<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-05 09:42:10 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>103</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e69c4c3aa7ac04b1dca84c51fe6263d52efa060" target='_blank'>
              Breast cancer through the lens of whole transcriptome spatial imaging
              </a>
            </td>
          <td>
            Claire Williams, Yi Cui, Michael Patrick, Giang T Ong, Terence Theisen, Megan Vandenberg, Joseph M. Beechem, Patrick Danaher
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Melanoma outcomes have dramatically improved over the past decade, but some patients still experience disease recurrence, particularly those who present at later stage of disease. Prior studies identified an altered immune microenvironment in the sentinel lymph node (SLN) including the presence of dysfunctional CD8 T cells and CD4 T regulatory cells (Tregs), but the spatial organization of the SLN and the interactions of individual cell types have not been extensively studied. To understand how spatial organization of immune cells in the SLN is related to outcomes, we performed spatial proteomic profiling of Stage I and II melanoma SLN to generate an atlas of cell types. Following multiplexed immunofluorescence imaging, deep learning-based segmentation and clustering, we identified 33 subsets of T cells, B cells and Tumor cells. Lymphoid region analyses established a foundational spatial map of the melanoma SLN, revealing higher regional frequencies of activated and memory CD4 T cells in Stage I SLN and consistent positioning of T cells at functional spatial locations. To better understand cellular interactions, we evaluated the immediate neighbors around each index cell using Effect Size Interaction mapping (ESI-map), a novel computational toolkit for understanding spatial interactions. Nearest-neighbor analyses across biological replicates revealed rewiring in cell-cell interacting pairs across disease progression and patient outcomes, particularly with respect to exhausted TOX+ CD8 T cells. Stage II patients who experienced disease recurrence had notable enrichment of cellular interactions between Tregs and TOX+ CD8 T cells that was associated with reduced expression of granzyme B and interferon-γ that was most pronounced in exhausted CD8 T cells. Together, these data demonstrate that the specific spatial interactions between Tregs and exhausted CD8 T cells in the SLN provide a novel immune signature for understanding melanoma outcomes. One Line Summary Treg-CD8 T cell interactions in the sentinel lymph node predict recurrence outcomes in Stage II SLN+ melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927407a38e9474aa6152c45daf9e0743472dc6ee" target='_blank'>
              Cellular interactions in the sentinel lymph node predict melanoma recurrence
              </a>
            </td>
          <td>
            Sabrina M. Solis, Yang Yang, Yee Hoon Foong, Anushka Dheer, K. Ma, M. S. Farooq, Shira G. Rosenberg, Sophie L. Gray-Gaillard, G. Karakousis, Xiaowei Xu, Rami S. Vanguri, Alexander C. Huang, R. Herati
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d2c494a1cd657524f51f9577b21d3baa396ea7" target='_blank'>
              Spatiotemporal regulation of cell cycle states within the complex tumor microenvironment
              </a>
            </td>
          <td>
            Gianni Zanardelli, Olivier Tassy, Maulik K. Nariya, Nacho Molina
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8ce4a4012ffb45da6b02201b0801273b5519b7e" target='_blank'>
              Integrative single-cell and spatial mapping of oxidative stress response uncovers GCLC+ mesenchymal tumor cell state linked with favorable outcomes in triple negative breast cancer
              </a>
            </td>
          <td>
            Nomeda Girnius, Tuulia Vallius, Wenqing Chen, I.-M. Launonen, S. Palomino-Echeverría, Jia-Ren Lin, Caitlin E. Mills, Silja Kauppila, Pauliina Kronqvist, A. Ellonen, Merja Perala, Eloise Withnell, Yu-An Chen, M. Secrier, S. Santagata, P. Sorger, Anniina Farkkila
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, I. Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatial transcriptomics has emerged as a transformative technology in biomedical research, offering unprecedented insights into gene and protein expression within their native tissue context. Unlike conventional bulk or single-cell sequencing approaches, spatial omics has the advantage of preserving the spatial structure of tissues, allowing researchers to directly map molecular information onto histological structures. This review provides an overview of the current state of spatial omics technologies, highlighting their application in cancer research. Spatial omics has enabled detailed characterization of the tumor microenvironment (TME), revealing spatial heterogeneity, immune cell infiltration patterns, and complex mechanisms of tumor progression and therapy resistance across various cancer types. The review covers future directions, including artificial intelligence–driven analytics, improved standardization, and cost reduction to accelerate clinical translation. Ultimately, spatial omics is poised to play a central role in precision oncology, enabling a deeper understanding of tumor biology and informing more effective individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a77ce136fd5fd93d240b66bc2895fcb841db1de" target='_blank'>
              Spatial omics: applications and utility in profiling the tumor microenvironment
              </a>
            </td>
          <td>
            Ji-Eun See, Sarah Barlow, Wani Arjumand, Hannah DuBose, Felipe Segato Dezem, Jasmine T. Plummer
          </td>
          <td>2025-12-01</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad9383618813c7fb20bfa893bf5368770766ebe" target='_blank'>
              Analysis of intratumoral immune heterogeneity reveals spatial organisation of immunosuppression in breast cancer
              </a>
            </td>
          <td>
            Christina Metoikidou, P. Bonté, Vadim K. Karnaukhov, Laëtitia Lesage, L. Djerroudi, André Nicolas, P. Sirven, A. Vincent-Salomon, Olivier Lantz, Emanuela Romano, Sebastián Amigorena
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Protein spatial imaging, rooted in spatial proteomics, enables the localization of proteins within their native environment, thus preserving structural and functional context. Its integration into cancer research has provided insights into tumor progression, especially in breast cancer, where ductal carcinoma in situ (DCIS) may transition into invasive breast cancer (IBC). Multiplexed ion beam imaging by time-of-flight (MIBI-TOF) has emerged as a powerful method to visualize protein distribution, tumor–stroma interactions, and immune cell dynamics. Evidence indicates that the tumor microenvironment (TME)—including stromal architecture, extracellular matrix (ECM), and immune cell infiltration—plays a central role in malignant progression. Here, we highlight recent advances in protein spatial imaging, discuss optimization strategies, and examine its implications for breast cancer progression and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a7eef8cc6668ecc3d6de49f721002ee2756192" target='_blank'>
              Spatial Proteomics Imaging to Decode Disease Progression: A Perspective on Tumor Microenvironments
              </a>
            </td>
          <td>
            Haonan Zhang
          </td>
          <td>2025-11-30</td>
          <td>Clinical Medicine and Pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Recent advances in spatial omics have revolutionized our ability to decode cell-positioning dynamics within three-dimensional (3D) culture models, such as organoids, tumor spheroids, and 3D bioprinting constructs, which faithfully mimic in vivo tissue architecture. By “spatially encoding” high-dimensional molecular information while preserving native microenvironmental context, spatial transcriptomics, proteomics, metabolomics, and epigenomics provide unprecedented maps of gene, protein, metabolite, and chromatin landscapes. When integrated with 3D culture systems, these approaches enable real-time visualization of how cells interact, self-organize, and respond to local biochemical and biophysical cues. Notably, spatial profiling of tumor spheroids has revealed discrete gene-expression gradients and region-specific metabolic heterogeneity, illuminating mechanisms by which the tumor microenvironment (TME) drives therapeutic resistance and immune evasion. Despite these insights, systematic integration of complex multimodal datasets and their translation into clinically actionable biomarkers remain formidable challenges. Emerging high-resolution spatial-omics platforms, which paired with precision-engineered 3D models such as bioprinted tissues and microfluidic organ-on-chip devices are beginning to bridge these gaps by enabling multiplexed, longitudinal analyses under physiologically relevant flow and mechanical stimulation. Looking ahead, ongoing improvements in imaging resolution, probe multiplexing, computational data-fusion, and standardized analytical pipelines are poised to deepen our understanding of tissue patterning, disease progression, and the spatial dynamics of therapeutic intervention. This review uniquely focuses on the seamless integration of cutting-edge spatial-omics technologies into 3D culture models, a synergy that provides unprecedented maps of gene, protein, metabolite, and chromatin landscapes while preserving native microenvironmental context, which will yield more predictive in vitro systems and accelerate the discovery of personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0a4b80969db3d8b9adfef962f9a44164de3a6d4" target='_blank'>
              Spatial omics in 3D culture model systems: decoding cellular positioning mechanisms and microenvironmental dynamics
              </a>
            </td>
          <td>
            Liwei Du, Huayu Yang
          </td>
          <td>2025-11-26</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="While advances in single-cell genomics have helped to chart the cellular components of tumor ecosystems, it has been more challenging to characterize their specific spatial organization and functional interactions. Here, we combine single-cell RNA-seq, spatial transcriptomics by Slide-seq, and in situ multiplex RNA analysis to create a detailed spatial map of healthy and dysplastic colon cellular ecosystems and their association with disease progression. We profiled inducible genetic CRC mouse models that recapitulate key features of human CRC, assigned cell types and epithelial expression programs to spatial tissue locations in tumors, and computationally used them to identify the regional features spanning different cells in the same spatial niche. We find that tumors were organized in cellular neighborhoods, each with a distinct composition of cell subtypes, expression programs, and local cellular interactions. Comparing to scRNA-seq and bulk RNA-seq data from human CRC, we find that both cell composition and layout features were conserved between the species, with mouse neighborhoods correlating with malignancy and clinical outcome in human patient tumors, highlighting the relevance of our findings to human disease. Our work offers a comprehensive framework that is applicable across various tissues, tumors, and disease conditions, with tools for the extrapolation of findings from experimental mouse models to human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af74f8f0f1a2428bab07b6671a3e436ab63d622f" target='_blank'>
              Spatially defined multicellular functional units in colorectal cancer revealed from single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            I. Avraham-Davidi, Simon Mages, J. Klughammer, Noa Moriel, Shinya Imada, Matan Hofree, Evan Murray, Jonathan Chen, Karin Pelka, Arnav Mehta, Genevieve M Boland, T. Delorey, Leah Caplan, Danielle Dionne, Robert Strasser, Jana Lalakova, Anežka Niesnerová, Hao Xu, Morgane Rouault, I. Tirosh, Hacohen Nir, Fei Chen, Omer Yilmaz, Jatin Roper, O. Rozenblatt-Rosen, Mor Nitzan, A. Regev
          </td>
          <td>2025-12-12</td>
          <td>eLife</td>
          <td>0</td>
          <td>197</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f7dcdd3510de0385797859e77f339a9cbb0192e" target='_blank'>
              Multidimensional Single-Cell Transcriptomic Profiling of Uterine Leiomyosarcomas Identifies Molecular Subtypes with Distinct Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Maria Korah, J. Agolia, Biren Reddy, R. Flojo, Amanda Gonçalves, Lilin Wang, Wesley Bobst, Brooke Liang, Kaylin A. Yip, Rosyli Reveron-Thornton, Chuner Guo, Robert Gruener, Andrea E. Delitto, Peter Y. Xie, Angela D. Tabora, Beatrice J. Sun, Sara K. Daniel, Babak Litkouhi, Oliver Dorigo, Nam Q. Bui, Minggui Pan, E. Moding, A. Kalbasi, Amanda R. Kirane, George A. Poultsides, Jeffrey A. Norton, Aaron M. Newman, G. Charville, K. Ganjoo, Deshka S. Foster, R. Stephanie Huang, Byrne Lee, Michael T. Longaker, Daniel J. Delitto
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7c241dbb9b8047ecee9d971f81cea0a1e193e3" target='_blank'>
              Multi-omic profiling provides insights into the heterogeneity, microenvironmental features, and biomarker landscape of small-cell lung cancer.
              </a>
            </td>
          <td>
            Mingchao Xie, Miljenka Vuko, Shashank Saran, Siyu Liu, Andrew G Chambers, Hana Baakza, Helen K Angell, Felicia Ng, C. Gay, R. Cardnell, F. Segerer, Alma Andoni, Jaime Rodriguez-Canales, Paul M Waring, Markus Schick, J. C. Barrett, L. Byers, Giulia Fabbri
          </td>
          <td>2025-12-02</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf21fbcae37403e9935121738032d46b9b3bab3c" target='_blank'>
              ProMPt: A modular preclinical platform for functional modelling of prostate cancer heterogeneity and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Nicole Pandell, Jacob Househam, Matteo Tartagni, Archana Thankamony, Florian Gabel, Roberto Rota, Elizabeth Flittner, B. Gurel, I. Figueiredo, G. Seed, A. Neeb, Mary Chol, W. Yuan, J. Clohessy, C. Tape, A. Sharp, J. D. De Bono, Marco Bezzi
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076f7d290241ef13795e81746d2f0f634b36afeb" target='_blank'>
              Star-Motifs: Revealing Single-Cell Spatiotypes from Routine Histology Using Star-Convex Neighborhoods
              </a>
            </td>
          <td>
            G. N. Gunesli, Muhammad Dawood, Lawrence S. Young, F. Minhas, S. E. A. Raza, N. Rajpoot
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background: Intratumour heterogeneity (ITH) is one of the key characteristics of cancer and is closely associated with patient prognosis, treatment resistance, and tumor metastasis. Nevertheless, the study of ITH in hepatocellular carcinoma (HCC) remains limited. Methods: The present study elucidated the influence of ITH on the tumor microenvironment (TME) in HCC. We applied Non-negative Matrix Factorization (NMF) analysis to a cohort of 78 single-cell RNA sequencing (scRNA-seq) HCC samples to systematically characterize ITH. Furthermore, by integrating spatial transcriptomics (ST) data from five HCC patients, we comprehensively analyzed the spatial organization and functional properties of distinct niches within HCC. We conducted a detailed analysis of the cell-type co-localization relationships within the TME and constructed a comprehensive atlas of HCC spatial organization. Results: We observed a co-localization relationship between hypoxia tumor cells, plasmalemma vesicle-associated protein (PLVAP+) endothelial cells (EC), and vascular endothelial growth factor A (VEGFA+) cancer-associated fibroblasts (CAF), suggesting a key role for hypoxia tumor cells in VEGFA+ CAF transformation and tumor angiogenesis. We identified a unique boundary region enriched with dendritic cells1 (DC1), interferon-expressing tumor cells, lymphatic EC, C–X–C Motif Chemokine Ligand 10 (CXCL10+) macrophages (Mac), and secreted phosphoprotein 1 (SPP1+) Mac located between the tumor-infiltrating immune cells and tumor regions. Furthermore, we found that CXCL10+ Mac and SPP1+ Mac, despite co-localizing in the boundary region, exhibit distinct functions, which may be attributed to their unique spatial locations, with the former being closer to the immune-infiltrated region and the latter more proximal to the tumor area. Conclusions: Our study highlights the critical role of spatial interactions between tumor cells and the microenvironment in HCC. The findings offer new insights into ITH and underscore the importance of spatial organization in understanding cancer biology and designing future precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e826ac6739dc6ba19b8b336aa7292a375efd8d3e" target='_blank'>
              Integrative Analysis of Single-Cell and Spatial Transcriptomics Reveals Intratumor Heterogeneity Shaping the Tumor Microenvironment in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-12-01</td>
          <td>Livers</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Multiplex imaging technologies have revolutionized our ability to study cellular behavior within the tissue microenvironment. Translating this complex data into meaningful biological insights requires a unified analytical framework. To address this, we developed SPACEMAP (Spatial Phenotyping And Classification with Enhanced Multiplex Analysis Pipeline), a comprehensive Python and Qupath-based platform for multiplex imaging analysis. SPACEMAP integrates image registration, segmentation, artifact removal, tissue and zone classification, spatial feature extraction, and a consolidated phenotyping approach into a single system. A core feature of SPACEMAP is its high-fidelity phenotyping. To evaluate classification performance, we benchmarked our method RESOLVE, against three established approaches, Leiden clustering, Self-Organizing Maps, and SCIMAP revealing substantial disagreement among them. SPACEMAP overcomes this through two complementary workflows: a machine learning model trained on expert-labeled cells, and a consensus classifier that integrates high-confidence cells across methods. Here, we validated SPACEMAP on in-house colorectal cancer samples and a public dataset, demonstrating its robustness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58864e00b009be9ccae3b0a7a4dc1ad7304393b8" target='_blank'>
              Resolving phenotyping discordance with SPACEMAP, an integrated machine learning framework
              </a>
            </td>
          <td>
            Bassel Dawod, Arely Perez Rodriguez, S. Diegeler, E. Elghonaimy, Megan Wachsman, Purva Gopal, David Hein, Paul H. Acosta, Andrew Jamieson, G. Danuser, Robert D. Timmerman, Satwik Rajaram, Todd A. Aguilera
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern, and a high relapse rate. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, where malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, we identify pericytes as the most active paracrine signaling hub within the tumor parenchyma. Their depletion through genetic engineering results in accelerated tumor progression and shortened survival. Mechanistic studies reveal that pericyte deficiency remodels the endothelium and impacts the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, the pericyte-deprived endothelium recruits perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages express Hepatocyte Growth Factor, which reinforces activation of its receptor tyrosine kinase MET on GBM cells harboring a pronounced mesenchymal subtype driven by the key phenotypic regulator Fosl1. Indeed, orthotopic implantation of MET-expressing GBM cells corroborates their superior tumor-initiating and invasive capabilities. Thus, pericytes represent critical modulators of GBM development by orchestrating a tumor-suppressive microenvironment, highlighting the importance of their preservation in therapy. Improved understanding of the tumor ecosystem in glioblastoma is critical for developing new treatment strategies for the disease. Here, the authors identify pericytes as an active paracrine signaling hub within the tumor parenchyma that orchestrates a tumor-suppressive microenvironment, suggesting pericyte preservation as a key feature of future therapeutic regimens for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c09a05790107e087a632df06714ceec3176e20b4" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, Mehrnaz Safaee Talkhoncheh, B. Phung, E. Cordero, R. Rosberg, E. Johansson, G. Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal cancer (CRC) brain metastases have a poor prognosis and limited treatment options, including resistance to radiation therapy. Little is known about the molecular and cellular mechanisms that enable CRC tumor cells to adapt to the brain and establish a supportive tumor microenvironment. To address this gap we used spatial transcriptomics to analyze 51 CRC brain metastases. A subset had matched primary colon tumors and longitudinally paired metastatic resections before and after radiation treatment. We identified the critical spatial cellular features of the tumor epithelium and the surrounding tumor microenvironment that support metastatic growth in the brain. CRC brain metastases developed a stromal microenvironment with abundant fibroblasts and tumor-associated macrophages. A fibroblast–macrophage cellular neighborhood promoted angiogenesis, extracellular matrix remodeling, and immune suppression. Tumor cells showed local adaptations. In endothelial-rich regions, they were proliferative whereas in macrophage-rich regions, they were more differentiated and immune evasive. Compared with paired primary tumors, CRC brain metastases showed increased chromosomal instability, with activation of RNA-processing, stress response, and junctional remodeling pathways. After radiation treatment, resistant clones had increased epithelial–mesenchymal transition, while the immunosuppressive stroma remained intact. We identified tumor-derived MIF, GDF15, PRSS3 and SEMA3C ligands and macrophage-derived SPP1 that have the potential to affect multiple cell types in the metastatic niche. These ligand-receptor interactions drive angiogenesis, stromal activation and immune suppression. In a macrophage–tumor–fibroblast co-culture model, knockout of SPP1 in macrophages led to reduced expression of lipid-metabolism related genes and disrupted tumor-promoting interactions. Together, these results indicate that CRC growth in the brain is sustained by a specific cellular organization with immunosuppressive multicellular interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121b9f0cf30a6fb662f64416fefc808a183dcc03" target='_blank'>
              Colorectal cancer relies on an immunosuppressive cellular topography and genomic adaptations for establishing brain metastases
              </a>
            </td>
          <td>
            A. Sathe, Mengrui Zhang, Xiangqi Bai, Ji In Kang, Rithika Meka, Huiyun Sun, Susan M. Grimes, Aparajita Khan, Mingen Liu, A. Luksik, M. Lim, Claudia K. Petrisch, Christopher M. Jackson, Hannes Vogel, Jeanne Shen, M. Gephart, Summer S. Han, Hanlee P. Ji
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafddf4eb7b4f74e9d7699b13e08df2afba97c1e" target='_blank'>
              Pan-cancer tumor classification by a holistic tumor microenvironment atlas
              </a>
            </td>
          <td>
            Shishang Qin, Xiao Du, Jinhu Li, Nan Jiang, Tian Diao, Yufei Bo, Qinhang Gao, Liangtao Zheng, Xinnan Ling, Qianqian Gao, Xiangjie Li, Sen Gao, Fei Tang, Wenjie Zhang, Chenwei Li, Peihong Fang, Linna Zhu, Dongfang Wang, Zemin Zhang
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Tumor heterogeneity drives drug resistance and relapse, influencing immune evasion and tumor progression. While intratumor heterogeneity has been extensively studied at the genomic level, its functional outcomes and interactions with the tumor microenvironment remain underexplored. In contrast, the functional outcome of heterogeneity and the interplay with the tumor microenvironment have not been addressed. In this study, we integrate multi-region spatial MS-based proteomics of 280 tumor regions, exome sequencing, and imaging to investigate spatial proteomic heterogeneity in breast cancer. Our findings reveal increased proteomic heterogeneity with tumor progression, independent of genomic heterogeneity but closely associated with microenvironmental differences. Integration with immune and stromal imaging highlighted a dynamic interplay where low-grade tumors exhibit constrained immune infiltration, and upon progression to higher grades, macrophages and T cells infiltrate. However, anti-inflammatory pathways involving kynurenine and prostaglandins are more highly expressed in infiltrated regions, suggesting that anti-tumorigenic activities are inhibited. Integration with the global protein network provides potential targetable mediators of immune evasion in breast cancer that can serve as the basis for future development of personalized breast cancer therapies. Intratumor heterogeneity (ITH) has been mainly studied at the genomic level. Here, the authors integrate multi-region proteomics of 280 tumor regions from 33 patients, exome sequencing, and imaging-based immune and stromal profiling to investigate functional spatial proteomic ITH in breast cancer and the interactions of cancer cells and the tumor microenvironment during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d48d5d89f33d4d9425895ba820536d951749d4" target='_blank'>
              Integrated spatial proteomic analysis of breast cancer heterogeneity unravels cancer cell phenotypic plasticity
              </a>
            </td>
          <td>
            M. Mardamshina, Shiri Karagach, Vishnu Mohan, Gali Arad, Daniela Necula, O. Golani, Liat Fellus-Alyagor, Anjana Shenoy, Kateryna Krol, Daniel Pirak, Nitay Itzhacky, I. Marin, Bruria Shalmon, Y. Addadi, Roded Sharan, E. Gal-Yam, Iris Barshack, Tamar Geiger
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e6a0c7866dff4b99709467cf25215e8947b79ac" target='_blank'>
              Coevolution of neoplastic and non-neoplastic reactive astrocyte states converges on mesenchymal-like and injury-response programs during murine glioblastoma progression and post-radiotherapy recurrence
              </a>
            </td>
          <td>
            D. Lindgren, R. Rosberg, K. Smolag, E. Johansson, D. Manou, J. Sjölund, S. Braun, B. Phung, K. Harbst, P. Jeannot, Maria Malmberg, C. Ceberg, G. Jönsson, Kristian Pietras, A. Pietras
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neuroblastoma (NB), the most common extracranial solid tumor in children, is characterized by remarkable cellular heterogeneity and clinical variability ranging from spontaneous regression to aggressive progression and relapse. Despite advances in multimodal therapies, including surgery, chemotherapy, radiotherapy, differentiation therapy, and immunotherapy—treatment resistance remains the principal barrier to improving survival in high-risk patients. Recent single-cell and spatial multi-omics studies have revolutionized our understanding of NB by revealing its developmental origins, lineage hierarchy, and adaptive evolution under therapeutic pressure. These technologies have delineated distinct cellular states along an adrenergic–mesenchymal continuum and uncovered the dynamic interplay between tumor cells and their microenvironment. Genetic instability, epigenetic reprogramming, and metabolic plasticity cooperate with immune and stromal remodeling to drive tumor persistence and relapse. At the molecular level, mechanisms such as MYCN-driven chromatin remodeling, super-enhancer reorganization, bypass signaling activation, quiescent persister programs, immune checkpoint engagement, and metabolic rewiring collectively enable therapeutic escape. Importantly, these processes are reversible, highlighting tumor plasticity as both a hallmark and a potential vulnerability of NB. Integrating single-cell transcriptomics, epigenomics, and spatial profiling provides an unprecedented framework to map resistance evolution, identify lineage-specific vulnerabilities, and guide rational combination strategies. Targeting epigenetic regulators, metabolic checkpoints, and immune suppressive networks in a temporally coordinated manner holds promise for converting NB from an adaptive to a controllable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance
              </a>
            </td>
          <td>
            Guoqian He, Si-Jia He, Xiao-yu Jing, Yi-Ling Dai, Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a5fb32298b06f7826592aac1dafed7fb246a78" target='_blank'>
              Genotype-phenotype correlations and de novo induction of cancer stem cells in Wilms tumor initiation
              </a>
            </td>
          <td>
            N. S. Pop, D. Koot, C.M. Brouwers, M. Linssen, J. Claassens, C.W.J. Cartlidge, D.D. Özdemir, K. S. Dolt, P. Hohenstein
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bdb3fc96051ec080fed6bcf88bde02391e145c9" target='_blank'>
              Integrative spatial multi-omics reveals prognostic tumor niches in female genital tumors
              </a>
            </td>
          <td>
            Tian Xu, Siyu Lin, Yuling Wu, Yufeng He, Zongxu Zhang, Yanping Zhao, Peng Zhang, Luyang Zhao, Wen Yang, M. Ye, Rui Zhang, Liang Wen, Pengfei Ren, Ce Luo, Zhe Zhang, Zexian Zeng, Yuanguang Meng
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb72dd83254c43d1127b19ea69b553e5fd51c4d" target='_blank'>
              TMPO promotes cellular dissemination and metastasis in circulating tumor cells
              </a>
            </td>
          <td>
            A. Giacobbe, Aleksandar Z Obradovic, Jinqiu Lu, Soonbum Park, Carlos Pedraz-Valdunciel, G. Fanelli, Aunika Zheng, Jaime Y Kim, Maya Stella Dixon, Jung Seung Nam, Florencia Picech, Caroline J. Laplaca, R. Virk, Matteo Di Bernardo, Alexander Chui, J. M. Arriaga, Stephanie N Afari, Francisca Nunes de Almeida, Min Zou, Helen Garcia, Brian D. Robinson, Hongshan Guo, S. Maheswaran, D. Haber, D. Miyamoto, D. Nanus, S. Tagawa, Tian Zheng, Massimo Loda, I. I. C. Chio, Michael Shen, P. Giannakakou, Andrea Califano, Peter A. Sims, Cory Abate-Shen
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb4486dfc086ad91b487a1c5269e719293310a3" target='_blank'>
              Machine perfusion and single-cell spatial transcriptome mapping identifies novel immune escape mechanisms in colorectal cancer liver metastasis
              </a>
            </td>
          <td>
            Peter Kok-Ting Wan, Ranchu Cheng, David Johnson, Carl Lee, Shihong Wu, Areeb Mian, Ahmet Hazini, Sorayya Moradi, Kate Friesen, Flurin Caviezel, Syed Hussain Abbas, Hatem Sadik, Adil Lakha, Keaton Jones, Girishkumar Kumaran, M. Bottomley, Rob Jones, Mark C. Coles, R. Bashford-Rogers, Constantin C. Coussios, Len Seymour, Robert Carlisle, Kerry Fisher, Alex Gordon-Weeks
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, M.-R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICI) have demonstrated prolonged efficacy in certain melanoma patients, yet a significant portion of patients do not experience clinical improvement, with the mechanisms underlying this resistance still not fully understood. Using established cell markers, we partitioned the single‐cell transcriptome into clusters, finding a notable link between NK cells and patient response to immunotherapy. We further identified four distinct subpopulations of NK cells, profiling marker gene sets and unique biological functions associated with each subpopulation. This analysis provides insights into the trajectory of NK cell development and differentiation, along with identifying the transcription factors driving these processes. The study pinpointed NK cluster 01 as pivotal in influencing patient sensitivity and prognosis during immunotherapy. Single‐cell transcriptome and spatial transcriptomics (ST) analysis revealed the proximity of NK cluster 01 cells to melanoma cells, hinting at a potential regulation of cell‐cell interaction via the IFN‐II signaling pathway network. ST analysis revealed the spatial arrangement and interaction of NK cluster 01 cells with melanoma cells. This study explores the feasibility of targeting NK cluster 01 cells with small molecule drugs via molecular docking, offering a promising approach to bolster the clinical utility of NK cell therapy. We comprehensively analyze the heterogeneity of NK cells within melanoma, elucidate the potential regulatory interactions between NK cells and other microenvironmental components, and establish a basis for the future clinical utilization of distinct NK cell subsets as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fbdc1aaba32b7600dd3221ae61c5e31ee96bd3" target='_blank'>
              Integrating Single‐Cell and Spatial Transcriptomics Reveals NK Cell Subpopulations Associated With Immunotherapy for Melanoma
              </a>
            </td>
          <td>
            Zhicheng Hu, Yongfei Chen, Hao Yang, Qiuming Pan, Hongrui Li, Shuting Li, Junxi Wang, Yudi Huang, Guanglong Huang, Shanqiang Qu
          </td>
          <td>2025-12-01</td>
          <td>Smart Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Breast cancer (BC) prognosis is fundamentally dictated by its pronounced molecular and cellular heterogeneity. While single-cell RNA sequencing (scRNA-seq) has detailed the tumor microenvironment (TME) cellular composition, a critical knowledge gap persists regarding the vascular heterogeneity of tumor endothelial cells (TECs) across different BC subtypes. Current TEC classification remains conventional, obscuring whether subtype-specific TECs possess unique transcriptomic signatures that influence treatment response, particularly to anti-angiogenic therapies. Given the pivotal role of tumor vasculature in proliferation and metastasis, and the existing research bias towards Triple-Negative BC (TNBC), a comprehensive investigation into the complexity of the vascular system is warranted. Furthermore, as lymph node metastasis (LNMT) is a major determinant of cancer mortality, distinguishing the endothelial and immune cell subtypes within LNMT microenvironments from primary tumors is essential for clarifying pathogenesis. Methods To address this, we integrated scRNA-seq data from primary BC tumors, ER_LN and healthy tissues, generating a robust dataset of 98,000 cells (~ 12 samples). Employing single-cell transcriptomics, spatial transcriptomics, and immunohistochemistry, we precisely delineated the endothelial cell (EC) heterogeneity within matched tumor and peri-tumoral tissues and scrutinized their intricate interactions with immune populations. Results We identified two previously uncharacterized, tumor-enriched endothelial cell subtypes, designated EC4 and EC5, which demonstrate subtype-specific functional adaptations and prognostic significance. EC4 cells, highly prevalent across BC, are principally characterized by antigen presentation, immune cell recruitment, and pro-inflammatory signaling. Conversely, EC5 cells, also enriched in BC, exhibit robust extracellular matrix (ECM) remodeling and potent tumor angiogenesis. We further characterized the functional divergence of EC4 and EC5 within ER tumors relative to HER2 tumors and ER_LN metastases. Our analysis reveals conserved endothelial programming mechanisms across BC subtypes, coexisting with distinct TME-driven transcriptional adaptations. Importantly, interactome analysis highlighted novel and subtype-specific communications between these novel EC subsets and immune cells, particularly CD8+T cells and macrophages. Experimental validation demonstrates that ECs overexpressing APP can mediate the M2 polarization of macrophages, underscoring diverse immunomodulatory roles for EC subsets across different BC contexts. Conclusions These findings offer critical and granular insights into the complex interplay between novel EC subtypes and the immune microenvironment in BC progression and metastasis. We establish that ECs are active and heterogeneous modulators of the TME, identifying specific therapeutic vulnerabilities within the tumor vasculature and providing a foundational blueprint for developing future precision immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67936270be24af1f65608417190d3a4e352d6fe3" target='_blank'>
              Single-cell profiling identifies heterogeneity of the immune microenvironment in healthy, primary and lymph node metastatic BC
              </a>
            </td>
          <td>
            Chuyu Liu, Feng Zeng, Shanshan Gao, Nengying Zhang, Yulan Cai
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, Christin-Therese Müller, Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the leading causes of cancer-related morbidity and mortality worldwide. Tumor sidedness, distinguishing left- (LCRC) and right-sided (RCRC) cancers, has emerged as a critical clinical determinant, influencing patient prognosis and therapeutic response. However, the cellular and molecular mechanisms underlying these differences remain poorly understood. Recent advances in single-cell RNA sequencing (sc-seq) provide high-resolution insights into CRC heterogeneity, revealing distinct tumor, immune, and stromal cell populations and their context-specific interactions. In this review, we synthesize sc-seq studies that dissect the molecular programs driving progression, therapy resistance, and metastasis in CRC. We highlight malignant subclusters characterized by metabolic reprogramming and spatially organized oncogenic signaling; specialized immune cell states, including macrophage subsets, exhausted T cells, and mast cells, that shape tumor immunity; and stromal elements such as cancer-associated fibroblasts and endothelial tip cells that remodel the extracellular matrix, promote angiogenesis, and foster immune evasion. Importantly, sc-seq demonstrates that LCRC and RCRC represent distinct multicellular ecosystems with differential immune recruitment and stromal signaling, underscoring the need for sidedness-informed therapeutic strategies. We propose that future interventions should target cell-cell communication networks and spatially defined tumor–microenvironment interactions to overcome heterogeneity and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03b9e3bca233f1317bcd2e2e43fb69a5fedfb984" target='_blank'>
              Sidedness matters: single-cell perspectives on left- and right-sided colorectal cancer
              </a>
            </td>
          <td>
            M. Yanova, Diana Maltseva, Alexander Tonevitsky
          </td>
          <td>2025-12-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a heterogeneous malignancy shaped by genetic alterations and immune microenvironmental interactions. While single-cell RNA sequencing (scRNA-seq) has uncovered diverse epithelial and T cell subsets, the spatial architecture of their crosstalk remains poorly understood. Methods We integrated scRNA-seq and spatial transcriptomics data from 36 CRC patients to comprehensively characterize epithelial and T cell heterogeneity, differentiation dynamics, and intercellular communication networks. Data preprocessing and integration were conducted using Seurat and Harmony. Cell trajectory inference was performed via CytoTRACE and Slingshot. Ligand-receptor signaling was assessed using CellChat, while spatial mapping was achieved using CellTrek. Results Nine epithelial and eight T cell subpopulations were identified, each exhibiting distinct transcriptional states, CNV burdens, and pseudotime trajectories. CD8⁺ T_GZMK cells displayed tumor-specific activation signatures and engaged in enriched communication with epithelial subsets through CEACAM, APP, and MIF signaling pathways. Spatial transcriptomics confirmed the in situ colocalization of CD8⁺ T_GZMK cells with PTP4A3⁺ epithelial cells in tumor regions, revealing spatially organized epithelial–immune niches. These interactions suggest potential mechanisms of immune modulation and tumor progression. Conclusion This study provides an integrative single-cell and spatial atlas of CRC, revealing structured epithelial–T cell communication and spatial architecture within the tumor microenvironment. Our findings offer novel insights into immune-epithelial crosstalk and identify signaling pathways that may serve as therapeutic targets or biomarkers for CRC precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6c4f2ec4971acec7d7b9addcd8348e71b0ec1ff" target='_blank'>
              Decoding epithelial-T cell interactions in colorectal cancer through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Rentao Yu, Xinyu Lu, Zhuangzhuang Yue
          </td>
          <td>2025-11-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The recruitment of the various types of cells into the tissue microenvironment and how these cells engage with other cells in the tissue sites play critical biological roles. However, it is difficult to study these processes on a genome-wide scale using traditional low-throughput experiments. To address this gap, we developed a fully interpretable multi-instance deep learner, coined “spacer”. Spacer digests the transcriptomics and spatial modalities of spatially resolved transcriptomics (SRT) data in alignment with the biological principles of the tissue spatial architecture. We deployed spacer to a panel of 17 high definition and 20 low definition SRT datasets, for studying how stromal and immune cells were recruited into tumors and heart during myocarditis. Spacer discovered novel biological insights not affordable by prior spatial data analysis tools, which were validated by orthogonal immuno-peptidomics, spatial T cell receptor (TCR) sequencing, and single cell sequencing experiments. We discovered genes that encode more immunogenic peptides and that are involved in developmental pathways are more potent in recruiting T cells to local tumor sites, while the recruitment of B cells and macrophages is stipulated by a different molecular program. On the other hand, expression of mucins in the tumor cells was found to repel T cell localization. For the engaging cell side, we also uncovered T cell-intrinsic features that determine their localization, validated by spatial-TCR-seq data. Spacer also unexpectedly revealed that CD4+ T cells, though fewer in numbers, are more responsive than CD8+ T cells in the heart during myocarditis. Collectively, this study establishes a new spatially resolved paradigm for studying cellular localization mechanisms in situ and shifts the paradigm from descriptive mapping to mechanistic discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e145cb49a8be785de3fc0ca3eaff00c47176e231" target='_blank'>
              An Interpretable Multi-instance Learner Decodes Cellular Recruitment from Spatially Resolved Transcriptomics
              </a>
            </td>
          <td>
            Jia Yao, Ji Seon Shim, Kelly Wong, James Zhu, Yunhe Liu, Sung Wook Kang, Kelli Bryan M. Burt, Jong Min Choi, Claire Lee, I. C. Julio, Luisa Maren Solis Soto, Sharia Hernandez, Karen Colbert, Wei Lu, Khaja B Khan, Paul F Mansfield, Hee-Jin Jang, Tao Yue, Changwei Li, Hao Zhu, Linghua Wang, Hyun-Sung Lee, Tao Wang
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor cell plasticity plays a key role in the progression of malignant neoplasms and therapy resistance, yet its spatial dynamics remain poorly understood. This study investigates the spatial distribution of epithelial–mesenchymal transition (EMT) and stemness, and their prognostic relevance in lung adenocarcinoma (LUAD). Single‐cell RNA sequencing was performed on 10 tumor fragments from three regions (core, core‐adjacent, and edge) of a single LUAD using a SURFSeq 5000 platform. Stemness potential was identified by CytoTRACE2, EMT by UCell and the Hallmark_epithelial_mesenchymal_transition signature, and tumor cells associated with disease prognosis by Scissor. LUAD displayed pronounced spatial heterogeneity of tumor cell plasticity. Stemness potential was heightened at the edge, while EMT was most prominent in the core. The correlation between EMT and stemness was slightly higher at the edge, though it remained negligible (R = 0.17, p = 5.5e−7). Higher EMT scores were observed in tumor cells associated with poor overall survival (r = 0.433, p < 2e−16), whereas stemness was prevalent in tumor cells associated with poor progression‐free survival (r = 0.251, p = 7.15e−6). Tumor cells associated with poor prognosis were enriched with the MYC‐targets v1 signature with PCBP1 and PA2G4 as the top contributing genes. Immunohistochemical data showed that PCBP1 and PA2G4 proteins predominantly localized within tumor cells of LUAD. Taken together, these findings highlight the spatial heterogeneity of tumor cell plasticity features in LUAD and uncover biomarkers associated with poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badf7545e441ad3330c93dcdb325456db2158597" target='_blank'>
              Spatial Dynamics of Tumor Cell Plasticity in Lung Adenocarcinoma Revealed by Region‐Specific Single Cell Transcriptomics
              </a>
            </td>
          <td>
            A. Khozyainova, Maxim E. Menyailo, Vera G. Subrakova, Dmitry M. Loos, P. Iamshchikov, Anton A Fedorov, M. S. Tretyakova, Ustinia A Bokova, Evgeny O Rodionov, Sergey V Miller, Evgeny B Topolnitsky, D. V. Piankov, Sergei A Korostelev, Evgeny V. Denisov
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Single-cell sequencing (scRNA-seq) has emerged as a pivotal technology for deciphering the complex cellular heterogeneity and tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC), positioning it as a critical tool for informing novel therapeutic strategies. This review explores how scRNA-seq reveals diverse cellular subpopulations and their functional roles within the PDAC TME, including malignant epithelial cells with transitional phenotypes, heterogeneous cancer-associated fibroblasts (CAFs), functionally distinct immune cells such as tumor-associated neutrophils (TANs) and macrophages (TAMs), and actively participating neural components like Schwann cells. These cellular constituents form specialized functional units that drive tumor progression, immune evasion, neural invasion, and therapy resistance through metabolic reprogramming, immunosuppressive signaling, and cellular plasticity. The review further examines technological advances in single-cell sequencing from 2023 to 2025, focusing on sample preprocessing innovations, multi-omics integration (combining transcriptomics with epigenomics and proteomics), spatial resolution enhancements, and customized computational tools that address PDAC-specific challenges. Clinically, single-cell sequencing enables precise cellular subtyping, identification of novel biomarkers, and development of personalized therapeutic approaches, including combination therapies targeting specific cellular subpopulations and their interactions. Despite these advances, significant challenges remain in standardizing clinical applications such as liquid biopsy for early detection and tumor microenvironment assessment for diagnostic staging, validating biomarkers like CLIC4, GAS2L1, Cytokeratins, Vimentin and N-cadherin in circulating tumor cells, and comprehensively integrating multi-omics data. Future research focusing on both technology refinement and biological validation will be essential for translating single-cell insights into improved diagnostic and therapeutic outcomes for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9576dbb6fe77bb5f04f2ad67b474589557936" target='_blank'>
              Single-Cell Sequencing Unravels Pancreatic Cancer: Novel Technologies Reveal Novel Aspects of Cellular Heterogeneity and Inform Therapeutic Strategies
              </a>
            </td>
          <td>
            Keran Chen, Zeyu Chen, Jinai Wang, Mo Zhou, Yun Liu, Bin Xu, Zhi-Jing Yu, Yiming Li, Guanhu Yang, Tiancheng Xu
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a05de91b68f189c5a7a8fad28c43d535bcdc834" target='_blank'>
              A spatial atlas of colorectal cancer reveals the influence of stromal niches on tumour differentiation
              </a>
            </td>
          <td>
            Andrew D. Pattison, Rebekah M. Engel, Wing Hei Chan, Spencer Greatorex, David Nickless, Liam Skinner, Camilla Cohen, Julie Hickey, Christine Georges, Anne L. Fletcher, P. McMurrick, Thierry Jardé, Lochlan Fennell, Helen E. Abud
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Breast cancer is composed of diverse cell populations, and this intratumoral heterogeneity profoundly affects clinical behavior. Here, we leveraged single cell RNA sequencing (scRNA-seq) of 68 breast cancer specimens to dissect tumor heterogeneity at high resolution. Unsupervised clustering identified all major cell types of the tumor microenvironment (TME)—including malignant epithelial cells, fibroblasts, T cells, macrophages, endothelial cells, and others—with striking variability in their proportions across molecular subtypes. For example, a BRCA1-mutant triple-negative breast cancer (TNBC) sample showed dense immune infiltration, whereas an estrogen receptor (ER)-positive tumor was mostly epithelial, consistent with known subtype differences in immunogenicity. We applied inference of copy number variations (inferCNV) to distinguish malignant epithelial cells, identifying ~90,000 tumor cells with significant copy-number aberrations enriched for cancer hallmark pathways. Re-clustering of these malignant cells revealed five discrete subpopulations. Notably, a KRT17-positive subcluster displayed the highest stemness score and a distinctive ETS-family transcription factor (ERG) regulon, suggesting a stem-like phenotype. Using The Cancer Genome Atlas (TCGA) cohort, we found that genes upregulated in this KRT17+ subpopulation, particularly NFKBIA, PDLIM4, and TCP1 stratified patient survival. An 8-gene risk signature derived from the KRT17 program segregated patients into high- and low-risk groups with markedly different outcomes. High-risk tumors were characterized by an immunosuppressive TME enriched in M2-like macrophages, whereas low-risk tumors more often harbored lymphocyte-predominant infiltrates. Focusing on TCP1, a chaperonin subunit upregulated in high-risk tumors, we demonstrate that TCP1 knockdown in breast cancer cell lines substantially impairs cell migration (~50% reduction in wound closure) and invasion (P < 0.01). These findings reveal functionally distinct malignant cell states within breast cancer and identify TCP1 as a promising therapeutic target to disrupt aggressive, stem-like tumor cell programs, ultimately guiding more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a4e16cf89ac6011cca0aca51da036b0952519" target='_blank'>
              Single-cell RNA-seq reveals breast cancer heterogeneity and identifies TCP1 as a therapeutic target in breast cancer
              </a>
            </td>
          <td>
            Houman Wu, Haiyang Du, Gao Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-12-09</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Thyroid cancer, the most common endocrine malignancy, is characterized by a unique and complex tumor microenvironment (TME). To unravel the high tumor heterogeneity and molecular mechanisms driving cancer progression, we performed single-cell RNA sequencing (scRNA-seq) analysis, enabling a comprehensive exploration of cellular diversity and molecular dynamics at single-cell resolution. We employed Principal Component Analysis (PCA) and Uniform Manifold Approximation and Projection (UMAP) for dimensionality reduction and subsequent identification of cellular clusters. Differential gene expression analysis across subclusters was conducted using the FindAllMarkers function, while the DoHeatmap function was utilized to visualize the distribution of differentially expressed genes. The AUCell algorithm was applied to evaluate pathway enrichment within specific cell subtypes. To decipher cellular communication networks, we integrated the CellChat and NicheNet algorithms, which revealed intricate intercellular signaling interactions. Finally, multiplex immunohistochemistry (mIHC) was performed to validate key cellular interactions identified in silico. By analyzing 405,077 single cells from 50 thyroid cancer samples (including papillary, anaplastic, and metastatic tumors) and 14 normal thyroid tissues, we identified four major cellular subpopulations through unbiased clustering based on gene expression patterns and representative cellular markers. The TME was found to encompass diverse immune, endothelial, and mesenchymal cell subtypes, including novel populations such as CD4 + HSPA1A + T cells. Functional pathway enrichment analysis highlighted the roles of abundant cell types in tumor progression. Cell-cell communication analysis uncovered potential immunotherapeutic targets and revealed critical crosstalk among hub niche cells, including APOE+ macrophages, EMT-like cancer-associated fibroblasts (CAFs), and RBP7+ endothelial cells. These findings were further validated by multiplex immunohistochemistry, confirming the spatial organization and interactions of these cell populations within the TME. Our study provides a comprehensive single-cell transcriptomic atlas of thyroid cancer, offering profound insights into tumor heterogeneity, the functional roles of key niche cells, and potential biomarkers for anticancer therapy. These findings not only enhance our understanding of thyroid cancer biology but also pave the way for the development of novel therapeutic strategies targeting the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b009366b4ed17ce1b669fc0314c882a610b52822" target='_blank'>
              Comprehensive single-cell RNA analysis reveals intertumoral microenvironment heterogeneity and hub niche of carcinogenesis in thyroid cancer
              </a>
            </td>
          <td>
            Hao Xu, Junjie Ma, Nanjun Li, Haoxiang Xuan, Shijie Lin, Yunuo Shen, Lu Yin, Ping Wang, Qianhui Xu
          </td>
          <td>2025-11-26</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumor vascular endothelial cells are essential constituents of the tumor microenvironment, responsible for delivering oxygen and nutrients that are vital for tumor growth and proliferation. As a hallmark of cancer progression, abnormal tumor vasculature contributes to tumor development through multiple mechanisms. Although anti-angiogenic therapies are widely used in the treatment of various cancers, the intrinsic heterogeneity of endothelial cells poses significant challenges regarding therapeutic efficacy. Therefore, further investigation into the heterogeneity of tumor endothelial cells is of paramount importance. The rapid advancement of single-cell sequencing technologies in recent years has facilitated the detailed characterization of heterogeneity among tumor endothelial cells at the single-cell level, thereby fostering a more precise understanding of the functional roles of individual cells within the tumor microenvironment. This technology has become an indispensable tool for investigating the heterogeneity of tumor endothelial cells, offering insights that could inform the refinement of future cancer treatments. In this review, we synthesize findings from the field of single-cell omics to elucidate the heterogeneous characteristics of tumor endothelial cells. We analyze recent advancements in single-cell technology used in the study of tumor cell heterogeneity in terms of both commonalities and distinctive features, covering aspects at the gene and cellular levels. In this review, we provide an overview of recent applications of single-cell sequencing technology in analyzing tumor endothelial cell heterogeneity, offering insights into the development of precise tumor therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a50144e46bbcc6cadd8fab9df5c73d57f4ff4f3" target='_blank'>
              Research Progress Regarding the Use of Single-Cell Sequencing Technology in Analyzing Tumor Endothelial Cell Pathophysiology
              </a>
            </td>
          <td>
            Shu Zhao, S. Liu, Wenxin Shao, Dong Liu
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54a8bb0d6bffac121a5e75b278c655207f421694" target='_blank'>
              Characterizing spatial functional microniches with SpaceTravLR
              </a>
            </td>
          <td>
            Koushul Ramjattun, Alyson Wang, Hannah Lee, Shilpi Giri, Yijia Chen, William A. MacDonald, Nathan D. Lord, Amanda C. Poholek, Youjin Lee, Jishnu Das
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor heterogeneity is a fundamental property of leukemia and lymphoma that drives disease progression, relapse, and therapy resistance. For decades, bulk genomic and transcriptomic analyses have provided a foundational understanding of these malignancies but have obscured the cellular diversity within the tumor ecosystem. The advent of high-throughput single-cell technologies, particularly single-cell RNA sequencing (scRNA-seq), has revolutionized our capacity to deconvolute this complexity. This review synthesizes how single-cell analyses are reshaping our understanding of tumor heterogeneity in hematological cancers. We explore the role of intra-clonal genetic diversity in Acute Myeloid Leukemia (AML) and B-cell Acute Lymphoblastic Leukemia (B-ALL), the functional and transcriptional plasticity of subpopulations, and the dynamic interplay between malignant cells and the immune microenvironment in both leukemia and lymphoma. We further highlight how the tumor microenvironment in lymphomas fosters heterogeneity and immune evasion. Furthermore, the integration of single-cell data with spatial transcriptomics and computational models is beginning to predict evolutionary trajectories, offering a window into future clinical behaviors. Critically, we discuss how single-cell approaches have identified pre-existing or emergent resistant subclones under therapeutic pressure, providing a mechanistic basis for treatment failure. The clinical implications are profound, paving the way for single-cell diagnostics for minimal residual disease (MRD) monitoring and the identification of novel therapeutic targets. We conclude that single-cell multi-omics is an indispensable tool for unraveling the multifaceted nature of tumor heterogeneity, with the potential to propel the field toward more precise and effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0baf5679635e6fcbb86a03a048b12087069db02" target='_blank'>
              Tumor Heterogeneity in Leukemia and Lymphoma: Insights from Single-Cell Analyses
              </a>
            </td>
          <td>
            Pang Hou, Wen
          </td>
          <td>2025-11-21</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ed421ed01090a9f895ebe8d07249f22a2618902" target='_blank'>
              SHEST: Single-cell-level artificial intelligence from haematoxylin and eosin morphology for cell type prediction and spatial transcriptomics reconstruction
              </a>
            </td>
          <td>
            Hoyeon Jeong, Junghan Oh, Donggeon Lee, Jae Hwan Kang, Yoon-La Choi
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastases represent one of the hardest obstacles in cancer treatment, accounting for many cancer-related deaths. Understanding the mechanisms that drive disease progression is essential to improve patient outcomes and develop more effective therapeutic strategies. This phenomenon appears to be elicited by dormant cancer cells (DCCs), which can persist undetected for extended periods of time, entering a non-proliferative, hibernation-like state that confers resistance to conventional therapies and facilitates immune evasion. Owing to their status as a rare and energy-restricted population, combined with the limitations of current medical imaging, DCCs often evade early detection, hindering timely intervention and effective clinical management. Consequently, a critical need exists to develop high-resolution detection systems and identify specific DCC targetable biomarkers. Here, a comprehensive overview of the current understanding of DCCs is reported, with a focus on recent advancements in experimental strategies for their identification and tracking, as well as therapeutic approaches currently under clinical investigation aimed at targeting these elusive cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9d13b10d52e65e6fb3d5e89c185ee227dee670" target='_blank'>
              Decoding cancer dormancy: integrative genomic, phenotypic and live-cell imaging analysis to reveal the hidden cancer cell reservoir
              </a>
            </td>
          <td>
            G. Porcelli, Caterina D’Accardo, Francesca Angeloro, Martina Cucchiara, Paola Bianca, Vincenzo Davide Pantina, Narges Roozafzay, Chiara Modica, Miriam Gaggianesi, Sebastiano Di Bella, G. Stassi, Alice Turdo, M. Todaro
          </td>
          <td>2025-11-12</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) enables the quantification of gene expression at single-cell resolution while preserving spatial context, but its utility is limited by small gene panels and challenges in accurate cell segmentation. To address these limitations, we present a graph autoencoder framework that integrates subcellular transcript distribution patterns with cell-level gene expression profiles for enhanced cell clustering in imaging-based ST (SPICEiST). The clustering performance of SPICEiST was systematically evaluated across several cancer datasets and gene panel sizes. The results demonstrate that SPICEiST consistently outperforms the conventional cell-level gene expression-based methods in distinguishing subtle differences in cell states, as measured by the number of cell clusters and clustering indices, such as the CHI and DBI. Moreover, the findings indicate that SPICEiST can further enhance the performance, even with advancements in cell segmentation, particularly for datasets with small gene panels. Overall, these improvements in cell clustering indices, CHI and DBI, were more pronounced in datasets with small gene panels of around 300 genes, in contrast to those with large panels containing over a thousand genes. Notably, SPICEiST also reveals more spatially intermixed and less compartmentalized cell clusters, a characteristic that better reflects the complex and heterogeneous nature of tumor microenvironments. This effect was especially evident in the datasets with large panels. These findings highlight the value of leveraging subcellular transcript patterns to overcome the inherent limitations of imaging-based ST, particularly for small gene panels, and may provide new insights into tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea647a2ed6ad5d25220e9cf9a6cab8978c9f7db5" target='_blank'>
              SPICEiST: subcellular RNA pattern enhances cell clustering of imaging-based spatial transcriptomics
              </a>
            </td>
          <td>
            Sungwoo Bae, Yuchang Seong, Dongjoo Lee, Hongyoon Choi
          </td>
          <td>2025-12-01</td>
          <td>Genomics & Informatics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/238054edb782d465f94eaf0c685da8f4229110b1" target='_blank'>
              An integrated platform for high-throughput phenospace learning of 3D multilineage organoid systems
              </a>
            </td>
          <td>
            Ryo Okuda, Christoph Harmel, Quan Xu, Héloise Mary, Philip Schulz, Linda Steinacher, Elisa D’Arcangelo, Bruno Gjeta, Marina Signer, Irineja Cubela, Marc Bickle, Matthias P. Lutolf, Lauriane Cabon, Ilya Lukonin, J. G. Camp
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cell state plasticity drives metastasis and therapy resistance in cancers. In melanoma, these behaviors map onto a melanocytic-to-mesenchymal-like continuum regulated by AP-1 transcription factors. However, how the AP-1 network encodes a limited set of discrete states, why their distributions vary across tumors, and what drives phenotypically consequential AP-1 state transitions remain unclear. We develop a mechanistic ODE model of the AP-1 network capturing their dimerization-controlled, co-regulated, competitive interactions. Calibrated to heterogeneous single-cell data across genetically diverse melanoma populations and combined with statistical learning, the model reveals network features explaining population-specific AP-1 state distributions. These features correlate with MAPK activity across tumor lines and with variability within clones, linking MAPK signaling to AP-1 states. The model predicts and experiments validate adaptive AP-1 reconfiguration under MAPK inhibition, inducing a dedifferentiated, therapy-resistant state that can be blocked by model-guided AP-1 perturbations. These results establish AP-1 as a configurable network and provide a computational framework for predicting and modulating AP-1 driven cell state plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80053bfbfe501e1acc3e16cb2b130e2f5342e9b1" target='_blank'>
              Cell state plasticity emerging from co-regulated, competitive, and configurable interactions within the AP-1 network
              </a>
            </td>
          <td>
            Yonatan N. Degefu, Magda Bujnowska, Douglas G. Baumann, Mohammad Fallahi-Sichani
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c38b6d1e04676ebe5c874a1af9317e2b5d3d46" target='_blank'>
              Single-cell Transcriptomic Profiling of Pancreatic Ductal Adenocarcinoma: Epithelial Reprogramming and Systemic Immune Exhaustion
              </a>
            </td>
          <td>
            Qinyu Xu
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background KRAS is one of the most frequently mutated genes in colorectal cancer (CRC) and plays a crucial role in tumorigenesis, progression, immune evasion, and treatment resistance. The pronounced heterogeneity within KRAS-mutant CRC highlights the urgent need for more precise and personalized therapeutic approaches. Methods To investigate this heterogeneity, we employed single-cell RNA sequencing and spatial transcriptomics to comprehensively characterize the tumor microenvironment of KRAS-mutant CRC. Data preprocessing and clustering were performed using Scanpy. Spatial cell-type deconvolution was conducted via Cell2location, whereas intercellular communication and spatial dependencies were analyzed using CellChat, MISTy, and stLearn. Results Our analyses revealed that KRAS-mutant tumor epithelial cells recruit Mono_S100A8 monocytes via the MDK_SDC4 signaling axis. Concurrently, surrounding Fib_CTHRC1 fibroblasts secrete collagen, which interacts with integrin receptors on KRAS-mutant epithelial cells and contributes to the exclusion of lymphocyte infiltration. Conclusion These cellular components collaboratively established an immunosuppressive spatial niche. These findings offer novel theoretical insights and potential targets for the development of immunoregulatory strategies tailored to KRAS-mutant CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059c9dfc013c2c370152c3fa0d39b4efc600e01e" target='_blank'>
              Single-cell and spatial transcriptome profiling identifies the immunosuppressive spatial niche in KRAS-mutant colorectal cancer
              </a>
            </td>
          <td>
            Sheng‐Li Yang, Chao Gu, Xinsheng Miao, Hao Zuo, Wei Xu, Yan Zhang, Wei Tang, Jianhua Zhu, Zheng Yuan, Xinhua Gu, Chenyi Zhong, Yueming Sun, Jiahui Zhou
          </td>
          <td>2025-12-01</td>
          <td>Journal for ImmunoTherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ffa550baca76e714d9b657ce67086b4b77a9db" target='_blank'>
              Deciphering Tumor Microenvironment Dynamics in Tumorigenesis and Lymph Node Metastasis of Esophageal Squamous Cell Carcinoma using Single-cell RNA Sequencing
              </a>
            </td>
          <td>
            Hansoll Na, T. Hong, Chung Lee, Y. Yang, Ha Eun Kim, Byung Jo Park, Hyun Ki Kim, Dae Joon Kim
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/465e60c37fcf50d018cbfc326971ebe12576816e" target='_blank'>
              Single-cell analysis reveals a universal pericyte signature associated with poor clinical outcome and immune T cell dysfunction in thyroid cancer and other cancers
              </a>
            </td>
          <td>
            Aarav Bhasin, Yubin Na, Jack Lawler, C. Nucera
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background: Despite the use of immunotherapy in esophageal squamous cell carcinoma (ESCC), treatment failure occurs occasionally in patients, yet the underlying mechanisms remain poorly understood. Methods: We conducted large-scale single-cell RNA sequencing (scRNA-seq) data analysis, which integrated seven independent datasets from 192 ESCC patients to yield over 440,000 high-quality single cells, to systematically characterize the tumor microenvironment (TME) landscape during ESCC progression and immunotherapy response. Additionally, we performed high-resolution spatial transcriptomics (stRNA-seq) using the 10x Visium HD platform on paired pre- and post-treatment tissues from two patients (one immunotherapy responder and one non-responder), which enhanced the findings from the scRNA-seq data and mapped therapy-induced TME at the spatial level. Multiplex immunohistochemistry was employed based on seven patients to confirm distinct patterns of intercellular crosstalk underlying differential therapeutic outcomes. Results: In scRNA-seq data, we found that B lineage cells were reduced during ESCC progression but were enriched in immunotherapy-resistant patients. Further analysis of malignant ESCC cells suggested that immunotherapy resistance might be associated with a subpopulation of tumor cells exhibiting aberrantly elevated cholesterol biosynthesis. Cell communication analysis of scRNA-seq and stRNA-seq data collectively revealed that immunotherapy resistance was linked to cellular crosstalk between cholesterol-biosynthetic tumor cells and germinal center (GC) B cells within tertiary lymphoid structures. Notably, single-cell, spatial data, and multiplex immunohistochemistry demonstrated that cholesterol biosynthesis-associated ESCC cells express elevated levels of MIF. This disrupts GC reactions by competing with the CXCL12-CXCR4 signaling axis via MIF-CXCR4 interactions, thereby impairing B cell-mediated immunity. Conclusions: MIF+ tumor cells in GCs may be a biomarker for predicting immunotherapy resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8a1ee6ff72d48f3e16c31d484505ab587cac16" target='_blank'>
              MIF-expressing tumor cells mediate immunotherapeutic resistance in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Jing Song, Xiaomei Song, Yue Xie, Hong Guo, Yupeng Cun
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is the most common histological subtype of renal cancer and remains a clinical challenge due to its frequent resistance to therapy and poor prognosis in advanced stages. Apoptosis, a fundamental tumor-suppressive mechanism, exhibits paradoxical roles in cancer, wherein apoptotic tumor cells can also contribute to immunosuppression and tumor progression. However, the spatial dynamics, transcriptional heterogeneity, and prognostic relevance of apoptosis-related gene programs in ccRCC remain poorly defined. We performed an integrative analysis combining single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and summary-based Mendelian randomization (SMR) to dissect apoptosis-related malignant cell states in ccRCC. Cancer cells were stratified based on apoptosis gene signatures and CASP9 expression. Cell–cell communication was assessed using CellChat and spatial interaction networks were constructed using RCTD and mistyR. SMR was employed to link genetically regulated CASP9 expression with renal cancer risk. A CASP9-associated prognostic model was developed using LASSO Cox regression and DeepSurv on TCGA and E-MTAB-1980 cohorts. We identified transcriptionally and spatially distinct apoptosis-high and apoptosis-low malignant cell subpopulations. Apoptosis-high tumor cells, characterized by elevated CASP9 expression, preferentially localized near macrophage-enriched stromal regions and exhibited stronger spatial clustering. Ligand–receptor modeling revealed directional signaling via the SPP1–CD44 axis between CASP9-high cancer cells and macrophages. SMR analysis provided genetic evidence supporting CASP9 as a causal gene for renal cancer. CASP9-high cells demonstrated distinct developmental trajectories and formed multicellular spatial modules with macrophages and cycling cells. A five-gene apoptosis-related signature derived from CASP9-stratified tumor cells robustly predicted patient survival across both training and validation cohorts. Low-risk patients exhibited enriched immune infiltration, increased immune checkpoint expression, and enhanced immune pathway activity. Our study reveals that apoptosis, particularly CASP9-driven programs, defines a spatially organized, immunosuppressive malignant cell state in ccRCC. CASP9 acts as both a genetic driver and spatial regulator of tumor–macrophage interactions, contributing to disease progression. The CASP9-associated risk model demonstrates strong prognostic utility and highlights apoptosis as a promising therapeutic axis in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22570e710b314543b7842267551dd125b21247d1" target='_blank'>
              Integrative SMR and single cell & spatial analysis reveals the spatial heterogeneity and prognostic value of CASP9-mediated apoptotic pathways in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Zijie Yu, Xinghan Yan, Wenchuan Shao, Lingchen Cai, Da Zhong, Xiyi Wei, Ninghong Song
          </td>
          <td>2025-11-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The bone marrow (BM) is a complex and compartmentalized tissue where spatial context plays a critical role in regulating cell behavior, signaling, and disease progression. To capture these dynamics, we apply spatial transcriptomics using the Visium Spatial Gene Expression platform on formalin-fixed paraffin-embedded (FFPE) BM sections from both healthy and Multiple Myeloma (MM) mouse models, as well as MM patient samples. Overcoming the technical challenges of working with mineralized long bone tissue, we develop a custom analytical framework integrating spatial and single-cell transcriptomic data to map cellular composition and interactions in situ. This approach enables the spatial characterization of transcriptionally heterogeneous malignant plasma cells (MM-PC) and their surrounding microenvironments. We identify spatially distinct gene programs linked to MM pathogenesis, including signatures of NETosis and IL-17 signalling, which are reduced in MM-PC–rich regions. Additionally, a transition gradient from effector to exhausted T cell phenotype is associated with increased remoteness from MM-PC. These spatial patterns are identified in FFPE BM biopsies from MM patients with varying tumor burdens. In summary, our study demonstrates both the capabilities and limitations of Visium technology in characterizing spatially regulated mechanisms underlying MM pathogenesis. Visium Spatial technology enables the characterization of transcriptionally heterogeneous malignant plasma cells in multiple myeloma, and identify spatially distinct gene programs linked to MM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55f354b66d39731b9cc107b87037fbdf9278486f" target='_blank'>
              Characterization of the bone marrow architecture of multiple myeloma using spatial transcriptomics
              </a>
            </td>
          <td>
            E. Muiños-López, Ana Rosa López-Pérez, Laura Sudupe, A. Vilas-Zornoza, S. Sarvide, Purificación Ripalda-Cemboráin, P. Aguirre-Ruiz, P. S. Martín-Uriz, M. Larrayoz, L. Alvarez-Gigli, M. Abengozar-Muela, I. Cenzano, Miguel Cócera, Javier Ruiz, Ignacio Sancho, Azari Bantan, Aleksandra Kurowska, Jin Ye, Phillip T. Newton, B. Paiva, L. Tamariz-Amador, J. R. Rodríguez-Madoz, Vincenzo Lagani, Jesper N. Tegnér, Borja Saez, J. Martinez-Climent, Isabel A. Calvo, D. Gómez-Cabrero, F. Prósper
          </td>
          <td>2025-11-20</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300bf1ce4707bea01a2973482ed7fc72841204e2" target='_blank'>
              Genomic heterogeneity drives distinct infiltration patterns in glioblastoma.
              </a>
            </td>
          <td>
            D. Harwood, S. Artzi, V. Pedersen, A. Locallo, M. Lü, David Scheie, D. Nørøxe, N. Hammouda, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-11-29</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Resistance to immune checkpoint blockade (ICB) remains a major challenge in lung adenocarcinoma (LUAD), with stromal mechanisms underlying CD8⁺ T cell exhaustion still poorly understood. By integrating single-cell, bulk, and spatial transcriptomic datasets using EcoTyper, we identified a distinct immunosuppressive ecotype, EC10, enriched for TGF-β signaling and epithelial–mesenchymal transition. EC10 exhibited spatial co-localization of fibroblasts, malignant epithelial cells, and exhausted CD8+ T cells, and was consistently associated with immune exclusion, poor progression-free survival, and elevated TIDE scores across four ICB-treated LUAD cohorts. Cell–cell communication analyses revealed a dominant TGFB1–SERPINE1 signaling axis originating from fibroblasts, linking stromal remodeling to T cell dysfunction. In contrast, EC12 represented an inflamed, ICB-responsive state enriched in interferon signaling. These findings define EC10 as a spatially organized, fibroblast-driven immunosuppressive ecosystem predictive of ICB resistance, and highlight the therapeutic potential of targeting the TGF-β axis in LUAD. This research identifies a fibroblast-driven immunosuppressive ecosystem in lung adenocarcinoma, linking TGF-β signaling to immune resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130525b051ffbdab87e5edbec0e75b95564de9c8" target='_blank'>
              Fibroblast TGF-β signaling defines spatial tumor ecosystems linked to immune checkpoint blockade resistance
              </a>
            </td>
          <td>
            Seo-Young Lee, Youngjoo Lee
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8efc06cf9c217f5e70cbbc0a284aede3a11ce200" target='_blank'>
              Cancers modulate p53 truncal neoantigen display to evade T cell detection
              </a>
            </td>
          <td>
            Koji Haratani, Bruce Reinhold, J. Duke-Cohan, Caroline G. Fahey, Kemin Tan, R. J. Mallis, Alexander Gusev, Kenneth L. Kehl, Jia Luo, Elizabeth L Holliday, Daniel J. Masi, Allyson Karmazyn, Katarzyna J. Zienkiewicz, Connor J Hennessey, Rafael B Blasco, Tran Thai, Grace M. Gibbons, Sophie Kivlehan, P. Lizotte, C. Paweletz, Andrew J. Aguirre, K. L. Ligon, Roberto Chiarle, Matthew J Lang, David A Barbie, Ellis L. Reinherz
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Understanding the evolutionary dynamics of clonal populations is essential for uncovering the principles of development, disease progression, and therapeutic resistance. Recent advances in single-cell lineage tracing and transcriptomics enable such analyses by combining heritable barcodes with cell-state information. Here, we present SCOUT (single-cell Ornstein–Uhlenbeck trees), a framework that models gene expression dynamics along single-cell lineage trees using Ornstein–Uhlenbeck processes to distinguish neutral drift from selective pressure. Using simulations, we demonstrate that SCOUT accurately classifies genes based on their underlying evolutionary models. We further validate SCOUT in Caenorhabditis elegans development, identifying biological processes under selection across distinct developmental contexts. Finally, we apply SCOUT to a lung adenocarcinoma xenograft model, revealing key regulators of metastatic progression and tumor microenvironmental adaptation. By integrating lineage and transcriptomic data, SCOUT provides a powerful evolutionary lens for dissecting the forces that shape cell fate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b1a07c3c29336ff660b3f4e1f28115198c90579" target='_blank'>
              SCOUT: Ornstein–Uhlenbeck modelling of gene expression evolution on single-cell lineage trees
              </a>
            </td>
          <td>
            Hannah Stuart, Aaron McKenna
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is one of the most aggressive brain tumors because its cells spread widely through the brain, making complete removal impossible. We implanted fluorescently labeled glioma stem-like cells (GSCs) into the mouse brain and observed their infiltrative growth using advanced 3D imaging. Within 6–10 weeks, tumors caused about 15% brain swelling and showed a strong preference for invading the corpus callosum, a major white matter tract. By measuring Sox2-positive cells, we found very high densities in the tumor bulk (over 4000 cells/mm2) but much lower levels at the invasive margin (about 30 cells/mm2) and a clear gradient in the corpus callosum (3010 to 1.5 cells/mm2 across 1 mm). Using CLARITY tissue clearing combined with light-sheet microscopy, we visualized how tumor cells infiltrate along blood vessels and brain structures, adopting distinct shapes and strategies. Single-cell gene analysis of the invading corpus callosum cells identified a 7-gene “invasion signature” (NES, CCND1, GUSB, NOTCH1, E2F1, EGFR, TGFB1), which was linked to worse survival and may represent new therapeutic targets. Abstract Background/Objectives: High-grade gliomas (HGGs), including glioblastomas, are among the most aggressive brain tumors due to their high intratumoral heterogeneity and extensive infiltration. Glioma stem-like cells (GSCs) frequently invade along white matter tracts such as the corpus callosum, but the molecular programs driving this region-specific invasion remain poorly defined. The aim of this study was to identify transcriptional signatures associated with GSC infiltration into the corpus callosum. Methods: We established an orthotopic xenograft model by implanting fluorescently labeled human GSCs into nude mouse brains. Tumor growth and invasion patterns were assessed using tissue clearing, light-sheet fluorescence microscopy, and histological analyses. To characterize region-specific molecular profiles, we performed microfluidic-based single-cell RNA expression analysis of 48 invasion- and stemness-related genes in cells isolated from the tumor bulk (TB) and corpus callosum (CC). Results: By six weeks post-implantation, GSCs displayed marked tropism for the corpus callosum, with distinct infiltration patterns captured by three-dimensional imaging. Single-cell gene expression profiling revealed significant differences in 7 of the 48 genes (14.6%) between TB- and CC-derived GSCs. These genes—NES, CCND1, GUSB, NOTCH1, E2F1, EGFR, and TGFB1—collectively defined a “corpus callosum invasion signature” (CC-Iv). CC-derived cells showed a unimodal, high-expression profile of CC-Iv genes, whereas TB cells exhibited bimodal distributions, suggesting heterogeneous transcriptional states. Importantly, higher CC-Iv expression correlated with worse survival in patients with low-grade gliomas. Conclusions: This multimodal approach identified a corpus callosum-specific invasion signature in glioma stem-like cells, revealing how local microenvironmental cues shape transcriptional reprogramming during infiltration. These findings provide new insights into the spatial heterogeneity of gliomas and highlight potential molecular targets for therapies designed to limit tumor spread through white matter tracts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561590a9b7ca738409c5933b00e64b421edbb629" target='_blank'>
              Innovative In Vivo Imaging and Single Cell Expression from Tumor Bulk and Corpus Callosum Reveal Glioma Stem Cells with Unique Regulatory Programs
              </a>
            </td>
          <td>
            Natália dos Santos, Aline Aquino, Friedrich Preusser, Fabio Rojas Rusak, E. H. Jandrey, Miyuki Uno, T. K. Furuya, C. Lancellotti, M. Maldaun, Roger Chammas, Stephan Preibisch, A. Camargo, C. Masotti, Erico Tosoni Costa
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Identifying drivers of cancer progression to guide treatment selection is hindered by our limited understanding of tumor heterogeneity and its impact on tumor evolution. Here, we delineate the phenotypic variability across ~300,000 cells collected from multiple tumor loci in primary prostate and matched locoregional metastases using single-cell chromatin accessibility and gene expression sequencing. We find inter-patient heterogeneity to be confined to malignant populations. Within individual tumor loci, we see phenotypic heterogeneity among malignant cell populations despite a shared clonal genotypic architecture. We also observe that malignant cell populations disseminating to locoregional lymph nodes mirror the clonal architecture and phenotypic heterogeneity across primary tumor loci, while shifting from canonical prostate-cancer states to non-canonical inflammatory-like states. Our findings suggest a bottleneck imposed during the dissemination process, funneling prostate cancer cells toward an inflammatory-like cell state. These insights into the interplay between phenotypic identity and clonal architecture refine our understanding of prostate cancer progression and suggest that convergence of cancer cells towards an inflammatory-like state underlies dissemination to lymph nodes, offering a critical framework for future studies into prostate cancer metastatic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b9d84066de3071bee8ea7d75cdd27aba04e0e02" target='_blank'>
              Prostate cancer cells converge to an inflammatory-like state upon metastatic dissemination
              </a>
            </td>
          <td>
            Tina Keshavarzian, K. Furlano, Giacomo Grillo, Lisanne Mout, C. Arlidge, Faizan Hasan, Ankita Nand, Migle Mikutenaite, Evdoxia Karadoulama, Ashish Goyal, Elisabeth L Pezzuto, Jessica Heilmann, Jakub Sykora, S. Minner, Thorsten Schlomm, G. Sauter, R. Simon, H. He, Joachim Weischenfeldt, C. Plass, Clarissa Gerhäuser, M. Lupien
          </td>
          <td>2025-12-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="ABSTRACT Head and neck squamous cell carcinoma (HNSCC) is the sixth most common cancer worldwide. HPV-negative HNSCC, arising in diverse upper airway mucosal niches, is particularly aggressive, with poor 5-y survival and a limited response to immune checkpoint inhibitors. A deeper understanding of the tumor-localized immune landscape is essential to uncover actionable immunotherapeutic targets. Here, we integrated two single-cell RNA sequencing (scRNA-seq) datasets from 29 samples totaling nearly 300,000 immune cells to dissect immune rewiring during tumor progression and lymph node metastasis in HPV-negative HNSCC. We identified distinct shifts in adaptive immune cell populations across 14 peripheral blood mononuclear cell (PBMC) and 21 tumor-infiltrating immune cell (TIC) states. Notably, TICs exhibited enriched interferon response and immunomodulatory gene signatures, in contrast to PBMCs, indicating tumor-specific immune imprinting. Ligand–receptor analysis revealed that immunosuppressive crosstalk between macrophages and cytotoxic cells was associated with advanced disease. To spatially validate these transcriptional states, we conducted multiplexed immunofluorescence profiling on nine locally invasive HPV-negative HNSCCs, all from the ventrolateral tongue mucosa. Spatial proteomics confirmed peritumoral enrichment of activated (CD107a+, ICOS+) NK and CD8+ T cells and intratumoral accumulation of exhausted (PD-1+, PD-L1+) phenotypes, mirroring pseudotime trajectories inferred from scRNA-seq. These findings highlight spatially localized cytotoxic cell exhaustion as a key immune evasion mechanism in HPV-negative HNSCC and underscore the value of integrating spatial and single-cell data to reveal therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e184d6ff5ae8303f9ef6f76673a4a076479b44fb" target='_blank'>
              Single-cell landscape of peripheral and tumor-infiltrating immune cells in HPV-negative HNSCC
              </a>
            </td>
          <td>
            Rômulo Gonçalves Agostinho Galvani, Adolfo Alexis Rojas Hidalgo, Carlos Alberto Biagi-Junior, B. Matuck, Jelte Martinus Maria Krol, Brittany T. Rupp, Nikhil V. Kumar, Khoa L. A. Huynh, Jinze Liu, Siddharth Sheth, Vinicius Maracaja-Coutinho, K. Byrd, Patricia Severino
          </td>
          <td>2025-12-30</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9542f45171a12fa01fc5a868a4f227534778e8" target='_blank'>
              Iron-Associated Mesenchymal Plasticity and Tumor Invasion in Glioblastoma
              </a>
            </td>
          <td>
            C. Petry, A. Boecke, Aso Omer Mohammed, N. Ghariwala, V. Jariwala, Mohammad Al Shhab, O. Buchholz, I. Vasilikos, J. M. Nakagawa, Andreas Vlachos, J. Grauvogel, M. Shah, Roland Rölz, Marco Prinz, U. G. Hofmann, Jürgen Beck, Kevin Joseph, V. Ravi
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8015c113e13d1290a915b03f174abb55236e475" target='_blank'>
              Longitudinal analysis of matched patient biospecimens reveals Neural Reprogramming of Cancer-Associated Fibroblasts Following Chemotherapy in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Aylin Z. Henstridge, Nandini Arya, Ahmed M. Elhossiny, Padma Kadiyala, Georgina Branch, Vaibhav Sahai, Nicole Peterson, Jorge D. Machicado, Richard S. Kwon, Allison Schulman, Erik Wamsteker, George M. Philips, Stacy Menees, Jonathan Xia, Arvind Rao, Jiaqi Shi, Timothy L. Frankel, Filip Bednar, Marina Pasca di Magliano, C. Lyssiotis, M. Truty, Martin Fernandez-Zapico, Eileen S. Carpenter
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="The tumor microenvironment (TME) has emerged as a promising source of prognostic biomarkers. To fully leverage its potential, analysis methods must capture complex interactions between different cell types. We propose HiGINE -- a hierarchical graph-based approach to predict patient survival (short vs. long) from TME characterization in multiplex immunofluorescence (mIF) images and enhance risk stratification in lung cancer. Our model encodes both local and global inter-relations in cell neighborhoods, incorporating information about cell types and morphology. Multimodal fusion, aggregating cancer stage with mIF-derived features, further boosts performance. We validate HiGINE on two public datasets, demonstrating improved risk stratification, robustness, and generalizability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d207f0757528c944b801713e062bc7358ff14382" target='_blank'>
              From Cells to Survival: Hierarchical Analysis of Cell Inter-Relations in Multiplex Microscopy for Lung Cancer Prognosis
              </a>
            </td>
          <td>
            Olle Edgren Schullerqvist, Jens Baumann, Joakim Lindblad, Love Nordling, Artur Mezheyeuski, Patrick Micke, Natavsa Sladoje
          </td>
          <td>2025-12-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Human breast cancer cells exhibit organotropism during metastasis, showing preferential homing to certain organs such as bone, lung, liver, and brain. Spatial genome organization plays a crucial role in oncogenic transformation and progression, but the extent to which chromosome architecture contributes to organ-specific metastatic traits is unclear. This work characterizes chromosome architecture changes associated with organotropic metastatic traits. By comparing a collection of human genomic data from different subtypes of localized and lung metastatic breast cancer cells with both normal and cancerous lung cells, we find important trends of genomic reorganization. The most striking differences in 3D genome compartments segregate cell types according to their epithelial vs. mesenchymal status. This epithelial–mesenchymal transition (EMT) compartment signature occurs at genomic regions distinct from transcription-defined EMT signatures, suggesting a separate layer of regulation. Specifically querying organotropism, we find 3D genome changes consistent with adaptations needed to survive in a new microenvironment, with lung metastatic breast cancer cells exhibiting compartment switch signatures that shift the genome architecture to a lung cell-like conformation and brain metastatic prostate cancer cells showing compartment shifts toward a brain-like state. TCGA patient data reveals gene expression changes concordant with these organ-permissive compartment changes. These results suggest that genome architecture provides an additional level of cell fate specification informing organotropism and enabling survival at the metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d470a8773036dd5ad3cf2b42848d5f3ff555882" target='_blank'>
              Rearrangement of 3D genome organization in breast cancer epithelial to mesenchymal transition and metastasis organotropism
              </a>
            </td>
          <td>
            Priyojit Das, Rebeca San Martin, Tian Hong, R. P. McCord
          </td>
          <td>2025-11-12</td>
          <td>eLife</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="HPV status is a key determinant of prognosis and treatment response in head and neck squamous cell carcinoma (HNSCC). To investigate how HPV influences the tumor-immune-stromal landscape, we performed high-dimensional spatial profiling, including its impact on spatial organization, tertiary lymphoid structures (TLSs), and spatially organized cellular neighborhoods. Tumor biopsies from HNSCC patients (n = 16; 7 HPV-positive, 9 HPV-negative) were stained with a multiplex immunofluorescence (mIF) panel focused on immune profiling. A deep learning-based analysis pipeline enabled the identification and phenotypic state profiling of 14 cell types. Tissues were segmented into four distinct tumor regions, and spatial neighborhoods and TLSs were identified and analyzed for differential cellular composition, activation states, and spatial interactions between HPV-positive and HPV-negative tumors. HPV-positive and HPV-negative tumors differ in their tumor microenvironment (TME) composition, tumor cell state and spatial organization. The TME of HPV-positive tumors exhibited a greater abundance of activated lymphocytes, B- and T-cell-enriched spatial neighborhoods, and PD-1–PD-L1 interactions within the tumor area, whereas HPV-negative tumors were dominated by fibroblast- and macrophage-rich niches. T- cells in HPV-positive tumors showed greater activation across neighborhoods and areas, while in HPV-negative tumors T- cells demonstrated enrichment of exhaustion and terminal differentiation markers such as PD-1 and CD57. HPV-positive tumor cells had increased IDO1, HLA-DR, and Ki67 positivity, whereas HPV-negative tumor cells were more frequently CD44 positive, reflecting a more stem-like phenotype. Importantly, TLSs in HPV-positive tumors were located closer to the tumor area and enriched in activated immune cells, including ICOS+ CD4 T- cells, memory T- cells, and CD21+ B- cells. In contrast, TLSs in HPV-negative tumors were more distant and enriched for immunosuppressive populations such as PD-1+/PD-L1+ Tregs and macrophages. HPV status defines distinct spatial immune architectures in HNSCC. HPV-positive tumors harbor immune-activating TLSs and cellular neighborhoods that support antitumor immunity, whereas HPV-negative tumors exhibit suppressive niches and stromal dominance. These findings highlight TLSs, particularly their proximity and composition, as key features of the HPV-stratified TME and potential biomarkers for immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec5a252b8e081f817dffff9e37e6b264cbbcf5" target='_blank'>
              Spatial profiling of HPV-stratified head and neck squamous cell carcinoma reveals distinct immune niches and microenvironmental architectures
              </a>
            </td>
          <td>
            E. Markovits, Dmytro Klymyshyn, Roni Froumine, Hailing Zong, M. Mints, Sangeetha Mahadevan, Kenneth Bloom, Jamie G. Bates, Gareth J. Thomas, Lauri Diehl, Oscar Puig, Abhishek Aggarwal
          </td>
          <td>2025-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is an aggressive brain tumor with limited treatment options, and the tumor microenvironment plays a major role in its progression. Among stromal cells, cancer-associated fibroblasts (CAFs) are known to influence tumor growth and immune responses, but their diversity in glioblastoma has not been well understood. In this study, we analyzed both bulk and single-cell RNA sequencing data to identify CAF subtype-associated patterns within glioblastoma. We discovered four CAF populations—immune, inflammatory, antigen-presenting, and myofibroblastic—each associated with poor patient survival and unique immune and metabolic features. We also identified Spleen Tyrosine Kinase (SYK) as a key regulatory gene shared across these CAF subtypes. Our findings highlight the importance of stromal heterogeneity in glioblastoma and suggest that targeting CAF-related signaling, including SYK, may offer new therapeutic opportunities for this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77eebcce441ee2b3c1609920849b71d1de5f2a30" target='_blank'>
              Dissecting CAF Heterogeneity in Glioblastoma Reveals Prognostic Subtypes and a Central Regulatory Role for Spleen Tyrosine Kinase (SYK)
              </a>
            </td>
          <td>
            Ji-Yong Sung, K. Hwang
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PAAD) metastasis is driven by complex tumor–microenvironment interactions. Here, we integrated single-cell and bulk transcriptomic analyses of 104,855 cells from 10 patients to delineate the cellular and molecular landscape of primary versus metastatic PAAD. We identified metastasis-associated epithelial (LMO7⁺, TOP2A⁺, PIGR⁺), fibroblast (IGKC⁺, RGS5⁺), and M2-like macrophage (APOE⁺, CD14⁺, FOLR2⁺, SPP1⁺) subpopulations, validated via bulk deconvolution. Functional analyses revealed upregulated Wnt signaling, epithelial–mesenchymal transition, and angiogenesis in metastatic epithelial and fibroblast compartments. Intercellular communication analysis highlighted SPP1-mediated macrophage–epithelial/fibroblast crosstalk involving key receptor–ligand pairs, contributing to immune suppression and metastatic niche formation. Integrating gene expression and cell proportions, we developed a prognostic model with high predictive accuracy (C-index > 0.85), stratifying patients into risk groups with distinct immune landscapes. Furthermore, PTK6 was identified as a driver of PAAD proliferation, migration, and invasion. Collectively, our study elucidates TME-driven mechanisms of PAAD metastasis, identifies prognostic and therapeutic targets, and provides a framework for precision intervention. Supplementary Information The online version contains supplementary material available at 10.1007/s10238-025-01974-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/759c03413ef4b1106fa2a25c75713885026e6884" target='_blank'>
              Single-cell transcriptomic analysis reveals key cell types and pathogenic genes associated with metastasis in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Shijun Shen, Zhiqiang Li, Hong Yang, Xinwei Zhang, Gang Chen, Chengzhou Pa
          </td>
          <td>2025-11-29</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Emerging imaging spatial transcriptomics (iST) platforms and coupled analytical methods can recover cell-to-cell interactions, groups of spatially covarying genes, and gene signatures associated with pathological features, and are thus particularly well-suited for applications in formalin fixed paraffin embedded (FFPE) tissues. Here, we benchmark the performance of three commercial iST platforms—10X Xenium, Vizgen MERSCOPE, and Nanostring CosMx—on serial sections from tissue microarrays (TMAs) containing 17 tumor and 16 normal tissue types for both relative technical and biological performance. On matched genes, we find that Xenium consistently generates higher transcript counts per gene without sacrificing specificity. Xenium and CosMx measure RNA transcripts in concordance with orthogonal single-cell transcriptomics. All three platforms can perform spatially resolved cell typing with varying degrees of sub-clustering capabilities, with Xenium and CosMx finding slightly more clusters than MERSCOPE, albeit with different false discovery rates and cell segmentation error frequencies. Taken together, our analyses provide a comprehensive benchmark to guide the choice of iST method as researchers design studies with precious samples in this rapidly evolving field. Wang, Huang, Nelson, Gao, and colleagues perform a head-to-head comparison of multiple platforms for imaging spatial transcriptomics, determining their relative sensitivity, specificity, and ability to identify major cell types in clinical pathology samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8345e62d987587be724e60e0529b77aaa194e516" target='_blank'>
              Systematic benchmarking of imaging spatial transcriptomics platforms in FFPE tissues
              </a>
            </td>
          <td>
            Huan Wang, Ruixu Huang, Jack Nelson, C. Gao, M. Tran, Anna Yeaton, Sachi Krishna, K. Felt, K. Pfaff, Teri Bowman, Scott J. Rodig, K. Wei, Brittany A. Goods, Samouil L. Farhi
          </td>
          <td>2025-11-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c1adf877d1b31f72e6e7bf9d982961bd5b1573" target='_blank'>
              Digital Spatial Pathway Mapping Reveals Prognostic Tumor States in Head and Neck Cancer
              </a>
            </td>
          <td>
            Julius Hense, M. J. Idaji, Laure Ciernik, Jonas Dippel, Fatma Ersan, Maximilian Knebel, Ada Pusztai, Andrea Sendelhofert, Oliver Buchstab, S. Fröhling, Sven Otto, Jochen Hess, P. Liokatis, Frederick Klauschen, Klaus-Robert Müller, A. Mock
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Cervical cancer, as one of the most common types of malignant tumors in women, poses a significant threat to the health and survival of women worldwide. Tumor drug resistance has emerged as a critical bottleneck severely limiting the efficacy of current cervical cancer treatment strategies, making it imperative to delve deeper into the underlying mechanisms for improved therapeutic outcomes. Methods Spatial transcriptomics and single-cell sequencing technologies were employed. Single-cell sequencing was used to uncover the heterogeneity in cervical cancer development. Pseudotime analysis, exploration of cell communication pathways, and spatial transcriptomics were further applied to investigate the roles of relevant molecules. Results This study reveals an abundance of PI3 + neoplastic (NEO) in tumors and SLC40A1 + NEO cells in HSIL samples. These cell populations may play crucial roles in the development of drug resistance, as they could potentially alter the tumor microenvironment and interfere with the efficacy of therapeutic agents. Pseudotime analysis, cell communication pathways, and spatial transcriptomics highlight the pivotal role of LGALS9 and its ligands, offering new insights for early diagnosis, molecular typing, prognosis evaluation, and personalized treatment of cervical cancer. Conclusion Our findings reveal that high expression of LGALS9 suppresses T cell function, fosters a strongly immunosuppressive tumor microenvironment, and is associated with chemotherapy resistance, ineffective immunotherapy, and poor prognosis. LGALS9 may serve as a biomarker for predicting immunotherapy response. This knowledge has the potential to facilitate early diagnosis, precise molecular typing, accurate prognosis evaluation, and the development of personalized treatment strategies that can overcome tumor drug resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f662f4b181b3395cd096885302fa365b2511567e" target='_blank'>
              Exploring tumor heterogeneity and drug resistance in cervical cancer through single-cell and bulk transcriptomics
              </a>
            </td>
          <td>
            Yinxia Yang, Sirong Cheng, Yuxuan Zhong, Pengyu Sun, Xiaoru Deng, Bohui Zhou
          </td>
          <td>2025-11-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Intrahepatic cholangiocarcinoma (ICC) is a highly heterogeneous malignancy with poor prognosis, in which tumor metastasis represents a critical event driving disease progression and treatment failure. Understanding the cellular drivers of ICC metastasis and establishing effective risk assessment models are essential for improving patient outcomes. Methods We analyzed the GSE201425 single-cell RNA sequencing dataset to explore the cellular origins and mechanisms of ICC metastasis. Copy number variation (CNV) profiling and clonal evolution analysis were performed to identify malignant epithelial cells. Pseudotime trajectory analysis revealed metastasis-associated epithelial cells (MAECs). Key biomarkers within MAECs were screened, and a Metastasis Index (Met-Index) based on one-class logistic regression was constructed to assess metastasis risk in bulk RNA-seq data from the TCGA-CHOL cohort. Multiplex immunofluorescence staining was performed in 34 clinical ICC specimens. Functional validation was conducted in HuCCT1 cells using siRNA-mediated gene silencing, followed by CCK-8 proliferation assays and wound healing assays. Results MAECs were identified as a distinct epithelial subpopulation enriched in metastatic ICC samples. Biomarkers including MMP7, FXYD2, and PTHLH were upregulated in MAECs and strongly correlated with metastatic status. The Met-Index effectively stratified patients into high- and low-risk groups, with the high Met-Index group exhibiting significantly worse overall survival (p = 0.005) and progression-free survival (p = 0.015). Immunofluorescence validation confirmed increased expression of MMP7, FXYD2, and PTHLH in metastatic tumors, which correlated with advanced clinicopathological features. Functional assays demonstrated that knockdown of these biomarkers inhibited HuCCT1 cell proliferation and suppressed cell migration capacity. Conclusions This study delineates the cellular origin and molecular drivers of ICC metastasis at the single-cell level, identifies MAECs as a metastasis-related subpopulation, and establishes a clinically relevant Met-Index with prognostic value. These findings provide potential biomarkers and therapeutic targets for the early identification and management of ICC patients at high risk of metastasis. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-15253-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27d855cd27f4e8545b1968d216eb341e556b791" target='_blank'>
              Single-cell multi-omics analysis reveals drivers of intrahepatic cholangiocarcinoma metastasis
              </a>
            </td>
          <td>
            Zetian Zhang, Hao Dou, Shangyumeng Zhao, Jiaqi Wang, Meiru Chen, Xiaoli Xie, Xiangcong Tian, Dongqiang Zhao
          </td>
          <td>2025-11-24</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e493ac7b6aab5ae1ab712cb247fef1395ea0b1a" target='_blank'>
              Quantitative profiling of intratumor immune heterogeneity identifies loss of immune diversity as a hallmark of cancer progression.
              </a>
            </td>
          <td>
            Qiqi Lu, Jian-qin Luo, Chia-Hao Tung, Xiaosheng Wang, Zhongming Zhao
          </td>
          <td>2025-12-16</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a formidable clinical challenge due to therapy resistance, metastasis, and relapse. Central to these processes are colorectal cancer stem cells (CCSCs), a dynamic subpopulation endowed with self-renewal capacity, plasticity, and heterogeneity. This review synthesizes recent advancements in understanding how CCSCs orchestrate tumor progression through intricate bidirectional crosstalk with the tumor immune microenvironment (TIME). We begin by elucidating the cellular origins of CCSCs, their profound intratumoral heterogeneity, and their remarkable phenotypic plasticity—driven by genetic, epigenetic, and metabolic reprogramming—which collectively serve as the root cause of therapeutic failure. A significant portion of our discussion is dedicated to deconstructing the immunosuppressive niche co-opted by CCSCs. We detail mechanisms of immune evasion and tolerance, highlighting how CCSCs modulate innate and adaptive immune cells—including NK cells, Tregs, dendritic cells, macrophages, neutrophils, and myeloid-derived suppressor cells—to foster an environment that supports stemness and suppresses cytotoxic attack. This reciprocal interaction forms a vicious cycle that perpetuates tumor survival and progression. Finally, we critically evaluate emerging therapeutic strategies that concurrently target CCSC-specific vulnerabilities and counteract immunosuppression. We explore the limitations of conventional chemotherapy and the promise of targeted therapies (e.g., Wnt inhibitors), immunotherapies (e.g., CAR-T, bispecific antibodies), and combination regimens designed to remodel the TIME and eradicate the CCSC reservoir. By integrating insights from single-cell omics and spatial biology, this review provides a comprehensive framework for overcoming therapy resistance and proposes novel precision medicine approaches for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b2fada714432c2d56f0013d3edefb668b69269" target='_blank'>
              Colorectal cancer stem cells crosstalk in tumor immune microenvironment and targeted therapeutic strategies
              </a>
            </td>
          <td>
            Yuxuan Duan, Anshu Li, Daojia Miao, Diaoyi Tan, Keshan Wang, Jian Shi
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Age is a dominant risk factor for all major breast cancer subtypes. However, the mechanisms by which aging influences tumor development remain unclear. Using a novel mouse model whereby breast cancer is induced in situ in young and old wild-type mice via intraductal delivery of a lentivirus encoding the HER2/neu oncogene, we found that old mice exhibited a higher oncogene-induced tumor burden than young mice. Old tumor cells showed reduced expression of interferon-related genes, particularly the T cell-recruiting chemokines Cxcl9 and Cxcl10, linked to their altered chromatin accessibility. CXCL9/10 expression also declined with age in human HER2+ tumors. Correspondingly, old tumors exhibited fewer T cells within tumor lesions. Targeted interventions showed that decreased expression of Cxcl9/10 is responsible for reduced T cell infiltration and weakened anti-tumor immunity. These results show how aged tumor cells are impaired in their recruitment of immune cells, leading to a defective anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38011df7fcbd774b6090832583a2578934863ea" target='_blank'>
              Aging alters tumor cell - T cell crosstalk to promote breast cancer progression
              </a>
            </td>
          <td>
            Shanshan Yin, Kay T Yeung, S. Mamde, Kate Lin, Armin Gandhi, Xue Lei, Andrew Davis, Rouven Arnold, Jing Yang, P. Adams
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fab0829fa668d3f68087f51374ae0be2ba943151" target='_blank'>
              Perturbational single-cell profiling of patient tumors defines lineage- and context-specific programs of innate immune resistance
              </a>
            </td>
          <td>
            C. Perry, A. Frey, Y. Fei, J. Wang, P. Costa, H. N. Amin, S. Ghadermarzi, D. Levine, W. Tong, A. Koda, M. Mackie, M. He, Y. Nie, K. Clulo, F. Ouerghi, J. Wei, T. Cordero Dumit, M. Yaskolko, M. Ding, A. Caldera, O. Kyrysyuk, L. Lum, J. W. Allen, W. Guo, A. Naqash, A. Elliot, A. Vanderwalde, M. Capelletti, T. Adeyelu, S. Barry, M. Hugaboom, A. Bacchiocchi, H. Kluger, M. Bosenberg, A. Iwasaki, D. Braun, J. Clune, D. van Dijk, K. Olino, J. Ishizuka
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Bladder cancer is a common urinary system malignancy with a complex immune microenvironment that affects treatment outcomes. Single-cell RNA sequencing technology offers new opportunities to understand cellular heterogeneity within tumors, but systematic analysis of immune-related genes in bladder cancer remains limited. Methods We performed multi-omics analysis on bladder cancer samples using TCGA database analysis, single-cell RNA sequencing, cell clustering, immune gene expression analysis, and network pharmacology to study cellular heterogeneity and potential therapeutic targets. Results We identified 8 upregulated immune genes significantly associated with poor survival: CCL20, IL17RB, IL37, PLAU, INSL4, KNG1, SLURP1, and PAEP. Single-cell analysis revealed major cell types including B cells, T cells, dendritic cells, epithelial cells, fibroblasts, mast cells, monocytes, and smooth muscle cells. We identified 5 distinct fibroblast subgroups and 10 T cell subgroups with different functional characteristics. Network pharmacology analysis of the PLAU gene showed that traditional Chinese medicine compounds had good binding capacity with the PLAU protein, validated by molecular dynamics simulation. Conclusion This study identified 8 prognosis-related bladder cancer immune genes and revealed tumor microenvironment heterogeneity. The findings, particularly the PLAU therapeutic target, provide important evidence for bladder cancer immunotherapy and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7804e2c979249e5eb150897b03ef7412bb3e32eb" target='_blank'>
              Single cell RNA sequencing decodes cellular heterogeneity and identifies prognostic immune signatures in bladder cancer microenvironment
              </a>
            </td>
          <td>
            Kai Cao, Yunfeng Shi, Bin Wu, Honglei Shi, Zhong Lv, Hongqian Guo, Rong Yang
          </td>
          <td>2025-11-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01df2c37dabe824163d0ae6c9689244efcafc8f6" target='_blank'>
              Multi-scale Network Topology Analysis Reveals Metabolic Reprogramming and Therapeutic Vulnerabilities in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Handan Cetin, P. Bogdan, S. Mumenthaler, Stacey D. Finley
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Imaging biological samples in three dimensions across scales is essential for capturing the complex spatial relationships that govern cancer initiation, invasion, and therapeutic response. As biological inquiry shifts from isolated molecular measurements toward spatially contextualized, multiomic profiling, new strategies have emerged to reconstruct tissue architecture at the whole-organ scale and subcellular resolution. These advances offer more anatomically faithful representations of tissue organization and open doors to integrating morphology with deep multiomic profiling in spatially resolved formats. As a result, we are improving our understanding of inter- and intratumoral heterogeneity and the key role of rare events and minority cell populations in tumor progression. The primary techniques used for 3D imaging of tumors include intact tissue imaging for targeted visualization of biological processes and serial sectioning for integration of diverse, multiomic platforms. In this review, we survey the major technologies used to image tumors in three dimensions, highlighting key methodologies, trade-offs, and recent innovations that make these approaches increasingly central to modern cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ec1eecfebb643d8f7b0229bfefe0ab87342355e" target='_blank'>
              Lessons from 3D Reconstruction of Cancer
              </a>
            </td>
          <td>
            Ashley L. Kiemen, Laura D. Wood
          </td>
          <td>2025-11-26</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4726aa2358332b95021c3fdf2dd12bc437304fe5" target='_blank'>
              Targeting CREB remodels the immune microenvironment to enhance immunotherapy responses in pancreatic cancer
              </a>
            </td>
          <td>
            S. Mehra, Supriya Srinivasan, Sudhakar Jinka, Varunkumar Krishnamoorthy, Vineet Kumar Gupta, Vanessa Garrido, Anna Bianchi, Andrew Adams, Manan Patel, H. Amirian, Luis A Nivelo, Karthik Rajkumar, Rimpi Khurana, Edmond W Box, Yuguang Ban, Oliver G McDonald, J. Datta, Kathleen E. DelGiorno, Nipun B Merchant, Sangeeta Goswami, A. Dosch, Nagaraj S. Nagathihalli
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b0c553f54a8eda4d3dc1abe42f3db69a41cfe1" target='_blank'>
              Single-cell and spatial transcriptomic profiling reveals distinct tumor microenvironment dynamics in cervical adenocarcinoma and squamous cell carcinoma.
              </a>
            </td>
          <td>
            Yaqin Wu, Bei Wei, Zhentong Wei, Hongyan Wang, Yunyun Liu, Dongdong Xu, Xiaokang Li, Liyuan Guo, Huaiwu Lu, Huiling Shang
          </td>
          <td>2025-12-19</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b4d870dc4ba763551ef81671814fc477f8046f" target='_blank'>
              CancerSTFormer enables multi-scale analysis of spot-resolution spatial transcriptomes and dissects gene and immune regulatory responses to targeted therapies
              </a>
            </td>
          <td>
            Benjamin S Strope, Dana Varghese, William Z Bowie, Stacy Wang, Qian Zhu
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Complex and dynamic networks of molecules are involved in human diseases. Single‐cell and spatial multiomics approaches have created new avenues for understanding the pathogenesis and diagnosis of diseases. Cell connections and characteristics in diseases may be examined more thoroughly by integration single‐cell and spatial multiomics. In this paper, we first reviewed the single‐cell and spatial multiomics approaches. Subsequently, the use of single‐cell and spatial multiomics to comprehend the mechanisms of human diseases, such as cancer (head and neck squamous cell carcinoma), neurodegenerative diseases, and aging, was discussed. Furthermore, we outline how deep learning approaches are now being applied to single‐cell and spatial multiomics data analysis in an effort to better define the pathogenic alterations upstream and the downstream molecular effects of diseases. Particularly, single‐cell and spatial multiomics are being utilized to help guide treatment plans, evaluate risks, and determine how they can affect precision medicine. Despite the relative youth of the field, the development of single‐cell coupled with spatial multiomics promises to provide a powerful tool for elucidating the pathogenesis of diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cc7cb5945e9a2b609b0a45825c131d67f4ba04e" target='_blank'>
              Single‐Cell and Spatial Multiomics: Applications for Diseases
              </a>
            </td>
          <td>
            Wentao Li, Chao Chen, Xin Zhu, Chenping Zhang
          </td>
          <td>2025-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cervical cancer (CC) is one of the most prevalent malignancies that affects women worldwide. It is strongly influenced by lifestyle and environmental factors, most notably persistent HPV infection and smoking. A defining feature of CC is its high degree of heterogeneity, which poses significant challenges for conventional sequencing and diagnostic approaches in fully capturing its oncogenetic landscape. The advent of single-cell RNA sequencing (scRNA-seq) has transformed this field, offering an unprecedented ability to dissect the tumor microenvironment at a single-cell resolution. By mapping cell–cell communication networks and characterizing gene expression across diverse cellular populations, scRNA-seq enables the identification of biomarkers with strong diagnostic and prognostic potential. Beyond biomarker discovery, this technology also facilitates the recognition of novel therapeutic targets, paving the way for more precise and effective treatment strategies. This review summarizes the applications of scRNA-seq in cervical cancer research, with a focus on methodological advances, key findings from recent studies, and their implications for improving diagnosis, prognosis, and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e029b6816cfdfef9fa928cd92a7a3e4aaf306a" target='_blank'>
              Implications of Single-Cell RNA Sequencing in Cervical Cancer: Unravelling the Molecular Landscape
              </a>
            </td>
          <td>
            Blessy Kiruba, Raja Rishi Raghavendar Raja Karthikeyan, Saurav Ram Anilkumar, Vino Sundararajan, S. Lulu S
          </td>
          <td>2025-12-02</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="To identify the cell types that are infected with KSHV and the immune interactions in Kaposi sarcoma (KS) lesions, we performed spatial transcriptomics with seven KS skin tumors. We used a single‐cell RNA‐sequencing reference data set from healthy skin donors with a method to conduct spatially informed cell‐type deconvolution for spatial transcriptomics. This allowed us to predict the relative amounts of each cell type within the patient sample sections. We included custom probes for five KSHV genes that allowed us to measure human and KSHV expression patterns at the same time. We then compared the spatial gene expression data of KS skin samples with six normal skin samples and found higher expression of marker genes corresponding to macrophages/dendritic cells, lymphatic endothelial cells, and vascular endothelial cells in the KS skin lesions when compared to normal skin samples. Our spatial transcriptomic results from thousands of spots across multiple KS tumors indicated a correlation between high levels of STC1 and decreased expression of macrophage markers. Together, these analyses offer potential mechanisms by which KSHV infection may remodel skin tissue, inhibit immune responses against KSHV infection, and confer resistance to anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0e2360131ded34f6dbf590cb05f764fc0e85cc" target='_blank'>
              Spatial Transcriptomics of Patients With Kaposi Sarcoma Identifies Mechanisms of Immune Evasion
              </a>
            </td>
          <td>
            Bahman Afsari, R. Ramaswami, K. Lurain, Takanobu Tagawa, Daphne Knudsen-Palmer, Guruswamy Mahesh, Ameera Mungale, R. Yarchoan, Joseph Ziegelbauer
          </td>
          <td>2025-12-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="In cancer, extrachromosomal DNA (ecDNA) contributes to tumor heterogeneity and is associated with poor prognosis, but studies on patient-derived ecDNA are relatively limited at single-cell resolution. Here, we introduce scCirclehunter, a framework designed to identify ecDNA from scATAC-seq data and assign ecDNA to specific cell populations. Leveraging scCirclehunter and available glioblastoma (GBM) datasets, we uncover the inter-cellular heterogeneity of ecDNA-carrying cells across GBM patients and trace the trajectories of malignant cells within a single patient that harbors multiple ecDNAs. By integrating scRNA-seq data, we use ecNR2E1 as an example to demonstrate that ecDNA drives tumor progression in GBM through several mechanisms. Additionally, our findings suggest a potential link between ecDNA and increased mitochondrial transfer frequency. Overall, scCirclehunter provides a novel framework for analyzing patient-specific ecDNAs with single-cell precision, offering insights into the role of ecDNA-carrying cells in driving GBM heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c481a654d2e51f6263c828b5c2a73535459ef5" target='_blank'>
              scCirclehunter delineates ecDNA-containing cells using single-cell ATAC-seq, with a focus on glioblastoma
              </a>
            </td>
          <td>
            Rong Jiang, Zhengmao Lu, Fang Li, Yibei Zhu, Manqiu Yang, Shufan Zhang, Ping Wu, Chengliang Gong, Yiyuan Fei, Yonghua Sang, Yulun Huang, Jiong Jiong Guo, Moli Huang
          </td>
          <td>2025-12-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The spatial organization of the tumor microenvironment (TME) encodes the selective pressures that drive cancer evolution, but many tools focus on local cell contacts or single-modality readouts. We developed Tumor Landscape Analysis (TLA), a modular framework that borrows from landscape ecology and spatial statistics to quantify heterogeneity, fragmentation, and diversity across scales, from single cells to whole-slide ecosystems.



 TLA ingests whole-cell or nuclear centroids and region-level segmentations to compute (i) point statistics such as nearest-neighbor distance, Ripley’s H, and Morisita-Horn colocalization; (ii) kernel-smoothed local spatial profiles including Getis-Ord Gi* hot and cold spots; and (iii) fragmentation, shape, and diversity metrics such as patch density, contagion, shape index, and Shannon entropy. We also introduce Local Microenvironments (LMEs), unsupervised ecological niches defined by joint cell-type abundance and spatial mixing. Pilot analyses were performed in ductal carcinoma in situ (DCIS; n=353 samples, 175 patients) with cell-level maps of epithelial, lymphocyte, and fibroblast populations, and in pancreatic ductal adenocarcinoma (PDAC; n=203 resected cases after neoadjuvant therapy) using cancer and stroma segmentations.



 In DCIS, spatial relationships rather than raw abundance associated with recurrence risk. Shorter lymphocyte-to-fibroblast minimum nearest-neighbor distance and lower Ripley’s H, indicating greater intermixing, correlated with earlier recurrence independent of cell counts. This highlights immune–stromal proximity as a candidate eco-evolutionary driver of persistence. In PDAC, tissue-level fragmentation captured prognostic architecture after therapy. Two shape-based signatures emerged: (1) stromal uniformity, where lower standard deviation of stromal patch shape index associated with improved disease-free survival, consistent with a more constrained, less reactive stroma; and (2) tumor geometric complexity, where higher mean cancer patch shape index, reflecting more irregular or dendritic nests, associated with lower recurrence risk, whereas compact nests predicted worse outcomes. Across datasets, LME maps localized niche phenotypes, such as mixed-immune or segmented tumor habitats, that align with hypothesized selection landscapes and provide inputs for fragmentation analysis.



 TLA operationalizes an eco-evolutionary view of cancer by unifying interpretable, multiscale spatial metrics in a single workflow that works with any rasterized data input. By quantifying how habitats, edges, and mixing patterns are organized, TLA reveals niche structures and architectural motifs linked to clinical outcomes and complements foundation models and modality-specific pipelines with theory-grounded, clinically translatable descriptors. Ongoing work integrates TLA features with molecular data and outcomes to derive robust biomarkers and to inform adaptive, niche-targeted strategies in precision oncology.



 Ryan M. Carr, Luis Cisneros, Merih Deniz. Toruner, Martin E. Fernandez-Zapico, Carlo Maley. Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f5d8ed56157756f83dcf8a730cae9e8b632c46" target='_blank'>
              Abstract A015: Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments
              </a>
            </td>
          <td>
            Ryan M. Carr, L. Cisneros, M. D. Toruner, Martín E Fernandez-Zapico, Carlo C. Maley
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor heterogeneity is one of the key factors leading to diagnostic errors, difficulty in identifying subtle cellular differences, and the occurrence of immune escape. Although single-cell RNA sequencing (scRNA-seq) has elucidated the molecular characteristics and heterogeneity at the single-cell level, it fails to retain spatial location information. In contrast, spatial transcriptomics (ST) technology can simultaneously acquire gene expression information and cellular localization, providing a unique perspective for studying the tumor microenvironment, cell-cell interactions, and other aspects. The integrated application of scRNA-seq and ST technologies enables the analysis of tumor heterogeneity from both molecular and spatial dimensions, offering comprehensive information support for precision medicine. The complementary advantages of these two technologies not only overcome the limitations of traditional methods but also promote the research on complex mechanisms such as tumor heterogeneity and the tumor microenvironment, accelerating the translational process of clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9514d53fae21da5a677d774e552caceeac1e73" target='_blank'>
              Research Progress in Single-Cell RNA Sequencing and Spatial Transcriptomics for Precision Oncology
              </a>
            </td>
          <td>
            Minxin Mo
          </td>
          <td>2025-12-18</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The tumor immune microenvironment plays a critical role in tumor progression and responses to immunotherapy. Nevertheless, its cellular complexity and heterogeneity remain incompletely understood. In this study, we employed high-resolution single-cell RNA sequencing on CD45+ immune cells isolated from ten syngeneic murine tumor models, representing seven distinct cancer types under treatment-naïve conditions, thereby enabling a comprehensive profiling of tumor-infiltrating immune cells. We identified seven principal immune cell populations and provided an in-depth characterization of T cells, NK/innate lymphoid cells, dendritic cells, monocytes/macrophages, and neutrophils. Cross-species analyses further delineated conserved immune cell states and transcriptomic features within the T cell and monocyte/macrophage compartments that are shared across syngeneic models and human tumors. To investigate the functional relevance of the predominant monocyte/macrophage compartment and the notable presence of neutrophils in syngeneic tumors, we evaluated responses to anti-PD-1 therapy across various models and analyzed the enrichment of monocyte/macrophage subsets in tumors that responded to treatment. Furthermore, we conducted neutrophil depletion experiments using anti-Ly6G antibodies, administered both as monotherapy and in combination with PD-1 blockade. Remarkably, an interferon-stimulated gene-high (ISGhigh) monocyte subset was significantly enriched in models responsive to anti-PD-1 therapy. Neutrophil depletion resulted in variable antitumor effects across models but failed to enhance the efficacy of PD-1 blockade. In summary, our single-cell profiling offered a detailed atlas of the immune microenvironment across multiple syngeneic mouse tumor models, thereby enabling rational model selection for immuno-oncology studies. We uncovered an ISGhigh monocyte subset enriched in anti-PD-1 responsive models, and showed the context-dependent effects of neutrophil depletion on tumor immunity and immunotherapy, underscoring the heterogeneity and functional divergence of immune cell sublineages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f925e5b4fe351a72672a38a0505d78e0be84e52" target='_blank'>
              Single-cell atlas of the tumor immune microenvironment across syngeneic murine models
              </a>
            </td>
          <td>
            Jia Wang, Bin Jiang, Minjuan Deng, Han Yan, Peilin Zhang, Wei Jin, Zhirong Shen
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Endometrial cancer is regarded as one of the most prevalent malignancies in women globally. Despite the advancements brought by multiple therapeutic strategies, the efficacy and effectiveness of treatment still appear to be diminished. Therefore, this accentuates the importance of understanding the tumour microenvironment (TME). Conventional methods, including immunohistochemistry (IHC), along with recently emerged techniques like single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST), have rejuvenated the notion of deciphering TME. From deconstructing tumour heterogeneity, identifying cell populations that play significant roles in treatment response, to discovering key biomarkers and therapeutic targets, these technologies lay the base for innovations. Importantly, this is the first comprehensive review that brings together scRNA-seq, ST, and IHC in Endometrial cancer. Merging these methods is geared towards creating a better grasp of the dynamics and interactions between diverse cells and the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bef0a0a3a7a3aad177138a9aca3da36e10bf85b" target='_blank'>
              Navigating tumour microenvironment in endometrial carcinoma: a comprehensive review integrating immunohistochemistry, single-cell RNA-sequencing and spatial transcriptomics
              </a>
            </td>
          <td>
            Oishee Mondal, Blessy Kiruba, Sajitha Lulu Sudhakaran, V. Sundararajan
          </td>
          <td>2025-11-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Glioma remains the most aggressive and therapy-resistant brain tumor, with a highly immunosuppressive tumor microenvironment. The role of inflammatory signaling in glioma progression and treatment response is poorly understood. Methods We performed single-cell RNA sequencing (scRNA-seq) analysis on 93,027 cells from 18 samples. Inflammation-related genes were identified using hdWGCNA and AUCell scoring. Multiple bulk RNA-seq and microarray datasets were integrated for validation. Machine learning algorithms, including CoxBoost, LASSO, and Random Survival Forest, were used to identify prognostic genes. Immune infiltration, immunotherapy response, and mutational landscape were analyzed using established computational tools. Findings COL8A1 was found to be a significant prognostic gene within a highly linked gene module connected to inflammation. Astrocytes, OPCs, and cancerous cells all had high levels of COL8A1 expression. In several cohorts, low survival was linked to high COL8A1 expression. The suppression of tumor migration and proliferation by COL8A1 knockdown was validated by functional tests. Multiple immunotherapy determinants, inhibitory immunological checkpoints, and immune cell infiltration all showed high correlations with COL8A1 expression. Additionally, it accurately forecasted the immune checkpoint blockade response. Through mutational profiling, we identified distinct somatic mutation patterns distinguishing COL8A1-high from COL8A1-low cancers. Conclusion By connecting tumor-intrinsic inflammation to immunological surveillance and treatment resistance, our study identified COL8A1 as a crucial inflammatory hub in glioma. In order to improve the results of immunotherapy for glioma, COL8A1 may be a useful therapeutic target and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf0463c9a68cc9f41847f121143f02d93fcd0752" target='_blank'>
              COL8A1 as a pro-inflammatory mediator bridges immune evasion and therapy resistance in glioma
              </a>
            </td>
          <td>
            Jiachong Wang, Jiale Li, Jun Peng, Chunyuan Zhang, Zigui Chen, Changfeng Miao, Chunhai Tang, Qisheng Luo
          </td>
          <td>2025-12-10</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Hepatocellular carcinoma (HCC) is a deadly and highly complex cancer. To better understand what makes it aggressive, we investigated the key genetic switches, known as transcription factors (TFs), that control tumor cell behavior. By analyzing vast amounts of genetic data from single cells, we identified a core network of nine TFs that drive HCC progression. We found that these TFs orchestrate a shift in tumor cells from a state of rapid growth to a more invasive and metabolically adapted state. This change also reshapes the tumor’s surroundings, creating an environment that suppresses the immune system, largely by recruiting and activating specific immune cells called SPP1+ macrophages. This coordinated action between tumor cells and immune cells promotes cancer growth and immune evasion. Our findings highlight this TF–macrophage axis as a promising new target for developing more effective HCC therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989aaab1ae1f0d6fa871b18d9e4cad44d18f7594" target='_blank'>
              Transcription Factor Networks Drive Tumor Progression and Immune Microenvironment Remodeling in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Sang Hoon Lee, Juwon Ahn, Wonbin Choi, Jina Kim, Joon Yeon Hwang, Jae-Hwan Kim, Hyaekang Kim, Woori Kwak
          </td>
          <td>2025-11-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05c188c0c6e2f128d8772ff9b6be6a1353be8fc1" target='_blank'>
              A Hybrid Multiscale Model for Predicting CAR-T Therapy Outcomes in Solid Tumors
              </a>
            </td>
          <td>
            M. Nikmaneshi, Lance L. Munn
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains one of the leading causes of cancer-related morbidity and mortality in men worldwide, and therapeutic resistance—particularly to immunotherapy—continues to limit clinical efficacy. Mounting evidence has positioned the immunosuppressive tumor microenvironment (TME) as a core driver of disease progression and a formidable barrier to effective immune-based interventions. In this review, we present a comprehensive and multi-dimensional analysis of the cellular, stromal, and molecular constituents of the immunosuppressive TME in PCa, highlighting its significant heterogeneity and context-dependent functions. We emphasize recent breakthrough insights obtained through single-cell RNA sequencing (scRNA-seq), spatial multi-omics, and high-dimensional imaging technologies, which are redefining our understanding of tumor–immune–stromal interactions. Based on these mechanistic findings, we examine precision therapeutic strategies aimed at remodeling the TME, including combinatorial immune checkpoint blockade, metabolic reprogramming, cytokine network regulation, and advanced nanomedicine-based delivery systems. Finally, we discuss translational opportunities and future research directions, underscoring the necessity of integrating advanced omics technologies with biomarker-driven clinical trial design to enable individualized, precision interventions and improve survival outcomes for patients with PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce05a3966abc2d6663f2f9fc06215e6d78446ea" target='_blank'>
              Tumor microenvironment-mediated immune evasion and resistance in prostate cancer: mechanisms, cross-talk, and therapeutic opportunities
              </a>
            </td>
          <td>
            Mingfeng Li, L. Kwantwi, Cuimei Wang, Qin Xiao
          </td>
          <td>2025-11-26</td>
          <td>Clinical and Experimental Medicine</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdc2f4fd014e0601edcc3f78406fd8afaf2448ea" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors.
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, D. Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson A. Weir, O. Ashenberg, Geraldine S Pinkus, Scott J. Rodig, Caroline Uhler, Evan Z. Macosko, Margaret Shipp, A. Louissaint, Fei Chen, Todd R Golub
          </td>
          <td>2025-12-20</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c14f73e77a88fcb7e1149959d53563fad60dc97" target='_blank'>
              Liquid Crystal-like self-organization of Glioblastoma Prevents cell density induced migratory arrest
              </a>
            </td>
          <td>
            Urszula Hohmann, Chalid Ghadban, J. Prell, Christian Strauss, Faramarz Dehghani, T. Hohmann
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Seminoma is the most common solid malignant tumor of the testis in young males, significantly impacting fertility. Approximately 20% of seminomas metastasize, markedly increasing recurrence risk and compromising quality of life. To investigate the poorly understood mechanisms driving seminoma metastasis, we performed a comprehensive analysis by integrating single-cell RNA sequencing, TCGA data mining, and cell biology assays. We identified significant intratumoral heterogeneity. A subset of tumor cells expressing DPPA4 and PSMA7 showed high stemness, enhanced self-renewal, and association with metastasis. Knockdown of these genes reduced sphere formation, tumor migration and proliferation in Tcam-2 and NCCIT cells. Conversely, tumor cells overexpressing PAGE5 and SAT1 exhibited reduced stemness, migratory capacity and proliferation. In the immune microenvironment, we identified IFNG+ T cells, which recruit and activate other antitumor immune cells. These cells secrete IFN-γ, which promotes tumor differentiation, reduces stemness, and mitigates tumor aggressiveness. Based on these findings, we further developed a molecular panel to aid in the identification of seminomas with a higher risk of metastasis. In conclusion, our study identifies unique molecular signatures that facilitate risk stratification based on metastatic potential, providing valuable insights for improving precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a200ee992d63f7f5d78cc885c7a3fb857907df52" target='_blank'>
              Single-cell analysis unravels divergent gene signatures shaping seminoma stemness and metastasis
              </a>
            </td>
          <td>
            Zhouliang Bian, Biying Chen, Jiahui Guo, Jiamin Zhang, Guangye Du, Shufang He, Haihua Yuan, Yue Zhou, Bin Jiang, Daliu Min, Dingwei Ye, Hengchuan Su, Yanjie Zhang
          </td>
          <td>2025-11-07</td>
          <td>Cell Death Discovery</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Molecular heterogeneity in head and neck squamous cell carcinoma (HNSCC) is well recognized, yet existing subtype frameworks remain largely descriptive and have not translated into therapeutic decision-making. Here, we establish a mechanistic platform that converts transcriptomic diversity into drug-actionable tumor states. Integrating multi-cohort RNA-seq from 727 tumors across five independent datasets, genome-scale CRISPR dependency maps, and pharmacologic screening, we define distinct tumor survival circuits across HPV-negative HNSCC and nominate subtype-matched therapeutic strategies. These circuits encompass a proliferative axis (MYC, MET/FAK, inflammatory and translational programs), an epithelial-differentiated/adhesion program, an EMT-like state with stromal activation, and mitochondrial/oxidative metabolic states, each mapping to selective liabilities (e.g., mitotic/autophagy control, ERBB/PI3K and cadherin signaling, OXPHOS/mitochondrial translation, and G2/M-integrin-Notch pathways, respectively). We then develop a transcriptomic predictor of EGFR-inhibitor response using machine learning and validate it in prospectively collected, fresh patient-derived 3D microtumors. The resulting 13-gene signature identifies erlotinib-responsive tumors (R = 0.93) and maps biologically to an epithelial-differentiated state, outperforming EGFR expression alone. Our study establishes a subtype-to-dependency-to-therapy framework, enabling precision stratification and providing a clinically feasible path for prospective biomarker deployment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29377070a0d6fb125a75d4b7cf89275cb55faab" target='_blank'>
              Subtype-Specific Dependencies and Drug Vulnerabilities Enable Precision Therapeutics in Head and Neck Cancer
              </a>
            </td>
          <td>
            Joel M Vaz, Songli Zhu, Mateo Useche, Lauren Shih, Emily Marchiano, Slobodan Beronja, Bruce E Clurman, Cristina Rodriguez, Brittany Barber, Marina Chan, Taranjit S. Gujral
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Hepatocellular carcinoma (HCC) harbors a dynamic tumor microenvironment (TME) in which macrophages are highly abundant and plastic. Under physiological conditions, macrophages switch between inflammatory and resolution/tissue-repair programs to maintain homeostasis; however, during hepatocarcinogenesis these programs are reprogrammed into tumor-associated macrophage (TAM) states that foster immune suppression, angiogenesis, and tumor progression. Purpose: To summarize macrophage heterogeneity and polarization mechanisms in HCC, and to highlight omics-informed therapeutic opportunities for targeting TAMs and improving precision immunotherapy. Research Design: This review summarizes physiological macrophage polarization and the mechanistic basis of macrophage reprogramming in the hepatocellular carcinoma immune microenvironment, integrating evidence from recent advances in single-cell sequencing, multi-omics, and spatial transcriptomics, with a focus on macrophage subset diversity, key regulatory pathways governing polarization and function, and emerging macrophage-targeted interventions and biomarkers. Results: Recent single-cell and spatial multi-omics studies reveal substantial TAM heterogeneity and plasticity in HCC. Macrophage-targeted strategies—including TAM depletion, phenotypic reprogramming, and exosome-mediated drug delivery—show encouraging preclinical efficacy. Macrophage-associated prognostic models and biomarkers may support individualized immunotherapeutic approaches. Conclusions: Macrophage polarization in HCC represents a dynamic continuum that is essential for homeostasis but is co-opted by tumors to drive immunosuppression and tissue remodeling. Advances in single-cell and spatial multi-omics are redefining TAM subsets and actionable pathways, enabling more rational macrophage-targeted therapies. However, challenges remain in standardizing TAM definitions, identifying robust predictive biomarkers, minimizing off-target effects, and optimizing combinations with immunotherapy. Integrating longitudinal multi-omics with AI-based modeling may help predict macrophage state transitions, guide patient-specific regimens, and advance precision medicine in HCC. Plain language summary Hepatocellular carcinoma is a highly aggressive cancer. Studies have found that immune cells within tumors—particularly macrophages—play a crucial role in cancer development and therapeutic response. Macrophages have two main “personalities”: the M1 type, which can kill cancer cells, and the M2 type, which supports cancer growth. They shift between these states in response to signals from the tumor microenvironment. Both liver-resident Kupffer cells and infiltrating macrophages are involved in this process. Various molecules, such as cytokines, enzymes, and exosomes, influence their polarization. This polarized state of macrophages, in turn, affects key tumor processes such as stemness, proliferation, metastasis, and angiogenesis. In recent years, scientists have been exploring drugs that specifically target macrophage polarization to suppress liver cancer, offering new directions for future therapy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b9c3d3369daa7c8bd747ff210bf91cca4945570" target='_blank'>
              Macrophage Polarization in the Tumor Microenvironment of Hepatocellular Carcinoma: From Mechanistic Insights to Translational Therapies
              </a>
            </td>
          <td>
            Xiaoqing Fu, Mingquan Pang, Zhixin Wang, Haijiu Wang
          </td>
          <td>2025-11-01</td>
          <td>Cancer Control: Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Melanoma brain metastasis (MBM) remains lethal with limited treatment efficacy. Meanwhile, the cellular origins and drivers of brain metastasis from melanoma have yet to be defined. Through integrated single-cell/single-nucleus RNA sequencing of 26 melanoma samples, we identified a pre-brain-metastatic tumor subpopulation (MBMATCs, MBM-associated tumor cells) within a conserved malignant cell trajectory (Mela3). MBMATCs exhibited activated pro-metastatic pathways and upregulated neural/adhesion genes (NRG3, NCAM1), suggesting a cellular origin for brain tropism. MBM ecosystems showed T cell exhaustion (elevated PD-1, HAVCR2, LAG3) and Treg enrichment. An MBM-Index derived from bulk RNA-seq accurately quantifies MBMATCs abundance, independently predicting poor overall survival in both TCGA-SKCM and validation cohorts. Furthermore, we assessed the clinical relevance of the MBM-Index and uncovered five candidate drugs with potential activity against MBMATCs. This study identifies MBMATCs as brain metastasis associated tumor cells and positions the MBM-Index as a biomarker for early stratification of melanoma patients at high risk of brain metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0332483b66f858c33c245348a73898413f2eeff5" target='_blank'>
              Single-cell transcriptomics profiling reveals cellular origins and molecular drivers underlying melanoma brain metastasis
              </a>
            </td>
          <td>
            Jingxin Zeng, Ling Lin, Yangyang Ma, Wenzhe Huang, Quan Luo, Ju Wen
          </td>
          <td>2025-11-24</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Formalin-fixed paraffin-embedded (FFPE) samples are the gold standard for tissue preservation in clinical and research settings. Current single-cell chromatin accessibility technologies cannot resolve cell-type-specific epigenetic profiles in FFPE tissues due to extensive DNA damage. We present scFFPE-ATAC, a high-throughput single-cell chromatin accessibility assay for FFPE samples that integrates an FFPE-adapted Tn5 transposase, ultra-high-throughput DNA barcoding (>56 million barcodes per run), T7 promoter-mediated DNA damage repair, and in vitro transcription. We benchmark scFFPE-ATAC on FFPE mouse spleen and validate its performance against fresh tissue. We apply it to human lymph node samples archived for 8–12 years and to lung cancer FFPE tissues, revealing distinct regulatory trajectories between tumor center and invasive edge. Analysis of archived follicular lymphoma and transformed diffuse large B-cell lymphoma samples identifies relapse- and transformation-associated epigenetic dynamics. scFFPE-ATAC enables retrospective, spatial, and mechanistic epigenetic studies in long-term archived specimens. The study introduces scFFPE-ATAC, a high-throughput single-cell chromatin accessibility profiling method enabling profiling from clinically archived formalin-fixed paraffin-embedded (FFPE) tissues, unlocking retrospective epigenetic insights across cancer and human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42464048e0ef78f0787ee6a93b5eb7756d4e366" target='_blank'>
              scFFPE-ATAC enables high-throughput single cell chromatin accessibility profiling in formalin-fixed paraffin-embedded samples
              </a>
            </td>
          <td>
            Ram Prakash Yadav, Pengwei Xing, Miao Zhao, P. Hollander, C. Strell, Minglu Xie, Maede Salehi, Emma Torell, Tobias Sjöblom, G. Enblad, Rose-Marie Amini, F. Swartling, I. Glimelius, P. Micke, Mats Hellström, Xingqi Chen
          </td>
          <td>2025-11-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d59ad194fb04ebf384288815f79948f2c11cf9f" target='_blank'>
              Humanized glioblastoma patient-derived orthotopic xenografts recreate a locally immunosuppressed human immune ecosystem amenable to immunotherapeutic modulation
              </a>
            </td>
          <td>
            P. M. Moreno-Sanchez, A. Oudin, Batuhan Kısakol, Heiko Dussmann, E. Klein, V. Baus, Camille Rolin, Carole Seguin-Devaux, Mahsa Rezaeipour, Aurélie Poli, A. Michelucci, J. Paggetti, E. Moussay, Jochen H. M. Prehn, Simone P. Niclou, A. Gołȩbiewska
          </td>
          <td>2025-11-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Aim Glioblastoma (GBM) exhibits pronounced cellular plasticity that contributes to therapeutic resistance. A facet of this adaptability is neuronal mimicry, whereby tumor cells acquire neuron-like features to engage with neural circuits. The coherence and biological significance of this process remain uncertain. Methods Spatial transcriptomic and proteomic datasets from Greenwald et al. [1] were analyzed alongside healthy cortical tissue from Ravi et al. [2] as a non-malignant reference. A permutation-based framework identified tumor–neuronal (TN) cells across Healthy, Core, Infiltrative, and Bulk regions, with stability verified by resampling and Jaccard overlap (CV < 10%). Results TN cells were enriched in Bulk (OR ≈ 70, p < 10−10) and Infiltrative (OR ≈ 7, p < 10−10) regions relative to Healthy. Integrative analyses revealed a synaptic-like transcriptional program marked by SNAP25, SYT1, and STMN2 (logFC ≈ 1–1.5), enriched for vesicle cycling and cytoskeletal remodeling pathways. Neuronal subtype markers (SLC17A7, GABRA1) were depleted, while malignant markers (CLU, GFAP, VIM) persisted. Single-cell proteomics confirmed co-localization of neuronal and tumor proteins within clustered, double-positive cells. Conclusion This study delineates a reproducible hybrid state with partial neuronal characteristics, suggesting that neuronal mimicry represents an adaptive, rather than lineage-converted, trajectory of GBM that may inform strategies to disrupt tumor–neuron crosstalk. References 1. Greenwald AC, et al. ‘Integrative spatial analysis reveals a multi-layered organization of glioblastoma.’ Cell 2024;187(10):2485–2501.e26. 2. Ravi V, Will P, Kueckelhaus J, et al. ‘Spatially resolved multi-omics deciphers bidirectional tumor-host interdependence in glioblastoma.’ Dryad 2025;10.5061/dryad.h70rxwdmj.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1722c03dee89d4877aa24eca4527f645a5b89d83" target='_blank'>
              Spatial multi-omics characterizes neuronal mimicry as a coherent hybrid state in glioblastoma
              </a>
            </td>
          <td>
            Claire Yuan, Jake Y Chen
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Triple-Negative Breast Cancer (TNBC) encompasses a biologically heterogeneous group of tumors, which can be classified into distinct molecular subtypes, namely basal-like 1 (BL1), basal-like 2 (BL2), immunomodulatory (IM), mesenchymal (M), mesenchymal stem-like (MSL), and luminal androgen receptor (LAR), with unique clinical and pathological characteristics. While immune features of these subtypes have been extensively characterized, the integration of non-immune stromal and structural components into our understanding of TNBC biology is only now being fully recognized. This narrative review synthesizes current evidence regarding differences in the non-immune microenvironment across TNBC molecular subtypes, with a focus on cancer-associated fibroblasts (CAFs), vascular features, extracellular matrix (ECM) dynamics, and epithelial–mesenchymal transition (EMT), along with metabolic–hypoxic reprogramming. Data from several studies are integrated to highlight subtype-specific signatures. Differences in stromal architecture and metabolic adaptations, potentially reflecting the underlying molecular heterogeneity, may hold prognostic or predictive significance and could inform personalized therapeutic strategies targeting the tumor–stroma interface.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/089f130c819509e80723565570647cc30d7f85ba" target='_blank'>
              Dissecting the Non-Immune Tumor Microenvironment in Triple-Negative Breast Cancer: Molecular Subtype-Specific Patterns and Prognostic Implications
              </a>
            </td>
          <td>
            A. Syrnioti, Eleni Timotheadou, Vasileios N. Papadopoulos, Georgia Syrnioti, T. Koletsa
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Spatial omics technologies enable high‐resolution mapping of transcriptomic, proteomic, and metabolic profiles within intact tissues, revealing how immune, stromal, and parenchymal cells interact in situ during inflammation. Chronic inflammation in skeletal muscle and the central nervous system is spatially organized within defined niches that shape disease progression and therapeutic response. In skeletal muscle, spatial analyses have uncovered fiber‐type‐specific vulnerability, regenerative trajectories, and immune–stromal crosstalk in disorders such as Duchenne muscular dystrophy and inclusion body myositis. In the central nervous system, these approaches have revealed compartmentalized neuroinflammation in multiple sclerosis, innate immune activation in amyotrophic lateral sclerosis, and immune evasion in glioma. Integration with single‐cell gene expression enables inference of cell–cell communication networks and identification of spatial gradients of immune activation and tissue remodeling. Despite major advances, clinical translation remains limited by small cohorts, methodological variability, and insufficient functional validation. As spatial profiling becomes more accessible, standardized, and scalable, it promises to stratify inflammatory disease states, identify tissue‐resident immune programs, and guide mechanism‐based therapies. Hence, spatial omics provide an unprecedented opportunity to resolve the cellular architecture of inflammation, revealing not only where immune activity occurs, but how it is orchestrated within complex tissue microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf8707390ef60e2d03a42073f1ba1cd24fde9c0" target='_blank'>
              Spatially Resolved Profiling of Compartmentalized Muscle and Brain Inflammation
              </a>
            </td>
          <td>
            Thorge Dobbertin, Lucas Schirmer
          </td>
          <td>2025-12-01</td>
          <td>European Journal of Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a82efc5b2c9cc4922bf6d2b205d9310648f4f84" target='_blank'>
              Single-cell multiomics reveals epigenetic rewiring of splenic memory B cells in murine malaria reinfection
              </a>
            </td>
          <td>
            Montserrat Coronado, Á. Vincelle-Nieto, Isabel G. Azcárate, S. Pérez-Benavente, A. Puyet, Amalia Díez, José M. Bautista, A. Reyes-Palomares
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Although non-muscle-invasive bladder cancer (NMIBC) frequently recurs and progresses despite BCG therapy, cellular mechanisms behind this remain unclear. Understanding the dynamics within the tumor microenvironment (TME) and identifying pathways associated with BCG resistance are crucial for improving NMIBC treatment. To elucidate transcriptional dynamics and key cellular interactions in the TME, we performed longitudinal single-nucleus RNA sequencing of 9 NMIBC cases with three pairs of treatment-naïve and disease progression samples. The resulting 58,037 nuclei were subject to cell type and subtyping annotations along with cell–cell interaction analyses. Our analysis revealed marked interpatient heterogeneity in malignant cells, accompanied by copy number alterations associated with disease progression. TGF-β signaling emerged as a key pathway, showing gradual enrichment from pre-treatment to post-progression samples. Within the tumor microenvironment, we identified distinct subtypes including LAMP3 + dendritic cells and iCAF fibroblasts that followed unique pseudotime trajectories linked to progression. Cell–cell interaction analysis further highlighted critical ligand–receptor pairs such as DSC2–DSG2 and ENG–BMPR2, which were associated with poor clinical outcomes. These interactions may represent hypothesis-generating candidates that warrant further validation. This study highlights key cellular and molecular mechanisms underlying bladder cancer progression Post-BCG treatment. Through single-nucleus RNA sequencing, we identified critical TME subtypes, TGF-β signaling involvement, and crucial ligand-receptor interactions such as DSC2-DSG2 and ENG-BMPR2, which may serve as preliminary biomarkers requiring confirmation in larger independent cohorts. These findings offer valuable insights into enhancement of treatment strategies and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b994449406e351724039c2b4ed65a466872b5c" target='_blank'>
              TGF-β signaling and tumor microenvironment dynamics in bladder cancer progression post-BCG therapy: a longitudinal single-nucleus RNA-seq study
              </a>
            </td>
          <td>
            Seo-Young Lee, Yun-Hee Lee, Tae-Min Kim, U-Syn Ha
          </td>
          <td>2025-11-10</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11151ef60d8479a0c0f55b160c123009b32bf93" target='_blank'>
              Single-cell transcriptomic analysis reveals the heterogeneity and functional characteristics of macrophage subpopulations in colon cancer.
              </a>
            </td>
          <td>
            Jiazheng Zhao, Baochun Wang, Xiao Li, Chunting Wei, Yi Min, Dayong Wang
          </td>
          <td>2025-11-23</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a powerful technique for understanding cellular diversity, but processing large patient cohorts to identify phenotype-associated cell populations remains challenging. Here, scPER (Estimating cell Proportions using single-cell RNA-seq Reference), a rigorous approach combining adversarial autoencoder and extreme gradient boosting to estimate tumor microenvironment cell compositions and identify phenotype-associated subclusters for bulk RNA-seq samples. Integrating scRNA-seq datasets from diverse studies, scPER constructed comprehensive reference panels and disentangled confounders from true signals. scPER achieved superior accuracy in cellular proportion estimation compared to CIBERSORTx, BayesPrism, Scaden, MuSiC, SCDC, DeSide and ReCIDE. It showed high accuracy in predicting metastatic melanoma immunotherapy response and identified a critical T cell subcluster expressing FCRL3 and SLAMF7. In metastatic urothelial cancer, scPER predicted TGFβ-mediated inhibition of CD4 naïve T cells to diminish PD-L1 checkpoint blockade efficacy. scPER enables robust integration of scRNA-seq datasets to estimate cellular proportions across tumors and identify clinically relevant cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8390e37b5363dcdab22134564264151d0e0926d0" target='_blank'>
              scPER: A Rigorous Computational Approach to Determine Cellular Subtypes in Tumors Aligned With Cancer Phenotypes From Total RNA Sequencing.
              </a>
            </td>
          <td>
            Bingrui Li, Xiaobo Zhou, R. Kalluri
          </td>
          <td>2025-11-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Immune-mediated glomerulonephritis (IMGN) is a major cause of kidney failure worldwide, yet the precise roles of T cells in its pathogenesis remain poorly understood. However, existing studies lack a comprehensive understanding of the characteristics and functional roles of IMGN T cells in the human context. Addressing this gap is crucial for advancing targeted therapies. By integrating single-cell RNA sequencing (sc-RNA-seq) data from three primary IMGN types—IgA nephropathy (IgAN), lupus nephritis (LN), and membranous nephropathy (MN)—we identified T cell subtype alterations at single-cell resolution. Utilizing advanced sc-RNA-seq computational pipelines, we constructed gene co-expression networks (GCNs), inferred T-cell differentiation trajectories, and assessed metabolic and intercellular signaling features. IMGN kidneys presented expanded T-cell compartments, with significant enrichment of cytotoxic natural killer T (NKT) cells and GZMK⁺ effector memory T (GZMK⁺ Tem) cells. Notably, LTB⁺ memory T cells (LTB⁺ Tm) were selectively elevated in IgAN and LN patients. A coexpression module centered on RGS1 was significantly correlated with 24-h proteinuria (p < 0.001). Metabolic profiling revealed subtype-specific disruptions in the glutathione (GSH) and 3-phospho-D-glyceroyl phosphate (3PD) pathways. Interaction analysis highlighted endothelial cells, mesangial cells, and fibroblasts as key mediators of pathogenic T-cell activation via defined ligand‒receptor pairs. This study provides the first comprehensive single-cell atlas of human IMGN T cells, revealing disease-specific T-cell states, metabolic signatures, and activation mechanisms. Our findings offer new insights into human renal immunopathology and identify promising therapeutic targets for IMGN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/355e0f6ab7752fce8d41720415bcb57b5ac71157" target='_blank'>
              Single-cell transcriptomic analysis reveals T cell heterogeneity and metabolic reprogramming in human immune-mediated glomerulonephritis
              </a>
            </td>
          <td>
            Jianbo Qing, Yiting Zhao, W. Cheungpasitporn, Jing Miao, C. Thongprayoon, Henglan Wu, Junnan Wu
          </td>
          <td>2025-11-16</td>
          <td>Autoimmunity</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background/Objectives: Achieving complete resection (R0) during secondary cytoreduction (SCR) is a pivotal prognostic factor for patients with platinum-sensitive relapsed ovarian cancer (PSROC). Methods: By integrating single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and functional assays on primary cells, we identified tumor microenvironment (TME) heterogeneity, particularly the presence of distinct subpopulations of cancer-associated fibroblasts (CAFs), as a critical determinant of surgical outcomes for SCR. Results: We characterized a previously unrecognized CAF-C7 subpopulation that was selectively enriched in tumors from non-R0 patients. Mechanistically, CAF-C7 promoted tumor cell migration, suppressed anoikis, and facilitated angiogenesis via the IGF1-IGF1R signaling axis, thereby contributing to multifocal recurrence and reducing the likelihood of complete resection. Importantly, the inhibition of the IGF1-IGF1R pathway effectively attenuated the pro-tumorigenic functions of CAF-C7. Conclusions: These findings uncover a novel cellular driver of unfavorable surgical outcomes in PSROC and suggest promising biomarkers and therapeutic targets for improving patient stratification and treatment in the context of SCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c69a7316901a4f26ec9560008414f1bd2cba29" target='_blank'>
              Single-Cell Analysis Links C7+ Cancer-Associated Fibroblasts with Incomplete Resection in Platinum-Sensitive Relapsed Ovarian Cancer
              </a>
            </td>
          <td>
            Longxia Li, Shangbing Gao, Yilizhati Maimaiti, Lifeng Lin, Xiaoxia Xing, Wei Wu, Yulian Chen, Mei-Chun Cai, Guanglei Zhuang, Rongyu Zang
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73743e96a9fb170b328b9a3bdc455c5b165f804f" target='_blank'>
              Single cell transcriptomic profiling uncovers shared inflammatory pathways and immune dysregulation underlying the asthma to lung cancer transition.
              </a>
            </td>
          <td>
            He Huang, Gang Teng, Jing Zhou, Ling Liu, Mei Dong, Wei Chen, Nianzhi Zhang
          </td>
          <td>2025-12-24</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Prostate cancer is classified as “immune-cold” due to limited infiltration of immune cells and no clear correlation between immune cells and clinical outcomes. However, immune cells are found in prostate cancers and the spatial relationships between these immune cells and cancer cells/glands have not been investigated, partly due to a lack of automated tools that classify both cancerous cells/glands. In this paper, we have developed an end-to-end tool (TOPAZ: Tissue Organization identification using sPAtial proteomics) that combines multiplexed single-cell protein data with histology images to: 1) predict cancerous versus non-cancerous epithelial cells using a Gaussian-mixture model; 2) predict cancerous/non-cancerous gland type using a principal curve estimation. Using TOPAZ to assign cancer and non-cancerous labels to cells and glands, we extracted multiscale spatial features from the classification results—including immune dense-region geometrical features and cell-to-gland distances— and correlated the features with risk of biochemical recurrence and cancer grade. Tissue-microarrays containing 753 cores from 217 prostate cancer patients underwent multiplexed immunofluorescent imaging (Cell DIVE, Leica) for epithelial cell markers (panCK26, S6, NaKATPase), basal cell markers (p63, CK5), a cancer cell marker (AMACR), and T cell markers (CD3, CD4, CD8, FOXP3, CD68). Cancerous/non-cancerous cell classification from TOPAZ achieved 82% sensitivity and 99% specificity against expert annotation, and the pipeline further predicted cancerous/non-cancerous glands without manual threshold tuning. Regulatory-T-cell and helper-T-cell percentages decreased, and macrophage percentage increased with grade increase (P < 0.05). When the median distance from cancerous gland centroids to the nearest regulatory or helper T-cell exceeded approximately 50 µm, the hazard of biochemical recurrence doubled (log-rank P < 0.01). The open-source Shiny app TOPAZ (https://chunglab.bmi.osumc.edu/TOPAZ) packages the workflow, predicting individual cell types and gland shapes. By combining probabilistic cell typing with gland-shape modeling, TOPAZ yields interpretable multiscale spatial features linked to prognosis and is released as an open web app for unrestricted use. Author Summary Spatial distribution of cancerous cells/glands and immune cell distribution has not been considered as a prognosticator in prostate cancer. In addition, automated tools that can quantify and integrate these distributions are lacking. We combined high-dimensional single-cell protein measurements with histology images to map gland structure in prostate cancer tissue. Our web-based tool, TOPAZ (https://chunglab.bmi.osumc.edu/TOPAZ) predicts whether each epithelial cell and gland in virtual H&E image is cancerous or not. Once the predictions are made, a spatial analysis workflow helps quantify spatial features of immune cells relative to the glands and correlate with recurrence risk and grades. Across 753 tissue cores from 217 prostate cancer patients, helper and regulatory-T-cells located more than about 50 µm away from cancerous epithelial glands were associated with a higher risk of biochemical recurrence. The pipeline provides new insights for researchers and pathologists into prostate cancer progression and biochemical recurrence through integration of spatial location of cancer glands and immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14615ea1ac9187e235a7c8fddba1789e76619bfb" target='_blank'>
              Annotation-Free Prediction of Cancer Cells and Glands and Spatial Analysis of Immune Cells
              </a>
            </td>
          <td>
            K. Jung, Soumya Ghose, Sanghee Cho, E. McDonough, Chrystal Chadwick, Robert West, James D. Brooks, Dongjun Chung, Fiona Ginty, R. Machiraju, Parag Mallick
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Spatial Transcriptomics (spTx) offers unprecedented insights into the spatial arrangement of the tumor microenvironment, tumor initiation/progression and identification of new therapeutic target candidates. However, spTx remains unlikely to be routinely used in the near future. Hematoxylin and eosin (H&E) stained histological slides, on the other hand, are routinely generated for a large fraction of cancer patients. Here, we present a deep learning-based approach for multiscale integration of spTx with tumor morphology (MISO). We train MISO to predict spTx from H&E and validate it on a dataset of 72 10X Genomics Visium samples. We further validate our approach on 348 samples from five indications from the MOSAIC consortium and show that MISO significantly outperforms competing methods in extensive benchmarks. We demonstrate that MISO enables near single-cell-resolution, spatially-resolved gene expression prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b66b910856826f08a930be56d475ed3036cfdbc" target='_blank'>
              A deep learning-based multiscale integration of spatial omics with tumor morphology
              </a>
            </td>
          <td>
            B. Schmauch, L. Herpin, Antoine Olivier, Thomas Duboudin, Rémy Dubois, Lucie Gillet, Alexandre Filiot, Jean-Baptiste Schiratti, Valentina Di Proietto, Delphine Le Corre, Alexandre Bourgoin, Julien Taïeb, Jean-François Emile, W. Fridman, Elodie Pronier, P. Laurent-Puig, Eric Y. Durand
          </td>
          <td>2025-11-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background: Tertiary Lymphoid Structures (TLS) are organized aggregates of immune cells that form in non-lymphoid tissues in response to chronic inflammation, including cancer. In melanoma, TLS presence within the tumor microenvironment is associated with a pre-existing anti-tumor immunity and linked to improved immunotherapy responses. However, their role in predicting recurrence, particularly in early-stage melanoma, remains unclear.Methods: We conducted a retrospective analysis of archival primary tumor samples from 65 patients with stage I-II melanoma and negative sentinel lymph node (Table 1). Tumor immune infiltrates and TLS were initially assessed using four-marker immunohistochemistry (IHC) panels targeting CD20, CD4, CD8, and Podoplanin (PDPN), a marker of stromal cells. TLS were objectively identified using the HDBSCAN AI-based clustering algorithm. Tumor samples were further analyzed for high-dimensional phenotyping and spatial analysis using a 17 markers panel with hyperplexed immunofluorescence imaging (HIFI). Associations between TLS features and clinical outcomes were evaluated.Results: IHC revealed that 88% of patients had peritumoral T cell infiltration, and TLS were detected in 54% of cases. TLS-positive patients had improved disease-free survival (DFS, p=0.2) and overall survival (OS, p=0.2) (Figure 1A, B). To investigate further, HIFI was performed on 8 patients (4 recurrent, 4 non-recurrent), revealing 13 distinct cell types: stromal (n=6), tumor (n=2), and immune (n=5) (Figure 1C). Spatial analysis identified 9 recurrent spatial modules (SM), each representing a distinct pattern of interaction between cell types. One SM, enriched for PD-L1⁺ stromal cells, was significantly associated with recurrence (p=0.019) (Figure 1D).Conclusions: TLS presence associates with improved DFS and OS in early-stage melanoma. However, beyond their mere presence, TLS spatial organization and local stromal-immune context could influence clinical outcomes. Our data demonstrated that a specific spatial module—defined by the proximity of PD-L1⁺ stromal cells—is associated with disease recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/978d2823a740581a2a5f2a0fdbf9de6949d49e50" target='_blank'>
              13 | Spatial contexture of tertiary lymphoid structures as a predictor of recurrence in early-stage melanoma
              </a>
            </td>
          <td>
            Italian Melanoma Intergroup
          </td>
          <td>2025-12-05</td>
          <td>Dermatology Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131d5075adfc8a892d5a30b581d5ddea0c0afd87" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular heterogeneity and prognostic subtypes in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhen Han, Yuan Gao, Yuxing Zhao, Jiahong Wu, Jia Chen, Jinling He, Hong Bai, Liang Qiao, Yali Ren, Lei Sun
          </td>
          <td>2025-12-22</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) is the most common malignancy among women, with its progression and prognosis significantly influenced by the tumor microenvironment (TME). Age-related differences in TME composition lead to distinct tumor behaviors: young patients (≤ 40 years) exhibit aggressive tumors, while elderly patients (> 70 years) experience immunosenescence and reduced therapy responses. We performed single-cell RNA sequencing (scRNA-seq) analysis on tumors from 10 breast cancer patients (5 ≤ 40 years, 5 ≥ 70 years), encompassing 33,664 high-quality cells. After cell annotation and batch correction, malignant epithelial cells were identified using inferCNV. We applied pseudotime trajectory analysis, pathway enrichment, and cell–cell communication profiling to investigate age-specific TME dynamics. Survival relevance was assessed using a GEO cohort (GSE20685) of young breast cancer patients, and immunohistochemical staining was performed on clinical tumor and fibroadenoma tissues to validate protein-level expression of key ISGs. In young patients, malignant epithelial cells showed gradual upregulation of interferon-stimulated genes (ISGs) such as IFI44, IFI44L, IFIT1, and IFIT3 along the pseudotime trajectory, suggesting their involvement in early tumorigenesis. High expression of these ISGs was significantly associated with poor overall survival in a young BC cohort (GSE20685). Immunohistochemical validation further confirmed elevated IFIT3 protein levels in young tumor tissues. In contrast, elderly patients had a TME enriched in macrophages and fibroblasts, with activation of immunosuppressive pathways (e.g., SPP1, COMPLEMENT). Our integrative analysis identifies ISGs as key transcriptional drivers of tumorigenesis in young breast cancer, with potential prognostic and therapeutic value. Despite limited sample size, the combination of single-cell transcriptomics, clinical survival data, and protein-level validation provides robust evidence of age-specific TME remodeling. These findings support the development of age-tailored immunotherapy strategies targeting interferon signaling in young patients and immune checkpoint pathways (e.g., LAG3, CTLA4) in elderly individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81afa54ab4f065b06519a2062a7ca2a21beb87a4" target='_blank'>
              Single-cell RNA sequencing reveals age-related heterogeneity in the tumor microenvironment of breast cancer patients
              </a>
            </td>
          <td>
            Zihao Li, Ji Feng, Mengyang An, Shuming Kou, Zengren Zhao, Jingjing Yu, Bo Liu, Fangjian Shang
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Colorectal carcinoma exhibits high heterogeneity, comprising subtypes that show poor efficacy of T cell-based immunotherapies, such as programmed cell death protein 1 (PD-1) checkpoint inhibitors. Although natural killer (NK) cells are considered a promising approach for cancer immunotherapy, it remains unclear what molecular mechanisms drive NK cell activation or suppression within the tumor microenvironment. Moreover, limitations in human tumor models that reflect the diversity of individual patient tumors hinder the ability to effectively select patients who would benefit most from NK cell-based therapies. Methods Here, we established a co-culture platform of genetically diverse colorectal cancer (CRC) patient-derived organoids (PDOs) with primary allogeneic NK cells. We performed bulk RNA sequencing analysis of sorted NK cells after exposure to PDOs and aligned gene expression signatures derived from our findings with publicly available single-cell RNA sequencing data of NK cells from peripheral blood and CRC tissues of patients. Moreover, we evaluated identified pathways using flow cytometry and IncuCyte live-cell imaging analysis to quantify phenotypic alterations and NK cell-mediated killing of PDOs over time, respectively. Ultimately, we tested CRISPR-Cas9-edited NK cells and PDOs, small molecule compounds, and clinically relevant monoclonal antibodies (mAbs) to increase NK cell potency. Results On co-culture, NK cells acquired common transcriptional signatures related to hypoxia and transforming growth factor-beta (TGF-β), similar to NK cells infiltrating CRC tissues of patients. In addition, we observed patient-specific differential PDO susceptibilities to NK cell-mediated lysis. Major histocompatibility complex class I deficiency and natural killer group 2, member D (NKG2D)-ligand expression on PDOs facilitated NK cell-mediated cytotoxicity, and induced phenotypic NK cell diversification related to activation and the acquisition of inflammation and tissue-residency-related transcriptional signatures. Genetic or pharmaceutical targeting of hypoxia-inducible factors HIF1A/EPAS1 or TGF-βR1, or the addition of anti-CEACAM1 mAbs, enhanced NK cell-mediated PDO killing or activation, respectively. Conclusions The NK cell/PDO co-culture platform allows the identification of both common and patient-specific impacts of the tumor microenvironment on NK cell function and can aid the development of patient-tailored immunotherapies. The majority of CRC (CMS2/CMS3) PDOs from our cohort were susceptible to NK cell-mediated killing and induced NK cell activation, highlighting the potential of NK cells for CRC immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfdbb13391ad7497b66effc8a4643668f2be84a" target='_blank'>
              Colorectal cancer organoids drive hypoxia, TGF-β, and patient-specific diversification of NK cell activation programs
              </a>
            </td>
          <td>
            Andreas von Kries, Irene Garcés-Lázaro, Bianca M Balzasch, Carsten Sticht, Indra A Shaltiel, Kim E. Boonekamp, Annika Sams, Alessia Triassi, Tomáš Hofman, Elke Burgermeister, J. Betge, Matthias P. Ebert, M. Boutros, Laura Helming, A. Stojanovic, A. Cerwenka
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Aberrant tumor glycosylation can alter immune recognition; however, the specific influence of glycan-lectin interactions on tumor progression remains poorly understood. Here, we identify the C-type lectin receptor CD301b (encoded by Mgl2) as a regulator of immune activity within the breast tumor microenvironment (TME). Using a murine triple-negative breast cancer model, we demonstrate that tumors expressing the Tn glycoantigen grow more rapidly, and this growth is facilitated by CD301b+ immune cells. Depletion or genetic loss of CD301b markedly suppressed tumor growth, indicating that CD301b promotes tumor progression through myeloid-tumor interactions. Phenotypic analyses revealed that CD301b+ cells within tumors are type 2 conventional dendritic cells (cDC2s), a subset known to influence immune polarization. Single-cell RNA sequencing of human breast cancers showed that the human ortholog CLEC10A is expressed in cDC2-like dendritic cells and select macrophage subsets, suggesting a conserved role for CD301+ myeloid populations. Transcriptomic profiling of tumors developed in Mgl2-deficient mice revealed a shift toward an inflammatory, immune-activated state consistent with enhanced antitumor immunity. Together, these findings establish a link between tumor glycosylation and lectin signaling of myeloid cells, highlighting CD301b as a potential target for reprogramming the tumor immune microenvironment in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401b567312697ac0756ae3899ebeedb84ca70dd0" target='_blank'>
              CD301b lectin expression in the breast tumor microenvironment augments tumor growth
              </a>
            </td>
          <td>
            A. Ozdilek, A. Paschall, Zahra Nawaz, Afaq M. M. Niyas, F. Avci
          </td>
          <td>2025-12-03</td>
          <td>Research Square</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Brain metastasis is a major contributor to melanoma-related mortality, arising from both genomic alterations and dynamic adaptations to the brain microenvironment. Tumor cells must cross the blood–brain barrier and exploit brain-specific signaling and metabolic landscapes to establish metastases. Brain metastases display molecular and phenotypic programs distinct from other organs, reflecting pre-existing and acquired mutations during progression. We analyzed primary melanoma, metastatic melanoma, and metastatic brain tumors genomic datasets to identify mutated proteins and signaling pathways that are common to or distinct among these tumor types. Our comprehensive computational analyses uncovered key pathways and proteins associated with brain metastasis. Brain metastases exhibit unique mutational landscapes and convergent alterations in neurotrophin, MAPK, and PI3K/AKT signaling compared with primary and metastatic melanoma. Proteins especially frequent in brain metastases pathways support survival, proliferation, and adaptation in the brain microenvironment. Pharmacology-wise, rational strategies such as combining c-KIT and VEGFR inhibitors can suppress converging oncogenic pathways and mitigate resistance in advanced melanoma. Co-targeting neurotrophin receptors with MAPK pathway members, particularly in combination with BRAF inhibitors, appears a promising approach for treating melanoma brain metastases. Our findings underscore the molecular heterogeneity of melanoma brain metastases, shaped by interconnected signaling networks, and reveal therapeutic targets that may diverge from those in other metastatic sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d4ecead949fc8a270a159656433a0bf5e5ad895" target='_blank'>
              Cellular Mechanisms Underlying Melanoma Brain Metastasis
              </a>
            </td>
          <td>
            B. R. Yavuz, Hyunbum Jang, Ruth Nussinov
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe18ec72d329e63e66769b08f8cae7347edc672" target='_blank'>
              Prognosis prediction using autophagy gene expression in osteosarcoma
              </a>
            </td>
          <td>
            Qing Liu, Cong Zhou, Ke Xu, Qikui Zhu, Junqin Lu, Chun Zhang, Wukun Xie, Guojiu Fang, Xue Zhou Chen, Dasheng Tian, Juehua Jing, Yize Li, Hongsheng Wang, Wenxiu Duan, Yihui Bi
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lymph node metastasis is a common metastatic route of gastric cancer. However, the heterogeneity of tumor cells and tumor microenvironment between primary gastric cancer and metastatic lymph nodes, as well as the driving mechanisms of metastasis, remain not fully understood. In this study, single-cell RNA sequencing and spatial transcriptomics analysis were performed on 4 gastric cancer patients (four primary tumors and two paired metastatic lymph nodes). Additionally, the TCGA database was used as an external validation dataset to validate the unfavorable clinical outcomes of necroptosis genes and the correlation between MDK-NCL and immune infiltration. Our single-cell analysis revealed significant heterogeneity in the tumor microenvironment between primary and metastatic lymph nodes of gastric cancer. Pseudotime analysis indicated that gastric cancer cells may follow two potential differentiation trajectories during lymph node metastasis. One type of cells possess the ability to migrate to lymph nodes, while the other type remain in the primary site during tumor progression. We observed that the expression of necroptosis-related gene CHMP3 was significantly associated with lymph node metastasis. Immunofluorescence further suggested upregulated CHMP3 protein expression in metastatic lymph node. Furthermore, we found that MDK-NCL interaction was active in metastatic lymph nodes. External validation using the TCGA cohort indicated that high MDK-NCL expression correlated with an immunosuppressive microenvironment. Finally, through integrated single-cell and spatial analysis, we observed that gastric cancer cells may remodel the tumor microenvironment via the MDK-NCL signaling pathway. In summary, our study revealed the dynamics of tumor cells in lymph node metastasis in GC and identified a subtype of GC cells with potential metastatic capability. Our data suggest that necroptosis and the MDK-NCL signaling pathway may be involved in facilitating lymph node metastasis, providing new insights into the mechanisms of GC progression and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a384ade012316bb3834d72822d6e02871f918f06" target='_blank'>
              Single-cell and spatial dissection of necroptosis spatiotemporal evolution driving lymph node metastasis in gastric cancer
              </a>
            </td>
          <td>
            Yuhua Hu, Feng-lin Shen, Honghong Zhang, Zhuowen Long, Xiaojun Zhao, Jiale Wen, Yijian Sheng, Junxing Huang, Yan Chen, Qing Guo
          </td>
          <td>2025-11-17</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Current therapeutic strategies for endometrial cancer are mainly based on aggressive histological types and molecular subtypes. However, ignoring the spatial distribution of immune/stromal cells fails to account for the heterogeneity of the local tumour microenvironment, leading to biased prediction of treatment response. The goal of precision medicine is to delineate the biological characteristics of local functional units based on molecular labelling, which adequately reflects spatially adaptive changes during treatment or metastasis. Single-cell resolution analysis of 40 endometrial cancer cases across four molecular subtypes was performed using imaging mass cytometry (IMC) to quantify the frequency, spatial distribution, and intercellular crosstalk of distinct immune and stromal cell populations. These ecosystem-level features were systematically correlated with clinical features and outcomes, including treatment response and survival. We further identified CD90 + clusters as key regulators of macrophage polarization and T-cell infiltration dynamics, with flow cytometry used to validate their functional role in tumour subtype specification and microenvironmental remodelling. Finally, machine learning-based spatial phenotyping was employed to construct molecular subtype-specific signatures and a highly accurate recurrence prediction model for high-risk endometrial cancer. Single-cell profiling revealed that CD90 + clusters constitute a critical immunomodulatory component within the tumour microenvironment, demonstrating significant enrichment in macrophage differentiation pathways and serving as key mediators of intercellular signalling networks. Furthermore, computational models integrating functional molecular signatures with cell–cell interaction profiles demonstrated high predictive accuracy for both molecular subtyping and recurrence risk stratification in patients with endometrial carcinoma. Our study establishes a spatial eco-context framework for molecular subtypes of endometrial cancer by integrating single-cell spatial multiomics data. This approach enables high-resolution mapping of tumour-immune-stromal interaction networks and reveals novel targets for personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63a8a4865fa920849f9dc4e8439a41c6d062126a" target='_blank'>
              Spatial ecostructural modelling of endometrial cancer identifies the key role of CD90 + CD105 + endothelial cells in tumour heterogeneity and predicts disease recurrence
              </a>
            </td>
          <td>
            Di Wu, Cinian Lv, Zhifeng Yan, Luyang Zhao, Lian Li, M. Ye, Mingyang Wang, Qingzhi Zhai, Nan Wang, Zheng Wang, Yuanguang Meng, Mingxia Li
          </td>
          <td>2025-11-17</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f4df7fa2e4800fd55af9dfd1c2bbdecb131c8f" target='_blank'>
              A guide to transcriptomic deconvolution in cancer.
              </a>
            </td>
          <td>
            Yaoyi Dai, Shuai Guo, Yidan Pan, C. Castignani, Matthew D. Montierth, P. van Loo, Wenyi Wang
          </td>
          <td>2025-12-02</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) exhibits substantial genetic and functional heterogeneities. We generated clonal spheroids (CS) from HNSCC cell lines and patient tumor cells, uncovering distinct functional differences. Hyperproliferative spheroids (HRPS) showed increased proliferation and tumorigenic potential, whereas hypoproliferative spheroids (HOPS) demonstrated enhanced migration and resistance to cisplatin and radiation. In patient tumor-derived spheroids, spheroid size distribution analysis across various TNM stages indicated that early-stage tumors (T2N0M0) predominantly formed HRPS, while later-stage tumors (T4N2M0) exhibited a higher prevalence of HOPS. Single-cell RNA sequencing (scRNA-seq) identified six clusters in the HOPS and four in the HRPS, each defined by unique gene expression profiles and pathway enrichment. HOPS were enriched with fibroblast, mesenchymal, and keratinizing epithelial cells, indicating a metastatic phenotype. They also expressed genes associated with stemness (KRT15 and ALDH3A1). Conversely, HRPS was characterized by proliferative genes, including TOP2A, AURKA, CKS2, AREG, and ANLN, suggesting an advanced HNSCC proliferative signature. Despite being generated from only two cells/well, the various clusters in both spheroid types mirrored the diverse tumor microenvironment in HNSCC, including neoplastic, epithelial, basal, fibroblast, and mesenchymal cells. Notably, the gene expression profiles of Cluster 3 in the HOPS and Cluster 1 in the HRPS closely matched the expression patterns observed in the HNSCC-TCGA dataset. These clusters also displayed a high transcriptional correlation between patient tumors and their xenografts, reinforcing the clonal nature of HNSCC’s genetic and functional diversity of HNSCC. This study provides a deeper understanding of the genetic and functional diversity of HNSCC tumors, which can aid in the development of more precise diagnostic tools by identifying distinct tumor subtypes. Distinguishing between hyper- and hypoproliferative spheroids could improve prognostic accuracy, helping to predict tumor behavior and patient outcomes. Moreover, the identification of specific gene expression profiles associated with proliferation, metastasis, and treatment resistance may guide personalized therapeutic strategies, enabling targeted treatment based on the unique tumor characteristics of individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/942cca58b9f76ca4806f68d559d87541dfee33c1" target='_blank'>
              Uncovering functional divergence and cellular clusters with specific gene signatures in HNSCC clonal spheroids
              </a>
            </td>
          <td>
            Jyoti Pandey, Md. Zubbair Malik, Ritis K Shyanti, Palak Parashar, Praveen K. Kujur, Deepali Mishra, Dhanir Tailor, Jee Min Lee, T. Kataria, Deepak Gupta, Hitesh Verma, Anshu Yadav, S. Kateriya, Sanjay V. Malhotra, Vibha Tandon, Rupesh Chaturvedi, Rana P. Singh
          </td>
          <td>2025-12-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0127956ee657e81733c7622d2b48c0407a2cab1" target='_blank'>
              Cell heterogeneity contributes to the variable response of HIV-1 to latency reversing agents
              </a>
            </td>
          <td>
            Rachel Topno, Hussein Karaki, Flavia Mazzarda, Oriane Pourcelot, Manon Philippe, K. Zibara, Ovidiu Radulescu, Edouard Bertrand
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Introduction Lung cancer is a highly heterogeneous disease, and the tumor microenvironment (TME) characteristics are closely related to disease progression and treatment outcomes. We elucidated the cell type-specific transcriptome landscape of cancer cells and the effect of the TME on lung adenocarcinoma (LUAD). Methods Single-cell RNA sequencing of the LUAD tissues and matched adjacent normal tissues of three patients in the early-stage via histopathological and immunohistochemical confirmation. Results The results revealed the landscape of 68,579 cells in LUAD microenvironment, and highly heterogeneous AT2 cells are crucial source of lung epithelial cell carcinogenesis. Genes KRT81, SPP1, PCDH7, SLC2A1, and TET1 were significantly upregulated in tumor tissues and associated with poor prognosis and survival, providing insights for exploring lung cancer biomarkers in future studies. Trajectory analysis identified ERBB4, SEMA4A, GCNT2 and SOX4 as key factors in AT2 cells that may promote cell proliferation, migration, or epithelial-mesenchymal transition (EMT) during the progression of lung adenocarcinoma (LUAD). We elucidated the transcription factor–target gene regulatory network involving NKX2-1 and TEAD1 in malignant tumor cells derived from AT2 cells. The findings indicated that malignant AT2 cells regulate communication between epithelial and immune cells in the TME by predicted FN1-CD44 and CADM1-CADM1 ligand-receptor interactions, which ultimately suppresses the host immune response. Discussion This comprehensive single-cell analysis increases our understanding of AT2 cells molecular and dynamics of metastatic lung cancer. In summary, single-cell RNA profiling of LUAD offers valuable prognostic insights based on AT2 cell types and identifies potential biomarkers for therapeutic responses, aiding the future development of LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68e15868fa220169dfea67c353cad3603481040a" target='_blank'>
              Single-cell RNA sequencing reveals heterogeneity among AT2 epithelial cells in the lung adenocarcinoma microenvironment
              </a>
            </td>
          <td>
            Zimeng Cao, Xuefeng Bai, Jing Li, Jinru Cai, Gang Cao, Yuncai Xiao, Wei Zhou
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8a1f1ccbeb7cda31923d2bd498d2b7cb3a690e5" target='_blank'>
              Metabolic zonation and characterization of tissue slices with spatial transcriptomics
              </a>
            </td>
          <td>
            Oier Etxezarreta Arrastoa, A. Pirona, Allon Wagner, N. Yosef
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) represents a significant global health challenge due to its molecular heterogeneity and the immunosuppressive nature of its tumor microenvironment (TME). Tumor-associated macrophages (TAMs), particularly the M2-polarized subset, are crucial in HCC progression, contributing to immune evasion, angiogenesis, and metastasis. However, the mechanisms driving TAM reprogramming and their metabolic interactions within the TME remain poorly understood. In this study, we applied single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to identify a novel metabolic-immune axis in HCC. Our findings revealed increased oxidative phosphorylation (OXPHOS) in tumor cells, which correlated with enhanced ribosomal activity in M2 macrophages, suggesting a metabolic coupling between the two. Tumor cells, under elevated OXPHOS, secrete factors that recruit macrophages and promote M2 polarization, thereby reinforcing immune suppression. Further analysis of The Cancer Genome Atlas (TCGA) dataset led to the development of a prognostic model based on OXPHOS-related genes, showing strong predictive power across multiple cohorts. These results suggest a novel OXPHOS-ribosome-M2 polarization axis and offer potential therapeutic targets for immunometabolic therapies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d98d705964e6108c751cf9dce67e7948e856f2d" target='_blank'>
              M2 macrophages and tumor cells engage in a metabolic feedback loop to drive HCC progression
              </a>
            </td>
          <td>
            Yimin Nong, Xiaolong Chen, Baoxin Zhang, Zhenying Lin, Wuriniletu, Chengyou Du, Qiang Liu
          </td>
          <td>2025-11-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocytes (TILs) play a critical role in the ability of the immune system to combat cancer, offering a foundation for personalized immunotherapies. However, the effectiveness of TILs is often reduced by problems like becoming less active, the tumor making the immune system weaker, and not lasting long in the tumor environment. Recent advancements in single-cell technologies, including single-cell RNA sequencing (scRNA-seq), single-cell T-cell receptor sequencing (scTCR-seq), and mass cytometry (CyTOF), have revolutionized our understanding of TIL heterogeneity and dynamics. These tools offer new perspectives on the diverse phenotypes, functional states, and spatial organization of TILs, enabling the identification of key exhaustion markers, regulatory pathways, and neoantigen-specific clones. Concurrently, genetic reprogramming strategies have emerged to address TIL limitations by reversing exhaustion, enhancing metabolic resilience, and improving persistence in vivo. This review explores the synergistic integration of single-cell technologies and genetic engineering in refining TIL-based therapies. We talk about how spatial transcriptomics can help us understand how TILs work in different areas of the body and how changing their epigenetics can help them become more effective at fighting cancer. Additionally, we highlight emerging approaches to overcome immunosuppressive barriers in the tumor microenvironment (TME), including targeting regulatory immune cells, neutralizing suppressive cytokines, and enhancing antigen presentation. Together, these strategies promise to unlock the full therapeutic potential of TILs, paving the way for more effective and durable cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8024b4f684c7b9b9c99e6af201613f98c8efa0" target='_blank'>
              Probing and enhancing tumor-infiltrating lymphocytes: insights from single-cell technologies and genetic reprogramming
              </a>
            </td>
          <td>
            Chaitanya Kumar, Ravi Verma, Ashok Sharma, V. V. Priya
          </td>
          <td>2025-11-17</td>
          <td>Exploration of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chemotherapy resistance remains a major clinical challenge in ovarian cancer treatment, limiting therapeutic efficacy and patient survival. Understanding the molecular mechanisms underlying chemoresistance at the single-cell level is crucial for developing targeted therapeutic strategies. We performed comprehensive single-cell RNA sequencing analysis on primary and chemoresistant ovarian cancer samples to characterize cellular heterogeneity and identify resistance-associated molecular signatures. Cell type identification, differential gene expression analysis, functional enrichment analysis, and immune microenvironment characterization were conducted to elucidate chemoresistance mechanisms. Computational predictions were validated by RT-qPCR analysis of four key resistance genes (BIRC5, ABCB1A, ABCG2, BCL2) in SKOV3 and SKOV3-CDDP cell lines. Our analysis identified nine distinct cell populations within the ovarian cancer microenvironment, revealing extensive transcriptional reprogramming in chemoresistant tumors. Key chemoresistance genes including Abcb1a, Abcg2, Bcl2, Birc5, Nek2, Inhba, Irs1, and Prc1 showed differential expression patterns across cell types. Epithelial cells and proliferating cells exhibited elevated expression of multiple resistance markers, particularly Birc5. Functional analysis revealed enrichment of ribosome biogenesis, protein synthesis machinery, and amoeboid-type cell migration pathways in resistant cells. Macrophage polarization states were significantly altered, and immune checkpoint molecules showed coordinated upregulation, indicating comprehensive immune microenvironment reprogramming. RT-PCR validation confirmed single-cell predictions with significant upregulation in resistant cells: BIRC5 (4.2-fold, p < 0.001), ABCB1A (3.8-fold, p < 0.001), ABCG2 (2.9-fold, p < 0.01), and BCL2 (2.1-fold, p < 0.05). This study provides a comprehensive single-cell atlas of chemoresistance in ovarian cancer, revealing cell type-specific resistance mechanisms and immune evasion strategies. The identified molecular signatures offer potential therapeutic targets for overcoming chemotherapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f229e22beb03e0ca3194ad4df8f24784e9b8331" target='_blank'>
              Single cell transcriptomic analysis reveals molecular mechanisms of chemoresistance and immune microenvironment remodeling in ovarian cancer
              </a>
            </td>
          <td>
            Xia Hu, Caifang Du, Dongjie Du, Yonghua Zhang, Haijian Gao, Minxian Tao
          </td>
          <td>2025-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed06b707f53d2e1de85e352e5c2852fcf93afdb7" target='_blank'>
              Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer
              </a>
            </td>
          <td>
            Maria Vidal-Sabanés, R. Navaridas, Núria Bonifaci, Ada Gay-Rua, Damià Ortega-Peinado, Joaquim Egea, Mario Encinas, Xavier Matias-Guiu, David Llobet-Navás, Xavier Dolcet
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Phenotypic plasticity plays a key role in cancer progression and metastasis, enabling cancer cells to adapt and evolve, but precisely how distinct axes governing phenotypic plasticity interact to shape tumour progression and patient outcomes remains unclear. We investigated five major interconnected axes of plasticity in ER-positive (ER+) breast cancer: Metabolic Reprogramming, Epithelial-to-Mesenchymal Plasticity (EMP), Luminal–Basal (Lineage) Switching, Stemness, and Drug-resistance using network dynamics simulations, integrative bulk and single-cell transcriptomic analyses and patient survival analyses. We show that these axes are not independent but drive one another, forming two mutually inhibiting ‘teams’ of nodes enabling specified cellular behaviour. One team (favouring high glycolysis, stem-like, basal-like, mesenchymal/hybrid and tamoxifen-resistant phenotype) was found to be associated with aggressive progression and worse survival. On the other hand, the opposing team (favouring high oxidative phosphorylation, non-stem-like, luminal-like, epithelial and tamoxifen-sensitive phenotype) correlated with better outcomes. Importantly, altering one axis of plasticity often drove coordinated responses along other axes and vice versa. Our findings establish phenotypic plasticity in cancer as a coordinated, multi-axis dynamical process, thus suggesting novel strategies to disrupt systems-level reprogramming enabling metastasis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acaf4b9f2872701e5c44aad69b0ee83629b9228" target='_blank'>
              Interconnected axes of phenotypic plasticity drive coordinated cellular behaviour and worse clinical outcomes in breast cancer
              </a>
            </td>
          <td>
            Ritesh Kumar Meena, Yijia Fan, Soundharya Ramu, Anjaney J. Pandey, Abhinav Kannan, Yashvita Subramanian, Yukta Subramanian, Jason T. George, M. Jolly
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Although well-differentiated thyroid carcinoma (WDTC) is characterized by a robust treatment response, aggressive subtypes, such as anaplastic thyroid carcinoma (ATC), remain highly lethal. To understand thyroid cancer evolution in both children and adults, we analyzed single-cell transcriptomes of 423,733 cells from 81 samples and spatially resolved key tumor and microenvironment populations across 28 tumors with spatial transcriptomics, including rare and unique composite WDTC/ATC tumors and pediatric diffuse sclerosing thyroid carcinomas. Additionally, we identified gene signatures of stromal cell populations in 5 large thyroid cancer bulk RNA-sequencing cohorts. Through this multi-institutional effort, we defined a population of POSTN+ myofibroblast cancer-associated fibroblasts (myCAFs) that are intimately associated with invasive tumor cells and correlate with poor prognosis, lymph node metastasis, and disease progression in thyroid carcinoma. We also revealed a population of inflammatory CAFs that are distant to tumor cells and are found in the inflammatory stromal microenvironment of autoimmune thyroiditis. Together, our study provides spatial profiling of thyroid cancer evolution in samples with mixed WDTC/ATC histopathology and identifies a prognostic myCAF subtype with potential clinical utility in predicting aggressive disease in both children and adults.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9674219bbd66cbdd5e33b2f92b59cb54d20e94cd" target='_blank'>
              An integrated single-cell and spatial transcriptomic atlas of thyroid cancer progression identifies prognostic fibroblast subpopulations.
              </a>
            </td>
          <td>
            M. Loberg, George J. Xu, Sheau-Chiann Chen, , Claudia C. Wahoski, Kailey P. Caroland, Megan L. Tigue, H. Hartmann, Jean-Nicolas Gallant, Courtney J. Phifer, Andres A Ocampo, Dayle K. Wang, Reilly G. Fankhauser, K. Karunakaran, , Maxime Tarabichi, Sophia M. Shaddy, J. Netterville, Sarah L. Rohde, Carmen C. Solórzano, Lindsay A. Bischoff, Naira Baregamian, Barbara A. Murphy, Jennifer H. Choe, Jennifer R. Wang, Eric C. Huang, Quanhu Sheng, Luciane T. Kagohara, Elizabeth M. Jaffee, Ryan H. Belcher, K. Lau, Fei Ye, Ethan Lee, V. Weiss
          </td>
          <td>2026-01-09</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Objective: The aim of this study was to develop and apply a dual-scale Capillary-Cell (CapCell) microscope to quantify spatial and temporal heterogeneity in tumor metabolism and vasculature during anti-angiogenic therapy. Impact Statement: This study introduces a dual-scale CapCell microscope, a novel imaging system to dynamically visualize metabolic and vascular adaptations in vivo. The platform reveals subregional features associated with treatment that are often missed by bulk analyses. Introduction: Tumor recurrence is often driven by microenvironmental heterogeneity in metabolism and perfusion. Given the importance of metabolic reprogramming in treatment response, the dual-scale CapCell microscope was designed to capture widefield and high-resolution images of metabolic–vascular coupling in vivo. Methods: The dual-scale CapCell microscope was implemented to image multiple endpoints including mitochondrial membrane potential and glucose uptake (widefield and high-resolution images) that are colocalized with vessel density and distance between vessels (high resolution). The CapCell was used to image 4T1 tumors grown in an orthotopic window chamber longitudinally following treatment with Combretastatin A-1 (CA1), a vascular-disrupting agent. Imaging was performed over a period of 8 days to evaluate the effects of CA1 administered on days 1 and 5. Results: Treated tumors showed a significant decrease in metabolism and vessel fraction, and a significant increase in the distance between vessels immediately following the first treatment. Within microregional areas, elevated mitochondrial activity was associated with vascular-dense regions, whereas increased glucose uptake was more apparent in less vascularized regions. Interestingly, the second treatment on day 6 had little effect on the tumor metabolism, and in fact, metabolism at this time point recovered to baseline levels despite a persistent reduction in vessel area fraction and no corresponding recovery in vascular proximity. Conclusion: The CapCell enables dual-scale, multiparametric imaging of tumor microenvironments, capturing spatial metabolic and vascular features often linked to poor therapeutic outcomes. This platform can inform therapeutic timing and guide the development of combination strategies by resolving critical tumor subpopulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11842a0eeea1ee8ef76c8bdfeeaff619c27c19c3" target='_blank'>
              Quantifying Spatiotemporal Heterogeneity of Tumor Metabolism and Vasculature with a Multiparametric Point-of-Investigation Microscope
              </a>
            </td>
          <td>
            Enakshi D Sunassee, Marcia Cunha Dos Santos, Riley J. Deutsch-Williams, Sanjana Sankholkar, Megan C. Madonna, Gregory M Palmer, N. Ramanujam
          </td>
          <td>2025-11-12</td>
          <td>BME Frontiers</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Rising incidence of early-onset cancers demands experimental systems that model how normal tissues traverse premalignant states to form clinically relevant, therapy-responsive tumors, and how host immunity and ancestry-linked genetics shape that trajectory. A major obstacle has been the lack of tractable in vivo platforms that enable genome-wide discovery while preserving continuous tumor evolution without chromosomal instability. We developed Stochastically Emergent Tumors (SETs), an organoid-derived in vivo evolution engine that addresses this gap and reframes discovery for early-onset and understudied patient groups. In this system, mismatch repair deficiency (MMRD) is induced in nonmalignant human organoids, which are passaged to accumulate stochastic point mutations and transplanted into mice to permit malignancy to emerge under authentic selective pressures. SETs evolve primarily through high-resolution point mutations rather than broad copy number changes, producing bioinformatically tractable clonal dynamics ideally suited for whole-genome driver discovery and machine learning. Compared with conventional models, SETs exhibit increased intertumoral heterogeneity and reproducible recovery of sensitizing and resistance alleles under selective pressure. As proof of principle in prostate cancer, endocrine therapy applied to prostate SET pools recovered known determinants of sensitivity and revealed novel drivers. Loss of ZFHX3 function, typically obscured within a multi-gene tumor suppressor locus in bulk cohorts, promoted luminal histology and sensitized tumors to androgen pathway inhibition in vivo, while alterations in KMT2D and CIC mediated resistance. Consistent with model predictions, ZFHX3 loss in patients was associated with markedly increased survival, a novel finding comparable in magnitude to the longest-surviving molecular subtypes of advanced prostate cancer. These results identify selective vulnerabilities that arise from stochastic evolution. We also defined an oncogenic threshold by relating pregraft mutation burden to tumor incidence: in a Pten-null background, roughly 900 coding mutations accumulated over 208 days were sufficient for malignant transformation in half of grafts. This quantifies the median number of stochastic events required for premalignant-to-malignant transition and establishes a measurable axis to assess how genetics and microenvironmental pressures shift that threshold. Because SETs generate neoantigen-rich point mutation landscapes, they can be adapted to immunocompetent and premalignant paradigms to interrogate tumor–immune coevolution. Coupled with spatial and functional readouts, SETs enable mapping of early T-cell surveillance, macrophage-mediated immune exclusion, and myeloid checkpoints that permit immune escape. SETs therefore provide a scalable, evolution-aware platform to identify lineage- and ancestry-associated drivers, immunopreventive targets, and biomarkers of early-onset cancer.



 Ruhollah Moussavi-Baygi, Matthew Ryan, Woogwang Sim, Samuel Hoelscher, Valbona Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, Cornelia Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, Charles Sawyers, Felix Feng, Hani Goodarzi, Rohit Bose. Reconstructing evolution with Stochastically Emergent Tumors (SETs) reveals in vivo vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr IA006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5e22cb0b560b5c34832027ffccc947dda1ce65" target='_blank'>
              Abstract IA006: Reconstructing evolution with Stochastically Emergent Tumors (SETs) reveals in vivo vulnerabilities
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel Hoelscher, V. Luga, Arun Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, Danika Bakke, C. Ding, Bradley Stohr, Peng Jin, Tejasveeta Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R Karthaus, Charles L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The spatial organization of the tumor microenvironment (TME) profoundly influences cancer biology. However, the cell types and spatial distribution driving lung adenocarcinoma (LUAD) invasion remain poorly understood. By integrating spatially resolved transcriptomics with scRNA-seq data, we identify a novel secretory leukocyte protease inhibitor (SLPI)-expressing AT2-like subpopulation that localizes specifically at the invasive tumor margin, which drives the transition from minimally invasive (MIA) to invasive (IA) LUAD. Functional characterization reveals that SLPI-expressing AT2-like cells upregulated Dickkopf-1(DKK1), enhancing tumor cell stemness and epithelial-mesenchymal transition (EMT). Furthermore, spatially co-localized MRC1-expressing resident tissue macrophages (RTM-TAMs) secrete pro-tumor cytokines upon interaction with SLPI+ tumor cells, alongside cancer-associated myofibroblasts (myo-CAFs) exhibiting reduced type I collagen production. These TME components establish a pro-tumorigenic niche and engaged in synergistic interactions with SLPI+ AT2-like cells to facilitate LUAD invasion. These findings reveal the specific cellular composition, spatial architecture, and functional crosstalk between tumor cells and TME subpopulations that orchestrated LUAD progression. The frontier molecular targets at the tumor invasive identified in this study can serve as a basis for developing novel therapeutic targets in the future and assist pathologists in accurately assessing patients' disease progression and survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef641f6801c52b34b7967192c024cc7cd0f87f78" target='_blank'>
              SLPI⁺ AT2-Like Cells Orchestrate Lung Adenocarcinoma Invasion via Wnt Pathway Activation and Stromal Crosstalk in a Spatially Defined Margin Niche.
              </a>
            </td>
          <td>
            Zhoufeng Wang, Guonian Zhu, Pan Tang, Yan Wang, Wenxin Luo, Wenpeng Song, Zhikang Pan, Bingjie Zheng, YongChao Jiang, Defu Xiao, Xin Jin, Yong Bai, Guowei Che, Weimin Li
          </td>
          <td>2025-11-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Metastatic outgrowth requires that cancer cells delaminate from the primary tumor, intravasate, survive in circulation, extravasate, migrate to, and proliferate at a distal site. Recurrent genetic drivers of metastasis remain elusive, suggesting that unlike the early steps of oncogenesis, metastasis drivers may be variable. We develop a framework for identifying metastasis regulators using CRISPR/Cas9-based screening in a genetically defined organoid model of colorectal adenocarcinoma. We conduct in vitro screens for invasion and migration alongside orthotopic, in vivo screens for gain of metastasis in a syngeneic mouse model. We identify CTNNA1 and BCL2L13 as bona fide metastasis-specific suppressors which do not confer any selective advantage in primary tumors. CTNNA1 loss promotes cell invasion and migration, and BCL2L13 loss promotes anchorage-independent survival and non-cell-autonomous changes to macrophage polarization. This study demonstrates proof of principle that large-scale genetic screening can be performed in tumor-organoid models in vivo and identifies novel regulators of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33c9f8937cc2d0d69608a2acde03a6c6afb3fab8" target='_blank'>
              Forward genetic screening in engineered colorectal cancer organoids identifies regulators of metastasis.
              </a>
            </td>
          <td>
            Xin Wang, Zvi Cramer, N. A. Leu, Keara Monaghan, K. Durning, S. Adams-Tzivelekidis, Joshua Rhoades, Jonathan Heintz, Yuhua Tian, Joshua Rico, Diego Mendez, R. Petroni, Austin King, Melissa S Kim, Rina Matsuda, Olivia Hanselman, Alice E. Shin, María F Carrera Rodríguez, Igor E. Brodsky, Anil Rustgi, Ning Li, Christopher Lengner, M. Andres Blanco
          </td>
          <td>2025-11-11</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="BACKGROUND
Meningioma brain invasion encumbers surgical resection and increases the risk of tumor recurrence, but the molecular mechanisms underlying this process are poorly understood.


METHODS
To identify molecular and cellular features of brain-invasive meningiomas, we (1) analyzed bulk RNA sequencing data from 199 meningiomas, including 33 brain-invasive tumors, (2) analyzed patient-matched single-cell RNA sequencing data of spatially mapped meningioma samples from the tumor core or brain-tumor interface (BTI), and (3) performed spatial transcriptomic sequencing of brain-invasive meningioma samples. Multiplexed immunofluorescence (IF) was used to validate bioinformatic spatial expression patterns. Functional interactions between meningioma cells and neurons were studied in meningioma/neuron co-cultures using confocal microscopy, multi-electrode array recordings, and live cell calcium imaging.


RESULTS
Transcriptomic analyses showed conserved enrichment of TGM2, S100A11, ZYX, and PDGFRA at the BTI across bulk, single-cell, and spatial RNA sequencing datasets. The expression of these genes at the BTI was confirmed using multiplexed IF, and single-cell bioinformatic and microscopy analyses further demonstrated enrichment of macrophages at the BTI. Co-culture assays showed neuronal hyperexcitability and increased proliferation of meningioma cells, suggesting functional communication between meningioma cells and the tumor microenvironment may contribute to meningioma growth in cases with brain invasion.


CONCLUSIONS
Meningioma brain invasion is defined by molecular remodeling of tumor cells and functional interactions within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4352a0cf42108bac27d918f350130e856ef324b" target='_blank'>
              Meningioma cell reprogramming and microenvironment interactions underlie brain invasion.
              </a>
            </td>
          <td>
            Ayush Aggarwal, Mark W. Youngblood, T. Picart, H. Najem, S. Oten, M. A. Cady, Stephen T. Magill, Craig M Horbinkski, James P. Chandler, A. Heimberger, K. Mirchia, S. Hervey-Jumper, David R. Raleigh
          </td>
          <td>2025-12-31</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e28d12e48520ac375b3cbad04e1c6d130662c96" target='_blank'>
              Deciphering Cancer Evolution Through Genomic Profiling of Patient-Derived Xenograft Together with Matched Primary Gallbladder Cancer.
              </a>
            </td>
          <td>
            Vipin Yadav, R. Kilambi, Archana Rastogi, Deepti Sharma, Perumal Nagarajan, Varun Suroliya, Anupam Bhattacharya, Vaishali Yadav, Pramod Gautam, Manju Kashayp, M. K. Kashyap, Vanya Garg, Anupama Kumari, N. Trehanpati, G. Ramakrishna
          </td>
          <td>2025-11-10</td>
          <td>Digestive diseases and sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Metastatic prostate cancer (mPC) is characterized by molecular and phenotypic heterogeneity. With increasing guideline‐driven use of metastatic biopsies, more mPC specimens are being evaluated in surgical pathology. However, unlike localized prostate cancer, no standardized framework currently exists to guide the diagnostic workup of metastatic biopsies or reliably determine phenotypic subtypes. While many mPCs retain conventional acinar features, a growing subset exhibits phenotypic plasticity – including loss of prostate epithelial identity and emergence of neuroendocrine or other divergent lineages. This phenotypic diversity often occurs in castration‐resistant prostate cancer as a mechanism of resistance to chronic androgen receptor pathway inhibition and is characterized by genomic alterations and epigenetic reprogramming. This review outlines the histologic and molecular spectrum of mPC and proposes a practical, pathology‐informed diagnostic approach integrating morphologic assessment and immunohistochemistry. Adoption of a standardized diagnostic framework and multidisciplinary integration will be useful for employing precision oncology in advanced mPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de38a46f132d3e7836729ee1ba6aae01604d174" target='_blank'>
              Molecular subtypes of metastatic prostate cancer: from pathophysiology to diagnosis
              </a>
            </td>
          <td>
            J. M. Dsouza, Erolcan Sayar, M. Schweizer, Stephanie Harmon, C. Morrissey, H. Beltran, Peter S. Nelson, Liang Cheng, C. C. Ding, Michael C. Haffner
          </td>
          <td>2025-12-12</td>
          <td>Histopathology</td>
          <td>1</td>
          <td>57</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common type of primary liver cancer. Tumor heterogeneity is a major obstacle to effective treatment and is poorly understood using traditional bulk sequencing methods. This review highlights the transformative role of single-cell and multi-omics technologies in determining the cellular and molecular complexities of HCC. We summarize recent advances in single-cell transcriptomics, epigenomics, multi-omics, and spatial transcriptomics platforms, emphasizing their applications in characterizing tumor subclones, cancer-associated fibroblast-immune interactions, circulating tumor cells, and immune-resistant phenotypes. Spatial approaches have revealed the architecture of cancer stem cell niches and tertiary lymphoid structures, providing unprecedented insights into tumor organization and microenvironmental crosstalk. Although still in their early stages, clinical trials have begun to incorporate these technologies, underscoring their translational potential. Single-cell and spatial omics have reshaped HCC research by enabling high-resolution profiling of tumor ecosystems and driving the discovery of biomarkers, therapeutic targets, and strategies for patient stratification. However, high cost, technical expertise, and limited accessibility, particularly in resource-constrained settings, are major barriers to its widespread adoption. Addressing these challenges is critical for translating these powerful approaches into clinical practice and for advancing precision medicine for the treatment of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0abad77c5fcae33c11d7d851ad3eb337d3881a5" target='_blank'>
              Exploring Single-Cell and Multi-Omics Technologies and Their Role in Unravelling Tumor Heterogeneity of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Charmi Jyotishi, Suresh Prajapati, Mansi Patel, Reeshu Gupta
          </td>
          <td>2025-12-17</td>
          <td>Journal of liver cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Aggrephagy plays a pivotal role in ovarian cancer (OC) progression by clearing misfolded proteins. However, the functional role of aggrephagy within the tumor microenvironment (TME) remains elusive. Non-negative matrix factorization (NMF) was employed to analyze 43 aggrephagy-related genes (ARGs) on 65,820 single cells from scRNA-seq data of 12 samples (7 OC tissues and 5 normal tissues). The OC cohort from 9 public databases and TIDE-based immunotherapy scores were used to identify prognosis and immune response patterns of these TME cell clusters. Furthermore, the association between ARGs and CD8+ T cells was validated by the ID8 ovarian cancer mouse subcutaneous tumor model. Fibroblasts, T cells, macrophages and B cells identified from scRNA-seq of OC samples were subjected to biological process enrichment and classified based on ARGs expression patterns. Additionally, it revealed that ARGs may be significantly associated with clinical and biological characteristics of OC and pseudotime trajectories of major TME cell types. Bulk-seq analysis revealed that these aggrephagy-associated TME cell subsets were closely linked to the prognosis of OC patients and exhibited significant heterogeneity in predicting response to immunotherapy. Among them, CAF and CD8+ T cells showed particularly significant immune responsiveness. Notably, CellChat analysis revealed that aggrephagy-associated subtypes of TME cells engage in diverse and extensive interactions with tumor epithelial cells. Meanwhile, significant co-localization of TUBA1B with CD8+ T cells was observed in OC tissues from the mice model. This study is the first to delineate the role of aggrephagy regulators-mediated intercellular communication of the TME in regulating OC growth and antitumor immunomodulation. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-04060-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f5dbf13eb9670595afcc13d733cac2504d66e7a" target='_blank'>
              Single-cell aggrephagy landscape delineates intercellular communication of tumor microenvironment associated with ovarian cancer progression and immunotherapy
              </a>
            </td>
          <td>
            Feng Fang, Jiemiao Chen, Xinran Li, Cao Zhou, Yilan Dai, Yuanyuan Chen, Xin Yang, Yong Xie, Fujie Qin, Huijiao Fu
          </td>
          <td>2025-11-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Understanding CD8+ tissue-resident memory T cells (TRM) spatial characteristics in hepatocellular carcinoma (HCC) is challenging, and clarifying the spatial feature changes following immunotherapy represents an urgent research gap. Methods This study employs a multi-omics approach to analyze the spatial distribution and intercellular interactions of TRM cells in HCC tissues using radiomics, single-cell sequencing, and multiplex immunofluorescence histochemistry (m-IHC). Results Our results show that the number of CD8+ TRM cells in HCC increases following immunotherapy. Furthermore, after dividing tumor tissues into the tumor core (TC), invasion margin (IM), and normal tissue (N), a increase in CD8+ TRM cells from the IM to the TC can be observed. Consistent with the results of single-cell sequencing analysis, this change in spatial characteristics may be associated with the interactions between CD8+ TRM cells and CD68+ cells. Discussion Immunotherapy can modify the spatial characteristics of CD8+ TRM cells via regulating their crosstalk with other immune cells, and the spatial distribution of CD8+ TRM cells in the HCC tumor microenvironment (TME) correlates with immune checkpoint blockade (ICB) therapeutic efficacy. Clarifying the mechanisms of action of immunotherapeutic drugs and developing a non-invasive radiomics model to predict CD8⁺ TRM cell dynamics will facilitate the clinical management of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/384dbc51e0f9efcd6dc85b035d8ba5b7b014c96f" target='_blank'>
              Multi-omics spatial characteristics of CD8+TRM cells in hepatocellular carcinoma and immunotherapy response prediction
              </a>
            </td>
          <td>
            Shipeng Li, Ju Ma, Xinqiang Li, Bingbing Qiao, Liancai Wang, Deyu Li, Hua Zhou, Haixu Xu, Jinzhen Cai
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2e088dcd4f395d6c7df482c6c35fcef0137dc41" target='_blank'>
              An epigenetic switch in vascular phenotype augments anti-tumor immunity
              </a>
            </td>
          <td>
            Dae Joong Kim, Mitchell T McGinty, S. Anandh, C. Riedstra, Yuvraj Sethi, Melanie R. Rutkowski, A. C. Dudley
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Effective antitumor immunity ultimately depends on the priming and activation of tumor-specific cytotoxic CD8+ T cells; however, the role of intratumoral cell-cell immune interactions remains incompletely understood. Recent work has revealed that the temporospatial colocalization of dendritic cells (DCs), helper T cells (Th), and cytotoxic T lymphocytes (CTL) within the tumor immune microenvironment following immune checkpoint blockade correlates with clinical response. Herein, we report the integration of more than one million spatially resolved single-cell profiles across six spatial proteomic and transcriptomic assays, which demonstrated that DC:Th:CTL three-cell-type clusters were common even in immunotherapy-naïve and highly desmoplastic tumors, such as fibrolamellar carcinoma and pancreatic ductal adenocarcinoma (PDAC). We found that these immune triads were enriched for functionally important type 1 conventional DC, mature DCs enriched in immunoregulatory molecules (mregDC), CXCL13+ Th, and GZMK+ effector CTL phenotypes. Subsequent multiplex immunofluorescence imaging of more than 450 primary PDAC tumors showed that the density of antigen-presenting cell (APC):Th:CTL three-cell-type clusters was correlated with intratumoral T-cell clonal expansion and improved overall survival. These findings suggest that DC:Th:CTL triads are conserved across solid tumors and highlight the importance of intratumoral spatial niches in mediating endogenous antitumor immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e981c34b939efc05dce5ef8fec4e92e8da12dbd" target='_blank'>
              Intratumoral three-cell-type clusters are a conserved feature of endogenous antitumor immunity.
              </a>
            </td>
          <td>
            Sheela R. Damle, J. A. Carter, K. E. Goodsell, Jose M B Pineda, L. K. Dickerson, Xiuyun Jiang, Jacqueline L Mudd, Thomas Walsh, H. Kenerson, Jack Cernak, S. S. B. Chauhan, Emily Beirne, Sujata Jana, Amanda L. Koehne, Kiran R Vij, Marianna B Ruzinova, R. Yeung, Shreeram Akilesh, Eric A. Collisson, Ryan C Fields, I. N. Crispe, Venu G. Pillarisetty
          </td>
          <td>2025-11-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Tracking small-molecule distribution in heterogeneous cell samples at single-cell resolution remains a major analytical challenge. Here, we present a tellurophene-functionalized analogue of the proteasome inhibitor Carfilzomib (TeCar) whose distribution can be followed by mass cytometric (MC) quantification while preserving target engagement and cytotoxicity. Structural and biochemical analyses confirm that TeCar binds the proteasome in a mode comparable to the clinically approved parent compound. Using MC, we demonstrate selective TeCar accumulation in malignant over immune cells within mixed populations, with cancer cells exhibiting 15 to 30-fold higher uptake. Tellurium signal correlates with proteasomal activity, and differential labeling among immune subsets reveals functional heterogeneity not captured by transcriptomics alone. These findings establish tellurophene tagging as a minimally perturbing and broadly applicable strategy for functional distribution studies at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297f01c1c4f8d9a53fbe0c9fdab22012abff3851" target='_blank'>
              Tellurophene-Tagged Carfilzomib Enables Single-Cell Mass Cytometric Mapping of Proteasome Activity.
              </a>
            </td>
          <td>
            Nicole Potter, A. Eddenden, Aleksandra Fomina, Anuroopa Dinesh, H. Jackson, A. McGuigan, M. Groll, Mark Nitz
          </td>
          <td>2025-11-18</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24dfa7ddd758cdfb84fb3b4e9f07b2ebd47b469" target='_blank'>
              Hidden-driver inference reveals synergistic brain-penetrant therapies for medulloblastoma
              </a>
            </td>
          <td>
            Jingjing Liu, Xu Yang, Mingrui Zhu, Xinran Dong, Honglei Zhou, B. Bianski, Barbara Jonchere, Wenwei Lin, Xiang Fu, Lei Yang, Burgess B. Freeman, Abigail S. Wang, Ruilin Jiang, Taosheng Chen, G. Robinson, M. Roussel, T. E. Merchant, Amar Gajjar, Jiyang Yu
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Therapeutic failure in cancer often arises from the ability of tumor cells to adapt to external stressors such as drug exposure, metabolic and oxidative stress, and immune surveillance within the tumor microenvironment (TME). This adaptive capacity is a significant factor contributing to poor clinical outcomes and the lethality associated with many aggressive cancer types. During tumor progression, a subpopulation of cells may acquire stem-like characteristics, leading to the development of cancer stem cells (CSCs), which play a pivotal role in resistance and disease recurrence. A key driver of this process is epigenetic reprogramming, which alters histone modifications and chromatin architecture in response to environmental stimuli. These modifications activate genes associated with stemness by creating transcriptionally permissive chromatin regions, while simultaneously silencing genes involved in differentiation, thereby maintaining the tumor cells’ undifferentiated state. This cellular plasticity enables cancer cells to undergo transdifferentiation or dedifferentiation, enhancing their capacity for survival and adaptation. Importantly, distinct transcriptional states are often regulated by specific epigenetic signatures, some of which are valuable for tumor subtype classification, prognosis assessment, and treatment response prediction. Furthermore, certain epigenetic biomarkers show considerable promise for detecting tumor recurrence and monitoring minimal residual disease with high sensitivity. In this review, we examine how epigenetic reprogramming influences chromatin plasticity to promote stemness and therapeutic resistance. Additionally, we discuss ongoing translational research aimed at leveraging these mechanisms to advance precision oncology, with the ultimate goal of improving diagnostic accuracy and therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72837cc909b2a1ccbe8a1449f9aafc89b0a2cfa3" target='_blank'>
              Epigenetic reprogramming as the nexus of cancer stemness and therapy resistance: implications for biomarker discovery
              </a>
            </td>
          <td>
            Chu Xin Ng, Shin Yuh Lee, Xin Yi Yap, Yong Hui Wong, Jian Sheng Loh, Kuan Ping Ang, Wen Hwei Lee, Wei Xu, Phelim Voon Chen Yong, Sau Har Lee
          </td>
          <td>2025-11-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7358fd5e226c097b2efb1a3758a0fef198cc402b" target='_blank'>
              Single cell spatial profiling of the matrisome identifies region-specific adhesion and signaling networks in glioblastoma.
              </a>
            </td>
          <td>
            Arpan De, Santiago A Forero, Ali Pirani, John E. Morales, Marisol De La Fuente-Granada, Sumod Sebastian, Jason T Huse, Leomar Ballester, J.S. Weinberg, Frederick F. Lang, Kadir C. Akdemir, Joseph H. McCarty
          </td>
          <td>2025-12-09</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a resilient subpopulation within tumors, capable of driving progression, metastasis, and recurrence. One mechanism that enables this plasticity is asymmetric cell division (ACD), a process by which CSCs generate both a self-renewing stem cell and a differentiated daughter cell. While traditionally associated with tissue development, ACD is now recognized as a dynamic and regulated feature of cancer biology, particularly in response to stress conditions such as hypoxia and radiation. In this review, we provide a comprehensive and mechanistic synthesis of how intrinsic factors such as polarity complexes, cell fate determinants (CFDs), spindle orientation cooperate with extrinsic cues from the tumor microenvironment to orchestrate ACD in CSCs. We explore how this process contributes to tumor heterogeneity, therapy resistance, and the emergence of quiescent, drug-tolerant CSCs across multiple malignancies, including brain, breast, colorectal, and hematologic cancers. Importantly, we highlight recent efforts to pharmacologically disrupt or redirect ACD using inhibitors of NOTCH, WNT, AURORA kinases, and MSI1, presenting ACD as a therapeutic vulnerability rather than a static trait. By shifting the focus from CSC markers to division mode, this review introduces a novel conceptual framework for targeting tumor hierarchy and plasticity. Understanding and manipulating ACD offers a promising frontier in precision oncology - one where altering the balance of cell fate decisions could limit relapse, reduce intratumoral complexity, and enhance long-term treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b911960284166687ce9462f6da53a77bba5e7d79" target='_blank'>
              Targeting asymmetric division in cancer stem cells: implications for relapse and resistance
              </a>
            </td>
          <td>
            Mona Latifi, Mahsa Sani, F. Sani
          </td>
          <td>2025-12-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Epithelial compartments play a central role in the progression and metastasis of colorectal cancer (CRC), yet the mechanisms underlying their transcriptional reprogramming across disease stages remain incompletely understood. We constructed comprehensive single-cell and spatial transcriptomic atlases encompassing normal colorectal tissue, primary CRC, and liver metastases (LMs), profiling over 200,000 high-quality cells. Specific coexpression modules and transcriptional regulators were identified and spatially mapped. Additionally, the clinical and functional roles of SCAND1 were investigated using western blot, RT-qPCR, and immunohistochemistry in clinical samples, as well as functional assays in CRC cell lines. Distinct coexpression modules were selectively enriched in normal epithelial cells and associated with vesicle transport, ion homeostasis, and barrier function, whereas LM-specific modules were linked to Wnt signalling, ribosome biogenesis, and cell cycle processes. High-CNV epithelial cells exhibited progressive activation of key transcription factors such as CEBPB, E2F, and the AP-1 family, with concurrent suppression of ARGLU1 and MTNR2L8, indicating coordinated remodelling of transcriptional and metabolic programs during metastasis. These changes were spatially validated and correlated with alterations in epithelial composition within the tumour core. Consistently, multiplex immunofluorescence (mIHC) confirmed the expression patterns of CEBPB, ARGLU1, S100A4 and FOSL1, across normal epithelium, CRC, and LM. Notably, SCAND1 was identified as a potential therapeutic target for malignant epithelial plasticity. We demonstrated for the first time that SCAND1 expression is significantly upregulated in CRC patients and is associated with superior diagnostic and prognostic value. Functional experiments revealed that SCAND1 promotes tumour cell proliferation, inhibits apoptosis, and enhances metastatic potential. Furthermore, co-culture of SCAND1-overexpressing tumour cells with T cells showed that SCAND1 overexpression attenuates T cell–mediated cytotoxicity, supporting an additional immune-evasive role. Our study elucidates the molecular logic of epithelial state transitions during CRC metastasis and underscores the promise of SCAND1 as a novel biomarker and therapeutic target in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/914a9adf8fcce69d1ec7290317d881a8d12ed794" target='_blank'>
              Dynamic remodelling of epithelial plasticity in colorectal cancer from single-cell and spatially resolved perspectives
              </a>
            </td>
          <td>
            Chengyuan Xu, Yun Pan, Siqi Zhang, Moubin Lin, Hailong Liu, Zicheng Yu
          </td>
          <td>2025-11-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Advanced prostate cancer, particularly castration-resistant disease, remains challenging to treat due to intratumoral heterogeneity, immune exclusion, and a suppressive tumor microenvironment. Within this ecosystem, cancer-associated fibroblasts shape tumor–stroma communication, but their marked heterogeneity and plasticity complicate classification and make indiscriminate fibroblast depletion potentially ineffective or even harmful. This review summarizes recent progress in fibroblast origins, functional subtypes, and fibroblast-driven mechanisms that promote tumor progression and therapy resistance, as well as emerging therapeutic opportunities in prostate cancer. We conducted a structured literature search of PubMed, ScienceDirect, and major publisher platforms (including Nature and SpringerLink) from database inception to 15 February 2025, supplemented by targeted manual screening of reference lists. Evidence from single-cell/spatial-omics and mechanistic studies indicates that prostate tumors contain multiple fibroblast programs that occupy distinct niches yet can interconvert. Across these studies, it was found that these fibroblasts contribute to immune suppression, extracellular matrix remodeling and stromal barrier formation, angiogenesis, and metabolic support, collectively limiting drug penetration and reinforcing immune evasion; therapeutic pressure can further rewire fibroblast states and resistance-associated signaling. Overall, the literature supports a shift toward function- and subtype-directed intervention rather than “one-size-fits-all” targeting, with promising directions including precision targeting and reversible reprogramming, rational combination strategies, and localized delivery approaches that reduce stromal barriers while preserving tissue homeostasis in high-risk and treatment-refractory prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ab55e2c580e0e641540a891419637d2d1e74e2e" target='_blank'>
              Targeting Cancer-Associated Fibroblasts in Prostate Cancer: Recent Advances and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Peng Chen, Junhao Chen, Peiqin Zhan, Xinni Ye, Li Zhao, Zhongsong Zhang, Jieming Zuo, Hong-jin Shi, Xiangyun Li, Songhong Wu, Yuanzhi Fu, Haifeng Wang, Shi Fu
          </td>
          <td>2025-12-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastasis drives treatment failure and cancer mortality, yet preclinical studies still rely heavily on subcutaneous xenograft models that prioritize tumor growth over metastatic biology. This disconnection from clinical reality significantly contributes to the high failure rate of experimental therapies in trials. Here, current mechanistic insights is integrated into metastasis and critically assess transplantation models to inform rational model selection for metastasis research. Tumor transplantation models exhibit distinct dissemination patterns governed by implantation methodology rather than intrinsic tumor properties. Subcutaneous models, while technically accessible, predominantly assess primary tumor growth and fail to capture critical metastatic steps like intravasation, pre‐metastatic niche formation, and organotropism. Orthotopic transplantation faithfully replicates native tissue microenvironments, enabling simultaneous assessment of tumor growth and metastatic potential. Intravascular models, while inducing rapid colonization, distort natural metastatic progression by skipping early dissemination stages. Metastatic site transplantation isolates microenvironmental impacts on tumor adaptation but fails to capture de novo metastatic initiation. Ultimately, three strategies is proposed for preventing metastasis: Eradicating circulating tumor cells, blocking colonization, and stifling outgrowth. This perspective catalyzes the strategic advancement of tumor metastasis models, thereby strengthening the reliability of preclinical findings and accelerating their clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e9a5079eba6d77348e239cd02e9f7fe18d9f93" target='_blank'>
              A Critical Reassessment of Tumor Metastasis Simulation in Murine Models: Insights Into Methodological Advances and Biological Relevance
              </a>
            </td>
          <td>
            Xiaoxi Li, Weijun Ren, Ling Liu, Chuan Liu, Xiang Chen
          </td>
          <td>2025-11-17</td>
          <td>Advanced Therapeutics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Intratumoral heterogeneity supports cancer progression and metastasis while limiting therapeutic efficacy. Comprehensive biological characterization of heterogeneous subclones in breast cancer may hold promise in battling against this deadly disease. Using single-cell transcriptomic analysis and in situ profiling of primary tumors, metastases, and circulating tumor cells from multiple patients with breast cancer, we identified a unique EPCAM+ CD68+ TREM2+ tumor subpopulation, likely resulting from the fusion of tumor cells and lipid-associated macrophages (LAM). The presence of these tumor-LAM fusion cells in the blood or in distinct metastatic sites was significantly correlated with metastatic progression. Stable fusion clonal lines established in vitro exhibited substantially enhanced proliferation, tumor initiation, and metastasis formation in mice. Integrative molecular and functional analyses revealed a critical role for SNX10 in mediating tumor-LAM fusion. Mechanistically, SNX10 physically interacted with the phospholipid scramblase ANO6 and maintained its protein stability by suppressing proteasome-mediated degradation. Furthermore, the transcription factor CBX3 directly targeted SNX10 promoter and modulated its expression. Fusion cells accumulated abundant intracellular lipid droplets and were highly sensitive to simvastatin treatment in vitro and in vivo. Together, this study uncovered that CBX3-SNX10-ANO6 signaling facilitates generation of an aggressive tumor-LAM fusion cell subpopulation that promotes metastasis, revealing an alternative metastatic mechanism and exposing putative therapeutic vulnerabilities.


SIGNIFICANCE
Single-cell transcriptomic profiling combined with functional and clinical validation identifies fusion of tumor cells and lipid-associated macrophages mediated by the CBX3-SNX10-ANO6 axis as a potentially targetable mechanism driving cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4271e1aeed14ff3c28d665da96c160262a128c0a" target='_blank'>
              Fusion of Tumor Cells with Lipid-Associated Macrophages Drives Metastatic Progression of Breast Cancer.
              </a>
            </td>
          <td>
            Yixin Cheng, Guanyin Huang, Xuefei Liu, Chun Wu, Jingru Lian, Jing Cao, Lianhui Duan, Boxi Zhao, Yufan Yang, Siqi Chen, Guanqiao Li, Liping Wu, Shuqian Zheng, Wenjing Wang, Tianyu Zha, Yang Yang, Yilin Lu, Jian Zhang, Hao Yu, Jun Tan, Feiqiu Wen, Zhenyu He, Xin Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Identifying cell types and subtypes in routine histopathology is fundamental for understanding disease. Existing tile-based models capture nuclear detail but miss the broader tissue context that influences cell identity. Current human annotations are coarse-grained and uneven across studies, making fine-grained, subtype-level classification difficult. In this study, we build a marker-guided dataset from Xenium spatial transcriptomics with single-cell resolution labels for more than two million cells across eight organs and 16 classes to address the lack of high-quality annotations. Leveraging this data resource, we introduce NuClass, a pathologist workflow inspired framework for cell-wise multi-scale integration of nuclear morphology and microenvironmental context. It combines Path local, which focuses on nuclear morphology from 224x224 pixel crops, and Path global, which models the surrounding 1024x1024 pixel neighborhood, through a learnable gating module that balances local and global information. An uncertainty-guided objective directs the global path to prioritize regions where the local path is uncertain, and we provide calibrated confidence estimates and Grad-CAM maps for interpretability. Evaluated on three fully held-out cohorts, NuClass achieves up to 96 percent F1 for its best-performing class, outperforming strong baselines. Our results demonstrate that multi-scale, uncertainty-aware fusion can bridge the gap between slide-level pathological foundation models and reliable, cell-level phenotype prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32b1de1af80a1656a995e86569031a135811200f" target='_blank'>
              Adaptive Multi-Scale Integration Unlocks Robust Cell Annotation in Histopathology Images
              </a>
            </td>
          <td>
            Yinuo Xu, Yan Cui, Mingyao Li, Zhi Huang
          </td>
          <td>2025-11-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="CD4 T follicular helper (Tfh) cells coordinate humoral immune responses within germinal centers (GC) of lymphoid tissue. Despite their critical roles in vaccination and autoimmunity, the gene expression programs that define functionally distinct human Tfh states— and the molecular pathways engaged by Tfh positioned within the GC niche—remain incompletely understood. This gap has limited translational efforts to monitor or therapeutically target specific Tfh states for clinical benefit. Here, we delineate human CD4 T cell heterogeneity in tonsils and peripheral blood using trimodal single-cell sequencing and spectral flow cytometry to define epigenomic, transcriptional, and proteomic features of distinct Tfh states. Tfh with a GC-like phenotype exhibited markedly increased chromatin accessibility and both mRNA and protein expression of G protein subunit gamma 4 (GNG4). In tonsil, single-cell spatial transcriptomics defined GNG4 expression as a distinguishing feature of activated Tfh states within spatially demarcated GC compartments, with greater specificity than conventionally GC-associated features such as BCL6, TOX2, and S1PR2. In contrast, GNG4− Tfh primarily localized to nonGC regions and exhibited a resting, Th17-polarized phenotype. Together, these data highlight GNG4 as a central feature of activated, GC-positioned Tfh cell identity in humans. One Sentence Summary GNG4 expression defines activated CD4 T follicular helper cells localized to the germinal center of human lymphoid tissue.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e07a3ba4b7925ada4a1cba1ab2a33fb871cfc9" target='_blank'>
              Multimodal analysis defines GNG4 as a distinguishing feature of germinal center-positioned CD4 T follicular helper cells in humans
              </a>
            </td>
          <td>
            Sam Barnett Dubensky, Yutong Zhu, Molly Gallagher, K. Kumashie, Tianyu Lu, Jonathan Tedesco, Nina De Luna, Katherine Premo, Yi Qi, Suzanna Rachimi, Emylette Cruz Cabrera, Bria Fulmer, Ijeoma C. Meremikwu, Ashley Carter, Sarah E. Henrickson, N. Romberg, Amy E. Baxter, Derek A. Oldridge, Laura A. Vella
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Secreted phosphoprotein 1 (SPP1+) macrophages are a recurrent and functionally critical immune cell subset across multiple cancer types. They drive adverse clinical outcomes by promoting immunosuppression, tumor invasion, metastasis, and therapy resistance. Given their prevalence and pivotal role, SPP1+ macrophages have become a major focus in cancer immunology and a promising target for therapeutic development. SPP1+ macrophages have been identified in a wide range of human malignancies through single-cell RNA sequencing and spatial transcriptomics studies. Their differentiation and maintenance are strongly influenced by reciprocal cellular interactions and hypoxic conditions within the tumor microenvironment (TME). Within the tumor microenvironment (TME), SPP1+ macrophages promote tumor progression by interacting with cancer-associated fibroblasts (CAFs) and helping to form a physical barrier that restricts immune cell infiltration into the tumor core. Specifically, they impair the recruitment of CD8+ T cells and promote T cell exhaustion (TEX). In this review, we focus on recent advances in understanding the differentiation of SPP1 macrophages in hypoxic tumor microenvironment and the role of SPP1+ macrophages in immunosuppression and their therapeutic implications in cancer. Targeting this subset of macrophages has emerged as a highly promising therapeutic strategy, with several approaches demonstrating encouraging results in preclinical models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7c7489149a391c5b3e442dad42f10334718f488" target='_blank'>
              SPP1+ macrophages in tumor immunosuppression: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Juanjuan Wang, Ya Wang, Yuqing Liu, Rongcun Yang
          </td>
          <td>2025-12-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Plasticity, or the ability to rapidly and reversibly change phenotypes, may help explain how a single progenitor cell eventually generates a tumor with many different cell phenotypes. Normal colon plasticity is characterized by a conserved and broadly permissive epigenome, where expression and phenotype are determined by the microenvironment instead of epigenetic remodeling. To determine whether this stem-like plasticity is retained during progression, gene expression was measured with spatial transcriptomics and compared with gene-level DNA methylation in two colorectal cancers (CRCs). Like normal colon, genes that were differentially expressed between regions, subclones, and phenotypes (superficial, invasive, and metastatic) tended to have lower DNA methylation variability. We propose a quantitative signal of plasticity that correlates gene epigenetic variability with gene expression variability. In this framework, negative correlation implies phenotypic plasticity, as more variably expressed genes tend to have less epigenetic variability. We verify the presence of this signal in multiple external single-cell RNA-Seq datasets, in both normal colon and CRC samples. Therefore, the plasticity of normal colon appears to be retained during progression. A CRC progenitor with a preconfigured plastic phenotype is poised for rapid growth because it expresses, as needed, transcripts required for progression with minimal epigenetic remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86cc1dfed608d8a0144edef3180374f9ebee40c0" target='_blank'>
              Epigenetic conservation infers that colorectal cancer progenitors retain the phenotypic plasticity of normal colon
              </a>
            </td>
          <td>
            Kelly Street, Yifan Zhang, Kimberly Siegmund, Darryl Shibata
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdaa3f119622f6caaf2aaa901d9eb1cf04bbc193" target='_blank'>
              Hypergraph-driven spatial multimodal fusion for precise domain delineation and tumor microenvironment decoding.
              </a>
            </td>
          <td>
            Chengyang Zhang, Xulong Li, Bo Li, Chenxun Deng, Mengran Li, Shiqi Zhang, Weijiang Yu, Hongyu Zhang, Zheng Wang, Yuedong Yang, Yuansong Zeng
          </td>
          <td>2025-12-01</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Pancreatic neuroendocrine tumors (panNETs) undergo high-grade (HG) transformation often, with the liver being a common site for metastasis​. Surgical resection being considered as long-term outcomes for selected patients. However, recurrence and grade transformation remain potential significant challenges, with recurrence rates approaching 50–95%.1,2 Despite extensive whole exome analysis and construction of phylogenetic trees, we were unable to identify either a consistent genetic driver or any specific patterns of tumor transformation.2 We are also concerned that some of our treatments may inadvertently contribute to grade transformation. To assess the treatment effects, we performed pre- and post- trans arterial embolization bulk-transcriptome analysis, which revealed cell population shifts in panNETs that may be linked to grade transformation.4To explore cellular shifts within the liver tumor microenvironment during panNET progression, we performed single-cell RNA sequencing. The resulting data was processed using Seurat.v5. The clusters of panNET cells, fibroblasts, hepatocytes, and immune cells were annotated by known markers. Additionally, we distinguished malignant cells from a reference of immune cells, fibroblasts, and hepatocytes by InferCNV. To understand the tumor progression, we performed trajectory analysis using monocle3. Differential metabolic activity between the panNETs grades were assessed using the scMetabolism package. Cell-cell communication and related ligand-receptor pairs were inferred using CellChatDB. Our results revealed distinct transcriptional trajectories, with pseudotime analysis demonstrating a trajectory of tumor evolution from low-grade (LG) to HG states. HG tumors were enriched for proliferative and aggressive subclusters compared to LG cells. Metabolic profiling further highlighted grade-specific adaptations, with HG tumors showing enhanced oxidative phosphorylation and TCA cycle across tumor cells, whereas LG tumors displayed relatively lower metabolic activity. Notably, immune cells were found metabolically active in LG via glycolysis. Conversely, HG panNETs exhibit glucose depletion in immune cells and less active microenvironment. The CellChat analysis revealed distinct cellular communication patterns in panNET grade. For instance, in HG tumors, signaling was primarily mediated between fibroblasts and panNET cells, with Glu-(SLC1A2+GLS)-GRM7 as the dominant ligand–receptor pair. In LG tumors, major interactions occurred among immune cells, hepatocytes, and panNET cells, with Glu-(SLC1A3+GLS)-GRIK2 representing the highest contributor. The results suggest that metabolic rewiring, altered intercellular signaling, and transcriptional plasticity underlie progression from LG to HG disease.
 Reference:
 1. Spolverato etal J Gastrointest Surg. 2017, 21(12):2039-2047
 2. Mayo etal Ann Surg Onc. 2011, 18(13):3657-65.
 3. Singh et al. Early ... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium. 2023.
 4. Singh et al. Post... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium.2023



 Himanshu N. Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jungeui Hong, Nitya Raj, Olca Basturk, Zeynep Tarcan, Christine A. Iacobuzio, Etay Ziv. Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f19a2d46016dbf4a94424ac84c3f39682a28b2d" target='_blank'>
              Abstract B025: Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors
              </a>
            </td>
          <td>
            Himanshu Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jung-Pyo Hong, Nitya Raj, O. Basturk, Z. Tarcan, Christine A. Iacobuzio, E. Ziv
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Immunohistochemistry (IHC) remains the gold standard for evaluating protein expression in tumor microenvironment analysis. This approach hinders robust correlation analyses between spatial heterogeneity in the tumor microenvironment and clinical outcomes like disease-free survival (DFS). To address these challenges, we developed an automated pipeline for quantitative IHC feature extraction. Our method integrates deep learning-based tumor segmentation with computational detection of invasive margins at varying distances. Deconvolution algorithms quantify diaminobenzidine (DAB) staining intensity across the tumor body and the invasive margin. Spatial heterogeneous DAB density patterns were subsequently analyzed for DFS correlation. Using 104 patient samples (57 training/47 validation) stained for CD3, CD8, CD31, and HIF-1α, we identified 2 prognostic feature categories (CD3/CD8 aggregated positive areas within the 0.25-mm peripheral zone extending outward from the tumor-invasive front and HIF1-α-positive areas within a 0.75-mm peripheral zone extending outward from the tumor-invasive front). Immune-related features demonstrated C-indices of 0.726 (training) and 0.626 (validation), while hypoxia-associated markers showed C-indices of 0.714 and 0.656, respectively. Integration of these features with pTNM staging enhanced DFS stratification compared to pTNM staging alone, improving C-indices from 0.702 to 0.819 (training) and 0.668 to 0.853 (validation). This automated pipeline addresses critical limitations in traditional IHC analysis by enabling: 1) Objective quantification of spatial DAB heterogeneity. 2) Identification of biologically interpretable prognostic features. 3) Enhanced predictive performance over conventional staging systems. Our findings suggest this methodology could standardize IHC-based prognostic assessments and inform personalized treatment strategies. Further validation in multicenter cohorts is warranted to confirm clinical applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c85bd6804a873bad087f9c925d5c02405f8fb31f" target='_blank'>
              Automatically quantifying spatial heterogeneity of immune and tumor hypoxia environment and predicting disease-free survival for patients with rectal cancer
              </a>
            </td>
          <td>
            Shenlun Chen, Zonglin Liu, Jiazhou Wang, Zhiyuan Zhang, Andre Dekker, L. Wee, P. Kalendralis, Yiqun Sun, Tong Tong
          </td>
          <td>2025-11-18</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eff27a80c0a6e450a7ee922148ee3e29c575c23" target='_blank'>
              From HeLa to human blood: extending single-cell proteomics to the smallest immune cells
              </a>
            </td>
          <td>
            I. Hendriks, Sara C. Buch-Larsen, Martin Rykær, Maico Y. Lechner, A. Kverneland, T. Arrey, Daniel Hermanson, E. Damoc, J. Olsen
          </td>
          <td>2025-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer is a highly heterogeneous disease, with many cancers containing multiple distinct subclones. While subclones are often seen as competitors (survival of the fittest), intratumor heterogeneity can also offer direct benefits to the tumor through cooperation between different clones. This has important clinical implications, as interdependent populations may present therapeutic vulnerabilities. Here, we review existing evidence for clonal cooperativity to address key questions and outline future developments based on six overarching principles: (a) secreted factors are important mediators of clonal cooperation; (b) (very) small subclones can significantly affect tumor behavior; (c) both genetic and nongenetic heterogeneity are substrates for cooperation; (d) nonmalignant cells from the tumor microenvironment can act as cooperating partners; (e) clonal cooperation occurs throughout different stages of cancer, from premalignancy to metastasis; and (f) clonal cooperation can promote therapy resistance by protecting otherwise sensitive populations. Together, these principles suggest clonal cooperation as an important mechanism in cancer. Lastly, we discuss how novel technological developments could address remaining gaps to open up new therapeutic strategies that exploit clonal cooperativity by targeting the tumor's weakest link.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb39ce21ba07d76bffe91c932b77c8c8a7e6988d" target='_blank'>
              Strength through diversity: how cancers thrive when clones cooperate.
              </a>
            </td>
          <td>
            Marije C. Kuiken, Maartje Witsen, E. Voest, K. Dijkstra
          </td>
          <td>2025-11-09</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba29f656998508469f49c53f9437fe122dadd98" target='_blank'>
              DYRK1A regulates cancer cell and cancer-associated fibroblast secretomes to foster an immunosuppressive microenvironment in pancreatic cancer
              </a>
            </td>
          <td>
            Silvia Pascual-Sabater, Sushmita Varhadi, Chiara Di Vona, Marta Celma, Cristina Fillat, S. de la Luna
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Background Colorectal cancer (CRC) continues to represent a major cause of cancer-related mortality worldwide, with its progression and therapeutic outcomes strongly shaped by the complexity and heterogeneity of the tumor immune microenvironment (TME). This review critically examines the cellular and molecular mechanisms driving immune evasion in CRC, emphasizing the dual roles of immune cell populations—including tumor-associated macrophages, neutrophils, dendritic cells, T cells, B cells, and natural killer cells—as well as non-cellular elements such as the extracellular matrix and extracellular vesicles. Objective A key objective is to evaluate recent developments in immunotherapeutic approaches, including immune checkpoint inhibitors, tumor vaccines, adoptive cell transfer, and novel combinatorial regimens, while addressing their therapeutic promise and inherent limitations, especially in microsatellite-stable (MSS) tumors that exhibit primary resistance to standard immunotherapies. Further analysis integrates perspectives on metabolic reprogramming within the TME, epigenetic alterations, and advances in engineered cellular therapies, thereby providing a comprehensive framework for overcoming immunosuppressive mechanisms. Discussion and conclusion Special consideration is directed toward the translational value of targeting immune-metabolic interactions and spatial dynamics within the TME. Ultimately, this work synthesizes current knowledge and outlines forward-looking strategies to advance personalized, multi-target immunotherapy, with the potential to reshape clinical paradigms in CRC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03fb3a1b140db8c8ae9afa2ff97c1f40f6b8cd5d" target='_blank'>
              Cellular characteristics of the immune microenvironment of colorectal cancer and progress in immunotherapy research
              </a>
            </td>
          <td>
            Chunbaixue Yang, Jianchun Fan, Yixuan Zhang, Yixiao Zhang, Lei Xia, Xinran Cao, Xueliang Wu
          </td>
          <td>2025-11-30</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Restoring functional β-cell mass through proliferation is a central goal of diabetes therapy, but progress has been hampered by the inability to isolate live, proliferating β-cells for study. Here, CD168 is identified as a conserved surface marker that specifically enriches for proliferating cells in mouse and human pancreatic islets. Single-cell RNA sequencing reveals a distinct cluster of islet cells with high proliferative activity, low insulin expression, and specific CD168 expression. Flow cytometry and immunostaining confirm CD168+ cells co-localize with Ki67+ proliferating cells, reside in G2/M phase, and comprise ≈0.5% of adult islet cells. Using a new CD168-CreERT2 mouse model for lineage tracing, it is demonstrated that CD168⁺ cells rapidly divide, forming two-cell clones within hours. Unbiased lineage tracing shows 87.4% of clones are uni-β lineage, with smaller proportions of uni-α, uni-δ, and multi-lineage. Integrated lineage tracing with multi-omics maps a ≈60-day maturation trajectory for nascent β-cells, involving progressive epigenome remodeling and shifting transcriptional networks. Notably, CD168 expression is conserved in proliferating cells of human islets and pancreatic neuroendocrine tumors, highlighting its clinical relevance. This work establishes CD168 as a marker for proliferating islet cells, providing a tool to study β-cell proliferation and novel insights into islet cell proliferation and maturation mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba802d29084b63c5d6c432929d0001d8e52e060e" target='_blank'>
              CD168 Identifies Proliferating Pancreatic Islet Cells in Murine and Human.
              </a>
            </td>
          <td>
            Shubo Yuan, Jiafu Li, Min Shao, Haili Bao, Ajun Geng, Jialin Yang, Yu Tao, Xinyi Chen, Tianxiong Xiao, Chunye Liu, Zhiyao Xie, Wenqian Song, Q. Yu, Hongxing Fu, Xu Han, Taochen He, Wenquan Wang, Jianfeng Chen, Sheng Yan, Shaohua Song, Liang Liu, Y. Zeng
          </td>
          <td>2025-12-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, J. Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, Oliver Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, G. Boons, Junbin Qian, S. Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related mortality worldwide, with tumor recurrence and metastasis posing significant challenges despite advances in targeted therapies and immunotherapy. Cellular dormancy, a reversible, quiescent state marked by cell cycle arrest, has emerged as a key driver of therapeutic resistance and disease relapse, particularly in small-cell lung cancer (SCLC) and non-small cell lung cancer (NSCLC). Multiple mechanisms, including autophagy, stress-adaptive signaling, microenvironmental cues, and epigenetic dysregulation, have been implicated in the regulation of dormancy and long-term cell survival. Among these, epigenetic modifications such as DNA methylation, histone modifications, and non-coding RNAs (ncRNAs) play pivotal roles in maintaining dormancy by repressing proliferative gene expression programs. Increasing evidence suggests that dormant tumor cells harbor distinct epigenomic signatures, which may serve as predictive biomarkers for minimal residual disease (MRD) and relapse risk. This review summarizes current advances in understanding the epigenetic regulation of cellular dormancy in lung cancer, with a particular emphasis on the interplay between epigenetic modifiers and oncogenic signaling pathways. Furthermore, emerging molecular targets and associated therapeutic agents currently under clinical evaluation are presented, emphasizing how a deeper understanding of the epigenetic landscape governing dormancy may inform the development of novel interventions to improve long-term clinical outcomes in lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f78e8931704f8d1c6b0d604cdcde3728bfe1b4a" target='_blank'>
              Unlocking Lung Cancer Cell Dormancy: An Epigenetic Perspective
              </a>
            </td>
          <td>
            F. Fabrizio
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Background Early tumour vascular invasion contributes to cancer progression. Tip cells, a subset of tumour endothelial cells, significantly decline after anti‐angiogenic therapy. However, their behaviour and the roles of their signature genes during early invasion are incompletely understood. Methods This study employed single‐cell transcriptomic analysis and 10x Genomics Visium spatial transcriptomics on fresh lung tissues from patients with pulmonary nodules and from KrasG12D (K) and KrasG12DTgfbr2−/− (KT) mice. The role of plasma vesicle‐associated protein (PLVAP), a tip cell marker, was further examined using survival databases, immunofluorescence, in vitro co‐culture, cell migration, invasion assays and endothelial tube formation. Results Tip cell proportions were elevated in early‐stage lung adenocarcinoma (LUAD) tissues and KT mice, with evidence suggesting they arise from capillaries type I. PLVAP expression was enriched in tumour endothelial cells, induced by TGFβ1, and negatively correlated with patient prognosis. Functionally, PLVAP promoted endothelial cell invasion, migration and angiogenesis, and regulated tumour cell invasiveness. Intercellular analysis revealed that some tip cells also expressed TGFβ1, which may act on adjacent tumour cells to enhance invasion during early tumour development. Conclusion Tip cells increased during early LUAD progression and likely evolved from capillaries type I. Their marker PLVAP was associated with poor prognosis and pro‐invasive endothelial behaviour. Tumour‐secreted TGFβ1 upregulated PLVAP in endothelial cells, promoting angiogenesis and tumour invasion. Additionally, tip‐cell‐derived TGFβ1 may further stimulate tumour aggressiveness, highlighting a reciprocal interaction that contributes to early tumour progression. Key points Tip cells expand during early LUAD progression and likely originate from capillary type I endothelial cells. Tumour‐derived TGFβ1 induces PLVAP expression in endothelial cells, linking tumour signals to vascular activation. PLVAP enhances endothelial cell migration, invasion and angiogenic capacity. Endothelial PLVAP promotes tumour cell invasiveness, revealing a reciprocal endothelial‐tumour interaction that drives early tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79021428fe4068225ddb196bfe79a293a09eade5" target='_blank'>
              PLVAP mediates the regulation of the tumour microenvironment in early‐stage lung adenocarcinoma
              </a>
            </td>
          <td>
            Linshan Xie, Mengting Sun, Yujie Zheng, Zezhong Yu, Hui Kong, Jinjie Yu, Shaohua Lu, Yong Zhang, Jie Hu, Hongyi Xin, Jian Zhou, Xiangdong Wang, Charles A Powell, Fred R Hirsch, Chunxue Bai, Yuanlin Song, Jun Yin, Dawei Yang
          </td>
          <td>2025-12-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) and pancreatic ductal adenocarcinoma (PDAC) are highly lethal cancers marked by profound epigenetic and metabolic reprogramming. Among the candidate biomarkers, the DNA methyltransferase DNMT3A and the guanine monophosphate synthetase (GMPS) have emerged as potential prognostic drivers, yet their roles across tumor contexts remain unclear. Here, we demonstrate the application of KMplotter to interrogated pan-cancer transcriptomic and survival datasets encompassing over 7000 patients, complemented by expression profiling of normal, tumor, and metastatic tissues, and integrated tumor microenvironment (TME) analyses. Elevated DNMT3A and GMPS expression correlated with worse overall survival in HCC, particularly in Asian patients, while in PDAC, high DNMT3A but low GMPS expression predicted favorable outcomes. Both genes were consistently upregulated in tumors relative to normal tissues, with further increases in metastatic HCC. Immune deconvolution revealed that DNMT3A was linked to Th2/Treg-enriched niches, whereas GMPS overexpression coincided with high mutational burden or stromal enrichment, fostering immunosuppressive microenvironments. Comparative analysis of toll-like receptor signatures highlighted divergent antigen-sensing pathways, with HCC reflecting viral-driven immune exhaustion and PDAC showing self-antigen–associated signaling. Collectively, these findings position DNMT3A and GMPS as context-dependent biomarkers that integrate metabolic and immune cues to shape prognosis in liver and pancreatic cancer, offering mechanistic insight and translational relevance for patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c71a88a2f1a7eea4d4abf0ff75d340e4a7fe137" target='_blank'>
              Comparative Tumor Microenvironment Analysis for HCC and PDAC Using KMplotter
              </a>
            </td>
          <td>
            Wen-Han Chang, Drashya Shah, Scott Myers, Michael Potts, Sanjive Qazi, Vuong Trieu
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The aim of this study was to characterize the transcriptional landscape of tuberous sclerosis complex-associated renal angiomyolipomas (TSC-AMLs) using spatial transcriptomics and to compare it with sporadic AMLs and renal cell carcinoma (RCC) to identify TSC-specific molecular features. Spatial gene expression analysis (CytAssist Visium) was performed on formalin-fixed, paraffin-embedded sections from one case each of TSC-AML, sporadic AML, and RCC. Unsupervised clustering revealed that TSC-AML and sporadic AML, unlike RCC, displayed triphasic tumor structures with vascular, smooth muscle, and adipose components expressing canonical perivascular epithelioid cell markers such as PMEL, MLANA, ACTB, and DES. TSC1 and TSC2 were downregulated, and mild upregulation of MTOR, MLST8, and RPTOR was observed, consistent with mTORC1 hyperactivation, whereas RCC showed broader pathway activation. Spatial pathway analysis indicated shared differentiation programs in smooth muscle regions but unique angiogenesis, extracellular matrix remodeling (Wnt and NABA matrisome), and lipid metabolic (PPAR signaling) enrichment in TSC-AML. Comparative profiling identified 42 genes uniquely upregulated in TSC-AML, including GRIA2, FABP4, and IL33, defining candidate TSC-AML–enriched features. Despite the single-case design, this pilot study provides hypothesis-generating insights into TSC-associated renal tumorigenesis and highlights potential biomarkers for future investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca795c8f856b060495a340d41a11434aceebd201" target='_blank'>
              Spatial gene expression profiling reveals the distinct microenvironment of tuberous sclerosis complex-associated angiomyolipoma
              </a>
            </td>
          <td>
            R. Watanabe, Tetsuya Fukumoto, N. Miura, M. Kurata, R. Kitazawa, T. Kikugawa, Takashi Saika
          </td>
          <td>2025-11-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) have emerged as critical prognostic and immunotherapeutic indicators in cancer, with their clinical significance modulated by spatial distribution patterns and density. Here, we performed integrated single-cell and spatial transcriptomic profiling of 30 gastric cancer (GC) specimens stratified by TLS spatial localization patterns. Comparative analysis shows pronounced enrichment of CXCL13+ T lymphocytes (TLCs), CXCR5+ germinal center B lymphocytes (gc_B cells), LAMP3+CD80+ activated dendritic cells (DCs), and SELP+ACKR1+ high endothelial venule (HEV) cells within intratumoral-TLS (iTLS) rich tumors compared to peritumoral-TLS (pTLS) and desert-TLS (dTLS) tumor subtypes. Multimodal cell–cell interaction analysis and functional experiments demonstrate that HEV expressed VCAM1 and ICAM1 recruits and activates CXCL13+ TLC through the CXCL13-ACKR1 pathway, which promotes TLS formation via CXCL13-CXCR5-dependent crosstalk with B lymphocytes. We further develop a single-cell/spatial TLS signature that captures the cellular ecosystem of iTLS-containing tumor, demonstrating predictive value for immunotherapy outcomes in GC patients. Tertiary lymphoid structures (TLS) within the tumour microenvironment are important for prognostic and immunotherapeutic indications in cancer. Here the authors use a spatial and single cell transcriptomics analysis of gastric cancer (GC) samples and show transcriptomic differences grouped by types of TLS and generate a prognostic transcriptomic signature to help predict outcome in GC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f70b23aaaa2341fd42d68d4bef2be6eb1c78ea" target='_blank'>
              Single-cell and spatial transcriptomics implicate a prognostic function of tertiary lymphoid structures in gastric cancer
              </a>
            </td>
          <td>
            Yanchun Wang, Guangyu Zhang, Xin Zhang, Gaoke Liu, Lihua Zhang, Lei Chen, Shuli Sang, Sheng Yao, Yang Fei, Zhichao Tian, Yu Zang, Hongwei Wang, Yiping Dong, Chunjie Liu, Xizhao Chen, Yan Liu, Min Zhang
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Cervical cancer is among the prevalent gynecological malignancies affecting women worldwide. The diverse cellular populations and their intricate communication patterns within the tumor environment significantly influence cancer development and metastasis. Our research focused on elucidating the cellular diversity, intercellular signaling networks, and underlying molecular processes in cervical cancer tissues via single-cell transcriptomic approaches. Methods Publicly available single-cell RNA sequencing (scRNA-seq) data for cervical cancer (GSE236738) were re-analysed to delineate the cellular composition and transcriptional programs of the tumour microenvironment. We applied UMAP and t-SNE algorithms for dimensionality reduction, alongside intercellular communication mapping, MIF pathway analysis, temporal trajectory modeling, and pathway enrichment studies, to comprehensively characterize the cellular architecture and molecular profiles within cervical cancer tissues. In addition, paired tumour and adjacent non-tumour cervical tissues from 20 patients were subjected to quantitative real-time PCR and enzyme-linked immunosorbent assay to confirm the transcriptional and protein-level expression of key genes identified by scRNA-seq. Results Analysis revealed substantial cellular and molecular diversity within cervical cancer specimens. Ten distinct cellular populations were characterized, including NK lymphocytes, epithelial components, and macrophage subsets, indicating pronounced cellular heterogeneity. Intercellular communication mapping revealed that stromal and epithelial populations are the primary coordinators of cell-to-cell signaling. MIF pathway activity peaked in cancer-associated fibroblasts, which exhibited distinct cell type distributions. Temporal analysis revealed dynamic cellular states and pinpointed critical regulatory elements, including ISG15. Pathway analysis emphasized the importance of neutrophil activity, mitochondrial respiration, and cell adhesion mechanisms in cancer progression. qPCR confirmed that ISG15 and TNFRSF18 were significantly upregulated in tumor tissues, while SDF4 was downregulated and NOC2L showed a moderate increase. ELISA results were consistent with transcript-level findings, demonstrating significant protein overexpression of ISG15 and TNFRSF18 in tumor tissues compared with controls. Conclusion Our investigation comprehensively characterized the cellular diversity and molecular features of the cervical cancer microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7329b5a877ad7165acf8e0004372f128c5c7552" target='_blank'>
              Mapping cellular interactions and communication landscapes in cervical cancer via single-cell transcriptomics
              </a>
            </td>
          <td>
            Shaoli Zhuang, Xiaoxiao Deng, Furong Fu, Zhang Zhang
          </td>
          <td>2025-11-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Diffuse gliomas remain lethal primary brain tumors. Immune-checkpoint inhibitors have not delivered durable benefit for most patients, reflecting myeloid-dominant immunosuppression and spatially organized immune exclusion. In this mini-review we summarize ligand–receptor multi-omics—single-cell RNA/CITE-seq, single-cell chromatin accessibility, and spatial proteo-transcriptomics—that resolve microglia- and monocyte-derived TAM programs and malignant state continua, and we appraise translational opportunities spanning TAM reprogramming (CSF1–CSF1R), perivascular SPP1–CD44 disruption, and innate–adaptive combinations targeting CD47–SIRPα, CD39–CD73, and PD-1/PD-L1. We also discuss challenges—including ontogeny-aware state definitions, heteromer-aware databases, chromatin gating of receivers (requiring accessible regulatory DNA for the receptor and its program), spatial registration, and limited assay standardization—that temper implementation. By integrating myeloid-informed readouts (SPP1–TAM burden, CD39–CD73 proximity, HMOX1+ IL-10 niches, serum IL-8), emerging strategies aim to restore antigen presentation, enable effector ingress, and remodel vascular–stromal interfaces. Our synthesis provides an appraisal of reproducible communication architectures in glioma and outlines pragmatic reporting standards and trial-ready pharmacodynamic endpoints for myeloid-informed precision immuno-oncology. We hope these insights will assist researchers and clinicians as they design multi-omics pipelines and interventions to convert suppressive ecosystems into responsive ones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e39bafbb89665949b18c8b3763ecd3648dcb8152" target='_blank'>
              Mapping TAM–tumor crosstalk in glioma via ligand–receptor multi-omics: mechanisms of immune evasion
              </a>
            </td>
          <td>
            Dong Zhang, Yiming Ma, Daxiong Feng
          </td>
          <td>2025-11-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The introduction of bowel cancer screening has led to an increase in the detection and removal of pre-malignant adenomas, however many of these would never progress to colorectal cancer (CRC) within a patient’s lifetime. This implies a key evolutionary bottleneck at the transition from adenoma to carcinoma, with immune surveillance likely playing a central role in suppressing the outgrowth of invasive cells. Here we analysed a cohort of “ca-in-ads” (adenomas with a small focus of cancer) to provide a unique snapshot of the invasive transition. We combined genomics, spatial transcriptomics, digital pathology and multiplex imaging, concurrently deriving the distribution of tumour clones, their gene expression and their interplay with the immune microenvironment. We performed multi-region low-coverage whole genome sequencing (lcWGS) and T-cell receptor sequencing (using FUME-TCRseq, a novel method developed in our lab) on archival ca-in-ads from 40 patients. For a subset of cases we performed spatial transcriptomics using the 10x Genomics Visium platform, including custom probes to detect T-cell clonotypes of interest. We applied a deep learning cell classifier to haematoxylin and eosin (H&E) stained sections to determine the abundance and spatial distribution of eight cell types, with detailed immunophenotyping on selected cases (using a 50-marker panel on Akoya’s PhenoCycler-Fusion platform). We found a significant increase in copy number alterations in carcinoma regions relative to adenomas, and this was accompanied by a large shift in the composition of the local T-cell repertoire. TCR sequencing revealed expanded T-cell clonotypes that were adenoma-specific or carcinoma-specific, and we validated the spatial exclusivity of these using a custom Visium assay. Although overall lymphocyte density remained unchanged, digital pathology and cyclic immunofluorescence revealed significantly increased infiltration of macrophage and neutrophil populations in regions of invasive cancer. Taken together our analysis suggests a striking reshaping of tumour-immune dynamics at the transition to an invasive phenotype in CRC. Understanding what governs this invasive shift could highlight new avenues for cancer prevention or identify individuals at risk of CRC.



 Ann-Marie Baker, Eleanor Smith, Chirine Sakr, Konstantin Bräutigam, Nick Trahearn, Holly Eggington, Miangela M. Lacle, Manuel Rodriguez-Justo, Simon J. Leedham, Trevor A. Graham. Multi-omic spatial analysis reveals reshaping of tumour-immune dynamics at the transition to invasive colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr PR004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b4fa90cafae68daf434680b6837e68209ae59d6" target='_blank'>
              Abstract PR004: Multi-omic spatial analysis reveals reshaping of tumour-immune dynamics at the transition to invasive colorectal cancer
              </a>
            </td>
          <td>
            Ann-Marie Baker, Eleanor Smith, Chirine Sakr, Konstantin Bräutigam, N. Trahearn, Holly Eggington, M. Laclé, M. Rodriguez-Justo, Simon J. Leedham, Trevor A. Graham
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/999c9f7dc52086f050a2aa67ed25e42b93fae763" target='_blank'>
              Single-cell RNA sequencing in osteosarcoma: applications in diagnosis, prognosis, and treatment
              </a>
            </td>
          <td>
            Christèle Asmar, Guy Awad, Marc Boutros, Simon Daccache, A. Chebly, C. Alix‐Panabières, H. Kourie
          </td>
          <td>2025-11-13</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="BACKGROUND
The development of gemcitabine resistance significantly diminishes treatment efficacy and worsens prognosis in patients with pancreatic ductal adenocarcinoma (PDAC). A comprehensive understanding of the molecular and cellular mechanisms driving gemcitabine resistance is thus essential for optimizing treatment strategies and improving survival outcomes in PDAC.


METHODS
An integrated analysis of bulk RNA sequencing (RNA-seq), single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics was performed to elucidate the molecular mechanisms of gemcitabine resistance. Tumor-associated epithelial and myeloid cell subsets were identified, and their interactions were characterized using clustering, pseudotime analysis, and cell-to-cell communication networks. Findings were validated through molecular and cellular experiments in PDAC cell lines, as well as multicolor immunofluorescence staining in clinical cohort samples.


RESULTS
The findings revealed that PRRX2+ epithelial cells (EC) and SPP1+ tumor-associated macrophages (TAM) were enriched in gemcitabine-resistant samples, with robust cell-to-cell communication observed between these two populations. PRRX2+ EC cells and SPP1+ TAM cells interacted through the TGFB1/TGFBR2 axis, promoting epithelial-mesenchymal transition (EMT) and extracellular matrix (ECM) remodeling, thereby contributing to drug resistance. The transcription factor PRRX2 was identified as a central regulator of this resistance mechanism, amplifying TGF-β signaling through TGFB1 to induce EMT and activating Fibronectin 1 to remodel the ECM, thereby enhancing an immunosuppressive tumor microenvironment. Furthermore, experimental downregulation of PRRX2 in pancreatic cancer cell lines resulted in reduced tumor cell migration. High expression levels of PRRX2+EC/SPP1+TAM were associated with poor prognosis and may indicate susceptibility to Dasatinib and antibody-drug conjugates (ADCs) targeting TROP2, MUC1, and CEACAM5.


CONCLUSIONS
The results revealed the critical role of PRRX2+EC/SPP1+TAM cell communication in mediating gemcitabine resistance in PDAC. These findings provide new insights into potential therapeutic strategies, highlighting the importance of targeting the PRRX2+EC/SPP1+TAM axis and supporting the use of multidrug regimens to overcome resistance and improve patient outcomes in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/add5ea3cff49a3e2c5c93db3e7092d7e12027eec" target='_blank'>
              Single-cell and spatial transcriptomics reveal the interaction between PRRX2-driven epithelial cells and SPP1+ macrophages in mediating gemcitabine resistance in pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Jingmei Liu, Zhilong Li, Guanhua Liu, Zhihua Pei, Weilin Liu, Wenhui Yang, Feng Li, Lijuan Huo
          </td>
          <td>2025-12-26</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="In this study, we analyzed publicly available scRNA-seq data from primary tumor tissues of 10 Retinoblastoma (RB) patients to explore tumor microenvironment (TME) heterogeneity. Our findings revealed distinct subpopulations of cone precursor (CP) cells, with higher proportions in invasive RB. Differential gene expression and pathway analysis highlighted functional diversity among CP subpopulations, with CP4 showing elevated TGF-β signaling in invasive RB. Cell–cell interaction analysis further identified rewiring of communication networks, with increased fibroblast–CP interactions in invasive tumors. Bulk RNA-seq identified two molecular subtypes, with subtype 1 showing an immunosuppressive TME. Finally, DOK7 was identified as a key gene associated with invasion, with functional assays confirming its role in promoting tumor progression. These results provide valuable insights into RB progression and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0641facdea378a09ac72da42fc0862a4720147" target='_blank'>
              Comprehensive analysis of single-cell and bulk RNA sequencing uncover tumor microenvironment diversity in invasive Retinoblastoma
              </a>
            </td>
          <td>
            Wei Wan, Xiaoqian Chen, Hongli Liu, Ling Yang, Pingping Huang
          </td>
          <td>2025-11-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) serve as a promising biomarker for tracking cancer metastasis, progression, and recurrence. Liquid biopsy techniques centered on CTC detection have demonstrated considerable potential due to their non-invasive nature and ability to provide real-time monitoring of tumor dynamics. However, conventional bulk CTC analyses fail to capture the intrinsic heterogeneity among CTC populations, obscuring crucial insights into tumor biology. Single-cell RNA sequencing (scRNA-seq) enables high-resolution characterization of CTC heterogeneity, offering new opportunities for precision oncology and mechanistic studies of tumor progression. Despite these advantages, the existing methodologies for single-CTC sequencing tend to suffer from inefficiencies, including low recovery rates, labor-intensive workflows, and contamination risks associated with multiple manual handling steps. To address these limitations, we present an integrated microfluidic protocol that consolidates CTC enrichment, purification, and single-cell sequencing into a unified workflow. The method employs dynamically controlled magnetic capture within a herringbone-structured chip, where vortex mixing and cumulative immunomagnetic bead binding enable robust, high-throughput CTC isolation with minimal cell damage. Subsequent purification using a leukocyte antibody-coated microfluidic chip effectively removes the non-target cells, further enhancing CTC purity through negative selection. Finally, a high-precision single-cell sequencing chip, designed based on differential flow resistance principles, facilitates efficient single-cell capture and pairing with uniquely barcoded microbeads. This novel platform overcomes the limitations of Poisson distribution-based methods, improving CTC utilization while minimizing microbead consumption and sequencing costs. Our integrated protocol significantly enhances CTC capture efficiency, purity, and single-cell sequencing throughput, making it well-suited for clinical applications and large-scale cancer research. By enabling a more precise and scalable analysis of CTC heterogeneity, this method has the potential to refine early cancer diagnosis, treatment monitoring, and mechanistic studies of metastasis, ultimately advancing the field of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9babafe8a10523678f47d2a96e859b16fafc3f98" target='_blank'>
              Microfluidics-based High-throughput Circulating Tumor Cell Sorting and Single-cell Sequencing Technology.
              </a>
            </td>
          <td>
            Zheyu Liu, Ye Tian, Zian Zhang, Lingling Wu, Liang Dong, Wei Xue
          </td>
          <td>2025-11-14</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is characterized by its highly aggressive phenotype and dismal outcome. Despite the benefit of adding immune checkpoint blockade to standard chemotherapy, tumors acquire the ability to evade immunosurveillance and develop resistance. To investigate these underlying mechanisms, we perform high-dimensional profiling of human and murine SCLC specimens. In matched primary and metastatic human samples, we observe MHC-I loss in metastases, highlighting its role in immune evasion. Correspondingly, silencing MHC-I in SCLC cells drastically reduces immune infiltration and promotes metastasis in mice. Using mass spectrometry and phospho-tyrosine kinase analyses, we identify ERBB2 signaling as a suppressor of MHC-I and driver of immune-modulatory transcripts. Mechanistically, genetic and pharmacologic blockade of ERBB2 induces MHC-I in a STING-dependent manner and prevents immune evasion in autochthonous murine SCLC. Strikingly, combining ERBB2 inhibition with anti-PD-1 elicits profound synergistic responses in preclinical models, suggesting this combination for future clinical trials in SCLC patients. While the emergence of immune checkpoint inhibitors has improved outcomes in patients with small cell lung cancer (SCLC), tumour that develop means of immune evasion become resistant. Here, the authors report that ERBB2 signalling induces loss of MHC Class I expression and subsequently immune evasion in preclinical models of SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f612388efd2f0a9c1ebc8d3e60e6b396ae989beb" target='_blank'>
              ERBB2 signaling drives immune cell evasion and resistance against immunotherapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Meder, C. Orschel, C. Bouchez, R. Gholamipoorfard, C. V. Orschel, David Stahl, Christoph Kreer, Mirjam Koker, Marieke Nill, Ilayda G Kocak, Ka-Won Noh, Xinlei Zhao, Leon Ullrich, B. Häupl, Josefine Jakob, Marie-Lisa Eich, Alexandra Florin, Holger Grüll, J. Wolf, Filippo Beleggia, R. Buettner, T. Oellerich, Florian Klein, J. Brägelmann, H. C. Reinhardt, Nima Abedpour, R. Ullrich
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Purpose
Colorectal cancer (CRC) lung metastases exhibit high recurrence rates after resection, underscoring the need for improved therapeutic strategies. This study aimed to characterize the tumor microenvironment (TME) of CRC lung metastases and identify the factors associated with recurrence.


Materials and Methods
Fifteen CRC patients who underwent lung metastasectomy were enrolled. Multiplex immunohistochemistry (IHC), whole exome sequencing, transcriptome profiling, and single-cell RNA sequencing (scRNA-seq) were conducted on matched tumor, adjacent and distant normal lung tissues. Immune cell populations and gene expression profiles were analyzed and correlated with clinical recurrence outcomes.


Results
Exome and transcriptome analyses revealed frequent TP53, KRAS, and APC mutations. Most tumors corresponded to consensus molecular subtypes 2 and 4, characterized by immune-depleted and fibrotic features. Tumors showed downregulation of effector T and NK cell signatures. IHC revealed reduced density and increased distance of CD8+ T cells and macrophages from the epithelial cells. scRNA-seq demonstrated increased regulatory T cells and decreased NK and effector T cells in tumor. Tumor-associated macrophages (TAMs), particularly SPP1 (osteopontin)-expressing subsets, were markedly enriched in tumor and correlated with suppressed effector T cella activity. High SPP1 expression was associated with early recurrence and poor overall survival. Patients with recurrence had higher proportion of PD-1+ CD8+ T cells in adjacent normal tissues.


Conclusion
Immunosuppressive features including enrichment of SPP1+ TAMs and depletion of effector T and NK cells contribute to recurrence after CRC lung metastasectomy. Therapeutic strategies targeting both TAMs and T cells may enhance clinical outcomes in this patient population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae972bf762bae15538dd4da3de79b673b4d58b2d" target='_blank'>
              Immunosuppressive Tumor Microenvironment in Colorectal Cancer Lung Metastases: Implications for Recurrence After Metastasectomy.
              </a>
            </td>
          <td>
            Minsuk Kwon, Minkyue Shin, M. An, Y.J. Jeon, T. Hong, Jung Kyong Shin, Sung Hee Lim, Yoonah Park, Y. Cho, Seung-Tae Kim, Y. Choi, Jeeyun Lee
          </td>
          <td>2025-12-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven by heterogeneous tumor cell populations and microenvironmental interactions. However, a comprehensive single-cell atlas across patients that captures this heterogeneity, and its clinical implications has been lacking. Such an atlas could reveal rare tumor subpopulations that underpin disease aggressiveness and offer new prognostic biomarkers or therapeutic targets. We integrated two high-quality single-cell RNA sequencing datasets from 29 CRC patients to construct a cellular atlas encompassing immune, stromal and epithelial compartments. Malignant epithelial cells were distinguished via inferCNV-based copy number alteration analysis and reclustered, yielding seven transcriptionally distinct malignant subpopulations. One malignant epithelial cluster, marked by high expression of the long non-coding RNA ELFN1-AS1, exhibited the highest stemness signature. From this stem-like cluster, we derived a four-gene prognostic model (RPL21, GAL, ELFN1-AS1, AIF1L). In the TCGA-COAD cohort, this model stratified patients into high-risk and low-risk groups with significantly different survival outcomes, with the high-risk group experiencing significantly worse survival. High-risk tumors were enriched for metabolic and translational pathways and displayed distinct immune and genomic features. AIF1L was identified as a hub gene within the signature, and its knockdown in CRC cells enhanced migration and invasion, functionally validating its role in tumor progression. Our study provides a high-resolution single-cell atlas of CRC and identifies a previously unrecognized stem-like tumor cell subpopulation with prognostic significance. These findings highlight novel prognostic biomarkers and suggest potential therapeutic targets (such as AIF1L) that could inform patient stratification and the development of targeted therapies for CRC. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-15241-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3d35184693bc48dd0fe85be6bfc55a03d6e4e7" target='_blank'>
              Single-cell RNA sequencing defines prognostic subtypes and identifies AIF1L as a therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Gao Li, Dingxue Wang, Guo Chen
          </td>
          <td>2025-12-26</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Oropharyngeal squamous cell carcinoma (OPC) associated with high-risk human papillomavirus (HPV), particularly HPV-16, generally shows favorable outcomes yet paradoxically exhibits a high incidence of early lymphatic metastasis. The immune mechanisms underlying this phenomenon remain unclear. Methods We conducted an exploratory spatial transcriptomic analysis using the GeoMx Digital Spatial Profiler on formalin-fixed, paraffin-embedded samples from six patients with palatine tonsil-derived OPC. Tumor tissue regions (TTRs) and lymphoid follicular regions (LFRs) were compared according to HPV status and nodal involvement. Results In HPV-positive LFRs, pathways related to B-cell apoptosis appeared downregulated, suggesting prolonged B-cell survival and antigen presentation. Metastasis-negative HPV-positive cases displayed a Th2-skewed immune profile in LFRs, with increased naïve B cells, plasma cells, eosinophils, M2 macrophages, and activated mast cells. In contrast, metastasis-positive cases showed increased T cell activation in LFRs and reduced proliferation-related signaling in TTRs. Pathways involving estrogen signaling and bile acid metabolism were also associated with metastatic behavior. Conclusion These exploratory findings suggest that peritumoral Th2-biased immunity within lymphoid structures may contribute to restraining lymphatic metastasis in HPV-positive OPC. Spatial transcriptomics may provide a high-resolution framework for investigating tumor–immune interactions and generating hypotheses for future mechanistic and clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb3a9ec6ff5bf8bd62696a72523c1ce32b123f2" target='_blank'>
              Exploratory spatial transcriptomic profiling of peritumoral Th2 immune polarization in HPV-positive oropharyngeal cancer
              </a>
            </td>
          <td>
            N. Wakisaka, M. Moriyama-Kita, Satoru Kondo, E. Kobayashi, Takayoshi Ueno, Y. Nakanishi, Kazuhira Endo, H. Sugimoto, T. Yoshizaki
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Mapping the spatial proteome signature within heterogeneous tissue microenvironment of clinical specimens offers unique insights into both physiological and pathological molecular phenotypes. However, to simultaneously visualize and discover these spatial proteome features is challenging. Herein, we introduce spatial visual proteomics (SVPro) which integrates single-cell proximity biotinylation coupled with fluorescence signal amplification via the trifunctional probe. This probe uniquely combines naphthylamine-mediated radical targeting, clickable fluorescence imaging, and biotinylation-facilitated proteome capture, enabling all three within a single tissue section. Strategic PEGylation and multivalent fluorescent conjugation of these probes improves labeling and visualization efficiency and specificity. SVPro allows cell-type proteomic profiling of neuronal subpopulations in distinct mouse brain regions and amyloid-β plaques in mouse brain of Alzheimer's disease model. Notably, we identified and validated distinct marker proteins associated with amyloid‑β plaques deposited in different brain regions. SVPro therefore emerges as a robust and high-resolution spatial proteomics approach for simultaneously visualizing the spatial architecture and proteome landscapes of intact tissue microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ff1f2a09a7c9f7d49b82dc133390ce911b3a49e" target='_blank'>
              In-Situ Spatial Visual Proteomics Enabled by Single-Cell-Type Proximity Biotinylation and Signaling Amplification.
              </a>
            </td>
          <td>
            Zhendong Zheng, Zhiyao Tang, Chang Li, Haonan Xiao, Yuan Li, An He, Yiheng Mao, Jiangnan Zheng, Mi Ke, Rui Gao, Dan Li, Cong Liu, Zhe Dong, Ruijun Tian
          </td>
          <td>2025-11-26</td>
          <td>Angewandte Chemie</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a712967a1f06be04c88812bdec6b8089c56c0b" target='_blank'>
              Antigen reactivity defines tissue-resident memory and exhausted T cells in tumors.
              </a>
            </td>
          <td>
            Thomas N. Burn, Jan Schröder, Luke C. Gandolfo, M. Osman, Elanor N. Wainwright, E. Lam, Keely M. McDonald, Rachel B. Evans, Shihan Li, Daniel Rawlinson, Lachlan Dryburgh, Ali Zaid, Z. Maliga, Dominick Schienstock, Philippa Meiser, H. J. Lee, Hongjin Lai, Marcela L. Moreira, Pirooz Zareie, , Lutfi Huq, Susan N. Christo, Justine Seow, Keith A. Ching, Stéphane M Guillaume, Kathy Knezevic, Simone L. Park, Maximilien Evrard, Jason Waithman, T. Gebhardt, Scott N. Mueller, G. Riddiough, M. V. Perini, S. C. Tsao, Terence P. Speed, P. Sorger, S. Loi, Francis R. Carbone, Stephanie Gras, Timothy S Fisher, Bas J Baaten, M. Dawson, Laura K. Mackay
          </td>
          <td>2025-12-29</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background The most aggressive type of skin cancer, melanoma, has a high prevalence of metastases and a poor prognosis. Despite advancements in treatment, drug resistance and tumor microenvironment heterogeneity, especially involving neuro-immune interactions, continue to exist. The goal of this study is to uncover the cellular heterogeneity of melanoma in order to pinpoint molecular targets and tumor-promoting subtypes. Methods Melanoma single-cell RNA sequencing data came from GEO. Twelve cell types were discovered after Harmony batch effect adjustment and Seurat quality control; melanoma cells were subtyped. Functional expression of differential genes was examined using gene ontology and gene set enrichment. Cytotrace measured subtype differentiation potency. PySCENIC revealed transcription factor regulatory networks, and CellChat predicted intercellular communication between malignant cell subtypes and other cell types. Functional experiments with A375 and MEWo cell lines—lentiviral shRNA knockdown, CCK-8 proliferation, wound healing, transwell migration, and flow cytometry apoptotic assays—confirmed the analysis. Results Single-cell RNA sequencing was utilized to analyze melanoma cell subtypes and their interactions in the tumor microenvironment. C2 IGFBP3+, which had the lowest CytoTRACE2 score and was enriched in late tumor stages, affected melanoma development. This subtype expressed prominent immunomodulatory pathways. The C2 subtype mediate signaling to fibroblasts/pericytes via the PROS pathway and myeloid/plasmacytoid dendritic cells via the MHC-II pathway. The C2 subtype was strongly linked with FOSL1 expression, which may affect gene transcription and illness progression. FOSL1 knockdown significantly increased apoptosis and decreased melanoma cell motility and proliferation in vitro. Conclusion We identified immunoregulatory C2 IGFBP3+ melanoma cell subtypes in our investigation, and FOSL1 was a crucial transcription factor that aided in cell migration, proliferation, and survival. Because the C2 subtype involves the MHC-II and PROS signaling pathways, which have been shown to have roles in neuroimmunology, neuroinflammation, and pain regulation, it may serve as a hub for neuro-immune interactions in the tumor microenvironment. Precision treatments for melanoma may be advanced by focusing on the FOSL1 or C2 subtype pathways, which may assist in overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34e116063cb6903b41d6e715ba8448ec0e1c090" target='_blank'>
              Single-cell transcriptomics identifies FOSL1-regulated IGFBP3+ melanoma subtype as a neuro-immunoregulatory signaling hub facilitating tumor progression
              </a>
            </td>
          <td>
            Wenjia Ge, Ziwei Zhang, Wenjie Chen, Zhijie Zhao, Hua-Bao Cai, Yantao Ding, Jin Xu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Immune evasion and epithelial-mesenchymal transition (EMT) are critical mechanisms driving tumor progression and therapy resistance in prostate cancer. In this study, we explored the role of TDP2 in modulating the tumor microenvironment (TME) through single-cell RNA sequencing and pathway enrichment analysis. Our results revealed that epithelial cells with high TDP2 expression extensively interact with myeloid cells, macrophages, and fibroblasts, thereby shaping immune responses and facilitating tumor progression. Specifically, TDP2 overexpression suppressed M1 macrophage polarization and dendritic cell (DC) maturation, leading to reduced CD8 + T cell activation and enhanced immune evasion. Additionally, TDP2-high expression was associated with enriched signaling pathways involved in EMT, including COLLAGEN, GALECTIN, MIDKINE (MK), and ONCOSTATIN M (OSM), which promoted tumor cell migration, invasion, and immune evasion. Survival analyses further demonstrated that high TDP2 expression correlated with poor clinical outcomes in prostate cancer patients. Overall, our findings identify TDP2 as a key regulator within the TME and suggest its potential utility as both a prognostic biomarker and therapeutic target in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c33c0d20e26f20d7665d000d3fb80f5eb193340" target='_blank'>
              TDP2 drives immune evasion and metastatic progression in prostate cancer
              </a>
            </td>
          <td>
            Huan Cao, Anyin Pan, Yuetao Chen, Fei Zheng
          </td>
          <td>2026-01-02</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61684838fc3c53a5cd2329af114682e03471ba75" target='_blank'>
              Single-Cell Mendelian Randomization Identifies Cell-Type Specific Genetic Drivers of Lung Cancer Subtypes
              </a>
            </td>
          <td>
            Q. Yang, C. Yan, X.-f. Wang
          </td>
          <td>2025-12-02</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer is a highly heterogeneous disease, driven by genomic and transcriptional changes that impact disease progression and metastatic potential. The interplay between clonal evolution, transcriptional plasticity, and tumour microenvironment is, however, poorly understood. Here, we leverage and integrate single-nuclei RNA sequencing and whole-genome sequencing from 43 spatially distinct tumour samples from five patients with locally advanced prostate cancer to reconstruct clonal evolution trajectories and transcriptional changes driving metastasis at single-cell resolution. We find extensive clonal heterogeneity, including both monophyletic and polyphyletic metastatic dissemination, and ongoing clonal evolution in the primary tumour after metastatic spread. Metastatic seeding converges on disease trajectories involving both genomic and transcriptional changes, including androgen receptor independence and activation of estrogen-, WNT- and JAK-STAT- pathway activity, in spatially distinct areas. Our findings suggest an intricate interplay between clonal evolution and cellular plasticity driving metastatic seeding and point toward more integrative prognostic markers for improved patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea384ac37d8c4c2acd2bd5fc9ab62cffd7ea81c" target='_blank'>
              Clonal evolution and transcriptional plasticity shape metastatic dissemination routes in prostate cancer
              </a>
            </td>
          <td>
            Migle Mikutenaite, Evdoxia Karadoulama, F. Favero, A. Locallo, Francisco German Rodriguez Gonzalez, Daria Kiriy, Tina Keshavarzian, K. Furlano, M. Graefen, Robert G. Bristow, Clarissa Gerhäuser, C. Plass, G. Sauter, R. Simon, M. Lupien, S. Minner, Thorsten Schlomm, Joachim Weischenfeldt
          </td>
          <td>2025-12-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Background Programmed cell death protein-1 (PD-1) blockade has shown promising clinical efficacy in hepatocellular carcinoma (HCC), yet the underlying immunological mechanisms governing response and resistance remain unclear. This study aimed to delineate the temporal and spatial dynamics of T-cell clonotypes and their relationship with pathological response in neoadjuvant PD-1 blockade therapy in HCC. By integrating T-cell receptor (TCR) repertoire analysis with transcriptomic profiling, we sought to elucidate the immune landscape alterations associated with treatment outcomes. Methods We analyzed longitudinal, matched tumor and blood samples from our previous clinical trial (NCT05471674), encompassing 114 specimens, including pre-nivolumab tumor biopsies, post-treatment resected tumors, adjacent non-tumorous livers, and peripheral blood collected before, after, and during follow-up from 19 treatment-naïve HCC. TCR clonality, diversity, and transcriptomic signatures were assessed to characterize immune responses. Results Elevated clonality of intratumoral TCR clonotypes (ITCs) post-treatment correlated with increased pathological tumor necrosis, predominantly driven by the most abundant top 1% ITCs. These dominant clonotypes occupied a significantly larger clonal space and demonstrated increased spatial sharing across tumor, non-tumorous liver, and peripheral blood compartments in nivolumab responders (nivo-Rs). Shared clonotypes were positively associated with tumor necrosis extent and cytolytic activity, suggesting active immune engagement. Post-treatment, shared clonotypes exhibited peripheral expansion and greater intratumoral dominance in nivo-Rs, whereas tumor unique intratumoral TCR clonotypes (ITCs) remained prevalent in non-responders (nivo-NRs). These divergent patterns were linked to higher chemokine expression within the tumor microenvironment of nivo-Rs and impaired human leukocyte antigen (HLA) class II antigen presentation in nivo-NRs, indicating distinct immune landscape configurations influencing therapeutic response. Conclusions Our findings demonstrate that the peripheral infiltration and expansion of high-frequency intratumoral T-cell clonotypes are critical drivers of pathological response to neoadjuvant PD-1 blockade in HCC, highlighting potential immune biomarkers and therapeutic targets to enhance immunotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/901935af9edd3945285033fa6917d3b60bbe154f" target='_blank'>
              Spatiotemporal dynamics of T cells in peripheral blood and tumor underlying differential responses to neoadjuvant PD-1 blockade in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yajing Zhang, Y. Tsui, J. Lee, T. Cheung, DanielWai-Hung Ho, I. Ng
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is the most aggressive primary brain tumor in adults, marked by rapid progression and poor prognosis. GBM may also harbor a complex virome, defined here as the total repertoire of viral elements within tumors, encompassing both exogenous viruses and reactivated endogenous retroviruses. While viral signatures have been observed in other cancers, their transcriptional activity in GBM and their distribution across tumor cells and extracellular compartments remain largely unexplored. Methods We analyzed 45 long- and small-RNA libraries from glioma stem-like cells, their extracellular fractions (exosomes, microvesicles, ribonucleoprotein complexes), and normal brain controls (BioProject PRJNA360129). Taxonomic profiling and functional annotation were performed using established pipelines (MetaPhlAn4, BLASTn, UniProtKB, InterPro), with fresh media controls to exclude background contamination. To validate discriminative power, viral RNA profiles were classified using machine learning -Random Forest models trained on nine species identified as glioblastoma-enriched by LEfSe. Results We detected ~3.41 million viral reads, with >99% from GBM cells and extracellular compartments and <1% from normal brain. Reads spanned 36 species across 11 phyla. Nine species were consistently enriched in glioblastoma: Saimiriine gammaherpesvirus 2, Human endogenous retrovirus K, Camelpox virus, Pestivirus A, Macacine gammaherpesvirus 5, Avian myelocytomatosis virus, Rabbit fibroma virus, Omegapapillomavirus 1, and Cyprinid herpesvirus 2 with genes mapping to oncogenic pathways. Random Forest models showed long-RNA profiles perfectly discriminated against tumor from normal samples (AUC = 1.0), whereas small-RNA profiles carried limited signal (AUC = 0.5). Tumor–extracellular comparisons revealed strong compartment-specific long-RNA signatures (AUC = 1.0), with partial overlap between tumor cells and exosomes in small-RNA data (AUC = 0.75). Conclusion Glioblastoma contains a distinct viral transcriptome; long-RNA profiles show strong diagnostic power, and compartment-specific signatures suggest biomarker potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f03aa5d91d0d223768b844afb632280b82ecb5f9" target='_blank'>
              The glioblastoma virome across tumor and extracellular compartments
              </a>
            </td>
          <td>
            M. Ghorbani, H. Karimi, Farnoosh Solati, Nazanin Anbarestani, Nooshin Khoshdoozmasouleh
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917b475731c2b6642606d58ea37295a894578bc5" target='_blank'>
              Decoding Neoantigen-encoding Tumor-Specific Transcripts Unveils a Shared Target Reservoir for Immunotherapy in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Peng Lin, Yifan Wen, Jingjing Zhao, Feifei Zhang, Yaoming Su, Hongwu Yu, Qiaojuan Li, Chengye Liu, Zhixiang Hu, Yan Li, Zhuting Fang, Linhui Liang, Shenglin Huang
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a crucial role in colorectal cancer by sustaining intratumoral heterogeneity, therapeutic resistance, and metastatic potential. CD133 (PROM1) is among the most frequently used surface markers for CSC identification, whereas TRIM28, a versatile epigenetic regulator, has been implicated in controlling CD133 expression and stem-like features. In this study, we performed a detailed molecular and functional analysis of Caco2 colorectal cancer cell clones with individual knockouts of CD133 or TRIM28. Elimination of CD133 neither altered global gene expression, as confirmed by transcriptome profiling, nor affected key cellular properties. In contrast, loss of TRIM28 led to a marked reduction in CD133 protein abundance and induced extensive molecular and phenotypic remodeling. TRIM28 knockout was associated with broad transcriptomic changes involving more than 500 differentially expressed genes, decreased proliferative activity monitored by time-lapse imaging, and reduced sensitivity to paclitaxel, cisplatin, and curcumin. Furthermore, immune evasion molecules CD24 and CD47 (“don’t eat me” signals) were strongly upregulated in TRIM28-deficient cells, consistently confirmed by both RNA-Seq and flow cytometry analyses. At the same time, imaging flow cytometry and mitochondrial activity assays indicated that these effects were not due to major shifts in mitotic index or bioenergetic status. Altogether, our results demonstrate that TRIM28, rather than CD133, functions as a central regulator of CSC-associated phenotypes in colorectal cancer. These findings highlight the importance of epigenetic context in CSC biology and may inform the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3d3eb05c9db6b57d94b54e8bdad26455b3ea6" target='_blank'>
              Contrasting Impacts of Targeted Disruption of the Cancer Stem Cell Marker CD133 and Its Epigenetic Regulator TRIM28 in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            I. Kholodenko, A. Saidova, D. Potashnikova, V. A. Arzumanian, D. Romashin, A. Tvorogova, E. Poverennaya, K. Yarygin, Yan S. Kim
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The Tracking Cancer Evolution Through Therapy (TRACERx) program represents the most comprehensive effort to characterize tumor evolution in real time. Through longitudinal, multiregion, and multiomic profiling of tumors—and particularly of non-small-cell lung cancer and clear cell renal cell carcinoma—TRACERx has illuminated the dynamic interplay between genetic, nongenetic, and (micro)environmental factors that drive cancer progression, immune evasion, and therapeutic resistance. A central insight from TRACERx has been that not all tumor evolution is genomic: Transcriptomic diversity, epigenetic alterations, RNA editing, and changes in cell–cell interactions also drive adaptation. Methodological innovations—including tumor-informed and ultrasensitive circulating tumor DNA assays, representative sequencing, and integrative immune–genomic analyses—have yielded biomarkers resistant to sampling bias and/or predictive of recurrence, metastasis, and treatment response. By demonstrating that intratumor heterogeneity is a key determinant of clinical outcome and revealing its molecular, transcriptional, and ecosystem-level drivers, TRACERx has established a framework for linking evolutionary dynamics to patient care. As both a scientific framework and a clinical paradigm, TRACERx demonstrates how adaptive, iterative research can refine evolutionary models, improve patient risk stratification, and inspire next-generation cancer evolution studies across malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a59431b9259c4928178579f6c460cf8e369f4f1" target='_blank'>
              TRACERx as an Optimized Paradigm for Understanding Cancer Evolution
              </a>
            </td>
          <td>
            Naomi Iris van den Berg, C. Swanton, S. Turajlic
          </td>
          <td>2025-12-24</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The tumor infiltrating T cells determine whether a patient would respond positively to immunotherapy. This paper has analyzed an mRNA scRNA-seq dataset of melanoma in the context of mapping heterogeneity in T-cells and the question of whether clinical features can shape cell clustering. Unsupervised clustering with k-means based on quality control followed by a description of the major T-cell groups by canonical immune markers. The identification of the differentially expressed genes that were identified was then done by a posteriori information based ranking approach. This study have created a confidence-weighted approach, which is able to merge Spearman and cosine similarity measures with marker-gene rules, to enhance cell-type annotation. Metadata analysis indicated that treatment status is an element that causes the state changes in / T cells pre- and post-checkpoint therapy. Combining graph-based Leiden clustering and k-means, this study refined the definition of the fine-grained T-cell states and proved that, despite the significant heterogeneity of T cells, the strong definition of subclass depends on the combination of several methods of clustering and the use of orthogonal knowledge of experiments. Pseudotime analysis found evidence in gradual transition between naive or early-activated states to cytotoxic or dysfunctional phenotypes that indicate intratumoral T-cell diversity captures a reflection not only of inherent developmental trajectories, but also of the effects due to therapy. Lastly, survival analysis revealed that there was a distinct relationship between clinical response and overall survival. These findings combined narrow the scope of the T-cell picture and offer understanding in the form of transcriptional programs connected with treatment outcomes. Refer to this link for supplementary materials: https://drive.google.com/drive/folders/1UYW-6EMjSUMVOwHxvuYdFSZwREqrGp89">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b245ce41e96da9548625adf2b4b3d69d756bc71f" target='_blank'>
              Single-Cell Transcriptomics Reveals T-Cell State Transitions and Their Association with Immunotherapy Response in the Melanoma Tumor Microenvironment
              </a>
            </td>
          <td>
            Shangyu Ning
          </td>
          <td>2025-12-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23430b4155c46739c54ef5b1d8a5d28f03c57cf0" target='_blank'>
              Spatial transcriptomics reveals distinct inflammatory and adaptive immune properties of rheumatoid arthritis synovial fibroblasts
              </a>
            </td>
          <td>
            Camilla R. L. Machado, Mina Yao, David L. Boyle, R. Benschop, James T. Parker, Wei Wang, Garry S. Firestein
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e643a6feb31e1193c4fe51251e45e7c7b63296ed" target='_blank'>
              Cross-Species Transcriptomic Integration Reveals a Conserved, MIRO1-Mediated Macrophage-to-T Cell Signaling Axis Driving Immunosuppression in Glioma
              </a>
            </td>
          <td>
            Zehui Du, Menghan Li, Brandon H Bergsneider, Andy P. Tsai, Kwang Bog Cho, Lily H Kim, John Choi, Gordon Li, T. Wyss-Coray, Michael Lim, Xinnan Wang
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (iCCA) is a highly metastatic tumor type, and iCCA with intrahepatic metastasis is associated with poor prognosis. Previous studies have confirmed that the tumor microenvironment (TME) is closely related to tumor metastasis. However, the state of the TME during iCCA liver metastasis remains unknown.


APPROACH AND RESULTS
We profiled 28 specimens from 16 patients with iCCA (with and without intrahepatic metastasis) using integrated single-cell and spatial transcriptomics, bulk RNA sequencing, whole-exome sequencing, and in vivo and in vitro functional experiments to characterize the specific TME changes during iCCA liver metastasis. we identified the metastatic tumor cells, COL3A1+ epithelial cells, and revealed that COL3A1+ epithelial cells-endothelial-to-mesenchymal transition cells crosstalk promoted tumor invasion by inducing mesenchymal transformation of endothelial cells, while COL3A1+ epithelial cells-VEGFA+ CCL4+ neutrophils crosstalk promoted tumor colonization by forming neutrophil extracellular traps.


CONCLUSIONS
We revealed the key biological mechanisms involved in the invasion and colonization phases of iCCA liver metastasis. Moreover, we provide valuable data for understanding iCCA liver metastasis and a possible avenue for the treatment of advanced iCCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd684325d487cd1d69724768d37142d24e7c6b41" target='_blank'>
              Spatiotemporal landscape of intrahepatic cholangiocarcinoma liver metastasis at the single-cell level.
              </a>
            </td>
          <td>
            Ziyu Xun, Hao Wang, Zhengfeng Xuan, Kexu Xiong, Na Zhang, J. Long, Huishan Sun, Yiran Li, Chengpei Zhu, M. Piao, Ting Zhang, Longhao Zhang, Shuofeng Li, Chengjie Li, Jiongyuan Li, Boyu Sun, Zi-Qi Zhou, Shanshan Wang, Ziyue Huang, Kai Liu, Yang Tan, Xiaohua Shi, Xiaobo Yang, Hanping Wang, Ling Lu, Haitao Zhao
          </td>
          <td>2025-12-10</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Pancreatic cancer, recognized as a refractory tumor, has an overall survival rate of less than 10%, and its mortality rate continues to rise. Due to the low immune activity induced by its unique tumor microenvironment, pancreatic cancer is classified as a “cold” tumor and is insensitive to current immunotherapies. However, little is still known about the identification and functional mechanisms of key regulatory molecules in the formation of “cold” tumors. In this study, we identified the pancreatic ductal cell niche through single-cell sequencing. By calculating CNV scores using inferCNV to distinguish malignant from non-malignant cells and analyzing differences in transcriptional levels, we constructed a 15-gene model. The inhibitory effect of UBE2H on the immune microenvironment was studied through single-cell sequencing analysis, including the inhibition of T cell function, impairment of antigen presentation in macrophages and neutrophils, and activation of neutrophil extracellular traps (NETs). Our results revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche, primarily observed in pancreatic cancer samples. Notably, single-cell sequencing analysis of UBE2H overexpression revealed an enhanced T cell PD-L1 interaction axis and a downregulation of antigen presentation pathways in macrophages and neutrophils (antigen processing and presentation, macrophage activation, and neutrophil activation). Further in vivo experiments confirmed that high UBE2H expression promotes tumor progression, leading to increased T cell exhaustion (PD1) and decreased activation (CD69, GZMB, IFNR) in the immune microenvironment, impaired antigen presentation of macrophages and neutrophils (H2Kb, I-Ab, I-A/E), and increased neutrophil extracellular trap formation (MPO, NE, CITH3). This study revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche and analyzed the inhibition of UBE2H on the immune microenvironment of pancreatic cancer through single-cell sequencing and in vivo experiments, providing important clues for the formation of “cold” tumors in pancreatic cancer and opening new directions for exploring new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ecc8251593af0fcef99c0ebc48949b3c7fba9f" target='_blank'>
              Integrated scRNA-seq and transcriptome analyses uncover the effects of UBE2H on the immune microenvironment regulation in pancreatic cancer
              </a>
            </td>
          <td>
            Fuxin Huang, Zhongyan Zhang, Jike Fang, Yue Chen, Jinhui Wei, Quanzhang Li, Chuanzhao Zhang, Shanzhou Huang, B. Hou
          </td>
          <td>2025-11-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/128cca3d1edd8a7877ad22a89e79521ff40bea75" target='_blank'>
              Single-cell sequencing in bladder cancer: new insights from tumor cell diversity to individualized treatment strategies.
              </a>
            </td>
          <td>
            Cen Liufu, Shuai Ye, Cong Yin, Tao Zhu, Jiahao Jiang, Yan Wang, Bentao Shi
          </td>
          <td>2025-12-03</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bc8c7de17d28677187bc796f682ca2aa0c8467e" target='_blank'>
              Single-cell analysis reveals the cellular and transcriptional diversity of thyrocytes in the normal pediatric thyroid
              </a>
            </td>
          <td>
            E. R. Reichenberger, Nicholas E. Bambach, Zach H Spangler, J. Ricarte-Filho, K. Hinkle, A. Isaza, Tricia Bhatti, A. J. Bauer, A. Franco
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a leading male malignancy worldwide, with metabolic reprogramming being a critical hallmark of its progression. Extracellular vesicles (EVs) derived from tissues directly reflect the tumor microenvironment, offering unique insights into cancer pathophysiology that are unattainable through cell line or biofluid-derived EVs. However, the functional roles of tissue-derived EVs in PCa metabolism remain poorly understood. Leveraging our expertise in murine PCa model establishment and EV isolation from prostate tissue, this study aimed to characterize functional differences between PCa and normal prostate tissue via proteomic analysis of tissue-derived sEVs. We orthotopically implanted luciferase-labeled PCa cells into nude mice to establish an in situ PCa model, confirmed tumor formation via in vivo imaging, and harvested tissues after 4 weeks. sEVs were isolated using ultracentrifugation combined with an iodixanol density cushion and characterized by transmission electron microscopy, nanoparticle tracking analysis, and protein marker profiling. Proteomic analysis identified 28 upregulated and 24 downregulated proteins in PCa-derived sEVs compared to normal controls. Subcellular localization revealed enrichment in the cytoplasm, while pathway analysis highlighted significant involvement in metabolic processes, particularly glycolysis, amino acid biogenesis, carbon metabolism, and pyruvate metabolism. Our study establishes a robust method for isolating prostate tissue sEVs and provides the first evidence that PCa tissue-derived sEVs exhibit profound metabolic pathway alterations. These findings enhance our understanding of PCa progression mechanisms and may facilitate the development of novel diagnostic biomarkers and therapeutic strategies targeting metabolic dysregulation in PCa. SUMMARY: In this study, we created a method to isolate prostate tissue small EVs, based on our knowledge of the murine prostate cancer model building. Our data suggested that prostate tissue small EVs proteins significantly changed in many metabolism pathways, such as Glycolysis, Biogenesis of amino acids, Carbon metabolism and Pyruvate metabolism. In this study, we are the first to report prostate tissue-derived EVs proteins enriched in alterations of cancer metabolism. These differential proteins in PCa tissue EVs reflect metabolic changes in PCa and may provide insights into the development of early diagnostic biomarkers or novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c976599295f2bbc1baf63bb2a61d05088e5e38" target='_blank'>
              Proteomics of Prostate Cancer Tissue Small Extracellular Vesicles Reveal Alteration of Metabolism.
              </a>
            </td>
          <td>
            Lijuan Yu, Ting Ding, R. Olofsson Bagge, Lei Zheng, Xiaoke Hao
          </td>
          <td>2025-11-17</td>
          <td>Proteomics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9b43f154dcd18b57efa496cdfa1d3835d1a5b6e" target='_blank'>
              EPI-SauriCas9-based mouse ovarian cancer models recapitulating pten deletion in patients.
              </a>
            </td>
          <td>
            Wutao Chen, Pengju He, Ling Ding, Wei-Hua Lou, Yishu Wang, Weiwei Shi, Zhangzhengyi Fan, Yumeng Sheng, Jing Luo, Zhixing Tan, You Wang, Wen Di, Xiaoping Ke, Bin Yu
          </td>
          <td>2025-12-29</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) occurs as a complex, multifaceted event driven by the interplay of tumor-permissive genetic mutations, the nature of the cellular origin, and microenvironmental stress. In this study, using primary human pancreatic acinar 3D organoids, we performed a CRISPR-KO screen targeting 199 potential tumor suppressors curated from clinical PDAC samples. Our data revealed significant enrichment of a list of candidate genes, with neurofibromatosis type 2 associated gene (NF2) emerging as the top target. Functional validation confirmed that loss of NF2 promoted the transition of PDAC to an invasive state, potentially through extracellular matrix modulation. NF2 inactivation was found to enhance PDAC cell fitness under nutrient starvation. This adaptation not only reinforced the oncogenic state but also conferred therapeutic resistance. Additionally, we found that NF2 loss was associated with fibroblast heterogeneity and cancer-stroma communication in tumor evolution. These findings establish NF2 as a critical tumor suppressor in PDAC and uncover its role in mediating nutrient adaptation and drug resistance. Importantly, this study provides additional insights into drug resistance mechanisms and potential therapeutic targets in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c5d227985bf39716bd4f69d5a354d37cc46442" target='_blank'>
              NF2 loss malignantly transforms human pancreatic acinar cells and enhances cell fitness under environmental stress
              </a>
            </td>
          <td>
            Yi Xu, Michael H. Nipper, Angel A. Dominguez, Chenhui He, Francis E Sharkey, Sajid Khan, Han Xu, Daohong Zhou, Lei Zheng, Yu Luan, Jun Liu, Pei Wang
          </td>
          <td>2026-01-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Due to their intrinsic heterogeneity and plasticity, pediatric brain tumors present highly complex clinical challenges. To provide insight into the molecular scene of these tumors, we generated spatial transcriptomics data from 19 distinct patients, 7 of which suffered a relapse of the disease. In this cohort, spanning 59 tissue sections across 8 diagnoses and 4 tumor grades, we recovered diagnosis-specific gene expression patterns that could be linked to processes characteristic of developmental stages. We also identified between 2 to 4 spatial archetypal niches per section, which could then be related to 11 main biological themes, and to distinct celltype-like transcriptomic signatures. For example, we noted strong spatial correlations between developmental archetypes and oligodendrocyte lineage signal in pilocytic astrocytoma. Lastly, we utilised spatially inferred copy-number variants for the profiling of relapse-associated tentative tumor clones. These clones were then related via spatial geographical analysis to regions with strong blood vessel signatures. Overall, these results provide vital details into the progression and maintenance of pediatric brain tumors, offering novel edges to be exploited in personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f69864dd9511e02d9a815817c172642be7190a7d" target='_blank'>
              Surveying the molecular landscape of pediatric brain tumors via spatially resolved transcriptomics
              </a>
            </td>
          <td>
            Javier Escudero Morlanes, Timo-Pekka K Lehto, L. Larsson, L. A. Galicia, A. Mollbrink, Alia Shamikh, Elisa Basmaci, Gabriela Prochazka, T. de Ståhl, Johanna Sandgren, F. Taylan, Bianca Tesi, Ann Nordgren, Andrew Erickson, Alastair D Lamb, Klas Blomgren, Monica Nistér, J. Lundeberg, R. Mirzazadeh, L. Kvastad
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9bb04ebfe32c3c0fbf91f302efd50e3e49a5efa" target='_blank'>
              Systematic analysis of immune cell motility leveraging the open intravital microscopy database Immunemap.
              </a>
            </td>
          <td>
            D. U. Pizzagalli, Pau Carrillo-Barberà, Himanshu Bansal, Elisa Palladino, Kevin Ceni, Benedikt Thelen, Alain Pulfer, Enrico Moscatello, Raffaella Fiamma Cabini, Johannes Textor, Inge M N Wortel, Rolf Krause, Santiago Fernandez Gonzalez
          </td>
          <td>2025-11-20</td>
          <td>The EMBO journal</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Spatial biology technologies enable high-dimensional profiling within intact tissues, revealing how molecular and cellular organization drives function and disease. As these platforms gain broader adoption, standardized analytical frameworks are needed to ensure data quality and reproducibility. Here, we present an end-to-end pipeline for the GeoMx Digital Spatial Profiler that simultaneously generates whole-transcriptome and 637-protein measurements from user-defined regions within the same tissue sections. The workflow integrates morphology-guided region selection, quality control, normalization, and multi-modal data interpretation. Applied to formalin-fixed cortical tissues from Alzheimer’s disease, dementia with Lewy bodies, amyotrophic lateral sclerosis, and controls, the framework resolves spatially distinct molecular domains. Transcript and protein signals diverge across amyloid plaque cores and surrounding glial-rich regions, with RNA–protein concordance varying by disease condition, while single-neuron profiling with and without pathogenic tau deposition illustrates protein assay sensitivity. This dataset provides a rigorously validated resource for spatial multi-omic analyses and establishes broadly applicable guidelines for reliable, reproducible profiling of complex tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f33c108429c953b8d97ff4a07ced8310b832aa9" target='_blank'>
              Spatial Multi-Omics Workflow and Analytical Guidelines for Alzheimer’s Neuropathology
              </a>
            </td>
          <td>
            Xuehan Sun, Hannah R. Hudson, Timothy C. Orr, Srinivas Koutarapu, Alyssa Rosenbloom, Matthew H. Ingalls, O. Braubach, C. D. Keene, Joseph M. Beechem, Miranda E. Orr
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background CD47 is a “don’t eat me” signal that is overexpressed in tumors to evade phagocytosis by tumor associated macrophages (TAM). Investigational agents targeting CD47, such as magrolimab, aim to induce phagocytosis of tumor cells by TAMs. Previously, two key TAM subsets have been identified: C1QC TAMs, which display pro-phagocytic activity, and SPP1 TAMs that express pro-angiogenic markers. We characterize CD47 expression and its relationship with tumor macrophages in solid tumor samples. Patients and methods Resectable tumors from head and neck squamous cell carcinoma (n=36) (HNSCC), breast cancer (n=37) (BC), and colorectal cancer (n=36) (CRC) were evaluated for CD47 expression by immunohistochemistry (IHC), two multiplex immunofluorescence panels were used to characterize TAM markers and T cell markers. RNA sequencing was also performed. Results CD47 protein expression was higher on tumor cells compared to stromal cells across tumor indications tested. Although CRC had the lowest prevalence for CD47 expression in primary tumors, we observed a marked increase in CD47 expression in CRC liver metastases. We developed an SPP1 TAM gene signature and validated a C1QC TAM gene signature to estimate TAM abundances from bulk RNA-Seq data. In the TAM mIF analysis, HNSCC had the highest macrophage density of the indications tested. We observed a positive correlation between a higher C1QC: SPP1 TAM ratio and macrophage phenotype and tumor T cell density. C1QC macrophage signatures correlate with tumor CD47 protein expression in BC and HNSCC samples, suggesting interplay between them. Conclusions We characterized CD47 expression across key solid tumor indications being evaluated clinically using anti-CD47 blockade agents: HNSCC, breast cancer and CRC. Using a CD47 IHC assay, we identified HNSCC as an indication with the highest CD47 expression. In addition, we quantified tumor macrophages using multiplex immunofluorescence (mIF) and determined that HNSCC had the highest density of TAMs. Compared to relatively low CD47 expression in primary CRC tumors, CRC liver metastases had very high CD47 expression. Quantification of TAM signatures and CD47 expression represent key biomarkers to monitor in patient samples during exploration of CD47-blockade agents in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31725c03ca6311073138a838325de55dcf4fe468" target='_blank'>
              CD47 expression in solid tumors correlates with phagocytic tumor-associated macrophage gene signature
              </a>
            </td>
          <td>
            Nicholas van Buuren, Mengshu Xu, Yi Zhang, Paola Correa, Shiva Zaboli, Azadeh Madjidi, Christina Moon, Szu-Wen Liu, Ruidong Li, Kai Sun, Shahed Iqbal, Abhishek Aggarwal, Min Wang, Li Li, Jared M Odegard, Kelli Boyd
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Spatial transcriptomics, which deals with tissue architecture in genetic investigation, is an innovative technique for examining cell heterogeneity and tissue organization. This review emphasizes major approaches, include spatially resolved transcriptome methods, immunohistochemistry as well as in situ hybridization, all of which permit the mapping of RNA molecules in their native tissue environment. These methods have proven essential in achieving our understanding of biological events such as tumor evolution, progression of cancer, and cancer tumor stem cell detection. Spatial transcriptomics, the study of patterns of gene expression in space, reveals the intricate nature of the tumor microenvironment (TME) and its effect on cancer biology. Although it delivers insight on the cellular connections that underlie disease, the significance of spatial transcriptomics in multiple organs has expanded.

Although its immense potential, there are still difficulties to be conquered, particularly within the areas of analysis of data, spatial resolution, and integration with other omics data. To be able to fully comprehend the complexities of tissues biology and ailments, this review additionally tackles future potential avenues, including the necessity for greater multiplexing, enhanced resolution, and the combination of functional genomics. With this synthesis, we intend provide an extensive summary of the state of spatial transcriptomics currently and demonstrate that it possesses the potential to improve precision medicine, cancer research, and our understanding of broader biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579b9e172c2769b4ad99d0da42fa24b04f1ee980" target='_blank'>
              Spatial Transcriptomics in Cancer Research: Methods and Applications
              </a>
            </td>
          <td>
            P. K. Kumar, K. M. Rao, Syed Mohiuddin Quadri, Bagadam Aparna, V. U. M. rao
          </td>
          <td>2025-12-18</td>
          <td>International Journal of Current Science Research and Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCC) with multinodular morphology were typically polyclonal and exhibited extensive heterogeneity. We explored the multiple-layer heterogeneity of HCC utilizing single-cell multiomics sequencing together with multiregional sampling. We found that Confluent Multi-Nodular samples exhibit more heterogeneous immune landscapes characterized by increased transcriptome heterogeneity and more complex immune-related interactions compared to Single Nodular (SN) samples. We identified differential DNA methylation patterns among distinct tumor foci in a subset of liver cancer patients. Global DNA hypomethylation in cancer cells predominantly occurs in partially methylated domains. We predicted and validated two genes, GADD45A and SNHG6, as potential drivers of DNA hypomethylation. Lastly, we demonstrated that DNA methylation distance serves as a more robust metric than gene expression distance for reconstructing tumor evolutionary trajectories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eb1f83715ac49e3d497001d0455b6e605a23122" target='_blank'>
              Interrogating subclonal heterogeneity of liver cancer with single-cell multi-omics analysis
              </a>
            </td>
          <td>
            Liwen Zhang, Dandan Hu, Xing Peng, Dan Su, Lei Chang, Yi Guo, Kai Tian, Qingpei Huang, Enze Deng, Zhuojie Su, Longxi Li, Xiong Xiong, Weiyuan Ma, Hongjian Qi, Yaojun Zhang, Yanxiao Zhang, Xiaoying Fan
          </td>
          <td>2025-11-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is an aggressive cancer thought to arise from impaired myogenesis. This can be substantially overcome by increasing the levels of pannexin 1 (PANX1), a critical component of the myogenic program, but the mechanism involved is unknown. Using RNA-seq, we have previously found that overexpression of PANX1 dramatically reshapes the transcriptomic landscape of RMS including downregulation of a myogenic modulator, APOBEC2 (apolipoprotein B mRNA editing enzyme catalytic subunit 2). Following this clue, we investigated the role of APOBEC2 in the PANX1-mediated suppression of RMS malignancy. Here we show that, using a panel of patient-derived RMS cell lines and tumor specimens, APOBEC2 is expressed in RMS, but that its levels are lower than those in both differentiating myoblasts and skeletal muscle. In most RMS cell lines examined, APOBEC2 accumulates during proliferation and sustains their stem-like characteristics, as evidenced by its ability to promote the growth of spheroids upon increased expression. Yet, ectopic PANX1 expression led to a marked downregulation of APOBEC2 across a large proportion of RMS cell lines assessed. Strikingly, these were the same cells in which PANX1 triggers multinucleation. We further reveal that, like healthy myoblasts progressing through myogenesis, the multinucleation observed here in RMS cells results from cell fusion. Importantly, in RMS cells engineered to overexpress APOBEC2, PANX1 no longer enhances cell fusion, but its other anti-tumorigenic properties are still preserved. Collectively, our data indicate that PANX1 promotes RMS cell fusion by downregulating APOBEC2 expression, driving these tumor cells further into the myogenic program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/398de031cdf1c2189823064d4c0ea5c0fa3f3f02" target='_blank'>
              Pannexin 1 induces Rhabdomyosarcoma cell fusion by downregulating APOBEC2
              </a>
            </td>
          <td>
            Alexandra Welten, Amit Bera, S. Langlois, X. Xiang, Keshav Gupta, Emily Freeman, Kyle N. Cowan
          </td>
          <td>2025-11-18</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/027f986afc66cae82f4bb168ec0c6120d17b62d0" target='_blank'>
              Transgelin defines pro-tumorigenic cancer-associated fibroblasts in pancreatic cancer.
              </a>
            </td>
          <td>
            Xingxing Wang, Keiko Shinjo, Kohei Kumegawa, R. Maruyama, Shinji Mii, Yukihiro Shiraki, Tastunori Nishimura, Y. Murofushi, Miho M. Suzuki, T. Kabasawa, Mitsuru Futakuchi, A. Kanai, Yutaka Suzuki, Atsushi Enomoto, Yutaka Kondo
          </td>
          <td>2025-12-16</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1100b9077e6e7f158509d2bfcdf86085e31d43e6" target='_blank'>
              Tumour-reactive heterotypic CD8 T cell clusters from clinical samples.
              </a>
            </td>
          <td>
            Sofía Ibáñez-Molero, J. Veldman, Juan Simon Nieto, Joleen J H Traets, Austin George, K. Hoefakker, Anita Karomi, R. Harkes, Bram van den Broek, Su Min Pack, L. Tas, Nils L Visser, Susan E. van Hal-van Veen, P. Alóndiga-Mérida, M. Alkemade, I. M. Seignette, R. Tissier, M. Nieuwland, M. van Baalen, Joanna Poźniak, Erik Mul, S. Tol, S. Stenqvist, L.M. Nilsson, Jonas A Nilsson, J. Haanen, W. V. van Houdt, D. Peeper
          </td>
          <td>2025-11-19</td>
          <td>Nature</td>
          <td>2</td>
          <td>59</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b2473c436968a0964a051f1901617b06af4fa0b" target='_blank'>
              Systemic activation and tissue infiltration of CD8 + CX3CR1 + T cells in non-small cell lung cancer treated with neoadjuvant immune checkpoint blockade.
              </a>
            </td>
          <td>
            Sisi Dai, Yongcheng Liu, Tingting Guo, Hong Luo, Guanglei Yang, Songliang Yu, Sifan Zhang, Lili Jiang, Jie Liu, Ye Wang, Daxing Zhu, Xuyu Cai
          </td>
          <td>2025-11-22</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Introduction The tumor microenvironment (TME) is a complex ecosystem where cellular interactions are spatially governed. Spatial transcriptomics (ST) technologies, sequencing-based Visium which only have spot level resolution and imaging-based Xenium which are limited by a small RNA panel, probe the complexity, but high dimensional data interpretation remains a challenge [1]. Current analyses rely on pre-defined cell annotation patterns, critically failing to account for higher-order structures in gene co-expression and cell co-localization [2]. Biologically, cellular function is defined by coordinated gene modules, not individual genes, and these relationships exhibit a hierarchical organization across multiple spatial scales. A simple gene expression matrix is insufficient to capture this multi-scale and hierarchical structure of gene–gene and cell–cell interactions, especially given that TME niches are highly variable and context-dependent, difficultly to make a fixed scale definition. Furthermore, the order of cells in discrete gene expression matrices can affect the ability of neural networks to analyze cell–cell interactions. To overcome this, we introduce CoSMIM (Covariance of Spatial Multi-scale Integrated Microenvironment) matrix. Our hypothesis is that gene–gene covariance matrices, computed across multiple scales of neighbor cells, provide a richer representation of the TME’s spatial heterogeneity functional architecture. We use a multi-decoder conditional variational autoencoder(CVAE) to learn a unified latent space for consistent cross-scale modeling. Its architecture can impute genes in targeted spatial panels and infer spatial covariance matrices for non-spatial single-cell data, thus bridging these modalities. Methodology Our framework consists of two core components: the construction of a CoSMIM niche representation and a CVAE generative model for learning a unified representation. The input to our method is ST data from dataset HEST-1 k and 10X human-breast cancer FFPE dataset [3,4] which has various ST platforms such as Visium, Xenium, CosMx, or MERFISH. The specific steps are as follows (Fig. 1b): Definition and construction of multi-scale neighborhoods Tumor spatial heterogeneity is not random but is driven by the organizational patterns of cellular communities at different scales. Precisely defining cellular niches within ST data remains a key challenge. Given the heterogeneity in cell size, defining neighborhoods using a fixed number of cells or a constant radius has limitations [5]. We propose an adaptive approach that constructs neighborhoods through a learned linear combination of different biological scales (Fig. 2&Eq.2). We predefine a series of spatial neighborhood representing distinct biological interaction ranges. The gene expression matrix of whole tissue is \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\mathrm{X}\in{R}^{n\times g}$\end{document}, \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $n$\end{document} is the number of cells and \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\mathrm{g}$\end{document} is number of genes. Each cell has a muti-scale niches matrix \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} ${E}_k^i\in{R}^{k\times g}\left(\mathrm{e}.\mathrm{g}.\mathrm{k}\in \mathrm{K}\left\{\mathrm{5,15,50}\right\}\right)$\end{document} of \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $k$\end{document} nearest neighbor cells (excluding the target cell \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $i$\end{document} itself). Let \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\mathrm{Cov}\in{\mathrm{R}}^{g\times g}$\end{document} be the target cell \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $i$\end{document}’s gene–gene covariance of \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $k$\end{document} nearest neighbor cells. \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\overline{X}$\end{document} global mean gene expression of gene g and \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\overline{E_k}$\end{document} is \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $k$\end{document} scale niche’s mean gene expression. \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $$ \mathrm{Co}{v}_k^{troditional}={\sum}_{i=1}^n Co{v}_k\left({E}_i\right)={\sum}_{i=1}^n\frac{1}{k}{\left({E}_i^k-\overline{E_i^k}\right)}^T\left({E}_i^k-\overline{E_i^k}\right) $$\end{document} \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $$ \mathrm{Co}{v}_k^{our\; method}={\sum}_{i=1}^n\mathrm{Co}{\mathrm{v}}_{\mathrm{k}}\left({E}_i^k\right)={\sum}_{i=1}^n\frac{1}{k}{\left({E}_i^k-\overline{X}\right)}^T\left({E}_i^k-\overline{X}\right) $$\end{document} \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $$ CoSMIM=\sum_{k={k}_1}^K{w}_k Co{v}_k^{ourmethod},{w}_k\ge 0,\sum_{k={k}_1}^K{w}_k=1(1) $$\end{document} We then evaluate and optimize the selection of these scales \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $K$\end{document} based on the discriminatory power of their corresponding eigenvalues in principal component analysis. These scales correspond to different biological contexts: Small-scale (5–15 cells) Primarily captures direct signaling and short-range paracrine signaling, such as direct contact between a tumor cell and an adjacent immune cell PD-1/PD-L1 or local growth factor support from a Cancer-Associated Fibroblast. Medium-scale (15–50 cells) Reflects the characteristics of a functional niche, such as an immunosuppressive or pro-angiogenic ‘community’ composed of several tumor, immune, and stromal cells. Large-scale (>$50cells) Characterizes the tissue region state, such as broad hypoxic zones, regions of lymphocytic infiltration, or fibrotic stromal areas, which are often governed by diffusive chemical gradients or physical barriers. The scale weights are learned by entropy-regularized log-det minimization: \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $$ {\mathrm{w}}^{\ast }=\arg \underset{\mathrm{w}\in{\Delta}^{\mathrm{K}-1}}{\min}\left(-\log \left|\Sigma \left(\mathrm{w}\right)+\upvarepsilon \mathrm{I}\right|+\uplambda{\sum}_{\mathrm{k}=1}^{\mathrm{K}}{\mathrm{w}}_{\mathrm{k}}\log{\mathrm{w}}_{\mathrm{k}}\right)\kern0.36em (2) $$\end{document} With \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\varepsilon >0$\end{document} for numerical stability and \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgreek} \usepackage{mathrsfs} \setlength{\oddsidemargin}{-69pt} \begin{document} $\lambda \ge 0$\end{document} (larger \documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{upgre">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90a83168d353c1d44c04ce73b829897d6c778eb0" target='_blank'>
              CoSMIM: a covariance-based multi-scale framework for integrated spatial microenvironment analysis
              </a>
            </td>
          <td>
            Bairui Du, H. Kiryu
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7a9a257557fe4b01c761e5ab2bf1e53a00ef799" target='_blank'>
              TIF1γ drives oral cancer recurrence by the transcriptional regulation of self-renewal genes, such as HES1
              </a>
            </td>
          <td>
            K. P. Padmaja, A. Mohan, G. M. Nair, Jyothy S Prabhu, Hafsa Shabeer, V. P. Snijesh, P. G. Balagopal, A. Thameem, Alan Jose, N. George, Jiss Maria Louis, Parvathy Visakh, Gayathri V Nair, Anna P Joseph, V. S. Jaikumar, Riya Ann Paul, Jackson James, T. T. Maliekal
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4393b9168f5ebc4a80f332a0eba92fab9bfeef3" target='_blank'>
              Single-Cell Multiomic Profiling Uncovers Radiation Dosage-Sensitive, Cluster-Specific Regulatory Dynamics in Glioblastoma
              </a>
            </td>
          <td>
            Khoi Huynh, Sara Nalina Barcik Weissman, Cheol Park, Maria L.G. Quiloan, Blake Sun Chang, Eric L Chang, Kelly Street, David Tran, A. Modrek
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary This review was conducted to address the complexities of breast cancer research, which is often limited by two-dimensional cell cultures that fail to accurately represent the tumor microenvironment. Although three-dimensional (3D) models are generally superior, the absence of standardized methodologies has resulted in inconsistent findings. The objective of this review was to systematically analyze how different 3D culture methods influence the epigenetic reprogramming of tumor cells, focusing on changes in DNA methylation, histone modifications, and chromatin organization, as well as the differential expression of various types of non-coding RNA. Our findings demonstrate that different 3D culture methods are not equivalent and yield distinct epigenetic signatures. This underscores the urgent need to standardize protocols and provide detailed reports on the properties of culture media to enhance reproducibility, which is crucial for identifying new therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/933fb504bd5970ab97b7de6c042e05e852c76571" target='_blank'>
              Epigenetic and Transcriptional Reprogramming in 3D Culture Models in Breast Cancer
              </a>
            </td>
          <td>
            Laura Cecilia Flores-García, Karla Rubio, Eloísa Ibarra-Sierra, M. B. Silva-Cázares, C. Palma-Flores, C. López-Camarillo
          </td>
          <td>2025-11-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The field effect describes the phenomena where environmental exposures, infection, and genetic predisposition result in molecular changes in cells that predispose them to developing cancer. Though this is a well-established concept in pathology, it remains underexplored in the context of high-resolution omics. We utilized the Diagnostic Evidence Gauge of Single Cells (DEGAS) deep transfer learning framework to analyze prostate cancer spatial transcriptomics to identify cells and tissues that are highly associated with cancer progression. DEGAS highlighted morphologically benign glands that had reduced expression of MSMB, a differentiation marker that is decreased in aggressive tumors. These glands have upregulated genes associated with antigen presentation and aggressive neoplasms. Integration of single-cell transcriptomics and deep learning image analysis separately revealed altered immune-cell infiltration, suggesting a complex interplay in the tumor environment facilitating aggressiveness. We used immunohistochemistry to quantify the MSMB protein (PSP-94) expression on morphologically normal and tumor tissues from patients with and without 5-year distant metastasis. Samples from patients who developed metastasis consistently showed lower fractions of positively stained cells, indicating a subtle yet significant "field effect" in seemingly benign regions. These proteomic results validate the transcriptomic findings and further underscore that inflammatory or immune-related changes in ostensibly normal tissue may contribute to aggressive disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68d7d78a2fef9048f9bcafbf03164f9002e0042" target='_blank'>
              Deep Transfer Learning Links Benign Glands to Prostate Cancer Progression via Transcriptomics.
              </a>
            </td>
          <td>
            Justin L Couetil, Ziyuan Liu, Chao Chen, A. Alomari, Kun Huang, Jie Zhang, Travis S. Johnson
          </td>
          <td>2025-11-29</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) progression involves dynamic remodeling of the tumor microenvironment (TME). However, the stage-specific dynamics of immune and stromal cell remodeling throughout LUAD progression remain incompletely understood. Here, the study systematically profiles the cellular composition and transcriptional states across multiple LUAD stages, integrating early-stage patient specimens with publicly available datasets encompassing advanced-stage disease. The analysis reveals a marked stage-dependent shift from a proliferative and immune-activated microenvironment in early LUAD to a hypoxia-enriched and immunosuppressive landscape in advanced disease. A distinct hypoxia-adapted epithelial tumor cell subpopulation (C5), exhibiting transcriptional features of metastasis, invasion, and hypoxia, and poor prognosis, is identified. Advanced LUAD featured immunosuppressive LGMN⁺ macrophages and STAT1-driven exhausted CD8⁺ T cells. FKBP11⁺ plasma B cells exhibited exhaustion-linked metabolic changes. POSTN⁺ CAFs emerged as central mediators of extracellular matrix (ECM) remodeling and immune exclusion. Collectively, the findings reveal a model of hypoxia-driven functional convergence, in which distinct TME components co-evolve toward phenotypes that collectively promote immune evasion, matrix remodeling, and tumor progression. These findings may provide insights into stage-specific cellular dynamics and highlight promising therapeutic targets for precision immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8876e01109cf8f024442bf59e35e9ed246f97f" target='_blank'>
              Characterizing Stage-Specific Cellular Dynamics and Microenvironmental Remodeling in Lung Adenocarcinoma by Single-Cell RNA Sequencing.
              </a>
            </td>
          <td>
            Bomiao Qing, Xiaolan Li, Xiang He, Junyi Wang, Yi Yang, Manling Jiang, Bingbing Yan, Lei Zhang, Anying Xiong, Q. Ran, Guoping Li
          </td>
          <td>2025-11-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb706d249cfb2775d83ca53d2e8588e8894f0cf6" target='_blank'>
              Dual Transcriptional Drivers of Immune Suppression in SACC: NOTCH1 for Cellular and MYB for Humoral Immunity
              </a>
            </td>
          <td>
            Guoliang Yang, Xudong Wang, Tian Ye, Tingyao Ma, Youmei Chen, Fang Nan, Qian Chen, Lu Kong, Xiao-hong Chen
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Precision medicine has evolved through distinct phases, from the origins of the Human Genome Project to mutation‐based targeted therapies. This editorial posits that “stereological cell biomedicine” could be a new approach promoting the development of the next generation of precision medicine. This emerging discipline transitions the focus from genomic data to the multi‐dimensional and spatiotemporal complexity of single cells. Driven by advances in Stereo single‐cell multi‐omics (Stereo Cell‐seq), spatial transcriptomics (Stereo‐seq), and single‐cell surfaceomics (sc‐surfaceome), this approach aims to capture the stereologically dynamic interactions between organelles within a cell and between cells in the tissue. We argue that understanding the spatiotemporal location of molecules, particularly protein interactions at organelle interfaces and on the cell surface, is as critical as their abundance for defining cellular function in health and disease. Integrating these high‐resolution measurements with artificial intelligence and computational modelling will bridge the gap between advanced omics and pathology. Initiatives such as the newly established European Stereo Cell Center (ESCC) signal a global shift towards this new paradigm, which promises to unlock novel diagnostic biomarkers and therapeutic targets for truly multi‐factorial and dynamic precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acdcd69ccf472862bb486c6c2a16a93191b0f96" target='_blank'>
              Stereo cell: A new approach to the next generation of clinical precision medicine
              </a>
            </td>
          <td>
            Wanxin Duan, Mingjie Wang, Yifei Liu, Celine Desoyer, Christian Baumgartner, Xiangdong Wang
          </td>
          <td>2025-11-28</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epidermal growth factor receptor (EGFR) mutation is a key oncogenic driver in lung adenocarcinoma (LUAD), but its impact on the tumor immune microenvironment (TIME) remains unclear. By integrating single-cell transcriptomes from 153 LUAD samples using machine learning, we generated an atlas of over one million cells that delineates immune heterogeneity. EGFR-mutant tumors exhibited enrichment of TIGIT+regulatory T cells, neutrophils, and macrophages, whereas wild-type tumors contained abundant ZNF683+CD8+tissue-resident memory T cells, diverse memory B cells, and FGFBP2+CD16high natural killer cells, reflecting an immune-active TIME. Non-negative matrix factorization defined five TIME subtypes, with EGFR-mutant patients clustering into immunosuppressive profiles linked to poor prognosis. Flow cytometry and mouse models confirmed the cytotoxic and PD-1 blockade-enhancing functions of FGFBP2+NK cells. These findings reveal distinct TIME landscapes in EGFR-mutant LUAD and illustrate the potential of machine learning-based immunogenomic analysis to inform precision immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad6284204cf8fc0e7c47d8fa667f142a64e34e5" target='_blank'>
              Machine learning identifies TIME subtypes linking EGFR mutations and immune states in lung adenocarcinoma
              </a>
            </td>
          <td>
            Zetian Gong, Mingjun Du, Ying Li, Bicheng Ye, Yuming Huang, Hui Gong, Wei Wang, Liang Chen, Zongli Ding, Pengpeng Zhang
          </td>
          <td>2025-11-26</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ebcdfb94400220dc54cbd6a2cfd578841de975f" target='_blank'>
              Integrated transcriptomic and single-cell analysis reveals cell cycle dysregulation and cellular heterogeneity in lung cancer.
              </a>
            </td>
          <td>
            Yuzi Zhang, Wencheng Che, Qingchuan Li, Wei Huang
          </td>
          <td>2025-12-30</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ca98a4e613e1447cb2daf8bdbf25c8434e308a3" target='_blank'>
              Multi-dimensional omics integrated machine learning framework identifies macrophage-fibroblast-tumor co-infiltration patterns to predict prognosis in gastric cancer.
              </a>
            </td>
          <td>
            Qi Wang, Yuan Ni, Sheng Lu, Benyan Zhang, Jun Ji, Qu Cai, Chao Yan, Feng Qi, M. Shi, Jun Zhang
          </td>
          <td>2025-12-09</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Prostate cancer exhibits a strong familial association, and its heritability indicates a significant contribution from germline variants. While genome-wide association studies (GWAS) have identified common germline variants associated with prostate cancer risk, translating these statistical associations into functional mechanisms has remained a long-standing challenge. Consequently, most of our understanding of the genetic basis of prostate cancer stems from extensive studies of somatic mutations, leaving the germline genetic architecture largely unresolved. Because most germline variants lie in the noncoding genome and complex human diseases are predominantly driven by regulatory mutations, we herein asked which prostate cell types mediate the functional effects of germline variants, and thus represent the most genetically vulnerable populations. We generated paired epigenomic and transcriptomic profiles from reference human prostate tissues. Integrating these single-cell data with large-scale GWAS data identified a terminally differentiated luminal epithelial subtype that mediates the strongest germline risk in prostate cancer. We subsequently developed a deep learning model to score ~17 million GWAS variants based on their predicted impact on altering local chromatin accessibility in this vulnerable luminal epithelial subtype, and identified high-confidence candidate loci where high-risk germline variants likely alter promoter accessibility in prostate cancer. The implicated genes were involved in several pathways in tumorigenesis, displayed strong dosage sensitivity, and converged on the androgen receptor (AR)-mediated regulon, a mechanism also observed for somatic mutations. Overall, by unveiling cell types and candidate loci that mediate germline risk, our study defines the cell-type-specific germline architecture in prostate cancer and provides a comprehensive framework for understanding cancer heritability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5e64e9e662cca846ddc03a92460330d25c0b2de" target='_blank'>
              Decoding the germline genetic architecture of prostate cancer at a single cell resolution
              </a>
            </td>
          <td>
            Cheng Wang, Tianqi Tang, Y. Wang, Jingjing Li
          </td>
          <td>2025-12-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Prostate cancer commonly presents as multifocal disease, with distinct tumours that are often genetically independent. Metastases, however, typically derive from a single dominant clone, highlighting the clinical challenge of identifying which lesion drives progression. Focal therapy, an emerging treatment strategy, aims to target only the most aggressive tumour focus or foci, sparing normal prostate structures and reducing morbidity. Yet, the lack of consistent genomic drivers in primary tumours complicates efforts to identify the focus most likely to progress. While prostate cancers are known to harbour widespread methylation alterations, the extent to which these epigenetic changes track with clonal origin remains poorly understood. To address this, we performed multiregional epigenomic sequencing of 189 prostate tissue samples, including 109 tumour foci and 80 non-tumour regions from 44 patients (2-8 tumour samples per case). Differential methylation analysis revealed widespread alterations distinguishing benign from tumour tissue, including a core ‘epigenetic trunk’ of >20,000 recurrently altered CpGs shared across cases. DNA variants called from non-methylation sites of the epigenomic sequencing closely matched variants from WGS in cases with WGS available. Phylogenetic trees constructed from DNA variants and phyloepigenetic trees from methylation sequencing revealed striking patterns. Some cases lacked shared DNA variants across all tumour foci, indicating a polyclonal tumour origin, yet these genetically unrelated foci shared extensive methylation alterations. This suggests widespread methylation convergence occurs in clonally distinct tumour foci. To dissect clonal relationships further, we applied two orthogonal lineage tracing approaches using the methylation sequencing data: fluctuating CpGs (sites that change stochastically over time) and highly entropic 8-mer ‘methylation barcodes’ within the protocadherin gene cluster. Fluctuating CpGs showed concordant states across tumours that shared DNA sequence variants, while the methylation barcodes likewise revealed high similarity among such tumours. Together, these methods support the independent evolutionary origins of polyclonal prostate tumour foci. Pathway analysis of recurrently altered methylation loci highlighted consistent enrichment in epithelial–mesenchymal transition, MYC signalling, DNA damage response, and hormonal signalling across cases, suggesting functional convergence at the pathway level despite divergent genetic origins, consistent with selective pressures acting on shared programmes. Together, these findings demonstrate that DNA methylation alterations represent fundamental and recurrent events in prostate cancer, more consistent than DNA sequence variants across multifocal disease. Lineage-tracing analyses further highlight that methylation processes record tumour evolutionary history, encoding both clonal relationships and convergent biology. These observations may inform methylation-based biomarkers and the rational selection of lesions for focal therapy.



 Tamsin J. Robb, Melissa Cheung, Rajbir Batra, Henson Lee Yu, John C. Thomas, Anne Warren, Andrew Lynch, Daniel Brewer, David Wedge, CRUK-ICGC Prostate Cancer Group, Charles Massie, Harveer Dev. Widespread methylation convergence in clonally distinct foci of multifocal prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90717932598b7c081fea25f61e12e100d9757059" target='_blank'>
              Abstract B003: Widespread methylation convergence in clonally distinct foci of multifocal prostate cancer
              </a>
            </td>
          <td>
            Tamsin J. Robb, Melissa Cheung, Rajbir Batra, Henson Lee Yu, John C. Thomas, Anne Y Warren, Andrew Lynch, Daniel Brewer, D. Wedge, Charles Massie, H. Dev
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Spatial omics technologies have transformed biomedical research by enabling high-resolution molecular profiling while preserving the native tissue architecture. These advances provide unprecedented insights into tissue structure and function. However, the high cost and time-intensive nature of spatial omics experiments necessitate careful experimental design, particularly in selecting regions of interest (ROIs) from large tissue sections. Currently, ROI selection is performed manually, which introduces subjectivity, inconsistency and a lack of reproducibility. Previous studies have shown strong correlations between spatial molecular patterns and histological features, suggesting that readily available and cost-effective histology images can be leveraged to guide spatial omics experiments. Here we present Smart Spatial omics (S2-omics), an end-to-end workflow that automatically selects ROIs from histology images with the goal of maximizing molecular information content in the ROIs. Through comprehensive evaluations across multiple spatial omics platforms and tissue types, we demonstrate that S2-omics enables systematic and reproducible ROI selection and enhances the robustness and impact of downstream biological discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faf506dbe520bf8961e03a7037d33ea836e35e91" target='_blank'>
              Smart spatial omics (S2-omics) optimizes region of interest selection to capture molecular heterogeneity in diverse tissues
              </a>
            </td>
          <td>
            Musu Yuan, Kaitian Jin, Hanying Yan, Amelia Schroeder, Chunyu Luo, Sicong Yao, Bernhard Dumoulin, Jonathan Levinsohn, Tianhao Luo, Jean R. Clemenceau, Inyeop Jang, Minji Kim, Yunhe Liu, Minghua Deng, E. E. Furth, Parker Wilson, Anupma Nayak, Idania Lubo, L. S. Solis Soto, Linghua Wang, Jeong Hwan Park, K. Suszták, Tae Hyun Hwang, Mingyao Li
          </td>
          <td>2025-11-26</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>64</td>
        </tr>

        <tr id="Recent advances in spatial molecular imaging technologies have enabled gene expression profiling alongside high-resolution imaging, providing unprece dented opportunities to resolve molecular heterogeneity at subcellular resolution. However, these technologies fail to fully capture cellular characteristics due to the limited number of genes they can detect, which hinderdownstream analysis. Spatial imaging data provide high-resolution and fine-grained morphology information, developing computational methods that effectively integrate image features with transcriptomic profiles is crucial for enabling comprehensive subcellular data analysis. In this study, we present SIMMT, an image-enhanced multi-modal contrastivetrans former framework for identifying spatial domains and en hancing subcellular data. In the framework, we design a dual transformer architecture to learn multi-modal representations for cells by modeling transcriptomics and morphological images respectively. To fully capture modality interactions within spatial contexts, we introduce a contrastive learning module that enhances cell representation by aligning tissue morphology and gene expression at the cell level. We tested SIMMT on subcellular spatial transcriptomics datasets from human lung cancer tissue, mouse brain tissue, human colorectal cancer tissue, and human ovarian cancer tissue. The results demonstrated that SIMMT consistently outperformed state-of-the-art methods in spatial clustering and gene expression pattern analysis. Our method also effectively demonstrated its ability to identify tumor spatial heterogeneity and uncover potential gene biomarkers in the human bronchiolar adenoma (BA) dataset. The code and dataset of SIMMT can be downloaded from https://github.com/LWanzi/SIMMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c3848a8a10be8ec3a9138c712dc99053df49cc" target='_blank'>
              Image-enhanced Multi-Modal Contrastive Transformer for subcellular spatial transcriptomics.
              </a>
            </td>
          <td>
            Wanwan Shi, Ying Liu, Qiu Xiao, Yuting Bai, Xiao Liang, Xinling Zeng, C. Kwoh, Jiawei Luo
          </td>
          <td>2025-11-17</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Tumor-Associated Macrophages play a significant role in the tumor microenvironment of breast cancer, influencing its growth and progression. Their presence is associated with key processes such as angiogenesis and immunomodulation, impacting the immune system response and tumor aggressiveness. This study employed various methods to analyze scRNA-seq and spatial transcriptome data. The scRNA-seq data underwent thorough processing and analysis, with a focus on breast cancer subtypes. The CellChat R package was employed to analyze and visualize cell-cell communication networks within the scRNA-seq data. hdWGCNA, conducted in R, involved multiple steps such as data preprocessing, gene network construction, module identification, module preservation analysis, and functional enrichment analysis. To analyze the pseudotime dynamics of endothelial cells, Monocle3 was utilized. The spatial data for normal skin and leprosy were obtained and analyzed using the SPATA2 and semla R packages, facilitating the integration of spatial and single-cell data. The study utilized scRNA-seq and spatial transcriptome data to comprehensively analyze 40 breast cancer samples, including ER+, HER2+, and TNBC subtypes, alongside normal mammary tissue. Distinct cellular clusters were identified, and myeloid cells were characterized into four subtypes. Comparative analyses across breast cancer subtypes highlighted variations in the expression of the CXCL10 gene, and hdWGCNA provided insights into molecular landscapes. Trajectory inference analysis revealed pseudotime dynamics in myeloid cells, with specific gene expression changes. Spatial transcriptome profiling in TNBC and HER2+ breast cancer showed unique cellular clusters and gene expression patterns within the tumor microenvironment. Spatial and single-cell data integration emphasized the distribution of cell types and spatial enrichment of gene expression signatures.  TAMs emerge as key players engaging in multifaceted interactions with neighboring cell types within the intricate context of breast cancer. These identified genes, particularly CXCL10, assume central roles in regulating various facets of myeloid cell physiology and modulating immune responses. CXCL10 expression increases in TNBC macrophages suggest a potential association with T lymphocyte activity and myeloid cell chemotaxis, implicating its significance in the intricate immune response dynamics within the TNBC microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/666c93cdc1f48635e7eb409847dc8e7a85718b81" target='_blank'>
              Spatial profiling of CXCL10 in breast cancer: tumor-associated macrophages' mediation of immune response
              </a>
            </td>
          <td>
            Natália Cordeiro de Albuquerque D’Alva, Ana Kelly da Silva Fernandes Duarte, Leonardo Vinícius da Silva Lima, Eduardo Paulo da Silva Lima, Caio Cesar Fernandes Nobre Porto, Maria Nathália de Menezes, Luis Filipe Pereira Rocha, Leonardo Prudêncio Coutinho de Almeida, Rafael de Oliveira Calaça Farias, Leonardo Barbosa da Costa, Salomão Belfort Sparapan de Melo, Raimundo Rodrigues de França Júnior, Rodolfo Tibério Ferreira Silva, Alberson Maylson Ramos DA Silva, Taina Santos Bezerra
          </td>
          <td>2025-12-04</td>
          <td>CONTRIBUCIONES A LAS CIENCIAS SOCIALES</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Bone metastases are a common and devastating complication of advanced prostate cancer (PCa), with up to 80–90% of men with castration-resistant PCa (CRPC) developing metastases to the bone. The development of bone metastasis serves as a critical indicator of worse long-term clinical outcomes. Understanding the factors driving PCa metastasis to the bone, including through myeloid cell reprogramming within the tumor microenvironment (TME), is critical. Our study examined published single-cell RNA sequencing data from 7 PCa patients with bone metastasis and 7 benign donors, focusing on myeloid cells across tumor, involved marrow, distal marrow, and benign bone factions. We used manual annotation methods and scANVI for whole integrated data annotation. Myeloid cells were sub-classed and manually annotated for classical monocytes (Mono1), intermediate monocytes (Mono2), non-classical monocytes (Mono3), myeloid dendritic cells (mDCs), tumor-infiltrating monocytes (TIMs), tumor-associated macrophages (TAMs), and osteoclasts. We applied pseudotime and used CellRank's Random-Walk Simulation for trajectory inference to determine final terminal states for the myeloid cells, i.e., tumor-associated macrophages (TAMs), myeloid dendritic cells (mDCs), and osteoclasts. Ranked driver gene lists for every sample type (benign, distal, involved, tumor) and cell type (Mono1/2/3, TIM) that were associated with the final terminal states were generated. QIAGEN Ingenuity Pathway Analysis (IPA) and Gene Set Enrichment Analysis (GSEA) were performed on driver gene lists across sample types and cell types, thus identifying upstream regulators influencing terminal state formation. IPA and GSEA identified ATF3, a stress-response regulator, and NF-κB, a central inflammatory mediator, to be significantly associated with TAM differentiation driver genes and were shared across extremes of sample types (benign & tumor) and cell types (Mono1 & TIM). CellOracle's in-silico perturbations were performed to assess the significance of ATF3 in context of the PCa bone TME while visualizing impacts on cell development. Through siRNA suppression of ATF3 in-vitro, followed by flow cytometry and qPCR for knockdown validation, our results show reduced M2/TAM differentiation with decreased associated markers, suggesting a role for ATF3 in myeloid reprogramming that may contribute to TAM-associated immunosuppression and the metastatic disease state



 Azimullah Rifai, Safiullah Rifai, Zumar Meher, Tausif Khan, Wei Guang, Linbo Wang, Mohamad Khan, Arif Hussain. Rewiring the tumor microenvironment: Uncovering lineage drivers of immunosuppressive tumor-associated macrophages in prostate cancer bone metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b3000b262dc2818edcc8efcb35668c9389013e" target='_blank'>
              Abstract A019: Rewiring the tumor microenvironment: Uncovering lineage drivers of immunosuppressive tumor-associated macrophages in prostate cancer bone metastasis
              </a>
            </td>
          <td>
            Azimullah Rifai, Safiullah Rifai, Zumar Meher, Tausif Khan, Wei Guang, Linbo Wang, Mohamad Khan, Arif Hussain
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are increasingly recognized as critical contributors to the limited efficacy of immunotherapy, particularly through the establishment of immune-excluded tumor microenvironments. However, the functional heterogeneity and spatial ecological roles of CAFs in melanoma remain poorly characterized. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq data to systematically dissect CAF molecular programs in melanoma using weighted gene co-expression network analysis (WGCNA). Spatial distribution, CellChat, and ligand–receptor interaction analyses were applied to construct CAF-centered immune exclusion networks. A functional scoring model (riskScore) based on the CAF-M1 module was developed to evaluate clinical relevance. CAF was spatially co-localized with endothelial cells, and engaged in strong crosstalk with multiple immune cell types via pathways such as COLLAGEN–integrin and THBS1–CD47. The derived riskScore model demonstrated robust prognostic value across TCGA-SKCM cohort. Through spatial aggregation and multifaceted immune signaling, CAF constructs a multilayered immune-exclusion network in melanoma, linking stromal remodeling to immune evasion. These findings offer novel insights into CAF-driven immune resistance and may inform future stratified immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca83751e1c8ec5203b2f4b9c36b4cadfbfd1df19" target='_blank'>
              Exploration of the roles of CAFs in melanoma based on single-cell transcriptomics and spatial transcriptomics
              </a>
            </td>
          <td>
            Sheng Hong, Yuhan Zhao, Anni Hu, Yizhong Zhang, Yuming Li, Yan Rong, Zukai Li, Wenzhuo Su, Lude Zhu
          </td>
          <td>2025-11-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinomas (cSCCs) arise from keratinocytes in the skin, but the molecular changes driving this transformation remain unclear. To better understand this process, we perform multi-omic profiling of keratinocytes, actinic keratoses, and cSCCs. Single-cell mutational analyses reveal that most keratinocytes have remarkably low mutation burdens; however, keratinocytes with TP53 or NOTCH1 mutations exhibit substantially higher burdens. These findings suggest that keratinocytes can withstand high dosages of cumulative ultraviolet radiation, but certain pathogenic mutations break these adaptive mechanisms, inducing a mutator phenotype. Mutational profiling of cSCCs adjacent to actinic keratoses reveals TERT promoter and CDKN2A mutations emerge in actinic keratoses, whereas additional mutations that inactivate ARID2 and activate the mitogen-activated protein kinase pathway delineate the transition to cSCC. Surprisingly, actinic keratoses are often not related to their neighboring cSCC. Spatial analyses reveal gene expression heterogeneity, including checkpoint molecule enrichment at invasive fronts, highlighting tumor and immune cell interactions. The evolution of cutaneous squamous cell carcinoma (cSCC) remains poorly understood. Here, the authors employ multi-omics and multi-scale analyses to explore the genetic evolution of keratinocytes to cSCC, finding key pathogenic mutations that break the resistance to ultraviolet radiation as well as spatial heterogeneity patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ace63c32efba22479251c352c77035a045dd5b73" target='_blank'>
              Genetic evolution of keratinocytes to cutaneous squamous cell carcinoma
              </a>
            </td>
          <td>
            Bishal Tandukar, Delahny Deivendran, Limin Chen, A. K. Bandari, Noel Cruz-Pacheco, H. Sharma, Meng Wang, Albert Xu, Daniel B. Chen, Christopher D. George, Annika L. Marty, Raymond J. Cho, Jeffrey B. Cheng, Drew K Saylor, P. Gerami, I. Yeh, Sarah T. Arron, Boris C. Bastian, A. H. Shain
          </td>
          <td>2025-11-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Solid tumors like lung cancer show significant mutational heterogeneity. A biopsy captures only focal aspects, limiting conclusions about overall tumor biology. This prospective study correlated CT-based radiomics features with genomic profiles to optimize biopsy site selection. Lung cancer patients underwent CT imaging, radiomics analysis, targeted biopsies, and whole-exome sequencing. Twelve non-redundant features were extracted, with JointEntropy guiding biopsy targeting. In 7 of 12 patients, over 10% of mutations were exclusive to one biopsy. Clonal reconstruction showed heterogeneous profiles with over two subclonal processes in 67% of cases. Unsupervised clustering of radiomics features revealed two distinct groups separated by entropy features, of which the entropy-rich cluster was associated with STK11 mutations. Our study demonstrates that integrating radiomics with localized genomic analysis enhances the understanding of tumoral heterogeneity and may improve the targeting of advanced tumor regions for diagnostic sampling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26c7a7dc0c38ff91f3e1c4cecb5414647d6488b0" target='_blank'>
              Targeting tumoral heterogeneity in lung cancer: a novel, CT-texture-guided targeted biopsy approach with exome sequencing
              </a>
            </td>
          <td>
            Alexander Hertel, A. Streuer, Steffen Diehl, Tobias Boch, Dominik Nörenberg, Anika Strittmatter, Frank G. Zöllner, Stefan O. Schoenberg, W. Hofmann, Sonja Loges, Daniel Nowak, Matthias F. Froelich
          </td>
          <td>2025-11-07</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most malignant and highly recurrent brain tumor. Although over half of the GBM patients are elderly patients, the understanding of how aging affects GBM progression remains limited. Clinical and genomic variation data of GBM patients from TCGA and CGGA databases were used for prognostic analysis. We collected single-cell transcriptome data of 88,908 cells from 13 primary GBM (pGBM) and 12 recurrent GBM (rGBM) patients. Age-related immune cells were identified through cell–cell communication and trajectory analysis. The results were validated by projecting the single-cell transcriptome profiles onto bulk data. Finally, we experimentally validated the results on syngeneic orthotopic models of younger and older mice. Prognostic analysis indicated the effect of age in pGBM patients is stronger than that in rGBM patients. Moreover, the mutational signatures in pGBM does not affect prognosis. Single-cell RNA sequencing analysis revealed age-related differences in immune cells between pGBM and rGBM, and identified microglia underwent significant cell state changes with aging only in pGBM patients. Next, we validated that high expression of HSPB1 in microglia from older pGBM patients is associated with poor prognosis. Finally, the syngeneic orthotopic model for aged mice exhibited more tumor invasion, with a shorter median survival time. Furthermore, the microglia within the tumor microenvironment (TME) of aged mice showed markedly high expression level of HSPB1. Our study highlights the crucial role of microglial aging in pGBM, reveals distinct age-related changes of immune cells in the TME between pGBM and rGBM, and offers valuable insights into clinical treatment strategies targeting elderly GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9790086781b49dfb66d77c2a7a61b8f32ab0b08" target='_blank'>
              Integrative analysis of single-cell and bulk transcriptome data reveals age-related immune cell alterations in primary glioblastoma associated with prognosis
              </a>
            </td>
          <td>
            Zefan Jing, Bojun Qiu, Chenyang Ai, Chunhui Wang, Xinrun Wang, Boyang Li, Lin Hou, Bin Yin, Wei Han, Xiaozhong Peng
          </td>
          <td>2025-11-12</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fb5388d0bd0362161f91d2111550a5aa12010f" target='_blank'>
              Inference of clonal hematopoiesis using the collective behaviour of DNA methylation states
              </a>
            </td>
          <td>
            Samuel. J. C. Crofts, Caleb M. Grenko, Neil A. Robertson, Kristina Kirschner, Tamir Chandra
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84d2ad509c801e8dff7695be2327a46d057fed6b" target='_blank'>
              CCR5 expression defines the functional heterogeneity of tumor-infiltrating CD8+ T cells and predicts enhanced antitumor immunity and favorable prognosis in non-small cell lung cancer.
              </a>
            </td>
          <td>
            Yilong Li, Kaiyuan Zhang, Lian Wang, Weiping Yao, Wenjing Fei, Keyu Cheng, Yan Li, Guichun Huang
          </td>
          <td>2025-12-23</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a90c7d87d145743b8f163f5d40e3c8d84fc50c6" target='_blank'>
              Single-cell phylodynamics reveal rapid late-stage colorectal cancer expansions
              </a>
            </td>
          <td>
            J. M. Alves, Kylie Chen, Sonia Prado-López, N. Estévez-Gómez, P. Alvariño, Juana Alonso, Monica V. Valecha, Laura Tomás, A. Zwaans, Ugnė Stolz, Tanja Stadler, Débora Chantada, J. Cameselle-Teijeiro, Alexei Drummond, D. Posada
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Colorectal cancer (CRC) often occurs in combination with sarcopenia (SP), which affects the prognosis of CRC, but the histopathological mechanisms underlying this process are unclear. Single-cell transcriptome technology provides a tool for resolving multi-organ pathological interactions, and previous studies have lacked direct molecular evidence of a trans-tissue microenvironment. To address this, we applied an integrated analytical approach. The integration of single-cell transcriptome data from 16 CRC tumor tissues (scRNA-seq) and 17 SP muscle tissues (snRNA-seq) was used to systematically resolve heterogeneity across tissue microenvironments using Harmony to correct for batch effects and combining cellular annotations, differential expression analyses, and CellChat communication modeling. Our analysis revealed that CRC and SP share 3 core cell types, namely, endothelial cells, fibroblasts and monocytes, but they show distinct transcriptional and functional differentiation in different tissues. CRC cells are enriched in the inflammatory, angiogenic and matrix remodeling pathways, accompanied by the upregulation of hypoxia-associated genes, whereas SP is biased toward muscle structural maintenance, metabolic regulation and repair functions. The MIF-CD74, COLLAGEN and THBS-CD47 pathways were active in CRC, whereas SDC4 and CD36-related signals were dominant in SP. VWF, PDGFRA and FCN1 were stably expressed in both groups, suggesting the existence of a conserved homeostatic regulatory network. In conclusion, this study reveals the cross-tissue functional heterogeneity of cell types shared by CRC and SP, suggests a set of potential pervasive regulators, and provides new perspectives for understanding the systemic effects of tumors and the microenvironmental mechanisms of sarcopenia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28aac551ab88d08b24ed6a70bb80afdcde46ff4f" target='_blank'>
              Single-cell transcriptome analysis elucidates the microenvironmental interactions between colorectal cancer and sarcopenia
              </a>
            </td>
          <td>
            Tihong Qiu, Xiaoying Wang, Jie Li, Yiping Guan, Hongyou Wen
          </td>
          <td>2025-11-28</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related deaths worldwide. Existing therapeutic options have limited efficacy, particularly for lung squamous cell carcinoma (LUSC), underscoring the critical need for the identification of new therapeutic targets. We previously demonstrated that the Transmembrane Serine Protease TMPRSS11B promotes the transformation of human bronchial epithelial cells and enhances lactate export from LUSC cells. Here, we evaluate the impact of TMPRSS11B activity on the host immune system and the tumor microenvironment (TME). Tmprss11b depletion significantly reduces tumor burden in immunocompetent mice and triggers an infiltration of immune cells. RNA FISH analysis and spatial transcriptomics in the autochthonous Rosa26-Sox2-Ires-GfpLSL/LSL; Nkx2-1fl/fl; Lkb1fl/fl (SNL) model reveal an enrichment of Tmprss11b expression in LUSC tumors, specifically in Krt13+ hillock-like cells. Furthermore, utilizing ultra-pH-sensitive nanoparticle imaging and metabolite analysis, we identify regions of acidification, elevated lactate, and enrichment of immunosuppressive (M2-like) macrophages in LUSC tumors. These results demonstrate that TMPRSS11B promotes an acidified and immunosuppressive TME and nominate this enzyme as a therapeutic target in LUSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edde3e6259df0a2e4e7c324d76c028ecfb0de34e" target='_blank'>
              TMPRSS11B promotes an acidified microenvironment and immune suppression in squamous lung cancer
              </a>
            </td>
          <td>
            Hari Shankar Sunil, Jean R. Clemenceau, Anthony Grichuk, Isabel Barnfather, S. R. Nakkireddy, Luke Izzo, Q. Feng, William Hartnett, Bret M. Evers, Lisa Thomas, I. Subramaniyan, Li Li, William C Putnam, Steven Hepensteil, Jingfei Zhu, Barrett L. Updegraff, John D. Minna, R. DeBerardinis, Tae Hyun Hwang, Jinming Gao, T. Oliver, Kathryn A. O’Donnell
          </td>
          <td>2025-11-10</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95f4955996275b1364c480a5de0b28f26332655d" target='_blank'>
              Structural basis for constitutive activation of oncogenic gp130 mutants in human inflammatory hepatocellular adenomas
              </a>
            </td>
          <td>
            Yi Zhou, Jing Cao, T. Ramlall, Luke L. McGoldrick, Jennifer Jones, Annan S. I. Cook, Raymond Leidich, Mark W. Sleeman, William C. Olson, P. Stevis, Matthew C. Franklin
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) play crucial and important role in cancer dynamics by affecting homeostasis, immunosuppression, and angiogenesis within the tumor microenvironment. Using single-cell transcriptomics, we constructed a comprehensive atlas of myeloid cell populations across healthy and pan-cancer tissues that revealed heterogeneity. Our analysis suggested that TAMs may arise from two distinct origins: C1QC+ TAMs, which likely are derived from resident tissue macrophages, SPP1+ TAMs and ISG15+ TAMs, which appear to originate from circulating monocytes. Regarding immature myeloid-derived suppressor cells (MDSCs), we highlighted THBS1+ MDSCs and their descendants, SPP1+ TAMs, as key contributors to tumor progression, immunosuppression, and angiogenesis. We proposed a dichotomous model for TAMs, in which C1QC+ TAMs are associated with better patient outcomes, whereas the THBS1+ MDSCs - SPP1+ TAMs lineage correlates with poorer survival and unfavorable response to immunotherapy. This study offers insight into the complex interactions among monocyte-macrophage subtypes and sheds light on TAM heterogeneity and its implications for cancer progression and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a07f8d41863dfbb5389df0779f0c2e6e40c36fc" target='_blank'>
              Pan-cancer single-cell RNA sequencing analysis refines multi-origin monocyte and macrophage lineages.
              </a>
            </td>
          <td>
            Truc Do Thanh Nguyen, Andrew J Lee, Hyun Jung Park, Nameeta Shah, Bayrta Mandzhieva, Dong-Sup Lee, Inkyung Jung, Woong-Yang Park
          </td>
          <td>2025-11-13</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The incidence of early-onset colorectal cancer (CRC), defined as cases diagnosed in individuals under 50, is rising globally. However, its molecular and immune characteristics remain poorly understood. In this study, we analyze single-cell RNA sequencing data from 168 CRC patients, aged 22 to 91, to investigate differences between early-onset and standard-onset CRC. We find a reduced proportion of tumor-infiltrating myeloid cells, a higher burden of copy number variations, and decreased tumor-immune interactions in early-onset CRC. Additionally, immune signatures unique to early-onset CRC are associated with differential responses to immunotherapy, underscoring the need for tailored therapeutic strategies for this group of patients. These findings provide valuable insights into the molecular and immune landscape of early-onset CRC, emphasizing the importance of developing targeted prevention and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da549e2629038aa4c9de27bbf167df537f9f53c" target='_blank'>
              Single-cell integration analysis reveals molecular signatures of the tumor microenvironment in early-onset colorectal cancer
              </a>
            </td>
          <td>
            Ting Peng, Qimin Zhan
          </td>
          <td>2025-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Introduction The analysis of tumor-infiltrating lymphocytes often requires techniques that expand their numbers, potentially introducing bias. To address this, we performed a detailed analysis of minimally cultured TILs to evaluate whether this approach better preserves their characteristics. Methods The TIL culture method was based solely on tumor tissue with low IL-2 supplementation to minimize artificial alterations. The validity of this approach was confirmed by the correlation between CD3+ T cell percentages in cultures and infiltration patterns observed by immunohistochemistry. Immunophenotyping, cytokine release, and TCR repertoire analysis were used to characterize CD4+ and CD8+ T cell subsets and their molecular features during minimal expansions. Results High TIL infiltration areas did not consistently correspond to an increased presence of any T cell subset; both CD4+ and CD8+ T cells frequently coexisted in these regions. In contrast, low TIL infiltration sections often displayed a higher proportion of CD4+ T cells. An inverse correlation between CD4+ T cell percentages and cytotoxic molecules was observed, indicating reduced cytotoxic activity in low-TIL sections with abundant CD4+ T cells. TCR repertoire analysis revealed differences between T cell subsets: CD4+ T cells were associated with longer TRA CDR3 nt and shorter TRB N(D)N nt lengths, along with lower diversity, while CD8+ T cells did not exhibit significant correlation with any TCR feature. Discussion This study highlights the distinct biological features of CD4⁺ and CD8⁺ TIL populations within the tumor microenvironment that can be preserved using a minimally expanded TIL approach. The observed associations between IHC patterns, T cell subset composition, cytotoxic potential, and TCR repertoire diversity help identify which biopsy regions yield TILs with greater therapeutic potential, thus providing guidance for TIL selection in immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c988d227329c53bf2d2653e81969a85f3a08fee" target='_blank'>
              Minimally expanded breast cancer tumor-infiltrating-lymphocytes provide guidance for therapeutic selection
              </a>
            </td>
          <td>
            A. Aran, Gonzalo Lázaro, Vicente Marco, V. Peg, Maitane Faus, L. Garrigós, J. Pérez-García, Javier Cortés, Mercè Martí
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11c95fbbadbecbdf667b52410258455d109e138e" target='_blank'>
              A comparative review of single-cell atlases: mapping cellular diversity across species and tissues.
              </a>
            </td>
          <td>
            Keyun Look, Pei En Goh, Long Chiau Ming, Shi Hui Law, Sang Loon Tan, Ee Wern Tan, Hooi‐Leng Ser, Bey-Hing Goh
          </td>
          <td>2025-12-02</td>
          <td>Cellular and molecular life sciences : CMLS</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099b7cbfc744841c3702cbfd3025a09c4f8cf0c1" target='_blank'>
              Targeting the HMGB1-IL32 pathway to alleviate T cell exhaustion in epithelial ovarian cancer.
              </a>
            </td>
          <td>
            Ankita Murmu, Balázs Győrffy
          </td>
          <td>2025-11-15</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa756c05989530f16b2f4c4778c0bfc833ec875f" target='_blank'>
              Deep Learning links TP53 genotype to expression-defined transcriptional program in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            M. Tsesmelis, Rebecca Andersson, Alba Ferrer Perez, Sara Montserrat Vazquez, M. Carolina Florian, M. Mulaw
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) progression is shaped by the tumor microenvironment, particularly tumor-associated macrophages (TAMs), which often adopt immunosuppressive functions. CD300e, a myeloid receptor involved in immune regulation, has an uncharacterized role in CRC. Methods Functional studies were conducted in azoxymethane/dextran sodium sulfate and MC38 murine CRC models using CD300e systemic and myeloid-specific CD300e knockout mice, and adoptive transfer experiments assessed macrophage-intrinsic effects. Human studies included analysis of CD300e expression in matched tumor and normal tissue from patients with CRC and in vitro co-culture of patient-derived colon tumor organoids with monocytes to study CD300e induction and TAM polarization. Results In vivo, CD300e deficiency led to reduced tumor burden, enhanced major histocompatibility complex expression on TAMs, and improved T-cell responses. CD300e-deficient macrophages exhibited increased phagocytic activity, antigen presentation, and support for T-cell proliferation and cytotoxicity. Adoptive transfer confirmed that macrophage-intrinsic CD300e expression is sufficient to suppress T-cell function and promote tumor growth. In patients with CRC, CD300e is selectively upregulated in tumor-infiltrating monocytes and macrophages, driving a suppressive phenotype marked by impaired antigen presentation. Tumor-derived signals in vitro induce CD300e expression and promote a protumorigenic macrophage profile. Conclusions Our findings identify CD300e as a critical regulator of macrophage-mediated immune suppression in CRC and a potential target for reprogramming TAMs to enhance immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c56c500ab6751223d28c46c0f7327dc521a283" target='_blank'>
              CD300e is a driver of the immunosuppressive tumor microenvironment and colorectal cancer progression via macrophage reprogramming
              </a>
            </td>
          <td>
            Annica Barizza, Stefania Vassallo, L. Masatti, Mattia Laffranchi, Sofia Giacometti, S. Lonardi, Mattia Bugatti, Sara Coletta, C. Della Bella, M. D’Elios, Simone Pizzini, A. Rosato, W. Vermi, Matteo Fassan, G. Spolverato, Silvano Sozzani, E. Calura, R. Sommaggio, Gaia Codolo
          </td>
          <td>2025-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Cellular senescence (CS) plays an important role in cancer development and treatment. However, the heterogeneity of CS among different types of cancer and its impacts on patient prognosis and therapy response remain to be fully elucidated. This study performed a comprehensive pan-cancer analysis on the CS landscape of 33 cancer types and 29 normal tissues. The molecular subgroups of CS were identified based on the expression of CS-related genes. Multi-platform data including prognosis, microbiota, immune microenvironment, multi-omics, and drug sensitivity were used to investigate the associations with the CS subgroups. Additionally, 12 single-cell datasets and 19 immunotherapy cohorts were collected to evaluate the CS subgroups. We characterized five pan-cancer CS subgroups with distinct biological features, named Inflamm-aging, DNA Damage Response, Autophagy, Immunologically Quiet, and Metabolic Disorder. The CS subgroups showed cancer-type and tissue-type specific distribution, and revealed significant associations with cancer prognosis, intratumoral microbiota, immunophenotypic features, and multi-omic alterations. The immunophenotypic features of the CS subgroups were verified by immunohistochemistry staining in tissue microarrays. Single-cell analyses indicated the CS heterogeneity among cancer types at the single-cell level. Furthermore, we developed a machine-learning model integrating CS-related cancer driver genes to infer the CS subgroups, and verified its prediction capability for immunotherapy response and prognosis in independent cohorts. Finally, potential therapeutic agents and targets were identified in the CS subgroups, which might have therapeutic implications for patients. Overall, this study provided a CS-derived classification scheme to expand the existing understanding of CS heterogeneity. The CS subgroups can reveal distinct CS states among cancer types and may contribute to improving personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994dcd388a6e6c3028419b6ef9776d711158f488" target='_blank'>
              Pan-cancer landscape of cellular senescence heterogeneity reveals associations with prognosis and therapy response
              </a>
            </td>
          <td>
            Linyan Chen, Tongshu Yang, Zhiyuan Zhang, Yuqi Yang, Sidian Long, Manhua Wang, Hao Zeng
          </td>
          <td>2025-11-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Despite the emergence of new therapies such as, immunotherapy, the treatment and diagnosis of Hepatocellular Carcinoma (HCC) still face many challenges, and the therapeutic outcomes for patients remain less than satisfactory. The study identified differentially expressed genes in HCC through differential analysis and then used univariate cox regression analysis to identify genes associated with prognosis. The intersection of these two sets of genes was used to obtain differentially expressed prognostic genes in HCC, which were then subjected to enrichment analysis. We analyzed two single-cell RNA sequencing (scRNA-seq) datasets from HCC patients, comprising 24,637 cells. Non-negative Matrix Factorization (NMF) clustering was used to identify cell cycle regulation in HCC tumor microenvironment (TME) cells, including three cell subpopulations: proliferating cells (PC), dendritic cells (DC), and macrophages (MAC). We employed the CellChat package to analyze cell–cell communication, the Monocle package for pseudotime trajectory analysis, and the SCENIC software package to study gene regulatory networks. Survival analysis was also performed using cell cycle-related features. A total of 26 clusters, including 15 major cell types, were identified in the HCC samples. Complex cell–cell communication networks were observed among these cell types. Enrichment analysis revealed that these cells were mainly enriched in pathways related to the cell cycle. The expression of cell cycle-related genes was elevated in tumor samples, and changes in cell cycle-related genes in specific subtypes were associated with different overall survival rates. The study focused on single-cell level data analysis of the cell cycle. The bubble plot results showed that the cell cycle scores were significantly upregulated in the PC, DC, and MAC subpopulations. Further subtyping revealed that these subtypes exhibited distinct biological states, cell–cell communication, and metabolic pathways. This study demonstrates that cell cycle regulation and cell–cell communication within the HCC tumor microenvironment impact tumor progression and patient prognosis. Cell cycle dysregulation in TME cells correlates with poor prognosis and immunotherapy efficacy, suggesting cell cycle targeting as a therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc3531834465d59ab5f37ade029201ae0f13d144" target='_blank'>
              Impact of single-cell cell cycle regulation of intercellular communication on the prognosis of hepatocellular carcinoma in the tumor microenvironment
              </a>
            </td>
          <td>
            Cong Hu, Rui Deng, Shuxiong Nong, Xinglang Mou
          </td>
          <td>2025-11-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly lethal malignancy with poor prognosis due to its resistance to conventional chemotherapy and limited number of targetable mutations. The recurrent loss-of-function mutation of the tumor suppressor ARID1A represents a key vulnerability potentially exploitable via synthetic lethality. To advance research in this area, we successfully established and characterized two novel CCA cell lines, SiSP-K01 and SiSP-K05, derived from moderately differentiated intrahepatic CCA. Both models were confirmed to carry ARID1A heterozygous loss and complete protein depletion, authentically reproducing the target genotype. Comprehensive molecular and phenotypic profiling (including karyotype, growth kinetics, 100% in vivo tumorigenesis, and tumor marker expression) confirmed their fidelity to the patient tissue and CCA lineage. Importantly, exome and transcriptome analysis not only validated the cell lines but also revealed an associated derangement of the PI3K/AKT signaling pathway. Based on this molecular finding, we performed drug screening and demonstrated that these ARID1A-deficient CCA cell lines are hypersensitive to inhibition by both PI3K/AKT and PARP inhibitors. The SiSP-K01 and SiSP-K05 cell lines are therefore critical new preclinical models suitable for functional studies of ARID1A deficiency and the development of targeted therapies for CCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef0e9dfbc0a68397e20306d980742627a07920fa" target='_blank'>
              Establishment of novel cholangiocarcinoma cell lines with ARID1A deficiency and preclinical validation of synthetic lethality therapies
              </a>
            </td>
          <td>
            Sunisa Prasopporn, G. Sittithumcharee, Jantappapa Chanthercrob, Somchai Limsrichamrern, Arada Hirankitti, Pimkanya More-krong, Sakda Sathirareuangchai, A. Chaiboonchoe, Somponnat Sampattavanich, Seiji Okada, Siwanon Jirawatnotai
          </td>
          <td>2025-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Simple Summary Cancer remains a leading cause of death worldwide, often because tumors can evade the immune system and become resistant to treatments. In this study, we investigated a protein called HMGA1, which is known to be present in high amounts in many cancers but whose broader role was not fully understood. By analyzing data from thousands of patients across numerous cancer types, we discovered that HMGA1 is frequently overactive due to specific chemical changes on its DNA. High levels of this protein were linked to poorer patient survival. We found that HMGA1 helps tumors hide from the body’s immune defenses and makes them resistant to a common class of drugs known as AKT inhibitors. We confirmed this drug resistance in breast cancer cells. Our work identifies HMGA1 as a key driver of cancer aggression and suggests that measuring its levels could help predict patient outcomes and guide more effective treatment strategies, particularly for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3e03334f80f9abed34dd7f9721e333bfb33a958" target='_blank'>
              Promoter Hypomethylation Unleashes HMGA1 to Orchestrate Immune Evasion and Therapy Resistance Across Cancers
              </a>
            </td>
          <td>
            Iram Shahzadi, T. Ahsan, Shoaib Anwaar, Wajid Zaman, Houjun Xia
          </td>
          <td>2025-12-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Regulatory T cells (Tregs), defined by the lineage‐specific transcription factor FoxP3, are crucial for immune regulation and have been studied extensively in mammals. However, avian Tregs remain poorly characterized, leaving gaps in our understanding of their evolutionary conservation and unique features. In this study, we investigated the phenotype of chicken Tregs to define reliable markers for their identification and characterization. We analyzed CD4+ splenocytes sorted into CD25negative, CD25low, and CD25high subpopulations using RNA sequencing. FOXP3 and other Treg‐associated genes were expressed in both CD25low and CD25high populations, showing that CD25 expression alone is insufficient to distinguish chicken Tregs. To refine the marker profile, we evaluated additional markers, including CTLA‐4 and GITR. Notably, we describe for the first time a chicken‐specific CTLA‐4 antibody, which uniquely stains CTLA‐4 exclusively in intracellular (ic) compartments, distinguishing it from mammalian counterparts. Single‐cell RNA sequencing further confirmed distinct FOXP3+ clusters enriched for expression of CTLA4 and TNFRSF18 (encoding GITR). While CTLA‐4's ic expression limits usability in functional assays, the combination of CD4+/CD25+/CTLA‐4+/GITR+ represents the most accurate characterization of putative chicken Tregs to date. These findings highlight evolutionary conservation and species‐specific differences in Treg markers, providing the foundation for future studies on chicken Treg functionality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ba9ce910cfe45c60bacadf709ff7e892561eafd" target='_blank'>
              Beyond FoxP3—Identification of a Chicken Regulatory T Cell Signature
              </a>
            </td>
          <td>
            Isabell Naumann, Edward S Ricemeyer, Daniel Elleder, Jiri Plachy, Dominik von la Roche, Kim Vučinić, Thomas W. Göbel, Bernd Kaspers, Simon P. Früh, S. Härtle
          </td>
          <td>2025-12-01</td>
          <td>European Journal of Immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="This study addresses the limited mechanistic understanding behind medical imaging for tumor microenvironment (TME) assessment. We developed a novel framework that analyzes tumor imaging heterogeneity index (TIHI)-correlated genes to uncover underlying TME biology and therapeutic vulnerabilities. DCE-MRI and mRNA data from 987 high-risk breast cancer patients in the I-SPY2 trial, together with mRNA data from 508 patients in GSE25066, were analyzed. TIHI-associated genes were identified via Pearson correlation, clustered via weighted gene co-expression network analysis (WGCNA), and subgroups were defined via non-negative matrix factorization (NMF). The clinical relevance of the image-to-gene comprehensive (I2G-C) subtype defined by subgroups was assessed using logistic regression and Cox analysis. I2G-C comprised four clusters with distinct immune and replication/repair functions. It further stratified receptor, PAM50, and RPS5 subtypes. The “immune+/replication+” was more likely to achieve pathological complete response (pCR) (OR = 2.587, p < 0.001), while the “immune−/replication−” was the least likely to achieve pCR (OR = 0.402, p < 0.001). The “immune+/replication+” showed sensitivity to pembrolizumab (OR = 10.192, p < 0.001) and veliparib/carboplatin (OR = 5.184, p = 0.006), while “immune-/replication-” responded poorly to pembrolizumab (OR = 0.086, p < 0.001). Additionally, “immune+/replication-” had the best distant recurrence-free survival (DRFS), whereas “immune-/replication+” had the worst (log-rank p = 6 × 10−4, HR = 5.45). By linking imaging heterogeneity directly to molecular subtypes and therapeutic response, this framework provides a robust, non-invasive surrogate for genomic profiling and a strategic tool for personalized neoadjuvant therapy selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaac41c7947552aff84e1e324fe7769979665178" target='_blank'>
              Tumor Imaging Heterogeneity Index-Inspired Insights into the Unveiling Tumor Microenvironment of Breast Cancer
              </a>
            </td>
          <td>
            Qingpei Lai, Xinzhi Teng, Jiang Zhang, Xinyu Zhang, Yufeng Jiang, Yao Pu, Peixin Yu, Wen Li, Tian Li, J. Cai, Ge Ren
          </td>
          <td>2025-11-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Computational pathology enables the automatic tissue analysis of Whole Slide Images (WSIs), offering unmatched possibilities to capture quantitative tumor microenvironment (TME) characteristics that are essential for patients' prognosis and therapy response. The existing clinical and digital biomarkers do not encompass the morphometric features and spatial interactions between vascular networks and immunological compartment in the TME. To address this challenge, this work presents a high- throughput quantitative framework for automatic segmentation and assessment of aberrant phenotypes of blood vessels and immune cell clusters in hematoxylin & eosin-stained WSIs, to construct the Vascular-Immune Pathomic (VIPath) biomarker. For our study, we utilized three public datasets of Colon Adenocarcinoma (COAD) and Stomach Adenocarcinoma (STAD) from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) projects: TCGA-COAD, TCGA-STAD, and CPTAC-COAD. Additionally, we collected two in-house gastric cancer cohorts of 80 and 51 patients : DBGC-M0 and DBGC-M1. The VIPath biomarker was incorporated in a Cox proportional hazards model trained on the TCGA-COAD. Then, it was validated for predicting Overall Survival (OS) in TCGA-STAD, DBGC-M0, and DBGC-M1, Disease Free Survival in CPTAC-COAD and second-line therapy Progression-free Survival (PFS-2) in DBGC-M1. VIPath encompasses features from both vascular and immunological compartments interacting in the TME. Results proved that VIPath was capable to significantly stratify risk groups for OS TCGA-STAD (p=0.018), OS DBGC-M0 (p=0.029), OS DBGC-M1 (p=0.014), and PFS-2 DBGC-M1 (p$< $0.005). Furthermore, when inserted in a Cox model, it led to an improvement of C-index and R2 over all other considered prognostic factors, i.e., p-TNM, ECOG, MSI, HER2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f526aa6f7bc4b1f2859a4decf8cee5c58634b7e" target='_blank'>
              Automated Pathomic Analysis of Angiogenesis and Immune Profiles Unveils an Interpretable Prognostic Biomarker in Colon and Gastric Cancers.
              </a>
            </td>
          <td>
            Michela Prunella, Nicola Altini, Rosalba D'Alessandro, Annalisa Schirizzi, Giampiero De Leonardis, Graziana Arborea, M. Savino, Anna Maria Valentini, R. Armentano, Angela Dalia Ricci, Claudio Lotesoriere, Raffaele Carli, M. Dotoli, Gianluigi Giannelli, Vitoantonio Bevilacqua
          </td>
          <td>2025-11-24</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ee12cd01ff0dc670471af7bea5ea93e2f7413e0" target='_blank'>
              INSM1 governs a neuronal progenitor state that drives glioblastoma in a human stem cell model.
              </a>
            </td>
          <td>
            Patrick A. DeSouza, Matthew Ishahak, Xuan Qu, Colin R. McCornack, Devi Annamalai, Diane D. Mao, Rajanikanth Vangipurapu, Yiwei Fu, Alexandre T. Vessoni, Ryan Cleary, Rowland H Han, Punn Augsornworawat, Timothy Woodiwiss, Darby Agovino, Braxton Sizemore, Jessica Kline, Maryam Borhani-Haghighi, Hao Chen, Sangami Pugazenthi, Hiroko Yano, Ting Wang, Luis F Z Batista, J. Millman, Albert H. Kim
          </td>
          <td>2025-12-07</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a highly dynamic network shaped by immune, vascular, and stromal interactions, further modulated by extracellular factors. The high tumor heterogeneity complicates treatment due to evolving resistance. Biodynamic imaging (BDI), a 3D coherence-gated holography technique, quantifies intracellular motion to assess phenotypic responses to therapy. We applied BDI to prostate cancer (PCa) treated with IL-27-based combinations to evaluate the dynamic responses of immunotherapy. This study is the first to integrate BDI with RNA-seq to correlate physiological changes with gene expression. Using a subcutaneous PCa model, tumors were treated ex vivo with chemotherapy and immunomodulatory agents. BDI enabled capturing motion frequency shifts (0.01–12.5 Hz), while RNA-seq revealed the molecular changes in the tumors. IL-27 showed potential to modulate the immunologically “cold” TME and enhance therapeutic efficacy. This integrative approach offers a novel platform for evaluating combination strategies in PCa and may inform more effective treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1439e864183bad47fa0679ec246c930f10dfa35f" target='_blank'>
              Exploring tumor dynamics and responses of prostate cancer to IL-27 based treatment combinations through biodynamic imaging and RNA sequencing analyses
              </a>
            </td>
          <td>
            Shreya Kumar, Dawith Lim, Harish Kothandaraman, Nadia Atallah-Lanman, David Nolte, John Turek, Marxa L. Figueiredo
          </td>
          <td>2025-11-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cervical cancer is a prevalent malignancy among women, yet the involvement of neutrophils in its tumor microenvironment remains insufficiently explored. This study utilized single-cell RNA sequencing (scRNA-seq) to delineate neutrophil subsets and elucidate their roles in disease progression and prognosis. Analysis of 20 cervical biopsy samples across different disease stages identified five neutrophil subsets (N0-N4), among which the N4 subset exhibited a marked increase during disease advancement. Spatial transcriptomics and tissue microarray analyses revealed that N4 neutrophils are enriched in tumor regions and are associated with genes implicated in proliferation, metastasis, and immune evasion. Functional characterization demonstrated that N4 promotes tumor progression via activation of the Wnt signaling pathway and extracellular matrix remodeling. A neutrophil infiltration-based risk model was established and validated through multi-omics approaches, highlighting its potential in prognostic prediction. These findings underscore the pivotal role of N4 neutrophils in cervical cancer and provide valuable insights for the development of targeted immunotherapies and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9eca7bc06f3fa7d3102669c9c6d4eefb56b347" target='_blank'>
              Single-Cell and Multiomic Analysis Reveals Neutrophil Heterogeneity and Prognostic Value in Cervical Lesions.
              </a>
            </td>
          <td>
            Ze Wang, Zhuang Liang, Shiyi Liu, C. Cao, Shimin Chen, Nan Yu, Xiaoyuan Huang, Tao Zhang
          </td>
          <td>2025-12-12</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Extensive studies have demonstrated the relationship between metabolic reprogramming and the tumor microenvironment. Here, we characterized the head and neck squamous cell carcinoma (HNSCC) evolutionary landscape using spatial metabolomics/transcriptomics, single-cell transcriptomics, and bulk multi-omics. Metabolic heterogeneity during HNSCC malignant transformation was identified, with significant enrichment in the purine metabolism. Integrating single-cell and bulk data, we developed a robust ligand-receptor-based signature (LRS) linked to NT5E, a key upstream regulator of purine metabolism, which served as an independent prognostic indicator. The low LRS subtype was associated with a high proportion of immune cell infiltration and improved response to immunotherapy. Notably, in vitro and in vivo experiments demonstrated that AMIGO2, a core molecule within the LRS, regulates tumor-associated purine metabolism, and that its downregulation suppresses tumor cell invasion and migration, inhibits myofibroblast differentiation, and promotes immune effector cell infiltration. Moreover, combining AMIGO2 targeting with anti-PD-1 therapy yielded superior efficacy. Consistent validation was also obtained in a clinical cohort of HNSCC and premalignancy patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca668a38f0d67d558807d21d034edd4de991a2" target='_blank'>
              Metabolomic and transcriptomic profiling of HNSCC identifies AMIGO2 as a therapeutic target modulating tumor microenvironment
              </a>
            </td>
          <td>
            Gan Liu, Xinfeng Yao, Yuchen Hou, Wentao Deng, Liyu Zhang, Shutong Li, Panpan Huang, Wenjiao Chang, Haiyan Huang, Lan Shi, Qianqian Zhou, Yanqing Wang, Jia Chen, Wenchao Gu, Xiaoling Ma, Yu Zhou, Honghong Liu, Shanshan Zhang, Zongcheng Yang
          </td>
          <td>2025-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Objective This study aimed to investigate cell-type-specific RNA methylation patterns in primary breast cancers and metastatic lymph nodes and explore their roles in cancer progression and immune regulation. Methods Using single-cell RNA sequencing data from five breast cancer patients (primary cancers and matched metastatic lymph nodes), we calculated RNA methylation scores per cell using the AUCell method. Cell types were annotated using Seurat. Differentially expressed gene analysis, pseudotime trajectory assessment, copy number variation analysis, and cell–cell communication analyses were performed between high- and low-methylation groups. Validation was conducted using bulk RNA sequencing data from The Cancer Genome Atlas. Results Metastatic lymph nodes showed significantly lower RNA methylation scores than primary cancers, particularly in T cells and B cells. Low methylation was correlated with enhanced cell differentiation, increased malignancy in epithelial cells, and reduced immune cell communication. Monocytes exhibited opposite trends. Differentially expressed genes from methylation groups stratified The Cancer Genome Atlas patients into two clusters with distinct survival and immune infiltration profiles. Conclusions RNA methylation is suppressed during breast cancer metastasis, affecting cell differentiation, immune communication, and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3a57d08895a4d132dadd9681e650758bc323c10" target='_blank'>
              Single-cell analysis reveal cell-type-specific RNA methylation patterns in breast cancer
              </a>
            </td>
          <td>
            Shengtao Huang, Yichen Lin, Xiaobin Liu, Kun Zhang, Qiuju Han, Ruilei Liu, Yong Zhuang
          </td>
          <td>2025-12-01</td>
          <td>The Journal of International Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The incidence of prostate cancer continues to increase, making it the second most common malignant tumor among men worldwide. Immunotherapy has emerged as a key therapeutic strategy for treating tumors. Numerous studies have established that the efficacy of tumor immunotherapy is closely associated with the tumor microenvironment and T cell subsets. However, the specific functions of certain T cell subsets in prostate cancer remain incompletely characterized. Therefore, this study aimed to systematically investigate the distribution patterns of T cell subsets within the tumor microenvironment of prostate cancer patients and their correlations with clinicopathological parameters. Therefore, we investigated the impact of T cells on the tumor microenvironment of prostate cancer at the single-cell level. We employed a variety of analytical methods to reveal the functions of T cells, including cell interaction analysis, time-series analysis, enrichment analysis, immune infiltration analysis, and other analytical approaches. By integrating bulk RNA-seq data, we constructed and validated a prognostic risk model based on T cell marker genes. Finally, we utilized the ssGSEA and ESTIMATE algorithms to explore the relationship between the prognostic risk model and immunotherapy. After quality control, 16,999 cells from the single-cell data were retained for downstream analysis. Our study focused on T cells, revealing the communication between various cell types and T cells. Pseudotime analysis showed that different T cell marker genes exhibited differential expression at various time points, corresponding to distinct biological processes. Enrichment analysis indicated that T cell marker genes were enriched in several immune-related pathways. From our analysis, BCAS2, EIF2S2, RIOK3, and ATP6V1E1 were ultimately identified as prognostic markers. Immune infiltration analysis revealed that high-risk patients had lower immune scores, stromal scores, and ESTIMATE scores and greater tumor purity compared to low-risk patients. We analyzed the mechanisms involving T cells in prostate cancer from multiple perspectives, constructed a prognostic model, and conducted immune infiltration analysis. Our findings contribute to the understanding of prostate cancer and its prognosis, providing valuable insights for future research and prognostic assessments in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94839f7e60978966361da4e926a8390bd98bb805" target='_blank'>
              Exploring the key functions of T cells and the regulation of the immune microenvironment in prostate cancer using single-cell RNA sequencing and bulk RNA sequencing.
              </a>
            </td>
          <td>
            Zhidu Wang, Yan Xing, Dongmei Shang, Xuefei Jin
          </td>
          <td>2025-12-31</td>
          <td>Autoimmunity</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Lung cancer symptoms typically do not appear until advanced stages, leading to late diagnoses. This delay is a major contributor to its poor prognosis, resulting in lung cancer being the leading cause of all cancer-related deaths worldwide. Essential biological understanding of how lung cancers arise and progress is still lacking. Studying premalignant lesions that eventually develop into invasive carcinomas helps to bridge this knowledge gap. We are focusing on adenocarcinomas (ACs), the most common subtype of lung cancer, and especially, activating mutations in the Kirsten rat sarcoma virus oncogene (KRAS), which is among the most common oncogenic mutations and has previously been reported in premalignant lung lesions. Given that KRAS signaling is growth-promoting, it is reasonable to assume that KRAS mut promotes tumor formation and progression. However, the extent of its influence, the timing of when it arises, and particularly its connection to histologic and phenotypic changes driving tumor evolution, remain unclear.



 We have collected a cohort from biopsy material, initially collected for diagnostic purposes from AC patients at varying stages of tumor evolution, with some matched ‘normal-histology’ samples when available. In total, tissue from 38 patients with AC premalignancies is available for investigation. To map KRAS mutations in situ, we employed a BaseScope assay with commercial detection probes against the different KRAS mut transcripts. To map KRAS statuses to cellular phenotypes in the microenvironment, we are running the 6K discovery panel on the CosMx platform (Bruker) on a subset of patients, which is a highly sensitive spatial transcriptomics technique at single-cell resolution. Spatial Mass Spectrometry (MS) tracking metabolic features on the timsTOF flex MALDI-2 (Bruker) is also being performed on a subset of patients.



 Initial tests of the BaseScope assay have revealed specific, yet not sensitive, signals of KRAS mut transcripts in premalignant lesions. While advanced cancers display distinct signals, premalignant lesions have fewer, if any, signals. This might be due to the lower expression levels of KRAS at the earlier stages. To address this, we are now applying a BaseScope assay with signal amplification to be able to track KRAS expressed at lower levels. Preliminary analyses of the Spatial MS matrix have revealed highly individual metabolomics features, with great inter-patient heterogeneity. This pattern is also reflected in bulk RNA-seq data, revealing expression profiles more similar across, e.g., Adenocarcinoma in Situ (AiS) and invasive AC from the same patient, than between the AiSs from different patients.



 By applying this state-of-the-art multi-omics approach to premalignant adenocarcinomas, we hope to identify key regulators of early lung cancer progression and to translate these findings to clinically relevant basic molecular diagnostics.



 Amanda Lindberg, Bartosz Sobocki, Patrick Micke, Carina Strell. In situ KRAS-mapping and spatial-omics characterization in lung premalignancy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A002.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23b558a4354aae9f9807afda6b85aec2e424e2cc" target='_blank'>
              Abstract A002:
 In situ
 KRAS-mapping and spatial-omics characterization in lung premalignancy
              </a>
            </td>
          <td>
            A. Lindberg, B. Sobocki, P. Micke, C. Strell
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b536ff0acd69dbca3af6bd3122c752115f69d4a" target='_blank'>
              Single-Cell Transcriptomic Profiling of GL261 Glioblastoma Cells Reveals Gene Expression Signatures Underlying Tumorigenicity.
              </a>
            </td>
          <td>
            Colton E Troxel, Ruby A Olvera, Emily A Freko, Suely S C Soeiro, Thandiswa Mdluli, Richard L Daniels
          </td>
          <td>2025-12-16</td>
          <td>Cellular and molecular neurobiology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4f815c5ea5ff8465ffcedac66d5a28bbf1f8878" target='_blank'>
              Metabolic profiling of therapy-induced senescent cancer cells via TPEF, MALDI-MS, and RNA-sequencing.
              </a>
            </td>
          <td>
            S. Ghislanzoni, F. Padelli, Matteo Niero, A. Bertolotti, A. Belfiore, Simone Torelli, A. Bresci, Andrea Masella, Silvia Betti, Dario Polli, Luca Agnelli, I. Bongarzone
          </td>
          <td>2025-12-17</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAM) exert both pro- and anti-tumoral functions that influence cancer progression and patient prognosis. However, single-cell RNA sequencing (scRNA-seq) studies have revealed that TAM heterogeneity remains incompletely characterized. By performing an unbiased, integrated in silico analyses of publicly available scRNA-seq datasets, comprising samples from blood, tumor, and non-tumoral mammary tissue (NTMT) from both breast cancer (BRCA) patients and healthy individuals, we identified seven transcriptional signatures corresponding to distinct TAM subsets, exhibiting unique functional profiles, including heightened interferon responses, scavenging, and matrix remodeling, the latter two being characteristic of tissue repair. Notably, none of these subsets aligned with the M1/M2 classification of macrophage (Mφ) polarization. Interferon-associated genes were predominantly enriched in blood monocytes, whereas tissue-repair-associated-signatures were more abundant in tissue resident Mφ, suggesting that TAMs bearing these signatures resemble monocyte-derived or tissue-resident Mφs, respectively. Importantly, TAM subsets expressing interferon-associated genes were associated with improved survival compared to tissue-repairing TAMs in a BRCA cohort from The Cancer Genome Atlas (TCGA). Additionally, one signature was heightened in peripheral monocytes from BRCA patients compared with healthy individuals, which was experimentally validated in a pilot study of Mexican BRCA patients. We concluded that these signatures are a closer description of TAM heterogeneity in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769e9d0c6f0718581121627c404c58103ae137e7" target='_blank'>
              Single cell analysis identifies inflammatory and tissue remodeling tumor associated macrophages distinct from M1/M2 paradigm.
              </a>
            </td>
          <td>
            Eric López-Huerta, Diana P Portales-Pérez, M. E. Martínez-Leija, Vianney Ortiz-Navarrete, E. Fuentes-Pananá
          </td>
          <td>2025-12-05</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most lethal brain tumors, characterized by extensive immune evasion and a macrophage-dominated tumor microenvironment (TME). However, the molecular determinants governing tumor-associated macrophage (TAM) states and their immunoregulatory functions remain poorly understood. We integrated bulk- and single-cell transcriptomic datasets (TCGA, CGGA, Ivy GAP, and Brain Immune Atlas) to systematically characterize the expression, prognostic relevance, and immune contexture of the myeloid biomarker membrane-spanning 4-domain A6A, MS4A6A, in GBM. Differential expression, survival, and pathway enrichment analyses were performed. Single-cell mapping and CellChat modeling delineated MS4A6A-associated TAM subpopulations, intercellular communication networks, and ligand–receptor signaling dynamics. Spatial transcriptomic validation and pharmacogenomic modeling were conducted to assess anatomic enrichment and therapeutic vulnerabilities. High MS4A6A expression predicted unfavorable survival and correlated with increased stromal and immune infiltration. Single-cell analyses localized MS4A6A predominantly to TAMs, especially Regulatory- and Ribo-TAM states enriched for antigen presentation, T-cell regulation, and ribosomal biogenesis pathways. CellChat analysis revealed that MS4A6A-high TAMs exhibited markedly enhanced communication with CD4+ T cells and Tregs through upregulated PGE2–PTGER2/PTGER4, PECAM1–CD38, and THBS1–CD36 signaling axes, implicating MS4A6A in prostaglandin-driven immune suppression. Spatial profiling confirmed preferential localization of MS4A6A within perivascular and angiogenic niches. Pharmacogenomic prediction indicated that MS4A6A-high tumors were more sensitive to ERK, mTOR, and CDK4/6 inhibition. MS4A6A defines a macrophage-centered, immunosuppressive ecosystem in GBM, mediated by the activation of the PGE2 signaling axis. These findings position MS4A6A both as a prognostic biomarker and as a potential therapeutic node linking myeloid reprogramming to actionable pathway vulnerabilities in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aace77685200c650b039b4e217d58ff78ecf5af" target='_blank'>
              The Myeloid Biomarker MS4A6A Drives an Immunosuppressive Microenvironment in Glioblastoma via Activation of the PGE2 Signaling Axis
              </a>
            </td>
          <td>
            Jianan Chen, Qiong Wu, Anders E. Berglund, Robert J. Macaulay, James J. Mulé, Arnold B. Etame
          </td>
          <td>2025-12-20</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tissue-resident memory T (TRM) cells, memory T cells that stably occupy tissues and contribute to immunosurveillance, induce favorable survival outcomes in solid tumors. While TRM cells have been observed in lymph nodes, their phenotype and prognostic significance in diffuse large B-cell lymphoma (DLBCL) remains uncharacterized. In this study, CD103+ T cells were quantified in DLBCL samples by immunofluorescence (IF) staining of tissue biopsies and flow cytometry of cell disaggregates, and linked with clinical outcomes. Across two patient cohorts, CD103+ T cells were identified in both nodal and extranodal DLBCL, and higher CD103+ T cell levels correlated with superior clinical outcomes. Single-cell RNA sequencing (scRNAseq) revealed ITGAE expressing T cells in both malignant and reactive lymph node (rLN) samples. However, transcriptional profiles differed, as a canonical TRM population was observed in the malignant setting. This TRM cluster was enriched for genes associated with cytotoxicity and activation and was validated in an external CITEseq dataset. Flow cytometry additionally confirmed protein expression of TRM markers (CXCR6, CD39, and PD-1) on CD69+CD103+ T cells. We assessed functional activity in co-culture experiments of CD103+ versus CD103- T cells with autologous CD20+ B cells, where CD103+ T cells displayed enhanced killing. CD103+ TRM cells in DLBCL represent a prognostically favorable population with an activated/cytotoxic T cell phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75e4cde340cefbbda9de07c5cdbbe7ec6ec4213" target='_blank'>
              Phenotypic characterization and prognostic impact of CD103+ tissue-resident memory T cells in diffuse large B cell lymphoma.
              </a>
            </td>
          <td>
            Gillian Savage, K. Milne, B. Collinge, Yifan Yin, Pauline Loos, Mary E Warren, Céline M. Laumont, Danielle Keith, Brad H Nelson, Leandro Venturutti, Andrew P. Weng, D. Scott, Christian Steidl, L. Evgin
          </td>
          <td>2025-11-21</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Breast cancer is the most common cancer among women and a leading contributor to disease-related mortality. While advancements in diagnostic techniques and the widespread dissemination of medical knowledge have improved outcomes, immunotherapy targeting the tumor microenvironment offers a promising approach to further reducing the disease burden. Methods We first explored cellular heterogeneity within breast cancer tumors using single-cell RNA sequencing and subsequently analyzed the association between the infiltration levels of various cell types and survival prognosis using a deconvolution algorithm combined with bulk RNA sequencing. Our analysis identified CXCL9 + macrophages as being associated with improved survival outcomes. Next, WGCNA was used to identify genes related to CXCL9 + macrophage infiltration, and a survival risk model was constructed based on these genes. Finally, experiments were conducted to validate the effects of CXCL9 + macrophages on breast cancer cells. Results We found that CXCL9 + macrophage primarily interacted with cytokines and activated T and NK cells through the CXCL9/10/11-CXCR3 axis. Increased infiltration, as observed in bulk RNA sequencing, was associated with improved patient survival. The risk model constructed based on CXCL9 + macrophage infiltration-related genes demonstrated strong efficacy in predicting patient survival outcomes. Finally, in vitro experiments confirmed that CXCL9 + macrophages inhibited the viability of breast cancer cells. Conclusion CXCL9 + macrophages could play a significant role in inhibiting breast cancer, and their infiltration into tumor tissues is associated with improved survival in breast cancer patients. Immunotherapy targeting CXCL9 + macrophages hold great potential as a therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c633a0c3546d68c439ee716b36ed43ccdccd0f56" target='_blank'>
              Infiltration of CXCL9+ macrophages confers a favorable prognosis in breast cancer: Insights from an integrated single-cell RNA and bulk RNA sequencing study
              </a>
            </td>
          <td>
            Bin Liang, Zhicai Duan, Siyu Long, Peng Zhou
          </td>
          <td>2025-12-01</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 A crucial adaptability trait of melanoma cells that persist under all selection pressures, including therapies, is their ability to efficiently switch from proliferation to quiescence. With the goal of improving therapeutic strategies for preventing melanoma recurrence, herein we describe an approach to modeling this trait in cell culture. Body imposes a variety of selection pressures and bottlenecks ranging from different components of immunity to metabolic challenges to enforce quiescence and influence cancer evolution. The rationale for the choice of model as cell culture is that it is ideal for modeling the realistic depth of quiescence and for evaluating therapeutic strategies that may inhibit highly abnormal but adaptable cancer cells. As it is not possible to replicate all such bottlenecks in cell culture, our approach is to apply as severe a bottleneck as possible, thus eliminating all proliferative (non-adaptable) cells, leaving behind rare cells surviving in deep quiescence. We hypothesize that if the quiescent cell state being modeled can survive a severe bottleneck, it can also survive other bottlenecks of similar or lower severity. A lack of glutamine proved to be a strong and prolonged selection pressure for the highly metastatic human melanoma cell line A375SM, killing more than 99% of cells and selecting rare cells based on their ability to survive in deep quiescence. After 4 weeks, cells gradually exited quiescence and proliferated indefinitely. Furthermore, we modeled even deeper quiescence lasting longer than 4 weeks by increasing the severity of the selection pressure using dialyzed serum in medium. The cells selected in this manner were much more resistant to paclitaxel than was their parental cell line. We obtained similar results with the highly metastatic mouse melanoma cell line B16-BL6. We conclude that our approach is suitable for modeling abnormal deep quiescence in melanoma that drives the disease, including recurrence and metastasis. The main strength of the approach is that, besides modeling genetic alterations, it incorporates suitable selection pressures and the time element that are important in cancer evolution. Although there is a limitation of lack of body-like microenvironments in cell culture, we believe that abnormal quiescent cancer cell state being modeled here is capable of overcoming restraints imposed by the body. We will also discuss our published and new observations with rare cells (0.01% in population) surviving in quiescence for several weeks in aggressive triple-negative breast cancer cell lines and their functional characteristics. Our results obtained with adaptable cells from SUM149 TNBC cell line, modeled as above for melanoma cells, suggest that they are stem-like quiescent cells. We will also discuss how we choose and evaluate therapies that would likely inhibit quiescent and slow-growing cancer cells prior to recurrence. This approach may significantly improve how we investigate deep intrinsic resistance and strategies to overcome it.



 Balraj Singh, Vanessa N. Sarli, Nikil Erry, Anthony Lucci. A cell culture model of quiescent melanoma cell state that governs cancer evolution, therapy resistance, and recurrence/metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b4de6eb92fbb6a26e62be45c5389b5a53529d5" target='_blank'>
              Abstract A010: A cell culture model of quiescent melanoma cell state that governs cancer evolution, therapy resistance, and recurrence/metastasis
              </a>
            </td>
          <td>
            Balraj Singh, V. Sarli, Nikil Erry, Anthony Lucci
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related mortality, representing a highly coordinated, multistep process in which malignant cells gain the ability to invade, survive in the circulation, and establish secondary tumors at distant sites. While genetic mutations initiate oncogenesis, accumulating evidence shows that epigenetic and epitranscriptomic regulators, encompassing DNA methylation, RNA modifications, and noncoding RNAs (ncRNAs), reshape metastatic phenotypes. This review integrates current insights into these mechanisms and their crosstalk, with a primary focus on their methylation modification. Given their plasticity and potential reversibility, these regulators are attractive targets for therapeutic intervention. Defining the dynamic interplay between DNA and RNA modifications and ncRNAs provides a coherent framework for controlling metastasis and guides the development of precision epigenetic strategies and biomarkers. Future research that integrates multi-omics approaches and spatial transcriptomics will be essential for revealing the epigenetic and epitranscriptomic layers of the metastatic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32e97223f0c41298eec4d68679c66b5c3087095" target='_blank'>
              Rewriting the Fate of Cancer: Epigenetic and Epitranscriptomic Regulators in the Metastatic Cascade
              </a>
            </td>
          <td>
            Toshifumi Hara, M. Subramanian
          </td>
          <td>2025-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is a highly prevalent malignancy with poor prognosis, largely driven by lymph node metastasis (LNM). Despite its clinical significance, the underlying mechanisms of LNM remain elusive. In this study, we used single-cell transcriptomic data to dissect the cellular and molecular interactions within metastatic lymph nodes (MET). Specifically, we analyzed Single-cell RNA sequencing (scRNA-seq) data from GSE195655, GSE140042, GSE227156, and GSE159929 (n = 41) to delineate cellular heterogeneity, intercellular communication networks, and functional enrichment in primary tumors (PT), MET, and non-metastatic lymph nodes (LN). Our analysis identified a subpopulation of tumor-associated macrophages (TAMs) distinctly enriched in MET, characterized by high expression of SPP1. Functional analysis revealed that this TAM subpopulation promotes angiogenesis through specific ligand-receptor interactions with endothelial cells (ECs), involving the SPP1-ITGa9b1 and FN1-ITGa2b1 signaling axes. Furthermore, we leveraged bulk RNA-seq for prognostic research. Immunohistochemistry (IHC) confirmed the increased density and number of blood and lymphatic vessels in MET. Spatial analysis via multiplex immunohistochemistry (mIHC) confirmed the preferential localization of SPP1+ TAMs near ECs in MET. These results suggest a potential communication between TAMs and ECs that contributes to LNM in HNSCC, providing critical insights for HNSCC prognosis and precision treatment. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-27796-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa35cd24399d837bd70f92cab5dbd70391a0453c" target='_blank'>
              Single-cell identification of an endothelial cell proximal SPP1+ macrophage population defines the metastatic vascular niche in lymph nodes
              </a>
            </td>
          <td>
            Zhentao Lao, Rukeng Tan, Mailikanmuhan Maitikuerban, Xiaofang Xu, Sien Zhang, Jingxin Ma, Dikan Wang, Guiqing Liao, Le Yang
          </td>
          <td>2025-12-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Intratumoral genomic heterogeneity (ITGH), the existence of genotypic and phenotypic variation within an individual tumor, is known to be a key mechanism in treatment resistance. Deviating gene Expression Profiling Tumor Heterogeneity 2 (DEPTH2) algorithm was developed to estimate ITGH using solely RNA expression data unlike the others that require both DNA- and RNA-expression data. A total of 7508 breast cancer patients from multiple independent cohorts were analyzed using DEPTH2. High DEPTH2 score patients were associated with worse overall survival consistently across all subtypes in SCAN-B, but not in TCGA and METABRIC cohort. Higher scores were also linked to increased cell proliferation, as evidenced by elevated Nottingham histological grades and Ki67 gene expression, as well as enrichment of the cell proliferation-related gene sets, and immune cell infiltrations. DEPTH2 score was significantly higher in triple negative breast cancer among the subtypes but did not reflect with lymph node and distal metastasis. DEPTH2 scores decreased in two but showed no change in another two cohorts after neoadjuvant chemotherapy (NAC). This score was not associated with pathological complete response after NAC in any subtypes across 3 cohorts. DEPTH2 score may not capture the entire biological aspects of ITGH in breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/674d3dd6eefd604ca7935c900aea2b2b00942a09" target='_blank'>
              DEPTH2 score was associated with cell proliferation and immune cell infiltrations but not with systemic treatment response in breast cancer
              </a>
            </td>
          <td>
            Kohei Chida, Rongrong Wu, A. M. Roy, Takashi Ishikawa, Kenichi Hakamada, K. Takabe
          </td>
          <td>2025-11-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a59f51e7895cef79efeacf0e1e7ab108d79743ff" target='_blank'>
              Machine learning-enabled spatial multi-omics uncovers lactate-driven targets and tumor microenvironmental reprogramming in cancer.
              </a>
            </td>
          <td>
            Yingzheng Tan, Wenliang Tan, Yanchao Liang, Yunzhu Long, Shuanghua Chen, Qihao Hu, Yangjing Ou, Jingli Fu, Huan Chen, Fangyuan Ren, Jun Ye, Qing Zhou, Sheng Li, Xiaojin He, Qianqian Wang, Yueming Shen, Haiyuan Lu, Daichao Wu, An-bo Gao, Xun Chen, Yukun Li
          </td>
          <td>2025-12-30</td>
          <td>NPJ digital medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c362676e5f313e967da9a02b557ed5fbcda507b0" target='_blank'>
              Single-cell and spatial transcriptome-based metabolism-immunity interaction network and therapeutic target discovery of matrine in cervical cancer.
              </a>
            </td>
          <td>
            Hong Zou, Wang Xiong, Yumei He
          </td>
          <td>2025-11-24</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Human papillomavirus (HPV) infection is associated with an increased risk of cutaneous squamous cell carcinoma (CSCC). A comprehensive understanding of the cellular heterogeneity of HPV-positive and -negative CSCC is crucial for improving diagnosis and preventing tumor progression. We conducted an integrated analysis of single-cell RNA and spatial transcriptomic data from different skin tissue sources to map the cellular landscape of the tumor microenvironment (TME) in both HPV positive and negative CSCC. Results were validated through multiplex immunohistochemistry (mIHC) and in vitro experiments. We identified 10 major cell types in CSCC and normal skin samples, including epithelial cells, myeloid cells, T cells, fibroblasts, endothelial cells, B cells, smooth muscle cells, mast cells, melanocytes, and hair follicle cells. Notably, fibroblasts were found to be associated with tumor progression in CSCC with or without HPV infected. We further identified eight major CAF subtypes in CSCC, with iCAFs-CXCL2 promoting tumor progression, while iCAFs-PLA2G2A acted to suppress tumor growth. The MDK-ITGA6 pair was found to mediate interactions between fibroblasts and epithelial cells in CSCC. mIHC analysis confirmed elevated expression of MDK and ITGA6 in CSCC samples. Additionally, cell co-culture experiments confirmed that MDK-mediated CAFs were shown to enhance tumor cell migration and invasion in CSCC. Our findings provide a comprehensive cellular atlas of CSCC, highlighting the association of CAFs in HPV infection and tumor progression of CSCC. These results also offer potential diagnostic and prognostic biomarkers for CSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65aef27a5d8cbdec4a99704bf580728fefb52e8e" target='_blank'>
              Multi-omics analysis unveils the role of cancer-associated fibroblasts in cutaneous squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Wang, Tingrui Li, Yichao Jin, Jingjing Chen, XinYang, Zhen Guan, Mei Jin, Jingxian Zhang, Liangheng Xu, Sizhen Tao, Chunguang Li, Chunping Ao
          </td>
          <td>2025-11-19</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Breast cancer (BC) is the most common malignancy among women, with an increasing incidence correlated with age and diverse subtypes exhibiting distinct prognoses. The tumor microenvironment (TME) in BC is complicated. It is now believed that BC may acquire invasive characteristics and even extra proliferative ability from the TME through various mechanisms. However, most studies predominantly focus on the heterogeneity of tumor cells in BC, lacking a comprehensive depiction of intercellular communication within BC. Therefore, the present study aimed to elucidate cellular communication in the TME by integrated bioinformatic analysis of bulk mRNA and single-cell mRNA sequencing, combined with certain validation in clinical samples. Methods We first utilized single-cell sequencing data from GSE176078 to find out the most important cell communication pairs for the tumor microenvironment in BC then we conducted bulk-sequencing analysis to identify the differential expressed genes. Through correlation analysis, we sort out the top five most relevant genes to the most important cell communication pairs. We then validated the expression of the key genes of the aforementioned cell communication pairs and the five differentially expressed genes by qPCR on clinical samples. Furthermore, we analyzed the immunological relevance of these genes via a novel approach at single-cell resolution. Results The results of single-cell analysis indicated that the CXC12-CXCR4 ligand-receptor pair in the CXCL pathway and the FGF7-FGFR1 ligand-receptor pair in the FGF pathway are the most important cell communication pairs of the TME in BC. Subsequent bulk sequencing analysis showed that CHRDL1, SCARA5, LYVE1, PI16, and SAA2 were the most important differentially expressed genes linked to these cell communication pairs. In addition, we validated the expression of the key genes of the two cell communication pairs and the five genes in clinical samples, observing that the trends fitted the computational results. Finally, we studied the association of these genes with immune cell infiltration at single-cell level and had it cross-validated in bulk sequencing data, finding out that there were significant connections. Conclusion In the tumor microenvironment of breast cancer, intercellular communication pairs of different cell types and molecules can exacerbate the development of breast cancer. among them, through the present study, we found that CXCL12-CXCR4 and FGF7-FGFR1 are the most important. Also, most significantly differentially expressed genes including CHRDL1, SCARA5, LYVE1, PI16, and SAA2 seemed to play a critical role in these mechanisms and immune cell infiltration, shaping the TME of BC. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03859-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2625cd3b8acf01f328a363b0abd8247b6fa35d" target='_blank'>
              Integrated bioinformatics analysis to elucidate cellular communication in the microenvironment of breast cancer
              </a>
            </td>
          <td>
            Man Wang, Hua Li, Xingong Zhang, Zhijian Zha, Mengjia Zhao, Qi Wang, Danfeng Zhang
          </td>
          <td>2025-11-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Why and how does cancer start? Building from a Symposium at the 2025 Society of Toxicology meeting, we convened a group of international experts to answer this seemingly simple question. As experimental evidence has evolved, perspectives on cancers' origins have shifted from the accumulation of DNA mutations in single cells to complex processes involving signals from an altered tissue microenvironment which promote tumorigenesis. Carcinogen exposures impact the biology of the microenvironment in complex and tissue-specific ways. These changes can include the infiltration of inflammatory cells that produce growth factors, neo-angiogenesis, morphological changes, and immune tolerance that avoids immune-mediated elimination. In this in-depth review, we discuss the evidence linking chemical-driven microenvironmental changes in the development of a range of solid and liquid tumors. We discuss specific phenotypic alterations, such as selection pressure driving clonal expansion and cellular plasticity and reacquisition of stem cell states, linked to carcinogen-induced changes in the microenvironment. We describe assays and biomarkers which can allow us to experimentally assess links between chemical exposures, the microenvironment, and cancer phenotypes. We end by discussing how understanding the role of the microenvironment and malignancy in toxicology is essential for accurate cancer hazard evaluation, development of next-generation risk assessment frameworks, identifying new strategies for cancer prevention, and improving patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c64691a327a7b8227d277fcb6bbfe9d59f21fba7" target='_blank'>
              Rethinking the Microenvironment's Role in Chemical-Induced Malignancy.
              </a>
            </td>
          <td>
            William H. Bisson, Richard A Currie, Emilia L Lim, Coraline Mlynarczyk, Erik J Tokar, Annamaria Colacci, Justin A. Colacino
          </td>
          <td>2025-12-17</td>
          <td>Toxicological sciences : an official journal of the Society of Toxicology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Acute myeloid leukemia is a genetically and cellularly heterogeneous disease. We characterize 120 AMLs using genomic and transcriptomic analyses, including single-cell RNA sequencing. Our results reveal an extensive cellular heterogeneity that distorts the bulk transcriptomic profiles. Selective examination of the transcriptional signatures of >90,000 immature AML cells identifies four main clusters, thereby extending current genomic classification of AML. Notably, NPM1-mutated AML can be stratified into two clinically relevant classes, with NPM1class I associated with downregulation of MHC class II and excellent survival following hematopoietic stem cell transplantation. NPM1class II is instead associated with resistance to allogeneic T cells in an ex vivo co-culture assay, and importantly, dismal survival following hematopoietic stem cell transplantation. These findings provide insights into the cellular state space of AML, define diagnostic entities, and highlight potential therapeutic intervention points. The clinical outcomes and treatment responses of acute myeloid leukemia (AML) patients are heterogeneous. Here, the authors use bulk and single-cell sequencing approaches and identify two transcriptomic subtypes within NPM1-mutated AML with distinct immune evasion properties and responses to hematopoietic stem cell transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0674b44d7df57e733fca27fc81a60b0eeaf92970" target='_blank'>
              The AML cellular state space unveils NPM1 immune evasion subtypes with distinct clinical outcomes
              </a>
            </td>
          <td>
            H. Lilljebjörn, Pablo Peña-Martínez, Hanna Thorsson, Rasmus Henningsson, M. Rissler, N. Landberg, Noelia Puente-Moncada, Sofia von Palffy, Vendela Rissler, Petr Stanek, Jonathan Desponds, Xiangfu Zhong, Gunnar Juliusson, Vladimir Lazarevic, Sören Lehmann, Magnus Fontes, H. Ågerstam, Carl Sandén, Christina Orsmark-Pietras, T. Fioretos
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background/Objectives: Tumor-induced immune reprogramming is increasingly recognized as a key mechanism by which cancers evade surveillance and promote disease progression. The interaction between cancer and immune cells within the tumor microenvironment (TME) can drive phenotypic and functional changes in immune populations, facilitating metastasis and immune evasion. Methods: In this study, we used co-culture models to expose THP1 monocytes to triple-negative breast cancer (TNBC) cells, MDA-MB-231 and BT-549, either directly or indirectly via tumor-conditioned media, to mimic tumor–immune cell communication. Transcriptomic and pathway analyses revealed that cancer-exposed monocytes adopt a reprogrammed phenotype marked by activation of pro-tumorigenic signaling pathways, enhanced proliferative capacity, and elevated expression of pro-inflammatory cytokines such as IL6. Results: Functional assays confirmed a significant increase in monocyte proliferation under both direct and indirect tumor exposure. Importantly, we demonstrated that this tumor-driven proliferation of THP1 cells could be suppressed by the STAT3 inhibitor STAT3-IN-12. This highlights the critical role of STAT3 signaling in mediating immune cell transformation and supporting a novel immunomodulatory approach for therapeutic intervention. Conclusions: These findings support the potential for targeting tumor-educated transcriptional programs as a novel immunomodulatory strategy in cancer treatment. Restoring immune cell homeostasis and suppressing pro-tumor phenotypes through pharmacological inhibition of the key signaling nodes such as STAT3 may complement existing cancer therapies. This study provides new insights into immune cell plasticity in cancer and identifies actionable strategies to counteract tumor-driven immune dysregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49c41652e9aa2af643e0781e40ec3d484d82be01" target='_blank'>
              Tumor–Immune Cell Crosstalk Drives Immune Cell Reprogramming Towards a Pro-Tumor Proliferative State Involving STAT3 Activation
              </a>
            </td>
          <td>
            Karen Norek, Jacob Kennard, K. Fuh, Robert D. Shepherd, Kristina D. Rinker, O. Kharenko
          </td>
          <td>2025-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73602df6625b44848edf1084a701c143743994eb" target='_blank'>
              Tumors Located in the Brain Impair the Frequency and Phenotype of Dendritic Cells in Blood and Tumor
              </a>
            </td>
          <td>
            B. Gardam, T. Gargett, E. Nam, Sidra Khan, Rebecca J Ormsby, Santosh I Poonnoose, Julie M Bracken, Anupama Pasam, S. Lenin, Briony L Gliddon, Melinda N. Tea, C. Shard, Stuart M. Pitson, Guillermo A Gomez, Katherine A. Pillman, Shahneen Sandhu, Michael P Brown, LM Ebert
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) is a highly invasive malignant tumor, and its tumor microenvironment (TME) plays a pivotal role in disease progression and treatment resistance. As key components of the TME, fibroblasts and keratinocytes are hypothesized to synergistically regulate tumor biological behavior. This study integrated single-cell transcriptomic data with bulk RNA-seq data from ESCC to investigate the interaction mechanisms between fibroblasts and keratinocytes within the tumor microenvironment. Rigorous quality control, standardization, and clustering analyses were performed on the single-cell data, identifying ten distinct cell subpopulations, including fibroblasts and keratinocytes. The proportions and expression profile differences of these two cell types were compared between experimental and control groups. Cell–cell communication analysis deciphered ligand-receptor pairs and associated signaling pathways governing intercellular interactions. Molecular subtyping of The Cancer Genome Atlas (TCGA) cohort samples was subsequently performed using non-negative matrix factorization (NMF) based on interaction-specific genes. A multi-gene prognostic model was constructed via least absolute shrinkage and selection operator (LASSO) regression, and its predictive efficacy was evaluated. Finally, functional enrichment, immune microenvironment assessment, and differential expression analyses revealed differences in biological functions and immune infiltration patterns across distinct risk subtypes. By integrating the gene expression profiles of these two cell types, a novel molecular classification system was successfully established, enabling the stratification of ESCC patients into C1(high-risk) and C2(low-risk) subtypes with marked differences in survival outcomes. Patients classified as subtype C2 had a median survival of 3.8 months, significantly longer than the 1.8 months observed in the high-risk subtype C1, highlighting the substantial clinical relevance of this classification. Further analysis revealed that the high-risk cohort exhibited significant enrichment in metabolism-related pathways, such as endocytosis and glyceride metabolism, whereas the low-risk cohort was predominantly associated with pathways involved in cell junctions and structural maintenance. Mechanistically, the interaction between these two cell types jointly promotes the reprogramming of the tumor immune microenvironment by regulating immune cell infiltration, HLA molecule expression, and immune checkpoint activity. Furthermore, the study identified key genes, including SLIT2 and SFRP1, which display cell-type-specific expression patterns in ESCC and hold potential as prognostic biomarkers or therapeutic targets. This research provides novel insights into the regulatory mechanisms of the ESCC tumor microenvironment, laying a theoretical foundation for the development of precision therapeutic strategies targeting the tumor microenvironment. This study elucidates the synergistic regulatory mechanisms between fibroblasts and keratinocytes within the tumor microenvironment of ESCC. These findings offer a novel theoretical foundation for molecular classification, prognosis prediction, and precision treatment of ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec3cca186de01a25c15c9e25673d6418ec7eece" target='_blank'>
              Mechanisms of synergistic regulation of the tumor microenvironment by fibroblasts and keratinocytes in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Yujie Weng, Rong Jia, Pengfei Ning
          </td>
          <td>2025-11-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND: The spatial organization of immune cells within tumors is an important area of investigation in studies of interactions between tumor cells and the tumor microenvironment. This is particularly relevant because some immune cells function through direct contact with their target cells, whereas others communicate over a distance via paracrine signaling involving cytokines. Thus, the topography of tumor cells and microenvironmental cells may determine the possibility and nature of intercellular interactions and thereby influence the functional effects of immune cells. AIM: This study aimed to compare the spatial transcriptomic profiles of tumor and stromal regions enriched in macrophages in triple-negative breast cancer. METHODS: Eight patients with triple-negative breast cancer were included. Spatial transcriptomic analysis was performed on formalin-fixed, paraffin-embedded tissue sections using high-throughput RNA sequencing with the 10x Visium platform. The annotated spots enriched in intraepithelial and stromal macrophages were used for downstream bioinformatic analysis. RESULTS: A total of 437 differentially expressed genes were identified between the two groups of spots containing macrophages with distinct spatial localization. Spots with intraepithelial macrophages were characterized by activation of processes related to cytokine and chemokine signaling, regulation of regulatory T-cell differentiation, organization of cell – cell contacts, wound healing, and inhibition of viral activity. Spots enriched in stromal macrophages demonstrated activation of biological processes associated with the regulation of angiogenesis, cell migration and recruitment, cell adhesion, and stromal remodeling. CONCLUSION: Macrophage topography within primary tumors of triple-negative breast cancer is associated with their functional characteristics. These fundamental findings may be useful for developing prognostic criteria and therapeutic approaches aimed at modulating the tumor microenvironment to improve long-term outcomes in patients with triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1996d432009ab729b97a00ec19d18afda8c70949" target='_blank'>
              Spatial Characterization of Macrophage-Enriched Tumor Regions in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            I. Patskan, A. Kalinchuk, Elisaveta Tsarenkova, Evgeniia Grigoryeva, I. Larionova, Natalia Popova, L. Tashireva
          </td>
          <td>2025-12-02</td>
          <td>Genes &amp; Cells</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background High endothelial venules (HEVs) are specialized blood vessels found primarily in lymphoid tissues, notably at the cortico-paracortical junction of lymph nodes (LNs). They play a key role in lymphocyte trafficking, facilitating the entry of lymphocytes into peripheral lymphoid organs. Objective Recent evidence indicates the presence of HEVs within tumor tissues, where they form part of tertiary lymphoid structures. This review aims to explore the structural and functional alterations of HEVs in tumors and LNs, and investigate their role in cancer prognosis and response to immunotherapy. Methods We conducted a comprehensive review of recent studies examining HEVs in tumors and LNs, with a focus on their involvement in immune responses and cancer progression. Additionally, the use of MECA-79 as a specific HEV marker was considered as a potential therapeutic target. Results HEVs within tumors have been associated with improved immune surveillance and better prognosis in certain cancers. The presence of HEVs in tumors correlates with enhanced immune cell infiltration and may influence the effectiveness of immunotherapy treatments. Targeting HEVs, particularly through MECA-79, holds promise as a novel therapeutic strategy to modulate tumor immunity. Conclusion Targeting HEVs in tumor tissues represents a promising avenue for cancer therapy. Further research is needed to optimize strategies for modulating HEV function and to better understand their role in immune responses to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/701871a3989e04068f26894b4756f3bd07c359ea" target='_blank'>
              Targeting high endothelial venules: potential strategies for cancer treatment
              </a>
            </td>
          <td>
            Weichang Yang, Guofeng Zhu, Shanshan Cai, Hongquan Xing, Aoyu Yang, Xiaoqun Ye
          </td>
          <td>2025-12-01</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [13, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>