<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-01 09:37:44 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/629f37baf89cd9a5cc6f98228b4de5cf8f891b4c" target='_blank'>
              ROSIE-Enabled Spatial Mapping Reveals Architectural Fragmentation and Immune Reprogramming in Lung Adenocarcinoma Evolution
              </a>
            </td>
          <td>
            Beibei Huang, Bo Zhu
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728733c32d8e17fbc34a5070c8b658170e0a8a22" target='_blank'>
              Identifying tissue states by spatial protein patterns related to chemotherapy response in triple-negative breast cancer
              </a>
            </td>
          <td>
            L.M. Luque, A. Khan, G. Torrisi, Tessa D. Green, D. Hardman, C. Owczarek, T. A. Phillips, D. Marks, M. Parsons, C. Sander, L. Schumacher
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f7dcdd3510de0385797859e77f339a9cbb0192e" target='_blank'>
              Multidimensional Single-Cell Transcriptomic Profiling of Uterine Leiomyosarcomas Identifies Molecular Subtypes with Distinct Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Maria Korah, J. Agolia, Biren Reddy, R. Flojo, Amanda Gonçalves, Lilin Wang, Wesley Bobst, Brooke Liang, Kaylin A. Yip, Rosyli Reveron-Thornton, Chuner Guo, Robert Gruener, Andrea E. Delitto, Peter Y. Xie, Angela D. Tabora, Beatrice J. Sun, Sara K. Daniel, Babak Litkouhi, Oliver Dorigo, Nam Q. Bui, Minggui Pan, E. J. Moding, A. Kalbasi, Amanda R. Kirane, George A. Poultsides, Jeffrey A. Norton, Aaron M. Newman, G. Charville, K. Ganjoo, Deshka S. Foster, R. Stephanie Huang, Byrne Lee, Michael T. Longaker, Daniel J. Delitto
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/235c8c18e964369ee970e129fd3fe5bd3b1a1aa6" target='_blank'>
              In silico reconstruction of primary and metastatic tumor architecture using GIS-augmented spatial transcriptomics
              </a>
            </td>
          <td>
            Jin Young Yoo, Sabrina Akter, Qianying Zuo, Amir Kazemi, Wenjun Wu, Mahima Goel, Audrey Lam, Debapriya Dutta, Betsy Barnick, M. Pekmezci, Maria Grosse Perdekamp, Aiman Soliman, Zeynep Madak-Erdogan
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d523903df00c8d74634f25362c4c022807a0ad86" target='_blank'>
              Spatial proteomics for investigating solid tumor resistance mechanisms
              </a>
            </td>
          <td>
            X. M. M. Zhou, A. D'Amiano, Charles Lu, Vrinda Madan, Sara Khoshniyati, J. Kollings, Noah E Sunshine, S. Surwase, Joel C. Sunshine
          </td>
          <td>2025-10-08</td>
          <td>Cancer and Metastasis Reviews</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Integration of scRNA-seq data from millions of cells revealed a high diversity of cell types in the healthy and diseased human lung. In a large and complex organ, constantly exposed to external agents, it is crucial to understand the influence of lung tissue topography or external factors on gene expression variability within cell types. Here, we apply three spatial transcriptomics approaches, to: (i) localize the majority of lung cell types, including rare epithelial cells within the tissue topography, (ii) describe consistent anatomical and regional gene expression variability within and across cell types, and (iii) reveal distinct cellular neighborhoods in specific anatomical regions and examine gene expression variations in them. We thus provide a spatially resolved tissue reference atlas in three representative regions of the healthy human lung. We further demonstrate its utility by defining previously unknown imbalances of epithelial cell type compositions in chronic obstructive pulmonary disease lungs. Our topographic atlas enables a precise description of characteristic regional cellular responses upon experimental perturbations or during disease progression. Using multiplexed spatially resolved transcriptomic approaches, the authors generate a topographic atlas of the healthy adult lung with location-related gene expression variability or neighbourhood changes as exemplified by COPD patient sample analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe80d855be30beaf44c2810f98505d59172a3d0" target='_blank'>
              Spatial single-cell atlas reveals regional variations in healthy and diseased human lung
              </a>
            </td>
          <td>
            A. Firsova, Sergio Marco Salas, L. B. Kuemmerle, X. Abalo, A. Sountoulidis, L. Larsson, Krishnaa T. Mahbubani, Jonas Theelke, Žaneta Andrusivová, Leire Alonso Galicia, A. Liontos, Tamás Balassa, Ferenc Kovács, Peter Horvath, Yuexin Chen, Janine Gote-Schniering, M. Stoleriu, Jürgen Behr, Kerstin B. Meyer, W. Timens, Herbert B. Schiller, Malte D. Luecken, F. Theis, J. Lundeberg, Mats Nilsson, M. Nawijn, Christos Samakovlis
          </td>
          <td>2025-11-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Melanoma plasticity drives immune evasion and therapy resistance through dynamic cell-state transitions beyond genetic alterations. Epigenetic remodeling critically influences such processes, yet its role in reshaping the tumor ecosystem under therapeutic pressure remains unresolved. Here, we profiled longitudinal biopsies from melanoma patients treated in the phase Ib NIBIT-M4 epi-immunotherapy clinical trial (NCT02608437), testing the combination of a DNMT1 inhibitor with anti-CTLA4 using single-cell multiome and high-resolution spatial transcriptomics. Integrated analyses resolved seven malignant meta-programs, including a rare Wnt/β-catenin–driven melanocytic state and a de-differentiated neural crest–like state enriched in non-responders. Spatial modeling revealed that homotypic clustering stabilizes resistant programs, with neural crest–like cells forming compact, centrally localized niches, whereas Wnt/β-catenin subpopulations displayed a bimodal architecture, either cohesive clusters sustained by adhesion or dispersed, transcriptionally plastic cells. Responders exhibited progressive enrichment of an antigen presentation/interferon program and coordinated remodeling of the tumor microenvironment with T and B cell expansion, whereas tumors from non-responder patients maintained stable composition of neural crest–like clusters. Epigenetic therapy reactivated transposable elements, providing both regulatory signals that prime innate immunity within microenvironment and generating antigens that drive immunoediting and immunogenicity of Antigen presentation/interferon cell states in responders. Finally, NFATC2 emerged as a master regulator of neural crests–like transcriptional phenotypes and promoter of resistance to therapeutic interventions in melanoma patients. NFATC2 perturbation was able to shift tumor cells towards more differentiated and immunogenic states. These findings reveal how epigenetic-based immunotherapy reshapes melanoma ecosystems, provide mechanistic insights into how multiple transcriptional programs promote tumor plasticity and resistance to both combinatorial therapies and immune checkpoint blockade, identify spatial clustering as a principle stabilizing resistant niches, and highlight β-catenin and NFATC2 as actionable vulnerabilities to overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9320984fb7ca80afa8b2d49d724dd86390a378" target='_blank'>
              Transposable Elements and Homotypic Niches Drive Immune Dynamics and Resistance in Melanoma Epigenetic-based immunotherapy
              </a>
            </td>
          <td>
            Erika Ciervo, Francesco Ceccarelli, A. D. Di Giacomo, Piera Grisolia, A. Covre, Z. M. Besharat, Antonio De Falco, F. Caruso, Luigi Laezza, Luigi Ferraro, Gloria Mas Martin, M. F. Lofiego, Tommaso Sani, E. Ferretti, Yan Guo, Sean B. Holden, R. Mortarini, A. Anichini, M. Maio, T. Noviello, M. Ceccarelli
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca44194b3435bd4a55c77f67d7a9d76e8205133e" target='_blank'>
              Deep multiplexed 50-marker imaging of circulating tumor cells expands actionable biomarker profiling for precision oncology
              </a>
            </td>
          <td>
            Tim J Mann, Ye Zheng, Tanzila Khan, Tim Huang, U. Nindra, Yafeng Ma, Alexander James, Daniel P. Neumann, F. Kohane, Ihuan Gunawan, F. Vafaee, W. Chua, Paul de Souza, Tara L. Roberts, Therese M. Becker, J. G. Lock
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5971868e6de8c14556026d2b663acac3f252b79d" target='_blank'>
              Genotype-phenotype correlations in Wilms tumor initiation
              </a>
            </td>
          <td>
            N. S. Pop, D. Koot, C.M. Brouwers, M. Linssen, J. Claassens, C.W.J. Cartlidge, D.D. Özdemir, K. S. Dolt, P. Hohenstein
          </td>
          <td>2025-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9ca9b266d6e3d48dee0fe7d9f0b768bc883a83" target='_blank'>
              Functional interrogation uncovers a critical role for a high-plasticity cell state in lung adenocarcinoma
              </a>
            </td>
          <td>
            J. Chan, Chun-Hao Pan, Jonathan Rub, Klavdija Krause, Emma Brown, Gary Guzman, Zeda Zhang, Hannah C Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076f7d290241ef13795e81746d2f0f634b36afeb" target='_blank'>
              Star-Motifs: Revealing Single-Cell Spatiotypes from Routine Histology Using Star-Convex Neighborhoods
              </a>
            </td>
          <td>
            G. N. Gunesli, Muhammad Dawood, Lawrence S. Young, F. Minhas, S. E. A. Raza, N. Rajpoot
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Spontaneously transformed mouse ovarian surface epithelial cell line ID8 generates a syngeneic mouse model, recapitulating the primary and metastatic stages of the malignancy hence, frequently used in pre-clinical evaluation. Due to inherent phenotypic plasticity, primary and metastatic tumors show contrasting phenotypic features dictated by the tissue niche. To address the tumor heterogeneity within the model we stratified it into a specific molecular subtype, which would provide an accurate setting for interpretation of experimental data. Stratification was performed on ID8 cell line and ID8 generated xenografts based on immunostaining, live cell imaging, histopathology, transcriptomics and proteomics. Tumor microenvironment (TME) investigation for each xenograft was done through flow cytometry analysis. ID8 cells present epithelial/mesenchymal (E/M) phenotype. Phenotypic plasticity retention in them causes class switching of ID8 cells in response to niche alteration, from E/M (in culture) to intermediate epithelial (iE) in orthotopic tumors while intermediate mesenchymal (iM) in subcutaneous tumors. Metastasis of ID8 primary tumor resulted into differentiated mesenchymal phenotype in ascites while differentiated epithelial phenotype in secondary intestinal tumors. Immune profiling of primary and secondary xenografts revealed differences in immune repertoire of tumors, highlighting subcutaneous tumor model to be inappropriate as compared to orthotopic model. Local tissue niche modulates the phenotype of tumors generated, transitioning through intermediate states towards a final commitment to either an epithelial or mesenchymal state that recapitulates the reported class switching in patient tumors. During disease progression, such heterogeneity of tumor cells at different niche(s) and their cross-talks with discrete TME at each site, will pose challenges for effective therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20023e2da4c4317c3adb1e410526c772109c27da" target='_blank'>
              ID8 cells manifest phenotypic plasticity and molecular heterogeneity of high-grade serous ovarian cancer in response to the local tissue niche
              </a>
            </td>
          <td>
            Ritika Gupta, S. Shravan, Swapna Balaji Agwane, R. M. Bankar, S. Bapat
          </td>
          <td>2025-10-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes TP53 and RB1. However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yueli Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Risk stratification remains a critical challenge in non-small cell lung cancer patients for optimal therapy selection. In this study, we develop an artificial intelligence-powered spatial cellomics approach that combines histology, multiplex immunofluorescence imaging and multimodal machine learning to characterize the complex cellular relationships of 43 cell phenotypes in the tumor microenvironment in a real-world retrospective cohort of 1168 non-small cell lung cancer patients from two large German cancer centers. The model identifies cell niches associated with survival and achieves a 14% and 47% improvement in risk stratification in the two main non-small cell lung cancer subtypes, lung adenocarcinoma and squamous cell carcinoma, respectively, combining niche patterns with conventional cancer staging. Our results show that complex immune cell niche patterns identify potentially undertreated high-risk patients qualifying for adjuvant therapy. Our approach highlights the potential of artificial intelligence powered multiplex imaging analyses to better understand the contribution of the tumor microenvironment to cancer progression and to improve risk stratification and treatment selection in non-small cell lung cancer. Risk stratification in non-small cell lung cancer (NSCLC) remains challenging. By combining multiplex immunofluorescence, H&E histology, and AI, the study identifies spatial “cell-niche” patterns that enhance survival prediction beyond UICC8 staging. These patterns reclassify many stage I patients as high risk, revealing potentially undertreated cases and establishing spatial tumor microenvironment features as clinically actionable biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53bd1a5c6021c6b38d3074a4cf442e836ada177" target='_blank'>
              AI-powered spatial cell phenomics enhances risk stratification in non-small cell lung cancer
              </a>
            </td>
          <td>
            Simon Schallenberg, Gabriel Dernbach, Sharon Ruane, Philipp Jurmeister, Cornelius Böhm, Kai Standvoss, Sandip Ghosh, M. Frentsch, M. Dragomir, Philipp Keyl, Corinna Friedrich, Il-Kang Na, Sabine Merkelbach-Bruse, A. Quaas, N. Frost, Kyrill Boschung, Winfried Randerath, G. Schlachtenberger, Matthias Heldwein, Ulrich Keilholz, Khosro Hekmat, Jens Rückert, R. Buettner, Angela Vasaturo, David Horst, Lukas Ruff, M. Alber, Klaus-Robert Müller, Frederick Klauschen
          </td>
          <td>2025-11-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b8f75faab2c4feb9da5df93770ed23963d387e7" target='_blank'>
              Nuclear Irregularity as a Universal Diagnostic Tool in Solid Tumors
              </a>
            </td>
          <td>
            Francesco Hamilton, Kane Foster, Irene Ghobrial
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5df09ad47e15df1d3b2fa365bedcbec8d739cdeb" target='_blank'>
              Unveiling Oligodendrocyte-Lineage Differentiation in the Glioblastoma Infiltrative Zone Through Spatial Transcriptomics
              </a>
            </td>
          <td>
            C. M. Argento, Enrico Sebastiani, E. F. Rosatti, D. Sighel, S. Longhi, A. Soldano, T. Tebaldi, Alessandro Quattrone, F. Corsini, Luciano Annicchiarico, M. Barbareschi, L. Pecciarini, P. L. Poliani
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 Mouse models are pivotal for translational research, enabling the detection of therapeutic-induced immune modifications within the tumor microenvironment (TME). We utilized a 40-marker panel targeting tumor and immune markers to investigate the spatial protein composition of the whole mouse embryo, mouse brain and mouse glioblastoma (GBM) tumor sections. Imaging Mass Cytometry™ (IMC™) technology was employed to capture the expression of 40 markers simultaneously while preserving tissue integrity and enabling a flexible workflow without autofluorescence artifacts. Data analysis revealed distinct spatial patterns of tumor and immune markers across the GBM, identifying key features such as necrotic cores, regions of high immune infiltration, proliferating tumor cells and diverse populations of infiltrating immune cells. The wide detection range of expression provided by IMC technology enabled visualization and analysis of clusters exhibiting varying levels of key GBM markers, suggesting a probable dual origin of the GBM, coexistence of pro- and anti-tumorigenic immune responses and overall extreme heterogeneity that can only be visualized with high-plex antibody panels. This approach demonstrates the capability of IMC technology to reveal the complex cellular and structural spatial landscape of the TME in GBM, providing valuable insights into tumor biology and potential therapeutic targets for patient stratification. For Research Use Only. Not for use in diagnostic procedures.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d885a41b02f8da25eb39b11abd7a0e0030ab5b6" target='_blank'>
              Characterization of cellular diversity and immune landscape in Mouse Glioblastoma using whole slide imaging modes of high-plex Imaging Mass Cytometry 2619
              </a>
            </td>
          <td>
            Qanber Raza, Nick Zabinyakov, T. Pfister, Nikesh Parsotam, David Howell, Liang Lim, James Pemberton, Yun-Jyh Chwee, C. Loh
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal cancer (CRC) brain metastases have a poor prognosis and limited treatment options, including resistance to radiation therapy. Little is known about the molecular and cellular mechanisms that enable CRC tumor cells to adapt to the brain and establish a supportive tumor microenvironment. To address this gap we used spatial transcriptomics to analyze 51 CRC brain metastases. A subset had matched primary colon tumors and longitudinally paired metastatic resections before and after radiation treatment. We identified the critical spatial cellular features of the tumor epithelium and the surrounding tumor microenvironment that support metastatic growth in the brain. CRC brain metastases developed a stromal microenvironment with abundant fibroblasts and tumor-associated macrophages. A fibroblast–macrophage cellular neighborhood promoted angiogenesis, extracellular matrix remodeling, and immune suppression. Tumor cells showed local adaptations. In endothelial-rich regions, they were proliferative whereas in macrophage-rich regions, they were more differentiated and immune evasive. Compared with paired primary tumors, CRC brain metastases showed increased chromosomal instability, with activation of RNA-processing, stress response, and junctional remodeling pathways. After radiation treatment, resistant clones had increased epithelial–mesenchymal transition, while the immunosuppressive stroma remained intact. We identified tumor-derived MIF, GDF15, PRSS3 and SEMA3C ligands and macrophage-derived SPP1 that have the potential to affect multiple cell types in the metastatic niche. These ligand-receptor interactions drive angiogenesis, stromal activation and immune suppression. In a macrophage–tumor–fibroblast co-culture model, knockout of SPP1 in macrophages led to reduced expression of lipid-metabolism related genes and disrupted tumor-promoting interactions. Together, these results indicate that CRC growth in the brain is sustained by a specific cellular organization with immunosuppressive multicellular interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121b9f0cf30a6fb662f64416fefc808a183dcc03" target='_blank'>
              Colorectal cancer relies on an immunosuppressive cellular topography and genomic adaptations for establishing brain metastases
              </a>
            </td>
          <td>
            A. Sathe, Mengrui Zhang, Xiangqi Bai, Ji In Kang, Rithika Meka, Huiyun Sun, Susan M. Grimes, Aparajita Khan, Mingen Liu, A. Luksik, M. Lim, Claudia K. Petrisch, Christopher M. Jackson, Hannes Vogel, Jeanne Shen, M. Gephart, Summer S. Han, 
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf8e063a68a88a96c8de0cef9efb9254b78c4b6" target='_blank'>
              A cellular and spatial atlas of TP53-associated tissue remodeling defines a multicellular tumor ecosystem in lung adenocarcinoma
              </a>
            </td>
          <td>
            William Zhao, Thinh Nguyen, Atharva Bhagwat, Akhil Kumar, Bruno Giotti, Benjamin Kepecs, Jason L. Weirather, N. Mahadevan, Åsa Segerstolpe, Komal Dolasia, Jamshid Abdul-Ghafar, Naomi Besson, Stephanie M Jones, Brian Y. Soong, Chendi Li, S. Vigneau, M. Slyper, Isaac Wakiro, Mei-ju Su, K. Helvie, Allison Frangieh, Judit Jané-Valbuena, Orr Ashenberg, Mark Awad, Asaf Rotem, Raphael Bueno, O. Rozenblatt-Rosen, K. Pfaff, Scott J. Rodig, Aaron N. Hata, Aviv Regev, B. Johnson, A. Tsankov
          </td>
          <td>2025-10-07</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="
 A major barrier in understanding adult diffuse gliomas lies in the inability to track tumor evolution in real time. To address this, we employed a novel method known as continuous lineage tracing, which integrates CRISPR/Cas9-based expressed DNA barcoding with single-cell RNA sequencing, enabling a phylogenetic approach to studying tumor development. Patient-derived glioma-initiating cell lines were engineered with expressed barcodes targeted by CRISPR/Cas9 and implanted into mice to create intracranial xenografts. Tumors underwent single-cell RNA sequencing; expressed barcodes were used to infer clonal relationships, and transcriptomic profiles enabled cell state classification. Phylogenetic lineage trees were reconstructed using lineage inference algorithms to characterize cell fitness, expansion, and plasticity. Our analysis uncovered extensive intra-clonal cell state heterogeneity, indicating active phenotype switching prior to therapy. We identified consistent transcriptional programs associated with tumor engraftment and in vivo clonal advantage. Lineage tracing revealed gene expression signatures linked to key phenotypes: fitness, enriched for neural-mesenchymal and injury-response pathways; expansion, associated with RNA splicing; and plasticity, correlated with cell cycle and DNA repair programs. Glioma stem cells appeared to span a transcriptional continuum from undifferentiated, high-fitness states to more differentiated, low-fitness states, with high-fitness cells potentially representing transitional phenotypes. We then validated these findings in a cohort of 185 surgically resected gliomas with matched bulk RNA and proteomic data. Phylogenetic gene signatures differed markedly between IDH-mutant and IDH-wildtype tumors. When stratified by tumor type, both fitness and expansion signatures were significantly associated with overall survival in GBM (median 12.6 months, p=0.041) and oligodendrogliomas (median 66 months, p=0.027). This study demonstrates the utility of continuous lineage tracing to reconstruct tumor evolution and identify transcriptional programs linked to tumor growth and prognosis. Our approach provides a powerful framework for dissecting glioma biology and identifying potential therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d67c2559a40c369b3a39bc8fe0aad62196164a" target='_blank'>
              STEM-24. SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS, EXPANSION AND PLASTICITY AXES IN ADULT DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Ajisebutu, Xiaojun Fan, Chloe Gui, Jeff Liu, Vikas Patil, A. Landry, Y. Ellenbogen, L. Yefet, Phooja Persaud, F. Nassiri, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor heterogeneity drives drug resistance and relapse, influencing immune evasion and tumor progression. While intratumor heterogeneity has been extensively studied at the genomic level, its functional outcomes and interactions with the tumor microenvironment remain underexplored. In contrast, the functional outcome of heterogeneity and the interplay with the tumor microenvironment have not been addressed. In this study, we integrate multi-region spatial MS-based proteomics of 280 tumor regions, exome sequencing, and imaging to investigate spatial proteomic heterogeneity in breast cancer. Our findings reveal increased proteomic heterogeneity with tumor progression, independent of genomic heterogeneity but closely associated with microenvironmental differences. Integration with immune and stromal imaging highlighted a dynamic interplay where low-grade tumors exhibit constrained immune infiltration, and upon progression to higher grades, macrophages and T cells infiltrate. However, anti-inflammatory pathways involving kynurenine and prostaglandins are more highly expressed in infiltrated regions, suggesting that anti-tumorigenic activities are inhibited. Integration with the global protein network provides potential targetable mediators of immune evasion in breast cancer that can serve as the basis for future development of personalized breast cancer therapies. Intratumor heterogeneity (ITH) has been mainly studied at the genomic level. Here, the authors integrate multi-region proteomics of 280 tumor regions from 33 patients, exome sequencing, and imaging-based immune and stromal profiling to investigate functional spatial proteomic ITH in breast cancer and the interactions of cancer cells and the tumor microenvironment during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d48d5d89f33d4d9425895ba820536d951749d4" target='_blank'>
              Integrated spatial proteomic analysis of breast cancer heterogeneity unravels cancer cell phenotypic plasticity
              </a>
            </td>
          <td>
            M. Mardamshina, Shiri Karagach, Vishnu Mohan, Gali Arad, Daniela Necula, O. Golani, Liat Fellus-Alyagor, Anjana Shenoy, Kateryna Krol, Daniel Pirak, Nitay Itzhacky, I. Marin, Bruria Shalmon, Y. Addadi, Roded Sharan, E. Gal-Yam, Iris Barshack, Tamar Geiger
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and lethal primary malignant tumor of the central nervous system. Advances in therapy are hindered by the complex intratumoral heterogeneity of GBM, where distinct malignant and non-malignant cellular states and interactions exist in spatially defined niches of the tumor microenvironment (TME), shaping both tumor behavior and treatment response. In this work, we define GBM biological reprogramming, TME recomposition, and cell-cell interactions in relation to spatially well defined Ivy Glioblastoma Atlas Project regions. Further, we apply a novel spatially informed approach to integrate biological reprogramming, as identified through gene co-expression network modules, with specific changes in cell-cell communications. Our results validate prior findings and offer novel insights into the spatial patterning of transcription factor regulation, cellular interactions, and biological pathway activity, in addition to informing rational combination therapies targeting spatial niche specific vulnerabilities. In addition, our work contributes a novel Visium spatial transcriptomics dataset of 14 samples for use by the GBM research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719b808c693c9788562c4b889cdeaf7fc68e251b" target='_blank'>
              Spatial transcriptomics of glioblastoma defines biologically and clinically significant reprogramming patterns across unique spatial microenvironments
              </a>
            </td>
          <td>
            V. Ravikumar, A. Maddox, Reva Kulkarni, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background The tumor microenvironment influences treatment response in ER-positive breast cancer, but what distinguishes responders from non-responders and how it changes during treatment is poorly understood. Methods ER-positive breast tumors treated with neoadjuvant chemotherapy with or without bevacizumab were profiled with bulk proteomics pre- (n = 95), on- (n = 84) and post-treatment (n = 100). A subset of tumors was profiled with spatial single-cell proteomics pre- (n = 13) and on-treatment (n = 11). Cell phenotypes, spatial location and activation states were determined, and cellular colocalization assessed with spatial metrics. Bulk and spatial features were evaluated against treatment response defined by residual cancer burden. Results Treatment with bevacizumab amplified chemotherapy effects on proteomic signaling. The immune contexture shifted from suppressive to supportive during treatment through decreased macrophage, regulatory and anergic T-cell density and increased colocalization between epithelial cells and CD8+, CD4+ T-cells and dendritic cells. At baseline, responders had high density of effector memory T-cells, while non-responders had more naïve T-cells. In addition, responders had increased colocalization of epithelial cells with macrophages, and effector memory T-cells with M1-like macrophages compared to non-responders. Conclusions Spatially distinct tumor-immune microenvironments influence response to neoadjuvant treatment, offering valuable insights for guiding treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd7292fed094fd219af6750590d28db3dc5a363b" target='_blank'>
              Spatial organization of the tumor-immune microenvironment in ER-positive breast cancer: remodeling during treatment and associations with clinical response
              </a>
            </td>
          <td>
            Maria Aanesland Dahle, Jan-Lukas Førde, E. V. Egeland, A. Creason, Cameron Watson, Ø. Garred, L. Prasmickaite, G. Mælandsmo, Gordon B. Mills, O. Engebraaten, M. H. Haugen
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Tumor thrombus (TT) worsens prognosis and complicates surgery in renal cell carcinoma (RCC), yet its formation mechanisms remain unclear. Here, we perform integrative single-cell and spatial transcriptomic analyses on 71 tissues and 48 sections from RCC patients with or without TT. The cellular and spatial atlas reveals distinct TT-associated tumor microenvironment remodeling characterized by the enrichment of FAP+ fibroblasts. These FAP+ fibroblasts are spatially contiguous to aggressive cancer cells and promote their malignant phenotypes in vitro. Their abundance inversely correlates with functional NK cells, suggesting roles in tumor invasion and immune evasion. Furthermore, single-cell multiomics analysis identifies tumor pericytes as a source of FAP+ fibroblasts and delineates transcription factor dynamics underlying pericyte-fibroblast transition. Finally, high levels of FAP+ fibroblasts are associated with poor prognosis and predict a weaker response to anti-VEGF-based therapy. In conclusion, our study highlights FAP+ fibroblasts as drivers of aggressive RCC with TT, suggesting potential therapeutic targets. Tumour thrombus (TT) complicates renal cell carcinoma (RCC) treatment. Here, authors conduct an integrative single-cell RNA sequencing and spatial transcriptomic analyses on RCC patients with and without TT and identify FAP+ fibroblasts as drivers for TT formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5472524a744e590345b74e85358cb939dc059d29" target='_blank'>
              FAP+ fibroblasts orchestrate tumor microenvironment remodeling in renal cell carcinoma with tumor thrombus
              </a>
            </td>
          <td>
            Jiacheng Ma, Yan Huang, Jie Chen, Yang Li, Rongyan Yao, Xiubin Li, Qiyang Liang, Xinran Chen, Cheng Peng, Kan Liu, Yuanjun Zhai, Xu Zhang, Xin Ma, Xiaowen Wang, Qingbo Huang, Fuchu He
          </td>
          <td>2025-10-23</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="Merkel cell carcinoma (MCC) is an aggressive skin cancer with neuroendocrine differentiation marked by high cellular plasticity, often manifesting as rapid therapy resistance. Although the cell-of-origin is presumed to be epithelial, epidermal localization of MCC is rarely observed, largely because in situ MCC is typically an incidental finding. Nevertheless, a subset of MCC tumors exhibits epidermotropism, wherein tumor cells are present in the epidermis. The behavior of cancer cells is profoundly influenced by the tumor microenvironment and interactions with neighboring cells. Notably, the normal counterparts of the cancer’s cell-of-origin have been shown to attenuate tumor aggressiveness. Thus, epidermotropic MCC presents a unique opportunity to explore the potential role of epidermal microenvironment in modulating tumor cell behavior. While the epidermotropic tumor nests share histological resemblance with their dermal counterparts, their transcriptomic profiles remain unexplored. Here, we employed high-definition spatial and single-cell transcriptomics to dissect the gene expression profiles of epidermotropic MCC cells, comparing them to MCC cells in the tumor core and those adjacent to blood vessels. Notably, epidermotropic MCC cells exhibit a transcriptomic signature reminiscent of cutaneous squamous cell carcinoma, characterized by upregulation of genes encoding keratins, S100A proteins, as well as calmodulin-like proteins 3 and 5. Mechanistically, this keratinocytic differentiation is associated with enhanced p63 activity, leading to the upregulation of PERP. Collectively, our study demonstrates that MCC cells can adopt a keratinocytic differentiation program in response to microenvironmental cues, underscoring the remarkable phenotypic plasticity of this malignancy and the importance of the microenvironment for tumor cell characteristics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4113c46dcadcc5e605fb92cfb97a958b41a5f2" target='_blank'>
              High-resolution spatial transcriptomics uncover epidermal-dermal divergences in Merkel cell carcinoma: spatial context reshapes the gene expression landscape
              </a>
            </td>
          <td>
            Kuan Cheok Lei, Nalini Srinivas, Mitalee Chandra, Vahan Serobyan, Selma Ugurel, Daniel Hoffmann, T. Kervarrec, Weng-Onn Lui, Jürgen C. Becker
          </td>
          <td>2025-10-23</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a highly heterogeneous cancer with complex tumor–immune interactions. This heterogeneity, particularly in tumor and immune cells, complicates treatment and prognostic evaluation. Although recent studies have revealed distinct tumor cell states and immune dysfunction in HCC, the molecular basis underlying tumor aggressiveness remains poorly understood. A deeper understanding of the molecular and functional diversity of both tumor and immune cell populations, especially the identification of stem-like tumor subpopulations and immunosuppressive mechanisms, along with the development of robust prognostic biomarkers, is essential for advancing precision oncology and improving clinical outcomes. Methods We integrated three publicly available single-cell RNA sequencing (scRNA-seq) datasets from GEO to delineate the cellular architecture of the HCC tumor microenvironment. Unsupervised clustering and dimensionality reduction were employed to identify major cell types and tumor subpopulations. Functional annotation was performed using canonical markers, Monocle, CytoTRACE, and AUCell scoring. H2AFZ was identified as a candidate oncogene and validated through in vitro knockdown experiments. The interaction between T cell subsets and tumor subpopulations were further characterized. A prognostic risk model was constructed using LASSO regression. Results Six major cell types were identified in HCC TME. Tumor cells were subdivided into three distinct clusters: Tumor_C0, Tumor_C1 and Tumor_C2. Tumor_C2 showed the highest stemness, pro-metastatic activity and immunogenic cell death signatures. H2AFZ was highly expressed in Tumor_C2 and associated with poor prognosis. The knockdown of H2AFZ reduced H2A.Z protein levels, inhibited proliferation, invasion, and induced apoptosis. T cell analysis revealed five subpopulations. It was found that Tumor_C2 interacts with the proliferative and exhausted T cell subpopulations, suggesting a potential functional relationship between them. The prognostic model based on tumor_C2 transcriptomic features effectively stratified patient survival across multiple cohorts, with robust AUCs and Kaplan-Meier survival distinctions. Conclusion We identified a proliferative, stem-like tumor cell subpopulation (Tumor_C2) in HCC characterized by high H2AFZ expression, which drives tumor aggressiveness. T cell analysis revealed significant interactions with Tumor_C2. Moreover, a prognostic model based on Tumor_C2 features effectively stratified patient survival across multiple cohorts. Together, these findings highlight potential therapeutic targets for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7024320dd03972a4da66c548f19e56393e35ae99" target='_blank'>
              Single-cell transcriptomics identifies an H2AFZ-driven proliferative tumor subpopulation associated with poor prognosis in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jihui Huo, Tao Yang, Kai Lei, Zeyao Wang, Zebin Chen, Qi Zhou
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98126a0a8ef060e1442a8189ca2cc50b5b3d5b2e" target='_blank'>
              Integrated cell atlas and tumoroids chart pancreatic cancer therapeutic targets
              </a>
            </td>
          <td>
            Quan Xu, Ryo Okuda, Bruno Gjeta, Christoph Harmel, Marina Signer, Matilde Lucioli, Małgorzata Santel, M. Seimiya, Cinzia Esposito, Karolina Guja-Jarosz, Ashley Maynard, S. Morinaga, Y. Miyagi, Tomoyuki Yamaguchi, Y. Ueno, Salvatore Piscuoglio, Daniel J. Müller, Hideki Taniguichi, Barbara Treutlein, J. G. Camp
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f17bf1cd2506f6b324c29c6b65cf368119834626" target='_blank'>
              Decrypting cellular engagement and recruitment from spatially resolved transcriptomics
              </a>
            </td>
          <td>
            Jia Yao, Ji Seon Shim, James Zhu, Sung Wook Kang, K. McCord, Woo Yong Chang, Jong Min Choi, Claire Lee, , William H. Hudson, Bryan M Burt, Tao Yue, Changwei Li, Hyun-Sung Lee, Tao Wang
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fa5a90466cca365ac23cb5111624676a9f4bea9" target='_blank'>
              Deciphering Cellular Ecosystems Driving Tumor Progression and Immune Escape from Spatial Transcriptomics and Single-Cell with COMPOTES
              </a>
            </td>
          <td>
            L. Herpin, Anaïs Chossegros, Roberta Codato, Josep Monserrat Sanchez, Jean El Khoury, Simon Grouard, V. Ducret, Alex Cornish, Baptiste Gross, Elodie Pronier, C. Hoffmann, Alberto Romagnoni, Eric Y. Durand, Almudena Espin Perez, Quentin Bayard
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696e778534089692f9510070ab5b7ccf3c2a11f2" target='_blank'>
              Single-cell spatial mapping reveals dynamic bone marrow microarchitectural alterations and enhances clinical diagnostics in MDS
              </a>
            </td>
          <td>
            Ryan Nachman, Aleksandra Kopacz, Caitlin Unkenholz, Jian Chai, Arvin Ruiz, Itzel Valencia, Jeanne Jiang, F. Socciarelli, Jiwoon Park, Christopher E. Mason, Ling Zhang, D. Sallman, Gail J. Roboz, Pinkal M. Desai, Justin Kaner, Joshua A Fein, Monica L Guzman, Neal Lindeman, Amy Chadburn, Madhu M Ouseph, Paul Simonson, Julia Geyer, Giorgio Inghirami, Shahin Rafii, David Redmond, Sanjay Patel
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is an aggressive neuroendocrine malignancy characterized by rapid onset of chemoresistance and poor clinical outcomes. Transcriptional heterogeneity among treatment-naïve SCLC tumors underlies four transcriptional subtypes, each with distinct clinical vulnerabilities. Though previously hypothesized to delineate a distinct subtype, expression of YAP1 is largely absent from treatment-naïve, pure SCLC. To characterize relapsed SCLC, circulating tumor DNA, circulating tumor cells, and core needle biopsies from SCLC patients and preclinical models following resistance to standard-of-care therapies were analyzed. In contrast to treatment-naïve SCLC, these analyses reveal an emergent YAP1-positive cell population that coincides with treatment resistance. These YAP1-positive cells exhibit characteristics of drug tolerant persister cells, including senescence, stemness, and plasticity, as YAP1 positive cells largely abandon features characteristic of SCLC to adopt those of large-cell neuroendocrine carcinoma (LCNEC). As a result of this SCLC-like to LCNEC-like evolution, YAP1-positive cells lack several clinically relevant SCLC surface targets (i.e., DLL3, SEZ6), but are enriched for others (i.e., B7-H3, TROP2). We propose a model where YAP1 expressing cells emerge with SCLC treatment resistance and characterize a tenacious subpopulation capable of diverging from the treatment naïve lineage and adopting features to evade therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df773895d5d3bb014e2dd4dc6276d432f2994491" target='_blank'>
              YAP1 defines an emergent, plastic population of relapsed small cell lung cancer
              </a>
            </td>
          <td>
            C. Stewart, K. Ramkumar, Runsheng Wang, Yuanxin Xi, A. Halliday, L. Diao, Qi Wang, Alejandra G Serrano, Sarah M. Groves, Simon Heeke, Azusa Tanimoto, L. Kaiser, Whitney E Lewis, M. Bose, Pedro Rocha, L. Karacosta, Vito Quaranta, Jing Wang, Julie George, Luisa Maren, Solis Soto, Bingnan Zhang, J. Heymach, L. Byers, C. Gay
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd437113833456e38aa31e8991ccb7e3780bc179" target='_blank'>
              Integrated 3D Light-Sheet and 2D Multiplex Imaging for Deep Histological Profiling of a Somatic Mouse Glioblastoma Model
              </a>
            </td>
          <td>
            M. Tiveron, Nathalie Coré, Kevin Bigott, Yliana Hurriaux Fontana, Maria Caccavalle, Lena Vilvandre, Fabio Al Yassouri, Victoria Schöppel, Silvia Rüberg, M. Jungblut, Dominique Figarella-Branger, A. Tchoghandjian, A. Bosio, Harold Cremer
          </td>
          <td>2025-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbc4db46760047f2ef427d64a22a5e7a556b7044" target='_blank'>
              Human cortico-vascular assembloids reveal a CELF2-AHNAK-dependent switch from neuronal to endothelial tropism in glioblastoma cells
              </a>
            </td>
          <td>
            M. Bertacchi, Annie Ladoux, Alexa Saliou, Gwendoline Maharaux, Clément Peux, L. Turchi, Béatrice Polo, Hervé Chneiweiss, E. El-Habr, M. Junier, Fabien Almairac, Fanny Burel-Vandenbos, Michèle Studer, T. Virolle
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is highly heterogeneous, with five-year survival rates dropping from $\sim$90% in localized disease to $\sim$15% with distant metastases. Disease progression is shaped not only by tumor-intrinsic alterations but also by the reorganization of the tumor microenvironment (TME). Metabolic, compositional, and spatial changes contribute to this progression, but considered individually they lack context and often fail as therapeutic targets. Understanding their coordination could reveal processes to alter the disease course. Here, we combined multiplexed ion beam imaging (MIBI) with machine learning to profile metabolic, functional and spatial states of 522 colorectal lesions with single-cell resolution. We observed recurrent stage-specific remodeling marked by a lymphoid-to-myeloid shift, stromal-cancer cooperation, and malignant metabolic shifts. Spatial organization of epithelial, stromal, and immune compartments provided stronger stratification of disease stage than tumor-intrinsic changes or bulk immune infiltration alone. To systematically model these coordinated changes, we condensed multimodal features into 10 latent factors of TME organization. These factors tracked disease progression, were conserved across cohorts, and revealed frequent multicellular metabolic niches and distinct, non-exclusive TME trajectories. Our framework MuVIcell exposes the elements that together drive CRC progression by grouping co-occurring changes across cell types and feature classes into coordinated multicellular programs. This creates a rational basis to therapeutically target TME reorganization. Importantly, the framework is scalable and flexible, offering a resource for studying multicellular organization in other solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884e760c0de1e86aa10844e17508931cd572069b" target='_blank'>
              Robust multicellular programs dissect the complex tumor microenvironment and track disease progression in colorectal adenocarcinomas
              </a>
            </td>
          <td>
            Loan Vulliard, Teresa Glauner, Sven Truxa, Miray Cetin, Yu-Le Wu, R. Simon, Laura Behm, Jovan Tanevski, Julio Saez-Rodriguez, G. Sauter, Felix J. Hartmann
          </td>
          <td>2025-10-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Spatial context is a fundamental driver of intra-tumor heterogeneity in glioblastoma (GBM), shaping the molecular landscape, cellular composition, and therapeutic vulnerabilities. While single-cell profiling has illuminated the diversity of GBM cell states, a critical gap remains in understanding how local microenvironments orchestrate gene expression programs and cellular plasticity within intact tumors. To address this, we analyzed spatial transcriptomics data from 44 GBM specimens, comprising over 128,000 tissue spots, to systematically map the biological programs and cellular architecture of distinct tumor niches. Leveraging machine learning, we derived a unified batch-adjusted data embedding that captured the local tissue context of each spot across all slides, allowing the study of spatially continuous rewiring of molecular programs. This approach enabled us to identify ligand–receptor interactions and transcription factors (TF) with spatially localized activity across tissue slides. Using cross-correlation metrics, we identified multiple TF modules specifically active in certain environments. Notably, the perivascular region exhibited the most diverse transcriptional activity, with multiple TF modules associated with tumor growth, division, and migration processes. Additional modules were uniquely enriched at the leading edge and hypoxic regions, as well as at the tumor–necrosis interface. Importantly, we identified a unique module active at the tumor core–leading edge interface, with corresponding spots enriched for alternative splicing processes, suggesting this could be a core driver of the invasive tumor phenotype. Dynamic gene network inference along a continuous hypoxic gradient uncovered context-specific hub genes and TFs that facilitate tumor metabolic adaptation and cell state shifts. These findings underscore the immense spatial heterogeneity of GBM, characterized by context-specific molecular rewiring of tumor cells driven by complex cellular interactions. Our study not only reveals new vulnerabilities but also highlights the need for spatially targeted combination therapies to disrupt the adaptive spectrum of cancer cell states in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fddeac5d22fe2de68ff3fa47ac318949e6c8ee0" target='_blank'>
              TMIC-118. Identifying spatially-localized biological programs and molecular vulnerabilities in glioblastomas
              </a>
            </td>
          <td>
            V. Ravikumar, Reva Kulkarni, A. Maddox, Arushi Tripathy, Emre Kiziltug, Dah-Luen Huang, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Organoid technology has emerged as a revolutionary tool in cancer research, offering physiologically accurate, three-dimensional models that preserve the histoarchitecture, genetic stability, and phenotypic complexity of primary tumors. These self-organizing structures, derived from adult stem cells, induced pluripotent stem cells, or patient tumor biopsies, recapitulate critical aspects of tumor heterogeneity, clonal evolution, and microenvironmental interactions. Organoids serve as powerful systems for modeling tumor progression, assessing drug sensitivity and resistance, and guiding precision oncology strategies. Recent innovations have extended organoid capabilities beyond static culture systems. Integration with microfluidic organoid-on-chip platforms, high-throughput CRISPR-based functional genomics, and AI-driven phenotypic analytics has enhanced mechanistic insight and translational relevance. Co-culture systems incorporating immune, stromal, and endothelial components now permit dynamic modeling of tumor–host interactions, immunotherapeutic responses, and metastatic behavior. Comparative analyses with conventional platforms, 2D monolayers, spheroids, and patient-derived xenografts emphasize the superior fidelity and clinical potential of organoids. Despite these advances, several challenges remain, such as protocol variability, incomplete recapitulation of systemic physiology, and limitations in scalability, standardization, and regulatory alignment. Addressing these gaps with unified workflows, synthetic matrices, vascularized and innervated co-cultures, and GMP-compliant manufacturing will be crucial for clinical integration. Proactive engagement with regulatory frameworks and ethical guidelines will be pivotal to ensuring safe, responsible, and equitable clinical translation. With the convergence of bioengineering, multi-omics, and computational modeling, organoids are poised to become indispensable tools in next-generation oncology, driving mechanistic discovery, predictive diagnostics, and personalized therapy optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c3effe5e0ade80f8d6b3e4aa8e2c7994edd86eb" target='_blank'>
              Organoids as Next-Generation Models for Tumor Heterogeneity, Personalized Therapy, and Cancer Research: Advancements, Applications, and Future Directions
              </a>
            </td>
          <td>
            Ayush Madan, Ramandeep Saini, Nainci Dhiman, Shu-Hui Juan, M. K. Satapathy
          </td>
          <td>2025-10-08</td>
          <td>Organoids</td>
          <td>2</td>
          <td>10</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, Michal R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Cancer is not composed of a single cell type but rather a complex community of cancer cells, immune cells, and supporting stromal cells that communicate with each other. These cellular conversations shape how each cancer grows, spreads, and responds to treatment. In this study, we compared single-cell sequencing data from seven different human cancers to explore how these cell interactions differ among tumor types. We found that pancreatic cancer contains many neutrophils, a type of immune cell that interacts mainly with other immune cells, while liver cancer lacks the usual fibroblast support cells. In contrast, esophageal and breast cancers were rich in fibroblasts that send growth signals to tumor cells, and thyroid cancer retained genes that may slow tumor progression. These differences help explain why some cancers behave more aggressively than others. Our findings provide a clearer picture of how the tumor environment varies among cancers and may guide the development of new strategies to treat solid tumors by targeting their surrounding cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95ce9b5897ed438ba40418ac1eb1c9c140e7a59a" target='_blank'>
              Comparative Oncology: Cross-Sectional Single-Cell Transcriptomic Profiling of the Tumor Microenvironment Across Seven Human Cancers
              </a>
            </td>
          <td>
            Riku Okamoto, Kota Okuno, Akiko Watanabe, Kanako Naito, Hiroyuki Minoura, Shumpei Shibaki, Kyonosuke Ikemura, Keiko Oki, Yu Kuroda, Shiori Fujino, Yusuke Nie, Nobuyuki Nishizawa, Eiichiro Watanabe, Mariko Kikuchi, Koshi Kumagai, Takahiro Yamanashi, Hiroshi Katoh, Hajime Takayasu, Takeo Sato, Takafumi Sangai, Yusuke Kumamoto, T. Naitoh, Naoki Hiki, Keishi Yamashita
          </td>
          <td>2025-10-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Neuroblastoma, a deadly pediatric cancer derived from sympathetic ganglia of the peripheral nervous system, frequently metastasizes, driving poor outcomes. Primary neuroblastomas are well-characterized, but the mechanisms underlying metastasis remain poorly understood. Here, by applying single-cell and spatial multi-omics analyses to primary and metastatic tumors, we found that lymph-node metastases in high-risk neuroblastomas display distinctive cellular heterogeneity and plasticity, marked by mesenchymal-like and stem-like states and heightened epithelial-to-mesenchymal transition activity compared to primary adrenal tumors. Additionally, compared to primary adrenal masses, the metastatic niche display increased immunosuppressive myeloid programs, heightened immune checkpoint signaling, and lymphocyte exhaustion, which are indicative of immune evasion and dysfunction. Notably, metastatic neuroblastomas show elevated eIF4F translation machinery and XPO1 levels. Dual inhibition of eIF4A and XPO1 synergistically halted tumor growth and prolonged survival in xenograft models. Together, our multi-omics studies reveal the molecular and cellular plasticity that contributes to therapy resistance and highlight exploitable therapeutic vulnerabilities in high-risk metastatic neuroblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03cc2aa2a287dcfe2c1cd1643687b026d41744b" target='_blank'>
              Single-cell and Spatial Omics Reveals Region-Specific Plasticity and Therapeutic Vulnerabilities in Metastatic High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Lai Man, Natalie Wu, Janet L. Oblinger, Dazhuan E Xin, Rohit Rao, Feng Zhang, Mike Adam, Oscar Lopez-Nunez, Sara Szabo, D. Allmen, Long-Sheng Chang, Brian D Weiss, Q. R. Lu
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Small cell carcinoma of the esophagus (SCCE) is a rare and highly aggressive malignancy with limited therapeutic options and poor prognosis. The paucity of clinical specimens and lack of established experimental models have hindered a comprehensive understanding of its cellular heterogeneity and tumor microenvironment. Methods We performed single-cell RNA sequencing on SCCE samples, and integrated them with publicly available scRNA-seq datasets from esophageal squamous cell carcinoma (ESCC), esophageal adenocarcinoma (EAC), and adjacent normal tissues (NT) from ESCC and EAC cases. An integrative transcriptomic analysis was conducted to identify cell types, infer malignant states, reconstruct differentiation trajectories, evaluate immune landscapes, and investigate fibroblast subtypes and cell–cell communication networks. Results SCCE tumors were characterized by a predominance of malignant epithelial cells and exhibited a profoundly immunosuppressed phenotype, with reduced immune infiltration and widespread downregulation of immune checkpoint genes. Malignant epithelial cells showed pronounced chromosomal instability and were classified into three transcriptionally distinct subtypes with divergent differentiation trajectories. The tumor microenvironment featured a complex stromal compartment, with enrichment of extracellular matrix fibroblasts (eCAFs) characterized by elevated ELF3 regulatory activity, and collagen-driven signaling predominantly mediated by inflammatory CAFs (iCAFs). SCCE also showed the most intricate cell–cell communication network among esophageal cancer subtypes. Conclusion Our single-cell atlas offers a detailed view of the cellular heterogeneity and microenvironmental complexity of SCCE, highlighting its distinct tumor architecture, immune exclusion, and stromal reprogramming. These findings provide a valuable resource for understanding SCCE biology and form a basis for future mechanistic and exploratory biological investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db8bbf656956c2fda0265e682ad202bf970eea8" target='_blank'>
              Single-cell transcriptomic analysis reveals epithelial and microenvironmental heterogeneity in small cell carcinoma of the esophagus
              </a>
            </td>
          <td>
            X. Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f71326f104bcdec5d1db3415b5ee11d22abad15" target='_blank'>
              Distinct tumor immune microenvironmental (TIME) landscapes drive divergent immunotherapy responses in glioblastoma
              </a>
            </td>
          <td>
            Linqian Weng, Krish Skandha Gopalan, Mélanie Guyot, M. Vanmechelen, Pouya Nazari, M. Duhamel, Clelia Donisi, Carla Pallarés-Moratalla, A. Garg, K. White, A.T. Byrne, Diether Lambrechts, F. De Smet, G. Bergers
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCLs) are a heterogeneous group of malignancies that can arise in lymph nodes or extranodal locations, including immune-privileged sites. Here, we applied highly multiplexed spatial transcriptomics and proteomics together with genomic profiling to characterize the immune microenvironment architecture of 78 DLBCL tumors. We define seven distinct cellular niches, each characterized by unique cellular compositions, spatial organizations and patterns of intercellular communication associated with niche-specific phenotypes of both T cells and tumor B cells. Among these, DLBCLs from immune-privileged sites showed abundant T cell infiltration into diffuse niches, where immune cells were intermixed with tumor B cells and bore transcriptional hallmarks of activation and effector function, suggesting that they may be primed for anti-tumor immunity. Spatial characterization of the DLBCL immune microenvironment, therefore, reveals cellular niches that foster divergent patterns of cell–cell interactions contributing to the phenotypic heterogeneity of both niche-resident tumor and immune cells. Analysis of the immune microenvironment of diffuse B cell lymphomas using spatial transcriptomics, proteomics and genomics highlights discrete cellular niches with divergent patterns of cell–cell communication that contribute to the phenotypic heterogeneity of both tumor and immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe40c75d4c6f7843c947dbae95bbc2601c4d486" target='_blank'>
              Multi-modal spatial characterization of tumor immune microenvironments identifies targetable inflammatory niches in diffuse large B cell lymphoma
              </a>
            </td>
          <td>
            Yibo Dai, A. Kizhakeyil, Dai Chihara, Xubin Li, Yunhe Liu, T. P. Sainz Zuniga, A. Wilson, Jared Henderson, Daniil Vibe, Arman Petrosyants, Connor Jacobson, Alexander Sarachakov, K. Nomie, K. Kryukov, Aleksander Bagaev, Ayushi Chauhan, J. Westin, Christopher R. Flowers, F. Vega, Linghua Wang, Michael R Green
          </td>
          <td>2025-10-21</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) remains a leading cause of cancer mortality, with liver metastasis being the principal determinant of poor prognosis, but the spatial mechanisms orchestrating metastatic niches remain elusive. Method To dissect the molecular and spatial dynamics of CRC progression, we constructed an integrative atlas using 35 single-cell RNA-seq datasets and spatial transcriptomics from primary tumors, liver metastases, and matched normal tissues. Malignant epithelial subpopulations were stratified via inferCNV and CytoTRACE analyses. Stromal-tumor interactions were dissected using CellChat and NicheNet, with functional validation through in vitro co-culture and immunohistochemistry. Result We identified a transcriptionally distinct epithelial subpopulation, termed high-malignancy CRC (High-M CRC), enriched in metastatic lesions and characterized by enhanced stemness, MYC-driven transcriptional activity, and glycolytic reprogramming. Stromal-tumor interaction analyses revealed that cancer-associated fibroblasts (CAFs), particularly matrix CAFs (mCAFs), promote malignant progression via the HGF-MET-MYC signaling axis. Spatial transcriptomic mapping confirmed the physical proximity and molecular co-localization of High-M CRC cells and mCAFs, along with enriched glycolysis and MYC expression at the cell-cell interface. In vitro functional validation demonstrated that CAF-derived HGF activates MET-MYC signaling in CRC cells, enhancing their invasion and proliferation—effects reversible by MET knockdown. Conclusion We unveil a spatially organized metabolic niche driven by stromal-tumor HGF-MET-MYC signaling. These findings offer novel insights into the stromal-tumor interaction and suggest actionable targets for therapeutic intervention in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dc67b8fc38aa04a6e00b35ef87b6cb65aaaec9" target='_blank'>
              A spatially resolved single-cell landscape of colorectal cancer liver metastasis reveals a stromal-tumor glycolytic signaling interaction
              </a>
            </td>
          <td>
            Jiahui Chen, Zukai Wang, Bingwang Zhu, Guoxian Guan
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e21c50186a7f4c02dcda40cb1f09f818f40176" target='_blank'>
              Histology and spatial transcriptomic integration revealed infiltration zone with specific cell composition as a prognostic hotspot in glioblastoma
              </a>
            </td>
          <td>
            L. Fidon, M. Lubrano, C. Hoffmann, Esther Baena, C. Thiriez, Luis Cano Ayestas, A. Cornish, E. Madissoon, V. Ducret, Christian Esposito, Eric Y. Durand, Benoît Schmauch, Elodie Pronier, Alberto Romagnoni, Almudena Espin Perez, M. Eckstein, R. Gottardo, T. Maragkou, M. Shelan, Almoatazbellah Youssef, Spencer S Watson, Marc Sanson, F. Bielle
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="In recent years, spatial transcriptomics (ST) has emerged as a groundbreaking technology with the potential to transform and accelerate our understanding of cellular crosstalk. While single-cell approaches have uncovered an unexpected level of cellular heterogeneity in both healthy and diseased tissues, they remain limited in their ability to capture cellular interactions in the native microenvironment. ST techniques bridge this gap by preserving anatomical information, enabling a direct investigation of spatially defined cellular interactions. This feature is particularly relevant in tissues such as skeletal muscle, where syncytial myofibers coexist with a heterogeneous set of interstitial cell populations. Spatial localization is a key factor during muscle regeneration, particularly as stem cell progression is driven by complex interactions between resident and recruited cell populations. Understanding these spatial dynamics is therefore critical to better characterize the fundamental mechanisms of muscle repair and identify aberrant signaling pathways of chronic injury or impaired regeneration. In this review, we will explore the various types of ST techniques, provide a brief summary of the available analytical tools, and highlight recent advancements in the skeletal muscle field enabled by the application of ST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1289a819036649b0e45823c7b193207d283e3d73" target='_blank'>
              Across the space: applications of spatial transcriptomic technology in healthy and diseased muscle
              </a>
            </td>
          <td>
            Laura Virtanen, C. D’Ercole, Lorenzo Giordani
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT The metabolic programs of immune cells influence their activation, differentiation, and effector functions. While much of immunometabolism has focused on cell‐intrinsic regulation, it is now clear that metabolic activity is profoundly influenced by the surrounding tissue environment. In tumors and other inflammatory settings, immune cells are shaped by nutrient gradients, hypoxia, and immunoregulatory metabolites, factors that are spatially heterogeneous and often poorly captured by traditional methods. This review highlights recent technological advances that enable spatially resolved analysis of immune metabolism, with an emphasis on multimodal integration and cancer as a model system. Mass spectrometry imaging (MALDI, DESI), high‐resolution platforms like SIMS, and vibrational imaging approaches such as Raman microscopy enable direct visualization of metabolites in tissue. Transcriptomic and proteomic data can be used to infer metabolic states, and computational models are being developed to integrate these diverse data layers. Together, these technologies are transforming the study of immunometabolism from dissociated cells to the intact tissue context. Key challenges remain in resolution, annotation, and data integration, but spatial immunometabolism holds particular promise for illuminating mechanisms of immune regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4ef9dd08ead570796d0b86fc9dc291d59a2835" target='_blank'>
              Spatial Immunometabolism: Integrating Technologies to Decode Cellular Metabolism in Tissues
              </a>
            </td>
          <td>
            F. Hartmann
          </td>
          <td>2025-11-01</td>
          <td>European Journal of Immunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Penile squamous cell carcinoma (PSCC) is a malignancy characterized by a poor prognosis and lack of reliable biomarkers, presenting considerable therapeutic challenges. Tertiary lymphoid structures (TLSs) are critical modulators of antitumor immunity; however, the immunological dynamics, particularly the roles of B cells and their interactions with naive T cells in PSCC tissues, remain inadequately understood. This study integrates transcriptomic approaches, including spatial transcriptomics, single‐cell sequencing (scRNA‐seq), and bulk RNA sequencing (bulk RNA‐seq), and immunohistochemistry to elucidate the immune architecture of and functional mechanisms within TLSs. The results reveal a positive correlation between TLS density and patient survival, with CD74⁺ B cells tending to be enriched during early TLS formation. These cells exhibit strong immune activation and a propensity to differentiate into plasma cells. By engaging with naive T cells through HLA‐DRA via ligand–receptor interactions, CD74⁺ B cells activate transcription factors, including NFKB1, NFKB2, NFATC1, NFATC2, FOS, and RUNX1, in naive T cells, thereby enhancing the immune response. Consequently, CD74⁺ B cells represent a compelling biomarker for and therapeutic target of PSCC, offering profound insights into the immunological mechanisms that drive PSCC progression and response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f775dcc4d8e646b4cd53d88e405d27443da1f7f" target='_blank'>
              Spatial Transcriptional Dynamics of CD74⁺ B Cells in Tertiary Lymphoid Structures Drive Immune Evolution in Penile Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Ting Xue, Chuangzhong Deng, Jingya Liu, Ru Yan, Jing Li, Xiheng Hu, Xueying Li, Xiao Xiao, Jietian Jin, Hongzhen Tang, Desi Chen, Zihan Zuo, Yujie Liang, Dongbin Wang, Bonan Chen, Hui Han, Zaishang Li
          </td>
          <td>2025-10-20</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Recent single-cell studies implicate the glutamatergic lineage of the rhombic lip (RL) as the origin of group (G) 3/4 medulloblastomas (MBs) and a stalled differentiation program as underlying their malignant transformation. It is unknown, however, if G3/4 MBs exhibit cellular plasticity as has been observed in glioma. The effect of therapy on cellular hierarchies in G3/4 MBs is unknown. We profiled N=57 G3/4 MB frozen specimens obtained from the University of California, San Francisco Brain Tumor Center and the Children’s Brain Tumor Network biobanks via single-nucleus RNA and epigenetic sequencing. This cohort included 43 diagnostic and 14 patient-matched recurrent specimens. We profiled N=32 FFPE specimens via Xenium single-cell spatial transcriptomics. We performed genetic lineage tracing in intracranial implants of G3 MB cell lines derived from primary and metastatic disease using engineered barcodes, and in clinical specimens using endogenous mutations. Remarkably, xenografts from primary tumors produced similar cell types, in similar proportions to those found in treatment-naive human MBs. Moreover, the distribution of cell types found in xenografts was conserved at the level of individual clones, indicating cell-intrinsic regulation of tumor composition. Single-cell phylogenetic analysis demonstrated significant rates of dedifferentiation events from unipolar brush-like cells to RL-progenitor phenotypes. This plasticity was increased in recurrent specimens and in metastatic models compared to models and samples derived from primary disease. Samples with higher plasticity exhibited greater spatial heterogeneity, while samples with lower rates of dedifferentiation had more structured spatial organization. These studies show that G3/4 MB cellular composition is homeostatic in treatment-naive disease and conserved at the level of individual clones. Yet, these tumors exhibit plasticity that is exacerbated by therapy and affects spatial organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7150b972b545d28e4de15c0584606a43a9a4dc" target='_blank'>
              STEM-18. GENETIC LINEAGE TRACING OF MEDULLOBLASTOMA UNDER THERAPY WITH SPATIAL CONTEXT
              </a>
            </td>
          <td>
            Bohyeon Yu, Jangham Jung, Abhinav Jain, Emanuele Rosatti, Joanna Phillips, Aaron A. Diaz
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c21e29753393c78c9a7147383291954089d02fd" target='_blank'>
              KDM5-driven transcriptional noise fuels plasticity-led awakening and relapse in paediatric cancer
              </a>
            </td>
          <td>
            Alejandro Allo Anido, Cecilia Roux, Emilia Chen, Siân Hamer, Abigail Shea, Ayeh Sadat Sadr, Christie English, Charlotte Butterworth, Harvey Che, Angella Bellini, Birgit Geoerger, Gudrun Schleiermacher, L. Chesler, Michael David Morgan, Alejandra Bruna
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The mortality rate of ovarian cancer remains disproportionately high compared to its incidence. This is partly due to a high level of intra-tumoral heterogeneity that promotes disease recurrence and treatment failure. In this study, we describe degrees of heterogeneity revealed by single-cell whole genome sequencing and spatial transcriptomics of five epithelial ovarian carcinomas. At the cellular level, we describe pseudo-diploid cells that match the malignant cell population in both somatic variant and copy number patterns. At the clonal and subclonal levels, we describe diversification associated with copy number gains and whole genome doubling. In multi-clonal samples, we infer evolutionary relationships from single cell copy number, loss of heterozygosity analysis, and somatic variant detection, and correlate these with tissue histology and gene expression programs. In one sample, we identify functionally consequential copy number alterations that contribute to molecular diversity, cell proliferation, and inflammation in a minor clone that persisted without major expansion alongside a more complex major clone. In another, we describe a complex evolutionary history including a spontaneous reversion of a driver mutation in a secondary clone, which correlated with a switch in oncogenic expression programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Spatial proteomics can capture cell organization, providing deep insights into diseases and catalyze significant advancements in clinical diagnosis, treatment, and outcomes. The tumor microenvironment (TME) is an organized signaling network consisting of extracellular matrix (ECM), tumor cells, stromal cells, and immune cells. Tissue histopathology, particularly immunohistochemistry (IHC), remains the most frequently used modality to assess the TME. However, despite advances in histopathology, the complexity of the TME often leads to the loss of critical information about tumor function and clinical outcomes. New technologies for high-dimensional protein mapping within intact tissues, such as multiplexed ion beam imaging by time of flight (MIBI-TOF), are increasingly employed to reveal the functional encoding of the TME. MIBI uses secondary ion mass spectrometry to image metal-tagged antibodies in tissue samples. This allows for the simultaneous quantification of dozens of antibodies, even those present in low abundance, at a subcellular level within a single scan, offering precise insights into complex biological processes. We have utilized spatial omics tools to investigate immune tolerance in diverse contexts such as human neuropathology, peanut allergy immunotherapy, pulmonary arterial hypertension, and cancer. Here, we demonstrate the application of MIBI in large-scale cancer clinical trials to analyze panels of ∼30 biomarkers.



 2U24CA224309



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9b866f97f205cb2e4a97a39048acf0269a7554b" target='_blank'>
              Application of Spatial Proteomics for Comprehensive Immune Monitoring in Cancer Clinical Trials 9251
              </a>
            </td>
          <td>
            S. Kowli, Alex Kong, C. Fullaway, Mina Pichavant, Sean C. Bendall, Michael R. Angelo, H. T. Maecker
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma (GBM) quiescent (Q) cell populations are hypothesized to contain cancer stem-like cells (CSC) that drive tumor growth, cellular heterogeneity, and recurrence. However, GBM tumors do not neatly resolve into developmental hierarchies and Q stem-like activities are difficult to assess. Here, we evaluated tumor Q subpopulations in patient-derived GBM xenograft tumors using live cell reporters, DNA label retention assays, and single cell genomics. Compared to adult neural stems cells (NSCs), GBM Q populations contain hybrid transcriptional states composed of networks found in both dormant and activated adult NSCs, resulting in constitutive expression of key Q egress transcription factors and their targets (e.g., AP-1 and CCND1/2). As a result, even the longest Q-residing cells (∼12 days) in xenograft tumors continuously cycle and fail to enter dormant Q states. We provide evidence and hypothesize that transient Q states in primary tumors arise as part of distinct proliferative compartments rather than deterministic developmental hierarchies driven by CSC activity. We further speculate that increases in basal translation rates drive Q instability in GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038a04e70ede8fe2ebdfee87bc265630d9451dc" target='_blank'>
              Characterization of quiescent subpopulations and proliferative compartments in glioblastoma
              </a>
            </td>
          <td>
            Anca B. Mihalas, Kelly Mitchell, Sonali Arora, Samantha A. O’Connor, Anoop P. Patel, Christopher L. Plaisier, Patrick J. Paddison
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 Deciphering the spatial biology of the tumor microenvironment (TME) can inform researchers about the likelihood of therapeutic success. Of particular interest is the spatial organization of immune cells, especially T cells, which can undergo exhaustion resulting in reduced effector functions due to expression of inhibitory markers such as PD-1. Imaging Mass Cytometry™ (IMC™) technology is a quantitative multiplexed spatial imaging technique for assessment of 40-plus biomarkers simultaneously on the same tissue slide without signal amplification or autofluorescence interference. We used IMC technology to investigate the spatial distribution and phenotypic characteristics of immune cells, focusing on exhausted T cells in the TME of multiple cancers. Various IMC imaging modes revealed striking heterogeneity of the TME, including clusters of lymphocytes at the tumor margins, indicative of immune surveillance. Pixel-clustering and single-cell analyses delineated distinct tumor areas based on extent of T cell exhaustion, immune cell density, cell proliferation and stromal components. These findings illustrate the power of IMC technology to elucidate the complex spatial landscape of the TME, which may aid in developing prognostic assessments and guide personalized therapeutic strategies for cancer treatments. For Research Use Only. Not for use in diagnostic procedures.



 Tumor Immunology: Checkpoints, Prevention, and Treatment (TIPT)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b22f9c7f1b4911c7dde0700548e2e19989a2ee" target='_blank'>
              High-plex spatial analysis of T cell exhaustion using Imaging Mass Cytomerty for immuno-oncology studies 2622
              </a>
            </td>
          <td>
            Qanber Raza, T. Pfister, Yun-Jyh Chwee, Nikesh Parsotam, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cba48427c8f5b3bb0c33906a8e8e094353876bb6" target='_blank'>
              SpNeigh: spatial neighborhood and differential expression analysis for high-resolution spatial transcriptomics
              </a>
            </td>
          <td>
            Jinming Cheng, Pierce K H Chow, Nan Liu
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Motivation Understanding cell fate determination is crucial in developmental biology and regenerative medicine. Although theoretical frameworks such as epigenetic landscape and gene regulatory networks have been proposed for decades, traditional studies have often been limited by population-averaging and low-throughput techniques, which obscure the heterogeneity of individual cells and fail to provide a systematic view of cell fate control. Recent advances in single-cell technologies have provided unprecedented resolution, revealing the complexity of cell fate decisions and driving the need for more sophisticated computational methods. Results In this review, we first emphasize experimental advances, such as single-cell multi-omics, lineage tracing, and perturbation techniques, which produce novel data modalities and enable dynamic tracking of cell fate transitions. We then discuss the modeling paradigms for cell fate studies and further assess the role of emerging AI tools in perturbation modeling and discuss the potential of single-cell and spatial foundation models. Additionally, we highlight several case studies on predicting and manipulating cell fates, and discuss key challenges and future directions of the field. Availability and implementation This work generates no new software.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/980bc63f4e85bb5bf85c3dca156b0326c0914eb4" target='_blank'>
              Decoding cell fate: integrated experimental and computational analysis at the single-cell level
              </a>
            </td>
          <td>
            Yutong Zhou, Shuyang Hou, Xinhao Miao, Guangxin Zhang, Zining Li, Di Zhang, Yongjie Lin, Yihan Lin
          </td>
          <td>2025-11-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastases represent one of the hardest obstacles in cancer treatment, accounting for many cancer-related deaths. Understanding the mechanisms that drive disease progression is essential to improve patient outcomes and develop more effective therapeutic strategies. This phenomenon appears to be elicited by dormant cancer cells (DCCs), which can persist undetected for extended periods of time, entering a non-proliferative, hibernation-like state that confers resistance to conventional therapies and facilitates immune evasion. Owing to their status as a rare and energy-restricted population, combined with the limitations of current medical imaging, DCCs often evade early detection, hindering timely intervention and effective clinical management. Consequently, a critical need exists to develop high-resolution detection systems and identify specific DCC targetable biomarkers. Here, a comprehensive overview of the current understanding of DCCs is reported, with a focus on recent advancements in experimental strategies for their identification and tracking, as well as therapeutic approaches currently under clinical investigation aimed at targeting these elusive cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9d13b10d52e65e6fb3d5e89c185ee227dee670" target='_blank'>
              Decoding cancer dormancy: integrative genomic, phenotypic and live-cell imaging analysis to reveal the hidden cancer cell reservoir
              </a>
            </td>
          <td>
            G. Porcelli, Caterina D’Accardo, Francesca Angeloro, Martina Cucchiara, Paola Bianca, Vincenzo Davide Pantina, Narges Roozafzay, Chiara Modica, Miriam Gaggianesi, Sebastiano Di Bella, G. Stassi, Alice Turdo, M. Todaro
          </td>
          <td>2025-11-12</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="The spontaneous regression of cancer lesions demonstrates the potential of immune surveillance; yet, these transient events have remained inaccessible to systematic study. Using longitudinal intravital microscopy in a carcinogen-induced model of head and neck cancer, we tracked premalignant lesions within the same animals for 24 weeks at single-cell resolution. This approach revealed three trajectories: progression, stability, or regression, and enabled dissection of the immune dynamics underlying each fate. Lesion outcome was dictated by the spatial organization of myeloid-derived antigen-presenting cells: regressing lesions were characterized by dense clusters of myeloid-derived cells associated with CXCL9+/CXCL10+ expression and T cell recruitment, whereas progressing lesions displayed sparse, non-clustered infiltration. Remarkably, transient myeloid clusters arose prior to any detectable lesion formation and consistently marked regions that would later develop into premalignant lesions. These findings identify spatiotemporal myeloid organization as an early determinant of tumor fate and provide a mechanistic framework for predicting and intercepting cancer at its inception. Summary Sentence Early myeloid architecture dictates cancer fate: dense CXCL9⁺/CXCL10⁺ clusters with T-cell enrichment accompany regression, whereas sparse infiltration predicts progression. Transient pre-lesional myeloid clusters emerge at future tumor sites, revealing immune organization as an early determinant of malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59ee0ed38702dcef1f68ff3c8ff2d5ef5e4fc717" target='_blank'>
              Longitudinal Imaging of the Premalignant Tumor Microenvironment Reveals Transient Myeloid States Predictive of Tumor Fate
              </a>
            </td>
          <td>
            Thomas D. Madsen, Desu Chen, Maria Hernandez, Sarah M. Hammoudeh, Marco Heydecker, Emily Chen, Salma Abu-Elnaj, Noemi Kedei, Weiye Wang, Roberto Weigert
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 Breast cancer (BC) prognosis is fundamentally dictated by its pronounced molecular and cellular heterogeneity. While single-cell RNA sequencing (scRNA-seq) has detailed the tumor microenvironment (TME) cellular composition, a critical knowledge gap persists regarding the vascular heterogeneity of tumor endothelial cells (TECs) across different BC subtypes. Current TEC classification remains conventional, obscuring whether subtype-specific TECs possess unique transcriptomic signatures that influence treatment response, particularly to anti-angiogenic therapies. Given the pivotal role of tumor vasculature in proliferation and metastasis, and the existing research bias towards Triple-Negative BC (TNBC), a comprehensive investigation into the complexity of the vascular system is warranted. Furthermore, as lymph node metastasis (LNMT) is a major determinant of cancer mortality, distinguishing the endothelial and immune cell subtypes within LNMT microenvironments from primary tumors is essential for clarifying pathogenesis.



 To address this, we integrated scRNA-seq data from primary BC tumors, ER_LN and healthy tissues, generating a robust dataset of 98,000 cells (~ 12 samples). Employing single-cell transcriptomics, spatial transcriptomics, and immunohistochemistry, we precisely delineated the endothelial cell (EC) heterogeneity within matched tumor and peri-tumoral tissues and scrutinized their intricate interactions with immune populations.



 We identified two previously uncharacterized, tumor-enriched endothelial cell subtypes, designated EC4 and EC5, which demonstrate subtype-specific functional adaptations and prognostic significance. EC4 cells, highly prevalent across BC, are principally characterized by antigen presentation, immune cell recruitment, and pro-inflammatory signaling. Conversely, EC5 cells, also enriched in BC, exhibit robust extracellular matrix (ECM) remodeling and potent tumor angiogenesis. We further characterized the functional divergence of EC4 and EC5 within ER tumors relative to HER2 tumors and ER_LN metastases. Our analysis reveals conserved endothelial programming mechanisms across BC subtypes, coexisting with distinct TME-driven transcriptional adaptations. Importantly, interactome analysis highlighted novel and subtype-specific communications between these novel EC subsets and immune cells, particularly CD8+T cells and macrophages. Experimental validation demonstrates that ECs overexpressing APP can mediate the M2 polarization of macrophages, underscoring diverse immunomodulatory roles for EC subsets across different BC contexts.



 These findings offer critical and granular insights into the complex interplay between novel EC subtypes and the immune microenvironment in BC progression and metastasis. We establish that ECs are active and heterogeneous modulators of the TME, identifying specific therapeutic vulnerabilities within the tumor vasculature and providing a foundational blueprint for developing future precision immunotherapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67936270be24af1f65608417190d3a4e352d6fe3" target='_blank'>
              Single-cell profiling identifies heterogeneity of the immune microenvironment in healthy, primary and lymph node metastatic BC
              </a>
            </td>
          <td>
            Chuyu Liu, Feng Zeng, Shanshan Gao, Nengying Zhang, Yulan Cai
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d08a85bdb5e7908e54fe64b535408e492e8893e" target='_blank'>
              Stage-resolved Spatial Multi-omics Reveals Myeloid Niches in Human Atherosclerotic Plaques
              </a>
            </td>
          <td>
            Pengbo Hou, Huan Cai, Zhanhong Liu, Jiankai Fang, Ziyi Wang, Shiqing Wang, Qingkang Lyu, Peishan Li, Changshun Shao, G. Melino, Klaus Ley, Bo Hu, Yufang Shi
          </td>
          <td>2025-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is the 4th leading cause of cancer death in the United States as of 2024. Its tumor microenvironment (TME) contains heterogenous inflammatory and immunosuppressive areas due to varying immunophenotypes relating to immune cell infiltration and exhaustion. CD4+ T helper cells can contribute to these dynamics, but defined roles of some subsets in PDAC remain debatable. We hypothesize that effector CD4+ T cell subsets spatially influence levels of inflammation in the PDAC TME to promote tumor metastasis. To investigate these dynamics, we designed a novel sequential multiplex immunofluorescent panel for murine FFPE slides consisting of 50 markers to spatially capture immune cells with activation/exhaustion states, epithelial-to-mesenchymal phenotypes, vessel infiltration, and stromal deposition in the TME. Using FFPE slides also improves preservation of structural morphology and translation to human tumor samples. Here, we applied the panel to primary murine PDAC tumors and liver metastases compared to normal tissues, revealing heterogenous MHC-I presentation from primary tumor cells, variations in CD4+ subsets in the primary tumor, and increased lymphocyte infiltration in liver metastases, validating our panel with recent spatial and dissociative multimodal publications. Ultimately, we will apply the panel to mice deficient in specific CD4+ subsets to better elucidate the spatial role of effector T helper cells in PDAC metastases.



 Supported by UAB Hematology & Oncology Startup Fund; Hirshberg Foundation for Pancreatic Cancer Reasearch



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d46f7f6b06d7c343cfa41f8977a3f99ebd086c73" target='_blank'>
              High-dimensional spatial analysis of murine tumor microenvironment highlights the heterogeneity of primary and metastatic pancreatic cancer immune landscape 3822
              </a>
            </td>
          <td>
            Christina R Larson, Ayushi Mandloi, Jace Baines, Richard Laube, Alana M. Franceski, R. Welner, Julienne Carstens
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background : Despite the use of immunotherapy in esophageal squamous cell carcinoma (ESCC), treatment failure occurs occasionally in patients, yet the underlying mechanisms remain poorly understood. Methods : We conducted large-scale single-cell RNA sequencing (scRNA-seq) data analysis, which integrated seven independent datasets from 192 ESCC patients to yield over 440,000 high-quality single cells, to systematically characterize the tumor microenvironment (TME) landscape during ESCC progression and immunotherapy response. Additionally, we performed high-resolution spatial transcriptomics (stRNA-seq) using the 10x Visium HD platform on paired pre-and post-treatment tissues from two patients (one immunotherapy responder and one non-responder), which enhanced the findings from the scRNA-seq data and mapped therapy-induced TME at the spatial level. Multiplex immunohistochemistry was employed based on seven patients to confirm distinct patterns of intercellular crosstalk underlying differential therapeutic outcomes. Results : In scRNA-seq data, we found that B lineage cells were reduced during ESCC progression but were enriched in immunotherapy-resistant patients. Further analysis of malignant ESCC cells suggested that immunotherapy resistance might be associated with a subpopulation of tumor cells exhibiting aberrantly elevated cholesterol biosynthesis. Cell communication analysis of scRNA-seq and stRNA-seq data collectively revealed that immunotherapy resistance was linked to cellular crosstalk between cholesterol-biosynthetic tumor cells and germinal center (GC) B cells within tertiary lymphoid structures. Notably, single-cell, spatial data, and multiplex immunohistochemistry demonstrated that cholesterol biosynthesis-associated ESCC cells express elevated levels of MIF. This disrupts GC reactions by competing with the CXCL12-CXCR4 signaling axis via MIF-CXCR4 interactions, thereby impairing B cell-mediated immunity. Conclusions : MIF + tumor cells in GCs may be a biomarker for predicting immunotherapy resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8a1ee6ff72d48f3e16c31d484505ab587cac16" target='_blank'>
              MIF-expressing tumor cells mediate immunotherapeutic resistance in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Jing Song, Xiaomei Song, Yue Xie, Hong Guo, Yupeng Cun
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Understanding the heterogeneity of Neurofibromatosis type 1 (NF1)-associated tumors and delineating the natural historical evolution of cell signaling are essential for interpreting tumor initiation, preventing tumor progression from benign plexiform neurofibromas (pNFs) to malignant peripheral nerve sheath tumors (MPNSTs), and engineering effective treatments. The neural crest-derived Schwann cell precursor (SCP)-like tumor population interacts with different cells in the tumor microenvironment (TME), particularly macrophages, continually shaping the intrinsic and extrinsic NF1 tumor heterogeneity. Through integrated analyses of single-cell RNA-seq (scRNA-seq) and spatial transcriptomics, we reveal that SPP1-CD44 signaling is initiated by SCP-like tumor cells in pNF, operating through autocrine mechanisms. However, in MPNST, a distinct subset of macrophages becomes the dominant SPP1 signaling source while the SCP-like cells maintain autocrine signaling. The role of SPP1 in tumorigenesis is validated by the significantly extended survival in the MPNST mouse model with cisNf1+/-;Trp53+/-;Spp1-/- configuration. Notably, our analysis of the pre-tumor stage in the DhhCre;Nf1-/- pNF mouse model demonstrates upregulated Spp1 expression compared to control tissue in Nes+ Schwann lineage cells. Together, these findings elucidate the natural historical dynamics of SPP1-CD44 signaling during tumor initiation and progression from pNF to MPNST, and highlight the SPP1-CD44 signaling axis as a potential therapeutic target to disrupt tumor stemness properties and reprogram the immune TME in malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c2e33340c1fce56bb09c0614c9943f6b1a4d428" target='_blank'>
              Natural history of SPP1 signaling in NF1 tumors
              </a>
            </td>
          <td>
            Kimani Njoya, Huda Zayed, Li Sun, Donia Alson, Oluwatosin Aina, Sajjad Khan, Ximei Veneklasen, Nikki K. Lytle, P. Chaluvally-Raghavan, Daochun Sun
          </td>
          <td>2025-10-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The bone marrow (BM) is a complex and compartmentalized tissue where spatial context plays a critical role in regulating cell behavior, signaling, and disease progression. To capture these dynamics, we apply spatial transcriptomics using the Visium Spatial Gene Expression platform on formalin-fixed paraffin-embedded (FFPE) BM sections from both healthy and Multiple Myeloma (MM) mouse models, as well as MM patient samples. Overcoming the technical challenges of working with mineralized long bone tissue, we develop a custom analytical framework integrating spatial and single-cell transcriptomic data to map cellular composition and interactions in situ. This approach enables the spatial characterization of transcriptionally heterogeneous malignant plasma cells (MM-PC) and their surrounding microenvironments. We identify spatially distinct gene programs linked to MM pathogenesis, including signatures of NETosis and IL-17 signalling, which are reduced in MM-PC–rich regions. Additionally, a transition gradient from effector to exhausted T cell phenotype is associated with increased remoteness from MM-PC. These spatial patterns are identified in FFPE BM biopsies from MM patients with varying tumor burdens. In summary, our study demonstrates both the capabilities and limitations of Visium technology in characterizing spatially regulated mechanisms underlying MM pathogenesis. Visium Spatial technology enables the characterization of transcriptionally heterogeneous malignant plasma cells in multiple myeloma, and identify spatially distinct gene programs linked to MM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55f354b66d39731b9cc107b87037fbdf9278486f" target='_blank'>
              Characterization of the bone marrow architecture of multiple myeloma using spatial transcriptomics
              </a>
            </td>
          <td>
            E. Muiños-López, Ana Rosa López-Pérez, Laura Sudupe, A. Vilas-Zornoza, S. Sarvide, Purificación Ripalda-Cemboráin, P. Aguirre-Ruiz, P. S. Martín-Uriz, M. Larrayoz, L. Alvarez-Gigli, M. Abengozar-Muela, I. Cenzano, Miguel Cócera, Javier Ruiz, Ignacio Sancho, Azari Bantan, Aleksandra Kurowska, Jin Ye, Phillip T. Newton, Bruno Paiva, L. Tamariz-Amador, J. R. Rodríguez-Madoz, Vincenzo Lagani, Jesper N. Tegnér, Borja Saez, J. Martinez-Climent, Isabel A. Calvo, D. Gómez-Cabrero, F. Prósper
          </td>
          <td>2025-11-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="


 One of the core issues surrounding solid tumor cancers is the inability to predict patient responsiveness to therapies. Although tumor biopsies are common, our ability to leverage these precious tissues for treatment plans has been technologically limited, as simple histological methods have lacked the resolution to elucidate the highly variable therapeutic response outcomes within similar patient groups. Deciphering these complexities will require quantitative measurement of cellular behaviors, interactions, and phenotypes within intact tissues. Our lab has pushed to bridge the gap between spatial immunoprofiling and spatial transcriptomics. To that end, we have successfully combined the IBEX (iterative bleaching extends multiplexity) platform with HiPlex RNAscope to visualize high-plex protein markers and high-plex RNA transcripts with single-cell resolution. “Bex-Plex” enables us to perform an unmodified multi-cycle IBEX protocol on fixed frozen murine tissues followed by a modified HiPlex RNAscope protocol (so far up to 12 probes), to ensure nuclear-level alignment. With Bex-Plex we can deeply profile both the transcriptional and functional state of all cells in the tumor microenvironment, while retaining their critical spatial information. The power to profile at this single-cell level can empower predictions about the nuanced cellular interactions that drive therapeutic responsiveness, potentially revolutionizing our approach to therapy-resistant solid tumors.



 Supported by NIH/NIGMS K99/R00 GM147841-01



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca97c1f5a8a88ed8f8a444e5de591397fc59dc55" target='_blank'>
              Leveraging spatial imaging technologies to decode cellular interactions in pancreatic cancer 3297
              </a>
            </td>
          <td>
            Catherine Murphy, Andrea Verhagen, Samantha Kimmel, Vivien I Maltez
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300bf1ce4707bea01a2973482ed7fc72841204e2" target='_blank'>
              Genomic heterogeneity drives distinct infiltration patterns in glioblastoma.
              </a>
            </td>
          <td>
            D. Harwood, S. Artzi, V. Pedersen, A. Locallo, M. Lü, David Scheie, D. Nørøxe, N. Hammouda, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-11-29</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Phenotypic plasticity is a major factor in tumor heterogeneity and treatment resistance. In particular, cancer stem cells (CSCs) represent a small subpopulation within tumors that possesses self-renewal and tumor-forming capabilities. Understanding reprogramming, maintenance, and lineage properties of CSCs requires dedicated tools to disentangle the respective influences of phenotypic inheritance and cell-cell interactions. Here, we set up ultra-wide field microscopy to image breast cancer cell lines expressing a stemness fluorescent reporter over several days. The fluorescent reporter distinguishes three phenotypes: CSCs, cancer differentiated cells (CDCs), and intermediate/transiting cancer cells (iCCs). Spatial statistics indicate significant zonation in which CSCs cluster together and are spatially separated from CDCs, forming patterns resembling niches. Surprisingly, single-cell time series reveal spontaneous reprogramming events from CDC to CSC even in unperturbed populations. We identify that such transitions are prone to arise during the cell cycle. Moreover, lineage analysis shows that the phenotype is partially inherited from ancestor cells. However, such heredity is not sufficient to explain the spatial properties of the cell population, which also depend on cell-cell interactions. Indeed, we find that phenotypic transitions of cancer cells are influenced by the phenotypic state of neighboring cells. Reprogramming into CSCs is respectively promoted and inhibited by the presence of CSCs and CDCs in the neighborhood. Altogether, our results disentangle how phenotypic inheritance and intercellular interactions orchestrate the spatio-temporal self-organization of cancer cell heterogeneity, maintaining a subpopulation of CSCs within niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0816aa85e620c96aaa89badbcea20ed0e370fd6d" target='_blank'>
              Spatial self-organization of cancer stem cell niches revealed by live single-cell imaging
              </a>
            </td>
          <td>
            Mathilde Brulé, Anaïs Horochowska, Emeline Fontaine, Raoul Torero-Ibad, Flavie Woesteland, Marie Denoulet, Jean Pesez, E. Adriaenssens, R. Toillon, X. Le Bourhis, Benjamin Pfeuty, Chann Lagadec, F. Anquez
          </td>
          <td>2025-10-27</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) invasion is shaped by dynamic clonal evolution within the spatial and cellular architecture of the brain. To investigate this process, we integrated multiparametric MRI, whole-exome sequencing (WES), bulk RNA-seq and single-nucleus RNA-seq (snRNA-seq) from 69 IDH-wildtype GBM patients, using spatially resolved biopsies from contrast-enhancing (CE) core tumor and non-enhancing (NE) invasive regions. Phylogenetic analysis revealed distinct clonal architectures. CE-enriched clones carried truncal mutations in conanical drivers of cell cycle and RTK signaling (TP53, PTEN, EGFR, PDGFRA/B), and private alterations in DNA repair genes (CDH11, PAX7, RAD51B). In contrast, NE-enriched clones exhibited private mutations in MAPK and Hippo pathways components (STAG2, BRAF, MAP2K3, CTNNA1), suggesting region-specific evolutionary trajectories. Furthermore, snRNAseq cell-cell communication analysis uncovered enriched ligand-receptor signaling in NE regions between malignant cells and brain-resident populations, including PTN—PTPRZ1 (myeloid), NRG-ERBB4 and SLIT-ROBO (neuronal, astrocytic, and oligodendrocytic). Bulk RNAseq deconvolution confirmed increased representation of neurons and glial cells in biopsies harboring NE-specific clones. These findings define a unique population of GBM clones localized at the infiltrative margin, marked by private MAPK/Hippo mutations and enhanced intracellular communication with the surrounding brain microenvironment. The recurrent engagement of neural-glial signaling pathways suggests that these NE-associated clones co-opt normal cellular networks to support invasion, immune evasion, and persistence following surgical resection. This study establishes a mechanistic link between spatial clonal evolution and tumor-host communication at the GBM infiltrative region and provide a framework for targeting cell-cell signaling circuits that sustain invasive tumor ecosystems and escape therapeutic treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee4b73c1f939dc48d12b3b70a8dbe5ab434d2210" target='_blank'>
              CSIG-31. REDEFINING THE GLIOBLASTOMA RESECTION MARGIN THROUGH SPATIAL CLONAL AND CELLULAR ECOSYSTEM ANALYSIS
              </a>
            </td>
          <td>
            Angad S. Beniwal, Junlin Wang, Xiaoling Lu, C. Sereduk, S. Migliozzi, L. Garofano, C. Plaisier, Leland Hu, R. Barajas, Junwen Wang, N. Tran
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Immunometabolic adaptations may induce tumor immune escape and immunotherapeutic resistance, representing crucial mechanisms in cancer progression. Understanding the metabolic rewiring of tumor-infiltrating immune cells as tumors advance could enhance current immune-oncology treatments. In this study, we investigated metabolic heterogeneity in immune cells within both tumor and adjacent normal tissue using single-cell transcriptome profiling of colon cancer. We also utilized the MC38 colorectal cancer model, a commonly employed mouse tumor model, to assess the metabolic atlas of major immune cell populations in tumor and normal tissue. We examined the immunometabolic features in tumor tissue and adjacent normal tissue using public single-cell transcriptomic datasets of colorectal cancer (CRC) patients, in which myeloid cells showed dominant metabolic activity. Using a mouse tumor model, we demonstrated distinct metabolic reprogramming of major immune cell types in tumor compared to normal tissue. Specifically, we observed increased glucose and lipid uptake, along with abundant lipid accumulation in tumor-infiltrating myeloid cells, particularly macrophages. Additionally, we identified diverse mitochondrial fitness and oxidative stress levels within the tumor immune microenvironment. Macrophages exhibited metabolic fitness, CD8+ T cells displayed mitochondrial depolarization, and neutrophils showed high oxidative stress. Furthermore, we investigated immunometabolic dynamics and observed augmented metabolic activity in immune cells infiltrating progressive and late stages of tumor development. Notably, intratumoral macrophages exhibited metabolic heterogeneity, characterized by robust lipid uptake and synthesis, which correlated with a pro-tumor phenotype and poor clinical outcomes. Overall, our study unveils the heterogeneity and dynamics of metabolic properties in immune cells within the tumor microenvironment. These findings provide insights for developing therapeutic strategies that target metabolism to enhance antitumor immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a595ba7e9cdf813a9e6dc8d95963965e79eecca" target='_blank'>
              Deciphering metabolic reprogramming of immune cells within the tumor microenvironment
              </a>
            </td>
          <td>
            Yinping Wang, Weijie Chen, Zida Wang, Siwen Cai, Xinnan Zhao, Jingsi Jin, Ting Gao, Junwen Qu
          </td>
          <td>2025-10-06</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Mass cytometry (CyTOF) and Imaging Mass Cytometry (IMC) provide single-cell resolution for over 50 protein markers, enabling unprecedented exploration of tumour and immune heterogeneity. We conducted a scoping review of 61 original studies (inception–2025), spanning 17 cancer types, to map current applications, analytical strategies, and emerging biological insights. 46 studies used CyTOF alone, 12 employed IMC exclusively, and 3 combined both platforms. Median panel sizes were 33.5 markers for CyTOF and 33 for IMC. While lineage and immune checkpoint markers were universal, phospho-epitopes, metabolic enzymes, and stromal proteins appeared in more focused subsets. Most studies followed a three-step analytical workflow: (i) segmentation or gating, (ii) unsupervised clustering, and (iii) downstream spatial or functional analyses. CyTOF investigations frequently identified exhausted CD8+ T-cell subsets (e.g., PD-1+TIM-3+CD39+), suppressive myeloid populations (e.g., CD163+HLA-DR− macrophages), and metabolically reprogrammed Tregs. IMC studies uncovered spatial patterns predictive of outcome, such as tertiary lymphoid structures (TLSs) and macrophage–T cell exclusion zones. Several studies proposed predictive immune signatures or integrated CyTOF with transcriptomic or spatial datasets. We identified five recurrent immunobiological motifs, CD8+ T-cell bifurcation, CD38+ TAM barriers, TLS maturity, CTLA-4+ NK-cell signatures and metabolically defined niches, highlighting convergent axes of resistance and response. Bioinformatic pipelines converged around FlowSOM or PhenoGraph clustering, CITRUS or elastic-net feature selection, and increasingly, machine learning and agent-based spatial modelling. Collectively, CyTOF and IMC are redefining biomarker discovery, therapeutic stratification, and virtual trial design in oncology, establishing high-dimensional CyTOF as a cornerstone of next-generation precision cancer medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c751da190ee62eb1876b5c48df077ef5372fdfb0" target='_blank'>
              Pan-cancer convergence of tumour–immune microenvironment motifs revealed by CyTOF and imaging mass cytometry
              </a>
            </td>
          <td>
            Alexandre Vallée, Alexandre Drezet, Maxence Arutkin
          </td>
          <td>2025-10-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="HPV status is a key determinant of prognosis and treatment response in head and neck squamous cell carcinoma (HNSCC). To investigate how HPV influences the tumor-immune-stromal landscape, we performed high-dimensional spatial profiling, including its impact on spatial organization, tertiary lymphoid structures (TLSs), and spatially organized cellular neighborhoods. Tumor biopsies from HNSCC patients (n = 16; 7 HPV-positive, 9 HPV-negative) were stained with a multiplex immunofluorescence (mIF) panel focused on immune profiling. A deep learning-based analysis pipeline enabled the identification and phenotypic state profiling of 14 cell types. Tissues were segmented into four distinct tumor regions, and spatial neighborhoods and TLSs were identified and analyzed for differential cellular composition, activation states, and spatial interactions between HPV-positive and HPV-negative tumors. HPV-positive and HPV-negative tumors differ in their tumor microenvironment (TME) composition, tumor cell state and spatial organization. The TME of HPV-positive tumors exhibited a greater abundance of activated lymphocytes, B- and T-cell-enriched spatial neighborhoods, and PD-1–PD-L1 interactions within the tumor area, whereas HPV-negative tumors were dominated by fibroblast- and macrophage-rich niches. T- cells in HPV-positive tumors showed greater activation across neighborhoods and areas, while in HPV-negative tumors T- cells demonstrated enrichment of exhaustion and terminal differentiation markers such as PD-1 and CD57. HPV-positive tumor cells had increased IDO1, HLA-DR, and Ki67 positivity, whereas HPV-negative tumor cells were more frequently CD44 positive, reflecting a more stem-like phenotype. Importantly, TLSs in HPV-positive tumors were located closer to the tumor area and enriched in activated immune cells, including ICOS+ CD4 T- cells, memory T- cells, and CD21+ B- cells. In contrast, TLSs in HPV-negative tumors were more distant and enriched for immunosuppressive populations such as PD-1+/PD-L1+ Tregs and macrophages. HPV status defines distinct spatial immune architectures in HNSCC. HPV-positive tumors harbor immune-activating TLSs and cellular neighborhoods that support antitumor immunity, whereas HPV-negative tumors exhibit suppressive niches and stromal dominance. These findings highlight TLSs, particularly their proximity and composition, as key features of the HPV-stratified TME and potential biomarkers for immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec5a252b8e081f817dffff9e37e6b264cbbcf5" target='_blank'>
              Spatial profiling of HPV-stratified head and neck squamous cell carcinoma reveals distinct immune niches and microenvironmental architectures
              </a>
            </td>
          <td>
            E. Markovits, Dmytro Klymyshyn, Roni Froumine, Hailing Zong, M. Mints, Sangeetha Mahadevan, Kenneth Bloom, Jamie G. Bates, Gareth J. Thomas, Lauri Diehl, Oscar Puig, Abhishek Aggarwal
          </td>
          <td>2025-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Metastasis remains the principal cause of cancer-related mortality, yet its distribution across organs is far from random. Instead, tumor cells exhibit organotropism, a consistent preference for colonizing specific distant tissues, a phenomenon shaped by anatomical constraints, molecular crosstalk, and microenvironmental compatibility. Far beyond mere mechanical entrapment in vascular beds, metastatic dissemination reflects a coordinated interplay between tumor-intrinsic programs and organ-specific niches. Tumor-derived extracellular vesicles, cytokines, and matrix-remodeling enzymes actively precondition distant sites through pre-metastatic niche formation, creating permissive microenvironments primed for colonization. Simultaneously, tissue-specific immune landscapes, stromal compositions, and mechanical architectures determine the fate of disseminated tumor cells, whether they are eliminated, enter dormancy, or form macrometastases. Phenotypic plasticity, metabolic reprogramming, and immune evasion mechanisms equip subclones with the capacity to exploit these unique niches. Across cancer types, reproducible patterns of organotropic metastasis not only guide clinical surveillance and therapeutic stratification but also reveal vulnerabilities in the metastatic cascade. This review synthesizes emerging mechanistic insights across anatomical, immunological, and molecular domains to construct a comprehensive framework of organotropism, highlighting therapeutic opportunities to intercept metastasis at organ-specific checkpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7663dab540302726b6f7b6337c8bf27a6cf336" target='_blank'>
              Deciphering organotropism reveals therapeutic targets in metastasis
              </a>
            </td>
          <td>
            Jhommara Bautista, Andrés López-Cortés
          </td>
          <td>2025-10-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7502a80e605765bec32ff6d8c49daea675b88242" target='_blank'>
              Stromal subsets modulate T-cell infiltration in early breast cancer
              </a>
            </td>
          <td>
            Julia Chen, Hanyun Zhang, Travis Ruan, Sunny Z. Wu, I. Slapetova, Ewan K. A. Millar, Peter Graham, Jodi Lynch, Lois Browne, Elgene Lim, Alexander Swarbrick
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32263f8daacb6eb2cfb9538393332506d52b7ecc" target='_blank'>
              Characterising cancer-stroma interactions through high-content phenotyping from microscopy time-lapses
              </a>
            </td>
          <td>
            Laura Wiggins, Jodie R. Malcolm, Karen Hogg, Peter J. O’Toole, Julie Wilson, W. Brackenbury
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a23889452eb3a744641d2c79b511978437ffd39c" target='_blank'>
              A new genetically engineered transplant model of glioma recapitulates key phenotypes of low- and high-grade gliomas
              </a>
            </td>
          <td>
            D. Forsythe, Shannon J. Oliver, A. Valkovic, Danielle Lewthwaite, Germaine Uys, Brittany Koning, David Eccles, L. Satgunaseelan, I. Hermans, S. Freytag, Jim R Whittle, Sarah A. Best, Melanie J. McConnell
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Deep learning and large language models can integrate complex datasets to uncover biological insights that are often undetectable through conventional analyses. With application to translational cancer research, these computational tools have positioned 3D patient-derived tumor avatars front and center as crucial data input sources. However, a major challenge remains: the lack of standardization in media composition in 3D patient-derived tumor models unpredictably affects cell behavior and limit the utility beyond predicting treatment responses. To address this unmet need, we developed a simple, reproducible perfusion circuit system to approximate in vivo physiology using autologous patient plasma. With peritoneal metastases and core needle biopsies across multiple tumor histologies, we demonstrate preservation of the tumor microenvironment for up to 48 hours using multi-modal interrogation techniques. With proof-of-concept experiments, we display the system’s ability to unveil complex drug-dependent biology within this time window. Standardizable, physiologically relevant platforms for 3D patient-derived tumor avatars will yield unprecedented insights through the integration of data from broad groups of patients and the use of an expanding armamentarium of artificial intelligence capabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7a2b5548999bc3e5693c3a46fe302fa33927e3b" target='_blank'>
              A simple circuit to sustain intact tumor microenvironments for complex drug interrogations
              </a>
            </td>
          <td>
            A. Sarvestani, Samantha Ruff, K. Remmert, Martha E Teke, Emily A Verbus, A. Saif, Marcial Garmendia-Cedillos, Alex Rossi, Ashley Rainey, K. Nomie, Surajit Sinha, Michael M. Wach, Carrie E Ryan, Stephanie N. Gregory, Priyanka P Desai, A. Panda, Emily Smith, Reed Ayabe, James D McDonald, Shreya Gupta, Tracey Pu, Tahsin M. Khan, Jacob T Lambdin, K. Luberice, Stephanie C Lux, Hanna Hong, Imani A Alexander, Sarfraz R. Akmal, Shahyan Rehman, Sophia Xiao, Jason Ho, Justine F. Burke, C. Hannah, Alexandra Livanova, Andrey Tyshevich, Stanislav Kurpe, Andrey Kravets, Basim Salem, Daniil Vibe, Arina Tkachukv, Anna Belozerova, A. Sarachakov, Evgeny Barykin, V. Kushnarev, E. Postovalova, A. Bagaev, Noemi Kedei, Maria Hernandez, Baktiar Karim, Donna Butcher, Jennifer Matta, Michael Kelly, Jatinder Singh, Todd D Prickett, James W Hodge, T. Andresson, Michael B Yaffe, Ethan M Shevach, David E. Kleiner, Jeremy L. Davis, Andrew M. Blakely, Jonathan M. Hernandez
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c1adf877d1b31f72e6e7bf9d982961bd5b1573" target='_blank'>
              Digital Spatial Pathway Mapping Reveals Prognostic Tumor States in Head and Neck Cancer
              </a>
            </td>
          <td>
            Julius Hense, M. J. Idaji, Laure Ciernik, Jonas Dippel, Fatma Ersan, Maximilian Knebel, Ada Pusztai, Andrea Sendelhofert, Oliver Buchstab, S. Fröhling, Sven Otto, Jochen Hess, P. Liokatis, Frederick Klauschen, Klaus-Robert Müller, A. Mock
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Intra- and inter-tumoral heterogeneity combined with immunosuppressive tumor microenvironments contribute to the poor outcomes associated with glioblastoma (GBM). Well-characterized immunocompetent models that recapitulate human GBM features are urgently needed to identify targets in the tumor microenvironment and develop novel therapeutics. Here, we used multi-omic approaches to characterize syngeneic mouse brain tumor stem cell lines in vitro and in orthotopically engrafted tumors.



 Whole-genome sequencing, transcriptomics, ATAC-sequencing, and imaging mass cytometry were used to characterize syngeneic brain tumor stem cell lines derived from Trp53+/-/Nf1+/- C57Bl6 mice. Mouse and human bulk, single-cell, and spatial sequencing datasets were analyzed for validation. CRISPR/Cas9 and shRNA were used for gene knockdowns. Tumor growth was investigated using orthotopic engraftment in syngeneic C57Bl6 mice.



 One of the syngeneic lines, mBT0309, generated tumors with histopathological characteristics of GBM. mBT0309 displayed amplification and high expression of Igf2. Copy number gains at the IGF2 locus were observed in human GBM tumors and stem cell lines. Furthermore, we determined that high IGF2 RNA expression is associated with poor survival in GBM patients. Imaging mass cytometry on mBT0309 tumors showed early infiltration of monocyte-derived macrophages, vascularization, and cell states characteristic of human GBM. Genetic targeting of Igf2 decreased in vitro cell growth, improved survival of engrafted mice, and decreased the percentage of Arginase-1+ macrophages in mBT0309 tumors.



 mBT0309 is a valuable syngeneic model for studying immunosuppression and therapeutic resistance in GBM. IGF2 offers promise as a valuable therapeutic target to combat tumor growth and immunosuppression in GBM patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3386883b6908859de06190f6d3a8fc6a7ff95c1" target='_blank'>
              IGF2 supports glioblastoma growth and immune evasion through a combination of tumor cell-intrinsic and -extrinsic mechanisms
              </a>
            </td>
          <td>
            Kyle Heemskerk, Samir Assaf, X. Hao, Shannon Snelling, M. Meode, R. Hassam, Orsolya Cseh, Smriti Kala, James Pemberton, Jennifer A. Chan, J. Cairncross, Peter Forsyth, V. Wee Yong, Reza Mirzaei, Samuel Weiss, F. Zemp, H. Artee Luchman
          </td>
          <td>2025-10-15</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance.
              </a>
            </td>
          <td>
            , Si-Jia He, Xiao-yu Jing, , Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer is a deadly disease affecting millions of people worldwide. Circulating tumor cells (CTCs) represent a critical link between primary malignancies and metastasis, acting as key players in cancer dissemination, progression, and recurrence. Although rare, CTCs offer a valuable, non-invasive window into tumor biology and the evolution of disease in patients. CTCs can exist as single cells in the circulation, but some are shed and travel in larger groups, referred to as CTC clusters. These clusters possess a greater oncogenic potential compared to individual CTCs. In this review, we aim to provide insight into the dynamic biological processes underlying CTC generation, biology, and survival, with a focus on epithelial-to-mesenchymal transition (EMT) and beyond like cancer stem cells (CSCs), cellular plasticity, and senescence. A crucial aspect of CTC biology is EMT, a process that imparts cancer cells with increased motility, invasiveness, resistance to apoptosis, and the ability to intravasate and evade the immune system. Beyond EMT the cancer cells show further plasticity, allowing epithelial tumor cells to adopt mesenchymal or hybrid phenotypes, which enables adaptation to a changing microenvironment and enhances therapy resistance. Moreover, a subset of cancer cells can acquire stem cell-like properties, including self-renewal and tumor-initiating capacity. EMT, along with processes such as dedifferentiation, contributes to the generation of cancer stem cells. In recent years, studies have also highlighted the complex and paradoxical role of senescence in CTC biology. While senescence typically results in permanent cell cycle arrest, in cancer cells it may be reversible and can promote tumor cell dormancy, immune evasion, and metastatic reactivation. By exploring the connections between CTCs, EMT, CSCs, plasticity, and senescence, we aim to shed light on the unique biology of CTCs, their metastatic potential, and their contributions to tumor heterogeneity. We hope that a better understanding of these processes will help advance the development of novel biomarkers and therapeutic targets for solid tumors beyond EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a99a639294b1ce88e2f796dfc138b9bd8dbb8609" target='_blank'>
              Circulating tumor cells: indicators of cancer progression, plasticity and utility for therapies
              </a>
            </td>
          <td>
            Tamás Richárd Linkner, Z. Nagy, Alexandra Kalmár, Eszter Farkas, Fruzsina Bányai, Nikolett Szakállas, István Takács, Béla Molnár
          </td>
          <td>2025-10-20</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02530b52b1f076b640bb9d3473208058e5838dbf" target='_blank'>
              Self-organization of tumor heterogeneity and plasticity
              </a>
            </td>
          <td>
            Carlos Pérez-González, D. Brückner, M. Di-Luoffo, Sophie Richon, R. Goswami, Meryem B Baghdadi, Florence Piastra-Facon, Neta Felsenthal, Réda Bouras, A. Fumagalli, Mirjam C van der Net, M. Gloerich, Salvatore Girardo, Jochen Guck, J. van Rheenen, J. Guillermet-Guibert, É. Hannezo, D. Vignjevic
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="To elucidate the immunological mechanisms underlying T cell exhaustion in papillary thyroid carcinoma (PTC) by integrating bulk and single-cell transcriptomic data, with a particular focus on the NECTIN–TIGIT interaction as a potential driver of immune evasion. Papillary thyroid carcinoma, while often indolent, can recur or progress in certain patients. Increasing evidence suggests that immune evasion within the tumor microenvironment (TME) plays a central role in PTC pathogenesis. Among the various immune checkpoint pathways, the NECTIN–TIGIT axis has emerged in other cancers as a key mediator of T cell dysfunction; yet, its role in PTC remains poorly characterized. We conducted an integrated analysis of publicly available bulk RNA-sequencing and single-cell RNA-sequencing (scRNA-seq) datasets from the gene expression omnibus (GEO). Differential gene expression, cell-type annotation, exhaustion marker profiling, and ligand–receptor interaction analyses were employed to investigate immune cell states and intercellular signaling. Drug sensitivity data from multiple pharmacogenomic databases were used to assess therapeutic implications. Our analysis revealed a high expression of TIGIT and associated exhaustion markers (PDCD1, CTLA4, and TOX) in PTC-infiltrating T cells, alongside a dominant presence of NECTIN2/3 in epithelial and endothelial compartments. The NECTIN–TIGIT interaction was found to suppress CD226 co-stimulatory signaling, promoting T cell exhaustion. Endothelial cells were identified as key ligands senders, creating an immunosuppressive microenvironment. Drug sensitivity correlations further suggested that TIGIT-mediated exhaustion may influence therapeutic responsiveness. This study identifies the NECTIN–TIGIT axis as a novel immunoregulatory pathway contributing to T cell exhaustion in PTC. These findings provide mechanistic insights into immune evasion and suggest NECTIN–TIGIT signaling as a potential biomarker and therapeutic target. Future wet-laboratory validation and clinical studies are needed to confirm the translational relevance of this axis in thyroid cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01aa67bc49eca70db519fdc5c0faa093c743d077" target='_blank'>
              Integrated bulk and single-cell transcriptomic profiling reveals NECTIN–TIGIT interaction underlies T cell exhaustion in papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Ahmad Golestanifar, Mohammad Reza Seyedtaghia, P. Mousavi
          </td>
          <td>2025-11-18</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Spatial tumour heterogeneity, which denotes the changes in cellular and molecular attributes across distinct locations within a tumour, significantly influences cancer diagnosis and treatment resistance. The heterogeneity of tumour cells inside a singular mass facilitates tumour development, metastasis, and the ineffectiveness of standard therapy. Comprehending the geographical distribution of tumour cells is crucial for formulating more efficient treatment regimens. Diverse methodologies are employed to investigate spatial heterogeneity, encompassing modern imaging techniques such as MRI, PET, and multiplexed imaging, alongside omics approaches including genomes, transcriptomics, and proteomics. These instruments offer insights into the tumour microenvironment and facilitate the identification of resistant subpopulations. The amalgamation of imaging and genomic data via radiogenomics has emerged as a viable methodology, providing an extensive perspective on the spatial and molecular intricacies of tumours. Principal findings reveal that spatial heterogeneity fosters medication resistance by establishing microenvironments characterised by varying oxygen levels, immunological infiltration, and genetic alterations, hence complicating the efficacy of monotherapy strategies. Hypoxic environments and immunological evasion significantly contribute to treatment resistance. Addressing geographical heterogeneity has the potential to enhance cancer treatments. By analysing the molecular and geographical characteristics of tumours, physicians can customise therapies more efficiently, minimising resistance and improving therapeutic results. This methodology signifies a vital advancement in precision medicine, providing more individualised and efficacious cancer therapies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/691858c0b45e6f3db66c28fcbc5355f8258e6fea" target='_blank'>
              Spatial Tumor Heterogeneity: The Next Frontier in Understanding Cancer Resistance
              </a>
            </td>
          <td>
            Kwesi Egyin Taylor, Hycent Jacob, T. Oladosu, Godwin Kenechukwu Nwajiugo, Motunrayo Victoria Adigun, Markus Saerimam Nzunde, C. Ugo
          </td>
          <td>2025-10-18</td>
          <td>Oncology, Nuclear Medicine and Transplantology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d38817463882bea59fae7ae417ac7b28c60ff93" target='_blank'>
              Multi-omics evaluation of cell lines as models for metastatic prostate cancer
              </a>
            </td>
          <td>
            Xueying Liu, Weixing Yu, Xiuyuan Jin, Yugang Wang, Ke Liu
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The diffusely infiltrative growth pattern of glioblastoma (GBM) is a major obstacle to effective therapy. A complex network of tumor intrinsic and microenvironmental features interact to shape the landscape of this growth pattern, but deciphering the molecular substrates of these interactions remains challenging. While single cell RNA sequencing has reshaped our understanding of GBM heterogeneity and uncovered cellular phenotypes of many cellular populations in these tumors, the spatial context of this heterogeneity could not be assessed with earlier technologies. By contrast, spatial transcriptomics platforms have the potential to map this transcriptome-wide information in situ. We recently reported the use of this technology to develop a layered model of spatial architecture in GBM and showed that hypoxia represents a key organizing feature of GBM heterogeneity. However, the limited resolution of this platform precluded assessment of whole-transcriptome features at truly cellular resolution. In this work, we now present our findings utilizing the next generation of this technology and showcase the ability to map whole transcriptome data in spatial context at true single cell resolution in primary patient GBM samples. We explore heterogeneous GBM phenotypes in the context of GBM spatial microarchitecture. Using copy number inference, we demonstrate the ability to assess clonal heterogeneity in situ and map the pattern of invasion of unique GBM clones. Finally, we discuss our progress to discover organizing principles underlying distinct patterns of GBM cell invasion. Taken together, we demonstrate the potential of single cell spatial transcriptomics to decipher the molecular heterogeneity of GBM microenvironment in situ and uncover the biology driving the aggressive infiltrative growth of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad815e1440b141545b8a8eee62737b92edef994" target='_blank'>
              TMIC-68. Deciphering the spatial architecture of transcriptional heterogeneity in glioblastoma at single cell resolution
              </a>
            </td>
          <td>
            C. Mount, Zebin Wen, Alissa C. Greenwald, S. Kovatsis, Jack Lu, D. Gerovasilis, I. Tirosh, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Current cancer therapies often fail due to a highly immunosuppressive tumor microenvironment (TME), which impedes immune cells from effectively targeting and eliminating cancer cells. Tumor-associated macrophages (TAMs) have been shown to compose up to 50% of a tumor’s mass and have been shown to exist on a polarization spectrum from anti-tumoral to pro-tumoral depending on the direct and indirect effects they have on tumors. The objective of this project is to establish pathways that control TAM tumor phenotypes. To study which TAM pathways are most amenable to manipulation to drive anti-tumor efficacy, we utilized a multi-omics approach which combines single cell RNA sequencing (scRNA-seq), spatial transcriptomics, and imaging mass spectrometry. We integrated publicly available datasets with datasets we developed from our mouse model of orthotopic pancreatic ductal adenocarcinoma (PDAC). These techniques allowed the identification of alterations in TAM metabolic networks associated with markers known to be a part of macrophage reprogramming such as Arg1, Spp1, and C1q, but the mechanism of how this shift can drive the phenotype is still to be elucidated. Future work will focus on identifying key genes involved in TAM metabolic reprogramming and assessing whether targeting these genes can modulate tumor growth by altering TAM activity. Ultimately, this research seeks to define actionable pathways that can be manipulated to enhance TAM-mediated anti-tumor responses.



 NIH K00CA274682-03



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d07758d1aaf12c832cc03f655246443f1a51f16" target='_blank'>
              Characterization of anti-tumor networks in tumor-associated macrophages 2377
              </a>
            </td>
          <td>
            Stephen Sakuma, Ming Sun, S. Kaech
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Micvotabart pelidotin (micvo) is an antibody–drug conjugate that targets the non-cellular extradomain-B of fibronectin, an extracellular matrix protein highly expressed in tumors compared to normal adult tissues. Micvo is designed to kill cancer cells, remodel the tumor microenvironment (TME), and mobilize an anti-tumor immune response. Micvo is currently in clinical development as a monotherapy (NCT05720117) and in combination with pembrolizumab for difficult-to-treat cancers (NCT06795412). Histological methods to characterize participants’ tumors before and after treatment with micvo are needed. Multiplex immunofluorescence (mIF) enables high-resolution phenotyping of tissue architecture and cellular interactions to inform biomarker development and pharmacodynamic assessments. This study aimed to establish mIF methodologies for integrated spatial assessment of tumor-immune interactions and stromal architecture to better characterize microenvironmental variation across tumor types for future use in biomarker discovery and characterization of pharmacodynamic responses to micvo. A practical method for staining six custom targets simultaneously was developed using Akoya’s Opal™ mIF kit, which employs iterative immunofluorescent labeling to visualize multiple markers within a single formalin-fixed, paraffin-embedded section while preserving spatial context. Utilizing this technology, two complementary mIF panels were designed, optimized and validated; one focused on tumor-immune interactions, and the other on key stromal features, including fibroblast subtypes and extracellular matrix components. Same-slide H&E and Masson’s trichrome staining were performed on mIF stained samples to provide additional context for tumor cell morphology and collagen distribution, respectively. A digital pathology workflow was developed to enable integrated spatial assessment of immune and stromal compartments by aligning and combining region annotations between sequential slides; positive-cell thresholds and region classifiers were guided by a board-certified pathologist. To validate this workflow for human clinical samples, each mIF panel was deployed and validated on commercially sourced whole-tissue slides of head and neck squamous cell carcinoma (HNSCC) and pancreatic ductal adenocarcinoma (PDAC), characterizing histologic and microenvironmental differences across indications. Digital pathology enabled precise spatial quantification, revealing differences in the composition and spatial organization of immune and stromal components of the TME between HNSCC and PDAC. These results demonstrate the feasibility of integrating tumor-immune and stromal mIF panels for high-resolution TME profiling, offering a scalable and customizable approach to investigate tumor heterogeneity and treatment response across solid tumors. The validated mIF workflow will be applied to clinical biopsies from micvo-treated participants to characterize pharmacodynamic changes in immune and stromal compartments.



 Justin Trickett, Sara Lewandowski, Krystal Watkins, Eila Crochiere, Victor Zota, Marsha Crochiere. Development of multiplex immunofluorescence workflows for characterizing tumor-immune and stromal compartments for pharmacodynamic assessments of solid tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A117.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3bd377e69e88173ec56ce3d1cf36113c55b24a8" target='_blank'>
              Abstract A117: Development of multiplex immunofluorescence workflows for characterizing tumor-immune and stromal compartments for pharmacodynamic assessments of solid tumors
              </a>
            </td>
          <td>
            Justin Trickett, Sara Lewandowski, Krystal Watkins, Eila Crochiere, Victor Zota, Marsha Crochiere
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype; however, clinically approved prognostic biomarkers and therapeutic options remain limited. This study aimed to investigate tumor dependency genes to identify novel therapeutic targets for TNBC. Methods Tumor dependency genes for TNBC were identified using the The Cancer Dependency Map (DEPMAP) database. The TCGA-BRCA dataset was utilized to analyze the expression, survival associations, and pathway enrichment of these genes. Single-cell datasets were employed to explore cellular trajectories and biological functions within tumor dependency gene-associated cell subpopulations. Genomic sequencing was used to investigate the somatic mutational landscape influencing the infiltration abundance of the tumor dependency-associated subpopulation. The METABRIC dataset assessed the impact of the tumor dependency-associated subpopulation on radiotherapy, chemotherapy, and combination therapy outcomes. Potential drugs were identified using the Connectivity Map (CMAP). Colony formation experiment and the CCK-8 experiment were performed to validate the biological function of gene. Results Four tumor dependency genes (TDGs) were identified. These genes were highly expressed in TNBC and associated with poor prognosis. Enrichment analysis revealed their significant involvement in cell cycle-related pathways. Single-cell analysis demonstrated that the tumor dependency-associated subpopulation (TDAS), defined by these four genes, resided at the differentiation terminus of epithelial/tumor cells and was linked to energy metabolism and cell proliferation pathways. Crucially, patients with high TDAS infiltration abundance were found to be unsuitable for surgery alone and should receive combined radiotherapy or chemotherapy. Potential therapeutic agents targeting the TDAS were screened. And in vitro experiments confirmed the cell proliferation role of candidate genes. Conclusion This study identifies four potential TNBC biomarkers for assessing TDAS abundance, providing novel insights and strategies for personalized TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c8aa1f75b0d7e0c49aabd5f19b6bf7360c1cf" target='_blank'>
              Integrating single-cell transcriptomics and whole-genome CRISPR CAS9 screen identifies a cell cluster associated with tumor dependency in triple-negative breast cancer
              </a>
            </td>
          <td>
            Su Liu, Shuo Wang, Guixin Wang, Yingxi Li, Zhigang Zhao, Yao Tian, Jun Cao
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Single-cell and spatial transcriptomics have become pivotal tools for elucidating the phenotypic heterogeneity of immune cells and identifying spatially recurrent immune niche states. These technologies have produced extensive datasets containing whole-transcriptome profiles of dissociated single cells and spatially resolved tissues. However, both approaches are subject to significant technical and biological batch effects, which complicates the assignment of cell types from single-cell data to spatial transcriptomics datasets and hinders the comparative analysis of spatial niche and region labels across tissue slices. To overcome these limitations, we developed Synchronized Deconvolution and Integration (SDI), a deep learning model designed to integrate dissociated single-cell and spatial transcriptomics data across diverse platforms and spatial resolutions. Pretrained on over 5 million dissociated cells and more than 1 million spatial spots from pan-cancer tissues, SDI demonstrates the ability to accurately project cell type labels from single-cell to spatial datasets, with adaptability to novel datasets. We show that SDI facilitates batch-unbiased deconvolution of immune cell composition and provides consistent predictions of tertiary lymphoid structures (TLS) across over 400 tissue slices from more than 10 cancer types. In conclusion, SDI enables integrated analyses linking immune cell types from both single-cell and spatial transcriptomics.



 None declared



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5508adcf16a988490ac8e07619e7513b2c493ae0" target='_blank'>
              Linking large-scale pan-cancer single-cell and spatial transcriptomics profiles using SDI 9110
              </a>
            </td>
          <td>
            Ziwei Xue, Wanlu Liu
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The global burden of cancer is rising, with treatment failures often due to the metastatic nature of late-stage malignancies. Circulating tumour cells (CTCs) are metastatic precursors shed from primary tumours, which survive in circulation, extravasate and colonise distant organs. The advent of high-throughput single-cell RNA sequencing (scRNA-seq) has revolutionised the investigation of transcriptomic landscape at single-cell resolution, enabling deep transcriptomic profiling, re-stratifying CTC subtypes and improving the detection of rare new subpopulations. Applications extend to understanding tumour microenvironments, characterising cellular heterogeneity, uncovering metastasis molecular mechanisms and improving prognosis and diagnostic strategies. A timeline of key milestones in CTC scRNA-seq research is also provided. Nevertheless, a knowledge gap remains due to unstandardised protocols and fragmented resources in CTC scRNA-seq research. We address this gap by proposing a 12-step CTC-specific scRNA-seq workflow to overcome methodological inconsistencies. This workflow spans the entire process from enrichment, single-cell sorting and sequencing to data pre-processing and downstream analyses, with a detailed compilation of data analysis tools. An in-depth discussion of the pros and cons of commonly used scRNA-seq tools is also included, specifically evaluating their suitability for CTC research. Additionally, emerging research frontiers, including the discovery of hybrid cells—fusion products of tumour and normal cells—and the integration of machine learning (ML) into scRNA-seq workflows, are explored. Future research should prioritise CTC scRNA-seq workflow standardisation, integrate ML-driven analysis and investigate rare and hybrid populations to advance metastasis research. This review supports these goals by guiding methods, informing tool selection and promoting data sharing for reproducibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d2d946c12a3009915b3b21748cb811af18ab2d4" target='_blank'>
              Single-cell RNA-sequencing of circulating tumour cells: A practical guide to workflow and translational applications
              </a>
            </td>
          <td>
            Francis Yew Fu Tieng, Learn-Han Lee, N. Ab Mutalib
          </td>
          <td>2025-10-06</td>
          <td>Cancer Metastasis Reviews</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6214e46642ec3f3780a0b2d3254221a9b00d6" target='_blank'>
              Comparative single-cell transcriptomics of orthotopic and subcutaneous gastric tumors reveal immune and stromal heterogeneity
              </a>
            </td>
          <td>
            Jinho Jang, Yoojeong Seo, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 The human brain contains a rich milieu of extracellular matrix (ECM) components that promote normal physiology and are often dysregulated in pathologies including the malignant cancer glioblastoma (GBM). We have used in situ single-cell spatial transcriptomic platforms to map the expression patterns of nearly 400 ECM-related genes in multiple GBM samples (IDH1 wild type). Our analysis identifies at least four different GBM cell populations with unique ECM expression profiles that show spatial enrichment in distinct intratumor regions. Spatial mapping also demonstrates largely non-overlapping expression signatures of various ECM components in stromal cell types, particularly in GBM vascular endothelial cells and reactive microglia/macrophages. Comparisons of GBM versus lower-grade II and III astrocytoma samples (IDH1 R132H) identifies differential expression of key ECM components, including elevated levels of select ECM glycoproteins (IGFBP2 and MGP) and ECM-affiliated proteins (ANXA1 and ANXA2). In addition, we detect spatially enriched expression of COL8A1 (collagen), LUM (proteoglycan), and POSTN (ECM glycoprotein) in perivascular stromal cells in GBM but not in lower grade tumors. Computational analysis of putative ligand-receptor interactions reveals novel communication networks between cancer cells and stromal components, particularly in regions of GBM microvascular proliferation and pseudopalisading necrosis. In summary, this comprehensive spatial map provides new insights into ECM control of tumor initiation and progression and identifies potential therapeutic targets within the GBM microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e40eadeccedca2699836ba1458071cfd535dcbfb" target='_blank'>
              TMIC-33. Single Cell Spatial Profiling of Matrisome Gene Expression in Glioblastoma
              </a>
            </td>
          <td>
            Joseph H. McCarty, Arpan De, Santiago A. Forero, Ali Pirani, John Morales
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Cancer-associated fibroblasts (CAFs) are key drivers of the mesenchymal, therapy-resistant phenotype observed in the CMS4 subtype of colorectal cancer (CRC). CAFs are abundant in the tumor stroma, where they orchestrate extracellular matrix (ECM) remodeling, activate pro-tumorigenic signaling (e.g., TGF-β, PDGF), and suppress anti-tumor immunity. These interactions are difficult to model in conventional 2D cultures, which lack the complexity of the tumor microenvironment (TME). To overcome this limitation, we utilized MCAT™ (MEDiC’s Casual and Actionable Target)—a physiologically relevant and scalable 3D platform—to model CMS4 tumor biology and identify cancer cell dependencies influenced by stroma. Using MCAT™, we performed genome-wide CRISPR knockout screens to uncover tumor vulnerabilities and resistance mechanisms emerging through tumor–stroma interactions.



 Two CMS4 CRC cell lines (HCT116, SW620) and human skin fibroblasts (BJ) were co-cultured at defined ratios and encapsulated in ECM ladden alginate hydrogel microcapsules to form 3D spheroids. Fibroblast-induced changes in cancer cell proliferation were tracked via live-cell imaging. Transcriptomic profiling was performed to assess fibroblast-driven changes. For genome-wide CRISPR screening, cancer cells expressing Cas9 were transduced with a pooled sgRNA library (∼60,000 sgRNAs targeting ∼20,000 genes). Co-cultures were maintained for four weeks to allow for functional selection under stromal influence.



 Fibroblast co-culture at a 1:3 (cancer:fibroblast) ratio significantly enhanced proliferation of both HCT116 and SW620 cells, whereas a 3:1 ratio showed no effect. qPCR analysis revealed upregulation of CMS4 markers (PDGFRA, c-KIT) in 3:1 co-cultures. Transcriptomic analysis identified ∼2,000 differentially expressed genes (DEGs) between 2D and 3D conditions. 3D co-culture significantly enriched mesenchymal traits, with upregulation of EMT markers (TWIST, VIM) and pathways related to ECM organization (COL5A1, BGN), ECM degradation (MMP2, TIMP3), and PDGF signaling—hallmarks of invasion and metastasis. Genome-wide CRISPR screening revealed 27 genes with a negative-growth phenotype and 92 genes with a positive-growth phenotype in the presence of fibroblasts. Notably, several TGF-β pathway regulators showed negative-growth phenotype only in co-culture, highlighting the stroma-specific role of this pathway. Genes involved in metabolic processes were enriched among negative hits, which could be potentially targeted for therapeutics.



 The MCAT™ platform enables physiologically relevant modeling of tumor–stroma interactions in CMS4 CRC. It supports high-throughput functional genomics to uncover context-specific vulnerabilities, providing a powerful tool for discovering novel therapeutic targets in aggressive, stroma-rich cancers.



 [1] Han, K., Pierce, S.E., Li, A. et al. CRISPR screens in cancer spheroids identify 3D growth-specific vulnerabilities.Nature 580, 136–141 (2020). https://doi.org/10.1038/s41586-020-2099-x



 Pranay Agarwal, Maya Tureez, Jojo Baek, Aashka Mehta, Kriti Bhardwaj, Ella Liu, Jina Song, Kyuho Han, Hong-Pyo Lee. MCAT™: A 3D Co-Culture Platform for Genome-Wide CRISPR Screening to Identify Stromal-Driven Vulnerabilities in CMS4 Colorectal Cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C124.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7c5230df580ab0427c0ffbb4a608fdc78d9141b" target='_blank'>
              Abstract C124: MCAT™: A 3D Co-Culture Platform for Genome-Wide CRISPR Screening to Identify Stromal-Driven Vulnerabilities in CMS4 Colorectal Cancer
              </a>
            </td>
          <td>
            Pranay Agarwal, Maya Tureez, Jojo Baek, Aashka Mehta, Kriti Bhardwaj, Ella Liu, Jina Song, Kyuho Han, Hong-Pyo Lee
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c7fc26d95585d6b0934d32560acffc12bc3781e" target='_blank'>
              Region-specific human brain organoids reveal synaptic and cell state drivers of glioblastoma invasion
              </a>
            </td>
          <td>
            Tarun N. Bhatia, Saisrinidhi Ganta, Mostafa Meselhe, Caitlin Sojka, A. Martija, Lisa Nieland, Uriel Rufen-Blanchette, Anson Sing, Alexia King, Brain Organoid Hub, A. Bhaduri, Kimberly Hoang, E. Nduom, Renee D. Read, Jeffrey Olson, Steven A. Sloan
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01df2c37dabe824163d0ae6c9689244efcafc8f6" target='_blank'>
              Multi-scale Network Topology Analysis Reveals Metabolic Reprogramming and Therapeutic Vulnerabilities in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Handan Cetin, P. Bogdan, S. Mumenthaler, Stacey D. Finley
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cancer‐associated fibroblasts (CAFs) are pivotal stromal components that shape the tumor microenvironment through their remarkable heterogeneity and dynamic functions. This review comprehensively examines the biology of CAFs, starting with their diverse cellular origins and induction mechanisms via key signaling pathways. We highlight the spatial‐temporal heterogeneity of CAF subsets and propose an updated classification system that encompasses eight functionally distinct subtypes based on recent single‐cell transcriptomic studies. The tumor‐promoting roles of CAFs are explored in depth, including their contributions to tumor behavior through novel mechanisms such as neutrophil extracellular traps formation, and multidrug resistance via metabolic reprogramming. A major focus is placed on CAF‐mediated immunotherapy resistance through immune checkpoint regulation, recruitment of immunosuppressive cells, and metabolite secretion. This review also highlights key technologies that are advancing CAF research, including patient‐derived 3D models, artificial intelligence‐enhanced spatial multiomics, clustered regularly interspaced short palindromic repeats (CRISPR)‐engineered mouse systems, and machine learning approaches for analyzing CAF heterogeneity and function. Finally, we evaluate therapeutic strategies targeting CAF activation, secreted factors, and immune crosstalk, while critically analyzing ongoing clinical trials. Overall, this review not only clarifies the biology of CAFs but also provides a translational framework for developing next‐generation stromal‐targeted therapies to overcome treatment resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1dae86be5576422570b1acdcb11b97738bdec4b" target='_blank'>
              Cancer‐Associated Fibroblasts: Origin, Classification, Tumorigenicity, and Targeting for Cancer Therapy
              </a>
            </td>
          <td>
            Chang Fan, Wenlong Zhu, Yuan Chen, Wanwan Zhu, Jin Ding
          </td>
          <td>2025-10-28</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy, poised to become the second leading cause of cancer-related deaths in North America. The paucity of neoantigen presentation in PDAC has been hypothesized to promote the development of an “immunologically-cold” tumor microenvironment (TME), driving poor clinical responses to immunotherapy. Despite the abundant immune infiltrate observed in PDAC, T cells display a limited cytotoxic capacity to kill malignant tumor cells which further diminishes throughout disease progression. Utilizing single cell RNA-sequencing coupled to T cell receptor profiling, we have characterized T cell populations in clinical specimens of inflammatory pre-malignant intraductal papillary mucinous neoplasia (IPMN) lesions and advanced stage PDAC tumors. Striking phenotypic and clonal dynamic differences were observed in both CD8 and CD4 T cell populations between IPMN and PDAC specimens which underly their distinct functional capabilities in the evolving TME. The spatial interactions and microenvironmental neighborhoods of neoantigen-reactive T cells in PDAC have yet to be explored in-depth and hold unexploited therapeutic potential. Utilizing the 10X Genomics Xenium platform with custom probes designed to hybridize with neo-antigen-specific T cell clones, we are interrogating the cellular interactions which drive T cell clonal expansion to identify therapeutically relevant vulnerabilities in PDAC malignant progression.



 Supported by National Insitutes of Health, Canadian Institutes of Health Research, and Princess Margaret Cancer Center Catalyst Grant.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f6400ccbc9c89325532880266eae31b90968879" target='_blank'>
              Defining molecular determinants of T cell spatial localization and clonal response throughout PDAC malignant progression 3030
              </a>
            </td>
          <td>
            Matthew Bianca, Sara Lamorte, Teresa Ciudad, Cristiane Naffah de Souza, Rene Quevedo, Tracy Mcgaha
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Imaging biological samples in three dimensions across scales is essential for capturing the complex spatial relationships that govern cancer initiation, invasion, and therapeutic response. As biological inquiry shifts from isolated molecular measurements toward spatially contextualized, multiomic profiling, new strategies have emerged to reconstruct tissue architecture at the whole-organ scale and subcellular resolution. These advances offer more anatomically faithful representations of tissue organization and open doors to integrating morphology with deep multiomic profiling in spatially resolved formats. As a result, we are improving our understanding of inter- and intratumoral heterogeneity and the key role of rare events and minority cell populations in tumor progression. The primary techniques used for 3D imaging of tumors include intact tissue imaging for targeted visualization of biological processes and serial sectioning for integration of diverse, multiomic platforms. In this review, we survey the major technologies used to image tumors in three dimensions, highlighting key methodologies, trade-offs, and recent innovations that make these approaches increasingly central to modern cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ec1eecfebb643d8f7b0229bfefe0ab87342355e" target='_blank'>
              Lessons from 3D Reconstruction of Cancer
              </a>
            </td>
          <td>
            Ashley L. Kiemen, Laura D. Wood
          </td>
          <td>2025-11-26</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spatially resolved transcriptomics (SRT) facilitates the study of cell–cell interactions within native tissue environments. To support method development and benchmarking, we introduce sCCIgen, a real-data-based simulator that generates high-fidelity synthetic SRT data with known interaction features. sCCIgen preserves transcriptomic and spatial characteristics and provides key interaction features, including cell colocalization, spatial dependence of gene expression, and gene–gene interactions between neighboring cells. It supports input from SRT data, single-cell expression data alone, and unpaired expression and spatial data. sCCIgen is interactive, user-friendly, reproducible, and well-documented for studying cellular interactions and spatial biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cffc0895b0c59f6ae8d74318f266f33f6e00809" target='_blank'>
              sCCIgen: a high-fidelity spatially resolved transcriptomics data simulator for cell–cell interaction studies
              </a>
            </td>
          <td>
            Xiaoyu Song, J. C. Chávez‐Fuentes, Weiping Ma, Weijia Fu, S. C. Shin, Pei Wang, Guo-Cheng Yuan
          </td>
          <td>2025-10-15</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01b5dde4ed4b216cb7cf78338e646166d480ca06" target='_blank'>
              Spatial proteomics of ovarian cancer precursors delineates early disease changes and drug targets.
              </a>
            </td>
          <td>
            Anuar Makhmut, M. Dragomir, Sonja Fritzsche, Markus Moebs, W. Schmitt, Eliane T. Taube, Fabian Coscia
          </td>
          <td>2025-11-13</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cancer cell plasticity, the dynamic ability of tumor cells to switch phenotypic states, drives tumor progression, metastasis, and therapy resistance, posing a formidable challenge to effective cancer management. This review comprehensively explores the genetic and epigenetic mechanisms underlying plasticity, highlighting how oncogenic mutations, chromosomal instability, and pathways like PI3K/AKT and RAS/MAPK enable adaptive behaviors such as epithelial-mesenchymal transition and stemness. Epigenetic regulators, including DNA methylation, histone modifications, and non-coding RNAs, amplify this adaptability, allowing rapid responses to therapeutic pressures. We examine the profound clinical implications, with plasticity fueling relapse in breast, lung, and prostate cancers, exacerbated by tumor microenvironment interactions. Challenges in detecting plastic states due to intratumor heterogeneity underscore the need for advanced diagnostics like single-cell genomics. Emerging interventions, from CRISPR-based gene editing and epigenetic therapies to patient-derived organoids and AI-driven combination strategies, offer transformative potential to disrupt plasticity and overcome resistance. This review illuminates the intricate genetic landscape of cancer cell plasticity, advocating for integrated approaches to revolutionize personalized oncology and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56a74b7b33fc882b2d862736314940182017fc5" target='_blank'>
              The Genetic Landscape of Cancer Cell Plasticity: Mechanisms, Implications for Therapy Resistance and Emerging Interventions
              </a>
            </td>
          <td>
            I. N. Monago, Nyerovwo Charity Okei, Chibuike Stephen Nzereogu, Taiwo Emmanuel Olowodasa, Tamasanni Omeiza Salawu, Ahmad Muhammad Garba, Doris Ada Ojum
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Biological and Pharmaceutical Sciences Archive</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Super-enhancers (SEs) are clusters of enhancers with potent regulatory capabilities. They play a crucial role in shaping cellular identity and driving the progression of various diseases, including cancer. SEs exhibit significant heterogeneity across different cell types and cancer subtypes. Analysis of SE landscapes can recapitulate existing classification systems and unveil novel SE-driven epigenetic subtypes. In this review, we summarized the latest advancements in cancer subtype identification based on SE-related characteristics, outlined the typical analytical workflows adopted in such studies, and explored the biological and clinical significance of SE-driven subtypes. Furthermore, we discussed the field’s key challenges and emerging technologies to highlight future research directions. SE analysis provides a robust framework for dissecting cancer heterogeneity. This approach offers novel epigenetic perspectives and support for the realization of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/285d6877af1dc808910fb0a7890277cebd5d2976" target='_blank'>
              Decoding cancer heterogeneity through super-enhancer landscapes: from subtype discovery to therapeutic opportunity
              </a>
            </td>
          <td>
            Shuyang Cai, Zhenchang Wang, Xiao Sun, Hao Huang
          </td>
          <td>2025-11-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Introduction Leiomyosarcoma (LMS) is a rare, aggressive cancer with limited treatment options at the metastatic stage. The response to immune checkpoint inhibitors (ICIs) is inconsistent, likely due to intratumoral heterogeneity, which is more pronounced in large tumours such as retroperitoneal LMS. Methods This study examined heterogeneity in four large treatment-naive LMS tumours (ten samples per tumour) by analysing immune cells, tertiary lymphoid structures (TLSs), checkpoint molecules, and cytokine secretion across different tumour regions. Results Significant region-dependent differences were observed in immune components, with TLSs present only at tumour margins and inconsistently across samples from the same tumour. Expression levels of programmed cell death 1 (PD-1) and programmed death ligand 1 (PD-L1) varied within individual tumours, and shared immune patterns were identified in specific regions, including elevated indoleamine 2,3-dioxygenase 1, absence of a particular macrophage subpopulation, and reduced PD-1 and lymphocyte activation gene 3 (LAG-3) expression at organ-adjacent margins. Anti-LAG-3 blockade altered cytokine and checkpoint molecule levels in a region-specific manner. Conclusion These findings highlight substantial intratumoral heterogeneity, which may contribute to the variable response to ICI therapy. As immune checkpoint molecule expression influences treatment eligibility, multiple biopsies from different tumour regions may be necessary to assess immune infiltration accurately and guide therapy decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f47a854ab32ca2eceb50d1b8d1469818b11328c" target='_blank'>
              Spatial immunoprofiling of retroperitoneal leiomyosarcomas reveals intratumoral heterogeneity in immune cell infiltration, checkpoint molecule expression, and tertiary lymphoid structures
              </a>
            </td>
          <td>
            Iva Benesova, Jan Balko, V. Tovazhnianska, M. Rataj, R. Lischke, J. Bartůňková, K. Kopeckova, T. Buchler, W. V. van Houdt, Yvonne Schrage, David Moura, J. Broto, Zuzana Ozaniak Střížová, A. Ozaniak
          </td>
          <td>2025-10-13</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95bb3b5b5807e6d181c0c3ffb7556186ba442136" target='_blank'>
              Nestin and SOX2 Maintain self-renewal Abilities of Different Pancreatic Cancer Stem Cell Populations.
              </a>
            </td>
          <td>
            L. Philipp, Patrick Hoffmann, L. Hattingen, Amelie Modi, Susanne Sebens
          </td>
          <td>2025-10-23</td>
          <td>Stem cell reviews and reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The tumor immune microenvironment plays a critical role in tumor progression and responses to immunotherapy. Nevertheless, its cellular complexity and heterogeneity remain incompletely understood. In this study, we employed high-resolution single-cell RNA sequencing on CD45+ immune cells isolated from ten syngeneic murine tumor models, representing seven distinct cancer types under treatment-naïve conditions, thereby enabling a comprehensive profiling of tumor-infiltrating immune cells. We identified seven principal immune cell populations and provided an in-depth characterization of T cells, NK/innate lymphoid cells, dendritic cells, monocytes/macrophages, and neutrophils. Cross-species analyses further delineated conserved immune cell states and transcriptomic features within the T cell and monocyte/macrophage compartments that are shared across syngeneic models and human tumors. To investigate the functional relevance of the predominant monocyte/macrophage compartment and the notable presence of neutrophils in syngeneic tumors, we evaluated responses to anti-PD-1 therapy across various models and analyzed the enrichment of monocyte/macrophage subsets in tumors that responded to treatment. Furthermore, we conducted neutrophil depletion experiments using anti-Ly6G antibodies, administered both as monotherapy and in combination with PD-1 blockade. Remarkably, an interferon-stimulated gene-high (ISGhigh) monocyte subset was significantly enriched in models responsive to anti-PD-1 therapy. Neutrophil depletion resulted in variable antitumor effects across models but failed to enhance the efficacy of PD-1 blockade. In summary, our single-cell profiling offered a detailed atlas of the immune microenvironment across multiple syngeneic mouse tumor models, thereby enabling rational model selection for immuno-oncology studies. We uncovered an ISGhigh monocyte subset enriched in anti-PD-1 responsive models, and showed the context-dependent effects of neutrophil depletion on tumor immunity and immunotherapy, underscoring the heterogeneity and functional divergence of immune cell sublineages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f925e5b4fe351a72672a38a0505d78e0be84e52" target='_blank'>
              Single-cell atlas of the tumor immune microenvironment across syngeneic murine models
              </a>
            </td>
          <td>
            Jia Wang, Bin Jiang, Minjuan Deng, Han Yan, Peilin Zhang, Wei Jin, Zhirong Shen
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3948f28751f8205d2389f7d1df87013dbea193d3" target='_blank'>
              RUNX1T1-HDAC Reprogramming of the HOX Code Signaling Drives a Targetable Pan-Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, Catherine Yijia Zhang, Xiao Jin, Yaru Xu, I. Y. Kim, Ping Mu
          </td>
          <td>2025-10-08</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05c188c0c6e2f128d8772ff9b6be6a1353be8fc1" target='_blank'>
              A Hybrid Multiscale Model for Predicting CAR-T Therapy Outcomes in Solid Tumors
              </a>
            </td>
          <td>
            M. Nikmaneshi, Lance L. Munn
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Pleomorphic adenoma (PA), the most common benign salivary gland tumor, harbors unpredictable risks of recurrence and malignant transformation into carcinoma ex pleomorphic adenoma (CXPA), posing significant clinical challenges. To better delineate the tumor transformation trajectory, we performed single-cell RNA sequencing of normal salivary gland, primary PA, recurrent PA (rPA), and CXPA. Cell trajectory reconstruction, differential expression gene identification, and key gene network analysis were integrated to characterize molecular transitions and intercellular crosstalk driving PA recurrence and malignant transformation. Immunohistochemistry was used to validate key findings. GALNT13 + myoepithelial cells were identified as CXPA-specific malignant progenitors, delineating early malignant conversion. Concurrently, MIF + myoepithelial cells exhibited enhanced tissue-destructive capabilities. Fibroblasts enforced fibrotic restraint in primary PA and drove extracellular matrix degradation in CXPA. The tumor microenvironment exhibited stage-specific adaptations, with CXPA favoring pro-inflammatory MIF-CD74/CD44 signaling and rPA adopting immunosuppressive traits. Stromal reprogramming and immune-editing dynamics collectively orchestrated microenvironmental adaptation, linking cellular heterogeneity to clinical aggressiveness. This study provides the first comprehensive molecular atlas of PA-to-CXPA transformation, revealing malignant specialization of the myoepithelial subpopulation, fibroblast-mediated stromal reprogramming, and immune-editing driven microenvironmental adaptation. These findings provide a framework for precision stratification of the malignant potential of PA, while positioning microenvironmental intervention as a cornerstone of future clinical strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/110371b54a7af37b631e968b70c953daf9f4053b" target='_blank'>
              Single-cell transcriptomics reveals cellular evolution underlying pleomorphic adenoma recurrence and malignant transformation
              </a>
            </td>
          <td>
            Yike Li, Zhixuan Li, Guile Zhao, Qi Han, Zhangfan Ding, Cheng Wang, Chunjie Li
          </td>
          <td>2025-10-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Melanoma shows heterogeneity across body sites like skin, acral skin, and the uvea, driven by molecular characteristics and genetic variations. However, comparative studies exploring the heterogeneity of melanoma across different anatomical sites remain limited, hindering a comprehensive understanding of its underlying biology. We proposed a research framework through bioinformatics to analyze the tumor ecosystems of cutaneous, acral, and uveal melanoma, from molecular characteristics to genetic variations at single-cell resolution. We found that oxidative phosphorylation (OXPHOS) is a critical driver of tumor cell evolution, with abnormal ribosomal gene and tumor suppressor expression observed in uveal melanoma (UM). Additionally, we screened for potential drug targets and drugs against tumor cells. In the immune microenvironment, acral melanoma (AM) and UM exhibit stronger immunosuppressive characteristics compared to cutaneous melanoma (CM). OXPHOS contributes to T cell cytotoxicity dysregulation in CM and AM, while interferon-γ is crucial in UM. Tumor cells may also induce T cell dysfunction through biological signals such as MIF-CD74 and HLA-E-NKG2A. This study offers valuable insights into melanoma heterogeneity, providing a comprehensive research framework for understanding the distinct molecular and immune characteristics of CM, AM, and UM, and potentially guiding the development of therapeutic strategies tailored to each melanoma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da658ce35ad37afa8afa0fc4adb1c52869ddb7f4" target='_blank'>
              Dissecting Melanoma Ecosystem Heterogeneity from Molecular Characteristics to Genetic Variation at Single-Cell Resolution
              </a>
            </td>
          <td>
            Congxue Hu, Liyuan Li, Tengyue Li, Baobin Qi, Wanqi Mi, He Yu, Kaiyue Yang, Qi Ou, Xia Li, Yunpeng Zhang
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Spatial Multiomics allows for context-based study of the tumor microenvironment (TME). TME consists of diverse cells that collectively influence tumor progression and immune activation. CD8 T-cells, essential for antitumor immunity, often become exhausted in the TME. PD1+ TCF1+ stem-like CD8 T-cells retains differentiation potential and can be rejuvenated through cytokine signaling and immune checkpoint modulation. Here we introduce a workflow for RNAscope™ Multiplex Fluorescent v2 assay combining RNA in-situ hybridization (ISH) and protein immunofluorescence to study cell marker (CD8, PD1) with RNA targets (TNFA, TCF7, IFNG) in a tumor microarray (TMA). The Manual Multiplex Protease-Free workflow enables simultaneous RNA and protein visualization without enzymatic disruption, preserving antigen integrity and tissue morphology. Using this Multiomics assay, we can study CD8 T-cell differentiation and activation. Co-detection of RNA transcripts and proteins provides insights into specific T cell phenotypes and their activation/exhaustion status. Data reveal robust co-expression patterns of these targets, particularly in the tumor-cell neighboring stromal and immune cells in breast, cervical and stomach cancers, reflecting immune activation and exhaustion across distinct TME niches. This methodology lays a foundation for refining immunotherapeutic strategies targeting PD1+ TCF7+ CD8 T-cells, enhancing effector functions, and advancing precision oncology interventions.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bea7d4ce3faf01c213d46ecb8b271a789ba794e6" target='_blank'>
              Spatial multiomics reveals T-cell activation and exhaustion states in the tumor microenvironment 3446
              </a>
            </td>
          <td>
            Steve Zhou, Yifan Wang, Debia Wakhloo, A. Dikshit, Li-chong Wang, Maithreyan Srinivasan
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) poses a great challenge for therapeutics development due to its heterogeneity and the lack of immunocompetent models that faithfully recapitulate human GBM. We sought to generate and characterize syngeneic mouse models for major GBM subtypes.



 Postnatal electroporation using the PiggyBac transposon/transposase for overexpression (OE) and CRISPR/Cas9 for knockout (KO) was used to generate models with common GBM alterations including EGFRvIII-OE/Cdkn2a-KO/Pten-KO, Nf1-KO/Pten-KO/p53-KO, and RasV12-OE/Cdk4-OE/p53-KO. Model characterization was done by HE staining, immunofluorescence staining, RNA sequencing and flow cytometry.



 All models displayed histopathological and molecular features of GBM such as pseudopalisiding necrosis, microvascular proliferation, and cellular atypia, as well as Gfap and Nestin expression. On the transcriptomic level, Nf1/Pten/p53 tumors showed enrichment for the human mesenchymal signature, while RasV12/Cdk4/p53 and EGFRvIII/Cdkn2a/Pten tumors resembled the classical and proneural signatures. Each tumor was comprised of four cellular states, i.e., astrocytic, mesenchymal, OPC-like, and NPC-like, suggestive of intratumoral heterogeneity. On examining the microenvironment, EGFRvIII/Cdkn2a/Pten tumors had a higher relative abundance of T cells (48.6% vs 6.3%, p=0.0084, 48.6% vs 8.8%, p=0.0376,), NK cells, (1.2% vs 0.4%, p=0.1425, 1.2% vs 0.19%, p=0.0315) and dendritic cells (7.5% vs 0.4%, p=0.0272) than the Nf1/Pten/p53 and RasV12/Cdk4/p53 models. RasV12/Cdk4/p53 tumors had greater infiltration of monocyte-derived macrophages than Nf1/Pten/p53 (35.9% vs 4.18%, p=0.0012) and EGFRvIII/Cdkn2a/Pten (35.9% vs 10.4%, p=0.0620) tumors as well as an elevated CD4+/CD8+ T cell ratio. In the Nf1/Pten/p53 model, microglia were the most abundant cells within the CD45+ compartment, surpassing levels observed in EGFRvIII/Cdkn2a/Pten (60.2% vs 4.6%, p=0.0548) and RasV12/Cdk4/p53 (60.2%vs7.5%, p=0.0564) tumors.



 We developed an array of syngeneic mouse models of GBM that recapitulate human inter- and intra- tumoral heterogeneity. Future studies will define the functional states of immune cells in the microenvironment and evaluate the impact of immune cell composition on the response to immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9fb466bb8d2e828409bfc681044d64171c2f26" target='_blank'>
              TMOD-40. Genetic drivers shape tumor phenotype and composition of the tumor microenvironment in syngeneic mouse models of glioblastoma
              </a>
            </td>
          <td>
            J. Pyczek, Shannon Snelling, Xueqing Lun, Bo Young Ahn, Heewon Seo, Miranda Yu, Daniela F. Quail, A. Morrissy, Jennifer A Chan
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous malignancy, with its tumor immune microenvironment (TIME) playing a crucial role in tumor progression, immune evasion, and treatment response. Despite insights from single-cell technologies, challenges such as small samples and inconsistent annotations have limited a comprehensive understanding. Thus, a large-scale, high-resolution single-cell atlas is needed to fully explore the heterogeneity of CRC TIME and guide personalized treatment. We analyzed single-cell transcriptomic data from 100 CRC samples (371,223 cells). Immune, stromal, and epithelial cells in the TIME were clustered, annotated, and functionally analyzed. Transcription factor networks were inferred using SCENIC, differentiation states assessed with CytoTRACE, and cell–cell communication analyzed via CellPhoneDB. TIME subtypes were constructed from bulk RNA-seq data, followed by survival analysis. SCRN1 was further validated through in vitro experiments. This study presents a comprehensive single-cell CRC atlas, revealing significant heterogeneity in T cells, B cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells within the TIME. We defined two immune ecological subtypes: subtype 1 (enriched in metabolic and motility pathways with poor prognosis) and subtype 2 (enriched in immune response pathways, with better prognosis and greater immunotherapy potential). High SCRN1 expression in malignant epithelial cells correlated with poor prognosis and metastasis. In vitro experiments confirmed the role of SCRN1 in promoting CRC cell proliferation and migration. This study provides a comprehensive characterization of the CRC TIME, links TIME subtypes to prognosis, and identifies SCRN1 as a potential therapeutic target, offering insights for CRC precision subtyping and immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c5be539fad2c0073ae6e62159eaef1e83a1ef" target='_blank'>
              The single-cell atlas of colorectal cancer reveals cellular heterogeneity and key molecular features of the tumor immune microenvironment
              </a>
            </td>
          <td>
            JunHu Li, ZhiQiang Liu, BoYu Zhang, Liuping You, Yuenan Huang
          </td>
          <td>2025-10-10</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08fb4029b669b488e3f7a2bf959803989f3d7dae" target='_blank'>
              Single-Cell RNA and ATAC Sequencing Reveal Cellular Heterogeneity and Chromatin Accessibility Dynamics in Young Chinese Breast Cancer Patients across Pre- and Post-Neoadjuvant Therapy
              </a>
            </td>
          <td>
            Xue Han, Bin Wang, Wenting Xiang, Fan Lin, Bo Sun, Beibei Yang, Zhihan Ruan, Shasha Liu, Shuo Li, Hong Liu, Jian Liu
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy with significant cellular heterogeneity, which poses a significant challenge for treatment. We integrated three single-cell RNA sequencing (scRNA-seq) datasets to construct an expression matrix, identify cell types, and assess copy number variation (CNV) scores, with a particular focus on ductal cells. Our study found that ductal cells are the primary source of malignant cells in PDAC and identified a high-malignancy subtype—ductal cell-3—where RACK1 was the most significantly differentially expressed gene. Through the analysis of public and clinical data, along with RT-qPCR and immunohistochemistry validation, we confirmed the high expression of RACK1 in PDAC tumors, which correlated with lymph node metastasis, clinical staging, histological differentiation, and prognosis. Further investigation revealed that RACK1 promotes PDAC cell proliferation, migration, and invasion. Sequencing analysis indicated that RACK1 inhibits the FOXO signaling pathway, promotes epithelial-mesenchymal transition (EMT), and facilitates the progression and metastasis of PDAC. This study provides new insights into the role of RACK1 in PDAC and its underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fac327fd52a6595ab14fc1ae00b1f2b7a0bcb1e" target='_blank'>
              Molecular mechanisms of RACK1-driven metastasis in pancreatic ductal adenocarcinoma revealed by single-cell RNA sequencing
              </a>
            </td>
          <td>
            Hangyu Liu, Tingxin Wang, Yuying Cui, Biao Zhang, Dong Shang, Huiyi Song
          </td>
          <td>2025-10-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Tumor progression is driven not only by the complex milieu of transcriptional cell states of distinct lineages that populate the tumor microenvironment (TME) but also by the interactions these states forge with one another. Systematically charting the landscape of such prognostic cell-state interactions (CSIs) remains difficult as single-cell datasets are small and rarely linked to clinical outcomes, whereas bulk profiles obscure cellular context. We developed CSI-TME, a computational pipeline that adapts the concept of synthetic lethality to infer prognostic cell-state interactions by directly utilizing the large cohorts of bulk transcriptomic datasets. We applied CSI-TME to IDH-mutant gliomas, deriving a highly reproducible cell-state interaction network (CSIN) that is predominantly pro-tumor and differentially activated in IDH-mut astrocytoma versus oligodendroglioma. Malignant cells within the CSIN resemble multiple neuronal lineages including, astrocyte-like and oligodendrocyte-progenitor-like program, and reveal key interactions between glioma stem cells and T cells. CSIN stratifies patient response to immune-checkpoint blockade therapy. Roughly 20% of interactions involve direct ligand-receptor communication, and their participating cell states significantly co-localize in spatial-transcriptomic datasets, most notably for a pro-tumorigenic interaction between tip-like endothelial cells and hypoxic malignant cells involving multiple ligand-receptor interactions. Integration with somatic-mutation profiles reveals that many interactions in CSIN exert anti-tumor effect early-on in glioma but shift toward pro-tumor roles as the disease progresses. Taken together, CSI-TME provides a unique way of identifying the prognostic and clinically relevant cell state interactions without the need for extensive single cell datasets and provides a framework for prioritizing therapeutic ligand-receptor targets, patient stratification, while providing new insight into how interactions among the cell states shape the TME in IDH-mutant glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3146cb53ad109ab0b9473e62c6d93d13f03288e3" target='_blank'>
              TMIC-77. Identification of Clinically Relevant Cell State Interactions in the Tumor Microenvironment of IDH-Mut Glioma
              </a>
            </td>
          <td>
            Arashdeep Singh, Bharati Mehani, Vishaka Gopalan, Ken Aldape, Sridhar Hannenhalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive and lethal adult brain tumor. The cellular heterogeneity within the tumor microenvironment (TME) plays a critical role in the complexity of treatment and poor survival. GBM is typically classified into 3 molecular subtypes—Classical, Mesenchymal, and Proneural—associated with EGFR, NF1, and PDGFRA genetic drivers, respectively. Our team previously generated human GBM-specific driver mutations using RCAS/tv-a gene transfer system to create immunocompetent genetically engineered mouse models (GEMMs) of murine GBM (mGBM) that share the same cell-of-origin. While we and others have shown that these GEMMs closely mimic myeloid cell composition and bulk expression profiles observed in human GBM (hGBM) subtypes, it is critical to identify species-specific similarities and differences in tumor microenvironment (TME) heterogeneity at cellular resolution. Hence, we reasoned that single-cell characterization of mGBM models and comparison to hGBM would highlight the cross-species similarities/differences and enable investigators to identify and validate the most relevant human target candidates. Towards this goal, we performed single-cell RNA sequencing (scRNA-seq), multicolor flow cytometry, and in-depth computational analysis on EGFR-, Nf1-, and PDGFB-driven GEMMs to dissect the effect of common GBM genetic drivers on the TME. To assess the human relevance, we compared our findings to bulk RNA-seq data from human GBM tumors with mutually exclusive EGFRvIII deletion, NF1 loss of function mutation, or PDGFRA amplification on chromosome 4. Our analysis revealed that PDGFB-driven tumors were more proliferative and enriched for Wnt signaling interactions, while EGFRvIII-driven tumors showed an elevated interferon signaling response. Moreover, Nf1-silenced tumors displayed higher myeloid abundance, myeloid immunosuppressive interactions involving Osteopontin, Treg infiltration, and expression of immune checkpoint molecule Ctla4. Overall, we established a human-mouse analytical platform for genotype-aware target discovery and validation, which offers promising new avenues for more effective, personalized treatments in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edea93ae20a51718ffae751a4a4ae63a990d752e" target='_blank'>
              TMIC-24. Single-cell decoding of the genotype-immunophenotype relationship in glioblastoma
              </a>
            </td>
          <td>
            N. Soni, Kavita Rawat, Zhihong Chen, Angela DiMauro, Bruno Giotti, D. Hambardzumyan, A. Tsankov
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is an incurable and aggressive brain cancer marked by profound intra-tumoral heterogeneity. Malignant cells exist in four core transcriptional states: stem-like (OPC-like and NPC-like) and more differentiated astrocytic (AC-like) or mesenchymal (MES-like) states. Using spatial transcriptomics, we previously identified a layered tumor structure of these states, with hypoxia emerging as an organizing driver, implicating oxygen availability as a central factor in cell state dynamics. However, how these cellular states interact with each other and with the tumor microenvironment remains incompletely understood.



 To uncover state-specific vulnerabilities for reducing GBM cell state diversity, we conducted a small-molecule screen in gliomasphere models using over 1,600 cysteine-reactive covalent inhibitors, assessing their effects on cell states via RNA-seq. We then employed the Sonar metabolic reporter to visualize NAD+ and NADH levels in gliomaspheres. Through experiments with orthotopic xenografts in mice and co-culture with human cortical organoids, we characterized distinct redox profiles across GBM cell states by single-cell and bulk RNA-seq. Spatial single-cell transcriptomics and mass spectrometry imaging (MSI) metabolomics on adjacent sections of frozen patient samples were used to identify preferential metabolic programs in specific cell states.



 Our small-molecule screen identified compounds that promote differentiation and are enriched for NRF2 activation, a master regulator of the antioxidant response. Spatial single-cell transcriptomics and mass spectrometry metabolomics experiments confirmed that MES-like cells exhibit a heightened glycolytic signature, whereas stem-like states preferentially depend on oxidative phosphorylation (OXPHOS) for their energetic needs. Finally, pharmacologic perturbations revealed that mitochondrial and glycolysis inhibition selectively deplete different cell state populations in gliomaspheres.



 Our findings provide deeper insight into the metabolic underpinnings of GBM cell states and support the rationale for combinatorial therapies aimed at reducing intratumoral heterogeneity that exploit their divergent dependencies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e25475481794c097c1d8706111cced1dc0e56d4b" target='_blank'>
              TMET-30. Uncovering the metabolic programs underlying malignant cell state heterogeneity in glioblastoma
              </a>
            </td>
          <td>
            Alexander Jucht, Maolin Ge, Rony Chanoch-Myers, Florian Ruiz, Gerard Baquer, Alissa C. Greenwald, Min Yang, Toshiro Hara, Yilin Fan, Channing Pooley, D. Gerovasilis, Charles Couturier, Nathalie Y R Agar, I. Tirosh, L. Bar-Peled, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Pancreatic cancer, specifically pancreatic ductal adenocarcinoma (PDAC), is notorious for its aggressive nature and dismal prognosis, ranking as a leading cause of cancer-related mortality worldwide. Despite advancements in surgical techniques, chemotherapies, and novel therapeutic approaches, the overall survival rates remain bleak, primarily due to the complex and dynamic tumor microenvironment (TME) which fosters cancer progression and therapeutic resistance. Central to the TME are cancer-associated fibroblasts (CAFs), which exhibit significant heterogeneity and play multifaceted roles in tumor progression, ranging from remodeling the extracellular matrix to modulating immune responses. Recent breakthroughs in single-cell sequencing and spatial omics have unraveled the diverse functional states and spatial organization of various cell types within the TME, including distinct subtypes of CAFs which differ vastly in their influence on pancreatic cancer pathophysiology. This review delves into the intricate interactions between CAFs and pancreatic cancer cells, highlighting how metabolic reprogramming and immune regulation by CAFs contribute to the high malignancy of PDAC. By integrating single-cell and spatial resolution data, we offer new insights into the metabolic crosstalk and immune landscape remodeling driven by CAFs, establishing a foundation for targeting these stromal interactions in therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2add2bd80d02d372a76aab262dddc13647852fe6" target='_blank'>
              Metabolic and immune crosstalk between cancer-associated fibroblasts and pancreatic cancer cells
              </a>
            </td>
          <td>
            Qiyao Zhang, Z. Cao, Shangcheng Yan, Bingrong Chen, Hao Wu, Hongtao Cao, Chen Lin, Ziwen Liu
          </td>
          <td>2025-10-17</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="


 Understanding the immune microenvironment requires precise delineation of immune cell populations and their spatial interactions within tissues. Advanced spatial-omics technologies enable simultaneous analysis of cellular biomarkers, spatial contexts, and tissue architecture. However, current analytical tools fall short in integrating this multidimensional data, limiting insights into immune cell functionality and heterogeneity. To address this, we present CellSNAP (Cell Spatio- and Neighborhood-informed Annotation and Patterning), a self-supervised computational framework that integrates molecular expression, spatial neighborhood contexts, and local histological features using graph neural networks. CellSNAP enables the de novo discovery of fine-grained immune cell subpopulations, overcoming the constraints of predefined cell atlases. Applying CellSNAP to spatial-omics datasets across various tissues, including spleen and lymphoma, revealed immune subpopulations with distinct spatial distributions and functional states, such as tumor-associated macrophages and spatially stratified B and T cells. These findings underscore the utility of integrating spatial and molecular data to advance immunological research, offering a robust tool for unraveling immune cell dynamics in health and disease.



 1P01AI177687-01, 75N93019C00071, DP2AI171139, R01AI149672, U24CA224331, NSF 2345215, NSF 2245575.



 Technological Innovations in Immunology (TECH)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6caeae1dc09bc627a0349265ceed8d4f08e8abb" target='_blank'>
              Cross-domain information fusion for enhanced immune cell population delineation in single-cell spatial-omics data 2975
              </a>
            </td>
          <td>
            Bokai Zhu
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spatial variable genes (SVGs) reveal critical information about tissue architecture, cellular interactions, and disease microenvironments. As spatial transcriptomics (ST) technologies proliferate, accurately identifying SVGs across diverse platforms, tissue types, and disease contexts has become both a major opportunity and a significant computational challenge. Here, we present a comprehensive benchmarking study of 20 state-of-the-art SVG detection methods using human slides from STimage-1K4M, a large-scale resource of ST data comprising 662 slides from more than 18 tissue types. We evaluate each method across a range of biologically and technically meaningful criteria, including recovery of pathologist-annotated domain-specific markers, cross-slide reproducibility, scalability to high-resolution data, and robustness to technical variation. Our results reveal marked differences in performance depending on tissue type, spatial resolution, and study design. Beyond benchmarking, we construct the first cross-tissue atlas of SVGs, enabling comparative analysis of spatial gene programs across cancer and normal tissues. We observe similarities between pairs of tissues that reflect developmental and functional relationships, such as high overlap between thymus and lymph node, and uncover spatial gene programs associated with metastasis, immune infiltration, and tissue-of-origin identity in cancer. Together, our work defines a framework for evaluating and interpreting spatial gene expression and establishes a reference resource for the ST community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ff70f866316ad376e66f985978ce8e7239e3c0" target='_blank'>
              Large-scale spatial variable gene atlas for spatial transcriptomics
              </a>
            </td>
          <td>
            Jiawen Chen, Jinwei Zhang, Dongsheng Peng, Yutong Song, Aitong Ruan, Yun Li, Didong Li
          </td>
          <td>2025-10-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6c4f2ec4971acec7d7b9addcd8348e71b0ec1ff" target='_blank'>
              Decoding epithelial-T cell interactions in colorectal cancer through single-cell and spatial transcriptomics.
              </a>
            </td>
          <td>
            Rentao Yu, Xinyu Lu, Zhuangzhuang Yue
          </td>
          <td>2025-11-28</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01265124a4e62c29b2b424c2546b4d711500fa92" target='_blank'>
              A gene program dictionary of human cells
              </a>
            </td>
          <td>
            Yupu Xu, Yifan Wang, Zhenxing Geng, Yue Qin, Shisong Ma
          </td>
          <td>2025-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Triple-Negative Breast Cancer (TNBC) encompasses a biologically heterogeneous group of tumors, which can be classified into distinct molecular subtypes, namely basal-like 1 (BL1), basal-like 2 (BL2), immunomodulatory (IM), mesenchymal (M), mesenchymal stem-like (MSL), and luminal androgen receptor (LAR), with unique clinical and pathological characteristics. While immune features of these subtypes have been extensively characterized, the integration of non-immune stromal and structural components into our understanding of TNBC biology is only now being fully recognized. This narrative review synthesizes current evidence regarding differences in the non-immune microenvironment across TNBC molecular subtypes, with a focus on cancer-associated fibroblasts (CAFs), vascular features, extracellular matrix (ECM) dynamics, and epithelial–mesenchymal transition (EMT), along with metabolic–hypoxic reprogramming. Data from several studies are integrated to highlight subtype-specific signatures. Differences in stromal architecture and metabolic adaptations, potentially reflecting the underlying molecular heterogeneity, may hold prognostic or predictive significance and could inform personalized therapeutic strategies targeting the tumor–stroma interface.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/089f130c819509e80723565570647cc30d7f85ba" target='_blank'>
              Dissecting the Non-Immune Tumor Microenvironment in Triple-Negative Breast Cancer: Molecular Subtype-Specific Patterns and Prognostic Implications
              </a>
            </td>
          <td>
            A. Syrnioti, Eleni Timotheadou, Vasileios N. Papadopoulos, Georgia Syrnioti, T. Koletsa
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 The immune receptor repertoire captures the fingerprint of immune responses shaped by pathogens, malignancies, and autoimmunity. Advances in spatial transcriptomics have enabled the study of immune repertoires within their spatial context, linking cellular localization to functional diversity. To support this, we present CIRCUSS (Comprehensive Immune Receptor Curation and Spatial Sequencing), an open-source Python package that allows for the robust analysis and visualization of spatial immune cell repertories, quantifying T- and B-cell receptor clonal diversity, expansion, and spatial distribution within tissues. CIRCUSS provides tools for annotating immune repertoires, tracking clonotypes across geographies, and detecting immune cell spatial clustering relative to the tumor microenvironment. Its modular, user-friendly commands enable seamless integration of spatial immune repertoire data with transcriptomics and histological analyses, enhancing the biological interpretation of immune responses. We demonstrate CIRCUSS’s utility by analyzing publicly available spatial datasets from targeted and whole transcriptome detection approaches across tissue sections from human lymphoid tissue, brain metastases, renal cell carcinoma, and melanoma uncovering how immune receptor variation and spatial organization drive localized immune responses. CIRCUSS enables novel insights into immune expansion mechanisms and therapeutic resistance, fostering broader exploration of spatial immunology.



 Research funded by Lustgarten Foundation and Break Through Cancer foundation.



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93751fb379adef50114d2b78cf230f4c349805f1" target='_blank'>
              CIRCUSS, a spatial immune receptor computational analysis tool, enables the characterization of therapeutic resistance to cancer immunotherapies 4342
              </a>
            </td>
          <td>
            Benjamin F. Green, Janelle M. Montagne, Mulan Bell, J. Taube, Elizabeth M. Jaffee, Luciane T. Kagohara, E. Fertig, Dimitri N. Sidiropoulos
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648afec99f4d26f113081261fc3d480301dc1dab" target='_blank'>
              Transcriptional reprogramming of tumor-infiltrating T cells during PD-1 blockade revealed through gene regulatory network and trajectory inference in squamous cell carcinoma
              </a>
            </td>
          <td>
            Roger Casals-Franch, Jordi Villà-Freixa, Lara Nonell, Adrián López García de Lomana
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Advances in our understanding of the tumor microenvironment (TME) will help inform the development of next generation immunotherapies and engineered immune-cell therapies. Spatially characterizing immune cells with cell marker protein and activation marker RNA is crucial. To address this, we have developed a high throughput RNAscope™ assay on the BOND RX for same slide RNA+protein detection. We utilized the new protease-free RNAscope™ Multiomic LS assay that can detect up to 6 RNA and protein targets on the same slide with TSA-based amplification strategy that offers a strong signal boost. Here, we demonstrate the use of our pre-conjugated antibody panel and a some unconjugated primary antibodies in multiple tumor tissues. Infiltrating Cytotoxic T cell lymphocytes were detected using CD8, GZMB and IFNG co-expression. Regulatory T cells were detected using CD4, FOXP3 co-expression. PanCK was used as a tumor marker to delineate tumor from stroma. Similarly, tumor associated macrophages were detected using CD163, CD68, IL10 and IL-1B expression. We were also able to identify distinct M1 and M2 macrophage populations in the tumor samples. The RNAscope Multiomic LS assay was successfully able to interrogate different subpopulation of immune cells with high quality RNA and protein signal while maintaining tissue morphology. The high throughput capability with strong signal boosting technology makes this assay an ideal tool for identifying diagnostic and therapeutic biomarkers.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b2afa7ff60daccddf4d93319b5feabb08eb8786" target='_blank'>
              Identifying immune cell phenotypes and their function by using spatial multiomics 3124
              </a>
            </td>
          <td>
            A. Dikshit, Debia Wakhloo, Julia S. Yu, Sonali A Deshpande
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Gastric cancer progression involves complex interactions among tumor cells, immune components, and stromal elements within the tumor microenvironment. However, a comprehensive understanding of cellular heterogeneity, spatial organization, and cell-cell communication in gastric cancer remains incomplete. Methods Single-cell RNA sequencing was performed on 252, 399 cells from six tissue types, spanning gastritis, intestinal metaplasia, primary tumors, adjacent normal tissue, and metastatic lesions. Integration with spatial transcriptomics enabled spatial mapping of cellular interactions. Pseudotime, cell-cell communication, and transcriptional heterogeneity analyses were conducted. Tumor stage-associated gene modules were identified using Weighted Gene Co-expression Network Analysis (WGCNA) of The Cancer Genome Atlas (TCGA) data. Finally, a deep learning-based prognostic model was developed and externally validated. Results Our analysis revealed dynamic remodeling of the tumor microenvironment during gastric cancer progression, characterized by the expansion of dysfunctional CD8+ T cells, pro-tumorigenic fibroblasts (e.g., ITGBL1+, PI16+, and ITLN1+), and altered myeloid populations. Stromal-immune crosstalk, particularly fibroblast-driven immunosuppressive signaling, was prominent. Spatial transcriptomics revealed the colocalization of immune and stromal cells, supporting spatially organized cellular interactions. WGCNA identified a gene module (657 genes) associated with T cell, myeloid, and stromal alterations, as well as tumor stage. A deep learning model based on this gene set accurately stratified patients according to survival in both TCGA and independent validation cohorts. Risk scores were correlated with clinical features, including tumor stage and therapeutic response. Conclusions Our integrative single-cell, spatial, and computational analysis provides a high-resolution map of gastric cancer microenvironment remodeling. We identified key stromal and immune subpopulations, extensive cellular communication networks, and spatial structures that collectively drive tumor progression and metastasis. The derived gene signature and prognostic model have the potential for clinical risk stratification and therapeutic targeting in gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f147ce2dccf2934265aaef857f3e2b638aa838fb" target='_blank'>
              Integrative multi-omics analysis of gastric cancer evolution from precancerous lesions to metastasis identifies a deep learning-based prognostic model
              </a>
            </td>
          <td>
            Yulin Ren, Xiaoyan Zhang, Ke Li, Shuning Xu, L. Qiao, Qun Li, Cheng Zhang, Ying Liu
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11151ef60d8479a0c0f55b160c123009b32bf93" target='_blank'>
              Single-cell transcriptomic analysis reveals the heterogeneity and functional characteristics of macrophage subpopulations in colon cancer.
              </a>
            </td>
          <td>
            Jiazheng Zhao, Baochun Wang, Xiao Li, Chunting Wei, Yi Min, Dayong Wang
          </td>
          <td>2025-11-23</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer metastasis involves a complex cascade of events, where cancer cells migrate from their site of origin to secondary sites via the lymphatic and circulatory system. During this process, some cancer subclones will successfully ‘seed’ at distant organs to generate lethal metastases. Here, we optimised a method for tracking cancer cells in metastatic breast cancer tumours and investigated their complex interplay with the lung vasculature using lentiviral-based optical barcoding (LeGO). Given the regional heterogeneity in lung tissue microenvironments as well as lobar asymmetry, we used light sheet microscopy to perform three-dimensional (3D) imaging of wholemount lung lobes. The results revealed that polychromatic metastases occurred less frequently than monochromatic metastases and were more likely to be located nearer to blood vessels in both spontaneous (i.e. mammary fat pad injections) and experimental (i.e. tail vein injections) mouse assays of metastasis. This 3D imaging and analytic pipeline can provide unique insights about metastatic heterogeneity and dynamics, and represents a new avenue for studying therapeutic response across large volumes of lung tissue.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6bd2742bda5feb8ccacf510a8780597af23f29" target='_blank'>
              LeGO-3D: 3D imaging of lung metastases and vascularisation using light sheet fluorescence microscopy
              </a>
            </td>
          <td>
            Sabrina M. Lewis, J. Berthelet, Lachlan W Whitehead, P. Rajasekhar, Farrah El-Saafin, C. Bell, Shalin Naik, D. Merino, Verena C Wimmer, K. Rogers
          </td>
          <td>2025-11-07</td>
          <td>npj Imaging</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer metastasis, the process by which tumor cells spread from the primary site to other organs, is one of the leading causes of death in cancer patients. However, due to its complexity and unpredictability, the study of cancer metastasis remains a significant challenge in medicine. In recent years, with the advancement of high-throughput sequencing technologies and single-cell transcriptomics, we could gain a deeper understanding of the molecular mechanisms and cellular heterogeneity underlying cancer metastasis. There is an accumulating volume of publicly available cancer metastasis research data. Nevertheless, these resources lack proper organization, hindering systematic analysis. In this study, we developed an integrated resource named Panmim (http://www.gdwk-bioinfo.com/pan_metastasis/home) to investigate the immune microenvironment of metastatic tumors. The database currently encompasses 90 single-cell RNA-seq datasets from metastatic cancers of diverse origins, encapsulating 14 distinct metastatic sites and 36 primary cancer types. Panmim facilitates a cellular-level comparison of similarities and differences between primary and metastatic cancers, encompassing pathway analysis, alterations in cellular metabolic pathways, cellular distribution preferences, and various aspects of intercellular communication. Panmim presents the analysis results through an intuitive interactive graphical interface, enabling users to explore and understand the complex biological phenomena during cancer metastasis more conveniently. This resource will become a valuable tool for biologists and bioinformaticians to study the mechanisms of cancer metastasis, providing important data support and scientific basis for the optimization of cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9f11b0d110baaaf542f6916b80b13cbf907d57" target='_blank'>
              Panmim: a resource of pan-cancer metastasis immune microenvironment
              </a>
            </td>
          <td>
            Xuelu Zhang, Simeng Hu, Huanjing Hu, Yongkun Li, Pengyin Nie, Xichen Hu, Ruidong Xue, Xiao Xiang, Lu Zheng
          </td>
          <td>2025-10-28</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Immune suppression in glioma is marked by systemic reprogramming of bone marrow-derived cells (BDMCs). Tumor-derived exosomes (TEX) exhibit the requisite biodistribution, genomic contents, and specificity for short and long-range reprogramming of BDMCs. However, in vivo studies have emerged as the major obstacle for discovery of TEX cell-specific glioma immunotherapy. We report a syngeneic murine glioma model capable of tracking endogenously released TEX in real-time.



 We used the RCAS retroviral system develop IDH-mutant and wild-type glioma mouse models with concurrent induction of spontaneous glioma and reporting of GFP-tagged TEX. Tumor and blood samples were analyzed using spectral flow cytometry, multi-plex immunofluorescence and scRNA sequencing to characterize specific TEX-targeted immune cells and their corresponding spatial topography.



 Our multimodal analyses reveal unique genotype-specificity of TEX-targeted immune ontogenies. In IDHm gliomas, myeloid cells were ubiquitously endowed with TEX and exhibited central topography within the tumor. Conversely, IDHwt TEX-containing myeloid cells were over-represented along the tumor periphery. These cells exhibited a dichotomous and polarized pattern of TEX uptake with dorsal (CD68/F4/80) and ventral (CD11b/F480) localization. Over 90% of CD4 T cells engulfed TEX and were highly enriched within the medial leading edge of the tumor. CD8 T cells contained the fewest proportion of TEX and were restrained to the meninges and peri-tumoral white matter. Remarkably, TEX were identified in non-neoplastic contralateral hemispheres and the meninges of tumor-bearing mice. Finally, flow analyses of blood samples detected an increase of TEX in circulating myeloid cells of IDHm tumor-bearing mice, emerging at 2 weeks and peaking at 6 weeks after tumor injection.



 We have developed a novel mouse model that allows unprecedented endogenous tracking of glioma tumor exosomes. Tumor exosomes engage peripheral macrophages and enable recruitment of these cells into the local tumor microenvironment. Our data supports engineering exosomes for glioma immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35dbe1308ea00e1781037da15c0c198dd059c70a" target='_blank'>
              TMOD-05. Glioma Exosomes Mediate Systemic Immune Suppression Through Myeloid Cell Targeting and Recruitment
              </a>
            </td>
          <td>
            Emily Xu, Jonathan Patterson, Jonathan H Sussman, Angelo Angione, Ebrar Akca, N. Amankulor
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer (BC) is the most common malignancy among women, with its progression and prognosis significantly influenced by the tumor microenvironment (TME). Age-related differences in TME composition lead to distinct tumor behaviors: young patients (≤ 40 years) exhibit aggressive tumors, while elderly patients (> 70 years) experience immunosenescence and reduced therapy responses. We performed single-cell RNA sequencing (scRNA-seq) analysis on tumors from 10 breast cancer patients (5 ≤ 40 years, 5 ≥ 70 years), encompassing 33,664 high-quality cells. After cell annotation and batch correction, malignant epithelial cells were identified using inferCNV. We applied pseudotime trajectory analysis, pathway enrichment, and cell–cell communication profiling to investigate age-specific TME dynamics. Survival relevance was assessed using a GEO cohort (GSE20685) of young breast cancer patients, and immunohistochemical staining was performed on clinical tumor and fibroadenoma tissues to validate protein-level expression of key ISGs. In young patients, malignant epithelial cells showed gradual upregulation of interferon-stimulated genes (ISGs) such as IFI44, IFI44L, IFIT1, and IFIT3 along the pseudotime trajectory, suggesting their involvement in early tumorigenesis. High expression of these ISGs was significantly associated with poor overall survival in a young BC cohort (GSE20685). Immunohistochemical validation further confirmed elevated IFIT3 protein levels in young tumor tissues. In contrast, elderly patients had a TME enriched in macrophages and fibroblasts, with activation of immunosuppressive pathways (e.g., SPP1, COMPLEMENT). Our integrative analysis identifies ISGs as key transcriptional drivers of tumorigenesis in young breast cancer, with potential prognostic and therapeutic value. Despite limited sample size, the combination of single-cell transcriptomics, clinical survival data, and protein-level validation provides robust evidence of age-specific TME remodeling. These findings support the development of age-tailored immunotherapy strategies targeting interferon signaling in young patients and immune checkpoint pathways (e.g., LAG3, CTLA4) in elderly individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81afa54ab4f065b06519a2062a7ca2a21beb87a4" target='_blank'>
              Single-cell RNA sequencing reveals age-related heterogeneity in the tumor microenvironment of breast cancer patients
              </a>
            </td>
          <td>
            Zihao Li, Ji Feng, Mengyang An, Shuming Kou, Zengren Zhao, Jingjing Yu, Bo Liu, Fangjian Shang
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The tumor microenvironment (TME) constitutes a highly complex and dynamic ecosystem wherein diverse leukocyte subsets engage in intricate cross-talk that ultimately dictates the balance between tumor progression and suppression. This review provides a comprehensive analysis of the dualistic functions of key immune cell populations-including natural killer (NK) cells, effector T cells, regulatory T cells (Tregs), myeloid-derived suppressor cells (MDSCs), and tumor-associated macrophages (TAMs)-within this milieu. Particular emphasis is placed on the Nigerian context, where unique genetic backgrounds, such as the high prevalence of sickle cell trait, and environmental factors, including endemic infections, significantly shape immune phenotypes and cancer pathogenesis. We explore how cutting-edge single-cell technologies are revolutionizing our understanding of the spatial architecture, functional plasticity, and cellular heterogeneity of these leukocytes in patient cohorts, revealing mechanisms of both immune surveillance and tumor-promoting immunosuppression. The paradoxical roles of these cells elucidate the variable clinical outcomes observed with immunotherapies and underscore the critical need for precision medicine strategies that are tailored to individual immune landscapes. Finally, we discuss emerging therapeutic modalities aimed at selectively depleting pro-tumor subsets like Tregs or MDSCs while activating anti-tumor effectors such as NK cells, which hold substantial promise for reshaping the TME and improving oncology outcomes both in Nigeria and across the globe.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d4138591bf3dda987df758be78f378fdcee1f4" target='_blank'>
              Leukocyte Subset Dynamics in the Tumor Microenvironment: Dual Roles, Single-Cell Insights, and Implications for Cancer Immunotherapy in Nigeria
              </a>
            </td>
          <td>
            Okluchi Hartumu
          </td>
          <td>2025-11-25</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Pediatric-type high-grade gliomas (HGGs) are aggressive brain tumors characterized by rapid growth, infiltrative behavior, and significant therapy resistance. One major challenge in treating HGGs involves the interplay of tumor cells to their microenvironment; specifically the blood-tumor barrier (BTB). Our studies combined histologic and transcriptomic evaluations of immunodeficient rodents intracortically injected with H3K27-altered and H3 WT glioma cells. Multiplex immunohistochemical staining provided temporospatial analysis of known cellular markers involved in BBB formation, including pericytes, endothelial tight-junction proteins, and myeloid cell markers. Regions of interest included the tumor, peritumor (+250µM), and distant (>250µM) areas. Analysis was conducted with the use of QuPath and cell segmentation performed using Mesmer; with expression proportions quantified for each cell/protein marker. Utilization of vessel isolation and nuclei extraction for sc-RNAseq (VINE-seq) allowed us to specifically explore the molecular landscape of vasculature within the tumor microenvironment. All rodent models were untreated and evaluated early to late post-intracranial injection (D20-50). Preliminary results in H3-K27-altered glioma models reveal elevated tight junctions within distal regions compared to the tumor (p<0.0001); demonstrating an intact BTB furthest from tumor bulk. Within the tumor, high Iba1+ macrophages (46%) and low neuronal (1%) expression was present (p<0.0001), while the intratumoral vasculature demonstrated the highest localization of glioma stem cells, and PDGFRβ+ during late disease progression (p<0.0001). VINE-seq identified unique cell-clusters in the tumor bearing vs. contralateral hemisphere at middle and late timepoints (D40-50). Ongoing VINE-seq studies will identify single cell expression changes both regionally and temporally with correlation to multiplex staining. These findings of blood-tumor barrier expression profiles among glioma cells offer a glimpse of changes for both the cellular and molecular landscape with both region and time specificity. Our research underscores the importance of understanding these interactions related to restrictions of CNS treatment and pediatric-type HGG progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da3676d783a5588622cda56cd645fcf0d5acc71b" target='_blank'>
              TMIC-08. Multiplex immunofluorescent imaging and VINE-seq demonstrates the temporospatial complexity of the blood-tumor-barrier in pediatric-type diffuse high grade glioma
              </a>
            </td>
          <td>
            Jatia Mills, Minhye Kwak, Sanghee Lim, D. Maric, Kory Johnson, Trevor McKee, Sadhana Jackson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains one of the most fatal brain tumors, in part because its profound spatial heterogeneity undermines the effectiveness of targeted therapies. Despite extensive pursuit, EGFR and VEGF inhibitor monotherapies have shown limited clinical benefit, yielding only brief remissions. We hypothesized that this therapeutic failure stems from region-specific variations in drug target expression within the tumor microenvironment, limiting the reach of monotherapies. To investigate this, we performed spatial transcriptomic profiling on 17 regions from 8 human GBM specimens, mapping gene expression to distinct anatomical compartments defined by the Ivy Glioblastoma Atlas Project: Core Tumor (CT), Core Tumor with Pseudopalisading Necrosis (CTpan), Core Tumor with Microvascular Proliferation (CTmvp), and Leading Edge (LE). Our analysis revealed striking spatial compartmentalization of therapeutic targets: EGFR expression was largely confined to the proliferative CT; VEGF ligands (VEGFA, VEGFB) were enriched within the hypoxic, necrotic CTpan; and VEGF receptors (FLT1, FLT4, KDR) were mostly localized to CTmvp, consistent with their roles in angiogenesis. Notably, none of these targets were expressed at the infiltrative Leading Edge, a compartment strongly implicated in tumor recurrence. These findings provide mechanistic insight into the limited efficacy of EGFR and VEGF inhibitor monotherapies in GBM: the spatial segregation of targets prevents uniform drug action across all tumor regions. By leveraging spatial transcriptomics, our study highlights the inadequacy of one-size-fits-all regimens and underscores the need for spatially informed, multi-agent strategies tailored to the tumor’s regional biology. These results support a shift toward treatment approaches that better align with the evolving landscape of tumor heterogeneity in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1153b38bc03abdb3629caf32aaea054bbbcdd6" target='_blank'>
              EPCO-49. SPATIAL TRANSCRIPTOMICS ILLUMINATES MECHANISMS OF MONOTHERAPY FAILURE IN GLIOBLASTOMA BY MAPPING TARGET EXPRESSION ACROSS TUMOR REGIONS
              </a>
            </td>
          <td>
            Emre Kiziltug, V. Ravikumar, Reva Kulkarni, A. Maddox, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits profound spatial heterogeneity driving therapeutic resistance, while the role of cancer-associated fibroblasts (CAFs) in orchestrating immunosuppressive niches remains incompletely defined. This study integrates single-nucleus RNA sequencing (snRNA-seq) and spatial transcriptomics (stRNA-seq) to map the cellular and molecular landscape of HCC. snRNA-seq identifies key cell populations-including fibroblasts, T_NK cells, and endothelial cells-using canonical marker genes. Spatial transcriptomics maps gene expression across tumor regions (core, invasive front, stroma) via the robust cell type decomposition (RCTD) algorithm. Immunofluorescence validates collagen deposition and POSTN spatial distribution, confirming T-cell exclusion patterns. The analysis identifies hypoxic metabolic myofibroblasts (hmmyCAFs) as central regulators of the tumor microenvironment. hmmyCAFs enrich at the invasive front, forming collagen-rich barriers that physically exclude CD8⁺ T cells. Simultaneously, they secrete POSTN to suppress immune checkpoint signaling and drive hypoxia-mediated glycolytic reprogramming of T-cell metabolism. Clinically, hmmyCAF activity and POSTN expression correlate with reduced progression-free survival and immunotherapy resistance. This multimodal study defines hmmyCAFs as triple architects of physical immunosuppression, molecular regulation, and metabolic remodeling. By linking collagen remodeling, POSTN-mediated checkpoint inhibition, and hypoxia-driven metabolic reprogramming to clinical outcomes, hmmyCAFs and POSTN may serve as potential indicators for evaluating the efficacy of immunotherapy in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aba5ba124e6d224e40973627e744d5081476427" target='_blank'>
              Spatial Transcriptomics and snRNA-seq Expose CAF Niches Orchestrating Dual Stromal-Immune Barriers in Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Yingxue Li, Changxiang Huan, Haoting Sun, Wei Zhang, Zhen Guo, Chuanyu Li, Jia Yao, Zhiqi Zhang, Shizhe Yu, Qiongzhu Dong, Lunxiu Qin, Jinze Li, Lianqun Zhou
          </td>
          <td>2025-10-07</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Current therapeutic strategies for endometrial cancer are mainly based on aggressive histological types and molecular subtypes. However, ignoring the spatial distribution of immune/stromal cells fails to account for the heterogeneity of the local tumour microenvironment, leading to biased prediction of treatment response. The goal of precision medicine is to delineate the biological characteristics of local functional units based on molecular labelling, which adequately reflects spatially adaptive changes during treatment or metastasis. Single-cell resolution analysis of 40 endometrial cancer cases across four molecular subtypes was performed using imaging mass cytometry (IMC) to quantify the frequency, spatial distribution, and intercellular crosstalk of distinct immune and stromal cell populations. These ecosystem-level features were systematically correlated with clinical features and outcomes, including treatment response and survival. We further identified CD90 + clusters as key regulators of macrophage polarization and T-cell infiltration dynamics, with flow cytometry used to validate their functional role in tumour subtype specification and microenvironmental remodelling. Finally, machine learning-based spatial phenotyping was employed to construct molecular subtype-specific signatures and a highly accurate recurrence prediction model for high-risk endometrial cancer. Single-cell profiling revealed that CD90 + clusters constitute a critical immunomodulatory component within the tumour microenvironment, demonstrating significant enrichment in macrophage differentiation pathways and serving as key mediators of intercellular signalling networks. Furthermore, computational models integrating functional molecular signatures with cell–cell interaction profiles demonstrated high predictive accuracy for both molecular subtyping and recurrence risk stratification in patients with endometrial carcinoma. Our study establishes a spatial eco-context framework for molecular subtypes of endometrial cancer by integrating single-cell spatial multiomics data. This approach enables high-resolution mapping of tumour-immune-stromal interaction networks and reveals novel targets for personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63a8a4865fa920849f9dc4e8439a41c6d062126a" target='_blank'>
              Spatial ecostructural modelling of endometrial cancer identifies the key role of CD90 + CD105 + endothelial cells in tumour heterogeneity and predicts disease recurrence
              </a>
            </td>
          <td>
            Di Wu, Cinian Lv, Zhifeng Yan, Luyang Zhao, Lian Li, M. Ye, Mingyang Wang, Qingzhi Zhai, Nan Wang, Zheng Wang, Yuanguang Meng, Mingxia Li
          </td>
          <td>2025-11-17</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8a1f1ccbeb7cda31923d2bd498d2b7cb3a690e5" target='_blank'>
              Metabolic zonation and characterization of tissue slices with spatial transcriptomics
              </a>
            </td>
          <td>
            Oier Etxezarreta Arrastoa, A. Pirona, Allon Wagner, N. Yosef
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) represents a significant global health challenge due to its molecular heterogeneity and the immunosuppressive nature of its tumor microenvironment (TME). Tumor-associated macrophages (TAMs), particularly the M2-polarized subset, are crucial in HCC progression, contributing to immune evasion, angiogenesis, and metastasis. However, the mechanisms driving TAM reprogramming and their metabolic interactions within the TME remain poorly understood. In this study, we applied single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to identify a novel metabolic-immune axis in HCC. Our findings revealed increased oxidative phosphorylation (OXPHOS) in tumor cells, which correlated with enhanced ribosomal activity in M2 macrophages, suggesting a metabolic coupling between the two. Tumor cells, under elevated OXPHOS, secrete factors that recruit macrophages and promote M2 polarization, thereby reinforcing immune suppression. Further analysis of The Cancer Genome Atlas (TCGA) dataset led to the development of a prognostic model based on OXPHOS-related genes, showing strong predictive power across multiple cohorts. These results suggest a novel OXPHOS-ribosome-M2 polarization axis and offer potential therapeutic targets for immunometabolic therapies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d98d705964e6108c751cf9dce67e7948e856f2d" target='_blank'>
              M2 macrophages and tumor cells engage in a metabolic feedback loop to drive HCC progression
              </a>
            </td>
          <td>
            Yimin Nong, Xiaolong Chen, Baoxin Zhang, Zhenying Lin, Wuriniletu, Chengyou Du, Qiang Liu
          </td>
          <td>2025-11-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Glioblastoma (GBM) is a highly aggressive brain tumor with a median survival of only 15 months. A major challenge in GBM management is the pronounced inter- and intratumoral heterogeneity, which complicates prognosis and therapy. Radiomics, the quantitative extraction of features from medical images, can capture this heterogeneity across the entire tumor volume, but the biological basis of radiographic phenotypes remains poorly understood.



 We integrated preoperative MRI-based radiomic stratification with multi-platform transcriptomics (bulk RNA-seq, single-cell RNA-seq, and spatial transcriptomics) in IDH-wildtype GBM patients. Unsupervised clustering of radiomic features identified four imaging subtypes.



 Group 4 emerged as a high-risk subtype associated with significantly worse survival and a distinctive MRI pattern of peripheral contrast enhancement. Transcriptomic analyses revealed that Group 4 tumors were enriched in cell-cycle and proliferation markers and exhibited neural stem cell–like gene expression signatures. Single-cell profiling confirmed an elevated proportion of stem-like malignant cells in this subtype. Spatial transcriptomics further demonstrated that these proliferative, stem-like programs were localized predominantly to the tumor periphery, corresponding to the rim-enhancing regions on MRI. Finally, we identified the developmental transcription factor VAX2 as a candidate driver of the Group 4 gene network; functional assays showed that VAX2 promotes GBM cell proliferation in vitro.



 Our findings link a radiomics-defined MRI phenotype to specific molecular programs and cell populations in GBM, suggesting that radiomic subtypes can serve as noninvasive biomarkers of tumor biology and highlighting potential therapeutic targets in aggressive, stem-like tumor cell populations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63325a32d6106e3ca9f5bff7f21292a347c5da7b" target='_blank'>
              MRI-based radiomic clustering identifies a glioblastoma subtype enriched for neural stemness and proliferative programs
              </a>
            </td>
          <td>
            Zhongyi Zhang, Yang Liu, Zhicong Zhang, Tian Gui, Youhui Chen, Qian Chen, Xiufu Wu, Li Sun, Sujie Li, Shuyang Wei
          </td>
          <td>2025-11-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chemotherapy resistance remains a major clinical challenge in ovarian cancer treatment, limiting therapeutic efficacy and patient survival. Understanding the molecular mechanisms underlying chemoresistance at the single-cell level is crucial for developing targeted therapeutic strategies. We performed comprehensive single-cell RNA sequencing analysis on primary and chemoresistant ovarian cancer samples to characterize cellular heterogeneity and identify resistance-associated molecular signatures. Cell type identification, differential gene expression analysis, functional enrichment analysis, and immune microenvironment characterization were conducted to elucidate chemoresistance mechanisms. Computational predictions were validated by RT-qPCR analysis of four key resistance genes (BIRC5, ABCB1A, ABCG2, BCL2) in SKOV3 and SKOV3-CDDP cell lines. Our analysis identified nine distinct cell populations within the ovarian cancer microenvironment, revealing extensive transcriptional reprogramming in chemoresistant tumors. Key chemoresistance genes including Abcb1a, Abcg2, Bcl2, Birc5, Nek2, Inhba, Irs1, and Prc1 showed differential expression patterns across cell types. Epithelial cells and proliferating cells exhibited elevated expression of multiple resistance markers, particularly Birc5. Functional analysis revealed enrichment of ribosome biogenesis, protein synthesis machinery, and amoeboid-type cell migration pathways in resistant cells. Macrophage polarization states were significantly altered, and immune checkpoint molecules showed coordinated upregulation, indicating comprehensive immune microenvironment reprogramming. RT-PCR validation confirmed single-cell predictions with significant upregulation in resistant cells: BIRC5 (4.2-fold, p < 0.001), ABCB1A (3.8-fold, p < 0.001), ABCG2 (2.9-fold, p < 0.01), and BCL2 (2.1-fold, p < 0.05). This study provides a comprehensive single-cell atlas of chemoresistance in ovarian cancer, revealing cell type-specific resistance mechanisms and immune evasion strategies. The identified molecular signatures offer potential therapeutic targets for overcoming chemotherapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f229e22beb03e0ca3194ad4df8f24784e9b8331" target='_blank'>
              Single cell transcriptomic analysis reveals molecular mechanisms of chemoresistance and immune microenvironment remodeling in ovarian cancer
              </a>
            </td>
          <td>
            Xia Hu, Caifang Du, Dongjie Du, Yonghua Zhang, Haijian Gao, Minxian Tao
          </td>
          <td>2025-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Human papillomavirus (HPV)-associated oropharyngeal cancer (OPC) is increasing in prevalence, but the drivers of metastasis remain poorly understood, which impacts the ability to personalise management decisions. Much of the genomic research to date focuses on the HPV-negative population. Here, we utilise single-cell and spatial single-cell techniques to understand the drivers of metastasis. Patients with HPV-positive OPC and cervical lymph node metastases treated with curative surgery had matched samples from the primary and lymph nodes collected for research. Single-cell RNA sequencing, single-cell spatial sequencing (Visium) and in-situ spatial platforms were performed. Cancer clones were delineated using inferred copy number variation. Expression phenotypes and interactions with the tumour microenvironment were compared between the metastasising and non-metastasising cancer clones. Individual cancer clones have varied abilities to metastasise and undergo clonal expansion in the lymph node, with only a subset of clones present in the primary expanding in the lymph node. Four mechanisms were identified as defining the metastatic phenotype, including protein translation adaptation, immunoproteasome dysfunction and immune evasion, suppression of the IFN immune response and cap-independent protein translation. This research elucidates multiple mechanisms driving the expansion of cancer clones in HPV-positive oropharyngeal cancer. By detailing the roles of translational adaptation, immunoproteasome dysfunction, suppression of the interferon immune response and cap-independent protein translation, we provide insights into how these processes contribute to immune evasion and tumour survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fda352f2dd3f74c2b6cb392920d418565101f0" target='_blank'>
              Protein translation dysregulation and immune cell evasion mediated by IFN and immunoproteasome downregulation define metastatic clones in HPV-related cancer of the oropharynx
              </a>
            </td>
          <td>
            Venessa T. Chin, W. Muskovic, Rachael A. McCloy, Drew R. Neavin, José Alquicira-Hernández, Himanshi Arora, A. Senabouth, K. Sudhakar, Patricia Keith, Ellie Spenceley, Dominik Kaczorowski, Peter Floros, Brett Leavers, Julia Crawford, Richard Gallagher, Joseph E. Powell
          </td>
          <td>2025-10-14</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Gliomas are lethal malignancies composed of heterogeneous and dynamic subpopulations of cellular states that resemble both normal neurodevelopmental cell types and adaptive responses to stress within the tumor microenvironment (TME). While cellular plasticity enables gliomas to survive environmental pressures, the mechanisms driving glioma state dynamics remain unclear. To explore how glioma state heterogeneity functionally and metabolically interacts with the brain TME, we conducted a multi-omic analysis across 392 glioma specimens, including patient tumors, orthotopic xenografts and gliomasphere cultures using bulk RNA/whole-exome sequencing and lipidomic profiling. Single-cell transcriptome sequencing of a diverse panel of glioma tumors (n = 21) integrated with bulk tumor transcriptome profiling revealed a spectrum of cellular identities associated with distinct lipidomic profiles. Comparison of matched patient and derived models showed that the non-native environments, such as gliomasphere culture, restricted glioma state diversity and enriched for states characterized by elevated de novo fatty acid synthesis. Across environmental contexts, glioma state plasticity was coupled to lipid metabolic plasticity, enabling tumors to concordantly remodel their state composition and lipid profiles in response to environmental constraints. Of note, certain tumors exhibited limited state and metabolic plasticity – specifically those enriched for states within oligodendroglial and neuronal lineages. These gliomas demonstrated reduced de novo lipid synthesis capacity, and a concordant increased reliance on exogenous lipid scavenging within the brain microenvironment for survival. Together, these results link glioma heterogeneity and plasticity to lipid metabolic reprogramming, and highlight how distinct cellular state profiles result in environment-dependent metabolic liabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149479392e37d2670e1ddd1d4b3e1788684e5fcb" target='_blank'>
              EPCO-63. LIPID METABOLIC HETEROGENEITY ACROSS GLIOMA CELLULAR STATES REVEALS ENVIRONMENTAL DEPENDENCIES
              </a>
            </td>
          <td>
            Nicholas A. Bayley, Jenna K. Minami, Cassidy Andrasz, C. Tse, Jennifer Salinas, Dimitri Cadet, A. Bhaduri, T. Cloughesy, Thomas G. Graeber, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocytes (TILs) play a critical role in the ability of the immune system to combat cancer, offering a foundation for personalized immunotherapies. However, the effectiveness of TILs is often reduced by problems like becoming less active, the tumor making the immune system weaker, and not lasting long in the tumor environment. Recent advancements in single-cell technologies, including single-cell RNA sequencing (scRNA-seq), single-cell T-cell receptor sequencing (scTCR-seq), and mass cytometry (CyTOF), have revolutionized our understanding of TIL heterogeneity and dynamics. These tools offer new perspectives on the diverse phenotypes, functional states, and spatial organization of TILs, enabling the identification of key exhaustion markers, regulatory pathways, and neoantigen-specific clones. Concurrently, genetic reprogramming strategies have emerged to address TIL limitations by reversing exhaustion, enhancing metabolic resilience, and improving persistence in vivo. This review explores the synergistic integration of single-cell technologies and genetic engineering in refining TIL-based therapies. We talk about how spatial transcriptomics can help us understand how TILs work in different areas of the body and how changing their epigenetics can help them become more effective at fighting cancer. Additionally, we highlight emerging approaches to overcome immunosuppressive barriers in the tumor microenvironment (TME), including targeting regulatory immune cells, neutralizing suppressive cytokines, and enhancing antigen presentation. Together, these strategies promise to unlock the full therapeutic potential of TILs, paving the way for more effective and durable cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8024b4f684c7b9b9c99e6af201613f98c8efa0" target='_blank'>
              Probing and enhancing tumor-infiltrating lymphocytes: insights from single-cell technologies and genetic reprogramming
              </a>
            </td>
          <td>
            Chaitanya Kumar, Ravi Verma, Ashok Sharma, V. V. Priya
          </td>
          <td>2025-11-17</td>
          <td>Exploration of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Metastatic brain tumors (BrM) are associated with significant morbidity and mortality. Despite significant advances in understanding the role of microbiome in cancer and tumor immunity, the impact of microbiome in BrM remains largely unknown. In this study, we used a prospectively collected cohort of BrM tumors to explore the spatial distribution of microbial elements in BrM and determine their correlation with the tumor microenvironment. We employed pan-bacterial 16S fluorescent in situ hybridization and Small Molecular Imaging (SMI, CosMx® Platform, NanoString) and demonstrated that bacterial signals can be detected in BrM tumors, are intracellular and distributed across tumor and immune cells. We investigated the spatial association of these bacterial signals with specific niches in the tumor microenvironment using SMI and Digital Spatial Profiling (DSP, GeoMx® Platform, NanoString). We demonstrated that tumor regions with high bacterial signals were associated with in anti-bacterial signatures indicative of TLR pathway activation, enrichment of neutrophils and neutrophil enzymes, as well as distinct metabolic signatures. Collectively, our studies introduce intracellular bacterial signal as a novel component of the brain tumor microenvironment and demonstrate its correlation with distinct immune signatures. These findings can guide future mechanistic and translational studies to elucidate the functional role of the intra-tumoral bacterial signal in brain tumor biology.



 This study was supported by the National Cancer Institute (F32CA260769), the Andrew M. McDougall Brain Metastasis Research Award, and the Dr. Marnie Rose Foundation.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c5515b9b890be2829ade266a16baf6b7c61224" target='_blank'>
              Intra-tumoral bacterial signals are associated with distinct immune niches in metastatic brain tumors 3692
              </a>
            </td>
          <td>
            Golnaz Morad, Ashish V Damania, Brenda Melendez, Bharat Singh, Matthew Wong, P. Sahasrabhojane, Yasmine M. Hoballah, Fabiana J. Veguilla, Sarah Johnson, J. Losh, Susan Bullman, N. Ajami, Sherise Ferguson, Jennifer A. Wargo
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1217f04795a563df5b0ffd6d061c6a9c9b363d93" target='_blank'>
              Lineage of origin-specific developmental programs drive the behaviors of malignant cells in an avian embryo model of human Medulloblastoma subgroups
              </a>
            </td>
          <td>
            Marion Mallet, Florian Martin, Karine Thoinet, Caroline Imbert, Moein Sarhadi, Jérémy Ganofsky, Jacob Torrejon Diaz, T. Fenouil, C. Delloye-Bourgeois, Julien Falk, Cécile Faure-Conter, Ludovic Telley, D. Meyronet, O. Ayrault, Valérie Castellani, S. Tauszig-Delamasure
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Metastatic outgrowth requires that cancer cells delaminate from the primary tumor, intravasate, survive in circulation, extravasate, migrate to, and proliferate at a distal site. Recurrent genetic drivers of metastasis remain elusive, suggesting that unlike the early steps of oncogenesis, metastasis drivers may be variable. We develop a framework for identifying metastasis regulators using CRISPR/Cas9-based screening in a genetically defined organoid model of colorectal adenocarcinoma. We conduct in vitro screens for invasion and migration alongside orthotopic, in vivo screens for gain of metastasis in a syngeneic mouse model. We identify CTNNA1 and BCL2L13 as bona fide metastasis-specific suppressors which do not confer any selective advantage in primary tumors. CTNNA1 loss promotes cell invasion and migration, and BCL2L13 loss promotes anchorage-independent survival and non-cell-autonomous changes to macrophage polarization. This study demonstrates proof of principle that large-scale genetic screening can be performed in tumor-organoid models in vivo and identifies novel regulators of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33c9f8937cc2d0d69608a2acde03a6c6afb3fab8" target='_blank'>
              Forward genetic screening in engineered colorectal cancer organoids identifies regulators of metastasis.
              </a>
            </td>
          <td>
            Xin Wang, Zvi Cramer, N. A. Leu, Keara Monaghan, K. Durning, S. Adams-Tzivelekidis, Joshua Rhoades, Jonathan Heintz, Yuhua Tian, Joshua Rico, Diego Mendez, R. Petroni, Austin King, Melissa S Kim, Rina Matsuda, Olivia Hanselman, Alice E. Shin, María F Carrera Rodríguez, Igor E. Brodsky, Anil Rustgi, Ning Li, Christopher Lengner, M. Andres Blanco
          </td>
          <td>2025-11-11</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Genomic alterations in glioblastoma (GB) drive distinct transcriptional subtypes, yet how they shape the tumour microenvironment (TME) architecture remains poorly understood. Here, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss, and PDGFB overexpression in Tp53−/− Nestin-tv-a mice, that recapitulated classical, mesenchymal, and proneural transcriptomic GB subtypes. We integrated single-cell RNA sequencing with regional (NanoString GeoMx) and single cell (10x Visium HD) spatial transcriptomics and proteomics data to acquire an in-depth characterization of the TME organization. At single cell level, NF1- and EGFRvIII-driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human GBs that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1043a078420592595b843047c99776c531341d4b" target='_blank'>
              TMIC-04. Genetic drives promote distinct microenvironment organization in glioblastoma
              </a>
            </td>
          <td>
            Junyi Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec585f40c1c2e8296bee75c248183641228fbd0" target='_blank'>
              Macrophages display spatiotemporal specificity in intrahepatic cholangiocarcinoma and drive tumour progression via OSM and THBS1
              </a>
            </td>
          <td>
            Tingjie Wang, Ruitao Long, Zhongyu Cheng, Mi Zhang, Bing Wei, Cong Lyu, Cuiyun Zhang, Guangbo Tang, Zihang Li, Lei Li, Yida Li, Xiaofei Zhu, Chao Shi, Ke Yang, Chuanrui Xu
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive and lethal primary brain tumor in adults, characterized by rapid proliferation, an immunologically inert microenvironment, diffuse tumor cell infiltration and therapy resistance. Progress in developing effective glioblastoma treatments is impeded by a lack of mechanistic insight into how GBM cells and the surrounding microenvironment respond to therapeutic interventions. To address this prominent gap, we developed an innovative on-therapy, longitudinal, and multi-regional tissue sampling approach in patients receiving rQNestin34.5v.2, an engineered oncolytic biologic, based on herpes simplex virus 1, to investigate the dynamic evolution of tumor and immune ecosystems during treatment. Using targeted spatial proteomics (CyCIF) alongside spatial metabolomics (MALDI-MSI), we analyzed over 100 samples from six patients, profiling immune, stromal, and malignant cells across both space and time. Our CyCIF data revealed that T and B cells infiltrate and colocalize within rQnestin34.5v.2–positive regions, forming clusters that reflect active immune interactions. These immune-rich zones were inversely correlated with SOX2⁺ and OLIG2⁺ tumor cells. Notably, patients with longer survival showed a progressive increase in CD3⁺ T cells, driven by expanding cytotoxic T cell populations alongside a reduction in CD4⁺ T cells. Despite increased immune infiltration, post-trial tissues displayed an immunosuppressive environment characterized by few cytotoxic T cells and reduced tumor cells. Integrating spatial proteomics and metabolomics, we observed elevated purine metabolites—including ATP, GTP, and xanthine—in regions of heightened immune infiltration. Extracellular ATP and GTP likely facilitate T cell activation and migration through purinergic signaling and GTPase pathways. These metabolic patterns suggest that purine metabolism plays a key role in shaping immunologically active tumor microenvironments following treatment. Together, these findings provide novel mechanistic insights into the evolving tumor-immune landscape during rQnestin34.5v.2 therapy in glioblastoma, highlighting purine metabolism as a potential driver of immune activation within the tumor microenvironment. This integrated spatial approach offers a powerful framework to guide the development of more effective immunotherapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82f2a6552b2a8d376cecc37c28c7caffa62bc8f2" target='_blank'>
              TMIC-70. Integrative spatiotemporal proteomic and metabolomic characterization of immune infiltration following intracranial injection of oncolytic immunotherapy in glioblastoma patients
              </a>
            </td>
          <td>
            Jennifer Gantchev, Gerard Baquer, Peter Zheng, Alicia D’Souza, Alexander L. Ling, Michael S Regan, G. Teamlab, Atul Deshpande, Forest M. White, Kenny Yu, E. Chiocca, Charles Couturier, Nathalie Y R Agar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) ranks among the most prevalent malignancies globally. Although treatment strategies have improved, the prognosis for patients with advanced HCC remains unfavorable. Tumor-associated macrophages (TAMs) play a dual role, exhibiting both anti-tumor and pro-tumor functions. In this study, we analyzed single-cell RNA sequencing data from 10 HCC tumor cores and 8 adjacent non-tumor liver tissues available in the dataset GSE149614. Using dimensionality reduction and clustering approaches, we identified six major cell types and nine distinct TAM subtypes. We employed Monocle2 for cell trajectory analysis, hdWGCNA for co-expression network analysis, and CellChat to investigate functional communication between TAMs and other components of the tumor microenvironment. Furthermore, we estimated TAM abundance in TCGA-LIHC samples using CIBERSORT and observed that the relative proportions of specific TAM subtypes were significantly correlated with patient survival. To identify TAM-related genes influencing patient outcomes, we developed a high-dimensional, gene-based transformer survival model. This model achieved superior concordance index (C-index) values across multiple datasets, including TCGA-LIHC, OEP000321, and GSE14520, outperforming other methods. Our results emphasize the heterogeneity of tumor-associated macrophages in hepatocellular carcinoma and highlight the practicality of our deep learning framework in survival analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/512cd6a71ab69bb5319b1934b848fe9e7e46835f" target='_blank'>
              Bioinformatics Analysis of Tumor-Associated Macrophages in Hepatocellular Carcinoma and Establishment of a Survival Model Based on Transformer
              </a>
            </td>
          <td>
            Zhuo Zeng, Shenghua Rao, Jiemeng Zhang
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Understanding how small molecules affect cellular morphology is essential for exploring their mechanisms of action (MoA) and identifying new therapeutic candidates. In this study, the Cell Painting Assay (CPA) was used to profile 196 small molecules in HCT116 colorectal cancer cells. By applying t-distributed stochastic neighbor embedding (t-SNE) followed by density-based clustering, 18 distinct phenotypic clusters were identified based on similarities in the quantitative morphological profiles generated from Cell Painting data. Although it was initially hypothesized that clustering would reflect known MoAs, most clusters showed only partial overlap with target-based classifications. Instead, compounds from different MoA classes converged on similar cellular phenotypes, suggesting common downstream effects or shared stress responses. Notably, compounds affecting DNA replication, mitosis, or transcriptional control appeared across multiple clusters, indicating functional diversity within morphologically similar groups. Clusters enriched with mTOR/PI3K inhibitors, spindle poisons, or transcriptional CDK blockers exhibited well-defined phenotypes, supporting the robustness of the assay. In contrast, compounds inducing more subtle phenotypes formed distinct micro-clusters, highlighting the method’s sensitivity. Overall, this study demonstrates that CPA can capture convergent phenotypic signatures that extend beyond target-based classification. These findings underscore the value of phenotype-driven screening for the functional annotation of chemical compounds and may help uncover unexpected relationships among molecules with diverse biological activities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b69630fa6518b96d44a14fa030997f1e5bc439" target='_blank'>
              Phenotypic profiling of small molecules using cell painting assay in HCT116 colorectal cancer cells
              </a>
            </td>
          <td>
            Velemir Lavrinenko, Ilona Donskaya, Vladimir Popov, Ekaterina Litau, Varvara Petrova, Stanislav Tyazhelnikov
          </td>
          <td>2025-10-29</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Spatial studies of the tumor microenvironment are crucial for understanding the complex interactions between different cell types and how these interactions contribute to disease progression. This poster presents a streamlined approach to spatial analysis of the tumor microenvironment in brain tissues, focusing on astrocytoma and oligodendroglioma. Additionally, the methodology was applied to Parkinson’s Disease versus healthy brain tissue to observe disease-specific alterations. We employed primary antibody conjugates to perform immunohistochemistry (IHC) on human tissue samples, utilizing an 8-plex panel of markers. The study incorporated IHC-validated antibody conjugates supplemented by an antibody labeling kit for disease-specific markers, facilitating high spatial resolution imaging without the need for lengthy workflows commonly associated with secondary amplification or cyclic stripping methods. Our findings demonstrate a significant increase in astrocytes and oligodendrocytes in the tumor tissue, as revealed by the increased presence of GFAP and Olig2, respectively, compared to healthy controls. The use of the EVOS S1000 imaging system made image capture and analysis remarkably straightforward, further simplifying the workflow. This approach provides clear advantages over traditional methods, highlighting its potential for broad application in immunological and neuropathological studies. For Research Use Only. Not for use in diagnostic procedures.



 Thermo Fisher Scientific



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/957e86629e50682381d4aded4eed8bfc06834bb3" target='_blank'>
              Streamlined spatial analysis of the tumor microenvironment and associated disease states using multiplex immunohistochemistry 9329
              </a>
            </td>
          <td>
            Mae Voeun, C. Langsdorf
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hematopoietic stem cell (HSC) therapy remains essential in treating blood disorders, autoimmune diseases, neurodegenerative conditions, and cancers. Despite its potential, challenges arise from the inherent heterogeneity of HSCs and the complexity of their regulatory niche. Recent advancements in single-cell RNA sequencing and chromatin accessibility sequencing have provided deeper insights into HSC markers and chromatin dynamics, highlighting the intricate balance between intrinsic and extrinsic regulatory mechanisms. Zebrafish models have emerged as valuable tools in HSC research, particularly through live imaging and cellular barcoding techniques. These models have allowed us to describe critical interactions between HSCs and embryonic macrophages, involving reactive oxygen species and calreticulin signaling. These are essential for ensuring HSC quality and proper differentiation, with implications for improving HSC transplant outcomes. Furthermore, the review examines clonal hematopoiesis, with a focus on mutations in epigenetic regulators such as DNMT3A, TET2, and ASXL1, which elevate the risk of myelodysplastic syndromes and acute myeloid leukemia. Emerging technologies, including in vivo cellular barcoding and CRISPR-Cas9 gene editing, are being investigated to enhance clonal diversity and target specific mutations, offering potential strategies to mitigate these risks. Additionally, macrophages play a pivotal role in maintaining HSC clonality and ensuring niche localization. Interactions mediated by factors such as VCAM-1 and CXCL12/CXCR4 signaling are crucial for HSC homing and the stress response, opening new therapeutic avenues for enhancing HSC transplantation success and addressing clonal hematopoiesis. This review synthesizes findings from zebrafish models, cutting-edge sequencing technologies, and novel therapeutic strategies, offering a comprehensive framework for advancing HSC biology and improving clinical outcomes in stem cell therapy and the treatment of hematologic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6985cdf565a94b5e4ce65dadd2a9ba4efa0f62f7" target='_blank'>
              Advancements in Hematopoietic Stem Cell Therapy: From Biological Pathways to Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Viviana Cortiana, H. Chorya, R. H. Abbas, Jade Gambill, Adhith Theyver, Chandler Park, Yan Leyfman
          </td>
          <td>2025-10-15</td>
          <td>Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 The World Health Organization provides clear criteria for diagnosing IDH1-wildtype glioblastoma (GBM), relying on traditional approaches to define pathoetiological features, genomic mutations, copy number variation, and aberrant expression of prognostic genes. While these methods guide diagnosis, they may lack the resolution to fully capture tumor heterogeneity, therapeutic response, or outcomes. To address this, we applied leading-edge technologies to 14 primary and recurrent tumors to add granularity to current diagnostic metrics and clinical classification. Aiming to better inform tumor subtyping and improve patient care, this integrative approach enabled us to identify malignant gene programs and localize prognostically relevant features at both single-cell and spatial levels. We analyzed 14 GBM tumors from 6 patients using spatial in situ sequencing (ISS) and single-nuclei RNA-sequencing. A custom ISS gene panel enabled spatial mapping of tumor-associated cell types and state markers, while matched snRNA-seq from adjacent tumor tissue provided transcriptome-wide profiles. One patient contributed 5 serially collected recurrent tumors, enabling longitudinal analysis of tumor evolution. Profiling these tumors using snRNA-seq identified canonical copy number alterations (+7/–10) and ligand-receptor interaction analysis showed an evocative ANXA1 signaling axis progress across recurrent tumors, which suggests possible therapeutic opportunities. Finally, the dataset enabled subcellular and spatial localization of the prognostic GBM biomarker, EGFRvIII, being enriched in oligodendrocyte progenitor (OPC)-like tumor cells, which provides exciting evidence of these cancer cell subtypes as the putative cell of origin for GBM. By extending beyond conventional diagnostic benchmarks, our molecular profiling reveals new insight into GBM biology that may support more refined tumor subtyping and prognostic assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb079e9e9eb8a8e1b502483557634499fe4e750" target='_blank'>
              BIOM-72. INTEGRATIVE SINGLE-NUCLEI SPATIAL TRANSCRIPTOMIC PROFILING OF PRIMARY AND RECURRENT HUMAN IDH-WILDTYPE GLIOBLASTOMA REVEALS CELL-SPECIFIC EGFRVIII EXPRESSION AND ANXA1 SIGNALING AXIS
              </a>
            </td>
          <td>
            Lauren Powers, Vaibhav Jain, Michael Aksu, Emily Hocke, Alan Smith, Stephanie Arvai, Ellora Haukefrers, Kevin Stevenson, Khooshbu Patel, Diane Satterfield, Elizabeth Thomas, Karen Abramson, Giselle López, R. McLendon, David M. Ashley, Simon Gregory
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Uveal melanoma (UM) is the most common intraocular malignancy, with poor prognosis in metastatic cases and limited response to conventional therapies. Despite advances in genetic stratification, the immunological landscape of primary UM remains poorly understood. Secondary data generation of single-cell RNA sequencing (scRNA-seq) of primary class 1 and class 2 (loss of BAP1) UM tumours and flow cytometric analysis of 8 primary UM tumour biopsies were used to characterize the tumour microenvironment, cellular composition, tumour–immune cell interactions, and stromal marker expression associated with tumour progression and immune infiltration. scRNA-seq analysis revealed 16 distinct cell clusters, including melanocytes, T cells, macrophages, and stromal cells. Class 2 tumours contained unique melanocyte subpopulations exhibiting chromosome 8 copy number variations and enriched in hypoxia, PI3K-Akt, and MAPK signalling pathways. Ligand–receptor analysis identified extensive interactions between these aggressive melanocytes and pericytes/macrophages. Flow cytometric analysis confirmed two distinct immune infiltrate profiles: low-infiltrate tumours dominated by CD14⁺ cells, and high-infiltrate tumours with CD8⁺ memory-like T cells expressing PD-1 and CD27. Stromal marker analysis revealed elevated expression of CD81 and NGFR in immune-excluded tumours, implicating them in metastatic potential. Our study reveals cellular and immunological heterogeneity within primary UM tumours. The identification of immunologically distinct tumour types, along with aggressive melanocyte subpopulations and stromal interactions, provides insight into UM pathogenesis and supports stratified immunotherapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddff61a0a880b01ec6e97ff402716c2ca58aa31b" target='_blank'>
              Identification of two distinct uveal melanoma subtypes based on the immune cell infiltration of the primary tumour
              </a>
            </td>
          <td>
            Konstantinos Panagiotidis, S. Owens, Gráinne Finegan, Noel Horgan, Stefanos Tsavdaridis, S. Kennedy, Achilleas Floudas
          </td>
          <td>2025-10-23</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma (GBM), a highly aggressive malignant brain tumor, leads to poor clinical outcomes and low long-term survival rates. Using bioinformatics approaches, we analyzed single-cell RNA sequencing (scRNA-seq) data (GSE273274) from glioma patient samples in this study. It was found that mesenchymal-like (MES-like) cells in the GBM center (GC) significantly facilitated GBM’s aggressive spread and microenvironmental reprogramming. Further analysis on MES-like cells suggested that these cells may participate in the progression from glial differentiation to invasive metastasis. Differential expression analysis, univariate Cox regression, and LASSO regression identified COL22A1 as a key gene. COL22A1 was highly expressed in GC compared to GBM surrounding tissue (GS) and exhibited potential connection to aberrant phosphatidylinositol 3-kinase (PI3K) signaling activation, indicating that COL22A1 may act as a potential regulator of GBM, promoting malignant processes such as proliferation, apoptosis resistance, migration, and invasion. In vitro experiments showed that COL22A1 overexpression (COL22A1-OE) U-87 cells exhibited enhanced proliferation, viability, migration, and invasion. Moreover, immunoblotting of BCL-2 and BAX revealed that COL22A1-OE increased resistance to apoptosis in U-87 cells. However, all these effects were reversed upon silencing of COL22A1, which suggests COL22A1 could be a promising new target for GBM therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a091b05d62e57c97e8673640ca9464f145f296ba" target='_blank'>
              Malignant progression of MES-like cells mediated by COL22A1 in the spatial heterogeneity of glioblastoma
              </a>
            </td>
          <td>
            Guangyuan Chen, Zhonghua Fu, Xulin He, Zhaoli Shen
          </td>
          <td>2025-10-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cooperation among phenotypically distinct sub-populations within a tumor plays a key role in cancer progression. In this study, we investigated how proteolytic heterogeneity supports collective cancer invasion. In invasive MDA-MB-231 breast cancer cells which exhibit considerable variability in MMP9 expression, we show that MMP9 knockdown cells are notably smaller and softer than control cells. A computational model revealed that the invasiveness of mixed clusters containing both proteolytic and non-proteolytic cells depends on cell-cell adhesion, with non-proteolytic cell invasion requiring close proximity to proteolytic neighbors. When we assigned non-proteolytic cells the same size and stiffness as proteolytic ones, the overall invasiveness declined–highlighting that small size and deformability of non-proteolytic cells are essential for sustained collective invasion. We validated these predictions experimentally using spheroid invasion assays showing that mixed spheroids of control and MMP9 knockdown cells are the most invasive. Together, our findings demonstrate that interplay between MMP9 expression and biophysical properties enables collective invasion through enrichment of and matrix degradation by high MMP9 expressing cells at the invasive front, and squeezing of low MMP9 expressing cells through the remodeled matrix.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc1f02ce1d96b32eaece7abb515cefba043a732" target='_blank'>
              MMP9 shapes cell mechanics to enable collective invasion in cancer
              </a>
            </td>
          <td>
            Sarbajeet Dutta, Sumon Kumar Saha, Anisha Karmakar, Nikita Sharma, Sudikshaa Vijayakumar, Shamik Sen, Sarbajeet Asadullah, Sumon Kumar Dutta, Anisha Karmakar Saha
          </td>
          <td>2025-10-29</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Muscle-invasive (MI) urothelial carcinoma (UC) is a clinically challenging malignancy with a poor prognosis. Understanding the cellular dynamics that drive UC progression is critical for the development of optimized therapeutic strategies. Through integrative analysis of large-scale single-cell transcriptomic datasets from non-muscle-invasive (NMI) and MI tumours and validation with spatial transcriptomic datasets, we systematically characterized immune cell dynamics and cancer cell plasticity during UC progression. Our analysis revealed an immunosuppressive tumour microenvironment and a subset of cancer cells with upregulated major histocompatibility complex II (MHC-II) expression in MI tumours. Notably, MHC-II⁺ cancer cells were induced by interferon-γ signalling, as confirmed through in vitro experiments, and exhibited phenotypic alterations characterized by enhanced proliferative and migratory capacities. Furthermore, MHC-II⁺ cancer cells spatially colocalized with CD8⁺ T cells, regulatory T cells, and SPP1⁺ macrophages, where they engaged with inhibitory receptors on these immune cells, promoted CD8⁺ T cell exhaustion and facilitated immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27b9542e0889bc085781e9b9db8fbe77cfa48b4c" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveal cellular heterogeneity and cancer cell-intrinsic major histocompatibility complex II expression in urothelial carcinoma
              </a>
            </td>
          <td>
            S. Xiong, Jian Fan, Cong Huang, Shi-ming He, Yuan Liang, Qi Zhang, Bin Guo, Yucai Wu, Mancheng Xia, Fangzhou Zhao, W. Ci, Liqun Zhou, Yanqing Gong, Xuesong Li
          </td>
          <td>2025-10-20</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39182f5f4a0a9fe0e0a7bf848e1130bc43d58a1d" target='_blank'>
              A unified multimodal single-cell framework reveals a discrete state model of hematopoiesis in mice
              </a>
            </td>
          <td>
            Kyle Ferchen, Xuan Zhang, Kairavee Thakkar, Guangyuan Li, David Bernardicius, Sidharth Sen, Priyanka Rawat, André Olsson, Sierra N Bennett, Crystal Potter, Fred D. Finkelman, Josh Croteau, Samantha A. Morris, Harinder Singh, Nathan Salomonis, Leighton Grimes
          </td>
          <td>2025-10-22</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Resistance to chemotherapy, targeted agents, and particularly immunotherapy remains the principal challenge in the management of gastrointestinal malignancies. This review aims to comprehensively delineate the molecular and microenvironmental drivers of resistance, with emphasis on mechanisms impacting immunotherapy response, and evaluate emerging, mechanism−guided interventions (including immunotherapeutic combinations) for precision therapy. We first examine intrinsic mechanisms—including drug−target alterations, dysregulated drug metabolism and efflux, hyperactivation of DNA damage repair pathways, and epigenetic remodeling—and extrinsic influences stemming from the tumor microenvironment and extracellular matrix remodeling. We then highlight epithelial–mesenchymal transition (EMT) as a critical nexus that integrates stromal cues with cell−intrinsic survival programs, thereby promoting drug efflux and immune evasion. Next, we discuss how single−cell and spatial omics, liquid biopsy, patient−derived organoids, and AI−enabled analytics facilitate subclone−level mapping of resistance networks and real−time tracking of clonal evolution. Finally, we review mechanism−based strategies—including KRAS G12C inhibitors, efflux−pump antagonists, apoptosis reactivators, and epigenetic/autophagy modulators—and propose an integrated, multimodal regimen leveraging immunotherapy where appropriate, informed by real-time drug sensitivity data (e.g., from liquid biopsy), dynamic biomarkers and AI−driven optimization to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebd9ae366f20576b8816fc8c01cf3154ec38755" target='_blank'>
              Integrated molecular and microenvironmental drivers of drug resistance in gastrointestinal cancers: mechanisms, immunotherapy challenges, and precision strategies
              </a>
            </td>
          <td>
            Heng Xu, Jiaan Lu, Jiangying Wu, Kangling Zhang, Xuancheng Zhou, Ziqi Gao, Jingqi Feng, Ziye Zhuang, Xiaolin Zhong
          </td>
          <td>2025-10-28</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="(1) Background: Hepatocellular carcinoma (HCC) remains a major cause of cancer mortality and is characterized by pronounced inter- and intra-tumoral heterogeneity and therapy resistance. We aimed to define core transcriptional circuits that drive HCC malignancy and to delineate how these programs shape the tumor microenvironment (TME). (2) Methods: We integrated single-cell, spatial, and bulk transcriptomic datasets from public cohorts. (3) Results: We identified nine tumor-restricted transcription factors (TFs)—HTATIP2, HES6, ILF2, E2F1, MYBL2, DDIT3, FOXM1, HMGA1, and ETV4—whose expression and regulon activity associated with malignant phenotypes and poor survival. These TFs organize a progression axis from an early proliferative state (cluster C4) toward an invasive, metabolically adapted state (cluster C1) enriched for hypoxia, epithelial–mesenchymal transition (EMT), and inflammatory signaling. The C1 state remodeled the TME by establishing an immunosuppressive niche marked by expansion of T regulatory cells (Treg) and by accumulation of SPP1+ macrophages. These macrophages, recruited and polarized by C1 tumor cells, exhibited M2-like, pro-angiogenic, and immunosuppressive features and engaged epithelial, immune, and stromal partners via SPP1-CD44 and SPP1-integrin interactions. (4) Conclusions: In summary, a tumor-intrinsic TF network cooperates with SPP1+ macrophage signaling to promote a permissive microenvironment and HCC progression. This integrated axis highlights tractable vulnerabilities for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989aaab1ae1f0d6fa871b18d9e4cad44d18f7594" target='_blank'>
              Transcription Factor Networks Drive Tumor Progression and Immune Microenvironment Remodeling in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Sang Hoon Lee, Ju Won Ahn, Wonbin Choi, Jina Kim, Joon Yeon Hwang, Jae-Hwan Kim, Hyaekang Kim, Woori Kwak
          </td>
          <td>2025-11-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Traditional pooled genetic dependency screens, such as DepMap, have proven powerful for identifying essential genes in cancer cell lines. However, these pooled formats are prone to the bystander effect, where secreted factors from wild-type cells support the survival of knockout (KO) cells within the same culture, potentially masking dependencies on specific secreted proteins. To address this limitation, we performed a well-by-well CRISPR screen targeting genes encoding secreted factors in the pancreatic ductal adenocarcinoma (PDAC) cell line HPAFII. This arrayed format eliminates cross-feeding and enables precise assessment of cell-intrinsic dependencies on targeted proteins and could thus expose hidden vulnerabilities. CellTiter-Glo (CTG) viability assays were used to assess gene lethality following target gene KOs. Furthermore, RNA sequencing verified effective disruption of target genes, establishing the specificity and reliability of the screening workflow. We observed strong growth inhibition in EFNA1 (Ephrin-A1) KO samples compared to lipofectamine-treated controls. However, this inhibitory effect was missed and underrepresented in the conventional pooled format. EFNA1 is a ligand for EPH receptors, involved in cell-cell communication, migration, and proliferation signaling. Differential gene expression analysis comparing EFNA1 KO to lipofectamine-treated controls revealed enriched pathways of oxidative phosphorylation, E2F targets, MYC targets, and DNA repair in EFNA1 KO samples. This result supports our observation of lethality in EFNA1 KO samples. It suggests that EFNA1 contributes to cell-cell communication and is critical for HPAFII cell survival. Its KO triggers stress pathways, including E2F, MYC, OXPHOS, and DNA repair, and acts as a suppressor of excessive proliferative programs. In conclusion, we identified EFNA1 as a critical non-cell-autonomous essential gene through an arrayed CRISPR screen, with its loss showing lethality to HPAFII cells exclusively in isolated settings, likely as a result of disrupted cell-cell interactions. Defining such dependencies may facilitate the discovery of directly druggable and more tumor-selective therapeutic strategies.



 Xingyi Shi, David Ruddy, Millicent Gabriel, Michelle Piquet, Kris Muskiewicz, Joel Wagner, Joshua Korn, Cory Johannessen. Arrayed CRISPR Screening Reveals EFNA1 as a Non-Cell-Autonomous Essential Gene in Pancreatic Cancer Cells [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd6ce939e1194e9cb9b823d2bb2e7b39fb1e35f" target='_blank'>
              Abstract C035: Arrayed CRISPR Screening Reveals EFNA1 as a Non-Cell-Autonomous Essential Gene in Pancreatic Cancer Cells
              </a>
            </td>
          <td>
            Xingyi Shi, David A Ruddy, Millicent Gabriel, M. Piquet, Kris Muskiewicz, Joel P Wagner, Joshua Korn, Cory Johannessen
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains a highly aggressive brain tumor with limited therapeutic options and significant immune evasion. Tumor-associated regulatory T cells (Tregs) and macrophages (TAMs) are central orchestrators of the GBM immune microenvironment; however, the dynamics and subtype-specific interactions between these populations during disease progression are still poorly understood. In this study, we utilized complementary mesenchymal and proneural murine GBM models to examine the differentiation trajectories and functional crosstalk of Tregs at single-cell resolution. Single-cell RNA sequencing (scRNA-seq) of tumor-infiltrating Tregs revealed that mesenchymal GBM promoted the expansion of an immunosuppressive Treg population characterized by elevated CCR8 (C-C chemokine receptor type 8), GITR (Glucocorticoid-Induced TNFR-Related protein), and IL1R2 (Interleukin 1 Receptor Type 2), which was less pronounced in proneural GBM. Trajectory analysis uncovered enhanced proneural-to-mesenchymal transition signatures, indicating dynamic Treg adaptation to evolving tumor states. Validation in human spatial transcriptomic GBM datasets revealed that the murine tumor Treg signature was enriched within mesenchymal regions and closely associated with myeloid-dense microenvironments. By leveraging a time-course scRNA-seq dataset from the mesenchymal GBM model, we observed a progressive expansion of the mesenchymal-like Treg signature paralleling tumor evolution and mesenchymal transition. Notably, CellChat-based cell–cell communication analysis demonstrated that Treg–myeloid interactions were not only enhanced over time but also that Treg subsets differentially engaged with immunosuppressive TAMs through unique ligand–receptor pairs, underscoring the dynamic and reciprocal adaptation between regulatory and myeloid compartments. Collectively, our study delineates a model in which GBM progression is shaped by the evolving interplay of Treg and myeloid programs, providing a framework for targeting context-specific immunosuppressive circuits in future immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a27d9dd70e278b9475e8a73470ac145030c21c" target='_blank'>
              IMMU-86. Dynamic Treg-myeloid crosstalk dictates glioblastoma progression
              </a>
            </td>
          <td>
            Sadashib Ghosh, Jeffrey M. Grimes, Stephen Opoku, J. Leavenworth
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Dedifferentiation programs are commonly enacted during breast cancer progression to enhance tumor cell fitness. Increased cellular plasticity within the neoplastic compartment of tumors correlates with disease aggressiveness, often culminating in greater resistance to cytotoxic therapies or augmented metastatic potential. Here, we found that subpopulations of dedifferentiated neoplastic breast epithelial cells express canonical leukocyte cell surface receptor proteins and have thus named this cellular program "immune mimicry." Analysis of public human breast tumor single-cell RNA-sequencing datasets and histopathological breast tumor specimens, as well as functional experiments in vitro in breast cancer cell lines and in vivo in murine transgenic and cell line-derived mammary cancer models, showed that neoplastic cells engaged in immune mimicry. Immune-mimicked neoplastic cells harbored hallmarks of dedifferentiation and were enriched in treatment-resistant and high-grade breast tumors. In aggressive breast cancer cell lines, anti-proliferative cytotoxic chemotherapies drove epithelial cells toward immune mimicry. Expression of the CD69 leukocyte activation protein by neoplastic cells conferred a proliferative advantage that facilitated early tumor growth. Together, these findings suggest that neoplastic breast epithelial cells upregulating leukocyte surface receptors potentiate malignancy and that neoplastic immune mimicry has potential clinical utility for patient prognosis and stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb7d3c8fc355d40193e7476934f2c0c4a7fa588a" target='_blank'>
              Neoplastic Immune Mimicry Potentiates Breast Tumor Progression.
              </a>
            </td>
          <td>
            Eric B Berens, Sokchea Khou, Elaine T Huang, Amber Hoffman, Briana Johnson, Nell Kirchberger, Shamaline Sivagnanam, Nicholas L. Calistri, Daniel Derrick, T. Liby, Ian C. Mclean, Aryn A Alanizi, Furkan Ozmen, T. Ozmen, G. Mills, E. S. Hwang, P. Schedin, Hugo Gonzalez Velozo, Z. Werb, Laura M Heiser, L. Coussens
          </td>
          <td>2025-10-23</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="Understanding tumor heterogeneity is essential for advancing cancer treatment. Clonal reconstruction methods play a pivotal role in deciphering this heterogeneity. Our goal is to develop a clonal reconstruction approach that is clinically applicable, easy to implement, and capable of delivering both high-speed performance and excellent reconstruction accuracy. We present MyClone, a probabilistic method designed to reconstruct the clonal composition of tumors using deep sequencing genomic data. MyClone processes read counts and copy number information of single nucleotide variants derived from deep sequencing data, enabling it to determine the mutational composition of clones and the cancer cell fractions of these mutations. Compared to existing clonal reconstruction methods, MyClone enhances clustering accuracy and cancer cell fraction prediction when applied to deep-targeted sequencing data and bulk tumor sequencing data with deep sequencing coverage. Additionally, MyClone achieves a substantial improvement in computational speed. We rigorously validated MyClone’s performance using both simulated and real clinical datasets and applied it to analyze a circulating tumor DNA sequencing dataset from 139 metastatic breast cancer patients. In this analysis, we explored mechanisms of drug resistance in metastatic breast cancer and identified 10 mutated genes potentially associated with drug resistance or sensitivity. For deeply sequenced data, MyClone outperforms existing methods on both targeted sequencing and bulk tumor data. Its high computational efficiency and reconstruction accuracy position MyClone as a promising tool for broad clinical application in cancer treatment. The source code is publicly available at https://github.com/Hansen0413/myclone-code.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/008b5a4c97ae8500318c69c4b98363d3ad251446" target='_blank'>
              MyClone: rapid and precise reconstruction of clonal population structures for tumors
              </a>
            </td>
          <td>
            Qihan Guo, Dan Lv, Bo Lan, Haili Qian, Fei Ma, Ting Chen
          </td>
          <td>2025-10-08</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Epithelial-mesenchymal transition (EMT) generates heterogeneity in circulating tumor cells (CTCs), affecting their biological properties and hampering their detection. This limits our understanding of the mechanisms underlying hematogenous dissemination, especially in early breast cancer (BC), where CTCs are rare. Here, we aimed to detect CTCs with different EMT statuses from BC patients. CTCs in blood samples from 107 BC patients were evaluated using immunomagnetic depletion and multi-marker immunofluorescence (EpCAM, E-cadherin, MCAM, cell surface vimentin, CD31, CD45), followed by single-cell transcriptomics. CTCs were detected in 51.9% of therapy-naïve early BC cases, with 3.8% showing only epithelial CTCs (eCTCs), 5.8% epithelial-mesenchymal (emCTCs), 26.0% mesenchymal (mCTCs), and 16.3% mixed phenotypes. CTC heterogeneity was more frequent in triple-negative (86%) than in luminal BC (17%, P = 0.008). Lymph node involvement strongly predicted dissemination of all CTC phenotypes, while tumor size correlated with mCTC abundance. Single-cell RNA sequencing revealed downregulation of ribosomal genes and translation inhibition in CTCs with mesenchymal features, linked to mTORC1 signaling. Findings were also validated in an independent dataset, highlighting vulnerabilities in CTCs during dissemination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93946204244173c531d16f8507a787b8cf8bd343" target='_blank'>
              Characterizing epithelial-mesenchymal transition-linked heterogeneity in breast cancer circulating tumor cells at a single-cell level.
              </a>
            </td>
          <td>
            Justyna Topa, J. Richert, Tomasz Stokowy, Alicja Staśczak, Mariusz Szajewski, Maciej Ciesielski, Petra M Grešner, Bartłomiej Tomasik, Łukasz Arcimowicz, Agnieszka Stankiewicz, G. Suchodolska, E. Senkus, W. Kruszewski, A. Żaczek, A. Markiewicz
          </td>
          <td>2025-10-05</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Glioblastomas function as intricate cellular networks that extend into the surrounding brain tissue enabling long distance communication. This malignant connectivity spans from the tumor core to remote infiltration zones, supporting the concept of glioblastoma as a whole-brain disease. With growing ethical considerations in biomedical research and the limitations inherent in cross-species comparisons between animal studies and human glioblastoma models, human tumor platforms remain constrained in their ability to replicate the full infiltration spectrum from the tumor core to single-cell dispersion. Here, we present a three-dimensional, fully human ex-vivo glioblastoma model („Core2Edge”) that replicates this extensive infiltration range while preserving the intratumoral heterogeneity of the original tumor. This model involves implanting fluorescently labeled human glioblastoma organoids (GBOs) into organotypic human brain slices, maintaining the genetic integrity and cytoarchitecture of both brain and tumor.



 The Core2Edge model involves a comprehensive workflow completed over 7–12 days. Key steps include brain slice preparation (~4-6 hours, depending on quantity), one day for initial culture before GBO staining and transplantation, a variable culture period (up to ten days), and fixation with different techniques (~8 hours). Following model preparation, downstream analyses such as imaging, sequencing, and proteomics were performed.



 Multiplex immunofluorescence confirmed cell composition consistent with histological staining. Spatial transcriptomics enabled detailed glioblastoma cell subtyping based on Neftel states. Advanced imaging, combining tissue expansion and light-sheet fluorescence microscopy, facilitated super-resolution, three-dimensional visualization of the GBO-brain slice system. This approach allows for the study of initial infiltration steps, in-depth analysis of the invasive front, exploration of cell-cell interactions between tumor cells and the tumor microenvironment, and offers a platform for drug screening and testing, reducing the need for animal models.



 The „Core2Edge“model represents an innovative ex-vivo platform replicating glioblastoma infiltration and heterogeneity. By preserving genetic integrity and cytoarchitecture, this fully human model bridges critical gaps in glioblastoma research, offering unparalleled insights into tumor behavior and potential therapeutic interventions.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f77d56f46efc9d565ffafe70a0c41870c961aee" target='_blank'>
              TMOD-28. Core2Edge: A human glioblastoma organoid and brain slice co-culture model capturing tumor infiltration and transcriptional heterogeneity from core to single-cell dispersion
              </a>
            </td>
          <td>
            A. Potthoff, A. Melhem, Barbara E F Pregler, J. Rodriguez-Gatica, L. Friker, Jake Thomas, Valeri Borger, M. Toma, M. Westhoff, V. Ravi, D. Heiland, A. Schlitzer, Torsten Pietsch, M. Hölzel, A. Waha, H. Vatter, M. K. Schwarz, U. Herrlinger, U. Kubitscheck, Matthias Schneider
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 Spatial biology is rapidly emerging as a powerful tool for deepening our knowledge of the complex, dynamic nature of tissue-specific immune responses. It involves investigating the interactions between lymphoid cells, antigen-presenting cells, and non-lymphoid cells within tissues. Multiplex immunohistochemistry (IHC) and immunofluorescence (IF) are ideal techniques for characterizing the spatial aspects of immune responses. To streamline accurate marker detection, emerging automated staining and imaging platforms require qualified antibodies with platform-specific and proven specificity and sensitivity. Using IHC antibodies from R&D SystemsTM, we used a systematic process to qualify antibodies for use on the Lunaphore COMET™ platform in murine lymphoid tissue. COMETTM offers a novel sequential immunofluorescence (seqIF™) approach that utilizes cycles of staining, imaging, and elution with unconjugated primary antibodies and fluorescently labeled secondary antibodies. In this study, we distinguish important cell types, including pan-cytokeratin+ epithelial cells, CD45+ hematopoietic cells, and CD31+ endothelial cells. Simultaneously, we can further characterize hematopoietic cells using phenotypic markers, including Ki67, the regulatory transcription factor FoxP3, and the activation marker CD86. This approach provides a powerful platform for characterizing tissue-specific immune responses, differentiating disease states, and identifying potential therapeutic targets.



 Technological Innovations in Immunology (TECH)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cafcb0d97c16e1cc46bb1d13d4f71af1e136bee6" target='_blank'>
              Spatial Distribution and Characterization of Mouse Lymphoid Tissue Using Fully Automated Multiplex Immunofluorescence on COMETTM 3909
              </a>
            </td>
          <td>
            Ruha Adelkar, Emily K Cartwright, A. Kalyuzhny, Jodi Hagen, Ashley Oliver, Brandon Wagner
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gastric cancer (GC) is a highly aggressive malignancy with a poor prognosis, closely linked to the tumor microenvironment (TME). Emerging evidence highlights the critical role of gastric cancer-associated mesenchymal stem cells (GC-MSCs) in recruiting neutrophils and facilitating neutrophil extracellular traps (NETs) formation, thereby remodeling the tumor microenvironment (TME) and promoting tumor progression, immune modulation, and metastasis. This study integrated single-cell RNA sequencing (scRNA-seq), Mendelian randomization (MR), and functional enrichment analyses to uncover the molecular underpinnings of GC. Transcriptomic data from public databases, including TCGA and GEO, were analyzed to explore cellular heterogeneity and the influence of NETs within the TME. MR analysis was conducted to establish causal relationships between key NET-related genes and GC. We integrated scRNA-seq, MR, and functional enrichment analyses to systematically dissect the roles of GC-MSCs and NETs in reshaping the tumor microenvironment. scRNA-seq identified 12 distinct cell types in the TME, with significantly elevated NET scores associated with GC-MSCs in the disease group. Functional enrichment analyses revealed that NET-associated marker genes were enriched in pathways such as oxidative phosphorylation and cytoplasmic translation. MR analysis confirmed EIF1 and RPS12 as key genes with causal links to GC progression, demonstrating robust associations with immune cell infiltration and critical signaling pathways. Additionally, transcriptional regulation analysis identified motifs and transcription factors governing these genes, while GSEA highlighted their involvement in pathways such as ribosome biogenesis and focal adhesion. This study elucidates the intricate interplay between GC-MSCs, NETs, and the immune microenvironment, offering novel insights into GC pathogenesis and potential therapeutic targets. EIF1 and RPS12 emerge as promising candidates for precision medicine, with broader implications for NET-related cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/616c438e8db3d32699e9535d88f238cc68085b7c" target='_blank'>
              Unveiling the role of gastric cancer-associated mesenchymal stem cells and neutrophil extracellular traps through multi-omics analysis
              </a>
            </td>
          <td>
            Zhengrui Li, Peng Luo, Zhaokai Zhou, J M Cao, Wen Zhang
          </td>
          <td>2025-11-20</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5bf5750b51fd180863b176de6d4f229a2cae4e" target='_blank'>
              Single-cell and spatial transcriptomics reveal lactylation-associated tumor cell clusters and define a prognostic risk model in glioblastoma.
              </a>
            </td>
          <td>
            Rui Han, Guangfan Chi, Dongjie Sun, Ziran Xu, Liangfu Zhou, Kan Xu
          </td>
          <td>2025-11-22</td>
          <td>BMC cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b5111159e92c7f129ff30372c9b9934bfcdb2c" target='_blank'>
              Single-cell RNA sequencing and large-scale bulk combination with machine learning reveal gastric cancer-related macrophage heterogeneity
              </a>
            </td>
          <td>
            Saisai Gong, Yu peng Zhao, Sheng Yang, Tong Liu, Chengyun Li, Yun Zhou, Geyu Liang
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce05a3966abc2d6663f2f9fc06215e6d78446ea" target='_blank'>
              Tumor microenvironment-mediated immune evasion and resistance in prostate cancer: mechanisms, cross-talk, and therapeutic opportunities.
              </a>
            </td>
          <td>
            Mingfeng Li, Louis Boafo Kwantwi, Cuimei Wang, Qin Xiao
          </td>
          <td>2025-11-26</td>
          <td>Clinical and experimental medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Diffuse midline gliomas (DMGs) are universally fatal pediatric brain tumours defined by their midline localization, early onset, and distinct mutational landscape. The H3.3K27M mutation is found in approximately 80% of DMGs, marking it as a key initiating event. However, H3.3K27M alone does not induce brain tumour formation in experimental models, suggesting that its oncogenic potential may be linked to specific developmental windows and susceptible precursor populations. Identifying the cellular states that are vulnerable to transformation is critical to understanding DMG initiation. To establish a reference framework for these vulnerable states, we generated a single-cell RNA sequencing atlas of normal hindbrain development. Using a Sox2-eGFP reporter mouse and FACS-based enrichment of Sox2+ precursors, we profiled over 120,000 cells across ten developmental stages, capturing critical transitions in neurogenesis and gliogenesis. To reveal regional differences, we integrated the atlas with a similarly generated forebrain atlas, providing a comprehensive map of neurodevelopmental trajectories where DMG mutations might exert their effects. Building on this developmental reference, we utilized inducible mouse models to introduce H3.3K27M mutations in precursors and employ lineage tracing to track their fate over time. When combined with loss of Trp53 and overexpression of PDGFRa, alterations prevalent in DMG, mice develop tumours that resemble the human disease. H3.3K27M-driven tumours consistently localize to the pons, in contrast to H3 wild-type tumours, which occur more variably across the forebrain and hindbrain. Tracking these pontine tumours using serial MRI revealed that tumours arise well before symptom onset and exhibit growth dynamics that vary by location and timing. Earlier-arising tumours are more aggressive and tend to occur in posterior locations. Together, our integrated approach combining lineage tracing, tumour modelling, and a developmental atlas provides a powerful platform to dissect the early events in DMG pathogenesis and identify developmental vulnerabilities that may be targeted for early intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0a1a4d1a35cbc94502034b6562096dc3dd8efc6" target='_blank'>
              STEM-27. TRACING THE ORIGINS OF H3.3K27M MUTANT DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            Caitlin Herrington, Evan Lubanszky, Quang Trinh, Luiza Lopes Pontual, Jannine Forst, Cynthia Hawkins, Peter Dirks
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neoadjuvant therapy plays an important role in the treatment of glioblastoma (GBM), but a considerable proportion of patients remain unresponsive to the combination of immune checkpoint blockade (ICB) and antiangiogenic therapy. Understanding the mechanisms underlying resistance to this treatment and developing novel therapeutic strategies are crucial. We integrate extensive single-cell and spatial transcriptomic data to dissect the cellular composition and spatial organization of the GBM tumor microenvironment and validate our findings through experiments such as multiplex immunohistochemistry and atomic force microscopy. We applied 101 machine learning algorithms to evaluate the prognostic and immunological value of COL6A3+ tumor-associated fibroblasts (TAFs) and GPNMB+ monocyte-derived macrophages (MDMs) in multiple GBM cohorts and immunotherapy cohorts. We constructed a stromal cell atlas in GBM and identified a distinct subset of COL6A3+ TAFs with functional characteristics of matrix fibroblasts. We found that COL6A3+ TAFs are significantly enriched in non-responders to neoadjuvant combination therapy. These fibroblasts drive the spatial-reprogramming of anti-tumorigenic MDMs into a pro-tumorigenic phenotype. In turn, these reprogrammed immunosuppressive GPNMB+ MDMs promote vascular fibrosis mediated by COL6A3+ TAFs through the GPNMB-ITGB5 interaction. Our findings highlight the critical role of COL6A3+ TAFs in regulating MDM function and spatial distribution, as well as their contribution to fibrotic tumor vasculature formation. Additionally, we propose targeting COL6A3+ TAFs with cilengitide as a potential therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/430d20f3bfe099d744acf27ed5b371550f5b9883" target='_blank'>
              Spatial-reprogramming derived GPNMB+ macrophages interact with COL6A3+ fibroblasts to enhance vascular fibrosis in glioblastoma
              </a>
            </td>
          <td>
            Yinfei Du, Xinmiao Long, Xuetong Li, Fan Guan, Wei Gao, Kun-Kang Deng, Shiyi Wang, Xiang Lin, Meng Huang, X. She, Shuai Chen, Minghua Wu
          </td>
          <td>2025-10-31</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9eea4d17ccda6da786c821ed59ad42654c9c3e4" target='_blank'>
              Tumor microenvironment in breast cancer: cellular crosstalk, pathways, and therapeutic insights
              </a>
            </td>
          <td>
            Manjari Sharma, Umesh Kumar, Shoorvir Singh
          </td>
          <td>2025-11-04</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The tumor microenvironment is highly heterogeneous, composed of diverse cell types with distinct phenotypes and characteristics. Gaining insights into these differences is essential for advancing cancer immunotherapy and research. Reliable results demand high-quality samples and standardized processing, however, generating single-cell suspensions from solid tumors is challenging due to variations in tumor type, size, texture, and developmental stage. Balancing enzymatic and mechanical dissociation is critical to preserve cell viability and yield. We developed the STEMprep™ Mouse Tumor Dissociation Kit for use with the STEMprep™ Automated Tissue Dissociation System, featuring a user-friendly instrument and specially designed sample tubes. This system includes an optimized enzyme formulation and a single protocol that works across diverse tumor types. Our findings demonstrate consistently high cell viability and yield (cells/mg tissue) across tumors: soft B16 melanoma (91.2 ± 3.3% and 1.5E4 ± 7.4E3 [n = 15]), medium-firm CT26 colon carcinoma (82.9 ± 6.1% and 2.6E4 ± 1.3E4 [n = 9]), and firm 4T1 mammary carcinoma (88.5 ± 4.0% and 2.7E4 ± 1.7E4 [n = 4]). This method maintains cell integrity and epitope availability for downstream analysis. Furthermore, STEMprep™-processed samples are compatible with EasySep™ magnetic cell isolation technology, and are functional in T cell suppression assays, enhancing tumor research workflows with efficiency, consistency, and reliability.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15528cd65eafa51df7aab4672196a35521811783" target='_blank'>
              Streamlined mouse tumor processing with STEMprep™: Automated, Efficient, and Reliable 3260
              </a>
            </td>
          <td>
            F. Antignano, G. Poon, So Ra Lee, Jenna Thompson, A. Chenery, Alice Liang, Payam Zachkani, Andrew Nobles, Bilal El yassem, Chris Ryan, Marc Delorme, Rodrigo Martins, Martin O’Keane, A. Kokaji, A. Eaves, Sharon Louis
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a highly malignant brain tumor that becomes resistance to current therapy partially due to interactions between tumor cells and tumor microenvironment (TME). To further understand these interactions and its patient prognosis association, we constructed a bulk RNA deconvolution method, the quantifying proportion of cells for GBM (QPC-GBM), to estimate cell compositions in GBM RNA samples. The QPC-GBM is a new combined methods with optimal results of each cell type from different deconvolution combinations of reference matrix and deconvolution approach with specific cell types and reference matrices from GBM single cell RNA seq datasets. By analyzing TCGA-GBM and CGGA-GBM datasets, we found the abundance of tumor and endothelial cells were positively correlated to each other, whereas their abundances were negatively correlated to GAMs and mural cells. Tumor and endothelial cells decreased while the oligodendrocytes increased in recurrent tumors. Endothelial cell abundance positively correlated to better prognosis whereas GAMs had opposite effect in newly diagnosed GBM. GAMs specifically enriched in TLR/LDL-related pathways were negatively correlated to survival and may serve as novel drug targets. This study provides a new method for studying GBM TME and links GBM TME cells to patient survival that drives new thoughts to future GBM therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0447dcd08b9510cd12e2ce81c45fe759af32d0f" target='_blank'>
              EPCO-14. QPC-GBM DECONVOLUTION REVEALS OPPOSING PROGNOSTIC ROLES OF GLIOMA-ASSOCIATED MYELOID AND ENDOTHELIAL CELLS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Chia-Hua Chen, Chiung-Yin Huang, Ian Yi-Feng Chang, Kuo-Chen Wei
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Gliomas and brain metastases are associated with poor prognosis, necessitating a deeper understanding of brain tumor biology and the development of effective therapeutic strategies. Although our group and others have demonstrated microbial presence in various tumors, recent controversies regarding cancer-type-specific intratumoral microbiota emphasize the importance of rigorous, orthogonal validation. This prospective, multi-institutional study included a total of 243 samples from 221 patients, comprising 168 glioma and brain metastases samples and 75 non-cancerous or tumor-adjacent tissues. Using stringent fluorescence in situ hybridization, immunohistochemistry and high-resolution spatial imaging, we detected intracellular bacterial 16S rRNA and lipopolysaccharides in both glioma and brain metastases samples, localized to tumor, immune and stromal cells. Custom 16S and metagenomic sequencing workflows identified taxa associated with intratumoral bacterial signals in the tumor microenvironment; however, standard culture methods did not yield readily cultivable microbiota. Spatial analyses revealed significant correlations between bacterial 16S signals and antimicrobial and immunometabolic signatures at regional, neighborhood and cellular levels. Furthermore, intratumoral 16S bacterial signals showed sequence overlap with matched oral and gut microbiota, suggesting a possible connection with distant communities. Together, these findings introduce microbial elements as a component of the brain tumor microenvironment and lay the foundation for future mechanistic and translational studies. A combination of high-resolution spatial imaging, spatial proteomics and transcriptional data reveals sparse and heterogeneous bacterial signals in gliomas and brain metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae0d1782181d2f2125dbd26294df4180d5bf9d82" target='_blank'>
              Microbial signals in primary and metastatic brain tumors
              </a>
            </td>
          <td>
            Golnaz Morad, Ashish V Damania, Brenda Melendez, Bharat Singh, Fabiana J. Veguilla, Rebecca A. Soto, Yasmine M. Hoballah, P. Sahasrabhojane, M. Wong, Mona M. Ahmed, Rene N. Rico, Kaitlyn N. Lewis, Khalida M Wani, D. Shamsutdinova, Rossana N. Lazcano Segura, D. Ingram, Eric A. Goethe, Abderrahman Day, Ivonne I Flores, Lauren K McDaniel, M. Chelvanambi, Sarah B. Johnson, Florentia Dimitriou, Pravesh Gupta, Shivangi Oberai, M. Zal, Phoebe Doss, Mohamed A Jamal, Eiko Hayase, Chetna Wathoo, Lisa M. Norberg, Stephanie L. Jenkins, Sara Nass, J. Gumin, L. Long, Jing Yang, Gina R. Bradley, M. Bekal, A. Dono, P. Pichardo-Rojas, Samuel W. Andrewes, Leomar Ballester, J. Losh, Jiyong Liang, Longfei Huo, Douglas Nielsen, Brittany C Parker Kerrigan, Priscilla K. Brastianos, N. Fowlkes, Chia-Chi Chang, Robert R. Jenq, C. Gomez-Manzano, Jason Huse, Michael A. Davies, A. Lazar, Krishna P. Bhat, Nitin Tandon, Y. Esquenazi, Christine B. Peterson, Vinay K. Puduvalli, Frederick F. Lang, Christopher D. Johnston, Susan Bullman, N. Ajami, S. Ferguson, J. Wargo
          </td>
          <td>2025-11-01</td>
          <td>Nature Medicine</td>
          <td>1</td>
          <td>49</td>
        </tr>

        <tr id="Therapeutic resistance remains a critical barrier in oncology, frequently leading to cancer relapse after initial treatment response. Growing evidence suggests the presence of drug-tolerant persisters (DTPs), a rare subpopulation of cancer cells that survives chemotherapy by entering a reversible specific adaptation. Unlike classical cell resistance, the DTP phenotype is independent of genetic changes and maintained through dynamic regulatory mechanisms. DTPs are phenotypically heterogeneous and can exhibit stem-like and quiescent cell phenotypes, non- or slow proliferation, and remarkable plasticity due to a di-pause-like state and executing epithelial–mesenchymal transition (EMT) or transdifferentiation processes. Despite advances in research, the molecular mechanisms underlying DTPs’ biology and their role in cancer relapse remain only partially understood. The review summarizes the current progress in processes that lead to the acquisition of cellular persistence status, which, in turn, constitute areas of vulnerability that can be exploited in cancer therapy. We highlight anti-DTP therapeutic strategies, including epigenetic modification, cell signaling and transcriptional regulation, metabolic reprogramming, and modification of cell interactions within the tumor microenvironment. Furthermore, we focus on the potential role of nanomaterials in the combat against DTPs. Nanoparticles not only act as part of the drug delivery process, enabling precise DTP targeting and enhancing intracellular drug accumulation, but their intrinsic properties can also be used to eradicate DTPs directly or by enhancing the effectiveness of other therapeutic strategies. The integrated approach offers strong potential to eliminate tumor persistence, prevent recurrence, and improve long-term patient outcomes beyond conventional therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495b98c4c7bc3661dae9aa58d4006e54132b7ae3" target='_blank'>
              Targeting Drug-Tolerant Persister Cancer Cells: Can Nanomaterial-Based Strategies Be Helpful for Anti-DTP Therapies?
              </a>
            </td>
          <td>
            Prachi Ghoderao, Eliza Kwiatkowska-Borowczyk, Sanjay Sahare, H. Dams-Kozlowska
          </td>
          <td>2025-11-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc219a64ec3d466c88c4dfd11649123286e2032b" target='_blank'>
              Cross-Species Insights from ART-D to Uncover Evolutionarily Conserved Oncogenic Mechanisms
              </a>
            </td>
          <td>
            Yifan Guo, Jiadong Zheng, Yixin Wu, Kun Zhao, Dian Lv, Wenhan Liu, Mengling Wang, Jin-Yu Lu, Wenyan Xu, Xianping Wang, Xianjue Ma
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive primary brain tumor, characterized by profound intratumoral heterogeneity. Recent studies have identified neural stem cells (NSCs) in the subventricular zone (SVZ) as potential cells-of-origin harboring early driver mutations. However, the stepwise transformation of these rare, mutation-bearing NSCs into heterogeneous tumor populations remains poorly understood. In this study, using a spontaneous somatic mouse model that recapitulates human gliomagenesis, as well as tumor-free SVZ tissues from GBM patients, we identified transcriptionally distinct precancerous cells harboring oncogenic mutations. These cells emerge through oligodendrocyte progenitor-like lineage specification and exhibit aberrant transcriptional programs, including dysregulated translation and extracellular matrix remodeling. Single-cell transcriptomic analysis revealed transitional cell states and progressive acquisition of gliomagenic programs, leading to the emergence of molecularly diverse malignant cell populations. Our findings demonstrate that these OPC-like precancerous cells serve as a critical intermediate in GBM evolution and a driver of intratumoral molecular heterogeneity. This work provides a single-cell atlas of early gliomagenesis and identifies a potential window for therapeutic intervention targeting precancerous cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f90544542d0fd8ecacde0e88ce21d50dda39f62" target='_blank'>
              EPCO-19. PRECANCEROUS OPC-LIKE CELLS DRIVE GLIOBLASTOMA EVOLUTION AND SHAPE INTRATUMORAL HETEROGENEITY
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, J. Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, Jeong Ho Lee
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/759c03413ef4b1106fa2a25c75713885026e6884" target='_blank'>
              Single-cell transcriptomic analysis reveals key cell types and pathogenic genes associated with metastasis in pancreatic adenocarcinoma.
              </a>
            </td>
          <td>
            Shijun Shen, Zhiqiang Li, Hong Yang, Xinwei Zhang, Gang Chen, Chengzhou Pa
          </td>
          <td>2025-11-29</td>
          <td>Clinical and experimental medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite advances in understanding systemic immune responses to Mycobacterium tuberculosis (Mtb), the localized immune dynamics within infected lymph nodes, particularly cell-type-specific transcriptional reprogramming and intercellular crosstalk, remain poorly defined, impeding targeted therapy development. By means of single-cell transcriptomics, the objective was to dissect the immune microenvironment and map intercellular crosstalk in Mtb-infected cervical lymph nodes, with the purpose of uncovering the mechanisms of localized immunity and immunopathology. Paired Mtb-positive cervical swollen lymph nodes (SLN) and adjacent Mtb-negative normal-appearing lymph nodes (NLN) from five cervical LNTB patients were analyzed using single-cell RNA sequencing (scRNA-seq). Computational network modeling (CellChat) and flow cytometry validation were employed to map immune cell heterogeneity and cell–cell communication. ScRNA-seq identified ten T cell subsets, ten B cell subsets, and six myeloid subsets, revealing conserved frequencies but substantial transcriptional reprogramming in SLN. The SLN exhibited extensive upregulation of pro-inflammatory pathways across T, B, and myeloid cells, accompanied by minimal alterations in subset frequencies. IL1B + macrophages in SLN showed an enrichment of genes associated with oxidative phosphorylation, antigen presentation, and inflammasome-related genes. The SLN demonstrated an increased cell–cell communication driven by the crosstalk between macrophage and CD8 + T/NKT cells. Validation through flow cytometry confirmed comparable proportions of immune subsets between the SLN and the NLN, which was consistent with the findings of scRNA-seq. This study delineates a spatially coordinated immune strategy in cervical LNTB. In this context, Mtb infection induces transcriptional and metabolic reprogramming rather than subset redistribution. The strengthened interaction between macrophages and T cells emphasizes that cellular immunity serves as the main impetus for bacterial containment. Nevertheless, there are trade-offs between inflammation and tissue integrity. These insights provide a framework for developing therapies that target intercellular networks to achieve a balance between immunity and pathology in LNTB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f7d9d58e6ee25e924ad6baff06b4c1c3d3fa38" target='_blank'>
              Single-cell transcriptomics of cervical lymph node tuberculosis reveals cellular heterogeneity and enhanced cellular communication
              </a>
            </td>
          <td>
            Guohui Xiao, Haijiang Wang, Su Zhang, Xuefeng Zhou, M. Ou, Xiangmei Zhang, Guobao Li, Aimei Liu, Shimin Shuai, Guoliang Zhang
          </td>
          <td>2025-10-21</td>
          <td>BMC Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) represent a small but critical subpopulation within breast tumors that drive tumor initiation, progression, metastasis, therapeutic resistance, and recurrence. BCSCs are characterized by cell surface markers such as CD44⁺/CD24⁻ and high aldehyde dehydrogenase 1 (ALDH1) activity, and are regulated by pluripotency-associated transcription factors including POU class 5 homeobox 1 (OCT4), SRY-Box transcription factor 2 (SOX2), and Nanog homeobox (NANOG). They exhibit remarkable plasticity and heterogeneity across different breast cancer subtypes, which is further enhanced by epithelial-mesenchymal transition and dynamic interactions with the tumor microenvironment, enabling immune evasion and adaptation to hostile conditions such as hypoxia. Functionally, BCSCs sustain tumorigenesis through self-renewal, contribute to metastatic dissemination as circulating tumor cells, and can enter dormancy in distant niches, leading to late recurrences. Their intrinsic resistance to chemotherapy, radiotherapy, and targeted therapies arises from quiescence, enhanced DNA damage repair, overexpression of drug efflux transporters, and metabolic reprogramming, posing major clinical challenges. Recent advances have improved BCSC identification and isolation through marker-based cell sorting, dye exclusion assays, mammosphere culture, and functional reporter systems. Conventional therapies often spare or enrich BCSCs, highlighting the need for novel treatments. Targeted strategies focusing on stemness-related signaling pathways such as Wnt, Notch, and Hedgehog, as well as regulators including NANOG, ALDH1 and programmed death-ligand 1 (PD-L1), are under investigation. Emerging approaches incorporating natural compounds, nanotechnology, and microenvironmental modulation show promise in overcoming therapeutic resistance. This review integrates current understanding of BCSC biology, heterogeneity, and microenvironmental interactions, providing valuable insights for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0824bd9fb688f1edb70aeb158a043b8788637c" target='_blank'>
              Breast cancer stem cells: Biological roles, microenvironmental interactions, and therapeutic targeting
              </a>
            </td>
          <td>
            Mou Yan
          </td>
          <td>2025-10-11</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer is a highly heterogeneous disease, with many cancers containing multiple distinct subclones. While subclones are often seen as competitors (survival of the fittest), intratumor heterogeneity can also offer direct benefits to the tumor through cooperation between different clones. This has important clinical implications, as interdependent populations may present therapeutic vulnerabilities. Here, we review existing evidence for clonal cooperativity to address key questions and outline future developments based on six overarching principles: (a) secreted factors are important mediators of clonal cooperation; (b) (very) small subclones can significantly affect tumor behavior; (c) both genetic and nongenetic heterogeneity are substrates for cooperation; (d) nonmalignant cells from the tumor microenvironment can act as cooperating partners; (e) clonal cooperation occurs throughout different stages of cancer, from premalignancy to metastasis; and (f) clonal cooperation can promote therapy resistance by protecting otherwise sensitive populations. Together, these principles suggest clonal cooperation as an important mechanism in cancer. Lastly, we discuss how novel technological developments could address remaining gaps to open up new therapeutic strategies that exploit clonal cooperativity by targeting the tumor's weakest link.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb39ce21ba07d76bffe91c932b77c8c8a7e6988d" target='_blank'>
              Strength through diversity: how cancers thrive when clones cooperate.
              </a>
            </td>
          <td>
            Marije C. Kuiken, Maartje Witsen, E. Voest, K. Dijkstra
          </td>
          <td>2025-11-09</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Immunotherapy is transforming CRC treatment, but the traditional cecal wall model lacks clinical relevance by failing to replicate human tumor location and microenvironment. To bridge this gap, we developed a novel site-specific non-surgical orthotopic model that closely mimics the clinical settings. Comparative tumor growth and immunophenotyping of these models were performed in BALB/c and C57BL/6 mice, representing microsatellite stable (MSS) and microsatellite instability (MSI) models, respectively, to reveal an immune-responsive CRC model. Flow cytometry was employed to analyze the tumor’s immune cell composition and functional state. We observed a significant decrease in CD4+ T cell infiltration in MSS tumors generated using the non-surgical procedure, accompanied by an increase in Tregs, indicative of a more immunosuppressive tumor microenvironment. Reduced PD-1 expression on CD8+ T cells in tumors from BALB/c mice further supports this finding, highlighting the potential for immune checkpoint blockade therapy in this model. Additionally, elevated KLRG1 expression on CD4+ T cells suggests impaired T cell function and reduced IL-2 production. Conversely, a less immunosuppressive phenotype in tumors generated using the non-surgical procedure compared to the cecal wall model was observed in the MSI model. These findings suggest that the non-surgical procedure is an immunologically predictive CRC model, facilitating the preclinical evaluation of potential therapies.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35c066272b7661ac2de52049f27591f533c0d238" target='_blank'>
              Tumor site-specific immune profiling and infiltration influence tumor microenvironment in colorectal cancer mouse models 4112
              </a>
            </td>
          <td>
            Samantha Sharma, Naresh Singh, Kevin Van der Jeught, George E Sandusky, Mateusz Opyrchal, Xinna Zhang, Xiongbin Lu
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) progression and recurrence are often driven by therapy-resistant cellular states emerging within hypoxic tumor niches. While hypoxia is a hallmark of GBM, the spatial transcriptomic heterogeneity that underpins resistance in these regions remains poorly characterized. Here, we employ pimonidazole labeling in conjunction with high-resolution spatial and single cell transcriptomics across 29 primary, treatment-naive human GBM samples, syngeneic mouse models, and patient-derived glioma-initiating cells exposed to hypoxia to define the hypoxia-specific tumor microenvironment and its associated cellular programs. We reveal spatially organized sub-niches within hypoxic regions, each supporting diverse cellular programs yet coexisting tumor lineage states with minimal transitions between cell states. Phylogenetic reconstruction based on inferred copy number variations demonstrates consistent evolutionary trajectories from non-hypoxic to hypoxic tumor loci. Specific copy number alterations were enriched in hypoxic regions and associated with immune infiltration and poor prognosis in MGMT-methylated tumors. Cross-species analyses confirmed the conservation of hypoxia-associated gene programs and spatial architectures. Together, our study delineates the cellular, evolutionary, and spatial complexity of hypoxia in GBM, identifies hypoxia-induced genomic alterations that may promote tumor fitness, and offers insights into new therapeutic vulnerabilities within hypoxic niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec812f9197d64f5c26991c527a5afc02eb1748" target='_blank'>
              TMIC-96. Integrated multi-omics reveals distinct hypoxic sub-niches and associated epigenomic and genomic alterations in glioblastoma
              </a>
            </td>
          <td>
            S. Mansouri, Jefferey Liu, Vikas Patil, Ruth Isserlin, Phoebe Lombard, Ronald Wu, Julio Sosa, S. Gandhi, Y. Ellenbogen, O. Singh, Hafsah Ali, C. Velásquez, Bradly Wouters, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Tumor microenvironments are complex and host diverse cell types and interactions. In-depth characterization of these environments requires detecting multiple biomarkers within the same sample, which is limited by traditional fluorescent immunohistochemistry (IHC) due to spectral constraints. Further, many high-plex IHC methods are time-consuming or require multiple imaging rounds. Here we developed a one-step multiplex labeling approach using a cocktail of primary antibody conjugates combined with spectral unmixing, enabling the simultaneous detection of 8 biomarkers and a nuclear counterstain. This streamlined approach offers high-plex biomarker detection within a timeframe similar to traditional single-antibody IHC workflows. Using the EVOS S1000 Spatial Imaging System and quantitative image analysis software, we identified multiple distinct myeloid (CD68+), lymphoid (CD20+, CD4+, FOXP3+, CD8+), and proliferating (Ki67+) cell subpopulations in normal and tumor colon tissue samples and compared their abundance and spatial distribution in relation to tissue architecture markers for smooth muscle actin and pan-cytokeratin. Additionally, we analyzed the cellular composition of a tertiary lymphoid structure in the colon tumor, believed to play a role in the immune response to colorectal cancer. These data highlight the advantages of high-plex biomarker detection for phenotyping complex tumor microenvironments using our streamlined labeling workflow and spectral unmixing approach.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d287e09a1ed64337a052860f72c1d3cae8bc8f" target='_blank'>
              Cellular phenotyping of tumor microenvironments with a high-plex fluorescent labeling and spectral unmixing approach 9333
              </a>
            </td>
          <td>
            Austin Harvey, Oggie Golub
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a formidable clinical challenge due to therapy resistance, metastasis, and relapse. Central to these processes are colorectal cancer stem cells (CCSCs), a dynamic subpopulation endowed with self-renewal capacity, plasticity, and heterogeneity. This review synthesizes recent advancements in understanding how CCSCs orchestrate tumor progression through intricate bidirectional crosstalk with the tumor immune microenvironment (TIME). We begin by elucidating the cellular origins of CCSCs, their profound intratumoral heterogeneity, and their remarkable phenotypic plasticity—driven by genetic, epigenetic, and metabolic reprogramming—which collectively serve as the root cause of therapeutic failure. A significant portion of our discussion is dedicated to deconstructing the immunosuppressive niche co-opted by CCSCs. We detail mechanisms of immune evasion and tolerance, highlighting how CCSCs modulate innate and adaptive immune cells—including NK cells, Tregs, dendritic cells, macrophages, neutrophils, and myeloid-derived suppressor cells—to foster an environment that supports stemness and suppresses cytotoxic attack. This reciprocal interaction forms a vicious cycle that perpetuates tumor survival and progression. Finally, we critically evaluate emerging therapeutic strategies that concurrently target CCSC-specific vulnerabilities and counteract immunosuppression. We explore the limitations of conventional chemotherapy and the promise of targeted therapies (e.g., Wnt inhibitors), immunotherapies (e.g., CAR-T, bispecific antibodies), and combination regimens designed to remodel the TIME and eradicate the CCSC reservoir. By integrating insights from single-cell omics and spatial biology, this review provides a comprehensive framework for overcoming therapy resistance and proposes novel precision medicine approaches for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b2fada714432c2d56f0013d3edefb668b69269" target='_blank'>
              Colorectal cancer stem cells crosstalk in tumor immune microenvironment and targeted therapeutic strategies
              </a>
            </td>
          <td>
            Yuxuan Duan, Anshu Li, Daojia Miao, Diaoyi Tan, Keshan Wang, Jian Shi
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Endometriosis (EM) is a chronic gynecological disorder that affects 5% to 10% of women of childbearing age, often causing pelvic pain and infertility. Fibrosis is a hallmark of EM progression, yet its underlying molecular drivers remain poorly understood. Emerging progress in single-cell and spatial transcriptomic technologies offer new opportunities to unravel the cellular heterogeneity and intercellular interactions driving fibrotic and immune remodeling in EM lesions. Methods We performed an integrative multi-omics analysis combining single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to dissect fibroblast heterogeneity and cell–cell communication networks in EM. ScRNA-seq data from 15 EM patients (GSE213216) were processed to identify transcriptionally distinct fibroblast subpopulations. Functional enrichment (GO, GSEA), stemness estimation (CytoTRACE), and trajectory inference were applied to reveal lineage plasticity. CellChat was used to infer intercellular signaling networks, and spatial transcriptomic data from two ectopic lesions (GSM6690475, GSM6690476) were analyzed to validate the spatial distribution of key ligand–receptor interactions. Results We identified 35 cell clusters across EM lesions, with Fibroblast and T/NK cells as dominant populations. Fibroblast were divided into five subtypes, which were associated with extracellular matrix remodeling, immune interactions, and metabolic regulation. Notably, the C2 CXCR4 + Fibroblast subpopulation exhibited high proliferative capacity and stemness characteristics, and mediated signaling pathways involved in immune and fibrotic responses through FN1. Spatial transcriptomic analysis confirmed the local enrichment of these Fibroblast in ectopic lesions, where they were associated with regions of active signaling. Conclusion This study revealed the transcriptional and spatial heterogeneity of Fibroblast in EM syndrome, and identified the C2 CXCR4 + Fibroblast subpopulation as a may represent key driver of fibrosis and immune regulation. Our integrated omics approach provided new mechanistic insights into the pathogenesis of EM and pointed out new targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ccad044a665115ac5faba149f5925920f3e3b05" target='_blank'>
              Fibroblast heterogeneity and FN1-mediated signaling in endometriosis revealed by single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Wenwen Shao, Hongmei Ju, Zhikai Xiahou, Sheng Fang, Rugen Yan, Chunyan Li, Yuan Xu, Pingping Cai
          </td>
          <td>2025-10-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Summary Whereas transcriptomic and spatial profiling offer static snapshots of tissue structure, mechanistic models use biological rules to predict how tissues evolve. We present the BioInformatics WalkThrough (BIWT) software to directly initialize spatial agent-based models from single-cell and spatial molecular data. We demonstrate how initialization strategies affect tumor–immune dynamics and spatial clustering, positioning BIWT as a software suite to generate data-driven virtual cells representing both experimental and clinical contexts. Availability and implementation The BIWT software is available at https://github.com/PhysiCell-Tools/PhysiCell-Studio. The sample dataset for running the BIWT is available at https://zenodo.org/records/16365625. The code and instructions for reproducing the use case example is available at https://github.com/drbergman/BIWT-Paper.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de85f70990a72aa732cef216c202f1c4a4e48e9" target='_blank'>
              BIWT: a bioinformatics walkthrough for embedding spatial multiomics in agent-based models for virtual cells
              </a>
            </td>
          <td>
            Daniel R. Bergman, Jeanette Johnson, Marwa Naji, Max Booth, Heber Lima da Rocha, Atul Deshpande, Dimitri N. Sidiropoulos, T. Lopez-Vidal, Randy Heiland, Luciane T. Kagohara, Robert A Anders, Lei Zheng, Elizabeth M. Jaffee, Genevieve Stein-O'Brien, Paul Macklin, E. Fertig
          </td>
          <td>2025-10-16</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="High-grade gliomas (HGGs) are aggressive brain tumors with limited treatment options and poor survival outcomes. Variants including isocitrate dehydrogenase (IDH)-wildtype, IDH-mutant, and histone 3 lysine to methionine substitution (H3K27M)-mutant subtypes demonstrate considerable tumor heterogeneity at the genetic, cellular, and microenvironmental levels. This presents a major barrier to the development of reliable models that recapitulate tumor heterogeneity, allowing for the development of effective therapies. Glioma tumor organoids (GTOs) have emerged as a promising model, offering a balance between biological relevance and practical scalability for precision medicine. In this study, we present a refined methodology for generating three-dimensional, multiregional, patient-derived GTOs across a spectrum of glioma subtypes (including primary and recurrent tumors) while preserving the transcriptomic and phenotypic heterogeneity of their source tumors. We demonstrate the feasibility of a high-throughput drug-screening platform to nominate multi-drug regimens, finding marked variability in drug response, not only between patients and tumor types, but also across regions within the tumor. These findings underscore the critical impact of spatial heterogeneity on therapeutic sensitivity and suggest that multiregional sampling is critical for adequate glioma model development and drug discovery. Finally, regional differential drug responses suggest that multi-agent drug therapy may provide better comprehensive oncologic control and highlight the potential of multiregional GTOs as a clinically actionable tool for personalized treatment strategies in HGG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba311ed25ba183347b470e69990beb4c5d96db4" target='_blank'>
              Precision Oncology for High-Grade Gliomas: A Tumor Organoid Model for Adjuvant Treatment Selection
              </a>
            </td>
          <td>
            Arushi Tripathy, Sunjong Ji, H. Serhan, Reka Chakravarthy Raghunathan, S. Syed, Visweswaran Ravijumar, Sunita Shankar, Dah-Luen Huang, Yazen Alomary, Yacoub Haydin, Tiffany Adam, Kelsey Wink, N. Clarke, C. Koschmann, Nathan M. Merrill, Toshiro Hara, S. Merajver, W. Al-Holou
          </td>
          <td>2025-10-01</td>
          <td>Bioengineering</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bb8a507427a3e0cfe9ea50f66dabfc18ac940f2" target='_blank'>
              Cancer Metastasis: Therapeutic Challenges and Opportunities
              </a>
            </td>
          <td>
            Sri Sathya Sandilya Garemilla, Manisha Choudhary Kadambala, S. C. Gampa, Swetha Chinthala, S. Garimella
          </td>
          <td>2025-10-16</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f06cd8f942853075fbd7169c14dc2622fac19fcf" target='_blank'>
              Comprehensive Bulk and Single-Cell RNA Sequencing Uncovers Senescence-Associated Biomarkers in Therapeutic Mesenchymal Stem Cells
              </a>
            </td>
          <td>
            Emese Pekker, Erda Qorri, Márton Zs. Enyedi, Valéria Szukacsov, Ferhan Ayaydin, Éva Szabó-Kriston, Bernadett Csányi, M. Mórocz, Farkas Sükösd, Endre Kiss- Tóth, Lajos Haracska
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a185da472d5e884ad916a2af2cce445855a3e882" target='_blank'>
              Single-cell lineage trajectory defines CDK inhibitor-sensitive cells-of-origin in esophageal squamous cell cancer
              </a>
            </td>
          <td>
            Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) have emerged as critical prognostic and immunotherapeutic indicators in cancer, with their clinical significance modulated by spatial distribution patterns and density. Here, we performed integrated single-cell and spatial transcriptomic profiling of 30 gastric cancer (GC) specimens stratified by TLS spatial localization patterns. Comparative analysis shows pronounced enrichment of CXCL13+ T lymphocytes (TLCs), CXCR5+ germinal center B lymphocytes (gc_B cells), LAMP3+CD80+ activated dendritic cells (DCs), and SELP+ACKR1+ high endothelial venule (HEV) cells within intratumoral-TLS (iTLS) rich tumors compared to peritumoral-TLS (pTLS) and desert-TLS (dTLS) tumor subtypes. Multimodal cell–cell interaction analysis and functional experiments demonstrate that HEV expressed VCAM1 and ICAM1 recruits and activates CXCL13+ TLC through the CXCL13-ACKR1 pathway, which promotes TLS formation via CXCL13-CXCR5-dependent crosstalk with B lymphocytes. We further develop a single-cell/spatial TLS signature that captures the cellular ecosystem of iTLS-containing tumor, demonstrating predictive value for immunotherapy outcomes in GC patients. Tertiary lymphoid structures (TLS) within the tumour microenvironment are important for prognostic and immunotherapeutic indications in cancer. Here the authors use a spatial and single cell transcriptomics analysis of gastric cancer (GC) samples and show transcriptomic differences grouped by types of TLS and generate a prognostic transcriptomic signature to help predict outcome in GC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f70b23aaaa2341fd42d68d4bef2be6eb1c78ea" target='_blank'>
              Single-cell and spatial transcriptomics implicate a prognostic function of tertiary lymphoid structures in gastric cancer
              </a>
            </td>
          <td>
            Yanchun Wang, Guangyu Zhang, Xin Zhang, Gaoke Liu, Lihua Zhang, Lei Chen, Shuli Sang, Sheng Yao, Yang Fei, Zhichao Tian, Yu Zang, Hongwei Wang, Yiping Dong, Chunjie Liu, Xizhao Chen, Yan Liu, Min Zhang
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Neutrophils are integral components of the bone marrow and stromal cell network. They express the immune checkpoint molecule PD-L1 and can induce T cell exhaustion, thereby promoting immunosuppression. In this study, we investigated whether tumor-derived Cldn7 deficiency could recruit polymorphonuclear neutrophils (PMNs), induce their metabolic reprogramming, and consequently drive their transition toward a pro-tumor phenotype, leading to the establishment of an immunosuppressive tumor microenvironment (TME). Using single-cell RNA sequencing, clinical sample validation, and both in vivo and in vitro experiments, we found that Cldn7 deficiency in colorectal cancer (CRC) results in a tumor microenvironment characterized by significantly increased infiltration of CD11b⁺ Ly6G⁺ neutrophils. This is accompanied by neutrophil metabolic reprogramming that facilitates their phenotypic shift toward a tumor-promoting state, which in turn suppresses the cytotoxic function of CD8⁺ T cells and contributes to the formation of an immunosuppressive TME, thereby accelerating CRC progression. Mechanistically, Cldn7 deficiency indirectly activates the NF-κB signaling pathway, leading to elevated secretion of chemokines such as CXCL1 that are responsible for PMN recruitment. Inhibition of the NF-κB/CXCL1 axis reduces PMN infiltration, decreases PD-L1 expression on neutrophils, suppresses neutrophil glycolysis and histone lactylation, reverses the exhausted phenotype of CD8⁺ T cells, thereby mitigating the immunosuppressive microenvironment. Furthermore, overexpression of Cldn7 enhances the efficacy of immune checkpoint blockade (ICB) therapy.Collectively, our findings indicate that Cldn7 deficiency not only contributes to immune evasion and malignant progression in CRC but also plays a critical role in immune modulation. Targeting PMN metabolic reprogramming and immunosuppressive function associated with Cldn7 loss may offer a promising strategy to improve the therapeutic efficacy of immunotherapy in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57fa7d965f4e46d2deb56381980d3616631c7424" target='_blank'>
              Claudin-7 deficiency induces metabolic reprogramming of neutrophils in the colorectal cancer microenvironment
              </a>
            </td>
          <td>
            Xiaoqing Liang, Dajin Yuan, Songyun Zhao, Jianbo Zhou, Kun Wang, Xiaoli Liu, Yin Liu, Huimin Li, Mengdi Hao, Wenbin Huang, Wenjie Li, Lei Ding
          </td>
          <td>2025-10-16</td>
          <td>Cell Death & Disease</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) and diffuse midline glioma (DMG) are two of the most aggressive brain tumors, marked by significant intratumoral heterogeneity and transcriptional plasticity, which drive resistance to therapy. These tumors share similar transcriptional programs, though the specific genes associated with these programs differ. Transitions between transcriptional states, particularly to the mesenchymal state, which is associated with greater resistance to treatment, complicate treatment strategies. A significant challenge in glioma research is the inability to dynamically track these state transitions in vitro and in vivo. To address this challenge, we introduce CRISPR-TAG reverse genetics, a novel approach that enables real-time monitoring of key transcriptional state markers in glioblastoma and diffuse midline glioma cell lines. Using CRISPR-TAG, we insert a fluorescent reporter sequence at the 3’ end of target genes, linking the fluorescent signal to endogenous gene expression. This allows us to dynamically track changes in gene expression as cells transition between states. We have successfully tagged 30 key cell state markers in patient-derived GBM and DMG cell lines, providing a robust model to investigate the upstream regulators of cell state transitions. By integrating CRISPR-TAG with global CRISPR screens and high-content pharmacological screens, we uncover genetic dependencies that underlie glioma plasticity. For example, we identify modulators of the mesenchymal cell state, including ANXA1 expression, which serves as a critical marker of this resistant state. Additionally, we demonstrate how our method allows in vivo tracking and modulation of cell state transitions in explant cultures, with AI-assisted analysis providing detailed insights into these transitions. In conclusion, CRISPR-TAG reverse genetics lays a critical foundation for building a high-resolution map of glioma state regulators, paving the way for targeted therapeutic strategies. By enabling the precise modulation of cell states, this method offers the potential to complement and enhance existing treatment modalities for GBM and DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eaf38521e14ac09d468437f1dbddc6b4207f994" target='_blank'>
              CNSC-29. RESOLVING THE REGULATION OF GLIOBLASTOMA CELL STATES BY CRISPR-TAG REVERSE GENETICS
              </a>
            </td>
          <td>
            Josephine Heinold, Madeleine Skeppås, Filippa Kruse, Clara Nordquist, Irem Uppman, Sven Nelander
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In this study, we analyzed publicly available scRNA-seq data from primary tumor tissues of 10 Retinoblastoma (RB) patients to explore tumor microenvironment (TME) heterogeneity. Our findings revealed distinct subpopulations of cone precursor (CP) cells, with higher proportions in invasive RB. Differential gene expression and pathway analysis highlighted functional diversity among CP subpopulations, with CP4 showing elevated TGF-β signaling in invasive RB. Cell–cell interaction analysis further identified rewiring of communication networks, with increased fibroblast–CP interactions in invasive tumors. Bulk RNA-seq identified two molecular subtypes, with subtype 1 showing an immunosuppressive TME. Finally, DOK7 was identified as a key gene associated with invasion, with functional assays confirming its role in promoting tumor progression. These results provide valuable insights into RB progression and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0641facdea378a09ac72da42fc0862a4720147" target='_blank'>
              Comprehensive analysis of single-cell and bulk RNA sequencing uncover tumor microenvironment diversity in invasive Retinoblastoma
              </a>
            </td>
          <td>
            Wei Wan, Xiaoqian Chen, Hongli Liu, Ling Yang, Pingping Huang
          </td>
          <td>2025-11-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Bladder cancer is a significant malignancy, for which prognostic prediction and understanding of the tumor immune microenvironment are crucial. B cells play a key regulatory role in this environment, making their study essential for advancing bladder cancer research. Methods In this study, a multi-omics analysis strategy combining single-cell RNA sequencing (scRNA-seq) and bulk RNA-seq was used to establish single-cell transcriptome profiles of tumor tissues from bladder cancer patients, focusing on B-cell populations and their interactions with other cell types in the tumor microenvironment. Large public databases were used to screen for key prognostic genes associated with bladder cancer B cells, and their biomarker expression was verified by in vitro experiments. Results Based on tumor samples from eight patients with bladder cancer and four normal samples, we selected 84, 967 cells for single-cell sequencing analysis. From these, we identified 10, 967 B cells and identified 508 key genes associated with B cells in bladder cancer from five different B cell subtypes. By integrating a large amount of RNA sequencing data, we identified VCL, FLNA, TAGLN, ACTA2, COL6A2, and CALD1 as potential biomarkers for B-cell-associated bladder cancer, and experimentally verified that these markers were significantly lower in bladder cancer patients than in the normal group, and were effective in predicting the survival rate of the patients and the status of the tumor immune microenvironment. Conclusions Using a combination of transcriptomic and experimental validation at single-cell and batch levels, this study provides insights into the key gene signatures of B cells from patients with bladder cancer and their roles in regulating the tumor immune microenvironment, providing new biomarkers and potential therapeutic targets for predicting patient’ prognosis and immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c8bfbe519c06579f4740a61199654ca1af067bc" target='_blank'>
              Comprehensive analysis of single-cell and bulk transcriptomes reveals key B-cell genes and immune microenvironment regulation in bladder cancer
              </a>
            </td>
          <td>
            Lijun Wang, Juan Yang, Zhangxiao Xu, Bo Tao, Yunpeng He, Yuan Zhao, Jian Wu, Yira Ma, Zitao Zhong, Lin Ye
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Malignant peripheral nerve sheath tumors (MPNST) are aggressive sarcomas with limited therapeutic options. Here, we present a novel sporadic murine model of Nf1 wild-type MPNST, driven by conditional expression of oncogenic BrafV600E and Pten loss in the glial lineage using the Plp1::CreERT2 driver. This model allows for highly penetrant and rapid tumor induction through spontaneous formation, localized initiation, or cell transplantation. Comparative analysis with Tyr::CreERT2-driven melanoma revealed striking phenotypic divergence despite shared genetic alterations, underscoring the importance of the cell of origin in shaping tumor identity. In this system, MPNST cells show refractory capacities to induce melanocytic trans-differentiation upon melanoma-promoting signaling cues such as canonical Wnt signaling gain of function (GOF) or increased of levels of the epigenetic mark H3K27Me3 upon Ezh2 GOF. Our findings emphasize the significance of lineage context in tumor initiation and provide a foundation for future mechanistic and therapeutic studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1b40c771fae54171400436daa93f306b770ba99" target='_blank'>
              A novel murine model for sporadic, malignant peripheral nerve sheath tumors, driven by BrafV600E and Pten loss.
              </a>
            </td>
          <td>
            Julien Debbache, Myriam Gwerder, Elisabeth Rushing, Lukas Sommer
          </td>
          <td>2025-10-09</td>
          <td>Disease models & mechanisms</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 The tumor immune microenvironment (TiME) accommodates significant transcriptomic and proteomic heterogeneity, and understanding this heterogeneity is key to the development of cancer treatments. Most analytical approaches provide only one type of biological readout and/or lack spatial information needed to decipher TiME complexity. We developed a new spatial multi-omics toolset to comprehensively study the TiME transcriptomic and proteomic readouts with single cell resolution. We used the CellScape™ Precise Spatial Proteomics platform with enhanced photobleaching in cyclic immunofluorescence (EpicIF) technology to detect 12+ RNA targets via HCR™ Gold RNA-FISH and 24+ protein markers via multiplex immunofluorescence (mIF) on the same formalin-fixed paraffin-embedded (FFPE) tissue sample. RNA-FISH target probes were added to the tissue sample after antigen retrieval, and tissues were then mounted in CellScape fluidic chambers to enable on-instrument staining, imaging, and signal removal. The RNA-FISH assay was followed by a 24-plex mIF assay containing antibodies against common immune-oncology biomarkers. Multiomic data from human tonsil, colon and breast tissues showed high conformity between RNA-FISH and mIF assays on the CellScape platform. The combined assay revealed differential biomarker expression profiles associated with immune-regulatory processes in the tumor samples and provided detailed insight into immune and inflammatory processes of the TiME.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5daaa251cd6bb773ad5618931f13decbb4ef34a" target='_blank'>
              Spatial multi-omic analysis of the tumor microenvironment using high-plex HCR RNA and proteomic immune profiling on the CellScape platform 4057
              </a>
            </td>
          <td>
            Matthew H. Ingalls, J. Boog, Daniel Jimenez Sanchez, Adam Maddox, Aneesh Acharya, Cameron Earl, Harry Choi, A. Christians, O. Braubach
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 The most aggressive type of skin cancer, melanoma, has a high prevalence of metastases and a poor prognosis. Despite advancements in treatment, drug resistance and tumor microenvironment heterogeneity, especially involving neuro-immune interactions, continue to exist. The goal of this study is to uncover the cellular heterogeneity of melanoma in order to pinpoint molecular targets and tumor-promoting subtypes.



 Melanoma single-cell RNA sequencing data came from GEO. Twelve cell types were discovered after Harmony batch effect adjustment and Seurat quality control; melanoma cells were subtyped. Functional expression of differential genes was examined using gene ontology and gene set enrichment. Cytotrace measured subtype differentiation potency. PySCENIC revealed transcription factor regulatory networks, and CellChat predicted intercellular communication between malignant cell subtypes and other cell types. Functional experiments with A375 and MEWo cell lines—lentiviral shRNA knockdown, CCK-8 proliferation, wound healing, transwell migration, and flow cytometry apoptotic assays—confirmed the analysis.




 Single-cell RNA sequencing was utilized to analyze melanoma cell subtypes and their interactions in the tumor microenvironment. C2
 IGFBP3
 +, which had the lowest CytoTRACE2 score and was enriched in late tumor stages, affected melanoma development. This subtype expressed prominent immunomodulatory pathways. The C2 subtype mediate signaling to fibroblasts/pericytes via the PROS pathway and myeloid/plasmacytoid dendritic cells via the MHC-II pathway. The C2 subtype was strongly linked with FOSL1 expression, which may affect gene transcription and illness progression. FOSL1 knockdown significantly increased apoptosis and decreased melanoma cell motility and proliferation
 in vitro
 .





 We identified immunoregulatory C2
 IGFBP3
 + melanoma cell subtypes in our investigation, and FOSL1 was a crucial transcription factor that aided in cell migration, proliferation, and survival. Because the C2 subtype involves the MHC-II and PROS signaling pathways, which have been shown to have roles in neuroimmunology, neuroinflammation, and pain regulation, it may serve as a hub for neuro-immune interactions in the tumor microenvironment. Precision treatments for melanoma may be advanced by focusing on the FOSL1 or C2 subtype pathways, which may assist in overcoming immunotherapy resistance.

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34e116063cb6903b41d6e715ba8448ec0e1c090" target='_blank'>
              Single-cell transcriptomics identifies FOSL1-regulated IGFBP3+ melanoma subtype as a neuro-immunoregulatory signaling hub facilitating tumor progression
              </a>
            </td>
          <td>
            Wenjia Ge, Ziwei Zhang, Wenjie Chen, Zhijie Zhao, Huabao Cai, Yantao Ding, Jin Xu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The c-MYC transcription factor is aberrantly expressed in most human cancers to enhance expression of proliferative gene programs. Owing to its pseudo-ordered structure and reliance on extensive and dynamic protein-protein interactions in distinct transcriptional regulatory complexes, defining context-specific MYC interactors has remained challenging. Therefore, mapping MYC-centered complex topologies in disease relevant models could identify components critical for its function which may serve as therapeutic targets in MYC-driven cancers. Here, we present a matched pair of photoproximity probes coupled with quantitative proteomics which enable context-dependent mapping of protein complex topology inside cells. We applied this spatially resolved, intracellular photoproximity (siPROX) profiling workflow to map MYC interactomes across temporal, spatial and disease-relevant contexts. Basal and inhibitor-treated profiles confirmed interactions with a wide range of known chromatin-associated transcriptional regulatory factors that define the extended MYC transcriptional bubble in live cells. Time-resolved mapping of inhibitor treated cells identified dynamic remodeling of numerous transcriptional regulatory factors and identified several BAF complex members (e.g., PBRM1 and SMARCC1)1 that persist in the presence of bromodomain inhibition. Furthermore, spatial MYC topology maps in small cell lung cancer cells confirmed the presence of BAF complex members under conditions where MYC induced target gene expression, altered cell morphology and enhanced proliferation. Lastly, loss of BAF function via inhibition of SMARCA2/4 ATPase activity resulted in rapid loss of chromatin-bound and nuclear MYC levels, downregulation of MYC-dependent transcripts and MYC-specific cell growth in several cancer cell models. Together, these data highlight the potential for siPROX to identify spatially resolved, dynamic TF interactors and highlight MYC-proximal BAF interactions as a targetable liability to regulate MYC-dependent transcription and proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f78a06fd992b15a6520a4d4a8e38c9a04a5e3b92" target='_blank'>
              Spatially-resolved Photoproximity Profiling of MYC Identifies a MYC-BAF Liability in Cancer Cells
              </a>
            </td>
          <td>
            Anthony J. Carlos, Shuyuan Huang, Dongbo Yang, Colin Swenson, Pratyasha Chakraborty, Shaopeng Yu, Benjamin D. Stein, Raymond E. Moellering
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a crucial role in colorectal cancer by sustaining intratumoral heterogeneity, therapeutic resistance, and metastatic potential. CD133 (PROM1) is among the most frequently used surface markers for CSC identification, whereas TRIM28, a versatile epigenetic regulator, has been implicated in controlling CD133 expression and stem-like features. In this study, we performed a detailed molecular and functional analysis of Caco2 colorectal cancer cell clones with individual knockouts of CD133 or TRIM28. Elimination of CD133 neither altered global gene expression, as confirmed by transcriptome profiling, nor affected key cellular properties. In contrast, loss of TRIM28 led to a marked reduction in CD133 protein abundance and induced extensive molecular and phenotypic remodeling. TRIM28 knockout was associated with broad transcriptomic changes involving more than 500 differentially expressed genes, decreased proliferative activity monitored by time-lapse imaging, and reduced sensitivity to paclitaxel, cisplatin, and curcumin. Furthermore, immune evasion molecules CD24 and CD47 (“don’t eat me” signals) were strongly upregulated in TRIM28-deficient cells, consistently confirmed by both RNA-Seq and flow cytometry analyses. At the same time, imaging flow cytometry and mitochondrial activity assays indicated that these effects were not due to major shifts in mitotic index or bioenergetic status. Altogether, our results demonstrate that TRIM28, rather than CD133, functions as a central regulator of CSC-associated phenotypes in colorectal cancer. These findings highlight the importance of epigenetic context in CSC biology and may inform the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3d3eb05c9db6b57d94b54e8bdad26455b3ea6" target='_blank'>
              Contrasting Impacts of Targeted Disruption of the Cancer Stem Cell Marker CD133 and Its Epigenetic Regulator TRIM28 in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            I. Kholodenko, A. Saidova, D. Potashnikova, V. A. Arzumanian, D. Romashin, A. Tvorogova, E. Poverennaya, K. Yarygin, Yan S. Kim
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Diffuse midline glioma (DMG) is an incurable pediatric brain tumor, with radiotherapy offering only transient benefit before inevitable recurrence. While recent studies have revealed the cellular complexity of DMG, the relationship between intratumoral heterogeneity and disease progression — particularly under therapeutic pressure — remains poorly understood. In this study, we integrate longitudinal single-cell- and spatial transcriptomics with single-cell chromatin accessibility profiling and radiomics to investigate how distinct DMG cell states and microenvironmental components evolve in response to treatment, contributing to resistance and the emergence of therapy-persistent niches. We applied this multi-omic approach to a unique cohort of 10 matched diagnostic and autopsy DMG samples obtained from children enrolled in the PNOC023 clinical trial. Spatial transcriptomic profiling revealed that post-treatment autopsy tissues showed consistent enrichment of oligodendrocyte progenitor cell-like malignant populations, suggesting that this stem-like subpopulation is selectively retained or expanded following radiotherapy in DMG. Moreover, integration of single-cell RNA and ATAC sequencing with cerebrospinal fluid microRNA analyses further revealed molecular signatures of resistance and uncovered minimally invasive biomarkers of treatment response. To complement our molecular analyses, we applied radiomic profiling of longitudinal MRI scans to non-invasively monitor changes in tumor heterogeneity over the course of treatment in this patient cohort. In parallel, we developed a data-driven mathematical model using patient-derived DMG xenografts to simulate tumor cell responses to radiotherapy and generate quantitative predictions of preclinical treatment outcomes. We employed this model to explore alternative radiation fractionation schemes and identify optimal dosing strategies that most effectively deplete therapy-resistant tumor cell populations in DMG. Together, these integrated spatial and molecular datasets provide a high-resolution framework for understanding how radiotherapy remodels the DMG tumor ecosystem. By identifying persistent cellular populations and spatial niches that survive treatment, this work lays the groundwork for future precision therapies aimed at targeting the cellular reservoirs that drive DMG recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4395d6f286b7b33609966b9d19ccc4d41894fd90" target='_blank'>
              RBIO-16. Mapping the Spatiotemporal Evolution of Diffuse Midline Glioma Under Radiotherapy
              </a>
            </td>
          <td>
            Costanza Lo Cascio, Shashank Katiyar, Simone Bruno, Carlos A O De Biagi Junior, Danielle Bitterman, Joana G. Marques, Sina Neyazi, Sharmistha Pal, David Kozono, Benjamin H. Kann, Charles Stiles, Sabine Mueller, Franziska Michor, D. Haas-Kogan, M. Filbin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Extensive studies have demonstrated the relationship between metabolic reprogramming and the tumor microenvironment. Here, we characterized the head and neck squamous cell carcinoma (HNSCC) evolutionary landscape using spatial metabolomics/transcriptomics, single-cell transcriptomics, and bulk multi-omics. Metabolic heterogeneity during HNSCC malignant transformation was identified, with significant enrichment in the purine metabolism. Integrating single-cell and bulk data, we developed a robust ligand-receptor-based signature (LRS) linked to NT5E, a key upstream regulator of purine metabolism, which served as an independent prognostic indicator. The low LRS subtype was associated with a high proportion of immune cell infiltration and improved response to immunotherapy. Notably, in vitro and in vivo experiments demonstrated that AMIGO2, a core molecule within the LRS, regulates tumor-associated purine metabolism, and that its downregulation suppresses tumor cell invasion and migration, inhibits myofibroblast differentiation, and promotes immune effector cell infiltration. Moreover, combining AMIGO2 targeting with anti-PD-1 therapy yielded superior efficacy. Consistent validation was also obtained in a clinical cohort of HNSCC and premalignancy patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca668a38f0d67d558807d21d034edd4de991a2" target='_blank'>
              Metabolomic and transcriptomic profiling of HNSCC identifies AMIGO2 as a therapeutic target modulating tumor microenvironment
              </a>
            </td>
          <td>
            Gan Liu, Xinfeng Yao, Yuchen Hou, Wentao Deng, Liyu Zhang, Shutong Li, Panpan Huang, Wenjiao Chang, Haiyan Huang, Lan Shi, Qianqian Zhou, Yanqing Wang, Jia Chen, Wenchao Gu, Xiaoling Ma, Yu Zhou, Honghong Liu, Shanshan Zhang, Zongcheng Yang
          </td>
          <td>2025-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a highly malignant pediatric brain tumor of the cerebellum characterized by extensive molecular, biological, and clinical heterogeneity. Current genetically engineered mouse models have only partially replicated this diversity, and harbor species-specific limitations owing to fundamental differences between mouse and human development. Therapeutic strategies that accommodate MB heterogeneity are required to improve patient outcomes but necessitate biologically faithful models that are predominantly lacking. To this end, we have established human pluripotent stem cell-derived cerebellar organoids that recapitulate early human cerebellar development and enable deep mechanistic studies into MB oncogenesis. Transcriptomic profiling and immunophenotyping of our organoids throughout developmental timepoints revealed cellular compositions and differentiation trajectories reminiscent of cerebellar development in vivo. As our cerebellar organoids faithfully reproduce putative lineages of origin for distinct MB subgroups, we’ve endogenously Cre-tagged stem cell lines and generated various expression constructs to manipulate known driver genes in specific cellular compartments to generate accurate human cerebellar organoid-based disease models. We furthered the utility of this ex vivo human system by establishing a cerebellar organoid/patient-derived xenograft (PDX) co-culture platform that fosters the sustainability of MB PDXs in culture. Upon co-culturing with cerebellar organoids, we observed sustained viability and proliferation of MB PDXs, both of which are unprecedented in standard in vitro cultures. Further, we observed differential electrophysiological properties in PDX-harboring organoids compared to standalone organoids and identified putative signaling pathways crucial for cellular crosstalk between tumor and their supporting microenvironment. Our ongoing multi-faceted MB modeling strategies leveraging cerebellar organoids engender new experimental opportunities otherwise incompatible with pre-existing approaches, providing an efficient platform for advancing disease biology and identifying therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9c6a6fee26a49ce1740bbf42bd5633cf13397a" target='_blank'>
              TMOD-13. Human pluripotent stem cell-derived cerebellar organoids as a robust platform for studying medulloblastoma pathogenesis
              </a>
            </td>
          <td>
            Stephanie C Wu, Rachel Xu, Yiran Li, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="


 Glioblastoma multiforme (GBM) is an aggressive malignancy characterized by rapid progression, heterogeneity, and resistance to conventional therapies. Traditional in vitro and animal models often fail to replicate the complex tumor microenvironment and cellular diversity of human GBM, limiting their utility in effective drug testing and translation to clinical application. To address these limitations, we developed a 3D Ex Vivo Patient Tissue (EVPT) assay for GBM which uses patient-derived tumor tissue. This approach maintains the tumor heterogeneity and cytoarchitecture of the tumor, enabling more clinically relevant assessment of therapeutic candidates. In this study, the EVPT model was used to test novel tumor-specific therapies, highlighting potential breakthroughs in the treatment of GBM.



 10 patient tumor tissues from GBM patients were procured from commercial tissue providers and processed within 24 hours of dissection to preserve the TME. Fresh, minimally processed tumor material was embedded in a protein-rich hydrogel and exposed to novel peptide-drug conjugates (PDC): a peptide carrier and a cytotoxic payload from Oncopeptides at various doses in 384-well format for 5-7 days. As controls, we included Staurosporine (maximum killing), Temozolomide (one of the few options and standard of care for GBM), as well as investigational drugs currently in clinical trials: QBS72S (Alkylating Agent) and Panobinostat (Histone Deacetylase (HDAC) Inhibitor). Phenotypic screening was conducted using our proprietary automated high-content imaging (HCI) platform. Additionally, sample characterization was performed on fixed tissue samples by the IHC, validating GBM tumor identity, proliferation and TME presence.



 All patient tissues responded strongly to Staurosporine, while exhibiting resistance to Temozolomide. QBS72S and Panobinostat showed sample-dependent killing activity but were outperformed by the PDC compounds, demonstrating over a five-fold increase in effectiveness. PDC compounds elicited strong to mild killing responses, reflecting the heterogeneity in the samples' reactions (>10-fold difference). Notably, the lead PDC compound demonstrated strong cytotoxic activity, effectively killing tumor cells at highest dose in 9 out of 10 samples.



 Our results demonstrate the novel PDCs' great potential over standard of care Temozolomide, highlighting its promise as a more effective therapeutic strategy for GBM. The EVPT platform effectively aided in lead compound selection, with a clear assay window between less and more potent compounds. Moreover, drug response heterogeneity was accurately captured between patients. This represents a major advancement in preclinical drug testing and glioblastoma therapy development, enhancing the likelihood of successful translation to clinical outcomes. Ultimately, this approach promises to improve treatment strategies and patient outcomes in the long term.



 Ezgi Kaya Aksoy, Ulrica Westermark, Kristin Hammer, Luca Gandini, Jolie Flach, Emma Spanjaard, Stefan Svensson. Gelius, Marrit Putker, Nataliia Beztsinna. A preclinical ex vivo model for glioblastoma captures patient heterogeneity in drug response [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B092.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5f6e0109b0e3b7c28458848e59c50edea2eb7e" target='_blank'>
              Abstract B092: A preclinical ex vivo model for glioblastoma captures patient heterogeneity in drug response
              </a>
            </td>
          <td>
            E. K. Aksoy, Ulrica Westermark, Kristin Hammer, L. Gandini, J. Flach, E. Spanjaard, S. Gelius, Marrit Putker, Nataliia Beztsinna
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary Traditional two-dimensional (2D) cell cultures are widely used in cancer research but fail to replicate the complex interactions that occur in tumors in vivo. Three-dimensional (3D) models more closely mimic the tumor microenvironment by preserving dynamic cell–matrix communication. In this study, we developed and characterized 3D spheroids from two breast cancer cell lines with distinct metastatic potential (MCF-7 and MDA-MB-231). Significant differences in cell morphology, epithelial-to-mesenchymal transition (EMT) markers expression, and functional behavior compared to 2D cultures were observed. The spheroids also showed the distinct expression of key receptors (ERs, EGFR, IGF1R) and matrix molecules (syndecans and matrix metalloproteinases). Bioinformatic analysis further revealed the clinical relevance of these matrix regulators in breast cancer prognosis. Overall, these findings introduce an informative, matrix-free cell culture platform for studying breast cancer progression and potentially exploring new therapeutic approaches. Abstract Background/Objectives: Conventional two-dimensional (2D) cell cultures offer valuable insights into cancer cell biology; however, they lack in replicating the complex interactions present in solid tumors. Therefore, research has shifted towards the development of three-dimensional (3D) cell models that recapitulate the dynamic cell–cell and cell–matrix interactions within the complex tumor microenvironment (TME), better resembling tumor growth and initial stages of dissemination. Extracellular matrix, a key component within the TME, regulates cell morphology and signaling, influencing key functional properties. Breast cancer remains the most frequently diagnosed cancer type in women and a leading cause of cancer-related mortality. Methods: The aim of the present study was the development of breast cancer cell-derived spheroids, utilizing two breast cancer cell lines with differential estrogen receptor (ER) expression profile, and their characterization in terms of morphology, functional properties, and expression of epithelial-to-mesenchymal transition (EMT) markers and matrix signatures implicated in breast cancer progression. To this end, the ERα-positive MCF-7, and ERβ-positive MDA-MB-231 breast cancer cell lines were utilized. Results: Our findings revealed notable phenotypic transitions between 2D and 3D cultures, which were further supported by differential EMT markers expression. Moreover, spheroids exhibited distinct expression profiles of key receptors [ERs, epidermal growth factor receptor (EGFR) and insulin-like growth factor receptor (IGF1R)] and matrix molecules (syndecans, and matrix metalloproteinases), accompanied by altered functional cell properties. Bioinformatic tools further emphasized the interplay between the studied matrix regulators and their prognostic relevance in breast cancer. Conclusions: Overall, this study introduces a simple yet informative 3D breast cancer model that captures key TME features to better predict cell behavior in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454047e64cbe106ca9e24ee78bd584b4344b2361" target='_blank'>
              Spheroid-Based 3D Models to Decode Cell Function and Matrix Effectors in Breast Cancer
              </a>
            </td>
          <td>
            Sylvia Mangani, Christos Koutsakis, Nikolaos E. Koletsis, Z. Piperigkou, Marco Franchi, Martin Götte, Nikos K. Karamanos
          </td>
          <td>2025-10-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Breast Cancer is one of the most prevalent malignancies and the cause of mortality among women. Despite technological advancements, diseases have inherent heterogeneity, especially in the tumor microenvironment (TME), which poses a significant challenge for effective therapy. The study explores the spatial heterogeneity of two key molecular markers, EPCAM (Epithelial Cell Adhesion Molecule) and ERBB2 (HER2).  The research was conducted based on spatial transcriptomics and a graph-based clustering method. EPCAM plays a crucial role in maintaining epithelial integrity, and its dysregulation facilitates tumor invasion and metastasis. On the other hand, ERBB2 is a well-characterized oncogene that drives proliferation in HER2-positive breast cancer. The study leveraged 10x Genomics Visium data and a high-resolution histopathology image to analyze gene expression and spatially variable genes (SVGs) within the tissue. The study identified eight functional spatial domains. ERBB2 expression was localized to tumor cores, which reflects the role as an oncogenic driver. On the other hand, EPCAM exhibited a wider distribution, highlighting the involvement in tumor-stroma interactions and epithelial-mesenchymal transition (EMT). The findings demonstrate the utility of spatial transcriptomics in two aspects. The first is resolving tumor heterogeneity, and the second is to underscore the importance of spatial context in understanding the roles of EPCAM and ERBB2 for cancer progression. The insights contribute to the development of precision medicine approaches that target a specific tumor region for improving the therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b117f399cc68a98eef4707a09f6308f49ac08645" target='_blank'>
              Spatial Heterogeneity of EPCAM and ERBB2 Expression in Breast Cancer: Insights from Spatial Transcriptomics
              </a>
            </td>
          <td>
            Muhammad Afrizal Amrustian, Bita Parga Zen, Aminatus Sa'adah
          </td>
          <td>2025-10-27</td>
          <td>International Journal on Advanced Science, Engineering and Information Technology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most common and deadliest primary brain tumor, with no effective therapies and a longstanding record of failed phase 3 clinical trials—often without mechanistic insight into why promising approaches fall short. To address this critical gap, we implemented a novel on-therapy, longitudinal, and multi-regional tissue sampling strategy in patients treated with rQNestin34.5v2, an engineered oncolytic biologic based on herpes simplex virus 1, to explore how tumor and immune ecosystems evolve during treatment. Using high-resolution spatial transcriptomics (Xenium) and targeted spatial proteomics (CyCIF), we profiled over 80 spatially and temporally distinct samples from six patients, capturing over half a million immune, stromal, and malignant cells. This unique dataset enabled unprecedented resolution of the dynamic interactions between tumor and immune compartments in situ over the course of therapy. Our preliminary analysis revealed discrete, virus-associated lymphoid-like structures that formed early after treatment initiation. These structures were enriched in T cells, B cells, and myeloid cells, and were associated with localized reductions in tumor cell density. Over time, we observed a shift from an initially inflamed microenvironment to a more immunosuppressive state. By the time of radiographic progression, these lymphoid structures had largely dissipated—underscoring the importance of on-treatment sampling in capturing transient, treatment-responsive immune architectures. While these findings are preliminary, they suggest a transient immune structure linked to early therapeutic response, whose collapse may contribute to treatment resistance. Our study illustrates how integrative, on-therapy spatial profiling can begin to reveal mechanisms of response and resistance in GBM, providing a framework for refining future immunotherapeutic strategies in brain tumors and other treatment-refractory cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf60db76ebc26534935de49d68b0018aa310569" target='_blank'>
              IMMU-67. On-Treatment Spatiotemporal Profiling of Glioblastoma During Oncolytic ImmunoTherapy
              </a>
            </td>
          <td>
            Charles Couturier, Jennifer Gantchev, Gerard Baquer, Peter Zhang, Alicia D’Souza, Michael S Regan, Kaitlyn Bootz, J. Bonney, Md Amin Hossain, G. Teamlab, Atul Deshpande, Forest M. White, Kenny Yu, E. Chiocca, Nathalie Y R Agar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains among the most aggressive and intractable human cancers, driven by profound intratumoral heterogeneity, dynamic cellular plasticity, and robust mechanisms of immune escape. To overcome these barriers, we developed an integrated discovery and engineering pipeline that combines single-nucleus RNA sequencing (snRNA-seq), antibody generation through both immunization and generative AI design, yeast-based directed evolution, and rationally engineered CAR-T cell architectures. We first assembled a high-resolution snRNA-seq atlas spanning malignant GBM, adult brain, and developmental brain tissues. Cross-comparative analysis revealed PTPRZ1 as a consistently expressed, tumor-enriched surface antigen with minimal expression in normal adult tissues. Further analysis across developmental datasets confirmed PTPRZ1 expression during neurodevelopment, particularly in glioma stem cells and oligodendrocyte progenitor-like states, and showed retention of expression across all major GBM differentiation trajectories. These findings suggest that PTPRZ1 marks a persistent cellular state within the glioma hierarchy, making it a stable and tractable immunotherapeutic target despite GBM’s plasticity. To generate high-affinity binders against PTPRZ1, we pursued two parallel strategies: (1) phage display from immunized hosts followed by affinity maturation via yeast-based directed evolution, and (2) de novo nanobody design using structure-aware generative models trained on antibody sequence and structural datasets. Candidate binders were refined through site-directed mutagenesis to enhance specificity and binding kinetics. Given the known challenges of targeting PTPRZ1, including extracellular domain shedding and glycosylation-dependent surface stability, we engineered a suite of TCR-mimetic CAR-T cells incorporating modular costimulatory domains to enable tunable activation thresholds and improved resistance to exhaustion. These constructs emulate core features of physiological TCR signaling, offering increased antigen sensitivity, functional persistence, and reduced tonic signaling in hostile tumor contexts. This work establishes a modular platform for precision antigen discovery and CAR-T cell engineering in GBM and validates PTPRZ1 as a compelling, developmentally anchored immunotherapeutic target. More broadly, it offers a scalable roadmap for rational design of next-generation cellular therapies against solid tumors that evade conventional targeting strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/693e783962995ec2b156273884091139270f0e5f" target='_blank'>
              EXTH-68. Targeting Tumor Heterogeneity And Plasticity in Glioblastoma With Novel CAR T Cells Targeting PTPRZ1
              </a>
            </td>
          <td>
            Aditya Mohan, Barbara Lipes, Charla Gentry, Rushil Yerabelli, Kisha Patel, Bhairavy J Puviindran, E. Blandford, Ariel Gonzalez, Daniel S. Wilkinson, John H. Sampson, Michael Gunn, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) evolves within a chronic inflammatory renal niche, where angiogenesis, metabolism, and immune programs synergize to reshape the tumor immune microenvironment (TIME). Recurrent renal damage and tissue hypoxia sustain NF-κB/STAT3 and HIF-VEGF signaling, while a sustained IFN-γ response enhances antigen presentation while inducing inhibitory checkpoints, promoting a state of “inflammation but constrained.” Single-cell and spatial studies reveal early microenvironment heterogeneity and the chemokine-checkpoint paradox: regions enriched in CXCL9/10 coexist with endothelial inertia, cancer-associated fibroblasts (CAFs)-mediated stromal barriers, and metabolic stress, which collectively exclude functional CD8+T cells. In advanced clear cell RCC, immuno-inflammatory, immuno-excluded, and immuno-desert phenotypes often coexist and undergo transitions during treatment, leading to heterogeneity in response to immune checkpoint blockade (ICB). We propose a modular perspective-the NF-κB/STAT3, HIF-VEGF, IFN-γ circuits and auxiliary regulatory factors-to link stage-specific biology with treatment matching. Integrative biomarkers couple IFN-γ characteristics with angiogenesis/stromal modules and spatial indicators, offering superior predictive power compared to single tests. These insights support the adoption of a combined strategy: integrating vascular normalization or stromal/myeloid cell reprogramming on the basis of ICB, and encouraging the use of longitudinal “immune snapshots” to guide intervention and precision immunotherapy for renal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bd0a59026e4807122514617a7c05a2fad79de65" target='_blank'>
              From chronic inflammation to immune escape: mapping the tumor microenvironment evolution in renal cell carcinoma
              </a>
            </td>
          <td>
            Hanjun Xu, Baojun Tu, Hui Li, Yong Shan
          </td>
          <td>2025-11-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The immune surveillance hypothesis posits that the immune system controls pre-cancerous cells prior to tumor formation. However, direct visualization of the process has been nearly impossible, because pre-cancerous cells are difficult to identify in conventional cancer models. To address this technical gap, our lab established a unique mouse model called MADM (Mosaic Analysis with Double Markers) that generates rare mutant cells with unequivocal GFP labeling. In a MADM model for triple negative breast cancer, we have discovered the specific and prominent presence of T/B cell-dominated immune aggregates adjacent to premalignant mutant ducts. Characterization via flow cytometry and spatial staining has revealed these aggregates are heterogenous in size, composition, and organization, and tend to dissipate as malignant tumors form. Interestingly, a significant portion of T cells in these aggregates are CD4+ Foxp3+ Tregs, which typically play an important role in preventing excess inflammation. We hypothesize that Tregs regulate immune surveillance against premalignancy to prevent excess inflammation/autoimmunity, but in doing so also prevent eradication of premalignant cells. We are testing this hypothesis using Treg ablation during premalignancy and expect that Treg ablated mice will show increased T/B cell presence and activation in the mammary gland and decreased premalignant cell expansion.



 Supported by the Basser Center for BRCA; University of Virginia (UVA) Pinn Scholarship; NIH/NCI R01-CA256199; UVA Cancer Center Training Grant



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e28dd574c9b2b053c06b1da358808d0a549f6508" target='_blank'>
              Uncovering the role of Tregs in immune surveillance against cancer with a mouse model that reveals premalignancy at single cell resolution 3379
              </a>
            </td>
          <td>
            Elaina K. Ball, Xian Zhou, Jianhao Zeng, Alexys Riddick, Hui Zong
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background The nephron progenitor cells generate approximately one million nephrons during human nephrogenesis. At 34-36 weeks of human genstation, silencing of the key kidney progenitor genes results in depletion of this progenitor pool, limiting the regeneration capacity of the mature kidney. Concurrently, the increasing incidence of end-stage kidney disease underscores the urgent need for innovative regenerative strategies. Methods We employed lentiviral vectors to ectopically induce two key kidney progenitor genes OSR1 and SIX2 individually or together in primary human adult kidney (hAK) cells. We then analyzed the cellular and molecular consequences through morphological assessments, functional assays, in vivo transplantation studies, and comprehensive transcriptional profiling. Results OSR1 and SIX2 induced distinct reprogramming processes with differential functional outcomes; SIX2 overexpression was found to maintain epithelial morphology while significantly enhancing proliferation and clonogenic efficiency. Transcriptionally, SIX2 established epithelialization and cell-cycle networks by downregulating proximal tubule markers while upregulating distal nephron markers and proliferation genes. In vivo, SIX2-expressing cells formed organized tubular structures with a distinct luminal architecture in a proof-of-concept model. In contrast, OSR1 overexpression was found to induce morphological changes and activate developmental morphogenetic pathways, including epithelial tube morphogenesis and canonical Wnt signaling; however, it did not enhance proliferation and showed minimal tubulogenic capacity in vivo. Unexpectedly, OSR1 overexpression led to malignant transformation in one clone and exhibited Wilms’-tumor-like features, including expression of kidney developmental markers (i.e., SIX2, NCAM1, and WT1) and blastemal phenotype. Conclusion Our findings suggest that SIX2 overexpression in primary hAK cells functionally confers enhanced self-renewal and tubulogenic capacity while transcriptionally inducing a proximal-to-distal tubular cell diversion with maintained proliferative programs. In contrast, OSR1 activates the broader developmental morphogenetic networks but poses potential oncogenic risks. The malignant transformation observed with OSR1 overexpression provides insights into the potential cellular origins of Wilms’ tumor and raises important safety considerations for regenerative medicine approaches involving developmental gene induction in adult kidney cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f6cb330465aacec4853d965847f4bf43b16c3ba" target='_blank'>
              OSR1 and SIX2 drive divergent transcriptional programs in human kidney cells: implications for regeneration and tumorigenesis
              </a>
            </td>
          <td>
            N. Pode-Shakked, O. Cohen-Zontag, Dorit Omer, O. Harari‐Steinberg, E. Vax, O. Pleniceanu, Benjamin Dekel
          </td>
          <td>2025-10-03</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Recent advances in single-cell multi-omics technologies have revolutionized cellular analysis, enabling comprehensive exploration of cellular heterogeneity, developmental trajectories, and disease mechanisms at unprecedented resolution. Foundation models, originally developed for natural language processing, are now driving transformative approaches to high-dimensional, multimodal single-cell data analysis. Frameworks such as scGPT and scPlantFormer excel in cross-species cell annotation, in silico perturbation modeling, and gene regulatory network inference. Multimodal integration approaches, including pathology-aligned embeddings and tensor-based fusion, harmonize transcriptomic, epigenomic, proteomic, and spatial imaging data to delineate multilayered regulatory networks across biological scales. Federated computational platforms facilitate decentralized data analysis and standardized, reproducible workflows, fostering global collaboration. Challenges persist, including technical variability across platforms, limited model interpretability, and gaps in translating computational insights into clinical applications. Overcoming these hurdles demands standardized benchmarking, multimodal knowledge graphs, and collaborative frameworks that integrate artificial intelligence with human expertise. This review synthesizes recent technological advancements and proposes actionable strategies to bridge single-cell multi-omics innovations with mechanistic biology and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd39e97dd01c5b51454dd49ad3ed59f55d8cf50a" target='_blank'>
              Transformative advances in single-cell omics: a comprehensive review of foundation models, multimodal integration and computational ecosystems
              </a>
            </td>
          <td>
            T. Yiu, Bin Chen, Haoyu Wang, Genyi Feng, Qiangqiang Fu, Huijing Hu
          </td>
          <td>2025-10-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f5dbf13eb9670595afcc13d733cac2504d66e7a" target='_blank'>
              Single-cell aggrephagy landscape delineates intercellular communication of tumor microenvironment associated with ovarian cancer progression and immunotherapy.
              </a>
            </td>
          <td>
            Feng Fang, Jiemiao Chen, Xinran Li, Cao Zhou, Yilan Dai, Yuanyuan Chen, Xin Yang, Yong Xie, Fujie Qin, Huijiao Fu
          </td>
          <td>2025-11-24</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 With the rise of immune checkpoint inhibitors (ICIs) for the treatment of metastatic renal cell carcinoma (RCC), understanding the role of T cells in the tumor microenvironment (TME) is critical. Single-cell transcriptomics have provided insights into T cell phenotypes, but functional validation is needed to develop novel therapies. We developed a patient-derived tumor model (PDTM) to assess inhibitory immune interactions on T cell function and anti-tumor activity in the RCC TME. Fresh RCC tumor samples were minced into ∼1 mm3 pieces and cultured in an air-liquid interface system embedded in a collagen matrix with or without anti-PD-1 antibody. The PDTM system maintained the TME’s cellular architecture and immune composition over three days, as confirmed by immunohistochemistry and hematoxylin and eosin staining. Flow cytometry (FCM) identified well-preserved CA9+ tumor cells and immune cell populations, including CD4+ and CD8+ T cells, regulatory T cells, NK cells, B cells, and myeloid subsets. Anti-PD-1 treatment with low-dose T cell stimulation resulted in increased activated CD8 T cells (CD3+CD8+CD25+) by FCM and significantly increased IFN-γ production (p < 0.001, paired t-test) by ELISA, demonstrating PD-1 blockade’s functional relevance ex vivo. Our novel PDTM system preserves the RCC TME for functional interrogation and has extensive applications in studying RCC, including assessing the impact of ICIs and novel combination therapies on T cell function.



 This work was supported by DOD KCRP KC230003 and NIH grant 1R37CA279822-01.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ee0ffefa1fb0f2f1e8a8071118f25e16b289fc7" target='_blank'>
              Development of ex vivo patient-derived models to uncover the role of the tumor immune microenvironment in renal cell carcinoma 4463
              </a>
            </td>
          <td>
            Ruchi Gupta, S. Kashima, V. Moritz, Katherine Sadak, Moreen Ng, Adebowale J. Adeniran, Peter Humphrey, M. Bosenberg, M. Hurwitz, Patrick Kenney, David Braun
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6316e334af69e0e24d88cebcc1c3792b9c71c172" target='_blank'>
              Archetype analysis of lung adenocarcinoma premalignancy links heterogeneity in premalignant lesions to diverging features of invasive disease
              </a>
            </td>
          <td>
            Kelley E. Anderson, Linh M. Tran, K. Krysan, Y. Kefella, Lingge Yu, Gregory A. Fishbein, Erika Rodriguez, Maryam Shabihkhani, Daniel P. Stefanko, E. Green, Gang Liu, Hanqiao Liu, Sherry Zhang, E. Kane, M. Mehrad, Avrum E. Spira, Stephen Dubinett, Eric J. Burks, S. Mazzilli, Marc E. Lenburg, J. Beane
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Cancer arises from genetic and epigenetic alterations that reprogram gene expression networks, driving unchecked proliferation. Glioblastoma (GBM), the most common and lethal primary brain tumor, has a median survival of about one year and a dismal 5% five-year survival rate. Despite extensive research, effective treatments remain elusive, underscoring the need for a deeper understanding of GBM biology. Although several genetic alterations have been implicated in gliomagenesis, driving genetic factors responsible for GBM are still obscure, and how these genetic changes interact with epigenetic dysregulation is poorly understood. Here, we identify CHD5, a gene located in the frequently deleted 1p36 chromosomal region, as recurrently deleted in over 20% of GBM cases and further downregulated epigenetically. To investigate the role of CHD5 in gliomagenesis, we generated a glioma-prone mouse model carrying conditional alleles of Pten, Trp53, and Chd5, and employed engineered neural stem cells (NSCs)—a major GBM cell-of-origin—to induce tumor formation. Loss of Chd5 impaired NSC differentiation, enhanced proliferation, and drove in vivo gliomagenesis. Mechanistically, Chd5 forms a NuRD complex that binds the Myc promoter and super-enhancers to regulate its expression. Chd5 loss leads to chromatin accessibility changes, Myc upregulation, and aggressive tumor development. Importantly, reactivating CHD5 in human GBM xenografts significantly extended survival. These findings establish CHD5 as a tumor suppressor in gliomagenesis, offer therapeutic insights for cancers with 1p36 deletions, and present a robust NSC-based platform for functional genetic studies and modeling GBM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccecf93321da1d6d66641061e1225be0f52f0874" target='_blank'>
              EPCO-68. CHD5 IS A TUMOR SUPPRESSOR IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Xueqin Sun, A. Mills
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 For glioblastoma (GBM), like other solid tumors, intratumoral heterogeneity is likely a critical factor in promoting tumor growth, homeostasis, and therapeutic response. Recent applications of single-cell genomics techniques have revealed the scope and nature of GBM cell diversity. However, these techniques do not allow for the experimental manipulation of these same tumor subpopulations, raising a critical barrier to understanding tumor biology and developing effective therapies. To overcome this barrier, we developed endogenous gene reporter systems to enable the isolation, characterization, and manipulation of key GBM subpopulations directly in patient tumor isolates. This approach utilizes streamlined techniques for gene editing in GBM stem-like cells (GSCs), which eliminate the need for the cloning of individual cells. As proof of concept, we applied this approach to two genes in hGSC isolates: OLIG1 (GBM-OPC marker) and SOX9 (GSC/NSC stem cell homeostasis). We targeted the 3’ exon of each gene so that EGFP or RFP are in-frame with the upstream exon but separated by a 2A ribosome skip sequence. We then employed these GSC reporter isolates to comprehensively screen human transcription factors using a CRISPR-Cas9 library and LV-ORF library (1836 genes and 3548 isoforms) to identify transcription factors that activate or repress OLIG1 and SOX9. From these screens we identified 15 genes necessary and sufficient for regulating OLIG1 promoter activity, including known and novel regulators. These included SOX9 itself, which we find is a key factor linking OPC identify with self-renewal of GSCs. In addition, we identified dozens of transcription factor isoforms (validated by long read RNA-seq) that impact reporter activity, as well as ectopic factors not normally expressed in our GSCs which, nonetheless, significantly alter OLIG1 and SOX9 expression. At this meeting, we report results of these screens and present a functional genetic map of factors affecting the GBM-OPC cell identity, self-renewal, and plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9966806d724ae473e21ea19603f70a36860465" target='_blank'>
              STEM-26. COMPREHENSIVE FUNCTIONAL GENOMIC SCREENS REVEAL HUMAN TRANSCRIPTION FACTORS REQUIRED TO MAINTAIN GLIOBLASTOMA OLIGODENDROCYTE PROGENITOR CELL-LIKE IDENTITY
              </a>
            </td>
          <td>
            Anca B. Mihalas, Kelly Mitchell, Sonali Arora, Colette Felton, Mikey Witkowski, Aiden Abadilla, Daniel A. Kuppers, Angela Brooks, Alice Berger, Patrick J. Paddison
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a8fc3f37c54ae77fa1e33d4bf2768934a457fd9" target='_blank'>
              Single-cell transcriptomic profiling of bam mutant tumor reveals germline heterogeneity and gcrf1 as a modulator in Drosophila germ cells
              </a>
            </td>
          <td>
            Zhipeng Sun, Yujun Zeng, T. Nystul, Guohua Zhong
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="T/NK-cell neoplasms represent rare and highly diverse cancers, distinguished by variability in their molecular architecture, local inflammatory milieu, and microenvironmental composition, which collectively underpin the diversity of clinical presentations and outcomes. The neoplastic tissue comprises malignant lymphoma/leukemic cells in concert with a spectrum of stromal elements and the acellular extracellular matrix (ECM), collectively constituting the lymphoma microenvironment (LME). These components engage in dynamic, reciprocal interactions, forming a self-regulating ecosystem capable of responding adaptively to both exogenous and endogenous stimuli. Historically, the LME was largely neglected in considerations of lymphomagenesis; however, emerging evidence highlights its pivotal role in driving core oncogenic processes, including sustained proliferative signaling, angiogenesis, immune evasion, and apoptotic resistance. Deciphering the intricate, multidirectional crosstalk among the cellular and acellular constituents of the T/NK-cell neoplastic microenvironment promises to deepen our understanding of disease biology and may inform the development of novel, mechanism-based therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f57f1ac49eb4898f862ff3159155ed73801efa79" target='_blank'>
              The Significance of the Microenvironment in T/Nk-Cell Neoplasms
              </a>
            </td>
          <td>
            Ivan Petković, Michele Ritucci, Ana Stojković, S. Stojnev, Aleksandar Popović, I. Conić, Milica Radić, Miljana Džunić, M. Krstić
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Pre-invasive breast cancer, or ductal carcinoma in situ (DCIS), shares many morphologic and genomic features with invasive breast cancer, yet most DCIS tumors remain indolent over decades. In this study, we performed spatial analyses of somatic mutation patterns in 18 DCIS tumors to infer the underlying growth dynamics. Experimental data combined with mathematical models and Bayesian computation revealed a growth model that mimics the cellular dynamics of branching morphogenesis, which governs normal mammary gland development during puberty. Unlike a traditional model of clonal evolution, this non-competitive growth model allowed for extensive DCIS spread without the selective pressures of subclone competition that promote the emergence of invasive phenotypes. This finding provides a framework to understand the biological basis for the limited progression of DCIS and adds rationale for ongoing clinical efforts to reduce DCIS overtreatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56eaf0d13e80db0fb1f5ee2e2242518bc8cf64f7" target='_blank'>
              Spatially Discontinuous Mutation Topographies in Ductal Carcinoma in Situ Reveal Non-Competitive Growth Dynamics.
              </a>
            </td>
          <td>
            Marc D. Ryser, Matthew A Greenwald, Daniel L. Monyak, I. Sorribes, Lorraine M. King, Allison Hall, J. Geradts, Donald L Weaver, D. Mallo, Shannon T. Holloway, Ethan Wu, Kevin A. Murgas, Robert B. West, Lars J. Grimm, Carlo C. Maley, Jeffrey R. Marks, Darryl Shibata, E. S. Hwang
          </td>
          <td>2025-10-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a leading male malignancy worldwide, with metabolic reprogramming being a critical hallmark of its progression. Extracellular vesicles (EVs) derived from tissues directly reflect the tumor microenvironment, offering unique insights into cancer pathophysiology that are unattainable through cell line or biofluid-derived EVs. However, the functional roles of tissue-derived EVs in PCa metabolism remain poorly understood. Leveraging our expertise in murine PCa model establishment and EV isolation from prostate tissue, this study aimed to characterize functional differences between PCa and normal prostate tissue via proteomic analysis of tissue-derived sEVs. We orthotopically implanted luciferase-labeled PCa cells into nude mice to establish an in situ PCa model, confirmed tumor formation via in vivo imaging, and harvested tissues after 4 weeks. sEVs were isolated using ultracentrifugation combined with an iodixanol density cushion and characterized by transmission electron microscopy, nanoparticle tracking analysis, and protein marker profiling. Proteomic analysis identified 28 upregulated and 24 downregulated proteins in PCa-derived sEVs compared to normal controls. Subcellular localization revealed enrichment in the cytoplasm, while pathway analysis highlighted significant involvement in metabolic processes, particularly glycolysis, amino acid biogenesis, carbon metabolism, and pyruvate metabolism. Our study establishes a robust method for isolating prostate tissue sEVs and provides the first evidence that PCa tissue-derived sEVs exhibit profound metabolic pathway alterations. These findings enhance our understanding of PCa progression mechanisms and may facilitate the development of novel diagnostic biomarkers and therapeutic strategies targeting metabolic dysregulation in PCa. SUMMARY: In this study, we created a method to isolate prostate tissue small EVs, based on our knowledge of the murine prostate cancer model building. Our data suggested that prostate tissue small EVs proteins significantly changed in many metabolism pathways, such as Glycolysis, Biogenesis of amino acids, Carbon metabolism and Pyruvate metabolism. In this study, we are the first to report prostate tissue-derived EVs proteins enriched in alterations of cancer metabolism. These differential proteins in PCa tissue EVs reflect metabolic changes in PCa and may provide insights into the development of early diagnostic biomarkers or novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c976599295f2bbc1baf63bb2a61d05088e5e38" target='_blank'>
              Proteomics of Prostate Cancer Tissue Small Extracellular Vesicles Reveal Alteration of Metabolism.
              </a>
            </td>
          <td>
            Lijuan Yu, Ting Ding, R. Olofsson Bagge, Lei Zheng, Xiaoke Hao
          </td>
          <td>2025-11-17</td>
          <td>Proteomics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Glioma evolution is associated with disease progression and therapeutic resistance. We recently identified a group of MAPK-driven glioma with CDKN2A homozygous deletion that are low-grade and immune enriched, characterized by the presence of antigen presenting cells (APCs) and activated cytotoxic CD8⁺ T cells. Over time, however, these tumors can progress to aggressive, higher-grade tumors. While low-grade tumors may be amenable to immunotherapeutic strategies, it is unknown how the tumor and its microenvironment evolve. To investigate this, we tested human tumors before and after progression within the same patients (n=7) over time (longitudinally) and across patients (n=21) using transcriptomic and proteomic approaches. In neoplastic cells, genetic evolution was accompanied by increased tumor cell proliferation and differentially expressed genes were enriched for telomere organization and maintenance (RTEL1, TERC, HSP90) and response to hypoxia (VEGFA, VEGFC, HIF1A). In parallel, the immune response became predominantly immunosuppressive, marked by an expansion of immunosuppressive macrophages, a reduction in APCs, including dendritic cells, and transcriptional reprogramming of T cells. At progression, T cells displayed increased exhaustion, diminished cytotoxicity (reduced GZMA), and enhanced expression of immunosuppressive mediators (IL-10 and TGF-β). Using spatial proteomics and spatial transcriptomics, we investigated factors potentially driving tumor–immune co-evolution. With disease progression, reductions in APCs and activated CD8 T cells were associated with altered cell-cell signaling. CXCL16–CXCR6 signaling between APCs and T cells was decreased and CXCL14, a chemokine implicated in recruitment of both CD8 T cells and APCs to tumors, was decreased. Indeed, CXCL14 expression correlated with an increased CD11c expressing population of antigen presenting cells. Thus, tumor progression is accompanied by a coordinated evolution of both tumor and immune compartments, driven in part by changes in cell-cell communication within the tumor microenvironment. Ongoing studies are investigating how targeting these communication pathways within the tumor microenvironment may alter tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f32c872c50770144e79a4ef320e0649d8c4401f" target='_blank'>
              TMIC-88. MAPK- driven glioma progression and reprogramming of the tumor-associated immune response
              </a>
            </td>
          <td>
            Elaheh Hashemi, Akshaya R.L, Lin Wang, Viva Voong, Hadeesha Piyadasa, Benjamin Oberlton, Spencer Bergland, Anupam Kumar, R. Prins, Sean C. Bendall, Michael Angelo, Mitchell S. Berger, S. Hervey-Jumper, H. Okada, Aaron A. Diaz, Joanna Phillips
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) exhibits significant transcriptomic diversity, limiting treatment efficacy. This study aims to identify conserved molecular pathways that remain stable in heterogeneous TNBC, providing reliable therapeutic and prognostic targets. Genomic diversity was assessed via Mutant-Allele Tumor Heterogeneity (MATH) in TCGA-TNBC cohorts. Single-cell RNA sequencing from TNBC samples (GSE176078) underwent non-negative matrix factorization (NMF) to identify intratumoral heterogeneity (ITH) metaprograms. Transcriptomic diversity was quantified using PCA-based cell-cell Euclidean distances. CellChat analyzed cellular communication in high- and low-diversity groups. Conserved pathways were identified using KEGG enrichment, ssGSEA, and spatial transcriptomics. And we confirmed the feasibility of targeting sphingolipid metabolism through in vitro and in vivo experiments. TNBC showed elevated genomic diversity, inversely correlated with immune infiltration and pathway activity. Single-cell RNA sequencing revealed ITH metaprograms across malignant subtypes, with Cancer Cycling cells showing proliferative enrichment. High transcriptomic diversity tumors exhibited enhanced Cancer Basal SC interactions with CAFs, promoting stemness and EMT. Multi-region TNBC dataset analysis confirmed sphingolipid metabolism as the most conserved pathway (high interpatient and low intratumoral heterogeneity), enriched in malignant regions and negatively correlated with immune cells. A prognostic model incorporating sphingolipid genes (SPTLC2, SGMS1, SGPP2) stratified high-risk patients. Inhibiting SPHK1 via PF-543 induced dose-dependent cell death. Sphingolipid metabolism could be a transcriptionally conserved pathway in TNBC. Its inhibition may overcome heterogeneity-driven resistance, and its prognostic value enables precise patient stratification, offering a potential strategy against TNBC plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/273a42854b1bca75f9d7c92c18142aeda8983ebb" target='_blank'>
              Conserved sphingolipid metabolism under transcriptomic diversity: a prognostic and therapeutic target in triple-negative breast cancer
              </a>
            </td>
          <td>
            Jian Li, Rui Chen, Xiaowu Wang, Qing Zheng, Ruijie Dai
          </td>
          <td>2025-11-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Anti-angiogenic therapies are commonly employed in colon cancer management, yet many patients eventually develop resistance and experience disease progression. Vasculogenic mimicry (VM)—the formation of tumor-derived vessel-like networks—has been recognized as one mechanism contributing to this resistance, although the underlying details remain incompletely understood. Here, by integrating bioinformatic analyses of publicly available datasets and validating the results in patient samples (n = 157), we identified follistatin-like 3 (FSTL3) as a critical factor predominantly expressed in colon cancer-associated fibroblasts (CCAFs), with its expression strongly correlating with increased VM formation, intratumoral blood vessels, and poor prognosis. Single-cell RNA sequencing of tumors from VM and non-VM patients revealed that hypoxia drives FSTL3 expression in CCAFs, leading to extracellular matrix remodeling and enhancing cancer cell endothelial-like plasticity. Mechanistically, FSTL3 binds to transferrin receptor (TfR1), an iron-uptake receptor on cancer cells, thereby activating the TfR1/AKT/mTOR pathway and elevating VE-Cadherin to support endothelial-like transformation, VM, and metastatic progression. Notably, FSTL3-targeting antibodies (aFSTL3) effectively inhibited VM and angiogenesis in both in vitro and in vivo models, while the combination of aFSTL3 with bevacizumab produced synergistic suppression of neovascular-like structures and distant metastases. These findings demonstrate a pivotal role for FSTL3+ CCAFs in facilitating VM through TfR1-mediated signaling and offer a promising dual-target approach to overcome anti-angiogenic therapy resistance in colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f12af58c0e4c8605b5eb8483da860b03f136a52" target='_blank'>
              Cancer-associated fibroblasts expressing FSTL3 promote vasculogenic mimicry formation and drive colon cancer malignancy
              </a>
            </td>
          <td>
            Leqian Ying, Yini Zhu, Lu Zhang, Min Ji, Meidan Wang, Lei Dong, Zhengcheng Yun, Yanping Chen, Jingyi Zhou, Chunchun Huang, Shengwei Zhang, Xuhong Yang, Hui Yang, Guichun Huang, Shukui Qin, Jinbing Xie, Lin Liu
          </td>
          <td>2025-10-06</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) represent the main immune cell population in various brain malignancies, but there is rare knowledge on the functional and, in particular, the prognostic role of TAMs in the meningioma (MGM) microenvironment. Here, we investigated TAM frequencies, activation state, survival-associated changes, and their association with tumor-infiltrating T lymphocytes (TILs) in two independent study samples comprising altogether 680 MGMs. To this end, we performed tissue cytometry analyses, quantified tissue cytokine levels, and integrated previously published TIL infiltration and microarray datasets in the discovery cohort comprising n = 195 clinically well-annotated cases. This was complemented by a DNA methylation-based deconvolution approach to predict TAM and TIL infiltration rates using immune cell-specific CpG sites as well as survival associations in an independent validation cohort of n = 485 MGMs. Our findings revealed substantial but heterogeneous TAM infiltration in newly diagnosed MGMs, with increased numbers of pro-tumoral TAMs in clinically aggressive tumors. Additional cytokine and transcriptome analyses corroborated the presence of an immunosuppressive niche in TAM-enriched MGMs. Importantly, a high frequency of pro-tumoral TAMs was associated with poor patient outcome, and high TAM infiltration was further identified as an independent prognostic factor for inferior survival, counteracting the beneficial prognostic effect of TILs. Moreover, methylation-based deconvolution analyses confirmed the opposing prognostic roles of TAMs and TILs in the validation cohort. Altogether, higher numbers of TAMs appear to be a hallmark of clinically aggressive behavior in newly diagnosed and recurrent MGMs. Unlike TILs, immunosuppressive TAMs seem to play a dominant role in the immunological landscape of MGMs with a significant negative impact on patient outcome, highlighting pro-tumoral TAMs to be an attractive treatment target in MGMs. Furthermore, our deconvolution approach presents a pipeline to computationally determine TAM and TIL infiltrates in the MGM microenvironment, which might be highly valuable for patient stratification for future immunotherapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9eb1c2486a92614ef9d3bad76049f5fefd9924e" target='_blank'>
              Tumor-associated macrophages in meningiomas: a novel biomarker for poor survival outperforming the benefits of T cells
              </a>
            </td>
          <td>
            C. Lotsch, R. Warta, Fang Liu, G. Jungwirth, C. Rommel, M. Barthel, K. Lamszus, A. Kessler, Niels Grabe, Mario Loehr, R. Ketter, C. Senft, S. Maas, P. Sievers, Manfred Westphal, Sandro M. Krieg, A. Unterberg, M. Simon, Andreas von Deimling, F. Sahm, D. Raleigh, C. Herold-Mende
          </td>
          <td>2025-10-09</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCC) with multinodular morphology were typically polyclonal and exhibited extensive heterogeneity. We explored the multiple-layer heterogeneity of HCC utilizing single-cell multiomics sequencing together with multiregional sampling. We found that Confluent Multi-Nodular samples exhibit more heterogeneous immune landscapes characterized by increased transcriptome heterogeneity and more complex immune-related interactions compared to Single Nodular (SN) samples. We identified differential DNA methylation patterns among distinct tumor foci in a subset of liver cancer patients. Global DNA hypomethylation in cancer cells predominantly occurs in partially methylated domains. We predicted and validated two genes, GADD45A and SNHG6, as potential drivers of DNA hypomethylation. Lastly, we demonstrated that DNA methylation distance serves as a more robust metric than gene expression distance for reconstructing tumor evolutionary trajectories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eb1f83715ac49e3d497001d0455b6e605a23122" target='_blank'>
              Interrogating subclonal heterogeneity of liver cancer with single-cell multi-omics analysis
              </a>
            </td>
          <td>
            Liwen Zhang, Dandan Hu, Xing Peng, Dan Su, Lei Chang, Yi Guo, Kai Tian, Qingpei Huang, Enze Deng, Zhuojie Su, Longxi Li, Xiong Xiong, Weiyuan Ma, Hongjian Qi, Yaojun Zhang, Yanxiao Zhang, Xiaoying Fan
          </td>
          <td>2025-11-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Despite decades of concerted research and clinical efforts, patient outcomes in glioblastoma (GBM) remain dismal. While chimeric antigen receptor (CAR)- T cell therapies in GBM have produced individual reports of efficacy, they have yet to meaningfully alter standard of care. Tumor heterogeneity, both inter- and intra-patient, likely contributes to these variable responses. Single-cell RNA sequencing (scRNA-seq) conducted in our lab has revealed that this transcriptional heterogeneity is organized along distinct cellular state axes. We sought to overcome this heterogeneity by designing CAR-T cells directed against these cellular states. We hypothesize that CAR-T cells targeting a defined GBM cell state will lead to a depletion of those populations. We report the development of a panel of cell state-targeting CAR-T cell candidates engineered for target-specific activity in vitro. To model the three-dimensional tumor environment, we developed patient-derived GBM organoids that recapitulate the cell state landscape seen in primary tumors. We treated GBM organoids with these cell state-targeting CAR-T cells and assessed cell states by scRNA seq. Unexpectedly, we observed convergent transcriptome shifts to a new cell state with overlapping features of mesenchymal (MES) and interferon-induced signatures. To target this dynamic, we designed a novel CAR-T cell directed against this population and observed feedforward target enrichment in response to therapy. We anticipate that our findings will inform rational design of translational CAR-T cell candidates for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae77281b5ad33a2d2f93d13dc0c418dca38401a5" target='_blank'>
              EXTH-37. Evaluating cell state-targeting CAR-T cells in glioblastoma
              </a>
            </td>
          <td>
            S. Kovatsis, C. Mount, E. Boxer, Jun Zhong, S. Dumont, D. Gerovasilis, Jack Lu, I. Tirosh, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a high-grade glioma marked by high intratumoral molecular heterogeneity, resistance to therapy and poor prognosis. Leveraging a multisampling approach, we investigated the relation between genomic and physical distance, and whether this information was captured by imaging phenotypes and patient outcomes. We profiled 79 spatially distinct tumor regions from 24 GBM patients (F/M = 0.60; median age = 65.5 years) using deep whole-exome sequencing (>300x coverage). Genomic distance was quantified by pairwise Euclidean distance of somatic mutational profiles and integrated with 3D neuronavigation-based spatial coordinates and progression-free survival (PFS). Two distinct growth patterns emerged: expansive, defined by strong correlation between molecular and spatial distance (RE = 0.6), and stochastic, characterized by molecular divergence uncoupled from spatial proximity (RS = -0.2). High molecular distance correlated with reduced PFS (R = -0.5538, p = 0.026), and stochastic expansion predicted unfavorable outcome (p = 0.035), frequently localized to the frontal lobe. Radiomic analysis from contrast-enhanced T1-weighted MRI revealed that molecular distance was positively correlated with heterogeneous texture features (e.g., GLCM entropy, NGTDM complexity; p < 0.05). Stochastic tumors were enriched for radiomic heterogeneity (e.g.), while expansive tumors displayed homogeneous imaging textures, suggesting convergent phenotypic adaptation in genomically diverse tumors. Functional annotation using neuromaps demonstrated that tumors with high molecular distance exhibited positive correlations with differentiation- and angiogenesis-related metrics (SA_axis, genePC1, CBV), and negative associations with expression of neurotransmitter receptors (acetylcholine, serotonin, dopamine). Stochastic tumors correlated with increased Magnetoencephalography (MEG) derived timescales, implicating higher neural network engagement and spatial fluctuations. These findings suggest a link between type of clonal expansion, radiomic-based structural imaging phenotype, and functional neuroanatomy in GBM, providing a multidimensional framework to interpret tumor behavior and identify clinically relevant radiogenomic signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/111ccc1f311e4f7d5b04bf8582bc8726a2476d94" target='_blank'>
              CNSC-23. MULTIDIMENSIONAL SPATIAL AND MOLECULAR MODELING OF REFRACTORY GLIOBLASTOMA EVOLUTION
              </a>
            </td>
          <td>
            Youjin Song, Y. Miller-Michlits, K. Nenning, L. Greutter, B. Kiesel, D. Lötsch-Gojo, J. Gesperger, J. Breu, D. Nam, G. Widhalm, J. K. Sa, A. Woehrer
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de62dfbaf4b4553a5a57c81de9ba67688e29afbe" target='_blank'>
              Microphysiological systems for metastasis research: a stepwise approach.
              </a>
            </td>
          <td>
            Vira Sharko, Ignacio Ochoa, Estela Solanas
          </td>
          <td>2025-10-22</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Therapeutic resistance remains a defining challenge in oncology, limiting the durability of current therapies and contributing to disease relapse and poor patient outcomes. This review systematically integrates recent progress in understanding the molecular, cellular, and ecological foundations of drug resistance across chemotherapy, targeted therapy, and immunotherapy. We delineate how genetic alterations, epigenetic reprogramming, post-translational modifications, and non-coding RNA networks cooperate with metabolic reprogramming and tumor microenvironment remodeling to sustain resistant phenotypes. The influence of the microbiome is highlighted as an emerging determinant of therapeutic response through immune modulation and metabolic cross-talk. By summarizing key regulatory circuits, We establishe a unified framework linking clonal evolution, metabolic adaptability, and tumor ecological dynamics. We further synthesizes novel therapeutic strategies that convert resistance mechanisms into therapeutic vulnerabilities, including synthetic lethality approaches, metabolic targeting, and disruption of stem cell and stromal niches. Advances in single-cell and spatial omics, liquid biopsy, and artificial intelligence are emphasized as transformative tools for early detection and real-time prediction of resistance evolution. This review also identifies major translational gaps in preclinical modeling and proposes precision oncology frameworks guided by evolutionary principles. By bridging mechanistic understanding with adaptive clinical design, this work provides an integrated roadmap for overcoming therapeutic resistance and achieving sustained, long-term cancer control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5732b68dc3b5afcc6ec00f3b3bd38479653a277" target='_blank'>
              Drug resistance in cancer: molecular mechanisms and emerging treatment strategies
              </a>
            </td>
          <td>
            Jinxin Li, Jiatao Hu, Yiren Yang, Hanzhong Zhang, Ying Liu, Yu Fang, Le Qu, Anqi Lin, Peng Luo, Aimin Jiang, Linhui Wang
          </td>
          <td>2025-11-17</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/394995c0fcb10ae0448160919d2f62f9453d3170" target='_blank'>
              Comprehensive single-cell analysis reveals mast cells’ roles in cancer immunity
              </a>
            </td>
          <td>
            Hao Qian, Wenbo Wang, Jiaomeng Pan, Ming Zhao, Li Gao, Jia Zeng, Hang Gong, Guangyan Zhangyuan, Zi Li, Mao Zhang, Xia Shen, Yuanchi Weng, Xiaxing Deng, Yingying Huang
          </td>
          <td>2025-10-10</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Antiretroviral therapy (ART) has transformed HIV into a manageable chronic condition. Stopping ART causes rapid viral rebound, revealing complex and latent tissue reservoirs that hinder efforts for a cure. Understanding the biological, cellular, and tissue-specific factors involved is key to overcoming this challenge. In-situ spatial biology and multiplex histology technology can provide novel insights into the spatial immune and anatomical dysregulation driven by HIV. We performed in-situ multiplex-spatial proteomic analysis using the Nanostring CosMx Human 64-plex Immuno-Oncology Protein Panel on fresh-frozen spleen and gut tissues from two people with HIV (PWH) (detectable and undetectable) from the Last Gift Rapid Research Autopsy Cohort. Data quality control and normalization was processed with the AtoMx pipeline. Despite viremia status, we found similar total protein expression in the spleen and colon tissue. Cell type annotation using the CELESTA pipeline revealed that the viremic donor had more vascular smooth muscle cells in the colon, while the aviremic donor had more epithelial and fibroblast cells. Immune profiling showed higher lectin specific protein expressions in immune cells of the aviremic donor, particularly in dendritic and epithelial cells in the colon compared to the viremic donor. These preliminary findings highlight tissue-specific immune and cellular changes in PWH, with future studies planned to compare these changes to those in people without HIV.



 Supported by NIH/NIAID AI-164559



 Technological Innovations in Immunology (TECH)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5645976e541610bcd1dcd387cc992098aa48a1ae" target='_blank'>
              Spatial biology discovery approaches to dissect in situ modification of site-specific tissue HIV reservoirs 2542
              </a>
            </td>
          <td>
            Stephen T. Yeung, P. Moar, Vanessa Gomez-Moreno, Davey M Smith, A. Chaillon, S. Gianella, L. Ndhlovu
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88201947fac2d05b37de4b53710cc1d1179ec932" target='_blank'>
              MEF2D-expressing cancer precursors reprogram tissue-resident macrophages to support liver tumorigenesis.
              </a>
            </td>
          <td>
            Junyu Xiang, Y. Ni, Jiajun Yu, Hui Sun, Zhiyun Gu, Ziyong Li, Tongwang Yang, Juan Feng, Li Su, Limei Liu, Jiatao Li, Jesús Prieto, Matías A Ávila, Juanjuan Shan, Na Zhuang, Cheng Qian
          </td>
          <td>2025-10-23</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f662f4b181b3395cd096885302fa365b2511567e" target='_blank'>
              Exploring tumor heterogeneity and drug resistance in cervical cancer through single-cell and bulk transcriptomics.
              </a>
            </td>
          <td>
            Yinxia Yang, Sirong Cheng, Yuxuan Zhong, Pengyu Sun, Xiaoru Deng, Bohui Zhou
          </td>
          <td>2025-11-27</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The biological heterogeneity between left- and right-sided colorectal cancer (CRC) poses a significant clinical challenge and the underlying regulatory mechanisms remain elusive. As an emerging hallmark of cancer, the contribution of circadian rhythm disruption (CRD) to this side-specific heterogeneity is unclear. By integrating single-cell and spatial transcriptomic analyses, this study shows that tumor cells in right-sided CRC exhibit significantly higher CRD scores and identifies the emergence of a NONO-positive tumor-cell subpopulation (NONO⁺ TC) as a key molecular feature of this phenotype. Spatial analysis further confirms that this NONO⁺ TC forms a tightly co-localized microenvironment with fibroblasts in situ. Notably, cell–cell communication analyses indicate that NONO expression does not augment the signal-sending capacity of tumor cells but instead reprograms them into highly efficient signal receivers. These augmented incoming signals predominantly originate from the more pro-tumorigenic myofibroblastic cancer-associated fibroblast (myCAF) subtype, indicating that NONO⁺ TC are particularly sensitive to malignant stromal inputs. In conclusion, our study delineates the dual role of NONO as both a circadian regulator and a pro-tumorigenic signaling hub. By enhancing tumor-cell reception of CAF-derived signals, NONO links CRD to the formation of NONO⁺ TC niches and a malignant microenvironment in right-sided CRC, providing new mechanistic insight into the spatial heterogeneity of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241b9e1e30a34229f91641257f9f4e9436ac45df" target='_blank'>
              NONO links circadian rhythm disruption and enhanced tumor-fibroblast crosstalk in right-sided colorectal cancer
              </a>
            </td>
          <td>
            Zhi-Hao Shang, Qin-Chang Zhang, Song-yang Xi, Kai Chen, Shao-Bo Guo, Zhou Zhou, Xin-Zhuo Zhan, Yun-Xia Wu, Xin-Yi Li, Hai-Bo Cheng, Xue-Jun Song, Gui-Hua Tian
          </td>
          <td>2025-10-31</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains one of the most difficult solid tumors to treat, owing in part to its remarkable cell state plasticity - a property that enables tumor cells to resist therapy, evade differentiation, and adapt to selective pressures. While targeted therapies have succeeded in subsets of patients, most tumors persist and eventually progress. To pursue a fundamentally new therapeutic strategy, we conducted an unbiased chemical screen using a custom-designed dual reporter system designed to broadcast cell state dynamics in real time by fluorescently tagging the endogenous SOX9 (aberrant stem-like) and KRT20 (differentiated) loci. This screen targeted epigenetic regulators capable of supporting CRC plasticity and preventing differentiation. We discovered that selective inhibition of histone deacetylases HDAC1/2 promotes differentiation and reduces tumor growth across human and mouse CRC models. This effect was not a consequence of cytotoxicity but rather epigenetic reprogramming. Through mass spectrometry-based histone modification profiling, we identified H3K27ac and H3K9ac as key epigenetic marks enriched at HDAC1/2-bound loci. These marks correlated with enhanced chromatin accessibility and the upregulation of differentiation-associated genes. Degradation of EP300, a major acetyltransferase, reversed the phenotype, confirming the functional relevance of histone acetylation in mediating this differentiation response. Furthermore, a focused CRISPR screen uncovered DAPK3 as a previously unrecognized downstream effector, providing a mechanistic foothold for future biomarker development. Our findings introduce a paradigm shift in CRC therapy: rather than targeting plastic tumor cells for elimination, we aim to force them into a differentiated, less adaptive state, making them more vulnerable to conventional and targeted treatments. This reprogramming effect is trackable via specific histone acetylation patterns, which we propose as candidate predictive biomarkers of response. Based on these insights, we hypothesize that HDAC1/2 inhibitors, when combined with KRAS or EZH2 inhibitors and/or chemotherapy, could yield synergistic therapeutic benefit. We are actively evaluating these combinations in preclinical models. In sum, this work positions HDAC1/2 inhibition as a precision epigenetic strategy to suppress CRC plasticity through enforced differentiation. By integrating multiomic profiling, functional genomics, and novel reporter technologies, we offer a blueprint for targeting the epigenetic underpinnings of therapy resistance in gastrointestinal malignancies, one that prioritizes reprogramming over destruction.



 Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, Sandor Spisak, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James Cleary, Jun Qi, Nilay Sethi. Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1992691ab8c82d8cb17a698db10f779b389e986a" target='_blank'>
              Abstract C031: Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Single-cell molecular profiling has emerged as a cutting-edge technology in biological research, playing a pivotal role in elucidating fundamental life processes and cellular heterogeneity. However, traditional methods generally provide end point measurements and require cell disruption, posing challenges for long-term monitoring of living cell events, e.g., cancer therapy. Tumor Treating Fields (TTFields), a noninvasive therapeutic modality, exert inhibitory effects on tumor growth and invasion by applying low-intensity intermediate-frequency alternating electric fields to tumor regions, whose molecular mechanism on cancer physiology remains poorly understood. In this study, we propose a multifunctional microchip capable of continuous in situ monitoring of molecular signatures in living single cells. The platform integrates spatial cell positioning, single-cellular multimolecule tracking with surface-enhanced Raman scattering (SERS), and TTFields treatment functionalities on a microchip. Using multiplexed analysis of liver cancer cells treated with TTFields and chemotherapy drugs, we implement spatial-temporal multiomics profiling for cancer therapy. Temporal tracking of critical molecular events associated with glycolysis, genetic stability, and membrane integrity reveals significant differences between the groups with and without TTFields treatment. Causal network analysis of molecular profiles uncovers the underlying mechanisms through which TTFields modulate oncogenic pathways. Our work demonstrates the effectiveness of the proposed microchip platform for real-time monitoring of living cell molecular dynamics, offering a powerful tool for precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889ada64ff2d00892ebd5cde7eb385bb8b31694b" target='_blank'>
              Spatial-Temporal Multiomics Profiling of Living Single Cells on Chip for Cancer Therapy.
              </a>
            </td>
          <td>
            Kunru Yu, Sheng-Zong Chen, Rong Zhu
          </td>
          <td>2025-10-12</td>
          <td>ACS sensors</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86ecdda13a72d7b290cf8fe9dd99c8ce634923c0" target='_blank'>
              Galectin-9-driven immunosuppressive macrophage population shapes the lymph node metastatic microenvironment.
              </a>
            </td>
          <td>
            Yao Zhou, Xiaowei Liu, Guixiu Xiao, Genpeng Li, Xiujing He, Leyi Tang, Xueyan Wang, Yujia Lin, Jianyong Lei, Jing Jing, Hubing Shi
          </td>
          <td>2025-10-25</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8876e01109cf8f024442bf59e35e9ed246f97f" target='_blank'>
              Characterizing Stage-Specific Cellular Dynamics and Microenvironmental Remodeling in Lung Adenocarcinoma by Single-Cell RNA Sequencing.
              </a>
            </td>
          <td>
            Bomiao Qing, Xiaolan Li, Xiang He, Junyi Wang, Yi Yang, Manling Jiang, Bingbing Yan, Lei Zhang, Anying Xiong, Q. Ran, Guoping Li
          </td>
          <td>2025-11-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Pleomorphic rhabdomyosarcoma is a rare soft-tissue tumor that occupies an uncertain middle ground between rhabdomyosarcoma and undifferentiated pleomorphic sarcoma. With its relative rarity, aggressiveness, and lack of detailed characterization, it presents a challenging task for therapeutic treatment. In this case study, we use single-cell transcriptomics to investigate the heterogeneous landscape of pRMS and the tumor microenvironment. We demonstrate that the tumor populations in pRMS have a clear division into myogenic and non-myogenic clusters, with the non-myogenic clusters having more numerous communication links with the immune populations. All pRMS tumor clusters use the MIF-CD74 pathway to suppress the immune response, while APP, PTN, and CXCL12 signaling are employed predominantly by the non-myogenic tumor clusters. The cytotoxic T cells in pRMS bear markers of exhaustion (LAG3, HAVCR2, EOMES), and the macrophages express myeloid checkpoint-related genes (SIGLEC1, SIRPA, CSF1R, HAVCR2). This transcriptomic data suggests that targeting MIF and APP signaling in pRMS may have therapeutic potential; however, studies on multiple-patient cohorts, protein verification, and in vitro and in vivo validation are still needed for clinical actionability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/247589e7166c26c19a2c0ff799f9b8b5c5af60e4" target='_blank'>
              Single-Cell Sequencing Reveals Novel Tumor Populations and Their Interplay with the Immune Microenvironment in a Pleomorphic Rhabdomyosarcoma
              </a>
            </td>
          <td>
            Elena E. Kopantseva, A. Ikonnikov, Maxim E. Menyailo, Timur I. Fetisov, A. Korobeynikova, K. Kirsanov, A. Tararykova, B. Bokhyan, Nikolay A. Kozlov, M. Yakubovskaya, E. Denisov
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) exhibits spatial and molecular heterogeneity. Understanding region-specific transcriptional programs within distinct tumor microenvironments is crucial for identifying niche-targeted therapeutic strategies. This study uses the Ivy Glioblastoma Atlas Project (Ivy GAP) to characterize transcriptional variation across histologically annotated tumor regions. We analyzed annotated bulk RNA-seq data from Ivy GAP, covering five tumor regions: cellular tumor (CT, n=111), infiltrating tumor (IT, n=24), leading edge (LE, n=19), microvascular proliferation (MVP, n=50), and perinecrotic zone (PNZ, n=65). Differential gene expression (DGE) analysis was performed using CT as reference. Enrichment analysis was conducted using Gene Ontology and KEGG databases. Prognostic significance of differentially expressed genes (DEG) was assessed using CGGA-GBM dataset for survival analysis. Finally, we performed a pseudo-spatial analysis by plotting normalized Fragments Per Kilobase of transcript per Million mapped fragments (FPKM) expression levels of DEG genes across tumor regions to infer expression dynamics across distinct regions. Each region exhibited distinct transcriptional profiles. IT and LE were enriched for synaptic transmission, neuronal signaling, calcium signaling and excitable membrane potential genes; MVP for angiogenesis and extracellular matrix remodeling; and PNZ for hypoxia-induced metabolic adaptation. Several genes, including CACNG3 (20.5 vs 13.6 m, p=0.03), HCN1 (19.8 vs 13.9 m, p=0.02), SULT4A1(19.2 vs 13.9 m, p=0.03), and SYCE1 (21.7 vs 13.6 m, p=0.0006), were strongly associated with survival in CGGA-GBM dataset. Pseudo-spatial mapping allowed visualization of region-specific gene expression gradients. This integrative analysis reveals spatially distinct molecular landscapes in GBM and identifies prognostic gene markers linked to specific tumoral microenvironments. Our findings define region-specific transcriptional programs in GBM, offering mechanistic insights into tumor microenvironment heterogeneity and nominating candidate genes for future functional validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eee57357cd7906869206ea1e359f6b55d3036c56" target='_blank'>
              BIOM-86. MAPPING REGION-SPECIFIC TRANSCRIPTOMIC PROGRAMS IN GLIOBLASTOMA FROM HISTOLOGICALLY ANNOTATED RNA-SEQ DATA
              </a>
            </td>
          <td>
            Biprojit Nath, S. Bobholz, A. Lowman, S. Duenweg, Benjamin Chao, F. Kyereme, Jennifer Connelly, E. K. Mrachak, Max O. Krucoff, Peter S LaViolette
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51715bb05893c4f00c959588b59a96a3be80d8a0" target='_blank'>
              Longitudinal investigation of prostate tumor spheroid proliferation with dynamic line-field optical coherence tomography
              </a>
            </td>
          <td>
            Stephanie Swanson, Keyu Chen, Elahe Cheraghi, E. Osei, K. Bizheva
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>30</td>
        </tr>

        <tr id="Within the tumor microenvironment (TME) of solid malignancies, tumor-infiltrating antibodies, have been identified as significant modulators of tumor progression and immune response. Tumor-infiltrating antibodies predominantly secreted by plasma cells but also including a small proportion of cancer-derived antibodies. This review aims to elucidate the multifaceted roles of tumor-infiltrating antibodies in the immunology of solid tumors, focusing on their dualistic nature within the TME. This review outlines the mechanisms of B cell activation, antibody class switching, plasma cell differentiation and antibody production, with a focus on their contributions to tumor immunity in solid cancers. Additionally, we discuss the emerging potential of tumor-infiltrating antibodies as both therapeutic targets and diagnostic biomarkers, offering insights that may inform future strategies in cancer treatment. Collectively, antibody functions are shaped by their isotypes: IgG is often associated with improved prognosis in various solid tumors. IgG1 and IgG3 generally mediate anti-tumor responses via antibody-dependent cell-mediated cytotoxicity (ADCC) and antibody-dependent cellular phagocytosis (ADCP), while IgG4 may impair immune effector functions and associate with immune tolerance. IgM, as an early humoral responder, enhances tumor surveillance through complement dependent cytoxicity (CDC), phagocytosis, and apoptosis induction. IgA predominantly promotes tumor progression through immune suppression. IgE exhibits context-dependent pro- and anti-tumor activities, though current evidence is limited, whereas the function of IgD remains largely unknown. Additionally, tumor-derived IgG promotes tumor growth, metastasis, and immune evasion. These findings may open new avenues of research to develop targeted therapies that modulate tumor-infiltrating antibodies, potentially improving the efficacy and safety profiles of current immunotherapeutic approaches. Overall, this review focuses on tumor-infiltrating antibodies in solid tumors and does not encompass hematological malignancies, aiming to provide a more precise understanding of antibody-mediated regulation within the solid tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8616bf287135d120e92e4c09a4ba7ac972e65717" target='_blank'>
              Unraveling the dual roles of tumor-infiltrating antibodies in solid tumors: friend or foe in the tumor microenvironment?
              </a>
            </td>
          <td>
            Lingrui Miao, Run Zhou, Longfei Zhang, Linlu Feng, Yiao Wang, Pan Song, Dong Lv, Tai-Min Shen
          </td>
          <td>2025-11-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The senescence of immune cells, including macrophages, that accompany the initiation and development of tumors has become a novel research hotspot. Recently, studies have reported the molecular characteristics of senescent macrophages (sMACs) in the tumor microenvironment (TME), including cell cycle arrest, senescence-associated secretory phenotype (SASP), and senescence-associated β-galactosidase phenotype (SA-β-gal), and these characteristics not only suggest that sMACs are functionally rich in the TME, but also have the potential to become biomarkers for the identification of sMACs. The in-depth study and analysis of sMACs dialogue and mediating the changes of signaling pathways related to tumor and immune cells will help us to better understand the balance between tumor and aging. Here, we review recent advances in sMACs, including phenotypical molecular characteristics, potential functions and intervention approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c92a1902632f894c6927cd5c2128a9f0efe7401" target='_blank'>
              Senescent macrophages in tumor: phenotypes, roles, and interventions
              </a>
            </td>
          <td>
            Wenhui Shen, Yueyu Huang, Xuping Yang, Yutian Zhang, Yiyi Pan, You Xiao, Jiahui Wang, Changchun Wang, Weimin Mao, An Zhao
          </td>
          <td>2025-10-06</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The immunosuppressive tumor environment (TME) is a major barrier to effective immunotherapy in ovarian cancer (OC), yet the regulatory drivers remain incompletely defined. Here, we applied a multi-omens workflow integrating single-cell RNA sequencing (scRNA-seq), bulk transcriptomics, immune decovolution, Mendelian randomization (MR), expression quantitative trait loci (eQTL) analysis, and functional validation to dissect the heterogeneity and regulatory circuits of tumor-associated macrophages (TAMs) in OC. We identified a selective expansion of intermediate monocytes (IMo) within the OC TME, which promoted immunosuppressive niche formation. MR analysis identified TNFRSF1B (tumor necrosis factor receptor superfamily member 1B, encoding TNFR2) as a causal risk gene, enriched in immunosuppressive myeloid cells across single-cell, spatial, and pan-cancer datasets. High TNFRSF1B expression was associated with M2-like polarization, elevated immune checkpoint signatures, and poor prognosis. Functional knockdown of TNFRSF1B in vitro reduced M2 polarization and cytokine secretion, while in vivo antibody blockade suppressed tumor growth and reversed CD8+ T cell exhaustion. Mechanistically, TNFRSF1B+ TAMs secreted SPP1, which engaged CD44 on CD8+ T cells and induced TIM3-mediated exhaustion. These findings nominate TNFRSF1B as both a prognostic biomarker and a promising immunotherapeutic target in OC and potentially other solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/474e9a188c92710a24fccf2b9a524577ead4f775" target='_blank'>
              TNFRSF1B-DRIVEN MYELOID PLASTICITY CONSTITUTES A CLINICALLY TRACTABLE IMMUNOSUPPRESSIVE AXIS ACROSS HUMAN CANCERS
              </a>
            </td>
          <td>
            Li Ou
          </td>
          <td>2025-11-25</td>
          <td>LIFE: International Journal of Health and Life-Sciences (ISSN 2454-5872)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Glioblastoma progression features extensive dissemination of cancer cells throughout the brain. Limited by neuroimaging resolution and lack of specific markers, malignant cell infiltration is clinically undetectable. This makes complete tumor removal impossible and relapse inevitable. During their exodus from the tumor core, glioblastoma cells are traversing distinct brain territories influencing their phenotypic cell state. To better understand the cellular and molecular dynamics of glioblastoma invasion, we profiled glioblastoma patient avatars and clinical samples using total, single-cell and spatial transcriptomics. In addition to gene expression data, active intracellular pathways of glioblastoma invasion were screened with high throughput activated kinase assays (Pamgene). We identified that human glioblastoma stem cells exhibiting proneural (PN) and neuron-progenitor like (NPC) transcriptomic traits in vitro formed highly infiltrative and connected xenografts in patient avatars. Consistent with this, PN/NPC glioblastoma stem cells cultured on primary hippocampal neurons exhibited higher connectivity and motility compared to mesenchymal-like cells. Leveraging single-cell and spatial transcriptome analytics of avatar and patient samples, we performed cell clustering, annotation, pseudotime and RNA velocity analyses to associate glioblastoma phenotypic cell states to cellular functions. We found that invasive glioblastoma cells re-activated genes and pathways related to neuron progenitor maturation when transitioning from the tumor core to the brain microenvironment. To verify whether invasive PN/NPC glioblastoma cell integration would alter brain functionality, we performed preclinical live cerebral blood volume recordings using noninvasive brain functional ultrasounds (fUS). As glioblastomas progressed, we detected a gradual alteration of brain functional connectivity. Modifications happened in brain areas featuring minimal, yet consistent glioblastoma cell colonization, identified by post-mortem pathology analyses. Altogether, our data shed light on the importance of glioblastoma cell adaptability and brain integration during the metastatic progression. They also provide a framework for detecting invisible clinical features such as tumor invasion, focusing on brain functional connectivity assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6de15ca728123a2849468be8f7f288fcf64e2b6b" target='_blank'>
              ANGI-14. GLIOBLASTOMA PROGRESSION FROM TRANSCRIPTOMIC REGULATION TO BRAIN FUNCTIONAL CONNECTIVITY
              </a>
            </td>
          <td>
            Madeleine Lackman, C. Bergström, Anaïs Virenque, Kato Herman, Aarti Venkatesan, Juli Udayani, Takashi Namba, Analiz Rodriguez, E. Castrén, P. Laakkonen, Sinem Karaman, V. Le Joncour
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1fe8757b9db03b1f80d6119b9cc7fd8b56ac919" target='_blank'>
              Pan-cancer analysis reveals that tumor microbiomes generate specific epitopes through transcriptional reprogramming
              </a>
            </td>
          <td>
            Jinman Fang, Yuxin Zhang, Wei Wang, Xiao Zuo, Tao Chen, Kecheng Li, Huiya Yang, Tianyang Zhu, Yuanliang Zhang, Dandan Wei, Jianfeng Zhang, Jialong Cui, Zhen Tang, Shouwu Zhang, Lirong Wang, Zhilei Li, Dushan Ding, Ruixia Sun, Dandan Zhu, Yin He, Yanfei Zhou, Chenkun Ye, Dandan Chen, Xueting Lang, Long Jiang, Qian Zhao, Lu Zhang, Hongzhi Wang, Yuanwei Zhang
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Objectives FAM49B has been shown to promote proliferation and metastasis of colorectal cancer (CRC) by stabilizing MYC through phosphorylation of NEK9; however, its role in shaping the immune suppressive tumor microenvironment (TME), particularly in macrophage polarization, remains unclear. Methods We applied multi-omics approaches to study CRC by integrating 33 scRNA-seq samples from 16 CRC patients, 2 paired spatial transcriptomics (ST) samples, and bulk RNA data to characterize malignant epithelial cells (High_FAM49B_EP) and tumor-associated macrophages (TAMs). Functional validation of FAM49B was conducted via knockdown experiments and proteomics analysis. Results A High_FAM49B_EP subpopulation was identified in primary tumors (PT) and liver metastases (LM), exhibiting elevated MYC signaling and association with poor prognosis. TAMs showed spatial heterogeneity: M1-like CXCL3+ TAMs predominated in PT, whereas M2-like SPP1+ TAMs were enriched in LM. CellChat analysis revealed that High_FAM49B_EP activated macrophage polarization through the MDK–NCL signaling axis. Pseudotime trajectory analysis confirmed differentiation from CXCL3+ to SPP1+ TAMs driven by upregulation of NCL. Spatial mapping showed co-localization of MDK+ epithelial cells with NCL+ TAMs in the immunosuppressive microenvironment. FAM49B knockdown significantly inhibited MDK expression and disrupted ECM–receptor interactions. Conclusions FAM49B promotes immunosuppressive TME formation by mediating TAM polarization via the MDK–NCL axis, suggesting the FAM49B–MDK–NCL pathway as a potential therapeutic target for CRC metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8878570ce8b97d296e4580e17e3836990c936b1" target='_blank'>
              Single-cell and spatial transcriptomics integration reveals FAM49B promotes tumor-associated macrophages polarization in colorectal cancer via the MK pathway
              </a>
            </td>
          <td>
            Tianyu Liu, Quchen Ding, Jin Gou, Chen Lu, Xingming Lu, Jiatong Chen, Y. E, Lianhong Li, Chongguo Zhang, Xiaojuan Zhu, Chunzhao Yu, Xiagang Luo
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 The aberrant properties of brain tumor vasculature are increasingly recognized as barriers to efficient delivery and distribution of bloodborne therapies. These aberrant properties emerge from molecular abnormalities that can be identified through comparisons with normal human brain vasculature. However, vascular cells are difficult to isolate and capture, which leads to under-representation in single-cell datasets and challenges identifying reproducible molecular differences among patient populations. To address these challenges, we have developed a computational strategy to isolate transcriptional profiles of vascular cells through integrative gene coexpression analysis of bulk transcriptomes. We have applied this strategy to ~100 gene expression datasets representing >12K normal adult human brain and adult malignant glioma samples and identified robust markers of glioma vasculature that are not observed in normal adult human brain vasculature. These markers include ENPEP (aminopeptidase A), which encodes a cell-surface protein that is implicated in blood pressure regulation and angiogenesis. We confirmed histologically that ENPEP is profoundly upregulated in hyperplastic blood vessels of glioblastomas (GBMs) and to a lesser extent IDH-mut gliomas but is not detectable in vascular cells of non-neoplastic adult human brain. Furthermore, over 6503 normal adult human brain samples profiled with diverse technology platforms, we found that ENPEP is expressed in the 8th percentile genome-wide. To characterize the functional significance of ENPEP in GBM, we acquired firibastat, an oral prodrug that crosses the blood-brain barrier and selectively inhibits ENPEP. We used the GBM14 FL intracerebral tumor xenograft model to investigate the anti-tumor efficacy of firibastat in vivo. Log-rank survival analysis revealed that firibastat delivered daily by oral gavage (500mg/kg) significantly extended survival (P = 0.04 vs. vehicle). These results support further study of ENPEP and other novel glioma vascular markers for potential applications as biomarkers and therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67db9014a7c696a41912c8e5d1a1d46c4a0b8bd9" target='_blank'>
              TMIC-54. Identifying and exploiting molecular abnormalities of glioma vasculature
              </a>
            </td>
          <td>
            Patrick Schupp, Rebecca Eliscu, Nirupama Korada, Fabrice Balavoine, S. Beccari, T. Ozawa, D. Raleigh, Joanna Phillips, Michael Oldham
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The generation of functionally mature hepatocytes from human induced pluripotent stem cells (iPSC) has the potential to replace primary human hepatocytes (PHH) as the gold standard in vitro model for drug screening, studies of hepatotoxicity as well as liver disease, and is considered a gateway technology to future cell therapy applications. However, we recently reported that current protocols for deriving hepatocyte-like cells (HLC) from iPSC fail to restrict differentiation to the hepatic lineage. Instead, single cell transcriptomics and protein expression analysis uncovered that current methods induce hybrid differentiation, leading to the establishment of abundant hepatic and intestinal gene expression signatures within individual HLC, thereby compromising hepatocyte functionality and phenotype. Differentiation of iPSC to HLC was performed and followed by analysis of transcriptome changes during iPSC to HLC differentiation in comparison to PHH. Differentiation pattern clustering (DiPaC) of differentially expressed genes (DEG) and downstream bioinformatic analysis identified the gene regulatory networks (GRN) involved in establishing hybrid differentiation signatures in HLC, indicating a major influence of caudal-domain type 2 (CDX2) in HLC-IEC hybrid differentiation. CRISPR Cas9-based genetic engineering was used to generate CDX2−/− iPSC, followed by HLC differentiation, transcriptomics and intestinal and hepatic protein expression analysis. The observed phenotypic changes in CDX2−/− HLC were verified by functional analysis of bile canaliculus transport kinetics as well as intestinal enzyme activity and compared to PHH. Transcriptome analysis of WT iPSC to HLC differentiation in comparison to PHH confirmed the establishment of intestinal differentiation gene expression signatures in HLC and indicated a major role of CDX2 in erroneous lineage decision making of HLC. Absence of CDX2 during iPSC to HLC differentiation promoted hepatic specification through the induction of HHEX and PROX1, while drastically reducing intestinal differentiation signatures. CDX2−/− HLC displayed phenotypic maturation of bile canaliculi, including loss of intestine-associated membrane proteins (e.g. IBAT, SI, LCT), increase in hepatic transporters (e.g. BSEP) and size reduction, leading to closer functional and architectural resemblance to bile canaliculi formed by PHH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4262aa3521d3e2a2d0d27e54a288918ba0f78a31" target='_blank'>
              Elimination of CDX2 restricts intestinal hybrid differentiation signatures in stem cell-derived hepatocyte-like cells
              </a>
            </td>
          <td>
            P. Nell, A. Thomitzek, D. Feuerborn, A. Scholtz-Illigens, Sarah M Seidel, K. Derksen, Lara Maria Chilinski, Kathrin Kattler-Lackes, Nils Blüthgen, M. Morkel, Jörn Walter, Karolina Edlund, Jörg Rahnenführer, J. Hengstler
          </td>
          <td>2025-10-07</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Macrophages in the tumor microenvironment (TME) can constitute up to 50% of tumor mass and play a critical role in cancer cell proliferation, invasion, and metastasis. While their contribution to extracellular matrix (ECM) degradation through matrix metalloproteinases (MMPs) has been explored, the role of other macrophage-derived factors in ECM remodeling and their impacts beyond degradation remain poorly understood. Here, we describe the development of a 3D collagen-based tumor spheroid model to investigate the impact of peripheral blood mononuclear cell (PBMC)-derived macrophages on cancer cell-ECM and cancer cell-macrophage interactions within the TME. We observed that cancer cells stimulated PBMC-derived macrophages into an M2-like phenotype and that tumor spheroid conditioned macrophages (TSCMs) shifted cancer cell populations toward phenotypes with greater invasion distances and reduced circularity, indicative of increased malignancy. Such observations can be explained by macrophage-mediated ECM remodeling. Specifically, we demonstrate that TSCMs secreted a variety of soluble factors that are known to contribute to ECM remodeling, including ECM degradation and fiber realignment. These processes collectively create a tumor-favoring environment by loosening the collagen matrix and aligning fibers that serve as invasion tracks for migrating tumor cells that facilitate cancer cell migration and invasion. This model provides a robust platform to study the interactions between cellular and non-cellular components in the TME and to identify the molecular mechanisms underlying cancer progression. These insights may aid in the development of novel therapeutic strategies targeting macrophage-mediated processes in cancer. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8ba56db9b76901c5ee694ce68b18c1a460709a" target='_blank'>
              Collagen-Based Tumor Spheroid Model for Investigating Tumor-Macrophage Interactions through Extracellular Matrix Remodeling
              </a>
            </td>
          <td>
            Luna Zhang, Sanab Abdi, Hannah M. Szafraniec, Paolo P. Provenzano, K. Schwertfeger, David K. Wood
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Neutrophils, the most abundant immune cells, have multifaceted functions in cancers. While most previous work reported pro-tumoral neutrophils, emerging data have shown anti-tumoral phenotypes directly killing tumor cells and activating cytotoxic T-cell immunity in different immunotherapy contexts. Therefore, there is a need to define anti-tumoral neutrophil markers and potential therapeutic targets. Here, we interrogate anti-tumoral phenotypes using bioinformatics analysis of ∼95,000 neutrophils from scRNA-Seq of 8 mouse models with different treatments, including Immune Checkpoint Blockade (ICB) and CD40 agonist. In addition, we generated multi-omics single-cells of neutrophils from immunological cold models using a combined neutrophil activation therapy. We found consistent markers of interferon-stimulated neutrophil phenotypes in multiple cancer types and treatments. The compendium of pan-cancer pan-treatment data helps identify potential markers of neutrophils with tumor-killing phenotypes found in microbial stimulation and tumor gene knock-out settings. Furthermore, our new scRNA-Seq data determined a subset associated with TNF and migration phenotypes. Lastly, cross-species analysis with data across 17 cancer types, including those treated with ICB, determined similar signatures in humans. Our results report the emerging phenotypes in anti-tumoral neutrophils revealed by single-cell omics, providing a foundation for future mechanistic and translational studies.



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f8821c014fd4074796e50f57262ffad11085ece" target='_blank'>
              Identification of anti-tumoral neutrophil states from pan-cancer single-cell transcriptome analysis of immunotherapies in preclinical mouse models and humans 3571
              </a>
            </td>
          <td>
            Huy Q. Dinh, Anqi Gao, Hien Pham
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Prostate cancer (PC) remains among the leading causes of cancer-related deaths with a better understanding of the mechanisms underlying its pathogenesis urgently needed. The synthesis of complement component C3 by tumor-infiltrating immune cells in the tumor microenvironment (TME) facilitates tumor progression via immunosuppression and angiogenesis support, denoting TME C3 as a therapeutic target. However, the role of C3 expressed by tumor cells themselves during oncogenesis remain inadequately characterized. Using data from multiple clinical PC cohorts we found that, surprisingly, C3 transcript and protein levels were significantly reduced in malignantly transformed regions when compared to benign regions of human PC tissues. C3 silencing in human benign primary prostate epithelial cells (PECs) caused uncontrolled proliferation while provision of intracellular C3a to the aggressive PC cell line DU145 reduced tumor burden in a xenograft mouse model, confirming the unexpected tumor-suppressing role of C3. Mechanistically, steady-state, endogenous C3 expression, C3a generation, and -receptor engagement in PECs maintains normal cell turnover through restraint of a PI3Kd-dependent MYC program upstream of ribosomal biogenesis. Collectively, these findings ascribe a non-traditional, tumor-suppressor function to cell intrinsic C3 and advocate for dissecting the context-specific roles of complement in malignancies to better inform on complement-targeted cancer therapies.



 NIH: ZIA/hl006223, ZIA/DK075149, R35GM138283 Swiss National Science Foundation: PCEFP3_194618



 Immune Mechanisms of Human Disease (HUM)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020832dd81a765c03d9c1ee584f3bf5577bab7a2" target='_blank'>
              Cell-autonomous complement C3 activity in prostate epithelial cells suppresses malignant transformation through tonic restrain of the MYC-tumorigenesis program 3135
              </a>
            </td>
          <td>
            Natalia Kunz, Jack A. Bibby, Nicolas S. Merle, Sumeyra Kartal, Hidekazu Yamamoto, Luopin Wang, Md Taj, Jianping Ma, Martin Kolev, Jonas Lotscher, Tamara Kellerhals, Maria L Balmer, Ashalia Chandra, Christoph Hess, Behdad Afzali, Prokar Dasgupta, Erin E. West, A. Sowalsky, Claudia Kemper
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="BACKGROUND
Changes in the cyst microenvironment in Polycystic Kidney Disease (PKD) may drive progressive cyst formation. Bulk- and single-cell RNA sequencing have advanced our understanding of altered signaling; however, the lack of spatial information has limited our insights into local gene expression and cellular communication near cysts.


METHODS
We used wild-type and Pkd1-deficient mouse kidneys to generate 10x Genomics Visium Spatial Gene Expression datasets. Utilizing our single-cell mouse kidney atlas and single cell sequencing data for spot deconvolution, we enhanced resolution and estimated enriched cell types. We analyzed spatial gene expression patterns and used a cyst-centered analysis to identify cyst-associated gene signature. Cell communication near cysts was investigated, identifying key ligand-receptors. Prioritized key factors were validated in tissues.


RESULTS
We observed enrichment of fibroblasts, injury-repair-related cell types, and diverse immune populations in PKD. Injury-repair-related cells were exclusively observed in PKD, predominantly localized within immune cell-dense regions near cysts. These cells collectively contributed to the altered gene expression profile in PKD, including cyst-associated signature genes related to inflammatory processes. Analysis of cellular communication in less-inflamed regions around cysts revealed the involvement of multiple cell types. Key ligand-receptor interactions were associated with cytokine signaling, fibrosis, cellular development, and repair. These included Angpt2, C3, Csf1, Cxcl12, Il34, Gas6, Il16, Mdk, Mif, Ptn, Sfrp2, Spp1, Sdc1, Tnc, Tnfsf12, Wnt5a. In addition, ECM proteins implicated in immune response, ECM remodeling, cell adhesion, and cell signaling were identified, such as Adam9, Adam10, Col1a1, Col3a1, Col4a2, Lamb2, Lamc1, Efnb1, Efnb2, Thbs1, Thbs2, and Vcam1. IHC confirmed expression of Syndecan-1-Collagen IV, Midkine-Integrin β1, CSF-1, Pleiotrophin, and Tenascin-C in cystic kidneys.


CONCLUSIONS
Spatial transcriptomics in PKD revealed enrichment of (myo)fibroblasts, immune, and injury-repair-related cells near cysts, creating a (pro)inflammatory and (pro)fibrotic niche. Key ligand-receptor and ECM interactions were identified and validated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecdcf5e98138976e4b0b6f2eafc40e1b84148be0" target='_blank'>
              Spatial Transcriptomics Reveals Injured Cells, Signature Genes, and Communication Patterns in the Cyst Microenvironment of Polycystic Kidney Disease.
              </a>
            </td>
          <td>
            Sevtap A Yasinoglu, Claudio Novella-Rausell, L. Wisse, K. Dijkstra, A. Mahfouz, Hans J. Baelde, DJM. Peters
          </td>
          <td>2025-10-10</td>
          <td>Journal of the American Society of Nephrology : JASN</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 The relevance of the spatial context of the increasing number of potential biomarkers within the tumor microenvironment (TME) has resulted in a need for reliable and accurate multiplex immunohistochemical (mIHC) assays. The SignalStar mIHC assay from Cell Signaling Technology is a novel approach that was utilized to amplify and visualize up to 8 targets within the same formalin fixed, paraffin-embedded (FFPE) tissue section without the need for cycling of antibodies or fluorophore deposition. Here, we compare each target in an 8-plex panel consisting of T-cell exhaustion markers to the canonical chromogenic IHC assay. We also demonstrate the value of amplification when detecting lower abundance functional markers by comparing the percent positivity of SignalStar amplified targets to that achieved with direct and indirect immunofluorescence. Furthermore, we highlight the flexibility of this assay via the usage of a SignalStar secondary antibody, which allows for the incorporation of any IHC-validated rabbit antibody into a panel. Lastly, we also compare the SignalStar assay to other mIHC assays with respect to signal intensity and percent cell positivity. Ultimately, our data demonstrates that the SignalStar mIHC assay is a powerful tool that accurately characterizes the complex TME, allowing researchers to better understand immunotherapeutic approaches and other treatment strategies.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5388ac320b15789afa2366c0bfc8f1b720971bda" target='_blank'>
              Spatial analysis of the tumor microenvironment can be achieved via the novel SignalStarTM multiplex immunohistochemistry assay 3651
              </a>
            </td>
          <td>
            J. Ziello, Derek Papalegis, Lily Vu, Sasha Tkachev, Gabriella Spang, Sarah R Klein
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Neoadjuvant treatment has demonstrated clinical benefits in advanced gallbladder cancer (GBC). However, the overall response rate remains suboptimal, and the underlying mechanisms driving treatment efficacy are not fully understood.


MATERIALS AND METHODS
This study aimed to evaluate the therapeutic effects of neoadjuvant chemo-immunotherapy (NAT) combining gemcitabine, nab-paclitaxel, and anti-PD-1 immunotherapy in advanced GBC, and to investigate the associated tumor microenvironment (TME) alterations. Single-cell RNA sequencing and multiplex immunohistochemistry were utilized to analyze the cellular composition of the TME in patients who successfully underwent downstaging.


RESULTS
NAT significantly reshaped the tumor-immune landscape, characterized by an expansion of follicular helper T (Tfh) cells and the formation of tertiary lymphoid structures (TLSs) in 50% of treated tumors. Inflammatory cancer-associated fibroblasts (CAFs) increased and exhibited upregulation of CCL19 and CXCL12, potentially promoting Tfh cell recruitment and TLS formation. Additionally, NAT led to an expansion of GZMB+ cytotoxic CD8+ T cells with an exhausted phenotype in GBC, but not in adjacent normal tissues. The treatment also increased the number and effector functions of natural killer cells while reducing tumor-promoting macrophages and angiogenesis-related CAFs. Furthermore, NAT decreased the cancer stem cell-like subpopulation while increasing a cancer cell subset with enhanced antigen-presenting capacity.


CONCLUSION
This study suggested the potential efficacy of NAT in advanced GBC and revealed alterations in the TME following treatment. The findings provided insights into the mechanisms underlying NAT responses and offered valuable directions for optimizing therapeutic strategies in GBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/363d50ed22da82003f2b54ca7ab634be84e764af" target='_blank'>
              Single-cell profiling unveils the immuno-favorable tumor microenvironment remodeling after successful neoadjuvant therapy for advanced gallbladder cancer.
              </a>
            </td>
          <td>
            Qiuyi Tang, Yi-Jian Zhang, Ziyou Wu, Tian-Yu Zhang, Xiao-ling Song, Yi-di Zhu, Zi-Yi Yang, Cheng Zhao, Jing-wei Zhao, Hua-kai Wang, Yue Yang, Yu Zhang, Wen-Ting Dai, Lin Jiang, Yang-Yang Zhai, Chuan-Xin Huang, Xiang-song Wu, Wei Gong
          </td>
          <td>2025-11-06</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Analyses of patient-derived cell lines have greatly enhanced discovery of molecular biomarkers and therapeutic targets. However, characterization of cellular morphological properties is limited. We studied cell morphologies of human pancreatic adenocarcinoma (PDAC) cell lines and their associations with drug sensitivity, gene expression, and functional properties. By integrating live cell and spatial messenger RNA imaging, we identified KRAS inhibitor–induced morphological changes specific for drug-resistant cells that correlated with gene expression changes. We then categorized a large panel of patient-derived PDAC cell lines into morphological and organizational subtypes and found differences in gene expression, therapeutic targeting potential, and metastatic proclivity. Patterns of cancer cell organization in human PDAC tissues stratified distinct gene expression signatures with clinical significance. In summary, we highlight the potential of cell morphological information in rapid, cost-effective assays to aid precision oncology efforts leveraging patient-derived in vitro models and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb047a3ab41c4e28c6535b371221906a6c927ab" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Background/Objectives Triple-negative breast cancer (TNBC) tumors are typically heterogeneous, predominantly epithelial tissues with discrete patches of mesenchymal-like TNBC cells, that differ in their invasiveness, proliferation potential and response to treatment. However, the contribution of mesenchymal-like and epithelial TNBC cells in the persistence of chemotherapy resistant disease remains poorly understood. Methods Mesenchymal-like and epithelial TNBC cell types were detected by multiplex fluorescent immunohistochemistry using antibodies against vimentin, Ki67, and Annexin A6 (AnxA6). Chemotherapy drug resistant mesenchymal-like and epithelial TNBC cell populations were established by pulse exposure and stepwise dose escalation and validated by 3D cultures and unbiased antibody arrays. Results Analysis of the response of stage IV TNBC tumors treated with six common chemotherapy regimens resulted in 36% complete response and 64% partial response with residual tumor sizes ranging from 0.5 to 37.0 mm. Treatment of TNBC cells with chemotherapy agents led to distinct resistance signatures including downregulation of survivin and upregulation of M-CSF and CXCL8/IL-8 in model mesenchymal-like, and upregulation of CCL2/MCP-1, CTSS and DKK-1 in model epithelial TNBC cells. The inhibitory phosphorylation of GSK-3β (p-S9) increased in paclitaxel resistant epithelial cells but decreased in resistant mesenchymal-like TNBC cells. Finally, chemotherapy resistance also activated p90 ribosomal S6 kinases (RSK1/2) in both cell types, while activation of mitogen- and stress-activated kinases (MSK1/2) was only observed in chemotherapy resistant epithelial TNBC cells. Conclusions These data reveal that chemotherapy resistance of epithelial and mesenchymal-like TNBC cellular subsets upregulated distinct profiles of proinflammatory and immune cell chemotactic cytokines and modulated the activities of GSK-3β, p90 RSK1/2 and the related MSK1/2. Targeting these factors and/or the associated signaling pathways may help overcome chemotherapy resistance in TNBC. Graphical Abstract TNBC tumors are typically heterogeneous, predominantly epithelial tissues with discrete patches of mesenchymal-like TNBC cells, that differ in their invasiveness, proliferation potential and response to treatment. This study reveals that Chemotherapy resistance in epithelial and mesenchymal-like TNBC cells is maintained by distinct signatures of proinflammatory and immune cell chemotactic cytokines, and activities of GSK-3β and p90 ribosomal S6 kinases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cfe5f04ad4ee0a30d00ab780556d22fdbb86b41" target='_blank'>
              Contribution of Mesenchymal-like and Epithelial Cellular Subsets to Chemotherapy Resistance in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            N. Vuong, O. Korolkova, Michael Izban, Nobelle I. Sakwe, Antonisha R. McIntosh, Destiny Ball, Perrin Black, Alayjha D Edwards, Billy R. Ballrad, S. Adunyah, A. Sakwe
          </td>
          <td>2025-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253b52aabc9f910e8e84f8a906624aeebc79b3e8" target='_blank'>
              Single-cell transcriptomic profiling reveals liver fibrosis in colorectal cancer liver metastasis.
              </a>
            </td>
          <td>
            Yiqiao Deng, Chengyao Guo, Xiaomeng Liu, Xin Li, Jianmei Liu, Wenjie Liu, Jinghua Chen, Zhen Huang, Ye‐fan Zhang, Xinyu Bi, Jian‐jun Zhao, Jianguo Zhou, Zhi-yu Li, Hongliang Wu, Baocai Xing, Qichen Chen, Hong Zhao
          </td>
          <td>2025-11-14</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are increasingly recognized as critical contributors to the limited efficacy of immunotherapy, particularly through the establishment of immune-excluded tumor microenvironments. However, the functional heterogeneity and spatial ecological roles of CAFs in melanoma remain poorly characterized. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq data to systematically dissect CAF molecular programs in melanoma using weighted gene co-expression network analysis (WGCNA). Spatial distribution, CellChat, and ligand–receptor interaction analyses were applied to construct CAF-centered immune exclusion networks. A functional scoring model (riskScore) based on the CAF-M1 module was developed to evaluate clinical relevance. CAF was spatially co-localized with endothelial cells, and engaged in strong crosstalk with multiple immune cell types via pathways such as COLLAGEN–integrin and THBS1–CD47. The derived riskScore model demonstrated robust prognostic value across TCGA-SKCM cohort. Through spatial aggregation and multifaceted immune signaling, CAF constructs a multilayered immune-exclusion network in melanoma, linking stromal remodeling to immune evasion. These findings offer novel insights into CAF-driven immune resistance and may inform future stratified immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca83751e1c8ec5203b2f4b9c36b4cadfbfd1df19" target='_blank'>
              Exploration of the roles of CAFs in melanoma based on single-cell transcriptomics and spatial transcriptomics
              </a>
            </td>
          <td>
            Sheng Hong, Yuhan Zhao, Anni Hu, Yizhong Zhang, Yuming Li, Yan Rong, Zukai Li, Wenzhuo Su, Lude Zhu
          </td>
          <td>2025-11-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Soft tissue sarcomas (STS) are characterized by abundant extracellular matrix (ECM) deposition, yet the functional contribution of specific ECM components remains poorly understood. Here, we identify Periostin (POSTN), a matricellular protein, as a regulator of sarcoma progression and the tumor immune microenvironment. Analysis of human sarcoma datasets revealed that high POSTN expression correlates with poor prognosis and elevated expression of ECM-related and myeloid cell-associated genes. In murine genetic models of sarcoma, tumors expressing high levels of Postn displayed enhanced expression of ECM genes and monocyte-recruiting cytokines. Functional silencing of Postn in vivo reduced tumor growth without altering tumor cell proliferation or intrinsic signaling pathways, suggesting a non-cell-autonomous mechanism. Instead, Postn-deficient tumors showed increased infiltration of CD4⁺ and CD8⁺ T cells and reduced proportions of immunosuppressive myeloid cells, including tumor-associated macrophages (TAMs). Single-cell RNA sequencing revealed that Postn silencing reprograms the myeloid compartment, increasing interferon-responsive and pro-inflammatory subsets. Mechanistically, recombinant POSTN promoted monocyte migration and maturation into macrophages in vitro, supporting its role as a chemoattractant and differentiation cue. Therapeutic neutralization of POSTN partially recapitulated the immunologic remodeling but was insufficient to reduce tumor burden as monotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcb661ad1542d712f7aa5599bc104c08fe2694e" target='_blank'>
              Periostin promotes sarcoma growth by promoting tumor-associated macrophage migration and differentiation.
              </a>
            </td>
          <td>
            Jinfen Xiao, Kristin Ishaya, E. Ko, Annaliese Fowler, Marina T Broz, Jlenia Guarnerio
          </td>
          <td>2025-11-20</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The clonal evolution model provides a framework for understanding the evolution of cancer cells. According to this model, cancer cells accumulate genetic mutations over time, and these mutations are passed down to their descendants, leading to genetic diversity within the tumor. Some of these mutations confer selective advantages, causing certain lineages of cancer cells (clones) to dominate and expand. However, this model is rooted in certain conceptual assumptions, which we propose to revisit by considering the potential involvement of retrotransposons in cancer initiation and progression. In recent years, it has become evident that transposable elements, particularly retrotransposons, play a significant role in driving cancer transformation and progression. We first review how current knowledge about retrotransposon activity aligns with the clonal evolution model by highlighting its ability to modulate cancer cell fitness. We then take a forward‐looking perspective to explore additional ways retrotransposons may also influence clonal dynamics beyond the current model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb2629152d1c164e115163464d0e7599e1333a7" target='_blank'>
              Revisiting Clonal Evolution Through the Light of Retrotransposons
              </a>
            </td>
          <td>
            Anaïs Lamoureux, Emilie Elvira-Matelot, Françoise Porteu, L. Laplane
          </td>
          <td>2025-10-09</td>
          <td>Bioessays</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 As cancer therapies improve and patients live longer, the incidence of brain metastases (BrMets) continues to rise. Among women with metastatic breast cancer, 10-15% develop BrMets, with rates as high as 30% in HER2+ and 50% in triple-negative. BrMets, represent highly evolved, therapy-resistant tumors with a median survival of 10 months and accompanied by severe neurologic decline. BrMets often emerge years after initial diagnosis, suggesting that metastatic cells must acquire brain-specific traits to survive. This process may involve transcriptional convergence—where tumor cells adopt brain-like programs, and brain-resident cells exhibit tumor-associated changes. Despite its clinical relevance, the molecular basis of this convergence remains poorly defined. Decoding, and targeting, these shared gene programs could reveal new therapeutic vulnerabilities.



 We performed spatial transcriptomic profiling on 235 patient tissue cores (BrMets, adjacent normal brain, primary breast tumors, and non-cancer brain). Using NanoString GeoMx, DSP, we measured the expression of 18,677 RNAs across 450 spatially-defined regions enriched for tumor, immune, and brain-resident cells. To interrogate transcriptional convergence, we developed the Equivalent Expression Index, a novel statistical tool to detect genes with biologically meaningful expression similarity across distinct cell populations.



 Our integrated analysis revealed two prognostic gene signatures. The Metastasis-Induced Brain Shared (MIBS-9) signature, comprising brain-like genes upregulated in tumor cells, was associated with longer patient survival. Conversely, the Adjacent brain-Resident Cancer Shared (ARCS-81) signature, defined by tumor-like gene expression in brain-resident cells, correlated with shorter survival. These signatures were externally validated and enriched for pathways related to neuronal remodeling, immune evasion, cellular communication, and cancer/metastasis.



 By coupling spatial transcriptomics with a novel equivalence-based algorithm, we uncover distinct, clinically relevant gene programs that mediate bidirectional adaptations between tumor and brain-resident cells. These findings illuminate key mechanisms of metastatic outgrowth and offer new targets for disrupting the breast cancer BrMet microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4c6b99afe4369f11ea5994698b0e8b275a5df02" target='_blank'>
              TMIC-89. Spatial transcriptomics reveals tumor–brain expression convergence and identifies prognostic gene signatures in breast cancer brain metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, Christine Yeh, Pablo Nunez Perez, Giuseppe Barisano, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia Plevritis, M. Gephart
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d80db710da61450f8b7912e87ba31ff737151f1" target='_blank'>
              The shared selection landscape of dog and human cancers
              </a>
            </td>
          <td>
            D. Genereux, Kate Megquier, Ross Swofford, Jason Turner-Maier, Michelle E. White, Vista Sohrab, Christopher Husted, Heather L. Gardner, Corrie Painter, Cheryl London, Elinor K. Karlsson
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Brain metastases (BrMs) represent a significant clinical challenge with increasing incidence and poor prognosis. Here, we present a comprehensive proteogenomic characterization of pan-cancer BrMs (n=923) with patient-matched primary tumors (n=82) and glioblastomas (n=20), representing the largest cohort to date. Through consensus clustering of transcriptomic data, we identified and validated four distinct molecular subtypes (CMS1-4) with unique biological features. Integrated analysis of the transcriptomic and proteomic revealed that CMS1 was activated with neural system-associated pathways, CMS2 was characterized by extracellular matrix remodeling and increased immune cell infiltration, CMS3 exhibited a more epithelial phenotype along with metabolic reprogramming and CMS4 showed upregulation of cell cycle-related genes with elevated stemness scores. These molecular features were further validated through immunohistochemistry and multiplex immunofluorescence. Genomic analysis revealed subtype-specific mutational signatures and copy number alterations, with CMS1 exhibiting lower tumor purity and CMS4 displaying the highest chromosomal instability. Survival analysis indicates that CMS4 was associated with the worst prognosis, while CMS1 and CMS2 demonstrated more favorable responsiveness to radiotherapy and immunotherapy, respectively. Using patient-derived organoid models, we identify subtype-specific therapeutic vulnerabilities, notably metabolic inhibition sensitivity in CMS3 and CDK4/6 inhibition response in CMS4. Single-nucleus RNA sequencing further elucidates the complex cellular ecosystems and intercellular communication patterns within each subtype. This study creates a multi-omics resource for understanding the metastatic adaptation to the brain microenvironment and exploring personalized treatment strategies in BrMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8325830d999701778469a9b0a790181d9230e305" target='_blank'>
              EPCO-01. PAN-CANCER PROTEOGENOMIC AND SPATIAL CHARACTERIZATION OF 1000 METASTATIC BRAIN TUMORS
              </a>
            </td>
          <td>
            Gao Zhang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is characterized by aggressive biological behavior and poor prognosis. However, TNBC exhibits higher immunogenicity than other breast cancer subtypes, making it more responsive to immunotherapy. Neoadjuvant chemotherapy (NAC) is the standard treatment for early high-risk TNBC; however, reliable biomarkers for predicting NAC response remain elusive. Tumor-infiltrating lymphocytes (TIL) are recognized as predictive markers of NAC response in TNBC, yet discordant cases remain, such as tumors with high TIL levels but poor response. This study aimed to further elucidate the immune environment of TNBC by analyzing TIL, programmed death-ligand 1 (PD-L1) expression, and tumor-stroma ratio using pretreatment biopsy tissue slides from 16 patients with TNBC treated with NAC. Multiplexed immunofluorescence for CD8, FOXP3, CD4, CD20, and CK was employed to investigate immune cell (IC) composition and spatial interactions within the tumor immune microenvironment. Cell to cell distance and comparison of subcellular proportion of IC and were analyzed by treatment response, TIL, and PD-L1 status. Significant differences were found in IC composition and distribution between patients achieving pathologic complete response (pCR) and those with residual disease (non-pCR). The pCR group exhibited significant enrichment of cluster of differentiation CD8 + IC and CD20 + IC in both tumor and stromal regions, suggesting their critical role in mediating an effective NAC response. In contrast, non-pCR cases showed higher proportions of immunosuppressive CD4 + FOXP3 + IC, particularly in the tumor region. High TIL levels were associated with pronounced B-T cell interactions, as evidenced by the significant clustering of CD20 + and CD8 + ICs near tumor cells, highlighting their cooperative role in antitumor immunity. In conclusion, our findings suggest that tumoral CD8 + and CD20 + ICs are pivotal determinants of NAC response in TNBC. The enrichment of CD20 + IC under high-TIL conditions underscores the potential role of B-T cell interactions in shaping immune-mediated chemotherapy responses. These insights provide a foundation for leveraging immune-based biomarkers to stratify patients with TNBC and optimize NAC outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efea310e4b9cebac8f99562fb5c52b2d67c2f334" target='_blank'>
              Spatial analysis of tumor immune microenvironment of TNBC with different neoadjuvant chemotherapy outcomes using multiplex Immunofluorescence
              </a>
            </td>
          <td>
            Inho Park, Su-Jin Shin, H. Go, Jiwon Ko, S. Bae, S. Ahn, Joon Jeong, Jee Hung Kim, Yoon Jin Cha
          </td>
          <td>2025-10-21</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Immune surveillance is believed to eradicate pre-cancerous cells to guard against cancer. However, its mechanisms remain unclear, as pre-cancerous cells are difficult to detect. To gain a glimpse into immune surveillance, our lab developed a genetically engineered mouse model recapitulating human triple negative breast cancer (TNBC), which generates rare, p53-Brca1 mutant cells with unequivocal GFP labeling. Guided by the visualization of premalignant cells, we observed tertiary lymphoid structure (TLS)-like immune aggregates near mutant ducts long before tumor formation. GeoMx-based spatial profiling showed not only activation signatures in T and B cells but also elevated interferon response genes in mutant ducts associated with immune aggregates when compared to those free of immune infiltration. FTY720 treatment at premalignancy, which blocks T and B cell egress from lymph nodes, led to increased mutant cells expansion and shortened tumor latency, demonstrating the restraint of mutant cell growth by the immune system. To clarify the roles of specific immune cells, we plan to deplete CD4 T, CD8 T, and B cells at premalignancy. We will also conduct longitudinal studies on T and B cells along the tumor evolution process from premalignancy to malignancy to understand how immune surveillance eventually falters. Taken together, our studies are poised to elucidate key mechanisms of immune surveillance in the premalignant stage of TNBC.



 Supported by NIH R01-CA256199; Basser Center for BRCA; the Pinn Scholarship (UVA); UVA Cancer Center Spatial Biology Funding; UVA Cancer Center Training Grant to X.Z.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ee7201e884081fced81f5a6fa79a355f932afc" target='_blank'>
              First glimpse of immune surveillance during premalignant progression of triple negative breast cancer 3378
              </a>
            </td>
          <td>
            Xian Zhou, Jianhao Zeng, Elaina Kiral Ball, Alexys Riddick, Pat Pramoonjago, Victor Engelhard, Hui Zong
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Phenotypic plasticity plays a pivotal role in cancer, enabling tumor cells to adapt to environmental pressures and evade therapeutic interventions by transitioning between distinct cellular states. However, the contribution of phenotypic plasticity to adaptive drug resistance in glioblastoma (GBM), one of the most lethal of all cancers, remains poorly understood. In this study, we identify that GBM tumor-initiating cells resembling normal radial glia (RG), which occupy the apex of normal neurodevelopment, are driven by aberrant epidermal growth factor receptor (EGFR) signaling. Using a suite of patient-derived GBM models, we demonstrate through global proteomics and single-cell RNA sequencing that pharmacological inhibition of EGFR triggers a lineage transition toward neuronal and oligodendrocyte progenitor (OPC)-like states. This shift is accompanied by activation of oncogenic RAS-MAPK signaling – despite robust and durable inhibition of EGFR activation – and is further modulated by brain microenvironmental cues, including synaptic and calcium-mediated signaling programs. Dual inhibition of EGFR and RAS-MAPK with novel, tumor-selective small molecules blocks these phenotypic transitions and enhances GBM cell death in EGFR-mutated GBM models. To determine the subpopulation dynamics of RAS-MAPK signaling within GBM neurodevelopmental lineages, we develop DENALI (Dual-Expression Nuclear reporter of ERK Activity and Lineage Identity) – a novel, high-complexity barcoded lentiviral vector and integrative fluorescence reporter system. Using DENALI, we investigate the clonal mechanisms driving lineage plasticity in GBM following oncogenic EGFR inhibition and couple adaptive RAS signaling programs to the emergence of neuronal and OPC-like states under EGFRi therapy. Together, our findings establish neurodevelopmental lineage plasticity as a key driver of adaptive resistance in GBM and support dual-inhibition strategies to improve therapeutic outcomes in patients with GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a2a811b34aed9bceff625fc38dcd958e60df8" target='_blank'>
              STEM-23. GBM NEURODEVELOPMENTAL LINEAGE PLASTICITY DRIVES RAPID ADAPTATION TO ONCOGENE TARGETED THERAPY
              </a>
            </td>
          <td>
            Dimitri Cadet, Toby Harris, Elisa Fazzari, Cassidy Andrasz, Nicholas A. Bayley, Thomas G. Graeber, T. Cloughesy, A. Bhaduri, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/906f1e267f0452865e325381ce2fb991c017ccf2" target='_blank'>
              Are Different Populations Fairly Represented in Single-Cell Omic Atlases?
              </a>
            </td>
          <td>
            Catrina Yang, Kavitharini Saravanan, Aryan Saharan, Kuan-lin Huang
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 The failure of current glioblastoma therapies is largely attributed to an incomplete understanding of the tumor’s inherent plasticity, a key driver of treatment resistance. Better preclinical models that possess intra-tumoral heterogeneity need to be integrated into therapeutic development pipelines. This will enable testing of effective therapeutic strategies that better target heterogeneity and identify those that would improve patient outcomes. Patient-derived glioblastoma organoids (GBOs) present a powerful pre-clinical platform for predicting the efficacy of therapeutics in a more heterogeneous, patient-proximal setting. The GBO approach involves culturing intact pieces of glioblastoma tumor without cell dissociation, maintaining parent tumor architecture, heterogeneity, and tumor microenvironment. Erythropoietin-producing human hepatocellular (Eph) receptors and ephrin ligands are cell surface proteins commonly overexpressed in cancers, with several family members being significantly overexpressed in a large proportion of glioblastoma tumors. We examined the efficacy of targeting Eph and ephrin family members, including EphA3, using antibody drug conjugate (ADC) and chimeric antigen receptor (CAR) T-cell therapy approaches in GBOs. We implemented assay efficacy readouts such as measurements of GBO size, viability, and immunofluorescence staining of cell death markers. Targeting Eph and ephrin cell surface proteins elicited potent yet variable cytotoxic responses both within and across GBO samples, underscoring their potential to address intra-tumoral heterogeneity. These findings identify Eph receptors and ephrin ligands as promising therapeutic targets in glioblastoma. Simultaneous targeting of multiple Eph/ephrin family members may enhance the ability to overcome tumor heterogeneity and support the development of more durable and effective treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb60a1068d2694df93913a8fcf3983dd27f6fa4" target='_blank'>
              EXTH-76. Validating the Eph family of receptors as therapeutic targets using patient-derived Glioblastoma organoids
              </a>
            </td>
          <td>
            N. Skarne, R. D’Souza, Paulo Martins, Onkar Mulay, L. Harris, E. Holton, A. Zaman, S. Akgül, Helen M. Palethorpe, Michael Colditz, H. Alexander, R. Jeffree, J. Powell, Kris Thurecht, Rajiv Khanna, Quan Nguyen, A. Boyd, Guillermo Gomez, Bryan W Day
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Cancer drug discovery and development rely on preclinical models that accurately reflect the molecular and functional characteristics of human tumors, while accounting for in vivo factors that influence drug efficacy, such as pharmacokinetics, metabolism and toxicity. Malignant gliomas are highly aggressive brain tumors that develop within the brain parenchyma, where their heterogeneous cellular composition engage in complex interactions with highly specialized brain cells, and a blood brain barrier that restricts drug penetration. When removed from this native environment, such as in culture or heterotopic in vivo environments (e.g., flank), gliomas either lose their molecular diversity or fail to grow altogether. Therefore, there is a critical need for physiologically relevant models that capture both the intra and inter-tumor diversity of glioma, as well as the organismal context required for drug development. Here, we present GliomaPDOX – a direct brain to brain glioma orthotopic xenograft biobank, consisting of more than 200 unique models that faithfully recapitulate the key molecular, histopathological, and proliferative features of their parental tumors. By incorporating a non-invasive, secreted reporter system to monitor tumor burden in real time—including drug-induced changes in intracranial tumor growth—we demonstrate the utility of GliomaPDOX for therapeutic evaluation. Together, this robust platform provides a physiologically relevant system to accelerate drug discovery and development for glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8aa8d2d661c245c11d742e94923975c10f7fc3da" target='_blank'>
              EXTH-02. GliomaPDOX – A direct brain-to-brain glioma xenograft library for drug discovery and development
              </a>
            </td>
          <td>
            E. Fernández, C. Tse, Jennifer Salinas, Nicholas A. Bayley, Lisa Ta, Laura Gosa, Henan Zhu, Cassidy Andrasz, Michael Vigman, F. Sanvito, B. Ellingson, Linda M Liau, T. Cloughesy, Thomas G. Graeber, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background: Epidermal growth factor receptor (EGFR) mutations occur frequently in lung adenocarcinoma (LUAD). However, the lack of practical preclinical models has limited our understanding of how EGFR mutations reshape the tumor immune microenvironment. Methods: EGFR E19del/E19del ; Trp53 -/- (EP) cell lines were established from a human EGFR exon 19 deletion mutated genetically engineered mouse model (GEMM). These cell lines were applied to generate orthotopic tumors in both immunodeficient and immunocompetent mice. Drug sensitivity assays were performed to evaluate responses to Osimertinib. Multi-omics analyses, including transcriptomic and metabolic profiling, were conducted to compare EP tumors with KRAS-mutant models and human EGFR-mutant LUAD. Results: EP cell lines formed tumors in both immunodeficient and immunocompetent mice and showed sensitivity to Osimertinib comparable to human EGFR-mutant cell lines. EP tumors further developed acquired resistance to EGFR-TKI therapy, accompanied by transcriptional reprogramming marked by enhanced epithelial-mesenchymal transition (EMT) and Wnt/ β -catenin signaling. Multi-omics analysis revealed that EP tumors closely recapitulate the molecular features of human EGFR-mutant LUAD, while exhibiting distinct transcriptomic and metabolic profiles compared to KRAS-mutant models. Immune profiling demonstrated a suppressed adaptive immune response in EP tumors, including reduced infiltration of tissue-resident memory T cell and increased M2-like polarization of macrophage. Conclusions: This study presents a novel preclinical model that practically represents EGFR-mutant LUAD. Unlike traditional models relying on exogenous EGFR mutations, EP cells are endogenously driven by EGFR signaling and accurately reflect the immunosuppressive microenvironment observed in patients. This model provides an efficient platform for investigating mechanisms of immune evasion and for developing innovative therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b4cb2f8d3e64660cd99ecdb622b3889f526bd3a" target='_blank'>
              A novel mouse cell line model reveals the tumor intrinsic and immune characteristics of EGFR-mutant lung cancer
              </a>
            </td>
          <td>
            Yueren Yan, Jun Shang, Chunnan Liu, Yuting Chen, Yufang Bao, Yue Zhao, F. Fan, Yifu Sun, Yudi Zhou, Yuke Wu, Yongbo Wang, Han Han, Haiquan Chen, Yunjian Pan
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 The tumor microenvironment (TME) is a highly complex multicellular system and has been shown to play a fundamental role in dictating tumorigenesis. High expression of regulatory surface molecules, cytokines, and chemokines in fibroblast niches within lung tumor stroma, paired with their colocalization with Treg cells, suggests cancer-associated fibroblast (CAF) niches may alter the functional response of Treg cells and influence anti-tumor immunity. Here, utilizing a transcriptomic approach, we have uncovered a distinct CAF population – which we term “imCAFs” – that is predominantly localized to tumor-derived tissue samples and is enriched for immunomodulatory gene signatures, T cell ligands, and chemoattractants. This population expresses a unique set of cell surface molecules that permit their identification and isolation from lung tumor tissue. Interestingly, CXCL9, a well-established T cell chemoattractant, was among the top differentially expressed genes in the imCAF population. We were able to demonstrate that TME Treg cells preferentially migrate to CXCL9 gradients, and colocalize with CXCL9-producing imCAFs in lung tumors. CXCL9-producing imCAFs attract a unique subset of Treg cells that highly upregulate the CXCL9 receptor, CXCR3. This subset of Treg cells is functionally distinct from CXCR3-negative Treg cells. Our analyses suggest that imCAFs play a critical role in dictating the recruitment and function of a specific subset of Treg cells to the TME in lung tumors.



 Supported by NIH/NHLBI R01HL148718; NIH/NIAID T32AI106711.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/909bf82020ae6c71525db0b4290dae269c849d42" target='_blank'>
              Immunomodulatory cancer-associated fibroblasts influence Treg cell dynamics in lung cancer 2712
              </a>
            </td>
          <td>
            Nicholas Arpaia, Olivia R. Ringham
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Adoptive T cell therapies (ACT) have revolutionized the management of hematologic malignancies; however, their efficacy in solid tumors remains limited. Accumulating evidence implicates the tumor microenvironment (TME) - a highly complex and immunosuppressive niche as a major barrier to their effectiveness. In this review, we propose that the next generation of ACT will require a fundamental shift from a reductionist focus on T cell engineering alone to an integrated approach that considers the interactions between immune cells and the TME. A comprehensive literature review identified several emerging strategies to enhance the efficacy of ACT, including reprogramming tumor vasculature, repolarizing immunosuppressive myeloid and stromal cells, leveraging oncolytic viruses to remodel antigen presentation, inducing acute sterile inflammation, and targeting the physical properties of the extracellular matrix. While many of these approaches remain in early-stage development, some have already progressed to clinical trials, indicating their potential for clinical translation. Additionally, we found that conventional therapies, such as surgery, chemotherapy, and radiotherapy, can be strategically integrated with ACT to improve therapeutic outcomes. These findings highlight a shift in the field toward more integrative approaches. Future advances will likely depend on reprogramming the TME to support T cell persistence and functions. Addressing these interconnected challenges will require closer collaboration between immunology, oncology, and bioengineering disciplines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5845940292899bbb8a96147c6b79326179c978e6" target='_blank'>
              Reprogramming the tumor microenvironment to boost adoptive T cell therapy
              </a>
            </td>
          <td>
            Maria Fernanda Meza Pacheco, Lee-Hwa Tai
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The incidence of early-onset colorectal cancer (CRC), defined as cases diagnosed in individuals under 50, is rising globally. However, its molecular and immune characteristics remain poorly understood. In this study, we analyze single-cell RNA sequencing data from 168 CRC patients, aged 22 to 91, to investigate differences between early-onset and standard-onset CRC. We find a reduced proportion of tumor-infiltrating myeloid cells, a higher burden of copy number variations, and decreased tumor-immune interactions in early-onset CRC. Additionally, immune signatures unique to early-onset CRC are associated with differential responses to immunotherapy, underscoring the need for tailored therapeutic strategies for this group of patients. These findings provide valuable insights into the molecular and immune landscape of early-onset CRC, emphasizing the importance of developing targeted prevention and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da549e2629038aa4c9de27bbf167df537f9f53c" target='_blank'>
              Single-cell integration analysis reveals molecular signatures of the tumor microenvironment in early-onset colorectal cancer
              </a>
            </td>
          <td>
            Ting Peng, Qimin Zhan
          </td>
          <td>2025-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25a860796dffeb4442ab095fa70e047b25f6f73a" target='_blank'>
              Single-cell transcriptomics of the myeloid milieu reveals an angiogenic niche in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Yechan Choi, Minkyu Shim, Suhn Hyung Kim, Duk Ki Kim, Juhee Jeong, J. Byeon, Giyong Jang, Ji-Yeon Kim, Paul Robson, Charles Lee, Han-Byoel Lee, Keehoon Jung
          </td>
          <td>2025-11-07</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Colorectal cancer (CRC) often occurs in combination with sarcopenia (SP), which affects the prognosis of CRC, but the histopathological mechanisms underlying this process are unclear. Single-cell transcriptome technology provides a tool for resolving multi-organ pathological interactions, and previous studies have lacked direct molecular evidence of a trans-tissue microenvironment. To address this, we applied an integrated analytical approach. The integration of single-cell transcriptome data from 16 CRC tumor tissues (scRNA-seq) and 17 SP muscle tissues (snRNA-seq) was used to systematically resolve heterogeneity across tissue microenvironments using Harmony to correct for batch effects and combining cellular annotations, differential expression analyses, and CellChat communication modeling. Our analysis revealed that CRC and SP share 3 core cell types, namely, endothelial cells, fibroblasts and monocytes, but they show distinct transcriptional and functional differentiation in different tissues. CRC cells are enriched in the inflammatory, angiogenic and matrix remodeling pathways, accompanied by the upregulation of hypoxia-associated genes, whereas SP is biased toward muscle structural maintenance, metabolic regulation and repair functions. The MIF-CD74, COLLAGEN and THBS-CD47 pathways were active in CRC, whereas SDC4 and CD36-related signals were dominant in SP. VWF, PDGFRA and FCN1 were stably expressed in both groups, suggesting the existence of a conserved homeostatic regulatory network. In conclusion, this study reveals the cross-tissue functional heterogeneity of cell types shared by CRC and SP, suggests a set of potential pervasive regulators, and provides new perspectives for understanding the systemic effects of tumors and the microenvironmental mechanisms of sarcopenia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28aac551ab88d08b24ed6a70bb80afdcde46ff4f" target='_blank'>
              Single-cell transcriptome analysis elucidates the microenvironmental interactions between colorectal cancer and sarcopenia
              </a>
            </td>
          <td>
            Tihong Qiu, Xiaoying Wang, Jie Li, Yiping Guan, Hongyou Wen
          </td>
          <td>2025-11-28</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) play crucial and important role in cancer dynamics by affecting homeostasis, immunosuppression, and angiogenesis within the tumor microenvironment. Using single-cell transcriptomics, we constructed a comprehensive atlas of myeloid cell populations across healthy and pan-cancer tissues that revealed heterogeneity. Our analysis suggested that TAMs may arise from two distinct origins: C1QC+ TAMs, which likely are derived from resident tissue macrophages, SPP1+ TAMs and ISG15+ TAMs, which appear to originate from circulating monocytes. Regarding immature myeloid-derived suppressor cells (MDSCs), we highlighted THBS1+ MDSCs and their descendants, SPP1+ TAMs, as key contributors to tumor progression, immunosuppression, and angiogenesis. We proposed a dichotomous model for TAMs, in which C1QC+ TAMs are associated with better patient outcomes, whereas the THBS1+ MDSCs - SPP1+ TAMs lineage correlates with poorer survival and unfavorable response to immunotherapy. This study offers insight into the complex interactions among monocyte-macrophage subtypes and sheds light on TAM heterogeneity and its implications for cancer progression and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a07f8d41863dfbb5389df0779f0c2e6e40c36fc" target='_blank'>
              Pan-cancer single-cell RNA sequencing analysis refines multi-origin monocyte and macrophage lineages.
              </a>
            </td>
          <td>
            Truc Do Thanh Nguyen, Andrew J Lee, Hyun Jung Park, Nameeta Shah, Bayrta Mandzhieva, Dong-Sup Lee, Inkyung Jung, Woong-Yang Park
          </td>
          <td>2025-11-13</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>6</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [16, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>