<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-31 09:29:07 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Spatial cellular context is crucial in shaping intratumor heterogeneity. However, understanding how each tumor establishes its unique spatial landscape and what factors drive the landscape for tumor fitness remains significantly challenging. Here, we analyzed over 2 million cells from 50 tumor biospecimens using spatial single-cell imaging and single-cell RNA sequencing. We developed a deep learning-based strategy to spatially map tumor cell states and the architecture surrounding them, which we referred to as Spatial Dynamics Network (SDN). We found that different tumor cell states may be organized into distinct clusters, or ‘villages’, each supported by unique SDNs. Notably, tumor cell villages exhibited village-specific molecular co-dependencies between tumor cells and their microenvironment and were associated with patient outcomes. Perturbation of molecular co-dependencies via random spatial shuffling of the microenvironment resulted in destabilization of the corresponding villages. This study provides new insights into understanding tumor spatial landscape and its impact on tumor aggressiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d9a8ffa548a2f4ea069cb9bc5e6205adbb07d1" target='_blank'>
              Tumor cell villages define the co-dependency of tumor and microenvironment in liver cancer
              </a>
            </td>
          <td>
            Meng Liu, M. Hernandez, Darko Castven, Hsin-Pei Lee, Wenqi Wu, Limin Wang, M. Forgues, Jonathan M. Hernandez, Jens U. Marquardt, Lichun Ma
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer deaths; nonetheless, how tumor cells adapt to vastly different organ contexts is largely unknown. To investigate this question, we generated a transcriptomic atlas of primary tumor and diverse metastatic samples from a patient with pancreatic ductal adenocarcinoma who underwent rapid autopsy. Unsupervised archetype analysis identified both shared and site-specific gene programs, including lipid metabolism and gastrointestinal programs prevalent in peritoneum and stomach wall lesions, respectively. We developed a probabilistic approach for inferring clonal phylogeny from single-cell and matched whole-exome data. Distantly related genetic clones in the peritoneum express the lipid metabolism program, likely due to signaling by the adipocyte-rich peritoneum environment, and cells in most clones express multiple programs, suggesting that transcriptomic plasticity is a prevalent feature of metastatic cells. These deeply annotated analyses using a patient-centric platform provide a model for investigating metastatic mechanisms and plasticity in advanced cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b77f53eedac93ceda8af4b898f6071abded0e4" target='_blank'>
              Transcriptomic plasticity is a hallmark of metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Alejandro Jiménez-Sánchez, S. Persad, Akimasa Hayashi, Shigeaki Umeda, Roshan Sharma, Yubin Xie, A. Mehta, Wungki Park, I. Masilionis, T. Chu, Feiyang Zhu, Jungeui Hong, R. Chaligné, Eileen M. O’Reilly, L. Mazutis, T. Nawy, I. Pe’er, Christine A. Iacobuzio-Donahue, D. Pe’er
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2886703d3c5dcc191767cc197fd5d1a7d169072" target='_blank'>
              A spatially resolved single cell proteomic atlas of Small Bowel Adenocarcinoma
              </a>
            </td>
          <td>
            Z. Dereli, B. Bozorgui, M. Sanchez, Nicholas Hornstein, Guillaume Thibault, Huamin Wang, Gordon B. Mills, John N. Weinstein, M. Overman, Anil Korkut
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3061625679fbc2f861a2180efc0a648894c17487" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment
              </a>
            </td>
          <td>
            Shoaib Ajaib, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F. Stead
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a crucial role in orchestrating tumor cell behavior and cancer progression. Recent advances in spatial profiling technologies have uncovered novel spatial signatures, including univariate distribution patterns, bivariate spatial relationships, and higher-order structures. These signatures have the potential to revolutionize tumor mechanism and treatment. In this review, we summarize the current state of spatial signature research, highlighting computational methods to uncover spatially relevant biological significance. We discuss the impact of these advances on fundamental cancer biology and translational research, address current challenges and future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed05a90f6b1d7a182ae84bb7e554725c91c44235" target='_blank'>
              Quantifying and interpreting biologically meaningful spatial signatures within tumor microenvironments
              </a>
            </td>
          <td>
            Si Jing, He-Qi Wang, Ping Lin, Jiao Yuan, Zhixuan Tang, Hong Li
          </td>
          <td>2025-03-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98801c9770ab85a801e0ca53d0db3280fc312fc" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/619b0b12a0a0a0f0529c78aba3cfb6008e7a474e" target='_blank'>
              Sparse deconvolution of cell type medleys in spatial transcriptomics
              </a>
            </td>
          <td>
            Nuray Sogunmez Erdogan, Deniz Eroglu
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc63f157070e0fd70ef6717ad423137ac275d75" target='_blank'>
              Stochastically Emergent Tumors offer in vivo whole genome interrogation of cancer evolution from non-malignant precursors
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel B. Hoelscher, V. Luga, Arun A. Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, C.K. Cornelia Ding, Bradley A. Stohr, Peng Jin, Tejasveeta V. Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R. Karthaus, C.L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Spatial omics is enabling unprecedented tissue characterization, but the ability to adequately compare spatial features across samples under different conditions is lacking. We propose a quantitative framework that catalogs significant, normalized, colocalizations between pairs of cell subpopulations, enabling comparisons among a variety of biological samples. We perform cell-pair colocalization analysis on multiplexed immunofluorescence images of assembloids constructed with lung adenocarcinoma (LUAD) organoids and cancer-associated fibroblasts derived from human tumors. Our data show that assembloids recapitulate human LUAD tumor-stroma spatial organization, justifying their use as a tool for investigating the spatial biology of human disease. Intriguingly, drug-perturbation studies identify drug-induced spatial rearrangements that also appear in treatment-naïve human tumor samples, suggesting potential directions for characterizing spatial (re)-organization related to drug resistance. Moreover, our work provides an opportunity to quantify spatial data across different samples, with the common goal of building catalogs of spatial features associated with disease processes and drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d655b9c3b36c5c88b85d107ee0f6e182ac411fa5" target='_blank'>
              A quantitative spatial cell-cell colocalizations framework enabling comparisons between in vitro assembloids and pathological specimens
              </a>
            </td>
          <td>
            Gina Bouchard, Weiruo Zhang, Ilayda Ilerten, Irene Li, Asmita Bhattacharya, Yuanyuan Li, W. Trope, Joseph B Shrager, Calvin J. Kuo, Michael G Ozawa, A. Giaccia, Lu Tian, Sylvia K Plevritis
          </td>
          <td>2025-02-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="We introduce Φ-Space ST, a platform-agnostic method to identify continuous cell states in spatial transcriptomics (ST) data using multiple scRNA-seq references. For ST with supercellular resolution, Φ-Space ST achieves interpretable cell type deconvolution with significantly faster computation. For subcellular resolution, Φ-Space ST annotates cell states without cell segmentation, leading to highly insightful spatial niche identification. Φ-Space ST harmonises annotations derived from multiple scRNA-seq references, and provides interpretable characterisations of disease cell states by leveraging healthy references. We validate Φ-Space ST in three case studies involving CosMx, Visium and Stereo-seq platforms for various cancer tissues. Our method revealed niche-specific enriched cell types and distinct cell type co-presence patterns that distinguish tumour from non-tumour tissue regions. These findings highlight the potential of Φ-Space ST as a robust and scalable tool for ST data analysis for understanding complex tissues and pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7881362e9ca8343d6116547e4bd2ccab35cc278" target='_blank'>
              Φ-Space ST: a platform-agnostic method to identify cell states in spatial transcriptomics studies
              </a>
            </td>
          <td>
            Jiadong Mao, Jarny Choi, Kim-Anh Lê Cao
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC), accounting for 90% of pancreatic neoplasms, is characterized by its poor prognosis, with a 5-year survival rate of only 12%. Most patients are diagnosed with metastatic or locally advanced disease, leaving only 15% eligible for curative resection. PDAC exhibits resistance to chemotherapy, targeted therapies, and immunotherapy, largely due to its highly heterogeneous tumor microenvironment (TME). In this study, we performed an integrative analysis of publicly available scRNA, spatial transcriptomics, and bulk RNA sequencing datasets to investigate the influence of TME composition and tumor architecture on PDAC progression, treatment response, and clinical outcomes. We identified TME subtypes with distinct cellular compositions, functional signatures, and immunomodulatory cell-cell interactions. Spatially distinct cellular niches and gene modules revealed heterogeneity across primary tumors and metastatic lesions. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications. These findings underscore the importance of integrating multi-omics approaches to unravel the complexity of the PDAC TME and highlight its potential to inform therapeutic strategies and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768379a4460d13ebf34f45a5af50c7d930ab8e2" target='_blank'>
              Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel F. Pozo de Mattos P., Marvin Paulo Lins, Julia Fontoura, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In this study, we developed an integrated single cell transcriptomic (scRNAseq) atlas of human breast cancer (BC), the largest resource of its kind, totaling > 600,000 cells across 138 patients. Rigorous integration and annotation of publicly available scRNAseq data enabled a highly resolved characterization of epithelial, immune, and stromal heterogeneity within the tumor microenvironment (TME). Within the immune compartment we were able to characterize heterogeneity of CD4, CD8 T cells and macrophage subpopulations. Within the stromal compartment, subpopulations of endothelial cells (ECs) and cancer associated fibroblasts (CAFs) were resolved. Within the cancer epithelial compartment, we characterized the functional heterogeneity of cells across the axes of stemness, epithelial-mesenchymal plasticity, and canonical cancer pathways. Across all subpopulations observed in the TME, we performed a multi-resolution survival analysis to identify epithelial cell states and immune cell types which conferred a survival advantage in both The Cancer Genome Atlas (TCGA) and METABRIC. We also identified robust associations between TME composition and clinical phenotypes such as tumor subtype and grade that were not discernible when the analysis was limited to individual datasets, highlighting the need for atlas-based analyses. This atlas represents a valuable resource for further high-resolution analyses of TME heterogeneity within BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7db0b505659425c4bfdef92aba55ca31e10f8c9" target='_blank'>
              A highly resolved integrated transcriptomic atlas of human breast cancers
              </a>
            </td>
          <td>
            Andrew Chen, Lina Kroehling, Christina S. Ennis, Gerald V. Denis, S. Monti
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88360a99c9dba219540b793cdb094157ffa2ec6a" target='_blank'>
              Lineage Tracing Reveals Clone-Specific Responses to Doxorubicin in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Daylin Morgan, Andrea L. Gardner, Amy Brock
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74fb81e29120e8309bf4f3e49e31706525768da7" target='_blank'>
              Integrating multi-modal transcriptomics identifies cellular subtypes with distinct roles in PDAC progression
              </a>
            </td>
          <td>
            Jun Wu, Tenghui Dai, Ziyue Li, Meng Pan, Wei Zhang, Hao Chen, Guansheng Zheng, Li Qiao, Qizhou Lian, Yang Liu, Jierong Chen
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06a703b5eeba2e901b0564ff4995a24e3b880d79" target='_blank'>
              Single-cell spatial analysis of pediatric high-grade glioma reveals a novel population of SPP1+/GPNMB+ myeloid cells with immunosuppressive and tumor-promoting capabilities
              </a>
            </td>
          <td>
            Thijs J M van den Broek, R. Hoogendijk, M. Kranendonk, J. Lammers, Akshaya L. Krishnamoorthy, Ravian L. van Ineveld, Milo Molleson, Vasily O. Tsvetkov, F. Ringnalda, Marc van de Wetering, Yan Su, John I. Bianco, Cristian Ruiz Moreno, Mario G Ries, E. Hoving, J. van der Lugt, Leila Akkari, David P. Schrijver, H. Stunnenberg, A. Rios, D. V. van Vuurden, A. Zomer
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Cellular organization is central to tissue function and homeostasis, influencing development, disease progression, and therapeutic outcomes. The emergence of spatial omics technologies, including spatial transcriptomics and proteomics, has enabled the integration of molecular and histological features within tissues. Analyzing these multimodal data presents unique challenges, including variable resolutions, imperfect tissue alignment, and limited or variable spatial coverage. To address these issues, we introduce CORAL, a probabilistic deep generative model that leverages graph attention mechanisms to learn expressive, integrated representations of multimodal spatial omics data. CORAL deconvolves low-resolution spatial data into high-resolution single-cell profiles and detects functional spatial domains. It also characterizes cell-cell interactions and elucidates disease-relevant spatial features. Validated on synthetic data and experimental datasets, including Stero-CITE-seq data from mouse thymus, and paired CODEX and Visium data from hepatocellular carcinoma, CORAL demonstrates robustness and versatility. In hepatocellular carcinoma, CORAL uncovered key immune cell subsets that drive the failure of response to immunotherapy, highlighting its potential to advance spatial single-cell analyses and accelerate translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/295042adfe589c5c9974e6c8e7ec5b6246fdd726" target='_blank'>
              Learning single-cell spatial context through integrated spatial multiomics with CORAL
              </a>
            </td>
          <td>
            Siyu He, Matthew Bieniosek, Dongyuan Song, Jingtian Zhou, Benjamin Chidester, Zhenqin Wu, Joseph Boen, Padmanee Sharma, Alexandro E. Trevino, James Zou
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/190acfc8ea743ab5360d22a148d803b01bb31afc" target='_blank'>
              A pathogenic subpopulation of human glioma associated macrophages linked to glioma progression
              </a>
            </td>
          <td>
            Kenny Kwok Hei Yu, Z. Abou-Mrad, Kristof Törkenczy, Isabell Schulze, Jennifer Gantchev, Gerard Baquer, Kelsey Hopland, Evan D. Bander, Umberto Tosi, Cameron Brennan, Nelson S. Moss, Pierre-Jacques Hamard, R. Koche, Caleb Lareau, N. Agar, T. Merghoub, Viviane Tabar
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Abstract Background The tumor microenvironment (TME) is a complex and dynamic ecosystem that is known to influence responses to immunotherapy. We leveraged single-cell spatial transcriptomics to systematically dissect the intricate complexity of the TME, in particular the cellular heterogeneity and spatial interactions. Their collective impact on immunotherapy efficacy was studied in the context of a homogeneous group of patients with vulvar high-grade squamous intraepithelial lesions (vHSIL) treated with an immunotherapeutic tumor-specific peptide vaccine. Methods We performed single-cell spatial transcriptomics on 20 pretreatment vHSIL lesions, stratified by clinical response to immunotherapeutic vaccination into complete responders (CR), partial responders (PR) and non-responders (NR). Using a 1,000-gene panel, we mapped over 274,000 single cells in situ, identifying 18 cell clusters and 99 distinct non-epithelial cell states. Findings were validated against public single-cell transcriptomic data sets to assess their broader relevance across tumor types. Results Profound heterogeneity within the TME was detected across the response groups. CR lesions exhibited a higher ratio of immune-supportive to immune-suppressive cells—a pattern mirrored in other solid tumors following neoadjuvant checkpoint blockade. Key immune populations enriched in CRs included CD4+CD161+ effector T cells and chemotactic CD4+ and CD8+ T cells. Conversely, PRs were characterized by increased proportions of T helper 2 cells and CCL18-expressing macrophages, which are associated with the recruitment of type 2 T cells and regulatory T cells. NRs displayed preferential infiltration with immunosuppressive fibroblasts. Distinct spatial immune ecosystems further defined response groups. Although a number of immune cells were detected in all patients, type 1 effector cells dominated interactions in CRs, type 2 cells were prominently interacting in PRs, while NRs lacked organized immune cell interactions. Conclusions This study underscores the dual importance of both cellular composition and spatial organization in steering clinical response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b353470be584af7d95e65b2aa9a5dffe4241b32d" target='_blank'>
              Single-cell spatial transcriptomics unravels cell states and ecosystems associated with clinical response to immunotherapy
              </a>
            </td>
          <td>
            Z. Abdulrahman, R. Slieker, Daniel McGuire, M. J. Welters, M. V. van Poelgeest, S. H. van der Burg
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80bfc8505face3c0cb655cd2772225105395f750" target='_blank'>
              Deep spatial proteomics of ovarian cancer precursor lesions delineates early disease changes and cell-of-origin signatures
              </a>
            </td>
          <td>
            Anuar Makhmut, M. Dragomir, Sonja Fritzsche, Markus Moebs, Wolfgang D. Schmitt, Eliane T Taube, Fabian Coscia
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors (TKIs) provide therapeutic benefit in EGFR mutation–positive non–small cell lung cancer, but some individuals develop early resistance. We performed spatial transcriptomics analysis of pre- and posttreatment tumor samples from the same patients to explore the underlying mechanisms of such early resistance. The proportion and activation of fibroblasts increased in association with the development of early resistance, whereas a distinct tumor cell cluster showed activation of tumor necrosis factor–α signaling via the NF-κB pathway even before treatment. Also in the early resistance sample, specific tumor cell clusters interacted with immune and stromal cells. Immature tertiary lymphoid structures (TLSs) were enriched in the early resistance sample, whereas mature TLSs were observed in the long-term response sample. These findings implicate tumor heterogeneity and an inflammatory tumor microenvironment in early EGFR-TKI resistance, providing insight into potential therapeutic strategies to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59cd540165dee7cba115de4f78c74d67fa9e26e" target='_blank'>
              Spatial dynamics of the tumor microenvironment associated with emerging resistance to targeted therapy in EGFR-mutated non–small cell lung cancer
              </a>
            </td>
          <td>
            Satoshi Nakamura, Daisuke Shibahara, Kentaro Tanaka, Yasuyuki Kishikawa, Mikiko Hashisako, K. Nakatomi, Noriaki Nakagaki, M. Kohno, Koichi Azuma, R. Ibusuki, K. Otsubo, Y. Yoneshima, E. Iwama, Yoshinao Oda, I. Okamoto
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8b6d1c2313c8e86b7cbcc29c94f8060fb9e7af4" target='_blank'>
              Abstract B048: Spatiotemporal analysis during tumor rejection reveals coordinated cellular dynamics underlying successful immune-checkpoint therapy
              </a>
            </td>
          <td>
            Ruan FV Medrano, Vladimir Sukhov, Maxim N Artyomov, Robert Schreiber
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Spatially resolved transcriptomics (SRT) has transformed tissue biology by linking gene expression profiles with spatial information. However, sequencing-based SRT methods aggregate signals from multiple cell types within capture locations (“spots”), masking cell-type-specific gene expression patterns. Traditional cell-type deconvolution methods estimate cell compositions within spots but fail to resolve cell-type-specific gene expression, limiting their ability to uncover critical biological processes such as cellular interactions and microenvironmental dynamics. Here, we present STged (spatial transcriptomic gene expression deconvolution), a novel computational framework that goes beyond traditional deconvolution by reconstructing cell-type-specific gene expression profiles from mixed spots. STged integrates graph-based spatial correlations and reference-derived gene signatures using a non-negative least-squares regression framework, achieving precise and biologically meaningful deconvolution. Comprehensive simulations show that STged consistently outperforms existing methods in accuracy and robustness. Applications to human pancreatic ductal adenocarcinoma and human squamous cell carcinoma datasets reveal its capacity to identify microenvironment-specific highly variable genes, reconstruct spatial cell–cell communication networks, and resolve tissue architecture at near-single-cell resolution. In mouse kidney tissues, STged uncovers dynamic spatial gene expression patterns and distinct gene programs, advancing our understanding of tissue heterogeneity and cellular dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db33671170dacba5e74f384054d9b15e8f2088ef" target='_blank'>
              Precise gene expression deconvolution in spatial transcriptomics with STged
              </a>
            </td>
          <td>
            Jia-Juan Tu, Hong Yan, Xiao-Fei Zhang, Zhixiang Lin
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="High-grade gliomas (HGG) are incurable brain malignancies in children and adults. Breakthrough advances in transcriptomic technologies unveiled the intricate diversity of cellular states and their spatial organization within HGGs. We qualitatively integrated 55 neoplastic transcriptomic signatures described in 17 single-cell and spatial RNA sequencing-based studies. Our review delineates a spectrum of cellular states, represented by the expression of specific genes, which can be conceptualized along a “reactive-developmental programs” axis. Additionally, we discussed the potential cues influencing these cellular states, including how spatial organization may impact transcriptomic dynamics. Leveraging these insightful discoveries, we discussed a novel, evolutive way to integrate the different transcriptomic signatures in two or three dimensions, incorporating developmental states, their proliferative capacity, and their possible transition towards reactive states. This integrated analysis illuminates the diverse cellular landscape of HGGs and provides a valuable resource for further elucidation of malignant mechanisms, and for the design of therapeutic endeavors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda3dd1ef6762a74a82f982637ae107d4fb74853" target='_blank'>
              An integrated perspective on single-cell and spatial transcriptomic signatures in high-grade gliomas
              </a>
            </td>
          <td>
            Célia Lemoine, Marc-Antoine Da Veiga, B. Rogister, Caroline Piette, Virginie Neirinckx
          </td>
          <td>2025-02-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98eeee496484cc85e8f46ccc055600cb6afe1b1e" target='_blank'>
              A harmonized ovarian cancer scRNA-seq atlas to dissect disease heterogeneity underlying metastatization and chemoresistance
              </a>
            </td>
          <td>
            M. Sallese, Vittorio Aiello, C. E. Villa, Pietro Lo Riso, Giuseppe Testa
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="NF2-related Schwannomatosis (NF2 SWN) is a rare disease characterised by the growth of multiple nervous system neoplasms, including bilateral vestibular schwannoma (VS). VS tumours are characterised by extensive leucocyte infiltration. However, the immunological landscape in VS and the spatial determinants within the tumour microenvironment that shape the trajectory of disease are presently unknown. In this study, to elucidate the complex immunological networks across VS, we performed imaging mass cytometry (IMC) on clinically annotated VS samples from NF2 SWN patients. We reveal the heterogeneity in neoplastic cell, myeloid cell and T cell populations that co-exist within VS, and that distinct myeloid cell and Schwann cell populations reside within varied spatial contextures across characteristic Antoni A and B histomorphic niches. Interestingly, T-cell populations co-localise with tumour-associated macrophages (TAMs) in Antoni A regions, seemingly limiting their ability to interact with tumorigenic Schwann cells. This spatial landscape is altered in Antoni B regions, where T-cell populations appear to interact with PD-L1+ Schwann cells. We also demonstrate that prior bevacizumab treatment (VEGF-A antagonist) preferentially reduces alternatively activated-like TAMs, whilst enhancing CD44 expression, in bevacizumab-treated tumours. Together, we describe niche-dependent modes of T-cell regulation in NF2 SWN VS, indicating the potential for microenvironment-altering therapies for VS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e66dea3c7770dc53e6070f848f2e765373a2eab3" target='_blank'>
              Spatial mapping of immune cell environments in NF2-related schwannomatosis vestibular schwannoma
              </a>
            </td>
          <td>
            Adam P. Jones, Michael J Haley, Miriam H Meadows, Grace E. Gregory, C. Hannan, Ana K Simmons, Leoma D. Bere, Daniel Lewis, Pedro Oliveira, Miriam J. Smith, Andrew T King, D Gareth R Evans, P. Paszek, David Brough, Omar N Pathmanaban, Kevin N. Couper
          </td>
          <td>2025-03-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407e0ecf5209e24e1ad27b286f77ee0657cbbbda" target='_blank'>
              Abstract A120: Single-Cell RNA Sequencing Combined with Lineage Tracing Identify Clonal Populations Across Melanoma Organ-Specific Metastasis
              </a>
            </td>
          <td>
            Veronica Aedo-Lopez
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Here we present STModule, a Bayesian method developed to identify tissue modules from spatially resolved transcriptomics that reveal spatial components and essential characteristics of tissues. STModule uncovers diverse expression signals in transcriptomic landscapes such as cancer, intraepithelial neoplasia, immune infiltration, outcome-related molecular features and various cell types, which facilitate downstream analysis and provide insights into tumor microenvironments, disease mechanisms, treatment development, and histological organization of tissues. STModule captures a broader spectrum of biological signals compared to other methods and detects novel spatial components. The tissue modules characterized by gene sets demonstrate greater robustness and transferability across different biopsies. STModule: https://github.com/rwang-z/STModule.git. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01441-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61c2f79171ce9194bffe681aecda7bf66be2273" target='_blank'>
              STModule: identifying tissue modules to uncover spatial components and characteristics of transcriptomic landscapes
              </a>
            </td>
          <td>
            Ran Wang, Yan Qian, Xiaojing Guo, Fangda Song, Zhiqiang Xiong, Shirong Cai, Xiuwu Bian, Man-Hon Wong, Qin Cao, Lixin Cheng, Gang Lu, K. Leung
          </td>
          <td>2025-03-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Spatial transcriptomics integrates transcriptomics data with histological tissue images, offering deeper insights into cellular organization and molecular functions. However, existing computational platforms mainly focus on genomic analysis, leaving a gap in the seamless integration of genomic and image analysis. To address this, we introduce Thor, a comprehensive computational platform for multi-modal analysis of spatial transcriptomics and histological images. Thor utilizes an anti-shrinking Markov diffusion method to infer single-cell spatial transcriptomes from spot-level data, effectively integrating cell morphology with spatial transcriptomics. The platform features 10 modules designed for cell-level genomic and image analysis. Additionally, we present Mjolnir, a web-based tool for interactive tissue analysis using vivid gigapixel images that display information on histology, gene expression, pathway enrichment, and immune response. Thor's accuracy was validated through simulations and ISH, MERFISH, Xenium, and Stereo-seq datasets. To demonstrate its versatility, we applied Thor for joint genomic-histology analysis across various datasets. In in-house heart failure patient samples, Thor identified a regenerative signature in heart failure, with protein presence confirmed in blood vessels through immunofluorescence staining. Thor also revealed the layered structure of the mouse olfactory bulb, performed unbiased screening of breast cancer hallmarks, elucidated the heterogeneity of immune responses, and annotated fibrotic regions in multiple heart failure zones using a semi-supervised approach. Furthermore, Thor imputed high-resolution spatial transcriptomics data in an in-house bladder cancer sample sequenced using Visium HD, uncovering stronger spatial patterns that align more closely with histology. Bridging the gap between genomic and image analysis in spatial biology, Thor offers a powerful tool for comprehensive cellular and molecular analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d882bc79c8c5d2b2813a2269a80bde6db2f397cb" target='_blank'>
              Thor: a platform for cell-level investigation of spatial transcriptomics and histology
              </a>
            </td>
          <td>
            Guangyu Wang, Pengzhi Zhang, Weiqing Chen, Tu Tran, , Kaylee Carter, Ibrahem Kandel, Shengyu Li, Li Lai, Qianqian Song, Keith Youker, Yu Yang, Keith Syson Chan, Xen Ping Hoi, Fotis Nikolo
          </td>
          <td>2025-03-10</td>
          <td>Research Square</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7461a1f413e55baf162ee7fd7309ee3cb97ce7a" target='_blank'>
              Deep visual multi-omics profiling reveals mechanisms that underly cancer cell differentiation and aggressiveness in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            H. A. Bolck, Ede Migh, A. Kriston, Natalia Zajac, Susanne Kreutzer, Tiberiu Totu, Peter Leary, Ferenc Kovács, D. Rutishauser, Sybille Pfammatter, Jonas Grossmann, Cassandra Litchfield, Marija Buljan, Niels J Rupp, Péter Horváth, Holger Moch
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly malignant tumor in women, characterized by high morbidity, mortality, and recurrence rates. Although surgical treatment, radiotherapy, and chemotherapy are the mainstays of current treatment methods, the high heterogeneity of TNBC results in unsatisfactory outcomes with low 5-year survival rates. Rapid advancements in omics technology have propelled the understanding of TNBC molecular biology. The emergence of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has significantly enhanced knowledge of tumor heterogeneity and the distribution, functionality, and intercellular interactions of various cell types within the tumor microenvironment, including tumor cells, T cells, B cells, macrophages, and fibroblasts. The present study provides an overview of the technical characteristics of scRNA-seq and ST, highlighting their applications in exploring TNBC heterogeneity, cell spatial distribution patterns, and intercellular interactions. This review aims to enhance the comprehension of TNBC at the cellular level for the development of effective therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db2a16a0771f525b51537059ff8768102022813" target='_blank'>
              Analysis of single-cell and spatial transcriptomics in TNBC cell-cell interactions
              </a>
            </td>
          <td>
            Yan Xin, Qiji Ma, Qiang Deng, Tielin Wang, Dongxu Wang, Gang Wang
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Lung adenocarcinoma, the most common subtype of lung cancer, is genomically complex, with tumors containing tens to hundreds of non-synonymous mutations. However, little is understood about how genes interact with each other to enable the evolution of cancer in vivo, largely due to a lack of methods for investigating genetic interactions in a high-throughput and quantitative manner. Here, we employed a novel platform to generate tumors with inactivation of pairs of ten diverse tumor suppressor genes within an autochthonous mouse model of oncogenic KRAS-driven lung cancer. By quantifying the fitness of tumors with every single and double mutant genotype, we show that most tumor suppressor genetic interactions exhibited negative epistasis, with diminishing returns on tumor fitness. In contrast, Apc inactivation showed positive epistasis with the inactivation of several other genes, including synergistic effects on tumor fitness in combination with Lkb1 or Nf1 inactivation. Sign epistasis was extremely rare, suggesting a surprisingly accessible fitness landscape during lung tumorigenesis. These findings expand our understanding of the interactions that drive tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e03ac1e8a6ee59b9db5be527feeaddbf8b397192" target='_blank'>
              Combinatorial In Vivo Genome Editing Identifies Widespread Epistasis and an Accessible Fitness Landscape During Lung Tumorigenesis
              </a>
            </td>
          <td>
            Jess D. Hebert, Yuning J Tang, Márton Szamecz, Laura Andrejka, Steven S. Lopez, D. Petrov, Gábor Boross, M. Winslow
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ca2d699ba974ef22ac49a1f2990095215e2dd0" target='_blank'>
              Bacterial population dynamics during colonization of solid tumors
              </a>
            </td>
          <td>
            Serkan Sayin, Motasem ElGamel, Brittany Rosener, Michael Brehm, Andrew Mugler, Amir Mitchell
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan A. Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4421024c849016285df42c8f390ac35abcedd51e" target='_blank'>
              Pan-cancer spatial characterization of key immune biomarkers in the tumor microenvironment
              </a>
            </td>
          <td>
            James R. Lindsay, Jennifer Altreuter, J. Alessi, Jason L. Weirather, Anita Giobbie-Hurder, Ian D Dryg, Katharina V. Hoebel, B. Sharma, K. Felt, F. S. Hodi, N. Lindeman, L. Sholl, E. Cerami, Jonathan A. Nowak, Mark M Awad, Scott J Rodig, William E. Lotter
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3aa6e9e33a49a78ba8f7c54e58b7d68564b492" target='_blank'>
              Applications of Multiplex Immunohistochemistry in Evaluating Spatiotemporal Heterogeneity of T Cells
              </a>
            </td>
          <td>
            Mercedes Machuca-Ostos, Tim de Martines, K. Yoshimura, Junichi Mitsuda, Sumiyo Saburi, Alisa Kimura, Hiroki Morimoto, Koichi Yoshizawa, Nana Sakurai, Nanako Murakami, Kayo Kitamoto, Makoto Yasuda, Y. Sugiyama, H. Ogi, Saya Shibata, Aya Miyagawa-Hayashino, Eiichi Konishi, Kyoko Itoh, Takahiro Tsujikawa, Shigeru Hirano
          </td>
          <td>2025-02-17</td>
          <td>Immuno</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Single-cell spatial omics analysis requires consideration of biological functions and mechanisms in a microenvironment. However, microenvironment analysis using bioinformatic methods is limited by the need to detect histological morphology and extend it to the surrounding area. In this study, we developed SpatialKNifeY (SKNY), an image-processing-based toolkit that detects spatial domains that potentially reflect histology and extends these domains to the microenvironment. Using spatial transcriptomic data from breast cancer, we applied the SKNY algorithm to identify tumor spatial domains, followed by clustering of the domains, trajectory estimation, and spatial extension to the tumor microenvironment (TME). The results of the trajectory estimation were consistent with the known mechanisms of cancer progression. We observed tumor vascularization and immunodeficiency at mid- and late-stage progression in TME. Furthermore, we applied the SKNY to integrate and cluster the spatial domains of 14 patients with metastatic colorectal cancer, and the clusters were divided based on the TME characteristics. In conclusion, the SKNY facilitates the determination of the functions and mechanisms in the microenvironment and cataloguing of the features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796429c918dab0f1b4d04958c50f7cf17d100c5e" target='_blank'>
              SpatialKNifeY (SKNY): Extending from spatial domain to surrounding area to identify microenvironment features with single-cell spatial omics data
              </a>
            </td>
          <td>
            Shunsuke A Sakai, Ryosuke Nomura, Satoi Nagasawa, SungGi Chi, Ayako Suzuki, Yutaka Suzuki, Mitsuho Imai, Y. Nakamura, Takayuki Yoshino, Shumpei Ishikawa, K. Tsuchihara, S. Kageyama, Riu Yamashita
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Intratumoral heterogeneity is the main cause of tumor treatment failure, varying across disease sites (spatial heterogeneity) and polyclonal properties of tumors that evolve over time (temporal heterogeneity). As our understanding of intratumoral heterogeneity, the formation of which is mainly related to the genomic instability, epigenetic modifications, plastic gene expression, and different microenvironments, plays a substantial role in drug-resistant as far as tumor metastasis and recurrence. Understanding the role of intratumoral heterogeneity, it becomes clear that a single therapeutic agent or regimen may only be effective for subsets of cells with certain features, but not for others. This necessitates a shift from our current, unchanging treatment approach to one that is tailored against the killing patterns of cancer cells in different clones. In this review, we discuss recent evidence concerning global perturbations of intratumoral heterogeneity, associations of specific intratumoral heterogeneity in lung cancer, the underlying mechanisms of intratumoral heterogeneity potentially leading to formation, and how it drives drug resistance. Our findings highlight the most up-to-date progress in intratumoral heterogeneity and its role in mediating tumor drug resistance, which could support the development of future treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d2f0eb0a60c64efe2bd230e8933a5f80d2d405" target='_blank'>
              Intratumoral heterogeneity and drug resistance in cancer
              </a>
            </td>
          <td>
            Yue-Chun Fu, Shao-Bo Liang, Min Luo, Xue-Ping Wang
          </td>
          <td>2025-03-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC), accounting for 30% of lung cancer cases, lacks adequate research due to limited understanding of its molecular abnormalities. Our study analyzed public LUSC datasets to explore the tumor microenvironment (TME) composition using scRNA-seq from two cohorts. Applying non-negative matrix factorization, we identified unique malignant cell phenotypes, or meta-programs (MPs), based on gene expression patterns. Survival analysis revealed the clinical relevance of these MPs. Findings illuminated a TME landscape enriched with immune cells—CD8 + T, exhausted T, CD4 + T, and naive T cells—and suggested roles for myeloid cells, like cDC1 and pDCs, in LUSC progression. Different MPs highlighted the heterogeneity of malignant cells and their clinical implications. Targeting MP-specific genes may enable personalized therapy, especially for early-stage LUSC. This study offers insights into immune cell function in tumor dynamics, identifies MPs, and paves the way for novel LUSC strategies, enhancing early intervention, personalized treatment, and prognosis, ultimately improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93916-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a302b866a51ae52b8b8d7352c3feab56f45e3d0e" target='_blank'>
              Single-cell RNA sequencing reveals potential therapeutic targets in the tumor microenvironment of lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Junda Fan, Yu Chen, Yue Gong, Hongmei Sun, Rui Hou, Xiaoya Dou, Yanping Zhang, Cheng Huo
          </td>
          <td>2025-03-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer cells adapt to treatment, leading to the emergence of clones that are more aggressive and resistant to anti-cancer therapies. We have a limited understanding of the development of treatment resistance as we lack technologies to map the evolution of cancer under the selective pressure of treatment. To address this, we developed a hierarchical, dynamic lineage tracing method called FLARE (Following Lineage Adaptation and Resistance Evolution). We use this technique to track the progression of acute myeloid leukemia (AML) cell lines through exposure to Cytarabine (AraC), a front-line treatment in AML, in vitro and in vivo. We map distinct cellular lineages in murine and human AML cell lines predisposed to AraC persistence and/or resistance via the upregulation of cell adhesion and motility pathways. Additionally, we highlight the heritable expression of immunoproteasome 11S regulatory cap subunits as a potential mechanism aiding AML cell survival, proliferation, and immune escape in vivo. Finally, we validate the clinical relevance of these signatures in the TARGET-AML cohort, with a bisected response in blood and bone marrow. Our findings reveal a broad spectrum of resistance signatures attributed to significant cell transcriptional changes. To our knowledge, this is the first application of dynamic lineage tracing to unravel treatment response and resistance in cancer, and we expect FLARE to be a valuable tool in dissecting the evolution of resistance in a wide range of tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f240aad73d27cb602765176a2877d013637b5963" target='_blank'>
              Hierarchical Lineage Tracing Reveals Diverse Pathways of AML Treatment Resistance
              </a>
            </td>
          <td>
            Rachel Saxe, Hannah Stuart, Abigail Marshall, Fahiima Abdullahi, Zoë Chen, F. Emiliani, Aaron McKenna
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Checkpoint inhibitors have led to remarkable benefits in non-small cell lung cancer (NSCLC), yet response rates remain below expectations. High-dimensional analysis and mechanistic experiments in clinical samples and relevant NSCLC models uncovered the immune composition of lung cancer tissues, providing invaluable insights into the functional properties of tumor-infiltrating T cells and myeloid cells. Among myeloid cells, type 1 conventional dendritic cells (cDC1s) stand out for their unique ability to induce effector CD8 T cells against neoantigens and coordinate antitumoral immunity. Notably, lung resident cDC1 are particularly abundant and long-lived and express a unique tissue-specific gene program, underscoring their central role in lung immunity. Here, we discuss recent insights on the induction and regulation of antitumoral T cell responses in lung cancer, separating it from the tissue-agnostic knowledge generated from heterogeneous tumor models. We focus on the most recent studies dissecting functional states and spatial distribution of lung cDC1 across tumor stages and their impact on T cell responses to neoantigens. Finally, we highlight relevant gaps and emerging strategies to harness lung cDC1 immunostimulatory potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2155ad4b9a79772729f8fb7af74e6020d7c7eb14" target='_blank'>
              Tissue-specific properties of type 1 dendritic cells in lung cancer: implications for immunotherapy
              </a>
            </td>
          <td>
            Lucía López Rodríguez, Roberto Amadio, G. M. Piperno, F. Benvenuti
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca97e870a1822de5485cb1751fdbfdd64fadf250" target='_blank'>
              Ontogeny Dictates Oncogenic Potential, Lineage Hierarchy, and Therapy Response in Pediatric Leukemia
              </a>
            </td>
          <td>
            Ke Wang, Shayan Saniei, Nikita Poddar, Subrina Autar, Saul Carcamo, Meghana Sreenath, Jack H Peplinski, R. Ries, Isabella G Martinez, Clifford Chao, Anna Huo-Chang Mei, Noshin Rahman, Levan Mekerishvili, Miguel Quijada-álamo, Grace Freed, Mimi Zhang, Katherine Lachman, Zayna Diaz, Manuel M Gonzalez, Jing Zhang, Giang Pham, Dan Filipescu, Mirela Berisa, Tommaso Balestra, J. Reisz, A. D’Alessandro, Daniel J Puleston, Emily Bernstein, Jerry E. Chipuk, M. Wunderlich, S. Tasian, Bridget K. Marcellino, Ian A. Glass, Christopher M Sturgeon, D. Landau, Zhihong Chen, E. Papapetrou, Franco Izzo, S. Meshinchi, Dan Hasson, Elvin Wagenblast
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) are key components of the hepatocellular carcinoma (HCC) tumor microenvironment (TME). regulating tumor proliferation, metastasis, therapy resistance, immune evasion via diverse mechanisms. A deeper understanding of the l diversity of CAFs is essential for predicting patient prognosis and guiding treatment strategies. Methods We examined the diversity of CAFs in HCC by integrating single-cell, bulk, and spatial transcriptome analyses. Results Using a training cohort of 88 HCC single-cell RNA sequencing (scRNA-seq) samples and a validation cohort of 94 samples, encompassing over 1.2 million cells, we classified three fibroblast subpopulations in HCC: HLA-DRB1 + CAF, MMP11 + CAF, and VEGFA + CAF based on highly expressed genes of which, which are primarily located in normal tissue, tumor boundaries, and tumor interiors, respectively. Cell trajectory analysis revealed that VEGFA + CAFs are at the terminal stage of differentiation, which, notably, is tumor-specific. VEGFA + CAFs were significantly associated with patient survival, and the hypoxic microenvironment was found to be a major factor inducing VEGFA + CAFs. Through cellular communication with capillary endothelial cells (CapECs), VEGFA + CAFs promoted intra-tumoral angiogenesis, facilitating tumor progression and metastasis. Additionally, a machine learning model developed using high-expression genes from VEGFA + CAFs demonstrated high accuracy in predicting prognosis and sorafenib response in HCC patients. Conclusions We characterized three fibroblast subpopulations in HCC and revealed their distinct spatial distributions within the tumor. VEGFA + CAFs, which was induced by hypoxic TME, were associated with poorer prognosis, as they promote tumor angiogenesis through cellular communication with CapECs. Our findings provide novel insights and pave the way for individualized therapy in HCC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06192-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2410faffb025161d800cf5841a3c4049fde9d25" target='_blank'>
              Integration of single-cell and spatial transcriptomics reveals fibroblast subtypes in hepatocellular carcinoma: spatial distribution, differentiation trajectories, and therapeutic potential
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-02-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f579a4d65bfde6d9ba50ff844febde4a92d36d5" target='_blank'>
              Clonal memory of colitis accumulates and promotes tumor growth
              </a>
            </td>
          <td>
            Surya Nagaraja, Lety Ojeda-Miron, Ruochi Zhang, Ena Oreskovic, Yan Hu, Daniel Zeve, Karina Sharma, Roni R. Hyman, Qiming Zhang, Andrew Castillo, David T. Breault, Ömer H Yilmaz, Jason D. Buenrostro
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf0b044513a32c1bfdbc6767c40e3363e83eee34" target='_blank'>
              Abstract B038: A self-supervised AI model leveraging spatial omics for analyzing tumor microenvironment heterogeneity in breast cancer only with H&E
              </a>
            </td>
          <td>
            Haenara Shin, Dongjoo Lee, Yooeun Kim, D. Lee, Kwon Joong Na, C. Cha, Hosub Park, Hongyoon Choi
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1fe686f97fcb46c56e3f6bfbef8e78fcca9cb41" target='_blank'>
              Pericyte-tumor crosstalk facilitates metastatic tumor cell latency through PIEZO1-activated lysophospholipid transfer
              </a>
            </td>
          <td>
            Tamara McErlain, Morgan J Glass, Elizabeth C McCulla, Caitlin A Madden, Lauren E Ziemer, Kirsten E Overdahl, Alan K. Jarmusch, Michael J. Kruhlak, A. Chesler, Howard H Yang, Maxwell P Lee, Cristina M Branco, Meera Murgai
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee29e34be0729357213c5d8d79a8e1845ecaf78" target='_blank'>
              Multidimensional characterization of cellular ecosystems in Hodgkin lymphoma
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew G Lytle, Yifan Yin, Makoto Kishida, Michael Li, Denise Smorra, Laura K. Hilton, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O'Brien, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Patino Rangel, Michael Hong, Shaocheng Wu, Eric Lee, Katsuyoshi Takata, T. Miyata‐Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander M Xu, Akil Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, Ryan D Morin, A. Alizadeh, K. Savage, David W. Scott, Christian Steidl
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The spatial arrangement of cells plays a critical role in determining their functions and interactions within tissues. However, single-cell RNA sequencing (scRNA-seq) dissociates cells from their native tissue context, resulting in a loss of spatial information. Here, we show that complex tissue structures can be reassembled from the gene expression profiles of dissociated cells. To achieve this, we developed LUNA, a generative AI model that reconstructs tissues conditioned solely on gene expressions of cells by learning spatial priors over existing spatially resolved datasets. We show that LUNA effectively reconstructs slices from the MERFISH whole mouse brain atlas with over 1.2 million cells, including cells from cell types never seen during model training. Applying LUNA to the mouse central nervous system scRNA-seq atlas, we show that LUNA is applicable for de novo generation of tissue structures. Additionally, LUNA can infer locations of nuclei lost during cell profiling with Slide-tags technology and correctly places cells belonging to spatially distinct compartments in a human metastatic melanoma sample. We envision that AI-driven tissue reassembly can help to overcome current technological limitations and advance our understanding of tissue organization and function, paving the way towards virtual tissue models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2575a743b02c9647eeddfbc3632592d6900d90aa" target='_blank'>
              Tissue reassembly with generative AI
              </a>
            </td>
          <td>
            Tingyang Yu, C. Ekbote, Nikita Morozov, Jiashuo Fan, Pascal Frossard, Stéphane d’Ascoli, M. Brbić
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Human Immunodeficiency Virus (HIV) is one of the most prevalent viruses, causing significant immune depletion in affected individuals. Current treatments can control HIV and prolong patients’ lives, but new challenges have emerged. Increasing incidence of cancers occur in HIV patients. Esophageal squamous cell carcinoma (ESCC) is one of the most common cancers observed in HIV patients. However, the spatial cellular characteristics of HIV-related ESCC have not been explored, and the differences between HIV-ESCC and typical ESCC remain unclear. Methods We performed spatial transcriptome sequencing on HIV-ESCC samples to depict the microenvironment and employed cell communication analysis and multiplex immunofluorescence to investigate the molecular mechanism in HIV-ESCC. Results We found that HIV-ESCC exhibited a unique cellular composition, with fibroblasts and epithelial cells intermixed throughout the tumor tissue, lacking obvious spatial separation, while other cell types were sparse. Besides, HIV-ESCC exhibited an immune desert phenotype, characterized by a low degree of immune cell infiltration, with only a few SPP1+ macrophages showing immune resistance functions. Cell communication analysis and multiplex immunofluorescence staining revealed that tumor fibroblasts in HIV-ESCC interact with CD44+ epithelial cells via COL1A2, promoting the expression of PIK3R1 in epithelial cells. This interaction activates the PI3K-AKT signaling pathway, which contributes to the progression of HIV-ESCC. Conclusions Our findings depict the spatial microenvironment of HIV-ESCC and elucidate a molecular mechanism in the progression of HIV-ESCC. This will provide us insights into the molecular basis of HIV-ESCC and potential treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02248-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba81ba07e36306408c53ef4726b3e3e8f09de47b" target='_blank'>
              Spatial transcriptomic analysis of tumor microenvironment in esophageal squamous cell carcinoma with HIV infection
              </a>
            </td>
          <td>
            Yuanli Zuo, Yang Jin, Gang Li, Yue Ming, Ting Fan, Yitong Pan, Xiaojun Yao, Yong Peng
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Spatial transcriptomics technology remains resource-intensive and unlikely to be routinely adopted for patient care soon. This hinders the development of novel precision medicine solutions and, more importantly, limits the translation of research findings to patient treatment. Here, we present DeepSpot, a deep-set neural network that leverages recent foundation models in pathology and spatial multi-level tissue context to effectively predict spatial transcriptomics from H&E images. DeepSpot substantially improved gene correlations across multiple datasets from patients with metastatic melanoma, kidney, lung, or colon cancers as compared to previous state-of-the-art. Using DeepSpot, we generated 1 792 TCGA spatial transcriptomics samples (37 million spots) of the melanoma and renal cell cancer cohorts. We anticipate this to be a valuable resource for biological discovery and a benchmark for evaluating spatial transcriptomics models. We hope that DeepSpot and this dataset will stimulate further advancements in computational spatial transcriptomics analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e5c9f327828cd33b4c0fe8505c7428b8e43eb60" target='_blank'>
              DeepSpot: Leveraging Spatial Context for Enhanced Spatial Transcriptomics Prediction from H&E Images
              </a>
            </td>
          <td>
            Kalin Nonchev, Sebastian Dawo, Karina Silina, Holger Moch, Sonali Andani, Tumor Profiler Consortium, Viktor H Koelzer, Gunnar R¨atsch
          </td>
          <td>2025-02-12</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Spatial transcriptomics provides insights into tissue architecture by linking gene expression with spatial localization. Current deconvolution methods rely heavily on single-cell RNA sequencing (scRNA-seq) references, which are costly and often unavailable, mainly if the tissue under evaluation is limited, such as in a core biopsy specimen. We present a novel tool, CITEgeist, that deconvolutes spatial transcriptomics data using antibody capture from the same slide as the reference, directly leveraging cell surface protein measurements from the same tissue section. This approach circumvents the limitations of scRNA-seq as a reference, offering a cost-effective and biologically grounded alternative. Our method employs mathematical optimization to estimate cell type proportions and gene expression profiles, incorporating sparsity constraints for robustness and interpretability. Benchmarks against state-of-the-art deconvolution methods show improved accuracy in cell type resolution, particularly in dense tumor microenvironments, while maintaining computational efficiency. This antibody-based tool advances spatial transcriptomics by providing a scalable, accurate, and reference-independent solution for deconvolution in complex tissues. We validate this tool by using a combined approach of simulated data and clinical samples by applying CITEgeist to translational pre-treatment and post-treatment ER+ breast tumors from an ongoing clinical trial, emphasizing the applicability and robustness of CITEgeist.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4308d97c43b1d495c969105587ab2b638fc0a951" target='_blank'>
              CITEgeist: Cellular Indexing of Transcriptomes and Epitopes for Guided Exploration of Intrinsic Spatial Trends
              </a>
            </td>
          <td>
            A. Chang, Brent T. Schlegel, N. Carleton, Priscilla F McAuliffe, S. Oesterreich, Russell Schwartz, Adrian V. Lee
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6fee264333ab195499909e9bd27046249a7701" target='_blank'>
              STmiR: A Novel XGBoost-Based Framework for Spatially Resolved miRNA Activity Prediction in Cancer Transcriptomics
              </a>
            </td>
          <td>
            Jiaqi Yuan, Peng Xu, Zheng Ye, Wenbin Liu
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51f0d69b3ba41ef9a508aa6acb9b650626ef267d" target='_blank'>
              MorphoITH: A Framework for Deconvolving Intra-Tumor Heterogeneity Using Tissue Morphology
              </a>
            </td>
          <td>
            Aleksandra Weronika Nielsen, H. E. Manoochehri, Hua Zhong, Vandana Panwar, V. Jarmale, Jay Jasti, Mehrdad Nourani, Dinesh Rakheja, J. Brugarolas, Payal Kapur, Satwik Rajaram
          </td>
          <td>2025-02-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background The identification of the complex spatial architecture of immune cell infiltration and its interaction mechanisms within tumor ecosystems provides crucial insights into therapeutic responses to neoadjuvant therapy in muscle-invasive bladder cancer (MIBC). This study aims to characterize the spatial features of distinct cell-type niches within the tumor microenvironment (TME) of patients with varying responses to neoadjuvant therapy. Methods We performed spatial transcriptomic profiling on six MIBC specimens obtained from a registered clinical trial (ChiCTR2000032359), generating whole-transcriptome spatial atlases to map the TME architecture. High-throughput analytical frameworks were employed to deconstruct the TME, and key findings were validated through immunohistochemistry and mouse model experiments. Results Our analysis revealed that tissues from complete responders exhibited greater infiltration of T and B cells, with the formation of tertiary lymphoid structure (TLS). Trajectory analysis identified CCL19/CCL21 as the key signaling molecules driving TLS formation in MIBC. Mouse experiments demonstrated that recombinant CCL19/CCL21 protein injections promoted intratumoral TLS formation and enhance the efficacy of immunotherapy. Furthermore, we observed significant intrinsic heterogeneity within individual tumors, which may contribute to the lack of therapeutic efficacy in MIBC. Conclusions This study underscores the critical role of TLS formation in the response to neoadjuvant therapy in MIBC. We identified CCL19/CCL21 as key drivers of TLS formation within MIBC tumors and potential immune-sensitizing agents. Additionally, the intrinsic heterogeneity of tumor should be considered a significant factor influencing therapeutic efficacy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06358-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d2414e88dc899f0c1cdec00aa234fe0cea16d7" target='_blank'>
              Spatial relationships and interactions of immune cell niches are linked to the pathologic response of muscle-invasive bladder cancer to neoadjuvant therapy
              </a>
            </td>
          <td>
            Wasilijiang Wahafu, Quan Zhou, Xihua Yang, Yongming Yang, Yuanyuan Zhao, Zhu Wang, Xiangpeng Kang, Xiongjun Ye, N. Xing
          </td>
          <td>2025-03-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a powerful tool for characterizing tumor heterogeneity, yet accurately identifying malignant cells remains challenging. Here, we propose scMalignantFinder, a machine learning tool specifically designed to distinguish malignant cells from their normal counterparts using a data- and knowledge-driven strategy. To develop the tool, multiple cancer datasets were collected, and the initially annotated malignant cells were calibrated using nine carefully curated pan-cancer gene signatures, resulting in over 400,000 single-cell transcriptomes for training. The union of differentially expressed genes across datasets was taken as the features for model construction to comprehensively capture tumor transcriptional diversity. scMalignantFinder outperformed existing automated methods across two gold-standard and eleven patient-derived scRNA-seq datasets. The capability to predict malignancy probability empowers scMalignantFinder to capture dynamic characteristics during tumor progression. Furthermore, scMalignantFinder holds the potential to annotate malignant regions in tumor spatial transcriptomics. Overall, we provide an efficient tool for detecting heterogeneous malignant cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a192a3c85f5ce13a8130114121b674f869202937" target='_blank'>
              scMalignantFinder distinguishes malignant cells in single-cell and spatial transcriptomics by leveraging cancer signatures
              </a>
            </td>
          <td>
            Qiaoni Yu, Yuan-Yuan Li, Yunqin Chen
          </td>
          <td>2025-03-27</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intratumor transcriptional heterogeneity (ITTH) presents a major challenge in cancer treatment, particularly due to the limited understanding of the diverse malignant cell populations and their relationship to therapy resistance. While single-cell sequencing has provided valuable insights into tumor composition, its high cost and technical complexity limits its use for large-scale tumor screening. In contrast, several databases collecting bulk RNA sequencing from multiple samples across various cancer types are available and could be used to profile ITTH. Several deconvolution approaches have been developed to infer cellular composition from such data. However, most of these methods rely on predefined markers or reference datasets, limiting the performance of such methods by the quality of used reference data. On the other hand, unsupervised approaches do not face such limitations, but existing methods have not been specifically adapted to characterize malignant cell states, focusing instead on general cell types. To address these gaps, we introduce CDState, an unsupervised method for inferring malignant cell subpopulations from bulk RNA-seq data. CDState utilizes a Nonnegative Matrix Factorization (NMF) model improved with sum-to-one constraints and a cosine similarity-based optimization to deconvolve bulk gene expression into distinct cell state-specific profiles, and estimate the abundance of each state across tumor samples. We validate CDState using bulkified single-cell RNA-seq data from five cancer types, showing that it outperforms existing unsupervised deconvolution methods in both cell state proportions and gene expression estimation. Applying CDState to 33 cancer types from the TCGA, we identified recurrent malignant cell programs, including epithelial-mesenchymal transition (EMT) and hypoxia as main drivers of tumor transcriptional heterogeneity. We further link the identified malignant states to patient clinical features, revealing states associated with worse patient prognosis. Finally, we find that alterations in genes such as KRAS or TP53, whose copy number or mutation status strongly correlate with specific malignant states, may play a key role in driving these cellular states. Overall, CDState provides a powerful and accessible approach to resolving intratumor heterogeneity using bulk RNA-seq data and emerges as a promising tool for advancing our understanding of malignant cell heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f16cea30bc9f4619225da6166ac6508ace6b994e" target='_blank'>
              CDState: an unsupervised approach to predict malignant cell heterogeneity in tumor bulk RNA-sequencing data
              </a>
            </td>
          <td>
            Agnieszka Kraft, Josephine Yates, Valentina Boeva
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell transcriptomic studies have identified distinct microglial subpopulations with shared and divergent gene signatures across development, aging and disease. Whether these microglial subsets represent ontogenically separate lineages of cells, or they are manifestations of plastic changes of microglial states downstream of some converging signals is unknown. Furthermore, despite the well-established role of enhancer landscapes underlying the identity of microglia, to what extent histone modifications and DNA methylation regulate microglial state switches at enhancers have not been defined. Here, using genetic fate mapping, we demonstrate the common embryonic origin of proliferative-region-associated microglia (PAM) enriched in developing white matter, and track their dynamic transitions into disease-associated microglia (DAM) and white matter-associated microglia (WAM) states in disease and aging contexts, respectively. This study links spatiotemporally discrete microglial states through their transcriptomic and epigenomic plasticity, while revealing state-specific histone modification profiles that govern state switches in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f71e88e7c209aa91ac01a49896d75e72d7b341cd" target='_blank'>
              Microglial plasticity governed by state-specific enhancer landscapes
              </a>
            </td>
          <td>
            Nicole Hamagami, Dvita Kapadia, Nora Abduljawad, Zuolin Cheng, Liam McLaughlin, Darsh Singhania, Kia M. Barclay, Jin Yang, Zhixin Sun, Peter Bayguinov, Guoqiang Yu, Harrison W. Gabel, Qingyun Li
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ae4e544698f6ef28ee1a58712b3dace0b7874a7" target='_blank'>
              A data-informed approach for engineering in-vitro experiment design to decipher key features of invasive breast cancer cell phenotypes
              </a>
            </td>
          <td>
            Lekha Shah, V. Breschi, Annalisa Tirella
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb1760a6c58270791a1de56692df43805b36e2d0" target='_blank'>
              Single Cell RNA Sequencing Analysis of Human Melanoma Reveals A Distinct Prognostic Myeloid Cell
              </a>
            </td>
          <td>
            M. U. Ahmad, Lyong Heo, Ramesh V. Nair, Saurabh Sharma, E. Maverakis, Amanda R. Kirane
          </td>
          <td>2025-03-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common malignant tumors of the digestive tract, with increasing incidence and mortality rates, posing a significant burden on human health. Its progression relies on various mechanisms, among which the tumor microenvironment and tumor-associated macrophages (TAMs) have garnered increasing attention. Macrophage infiltration in various solid tumors is associated with poor prognosis and is linked to chemotherapy resistance in many cancers. These significant biological behaviors depend on the heterogeneity of macrophages. Tumor-promoting TAMs comprise subpopulations characterized by distinct markers and unique transcriptional profiles, rendering them potential targets for anticancer therapies through either depletion or reprogramming from a pro-tumoral to an anti-tumoral state. Single-cell RNA sequencing technology has significantly enhanced our research resolution, breaking the traditional simplistic definitions of macrophage subtypes and deepening our understanding of the diversity within TAMs. However, a unified elucidation of the nomenclature and molecular characteristics associated with this diversity remains lacking. In this review, we assess the application of conventional macrophage polarization subtypes in colorectal malignancies and explore several unique subtypes defined from a single-cell omics perspective in recent years, categorizing them based on their potential functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff246fd6cb059fea5f5e9204ca830e7b6e029678" target='_blank'>
              Characterizing macrophage diversity in colorectal malignancies through single-cell genomics
              </a>
            </td>
          <td>
            Tingshuo Zhao, Yinyi Luo, Yuanjie Sun, Zhigang Wei
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Growing evidence supports the importance of characterizing the organizational patterns of various cellular constituents in the tumor microenvironment in precision oncology. Most existing data on immune cell infiltrates in tumors, which are based on immune cell counts or nearest neighbor-type analyses, have failed to fully capture the cellular organization and heterogeneity. Methods We introduce a computational algorithm, termed Tumor-Immune Partitioning and Clustering (TIPC), that jointly measures immune cell partitioning between tumor epithelial and stromal areas and immune cell clustering versus dispersion. As proof-of-principle, we applied TIPC to a prospective cohort incident tumor biobank containing 931 colorectal carcinoma cases. TIPC identified tumor subtypes with unique spatial patterns between tumor cells and T lymphocytes linked to certain molecular pathologic and prognostic features. T lymphocyte identification and phenotyping were achieved using multiplexed (multispectral) immunofluorescence. In a separate hepatocellular carcinoma cohort, we replaced the stromal component with specific immune cell types—CXCR3+CD68+ or CD8+—to profile their spatial relationships with CXCL9+CD68+ cells. Results Six unsupervised TIPC subtypes based on T lymphocyte distribution patterns were identified, comprising two cold and four hot subtypes. Three of the four hot subtypes were associated with significantly longer colorectal cancer (CRC)-specific survival compared to a reference cold subtype. Our analysis showed that variations in T-cell densities among the TIPC subtypes did not strictly correlate with prognostic benefits, underscoring the prognostic significance of immune cell spatial patterns. Additionally, TIPC revealed two spatially distinct and cell density-specific subtypes among microsatellite instability-high colorectal cancers, indicating its potential to upgrade tumor subtyping. TIPC was also applied to additional immune cell types, eosinophils and neutrophils, identified using morphology and supervised machine learning; here two tumor subtypes with similarly low densities, namely ‘cold, tumor-rich’ and ‘cold, stroma-rich’, exhibited differential prognostic associations. Lastly, we validated our methods and results using The Cancer Genome Atlas colon and rectal adenocarcinoma data (n = 570). Moreover, applying TIPC to hepatocellular carcinoma cases (n = 27) highlighted critical cell interactions like CXCL9-CXCR3 and CXCL9-CD8. Conclusions Unsupervised discoveries of microgeometric tissue organizational patterns and novel tumor subtypes using the TIPC algorithm can deepen our understanding of the tumor immune microenvironment and likely inform precision cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95d3eb1387144868bd261357332833886cbc29f0" target='_blank'>
              Tumor-immune partitioning and clustering algorithm for identifying tumor-immune cell spatial interaction signatures within the tumor microenvironment
              </a>
            </td>
          <td>
            M. Lau, J. Borowsky, J. Väyrynen, K. Haruki, Melissa Zhao, Andressa Dias Costa, Simeng Gu, Annacarolina da Silva, Tomotaka Ugai, Kota Arima, Minh N Nguyen, Yasutoshi Takashima, Joe Yeong, D. Tai, Tsuyoshi Hamada, Jochen K Lennerz, Charles S Fuchs, Catherine J Wu, J. Meyerhardt, S. Ogino, J. Nowak
          </td>
          <td>2025-02-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="Cancers display cellular, genetic and epigenetic heterogeneity, complicating disease modeling. Multiple cell states defined by gene expression have been described in lung adenocarcinoma (LUAD). However, the functional contributions of cell state and the regulatory programs that control chromatin and gene expression in the early stages of tumor initiation are not well understood. Using single-cell RNA and ATAC sequencing in Kras/p53-driven tumor organoids, we identified two major cellular states: one more closely resembling alveolar type 2 (AT2) cells (SPC-high), and the other with epithelial-mesenchymal-transition (EMT)-associated gene expression (Hmga2-high). Each state exhibited distinct transcription factor networks, with SPC-high cells associated with TFs regulating AT2 fate and Hmga2-high cells enriched in Wnt- and NFκB-related TFs. CD44 was identified as a marker for the Hmga2-high state, enabling functional comparison of the two populations. Organoid assays and orthotopic transplantation revealed that SPC-high, CD44-negative cells exhibited higher tumorigenic potential within the lung microenvironment. These findings highlight the utility of organoids in understanding chromatin regulation in early tumorigenesis and identifying novel early-stage therapeutic targets in Kras-driven LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729ab293566d42d02b2188c02492d9921394583a" target='_blank'>
              Organoid modeling reveals the tumorigenic potential of the alveolar progenitor cell state
              </a>
            </td>
          <td>
            Jingyun Li, Susanna M Dang, Shreoshi Sengupta, Paul Schurmann, A. F. Dost, Aaron L. Moye, M. F. Trovero, Sidrah Ahmed, Margherita Paschini, Preetida J Bhetariya, Roderick Bronson, Shannan J. Ho Sui, Carla F. Kim
          </td>
          <td>2025-02-10</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Characterizing the variety of cell types in the tumor microenvironment (TME) and their organization into cellular communities is vital for elucidating the biological diversity of cancer and informing therapeutic strategies. Methods Here, we employed a machine learning-based algorithm framework, EcoTyper, to analyze single-cell transcriptomes from 139 patients with head and neck squamous cell carcinoma (HNSC)and gene expression profiles from 983 additional HNSC patients, aiming to delineate the fundamental cell states and ecosystems integral to HNSC. Results A diverse landscape of 66 cell states and 9 ecosystems within the HNSC microenvironment was identified, revealing classical cell types while also expanding upon previous immune classifications. Survival analysis revealed that specific cell states and ecotypes (ecosystems) are associated with patient prognosis, underscoring their potential as indicators of clinical outcomes. Moreover, distinct cell states and ecotypes exhibited varying responses to immunotherapy and chemotherapy, with several showing promise as predictive biomarkers for treatment efficacy. Conclusion Our large-scale integrative transcriptome analysis provides high-resolution insights into the cellular states and ecosystems of HNSC, facilitating the discovery of novel biomarkers and supporting the development of precision therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06299-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd5809c9e6b6aaac22c97a1616ff68cb7ac8d0d" target='_blank'>
              High-dimensional deconstruction of HNSC reveals clinically distinct cellular states and ecosystems that are associated with prognosis and therapy response
              </a>
            </td>
          <td>
            Lei Xiao, Zhe Shen, Zhaoyu Pan, Yuanzheng Qiu, Donghai Huang, Yong Liu, Chao Liu, Xin Zhang
          </td>
          <td>2025-03-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous tumor, with significant variation in malignant cells, posing challenges for treatment and prognosis. However, this heterogeneity offers opportunities for personalized therapy. Methods The consensus non-negative matrix factorization algorithm was employed to analyze single-cell transcriptomic data from CRC, which helped identify malignant cell expression programs (MCEPs). Subsequently, a crosstalk network linking MCEPs with immune/stromal cell trajectory development was constructed using Monocle3 and NicheNet. Additionally, bulk RNA-seq data were utilized to systematically explore the relationships between MCEPs, clinical features, and genetic mutations. A prognostic model was then established through Lasso and Cox regression analyses, integrating clinical data into a nomogram for personalized risk prediction. Furthermore, key genes associated with MCEPs and their potential therapeutic targets were identified using protein-protein interaction networks, followed by molecular docking to predict drug-binding affinity. Results We classified CRC malignant cell transcriptional states into eight distinct MCEPs and successfully constructed crosstalk networks between these MCEPs and immune or stromal cells. A prognostic model containing 15 genes was developed, demonstrating an AUC greater than 0.8 for prognostic evaluation over 1 to 10 years when combined with clinical features. A key drug target gene TIMP1 was identified, and several potential targeted drugs were discovered. Conclusion This study demonstrated that characterization of the malignant cell transcriptional programs could effectively reveal the biological features of highly heterogeneous tumors like CRC and exhibit significant potential in tumor prognosis assessment. Our research provides new theoretical and practical directions for CRC prognosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Immune evasion by cancer cells involves reshaping the tumor microenvironment (TME) via communication with non-malignant cells. However, resistance-promoting interactions during treatment remain lesser known. Here we examine the composition, communication, and phenotypes of tumor-associated cells in serial biopsies from stage II and III high-risk estrogen receptor positive (ER+ ) breast cancers of patients receiving endocrine therapy (letrozole) as single agent or in combination with ribociclib, a CDK4/6-targeting cell cycle inhibitor. Single-cell RNA sequencing analyses on longitudinally collected samples show that in tumors overcoming the growth suppressive effects of ribociclib, first cancer cells upregulate cytokines and growth factors that stimulate immune-suppressive myeloid differentiation, resulting in reduced myeloid cell- CD8 + T-cell crosstalk via IL-15/18 signaling. Subsequently, tumors growing during treatment show diminished T-cell activation and recruitment. In vitro, ribociclib does not only inhibit cancer cell growth but also T cell proliferation and activation upon co-culturing. Exogenous IL-15 improves CDK4/6 inhibitor efficacy by augmenting T-cell proliferation and cancer cell killing by T cells. In summary, response to ribociclib in stage II and III high-risk ER +  breast cancer depends on the composition, activation phenotypes and communication network of immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25a85c2bcf9c97b3611062a72118d56273a4ef13" target='_blank'>
              Cellular interactions within the immune microenvironment underpins resistance to cell cycle inhibition in breast cancers
              </a>
            </td>
          <td>
            Jason I. Griffiths, Patrick A. Cosgrove, Eric F. Medina, Aritro Nath, Jinfeng Chen, Fred Adler, Jeffrey T. Chang, Qamar J Khan, Andrea H Bild
          </td>
          <td>2025-03-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5782e67aa86bd64459296bc264a422b250cba97" target='_blank'>
              SCOIGET: Predicting Spatial Tumor Evolution Pattern by Inferring Spatial Copy Number Variation Distributions
              </a>
            </td>
          <td>
            Yujia Zhang, Yitao Yang, Yan Kong, Bingxu Zhong, Kenta Nakai, Hui Lu
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Understanding the intricate tumor microenvironment (TME) is crucial for elucidating the mechanisms underlying the progression of cervical squamous cell carcinoma (CSCC) and its response to anti-PD-1 therapy. Methods In this study, we characterized 50,649 cells obtained from the CSCC for single-cell RNA sequencing and integrated bulk sequencing data from The Cancer Genome Atlas (TCGA) and clinical samples to explore their cell composition, metabolic processes, signaling pathways, specific transcription factors, lineage tracking and response to immunotherapy. In vivo experiments were performed to validate the function of key cell subsets. Results We identified ten major cell type and 35 subsets of stromal and immune cells in TME and observed distinct patterns in the metabolic processes and signaling pathways of these cells between tumor and normal tissues. Furthermore, PCNA clamp-associated factor (PCLAF)+ tumor-associated epithelial cell (TAEpis) was negatively correlated with the number of C-X-C motif chemokine ligand 13 (CXCL13)+ CD8+ T cells, overall survival, and response to anti-programmed cell death-1(PD-1) therapy in patients with CSCC. Both in vivo and in vitro experiments demonstrated that PCLAF+ TAEpis promotes the apoptosis of CD8+ T and tumor growth, while also inhibiting T cell infiltration and function. Conclusion Our findings illuminate the heterogeneity of the complex TME in CSCC and offer evidence supporting PCLAF+ TAEpis as a promising therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03725-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) has advanced our understanding of tissue regionalization by enabling the visualization of gene expression within whole-tissue sections, but current approaches remain plagued by the challenge of achieving single-cell resolution without sacrificing whole-genome coverage. Here we present Spotiphy (spot imager with pseudo-single-cell-resolution histology), a computational toolkit that transforms sequencing-based ST data into single-cell-resolved whole-transcriptome images. Spotiphy delivers the most precise cellular proportions in extensive benchmarking evaluations. Spotiphy-derived inferred single-cell profiles reveal astrocyte and disease-associated microglia regional specifications in Alzheimer's disease and healthy mouse brains. Spotiphy identifies multiple spatial domains and alterations in tumor-tumor microenvironment interactions in human breast ST data. Spotiphy bridges the information gap and enables visualization of cell localization and transcriptomic profiles throughout entire sections, offering highly informative outputs and an innovative spatial analysis pipeline for exploring complex biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/022fdf3eed7c33ba165a376f1b3fb14ad94870ef" target='_blank'>
              Spotiphy enables single-cell spatial whole transcriptomics across an entire section.
              </a>
            </td>
          <td>
            Jiyuan Yang, Ziqian Zheng, Yun Jiao, Kaiwen Yu, Sheetal Bhatara, Xu Yang, Sivaraman Natarajan, Jiahui Zhang, Qingfei Pan, John Easton, Koon-Kiu Yan, Junmin Peng, Kaibo Liu, Jiyang Yu
          </td>
          <td>2025-03-12</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung cancer is a leading cause of cancer-related mortality worldwide, largely due to its heterogeneity and intrinsic drug resistance. Malignant pleural effusions (MPEs) provide diverse tumor cell populations ideal for studying these complexities. Although chemotherapy and targeted therapies can be initially effective, subpopulations of cancer cells with phenotypic plasticity often survive treatment, eventually developing resistance. Here, we integrated single-cell isolation and three-dimensional (3D) spheroid culture to dissect subclonal heterogeneity and drug responses, aiming to inform precision medicine approaches. Using A549 lung cancer cells, we established a cisplatin-resistant line and isolated three resistant subclones (Holoclone, Meroclone, Paraclone) via single-cell sorting. In 3D spheroids, Docetaxel and Alimta displayed higher IC50 values than in 2D cultures, suggesting that 3D models better reflect clinical dosing. Additionally, MPE-derived Holoclone and Paraclone subclones exhibited distinct sensitivities to Giotrif and Capmatinib, revealing their heterogeneous drug responses. Molecular analyses confirmed elevated ABCB1, ABCG2, cancer stem cell (CSC) markers (OCT4, SOX2, CD44, CD133), and epithelial–mesenchymal transition (EMT) markers (E-cadherin downregulation, increased Vimentin, N-cadherin, Twist) in resistant subclones, correlating with enhanced migration and invasion. This integrated approach clarifies the interplay between heterogeneity, CSC/EMT phenotypes, and drug resistance, providing a valuable tool for predicting therapeutic responses and guiding personalized, combination-based lung cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6813439ca1c219dd03651dca35df5520579d3204" target='_blank'>
              Application of an Integrated Single-Cell and Three-Dimensional Spheroid Culture Platform for Investigating Drug Resistance Heterogeneity and Epithelial–Mesenchymal Transition (EMT) in Lung Cancer Subclones
              </a>
            </td>
          <td>
            Shin-Hu Chen, Jian-Hong Yu, Yu-Chun Lin, Yi-Ming Chang, Nien-Tzu Liu, Su-Feng Chen
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Neurodegenerative diseases and infections can produce lasting effects on brain function, yet the spatial molecular mechanisms underlying these changes remain poorly understood. Here, we present high-resolution spatial transcriptomics of 40 postmortem brain samples from patients with Parkinson’s disease, frontotemporal dementia, dementia with Lewy bodies, and severe COVID-19. Analyzing over 1.5 million spatially resolved cells across dorsolateral prefrontal cortex and anterior cingulate cortex revealed disease-specific transcriptional signatures with pronounced layer-and region-specificity. In Parkinson’s disease, we identified stressed neurons creating distinctive microenvironmental gradients where metabolic and protein degradation pathways are elevated near stress epicenters, while regenerative processes increase with distance. COVID-19 brains displayed extensive peripheral immune cell infiltration, particularly in the subcortical white matter, accompanied by compromised blood-brain barrier and coordinated neuroinflammatory responses from microglia, astrocytes, and endothelial cells. Integration of miRNA sequencing with spatial transcriptomics uncovered layer-specific regulatory patterns, including neuroinflammation-associated miR-155. This atlas provides unprecedented insights into disease pathology and highlights the critical importance of spatial molecular context in understanding brain disorders. Key Messages [1] A high-resolution single-cell spatial transcriptomics atlas of the dorsolateral prefrontal cortex and anterior cingulate cortex across neurodegenerative conditions and severe COVID-19 [2] Region-and layer-specific transcriptional dysregulation across disease comparisons reveals disease-specific differential vulnerability [3] Metabolically stressed cells found selectively in the anterior cingulate cortex of Parkinson’s disease patients but not in dementia with Lewy bodies, along with detailed characterization of their spatial microenvironment [4] Peripheral immune cell clusters identified in the white matter of the cerebral cortex of COVID-19 patients, with detailed characterization of their spatial microenvironment [5] Integration of bulk miRNA sequencing reveals cortical layer-specific miRNA regulatory patterns">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97c071df2a109d09f7501e38a14ec4947fe8e317" target='_blank'>
              Single-cell Spatial Transcriptomics Reveals Disease-specific Microenvironmental Niches in Neurodegeneration and COVID-19
              </a>
            </td>
          <td>
            Simon Graf, N. Ludwig, Matthias Flotho, Annika Engel, I. F. Diks, Paula Zimmer, Wiebke M. Jürgens-Wemheuer, Arne Wrede, Daniela Mirzac, G. González-Escamilla, Victoria Wagner, Vilas Menon, Sergiu Groppa, P. D. De Jager, Lai Yiwei, Miguel A. Esteban, Walter J. Schulz-Schaeffer, Andreas Keller
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Background Cancer development is closely associated with complex alterations in the tumor microenvironment (TME). Among these, immune cells within the TME play a huge role in personalized tumor diagnosis and treatment. Objectives This review aims to summarize the diversity of immune cells in the TME, their impact on patient prognosis and treatment response, and the contributions of single‐cell RNA sequencing (scRNA‐seq) in understanding their functional heterogeneity. Methods We analyzed recent studies utilizing scRNA‐seq to investigate immune cell populations in the TME, focusing on their interactions and regulatory mechanisms. Results ScRNA‐seq reveals the functional heterogeneity of immune cells, enhances our understanding of their role in tumor antibody responses, and facilitates the construction of immune cell interaction networks. These insights provide guidance for the development of cancer immunotherapies and personalized treatment approaches. Conclusion Applying scRNA‐seq to immune cell analysis in the TME offers a novel pathway for personalized cancer treatment. Despite its promise, several challenges remain, highlighting the need for further advancements to fully integrate scRNA‐seq into clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36e80d2a964412d958098f4ad446c2f9779f2b7e" target='_blank'>
              Analyze the Diversity and Function of Immune Cells in the Tumor Microenvironment From the Perspective of Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Lujuan Ma, Yu Luan, Lin Lu
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Spatially resolved transcriptomics has transformed our ability to study complex tissues at the cellular and subcellular resolution. However, targeted spatial technologies require pre-selected gene panels, which are typically curated based on existing biological knowledge and prior research hypotheses. While current methods often prioritize capturing cell type information, we argue that an effective gene panel should also capture cell type diversity, cell states, pathway-level information, and minimize redundancy. To address these broader requirements, we developed a gene panel characterization platform that characterizes panels across multiple perspectives, thus allowing us to compare panels comprehensively. Notably, computationally constructed gene panels performed competitively in capturing major cell types when compared to our in-house manually curated panel. However, refined manual curation offered distinct advantages, particularly in capturing minor and rare cell types and exhibited lower information redundancy comparatively. Building on this framework, we integrated these metrics into a deep learning platform, panelScope, leveraging them as a loss function to design holistic gene panels. Using an acute myeloid leukemia (AML) dataset with 42 well-defined cell types and the 5K Xenium panel from 10X Genomics, we demonstrate the utility of our framework in comprehensively characterizing gene panels, enabling the design of tailored panels that address diverse research needs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef9065cee3317a4c66d56d6484e04cef47751f06" target='_blank'>
              Multi-view gene panel characterization for spatially resolved omics
              </a>
            </td>
          <td>
            Daniel Kim, Wenze Ding, Akira Nguyen Shaw, Marni Torkel, Cameron J Turtle, Pengyi Yang, J. Yang
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc383607243031294dc9e768566a894cfedd572c" target='_blank'>
              Glioblastoma stem cell morphotypes convey distinct cell states and clinically relevant functions
              </a>
            </td>
          <td>
            C. Barelli, Matteo Bonfanti, Filippo Mirabella, Dario Ricca, Negin Alizadehmohajer, Viktoria Sokolova, Alberto Campione, G. Sicuri, C. Peano, Stefania Faletti, Roberto Stefini, N. Kalebic
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent advances in spatial transcriptomics (ST) technologies have transformed our ability to profile gene expression while retaining the crucial spatial context within tissues. However, existing ST platforms suffer from high costs, long turnaround times, low resolution, limited gene coverage, and small tissue capture areas, which hinder their broad applications. Here we present iSCALE, a method that predicts super-resolution gene expression and automatically annotates cellular-level tissue architecture for large-sized tissues that exceed the capture areas of standard ST platforms. The accuracy of iSCALE were validated by comprehensive evaluations, involving benchmarking experiments, immunohistochemistry staining, and manual annotation by pathologists. When applied to multiple sclerosis human brain samples, iSCALE uncovered lesion associated cellular characteristics that were undetectable by conventional ST experiments. Our results demonstrate iSCALE’s utility in analyzing large-sized tissues with automatic and unbiased tissue annotation, inferring cell type composition, and pinpointing regions of interest for features not discernible through human visual assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca1037b8c19366713347f95c55ebd14edfcd843" target='_blank'>
              Scaling up spatial transcriptomics for large-sized tissues: uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE
              </a>
            </td>
          <td>
            Amelia Schroeder, Melanie Loth, Chunyu Luo, Sicong Yao, Hanying Yan, Daiwei Zhang, Sarbottam Piya, Edward Plowey, Wen-Tuo Hu, Jean R. Clemenceau, Inyeop Jang, Minji Kim, Isabel Barnfather, Su Jing Chan, Taylor L. Reynolds, Thomas Carlile, Patrick Cullen, Ji-Youn Sung, Hui-Hsin Tsai, Jeong Hwan Park, Tae Hyun Hwang, Baohong Zhang, Mingyao Li
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1221249520ea9d5b52e0d651f4686ac33466a610" target='_blank'>
              Staining-free Visualization of Cell Boundaries in Sliced Tissues for Accurate Binning of Gene Expression in Spatial Transcriptomics
              </a>
            </td>
          <td>
            Zhiming Zhang, Xiaoxia Chen, Youyuan Chen, Zixuan Liao, T. Liu
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff64eb12bd7d9a78fd223bedc136a7a4d7a78d40" target='_blank'>
              Abstract B068: A novel murine model for interrogating immune response to tumor heterogeneity in non-small cell lung cancer
              </a>
            </td>
          <td>
            M. Oh, J. Abascal, L. Tran, R. Salehi-Rad, R. Lim, C. Dumitras, Cara E. Yean, Sunny Xiao, Bitta Kahangi, William P Crosson, E. P. Reyes, Nalani Coleman, Austin K. Rennels, K. Krysan, Bin Liu, Steven M Dubinett
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12d9a96ad448410a624d4884f42c080b637c2227" target='_blank'>
              Whole-slide multiplexed tissue imaging of prostate adenocarcinoma to investigate immune niches in high- versus low-grade tumors.
              </a>
            </td>
          <td>
            Ali Amiryousefi, Jeremiah Wala, B. Labadie, Aishwarya Atmakuri, K. Chaudagar, Eamon Toye, Andréanne Gagné, Jia-Ren Lin, E. Van Allen, Akash Patnaik, P. Sorger
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cellular plasticity enables cancer cells to adapt non-genetically, thereby preventing therapeutic success. The epithelial-mesenchymal transition (EMT) is a type of plasticity linked to resistance and metastasis. However, its exact impact on population diversity and its dynamics under chemotherapy is unknown. We used single-cell transcriptomics to investigate phenotypic diversity dynamics upon treatment in two in vitro models of triple negative breast cancer (TNBC), where EMT-driven plasticity is either induced or spontaneously occurring. We report that EMT-driven plasticity confers higher phenotypic cell–cell variability (p < 0.001) while enriching for stem-like cells. Genetic and phenotypic cell–cell variability were not consistently correlated. High-plasticity populations displayed more pre-adapted cells before treatment (p = 0.03). In a population displaying spontaneous EMT and phenotypic variation, pre-adapted cells were a rare minority of high-scoring outliers whose expression patterns correlated with survival in TNBC patients subjected to chemotherapy (p = 0.03). Higher plasticity was not associated with a partial EMT status. Our results provide novel insights on how EMT-driven plasticity promotes a prospective diversification process increasing population phenotypic diversity, which can yield rare pre-adapted states before treatment. This highlights the need to tackle phenotypic diversity prior to treatment in high-plasticity tumours. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03637-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a529afcdab161e3ac0972bc45b1449999ddb519" target='_blank'>
              EMT-driven plasticity prospectively increases cell–cell variability to promote therapeutic adaptation in breast cancer
              </a>
            </td>
          <td>
            Lauriane Muller, Frédérique Fauvet, Christelle Chassot, Francesca Angileri, Angèle Coutant, Cyril Dégletagne, Laurie Tonon, P. Saintigny, Alain Puisieux, A. Morel, Maria Ouzounova, Pierre Martinez
          </td>
          <td>2025-02-03</td>
          <td>Cancer Cell International</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d948793b1aa3c6db446f2356cc375541bc62bfa6" target='_blank'>
              A Multimodal Framework to Uncover Drug-Responsive Subpopulations in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Yue Wang, Santiago Haase, Austin A. Whitman, Adriana Beltran, Philip M. Spanheimer, Elizabeth Brunk
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Gastric cancer is a highly aggressive malignancy characterized by a complex tumor microenvironment (TME). Cancer-associated fibroblasts (CAFs), which are a key component of the TME, exhibit significant heterogeneity and play crucial roles in tumor progression. Therefore, a comprehensive understanding of CAFs is essential for developing novel therapeutic strategies for gastric cancer. Methods This study investigates the characteristics and functional information of CAF subtypes and explores the intercellular communication between CAFs and malignant epithelial cells (ECs) in gastric cancer by analyzing single-cell sequencing data from 24 gastric cancer samples. CellChat was employed to map intercellular communication, and Seurat was used to integrate single-cell sequencing data with spatial transcriptome data to reconstruct a comprehensive single-cell spatial map. The spatial relationship between apCAFs and cancer cells was analyzed using multicolor immunohistochemistry. Results Cells were categorized into nine distinct categories, revealing a positive correlation between the proportions of epithelial cells (ECs) and fibroblasts. Furthermore, six fibroblast subpopulations were identified: inflammatory (iCAFs), pericytes, matrix (mCAFs), antigen-presenting (apCAFs), smooth muscle cells (SMCs), and proliferative CAFs (pCAFs). Each of these subpopulations was linked to various biological processes and immune responses. Malignant ECs exhibited heightened intercellular communication, particularly with CAF subpopulations, through specific ligand-receptor interactions. High-density regions of CAF subpopulations displayed spatial exclusivity, with pericytes serving as a source for iCAFs, mCAFs, and apCAFs. Notably, malignant ECs and apCAFs showed increased interactions, with certain ligand-receptor pairs potentially impacting the prognosis of gastric cancer. Multiplex immunohistochemistry (mIHC) confirmed the close spatial proximity of apCAFs to cancer cells in gastric cancer. Conclusion Our study provided a comprehensive characterization of CAF heterogeneity in gastric cancer and revealed the intricate intercellular networks within the TME. The identified CAF subpopulations and their interactions with malignant cells could serve as potential therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06376-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24163c4c446c34923a67d83b5c03139aa6a3cb4" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics reveal the heterogeneity and intercellular communication of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xijie Zhang, Bo Ren, Bo Liu, Rui Wang, Sen Li, Yuzhou Zhao, Wence Zhou
          </td>
          <td>2025-03-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e78c6ebe3a3bbb6920324a2995126ad2f48328da" target='_blank'>
              PatientProfiler: A network-based approach to personalized medicine
              </a>
            </td>
          <td>
            Veronica Lombardi, Lorenzo Di Rocco, Eleonora Meo, Veronica Venafra, Elena Di Nisio, Valerio Perticaroli, Mihail Lorentz Nicolaeasa, Chiara Cencioni, Francesco Spallotta, Rodolfo Negri, F. Sacco, Livia Perfetto
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) is a cutting-edge technology that provides comprehensive insights into gene expression patterns from a spatial perspective. A key research focus within this field is spatial domain identification, which is essential for exploring tissue organization, biological development, and disease mechanisms. Although methods have been developed, they still face challenges in modeling the gene expression information together with the spatial locations, resulting in suboptimal accuracy. We introduce DisConST (Distribution-aware Contrastive Learning for Spatial Transcriptomics), a novel deep-learning method designed to improve spatial domain detection within spatial transcriptomics datasets. DisConST addresses key challenges, such as the high dropout rates and the complex integration of spatial and gene expression data, by incorporating contrastive learning strategies that are aware of the underlying data distributions. It employs the zero-inflated negative binomial (ZINB) distribution, along with graph contrastive learning, to generate more informative latent representations. These representations efficiently integrate spatial positions, transcriptomic profiles, and cell-type proportions within spots. We validated DisConST across diverse ST datasets of tissues, organs, and embryos from various sequencing platforms in both normal and disease states. Our results consistently demonstrated that DisConST achieves superior spatial domain recognition accuracy compared to existing state-of-the-art methods. Furthermore, our experiments highlighted the utility of DisConST in advancing research on tissue organization, embryonic development, and tumor immune microenvironment dissection. The source code for DisConST is freely available at https://github.com/Zhenpm/DisConST/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745c9ac0a0fd2c2ed7fb914bb3fa37f78a95e73c" target='_blank'>
              DisConST: Deciphering Spatial Domains Using Distribution-aware Contrastive Learning for Spatial Transcriptomics
              </a>
            </td>
          <td>
            Peimeng Zhen, Xiaofeng Wang, Han Shu, Jialu Hu, Yongtian Wang, Jiajie Peng, Xuequn Shang, Jing Chen, Tao Wang
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58e073c2b3967f3720af1b89697ce0e1aad33c4c" target='_blank'>
              Abstract B086: Disease-critical macrophage populations interact with stroma to modulate the tumor microenvironment
              </a>
            </td>
          <td>
            Weiwei Zhong, Mitra Ansariola, Micaela Tatman, Rachel Wester, Kim Van Naarden Braun, Shameek Biswas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebeb9fa82784b82f60e0038c9b75808f64425555" target='_blank'>
              Merkel cell polyomavirus infection and persistence modelled in skin organoids
              </a>
            </td>
          <td>
            Silvia Albertini, Manja Czech‐Sioli, Thomas Günther, Sanamjeet Virdi, Patrick Blümke, Lisann Röpke, Veronika Brinschwitz, M. Dandri, L. Allweiss, Rudolf Reimer, Carola Schneider, Arne Hansen, S. Krasemann, E. Wyler, M. Landthaler, A. Grundhoff, Nicole Fischer
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Spatial transcriptomics facilitates gene expression analysis of cells in their spatial anatomical context. Batch effects hinder visualization of gene spatial patterns across samples. We present the Crescendo algorithm to correct for batch effects at the gene expression level and enable accurate visualization of gene expression patterns across multiple samples. We show Crescendo’s utility and scalability across three datasets ranging from 170,000 to 7 million single cells across spatial and single-cell RNA sequencing technologies. By correcting for batch effects, Crescendo enhances spatial transcriptomics analyses to detect gene colocalization and ligand-receptor interactions and enables cross-technology information transfer. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03479-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/807d24ffeaa1b756156a596b5f9b0d95ce824b53" target='_blank'>
              Batch correcting single-cell spatial transcriptomics count data with Crescendo improves visualization and detection of spatial gene patterns
              </a>
            </td>
          <td>
            N. Millard, Jonathan H Chen, Mukta G. Palshikar, K. Pelka, Maxwell Spurrell, Colles Price, Jiang He, N. Hacohen, Soumya Raychaudhuri, Ilya Korsunsky
          </td>
          <td>2025-02-25</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Significance Chemotherapeutics effectively kill cancer cells but can also induce mutations in surviving cells, leading to therapy resistance and secondary malignancies. We analyzed the DNA rearrangements caused by doxorubicin in murine HER2+ breast cancer organoids, focusing on genomic structural variants (SVs). Using single-cell sequencing, we revealed extensive therapy-induced SVs and genomic instability in a subset of posttreatment cells. Additionally, by advancing computational methods to analyze nucleosome occupancy (NO) in different cell types of the mammary lineage, we link SVs to gene expression changes that could potentially lead to breast cancer progression. This research underscores the need for strategies to minimize drug-induced genomic damage to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95047a549d97c358dede6e2ed75d74511cbd0c06" target='_blank'>
              Structural variant and nucleosome occupancy dynamics postchemotherapy in a HER2+ breast cancer organoid model
              </a>
            </td>
          <td>
            Maja Starostecka, Hyobin Jeong, Patrick Hasenfeld, Eva Benito-Garagorri, Tania Christiansen, Catherine Stober Brasseur, Maise Gomes Queiroz, Marta Garcia Montero, Martin Jechlinger, J. Korbel
          </td>
          <td>2025-02-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) are pivotal components of the breast cancer (BC) tumor microenvironment (TME), significantly influencing tumor progression and response to therapy. However, the heterogeneity and specific roles of TAM subpopulations in BC remain inadequately understood. Methods We performed an integrated analysis of single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data from BC patients to comprehensively characterize TAM heterogeneity. Utilizing the MetaTiME computational framework and consensus clustering, we identified distinct TAM subtypes and assessed their associations with clinical outcomes and treatment responses. A machine learning-based predictive model was developed to evaluate the prognostic significance of TAM-related gene expression profiles. Results Our analysis revealed three distinct TAM subgroups. Notably, we identified a novel macrophage subtype, M_Macrophage-SPP1-C1Q, characterized by high expression of SPP1 and C1QA, representing an intermediate differentiation state with unique proliferative and oncogenic properties. High infiltration of M_Macrophage-SPP1-C1Q was significantly associated with poor overall survival (OS) and chemotherapy resistance in BC patients. We developed a Random Forest (RF)-based predictive model, Macro.RF, which accurately stratified patients based on survival outcomes and chemotherapy responses, independent of established prognostic parameters. Conclusion This study uncovers a previously unrecognized TAM subtype that drives poor prognosis in BC. The identification of M_Macrophage-SPP1-C1Q enhances our understanding of TAM heterogeneity within the TME and offers a novel prognostic biomarker. The Macro.RF model provides a robust tool for predicting clinical outcomes and guiding personalized treatment strategies in BC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03750-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/287d3599b1ab22b44316058fde7d65a5ef96c17d" target='_blank'>
              Integrated single-cell and bulk transcriptomic analysis identifies a novel macrophage subtype associated with poor prognosis in breast cancer
              </a>
            </td>
          <td>
            Qing Wang, Yushuai Yu, Liqiong Ruan, Mingyao Huang, Wei Chen, Xiaomei Sun, Jun Liu, Zirong Jiang
          </td>
          <td>2025-03-27</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac4b19bdc51a77041e08cca430b7dd7ca4fef49" target='_blank'>
              Abstract PR007: Heterotypic CD8 T cell clusters isolated from clinical samples are distinct and enriched for antitumor activity
              </a>
            </td>
          <td>
            D. Peeper, Sofía Ibáñez-Molero, Johanna Veldman, Juan Simon-Nieto, J. Traets, Austin George, K. Hoefakker, Anita Karomi, Jonas Nilsson, John Haanen, W. V. van Houdt
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) represent a central cell population of the tumor microenvironment (TME). Recently, single‐cell RNA‐sequencing (scRNA‐seq) analyses of primary tumors of different cancer entities yielded different classifications of CAF subsets underscoring the heterogeneity of CAFs within the TME. Here, we analyzed the transcriptional signatures of approximately 8400 CAFs and normal fibroblasts by scRNA‐seq and compared genetic profiles of CAFs from murine melanoma primary tumors to CAFs from corresponding melanoma lung metastases. This revealed distinct subsets for primary tumor and metastasis‐specific CAF populations, respectively. Combined with the spatial characterization of metastasis CAFs at the RNA and protein level, scRNA analyses indicate tumor‐dependent crosstalk between neutrophils and CAFs, mediated via SAA3 and IL1b‐related signaling pathways, which can be recapitulated in vitro. Analyzing tissue sections of human patient samples, this interaction was found to be present in human melanoma metastasis. Taken together, our data highlight unique characteristics of metastasis CAFs with potential therapeutic impact for melanoma metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae4d264150cbc5dcfaa11bf8a1b1f48e7dcf511" target='_blank'>
              A Unique Signature for Cancer‐Associated Fibroblasts in Melanoma Metastases
              </a>
            </td>
          <td>
            Saskia Tauch, Joschka Hey, Bettina Kast, N. Gengenbacher, Lena Weiß, Melanie Sator‐Schmitt, S. Lohr, Alexander Brobeil, P. Schirmacher, Jochen Utikal, H. Augustin, C. Plass, Peter Angel
          </td>
          <td>2025-02-09</td>
          <td>Pigment Cell & Melanoma Research</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce737a0ad997a56048c24df55247b712e440df3d" target='_blank'>
              Abstract PR002: T cell engager therapy affects the spatial distribution and phenotype of T cells in the tumor microenvironment
              </a>
            </td>
          <td>
            Billy Tomaszewski, Matthew Curtis, Conrad Foo, Gu Zhang, Patrick S Chang, Robyn Clark, Thao Nguyen, Joshua Webster, Sandra Rost, Raj Jesudason, Robert Piskol, Klara Totpal, Lisa Mcginnis, Kevin B Walsh, Weilan Ye
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4672987b5633b49ff2536422175c3660e10c3a" target='_blank'>
              Identification of a shared persistence program in triple-negative breast cancer across treatments and patients
              </a>
            </td>
          <td>
            Léa Baudre, Grégoire Jouault, Pacôme Prompsy, Mélissa Saichi, Sarah Gastineau, Christophe Huret, Laura Sourd, A. Dahmani, Elodie Montaudon, F. Dingli, D. Loew, Elisabetta Marangoni, J. Marsolier, C. Vallot
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77e71e2d20d9e52879b8c2d8cc5014dd16b7a335" target='_blank'>
              SVCFit: Inferring structural variant cellular fraction in tumors
              </a>
            </td>
          <td>
            Yunzhou Liu, Jiaying Lai, Laura D. Wood, Rachel Karchin
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a68270f6302f38ef1495b09baefc46d92bf7cdcd" target='_blank'>
              Reconstructing signaling histories of single cells via perturbation screens and transfer learning
              </a>
            </td>
          <td>
            Nicholas T. Hutchins, Miram Meziane, Claire Lu, Maisam Mitalipova, David Fischer, Pulin Li
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Previous studies have highlighted the capacity of brain cancer cells to functionally interact with the tumour microenvironment (TME). This TME-cancer crosstalk crucially contributes to tumour cell invasion and disease progression. In this study, we performed spatial transcriptomic sequencing analysis of a complete annotated tumour-infiltrated brainstem from a single diffuse intrinsic pontine glioma (DIPG) patient. Gene signatures from ten sequential tumour regions were analysed to assess mechanisms of disease progression and oncogenic interactions with the TME. We identified four distinct tumour subpopulations and assessed respective ligand-receptor pairs that actively promote DIPG tumour progression via crosstalk with endothelial, neuronal and immune cell communities. Our analysis found potential targetable mediators of tumour-to-TME communication, including members of the complement component system and the neuropeptide/GPCR ligand-receptor pair ADCYAP1-ADCYAP1R1. These interactions could influence DIPG tumour progression and represent novel therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-01952-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ea420e4a0256eeeba7153fed74e4eb3b92c386" target='_blank'>
              Spatial analysis of a complete DIPG-infiltrated brainstem reveals novel ligand-receptor mediators of tumour-to-TME crosstalk
              </a>
            </td>
          <td>
            Anja Kordowski, Onkar Mulay, Xiao Tan, Tuan Vo, Ulrich Baumgartner, Mellissa Maybury, Timothy Hassall, L. Harris, Quan Nguyen, B. Day
          </td>
          <td>2025-02-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC.
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e047830d6d15c84fdba5a27411c84ea5a08197" target='_blank'>
              Multi-Modal Disentanglement of Spatial Transcriptomics and Histopathology Imaging
              </a>
            </td>
          <td>
            Hassaan Maan, Zongliang Ji, Elliot Sicheri, Tiak Ju Tan, Alina Selega, Ricardo Gonzalez, Rahul G Krishnan, Bo Wang, Kieran R. Campbell
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are highly heterogeneous cells and important components of the breast tumor microenvironment (TME). However, their role and clinical value in ER-positive breast cancer have not been fully clarified. Our study aims to comprehensively characterize the heterogeneity, potential biological functions, and molecular mechanisms of CAFs in ER-positive breast cancer within the tumor microenvironment using multi-omics data, to provide new strategies for the diagnosis and treatment of ER-positive breast cancer patients. In this study, we found that COL1A2(+) MMP1(+) and COL1A2(+) MMP1(-) CAFs were associated with unfavorable prognosis. The dynamic evolution and cell-cell communications of CAFs were analyzed, revealing that COL1A2(+) MMP1(+/-) CAFs show extensive crosstalk with tumor-associated macrophages (TAMs), contributing to an immunosuppressive TME. Moreover, the somatic mutation of TP53 may be a potential indicator for evaluating the infiltration of COL1A2(+) MMP1(+/-) CAFs. Finally, an MRI-based radiomic model was constructed to estimate the abundance of these CAFs. In conclusion, our findings provide a theoretical basis for targeting CAFs and offer a noninvasive approach to evaluate the infiltration level of COL1A2(+) MMP1(+/-) CAFs. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03705-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9547a039c0fe93efd8fd0787388be5b2aa0012a6" target='_blank'>
              Integrating spatial and single-cell transcriptomes reveals the role of COL1A2(+) MMP1(+/-) cancer-associated fibroblasts in ER-positive breast cancer
              </a>
            </td>
          <td>
            Zhi-Hao Yu, Huan-Ling Xu, Shuo Wang, Yingxi Li, Guixin Wang, Yao Tian, Zhaohui Chen, Wenbin Song, Long He, Xin Wang, Xuchen Cao, Yue Yu
          </td>
          <td>2025-03-07</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Spatial omics enable the characterization of colocalized cell communities that coordinate specific functions within tissues. These communities, or niches, are shaped by interactions between neighboring cells, yet existing computational methods rarely leverage such interactions for their identification and characterization. To address this gap, here we introduce NicheCompass, a graph deep-learning method that models cellular communication to learn interpretable cell embeddings that encode signaling events, enabling the identification of niches and their underlying processes. Unlike existing methods, NicheCompass quantitatively characterizes niches based on communication pathways and consistently outperforms alternatives. We show its versatility by mapping tissue architecture during mouse embryonic development and delineating tumor niches in human cancers, including a spatial reference mapping application. Finally, we extend its capabilities to spatial multi-omics, demonstrate cross-technology integration with datasets from different sequencing platforms and construct a whole mouse brain spatial atlas comprising 8.4 million cells, highlighting NicheCompass' scalability. Overall, NicheCompass provides a scalable framework for identifying and analyzing niches through signaling events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14cda68a9739dfb6fdc7acbc975a0b67601ec1ae" target='_blank'>
              Quantitative characterization of cell niches in spatially resolved omics data.
              </a>
            </td>
          <td>
            Sebastian Birk, Irene Bonafonte-Pardàs, Adib Miraki Feriz, Adam Boxall, E. Agirre, Fani Memi, A. Maguza, Anamika Yadav, Erick Armingol, Rong Fan, Gonçalo Castelo-Branco, F. Theis, O. Bayraktar, Carlos Talavera-López, M. Lotfollahi
          </td>
          <td>2025-03-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The spatial organization of adaptive immune cells within lymph nodes is critical for understanding immune responses during infection and disease. Here, we introduce AIR-SPACE, an integrative approach that combines high-resolution spatial transcriptomics with paired, high-fidelity long-read sequencing of T and B cell receptors. This method enables the simultaneous analysis of cellular transcriptomes and adaptive immune receptor (AIR) repertoires within their native spatial context. We applied AIR-SPACE to mouse popliteal lymph nodes at five distinct time points after Vaccinia virus footpad infection and constructed a comprehensive map of the developing adaptive immune response. Our analysis revealed heterogeneous activation niches, characterized by Interferon-gamma (IFN-γ) production, during the early stages of infection. At later stages, we delineated sub-anatomical structures within the germinal center (GC) and observed evidence that antibody-producing plasma cells differentiate and exit the GC through the dark zone. Furthermore, by combining clonotype data with spatial lineage tracing, we demonstrate that B cell clones are shared among multiple GCs within the same lymph node, reinforcing the concept of a dynamic, interconnected network of GCs. Overall, our study demonstrates how AIR-SPACE can be used to gain insight into the spatial dynamics of infection responses within lymphoid organs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ff2bb39af0926fb7ea6baef50f5c517a318f30" target='_blank'>
              A Temporal and Spatial Atlas of Adaptive Immune Responses in the Lymph Node Following Viral Infection
              </a>
            </td>
          <td>
            Shaowen Jiang, Madhav Mantri, Viviana I. Maymi, S. Leddon, Peter Schweitzer, Subash Bhandari, Chase Holdener, Ioannis Ntekas, Christopher Vollmers, A. Flyak, Deborah J. Fowell, Brian D. Rudd, I. De Vlaminck
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Clinical analyses demonstrated treatment naive DDR status was associated with different responses to frontline chemotherapy. Using longitudinal liquid biopsies, we found that DDR Intermediate and High tumors exhibited subtype switching and coincident emergence of heterogenous phenotypes following frontline treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes and may be associated with different chemotherapy responses and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02291-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671674f412412e8378b84a598f7d2a87a83c70ce" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2025-03-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec81235338a787d945989660c2ddefa3be4c0bd" target='_blank'>
              Novel CHI3L1-Associated Angiogenic Phenotypes Define Glioma Microenvironments: Insights From Multi-Omics Integration.
              </a>
            </td>
          <td>
            Yu-Hang Zhao, Yu-Xiang Cai, Zhi-yong Pan, F. Tang, Chao Ma, Ze-Fen Wang, Gang Li, Hang Chang, Su-Fang Tian, Zhi-Qiang Li
          </td>
          <td>2025-02-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Primary pulmonary lymphoepithelioma-like carcinoma (PPLELC) is a rare subtype of non-small-cell lung cancer. Duo to the current lack of precise targeted therapies, there is an urgent need to identify novel therapeutic targets. In this study, we perform single-nucleus transcriptome analysis on PPLELC samples to reveal the molecular tumor heterogeneity and characterize the functional states of immune cells within the tumor microenvironment. We identify a critical malignant subpopulation of PPLELC characterized by elevated expression of AKT3 and FGFR2. Higher expression levels of AKT3 and FGFR2 are associated with poorer patient outcomes. Moreover, treatment with either an AKT3 inhibitor or an FGFR2 inhibitor significantly attenuates tumor progression in patient-derived xenograft models. Our findings highlight AKT3 and FGFR2 as potential therapeutic targets and prognostic biomarkers, providing valuable insights for the development of rational targeted therapies and immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a403bcbf00350f8809a4da6d588cc795243cc7f5" target='_blank'>
              Single-cell analysis reveals transcriptomic features and therapeutic targets in primary pulmonary lymphoepithelioma-like carcinoma
              </a>
            </td>
          <td>
            Binghua Tan, Ke Xu, Y. Lyu, Yicheng Liang, Ruihao Liang, Kai Lei, Jialu Liang, Jing Huang, Kefeng Wang, Duoguang Wu, Wenjian Wang, Xueting Hu, Kexi Wang, Minghui Wang, Huayue Lin
          </td>
          <td>2025-03-08</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Spatial transcriptomics is a cutting-edge technology that analyzes gene expression at the cellular level within tissues while integrating spatial location information. This concept, which combines high-plex RNA sequencing with spatial data, emerged in the early 2010s. Spatial transcriptomics has rapidly expanded with the development of technologies such as in situ hybridization, in situ sequencing, in situ spatial barcoding, and microdissection-based methods. Each technique offers advanced mapping resolution and precise spatial assessments at the single-cell level. Over the past decade, the use of spatial transcriptomics on clinical samples has enabled researchers to identify gene expressions in specific diseased foci, significantly enhancing our understanding of cellular interactions and disease processes. In the field of rheumatology, the complex and elusive pathophysiology of diseases such as rheumatoid arthritis, systemic lupus erythematosus, and Sjögren’s syndrome remains a challenge for personalized treatment. Spatial transcriptomics provides insights into how different cell populations interact within disease foci, such as the synovial tissue, kidneys, and salivary glands. This review summarizes the development of spatial transcriptomics and current insights into the pathophysiology of autoimmune rheumatic diseases, focusing on immune cell distribution and cellular interactions within tissues. We also explore the potential of spatial transcriptomics from a clinical perspective and discuss the possibilities for translating this technology to the bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95f806851975b88a84521b58227335a097ae890e" target='_blank'>
              Spatial transcriptomics in autoimmune rheumatic disease: potential clinical applications and perspectives
              </a>
            </td>
          <td>
            Atsuko Tsujii Miyamoto, Hiroshi Shimagami, Atsushi Kumanogoh, Masayuki Nishide
          </td>
          <td>2025-02-20</td>
          <td>Inflammation and Regeneration</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0761133026ec3ba173f2c421844c6435e13b010" target='_blank'>
              Real-Time Tracking of Tumor Invasion Dynamics in Organotypic Brain Slices at Subcellular Resolution
              </a>
            </td>
          <td>
            Jana B. Petr, Meng-Syuan Lin, Dina Hochuli, Andreas Hierlemann, Martin Baumgartner, Mario M. Modena
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3418175c8e687d05cd4cb3259c7f39ab58b56c25" target='_blank'>
              Dynamic responses to rejection in the transplanted human heart revealed through spatial transcriptomics
              </a>
            </td>
          <td>
            K. Amancherla, Angela M. Taravella Oill, Xavier Bledsoe, A. L. Williams, Nelson Chow, Shilin Zhao, Quanhu Sheng, David W. Bearl, Robert D. Hoffman, J. Menachem, Hasan K. Siddiqi, Douglas M. Brinkley, E. D. Mee, Niran Hadad, Vineet Agrawal, Jeffrey Schmeckpepper, A. Rali, Stacy Tsai, E. Farber-Eger, Quinn S Wells, Jane E. Freedman, Nathan R. Tucker, K. Schlendorf, Eric R. Gamazon, Ravi V. Shah, N. Banovich
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Immune cells exhibit functional heterogeneity beyond what is resolved by classical definitions of subpopulations based on cell surface expression of receptors. To develop efficient and personalized cell-based immunotherapies, we need to resolve this heterogeneity and understand the underlying parameters that dictate cellular responses to specific target cells. For this, new methods are required that can identify and harvest immune cells with specific functions, e.g., high cytotoxic potential, to form clonally expanded cells or to assess molecular or genetic signatures. In this study, we evaluate a system for non-destructive, live cell picking and release in deep, high-aspect ratio microwells and test it for isolation of individual natural killer (NK) cells. We assess its performance at retrieving and releasing beads from microwells and demonstrate its potential for single NK cell isolation with intact viability. We also implement a semi-automated workflow for functional single-cell screening of NK cell behavior in microwell arrays followed by single-cell identification and isolation, demonstrating the potential for functional screening and isolation of serial killing immune cells. Our evaluation concludes that this cell isolation system, in combination with microwell arrays, offers opportunities for improved understanding of NK cell biology with applications towards cell therapy. However, its limited throughput hinders large-scale applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba79619eb3501acae9caddc99dc83ce718f2ee3c" target='_blank'>
              Isolation of individual natural killer cells from deep, high-aspect ratio microwell arrays - an evaluative study
              </a>
            </td>
          <td>
            Quentin Verron, Niklas Sandström, Hanna van Ooijen, K. Guldevall, K. Olofsson, T. Frisk, Björn Önfelt
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary This study explores the potential of canine patient-derived organoids (PDOs) in identifying mammary tumor-specific weaknesses by comparing tumor cells with matching normal mammary PDOs from the same patient. For this purpose, two customized CRISPR/Cas9 screens were performed to identify genes that are essential for tumor cells to survive but dispensable for normal mammary epithelial cells. CDK2 was subsequently identified as an important vulnerability in PDOs derived from a canine mammary tumor patient, which can be targeted by an existing inhibitor, PF3600. Although we only used tumors and normal PDOs from a single patient, our approach exemplifies how to discover novel, more individualized anti-cancer treatments for dogs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a341795bab91ddb8ebd61b09b4f9f74bbe497648" target='_blank'>
              Individualized Pooled CRISPR/Cas9 Screenings Identify CDK2 as a Druggable Vulnerability in a Canine Mammary Carcinoma Patient
              </a>
            </td>
          <td>
            Marine Inglebert, Martina Dettwiler, Chang He, Enni Markkanen, Lennart Opitz, A. Naguleswaran, Sven Rottenberg
          </td>
          <td>2025-02-01</td>
          <td>Veterinary Sciences</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Background Model organisms allowing for longitudinal examinations of liver disease pathogenesis are pivotal for the development of new therapeutic modalities. The eastern North American woodchuck develops chronic hepatitis and liver cancer after woodchuck hepatitis virus (WHV) infection, mirroring aspects of the natural history of the human hepatitis B virus (HBV). However, the cellular landscape of the woodchuck liver and the cell-level relevance of WHV infection to HBV infection is currently uncharacterized. Methods We employed single-cell RNA sequencing (RNA-seq) to generate an atlas of healthy woodchuck liver (63,389 cells, n=8) and peripheral blood mononuclear cells (PBMCs) (26,972 cells, n=7). Cell-specific and hepatic zonation gene signatures were validated using spatial transcriptomics (n=1). We employed our atlas to examine immune activation in stimulated precision cut-liver slices (PCLS) and disease-related pathway activation in chronic WHV infection (11,797 cells, n=3). We further employed our atlas to examine shared disease pathways between WHV infection and human HBV infection. Results Our atlas revealed woodchuck hepatic cellular diversity comparable to human and murine livers. Applying single-nucleus RNA-seq to PMA/ionomycin-stimulated precision cut liver slices revealed inflammation-associated activation signatures in T cell, myeloid and endothelial cell compartment. Finally, we describe intrahepatic T cells in chronic WHV hepatitis with both exhaustion and activation-associated signatures that resemble intrahepatic T cell genes signatures described in human chronic HBV. Conclusions We present a multi-omic atlas of healthy, diseased and ex vivo stimulated woodchuck liver. By identifying shared pathological processes between WHV and HBV infections, our findings reinforce the value of this preclinical model in translational research. This resource aims to advance studies on HBV pathogenesis and oncogenesis to speed the development of novel therapeutic strategies. Impact and Implications/Lay summary The liver plays important roles in metabolism, detoxification, and immune processes; liver transplantation is often the only treatment option for severe chronic liver diseases. Therefore, developing animal models that reflect human liver disease and can be studied throughout the disease course is crucial for the discovery of new treatment options. The woodchuck is an animal that develops chronic hepatitis and liver cancer after infection with woodchuck hepatitis virus, which models the human hepatitis type B virus infection (HBV) and associated hepatic carcinoma. However, our understanding of the cells that compose the woodchuck liver is limited, making it challenging to design and test cell-based therapeutics. In this study, we atlased the healthy and chronically infected woodchuck liver, found that liver cell types in woodchuck resemble those in humans, and employed the atlas to show similarities between WHV and HBV at the cell level, reinforcing the potential of WHV-infected woodchuck as a model for human HBV disease. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8ef9bc878aedeb55d6c6cc4171d96693eccf30b" target='_blank'>
              A Single-Cell Woodchuck Liver Atlas Identifies Healthy and Disease-related Cellular Programs Conserved in Human
              </a>
            </td>
          <td>
            Zoe A. Clarke, Jawairia Atif, Xinle Wang, Lewis Y. Liu, Lawrence Wood, Damra Camat, Sai W. Chung, Xue-Zhong Ma, Justin Manuel, Si Lok, Timothy Lau, C. Thoeni, Tomasz I. Michalak, Ian D. McGilvray, Gary D. Bader, S. Macparland
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="As spatial molecular data grow in scope and resolution, there is a pressing need to identify key spatial structures associated with disease. Current approaches often rely on hand-crafted features such as local abundances of manually annotated, discrete cell types, which may overlook important signals. Here we introduce variational inference-based microniche analysis (VIMA), a method that combines deep learning with principled statistics to discover associated spatial features with greater flexibility and precision. VIMA uses a variational autoencoder to extract numerical “fingerprints” from small tissue patches that capture their biological content. It uses these fingerprints to define a large number of “microniches” – small, potentially overlapping groups of tissue patches with highly similar biology that span multiple samples. It then uses rigorous statistics to identify microniches whose abundance correlates with case-control status. We show in simulations that VIMA is well calibrated and more powerful and accurate than other approaches. We then apply VIMA to a 140-gene spatial transcriptomics dataset in Alzheimer’s dementia, a 54-marker CO-Detection by indEXing (CODEX) dataset in ulcerative colitis (UC), and a 7-marker immunohistochemistry dataset in rheumatoid arthritis (RA), in each case recapitulating known biology and identifying novel spatial features of disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55b596129875ea6fffe7088bad79d02d94a1bbd2" target='_blank'>
              Powerful and accurate case-control analysis of spatial molecular data with deep learning-defined tissue microniches
              </a>
            </td>
          <td>
            Yakir A. Reshef, Lakshay Sood, Michelle Curtis, Laurie Rumker, Daniel J. Stein, Mukta G. Palshikar, S. Nayar, Andrew Filer, A. Jonsson, Ilya Korsunsky, Soumya Raychaudhuri
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="We report proteogenomic analysis of locally advanced cervical cancer (LACC). Exome-seq data revealed predominant alterations of keratinization-TP53 regulation and O-glycosylation-TP53 regulation axes in squamous and adeno-LACC, respectively, compared to in early-stage cervical cancer. Integrated clustering of mRNA, protein, and phosphorylation data identified six subtypes (Sub1-6) of LACC among which Sub3, 5, and 6 showed the treatment-resistant nature with poor local recurrence-free survival. Elevated immune and extracellular matrix (ECM) activation mediated by activated stroma (PDGFD and CXCL1high fibroblasts) characterized the immune-hot Sub3 enriched with MUC5AChigh epithelial cells (ECs). Increased epithelial-mesenchymal-transition (EMT) and ECM remodeling characterized the immune-cold squamous Sub5 enriched with PGK1 and CXCL10high ECs. We further demonstrated that CIC mutations could trigger EMT activation by upregulating ETV4, and the elevation of the immune checkpoint PVR and neutrophil-like myeloid-derived suppressive cells (FCN1 and FCGR3Bhigh macrophages) could cause suppression of T-cell activation in Sub5. Increased O-linked glycosylation of mucin characterized adeno-LACC Sub6 enriched with MUC5AChigh ECs. These results provide a battery of somatic mutations, cellular pathways, and cellular players that can be used to predict treatment-resistant LACC subtypes and can serve as potential therapeutic targets for these LACC subtypes. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02256-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f895c0a9b62dd33a0cc6a786a1ce0dbaa8e88ab" target='_blank'>
              Proteogenomic characterization of molecular and cellular targets for treatment-resistant subtypes in locally advanced cervical cancers
              </a>
            </td>
          <td>
            Do Young Hyeon, Dowoon Nam, Hye-Jin Shin, Juhee Jeong, Eunsoo Jung, Soo Young Cho, Dong Hoon Shin, Ja-Lok Ku, H. Baek, Chong Woo Yoo, Eun-Kyung Hong, Myong Cheol Lim, Sang-Jin Lee, Young-Ki Bae, Jong Kwang Kim, Jingi Bae, Wonyoung Choi, Sujin Kim, Seunghoon Back, Chaewon Kang, Inamul Hasan Madar, Hokeun Kim, Suhwan Kim, Duk Ki Kim, Jihyung Kang, Geon Woo Park, Ki Seok Park, Y. Shin, Sang Soo Kim, K. Jung, Da-som Hwang, Sang-Won Lee, Joo-Young Kim
          </td>
          <td>2025-03-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer cell lines are valuable models for studying tumor biology, yet their genomic evolution during culture can compromise experimental reproducibility. We conducted a detailed genomic analysis of the triple-negative breast cancer cell line MDA-MB-231, examining sublines obtained from different sources, at various time points, and across distinct passages. We introduce the concept of intraline heterogeneity (ILH) to highlight the genomic variability observed among these sublines. Our analyses revealed extensive genomic diversity, including differences in single nucleotide variants (SNVs) and copy number alterations (CNAs). In particular, CNAs exhibited remarkable heterogeneity, with pronounced chromosomal gains and losses between sublines, underscoring the impact of genomic instability on ILH. These findings suggest that ILH may influence experimental outcomes, emphasizing the importance of considering passage-specific genomic characterization to ensure consistency and reliability in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7a90a295650f938dfb30675bf97324372367f9" target='_blank'>
              Intraline genomic heterogeneity of the triple-negative breast cancer MDA-MB-231 cell line
              </a>
            </td>
          <td>
            Nair Varela-Rouco, N. Estévez-Gómez, C. Fernández-Santiago, Laura Tomás, Miriam Pérez, Daniel García-Souto, J. Pasantes, R. Piñeiro, J. M. Alves, D. Posada
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59376f615ed819023304612aaccafe8e9882b1f" target='_blank'>
              Mapping T cell dynamics to molecular profiles through behavior-guided transcriptomics.
              </a>
            </td>
          <td>
            A. K. Wezenaar, U. Pandey, F. Keramati, M. Hernandez-Roca, P. Brazda, M. Barrera Román, A. Cleven, F. Karaiskaki, T. Aarts-Riemens, S. de Blank, P. Hernandez-Lopez, S. Heijhuurs, A. Alemany, J. Kuball, Z. Sebestyen, J. F. Dekkers, H. Stunnenberg, M. Alieva, A. Rios
          </td>
          <td>2025-02-10</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/520e7a98e4f661979d2c0ecb98d246afd9f267aa" target='_blank'>
              Translating genetics into tissue: inflammatory cytokine-producing TAMs and PD-L1 tumor expression as poor prognosis factors in cutaneous melanoma
              </a>
            </td>
          <td>
            Celia Barrio-Alonso, Alicia Nieto-Valle, Lucía Barandalla-Revilla, J. Avilés-Izquierdo, V. Parra-Blanco, Paloma Sánchez-Mateos, Rafael Samaniego
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary This study investigates metastatic melanoma through an in-depth analysis of molecular and genetic characteristics of tumor samples. Five distinct melanoma subtypes were identified based on the protein expression profiles within the tumors and their surrounding microenvironments. Our findings highlight the role of specific mutations, particularly in the BRAF gene, which can affect patient survival, with some patients showing better outcomes due to a particular immune response triggered by the mutation. Additionally, our study uncovered genetic changes in melanoma tumors that could support the development of treatments that target the immune system. The study shows that proteins linked to survival are distributed in complex patterns in both primary tumors and metastases, offering deeper insights into the molecular heterogeneity of melanoma. By combining detailed molecular analysis with clinical and tissue examination, this research provides new insights into melanoma biology and offers guidance for improving treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557557036af2f6be01a0b554ada4c910b8468881" target='_blank'>
              Proteogenomic Profiling of Treatment-Naïve Metastatic Malignant Melanoma
              </a>
            </td>
          <td>
            M. Kuras, L. Betancourt, R. Hong, L. Szadai, Jimmy Rodriguez, Peter Horvatovich, I. Pla, Jonatan Eriksson, B. Szeitz, Bartłomiej Deszcz, Charlotte Welinder, Y. Sugihara, Henrik Ekedahl, Bo Baldetorp, C. Ingvar, L. Lundgren, H. Lindberg, Henriett Oskolás, Z. Horváth, M. Rezeli, Jeovanis Gil, R. Appelqvist, L. Kemény, Johan Malm, Aniel Sanchez, A. M. Szasz, Krzysztof Pawłowski, E. Wieslander, David Fenyö, I. Németh, Gyorgy Marko-Varga
          </td>
          <td>2025-02-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Understanding how cells respond differently to perturbation is crucial in cell biology, but existing methods often fail to accurately quantify and interpret heterogeneous single-cell responses. Here we introduce the perturbation-response score (PS), a method to quantify diverse perturbation responses at a single-cell level. Applied to single-cell perturbation datasets such as Perturb-seq, PS outperforms existing methods in quantifying partial gene perturbations. PS further enables single-cell dosage analysis without needing to titrate perturbations, and identifies ‘buffered’ and ‘sensitive’ response patterns of essential genes, depending on whether their moderate perturbations lead to strong downstream effects. PS reveals differential cellular responses on perturbing key genes in contexts such as T cell stimulation, latent HIV-1 expression and pancreatic differentiation. Notably, we identified a previously unknown role for the coiled-coil domain containing 6 (CCDC6) in regulating liver and pancreatic cell fate decisions. PS provides a powerful method for dose-to-function analysis, offering deeper insights from single-cell perturbation data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/665524668bbf5054da27a38a2c50953ffb070f2b" target='_blank'>
              Decoding heterogeneous single-cell perturbation responses
              </a>
            </td>
          <td>
            Bicna Song, Dingyu Liu, W. Dai, Natalie F. McMyn, Qingyang Wang, Dapeng Yang, Adam Krejci, A. Vasilyev, Nicole Untermoser, Anke Loregger, Dongyuan Song, Breanna Williams, Bess P. Rosen, Xiaolong Cheng, Lumen Chao, Hanuman T Kale, Hao Zhang, Yarui Diao, T. Bürckstümmer, Janet D Siliciano, J. Li, R. Siliciano, Danwei Huangfu, Wei Li
          </td>
          <td>2025-02-26</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>110</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8af0488d6a2ccbb080a6ebfdc837df22774e9cf9" target='_blank'>
              Integrated interactome, proteomic and functional analyses reveal molecular pathways driving L1TD1-induced aggressiveness in CNS embryonal tumor cells.
              </a>
            </td>
          <td>
            T. Mitsugi, Matt Sherwood, Elisa Helena Farias Jandrey, Amanda Faria Assoni, Joseph Bell, Brandon Coke, Paul Skipp, Rob M. Ewing, Oswaldo Keith Okamoto
          </td>
          <td>2025-02-06</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f69de6f588f5dc88e965e9e468e394e888395b6" target='_blank'>
              Single-cell and spatial transcriptomics reveal a tumor-associated macrophage subpopulation that mediates prostate cancer progression and metastasis.
              </a>
            </td>
          <td>
            Shenglin Mei, Hanyu Zhang, Taghreed Hirz, N. Jeffries, Yanxin Xu, Ninib Baryawno, Shulin Wu, Chin-Lee Wu, Akash Patnaik, P. Saylor, David B. Sykes, Douglas M Dahl
          </td>
          <td>2025-03-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="BACKGROUND
The heterogeneity of tumors significantly impacts on colorectal cancer (CRC) progression. However, the influence of this heterogeneity on the spatial architecture of CRC remains largely unknown.


METHODS
Spatial transcriptomic (ST) analysis of AOM/DSS-induced colorectal cancer (CRC), integrated with single-cell RNA sequencing, generated a comprehensive spatial atlas of CRC. Pseudotime trajectory, stemness evaluation, and cell-cell communication analyses explored how CD44+ tumor cells at the leading edge remodel the tumor microenvironment (TME). In vitro experiments and immunofluorescence staining of clinical samples validated pleiotrophin (PTN) signaling in promoting cancer-associated fibroblasts (CAFs) phenotypic transition and CRC progression.


RESULTS
Our findings revealed a distinctive layered ring-like structure within CRC tissues, where CD44+ tumor cells exhibiting high stemness were positioned at the tumor's leading edge. Inflammatory CAFs (iCAFs)-like, myofibroblastic CAFs (myCAFs)-like cells and pro-tumorigenic neutrophils primarily located at the tumor edge, in proximity to CD44+ tumor cells. CD44+ tumor cells then triggered the phenotypic transition of CAFs into iCAF-like and myCAF-like cells through PTN signaling.


CONCLUSIONS
Our results provide distinctive insights into how tumor heterogeneity reshapes the TME at the leading edge of tumor, thereby promoting CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/562ecaa8f1395c90f98e5b2a88e96773ab95655e" target='_blank'>
              CD44+ cells enhance pro-tumor stroma in the spatial landscape of colorectal cancer leading edge.
              </a>
            </td>
          <td>
            F. Tang, Yong-Gui Zhu, Jia Shen, Bowen Yuan, Xiang He, Yuxi Tian, Liang Weng, Lunquan Sun
          </td>
          <td>2025-03-12</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd1fbf6a111cb790d6d05a256a03567072e5318c" target='_blank'>
              AI-Driven Variant Annotation for Precision Oncology in Breast Cancer
              </a>
            </td>
          <td>
            Kriti Shukla, Yue Wang, Philip M. Spanheimer, Elizabeth Brunk
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36c2c77f99d55aa9b36b1f371ad8b8cec2aac50" target='_blank'>
              Evolutionary fingerprints of epithelial-to-mesenchymal transition.
              </a>
            </td>
          <td>
            Luigi Perelli, Li Zhang, Sarah Mangiameli, Francesca Giannese, Krishnan K. Mahadevan, F. Peng, F. Citron, Hania Khan, Courtney Le, Enrico Gurreri, F. Carbone, Andrew J. C. Russell, M. Soeung, T. N. Lam, Sebastian Lundgren, Sujay Marisetty, Cihui Zhu, Desiree Catania, Alaa M T Mohamed, Ningping Feng, J. Augustine, A. Sgambato, G. Tortora, Giulio F. Draetta, Giovanni Tonon, Andrew Futreal, Virginia Giuliani, A. Carugo, A. Viale, Michael P. Kim, T. Heffernan, Linghua Wang, Raghu Kalluri, D. Cittaro, Fei Chen, G. Genovese
          </td>
          <td>2025-03-05</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related deaths. An enhanced understanding of the immune microenvironments within these tumours may foster more precise and efficient treatment, particularly for immune-targeted therapies. The spatial architectural differences between lung adenocarcinoma (LUAD) and lung squamous cell carcinoma (LUSC) are relatively unexplored. Here, we applied imaging mass cytometry to a balanced cohort of LUAD and LUSC patients, matched for clinical factors such as age, sex, and smoking history, to analyze 204 histopathology images of tumours from 102 individuals with non-small cell lung cancer (NSCLC). By analyzing interactions and broader cellular networks, we interrogate the tumour microenvironment to understand how immune cells are spatially organized in clinically matched adenocarcinoma and squamous cell carcinoma subsets. This spatial analysis revealed distinct patterns of immune cell aggregation, particularly among macrophage populations, that correlated with patient prognosis differentially in adenocarcinoma and squamous cell carcinoma, suggesting potential new strategies for therapeutic intervention. Our findings underscore the importance of analyzing NSCLC histological subtypes separately when investigating the spatial immune landscape, as microenvironmental characteristics and cellular interactions differed by subtype. Recognizing these distinctions is essential for designing precision therapies tailored to each subtype’s unique immune architecture, ultimately enhancing patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33631c265f68921d79fbd004eb59ad3a5ea83b81" target='_blank'>
              Spatially mapping the tumour immune microenvironments of non-small cell lung cancer
              </a>
            </td>
          <td>
            Lysanne Desharnais, Mark Sorin, M. Rezanejad, Bridget Liu, E. Karimi, Aline Atallah, Anikka M Swaby, Miranda W. Yu, Samuel Doré, Saskia Hartner, B. Fiset, Yuhong Wei, Baharak Kadang, R. Rayes, Philippe Joubert, Sophie Camilleri-Broët, P. Fiset, D. Quail, Jonathan D Spicer, L. Walsh
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background: Prostate cancer, as an androgen-dependent malignant tumor in older men, has attracted the attention of a wide range of clinicians. BCR remains a significant challenge following early prostate cancer treatment. Methods: The specific expression pattern of the Anoikis gene set in prostate cancer cells was first explored by single-cell and spatial transcriptomics analysis. Genes causally associated with prostate cancer were screened using Summary-data-based Mendelian Randomization (SMR). Subsequently, we explored the role and mechanism of CASP8 in prostate cancer cells and defined a new cell type: the CASP8 T cell. We constructed a prediction model that can better predict the BCR of prostate cancer, and explored the differences in various aspects of clinical subgroups, tumor microenvironments, immune checkpoints, drug sensitivities, and tumor-immune circulations between high- and low-risk groups. The results of SMR analysis indicated that CASP8 could increase the risk of prostate cancer. Based on the differential genes of CASP8-positive and -negative T cells, we constructed a four-gene prognostic model with a 5-year AUC of 0.713. Results: The results revealed that high-risk prostate cancer BCR patients had various characteristics such as higher tumor purity, higher BCR rate, downregulated SIRPA immune checkpoints, and unique drug sensitivity. Conclusions: In summary, CASP8 may be a potential biomarker for prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eeb3359d57a0925984c84f7f06ac46e83ea3d116" target='_blank'>
              Multi-Omics Analysis of the Anoikis Gene CASP8 in Prostate Cancer and Biochemical Recurrence (BCR)
              </a>
            </td>
          <td>
            Shan Huang, Hang Yin
          </td>
          <td>2025-03-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Objective Active surveillance (AS) offers a viable alternative to surgical intervention for the management of indolent papillary thyroid carcinoma (PTC), helping to minimize the incidence of unnecessary treatment. However, the broader adoption of AS is hindered by the need for more reliable diagnostic markers. This study aimed to identify the differences between indolent and progressive PTC and find new targets for biomarker development and therapeutic strategies. Methods We used single-cell RNA sequencing (scRNA-seq) to analyze cellular differences in 10 early-stage PTC tumors. Findings were validated in an additional 25 tumors using cell co-culture, migration assays, immunofluorescence staining, flow cytometry, and analysis of data from The Cancer Genome Atlas (TCGA). Results Tumor-infiltrating B cells (TIL-B), particularly germinal center B cells (GC-B), were more abundant in indolent PTC. These cells suppressed thyroid cell proliferation in both indolent and progressive cases, though indolent PTC had a higher capacity to recruit peripheral B cells. In indolent cases, TIL-B cells showed increased proliferation and formed clusters within tertiary lymphoid structures (TLS). PTPRC-CD22 interactions were identified as potential drivers of TIL-B cell proliferation. Markers linked to GC-B cells, such as LMO2, were highlighted as potential diagnostic and prognostic indicators for indolent PTC. Conclusion This study provides insights into the cellular landscape of early-stage PTC, revealing distinct tumor and immune microenvironment features in indolent and progressive cases. These findings advance the understanding of indolent PTC biology and support the development of reliable diagnostic and prognostic biomarkers. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03341-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c059a59d61bc9cea405825a6d5d7499c3788b8e" target='_blank'>
              Single-cell transcriptomics analysis reveals that the tumor-infiltrating B cells determine the indolent fate of papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Chunmei Li, Pei Wang, Zhizhong Dong, Weihan Cao, Yan-Jun Su, Jian-ming Zhang, Shuyan Zhao, Zhiyuan Wang, Zi Lei, Li Shi, Ruochuan Cheng, Wen Liu
          </td>
          <td>2025-03-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast tumors have traditionally been categorized into four widely recognized subtypes. Recent single-cell advancements have, however, shown that individual tumors can contain more than one subtype, exhibiting their intrinsic heterogeneity. Relatedly, whilst the crucial role of cell cycle regulation in cancer development is widely acknowledged, the specific impact of cell cycle phases on oncogenic signaling and processes in each of these subtypes remains unresolved. Here, we examine the differences in transcriptomic cancer hallmarks across and within cell cycle phases and breast cancer subtypes in two large single-cell cohorts. We show that taking cell cycle phases into account identifies a broader pool of relevant cancer hallmark pathways while preserving the hallmarks found when cell cycle phases are ignored. Moreover, we identify FDA approved drugs that can target diverse hallmarks within specific phase-subtype cell combinations. This study highlights the critical role of cell cycle phases in advancing our understanding of breast cancer biology. The distinct biological mechanisms observed in each phase-subtype group suggest that targeting these hallmarks may provide a more precise and effective therapeutic strategy for breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05c59d00d73d7927897f20d50ad9f322baa1c2ce" target='_blank'>
              Decoding Cell Cycle Phase Variations in Cancer Hallmarks Across Breast Cancer Subgroups
              </a>
            </td>
          <td>
            Miguel Castresana-Aguirre, A. Matikas, Linda S. Lindström, N. Tobin
          </td>
          <td>2025-03-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The spatial proximity of CD8+ T cells to tumor cells critically influences the efficacy of neoadjuvant therapy (NAT) in breast cancer (BC). In this study, we evaluated whether the presence of CD8+ T cells and other immune cells near cancer cells predicts treatment outcomes across various BC subtypes. We analyzed pre- and post-NAT biopsies from 104 BC patients using multiplex immunofluorescence (mIF) and immunohistochemistry (IHC) to assess the distribution of immune markers, including CD8+ T cells, CD68+ macrophages, FoxP3+ regulatory T cells. Our findings revealed that a higher percentage of CD8+ T cells within 20 µm of cancer cells (N20-CD8+ T cells) was strongly correlated with improved pathological complete response (pCR), disease-free survival (DFS), and overall survival (OS), regardless of tumor subtype or NAT regimen. Moreover, a positive correlation between CXCL9 expression and N20-CD8+ T cells suggests that CXCL9 may facilitate the recruitment of CD8+ T cells to tumor cells. Our study emphasizes the link between immune cell composition and location, and patient outcomes with NAT. Focusing on the spatial dynamics of CD8+ T cells could significantly advance personalized treatment strategies and the development of targeted immunotherapies in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6f831530f176b5b34be3de9d6c41ac294ce3eb8" target='_blank'>
              Spatial proximity of CD8+ T cells to tumor cells predicts neoadjuvant therapy efficacy in breast cancer
              </a>
            </td>
          <td>
            Hongling Liang, Jianqing Huang, Hongsheng Li, Weixing He, Xiang Ao, Zhi Xie, Yu Chen, Zhiyi Lv, Leyao Zhang, Yanhua Zhong, X. Tan, Guodong Han, Jie-Qiong Zhou, Ni Qiu, Ming Jiang, Haoming Xia, Yong-Tao Zhan, Lei Jiao, Jie Ma, Derek C Radisky, Jia Huang, Xuchao Zhang
          </td>
          <td>2025-02-10</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20c3f321acba286bf84a8f4af78044117acbf8c" target='_blank'>
              Abstract B105: Predicting spatial transcriptomics from histopathology with Path2Space: AI charts breast cancer patients tumor immune microenvironment on an unprecedented scale
              </a>
            </td>
          <td>
            Roshan Lodha, E. Shulman, Emma M. Campagnolo, Thomas Cantore, Tom Hu, M. Nasrallah, Danh-Tai Hoang, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0f6840421639f0eeeffb398c6fad844f8d444a6" target='_blank'>
              InSituPy – A framework for histology-guided, multi-sample analysis of single-cell spatial transcriptomics data
              </a>
            </td>
          <td>
            Johannes Wirth, Anna Chernysheva, Birthe Lemke, Isabel Giray, Aitana Egea Lavandera, K. Steiger
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc8b763d2e3ec3323e5145b28e5d2c4495cc407" target='_blank'>
              Abstract B040: Transcriptomic meta-analysis identifies robust, tissue-agnostic gene expression signature of immune checkpoint blockade response
              </a>
            </td>
          <td>
            Y. Guo, Aaron C Tan, Tanmay Kulshrestha, M. Chang, Dawn Lau, Lan Ying Wang, Timothy Tay, Irfahan Kassam, Egor Revkov, Simone Rizzetto, I. Tan, Daniel S W Tan, A. Skanderup
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Neoantigen vaccines are under investigation for various cancers, including epidermal growth factor receptor (EGFR)-driven lung cancers1,2. We tracked the phylogenetic history of an EGFR mutant lung cancer treated with erlotinib, osimertinib, radiotherapy and a personalized neopeptide vaccine (NPV) targeting ten somatic mutations, including EGFR exon 19 deletion (ex19del). The ex19del mutation was clonal, but is likely to have appeared after a whole-genome doubling (WGD) event. Following osimertinib and NPV treatment, loss of the ex19del mutation was identified in a progressing small-cell-transformed liver metastasis. Circulating tumour DNA analyses tracking 467 somatic variants revealed the presence of this EGFR wild-type clone before vaccination and its expansion during osimertinib/NPV therapy. Despite systemic T cell reactivity to the vaccine-targeted ex19del neoantigen, the NPV failed to halt disease progression. The liver metastasis lost vaccine-targeted neoantigens through chromosomal instability and exhibited a hostile microenvironment, characterized by limited immune infiltration, low CXCL9 and elevated M2 macrophage levels. Neoantigens arising post-WGD were more likely to be absent in the progressing liver metastasis than those occurring pre-WGD, suggesting that prioritizing pre-WGD neoantigens may improve vaccine design. Data from the TRACERx 421 cohort3 provide evidence that pre-WGD mutations better represent clonal variants, and owing to their presence at multiple copy numbers, are less likely to be lost in metastatic transition. These data highlight the power of phylogenetic disease tracking and functional T cell profiling to understand mechanisms of immune escape during combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Background Diffuse large B-cell lymphoma (DLBCL) is a clinically heterogeneous malignancy with diverse patient outcomes, largely influenced by the tumor microenvironment (TME). Understanding the roles of fibroblasts and macrophages within the TME is essential for developing personalized therapeutic strategies in DLBCL. Methods This study is a multi-omics approach, integrating spatial transcriptomics (n = 11), bulk transcriptomics (n = 2,499), immunohistochemistry (IHC, n = 37), multiplex immunofluorescence (mIF, n = 56), and plasma samples (n = 240) to identify and characterize fibroblast and tumor-associated macrophage subtypes in the TME. Hub genes for LYZ+ fibroblasts and FN1+ macrophages were selected through univariate Cox regression and random forest analyses. Their prognostic significance was validated using IHC, mIF, and autoantibody assays in DLBCL patients treated with R-CHOP and in non-small cell lung cancer (NSCLC) patients receiving immune checkpoint inhibitors (ICIs). Results Fibroblasts and macrophages were classified into two distinct subtypes. Patients with higher LYZ+ fibroblasts infiltration demonstrated superior prognosis, which was associated with increased infiltration of FN1+ macrophages. Key hub genes identified for LYZ+ fibroblasts included LYZ, ANPEP, CSF3R, C15orf48, LILRB4, CLEC7A, and COL7A1, while hub FN1+ macrophages genes included COL1A1, FN1, APOE, DCN, MMP2, SPP1, COL3A1, and COL1A2. Independent prognostic markers in DLBCL treated with R-CHOP and NSCLC treated with ICIs were identified, including LYZ and LILRB4 at both protein and mRNA levels, and COL1A2 autoantibodies (p < 0.05). In DLBCL patients treated with R-CHOP, FN1 mRNA and autoantibody levels were also prognostic markers (p < 0.05). In NSCLC treated with ICIs, COL3A1 autoantibody was prognostic marker (p < 0.05). Conclusions This study identified a prognostically relevant LYZ+ fibroblasts and FN1+ macrophages in DLBCL. The hub genes associated with these subtypes represent potential biomarkers, providing insights into improving patient outcomes in DLBCL. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-03968-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2ac9b04d746f5cf36d236d7ac9a449435141cc" target='_blank'>
              Spatial transcriptomics reveals prognostically LYZ+ fibroblasts and colocalization with FN1+ macrophages in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            L. Dai, N. Lou, Liling Huang, Lin Li, Le Tang, Yuankai Shi, Xiaohong Han
          </td>
          <td>2025-02-25</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="T cell-based immunotherapies hold promise in treating cancer by leveraging the immune system’s recognition of cancer-specific antigens1. However, their efficacy is limited in tumours with few somatic mutations and substantial intratumoural heterogeneity2–4. Here we introduce a previously uncharacterized class of tumour-wide public neoantigens originating from RNA splicing aberrations in diverse cancer types. We identified T cell receptor clones capable of recognizing and targeting neoantigens derived from aberrant splicing in GNAS and RPL22. In cases with multi-site biopsies, we detected the tumour-wide expression of the GNAS neojunction in glioma, mesothelioma, prostate cancer and liver cancer. These neoantigens are endogenously generated and presented by tumour cells under physiologic conditions and are sufficient to trigger cancer cell eradication by neoantigen-specific CD8+ T cells. Moreover, our study highlights a role for dysregulated splicing factor expression in specific cancer types, leading to recurrent patterns of neojunction upregulation. These findings establish a molecular basis for T cell-based immunotherapies addressing the challenges of intratumoural heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs), known for facilitating the progression and metastasis of colorectal cancer (CRC), have become a promising therapeutic target. However, the significant heterogeneity of CAFs and their intricate crosstalk with tumor cells present substantial challenges in the development of precise and effective therapeutic strategies. Methods Single-cell RNA sequencing (scRNA-seq) technology was used to identify various cell subtypes. Spatial transcriptomics (ST) was employed to map the spatial niches and colocalization patterns of these cell subtypes. Cell-cell interactions among these subtypes were analysed via CellChat and NicheNet software. Tumor cell invasion, migration, and proliferation were assessed through wound healing assays, transwell assays, colony formation assays, and xenograft mouse models. Results We identified a significant spatial colocalization between CTHRC1+ CAFs and a distinct subtype of malignant epithelial cells, both residing within the EMT-active spatial niche. Our results demonstrate that CTHRC1+ CAFs, as a major source of WNT5A, promote epithelial-mesenchymal transition (EMT) and enhance tumor cell invasiveness by upregulating MSLN expression in adjacent malignant epithelial cells. This signaling axis contributes significantly to CRC progression and metastasis. Conclusions Targeting the CTHRC1+ CAF-WNT5A-MSLN signaling axis presents a promising therapeutic strategy for advanced CRC patients. Our study provides new insights into the role of CAFs in CRC progression and offers potential avenues for developing targeted therapies to disrupt this pathway. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06236-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efd1362638fd8682aac622f2d9c6d7d60b693242" target='_blank'>
              Single-cell and spatial transcriptome profiling reveal CTHRC1+ fibroblasts promote EMT through WNT5A signaling in colorectal cancer
              </a>
            </td>
          <td>
            Yunfei Lu, Yang Chen, Zhenling Wang, Hengyang Shen, Lei Xu, Changzhi Huang, Ying Tong, Yu Shao, Hongqiang Zhang, Zan Fu
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35da01c8609a7028b2e2ec26d0b8011e3cf58489" target='_blank'>
              A single-cell transcriptomic comparison between small intestinal neuroendocrine tumors and their progenitor
              </a>
            </td>
          <td>
            Fredrik Axling, Elham Barazeghi, Per Hellman, O. Norlén, S. Backman, P. Stålberg
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Peritoneal carcinomatosis is a common yet deadly manifestation of gastrointestinal cancers, with few effective treatments. To identify targetable determinants of peritoneal metastasis, we focused on appendiceal adenocarcinoma (AC), a gastrointestinal cancer that metastasizes almost exclusively to the peritoneum. Current treatments are extrapolated from colorectal cancer (CRC), yet AC has distinct genomic alterations, mucinous morphology and peritoneum restricted metastatic pattern. Further, no stable preclinical models of AC exist, limiting drug discovery and representing an unmet clinical need. We establish a first-in-class stable biobank of 16 long-term cultured AC patient-derived organoids (PDOs), including 3 matched, simultaneously resected primary AC-peritoneal carcinomatosis (AC-PC) pairs. By enriching for cancer cells, AC PDOs enable accurate genomic characterization relative to paucicellular AC tissue. We establish an organoid orthotopic intraperitoneal xenograft model that recapitulates diffuse peritoneal carcinomatosis and show that PC-organoids retain increased metastatic capacity, decreased growth factor dependency and sensitivity to standard of care chemotherapy relative to matched primary AC organoids. Single cell profiling of AC-PC pairs reveals dedifferentiation from mucinous differentiated states in primary AC into intestinal stem cell and fetal progenitor states in AC-PC, with upregulation of oncogenic signaling pathways. Through hypothesis-driven drug testing, we identify KRASMULTI-ON inhibitor RMC-7977 and Wnt-targeting tyrosine kinase inhibitor WNTinib as novel, clinically actionable strategies to target AC-PC more effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3baef141d668078d350e52fdb87357fa4d118d" target='_blank'>
              Paired primary-metastasis patient-derived organoids and mouse models identify phenotypic evolution and druggable dependencies of peritoneal metastasis from appendiceal cancer
              </a>
            </td>
          <td>
            Ahmed Mahmoud, Philip H. Choi, Christine Sukhwa, Jura Pintar, H. Walch, Nan Zhao, J. Bermeo, Sebastian K Chung, Manisha Raghavan, Samhita Bapat, Qingwen Jiang, G. Karagkounis, Julia Meredith, Michael Giarrizzo, C. Firat, A. Cercek, Michael B Foote, Nikolaus Schultz, W. Chatila, G. Nash, J. Shia, F. Sánchez-Vega, Steven M Larson, Arvin C. Dar, Neal Rosen, K. Ganesh
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Simple Summary Pancreatic cancer has a low survival rate with limited treatment options. Perineural invasion (PNI), where cancer cells infiltrate the nerves in the pancreas, is associated with a poor prognosis and local recurrence. Pancreatic cancer is one of the most painful cancers, with PNI contributing to the pain experienced by patients. However, the mechanisms involved in the cancer invasion of nerves are not completely understood. Here, we used spatial transcriptomics to study the expression of both cancer and nerve cells in microscopic regions of PNI and compared it to regions without PNI evidence to better understand the mechanisms involved in PNI. We identified novel mechanisms that may impact cancer growth, PNI and pain experienced by patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1818d7eac1f6857a9dc4b88e4ee2f868f4f36db6" target='_blank'>
              Spatial Transcriptomics Reveals Novel Mechanisms Involved in Perineural Invasion in Pancreatic Ductal Adenocarcinomas
              </a>
            </td>
          <td>
            V. Lakis, Noni L Chan, Ruth J Lyons, Nicola Blackburn, Tam Hong Nguyen, Crystal Chang, Andrew Masel, Nicholas West, Glen M. Boyle, A. Patch, Anthony J. Gill, K. Nones
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ed6afc1fbea4684dd9546a568ad33c0ccc1b6d" target='_blank'>
              Mapping molecular landscapes in triple-negative breast cancer: insights from spatial transcriptomics.
              </a>
            </td>
          <td>
            F. Al-Mansour, H. Almasoudi, Ali M Albarrati
          </td>
          <td>2025-03-22</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Diabetes mellitus (DM) is a global health crisis affecting millions, with islet transplantation emerging as a promising treatment strategy to restore insulin production. This review synthesizes the current research on single-cell and spatial transcriptomics in the context of islet transplantation, highlighting their potential to revolutionize DM management. Single-cell RNA sequencing, offers a detailed look into the diversity and functionality within islet grafts, identifying specific cell types and states that influence graft acceptance and function. Spatial transcriptomics complements this by mapping gene expression within the tissue’s spatial context, crucial for understanding the microenvironment surrounding transplanted islets and their interactions with host tissues. The integration of these technologies offers a comprehensive view of cellular interactions and microenvironments, elucidating mechanisms underlying islet function, survival, and rejection. This understanding is instrumental in developing targeted therapies to enhance graft performance and patient outcomes. The review emphasizes the significance of these research avenues in informing clinical practices and improving outcomes for patients with DM through more effective islet transplantation strategies. Future research directions include the application of these technologies in personalized medicine, developmental biology, and regenerative medicine, with the potential to predict disease progression and treatment responses. Addressing ethical and technical challenges will be crucial for the successful implementation of these integrated approaches in research and clinical practice, ultimately enhancing our ability to manage DM and improve patient quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4d2a871c9156c1f2749ccafc8536877666a5a4e" target='_blank'>
              Single-cell genomics and spatial transcriptomics in islet transplantation for diabetes treatment: advancing towards personalized therapies
              </a>
            </td>
          <td>
            Lisha Mou, Tony Bowei Wang, Yuxian Chen, Ziqi Luo, Xinyu Wang, Zuhui Pu
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Spatial transcriptomics has emerged as an invaluable tool, helping to reveal molecular status within complex tissues. Nonetheless, these techniques have a crucial challenge: the absence of single-cell resolution, resulting in the observation of multiple cells in each spatial spot. While reference-based deconvolution methods have aimed to solve the challenge, their effectiveness is contingent upon the quality and availability of single-cell RNA (scRNA) datasets, which may not always be accessible or comprehensive. In response to these constraints, our study introduces STsisal, a reference-free deconvolution method meticulously crafted for the intricacies of spatial transcriptomics (ST) data. STsisal leverages a novel approach that integrates marker gene selection, mixing ratio decomposition, and cell type characteristic matrix analysis to discern distinct cell types with precision and efficiency within complex tissues. The main idea of our method is its adaptation of the SISAL algorithm, which expertly disentangles the ratio matrix, facilitating the identification of simplices within the ST data. STsisal offers a robust means to unveil the intricate composition of cell types in spatially resolved transcriptomic data. To verify the efficacy of STsisal, we conducted extensive simulations and applied the method to real data, comparing its performance against existing techniques. Our findings highlight the superiority of STsisal, underscoring its utility in capturing the cell composition within complex tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/537297555122a8c531c29f52cfb730430bee08e6" target='_blank'>
              STsisal: a reference-free deconvolution pipeline for spatial transcriptomics data
              </a>
            </td>
          <td>
            Yinghao Fu, Leqi Tian, Weiwei Zhang
          </td>
          <td>2025-03-03</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f8ba9e3fcdae65a1005fa8694699f4a2467c8be" target='_blank'>
              Intratumour oxidative hotspots provide a niche for cancer cell dissemination.
              </a>
            </td>
          <td>
            Yoshifumi Ueda, Shigeki Kiyonaka, L. Selfors, Keisuke Inoue, Hiroshi Harada, Tomohiro Doura, Kunishige Onuma, Makoto Uchiyama, Ryuhei Kurogi, Yuji Yamada, Jiacheng H Sun, Reiko Sakaguchi, Yuki Tado, Haruki Omatsu, Harufumi Suzuki, M. Aoun, Takahiro Nakayama, Taketoshi Kajimoto, Tetsuya Yano, Rikard Holmdahl, I. Hamachi, Masahiro Inoue, Yasuo Mori, Nobuaki Takahashi
          </td>
          <td>2025-02-21</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="The focus of cancer immunotherapy has traditionally been on immune cells and tumor cells themselves, often overlooking the tumor secretome. This review provides a comprehensive overview of the intricate relationship between tumor cells and the immune response in cancer progression. It highlights the pivotal role of the tumor secretome - a diverse set of molecules secreted by tumor cells - in significantly influencing immune modulation, promoting immunosuppression, and facilitating tumor survival. In addition to elucidating these complex interactions, this review discusses current clinical trials targeting the tumor secretome and highlights their potential to advance personalized medicine strategies. These trials aim to overcome the challenges of the tumor microenvironment by designing therapies tailored to the secretome profiles of individual cancer patients. In addition, advances in proteomic techniques are highlighted as essential tools for unraveling the complexity of the tumor secretome, paving the way for improved cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0c4d7e98a1c7e44d04ba5fa342a0635434e2cbe" target='_blank'>
              Tumor secretome shapes the immune landscape during cancer progression
              </a>
            </td>
          <td>
            Jianqiang Yang, Sijia Tang, Nabil F. Saba, C. Shay, Yong Teng
          </td>
          <td>2025-02-10</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="Mesothelioma cell heterogeneity encompasses diverse morphological and molecular characteristics observed within tumors, significantly impacting disease progression, treatment outcomes, and the development of targeted therapies. This heterogeneity has long posed challenges for accurate diagnosis and effective treatment, but understanding its complexities offers the potential for novel diagnostic modalities and therapeutic interventions. This study employed single-cell RNA sequencing (scRNA-seq) to investigate mesothelioma cell heterogeneity from various sources, including cell culture (CC), peritoneal lavage (Lav) from the tumor microenvironment, and circulating tumor cells (CTC) in murine models. Gene set enrichment analysis was used to identify distinct gene signatures for each subpopulation. The results revealed unique characteristics for mesothelioma cells depending on their origin. In the CC group, up-regulated genes were primarily involved in tumor cell cycle control, proliferation, and apoptosis. In the CTC group, up-regulated genes were associated with cancer cell stemness. The Lav group showed up-regulated genes facilitating interactions between tumor cells and the microenvironment, such as epithelial–mesenchymal transition and immune responses mediated by IFN-α and IFN-γ. Some pathways were shared among all tumor cells, suggesting the potential for transitioning between functional states under specific conditions. This may be the first study to explore circulating mesothelioma cell heterogeneity using scRNA-seq. The distinct gene signatures identified in each mesothelioma cell subpopulation likely play critical roles in tumor initiation and progression, offering potential novel targets for therapeutic intervention. These findings could help inform the development of more effective, personalized treatments for mesothelioma, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38354bc1c5ef98aa13d3c5253963e1d30d5518bf" target='_blank'>
              Mesothelioma cell heterogeneity identified by single cell RNA sequencing
              </a>
            </td>
          <td>
            Licun Wu, Zongjie Wang, Amin Zia, S. Kelley, Marc de Perrot
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20f7ade1ee5b9d0bf71b29249fe64f13f03ddc98" target='_blank'>
              Characterization of single neurons reprogrammed by pancreatic cancer.
              </a>
            </td>
          <td>
            Vera Thiel, Simon Renders, Jasper Panten, Nicolas Dross, K. Bauer, D. Azorin, Vanessa Henriques, V. Vogel, C. Klein, Aino-Maija Leppä, Isabel Barriuso Ortega, J. Schwickert, I. Ourailidis, Julian Mochayedi, Jan-Philipp Mallm, Carsten Müller-Tidow, Hannah Monyer, J. Neoptolemos, T. Hackert, O. Stegle, D. T. Odom, R. Offringa, A. Stenzinger, Frank Winkler, M. Sprick, A. Trumpp
          </td>
          <td>2025-02-17</td>
          <td>Nature</td>
          <td>3</td>
          <td>86</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirical testing of all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated five compounds as potent PGCC inhibitors using cell lines and patient-derived models. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0482d6424427dcdf25109ef995747da0a47de141" target='_blank'>
              High-Throughput Empirical and Virtual Screening To Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yuan Zhang, Daniel D Brown, S. Oesterreich, Adrian V Lee, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2025-03-04</td>
          <td>Analytical Chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are pivotal biomarkers in tumor metastasis, however, the underlying molecular mechanism of CTCs behavioral heterogeneity during metastasis remains unexplored. Here, an integrative workflow is developed to link behavior characteristics to metabolic profiling within individual CTCs, which simulates the metastatic process on a microfluidic system and combined with single-cell mass spectrometry (MS) detection. Spheroid-derived HCT116 cells are tracked and extracted via a temporary vascular system, revealing various arrest patterns under biomimetic vascular shear flow. Downstream MS analysis characterizes 17 cellular metabolites and associates metabolic profiles with de-adhesion behaviors of the same CTCs, identifying a potential high-metastatic subpopulation with enhanced arrest ability and evaluating critical metabolites involved in metastasis pathways. Additionally, the metastasis-inhibiting effect of anti-tumor drug 5-fluorouracil by reducing high-metastatic cells in spheroids is elucidated. This approach offers a valuable opportunity to dissect the interplay of the metastatic behavior and metabolic profiles of CTCs and foster insights into the molecular mechanisms underlying behavioral phenotypes in the tumor metastasis process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32d3e81bfd80ccc573739b0a4ae2229795ed81e7" target='_blank'>
              Linking Metastatic Behavior and Metabolic Heterogeneity of Circulating Tumor Cells at Single-Cell Level Using an Integrative Microfluidic System.
              </a>
            </td>
          <td>
            Ying Hou, Jiaxu Lin, Hongren Yao, Zengnan Wu, Yongning Lin, Jinming Lin
          </td>
          <td>2025-02-17</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Pituitary neuroendocrine tumors (PitNETs) are one of the most common intracranial tumors with diverse clinical manifestations. Current pathological classification systems rely primarily on histological hormone staining and transcription factors (TFs) expression. While effective in identifying three major lineages, molecular characteristics based on hormones and TFs lack sufficient resolution to fully capture the complex tumor heterogeneity. Transcriptional diversity by alternative splicing (AS) offered additional insight to address this challenge. Here, we perform bulk and full-length single-cell RNA sequencing to comprehensively investigate AS dysregulation across all PitNET lineages. We reveal pervasive splicing dysregulations that better depict tumor heterogeneity. Additionally, we delineate fundamental splicing heterogeneity at single-cell resolution, confirming bulk findings and refining splicing dysregulation varying among tumor cell types. Notably, we effectively distinguish the silent corticotroph subtype and define a distinct TPIT lineage subtype, which is associated with worse clinical outcomes and increased splicing abnormalities driven by altered ESRP1 expression. In conclusion, our results characterize the subtype specific AS landscape in PitNETs, enhancing the understanding of the PitNETs subtyping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d1d629c974d4b9e986ad888f6cfe60cc9b53f8" target='_blank'>
              Splicing diversity enhances the molecular classification of pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Yue Huang, Jing Guo, Xueshuai Han, Yang Zhao, Xuejing Li, Peiqi Xing, Yulou Liu, Yingxuan Sun, Song Wu, Xuan Lv, Lei Zhou, Yazhuo Zhang, Chuzhong Li, Weiyan Xie, Zhaoqi Liu
          </td>
          <td>2025-02-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most prevalent and aggressive histological subtype of ovarian cancer and often presents with metastatic disease. The drivers of metastasis in HGSOC remain enigmatic. APOBEC3A (A3A), an enzyme that generates mutations across various cancers, has been proposed as a mediator of tumor heterogeneity and disease progression. However, the role of A3A in HGSOC has not been explored. We observed an association between high levels of APOBEC3-mediated mutagenesis and poor overall survival in primary HGSOC. We experimentally addressed this correlation by modeling A3A expression in HGSOC, and this resulted in increased metastatic behavior of HGSOC cells in culture and distant metastatic spread in vivo, which was dependent on catalytic activity of A3A. A3A activity in both primary and cultured HGSOC cells yielded consistent alterations in expression of epithelial-mesenchymal transition (EMT) genes resulting in hybrid EMT and mesenchymal signatures, providing a mechanism for their increased metastatic potential. Inhibition of key EMT factors TWIST1 and IL-6 resulted in mitigation of A3A-dependent metastatic phenotypes. Our findings define the prevalence of A3A mutagenesis in HGSOC and implicate A3A as a driver of HGSOC metastasis via EMT, underscoring its clinical relevance as a potential prognostic biomarker. Our study lays the groundwork for the development of targeted therapies aimed at mitigating the deleterious effect of A3A-driven EMT in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5397c063e336101c76892114dddd3f14218105fb" target='_blank'>
              APOBEC3A drives ovarian cancer metastasis by altering epithelial-mesenchymal transition
              </a>
            </td>
          <td>
            J. Devenport, Thi Tran, Brooke R. Harris, Dylan F. Fingerman, Rachel A. DeWeerd, Lojain H Elkhidir, Danielle LaVigne, Katherine C Fuh, Lulu Sun, Jeffrey J Bednarski, Ronny Drapkin, Mary M Mullen, Abby M. Green
          </td>
          <td>2025-03-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f973214ce129b073da8e0d3c1c3cc399902a2e8" target='_blank'>
              Spatial single-cell proteomics landscape decodes the tumor microenvironmental ecosystem of intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Libing Hong, Jie Mei, Xuqi Sun, Yifan Wu, Zhen Dong, Yuzhi Jin, Liaoliao Gao, Jinlin Cheng, Weihong Tian, Chuan Liu, Bin Li, Pingping Hu, Lulu Liu, Shan Xin, Xiaomeng Dai, Peng Zhao, Rongping Guo, Minshan Chen, Jingping Yun, Bo Lin, Wei Wei, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-02-25</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Summary Spatial and temporal intra-tumor heterogeneity drives tumor evolution and therapy resistance. Existing visualization tools often fail to capture both dimensions simultaneously. To address this, we developed Jellyfish, a tool that integrates phylogenetic and sample trees into a single plot, providing a holistic view of tumor evolution and capturing both spatial and temporal evolution. Available as a JavaScript library and R package, Jellyfish generates interactive visualizations from tumor phylogeny and clonal composition data. We demonstrate its ability to visualize complex subclonal dynamics using data from ovarian high-grade serous carcinoma. Availability and implementation Jellyfish is freely available with MIT license at https://github.com/HautaniemiLab/jellyfish (JavaScript library) and https://github.com/HautaniemiLab/jellyfisher (R package).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0210199a3e67c966717695406003ace2c046fe" target='_blank'>
              Jellyfish: integrative visualization of spatio-temporal tumor evolution and clonal dynamics
              </a>
            </td>
          <td>
            K. Lavikka, Altti Ilari Maarala, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-02-25</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most prevalent malignant brain tumor with poor prognosis. Although chromatin intratumoral heterogeneity is a characteristic feature of GBM, most current studies are conducted at a single tumor site. To investigate the GBM-specific 3D genome organization and its heterogeneity, we conducted Hi-C experiments in 21 GBM samples from nine patients, along with three normal brain samples. We identified genome subcompartmentalization and chromatin interactions specific to GBM, as well as extensive intertumoral and intratumoral heterogeneity at these levels. We identified copy number variants (CNVs) and structural variations (SVs) and demonstrated how they disrupted 3D genome structures. SVs could not only induce enhancer hijacking but also cause the loss of enhancers to the same gene, both of which contributed to gene dysregulation. Our findings provide insights into the GBM-specific 3D genome organization and the intratumoral heterogeneity of this organization and open avenues for understanding this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c40cfec55257420ce8df49630fbce7033db9c8" target='_blank'>
              Spatial 3D genome organization reveals intratumor heterogeneity in primary glioblastoma samples
              </a>
            </td>
          <td>
            Qixuan Wang, Juan Wang, Radhika Mathur, Mark Youngblood, Q. Jin, Ye Hou, L. Stasiak, Yu Luan, Hengqiang Zhao, S. Hilz, Chibo Hong, Susan M Chang, Janine M Lupo, Joanna J Phillips, Joseph F Costello, Feng Yue
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Genomic alterations are the driving force behind pancreatic cancer (PC) tumorigenesis, but they do not fully account for its diverse phenotypes. Investigating the epigenetic landscapes of PC offers a more comprehensive understanding and could identify targeted therapies that enhance patient survival. In this study, we have developed a new promising methodology of spatial epigenomics that integrates multiplexed molecular imaging with convolutional neural networks. Then, we used it to map epigenetic modification levels in the six most prevalent PC subtypes. We analyzed and semi-quantified the resulting molecular data, revealing significant variability in their epigenomes. DNA and histone modifications, specifically methylation and acetylation, were investigated. Using the same technique, we examined DNA conformational changes to further elucidate the transcriptional regulatory mechanisms involved in PC differentiation. Our results revealed that the foamy-gland and squamous-differentiated subtypes exhibited significantly increased global levels of epigenetic modifications and elevated Z-DNA ratios. Overall, our findings may suggest a potentially reduced efficacy of therapeutics targeting epigenetic regulators for these subtypes. Conversely, the conventional ductal PC subtype has emerged as a promising candidate for treatment with epigenetic modulators. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-90087-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61e79fb6328397b7897b70522e3b8824a0291600" target='_blank'>
              Spatial recognition and semi-quantification of epigenetic events in pancreatic cancer subtypes with multiplexed molecular imaging and machine learning
              </a>
            </td>
          <td>
            Krzysztof Szymoński, Natalia Janiszewska, K. Sofińska, Katarzyna Skirlińska-Nosek, Dawid Lupa, Michał Czaja, Marta Urbańska, Katarzyna Jurkowska, Kamila Konik, Marta Olszewska, Dariusz Adamek, Kamil Awsiuk, E. Lipiec
          </td>
          <td>2025-02-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Single-cell proteomics is an emerging field with significant potential to characterize heterogeneity within biological tissues. It offers complementary insights to single-cell transcriptomics by revealing unbiased proteomic changes downstream of the transcriptome. Recent advancements have focused on enhancing proteome coverage and depth, mostly in cultured cell lines, and a few recent studies have explored the potential of analyzing tissue micro-samples but were limited to homogenous peripheral tissues. In this current work, we utilize the power of spatial single cell-proteomics through immunostaining-guided laser capture microdissection (LCM) coupled with LC-MS to investigate the heterogenous central nervous system. We used this method to compare neuronal populations from cortex and substantia nigra, two brain regions associated with motor and cognitive function and various neurological disorders. Moreover, we used the technique to understand the neuroimmune changes associated with stab wound injury. Finally, we focus our application on the peripheral nervous system, where we compare the proteome of the myenteric plexus cell ganglion to the nerve bundle. This study demonstrates the utility of spatial single-cell proteomics in neuroscience research toward understanding fundamental biology and the molecular drivers of neurological conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e060f62a68156851b45076f66f6e5883e7b7f674" target='_blank'>
              Molecularly-guided spatial proteomics captures single-cell identity and heterogeneity of the nervous system
              </a>
            </td>
          <td>
            Sayan Dutta, Marion Pang, G. Coughlin, Sirisha Gudavalli, Michael L Roukes, Tsui-Fen Chou, V. Gradinaru
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Crohn’s disease (CD) involves a complex intestinal microenvironment driven by chronic inflammation. While single-cell RNA sequencing has provided valuable insights into this biology, the spatial context is lost during single-cell preparation of mucosal biopsies. To deepen our understanding of the distinct inflammatory signatures of CD and overcome the limitations of single-cell RNA sequencing, we combined spatial transcriptomics of frozen CD surgical tissue sections with single-cell transcriptomics of ileal CD mucosa. Coexpressed genes and cell-cell communication from single-cell analyses and factorized genes from spatial transcriptomics revealed overlapping pathways affected in inflamed CD, like antigen presentation, phagosome activity, cell adhesion, and extracellular matrix. Within the pathways, early epithelial cells showed evidence of significant changes in gene expression and subtype composition, while spatial mapping revealed the location of the events, particularly antigen presentation from epithelial cells in the base of the crypt. Furthermore, we identified early epithelial cells as a potential mediator of the MHC class II pathway during inflammation, which we validated by spatial transcriptomics cell subtype deconvolution. Therefore, the inflammation from CD appears to change the types of interactions detectable between epithelial cells with immune and mesenchymal cells, likely promoting the conditions for more macrophage infiltration into these inflammatory microdomains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00594471c2ef5f7cea997fb7b198a677a1e23c9" target='_blank'>
              Altered inflammatory mucosal signatures within their spatial and cellular context during active ileal Crohn’s disease
              </a>
            </td>
          <td>
            Vasantha L. Kolachala, S. Maddipatla, Shanta Murthy, Yeonjoo Hwang, A. Dodd, Garima Sharma, Sachith Munasinghe, R. Pelia, S. Venkateswaran, M. Anbazhagan, Tarun Koti, Navdeep Jhita, Gaurav N Joshi, Chrissy A. Lopez, Duke Geem, Hong Yin, David J. Cutler, Peng Qiu, J. Matthews, S. Kugathasan
          </td>
          <td>2025-03-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb3b0d5b454cf059f699fd40f05952bb5941c97" target='_blank'>
              Unveiling spatial heterogeneity in medulloblastoma: a multi-omics analysis of cellular state and geographical organization.
              </a>
            </td>
          <td>
            Jiankang Li, Hailong Liu, Ziwei Wang, Jiao Zhang, Xuan Chen, Craig Daniels, Xiaochong Wu, Olivier Saulnier, Hiromichi Suzuki, Pasqualino de Antonellis, Alexandra Rasnitsyn, Winnie W. Ong, Evan Y Wang, Liam D Hendrikse, Yu Su, Yu Tian, Dongming Han, Ruohan Wang, Jialin Mo, Fei Liu, Kaiwen Deng, Dongyang Wang, Zhaoyang Feng, Yifei Jiang, Yanong Li, Yuting Ma, Zijia Liu, Meiyu Li, Peiyi Tian, Yanfeng Shi, Yong Jiang, Tao Yang, Shouwei Li, Jianfeng Liang, Jingchuan Wu, Ying Wang, Wanjing Zou, Yina Jiang, Lusheng Wang, Fang Chen, Xin Jin, Shuaicheng Li, Xiaoguang Qiu, Chunde Li, Ya Gao, Yujie Tang, Michael D. Taylor, Tao Jiang
          </td>
          <td>2025-02-17</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4955ba693d4e35ecf04dfaea41f95d050a589d61" target='_blank'>
              Abstract A029: Highly-resolved integrated single cell atlas of human breast cancer enables hypothesis-driven analysis of tumor heterogeneity
              </a>
            </td>
          <td>
            Andrew Chen, Lina Kroehling, Christina S. Ennis, Gerald V. Denis, Stefano Monti
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5411691311f3ea382bc8274891edc9e740ebfb77" target='_blank'>
              Non-invasive approach for endoluminal biopsy coupled with single-cell proteomics allows for immune characterization of intracranial aneurysms
              </a>
            </td>
          <td>
            J. Antonios, B. Gultekin, B. Theriault, K. Yalcin, D. Miyagishima, N. Adenu-Mensah, N. Sujijantarat, A. Koo, J. Haynes, P. Cedeno, M. Johnson, R. Hebert, C. Matouk, T. Barak, M. Gunel
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cfebdd69571f854dae1eb84acf1f1884a093ad9" target='_blank'>
              Measuring Longitudinal Genome-wide Clonal Evolution of Pediatric Acute Lymphoblastic Leukemia at Single-Cell Resolution
              </a>
            </td>
          <td>
            Yakun Pang, Tamara Prieto, V. Gonzalez-Pena, Athena Aragon, Yuntao Xia, Sheng Kao, Sri Rajagopalan, John Zinno, Jean Quentin, Julien Laval, Dennis Yuan, Nathaniel D. Omans, David Klein, Matthew MacKay, I. De Vlaminck, John Easton, William Evans, D. Landau, Charles Gawad
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Single-cell image analysis is crucial for studying drug effects on cellular morphology and phenotypic changes. Most studies focus on single cell types, overlooking the complexity of cellular interactions. Here, we establish an analysis pipeline to extract phenotypic features of cancer cells cultured with fibroblasts. Using high-content imaging, we analyze an oncology drug library across five cancer and fibroblast cell line co-culture combinations, generating 61,440 images and ∼170 million single-cell objects. Traditional phenotyping with CellProfiler achieves an average enrichment score of 62.6% for mechanisms of action, while pre-trained neural networks (EfficientNetB0 and MobileNetV2) reach 61.0% and 62.0%, respectively. Variability in enrichment scores may reflect the use of multiple drug concentrations since not all induce significant morphological changes, as well as the cellular and genetic context of the treatment. Our study highlights nuanced drug-induced phenotypic variations and underscores the morphological heterogeneity of ovarian cancer cell lines and their response to complex co-culture environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4573d60560710e744b74b82ab85a673cb84bfba" target='_blank'>
              Evaluating feature extraction in ovarian cancer cell line co-cultures using deep neural networks
              </a>
            </td>
          <td>
            Osheen Sharma, Greta Gudoityte, Rezan Minozada, Olli P. Kallioniemi, Riku Turkki, L. Paavolainen, B. Seashore-Ludlow
          </td>
          <td>2025-02-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Understanding lung cancer evolution can identify tools for intercepting its growth. In a landscape analysis of 1024 lung adenocarcinomas (LUAD) with deep whole-genome sequencing integrated with multiomic data, we identified 542 LUAD that displayed diverse clonal architecture. In this group, we observed an interplay between mobile elements, endogenous and exogenous mutational processes, distinct driver genes, and epidemiological features. Our results revealed divergent evolutionary trajectories based on tobacco smoking exposure, ancestry, and sex. LUAD from smokers showed an abundance of tobacco-related C:G>A:T driver mutations in KRAS plus short subclonal diversification. LUAD in never smokers showed early occurrence of copy number alterations and EGFR mutations associated with SBS5 and SBS40a mutational signatures. Tumors harboring EGFR mutations exhibited long latency, particularly in females of European-ancestry (EU_N). In EU_N, EGFR mutations preceded the occurrence of other driver genes, including TP53 and RBM10. Tumors from Asian never smokers showed a short clonal evolution and presented with heterogeneous repetitive patterns for the inferred mutational order. Importantly, we found that the mutational signature ID2 is a marker of a previously unrecognized mechanism for LUAD evolution. Tumors with ID2 showed short latency and high L1 retrotransposon activity linked to L1 promoter demethylation. These tumors exhibited an aggressive phenotype, characterized by increased genomic instability, elevated hypoxia scores, low burden of neoantigens, propensity to develop metastasis, and poor overall survival. Re-activated L1 retrotransposition-induced mutagenesis can contribute to the origin of the mutational signature ID2, including through the regulation of the transcriptional factor ZNF695, a member of the KZFP family. The complex nature of LUAD evolution creates both challenges and opportunities for screening and treatment plans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, Monia Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank Colon-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, C. A. Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b71083b48d8d0e95bc226f7370321af11b26e25b" target='_blank'>
              Spatial and single-cell transcriptomic analysis reveals fibroblasts dependent immune environment in colorectal cancer.
              </a>
            </td>
          <td>
            Hang Jia, Xiang Liu, Guimin Wang, Yue Yu, Ning Wang, Tianshuai Zhang, Liqiang Hao, Wei Zhang, Guanyu Yu
          </td>
          <td>2025-03-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b338d775526840d18d525abb3d293c825f24385" target='_blank'>
              Abstract B011: Harnessing epigenetic and metabolic biomarkers for targeted immunotherapy in colorectal cancer: Advances in precision medicine for solid tumors
              </a>
            </td>
          <td>
            Ji Min Lee
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Metastasis is a leading cause of cancer-related deaths, with the liver being the most frequent site of metastasis in colorectal cancer. Previous studies have predominantly focused on the influence of the primary tumor itself on metastasis, with relatively limited research examining the changes within target organs. Methods Using an orthotopic mouse model of colorectal cancer, single-cell sequencing was employed to profile the transcriptomic landscape of pre-metastatic and metastatic livers. The analysis focused on identifying cellular and molecular changes within the hepatic microenvironment, with particular emphasis on inflammatory pathways and immune cell populations. Results A neutrophil subpopulation with high Prok2 expression was identified, showing elevated levels in the pre-metastatic and metastatic liver. Increased infiltration of Prok2⁺ neutrophils correlated with poor prognosis in liver metastatic colorectal cancer patients. In the liver metastatic niche (MN), these neutrophils showed high App and Cd274 (PD-L1) expression, suppressing macrophage phagocytosis and promoting T-cell exhaustion. Conclusion A Prok2⁺ neutrophil subpopulation infiltrated both pre-metastatic and macro-metastatic liver environments, potentially driving immunosuppression through macrophage inhibition and T-cell exhaustion. Targeting Prok2⁺ neutrophils could represent a novel therapeutic strategy for preventing liver metastasis in colorectal cancer patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06351-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ae7b5dd668485b7f400a3f84e79d9d667bb33c" target='_blank'>
              Single-cell transcriptomic analysis reveals dynamic changes in the liver microenvironment during colorectal cancer metastatic progression
              </a>
            </td>
          <td>
            Yue Jiang, Guojie Long, Xiaoming Huang, Wenyu Wang, B. Cheng, Weidong Pan
          </td>
          <td>2025-03-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="We identified in human breast cancer a scarce population of Tumor-Associated Monocytes (TAMs) endowed with a pro-metastatic activity and associated with reduced distant-metastasis free survival of patients. We developed a novel framework combining computational and experimental methodologies to dampen TAM pro-metastatic activity. Multi-modal experimental data from TAMs exposed in vitro to a series of perturbations were collected to build a Boolean dynamical network of TAMs. This framework successfully identified the biological pathways underlying TAM pro-metastatic activity and predicted potent pharmacological interventions that inhibited the pro-metastatic activity of TAMs isolated from patient tumors. This study showcases the power of integrating computational predictions with experimental validation in identifying new therapeutic avenues that can be extended to other cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b7bb8f484b312f30dfdbd661789ddfd12b48726" target='_blank'>
              A combined computational and experimental approach to successfully predict the behavior of metastatic tumor-associated macrophages in human breast cancer
              </a>
            </td>
          <td>
            Amel Bekkar, A. Pabois, Isaac Crespo, Riccardo Turrini, Ioannis Xenarios, M. Doucey
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a279852f8f3e4dea6bf2c322ab3c789c81b3e6c" target='_blank'>
              How cancer emerges: Data-driven universal insights into tumorigenesis via hallmark networks
              </a>
            </td>
          <td>
            Jiahe Wang, Yan Wu, Yuke Hou, Yang Li, Dachuan Xu, Changjing Zhuge, Yue Han
          </td>
          <td>2025-02-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer is a manifestation of dysfunctional cell states. It emerges from an interplay of intrinsic and extrinsic factors that disrupt cellular dynamics, including genetic and epigenetic alterations, as well as the tumor microenvironment. This complexity can make it challenging to infer molecular causes for treating the disease. This may be addressed by system-wide computer models of cells, as they allow rapid generation and testing of hypotheses that would be too slow or impossible to perform in the laboratory and clinic. However, so far, such models have been impeded by both experimental and computational limitations. In this perspective, we argue that they can now be achieved using deep learning algorithms to integrate omics data and prior knowledge of molecular networks. Such models would have many applications in precision oncology, e.g., for identifying drug targets and biomarkers, predicting resistance mechanisms and toxicity effects of drugs, or simulating cell-cell interactions in the microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8844ef9ba3f77a8686c18b6b08bd0f64a9f348" target='_blank'>
              Towards an interpretable deep learning model of cancer
              </a>
            </td>
          <td>
            Avlant Nilsson, N. Meimetis, Douglas A. Lauffenburger
          </td>
          <td>2025-02-14</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Introduction The molecular pathogenesis of ocular adnexal mucosa-associated lymphoid tissue (MALT) lymphoma and IgG4-related ophthalmic disease (IgG4-ROD) remains incompletely understood. Differentiating between the two diseases is vital given that the diagnostic evaluation and treatment approaches can vary significantly; this difficulty in distinction is exacerbated by the absence of specific biomarkers. This study aimed to investigate the differences between these two diseases based on their cellular composition, transcriptional heterogeneity, and the immune microenvironment using single-cell RNA transcriptional sequencing (scRNA-seq) technology. Methods We collected orbital lacrimal gland region tissue samples from three patients with MALT lymphoma and another three with IgG4-ROD and performed single-cell sequencing experiments. Subsequently, we conducted bioinformatics analyses, including cell subpopulation segmentation and inter-group comparison, tumor cell identification, functional enrichment analysis, and pseudotime trajectory analysis. Furthermore, we analyzed the cellular communication between tumor B-cell and T-cell subsets within the immune microenvironment of MALT lymphoma tissues. We performed immunofluorescence assays to verify the co-expression of receptor-ligand pairs. Results A total of six major cell subpopulations were identified, with B-cells and T-cells being the predominant cell types. All B-cell subpopulations in MALT lymphomas are malignant, exhibiting significant intratumoral and intertumoral heterogeneity. Reclustering of the T-cell subpopulation identified five major T-cell subpopulations. Pseudotime analysis revealed that CD4+ naive T-cells in MALT lymphoma patients were highly likely to differentiate into follicular helper T-cells, whereas, in IgG4-ROD patients, CD4+ naive T-cells were highly likely to differentiate into regulatory T-cells. Intercellular communication analysis revealed that the CD27-CD70 immune checkpoint receptor−ligand pair and CXCL13-CXCR5 chemokine receptor−ligand pair were significantly upregulated between malignant B-cells and T-cells subpopulations. Conclusion This study is the first to conduct a comparative single-cell transcriptome sequencing analysis of ocular adnexal MALT lymphoma and IgG4-ROD. Our results reveal the cellular composition, key pathways, and critical immune microenvironment implicated in the development of these two diseases. These findings provide important insights into the pathogenesis of these two diseases and highlight the differences between them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5b84eefafe3887c5d3f9997b2994a421a348df" target='_blank'>
              Single-cell RNA-sequencing reveals cellular heterogeneity and immune microenvironment characteristics between ocular adnexal mucosa-associated lymphoid lymphoma and IgG4−related ophthalmic disease
              </a>
            </td>
          <td>
            Yu Yang, Yujiao Wang, Xuelian Jin, W. He
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Oral cancer is one of the most prevalent malignant neoplasms globally, with its microenvironment being extremely complicated and heterogeneous with regard to distribution of different types of cells, which creates a barrier against treatment. We integrated single-cell RNA sequencing, pseudotime analysis, and intercellular communication to identify subtypes, state transitions, and interactions in the development of oral cancer and their roles in tumor progression and immune response. Methods Principal component analysis (PCA) followed by dimensionality reduction techniques (UMAP and t-SNE) revealed multiple populations in the single-cell data from oral cancer. Analysis of gene expression patterns associated individual genes with distinct principal components that described features of cell subtypes or states in the oral cancer microenvironment. We also conducted gene set enrichment analysis (GSEA) and cell–cell communication network analysis to investigate the important signaling pathways as well as potential cell–cell communications. Results Gene expression patterns linked to oral cancer cell subtypes and states in this study included cell membrane, signal transduction and immune response in different biological processes. Pseudotime analysis showed differentiation trajectories and expression of essential differentiation genes were also altered. Cell–cell communication network analysis revealed that myeloid cells interacted heavily with myeloid cells and they also had strong interactions with endothelial cells. Analysis of the MIF signaling pathway network showed participation of various cell types in MIF signaling, suggesting its potential significance in the oral cancer microenvironment. Conclusion The intercellular communication complexity of TME in oral cancer is reported here with the first-ever single cell analysis and data providing new perspective regarding the heterogeneity of oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8d5845a6bd00754561294af9f1ae51a6774014" target='_blank'>
              Single-cell pseudotime and intercellular communication analysis reveals heterogeneity and immune microenvironment in oral cancer
              </a>
            </td>
          <td>
            Hanjun Liu, Lian Xie, Xuemin Xing, Lili Hou, Jinfeng Zhang, Ling Yang
          </td>
          <td>2025-02-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastomas are the most prevalent primary brain tumors and are associated with a dramatically poor prognosis. Despite an intensive treatment approach, including maximal surgical tumor removal followed by radio- and chemotherapy, the median survival for glioblastoma patients has remained around 18 months for decades. Glioblastoma is distinguished by its highly complex mechanisms of immune evasion and pronounced heterogeneity. This variability is apparent both within the tumor itself, which can exhibit multiple phenotypes simultaneously, and in its surrounding microenvironment. Another key feature of glioblastoma is its “cold” microenvironment, characterized by robust immunosuppression. Recent advances in single-cell RNA sequencing have uncovered new promising insights, revealing previously unrecognized aspects of this tumor. In this review, we consolidate current knowledge on glioblastoma cells and its microenvironment, with an emphasis on their biological properties and unique patterns of molecular communication through signaling pathways. The evidence underscores the critical need for personalized poly-immunotherapy and other approaches to overcome the plasticity of glioblastoma stem cells. Analyzing the tumor microenvironment of individual patients using single-cell transcriptomics and implementing a customized immunotherapeutic strategy could potentially improve survival outcomes for those facing this formidable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf00ce3ce13ab5e7ee7ff1ae10bf120f0be9305e" target='_blank'>
              Molecular biology of the deadliest cancer – glioblastoma: what do we know?
              </a>
            </td>
          <td>
            Aly Ismailov, Aldo Spallone, Alexey Belogurov, Alan Herbert, Maria Poptsova
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tyrosine kinase inhibitors (TKIs) have transformed the treatment of epidermal growth factor receptor (EGFR)-mutant non-small cell lung cancer. However, treatment resistance remains a major challenge in clinical practice. The tumor microenvironment (TME) is a complex system composed of tumor cells, immune and non-immune cells, and non-cellular components. Evidence indicates that dynamic changes in TME during TKI treatment are associated with the development of resistance. Research has focused on identifying how each component of the TME interacts with tumors and TKIs to understand therapeutic targets that could address TKI resistance. In this review, we describe how TME components, such as immune cells, fibroblasts, blood vessels, immune checkpoint proteins, and cytokines, interact with EGFR-mutant tumors and how they can promote resistance to TKIs. Furthermore, we discuss potential strategies targeting TME as a novel therapeutic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f6f1b01ac68ddc2dc3220a920e6b272e7041d2" target='_blank'>
              Targeting the Tumor Microenvironment in EGFR-Mutant Lung Cancer: Opportunities and Challenges
              </a>
            </td>
          <td>
            Jeong Uk Lim, Junyang Jung, Y. Kim, Chi Young Kim, Sang Hoon Lee, Dong Won Park, Sue In Choi, W. Ji, C. Yeo, Seung Hyeun Lee
          </td>
          <td>2025-02-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Ongoing mutagenesis in cancer drives genetic diversity throughout the natural history of cancers. As the activities of mutational processes are dynamic throughout evolution, distinguishing the mutational signatures of ‘active’ and ‘historical’ processes has important implications for studying how tumors evolve. This can aid in understanding mutagenic states at the time of presentation, and in associating active mutational process with therapeutic resistance. As bulk sequencing primarily captures historical mutational processes, we studied whether ultra-low-coverage single-cell whole-genome sequencing (scWGS), which measures the distribution of mutations across hundreds or thousands of individual cells, could enable the distinction between historical and active mutational processes. While technical challenges and data sparsity have limited mutation analysis in scWGS, we show that these data contain valuable information about dynamic mutational processes. To robustly interpret single nucleotide variants (SNVs) in scWGS, we introduce ArtiCull, a method to identify and remove SNV artifacts by leveraging evolutionary constraints, enabling reliable detection of mutations for signature analysis. Applying this approach to scWGS data from pancreatic ductal adenocarcinoma (PDAC), triple-negative breast cancer (TNBC), and high-grade serous ovarian cancer (HGSOC), we uncover temporal and spatial patterns in mutational processes. In PDAC, we observe a temporal increase in mismatch repair deficiency (MMRd). In cisplatin-treated TNBC patient-derived xenografts, we identify therapy-induced mutagenesis and inactivation of APOBEC3 activity. In HGSOC, we show distinct patterns of APOBEC3 mutagenesis, including late tumor-wide activation in one case and clade-specific enrichment in another. Additionally, we detect a clone-specific increase in SBS17 activity, in a clone previously linked to recurrence. Our findings establish ultra-low-coverage scWGS as a powerful approach for studying active mutational processes that may influence ongoing clonal evolution and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed6d1d9f289244977b2dfb8fbeaff86f943a227d" target='_blank'>
              Inferring active mutational processes in cancer using single cell sequencing and evolutionary constraints
              </a>
            </td>
          <td>
            Gryte Satas, Matthew A. Myers, Andrew McPherson, Sohrab P. Shah
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, Xiaolong Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Simple Summary Lung cancer, particularly non-small cell lung cancer (NSCLC), remains a leading cause of cancer-related deaths worldwide due to its complexity and resistance to therapy. This review explores the critical role of stem cells in lung cancer progression, focusing on how genetic mutations and the tumor microenvironment drive cancer stem cell activity, leading to therapy resistance and tumor relapse. By understanding these mechanisms, the research aims to identify potential therapeutic targets and strategies to improve treatment outcomes for NSCLC patients. This work presents the latest developments of novel personalized therapies and enhances the effectiveness of existing treatments, offering new insights and information for patients and the broader research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8d3b80e9aee70a721cb7f7cf165886e84b84b9b" target='_blank'>
              Oncogenic Mutations and the Tumor Microenvironment: Drivers of Non-Small Cell Lung Cancer Progression
              </a>
            </td>
          <td>
            A. Mitrakas, Christos Kakouratos, I. Lamprou, Erasmia T. Xanthopoulou, M. Koukourakis
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8148f78a26bf47463c0acd3c067c50a7bec19c0" target='_blank'>
              Spatio-temporal dynamics of M1 and M2 macrophages in a multiphase model of tumor growth
              </a>
            </td>
          <td>
            I. Lampropoulos, Panayotis Kevrekidis, Christos Zois, Helen Byrne, Mihail E Kavousanakis
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b216340a3ee900bc1d9420b9d6cc7048a9f68c8" target='_blank'>
              Radiogenomics of intrahepatic cholangiocarcinoma predicts immunochemotherapy response and identifies therapeutic target.
              </a>
            </td>
          <td>
            Gu-Wei Ji, Zheng Xu, Shuo-Chen Liu, Shu-Ya Cao, C. Jiao, Ming Lu, Biao Zhang, Yue Yang, Qing Xu, Xiao-Feng Wu, Ke Wang, Yong-xiang Xia, Xiang-Cheng Li, Xue-Hao Wang
          </td>
          <td>2025-02-10</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="N6-methyladenosine (m6A), the most abundant internal RNA modification in humans, regulates most aspects of RNA processing. Prostate cancer is characterized by widespread transcriptomic dysregulation; therefore, we characterized the m6A landscape of 162 localized prostate tumors with matched DNA, RNA and protein profiling. m6A abundance varied dramatically across tumors, with global patterns emerging via complex germline-somatic cooperative regulation. Individual germline polymorphisms regulated m6A abundance, cooperating with somatic mutation of cancer driver genes and m6A regulators. The resulting complex patterns were associated with prognostic clinical features and established the biomarker potential of global and locus-specific m6A patterns. Tumor hypoxia dysregulates m6A profiles, bridging prior genomic and proteomic observations. Specific m6A sites, such as those in VCAN, drive disease aggression, associating with poor outcomes, tumor growth and metastasis. m6A dysregulation is thus associated with key events in the natural history of prostate cancer: germline risk, microenvironmental dysregulation, somatic mutation and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19695af6dbb1a44b8e5db5cde53d9277c7377300" target='_blank'>
              The landscape of N6-methyladenosine in localized primary prostate cancer.
              </a>
            </td>
          <td>
            Xin Xu, Helen Zhu, Rupert Hugh-White, J. Livingstone, Stefan E. Eng, Nicole Zeltser, Yujuan Wang, Kinga Pajdzik, Sujun Chen, Kathleen E. Houlahan, Wenqin Luo, Shun Liu, Xi Xu, Minzhi Sheng, Wangyuan Guo, J. Arbet, Yuxi Song, Miranda Wang, Yong Zeng, Shiyan Wang, Guanghui Zhu, Tingxiao Gao, Wei Chen, Xinpei Ci, Wenjie Xu, Kexin Xu, M. Orain, V. Picard, H. Hovington, A. Bergeron, L. Lacombe, Bernard Têtu, Yves Fradet, Mathieu Lupien, Gong-Hong Wei, Marianne Koritzinsky, Robert G Bristow, Neil Fleshner, Xue Wu, Yang Shao, Chuan He, Alejandro Berlin, T. H. van der Kwast, Hon Leong, P. Boutros, H. H. He
          </td>
          <td>2025-03-24</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), are a critical subpopulation within tumours, and are defined by their capacity for self-renewal, differentiation, and tumour initiation. These unique traits contribute to tumour progression, metastasis, and resistance to conventional treatments like chemotherapy and radiotherapy, often resulting in cancer recurrence and poor patient outcomes. As such, CSCs have become focal points in developing advanced cancer therapies. This review highlights progress in CSC-targeted treatments, including chimeric antigen receptor T-cell (CAR-T) therapy, immunotherapy, molecular targeting, and nanoparticle-based drug delivery systems. Plant-derived compounds and gene-editing technologies, such as clustered regularly interspaced short palindromic repeats (CRISPR), are explored for their potential to enhance precision and minimize side effects. Metabolic pathways integral to CSC survival, such as mitochondrial dynamics, mitophagy (regulated by dynamin-related protein 1 [DRP1] and the PINK1/Parkin pathway), one-carbon metabolism, amino acid metabolism (involving enzymes like glutaminase (GLS) and glutamate dehydrogenase (GDH]), lipid metabolism, and hypoxia-induced metabolic reprogramming mediated by hypoxia-inducible factors (HIF-1α and HIF-2α), are examined as therapeutic targets. The adaptability of CSCs through autophagy, metabolic flexibility, and epigenetic regulation by metabolites like α-ketoglutarate, succinate, and fumarate is discussed. Additionally, extracellular vesicles and nicotinamide adenine dinucleotide (NAD⁺) metabolism are identified as pivotal in redox balance, DNA repair, and epigenetic modifications. Addressing challenges such as tumour heterogeneity, immune evasion, and treatment durability requires interdisciplinary collaboration. Advancing CSC-targeted therapies is essential for overcoming drug resistance and preventing cancer relapse, paving the way for transformative cancer treatments. This review underscores the importance of leveraging innovative technologies and fostering collaboration to revolutionize cancer treatment. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdfd29a8c49b166f1e1f2b3241c057ea76b860a3" target='_blank'>
              Unveiling the future of cancer stem cell therapy: a narrative exploration of emerging innovations
              </a>
            </td>
          <td>
            Joseph Nhyira Obisi, Abike Ndidiamaka Josephine Abimbola, O. Babaleye, Peter Kwame Atidoglo, Saviour Godswealth Usin, Eudora Nwanaforo, Faith Sutu Patrick-Inezi, I. V. Fasogbon, Joseph Chimezie, Christianah Adebimpe Dare, Oluwadoyinsayemi Oluwadamilare Kuti, Daniel Ejim Uti, Humphrey Chukwudi Omeoga
          </td>
          <td>2025-03-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8e1ac36ae0a43d7b28635df6f8edaf8554a72a9" target='_blank'>
              SpaDCN: Deciphering Spatial Functional Landscape from Spatially Resolved Transcriptomics by Aligning Cell-Cell Communications.
              </a>
            </td>
          <td>
            Xiaosheng Bai, Xinyu Bao, Chuanchao Zhang, Qianqian Shi, Luonan Chen
          </td>
          <td>2025-02-17</td>
          <td>Small methods</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by pervasive copy number alterations (CNAs), significantly contributes to cancer progression and therapeutic resistance. CNAs drive intratumoral genetic heterogeneity, creating distinct subpopulations whose interactions may shape tumor evolution through frequency-dependent selection. Here, we introduce, ECO-K (Ecological-Karyotypes), an inverse game theory framework that infers subpopulation interactions from longitudinal single-cell whole genome sequencing data. Applying this approach to serially-passaged, triple-negative breast cancer (TNBC) cell lines and patient-derived xenografts, we systematically identify frequency-dependent selection dynamics governed by karyotypic diversity. We evaluate evidence for subpopulation interactions that may influence tumor fitness, highlighting candidate karyotypes that serve as ecological interaction hubs. This framework provides testable predictions of intratumoral ecological dynamics and highlights the potential of targeting karyotypically distinct subpopulations to disrupt tumor evolution strategically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffff279b4407d3734303c6d5c0f841102d26d8b" target='_blank'>
              Inverse Game Theory Characterizes Frequency-Dependent Selection Driven by Karyotypic Diversity in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Noemi Andor
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccad37f8db740ee741e31d76d75cb5a07f7e923f" target='_blank'>
              Biology as vulnerability in follicular lymphoma: genetics, epigenetics, and immunogenetics.
              </a>
            </td>
          <td>
            F. Araujo-Ayala, W. Béguelin
          </td>
          <td>2025-03-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Breast cancer remains a significant global health challenge due to its complexity, which arises from multiple genetic and epigenetic mutations that originate in normal breast tissue. Traditional machine learning models often fall short in addressing the intricate gene interactions that complicate drug design and treatment strategies. In contrast, our study introduces GEMDiff, a novel computational workflow leveraging a diffusion model to bridge the gene expression states between normal and tumor conditions. GEMDiff augments RNAseq data and simulates perturbation transformations between normal and tumor gene states, enhancing biomarker identification. GEMDiff can handle large-scale gene expression data without succumbing to the scalability and stability issues that plague other generative models. By avoiding the need for task-specific hyper-parameter tuning and specific loss functions, GEMDiff can be generalized across various tasks, making it a robust tool for gene expression analysis. The model’s ability to augment RNA-seq data and simulate gene perturbations provides a valuable tool for researchers. This capability can be used to generate synthetic data for training other machine learning models, thereby addressing the issue of limited biological data and enhancing the performance of predictive models. The effectiveness of GEMDiff is demonstrated through a case study using breast mRNA gene expression data, identifying 307 core genes involved in the transition from a breast tumor to a normal gene expression state. GEMDiff is open source and available at https://github.com/xai990/GEMDiff.git under the MIT license.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3472d3e5ad8a58f2f1d0054cb37345e502bfbbc" target='_blank'>
              GEMDiff: a diffusion workflow bridges between normal and tumor gene expression states: a breast cancer case study
              </a>
            </td>
          <td>
            Xusheng Ai, Melissa C. Smith, F. A. Feltus
          </td>
          <td>2025-03-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background In the course of tumor treatment, radiation therapy (RT) not only kills cancer cells, but also induces complex biological effects in non-malignant cells around cancer cells. These biological effects such as angiogenesis, changes in stromal composition and immune cell infiltration remodel the tumor microenvironment (TME). As one of the major components of the TME, Cancer‑associated fibroblasts (CAFs) are not only involved in tumorigenesis, progression, recurrence, and metastasis but also regulate the tumor-associated immune microenvironment. CAFs and tumor cells or immune cells have complex intercellular communication in the context of tumor radiation. Main content. Different cellular precursors, spatial location differences, absence of specific markers, and advances in single-cell sequencing technology have gradually made the abundant heterogeneity of CAFs well known. Due to unique radioresistance properties, CAFs can survive under high doses of ionizing radiation. However, radiation can induce phenotypic and functional changes in CAFs and further act on tumor cells and immune cells to promote or inhibit tumor progression. To date, the effect of RT on CAFs and the effect of irradiated CAFs on tumor progression and TME are still not well defined. Conclusion In this review, we review the origin, phenotypic, and functional heterogeneity of CAFs and describe the effects of RT on CAFs, focusing on the mutual crosstalk between CAFs and tumor or immune cells after radiation. We also discuss emerging strategies for targeted CAFs therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4e7d8569cb5aa3f3831a8692c395636f7587796" target='_blank'>
              Cancer‑associated fibroblasts: a pivotal regulator of tumor microenvironment in the context of radiotherapy
              </a>
            </td>
          <td>
            Linhui Zheng, Wenqi Cai, Yuan Ke, Xiaoyan Hu, Chunqian Yang, Runze Zhang, Huachao Wu, Dong Liu, Haijun Yu, Chaoyan Wu
          </td>
          <td>2025-03-20</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Tumour‐associated microbiota are integral components of the tumour microenvironment (TME). However, previous studies on intratumoral microbiota primarily rely on bulk tissue analysis, which may obscure their spatial distribution and localized effects. In this study, we applied in situ spatial‐profiling technology to investigate the spatial distribution of intratumoral microbiota in breast cancer and their interactions with the local TME. Using 5R 16S rRNA gene sequencing and RNAscope FISH/CISH on patients’ tissue, we identified significant spatial heterogeneity in intratumoral microbiota, with Fusobacterium nucleatum (F. nucleatum) predominantly localized in tumour cell‐rich areas. GeoMx digital spatial profiling (DSP) revealed that regions colonized by F. nucleatum exhibit significant influence on the expression of RNAs and proteins involved in proliferation, migration and invasion. In vitro studies indicated that co‐culture with F. nucleatum significantly stimulates the proliferation and migration of breast cancer cells. Integrative spatial multi‐omics and co‐culture transcriptomic analyses highlighted the MAPK signalling pathways as key altered pathways. By intersecting these datasets, VEGFD and PAK1 emerged as critical upregulated proteins in F. nucleatum‐positive regions, showing strong positive correlations with MAPK pathway proteins. Moreover, the upregulation of VEGFD and PAK1 by F. nucleatum was confirmed in co‐culture experiments, and their knockdown significantly reduced F. nucleatum‐induced proliferation and migration. In conclusion, intratumoral microbiota in breast cancer exhibit significant spatial heterogeneity, with F. nucleatum colonization markedly altering tumour cell protein expression to promote progression and migration. These findings provide novel perspectives on the role of microbiota in breast cancer, identify potential therapeutic targets, and lay the foundation for future cancer treatments. Key points Intratumoral Fusobacterium nucleatum exhibits significant spatial heterogeneity within breast cancer tissues. F. nucleatum colonization alters the expression of key proteins involved in tumour progression and migration. The MAPK signalling pathway is a critical mediator of F. nucleatum‐induced breast cancer cell proliferation and migration. VEGFD and PAK1 are potential therapeutic targets to mitigate F. nucleatum‐induced tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b181d8c04116c66c673f00453e25d5cfea9b8649" target='_blank'>
              Integrated spatial multi‐omics profiling of Fusobacterium nucleatum in breast cancer unveils its role in tumour microenvironment modulation and cancer progression
              </a>
            </td>
          <td>
            Feng Zhao, Rui An, Yilei Ma, Shaobo Yu, Yuzhen Gao, Yanzhong Wang, Haitao Yu, Xinyou Xie, Jun Zhang
          </td>
          <td>2025-03-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5d48556d03df2b88d1c06ed57c2bca71cfaf9b2" target='_blank'>
              Spatial genomics reveals cholesterol metabolism as a key factor in colorectal cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Andrew J. Kavran, Yulong Bai, Brian Rabe, Anna Kreshock, Andrew Fisher, Yelena Cheng, Anne Lewin, Chao Dai, Matthew J. Meyer, Konstantinos J. Mavrakis, Anna Lyubetskaya, Eugene Drokhlyansky
          </td>
          <td>2025-03-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) presents significant treatment challenges due to its high heterogeneity and complex intercellular interactions. Further exploration of CRC subtypes and interactions among tumor-specific clusters will facilitate the development of personalized treatment strategies. Methods Single-cell RNA sequencing and bulk RNA sequencing datasets were integrated to determine CRC metabolic subtypes by hierarchical clustering. The analysis was further extended to cellchat, pseudotime, immune infiltration, and clinicopathological relevance to explore the characteristics of secreted frizzled related protein 2 (SFRP2) + cancer-associated fibroblast (CAF) clusters, and validated by spatial transcriptomics (ST), in vivo experiments, and multiple immunohistochemistry (mIHC). Results CRC samples were stably classified into three heterogeneous metabolic subtypes, each exhibiting different microenvironment and CAF heterogeneity, particularly in the distribution of SFRP2 + CAF, which was aligned with metabolic activity. SFRP2 + CAF exhibits high extracellular matrix (ECM) activity and is closely involved in cellular communication, not only promoting the malignant progression of cancer cells but also inducing the differentiation of Tregs. Compared to responders of chemotherapy, the proportion of SFRP2 + CAFs is significantly increased in non-responders. Importantly, mIHC and ST analyses confirm that cancer cells with low expression of agmatinase (AGMAT) can recruit SFRP2 + CAFs, and Treg infiltration surrounding SFRP2 + CAFs was observed. AGMAT combined with oxaliplatin showed the best efficacy in vivo, which may be associated with the inhibition of SFRP2 + CAF infiltration. Conclusions Our study identified and described the potential protumor biological properties of SFRP2 + CAFs, and AGMAT may be a valuable target for disrupting their properties. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06103-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/997e954cb156caa69cf5f6fb240e0f9297f38fc1" target='_blank'>
              Single-cell and spatial-resolved profiling reveals cancer-associated fibroblast heterogeneity in colorectal cancer metabolic subtypes
              </a>
            </td>
          <td>
            Youpeng Wang, Xingfeng Qiu, Qinghai Li, Jiale Qin, Lvlan Ye, Xiang Zhang, Xingxiang Huang, Xiangqiong Wen, Ziyang Wang, Weiling He, Yuqin Di, Qi Zhou
          </td>
          <td>2025-02-11</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Kaposi sarcoma (KS) is an AIDS-defining cancer and a significant global health challenge caused by KS-associated herpesvirus (KSHV). NGS-based approaches have profiled KS lesions in a minimal number of studies compared with other neoplastic diseases. Here we present a compiled and harmonized dataset of 131 KS and non-tumor cutaneous samples in the context of their predicted pathway activities, immune infiltrate, KSHV and HIV gene expression profiles, and their associated clinical data representing patient populations from Argentina, United States (USA), and Sub-Saharan Africa cohorts. RNA-seq data from 9 Argentinian KS lesions were generated and integrated with previously published datasets derived from the USA and sub-Saharan African cohorts from Tanzania, Zambia, and Uganda. An unsupervised analysis of 131 KS-related samples allowed us to identify four KS clusters based on their host and KSHV gene expression profiles, immune infiltrate, and the activity of specific signaling pathways. The compiled RNA-seq profile is shared with the research community through the UCSC Xena browser for further visualization, download, and analysis (https://kaposi.xenahubs.net/ ). These resources will allow biologists without bioinformatics knowledge to explore and correlate the host and viral transcriptome in a curated dataset of different KS RNA-seq-based cohorts, which can lead to novel biological insights and biomarker discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d234979ed4df4c2f534d8267ccbd3a7d7ce309" target='_blank'>
              Integrative Functional Genomics Analysis of Kaposi Sarcoma Cohorts
              </a>
            </td>
          <td>
            E. Lacunza, Valeria Fink, J. Naipauer, María E. Salas, Ana M. Gun, M. Goldman, Jingchun Zhu, Sion Williams, María I. Figueroa, P. Cahn, Omar Coso, Ethel Cesarman, Juan C. Ramos, M. C. Abba
          </td>
          <td>2025-03-11</td>
          <td>Research Square</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e89dc2d087d9d2c969c5bb0a4b45e01fc9cc0f7a" target='_blank'>
              Single-cell and spatial genomic landscape of non-small cell lung cancer brain metastases.
              </a>
            </td>
          <td>
            Somnath Tagore, Lindsay A. Caprio, Amit Dipak Amin, K. Bestak, Karan Luthria, Edridge K. D’Souza, Irving Barrera, Johannes C. Melms, Sharon Wu, Sinan Abuzaid, Yiping Wang, Viktoria Jakubikova, Peter Koch, D. Z. Brodtman, Banpreet Bawa, S. Deshmukh, Leon Ebel, Miguel A. Ibarra-Arellano, A. Jaiswal, Carino Gurjao, Jana Biermann, Neha Shaikh, Priyanka Ramaradj, Yohanna Georgis, Galina G Lagos, Matthew I Ehrlich, P. Ho, Zachary H. Walsh, Meri Rogava, Michelle Garlin Politis, Devanik Biswas, A. Cottarelli, Nikhil Rizvi, CA Shu, Benjamin O Herzberg, N. Anandasabapathy, G. Sledge, Emmanuel Zorn, Peter D Canoll, Jeffrey N. Bruce, Naiyer A Rizvi, Alison M Taylor, Anjali Saqi, H. Hibshoosh, Gary K Schwartz, Brian S Henick, Fei Chen, Denis Schapiro, Parin Shah, Benjamin Izar
          </td>
          <td>2025-02-27</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539a7161f9d5e59af0e8571e13d64e98e255b165" target='_blank'>
              Single-cell sequencing reveals the role of aggrephagy-related patterns in tumor microenvironment, prognosis and immunotherapy in endometrial cancer
              </a>
            </td>
          <td>
            Yuquan Yuan, Chunyan Ren, Jin Shu, Keyang Zhu, Ganghui Li, Bao Liu, Jianrong Huang, Yinde Huang, Chengzhi Zhao
          </td>
          <td>2025-03-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most lethal brain cancer, with GBM stem cells (GSCs) driving therapeutic resistance and recurrence. Targeting GSCs offers a promising strategy for preventing tumor relapse and improving outcomes. We identify SUV39H1, a histone-3, lysine-9 methyltransferase, as critical for GSC maintenance and GBM progression. SUV39H1 is upregulated in GBM compared with normal brain tissues, with single-cell RNA-seq showing its expression predominantly in GSCs due to super-enhancer–mediated activation. Knockdown of SUV39H1 in GSCs impaired their proliferation and stemness. Whole-cell RNA-seq analysis revealed that SUV39H1 regulates G2/M cell cycle progression, stem cell maintenance, and cell death pathways in GSCs. By integrating the RNA-seq data with ATAC-seq data, we further demonstrated that knockdown of SUV39H1 altered chromatin accessibility in key genes associated with these pathways. Chaetocin, an SUV39H1 inhibitor, mimics the effects of SUV39H1 knockdown, reducing GSC stemness and sensitizing cells to temozolomide, a standard GBM chemotherapy. In a patient-derived xenograft model, targeting SUV39H1 inhibits GSC-driven tumor growth. Clinically, high SUV39H1 expression correlates with poor glioma prognosis, supporting its relevance as a therapeutic target. This study identifies SUV39H1 as a crucial regulator of GSC maintenance and a promising therapeutic target to improve GBM treatment and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc58ec84c5d0f0ea9dd38953926222f20c648f30" target='_blank'>
              SUV39H1 maintains cancer stem cell chromatin state and properties in glioblastoma
              </a>
            </td>
          <td>
            Chunying Li, Qiqi Xie, Sugata Ghosh, Bihui Cao, Yuanning Du, G. V. Vo, Timothy Y. Huang, Charles Spruck, Richard L Carpenter, Y. A. Wang, Q. R. Lu, K. Nephew, Jia Shen
          </td>
          <td>2025-03-10</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is a highly aggressive brain tumor associated with poor survival outcomes and is driven by a complex tumor microenvironment (TME) that promotes tumor progression and treatment resistance. To explore the role of the TME in GBM, we analyzed glioma-related microarray and single-cell RNA sequencing (scRNA-seq) datasets from the Gene Expression Omnibus (GEO). Functional enrichment and weighted gene coexpression network analyses revealed distinct immune profiles, metabolic alterations, and differences in chemotherapeutic drug sensitivity between the high-risk and low-risk patient groups. scRNA-seq data processed with the ‘Seurat’ package were used to identify differentially expressed genes in pericytes, endothelial cells, and glioma cells, particularly those involved in extracellular matrix (ECM) remodeling. A 17-gene prognostic signature developed through Cox regression and LASSO analyses revealed that key genes (COL1A1, COL4A1, and VIM) were significantly associated with survival outcomes in GBM patients. Drug sensitivity analyses using data from the Genomics of Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) identified potential targeted therapies for GBM, including SB-505,124, staurosporine, and AZD8186. This integrative study underscores the critical roles of the ECM and synaptic remodeling in GBM and suggests novel therapeutic targets to improve personalized treatment strategies for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/456759f2943a08a601ac9cd5e20189bf3d373750" target='_blank'>
              Molecular mechanisms and therapeutic targets in glioblastoma multiforme: network and single-cell analyses
              </a>
            </td>
          <td>
            Xiangyu Chen, Xiao Zhong, Feifei Zhang, Xiaomei Zhou, Xiaofeng Yue, Xueru Li
          </td>
          <td>2025-03-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5372012557b72610a59f72be2bf39f2688654856" target='_blank'>
              Spatial distribution of the proteome in human body and cancers
              </a>
            </td>
          <td>
            Liang Yue, Wenhao Jiang, Sainan Li, Meng Luo, Ning Fan, Xiaolu Zhan, Kexin Chen, Tian Lu, Fang Guo, Dongwei Li, Weigang Ge, Zongxiang Nie, M. Lyu, A. Jun, Yingrui Wang, Yingdan Chen, Zhenhai Fu, Nan Xiang, Lu Li, Fengchao Yu, Guo Ci Teo, Alexey I. Nesvizhskii, Meng Wang, Michael P. Snyder, Ben C. Collins, Ruedi Aebersold, Fei Xu, Yingchuan Zhu, Tong Liu, Yan Li, Tiannan Guo
          </td>
          <td>2025-02-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background CD4+ T cells are a highly differentiated cell type that maintain enough transcriptomic plasticity to cycle between activated and memory statuses. How the 1D chromatin state and 3D chromatin architecture support this plasticity is under intensive investigation. Methods Here, we wished to test a commercially available in situ Hi-C kit (Arima Genomics Inc.) to establish whether published performance on limiting cell numbers from clonal cell lines copies across to a primary immune cell type. We achieved comparable contact matrices from 50,000, 250,000, and 1,000,000 memory CD4+ T-cell inputs. We generated multiple Hi-C and RNA-seq libraries from the same biological blood donors under three separate conditions: unstimulated fresh ex vivo, IL-2-only stimulated, and T cell receptor (TCR)+CD28+IL-2-stimulated, conferring increasingly stronger activation signals. We wished to capture the magnitude and progression of 3D chromatin shifts and correlate these to expression changes under the two stimulations. Results Although some genome organization changes occurred concomitantly with changes in gene expression, at least as many changes occurred without corresponding changes in expression. Counter to the hypothesis that topologically associated domains (TADs) are largely invariant structures providing a scaffold for dynamic looping contacts between enhancers and promotors, we found that there were at least as many dynamic TAD changes. Stimulation with IL-2 alone triggered many changes in genome organization, and many of these changes were strengthened by additional TCR and CD28 co-receptor stimulation. Conclusions This suggests a stepwise process whereby mCD4+ T cells undergo sequential buildup of 3D architecture induced by distinct or combined stimuli likely to “prime” or “deprime” them for expression responses to subsequent TCR-antigen ligation or additional cytokine stimulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd1aa3f310b47dde1e0fdd44c2b4206007ebe632" target='_blank'>
              Memory CD4+ T cells sequentially restructure their 3D genome during stepwise activation
              </a>
            </td>
          <td>
            Alexander I. Ward, J. I. de las Heras, Eric C. Schirmer, A. Fassati
          </td>
          <td>2025-02-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Cancer metastasis, a process in which cancer cells migrate to secondary sites, accounts for 90% of cancer deaths. While machine learning has been used to predict metastasis in a variety of ways, they tend to be specific to tumor types or classification tasks. Here, we provide a pan-cancer machine learning model that is easily interpretable and directly links the transcriptomic and proteomic profiles of hundreds of cancer cell lines from DepMap with metastasis in a tissue-agnostic manner. We show that, for the relatively small sample size here, linear models perform just as well as non-linear models, including neural networks. We also demonstrate that transcriptomics outperforms proteomics for predictive tasks. However, the combination of both transcriptomics and proteomics leads to an overall increase in performance, indicating that these modalities can complement each other. Finally, we demonstrate that those features identified as most important by the joint omics linear model fit are over-represented in biological pathways related to metastasis, suggesting that the model learns relevant information that may be useful in identifying universal biomarkers or therapeutic targets of cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4778538fa7f9077b250641d66da47d7e300ae2d7" target='_blank'>
              Joint linear modeling of transcriptomics and proteomics is predictive of cancer metastasis
              </a>
            </td>
          <td>
            Ronit Sharma, N. Meimetis, Arjana Begzati, S. Nagar, Benjamin Kellman, Hratch M. Baghdassarian
          </td>
          <td>2025-02-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Motivation Spatial transcriptomic technologies allow researchers to explore the diversity and specificity of gene expression within their original tissue structure. Accurately identifying regions that are spatially coherent in both gene expression and physical tissue structures is an emerging topic, but challenging due to the lack of ground truth labels which renders complicating validation of clustering consistency and reproducibility. This highlights a need for a computational evaluation framework to rigorously and unbiasedly assess clustering performance. Results To address this gap, we propose STEAM (Spatial Transcriptomics Evaluation Algorithm and Metric), a user-friendly computational pipeline designed to evaluate the consistency and reliability of clustering results by leveraging machine learning classification and prediction methods, with the goal of maintaining the spatial proximity and gene expression patterns within clusters. We benchmarked STEAM on various public datasets, spanning multi-cell to single-cell resolution, as well as spatial transcriptomics and proteomics. The results highlighted its robustness and generalizability through comprehensive statistical evaluation metrics, such as Kappa score, F1 score, accuracy, and adjusted rand index. Notably, STEAM supports multi-sample training, enabling cross-replicate clustering consistency assessment. Moreover, STEAM provides practical guidance by comparing clustering results across multiple approaches; here, we evaluated four different methods, including spatial-aware and spatial-ignorant approaches. In summary, we believe that STEAM provides researchers a promising tool for evaluating clustering robustness and benchmarking clustering performance for spatial omics data, offering valuable insights to drive reproducible discoveries in spatial biology. Availability and implementation Source code and the R software tool STEAM are available from https://github.com/fanzhanglab/STEAM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a0bb40c1ac826ccb8d6dd553bb5b01c202994f" target='_blank'>
              STEAM: Spatial Transcriptomics Evaluation Algorithm and Metric for clustering performance
              </a>
            </td>
          <td>
            Samantha Reynoso, Courtney Schiebout, Revanth Krishna, Fan Zhang
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56c8bae667d5d467ee9b1edf8b1b3bbcf92dc895" target='_blank'>
              Single-cell sequencing for lung cancer research: Progress and prospects
              </a>
            </td>
          <td>
            Zhongsha Li, Wei Su, Bingbing Bai, Yuesong Wu, Baoyin Zhang, Dejuan Xiang, Gilwa Borjigin, Jinhai Deng, Jian Sun, Mingzhu Yin
          </td>
          <td>2025-03-12</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The amazing diversity of B cells within the tumor microenvironment is the basis for the diverse development of B cell-based immunotherapies. Here, we focus on elucidating the mechanisms of tumor intervention mediated by four tumor-infiltrating B lymphocytes. Naive B cells present the initial antigen, germinal center B cell subsets enhance antibody affinity, and immunoglobulin subtypes exert multiple immune effects, while regulatory B cells establish immune tolerance. Together they reflect the complexity of the changing dynamics of cancer immunity. Additionally, we have investigated the dynamic effects of tumor-infiltrating B lymphocytes in immunotherapy and their relationship to prognosis, providing new insights into potential treatment strategies for patients. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-024-03936-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/787ec1e24f0f9b35e2346f89facf0c57ddb91948" target='_blank'>
              Dynamic roles of tumor-infiltrating B lymphocytes in cancer immunotherapy
              </a>
            </td>
          <td>
            Shishengnan Song, Chong Wang, Yangchao Chen, Xiaorong Zhou, Yi Han, Haijian Zhang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Glioblastoma is the most common and aggressive malignant brain tumour in the adult population and its prognosis is dismal. The heterogeneous nature of the tumour, to which epigenetic dysregulation significantly contributes, is among the main therapeutic challenges of the disease. Results We have leveraged SYNGN, an experimental pipeline enabling the syngeneic comparison of glioblastoma stem cells and expanded potential stem cell (EPSC)-derived neural stem cells to identify regulatory features driven by chromatin remodelling specifically in glioblastoma stem cells. Conclusions We show epigenetic regulation of the expression of genes and related signalling pathways contributing to glioblastoma development. We also identify novel epigenetically regulated druggable target genes on a patient-specific level, including SMOX and GABBR2. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02127-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba7e1fd60eab1467a858b79695c0c9e7c85c584" target='_blank'>
              Mapping chromatin remodelling in glioblastoma identifies epigenetic regulation of key molecular pathways and novel druggable targets
              </a>
            </td>
          <td>
            C. Vinel, James Boot, Weiwei Jin, N. Pomella, Alexandra Hadaway, C. Mein, N. Zabet, Silvia Marino
          </td>
          <td>2025-02-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer driver mutations are defined by their high prevalence in cancers and presumed rarity in normal tissues. However, recent studies show that positive selection in normal epithelia can increase the prevalence of some cancer drivers. To determine their true cancer-driving potential, it is essential to evaluate how frequent these mutations are in normal tissues and what are their phenotypes. Here, we explore the bioavailability of somatic variants by quantifying age-related mutational burdens in normal human colonic epithelium using immunodetection in FFPE samples (N = 181 patients). Positive selection of variants of tumour suppressor genes PTEN and ARID1A associates with monoallelic gene loss as confirmed by CRISPR/Cas9 mutagenesis and changes in their downstream effectors. Comparison of the mutational burden in normal tissue and colorectal cancers allows quantification of cancer driver potency based on relative representation. Additionally, immune exclusion, a cancer hallmark feature, is observed within ARID1A-deficient clones in histologically normal tissue. The behaviour resulting from haploinsufficiency of PTEN and ARID1A demonstrates how somatic mosaicism of tumour suppressors arises and can predispose to cancer initiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab8b68e7b5a2f581513237acff3c4e88418f66b" target='_blank'>
              Haploinsufficient phenotypes promote selection of PTEN and ARID1A-deficient clones in human colon
              </a>
            </td>
          <td>
            Nefeli Skoufou-Papoutsaki, Sam Adler, Shenay Mehmed, Claire Tume, Cora Olpe, Edward Morrissey, Richard Kemp, Anne-Claire Girard, Elisa B. Moutin, C. Chilamakuri, Jodi L Miller, Cecilia Lindskog, Fabian Werle, Kate Marks, Francesca Perrone, Matthias Zilbauer, D. Tourigny, Doug Winton
          </td>
          <td>2025-02-07</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Glioblastoma (GBM) is the most common and aggressive primary malignant brain tumor of adults. Current therapeutic options yield dismal prognoses that have remained essentially unchanged over nearly two decades. Diffuse growth patterns, high intratumoral heterogeneity, and variable blood–brain barrier integrity limit treatment efficacy, creating challenges that rational small molecule design has not overcome. Antibody–drug conjugates have shown some promise, leading us to hypothesize that smaller folded DNA aptamers, developed in vivo via principles of natural selection, might eventually have advantages for drug delivery. Here, we document the first in vivo DNA aptamer selection involving an orthotopic patient-derived xenograft GBM mouse model to identify tumor-homing DNA aptamers. We demonstrate the preferential accumulation of these aptamers in the tumor relative to other tissues 4 h after intraperitoneal injection. The aptamers can be detected by quantitative polymerase chain reaction, fluorescent tumor staining, and stain GBM tumor section from untreated mice and the GBM tumor cells in culture. Two of three candidates are selective for the target cell line in vitro and do not bind other human tumor cells. In vivo selection of tumor-specific DNA aptamers demonstrates a novel approach for diagnostics or toxin delivery that might allow for the development of individualized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a69e4c2d2724fca734736e524ece9241f19793a" target='_blank'>
              In vivo selection of anti-glioblastoma DNA aptamers in an orthotopic patient-derived xenograft model
              </a>
            </td>
          <td>
            Caroline D. Doherty, Brandon A. Wilbanks, Sonia Jain, Keenan S Pearson, Katie K Bakken, D. Burgenske, Nay Won Lett, J. Sarkaria, L. Maher
          </td>
          <td>2025-02-18</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and lethal malignancy, with a five-year survival rate of just 50% for cases of locally advanced disease. Chromosomal aberrations, particularly the deletion of the short arm of chromosome 3 (3p), have been strongly associated with poor prognosis and more aggressive tumor phenotypes. The tumor microenvironment (TME) plays a pivotal role in tumor progression and resistance to therapy. This study aims to elucidate the impact of 3p deletion on the TME, immune cell infiltration, and treatment response in HNSCC, to identify novel therapeutic targets to improve patient outcomes. We analyzed single-cell RNA sequencing (scRNA-seq) data from the Gene Expression Omnibus (GEO) and bulk transcriptome data from The Cancer Genome Atlas (TCGA). Pseudo-time trajectory and cell-cell communication analyses were performed with the Monocle and CellChat packages. The Wilcoxon test was used to evaluate the differential gene expression between wild-type (wt) and mutant (mut) groups. Prognostic models were developed using the Least Absolute Shrinkage and Selection Operator (LASSO) algorithm and Cox regression analyses to find the genes related to survival, with survival analysis conducted via Kaplan-Meier curves. Gene set enrichment analysis (GSEA) was employed to investigate pathway dysregulation, and immune cell infiltration was assessed using various immune scoring methodologies to explore the differences immune environment. The Tumor Immune Dysfunction and Exclusion (TIDE) database was utilized to predict the potential efficacy of immune checkpoint inhibitors. mRNA and protein expression levels of SPP1 were examined by RT-qPCR and Western blotting, while cell proliferation was assessed using the CCK8 assay. The mut group demonstrated significant alterations in cellular composition, characterized by increased endothelial cells and macrophages and decreased fibroblasts and CD8 + T cells, indicative of an immunosuppressive TME. Differential expression analysis revealed downregulation of immune pathways, including antigen processing and presentation, T cell receptor signaling, and B cell receptor signaling pathways in the mut group, along with enhanced metabolic activity in glycolysis and lipid metabolism. The prognostic model identified nine key genes associated with poor survival in HNSCC. The mut group exhibited poorer overall survival and a more immunosuppressive microenvironment compared to the wt group, which correlated with the outcomes observed in high-risk versus low-risk groups. High-risk patients also showed a diminished response to immunotherapy compared to low-risk patients. Additionally, SPP1 emerged as a critical gene associated with chemotherapy resistance and macrophage M2 polarization. This study demonstrates that 3p deletion significantly reshapes the TME, contributing to poor prognosis in HNSCC by fostering an immunosuppressive environment and enhancing chemoresistance. These findings highlight the potential for developing targeted therapies that address the genetic and immunological landscape of HNSCC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92078-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="To better understand the homeostatic mechanisms governing melanocytes, we performed deep phenotyping of clonal expansions of single melanocytes from human skin. In total, we interrogated the mutational landscapes, gene expression profiles, and morphological features of 297 melanocytes from 31 donors. To our surprise, a population of melanocytes with low mutation burden was maintained in sun damaged skin. These melanocytes were more stem-like, smaller, less dendritic and displayed distinct gene expression profiles compared to their counterparts with high mutation burdens. We used single-cell spatial transcriptomics (10X Xenium) to reveal the spatial distribution of melanocytes inferred to have low and high mutation burdens (LowMut and HighMut cells), based on their gene expression profiles. LowMut melanocytes were found in hair follicles as well as in the interfollicular epidermis, whereas HighMut melanocytes resided almost exclusively in the interfollicular epidermis. We propose that melanocytes in the hair follicle occupy a privileged niche, protected from UV radiation, but periodically migrate out of the hair follicle to replenish the interfollicular epidermis after waves of photodamage. More broadly, our study illustrates the advantages of a cell atlas that includes mutational information, as cells can change their cellular states and positional coordinates over time, but mutations are like scars, providing a historical record of the homeostatic processes that were operative on each cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35fcdb72f7ddf292a52ab622c06a7d378c43f17" target='_blank'>
              Somatic mutations distinguish melanocyte subpopulations in human skin
              </a>
            </td>
          <td>
            Bishal Tandukar, Delahny Deivendran, Limin Chen, Neda Bahrani, Beatrice Weier, Harsh Sharma, Noel Cruz-Pacheco, Min Hu, Kayla Marks, Rebecca G. Zitnay, Aravind K. Bandari, Rojina Nekoonam, I. Yeh, R. Judson-Torres, A. H. Shain
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01243dcc84f5d3d7fbdd4aa36ab8b1b63e350fae" target='_blank'>
              Abstract A086: Myeloid suppression within metastatic tumor microenvironments in advanced breast cancer—friend or foe to checkpoint inhibition?
              </a>
            </td>
          <td>
            Batul Al-zubeidy, Edgar Gonzalez, A. Baugh, Dominic Zavala, Mathew Jacobo, Michael F. Press, E. R. Torres
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f6e80235cbbb65b993c05ceb81c0545572bfc88" target='_blank'>
              Robust Spatial Cell-Type Deconvolution with Qualitative Reference for Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Qishi Dong, Yi Yang, Ziye Luo, Haipeng Shen, Xingjie Shi, Jin Liu
          </td>
          <td>2025-03-09</td>
          <td>Small methods</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad8fec02f869aff60be41da2c4550105b35eaada" target='_blank'>
              Functional Tumor Boundaries Revealed by Dynamic Electroimpedance Imaging Under Weak Stimuli: Dissipative Structures in Melanoma Area
              </a>
            </td>
          <td>
            Yuri F. Babich
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Intrahepatic cholangiocarcinoma (iCCA) may be genomically subclassified by the presence of potentially actionable molecular aberrations, of which pathogenic alterations in isocitrate dehydrogenase (IDH)1 and fibroblast growth factor receptor (FGFR)2 are the most frequently observed. The impact of these molecular alterations on the tumor immune microenvironment remains incompletely understood. Methods: We performed a high-parameter spatial immune phenotyping of iCCA samples with pathogenic FGFR2 or IDH1 alterations and FGFR2/IDH1 wild-type controls at the single-cell level using CO-Detection by indEXing. Results:﻿ A total of 24 tumors were examined. Tumors with FGFR2 alterations were characterized by fewer CD8+ T cells and “M2-like” macrophages but higher levels of polymorphonuclear myeloid-derived suppressor cells as compared to FGFR2 wild-type tumors. Spatial relationships between polymorphonuclear myeloid-derived suppressor cells and multiple other cell types in the tumor microenvironment (including tumor cells, CD4+, and CD8+ T cells) were enriched in tumors with FGFR2 alterations. Tumors with IDH1 mutations had a trend toward more fibroblasts and were characterized by a closer proximity of tumor cells to CD4+ T cells, and between macrophages and multiple structural tumor microenvironment components as compared to other subtypes. Conclusions: iCCAs with pathogenic FGFR2 fusions/rearrangements and IDH1 mutations have distinct immunophenotypes. Tailoring immunotherapeutic approaches to specific molecular subsets could improve treatment outcomes across the divergent molecularly defined iCCA subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/343c87af8f0d2b156de3f86c823598f7aa36e2b7" target='_blank'>
              Deep immune profiling of intrahepatic cholangiocarcinoma with CODEX multiplexed imaging
              </a>
            </td>
          <td>
            M. Baretti, Soumya Shekhar, Vaibhav Sahai, Daniel H. Shu, Kathryn Howe, Valerie Gunchick, N. Assarzadegan, Emma Kartalia, Qingfeng Zhu, E. Hallab, Archit Sheth-Shah, Aya Kondo, Nilofer S Azad, M. Yarchoan
          </td>
          <td>2025-02-19</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Abstract Solid tumours are intricate and highly heterogeneous ecosystems, which grow in and invade normal organs. Their progression is mediated by cancer cells’ interaction with different cell types, such as immune cells, stromal cells and endothelial cells, and with the extracellular matrix. Owing to its high incidence, aggressive growth and resistance to local and systemic treatments, liver cancer has particularly high mortality rates worldwide. In recent decades, spatial heterogeneity has garnered significant attention as an unfavourable biological characteristic of the tumour microenvironment, prompting extensive research into its role in liver tumour development. Advances in spatial omics have facilitated the detailed spatial analysis of cell types, states and cell‒cell interactions, allowing a thorough understanding of the spatial and temporal heterogeneities of tumour microenvironment and informing the development of novel therapeutic approaches. This review illustrates the latest discovery of the invasive zone, and systematically introduced specific macroscopic spatial heterogeneities, pathological spatial heterogeneities and tumour microenvironment heterogeneities of liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27ba251a9613c3090a1b350987469c7d7cfc3f02" target='_blank'>
              Spatial‒temporal heterogeneities of liver cancer and the discovery of the invasive zone
              </a>
            </td>
          <td>
            Jiayan Yan, Zhifeng Jiang, Shiyu Zhang, Qichao Yu, Yijun Lu, Runze Miao, Zhaoyou Tang, Jia Fan, Liang Wu, Dan G. Duda, Jian Zhou, Xinrong Yang
          </td>
          <td>2025-02-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/903982cf411415d6dcdb66121d6c8e705f6172b9" target='_blank'>
              The Role of Cancer Stem Cells in Uveal Melanoma
              </a>
            </td>
          <td>
            Piotr Świerzewski, Bożena Romanowska-Dixon, Martyna Elas
          </td>
          <td>2025-02-14</td>
          <td>Ophthalmology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Understanding the proteomic-level heterogeneity of the tumor microenvironment (TME) in colorectal cancer (CRC) is crucial due to its well-known heterogeneity. While heterogenous CRC has been extensively characterized at the molecular subtype level, research into the functional heterogeneity of fibroblasts, particularly their relationship with extracellular matrix (ECM) alterations, remains limited. Addressing this gap is essential for a comprehensive understanding of CRC progression and the development of targeted therapies.


METHODS
24 tissue samples from 21 CRC patients, along with adjacent normal tissues (NAT), were collected and decellularized using a detergent-based method to enrich the ECM component. Proteomic analysis of ECM-enriched samples was performed using tandem mass tag (TMT) spectrometry, followed by statistical analysis including differential expression protein (DEP) analysis. Single-cell RNA sequencing (scRNA-Seq) data from public datasets were integrated and analyzed to delineate cell states within the TME. Bulk tissue RNA-Seq and bioinformatics analysis, including consensus molecular subtype (CMS) classification and single-cell level deconvolution of TCGA bulk RNA-seq data, were conducted to further explore gene expression patterns and TME composition.


RESULTS
Differential cellular origin of the NAT and tumorous ECM proteins were identified, revealing 110 ECM proteins enriched in NAT and 28 ECM proteins in tumor tissues. Desmoplastic and WNT5A+ inflammatory fibroblasts were indicated as the sources of tumor-enriched ECM proteins, while ADAMDEC1+ expressing fibroblasts and PI16+ expressing fibroblast were identified as the sources of NAT-enriched ECM proteins. Deconvolution of bulk RNA-seq of CRC tissues discriminated CMS-specific fibroblast state, reflecting the biological traits of each CMS subtype. Specially, seven ECM genes specific to mesenchymal subtype (CMS4), including PI16+ fibroblast-related 4 genes (SFRP2, PRELP, OGN, SRPX) and desmoplastic fibroblast-related 3 genes (THBS2, CTHRC1, BGN), showed a significant association with poorer survival in patient with CRC.


CONCLUSION
We conducted an extracellular matrix (ECM)-focused profiling of the TME by integrating quantitative proteomics with single-cell RNA sequencing (scRNA-seq) data from CRC patients. We identified the ECM proteins of NAT and tumor tissue, and established a cell-matrisome database. We defined mesenchymal subtype-specific molecules associated with specific fibroblast subtypes showing a significant association with poorer survival in patients with CRC. Our ECM-focused profiling of tumor stroma provides new insights as indicators for biological processes and clinical endpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2f30006d88490586d800953c99040977f4d92b" target='_blank'>
              Differential cellular origins of the extracellular matrix of tumor and normal tissues according to colorectal cancer subtypes.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Sangkeun Park, Jun Hyeong Lee, Shin Young Chang, Sang Mi Oh, S. Mun, Junho Kang, Jong-Eun Park, Jung Kyoon Choi, Tae Il Kim, Jin Young Kim, Pilnam Kim
          </td>
          <td>2025-03-03</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0608cf84711b7461bf4c33725871d3360fc3c9a5" target='_blank'>
              Abstract B047: Stromal senescence and response to immune checkpoint inhibition in melanoma
              </a>
            </td>
          <td>
            Giselle Burton Sojo, Marcel O. Schmidt
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Fanconi Anemia (FA) is a heritable syndrome characterized by DNA damage repair deficits, frequent malformations and a significantly elevated risk of bone marrow failure, leukemia, and mucosal head and neck squamous cell carcinomas (HNSCC). Hematopoietic stem cell gene therapy can prevent marrow failure and lower leukemia risk, but mucosal gene therapy to lower HNSCC risk remains untested. Major knowledge gaps include an incomplete understanding of how rapidly gene-corrected cellular lineages could spread through the oral epithelium, and which delivery parameters are critical for ensuring efficient gene correction. To answer these questions, we extended an agent-based model of the oral epithelium to include the delivery of gene correction in situ to FA cells and the competitive dynamics between cellular lineages with and without gene correction. We found that only gene-corrected lineages with substantial proliferative advantages (probability of resisting displacement out of the basal layer ≥ 0. 1) could spread on clinically relevant timelines, and that these lineages were initially at high risk of loss in the generations following correction. Delivering gene correction to many cells minimizes the risk of loss, while delivery to many distinct locations within a tissue maximizes the rate of spread. To determine the impact of mucosal gene therapy in preventing the clonal expansion of pre-cancerous mutations, we compared the expected burden of TP53 mutations in simulated tissue sections with and without gene correction. We found that when FA cells have elevated genome instability or a TP53-dependent proliferative advantage, gene correction can substantially reduce the accumulation of pro-tumorigenic mutations. This model illustrates the power of computational frameworks to identify critical determinants of therapeutic success to enable experimental optimization and support novel and effective gene therapy applications. Author summary We investigated factors influencing the success of oral mucosal gene therapy for Fanconi Anemia (FA), a genetic syndrome marked by DNA repair defects in conjunction with a heightened risk of cancer. We used a computational model of the oral epithelium to determine how gene therapy corrected cells compete with FA background cells and the best gene delivery approaches to promote effective tissue replacement by gene-corrected cells. We find that gene-corrected cells require strong proliferative advantages to spread effectively, and that initially delivering gene correction to more cells reduces the chance that these cells are stochastically eliminated before they can spread. We also demonstrate that gene correction reduces the accumulation of pro-tumorigenic TP53 mutations in an FA context, where genomic instability can elevate the mutation rate and FA-specific selective pressures could favor accelerated TP53 clonal expansion. This research provides a useful framework for guiding mucosal gene therapy experiments and the development of effective oral gene therapy protocols for cancer prevention in FA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb3b56fe96b81aec45dd517d7b11cc8ca9c35" target='_blank'>
              Epithelial competition determines gene therapy potential to suppress Fanconi Anemia oral cancer risk
              </a>
            </td>
          <td>
            Hunter L. Colegrove, Raymond J Monnat, Alison F Feder
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f3b584bb3e59598c626453f550ef2ecaeb21de4" target='_blank'>
              Abstract A042: Uncovering the mechanism of synergistic anti-tumor effects in FAP-CD40 + PD1-IL2v combination therapy
              </a>
            </td>
          <td>
            Thuy Trinh Nguyen, Leo Kunz, Ines Grazina de Matos, Christine Trumpfheller, P. Umaña
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acb5f12dc4ee23891e41c491f92b49e0c1d5cff9" target='_blank'>
              Targeting CXCR4-expressing TAMs in muscle-invasive bladder cancer to enhance tumor control after immunotherapy
              </a>
            </td>
          <td>
            E. Desponds, H. El ahanidi, Nagham Alouche, Hana Zdimerova, Stéphanie Favre, Giulio Zanette, Sina Nassiri, Daniel Benamran, Petros K Tsantoulis, Mohammed Attaleb, Hélène Maby-El Hajjami, Julien Dagher, Vanessa Gourhand, K. Balabanian, M. Espéli, S. Luther, Camilla Jandus, M. Leblond, G. Verdeil
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Despite advances, Glioblastoma (GBM) treatment remains challenging due to its rapid progression and resistance to therapies. Objectives This study aimed to investigate the role of anoikis—a mechanism by which cells evade programmed cell death upon detachment from the extracellular matrix—in GBM progression and prognosis. Methods Utilizing single-cell sequencing and bulk-transcriptome sequencing data from TCGA, GEO, and CGGA databases, we performed comprehensive bioinformatics analyses. We identified anoikis-related genes, constructed a prognostic model using 101 machine learning algorithms, and validated its clinical utility across multiple cohorts.Finally, we also verified the expression of model genes and the function of key gene in clinical samples and cell lines. Results Single-cell sequencing revealed heterogeneous expression of anoikis-related genes across distinct cell populations within GBM. MES-like Malignant cells and Myeloids exhibited higher enrichment of these genes, implicating their role in anoikis resistance and tumor aggressiveness. The prognostic model, based on identified genes, effectively stratified patients into high-risk and low-risk groups, demonstrating significant differences in survival outcomes. Mutation and tumor microenvironment analyses highlighted distinct genetic landscapes and immune cell infiltration patterns associated with different risk groups. SLC43A3 emerged as a key gene, showing significant upregulation in tumor tissues and correlating with poor prognosis in GBM. Conclusion This study provides insights into the molecular mechanisms of anoikis resistance in GBM, underscoring its critical role in tumor progression and patient prognosis. The developed prognostic model offers a promising tool for personalized treatment strategies and warrants further exploration of targeted therapies to improve outcomes for GBM patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03752-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90776aa9e4e44d828b89f11f0b82b3f7faa8093b" target='_blank'>
              Unraveling anoikis in glioblastoma: insights from single-cell sequencing and prognostic modeling
              </a>
            </td>
          <td>
            Qikai Tang, Chenfeng Ma, Jiaheng Xie, Qixiang Zhang, Bingtao Zhang, Weiqi Bian, Qingyu Lu, Zeyu Wan, Wei Wu
          </td>
          <td>2025-03-26</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8677e695bbefb8bee72d8f6205f8c94375119315" target='_blank'>
              Developmental Origins and Oncogenesis in Medulloblastoma.
              </a>
            </td>
          <td>
            K. Kiang, Yogesh K H Wong, Soma Sengupta, Martine F. Roussel, Q. R. Lu
          </td>
          <td>2025-02-14</td>
          <td>Annual review of neuroscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastomas (GBMs), among the most aggressive and resilient brain tumors, characteristically exhibit high angiogenic potential, leading to the formation of a dense yet aberrant vasculature, both morphologically and functionally. With these premises, numerous expectations were initially placed on anti-angiogenic therapies, soon dashed by their limited efficacy in concretely improving patient outcomes. Neovascularization in GBM soon emerged as a complex, dynamic, and heterogeneous process, hard to manage with the classical standard of care. Growing evidence has revealed the existence of numerous non-canonical strategies of angiogenesis, variously exploited by GBM to meet its ever-increasing metabolic demand and differently involved in tumor progression, recurrence, and escape from treatments. In this review, we provide an accurate description of each neovascularization mode encountered in GBM tumors to date, highlighting the molecular players and signaling cascades primarily involved. We also detail the key architectural and functional aspects characteristic of the GBM vascular compartment because of an intricate crosstalk between the different angiogenic networks. Additionally, we explore the repertoire of emerging therapies against GBM that are currently under study, concluding with a question: faced with such a challenging scenario, could combined therapies, tailored to the patient’s genetic signatures, represent an effective game changer?">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64cd1c95a63b7c6947153915736e2f9dc103a024" target='_blank'>
              Understanding Neovascularization in Glioblastoma: Insights from the Current Literature
              </a>
            </td>
          <td>
            Mariagiovanna Ballato, Emanuela Germanà, Gabriele Ricciardi, Walter Giordano, Pietro Tralongo, Mariachiara Buccarelli, G. Castellani, L. Ricci-Vitiani, Q. D’Alessandris, Giuseppe Giuffrè, Cristina Pizzimenti, V. Fiorentino, Valeria Zuccalà, A. Ieni, M. Caffo, Guido Fadda, M. Martini
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6fe7c1118b5b40de590086593aa209906bee3a7" target='_blank'>
              Abstract A049: Application of a combinatorial functional precision medicine platform to predict immunotherapy response in DLBCL
              </a>
            </td>
          <td>
            Sharon Pei Yi Chan, J. Lim, M. Rashid, Edward Kai-Hua Chow
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49eb9686b4a983fe402a837ae9d2a07843ebb7fe" target='_blank'>
              Knockdown of DNAJC12 slows tumor progression and affects tumor radiosensitivity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiao Ju, Jianbo Zhang, Linke Yang, Pei Li, Ping Wang
          </td>
          <td>2025-03-06</td>
          <td>Molecular &amp; Cellular Biomechanics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell death is critical in tumor biology. The common cancer therapies can cause cell death and alleviate tumor, while the cancer cells can develop a resistance to cell death and survive from the therapies. Thus, not only observing the alternative mechanisms of tumor cells resistant to cell death, but also understanding the intricate dynamics of cell death processes within the tumor microenvironment (TME), are essential for tailoring effective therapeutic strategies. High-throughput sequencing technologies have revolutionized cancer research by enabling comprehensive molecular profiling. Recent advances in single cell sequencing have unraveled the heterogeneity of TME components, shedding light on their complex interactions. In this review, we explored the interplay between cell death signaling and the TME, summarised the potential drugs inducing cell death in pre-clinical stage, reviewed some studies applying next-generation sequencing technologies in cancer death research, and discussed the future utilization of updated sequencing platforms in screening novel treatment methods targeted cell death. In conclusion, leveraging multi-omics technologies to dissect cell death signaling in the context of the TME holds great promise for advancing cancer research and therapy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c40c61d7dcc3833421bbc3568f9b1a472c0c4795" target='_blank'>
              Cell death in tumor microenvironment: an insight for exploiting novel therapeutic approaches
              </a>
            </td>
          <td>
            Wenxin Wang, Tong Li, Kui Wu
          </td>
          <td>2025-03-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166ffe8ca69e378ce5cd3ea06cb57373f1b4e28e" target='_blank'>
              High-purity CTC RNA sequencing identifies prostate cancer lineage phenotypes prognostic for clinical outcomes.
              </a>
            </td>
          <td>
            Marina N. Sharifi, J. Sperger, Amy K Taylor, Katharine E Tippins, Shannon R Reese, Viridiana Carreno, K. Kaufmann, Alex H Chang, Luke A Nunamaker, Charlotte Linebarger, Leilani Mora-Rodriguez, J. Schehr, H. Krause, K. Helzer, M. Bootsma, G. Blitzer, John M Floberg, C. Kyriakopoulos, H. Emamekhoo, Elizabeth I. Heath, Meghan Wells, S. Tagawa, M. Sjöström, A. Choudhury, Menggang Yu, Andrew J. Armstrong, D. Rathkopf, H. Beltran, Peter S Nelson, Felix Y. Feng, S. Dehm, D. Kosoff, Xiao X Wei, Rana R Mckay, Shuang G. Zhao, Joshua M Lang
          </td>
          <td>2025-02-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="The tumor boundary of breast cancer represents a highly heterogeneous region. In this area, the interactions between malignant and non-malignant cells influence tumor progression, immune evasion, and drug resistance. However, the spatial transcriptional profile of the tumor boundary and its role in the prognosis and treatment response of breast cancer remain unclear.Utilizing the Cottrazm algorithm, we reconstructed the intricate boundaries and identified differentially expressed genes (DEGs) associated with these regions. Cell-cell co-positioning analysis was conducted using SpaCET, which revealed key interactions between tumor-associated macrophage (TAMs) and cancer-associated fibroblasts (CAFs). Additionally, Lasso regression analysis was employed to develop a malignant body signature (MBS), which was subsequently validated using the TCGA dataset for prognosis prediction and treatment response assessment.Our research indicates that the tumor boundary is characterized by a rich reconstruction of the extracellular matrix (ECM), immunomodulatory regulation, and the epithelial-to-mesenchymal transition (EMT), underscoring its significance in tumor progression. Spatial colocalization analysis reveals a significant interaction between CAFs and M2-like tumor-associated macrophage (TAM), which contributes to immune exclusion and drug resistance. The MBS score effectively stratifies patients into high-risk groups, with survival outcomes for patients exhibiting high MBS scores being significantly poorer. Furthermore, drug sensitivity analysis demonstrates that high-MB tumors had poor response to chemotherapy strategies, highlighting the role of the tumor boundary in modulating therapeutic efficacy.Collectively, we investigate the spatial transcription group and bulk data to elucidate the characteristics of tumor boundary molecules in breast cancer. The CAF-M2 phenotype emerges as a critical determinant of immunosuppression and drug resistance, suggesting that targeting this interaction may improve treatment responses. Furthermore, the MBS serves as a novel prognostic tool and offers potential strategies for guiding personalized treatment approaches in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b10699232e4b3f1c95f3a7d6be8ae3942b8fc0f" target='_blank'>
              Spatial multi-omics analysis of tumor-stroma boundary cell features for predicting breast cancer progression and therapy response
              </a>
            </td>
          <td>
            Yuanyuan Wu, Youyang Shi, Zhanyang Luo, Xiqiu Zhou, Yonghao Chen, Xiaoyun Song, Sheng Liu
          </td>
          <td>2025-03-26</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e9a98022a15f5fa6336dcfa874e713b8fce8db6" target='_blank'>
              AGP-Net: A Universal Network for Gene Expression Prediction of Spatial Transcriptomics
              </a>
            </td>
          <td>
            Yan Yang, Xuesong Li, Liyuan Pan, Guoxun Zhang, Liu Liu, Eric A. Stone
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Changes in cell type composition play an important role in human health and disease. Recent advances in single-cell technology have enabled the measurement of cell type composition at increasing cell lineage resolution across large cohorts of individuals. Yet this raises new challenges for statistical analysis of these compositional data to identify changes in cell type frequency. We introduce crumblr (DiseaseNeurogenomics.github.io/crumblr), a scalable statistical method for analyzing count ratio data using precision-weighted linear mixed models incorporating random effects for complex study designs. Uniquely, crumblr performs statistical testing at multiple levels of the cell lineage hierarchy using a multivariate approach to increase power over tests of one cell type. In simulations, crumblr increases power compared to existing methods while controlling the false positive rate. We demonstrate the application of crumblr to published single-cell RNA-seq datasets for aging, tuberculosis infection in T cells, bone metastases from prostate cancer, and SARS-CoV-2 infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4cb0d543e456b8e0b6125dd4a4f449c19fe2812" target='_blank'>
              Fast, flexible analysis of differences in cellular composition with crumblr
              </a>
            </td>
          <td>
            Gabriel E. Hoffman, P. Roussos
          </td>
          <td>2025-01-31</td>
          <td>Research Square</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Background Semaphorins (SEMAs), originally identified as axon guidance factors, have been found to play crucial roles in tumor growth, invasiveness, neoangiogenesis, and the modulation of immune responses. However, the prognostic value of SEMA-related genes in colorectal cancer (CRC) remains unclear. Methods We applied a novel machine learning framework that incorporated 10 machine learning algorithms and their 101 combinations to construct a SEMAs-related score (SRS). Multi-omics analysis was performed, including single-cell RNA sequencing (scRNA-seq), and spatial transcriptome (ST) to gain a more comprehensive understanding of the SRS. A series of cell experiments were conducted to prove the impact of key genes on CRC biological behavior. Result A consensus SRS was finally constructed based on a 101-combination machine learning computational framework, demonstrating outstanding performance in predicting overall survival. Moreover, distinct biological functions, mutation burden, immune cell infiltration, and immunotherapy response were observed between the high- and low-SRS groups. scRNA-seq and ST demonstrated unique cellular heterogeneity in CRC. We observed that SRS-high and SRS-low malignant epithelial cells exhibit different biological characteristics. High SRS malignant epithelial cells interact with myeloid and endothelial cells via SPP1 and COL4A2-ITGAV-ITGB8 pathways, respectively. Low SRS cells engage with myeloid and endothelial cells through MIF and JAG1-NOTCH4 pathways. Additionally, knocking down SEMA4C significantly inhibits the proliferation and invasion of CRC cells, while promoting apoptosis in vitro. Conclusion SRS could serve as an effective tool to predict survival and identify potential patients benefiting from immunotherapy in CRC. It also reveals tumor heterogeneity and provides valuable biological insights in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b27be243d13affabe786e086c79b848143a85be" target='_blank'>
              Integrative analysis of semaphorins family genes in colorectal cancer: implications for prognosis and immunotherapy
              </a>
            </td>
          <td>
            Jiahao Zhu, Benjie Xu, Zhixing Wu, Zhiwei Yu, Shengjun Ji, Jie Lian, H. Lu
          </td>
          <td>2025-03-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The sentinel lymph node (SLN) is the first lymph node encountered by a metastatic cancer cell and serves as a predictor of poor prognosis, as patients with clinically occult SLN metastases are classified as stage III with elevated rates of recurrence and diminished overall survival. However, the dynamics of immune infiltrates in SLNs remain poorly characterized. Here, using an unbiased Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq) technique, we profiled 97,777 cells from SLN tissues obtained from patients with stages I/II and III cutaneous melanoma. We describe the transcriptional programs of a multitude of T, B, and myeloid cell subtypes in SLNs. Based on the proportions of cell types, we determined that SLN subtypes are stratified along a naive → activated axis; patients with a 'high activated' signature score appear to be undergoing a robust melanoma antigen-driven adaptive immune response and thus, could be responsive to immunotherapy. Additionally, we identified transcriptomic signatures of SLN-infiltrating dendritic cell subsets that compromise anti-tumor immune responses. Our analyses provide valuable insights into tumor-driven immune changes in the SLN tissue, offering a powerful tool for the informed design of immune therapies for high-risk melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ff6ed2d70346b46dbac007982b480af5731bdaf" target='_blank'>
              Single-cell transcriptomics of melanoma sentinel lymph nodes identifies unique immune cell signatures associated with metastasis.
              </a>
            </td>
          <td>
            Eric Engelbrecht, Bryce F Stamp, Lewis C. Chew, Omar S Sarkar, Phillip Harter, Sabine J Waigel, Eric C Rouchka, J. Chariker, Andrei Smolenkov, Jason A Chesney, K. McMasters, Corey T Watson, K. Yaddanapudi
          </td>
          <td>2025-03-06</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Background Glioma, the most common primary cancer of the central nervous system, characterizes significant heterogeneity, presenting major challenges for therapeutic approaches and prognosis. In this study, the interactions between malignant glioma cells and macrophages/monocytes, as well as their influence on tumor progression and treatment responses, were explored using comprehensive single-cell RNA sequencing analysis. Methods RNA-seq data from the TCGA and CGGA databases were integrated and an in-depth analysis of glioma samples was performed using single-cell RNA sequencing, functional enrichment analysis, developmental trajectory analysis, cell–cell communication analysis, and gene regulatory network analysis. Furthermore, a prognostic model based on risk scores was developed, and its predictive performance was assessed through immune cell infiltration analysis and immune treatment response evaluation. Results Fourteen distinct glioma cellular subpopulations, seven primary cell types, and four macrophage/monocyte subtypes were identified. Developmental trajectory analysis offered insights into the origins and heterogeneity of malignant cells as well as macrophages/monocytes. Cell communication analysis revealed the interaction of macrophages and monocytes with malignant cells through several pathways, including the macrophage migration inhibitory factor and secreted phosphoprotein 1 pathways, engaging in key ligand-receptor interactions that influence tumor behavior. Categorization based on these communication characteristics was significantly correlated with overall survival. Immune cell infiltration analysis highlighted variations in immune cell abundance across different subgroups, possibly linked to differing responses to immunotherapy. This predictive model, comprising 29 prognostic genes, demonstrated high accuracy and robustness across multiple independent cohorts. Conclusion This study reveals the complex heterogeneity of the glioma microenvironment and enhances the understanding of diverse characteristics of glioma cell subsets. At the same time, it lays a foundation for the development of therapeutic strategies and prognostic models targeting the glioma microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b83f8e329366e14ce122de3c8f5608b479cd4df9" target='_blank'>
              Mapping glioma progression: single-cell RNA sequencing illuminates cell–cell interactions and immune response variability
              </a>
            </td>
          <td>
            Xia Li, Shenbo Chen, Ming Ding, Hui Ding, Kun Yang
          </td>
          <td>2025-03-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Newly diagnosed prostate cancers differ dramatically in mutational composition and lethality. The most accurate clinical predictor of lethality is tumor tissue architecture, quantified as tumor grade. To interrogate the evolutionary origins of prostate cancer heterogeneity, we analyzed 666 prostate tumor whole genomes. We identified a compendium of 223 recurrently mutated driver regions, most influencing downstream mutational processes and gene expression. We identified and validated individual germline variants that predispose tumors to acquire specific somatic driver mutations: these explain heterogeneity in disease presentation and ancestry differences. High-grade tumors have a superset of the drivers in lower-grade tumors, including increased frequency of BRCA2 and MYC mutations. Grade-associated driver mutations occur early in tumor evolution, and their earlier occurrence strongly predicts cancer relapse and metastasis. Our data suggest high- and low-grade prostate tumors both emerge from a common pre-malignant field, influenced by germline genomic context and stochastic mutation-timing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9c23cc7532d469971ff1994cdcbf780a4650cf6" target='_blank'>
              The Germline and Somatic Origins of Prostate Cancer Heterogeneity.
              </a>
            </td>
          <td>
            Takafumi N. Yamaguchi, Kathleen E. Houlahan, Helen Zhu, Natalie J Kurganovs, J. Livingstone, Natalie S. Fox, Jiapei Yuan, Jocelyn Sietsma Penington, Chol-Hee Jung, T. Schwarz, Weerachai Jaratlerdsiri, J. van Riet, P. Georgeson, S. Mangiola, Kodi Taraszka, R. Lesurf, Jue Jiang, K. Chow, Lawrence E Heisler, Yu-Jia Shiah, Susmita G Ramanand, Michael J Clarkson, Anne Nguyen, S. M. G. Espiritu, R. Stuchbery, Richard Jovelin, Vincent Huang, Connor Bell, Edward O'Connor, Patrick J. McCoy, Christopher M. Lalansingh, Marek Cmero, A. Salcedo, Eva K F Chan, Lydia Y. Liu, Phillip D Stricker, Vinayak Bhandari, R. Bornman, Dorota H. Sendorek, A. Lonie, S. Prokopec, M. Fraser, J. Peters, Adrien Foucal, S. Mutambirwa, Lachlan Mcintosh, M. Orain, Matthew J. Wakefield, V. Picard, Daniel J Park, H. Hovington, Michael Kerger, A. Bergeron, Veronica Y. Sabelnykova, J-H. Seo, M. Pomerantz, Noah Zaitlen, Sebastian M. Waszak, Alexander Gusev, L. Lacombe, Yves Fradet, Andrew Ryan, A. Kishan, M. Lolkema, Joachim Weischenfeldt, Bernard Têtu, Anthony J Costello, Vanessa M Hayes, R. J. Hung, H. He, John D McPherson, Bogdan Pasaniuc, T. H. van der Kwast, AT Papenfuss, M. Freedman, B. Pope, Robert G Bristow, Ram-Shankar Mani, N. Corcoran, J. Reimand, Christopher M Hovens, P. Boutros
          </td>
          <td>2025-02-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly malignancy and new treatments are needed to improve survival rates. Many lung tumours exhibit a feature called chromosomal instability, which is characterized by ongoing acquisition of genetic alterations that promote tumour aggression. Interestingly, chromosomal instability is a “double-edged sword”. A moderate level can support tumour growth and progression but a high level causes cell death. Therefore, chromosomal instability represents a weakness in cancer cells that could be exploited to kill lung tumours. Kinesins are a family of proteins with several members having key roles in chromosome segregation during mitosis. Because of this function, inhibiting mitotic kinesins can amplify chromosomal instability. This review summarizes the potential of targeting kinesins to elevate chromosomal instability to lethal levels as a strategy for killing chromosomally unstable lung cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33b20af0b1c35c714611ceec53e3a323336edcf" target='_blank'>
              Myeloid cells: key players in tumor microenvironments.
              </a>
            </td>
          <td>
            Qiaomin Hua, Zhixiong Li, Yulan Weng, Yan Wu, Limin Zheng
          </td>
          <td>2025-03-06</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="At present, only a limited fraction of patients with extensive-stage small cell lung cancer (ES-SCLC) achieve a sustained response to immune checkpoint blockade (ICB) therapy. The factors that drive therapeutic efficacy remain poorly delineated, and the field is devoid of reliable predictive biomarkers to guide personalized treatment decisions. Therefore, we conducted RNA sequencing of tumor samples from 21 patients prior to treatment to identify expression patterns associated with lasting benefit and used weighted gene co-expression network analysis (WGCNA) to identify key genes associated with favorable outcomes of chemotherapeutic immunotherapy. Multiplex immunofluorescence (mIF) quantification and reanalysis of publicly available datasets were used to validate the hub gene's association with the immune microenvironment and immunotherapy efficacy. The functional significance of the hub gene was further investigated in cellular models. We found that the durable clinical benefit (DCB) group exhibited significantly elevated levels of inflammation and interferon response compared to the no-durable benefit (NDB) group, alongside a notably lower proportion of Tregs and distinct metabolic features. Lactotransferrin (LTF) was identified as a hub gene associated with durable therapeutic benefits in chemo-immunotherapy. By further analysis, we proved that LTF acts as a tumor suppressor in small cell lung cancer, impacting cell proliferation, migration, and invasiveness. It also inhibits lipid metabolism in these cells. Elevated LTF expression is linked to better chemo-immunotherapy outcomes, suggesting its potential as a predictive biomarker for first-line treatment response in ES-SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e523656065fe7f0e2896bb7e39b310f5c10dff08" target='_blank'>
              LTF as a Potential Predictive Biomarker for Durable Benefit From First-Line Chemo-Immunotherapy in Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Shimo Shen, Yili Wu, Zhuowei Shao, You Li, D. Peng, Bing Li, Zhou Zhang, Shibo Wu
          </td>
          <td>2025-03-17</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e110685415835cdd67555abc6bde1dc3e70baed" target='_blank'>
              Interactions Between Ploidy and Resource Availability Shape Clonal Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, J. Teer, Didem Ilter, Richard Beck, Jackson P Cole, Ana M Forero Pinto, Joanne D Tejero, Elisa Scanu, Thomas Veith, William Dominguez-Viqueira, Konstantin Maksin, Francisco Carrillo-Perez, Olivier Gevaert, Xiaonan Xu, F. Karreth, Mahmoud A Abdalah, Giada Fiandaca, Stefano Pasetto, Sandhya Prabhakaran, Andrew Schultz, A. M. E. Ojwang', J. Barnholtz-Sloan, Joaquim M Farinhas, Ana P. Gomes, Parag Katira, Noemi Andor
          </td>
          <td>2025-02-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a deadly cancer with an aggressive nature, and how CRC tumor cells manage to translocate and proliferate in a new tissue environment remains not fully understood. Recently, higher-order chromatin structures and spatial genome organization are increasingly implicated in diseases including cancer, but in-depth studies of three-dimensional genome (3D genome) of metastatic cancer are currently lacking, preventing the understanding of the roles of genome organization during metastasis. Here we perform multi-omics profiling of matched normal colon, primary tumor, lymph node metastasis, liver metastasis and normal liver tissue from CRC patients using Hi-C, ATAC-seq and RNA-seq technologies. We find that widespread alteration of 3D chromatin structure is accompanied by dysregulation of genes including SPP1 during the tumorigenesis or metastasis of CRC. Remarkably, the hierarchy of topological associating domain (TAD) changes dynamically, which challenges the traditional view that the TAD structure between tumor and normal tissue is conservative. In addition, we define compartment stability score to measure large-scale alteration in metastatic tumors. To integrate multi-omics data and recognize candidate genes driving cancer metastasis, a pipeline is developed based on Hi-C, RNA-seq and ATAC-seq data. And three candidate genes ARL4C, FLNA, and RGCC are validated to be associated with CRC cell migration and invasion using in vitro knockout experiments. Overall, these data resources and results offer new insights into the involvement of 3D genome in cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4405a7f71591c3b2e80c4392e32cd43a5138621" target='_blank'>
              3D genome landscape of primary and metastatic colorectal carcinoma reveals the regulatory mechanism of tumorigenic and metastatic gene expression
              </a>
            </td>
          <td>
            Xiang Xu, Jingbo Gan, Zhaoya Gao, Ruifeng Li, Dandan Huang, Lin Lin, Yawen Luo, Qian Yang, Jingxuan Xu, Yaru Li, Qing Fang, Ting Peng, Yaqi Wang, Zihan Xu, An Huang, Haopeng Hong, F. Lei, Wensheng Huang, Jianjun Leng, Tingting Li, Xiaochen Bo, Hebing Chen, Cheng Li, Jin Gu
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ac5e3595c3552c277cd820f66cb6eb10faa0266" target='_blank'>
              MDA-9/Syntenin as a therapeutic cancer metastasis target: current molecular and preclinical understanding.
              </a>
            </td>
          <td>
            Swadesh K. Das, Paul B Fisher
          </td>
          <td>2025-03-08</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="This study comprehensively investigated keratinocyte subpopulation heterogeneity and developmental trajectories during skin aging using single-cell sequencing, transcriptomics, and facial aging-related genome-wide association studies (GWAS) data. We identified three major subpopulations: basal cells (BCs), spinous cells (SCs), and IFI27+ keratinocytes. Single-cell pseudotime analysis revealed that basal cells can differentiate along two distinct paths: toward spinous differentiation or the inflammatory state. With aging, the proportion of IFI27+ cells significantly increased, displaying more active inflammatory and immunomodulatory signals. Through cell–cell communication analysis, we found that the signaling pathways, including NOTCH, PTPR, and PERIOSTIN, exhibited distinct characteristics along different branches. Integration of the GWAS data revealed significant loci on chromosomes 2, 3, 6, and 9 that were spatially correlated with key biological pathways (including antigen processing, oxidative stress, and apoptosis). These findings reveal the complex cellular and molecular mechanisms underlying skin aging, offering potential targets for novel diagnostic approaches and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7d6ff4c88e6b6ec5e9e3e8325f958c0ceb91730" target='_blank'>
              Inflammatory Transformation of Skin Basal Cells as a Key Driver of Cutaneous Aging
              </a>
            </td>
          <td>
            Shupeng Liu, Sheng Lu, Zhiping Pang, Jiacheng Li, Meijuan Zhou, Zhenhua Ding, Zhijun Feng
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad75c766d4e07168f2e27a4c036f888ffea4b32" target='_blank'>
              Integrating model systems and genomic insights to decipher mechanisms of cancer metastasis.
              </a>
            </td>
          <td>
            Michelle M Leung, C. Swanton, N. Mcgranahan
          </td>
          <td>2025-03-10</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Malignant pleural mesothelioma (MPM) is a rare and aggressive malignancy with limited therapeutic options. To improve patients management and treatment, more precise stratification strategies are needed. This study aimed to characterize the phenogenomic landscapes of MPM and to understand their influence on patients clinical outcomes. Methods We conducted a phenogenomic analysis on 22 MPM patients using two high throughput approaches: imaging mass cytometry (IMC) with whole exome sequencing (WES). Resulting profiles were addressed for their clinical relevance to predict patients prognosis. Results IMC revealed a highly heterogeneous tumor microenvironment (TME) with distinct tumor cell subpopulations. Notably, we identified a novel sarcomatoid-like cellular cluster associated with poor prognosis. The TME was also infiltrated with immune cells including macrophages and CD4+ T lymphocytes, that were more abundant in patients with favorable clinical outcomes. WES identified a complex genomic landscape with limited prognostic value for individual genetic alterations. However, tumor mutational burden (TMB) emerged as a potential predictive biomarker, inversely correlating with immune cell infiltration, particularly macrophages and CD4+ T lymphocytes. Conclusions Our findings underscore the intricate interplay between the tumor genome, TME composition, and clinical outcomes in MPM. These data support the potential of integrating genomic and TME profiling to develop more precise patient stratification strategies and potentially optimize therapeutic approaches, including immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06193-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eee1a8b871b1885a1b34200cb093a48900beddf" target='_blank'>
              The phenogenomic landscapes of pleural mesothelioma tumor microenvironment predict clinical outcomes
              </a>
            </td>
          <td>
            Federica Torricelli, Filomena Spada, Cynthia Bishop, K. Todd, Daisuke Nonaka, N. Petrov, Manuela Terranova Barberio, Alan G Ramsay, Richard Ellis, A. Ciarrocchi, B. Apollonio, Andrea Billè
          </td>
          <td>2025-02-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The spatial structure of cells is highly organized at multiscale levels from global spatial domains to local cell type heterogeneity. Existing methods for analyzing spatially resolved transcriptomics (SRT) are separately designed for either domain alignment across multiple slices or deconvoluting cell type compositions within a single slice. To this end, a novel deep learning method, SMILE, is proposed which combines graph contrastive autoencoder and multilayer perceptron with local constraints to learn multiscale and informative spot representations. By comparing SMILE with the state-of-the-art methods on simulation and real datasets, the superior performance of SMILE is demonstrated on spatial alignment, domain identification, and cell type deconvolution. The results show SMILE's capability not only in simultaneously dissecting spatial variations at different scales but also in unraveling altered cellular microenvironments in diseased conditions. Moreover, SMILE can utilize prior domain annotation information of one slice to further enhance the performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34c3b8e00d794aa11d4133747984afb67d6fa899" target='_blank'>
              Multiscale Dissection of Spatial Heterogeneity by Integrating Multi-Slice Spatial and Single-Cell Transcriptomics.
              </a>
            </td>
          <td>
            Yuqi Chen, Caiwei Zhen, Yuanyuan Mo, Juan Liu, Lihua Zhang
          </td>
          <td>2025-02-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Profiling circulating cell-free DNA (cfDNA) has become a fundamental practice in cancer medicine, but the effectiveness of cfDNA at elucidating tumor-derived molecular features has not been systematically compared to standard single-lesion tumor biopsies in prospective cohorts of patients. The use of plasma instead of tissue to guide therapy is particularly attractive for patients with small cell lung cancer (SCLC), due to the aggressive clinical course of this cancer, which makes obtaining tumor biopsies exceedingly challenging. Methods In this study, we analyzed a prospective cohort of 49 plasma samples obtained before, during, and after treatment from 20 patients with recurrent SCLC. We conducted cfDNA low-pass whole genome sequencing (0.1X coverage), comparing it with time-point matched tumor characterized using whole-exome (130X) and transcriptome sequencing. Results A direct comparison of cfDNA and tumor biopsy revealed that cfDNA not only mirrors the mutation and copy number landscape of the corresponding tumor but also identifies clinically relevant resistance mechanisms and cancer driver alterations not detected in matched tumor biopsies. Longitudinal cfDNA analysis reliably tracks tumor response, progression, and clonal evolution. Sequencing coverage of plasma DNA fragments around transcription start sites showed distinct treatment-related changes and captured the expression of key transcription factors such as NEUROD1 and REST in the corresponding SCLC tumors. This allowed for the prediction of SCLC neuroendocrine phenotypes and treatment responses. Conclusions cfDNA captures a comprehensive view of tumor heterogeneity and evolution. These findings have significant implications for the non-invasive stratification of SCLC, a disease currently treated as a single entity. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01438-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75bdacf15f78eabb80ad03155a6f7ee66350fd9" target='_blank'>
              Genomic alterations and transcriptional phenotypes in circulating free DNA and matched metastatic tumor
              </a>
            </td>
          <td>
            N. Takahashi, L. Pongor, Shivam P. Agrawal, Mariya Shtumpf, Ankita Gurjar, V. Rajapakse, Ahmad Shafiei, Christopher W. Schultz, Sehyun Kim, Diana Roame, Paula Carter, R. Vilimas, Samantha Nichols, Parth Desai, W. Figg, Mohammad Bagheri, V. B. Teif, Anish Thomas
          </td>
          <td>2025-02-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0b1fd2e12bcf45ef4e0b8a0b897b112e846447" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Takafumi N. Yamaguchi, Yu-Jia Shiah, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, Amar U. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77eda520356c4bdca968c15e0f1c01aca84131b9" target='_blank'>
              A tissue multiplexing approach to maximise spatial transcriptomics output from precious pathology specimens
              </a>
            </td>
          <td>
            Y. Kao, Katie J. Lee, Chenhao Zhou, Andrew Causer, Harald Oey, Kiarash Khosrotehrani, Blake O’Brien, Angus Collins, Quan Nguyen, H. Soyer, Mitchell S. Stark, Quan Nguyen Orcid
          </td>
          <td>2025-02-08</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neuroendocrine bladder cancer (NEBC) is a rare but highly aggressive cancer, representing approximately 1% of urinary bladder cancer. The most common NEBC is small cell bladder cancer (SCBC), characterized by high rates of recurrence, chemotherapy resistance, and early mortality. SCBC is histologically identical to small cell lung cancer (SCLC) but remains significantly understudied. Advances in next-generation sequencing techniques have partially elucidated the molecular characteristics of NEBC and identified druggable targets. This review compiles recent studies on human NEBC samples, summarizing key findings on their genomic alterations and molecular subtyping. Notably, it highlights specific mutations in the TERT promoter and epigenetic modifiers in NEBC, as well as molecular subtyping based on lineage-specific transcription factors, including ASCL1, NEUROD1, and POU2F3. Furthermore, this review explores the significant tumor heterogeneity and cellular plasticity observed in NEBC and discusses its cell of origin and potential therapeutic targets (MET inhibitor or DLL3) identified by preclinical NEBC models. Emerging evidence suggests that NEBC may share a common origin with urothelial carcinoma (UC), arising from a UC precursor. Advancing our understanding of NEBC tumorigenesis and identifying druggable targets will enhance treatment outcomes for patients with NEBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fc64b4fb90b8829827b2a0ce83dfe4e9620085" target='_blank'>
              Molecular Characteristics, Heterogeneity, Plasticity, and Cell of Origin of Neuroendocrine Bladder Cancer
              </a>
            </td>
          <td>
            Dongbo Xu, Qiang Li
          </td>
          <td>2025-03-06</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfc8185bbc4036b6f9e694f8413b4a6f52d259e" target='_blank'>
              How is single-cell RNA sequencing contributing to the advancement of cancer therapeutics?
              </a>
            </td>
          <td>
            Keun-Yeong Jeong
          </td>
          <td>2025-04-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Tertiary lymphoid structures (TLS) correlate with tumour prognosis and immunotherapy responses in gastric cancer (GC) studies. However, understanding the complex and diverse immune microenvironment within TLS requires comprehensive analysis. Methods We examined the prognostic impact of TLS within the tumour core (TC) of 59 GC patients undergoing immunotherapy. Multispectral fluorescence imaging was employed to evaluate variations in immune cell infiltration across different TLS sites among 110 GC patients, by quantifying immune cell density and spatial characteristics. We also generated a single-cell transcriptomic atlas of TLS-positive (n = 4) and TLS-negative (n = 8) microenvironments and performed spatial transcriptomics (ST) analysis on two samples. Results TLS presence in the TC significantly correlated with improved immune-related overall survival (P = 0.049). CD8+LAG-3−PD-1+TIM-3−, CD4+PD-L1+, and CD4+FoxP3− T cell densities were significantly higher in the TLS within TC compared to tumour and stromal regions. Immune cells within TLS exhibited closer intercellular proximity than those outside TLS. Five key density and spatial characteristics of immune cells within TLS in the TC were selected to develop the Density and Spatial Score risk model. Single-cell RNA sequencing revealed strong intercellular interactions in the presence of TLS within the microenvironment. However, TLS-absent environment facilitated tumour cell interactions with immune cells through MIF- and galectin-dependent pathways, recruiting immunosuppressive cells. ST analysis confirmed that T and B cells co-localise within TLS, enhancing immune response activation compared to cancer nests and exerting a strong anti-tumour effect. Conclusions TLS presence facilitates frequent cell-to-cell communication, forming an active immune microenvironment, highlighting the prognostic value of TLS. Supplementary Information The online version contains supplementary material available at 10.1186/s12916-025-03889-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/197782f3d8b81c382c4e0d5197a6f09c7baeb2a9" target='_blank'>
              Immune microenvironment spatial landscapes of tertiary lymphoid structures in gastric cancer
              </a>
            </td>
          <td>
            Yi Xie, Haoxin Peng, Yajie Hu, Keren Jia, Jiajia Yuan, Dan Liu, Yanyan Li, Xujiao Feng, Jian Li, Xiaotian Zhang, Yu Sun, Lin Shen, Yang Chen
          </td>
          <td>2025-02-04</td>
          <td>BMC Medicine</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bb874396859196c4013ee0fce50222acff4e7b2" target='_blank'>
              Neutrophils physically interact with tumor cells to form a signaling niche promoting breast cancer aggressiveness.
              </a>
            </td>
          <td>
            Sandra Camargo, Ori Moskowitz, Amir Giladi, Maiia Levinson, Roi Balaban, Shani Gola, Alice Raizman, Kelly Lipczyc, Alon Richter, Noa Keren-Khadmy, Oren Barboy, Yael Dugach, Yaron Carmi, A. Sonnenblick, Merav Cohen
          </td>
          <td>2025-03-07</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Brain metastasis represents a significant challenge in oncology, driven by complex molecular and epigenetic mechanisms that distinguish it from primary tumors. While recent research has focused on identifying genomic mutation drivers with potential clinical utility, these strategies have not pinpointed specific genetic mutations responsible for site-specific metastasis to the brain. It is now clear that successful brain colonization by metastatic cancer cells requires intricate interactions with the brain tumor ecosystem and the acquisition of specialized molecular traits that facilitate their adaptation to this highly selective environment. This is best exemplified by widespread transcriptional adaptation during brain metastasis, resulting in aberrant gene programs that promote extravasation, seeding, and colonization of the brain. Increasing evidence suggests that epigenetic mechanisms play a significant role in shaping these pro-brain metastasis traits. This review explores dysregulated chromatin patterns driven by chromatin remodeling, histone modifications, DNA/RNA methylation, and other epigenetic regulators that underpin brain metastatic seeding, initiation, and outgrowth. We provide novel insights into how these epigenetic modifications arise within both the brain metastatic tumor and the surrounding brain metastatic tumor ecosystem. Finally, we discuss how the inherent plasticity and reversibility of the epigenomic landscape in brain metastases may offer new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Breast cancer is the most common type of cancer in women and the second leading cause of death by cancer. Despite recent advances, the mortality rate remains high, underlining the need to develop new therapeutic approaches. The complex interaction between cancer cells and the tumor microenvironment (TME) is crucial in determining tumor progression, therapy response, and patient prognosis. Understanding the role of immune cells in carcinogenesis and tumor progression can help improve targeted therapeutic options, increasing the likelihood of a favorable prognosis. Therefore, this review aims to critically analyze the complex interaction between tumor cells and immune cells, emphasizing the clinical and therapeutic implications. Additionally, we explore advances in immunotherapies, with a focus on immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c335e51d434a9f10b3300d0d306ba2af8135f4e1" target='_blank'>
              Breast Cancer and Tumor Microenvironment: The Crucial Role of Immune Cells
              </a>
            </td>
          <td>
            Tânia Moura, P. Laranjeira, Olga Caramelo, Ana M. Gil, Artur Paiva
          </td>
          <td>2025-02-28</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25aeeccc03e491382af67e5b76f7835dfb0b4a41" target='_blank'>
              508 The Largest Patient-Derived Meningioma Organoid Bank Demonstrates a Robust Model for Transcriptional Analysis of Tumors
              </a>
            </td>
          <td>
            Youssef M. Zohdy, Arman Jahangiri, Fadi Jacob, A. Aksionau, Alejandra Rodas, Zhexing Wen, Gustavo Pradilla, Tomas Garzon-Muvdi
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Although immune checkpoint (IC) inhibition is a major treatment modality in cancer-immunotherapy, multiple cancers show low response. Our in-silico exploration by mining cancer datasets using R2, available clinical trial data, and Kaplan–Meier analysis from GEPIA depicted that unlike low-responder (LR) cancers, high-responder (HR) cancers furnish higher IC expression, that upon lowering may provide better prognosis. Contrastingly, pancreatic adenocarcinoma (PAAD) demonstrated high IC expression but low immunotherapy-response. Infiltration scores from TIMER2.0 revealed higher pro-tumor immune subsets and cancer-associated fibroblasts (CAFs) while depicting lower anti-tumor immune subsets in PAAD as compared to HR lung adenocarcinoma (LUAD). Additionally, bioinformatic tool cBioportal showed lesser tumor mutational burden, mismatch repair deficiency and greater percent of driver mutations in TP53, KRAS and CDKN2A in PAAD, supporting its higher immunotherapy-resistance than LUAD. Our search for the ‘key’ immunotherapy response-deciding factor(s) revealed cancer stem cells (CSCs), the known contributors of therapy-resistance and immuno-evasion, to be positively correlated with above-mentioned driver mutations, pro-tumor immune and CAF subsets; and that PAAD furnished higher expression of CSC genes than LUAD. UMAP/tSNE analyses revealed that high CSC signature is positively correlated with immunotherapy-resistance genes and pro-cancer immune cells, while negatively with cytotoxic-T cells in PAAD. Our in-silico study explains the low immunotherapy-response in high IC-expressing PAAD, wherein CSC plays a pivotal role. Further exploration portrayed correlation of CSCs with immunotherapy-resistance in other LR and HR cancers too, substantiating the need for personalized CSC evaluation and targeting for successful immunotherapy outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93924-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de30629094c5f0558bed2894abe95ee38a9c5b9" target='_blank'>
              In-silico analysis unveiling the role of cancer stem cells in immunotherapy resistance of immune checkpoint-high pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Udit Basak, Sumon Mukherjee, Sourio Chakraborty, Gaurisankar Sa, S. G. Dastidar, Tanya Das
          </td>
          <td>2025-03-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af090730ccebba2da2c3e50d4cbf6a340cfee095" target='_blank'>
              In Search of Representative Translational Cancer Model Systems.
              </a>
            </td>
          <td>
            Hannah E. Trembath, Philip M. Spanheimer
          </td>
          <td>2025-02-01</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098b5888d894097b207e005b18110c0c81abf3d8" target='_blank'>
              Modeling cell differentiation in neuroblastoma: insights into development, malignancy, and treatment relapse
              </a>
            </td>
          <td>
            Simon F. Martina-Perez, Luke A. Heirene, Jennifer C. Kasemeier, P. Kulesa, Ruth E. Baker
          </td>
          <td>2025-02-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bd9fa8c8b49a99cb9a4bb9596137f5ef0a74d" target='_blank'>
              Sc-TUSV-Ext: Single-Cell Clonal Lineage Inference from Single Nucleotide Variants, Copy Number Alterations, and Structural Variants.
              </a>
            </td>
          <td>
            N. Bristy, Xuecong Fu, Russell Schwartz
          </td>
          <td>2025-03-06</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a517176c60f9a80290d19966f93c06f80af4cac7" target='_blank'>
              Emerging technologies of single-cell multi-omics.
              </a>
            </td>
          <td>
            Yi June Kim, Koichi Takahashi
          </td>
          <td>2025-02-06</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Recent oncological research has intensely focused on the tumor immune microenvironment (TME), particularly the functions of CD4 + T lymphocytes. CD4+ T lymphocytes have been implicated in antigen presentation, cytokine release, and cytotoxicity, suggesting their contribution to the dynamics of the TME. Furthermore, the application of single-cell sequencing has yielded profound insights into the phenotypic diversity and functional specificity of CD4+ T cells in the TME. In this review, we discuss the current findings from single-cell analyses, emphasizing the heterogeneity of CD4+ T cell subsets and their implications in tumor immunology. In addition, we review the critical signaling pathways and molecular networks underpinning CD4+ T cell activities, thereby offering novel perspectives on therapeutic targets and strategies for cancer treatment and prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06167-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8641c47a99cb4e08fcabebc4fec5794fc9e23e2f" target='_blank'>
              Role of CD4+ T cells in cancer immunity: a single-cell sequencing exploration of tumor microenvironment
              </a>
            </td>
          <td>
            Qi An, Li Duan, Yuanyuan Wang, Fuxin Wang, Xiangyu Liu, Chao Liu, Qinyong Hu
          </td>
          <td>2025-02-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains a leading cause of cancer-related mortality, with over one million new cases and 769,000 deaths reported in 2020. Despite advancements in chemotherapy, surgery, and targeted therapies, delayed diagnosis due to overlooked early symptoms leads to poor prognosis. Methods We integrated bulk RNA sequencing and single-cell RNA sequencing datasets from TCGA, GEO, and OMIX001073, employing normalization, batch effect correction, and dimensionality reduction methods to identify key cell populations associated with GC invasion and epithelial-mesenchymal transition (EMT), as well as analyze the tumor immune microenvironment. Results Our analysis identified the MUC5AC+ malignant epithelial cell cluster as a significant player in GC invasion and EMT. Cluster 1, representing this cell population, exhibited higher invasion and EMT scores compared to other clusters. Survival analysis showed that high abundance in cluster 0 correlated with improved survival rates (P=0.012), whereas cluster 1 was associated with poorer outcomes (P=0.045). A prognostic model highlighted ANXA5 and GABARAPL2 as two critical genes upregulated in GC tumors. High-risk patients demonstrated increased immune cell infiltration and worse prognosic. Analysis of tumor mutation burden (TMB) indicated that patients with low TMB in the high-risk group had the worst prognosis. Wet-lab validation experiments confirmed the oncogenic role of ANXA5, showing its facilitation of cell proliferation, invasion, and migration while suppressing apoptosis. Conclusion This study offers novel insights into the subpopulations of malignant epithelial cells in GC and their roles in tumor progression. It provides a prognostic model and potential therapeutic targets to combat GC, contributing crucial understanding to the fundamental mechanisms of drug resistance in gastrointestinal cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f9fbc569ffcfa0009c2b9e523878c6293cec12b" target='_blank'>
              Integrative analysis of single-cell and bulk RNA sequencing reveals the oncogenic role of ANXA5 in gastric cancer and its association with drug resistance
              </a>
            </td>
          <td>
            Deng-Gang Chen, Peng Zhang, Li Gong, Hailang Wei, Guanghui Yu, Tingting Zhang, Chen Bai
          </td>
          <td>2025-03-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumor microenvironment (TME) milieu directs a plethora of tumor-modulating functions. Recent years have seen pivotal breakthroughs in our understanding of the TME's role in tumor initiation and progression, with tangible clinical applications. Individual components of the TME exert their function predominantly by cell-cell crosstalk, both in the form of physical interaction and secreted factors. Notably, different spatial niches represent exclusive signaling hubs in the TME, propagating pro- or antitumoral functions. The exploration of these interactions has been vastly facilitated by novel molecular technologies, each of which provides a different perspective on this intricate intercellular communication network. Together, these complementary methods paint a detailed, high-resolution map of the TME's interaction landscape. In this viewpoint, we explore how cellular interactions can unlock a new level of understanding of TME complexity, and discuss the promises and challenges of characterizing tumors based on their cellular interaction footprint.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/072aea9d500a6665bd7f3f737b00c34169b4d84a" target='_blank'>
              Decoding multicellular interaction networks-a new horizon in tumor microenvironment research.
              </a>
            </td>
          <td>
            Roi Balaban, Merav Cohen
          </td>
          <td>2025-02-05</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aea618f90a9672b554b4f10cc0dc3c6cd3c4cdf0" target='_blank'>
              Abstract B089: Assessing integrin αvβ5 as a therapeutic target suppressing tumor progression and enhancing immune responses in non-small cell lung cancer
              </a>
            </td>
          <td>
            Bitta Kahangi, William P Crosson, Ji-Ann Lee, Nalani Coleman, C. Dumitras, Hong Jiang, Tian-Hao Zhang, J. Abascal, E. P. Reyes, Austin K. Rennels, R. Salehi-Rad, L. Tran, Michael Palazzolo, D. Conklin, Steven M Dubinett, Bin Liu
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Large-scale changes in the structure and cellular makeup of the distal lung are a hallmark of pulmonary fibrosis (PF), but the spatial contexts that contribute to disease pathogenesis have remained uncertain. Using image-based spatial transcriptomics, we analyzed the gene expression of 1.6 million cells from 35 unique lungs. Through complementary cell-based and innovative cell-agnostic analyses, we characterized the localization of PF-emergent cell types, established the cellular and molecular basis of classical PF histopathologic features and identified a diversity of distinct molecularly defined spatial niches in control and PF lungs. Using machine learning and trajectory analysis to segment and rank airspaces on a gradient of remodeling severity, we identified compositional and molecular changes associated with progressive distal lung pathology, beginning with alveolar epithelial dysregulation and culminating with changes in macrophage polarization. Together, these results provide a unique, spatially resolved view of PF and establish methods that could be applied to other spatial transcriptomic studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c52c76852d3ba58984faf618de957dc0aff431e" target='_blank'>
              Spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis
              </a>
            </td>
          <td>
            A. Vannan, Ruqian Lyu, A. L. Williams, N. Negretti, E. D. Mee, J. Hirsh, S. Hirsh, Niran Hadad, David Nichols, C. Calvi, Chase J Taylor, V. Polosukhin, A. P. Serezani, A. S. McCall, J. Gokey, Heejung Shim, Lorraine B. Ware, M. Bacchetta, C. Shaver, Timothy S. Blackwell, Rajat Walia, J. Sucre, JA Kropski, Davis J. McCarthy, N. Banovich
          </td>
          <td>2025-02-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bb25c7b8caf9c1e00291f67d305dabd29b4533" target='_blank'>
              Spatial Control of CAR T Cell Activation Using Tumor-Homing Polymers.
              </a>
            </td>
          <td>
            Clinton M Heinze, Trey J. Pichon, Abe Y Wu, Michael L Baldwin, James Matthaei, Kefan Song, Meilyn Sylvestre, Joshua Gustafson, Nathan J White, Michael C Jensen, S. Pun
          </td>
          <td>2025-02-04</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Simple Summary Epitranscriptomic modifications impact RNA stability and function, influencing key biological processes. While initially behind epigenetics, the field has advanced in the last decade, linking these modifications to development and diseases like cancer. High-grade gliomas, such as glioblastoma, remain challenging due to their aggressiveness and lack of effective treatments. Understanding the impact of epitranscriptomic alterations in gliomas may provide new insights into tumor progression and therapeutic strategies. This review provides an overview of RNA modifications in tRNAs, rRNAs, miRNAs, and mRNAs, exploring the role of the proteins involved in epitranscriptomics and their dysregulation in gliomas. We discuss how this imbalance affects pathways in glioma progression and highlight recent advancements in RNA modification-based therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bcb06b4f2cf9c15faeac2d418f2e3e672130f79" target='_blank'>
              Epitranscriptomics in the Glioma Context: A Brief Overview
              </a>
            </td>
          <td>
            Pablo Santamarina-Ojeda, Agustin F. Fernández, Mario F. Fraga
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fa697cc0365f103131eebd290f857c508fd5863" target='_blank'>
              Tumor Cell Plasticity and Stromal Microenvironment Distinguish Papillary and Follicular Growth Patterns in a Mouse Model of BRAFV600E-induced Thyroid Cancer.
              </a>
            </td>
          <td>
            E. Schoultz, C. Moccia, Shawn Liang, Ellen Johansson, Mikael Nilsson
          </td>
          <td>2025-02-17</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance.
 Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features.
 Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations.
 Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF -mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0ef45b0d0460ea2916e782f771c000702057b" target='_blank'>
              1347 Using Living Human Glioblastoma Samples to Develop Tumor-Targeted AAV Gene Therapy Vectors
              </a>
            </td>
          <td>
            M. Guevara, JP McGinnis, Joshua Ortiz-Guzman, Sai Mallannagari, Daniel Curry, Benjamin R. Arenkiel
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/806ce0b8ce24491e542e1befccfa41c6236040a0" target='_blank'>
              Circulating tumor cell (CTC) expression patterns of cell surface targets in metastatic prostate cancer.
              </a>
            </td>
          <td>
            Kendra D. Marr, J. Sperger, Amy K Taylor, Katharine E Tippins, Shannon R Reese, K. Helzer, M. Bootsma, G. Blitzer, John M Floberg, D. Kosoff, Rana R McKay, Xiao X Wei, Shuang Zhao, Joshua M Lang, Marina N. Sharifi
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059cc807139dc64898d45632991b55c44cdfb1d6" target='_blank'>
              Abstract A108: Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients
              </a>
            </td>
          <td>
            Muyi Yang, Sólrún Kolbeinsdóttir, Katarina Hammarlund, V. Grozman, Martin Enge, Suzanne Egyhazi-Brage, H. Helgadottir
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Anti-PD-1/PD-L1 immune checkpoint blockade (ICB) therapy has revolutionized clinical cancer treatment, while abnormal PD-L1 or HLA-I expression in patients can significantly impact the therapeutic efficacy. Somatic mutations in cancer cells that modulate these critical regulators are closely associated with tumor progression and ICB response. However, a systematic interpretation of cancer immune-related mutations is still lacking. Here, we harnessed the ABEmax system to establish a large-scale sgRNA library encompassing approximately 820,000 sgRNAs that target all feasible serine/threonine/tyrosine residues across the human genome, which systematically unveiled thousands of novel mutations that decrease or augment PD-L1 or HLA-I expression. Beyond residues associated with phosphorylation events, our screens also identified functional mutations that affect mRNA or protein stability, DNA binding capacity, protein-protein interactions, and enzymatic catalytic activity, leading to either gene inactivation or activation. Notably, we uncovered certain mutations that concurrently modulate PD-L1 and HLA-I expression, represented by the clinically relevant mutation SETD2_Y1666. We demonstrated that this mutation induces consistent phenotypic effects across multiple cancer cell lines and enhances the efficacy of immunotherapy in different tumor models. Our findings provide an unprecedented resource of functional residues that regulate cancer immunosurveillance, offering valuable guidance for clinical diagnosis, ICB therapy, and the development of innovative drugs for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0151a36f604dedfea97d97f1dfa3c6ae2a57eb97" target='_blank'>
              Massively parallel interrogation of human functional variants modulating cancer immunosurveillance
              </a>
            </td>
          <td>
            Ying Liu, Yongshuo Liu, Xuran Niu, Ang Chen, Yizhou Li, Ying Yu, Binrui Mo, Zhiheng Liu, Tao Xu, Jie Cheng, Zeguang Wu, Wensheng Wei
          </td>
          <td>2025-03-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Ductal carcinoma in situ (DCIS) is a risk factor for subsequent invasive breast cancer. To identify events in DCIS that lead to invasive cancer, we performed single-cell RNA-sequencing (scRNA-seq) on DCIS lesions and matched normal breast tissue. Inferred copy number variation (CNV) was used to identify neoplastic epithelial cells from clinical specimens, which contained a mixture of DCIS and normal ducts. Phylogenetic analysis demonstrated intratumoral clonal heterogeneity that was associated with significant gene expression differences. Classification of epithelial cells into mammary cell states revealed that subclones contained a mixture of cell states, suggesting an ongoing pattern of differentiation after neoplastic transformation. Cell state proportions were significantly different based on estrogen receptor (ER) expression with ER-negative DCIS more closely resembling the distribution in the normal breast, particularly with respect to cells with basal characteristics. Specific alterations in cell state proportions were associated with progression to invasive cancer in a cohort of DCIS with longitudinal outcome. Ongoing transcription of key basement membrane (BM) genes occurred in specific subsets of epithelial cell states, including basal/myoepithelial, which are diminished in DCIS. In the transition to invasive breast cancer, the BM protein laminin, but not COL4, was altered in DCIS adjacent to invasion. Loss of COL4, but not laminin, in an in vitro DCIS model led to an invasive phenotype. These findings suggest that the process of invasion is a loss-of-function event due to an imbalance in critical cell populations essential for BM integrity rather than a gain of an invasive phenotype by neoplastic cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5661c366f2eb52b8794cdbe190cb026b878cad" target='_blank'>
              Single Cell Expression Analysis of Ductal Carcinoma in Situ Identifies Complex Genotypic-Phenotypic Relationships Altering Epithelial Composition.
              </a>
            </td>
          <td>
            Xiaodi Qin, Siri H Strand, Marissa R Lee, Aashrith Saraswathibhatla, David G P van IJzendoorn, ChunFang Zhu, S. Vennam, S. Varma, Allison Hall, Rachel E Factor, Lorraine M. King, Lunden Simpson, Xiaoke Luo, G. Colditz, Shu Jiang, Ovijit Chaudhuri, E. S. Hwang, Jeffrey R. Marks, K. Owzar, Robert B. West
          </td>
          <td>2025-03-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7ca6a31f84637afae275f44ff0e8aae63741e5a" target='_blank'>
              Exploring the impact of RPN1 on tumorigenesis and immune response in cancer.
              </a>
            </td>
          <td>
            P. Luo, Zhimin Li, Haodong He, Yuanbin Tang, Lijun Zeng, Lunqi Luo, Lianjie Ouyang, Meiling Wen, Yuehua Li, Yongjun Jiang
          </td>
          <td>2025-03-13</td>
          <td>FASEB journal : official publication of the Federation of American Societies for Experimental Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de5a20b3e991f355b9226c4775967ff31f0cec4a" target='_blank'>
              Bridging the gap between histopathology and genomics: Spotlighting spatial omics.
              </a>
            </td>
          <td>
            Sujata Jana, Raisa A. Glabman, Amanda L. Koehne
          </td>
          <td>2025-03-26</td>
          <td>Veterinary pathology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f7b4d0623804b6e4ad33245c8e8741d967977b" target='_blank'>
              Relationship of spatial transcriptomic analyses with distance-associated transcriptomic spectrum and morphology-specific signatures of localized prostate cancer.
              </a>
            </td>
          <td>
            Brendan Raizenne, Hanbing Song, Nancy Y. Greenland, Maggie Tsui, Jared Khan, Taiqi Li, Chih-Hao Chang, M. Cooperberg, Franklin W. Huang
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Background & Aims Patients with colorectal cancer have heterogeneous clinical responses to chemotherapy, although clinical guidelines advise little variability in treatment selection based on molecular tumor features. Precision oncology research typically utilizes patient-derived tumor organoids (PDTO) to predict clinical outcomes, but such efforts are often not directed towards identification of molecular factors underlying differential responses to therapy. Methods Bulk RNA-sequencing was performed on treatment-naive PDTOs, and gene expression data was combined to drug sensitivity data to identify transcriptomic features associated with low in vitro sensitivity to chemotherapy. Whole-exome sequencing was performed on primary tumors to infer the somatic mutations of PDTOs and used to identify somatic mutations associated with differential in vitro drug responses. Publicly available gene expression and drug sensitivity data sets were used to validate the results. RNA interference was used for functional validation. Results PDTOs with low chemosensitivity had high JAK-STAT pathway activity resulting from high expression of interferon-stimulated genes. Evidence from single-cell RNA-sequencing confirmed chemotherapy-induced expression of interferon-stimulated genes in epithelial cells of cancers with partial response. EPSTI1 knockdown decreased cancer cell viability and sensitized cells to chemotherapy. Conclusions Sustained interferon signaling in epithelial cancer cells contributes to incomplete pathologic response in colorectal cancer. The findings highlight the potential of JAK-STAT inhibition or TRAIL pathway activation to enhance chemotherapy efficacy. Future studies investigating pharmacologic modulation of these pathways in preclinical CRC models are needed to determine their viability as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ab87f4ae646ab5e983158fb38a2489ac5894d6" target='_blank'>
              Sustained epithelial interferon signaling modulates incomplete pathologic response in colorectal cancer
              </a>
            </td>
          <td>
            E. Erkan, Emmi Hämäläinen, Julia Kolikova, Kornelia Kuc, K. Ojala, Meiju Kukkonen, Ismaïl Hermelo, S. Koskensalo, T. Tarvainen, Alli Leppä, I. Keränen, Carola Haapamäki, E. Karjalainen, Monica Carpelan-Holmström, L. Renkonen-Sinisalo, Laura E Koskenvuo, P. Puolakkainen, J. Mecklin, Elina Virolainen, M. Nykter, L. Aaltonen, Anna H. Lepistö, Ari Ristimäki, Toni T Seppälä
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Cell division drives somatic evolution but is challenging to quantify. We developed a framework to count cell divisions with DNA replication-related mutations in polyguanine homopolymers. Analyzing 505 samples from 37 patients, we studied the milestones of colorectal cancer evolution. Primary tumors diversify at ~250 divisions from the founder cell, while distant metastasis divergence occurs significantly later, at ~500 divisions. Notably, distant but not lymph node metastases originate from primary tumor regions that have undergone surplus divisions, tying subclonal expansion to metastatic capacity. Then, we analyzed a cohort of 73 multifocal lung cancers and showed that the cell division burden of the tumors' common ancestor distinguishes independent primary tumors from intrapulmonary metastases and correlates with patient survival. In lung cancer too, metastatic capacity is tied to more extensive proliferation. The cell division history of human cancers is easily accessible using our simple framework and contains valuable biological and clinical information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97569fe84c28d5e70fd4d5d44460cfdc4508b03d" target='_blank'>
              Quantifying cell divisions along evolutionary lineages in cancer.
              </a>
            </td>
          <td>
            Martin Blohmer, David M Cheek, Wei-Ting Hung, Maria Kessler, Foivos Chatzidimitriou, Jiahe Wang, William Hung, I-Hsiu Lee, Alexander N Gorelick, Emma Ce Wassenaar, Ching-Yeuh Yang, Yi-Chen Yeh, H. Ho, Dorothee Speiser, Maria M Karsten, Michael Lanuti, Sara I Pai, O. Kranenburg, J. Lennerz, Teh-Ying Chou, Matthias Kloor, Kamila Naxerova
          </td>
          <td>2025-02-04</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e95e6d9d4fbec0ea9d09f8e2bd810ce1221be487" target='_blank'>
              Unveiling the Role of the Cellular Tumor Microenvironment and the Therapeutic Targets it Provides in Cutaneous T-Cell Lymphoma.
              </a>
            </td>
          <td>
            Nikolaos A. Chinas, S. Kaliampou, V. Nikolaou
          </td>
          <td>2025-03-08</td>
          <td>Current oncology reports</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fffa6682c8cf8e3d73d30e552a59d8713e1c83f9" target='_blank'>
              Abstract A073: Digital assessment of tertiary lymphoid structures in cancer using whole slide scans
              </a>
            </td>
          <td>
            S. M. Park, Luis Munoz-Erazo, Shelly Lin, J. C. Chun-Jen, Lisa Y.Y. Zhou, J. Rhodes, Taesung Jeon, Sonya Fenton, John L McCall, Roslyn A. Kemp, P. R. Dunbar
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is characterized by expanding blood cell clones carrying somatic mutations in healthy aged individuals and is associated with various age-related diseases and all-cause mortality. While CH mutations affect diverse genes associated with myeloid malignancies, their mechanisms of expansion and disease associations remain poorly understood. We investigate the relationship between clonal fitness and clinical outcomes by integrating data from three longitudinal aging cohorts (n=713, observations=2,341). We demonstrate pathway-specific fitness advantage and clonal composition influence clonal dynamics. Further, the timing of mutation acquisition is necessary to determine the extent of clonal expansion reached during the host individual’s lifetime. We introduce MACS120, a metric combining mutation context, timing, and variant fitness to predict future clonal growth, outperforming traditional variant allele frequency measurements in predicting clinical outcomes. Our unified analytical framework enables standardized clonal dynamics inference across cohorts, advancing our ability to predict and potentially intervene in CH-related pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df776ea20c82ef08bfb6dc73e82dc94bd703a7e7" target='_blank'>
              Clinical progression of clonal hematopoiesis is determined by a combination of mutation timing, fitness, and clonal structure
              </a>
            </td>
          <td>
            Eric Latorre-Crespo, Neil A. Robertson, E. Kosebent, Louise Macgillivray, Lee Murphy, M. Uddin, Eric Whitsel, Michael C. Honigberg, Alexander G. Bick, A. Reiner, Valeria Orru, Michele Marongiu, Francesco Cucca, E. Fiorillo, I. Deary, Sarah Harris, Simon Cox, Riccardo Marioni, Linus Schumacher, Tamir Chandra, Kristina Kirschner
          </td>
          <td>2025-03-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6248a04912b285f929efafcc71ddd28019bc2f15" target='_blank'>
              Clonal hematopoiesis is clonally unrelated to multiple myeloma and is associated with specific microenvironmental changes.
              </a>
            </td>
          <td>
            M. Lionetti, Margherita Scopetti, A. Matera, Akihiro Maeda, Alessio Marella, Francesca Lazzaroni, Giancarlo Castellano, S. Fabris, Stefania Pioggia, Silvia Lonati, Alfredo Marchetti, Alessandra Cattaneo, Marta Tornese, A. Neri, Claudia Leoni, L. Pettine, Valentina Traini, Ilaria Silvestris, M. Barbieri, G. Fabbiano, D. Ronchetti, E. Taiana, C. De Magistris, M. D. Da Via', Francesco Passamonti, N. Bolli
          </td>
          <td>2025-03-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29965c5d14b51c6573686cfc61d8b621a40178b6" target='_blank'>
              Abstract IA019: Addressing genomic heterogeneity in glioblastoma personalized vaccine design
              </a>
            </td>
          <td>
            Elaine R Mardis, E. Garfinkle, T. Johanns, Katherine E Miller, Gavin P Dunn
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Pluripotent stem cells provide a scalable approach to analyse molecular regulation of cell differentiation across developmental lineages. Here, we engineer barcoded induced pluripotent stem cells to generate an atlas of multilineage differentiation from pluripotency, encompassing an eight-day time course with modulation of WNT, BMP, and VEGF signalling pathways. Annotation of in vitro cell types with reference to in vivo development reveals diverse mesendoderm lineage cell types including lateral plate and paraxial mesoderm, neural crest, and primitive gut. Interrogation of temporal and signalling-specific gene expression in this atlas, evaluated against cell type-specific gene expression in human complex trait data highlights the WNT-inhibitor gene TMEM88 as a regulator of mesendodermal lineages influencing cardiovascular and anthropometric traits. Genetic TMEM88 loss of function models show impaired differentiation of endodermal and mesodermal derivatives in vitro and dysregulated arterial blood pressure in vivo. Together, this study provides an atlas of multilineage stem cell differentiation and analysis pipelines to dissect genetic determinants of mammalian developmental physiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1610b8e09ca06ead7a9826cda40c37396bdae40e" target='_blank'>
              Atlas of multilineage stem cell differentiation reveals TMEM88 as a developmental regulator of blood pressure
              </a>
            </td>
          <td>
            Sophie Shen, Tessa Werner, S. Lukowski, Stacey Andersen, Yuliangzi Sun, W. J. Shim, Dalia Mizikovsky, Sakurako Kobayashi, Jennifer Outhwaite, Han Sheng Chiu, Xiaoli Chen, G. Chapman, E. M. Martin, Di Xia, Duy Pham, Zezhuo Su, Daniel Kim, Pengyi Yang, Men Chee Tan, Enakshi Sinniah, Qiongyi Zhao, Sumedha Negi, Meredith A. Redd, Joseph E Powell, Sally L. Dunwoodie, Patrick P L Tam, Mikael Bodén, J. W. Ho, Quan H Nguyen, N. Palpant
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf77d83d4281bc62f48275909f7e2fd555ec0bfd" target='_blank'>
              Abstract B059: Mechanisms of Sustainable T-cell Recruitment in a Perfusion-Enabled 3D Interstitial Network
              </a>
            </td>
          <td>
            Duy T. Nguyen, Laxmi Swetha Karanam, Said J Cifuentes Maury, Zoe Zhou
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Neoantigen-based cancer vaccine therapy represents a promising precision oncology strategy that targets unique tumor-specific mutations to elicit a robust immune response. This therapeutic approach is designed to harness the host’s immune response against tumor-specific neoantigens to eliminate cancer cells. The efficacy of neoantigen vaccines dependents on the coordinated action of diverse immune cells, including T lymphocytes, dendritic cells, B lymphocytes, natural killer cells, and macrophages. Each cell type plays a distinct and crucial role in recognizing, targeting, and destroying malignant cells. Understanding the mechanisms governing both individual and collective immune cell dynamics is crucial for therapeutic success. This comprehensive review systematically explores the mechanisms of neoantigen-specific immune cells, their dynamic interactions, and clinical application progress, aiming to unveil their potential value and future development in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d77226d01d41a06fee99f24389f787e9c744c885" target='_blank'>
              Unveiling the immunological landscape: comprehensive characterization of neoantigen-reactive immune cells in neoantigen cancer vaccines
              </a>
            </td>
          <td>
            Weijiang Liu, Hong Su, Peng Guo, Liping Hong, Xishan Hao, Xiaoling Li
          </td>
          <td>2025-03-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c13871b20a0df0f49db2baf60ca96fdc2c42238" target='_blank'>
              Spatially-structured inflammatory response in the presence of a uniform stimulus
              </a>
            </td>
          <td>
            Elizabeth R. Jerison, Nicolas Romeo, Stephen R. Quake
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Retrospective data support overall survival (OS) advantage to high clonal tumor mutation burden (cTMB), high clonal neoantigen load (cNEO) and low intratumor heterogeneity (ITH) in cancer patients who receive immunotherapy. In order to explore this relationship prospectively with Vigil, a triple function targeted immunotherapy involving ovarian cancer patients in long term follow up of the Phase 2b VITAL trial, we developed an exome sequencing procedure and associated bioinformatics pipeline to determine clonal signal patterns. DNA libraries containing exome sequences tagged with unique molecular identifiers (UMI) were prepared from paired samples and sequenced on Illumina sequencers to high coverage depths of ~ 930X (tumor) and ~ 130X (normal). Raw sequence reads were processed into optimized binary alignment map (BAM) files, using the UMI information. The BAM files were inputted into modules for calling MHC-I alleles, annotating single nucleotide variants (SNVs) and small insertions/deletions (InDels), and for determination of allelic copy number. The outputs were used to predict the sequence of peptide neoantigens and to perform clonality analysis in order to assign each SNV and InDel in a patient tumor sample to a primary clone or subclone. The Clonal Neoantigen pipeline was further assessed using whole exome Illumina sequencing data from three previously published studies. Evaluation of the pipeline using synthetic sequencing data from a sub-clonal deconvolution tool benchmarking study, showed positive predictive value (PPV) and positive percent agreement (PPA) of > 97.5% and > 96.5%, respectively, for SNV and InDel detection with minimum requirements for variant density and allele fraction. Haplotype calls from the Clonal Neoantigen pipeline MHC-I/ MHC-II typing module matched a published benchmark for 91.5% of the calls in a sample of 99 patients. Analysis of exome sequencing data from 14 patients with advanced melanoma revealed a strong correlation between cTMB values determined by the Clonal Neoantigen pipeline as compared to those calculated from the published data (R2 = 0.99). Following validation, the wet lab process and Clonal Neoantigen pipeline was applied to a set of matched normal, tumor, and Vigil product samples from 9 (n = 27 samples) ovarian cancer subjects entered into the VITAL (CL-PTL-119) trial. Results demonstrated marked correlation (R2 = 0.98) of cTMB between tumor used to construct Vigil and Vigil product. Correlation between tumor and Vigil for the cNEO and ITH metrics, showed R2 values of 0.95 and 0.87, respectively. The consistency of the Clonal Neoantigen pipeline results with previously published data as well as the agreement between results for tumor and Vigil for the entire system provide a strong basis of support for utilization of this pipeline for prospective determination of cTMB, cNEO, and ITH values in clinical tumor tissue in order to explore possible correlative relationships with clinical response parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16718d2277420c09078544ae1939c2c13bd37fd2" target='_blank'>
              Exome sequencing shows same pattern of clonal tumor mutational burden, intratumor heterogenicity and clonal neoantigen between autologous tumor and Vigil product
              </a>
            </td>
          <td>
            David Willoughby, E. Bognar, L. Stanbery, Casey Nagel, G. Wallraven, Aman Pruthi, Nicholas Bild, Ericca Stamper, Donald Rao, Adam Walter, John Nemunaitis
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57fc0173898db1615005d0ab09ce022fdfc322eb" target='_blank'>
              The Transcriptomic Landscapes of Appendiceal Primary and Metastatic Tumors are Distinct.
              </a>
            </td>
          <td>
            Margaret A. Park, Richard Jacobson, Maria Genilo-Delgado, Amir Mohammadi, Carlos M. Morán-Segura, Solomon Alhassan, Yukihiro Nakanishi, J. Permuth, I. Imanirad, S. Dineen
          </td>
          <td>2025-02-22</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related deaths, of which adenocarcinoma is the most common subtype. Despite this, lung adenocarcinoma and its metastasis are poorly understood, due to difficulties in feasibly recapitulating disease progression and predicting clinical benefits of therapy. We outline a methodology to develop immunogenic orthotopic lung adenocarcinoma mouse models, by injecting cell-specific cre viruses into the lung of a genetically engineered mouse, which mirrors cancer progression defined by the International Association for the Study of Lung Cancer. Evaluation of different cre virus/concentrations models demonstrate remarkable consistency in cancer initiation and metastasis, allowing for high throughput, while showing differences in timing and severity, offering greater flexibility when selecting models. Histological and immune profiles reflect clinical observations suggesting similar mechanisms are recapitulated and preliminary data show resultant tumors to be responsive to clinical treatments. We present a clinically relevant, next-generation murine model for studying lung adenocarcinoma. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-90344-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81dd756da5910df487dda469964a4404cb5624ff" target='_blank'>
              Methodology of murine lung cancer mimics clinical lung adenocarcinoma progression and metastasis
              </a>
            </td>
          <td>
            Edison Q Kim, Emily Y. Kim, Eric P Knott, Yujie Wang, Cheng-Bang Chen, Jose R Conejo-Garcia, M. Wangpaichitr, Diane C Lim
          </td>
          <td>2025-02-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), the most common form of primary liver cancer, is a leading cause of cancer-related mortality worldwide1,2. HCC occurs typically from a background of chronic liver disease, caused by a spectrum of predisposing conditions. Tumour development is driven by the expansion of clones that accumulate progressive driver mutations3, with hepatocytes the most likely cell of origin2. However, the landscape of driver mutations in HCC is broadly independent of the underlying aetiologies4. Despite an increasing range of systemic treatment options for advanced HCC, outcomes remain heterogeneous and typically poor. Emerging data suggest that drug efficacies depend on disease aetiology and genetic alterations5,6. Exploring subtypes in preclinical models with human relevance will therefore be essential to advance precision medicine in HCC7. Here we generated a suite of genetically driven immunocompetent in vivo and matched in vitro HCC models. Our models represent multiple features of human HCC, including clonal origin, histopathological appearance and metastasis. We integrated transcriptomic data from the mouse models with human HCC data and identified four common human–mouse subtype clusters. The subtype clusters had distinct transcriptomic characteristics that aligned with the human histopathology. In a proof-of-principle analysis, we verified response to standard-of-care treatment and used a linked in vitro–in vivo pipeline to identify a promising therapeutic candidate, cladribine, that has not previously been linked to HCC treatment. Cladribine acts in a highly effective subtype-specific manner in combination with standard-of-care therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cf01180ccde9b6fda123113aa98da6a623ebc73" target='_blank'>
              Human-correlated genetic models identify precision therapy for liver cancer
              </a>
            </td>
          <td>
            M. Müller, S. May, Holly Hall, Timothy J. Kendall, Lynn McGarry, Lauriane Blukacz, Sandro Nuciforo, Anastasia Georgakopoulou, Thomas Jamieson, Narisa Phinichkusolchit, Sandeep Dhayade, Toshiyasu Suzuki, J. Huguet-Pradell, Ian R Powley, Leah Officer-Jones, R. Pennie, R. Esteban-Fabró, Albert Gris-Oliver, R. Pinyol, George L Skalka, J. Leslie, Matthew Hoare, Joep Sprangers, Gaurav Malviya, Agata Mackintosh, Emma Johnson, M. McCain, John Halpin, Christos Kiourtis, C. Nixon, Graeme Clark, W. Clark, R. Shaw, A. Hedley, Thomas M Drake, Ee Hong Tan, Matt Neilson, Daniel J Murphy, David Y Lewis, Helen L Reeves, John Le Quesne, Derek A Mann, Leo M. Carlin, K. Blyth, J. M. Llovet, Markus H Heim, Owen J. Sansom, Crispin J Miller, Thomas G Bird
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c1e4bc107eff35ffcc4410b1199b818c7c8a640" target='_blank'>
              ATAD2 Drives Prostate Cancer Progression to Metastasis.
              </a>
            </td>
          <td>
            Anindita Dutta, A. Rodriguez-Calero, Kacey Ronaldson-Bouchard, Anne Offermann, Daoud Rahman, Twinkle Bapuji Vhatkar, Dan Hasson, M. Alshalalfa, E. Davicioni, R. Karnes, Mark A. Rubin, G. Vunjak‐Novakovic, Cory Abate-Shen, J. M. Arriaga
          </td>
          <td>2025-02-05</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>132</td>
        </tr>

        <tr id="Background Survival of patients with metastatic castration-resistant prostate cancer (mCRPC) depends on the site of metastatic dissemination. Methods Patients with mCRPC were prospectively included in the CPCT-02 metastatic site biopsy study. We evaluated whole genome sequencing (WGS) of 378 mCRPC metastases to understand the genetic traits that affect metastatic site distribution. Results Our findings revealed that RB1, PIK3CA, JAK1, RNF43, and TP53 mutations are the most frequent genetic determinants associated with site selectivity for metastatic outgrowth. Furthermore, we explored mutations in the non-coding genome and found that androgen receptor (AR) chromatin binding sites implicated in metastatic prostate cancer differ in mutation frequencies between metastatic sites, converging on pathways that impact DNA repair. Notably, liver and visceral metastases have a higher tumor mutational load (TML) than bone and lymph node metastases, independent of genetic traits associated with neuroendocrine differentiation. We found that TML is strongly associated with DNA mismatch repair (MMR)-deficiency features in these organs. Conclusions Our results revealed gene mutations that are significantly associated with metastatic site selectivity and that frequencies of non-coding mutations at AR chromatin binding sites differ between metastatic sites. Immunotherapeutics are thus far unsuccessful in unselected mCRPC patients. We found a higher TML in liver and visceral metastases compared to bone and lymph node metastases. As immunotherapeutics response is associated with mutational burden, these findings may assist in selecting mCRPC patients for immunotherapy treatment based on organs affected by metastatic disease. Trial registration number NCT01855477. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01445-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8017838415dd6bdeb817e19ffdcf7da3197701" target='_blank'>
              Whole genome sequencing of 378 prostate cancer metastases reveals tissue selectivity for mismatch deficiency with potential therapeutic implications
              </a>
            </td>
          <td>
            Daniel J Vis, S. Palit, Marie Corradi, Edwin Cuppen, Niven Mehra, M. Lolkema, Lodewyk F. A. Wessels, M. S. van der Heijden, W. Zwart, A. Bergman
          </td>
          <td>2025-03-20</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c10e56461ea00a2fad1aa2d1a734733ab751d66f" target='_blank'>
              Integrative multi-omic profiling of the neoantigen landscape of glioblastoma for the development of therapeutic vaccines reveals vast heterogeneity in immunogenic signatures
              </a>
            </td>
          <td>
            Qingtang Lin, Yukui Wei, Geng Xu, Leiming Wang, Feng Ling, Xiaojie Chen, Ye Cheng, Yiming Zhou
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d654bf452e2e4a90e37163a7af80b45f3f9eb5df" target='_blank'>
              Abstract A038: Inducing tumor associated tertiary lymphoid structures using cellular therapy
              </a>
            </td>
          <td>
            Nour Amwas, D. Alizadeh, Christine J Brown, Victor Chiu, Colt A. Egelston, D. Gumber, Hanmin Wang, John Williams, L. Wang
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Advances in spatially-resolved transcriptomics (SRT) technologies have propelled the development of new computational analysis methods to unlock biological insights. The lowering cost of SRT data generation presents an unprecedented opportunity to create large-scale spatial atlases and enable population-level investigation, integrating SRT data across multiple tissues, individuals, species, or phenotypes. Here, unique challenges are described in the SRT data integration, where the analytic impact of varying spatial and biological resolutions is characterized and explored. A succinct review of spatially-aware integration methods and computational strategies is provided. Exciting opportunities to advance computational algorithms amenable to atlas-scale datasets along with standardized preprocessing methods, leading to improved sensitivity and reproducibility in the future are further highlighted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76adb13a2a1a29db8847f590c746a2e59e284b7" target='_blank'>
              Integrating Spatially-Resolved Transcriptomics Data Across Tissues and Individuals: Challenges and Opportunities.
              </a>
            </td>
          <td>
            Boyi Guo, Wodan Ling, S. H. Kwon, Pratibha Panwar, S. Ghazanfar, Keri Martinowich, SC Hicks
          </td>
          <td>2025-02-11</td>
          <td>Small methods</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Malignant mesothelioma is a highly aggressive cancer with a poor prognosis and limited therapeutic options. The tumor microenvironment (TME) plays a pivotal role in driving tumor progression, with immune cells influencing disease outcomes. However, the molecular mechanisms underpinning mesothelioma’s progression remain insufficiently understood. HLA-C, a class I major histocompatibility complex (MHC) molecule, has been implicated in immune modulation and cancer progression, but its specific role in mesothelioma has yet to be thoroughly investigated. Methods This study employed a comprehensive multi-omics approach, integrating single-cell RNA sequencing, expression quantitative trait loci (eQTL) analysis, and Mendelian randomization (MR), to elucidate the role of HLA-C in mesothelioma progression. We first analyzed HLA-C expression within the TME, with particular focus on immune cells, especially macrophages. Survival analysis was conducted using data from the TCGA mesothelioma cohort to assess the clinical relevance of HLA-C expression. We utilized mediated MR analysis to investigate the impact of DNA methylation on HLA-C expression, identifying key mediators such as inflammatory cytokines, immune cell populations, blood cell types, and metabolites that could potentially influence patient prognosis. Results HLA-C was predominantly expressed in macrophages, T cells, and NK cells within the TME, and higher expression levels were associated with improved patient survival. MR analysis revealed that DNA methylation regulates HLA-C expression, which in turn impacts mesothelioma outcomes. Mediated MR analysis, encompassing 91 inflammatory cytokines, 731 immune cell populations, 91 blood cell types, and 1400 metabolites, highlighted several critical mediators of HLA-C’s effect on prognosis, including IL-10, CD33 expression on CD33dim HLA DR- myeloid cells, the reticulocyte perturbation response, and the ADP-to-citrate ratio. Gene set enrichment analysis (GSEA) showed significant enrichment of immune-related and inflammatory pathways in patients with high HLA-C expression. Conclusion HLA-C, regulated by DNA methylation, plays a central role in mesothelioma prognosis by modulating immune responses, inflammatory cytokines, blood cell populations, and metabolic processes within the TME. Our findings suggest that HLA-C could serve as both a prognostic biomarker and a potential therapeutic target for mesothelioma, offering new insights into the molecular mechanisms driving this aggressive cancer. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04012-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4904fecc6fa927af49272155c376129c6ffe55e9" target='_blank'>
              DNA methylation-regulated HLA-C expression modulates immune responses and metabolic alterations to influence prognosis in mesothelioma
              </a>
            </td>
          <td>
            Huilin Zhang, Luanxue Yu, Yunqing Guo, Jiawei Ming, Zhenying Guo
          </td>
          <td>2025-03-25</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7891a3a8df530bd356f3a5da9253356957c1253d" target='_blank'>
              Hyper-mutational processes provide a head-start for non-optimal cancer driver mutations explaining atypical KRAS variants
              </a>
            </td>
          <td>
            Michael D. Nicholson, Ian Tomlinson
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [19],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>