<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-10-28 06:32:52 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral heterogeneity is intrinsically comprised of molecular alterations of tumor cells and extrinsically from interconnections with microenvironments. This study explores the spatial heterogeneity of ovarian clear cell carcinoma (OCCC), a rare cancer with significance to East Asian women. We profiled 21 tumors from matched ovarian and metastatic sites with spatial transcriptomic (ST) platforms including GeoMx and Visium CytAssist. Three subclusters of OCCC tumor cells enriched in oxidative phosphorylation, epithelial-mesenchymal transition, and inflammation pathways were identified with preferential geospatial localizations in tumor centers and invasive margins/tumor-stromal interfaces. These ST subclusters correlated with epithelial-mesenchymal gradients and prognostic features in larger cohorts. Single-cell ST in CosMx identified 9 cancer cell populations with organ-specific tropism. Trajectory analysis indicated a NEAT1-high subclone having founder characteristics, progressively dominating the metastatic niche, displaying mesenchymal traits. This founder feature correlated with clinical outcomes in PanCancer datasets. Our findings highlight the adaptive and context-specific interactions between cancer cells and microenvironments within spatial complexities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8349570f5b5a5cc83b0effd91ea241eef3a2ce6c" target='_blank'>
              Spatial Transcriptomic Profiling Of Advanced Ovarian Clear Cell Carcinoma Reveals Intra-Tumor Heterogeneity In Epithelial-Mesenchymal Gradient
              </a>
            </td>
          <td>
            Thang Truong Le, Duncan Yi-Te Wang, Ko-Chen Chen, Sebastian Rui-Gu Yan, Pei-Yu Chu, Tuan Zea Tan, Yi-Chia Chiu, Jia-Yuh Sheu, , Jieru Ye, Ya-Ting Tai, Hsueh-Fen Juan, Feng-Chiao Tsai, , Lin-Hung Wei, 
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba675eb14cd995fdc082910df7aeef7f5f1a429b" target='_blank'>
              Immune landscape of oncohistone-mutant gliomas reveals diverse myeloid populations and tumor-promoting function
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Alva Annett, E. Karimi, D. G. Topouza, Morteza Rezanejad, Yitong Liu, Michael McNicholas, Eduardo G Gonzalez Santiago, Dhana Llivichuzhca-Loja, Arne Gehlhaar, Selin Jessa, Antonella De Cola, Bhavyaa Chandarana, Caterina Russo, D. Faury, Geoffroy Danieau, Evan Puligandla, Yuhong Wei, M. Zeinieh, Qing Wu, S. Hébert, N. Juretic, Emily M. Nakada, Brian Krug, Valérie Larouche, Alexander G Weil, Roy W. R. Dudley, Jason Karamchandani, S. Agnihotri, D. Quail, B. Ellezam, Liza Konnikova, L. Walsh, Manav Pathania, Claudia L Kleinman, Nada Jabado
          </td>
          <td>2024-09-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Breast cancer progression to visceral organs such as lung and liver is regarded as a dreadful event, unequivocally associated with a poor prognosis. Yet, these vital sites are characterized by highly diverse cellular microenvironments and physiological functions, suggesting that they may influence cancer cells behavior in divergent ways. Unexpectedly, we find that while the liver microenvironment fosters metastasis-boosting properties of its metastatic seeds and favors secondary spread, the lungs impose a drastic roadblock to the same processes. Using patient data and mouse models of breast cancer that spontaneously metastasize to visceral organs, coupled with genetic barcode-mediated metastasis tracing, niche labeling technology and single cell analysis of both tumor cells and their direct microenvironment at each site, we characterize this dichotomy and identify differential microenvironmental factors that contribute to liver vs lung metastasis biology. Mechanistically, we uncover pulmonary cytokines, such as Granulin, whose activity naturally blocks metastatic properties. Conversely, we identify BMP2-producing endothelial cells within the liver metastatic niche, capable to enhance further spread. Together, we reveal an unexpected contrast in the site-specific behavior of visceral metastases in triple-negative breast cancer models, highlighting microenvironmental factors that contribute to its geographical diversity, as well as organ-specific opportunities for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9abbeb3d8e83ff0a32a06d5c39fa11a5d2c21ff" target='_blank'>
              Mapping the niche of breast cancer metastases in lung and liver
              </a>
            </td>
          <td>
            Magdalena K. Sznurkowska, Francesc Castro-Giner, Yongzhan Zhang, Alexander Ring, F. Schwab, Francesca Albrecht, Kirsten D. Mertz, Karin Strittmatter, Lauren L. Ozimski, Selina Budinjas, G. Auray, C. Giachino, Verdon Taylor, Ana Gvozdenovic, I. Malanchi, Heike Frauchiger-Heuer, Andreas Wicki, M. Vetter, Nicola Aceto
          </td>
          <td>2024-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e27ecfb76604dddc8d09da4d1bbaaf7b66725bf" target='_blank'>
              Spatial multiplexed immunofluorescence analysis reveals coordinated cellular networks associated with overall survival in metastatic osteosarcoma
              </a>
            </td>
          <td>
            Ryan A Lacinski, Sebastian A. Dziadowicz, Vincent K. Melemai, Brody Fitzpatrick, John J. Pisquiy, Tanya Heim, Ines Lohse, Karen Schoedel, Nicolas J. Llosa, Kurt R Weiss, B. Lindsey
          </td>
          <td>2024-09-27</td>
          <td>Bone Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background: HCC is a highly vascular tumor, and many effective drug regimens target the tumor blood vessels. Prior bulk HCC subtyping data used bulk transcriptomes, which contained a mixture of parenchymal and stromal contributions. Methods: We utilized computational deconvolution and cell-cell interaction analyses to cell type-specific (tumor-enriched and vessel-enriched) spatial transcriptomic data collected from 41 resected HCC tissue specimens. Results: We report that the prior Hoshida bulk transcriptional subtyping schema is driven largely by an endothelial fraction, show an alternative tumor-specific schema has potential prognostic value, and use spatially paired ligand-receptor analyses to identify known and novel (LGALS9 tumor-HAVCR2 vessel) signaling relationships that drive HCC biology in a subtype-specific and potentially targetable manner. Conclusions: Our study leverages spatial gene expression profiling technologies to dissect HCC heterogeneity and identify heterogeneous signaling relationships between cancer cells and their endothelial cells. Future validation and expansion of these findings may validate novel cancer-endothelial cell interactions and related drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e5076806fef62fc7b47a9b3751a8891a984dc18" target='_blank'>
              HCC spatial transcriptomic profiling reveals significant and potentially targetable cancer-endothelial interactions
              </a>
            </td>
          <td>
            Chenyue Lu, Amaya Pankaj, Michael J. Raabe, Cole Nawrocki, Ann Liu, Nova Xu, Bidish Kumar Patel, Matthew J. Emmett, Avril K Coley, Cristina R. Ferrone, Vikram Deshpande, I. Bhan, Y. Hoshida, David T. Ting, Martin J. Aryee, Joseph W Franses
          </td>
          <td>2024-09-27</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PDAC) is a rapidly progressing cancer that responds poorly to immunotherapies. Intratumoral tertiary lymphoid structures (TLS) have been associated with rare long-term PDAC survivors, but the role of TLS in PDAC and their spatial relationships within the context of the broader tumor microenvironment remain unknown. We generated a spatial multi-omics atlas encompassing 26 PDAC tumors from patients treated with combination immunotherapies. Using machine learning-enabled H&E image classification models and unsupervised gene expression matrix factorization methods for spatial transcriptomics, we characterized cellular states within TLS niches spanning across distinct morphologies and immunotherapies. Unsupervised learning generated a TLS-specific spatial gene expression signature that significantly associates with improved survival in PDAC patients. These analyses demonstrate TLS-associated intratumoral B cell maturation in pathological responders, confirmed with spatial proteomics and BCR profiling. Our study also identifies spatial features of pathologic immune responses, revealing TLS maturation colocalizing with IgG/IgA distribution and extracellular matrix remodeling. GRAPHICAL ABSTRACT HIGHLIGHTS Integrated multi-modal spatial profiling of human PDAC tumors from neoadjuvant immunotherapy clinical trials reveal diverse spatial niches enriched in TLS. TLS maturity is influenced by tumor location and the cellular neighborhoods in which TLS immune cells are recruited. Unsupervised machine learning of genome-wide signatures on spatial transcriptomics data characterizes the TLS-enriched TME and associates TLS transcriptomes with survival outcomes in PDAC. Interactions of spatially variable gene expression patterns showed TLS maturation is coupled with immunoglobulin distribution and ECM remodeling in pathologic responders. Intratumoral plasma cell and immunoglobin gene expression spatial dynamics demonstrate trafficking of TLS-driven humoral immunity in the PDAC TME. Significance We report a spatial multi-omics atlas of PDAC tumors from a series of immunotherapy neoadjuvant clinical trials. Intratumorally, pathologic responders exhibit mature TLS that propagate plasma cells into malignant niches. Our findings offer insights on the role of TLS-associated humoral immunity and stromal remodeling during immunotherapy treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b4043873d18d3b758a620d9a4f90e32bea7034" target='_blank'>
              Spatial multi-omics reveal intratumoral humoral immunity niches associated with tertiary lymphoid structures in pancreatic cancer immunotherapy pathologic responders
              </a>
            </td>
          <td>
            Dimitri N. Sidiropoulos, Sarah M. Shin, Meredith Wetzel, Alexander Girgis, Daniel Bergman, Ludmila Danilova, Susheel Perikala, Daniel H. Shu, Janelle M. Montagne, Atul Deshpande, James Leatherman, Lucie Dequiedt, Victoria Jacobs, Aleksandra Ogurtsova, Guanglan Mo, Xuan Yuan, Dmitrijs Lvovs, G. Stein-O’Brien, M. Yarchoan, Qingfeng Zhu, Elizabeth I. Harper, A. Weeraratna, A. Kiemen, Elizabeth M. Jaffee, Lei Zheng, W. Ho, Robert A. Anders, E. Fertig, Lusciane T. Kagohara
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Malignant melanoma is an aggressive cancer, with a high risk of metastasis and mortality rates, characterized by cancer cell heterogeneity and complex tumor microenvironment (TME). Single cell biology is an ideal and powerful tool to address these features at a molecular level. However, this approach requires enzymatic cell dissociation that can influence cellular coverage. By contrast, single nucleus RNA sequencing (snRNA-seq) has substantial advantages including compatibility with frozen samples and the elimination of a dissociation-induced, transcriptional stress response. To better profile and understand the functional diversity of different cellular components in melanoma progression, we performed snRNA-seq of 16,839 nuclei obtained from tumor samples along the growth of murine syngeneic melanoma model carrying a BRAFV600E mutation and collected 9 days or 23 days after subcutaneous cell injection. We defined 11 different subtypes of functional cell clusters among malignant cells and 5 different subsets of myeloid cells that display distinct global transcriptional program and different enrichment in early or advanced stage of tumor growth, confirming that this approach was useful to accurately identify intratumor heterogeneity and dynamics during tumor evolution. The current study offers a deep insight into the biology of melanoma highlighting TME reprogramming through tumor initiation and progression, underlying further discovery of new TME biomarkers which may be potentially druggable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d37af9fbd7e323bb1c7c0e6480590eefcaf22d16" target='_blank'>
              Single-Nuclei Transcriptome Profiling Reveals Intra-Tumoral Heterogeneity and Characterizes Tumor Microenvironment Architecture in a Murine Melanoma Model
              </a>
            </td>
          <td>
            S. Parab, Valery Sarlo, S. Capellero, Luca Palmiotto, A. Bartolini, Daniela Cantarella, M. Turi, Annamaria Gullà, Elena Grassi, Chiara Lazzari, Marco Rubatto, Vanesa Gregorc, Fabrizio Carnevale-Schianca, Martina Olivero, Federico Bussolino, V. Comunanza
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) resides in an immune rich microenvironment, yet, immune-based therapies have faltered in eliciting durable responses. Bridging this paradox requires a comprehensive understanding of leukemic interactions within the bone marrow microenvironment. We optimized a high-throughput tissue-microarray based pipeline for high-plex spatial immunofluorescence and mass cytometry imaging on a single slide, capturing immune, tumor, and structural components. Using unbiased clustering on the spatial K function, we unveiled the presence of tertiary lymphoid-like aggregates in bone marrow which we validated using spatial transcriptomics and an independent proteomics approach. We then found validated TLS signatures predictive of outcomes in AML using an integrated public 480 patient transcriptomic dataset. By harnessing high-plex spatial proteomics, we open the possibility of discovering of novel structures and interactions that underpin leukemic immune response. Further, our study’s methodologies and resources can be adapted for other bone marrow diseases where decalcification and autofluorescence present challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1b1bad3b36f52c6886c710f3a8b1e66619ef44" target='_blank'>
              Multimodal Spatial Proteomic Profiling in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Christopher Ly, Ivo Veletic, Christopher D. Pacheco, Enes Dasdemir, F. Jelloul, Sammy Ferri-Borgongo, Akshay V Basi, Javier A. Gomez, Jessica L Root, P. Reville, Padmanee Sharma, Sreyashi Basu, Andres E. Quesada, Carlos Bueso-Ramos, T. Manshouri, Miriam Garcia, Jared K. Burks, Hussein A. Abbas
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0222f474962325ec3943ada79fb1a6a8a5016a" target='_blank'>
              Spatial resolved transcriptomics reveals distinct cross-talk between cancer cells and tumor-associated macrophages in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Zhao-Ru Dong, Meng-Ya Zhang, Ling-Xin Qu, Jie Zou, Yong-Heng Yang, Yun-Long Ma, Chuncheng Yang, Xue-Lei Cao, Li-Yuan Wang, Xiao-Lu Zhang, Tao Li
          </td>
          <td>2024-09-11</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Objective Single-cell technologies have revolutionized our understanding of the phenotypic and transcriptional diversity of aortic leukocytes in atherosclerotic humans and mice. However, enzymatically dissociated tissues lose the spatial context of plaque cells in situ. Here we utilized imaging mass cytometry (IMC) combining with single-cell RNA sequencing (scRNA-seq) to characterize the spatial distribution dynamics, phenotypic transitions, metabolic and functional phenotypes, and the intercellular interaction networks of plaque cells during atherosclerotic progression. Additionally, the dynamic immune landscape of circulating leukocytes associated with atherosclerosis was characterized using cytometry of time of flight (CyTOF). Approach and Results A highly multiplexed IMC panel with 33 metal-conjugated antibodies was designed to generate 11 highly multiplexed histology images of aortic root tissues from ApoE-/- mice on high-fat diet at different stage of atherosclerosis. Using histoCAT, we identified 8 principal cell subtypes with distinct phenotypic and geographic dynamics. Furthermore, IMC-defined cell subsets partially corresponded to scRNA-seq-annotated aortic cell subtypes, including 4 macrophage subsets, neutrophils, smooth muscle cells (SMCs) and SMC-derived SEMs (Stem cell, endothelial cell and macrophage-like cell). Activation of inflammatory pathways, increased oxidative phosphorylation and augmented osteoclast differentiation were observed in macrophage populations, SMCs and SEMs from an early stage to advanced stage of atherosclerosis. Notably, cell neighborhood analysis by IMC uncovered multifaceted cell-cell interactions within the plaque, in particular in neutrophil-mediated interactions with smooth muscle cells and macrophages, which were confirmed by ligand-receptor interactions based on scRNA-seq data. Additionally, characterization of the peripheral immune cells by CyTOF revealed an increased ratio of myeloid cells to lymphocytes, and certain neutrophil and monocyte subpopulations also exhibited enhanced lipid metabolism and glycolysis as well as activated inflammatory signaling. Conclusion This study provides a dynamic spatiotemporal landscape of atherosclerotic lesions and peripheral leukocytes. The new information based on IMC may help understand atherosclerotic pathology and develop novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47ce553643345229db62995608b4a068983d0b49" target='_blank'>
              Molecular and spatiotemporal characterization of cells in murine atherosclerotic plaques
              </a>
            </td>
          <td>
            Pengbo Hou, Zhanhong Liu, Jiankai Fang, Ziyi Wang, Shisong Liu, Shiqing Wang, Gerry Melino, Peishan Li, Yufang Shi, Changshun Shao
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is an immensely complex ecosystem1,2. This complexity underlies difficulties in elucidating principles of spatial organization and using molecular profiling of the TME for clinical use3. Through statistical analysis of 96 spatial transcriptomic (ST-seq) datasets spanning twelve diverse tumor types, we found a conserved distribution of multicellular, transcriptionally covarying units termed ‘Spatial Groups’ (SGs). SGs were either dependent on a hierarchical local spatial context – enriched for cell-extrinsic processes such as immune regulation and signal transduction – or independent from local spatial context – enriched for cell-intrinsic processes such as protein and RNA metabolism, DNA repair, and cell cycle regulation. We used SGs to define a measure of gene spatial heterogeneity – ‘spatial lability’ – and categorized all 96 tumors by their TME spatial lability profiles. The resulting classification captured spatial variation in cell-extrinsic versus cell-intrinsic biology and motivated class-specific strategies for therapeutic intervention. Using this classification to characterize pre-treatment biopsy samples of 16 non-small cell lung cancer (NSCLC) patients outside our database distinguished responders and non-responders to immune checkpoint blockade while programmed death-ligand 1 (PD-L1) status and spatially unaware bulk transcriptional markers did not. Our findings show conserved principles of TME spatial biology that are both biologically and clinically significant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a400ff77ff3e3d5b4307dd66b458357a7b860383" target='_blank'>
              Conserved principles of spatial biology define tumor heterogeneity and response to immunotherapy
              </a>
            </td>
          <td>
            Vivek Behera, Hannah Giba, Ue-yu Pen, Anna Di Lello, Benjamin A. Doran, Alessandra Esposito, Apameh Pezeshk, C. Bestvina, Justin Kline, M. Garassino, Arjun S. Raman
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fae692b36b9cb8dfff150568e46638865d4946" target='_blank'>
              Single-cell and spatial proteo-transcriptomic profiling reveals immune infiltration heterogeneity associated with neuroendocrine features in small cell lung cancer
              </a>
            </td>
          <td>
            Ying Jin, Yuefeng Wu, Alexandre Reuben, Liang Zhu, C. Gay, Qingzhe Wu, Xintong Zhou, Haomin Mo, Qi Zheng, Junyu Ren, Zhaoyuan Fang, Teng Peng, Nan Wang, Liang Ma, Yun Fan, Hai Song, Jianjun Zhang, Ming Chen
          </td>
          <td>2024-09-04</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is heterogeneous with low tumor purity, prominent microenvironment and complex architecture, which preclude identification of shared tumor intrinsic biology within and across patients. We overcame these challenges by applying complementary spatial omics approaches – providing necessary resolution and context – to primary untreated PDAC tumors from 39 patients capturing 341,949 low-bulk and 531,718 single-cell spatial transcriptomes. Of these, 59,403 low-bulk profiles and 205,665 single-cells represented tumor cells. We leveraged this data to build on prior reports of tumor cell subtypes in PDAC. We identify 5 distinct malignant subtypes, excluding ductal-like and intraepithelial neoplasia cells, that span the classical-to-basal spectrum. One tumor subtype is highly proliferative and enriched for the DNA synthesis-condensation-mitosis transcriptional network suggesting this subset disproportionately contribute to tumor growth. Strikingly, the spectrum of tumor cell subtypes was present within all patients, suggesting that subtype-based treatment stratification may need refinement. We find that each subtype has its own distinct regulators and histology. The classical subtype has extensive cell intrinsic regulation, higher cell density and lower extracellular matrix content. In contrast, the basal subtype phenotype results from a combination of cell intrinsic and tumor microenvironment (TME) interactions, wherein the basal subtype has a lower cell density, is surrounded by activated stroma and dense collagen matrix, and is poised for hypoxic adaptation. We derive the composition of cellular neighborhoods providing an explanation for the complex architecture seen in PDAC. Tumor cell neighborhoods stratified based on which side of the classical-to-basal spectrum they fell. For example, cells towards the classical-side of the spectrum had more homogenous tumor neighborhoods that generally lacked intermediate and basal tumor cells. In contrast, cells towards the basal-side of the spectrum were likely to contain classical neighbors. More broadly, neighborhoods along this continuum also contained different proportions of immune and stromal cells, cell densities, collagen deposition, and TME adaptation. These observations support a model where tumor subtypes exist within disparate niches. Lastly, our atlas unifies the catalog of previously reported stromal cells (fibroblasts and stellate cells versus myofibroblastic and inflammatory cancer associated fibroblasts (CAF)) and describes a novel interferon stimulated gene (ISG) resistance signature (ISG.RS) fibroblast. We map the location of these various stromal cells relative to tumor subtypes. For example, myofibroblast CAFs and ISG.RS fibroblasts are enriched near the basal tumor subtype whereas inflammatory CAF abundance increases with distance away from tumor cells. In sum, our atlas provides a lens for stratifying tumor complexity in a cancer cell centric manner and provides a path for testing therapeutic hypothesis in the right cell subtype and context.
 Citation Format: Brian Rabe, Anna Lyubetskaya, Andrew Kavran, Yulong Bai, Andrew Fisher, Hannah Pliner, Alba Font-Tello, Anne Lewin, Ruifeng Hu, Alexandre P Alloy, Yelena Cheng, Chao Dai, Yunfan Fan, Constance Brett, Todd Brett, Lauren Giampapa, Soeren Stahlschmidt, Fayaz Seifuddin, Steven Vasquez Grinnell, Daniel Carrera, Carlos Rios, Tom Lila, Pradeep Kar, Abhishek Shukla, Rachael Bashford Rogers, Lara Heij, Mike Mason, Ryan Golhar, Isaac Neuhaus, Enas Abu Shah, Shivan Sivakumar, Jimena Trillo Tinoco, Benjamin J Chen, Konstantinos J Mavrakis, Eugene Drokhlyansky. In situ multi-modal characterization of pancreatic ductal adenocarcinoma reveals tumor cell identity as a defining factor of the surrounding microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B081.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566e9b3495ee9764c389c226c9c17e824614540e" target='_blank'>
              Abstract B081: In situ multi-modal characterization of pancreatic ductal adenocarcinoma reveals tumor cell identity as a defining factor of the surrounding microenvironment
              </a>
            </td>
          <td>
            Brian Rabe, Anna Lyubetskaya, Andrew Kavran, Yulong Bai, Andrew Fisher, Hannah Pliner, Alba Font-Tello, Anne Lewin, Ruifeng Hu, Alexandre P. Alloy, Yelena Cheng, Chao Dai, Yunfan Fan, Constance Brett, Todd Brett, Lauren Giampapa, Soeren Stahlschmidt, Fayaz Seifuddin, Steven Vasquez Grinnell, Daniel Carrera, Carlos Rios, Tom Lila, Pradeep Kar, Abhishek Shukla, Rachael Bashford Rogers, Lara R. Heij, Mike Mason, R. Golhar, Isaac Neuhaus, E. A. Shah, Shivan Sivakumar, Jimena Trillo Tinoco, Benjamin Chen, Konstantinos J Mavrakis, Eugene Drokhlyansky
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Diffuse Large B-cell Lymphoma (DLBCL) is the most prevalent subtype of non-Hodgkin’s lymphoma for which current therapeutic strategies remain insufficient. The diffuse nature of DLBCL, lacking distinct tissue structures, represents a challenge to elucidate the cellular organization and interactions within the tumor microenvironment (TME). In this study, we applied spatial transcriptomics to identify spatially-resolved gene expression profiles in 10 DLBCL tissue samples, identifying distinct immune cell infiltration and colocalization patterns. These profiles were classified into six cellular ecosystems (Cell-Eco) that differ in cellular composition, functional patterns, and neighborhood characteristics. The spatially-resolved Cell-Eco signatures provided prognostic scores that stratified patients with different overall survival rates. We also found that C1q+ tumor-associated macrophages are the primary cells interacting with malignant B cells and influencing the spatial architecture of the TME. This study provides novel biological insights into the complexity of the TME in DLBCL and highlights the potential prognostic value of its spatial organization. Graphical abstract Key findings Spatial transcriptomics classifies DLBCL tissues based on immune cell infiltration and colocalization patterns. DLBCL tumor microenvironment consists of cellular ecosystems (Cell-Eco) that differ in cellular composition, transcriptomic profiles and neighborhood characteristics. Spatially-resolved Cell-Eco signatures stratify patients with different overall survival. C1q+ tumor-associated macrophages primarily interact with malignant B cells and contribute to the spatial organization of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ebc56c31f2327f9141f086b678169242b28d1d" target='_blank'>
              Spatial transcriptomics unveils immune cellular ecosystems associated with patient survival in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Alba Díaz Herrero, Hector F Pelaez-Prestel, Lucile Massenet-Regad, Maëva Veyssiere, Julien Calvani, Caterina Cristinelli, Jacqueline Lehmann-Che, Véronique Meignin, Catherine Thieblemont, Véronique Blanc, V. Soumelis, P. Tonnerre
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background Smooth muscle cell (SMC) plasticity and phenotypic switching play prominent roles in the pathogenesis of multiple diseases, but their role in tumorigenesis is unknown. We investigated whether and how SMC diversity and plasticity plays a role in tumor angiogenesis and the tumor microenvironment. Methods and Results We use SMC-specific lineage-tracing mouse models and single cell RNA sequencing to observe the phenotypic diversity of SMCs participating in tumor vascularization. We find that a significant proportion of SMCs adopt a phenotype traditionally associated with macrophage-like cells. These cells are transcriptionally similar to ‘resolution phase’ M2b macrophages, which have been described to have a role in inflammation resolution. Computationally predicted by the ligand-receptor algorithm CellChat, signaling from BST2 on the surface of tumor cells to PIRA2 on SMCs promote this phenotypic transition; in vitro SMC assays demonstrate upregulation of macrophage transcriptional programs, and increased proliferation, migration, and phagocytic ability when exposed to BST2. Knockdown of BST2 in the tumor significantly decreases the transition towards a macrophage-like phenotype, and cells that do transition have a comparatively higher inflammatory signal typically associated with anti-tumor effect. Conclusion As BST2 is known to be a poor prognostic marker in multiple cancers where it is associated with an M2 macrophage-skewed TME, these studies suggest that phenotypically switched SMCs may have a previously unidentified role in this immunosuppressive milieu. Further translational work is needed to understand how this phenotypic switch could influence the response to anti-cancer agents and if targeted inhibition of SMC plasticity would be therapeutically beneficial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcad9e00e00b0b1e9b4d096ed61477beb04a0767" target='_blank'>
              BST2 induces vascular smooth muscle cell plasticity and phenotype switching during cancer progression
              </a>
            </td>
          <td>
            Caitlin F. Bell, Richard A. Baylis, N. G. Lopez, W. Ma, Hua Gao, Fudi Wang, Sharika Bamezai, Changhao Fu, Yoko Kojima, Shaunak S. Adkar, Lingfeng Luo, Clint L. Miller, N. Leeper
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Solid tumours exhibit a well‐defined architecture, comprising a differentiated core and a dynamic border that interfaces with the surrounding tissue. This border, characterised by distinct cellular morphology and molecular composition, serves as a critical determinant of the tumour's invasive behaviour. Notably, the invasive border of the primary tumour represents the principal site for intravasation of metastatic cells. These cells, known as circulating tumour cells (CTCs), function as ‘seeds’ for distant dissemination and display remarkable heterogeneity. Advancements in spatial sequencing technology are progressively unveiling the spatial biological features of tumours. However, systematic investigations specifically targeting the characteristics of the tumour border remain scarce. In this comprehensive review, we illuminate key biological insights along the tumour body‐border‐haematogenous metastasis axis over the past five years. We delineate the distinctive landscape of tumour invasion boundaries and delve into the intricate heterogeneity and phenotype of CTCs, which orchestrate haematogenous metastasis. These insights have the potential to explain the basis of tumour invasion and distant metastasis, offering new perspectives for the development of more complex and precise clinical interventions and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f6a6ffd5124276ada6cdb3c914a272e4284a04" target='_blank'>
              Single‐cell and spatial omics unravel the spatiotemporal biology of tumour border invasion and haematogenous metastasis
              </a>
            </td>
          <td>
            Xifu Cheng, Yuke Cao, Xiangyi Liu, Yuanheng Li, Qing Li, Dian Gao, Qiongfang Yu
          </td>
          <td>2024-09-30</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast cancer is the most prevalent malignant tumor among women with high heterogeneity. Traditional techniques frequently struggle to comprehensively capture the intricacy and variety of cellular states and interactions within breast cancer. As global precision medicine rapidly advances, single-cell RNA sequencing (scRNA-seq) has become a highly effective technique, revolutionizing breast cancer research by offering unprecedented insights into the cellular heterogeneity and complexity of breast cancer. This cutting-edge technology facilitates the analysis of gene expression profiles at the single-cell level, uncovering diverse cell types and states within the tumor microenvironment. By dissecting the cellular composition and transcriptional signatures of breast cancer cells, scRNA-seq provides new perspectives for understanding the mechanisms behind tumor therapy, drug resistance and metastasis in breast cancer. In this review, we summarized the working principle and workflow of scRNA-seq and emphasized the major applications and discoveries of scRNA-seq in breast cancer research, highlighting its impact on our comprehension of breast cancer biology and its potential for guiding personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d20b5e78664c0194ad04247bdfc15101c7c5f63" target='_blank'>
              Single-Cell RNA-Sequencing: Opening New Horizons for Breast Cancer Research
              </a>
            </td>
          <td>
            Lingyan Xiang, Jie Rao, Jingping Yuan, Ting Xie, Honglin Yan
          </td>
          <td>2024-08-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Human health depends on complex processes of intercellular interactions and single-cell type-specific functions. Dramatic improvements in single-cell RNA sequencing (scRNA-seq) have opened up the possibility to identify functional states of single cells. Still, spatial methods are crucial for under-standing transcript location in relation to tissue architecture. Here, we con-structed a large-scale mapping strategy for proteins based on scRNA-seq and fluorescent-based multiplex immunohistochemistry (mIHC). We focused on the spatiotemporal landscape of 12 distinct germ cell states in human testis and performed an in-depth characterization of ∼500 germ cell proteins. Quantitative spatial localization data based on a custom-built image analysis pipeline allowed us to cluster proteins according to expression, forming the basis for functional analysis. mRNA and protein expression dynamics showed multiple cases with low levels of co-expression, highlighting the necessity of studying protein levels in single-cell mapping projects. The pre-sented workflow holds promise for proteome-wide tissue studies in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80790c055a4a9b5f49862e97b9882e5929f77890" target='_blank'>
              A spatiotemporal atlas of human spermatogenesis based on single-cell transcriptomics and multiplex antibody imaging
              </a>
            </td>
          <td>
            Feria Hikmet, Loren Méar, Jonas Gustavsson, Gisele Miranda, Cheng Zhang, Borbala Katona, Rutger Schutten, K. von Feilitzen, Mattias Forsberg, Jan-Bernd Stukenborg, Mathias Uhlén, Cecilia Lindskog
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary Phenotypic diversity in tumors makes treatment difficult and leads to poor prognosis. A better understanding of what causes intratumoral heterogeneity and differences in cells that are within the same tumor will advance therapeutic options and improve patient outcomes. This review will summarize recent studies that have examined the various cancer cell phenotypes along the epithelial–mesenchymal spectrum present in tumors and how epigenetic factors and the mechanical properties of tumor tissue mediate these phenotypes. This review will also discuss the therapeutic implications of intratumoral heterogeneity, such as drug resistance and tumor recurrence, and highlight opportunities to address current limitations in cancer treatments. Abstract The tumor microenvironment comprises various cell types and experiences dynamic alterations in physical and mechanical properties as cancer progresses. Intratumoral heterogeneity is associated with poor prognosis and poses therapeutic challenges, and recent studies have begun to identify the cellular mechanisms that contribute to phenotypic diversity within tumors. This review will describe epithelial–mesenchymal (E/M) plasticity and its contribution to phenotypic heterogeneity in tumors as well as how epigenetic factors, such as histone modifications, histone modifying enzymes, DNA methylation, and chromatin remodeling, regulate and maintain E/M phenotypes. This review will also report how mechanical properties vary across tumors and regulate epigenetic modifications and E/M plasticity. Finally, it highlights how intratumoral heterogeneity impacts therapeutic efficacy and provides potential therapeutic targets to improve cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b710fff2d564553c94ffb93440614a32c0f2e22e" target='_blank'>
              Epithelial–Mesenchymal Plasticity and Epigenetic Heterogeneity in Cancer
              </a>
            </td>
          <td>
            Jessica L Sacco, Esther W. Gomez
          </td>
          <td>2024-09-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related deaths. It is unclear how intratumor heterogeneity (ITH) contributes to metastasis and how metastatic cells adapt to distant tissue environments. The study of these adaptations is challenged by the limited access to patient material and a lack of experimental models that appropriately recapitulate ITH. To investigate metastatic cell adaptations and the contribution of ITH to metastasis, we analyzed single-cell transcriptomes of matched primary tumors and metastases from patient-derived xenograft models of breast cancer. We found profound transcriptional differences between the primary tumor and metastatic cells. Primary tumors upregulated several metabolic genes, whereas motility pathway genes were upregulated in micrometastases, and stress response signaling was upregulated during progression. Additionally, we identified primary tumor gene signatures that were associated with increased metastatic potential and correlated with patient outcomes. Immune-regulatory control pathways were enriched in poorly metastatic primary tumors, whereas genes involved in epithelial-mesenchymal transition were upregulated in highly metastatic tumors. We found that ITH was dominated by epithelial-mesenchymal plasticity (EMP), which presented as a dynamic continuum with intermediate EMP cell states characterized by specific genes such as CRYAB and S100A2. Elevated expression of an intermediate EMP signature correlated with worse patient outcomes. Our findings identified inhibition of the intermediate EMP cell state as a potential therapeutic target to block metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4040c30de029de626b99afe51b1e8ce55165671f" target='_blank'>
              Single-cell analysis of breast cancer metastasis reveals epithelial-mesenchymal plasticity signatures associated with poor outcomes
              </a>
            </td>
          <td>
            Juliane Winkler, Weilun Tan, Catherine M. M. Diadhiou, Christopher S. McGinnis, Aamna J. Abbasi, Saad Hasnain, Sophia Durney, Elena Atamaniuc, Daphne A. Superville, L. Awni, Joyce V. Lee, Johanna H. Hinrichs, Patrick S. Wagner, Namrata Singh, Marco Y. Hein, M. Borja, Angela M. Detweiler, Su-Yang Liu, Ankitha Nanjaraj, Vaishnavi Sitarama, H. Rugo, Norma Neff, Zev J. Gartner, Angela Oliveira Pisco, Andrei Goga, S. Darmanis, Z. Werb
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>117</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54370344e70d12f9e093a0f96e506dd839c6df0" target='_blank'>
              Deciphering breast cancer dynamics: insights from single-cell and spatial profiling in the multi-omics era
              </a>
            </td>
          <td>
            Xin Xiong, Xin Wang, Cui-Cui Liu, Zhi-Ming Shao, Ke-Da Yu
          </td>
          <td>2024-09-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background & Aims: Hepatocellular carcinoma (HCC) develops in the context of chronic inflammation, however, the opposing roles the immune system plays in both the development and control of tumours is not fully understood. Mapping immune cell interactions across the distinct tissue regions could provide greater insight into the role individual immune populations have within tumours. Approach & Results: Using highly multiplexed imaging mass cytometry, we have mapped the immune landscape of tumour, invasive margin, and adjacent non-tumour regions across sixteen resected tumours comprising of 144 regions of interest and over 10,000 parameters. We show immune cell densities remain largely consistent across these three regions, except for subsets of monocyte-derived macrophages which are enriched within the tumours. Mapping cellular interactions across these regions in an unbiased manner identifies immune neighbourhoods comprised of tissue-resident T cells, dendritic cells, and various macrophage populations around perivascular spaces. Importantly, we identify multiple immune cells within these neighbourhoods interacting with VEGF+ perivascular macrophages. Conclusions: Cellular interactions, but not cell densities, differ between intratumoural and adjacent non-tumour regions in HCC. Unique intratumoural immune neighbourhoods around the perivascular space points to an altered landscape within tumours. Enrichment of VEGF+ perivascular macrophages within these tumours could play a key role in angiogenesis and vascular permeability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f97ce36e43e0dfde11d3ba7e6fa851a271fe9a2" target='_blank'>
              Spatial mapping of the HCC landscape identifies unique intratumoral perivascular-immune neighborhoods
              </a>
            </td>
          <td>
            Felix Marsh-Wakefield, C. Santhakumar, A. L. Ferguson, T. Ashhurst, Joo-Shik Shin, Fiona H. X. Guan, Nicholas J. Shields, Barry J. Platt, Givanna H. Putri, Ruta Gupta, Michael Crawford, Carlo Pulitano, C. Sandroussi, Jerome M Laurence, Ken Liu, Geoffrey W McCaughan, U. Palendira
          </td>
          <td>2023-09-15</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cell states are modulated by intrinsic driving forces such as gene expression noise and extrinsic signals from the tissue microenvironment. The distinction between intrinsic and extrinsic cell state determinants is essential for understanding the regulation of cell fate in tissues during development, homeostasis and disease. The rapidly growing availability of single-cell resolution spatial transcriptomics makes it possible to meet this challenge. However, available computational methods to infer topological tissue domains, spatially variable gene expression, or ligand-receptor interactions are limited in capturing cell state changes driven by crosstalk between individual cell types within the same niche. We present NiCo, a computational framework for integrating single-cell resolution spatial transcriptomics with matched single-cell RNA-sequencing reference data to infer the influence of the spatial niche on the cell state. By applying NiCo to mouse embryogenesis, adult small intestine and liver data, we demonstrate the capacity to predict novel niche interactions that govern cell state variation underlying tissue development and homeostasis. In particular, NiCo predicts a feedback mechanism between Kupffer cells and neighboring stellate cells limiting stellate cell activation in the normal liver. NiCo provides a powerful tool to elucidate tissue architecture and to identify drivers of cellular states in local niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a330dd175c406437712c3044cd06a601fe89865f" target='_blank'>
              NiCo Identifies Extrinsic Drivers of Cell State Modulation by Niche Covariation Analysis
              </a>
            </td>
          <td>
            Ankit Agrawal, Stefan Thomann, Sukanya Basu, D. Grün
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Bone marrow (BM) is a common site for solid tumor metastasis, often causing poor outcome. Here, we define the characteristics of BM-disseminated tumor cells (DTCs) using neuroblastoma as a model. We combined single-cell RNA-sequencing (scRNA-seq) and cell-surface protein analysis using 7 paired BM and primary tumor (PT) samples and found that DTCs contain a higher percentage of cycling cells and higher expression of neurodevelopmental genes compared to corresponding PT cells. In 6 patients, the copy number variation profile differed between PT cells and DTCs, indicating spatial heterogeneity. Within the BM, we detected dormant DTCs with potentially reduced chemosensitivity; this population contained cells expressing low levels of the immunotherapeutic antigen GD2 and increased NGFR expression. In conclusion, we characterized DTCs that are particularly challenging to target, offering new avenues for developing therapeutic strategies designed to target all subpopulations within the highly complex metastatic site, thereby preventing the development of drug-resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fc1c99c4919cd41656eef13d151b98e6e5eb661" target='_blank'>
              Metastatic tumor cells in bone marrow differ from paired neuroblastoma tumor and contain subsets with therapy-resistant characteristics
              </a>
            </td>
          <td>
            Caroline Hochheuser, A. Boltjes, Kaylee M. Keller, Simon Tol, Marieke van de Mheen, Carolina Pita Barros, Zeinab van Gestel-Fadaie, André B. P. van Kuilenburg, Sander van Hooff, C. Voermans, Jan J. Molenaar, G. Tytgat, Ilse Timmerman
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9797ff0fc8da1478461b88997f4f8cb4c2bab8" target='_blank'>
              Single-cell and spatial sequencing identifies senescent and germinal tumor cells in adamantinomatous craniopharyngiomas
              </a>
            </td>
          <td>
            Xianlong Wang, Jincheng Lin, Hongxing Liu, Chuan Zhao, Zhiwei Tu, Dapeng Xu, En Zhang, Zhongqing Zhou, Xueling Qi, Xingfu Wang, Zhixiong Lin
          </td>
          <td>2024-09-02</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19dd1cc421c82853f5d16e4b6c21745d87d8c500" target='_blank'>
              Characteristics and transcriptional regulators of spontaneous epithelial–mesenchymal transition in genetically unperturbed patient-derived non-spindled breast carcinoma
              </a>
            </td>
          <td>
            H. Lien, Hui-Chieh Yu, Wen-Hsuan Yu, Su-Fang Lin, Tom Wei-Wu Chen, I. Chen, Li-Ping Hsiao, Ling-Chun Yeh, Yu-Chia Li, Chiao Lo, Yen-Shen Lu
          </td>
          <td>2024-09-10</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains an intractable cancer characterized by significant inter- and intra-patient heterogeneity. Recent research has established connections between key genetic drivers and distinct transcriptional profiles in GBM, yet the interactions between genetic changes and the transcriptional and spatial heterogeneity remain unclear. This study used genetically engineered mouse models, coupled with spatially resolved and single-cell RNA sequencing, to explore how specific genetic alterations influence the tumor microenvironment (TME) and cellular responses spatially.



 We used nestin-Tva C57BL/6 mice and generated de novo GBM using overexpression of oncogenes in combinations with short hairpin RNAs (shRNAs) or deletion to various tumor suppressors, including Tp53, Cdkn2A, and Pten. We modeled key driver mutations for GBM: PDGFB, NF1, and EGFR. We conducted single-cell RNA sequencing, combined with spatially resolved transcriptomics and proteomics. Data integration techniques, including horizontal (mutual nearest neighbors) and vertical (weighted nearest neighbors) approaches, were used and analyzed with the SPATA2 software. Advanced computational methods, such as graph neural networks and single-cell deconvolution, were applied to examine GBM genotypes and the spatial variability within tumors and their TMEs.



 Our findings revealed persistent cellular heterogeneity among tumor cells under various genetic conditions. However, specific genetic alterations influenced the cells toward dominant transcriptional states. NF1 and EGFR mutations resulted in shifts towards MES and AC-like states, respectively, unlike PDGFB, which promoted a NPC/OPC-like state. These transcriptional shifts were strongly associated with distinct microenvironmental changes, such as enhanced neuronal signaling in PDGFB-driven GBMs versus pronounced immune infiltration in NF1 and EGFR models. Utilizing weighted co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA) alongside spatial-niche deconvolution, we identified a mesenchymal transition leading to an immunosuppressive TME in NF1-driven GBMs and increased lymphocyte presence in EGFR-amplified GBMs, especially around blood vessels. Similar spatial patterns were observed in human spatially resolved transcriptomic data, highlighting the complexity and variability of genetic drivers in human GBMs.



 Our genetically engineered mouse models provide a valuable platform for investigating GBM with defined driver mutations. By leveraging sophisticated analytical techniques and computational modeling, this study underscores the potential to elucidate the interactions between genetic drivers and the tumor microenvironment. Our approach paves the way for developing targeted therapies and precision medicine strategies in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0056b4191c0de2f537409a208d3f06f5659352" target='_blank'>
              OS07.7.A DISSECTING SPATIAL DIVERSITY OF GLIOBLASTOMA GENOTYPES
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Hambardzumyan, D. Heiland
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Head and neck squamous cell carcinoma (HNSCC) is a complex and varied cancer that is difficult to treat due to its biological diversity. Single-cell RNA sequencing (scRNAseq) offers a way to study HNSCC by examining individual cells within tumors. This technology helps identify different cell types and their roles in cancer development, diagnosis, and treatment. By understanding how various cells respond to treatments like chemotherapy and immunotherapy, scRNAseq can enhance the study of cancer biology and improve therapeutic outcomes. This review highlights the significant impact of scRNAseq on HNSCC research and its potential to advance the study of other cancers. Abstract Background/Objectives: Head and neck squamous cell carcinoma (HNSCC) is a highly heterogeneous and the most common form of head and neck cancer, posing significant challenges for disease management. The objective of this review is to assess the utility of single-cell RNA sequencing (scRNAseq) in addressing these challenges by enabling a detailed characterization of the tumor microenvironment (TME) at the cellular level. Methods: This review compiles and analyzes current strategies that utilize scRNAseq and other single-cell technologies in HNSCC research. Results: For HNSCC etiology, scRNAseq allows for the construction of cellular atlases, characterization of different cell types, and investigation of genes and processes involved in cancer initiation, development, and progression within the TME. In terms of HNSCC diagnosis and prognosis, the resolution offered by scRNAseq enables the identification of cell type-specific signatures, enhancing prognostic models and disease stratifiers for patient outcome assessments. Regarding HNSCC treatment, scRNAseq provides insights into cellular responses to various treatments, including radiotherapy, chemotherapy, and immunotherapy, contributing to a better understanding of treatment efficacy and patient outcomes. Conclusions: This review highlights the contributions of scRNAseq to HNSCC research, addressing its cellular and biological complexity, and emphasizes its potential for advancing research and clinical practice in other cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e831682e82f2a5f8a0b8ba438418409c9819bff6" target='_blank'>
              Unravelling the Complexity of HNSCC Using Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Cristina Conde-Lopez, Divyasree Marripati, M. Elkabets, Jochen Hess, Ina Kurth
          </td>
          <td>2024-09-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Microscopy and genomics are both used to characterize cell function, but approaches to connect the two types of information are lacking, particularly at subnuclear resolution. While emerging multiplexed imaging methods can simultaneously localize genomic regions and nuclear proteins, their ability to accurately measure DNA-protein interactions is constrained by the diffraction limit of optical microscopy. Here, we describe expansion in situ genome sequencing (ExIGS), a technology that enables sequencing of genomic DNA and superresolution localization of nuclear proteins in single cells. We applied ExIGS to fibroblast cells derived from an individual with Hutchinson-Gilford progeria syndrome to characterize how variation in nuclear morphology affects spatial chromatin organization. Using this data, we discovered that lamin abnormalities are linked to hotspots of aberrant euchromatin repression that may erode cell identity. Further, we show that lamin abnormalities heterogeneously increase the repressive environment of the nucleus in tissues and aged cells. These results demonstrate that ExIGS may serve as a generalizable platform for connecting nuclear abnormalities to changes in gene regulation across disease contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e86969e327346d3a55aa56def236fca89f4284c6" target='_blank'>
              Expansion in situ genome sequencing links nuclear abnormalities to hotspots of aberrant euchromatin repression
              </a>
            </td>
          <td>
            Ajay S. Labade, Zachary D. Chiang, Caroline Comenho, Paul L. Reginato, Andrew C. Payne, Andrew S. Earl, Rojesh Shrestha, Fabiana M. Duarte, Ehsan Habibi, Ruochi Zhang, George M. Church, E. Boyden, Fei Chen, Jason D. Buenrostro
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Recent advancements in spatial biology have revolutionized our understanding of the organization and functional dynamics of cells and tissues. In this study, we applied Molecular Pixelation (MPX), a single-cell spatial proteomics assay, to investigate the modulation of the cell surface proteome in an in vitro drug screening model using the ETV6::RUNX1 acute lymphoblastic leukemia (ALL) cell line, Reh. Specifically, we focused on the in vitro response to fludarabine, a chemotherapeutic agent used prior to allogenic stem cell transplantation and chimeric antigen receptor (CAR)-T cell therapy in high-risk, refractory, or relapsed ALL patients. Using MPX, we quantified changes in protein abundance, spatial distribution, and colocalization of 76 targeted cell surface proteins in Reh cells before and after fludarabine treatment. Our analysis revealed 25 proteins with altered abundance, 24 proteins with increased polarity, and 138 protein pairs with modified colocalization following treatment. Notably, the tetraspanins CD82 and CD53, which are known for their roles in chemotherapy resistance, exhibited increased abundance, polarization, and colocalization post-treatment, suggesting their potential as a therapeutic scaffold. These findings underscore the unique ability of spatially resolved single-cell proteomics to uncover nuanced cellular responses that would otherwise remain undetected.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f5a352c544200f32c43afeccf0c1c6f723f3d81" target='_blank'>
              Mapping the Spatial Proteome of Leukemia Cells Undergoing Fludarabine Treatment
              </a>
            </td>
          <td>
            , Henrik Gezelius, Anna Pia Enblad, Anders Lundmark, Mariya Lysenkova Wiklander, O. Krali, Martin Åberg, Arja Harila, A. Raine, Claes Andersson, J. Nordlund
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The interaction of glioblastoma (GB) and microglia is critical due to its implications for tumor progression, immune response modulation, and potential therapeutic strategies. However, the role of microglia in GB pathogenesis remains unclear, especially regarding the in vivo dynamics of their interplay. Performing three-photon imaging in an autochthonous, immunocompetent mouse GB model, we examined tumor/microglia dynamics within previously inaccessible regions at the GB far infiltration zone in the corpus callosum. Initially, microglia increased tissue surveillance upon encountering GB-cells in sparsely infiltrated areas. In contrast, when GB-cell density increased, microglia reduced surveillance, suggesting a biphasic response to tumor invasion. Additionally, microglia were not uniformly attracted to infiltrating GB-cells; only a subset moved directionally toward them within a defined spatial range. This study provides insight into the heterogeneity of the immune response to tumor invasion and the dynamics of microglia-GB interactions in vivo. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39820cb3045ee584de7bf3d731fda31accb858a" target='_blank'>
              Dynamics of microglia-glioblastoma crosstalk at the far infiltration zone
              </a>
            </td>
          <td>
            F. Nebeling, Falko Fuhrmann, M. Mittag, Fabrizio Mussachio, Nala Gockel, Lea L Friker, S. Leonardelli, S. Filser, A. Deli, Miriam Stork, D. Bano, Torsten Pietsch, S. Parrinello, M. Hölzel, U. Herrlinger, P. Salomoni, M. Fuhrmann
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Predicting the progression of solid cancers based solely on genetics is challenging due to the influence of the tumor microenvironment (TME). For colorectal cancer (CRC), tumors deficient in mismatch repair (dMMR) are more immune infiltrated than mismatch repair proficient (pMMR) tumors and have better prognosis following resection. Here we quantify features of the CRC TME by combining spatial profiling with genetic analysis and release our findings via a spatially enhanced version of cBioPortal that facilitates multi-modal data exploration and analysis. We find that ∼20% of pMMR tumors exhibit similar levels of T cell infiltration as dMMR tumors and that this is associated with better survival but not any specific somatic mutation. These T cell-infiltrated pMMR (tipMMR) tumors contain abundant cells expressing PD1 and PDL1 as well as T regulatory cells, consistent with a suppressed immune response. Thus, like dMMR CRC, tipMMR CRC may benefit from immune checkpoint inhibitor therapy. SIGNIFICANCE pMMR tumors with high T cell infiltration and active immunosuppression are identifiable with a mid-plex imaging assay whose clinical deployment might double the number of treatment-naïve CRCs eligible for ICIs. Moreover, the low tumor mutational burden in tipMMR CRC shows that MMR status is not the only factor promoting immune infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb91d144afd07ee13d239cb16bf5689a34f70783" target='_blank'>
              Integrating spatial profiles and cancer genomics to identify immune-infiltrated mismatch repair proficient colorectal cancers
              </a>
            </td>
          <td>
            Jeremiah Wala, Ino de Bruijn, S. Coy, Andréanne Gagné, Sabrina Chan, Yu-An Chen, J. Hoffer, J. Muhlich, N. Schultz, Sandro Santagata, P. Sorger
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bca16ff9172b241a4b47443f7584ae07d099226" target='_blank'>
              Transcriptomic and proteomic spatial profiling of pediatric and adult diffuse midline glioma H3 K27-Altered
              </a>
            </td>
          <td>
            Sudarshawn Damodharan, Jack M. Shireman, Elliot Xie, Emily Distler, Christina Kendziorski, Mahua Dey
          </td>
          <td>2024-09-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Anaplastic lymphoma kinase (ALK) rearrangement is a major oncogenic driver in non-small cell lung cancer (NSCLC). While ALK tyrosine kinase inhibitors have shown promising therapeutic effects, overcoming resistance with immunotherapy becomes necessary when resistance develops. However, various clinical trials have revealed that their efficacies remain limited. To investigate the tumor microenvironment (TME) factors contributing to poor immune checkpoint blockade responses in ALK-positive patients, we performed single-cell RNA and ATAC sequencing on lung adenocarcinoma (LUAD) tumors with and without ALK rearrangements. Integrative analysis with additional public LUAD cohorts revealed distinct immune landscapes in ALK-positive tumors, marked by enriched innate immunity and depleted adaptive immunity. ALK-positive malignant cells exhibit higher stemness and aggressive phenotype. Tumor-associated macrophages (TAMs) in these tumors predominantly maintain pro-tumoral M2-like states, reinforcing immune suppression. B cells show reduced immune reactivity and impaired tertiary lymphoid structure formation, while CD8+ T cells display bystander-like signatures and reduced tumor reactivity. Single-cell chromatin accessibility profiles combined with regulatory network analysis suggest that differences in transcription factor activities, rather than chromatin accessibility, may underlie T cell dysfunction. These findings provide insights into the immunosuppressive TME of ALK-positive LUAD, potentially explaining the failure of recent immunotherapy trials and highlighting targets for improving efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb7277d768d3078fb9b8e8f161ef723f32126" target='_blank'>
              Single-cell multi-omics reveals tumor microenvironment factors underlying poor immunotherapy responses in ALK-positive lung cancer
              </a>
            </td>
          <td>
            S. Baek, Euijeong Sung, Gamin Kim, M. Hong, Chang Young Lee, H. Shim, Seong Yong Park, Hye Ryun Kim, Insuk Lee
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba40a48a9694556eef4d0d290c4702c1cf12dc5" target='_blank'>
              Spatially resolved analysis of pancreatic cancer identifies therapy-associated remodeling of the tumor microenvironment.
              </a>
            </td>
          <td>
            C. Shiau, Jingyi Cao, Dennis Gong, M. Gregory, Nicholas J Caldwell, Xunqin Yin, Jae-Won Cho, Peter L. Wang, J. Su, Steven Wang, J. Reeves, T. Kim, Youngmi Kim, Jimmy A. Guo, N. A. Lester, Jung Woo Bae, Ryan Zhao, Nathan Schurman, J. L. Barth, M. L. Ganci, Ralph Weissleder, Tyler Jacks, M. Qadan, Theodore S. Hong, J. Y. Wo, Hannah J Roberts, J. Beechem, Carlos Fernández-del Castillo, M. Mino-Kenudson, David T Ting, M. Hemberg, W. L. Hwang
          </td>
          <td>2024-09-03</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Serrated lesions are common precancerous pathways in colorectal cancer (CRC), but the process by which they progress to malignancy remains unclear. We aimed to elucidate this progression through a single-cell RNA landscape. We conducted single-cell RNA sequencing on three normal colonic tissues and fifteen SLs (including HPs, SSLs, SSLD, and TSAs) and integrated these data with datasets containing tumor samples. We identified three invasive malignant epithelial cell subtypes related to CRC progression: SLC1, SLC2, and tumor cell. SLC1, specific to SSLs, is involved in cell proliferation and shows a continuum of malignancy in gene expression. TSA-specific SLC2 exhibited FOXQ1 upregulation and active EMT, indicating invasiveness. The trajectory analysis showed that HPs do not progress to cancer, and different SL types are linked to the MSI status of advanced CRCs. We validated molecular drivers in premalignant lesions and later carcinogenesis. In the tumor microenvironment, CAF and pre-CAF fibroblast subtypes associated with progression were identified. During the premalignant stage, SLC1 triggered CD8+ T cell responses, while at the advanced stage, CAFs promoted tumor invasion and metastasis via FN1-CD44, influencing tumor progression and the treatment response. Our findings highlight transcriptional changes across serrated pathway stages, aiding in early CRC diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f52080cc10a19d5a436d2837e1d1e6040efb30" target='_blank'>
              Single-Cell Transcriptomics Reveals Cellular Heterogeneity and Drivers in Serrated Pathway-Driven Colorectal Cancer Progression
              </a>
            </td>
          <td>
            Jiahui Wang, Yu Zhang, Xinyi Chen, Qi Sheng, Junluo Yang, Yuyao Zhu, Yuhang Wang, Fangrong Yan, Jingya Fang
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The transcriptional state of a cell provides a multi-parameter representation of gene expression programs that reflect its identity and phenotype. However, current transcriptomic profiling technologies result in destruction of the biological sample, preventing direct analysis of transcriptional dynamics in the same living cells over time. Here, we developed a synthetic RNA export system called cellular ‘self-reporting’ to address this fundamental technological limitation. We repurposed the murine leukemia virus retroviral protein Gag to enable diverse types of immortalized and primary mammalian cells to package cellular RNA molecules in virus-like particles (VLPs) for export into the extracellular environment. We applied self-reporting to interrogate the transcriptome-wide dynamics that occur during neuronal differentiation from induced pluripotent stem cells and detected gene expression profiles from individual live cells. Leveraging this genetically encodable approach, we expanded the capabilities of self-reporting through molecular engineering of VLP components. Pseudotyping VLPs with epitope-tagged envelope proteins enabled multiplexed selective live-cell readout of transcriptional states from heterogeneous co-cultures. Furthermore, structure-guided protein engineering of Gag fusions with human RNA binding domains improved the mRNA representation in self-reporting readouts and enabled the directed export of libraries of synthetic barcode transcripts. Taken together, this work establishes self-reporting as a facile and broadly enabling technology for live-cell, transcriptome-scale profiling of dynamic processes across diverse cell types and biological applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd9363ab1dcbb456eb0b3e0944d84bfdb375d0c" target='_blank'>
              Live-cell transcriptomics with engineered virus-like particles
              </a>
            </td>
          <td>
            M. Najia, Jacob Borrajo, Anna Le, FuNien Tsai, Jeremy Y. Huang, Linda G. Griffith, George Q Daley, P. Blainey
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="In clinical applications, spatial data collected under varying conditions, time points, or patients often lack discernible structural alignment. Computational tools designed to align adjacent tissue sections are unsuited for dealing with this structural heterogeneity. There is a growing demand for methods that can effectively align and compare spatial data in the absence of obvious visual correspondence. To address this challenge, we developed an interpretable cell mapping strategy by considering spatial context at various scales. Our approach outperforms existing mapping tools in dealing with heterogeneous samples and is flexible enough to map cells across samples, technologies, resolutions, developmental and regenerative time. Using our approach, we showed spatiotemporal decoupling of cells during development. We even performed alignment for a population of spatial data from cancer patients to identify sub- populations. Our interpretable mapping approach facilitates systemic comparison and analysis of heterogeneous spatial data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3efb8af02c2551620bc2f4404557c0e501a57546" target='_blank'>
              Multi-scale and multi-context interpretable mapping of cell states across heterogenous spatial samples
              </a>
            </td>
          <td>
            P. Martin, Wenqi Wang, Hyobin Kim, Henrietta Holze, Kyoung Jae Won
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Astrocytes comprise ∼50% of all brain cells and present distinct morphological, molecular and functional properties in different brain regions. In glioblastoma (GBM), an aggressive primary brain tumour, tumour-associated astrocytes (TAAs) become activated and exhibit different transcriptomic profiles, morphology and functions supporting disease progression. Heterogeneity and specific roles of TAAs within various regions of tumours are poorly known. Advancements of single-cell and spatial transcriptomics allow to profile tumours at unprecedented resolution revealing cell phenotypes, hidden functionalities and spatial architecture in disease-specific context. We combined spatial transcriptomics and multiple immunofluorescent staining to visualize TAAs heterogeneity and location of various subpopulations in intracranial murine gliomas. Using distinct gene expression profiles, we identified subtypes of TAAs with distinct localization and inferred their specialized functionalities. Gene signatures associated with TAAs reflected their reprograming in the tumour microenvironment (TME), revealed their multiple roles and potential contributing factors shaping the local milieu. Using spatial correlation analysis of the spots, we inferred the interactome of Slc1a2 (encoding a glutamate transporter) with the other markers of TAAs based on segregated areas of the tumour. The designer RGD peptide blocking tumour-microglia communications, alters the spatial distribution of TAAs in experimental gliomas providing insights into potential mechanisms. Spatial transcriptomics combined with multiple staining unveils multiple functional phenotypes of TAAs and interactions within TME. It shows their distinct morphology and unveils different roles in various regions of the tumour. We demonstrate the glioma-induced heterogeneity of TAAs and their adaption to the pharmacologically-induced modification of the immunosuppressive TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2a092533402eea8774b3945747486b079a48e5e" target='_blank'>
              Integration of spatial transcriptomics with immunofluorescence staining reveals spatial heterogeneity and plasticity of astrocytes in experimental glioblastomas
              </a>
            </td>
          <td>
            Mitrajit Ghosh, P. Pilanc-Kudlek, Karol Jacek, Szymon Baluszek, Katarzyna Poleszak, P. Szadkowska, B. Gielniewski, Aleksandra Ellert Miklaszewska, Bozena Kaminska
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Accurately deciphering tumor cellular heterogeneity is crucial for developing effective treatments. While single-cell epigenomics assays offer powerful insights into studying tumor heterogeneity beyond the transcriptional level, their application remains a significant challenge. To address this, we introduce Multimodal-based Analysis of scATAC-Seq data (MAAS), a method designed to identify functional tumor cell subpopulations and reconstruct their lineages from single-cell sequencing assay for transposase-accessible chromatin (scATAC-seq). MAAS uniquely integrates multimodal information, encompassing chromatin accessibility, copy number variations (CNVs), and single-nucleotide variants, through self-expressive multimodal matrix factorization. MAAS outperforms existing methods in accuracy and robustness on both simulated and real datasets. When applied to multiple glioma scATAC-seq datasets, MAAS uncovered a previously hidden tumor cell subpopulation resistant to temozolomide treatment. Furthermore, MAAS effectively detected clinically relevant subpopulations with low CNVs, such as those found in pediatric ependymoma and B-cell lymphoma. Additionally, we developed a novel MAAS-derived clinical signature that provides superior prognostic prediction than traditional clinicopathologic features across multiple cancer types. In summary, MAAS significantly advances the identification of clinically relevant tumor cell subpopulations, thereby accelerating the discovery of potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b29ddaa37d46a0d04b1320e8bee3cd217df7912" target='_blank'>
              Multimodal integration of single cell ATAC-seq data enables highly accurate delineation of clinically relevant tumor cell subpopulations
              </a>
            </td>
          <td>
            Kewei Xiong, Ruofan Ding, Yangmei Qin, Xudong Zou, Jiguang Wang, Chen Yu, Lei Li
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8b0d44c911ccfcc30fd21e07313533937dc5a4" target='_blank'>
              Ultralow-dose irradiation enables engraftment and intravital tracking of disease initiating niches in clonal hematopoiesis
              </a>
            </td>
          <td>
            Kevin Lee, Wimeth Dissanayake, Melissa MacLiesh, Cih-Li Hong, Zi Yin, Yuko Kawano, Christina M Kaszuba, Hiroki Kawano, Emily R. Quarato, Brian Marples, Michael Becker, Jeevisha Bajaj, Laura M Calvi, Shu-Chi A. Yeh
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfb73fa1be8db27e4aafae1a4087794eca77af2e" target='_blank'>
              Spatial interactions modulate tumor growth and immune infiltration
              </a>
            </td>
          <td>
            S. Marzban, Sonal Srivastava, Sharon Kartika, Rafael Bravo, Rachel Safriel, Aidan Zarski, Alexander R. A. Anderson, Christine H Chung, Antonio L. Amelio, Jeffrey West
          </td>
          <td>2024-09-30</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The tumor microenvironment of brain metastases has become a focus in the development of immunotherapeutic drugs. However, countless brain metastasis patients have not experienced clinical benefit. Thus, understanding the immune cell composition within brain metastases, and how the immune cells interact with each other and other microenvironmental cell types, may be critical for optimizing immunotherapy. We applied spatial whole transcriptomic profiling with extensive multiregional sampling (19-30 regions per sample) and multiplex immunohistochemistry on formalin-fixed, paraffin-embedded lung cancer brain metastasis samples. We performed deconvolution of gene expression data to infer the abundances of immune cell populations and inferred spatial relationships from the multiplex immunohistochemistry data. We also described cytokine networks between immune and tumor cells and used a protein language model to predict drug-target interactions. Finally, we performed deconvolution of bulk RNA data to assess the prognostic significance of immune-metastatic tumor cellular networks. We show that immune cell infiltration has a negative prognostic role in lung cancer brain metastases. Our in-depth multiomics analyses further reveal recurring intratumoral immune heterogeneity and the segregation of myeloid and lymphoid cells into distinct compartments that may be influenced by distinct cytokine networks. By employing computational modeling, we identify drugs that may target genes expressed in both tumor core and regions bordering immune infiltrates. Finally, we illustrate the potential negative prognostic role of our immune-metastatic tumor cellular networks. Our findings advocate for a paradigm shift from focusing on individual genes or cell types, towards targeting networks of immune and tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488bc40a00ebddc1b6645e7141cbe9f55941284e" target='_blank'>
              Spatial multiomics reveal intratumoral immune heterogeneity with distinct cytokine networks in lung cancer brain metastases.
              </a>
            </td>
          <td>
            Gustav Christensson, M. Bocci, Julhash U. Kazi, Geoffroy Durand, Gustav Lanzing, Kristian Pietras, Hugo Gonzalez Velozo, C. Hagerling
          </td>
          <td>2024-10-14</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1374088281d4f8b66ab8cd65f25ae26c49ad31" target='_blank'>
              Spatial transcriptomics reveals tumor-derived SPP1 induces fibroblast chemotaxis and activation in the hepatocellular carcinoma microenvironment
              </a>
            </td>
          <td>
            W. Tong, Tianze Wang, Yi Bai, Xingpeng Yang, Pinsheng Han, Liuyang Zhu, Yamin Zhang, Zhongyang Shen
          </td>
          <td>2024-09-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma is the most hypovascular solid tumor, imposing harsh constraints on tumor metabolism. Cancer cells adapt to reduced access to vascular-derived nutrients and O2 by cooperating with abundant stromal cells. Specifically, critical nutrients including alanine, proline, glycosaminoglycans, branched chain ketoacids, lysophosphatidic acid, and unsaturated fatty acids can be provided by fibroblasts in vitro and in vivo. However, the extent of metabolic support provided by other cell types in the PDAC TME is relatively unexplored. We hypothesized that cell types that are cooperating metabolically with cancer cells will be spatially positioned near cancer cells in these tumors. We performed Xenium spatial transcriptomics on 8 treatment-naive PDAC resections and generated a spatial atlas of nearly 700,000 cells across a range of expected cell types. We noted striking neutrophil aggregates, nucleating in the glands of classical-like malignant cells and elaborating extraglandularly. Cellular neighborhood analysis demonstrates neutrophil aggregation is strongly correlated with malignant cell proliferation and low endothelial content, suggesting neutrophils may be an important source of pro-tumoral, non-vascular nutrients in the PDAC TME. Reanalysis of published scRNAseq data revealed 3 neutrophil metabolic states in human PDAC tumors. Metabolic cluster 7 neutrophils were found only in PDAC patient samples but not healthy controls and were enriched for Type I interferon response as well as pentose phosphate pathway and iron uptake. Metabolic cluster 2 neutrophils were strikingly depleted from PDAC tumor samples compared to blood, suggesting decreased infiltration or increased cell death. Cluster 2 neutrophils were enriched for several catabolic pathways including glycogenolysis, autophagy, and proteasome, and selectively express the NETosis gene PADI4. Multiplexed immunofluorescence demonstrated that neutrophil aggregates in PDAC are citH3+/MPO+ large neutrophil extracellular traps (NETs). We reasoned pancreatic cancer cells could proliferate under tumor-like metabolic stress by repurposing amino acids liberated by autophagy, glucose liberated by glycogenolysis, or simply consuming bulk neutrophil dead cell debris via macropinocytosis. Using tumor interstitial fluid media (TIFM) and 0.5% O2 to mimic tumor-like metabolic stress, we cocultured pancreatic cancer cells with human primary neutrophils and HL60-derived neutrophil-like cells. However, under tumor-like metabolic conditions, coculture with human primary neutrophils failed to substantially increase cancer cell proliferation and NETotic neutrophils were cytotoxic. However, we noticed enhanced neutrophil viability in tumor-like metabolic conditions, suggesting metabolic regulation of neutrophil cell death. Together, these data shed light on the metabolic phenotypes and interactions with cancer cells of neutrophils in the PDAC tumor microenvironment.
 Citation Format: Carson D Poltorack, Sydney M Shaffer, M Celeste Simon. Spatial and single cell transcriptomics uncovers metabolic reprogramming and NETosis of neutrophils in human pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A065.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e97afc3e27a38cdf1cbacb0a0b2dbe91b12facfe" target='_blank'>
              Abstract A065: Spatial and single cell transcriptomics uncovers metabolic reprogramming and NETosis of neutrophils in human pancreatic cancer
              </a>
            </td>
          <td>
            Carson D. Poltorack, Sydney M Shaffer, M. C. Simon
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Multiplex imaging technologies have developed rapidly over the past decades. The advancement of multiplex imaging has been driven in part by the recognition the spatial organisation of cells can represent important prognostic biomarkers and that simply studying the composition of cells in diseased tissue is often insufficient. There remains a lack of tools that can perform spatial analysis at the level of cellular aggregates (a common histopathological presentation) such as tumors and granulomas, with most analysis packages focussing on smaller regions of interest and potentially missing patterns in the overall lesion structure and cellular distribution. Here we present a protocol to quantify the cellular structure of entire tissue lesions built around two novel metrics. The Total Cell Preference Index reports whether a lesion tends to change in density in its central vs peripheral areas and can indicate the extent of necrosis across the entire lesion. The Immune Cell Preference Index then reports whether each immune cell-type is located more centrally or peripherally across the entire lesion. The output of both indexes is a single number readout for simple interpretation and visualization, and they can be applied to lesions of any size or shape. Additionally, this protocol can be applied to any slide-scanning multiplexed imaging system, either based on protein and nucleic acid staining. Finally, the protocol uses the open-source software QuPath and can be utilized by researchers with a basic understanding of QuPath with the full protocol able to be applied on pre-generated images within one hour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd048c5276460698536d41824ce54a133b750a5e" target='_blank'>
              A highly adaptable protocol for mapping spatial features of cellular aggregates in tissues
              </a>
            </td>
          <td>
            Andrew Sawyer, Nick Weingaertner, E. Patrick, Carl G Feng
          </td>
          <td>2024-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is the most prevalent type of kidney cancer characterized by its diverse tumor composition featuring various subclonal populations that hinder effective treatment responses. Tumoroids present an avenue for modelling this diversity and replicating the intricate tumor heterogeneity. Spatial transcriptomics preserves the spatial context of gene expression enabling us to study distinct tumor areas and the influence on overall diversity. Our spatial transcriptomics analysis uncovered tumor clusters with distinct genetic profiles that showcase various functional areas in depth and offer valuable understandings into the diversity of ccRCC types. Some of these tumor clusters exhibited activity in genes responsible for protein catabolism and reduced abundance of genes related to mitochondrial respiration processes. We also show isoform expression within tumoroids, in particular glutaminase (GLS) especially with the prevalence of the highly metabolically active GAC isoform that is expressed in regions where mitochondrial gene abundance is lower; whereas the KGA isoform displayed a more focal expression pattern. Combining long-read spatial transcriptomics with organoid models presents a novel strategy for unravelling gene and transcript level complexity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8b349aa856c9b8fd9c5542bf55647949ae547e0" target='_blank'>
              Long-read Spatial Transcriptomic Profiling of Patient-derived ccRCC Tumoroids Reveals Heterogeneity in Isoform and Gene Expression
              </a>
            </td>
          <td>
            Mustafa Elshani, Hazem Abdullah, Ying Zhang, Alexander Laird, Peter Mullen, David J. Harrison
          </td>
          <td>2024-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Castleman disease (CD) is inflammatory lymphoproliferative disorder of unclear etiology. To determine the cellular and molecular basis of CD, we analyzed the spatial proteome of 4,485,009 single cells, transcriptome of 50,117 single nuclei, immune repertoire of 8187 single nuclei, and pathogenic mutations in Unicentric CD, idiopathic Multicentric CD, HHV8-associated MCD, and reactive lymph nodes. CD was characterized by increased non-lymphoid and stromal cells that formed unique microenvironments where they interacted with lymphoid cells. Interaction of activated follicular dendritic cell (FDC) cytoplasmic meshworks with mantle zone B cells was associated with B cell activation and differentiation. VEGF, IL-6, MAPK, and extracellular matrix pathways were elevated in stromal cells of CD. CXCL13+ FDCs, PDGFRA+ T-zone reticular cells (TRC), and ACTA2-positive perivascular reticular cells (PRC) were identified as the predominant source of increased VEGF expression and IL-6 signaling in CD. VEGF expression by FDCs was associated with peri-follicular neovascularization. FDC, TRC and PRC of CD activated JAK-STAT, TGFβ, and MAPK pathways via ligand-receptor interactions involving collagen, integrins, complement components, and VEGF receptors. T, B and plasma cells were polyclonal but showed class-switched and somatically hypermutated IgG1+ plasma cells consistent with stromal cell-driven germinal center activation. In conclusion, our findings show that stromal cell activation and associated B-cell activation and differentiation, neovascularization and stromal remodeling underlie CD and suggest new targets for treatment. Key points Castleman Disease is characterized by activation and proliferation of CXCL13+ FDCs, PDGFRA+ reticular cells, and ACTA2-positive PRCs. VEGF and IL-6 from lymph node stromal cells are associated with B-cell activation and differentiation, endothelial proliferation, and inflammation in CD">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6effec836eaf8e7b0862b8cace8add3a480a246e" target='_blank'>
              Spatial and Single Cell Mapping of Castleman Disease Reveals Key Stromal Cell Types and Cytokine Pathways
              </a>
            </td>
          <td>
            David Smith, Anna Eichinger, Andrew Rech, Julia Wang, E. Esteva, Arta Seyedian, Xiaoxu Yang, Mei Zhang, Dan Martinez, Kai Tan, Minjie Luo, Christopher Y Park, Boris Reizis, Vinodh Pillai
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Microglia, the resident immune cells of the Central Nervous System (CNS), play crucial roles in homeostasis, immune responses, and the modulation of neuronal functions. In brain tumors, microglia are difficult to distinguish from infiltrating macrophages and are collectively called Tumor-Associated Macrophages (TAMs). While these cells exhibit plasticity, support tumor growth, and suppress antitumor immune responses, their identity and function remain largely unknown. Cellular senescence, a state of growth arrest associated with the upregulation of the cell cycle inhibitor p16INK4A (Cdkn2a), can both suppress and promote tumor progression via the secretion of inflammatory and tissue remodeling factors, collectively known as Senescence-Associated Secretory Phenotype (SASP). By leveraging existing single-cell RNA sequencing (scRNA-seq) data of murine brain tumors, we identified a population of Cdkn2a-positive TAMs involved in immune regulation and neurogenesis, characterized by high expression of Emb and Cd93, which we named MECC (Microglia-like, and Emb-, CD93- and Cdkn2a-positive) cells. In humans, these cells were more prevalent in recurrent glioblastoma and associated with poorer prognosis. Comparative analysis of brain samples from patients with neurodegeneration and healthy older individuals revealed the specificity of MECCs for brain tumors. These findings suggest that MECCs are a unique and distinct cell population associated with aggressive brain tumors, which could potentially serve as targets for novel treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2a0c4ffc5a390cae95577c04e0afb14d0a9ab78" target='_blank'>
              Transcriptional profiling reveals a previously unknown population of Cdkn2a-positive tumor-associated macrophages in aggressive brain cancer
              </a>
            </td>
          <td>
            Letícya Simone Melo dos Santos, Lucas Kich Grun, Juliete Nathali Scholl, F. M. Barbé-Tuana, M. Demaria
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a7e2a6359020fadce11d67094c49cba1e4ed0b" target='_blank'>
              Multi-scale transcriptomics reveals that specific tumor cells promote lung adenocarcinoma metastasis through crosstalk with the microenvironment
              </a>
            </td>
          <td>
            Qi Cai, Lin Shi, Meng-ya Zhang, Peng Chen
          </td>
          <td>2024-10-03</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Recent advances in neuro-oncology highlighted the meninges as an important immunological hub at the crossroads between systemic immunity and the central nervous system. Meningiomas are predominantly benign tumors arising from meningeal cells. However, about 20% are high-grade cases, commonly refractory to standard treatment, showing high recurrence rates. Methylation-based classification has crucially improved meningioma patient stratification. However, the biological processes that govern differences between these classes remain unknown. Here, we investigate the role of differential immune infiltration, cellular communities, and their spatial organization. In light of previously found differential immune infiltration across methylation classes, this study aims to spatially resolve the role of myeloid cells and study their influence on tumor progression and recurrence.



 We deployed multiplexed ion beam imagining and spatial transcriptomics on a retrospective cohort with extensive clinical annotations. Analyzing single-cell transcriptomic data of a subset of patients (n=22), we delineated the expression of gene and protein markers that were used to design an antibody panel comprising 41 targets for cell-type identification, phenotypic and metabolic characterization at the protein level. Additionally, we curated a targeted gene panel for spatial transcriptomics to perform granular spatial analysis. An integration of histological, spatial transcriptomic, and protein layers provides a comprehensive multi-omics approach to studying spatial cellular interactions.



 Transcriptional programs and cell populations observed in single-nuclei data were captured in the spatial context revealing differences in the myeloid compartment across meningioma methylation classes. In specific classes, myeloid cells constitute a significant part of the tumor bulk warranting further investigation into their phenotypic differences. Accordingly, our analyses revealed a distinct phenotypic macrophage landscape along disease progression. Furthermore, spatial profiling of the metabolic regulome displayed distinct niche formation within perivascular and tumor compartments.



 In this study we confirm that macrophages undergo a phenotypic switch from more tolerogenic in low-grade to inflammatory in high-grade cases of meningioma. Correspondingly, specific phenotypic and metabolic features of myeloid and tumor cells contribute to tumor progressions. Considering the limited targeted therapy options and poor responses to immunotherapy in meningioma patients, we hope that our spatial multi-omics analysis may reveal recurrent infiltration patterns of phenotypically distinct cell types that could be informative of biological processes and predictive of disease outcome.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c6aee05db85854153c8ed4386e1bdf7d8947c0" target='_blank'>
              P02.23.A SPATIAL PROFILING OF THE MYELOID COMPARTMENT IN MENINGIOMA REVEALS DISTINCT PHENOTYPIC HETEROGENEITY ACROSS METHYLATION CLASSES
              </a>
            </td>
          <td>
            D. Calafato, Y. Tang, G. Rukhovich, B. Patel, A. Suwala, F. Sahm, M. Gerstung, F. Hartmann
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Despite recent advances in cancer treatment, only 20-40% of all patients respond to immunotherapy. Further, efficacy varies highly by tumor type. Tumors with high infiltration of canonically anti-tumoral immune cells, such as lung cancers, also tend to display increased sensitivity to treatment. In contrast, those with higher infiltration of pro-tumoral immune cells, such as pancreatic cancers, are much less amenable to immunotherapy. These differences in immune infiltration have been linked to differential transcriptional regulation of immune cell recruiting factors, suggesting that perhaps tumors can intrinsically assemble different immune microenvironments in a tissue-of-origin dependent manner. Thus, it is necessary to better understand the factors that lead to heterogeneity of immune cell infiltration, how these mechanisms differ across tumor types, and how they may be perturbed to improve therapy responses. Here, we utilized two panels of clonal cell lines both driven by Kras-G12D and Tp53-R172H mutations but derived from two different tissues of origin: the lung and pancreas. These panels serve to recapitulate the immune heterogeneity of non-small cell lung cancer and pancreatic ductal adenocarcinoma, respectively, while controlling for genetic drivers that may influence immune infiltration. Analyses of subcutaneously implanted lung and pancreatic-origin tumors revealed that the overall abundance of immunosuppressive granulocytic myeloid-derived suppressor cells (gMDSCs) is significantly higher in pancreatic-origin tumors while the infiltration of dendritic cells is much lower as compared to that of lung-origin tumors, despite controlling for engraftment site. Likewise, lung, but not pancreatic-origin tumors exhibit more rapid tumor growth upon natural killer (NK) cell depletion, suggesting a lung-origin specific role of NK cells in controlling tumor burden. There were also similarities in immune archetype, with T-cell abundance negatively correlating with that of gMDSCs. However, whereas “T-cell inflamed” pancreatic tumors partially respond to immune checkpoint blockade, their lung-origin counterparts do not. We hypothesized that these differences in immune phenotype and subsequent therapy response may be due in part to somatic copy number alterations (SCNAs). We performed sparse whole genome sequencing and identified substantial genomic rearrangements in both lung and pancreatic-origin tumors. Notably, the specific regions of rearrangements differed highly between the two panels, with lung-origin clones showing more amplifications in chromosomes 2 and 11 and deletions in chromosomes 16 and 18. These results suggest that there are indeed tissue-of-origin specific differences in immune phenotype that can influence therapy outcome. Ongoing work seeks to further elucidate specific tumor cell-intrinsic factors that can be leveraged to skew the tumor immune microenvironment towards a more anti-tumoral phenotype and ultimately increase sensitivity to immunotherapy in a site-specific manner.
 Citation Format: Jane Xie, Timour Baslan, Ben Stanger. Tissue-of-origin determines immune cell heterogeneity and response to therapy via unique tumor cell-intrinsic factors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A046.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/882907c818197419b7f2d1a08e333d371fa45d1f" target='_blank'>
              Abstract A046: Tissue-of-origin determines immune cell heterogeneity and response to therapy via unique tumor cell-intrinsic factors
              </a>
            </td>
          <td>
            Jane Xie, T. Baslan, Ben Z. Stanger
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Spatially resolved transcriptomics (SRT) is poised to advance our understanding of cellular organization within complex tissues under various physiological and pathological conditions at unprecedented resolution. Despite the development of numerous computational tools that facilitate the automatic identification of statistically significant intra-/inter-slice patterns (like spatial domains), these methods typically operate in an unsupervised manner, without leveraging sample characteristics like physiological/pathological states. Here we present PASSAGE (Phenotype Associated Spatial Signature Analysis with Graph-based Embedding), a rationally-designed deep learning framework for characterizing phenotype-associated signatures across multiple heterogeneous spatial slices effectively. In addition to its outstanding performance in systematic benchmarks, we have demonstrated PASSAGE’s unique capability in identifying sophisticated signatures in multiple real-world datasets. The full package of PASSAGE is available at https://github.com/gao-lab/PASSAGE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/270944202d0822bdc6128135ef2baaa8fa8c720b" target='_blank'>
              Learning phenotype associated signature in spatial transcriptomics with PASSAGE
              </a>
            </td>
          <td>
            Chen-Kai Guo, Chen-Rui Xia, Guangdun Peng, Zhi-Jie Cao, Ge Gao
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Adult-type diffuse gliomas pose significant clinical challenges due to their complex cellular makeup and evolving nature. Understanding this heterogeneity is crucial for developing effective targeted therapies.



 To understand glioma heterogeneity, we performed single-cell and single-nucleus RNA sequencing on 66 tumours (single surgery) and 54 paired (two surgeries) longitudinal samples. This dataset comprises approximately 900,000 cells, enabling us to investigate both the conserved cellular landscape and dynamic cellular evolution across different glioma subtypes.



 Employing a novel phylogeny-based approach, we showed that the astrocyte-like glioma stem cell (GSC) is the origin of diverse glioma lineages. This finding represents a paradigm shift in our understanding of glioma development, highlighting the GSC as a potential therapeutic target. The GSC’s resemblance to healthy adult neural stem cells provides insight into the roots of glioma heterogeneity and suggests a shared origin for intratumoral diversity. Our analysis revealed a consistent cellular architecture spanning astrocytomas, oligodendrogliomas, and glioblastomas (GBMs). This architecture comprises seven recurring malignant cell states: neuro-, oligo-, neuro-oligo-lineage, cycling (G1/S, G2/M), hypoxia-associated, and a quiescent astrocyte-like GSC. Notably, the proportions of these cell states differ among glioma subtypes. We further show that GBMs exhibit significantly higher T-cell and myeloid cell infiltration than IDH-mutant gliomas. Paradoxically, this correlates with a worse prognosis, suggesting profound T-cell dysfunction and a complex interplay between myeloid cells and the tumour. Longitudinal analysis underscored the evolving nature of the glioma’s malignant cell populations and immune landscape. We identified subtype-specific T-cell and myeloid gene expression profiles, including targets like PARD6G (GBM) and CXCL13 (astrocytoma), which offer a foundation for developing targeted immunotherapies and prognostic biomarkers.



 Our study provides a model of glioma heterogeneity and evolution. This model integrates a shared astrocyte-like GSC driving cellular diversity alongside distinct, dynamic immune landscapes across different glioma subtypes. These findings offer unprecedented potential for developing transformative treatment strategies. Future research functionally validating the GSC and its impact on tumour dynamics will be crucial for translating these insights into improved patient outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dde712df597ac070934e4c044b4a4b9ad246fe2" target='_blank'>
              KS01.5.A A BRAIN CANCER CELL ATLAS REVEALS SHARED ASTROCYTE-LIKE STEM CELL STATES DRIVE INTRATUMOURAL HETEROGENEITY AND SHAPE IMMUNE DYNAMICS IN ADULT-TYPE DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Zaman, W. Muskovic, E. Holton, M. Pandya, R. Zakanovic, V. Chin, E. Lam, J. Yeung, M. Sughrue, C. Teo, J. Powell
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite the crucial role of T cell clones in the anti-tumor activity, their characterization and association with clinical outcome following immune checkpoint inhibitors (ICI) is lacking. Here we analyzed paired single-cell RNA-sequencing/T-cell receptor sequencing of 767,606 T cells from 460 samples spanning 6 cancer types. We found a robust signature of response based on expanded CD8+ clones that differentiates between responders and non-responders. Analysis of persistent clones showed transcriptional changes that are differentially induced by therapy in the different response groups, suggesting an improved reinvigoration capacity in responding patients. Moreover, a gene trajectory analysis revealed changes in the pseudo-temporal state of de-novo clones that are associated with response to therapy. Lastly, we found that clones shared between tumor and blood are more abundant in non-responders and execute distinct transcriptional programs. Overall, our results highlight differences in clonal transcriptional states that are linked to patient response, offering valuable insights into the mechanisms driving effective anti-tumor immunity. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3504c21c4fc5c6f9ee1a5a18b7688458b8f59956" target='_blank'>
              Single-cell meta-analysis of T cells reveals clonal dynamics of response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            Ofir Shorer, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Uterine carcinosarcoma (UCS) is a rare gynecologic cancer with unfavorable prognosis. Its biphasic histologic components are known to associated with outcomes. Dissecting the tumor heterogeneity within UCS is crucial to discover molecular biomarkers for patient stratification. The aim of this study was to utilize spatial profiling technology to analyze immune presentation in different histological components within UCS tumors. The study included 23 UCS patients whose tumor sections were subjected to nanoString GeoMx Digital Spatial Profiling (DSP) to profile 30 immune-related proteins in 491 segments. We identified distinct cluster-labeled subtypes showing varied immune marker expressions. The immune presentations within the carcinoma and sarcoma components showed significant differences. Markers indicating M2 macrophages (CD45+CD163+) and epithelial-mesenchymal transition (EMT) (SMA, fibronectin) were highly expressed in the sarcoma components. The immune presentation subtypes within each histological component also differed between patients with recurrence in 5 years and without recurrence over 5 years. The cluster-labeled subtype with high Neutrophil/PMN-MDSC marker expression (CD66b) was associated with longer survival, whereas the subtype with high NK cell marker expressions (CD56) was linked to poor prognosis. Our result suggested that immune features within the carcinoma component of UCS tumors would determine patient outcomes and should be considered as a strategy for patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/677b6a5b9abbf69e6cc6fe6fd37eee258299da27" target='_blank'>
              Inter- and intra-tumor heterogeneous immune presentation in primary uterine carcinosarcoma determined by digital spatial profiling
              </a>
            </td>
          <td>
            Ying-Xuan Li, Wei-Chou Lin, Ko-Chen Chen, Wen-Chun Chang, Chin-Jui Wu, Lin-Hung Wei, Ruby Yun-Ju Huang, Bor-Ching Sheu
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8531cded1ac1312b87ee916c1c14bac956c911de" target='_blank'>
              Computational immune synapse analysis reveals T-cell interactions in distinct tumor microenvironments
              </a>
            </td>
          <td>
            Victor G Wang, Zichao Liu, Jan Martinek, Ali Foroughi pour, Jie Zhou, Hannah Boruchov, Kelly Ray, Karolina Palucka, Jeffrey H. Chuang
          </td>
          <td>2024-09-28</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Adrenocortical carcinoma (ACC) is a rare endocrine malignancy with a poor prognosis and limited treatment options. Bulk genomic characterization of ACC has not yielded obvious therapeutic or immunotherapeutic targets, yet novel therapies are needed. We hypothesized that elucidating the intratumoral cellular heterogeneity by single nuclei RNA sequencing analyses would yield insights into potential therapeutic vulnerabilities of this disease. In addition to characterizing the immune cell and fibroblast landscape, our analyses of single nuclei gene expression profiles identified an adrenal cortex cell cluster exhibiting a program of replication stress and DNA damage response in primary and metastatic ACC. In vitro assessment of replication stress and DNA damage response using an ACC cell line and a series of newly-derived hormonally active patient-derived tumor organoids revealed ATR sensitivity. These findings provide novel mechanistic insight into ACC biology and suggest that an underlying dependency on ATR may be leveraged therapeutically in advanced ACC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1469f57c44a466002613ce12061fcafe073cc550" target='_blank'>
              Single Nuclei Sequencing Reveals Intratumoral Cellular Heterogeneity and Replication Stress in Adrenocortical Carcinoma
              </a>
            </td>
          <td>
            Liudmila V. Popova, Elizabeth A. R. Garfinkle, Daniel M. Chopyk, Jaye B. Navarro, Adithe Rivaldi, Yaoling Shu, Elena Lomonosova, J. Phay, Barbra S. Miller, Swati Sattuwar, Mary Mullen, Elaine R. Mardis, Katherine E. Miller, Priya H. Dedhia
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Gliomas are aggressive brain tumors associated with a poor prognosis. Cancer stem cells (CSCs) play a significant role in tumor recurrence and resistance to therapy. This study aimed to identify and characterize glioma stem cells (GSCs), analyze their interactions with various cell types, and develop a prognostic signature. Methods Single-cell RNA sequencing data from 44 primary glioma samples were analyzed to identify GSC populations. Spatial transcriptomics and gene regulatory network analyses were performed to investigate GSC localization and transcription factor activity. CellChat analysis was conducted to infer cell-cell communication patterns. A GSC signature (GSCS) was developed using machine learning algorithms applied to bulk RNA sequencing data from multiple cohorts. In vitro and in vivo experiments were conducted to validate the role of TUBA1C, a key gene within the signature. Results A distinct GSC population was identified, characterized by high proliferative potential and an enrichment of E2F1, E2F2, E2F7, and BRCA1 regulons. GSCs exhibited spatial proximity to myeloid-derived suppressor cells (MDSCs). CellChat analysis revealed an active MIF signaling pathway between GSCs and MDSCs. A 26-gene GSCS demonstrated superior performance compared to existing prognostic models. Knockdown of TUBA1C significantly inhibited glioma cell migration, and invasion in vitro, and reduced tumor growth in vivo. Conclusion This study offers a comprehensive characterization of GSCs and their interactions with MDSCs, while presenting a robust GSCS. The findings offer new insights into glioma biology and identify potential therapeutic targets, particularly TUBA1C, aimed at improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c2780868fec467184ab80544d5944cba2384cf" target='_blank'>
              From single-cell to spatial transcriptomics: decoding the glioma stem cell niche and its clinical implications
              </a>
            </td>
          <td>
            Lei Cao, Xu Lu, Xia Wang, Hao Wu, Xiaye Miao
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6f6c8a044b42cc74a45de30bf8885e624a6a702" target='_blank'>
              Single-cell transcriptional atlas of tumor-associated macrophages in breast cancer
              </a>
            </td>
          <td>
            Yupeng Zhang, Fan Zhong, Lei Liu
          </td>
          <td>2024-09-04</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Since the approval of the first checkpoint inhibitor, ipilimumab (α-CTLA4), there continues to be enthusiasm around leveraging the immune system to eradicate solid tumors. However, unknowns about the cellular mechanism(s) that cause some patients to durably respond while others relapse, continue to stymie the field. To support ongoing efforts elucidating cell subsets responsible for differences in patient responses, we aim to evaluate the functional heterogeneity of T-cell subsets responding to checkpoint inhibitors in mouse models of solid tumors. Single-cell RNA-seq has enabled discoveries of transcriptionally defined cell subsets, but understanding their function require inference from transcription, flow-based validation biased by pre-selection of subsets with specific markers, or reliance on surrogate markers for cell-extrinsic functions (e.g., cytotoxicity). Thus, functional heterogeneity could be obscured. To address this, we leveraged a novel technology that converges live, cellular functions with transcriptomics at the resolution of single cell-cell interactions. This technology captures tens of thousands of individual cells or defined combinations of groups of cells in bio-compatible hydrogel compartments, enabling longitudinal imaging over days to weeks. Throughout this period, functions are characterized via bright-field or immunofluorescence readouts, including antibodies, cytokine capture beads, and dyes. Cells are then lysed within compartments to generate barcoded libraries for mRNA-sequencing and gene signatures from individual cells computationally deconvoluted. With this technology, we developed a novel assay that enables longitudinal, unbiased functional characterization of individual T cells mapped to single-cell transcriptomics. Single mouse primary T cells were encapsulated with one or more MC38 target cells and imaged over time. We validated combinations of different readouts can be measured from the same T-cell, including cell-counts for proliferation, CD8/3 expression to differentiate T cells from target cells, activation markers, and / or target cell death via Sytox. Thus, single T cells can be multi-functionally evaluated over time, enabling the study of attributes inaccessible at high scale with prior technologies, including kinetics of killing or the number of target cells killed by a single T-cell. For assays combining cytotoxicity and surface receptors, we generated barcoded libraries from functionally characterized T cells for sequencing and developed an analytical pipeline that deconvolutes mRNA from single T cells mapped to their multi-functional profiles, to identify novel pathways that regulate anti-tumor functions. We will leverage this workflow to functionally characterize single T cells in tumor-bearing mice treated with checkpoint inhibitors and map the data to underlying transcriptomics. In doing so, we aim to discover functional subsets, potentially with identical transcriptional profiles, to support hypothesis generation around mechanisms governing anti-tumor efficacy of checkpoint inhibitors.
 Citation Format: Jacqueline Yee, Praveer Sharma, Yunmin Li, Isaac Thomas, Pier F Gherardini, Teresa Ai, Mostafa Ronaghi, Gary Schroth, Matthew H Spitzer. Novel workflow for characterizing T-cell functional heterogeneity in response to checkpoint inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b89db5bb53ca20d5ac128fa18040b839f5041ca9" target='_blank'>
              Abstract A025: Novel workflow for characterizing T-cell functional heterogeneity in response to checkpoint inhibitors
              </a>
            </td>
          <td>
            Jacqueline Yee, Praveer Sharma, Yunmin Li, Isaac Thomas, Pier F Gherardini, Teresa Ai, Mostafa Ronaghi, Gary Schroth, Matthew H Spitzer
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdce58909f9de06504a3ce2e2bc45ebf1454164f" target='_blank'>
              Spatial single-cell protein landscape reveals vimentinhigh macrophages as immune-suppressive in the microenvironment of hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Xinyao Qiu, Tao Zhou, Shuai Li, Jianmin Wu, Jing Tang, Guosheng Ma, Shuai Yang, Ji Hu, Kaiting Wang, Siyun Shen, Hongyang Wang, Lei Chen
          </td>
          <td>2024-09-26</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The development and progression of diseases in multicellular organisms unfold within the intricate three-dimensional body environment. Thus, to comprehensively understand the molecular mechanisms governing individual development and disease progression, precise acquisition of biological data, including genome, transcriptome, proteome, metabolome, and epigenome, with single-cell resolution and spatial information within the body’s three-dimensional context, is essential. This foundational information serves as the basis for deciphering cellular and molecular mechanisms. Although single-cell multi-omics technology can provide biological information such as genome, transcriptome, proteome, metabolome, and epigenome with single-cell resolution, the sample preparation process leads to the loss of spatial information. Spatial multi-omics technology, however, facilitates the characterization of biological data, such as genome, transcriptome, proteome, metabolome, and epigenome in tissue samples, while retaining their spatial context. Consequently, these techniques significantly enhance our understanding of individual development and disease pathology. Currently, spatial multi-omics technology has played a vital role in elucidating various processes in tumor biology, including tumor occurrence, development, and metastasis, particularly in the realms of tumor immunity and the heterogeneity of the tumor microenvironment. Therefore, this article provides a comprehensive overview of spatial transcriptomics, spatial proteomics, and spatial metabolomics-related technologies and their application in research concerning esophageal cancer, gastric cancer, and colorectal cancer. The objective is to foster the research and implementation of spatial multi-omics technology in digestive tumor diseases. This review will provide new technical insights for molecular biology researchers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77672e163c08fe2553e07120e8f19a354c4e1b31" target='_blank'>
              The burgeoning spatial multi-omics in human gastrointestinal cancers
              </a>
            </td>
          <td>
            Weizheng Liang, Zhenpeng Zhu, Dandan Xu, Peng Wang, Fei Guo, Haoshan Xiao, Chenyang Hou, Jun Xue, Xuejun Zhi, Rensen Ran
          </td>
          <td>2024-09-13</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Many aspects of breast cancer metastasis remain poorly understood, despite its clinical importance. In this issue of the JCI, Winkler et al. have applied an elegant patient-derived xenograft (PDX) model to map the transcriptomes of single cells in matched primary tumors and lung metastases across 13 breast cancer PDX models. They identified distinct transcriptional changes associated with metastatic evolution in lowly and highly metastatic primary tumors. Furthermore, by classifying the “epithelial-mesenchymal plasticity” (EMP) state of single cells, they revealed that considerable EMP heterogeneity exists among primary and metastatic human breast cancer cells. However, the EMP profile of a tumor does not change substantially upon metastasis. These findings give an unprecedentedly detailed view into the transcriptional heterogeneity and evolution of metastatic human breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54971b7e64340d8e740cdf1de341d57f2e78ee5a" target='_blank'>
              Mapping the transcriptional evolution of human metastatic breast cancer
              </a>
            </td>
          <td>
            Melissa Q. Reeves
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/134c6a39946a95594365892da7bfa505a4ad41ab" target='_blank'>
              The prognostic effect of infiltrating immune cells is shaped by proximal M2 macrophages in lung adenocarcinoma
              </a>
            </td>
          <td>
            Jian-Rong Li, Vikram R Shaw, Yupei Lin, Xiang Wang, M. Aminu, Yong Li, Jia Wu, Jianjun Zhang, Christopher I Amos, Chao Cheng
          </td>
          <td>2024-09-04</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ab5a7fdae99ad93f99247468e27d3a8b9633164" target='_blank'>
              Spatiotemporal modeling reveals high-resolution invasion states in glioblastoma
              </a>
            </td>
          <td>
            Varsha Thoppey Manoharan, Aly O. Abdelkareem, Gurveer Gill, Samuel Brown, Aaron H. Gillmor, Courtney Hall, Heewon Seo, Kiran Narta, Sean Grewal, Ngoc Ha Dang, Bo Young Ahn, Kata Osz, X. Lun, Laura Mah, Franz Zemp, Douglas J Mahoney, Donna L Senger, Jennifer A. Chan, A. Morrissy
          </td>
          <td>2024-10-10</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) generally acquire immunosuppressive and tumor-promoting phenotypes, which may contribute to tumor resistance to immunotherapy. We previously showed that suppression of microRNA activity through genetic Dicer1 inactivation rewires TAM’s transcriptomes and prompts their immunostimulatory activation. This phenotypic switch enhanced recruitment and activation of CD8+ cytotoxic T cells (CTLs) and improved the efficacy of immunotherapy in mouse cancer models. Here, we performed single-cell RNA sequencing of whole tumors grown in either wild-type mice or mice with macrophage-specific Dicer1 deletion. The analysis of multiple cell populations, including several discrete monocyte and macrophage subsets, indicated broad and convergent immunostimulatory programming of the tumor microenvironment, which was dependent on CTL-derived interferon-gamma (IFNγ), in mice with DICER-deficient macrophages. Intriguingly, dynamic inferences on monocyte/macrophage ontogeny and differentiation by pseudotime analysis revealed trajectories associated with progression into cell cycle, monocyte-to-macrophage differentiation, and transition from an immunostimulatory to an immunosuppressive phenotype in tumors with DICER-proficient macrophages. Dicer1 deficiency interfered with this trajectory and stalled TAMs at an intermediate state between immature monocytes and macrophages with T cell-stimulatory capacity, thereby impeding immunosuppressive TAM development. This translated into enhanced response to antiangiogenic immunotherapy in an immunotherapy-resistant model of non-small cell lung cancer. Cycling/M2-like macrophages were conserved in human melanoma and hepatocellular carcinoma and should represent a more promising therapeutic target than the bulk of TAMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37984a76dd2d17ad083124c8404831278705b7c5" target='_blank'>
              The immune landscape of tumor-associated macrophage reprogramming
              </a>
            </td>
          <td>
            Florent Duval, Joao Lourenco, Mehdi Hicham, G. Boivin, Alan Guichard, Celine Wyser-Rmili, Nadine Fournier, Michele De Palma, Nahal Mansouri
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Cancer-associated fibroblast (CAF) composition within the same organ varies across different cancer subtypes. Distinct CAF subtypes exhibit unique features due to interactions with immune cells and the tumor microenvironment. However, data on CAF subtypes in individuals with vestibular schwannoma (VS) are lacking. Therefore, we aimed to distinguish CAF subtypes at the single-cell level, investigate how stem-like CAF characteristics influence the tumor immune microenvironment, and identify CAF subtype-specific metabolic reprogramming pathways that contribute to tumor development. Data were analyzed from three patients with VS, encompassing 33,081 single cells, one bulk transcriptome cohort, and The Cancer Genome Atlas Pan-Cancer database (RNA sequencing and clinical data). Our findings revealed that antigen-presenting CAFs are linked to substantially heightened immune activity, supported by metabolic reprogramming, which differs from tumorigenesis. High expression of the stem-like CAF gene signature correlated with poor prognosis in low-grade gliomas within the pan-cancer database. This is the first study to classify CAF subtypes in VS patients and identify a therapeutic vulnerability biomarker by developing a stem-like CAF gene signature. Personalized treatments tailored to individual patients show promise in advancing precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd8e5819e3704f5476a3b3da2ab4be193ceaa21" target='_blank'>
              Cancer-Associated Fibroblast Subtypes Reveal Distinct Gene Signatures in the Tumor Immune Microenvironment of Vestibular Schwannoma
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jung Woo Lee
          </td>
          <td>2024-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="In the tumor microenvironment of glioblastoma, myeloid cells act as a double-edged sword—they are a major cellular component modulating the immune response while presenting a potential therapeutic target. Our study highlights the voltage-gated proton channel Hv1, predominantly expressed in myeloid cells, as a crucial regulator of their physiological functions. We discovered a strong correlation between increased Hv1 expression and poor prognosis in glioblastoma patients. Depleting Hv1 significantly extended survival in a mouse model of glioma. Employing multiple novel transgenic mouse lines, we demonstrated that Hv1 is upregulated in response to tumor presence, with glioma-associated macrophages as the principal contributors. Specifically, we identified that Hv1 in infiltrating macrophages as the major driver of survival phenotype differences. Through a combination of in vivo two-photon imaging, spectral flow cytometry, and spatial transcriptomic sequencing, we found that Hv1 depletion leads to reduced macrophage infiltration and enhanced antigen presentation, ultimately fostering a stronger adaptive immune response. These findings establish the Hv1 channel as a crucial new immune regulator within the brain tumor milieu, offering a promising target for reprogramming macrophage function to combat glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ec62c1fa4b300aaed4eedd2b71b6f093ab4c831" target='_blank'>
              Deficiency of Hv1 Proton Channel Enhances Macrophage Antigen Presentation and Anti-Tumor Responses in Glioma
              </a>
            </td>
          <td>
            Jiaying Zheng, Lingxiao Wang, Shunyi Zhao, K. Ayasoufi, E. Goddery, P. Strege, Yushan Wan, Wenjing Zhang, Manling Xie, Arthur Beyder, , Long-Jun Wu
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Innovative identification technologies for hematopoietic stem cells (HSCs) have advanced the frontiers of stem cell biology. However, most analytical techniques capture only a single snapshot, disregarding the temporal context. A comprehensive understanding of the temporal heterogeneity of HSCs necessitates live-cell, real-time and non-invasive analysis. Here, we developed a prediction system for HSC diversity by integrating single-HSC ex vivo expansion technology with quantitative phase imaging (QPI)-driven machine learning. By analyzing single-cell kinetics with QPI, we discovered previously undetectable diversity among HSCs that snapshot analysis fails to capture. Our QPI-driven algorithm quantitatively evaluates the stemness of individual HSCs and incorporates temporal information to significantly improve prediction accuracy. This platform marks a paradigm shift from “identification” to “prediction”, enabling us to forecast HSC status by analyzing their past temporal kinetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41d59e33033757dcb2fbbe006d53b555326dd5cf" target='_blank'>
              Quantitative phase imaging with temporal kinetics predicts hematopoietic stem cell diversity
              </a>
            </td>
          <td>
            Takao Yogo, Yuichiro Iwamoto, Hans Jiro Becker, Takaharu Kimura, Ayano Sugiyama-Finnis, Tomomasa Yokomizo, Toshio Suda, Sadao Ota, Satoshi Yamazaki
          </td>
          <td>2024-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/890b64940fb0066cdb995ca79773b85be943b6fd" target='_blank'>
              Integrated multiomics revealed adenosine signaling predict immunotherapy response and regulate tumor ecosystem of melanoma
              </a>
            </td>
          <td>
            Yantao Xu, Poyee Lau, Xiang Chen, Shuang Zhao, Yilang He, Zixi Jiang, Xiang Chen, Guanxiong Zhang, Hong Liu
          </td>
          <td>2024-09-15</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="(1) Background: This study investigates the tumor immune microenvironment, focusing on immune cell distribution in lung adenocarcinoma. (2) Methods: We evaluated fifty cases of lung adenocarcinoma, and suitable areas for further studies were annotated on the histological slides. Two tumor cores per case were obtained, one from the tumor’s center and another from its periphery, and introduced into three paraffin receptor blocks for optimized processing efficiency. The 4-micrometer-thick tissue microarray sections were stained for H&E and for CD68, CD163, CD8, CD4, and PD-L1; (3) Results: Our investigation revealed significant correlations between PD-L1 expression in tumor cells and the presence of CD163+ macrophages, between CD4+ cells and CD8+, CD68+, and CD163+ cells, and also between CD8+ T cells and CD163+ cells. Additionally, while we observed some differences in cellular components and densities between the tumor center and periphery, these differences were not statistically significant. However, distinct correlations between PD-L1 and immune cells in these regions were identified, suggesting spatial heterogeneity in the immune landscape. (4) Conclusions: These results emphasize the intricate interactions between immune cells and tumor cells in lung adenocarcinoma. Understanding patient spatial immune profile could improve patient selection for immunotherapy, ensuring that those most likely to benefit are identified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebf90ba602f501858b8c8085e2f07447aa272fce" target='_blank'>
              Analyzing the Spatial Distribution of Immune Cells in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Florina Almarii, María Sajin, George Simion, Simona O. Dima, Vlad Herlea
          </td>
          <td>2024-08-30</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d593a4bab5e1bd9b8d5a096791ffe052f667c34" target='_blank'>
              Computational identification of surface markers for isolating distinct subpopulations from heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Andrea L. Gardner, T. Jost, Daylin Morgan, Amy Brock
          </td>
          <td>2024-10-17</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Several methods for cell cycle inference from sequencing data exist and are widely adopted. In contrast, methods for classification of cell cycle state from imaging data are scarce. We have for the first time integrated sequencing and imaging derived cell cycle pseudo-times for assigning 449 imaged cells to 693 sequenced cells at an average resolution of 3.4 and 2.4 cells for sequencing and imaging data respectively. Data integration revealed thousands of pathways and organelle features that are correlated with each other, including several previously known interactions and novel associations. The ability to assign the transcriptome state of a profiled cell to its closest living relative, which is still actively growing and expanding opens the door for genotype-phenotype mapping at single cell resolution forward in time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73340e559ff9e02cb63ea7b7366f2d2d5be7e9ec" target='_blank'>
              Cell identity revealed by precise cell cycle state mapping links data modalities
              </a>
            </td>
          <td>
            Saeed Alahmari, Andrew Schultz, Jordan Albrecht, Vural Tagal, Zaid Siddiqui, Sandhya Prabhakaran, I. El Naqa, Alexander Anderson, Laura M. Heiser, Noemi Andor
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Esophageal cancer comprises two main subtypes: esophageal squamous cell carcinoma (ESCC) and esophageal adenocarcinoma (EAC). Previous studies have revealed distinct genomic characteristics between these subtypes, with ESCC sharing similarities with head and neck squamous cell carcinoma (HNSCC), and EAC aligning with gastric adenocarcinoma (GAC). Additionally, recent immunotherapy clinical trials have shown a higher response rate in ESCC and HNSCC compared to EAC and GAC. However, how the tumor microenvironment contributes to these varied immunotherapy responses remains unclear. In this study, through a comparative analysis of single-cell tumor transcriptomes from 35 patients with ESCC, EAC, HNSCC, or GAC, we identified two groups with tumor microenvironment distinctions: ESCC and HNSCC versus EAC and GAC, consistent with their genomic classifications. Malignant epithelial cells displayed distinct separations based on histological origin. In the tumor immune microenvironment, we observed an enrichment of CXCL13+CD8+ T cells and CXCL9+CXCL10+ tumor-associated macrophages (TAMs) in ESCC and HNSCC, which activate cellular immunity through interferon-γ. In contrast, EAC and GAC exhibited a high presence of heat-shock protein-expressing CD8+ T cells and MARCO+ TAMs. These immune signatures help explain the varied immunotherapy responses among these cancer subtypes and successfully predict immunotherapy outcomes across diverse cancer types, underscoring their clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251c1e9cc81c4dd68a06455379c16a74217063d7" target='_blank'>
              Tumor microenvironment distinctions between esophageal cancer subtypes explain varied immunotherapy responses
              </a>
            </td>
          <td>
            S. Baek, Junha Cha, M. Hong, Gamin Kim, Yoon Woo Koh, Dahee Kim, M. Hemberg, Seong Yong Park, Hye Ryun Kim, Insuk Lee
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162a418f3caf8634ccba24bf052ce0feed7db43f" target='_blank'>
              Unraveling molecular characteristics and tumor microenvironment dynamics of neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            David Heimdörfer, N. Artamonova, Zoran Culig, Isabel Heidegger
          </td>
          <td>2024-10-16</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eadaab3d293ff876f51403f3620a7ad83b9f29" target='_blank'>
              Spatial transcriptomic analysis of primary and metastatic pancreatic cancers highlights tumor microenvironmental heterogeneity.
              </a>
            </td>
          <td>
            A. Khaliq, Meenakshi Rajamohan, Omer Saeed, Kimia Mansouri, Asif Adil, Chi Zhang, Anita Turk, Julienne L. Carstens, Michael House, Sikander Hayat, G. Nagaraju, Sam G Pappas, Y. A. Wang, N. Zyromski, Mateusz Opyrchal, Kelvin P Lee, Heather M. O’Hagan, Bassel El Rayes, Ashiq Masood
          </td>
          <td>2024-09-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc9bc6fb64c943c109f90f687d279d53c1cce3e" target='_blank'>
              Integrating microarray-based spatial transcriptomics and RNA-seq reveals tissue architecture in colorectal cancer
              </a>
            </td>
          <td>
            Zheng Li, Xiao-Jie Zhang, Chongyuan Sun, Zefeng Li, He Fei, Dongbing Zhao
          </td>
          <td>2024-09-17</td>
          <td>J. Big Data</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Today, the intratumoral composition is a relevant factor associated with the progression and aggression of cancer. Although it suggests a metabolic interdependence among the subpopulations inside the tumor, a detailed map of how this interdependence contributes to the malignant phenotype is still lacking. To address this issue, we developed a systems biology approach integrating single-cell RNASeq and genome-scale metabolic reconstruction to map the metabolic cross-feeding among the subpopulations previously identified in the spheroids of MCF7 breast cancer. By calibrating our model with expression profiles and the experimental growth rate, we concluded that the reverse Warburg effect emerges as a mechanism to optimize community growth. Furthermore, through an in silico analysis, we identified lactate, alpha-ketoglutarate, and some amino acids as key metabolites whose disponibility alters the growth rate of the spheroid. Altogether, this work provides a strategy for assessing how space and intratumoral heterogeneity influence the metabolic robustness of cancer, issues suggesting that computational strategies should move toward the design of optimized treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c54b7152ffb9e525c998dbbf7f6c9c4f3769b3" target='_blank'>
              Intratumoral Heterogeneity and Metabolic Cross-Feeding in a Three-Dimensional Breast Cancer Culture: An In Silico Perspective
              </a>
            </td>
          <td>
            Jorge E. Arellano-Villavicencio, Aarón Vázquez-Jiménez, J. J. Oropeza-Valdez, Cristian Padron-Manrique, H. Prado-García, Armando R. Tovar, O. Reséndis-Antonio
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genetic perturbation of T cell receptor (TCR) T cells is a promising method to un-lock better TCR T cell performance to create more powerful cancer immunotherapies, but understanding the changes to T cell behavior induced by genetic perturbations remains a challenge. Prior studies have evaluated the effect of different genetic modifications with cytokine production and metabolic activity assays. Live-cell imaging is an inexpensive and robust approach to capture TCR T cell responses to cancer. Most methods to quantify T cell responses in live-cell imaging data use simple approaches to count T cells and cancer cells across time, effectively quantifying how much space in the 2D well each cell type covers, leaving actionable information unexplored. In this study, we characterize changes in TCR T cell’s interactions with cancer cells from live-cell imaging data using explainable artificial intelligence (AI). We train convolutional neural networks to distinguish behaviors in TCR T cell with CRISPR knock outs of CUL5, RASA2, and a safe harbor control knockout. We use explainable AI to identify specific interaction types that define different knock-out conditions. We find that T cell and cancer cell coverage is a strong marker of TCR T cell modification when comparing similar experimental time points, but differences in cell aggregation characterize CUL5KO and RASA2KO behavior across all time points. Our pipeline for discovery in live-cell imaging data can be used for characterizing complex behaviors in arbitrary live-cell imaging datasets, and we describe best practices for this goal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aebfd6c42a5e597f88022e8f6d3b947f59ad831" target='_blank'>
              Understanding TCR T cell knockout behavior using interpretable machine learning
              </a>
            </td>
          <td>
            Marcus Blennemann, Archit Verma, Stefanie Bachl, J. Carnevale, Barbara E. Engelhardt
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Somatic mutations inactivating TET2 are among the most common drivers of clonal hematopoiesis (CH). While TET2 inactivation is associated with monocyte-derived inflammation and improved chimeric antigen-receptor-T cell function, its impact on immunotherapy response is unknown. In our mouse model, hematopoietic Tet2 mutation enhanced immune checkpoint blockade (ICB) response. Enhanced ICB response with Tet2 mutation required phagocytes, CD4 and CD8 T cells. Mechanistically, in Tet2-mutant tumor-infiltrating leukocytes (TILs), ICB preferentially induced anti-tumor states and restricted cell states linked to tumor progression. Tet2-mutant monocytes activated costimulatory programs, while Tet2-mutant T cells showed enhanced T cell memory signatures, lesser exhaustion and decreased regulatory phenotype. Our murine data was clinically relevant, since we found that melanomas from patients with TET2 driver mutation-CH (TET2-CH) showed enhanced immune infiltration, T cell activation, and T cell memory programs. In melanoma patients treated with ICB, TET2-CH was associated with 6-fold greater odds of clinical benefit. Collectively, our data establishes that hematopoietic Tet2 inactivation primes leukocytes for anti-tumor states associated with immunotherapy response and provides a potential biomarker for personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3e6160e702c2224ea8ff0222f9be539e313873" target='_blank'>
              Hematopoietic Tet2 inactivation enhances the response to checkpoint blockade immunotherapy
              </a>
            </td>
          <td>
            R. Vanner, Suraj Bansal, Marco M. Buttigeig, Andy G. X. Zeng, Vincent Rondeau, Darryl Y. Chan, Michelle A. Chan-Seng-Yue, Liqing Jin, Jessica Mcleod, E. Donato, Patrick Stelmach, C. Vlasschaert, Yitong Yang, Aarushi Gupta, Sofia Genta, Enrique Sanz Garcia, LI Shlush, Mauricio Ribeiro, Marcus O Butler, Sagi Abelson, Mark D. Minden, Steven M. Chan, Michael J. Rauh, A. Trumpp, John E. Dick
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Cell morphologies represent a phenotypic integration of intrinsic cellular processes, changes in cell state, and interactions with neighboring cells. While analyses of large cell line collections such as the Cancer Dependency Map (DepMap) have greatly enhanced discovery of novel molecular biomarkers and therapeutic targets, morphological analysis of these models has not yet been performed. Hence, there is a rich opportunity to discover novel regulators and functional properties of cell morphologies by linking image-based profiling to -omics based assays. Here, we studied the transcriptional and functional properties of 16 human pancreatic cancer cell lines, associations with distinct cell morphologies, and treatment-induced remodeling. At baseline, we observed extensive morphological heterogeneity across and within cell lines, as well as differences in multicellular organization. We categorized cell line morphologies into epithelial, mesenchymal, and spheroid, and organizational patterns into tightly aggregated, multilayered, and dispersed. We performed differential expression analysis across morphological subtypes utilizing DepMap transcriptomics data and queried functional correlates including CRISPR dependency, drug sensitivity, and proclivity for metastasis. For example, we discovered that the mesenchymal morphology and multilayered organizational pattern [WH3] were associated with metastatic potential, while expressing reduced levels of tight junction and cell adhesion genes. We then explored treatment associated morphological changes and their potential as cost-effective biomarkers for cell-intrinsic resistance to chemotherapy (e.g., 5-FU and gemcitabine) and KRAS inhibitors (e.g., MRTX1133 and RMC- 6236). Cells were treated continuously and monitored with live cell imaging (Incucyte SX5) followed by Cell Painting at the three-day endpoint. We observed conserved morphological responses to therapy across cell lines including a shift towards spindle-like cell shapes with neurite-like projections in chemotherapy-treated conditions, and a cell bloating phenomena characterized by an increase in cell area in KRAS inhibitor treated conditions. To directly link transcriptional state to morphology, we performed 1000-plex spatial molecular imaging (Nanostring CosMx) at subcellular resolution to pools of cell lines, demonstrating that morphological diversity within cell lines corresponded to distinct transcriptional states. In conclusion, this study highlights the potential of harnessing cell morphological information in a rapid, cost-effective phenotyping assay to aid precision oncology efforts leveraging patient- derived in vitro models.
 Citation Format: Dennis Gong, William L Hwang. Morphological diversity predicts functional traits in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B076.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ee8c48f6e344128f81cda79e9fefc7cbd605a4f" target='_blank'>
              Abstract B076: Morphological diversity predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, W. L. Hwang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f337210da04933a9ae37820fdc6d7b7fe43d5d5" target='_blank'>
              CALB2 drives pancreatic cancer metastasis through inflammatory reprogramming of the tumor microenvironment
              </a>
            </td>
          <td>
            Jinxin Tao, Yani Gu, Zeyu Zhang, Guihu Weng, Yueze Liu, Jie Ren, Yanan Shi, J. Qiu, Yuanyang Wang, Dan Su, Ruobing Wang, Yifan Fu, Tao Liu, Liyuan Ye, W. Luo, Hao Chen, Gang Yang, Zhe Cao, Hua Huang, Jianchun Xiao, Bo Ren, Lei You, Taiping Zhang, Yupei Zhao
          </td>
          <td>2024-10-03</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Acquiring detailed spatial information about intratumor microbiota in situ is challenging, which leaves 3D distributions of microbiota within entire tumors largely unexplored. Here, a modified iDISCO-CUBIC tissue clearing and D-amino acid microbiome labeling-based (miCDaL) strategy are proposed, that integrates microbiota in situ labeling, tissue clearing, and whole-mount tissue imaging to enable 3D visualization of indigenous intratumor microbiota. Leveraging whole-mount spatial resolution and centimeter-scale imaging depth, the 3D biogeography of microbiota is successfully charted across various tumors at different developmental stages, providing quantitative spatial insights in relation to host tumors. By incorporating an immunostaining protocol, 3D imaging of the immunologic microenvironment is achieved in both murine and human mammary tumors that is previously assumed to be bacteria-free. Notably, immune infiltrates, including T cells and NK cells, and tertiary lymphoid structures are conspicuously absent in bacteria-colonized regions. This 3D imaging strategy for mapping Indigenous intratumor microbiota offers valuable insights into host-microbiota interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6fdb761437877e43564b6889311f1059c2228cb" target='_blank'>
              Whole-Tumor Clearing and Imaging of Intratumor Microbiota in Three Dimensions with miCDaL Strategy.
              </a>
            </td>
          <td>
            Yuezhou Wang, Zile Jiang, Kai Zhang, Huimin Tang, Guimei Wang, Jinshan Gao, Guanghui He, Baoyue Liang, Li Li, Chaoyong Yang, Xianming Deng
          </td>
          <td>2024-10-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a subpopulation of cancer cells that are believed to initiate and drive cancer progression. In animal models, xenotransplanted CSCs have demonstrated the ability to produce tumors. Since their initial isolation in blood cancers, CSCs have been identified in various solid human cancers, including oral squamous cell carcinoma (OSCC). In addition to their tumorigenic properties, dysregulated stem-cell-related signaling pathways—Wnt family member (Wnt), neurogenic locus notch homolog protein (Notch), and hedgehog—have been shown to endow CSCs with characteristics like self-renewal, phenotypic plasticity, and chemoresistance, contributing to recurrence and treatment failure. Consequently, CSCs have become targets for new therapeutic agents, with some currently in different phases of clinical trials. Notably, small molecule inhibitors of the hedgehog signaling pathway, such as vismodegib and glasdegib, have been approved for the treatment of basal cell carcinoma and acute myeloid leukemia, respectively. Other strategies for eradicating CSCs include natural compounds, nano-drug delivery systems, targeting mitochondria and the CSC microenvironment, autophagy, hyperthermia, and immunotherapy. Despite the extensive documentation of CSCs in OSCC since its first demonstration in head and neck (HN) SCC in 2007, none of these novel pharmacological approaches have yet entered clinical trials for OSCC patients. This narrative review summarizes the in vivo and in vitro evidence of CSCs and CSC-related signaling pathways in OSCC, highlighting their role in promoting chemoresistance and immunotherapy resistance. Additionally, it addresses methodological challenges and discusses future research directions to improve experimental systems and advance CSC studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7383ff1c5890f459470549a8d4649249c258e16c" target='_blank'>
              Cancer Stem Cells in Oral Squamous Cell Carcinoma: A Narrative Review on Experimental Characteristics and Methodological Challenges
              </a>
            </td>
          <td>
            Surendra Kumar Acharya, Saptarsi Shai, Yee Fan Choon, I. Gunardi, F. Hartanto, Kathreena Kadir, A. Roychoudhury, R. Amtha, V. K. Vincent-Chong
          </td>
          <td>2024-09-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is one of the most lethal cancers, with a 5-year survival rate of only 13%. PDAC develops from precursor lesions, most commonly Pancreatic Intraepithelial Neoplasia (PanIN). Studying PanIN in human patients is difficult due to its microscopic nature and the lack of indication for sampling the pancreas in the absence of disease. In collaboration with Gift of Life - Michigan, a local organ and tissue donor organization, we obtained over 80 pancreata for research. Surprisingly, we found PanIN in the vast majority of the organs, regardless of the donor’s age; the prevalence of PanIN compared to the relatively low incidence of PDAC indicates that most lesions are destined not to progress. To gather insight into factors that might distinguish lesions designed to progress from lesions that are likely to remain dormant, we sought to characterize both epithelial cells and their supporting microenvironment. Using spatial transcriptomics, we determined that PanIN epithelial cells are transcriptionaally similar to pancreatic cancer; further, limited genetic characterization revealed that they contain similar oncogenic KRAS mutations as pancreatic malignancy. We then sought to determine whether factors in the microenvironment distinguish sporadic PanIN from those associated with overt malignancy. We used 10x Genomic CytAssist Visium technology and profiled an initial batch of 7 tissue sections from donor pancreata and an equal number from tumors. We employed the BayesPrism algorithm to deconvolute the Visium data (which lacked single-cell resolution), using our single-cell RNAseq atlas to define stroma- and epithelial-specific expression profiles. We then segmented the tissue into epithelial and stromal compartments. We further identified various stromal subpopulations, with macrophages and fibroblasts being the most abundant cell types. We defined subpopulations of epithelial cells, fibroblasts, and macrophages through unsupervised clustering. In both donor samples and tumors, we identified heterogeneous populations of stromal cells present in specific geographic patterns, with PanIN-adjacent stroma clearly distinct from stroma surrounding healthy ducts. Cell-cell neighborhood analysis revealed a distinct geographic arrangement of epithelial and stromal compartments, indicating potential communication patterns. Independent unsupervised statistical analysis using Non-negative Matrix Factorization (NMF) using Coordinated Gene Association in Pattern Sets (CoGAPS) R package also revealed similar patterns. Our next steps involve validating gene expression signatures that define each subpopulation and identifying tumor-specific or PanIN-specific genes. Ultimately, we aim to investigate interactions between epithelial cells, fibroblasts, and macrophages to identify potential tumor-promoting or tumor-restraining pathways.
 Citation Format: Ahmed M. Elhossiny, Jude Okoye, Imade Williams, Yaqing Zhang, Hannah Watkoske, Atul Deshpande, Elana Fertig, Arvind Rao, Eileen S. Carpenter, Timothy L. Frankel, Marina Pasca di Magliano. Spatial transcriptomics reveals heterogeneity of epithelial and stromal compartments in healthy and tumor pancreas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A046.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f587649d06619c2f34a308f750ab4e8707c4ec42" target='_blank'>
              Abstract A046: Spatial transcriptomics reveals heterogeneity of epithelial and stromal compartments in healthy and tumor pancreas
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, Jude Okoye, Imade Williams, Yaqing Zhang, Hannah R Watkoske, Atul Deshpande, E. Fertig, Arvind Rao, Eileen S Carpenter, Timothy L. Frankel, M. Pasca di Magliano
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is a highly aggressive tumor, and despite standard-of-care therapy, tumor recurrence is inevitable. Recurrent glioblastoma (rGBM) remains a challenge to treat, given that it predominantly forms within a highly inflammatory tumor microenvironment (TME) that contributes to treatment resistance. However, the complex immune landscape of rGBM and how to effectively characterize it for therapeutic targeting remains poorly understood. To address this gap, this review exhaustively examines the existing body of literature on the immune characteristics of rGBM with a focus on the role of glioma-associated microglia and macrophages (GAMMs). We examine this inflammatory landscape through the lens of histological-based assessments given histological staining remains the gold standard of diagnostic identification of rGBM. We review canonical and emerging cell-specific markers of GAMMs and provide a guide of how these histological-based assessments can aid in characterizing the TME of rGBM and dissect how this landscape shifts in the context of treatment response. Abstract The glioblastoma (GBM) tumor microenvironment consists of a heterogeneous mixture of neoplastic and non-neoplastic cells, including immune cells. Tumor recurrence following standard-of-care therapy results in a rich landscape of inflammatory cells throughout the glioma-infiltrated cortex. Immune cells consisting of glioma-associated macrophages and microglia (GAMMs) overwhelmingly constitute the bulk of the recurrent glioblastoma (rGBM) microenvironment, in comparison to the highly cellular and proliferative tumor microenvironment characteristic of primary GBM. These immune cells dynamically interact within the tumor microenvironment and can contribute to disease progression and therapy resistance while also providing novel targets for emerging immunotherapies. Within these varying contexts, histological-based assessments of immune cells in rGBM, including immunohistochemistry (IHC) and immunofluorescence (IF), offer a critical way to visualize and examine the inflammatory landscape. Here, we exhaustively review the available body of literature on the inflammatory landscape in rGBM as identified through histological-based assessments. We highlight the heterogeneity of immune cells throughout the glioma-infiltrated cortex with a focus on microglia and macrophages, drawing insights from canonical and novel immune-cell histological markers to estimate cell phenotypes and function. Lastly, we discuss opportunities for immunomodulatory treatments aiming to harness the inflammatory landscape in rGBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4001488ad8c82d2b66d1744897ff8a71e5f6d40" target='_blank'>
              Unveiling the Inflammatory Landscape of Recurrent Glioblastoma through Histological-Based Assessments
              </a>
            </td>
          <td>
            Nicholas B. Dadario, D. Boyett, Damian E Teasley, Peter J. Chabot, Nathan J. Winans, Michael G. Argenziano, Colin P. Sperring, P. Canoll, Jeffrey N Bruce
          </td>
          <td>2024-09-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aab9291d16699439a2d6b53e165a3605cd9a24fb" target='_blank'>
              Unraveling the spatial organization and development of human thymocytes through integration of spatial transcriptomics and single-cell multi-omics profiling
              </a>
            </td>
          <td>
            Yanchuan Li, Huamei Li, Cheng Peng, Ge Meng, Yijun Lu, Honglin Liu, Li Cui, Huan Zhou, Zhu Xu, Lingyun Sun, Lihong Liu, Qing Xiong, Beicheng Sun, Shiping Jiao
          </td>
          <td>2024-09-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract The intricate origins, subsets, and characteristics of TCR (T Cell Receptor) s, along with the mechanisms underpinning the antitumor response of tumor-infiltrating T lymphocytes within the tumor microenvironment (TME) remain enigmatic. Recently, the advent of single-cell RNA+TCR-sequencing (scRNA+TCR seq) has revolutionized TME analysis, providing unprecedented insight into the origins, cell subsets, TCR CDR3 compositions, and the expression patterns of response/depletion factors within individual tumor-infiltrating T lymphocytes. Our analysis of the shared scRNA+TCR seq dataset revealed a substantial presence of dual TCR T cells, characterized by clonal hyperplasia and remarkable migratory prowess across various tissues, including blood, normal, peritumoral, and tumor tissues in non-small cell lung cancer patients. Notably, dual TCR CD8+T cells predominantly fell within the CXCL13+subset, displaying potent antitumor activity and a strong preference for tumor tissue residency. Conversely, dual TCR CD4+T cells were predominantly classified as CD5+ or LMNA+subsets, exhibiting a more even distribution across diverse tissue types. By harnessing scRNA+TCR seq and other cutting-edge technologies, we can delve deeper into the effects and mechanisms that regulate the antitumor response or tolerance of dual TCR T cells. This innovative approach holds immense promise in offering fresh perspectives and avenues for advancing research on TIL (Tumor infiltrating lymphocyte)s within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fa169a05a649ecec0b9e24325f2069cfcccb771" target='_blank'>
              scRNA+ TCR-seq revealed dual TCR T cells antitumor response in the TME of NSCLC
              </a>
            </td>
          <td>
            Qi Peng, Yuanyuan Xu, Xinsheng Yao
          </td>
          <td>2024-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Background: Hepatoblastoma (HB) and hepatocellular carcinoma (HCC) are the most common malignant hepatocellular tumors seen in children. Patients with vascular invasion and metastasis generally have poor outcomes. The aim of this study was to delineate the unique cellular changes that drive dissemination by performing comprehensive molecular and phenotypic profiling of samples representing disseminating cells, including primary circulating tumor cells (CTCs) from patients and orthotopic patient-derived xenograt (PDX) mouse models and cell lines grown in vitro in the presence of other cells that represent the tumor microenvironment (TME). Methods: With primary patient whole blood samples, we used a RosetteSep CD45 Depletion Cocktail (Stem Cell Technologies) to enrich CTCs. We validated that these cells were CTCs by staining for a panel of validated markers, including indocyanine green (ICG), Glypican-3 (GPC3), and DAPI. We then performed single cell RNA sequencing (scRNA-seq, 10x Genomics) to comprehensively analyze gene expression of these cells. To understand tumor dissemination mechanisms influenced by the TME, we grew HepT1 HB cells in the presence of supernatant from human umbilical vein endothelial cells (HUVECs). Specifically, HUVEC and HepT1 cells were cultured in separate plates in a combination of endothelial cell medium and minimal essential medium in a 1:1 ratio. The supernatant from each plate was then added to HepT1 cells, and we examined resulting transcriptomic changes with bulk RNA-seq and phenotypic alterations with proliferation (CCK8), scratch (Incucyte), and invasion (Boyden chamber) assays. We also worked to develop a pipeline to establish stable cell lines with primary patient- and murine-derived CTCs. Results: Using our ICG/GPC3/DAPI panel, we showed that the cells isolated from patient whole blood samples were CTCs. scRNA-seq analyses of these samples revealed that there was upregulation of key pathways in CTCs, including NRF2 activity, compared to low-risk primary HB tumors. Factors secreted by HUVECs influenced HepT1 cell phenotypes and gene expression. Specifically, we showed significantly higher migration and faster wound closure when HepT1 cells were grown in the presence of secreted factors from HUVECs, compared to control HepT1 cells. Conclusions: This study provides a comprehensive transcriptomic landscape of pediatric liver cancer CTCs. This work also underscores the importance of the interactions between endothelial cells and tumor cells in the initiation of metastasis. Taken together, this work builds a strong foundation for future studies elucidating the specific mechanisms of how pediatric liver tumor cells successfully disseminate with the overall goal of developing novel therapeutic regimens that target these mechanisms.
 Citation Format: Priyanka Rao, Andres Espinoza, Roma Patel, Mohammad J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield. Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B072.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165a476b6bcad6280ce2606b1f9945d425fc4ed9" target='_blank'>
              Abstract B072: Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer
              </a>
            </td>
          <td>
            Priyanka Rao, Andres F. Espinoza, R. Patel, M. J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44ac4365f3d537cc727149bd2f0c2b4a261aed5" target='_blank'>
              The neuroendocrine transition in prostate cancer is dynamic and dependent on ASCL1.
              </a>
            </td>
          <td>
            Rodrigo Romero, Tinyi Chu, Tania J González Robles, Perianne Smith, Yubin Xie, Harmanpreet Kaur, Sara Yoder, Huiyong Zhao, Chenyi Mao, Wenfei Kang, Maria V. Pulina, K. Lawrence, Anuradha Gopalan, S. Zaidi, K. Yoo, Jungmin Choi, Ning Fan, Olivia Gerstner, W. Karthaus, E. deStanchina, Kelly V. Ruggles, P. Westcott, R. Chaligné, D. Pe’er, C. Sawyers
          </td>
          <td>2024-10-11</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="T cell heterogeneity is a cornerstone of the adaptive immune response, with CD8+ T cells playing a pivotal role in mediating cytotoxic responses to pathogens and tumors. While traditional studies have focused on phenotypic and functional diversity through surface marker expression and functional assays, the significance of biophysical properties in delineating T cell subpopulations remains underexplored. Here, we show that resting T cells exhibit a bimodal distribution in buoyant mass, which is not captured by surface marker based canonical immunophenotypes or physical parameters such as volume, density, dry mass, or light scattering. Functionally, we demonstrate that T cells with lower buoyant mass exhibit reduced proliferation, tend towards effector differentiation, and become more exhausted in the presence of innate tumor-secreted factors. Conversely, T cells with higher buoyant mass show greater memory potential and resistance to exhaustion. Our study not only provides insights into T cell biology but also suggests buoyant mass as a promising label-free biomarker to potentiate T cell-based therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5377b1937748d2da8a9f6d4a6c6c5712c248ef8" target='_blank'>
              Bimodal cell mass distribution separates CD8+ T cells into two distinct types with divergent differentiation dynamics
              </a>
            </td>
          <td>
            Jiaquan Yu, Ye Zhang, Sarah M. Duquette, Gianfranco L. Yee, Teresa Dinter, Thomas R. Usherwood, Weida Wu, Teemu P. Miettinen, Stefani Spranger, S. Manalis
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="The tumor microenvironment (TME) comprises a diverse array of cells, both cancerous and non-cancerous, including stromal cells and immune cells. Complex interactions among these cells play a central role in driving cancer progression, impacting critical aspects such as tumor initiation, growth, invasion, response to therapy, and the development of drug resistance. While targeting the TME has emerged as a promising therapeutic strategy, there is a critical need for innovative approaches that accurately replicate its complex cellular and non-cellular interactions; the goal being to develop targeted, personalized therapies that can effectively elicit anti-cancer responses in patients. Microfluidic systems present notable advantages over conventional in vitro 2D co-culture models and in vivo animal models, as they more accurately mimic crucial features of the TME and enable precise, controlled examination of the dynamic interactions among multiple human cell types at any time point. Combining these models with next-generation technologies, such as bioprinting, single cell sequencing and real-time biosensing, is a crucial next step in the advancement of microfluidic models. This review aims to emphasize the importance of this integrated approach to further our understanding of the TME by showcasing current microfluidic model systems that integrate next-generation technologies to dissect cellular intra-tumoral interactions across different tumor types. Carefully unraveling the complexity of the TME by leveraging next generation technologies will be pivotal for developing targeted therapies that can effectively enhance robust anti-tumoral responses in patients and address the limitations of current treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee78b792b563d1d9bcfce83a027e1da609d44dc5" target='_blank'>
              Improving tumor microenvironment assessment in chip systems through next-generation technology integration
              </a>
            </td>
          <td>
            Daniela Gaebler, Stephanie J. Hachey, Christopher C. W. Hughes
          </td>
          <td>2024-09-25</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Increasing evidence suggests the tumor microenvironment (TME) governs solid tumor response to immune checkpoint inhibition (ICI). Decoding the relationship between the cell compositional diversity of the tumor microenvironment (TME) and the therapeutic outcomes has been a longstanding problem in solid tumor research. In this work, we develop a systems-theoretic formalism to decipher the key mechanisms of growth, proliferation, immune evasion, and drug resistance that run common across solid tumors in the context of Immune Checkpoint Inhibitors (ICI). We reconstructed a core TME network in common across most solid tumors, containing multiple tumor and non-tumor cell types in distinct functional states, and molecular agents mediating cellular signaling and cell-cell interactions. Our analysis shows that the core TME network is sufficient to yield a multiplicity of attractors corresponding to clinically observed TME subtypes namely, immune or fibro dominated, immune or fibro desert, and immune and fibro deficient. Importantly, the reachability around the pre-ICI attractors governs the response to ICI explaining the TME subtype-specific therapy outcomes. We analyzed the attractor transition network to identify subtype-specific combination therapies that can drive unresponsive TME to a responsive subtype. We derived mathematical conditions relating TME balances to determine the limits of the efficacy of combination therapies. Our results hold for a large class of smooth biochemical kinetics with monotone and bounded interactions and (semi-)concave proliferation rules. The analytical findings have been verified with extensive simulation of different TME sub-types. Overall, we propose a generalized systems formalism that accounts for the TME properties governing ICI response and can aid in designing intervention strategies for improved tumor prognosis. Significance Statement Non-responsive therapy outcomes have been a persistent problem in cancer treatment. Predicting the possibility of non-responsiveness to a particular therapy from the pre-treatment composition of the tumor microenvironment (TME) aids in designing appropriate combination treatment strategies toward an improved prognosis. The present work develops a systems-theoretic formalism that aims to unfold the mechanisms behind solid tumors’ growth, non-responsiveness, and recurrence. Unlike a single model, the proposed formalism does not assume any particular kinetics, barring some minimal assumptions. This enables us to explain some of the relevant observations made by the recent experimental studies. Finally, the closed-form conditions obtained for responsivity and recurrence can also guide novel therapeutic strategies that may be able to restore responsiveness to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657fb6c08b911bd43c20c8cf26d56275f3634fc0" target='_blank'>
              From tumor microenvironment to immuno-therapeutic outcomes for solid tumors: A systems theoretic approach
              </a>
            </td>
          <td>
            Priyan Bhattacharya, R. Vadigepalli
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs), and circulating tumor related cells, are extremely rare cells that intravasate from the tumor into the circulatory system and can be captured via a liquid biopsy. Although CTCs contribute to the metastatic cascade, and diverse phenotypes of CTCs have been observed – including the cytokeratin expressing CTCs, CTC clusters, large polyploid CTCs, and CTCs undergoing epithelial to mesenchymal transition – little is known about their functionality. By virtue of CTCs being rare, a detection method that maximizes the information obtained per cell would be ideal to understand their biology and for use in diagnostic approaches. The challenge is that rare cell detection necessitates extensive processing steps, during which molecular content, such as RNA and DNA, needs to be preserved for downstream single cell analysis. We developed GEnomic, Transcriptomic, and MOrphological profiling of Rare cells (GeTMoR), a method that extends the High Definition Single Cell Assay for detecting rare cancer related cells to simultaneously image and profile the genome and transcriptome from single rare cells. We validated GeTMoR by spiking in cancer cell lines into whole blood to evaluate the quality of recovered gene expression and copy number profile from the same cell. The GeTMoR approach provides the ability to link the phenotype of rare cells, including CTCs, to their genome and transcriptome, thereby enabling insight into rare cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6df3d1f532e0cca0520b19b2419d2d9cd51ad9" target='_blank'>
              GeTMoR: Simultaneous genomic, transcriptomic, and morphological profiling of rare single cells
              </a>
            </td>
          <td>
            Rishvanth K. Prabakar, Michael J. Schmidt, Peter Kuhn, James B. Hicks
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is a heterogenous disease comprised of malignant cell states and an immunosuppressive microenvironment that underlie radiotherapy (RT) resistance. Here we integrate genome wide and single cell in vivo functional genomic screens of preclinical models with single nuclei and spatial transcriptomics of human tumors to identify therapeutic vulnerabilities driving GBM radioresistance.



 Perturb-seq of intracranial GBM models was performed using human (GBM43) or mouse (GL261, SB28) GBM cells expressing CRISPR interference (CRISPRi) machinery, with sgRNA libraries delivered to intracranial tumors using convection enhanced delivery. Genome wide dual sgRNA CRISPRi screens were performed in GBM cells (GBM43, LN18, GL261, SB28) ± RT. Functional genomic analyses across preclinical models were integrated with single nuclei and spatial transcriptomics from human GBMs. Validation of genetic phenotypes was performed using the DNA-PK inhibitor peposertib to treat GBM cells and orthotopic mouse models (SB28, GBM43, GBM6), which were analyzed using IF and single cell RNA sequencing.



 Genome wide CRISPRi screens identified 114 radiation sensitizing targets. In vivo perturb-seq of radiation sensitizing targets reprogrammed 25 gene modules that spanned homeostatic, metabolic, and inflammatory pathways and nominated DNA-PK as a pleotropic radiation sensitizer that regulates interactions between malignant cells and the myeloid microenvironment. Genetic and pharmacologic inhibition of DNA-PK revealed radiotherapy dependent reprograming of cell states through cell intrinsic mechanisms (e.g. cell cycle, proteasome) and cell extrinsic pathways (e.g. TNFα, IFN). Peposertib sensitized GBMs to radiotherapy and extended survival in mice harboring intracranial xenografts. DNA-PK inhibition induced cGAS/STING and IFN signaling in malignant cells (n = 58,161) that were associated with pro-inflammatory polarization of infiltrating myeloid cells (n = 22,067). These alterations were synergistic with RT but were not induced by RT alone. Spatial transcriptomics of 8 matched primary-recurrent GBMs (20,793 transcriptomes) from patients treated with RT and peposertib, and single nuclei transcriptomics from 100 GBMs (76 matched primary-recurrent tumors, 4 after treatment with RT and peposertib, 356,997 nuclei) demonstrated malignant cell differentiation and enrichment of pro-inflammatory macrophages in the tumor microenvironment after inhibition of DNA-PK.



 Integration of genome wide CRISPRi screens and in vivo perturb-seq in preclinical models with single nuclei and spatial transcriptomic analyses of human tumors enables discovery and mechanistic validation of therapeutic targets in GBM. These results reveal DNA-PK as a target for radiation sensitization of tumor and myeloid microenvironment cells in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59313aeff0a382ec9e51c8a10cae8b225d6db462" target='_blank'>
              OS07.1.A INTEGRATIVE SINGLE NUCLEI, SPATIAL, AND FUNCTIONAL GENOMIC ANALYSES REVEAL DNA-PK DRIVES GLIOBLASTOMA RADIORESISTANCE THROUGH TUMOR CELLS AND THE MYELOID MICROENVIRONMENT
              </a>
            </td>
          <td>
            J. Liu, N. Majd, C. Zou, K. Mirchia, J. Pak, K. Foster, K. Lin, T. Casey-Clyde, K. Seo, S. Pan, E. Payne, H. Vasudevan, T. Ozawa, M. Berger, J. de Groot, W. Weiss, L. Gilbert, D. Raleigh
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968920bf1360bd97ef564975b23fe9a8085f15c4" target='_blank'>
              Microdissection tools to generate organoids for modeling the tumor immune microenvironment.
              </a>
            </td>
          <td>
            Seth C. Cordts, Kanako Yuki, Maria F Henao Echeverri, Balasubramanian Narasimhan, C. Kuo, Sindy K. Y. Tang
          </td>
          <td>2024-09-10</td>
          <td>Microsystems & nanoengineering</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is one of the deadliest types of cancer with median overall survival of little more than a year. It is the prototypic example of a molecularly diverse tumor associated with multiple subclones and high transcriptional plasticity. We here set out to map the clonal architecture and link it with growth patterns, transcriptional states and drug response in-vitro.



 Seventy-eight multiregional samples from 24 GBM patients (F/M ratio = 0.60, median age= 65.5ys) were profiled using deep whole-exome sequencing with a nominal coverage of >300x. Clonal complexity was assessed by the Euclidean distance based on somatic mutations between regions. Molecular distance was integrated with 3D neuronavigation-based physical distances and patient survival. Fresh tumor slices of 8 patients were cultured in minimal essential media and treated with 4.2Gy plus 200µM temozolomide for 48h. Spatial Transcriptomics was performed on matched tumor slices with /without treatment.



 Integrative characterization revealed two evolutionary trajectories termed expansive and stochastic diversification based on the correlation between physical and molecular distance (RE=0.6, RS=-0.2). High molecular distance was associated with reduced PFS (R=-0.5538, p=0.026). Tumors with expansive evolution demonstrated volume-based growth that directly correlated with increased molecular diversification, while stochastic models showed irregular and discordant patterns. Patients with stochastic evolution were characterized by unfavorable prognosis (p= 0.35) and tumors appeared more frequently in the frontal lobe. In-vitro drug response varied considerably between patients based on cell proliferation (minus 0.5-13.8%), apoptosis (30.1-44.4%), and viability (minus 7.8-34.1%). Under treatment an increase of mesenchymal states with peculiar redistribution of transcriptional patterns and increased clonal complexity was observed.



 In sum, we report on an ongoing study in glioblastoma that seeks to link the clonal architecture with tumor growth, transcriptional patterns and in-vitro drug response. Our study will provide new insights into the clinical relevance of the clonal composition of this deadly cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f41aef776a5ac27081fe2094b9b9942292ad4e2a" target='_blank'>
              P01.02.B CLONAL ARCHITECTURE, 3D TUMOR GROWTH, TRANSCRIPTIONAL STATES AND IN-VITRO DRUG RESPONSE IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Y. Song, Y. Miller-Michlits, L. Greutter, K. Nenning, B. Kiesel, D. Loetsch-Gojo, V. Redl, J. Gesperger, J. Jana Breu, D. Nam, G. Widhalm, J. K. Sa, A. Woehrer
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Immunotherapies such as immune checkpoint inhibitors (ICIs) have greatly improved patient outcomes in multiple tumor types, but ICIs have failed to achieve responses in many patients. Meta-analyses of patient cohorts treated with ICIs have observed that worse clinical outcomes are strongly correlated with high levels of intratumoral genetic heterogeneity (ITH). ITH occurs when tumor cells mutate as they divide, typically due to oxidative stress or DNA damage from rapid proliferation and continued carcinogenic insults, generating tumor subclones. However, the mechanisms by which ITH impairs the anti-tumor immune response are not fully understood. We propose a novel mechanism for how ITH in highly T cell-infiltrated inflamed tumors can adversely affect T cell behavior. We hypothesize that strongly pro-inflammatory tumor subclones secreting robust pro-inflammatory chemokine profiles can dominate T cell trafficking within the tumor, resulting in a weakened immune response against other tumor subclones. Using a murine squamous cell skin carcinoma (SCC) model, we have demonstrated that in immune-inflamed tumors, the spatial distribution and behavior of tumor-infiltrating T cells is influenced by ITH. A strongly pro-inflammatory tumor line expressing an endogenous RFP fluorophore was mixed with a weakly pro-inflammatory tumor line expressing an endogenous YFP fluorophore at a 1:1 ratio. The resulting tumors contained distinct RFP+ and YFP+ regions. These were analyzed by immunofluorescence (IF) and by manually dissecting the tumors and analyzing each region by flow cytometry. Flow cytometry analysis determined that the strongly pro-inflammatory RFP+ regions contained significantly more total T cells and CD4+ T cells than YFP+ regions. Furthermore, FoxP3- “conventional” CD4+ T cells were significantly more likely to be “terminal effector-like” CD44+CD62L- in RFP+ regions, and significantly more likely to be “naïve-like” CD62L+CD44- in YFP+ regions. IF analysis confirmed the preferential distribution of CD4+ T cells in RFP+ regions, and observed that this spatial pattern was evident as early as 11 days after subcutaneous injection. Notably, this localization trend did not occur with CD8+ T cells, which were evenly dispersed throughout the tumors. ELISA analysis of lysates of both tumor cell lines determined that the pro-inflammatory RFP+ cell line secretes significantly higher levels of the chemokine CCL5, which has been observed by other studies to selectively mediate CD4+ T cell chemotaxis. These results demonstrate that intratumoral CD4+ trafficking is shaped by the spatial distribution of specific tumor subclones, and suggest tumor cell chemokine signaling may directly mediate T cell spatial localization and behavior within tumors.
 Citation Format: Robert Letchworth, Savannah Hughes, Abigail Keku, Melissa Reeves. Pro-inflammatory tumor cells drive CD4+ T cell localization within heterogeneous tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aeeff4ff2175551d0a146978acdaeb3ec633ea2b" target='_blank'>
              Abstract A033: Pro-inflammatory tumor cells drive CD4+ T cell localization within heterogeneous tumors
              </a>
            </td>
          <td>
            Robert Letchworth, Savannah Hughes, Abigail Keku, Melissa Q Reeves
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cervical cancer remains the fourth most common cancer affecting women worldwide, and incidences of other HPV-related cancers continue to rise. For the development of effective prevention strategies in high-risk patients, we aimed to better understand the roles of inflammatory pathways and the tumour microenvironment as the main driver of progression to malignancy in HPV-infected tissues. We analysed the spatial organisation of seven samples of HPV+ high-grade squamous intraepithelial lesion (HSIL) and cervical intraepithelial neoplasia 3 (CIN3), comparing tumour heterogeneity and immune microenvironments between pre-malignant (neoplastic) and adjacent cervical tissues. We observed evidence of immune suppression within the neoplastic regions across all samples and identified distinct immune clusters for each dysplastic lesion. Previous single-cell data analyses in an HPV16 E7 oncoprotein-driven transgenic mouse model suggested a potential role for IL34-CSF1R signalling in immune modulation, where low IL34 expression was associated with Langerhans cell dysfunction, and, in cervical cancer, with poor patient outcome. Here, we observed that IL34-CSF1R co-expression was absent within HPV-associated neoplastic tissues but present in adjacent normal tissue regions. Additionally, we identified enrichment of an M2 gene signature in neoplastic tissue, while adjacent tissue was enriched with a pro-inflammatory M1 gene signature. Our findings provide bio-pathological insights into the spatial cellular and molecular mechanisms underlying HPV-associated cervical cancer immune regulation and suggest a strategy to modulate the immune system in HPV-positive neoplastic cervical and other tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78e69a13d70e65b0ae755f807b9334672667003c" target='_blank'>
              Spatial analysis of HPV associated cervical intraepithelial neoplastic tissues demonstrate distinct immune signatures associated with cervical cancer progression
              </a>
            </td>
          <td>
            Gianna Pavilion, Hani Vu, Zherui Xiong, Thi Viet Trinh Dang, Michael Walsh, Andrew Causer, J. Chandra, Quan Nguyen, Ian H. Frazer
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbcb18aaf8922f16ad19ad9eeb7b0ec8bc2789d" target='_blank'>
              Serial single-cell RNA sequencing unveils drug resistance and metastatic traits in stage IV breast cancer
              </a>
            </td>
          <td>
            K. Otsuji, Yoko Takahashi, Tomo Osako, Takayuki Kobayashi, T. Takano, S. Saeki, Liying Yang, Satoko Baba, Kohei Kumegawa, Hiromu Suzuki, Tetsuo Noda, Kengo Takeuchi, Shinji Ohno, T. Ueno, Reo Maruyama
          </td>
          <td>2024-10-03</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e8c0aed3c023b673b9bb62e9f661eb422606ebf" target='_blank'>
              A deep-learning model for characterizing tumor heterogeneity using patient-derived organoids
              </a>
            </td>
          <td>
            Kosuke Takagi, Motoki Takagi, Gen Hiyama, Kazuhito Goda
          </td>
          <td>2024-10-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb1ad46e8ba824353a9e31f8eb7a231bf91169bb" target='_blank'>
              Integrative multi-omics analysis reveals the role of tumor-associated endothelial cells and their signature in prognosis of intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Hao Jiang, Biao Gao, Zihe Meng, Yafei Wang, T. Jiao, Junfeng Li, Xuerui Li, Yinbiao Cao, Xianzhou Zhang, Chonghui Li, Shichun Lu
          </td>
          <td>2024-10-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Ethnic disparities in breast cancer (BC) outcomes have been reported, with lower survival rates among black women compared to white. Additionally, the unfavorable triple negative (TNBC) sub-type is more prevalent in black women and within this population, risk of mortality is again higher compared to white women. Biological contributors to this disparity are less clearly understood than socio-economic factors. Recent studies recognize different breast tumour immune microenvironments between black and white BC patients, necessitating further research into ethnicity-related differences in anti-tumor immune responses. Such responses begin in regional lymph nodes (LNs) with tumor neoantigen presentation. Here, we generate single-cell spatial maps of involved TNBC LN cores in treatment naïve black and white women. To guide design of a phenotyping panel for LNs, we profiled blood samples (n=95) using high-dimensional flow cytometry from newly diagnosed black and white BC patients. We found differential skewing of NK cell subsets, lower memory B cell (identified as IgM+CD27+IgD- & IgM+CD27+IgD+ cells) abundance and higher CD4+ T effector memory presence in black patients compared to white. Additionally, higher antigen presentation capacity was observed with increased intermediate monocytes and conventional DCs, confirmed by MS-based proteomics. Using highly multiplexed imaging with a panel of 26 markers, we performed single-cell spatial immune profiling to characterize cellular phenotypes, neighborhoods and interactions within a cohort of LN core biopsies from black patients and white patients. A total of 536,113 cells were segmented for analysis. Cell abundance analysis revealed higher CD8+ T effector memory and CD4+ T cell subsets in white patients, except for CD4+ T effector memory cells which were higher in black patients, consistent with blood profiling. Cellular neighborhood analysis revealed black patient tumor-rich regions were enriched for M2 macrophages while white patients had no such enrichment. In these white patients, a neighborhood resembling canonical follicular structures almost exclusively containing B cells was identified. Interestingly in these black patients, analogous B cell rich regions also comprised a heterogeneous array of other immune cells. Further analysis to explore B cell zones is ongoing. Preliminary interaction analysis identified immunosuppressive crosstalk in black patients with enriched Treg & CD8+ T cell and Treg & NK interactions, not observed in white patients. These initial findings suggest ethnicity-based variation in LN architectures from treatment naive patients with TNBC, supporting the hypothesis that variation in immune programming could translate into differences in pro and anti-tumor immunity, contributing to disparities in outcome amongst black and white patients.
 Citation Format: Paul R. Buckley, Melek Akay, Esme Carpenter, Rosalind Graham, Helen Kakkassery, Thanussuyah Alaguthurai, Priya Hossain, Patrycja Gazińska, Sheeba Irshad. Differential single-cell spatial immune cell profiles and immune cell interactions identified within lymph nodes metastases of black and white women diagnosed with triple negative breast cancers [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C142.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c431f2f91ac1c49d9f19dea4541265da8c85d8d6" target='_blank'>
              Abstract C142: Differential single-cell spatial immune cell profiles and immune cell interactions identified within lymph nodes metastases of black and white women diagnosed with triple negative breast cancers
              </a>
            </td>
          <td>
            Paul Buckley, Melek Akay, Esme Carpenter, Rosalind Graham, Helen Kakkassery, T. Alaguthurai, Priya Hossain, Patrycja Gazińska, S. Irshad
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Glioblastoma is the most common malignant primary brain tumor in adults with a poor survival and an unmet need for more effective therapies. Macroscopically, as observed through MR imaging and microscopy, these tumors are characterized by distinct tumor regions: a contrast-enhancing tumor core with a high cancer cell density and an infiltration zone that appears T2/FLAIR hyperintense without the uptake of contrast agent and consists of infiltrating tumor cells as well as non-malignant cells of the microenvironment. Previous genomic and transcriptomic studies revealed molecular differences between the tumor core and the infiltration zone. However, functional differences, such as drug sensitivity, between tumor regions are still largely unknown.



 To address this gap, we collected region-specific glioblastoma patient tissue samples from both the contrast-enhancing tumor core and the non-contrast-enhanced but 5-aminolevulinic acid (5-ALA) fluorescent infiltration zone directly from surgery. We utilized pharmacoscopy, a microscopy-based single-cell drug screening approach, to identify drug responses across regions and patients.



 By profiling a library of 60 drugs across regions in a cohort of 23 patients, we identified region-specific drug responses and developed a complementarity score to pinpoint drug combinations capable of targeting both the tumor core and the infiltration zone. These combination treatments exhibited highly promising anti-tumor activity across regions and patients.



 Given the high levels of functional spatial intra- and inter-tumor heterogeneity, relying on a single drug across patients may not be feasible for effective patient treatment. Thus, there is a strong rationale to administer drug combinations that not only consider regional heterogeneity but also account for patient-specific variations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fa1d38bc5a81ab129f0d7423cbbd80eb88f3cf3" target='_blank'>
              P01.13.B SPATIALLY RESOLVED FUNCTIONAL PROFILING FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Buck, S. Lee, R. Wegmann, J. Mena, F. Vasella, J. Friesen, L. Regli, E. Le Rhun, A. Zeitlberger, M. Neidert, N. Tatari, G. Hutter, M. Weller, B. Snijder, T. Weiss
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Even in its localized stages, pancreatic ductal adenocarcinoma (PDAC) is virtually incurable, characterized by early dissemination and disease recurrence. The liver is the most common site of metastasis, and in most cases hepatic recurrences drive survival outcomes. This underscores the critical need to target and prevent liver metastases. We leveraged a novel biobanking strategy to perform paired single-cell RNA (scRNA-seq) and TCR sequencing (scTCR-seq) to profile multisite immunity in patients with localized PDAC across the tumor, peripheral blood, and premetastatic liver. This analysis of three patients included three tumor, three blood, and nine liver (three per patient from separate segments) samples, and generated 85,748 cells for downstream analysis. We evaluated populations of tumor-expanded T cells as potential indicators of antitumoral immunity, and utilized validated computational approaches to identify tumor-reactive T cells based on key markers of dysfunction and exhaustion. Gene set enrichment analysis was performed on tumor infiltrating lymphocytes (TILs) using signatures specific to tumor-reactive CD4 and CD8 T cells (“NeoTCR4” and “NeoTCR8,” respectively), and scored these using the AUCell package in R. Overall, 153 CD4 and 22 CD8 T cells fit candidate NeoTCR4 and NeoTCR8 signature states, representing 5.6% and 0.8% of the total TIL population respectively. These nominated T cells expressed key marker genes associated with tumor specificity such as CXCL13, TIGIT, PDCD1, ENTPD1, and TOX. Thirteen of these predicted tumor-reactive T cell clones identified within the tumor were also detected within our liver samples, suggestive of a potential antitumoral immune response in the premetastatic liver space. We next performed 10x Xenium in situ spatial transcriptomic analysis of primary tumor, normal adjacent pancreas, and premetastatic liver from two of these patients, using tissue microarrays to incorporate replicate 2mm core biopsies from each of these sites. A custom designed gene panel, guided by our scRNA-seq dataset, targeted tumor-specific mutations (e.g. KRAS G12V, G12D, G12R, and Q61) and immune related genes with a specific focus on markers of T cell activity and exhaustion. Leveraging these latter marker genes, which represent a subset of the NeoTCR4 and NeoTCR8 signatures, we identified a population of exhausted TILs within the tumor microenvironment, representing potentially tumor-reactive T cells whose interactions can now be explored at spatial resolution. Future directions are aimed at experimentally validating the tumor reactivity of the predicted neoantigen-specific TILs within our dataset and characterizing the immune neighborhoods that exist in both the tumor and liver of PDAC patients. This work establishes exciting opportunities for novel T-cell therapy in the treatment of PDAC, and we have already implemented these experimental findings in a forthcoming clinical trial investigating the applicability of this predictive signature score for the generation of TCR-engineered lymphocyte therapy in patients with PDAC.
 Citation Format: Elishama N Kanu, Ashley A Fletcher, Sri Krishna, Frank J Lowery, Haotian Zhuang, Ethan S Agritelley, Jiayin Bao, Austin M Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, Michael E Lidsky, Sabino Zani, Dan G Blazer III, Peter J Allen, Nicholas D Klemen, Zhicheng Ji, Daniel P Nussbaum, Erika J Crosby. Single-cell and spatial analysis of the immune landscape unveils a subset of potentially tumor-reactive T cells in patients with localized PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb49745088796a4e1e8fb6d2e9a57e189de99728" target='_blank'>
              Abstract B052: Single-cell and spatial analysis of the immune landscape unveils a subset of potentially tumor-reactive T cells in patients with localized PDAC
              </a>
            </td>
          <td>
            Elishama N. Kanu, Ashley A. Fletcher, Sri Krishna, Frank J Lowery, Haotian Zhuang, Ethan S Agritelley, Jiayin Bao, A. Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, M. Lidsky, S. Zani, D. Blazer, Peter J. Allen, Nicholas D. Klemen, Zhicheng Ji, Daniel P. Nussbaum, Erika J. Crosby
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b28b9f10dc7dee52d960145a8bb408a48e4a08ef" target='_blank'>
              Multiparametric profiling of HER2-enriched extracellular vesicles in breast cancer using Single Extracellular VEsicle Nanoscopy
              </a>
            </td>
          <td>
            Nan Jiang, Andras Saftics, Eugenia Romano, I. Ghaeli, Cristal Resto, V. Robles, Saumya Das, K. Van Keuren-Jensen, Victoria L Seewaldt, T. Jovanović-Talisman
          </td>
          <td>2024-09-28</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Gliomas comprise a diverse spectrum of related tumor subtypes with varying biological and molecular features and clinical outcomes. Advances in detailed genetic and epigenetic characterizations along with an appreciation that subtypes associated with developmental origins, including brain location and patient age, have shifted glioma classification from the historical reliance on histopathological features to updated categories incorporating molecular signatures and spatiotemporal incidence. Within a subtype, individual gliomas show cellular heterogeneity, generally containing subpopulations resembling different types of normal glial and progenitor cells. In addition to tumor-autonomous mechanisms of aberrant growth regulation driven by genetic mutations and signaling between tumor cells, interactions with the tumor microenvironment, including neurons, astrocytes, oligodendrocyte precursor cells, and the immune microenvironment play important roles in driving glioma growth and influencing response to treatment. The emerging understanding of the complex contributions of normal brain to glioma growth represents new opportunities for therapeutic advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b4b46271e9cdc90000ffc4d7028f66e1ab4920c" target='_blank'>
              Glial Malignancies.
              </a>
            </td>
          <td>
            Suzanne J Baker, Hui Zong, Michelle Monje
          </td>
          <td>2024-09-16</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Summary Imagine if we could simultaneously predict spatial protein expression in tissues from their routine Hematoxylin and Eosin (H&E) stained images, and create tissue images given protein expression profiles thus enabling virtual simulations of how protein expression alterations impact histology in complex diseases like cancer. Such an approach could lead to more informed diagnostic and therapeutic decisions for precision medicine at lower costs and shorter turnaround times, more detailed insights into underlying disease pathology as well as improvement in predictive and generative performance. In this study, we investigate the intricate correlation between protein expressions obtained from Hyperion mass cytometry and histopathological microstructures in conventional H&E stained glioblastoma (GBM) samples, unveiling morphological patterns and cellular-level spatial alterations associated with protein expression changes. To model these complex relationships, we propose a novel generative-predictive framework called Ouroboros for producing H&E images from protein expressions and simultaneously predicting protein expressions from H&E images. Our comprehensive sample-independent validation over 9920 tissue spots from 4 GBM samples encompassing visual image analysis, quantitative analysis, subspace alignment and perturbation experiments shows that the proposed generative-predictive approach offers significant improvements in predicting protein expression from images in comparison to baseline methods as well as accurate generation of virtual GBM sample images. This proof of concept study can contribute to advancing our understanding of histological responses to protein expression perturbations and lays the foundations for further developments in this area. Availability and implementation Implementation and associated data for the proposed approach are available at the URL: https://github.com/Srijay/Ouroboros.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67ded4a0599c7810f90359134e15ada22ea589a" target='_blank'>
              Ouroboros: cross-linking protein expression perturbations and cancer histology imaging with generative-predictive modeling
              </a>
            </td>
          <td>
            Srijay Deshpande, Sokratia Georgaka, Michael Haley, Robert Sellers, James Minshull, J. Nallala, Martin Fergie, Nicholas Stone, N. Rajpoot, Syed-Murtuza Baker, Mudassar Iqbal, Kevin Couper, Federico Roncaroli, F. Minhas
          </td>
          <td>2024-09-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Understanding the mechanisms by which cancer cells switch between different adaptive states and evade therapeutic interventions is essential for clinical management. In this study, the in vivo cellular dynamics of a new chronic myeloid leukaemia cell line displaying altered phenotype and resistance to tyrosine kinase inhibitors were investigated in correlation with their parental cells for invasiveness/metastasis, angiogenic potential and population kinetics. We showed that the cells exhibiting drug resistance and plastic phenotype possess an increased capacity for invasion compared to their parental cells, that exposure to imatinib mesylate has the potential to enhance cellular motility and that in a leukaemic cell population, even a minority of plastic cells exhibit improved migratory ability. Furthermore, we show that these plastic cells have angiogenic and extravasation potential. The present study provides significant insights into the cellular dynamics displayed by a TKI‐resistant, phenotypically plastic CML cell line, using a zebrafish (Danio rerio) xenograft model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8debdaa7aac5be72ea83fbcd5d951e3b97abea7b" target='_blank'>
              Phenotypically plastic drug‐resistant chronic myeloid leukaemia cell line displays enhanced cellular dynamics in a zebrafish xenograft model
              </a>
            </td>
          <td>
            Seda Baykal, Zeynep Yuce, G. Ozhan
          </td>
          <td>2024-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor heterogeneity often leads to substantial differences in responses to same drug treatment. The presence of pre-existing or acquired drug-resistant cell sub-populations within a tumor survive and proliferate, ultimately resulting in tumor relapse and metastasis. The drug resistance is the leading cause of failure in clinical tumor therapy. Therefore, accurate identification of drug-resistant tumor cell subpopulations could greatly facilitate the precision medicine and novel drug development. However, the scarcity of single-cell drug response data significantly hinders the exploration of tumor cell resistance mechanisms and the development of computational predictive methods. In this paper, we propose scDrugAtlas, a comprehensive database devoted to integrating the drug response data at single-cell level. We manually compiled more than 100 datasets containing single-cell drug responses from various public resources. The current version comprises large-scale single-cell transcriptional profiles and drug response labels from more than 1,000 samples (cell line, mouse, PDX models, patients and bacterium), across 66 unique drugs and 13 major cancer types. Particularly, we assigned a confidence level to each response label based on the tissue source (primary or relapse/metastasis), drug exposure time and drug-induced cell phenotype. We believe scDrugAtlas could greatly facilitate the Bioinformatics community for developing computational models and biologists for identifying drug-resistant tumor cells and underlying molecular mechanism. The scDrugAtlas database is available at: http://drug.hliulab.tech/scDrugAtlas/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c67f110b19aa753b80dffa15ad6e0a2ac3b41aaf" target='_blank'>
              scDrugAtlas: an integrative single-cell drug response atlas for unraveling tumor heterogeneity in therapeutic efficacy
              </a>
            </td>
          <td>
            Wei Huang, Xinda Ren, Yinpu Bai, Hui Liu
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Introduction This study aims at elucidating mechanisms of immune infiltration in osteosarcoma (OSA) to identify novel therapeutic strategies. Methods Here, joint analysis of spatial transcriptomics (ST, Visium) data from 26 formalin-fixed paraffin embedded OSA samples at diagnosis (n=9) and relapse (n=17, MAPPYACTS NCT02613962 & OS2006 NCT00470223) from 19 donors (including 5 diagnosis/relapse pairs) was performed using the CellsFromSpace unbiased reference-free signal deconvolution and analysis workflow. This methodology enabled the definition of the topology and the molecular signatures of the cancerous, stromal and immune components across all samples, and allowed for the phenotyping of individual tumors based on cancer subset composition. Spatial colocalization analysis was performed using Lee’s bivariate spatial association measure of different cell populations, enabling the characterization of tumor-infiltrating immune cells found across samples. Similar ST analysis of Patient-derived xenograft (PDX) samples was performed on fresh-frozen samples. Subsequent separation of human and mouse reads using the Xenome algorithm allowed for a clear dissection of the cancer vs. microenvironment compartments. Preliminary signature validation on larger cohorts was performed on bulk RNAseq samples from the OS2006 and MAPPYACTS cohorts. Results Our joint spatial transcriptomics analysis of 26 primary and relapse tumors revealed 64 transcriptomic signatures associated with cancer cells, some of which were shared across multiple samples. These signatures could be consolidated into 18 broad phenotypic categories frequently detectable in various samples, showcasing significant intratumoral heterogeneity within the cancerous compartment. By averaging the composition per sample, we found that samples clustered based on the relative intratumoral abundance of cancer phenotypes, indicating the presence of at least two main differentiation archetypes in OSA: “High intratumoral heterogeneity” and “Undifferentiated.” Additionally, spatial niche analysis of immune cells within tumors identified immune populations consistently associated with tumors across samples. This analysis revealed a specific myeloid lineage capable of infiltrating tumors in all samples, likely mediated by chemotactic signals linked to cancer differentiation, as suggested by ligand-receptor analysis. To validate these findings, PDX models of OSA were analyzed, confirming the presence of a similar phenotype of tumor-infiltrating myeloid cells and underscoring the relevance of PDX models in OSA research. Furthermore, preliminary analyses querying the transcriptomic signature of infiltrating myeloid cells in larger bulk RNAseq cohorts revealed promising associations with patient outcomes, detectable even at diagnosis. Conclusions Our research elucidates patterns of intratumoral phenotypic heterogeneity in OSA and identifies the primary immune cell populations specifically infiltrating OSA tumors. This paves the way for developing novel therapies by leveraging these cellular tropism mechanisms.
 Citation Format: Gaël Moquin-Beaudry, Maria Eugenia Marques Da Costa, Hanane Zair, Corentin Thuilliez, Pierre Khneisser, Nicolas Signolle, Jean-Yves Scoazec, Birgit Geoerger, Nathalie Gaspar, Antonin Marchais. Characterizing the spatial transcriptomic landscape of Osteosarcoma from diagnosis to relapse [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0f2ff33e6fea180aea3b91082872e0d960db848" target='_blank'>
              Abstract B033: Characterizing the spatial transcriptomic landscape of Osteosarcoma from diagnosis to relapse
              </a>
            </td>
          <td>
            G. Moquin-Beaudry, Maria Eugénia Marques da Costa, Hanane Zair, Corentin Thuilliez, Pierre Khneisser, Nicolas Signolle, J. Scoazec, Birgit Geoerger, Nathalie Gaspar, A. Marchais
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f4bbafe57f1da4d23af55ae0496a72e14379c17" target='_blank'>
              Deep learning pipeline for automated cell profiling from cyclic imaging
              </a>
            </td>
          <td>
            Christian Landeros, Juhyun Oh, Ralph Weissleder, Hakho Lee
          </td>
          <td>2024-10-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/530d6f24e583e529ad460451ca4a394ebf382c85" target='_blank'>
              Mucosa-like differentiation of head and neck cancer cells is inducible and drives the epigenetic loss of cell malignancy
              </a>
            </td>
          <td>
            F. Oppel, S. Gendreizig, L. Martínez-Ruiz, J. Florido, Alba López-Rodríguez, H. Pabla, Lakshna Loganathan, Leonie Hose, P. Kühnel, P. Schmidt, M. Schürmann, J. M. Neumann, F. Viyof Ful, L. Scholtz, Dina Ligum, Frank Brasch, Karsten Niehaus, Germaine Escames, Tobias Busche, Jörn Kalinowski, Peter Goon, Holger Sudhoff
          </td>
          <td>2024-10-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Testicular tumors are the most common malignancy of young men and tumors affecting the testis are caused by somatic mutations in germ or germ-like cells. The PI3K pathway is constitutively activated in about one third of testicular cancers. To investigate the role of the PI3K pathway in transforming stem-like cells in the testis, we investigated tumors derived from mice with post-natal, constitutive activation of PI3K signaling and homozygous deletion of tumor suppressor Pten, targeted to nestin expressing cells. Mice developed aggressive tumors, exhibiting heterogeneous histopathology and hemorrhaging. The tumors resemble the rare testis tumor type, testicular sex cord–stromal Leydig cell tumors. Single cell resolution spatial tissue analysis demonstrated that T-cells are the dominant tumor infiltrating immune cell type, with very few infiltrating macrophages observed in the tumor tissue, with CD8+ T-cells predominating. Further analysis showed that immune cells preferentially localize to or accumulate within stromal regions. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee742f58eb6122ac85c8bf499863f34f30c2b9b5" target='_blank'>
              Testicular sex cord–stromal tumors in mice with constitutive activation of PI3K and loss of Pten
              </a>
            </td>
          <td>
            Marija Dinevska, Lachlan Mcaloney, Samuel S. Widodo, G. Filiz, Jeremy Anderson, Sebastian Dworkin, Simon P. Windley, Dagmar Wilhelm, Theo Mantamadiotis
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Glioblastoma, a highly aggressive brain tumor, presents a daunting prognosis with an average survival of under 15 months due to its invasive nature. Understanding the specific behaviors of individual tumor cells in terms of volume, shape, and movement dynamics is crucial for improved patient stratification and understanding tumor mechanisms.



 we developed Single-Cell Behavior Live Imaging (ScBLI) analysis, a novel approach combining live imaging with advanced mathematical processing. Our method enables precise analysis of morphological and behavioral traits in patient-derived tumor cells cultured in vitro. Leveraging sophisticated cell tracking methods, we analyzed 21 glioblastoma and 1 healthy astrocyte (HA) cell cultures, to capture detailed movement information. Our analyses extend beyond mere tracking; we extrapolated an overarching mobility strategy employed by the cells within each culture. By measuring the delta scaling parameter, we discern between subdiffusive motion, random Brownian motion, and more organized efficient strategies such as a typical Levy Walk. We correlate variations in delta scaling with cell applied stimuli, such as invasion and antineoplastic agents, enhancing our understanding of tumor cell behavior.



 We analyzed 100 cell trajectories per culture, totaling 2200 trajectories. Parameters as distance traveled, volume, track displacement, directional change rate, circularity, mean velocity, and confinement ratio were used to identify 3 distinct cell populations with different distributions within each GBM and HA culture facilitating the stratification of samples into 3 clusters. Moreover, the delta scaling was divided into three groups (0.28 to 0.80), and its integration with clinical data revealed an increased percentage of patients with shorter overall survival as scaling increased. Higher delta scaling indicated a more organized system adapted to challenging environments. Transcriptome analysis unveiled gene expression differences, highlighting potential therapeutic targets.



 This approach provides a deeper understanding of how tumor cells navigate and occupy the available space, shedding light on their dynamic behavior in vitro. Our results indicate that cancer cells must preserve the control of their locomotion to explore space in an organized and efficient way for survival, similar to the organized and functional behavior of healthy cells. By integrating the unique characteristics of each patient’s tumor cell behavior with clinical data, this approach offers a new method for stratifying patients with different survival probabilities. Overall, by combining imaging, image processing algorithms, and dynamic modeling, we provide a valuable framework for understanding the complex dynamics of glioblastoma and paving the way for more personalized and effective treatment approaches.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6a935ed764568a05683204598acba4c1c6581d6" target='_blank'>
              OS07.8.A INNOVATIVE DYNAMIC STRATIFICATION IN GLIOBLASTOMA: ANALYZING TUMOR CELL CHARACTERISTICS AND MOBILITY STRATEGIES THROUGH SINGLE CELL BEHAVIOR LIVE IMAGING (SCBLI)
              </a>
            </td>
          <td>
            G. Ferri, M. Morelli, A. Pastore, F. Lessi, F. Di Lorenzo, S. Franceschi, F. Pieri, C. Gambacciani, Y. Shah, L. Tonello, O. Santonocito, A. D. Di Stefano, L. Palatella, P. Grigolini, C. Mazzanti
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Glioblastoma (GB) is an aggressive brain cancer with an average survival of 14.6 months. Effective treatment is hindered by GB’s plasticity and invasive phenotype. Little is known regarding the molecular mechanisms driving invasion heterogeneity.



 We developed a bioengineered model of brain-like matrix, which together with patient-derived spheroids, allows study of GB’s migration in vitro. Twenty patient-derived cell lines with different intrinsic (genomic, transcriptomic) characteristics in several extrinsic (extracellular matrix stiffness, hypoxia vs normoxia) conditions were tested.



 Time-lapse video analysis identified recurrent migratory phenotypes across the cell lines. GB’s subtype- proneural, neural, classical, and mesenchymal- did not cluster cell lines with similar migratory phenotypes. An increase in matrix stiffness led to a switch in migratory phenotype in certain cell lines.



 We have identified finite and periodic migration phenotypes of GB in different conditions.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0c9fed4fcde7dc85545d88aa10cc4b4c6e9e390" target='_blank'>
              P01.01.A CHARACTERIZATION OF GLIOBLASTOMA’S MIGRATION HETEROGENEITY THROUGH A PATIENT-DERIVED 3D MODEL
              </a>
            </td>
          <td>
            A. Pujol-Castiblanque, T. Savin, A. E. Markaki, R. Mair
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 High-grade glioma (HGG) is the most common and aggressive form of primary brain cancer with poor prognosis. Standard of care surgery employs Magnetic Resonance Imaging (MRI) to identify the contrast enhancing (CE) core for resection, leaving behind an unresected non-enhancing (NE) invasive rim of tumor which borders healthy brain tissue and invariably contributes to recurrence. Recent multi-omic analyses identified predominant neuronal and immune signatures within NE tumor; understanding these signatures could provide key insights to combatting recurrence and improving patient outcomes. To this end, we utilized a novel graph database method to integrate multi-omic data from a unique dataset of multi-regional spatially-localized biopsies from the CE core and invasive NE rim from a large cohort of patients with HGG.



 159 IDH-wildtype tumor biopsy samples from the NE and CE regions of 74 glioma patients were profiled by bulk RNA sequencing. Differentially expressed genes (DEG) (n=1159) across NE and CE samples were selected using EdgeR. Samples were hierarchically clustered (method = median) for each DEG and represented as nodes in a Neo4j graph network. Edges were generated between nodes sharing ≥ 1 sample. Modularity optimization was used to identify communities of densely connected nodes. Genes occurring exclusively within NE communities were localized by cell type using external single-cell RNA sequencing datasets and selected as candidate source nodes for NE network random walks. RBFOX1 was selected as a source (walk length = 5; 10,000 walks/source node). Random walks traversing samples with significantly elevated RBFOX1 expression (one-tailed test) were kept for analysis. Genes occurring at significantly high frequency (one-tailed test) within random walks were ranked by correlation with RBFOX1 expression in NE samples and analyzed by gene set enrichment analysis (GSEA) for biological processes using clusterProfiler.



 RBFOX1, a neurodevelopmental regulator, was upregulated (logFC = 1.85) in NE samples and uniquely occurred in NE communities. We show RBFOX1 is differential expressed in the aggressive Neuronal glioma subtype and enriched in neuroepithelial/astrocyte subpopulations across single cell HGG datasets. NE network random walks identified 201 DEGs associated with RBFOX1. GSEA revealed upregulation of neurodevelopmental signatures and downregulation of proliferative pathways amongst RBFOX1’s network.



 RBFOX1’s genetic network within NE tumor represents neurodevelopmental processes potentially underlying glioma invasiveness within the NE region. As such, understanding the mechanistic role of key genes within the RBFOX1 network may provide insight into novel strategies for targeting these invasive cells within an aggressive subtype of glioma in the NE rim.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/795027c22420b28550f4273c8f2b9d812ca4be9c" target='_blank'>
              P18.35.B RBFOX1 GENE INFLUENCES NEURONAL SUBTYPE PLASTICITY IN INVASIVE HIGH GRADE GLIOMA CELLS
              </a>
            </td>
          <td>
            M. Flick, V. Stevens, C. Sereduk, C. L. Plaisier, R. F. Barajas, L. Hu, K. Kannan, N. Tran
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Gliomas are primary brain tumors that develop from glial cells within the central nervous system and are among the deadliest human cancers. Glioblastoma (GBM) is the most malignant form of glioma. NLRX1 is an innate immune pattern recognition receptor that exhibits tumor-suppressive and tumor-promoting effects that may be cancer or cell-type, context-dependent, aided by differences in the microenvironment. Here, we report that NLRX1 is differentially expressed in microglia, astrocytes, GBM cell lines, and glioma patient tissues. siRNA-mediated silencing of Nlrx1 decreases the ability of the GBM cell line, LN-229, to proliferate and migrate. Nlrx1-/- GBM cells exhibit attenuated ability to generate 3D spheroids and enhanced capability to form tunneling nanotubes. Moreover, Nlrx1-/- GBM cells show decreased expression of autophagy markers, suggesting that NLRX1 plays a role in maintaining autophagy in GBM. In summary, our findings indicate that NLRX1 may modulate GBM pathophysiology by regulating GBM cell proliferation, migration, and metabolism. We believe our understanding of NLRX1 in GBM pathophysiology paves the potential development of GBM-targeting therapeutics that may delay disease progression and/or improve survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95a9293ab6ceae7c379f276e26000ebf9b80a6b9" target='_blank'>
              Innate Immune Receptor NLRX1: Potential Modulator of Glioblastoma Pathophysiology
              </a>
            </td>
          <td>
            Durgesh Meena, Divya Shivakumar, Sushmita Rajkhowa, Neermita Bhattacharya, Priya Solanki, Shalini Chhipa, Vikas Janu, M. Garg, J. Gosal, Sushmita Jha
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Isocitrate dehydrogenase 1/2 mutant (IDHmt) astrocytoma is considered a T cell-deprived tumor, yet little is known regarding the exact phenotype that relates to T cell exclusion. To aid better understanding of this mechanism, we systematically characterized the immune microenvironment in IDHmt astrocytoma.



 We performed multiplex immunofluorescence (IF) stainings for CD4+ and CD8+ T cells, B cells, tumor associated macrophages (TAMs) and tumor cells on 75 IDHmt astrocytoma samples from which we collected and analyzed 1295 images. We performed bulk RNA sequencing on 158 IDHmt astrocytoma samples, accompanied with single nucleus RNA sequencing and spatial RNA and protein profiling on samples from the same cohort. We integrated and analyzed all datasets to acquire a robust characterization of the microenvironment organization.



 We demonstrated that when present, T cells were either distributed in the tumor stroma, or in the majority of cases, contained in the perivascular space (i.e. T cell cuffs). We found that the tumor stroma surrounding T cell cuffs showed significant local accumulation of gemistocytic tumor cells (GTCs), a tumor cell type that histologically shows a large eosinophilic cytoplasm and ectopically placed nucleus. GTC-high tumors (>20% of all tumor cells) demonstrated enhanced expression of genes related to immune cell migration and activation that were reliably expressed by TAMs and T cells. Notably, we found that GTCs formed a unique transcriptional cell sub-cluster that was part of the ‘astrocyte-like’ tumor cell state. GTCs expressed markers such as CD44, TNC and CRYAB that are specific for glial scarring, a central nervous system specific mechanism for immune cell exclusion. Surrounding T cell cuffs, GTCs co-localized with TAMs that transcriptomically resembled reactive microglia and showed increased expression of the chemo-attractants CCL3 and CCL4. Together, GTCs and reactive microglia expressed receptor-ligand pairs that take part in reactive astrogliosis and glial scarring, such as Osteopontin (SPP1) - CD44 and IL-1β - IL1R1. We validated this spatial pattern on protein level using whole-slide multiplex IF stainings of GTC-low and GTC-high tumors and showed specific local accumulation of CD44 positive GTCs and TAMs around T cell cuffs in GTC high tumors.



 Using different modalities of bulk, spatial and single cell profiling for RNA and protein, we uncovered a previously unknown spatial pattern of co-localizing GTCs and reactive microglia around T cell cuffs in IDHmt astrocytoma. Together, we show that IDHmt astrocytomas organize a cellular network of tumor cells and leukocytes that mimics glial scarring and can aid immune cell exclusion. Characterizations of these complex localized interactions between tumor- and immune cell subsets aids a more specific target selection for immune therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a0b251a3c8047c4a4b52d909d37695b06564f8" target='_blank'>
              OS11.6.A GEMISTOCYTIC TUMOR CELLS AND IMMUNE REACTIVE MICROGLIA PROGRAMMED FOR GLIAL SCARRING CHARACTERIZE T CELL CONTAINMENT IN IDH-MUTANT ASTROCYTOMA
              </a>
            </td>
          <td>
            L. van Hijfte, M. Geurts, I. D. de Heer, S. Ghisai, H. Balcioglu, Y. Hoogstrate, W. Vallentgoed, R. Head, B. Westerman, P. Wesseling, J. Joyce, R. Debets, P. French
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ad6a53d090caa0cff0b86e7a380a1302888db4" target='_blank'>
              Circulating tumor cells: from new biological insights to clinical practice
              </a>
            </td>
          <td>
            Xuyu Gu, Shiyou Wei, Xin Lv
          </td>
          <td>2024-09-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastatic breast cancer remains largely incurable, and the mechanisms underlying the transition from primary to metastatic breast cancer remain elusive. We analyzed the complex landscape of primary and metastatic breast cancer using scRNA-seq data from twenty-three female patients with either primary or metastatic disease to elucidate the genetic and molecular mechanisms underlying changes in the metastatic tumor ecosystem. We identify specific subtypes of stromal and immune cells critical to forming a pro-tumor microenvironment in metastatic lesions, including CCL2+ macrophages, cytotoxic T cells with an exhausted gene signature, and FOXP3+ regulatory T cells. Analysis of cell-cell communication highlights a marked decrease in tumor-immune cell interactions in metastatic tissues, likely strengthening the immunosuppressive microenvironment. In contrast, primary breast cancer samples displayed increased activation of the TNF- signaling pathway via NF-kB, indicating a potential therapeutic target. Our study comprehensively characterizes the transcriptional landscape encompassing primary and metastatic breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f83bcd576506ee2a8774cb0cfd1433b6936e5b1" target='_blank'>
              Single-cell RNA sequencing reveals immunosuppressive pathways associated with metastatic breast cancer
              </a>
            </td>
          <td>
            F. Ozmen, T. Y. Ozmen, A. Ors, M. Janghorban, M. J. Rames, X. Li, F. Behbod, G. B. Mills, H. Mohammed
          </td>
          <td>2024-09-26</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Asymmetric partition of fate determinants during cell division is a hallmark of cell differentiation. Recent works suggested that such a mechanism is hijacked by cancer cells to increase both their phenotypic heterogeneity and plasticity and in turn their fitness. To quantify fluctuations in the partitioning of cellular elements, imaging-based approaches are used, whose accuracy is limited by the difficulty of detecting cell divisions. Our work addresses this gap proposing a general method based on high-throughput flow cytometry measurements coupled with a theoretical frameworks. We applied our method to a panel of both normal and cancerous human colon cells, showing that different kinds of colon adenocarcinoma cells display very distinct extents of fluctuations in their cytoplasm partition, explained by an asymmetric division of their size. To test the accuracy of our population-level protocol, we directly measure the inherited fractions of cellular elements from extensive time-lapses of live-cell laser scanning microscopy, finding excellent agreement across the cell types. Ultimately, our flow cytometry-based method promise to be accurate and easily applicable to a wide range of biological systems where the quantification of partition fluctuations would help accounting for the observed phenotypic heterogeneity and plasticity">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89534ca95bbe7d2c00c135f356aa0991c7b534e4" target='_blank'>
              Robust assessment of asymmetric division in colon cancer cells
              </a>
            </td>
          <td>
            Domenico Caudo, Chiara Giannattasio, Simone Scalise, V. Turris, Fabio Giavazzi, G. Ruocco, Giorgio Gosti, Giovanna Peruzzi, Mattia Miotto
          </td>
          <td>2024-09-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87d5f0ed69b574960ec0cf8e7cfd67555c4fa973" target='_blank'>
              Drug resistant pancreatic cancer cells exhibit altered biophysical interactions with stromal fibroblasts in imaging studies of 3D co-culture models
              </a>
            </td>
          <td>
            Eric Struth, M. Labaf, V. Karimnia, Yiran Liu, Gwendolyn Cramer, Joanna B. Dahl, Frank J Slack, Kourosh Zarringhalam, Jonathan P Celli
          </td>
          <td>2024-09-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Tumours are complex ecosystems comprised of tumour cells and various normal cell types. The interactions among these cells with diverse molecular characteristics lead to intratumoral heterogeneity and cancer evolution. However, little is known about the spatial organisation of the glioma tumour ecosystem, the cellular interactions therein, and their influence on patient outcomes. Spatially resolved transcriptomics, which measures the activity of hundreds of genes in situ, offers new avenues to address these long-standing questions.



 We assembled a clinically annotated cohort of glioma patients spanning 278 specimens from low- and high-grade IDH-mutant astrocytoma, IDH-mutant and 1p/19q co-deleted oligodendroglioma, ependymoma, and glioblastoma. Using tissue microarrays from archival FFPE, we performed spatial transcriptomics profiling with the 10x Xenium platform. A custom panel of 344 genes, selected based on publicly available single-cell RNA sequencing datasets of gliomas, was interrogated.



 The spatial transcriptomics data enabled annotation and spatial mapping of more than 20 tumour microenvironment (TME) cell types and various tumour cell types present across the 5 glioma entities. We observed broadly conserved transcriptional profiles of TME cell types across gliomas, but TME cell type composition and spatial organisation varied depending on tumour entity. In particular, lower amounts of macrophages and microglia as well as oligodendrocytes and oligodendrocyte progenitor cells were observed in oligodendroglioma tumours compared to other entities. High-grade astrocytoma samples contained more myeloid dendritic cells and monocytes in comparison to low-grade astrocytomas. A lower proportion of vascular cells correlated with better overall survival for glioblastomas and low- and high-grade astrocytomas. Tumour cells exhibited mostly entity-specific gene expression, but also distinct tumour cell states including AC-, OPC-, NPC-, and MES-like cell states.



 Our work demonstrates the utility of the 10x Xenium platform for high-resolution spatial profiling of expression in archival FFPE samples. Preliminary mapping of the TME and tumour cell types unravelled a complex spatial architecture of the glioma ecosystems. Distinct compositions of the TME were found in each glioma subtype, which may help understand different mechanisms of malignancy. Associations between cellular spatial patterns and clinical data available for FFPE samples have the potential to provide better insights for improved patient stratification and treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5877e1818dcb8b95627446b49e8f170fc56f766" target='_blank'>
              P01.03.A DECIPHERING THE TUMOUR ECOSYSTEM IN GLIOMAS USING SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            Z. Seferbekova, D. Calafato, G. Rukhovich, A. Patel, M. Ritter, F. Ippen, A. Suwala, F. Sahm, M. Gerstung
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Isocitrate dehydrogenase (IDH) mutant gliomas are progressive, invariably fatal brain tumors affecting young and middle-age adult patients. Previous single-cell transcriptomic characterization of IDH-mutant gliomas has uncovered a heterogeneity of cell states, with a developmental hierarchy that starts with neural progenitor-like stem cells that transition to more differentiated malignant cell states. Despite this understanding, the dynamics of these cell state transitions as tumors undergo grade progression, and the factors that facilitate or restrict cell state transitions remain poorly understood.



 Here, we characterize the cell states of IDH-mutant gliomas via single-cell gene expression (scRNA-seq) and single-cell analyses of accessible chromatin (scATAC-seq) in tumors spanning all grades using fresh patient samples and compare them with transcriptional programs and chromatin accessibility profiles of normal brain cells. We perform mechanism validation experiments using IDH-mutant glioma organoids.



 We find that tumor transcriptional cell states experience a marked shift as tumors undergo progression. In low-grade tumors, predominant cell states mimic slow-cycling oligodendrocyte progenitor-like cells (OPCs). With grade progression, predominant cell states transition to resemble highly proliferative neural progenitor-like cells (NPCs). Additionally, scATAC-seq analyses revealed novel permissive chromatin states, which could facilitate this transition. Genetic analyses identified several molecular alterations in high-grade tumors, including loss of DNA methylation, oncogene amplification, and the deletion of genes associated with the interferon (IFN) response. Those IFN genes, as well as tumor suppressors, are initially repressed by DNA hypermethylation in low-grade tumors. Lastly, we validate the role of DNA hypomethylation in favoring immune evasion with grade progression using an IDH-mutant glioma organoid model.



 Overall, our findings provide evidence that tumor progression in IDH-mutant gliomas is driven by an interplay of epigenetic and genetic mechanisms, with the former being determinant in low-grade tumors and the latter in high-grade tumors. The emergence of genetically-driven mechanisms promotes more mitotically active cell states as well as decreased anti-tumoral immune activation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13fd62dc7724c0acd329992464a530f1dbcd46aa" target='_blank'>
              KS02.6.A SINGLE-CELL MULTIMODAL ANALYSIS OF IDH-MUTANT GLIOMAS UNCOVERS SHIFT FROM EPIGENETIC TO GENETIC DRIVERS WITH TUMOR PROGRESSION
              </a>
            </td>
          <td>
            L. G. Gonzalez Castro, J. Wu, S. Battaglia, C. El Farran, J. D’Antonio, T. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 Introduction: Pediatric sarcomas are the second most common solid tumor in children and adolescence. Despite advances in multimodal therapies, one-third of sarcoma patients do not survive the disease. The development of new treatments for sarcomas is hindered by a limited understanding of the tumor microenvironment (TME), especially the role of immune cells such as tumor-associated macrophages (TAMs). The abundance of TAMs significantly impacts the clinical outcomes of pediatric sarcoma patients, yet our knowledge of their molecular heterogeneity and interactions with other cell types in the TME remains limited. Methods: To characterize the transcriptomic and epigenomic heterogeneity of sarcoma TME, we profiled 32 treatment-naïve tumor samples from 5 pediatric sarcoma subtypes using the single-nuclei RNA+ATAC-Seq multiome assay. We also interrogated the spatial organization of the sarcoma TME using the multiplexed immunohistochemistry assay, Co-detection by indexing (CODEX), with a focus on the diversity and inter-cellular interactions of TAMs. We utilized in vitro co-culture models of human monocyte-derived macrophages, CD8 T cells and malignant sarcoma cells to validate immunosuppressive signaling interactions between TAMs and CD8 T cells predicted based on our multiome and CODEX data. Results: We sequenced 175,964 and 178,000 high-quality cells from RNA and ATAC modality respectively. Analysis of cell type frequencies across sarcoma subtypes showed that TAMs were the most abundant immune cells in the TME. We identified seven TAM subpopulations with distinct gene expression and epigenomic signatures. Among these, SPP1+ and C1QC+ TAM subsets exhibited higher immunosuppressive potential. By integrating gene expression and chromatin accessibility data, we identified key transcription factors that drive the transcriptional phenotypes of SPP1+/C1QC+ TAMs, including RUNX1 and ELF2. By ligand-receptor analysis using our snRNA-Seq data, we identified signaling ligand-receptor pairs that potentially mediate immunosuppression by SPP1+/C1QC+ TAMs, including SPP1-CD44 and HLA-E-KLRC1. Additionally, interactions like SEMA3D-NRP1 and LTBP3-ITGB5 between malignant cells and SPP1+/C1QC+ TAMs were observed in multiple sarcoma subtypes and significantly associated with inferior prognosis. Furthermore, our CODEX data revealed spatial co-localization between SPP1+TAMs and T cells, further corroborating the immune regulatory role of SPP1+TAMs in the sarcoma TME. Finally, our in vitro experiments revealed that macrophages co-cultured with sarcoma cell lines (RD and RH30) showed increased expression of C1QC/SPP1+ TAM markers, including CD163 and CD109, and demonstrated enhanced immunosuppressive effect on CD8+ T cells. Conclusion: This study presents a comprehensive single-cell multi-omics and spatial omics analysis of the pediatric sarcoma microenvironment. By identifying specific pro-tumorigenic TAM subsets and signaling interactions associated with inferior patient outcome, this study reveals novel insights into the role of TAMs in pediatric sarcomas.
 Citation Format: Zhan Zhang, Kyung Jin Ahn, Rumeysa Biyik-Sit, Changya Chen, Anusha Thadi, Chia-hui Chen, William Molina, Brian Lockhart, Theodore Laetsch, Lea Surrey, Malay Haldar, Kathrin Bernt, Vinodh Pillai, Kai Tan. Dissecting pediatric sarcoma microenvironment using single-cell and spatial multi-omics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B061.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea4e7ffe970f4c90d3fbc5950148fc0b9f42efbd" target='_blank'>
              Abstract B061: Dissecting pediatric sarcoma microenvironment using single-cell and spatial multi-omics
              </a>
            </td>
          <td>
            Zhan Zhang, Kyung Jin Ahn, Rumeysa Biyik‐Sit, Changya Chen, Anusha Thadi, Chia-Hui Chen, William Molina, Brian Lockhart, Theodore W Laetsch, Lea Surrey, Malay Haldar, Kathrin M Bernt, Vinodh Pillai, Kai Tan
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Hotspot mutations in the IDH genes serve as tumor-initiating events that define a subset of adult diffuse glioma that can further be divided into either oligodendroglioma or astrocytoma subtypes. Despite standard treatment, IDH-mutant gliomas inevitably recur. Previous studies suggest that therapeutic resistance may result from a combination of intratumoral cellular heterogeneity, epigenetic evolution, and acquired genetic alterations. However, how these multilayered molecular features interact to influence the evolutionary paths of IDH-mutant gliomas remains incompletely understood.



 To chart this evolution, 75 glioma samples were longitudinally collected from 35 patients (n = 13 oligodendroglioma, n = 22 astrocytoma) and profiled using single nucleus RNA sequencing, ATAC sequencing, and bulk DNA sequencing.



 Analysis of 331,016 nuclei (snRNA) revealed eight tumor microenvironment cell types and five pan-IDH-mutant malignant cellular states. The malignant states were distributed along a cellular hierarchy of stem-like populations (neural progenitor cell-like, oligodendrocyte progenitor cell-like, and undifferentiated) and more differentiated populations (mesenchymal-like and astrocyte-like) with cycling cells enriched in the stem-like populations. Joint open chromatin accessibility data (snATAC) was available for a subset of malignant cells (71,088 nuclei) and state-specific differentially accessible peaks supported that these states are epigenetically encoded. Across both subtypes, higher tumor grade was associated with reduced malignant cell differentiation and increased cycling populations. Between the two time points, we identified that there was a longitudinal increase in the cycling, undifferentiated, and mesenchymal-like populations with a corresponding decrease in the astrocyte-like population. Longitudinal genetic analysis revealed that 19 of 35 tumors acquired at least one of the following genetic alterations: treatment-associated hypermutation, cell cycle alteration, or large changes in copy number alterations. Importantly, tumors that acquired these key genetic alterations demonstrated significant shifts towards reduced differentiation and increased cycling populations while those tumors that did not had more stable malignant profiles.



 Collectively, our results suggest a common cellular hierarchy across IDH-mutant gliomas with a shift towards reduced differentiation, increased cycling populations during disease progression that is driven by acquired genetic alterations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d952023b3d2c6da79f53a908951847332e2e66b4" target='_blank'>
              KS02.7.A LONGITUDINAL SINGLE-CELL ANALYSES IDENTIFY DRIVERS OF GENETIC, EPIGENOMIC, AND CELLULAR EVOLUTION IN IDH-MUTANT GLIOMA
              </a>
            </td>
          <td>
            K. Johnson, A. Spitzer, F. Varn, M. Nomura, L. Garofano, T. Chowdhury, K. J. Anderson, F. D’Angelo, L. Bussema, S. Gritsch, Y. Oh, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, A. Lipsa, F. Hertel, Z. Zhao, Q. Wang, T Jiang, B. Hermes, N. Sanai, A. Gołȩbiewska, S. Niclou, J. Huse, W. Yung, A. Lasorella, M. Suvà, A. Iavarone, I. Tirosh, R. Verhaak
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Chimeric antigen receptor T cell therapies have achieved great success in eradicating some liquid tumors, whereas the preclinical results in treating solid tumors have proven less decisive. One of the principal challenges in solid tumor treatment is the physical barrier composed of a dense extracellular matrix, which prevents immune cells from penetrating the tissue to attack intratumoral cancer cells. Here, we improve immune cell infiltration into solid tumors by manipulating septin-7 functions in cells. Using protein allosteric design, we reprogram the three-dimensional structure of septin-7 and insert a blue light-responsive light-oxygen-voltage-sensing domain 2 (LOV2), creating a light-controllable septin-7-LOV2 hybrid protein. Blue light inhibits septin-7 function in live cells, inducing extended cell protrusions and cell polarization, enhancing cell transmigration efficiency through confining spaces. We genetically edited human natural killer cell line (NK92) and mouse primary CD8+ T-cells expressing the engineered protein, and we demonstrated improved penetration and cytotoxicity against various tumor spheroid models. Our proposed strategy to enhance immune cell infiltration is compatible with other methodologies and therefore, could be used in combination to further improve cell-based immunotherapies against solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ea19d2e43098962f3eaf53cfad11c6cdf08f3ff" target='_blank'>
              Optogenetically engineered Septin-7 enhances immune cell infiltration of tumor spheroids.
              </a>
            </td>
          <td>
            Jiaxing Chen, Brianna Hnath, Congzhou M. Sha, Lynne Beidler, Todd D Schell, Nikolay V. Dokholyan
          </td>
          <td>2024-10-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6866382555382de9b8b8e3c83922b3e82d715085" target='_blank'>
              Defining heritability, plasticity, and transition dynamics of cellular phenotypes in somatic evolution.
              </a>
            </td>
          <td>
            Joshua S. Schiffman, Andrew R D'Avino, Tamara Prieto, Yakun Pang, Yilin Fan, Srinivas Rajagopalan, Catherine Potenski, Toshiro Hara, M. Suvà, Charles Gawad, Dan A Landau
          </td>
          <td>2024-09-24</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 The structured layers of the human neocortex stand in sharp contrast to the disordered arrangement found in malignant CNS tumors. Glioblastoma (GBM), with its diverse characteristics, presents formidable hurdles in both research and treatment, as successful approaches demand a uniform biological understanding. In this study, we capitalize on recent advances in spatial biology, employing integrated spatially resolved transcriptomics and metabolomics to identify the factors driving the unique architectural changes observed in GBM.



 We performed spatial metabolomics (MALDI) and transcriptomics (Visium n=43 and MERFISH n=25) of glioblastoma patients. Computational analysis was performed with the SPATA2 R package.



 We found that most spatially recurrent pattern in glioblastoma emerge in a gradient-like organization,particularly influenced by proximity to areas of necrosis and hypoxia. Samples without hypoxic regions demonstrated less spatial organization. Metabolomic and transcriptomic profiling revealed enhanced responses to hypoxia coupled with reduced cell proliferation and elevated expression indicative of the S-phase in cell division in regions near necrosis. The vicinity to hypoxic areas appeared to be associated with increased chromosomal alterations as well as extracellular matrix remodeling and epithelial-to-mesenchymal transition, implying that the invasive capabilities of GBM cells originate from these hypoxic niches. By further integrating spatially resolved TCR-sequencing, we identified that hypoxic gradients drive defined T cell response with higher likelihood of T cell dysfunction within a distance of 235 μm (sd: 132 μm) to hypoxia. This suggests that the presence of necrosis, hypoxia, and hypoxia associated metabolites could serve as potential predictive markers for T-cell behavior and exhaustion in glioblastoma.



 Our investigation provides a comprehensive examination of the impact of two common environments in glioblastoma: necrosis and hypoxia. We demonstrate that the intricate structure of glioblastoma is not merely shaped by genetic chance and random events. Rather, it is substantially molded by spatial proximity to critical elements such as necrosis and hypoxia. This understanding contributes to an advancing framework aimed at unraveling the complex biology of glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e118b0e7f81b796b7529c20a3b8d7d117791b7f5" target='_blank'>
              OS07.3.A DYNAMIC METABOLIC SHIFTS RESHAPE GLIOBLASTOMA: IMPACTS ON SPATIAL STRUCTURE AND IMMUNE RESPONSE
              </a>
            </td>
          <td>
            N. Conrad, J. Benotmane, G. Villa, O. Schnell, D. Heiland, J. Kueckelhaus
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is the third leading cause of cancer-related death in the US. This poor prognosis is related to the complex tumor microenvironment and tumor heterogeneity that drives resistance to chemotherapy. Here the heterogeneity of the stromal, immune, and malignant epithelial cell populations enables signaling and metabolic crosstalk mechanisms that drive immune suppression and resistance to therapy. Recently, our group unveiled that multiple distinct metabolic subclasses of cancer cells coexist within individual PDAC tumors. Among these, two PDAC clonal populations can engage in metabolic symbiosis that supports mitochondrial metabolism. One subtype has a constitutively high integrated stress response (ISR), enabling resistance to mitochondrial inhibitors. Further, these cells release asparagine that supports the mitochondrial metabolism of the other subtype that are more sensitive to mitochondrial inhibition. Given these observations, we are developing approaches to disrupt this metabolic symbiosis. Clonal PDAC cells with constitutively active ISR exhibit increased sensitivity to inhibition of the redox activity of Redox factor 1/Apurinic-apyrimidinic endonuclease 1 (Ref-1). Importantly, this can be leveraged to disrupt the support of the mitochondrial metabolism of clonal symbiotic pairs. Moreover, polyclonal PDAC cell lines representative of the heterogeneity seen in PDAC patients show a highly synergistic response to the combination of Ref-1 redox inhibitors with mitochondrial inhibitors. These data suggest this treatment combination will be synergistic in polyclonal tumors that exist in most PDAC patients. Overall, these data have the potential to lead to new avenues to directly target heterogenous populations of cancer cells to treat pancreatic cancer- an urgent clinical need.
 Citation Format: Taryn Morningstar, Cecily Anaraki, Lorenzo Scipioni, Giulia Tedeschi, Michelle Digman, Aimee Edinger, Claus Jorgensen, Melissa Fishel, Christopher J Halbrook. Targeting metabolic heterogeneity in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C045.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0412df26641d71ff1a5183fc1b222c74aa39cae6" target='_blank'>
              Abstract C045: Targeting metabolic heterogeneity in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Taryn Morningstar, Cecily Anaraki, Lorenzo Scipioni, Giulia Tedeschi, M. Digman, Aimee Edinger, Claus Jorgensen, M. Fishel, Christopher J. Halbrook
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract The heterogeneity of tumor clones drives the selection and evolution of distinct tumor cell populations, resulting in an intricate and dynamic tumor evolution process. While tumor bulk DNA sequencing helps elucidate intratumor heterogeneity, challenges such as the misidentification of mutation multiplicity due to copy number variations and uncertainties in the reconstruction process hinder the accurate inference of tumor evolution. In this study, we introduce a novel approach, REconstructing Tumor Clonal Heterogeneity and Evolutionary Relationships (RETCHER), which characterizes more realistic cancer cell fractions by accurately identifying mutation multiplicity while considering uncertainty during the reconstruction process and the credibility and reasonableness of subclone clustering. This method comprehensively and accurately infers multiple forms of tumor clonal heterogeneity and phylogenetic relationships. RETCHER outperforms existing methods on simulated data and infers clearer subclone structures and evolutionary relationships in real multisample sequencing data from five tumor types. By precisely analysing the complex clonal heterogeneity within tumors, RETCHER provides a new approach to tumor evolution research and offers scientific evidence for developing precise and personalized treatment strategies. This approach is expected to play a significant role in tumor evolution research, clinical diagnosis, and treatment. RETCHER is available for free at https://github.com/zlsys3/RETCHER.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f822108aa29d6372cc1d56eba22e5f8f0a1fffc3" target='_blank'>
              Reconstructing tumor clonal heterogeneity and evolutionary relationships based on tumor DNA sequencing data
              </a>
            </td>
          <td>
            Zhen Wang, Yanhua Fang, Ruoyu Wang, Liwen Kong, Shanshan Liang, Shuai Tao
          </td>
          <td>2024-09-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) disease progression involves complex cell state transitions in both malignant and non-malignant populations within the tumor microenvironment. To uncover PDAC cell states and epigenetic features associated with clinical outcome, we profiled 40 tumor samples (33 primary tumors and 7 metastases) from 39 patients with single cell ATAC-seq, along with 10 sample-matched single cell RNA-seq profiles. Through topic modeling of high dimensional chromatin data, we uncovered distinct cis-regulatory networks underlying the classical and basal-like molecular subtypes of PDAC, which showed differential activity in basal-classical hybrid cells. Furthermore, delineation of epigenetic alterations by basal, classical, and hybrid subtypes revealed a core set of chromatin regions opened in all PDAC subtypes relative to ductal and acinar cells. These universally open regions showed striking overlap with gastrointestinal enhancers, highlighting a common endoderm lineage infidelity process across pancreatic tumors. Within the stromal compartment, we identified novel regulators of myofibroblastic cancer-associated fibroblasts (myCAFs) and inflammatory CAFs (iCAFs) from enhancer networks active in each CAF subtype. Importantly, our data uncover a highly prognostic gene program active in only one of two distinct myCAF chromatin states found in our data, highlighting a putative tumor-promoting myCAF subpopulation. Our work provides a high-resolution description of epigenetic cell state heterogeneity in PDAC, revealing novel regulators and gene programs associated with PDAC subtypes and clinical outcome.
 Citation Format: Kevin MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, Carl Pelz, Trent Waugh, Ethan Agritelle, John Muschler, Andrew Adey, Rosalie Sears. Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55da9db59167776bfc809f1ebb52c0cfa3226d09" target='_blank'>
              Abstract C078: Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes
              </a>
            </td>
          <td>
            Kevin A MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, C. Pelz, Trent Waugh, Ethan Agritelle, John L Muschler, Andrew C. Adey, Rosalie C. Sears
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c56d7dfef26c62111d5f106e161ec720c881a32b" target='_blank'>
              A new exploration: characterization of the differentiation trajectory of prostate cancer cells
              </a>
            </td>
          <td>
            Jiyu Yang, Changyou Wang, Xiao Ma, Jie Li, Haoran Yuan, Renzhen Tan, Ling Ling, Xianchun Zhou
          </td>
          <td>2024-09-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A variety of abnormal epithelial cells and immature and mature immune cells in thymic epithelial tumors (TETs) affect histopathological features, the degree of malignancy, and the response to treatment. Here, gene expression, trajectory inference, and T cell antigen receptor (TCR)-based lineage tracking are profiled in TETs at single-cell resolution. An original subpopulation of KRT14+ progenitor cells with a spindle cell phenotype is shown. An abnormal infiltration of immature T cells with a TCR hyper-rearrangement state is revealed, due to the lack of CCL21+ medullary epithelial cells. For thymic carcinoma, the novel biomarkers of MSLN, CCL20, and SLC1A5 are identified and observed an elevated expression of LAG3 and HAVCR2 in malignant tumorn-infiltrating mature T cells. These common features based on the single-cell populations may inform pathological reclassification of TETs. Meanwhile, it is found that macrophages (MACs) attract thymic tumor cells through the LGALS9-SLC1A5 axis, providing them with glutamine to elicit metabolic reprogramming. This MAC-based metabolic pattern can promote malignancy progression. Additionally, an interactive immune environment in TETs is identified that correlates with the infiltration of abnormal FOXI1+ CFTR- ionocytes. Collectively, the data broaden the knowledge of TET cellular ecosystems, providing a basis for tackling histopathological diagnosis and related treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a22718f78c121fcb0eb8985d234fb77b5499ccc3" target='_blank'>
              Abnormal Cellular Populations Shape Thymic Epithelial Tumor Heterogeneity and Anti-Tumor by Blocking Metabolic Interactions in Organoids.
              </a>
            </td>
          <td>
            Xuefei Liu, Changchun Wang, Yueyu Huang, Qiaoli Lv, Chang Yu, Jianghua Ying, Lianhui Duan, Yangzhong Guo, Guanyin Huang, Wenhui Shen, Ming Jiang, Weimin Mao, Zhixiang Zuo, An Zhao
          </td>
          <td>2024-09-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Although the combination of immunotherapy and radiotherapy (RT) for the treatment of malignant tumors has shown rapid development, the insight of how RT remodels the tumor microenvironment to prime antitumor immunity involves a complex interplay of cell types and signaling pathways, much of which remains to be elucidated. Four tumor samples were collected from the same abdominal wall metastasis site of the patient with gastric cancer at baseline and during fractionated RT for single-cell RNA and T-cell receptor sequencing. The Seurat analysis pipeline and immune receptor analysis were used to characterize the gastric cancer metastasis ecosystem and investigated its dynamic changes of cell proportion, cell functional profiles and cell-to-cell communication during RT. Immunohistochemical and immunofluorescent staining and bulk RNA sequencing were applied to validate the key results. We found tumor cells upregulated immune checkpoint genes in response to RT. The infiltration and clonal expansion of T lymphocytes declined within tumors undergoing irradiation. Moreover, RT led to the accumulation of proinflammatory macrophages and natural killer T cells with enhanced cytotoxic gene expression signature. In addition, subclusters of dendritic cells and endothelial cells showed decrease in the expression of antigen present features in post-RT samples. More ECM component secreted by myofibroblasts during RT. These findings indicate that RT induced the dynamics of the immune response that should be taken into consideration when designing and clinically implementing innovative multimodal cancer treatment regimens of different RT and immunotherapy approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/523682bb135dcad73fb3cd605aea7d93bedb4b8c" target='_blank'>
              Dynamic single-cell sequencing unveils the tumor microenvironment evolution of gastric cancer abdominal wall metastases during radiotherapy.
              </a>
            </td>
          <td>
            Qianqian Mao, Zhenzhen Wu, Yonghong Lai, Ling Wang, Qiongzhi Zhao, Xi Xu, Xiansheng Lu, Wenjun Qiu, Zhi-Huan Zhang, Jiani Wu, Gaofeng Wang, R. Zhou, Jianhua Wu, Huiying Sun, N. Huang, Xiatong Huang, Luyang Jiang, Yiran Fang, Yuyun Kong, Li Liang, J. Bin, Yulin Liao, Min Shi, Wangjun Liao, D. Zeng
          </td>
          <td>2024-09-26</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) that is refractory to treatment frequently leads to extensive systemic and multi-organ metastases. PDAC is characterized as an immune “cold” cancer; however, the persistence of this “cold” tumor immune microenvironment (TIME) across distant metastases, and the mechanisms by which metastases influence their respective TIME, remain poorly understood. In this study, we conducted spatial protein expression profiling of the TIME using the Nanostring GeoMx Digital Spatial Profiling (DSP) platform. We employed a 43- plex panel on 54 tissue samples, including 14 primary tumors and 40 matched metastases from organs commonly affected by PDAC metastases (liver, lung, and peritoneum), obtained through a rapid (“warm”) autopsy program. We quantified protein expression in three cellular compartments: PanCK+ epithelial cells, α-SMA+ fibroblasts, and CD45+ immune cells, across 181 regions of interest (ROIs). Our analysis revealed a significant enrichment of CD45+ immune cells in lung metastases compared to other organ sites, with a notably lower presence of these cells in liver metastases. Focusing on the immune cell compartment, we analyzed the CD45 segments for further investigation. After normalization using housekeeping proteins, including Histone H3, GAPDH, and ribosomal protein S6, we found that the immune profiles were most distinct between liver and lung metastases. Specifically, liver metastases exhibited the greatest deviation in immune profile compared to primary PDAC, lung, and peritoneal metastases. We scored protein expression signatures for various immune cell subsets based on specific marker expressions. When comparing liver to lung metastases, scores for T cells, myeloid cells, checkpoint markers, macrophages, and interferon signaling were significantly lower in the liver, underscoring the immune “coldness” of the liver metastasis environment. Our cohort included six untreated patients and nine patients treated with chemotherapies. We compared protein expressions in each organ site between untreated and treated patients. Notably, treatment induced divergent expression patterns of multiple proteins between liver and lung metastases. While treatment significantly reduced the expression of several immune markers, including CD45 (total immune), CD3 (T cell), CD4 (T helper cell), CD45RO (memory T cell), CD163 (M2 macrophage), and B7-H3 (checkpoint marker), in lung metastases, these markers were significantly upregulated in liver metastases of treated patients. This finding indicates a pronounced divergence in TIME modulation in response to treatment between liver and lung metastases, potentially fostering a treatment-resistant environment in PDAC patients with multi- organ metastasis. Consequently, this urges the need to develop tailored therapeutic strategies for PDAC metastasis, taking into account the distinct immune landscapes of different metastasis sites.
 Citation Format: Jimin Min, Vittorio Branchi, Kimal I Rajapakshe, Nisha Narayanan, Guangsheng Pei, Linghua Wang, Jean L Grem, Thomas C Caffrey, Paul M Grandgenett, Michael A Hollingsworth, Paola A Guerrero, Anirban Maitra. Organ-specific diversity of tumor immune microenvironment in metastatic pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B073.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd4226ecd000fb900607928bb9c305d2b12ea81" target='_blank'>
              Abstract B073: Organ-specific diversity of tumor immune microenvironment in metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Jimin Min, Vittorio Branchi, K. Rajapakshe, Nisha Narayanan, Guangsheng Pei, Linghua Wang, Jean L. Grem, T. Caffrey, P. Grandgenett, Michael A Hollingsworth, Paola A Guerrero, Anirban Maitra
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a68f84c9fb8e36a5e9dc6d992ffaa775efbb85db" target='_blank'>
              SMAC mimetic drives microglia phenotype and glioblastoma immune microenvironment
              </a>
            </td>
          <td>
            E. Snacel-Fazy, A. Soubéran, M. Grange, Kevin Joseph, C. Colin, Philippe Morando, H. Luche, A. Pagano, S. Brustlein, F. Debarbieux, Soline Toutain, C. Siret, S. A. van de Pavert, G. Rougon, D. Figarella-Branger, V. Ravi, E. Tabouret, A. Tchoghandjian
          </td>
          <td>2024-09-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cancer is characterized by uncontrolled cell growth and spread into surrounding tissue. According to the North American Association of Central Cancer Registries (NAACCR) and the National Center for Health Statistics, there were 1,918,030 cancer cases and a total of 609,360 deaths estimated in the US in 2022. Although the direct cause of cancer is not fully understood, it is recognized that the immune system plays a major role in the development and progression of tumors. Investigating tumor-associated macrophages (TAMs) is crucial for understanding the immunosuppressive role that they may play in the tumor microenvironment, which makes them a target for immunotherapies to combat cancer. However, angiogenesis, tumor initiation and invasion promoting signals prevent gaining deeper insight into the intricate molecular mechanisms involved in TAMs and the tumor microenvironment. While researchers in the past have used population samples to analyze cells, recent efforts have incorporated single-cell technologies to study the complex mixture of cells in a tumor, monitor the effects of immunotherapies and more. With the revolutionary advent of the first single-cell RNA sequencing (scRNA-seq), researchers have since been able to gain insight into single-cell data with unprecedented accuracy and develop new immunotherapies through different single-cell technologies, such as polymethylsiloxane (PDMS) and Multiplexed Ion Beam Imaging (MIBI) multiplexing. In this review, we underline the benefits of adopting single-cell technologies to identify distinct cell types in complex environments (i.e., tumors) and highlight the potential application of these technologies along with various multiplexing techniques to immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64f7bdccd7146f521a4c249388cdbf8225adf0ff" target='_blank'>
              A Review on Single-Cell analysis of Macrophages in Cancer
              </a>
            </td>
          <td>
            Isaiah Yim, Amirali Assylkhan
          </td>
          <td>2024-09-17</td>
          <td>STEM Fellowship Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The role of fibroblasts in determining tissue topography and immune cell organisation within chronically inflamed tissues is poorly understood. Herein, we use multi-omic spatial analysis to define the cellular zonation pattern of the synovium in patients with inflammatory arthritis, identifying discrete tissue niches underpinned by spatially programmed subsets of synovial fibroblasts. We observe that perivascular fibroblasts switch on distinct matrix programs in response to cytokine signalling from neighbouring cells, forming adapted tissue niches that either permit or restrict immune cell trafficking. Specifically, IFN-γ−responsive fibroblasts form a pathogenic lymphocyte-permissive niche that supports the persistence of leukocytes in the tissue, whilst TGF-β−responsive, matrix-synthesising fibroblasts comprise a reparative niche, composed of a collagen-rich barrier around blood vessels that restricts leukocyte migration and promotes resolution of tissue inflammation. Augmentation of such endogenous pathways to promote resolution of inflammation may offer therapeutically tractable approaches for restoration of tissue homeostasis. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149e36f3b7492866ab6678df91b844862a78208e" target='_blank'>
              Spatial programming of fibroblasts promotes resolution of tissue inflammation through immune cell exclusion
              </a>
            </td>
          <td>
            Patricia Reis Nisa, Christopher B Mahony, Annie Hackland, Elizabeth Clay, Paulynn Suyin Chin, Lucy-Jayne Marsh, Chrissy Bolton, S. Kemble, Charlotte G. Smith, Georgiana Neag, Ruchir Singh, Jason D Turner, A. Filer, Karim Raza, Jean-Baptiste Richard, K. Midwood, C. D. Buckley, Kevin Wei, H. McGettrick, Mark Coles, Adam P. Croft
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Abstract Tumor‐associatedmacrophages (TAMs) exhibit remarkable heterogeneity in glioblastoma. Spatially resolved single‐cell transcriptomic studies identified a monocyte‐derived TAM subset localized in the peri‐necrotic niche, driven by hypoxic cues to acquire ahypoxia response signature. These hypoxia‐TAMs destabilize endothelial adherens junctions through adrenomedullin paracrine signaling, promoting the formation of hyperpermeable neovasculature that impedes drug delivery. Blocking adrenomedullin produced by hypoxia‐TAMs restores vascular integrity, increases drug deliveryinto tumors, and provides combinatorial therapeutic benefits. Here we discuss the heterogeneity of TAMs regarding functional states and locations in glioblastomas, and propose future directions for studying the temporospatial dynamics of multifaceted TAM. Highlights Single‐cell omics reveal a functionally and spatially distinct hypoxia‐TAM subset in glioblastoma. Adrenomedullin secreted by hypoxia‐TAM destabilizes tumor vasculature and its blockade enhances vessel integrity and drug delivery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989fdc1e11579a80529898e9b0f1ec0aa652779e" target='_blank'>
              Identification of hypoxic macrophages in glioblastoma: Unveiling therapeutic insights from tumour microenvironment analysis
              </a>
            </td>
          <td>
            Zhen Qin, Xiu-Wu Bian, Yu Shi
          </td>
          <td>2024-09-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Gliomas are malignant tumors of neuronal support cells within the central nervous system (CNS) and are characterized by poor overall prognoses and limited treatment options due to their infiltrative growth patterns. The neural tumor microenvironment, composed of benign neurons, neuroglia, endothelial cells, and intravascular white blood cells, is a target-rich site for potential chemotherapeutic agents. This study assessed cell proliferation rates, white blood cell components, and a limited number of nuclear, cytoplasmic, and membrane markers using immunohistochemistry (IHC) assays on formalin-fixed and paraffin-embedded benign and glial tumor tissue samples from the CNS. It was observed that glioma tissues had increased rates of glial cell proliferation and significant increases in the number of observed T-lymphocytes and granulocytes but decreased expression of markers Somatostatin receptor 2 (SSTR2), L1 cell adhesion molecule (L1CAM), and GATA binding protein 3 (GATA3) when compared to benign tissue samples. Understanding the lack of protein expression and population expansion potential of the glioma microenvironment in greater detail could help identify valuable therapeutic target combinations for future treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f85d962de2c676129924d90f82a63311b9f2192e" target='_blank'>
              The impact of gliomas on the normal brain microenvironment: a pilot study.
              </a>
            </td>
          <td>
            Nicole Riha, Jacen S Moore, Sheila L Criswell
          </td>
          <td>2024-10-01</td>
          <td>Journal of histotechnology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="A significant challenge in analyzing spatial transcriptomics data is the effective and efficient detection of spatially variable genes (SVGs), whose expression exhibits non-random spatial patterns in tissues. Many SVGs show spatial variation in expression that is highly correlated with cell type categories or compositions, leading to the concept of cell type-specific spatially variable genes (ctSVGs). Existing statistical methods for detecting ctSVGs treat cell type-specific spatial effects as fixed effects when modeling, resulting in a critical issue: the testing results are not invariant to the rotation of spatial coordinates. Additionally, an SVG may display random spatial patterns within a cell type, and a ctSVG may exhibit random spatial patterns from a general perspective, indicating that an SVG does not necessarily have to be a ctSVG, and vice versa. This poses challenges in real analysis when detecting SVGs or ctSVGs. To address these problems, we propose STANCE, a unified statistical model developed to detect both SVG and ctSVGs in spatial transcriptomics. By integrating gene expression, spatial location, and cell type composition through a linear mixed-effect model, STANCE enables the identification of both SVGs and ctSVGs in an initial stage, followed by a second stage test dedicated to ctSVG detection. Its design ensures robustness in complex scenarios and the results are spatial rotation invariant. We demonstrated the performance of STANCE through comprehensive simulations and analyses of three public datasets. The downstream analyses based on ctSVGs detected by STANCE suggest promising future applications of the model in spatial transcriptomics and various areas of genome biology. A software implementation of STANCE is available at https://github.com/Cui-STT-Lab/STANCE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d67f5df398d264ed4c0dfa9fbb81e5f7b52f6c2a" target='_blank'>
              STANCE: a unified statistical model to detect cell-type-specific spatially variable genes in spatial transcriptomics
              </a>
            </td>
          <td>
            Haohao Su, Yuesong Wu, Bin Chen, Yuehua Cui
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNAseq) studies have unveiled large transcriptomic heterogeneity within both human and mouse dermal fibroblasts, but a consensus atlas that spans both species is lacking. Here, by studying 25 human and 9 mouse datasets through a semi-supervised procedure, we categorize 15 distinct human fibroblast populations across 5 main axes. Analysis of human fibroblast markers characteristic of each population suggested diverse functions, such as position-dependent ECM synthesis, association with immune responses or structural roles in skin appendages. Similarly, mouse fibroblasts were categorized into 17 populations across 5 axes. Comparison of mouse and human fibroblast populations highlighted similarities suggesting a degree of functional overlap, though nuanced differences were also noted: transcriptomically, human axes seem to segregate by function, while mouse axes seem to prioritize positional information over function. Importantly, addition of newer datasets did not significantly change the defined population structure. This study enhances our understanding of dermal fibroblast diversity, shedding light on species-specific distinctions as well as shared functionalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5503e605fc80d07b2d3beb41daf9c339ba1c0401" target='_blank'>
              A consensus single-cell transcriptomic atlas of dermal fibroblast heterogeneity
              </a>
            </td>
          <td>
            A. M. Ascensión, A. Izeta
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background Medulloblastoma (MB) is a pediatric brain tumor that develops in the cerebellum, representing one of the most common malignant brain cancers in children. Standard treatment includes surgery, chemotherapy, and radiation, but despite a 5-year survival rate of approximately 70%, these therapies often lead to significant neurological damage in the developing brain. This underscores the urgent need for less toxic, more effective therapeutic alternatives. Recent advancements in cancer immunotherapy, including immune checkpoint inhibitors and CAR-T cell therapy, have revolutionized cancer treatment. One promising avenue is the use of Gamma Delta (γδ) T cells, a unique T cell population with potential advantages such as non-alloreactivity, potent tumor cell lysis, and broad antigen recognition. However, their capacity to recognize and target MB cells remains underexplored. Methods To investigate the therapeutic potential of γδT cells against MB, we analyzed the proportion and status of MB-infiltrated γδ T cells within patient datasets. We next investigated the expression of γδ T cell ligands on MB cells. Finally, we generated various human γδ T cell subpopulations and evaluated their activation potential and cytotoxicity against a panel of MB cell lines and patient derived cells. Results Our findings identified Ephrin-A2 receptor and the phosphoantigen/Butyrophilin complex as key ligands, activating Vγ9Vδ1 and Vγ9Vδ2 T cells, respectively, leading to significant MB cell lysis in both monolayer and spheroid models. Importantly, preliminary safety data showed that γδT cells did not target differentiated neurons or neuroepithelial stem cells derived from induced pluripotent stem cells, underscoring the selectivity and safety of this approach. Conclusion In conclusion, γδT cells trigger an efficient and specific killing of MB, and would offer a promising novel therapeutic strategy, with potential for broader application to other pediatric brain tumors. Key messages Medulloblastoma patients often experience significant long-term side effects from current standard treatments. Immunotherapy has emerged as a promising alternative to conventional therapeutic approaches. In our study, we demonstrated that γδ T cells can efficiently and specifically target medulloblastoma cells without causing harm to healthy neuronal tissue. These findings suggest that γδ T cell therapy may provide therapeutic benefits while potentially reducing treatment-related toxicity in medulloblastoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7728d8e6eb2e9fe32658d85f902861981a9e4d1e" target='_blank'>
              Ephrin-A2 and Phosphoantigen-Mediated Selective Killing of Medulloblastoma by γδT Cells Preserves Neuronal and Stem Cell Integrity
              </a>
            </td>
          <td>
            Lola Boutin, Mingzhi Liu, Julie Déchanet Merville, Oscar Bedoya-Reina, M. Wilhelm
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) patients with tumors enriched for the basal-like molecular subtype exhibit severe resistance to standard of care treatments and have significantly worse overall survival, compared to patients with classical subtype enriched tumors. It is therefore important to develop resources that allow identification of novel targets in a statistically rigorous way to overcome therapeutic challenges. Here we compiled a single cell RNA sequencing (scRNAseq) atlas of the human pancreas with 229 individual patient samples, aggregated from raw data obtained from published studies. We mapped cell-type specific scRNAseq gene signatures in newly generated (n=12) and existing (n=13) spatial transcriptomics (ST) data from PDAC samples. Analysis of tumor cells from our scRNAseq atlas revealed nine transcriptionally distinct populations, two of which strongly align with a basal gene signature, correlating with worse overall survival in PDAC bulk RNAseq datasets. Deconvolution showed one of the basal populations identified to be the predominant tumor subtype in non- dissociated ST tissues and in vitro tumor cell and organoid PDAC lines. In bulk RNAseq datasets, we uncovered that myofibroblasts (myFb) expressing CXCL10 consistently correlated with both basal subtypes and were found near basal tumor cells using immunostaining. We also identified that besides immune cells, ductal cells also express CXCR3, the receptor for CXCL10, further suggesting a relationship between these cell types in PDAC tumor microenvironment (TME). Overall, we show that our newly built scRNAseq atlas, containing over 700,000 cells, integrated with ST data is a powerful resource, given that it corroborates numerous PDAC TME observations in the literature, with increased statistical power. Moreover, we uncovered a novel association between CXCL10+ myFb and basal tumor cells, underscoring the presence of unique spatial niches in PDAC that should be explored for the development of targeted therapies for this dismal disease.
 Citation Format: Ian M Loveless, Samantha B Kempt, Yuesong Wu, Daniel J Salas-Escabillas, Kailee Hartway, Madison George, Yetunde Makinwa, Thais Stockdale, Samuel D Zwernik, Kendyll Gartrelle, Rohit G Reddy, Daniel Long, Allison Wombell, Julie M Clark, Albert M Levin, David Kwon, Ling Huang, Ralph Francescone, Débora B Vendramini-Costa, Ben Stanger, Adam Alessio, Andrew M Waters, Yuehua Cui, Brian Theisen, Howard C Crawford, Nina G Steele. Human pancreatic cancer single cell atlas reveals association of CXCL10 positive fibroblasts and basal subtype tumor cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B068.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb2cdcff90781c8782f8a20266c95fd99d51d7f6" target='_blank'>
              Abstract B068: Human pancreatic cancer single cell atlas reveals association of CXCL10 positive fibroblasts and basal subtype tumor cells
              </a>
            </td>
          <td>
            Ian Loveless, Samantha B Kempt, Yuesong Wu, Daniel J. Salas-Escabillas, Kailee Hartway, Madison George, Yetunde Makinwa, Thais Stockdale, Samuel Zwernik, K. Gartrelle, Rohit G Reddy, Daniel W Long, Allison Wombell, Julie Clark, Albert Levin, David Kwon, Ling Huang, R. Francescone, D. Vendramini-Costa, Ben Z. Stanger, Adam Alessio, Andrew M. Waters, Yuehua Cui, Brian Theisen, Howard C. Crawford, Nina G. Steele
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Background Major histocompatibility complex class I (MHC-I) loss is frequent in non-small cell lung cancer (NSCLC) rendering tumor cells resistant to T cell lysis. NK cells kill MHC-I-deficient tumor cells, and although previous work indicated their presence at NSCLC margins, they were functionally impaired. Within, we evaluated whether NK cell and CD8 T cell infiltration and activation vary with MHC-I expression. Methods We used single-stain immunohistochemistry (IHC) and Kaplan-Meier analysis to test the effect of NK cell and CD8 T cell infiltration on overall and disease-free survival. To delineate immune covariates of MHC-I-disparate lung cancers, we used multiplexed immunofluorescence (mIF) imaging followed by multivariate statistical modeling. To identify differences in infiltration and intercellular communication between IFNγ-activated and non-activated lymphocytes, we developed a computational pipeline to enumerate single-cell neighborhoods from mIF images followed by multivariate discriminant analysis. Results Spatial quantitation of tumor cell MHC-I expression revealed intratumoral and intertumoral heterogeneity, which was associated with the local lymphocyte landscape. IHC analysis revealed that high CD56+ cell numbers in patient tumors were positively associated with disease-free survival (HR=0.58, p=0.064) and overall survival (OS) (HR=0.496, p=0.041). The OS association strengthened with high counts of both CD56+ and CD8+ cells (HR=0.199, p<1×10−3). mIF imaging and multivariate discriminant analysis revealed enrichment of both CD3+CD8+ T cells and CD3–CD56+ NK cells in MHC-I-bearing tumors (p<0.05). To infer associations of functional cell states and local cell–cell communication, we analyzed spatial single-cell neighborhood profiles to delineate the cellular environments of IFNγ+/– NK cells and T cells. We discovered that both IFNγ+ NK and CD8 T cells were more frequently associated with other IFNγ+ lymphocytes in comparison to IFNγ– NK cells and CD8 T cells (p<1×10–30). Moreover, IFNγ+ lymphocytes were most often found clustered near MHC-I+ tumor cells. Conclusions Tumor-infiltrating NK cells and CD8 T cells jointly affected control of NSCLC tumor progression. Coassociation of NK and CD8 T cells was most evident in MHC-I-bearing tumors, especially in the presence of IFNγ. Frequent colocalization of IFNγ+ NK cells with other IFNγ+ lymphocytes in near-neighbor analysis suggests NSCLC lymphocyte activation is coordinately regulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bffb606e7f7747c3416045b156baec2a760c44c" target='_blank'>
              Spatial colocalization and combined survival benefit of natural killer and CD8 T cells despite profound MHC class I loss in non-small cell lung cancer
              </a>
            </td>
          <td>
            Remziye E Wessel, Nardin Ageeb, Joseph M. Obeid, Ileana S Mauldin, Kate A. Goundry, Gabriel F. Hanson, Mahdin Hossain, Chad Lehman, R. Gentzler, Nolan A Wages, C. Slingluff, T. Bullock, Sepideh Dolatshahi, Michael G. Brown
          </td>
          <td>2024-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befebce4fc2c4b9cfeedafc2e4927bdb0544dbd9" target='_blank'>
              Mechanisms that clear mutations drive field cancerization in mammary tissue
              </a>
            </td>
          <td>
            Marta Ciwinska, H. Messal, Hristina R Hristova, C. Lutz, Laura Bornes, Theofilos Chalkiadakis, Rolf Harkes, Nathalia S M Langedijk, S. Hutten, Renée X Menezes, Jos Jonkers, S. Prekovic, Ben Simons, C. Scheele, J. van Rheenen
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>2</td>
          <td>39</td>
        </tr>

        <tr id="
 Introduction: A complex immunosuppressive tumor microenvironment (TME) is a distinctive feature of pancreatic ductal adenocarcinoma (PDAC). The TME consists of an admixture of cell types including epithelial, endothelial, immune cells of both myeloid and lymphoid origin, and cancer-associated fibroblasts (CAFs). The traditionally defined subtypes of PDAC tumor cells include classical, associated with the retention of epithelial markers and a better prognosis, and basal, which is characterized by a quasi-mesenchymal phenotype with poorer responses to chemotherapy and worse overall survival. Additionally, CAF subtypes are associated with specific roles within the TME. These include inflammatory (iCAF), with the capacity to secrete IL-6, myofibroblastic (myCAF), defined by high levels of alpha smooth muscle actin (αSMA) expression, and antigen presenting (apCAF), defined by expression of HLA-DR and antigen processing and presentation capacity. We hypothesized that interactions between CAFs and tumor cells altered tumor cell phenotype and could be delineated. Methods: Using multi-omics platforms, we evaluated the impact of CAF populations on tumor cell phenotype. Our approach integrated multiplex immunohistochemistry, imaging mass cytometry (IMC) and RNA-scope of patient tumor tissues with functional in vitro studies from matched patient-derived organoid (PDO) and CAF cocultures. The dynamic impact of CAFs on PDOs was examined using a combination of bulk RNA sequencing, multiparametric flow cytometry and secretome analysis, in addition to RT-qPCR and western blot comparing cocultures to matched PDO monocultures. Results: IMC of 15 patient tumors revealed distinct neighborhoods of CAFs and tumor cells. iCAFs were in close proximity to classical tumor cells, while myCAFs were in close proximity to basal tumor cells. Further, in our PDO-CAF co-cultures the PDOs underwent a shift from classical to basal subtype in 42% of co-cultures, a phenomenon also inferred in patient tissues by associating collagen content with classical or basal designation. We also captured a tumor cell population expressing both basal and classical markers that may represent a transition state. The shift from classical to basal tumor cell subtype was accompanied by enhanced epithelial-mesenchymal transition and a loss of IL-8 secretion. Interestingly, when we examined immune cell distribution within tumor tissues, T cells were both the most proximal and the most abundant immune cell type in relation to tumor cells. Moreover, the highest frequency of total and activated T cells was seen in close association with basal tumor cells. Conclusions: Our study underscores the importance of understanding the complex interplay between CAFs and tumor cells in PDAC. Further, we identify a relationship between T cells and tumor cell subtypes. By leveraging high-dimensional data and patient-derived models, we provide new insights into the spatial and functional dynamics of the PDAC TME, paving the way for novel therapeutic strategies to reprogram the TME and enhance patient survival.
 Citation Format: Jacquelyn W Zimmerman, Samantha Guinn, Mili Ramani, Brayan Perez, Joseph A Tandurella, Daniel J Zabransky, Jae W Lee, Emma Kartalia, Jignasha Patel, Sarah Shin, Alexei Hernandez, Courtney Cannon, Nicole Gross, Soren Charmsaz, Elana J Fertig, Richard A Burkhart, Won J Ho, Elizabeth M Jaffee. Cancer associated fibroblasts drive classical to basal change associated with increased T cell presence in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A032.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b1b3de60c255399613042a94100d74cd1acaab3" target='_blank'>
              Abstract A032: Cancer associated fibroblasts drive classical to basal change associated with increased T cell presence in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jacquelyn W. Zimmerman, Samantha Guinn, Mili Ramani, Brayan Perez, Joseph A Tandurella, Daniel J. Zabransky, Jae W. Lee, Emma Kartalia, Jignasha Patel, Sarah M. Shin, Alexei Hernandez, Courtney D Cannon, Nicole E. Gross, Soren Charmsaz, E. Fertig, Richard A Burkhart, W. Ho, Elizabeth M. Jaffee
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="


 Glioblastoma trigger systemic immune responses, resulting in elevated white blood cell counts and elevated temperatures, as the body attempts to combat tumor cells. However, glioblastoma reprograms immune cells to evade immune-mediated destruction and promote tumor growth. While targeted immunotherapies have shown promise in some cancers like melanoma, they have not been efficacious for malignant brain tumors, leading to significant societal costs due to high morbidity and mortality rates.



 Understanding immune cell trafficking is crucial for developing effective immunotherapies. We conducted spatially resolved single-cell analyses to investigate immune cell entry into glioblastoma. Integrating these data with multi-omic sequencing analyses of anatomically dissected specimens from glioblastoma-affected human brain border regions and tumor cores revealed distinct paths of immune cell trafficking into glioblastoma.



 One path involves an ordered entry of mononuclear cells through the subarachnoid and perivascular spaces, guided by chemotactic cues. Cells in these regions display markers of immune activation, suggesting immune priming. Another path occurs at sites of vascular leakage due to hypoxia and necrosis, where a mix of mononuclear cells and neutrophils accumulate. These cells exhibit immunosuppressive transcriptomic programs, regulated by the phagocytosis-inducing transcription factor MITF and downstream metabolic changes controlled by PPAR-gamma. Ongoing analyses aim to identify pharmacological agents that can modulate immune cell reprogramming at hypoxic niches.



 In summary, we have identified distinct modes of immune cell trafficking in glioblastoma. Chemotaxis-mediated trafficking via the brain border regions appears to prime immune cells, while entry at hypoxic niches is associated with immune suppression. Pharmacological agents shielding immune cells from immunosuppressive molecular cues in the tumor microenvironment show promise for glioblastoma immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8f5d39f4dab2e338bbcd19df4cc3dd4650f4fd" target='_blank'>
              P02.20.A NICHE-DEPENDENT MODES OF IMMUNE CELL TRAFFICKING IN GLIOBLASTOMA OFFER TARGETS FOR NOVEL IMMUNOTHERAPIES
              </a>
            </td>
          <td>
            R. Sankowski, A. Dalmau Gasull, M. Prinz
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Despite the growing insights into the cellular heterogeneity of pediatric brain tumors, the transcriptional changes and spatial reorganization during disease progression remain largely unexplored. Here, we present an in-depth single-cell and 10X Xenium spatial transcriptomic characterization of 24 matched pediatric high-grade gliomas (pHGGs) in children, adolescents and young adults (AYA) profiled at both diagnosis and recurrence - constituting a total of 55 tissue samples. By stratifying the transcriptional programs activated at recurrence, we identify two distinct global response sets. Hemispheric AYA tumors predominantly exhibit upregulation of extracellular matrix pathways at recurrence, whereas midline tumors preferentially activate generic stress response programs. On a cellular level, we find that heterogeneity within the malignant tumor compartment remains largely unchanged at recurrence. However, a subset of AYA patients with hemispheric pHGG shows enrichment of the neural-progenitor-like (NPC-like) tumor cell state, and compositional changes in the immune and normal cell landscape, characterized by a global decrease in myeloid cells and an increase in oligodendrocyte fractions. Those changes lead, in turn, to enhanced NPC-like tumor cell-to-oligodendrocyte interactions. Spatial analysis of our extensive cohort at unprecedented subcellular resolution further reveals distinct cellular and spatial features associated with tumor progression, particularly in the remodeling of cell-cell interactions and tumor cell neighborhoods. Overall, this study provides a comprehensive longitudinal single-cell and spatial atlas of pHGG, uncovering the extensive cellular heterogeneity associated with tumor progression. Our findings highlight the role of intrinsic and extrinsic tumor adaptations in shaping disease progression in pHGG, offering potential novel therapeutic targets.
 Citation Format: Sara G. Danielli, Sina Neyazi, Olivia A. Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, Johannes Gojo, Keith L. Ligon, Lissa Baird, Sanda Alexandrescu, Jennifer Cotter, Michael Prados, Adam Resnick, Lin Wang, Michelle Monje, Aaron Diaz, Mariella G. Filbin. Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca1e500481d6ce9fb3ed15507a312915878f86c" target='_blank'>
              Abstract A052 Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults
              </a>
            </td>
          <td>
            Sara G Danielli, S. Neyazi, Olivia A Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob S Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, J. Gojo, Keith L. Ligon, Lissa Baird, S. Alexandrescu, Jennifer Cotter, Michael Prados, Adam C. Resnick, Lin Wang, M. Monje, Aaron A. Diaz, Mariella G. Filbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b7649961c3767077277c2ff33140a98201a7bd6" target='_blank'>
              Characterization and spatial distribution of infiltrating lymphocytes in medullary, and lymphocyte-predominant triple negative breast cancers
              </a>
            </td>
          <td>
            A. Alfaro, C. Catelain, H. El-Masri, P. Rameau, M. Lacroix-Triki, JY. Scoazec, V. Marty, F. Mosele, B. Pistilli
          </td>
          <td>2024-09-14</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de9b939e6665a85e15c06d7919e4ac22bd18c3f" target='_blank'>
              Cellular origin and clonal evolution of human dedifferentiated liposarcoma
              </a>
            </td>
          <td>
            N. Gruel, C. Quignot, Laëtitia Lesage, Sophie El Zein, S. Bonvalot, Dimitri Tzanis, Khadija Ait Rais, Fabien Quinquis, Bastien Manciot, J. Vibert, Nadine El Tannir, A. Dahmani, Héloise Derrien, Didier Decaudin, Ivan Bièche, L. Courtois, O. Mariani, Laetitia K Linares, L. Gayte, Sylvain Baulande, J. Waterfall, O. Delattre, G. Pierron, Sarah Watson
          </td>
          <td>2024-09-12</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>40</td>
        </tr>

        <tr id="Although generally unknown, the age of a newly diagnosed tumor encodes valuable etiologic and prognostic information. Here, we estimate the age of breast cancers, defined as the time from the start of growth to detection, using a measure of epigenetic entropy derived from genome-wide methylation arrays. Based on an ensemble of neutrally fluctuating CpG (fCpG) sites, this stochastic epigenetic clock differs from conventional clocks that measure age-related increases in methylation. We show that younger tumors exhibit hallmarks of aggressiveness, such as increased proliferation and genomic instability, whereas older tumors are characterized by elevated immune infiltration, indicative of enhanced immune surveillance. These findings suggest that the clock captures a tumor’s effective growth rate resulting from the evolutionary-ecological competition between intrinsic growth potential and external systemic pressures. Because of the clock’s ability to delineate old and stable from young and aggressive tumors, it has potential applications in risk stratification of early-stage breast cancers and guiding early detection efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38dff77a79c0945793feb47e32141039b96a0e80" target='_blank'>
              Mapping the Temporal Landscape of Breast Cancer Using Epigenetic Entropy
              </a>
            </td>
          <td>
            Daniel L. Monyak, Shannon T. Holloway, Graham J. Gumbert, Lars J. Grimm, E. S. Hwang, Jeffrey R. Marks, Darryl Shibata, Marc D. Ryser
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Glioblastoma (GB) stem cells (GSCs) are described to recapitulate the expression signatures of stem and progenitor cell classes in the neurodevelopmental hierarchy and are implicated in driving therapeutic resistance. However, the heterogeneity of both the GSC subclasses and their expression of stemness markers hinders comprehensive labelling of these cells. Cellular dormancy is a key feature of GSCs that can be captured by label retention method to track slowly dividing cells. We used CellTrace (CT) dye to identify long term label retaining GB cells (LRGC) in orthotopic xenograft model. We analysed the distribution of LRGC versus the tumour bulk in histology. Additionally, we analysed the transcriptional signature by FAC-sorting LGRC for single cell RNA sequencing. Further, we demonstrated that LRCs survived through temozolomide (TMZ) treatment in vivo and re-initiated tumour proliferation in a faster pace than other surviving glioma cells, signifying its role in therapeutic relapse.



 To label LRGC, patient derived GB cell lines were stained with CT dye before transplantation into the striatum of immunodeficient mice. After 6 weeks of growth the brains were cryosectioned, histologically stained and imaged via confocal microscopy, followed by histologic AI assisted analysis with the AIVIA software. LRGC and non-label retaining tumor cells were FAC-sorted for single-cell RNA sequencing (SMART seq2). To explore the role of LGRC in therapeutic resistance, tumors were treated with TMZ. Re-initiation of tumor growth was tracked by subcutaneous surgical implantation of BrdU pumps at different timepoints after therapy. The proliferation patterns of LRGC versus other surviving tumor cells were analysed in histology again using an AI assisted software.



 LRGC could be identified in histology 6 weeks after tumor growth. LRGC were found near the centre of tumor but rarely at the invasion front. Single cell RNA-seq of LRGC showed differentially expressed pathways related to lipid metabolism and neurodevelopment. Re-initiation of tumor growth started 12-14 days after therapy. LRGC showed a more intense proliferation compared to other surviving tumor cells, indicating the ability of LRGC to not only survive TMZ treatment, but also drive GB relapse.



 LRGC could be visualized 6-8 weeks after tumor growth in an orthotopic xenograft model. On a transcriptional level these cells share similarities with stem cells in normal brain development but also show distinct features. An important role of LRGC in re-initiation of tumor growth after therapy could be confirmed.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863c1df5d02a6308a5d876cfe8995a6849735071" target='_blank'>
              P03.02.B DECIPHERING THE IDENTITY AND CHARACTERISTICS OF LONG-TERM LABEL RETAINING GLIOBLASTOMA CELLS AND THEIR ROLE IN TUMOUR RELAPSE
              </a>
            </td>
          <td>
            H. Chow, A. Kourtesakis, L. Hai, S. Schaffranek, S. Pusch, D. Hoffmann, D. Reibold, W. Wick, R. Wagener
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Cancer initiation and progression are always associated with the acquisition and accumulation of genetic mutations which drive the malignant transformation of the cancer cells of origin. It has been shown that different driver mutations can generate tumors with distinct pathological features. How driver mutations interact with tumor cells of origin and how such interactions dictate the fate of the cell of origin and the tumor pathological features are the fundamental questions in cancer biology.



 We developed a set of novel “all-in-one” glioma lineage-tracing model system based the intra-brain ventricle electroporation and transposon-mediated conditional gene editing techniques. By using this system in genetically engineered mouse models, we generated glioma models with the top driver mutation combinations found in IDH-WT GBM patients and trace their initiation and progression with single-cell resolution, therefore we combined lineage tracing with multi-omics analysis almost recapitulating the full driver-mutation spectrum found in glioblastoma patients.



 We found that driver gene mutations combinatorially dictate the early cell fate of tumor development. Driver gene mutations and cell fate combinatorially determine the prognosis and malignancy of glioma. Driver gene mutations combinatorially affect the molecular phenotypes and the treatment of end-stage tumor via deterministic CNVs. Synergistic or exclusive patterns of driver gene mutation combinations regulate the initiation and progression of gliomagenesis.



 The results will help to address those fundamental questions pertinent to the relationships between driver mutations, tumor evolution pathways and the final tumor phenotypes. The results will also help to understand the nature of gliomagenesis and to develop novel approaches for glioma intervention and treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f1a14c5a8062e0c2c1784f57135d696cc3437b" target='_blank'>
              P01.04.B DISSECT THE ROLE OF DRIVER GENE MUTATIONS IN THE INITIATION AND PROGRESSION OF GLIOMA
              </a>
            </td>
          <td>
            A. Tian
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is the most common type of malignancy in children. ALL prognosis after initial diagnosis is generally good; however, patients suffering from relapse have a poor outcome. The tumor microenvironment is recognized as an important contributor to relapse, yet the cell-cell interactions involved are complex and difficult to study in traditional experimental models. In the present study, we established an innovative larval zebrafish xenotransplantation model, that allows the analysis of leukemic cells (LCs) within an orthotopic niche using time-lapse microscopic and flow cytometric approaches. LCs homed, engrafted and proliferated within the hematopoietic niche at the time of transplant, the caudal hematopoietic tissue (CHT). A specific dissemination pattern of LCs within the CHT was recorded, as they extravasated over time and formed clusters close to the dorsal aorta. Interactions of LCs with macrophages and endothelial cells could be quantitatively characterized. This zebrafish model will allow the quantitative analysis of LCs in a functional and complex microenvironment, to study mechanisms of niche mediated leukemogenesis, leukemia maintenance and relapse development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93d8b671ae9c63ac39a6096f61f5f4b266ff65" target='_blank'>
              In vivo monitoring of leukemia-niche interactions in a zebrafish xenograft model
              </a>
            </td>
          <td>
            A. Arner, Andreas Ettinger, Bradley Wayne Blaser, Bettina Schmid, I. Jeremias, Nadia Rostam, Vera Binder-Blaser
          </td>
          <td>2024-08-30</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93313442cbce819fc17c1c31b35718987dafc2d3" target='_blank'>
              NPM1 inhibits tumoral antigen presentation to promote immune evasion and tumor progression
              </a>
            </td>
          <td>
            Xin Wang, Yangyang Chai, Yuan Quan, Jiaming Wang, Jiaying Song, Wenkai Zhou, Xiaoqing Xu, Henan Xu, Bingjing Wang, Xuetao Cao
          </td>
          <td>2024-10-14</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary The tumor microenvironment (TME) refers to the area immediately surrounding a cancerous tumor that influences the tumor’s behavior and how the immune system reacts to it. Central to the TME environment are special immune cells—tumor-associated macrophages (TAMs)—that, rather than restrict the tumor, actually facilitate tumor growth and metastasis. Thus, targeting these TAMs can help to eliminate cancer. In this article, we review the biology of TAMs and delve into the signaling mechanisms underlying the polarization of macrophage phenotypes, their plasticity, and therapeutic implications. Abstract Within the tumor microenvironment, myeloid cells constitute a dynamic immune population characterized by a heterogeneous phenotype and diverse functional activities. In this review, we consider recent literature shedding light on the increasingly complex biology of M2-like immunosuppressive tumor-associated macrophages (TAMs), including their contribution to tumor cell invasion and metastasis, stromal remodeling (fibrosis and matrix degradation), and immune suppressive functions, in the tumor microenvironment (TME). This review also delves into the intricate signaling mechanisms underlying the polarization of diverse macrophage phenotypes, and their plasticity. We also review the development of promising therapeutic approaches to target these populations in cancers. The expanding knowledge of distinct subsets of immunosuppressive TAMs, and their contributions to tumorigenesis and metastasis, has sparked significant interest among researchers regarding the therapeutic potential of TAM depletion or phenotypic modulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0ebf1f65314e32b8f73ed7814f661194d4b5d27" target='_blank'>
              Tumor-Associated Macrophages as Major Immunosuppressive Cells in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Anghesom Ghebremedhin, Dipti Athavale, Yanting Zhang, Xiaodan Yao, Curt Balch, Shumei Song
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Glioblastoma multiform (GBM), a grade IV glioma, is the most common and aggressive cancer in the central nervous system. Current treatment for GBM includes surgical resection, radiation, and the frontline DNA alkylating drug, temozolomide (TMZ). The current median survival for GBM patients is about 14.5 months with 5% patients surviving up to 5 years. We propose that discerning distinct subsets within heterogeneous GBM will provide avenues for research to improve new therapies. We used different methods to isolate GBM cell subsets. These include stable transfectants of GBM cell lines with a lentiviral system in which green fluorescence protein (GFP) is regulated with tandem repeats of Oct4a and Sox2 response elements. Parallel studies with a plasmid using the full-length regulatory region of Oct4a indicated reduced efficiency in separating cell subsets, relative to SORE6-GFP lentivirus. Stem cell-linked gene expressions and function studies such as ALDH1, tumorsphere and in vivo passaging of GFP hi subsets confirmed the presence of cancer stem cells (CSCs). We also studied a more efficient method that could be relevant for primary GBM cells. We selected tumorspheres by plating heterogeneous GBM cells and then serially passaged the spheres. Studies for stem cell genes indicated that this method could be used for primary GBM cells. Overall, this study provided insights into methods to isolate GBM subsets, including primary GBM cells. The advantages of the methods are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492e350655b514b418d1c77aa6fb2234fa8abba3" target='_blank'>
              A Platform to Establish a Working Hierarchy of Glioblastoma Multiforme Cells:
              </a>
            </td>
          <td>
            V. H. Savanur, Anushka Sarkar, Andrew Petryna, Ky Nguyen, Jesus Benites-Sandoval, Marina Gergues, Arash Hatefi, P. Rameshwar
          </td>
          <td>2024-09-24</td>
          <td>International Journal of Translational Science</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Glioblastoma is characterized by heterogeneous malignant cells that are functionally integrated within the neuroglial microenvironment. Here, we model this ecosystem by growing glioblastoma into long-term cultured human cortical organoids that contain the major neuroglial cell types found in the cerebral cortex. Single-cell RNA-seq analysis suggests that, compared to matched gliomasphere models, glioblastoma cortical organoids (GCO) more faithfully recapitulate the diversity and expression programs of malignant cell states found in patient tumors. Additionally, we observe widespread transfer of glioblastoma transcripts and GFP proteins to non-malignant cells in the organoids. Mechanistically, this transfer involves extracellular vesicles and is biased towards defined glioblastoma cell states and astroglia cell types. These results extend previous glioblastoma-organoid modeling efforts and suggest widespread intercellular transfer in the glioblastoma neuroglial microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6299bf8e02b097a87b6cfa72a31937f20578aae9" target='_blank'>
              Glioblastoma-cortical organoids recapitulate cell state heterogeneity and intercellular transfer.
              </a>
            </td>
          <td>
            Vamsi Mangena, Rony Chanoch-Myers, Rafaela Sartore, Bruna Paulsen, S. Gritsch, Hannah Weisman, Toshiro Hara, X. Breakefield, Koen Breyne, Aviv Regev, K. Chung, Paola Arlotta, I. Tirosh, M. Suvà
          </td>
          <td>2024-10-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Liver cancer represents a substantial global health challenge, contributing significantly to worldwide morbidity and mortality. It has long been understood that tumors are not composed solely of cancerous cells, but also include a variety of normal cells within their structure. These tumor-associated normal cells encompass vascular endothelial cells, fibroblasts, and various inflammatory cells, including neutrophils, monocytes, macrophages, mast cells, eosinophils, and lymphocytes. Additionally, tumor cells engage in complex interactions with stromal cells and elements of the extracellular matrix (ECM). Initially, the components of what is now known as the tumor microenvironment (TME) were thought to be passive bystanders in the processes of tumor proliferation and local invasion. However, recent research has significantly advanced our understanding of the TME’s active role in tumor growth and metastasis. Tumor progression is now known to be driven by an intricate imbalance of positive and negative regulatory signals, primarily influenced by specific growth factors produced by both inflammatory and neoplastic cells. This review article explores the latest developments and future directions in understanding how the TME modulates liver cancer, with the aim of informing the design of novel therapies that target critical components of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d21309486c5cd52dddda7197045d0a0b43a28e44" target='_blank'>
              Beyond Cancer Cells: How the Tumor Microenvironment Drives Cancer Progression
              </a>
            </td>
          <td>
            Hussein Sabit, B. Arneth, Shaimaa Abdel-Ghany, Engy F. Madyan, Ashraf H. Ghaleb, Periasamy Selvaraj, Dong M. Shin, Ramireddy Bommireddy, Ahmed Elhashash
          </td>
          <td>2024-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f00e386abbf367932af2ed3eee8d073da659417c" target='_blank'>
              Multi-stage mechanisms of tumor metastasis and therapeutic strategies
              </a>
            </td>
          <td>
            Zaoqu Liu, Jingqi Chen, Yuqing Ren, Shutong Liu, Yuhao Ba, Anning Zuo, Peng Luo, Q. Cheng, Hui-Xian Xu, Xinwei Han
          </td>
          <td>2024-10-11</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Background Cell death modulates the biological behaviors of tumors. However, the comprehensive role of the multiple forms of cell death in lower-grade glioma (LGG) is unknown. Methods We collected the transcriptional data of LGG patients from public repositories to comprehensively examine six cell death patterns (autophagy, apoptosis, cuproptosis, necroptosis, ferroptosis, and pyroptosis) in LGG samples and systematically correlated these patterns with patient survival, underlying biological processes, and drug sensitivity using serial bioinformatics analysis, clinical sample validation and in vitro assays. Results We identified and independently validated three reproducible cell death-based clusters associated with distinct clinical outcomes and tumor microenvironment characteristics. The Tumor Immune Dysfunction and Exclusion algorithm was applied for predicting how these three clusters would respond to immune checkpoint blockade (ICB) therapy; we found potential resistance of cluster B to ICB therapy. We also performed drug screening to identify cluster-specific drugs. Furthermore, a scoring system, designated as the CDPM score, was developed to estimate the cell death patterns of patients with LGG; this system could predict both LGG patients’ prognosis and immunotherapy efficacy. By performing multiplex immunofluorescence staining, we validated the correlations of GNAL expression with the molecular and clinical features of LGG in an independent LGG cohort. Finally, in vitro assays showed that GNAL promoted apoptosis and inhibited the proliferation of LGG cells. Conclusion The new cell death-based subtype system indicates several features of LGG biology and reveals novel insights into the use of precision medicine for treating LGG. The CDPM score could be used to predict the immunotherapy response and prognosis of LGG patients; moreover, it could indicate a novel direction for improving LGG management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7766cfe0385ddf0c9af53006d23560398f64acd" target='_blank'>
              An integrative analysis based on multiple cell death patterns identifies an immunosuppressive subtype and establishes a prognostic signature in lower-grade glioma
              </a>
            </td>
          <td>
            Hao Lian, Jiajia Wang, Shan Yan, Kui Chen, Lilun Jin
          </td>
          <td>2024-10-10</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="PURPOSE
Immunotherapies have led to a paradigm shift in the treatment of hepatocellular carcinoma (HCC). Studies have revealed the single-cell catalogues of tumor-infiltrating immune cells and the trajectories of their differentiation. Nevertheless, the spatial distribution of these immune cells with distinct phenotypes in tumor microenvironment and their clinicopathological significance in resectable and unresectable HCCs are still largely unclear.


EXPERIMENTAL DESIGN
We analyzed the spatial dynamics of intratumoral CD4 and CD8 T cells and their association with B and plasma cells using 283 surgically resected HCCs, 58 unresectable HCC samples before combined immunotherapy (atezolizumab plus bevacizumab [Atezo+Bev]), and autopsy specimens from 50 cases of advanced-stage HCCs through multiplex immunohistochemistry combined with transcriptomic and driver gene mutation analysis. Classification based on the spatial dynamics of T and B cell responses (refined immunosubtype) was developed and its clinicopathological significance was analyzed.


RESULTS
We found that stem-like CD4 and CD8 T cells were mainly observed in T cell aggregates. Differentiation of T follicular helper cells were associated with the development of tertiary lymphoid structures, whereas differentiation of CD4 TCXCL13 cells with phenotype resembling T peripheral helper cells were associated with the development of lymphoplasmacytic microenvironment. The refined immunosubtype could predict clinical outcomes of resectable HCC after surgery and unresectable HCC after Atezo+Bev therapy. The immune microenvironment of metastatic lesions tended to reflect those of primary lesions.


CONCLUSIONS
We revealed the spatial dynamics of T and B cell response in HCC, which is closely associated with the clinical outcome after surgical resection or Atezo+Bev therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30a9933c34558056c95ceeec869060a2ac77726f" target='_blank'>
              Spatial Dynamics of T and B Cell Response Predicts Clinical Outcome of Resectable and Unresectable Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Yutaka Kurebayashi, K. Sugimoto, Hanako Tsujikawa, Kosuke Matsuda, Rui Nomura, A. Ueno, Y. Masugi, Ken Yamazaki, Kathryn Effendi, Hirohito Takeuchi, Takao Itoi, Y. Hasegawa, Yuta Abe, M. Kitago, H. Ojima, Michiie Sakamoto
          </td>
          <td>2024-10-17</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Dclk1 expression defines a rare population of cells in the normal pancreas whose frequency is increased at early stages of pancreatic tumorigenesis. The identity and the precise roles of Dclk1 expressing cells in pancreas have been matter of debate, although evidence suggests their involvement in a number of key functions, including regeneration and neoplasia. We employed a recently developed Dclk1 reporter mouse model and single cell RNAseq analysis to define Dclk1 expressing cells in normal pancreas and pancreatic neoplasia. In normal pancreas, Dclk1 epithelial expression identifies subsets of ductal, islet and acinar cells. In pancreatic neoplasia, Dclk1 expression identifies five epithelial cell populations, among which acinar-to-ductal metaplasia (ADM)-like cells and tuft-like cells are predominant. These two cell populations play opposing roles in pancreatic neoplasia, with Dclk1+ ADM-like cells sustaining tumor growth while Dclk1+ tuft-like cells restraining tumor progression. The differentiation of Kras mutant acinar cells into Dclk1+ tuft-like cells requires the activation of the transcription factor SPIB and is further supported by a cellular paracrine loop involving cancer group 2 innate lymphoid cells (ILC2) and cancer activated fibroblasts (CAFs) that provide IL13 and IL33, respectively. In turn, Dclk1+ tuft-like cells release angiotensinogen that plays protective roles against pancreatic neoplasia. Overall, our study provides novel insights on the biology of Dclk1+ cells in normal pancreas and unveils a protective axis against pancreatic neoplasia, involving CAFs, ILC2 and Dclk1+ tuft-like cells, which ultimately results in angiotensinogen release.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297c14738f38a2df2fd03a4a3c024c764c7c76b3" target='_blank'>
              Regulatory network analysis of Dclk1 gene expression reveals a tuft cell-ILC2 axis that inhibits pancreatic tumor progression
              </a>
            </td>
          <td>
            G. Valenti, P. Laise, Ryota Takahashi, Feijin Wu, Tuo Ruan, A. Vasciaveo, Zhengyu Jiang, Masaki Sunagawa, M. Middelhoff, Henrik Nienhüser, Na Fu, E. Malagola, Y. Hayakawa, Alina C. Iuga, Andrea Califano, Timothy C Wang
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Alpha-smooth muscle actin (α-SMA) expression in the stroma is linked to the presence of cancer-associated fibroblasts and is known to correlate with worse outcomes in various tumors. In this study, using a GeoMx digital spatial profiling approach, we characterized the gene expression of the tumor and α-SMA-expressing stromal cell compartments in pancreatic neuroendocrine tumors (PanNETs). The profiling was performed on tissues from eight retrospective cases (three grade 1, four grade 2, and one grade 3). Selected regions of interest were segmented geometrically based on tissue morphology and fluorescent signals from synaptophysin and α-SMA markers. The α-SMA-expressing stromal-cell-associated genes were involved in pathways of extracellular matrix modification, whereas, in tumor cells, the gene expression profiles were associated with pathways involved in cell proliferation. The comparison of gene expression profiles across all three PanNET grades revealed that the differences between grades are not only present at the level of the tumor but also in the α-SMA-expressing stromal cells. Furthermore, the tumor cells from regions with a rich presence of adjacent α-SMA-expressing stromal cells revealed an upregulation of matrix metalloproteinase-9 (MMP9) expression in grade 3 tumors. This study provides an in-depth characterization of gene expression profiles in α-SMA-expressing stromal and tumor cells, and outlines potential crosstalk mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fe06c12f5865d2f0fe11fa3ad5ad66746e58fe" target='_blank'>
              Tumor and α-SMA-expressing stromal cells in pancreatic neuroendocrine tumors have a distinct RNA profile depending on tumor grade.
              </a>
            </td>
          <td>
            H. Niedra, R. Peculis, Rihards Saksis, Ilona Mandrika, Sofija Vilisova, Jurijs Nazarovs, Austra Breikša, Aija Gerina, Julie Earl, Ignacio Ruz-Caracuel, M. Rosas, A. Puķītis, Natalja Senterjakova, V. Rovite
          </td>
          <td>2024-09-08</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Pancreatic cancer, recognized as one of the most lethal cancer types, exhibits a dismal five-year survival rate below 12%. The genetic and epigenetic heterogeneity in human cancer poses a tremendous challenge in understanding disease mechanisms and developing targeted therapeutics. Current mouse models in the field can recapitulate some features of human pathophysiology but have limitations. While tumor formation in patients is focal, most genetically engineered mouse models (GEMMs) express oncogenes throughout the pancreas. GEMMs also have limited flexibility to manipulate gene expression and model the heterogeneity of human cancer. Additionally, GEMMs require significant time for crossing and tumor formation. Tumor transplantation models offer greater flexibility, however, they often require immunodeficient mice, which limits their applications in modeling the tumor microenvironment. To overcome the limits of current mouse models, we combined the piggyBac transposon system, CRISPR/Cas9, and in vivo electroporation to engineer somatic pancreatic tissue to induce focal tumor formation in immune-competent mice. With this approach, we engineered wildtype pancreata to express oncogenic KrasG12D while knocking out multiple tumor suppressors. We optimized this protocol by generating multiplexed gRNA constructs and cell type-specific promoters to improve reproducibility. Low-grade tumors formed within 10 days after electroporation, progressed to higher grade within a month, and metastasized to peritoneal organs during disease progression. Mouse tumors also recapitulated histopathological features in human patients. Lastly, we used our novel mouse model to study how mutant Kras disrupts post-transcriptional regulation by microRNA to drive pancreatic tumorigenesis. This model offers a powerful platform for manipulating gene expression and functionality to model cancer heterogeneity, ultimately advancing precision oncology's prospects for treating human patients.
 Citation Format: Ziyue (Zoey) Yang, Mara Cardenas, Akihiko Miyauchi, Caroline Snider, Ronald Parchem. Generation of a flexible pancreatic cancer mouse model via somatic tissue engineering [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02653c15c1c4443d9e4685c7c408a134b796978a" target='_blank'>
              Abstract A006: Generation of a flexible pancreatic cancer mouse model via somatic tissue engineering
              </a>
            </td>
          <td>
            Z. Yang, Mara Cardenas, Akihiko Miyauchi, Caroline Snider, Ronald Parchem
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The bone marrow microenvironment is critical for B-cell acute lymphoblastic leukemia (B-ALL) but its cellular heterogeneity remains poorly defined. Here, we employed single-cell RNA sequencing to comprehensively characterize the stromal and hematopoietic niches in pediatric B-ALL. Our analysis revealed two distinct mesenchymal stromal cell (MSC) populations as primary leukemia-supportive niches: early mesenchymal progenitors and adipogenic progenitors. Single-cell transcriptomic analysis infers that ALL blasts use distinct cell-cell interactions to communicate with the different stromal populations. Purified adipogenic progenitors from the bone of children with ALL support survival of the leukemic blasts ex vivo and their signature is enriched in relapse samples. Our data establish adipogenic progenitors as a distinct and novel component of the ALL niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f24fbb432c05f1e5ee3ed61b3b778c9a293df20" target='_blank'>
              Distinct Stromal Cell Populations Define the B-cell Acute Lymphoblastic Leukemia Microenvironment
              </a>
            </td>
          <td>
            Mauricio N. Ferrao Blanco, Bexultan Kazybay, M. Belderbos, Olaf Heidenreich, H. J. Vormoor
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/484efc8a4ee08b37da85a0f29cd67a40a824997b" target='_blank'>
              Spatial analysis reveals targetable macrophage-mediated mechanisms of immune evasion in hepatocellular carcinoma minimal residual disease.
              </a>
            </td>
          <td>
            Lea Lemaitre, Nia Adeniji, Akanksha Suresh, Reshma Reguram, Josephine Zhang, J. Park, Amit Reddy, Alexandro E. Trevino, Aaron T. Mayer, A. Deutzmann, Aida S Hansen, Ling Tong, Vinodhini Arjunan, N. Kambham, B.C. Visser, M. Dua, C. A. Bonham, N. Kothary, H. B. D'Angio, Ryan Preska, Yanay Rosen, James Y. Zou, Vivek Charu, Dean W Felsher, R. Dhanasekaran
          </td>
          <td>2024-09-20</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Simple Summary Spatial transcriptomics is a technique of measuring gene expression with spatial resolution on a tissue slide, which is especially useful in cancer research. This scoping study reviews 41 articles published by the end of 2023 to examine current trends for employed methods, applications, and data analysis approaches for spatial transcriptomics, identifying challenges and practical uses of this technique in studying cancer. The research shows that cancer is a major focus in spatial transcriptomics studies, with certain tools and methods being particularly popular among scientists. However, many studies do not fully explain their data processing methods, making it hard to reproduce their results. Emphasis on transparent sharing of analysis scripts and tailored single-cell analysis methods for ST is recommended to enhance study reproducibility and reliability in the domain. This work may stand as a spatial transcriptomics developmental snapshot and reference to contemporary neoplasm biology research conducted through this technique. Abstract Spatial transcriptomics (ST) examines gene expression within its spatial context on tissue, linking morphology and function. Advances in ST resolution and throughput have led to an increase in scientific interest, notably in cancer research. This scoping study reviews the challenges and practical applications of ST, summarizing current methods, trends, and data analysis techniques for ST in neoplasm research. We analyzed 41 articles published by the end of 2023 alongside public data repositories. The findings indicate cancer biology is an important focus of ST research, with a rising number of studies each year. Visium (10x Genomics, Pleasanton, CA, USA) is the leading ST platform, and SCTransform from Seurat R library is the preferred method for data normalization and integration. Many studies incorporate additional data types like single-cell sequencing and immunohistochemistry. Common ST applications include discovering the composition and function of tumor tissues in the context of their heterogeneity, characterizing the tumor microenvironment, or identifying interactions between cells, including spatial patterns of expression and co-occurrence. However, nearly half of the studies lacked comprehensive data processing protocols, hindering their reproducibility. By recommending greater transparency in sharing analysis methods and adapting single-cell analysis techniques with caution, this review aims to improve the reproducibility and reliability of future studies in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/546384150762685b69a172f984b5438d683edf6a" target='_blank'>
              Scoping Review: Methods and Applications of Spatial Transcriptomics in Tumor Research
              </a>
            </td>
          <td>
            Kacper Maciejewski, Patrycja Czerwińska
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 The complex biology of pancreatic ductal adenocarcinoma (PDAC) remains a significant challenge and limits overcoming its dismal prognosis. The tumor microenvironment (TME), shaped by dynamic interactions between cancer-associated fibroblasts (CAFs), immune and tumor cells, plays a crucial role in PDAC pathogenesis. Uncharted field lies in the impact of specific gene mutations within the tumor epithelium on orchestrating distinct oncogenic TME patterns. Our investigation into homologous recombination deficiency (HRD), caused by mutations in genes like ATM, reveals its role in promoting PDAC aggressiveness and triggering a pronounced desmoplastic reaction, suggesting a unique TME programing. Through single-cell-resolved and multiomics analyses of murine ATM and/or P53-depleted PDACs, and validation in human cases, we unveil transcriptional, translational, and secretory regulatory events, shed light on the intricate interplay between tumor genetics and PDAC microenvironment and pave the way for the elaboration of novel personalized therapies. Murine tumor analysis using single-nucleus multiomics RNA and ATAC sequencing, alongside histological validation in human tissues, revealed specific enrichment of myCAFs in ATM-deficient tumors and reduced populations of T cells and macrophages. Molecular signatures and cell-cell communication inference uncovered ATM-specific interactions within the TME, highlighting TGF-β signaling as pivotal in mediating dialog between CAFs and an aggressive EMT-like tumor cell subset prevalent in ATM-depleted tumors. Additionally, distinct communication patterns were observed between CAFs and immune cells, associated with specific epigenetic and molecular profiles. Mechanistically, transcriptomics, secretomics and proteomics investigations confirmed increased TGFβ release by ATM-deficient PDAC cells and identified a reactive oxygen species (ROS)-mediated mechanism affecting tumor cell behavior and interactions with CAFs. Consistently, a combinatorial therapy targeting TGF-β-mediated tumor-stroma dialog reversed cancer-promoting TME remodeling and exacerbated FOLFIRINOX cytotoxic effects, in vivo, exclusively in ATM-deficient HRD PDACs. Dissecting the biology of ATM-deficient HRD malignant cells, our study unveils their pivotal role in orchestrating oncogenic communication networks with CAFs and immune cells, along with specific tumor promoting stromal reprogramming. Our findings paint a comprehensive picture of how distinct tumor-stroma interactions reshape the TME, steering it towards a cancer-promoting outcome and shed light on the potential to exploit genotype-specific vulnerabilities via tailored tumor-stroma interference strategies.
 Citation Format: Elodie Roger, Hannah M. Mummey, Eleni Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, Lukas Perkhofer, Johann Gout, Alexander Kleger. Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fd93ca5c41a43eb8ee9eb151fa6f7aeb554c533" target='_blank'>
              Abstract A055: Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer
              </a>
            </td>
          <td>
            E. Roger, Hannah M. Mummey, E. Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, L. Perkhofer, J. Gout, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Simple Summary Gliomas are diverse types of tumors originating from glial cells. Among these tumors, glioblastoma multiforme (GBM) is the most aggressive form and has a poor prognosis and high mortality rate, with treatment success largely influenced by patient age, tumor location, and genetic mutations. Glioma cells can reprogram their tumor microenvironment (TME) to support tumor proliferation and survival, interacting with other cells through cytokines and growth factors. The TME of gliomas is highly heterogeneous, complex, and often immunosuppressive, interfering with treatment strategies. Current therapeutic strategies focus on modulating immune cell activities and enhancing anti-tumor responses. T-cell-based immunotherapies are being explored to target glioma cells. These include chimeric antigen receptor (CAR) T-cell therapy and T-cell receptor (TCR) therapy, which involve engineering T cells to recognize tumor antigens. However, these therapies face multiple challenges including tumor heterogeneity, immunosuppression by T cell exhaustion, and altered antigen presentation, etc. Future research should focus on boosting T cell functions, targeting immune checkpoints, and developing more effective delivery methods to improve treatment outcomes for glioma patients. Abstract Glioma is known for its immunosuppressive microenvironment, which makes it challenging to target through immunotherapies. Immune cells like macrophages, microglia, myeloid-derived suppressor cells, and T lymphocytes are known to infiltrate the glioma tumor microenvironment and regulate immune response distinctively. Among the variety of immune cells, T lymphocytes have highly complex and multifaceted roles in the glioma immune landscape. T lymphocytes, which include CD4+ helper and CD8+ cytotoxic T cells, are known for their pivotal roles in anti-tumor responses. However, these cells may behave differently in the highly dynamic glioma microenvironment, for example, via an immune invasion mechanism enforced by tumor cells. Therefore, T lymphocytes play dual roles in glioma immunity, firstly by their anti-tumor responses, and secondly by exploiting gliomas to promote immune invasion. As an immunosuppression strategy, glioma induces T-cell exhaustion and suppression of effector T cells by regulatory T cells (Tregs) or by altering their signaling pathways. Further, the expression of immune checkpoint inhibitors on the glioma cell surface leads to T cell anergy and dysfunction. Overall, this dynamic interplay between T lymphocytes and glioma is crucial for designing more effective immunotherapies. The current review provides detailed knowledge on the roles of T lymphocytes in the glioma immune microenvironment and helps to explore novel therapeutic approaches to reinvigorate T lymphocytes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/038df71f36526b5ab1e0ec2eb1e14e94beaba15f" target='_blank'>
              Role of T Lymphocytes in Glioma Immune Microenvironment: Two Sides of a Coin
              </a>
            </td>
          <td>
            Laiba Noor, Arun Upadhyay, Vibhuti Joshi
          </td>
          <td>2024-10-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Astrocytoma is a diffuse and heterogeneous glioma that infiltrates the brain. Despite post-operative therapy including radiation and chemotherapy, 52% of patients’ tumours recur within five-years of surgery. These tumours are characterised by mutations in the IDH1 or IDH2 gene that confer metabolic alterations, most notably the production of the onco-metabolite 2-hydroxyglutarate. To reveal the metabolic alterations and their upstream regulation across the heterogeneous tumour microenvironment, we applied spatial metabolomics and spatial transcriptomics to sequential sections from resected tumour samples of patients with low grade astrocytoma.



 Spatial assessment of heterogeneity was performed on serial sections using spatial metabolomics (atmospheric pressure MALDI-TOF) and spatial transcriptomics (10X Genomics Xenium). This necessitated extensive optimisation of image registration across modalities and sequential sections to create three dimensional models to deeply interrogate heterogeneity within tumours.



 Combining spatial transcriptomics and spatial metabolomics across 8 matched sequential slices of the same patient tumour, enabled the identification of metabolic niches associated with distinct cell type composition and transcriptional regulation. Strikingly, normal domains that were present at the leading edge of tumours could be identified both metabolically and transcriptomically.



 Understanding the heterogeneity of astrocytomas on a spatial level reveals unique domains and organisation of tumour and immune cell types. Matching these transcriptomic and cell state information with metabolic insights leads to pathway insights that will be essential to the identification of druggable targets. Our pipeline combining spatial transcriptomics and metabolomics with tissue histology reveals these biologically important domains.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbf36889624e6059d6b35a03a0d2ffccf6ee4adc" target='_blank'>
              OS07.5.A MULTI-MODAL EXAMINATION OF SPATIAL HETEROGENEITY IN THE ASTROCYTOMA MICROENVIRONMENT
              </a>
            </td>
          <td>
            T. Lu, L. Freytag, J. Moffet, J. Kriel, J. Whittle, S. Freytag, S. A. Best
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Uveal melanoma (UM) is a rare type of cancer of the eye that develops from a cell type called melanocytes, which are present in the uveal tract. These patients have a high risk of liver metastasis, usually many years after the initial diagnosis, which is life threatening. We discovered a population of cells in the primary tumor called dual-nature hybrid cells (DNCs) similar to previously discovered cells in the blood (i.e., circulating hybrid cells, CHCs) with shared characteristics of both cancer cells and white blood cells. In this paper, we determined the types of white blood cells capable of participating in the formation of a hybrid cell with a cancer cell, and we re-created these DNCs in a cell culture. The study of the cells of this tumor, particularly DNCs, is crucial to better understanding dissemination and uncovering a possible target for treatment of this systemic disease. Abstract Background: Metastasis, occurring years after primary diagnosis, represents a poor prognosis in uveal melanoma (UM)-affected individuals. The nature of cells involved in this process is under debate. Circulating hybrid cells that have combined tumor and immune cell features found in blood were predictive of metastasis and may correspond to dual-nature cells (DNC) in the primary tumor. Herein, we sought to determine the presence of DNCs in primary UM tumors, the cell types involved in their genesis, and their ability to be formed in vitro. Methods: UM lesions (n = 38) were immunolabeled with HMB45 in combination with immune-cell-specific antibodies. In parallel, we co-cultured UM cells and peripheral blood mononuclear cells (PBMCs) to analyze DNC formation. Results: HMB45+/CD45+ DNCs were present in 90% (26/29) of the tumors, HMB45+/CD8+ DNCs were present in 93% (26/28), and HMB45+/CD68+ DNCs were present in 71% (17/24). DNCs formed with CD8+ and CD68+ cells were positively correlated to the infiltration of their respective immune cells. Notably, UM cells were prone to hybridize with PBMCs in vitro. Conclusions: This phenotypical characterization of DNCs in UM demonstrates that CD8+ T-cells and macrophages are capable of DNC formation, and they are important for better understanding metastatic dissemination, thus paving the path towards novel therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c17116b61c31885d971f0798b1beb4e75b2a6ef8" target='_blank'>
              The Phenotypical Characterization of Dual-Nature Hybrid Cells in Uveal Melanoma
              </a>
            </td>
          <td>
            Emily Marcotte, Alicia Goyeneche, Mohamed Abdouh, J. Burnier, MiguelN Burnier
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9875ac28ea781567d1b798ee476f9d9fa7efb581" target='_blank'>
              Miniaturized protein profiling permits targeted signaling pathway analysis in individual circulating tumor cells to improve personalized treatment
              </a>
            </td>
          <td>
            Mahdi Rivandi, A. Franken, Liwen Yang, A. Abramova, Nadia Stamm, J. Eberhardt, B. Gierke, Meike Beer, Tanja Fehm, D. Niederacher, Michael Pawlak, Hans Neubauer
          </td>
          <td>2024-09-20</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Cell fate decisions, such as proliferation, differentiation, and death, are driven by complex molecular interactions and signaling cascades. While significant progress has been made in understanding the molecular determinants of these processes, historically, cell fate transitions were identified through light microscopy that focused on changes in cell morphology and function. Modern techniques have shifted towards probing molecular effectors to quantify these transitions, offering more precise quantification and mechanistic understanding. However, challenges remain in cases where the molecular signals are ambiguous, complicating the assignment of cell fate. During viral infection, programmed cell death (PCD) pathways, including apoptosis, necroptosis, and pyroptosis, exhibit complex signaling and molecular crosstalk. This can lead to simultaneous activation of multiple PCD pathways, which confounds assignment of cell fate based on molecular information alone. To address this challenge, we employed deep learning-based image classification of dying cells to analyze PCD in single Herpes Simplex Virus-1 (HSV-1)-infected cells. Our approach reveals that despite heterogeneous activation of signaling, individual cells adopt predominantly prototypical death morphologies. Nevertheless, PCD is executed heterogeneously within a uniform population of virus-infected cells and varies over time. These findings demonstrate that image-based phenotyping can provide valuable insights into cell fate decisions, complementing molecular assays.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23ea913674617fc1b694df526cfac8afeb0fe5af" target='_blank'>
              Deep learning-based image classification reveals heterogeneous execution of cell death fates during viral infection
              </a>
            </td>
          <td>
            Edoardo Centofanti, Alon Oyler-Yaniv, Jennifer Oyler-Yaniv
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND
Cancer-Associated Fibroblasts (CAFs) constitute a heterogeneous group of cells critical for the remodeling of the tumor microenvironment (TME). Given their significant impact on tumor progression, particularly in skin cancers, a deeper understanding of their characteristics and functions is essential.


METHODS
This study employed a single-cell transcriptomic analysis to explore the diversity of CAFs within three major types of skin cancer: basal cell carcinoma, melanoma, and head and neck squamous cell carcinoma. We applied analytical techniques, including Gene Ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), Gene Set Enrichment Analysis (GSEA), pseudotime tracking, metabolic profiling, and stemness assessment to delineate and define the functional attributes of identified CAF subgroups.


RESULTS
Our analysis successfully delineated nine distinct CAF subgroups across the studied tumor types. Of particular interest, we identified a novel CAF subtype, designated as C0, exclusive to basal cell carcinoma. This subtype exhibits phenotypic traits associated with invasive and destructive capabilities, significantly correlating with the progression of basal cell carcinoma. The identification of this subgroup provides new insights into the role of CAFs in cancer biology and opens avenues for targeted therapeutic strategies.


CONCLUSION
A pan-cancer analysis was performed on three cancers, BCC, MA, and HNSCC, focusing on tumor fibroblasts in TME. Unsupervised clustering categorized CAF into nine subpopulations, among which the C0 subpopulation had a strong correspondence with BCC-CAF and an invasive- destructive-related phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b65ae0c3c55b73c69971a741aaa77cb534354a3" target='_blank'>
              Pan-Cancer Single-Cell Analysis Revealing the Heterogeneity of Cancer-Associated Fibroblasts in Skin Tumors.
              </a>
            </td>
          <td>
            Yichi Zhang, Zhijie Zhao, Wenyi Huang, Byeong Seop Kim, Li Lin, Xin Li, Mengyuan Hou, Li Li, Yan Zhang, Wenjing Xi, G. Chai
          </td>
          <td>2024-09-25</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) -- the expansion of somatically-mutated hematopoietic cells in blood -- is common in solid cancers. CH is associated with systemic inflammation that may lead to cancer, but its impact on tumor biology is underexplored. Here, we report the effects of CH on the tumor microenvironment (TME) using 1,550 treatment-naive patient samples from the CPTAC cohort. CH was present in 18.3% of patients, with one-third of CH mutations also detectable in tumor-derived DNA from the same individual (CH-Tum), reflecting CH-mutant leukocyte infiltration. The presence of CH-Tum was associated with worse survival across cancers, particularly for glioblastoma. Transcriptomics and proteomics revealed that CH drives inflammation in the TME in a cancer- and CH driver-specific manner, and may improve immunotherapy responses. In glioblastoma, CH associated with pronounced macrophage infiltration, inflammation, and an aggressive, mesenchymal phenotype. Our findings demonstrate that CH shapes the TME, with potential applications as a biomarker in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df335cc5550a0c8767d76bf480da6e95ae3c38e1" target='_blank'>
              Inflammatory reprogramming of the tumor microenvironment by infiltrating clonal hematopoiesis is associated with adverse outcomes in solid cancer
              </a>
            </td>
          <td>
            M. Buttigieg, C. Vlasschaert, A. G. Bick, R. Vanner, M. Rauh
          </td>
          <td>2024-08-30</td>
          <td>None</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Effective drug discovery relies on combining target knowledge with functional assays and multi-omics data to understand chemicals’ molecular actions. However, the relationship between cell morphology and gene expression over time and across cell lines remains unclear. To explore this, we analyzed Cell Painting and L1000 data for 106 compounds across three cell lines from osteoblast, lung, and breast tumors (U2OS, A549, and MCF7) at three time points (6h, 24h, 48h) using a 10µM concentration. We found significant time and cell line effects in Cell Painting data, with less pronounced effects in transcriptomics. Using Weighted Gene Co-expression Network Analysis (WGCNA) and enrichment analysis, we identified connections between cell morphology and gene deregulation for chemicals with similar biological effects (e.g., HDAC and CDK inhibitors). These findings suggest that while Cell Painting shows distinct patterns, both technologies offer complementary insights into compound-induced cellular changes, enhancing drug discovery and chemical risk assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4e8eb6d8184a42daa0a1c6534359ca9aca19c28" target='_blank'>
              Cell morphology and gene expression: tracking changes and complementarity across time and cell lines
              </a>
            </td>
          <td>
            Vanille Lejal, David Rouquié, Olivier Taboureau
          </td>
          <td>2024-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The composition of T cell subsets and tumor-specific T cell interactions within the tumor microenvironment (TME) contribute to the heterogeneity observed in breast cancer. Moreover, aberrant tumor metabolism is often intimately linked to dysregulated anti-tumor immune function of T cells. Identifying key metabolic genes that affect immune cell interactions thus holds promise for uncovering potential therapeutic targets in the treatment of breast cancer. This study leverages single-cell transcriptomic data from breast cancer to investigate tumor-specific T-cell subsets and their interacting subnetworks in the TME during cancer progression. We further assess the metabolic pathway activities of tumor-specifically activated T-cell subsets. The results reveal that metabolic pathways involved in insulin synthesis, secretion, degradation, as well as fructose catabolism, significantly influence multiple T cell interactions. By integrating the metabolic pathways that significantly up-regulate T cells in tumors and influence their interactions, we identify key abnormal metabolic genes associated with T-cell collaboration and further develop a breast cancer risk assessment model. Additionally, using gene expression profiles of prognosis-related genes significantly associated with aberrant metabolism and drug IC50 values, we predict targeted drugs, yielding potential candidates like GSK-J4 and PX-12. This study integrate the analysis of abnormal T-cell interactions and metabolic pathway abnormalities in the breast cancer TME, elucidating their roles in cancer progression and providing leads for novel breast cancer therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f74ba362286710c5155a46d5e09eccd304f2cf3d" target='_blank'>
              Computational dissection of the regulatory mechanisms of aberrant metabolism in remodeling the microenvironment of breast cancer.
              </a>
            </td>
          <td>
            , , Yu Zhao, Na Sun, , Juan Xu
          </td>
          <td>2024-10-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The alteration of the glioblastoma genome by epigenetic mechanisms that share functions with normal developmental processes, such as self-renewal and fate specification of neural stem cells (NSC), is a key piece of evidence that links brain cancer pathogenesis with dysregulated stem cell functions. A patient-specific comparison of glioblastoma stem cells (GSC) with NSC, a putative cell of origin of at least a proportion of these tumours, is not feasible as patient-matched endogenous NSC are not surgically accessible and all epigenetic studies in glioblastoma have so far compared epigenetic changes of different tumours with each other or to comparators obtained from foetal brains or an unrelated donor. We reasoned that availability of syngeneic GSC and NSC pairs would allow to identify crucial epigenetic differences on a patient-specific basis and would provide essential therapeutic contrast to define disease-and patient-intrinsic biomarkers of drug response that are less confounded by germline variation.



 We have derived GSC from IDH-wildtype glioblastoma and harnessed state-of-the-art stem cell technologies to generate patient-matched fibroblast-derived EPSC, which were induced to NSC (iNSC). We generated transcriptome, DNA methylome, ChIP-Seq (H3K4me3, H3K27ac, H3K3me3, H3K27me3) and ATACSeq datasets for 10 GIC/iNSC pairs.



 Comparative integrated analysis has shown that redistribution of selected histone marks as well as shifts in chromatin states across the genome identifies differentially expressed genes, including novel druggable regulatory mechanisms in GSC, which are specific of the neoplastic context. In particular, we identified METTL7B as an essential regulator of lineage specification and plasticity in human glioblastoma, with impact on both tumour size and invasiveness in an in vivo xenograft model. Single cell transcriptomic analysis of these tumours and of cerebral organoids derived from expanded potential stem cells (EPSCs) overexpressing METTL7B reveal a regulatory role for the gene in the neural stem cells-to-astrocyte differentiation trajectory. Mechanistically, METTL7B downregulates the expression of key neuronal differentiation players, including SALL2, via post-translational modifications of histone marks.



 SYNGN is an epigenetic platform which can identify novel disease mechanisms and druggable targets in glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be96bd21ac249c6fcb9083c430d82766e4fcbf5a" target='_blank'>
              P01.14.B SYNGN: A NOVEL EPIGENETIC PLATFORM TO DISCOVER PATIENT-SPECIFIC DISEASE MECHANISMS AND DRUGGABLE TARGETS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            S. Marino, C. Vinel, M. Constantinou
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pancreatic cancer is a highly lethal and aggressive disease, commonly characterized by a unique tumor microenvironment, where mutant KRAS signaling plays a pivotal role in its pathophysiology. To deepen our understanding of KRAS-dependent signaling in earliest tumor development, oncogenic KRAS (KRASG12D) expression was induced in human stem cell-derived pancreatic ductal-like organoids (PDLOs), followed by time-resolved single-cell RNA and bulk ATAC sequencing. This unique precancerous state model facilitated a comprehensive characterization of early oncogenic events at cellular resolution. Furthermore, a machine learning approach was applied to accurately predict KRAS-dependent cell identities. Within seven days, mutated KRAS triggers distinct gene signature programs related to extracellular matrix remodeling and inflammation. Although derived from dysplastic yet non-cancerous states, these gene signatures negatively correlated with overall survival in PDAC (pancreatic ductal adenocarcinoma) patients. The KRAS-dependent signatures retrieved with the machine learning approach allowed the grouping of 24 published PDAC cases, showing differences in niche composition and communication patterns. Various co-culture systems demonstrated that KRASG12D-expressing PDLOs could activate pancreatic stellate cells, inducing a cancer- associated fibroblast (CAF)-like state. Simultaneously, these KRAS-activated organoids were protected from T cell infiltration in vitro. Furthermore, in silico approaches were employed to reconstruct a virtual pancreatic cancer space, demonstrating that our PDLOs transcriptionally bridge healthy and malignant ductal states. This also revealed potential KRAS-driven pathways mediating CAF activation and T cell shielding preceding cancer onset. In summary, our human PDLO model enables us to explore the transition from normal to cancerous states, addressing gaps in our understanding of tumor development and tumor niche preparation.
 Citation Format: Chantal Allgoewer, Markus Breunig, Meike Hohwieler, Medhanie Mulaw, Alexander Kleger. Single-cell profiling unleashes KRAS-driven redrafting of the tumor microenvironment before cancer onset [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc4900be652d49c2d242f617b8845d915d29b" target='_blank'>
              Abstract C010: Single-cell profiling unleashes KRAS-driven redrafting of the tumor microenvironment before cancer onset
              </a>
            </td>
          <td>
            Chantal Allgoewer, M. Breunig, M. Hohwieler, Medhanie M Mulaw, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Prostate cancer (PC), which is a disease driven by the activity of the androgen receptor (AR), is the most commonly diagnosed malignancy and despite advances in diagnostic and treatment strategies, PC is the second most common cause of cancer mortality in men (Bray et al., 2018). Taxane-based chemotherapy is the only chemotherapy that prolongs survival in metastatic PC patients (Petrylak et al., 2004; Tannock et al., 2004). At the cellular level, taxanes bind to and stabilize microtubules (MTs) inhibiting all MT-dependent intracellular pathways. MTs are highly dynamic polymers that stochastically switch between phases of growth, shrinkage, and pause (Jordan and Wilson, 2004). Altered MT dynamics endow cancer cells with both survival and migratory advantages (Mitchison, 2012). Taxanes inhibit MT dynamics and alter the spatial organization of the MT network, thereby inhibiting intracellular trafficking of molecular cargo critical for tumor survival. In PC specifically, taxanes inhibit transcriptional activity downstream of MT stabilization (Thadani-Mulero et al., 2012) and AR nuclear accumulation (Darshan et al., 2011; Zhu et al., 2010). Different tubulin inhibitors, even from within the same structural class as the taxanes, affect distinct parameters of MT dynamics (Jordan and Wilson, 2004), yet the selection of taxane for chemotherapy is not based on the particular patterns of dynamic behavior of the MT cytoskeleton in individual patients. We envisage that systematic characterization using quantitative analysis of MT dynamics in PC patient cells expressing clinically relevant protein isoforms (Matov et al., 2024; Thoma et al., 2010), before and after treatment with each of the taxanes, will allow us to identify criteria for the selection of the most suitable drug combination at the onset of treatment. We link MT dynamics in the presence of AR variants and sensitivity/resistance to taxanes and connect fundamental research with clinically relevant concepts to elucidate cellular mechanisms of clinical response to taxanes and, thus, advance the customization of therapy. Our computational live-cell analysis addresses questions in the context of the inherent differences in MT homeostasis as a function of AR content in PC cells, the specific parameters of MT dynamics each of the taxanes affects, and how can this information be used to match endogenous patterns of MT dynamics with drug-modulated MT behavior. We investigate whether the sensitivity to taxanes, evaluated by computational analysis of MTs, can be linked to gene expression driven by AR and its variants, and whether the resistance to taxanes be linked to the presence of a specific AR splice variant, and can we identify which of the taxanes will be most effective based on the endogenous patterns of MT dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e90d5e5c268c6138c1b50b9fd253e67162113bd" target='_blank'>
              Computational Analysis of Treatment Resistant Cancer Cells
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-08-31</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Bone metastasis (BM) is a mortality-related event of late-stage cancer, with non-small cell lung cancer (NSCLC) being a common origin for BM. However, the detailed molecular profiling of the metastatic bone ecosystem is not fully understood, hindering the development of effective therapies for advanced patients. In this study, we examined the cellular heterogeneity between primary tumours and BM from tissues and peripheral blood by single-cell transcriptomic analysis, which was verified using multiplex immunofluorescence staining and public datasets. Our results demonstrate a senescent microenvironment in BM tissues of NSCLC. BM has a significantly higher infiltration of malignant cells with senescent characteristics relative to primary tumours, accompanied by aggravated metastatic properties. The endothelial-mesenchymal transition involved with SOX18 activation is related to the cellular senescence of vascular endothelial cells from BM. CD4Tstr cells, with pronounced stress and senescence states, are preferentially infiltrated in BM, indicating stress-related dysfunction contributing to the immunocompromised environment during tumour metastasis to bone. Moreover, we identify the SPP1 pathway-induced cellular crosstalk among T cells, vascular ECs and malignant cells in BM, which activates SOX18 and deteriorates patient survival. Our findings highlight the roles of cellular senescence in modulating the microenvironment of BM and implicate anti-senescence therapy for advanced NSCLC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae5ee75b17730b9ef56c88719d46b498e6167453" target='_blank'>
              Single-cell transcriptomic analysis of the senescent microenvironment in bone metastasis.
              </a>
            </td>
          <td>
            Shenglin Wang, Lu Ao, Huangfeng Lin, Hongxiang Wei, Zhaoyang Wu, Shuting Lu, Fude Liang, R. Shen, Huarong Zhang, Tongjie Miao, Xiaopei Shen, Jianhua Lin, Guangxian Zhong
          </td>
          <td>2024-09-04</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) constitute a prominent immune cell population within various solid cancers, playing a pivotal role in disease progression. Their increased numbers and frequencies often strongly correlate with resistance to therapy and reduced overall survival rates. Within the complex ecosystem of the tumor microenvironment (TME), activated cancer-associated fibroblasts (CAFs) are expanded and contribute significantly to tumor growth and metastasis, and to chemo- or immune-therapy resistance. CAFs exert a critical influence on TAM phenotype and functions by orchestrating the reprogramming of tissue-infiltrating monocytes, thereby modulating their survival and differentiation. This reciprocal interaction between TAMs and CAFs forms a crucial axis in fostering a suppressive crosstalk within the TME, mediated by a diverse array of signals exchanged between these cell types. Recent advancements in single-cell RNA sequencing (sc-RNA seq) technologies and spatial transcriptomics have enhanced our comprehension of the signaling dynamics at the interface between TAMs and CAFs, including their spatial distribution within the tissue. In this review, we delve into the latest discoveries elucidating the biology of TAM and CAF crosstalk. We examine the complexity of TAM-CAF and CAF-TAM interactions within the TME of solid cancers, with particular focus on ligand-receptor interactions and clinically significant targets for novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba817d130671717874a36642fd784b7dbabebd41" target='_blank'>
              At the interface of tumor-associated macrophages and fibroblasts: immune-suppressive networks and emerging exploitable targets.
              </a>
            </td>
          <td>
            E. Timperi, Hugo Croizer, Darya Khantakova, Mhaidly Rana, Martina Molgora, Jennifer L. Guerriero, F. Mechta-Grigoriou, Emanuela Romano
          </td>
          <td>2024-09-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3db9b2bc9f499477ff494631d94e30c054c0bf" target='_blank'>
              A Boolean model explains phenotypic plasticity changes underlying hepatic cancer stem cells emergence
              </a>
            </td>
          <td>
            Alexis Hernández-Magaña, Antonio Bensussen, Juan Carlos Martínez-García, E. Álvarez-Buylla
          </td>
          <td>2024-09-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) it though to develop through induction of different precursor lesions, however, the normal cell type that gives rise to these precancerous lesions and PDAC is unclear. Recent studies using mouse models and ex vivo cultured human cells suggest that the cellular origin of PDAC may be either acinar or ductal cells. Potential transcriptional markers associated with PDAC of different cellular origins have been identified. Here, we interrogate whether these markers are predictive of cellular of origin across models using mouse models where acinar and ductal cells (Ptf1aCreER or Sox9CreER) expressing oncogenic Kras in the absence of Trp53 or Pten give rise to PDAC tumors. We found that acinar- and ductal-cell-derived PDAC exhibit transcriptional differences both in vivo and in vitro when comparing within one genotype or sample source type (e.g. bulk tumor or primary cell lines). However, previously predicted markers of cellular origin had variable expression across models and sample types suggestive of differences specific to those sample sets. Using our larger sample sets and multiple genotypes and source types, we proposed a more comprehensive set of cell of origin markers for consideration. Application of these markers to our own data sets, however, failed to properly segregate acinar- and ductal-cell-derived PDAC into separate groups. This suggests that the transcriptional heterogeneity observed in our sample set was not driven by the tumors arising from different cellular origins. We also examined the relationship between cell of origin and patient PDAC transcriptional subtypes using our mouse model datasets. Our findings support previous research indicating that mouse models of acinar-cell-derived PDAC favor those of a classical subtype. However, our mouse PDAC samples all express similar or very low levels of the genes denoting the basal subtype, suggesting that the basal subtype was not strongly present in any of the datasets we examined. In summary, our data suggest that acinar and ductal cells converge into a very similar PDAC transcriptional state and that, if differences can be identified, they will likely be identified by non-transcript based readouts.
 Citation Format: Ken Chu, Alex Lee, Karnjit Sarai, Yan Dou, Wesley Hunt, Emma Croft, Atefeh Samani, Farnaz Taghizedeh, Claire Dubois, Stephane Flibotte, Janel Kopp. Cell of origin does not underlie transcription heterogeneity in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7619576b4dd9f9188da32a5b5d7af49469c5cb88" target='_blank'>
              Abstract A025: Cell of origin does not underlie transcription heterogeneity in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ken Chu, Alex Y. L. Lee, Karnjit Sarai, Yan Dou, Wesley Hunt, Emma Croft, Atefeh Samani, Farnaz Taghizedeh, Claire L. Dubois, Stephane Flibotte, Janel L. Kopp
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Objectives: Mesenchymal neoplasms, or sarcomas, are a diverse and diagnostically challenging group including >150 histotypes, and are biased to pediatric patients, comprising 20% of pediatric solid tumor diagnoses compared to 1% in adults. A universal molecular taxonomy and classification system for sarcoma would be an invaluable tool for patient diagnosis and subtype discovery. We previously demonstrated a transcriptional pan-cancer taxonomy and classifier; however, we neither captured the full diversity of mesenchymal neoplasms nor classified preclinical models. We therefore created a second-generation, mesenchymal-specific atlas with improved diagnostic coverage, refining our ability to describe the molecular landscape of sarcoma. Methods: Fresh frozen, poly-A enriched RNA-seq datasets including mesenchymal tumors (n=24) were combined with the UCSC Treehouse Childhood Cancer Compendium. Transcriptomic data (n=14,315) were uniformly processed and all diagnostic labels were harmonized to ICD-O v3.2. To create a taxonomy, processed expression matrices were clustered one level deep and mesenchymal classes were selected and clustered to completion using established tools. To determine cluster identities, each cluster was annotated using gene expression profiles and available clinical and molecular data. We then trained a companion classifier to classify preclinical models. RNA-seq data for cell lines were obtained from the Cancer Cell Line Atlas and patient derived xenografts (PDX) from the UCSC Treehouse database. Results: All mesenchymal neoplasms (n=2,153, 82 ICD-O codes) form one cluster at the pan-cancer level, an increase of 190% in sample size and 200% in histotype diversity over our previous effort[YB1](n=1,125, 43 codes). The resulting taxonomy comprises 153 clusters over seven hierarchical levels. Out of 109 terminal nodes, 60% represent a consensus diagnosis (defined as >50% of cases classified under a single code). Of these, 50% represent novel subtypes. The remaining 40% of terminal nodes are diagnostically heterogeneous, with 25% of these containing a unifying genetic lesion, suggesting current diagnostic conventions do not capture the diversity of existing molecular entities. Sarcoma cell lines (n=54) show variable assignment to their disease of origin: several histotypes lose disease specificity, while fusion-driven histotypes conserve parental identity. Fusion-plasmid-generated models fail to recapitulate their in vivo profile. PDX’s (n=33) broadly maintain expected classification. Conclusion: RNA-seq continues to facilitate subtype discovery and disease classification in ongoing patients and preclinical models. Our results suggest a large portion of mesenchymal entities require molecular, rather than histotypic definitions. Classification of preclinical models resolves their suitability for disease modeling. The expression profiles outputted by our methods can be leveraged for future therapeutic nominations. We are expanding this effort to reach 3,000 samples this year and are open to the community to contribute.
 Citation Format: Joshua O. Nash, Pedro L. Ballestar, Scott Davidson, Astra Schwertschkow, Yael Babichev, Jodi Lees, Noa Alon, Nalan Gokgoz, Stephen M. Yu, Kyoko Yuki, Miranda Lorenti, Zhanqin Liu, Alaina McGoey, Famida Spatare, Bernard Castro, Kim Tsoi, Hagit Peretz-Soroka, Jack Brzezinski, Anita Villani, Albiruni Razaq, Abha Gupta, Elizabeth Demicco, Joanna Przybyl, Matt van de Rijn, Livia Garzia, Jay Wunder, Irene L. Andrulis, David Malkin, Rose Chami, Brendan C. Dickson, Rebecca A. Gladdy, Adam Shlien. A transcriptional atlas provides a universal diagnostic platform for mesenchymal tumors and validation of preclinical models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee427e26b14b0caf99c5649ed800749acc3f621a" target='_blank'>
              Abstract B034: A transcriptional atlas provides a universal diagnostic platform for mesenchymal tumors and validation of preclinical models
              </a>
            </td>
          <td>
            Joshua O Nash, Pedro L. Ballestar, Scott Davidson, Astra Schwertschkow, Y. Babichev, Jodi Lees, N. Alon, N. Gokgoz, S. M. Yu, Kyoko E. Yuki, Miranda Lorenti, Zhanqin Liu, Alaina McGoey, Famida Spatare, Bernard Castro, Kim Tsoi, Hagit Peretz-Soroka, J. Brzezinski, A. Villani, Albiruni Razaq, Abha A. Gupta, Elizabeth G. Demicco, Joanna Przybyl, Matt van de Rijn, Livia Garzia, Jay S. Wunder, I L Andrulis, D. Malkin, R. Chami, Brendan C. Dickson, R. Gladdy, A. Shlien
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Background: Pancreatic ductal adenocarcinoma (PDAC) is highly resistant to chemoradiation and immune therapies. There is limited biomarker to identify subsets of PDAC that responds to immunotherapy or targeted therapy. We have identified a subset of PDACs (approximately 25%) carrying COMPASS-like complex (CLC) gene mutations. Analysis of the clinicopathologic and molecular features of this PDAC subset revealed that these tumors are an aggressive group of PDACs with poor or squamous histologic differentiation, increased rates of TP53 mutations, and poor patient survival compared to control PDACs. Our preliminary data showed that PDACs carrying mutations in two members of the CLC genes (KDM6A and KMT2D) have immunosuppressive tumor microenvironment (TME), suggesting that CLC genes are candidate biomarkers for immunotherapy. However, a comprehensive analysis of the immune composition, immune checkpoint expression, and the relationship between tumor cells and immune cells in CLC gene mutated PDACs compared to controls has not been performed. In this study, we characterized the immune composition in CLC gene mutated PDACs compared to paired control PDACs using tyramide signal amplification multiplex fluorescent immunohistochemistry (mfIHC). Methods: Human PDAC tissue slides were stained with 2 mfIHC panels and cell types were defined by different markers: panel 1 (PanCK, CD163, PD-L1, CD3, CD8, FoxP3) and panel 2 (PanCK, CD163, CD3, CD8, TIGIT, TIM3). Random image sections were selected in each sample (5∼8 sections per sample) and divided into wild-type (WT) group (n=57) and mutant group (n=41) and analyzed using InForm Cell Analysis software. Tissue segmentation training was performed to separate stromal and epithelial elements. Cell percentage, cell density, and nearest neighbor distance of each cell type were calculated using R program. Results: The CLC gene mutation is associated with elevated epithelial cells (ECs) and PDL1+ ECs and increased infiltration of exhausted cytotoxic T cells (TIGIT+/TIM3+ CD8 T cells). CLC gene mutated PDACs are characterized by spatial proximity between exhausted T cells (TIGIT+) and epithelial cells/antigen presenting cells (APCs). Higher number of epithelial cells is associated with reduced infiltration of helper T cells in the mutant PDAC tissue. We also observed that high numbers of APC in proximity to CD4 T cells were positively correlated with the proximity of EC and CD8 cytotoxic T cells as well as the proximity of CD4 T cells and cytotoxic T cells in the WT group but not in the CLC gene mutant group. Conclusion: Our results provide insight into the effects of CLC gene mutations on shaping the immune microenvironment of PDAC and suggest that CLC gene mutated PDACs have an immunosuppressive immune microenvironment than other PDACs and may respond differently to immunotherapy.
 Citation Format: Shungang Zhang, Elaina Daniels, Jake McGue, Hong Sun Kim, Dafydd Thomas, Timothy Frankel, Jiaqi Shi. Multiplex fluorescent immunohistochemistry reveals immunosuppressive tumor microenvironment in COMPASS-like complex gene mutated pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2790a085665b4a36a3c8b84d6bede9ac8c16c70" target='_blank'>
              Abstract A040: Multiplex fluorescent immunohistochemistry reveals immunosuppressive tumor microenvironment in COMPASS-like complex gene mutated pancreatic cancer
              </a>
            </td>
          <td>
            Shungang Zhang, Elaina Daniels, Jake McGue, H. Kim, Dafydd Thomas, Timothy L. Frankel, Jiaqi Shi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0102937166e631177aafa289ad76311941f3d10" target='_blank'>
              Genomic and immune heterogeneity of multiple synchronous lung adenocarcinoma at different developmental stages
              </a>
            </td>
          <td>
            Yue Zhao, Jian Gao, Jun Wang, F. Fan, Chao Cheng, Danwen Qian, Ran Guo, Yang Zhang, Ting Ye, Marcellus Augustine, Yicong Lin, Jun Shang, Hang Li, Yunjian Pan, Qingyuan Huang, Haiqing Chen, Han Han, Zhendong Gao, Qiming Wang, Shiyue Zhang, Mou Zhang, F. Fu, Yueren Yan, Shanila Fernandez Patel, R. Vendramin, Hui Yuan, Yawei Zhang, Jiaqing Xiang, Hong Hu, Yihua Sun, Yuan Li, Kevin Litchfield, Zhiwei Cao, Haiquan Chen
          </td>
          <td>2024-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Background: Triple-negative breast cancer (TNBC) accounts for 15-20% of breast cancer (BC) cases but due to its aggressive nature and the lack of targeted treatment options, represents a disproportionately higher number of BC-related deaths. Compared to other BC subtypes, TNBCs are more inflamed with M2-like tumor-associated macrophages (TAMs) and cytotoxic T-lymphocytes (CTLs) which are associated with poor and good prognosis, respectively. Clinical trials have proven that immunogenic cancers can be effectively treated through immune checkpoint blockade (ICB), a treatment strategy that prevents cancer cells from silencing CTL-mediated attack. However, CTLs can be suppressed by M2-like TAMs which can limit ICB efficacy and support tumor growth. The hedgehog (HH) signaling pathway is highly activated in TNBC and higher expression of GLI1 is associated with worse overall survival. Recent pan-cancer analyses highlight a significant correlation between activated HH signaling and characteristics of immune evasion. We aim to clarify how activated HH signaling regulates resident innate and adaptive immune cells in the tumor microenvironment (TME) in TNBC. Methods: Primary K14-cre; Brca1 fl/fl ; P53 fl/fl (KBP) murine TNBC cells were transfected with CRISPR/Cas9 + sgRNA targeting GLI1. After isolating single cell clones and confirming GLI1 knockout (KO), KBP and KBP-GLI1-KO cells were orthotopically injected into the mammary fat pad of syngeneic immune-competent female mice and allowed to grow for 6-8 weeks. Tumor volume was measured once tumors were palpable, and tumors were harvested once the largest tumor reached ethical endpoint. Tumors were weighed at harvest and fluorescence activated cell sorting (FACS) was performed to determine if there was an effect on the composition of immune populations. Results: At harvest, KBP-GLI1-KO tumors were delayed in growth and significantly smaller than KBP tumors. FACS analysis showed that GLI1-KO tumors had increased CD4+ and CD8+ T cells and a reduced amount of Foxp3+ CD4+ Tregs. Furthermore, GLI1-KO tumors had significantly fewer F4/80+ CD11b+ TAMs within the TME. Amongst TAM populations, there was reduced CD206+ M2-like TAMs and an increase in CD80+ M1-like TAMs upon loss of GLI1. Similar trends were observed when KBP allografts were treated with the GLI1 inhibitor, GANT61. Immunohistochemical and RNAseq analysis are ongoing. Similar experiments with other KBP-GLI1-KO clones are currently being done. Conclusion: Here, we show that tumoral GLI1 not only promotes tumor growth, but also supports an immunosuppressive TME in TNBC. Abolishing GLI1 converted tumors to a pro-inflammatory phenotype marked by an increase in CD4+ and CD8+ T cells and a reduction in CD206+ M2-like TAMs. Future experiments are aimed at revealing the mechanisms by which GLI1 regulates immune cell recruitment to the TME and testing the effect of combination immunotherapy. Overall, these preliminary findings identify GLI1 as a potential therapeutic target to limit tumor growth and promote anti-tumor immunity.
 Citation Format: Aidan J Gray, Wanda Marini, Kiichi Murakami, Michael Reedijk. Tumoral GLI1 controls the resident innate and adaptive tumor immune microenvironment in triple negative breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr B022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a42f1e3465f38088513c262358ea59c76984656c" target='_blank'>
              Abstract B022: Tumoral GLI1 controls the resident innate and adaptive tumor immune microenvironment in triple negative breast cancer
              </a>
            </td>
          <td>
            Aidan J Gray, Wanda Marini, K. Murakami, M. Reedijk
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e985501495b95f9eef3f50592dfe9ef56c04c3f" target='_blank'>
              Digital spatial profiling identifies the tumor center as a topological niche in prostate cancer characterized by an upregulation of BAD
              </a>
            </td>
          <td>
            Ann-Kathrin Huber, A. Kaczorowski, Felix Schneider, Sarah Böning, M. Görtz, David Langhoff, C. Schwab, Albrecht Stenzinger, Markus Hohenfellner, A. Duensing, Stefan Duensing
          </td>
          <td>2024-08-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="In the first pan-cancer analysis of the tumor vasculature, Pan and colleagues profile nearly 200,000 endothelial and mural cells (ECs and MCs), identifying novel subclusters and cell states using consensus trajectory inference. They identify differentiation trajectories in vascular and lymphatic endothelial cells, and subtype the pericyte population. During sprouting angiogenesis, venous cells dedifferentiate and transition to capillary and, finally, arterial ECs. Capillary ECs transition via "three angiogenic stages" (SI-SIII) during which APLN+ TipS1 cells were identified as potential modulators of tumor-induced neovascularization and anti-angiogenic therapy response. In lymphatic ECs, differentiation was inversely correlated between the lymphangiogenic (T1) and antigen-presenting (T2) trajectories, with T2 associated with a better prognosis. While several pericyte clusters were identified, BASP1+ matrix-associated pericytes were associated with APLN+ TipS1 cells and had a worse prognosis. These findings present transcriptional validation of previous experimental findings and serve as a resource to examine the tumor vascular microenvironment in detail.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5f776484aa47a465eea06a0fd10d87081b85a4" target='_blank'>
              The Pan-tumor Vasculature Under the Transcriptomic Magnifying Glass.
              </a>
            </td>
          <td>
            Krish Skandha Gopalan, G. Bergers
          </td>
          <td>2024-09-06</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3b4d74609d2880453dad8460ce60ad53367b365" target='_blank'>
              MerlinS13 phosphorylation regulates meningioma Wnt signaling and magnetic resonance imaging features
              </a>
            </td>
          <td>
            C. D. Eaton, Lauro Avalos, S. J. Liu, Zhenhong Chen, Naomi Zakimi, T. Casey-Clyde, Paola Bisignano, Calixto-Hope G Lucas, Erica Stevenson, A. Choudhury, H. Vasudevan, Stephen T. Magill, Jacob S Young, N. Krogan, J. Villanueva‐Meyer, D. Swaney, D. Raleigh
          </td>
          <td>2024-09-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Glioblastoma (GBM) is known as the most aggressive brain tumor and is characterized by a high heterogeneity and a median patient survival time around 15 months. Phenotyping based on cell mechanics is increasingly recognized as a potential prognostic marker for tumor aggressiveness. We have previously shown that cell and nuclear mechanical properties vary between different grades of gliomas and may be used to differentiate between GBM of different aggressiveness. Here, we find that the levels of lamin proteins can serve as an indicator of GBM aggressiveness. In patient-derived GBM cell lines, we found that cells from different GBM express different lamin levels. Nuclear size correlates positively with the ratio between lamin A and lamin B1 while nuclear stiffness increases with the levels of both lamin A and lamin B1. A simple mechanical model suggests that lamin A and lamin B1 act like springs in series. We also show that cells proliferate faster in GBM cell lines expressing higher lamin A levels. Downregulating lamin A expression in these cells reverse the aggressive phenotype. In contrast with breast cancer cells for which reduced lamin A levels favor cell migration in a confined environment, increased levels of lamin A may facilitate the invasion of more aggressive GBM cell lines in the soft environment of the brain. Furthermore, since nuclear deformation is a hallmark of malignancy in cancer cells, our results suggest that nuclear shape and mechanics may serve as prognosis biomarkers for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07960217e9f3d9dc620d20f2cac26b49d0fcb858" target='_blank'>
              Lamins regulate nuclear mechanics and shape to control glioblastoma cell proliferation, migration and invasion
              </a>
            </td>
          <td>
            Xiuyu Wang, David Pereira, Isabelle Perfettini, Florent Peglion, Ryszard Wimmers, Ananya Roy, K. Forsberg-Nilsson, Alexandre Baffet, Sandrine Etienne-Manneville, Jean-Baptiste Manneville
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a6029311c7be5cdea36f9e6f715640956c9f739" target='_blank'>
              Biophysically interpretable inference of cell types from multimodal sequencing data
              </a>
            </td>
          <td>
            T. Chari, Gennady Gorin, L. Pachter
          </td>
          <td>2024-09-01</td>
          <td>Nature computational science</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="BACKGROUND
Coronary atherosclerotic plaques susceptible to acute coronary syndrome have traditionally been characterized by their surrounding cellular architecture. However, with the advent of intravascular imaging, novel mechanisms of coronary thrombosis have emerged, challenging our contemporary understanding of acute coronary syndrome. These intriguing findings underscore the necessity for a precise molecular definition of plaque stability. Considering this, our study aimed to investigate the vascular microenvironment in patients with stable and unstable plaques using spatial transcriptomics.


METHODS
Autopsy-derived coronary arteries were preserved and categorized by plaque stability (n=5 patients per group). We utilized the GeoMx spatial profiling platform and Whole Transcriptome Atlas to link crucial histological morphology markers in coronary lesions with differential gene expression in specific regions of interest, thereby mapping the vascular transcriptome. This innovative approach allowed us to conduct cell morphological and spatially resolved transcriptional profiling of atherosclerotic plaques while preserving crucial intercellular signaling.


RESULTS
We observed intriguing spatial and cell-specific transcriptional patterns in stable and unstable atherosclerotic plaques, showcasing regional variations within the intima and media. These regions exhibited differential expression of proinflammatory molecules (eg, IFN-γ [interferon-γ], MHC class II, proinflammatory cytokines) and prothrombotic signaling pathways. By using lineage tracing through spatial deconvolution of intimal CD68+ (cluster of differentiation 68) cells, we characterized unique, intraplaque subpopulations originating from endothelial, smooth muscle, and myeloid lineages with distinct regional locations associated with plaque instability. In addition, unique transcriptional signatures were observed in vascular smooth muscle and CD68+ cells among plaques exhibiting coronary calcification.


CONCLUSIONS
Our study illuminates distinct cell-specific and regional transcriptional alterations present in unstable plaques. Furthermore, we characterize the first spatially resolved, in situ evidence supporting cellular transdifferentiation and intraplaque plasticity as significant contributors to plaque instability in human coronary atherosclerosis. Our results provide a powerful resource for the identification of novel mediators of acute coronary syndrome, opening new avenues for preventative and therapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a73fe256c8ad5204b69d364f2110a060834134" target='_blank'>
              Spatial Transcriptomic Approach to Understanding Coronary Atherosclerotic Plaque Stability.
              </a>
            </td>
          <td>
            M. Gastanadui, Camilla Margaroli, Silvio Litovsky, Robert P Richter, Dezhi Wang, D.D. Xing, J. Wells, Amit Gaggar, V. Nanda, Rakesh P Patel, Gregory A Payne
          </td>
          <td>2024-09-05</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Triple-negative breast cancer (TNBC) is the most aggressive subtype of breast cancer. TNBC displays exceptional transcriptional heterogeneity, both between patients and intratumorally, and disproportionately affects underserved women. Hypoxia, a state of low oxygen levels, is often present in the solid tumor microenvironment. In TNBC, hypoxia is a key driver of aggressiveness and therapeutic resistance. Past bulk RNA sequencing (RNAseq) studies in cancer research have lacked spatial resolution, failing to uncover precise transcriptional differences within tumor regions. This spatial limitation has been especially problematic for studying transcriptional heterogeneity of the cellular hypoxia response within tumors. By integrating single-cell RNA sequencing and spatial transcriptomic analyses in a TNBC xenograft tumor model, our research has uncovered a novel hypoxia gene signature, specific to cancer cells. Elevated expression of this hypoxia signature predicts poorer prognosis in breast cancer patients, especially within TNBC patients and those of African ancestry. This suggests that hypoxia may play a role in the racial disparities observed in TNBC prognosis, potentially informing treatment strategies. This workflow introduces a novel framework for translating general spatial transcriptomics data into clinically relevant and accurate gene signatures.
 Citation Format: Kent Schechter, Long C. Nguyen, Geetha P. Yerradoddi, Rania Bassiouni, John D. Carpten, Marsha R. Rosner. Novel cancer-specific hypoxia signature predicts poor prognosis and highlights racial disparities in breast cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C105.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43ba633337a4ee769f8bd92a6f7364c821869174" target='_blank'>
              Abstract C105: Novel cancer-specific hypoxia signature predicts poor prognosis and highlights racial disparities in breast cancer
              </a>
            </td>
          <td>
            Kent Schechter, Long C Nguyen, G. P. Yerradoddi, R. Bassiouni, John D. Carpten, M. Rosner
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Advances in omics technologies have allowed spatially resolved molecular profiling of single cells, providing a window not only into the diversity and distribution of cell types within a tissue but also into the effects of interactions between cells in shaping the transcriptional landscape. Cells send chemical and mechanical signals which are received by other cells, where they can subsequently initiate context-specific gene regulatory responses. These interactions and their responses shape the individual molecular phenotype of a cell in a given microenvironment. RNAs or proteins measured in individual cells, together with the cells' spatial distribution, provide invaluable information about these mechanisms and the regulation of genes beyond processes occurring independently in each individual cell. SpaCeNet is a method designed to elucidate both the intracellular molecular networks (how molecular variables affect each other within the cell) and the intercellular molecular networks (how cells affect molecular variables in their neighbors). This is achieved by estimating conditional independence relations between captured variables within individual cells and by disentangling these from conditional independence relations between variables of different cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8ef214444a5271219b33a7d73c5e44038be1f04" target='_blank'>
              Spatial Cellular Networks from omics data with SpaCeNet.
              </a>
            </td>
          <td>
            Stefan Schrod, Niklas Lück, R. Lohmayer, S. Solbrig, Dennis Völkl, Tina Wipfler, K. Shutta, Marouen Ben Guebila, Andreas Schäfer, Tim Beißbarth, Helena U. Zacharias, Peter J Oefner, John Quackenbush, M. Altenbuchinger
          </td>
          <td>2024-09-04</td>
          <td>Genome research</td>
          <td>2</td>
          <td>12</td>
        </tr>

        <tr id="


 Meningiomas represent the most common primary brain malignancies in adults with a subset of tumors exhibiting aggressive clinical behavior. Immunotherapy might present a new treatment strategy but is highly dependent on the immunological composition of the tumor microenvironment. Our previous data have shown that tumor-associated macrophages (TAMs) make up the main immune cell population in meningiomas with a significant negative impact on patient outcome. In this study, we investigated whether TAMs from meningioma tissue could be reprogrammed to an immunologically active and tumoricidal phenotype.



 For this purpose, CD11b+ sorted macrophages derived from tumor samples from > 40 patients including a substantial number of clinically aggressive meningiomas were treated with small molecule inhibitors targeting the colony-stimulating factor-1 receptor (CSF-1R). In a first analysis, the direct treatment response of CD11b+ patient-derived macrophages has been investigated by various techniques including flow cytometry and bulk RNA-sequencing of treated TAMs and further analysis of the macrophage-conditioned media after treatment. In addition, we studied the influence of CSF-1R-targeted macrophage treatment on the phenotype and functional activity of T cells to assess a potential indirect treatment response in the tumor microenvironment.



 Our data revealed that CSF-1R-targeted treatment of CD11b+ TAMs induced significant changes in the protein and gene expression of macrophage polarization markers towards a more immunologically active state and a significantly higher metabolic nitric oxide production as another sign of immunological activation. Subsequent analysis of indirect effects on T cells showed not only a significantly increased expression of the T cell activation marker CD69+, but also a significantly increased tumor cell killing by autologous T cells after macrophage-targeted treatment.



 Together these data suggest both a direct and indirect CSF-1R-targeted macrophage treatment response in the local tumor microenvironment and give first promising results on the efficacy of macrophage-targeted immunotherapy in human meningiomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/249de0d80cf126001967c7b750ee6cedeab5320f" target='_blank'>
              P19.14.A TARGETING TUMOR-ASSOCIATED MACROPHAGES IN THE IMMUNE-MICROENVIRONMENT OF MENINGIOMAS
              </a>
            </td>
          <td>
            C. Lotsch, R. Warta, P. Dao Trong, S. Krieg, F. Sahm, A. Unterberg, C. Herold-Mende
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880d6fbe977b9fa0abf41b1330834beac1e2a6aa" target='_blank'>
              A patient-derived cell model for malignant transformation in IDH-mutant glioma
              </a>
            </td>
          <td>
            Olga Kim, Zach Sergi, Guangyang Yu, Kazu Yamamoto, Martha Quezado, Zied Abdullaev, Danel R Crooks, S. Kishimoto, Qi Li, Peng Lu, Burchelle N. Blackman, T. Andresson, Xiaolin Wu, Bao Tran, Jun S. Wei, Wei Zhang, Meili Zhang, Hua Song, Javed Khan, Murali C. Krishna, J. Brender, Jing Wu
          </td>
          <td>2024-09-10</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 Adult glioma patients rarely benefit from immunotherapy. This lack of efficacy has been related to gliomas immune “cold” microenvironment, which is commonly observed in these tumors and reduces lymphocyte-mediated antitumor responses. Here, we investigated the immune microenvironment of rare adult gliomas with high lymphocytic infiltration in order to evaluate the biology of this unusual immune response, its prognostic value and its potential role as predictive biomarker for immunotherapy strategies.



 Two retrospective cohorts of adult gliomas were analyzed: a supervised cohort of 206 samples enriched for tumors with high lymphocytic infiltration (including glioblastoma IDH wildtype [GBM], gemistocytic variant of astrocytoma IDH-mutant, ganglioglioma [GGG] and pleomorphic xanthoastrocytoma [PXA], and hypermutated mismatch repair deficient gliomas [MMRd]), and a consecutive cohort of 112 samples as control of the usual epidemiology. Immune microenvironment was characterized through (i) an analysis of spatial patterns and semi-automated quantification of T lymphocytes on CD3 immunolabeling and (ii) deconvolution of immune cells content from bulk RNA sequencing of FFPE tumor tissue. Cellular and spatial patterns of immune infiltration were correlated with demographic, histomolecular and outcome features. We investigated the underlying biology through multiplex immunolabeling and single-cell RNAseq of FACS-sorted intratumoral immune cells. Results: We identified three main spatial patterns (perivascular, peritumoral, diffuse intratumoral) of T cells infiltration in addition to the most usual aspect of scarce tumor-infiltrating lymphocytes. T cell proportion exhibited a wide range of distribution and was significantly associated with tumor type and IDH-status, with GBM, PXA and GGG exhibiting the highest proportions. Using transcriptomic signatures of immune cells and deconvolution of bulk RNAseq data, we observed recurrent patterns of associated immune cell types across diverse tumor types. We analyzed the correlations of these spatial and cellular immune patterns with tumor types and genetic alterations. Multiplex immunolabeling and single cell RNAseq revealed distinct T cells subpopulations, as well as potential functional states and cell-cell communication pathways driving these states. T cell infiltration had a significant prognostic impact among GBM patients.



 Adult gliomas with high lymphocytic content stands for a population of interest to better understand immune responses in the brain parenchyma and develop novel immunotherapy clinical trials. Our results suggest that therapeutic evaluation requires an adjustment due to the prognostic impact of immune microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/204034fedcd671ddafac89899df457a2d6c8c475" target='_blank'>
              OS12.7.A INTEGRATED DIGITAL PATHOLOGY AND TRANSCRIPTOMIC ANALYSIS OF ADULT GLIOMAS WITH HIGH LYMPHOCYTE INFILTRATION IDENTIFIES CELLULAR AND SPATIAL DETERMINANTS OF IMMUNE INFILTRATION WITH PROGNOSTIC VALUE
              </a>
            </td>
          <td>
            L. Salhi, J. Lerond, A. A. Nadaradjane, L. Drouiche, M. M. Otero Pizarro, L. Hanna Doumit Sakr, K. Labrèche, S. Kacimi, C. Bellamy, A. Picca, L. Stimmer, J. Brocic, Q. Letourneur, B. Barka, C. Carpentier, Y. Marie, H. Madry, J. Bhalshankar, R. Nicolle, H. Malaizé, B. Mathon, K. Mokhtari, A. Idbaih, K. Hoang-Xuan, M. Sanson, A. Alentorn, A. Duval, P. Le Rouzic, M. Dieu-Nosjean, I. Bloch, M. Touat, F. Bielle
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The impact of aging on the immune landscape of luminal breast cancer (Lum-BC) is poorly characterized. Understanding the age-related dynamics of immune editing in Lum-BC is anticipated to improve the therapeutic benefit of immunotherapy in older patients. To this end, here we applied the 'multiple iterative labeling by antibody neo-deposition' (MILAN) technique, a spatially resolved single-cell multiplex immunohistochemistry method. We created tissue microarrays by sampling both the tumor center and invasive front of luminal breast tumors collected from a cohort of treatment-naïve patients enrolled in the prospective monocentric IMAGE (IMmune system and AGEing) study. Patients were subdivided into three nonoverlapping age categories (35-45 = 'young', n = 12; 55-65 = 'middle', n = 15; ≥70 = 'old', n = 26). Additionally, depending on localization and amount of cytotoxic T lymphocytes, the tumor immune types 'desert' (n = 22), 'excluded' (n = 19), and 'inflamed' (n = 12) were identified. For the MILAN technique we used 58 markers comprising phenotypic and functional markers allowing in-depth characterization of T and B lymphocytes (T&B-lym). These were compared between age groups and tumor immune types using Wilcoxon's test and Pearson's correlation. Cytometric analysis revealed a decline of the immune cell compartment with aging. T&B-lym were numerically less abundant in tumors from middle-aged and old compared to young patients, regardless of the geographical tumor zone. Likewise, desert-type tumors showed the smallest immune-cell compartment and were not represented in the group of young patients. Analysis of immune checkpoint molecules revealed a heterogeneous geographical pattern of expression, indicating higher numbers of PD-L1 and OX40-positive T&B-lym in young compared to old patients. Despite the numerical decline of immune infiltration, old patients retained higher expression levels of OX40 in T helper cells located near cancer cells, compared to middle-aged and young patients. Aging is associated with important numerical and functional changes of the immune landscape in Lum-BC. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769ef243851690165ce0b362f0b78f63d7b88eb4" target='_blank'>
              Dissecting the immune infiltrate of primary luminal B-like breast carcinomas in relation to age.
              </a>
            </td>
          <td>
            S. Hatse, Y. Lambrechts, Asier Antoranz Martinez, M. De Schepper, T. Geukens, H. Vos, L. Berben, J. Messiaen, Lukas Marcelis, Yannick Van Herck, P. Neven, A. Smeets, C. Desmedt, F. De Smet, Francesca M Bosisio, H. Wildiers, G. Floris
          </td>
          <td>2024-09-29</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a030314e31860990960812246eddebd6ed05b0d" target='_blank'>
              Immunotherapy and pan-apoptotic characterization of the tumor microenvironment in gastric cancer (STAD): a single-cell multidimensional analysis
              </a>
            </td>
          <td>
            Sheng Zhang, Jianhong Wang, Huan Zhang, Benhua Li, Shun Gao
          </td>
          <td>2024-10-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The purpose of this study was to evaluate the suitability of paraffin-embedded (FFPE) samples and fixed fresh (FF) samples for single-cell RNA sequencing (scRNAseq). To this end, we compared whether single-cell profiles from FFPE and matched FF tissue samples from one invasive carcinoma of no especial type carcinoma (invasive ductal carcinoma –IDC) and one invasive lobular carcinoma (ILC) were similar when captured using the common immunohistochemical (IHC) and immune features of the tumors. FF and FFPE tissue derived libraries showed high-quality parameters. The cell heterogeneity obtained was similar, although, there were some differences in the number of cells captured, but there were no specific cell populations captured exclusively by one of the two different approaches. The combined analysis of the four samples identified six types of epithelial cells (normal basal cells, four subpopulations of neoplastic cells in the IDC and one in the ILC), and different tumoral microenvironment populations. scRNAseq results from epithelial neoplastic cells were concordant with common IHC markers. In addition, after quantitative assessment of the whole slide images using QuPath, the proportion of immune cells identified by IHQ in FFPE sections were similar to those obtained by scRNAseq. Confirming the suitability of scRNAseq using FFPE tissue, this technique identified one subpopulation of neoplastic cells present only in the IDC, characterized by the expression of genes related with multi-ciliated cell (MCC) differentiation (FOXJ1, ROPN1L, TPPP3, CFAP45). IHC analysis of FOXJ1 expression confirmed the presence of positive cells only in the IDC in a similar proportion to that obtained by scRNAseq. Analysis of FOXJ1 in a cohort of 214 ER-positive invasive carcinomas demonstrated protein expression in at least 1% of cells in 33% of tumors, suggesting frequent focal MCC differentiation. Our results support the suitability of scRNAseq analysis using FFPE tissue and identified a subpopulation of neoplastic MCC in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2bd0963b0249a16a0871656c5b25b2c0524782" target='_blank'>
              Single-cell RNA sequencing on formalin-fixed and paraffin-embedded (FFPE) tissue identified multi-ciliary cells in breast cancer
              </a>
            </td>
          <td>
            Silvia González-Martínez, Irene Carretero-Barrio, Belén Pérez-Miés, Val Fernández-Lanza, Mónica García-Cosío, T. Caniego-Casas, David Hardisson, I. Esteban-Rodríguez, J. Cortés, José Palacios
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Integrin family transmembrane receptors mediate dynamic interactions between cells and their extracellular microenvironment. The heterogeneous interaction partners of integrins directly regulate cell adhesion, motility, proliferation, and intracellular signaling. Despite the recognized importance of protein-protein interactions and the formation of signaling hubs around integrins, the ability to detect and quantify these dynamic binding partners with high spatial and temporal resolution remains challenging. Here, we developed an integrin-family-directed quantitative photoproximity protein interaction (PhotoPPI) profiling method to detect and quantify native integrin-centered protein social networks on live cells and tissues without the need for genetic manipulation, antibodies, or non-physiologic cell culture conditions. We drafted quantitative maps of integrin-centered protein social networks, highlighting conserved and unique binding partners between different cell types and cellular microenvironments. Comparison of integrin social networks in cancer cell lines of diverse tissue of origin and disease state identified specific AND-gate binding partners involved cell migration, microenvironmental interactions and proliferation that serve as markers of tumor cell metastatic state. Finally, we identified unique combinations – or barcodes - of integrin-proximal proteins on the surface of pre- and post-metastatic triple negative breast cancer (TNBC) cells whose expression strongly correlate with both positive and negative disease progression and outcomes in TNBC patients. Taken together, these data provide the first family-wide high-resolution maps of native protein interactors on live cells and identify dynamic integrin-centered social networks as potential AND-gate markers of cell identity, microenvironmental context and disease state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5900ee56d3da605dca7c4bf45b4b9da559c49721" target='_blank'>
              Family-Wide Photoproximity Profiling of Integrin Protein Social Networks in Cancer
              </a>
            </td>
          <td>
            Anthony J. Carlos, Dongbo Yang, Deborah M. Thomas, Shuyuan Huang, Keira I. Harter, R. Moellering
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b34bf47262137f174937f9ce2b2938693cc40a" target='_blank'>
              Genome-wide RNA-Seq identifies TP53-mediated embryonic stem cells inhibiting tumor invasion and metastasis
              </a>
            </td>
          <td>
            Yatong Li, Yongna Fan, Yunyi Xie, Limin Li, Juan Li, Jingyi Liu, Zhengyu Jin, Huadan Xue, Zhiwei Wang
          </td>
          <td>2024-10-16</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a prevalent malignant pediatric brain tumor with high mortality rates despite currentmultimodal therapies. MYC-driven Group 3 MB represents a highly aggressive subgroup with poor prognosis and limited treatment options. To systematically identify genes representing therapeutic vulnerabilities in MYC-driven MB, we performed a CRISPR-Cas9 screen with a specific focus on 1140 druggable genes across three MB cell lines. CDK8 emerged as a significantly selected gene crucial for MYC-driven MB growth, for which clinically relevant inhibitors are in early phase trials. CDK8 gene expression is high in group 3 MB (Colorado cohort) and specifically high in MYC driven MB subtypes and elevated expression of CDK8 is an adverse prognostic marker even in high-risk group 3 tumors. CDK8 is highly expressed in progenitor populations identified in our recent single-cell RNA-seq analysis and similarly expressed in cycling progenitors in murine models of MB. CDK8 depletion in MB cells using CRISPR-Cas9-induced knockout and lentivirus-mediated CDK8 shRNAs resulted in decreased i cell proliferation and MYC protein. Additionally, CDK8 depletion significantly decreased neurosphere growth in MB cells and supressed orthotopic xenograft growth in vivo. Evaluation of eight CDK8 selective inhibitors demonstrated a broad range of half-maximal inhibitory concentrations (IC50) across three G3-MB cell lines with RVU120 showing the maximum potency, RVu120 supressed colony growth in vitro in MB cells but not normal astrocytes. In vivo RVU120 potently supressed growth of orthotopic xenografts as evaluated by MRI and prolonged survival. RNA-seq analysis revelaed that CDK8 depletion leads to repression of mRNA translation and ribosome biogenesis. GSEA indicated that the loss of CDK8 resulted in strong negative enrichment in the transcriptomic profile associated with stemness gene sets and positive enrichment in differentiation gene sets. We performed a genome-wide analysis to map the occupancy of CDK8 and key histone markers using CUT&RUN in three G3-MB cell lines. We found that CDK8 functions as a transcriptional activator affecting the translation program. In vitro, inhibition of protein tranlation via mTOR supression was synergistic with CDK8 inhibition. We found that this synthetic lethality was relevant in vivo,where CDK8i and mTORi combined to supress xenograft growth and prolong survival. Taken together, these studies established the therapeutic efficacy of combination treatment with mTOR and CDK8 inhibitors in vivo and in vitro, opening an alternate path for biologically based therapeutic trials in MYC-driven MB.
 Citation Format: Dong Wang, Breunna Brunt, Bethany Veo, Milena Mazan, Tomasz Rzmyski, Sujatha Venkataraman, Nathan Dahl, Rajeev Vibhakar. Understanding the role of CDK8 in protein synthesis for treating MYC-driven medulloblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A082.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309ac06061039702b54427736670e453090c19d2" target='_blank'>
              Abstract A082 Understanding the role of CDK8 in protein synthesis for treating MYC-driven medulloblastoma
              </a>
            </td>
          <td>
            Dong Wang, Breunna Brunt, Bethany L. Veo, Milena Mazan, Tomasz Rzmyski, S. Venkataraman, N. Dahl, Rajeev Vibhakar
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Curcumin is widely recognized for its diverse antitumor properties, ranging from breast cancer to many other types of cancers. However, its role in the tumor microenvironment remains to be elucidated. In this study, we established a 3D tumor spheroids model that can simulate the growth environment of tumor cells and visualized the antitumor metabolic alteration caused by curcumin using mass spectrometry imaging technology. Our results showed that curcumin not only exerts a profound impact on the growth and proliferation of breast cancer cells but in situ multivariate statistical analysis also reveals the significant effect on the overall metabolic profile of tumor spheroids. Meanwhile, our visualization map characterized curcumin metabolic processes of reduction and glucuronidation in tumor spheroids. More importantly, abnormal metabolic pathways related to lipid metabolism and polyamine metabolism were also remodeled at the metabolite and gene levels after curcumin intervention. These insights deepen our comprehension of the regulatory mechanism of curcumin on the tumor metabolic network, furnishing powerful references for antitumor treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9708e466bdcc061f5dd608fcbfa98f940da2834e" target='_blank'>
              Spatial Metabolomics Profiling Reveals Curcumin Induces Metabolic Reprogramming in Three-Dimensional Tumor Spheroids
              </a>
            </td>
          <td>
            Zihan Zhu, Yaqi Zhang, Lei Wang, Haoyuan Geng, Min Li, Shiping Chen, Xiao Wang, Panpan Chen, Chenglong Sun, Chao Zhang
          </td>
          <td>2024-09-01</td>
          <td>Metabolites</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 For many years, glioblastoma (GBM) was thought to be restricted to the brain. Nevertheless, a growing body of evidence indicates that, like many other cancers, hematogenic dissemination is a reality. Detection of circulating tumor cells (CTCs) in the blood of GBM patients offers a non-invasive means to assess tumor-related information. Our aim is to use GBM CTCs to obtain knowledge about the tumor and improve the current GBM diagnosis and prognosis.



 Blood samples were collected from 18 primary tumor cases, 16 recurrences, 5 healthy controls, and 4 special cases. CTCs were enriched with Parsortix Cell Separation System and analysed on DEPArray system. The CTCs have been counted for all the samples. Copy Number Alterations (CNAs) analysis was performed on some of the isolated CTCs, examining a minimum of 1 to a maximum of 10 cells per sample. We performed also single-cell RNA sequencing (scRNAseq) analysis on a primary cancer cell line derived from a GBM primary tumor and we anlayzed CNAs in all the cell populations with Scevan tool.



 Cell counting demonstrated in both primary and recurrent GBM patients’ blood a statistically significantly higher number of CTCs compared to healthy controls (average of CTCs number 15.4 vs 1). Notably, primary GBM cases exhibited a greater number of CTCs compared to recurrent GBM cases, with averages of 18.2 and 12.4 CTCs, respectively. Based on these results provided by the cell count, we established a cut-off of 3 CTCs to identify a tumor sample. Moreover, among the four special cases examined, which included one lymphoma, one case of radionecrosis, and two specific tumors (an oligodendroglioma and a xanthoastrocytoma), our established cut-off was validated, affirming the accuracy of our CTCs detection method. Furthermore, CNAs analysis has revealed a completely novel finding observed thus far in GBM, specifically that approximately 51% of the CTCs isolated and analyzed harbored alterations, while the remainder 49% exhibited a wild-type profile at level of copy number alterations. This novel finding was corroborated by scRNAseq analysis of the GBM primary cancer cell line analyzed, wherein both wild-type cells and CNAs were observed with a percentage of CNAs of 51% and 49% respectively, mirroring the distibution observed in CTCs



 Our study demonstrates the utility of CTCs as a non-invasive diagnostic tool for GBM. Elevated CTCs counts, particularly in primary GBM cases, underscore their potential as biomarkers for disease progression. Additionally, our identification of copy number wild-type CTCs highlights the heterogeneity within CTC populations, which reflects the heterogeneous structure typical of GBM itself offering valuable insights into tumor biology and in understanding CTCs functions in the bloodstream.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff80c6c604aef64c944ec65a0ee5c23ebc6e79c9" target='_blank'>
              P08.02.B BEYOND THE EXPECTED: CIRCULATING TUMOR CELLS IN GLIOBLASTOMA REVEAL HIDDEN HETEROGENEITY AND BECOME POTENTIAL DIAGNOSTIC BIOMARKERS
              </a>
            </td>
          <td>
            F. Lessi, M. Morelli, A. Di Gangi, F. Di Lorenzo, S. Franceschi, P. Aretini, A. Pastore, G. Ferri, M. Menicagli, F. Pasqualetti, C. Gambacciani, F. Pieri, G. Grimod, A. D. Di Stefano, O. Santonocito, C. Mazzanti
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9918c8bdd0094035be3fb2b57ac7bbf09b9c90ee" target='_blank'>
              Comprehensive analysis of bulk, single-cell RNA sequencing, and spatial transcriptomics revealed IER3 for predicting malignant progression and immunotherapy efficacy in glioma
              </a>
            </td>
          <td>
            Qi Wang, Chunyu Zhang, Ying Pang, Meng Cheng, Rui Wang, Xu Chen, Tongjie Ji, Yuntong Yang, Jing Zhang, Chunlong Zhong
          </td>
          <td>2024-10-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The phenotypic plasticity of cancer cells has recently emerged as an important factor of treatment failure. The mechanisms of phenotypic plasticity are not fully understood. One of the hypotheses is that the degree of chromatin accessibility defines the easiness of cell transitions between different phenotypes. To test this, a method to compare overall chromatin accessibility between cells in a population or between cell populations is needed. We propose to measure chromatin accessibility by fluorescence signal from nuclei of cells stained with DNA binding fluorescent molecules. This method is based on the observations that small molecules bind nucleosome-free DNA more easily than nucleosomal DNA. Thus, nuclear fluorescence is proportional to the amount of nucleosome-free DNA, serving as a measure of chromatin accessibility. We optimized the method using several DNA intercalators and minor groove binders and known chromatin-modulating agents and demonstrated that chromatin accessibility is increased upon oncogene-induced transformation and further in tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf3429cc71eb3bb1823cf988315e0385b13e7d52" target='_blank'>
              Image-Based Quantitative Single-Cell Method Showed Increase of Global Chromatin Accessibility in Tumor Compared to Normal Cells
              </a>
            </td>
          <td>
            Mairead Commane, Vidula Jadhav, Katerina Leonova, Brian Buckley, Henry Withers, Katerina Gurova
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background/Objectives: Despite current treatments extending the lifespan of Glioblastoma (GBM) patients, the average survival time is around 15–18 months, underscoring the fatality of GBM. This study aims to investigate the impact of sample heterogeneity on gene expression in GBM, identify key metabolic pathways and gene modules, and explore potential therapeutic targets. Methods: In this study, we analysed GBM transcriptome data derived from The Cancer Genome Atlas (TCGA) using genome-scale metabolic models (GEMs) and co-expression networks. We examine transcriptome data incorporating tumour purity scores (TPSs), allowing us to assess the impact of sample heterogeneity on gene expression profiles. We analysed the metabolic profile of GBM by generating condition-specific GEMs based on the TPS group. Results: Our findings revealed that over 90% of genes showing brain and glioma specificity in RNA expression demonstrate a high positive correlation, underscoring their expression is dominated by glioma cells. Conversely, negatively correlated genes are strongly associated with immune responses, indicating a complex interaction between glioma and immune pathways and non-tumorigenic cell dominance on gene expression. TPS-based metabolic profile analysis was supported by reporter metabolite analysis, highlighting several metabolic pathways, including arachidonic acid, kynurenine and NAD pathway. Through co-expression network analysis, we identified modules that significantly overlap with TPS-correlated genes. Notably, SOX11 and GSX1 are upregulated in High TPS, show a high correlation with TPS, and emerged as promising therapeutic targets. Additionally, NCAM1 exhibits a high centrality score within the co-expression module, which shows a positive correlation with TPS. Moreover, LILRB4, an immune-related gene expressed in the brain, showed a negative correlation and upregulated in Low TPS, highlighting the importance of modulating immune responses in the GBM mechanism. Conclusions: Our study uncovers sample heterogeneity’s impact on gene expression and the molecular mechanisms driving GBM, and it identifies potential therapeutic targets for developing effective treatments for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86555fa3f48ea9588a98ba186b90071e4d847c59" target='_blank'>
              Unveiling the Molecular Mechanisms of Glioblastoma through an Integrated Network-Based Approach
              </a>
            </td>
          <td>
            Ali Kaynar, Woonghee Kim, Atakan Burak Ceyhan, Cheng Zhang, Mathias Uhlén, Hasan Turkez, S. Shoaie, A. Mardinoğlu
          </td>
          <td>2024-10-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="


 High-grade glioma (HGG) is the most common and aggressive form of primary brain cancer with poor prognosis. Standard of care surgery employs Magnetic Resonance Imaging to identify the contrast enhancing (CE) core for resection, typically leaving behind a non-enhancing (NE) invasive rim of tumor which invariably contributes to recurrence. Recent multi-omics analyses identified predominant neuronal and immune signatures within NE tumor, which could provide key insights for combatting recurrence. We present the novel application of graph network analyses to integrate multi-omic data from a unique dataset of multi-regional spatially-localized CE and NE biopsy samples from a large cohort of patients with HGG.



 159 IDH-wildtype tumor biopsy samples from NE and CE regions of 74 glioma patients were profiled by RNA sequencing. Differentially expressed genes (DEGs) across NE and CE samples (n = 1159) were determined using the EdgeR package, and represented as nodes within a Neo4j graph network. Samples were hierarchically clustered for each gene (method = median), and edges were generated between gene nodes sharing ≥ 1 sample. Modularity optimization was used to identify communities of densely connected nodes. Genes occurring exclusively within NE communities were localized by cell type using external single-cell RNA sequencing datasets and identified as source node candidates for NE network random walks. These were performed with CSF2RA nodes as source (walk length = 5; 10,000 walks/source node). Random walks traversing samples with significantly elevated CSF2RA expression (one-tailed test) were kept for analysis. Genes occurring at high frequency (one-tailed test) within random walks were ranked by correlation with CSF2RA expression in NE samples and analyzed by gene set enrichment analysis (GSEA) for biological processes using clusterProfiler.



 CSF2RA, an immune signaling receptor gene, was upregulated (logFC = 0.789) in NE samples and uniquely occurred within NE communities. We show its differential expression within the immune-enriched Glycolytic/Plurimetabolic (GPM) glioma subtype and confirm its enrichment in myeloid populations across single cell HGG datasets. NE network random walks identified 46 DEGs associated with CSF2RA. GSEA revealed upregulation of proliferative and metabolic signatures and downregulation of signaling pathways within CSF2RA’s network.



 CSF2RA’s genetic network in NE tumor represents a potential mechanism for immunologic facilitation of glioma progression within this region. Given the expression of CSF2 ligand by glioma cells, which plays a crucial role in recruiting myeloid cells during peripheral cancer progression, modulating the expression of key genes in the CSF2RA network may be a promising approach for perturbing the crosstalk signals between immune and tumor cells in the NE region.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/858c2e4adb493cdf971d2aaf4940d678ed0ad07a" target='_blank'>
              OS07.9.A NETWORK ANALYSIS IDENTIFIES CSF2RA’S GENETIC COMMUNITY AS A REGULATOR OF PROGRESSION IN THE INVASIVE REGION OF HIGH GRADE GLIOMA
              </a>
            </td>
          <td>
            V. Stevens, M. Flick, C. Sereduk, C. L. Plaisier, R. F. Barajas, L. Hu, K. Kannan, N. Tran
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="PURPOSE
Malignant peripheral nerve sheath tumors (MPNST) are aggressive soft tissue sarcomas and the leading cause of mortality in individuals with neurofibromatosis type 1 (NF1). Despite many clinical trials, outcomes for patients with MPNST have remained stagnant and most succumb to their disease; thus, novel therapeutic approaches are needed. A better understanding of the MPNST immune ecosystem will aid in the development of strategies to activate the immune system against the tumor. Herein, we profile the tumor immune microenvironment (TIME) in NF1-associated peripheral nerve sheath tumors (PNST) to discover insights on the role that tumor-infiltrating immune cells play in malignant transformation.


EXPERIMENTAL DESIGN
Utilizing fresh and formalin-fixed, paraffin-embedded tissue from patients diagnosed with NF1-PNST, we dissected the TIME by using immunohistochemistry, multiparameter flow cytometry, and comparative transcriptomic studies.


RESULTS
Immunophenotyping confirmed increased immune cells infiltration during malignant progression, with a predominance of infiltrating myeloid cells, particularly CD163+ tumor-associated macrophages (TAM). The T cells within MPNST exhibited signs of tumor activation, characterized by high PD-1 expression. Additionally, MPNST specimens demonstrated elevated levels of immunosuppressive TAM, with heightened PD-L1 expression. The proportion of CD163+ myeloid cells within the TIME correlated with poorer progression free survival. Notably, loss of H3K27 trimethylation correlated with low immune cell infiltration in MPNST.


CONCLUSIONS
Malignant transformation of NF1-PNST is characterized by an immunosuppressive microenvironment comprising of TAM with high expression of PD-L1, which are associated with inferior outcomes. These findings suggest a clinical potential of immune modulating therapeutics that can unleash an anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e45c2d277458a04b199f1d217e12500b372f16bc" target='_blank'>
              Multi-dimensional immunotyping of human NF1-associated peripheral nerve sheath tumors uncovers tumor-associated macrophages as key drivers of immune evasion in the tumor microenvironment.
              </a>
            </td>
          <td>
            Lindy Zhang, Alexandre Maalouf, Stavriani C Makri, Jineta Banerjee, Aditya Suru, Ada Tam, A. Calizo, K. Pollard, Jiawan Wang, Ludmila Danilova, Maria Ioannou, Adam S Levin, Carol D Morris, Daniel S Rhee, Allan J. Belzberg, J. Blakeley, Brian H Ladle, Drew M Pardoll, Calixto-Hope G Lucas, Fausto J Rodriguez, John M. Gross, Robert A. Anders, C. Pratilas, Nicolas J Llosa
          </td>
          <td>2024-09-25</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Granulomas are a key pathological feature of tuberculosis (TB), characterized by cell heterogeneity, spatial composition, and cellular interactions, which play crucial roles in granuloma progression and host prognosis. This study aims to analyze the transcriptome profiles of cell populations based on their spatial location and to understand the core transcriptome characteristics of granuloma formation and development. Methods In this study, we collected four clinical biopsy samples including Mycobacterium tuberculosis (Mtb) infected lung (MTB-L) and omentum tissues (MTB-O), as well as two lung and omentum biopsies from non-TB patients. The tissues were analyzed by spatial transcriptomics to create a spatial atlas. Utilizing cell enrichment scores and intercellular communication analysis, we investigated the transcriptome signatures of cell populations in various spatial regions and identified genes that may play a decisive role in the formation of pulmonary and omental tuberculosis granulomas. To validate our major findings, an in vitro TB model based on organoid-macrophage co-culture was established. Results Spatial transcriptomics mapped the cell composition and spatial distribution characteristics of tuberculosis granulomas in lung and omental tissues infected with Mtb. The characteristics and evolutionary relationships of major cell populations in granulomas reveal a shift in the immune microenvironment: from a predominance of B cells and fibroblasts in pulmonary granulomas to a predominance of myeloid cells and fibroblasts in omental granulomas. Furthermore, our data identified key differentially expressed genes across cell clusters and regions, showing that upregulation of collagen genes is a common feature of granulomas. Using an organoid-macrophage co-culture model, we demonstrated the notable efficacy of Thrombospondin-1 (THBS1) in reducing protein expression levels related to extracellular matrix remodeling. Conclusion These results provide insights into the pathogenesis and development of tuberculosis, enhancing our understanding of the composition and interactions of tuberculosis granuloma cells from a spatial perspective, and pave the way for novel adjuvant treatments for tuberculosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/466e12ea36c10b55f6829e91a348fd47392619ea" target='_blank'>
              Spatial transcriptomic sequencing reveals immune microenvironment features of Mycobacterium tuberculosis granulomas in lung and omentum
              </a>
            </td>
          <td>
            Xiaochen Qiu, Pengfei Zhong, Liang Yue, Chaofan Li, Zhimin Yun, Guangqian Si, Mengfan Li, Zhi Chen, Yingxia Tan, Pengtao Bao
          </td>
          <td>2024-09-23</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7199657c55f0905635cd26958d5762b121b0a3f2" target='_blank'>
              Integrated multi-omics assessment of lineage plasticity in a prostate cancer patient with brain and dural metastases
              </a>
            </td>
          <td>
            Megan L Ludwig, David Moline, Alec Horrmann, Ella Boytim, Gabrianne Larson, Ali T Arafa, Masooma Sayeda, John R Lozada, Hannah E Bergom, Abderrahman Day, Sandhyarani Dasaraju, S. Dehm, Paari J. Murugan, Justin H. Hwang, J. Drake, E. S. Antonarakis
          </td>
          <td>2024-09-30</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Understanding cancer mechanisms, defining subtypes, predicting prognosis and assessing therapy efficacy are crucial aspects of cancer research. Gene-expression signatures derived from bulk gene expression data have played a significant role in these endeavors over the past decade. However, recent advancements in high-resolution transcriptomic technologies, such as single-cell RNA sequencing and spatial transcriptomics, have revealed the complex cellular heterogeneity within tumors, necessitating the development of computational tools to characterize tumor mass heterogeneity accurately. Thus we implemented signifinder, a novel R Bioconductor package designed to streamline the collection and use of cancer transcriptional signatures across bulk, single-cell, and spatial transcriptomics data. Leveraging publicly available signatures curated by signifinder, users can assess a wide range of tumor characteristics, including hallmark processes, therapy responses, and tumor microenvironment peculiarities. Through three case studies, we demonstrate the utility of transcriptional signatures in bulk, single-cell, and spatial transcriptomic data analyses, providing insights into cell-resolution transcriptional signatures in oncology. Signifinder represents a significant advancement in cancer transcriptomic data analysis, offering a comprehensive framework for interpreting high-resolution data and addressing tumor complexity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e48fa67beea950c1a2e803890d2bcf508112396" target='_blank'>
              Exploring public cancer gene expression signatures across bulk, single-cell and spatial transcriptomics data with signifinder Bioconductor package
              </a>
            </td>
          <td>
            S. Pirrotta, L. Masatti, Anna Bortolato, Anna Corrà, Fabiola Pedrini, Martina Aere, Giovanni Esposito, Paolo Martini, Davide Risso, Chiara Romualdi, E. Calura
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Even though epigenetic factors contribute to oncogenesis, most human cancer models still assume that disease originates from driver DNA mutations. Thus, it is still unclear if non-genetic mechanisms are sufficient to trigger malignant transformation. Special AT-rich binding protein 2 (SATB2) is a chromatin organizer that brings distal DNA elements into close proximity, thus remodeling chromatin structures to reprogram cell-specific and/or developmentally-sensitive gene networks. Here, we discover that SATB2 generates a co-expressed circ3915 RNA that is translated into a peptide and co-locates with SATB2 in the cell nucleus. Ectopic SATB2 or circ3915 over- expression rearranges global chromatin accessibility, generates KRAS- and NFE2L2-like oncogenic gene expression patterns, and transforms lung epithelial cells independent of driver mutations. Thus, oncogenic pathways can be activated in mammalian cells without pre-disposing mutations in oncogenes or epigenetic regulators. Graphical abstract:">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2809e3fd65e4f1849899e9487a4c550596b35a99" target='_blank'>
              SATB2 and circ3915 RNA chromatin dysregulation drive KRAS-like oncogenic transformation
              </a>
            </td>
          <td>
            Rebekah Eleazer, Smitha George, Luke Shoemaker, Wesley N. Saintilnord, Kin H. Lau, Darrell Chandler, Yvonne Fondufe-Mittendorf
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mechanotransduction is a collection of pathways in which the cells reprogram themselves by sensing mechanical stimuli. Cells use biological cues to interpret the physiological stresses and respond to changing conditions by modifying the cellular and ECM architecture. This feedback loop regulates a variety of cellular processes, including migration, growth, differentiation, and death, which is essential for the network stability to work together in a coordinated manner. The effect of stress on cancer progression and the role of mechanics as a critical inducer in determining the cancer cell fate has been studied. This review discusses the progression of cancer cells to epithelial to mesenchymal transitions. It examines tumor microenvironment models, such as spheroids, bio-printing, and microfluidics, and how they recapitulate the tumor microenvironment. These offer certain benefits and help replicate the fundamental behavior in vivo conditions. We further discuss mechanosensing, the associated signaling molecules, and how it modulates the cancer drug resistance and transduction pathways that implicate cancer treatment. The difficulties with the existing methods and the prospects for additional study that may be applied in this area are discussed, and how they allow for new therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c5da4c6457c9e0269895f58a987813d2f60e71" target='_blank'>
              Mechanotransduction alterations in tissue-engineered tumor models for new drug interventions
              </a>
            </td>
          <td>
            Nipun Jain, Yusuf Olatunji Waidi, Souvik Debnath, V. Vannaladsaysy, Sudipto Datta
          </td>
          <td>2024-09-03</td>
          <td>Pharmacy &amp; Pharmacology International Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f8959f0cd1763d135183d030a6b223aefb572b8" target='_blank'>
              Therapeutic targeting of differentiation-state dependent metabolic vulnerabilities in diffuse midline glioma
              </a>
            </td>
          <td>
            Nneka E. Mbah, Amy L. Myers, Peter Sajjakulnukit, Chan Chung, Joyce K Thompson, H. Hong, Heather Giza, Derek Dang, Z. Nwosu, Mengrou Shan, Stefan R. Sweha, Daniella D Maydan, Brandon Chen, Li Zhang, Brian Magnuson, Zirui Zhu, Megan D Radyk, Brooke Lavoie, Viveka N Yadav, Imhoi Koo, Andrew D. Patterson, Daniel R. Wahl, Luigi Franchi, S. Agnihotri, C. Koschmann, S. Venneti, C. Lyssiotis
          </td>
          <td>2024-10-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/394c20d5ef3bbb71528bea2f3875ee00b262a370" target='_blank'>
              Glioblastoma cells increase expression of notch signaling and synaptic genes within infiltrated brain tissue.
              </a>
            </td>
          <td>
            D. Harwood, V. Pedersen, N. S. Bager, A. Schmidt, T. O. Stannius, A. Areškevičiūtė, Knud Josefsen, D. Nørøxe, David Scheie, Hannah E. Rostalski, Maya Jeje Schuang Lü, A. Locallo, Ulrik Lassen, F. O. Bagger, J. Weischenfeldt, D. H. Heiland, Kristoffer Vitting-Seerup, S. Michaelsen, BW Kristensen
          </td>
          <td>2024-09-09</td>
          <td>Nature communications</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="The expression of PD-L1 on tumor cells (TCs) is used as an immunotherapy biomarker in lung cancer, but heterogeneous intratumoral expression is often observed. Using a Digital Spatial Profiling, we performed proteomic and whole-transcriptomic analyses of TCs and immune cells (ICs) in spatially matched areas based on tumor PD-L1 expression and the status of the immune microenvironment. Our findings were validated using immunohistochemistry, The Cancer Genome Atlas, and immunotherapy cohorts. ICs in areas with high PD-L1 expression on TCs showed more features indicative of immunosuppression and exhaustion than ICs in areas with low PD-L1 expression on TCs. TCs highly expressing PD-L1 within immune-inflamed (IF) areas show up-regulation of pro-inflammatory processes, whereas TCs highly expressing PD-L1 within immune-deficient (ID) areas show up-regulation of various metabolic processes. Using differentially expressed genes of TCs between the IF and ID areas, we identified a novel prognostic gene signature for lung cancer. In addition, a high ratio of CD8+ cells to M2 macrophages was found to predict favorable outcomes in patients with PD-L1-expressing lung cancer after immune checkpoint inhibitor therapy. This study demonstrates that TCs and ICs have distinct spatial features within the tumor microenvironment that are related to tumor PD-L1 expression and IC infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc0a9207938d739ad9fb6635845dbbb9be7ec54" target='_blank'>
              Spatially resolved whole-transcriptomic and proteomic profiling of lung cancer and its immune-microenvironment according to PD-L1 expression.
              </a>
            </td>
          <td>
            Yoon Kyung Jeon, Jaemoon Koh, Dongjoo Lee, Sehui Kim, Seung Geun Song, B. Han, Hyein Jeong, Young A Kim, B. Keam, Se-Hoon Lee, K. Kim, Doo Hyun Chung
          </td>
          <td>2024-09-05</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="
 Patient-derived organoids (PDOs) have emerged as a powerful tool for modeling human cancers due to their ability to closely mirror in vivo architecture and preserve the genetic landscape of tumors. They offer a crucial platform for studying disease progression and treatment responses. In this study, we present a comprehensive characterization of a unique collection of PDO cultures derived from pancreatic ductal adenocarcinoma (PDAC), a particularly aggressive malignancy often diagnosed in advanced stages with limited treatment options and poor prognosis. Our investigation explores morphological and genetic profiling of 10 PDAC PDOs, specifically identifying distinct drug-response patterns. The morphological characteristics of these PDO cultures were investigated by microscopy, while the mutational status was acquired by panel sequencing. Subsequently, PDOs were treated with a panel of 10 inhibitors, either individually or in combination. The response to these treatments was assessed through live-cell imaging (Incucyte®) to capture and quantify changes in organoid morphology and endpoint measurements of cellular viability using a CellTiter-Glo® assay. Our detailed in vitro drug sensitivity testing revealed a remarkable diversity in the responses of the PDAC PDOs to different chemotherapeutic agents, including those currently under clinical trials. Of notable significance is the depth of understanding we have achieved, allowing us to establish potential correlations between PDO morphology, genetic mutations, and drug reactions. This finding provides a key insight into the complex interplay of these factors and their impact on treatment outcomes, offering valuable insights into understanding individual tumor subtypes' sensitivity or resistance to treatments. The PDOs exhibited a spectrum of cancer-associated mutations, including KRAS, TP53, SMAD4, CDKN2A, and ARID1A, reflecting the heterogeneity observed in PDAC patients. Morphologically, the organoids displayed various growth patterns, ranging from cystic and regular structures to filled or unfilled, granular, dark, and light membrane structures. Drug treatments underscored the diverse and unique drug-response profiles within the PDOs, with some demonstrating sensitivity to multiple treatments. In contrast, others exhibited resistance, highlighting the intricate nature of drug responses within this model. Overall, our findings emphasize the ability of PDAC organoids to faithfully capture the complexity and heterogeneity of the disease, making them promising platforms for guiding personalized oncology approaches. The diverse drug response profiles observed in PDOs advocate leveraging this model to develop precision medicine strategies tailored to patients' characteristics, ultimately paving the way for more effective and personalized treatment regimens in the fight against PDAC.
 Citation Format: Constanza Tapia Contreras, Günter Schneider, Denise Müller, Kimia Mirzakhani, Karly Conrads, Silke Kaulfuß, Tim Beißbarth, Gabriela Salinas, Nils Beyer, Sophia Reinländer, Ulrix Sax, Elisabeth Heßmann, Volker Ellenrieder, Philipp Ströbel, Michael Ghadimi. Patient-Derived Organoid Cultures for Personalized Therapies and Targeted Drug Screening Applications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A023.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a23581700d7eba1d7151f1cd1e85577199cc66e" target='_blank'>
              Abstract A023: Patient-Derived Organoid Cultures for Personalized Therapies and Targeted Drug Screening Applications
              </a>
            </td>
          <td>
            Constanza Tapia Contreras, Günter Schneider, Denise Müller, Kimia Mirzakhani, Karly Conrads, S. Kaulfuß, Tim Beißbarth, Gabriela Salinas, Nils Beyer, Sophia Reinländer, Ulrix Sax, E. Hessmann, V. Ellenrieder, Philipp Ströbel, Michael Ghadimi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Non-Hispanic Black women (BW) are diagnosed with more aggressive breast cancer subtypes at younger ages and have higher mortality rates compared to non-Hispanic White women (WW). Within the breast microenvironment emergent data shows that stromal collagen plays a significant role in aggressiveness of breast cancer linked to ancestral traits. However, the molecular composition of collagen stroma has yet to be linked to ancestry and aggressiveness of breast cancer. The purpose of this study was to evaluate the microenvironment within ancestry-defined normal breast and triple-negative breast cancer (TNBC) to understand spatial regulation of collagen stroma that may link to ancestral differences in cancer. Multimodal, multiplexed spatial analyses were performed. Techniques included Second Harmonic Generation (SHG) to measure collagen fibers, mass spectrometry imaging to measure the collagen proteome, and mass-tagged antibody panels to investigate immune signatures in TNBC. Normal breast tissues were obtained from the Susan G. Komen Tissue Bank, African American (AA; n=20) and European American (EA; n=20) samples were annotated with risk scores, age, Body Mass Index (BMI), Breast Imaging-Reporting and Data System (BI-RADS), continental ancestry, and family history. Triple-negative breast cancer (TNBC, n=80) tissues from BW with extensive clinical annotations including age, BMI, stage, metastasis, and survival were evaluated in comparison to 20+ TNBC tissues from WW. In the normal breast cohort, evaluation of BMI (overweight and obese) within ancestry groups showed significant but opposing differences in fiber width measurements. As BMI increases, AA showed statistically thicker fibers while EA showed statistically thinner fibers. At the protein level, two types of collagens were significantly higher in EA and one type is higher in AA. Within the TNBC cohort, collagen fiber length and width measurements by SHG were significantly shorter and thinner in stage II compared to stage I but no other stage comparisons were significant. This may indicate that ECM remodeling occurs at earlier stages in TNBC. Initial analyses showed that collagen peptide intensities significantly vary by TNBC stage (75 peptides, ANOVA test FDR p-value < 0.05). In the cohort of women who survived, 90 ECM peptides significantly correlated with collagen fiber widths, indicating a relationship between collagen physical and molecular properties within a defined region. Seven immune markers defined differences in survival when grouped by high vs low expression within tumor regions. These markers include CD20 (Hazard Ratio HR = 7.99, p-value = 0.002) and PD-1 (HR = 3.68, p-value = 0.048), which have both been studied in breast cancer. Current work focuses on collagen peptide post-translational modifications that may differ in TNBC patients by vital status and are detected in normal breast at risk of cancer. These findings support that molecular composition in breast stroma contributes to aggressive ancestral traits of breast cancer risk and triple-negative breast cancer.
 Citation Format: Jaclyn B. Dunne, Heather Jensen-Smith, Laura Spruill, Taylor S. Hulahan, Jade K. Macdonald, George E. Sandusky, M. A. Hollingsworth, Anand S. Mehta, Richard R. Drake, Jeffrey R. Marks, Harikrishna Nakshatri, Marvella E. Ford, Peggi M. Angel. Multimodal and multiplexed spatial regulation of the breast collagen proteome: Comparative analysis in Black and White women from normal breast tissue and triple-negative breast cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d058388e9d844a6c66ca907f1790256ff73b1f8" target='_blank'>
              Abstract C003: Multimodal and multiplexed spatial regulation of the breast collagen proteome: Comparative analysis in Black and White women from normal breast tissue and triple-negative breast cancer
              </a>
            </td>
          <td>
            Jaclyn B. Dunne, Heather Jensen-Smith, Laura S Spruill, Taylor S. Hulahan, Jade K Macdonald, George E Sandusky, M. A. Hollingsworth, A. Mehta, Richard R. Drake, Jeffrey R. Marks, Hari Nakshatri, Marvella E. Ford, Peggi M. Angel
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Signaling pathways induce stereotyped transcriptional changes as stem cells progress into mature cell types during embryogenesis. Signaling perturbations are necessary to discover which genes are responsive or insensitive to pathway activity. However, gene regulation is additionally dependent on cell state-specific factors like chromatin modifications or transcription factor binding. Thus, transcriptional profiles need to be assayed in single cells to identify potentially multiple, distinct perturbation responses among heterogeneous cell states in an embryo. In perturbation studies, comparing heterogeneous transcriptional states among experimental conditions often requires samples to be collected over multiple independent experiments. Datasets produced in such complex experimental designs can be confounded by batch effects. We present Design-Aware Integration of Single Cell ExpEriments (DAISEE), a new algorithm that models perturbation responses in single-cell datasets with a complex experimental design. We demonstrate that DAISEE improves upon a previously available integrative non-negative matrix factorization framework, more efficiently separating perturbation responses from confounding variation. We use DAISEE to integrate newly collected single-cell RNA-sequencing datasets from 5-hour old zebrafish embryos expressing optimized photoswitchable MEK (psMEK), which globally activates the extracellular signal-regulated kinase (ERK), a signaling molecule involved in many cell specification events. psMEK drives some cells that are normally not exposed to ERK signals towards other wild type states and induces novel states expressing a mixture of transcriptional programs, including precociously activated endothelial genes. ERK signaling is therefore capable of introducing profoundly new gene expression states in developing embryos. Significance Statement Signaling perturbations produce heterogeneous transcriptional responses that must be measured at the single-cell level. Data integration techniques allow us to model these responses which, however, can be confounded by batch effects. We present a computational tool (DAISEE) for extracting the common and perturbation-specific features of single-cell datasets representing multiple experimental conditions while achieving efficient batch effect correction. DAISEE outperforms its predecessor and will enable accurate analysis of a broad range of single-cell datasets. DAISEE applied to new single-cell RNA sequencing data from zebrafish embryos shows that gain-of-function signaling perturbations can induce novel states. Our analysis suggests that a wild type endothelial cell-specification program can be activated in abnormal developmental contexts when the extracellular signal-regulated kinase (ERK) pathway is deregulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9843baa0dda4952c851210ee811a954b61353c56" target='_blank'>
              Disrupted developmental signaling induces novel transcriptional states
              </a>
            </td>
          <td>
            Aleena Patel, Vanessa Gonzalez, Triveni Menon, Stanislav Y. Shvartsman, R. Burdine, M. Avdeeva
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a40aec86f303beb0c2a44fc542c17c17169d4e43" target='_blank'>
              Prognosis prediction via histological evaluation of cellular heterogeneity in glioblastoma
              </a>
            </td>
          <td>
            Mari Kirishima, Seiya Yokoyama, T. Akahane, Nayuta Higa, H. Uchida, Hajime Yonezawa, Kei Matsuo, Junkoh Yamamoto, Koji Yoshimoto, Ryosuke Hanaya, Akihide Tanimoto
          </td>
          <td>2024-10-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f411da274f61a050cd3f7ed458f7796f8e2b06" target='_blank'>
              Exosomal miR-106a-5p from highly metastatic colorectal cancer cells drives liver metastasis by inducing macrophage M2 polarization in the tumor microenvironment
              </a>
            </td>
          <td>
            Yahang Liang, Junyu Li, Yu-Wen Yuan, Houqiong Ju, H. Liao, Mingming Li, Yang Liu, Yao Yao, Lingling Yang, Taiyuan Li, Xiong Lei
          </td>
          <td>2024-10-09</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Transcription factor deregulation potently drives melanoma progression by dynamically and reversibly controlling gene expression programs. We previously identified the small MAF family transcription factor MAFG as a putative driver of melanoma progression, prompting an in-depth evaluation of its role in melanoma. MAFG expression increases with human melanoma stages and ectopic MAFG expression enhances the malignant behavior of human melanoma cells in vitro, xenograft models, and genetic mouse models of spontaneous melanoma. Moreover, MAFG induces a melanoma phenotype switch from a melanocytic state to a more dedifferentiated state. Mechanistically, MAFG interacts with the lineage transcription factor MITF which is required for the pro-tumorigenic effects of MAFG. MAFG and MITF co-occupy numerous genomic sites and MAFG overexpression influences the expression of genes harboring binding sites for the MAFG∼MITF complex. These results establish MAFG as a potent driver of melanomagenesis through dimerization with MITF and uncover an unappreciated mechanism of MITF regulation. Significance statement MITF is critically involved in melanoma progression and phenotype switching. We discovered that MAFG interacts with MITF to influence expression of MITF target genes and facilitate a shift toward a dedifferentiated melanoma cell state. This study demonstrates that MAFG promotes melanomagenesis by influencing MITF activity, an unappreciated mechanism of MITF regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b9b3d9b3d18ae635dff9248bc5db6f5ca715aae" target='_blank'>
              The small MAF transcription factor MAFG co-opts MITF to promote melanoma progression
              </a>
            </td>
          <td>
            Olga Vera, Michael Martinez, Zulaida Soto-Vargas, Kaizhen Wang, Xiaonan Xu, Sara Ruiz-Buceta, Nicol Mecozzi, Manon Chadourne, Benjamin Posorske, Ariana M. Angarita, Ilah Bok, Qian Liu, Harini Murikipudi, Yumi Kim, Jane L. Messina, Kenneth Y. Tsai, Michael B. Major, Eric K. Lau, Xiaoqing Yu, Inmaculada Ibanez-de-Caceres, F. Karreth
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="MDM4 is upregulated in the majority of melanoma cases and has been described as a “key therapeutic target in cutaneous melanoma”. Numerous isoforms of MDM4 exist, with few studies examining their specific expression in human tissues. The changes in splicing of MDM4 during human melanomagenesis are critical to p53 activity and represent potential therapeutic targets. Compounding this, studies relying on short reads lose “connectivity” data, so full transcripts are frequently only inferred from the presence of splice junction reads. To address this problem, long-read nanopore sequencing was utilized to read the entire length of transcripts. Here, MDM4 transcripts, both alternative and canonical, are characterized in a pilot cohort of human melanoma specimens. RT-PCR was first used to identify the presence of novel splice junctions in these specimens. RT-qPCR then quantified the expression of major MDM4 isoforms observed during sequencing. The current study both identifies and quantifies MDM4 isoforms present in melanoma tumor samples. In the current study, we observed high expression levels of MDM4-S, MDM4-FL, MDM4-A, and the previously undescribed Ensembl transcript MDM4-209. A novel transcript lacking both exons 6 and 9 is observed and named MDM4-A/S for its resemblance to both MDM4-A and MDM4-S isoforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb9d00c41bc5b70987f8e1b5ab3d2577df46d0b8" target='_blank'>
              Long-Read MDM4 Sequencing Reveals Aberrant Isoform Landscape in Metastatic Melanomas
              </a>
            </td>
          <td>
            Nehaal Patrick, Michael Markey
          </td>
          <td>2024-08-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b49713db7ad3aa08fe609faeb0a2e80697cb2aa" target='_blank'>
              Integrated analyses reveal two molecularly and clinically distinct subtypes of H3 K27M-mutant diffuse midline gliomas with prognostic significance
              </a>
            </td>
          <td>
            L. Stegat, Alicia Eckhardt, Antonia Gocke, S. Neyazi, Lara C Pohl, S. Schmid, Matthias Dottermusch, S. Frank, H. Pinnschmidt, Jochen Herms, M. Glatzel, M. Snuderl, L. Schweizer, C. Thomas, J. Neumann, Mario M. Dorostkar, U. Schüller, A. Wefers
          </td>
          <td>2024-09-10</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Simple Summary Currently, there are no effective therapies for glioblastoma. Infiltrating myeloid cells contributes significantly to the immune suppressive tumor microenvironment that is characteristic of GBM. Monocytic myeloid-derived suppressor cells are chief immune suppressive cells found in the glioma microenvironment. Understanding the mechanisms of M-MDSC differentiation and T-cell suppression is imperative for generating therapies that target this tumor-supportive cell population. In this study, we found that glioma-secreted CSF1R ligands, M-CSF and IL-34, promote M-MDSCs to suppress CD8 T cells. These M-MDSCs partially utilize nitric oxide synthase to illicit their suppressive activity. However, spatial RNAseq points to glioma microenvironment niches driving M-MDSC heterogeneity. Our findings identify key regulators of differentiation and suppressive mechanisms of M-MDSCs and confirm the importance of targeting this cell population in glioma. Abstract Glioblastoma (GBM) is the most common malignant primary brain tumor, resulting in poor survival despite aggressive therapies. GBM is characterized by a highly heterogeneous and immunosuppressive tumor microenvironment (TME) made up predominantly of infiltrating peripheral immune cells. One significant immune cell type that contributes to glioma immune evasion is a population of immunosuppressive cells, termed myeloid-derived suppressor cells (MDSCs). Previous studies suggest that a subset of myeloid cells, expressing monocytic (M)-MDSC markers and dual expression of chemokine receptors CCR2 and CX3CR1, utilize CCR2 to infiltrate the TME. This study evaluated the mechanism of CCR2+/CX3CR1+ M-MDSC differentiation and T cell suppressive function in murine glioma models. We determined that bone marrow-derived CCR2+/CX3CR1+ cells adopt an immune suppressive cell phenotype when cultured with glioma-derived factors. Glioma-secreted CSF1R ligands M-CSF and IL-34 were identified as key drivers of M-MDSC differentiation while adenosine and iNOS pathways were implicated in the M-MDSC suppression of T cells. Mining a human GBM spatial RNAseq database revealed a variety of different pathways that M-MDSCs utilize to exert their suppressive function that is driven by complex niches within the microenvironment. These data provide a more comprehensive understanding of the mechanism of M-MDSCs in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e96f076b3e4cde53f5f74be1b3ae929a72dd0c8" target='_blank'>
              CSF1R Ligands Expressed by Murine Gliomas Promote M-MDSCs to Suppress CD8+ T Cells in a NOS-Dependent Manner
              </a>
            </td>
          <td>
            Gregory P. Takacs, Julia S. Garcia, Caitlyn A. Hodges, C. Kreiger, Alexandra Sherman, Jeffrey K. Harrison
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Like many pediatric cancers, Ewing sarcoma (EwS) is a genomically quiet disease. Driven by an oncogenic fusion protein, usually EWSR1-FLI1, the epigenome of this genetically homogenous cancer is a driver of clinical outcomes, where heightened epigenetic heterogeneity within tumors is associated with metastatic disease. While prognosis for localized disease is moderate, outcomes are dismal for patients who present with advanced, relapsed, or refractory disease and there is a lack of targeted therapies in this setting. We have previously described a targeted combination therapy harnessing DDK and WEE1 inhibition exploits elevated endogenous replication stress in EwS to force cells into mitotic catastrophe in vitro. While our experiments evaluating the efficacy of this combination in vivo have been extremely encouraging, the data have shown that individual mice receiving the same treatment have variable patterns of outgrowth after treatment cessation, with some mice demonstrating moderate increases in tumor volume and some mice demonstrating little to no tumor outgrowth. This differential response to therapy is most likely to occur through epigenetic dysregulation. Thus, we aim to interrogate intertumoral epigenetic heterogeneity and examine epigenetic profiles associated with treatment response using samples from in vivo studies. Samples were analyzed from a CDX model using TC32 cells injected subcutaneously. Reduced representation bisulfite sequencing was performed on a control arm (n = 5) and the following three treatment arms, where the DDK inhibitor is TAK-931 (simurosertib) and the WEE1 inhibitor is MK1775 (adavosertib): 1) Irinotecan + Temozolomide (n=9), 2) Irinotecan (full dose) + DDKi + WEE1i (n=7), 3) Irinotecan (half dose) + DDKi + WEE1i (n=9), for a total of 30 samples. Our analysis of this data evaluates epigenetic changes between treatment conditions and increases our understanding of epigenetic heterogeneity in EwS.
 Citation Format: Emily Isenhart, Ajay Gupta, Joyce Ohm. Intertumoral epigenetic heterogeneity in response to a novel therapy in Ewing sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A056.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366bd5a5e10d37a86e17654dd6f774d1f8bf9952" target='_blank'>
              Abstract A056 Intertumoral epigenetic heterogeneity in response to a novel therapy in Ewing sarcoma
              </a>
            </td>
          <td>
            E. Isenhart, Ajay Gupta, Joyce Ohm
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0fe672a884b6c86c84e083c9b2ab93ac24f65f" target='_blank'>
              Multiplex, single-cell CRISPRa screening for cell type specific regulatory elements
              </a>
            </td>
          <td>
            Florence M. Chardon, Troy A. McDiarmid, Nicholas F. Page, R. Daza, Beth K. Martin, Silvia Domcke, Samuel Regalado, Jean-Benoît Lalanne, Diego Calderon, Xiaoyi Li, L. Starita, Stephan J. Sanders, N. Ahituv, J. Shendure
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>49</td>
        </tr>

        <tr id="
 Malignant rhabdoid tumors (MRT) are rare aggressive tumors of infancy characterized by the biallelic inactivation of SMARCB1. Their lineage of origin remains uncertain, but increasing evidence suggest a neural-crest origin. By analyzing the transcriptome of MRT occurring in peripheral and cranial nerves, we observe that their expression profiling was not distinct from any other tumor localization. We therefore investigated whether the development of Smarcb1-deficient tumors emerging from embryonic precursors of peripheral nerves in mice could recapitulate human MRT. we generated conditional Smarcb1 knockout in the Prss56-Cre mouse strain thereby targeting boundary cap cells which are neural crest derived Schwann cell precursor. All Smarcb1Flox/Flox;Prss56Cre+/+ showed ataxia within a median of two months, due to the development of massive intracranial tumor in the ponto-cerebellum angle. Smarcb1Flox/Flox;Prss56Cre+/- mainly developed paraplegia related to nerve root tumors, within a median delay of 7 months. Interestingly, transcriptome profiling confirmed the good correlation between mouse peripheral nerve tumors and human MRT; intracranial tumors in this new model correlated with MYC-ATRTs but with some specific features not observed in their human counterparts. All tumors harbored a significant immune infiltration, with some clonal T-cell expansion. Altogether, the Smarcb1Flox/Flox;Prss56-Cre mouse tumors offer a new model for human MRT and promising perspectives for immune-based innovative treatments.
 Citation Format: Zhi-Yan HAN, Mamy Andrianteranagna, Stéphanie Fitte-Duval, Valeria Manriquez, Magali Frah, Rachida Bouarich, Sandrina Turcynski, Arnault Tauziède-Espariat, Kevin Beccaria, Julien Masliah-Planchon, Christine Bourneix, Delphine Guillemot, Stéphanie Reynaud, Gaelle Pierron, Maëva Veyssiere, Owen Hoare, Céline Chauvin, Grégory Thomson, Didier Surdez, Pascale Varlet, Christelle Dufour, Volodia Dangouloff-Ros, Joshua J. Waterfall, Eliane Piaggio, Olivier Delattre, Piotr Topilko, Franck Bourdeaut. Targeting boundary cap cells leads to mouse Smarcb1-deficient peripheral nerve tumors recapitulating human peripheral rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624aa14765b3c481a1fc38ac19c8ad7677137eda" target='_blank'>
              Abstract A006 Targeting boundary cap cells leads to mouse Smarcb1-deficient peripheral nerve tumors recapitulating human peripheral rhabdoid tumors
              </a>
            </td>
          <td>
            Zhi-Yan Han, Mamy Andrianteranagna, Stéphanie Fitte-Duval, Valeria Manriquez, Magali Frah, Rachida Bouarich, Sandrina Turcynski, A. Tauziède-Espariat, K. Beccaria, J. Masliah-Planchon, C. Bourneix, D. Guillemot, S. Reynaud, G. Pierron, Maëva Veyssiere, Owen Hoare, C. Chauvin, Grégory Thomson, D. Surdez, Pascale Varlet, Christelle Dufour, V. Dangouloff-Ros, J. Waterfall, Eliane Piaggio, Olivier Delattre, Piotr Topilko, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Glioblastoma (GBM) has a poor prognosis with a high recurrence and low survival rate. Previous RNA-seq analyses have revealed that alternative splicing (AS) plays a role in GBM progression. Here, we present a novel AS analysis method (Semi-Q) and describe its use to identify GBM-specific AS events. We analyzed RNA-seq data from normal brain (NB), normal human astrocytes (NHAs) and GBM samples, and found that comparison between NHA and GBM was especially informative. Importantly, this analysis revealed that genes encoding cell migration-related proteins, including filamins (FLNs) and actinins (ACTNs), were among those most affected by differential AS. Functional assays revealed that dysregulated AS of FLNA, B and C transcripts produced protein isoforms that not only altered transcription of cell proliferation-related genes but also led to enhanced cell migration, resistance to cell death and/or mitochondrial respiratory function, while a dysregulated AS isoform of ACTN4 enhanced cell migration. Together, our results indicate that cell migration and actin cytoskeleton-related genes are differentially regulated by AS in GBM, supporting a role for AS in facilitating tumor growth and invasiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/093613fea651e9c7c835c1557c39bae9a76fdea0" target='_blank'>
              Splicing dysregulation in glioblastoma alters the function of cell migration-related genes
              </a>
            </td>
          <td>
            Minu Seong, Pedro Bak-Gordon, Zhaoqi Liu, Peter D Canoll, James L. Manley
          </td>
          <td>2024-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e1b808c0ad8ebc0432fdf9718775113703bf08" target='_blank'>
              Multi-regional genomic and transcriptomic characterization of a melanoma-associated oral cavity cancer provide evidence for CASP8 alteration-mediated field cancerization
              </a>
            </td>
          <td>
            Shouvik Chakravarty, Arnab Ghosh, Chitrarpita Das, Subrata Das, S. Patra, Arindam Maitra, S. Ghose, N. Biswas
          </td>
          <td>2024-09-07</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Background Osteosarcoma (OS) survival rates and outcome have not improved in 50 years since the advent of modern chemotherapeutics. Thus, there is a critical need for an improved understanding of the tumor microenvironment to identify better therapies. Extracellular matrix (ECM) deposition and hypoxia are known to abrogate the efficacy of various chemical and cell‐based therapeutics. Here, we aim to mechanistically investigate the combinatorial effects of hypoxia and matrix deposition with the use of OS spheroids. Methods We use two murine OS cell lines with differential metastatic potential to form spheroids. We form spheroids of two sizes, use ascorbate‐2‐phosphate supplementation to enhance ECM deposition, and study cell response under standard (21% O2) and physiologic (5% O2) oxygen tensions. Finally, we examine chemotherapeutic responses to doxorubicin treatment. Results ECM production and oxygen tension are key determinants of spheroid size through cell organization based on nutrient and oxygen distribution. Interestingly, highly metastatic OS is more susceptible to chemotherapeutics compared to less metastatic OS when matrix production increases. Together, these data suggest that dynamic interactions between ECM production and oxygen diffusion may result in distinct chemotherapeutic responses despite inherent tumor aggressiveness. Conclusion This work establishes OS spheroids as a valuable tool for early OS tumor formation investigation and holds potential for novel therapeutic target and prognostic indicator discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e5e2fb20e2a4e399455c45af17440bdc5ed9071" target='_blank'>
              Extracellular matrix production and oxygen diffusion regulate chemotherapeutic response in osteosarcoma spheroids
              </a>
            </td>
          <td>
            Isabel S Sagheb, Thomas P Coonan, R. L. Randall, Katherine H Griffin, J. K. Leach
          </td>
          <td>2024-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 The localization of gliomas in the brain is a critical prognostic factor, reﬂecting the genetic makeup of their originating cells. Variations in transcriptomic proﬁles across different regions and subtypes may further shed light on the mechanisms driving tumor development. Our research delves into the complex interplay between glioma locations and their transcriptomic signatures, seeking to dissect the spatial and genetic intricacies contributing to glioma diversity.



 In this study, we conducted a comprehensive analysis of preoperative anatomical MRI and transcriptomic data from 242 glioma patients to investigate the relationship between spatial distribution and tumor transcriptomic proﬁles. The MRI data were standardized by registration to the MNI-152 brain space. Tumors were segmented, and the brain was partitioned into 84 discrete subregions for detailed examination. Transcriptomic data underwent preprocessing, with principal component analysis employed to reduce dimensionality. Statistical analyses were used to assess the relationship between glioma description across different subtypes and PCA results. Additionally, we identify differentially expressed genes and common genes across regions for various glioma subtypes. Comparative analyses with the Allen Brain Atlas (divided into 84 subregions with expression data) were also performed to discern transcriptional differences between healthy individuals and glioma patients, further enhancing our understanding of glioma biology at the molecular level.



 We generated 21 frequency maps detailing regional glioma occurrences, categorized by biomarkers, subtypes, and grades. Notably, some of these maps showed signiﬁcant correlations with expression principal component analyses (PCAs). Our ﬁndings include a set of genes consistently expressed across subregions, alongside region speciﬁc differentially expressed genes. Additionally, we identiﬁed distinct gene expression patterns when comparing glioma patients to healthy controls in the Allen Brain Atlas, highlighting potential molecular underpinnings of glioma heterogeneity.



 Our analysis revealed notable correlations between glioma subtype locations and transcriptomic proﬁles, underscoring the role of transcriptomic signatures in explaining regional variations in glioma incidence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9710a81388df04aff0d2613923e0becf7f6f7df7" target='_blank'>
              INTEGRATING TRANSCRIPTOMICS WITH SPATIAL RADIOGRAPHIC ATLASES FOR GLIOMA ANALYSIS
              </a>
            </td>
          <td>
            Mr Yonghao Li, Mr Yizhou Wan, Prof Stephen Price
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Introduction Cancer-associated fibroblasts (CAFs) are key players in the fibrotic tumour microenvironment (TME) of pancreatic cancer. Recent evidence has demonstrated the existence of multiple CAF subtypes, however there is relatively little evidence validating and corroborating these CAF subtypes in resected human pancreatic cancer. Furthermore, the spatial interactions between CAF subtypes and the epithelial and immune compartments of the pancreatic cancer TME is not well understood. Methods We designed and optimised a bespoke 8-plex immunofluorescence panel for the purposes of CAF subtype profiling and elucidation of CAF spatial biology: panCK, FAP, SMA, CTGF, S100A4, IL6, LIF, PDGFR. This was applied to a cohort of n=215 resected pancreatic cancer, well characterised with highly granular clinico-pathological and transcriptomic data through the Australian Pancreatic Cancer Genome Initiative. Spatial analysis was performed with novel computational biology techniques. Results We demonstrate seven CAF subtypes which cluster to form distinct, highly prognostic “enrichment profiles” (EPs). The cold EP represents quiescent CAFs in close proximity to immune cells (multivariate HR 0.45, 95% CI 0.30-0.68, p<0.001), and the fibrotic EP represents activated CAFs in close proximity to epithelial cells (multivariate HR 2.22, 95% CI 1.48-3.35, p<0.001). We also demonstrate the prognostic power of high spatial entropy (loss of tissue architecture, correlating with the fibrotic EP) vs low spatial entropy (preservation of tissue architecture, correlating with the cold EP) (HR 2.22, 95% CI 1.47-3.35, p<0.001). Conclusion We begin to unravel the elusive spatial biology of pancreatic CAFs in a large resected human cohort. We see changes in immune cell infiltrate, modulation of cellular relationships, and greater tissue disorganisation as a result of CAF activation. These features are highly prognostic and we hypothesise they are driven by changes in crosstalk between TME compartments. Current work focuses on application of these techniques to a large resected neoadjuvantly-treated cohort, allowing further interrogation of the notorious fibrotic post-neoadjuvant pancreatic cancer TME. If accepted for presentation we look forward to presenting both cohorts simultaneously.
 Citation Format: Adam Bryce, Stephan Dreyer, Fieke Froeling, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, John Le Quesne, David Chang. Interrogating the spatial biology of pancreatic cancer-associated fibroblasts in resected human pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A053.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179192d42336b982da176e9aa5fe870344303cba" target='_blank'>
              Abstract A053: Interrogating the spatial biology of pancreatic cancer-associated fibroblasts in resected human pancreatic cancer
              </a>
            </td>
          <td>
            A. Bryce, S. Dreyer, F. Froeling, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, John Le Quesne, David Chang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Background: One of the most significant unmet clinical challenges in paediatric oncology is the development of novel therapeutic strategies for recurrent medulloblastoma (R-MB). MYC-driven MBs are defined as classically cold tumours with a low incidence of infiltrating immune cells, resulting in a therapeutic challenge. The immune landscape of recurrent medulloblastoma is yet to explored. Understanding the complex interplay between the immune system and tumour biology is pivotal for comprehending disease progression and devising tailored therapeutic strategies that consider the distinct molecular and immunological profiles of both primary and recurrent MYC-driven tumours. The identification of cell-cell communications, particularly ligand–receptor pairs, allows for the inference of significant intercellular communications based on the expression of corresponding genes. Consequently, we hypothesised that the most influential cell-cell interactions within the tumour immune microenvironment (TME) of MYC-driven MB could unveil prevalent immune-suppressive interactions and potential vulnerabilities for therapeutic exploration. Methods/Results: Paired primary-recurrent bulk RNA-sequencing data, confirmed myeloid cells as the most infiltrating immune cell type in group3-MB and group4-MB. Comprehensive spatial phenotypic and cell-cell communication analyses corroborated this discovery, validating an increased incidence of macrophages in the matched-recurrent tumours and phenotypic markers of advanced immunosuppression. Subsequently, we used innovative algorithms for 10X MB single-cell data to predict interactions between tumour-cell ligands and immune-cell receptors within the TME; macrophages emerged as the core immune-cells involved in interactions throughout the TMEs, with the most significant ligand-receptor interaction and inflammatory response between MIF and CD74.In-depth immunohistochemistry analyses of primary and recurrent group3 and group4 tumours, and exhaustive tissue microarrays demonstrated expression of both CD74 and MIF, with limited expression of CD74 within the brain. To investigate the therapeutic potential of CD74, we developed recurrent, immune competent MYC-driven medulloblastoma mouse models. Comprehensive deconvolution analysis confirmed the TME integrity of our models to mirror that of the human disease. Locoregional delivery and repeat dosing of a bioactive-CD74 peptide demonstrated significant tumour reduction in our immune-competent mouse models, demonstrating the impact of our prediction algorithm and significant therapeutic potential of targeting the CD74-MIF axis in MYC-driven primary and recurrent MB. Conclusions: Essential cellular interactions and targetable therapeutic vulnerabilities have been identified in the tumour-microenvironment of MYC-driven primary-recurrent MB.
 Citation Format: Ben Draper, Dean Thompson, Alaide Morcavallo, Bethany Remeniuk, John Anderson, Louis Chesler, Steve Clifford, Frank Huang, Laura K. Donovan. Unveiling the immune landscape of recurrent, MYC-driven medulloblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B063.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96d2ace9063c026bc5c1042bf3f0c9b788241cdc" target='_blank'>
              Abstract B063: Unveiling the immune landscape of recurrent, MYC-driven medulloblastoma
              </a>
            </td>
          <td>
            Benjamin Draper, Dean Thompson, A. Morcavallo, Bethany Remeniuk, John Anderson, Louis Chesler, Steve Clifford, Frank Huang, Laura K. Donovan
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The membrane composition of extracellular vesicles (EVs) largely reflects that of the plasma membrane of the cell of origin. We therefore hypothesized that EVs could be used for immunizations to generate monoclonal antibodies against well-known tumor antigens but possibly also against hitherto unknown tumor-associated target molecules. From an immunization experiment, we obtained a monoclonal antibody specific for SRRM2, an RNA-binding protein involved in splicing and a major component of nuclear speckles. Here, we used this antibody to demonstrate that SRRM2 is exposed on the surface of most cancer cell lines from various entities and, even more important, on cancer cells in vivo. Moreover, we demonstrated that SRRM2-specific CAR-T cells are functional in vitro and in vivo. Collectively, we identified SRRM2 as a promising new target molecule exposed on the cancer cell surface and showed that our SRRM2-specific antibody can be used as a basis for the development of new targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e28b566c1806f22e74663da9c96c01cab220620e" target='_blank'>
              The Nuclear Speckles Protein SRRM2 Is Exposed on the Surface of Cancer Cells
              </a>
            </td>
          <td>
            Markus Kellner, Julia Hörmann, Susanne Fackler, Yuanyu Hu, Tielin Zhou, Lin Lu, Ibrahim Ilik, Tuğçe Aktaş, Regina Feederle, Stefanie M. Hauck, Olivier Gires, Kathrin Gärtner, Lietao Li, Reinhard Zeidler
          </td>
          <td>2024-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Glioblastoma Multiforme (GBM) is a high-grade glioma known to infiltrate the surrounding brain parenchyma inducing a direct neuronal damage. Indeed, GBM mass is not an isolated and self-sustained entity but rather participates and interacts with the surrounding neuronal tissue. Specifically, tumoral cells induce neuron hyper-excitability while peritumoral cells seem to regulate tumor proliferation and progression. GBM affects 3 in 100,000 people and has a median survival of 15 months; it remains poorly understood and lethal. Therefore, gaining a deeper understanding of the alterations occurring in peritumoral tissues during GBM progression is of paramount importance.



 We investigate the plastic rearrangements of motor cortical areas in two different glioma-bearing mouse models using optogenetic stimulation, two photon imaging and immunohistochemical analyses. Specifically, we took advantage of Thy1-ChR2 transgenic mice, injected with two syngeneic lines of murine glioma cells (i.e. GL261 and PDGF+ Trp53-/- cells) into the motor cortex. Mice were optogenetically stimulated in 120 points per session (covering the entire extent of forelimb motor representation) at 3 different time points: baseline (before glioma cells injection), 14 and 21 days after tumor implantation.



 We show that both models of glioma-bearing mice underwent a strong remapping of forelimb motor territory. Namely, an increased threshold required for eliciting a forelimb movement was coupled with a decrease of forelimb specificity in the peritumoral area with respect to naïve mice. In addition, glioma-bearing mice showed a decrease in the number of dendritic spines, along with reduced density of perineuronal nets (PNNs) and inhibitory interneuron markers in the peritumoral cortex.



 We provide important insight into the interaction between GBM and surrounding tissue that can be used for developing more effective treatments to improve patients’ quality of life and survival.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6297cf9746d4f9b221b8e8f9f648c2b95db0018f" target='_blank'>
              P02.22.A REARRANGEMENTS OF PERITUMORAL TISSUE THAT OCCUR ALONG WITH GLIOMA PROGRESSION
              </a>
            </td>
          <td>
            E. Mori, C. Spalletti, M. Scalera, S. Haddad, D. Cangi, V. Pillai, E. Parmigiani, M. Mainardi, S. Landi, M. Caleo, E. Vannini
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intravital microscopy has enabled the study of immune dynamics in the pulmonary microvasculature, but many key events remain unseen because they occur in deeper lung regions. We therefore developed a technique for stabilized intravital imaging of bronchovascular cuffs and collecting lymphatics surrounding pulmonary veins in mice. Intravital imaging of pulmonary lymphatics revealed ventilation-dependence of steady-state lung lymph flow and ventilation-independent lymph flow during inflammation. We imaged the rapid exodus of migratory dendritic cells through lung lymphatics following inflammation and measured effects of pharmacologic and genetic interventions targeting chemokine signaling. Intravital imaging also captured lymphatic immune surveillance of lung-metastatic cancers and lymphatic metastasis of cancer cells. To our knowledge, this is the first imaging of lymph flow and leukocyte migration through intact pulmonary lymphatics. This approach will enable studies of protective and maladaptive processes unfolding within the lungs and in other previously inaccessible locations. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ad7292e3dc5444dbf480a1cf6ce97a64b4f2de" target='_blank'>
              Intravital imaging of pulmonary lymphatics in inflammation and metastatic cancer
              </a>
            </td>
          <td>
            S. Cleary, Longhui Qiu, Yurim Seo, P. Baluk, Dan Liu, N. Serwas, Jason G. Cyster, Donald M. McDonald, Matthew F. Krummel, M. Looney
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, S. Shen, S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Many solid tumors including breast cancer can exhibit early dissemination and dormancy-in which cancer cells spread early in the disease process and survive long periods without detectable growth. These early disseminated tumor cells sometimes reactivate and lead to incurable metastatic disease years or even decades after curative-intent therapy for the primary tumor. We are just beginning to understand the role of the immune system in this process in part because of improvements in immunocompetent models as well as technological advances such as single-cell genomics and spatial transcriptomics. In this issue of Cancer Research, Bushnell and colleagues showed that NK cells are important in this context. The authors found that disseminated tumor cells and quiescent cells express higher levels of MHC 1 but are resistant to NK-cell-mediated immunity. The proposed mechanism involves the STING pathway and transcription factors Sox2 and Bach1. As other studies have highlighted the importance of T-cell immunity, this work reaffirms the importance and diversity of immune regulation of dormancy and suggests the need for future studies to flesh out mechanistic details and predict when each type of immunity is most important. See related article by Bushnell et al., p. 3337.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9576e1dca44b3bd419b3eed082360a6cebe6d307" target='_blank'>
              Breast Cancer Disseminated Tumor Cells: Do They Stay and Fight or Run and Hide?
              </a>
            </td>
          <td>
            F. Cackowski, H. Korkaya
          </td>
          <td>2024-10-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 The management of diffuse midline gliomas (DMG) is one of the greatest challenges in the field of pediatric oncology. Available treatments do not improve the poor survival rates, and preclinical research is struggling to find new targets that could have the potential to revolutionize the therapeutic approach. In most of the current preclinical tools, microglia and macrophages are not taken into account, despite being the most abundant immune players in the brain tumor microenvironment. To address this issue, we established neural organoids and macrophages (iMacs) from induced pluripotent stem cells (iPSC) derived from pediatric patients treated for DMG ("autologous immuno-organoids"). We have reprogrammed fibroblasts into iPSC from 9 children with DMG and generated autologous immuno-organoids for 4 of them. These "avatars" provide a unique preclinical model to which tumor cells from the same patient can be added, enabling us to better understand the crosstalk between the tumor and its immunological microenvironment, and to evaluate new immunotherapies. In particular, we show here that iMacs decrease tumor proliferation and tumor cell-invasion capability in neural organoids, with such effect reduced upon iMac depletion using CSF1R inhibitor treatment. Genes/signaling pathways identified by multi-omics analysis and validated at proteomic level show a reciprocal reprogramming of tumor cells and iMacs over time. Altogther, our approach reveals an unique tumor macrophage crosstalk that will be targeted by tailored immunotherapies or gene modification to validate their impact on tumor progression.
 Citation Format: Florent Ginhoux, Claudia Pasqualini. Mapping microglia – tumor crosstalk in pediatric brain cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr IA016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13e82f8323ca1ef256cf15c794e5b4bbac237db1" target='_blank'>
              Abstract IA016: Mapping microglia – tumor crosstalk in pediatric brain cancer
              </a>
            </td>
          <td>
            Florent Ginhoux, Claudia Pasqualini
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The cell cycle governs the proliferation, differentiation, and regeneration of all eukaryotic cells. Profiling cell cycle dynamics is therefore central to basic and biomedical research spanning development, health, aging, and disease. However, current approaches to cell cycle profiling involve complex interventions that may confound experimental interpretation. To facilitate more efficient cell cycle annotation of microscopy data, we developed CellCycleNet, a machine learning (ML) workflow designed to simplify cell cycle staging with minimal experimenter intervention and cost. CellCycleNet accurately predicts cell cycle phase using only a fluorescent nuclear stain (DAPI) in fixed interphase cells. Using the Fucci2a cell cycle reporter system as ground truth, we collected two benchmarking image datasets and trained two ML models—a support vector machine (SVM) and a deep neural network—to classify nuclei as being in either the G1 or S/G2 phases of the cell cycle. Our results suggest that CellCycleNet outperforms state-of-the-art SVM models on each dataset individually. When trained on two image datasets simultaneously, CellCycleNet achieves the highest classification accuracy, with an improvement in AUROC of 0.08–0.09. The model also demonstrates excellent generalization across different microscopes, achieving an AUROC of 0.95. Overall, using features derived from 3D images, rather than 2D projections of those same images, significantly improves classification performance. We have released our image data, trained models, and software as a community resource.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f17793307104d6eb05316e7fe271398598a89d" target='_blank'>
              Predicting cell cycle stage from 3D single-cell nuclear-stained images
              </a>
            </td>
          <td>
            Gang Li, Eva K. Nichols, Valentino E. Browning, Nicolas J. Longhi, Conor Camplisson, Brian J. Beliveau, W. S. Noble
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Aberrant expression of repeat RNAs in pancreatic ductal adenocarcinoma (PDAC) mimics viral-like responses with implications for tumor cell state and the surrounding microenvironment. To better understand the relationship of repeat RNAs in human PDAC, we employed the NanoString CosMx™ spatial molecular imaging (SMI) platform, which utilizes a 1,000-plex RNA panel with custom repeat RNA probes targeting long interspersed nuclear element 1 (LINE-1) retrotransposon ORF1 and ORF2, HSATII satellite (SAT) repeat, and two human endogenous retroviruses (HERV-K and HERV-H) in 46 primary tumors. This analysis revealed correlations of high repeat RNA expression with alterations in the epithelial state of PDAC cells and the myofibroblast phenotype in cancer-associated fibroblasts (CAFs). The loss of cellular identity observed with dosing of extracellular vesicles (EVs) and individual repeat RNAs in PDAC and CAF cell culture models points to cell-cell intercommunication involving these viral-like elements. Differences in the PDAC and CAF response are driven by distinct innate immune signaling pathways through interferon regulatory transcription factor 3 (IRF3). Altogether, our data indicate that cell context-specific viral-like responses driven by tumor cells have broad impacts on single-cell heterogeneity within cancer cells and the pancreatic cancer microenvironment.
 Citation Format: Eunae You, Patrick Danaher, Chenyue Lu, Siyu Sun, Luli Zou, Ildiko Phillips, Alexandra Rojas, Natalie Ho, Yuhui Song, Michael Raabe, Katherine Xu, Peter Richieri, Hao Li, Natalie Aston, Rebecca Porter, Bidish Patel, Linda Nieman, Nathan Schurman, Briana Hudson, Khrystyna North, Sarah Church, Vikram Deshpande, Andrew Liss, Tae Kim, Yi Cui, Youngmi Kim, Benjamin Greenbaum, Martin Aryee, David Ting. Repeat RNA mediated disruption of cellular plasticity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A039.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffeaf733e03a440a3ea9633bad9509d48160c804" target='_blank'>
              Abstract A039: Repeat RNA mediated disruption of cellular plasticity in pancreatic cancer
              </a>
            </td>
          <td>
            Eunae You, Patrick Danaher, Chenyue Lu, Siyu Sun, Luli Zou, Ildiko E Phillips, Alexandra Rojas, Natalie Ho, Yuhui Song, Michael J. Raabe, Katherine Xu, Peter Richieri, Hao Li, Natalie Aston, Rebecca L. Porter, Bidish Kumar Patel, Linda T. Nieman, Nathan Schurman, Briana M. Hudson, Khrystyna North, Sarah Church, Vikram Deshpande, Andrew S. Liss, T. Kim, Yi Cui, Youngmi Kim, Benjamin Greenbaum, Martin J. Aryee, David T. Ting
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The tumor microenvironment (TME) can be regarded as a complex and dynamic microecosystem generated by the interactions of tumor cells, interstitial cells, the extracellular matrix, and their products and plays an important role in the occurrence, progression and metastasis of tumors. In a previous study, we constructed an IEO model (prI-, prE-, and pOst-metastatic niche) according to the chronological sequence of TME development. In this paper, to fill the theoretical gap in spatial heterogeneity in the TME, we defined an MCIB model (Metabolic, Circulatory, Immune, and microBial microenvironment). The MCIB model divides the TME into four subtypes that interact with each other in terms of mechanism, corresponding to the four major links of metabolic reprogramming, vascular remodeling, immune response, and microbial action, providing a new way to assess the TME. The combination of the MCIB model and IEO model comprehensively depicts the spatiotemporal evolution of the TME and can provide a theoretical basis for the combination of clinical targeted therapy, immunotherapy, and other comprehensive treatment modalities for tumors according to the combination and crosstalk of different subtypes in the MCIB model and provide a powerful research paradigm for tumor drug-resistance mechanisms and tumor biological behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6e1c21293602e7c8448df8e77945274f0be047" target='_blank'>
              The MCIB Model: A Novel Theory for Describing the Spatial Heterogeneity of the Tumor Microenvironment
              </a>
            </td>
          <td>
            Minghao Guo, Yinan Sun, Xiaohui Wang, Zikun Wang, Xun Yuan, Xinyi Chen, Xianglin Yuan, Lu Wang
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Introduction Multiple myeloma (MM) is an incurable hematological malignancy with high chromosome instability and heavy dependence on the immunosuppressive bone marrow microenvironment. P53 mutations are adverse prognostic factors in MM; however, clinically, some patients without P53 mutations also exhibit aggressive disease progression. DNp73, an inhibitor of TP53 tumor suppressor family members, drives drug resistance and cancer progression in several solid malignancies. Nevertheless, the biological functions of DNp73 and the molecular mechanisms in myelomagenesis remain unclear. Methods The effects of DNp73 on proliferation and drug sensitivity were assessed using flow cytometry and xenograft models. To investigate the mechanisms of drug resistance, RNA-seq and ChIP-seq analyses were performed in MM cell lines, with validation by Western blot and RT-qPCR. Immunofluorescence and transwell assays were used to assess DNA damage and cell invasion in MM cells. Additionally, in vitro phagocytosis assays were conducted to confirm the role of DNp73 in immune evasion. Results Our study found that activation of NF-κB-p65 in multiple myeloma cells with different p53 mutation statuses upregulates DNp73 expression at the transcriptional level. Forced expression of DNp73 promoted aggressive proliferation and multidrug resistance in MM cells. Bulk RNA-seq analysis was conducted to assess the levels of MYCN, MYC, and CDK7. A ChIP-qPCR assay was used to reveal that DNp73 acts as a transcription factor regulating MYCN gene expression. Bulk RNA-seq analysis demonstrated increased levels of MYCN, MYC, and CDK7 with forced DNp73 expression in MM cells. A ChIP-qPCR assay revealed that DNp73 upregulates MYCN gene expression as a transcription factor. Additionally, DNp73 promoted immune evasion of MM cells by upregulating MYC target genes CD47 and PD-L1. Blockade of the CD47/SIRPα and PD-1/PD-L1 signaling pathways by the SIRPα-Fc fusion protein IMM01 and monoclonal antibody atezolizumab significantly restored the anti-MM activity of macrophages and T cells in the microenvironment, respectively. Discussion In summary, our study demonstrated for the first time that the p53 family member DNp73 remarkably induces proliferation, drug resistance, and immune escape of myeloma cells by directly targeting MYCN and regulating the MYC pathway. The oncogenic function of DNp73 is independent of p53 status in MM cells. These data contribute to a better understanding of the function of TP53 and its family members in tumorigenesis. Moreover, our study clarified that DNp73 overexpression not only promotes aggressive growth of tumor cells but, more importantly, promotes immune escape of MM cells through upregulation of immune checkpoints. DNp73 could serve as a biomarker for immunotherapy targeting PD-L1 and CD47 blockade in MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29c2317b2b047eb0eb6e4a0882a65574253f5b39" target='_blank'>
              DNp73 enhances tumor progression and immune evasion in multiple myeloma by targeting the MYC and MYCN pathways
              </a>
            </td>
          <td>
            Lanting Liu, Dasen Gong, Hao Sun, Fangshuo Feng, Jie Xu, Xiyue Sun, L. Gong, Zhen Yu, Teng-Ching Fang, Yan Xu, R. Lyu, Tingyu Wang, Wentian Wang, Wenzhi Tian, Lugui Qiu, Gang An, Mu Hao
          </td>
          <td>2024-09-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Introduction Reprogramming of pancreas cell fate drives development of pancreatic ductal adenocarcinoma (PDAC). Acinar cells, the most probable origin of pancreatic cancer, undergo a rapid cell identity switch towards a duct-like phenotype upon KrasG12D expression and when combined with pancreatitis or the loss of acinar differentiation factors. Metaplastic and neoplastic duct-like cells are heterogeneous with a proportion acquiring features reminiscent of gastric lineages. While some gastric signatures are maintained in the classical PDAC subtype, they are eroded in the more aggressive, basal-like PDAC. Since subtype identity has a major impact on prognosis and therapeutic targetability, druggable targets that regulate cellular reprogramming in pancreatic cancer can be exploited to increase sensitivity to therapy. Methods To elucidate mechanisms responsible for early reprogramming, pancreatic tissue of mice with conditional KrasG12D expression and loss of Pdx1 was analyzed by single-nucleus ATAC-seq. Expression of identified target genes was studied in precancerous lesions and PDAC by using multiplex RNAscope and IHC staining. Computational analyses of publicly available sequencing data were used to establish correlation to PDAC subtype identity. Genes were functionally studied in PDAC cell lines. Results By performing snATAC-seq and RNA-seq of early transformed pancreatic tissue, we discovered that acinar cells with the combined expression of KrasG12D and loss of Pdx1 activate expression of a gastric metaplastic gene signature accompanied by elevated levels of the receptor kinase Ror2. Ror2 is also highly expressed in distinct subpopulations of metaplastic and neoplastic cells, associated with a gastric neck cell phenotype and enhanced proliferative capacity. In contrast, Ror2Low lesions are characterized by a gastric pit cell-like and a senescent phenotype. Genetic ablation of Ror2 resulted in a shift in lesion identity, enriching those with a pit cell and senescent identity. In PDAC, we found that Ror2 anti-correlates with a similar gastric pit cell phenotype that is maintained in the classical subtype PDAC, but is strongly associated with the more aggressive basal-like subtype. Overexpression of ROR2 in human PDAC cell lines with a classical differentiation induced loss of the classical gene signature as well as epithelial-to-mesenchymal transition. Moreover, ROR2 enforces a strong dependency on AKT signaling, causing increased vulnerability of ROR2-expressing cells to AKT inhibition, but increased resistance to the KRAS inhibitor MRTX1133. Conclusions We discovered Ror2 as a critical determinant of precancerous lesion as well as PDAC subtype identity. Its role in driving an aggressive PDAC phenotype that is inherently resistant to Kras inhibition suggests that inhibiting this receptor tyrosine kinase will enhance sensitivity to the new generation of targeted therapies.
 Citation Format: Simone Benitz, Alec Steep, Malak Nasser, Jonathan Preall, Ujjwal Mukund Mahajan, Holly McQuithey, Ian Loveless, Erick T. Davis, Hui-Ju Wen, Daniel W. Long, Thomas Metzler, Samuel Zwernik, Michaela Louw, Donald Rempinski, Daniel Salas-Escabillas, Sydney Brender, Linghao Song, Ling Huang, Brian K. Theisen, Zhenyu Zhang, Nina G. Steele, Ivonne Regel, Filip Bednar, Howard C. Crawford. Ror2, a Novel Key Regulator Driving Cell Fate Decisions throughout Pancreatic Carcinogenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C075.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada52d632c14a2e67876713dc628440690c3b38e" target='_blank'>
              Abstract C075: Ror2, a Novel Key Regulator Driving Cell Fate Decisions throughout Pancreatic Carcinogenesis
              </a>
            </td>
          <td>
            S. Benitz, Alexander Steep, Malak M Nasser, Jonathan Preall, U. Mahajan, Holly McQuithey, Ian Loveless, Erick T Davis, Hui-Ju Wen, Daniel W Long, Thomas Metzler, Samuel Zwernik, Michaela Louw, D. Rempinski, Daniel J. Salas-Escabillas, Sydney M Brender, Linghao Song, Ling Huang, Brian Theisen, Zhenyu Zhang, Nina G. Steele, I. Regel, Filip Bednar, Howard C. Crawford
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Gynecological cancers, notably breast and ovarian cancers, exhibit significant heterogeneity, complicating treatment strategies and impacting patient outcomes. Traditional cancer models often fail to capture the complexity and diversity of these tumors. Induced pluripotent stem cells (iPSCs) offer a promising alternative for modeling cancer due to their ability to differentiate into various cell types. This study aims to establish and characterize iPSC-derived models of breast and ovarian cancers to explore their heterogeneity and therapeutic responses. 
We cultured iPSCs and differentiated them into breast and ovarian cancer cell lineages using lineage-specific protocols. Differentiation was confirmed by the expression of specific markers (CK14, CK18 for breast cancer; CA125, CK7 for ovarian cancer). We generated tumor spheroids from the differentiated cells and assessed their morphology, size, and viability. Functional assays revealed significant differences in invasive and migratory capabilities between the two cancer models. 
Gene and protein expression analyses highlighted the upregulation of BRCA1 and BRCA2 in breast cancer models and higher TP53 expression in ovarian cancer models. Proliferation assays demonstrated variability in drug sensitivity, with breast cancer spheroids showing higher sensitivity to trastuzumab and ovarian cancer spheroids to olaparib. Apoptosis assays indicated higher basal and treatment-induced apoptotic activity in ovarian cancer spheroids. Angiogenesis potential, assessed using HUVEC tube formation assays, was greater in ovarian cancer models. 
Our results validate the use of iPSC-derived models for studying gynecological cancer heterogeneity. These models accurately reflect the molecular and functional diversity observed in patient tumors, providing a robust platform for investigating cancer biology and evaluating therapeutic strategies. Future research should expand these models to include additional gynecological cancers, incorporate patient-derived iPSCs for personalized medicine, and utilize multi-omics approaches to further understand cancer heterogeneity and improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0a0aa8bdddeea537f2d24a9ca65d650114826a0" target='_blank'>
              Study of Gynecological Cancer Heterogeneity in Breast and Ovarian Cancer Models Using Induced Pluripotent Stem Cells
              </a>
            </td>
          <td>
            King Williams, E. Godswill, Chiamaka Mercy, Uchechukwu Gift Chinenye, Ochai Onyiba Blessing, Imonikhu Francis, Ajifolawe Oladotun, Franklin Uche, Williams Williams
          </td>
          <td>2024-09-03</td>
          <td>Journal of Cancer and Tumor International</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35826bd0e07cbabe1eca463c7fc2d242b98d2d06" target='_blank'>
              Combined single cell and spatial transcriptome analysis reveals cellular heterogeneity of hedgehog pathway in gastric cancer.
              </a>
            </td>
          <td>
            Guoliang Zhang, Guojun Xia, Chunxu Zhang, Shaodong Li, Huangen Wang, Difeng Zheng
          </td>
          <td>2024-09-09</td>
          <td>Genes and immunity</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56281f9e2c04f45376f07cc22699136fff5305aa" target='_blank'>
              Exploring the interaction between immune cells in the prostate cancer microenvironment combining weighted correlation gene network analysis and single-cell sequencing: An integrated bioinformatics analysis
              </a>
            </td>
          <td>
            Danial Hashemi Karoii, Sobhan Bavandi, Melika Djamali, Ali Shakeri Abroudi
          </td>
          <td>2024-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma, IDH-wildtype WHO Grade 4 (GB), is the most common primary malignant brain tumor. Despite significant advancements in surgical techniques allowing for a greater extent of resection, the use of targeted radiotherapy along with the application of novel anticancer drugs, GB remains an incurable disease with a median survival of 16 months. One of the key factors contributing to the development of therapeutic resistance and recurrence is the intratumoral heterogeneity of GB. Although surgical therapy aims at maximal safe resection with the goal of gross total tumor resection (GTR) when feasible, a considerable heterogeneous residual tumor population remains at the resection margin. These residual tumor populations encompass various genomic, epigenetic, and microenvironmental variations, creating a pool from which treatment-resistant clones can emerge and subsequently become a source of GB recurrence. This pilot study aims to analyze the molecular characteristics of these high-risk cell populations in the peritumoral infiltration zone using next-generation sequencing (NGS), with the objective of identifying driving factors contributing to early relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37e369c3e6e516e2fccd96f8a11d4f444a655ce0" target='_blank'>
              P01.10.A INVESTIGATING GLIOBLASTOMA BIOMARKERS IN THE INFILTRATION ZONE: TOPOGRAPHIC LINKS TO AN EARLY RELAPSE
              </a>
            </td>
          <td>
            M. Barak, J. Brychta, M. Hendrych, T. Holecek, K. Holikova, T. Kazda, M. Hermanova, J. Šána, R. Jancalek
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is a highly diffusive tumor which restricts the efficacy of surgical resection and facilitates tumor recurrence. Matrix metalloproteinase 2 (MMP2) is the extracellular matrix protease that is activated by MMP14 expressed by microglia and plays a central role in tissue remodeling and tumor progression. The R132H mutation in isocitrate dehydrogenase 1 (IDH1R132H) is commonly observed in gliomas, results in hypermethylator phenotype and in GBMs is associated with better survival. The impacts of IDH1R132H on the tumor microenvironment of mIDH1/mATRX/mTP53 gliomas remains unexplored.



 We employed unique mouse glioma models which mimic human GBM pathology, coupled with high-dimensional technologies (CITEseq and spatial transcriptomics) to analyze and visualize the immune landscape in different gliomas. Additionally, we performed gelatin zymography, qPCR, wound healing assays, and DNA methylation qPCR techniques to validate the obtained results. MMP-2 expression/methylation was studied in the glioma TCGA dataset.



 The expression and activity of Mmp2 were different in experimental gliomas with different IDH1 status. Spatial transcriptomics demonstrated that Mmp2 expression is localized within the IDH1 wild-type tumors and absent in IDH1 mutated ones. Gelatin zymography showed the presence of Mmp2 activity in extracts from IDH1 wild-type tumors. Methylation of Mmp2 gene promoter was increased and resulted in decreased Mmp2 expression in IDH1 mutated gliomas. The IDH1 wild-type gliomas showed more diffuse growth patterns (tumors were stained with Nestin antibody) whereas IDH1 mutant gliomas showed more constrained growth patterns. In the TCGA glioma dataset, methylation of the MMP2 gene promoter was increased which was associated with decreased MMP2 expression in IDH1 mutated gliomas.



 We found that the IDH1 mutation results in hypermethylation of the Mmp2 gene promoter, leading to decreased Mmp2 expression and Mmp2 activity in experimental gliomas and human GBMs. Reduced Mmp2 activity and decreased tumor invasion result in more restricted tumor growth, which may facilitate resection, and radiotherapy, thereby improving patient survival. Studies were supported by National Science Center Poland grant 2020/39/B/NZ4/02683 (BK) and PASIFIC Call 1 PAS (MG).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81ebfa9c8baa458907e58b8a2d93e1bdfd385428" target='_blank'>
              P02.24.B THE IMPACT OF THE IDH1 R132H MUTATION ON MMP2 METHYLATION, EXPRESSION AND INVASION IN EXPERIMENTAL GLIOMAS AND HUMAN GBMS
              </a>
            </td>
          <td>
            A. M. Lenkiewicz, A. Ibragimova, M. Ghosh, P. Szadkowska, A. Ellert-Miklaszewska, S. Baluszek, K. Jacek, M. Castro, B. Kaminska
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Diffuse intrinsic pontine gliomas (DIPGs), a major type of pediatric high-grade gliomas located in the pons, are the leading cause of death in children with brain cancer. A subset (20-25%) of DIPGs harbor a lysine 27-to-methionine (K27M) mutation in HIST1H3B, which encodes histone H3.1, and an activating ACVR1 mutation. The occurrence of this pair of mutations in DIPGs, but not in pediatric gliomas in other anatomical locations, suggests the existence of a pontine-specific niche that favors DIPG gliomagenesis. Unfortunately, the identity of the underlying pontine niche remains elusive as available mouse models fail to recapitulate the anatomic specificity that characterizes DIPGs. Herein we show that the trigeminal root entry zone (TREZ), a pontine structure where several major axon tracts intersect, is enriched with proliferating oligodendrocyte-lineage cells during brainstem development. Introducing both H3.1K27M and activating Acvr1 and Pik3ca mutations (which co-occur frequently with H3.1K27M in human DIPGs) into the mouse brain leads to rapid gliomagenesis. This pathology recapitulates the pons specificity of DIPGs as glioma cells proliferate on axon tracts at the TREZ. We further show that a hyaluronan receptor important for cell stemness (HMMR) plays a key role in glioma cell proliferation at the TREZ. We propose that H3.1K27M and its co-occurring mutations drive pontine specific gliomagenesis by inducing a proliferative response of oligodendrocyte-lineage cells with enhanced stemness on large TREZ axon tracts. One-Sentence Summary The trigeminal root entry zone underlies pontine-specific gliomagenesis driven by H3.1K27M and its co-occurring mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edde84a5692dddffb8e8ac4f9ce9076078396750" target='_blank'>
              A pontine-specific axonal niche supports de novo gliomagenesis
              </a>
            </td>
          <td>
            Zhigang Xie, Adrija Pathak, V. Bankaitis
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="In esophageal squamous cell carcinoma (ESCC), lymph node (LN) metastasis is associated with poor survival. Emerging evidence has demonstrated elevated CD8+ T-cell levels in metastatic LNs following immunotherapy and increased chemoresistance. However, the underlying regulatory mechanisms of CD8+ T cells in chemoresistant/metastatic patients have not been elucidated. Given that metastasis is linked to aberrant splicing patterns, transcripts with alternative splicing forms also play a critical role. With spatial transcriptomics (ST), spatial isoform transcriptomics (SiT), single-cell RNA sequencing (scRNA-seq), and T-cell receptor (TCR) sequencing, the spatial isoforms are analyzed quantitatively in human solid tumors and LNs. These isoforms are classified according to expression and spatial distribution patterns and proposed that the temporal heterogeneity in isoforms is attributed to isoform biogenesis dynamics. C1QC+ tumor-associated macrophages (TAMs) contribute to the formation of metastases in the context of both immunotherapy and chemotherapy. Additionally, CD74 isoforms can be targeted by mRNA drugs, such as antisense oligonucleotides (ASOs) and RNA interference (RNAi), to prevent chemoresistance and metastasis. Overall, this work suggests that C1QC+ TAMs interact with CD8+ CXCL13+ Tex cells via enrichment with the CD74 isoform in the ESCC 's metastatic lymph node.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b661dec030c9bd326290c3a36f5208dd2dc40655" target='_blank'>
              Spatial Isoforms Reveal the Mechanisms of Metastasis.
              </a>
            </td>
          <td>
            Yin Yin, Yuhao Wang, Xiaofei Yu, Yang Li, Yahui Zhao, Yanfeng Wang, Zhihua Liu
          </td>
          <td>2024-09-23</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9108cf4e9dae8634b2222365d27b43224c1ce0ff" target='_blank'>
              Deciphering the ghost proteome in ovarian cancer cells by deep proteogenomic characterization
              </a>
            </td>
          <td>
            Diego Fernando Garcia-Del Rio, M. Derhourhi, Amélie Bonnefond, Sébastien Leblanc, Noé Guilloy, X. Roucou, S. Eyckerman, Kris Gevaert, M. Salzet, T. Cardon
          </td>
          <td>2024-09-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="The genomic features of pancreatic ductal adenocarcinoma (PDAC) have been well described, yet the evolutionary contexts within which those features occur remains unexplored. We studied the genome landscapes, phylogenies and clonal compositions of 91 PDACs in relation to clinicopathologic features. There was no difference in the number of driver mutations or the evolutionary timing that each mutation occurred. High truncal density, a metric of the accumulation of somatic mutations in the lineage that gave rise to each PDAC, was significantly associated with worse overall survival. Polyclonal, monoclonal or mixed polyclonal/monoclonal metastases were identified across the cohort highlighting multiple forms of inter-tumoral heterogeneity. Advanced stage and treated PDACs had higher odds of being polyclonal, whereas oligometastatic PDACs had fewer driver alterations, a lower fractional allelic loss and increased likelihood of being monoclonal. In sum, our findings reveal novel insights into the dynamic nature of the PDAC genome beyond established genetic paradigms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bec3471b710f3087aaa9089b00fcdde1b15c6db" target='_blank'>
              The Evolutionary Forest of Pancreatic Cancer.
              </a>
            </td>
          <td>
            Katelyn M Mullen, Jungeui Hong, M. Attiyeh, Akimasa Hayashi, Hitomi Sakamoto, Z. Kohutek, Caitlin A McIntyre, Haochen Zhang, Alvin P. Makohon-Moore, Amanda Zucker, L. D. Wood, Matthew A. Myers, Brian J. Arnold, S. Zaccaria, Joanne F Chou, M. Capanu, N. Socci, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-10-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667549573815f8b84f3de00bdda0de7e5af3d623" target='_blank'>
              Emerin mislocalization during chromatin bridge resolution can drive prostate cancer cell invasiveness in a collagen-rich microenvironment
              </a>
            </td>
          <td>
            Marta Popęda, Kamil Kowalski, Tomasz Wenta, Galina V Beznoussenko, Michał Rychłowski, Alexander A. Mironov, Zeno Lavagnino, Sara Barozzi, Julia Richert, Rebecca Bertolio, Kamil Myszczyński, J. Szade, Michał Bieńkowski, Kevin Miszewski, Marcin Matuszewski, A. Żaczek, Luca Braga, Giannino Del Sal, N. Bednarz-Knoll, P. Maiuri, P. Nastały
          </td>
          <td>2024-09-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 One key biological challenge in glioblastoma (GB) is the diffuse invasion of GB cells into the brain parenchyma beyond the margins of even supramaximal resection strategies. Various cell-intrinsic and extrinsic factors have been shown to regulate GB invasion, but it is unknown whether invasive GB cells acquire a persistent distinct genetic and epigenetic profile in response to environmental cues.



 Highly invasive isogenic subpopulations of GB cells (GBinv+ cells) were generated by serial invasion through matrix-coated transwell membranes. RNA sequencing and array-based DNA methylation profiling of control and GBinv+ cells were performed. Differentially expressed genes (DEGs) and differentially methylated probes (DMPs) were analysed using DESeq and RnBeads R package, respectively. Functional correlations of DEGs and DMPs were analysed using Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG). Candidate genes were validated using TCGA-LGG/GBM datasets and experimentally confirmed by genetic modulation of gene expression changes and applying classical in vitro invasion assays as well as the GLIoma Cerebral Organoid (GLICO) model for complex ex vivo tumor cell invasion analysis.



 In classical in vitro invasion assays, the invasion potential of GBinv+ cells were more than 10-fold increased relative to the isogenic control cells baseline (p < 0.0001, students t-test). Increased invasion was also demonstrated using the GLICO model. GBinv+ cells were found to have unique transcriptomic and DNA-methylation profiles that persisted for >2 weeks after removal of the stimulus. Genetic deletion of candidate genes identified by profiling reduced invasive properties.



 Our results indicate that the invasive sub-population of GB cells is characterized by a specific genetic and epigenetic profile and has enduring functional properties. The functional validation of the identified genes associated with invasion underscores their clinical relevance and suggests they may represent interesting new targets for experimental therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0482b1d060ae423ff8b68090f02deb204eff56a" target='_blank'>
              P01.12.A PERSISTENT GENETIC AND EPIGENETIC CHANGES ARE INDUCED BY INVASION STIMULI IN GLIOBLASTOMA CELLS
              </a>
            </td>
          <td>
            T. Müller, S. Schreiner, C. Cialini, U. Salcedo, M. Mittelbronn, K. H. Plate, J. Steinbach, A. Hau
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PURPOSE
The spatial variability and clinical relevance of the tumour immune microenvironment (TIME) are still poorly understood for hepatocellular carcinoma (HCC). Here we aim to develop a deep learning (DL)-based image analysis model for the spatial analysis of immune cell biomarkers, and microscopically evaluate the distribution of immune infiltration.


EXPERIMENTAL DESIGN
Ninety-two HCC surgical liver resections and 51 matched needle biopsies were histologically classified according to their immunophenotypes: inflamed, immune-excluded, and immune-desert. To characterise the TIME on immunohistochemistry (IHC)-stained slides, we designed a multi-stage DL algorithm, IHC-TIME, to automatically detect immune cells and their localisation in TIME in tumour-stromal, centre-border segments.


RESULTS
Two models were trained to detect and localise the immune cells on IHC-stained slides. The framework models, i.e. immune cell detection models and tumour-stroma segmentation, reached 98% and 91% accuracy, respectively. Patients with inflamed tumours showed better recurrence-free survival than those with immune-excluded or immune desert tumours. Needle biopsies were found to be 75% accurate in representing the immunophenotypes of the main tumour. Finally, we developed an algorithm that defines immunophenotypes automatically based on the IHC-TIME analysis, achieving an accuracy of 80%.


CONCLUSIONS
Our DL-based tool can accurately analyse and quantify immune cells on IHC-stained slides of HCC. The microscopical classification of the TIME can stratify HCCs according to the patient prognosis. Needle biopsies can provide valuable insights for TIME-related prognostic prediction, albeit with specific constraints. The computational pathology tool provides a new way to study the HCC TIME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43d6196ecc3cd0fe4ccb32f9d98d18fada8fb6b0" target='_blank'>
              Hepatocellular Carcinoma Immune Microenvironment Analysis: A Comprehensive Assessment with Computational and Classical Pathology.
              </a>
            </td>
          <td>
            Caner Ercan, S. Renne, L. di Tommaso, Charlotte K. Y. Ng, S. Piscuoglio, Luigi Terracciano
          </td>
          <td>2024-09-12</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>1</td>
          <td>47</td>
        </tr>

        <tr id="Cellular senescence (CS) and quiescence (CQ) are stress responses characterised by persistent and reversible cell cycle arrest, respectively. These phenotypes are heterogeneous, dependent on the cell type arrested and the insult inciting arrest. Because a universal biomarker for CS has yet to be identified, combinations of senescence-associated biomarkers linked to various biological stress responses including lysosomal activity (β-galactosidase staining), inflammation (senescence-associated secretory phenotypes, SASPs), and apoptosis (senescent cell anti-apoptotic pathways) are used to identify senescent cells. Using in vitro human bulk RNA-seq datasets, we find that senescent states enrich for various stress responses in a cell-type, temporal, and insult-dependent manner. We further demonstrate that various gene signatures used to identify senescent cells in the literature also enrich for stress responses, and are inadequate for universally and exclusively identifying senescent samples. Genes regulating stress responses – including transcription factors and genes controlling chromatin accessibility – are contextually differentially expressed, along with key enzymes involved in metabolism across arrest phenotypes. Additionally, significant numbers of SASP proteins can be predicted from senescent cell transcriptomes and also heterogeneously enrich for various stress responses in a context-dependent manner. We propose that ‘senescence’ cannot be meaningfully defined due to the lack of underlying preserved biology across senescent states, and CS is instead a mosaic of stress-induced phenotypes regulated by various factors, including metabolism, TFs, and chromatin accessibility. We introduce the concept of Stress Response Modules, clusters of genes modulating stress responses, and present a new model of CS and CQ induction conceptualised as the differential activation of these clusters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9e3857a67639ce9c496402c857eab9b897345d6" target='_blank'>
              Mosaic Regulation of Stress Pathways Underlies Senescent Cell Heterogeneity
              </a>
            </td>
          <td>
            R. Avelar, Thomas Duffield, Cyril Lagger, Nikita Krstevska, Marian Breuer, J. de Magalhães
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="The chromatins are folded into three-dimensional (3D) structures inside cells, which coordinates the regulation of gene transcription by the non-coding regulatory elements. Aberrant chromatin 3D folding has been shown in many diseases, such as acute myeloid leukemia (AML), and may contribute to tumorigenesis. The anthracycline topoisomerase II inhibitors can induce histone eviction and DNA damage. We performed genome-wide high-resolution mapping of the chemotherapeutic effects of various clinically used anthracycline drugs. ATAC-seq was used to profile the histone eviction effects of different anthracyclines. TOP2A ChIP-seq was used to profile the potential DNA damage regions. Integrated analyses show that different anthracyclines have distinct target selectivity on epigenomic regions, based on their respective ATAC-seq and ChIP-seq profiles. We identified the underlying molecular mechanism that unique anthracycline variants selectively target chromatin looping anchors via disrupting CTCF binding, suggesting an additional potential therapeutic effect on the 3D genome. We further performed Hi-C experiments, and data from K562 cells treated with the selective anthracycline drugs indicate that the 3D chromatin organization is disrupted. Furthermore, AML patients receiving anthracycline drugs showed altered chromatin structures around potential looping anchors, which linked to distinct clinical outcomes. Our data indicate that anthracyclines are potent and selective epigenomic targeting drugs and can target the 3D genome for anticancer therapy, which could be used for personalized medicine to treat tumors with aberrant 3D chromatin structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/760450c07be781a1d46120b324cb67687f8de734" target='_blank'>
              Targeting the 3D genome by anthracyclines for chemotherapeutic effects
              </a>
            </td>
          <td>
            Minkang Tan, Shengnan Sun, Yuchen Liu, Andrea A. Perreault, D. Phanstiel, Liping Dou, Baoxu Pang
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b681524478cf528a97b828064b23970db22097a" target='_blank'>
              Epigenomic heterogeneity as a source of tumour evolution.
              </a>
            </td>
          <td>
            Marthe Laisné, Mathieu Lupien, C. Vallot
          </td>
          <td>2024-10-16</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54957bdfa5b4be55fc758aeedcf61e34d1b8b67f" target='_blank'>
              Exploring the heterogeneity of interstitial cells of Cajal and their properties in gastrointestinal mesenchymal tumors applying single-cell RNA sequencing analysis
              </a>
            </td>
          <td>
            Yongjun Zhu, Rui Zhang, Shipai Zhang, Haijun Hu
          </td>
          <td>2024-09-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The discovery of biomarkers, essential for successful drug development, is often hindered by the limited availability of tissue samples, typically obtained through core needle biopsies. Standard 'omics platforms can consume significant amounts of tissue, forcing scientist to trade off spatial context for high-plex assays, such as genome-wide assays. While bulk gene expression approaches and standard single-cell transcriptomics have been valuable in defining various molecular and cellular mechanisms, they do not retain spatial context. As such, they have limited power in resolving tissue heterogeneity and cell-cell interactions. Current spatial transcriptomics platforms offer limited transcriptome coverage and have low throughput, restricting the number of samples that can be analyzed daily or even weekly. While the Digital Spatial Profiling (DSP) method does not provide single-cell resolution, it presents a significant advancement by enabling scalable whole transcriptome and ultrahigh-plex protein analysis from distinct tissue compartments and structures using a single tissue slide. These capabilities overcome significant constraints in biomarker analysis in solid tissue specimens. These advancements in tissue profiling play a crucial role in deepening our understanding of disease biology and in identifying potential therapeutic targets and biomarkers. To enhance the use of spatial biology tools in drug discovery and development, the DSP Scientific Consortium has created best practices guidelines. These guidelines, built on digital spatial profiling data and expertise, offer a practical framework for designing spatial studies and using current and future spatial biology platforms. The aim is to improve tissue analysis in all research areas supporting drug discovery and development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d05b5aa3ccb41d371a5372e1a65f3513ef0d582b" target='_blank'>
              A best practices framework for spatial biology studies in drug discovery and development: enabling successful cohort studies using digital spatial profiling.
              </a>
            </td>
          <td>
            David Krull, Premi Haynes, Anil Kesarwani, Julien Tessier, Benjamin J Chen, Kelly Hunter, Deniliz Rodriguez, Yan Liang, Jim Mansfield, Maxine McClain, Corinne Ramos, Edward Bonnevie, Esperanza Anguiano
          </td>
          <td>2024-09-03</td>
          <td>Journal of histotechnology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ovarian cancer is the sixth leading cause of cancer-related mortality among individuals with ovaries, and high-grade serous ovarian cancer (HGSOC) is the most common and lethal subtype. Characterized by a distinct and aggressive metastatic pattern, HGSOC can originate in the fallopian tube with the transformation of fallopian tube epithelial (FTE) cells, which metastasize to the ovary and subsequently to the omentum and peritoneal cavity. The omentum is a privileged metastatic site, and the metabolic exchange underlying omental metastasis could provide enzyme or receptor targets to block spread. In this study, we adapted a mass spectrometry imaging (MSI) protocol to investigate spatial location of 3D cocultures of tumorigenic FTE cells when grown in proximity to murine omental explants as a model of early metastatic colonization. Our analysis revealed several altered metabolites in tumorigenic FTE/omentum cocultures, namely changes in branched-chain amino acids (BCAA), including valine. We quantified the heightened consumption of valine, other BCAAs, and other amino acid-derived metabolites in omental cocultures using LC-MS assays. Our analysis revealed that metabolite concentrations when monitored with MSI from cell culture media in living culture systems have notable considerations for how MSI data may produce signatures that induce ionization suppression. Supplementation with valine enhanced proliferation and mTOR signaling in tumorigenic FTE cells, suggesting the potential of BCAA’s as a nutrient utilized by tumor cells during omental colonization and a possible target for metastasis. SIGNIFICANCE This study uncovers altered amino acid metabolism, specifically increased BCAA catabolism, at the interface of ovarian cancer cells and omental tissue in a coculture model of HGSOC secondary metastasis. Enhanced BCAA catabolism may promote cancer cell proliferation through mTOR signaling, presenting potential therapeutic value. These findings deepen our understanding of HGSOC pathogenesis and the metastatic tumor microenvironment, offering insights for developing new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe5b80a1daade6bcc4d3acb9445f9be702ff225f" target='_blank'>
              Branched-chain amino acid catabolism promotes ovarian cancer cell proliferation via phosphorylation of mTOR
              </a>
            </td>
          <td>
            Hannah J. Lusk, Monica A Haughan, Tova M Bergsten, Joanna E. Burdette, Laura M. Sanchez
          </td>
          <td>2024-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 20],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>