<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-17 09:41:40 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="


 Glioblastoma are hallmarked by clonal evolution and transcriptional heterogeneity linked to genetic and environmental diversity resulting in therapy resistance and disease progression. In this study, we integrate heritable molecular barcoding and spatial transcriptomics to enable high-resolution cell lineage tracing in the spatial context of the glioblastoma TME.



 Patient-derived glioblastoma cells from two individual cases were isolated from anatomically and pathologically distinct regions: the primary tumor core, the tumor periphery, and the recurrent lesion. Each subline was transduced with predefined FlexTrack lentiviral plasmids encoding GFP, resulting in uniquely barcoded (9N) cell lines. Immediately before injection, the barcoded glioblastoma populations were pooled and introduced into ex vivo human cortical slices. Live-imaging was used to trace GFP-positive, barcoded cell populations. After incubation, the fresh-frozen slices were processed for spatial transcriptomics and the line-specific barcodes were mapped according to the 3′ poly(A) tails of captured transcripts. Graph neural networks based on single-cell deconvolution were employed to spatially analyze the cellular ecosystems.



 We successfully deconvoluted and reconstructed the trajectories of injected glioblastoma cells linking their transcriptome phenotypes along with clonal identity. Multi-omic integration of spatial copy-number alterations and barcoding based pseudo-trajectories enabled reconstruction of the temporal transcriptional dynamics linked to the heterogeneity of tumor ecosystems.



 This study serves as a proof-of-concept for integrating molecular barcoding and spatial transcriptomics in cortical slice culture. Our approach to molecular barcoding in complex model systems opens new avenues to investigate the contextual dynamics of clonal evolution in glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f785ed6bb81bfa9c1d1d1cee449d66b313a3ee5" target='_blank'>
              P02.23.B MOLECULAR BARCODING OF GLIOBLASTOMA IN HUMAN ORGANOTYPIC CORTICAL SLICE CULTURE
              </a>
            </td>
          <td>
            G. Kastner, M. Mühlbauer, J. Jung, S. Langer, L. Müller, A. Baginska, P. Berger, C. Dobersalske, S. Ertmer, F. M. Gerach, N. Neidert, K. Herrmann, L. Rauschenbach, L. van Hijfte, U. Sure, K. D. Kokkaliaris, B. M. Grüner, D. Heiland, B. Scheffler
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8ce4a4012ffb45da6b02201b0801273b5519b7e" target='_blank'>
              Integrative single-cell and spatial mapping of oxidative stress response uncovers GCLC+ mesenchymal tumor cell state linked with favorable outcomes in triple negative breast cancer
              </a>
            </td>
          <td>
            Nomeda Girnius, Tuulia Vallius, Wenqing Chen, I.-M. Launonen, S. Palomino-Echeverría, Jia-Ren Lin, Caitlin E. Mills, Silja Kauppila, Pauliina Kronqvist, Antti Ellonen, Merja Perala, Eloise Withnell, Yu-An Chen, M. Secrier, S. Santagata, P. Sorger, Anniina Farkkila
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728733c32d8e17fbc34a5070c8b658170e0a8a22" target='_blank'>
              Identifying tissue states by spatial protein patterns related to chemotherapy response in triple-negative breast cancer
              </a>
            </td>
          <td>
            L.M. Luque, A. Khan, G. Torrisi, Tessa D. Green, D. Hardman, C. Owczarek, T. A. Phillips, D. Marks, M. Parsons, C. Sander, L. Schumacher
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Tumors acquire a set of functional capabilities, known as the Hallmarks of Cancer, to thrive and grow. While extensive research has revealed how gene and protein expression networks contribute to these traits, the relationships between tumor cell states, spatial organization of immune and stromal cells in the tumor microenvironment, and tumor genetics, remains poorly understood. To address this, we constructed a cohort of primary tumor tissues from 117 pancreatic ductal adenocarcinoma (PDAC) patients. Using Multiplexed Ion-Beam Imaging (MIBI) with a 40-antibody panel, we mapped the spatial organization of major cell types and their phenotypes. We also conducted targeted genomic and bulk proteomic analyses to explore how spatial organization patterns align with tumor genetics and proteome profiles. We identified phenotypic subtypes of tumor epithelium that shape distinct microenvironmental niches: (i) Glycolytic tumors expressing proteins such as GLUT1, LDHA, MCT1 and HIF1α reside in hypoxic niches. They are encased by dense fibrotic stroma and lacking vasculature and immune cell infiltration. (ii) Tumor and metaplastic epithelium that exhibit an oxidative proteome profile, are highly enriched for ECAD and PD-L1 expression, and are situated near pancreatic epithelium and vasculature, with significant immune infiltration. (iii) Myeloid-interacting tumors expressing CXCL5, heavily infiltrated with neutrophils and characterized by low-collagen and high FAP positive fibroblasts. Notably, neutrophils accumulate within the lumen of tumor ducts where they exhibit increased apoptosis, DNA damage, and netosis. These highly neutrophil-infiltrated tumors, associate with poor disease outcome. The genetic landscape of PDAC influences these spatial and phenotypic patterns. Patients with TGFβ/SMAD pathway mutations, demonstrate an increased proportion of basal tumor cells accompanied by reduced lymphocytic infiltration. Similarly, distinct TP53 variants shape immune-cell infiltration: patients harboring missense mutations show increased lymphocytic infiltration compared to those with truncating mutations. These findings underscore the strong link between tumor genetics, phenotypic heterogeneity and cellular architecture of the PDAC microenvironment, offering new insights into collective tumor behavior.



 Ofer Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, Inti Zlobec, Tamar Geiger, Leeat Keren. Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B123.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb5d6211d2e33680561c9558492cf85ff16b33" target='_blank'>
              Abstract B123: Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization
              </a>
            </td>
          <td>
            O. Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, I. Zlobec, Tamar Geiger, Leeat Keren
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/235c8c18e964369ee970e129fd3fe5bd3b1a1aa6" target='_blank'>
              In silico reconstruction of primary and metastatic tumor architecture using GIS-augmented spatial transcriptomics
              </a>
            </td>
          <td>
            Jin Young Yoo, Sabrina Akter, Qianying Zuo, Amir Kazemi, Wenjun Wu, Mahima Goel, Audrey Lam, Debapriya Dutta, Betsy Barnick, M. Pekmezci, Maria Grosse Perdekamp, Aiman Soliman, Zeynep Madak-Erdogan
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d523903df00c8d74634f25362c4c022807a0ad86" target='_blank'>
              Spatial proteomics for investigating solid tumor resistance mechanisms
              </a>
            </td>
          <td>
            X. M. M. Zhou, A. D'Amiano, Charles Lu, Vrinda Madan, Sara Khoshniyati, J. Kollings, Noah E Sunshine, S. Surwase, Joel C. Sunshine
          </td>
          <td>2025-10-08</td>
          <td>Cancer and Metastasis Reviews</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Integration of scRNA-seq data from millions of cells revealed a high diversity of cell types in the healthy and diseased human lung. In a large and complex organ, constantly exposed to external agents, it is crucial to understand the influence of lung tissue topography or external factors on gene expression variability within cell types. Here, we apply three spatial transcriptomics approaches, to: (i) localize the majority of lung cell types, including rare epithelial cells within the tissue topography, (ii) describe consistent anatomical and regional gene expression variability within and across cell types, and (iii) reveal distinct cellular neighborhoods in specific anatomical regions and examine gene expression variations in them. We thus provide a spatially resolved tissue reference atlas in three representative regions of the healthy human lung. We further demonstrate its utility by defining previously unknown imbalances of epithelial cell type compositions in chronic obstructive pulmonary disease lungs. Our topographic atlas enables a precise description of characteristic regional cellular responses upon experimental perturbations or during disease progression. Using multiplexed spatially resolved transcriptomic approaches, the authors generate a topographic atlas of the healthy adult lung with location-related gene expression variability or neighbourhood changes as exemplified by COPD patient sample analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe80d855be30beaf44c2810f98505d59172a3d0" target='_blank'>
              Spatial single-cell atlas reveals regional variations in healthy and diseased human lung
              </a>
            </td>
          <td>
            A. Firsova, Sergio Marco Salas, L. B. Kuemmerle, X. Abalo, A. Sountoulidis, L. Larsson, Krishnaa T. Mahbubani, Jonas Theelke, Žaneta Andrusivová, Leire Alonso Galicia, A. Liontos, Tamás Balassa, Ferenc Kovács, Peter Horvath, Yuexin Chen, Janine Gote-Schniering, M. Stoleriu, Jürgen Behr, Kerstin B. Meyer, W. Timens, Herbert B. Schiller, Malte D. Luecken, F. Theis, J. Lundeberg, Mats Nilsson, M. Nawijn, Christos Samakovlis
          </td>
          <td>2025-11-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Abstract The heterogeneous nature of cell populations in human tumors is a major contributor to tumor evolution, including and perhaps most importantly in response to treatment. Here, we review current knowledge on tumor heterogeneity and cell state plasticity in small cell lung cancer (SCLC), a fast growing and highly metastatic form of lung cancer which develops rapid resistance to therapy. There is a pressing need to expand treatment options for patients with SCLC, which requires a better understanding of the mechanisms by which this disease is able to rapidly grow and evolve in response to therapy. Our current understanding points to epigenetic rather than genetic factors in defining major aspects of inter- and intra-tumoral heterogeneity in SCLC. SCLC is overall considered to be a neuroendocrine (NE) cancer type but SCLC tumors harbor a wide diversity of cancer cell states, including both NE and non-neuroendocrine (non-NE) states, defined by their mutually exclusive expression of a set of transcription factors such as ASCL1, NEUROD1, and POU2F3. The immune microenvironments of SCLC tumors also contain a great deal of heterogeneity. Here, we discuss the different SCLC cell states associated with their defining transcription factors, as well as the epigenetic mechanisms regulating the ability of SCLC cells to switch from one state to another. We further discuss how the composition of SCLC tumors and the surrounding immune cells may affect the response to chemotherapy and immunotherapy. Being able to control plasticity and heterogeneity in SCLC may in the future offer unique opportunities to improve treatment efficacy in this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f9c75cb099eaa87426731186cf92ba20e24f46a" target='_blank'>
              Tumor Heterogeneity and Plasticity in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Gina Duronio, Julien Sage
          </td>
          <td>2025-10-01</td>
          <td>Lung Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9ca9b266d6e3d48dee0fe7d9f0b768bc883a83" target='_blank'>
              Functional interrogation uncovers a critical role for a high-plasticity cell state in lung adenocarcinoma
              </a>
            </td>
          <td>
            J. Chan, Chun-Hao Pan, Jonathan Rub, Klavdija Krause, Emma Brown, Gary Guzman, Zeda Zhang, Hannah C Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The spatial organization and composition of the tumor-immune microenvironment (TME) play a critical role in shaping the progression of many solid cancers, but the organization of the TME in primary prostate cancer (PCa) remains poorly characterized. We therefore profiled the abundance and spatial distributions of major cell types involved in adaptive immunity in 29 radical prostatectomy specimens stratified into high (HGG; n=14) and low Gleason-grade (LGG; n=15). Compared to LGG, HGG PCa exhibited significantly greater B and T cell infiltration with many immune cells organized into clusters, some of which resembled tertiary lymphoid structures (TLSs). In HGG tumors, these clusters were dense, symmetric, rich in PD-1+ T cells, and frequently proximate to the tumor compartment. LGG clusters were less well organized, and T cell depleted. Thus, a subset of high-grade PCa harbor organized immune clusters that may play a role in tumor control and contain therapeutically targetable T and B cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e5e762dad0168906d48c0967522a47db6d0ac" target='_blank'>
              Spatially organized lymphocytic microenvironments in high grade primary prostate tumors
              </a>
            </td>
          <td>
            Ali Amiryousefi, Jeremiah Wala, Jia-Ren Lin, Brian Labadie, A. Atmakuri, Z. Maliga, Eamon Toye, K. Chaudagar, Madeleine S. Torcasso, S. Coy, G. Fanelli, Brigette Kobs, F. Socciarelli, Andréanne Gagné, Eliezer Van Allen, Akash Patnaik, P. Sorger
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Spontaneously transformed mouse ovarian surface epithelial cell line ID8 generates a syngeneic mouse model, recapitulating the primary and metastatic stages of the malignancy hence, frequently used in pre-clinical evaluation. Due to inherent phenotypic plasticity, primary and metastatic tumors show contrasting phenotypic features dictated by the tissue niche. To address the tumor heterogeneity within the model we stratified it into a specific molecular subtype, which would provide an accurate setting for interpretation of experimental data. Stratification was performed on ID8 cell line and ID8 generated xenografts based on immunostaining, live cell imaging, histopathology, transcriptomics and proteomics. Tumor microenvironment (TME) investigation for each xenograft was done through flow cytometry analysis. ID8 cells present epithelial/mesenchymal (E/M) phenotype. Phenotypic plasticity retention in them causes class switching of ID8 cells in response to niche alteration, from E/M (in culture) to intermediate epithelial (iE) in orthotopic tumors while intermediate mesenchymal (iM) in subcutaneous tumors. Metastasis of ID8 primary tumor resulted into differentiated mesenchymal phenotype in ascites while differentiated epithelial phenotype in secondary intestinal tumors. Immune profiling of primary and secondary xenografts revealed differences in immune repertoire of tumors, highlighting subcutaneous tumor model to be inappropriate as compared to orthotopic model. Local tissue niche modulates the phenotype of tumors generated, transitioning through intermediate states towards a final commitment to either an epithelial or mesenchymal state that recapitulates the reported class switching in patient tumors. During disease progression, such heterogeneity of tumor cells at different niche(s) and their cross-talks with discrete TME at each site, will pose challenges for effective therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20023e2da4c4317c3adb1e410526c772109c27da" target='_blank'>
              ID8 cells manifest phenotypic plasticity and molecular heterogeneity of high-grade serous ovarian cancer in response to the local tissue niche
              </a>
            </td>
          <td>
            Ritika Gupta, S. Shravan, Swapna Balaji Agwane, R. M. Bankar, S. Bapat
          </td>
          <td>2025-10-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has emerged as an advanced biological technology capable of resolving the complexity of cancer landscapes at single-cell resolution. Spatial transcriptomics(ST), as an innovative complementary approach, effectively compensates for the lack of spatial information inherent in scRNA-seq data. This review explores the rapidly evolving integration of scRNA-seq and ST and their transformative role in deciphering the tumor microenvironment (TME). We highlight how these technologies jointly uncover cellular heterogeneity, stromal-immune interactions, and spatial niches driving tumor progression and therapy resistance. Moving beyond previous reviews, we emphasize emerging computational strategies for data integration—including deconvolution and mapping approaches—and evaluate their applications in characterizing immune evasion, fibroblast diversity, and cell-cell communication networks. Ultimately, this review provides a forward-looking perspective on how spatial multi-omics are poised to advance precision oncology through spatially-informed biomarkers and diagnostic tools. We conclude that the full clinical potential of these technologies relies on closing the gap between analytical innovation and robust clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84801b1d17cdb40a9473ba25f6bac709b5c4c887" target='_blank'>
              Single-cell and spatial transcriptomics integration: new frontiers in tumor microenvironment and cellular communication
              </a>
            </td>
          <td>
            Wenxin Shi, Zhiqiang Zhang, Xiaotong Xu, Yanpeng Tian, Li Feng, Xianghua Huang, Yanfang Du, Zhongkang Li
          </td>
          <td>2025-10-02</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Quantitative investigation of how the spatial organization of cells within the tumor microenvironment associates with disease progression, patient outcomes, and that cell’s phenotypic state remains a key challenge in cancer biology. High-dimensional multiplexed imaging offers an opportunity to explore these relationships at single-cell resolution. Methods We developed a computational pipeline to quantify and analyze the neighborhood profiles of individual cells in multiplexed immunofluorescence images. The pipeline characterizes spatial co-localization patterns within the tumor microenvironment and applies interpretable supervised machine learning models, specifically orthogonal partial least squares analysis (OPLS), to identify spatial relationships predictive of cell states and clinical phenotypes. Results We applied this framework to a previously published non-small cell lung cancer (NSCLC) cohort across four applications. At the cellular level, we identified neighborhood features associated with lymphocyte activation states. At the tumor-immune interface, we demonstrated that the immune cell composition surrounding major histocompatibility complex class I-expressing (MHC I+) tumor cells could distinguish adenocarcinoma from squamous cell carcinoma. At the patient level, spatial features predicted tumor grade. Discussion By integrating cell-segmented imaging data with interpretable modeling, our pipeline reveals key spatial determinants of tumor biology. These findings generate testable mechanistic hypotheses about intercellular interactions and support the development of spatially informed prognostic and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a735226da4130f5ff72da463b9cc63eeddebebdb" target='_blank'>
              A flexible systems analysis pipeline for elucidating spatial relationships in the tumor microenvironment linked with cellular phenotypes and patient-level features
              </a>
            </td>
          <td>
            Gabriel F. Hanson, Kate A. Goundry, Remziye E. Wessel, Michael G Brown, Timothy N. J. Bullock, Sepideh Dolatshahi
          </td>
          <td>2025-09-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes TP53 and RB1. However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yueli Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Risk stratification remains a critical challenge in non-small cell lung cancer patients for optimal therapy selection. In this study, we develop an artificial intelligence-powered spatial cellomics approach that combines histology, multiplex immunofluorescence imaging and multimodal machine learning to characterize the complex cellular relationships of 43 cell phenotypes in the tumor microenvironment in a real-world retrospective cohort of 1168 non-small cell lung cancer patients from two large German cancer centers. The model identifies cell niches associated with survival and achieves a 14% and 47% improvement in risk stratification in the two main non-small cell lung cancer subtypes, lung adenocarcinoma and squamous cell carcinoma, respectively, combining niche patterns with conventional cancer staging. Our results show that complex immune cell niche patterns identify potentially undertreated high-risk patients qualifying for adjuvant therapy. Our approach highlights the potential of artificial intelligence powered multiplex imaging analyses to better understand the contribution of the tumor microenvironment to cancer progression and to improve risk stratification and treatment selection in non-small cell lung cancer. Risk stratification in non-small cell lung cancer (NSCLC) remains challenging. By combining multiplex immunofluorescence, H&E histology, and AI, the study identifies spatial “cell-niche” patterns that enhance survival prediction beyond UICC8 staging. These patterns reclassify many stage I patients as high risk, revealing potentially undertreated cases and establishing spatial tumor microenvironment features as clinically actionable biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53bd1a5c6021c6b38d3074a4cf442e836ada177" target='_blank'>
              AI-powered spatial cell phenomics enhances risk stratification in non-small cell lung cancer
              </a>
            </td>
          <td>
            Simon Schallenberg, Gabriel Dernbach, Sharon Ruane, Philipp Jurmeister, Cornelius Böhm, Kai Standvoss, Sandip Ghosh, M. Frentsch, M. Dragomir, Philipp Keyl, Corinna Friedrich, Il-Kang Na, Sabine Merkelbach-Bruse, A. Quaas, N. Frost, Kyrill Boschung, Winfried Randerath, G. Schlachtenberger, Matthias Heldwein, Ulrich Keilholz, Khosro Hekmat, Jens Rückert, R. Buettner, Angela Vasaturo, David Horst, Lukas Ruff, M. Alber, Klaus-Robert Müller, Frederick Klauschen
          </td>
          <td>2025-11-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive disease with a dense fibrotic stroma and is often accompanied by chronic inflammation. Peritumoral inflammation is typically viewed as a reaction to nearby tumor growth. Here, we report that the inflamed pancreatic lobules are frequently invaded by tumor cells, forming a distinct, non-fibrotic tumor niche. Using a semi-supervised machine learning approach for annotations of clinical samples and multiplex protein profiling, we show that tumor cells at the invasion front are closely associated with acinar cells undergoing damage-induced changes, and with activated fibroblasts expressing markers of injury. The invaded lobules are linked to classical tumor phenotypes, in contrast to fibrotic areas where tumor cells display a more basal profile, highlighting microenvironment-dependent tumor subtype differences. In female mice, lobular invasion similarly aligns with the classical tumor phenotype. Together, our data reveal that pancreatic tumors colonize injured lobules, creating a unique niche that shapes tumor characteristics and contributes to disease biology. Peritumoural pancreatitis and lobular injury are commonly seen in pancreatic cancer. Here, the authors reveal interactions between invading tumour cells and inflammation-related lobular cells, linked to switching from a basal to classical tumor cell phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33fdbbbab057fd0e654aba5735a7c83d83362e84" target='_blank'>
              An injury-associated lobular microniche is associated with the classical tumor cell phenotype in pancreatic cancer
              </a>
            </td>
          <td>
            Sara Söderqvist, Annika Viljamaa, Natalie Geyer, Anna-Lena Keller, Kseniya Ruksha, Carina Strell, Neda Hekmati, Alexandra Niculae, J. Engstrand, E. Sparrelid, Caroline Salmén, Tânia D. F. Costa, Miao Zhao, Staffan Strömblad, Argyro Zacharouli, P. Ghorbani, Sara Harrizi, Yousra Hamidi, Olga Khorosjutina, Stefina Milanova, Bernhard Schmierer, Béla Bozóky, C. Fernández Moro, Marco Gerling
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c74617e8894674076d6153fb81e22ba9cb23b96" target='_blank'>
              Comparison of Human Melanoma Single Cell Profiles to Evolutionary Medicine Model Xiphophorus Provides Insights in Disease Control
              </a>
            </td>
          <td>
            Yanting Xing, Rachel Carrol, X. Maggs, Wes C. Warren, Manfred Schartl, Yuan Lu
          </td>
          <td>2025-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is a molecularly diverse form of lung cancer characterized by distinct oncogenic driver mutations that influence both tumour biology and clinical outcomes. Understanding the interplay between these oncogenic drivers and the tumour microenvironment (TME) is crucial for improving therapeutic strategies and patient management. Here, we investigate the impact of driver mutations on the composition and spatial architecture of the TME in LUAD. Using imaging mass cytometry (IMC), we analyse tumour samples from 157 LUAD patients, integrating genomic and clinical data to link specific mutations with tumour characteristics. Unique patterns are associated with mutated KRAS and EGFR tumours with TP53 co-mutations, suggesting these co-mutations reshape the TME and promote resistance to tyrosine kinase inhibitors (TKIs). Overall, our findings highlight the complex interplay between oncogenic driver mutations and the TME in LUAD, underscoring the importance of integrating genomic and cellular data to understand the underlying tumour behaviour and prognosis. Oncogenic driver mutations can influence the tumour biology of lung adenocarcinomas. Here, authors analyse 157 LUAD samples using imaging mass cytometry to reveal how these mutations impact the tumour microenvironment and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05eac26faa3f5fa852d857cbc84d4c9c3228ae87" target='_blank'>
              Oncogenic driver mutations underlie the spatial tumour immune landscape of non-small cell lung cancer
              </a>
            </td>
          <td>
            Saskia Hartner, H. Abolfathi, M. Rezanejad, Bridget Liu, E. Karimi, Dakota Rogers, Mark Sorin, Samuel Doré, Lysanne Desharnais, M. Orain, William Enlow, A. Gagné, Yuhong Wei, Y. Bossé, D. Quail, Philippe Joubert, L. Walsh
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="


 Adult gliomas are a diverse class of tumors with substantial molecular and pathological variation. While prior single-cell omics studies have characterized intratumoral heterogeneity, they often examined tumors in isolation, thus overlooking the recurrence of tumor states across subtypes. Given their shared environment, understanding the conservation of tumor states across gliomas is essential.



 To address this, we analyzed 310 tumor cores from 284 patients spanning oligodendroglioma, low- and high-grade astrocytoma, glioblastoma, and ependymoma, using single-cell spatial transcriptomics with a 344-gene panel (10x Xenium). This approach yielded a dataset of 2.8 million cells, of which 84% were tumor and 16% represented 18 non-malignant cell types from the tumor microenvironment (TME). To further investigate local cellular interactions, we applied an extended spatial transcriptomics panel (5,096 genes) alongside multiplexed immunofluorescence (57 proteins) to a validation cohort of low-grade astrocytomas.



 Our analysis revealed tumor states recurring across gliomas, albeit in varying proportions, organized into neighborhoods that reflect local cell-cell interactions. For instance, proliferating tumor cells tended to co-localize with oligodendro- and neuro-progenitor-like tumor states, suggesting that these programs may give rise to proliferating cells. Spatial neighborhoods were primarily subtype-specific, indicating that tissue organization and not cell composition alone, underlies glioma heterogeneity. We next assessed the relationship between tumor and immune cells, revealing entity-specific immune microenvironments: astrocytoma and ependymoma exhibited distinct immune responses (T cell- versus myeloid-driven inflammation), whereas oligodendroglioma displayed a low inflammatory landscape. Notably, microglia were consistently observed in proximity to tumor cells across all glioma subtypes, highlighting their conserved role in modulating anti-tumor immunity. Methylation profiling stratified these tumors into 13 molecular subtypes, each associated with distinct neighborhoods. Glioblastoma subtypes RTK1, RTK2, and MES, in particular, displayed unique tissue architectures consistent with their subtype-specific methylation profiles.



 Collectively, these findings link microanatomical features to glioma heterogeneity by connecting tumor states to their microenvironments. Regional differences, such as those between the tumor core and infiltrating edge, shape cellular composition with implications to treatment response. By capturing such spatially dependent patterns, our work provides a framework for advancing the understanding of glioma biology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34fbb58728bc44ec20e86f909612c57947c86652" target='_blank'>
              KS03.6.A THE RECURRENT TISSUE STATES ACROSS ADULT GLIOMAS AS REVEALED BY SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            A. Mathioudaki, Z. Seferbekova, M. Ritter, D. Calafato, G. Rukhovich, F. Hinz, F. Ippen, E. Popova, N. Koeberer, P. Sant, S. Schinkewitsch, J. Malm, S. Dietrich, C. Herold-Mende, N. Etminan, A. Wick, S. Krieg, M. Platten, A. von Deimling, F. Sahm, A. Suwala, M. Gerstung
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="High-resolution spatial transcriptomics (ST) data provide valuable insights into the molecular dynamics underlying complex biological processes. However, their widespread application remains limited due to high costs and technical challenges. Here, we present PRTS (Pathology-driven Reconstruction of Transcriptomic States), a novel framework that predicts single-cell-resolution ST data directly from histological images. Our results demonstrated that PRTS generated transcriptomic profiles for about 60,000 analyzable cell tiles per tissue section, representing an approximately 27-fold increase in analytical units compared to conventional ST spots and remarkably enhancing spatial resolution. Notably, PRTS achieves accurate cell-level transcriptomic predictions using only hematoxylin-and-eosin-stained tissue images. This method transforms costly ST technologies into a practical and scalable tool, offering a cost-efficient solution for comprehensive ST profiling in hematoxylin-and-eosin-based disease research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdf408653ada92ca0b84be1f0e89d28a32eb782d" target='_blank'>
              PRTS: Predicting Single-Cell Spatial Transcriptomic Maps from Histological Images
              </a>
            </td>
          <td>
            Jingyi Wen, Lingxuan Zou, Jiying Liu, Xi Guo, Changkun Liu, Bensu Wang, Tongtong Deng, Chang Liu, Risheng Tang, Yanbin Yang, Yucheng Huang, Lijia Yang, Hui Wang, Zihao Li, Shengming Lin, Shipping Liu, Yuhu Zhang, Z. Hao, Haiyu Zhou, Han Huang, Fei Ling
          </td>
          <td>2025-10-06</td>
          <td>Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719b808c693c9788562c4b889cdeaf7fc68e251b" target='_blank'>
              Spatial transcriptomics of glioblastoma defines biologically and clinically significant reprogramming patterns across unique spatial microenvironments
              </a>
            </td>
          <td>
            V. Ravikumar, A. Maddox, Reva Kulkarni, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) is evolving as a pivotal technology in studying tumor biology and associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. We use serial 5 μm sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma samples in tissue microarrays to compare the performance of the ST platforms (CosMx, MERFISH, and Xenium (uni/multi-modal)) in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx, and hematoxylin and eosin staining data. In addition to an objective assessment of automatic cell segmentation and phenotyping, we perform a manual phenotyping evaluation to assess pathologically meaningful comparisons between ST platforms. Here, we show the intricate differences between the ST platforms, reveal the importance of parameters such as probe design in determining the data quality, and suggest reliable workflows for accurate spatial profiling and molecular discovery. Spatial cell distribution within a tissue microenvironment is a rapidly advancing field. Here, authors assess three commercially available single-cell resolution spatial transcriptomics approaches (CosMx, MERFISH, and Xenium) to inform which technology outperforms for immune profiling of solid tumors using patient samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7415db7d075b9d1e62e79a2741843a3cf68a8b2" target='_blank'>
              Comparison of imaging based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed paraffin-embedded tumor samples
              </a>
            </td>
          <td>
            Nejla Ozirmak Lermi, Max Molina Ayala, Sharia Hernandez, Wei Lu, Khaja B Khan, Alejandra Serrano, Idania Lubo, Leticia Hamana, K. Tomczak, Sean Barnes, J. Dou, Qingnan Liang, , Ahmed N. Alrawi, Claudio A. Arrechedera, Kimberly S. Ayers, Caludia A. Bedoya, Elizabeth Burton, Connie A. Chon, Randy Chu, S. Crosby, Jonathan Do, Cibelle FP Lima, Fu Szu-Chin, Andy Futreal, Ana L. Garcia, Celia Garica-Prieto, S. Gite, C. Gumbs, Kristin J. Hargraves, Meng He, Chacha Horombe, H. Ibarguen, Stacy Jackson, Jeena Jacob, Isha Khanduri, Walter K. Kinyua, M. Knafl, Wenhua Lang, L. Little, S. Mallampati, Mary GT Mendoza, F. Meric-Bernstam, M. M. Mohammad, Mario LM Piubelli, Sabitha Prabhakaran, K. R. Shaw, Xiaofei Song, Sandesh Subramanya, Baohua Sun, Shumaila Virani, Wan-Jan Wang, Scott E. Woodman, Mingchu Xu, Jianhua Zhang, Qingxiu C. Zhang, Shanyu Zhang, M. Gabriela Raso, Ximing Tang, Mei Jiang, B. Sanchéz-Espiridion, A. Weissferdt, J. Heymach, Jianjun Zhang, B. Sepesi, T. Cascone, Anne Tsao, Mehmet Altan, Reza J Mehran, Don L. Gibbons, Ignacio I. Wistuba, C. Haymaker, Ken Chen, L. S. Solis Soto
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 A major barrier in understanding adult diffuse gliomas lies in the inability to track tumor evolution in real time. To address this, we employed a novel method known as continuous lineage tracing, which integrates CRISPR/Cas9-based expressed DNA barcoding with single-cell RNA sequencing, enabling a phylogenetic approach to studying tumor development. Patient-derived glioma-initiating cell lines were engineered with expressed barcodes targeted by CRISPR/Cas9 and implanted into mice to create intracranial xenografts. Tumors underwent single-cell RNA sequencing; expressed barcodes were used to infer clonal relationships, and transcriptomic profiles enabled cell state classification. Phylogenetic lineage trees were reconstructed using lineage inference algorithms to characterize cell fitness, expansion, and plasticity. Our analysis uncovered extensive intra-clonal cell state heterogeneity, indicating active phenotype switching prior to therapy. We identified consistent transcriptional programs associated with tumor engraftment and in vivo clonal advantage. Lineage tracing revealed gene expression signatures linked to key phenotypes: fitness, enriched for neural-mesenchymal and injury-response pathways; expansion, associated with RNA splicing; and plasticity, correlated with cell cycle and DNA repair programs. Glioma stem cells appeared to span a transcriptional continuum from undifferentiated, high-fitness states to more differentiated, low-fitness states, with high-fitness cells potentially representing transitional phenotypes. We then validated these findings in a cohort of 185 surgically resected gliomas with matched bulk RNA and proteomic data. Phylogenetic gene signatures differed markedly between IDH-mutant and IDH-wildtype tumors. When stratified by tumor type, both fitness and expansion signatures were significantly associated with overall survival in GBM (median 12.6 months, p=0.041) and oligodendrogliomas (median 66 months, p=0.027). This study demonstrates the utility of continuous lineage tracing to reconstruct tumor evolution and identify transcriptional programs linked to tumor growth and prognosis. Our approach provides a powerful framework for dissecting glioma biology and identifying potential therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d67c2559a40c369b3a39bc8fe0aad62196164a" target='_blank'>
              STEM-24. SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS, EXPANSION AND PLASTICITY AXES IN ADULT DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Ajisebutu, Xiaojun Fan, Chloe Gui, Jeff Liu, Vikas Patil, A. Landry, Y. Ellenbogen, L. Yefet, Phooja Persaud, F. Nassiri, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor thrombus (TT) worsens prognosis and complicates surgery in renal cell carcinoma (RCC), yet its formation mechanisms remain unclear. Here, we perform integrative single-cell and spatial transcriptomic analyses on 71 tissues and 48 sections from RCC patients with or without TT. The cellular and spatial atlas reveals distinct TT-associated tumor microenvironment remodeling characterized by the enrichment of FAP+ fibroblasts. These FAP+ fibroblasts are spatially contiguous to aggressive cancer cells and promote their malignant phenotypes in vitro. Their abundance inversely correlates with functional NK cells, suggesting roles in tumor invasion and immune evasion. Furthermore, single-cell multiomics analysis identifies tumor pericytes as a source of FAP+ fibroblasts and delineates transcription factor dynamics underlying pericyte-fibroblast transition. Finally, high levels of FAP+ fibroblasts are associated with poor prognosis and predict a weaker response to anti-VEGF-based therapy. In conclusion, our study highlights FAP+ fibroblasts as drivers of aggressive RCC with TT, suggesting potential therapeutic targets. Tumour thrombus (TT) complicates renal cell carcinoma (RCC) treatment. Here, authors conduct an integrative single-cell RNA sequencing and spatial transcriptomic analyses on RCC patients with and without TT and identify FAP+ fibroblasts as drivers for TT formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5472524a744e590345b74e85358cb939dc059d29" target='_blank'>
              FAP+ fibroblasts orchestrate tumor microenvironment remodeling in renal cell carcinoma with tumor thrombus
              </a>
            </td>
          <td>
            Jiacheng Ma, Yan Huang, Jie Chen, Yang Li, Rongyan Yao, Xiubin Li, Qiyang Liang, Xinran Chen, Cheng Peng, Kan Liu, Yuanjun Zhai, Xu Zhang, Xin Ma, Xiaowen Wang, Qingbo Huang, Fuchu He
          </td>
          <td>2025-10-23</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c036e69ca133347b296f15467d94c4fe32de6f8f" target='_blank'>
              A Tonic Signaling Code Predicts CAR-T Cell Efficacy in Diffuse Midline Glioma
              </a>
            </td>
          <td>
            Emily B. Deng, Xiaowen Zhong, Dazhuan E Xin, Upendra K. Soni, Wenkun Ma, Xiao Huang, Mingjun Cai, Po-Yu Liang, Jun Bai, Qian Qin, Shreya Mishra, Ming Hu, Arman E. Bayat, Jiajie Diao, Mei Xin, Natasha Pillay-Smiley, T. Hummel, Charles B. Stevenson, Jessica B Foster, Peter de Blank, Scott Raskin, C. Koschmann, Joseide A. Cancelas, Yi Zheng, Q. R. Lu
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd7292fed094fd219af6750590d28db3dc5a363b" target='_blank'>
              Spatial organization of the tumor-immune microenvironment in ER-positive breast cancer: remodeling during treatment and associations with clinical response
              </a>
            </td>
          <td>
            Maria Aanesland Dahle, Jan-Lukas Førde, E. V. Egeland, A. Creason, Cameron Watson, Ø. Garred, L. Prasmickaite, G. Mælandsmo, Gordon B. Mills, O. Engebraaten, M. H. Haugen
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The immune system is deeply shaped by its anatomical context, with spatial organization emerging as a fundamental principle of immune regulation. Recent advances in spatial omics technologies—encompassing transcriptomics, proteomics, metabolomics, lipidomics, and phosphoproteomics—have revolutionized our ability to study immune processes within intact tissue environments. By preserving spatial coordinates while capturing high‐dimensional molecular data, these technologies offer unprecedented insight into how immune cell states and functions are governed by local cues and tissue architecture. In this review, we provide an overview of the major spatial omics platforms, emphasizing methodologies that have gained traction within the immunology community and in our own research. We then illustrate how these tools have begun to elucidate the logic of immune compartmentalization across anatomically complex tissues. While not exhaustive, we highlight selected examples from the intestine, secondary lymphoid organs, and liver to show how spatial omics has uncovered region‐specific immune programs, microenvironmental niches, and context‐dependent signaling pathways. Together, these studies demonstrate how spatial omics technologies are redefining immunological inquiry—shifting the focus from isolated cell types to their spatially embedded roles in tissue physiology and pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e33ddb8a880f7c8c895cc2a4a4ca66e664dd1a05" target='_blank'>
              From Histology to High‐Resolution Mapping: The Rise of Spatial Omics in Immunology
              </a>
            </td>
          <td>
            Marco De Giovanni, D. Inverso, Matteo Iannacone
          </td>
          <td>2025-10-01</td>
          <td>European Journal of Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98126a0a8ef060e1442a8189ca2cc50b5b3d5b2e" target='_blank'>
              Integrated cell atlas and tumoroids chart pancreatic cancer therapeutic targets
              </a>
            </td>
          <td>
            Quan Xu, Ryo Okuda, Bruno Gjeta, Christoph Harmel, Marina Signer, Matilde Lucioli, Małgorzata Santel, M. Seimiya, Cinzia Esposito, Karolina Guja-Jarosz, Ashley Maynard, S. Morinaga, Y. Miyagi, Tomoyuki Yamaguchi, Y. Ueno, Salvatore Piscuoglio, Daniel J. Müller, Hideki Taniguichi, Barbara Treutlein, J. G. Camp
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a highly heterogeneous cancer with complex tumor–immune interactions. This heterogeneity, particularly in tumor and immune cells, complicates treatment and prognostic evaluation. Although recent studies have revealed distinct tumor cell states and immune dysfunction in HCC, the molecular basis underlying tumor aggressiveness remains poorly understood. A deeper understanding of the molecular and functional diversity of both tumor and immune cell populations, especially the identification of stem-like tumor subpopulations and immunosuppressive mechanisms, along with the development of robust prognostic biomarkers, is essential for advancing precision oncology and improving clinical outcomes. Methods We integrated three publicly available single-cell RNA sequencing (scRNA-seq) datasets from GEO to delineate the cellular architecture of the HCC tumor microenvironment. Unsupervised clustering and dimensionality reduction were employed to identify major cell types and tumor subpopulations. Functional annotation was performed using canonical markers, Monocle, CytoTRACE, and AUCell scoring. H2AFZ was identified as a candidate oncogene and validated through in vitro knockdown experiments. The interaction between T cell subsets and tumor subpopulations were further characterized. A prognostic risk model was constructed using LASSO regression. Results Six major cell types were identified in HCC TME. Tumor cells were subdivided into three distinct clusters: Tumor_C0, Tumor_C1 and Tumor_C2. Tumor_C2 showed the highest stemness, pro-metastatic activity and immunogenic cell death signatures. H2AFZ was highly expressed in Tumor_C2 and associated with poor prognosis. The knockdown of H2AFZ reduced H2A.Z protein levels, inhibited proliferation, invasion, and induced apoptosis. T cell analysis revealed five subpopulations. It was found that Tumor_C2 interacts with the proliferative and exhausted T cell subpopulations, suggesting a potential functional relationship between them. The prognostic model based on tumor_C2 transcriptomic features effectively stratified patient survival across multiple cohorts, with robust AUCs and Kaplan-Meier survival distinctions. Conclusion We identified a proliferative, stem-like tumor cell subpopulation (Tumor_C2) in HCC characterized by high H2AFZ expression, which drives tumor aggressiveness. T cell analysis revealed significant interactions with Tumor_C2. Moreover, a prognostic model based on Tumor_C2 features effectively stratified patient survival across multiple cohorts. Together, these findings highlight potential therapeutic targets for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7024320dd03972a4da66c548f19e56393e35ae99" target='_blank'>
              Single-cell transcriptomics identifies an H2AFZ-driven proliferative tumor subpopulation associated with poor prognosis in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jihui Huo, Tao Yang, Kai Lei, Zeyao Wang, Zebin Chen, Qi Zhou
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696e778534089692f9510070ab5b7ccf3c2a11f2" target='_blank'>
              Single-cell spatial mapping reveals dynamic bone marrow microarchitectural alterations and enhances clinical diagnostics in MDS
              </a>
            </td>
          <td>
            Ryan Nachman, Aleksandra Kopacz, Caitlin Unkenholz, Jian Chai, Arvin Ruiz, Itzel Valencia, Jeanne Jiang, F. Socciarelli, Jiwoon Park, Christopher E. Mason, Ling Zhang, D. Sallman, Gail J. Roboz, Pinkal M. Desai, Justin Kaner, Joshua A Fein, Monica L Guzman, Neal Lindeman, Amy Chadburn, Madhu M Ouseph, Paul Simonson, Julia Geyer, Giorgio Inghirami, Shahin Rafii, David Redmond, Sanjay Patel
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fa5a90466cca365ac23cb5111624676a9f4bea9" target='_blank'>
              Deciphering Cellular Ecosystems Driving Tumor Progression and Immune Escape from Spatial Transcriptomics and Single-Cell with COMPOTES
              </a>
            </td>
          <td>
            L. Herpin, Anaïs Chossegros, Roberta Codato, Josep Monserrat Sanchez, Jean El Khoury, Simon Grouard, V. Ducret, Alex Cornish, Baptiste Gross, Elodie Pronier, C. Hoffmann, Alberto Romagnoni, Eric Y. Durand, Almudena Espin Perez, Quentin Bayard
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64783d2fd1c1cb64aa22b77e49df40179825bbfc" target='_blank'>
              Inferring cellular communication through mapping cells in space using Tangram 2
              </a>
            </td>
          <td>
            Hejin Huang, Alma Andersson, Sunny Z. Wu, Gabriele Scalia, K. H. Hoi, Jenna Collier, Graham Heimberg, Hector Corrada-Bravo, Shreya Gaddam, Jan-Christian Huetter, Sören Müller, Shannon J. Turley, Dave Richmond, Aicha Bentaieb, Tommaso Biancalani
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Meningiomas represent the most common primary intracranial tumors, accounting for approximately one-third of adult CNS tumors. Despite their frequency, model systems that accurately replicate the complex tumor-microenvironment interactions across meningioma grades remain underdeveloped, limiting translational progress. This study aimed to establish and characterize human meningioma tumor models to interrogate grade-specific cellular behaviors and microenvironmental dynamics



 Patient-derived meningioma cell lines were established through mechanical and enzymatic dissociation of surgically resected tumors, followed by expansion in specialized culture media. GFP transfection was performed to enable live-cell tracking. For ex vivo modeling, human neocortical slices — obtained from access tissue from non-meningioma surgeries, either 2 cm distal to the glioma core or from epilepsy surgeries under neuronavigation guidance — were injected with GFP-labeled meningioma cells. Tumor progression, cluster formation, and tissue integration were assessed over six days using longitudinal live imaging and immunofluorescence staining. Single-cell RNA sequencing (scRNA-seq) was employed to resolve cellular heterogeneity and tumor-microenvironmental interactions. Machine learning-based image segmentation combined with manual curation enabled precise quantification of tumor cell distribution and interaction patterns within the host tissue.



 Grade 3 meningioma cells demonstrated significantly enhanced cluster formation, tissue invasion, and microenvironmental remodeling compared to lower-grade counterparts. Ex vivo models recapitulated patient tumor behaviors, with scRNA-seq revealing upregulation of genes associated with mesenchymal transition, extracellular matrix remodeling, and immune modulation in Grade 3 tumors. Microenvironmental analysis indicated that higher-grade tumors established more extensive cell-cell interaction networks and induced greater astrocytic and microglial activation relative to Grade 1 and 2 models. Furthermore, immune profiling revealed a more immunosuppressive environment, with myeloid-derived suppressor cells, reducing immune surveillance and facilitating tumor evasion. Additionally, there was increased activation of astrocytes and microglia, contributing to a supportive tumor microenvironment.



 Our patient-derived meningioma models reveal grade-specific differences in tumor behavior and TME interactions. Grade 3 meningiomas show aggressive clustering and microenvironmental remodeling, offering insights into malignant progression. These models provide a high-fidelity platform for studying tumor biology and preclinical evaluation of therapies targeting tumor-stroma interactions
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b180fe0ce6bc7373d74452191faf8369d3bd9e2c" target='_blank'>
              P13.17.B HUMAN MENINGIOMA TUMOR MODELS REVEAL GRADE-SPECIFIC MICROENVIRONMENTAL INTERACTIONS
              </a>
            </td>
          <td>
            A. Mohammed, K. Joseph, D. Depro, G. Nishita, V. Jariwala, A. Boecke, M. Al Shhab, I. Vasilikos, J. M. Nakagawa, T. Juratli, R. Roelz, M. Shah, J. Beck, J. Grauvogel, V. Ravi
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is highly heterogeneous, with five-year survival rates dropping from $\sim$90% in localized disease to $\sim$15% with distant metastases. Disease progression is shaped not only by tumor-intrinsic alterations but also by the reorganization of the tumor microenvironment (TME). Metabolic, compositional, and spatial changes contribute to this progression, but considered individually they lack context and often fail as therapeutic targets. Understanding their coordination could reveal processes to alter the disease course. Here, we combined multiplexed ion beam imaging (MIBI) with machine learning to profile metabolic, functional and spatial states of 522 colorectal lesions with single-cell resolution. We observed recurrent stage-specific remodeling marked by a lymphoid-to-myeloid shift, stromal-cancer cooperation, and malignant metabolic shifts. Spatial organization of epithelial, stromal, and immune compartments provided stronger stratification of disease stage than tumor-intrinsic changes or bulk immune infiltration alone. To systematically model these coordinated changes, we condensed multimodal features into 10 latent factors of TME organization. These factors tracked disease progression, were conserved across cohorts, and revealed frequent multicellular metabolic niches and distinct, non-exclusive TME trajectories. Our framework MuVIcell exposes the elements that together drive CRC progression by grouping co-occurring changes across cell types and feature classes into coordinated multicellular programs. This creates a rational basis to therapeutically target TME reorganization. Importantly, the framework is scalable and flexible, offering a resource for studying multicellular organization in other solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884e760c0de1e86aa10844e17508931cd572069b" target='_blank'>
              Robust multicellular programs dissect the complex tumor microenvironment and track disease progression in colorectal adenocarcinomas
              </a>
            </td>
          <td>
            Loan Vulliard, Teresa Glauner, Sven Truxa, Miray Cetin, Yu-Le Wu, R. Simon, Laura Behm, Jovan Tanevski, Julio Saez-Rodriguez, G. Sauter, Felix J. Hartmann
          </td>
          <td>2025-10-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0d7b578f97dbff41a41bb90de3b031320cfe41" target='_blank'>
              Unravelling drug resistant proteotypes through phenotype-resolved proteomics of single-cell derived colonies
              </a>
            </td>
          <td>
            Di Qin, Ridhima Das, S. Schallenberg, K. Riecken, I. Tinhofer, Fabian Coscia
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Stem-like gastric cancer (GC) is an aggressive molecular subtype marked by poor prognosis and limited response to immune checkpoint blockade (ICB). The spatial mechanisms driving this resistance remain unclear. We conducted spatially resolved single-cell transcriptomic profiling of diffuse-type GC tissues to uncover the spatial architecture and functional diversity of tumor and stromal populations. Cellular heterogeneity and region-specific signaling pathways were characterized using integrative bioinformatics analyses. We identified transcriptionally diverse, high-entropy cell populations predominantly localized in the deep tumor regions. These included unique endothelial and fibroblast subsets enriched for pro-tumorigenic and immune-regulatory signaling. A notable finding was the engagement of deep-region endothelial cells in VISFATIN (extracellular NAMPT) signaling through the ITGA5–ITGB1 integrin axis, associated with immune evasion and poor prognosis. This endothelial signaling program is distinct from and functionally independent of cancer-associated fibroblast (CAF)-mediated pathways. Elevated expression of the NAMPT–ITGA5–ITGB1 axis was observed in ICB non-responders and correlated with reduced overall survival. Our study delineates spatially defined cellular programs that contribute to immune escape in stem-like GC, highlighting a novel VISFATIN–integrin signaling axis as a potential biomarker and therapeutic target in immunotherapy-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dd6bcdbe10e93d5e57b978f2adeff6ec26c9091" target='_blank'>
              Spatially resolved endothelial signaling via nampt-itga5 drives immune evasion in stem-like gastric cancer
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jae-Ho Cheong, Eui Tae Kim
          </td>
          <td>2025-09-29</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Spatial context is a fundamental driver of intra-tumor heterogeneity in glioblastoma (GBM), shaping the molecular landscape, cellular composition, and therapeutic vulnerabilities. While single-cell profiling has illuminated the diversity of GBM cell states, a critical gap remains in understanding how local microenvironments orchestrate gene expression programs and cellular plasticity within intact tumors. To address this, we analyzed spatial transcriptomics data from 44 GBM specimens, comprising over 128,000 tissue spots, to systematically map the biological programs and cellular architecture of distinct tumor niches. Leveraging machine learning, we derived a unified batch-adjusted data embedding that captured the local tissue context of each spot across all slides, allowing the study of spatially continuous rewiring of molecular programs. This approach enabled us to identify ligand–receptor interactions and transcription factors (TF) with spatially localized activity across tissue slides. Using cross-correlation metrics, we identified multiple TF modules specifically active in certain environments. Notably, the perivascular region exhibited the most diverse transcriptional activity, with multiple TF modules associated with tumor growth, division, and migration processes. Additional modules were uniquely enriched at the leading edge and hypoxic regions, as well as at the tumor–necrosis interface. Importantly, we identified a unique module active at the tumor core–leading edge interface, with corresponding spots enriched for alternative splicing processes, suggesting this could be a core driver of the invasive tumor phenotype. Dynamic gene network inference along a continuous hypoxic gradient uncovered context-specific hub genes and TFs that facilitate tumor metabolic adaptation and cell state shifts. These findings underscore the immense spatial heterogeneity of GBM, characterized by context-specific molecular rewiring of tumor cells driven by complex cellular interactions. Our study not only reveals new vulnerabilities but also highlights the need for spatially targeted combination therapies to disrupt the adaptive spectrum of cancer cell states in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fddeac5d22fe2de68ff3fa47ac318949e6c8ee0" target='_blank'>
              TMIC-118. Identifying spatially-localized biological programs and molecular vulnerabilities in glioblastomas
              </a>
            </td>
          <td>
            V. Ravikumar, Reva Kulkarni, A. Maddox, Arushi Tripathy, Emre Kiziltug, Dah-Luen Huang, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75be2c8e54a33a7fe17a0e30ce626e667d1c6218" target='_blank'>
              Revealing Shared Tumor Microenvironment Dynamics Related to Microsatellite Instability Across Different Cancers Using Cellular Social Network Analysis
              </a>
            </td>
          <td>
            N. Zamanitajeddin, M. Jahanifar, M. Eastwood, G. Gunesli, M. J. Arends, N. Rajpoot
          </td>
          <td>2025-09-29</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03cc2aa2a287dcfe2c1cd1643687b026d41744b" target='_blank'>
              Single-cell and Spatial Omics Reveals Region-Specific Plasticity and Therapeutic Vulnerabilities in Metastatic High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Lai Man, Natalie Wu, Janet L. Oblinger, Dazhuan E Xin, Rohit Rao, Feng Zhang, Mike Adam, Oscar Lopez-Nunez, Sara Szabo, D. Allmen, Long-Sheng Chang, Brian D Weiss, Q. R. Lu
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Owing to the complexity of TME components and the heterogeneity of cancer cells, the relationship between the niches of TME and prognosis in breast ductal carcinoma remains unknown. The staged characteristics of corresponding cancer cell behaviors are unclear. Our study aims to reveal spatial structures and specific cellular information of TME and cancer cells subgroups during the progression from DCIS to IDC and lymph node metastasis. Single-cell sequencing, spatial transcriptomics, bulk RNA sequencing datasets were used to explore the changes in microenvironmental components and transcriptional programs of tumor cells during the progression of breast ductal carcinoma. Immunohistochemistry, multiplex immunofluorescence, flow cytometry cell cycle detection, invasion migration experiments, and WB imprinting were employed for validation. Analysis of TME cell type subsets revealed the accumulation of TEX, iTreg, and stress-phenotype TAM in the mammary gland in situ during the invasion process. Lymphatic metastases exhibited enrichment of nTregs and a more naïve-like CD8 T cell population. Spatial analysis and survival analysis showed that the spatial niches of CD4 TN and phagocytic-phenotype macrophages were associated with a favorable prognosis, and these niches were lost during disease progression. The proliferative subpopulation of breast ductal carcinoma was enriched in lymphatic metastatic tissues, expressing high levels of FAM111B and exhibiting intense TCA and oxidative phosphorylation metabolism. Silencing FAM111B led to cell cycle arrest, decreased invasion and migration abilities, and downregulation of core mediator genes for cuproptosis and disulfidptosis. The stage-specific microenvironmental characteristics of breast ductal carcinoma correspond to some extent to the behavior of tumor cells. During the progression of ductal carcinoma in breast tissue, the establishment of an immunosuppressive microenvironment occurs. The microenvironmental spectrum at lymph node metastases differs somewhat, corresponding to a more enriched turnover of cancer cell proliferation and death. Inhibitors of FAM111B and inducers of cuproptosis and disulfidptosis may serve as potential therapeutic targets for proliferative subgroups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfce9a8ff47f521350ed382190ba1004f756bb5c" target='_blank'>
              Spatiotemporal microenvironment landscape and malignant epithelial pattern transition in breast ductal carcinoma progression
              </a>
            </td>
          <td>
            Xifu Cheng, Wenjuan Zeng, Bingzhe Yin, Jiawei Gui, Hengbin Zhang, Zhenxing Lv, Simin Zhang, Yao Zhou
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Human pancreatic ductal adenocarcinomas (PDAC) are composed of sparse, heterogeneous tumour cell populations embedded within a dense, desmoplastic stroma and are highly resistant to existing therapies. Guided by underlying genomic aberrations, cancer cell states and phenotypes interact with their microenvironment to shape disease trajectory and therapeutic response. To quantify the cellular phenotypic heterogeneity of these complex tumour-microenvironment interactions we designed custom multiplexed histopathology imaging panels against single cell RNA sequencing-defined tumour, immune, and stromal cell compartments. Using imaging mass cytometry (IMC), we profiled the co-localization and organization of 83 cell types and their functional states in 221 resected pancreatic tumours. This identified extensive inter- and intra-patient heterogeneity including the presence of both classical and basal cell types in most patients and a gradient of pancreatic epithelial identity confirmed by scRNAseq. This single cell content defined an expanded spectrum of classifiable intermediate tumour subtypes between classical and basal which have specific associations to neighboring immune and stroma cellular content, ploidy, established broad transcriptional subtypes, heterogeneity, and patient outcome. Eight reproducible cancer microenvironments were quantified, empowering cross-tumour comparisons. Matched whole genome sequencing (WGS) identified aberrations associated with specific tumour phenotypes and microenvironments. These results support asynchronous tumour and microenvironment genotype-phenotype axes which subdivide basal to classical tumour polarization and categorize co-existing microenvironments from stiff, ECM-rich, immune suppressed to immune infiltrated regions alongside fibrovascularized CD105+ stroma. This is based upon spatially heterogeneous co-occurring microenvironment niches and correlated tumour phenotypes which are associated with KRAS or MYC amplification and CDKN2A or LATS2 deletion on one axis, mutation of epigenetic modifiers in intermediate and hybrid subtypes, and RNF43 deletion on the other axis. To deeply profile the signalling pathways and enriched biological processes characteristic of each tissue state, we performed deep proteomics measurements of each microenvironment and tumor phenotype through whole-slide IMC informed laser capture microdissection and mass spectrometry of 273 micro-regions from a subset of patients. Finally, to identify a robust, minimized set of outcome-relevant features, we used a multi-modal technology optimized machine learning model trained to predict overall survival. We compare the predictive potential of WGS and spatial proteomics and show that a combination of genomic and cellular content outperforms clinical features and platform specific models thereby demonstrating the synergistic benefit of integrated multi-modal data. Our findings classify the spatial organization of human PDAC to identify an intertwined tumour/microenvironment axis that is influenced by specific genomic aberrations.



 Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Beth Sunnucks, Jennifer Gorman, Chengxin Yu, Michael Geuenich, Sheng Ben Liang, Barbara Gruenwald, Ayelet Borgida, Cassandra Wong, Brendon Seale, Zhen Yuan Lin, Edward Chen, Golnaz Abazari, Miralem Mrkjonic, Julie Wilson, Kieran Campbell, Anne-Claude Gingras, Rob Grant, Grainne O'Kane, Faiyez Notta, Steve Gallinger, Hartland Jackson. Spatial proteomics extend tumour subtype and microenvironment classifications across genotype-phenotype axes of pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B128.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61e2ac6f788886b4d295a71549b94c0503bd2fb" target='_blank'>
              Abstract B128: Spatial proteomics extend tumour subtype and microenvironment classifications across genotype-phenotype axes of pancreatic cancer
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Beth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng Ben Liang, Barbara Gruenwald, A. Borgida, Cassandra J. Wong, Brendon Seale, Z. Lin, Edward L. Y. Chen, Golnaz Abazari, Miralem Mrkjonic, Julie M. Wilson, Kieran R. Campbell, A. Gingras, Robert C. Grant, G. O’Kane, Faiyez Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d4f1e5ea74f6620f6b4fbf1988331e79868e36" target='_blank'>
              Interpretable Deep Learning Reveals Biologically Relevant Spatial Gene Expression Patterns in Lung Tumors and their Microenvironment
              </a>
            </td>
          <td>
            Vibha R. Rao, Adrienne A Workman, Liang Lu, Xiaoying Liu, Shrey S. Sukhadia
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The skin is the largest human organ and a site of significant disease burden, yet its cellular and molecular organization across the body are largely undefined. Here, we construct a spatially-resolved single-cell atlas of 1.2 million cells from normal adult human skin to localize 45 cell types across 15 anatomic sites. We define principles of organ-wide cell composition, including axes of cell diversity and specialization, and distinguish site-enriched cell types. Each body site is comprised of 10 multicellular neighborhoods that define cell-cell communication. Notably, we identify a perivascular neighborhood enriched for immune-stromal crosstalk with features resembling a homeostatic immune niche similar to skin-associated lymphoid tissue. Finally, mapping these neighborhoods onto skin disease reveals pathogenic neighborhood disruptions, including pan-disease immune alterations in the perivascular neighborhood. We present a framework charting the skin’s multiscale spatial organization across a molecular to macroanatomic scale. This work advances our understanding of organ-wide skin cellular organization and communication, and its architectural disruption in disease. HIGHLIGHTS Human skin MERFISH spatial atlas of 1.2 million cells from 114 samples and 22 donors Cellular diversity varies along a central to peripheral body site spatial axis 10 multicellular neighborhoods define skin microanatomy and homeostatic interactions Human skin diseases feature spatial and transcriptional neighborhood remodeling">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc0e719289efff10e98b7b37485150f9c3fc314f" target='_blank'>
              A single-cell spatial transcriptomic census of human skin anatomy
              </a>
            </td>
          <td>
            Paula Restrepo, Alexis Wilder, Aubrey Houser, Angie Ramirez, M. Hren, Raman Gill, Abiha Kazmi, Larry Chen, D. Demircioglu, D. Hasson, A. Soto, Stephanie McQuillan, Edgar Gonzalez-Kozlova, R. Brody, Benjamin Ungar, Maria Kasper, Phillip Torina, Jesse M. Lewin, Sacha Gnjatic, Sai Ma, Andrew L. Ji
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c81035972b65c627d1b19e7302e769d3ddd0b086" target='_blank'>
              A Single-Cell Atlas of the Breast Cancer Microenvironment Identifies Subtype-Specific Tumor-Immune Landscapes and Vulnerabilities
              </a>
            </td>
          <td>
            Liron Zisman Schachter, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCLs) are a heterogeneous group of malignancies that can arise in lymph nodes or extranodal locations, including immune-privileged sites. Here, we applied highly multiplexed spatial transcriptomics and proteomics together with genomic profiling to characterize the immune microenvironment architecture of 78 DLBCL tumors. We define seven distinct cellular niches, each characterized by unique cellular compositions, spatial organizations and patterns of intercellular communication associated with niche-specific phenotypes of both T cells and tumor B cells. Among these, DLBCLs from immune-privileged sites showed abundant T cell infiltration into diffuse niches, where immune cells were intermixed with tumor B cells and bore transcriptional hallmarks of activation and effector function, suggesting that they may be primed for anti-tumor immunity. Spatial characterization of the DLBCL immune microenvironment, therefore, reveals cellular niches that foster divergent patterns of cell–cell interactions contributing to the phenotypic heterogeneity of both niche-resident tumor and immune cells. Analysis of the immune microenvironment of diffuse B cell lymphomas using spatial transcriptomics, proteomics and genomics highlights discrete cellular niches with divergent patterns of cell–cell communication that contribute to the phenotypic heterogeneity of both tumor and immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe40c75d4c6f7843c947dbae95bbc2601c4d486" target='_blank'>
              Multi-modal spatial characterization of tumor immune microenvironments identifies targetable inflammatory niches in diffuse large B cell lymphoma
              </a>
            </td>
          <td>
            Yibo Dai, A. Kizhakeyil, Dai Chihara, Xubin Li, Yunhe Liu, T. P. Sainz Zuniga, A. Wilson, Jared Henderson, Daniil Vibe, Arman Petrosyants, Connor Jacobson, Alexander Sarachakov, K. Nomie, K. Kryukov, Aleksander Bagaev, Ayushi Chauhan, J. Westin, Christopher R. Flowers, F. Vega, Linghua Wang, Michael R Green
          </td>
          <td>2025-10-21</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59ee0ed38702dcef1f68ff3c8ff2d5ef5e4fc717" target='_blank'>
              Longitudinal Imaging of the Premalignant Tumor Microenvironment Reveals Transient Myeloid States Predictive of Tumor Fate
              </a>
            </td>
          <td>
            Thomas D. Madsen, Desu Chen, Maria Hernandez, Sarah M. Hammoudeh, Marco Heydecker, Emily Chen, Salma Abu-Elnaj, Noemi Kedei, Weiye Wang, Roberto Weigert
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Understanding tumor heterogeneity at the resolution of individualized cell–cell communication networks (CCCNs) remains a major computational challenge in precision oncology. Existing inference methods largely rely on population-level correlation and thus fail to capture patient-specific signaling patterns across diverse cell types. To address this limitation, we developed an integrative computational framework combining the nested hierarchical Dirichlet process (nHDP) model for identifying hierarchically structured gene expression modules, with instance-specific Greedy Fast Causal Inference (iGFCI) for inferring individualized CCCNs (iCCCNs) in colorectal cancer (CRC). Applied to large-scale single-cell RNA-seq data from over 625,000 cells, our model successfully decomposed complex gene expression modules GEMs, potentially representing the cell lineage and cellular signaling states, and uncovered iCCCNs across detailed cell subtypes. We further used TCGA bulk RNA-seq data with survival data to validate the clinical relevance of these individualized gene expression module causal interactions, demonstrating their potential as robust prognostic signatures in CRC. Finally, we used principled causal inference methods to search for ligand-receptor pairs that mediate cell-cell communication. This framework enables mechanistic insights into immune evasion. Our computational method represents a significant advance toward realizing personalized oncology, enabling precise patient stratification and identification of actionable biomarkers for improved therapeutic targeting in cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4e906c5fe00a93c967af7c5aabe8332c82ec34" target='_blank'>
              Inferring Personalized Cell-Cell Communication Networks in Colorectal Cancer with Individualized Causal Discovery
              </a>
            </td>
          <td>
            Aodong Qiu, Binfeng Lu, Gregory F. Cooper, Xinghua Lu, Lujia Chen
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) remains a leading cause of cancer mortality, with liver metastasis being the principal determinant of poor prognosis, but the spatial mechanisms orchestrating metastatic niches remain elusive. Method To dissect the molecular and spatial dynamics of CRC progression, we constructed an integrative atlas using 35 single-cell RNA-seq datasets and spatial transcriptomics from primary tumors, liver metastases, and matched normal tissues. Malignant epithelial subpopulations were stratified via inferCNV and CytoTRACE analyses. Stromal-tumor interactions were dissected using CellChat and NicheNet, with functional validation through in vitro co-culture and immunohistochemistry. Result We identified a transcriptionally distinct epithelial subpopulation, termed high-malignancy CRC (High-M CRC), enriched in metastatic lesions and characterized by enhanced stemness, MYC-driven transcriptional activity, and glycolytic reprogramming. Stromal-tumor interaction analyses revealed that cancer-associated fibroblasts (CAFs), particularly matrix CAFs (mCAFs), promote malignant progression via the HGF-MET-MYC signaling axis. Spatial transcriptomic mapping confirmed the physical proximity and molecular co-localization of High-M CRC cells and mCAFs, along with enriched glycolysis and MYC expression at the cell-cell interface. In vitro functional validation demonstrated that CAF-derived HGF activates MET-MYC signaling in CRC cells, enhancing their invasion and proliferation—effects reversible by MET knockdown. Conclusion We unveil a spatially organized metabolic niche driven by stromal-tumor HGF-MET-MYC signaling. These findings offer novel insights into the stromal-tumor interaction and suggest actionable targets for therapeutic intervention in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dc67b8fc38aa04a6e00b35ef87b6cb65aaaec9" target='_blank'>
              A spatially resolved single-cell landscape of colorectal cancer liver metastasis reveals a stromal-tumor glycolytic signaling interaction
              </a>
            </td>
          <td>
            Jiahui Chen, Zukai Wang, Bingwang Zhu, Guoxian Guan
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7bbdd81d510f831417ecf76eb25316f13e6cad2" target='_blank'>
              SpaPheno: Linking Spatial Transcriptomics to Clinical Phenotypes with Interpretable Machine Learning
              </a>
            </td>
          <td>
            Bin Duan, Xiaojie Cheng, Hua Zhou
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195d5e43291b6baf82f9d6ab8680e6d30406a98c" target='_blank'>
              Differential colocalization of cell-type pairs in single-cell spatial omics with DC-SPOMIC
              </a>
            </td>
          <td>
            Jacob Chang, Almudena Espín Pérez, Perla Molina, Lu Tian, Sylvia K Plevritis
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617cbae0aef811bf0bd717218d2f691f2de87645" target='_blank'>
              Germline genetic risk converges through intercellular crosstalk in tumor microenvironment
              </a>
            </td>
          <td>
            X. Bai, Yang Liu, Shuai He, Chun-Ling Luo, G.-W. Lin, Dong-Mei Chi, Shuang-Yan Ye, Xiu-Zhi Wang, , You Zhou, Shu-Qiang Liu, P.-P. Wei, Xin-Yuan Guan, Hai-Qiang Mai, Soon Thye Lim, Jian-Jun Liu, C. Khor, Melvin Lee, Kiang Chua, Jin-Xin Bei, Yanni Zeng
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="
 Recent single-cell studies implicate the glutamatergic lineage of the rhombic lip (RL) as the origin of group (G) 3/4 medulloblastomas (MBs) and a stalled differentiation program as underlying their malignant transformation. It is unknown, however, if G3/4 MBs exhibit cellular plasticity as has been observed in glioma. The effect of therapy on cellular hierarchies in G3/4 MBs is unknown. We profiled N=57 G3/4 MB frozen specimens obtained from the University of California, San Francisco Brain Tumor Center and the Children’s Brain Tumor Network biobanks via single-nucleus RNA and epigenetic sequencing. This cohort included 43 diagnostic and 14 patient-matched recurrent specimens. We profiled N=32 FFPE specimens via Xenium single-cell spatial transcriptomics. We performed genetic lineage tracing in intracranial implants of G3 MB cell lines derived from primary and metastatic disease using engineered barcodes, and in clinical specimens using endogenous mutations. Remarkably, xenografts from primary tumors produced similar cell types, in similar proportions to those found in treatment-naive human MBs. Moreover, the distribution of cell types found in xenografts was conserved at the level of individual clones, indicating cell-intrinsic regulation of tumor composition. Single-cell phylogenetic analysis demonstrated significant rates of dedifferentiation events from unipolar brush-like cells to RL-progenitor phenotypes. This plasticity was increased in recurrent specimens and in metastatic models compared to models and samples derived from primary disease. Samples with higher plasticity exhibited greater spatial heterogeneity, while samples with lower rates of dedifferentiation had more structured spatial organization. These studies show that G3/4 MB cellular composition is homeostatic in treatment-naive disease and conserved at the level of individual clones. Yet, these tumors exhibit plasticity that is exacerbated by therapy and affects spatial organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7150b972b545d28e4de15c0584606a43a9a4dc" target='_blank'>
              STEM-18. GENETIC LINEAGE TRACING OF MEDULLOBLASTOMA UNDER THERAPY WITH SPATIAL CONTEXT
              </a>
            </td>
          <td>
            Bohyeon Yu, Jangham Jung, Abhinav Jain, Emanuele Rosatti, Joanna Phillips, Aaron A. Diaz
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are key regulators of the tumor microenvironment, yet their spatial organization and immunomodulatory functions in H. pylori-associated gastric cancer (GC) remain incompletely understood. We profiled formalin-fixed paraffin-embedded (FFPE) tumors from 71 GC patients using spatial transcriptomics and integrated single-cell RNA-seq from three independent cohorts (China, USA, and Singapore). CAF-immune cell colocalization was quantified by neighborhood enrichment and aggregation index score. Ligand-receptor inference and trajectory analysis resolved CAF signaling and state transitions. To delineate post-transcriptional control, ARE-motif scanning and expression correlations were combined with laser-assisted crosslinking and immunoprecipitation sequencing (LACE-seq) to nominate ZFP36 targets. Immune contexture and prognostic associations were evaluated using CIBERSORT-ABS and Kaplan-Meier analyses in The Cancer Genome Atlas (TCGA) and Asian Cancer Research Group (ACRG) cohorts. We first defined the spatial distributions of the four CAF subtypes reported in prior studies and found that their immune associations varied across histologic and infection-defined GC subtypes. In H. pylori-positive tumors, THBS1⁺ CAFs were spatially enriched near regulatory T cells (Tregs) and were associated with local immunosuppression through WNT5-FZD interactions. In parallel, ZFP36 bound AU-rich elements within the FN1 3′ untranslated region (3'UTR), destabilizing FN1 mRNA and thereby diminishing FN1⁺ CAF-mediated cytotoxic T lymphocyte (CTL) activation. Together, these axes promoted Treg accumulation and suppression of CTL activation. These findings reveal infection-associated stromal programs that shape the immune landscape in GC and highlight CAF-directed pathways as potential therapeutic targets in H. pylori-associated GC. H. pylori-induced CAF-mediated immunosuppression in GC. 1. H. pylori infection triggers transcriptional reprogramming of CAFs within the tumor microenvironment of GC. 2. The CAF subset marked by THBS1 is spatially enriched adjacent to Tregs and facilitates their recruitment via WNT5-FZD ligand-receptor signaling. 3. The RNA-binding protein ZFP36 binds AU-rich elements in the FN1 3’UTR, destabilizing FN1 mRNA and thereby attenuating CTL activation. 4. The THBS1-WNT5 axis stabilizes Tregs, whereas the ZFP36-FN1 axis limits CTL activation, converging to drive local immunosuppression in H. pylori-positive GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7f1c559549d2edda98547851b085fedd496597a" target='_blank'>
              Spatial and functional dissection of cancer-associated fibroblasts-mediated immune modulation in H. pylori-associated gastric cancer
              </a>
            </td>
          <td>
            Bonan Chen, Hongzhen Tang, Xiaohong Zheng, Fuda Xie, Peiyao Yu, Yang Lyu, Tiejun Feng, Jialin Wu, Jingya Liu, Yi Xu, A. H. Cheung, Canbin Fang, Zhangding Wang, Shouyu Wang, Justin Chak Ting Cheung, Yujuan Dong, Ruoxi Tian, Yigan Zhang, Cheng Lu, C. Wong, Jun Yu, William K. K. Wu, Elke Burgermeister, Man Tong, Fengbin Zhang, Wei Kang, Kam Tong Leung, K. To
          </td>
          <td>2025-11-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Metastases represent one of the hardest obstacles in cancer treatment, accounting for many cancer-related deaths. Understanding the mechanisms that drive disease progression is essential to improve patient outcomes and develop more effective therapeutic strategies. This phenomenon appears to be elicited by dormant cancer cells (DCCs), which can persist undetected for extended periods of time, entering a non-proliferative, hibernation-like state that confers resistance to conventional therapies and facilitates immune evasion. Owing to their status as a rare and energy-restricted population, combined with the limitations of current medical imaging, DCCs often evade early detection, hindering timely intervention and effective clinical management. Consequently, a critical need exists to develop high-resolution detection systems and identify specific DCC targetable biomarkers. Here, a comprehensive overview of the current understanding of DCCs is reported, with a focus on recent advancements in experimental strategies for their identification and tracking, as well as therapeutic approaches currently under clinical investigation aimed at targeting these elusive cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9d13b10d52e65e6fb3d5e89c185ee227dee670" target='_blank'>
              Decoding cancer dormancy: integrative genomic, phenotypic and live-cell imaging analysis to reveal the hidden cancer cell reservoir
              </a>
            </td>
          <td>
            G. Porcelli, Caterina D’Accardo, Francesca Angeloro, Martina Cucchiara, Paola Bianca, Vincenzo Davide Pantina, Narges Roozafzay, Chiara Modica, Miriam Gaggianesi, Sebastiano Di Bella, G. Stassi, Alice Turdo, M. Todaro
          </td>
          <td>2025-11-12</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are central architects of the prostate cancer (PCa) microenvironment, yet their phenotypic diversity and druggable vulnerabilities remain largely uncharted. Here, we present an integrative multi-omics characterization of primary ex vivo CAFs from seven treatment-naïve PCa patients. Using single-cell RNA sequencing (scRNA-seq), we uncover substantial transcriptional heterogeneity among CAFs, with distinct gene expression programs related to extracellular matrix remodeling, inflammation, immune modulation, and metabolic reprogramming. This phenotypic diversity was further supported by variable expression of canonical stromal markers, including FAP, SULF1, VIM, CAV1, and αSMA. Transcription factor network analysis revealed SOX, FOX, and STAT3 family members as key regulators of pro-tumorigenic CAF states. To probe therapeutic vulnerabilities, we performed high-throughput drug sensitivity and resistance testing (DSRT) across 396 oncology compounds. CAFs exhibited broad sensitivity to multikinase inhibitors, with dasatinib, midostaurin, and FGFR inhibitors (AZD4547, erdafitinib) emerging as top stromal-directed candidates. These findings underscore the plasticity of prostate CAFs and reveal actionable vulnerabilities, supporting the development of targeted stromal therapies to disrupt tumor-stroma interactions in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0342abe6ecbf94b88ae457a3f680a1e8d054b454" target='_blank'>
              Molecular profiling of ex vivo prostate cancer CAF models captures stromal heterogeneity and drug vulnerabilities
              </a>
            </td>
          <td>
            Frida Rantanen, A. Murumägi, Mariliina Arjama, K. Välimäki, Elina Multamäki, T. Mirtti, Antti S. Rannikko, T. Pellinen, Daniela Ungureanu, Olli Kallioniemi
          </td>
          <td>2025-11-06</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d4b0eb929a81f46df1d1e2e57abece7f236b787" target='_blank'>
              PDAC tumor Immune system escape architecture: Analysis of transcriptomics of mechanisms that TME is tackling immune system in Pancreatic Ductal Adenocarcinoma Tumors
              </a>
            </td>
          <td>
            Abbas Zareei, Hossein Allahdadi
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Heterogeneity within the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment (TME), across tumor locations, and between patients largely contributes to poor treatment response and PDAC’s status as the fourth leading cause of cancer death in the United States. In order to identify suitable targets for innovative therapeutic interventions, it is necessary to understand and mechanistically test TME constituents, their infiltration, and their interaction dynamics in the context of primary and metastatic tumor locations using pre-clinical modeling. Therefore, we designed a novel 46-marker multiplex immunofluorescent staining panel for murine formalin-fixed paraffin-embedded (FFPE) tissues. This panel enables the spatial quantification of immune cell populations and activation/exhaustion states, epithelial-to-mesenchymal phenotypes, cancer-associated fibroblast heterogeneity, hematologic and lymphatic vessel infiltration, neuronal infiltration, and extracellular matrix deposition in the TME. We quantified the composition and spatial infiltration of the TME on matched murine primary and liver tumor samples from tandem orthotopic and hemi-spleen transplants of three distinct primary murine PDAC cell lines derived from the KPC model (KrasG12D; TP53R172H; PDX1-Cre; RosaEYFP). The resulting primary tumors from each cell line showed inter-tumoral heterogeneity as their distinct levels of immune infiltration had varying clustering patterns. Spatial analysis further revealed heterogeneous cancer cell MHC-I expression, immune cell exhaustion, and myeloid and lymphoid populations correlating with cell line. However, all liver nodules were encased by a barrier of immune cells separating the tumor from normal liver parenchyma irrespective of cell line. This observation reflects our recent spatial transcriptomic analysis of patient-derived matched liver and primary tumors, supporting the use of these pre-clinical models for future mechanistic investigations. Altogether, these data highlight the complexity of PDAC TMEs and the site-specific differences between primary and metastatic tumor composition.



 Christina R. Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert Welner, Julienne L. Carstens. High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B110.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335836491342c8b5df5a982f9e0790e87ffd1d16" target='_blank'>
              Abstract B110: High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity
              </a>
            </td>
          <td>
            Christina R Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c38b6d1e04676ebe5c874a1af9317e2b5d3d46" target='_blank'>
              Single-cell Transcriptomic Profiling of Pancreatic Ductal Adenocarcinoma: Epithelial Reprogramming and Systemic Immune Exhaustion
              </a>
            </td>
          <td>
            Qinyu Xu
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Intratumoral heterogeneity in pancreatic ductal adenocarcinoma (PDAC) is characterized by the coexistence of classical epithelial, basal, and mesenchymal cancer cell states. Targeting cancer cell plasticity—the ability of tumor cells to adapt to extrinsic pressures via non-genetic mechanisms—has emerged as a promising therapeutic strategy. In parallel, the immunosuppressive tumor microenvironment remains a major barrier to effective treatment in PDAC. Yet, whether specific cell states possess distinct functional properties that drive key tumor behaviors—such as phenotypic plasticity or immune evasion—remains poorly understood. Using genetically engineered mouse models driven by oncogenic Kras G12D and loss of Trp53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We find that the KPC model harbors a basal state, which harbors transcriptional signature that aligns with the "basal" signature in human PDAC and is linked to poor prognosis and chemoresistance. The basal state co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. To functionally interrogate the basal state in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the basal state has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the basal state is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the basal state over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we find that the basal state is a key driver of therapeutic resistance: rather than being eliminated by treatment, it persists through phenotypic adaptation in response to both KRAS-targeted therapies and cytotoxic chemotherapy. Co-elimination of the basal state alongside these therapies leads to deeper and more durable responses, underscoring its functional role in tumor persistence under therapy. In parallel, we observe that the basal state plays a central role in enforcing immunosuppression in PDAC tumors. Basal state ablation rapidly reshapes the tumor immune microenvironment, and we observe loss of cancer-associated fibroblasts, polarization of PDAC-resident macrophages toward a pro-inflammatory phenotype, and increased infiltration of T and NK cells. Cytokine profiling of basal-ablated tumors revealed a dramatic reduction in multiple immune-modulatory and immunosuppressive cytokines. Taken together, our data indicate that PDAC harbors a striking dependency on the basal state for both therapeutic resistance and immune evasion, and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.



 Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C. Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Zeynep C. Tarcan, Olca Basturk, Rona Yaeger, James G. Christensen, Yan Yan, Elisa de Stanchina, Dana Pe'er, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B108.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6879ea5062e58b2315581060e5992acf02efdbb" target='_blank'>
              Abstract B108: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Z. Tarcan, O. Basturk, R. Yaeger, James G Christensen, Yan Yan, E. de Stanchina, D. Pe’er, Tuomas Tammela
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="


 Recent studies using single-cell RNA sequencing (scRNA-seq) have provided insights into the cellular composition and molecular states in glioblastoma (GBM). However, these studies have not identified significant differences between primary and recurrent GBMs. We hypothesize that other factors, such as spatial organization and cellular interactions, may provide key information driving recurrence in GBM. Spatial transcriptomics techniques, such as MERFISH (Multiplexed Error-Robust Fluorescence In Situ Hybridization), provide single-cell spatial resolution of tissues, enabling the identification of distinct molecular features. MERFISH, in particular, allows for the 3D reconstruction of gene expression signatures in consecutive tissue slices, providing a detailed spatial map of the tumor cells and surrounding environment. This technology is invaluable in studying longitudinal GBMs, where changes in tumor organization, immune cell interactions, and mechanisms of therapeutic resistance remain poorly understood.



 We analyzed 102 tumor samples consisting of primary and recurrent GBMs and normal brain tissue using Visium and MERSCOPE technologies. We used supervised artificial intelligence (AI)-based learning, in-silico perturbation and explainable-AI to identify distinct spatial and cell-type-specific transcriptomic reorganization associated with GBM recurrence.



 We developed a spatial transcriptomics atlas of GBM at sub-cellular resolution. Our analysis revealed distinct cellular neighborhoods in the primary and recurrent GBM tumors, with changes leading to the dynamic spatial reorganization of GBM at recurrence. The 3D spatial images acquired provided a comprehensive view of the tumor cells interacting with the tumor microenvironment in situ. Our explainable-AI models successfully identified distinct spatial transcriptomic features associated with recurrence in GBM. The identified spatially distinct cellular neighborhoods reveal dynamic shift in cellular organization following in silico perturbation.



 This study provides a high-resolution spatial map of longitudinal GBM that offers new insights into the dynamic cellular reorganization of GBM at recurrence. Our findings highlight the role of cellular reorganization during tumor recurrence in GBM, a potential avenue for targeted therapeutic intervention.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4929df1bf2a8e42843ae0cfe5558a1691cc6412" target='_blank'>
              P04.05.B DEEP LEARNING ANALYSIS OF SPATIALLY RESOLVED SINGLE-CELL TRANSCRIPTOMICS UNCOVERS CELLULAR REORGANIZATION IN GLIOBLASTOMA AT RECURRENCE
              </a>
            </td>
          <td>
            Y. Yabo, E. Grabis, T. Peng, J. Benotmane, J. Kueckelhaus, J. Zhang, G. Villa, N. Neidert, M. Prinz, J. Beck, A. Ahmed, O. Schnell, F. Sahm, V. Venkataramani, F. Ricklefs, R. Sankowski, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is the most common histological subtype of renal cancer and remains a clinical challenge due to its frequent resistance to therapy and poor prognosis in advanced stages. Apoptosis, a fundamental tumor-suppressive mechanism, exhibits paradoxical roles in cancer, wherein apoptotic tumor cells can also contribute to immunosuppression and tumor progression. However, the spatial dynamics, transcriptional heterogeneity, and prognostic relevance of apoptosis-related gene programs in ccRCC remain poorly defined. We performed an integrative analysis combining single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and summary-based Mendelian randomization (SMR) to dissect apoptosis-related malignant cell states in ccRCC. Cancer cells were stratified based on apoptosis gene signatures and CASP9 expression. Cell–cell communication was assessed using CellChat and spatial interaction networks were constructed using RCTD and mistyR. SMR was employed to link genetically regulated CASP9 expression with renal cancer risk. A CASP9-associated prognostic model was developed using LASSO Cox regression and DeepSurv on TCGA and E-MTAB-1980 cohorts. We identified transcriptionally and spatially distinct apoptosis-high and apoptosis-low malignant cell subpopulations. Apoptosis-high tumor cells, characterized by elevated CASP9 expression, preferentially localized near macrophage-enriched stromal regions and exhibited stronger spatial clustering. Ligand–receptor modeling revealed directional signaling via the SPP1–CD44 axis between CASP9-high cancer cells and macrophages. SMR analysis provided genetic evidence supporting CASP9 as a causal gene for renal cancer. CASP9-high cells demonstrated distinct developmental trajectories and formed multicellular spatial modules with macrophages and cycling cells. A five-gene apoptosis-related signature derived from CASP9-stratified tumor cells robustly predicted patient survival across both training and validation cohorts. Low-risk patients exhibited enriched immune infiltration, increased immune checkpoint expression, and enhanced immune pathway activity. Our study reveals that apoptosis, particularly CASP9-driven programs, defines a spatially organized, immunosuppressive malignant cell state in ccRCC. CASP9 acts as both a genetic driver and spatial regulator of tumor–macrophage interactions, contributing to disease progression. The CASP9-associated risk model demonstrates strong prognostic utility and highlights apoptosis as a promising therapeutic axis in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22570e710b314543b7842267551dd125b21247d1" target='_blank'>
              Integrative SMR and single cell & spatial analysis reveals the spatial heterogeneity and prognostic value of CASP9-mediated apoptotic pathways in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Zijie Yu, Xinghan Yan, Wenchuan Shao, Lingchen Cai, Da Zhong, Xiyi Wei, Ninghong Song
          </td>
          <td>2025-11-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The ability to interrogate changes within the tumor microenvironment (TME) before, during and following therapeutic intervention could yield important understanding of treatment response and causes for disease progression. Yet, the role of investigational tissue analysis faces key challenges in the clinical setting and the value of integrating longitudinal biopsies with emerging multimodal molecular analyses (“Multi-omics”) remains to be defined. In this study, we conducted a multicenter phase 2 clinical trial examining the effect of a novel cytotoxic T-lymphocyte-associated protein 4/programmed death-ligand 1 bispecific antibody in combination with a dual-epitope blocking anti-human epidermal growth factor receptor 2 antibody in treatment-resistant metastatic breast cancer. We performed longitudinal sampling of patient tumor tissues before and following treatment. Single-cell RNA and T cell receptor sequencing from 334,183 cells from site-matched tumors reveals significant temporal shift of various immune cell populations and phenotypes within the TME associated with treatment responses. Conversely, regulatory T cells were activated while effector T cells, natural killer cells, and dendritic cells were significantly depleted in non-responding tumors. Taken together, these results support that longitudinal analysis of TME to generate multiomics data that can lead to rich insight into disease process and to provide clinical value in evaluating treatment responses. Trial registration number NCT04521179.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97d0a38c945abea56c7fcf08591e230c989d914" target='_blank'>
              Longitudinal tissue analysis reveals microenvironmental changes correlate with combined immunotherapy and targeted therapy response in metastatic breast cancer
              </a>
            </td>
          <td>
            Jianyou Liao, Jien-Ren Wang,, Hengyu Li, Zhijun Liu, Z. Tian, Xinying Lv, Jianjian Peng, Chuangui Song, Jieqiong Liu
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4534f83212d853ecc1eca8aac314fc0a39255797" target='_blank'>
              Integrative Spatial Modelling of Cellular Plasticity using Graph Neural Networks and Geostatistics
              </a>
            </td>
          <td>
            Eloise Withnell, Cenk Celik, M. Secrier
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Phenotypic plasticity is a major factor in tumor heterogeneity and treatment resistance. In particular, cancer stem cells (CSCs) represent a small subpopulation within tumors that possesses self-renewal and tumor-forming capabilities. Understanding reprogramming, maintenance, and lineage properties of CSCs requires dedicated tools to disentangle the respective influences of phenotypic inheritance and cell-cell interactions. Here, we set up ultra-wide field microscopy to image breast cancer cell lines expressing a stemness fluorescent reporter over several days. The fluorescent reporter distinguishes three phenotypes: CSCs, cancer differentiated cells (CDCs), and intermediate/transiting cancer cells (iCCs). Spatial statistics indicate significant zonation in which CSCs cluster together and are spatially separated from CDCs, forming patterns resembling niches. Surprisingly, single-cell time series reveal spontaneous reprogramming events from CDC to CSC even in unperturbed populations. We identify that such transitions are prone to arise during the cell cycle. Moreover, lineage analysis shows that the phenotype is partially inherited from ancestor cells. However, such heredity is not sufficient to explain the spatial properties of the cell population, which also depend on cell-cell interactions. Indeed, we find that phenotypic transitions of cancer cells are influenced by the phenotypic state of neighboring cells. Reprogramming into CSCs is respectively promoted and inhibited by the presence of CSCs and CDCs in the neighborhood. Altogether, our results disentangle how phenotypic inheritance and intercellular interactions orchestrate the spatio-temporal self-organization of cancer cell heterogeneity, maintaining a subpopulation of CSCs within niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0816aa85e620c96aaa89badbcea20ed0e370fd6d" target='_blank'>
              Spatial self-organization of cancer stem cell niches revealed by live single-cell imaging
              </a>
            </td>
          <td>
            Mathilde Brulé, Anaïs Horochowska, Emeline Fontaine, Raoul Torero-Ibad, Flavie Woesteland, Marie Denoulet, Jean Pesez, E. Adriaenssens, R. Toillon, X. Le Bourhis, Benjamin Pfeuty, Chann Lagadec, F. Anquez
          </td>
          <td>2025-10-27</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Mass cytometry (CyTOF) and Imaging Mass Cytometry (IMC) provide single-cell resolution for over 50 protein markers, enabling unprecedented exploration of tumour and immune heterogeneity. We conducted a scoping review of 61 original studies (inception–2025), spanning 17 cancer types, to map current applications, analytical strategies, and emerging biological insights. 46 studies used CyTOF alone, 12 employed IMC exclusively, and 3 combined both platforms. Median panel sizes were 33.5 markers for CyTOF and 33 for IMC. While lineage and immune checkpoint markers were universal, phospho-epitopes, metabolic enzymes, and stromal proteins appeared in more focused subsets. Most studies followed a three-step analytical workflow: (i) segmentation or gating, (ii) unsupervised clustering, and (iii) downstream spatial or functional analyses. CyTOF investigations frequently identified exhausted CD8+ T-cell subsets (e.g., PD-1+TIM-3+CD39+), suppressive myeloid populations (e.g., CD163+HLA-DR− macrophages), and metabolically reprogrammed Tregs. IMC studies uncovered spatial patterns predictive of outcome, such as tertiary lymphoid structures (TLSs) and macrophage–T cell exclusion zones. Several studies proposed predictive immune signatures or integrated CyTOF with transcriptomic or spatial datasets. We identified five recurrent immunobiological motifs, CD8+ T-cell bifurcation, CD38+ TAM barriers, TLS maturity, CTLA-4+ NK-cell signatures and metabolically defined niches, highlighting convergent axes of resistance and response. Bioinformatic pipelines converged around FlowSOM or PhenoGraph clustering, CITRUS or elastic-net feature selection, and increasingly, machine learning and agent-based spatial modelling. Collectively, CyTOF and IMC are redefining biomarker discovery, therapeutic stratification, and virtual trial design in oncology, establishing high-dimensional CyTOF as a cornerstone of next-generation precision cancer medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c751da190ee62eb1876b5c48df077ef5372fdfb0" target='_blank'>
              Pan-cancer convergence of tumour–immune microenvironment motifs revealed by CyTOF and imaging mass cytometry
              </a>
            </td>
          <td>
            Alexandre Vallée, Alexandre Drezet, Maxence Arutkin
          </td>
          <td>2025-10-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) invasion is shaped by dynamic clonal evolution within the spatial and cellular architecture of the brain. To investigate this process, we integrated multiparametric MRI, whole-exome sequencing (WES), bulk RNA-seq and single-nucleus RNA-seq (snRNA-seq) from 69 IDH-wildtype GBM patients, using spatially resolved biopsies from contrast-enhancing (CE) core tumor and non-enhancing (NE) invasive regions. Phylogenetic analysis revealed distinct clonal architectures. CE-enriched clones carried truncal mutations in conanical drivers of cell cycle and RTK signaling (TP53, PTEN, EGFR, PDGFRA/B), and private alterations in DNA repair genes (CDH11, PAX7, RAD51B). In contrast, NE-enriched clones exhibited private mutations in MAPK and Hippo pathways components (STAG2, BRAF, MAP2K3, CTNNA1), suggesting region-specific evolutionary trajectories. Furthermore, snRNAseq cell-cell communication analysis uncovered enriched ligand-receptor signaling in NE regions between malignant cells and brain-resident populations, including PTN—PTPRZ1 (myeloid), NRG-ERBB4 and SLIT-ROBO (neuronal, astrocytic, and oligodendrocytic). Bulk RNAseq deconvolution confirmed increased representation of neurons and glial cells in biopsies harboring NE-specific clones. These findings define a unique population of GBM clones localized at the infiltrative margin, marked by private MAPK/Hippo mutations and enhanced intracellular communication with the surrounding brain microenvironment. The recurrent engagement of neural-glial signaling pathways suggests that these NE-associated clones co-opt normal cellular networks to support invasion, immune evasion, and persistence following surgical resection. This study establishes a mechanistic link between spatial clonal evolution and tumor-host communication at the GBM infiltrative region and provide a framework for targeting cell-cell signaling circuits that sustain invasive tumor ecosystems and escape therapeutic treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee4b73c1f939dc48d12b3b70a8dbe5ab434d2210" target='_blank'>
              CSIG-31. REDEFINING THE GLIOBLASTOMA RESECTION MARGIN THROUGH SPATIAL CLONAL AND CELLULAR ECOSYSTEM ANALYSIS
              </a>
            </td>
          <td>
            Angad S. Beniwal, Junlin Wang, Xiaoling Lu, C. Sereduk, S. Migliozzi, L. Garofano, C. Plaisier, Leland Hu, R. Barajas, Junwen Wang, Nhan Tran
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is classified into epithelial and mesenchymal subtypes based on transcriptional profiling, and these subtypes are associated with varied patient outcomes. Differences in the tumor microenvironment (TME), particularly in cancer-associated fibroblasts (CAFs), may contribute to this variation. scRNA-seq studies have observed heterogeneous CAF populations in PDAC, although the relationship between tumor subtype and CAF phenotype is not well understood. To investigate this relationship, we utilized epithelial and mesenchymal PDAC cell lines in a xenograft model to examine whether these subtypes influence CAF phenotypes.



 Xenograft models were generated by subcutaneously injecting BALB/c nude mice with PDAC cell lines of either the epithelial subtype (BxPC-3) or the mesenchymal subtype (MIA PaCa-2). Tumors were then harvested and dissociated for scRNA-seq to investigate the TME. Human tumor (graft) and murine stromal (host) cells were computationally separated using the Xenocell tool. Subsequent analyses focused on fibroblasts within the TME, which were subclustered to assess CAF heterogeneity between PDAC subtypes. To characterize transcriptional programs and identify CAF features, we performed high-dimensional weighted gene co-expression network analysis (hdWGCNA) and analyzed differentially expressed genes. Finally, spatial transcriptomics was conducted on tumor sections from BxPC-3 and MIA PaCa-2 xenografts to validate CAF clusters observed in scRNA-seq.



 Distinct CAF clusters were identified in the xenograft tumors, including LRRC15+ CAFs, RGS5+ CAFs, and inflammatory CAFs (iCAFs). LRRC15+ CAFs were predominant in tumors derived from the epithelial subtype (BxPC-3), while RGS5+ CAFs were enriched in tumors from the mesenchymal subtype (MIA PaCa-2). LRRC15+ CAFs exhibited a myofibroblastic CAF (myCAF) phenotype characterized by extracellular matrix production, with high expression of Lrrc15, Postn, and Fbln1. RGS5+ CAFs expressed canonical pericyte markers such as Rgs5 and Cox4i2, along with CAF-associated genes including Acta2. These CAF clusters also showed distinct expression patterns of collagens and secreted factors. In particular, Col11a1, Col12a1, and Wnt5a were upregulated in LRRC15+ CAFs, while Col4a1, Col10a1, and Tgfb1 were upregulated in RGS5+ CAFs. Spatial transcriptomics confirmed that gene modules associated with Lrrc15 and Rgs5, inferred from hdWGCNA, were differentially expressed between Acta2-positive stromal regions of epithelial and mesenchymal PDACs.



 These results suggest that epithelial and mesenchymal PDAC subtypes directly shape distinct CAF populations. Notably, the enrichment of RGS5+ CAFs in mesenchymal tumors may underlie their metastatic potential and poor clinical outcomes.



 Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, Jung Hyun Jo. Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B105.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020656bad62fdcf52b16552c5f7369030a0e1205" target='_blank'>
              Abstract B105: Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model
              </a>
            </td>
          <td>
            Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo-Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, J. Jo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Gliosarcoma, a rare and aggressive variant of IDH-wildtype glioblastoma, is characterized by striking intra-tumoral heterogeneity with alternating glial and mesenchymal differentiation. The mesenchymal component is often enriched at recurrence and associated with therapy resistance, suggesting treatment-driven clonal selection. Despite its poor prognosis and distinct histology, gliosarcoma remains underexplored at the molecular level, with no systematic profiling of its cellular or spatial architecture - limiting progress toward targeted therapies.



 Single-nucleus RNA sequencing (snRNA-seq) was performed on eight FFPE gliosarcoma samples, complemented by spatial transcriptomic profiling (10X Visium) in four matched cases. Multimodal integration enabled high-resolution mapping of malignant and non-malignant compartments. To enhance spatial context extraction and to model tumor-microenvironment interactions, advanced computational approaches, including graph neural networks (GNNs) and convolutional neural networks (CNNs), are being developed.



 Integration of snRNA-seq and spatial transcriptomic data revealed a distinct population of stromal-like malignant cells enriched for mesenchymal signatures. Compared to conventional glioblastoma, gliosarcoma samples exhibited a reduced presence of neural progenitor cell (NPC-like) and oligodendrocyte progenitor cell (OPC-like) states, suggesting early dominance or selective expansion of the mesenchymal compartment. Spatial annotation of the tumor microenvironment revealed immune-rich niches populated by disease-associated microglia (DIMs), damage-associated macrophages (DAMs), and macrophages (MACs). Preliminary spatial modeling indicates potential interactions between immune population and mesenchymal-like tumor cells that may support therapy-resistant niches.



 This ongoing study integrates single-nucleus and spatial transcriptomic profiling to characterize mesenchymal transition and resistance in gliosarcoma. Early findings highlight a distinct mesenchymal-like malignant population and associated immune environments that may drive resistance. Further computational modeling aims to uncover key cellular interactions and molecular programs, that could inform novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47d9bc10df09a19173093cb2520a4af4dc46d0d5" target='_blank'>
              P14.25.B DECODING MORPHO-MOLECULAR DIVERSITY IN GLIOSARCOMA AND INSIGHTS INTO RESISTANCE USING SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            D. Savran, N. Schoebe, N. Berghaus, D. Reuss, M. Ratliff, C. Herold-Mende, S. Krieg, W. Wick, A. von Deimling, D. Heiland, F. Sahm, P. Sievers
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="ABSTRACT The immune landscape of healthy prostate and its alterations during prostate cancer (PCa) progression remain poorly characterized. Using scRNA-sequencing and multiparametric flow-cytometry analysis, we comprehensively characterized immune cells in wild-type and PTEN(i)pe−/− mouse prostates, a model that closely recapitulates human PCa. PCa in PTEN(i)pe−/− is marked by the recruitment of tumor-associated neutrophils (TANs), which represent the dominant immune cell population and resolved into eight distinct states, Trem2+ tumor-associated-macrophages (TAMs), and exhausted CD8+ T cells. Trem2+ TAMs differ from the three main resident macrophage populations in the healthy prostate, exhibiting a strong metabolic and immunosuppressive signature, likely driven by the MIF/HIF1A-signaling axis. This study provides the first detailed characterization of immune cells in the healthy mouse prostate and reveals changes in the immune landscape associated with prostate cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb0c0592aa197777e78a6d1020c20a4157e6fc" target='_blank'>
              Characterization of the immune landscape in healthy mouse prostate and during prostate cancer progression
              </a>
            </td>
          <td>
            Despoina Pervizou, Joanna De Chiara, L. Spinelli, Maia Nestor-Martin, Lionel Chasson, Kateryna Len-Tayon, Darya Yanushko, Frédéric Fiore, Marc Bajénoff, Bernard Malissen, Daniel Metzger, Gilles Laverny, S. Henri
          </td>
          <td>2025-09-26</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b15ed383fa686f49d7de283de5e0eaf7d1cc0c" target='_blank'>
              A Rare Multipotent Peg-like Epithelial Cell is a Candidate Cell-of-Origin for High-Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Megan L. Ritting, Wenmei Yang, Syed Mohamed Musheer Aalam, Hui Zhao, Liang Feng, Jianning Song, Mihai G. Dumbrava, W. Ismail, Dong-Gi Mun, Chunling Hu, Opalina Roy, Keyur Chaludiya, Geng-Xian Shi, Daphne Norma Crasta, Kenneth Schaufelberger, Jeffrey R. Janus, Guruprasad Kalthur, S. Weroha, Scott H. Kaufmann, A. Sadanandam, David J. H. F. Knapp, Chen Wang, Akhilesh Pandey, Alexandre Gaspar-Maia, F. Couch, Mark E. Sherman, J. Bakkum-Gamez, Nagarajan Kannan
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>125</td>
        </tr>

        <tr id="Metastasis remains the principal cause of cancer-related mortality, yet its distribution across organs is far from random. Instead, tumor cells exhibit organotropism, a consistent preference for colonizing specific distant tissues, a phenomenon shaped by anatomical constraints, molecular crosstalk, and microenvironmental compatibility. Far beyond mere mechanical entrapment in vascular beds, metastatic dissemination reflects a coordinated interplay between tumor-intrinsic programs and organ-specific niches. Tumor-derived extracellular vesicles, cytokines, and matrix-remodeling enzymes actively precondition distant sites through pre-metastatic niche formation, creating permissive microenvironments primed for colonization. Simultaneously, tissue-specific immune landscapes, stromal compositions, and mechanical architectures determine the fate of disseminated tumor cells, whether they are eliminated, enter dormancy, or form macrometastases. Phenotypic plasticity, metabolic reprogramming, and immune evasion mechanisms equip subclones with the capacity to exploit these unique niches. Across cancer types, reproducible patterns of organotropic metastasis not only guide clinical surveillance and therapeutic stratification but also reveal vulnerabilities in the metastatic cascade. This review synthesizes emerging mechanistic insights across anatomical, immunological, and molecular domains to construct a comprehensive framework of organotropism, highlighting therapeutic opportunities to intercept metastasis at organ-specific checkpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7663dab540302726b6f7b6337c8bf27a6cf336" target='_blank'>
              Deciphering organotropism reveals therapeutic targets in metastasis
              </a>
            </td>
          <td>
            Jhommara Bautista, Andrés López-Cortés
          </td>
          <td>2025-10-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Spatial transcriptomic data enable study of cell–cell communication, yet current analysis tools often fail to provide dynamic, interpretable estimates of interactions and their spatial range across tissue. We present AMICI, an interpretable attention framework that jointly estimates interaction length scales, adaptively resolves sender–receiver subpopulations, and links communication to downstream gene programs. AMICI recovers ground-truth interactions in semi-synthetic data, uncovers gene programs linked to cell communication in the mouse cortex, and reveals length-scale-dependent tumor–immune signaling that reinforces estrogen receptor (ER) programs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71fbc9a6d79bce1e371c9e03697ca819ffcb1d27" target='_blank'>
              AMICI: Attention Mechanism Interpretation of Cell-cell Interactions
              </a>
            </td>
          <td>
            Justin Hong, Khushi Desai, Tu Duyen Nguyen, Achille Nazaret, Nathan Levy, Can Ergen, G. Plitas, Elham Azizi
          </td>
          <td>2025-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Intratumoral heterogeneity in glioblastoma (GBM) remains a major barrier to effective therapy. Spatial variations in tumor biology contribute to treatment resistance and recurrence but remain underexplored in preclinical models. To address this, we developed spatially paired GBM models derived from anatomically distinct tumor regions—the invasive cortical margin and the solid tumor core—using both patient-derived tumor tissue (Pt-NBRX121GBM-Cortex and Pt-NBRX121GBM-Solid) and matched xenograft tumors (IC-NBRX121GBM-Cortex and IC-NBRX121GBM-Solid). From each region, we established three-dimensional organoid cultures and patient-derived orthotopic xenografts (PDOX), preserving regional biological characteristics. We conducted longitudinal survival studies in SCID mice, confocal imaging to assess organoid structure and metabolism, single-cell RNA sequencing (scRNA-seq), and high-throughput drug screening. PDOX models from cortex and solid regions showed moderate differences in median survival (76.0 to 88.5 days), but no statistically significant differences under comparable tumor burden, suggesting similar in vivo aggressiveness. In contrast, organoid analyses revealed pronounced region- and source-specific distinctions. Cortex-derived organoids, especially larger ones, exhibited disorganized morphology, peripheral mitochondrial clustering, and metabolic stress, while solid-derived organoids maintained spherical structure, strong mitochondrial-nuclear association, and high proliferative capacity. In vivo, cortex-derived tumors displayed diffuse infiltration, elevated mitochondrial activity, and mesenchymal-like gene expression, whereas solid-core tumors showed compact growth, increased proliferation, and hypoxia-driven angiogenesis. scRNA-seq confirmed distinct transcriptional programs consistent with these phenotypes. High-throughput drug screening uncovered region-specific therapeutic vulnerabilities, with cortex-derived models showing sensitivity to anti-migratory compounds and solid-derived tumors responding to agents targeting angiogenesis and metabolism. This spatially defined GBM organoid and PDOX platform captures key features of tumor heterogeneity across structure, metabolism, gene expression, and therapy response. These models provide a robust and scalable system for dissecting region-specific tumor biology and for advancing precision therapeutic strategies in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93969cee50e27415ebc83219ceaa447b23e4c103" target='_blank'>
              TMOD-38. Spatially Paired Glioblastoma Organoid and Xenograft Model derived from tumor core and invasive front Reveal Distinct Metabolism, Regional Tumor Behaviors and Therapy Responses
              </a>
            </td>
          <td>
            Xin Zhai, Tongchao Jiang, E. Ciolak, Jinnan Chen, Zilu Huang, M. S. Palacios, Robert C. Rostomily, James Bellis, N. Wadhwani, Xiaonan Li, Yuchen Du
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human breast cancer cells exhibit organotropism during metastasis, showing preferential homing to certain organs such as bone, lung, liver, and brain. Spatial genome organization plays a crucial role in oncogenic transformation and progression, but the extent to which chromosome architecture contributes to organ-specific metastatic traits is unclear. This work characterizes chromosome architecture changes associated with organotropic metastatic traits. By comparing a collection of human genomic data from different subtypes of localized and lung metastatic breast cancer cells with both normal and cancerous lung cells, we find important trends of genomic reorganization. The most striking differences in 3D genome compartments segregate cell types according to their epithelial vs. mesenchymal status. This epithelial–mesenchymal transition (EMT) compartment signature occurs at genomic regions distinct from transcription-defined EMT signatures, suggesting a separate layer of regulation. Specifically querying organotropism, we find 3D genome changes consistent with adaptations needed to survive in a new microenvironment, with lung metastatic breast cancer cells exhibiting compartment switch signatures that shift the genome architecture to a lung cell-like conformation and brain metastatic prostate cancer cells showing compartment shifts toward a brain-like state. TCGA patient data reveals gene expression changes concordant with these organ-permissive compartment changes. These results suggest that genome architecture provides an additional level of cell fate specification informing organotropism and enabling survival at the metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d470a8773036dd5ad3cf2b42848d5f3ff555882" target='_blank'>
              Rearrangement of 3D genome organization in breast cancer epithelial to mesenchymal transition and metastasis organotropism
              </a>
            </td>
          <td>
            Priyojit Das, Rebeca San Martin, Tian Hong, R. P. McCord
          </td>
          <td>2025-11-12</td>
          <td>eLife</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32263f8daacb6eb2cfb9538393332506d52b7ecc" target='_blank'>
              Characterising cancer-stroma interactions through high-content phenotyping from microscopy time-lapses
              </a>
            </td>
          <td>
            Laura Wiggins, Jodie R. Malcolm, Karen Hogg, Peter J. O’Toole, Julie Wilson, W. Brackenbury
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Adult-type granulosa cell tumors (AGCT) are rare ovarian tumors with few effective treatments for recurrent disease. To elucidate spatial features and cellular interactions within the AGCT tumor microenvironment, we applied imaging mass cytometry using a 34-marker panel on 130 regions from 24 AGCT samples, profiling more than 900,000 single cells. Analysis confirmed the immune “cold” phenotype of AGCTs and showed higher macrophage abundance in recurrent compared with primary tumors. We observed substantial heterogeneity in tissue architecture across samples, including variable presence of FOXL2+ cells embedded in collagen-rich regions (FOXL2+COL1A1+ cells). Based on tumor microenvironment composition, we defined two AGCT subtypes: AGCT-1 and AGCT-2 with distinct FOXL2+ cell distributions, differences in progesterone receptor expression, and unique transcriptomic profiles. Our findings highlight the role of macrophages, Foxl2+ subpopulations, and the extracellular matrix in AGCT progression and suggest AGCT subtype–specific vulnerabilities that could inform personalized therapies for this rare malignancy. Significance: We discovered two histologically and molecularly distinct forms of AGCTs that differ in cell composition, immune activity, and hormone signals. These findings point to new opportunities for more personalized treatment of this rare ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/374bc6a823e616d6fd56470b772ded7e796990b4" target='_blank'>
              Multiplexed Imaging Mass Cytometry Reveals Tumor-immune Microenvironment–dependent Hormone Receptor Expression in Adult-Type Ovarian Granulosa Cell Tumors
              </a>
            </td>
          <td>
            Eleonora Y Khlebus, Veena K Vuttaradhi, S. Ferri-Borgogno, Allison L. Brodsky, B. Lawson, Samuel C. Mok, R. T. Hillman
          </td>
          <td>2025-10-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract The complex interplay between metal abundance, transport mechanisms, cell distribution, and tumor progression-related biological pathways (e.g. metabolism, collagen remodeling) remains poorly understood. Traditionally, genes and metals have been studied in isolation, limiting insights into their interactions. Recent advances in spatial transcriptomics and elemental profiling now enable comprehensive exploration of tissue-wide metal-gene interactions, though integration remains challenging. In this proof-of-concept study, we investigated metal-dependent signaling within the tumor microenvironment of a unique colorectal cancer (CRC) tumor. We implemented a spatial multimodal workflow which integrated elemental imaging, gene expression, cellular composition, and histopathological features to uncover metals-related pathways through spatially resolved gene expression correlation analyses. Preliminary findings revealed significant associations, for instance: elevated iron correlated with mesenchymal phenotypes located at the tumor’s proliferative front, correlating with expression of genes involved in the epithelial-to-mesenchymal transition pathways, and extracellular matrix remodeling. Preliminary observations from this single sample revealed that high copper concentrations were localized to regions of active tumor growth and were associated with increased expression of immune response genes. This proof-of-concept workflow demonstrates the feasibility of integrating elemental imaging with spatial transcriptomics to identify metals-based gene correlates. Future application of this workflow to larger patient cohorts will pave the way for expansive comparisons across the metallome and transcriptome, ultimately identifying novel targets for tumor progression biomarkers and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/583ee888bce168b26048a39ba23b693a3c315972" target='_blank'>
              Integration of elemental imaging and spatial transcriptomic profiling for proof-of-concept metals-based pathway analysis of colon tumor microenvironment
              </a>
            </td>
          <td>
            A. Srivastava, N. Shaik, Yunrui Lu, M. Chan, A. Diallo, John P. Zavras, Serin Han, T. Punshon, B. Jackson, L. Vahdat, Xiaoying Liu, V. Mittal, K. Lau, J. Gui, Louis J. Vaickus, J. Hoopes, F. Kolling, L. Perreard, J. Marotti, J. Levy
          </td>
          <td>2025-10-01</td>
          <td>Metallomics: Integrated Biometal Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 H3K27 altered diffuse midline gliomas (DMG) are aggressive brain tumors that arise centrally in the brain. These tumors are not amenable to surgery and no curative therapy exists. DMG exhibits profound heterogeneity, with tumor cells displaying dynamic and interchangeable cellular states that recapitulate aspects of normal development. In parallel, the tumor microenvironment (TME) is highly diverse, further contributing to the complexity of the disease. A better understanding of the DMG ecosystem is needed for therapeutic target identification. In this study, publicly available single cell RNA seq data of DMG (Jessa et al. Nat. Genet. 2022) were analyzed to define key molecular characteristics of DMG. Xenium spatial transcriptomics and multiplex immunohistochemistry were performed on FFPE patient samples collected at diagnosis to study the tumor and TME cells in their native spatial context. In DMG, both the glioma cellular composition and the heterogeneity of the TME varied based on the anatomical location of the tumor. Tumors located in the pons exhibited a high proportion of astrocyte-like glioma cells. In contrast, thalamic DMG showed a greater abundance of oligodendrocyte-like tumor cells, suggesting a regional bias in cellular differentiation states. Furthermore, the TME in thalamic tumors was notably enriched with immunotolerant macrophage populations, expressing molecules associated with an anti-inflammatory, pro-tumoral phenotype. With spatial profiling, distinct tumor niches were identified, showing heterogeneity among different samples and underscoring the complex and spatially organized nature of both tumor cell states and the surrounding microenvironment in DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/408581e61f93671e8e7b1c9fc7f6b7a6669a068a" target='_blank'>
              EPCO-24. DISSECTING THE ECOSYSTEM OF PEDIATRIC H3K27 ALTERED DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            Julie Morscio, Chiara Caprioli, C. González-Blas, Pierluigi Calò, K. Vandereyken, Thierry Voet, R. Sciot, Sandra Jacobs, F. de Smet
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Understanding the cellular origins and early evolutionary dynamics that drive the initiation of carcinogenesis is critical to advancing early detection and prevention strategies. By characterizing key molecular, cellular and niche events at the precancerous tipping point of early gastric cancer (EGC), we aimed to develop more precise screening tools and design targeted interventions to prevent malignant transformation at this stage. We utilized our AI models to integrate spatial multimodal data from nine EGC endoscopic submucosal dissection (ESD) samples (covering sequential stages from normal to cancer), construct a spatial-temporal profile of disease progression, and identify a critical tipping point (PMC_P) characterized by an immune-suppressive microenvironment during early cancer development. At this stage, inflammatory pit mucous cells with stemness (PMC_2) interact with fibroblasts via NAMPT⟶\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\longrightarrow$$\end{document}ITGA5/ITGB1 and with macrophages via AREG⟶\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\longrightarrow$$\end{document}EGFR/ERBB2 signaling, fostering cancer initiation. We established gastric precancerous cell lines and organoids to demonstrate that NAMPT and AREG promote cellular proliferation in vitro. Furthermore, in the transgenic CEA-SV40 mouse model, targeting AREG and/or NAMPT disrupted key cell interactions, inhibited the JAK-STAT, MAPK, and NFκB pathways, and reduced PD-L1 expression, which was also confirmed by western blot in vitro. These interventions delayed disease progression, reversed the immunosuppressive microenvironment, and prevented malignant transformation. Clinical validation was conducted using endoscopically resected EGC specimens. Our study provides a precise spatiotemporal depiction of EGC development and identifies novel diagnostic markers and therapeutic targets for early intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0410e9abe73bbef3e06750644cca2f27fd69ccca" target='_blank'>
              Spatiotemporal multi-omics analysis uncovers NAD-dependent immunosuppressive niche triggering early gastric cancer
              </a>
            </td>
          <td>
            Pingting Gao, Chunman Zuo, Wei Yuan, Jiabin Cai, Xiaoqiang Chai, Ruijie Gong, Jia Yu, Lu Yao, Wei Su, Zuqiang Liu, Shengli Lin, Yun Wang, Mingyan Cai, Lili Ma, Quanlin Li, Pinghong Zhou
          </td>
          <td>2025-09-22</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Intra- and inter-tumoral heterogeneity combined with immunosuppressive tumor microenvironments contribute to the poor outcomes associated with glioblastoma (GBM). Well-characterized immunocompetent models that recapitulate human GBM features are urgently needed to identify targets in the tumor microenvironment and develop novel therapeutics. Here, we used multi-omic approaches to characterize syngeneic mouse brain tumor stem cell lines in vitro and in orthotopically engrafted tumors.



 Whole-genome sequencing, transcriptomics, ATAC-sequencing, and imaging mass cytometry were used to characterize syngeneic brain tumor stem cell lines derived from Trp53+/-/Nf1+/- C57Bl6 mice. Mouse and human bulk, single-cell, and spatial sequencing datasets were analyzed for validation. CRISPR/Cas9 and shRNA were used for gene knockdowns. Tumor growth was investigated using orthotopic engraftment in syngeneic C57Bl6 mice.



 One of the syngeneic lines, mBT0309, generated tumors with histopathological characteristics of GBM. mBT0309 displayed amplification and high expression of Igf2. Copy number gains at the IGF2 locus were observed in human GBM tumors and stem cell lines. Furthermore, we determined that high IGF2 RNA expression is associated with poor survival in GBM patients. Imaging mass cytometry on mBT0309 tumors showed early infiltration of monocyte-derived macrophages, vascularization, and cell states characteristic of human GBM. Genetic targeting of Igf2 decreased in vitro cell growth, improved survival of engrafted mice, and decreased the percentage of Arginase-1+ macrophages in mBT0309 tumors.



 mBT0309 is a valuable syngeneic model for studying immunosuppression and therapeutic resistance in GBM. IGF2 offers promise as a valuable therapeutic target to combat tumor growth and immunosuppression in GBM patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3386883b6908859de06190f6d3a8fc6a7ff95c1" target='_blank'>
              IGF2 supports glioblastoma growth and immune evasion through a combination of tumor cell-intrinsic and -extrinsic mechanisms
              </a>
            </td>
          <td>
            Kyle Heemskerk, Samir Assaf, X. Hao, Shannon Snelling, M. Meode, R. Hassam, Orsolya Cseh, Smriti Kala, James Pemberton, Jennifer A. Chan, J. Cairncross, Peter Forsyth, V. Wee Yong, Reza Mirzaei, Samuel Weiss, F. Zemp, H. Artee Luchman
          </td>
          <td>2025-10-15</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal and immunologically "cold" cancer, with a 5-year survival rate of 13%. Despite the success of immunotherapy in other malignancies, its efficacy in PDAC remains limited due to complex immune evasion mechanisms and a pro-tumorigenic immune microenvironment. Several lines of evidence suggest that PDAC progresses from acinar-to-ductal metaplasia (ADM) to pancreatic intraepithelial neoplasia (PanIN) lesions to invasive cancer, accompanied by increasing immunosuppression. We hypothesize that understanding the spatial and temporal transitions of immune cell states during tumor evolution is key to uncovering mechanisms of immune dysfunction and guiding new therapeutic strategies. Using single-cell RNA sequencing (scRNA-seq) and multispectral imaging, we aim to uncover comprehensive, large-scale, and longitudinal changes in the immune microenvironment during PDAC progression. Here, we utilized the KrasG12D/+; Trp53R172H/+; Ptf1a-Cre (KPC) genetically engineered mouse model, and through longitudinal histological analysis of pancreas, we delineated the trajectory of disease progression, with PanIN I-II lesions appearing by 8–9 weeks, PanIN III by 11–12 weeks, and locally invasive PDAC by 16 weeks of age. To study the temporal evolution of immune cell-state transitions, we performed scRNA-seq on CD45+ enriched immune cells isolated from KPC mouse pancreas collected at 4, 8, 12, and 16 weeks using the 10x Genomics Chromium platform. To examine the spatial dynamics of immune cells, we performed multispectral imaging at defined time points using the Lunaphore COMET platform with a panel of over 15 antibodies, enabling simultaneous visualization of several markers on a single tissue section. In our analysis to date, we identified subsets of pro-tumorigenic macrophages emerging during PDAC development, along with a marked increase in myeloid-derived suppressor cells (MDSCs) in late-stage tumors. We also observed dynamic shifts in other immune cell populations evolving alongside tumor progression. Notably, we uncovered distinct subsets of monocytes, macrophages, and T cells exhibiting tumor-promoting potential, progressively enriched during the advanced stages of PDAC. Using multispectral imaging, we generated a comprehensive spatial map of the immune microenvironment across PDAC progression, capturing patterns of immune exclusion, infiltration, and cell–cell neighborhood interactions. Our study highlights the multi-layered immune dysfunction that evolves from early to late stages of PDAC - features that are challenging to capture in human samples. CD45+ enrichment improved the detection of rare and transient immune subsets by mitigating the masking effects caused by abundant cancer and stromal cells. Together, these findings provide an unbiased and detailed view of the evolving immune landscape in PDAC, laying the foundation for identifying microenvironmental and cell-intrinsic factors that either constrain or promote malignant progression, as well as for developing immune-based therapeutic strategies.



 Atul Verma, Doreen Klingler, Linda Bergmayr, Tanvi V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug. Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0f49d549158981070f5d866ac858fc605d13b" target='_blank'>
              Abstract A106: Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression
              </a>
            </td>
          <td>
            A. Verma, Doreen Klingler, Linda Bergmayr, T. V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="


 Diffuse midline glioma (DMG) is an aggressive pediatric brain tumor with no curative treatment. The tumor microenvironment (TME) of DMG, shaped by its unique anatomical location and developmental context, poses significant therapeutic hurdles. While the TME has been characterized by a predominance of tumor-associated microglia and sparse infiltration of blood-derived immune cells, a comprehensive understanding of immune-tumor cell interactions and how to effectively target them with immunotherapies remains challenging.



 To dissect the DMG TME, we employed a multi-omics approach that integrated single-nucleus RNA sequencing, spatial transcriptomics, and high-dimensional imaging on both patient samples and an immunocompetent in utero electroporation murine DMG model. We characterized the spatial and phenotypic architecture of the tumors and validated a novel immune target using syngeneic mouse models and ex vivo co-culture experiments.



 Our analysis revealed pronounced intratumoral heterogeneity, with two spatially distinct patterns of cancer-immune interaction. A mesenchymal-like niche promoted the recruitment of blood-derived immune cells, whereas another pattern exhibited an immunologically cold phenotype, reminiscent of the immune-privileged healthy brain. Spatially resolved analysis of cell-cell communication within the TME identified the VISTA immune checkpoint pathway as a key regulator of these patterns. Targeting of this axis in a syngeneic mouse model led to significant tumor reduction and 100% survival in mice, highlighting its therapeutic potential. Further validation demonstrated that microglia play a critical role in orchestrating immune cell recruitment and anti-tumor activity, and that modulating the VISTA pathway can reshape the immune response.



 Our findings uncover the spatial and cellular complexity of the DMG immune microenvironment and identify the VISTA pathway as a critical modulator of immune suppression. By demonstrating robust therapeutic efficacy upon VISTA blockade, this study lays a strong foundation for developing targeted immunotherapies in DMG and opens new avenues for overcoming immune resistance in pediatric brain tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3d8c86dfadea2de1a37fb282c8875e761a9bc8" target='_blank'>
              JS07.7.A VISTA IS A CRITICAL AND TARGETABLE IMMUNE CHECKPOINT PATHWAY IN DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            R. Collot, C. Ruiz-Moreno, C. Honhoff, A. Wezenaar, H. Ariese, H. Johnson, B. M. Vervoort, J. Bunt, E. Bokobza, H. G. Rebel, B. T. Te Pas, P. Robe, M. Kool, M. Kranendonk, D. V. van Vuurden, E. Hulleman, E. J. Wehrens, A. Zomer, H. Stunnenberg, A. Rios
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a fundamental role in tumor progression. Cancer cells interact with their surroundings to establish a supportive niche through structural changes and paracrine signaling. Cells around transformed tumor cells contribute to cancer development, while infiltrating immune cells in this aggressive TME often become exhausted. Solid tumors, especially the most invasive types such as pancreatic ductal adenocarcinoma, are notably stiff mechanically, with cross-linking enzymes significantly affecting the survival of cancer cells in both primary tumors and metastatic sites. In this review, we highlight recent key contributions to the field, focusing on single-cell sequencing of stromal cells, which are increasingly seen as highly heterogeneous yet classifiable into distinct subtypes. These new insights enable the development of effective co-treatment approaches that could significantly enhance current and novel therapies against the most aggressive cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5047b2e8e1a1022055daa398bbad8bc791fe908c" target='_blank'>
              Microenvironment of Solid Tumors.
              </a>
            </td>
          <td>
            T. Korneenko, Nikolay B. Pestov, M. Shakhparonov, Nickolai A. Barlev
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in bioscience</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Metastasis remains the leading cause of cancer-related death, yet the biological determinants that enable tumor cells to disseminate and colonize distant organs are incompletely understood. Emerging evidence identifies the microbiome, not merely as a bystander, but as an active architect of the metastatic cascade. Microbial communities residing in the gut, mucosal barriers, and within tumors shape metastatic progression by modulating immune surveillance, stromal remodeling, oncogenic signaling, and therapy response. Intratumoral and even intracellular microbes regulate epithelial–mesenchymal transition, angiogenesis, and immune escape, while gut-derived metabolites condition pre-metastatic niches and alter systemic immunity. Technological advances in spatial transcriptomics, single-cell multi-omics, and metagenomics have revealed a spatially organized, functionally integrated microbial ecosystem within tumors, challenging long-held assumptions of sterility in cancer biology. This review synthesizes five converging dimensions of this paradigm: microbial interactions in the metastatic tumor microenvironment; microbiome-mediated immunoediting and metastatic escape; the role of intratumoral and intracellular bacteria in dissemination; spatial-multi-omic approaches to map microbial niches; and microbial biomarkers predictive of metastasis and therapy outcomes. Collectively, these findings recast the microbiome as a critical and targetable determinant of metastasis. Deciphering the tumor–microbe–host triad holds transformative potential for biomarker development, therapeutic innovation, and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15720d6be643a5b1bcd7bdc88b245af42661e241" target='_blank'>
              Microbial signatures in metastatic cancer
              </a>
            </td>
          <td>
            Jhommara Bautista, María Paula Fuentes-Yépez, Joseth Adatty-Molina, Andrés López-Cortés
          </td>
          <td>2025-09-30</td>
          <td>Frontiers in Medicine</td>
          <td>3</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Motivation Tumor clonal evolution represents a dynamic ecosystem underpinned by genetic alterations and Darwinian selection, posing major challenges due to intratumoral heterogeneity and therapy resistance. Although computational methods have advanced significantly, current tools often focus on single data modalities, leaving important gaps in modeling spatial and non-genetic evolution. This review systematically surveys and assesses algorithmic progress across diverse study designs to identify key limitations and future directions. Results We systematically evaluate over 20 computational tools across four study designs—cross-sectional, regional bulk, single-cell, and lineage tracing—and perform benchmarking of seven comparable tools. Multiomics integration approaches are shown to improve phylogenetic inference, yet challenges remain in mutation ordering and polyclonal detection. A novel spatiotemporal framework is proposed to link phylogenetic branch lengths with spatial transcriptomic gradients. Future efforts should prioritize multimodal data integration, scalable computational architectures, and clinically applicable models to translate evolutionary insights into precision oncology. Availability and implementation This review provides a comprehensive survey and benchmarking of existing methods. The code and data used to generate the benchmarking figures and results are available at https://github.com/zlsys3/review_benchmark/tree/main/figure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d2b14817118e93c14f870881445bd16f81118f7" target='_blank'>
              Computational strategies in tumor phylogenetics: evaluating multimodal integration and methodological trade-offs across study designs
              </a>
            </td>
          <td>
            Chenghan Jiang, Zhe Wang, Ruoyu Wang, Shanshan Liang, Shuai Tao
          </td>
          <td>2025-10-01</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive brain cancer with a poor prognosis and is challenging to study due to its high degree of inter- and intra-tumoral heterogeneity and complex tumor microenvironment. Current in vitro models of GBM, including patient-derived organoids and 2D and 3D cell cultures can recapitulate aspects of the GBM biology, but interactions with normal cells of the central nervous system remain challenging to model. Herein, we introduce a new 3D biomimetic brain cancer microtissue (BCM) developed by co-culturing rat cortical microtissues with rat glioma cell lines. Leveraging this model, we have characterized glioma cell motility, invasiveness, and interactions with neurons, astrocytes, and microglia. GBM cell behavior in the BCM model is consistent with that observed in vivo. This robust and versatile platform will enable future high-throughput therapeutic screening and detailed insights into primary glioma biology and potentially other brain tumors and cancers that metastasize to the brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84026343603df03139d750fc53943115b170da32" target='_blank'>
              A brain cancer microtissue model for studying tumor cell and neural cell interactions
              </a>
            </td>
          <td>
            Dominick Calvao, Francesca Vecchio, Ahbid Zein-Sabatto, Monica Ocitti, Rachel Vidomlanski, Andrea Schmidt, Michelle Shub, Haneesh Kesari, Mariano S. Viapiano, Sean E. Lawler, Diane Hoffman-Kim
          </td>
          <td>2025-10-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Most meningiomas are benign, slow-growing tumors. However, CNS WHO grades 2 and 3 meningiomas display significantly higher aggressiveness and poorer prognosis. Recent investigations into the molecular biology of meningiomas have greatly increased our understanding of these tumors. Furthermore, emergent spatial transcriptomic sequencing methods now provide an unprecedented opportunity to study molecular mechanisms of tumor development within a spatial context and on a single-cell level. This study aims to uncover specific microenvironmental features, cell-cell interactions and transcriptional regulatory mechanisms present in different molecular classes and subclones of meningiomas.



 We employed single-nuclei RNA sequencing and spatial transcriptomic profiling on a matching set of meningiomas (n=22). We developed a method based on non-negative matrix factorization (NMF) to analyze intra-tumor heterogeneity from both single-cell and spatial transcriptional profiles across samples. Trajectory analysis was performed to screen for expression signatures selectively up-regulated around specific morphological features of the tumors. We clustered cells into types of niches based on compositions of their spatial neighborhoods.



 We annotated cell types in the tumor microenvironment (TME), and identified cellular states of meningioma tumor cells in single-cell and spatial transcriptomic data. Tumors of different subclasses display drastic differences in spatial cell state compositions, suggesting distinct modes of microenvironmental organization. We identified conserved patterns of tumor-TME cell state colocalization and interaction. Benign NF2-mutant meningiomas are characterized by extensive infiltration of myeloid cells which trigger a series of immune and stress response pathways in neighboring tumor cells. Among more malignant tumors, a decrease of immune cell populations is coupled with a marked shift in the TME toward tumor-supportive phenotypes. Moreover, we observed reduced tumor-infiltrative capabilities among tumor-associated myeloid cells and the formation of compartmentalized spatial niches of interacting tumor cells. Additionally, various tumor transcriptional signatures related to extracellular matrix remodeling, fibroblastic markers, and meningothelial signatures were mapped to distinct morphological structures across samples.



 Our results reveal that the meningioma TME is heterogenous at the cell state level, with tumor cells and normal cells forming niches that have varying functions in tumor pathophysiology and characterize molecular classes of meningiomas. A series of corresponding RNA markers and spatial signatures were identified, which could potentially be used to increase precision in diagnosis and stratification of meningiomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb7c99c73e50db41072f3f2a1080df073fd8ef43" target='_blank'>
              OS13.6.A A SINGLE-CELL AND SPATIAL ATLAS OF MENINGIOMA HETEROGENEITY ACROSS TUMOR CLASSES
              </a>
            </td>
          <td>
            Y. Tang, R. Banan, S. Maas, D. Calafato, R. Michael, J. Kada Benotmane, R. Yan, G. Rukhovich, Z. Ferdinand, V. Venkataramani, A. Suwala, I. Bludau, S. Krieg, W. Wick, A. von Deimling, D. Heiland, M. Gerstung, B. Patel, F. Sahm
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the most lethal brain tumor in adults with remarkable cellular plasticity and complex anatomical features. While single-cell RNA sequencing (scRNA-seq) has revealed its cellular heterogeneity, the spatial cellular organization within the anatomical features remains elusive.



 Here, we provide a comprehensive spatial transcriptomic dataset comprising 115,914 transcriptomes from 32 genotyped tissue sections.



 Leveraging Xenium in situ technology, we generated spatial transcriptomic data with subcellular-resolution, significantly improving the granularity of our understanding of GBM cellular architecture. In addition, our analyses identified oligodendrocyte subtypes specifically localized to tumor cores and perivascular regions with molecular profiles associated with poor prognosis.



 This publicly accessible GBM atlas provides a valuable resource for understanding the complex cellular architecture of GBM and facilitating the development of novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1717e2c5370026ee08f9c770891fbab0aaa81409" target='_blank'>
              P02.08.A A SPATIALLY RESOLVED HUMAN GLIOBLASTOMA ATLAS REVEALS DISTINCT CELLULAR AND MOLECULAR PATTERNS OF ANATOMICAL NICHES
              </a>
            </td>
          <td>
            H. Park, P. Sonpatki, C. Park, W. Park, N. Shah
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Endothelial cells are ubiquitously present in the human body and line the luminal surface of blood and lymphatic vessels. The oxygen-dependence of cells impacts their proximity to blood vessels, and consequently, to endothelial cells depending on their functional properties and priorities. This paper presents cell-to-nearest-endothelial-cell distance distributions for various cell types using 399 spatially resolved omics datasets from 14 studies comprising 12 tissue types with a total of 47,349,496 cells. Additionally, we developed an open-source web-based interactive tool, Cell Distance Explorer, that allows researchers to interactively visualize cell graphs and linkages in 2D and 3D datasets. Finally, we present a hierarchical neighborhood analysis focused on the endothelial cell neighborhoods in small and large intestine datasets. This paper provides an open-access resource (datasets, tools, and analyses) to characterize and compare cell distances and cell neighborhoods in spatially resolved omics data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b8d116f12142b4777b553dbbc51f1021b9f8b2" target='_blank'>
              Exploring endothelial cell environments across organs in spatially resolved omics data
              </a>
            </td>
          <td>
            Yashvardhan Jain, J. Jepson, Roy Chen, Elizabeth Maier, B. Herr, Aleix Puig-Barbe, Ellen M. Quardokus, Danial Qaurooni, Clarence Yapp, Samuel L. Ewing, Archibald Enninful, Negin Farzad, Andreas Bueckle, Quinn T. Easter, B. Matuck, Chenchen Zhu, Emma Monte, J. Purkerson, Matthew Jehrio, Ravi S. Misra, Rong Fan, Fiona Ginty, A. Karunamurthy, Jean Fan, Martha Campbell-Thompson, Gloria S. Pryhuber, K. Byrd, John W. Hickey, Katy Börner
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Spatial tumour heterogeneity, which denotes the changes in cellular and molecular attributes across distinct locations within a tumour, significantly influences cancer diagnosis and treatment resistance. The heterogeneity of tumour cells inside a singular mass facilitates tumour development, metastasis, and the ineffectiveness of standard therapy. Comprehending the geographical distribution of tumour cells is crucial for formulating more efficient treatment regimens. Diverse methodologies are employed to investigate spatial heterogeneity, encompassing modern imaging techniques such as MRI, PET, and multiplexed imaging, alongside omics approaches including genomes, transcriptomics, and proteomics. These instruments offer insights into the tumour microenvironment and facilitate the identification of resistant subpopulations. The amalgamation of imaging and genomic data via radiogenomics has emerged as a viable methodology, providing an extensive perspective on the spatial and molecular intricacies of tumours. Principal findings reveal that spatial heterogeneity fosters medication resistance by establishing microenvironments characterised by varying oxygen levels, immunological infiltration, and genetic alterations, hence complicating the efficacy of monotherapy strategies. Hypoxic environments and immunological evasion significantly contribute to treatment resistance. Addressing geographical heterogeneity has the potential to enhance cancer treatments. By analysing the molecular and geographical characteristics of tumours, physicians can customise therapies more efficiently, minimising resistance and improving therapeutic results. This methodology signifies a vital advancement in precision medicine, providing more individualised and efficacious cancer therapies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/691858c0b45e6f3db66c28fcbc5355f8258e6fea" target='_blank'>
              Spatial Tumor Heterogeneity: The Next Frontier in Understanding Cancer Resistance
              </a>
            </td>
          <td>
            Kwesi Egyin Taylor, Hycent Jacob, T. Oladosu, Godwin Kenechukwu Nwajiugo, Motunrayo Victoria Adigun, Markus Saerimam Nzunde, C. Ugo
          </td>
          <td>2025-10-18</td>
          <td>Oncology, Nuclear Medicine and Transplantology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d38817463882bea59fae7ae417ac7b28c60ff93" target='_blank'>
              Multi-omics evaluation of cell lines as models for metastatic prostate cancer
              </a>
            </td>
          <td>
            Xueying Liu, Weixing Yu, Xiuyuan Jin, Yugang Wang, Ke Liu
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic non-small cell lung cancer (mNSCLC) cells carry heterogeneity, not only among different subtypes but also within a single tumor. Most evidence suggests that mNSCLC exploits specific molecular drivers and mechanisms to maintain physiology, metabolism, and immune evasion during tumorigenesis. Genome-wide association studies also revealed particular mutations in the oncogenic drivers supporting tumor cell proliferation and survival, resulting in aggressive and drug-resistant phenotypes of mNSCLC. While significant progress has been made in understanding mNSCLC at the genetic and molecular levels, a considerable gap remains in understanding the dynamic interplay between intrinsic factors—particularly key tumor-associated cells—and tumor immune microenvironment (TIME) during metastasis. Hence, this review highlights histological and genetic characteristics, emphasizes the clinical relevance of metastasis, and the roles of tumor-associated cells in shaping the immunosuppressive tumor microenvironment (TME) in mNSCLC. Understanding these intricate features and mechanisms is crucial for identifying novel therapeutic targets and improving strategies to combat mNSCLC progression in diagnosed patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/931263caf4b3388e1caa012daaf5acb9be29f764" target='_blank'>
              Unraveling the molecular-pathological characteristics and cellular complexity of the tumor immune microenvironment in metastatic non-small cell lung cancer
              </a>
            </td>
          <td>
            S. Bharadwaj, J. M. Mierzwicka, Lucie Vaňková, Petr Malý
          </td>
          <td>2025-09-29</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 The diffusely infiltrative growth pattern of glioblastoma (GBM) is a major obstacle to effective therapy. A complex network of tumor intrinsic and microenvironmental features interact to shape the landscape of this growth pattern, but deciphering the molecular substrates of these interactions remains challenging. While single cell RNA sequencing has reshaped our understanding of GBM heterogeneity and uncovered cellular phenotypes of many cellular populations in these tumors, the spatial context of this heterogeneity could not be assessed with earlier technologies. By contrast, spatial transcriptomics platforms have the potential to map this transcriptome-wide information in situ. We recently reported the use of this technology to develop a layered model of spatial architecture in GBM and showed that hypoxia represents a key organizing feature of GBM heterogeneity. However, the limited resolution of this platform precluded assessment of whole-transcriptome features at truly cellular resolution. In this work, we now present our findings utilizing the next generation of this technology and showcase the ability to map whole transcriptome data in spatial context at true single cell resolution in primary patient GBM samples. We explore heterogeneous GBM phenotypes in the context of GBM spatial microarchitecture. Using copy number inference, we demonstrate the ability to assess clonal heterogeneity in situ and map the pattern of invasion of unique GBM clones. Finally, we discuss our progress to discover organizing principles underlying distinct patterns of GBM cell invasion. Taken together, we demonstrate the potential of single cell spatial transcriptomics to decipher the molecular heterogeneity of GBM microenvironment in situ and uncover the biology driving the aggressive infiltrative growth of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad815e1440b141545b8a8eee62737b92edef994" target='_blank'>
              TMIC-68. Deciphering the spatial architecture of transcriptional heterogeneity in glioblastoma at single cell resolution
              </a>
            </td>
          <td>
            C. Mount, Zebin Wen, Alissa C. Greenwald, S. Kovatsis, Jack Lu, D. Gerovasilis, I. Tirosh, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype; however, clinically approved prognostic biomarkers and therapeutic options remain limited. This study aimed to investigate tumor dependency genes to identify novel therapeutic targets for TNBC. Methods Tumor dependency genes for TNBC were identified using the The Cancer Dependency Map (DEPMAP) database. The TCGA-BRCA dataset was utilized to analyze the expression, survival associations, and pathway enrichment of these genes. Single-cell datasets were employed to explore cellular trajectories and biological functions within tumor dependency gene-associated cell subpopulations. Genomic sequencing was used to investigate the somatic mutational landscape influencing the infiltration abundance of the tumor dependency-associated subpopulation. The METABRIC dataset assessed the impact of the tumor dependency-associated subpopulation on radiotherapy, chemotherapy, and combination therapy outcomes. Potential drugs were identified using the Connectivity Map (CMAP). Colony formation experiment and the CCK-8 experiment were performed to validate the biological function of gene. Results Four tumor dependency genes (TDGs) were identified. These genes were highly expressed in TNBC and associated with poor prognosis. Enrichment analysis revealed their significant involvement in cell cycle-related pathways. Single-cell analysis demonstrated that the tumor dependency-associated subpopulation (TDAS), defined by these four genes, resided at the differentiation terminus of epithelial/tumor cells and was linked to energy metabolism and cell proliferation pathways. Crucially, patients with high TDAS infiltration abundance were found to be unsuitable for surgery alone and should receive combined radiotherapy or chemotherapy. Potential therapeutic agents targeting the TDAS were screened. And in vitro experiments confirmed the cell proliferation role of candidate genes. Conclusion This study identifies four potential TNBC biomarkers for assessing TDAS abundance, providing novel insights and strategies for personalized TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c8aa1f75b0d7e0c49aabd5f19b6bf7360c1cf" target='_blank'>
              Integrating single-cell transcriptomics and whole-genome CRISPR CAS9 screen identifies a cell cluster associated with tumor dependency in triple-negative breast cancer
              </a>
            </td>
          <td>
            Su Liu, Shuo Wang, Guixin Wang, Yingxi Li, Zhigang Zhao, Yao Tian, Jun Cao
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The global burden of cancer is rising, with treatment failures often due to the metastatic nature of late-stage malignancies. Circulating tumour cells (CTCs) are metastatic precursors shed from primary tumours, which survive in circulation, extravasate and colonise distant organs. The advent of high-throughput single-cell RNA sequencing (scRNA-seq) has revolutionised the investigation of transcriptomic landscape at single-cell resolution, enabling deep transcriptomic profiling, re-stratifying CTC subtypes and improving the detection of rare new subpopulations. Applications extend to understanding tumour microenvironments, characterising cellular heterogeneity, uncovering metastasis molecular mechanisms and improving prognosis and diagnostic strategies. A timeline of key milestones in CTC scRNA-seq research is also provided. Nevertheless, a knowledge gap remains due to unstandardised protocols and fragmented resources in CTC scRNA-seq research. We address this gap by proposing a 12-step CTC-specific scRNA-seq workflow to overcome methodological inconsistencies. This workflow spans the entire process from enrichment, single-cell sorting and sequencing to data pre-processing and downstream analyses, with a detailed compilation of data analysis tools. An in-depth discussion of the pros and cons of commonly used scRNA-seq tools is also included, specifically evaluating their suitability for CTC research. Additionally, emerging research frontiers, including the discovery of hybrid cells—fusion products of tumour and normal cells—and the integration of machine learning (ML) into scRNA-seq workflows, are explored. Future research should prioritise CTC scRNA-seq workflow standardisation, integrate ML-driven analysis and investigate rare and hybrid populations to advance metastasis research. This review supports these goals by guiding methods, informing tool selection and promoting data sharing for reproducibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d2d946c12a3009915b3b21748cb811af18ab2d4" target='_blank'>
              Single-cell RNA-sequencing of circulating tumour cells: A practical guide to workflow and translational applications
              </a>
            </td>
          <td>
            Francis Yew Fu Tieng, Learn-Han Lee, N. Ab Mutalib
          </td>
          <td>2025-10-06</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c7fc26d95585d6b0934d32560acffc12bc3781e" target='_blank'>
              Region-specific human brain organoids reveal synaptic and cell state drivers of glioblastoma invasion
              </a>
            </td>
          <td>
            Tarun N. Bhatia, Saisrinidhi Ganta, Mostafa Meselhe, Caitlin Sojka, A. Martija, Lisa Nieland, Uriel Rufen-Blanchette, Anson Sing, Alexia King, Brain Organoid Hub, A. Bhaduri, Kimberly Hoang, E. Nduom, Renee D. Read, Jeffrey Olson, Steven A. Sloan
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 The tumour microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC) is comprised of heterogenous malignant epithelium and diverse immune infiltrates within a dense, fibrotic stroma. This multi-compartmental TME includes myeloid and lymphoid lineages, cancer-associated fibroblasts (CAFs), mural and endothelial cells, and tumour cells embedded in a rich extracellular matrix (ECM). Efforts to profile and perturb compartments or cellular phenotypes in isolation have often led to unintended adverse effects on neighbouring cells, compromising drug efficacy. This drives the need for an integrated, spatially-resolved understanding of the architecture and molecular features underlying the organization of the TME. To address this, we leveraged single-cell RNA sequencing (scRNAseq) data from eight distinct datasets to design three custom multiplexed imaging mass cytometry (IMC) panels against tumour, immune, and stromal targets. These were applied to three serial sections of a tissue microarray of 221 PDAC patients. Segmentation and clustering of 7.9 million cells revealed 83 distinct cell types and their functional states, including biophysical responses. Advanced image alignment across serial sections enabled the incorporation of TME-wide features and resolved eight spatially recurrent PDAC microenvironments neighbouring phenotypically distinct epithelial ducts. Two microenvironments, ECM-rich and immune-suppressed, were associated with worse overall survival and frequently co-occurred with a third, stiff matrix, characterized by stroma enriched in phospho-myosin light chain 2 expressing CAFs. In contrast, the immune infiltrated stroma microenvironment was linked to improved clinical outcomes and often co-occurred with CD105+ fibrovascular and immune infiltrated microenvironments. Matched whole-genome sequencing (n = 182/221) revealed that patients dominant in ECM-rich regions harboured 12p11.21 (KRAS) and 8q24.21 (MYC and POU5F1B) amplifications, and 17q22 (RNF43) deletions, and stiff matrix were associated with 13q33.3 (LATS2) deletion. To link spatial niches to proteomic signatures, we performed whole-slide IMC-guided laser capture microdissection and mass spectrometry, and further integrated differentially expressed proteins with transcriptomes from an external scRNAseq cohort (n = 163). ECM-rich and immune suppressed regions showed signatures of mechanically induced immune suppression, including NETosis and innate immune surveillance via pattern recognition receptors. Contrastingly, immune infiltrated microenvironments were enriched for cytotoxic activity and immune trafficking programs, while immune infiltrated stroma displayed signatures associated with oxidative stress responses. These findings define unique spatial ecosystems within the PDAC TME that are linked to specific genomic alterations, functional states and clinical outcomes, highlighting their therapeutic relevance and potential as targets to overcome the challenges of treating PDAC.



 Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Cassandra J. Wong, Brendon Seale, Zhen Yuan Lin, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steven Gallinger, Hartland W. Jackson. Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B115.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e942fb4c97db1a6398873aeae226bf3656a085" target='_blank'>
              Abstract B115: Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, A. Borgida, Cassandra J. Wong, Brendon Seale, Z. Lin, Edward L. Y. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 The human brain contains a rich milieu of extracellular matrix (ECM) components that promote normal physiology and are often dysregulated in pathologies including the malignant cancer glioblastoma (GBM). We have used in situ single-cell spatial transcriptomic platforms to map the expression patterns of nearly 400 ECM-related genes in multiple GBM samples (IDH1 wild type). Our analysis identifies at least four different GBM cell populations with unique ECM expression profiles that show spatial enrichment in distinct intratumor regions. Spatial mapping also demonstrates largely non-overlapping expression signatures of various ECM components in stromal cell types, particularly in GBM vascular endothelial cells and reactive microglia/macrophages. Comparisons of GBM versus lower-grade II and III astrocytoma samples (IDH1 R132H) identifies differential expression of key ECM components, including elevated levels of select ECM glycoproteins (IGFBP2 and MGP) and ECM-affiliated proteins (ANXA1 and ANXA2). In addition, we detect spatially enriched expression of COL8A1 (collagen), LUM (proteoglycan), and POSTN (ECM glycoprotein) in perivascular stromal cells in GBM but not in lower grade tumors. Computational analysis of putative ligand-receptor interactions reveals novel communication networks between cancer cells and stromal components, particularly in regions of GBM microvascular proliferation and pseudopalisading necrosis. In summary, this comprehensive spatial map provides new insights into ECM control of tumor initiation and progression and identifies potential therapeutic targets within the GBM microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e40eadeccedca2699836ba1458071cfd535dcbfb" target='_blank'>
              TMIC-33. Single Cell Spatial Profiling of Matrisome Gene Expression in Glioblastoma
              </a>
            </td>
          <td>
            Joseph H. McCarty, Arpan De, Santiago A. Forero, Ali Pirani, John Morales
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 PDAC is the most hypovascular solid tumor, imposing harsh constraints on tumor metabolism. Cancer cells cope with poor access to vascular-derived nutrients and O2 by cooperating with abundant stromal cells, receiving alanine, proline, glycosaminoglycans, branched chain ketoacids, lysophosphatidic acid, and unsaturated fatty acids from fibroblasts. However, the extent of metabolic support provided by other cell types in the PDAC TME is unexplored. We reasoned that metabolically cooperative cell types would be spatially positioned near cancer cells in these tumors, and that cooperative spatial arrangements would be enriched under conditions of relative hypovascularity. We first performed Xenium spatial transcriptomics on a cohort of 8 treatment-naive human PDAC resections and generated a spatial atlas of nearly 700,000 cells across a range of expected cell types. Niche analysis uncovered 17 recurrent cellular neighborhoods in the data, including an endothelial-enriched niche. We find that tumor-associated neutrophils form dense aggregates within malignant glands in tumors with low endothelial content, and form in tumor regions which are most distal from vasculature. In a separate cohort of 4 additional PDAC tumors, the expression of neutrophil chemokines CXCL1/2/3/5 was highest in a subpopulation of malignant cells distant from endothelium. These data suggest metabolic stress secondary to poor vascularity drives recruitment of neutrophils via the CXCL1/2/3/5 axis. Indeed, when exposed to Tumor Interstitial Fluid Media (TIFM), cancer cells upregulated the transcription and/or secretion of these cytokines and G-CSF. Since the formation of neutrophil aggregates is driven by cancer cell nutrient derangement, we hypothesized that cancer cells could cooperate metabolically with neutrophils to alleviate nutrient stress driven by poor vascularity. Neutrophil co-culture promoted cancer cell proliferation under TIFM, but not RPMI, while under lethal combination TIFM and serum starvation, co-culture enabled enhanced survival of cancer cells. Lastly, to understand what metabolic processes could be fueled by neutrophil aggregates in pancreatic cancer, we designed a fully custom spatial transcriptomics panel targeting ∼360 metabolic enzymes and profiled 4 additional PDAC cases, performing dispersed whole-transcriptome scFFPEseq on an adjacent tissue section. We integrated data across the spatial and dispersed modalities to enable spatial projection of whole-transcriptome-derived information. Cancer cells located near neutrophil aggregates had significantly enhanced expression of vitamin B6 salvage pathway, suggesting that one critical shared metabolite between neutrophils and cancer cells is the B6 precursor pyridoxine. In total, we have deployed multiple spatial and single cell RNA profiling technologies to uncover a novel node of metabolic crosstalk in pancreatic cancer between neutrophils and cancer cells. Future work is needed to confirm transfer of specific metabolites and the downstream sequelae of their provision for tumor growth.



 Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M Celeste. Simon. Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec70e5707c093ceceed418e17d0f9e9a3b6c60c" target='_blank'>
              Abstract B127: Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer
              </a>
            </td>
          <td>
            Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M. C. Simon
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Cancer-associated fibroblasts (CAFs) are critical mediators of tumor-stroma crosstalk in pancreatic ductal adenocarcinoma (PDAC). While bulk and single-cell transcriptomic studies have uncovered CAF heterogeneity, the spatial organization of CAF subtypes and their physical interactions with tumor epithelial subtypes remain incompletely understood. Here, we integrate bulk, single-cell, and spatial transcriptomic datasets to systematically interrogate the molecular, cell, and spatial interactions between tumor-promoting CAFs (proCAFs), tumor-restraining CAFs (restCAFs), and basal-like versus classical tumor subtypes across multiple biological scales. Ligand-receptor analysis of single-cell RNA-seq data revealed robust signaling between proCAFs and basal-like tumor cells, particularly through integrin-mediated (COL11A1–ITGA11) pathways, suggesting subtype-specific stromal-epithelial crosstalk within distinct tumor microenvironments. Histopathologic validation via immunofluorescence staining confirmed that proCAF regions are preferentially located adjacent to tumor epithelium and are characterized by reactive, myxoid stroma, whereas restCAF regions exhibit a fibrous, hypocellular morphology. To further elucidate spatial relationships, we performed 10x Genomics Visium spatial transcriptomics on seven primary PDAC tumors. Application of the DeCAF classifier to Visium spots demonstrated a distinct radial gradient of proCAF expression, peaking at the tumor-stroma interface and diminishing with distance—indicative of spatial colocalization with basal-like programs. Notably, proCAF gene signatures were specifically enriched in stromal regions adjacent to basal-like tumor zones, while classical tumor regions were spatially associated with heterogeneous CAF compositions. This subtype-specific spatial organization and paracrine signaling may underlie the clinical observation that basal-like tumors are significantly more likely to co-occur with proCAF stroma (64.5%, p = 2.17e–5). Importantly, patients harboring both basal-like and proCAF subtypes exhibited the poorest clinical outcomes (median overall survival: 11.0 months), compared to those with classical/restCAF profiles (median overall survival: 30.4 months, p < 0.001). Collectively, these multidimensional data define spatially organized and functionally interactive tumor-stroma niches in PDAC. Our findings highlight a co-evolutionary relationship between proCAFs and basal-like tumor cells that contributes to an aggressive disease phenotype, and underscore the potential of spatially resolved tumor-stroma profiling for improving patient stratification and guiding therapeutic targeting of the tumor microenvironment.



 Xianlu Laura. Peng, Ian McCabe, Elena Kharitonova, Ryan Zhao, Changfei Luan, Arthi Hariharan, Jaewon Lee, Nancy Kren, Yi Xu, Joseph Kearney, Justin Su, Michelle LaBella, Susan Tsai, Brian Belt, Roheena Panni, David Linehan, Yuliya Pylayeva-Gupta, Alina Iuga, Naim Rashid, Jen Jen Yeh. Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B100.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af6682e5a69c7a44f5aef0626bbff15428b91a16" target='_blank'>
              Abstract B100: Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications
              </a>
            </td>
          <td>
            X. Peng, Ian C McCabe, Elena V Kharitonova, Ryan T Zhao, C. Luan, Arthi Hariharan, Jaewon J Lee, Nancy P. Kren, Yi Xu, Joseph F Kearney, Justin Su, M. LaBella, Susan Tsai, Brian Belt, R. Panni, David Linehan, Y. Pylayeva-Gupta, Alina C Iuga, Naim U Rashid, J. J. Yeh
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5994676ec22291e594d0de0ef39f5ccc905b6ab" target='_blank'>
              Integrative analysis of spatiotemporal transcriptomics delineates dynamic cell states in squamous tumorigenesis
              </a>
            </td>
          <td>
            Varsha Sreekanth, Timur Rusanov, Farah Yousef, Mark A. Sanborn, Su Yeon Yeon, Vijayalakshmi Ananthanarayanan, Olga Karginova, Jalees Rehman, A. Salahudeen
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Glioblastoma (GB) is a particularly aggressive brain tumor which is characterized by a low overall survival due to rapid formation of neuron-tumor-networks. The neural epigenetic subtype was linked to increased tumor-neural connectivity but the mechanisms underlying this integration remain poorly understood. Here, we aim to investigate the impact of the microenvironmental ecosystem on neuron-tumor synaptogenesis leveraging a human cortical slice model and single cell spatially resolved transcriptomic analysis.



 A human cortical slice model was employed to visualize and assess tumor-neuronal interactions using advanced retrograde tracing techniques in human derived cell lines (n = 12). This system was based on an EnvA-pseudotyped G-protein-deleted rabies virus. Spatial transcriptomics was conducted using multiplexed error-robust fluorescence in situ hybridization analysis (MERFISH). Additionally, we conducted immunohistochemical staining and confocal microscopy.



 Spatial transcriptomic evaluation of neural high and low tumors revealed distinct microenvironmental ecosystems within the infiltrating rim of the tumors. Neural-high tumors exhibited a higher presence of synaptogenesis-inducing inflammatory myeloid cells driving neuronal synaptogenesis. Retrograde tracing of neural high vs low cell lines confirmed increased synaptic density and neuronal coupling in the neural high phenotype, which were associated with higher invasiveness, supported by stRNA-seq, which identified genes linked to synaptogenesis and connectivity.



 Neural-high GB shows enhanced neuronal connectivity and invasiveness, contributing to accelerated tumor progression, which underlines the prognostic value of epigenetic tumor profiling and opens ways for novel therapeutic approaches.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ab32a7b8c0ed1e91d6c055b2bda26b2946992d4" target='_blank'>
              P02.15.B GLIOBLASTOMA IN THE CONNECTOME: NEURAL PHENOTYPES PREDICT TUMOR INTEGRATION AND CONNECTIVITY
              </a>
            </td>
          <td>
            L. Menzl, G. Villa, Y. Yabo, V. Venkataramani, F. Ricklefs, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is characterized by profound intratumoral heterogeneity and cellular plasticity, which drive therapeutic resistance. While molecular subtyping into classical and basal-like states has provided a framework, the biology of cells transitioning between these phenotypes remains poorly understood. This study aimed to characterize and validate a transitional, co-expressor cell state at the single-cell, spatial, and proteomic levels to elucidate mechanisms of subtype plasticity. We employed an integrative, multi-platform approach combining single-cell RNA sequencing (scRNA-seq), multiplex immunohistochemistry (mIHC), and laser microdissection with mass spectrometry-based proteomics (LMD-MS) to investigate tumor cells co-expressing the classical marker GATA6 and the basal-like marker KRT17. scRNA-seq analysis revealed that a significant population of tumor cells co-expresses GATA6 and KRT17, existing along a transcriptional continuum between the classical and basal-like poles. Pseudotime trajectory analysis identified a path from the GATA6+ classical state towards the KRT17+/GATA6+ transitional state, governed by the upregulation of pathways including MYC signaling and epithelial-mesenchymal transition (EMT). To validate these findings spatially, mIHC confirmed the presence of KRT17+/GATA6+ double-positive cells, frequently located at the interface between well-differentiated, GATA6-rich glandular structures and poorly differentiated, KRT17-rich infiltrative fronts. Furthermore, LMD-MS was used to provide proteomic confirmation. By isolating pure populations of GATA6+ and double-positive cells, we confirmed that transitional cells are enriched in proteins associated with metabolic reprogramming and cellular motility. Our findings demonstrate that PDAC plasticity is not a simple switch but a dynamic continuum involving a tangible transitional cell state. These co-expressor cells, with their unique metabolic and signaling dependencies, represent a key driver of tumor progression and a potential reservoir for therapeutic resistance. Targeting the specific vulnerabilities of this transitional phenotype offers a promising strategy to overcome plasticity, prevent relapse, and improve therapeutic outcomes in PDAC.



 Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, Natalia Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth Shroyer. Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B122.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a10f45ec5eb895ff99dbba41e30b63f2ad6188" target='_blank'>
              Abstract B122: Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, N.Ye Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth R. Shroyer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ovarian cancer represents a typically immune “cold” tumor, where obvious immunosuppression, spatial T-cell exclusion, and cellular dysfunction collectively limit immunotherapy effectiveness. Especially in high-grade serous ovarian carcinoma (HGSOC), the immune low-response state is driven by complex interactions among tumor-associated macrophages (TAMs), suppressive stromal networks, and the T-cell compartment (regulatory T cells, Tregs, and exhausted effector T cells). Emerging multi-omics technologies—particularly single-cell RNA sequencing and spatial transcriptomics—have showed the heterogeneity and spatial immune organization underlying this suppressed state. Here, we integrate these datasets to describe TAM phenotypes and spatial niches, T-cell exhaustion, Tregs accumulation, NK-cell dysfunction, and stromal barriers that enforce exclusion. We then derive phenotype-guided combination strategies to remodel the tumor microenvironment and improve responsiveness to immune checkpoint blockade. This synthesis provides a concise, multi-dimensional framework for precision immunotherapy and for overcoming resistance in immune-low ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880f6c405f773eb38fd078c68256074438417fa2" target='_blank'>
              Decoding immune low-response states in ovarian cancer: insights from single-cell and spatial transcriptomics for precision immunotherapy
              </a>
            </td>
          <td>
            Yalan Yan, Jiaan Lu, Huanyu Luo, Zizhang Wang, Ke Xu, Lexin Wang, Qin Wang
          </td>
          <td>2025-09-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d74e30f87b2d3e252ed12332d86f09f04feb4c91" target='_blank'>
              CTDP: Identifying cell types associated with disease phenotypes using scRNA-seq data
              </a>
            </td>
          <td>
            Chonghui Liu, Murong Zhou, Zhidong Wang, Chen Yang, Yan Zhang, Zhongjun Jiang, Yu-Hang Yin, Zeyu Luo, Tianjiao Zhang, Guohua Wang, Lei Yuan
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Tumor resident microbes are a well-recognized component of the tumor microenvironment. Microbial subcellular location within different tumor types along with their functionality remains to be determined. Bulk microbial profiling techniques, however, lack subcellular and spatial transcriptomic resolution. To address these limitations, we developed advanced quantitative fluorescent imaging methodologies that allowed for visualization of microbial cellular compartmentalization across different tumor types. Furthermore, we performed spatial microbial transcriptomics at the regional and single cell levels to interrogate microbial regulation of signaling within pancreatic cancer cells. To confirm presence of viable microbes within tumors, we performed multiplexed culturomics of patient tissue samples followed by Whole Genomic Sequencing (WGS) analysis. We used in vitro co-culture assays together with in vivo animal experiments for functional biological validation. Overall, our results identified several pathways under microbial regulation within cancer cells that can drive immune evasion through impaired antigen presentation. In summary, using multiple complimentary novel methodologies we characterize the microbial niche of tumors (MiNT) that uncovers microbial regulation of host cells signaling and reshaping of the surrounding tumor microenvironment.



 Vidhi Chandra, Le Li, Seyda Baydogan, Haoyue Liu, Javier Gomez, Steven B. Maron, Daniel Carvajal-Hausdorf, Erick M. Riquelme, Mark W. Hurd, Anirban Maitra, Luis Diaz, Ismet Sahin, Jared Burks, Michael P. Kim, Florencia McAllister. Functional interrogation of the pancreatic intratumoral microbial niche reveals tumor modulation and reshaping of the microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B011.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9d435ae53244dd8930ff7ec47bc24ea3f53880a" target='_blank'>
              Abstract B011: Functional interrogation of the pancreatic intratumoral microbial niche reveals tumor modulation and reshaping of the microenvironment
              </a>
            </td>
          <td>
            Vidhi Chandra, Le Li, Seyda Baydogan, Haoyue Liu, Javier A Gomez, S. Maron, Daniel Carvajal-Hausdorf, Erick Riquelme, M. Hurd, Anirban Maitra, Luis A. Diaz, I. Sahin, Jared K. Burks, Michael P Kim, Florencia McAllister
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary Cervical biofluids collected during routine Pap smears contain a mixture of epithelial and immune cells that reflect cervical health and disease. This study demonstrates proof-of-concept that multicolor flow cytometry can be applied to these samples to identify and characterize cell populations in normal cervical samples as well as in different stages of cervical cancer. Our analysis identified major cell populations, including different types of epithelial cells and immune cells, and revealed that their composition within the samples changes with disease stage. We observed variability in both cellular composition and marker expression, highlighting the need for additional markers and larger patient cohorts. Nonetheless, this minimally invasive approach could potentially be used to monitor tumor cells, immune responses, and therapeutic outcomes over time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/565b70c9fdb10e2ce7ca145493da61430604f525" target='_blank'>
              Noninvasive Cell Population Profiling of Normal and Dysplastic Cervical Biofluids by Multicolor Flow Cytometry as a Promising Tool for Companion Diagnostics
              </a>
            </td>
          <td>
            Christoph Berger, Wolf Dietrich, Manuela Richter, Florian Kellner, Christian Kühne, Katharina Strasser
          </td>
          <td>2025-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pleomorphic adenoma (PA), the most common benign salivary gland tumor, harbors unpredictable risks of recurrence and malignant transformation into carcinoma ex pleomorphic adenoma (CXPA), posing significant clinical challenges. To better delineate the tumor transformation trajectory, we performed single-cell RNA sequencing of normal salivary gland, primary PA, recurrent PA (rPA), and CXPA. Cell trajectory reconstruction, differential expression gene identification, and key gene network analysis were integrated to characterize molecular transitions and intercellular crosstalk driving PA recurrence and malignant transformation. Immunohistochemistry was used to validate key findings. GALNT13 + myoepithelial cells were identified as CXPA-specific malignant progenitors, delineating early malignant conversion. Concurrently, MIF + myoepithelial cells exhibited enhanced tissue-destructive capabilities. Fibroblasts enforced fibrotic restraint in primary PA and drove extracellular matrix degradation in CXPA. The tumor microenvironment exhibited stage-specific adaptations, with CXPA favoring pro-inflammatory MIF-CD74/CD44 signaling and rPA adopting immunosuppressive traits. Stromal reprogramming and immune-editing dynamics collectively orchestrated microenvironmental adaptation, linking cellular heterogeneity to clinical aggressiveness. This study provides the first comprehensive molecular atlas of PA-to-CXPA transformation, revealing malignant specialization of the myoepithelial subpopulation, fibroblast-mediated stromal reprogramming, and immune-editing driven microenvironmental adaptation. These findings provide a framework for precision stratification of the malignant potential of PA, while positioning microenvironmental intervention as a cornerstone of future clinical strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/110371b54a7af37b631e968b70c953daf9f4053b" target='_blank'>
              Single-cell transcriptomics reveals cellular evolution underlying pleomorphic adenoma recurrence and malignant transformation
              </a>
            </td>
          <td>
            Yike Li, Zhixuan Li, Guile Zhao, Qi Han, Zhangfan Ding, Cheng Wang, Chunjie Li
          </td>
          <td>2025-10-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="High-throughput clonal tracing of primary human samples relies on naturally occurring barcodes, such as somatic mitochondrial DNA (mtDNA) mutations detected via single-cell ATAC-seq (mtscATAC-seq). Fresh-frozen clinical specimens preserve tissue architecture but compromise cell integrity, thereby precluding their use in multi- omic approaches such as mitochondrial genotyping at single-cell resolution. Here, we introduce Cryo-mtscATAC-seq, a broadly applicable method for diverse pathophysiological contexts to isolate nuclei with their associated mitochondria (“CryoCells”) from frozen samples for high-throughput clonal analysis. We applied Cryo-mtscATAC-seq to the neurodegenerated human brain, glioblastoma (GBM), pediatric neuroblastoma, and human aorta, and implemented mitobender, a computational tool to reduce ambient mtDNA in single-cell assays. Our approach revealed regional clonal gliogenesis and microglial expansions in amyotrophic lateral sclerosis (ALS), persistence of oligodendrocyte progenitor cell (OPC)-like clones in GBM recurrence, mtDNA depth heterogeneity after neuroblastoma chemotherapy, and oligoclonal proliferation of smooth muscle cells in human aorta. In conclusion, Cryo-mtscATAC-seq broadly extends mtDNA genotyping to archival frozen specimens across tissue types, opening new avenues for investigation of cell state- informed clonality in human health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c60dee046cd06ab111daf9f8ce07ff1e2d339ed7" target='_blank'>
              Cryo-mtscATAC-seq for single-cell mitochondrial DNA genotyping and clonal tracing in archived human tissues
              </a>
            </td>
          <td>
            Maren Salla, B. Obermayer, Marie Cotta, Ekaterina Friebel, Juliana Campo-Garcia, Georgia Charalambous, Roemel Jeusep Bueno, Dustin Lieu, Patryk Dabek, Ashley Helmuth, G. Tellides, Roland Assi, Katrin Bankov, Marco Lodrini, H. Deubzer, D. Beule, Hattie Chung, Helena Radbruch, David Capper, Frank L. Heppner, Sarah C. Starossom, Caleb A. Lareau, Ilon Liu, Leif S. Ludwig
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous malignancy, with its tumor immune microenvironment (TIME) playing a crucial role in tumor progression, immune evasion, and treatment response. Despite insights from single-cell technologies, challenges such as small samples and inconsistent annotations have limited a comprehensive understanding. Thus, a large-scale, high-resolution single-cell atlas is needed to fully explore the heterogeneity of CRC TIME and guide personalized treatment. We analyzed single-cell transcriptomic data from 100 CRC samples (371,223 cells). Immune, stromal, and epithelial cells in the TIME were clustered, annotated, and functionally analyzed. Transcription factor networks were inferred using SCENIC, differentiation states assessed with CytoTRACE, and cell–cell communication analyzed via CellPhoneDB. TIME subtypes were constructed from bulk RNA-seq data, followed by survival analysis. SCRN1 was further validated through in vitro experiments. This study presents a comprehensive single-cell CRC atlas, revealing significant heterogeneity in T cells, B cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells within the TIME. We defined two immune ecological subtypes: subtype 1 (enriched in metabolic and motility pathways with poor prognosis) and subtype 2 (enriched in immune response pathways, with better prognosis and greater immunotherapy potential). High SCRN1 expression in malignant epithelial cells correlated with poor prognosis and metastasis. In vitro experiments confirmed the role of SCRN1 in promoting CRC cell proliferation and migration. This study provides a comprehensive characterization of the CRC TIME, links TIME subtypes to prognosis, and identifies SCRN1 as a potential therapeutic target, offering insights for CRC precision subtyping and immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c5be539fad2c0073ae6e62159eaef1e83a1ef" target='_blank'>
              The single-cell atlas of colorectal cancer reveals cellular heterogeneity and key molecular features of the tumor immune microenvironment
              </a>
            </td>
          <td>
            JunHu Li, ZhiQiang Liu, BoYu Zhang, Liuping You, Yuenan Huang
          </td>
          <td>2025-10-10</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08fb4029b669b488e3f7a2bf959803989f3d7dae" target='_blank'>
              Single-Cell RNA and ATAC Sequencing Reveal Cellular Heterogeneity and Chromatin Accessibility Dynamics in Young Chinese Breast Cancer Patients across Pre- and Post-Neoadjuvant Therapy
              </a>
            </td>
          <td>
            Xue Han, Bin Wang, Wenting Xiang, Fan Lin, Bo Sun, Beibei Yang, Zhihan Ruan, Shasha Liu, Shuo Li, Hong Liu, Jian Liu
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566fc9fd9c1a3c438b8fdc4f08cbb7f2ae09af61" target='_blank'>
              Joint analysis of expression and variation at single cell resolution by scVar
              </a>
            </td>
          <td>
            Wei Zhao, Fei Yang, Wenting Zong, Xinchang Zheng, Yiming Bao
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy with significant cellular heterogeneity, which poses a significant challenge for treatment. We integrated three single-cell RNA sequencing (scRNA-seq) datasets to construct an expression matrix, identify cell types, and assess copy number variation (CNV) scores, with a particular focus on ductal cells. Our study found that ductal cells are the primary source of malignant cells in PDAC and identified a high-malignancy subtype—ductal cell-3—where RACK1 was the most significantly differentially expressed gene. Through the analysis of public and clinical data, along with RT-qPCR and immunohistochemistry validation, we confirmed the high expression of RACK1 in PDAC tumors, which correlated with lymph node metastasis, clinical staging, histological differentiation, and prognosis. Further investigation revealed that RACK1 promotes PDAC cell proliferation, migration, and invasion. Sequencing analysis indicated that RACK1 inhibits the FOXO signaling pathway, promotes epithelial-mesenchymal transition (EMT), and facilitates the progression and metastasis of PDAC. This study provides new insights into the role of RACK1 in PDAC and its underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fac327fd52a6595ab14fc1ae00b1f2b7a0bcb1e" target='_blank'>
              Molecular mechanisms of RACK1-driven metastasis in pancreatic ductal adenocarcinoma revealed by single-cell RNA sequencing
              </a>
            </td>
          <td>
            Hangyu Liu, Tingxin Wang, Yuying Cui, Biao Zhang, Dong Shang, Huiyi Song
          </td>
          <td>2025-10-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Seminoma is the most common solid malignant tumor of the testis in young males, significantly impacting fertility. Approximately 20% of seminomas metastasize, markedly increasing recurrence risk and compromising quality of life. To investigate the poorly understood mechanisms driving seminoma metastasis, we performed a comprehensive analysis by integrating single-cell RNA sequencing, TCGA data mining, and cell biology assays. We identified significant intratumoral heterogeneity. A subset of tumor cells expressing DPPA4 and PSMA7 showed high stemness, enhanced self-renewal, and association with metastasis. Knockdown of these genes reduced sphere formation, tumor migration and proliferation in Tcam-2 and NCCIT cells. Conversely, tumor cells overexpressing PAGE5 and SAT1 exhibited reduced stemness, migratory capacity and proliferation. In the immune microenvironment, we identified IFNG+ T cells, which recruit and activate other antitumor immune cells. These cells secrete IFN-γ, which promotes tumor differentiation, reduces stemness, and mitigates tumor aggressiveness. Based on these findings, we further developed a molecular panel to aid in the identification of seminomas with a higher risk of metastasis. In conclusion, our study identifies unique molecular signatures that facilitate risk stratification based on metastatic potential, providing valuable insights for improving precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a200ee992d63f7f5d78cc885c7a3fb857907df52" target='_blank'>
              Single-cell analysis unravels divergent gene signatures shaping seminoma stemness and metastasis
              </a>
            </td>
          <td>
            Zhouliang Bian, Biying Chen, Jiahui Guo, Jiamin Zhang, Guangye Du, Shufang He, Haihua Yuan, Yue Zhou, Bin Jiang, Daliu Min, Dingwei Ye, Hengchuan Su, Yanjie Zhang
          </td>
          <td>2025-11-07</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) poses a great challenge for therapeutics development due to its heterogeneity and the lack of immunocompetent models that faithfully recapitulate human GBM. We sought to generate and characterize syngeneic mouse models for major GBM subtypes.



 Postnatal electroporation using the PiggyBac transposon/transposase for overexpression (OE) and CRISPR/Cas9 for knockout (KO) was used to generate models with common GBM alterations including EGFRvIII-OE/Cdkn2a-KO/Pten-KO, Nf1-KO/Pten-KO/p53-KO, and RasV12-OE/Cdk4-OE/p53-KO. Model characterization was done by HE staining, immunofluorescence staining, RNA sequencing and flow cytometry.



 All models displayed histopathological and molecular features of GBM such as pseudopalisiding necrosis, microvascular proliferation, and cellular atypia, as well as Gfap and Nestin expression. On the transcriptomic level, Nf1/Pten/p53 tumors showed enrichment for the human mesenchymal signature, while RasV12/Cdk4/p53 and EGFRvIII/Cdkn2a/Pten tumors resembled the classical and proneural signatures. Each tumor was comprised of four cellular states, i.e., astrocytic, mesenchymal, OPC-like, and NPC-like, suggestive of intratumoral heterogeneity. On examining the microenvironment, EGFRvIII/Cdkn2a/Pten tumors had a higher relative abundance of T cells (48.6% vs 6.3%, p=0.0084, 48.6% vs 8.8%, p=0.0376,), NK cells, (1.2% vs 0.4%, p=0.1425, 1.2% vs 0.19%, p=0.0315) and dendritic cells (7.5% vs 0.4%, p=0.0272) than the Nf1/Pten/p53 and RasV12/Cdk4/p53 models. RasV12/Cdk4/p53 tumors had greater infiltration of monocyte-derived macrophages than Nf1/Pten/p53 (35.9% vs 4.18%, p=0.0012) and EGFRvIII/Cdkn2a/Pten (35.9% vs 10.4%, p=0.0620) tumors as well as an elevated CD4+/CD8+ T cell ratio. In the Nf1/Pten/p53 model, microglia were the most abundant cells within the CD45+ compartment, surpassing levels observed in EGFRvIII/Cdkn2a/Pten (60.2% vs 4.6%, p=0.0548) and RasV12/Cdk4/p53 (60.2%vs7.5%, p=0.0564) tumors.



 We developed an array of syngeneic mouse models of GBM that recapitulate human inter- and intra- tumoral heterogeneity. Future studies will define the functional states of immune cells in the microenvironment and evaluate the impact of immune cell composition on the response to immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9fb466bb8d2e828409bfc681044d64171c2f26" target='_blank'>
              TMOD-40. Genetic drivers shape tumor phenotype and composition of the tumor microenvironment in syngeneic mouse models of glioblastoma
              </a>
            </td>
          <td>
            J. Pyczek, Shannon Snelling, Xueqing Lun, Bo Young Ahn, Heewon Seo, Miranda Yu, Daniela Quail, A. Morrissy, Jennifer A Chan
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Pancreatic cancer, specifically pancreatic ductal adenocarcinoma (PDAC), is notorious for its aggressive nature and dismal prognosis, ranking as a leading cause of cancer-related mortality worldwide. Despite advancements in surgical techniques, chemotherapies, and novel therapeutic approaches, the overall survival rates remain bleak, primarily due to the complex and dynamic tumor microenvironment (TME) which fosters cancer progression and therapeutic resistance. Central to the TME are cancer-associated fibroblasts (CAFs), which exhibit significant heterogeneity and play multifaceted roles in tumor progression, ranging from remodeling the extracellular matrix to modulating immune responses. Recent breakthroughs in single-cell sequencing and spatial omics have unraveled the diverse functional states and spatial organization of various cell types within the TME, including distinct subtypes of CAFs which differ vastly in their influence on pancreatic cancer pathophysiology. This review delves into the intricate interactions between CAFs and pancreatic cancer cells, highlighting how metabolic reprogramming and immune regulation by CAFs contribute to the high malignancy of PDAC. By integrating single-cell and spatial resolution data, we offer new insights into the metabolic crosstalk and immune landscape remodeling driven by CAFs, establishing a foundation for targeting these stromal interactions in therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2add2bd80d02d372a76aab262dddc13647852fe6" target='_blank'>
              Metabolic and immune crosstalk between cancer-associated fibroblasts and pancreatic cancer cells
              </a>
            </td>
          <td>
            Qiyao Zhang, Z. Cao, Shangcheng Yan, Bingrong Chen, Hao Wu, Hongtao Cao, Chen Lin, Ziwen Liu
          </td>
          <td>2025-10-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive and lethal adult brain tumor. The cellular heterogeneity within the tumor microenvironment (TME) plays a critical role in the complexity of treatment and poor survival. GBM is typically classified into 3 molecular subtypes—Classical, Mesenchymal, and Proneural—associated with EGFR, NF1, and PDGFRA genetic drivers, respectively. Our team previously generated human GBM-specific driver mutations using RCAS/tv-a gene transfer system to create immunocompetent genetically engineered mouse models (GEMMs) of murine GBM (mGBM) that share the same cell-of-origin. While we and others have shown that these GEMMs closely mimic myeloid cell composition and bulk expression profiles observed in human GBM (hGBM) subtypes, it is critical to identify species-specific similarities and differences in tumor microenvironment (TME) heterogeneity at cellular resolution. Hence, we reasoned that single-cell characterization of mGBM models and comparison to hGBM would highlight the cross-species similarities/differences and enable investigators to identify and validate the most relevant human target candidates. Towards this goal, we performed single-cell RNA sequencing (scRNA-seq), multicolor flow cytometry, and in-depth computational analysis on EGFR-, Nf1-, and PDGFB-driven GEMMs to dissect the effect of common GBM genetic drivers on the TME. To assess the human relevance, we compared our findings to bulk RNA-seq data from human GBM tumors with mutually exclusive EGFRvIII deletion, NF1 loss of function mutation, or PDGFRA amplification on chromosome 4. Our analysis revealed that PDGFB-driven tumors were more proliferative and enriched for Wnt signaling interactions, while EGFRvIII-driven tumors showed an elevated interferon signaling response. Moreover, Nf1-silenced tumors displayed higher myeloid abundance, myeloid immunosuppressive interactions involving Osteopontin, Treg infiltration, and expression of immune checkpoint molecule Ctla4. Overall, we established a human-mouse analytical platform for genotype-aware target discovery and validation, which offers promising new avenues for more effective, personalized treatments in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edea93ae20a51718ffae751a4a4ae63a990d752e" target='_blank'>
              TMIC-24. Single-cell decoding of the genotype-immunophenotype relationship in glioblastoma
              </a>
            </td>
          <td>
            N. Soni, Kavita Rawat, Zhihong Chen, Angela DiMauro, Bruno Giotti, D. Hambardzumyan, A. Tsankov
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Tumor progression is driven not only by the complex milieu of transcriptional cell states of distinct lineages that populate the tumor microenvironment (TME) but also by the interactions these states forge with one another. Systematically charting the landscape of such prognostic cell-state interactions (CSIs) remains difficult as single-cell datasets are small and rarely linked to clinical outcomes, whereas bulk profiles obscure cellular context. We developed CSI-TME, a computational pipeline that adapts the concept of synthetic lethality to infer prognostic cell-state interactions by directly utilizing the large cohorts of bulk transcriptomic datasets. We applied CSI-TME to IDH-mutant gliomas, deriving a highly reproducible cell-state interaction network (CSIN) that is predominantly pro-tumor and differentially activated in IDH-mut astrocytoma versus oligodendroglioma. Malignant cells within the CSIN resemble multiple neuronal lineages including, astrocyte-like and oligodendrocyte-progenitor-like program, and reveal key interactions between glioma stem cells and T cells. CSIN stratifies patient response to immune-checkpoint blockade therapy. Roughly 20% of interactions involve direct ligand-receptor communication, and their participating cell states significantly co-localize in spatial-transcriptomic datasets, most notably for a pro-tumorigenic interaction between tip-like endothelial cells and hypoxic malignant cells involving multiple ligand-receptor interactions. Integration with somatic-mutation profiles reveals that many interactions in CSIN exert anti-tumor effect early-on in glioma but shift toward pro-tumor roles as the disease progresses. Taken together, CSI-TME provides a unique way of identifying the prognostic and clinically relevant cell state interactions without the need for extensive single cell datasets and provides a framework for prioritizing therapeutic ligand-receptor targets, patient stratification, while providing new insight into how interactions among the cell states shape the TME in IDH-mutant glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3146cb53ad109ab0b9473e62c6d93d13f03288e3" target='_blank'>
              TMIC-77. Identification of Clinically Relevant Cell State Interactions in the Tumor Microenvironment of IDH-Mut Glioma
              </a>
            </td>
          <td>
            Arashdeep Singh, Bharati Mehani, Vishaka Gopalan, Ken Aldape, Sridhar Hannenhalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is an incurable and aggressive brain cancer marked by profound intra-tumoral heterogeneity. Malignant cells exist in four core transcriptional states: stem-like (OPC-like and NPC-like) and more differentiated astrocytic (AC-like) or mesenchymal (MES-like) states. Using spatial transcriptomics, we previously identified a layered tumor structure of these states, with hypoxia emerging as an organizing driver, implicating oxygen availability as a central factor in cell state dynamics. However, how these cellular states interact with each other and with the tumor microenvironment remains incompletely understood.



 To uncover state-specific vulnerabilities for reducing GBM cell state diversity, we conducted a small-molecule screen in gliomasphere models using over 1,600 cysteine-reactive covalent inhibitors, assessing their effects on cell states via RNA-seq. We then employed the Sonar metabolic reporter to visualize NAD+ and NADH levels in gliomaspheres. Through experiments with orthotopic xenografts in mice and co-culture with human cortical organoids, we characterized distinct redox profiles across GBM cell states by single-cell and bulk RNA-seq. Spatial single-cell transcriptomics and mass spectrometry imaging (MSI) metabolomics on adjacent sections of frozen patient samples were used to identify preferential metabolic programs in specific cell states.



 Our small-molecule screen identified compounds that promote differentiation and are enriched for NRF2 activation, a master regulator of the antioxidant response. Spatial single-cell transcriptomics and mass spectrometry metabolomics experiments confirmed that MES-like cells exhibit a heightened glycolytic signature, whereas stem-like states preferentially depend on oxidative phosphorylation (OXPHOS) for their energetic needs. Finally, pharmacologic perturbations revealed that mitochondrial and glycolysis inhibition selectively deplete different cell state populations in gliomaspheres.



 Our findings provide deeper insight into the metabolic underpinnings of GBM cell states and support the rationale for combinatorial therapies aimed at reducing intratumoral heterogeneity that exploit their divergent dependencies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e25475481794c097c1d8706111cced1dc0e56d4b" target='_blank'>
              TMET-30. Uncovering the metabolic programs underlying malignant cell state heterogeneity in glioblastoma
              </a>
            </td>
          <td>
            Alexander Jucht, Maolin Ge, Rony Chanoch-Myers, Florian Ruiz, Gerard Baquer, Alissa C. Greenwald, Min Yang, Toshiro Hara, Yilin Fan, Channing Pooley, D. Gerovasilis, Charles Couturier, Nathalie Y R Agar, I. Tirosh, L. Bar-Peled, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d4e23f0e2eb9b451cd70e81ca9b082a2f8b6469" target='_blank'>
              Advancements in single-cell sequencing for cervical cancer research.
              </a>
            </td>
          <td>
            Congli Pu, Biyuan Xing, Shujie Wang, Zhao Liu, Yingchao Zhao
          </td>
          <td>2025-11-06</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Background Pancreatic ductal adenocarcinoma (PDAC) is an exceptionally lethal malignancy, with high percents of patients presenting with liver metastases (LM). However, the mechanisms driving liver metastases remain critical bottlenecks requiring urgent exploration. Objective To identify the key cellular subsets driving PDAC liver metastases, elucidate their interactions with the metastatic microenvironment, and define the underlying mechanisms of liver colonization. Materials and Methods Integrated single‐cell transcriptomic analysis was performed using scRNA‐seq data of PT and LM. The expression of signature genes within the identified cell subset was validated using clinical samples from PDAC PT and LM patients. Furthermore, ligand‐receptor network analysis was conducted between the specific tumor cell subset and key immune cells. Results We identified a novel liver‐enriched metastatic subset (LEMS), a terminally differentiated malignant cell subpopulation characterized by metabolic reprogramming and hyperactivation of immunosuppressive pathways. We further validated the LEMS signature genes, oxidized low‐density lipoprotein receptor 1 (OLR1) and solute carrier family 7 member 7 (SLC7A7), as potential diagnostic biomarkers for liver metastases. Importantly, we found that SPP1+ macrophages interacted with LEMS via ligand‐receptor networks, thereby driving invasion and immune evasion. Discussion We revealed the highly malignant features of LEMS and crosstalk between LEMS and SPP1+ macrophages in liver metastases. However, it is necessary to expand clinical cohorts and in vivo models to comprehensively elucidate the specific mechanistic interactions between LEMS and macrophages. Conclusion We delineated LEMS as an enriched subset in LM and proposed targeting of LEMS‐SPP1+ macrophage interactions as a therapeutic strategy to disrupt metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a6cf6b6420fa24913d08de22b378b6164ad08ae" target='_blank'>
              Single‐Cell Transcriptomic Analysis Identifies a Novel OLR1 + SLC7A7 + Liver‐Enriched Metastatic Subset With Immunometabolic Rewiring in Pancreatic Cancer
              </a>
            </td>
          <td>
            Xuan Yang, Xinyuan Chen, Zixin Wang, Yanfang Liu, Huiting Hu, Qingru Wu, Hailing Zhang, Yu Xiong, Xin Li, Xiaotao Cheng, Xiaoyu Ruan, Yan Gu
          </td>
          <td>2025-11-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Liver metastases represent a leading cause of mortality in pancreatic ductal adenocarcinoma (PDAC). Recent clinical evidence has revealed distinct immune ecosystems in the liver tumor microenvironment, yet mechanistic insight into the cellular and molecular mechanisms driving those diverse immune landscapes remains unclear. Here we report, computational cellular signaling analysis of single-cell RNA sequencing data from matched primary and liver metastatic samples isolated at early and late time points from the spontaneous genetically engineered mouse model KPC (Pdx1Cre; KrasLSL-G12D; Tp53LSL-R172H). This analysis uncovered several immunomodulatory axes between cancer cells-T cells and cancer cells- macrophages. Validation of these findings were performed using primary tumor cell lines, isolated from the same or littermate tumors as the scRNA-seq dataset. Bulk RNA-seq analysis of these cell lines showed a retention of cancer-derived ligands in culture. In vivo validation using the hemisplenic liver metastatic transplant model confirmed that cell lines isolated from primary tumors, which metastasized spontaneously to the liver, retained liver metastatic outgrowth potential. Flow cytometry and multiplex cytokine profiling of isolated cell lines, non-involved liver, and liver tumors revealed distinct immune population shifts within the metastatic niche and confirmed cancer cell derived immunomodulatory cytokines. This study highlights the interplay between cancer-intrinsic signaling and the immune landscape in PDAC liver metastases and supports future mechanistic exploration.



 Ayushi Mandloi, Jace Baines, Meet Patel, Christina R. Larson, Robert S. Welner, Julienne L. Carstens. Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A081.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fce5d9cdc25fe3d365bf9aa1c45eddedfbe9968" target='_blank'>
              Abstract A081: Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth
              </a>
            </td>
          <td>
            Ayushi Mandloi, Jace Baines, Meet Patel, Christina R Larson, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72005e917b50c2e5c4b78fb42f30196a93fe8580" target='_blank'>
              Transcriptional Profiling of Commonly Used Liver Cancer Cell Lines Reveals Disease-Specific Modeling Potential and Authentication Concerns
              </a>
            </td>
          <td>
            Nima Sarfaraz, Gail R. Rosen, Trevor Winkler, Risha Patel, Srinivas Somarowthu, Michael Bouchard
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive disease with one of the lowest five-year survival rates of all cancers. Low grade, premalignant pancreatic intraepithelial neoplasias (PanINs) are common in the general population, but despite expression of mutant KRAS, the vast majority of these lesions will never progress to cancer. Understanding how additional genetic or epigenetic changes as well as changes to the tumor microenvironment contribute to PanIN to PDAC progression is critical and has the potential to reveal mechanisms of interception for this deadly disease. Our previous work identified the prolyl isomerase PIN1 as a critical factor in the tumor microenvironment for growth of a subset of pancreatic cancer cell lines. Loss of PIN1 in the tumor microenvironment restricted in vivo growth of multiple KPC cell lines. We identified distinct fibroblast phenotypes in the absence of microenvironmental PIN1 expression or function. These changes in fibroblast state drive significant changes in the extracellular matrix and growth factor expression, both of which impact the crosstalk to and behavior of the neoplastic cells. We are continuing to use conditional PIN1 knockout mice to interrogate the specific cell types responsible for the observed tumor-restrictive phenotype and are using single cell sequencing to better understand how changes to individual cell types affect multiple cell populations to inhibit tumor growth. In addition, we are comparing the fibroblast phenotypes observed in our mouse models to those found in tumor-promoting versus tumor-restrictive microenvironments in human tissues. These tissues include autopsy pancreata with low grade PanIN lesions as well as PDAC resection specimens with low and high grade PanINs. We are creating organoid and cell line models of epithelial and stromal cells from normal human pancreata to support complex co-culture engineered models of early disease progression. By integrating across mouse and human model systems, we strive to understand the crosstalk that permits progression of pancreatic cancer and to identify targets to intercept this process.



 Tana Gazdik, Madeline Kuhn, Chloe Bowman, Eric Carlson, Kyra Lindley, Ellen M. Langer. Interrogating microenvironmental changes that impact early pancreatic cancer development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4475acff7b0cbed38743a8484c9c3b1b2adb82c" target='_blank'>
              Abstract A031: Interrogating microenvironmental changes that impact early pancreatic cancer development
              </a>
            </td>
          <td>
            Tana Gazdik, Madeline R Kuhn, Chloe L Bowman, Eric J Carlson, Kyra A Lindley, Ellen M. Langer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Although non-muscle-invasive bladder cancer (NMIBC) frequently recurs and progresses despite BCG therapy, cellular mechanisms behind this remain unclear. Understanding the dynamics within the tumor microenvironment (TME) and identifying pathways associated with BCG resistance are crucial for improving NMIBC treatment. To elucidate transcriptional dynamics and key cellular interactions in the TME, we performed longitudinal single-nucleus RNA sequencing of 9 NMIBC cases with three pairs of treatment-naïve and disease progression samples. The resulting 58,037 nuclei were subject to cell type and subtyping annotations along with cell–cell interaction analyses. Our analysis revealed marked interpatient heterogeneity in malignant cells, accompanied by copy number alterations associated with disease progression. TGF-β signaling emerged as a key pathway, showing gradual enrichment from pre-treatment to post-progression samples. Within the tumor microenvironment, we identified distinct subtypes including LAMP3 + dendritic cells and iCAF fibroblasts that followed unique pseudotime trajectories linked to progression. Cell–cell interaction analysis further highlighted critical ligand–receptor pairs such as DSC2–DSG2 and ENG–BMPR2, which were associated with poor clinical outcomes. These interactions may represent hypothesis-generating candidates that warrant further validation. This study highlights key cellular and molecular mechanisms underlying bladder cancer progression Post-BCG treatment. Through single-nucleus RNA sequencing, we identified critical TME subtypes, TGF-β signaling involvement, and crucial ligand-receptor interactions such as DSC2-DSG2 and ENG-BMPR2, which may serve as preliminary biomarkers requiring confirmation in larger independent cohorts. These findings offer valuable insights into enhancement of treatment strategies and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b994449406e351724039c2b4ed65a466872b5c" target='_blank'>
              TGF-β signaling and tumor microenvironment dynamics in bladder cancer progression post-BCG therapy: a longitudinal single-nucleus RNA-seq study
              </a>
            </td>
          <td>
            Seo-Young Lee, , Tae-Min Kim, U-Syn Ha
          </td>
          <td>2025-11-10</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality worldwide. However, the spatial and temporal dynamics underlying its development remain poorly characterized. This study employs spatial transcriptomics (ST) to investigate the progression of intestinal tumors in APC Min/+ mice across multiple time points. We identified distinct transcriptional profiles between tumor and normal tissues, resolving six major cell types through integrated dimensionality reduction and pathological annotation. Pseudo-time trajectory analysis revealed increased expression of MMP11 and MYL9 in later stages of tumor progression. Analysis of human CRC cohorts from the TCGA database further confirmed that high expression of these genes is associated with advanced clinical stages and promotes tumor proliferation and invasion. Temporal gene expression dynamics indicated enrichment of cancer-related pathways concurrent with suppression of lipid and amino acid metabolism. Notably, genes in the DEFA family were significantly upregulated in normal tissues compared to tumor tissues. Functional validation showed that DEFA3 inhibits colon cancer cell migration and proliferation in vitro. These demonstrate the value of ST in resolving spatiotemporal heterogeneity in CRC and identify both MMP11/MYL9 and DEFA3 as potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a441608f72da7375d3ddfdc034535ffa428b9ffc" target='_blank'>
              Investigating the Development of Colorectal Cancer Based on Spatial Transcriptomics
              </a>
            </td>
          <td>
            Zhaoyao Qi, Guoqing Gu, Huanwei Huang, Beile Lyu, Yibo Liu, Wei Wang, Xu Zha, Xicheng Liu
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Breast cancer brain metastases are associated with a poor prognosis, and there is a high demand for improved personalized treatment strategies. It is still poorly understood which breast cancer brain metastases patients benefit from immune checkpoint blockade.



 Here, we multimodally characterized the immune niche of breast cancer brain metastases at single-cell resolution in a clinically annotated patient cohort (n = 156), combining multiplexed imaging and cell type quantification by tissue cytometry, single-nuclei RNA-sequencing, flow cytometry, spatial transcriptomics, and functional studies in patient-derived organoids.



 Overall, infiltration of breast cancer brain metastases by CD8 T cells was strongly heterogeneous between patients, and high numbers were prognostically favorable. In-depth characterization of cell types and integrative analyses including clinical data revealed distinct immune landscapes across patients, two of which independently predicted prolonged overall survival: i) Breast cancer brain metastases containing T and B cells accumulating together with characteristic cell communities in stromal tertiary lymphoid structures, and ii) breast cancer brain metastases characterized by a high content of tissue resident-like CD8 memory T cells (CD8 TRM-like cells) spatially enriched within tumor islands. CD8 TRM-like cells exhibited high levels of activation, checkpoint, and functional markers and showed tumor cell killing capacity ex vivo, suggesting them to be critical players of tumor immune control.



 Together, our work provides novel insights into anti-tumor immunity in breast cancer brain metastases and reveals prognostic immune biomarkers with potential translational relevance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb6c7f193a5bf8137e26dc814e6139f92f208a4" target='_blank'>
              KS03.5.A DISSECTING THE CELLULAR ARCHITECTURE OF BREAST CANCER BRAIN METASTASES REVEALS PROGNOSTICALLY DISTINCT IMMUNE LANDSCAPES
              </a>
            </td>
          <td>
            L. Jassowicz, F. Feng, R. Warta, F. Sor, J. Soller, A. Wöllner, C. Lotsch, J. Wong, J. Schlegel, L. Frank, P. Dao Trong, G. Jungwirth, A. Nohman, K. Maaß, M. Barthel, A. Unterberg, S. Krieg, K. Rippe, A. Abdollahi, A. von Deimling, J. Malm, P. Lichter, M. Zapatka, M. Seiffert, C. Herold-Mende
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648afec99f4d26f113081261fc3d480301dc1dab" target='_blank'>
              Transcriptional reprogramming of tumor-infiltrating T cells during PD-1 blockade revealed through gene regulatory network and trajectory inference in squamous cell carcinoma
              </a>
            </td>
          <td>
            Roger Casals-Franch, Jordi Villà-Freixa, Lara Nonell, Adrián López García de Lomana
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 MAPK pathway alterations occur in 10-20% of diffuse midline glioma, H3K27-altered (DMG-H3K27), resulting in a proposed novel subtype of the disease with distinct clinical features. In a previous co-clinical trial platform, we showed an increased sensitivity to MEK inhibitors in DMG-MAPK patient-derived models, though resistance occurred in vitro, in vivo and clinically. To understand the nature of these resistance mechanisms, we used sophisticated coupling of retrievable high-complexity barcoding and single cell RNA sequencing in DMG-MAPK cells. Lineage tracing conducted in several technical replicate experiments showed a subset of clones is consistently enriched across replicates when treated with a GI90 concentration of trametinib for 11 weeks. Interrogation of clonal dynamics using mathematical modelling showed barcodes commonly enriched across replicates exhibit significantly higher clonal fitness compared to replicate-specific enriched barcodes, suggesting a deterministic pattern of resistance. Serial single cell RNA sequencing analysis at weeks 5, 8 and 11 post-treatment initiation showed some resistant H3.3K27M/BRAF-mutant cells adopted astrocyte-like and neural progenitor-like metaprograms in which barcodes were commonly enriched across two cell states, suggesting a subset of clones circumvent trametinib treatment by phenotypic switching; simultaneously, a late-emerging resistant population of cells activate a distinct mesenchymal program. Furthermore, we determine the previously identified MAP2K1 (MEK1) mutations to arise specifically from a small population of radial glia-like stem cells that appear acutely predisposed to the acquisition of these mutations. Together, our data suggest two clonally and mechanistically distinct resistance mechanisms can be active in DMG-MAPK in response to MEK inhibition, involving both transcriptional plasticity and a progenitor-like state primed for the acquisition of genetic determinants which circumvent drug response. With trametinib-resistant cells being highly sensitive to selective SRC inhibition, evolutionary-guided combinatorial treatment schedules may provide for strategies to significantly lengthen time taken for resistance development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c54926c3fb10216fcf5c70e35738abc3a296f32" target='_blank'>
              DDDR-39. Untangling MEK inhibitor resistance in DMG-MAPK using coupled lineage tracing and single cell RNA sequencing
              </a>
            </td>
          <td>
            Molina Das, Y. Grabovska, Haider Tari, A. Mackay, Rebecca F Rogers, Jared Collins, Jiin Song, Laura Bevington, A. Burford, Drenusha Sejdiu, R. Beroukhim, P. Bandopadhayay, Chris Jones
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>151</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) arises from precursor lesions, the most common of which are Pancreatic Intraepithelial Neoplasia (PanIN). PanINs are microscopic and cannot be detected in live individuals; as such, PanIN characterization has occurred prevalently in the context of cancer-bearing pancreata (adjacent normal) rather than in healthy tissue. Recently, through a unique collaboration with Gift of Life Michigan, we showed that PanIN prevalence is over 60% across age groups in donor healthy tissues, with high prevalence even in young individuals. Given PanIN prevalence and the rarity of pancreatic cancer, we hypothesized that the majority of PanIN is not fated to progress to malignancy. We postulated that a combination of neoplastic cell-intrinsic factors and components of the microenvironment controls progression to malignancy. We utilized spatial transcriptomics using the Visium platform (n = 14) and scRNASeq (n = 45) to dissect gene expression profiles and characterize spatial domains in donor healthy, adjacent-normal, and tumor tissues. We developed a spatial domain identification workflow that integrates spatial transcriptomics with the scRNASeq atlas, leveraging cell-type deconvolution, spatially informed clustering and pathology annotation. This approach allowed us to study the co-evolution of epithelial and stromal cells over the progression of pancreatic cancer. We were able to delineate distinct epithelial spatial domains—including Acinar, Ductal, and PanIN structures—in both donor and tumor tissues, as well as tumor-specific regions such as glandular tumor and poorly differentiated tumor. Additionally, we identified key stromal niches, including immune-rich regions such as tertiary lymphoid structures (TLS), plasma cell clusters, and macrophage-enriched zones, alongside non-immune fibrotic domains. We began by analyzing the epithelial domains and observed that while all acinar and ductal domains formed transcriptionally distinct clusters, neoplastic cells formed a continuum from sporadic healthy donor-associated PanINs to well- and poorly-differentiated tumors. We identified key genes that exhibit expression changes throughout the progression trajectory. Secondly, we analyzed characteristics of the stroma surrounding sporadic lesions, well-differentiated, and poorly-differentiated tumors. We observed key differences in fibroblasts between healthy donor-PanIN and tumorigenic regions, as fibroblasts surrounding donor-associated PanIN lacked known cancer-associated fibroblast (CAF) markers such as Alpha-Smooth Muscle Actin (α-SMA). We also observed distinct differences in immune cells, where there were more abundant plasma and CD8+ T-cells surrounding donor-associated PanIN, while tumorigenic regions showed macrophage enrichment. We then performed immunostaining and confocal imaging to validate the defined markers and specific cell populations. Our work defines the PanIN microenvironment as distinct from that of PDAC and maps the co-evolution of cancer cells and stroma over the course of carcinogenesis.



 Ahmed M. Elhossiny, Jude Okoye, Padma Kadiyala, Alexander Bray, Jamie Mills, Hannah Watkoske, Yaqing Zhang, Jiaqi Shi, Arvind Rao, Elana J. Fertig, Eileen S. Carpetner, Timothy Frankel, Marina Pasca di Magliano. Spatially Resolved Insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A052.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8f4f9e15e43361f1e76f4bf1e3cc3e1d04354d0" target='_blank'>
              Abstract A052: Spatially Resolved Insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, J. Okoye, Padma Kadiyala, Alexander Bray, Jamie N. Mills, Hannah R Watkoske, Yaqing Zhang, Jiaqi Shi, Arvind Rao, E. Fertig, Eileen S. Carpetner, Timothy L. Frankel, Marina Pasca di Magliano
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pediatric-type high-grade gliomas (HGGs) are aggressive brain tumors characterized by rapid growth, infiltrative behavior, and significant therapy resistance. One major challenge in treating HGGs involves the interplay of tumor cells to their microenvironment; specifically the blood-tumor barrier (BTB). Our studies combined histologic and transcriptomic evaluations of immunodeficient rodents intracortically injected with H3K27-altered and H3 WT glioma cells. Multiplex immunohistochemical staining provided temporospatial analysis of known cellular markers involved in BBB formation, including pericytes, endothelial tight-junction proteins, and myeloid cell markers. Regions of interest included the tumor, peritumor (+250µM), and distant (>250µM) areas. Analysis was conducted with the use of QuPath and cell segmentation performed using Mesmer; with expression proportions quantified for each cell/protein marker. Utilization of vessel isolation and nuclei extraction for sc-RNAseq (VINE-seq) allowed us to specifically explore the molecular landscape of vasculature within the tumor microenvironment. All rodent models were untreated and evaluated early to late post-intracranial injection (D20-50). Preliminary results in H3-K27-altered glioma models reveal elevated tight junctions within distal regions compared to the tumor (p<0.0001); demonstrating an intact BTB furthest from tumor bulk. Within the tumor, high Iba1+ macrophages (46%) and low neuronal (1%) expression was present (p<0.0001), while the intratumoral vasculature demonstrated the highest localization of glioma stem cells, and PDGFRβ+ during late disease progression (p<0.0001). VINE-seq identified unique cell-clusters in the tumor bearing vs. contralateral hemisphere at middle and late timepoints (D40-50). Ongoing VINE-seq studies will identify single cell expression changes both regionally and temporally with correlation to multiplex staining. These findings of blood-tumor barrier expression profiles among glioma cells offer a glimpse of changes for both the cellular and molecular landscape with both region and time specificity. Our research underscores the importance of understanding these interactions related to restrictions of CNS treatment and pediatric-type HGG progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da3676d783a5588622cda56cd645fcf0d5acc71b" target='_blank'>
              TMIC-08. Multiplex immunofluorescent imaging and VINE-seq demonstrates the temporospatial complexity of the blood-tumor-barrier in pediatric-type diffuse high grade glioma
              </a>
            </td>
          <td>
            Jatia Mills, Minhye Kwak, Sanghee Lim, D. Maric, Kory Johnson, Trevor McKee, Sadhana Jackson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/466091ce59415bf8e8d644d95fdb8949a53fb96b" target='_blank'>
              Cancer-associated fibroblasts drive metabolic heterogeneity in KRAS-mutant colorectal cancer cells
              </a>
            </td>
          <td>
            Elizabeth Elton, Niki Tavakoli, Handan Cetin, Stacey D. Finley
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf8e063a68a88a96c8de0cef9efb9254b78c4b6" target='_blank'>
              A cellular and spatial atlas of TP53-associated tissue remodeling defines a multicellular tumor ecosystem in lung adenocarcinoma.
              </a>
            </td>
          <td>
            William Zhao, Thinh Nguyen, Atharva Bhagwat, Akhil Kumar, Bruno Giotti, Benjamin Kepecs, Jason L. Weirather, N. Mahadevan, Åsa Segerstolpe, Komal Dolasia, Jamshid Abdul-Ghafar, Naomi Besson, Stephanie M Jones, Brian Y. Soong, Chendi Li, S. Vigneau, M. Slyper, Isaac Wakiro, Mei-ju Su, K. Helvie, Allison Frangieh, Judit Jané-Valbuena, Orr Ashenberg, Mark Awad, Asaf Rotem, Raphael Bueno, O. Rozenblatt-Rosen, K. Pfaff, Scott J. Rodig, Aaron N. Hata, Aviv Regev, B. Johnson, A. Tsankov
          </td>
          <td>2025-10-07</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, Anoop Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Glioblastomas function as intricate cellular networks that extend into the surrounding brain tissue enabling long distance communication. This malignant connectivity spans from the tumor core to remote infiltration zones, supporting the concept of glioblastoma as a whole-brain disease. With growing ethical considerations in biomedical research and the limitations inherent in cross-species comparisons between animal studies and human glioblastoma models, human tumor platforms remain constrained in their ability to replicate the full infiltration spectrum from the tumor core to single-cell dispersion. Here, we present a three-dimensional, fully human ex-vivo glioblastoma model (,,Core2Edge”) that replicates this extensive infiltration range while preserving the intratumoral heterogeneity of the original tumor. This model involves implanting fluorescently labeled human glioblastoma organoids (GBOs) into organotypic human brain slices, maintaining the genetic integrity and cytoarchitecture of both brain and tumor.



 The Core2Edge model involves a comprehensive workflow completed over 7-12 days. Key steps include brain slice preparation (~4-6 hours, depending on quantity), one day for initial culture before GBO staining and transplantation, a variable culture period (up to ten days), and fixation with different techniques (~8 hours). Following model preparation, downstream analyses such as imaging, sequencing, and proteomics were performed.



 Multiplex immunofluorescence confirmed cell composition consistent with histological staining. Spatial transcriptomics enabled detailed glioblastoma cell subtyping based on Neftel states. Advanced imaging, combining tissue expansion and light-sheet fluorescence microscopy, facilitated super-resolution, three-dimensional visualization of the GBO-brain slice system. This approach allows for the study of initial infiltration steps, in-depth analysis of the invasive front, exploration of cell-cell interactions between tumor cells and the tumor microenvironment, and offers a platform for drug screening and testing, reducing the need for animal models.



 The,,Core2Edge“model represents an innovative ex-vivo platform replicating glioblastoma infiltration and heterogeneity. By preserving genetic integrity and cytoarchitecture, this fully human model bridges critical gaps in glioblastoma research, offering unparalleled insights into tumor behavior and potential therapeutic interventions.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7215ed71357bb240e167b383e1fb29c1a6a3d4" target='_blank'>
              P01.24.B CORE2EDGE: A HUMAN GLIOBLASTOMA ORGANOID AND BRAIN SLICE CO-CULTURE MODEL CAPTURING TUMOR INFILTRATION AND TRANSCRIPTIONAL HETEROGENEITY FROM CORE TO SINGLE-CELL DISPERSION
              </a>
            </td>
          <td>
            M. Schneider, A. Potthoff, A. Melhem, B. Pregler, J. Rodriguez-Gatica, J. Thomas, L. Friker, M. Westhoff, V. Ravi, D. Heiland, A. Schlitzer, T. Pietsch, M. Hölzel, A. Waha, H. Vatter, M. K. Schwarz, U. Kubitscheck, U. Herrlinger
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits profound spatial heterogeneity driving therapeutic resistance, while the role of cancer-associated fibroblasts (CAFs) in orchestrating immunosuppressive niches remains incompletely defined. This study integrates single-nucleus RNA sequencing (snRNA-seq) and spatial transcriptomics (stRNA-seq) to map the cellular and molecular landscape of HCC. snRNA-seq identifies key cell populations-including fibroblasts, T_NK cells, and endothelial cells-using canonical marker genes. Spatial transcriptomics maps gene expression across tumor regions (core, invasive front, stroma) via the robust cell type decomposition (RCTD) algorithm. Immunofluorescence validates collagen deposition and POSTN spatial distribution, confirming T-cell exclusion patterns. The analysis identifies hypoxic metabolic myofibroblasts (hmmyCAFs) as central regulators of the tumor microenvironment. hmmyCAFs enrich at the invasive front, forming collagen-rich barriers that physically exclude CD8⁺ T cells. Simultaneously, they secrete POSTN to suppress immune checkpoint signaling and drive hypoxia-mediated glycolytic reprogramming of T-cell metabolism. Clinically, hmmyCAF activity and POSTN expression correlate with reduced progression-free survival and immunotherapy resistance. This multimodal study defines hmmyCAFs as triple architects of physical immunosuppression, molecular regulation, and metabolic remodeling. By linking collagen remodeling, POSTN-mediated checkpoint inhibition, and hypoxia-driven metabolic reprogramming to clinical outcomes, hmmyCAFs and POSTN may serve as potential indicators for evaluating the efficacy of immunotherapy in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aba5ba124e6d224e40973627e744d5081476427" target='_blank'>
              Spatial Transcriptomics and snRNA-seq Expose CAF Niches Orchestrating Dual Stromal-Immune Barriers in Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Yingxue Li, Changxiang Huan, Haoting Sun, Wei Zhang, Zhen Guo, Chuanyu Li, Jia Yao, Zhiqi Zhang, Shizhe Yu, Qiongzhu Dong, Lunxiu Qin, Jinze Li, Lianqun Zhou
          </td>
          <td>2025-10-07</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a39d99b88fdac73c609342d555fba7b0f41448d" target='_blank'>
              Regulatory network and spatial modeling reveal cooperative mechanisms of resistance and immune escape in ER+ breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Sarthak Sahoo, M. Jolly, Jason T. George
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Highlights What are the main findings? Poor prognostic ligand–receptor pairs are localized in specific regions within glioblastoma tissues. What is the implication of the main finding? Our findings provide new insights into pathological cell–cell interactions and treatment for glioblastoma. Abstract Glioblastoma (GBM) is an aggressive and lethal malignant brain tumor. Cell–cell interactions (CCIs) in the tumor microenvironment, mediated by ligand–receptor (LR) pairs, are known to contribute to its poor prognosis. However, the prognostic influence of CCIs on patients with GBM and the spatial expression profiles of such LR pairs within tumor tissues remain incompletely understood. This study aimed to identify prognostic LR pairs in GBM and their intratumoral localization via multitranscriptomic analysis. The CCIs among GBM cells as well as between GBM and niche cells were comprehensively evaluated using 40,958 cells in single-cell RNA sequencing datasets. They were found to form intercellular networks in GBM by specific LR pairs, which were mainly implicated in extracellular matrix (ECM)-related biological processes. Survival analysis revealed that 13 LR pairs related to ECM biological processes contributed to poor prognosis (p < 0.05, and 95% confidence intervals > 1). Notably, our spatial transcriptomic analysis using three independent GBM cohorts revealed that the identified poor prognostic LR pairs were localized in specific regions within GBM tissues. Although the clinical importance of these LR pairs requires further investigation, our findings suggest potential therapeutic targets for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/102a7c96b1dcd02ce81762c455b5d0717c9c4822" target='_blank'>
              Integrated Spatial and Single-Cell Transcriptomics Reveals Poor Prognostic Ligand–Receptor Pairs in Glioblastoma
              </a>
            </td>
          <td>
            Makoto Yoshimoto, Kengo Sugihara, Kazuya Tokumura, Shohei Tsuji, E. Hinoi
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The metastatic tumor microenvironment (TME) is a highly dynamic and heterogeneous ecosystem that plays a critical role in promoting cancer cell colonization, immune escape, and resistance to therapy. Recent advances in multi-omics technologies—including genomics, transcriptomics, epigenomics, proteomics, and metabolomics—have enabled a systems-level understanding of the molecular reprogramming that occurs in the TME following metastasis. In this review, we systematically summarize emerging findings from recent multi-omics studies that dissect cellular composition, signaling pathways, immune landscape, and metabolic rewiring within the metastatic TME. We highlight key molecular signatures and intercellular interactions that drive metastatic progression and therapy resistance. Furthermore, we discuss how integrative multi-omics data are being leveraged to identify actionable targets and to design novel immunotherapeutic and molecular precision strategies tailored to the metastatic niche. These insights provide a scientific rationale for the development of TME-targeted approaches in the treatment of advanced-stage cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1202cb3e5acfc52d10d5fc247d4fca996a2d6f2e" target='_blank'>
              Multi-omics analysis of the tumor microenvironment after metastasis: advancing toward personalized immunotherapy and molecular targeted strategies
              </a>
            </td>
          <td>
            Qing Gao, Huqiang Wu, Li Duan, Guanhua Lv, Dongmei Quan
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec585f40c1c2e8296bee75c248183641228fbd0" target='_blank'>
              Macrophages display spatiotemporal specificity in intrahepatic cholangiocarcinoma and drive tumour progression via OSM and THBS1
              </a>
            </td>
          <td>
            Tingjie Wang, Ruitao Long, Zhongyu Cheng, Mi Zhang, Bing Wei, Cong Lyu, Cuiyun Zhang, Guangbo Tang, Zihang Li, Lei Li, Yida Li, Xiaofei Zhu, Chao Shi, Ke Yang, Chuanrui Xu
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Meningiomas are the most common primary intracranial tumors, yet their cellular diversity, tumor-stroma interactions, and the relationship between neoplastic and normal meningeal cells remain incompletely understood. While recent studies have provided valuable single-cell insights into specific tumor contexts, a unified atlas encompassing both meningioma and healthy meningeal tissue across patients and studies is lacking.



 We constructed MeningioMap, a comprehensive single-cell atlas by harmonizing three publicly available datasets with in-house single-cell RNA sequencing (scRNA-seq) of surgically resected meningiomas and matched adjacent meningeal tissue. The dataset encompasses over 300,000 high-quality cells from 40+ patients, spanning diverse tumor grades, anatomical locations, histopathological subtypes, and genders. Standardized preprocessing, robust batch correction, and integrative clustering pipelines were employed. Preliminary spatial transcriptomic analyses further validated key tumor-stromal interactions predicted from scRNA-seq profile.



 Meningioma delineates a high-resolution taxonomy of malignant and non-malignant cell states within the meningioma ecosystem. We identified distinct malignant subpopulations characterized by specific copy number alterations, transcriptional programs, and clinical associations with tumor grade. Comparative analysis with matched meninges uncovered conserved meningeal lineages and unveiled tumor-specific reprogramming events. Spatial transcriptomics confirmed localized immune remodeling and stromal activation signatures at the tumor-meninges interface. Notably, we identified a novel transitional malignant-stromal phenotype enriched in invasive tumor fronts, suggesting a potential driver of recurrence and therapeutic resistance.



 MeningioMap represents the most extensive single-cell resource for meningioma and meningeal research to date. Beyond offering a foundational reference for tumor heterogeneity and cellular dynamics, it provides critical insights into tumor-microenvironment crosstalk, informing future strategies for precision diagnostics, risk stratification, and targeted therapeutic development. An interactive, publicly accessible portal will facilitate broad scientific and clinical utility.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5ef5fbb1b4ae669071d94a0dad7887a5b662c5" target='_blank'>
              P13.18.A MENINGIOMAP: A SINGLE-CELL ATLAS OF MENINGIOMA ECOSYSTEMS AND MICROENVIRONMENTAL NETWORKS
              </a>
            </td>
          <td>
            A. Boecke, A. Mohammed, D. Das, G. Nishita, R. Roelz, M. Shah, T. Juratli, J. Beck, J. Grauvogel, K. Joseph, V. Ravi
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Genomic alterations in glioblastoma (GB) drive distinct transcriptional subtypes, yet how they shape the tumour microenvironment (TME) architecture remains poorly understood. Here, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss, and PDGFB overexpression in Tp53−/− Nestin-tv-a mice, that recapitulated classical, mesenchymal, and proneural transcriptomic GB subtypes. We integrated single-cell RNA sequencing with regional (NanoString GeoMx) and single cell (10x Visium HD) spatial transcriptomics and proteomics data to acquire an in-depth characterization of the TME organization. At single cell level, NF1- and EGFRvIII-driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human GBs that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1043a078420592595b843047c99776c531341d4b" target='_blank'>
              TMIC-04. Genetic drives promote distinct microenvironment organization in glioblastoma
              </a>
            </td>
          <td>
            Junyi Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) exhibits marked plasticity and intense microenvironmental crosstalk. We aimed to delineate mesenchymal programs with spatial resolution, clinical relevance, and mechanistic anchors. Methods We integrated single-cell RNA-seq, bulk transcriptomes, and Visium spatial data. After rigorous QC and Harmony integration, we annotated 12 cell states using canonical markers, decoupler-based ORA, and AUCell. Tumor boundaries were defined by inferCNV/CopyKAT; developmental potential by CytoTRACE2 and PHATE. Post-translational modification (PTM) axes were scored from curated gene sets. A cell type-aware GNN linked bulk expression to a patient-similarity graph for survival modeling and gene-level hazard attribution. Network convergence combined bulk WGCNA (TCGA/CGGA), single-cell hdWGCNA, BayesPrism deconvolution, and external GEO validation. Ligand–receptor (LR) signaling was inferred with LIANA+, embedded in a signed causal network, and mapped spatially. ARRDC3 expression was assessed in GBM tissues; U251 gain- and loss-of-function assays evaluated proliferation and migration. Results We resolved major GBM states, including two mesenchymal programs (MES1-like, MES2-like). CNV-high regions marked malignant cores, and CytoTRACE2 identified high-potency niches within MES2-like and Proliferation states along non-linear trajectories. PTM landscapes segregated by state; S-nitrosylation, glycosylation, and lactylation were enriched in mesenchymal programs. A GNN risk score stratified overall survival in TCGA (n=157) and generalized to CGGA-325 (n=85) and CGGA-693 (n=140). MES2-like abundance remained an independent adverse predictor (HR = 2.31; 95% CI, 1.04–5.10). MES2-high tumors upregulated EMT, TNFα/NF-κB, JAK/STAT, hypoxia, angiogenesis, and glycolysis; S-nitrosylation associated with increased hazard. Cross-modal convergence defined a conservative MES2 core enriched for ECM remodeling, collagen modification, focal adhesion, and TGF-β regulation. LR analysis prioritized a TAM-to-MES2 axis (e.g., GRN–TNFRSF1A, ADAM9/10/17–ITGB1, TGFB1–ITGB1/EGFR) converging on a CEBPD-centered module. Spatial mapping localized MES2 hotspots within CNV-defined territories and revealed a TNFRSF1A–CEBPD–ARRDC3 focus at an infiltrative rim. ARRDC3 was upregulated in GBM tissues; in U251 cells, knockdown promoted and overexpression suppressed proliferation and migration, indicating context-dependent roles. Conclusions MES2-like GBM is an ECM-driven, stress-adapted state with strong prognostic impact. We nominate CEBPD and TNFRSF1A/ITGB1 as actionable nodes and identify ARRDC3 as a spatially restricted effector with context-dependent tumor-modulatory functions warranting therapeutic exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8942e39c98213dbf5399f61de91b7075a89e9eb2" target='_blank'>
              Single-cell/spatial integration reveals an MES2-like glioblastoma program orchestrated by immune communication and regulatory networks
              </a>
            </td>
          <td>
            Chonghui Zhang, Lu Tan, Kaijian Zheng, Yifan Xu, Junshan Wan, Jinpeng Wu, Chao Wang, Pin Guo, Yugong Feng
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Understanding how small molecules affect cellular morphology is essential for exploring their mechanisms of action (MoA) and identifying new therapeutic candidates. In this study, the Cell Painting Assay (CPA) was used to profile 196 small molecules in HCT116 colorectal cancer cells. By applying t-distributed stochastic neighbor embedding (t-SNE) followed by density-based clustering, 18 distinct phenotypic clusters were identified based on similarities in the quantitative morphological profiles generated from Cell Painting data. Although it was initially hypothesized that clustering would reflect known MoAs, most clusters showed only partial overlap with target-based classifications. Instead, compounds from different MoA classes converged on similar cellular phenotypes, suggesting common downstream effects or shared stress responses. Notably, compounds affecting DNA replication, mitosis, or transcriptional control appeared across multiple clusters, indicating functional diversity within morphologically similar groups. Clusters enriched with mTOR/PI3K inhibitors, spindle poisons, or transcriptional CDK blockers exhibited well-defined phenotypes, supporting the robustness of the assay. In contrast, compounds inducing more subtle phenotypes formed distinct micro-clusters, highlighting the method’s sensitivity. Overall, this study demonstrates that CPA can capture convergent phenotypic signatures that extend beyond target-based classification. These findings underscore the value of phenotype-driven screening for the functional annotation of chemical compounds and may help uncover unexpected relationships among molecules with diverse biological activities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b69630fa6518b96d44a14fa030997f1e5bc439" target='_blank'>
              Phenotypic profiling of small molecules using cell painting assay in HCT116 colorectal cancer cells
              </a>
            </td>
          <td>
            Velemir Lavrinenko, Ilona Donskaya, Vladimir Popov, Ekaterina Litau, Varvara Petrova, Stanislav Tyazhelnikov
          </td>
          <td>2025-10-29</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is marked by an immunologically inert tumor microenvironment (TME), which may explain why immunotherapies have generally failed to improve survival. While TME features have been linked to clinical outcomes, prior studies were limited by small sample sizes and narrow cell type resolution, precluding a full understanding of TME multicellular dynamics. To address this, we assembled 1,979 clinically annotated GBM RNA-seq profiles and benchmarked six deconvolution algorithms using pseudobulk data from a large GBM single-cell RNA-seq dataset. Bayesian deconvolution (InstaPrism) emerged as a top performer and was used to characterize the TME across 39 cell states. We then stratified tumors by their dominant malignant cell state (AC-like, MES-like, NPC-like, or OPC-like). We identified lymphoid-dominant and myeloid-dominant immune compositions within each GBM subtype. We demonstrate that lymphoid-dominant, immune-activated tumors associate with significantly improved overall survival within the aggressive MES-like group, but not within other groups. Immune activation was defined by a conserved stress-inflammatory module consisting of general inflammatory activation, tissue remodeling, and stress. In tumors dominated by MES-like cell states, immune cells within the TME were driven by robust interferon alpha and gamma gene signatures. Sobel mediation analysis pinpointed Stimulator of interferon genes (STING) upregulation as a potential upstream orchestrator within this group. Interestingly, STING activation showed no beneficial effect in AC-like or NPC-like tumors and was detrimental in OPC-like GBM, highlighting a group-specific biology. To our knowledge, this represents the most comprehensive analysis of the GBM TME to date in terms of patient number (n=1,979) and TME cell type plexy (n=39). The findings offer new understanding of GBM multicellular organization and how different cell types coordinate within the tumor. These data may help design rational combinations of immunomodulatory therapies and guide patient selection for next-generation clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91fc74608f0fb0d307e1af10e8fcfdf8bbe8b02e" target='_blank'>
              TMIC-63. Microenvironment archetypes in 1,979 glioblastomas reveal cell state-specific drivers of immune recruitment and survival benefit
              </a>
            </td>
          <td>
            Kevin J. Tu, Zachary J. Reitman
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chemotherapy resistance remains a formidable challenge to the treatment of high-grade serous ovarian cancer (HGSOC). The drug-tolerant cells may originate from a small population of inherently resistant cancer stem cells (CSCs) in primary tumors. In contrast, sufficient evidence suggests that drug tolerance can also be transiently acquired by nonstem cancer cells. Regardless of the route, key regulators of this plastic process are poorly understood. Here, we utilized multiomics, tumor microarrays, and epigenetic modulation to demonstrate that SOX9 is a key chemo-induced driver of chemoresistance in HGSOC. Epigenetic upregulation of SOX9 was sufficient to induce chemoresistance in multiple HGSOC lines. Moreover, this upregulation induced the formation of a stem-like subpopulation and significant chemoresistance in vivo. Mechanistically, SOX9 increased transcriptional divergence, reprogramming the transcriptional state of naive cells into a stem-like state. Supporting this, we identified a rare cluster of SOX9-expressing cells in primary tumors that were highly enriched for CSCs and chemoresistance-associated stress gene modules. Notably, single-cell analysis showed that chemo treatment results in rapid population-level induction of SOX9 that enriches for a stem-like transcriptional state. Altogether, these findings implicate SOX9 as a critical regulator of early steps of transcriptional reprogramming that lead to chemoresistance through a CSC-like state in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016a8c0db4a1babe91a08a91f054acbb57a8720c" target='_blank'>
              SOX9 drives a stem-like transcriptional state and platinum resistance in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            A. Duval, Fidan Seker-Polat, Magdalena Rogozinska, Meric Kinali, A. Walts, Ozlem Neyisci, Yaqi Zhang, Zhonglin Li, Edward Tanner, Allison E. Grubbs, Sandra Orsulic, Daniela Matei, M. Adli
          </td>
          <td>2025-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is one of the most aggressive and deadly brain tumors, with a median survival of ~14 months. Its poor prognosis is largely due to its ability to infiltrate healthy brain tissue both as single cells and in collectives. A deeper understanding of both individual and collective GBM cell motility could reveal new insights for patient-specific therapies. We applied live-cell imaging and mathematical modeling to characterize migration in patient-derived GBM cells, aiming to identify stratification markers and therapeutic vulnerabilities.



 We developed Single-Cell Behavior Live Imaging (ScBLI), an approach integrating live imaging with computational analysis to study the dynamics of GBM and healthy astrocyte (HA) cultures. We analyzed 21 GBM cell lines and one HA culture. Using TrackMate and Cellpose for segmentation and tracking, we extracted data on trajectories, velocities, distances, and morphological features (e.g., volume, circularity). Delta scaling was applied to classify trajectories as subdiffusive, Brownian, or Levy-like. Functional assays evaluated responses to migration stimuli and therapeutics. We also assessed collective migration behavior and integrated these data with single-cell analyses. All findings were correlated with clinical outcomes and transcriptomic profiling.



 We tracked 100 cells per sample, analyzing 2200 trajectories. Each culture contained three distinct migratory subpopulations based on metrics such as displacement, velocity, confinement ratio, and cellular organization. GBM samples were grouped into three behavioral clusters. Delta scaling values (0.28-0.80) identified corresponding migration modes. Higher delta values indicated more structured, Levy-like migration and were associated with shorter patient survival, suggesting a relationship between organized motility and increased invasiveness. Transcriptomic analysis revealed differential gene expression signatures among the clusters, potentially linked to migration phenotypes.



 Our study presents a robust framework to characterize GBM migration dynamics across patient-derived cultures. By integrating individual and collective motility data with clinical and transcriptomic information, we identified three distinct behavioral profiles with prognostic value. Delta scaling proved to be an effective metric for stratifying tumors based on migratory efficiency and aggressiveness. These insights highlight how structured motility may facilitate tumor spread and serve as a functional biomarker. Moreover, the identification of migration-associated gene signatures opens avenues for the development of targeted therapies aimed at disrupting invasive behavior. Ultimately, our findings support the use of dynamic phenotyping in guiding personalized treatment strategies for GBM patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53502cc74a0ab473690a657868f9d225b25d02d6" target='_blank'>
              P12.34.A TRACKING INDIVIDUAL AND COLLECTIVE CELLULAR DIFFUSION IN PATIENT-DERIVED PRIMARY GBM CELL LINES: EXPLORING CELL DIFFUSION PROPERTIES AS A POTENTIAL VULNERABILITY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            M. Morelli, G. Ferri, T. Hohmann, F. Lessi, F. Di Lorenzo, A. Pastore, S. Franceschi, F. Pieri, C. Gambacciani, O. Santonocito, Y. Shah, L. Tonello, A. D. Di Stefano, P. Grigolini, L. Palatella, C. Mazzanti
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 The reciprocal interaction between neural and immune components in the tumor microenvironment has emerged as a critical area of study in cancer neuroscience. Recent data has shown the increasing role of myeloid cells in enabling an immunosuppressive microenvironment in glioblastoma. However, the nature and the role of T cells and other immune components forming tumor-immune synapses remain to be fully explored. Here we aim to demonstrate how immunological stimuli and inflammatory changes affect the neural connections within and around glioblastoma cells using advanced spatial transcriptomics.



 The study employs human cortical slice models, inoculated with tumor cells carrying retrograde tracing viruses, specifically a rabies virus, to trace neuron-tumor connections. This was followed by the inoculation of reactive T cells. We used the MERSCOPE system, which allows for the 3D reconstruction of gene expression signatures in consecutive tissue slices, spanning multiple Z-depths by combining this cutting-edge technology with new segmentation approaches.



 Our spatial transcriptomic analyses revealed different cellular ecosystems in the human cortical slice model. Enhanced inflammatory stimuli can increase local tumor growth and infiltration and neuronal connectivity. We found that inhibition of T cell activation using JAK-STAT inhibition can reverse the reactive hyperconnectivity. We further identified key transcription regulators defining immune cell dynamics and triggering immunological stimuli within tumor-immune synapses.



 The integrative multi-omic approach using imaging-based analysis and 3D single-cell spatial transcriptomics technique is a powerful tool that allows a comprehensive understanding of how immune cells influence the neural connectivity and synaptic reactivity within the peritumoral microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5833d83bdb7f3aebf492021ffdbee5e8a292416d" target='_blank'>
              P04.16.A DECIPHERING NEURAL-IMMUNE RECIPROCAL INTERACTIONS IN GLIOBLASTOMA USING 3D SINGEL-CELL SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            T. Peng, A. Merdan, Y. Yabo, V. Venkataramani, F. Ricklefs, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) progression and treatment resistance are closely linked to cancer stem cells (CSCs), yet their spatial dynamics and regulatory mechanisms remain poorly characterized. This study integrates single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) to define CRC stem cell subtypes (Co-CSS), map their interactions within the tumor microenvironment, and identify novel prognostic markers driving clinical outcomes. Methods scRNA-seq was used to analyze cellular heterogeneity in CRC, identifying key Co-CSS. CytoTRACE was applied to assess stemness. ST further mapped the distribution and interactions of Co-CSS with the tumor microenvironment. Gene intersection analysis identified NPDC1 and NSMF, whose high expression correlated with poor prognosis. Patient classification based on NPDC1 and NSMF expression highlighted treatment response variability, suggesting targets for precision medicine. Results The findings indicate that Co-CSS in CRC can modulate the tumor microenvironment and influence the progression of the disease through various ligand-receptor interactions. It is possible that Co-CSS is regulated by fibroblasts through the WNT signaling pathway. The expression levels of NPDC1 and NSMF are significantly correlated with the overall survival of CRC patients, suggesting their potential value as independent prognostic markers. Conclusions This study offers insights into the role of Co-CSS in CRC through ST and gene expression profiling, uncovering novel markers and pathways. It redefines the interplay between Co-CSS and the tumor microenvironment, offering a foundation for future mechanistic studies and therapeutic interventions aimed at targeting Co-CSS to improve CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed26e5557feb0443a11a342bd7bb1b7f2d659826" target='_blank'>
              The role of cancer stem cells in the progression of colorectal cancer and the tumor microenvironment: new perspectives from single-cell RNA sequencing and spatial transcriptomics analyses
              </a>
            </td>
          <td>
            Yuting Wu, Xianglin Liu, Wenqiang Liu, Qingliang Jiang, Yangyang Li, Hengyu Li, Liqiang Hao
          </td>
          <td>2025-10-01</td>
          <td>Translational Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Chimeric antigen receptor T-cell (CAR-T) therapy has emerged as a breakthrough immunotherapy for multiple cancers, yet its efficacy remains limited in glioblastoma due to tumor heterogeneity, immune-evasion of the microenvironment, and rapid T-cell dysfunction. The aim of our study is to investigate the spatially differential expression of CAR-T targets in glioblastoma. We want to focus on differential expression of CAR-T targets in tumor cell subtypes, differences between tumor core and infiltration zone as well as reactive immune zones.



 We analyzed spatial transcriptomic data from 62 glioblastoma specimens using SPATA2 software, augmented by advanced computational tools such as scVI (data integration), Cell2Location (single-cell deconvolution), graph neural networks, gene set enrichment analysis (GSEA), and spatial annotation screening (SAS). CAR-T antigens were chosen from existing published clinical and pre-clinical CAR-T cell trials.



 Spatial mapping revealed a pronounced gradient of NPC-like cells with a higher abundance of NPC-like cells in the infiltration zone and resection margins—regions critical for residual disease persistence after resection. Clinical CAR-T targets (e.g., ERBB2, PTPRZ1) were highly expressed in AC-like and OPC-like dominated niches while nearly absent in NPC-rich areas, suggesting a blind spot for current CAR-T therapies. Single-cell deconvolution further correlated NPC-like dominance to increased T-cell infiltration, while GSEA and clonal dynamics analysis of the T-cell compartment suggested downregulated functioning subclones in NPC-rich spatial neighborhoods, underscoring potential immune evasion mechanisms in these regions.



 Our study exposes a fundamental limitation of current CAR-T designs: their inefficacy against NPC-like cells that dominate the infiltration zone and therefore the post-surgical glioblastoma ecosystem. By pinpointing spatial and phenotypic resistance hubs, we provide a roadmap for refining CAR-T targets to address this gap, potentially enhancing adjuvant immunotherapy for glioblastoma. Protein-level validation of key findings is currently ongoing to further substantiate the results.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe7508f42aeb973a2f44bba592abe40968ca576d" target='_blank'>
              P04.13.B SPATIAL HETEROGENEITY OF CAR-T ANTIGEN EXPRESSION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Y. Kong, J. Zhang, F. Sahm, G. Kastner, M. Mühlbauer, E. Grabis, P. Heinrichs, J. Straehle, R. Sankowski, R. Roelz, J. Beck, D. Heiland, N. Neidert
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma (PDAC) exhibits a huge range of genomic plasticity due to prevalent genomic instability across tumors. Longitudinal whole genome sequencing of patients at primary diagnosis and metastatic disease progression have shown KRAS copy number amplification and allelic imbalance as drivers of classical to basal-like phenotypic switch, resulting in propensity for whole genome duplication, genomic instability and aggressive metastasis. High-plex spatial proteomics have revealed that primary tumors are phenotypically mixed, but the relationship between intratumorally distinct phenotypes and their genomics remain unclear as existing genomics relies on bulk captured tumors. Therefore, this drives the need to spatially resolve the intra-tumoral genomic heterogeneity associated with distinct tumor phenotypes. We leveraged whole-slide high-plex Imaging Mass Cytometry (IMC) to stain a phenotypically mixed primary tumor with 41 different markers covering the classical (AGR2, TFF1, CEACAM6) and basal (TP63, KRT5, S100A2, CAV1) phenotype, pan-immune markers, epithelial transcription factors (GATA6, FOXA2, PDX1) and stroma. Bulk whole genome sequencing (30X) reveals that this tumor had undergone losses of chr 1p, 6p, 9, 17 and 8, in addition to 12p focal amplification and whole genome doubling. To address the association of spatial phenotypes to regional genomics, IMC guided laser capture microdissection (LCM) was performed on a serial section aligned to 4 classical (C1-4), 4 basal (B1-4) and 4 hybrid-staining (H1-4) regions, followed by independent shallow whole genome sequencing (sWGS) at 2X coverage for each captured region (n = 12). Quality control, alignment and variant calling was performed utilizing standard whole-genome sequencing workflow. Integer copy number was determined through 100kb bins and segmentation. A full range of ploidy solutions (2N, 3N and 4N) was fit for each region using Absolute Copy Number Estimation (ACE) and the best fit was used for each region. We observed that in region C2, losses in many chromosomes (1p, 6, 9, 17, 18) but had no arm level amplifications, indicating a mitotic defect. At region C3, we observed a small increase in the 12p segment harboring the KRAS locus, suggesting that a sub-clonal event had occurred resulting in amplification. At region H3, a phenotypically mixed region, a strong 12p amplification was observed along with widespread genomic instability. Finally, at region B1, the whole genome duplication is observed. Our initial pilot reveals a stepwise, orchestrated genome evolution process in a phenotypically mixed primary tumor which was not obvious in bulk whole genome sequencing, highlighting the need for spatially informed regional capture of tumors to uncover phenotypically and potentially clinically important genomic alterations.



 Tiak Ju Tan, Deni Van. Sechie, Ferris Nowlan, Noor Shakfa, Elizabeth Sunnucks, Sibyl Drissler, Jenn Gorman, Michael J. Geuenich, Chengxin Yu, Grannie O. Kane, Faiyaz Notta, Rob Grant, Julie Wilson, Steve Gallinger, Kieran R. Campbell, Hartland W. Jackson. Imaging mass cytometry guided shallow whole genome sequencing for phenotype informed regional copy number profiling in a pancreatic tumor [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B113.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e0d82bf993f8cb23419e03138a40de013a09be1" target='_blank'>
              Abstract B113: Imaging mass cytometry guided shallow whole genome sequencing for phenotype informed regional copy number profiling in a pancreatic tumor
              </a>
            </td>
          <td>
            Tiak Ju Tan, Deni Van. Sechie, Ferris Nowlan, Noor Shakfa, Elizabeth Sunnucks, Sibyl Drissler, Jennifer L. Gorman, Michael J. Geuenich, Chengxin Yu, Grannie O. Kane, F. Notta, Robert C. Grant, Julie M. Wilson, Stephen Gallinger, Kieran R. Campbell, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Glioblastoma (GBM), a highly aggressive malignant brain tumor, leads to poor clinical outcomes and low long-term survival rates. Using bioinformatics approaches, we analyzed single-cell RNA sequencing (scRNA-seq) data (GSE273274) from glioma patient samples in this study. It was found that mesenchymal-like (MES-like) cells in the GBM center (GC) significantly facilitated GBM’s aggressive spread and microenvironmental reprogramming. Further analysis on MES-like cells suggested that these cells may participate in the progression from glial differentiation to invasive metastasis. Differential expression analysis, univariate Cox regression, and LASSO regression identified COL22A1 as a key gene. COL22A1 was highly expressed in GC compared to GBM surrounding tissue (GS) and exhibited potential connection to aberrant phosphatidylinositol 3-kinase (PI3K) signaling activation, indicating that COL22A1 may act as a potential regulator of GBM, promoting malignant processes such as proliferation, apoptosis resistance, migration, and invasion. In vitro experiments showed that COL22A1 overexpression (COL22A1-OE) U-87 cells exhibited enhanced proliferation, viability, migration, and invasion. Moreover, immunoblotting of BCL-2 and BAX revealed that COL22A1-OE increased resistance to apoptosis in U-87 cells. However, all these effects were reversed upon silencing of COL22A1, which suggests COL22A1 could be a promising new target for GBM therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a091b05d62e57c97e8673640ca9464f145f296ba" target='_blank'>
              Malignant progression of MES-like cells mediated by COL22A1 in the spatial heterogeneity of glioblastoma
              </a>
            </td>
          <td>
            Guangyuan Chen, Zhonghua Fu, Xulin He, Zhaoli Shen
          </td>
          <td>2025-10-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Recurrent/metastatic head and neck squamous cell carcinoma (HNSCC) is an aggressive malignancy with a significant unmet need for enhancing immunotherapy response given current modest efficacy. Here, we perform an in vivo CRISPR screen in an HNSCC mouse model to identify immune evasion genes. We identify several regulators of immune checkpoint blockade (ICB) response, including the ubiquitin C-terminal hydrolase 5 (UCHL5). Loss of Uchl5 in tumors increases CD8+ T cell infiltration and improved ICB responses. Uchl5 deficiency attenuates extracellular matrix (ECM) production and epithelial-mesenchymal-transition (EMT) transcriptional programs, which contribute to stromal desmoplasia, a histologic finding we describe as associated with reduced anti-PD1 response in human HNSCCs. COL17A1, a collagen highly and specifically expressed in HNSCC, mediates in part Uchl5-mediated immune evasion. Our findings suggest an unappreciated role for UCHL5 in promoting EMT in HNSCC and highlight ECM modulation as a strategy to improve immunotherapy responses. UCHL5 is a deubiquitinating enzyme that cleaves Lys-48-linked polyubiquitin chains. Here, the authors discover through in-vivo CRISPR-Cas9 screens that Uchl5 is involved in immune evasion and modulation of extracellular matrix deposition in head and neck squamous cell carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fd6c2af8dceba0dfa5c4863b05099dccf3b0af2" target='_blank'>
              In vivo CRISPR screening in head and neck cancer reveals Uchl5 as an immunotherapy target
              </a>
            </td>
          <td>
            Cong Fu, R. Saddawi-Konefka, Jordan M. Chinai, Sarah Y. Kim, Ashwin V. Kammula, Jonathan J Perera, Aiping Jiang, Payal Tiwari, Erin N Kistler, Shiqi Tang, Sarah M Luna, Kayla J. Colvin, Juan Dubrot, Seth Anderson, Rachel A Fetterman, Cun Lan Chuong, Sarah K. Lane-Reticker, Collins K. Cheruiyot, Audrey J. Muscato, Zahra Alipour, Douglas R. Adkins, Gabriel K Griffin, Bradley E Bernstein, A. Egloff, Kathleen B. Yates, R. Chernock, J. Gutkind, R. Uppaluri, R. Manguso
          </td>
          <td>2025-09-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 The World Health Organization provides clear criteria for diagnosing IDH1-wildtype glioblastoma (GBM), relying on traditional approaches to define pathoetiological features, genomic mutations, copy number variation, and aberrant expression of prognostic genes. While these methods guide diagnosis, they may lack the resolution to fully capture tumor heterogeneity, therapeutic response, or outcomes. To address this, we applied leading-edge technologies to 14 primary and recurrent tumors to add granularity to current diagnostic metrics and clinical classification. Aiming to better inform tumor subtyping and improve patient care, this integrative approach enabled us to identify malignant gene programs and localize prognostically relevant features at both single-cell and spatial levels. We analyzed 14 GBM tumors from 6 patients using spatial in situ sequencing (ISS) and single-nuclei RNA-sequencing. A custom ISS gene panel enabled spatial mapping of tumor-associated cell types and state markers, while matched snRNA-seq from adjacent tumor tissue provided transcriptome-wide profiles. One patient contributed 5 serially collected recurrent tumors, enabling longitudinal analysis of tumor evolution. Profiling these tumors using snRNA-seq identified canonical copy number alterations (+7/–10) and ligand-receptor interaction analysis showed an evocative ANXA1 signaling axis progress across recurrent tumors, which suggests possible therapeutic opportunities. Finally, the dataset enabled subcellular and spatial localization of the prognostic GBM biomarker, EGFRvIII, being enriched in oligodendrocyte progenitor (OPC)-like tumor cells, which provides exciting evidence of these cancer cell subtypes as the putative cell of origin for GBM. By extending beyond conventional diagnostic benchmarks, our molecular profiling reveals new insight into GBM biology that may support more refined tumor subtyping and prognostic assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb079e9e9eb8a8e1b502483557634499fe4e750" target='_blank'>
              BIOM-72. INTEGRATIVE SINGLE-NUCLEI SPATIAL TRANSCRIPTOMIC PROFILING OF PRIMARY AND RECURRENT HUMAN IDH-WILDTYPE GLIOBLASTOMA REVEALS CELL-SPECIFIC EGFRVIII EXPRESSION AND ANXA1 SIGNALING AXIS
              </a>
            </td>
          <td>
            Lauren Powers, Vaibhav Jain, Michael Aksu, Emily Hocke, Alan Smith, Stephanie Arvai, Ellora Haukefrers, Kevin Stevenson, Khooshbu Patel, Diane Satterfield, Elizabeth Thomas, Karen Abramson, Giselle López, R. McLendon, David M. Ashley, Simon Gregory
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Glioblastoma relapse remains inevitable, placing these tumors among the most difficult to treat. Treatment challenges include inherent drug resistance, tumour’s plastic nature, and suppressive tumour microenvironment (TME). In other cancers, treatment-induced plasticity has been associated with the emergence of drug-tolerant persister cells, which survive therapy by undergoing reversible phenotypic changes towards resistant states. The role of persisters in glioblastoma remains to be unravelled. It is unclear to what extent the responses are patient- and treatment-specific, and how much they depend on the tumor microenvironment.



 We applied a cohort of molecularly characterized patient-derived models to investigate patient-specific responses to chemotherapeutics (Temozolomide, VAL-083). Treatment efficacy and its correlation with molecular features were assessed ex vivo using organoids and stem-like cultures as well as in vivo through orthotopic xenografts. To mimic clinical treatment schedules, we assessed functional responses in time, incorporating prolonged exposure to treatment and drug holiday period over several weeks. Analysis included tumor cell viability, cell cycle, DNA damage and repair, senescence and quiescence features. Phenotypic adaptation of glioblastoma and microenvironmental cells in vivo were analysed with MRI, single-cell RNA-seq, spatial transcriptomics and IHC.



 Assessment of drug efficacy across a cohort of patient-derived models revealed resistant versus adaptive responder groups, which were treatment-specific. While resistant cells did not show functional changes in cell viability and cell cycle upon prolonged drug exposure, sensitive cells show a biphasic growth curve characteristic of persister cells. The adaptation towards the least proliferative phenotype was reversible during drug holiday period, further confirming a role of a transient drug-tolerant state. Chemotherapeutic treatment in orthotopic xenografts partially reduced tumor growth in vivo and activated apoptosis and DNA repair pathways in glioblastoma cells. These changes coincided with an increased infiltration of tumor-associated macrophages.



 Our findings highlight the critical roles of both tolerance and intrinsic resistance mechanisms. The responder groups appear to be specific to the type of treatment and the unique characteristics of each patient’s tumor, emphasizing the need to evaluate treatment-induced plasticity in a patient-specific context. Glioblastoma cells are further supported by the adaptation of the microenvironment towards a tumor-supportive crosstalk.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ea2a43463b7a7c67d446cbd650a87fb74e2953" target='_blank'>
              P01.18.B MODELING GLIOBLASTOMA TREATMENT RESPONSES: INSIGHTS INTO TUMOR PLASTICITY AND THE ROLE OF THE TUMOR MICROENVIRONMENT
              </a>
            </td>
          <td>
            M. de Lucas Sanz, H. Allahverdi, B. Nosirov, P. Moreno Sánchez, M. Baroni Milan, A. Lipsa, M. Rezaeipour, C. Dording, D. Brown, J. M. Langlands, K. Grzyb, Y. Yabo, J. Zhang, D. Heiland, A. Michelucci, P. Nazarov, A. Gołȩbiewska
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4869e98a51fddec49b56e84ff1fb2955ba21a8dd" target='_blank'>
              Clonal selection supported by single cell Dna sequencing reveals hormonal adaptation and resistance in locally advanced breast cancer During Neoadjuvant Aromatase Inhibition
              </a>
            </td>
          <td>
            Denise G. O’Mahony, T. Lesluyes, Ina S. Brorson, Patrik Vernhoff, Ksenia Sokolova, M. R. Aure, G. G. Alnæs, Rebecca Mae Hoøen, Arvind Y.M. Sundaram, Chandra L. Theesfeld, Stephanie B Geisler, Torill Sauer, N. Bahrami, R. Parulkar, Andliena Tahiri, T. Lüders, Olga G. Troyanskaya, Charles J. Vaske, P. van Loo, Jürgen Geisler, Vessela N. Kristensen
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Understanding tumor heterogeneity is essential for advancing cancer treatment. Clonal reconstruction methods play a pivotal role in deciphering this heterogeneity. Our goal is to develop a clonal reconstruction approach that is clinically applicable, easy to implement, and capable of delivering both high-speed performance and excellent reconstruction accuracy. We present MyClone, a probabilistic method designed to reconstruct the clonal composition of tumors using deep sequencing genomic data. MyClone processes read counts and copy number information of single nucleotide variants derived from deep sequencing data, enabling it to determine the mutational composition of clones and the cancer cell fractions of these mutations. Compared to existing clonal reconstruction methods, MyClone enhances clustering accuracy and cancer cell fraction prediction when applied to deep-targeted sequencing data and bulk tumor sequencing data with deep sequencing coverage. Additionally, MyClone achieves a substantial improvement in computational speed. We rigorously validated MyClone’s performance using both simulated and real clinical datasets and applied it to analyze a circulating tumor DNA sequencing dataset from 139 metastatic breast cancer patients. In this analysis, we explored mechanisms of drug resistance in metastatic breast cancer and identified 10 mutated genes potentially associated with drug resistance or sensitivity. For deeply sequenced data, MyClone outperforms existing methods on both targeted sequencing and bulk tumor data. Its high computational efficiency and reconstruction accuracy position MyClone as a promising tool for broad clinical application in cancer treatment. The source code is publicly available at https://github.com/Hansen0413/myclone-code.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/008b5a4c97ae8500318c69c4b98363d3ad251446" target='_blank'>
              MyClone: rapid and precise reconstruction of clonal population structures for tumors
              </a>
            </td>
          <td>
            Qihan Guo, Dan Lv, Bo Lan, Haili Qian, Fei Ma, Ting Chen
          </td>
          <td>2025-10-08</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) chemotherapy regimens are largely uniform, tailored mainly to patient fitness or rare genomic alterations. This strategy fails to capitalize on the disease’s substantial molecular heterogeneity, including variations in copy number alterations, immune infiltration, stromal composition, and cancer cell phenotypes. These features are not independent; they reflect interconnected biological processes and co-evolving pathways that shape tumour progression. Multi-omic integration is therefore essential—not only to capture the complexity of PDAC biology, but also to clarify the molecular basis of tumours classified as ‘intermediate’ by current stromal and cancer taxonomies, thereby enabling rational therapeutic targeting. To this end, we performed imaging mass cytometry on three serial sections of a PDAC tissue microarray (221 resected tumours, ∼4 cores each), generating >800 multiplexed images (40–43 channels) focused on epithelial, immune, or stromal biomarkers. We profiled 76 immune and stromal cell types and states (hypoxic, proliferative, apoptotic, under tension), as well as six tumour phenotypes defined by expression of epithelial transcription factors (GATA6, FOXA2, PDX1), classical markers (AGR2, TFF1, CEACAM6), basal markers (TP63, KRT5, S100A2, CAV1), and other PDAC-associated proteins (S100A4, MMP7, MUC16). These six cancer cell types captured the classical and basal PDAC signatures, along with four discrete “intermediate” states with distinct associations to stromal heterogeneity, RNA subtype (n = 92), tumour ploidy (n = 182), and patient outcome. These phenotypes are also detectable in unmatched single-cell RNA-seq data (n = 163), though with more overlap in marker expression. Using matched 30X whole genome sequencing (n = 182), we identified mutations and copy number alterations linked to shifts in cancer and stromal cell phenotypes, raising the question of which molecular axis best informs clinical prognosis. To address this, we applied a modified version of Stabl, a Lasso-based machine learning approach, to compare across omic layers and identify features most strongly associated with overall survival. Per-modality analysis showed that models incorporating omics outperformed those based on clinical features alone, with imaging data slightly outperforming genomics. Cross-omic integration revealed several prognostically relevant copy number aberrations, fibroblast phenotypes, and cancer cell states. By consolidating information on tumour phenotypes, stromal niches, and genomic alterations, this work aims to focus future drug discovery on the most clinically impactful molecular features in PDAC.



 Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steve Gallinger, Hartland W. Jackson. Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B116.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5cd553302915b28eaa98522e24327f9aa96cbdc" target='_blank'>
              Abstract B116: Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival
              </a>
            </td>
          <td>
            Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, A. Borgida, Edward L. Y. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background: Intrahepatic cholangiocarcinoma (ICC) has a poor prognosis due to late-stage presentation and ineffective systemic therapies. Targeting the tumor microenvironment (TME) in ICC offers new therapeutic possibilities, particularly through tumor-associated macrophages (TAM), which can both promote and inhibit tumor progression. The current study utilized multi-omics analysis to characterize the gene signature of TAM and explore its therapeutic potential in ICC. Methods: Public GEO datasets provided the basis for analysis. Single-cell RNA sequencing (scRNA-seq) data from five ICCs, three adjacent non-tumorous tissues (ANTs), and four healthy liver samples were examined with Python. To validate scRNA-seq findings, bulk RNA-seq data from 27 ICC and 27 matched ANT samples were assessed using R. Differentially expressed genes were identified with adjusted p-values <0.01 and log2-fold changes >1 or <−1. CIBERSORT pipeline analyzed 22 immune cell subtypes in bulk RNA-seq data. STRING database analyzed the contribution of unique TAM-related genes to networks of protein–protein interactions. Results: TAM population demonstrated phenotypic heterogeneity exhibiting partial gene signatures of inflammatory (MS1) and anti-inflammatory (MS2) macrophages. Unique TAM-associated markers, TREM2, CD9, and PRMT10, showed variable expression within the TAM subpopulation. Bulk RNAseq analysis confirmed the scRNA-seq results, highlighting overexpression of TREM2 and CD9 in most ICC samples versus ANT. Immune cell deconvolution revealed decreased MS1 and MS2 macrophages in ICC, and alterations in adaptive immune profile, suggesting immunotolerant TME. STRING database defined TREM2-LGALS3 axis as a potential target for anti-tumor therapies. Conclusions: TAM represents a unique heterogenous population which is primarily found in ICC TME versus ANT or healthy liver tissue The non-uniform expression of unique gene signature demonstrates additional heterogeneity in the TAM subpopulation and suggests that TREM2+ TAM may be desirable targets for anti-TREM2-LGALS3 immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be67d95f17acd65abd3873af222e463c89ca98c4" target='_blank'>
              Defining Gene Signature of Tumor-Associated Macrophages in Intrahepatic Cholangiocarcinoma as Target for Immunotherapy Using Single Cell and Bulk RNA Sequencing
              </a>
            </td>
          <td>
            J. S. Badshah, Ryan M. Lee, Andrea Reitsma, M. Melcher, Olivia M. Martinez, Sheri M. Krams, Daniel J. Delitto, Varvara A. Kirchner
          </td>
          <td>2025-10-29</td>
          <td>Livers</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) cancer associated fibroblasts (CAFs) are a complex and heterogenous group of cells and are a key component of the tumour microenvironment (TME). Our understanding of CAF functional diversity has increased in recent years, however certain aspects of their biology remain elusive, such as the spatial organisation of CAFs, their interactions with other TME compartments, and their contribution to global TME architecture. Better understanding of these will significantly accelerate therapeutic development.



 We performed multiplex immunofluorescence on tissue microarrays of resected human pancreatic cancer (n=227) with an 8-plex CAF panel and a molecular subtyping panel. After assigning bulk cell type (CAF – FAP+CK-, Epithelium – FAP-CK+, Other – FAP-CK-), we integrated CAF morphology, spatial pseudotime analysis, and TME architectural scoring in a hierarchical manner (pixel > cell > neighborhood > tissue module > patient) to define architectural subtypes of PDAC with internal validation.



 Architectural subtypes are defined as follows (name, dominant CAF, median disease-specific survival): Cohesive subtype (iCAF, 40 months) Rich in dense, cohesive, and tightly-packed epithelium. Sparse, loose stroma with a heterogenous population of iCAF-like CAFs which exhibit migration away from epithelium on spatial pseudotime modelling. Inflammatory subtype (roundCAF, 31 months) Dominated by a rich infiltrate of presumed immune cells (FAP-CK-). Highly organised with low spatial entropy. Associated with the classical subtype. CAFs have “cold” marker expression, are round in shape with a smooth surface, and exhibit migration towards epithelium on spatial pseudotime modelling. Activated subtype (Enlarged myCAF, 17 months) Dense stroma infiltrated by large myCAF-like CAFs. High spatial entropy with loss of TME compartmentalisation. Immune excluded. Strongly associated with the squamous/basal-like subtype. Fragmented subtype (Stellate myCAF, 13 months) Loose, dis-cohesive epithelium infiltrated by a CAF-dense stroma rich in myCAF-like CAFs with stellate morphology. High CAF-epithelial cell spatial contact. Associated with poor histological differentiation and poorest prognosis. Hypothesised to represent impending or established (occult) metastasis.



 Morphospatial profiling reveals striking associations between CAF shape, function, and spatial niche, and adds crucial perspective to our understanding of CAF biology beyond current methods. Combined with spatial pseudotime analysis, it offers exciting insight into CAFs in transitory states between subtypes, allowing maximum leverage of datasets which lack a true temporal dimension. In addition, architectural subtyping reveals the complex and heterogenous spatial organisation of the pancreatic TME, and the co-evolution of CAFs with these organisational changes. Integrating this with histopathological analysis lays foundations for the development of novel spatial biomarkers of prognosis, therapeutic development, and treatment response.



 Adam Bryce, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, Leonor Schubert Santana, Fiona Ballantyne, Catherine Ficken, Ian Powley, Australian Pancreatic Cancer Genome Initiative (APGI), John Le Quesne, Fieke Froeling, Stephan Dreyer, David Chang. Morphospatial profiling of cancer-associated fibroblasts reveals architectural subtypes of pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B121.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76e85ad242be0ed691b8aeea89c3fdd7c71569d" target='_blank'>
              Abstract B121: Morphospatial profiling of cancer-associated fibroblasts reveals architectural subtypes of pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            A. Bryce, Silvia Martinelli, R. Pennie, Leah Officer-Jones, Leonor Schubert Santana, Fiona Ballantyne, C. Ficken, Ian R Powley, Australian Pancreatic Cancer Genome Initiative (APGI), John Le Quesne, F. Froeling, S. Dreyer, David Chang
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Phenotypic heterogeneity - distinct molecular and behavioral variations within a population -significantly influences collective invasion and tumor progression. Here, we employ a molecular approach to explore how the underlying metabolic heterogeneity in non-small cell lung cancer (NSCLC) influences invasion and pack patterning. Assessing three-dimensional (3D) pack patterning revealed invasive heterogeneity across NSCLC cell lines and patient-derived samples. Flow cytometry identified IL13RA2 as a biomarker for invasive potential, enabling isolation of subpopulations with distinct invasive characteristics. By integrating a cell surface biomarker (IL13RA2±) with mitochondrial membrane potential (TMRM), we identified and isolated three distinct subpopulations. Two-dimensional (2D) analyses revealed differences in mitochondrial polarity and transcriptional programs associated with migration and oxygen-sensitivity. In 3D, these subpopulations invaded with distinct patterns, from contiguous circular packs to structured chains. Assessments under varied oxygen tension demonstrated that oxygen availability and subpopulation metabolism together influence collective invasion patterning. When recombined at ratios recapitulating the original population, both stochastic and opportunistic cooperative dynamics emerged, dependent on subpopulation composition and oxygen levels. Our molecular approach, integrating cell surface and metabolic characteristics, enables isolation of unique subpopulations and demonstrates that phenotypic and metabolic heterogeneity, population composition, and oxygen availability collectively pattern invasion packs and drive collective invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14710731ead2715e9cfc57e6f8afb7982d2087db" target='_blank'>
              Metabolic programming defines oxygen sensitive subpopulation hierarchies and patterning in collective invasion.
              </a>
            </td>
          <td>
            Veronika Y. Matsuk, Tala O. Khatib, Landon J. Marcus, Isaac E. Robinson, Yuan Liu, Janani K Pasupathy, Mala Shanmugam, Janna K Mouw, Adam I. Marcus
          </td>
          <td>2025-09-24</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Brain-invasion of meningioma is a hallmark of malignant behavior and linked to higher risk of surgical resection and poor clinical outcome. Here, we hypothesized that meningioma tumor ecosystems with invasive behavior enhance cellular communication with residual brain cells allowing an integration of tumor cells into neuronal circuits. To this end, we profiled 42 tumors with and without brain invasive pattern leveraging single cell (Xenium) and array-based (Visium) spatially resolved transcriptomics followed by a spatial graph attention model (sGAT) trained to identify the distance of tumor ecosystems to the brain-tumor boarder. Leveraging explainable AI techniques (integrated gradients and attention) we identified enhanced SPP1+ myeloid cells enriched at the infiltrative boarder. Cellular neighborhood analysis identified enriched proximity of pro-inflammatory myeloid cells, activated astrocytes and excitatory neurons within a distance of 50-100µm to the infiltrative boarder followed by OPC- and oligodendrocytes enhancement peaking at at 200µm distance. To further investigate the tumor-neuronal interaction we leveraged tumor neuronal co-culture systems with genetically diverse meningioma cell models followed by rabies-ΔG-GFP retrograde tracing model demonstrating high degree of neuronal connectivity after 5 days co-culture. To confirm electrically synaptic transmission, patch-clamp and ca2+ imaging revealed functional synaptic contact predominantly activated by acetylcholine (puff). Cross-validation with spatial and single cell transcriptomic data revealed a meningioma neuron-like cell population enriched for acetylcholine receptor expression and localized mainly in the invasive front. Using super-resolution microscopy in brain-invasive meningioma specimens, we demonstrated putative neuron-to-tumor synapses at brain-tumor interfaces. In summary, invasive meningioma are marked by inflammatory reshape activating microglia and astrocytes and accumulation neuronal input on meningioma tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22186b7a6b175f66d08a9c29db91c360e9425639" target='_blank'>
              TMIC-86. Meningioma brain-invasion is linked to inflammatory induced tumor-neuronal connectivity
              </a>
            </td>
          <td>
            J. Benotmane, Yiheng Tang, B. Patel, D. Savran, M. Ratliff, C. Herold-Mende, Sandro M. Krieg, Wolfgang Wick, A. von Deimling, M. Gerstung, I. Bludau, Ekin Reyhan, Svenja K. Tetzlaff, Nina Drewa, Elijah Keifert, Mahammad Bashirov, N. Layer, Y. Yabo, L. van Hijfte, Sybren L. N. Maas, V. Venkataramani, D. Heiland, F. Sahm
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="


 We present a novel integration of data-driven single-cell analysis with mechanistic mathematical modeling to predict the impact of cancer-associated fibroblasts (CAF) on pancreatic ductal adenocarcinoma (PDAC) invasion. Bioinformatic analyses of high-throughput single-cell and spatial molecular assays give us access to human tumor gene expression at the time of measurement, enabling direct characterization of the human tumor microenvironment (TME), complementing our understanding of cell behavior gained from preclinical models. Notably, applying our Bayesian non-negative matrix factorization algorithm CoGAPS to single-cell RNA-seq data from PDAC allowed us to associate a gene program of concurrent epithelial-to-mesenchymal transition (EMT) and inflammation in epithelial cells with the presence of CAFs, validated through organoid co-culture experiments. We hypothesize that this gene program represents a mechanism by which CAFs promote invasive behavior in neoplastic cells over time. Testing this hypothesis requires inference of temporal changes, which is not possible with single-timepoint data collection in genomics data. In contrast to data-driven bioinformatics, mechanistic mathematical modeling can capture processes that happen over time and make predictions about a system. Agent-based models (ABMs) are one form of such mechanistic mathematical models that are well suited for temporal modeling of cellular phenotypes. We developed new software for ABMs that abstract cells into software agents with individual states and their own rules of behavior from single-cell and spatial molecular assays for temporal inference. We apply this technique to understand how CAFs in the PDAC microenvironment support the series of phenotype transitions comprising tumor progression and invasion, by encoding the CAF-induced ability of tumor cells to migrate and a cessation of proliferation in tumor cells, induced by CAF-secreted, non-diffusable substrate as inferred from our genomics analysis. While this provides a virtual framework to test the impact of varying CAF density on tumor cell invasion, it initially lacked integration with the spatial architecture of human tumors. We then apply the ABM to human PDAC spatial transcriptomics data to forecast tumor behavior over time in a spatially-informed setting, varying the hypothesized mechanisms. This revealed that a substrate-driven mechanism results in a broad mesenchymal-like zone surrounding epithelial-like outgrowths, while a cell contact-mediated mechanism results in a single-cell-wide boundary surrounding more significant outgrowth. When we compared these to real tumor compositions, this favors the explanation that secreted CAF substrate drives EMT in PDAC neoplastic cells, which we then validate in organoids with CAF conditioned media. Our work demonstrates how integrating ABMs with genomic and spatial data enables testing of mechanistic hypotheses about tumor-stroma interactions and provides deeper insights into PDAC microenvironments, potentially informing strategies to intercept disease progression.



 Jeanette I. Johnson, Daniel R. Bergman, Tamara Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Yoseph W. Dance, Jacob T . Mitchell, Elizabeth M. Jaffee, Neeha Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, Ashley L. Kiemen, Luciane T. Kagohara, Young H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O'Brien, Elana J. Fertig. Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d57a26173bbcb8bbded169c7c20718d2482d35" target='_blank'>
              Abstract A025: Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion
              </a>
            </td>
          <td>
            J. Johnson, Daniel R Bergman, T. Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Y. Dance, Jacob T. Mitchell, Elizabeth M. Jaffee, N. Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, A. Kiemen, Luciane T. Kagohara, Y. H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O’Brien, E. Fertig
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="PURPOSE
The Tumor Microenvironment (TME) has a vital role in cancer survival and progression, and may play roles in drug resistance and immune escape. To date, few studies have detailed the TME of Ewing sarcoma (EwS).


EXPERIMENTAL DESIGN
We performed spatially resolved transcriptomics of primary treatment-naïve EwS tumor biopsies from patients with or without clinical metastasis, complemented by high-plex spatial proteomic analysis.


RESULTS
We discovered greater stromal enrichment in localized EwS primary tumors compared to metastasis-associated EwS primary tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumor role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling connectivity, suggesting a potential immune-evasive mechanism.


CONCLUSIONS
In addition to the immune recruitment role of tumor-associated stroma, our findings provide spatial insight into the TME of EwS and provide a rationale for the preclinical investigation of MIF as a potential target for Ewing sarcoma immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f9c77e0153cfc111b2db6ef0b69632696638603" target='_blank'>
              Spatial profiling identifies tumor-associated stroma enrichment and MIF as potential immunotherapy targets in primary Ewing sarcomas.
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-09-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc219a64ec3d466c88c4dfd11649123286e2032b" target='_blank'>
              Cross-Species Insights from ART-D to Uncover Evolutionarily Conserved Oncogenic Mechanisms
              </a>
            </td>
          <td>
            Yifan Guo, Jiadong Zheng, Yixin Wu, Kun Zhao, Dian Lv, Wenhan Liu, Mengling Wang, Jin-Yu Lu, Wenyan Xu, Xianping Wang, Xianjue Ma
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) represent a small but critical subpopulation within breast tumors that drive tumor initiation, progression, metastasis, therapeutic resistance, and recurrence. BCSCs are characterized by cell surface markers such as CD44⁺/CD24⁻ and high aldehyde dehydrogenase 1 (ALDH1) activity, and are regulated by pluripotency-associated transcription factors including POU class 5 homeobox 1 (OCT4), SRY-Box transcription factor 2 (SOX2), and Nanog homeobox (NANOG). They exhibit remarkable plasticity and heterogeneity across different breast cancer subtypes, which is further enhanced by epithelial-mesenchymal transition and dynamic interactions with the tumor microenvironment, enabling immune evasion and adaptation to hostile conditions such as hypoxia. Functionally, BCSCs sustain tumorigenesis through self-renewal, contribute to metastatic dissemination as circulating tumor cells, and can enter dormancy in distant niches, leading to late recurrences. Their intrinsic resistance to chemotherapy, radiotherapy, and targeted therapies arises from quiescence, enhanced DNA damage repair, overexpression of drug efflux transporters, and metabolic reprogramming, posing major clinical challenges. Recent advances have improved BCSC identification and isolation through marker-based cell sorting, dye exclusion assays, mammosphere culture, and functional reporter systems. Conventional therapies often spare or enrich BCSCs, highlighting the need for novel treatments. Targeted strategies focusing on stemness-related signaling pathways such as Wnt, Notch, and Hedgehog, as well as regulators including NANOG, ALDH1 and programmed death-ligand 1 (PD-L1), are under investigation. Emerging approaches incorporating natural compounds, nanotechnology, and microenvironmental modulation show promise in overcoming therapeutic resistance. This review integrates current understanding of BCSC biology, heterogeneity, and microenvironmental interactions, providing valuable insights for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0824bd9fb688f1edb70aeb158a043b8788637c" target='_blank'>
              Breast cancer stem cells: Biological roles, microenvironmental interactions, and therapeutic targeting
              </a>
            </td>
          <td>
            Mou Yan
          </td>
          <td>2025-10-11</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive primary brain tumor, characterized by profound intratumoral heterogeneity. Recent studies have identified neural stem cells (NSCs) in the subventricular zone (SVZ) as potential cells-of-origin harboring early driver mutations. However, the stepwise transformation of these rare, mutation-bearing NSCs into heterogeneous tumor populations remains poorly understood. In this study, using a spontaneous somatic mouse model that recapitulates human gliomagenesis, as well as tumor-free SVZ tissues from GBM patients, we identified transcriptionally distinct precancerous cells harboring oncogenic mutations. These cells emerge through oligodendrocyte progenitor-like lineage specification and exhibit aberrant transcriptional programs, including dysregulated translation and extracellular matrix remodeling. Single-cell transcriptomic analysis revealed transitional cell states and progressive acquisition of gliomagenic programs, leading to the emergence of molecularly diverse malignant cell populations. Our findings demonstrate that these OPC-like precancerous cells serve as a critical intermediate in GBM evolution and a driver of intratumoral molecular heterogeneity. This work provides a single-cell atlas of early gliomagenesis and identifies a potential window for therapeutic intervention targeting precancerous cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f90544542d0fd8ecacde0e88ce21d50dda39f62" target='_blank'>
              EPCO-19. PRECANCEROUS OPC-LIKE CELLS DRIVE GLIOBLASTOMA EVOLUTION AND SHAPE INTRATUMORAL HETEROGENEITY
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, J. Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, Jeong Ho Lee
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background Endometriosis (EM) is a chronic gynecological disorder that affects 5% to 10% of women of childbearing age, often causing pelvic pain and infertility. Fibrosis is a hallmark of EM progression, yet its underlying molecular drivers remain poorly understood. Emerging progress in single-cell and spatial transcriptomic technologies offer new opportunities to unravel the cellular heterogeneity and intercellular interactions driving fibrotic and immune remodeling in EM lesions. Methods We performed an integrative multi-omics analysis combining single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to dissect fibroblast heterogeneity and cell–cell communication networks in EM. ScRNA-seq data from 15 EM patients (GSE213216) were processed to identify transcriptionally distinct fibroblast subpopulations. Functional enrichment (GO, GSEA), stemness estimation (CytoTRACE), and trajectory inference were applied to reveal lineage plasticity. CellChat was used to infer intercellular signaling networks, and spatial transcriptomic data from two ectopic lesions (GSM6690475, GSM6690476) were analyzed to validate the spatial distribution of key ligand–receptor interactions. Results We identified 35 cell clusters across EM lesions, with Fibroblast and T/NK cells as dominant populations. Fibroblast were divided into five subtypes, which were associated with extracellular matrix remodeling, immune interactions, and metabolic regulation. Notably, the C2 CXCR4 + Fibroblast subpopulation exhibited high proliferative capacity and stemness characteristics, and mediated signaling pathways involved in immune and fibrotic responses through FN1. Spatial transcriptomic analysis confirmed the local enrichment of these Fibroblast in ectopic lesions, where they were associated with regions of active signaling. Conclusion This study revealed the transcriptional and spatial heterogeneity of Fibroblast in EM syndrome, and identified the C2 CXCR4 + Fibroblast subpopulation as a may represent key driver of fibrosis and immune regulation. Our integrated omics approach provided new mechanistic insights into the pathogenesis of EM and pointed out new targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ccad044a665115ac5faba149f5925920f3e3b05" target='_blank'>
              Fibroblast heterogeneity and FN1-mediated signaling in endometriosis revealed by single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Wenwen Shao, Hongmei Ju, Zhikai Xiahou, Sheng Fang, Rugen Yan, Chunyan Li, Yuan Xu, Pingping Cai
          </td>
          <td>2025-10-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 The tumor microenvironment (TME) of glioblastoma (GBM) is highly complex and dynamic, playing a pivotal role in tumor progression and therapeutic resistance. Comprising diverse components including blood vessels, immune cells, and the blood-brain barrier, the heterogeneous TME forms distinct niches that critically shape tumor behavior. Recent TME-based classification has delineated three subtypes TMELow, TMEMed, and TMEHigh distinguished by varying vascular and immune cell compositions.



 This study focuses on elucidating the vascular interactions within the GBM-TME interface. To achieve this, we developed a brain-specific 3D vascular network by integrating primary human brain-derived endothelial and stromal cells within an extracellular matrix (ECM) on a microfluidic platform. Following the establishment of the brain vascular network model, GBM cell were incorporated to examine tumor-vasculature interactions in a 3D context. GBM integration was performed via two distinct strategies: co-seeding with vascular-forming cells within the ECM at the start of culture, or grafting GBM cells onto a pre-established vascular bed culture to facilitate invasion assays.



 The resulting perfusable model recapitulates key features of the brain vasculature, including the expression of adherens junction markers CD31 and VE-cadherin, tight junction marker Claudin-5, and the strategic alignment of stromal cells along vascular structures. Preliminary results from the integration of GBM in the vascular network model indicate that, irrespective of the seeding method, GBM cells preferentially localize along vascular structures, suggesting an intrinsic tropism toward the vasculature. Notably, co-cultures of GBM with vascular networks exhibited significantly higher proliferative and invasive capacity compared to GBM monocultures, highlighting the pro-tumorigenic influence of the vascular microenvironment.



 The brain vascular network model demonstrates physiological relevance, robustness, and scalability making it suitable for medium- to high-throughput applications. This multifaceted platform offers a unique opportunity to functionally model and stratify GBM TME subtypes in vitro, with the potential to uncover novel therapeutic targets within the vascular and immune niches of the tumor ecosystem.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2431f466f2da1d2f9dc24174aabddca7aa824610" target='_blank'>
              P02.11.B MODELING GLIOBLASTOMA-VASCULATURE INTERACTION ON-CHIP
              </a>
            </td>
          <td>
            P. Emeh, A. Fetah, J. Admiraal, M. Bokkers, K. Jimenez-Cowell, N. Wevers, M. Lamfers, C. Dirven, T. Burton, K. Queiroz
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) progression and recurrence are often driven by therapy-resistant cellular states emerging within hypoxic tumor niches. While hypoxia is a hallmark of GBM, the spatial transcriptomic heterogeneity that underpins resistance in these regions remains poorly characterized. Here, we employ pimonidazole labeling in conjunction with high-resolution spatial and single cell transcriptomics across 29 primary, treatment-naive human GBM samples, syngeneic mouse models, and patient-derived glioma-initiating cells exposed to hypoxia to define the hypoxia-specific tumor microenvironment and its associated cellular programs. We reveal spatially organized sub-niches within hypoxic regions, each supporting diverse cellular programs yet coexisting tumor lineage states with minimal transitions between cell states. Phylogenetic reconstruction based on inferred copy number variations demonstrates consistent evolutionary trajectories from non-hypoxic to hypoxic tumor loci. Specific copy number alterations were enriched in hypoxic regions and associated with immune infiltration and poor prognosis in MGMT-methylated tumors. Cross-species analyses confirmed the conservation of hypoxia-associated gene programs and spatial architectures. Together, our study delineates the cellular, evolutionary, and spatial complexity of hypoxia in GBM, identifies hypoxia-induced genomic alterations that may promote tumor fitness, and offers insights into new therapeutic vulnerabilities within hypoxic niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec812f9197d64f5c26991c527a5afc02eb1748" target='_blank'>
              TMIC-96. Integrated multi-omics reveals distinct hypoxic sub-niches and associated epigenomic and genomic alterations in glioblastoma
              </a>
            </td>
          <td>
            S. Mansouri, Jefferey Liu, Vikas Patil, Ruth Isserlin, Phoebe Lombard, Ronald Wu, Julio Sosa, S. Gandhi, Y. Ellenbogen, O. Singh, Hafsah Ali, C. Velásquez, Bradly Wouters, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer mortality, with minimal improvements in 5-year survival over the past decade. While PDAC originates from premalignant lesions, only a minority of these lesions advance further into malignancy. Distinguishing which lesions are likely to progress would transform enabling timely preventative surgical intervention, but remains a key challenge. In addition to the more well-characterized genetic mutations associated with PDAC onset, recent studies suggest that inflammation in the pancreas microenvironment promotes epithelial chromatin accessibility remodeling to increase their plasticity. This plasticity is thought to enhance cellular sensitivity in response to local signaling cues, which varies drastically across PDAC tumors. We propose that the cell lineage fate of the premalignant epithelial cells is heavily influenced by their surrounding microenvironment. To investigate this, we are using the CosMx spatial transcriptomics platform to profile cellular composition heterogeneity across pancreatic tissue cores from KrasLSL-G12D; Trp53LSL-R172H; Pdx1-Cre (KPC) mouse model of PDAC. By analyzing spatial heterogeneity within the tumor microenvironment, we aim to delineate how local microenvironmental differences might influence epithelial lineage fate decisions into PDAC. Our findings will help understand the factors driving progression from early lesions to invasive cancer, supporting efforts in early PDAC detection and informing clinical strategies for surgical resection.



 Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P. Morton, Crispin J. Miller. Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B118.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb056397ddb7c8e4e0421ffdba57d9fda0a6959d" target='_blank'>
              Abstract B118: Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P Morton, Crispin J. Miller
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Gliomas are the most common malignant brain tumors, characterized by their aggressive nature and poor prognosis. Despite advancements in radiation and chemotherapy, the five-year survival rate for patients with glioma remains low. A significant challenge is the lack of experimental models that accurately replicate the tumor microenvironment and cellular heterogeneity of human gliomas, limiting the understanding of disease mechanisms and the development of effective therapies.



 Here, we present a novel approach to generate glioma organoids with a microenvironment (GlioME) from patient-derived glioma tissue. These organoids maintain the genetic and epigenetic characteristics of the primary tumor and preserve cell-to-cell interactions within the tumor microenvironment, including resident immune cells. Bulk RNA sequencing, whole exome sequencing, and DNA methylation analysis were used to confirm the molecular similarities between the organoids and primary glioma tissues. Immunofluorescence and flow cytometry were used to assess immune cell viability, comparing GlioME with floating glioma organoids.



 Immunofluorescence, flow cytometry, and gene sequencing revealed that the organoids retained an immune microenvironment consistent with that of the original tumor. Unlike traditional cell lines and animal models, GlioMEs retain the immune microenvironment, allowing faster and more accurate immunotherapy screening. GlioME exhibited high responsiveness to chemotherapy and targeted therapy, demonstrating its potential for therapeutic screening applications. Notably, GlioME accurately predicted patient response to the recently approved MET inhibitor, vebreltinib.



 In summary, our results demonstrate that GlioME can retain the parental glioma microenvironment in vitro and reliably test various therapeutic drugs. This organoid model provides a reliable in vitro platform for glioma microenvironment-related research and clinical drug screening.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4662e63a3116553f2fb291a5b7c518e12a2b2372" target='_blank'>
              P12.31.B A NOVEL ORGANOID MODEL RETAINING THE GLIOMA MICROENVIRONMENT FOR PERSONALIZED DRUG SCREENING AND THERAPEUTIC EVALUATION
              </a>
            </td>
          <td>
            C. Zheng, Q. Chen, Z. Fang, P. Wang, D. Zhang, Y. Feng, Y. Zhang, T. Jiang, L. Zheng, Z. Bao
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Pancreatic intraepithelial neoplasms (PanINs), precursor lesions of pancreatic ductal adenocarcinoma (PDAC) that are abundant healthy individuals, rarely progress to cancer despite universally possessing many of the driver alterations associated with malignancy. Recent single cell profiling efforts of PanIN progression in vivo have described marked heterogeneity in chromatin accessibility and transcriptional identity within these neoplasias, but it remains unclear how discrete epigenetic factors play a role in shaping tumorigenesis, or what distinguishes rare neoplastic cells capable of seeding invasive carcinomas from their benign neighbors. Given that more than 80% of patients with PDAC present with unresectable or metastatic disease, further studies on the molecular features of PanIN progression are needed to enable methods of early detection and intervention. Unfortunately, owing to their microscopic size and challenges associated with the isolation of PanINs from surrounding normal epithelium and stroma, it is particularly difficult to address these knowledge gaps in vivo. By generating complex murine derived organoid models, we have built a system that allows us to faithfully reconstitute the evolution from benign PanIN to PDAC in vitro, as well as in vivo through orthotopic implantation into syngeneic mice. As they are easily perturbable, readily generate large amounts of biomass, and closely resemble the stages of PDAC initiation in vivo, these models can be leveraged to directly assay how chromatin remodelers influence malignant transformation. Further, pairing these organoids with our highly multiplexed, transcribable DNA barcoding system and single cell epi multiomics will enable us to study the evolutionary dynamics, chromatin landscapes, and transcriptional identities of clonal populations of PanIN organoids as they progress to cancer. Through these studies, we will gain unprecedented insights into the underpinnings of PDAC initiation, and thus may open up new avenues for the early detection of tumorigenesis.



 Connor J. Hennessey, Minh T. Than, Ben Z. Stanger, Sydney M. Shaffer. Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90b593e2ad7a725999b9d873ea37a620072eeba" target='_blank'>
              Abstract B106: Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models
              </a>
            </td>
          <td>
            Connor J. Hennessey, Minh T Than, Ben Z. Stanger, Sydney M. Shaffer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) and diffuse midline glioma (DMG) are two of the most aggressive brain tumors, marked by significant intratumoral heterogeneity and transcriptional plasticity, which drive resistance to therapy. These tumors share similar transcriptional programs, though the specific genes associated with these programs differ. Transitions between transcriptional states, particularly to the mesenchymal state, which is associated with greater resistance to treatment, complicate treatment strategies. A significant challenge in glioma research is the inability to dynamically track these state transitions in vitro and in vivo. To address this challenge, we introduce CRISPR-TAG reverse genetics, a novel approach that enables real-time monitoring of key transcriptional state markers in glioblastoma and diffuse midline glioma cell lines. Using CRISPR-TAG, we insert a fluorescent reporter sequence at the 3’ end of target genes, linking the fluorescent signal to endogenous gene expression. This allows us to dynamically track changes in gene expression as cells transition between states. We have successfully tagged 30 key cell state markers in patient-derived GBM and DMG cell lines, providing a robust model to investigate the upstream regulators of cell state transitions. By integrating CRISPR-TAG with global CRISPR screens and high-content pharmacological screens, we uncover genetic dependencies that underlie glioma plasticity. For example, we identify modulators of the mesenchymal cell state, including ANXA1 expression, which serves as a critical marker of this resistant state. Additionally, we demonstrate how our method allows in vivo tracking and modulation of cell state transitions in explant cultures, with AI-assisted analysis providing detailed insights into these transitions. In conclusion, CRISPR-TAG reverse genetics lays a critical foundation for building a high-resolution map of glioma state regulators, paving the way for targeted therapeutic strategies. By enabling the precise modulation of cell states, this method offers the potential to complement and enhance existing treatment modalities for GBM and DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eaf38521e14ac09d468437f1dbddc6b4207f994" target='_blank'>
              CNSC-29. RESOLVING THE REGULATION OF GLIOBLASTOMA CELL STATES BY CRISPR-TAG REVERSE GENETICS
              </a>
            </td>
          <td>
            Josephine Heinold, Madeleine Skeppås, Filippa Kruse, Clara Nordquist, Irem Uppman, Sven Nelander
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Bladder cancer is a significant malignancy, for which prognostic prediction and understanding of the tumor immune microenvironment are crucial. B cells play a key regulatory role in this environment, making their study essential for advancing bladder cancer research. Methods In this study, a multi-omics analysis strategy combining single-cell RNA sequencing (scRNA-seq) and bulk RNA-seq was used to establish single-cell transcriptome profiles of tumor tissues from bladder cancer patients, focusing on B-cell populations and their interactions with other cell types in the tumor microenvironment. Large public databases were used to screen for key prognostic genes associated with bladder cancer B cells, and their biomarker expression was verified by in vitro experiments. Results Based on tumor samples from eight patients with bladder cancer and four normal samples, we selected 84, 967 cells for single-cell sequencing analysis. From these, we identified 10, 967 B cells and identified 508 key genes associated with B cells in bladder cancer from five different B cell subtypes. By integrating a large amount of RNA sequencing data, we identified VCL, FLNA, TAGLN, ACTA2, COL6A2, and CALD1 as potential biomarkers for B-cell-associated bladder cancer, and experimentally verified that these markers were significantly lower in bladder cancer patients than in the normal group, and were effective in predicting the survival rate of the patients and the status of the tumor immune microenvironment. Conclusions Using a combination of transcriptomic and experimental validation at single-cell and batch levels, this study provides insights into the key gene signatures of B cells from patients with bladder cancer and their roles in regulating the tumor immune microenvironment, providing new biomarkers and potential therapeutic targets for predicting patient’ prognosis and immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c8bfbe519c06579f4740a61199654ca1af067bc" target='_blank'>
              Comprehensive analysis of single-cell and bulk transcriptomes reveals key B-cell genes and immune microenvironment regulation in bladder cancer
              </a>
            </td>
          <td>
            Lijun Wang, Juan Yang, Zhangxiao Xu, Bo Tao, Yunpeng He, Yuan Zhao, Jian Wu, Yira Ma, Zitao Zhong, Lin Ye
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8be9d1c5a1b99f57bdae54da32d36142c606a15" target='_blank'>
              A unified analysis of cell-type and trajectory-associated pathways in single-cell data using Phoenix
              </a>
            </td>
          <td>
            Yudit Halperin, D. Nachmani, Michal Rabani
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="In this study, we analyzed publicly available scRNA-seq data from primary tumor tissues of 10 Retinoblastoma (RB) patients to explore tumor microenvironment (TME) heterogeneity. Our findings revealed distinct subpopulations of cone precursor (CP) cells, with higher proportions in invasive RB. Differential gene expression and pathway analysis highlighted functional diversity among CP subpopulations, with CP4 showing elevated TGF-β signaling in invasive RB. Cell–cell interaction analysis further identified rewiring of communication networks, with increased fibroblast–CP interactions in invasive tumors. Bulk RNA-seq identified two molecular subtypes, with subtype 1 showing an immunosuppressive TME. Finally, DOK7 was identified as a key gene associated with invasion, with functional assays confirming its role in promoting tumor progression. These results provide valuable insights into RB progression and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0641facdea378a09ac72da42fc0862a4720147" target='_blank'>
              Comprehensive analysis of single-cell and bulk RNA sequencing uncover tumor microenvironment diversity in invasive Retinoblastoma
              </a>
            </td>
          <td>
            Wei Wan, Xiaoqian Chen, Hongli Liu, Ling Yang, Pingping Huang
          </td>
          <td>2025-11-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 There are four established tumor cell states in glioblastoma including astrocyte-like, oligodendrocyte precursor cell-like, neural progenitor cell-like, and mesenchymal-like states. Single-cell RNA sequencing has showed that they are all present with varying distribution between patients. However, since surgery of glioblastoma follows the principle of maximal safe resection, studies have been mainly limited to the necrotic core and contrast-enhancing zone.



 Here, we investigated the cellular organization of the transcriptional states within and outside of the contrast-enhancing zone. This was achieved through a unique set of supramarginal resected glioblastoma samples using large-scale single-cell RNA sequencing of fresh tissue by systematic multi-region sampling from a large number of spatial locations based on 5-ALA fluorescence.



 We discovered that the mesenchymal state is region-specific, highest in the core, at the center of the tumor, shows a gradual decrease with distance from the core and is absent in the low-fluorescent zone and in the periphery, regardless of direction. More importantly, the mesenchymal signature is not only present in tumor cells but also in various other cell types including microglia, macrophages, oligodendrocytes and astrocytes.



 Altogether, this argues that the mesenchymal state is extrinsic, driven by the macro environment, can be induced in completely different cell types and its regional distribution suggests wound-healing and hypoxia to be the main drivers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15d224367def8e1deca00ec99d840b021d59cf13" target='_blank'>
              P02.32.A SINGLE CELL RNA-SEQUENCING OF SUPRAMARGINAL GLIOBLASTOMA RESECTIONS REVEAL THAT THE MESENCHYMAL SIGNATURE IS CONFINED TO THE TUMOR BULK AND IS SHARED ACROSS MALIGNANT AND NON-MALIGNANT CELL TYPES
              </a>
            </td>
          <td>
            J. Kjaer Jacobsen, L. Hu, C. Mannens, K. Siletti, O. Persson, S. Linnarsson
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="
 Diffuse midline glioma (DMG) is an incurable pediatric brain tumor, with radiotherapy offering only transient benefit before inevitable recurrence. While recent studies have revealed the cellular complexity of DMG, the relationship between intratumoral heterogeneity and disease progression — particularly under therapeutic pressure — remains poorly understood. In this study, we integrate longitudinal single-cell- and spatial transcriptomics with single-cell chromatin accessibility profiling and radiomics to investigate how distinct DMG cell states and microenvironmental components evolve in response to treatment, contributing to resistance and the emergence of therapy-persistent niches. We applied this multi-omic approach to a unique cohort of 10 matched diagnostic and autopsy DMG samples obtained from children enrolled in the PNOC023 clinical trial. Spatial transcriptomic profiling revealed that post-treatment autopsy tissues showed consistent enrichment of oligodendrocyte progenitor cell-like malignant populations, suggesting that this stem-like subpopulation is selectively retained or expanded following radiotherapy in DMG. Moreover, integration of single-cell RNA and ATAC sequencing with cerebrospinal fluid microRNA analyses further revealed molecular signatures of resistance and uncovered minimally invasive biomarkers of treatment response. To complement our molecular analyses, we applied radiomic profiling of longitudinal MRI scans to non-invasively monitor changes in tumor heterogeneity over the course of treatment in this patient cohort. In parallel, we developed a data-driven mathematical model using patient-derived DMG xenografts to simulate tumor cell responses to radiotherapy and generate quantitative predictions of preclinical treatment outcomes. We employed this model to explore alternative radiation fractionation schemes and identify optimal dosing strategies that most effectively deplete therapy-resistant tumor cell populations in DMG. Together, these integrated spatial and molecular datasets provide a high-resolution framework for understanding how radiotherapy remodels the DMG tumor ecosystem. By identifying persistent cellular populations and spatial niches that survive treatment, this work lays the groundwork for future precision therapies aimed at targeting the cellular reservoirs that drive DMG recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4395d6f286b7b33609966b9d19ccc4d41894fd90" target='_blank'>
              RBIO-16. Mapping the Spatiotemporal Evolution of Diffuse Midline Glioma Under Radiotherapy
              </a>
            </td>
          <td>
            Costanza Lo Cascio, Shashank Katiyar, Simone Bruno, Carlos A O De Biagi Junior, Danielle Bitterman, Joana G. Marques, Sina Neyazi, Sharmistha Pal, David Kozono, Benjamin H. Kann, Charles Stiles, Sabine Mueller, Franziska Michor, D. Haas-Kogan, M. Filbin
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Nodular lymphocyte-predominant Hodgkin lymphoma (NLPHL) is a rare cancer, and few studies have comprehensively investigated the immune microenvironment and rare lymphocyte-predominant (LP) cells. Here we develop a NLPHL specific lymphocyte-predominant ecotype (LPE) model to identify 34 distinct cell states across 14 cell types that co-occur within 3 LPEs for 171 cases. LPE1 and LPE2 were characterized by immunosuppressive microenvironments with high expression of B2M on LP cells, CD8 T-cell exhaustion, immune checkpoint genes expressed by follicular T-cells, and an improved freedom from progression compared to LPE3 in training (n = 109, with 65% LPE1/2) and validation cohorts (n = 62, with 61% LPE1/2). We validate the co-occurrence and co-localization of cell states using spatial transcriptomics. Protein expression of HLA-I and HLA-II on LP cells and SSTR2 on dendritic cells was predictive of LPE1 (C-statistic=0.69), LPE2 (C-statistic=0.79), and LPE3 (C-statistic=0.60). This study establishes a clinically relevant biologic categorization for NLPHL. Nodular lymphocyte-predominant Hodgkin lymphoma (NLPHL) is a rare cancer. Here, the authors develop a NLPHL specific model to identify 34 distinct cell states across 14 cell types that co-occur within 3 lymphocyte predominant ecotypes (LPEs) for 171 cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687e04fd2e7cfd53da0cba5d8ceaacb2510f65e8" target='_blank'>
              Distinct cell state ecosystems for nodular lymphocyte-predominant Hodgkin lymphoma
              </a>
            </td>
          <td>
            Ajay Subramanian, Shengqin Su, J. Flerlage, S. Alig, S. Younes, L. Marks, Chelsea C. Pinnix, F. Vega, Raphael Steiner, Priyadarshini Kumar, Heidi Mocikova, A. Sýkorová, V. Procházka, Cristiane Milito, Pamela Allen, D. Paulino, Alan Ramsay, T. Flerlage, Monica Palese, Robert West, ChunFang Zhu, Troy Noordenbos, J. Schroers-Martin, Shuchun Zhao, N. J. Park, A. Kalbasi, E. J. Moding, Aaron M. Newman, R.H. Advani, Richard T Hoppe, Maximilian Diehn, Y. Natkunam, A. Alizadeh, M. Binkley
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 Glioblastoma is the most common malignant primary brain tumor in adults and associated with a poor prognosis and a median survival of approximately 15 months, largely due to the lack of effective therapies and inevitable recurrence. A key challenge is the spatial and cellular heterogeneity, evident on MRI as a necrotic center, a contrast-enhancing tumor core, and a non-enhancing, T2/FLAIR-hyperintense infiltration zone. The diffuse infiltration of tumor cells into surrounding brain tissue impedes complete surgical resection and drives relapse from persisting residual cells. While regional molecular differences have been well characterized, their functional implications - particularly drug sensitivities - remain insufficiently understood.



 To address this, we performed MRI- and 5-aminolevulinic acid (5-ALA)-guided tissue sampling from the tumor core and from the infiltration zone of patients with glioblastoma. We utilized pharmacoscopy—a single-cell, microscopy-based drug screening platform—to evaluate region- and tumor cell type-specific drug responses. Cancer cell types were defined based on differential expression of nestin (marking stem-like cells) and S100B (marking more differentiated tumor cells).



 Profiling a library of 59 drugs across region-specific tumor samples from 23 patients revealed pronounced heterogeneity in drug responses across patients, tumor regions and tumor cell states. To overcome this heterogeneity, we computationally predicted complementary drug pairs and validated 20 drug combinations in an independent cohort. The combinations demonstrated enhanced efficacy compared to single agents and effectively addressed glioblastoma’s spatial, cellular, and inter-patient heterogeneity. Notably, drug combinations of oncology and neuroactive drugs outperformed those involving agents from the same drug class (oncology + oncology or neuroactive + neuroactive). Ongoing mechanistical explorations reveal first insights into mechanisms of the complementary drug efficacy.



 Our findings underscore the limitations of monotherapies for patients with glioblastoma and highlight the potential of rationally designed drug combinations tailored to the tumor’s spatial and functional complexity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60062d1c4566c5af52ee0940de47f3ed99103b0" target='_blank'>
              OS12.5.A SPATIALLY RESOLVED FUNCTIONAL PROFILING FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Buck, S. Lee, I. Sakic, R. Wegmann, J. Mena, F. Vasella, J. Friesen, L. Regli, E. Le Rhun, A. Zeitlberger, N. Tatari, G. Hutter, M. Neidert, M. Weller, B. Snijder, T. Weiss
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Glioblastoma multiforme (GBM) is an aggressive malignancy characterized by rapid progression, heterogeneity, and resistance to conventional therapies. Traditional in vitro and animal models often fail to replicate the complex tumor microenvironment and cellular diversity of human GBM, limiting their utility in effective drug testing and translation to clinical application. To address these limitations, we developed a 3D Ex Vivo Patient Tissue (EVPT) assay for GBM which uses patient-derived tumor tissue. This approach maintains the tumor heterogeneity and cytoarchitecture of the tumor, enabling more clinically relevant assessment of therapeutic candidates. In this study, the EVPT model was used to test novel tumor-specific therapies, highlighting potential breakthroughs in the treatment of GBM.



 10 patient tumor tissues from GBM patients were procured from commercial tissue providers and processed within 24 hours of dissection to preserve the TME. Fresh, minimally processed tumor material was embedded in a protein-rich hydrogel and exposed to novel peptide-drug conjugates (PDC): a peptide carrier and a cytotoxic payload from Oncopeptides at various doses in 384-well format for 5-7 days. As controls, we included Staurosporine (maximum killing), Temozolomide (one of the few options and standard of care for GBM), as well as investigational drugs currently in clinical trials: QBS72S (Alkylating Agent) and Panobinostat (Histone Deacetylase (HDAC) Inhibitor). Phenotypic screening was conducted using our proprietary automated high-content imaging (HCI) platform. Additionally, sample characterization was performed on fixed tissue samples by the IHC, validating GBM tumor identity, proliferation and TME presence.



 All patient tissues responded strongly to Staurosporine, while exhibiting resistance to Temozolomide. QBS72S and Panobinostat showed sample-dependent killing activity but were outperformed by the PDC compounds, demonstrating over a five-fold increase in effectiveness. PDC compounds elicited strong to mild killing responses, reflecting the heterogeneity in the samples' reactions (>10-fold difference). Notably, the lead PDC compound demonstrated strong cytotoxic activity, effectively killing tumor cells at highest dose in 9 out of 10 samples.



 Our results demonstrate the novel PDCs' great potential over standard of care Temozolomide, highlighting its promise as a more effective therapeutic strategy for GBM. The EVPT platform effectively aided in lead compound selection, with a clear assay window between less and more potent compounds. Moreover, drug response heterogeneity was accurately captured between patients. This represents a major advancement in preclinical drug testing and glioblastoma therapy development, enhancing the likelihood of successful translation to clinical outcomes. Ultimately, this approach promises to improve treatment strategies and patient outcomes in the long term.



 Ezgi Kaya Aksoy, Ulrica Westermark, Kristin Hammer, Luca Gandini, Jolie Flach, Emma Spanjaard, Stefan Svensson. Gelius, Marrit Putker, Nataliia Beztsinna. A preclinical ex vivo model for glioblastoma captures patient heterogeneity in drug response [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B092.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5f6e0109b0e3b7c28458848e59c50edea2eb7e" target='_blank'>
              Abstract B092: A preclinical ex vivo model for glioblastoma captures patient heterogeneity in drug response
              </a>
            </td>
          <td>
            E. K. Aksoy, Ulrica Westermark, Kristin Hammer, L. Gandini, J. Flach, E. Spanjaard, S. Gelius, Marrit Putker, Nataliia Beztsinna
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary Traditional two-dimensional (2D) cell cultures are widely used in cancer research but fail to replicate the complex interactions that occur in tumors in vivo. Three-dimensional (3D) models more closely mimic the tumor microenvironment by preserving dynamic cell–matrix communication. In this study, we developed and characterized 3D spheroids from two breast cancer cell lines with distinct metastatic potential (MCF-7 and MDA-MB-231). Significant differences in cell morphology, epithelial-to-mesenchymal transition (EMT) markers expression, and functional behavior compared to 2D cultures were observed. The spheroids also showed the distinct expression of key receptors (ERs, EGFR, IGF1R) and matrix molecules (syndecans and matrix metalloproteinases). Bioinformatic analysis further revealed the clinical relevance of these matrix regulators in breast cancer prognosis. Overall, these findings introduce an informative, matrix-free cell culture platform for studying breast cancer progression and potentially exploring new therapeutic approaches. Abstract Background/Objectives: Conventional two-dimensional (2D) cell cultures offer valuable insights into cancer cell biology; however, they lack in replicating the complex interactions present in solid tumors. Therefore, research has shifted towards the development of three-dimensional (3D) cell models that recapitulate the dynamic cell–cell and cell–matrix interactions within the complex tumor microenvironment (TME), better resembling tumor growth and initial stages of dissemination. Extracellular matrix, a key component within the TME, regulates cell morphology and signaling, influencing key functional properties. Breast cancer remains the most frequently diagnosed cancer type in women and a leading cause of cancer-related mortality. Methods: The aim of the present study was the development of breast cancer cell-derived spheroids, utilizing two breast cancer cell lines with differential estrogen receptor (ER) expression profile, and their characterization in terms of morphology, functional properties, and expression of epithelial-to-mesenchymal transition (EMT) markers and matrix signatures implicated in breast cancer progression. To this end, the ERα-positive MCF-7, and ERβ-positive MDA-MB-231 breast cancer cell lines were utilized. Results: Our findings revealed notable phenotypic transitions between 2D and 3D cultures, which were further supported by differential EMT markers expression. Moreover, spheroids exhibited distinct expression profiles of key receptors [ERs, epidermal growth factor receptor (EGFR) and insulin-like growth factor receptor (IGF1R)] and matrix molecules (syndecans, and matrix metalloproteinases), accompanied by altered functional cell properties. Bioinformatic tools further emphasized the interplay between the studied matrix regulators and their prognostic relevance in breast cancer. Conclusions: Overall, this study introduces a simple yet informative 3D breast cancer model that captures key TME features to better predict cell behavior in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454047e64cbe106ca9e24ee78bd584b4344b2361" target='_blank'>
              Spheroid-Based 3D Models to Decode Cell Function and Matrix Effectors in Breast Cancer
              </a>
            </td>
          <td>
            Sylvia Mangani, Christos Koutsakis, Nikolaos E. Koletsis, Z. Piperigkou, Marco Franchi, Martin Götte, Nikos K. Karamanos
          </td>
          <td>2025-10-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6b221cace1b09faa76330c09ce1ca8391e4017e" target='_blank'>
              GLIS3 Marks a Neural-like Progenitor Cell State that Drives Metastasis in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Dennis Gong, Jimmy A. Guo, Jennifer Su, Sueda Cetinkaya, Connor J Hennessey, Patrick Yu, Ananya D. Jambhale, Peter L. Wang, Nicholas J Caldwell, Ashley Lam, Junning Wang, C. Shiau, Kevin S. Kapner, Laleh Abbassi, Seema Chugh, Shatavisha Dasgupta, Jonathan O Nowak, B. Wolpin, M. L. Zhang, Mari Mino-Kenudson, T. Jacks, Andrew J. Aguirre, William L. Hwang
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Breast Cancer is one of the most prevalent malignancies and the cause of mortality among women. Despite technological advancements, diseases have inherent heterogeneity, especially in the tumor microenvironment (TME), which poses a significant challenge for effective therapy. The study explores the spatial heterogeneity of two key molecular markers, EPCAM (Epithelial Cell Adhesion Molecule) and ERBB2 (HER2).  The research was conducted based on spatial transcriptomics and a graph-based clustering method. EPCAM plays a crucial role in maintaining epithelial integrity, and its dysregulation facilitates tumor invasion and metastasis. On the other hand, ERBB2 is a well-characterized oncogene that drives proliferation in HER2-positive breast cancer. The study leveraged 10x Genomics Visium data and a high-resolution histopathology image to analyze gene expression and spatially variable genes (SVGs) within the tissue. The study identified eight functional spatial domains. ERBB2 expression was localized to tumor cores, which reflects the role as an oncogenic driver. On the other hand, EPCAM exhibited a wider distribution, highlighting the involvement in tumor-stroma interactions and epithelial-mesenchymal transition (EMT). The findings demonstrate the utility of spatial transcriptomics in two aspects. The first is resolving tumor heterogeneity, and the second is to underscore the importance of spatial context in understanding the roles of EPCAM and ERBB2 for cancer progression. The insights contribute to the development of precision medicine approaches that target a specific tumor region for improving the therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b117f399cc68a98eef4707a09f6308f49ac08645" target='_blank'>
              Spatial Heterogeneity of EPCAM and ERBB2 Expression in Breast Cancer: Insights from Spatial Transcriptomics
              </a>
            </td>
          <td>
            Muhammad Afrizal Amrustian, Bita Parga Zen, Aminatus Sa'adah
          </td>
          <td>2025-10-27</td>
          <td>International Journal on Advanced Science, Engineering and Information Technology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most common and deadliest primary brain tumor, with no effective therapies and a longstanding record of failed phase 3 clinical trials—often without mechanistic insight into why promising approaches fall short. To address this critical gap, we implemented a novel on-therapy, longitudinal, and multi-regional tissue sampling strategy in patients treated with rQNestin34.5v2, an engineered oncolytic biologic based on herpes simplex virus 1, to explore how tumor and immune ecosystems evolve during treatment. Using high-resolution spatial transcriptomics (Xenium) and targeted spatial proteomics (CyCIF), we profiled over 80 spatially and temporally distinct samples from six patients, capturing over half a million immune, stromal, and malignant cells. This unique dataset enabled unprecedented resolution of the dynamic interactions between tumor and immune compartments in situ over the course of therapy. Our preliminary analysis revealed discrete, virus-associated lymphoid-like structures that formed early after treatment initiation. These structures were enriched in T cells, B cells, and myeloid cells, and were associated with localized reductions in tumor cell density. Over time, we observed a shift from an initially inflamed microenvironment to a more immunosuppressive state. By the time of radiographic progression, these lymphoid structures had largely dissipated—underscoring the importance of on-treatment sampling in capturing transient, treatment-responsive immune architectures. While these findings are preliminary, they suggest a transient immune structure linked to early therapeutic response, whose collapse may contribute to treatment resistance. Our study illustrates how integrative, on-therapy spatial profiling can begin to reveal mechanisms of response and resistance in GBM, providing a framework for refining future immunotherapeutic strategies in brain tumors and other treatment-refractory cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf60db76ebc26534935de49d68b0018aa310569" target='_blank'>
              IMMU-67. On-Treatment Spatiotemporal Profiling of Glioblastoma During Oncolytic ImmunoTherapy
              </a>
            </td>
          <td>
            Charles Couturier, Jennifer Gantchev, Gerard Baquer, Peter Zhang, Alicia D’Souza, Michael S Regan, Kaitlyn Bootz, J. Bonney, Md Amin Hossain, G. Teamlab, Atul Deshpande, Forest M. White, Kenny Yu, E. Chiocca, Nathalie Y R Agar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Treatment of locally advanced unresectable pancreatic cancer remains a major clinical challenge due to pronounced heterogeneity and resistance to standard regimens. Increasing evidence highlights the critical role of the tumor microenvironment (TME) in shaping therapeutic response and driving drug resistance. In this minireview, we summarize recent advances in TME phenotyping and its potential to guide precision therapy. A four-dimensional framework integrating stromal, immune, genomic, and metabolic features has been proposed to better characterize TME heterogeneity. Preclinical and clinical studies indicate that strategies targeting the stroma, modulating immunity, or exploiting genomic vulnerabilities such as homologous recombination deficiency may enhance the efficacy of chemotherapy, immunotherapy, and targeted agents. Dynamic biomarkers, including circulating tumor DNA and carbohydrate antigen 19-9, also show promise for real-time therapy adaptation, although their clinical application remains limited. By synthesizing current evidence, we emphasize the importance of individualized treatment strategies that account for TME complexity. While encouraging, the translation of multiomics phenotyping and biomarker monitoring into routine clinical practice requires standardization, prospective validation, and integration of novel technologies. Future research should focus on establishing reproducible TME-guided models to enable dynamic and personalized therapy for patients with unresectable pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9ebed50c75b679e8298f471ae2f8ee5933e4197" target='_blank'>
              Tumor microenvironment phenotyping guides precision therapy in unresectable pancreatic cancer
              </a>
            </td>
          <td>
            Kai Zhao, Ming-Ming Xiao, Yong-Sheng Yang, Xiao Xiao
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains among the most aggressive and intractable human cancers, driven by profound intratumoral heterogeneity, dynamic cellular plasticity, and robust mechanisms of immune escape. To overcome these barriers, we developed an integrated discovery and engineering pipeline that combines single-nucleus RNA sequencing (snRNA-seq), antibody generation through both immunization and generative AI design, yeast-based directed evolution, and rationally engineered CAR-T cell architectures. We first assembled a high-resolution snRNA-seq atlas spanning malignant GBM, adult brain, and developmental brain tissues. Cross-comparative analysis revealed PTPRZ1 as a consistently expressed, tumor-enriched surface antigen with minimal expression in normal adult tissues. Further analysis across developmental datasets confirmed PTPRZ1 expression during neurodevelopment, particularly in glioma stem cells and oligodendrocyte progenitor-like states, and showed retention of expression across all major GBM differentiation trajectories. These findings suggest that PTPRZ1 marks a persistent cellular state within the glioma hierarchy, making it a stable and tractable immunotherapeutic target despite GBM’s plasticity. To generate high-affinity binders against PTPRZ1, we pursued two parallel strategies: (1) phage display from immunized hosts followed by affinity maturation via yeast-based directed evolution, and (2) de novo nanobody design using structure-aware generative models trained on antibody sequence and structural datasets. Candidate binders were refined through site-directed mutagenesis to enhance specificity and binding kinetics. Given the known challenges of targeting PTPRZ1, including extracellular domain shedding and glycosylation-dependent surface stability, we engineered a suite of TCR-mimetic CAR-T cells incorporating modular costimulatory domains to enable tunable activation thresholds and improved resistance to exhaustion. These constructs emulate core features of physiological TCR signaling, offering increased antigen sensitivity, functional persistence, and reduced tonic signaling in hostile tumor contexts. This work establishes a modular platform for precision antigen discovery and CAR-T cell engineering in GBM and validates PTPRZ1 as a compelling, developmentally anchored immunotherapeutic target. More broadly, it offers a scalable roadmap for rational design of next-generation cellular therapies against solid tumors that evade conventional targeting strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/693e783962995ec2b156273884091139270f0e5f" target='_blank'>
              EXTH-68. Targeting Tumor Heterogeneity And Plasticity in Glioblastoma With Novel CAR T Cells Targeting PTPRZ1
              </a>
            </td>
          <td>
            Aditya Mohan, Barbara Lipes, Charla Gentry, Rushil Yerabelli, Kisha Patel, Bhairavy J Puviindran, E. Blandford, Ariel Gonzalez, Daniel S. Wilkinson, John H. Sampson, Michael Gunn, P. Fecci, Anoop Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern and a high rate of relapse. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, i.e. a hypoxic niche, a leading-edge niche, and a peri-vascular niche, in which malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment, including immune cells, astrocytes, and vascular cells.



 By analysis of single-cell transcriptomic data of human GBM and transgenic mouse models of GBM, we unexpectedly identified pericytes intimately associated with the endothelium as the most active paracrine signaling hub within the tumor parenchyma. Exclusive signaling axes emanating from pericytes were received by endothelial cells, malignant cells, astrocytes, and immune cells. Depletion of pericytes through genetic engineering in several different transgenic and orthotopic mouse models of GBM demonstrated accelerated tumor progression, a disrupted blood-brain-barrier, and premature death of pericyte-poor mice. Mechanistic studies revealed that pericyte deficiency altered the cellular composition of GBM, remodeled the endothelium, and impacted on the immune cell landscape. Specifically, endothelial cells deprived of pericyte association upregulated Colony Stimulating Factor (CSF)-1, which in turn attracted peri-vascular tumor-associated macrophages polarized towards an immune-suppressive phenotype. In the absence of pericytes, the recruited macrophages expressed Hepatocyte Growth Factor (HGF), which reinforced activation of its receptor tyrosine kinase c-Met on a subset of GBM cells harboring an extreme mesenchymal-like state. Indeed, orthotopic implantation of isolated c-Met-expressing GBM cells corroborated their superior tumor-initiating capability and invasive phenotype, compared to cells negative for c-Met.



 We infer that the pericyte represents a critical modulator of GBM development by orchestrating a tumor-suppressive microenvironment; our findings thus highlight the importance of pericyte preservation in the face of current and future GBM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a67e2a70cd46f28d9a303424953a15720ec17e" target='_blank'>
              P02.01.B PERICYTES ORCHESTRATE A TUMOR-SUPPRESSIVE MICROENVIRONMENT BY IMPINGING ON THE CROSSTALK BETWEEN MACROPHAGES AND TUMOR-INITIATING CELLS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, M. Safaee Talkhoncheh, B. Phung, E. Cordero, E. Johansson, R. Rosberg, G. B. Jönsson, A. Pietras, K. Pietras
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Hypoxia is a critical driver of tumor aggressiveness in high-grade gliomas, yet its cell-type-specific effects on immune cell populations within the tumor microenvironment (TME) remain poorly understood.



 We investigated how hypoxia shapes the spatial distribution and functional states of monocyte-derived macrophages (MDMs) and brain-resident microglia (MG) in diffuse astrocytomas and glioblastomas (GB) by using cyclic immunohistochemistry (cIHC), single-cell RNA sequencing, spatial transcriptomics, and in vitro cell culture models.



 Hypoxia induces divergent responses in these myeloid subsets, driving spatial immune patterning. CD163⁻ MDMs dominate hypoxic niches in GB, while MG populations are excluded from these regions, correlating with hypoxia-induced TNF upregulation, stress response signatures, and dampened interferon responses. Conversely, MDMs exhibit a shift towards hypoxia-associated immunosuppressive traits and altered metabolism. GBs display elevated hypoxic intensity compared to diffuse astrocytomas, as supported by hypoxia-response gene expression in TCGA dataset. In vitro, MG show heightened sensitivity to hypoxia-associated acidity compared to MDMs, suggesting together with microglial hypoxia response genes that their exclusion from hypoxic zones results from intrinsic vulnerability.



 Our findings reveal that hypoxia remodels the TME by promoting immunosuppressive MDM accumulation and depletion of MG, creating spatially distinct immune landscapes that may underlie glioma progression. These results highlight hypoxia-driven immune dysregulation as a therapeutic target and underscore the importance of cell-type-specific strategies to counteract TME-driven immunosuppression in malignant gliomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad214ece57abc6b1e7d242c9f8b34d0a20415450" target='_blank'>
              P02.29.B HYPOXIA SHAPES TUMOR IMMUNE MICROENVIRONMENT THROUGH CELL-TYPE DEPENDENT RESPONSES IN DIFFUSE ASTROCYTOMAS
              </a>
            </td>
          <td>
            A. Tiihonen, I. Salonen, I. Koivisto, A. Ritamäki, S. Jaatinen, T. Hyvärinen, J. Lotila, J. Kreutzer, M. Valkonen, G. Karabiyik, S. Mäntylä, M. Mohammadlou, M. Hoikka, J. Beck, R. Rölz, M. Marttinen, M. Nykter, J. Haapasalo, P. Ruusuvuori, S. Parkkila, P. Kallio, S. Hagman, J. Kesseli, V. Ravi, H. Haapasalo, A. Jukkola, K. Joseph, K. J. Rautajoki
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f94703bbaec5cc0e9f11f2c621c2b6ffe36e730" target='_blank'>
              Epithelial-Mesenchymal Plasticity is Associated with Immunosuppressive Features in Canine Mammary Carcinomas
              </a>
            </td>
          <td>
            Kimaya M. Bakhle, Sophie Nelissen, Lynna Li, Ioannis Ntekas, Iwijn De Vlaminck, Santiago Peralta, Andrew D. Miller, Anushka Dongre
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 Ovarian cancer (OvCa) remains largely refractory to immune checkpoint blockade (ICB), owing to a profoundly immunosuppressive tumor microenvironment (TME). Single-cell RNA sequencing (scRNA-seq) of patient tumors has revealed a TME enriched in immunosuppressive macrophages and characterized by exclusion of T cells from the tumor core, features strongly associated with ICB resistance. However, the molecular mechanisms underlying the establishment of this immune landscape remain poorly understood. To identify regulators of OvCa immunity, we employed Perturb-map, a first-of-its-kind spatial functional genomics platform that enables multiplexed CRISPR/Cas9 knockout of candidate genes while preserving the spatial context of tumor–TME interactions in situ. We targeted 35 genes, prioritized through ligand–receptor interaction analysis of human OvCa scRNA-seq data, in an immunocompetent OvCa mouse model. Among these, IL-4 emerged as a key driver of ICB resistance. While IL-4 knockout did not affect tumor growth in untreated animals, it substantially remodeled the local TME and rendered tumors responsive to anti-PD-1 therapy. Unexpectedly, we found that both mouse and human ovarian cancer cells are a source of IL-4, a cytokine traditionally restricted to immune cells. Analysis of patient tumors revealed that tumor-associated macrophages exhibit a robust IL-4, induced transcriptional signature, implicating IL-4 in shaping the immunosuppressive TME in human disease, consistent with our preclinical findings (Cell, 2024). Furthermore, we observed that standard-of-care chemotherapy regimens, including carboplatin, paclitaxel, and cisplatin—induce a dose-dependent increase in IL-4 expression in both murine and human OvCa cell lines. This chemotherapy-induced IL-4 upregulation may contribute to immune evasion and recurrence following initial therapeutic response. Therapeutically, combined blockade of IL-4Rα and PD-1 significantly delayed tumor progression in vivo compared to monotherapy, suggesting that IL-4 signaling is both a necessary and targetable mechanism of immune resistance. These findings are clinically actionable, as Dupilumab, an FDA-approved anti-IL-4Rα antibody, is already in clinical use and well-tolerated. Our results uncover a previously unrecognized, chemotherapy-induced immunoregulatory axis in ovarian cancer and provide a strong rationale for therapeutic strategies that combine IL-4 pathway inhibition with ICB to overcome resistance and improve patient outcomes.



 Gurkan Mullaoglu, Hunter Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian Brown, Alessia Baccarini. Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462188b4ccc770fdbf7b28c6a3fda711e0408972" target='_blank'>
              Abstract PR015: Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Gurkan Mullaoglu, Hunter T. Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian D. Brown, Alessia Baccarini
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most common brain cancers diagnosed in children, often leading to life-long neurological impairments. Whilst targeted inhibitors exist, their effects are often short-lived, highlighting the urgent need for innovative treatments. Notably, some pLGG tumors spontaneously regress with age, suggesting that changes in the tumor microenvironment can arrest tumor growth. In this work, we aim to characterize which signaling interactions promote the growth of pLGG cells and how these interactions change dynamically with time. To achieve this, we have optimized a protocol for introducing pLGG mutations into developing mouse brains via in utero electroporation. This study revealed that brains harboring the common KIAA1549::BRAF fusion mutation have a striking phenotype characterized by increased astrocyte reactivity and myeloid cell infiltration. To better understand how signals derived from immune cells may impact pLGG cell fitness, we are additionally performing a high-throughput cytokine screen. We are using barcoded cytokine constructs tethered to the cell membrane to comprehensively screen the impact of cytokine signaling on the growth of neural stem cells engineered to overexpress common pLGG mutations. Together this work aims to unravel which signaling factors govern the age-dependent growth of pLGG tumors, which may reveal targetable aspects of the tumor microenvironment amenable to therapeutic intervention.



 Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy Chang, Ria Kedia, Michelle Boisvert, David Jones, Timothy Phoenix, Pratiti Bandopadhayay. Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f149e2a5701b9900d3e20ea1c251a5956b1b56d2" target='_blank'>
              Abstract B037: Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas
              </a>
            </td>
          <td>
            Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy H. Chang, Ria Kedia, Michelle Boisvert, David Jones, T. Phoenix, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Meningiomas represent the most prevalent primary intracranial tumors, with the majority classified as WHO grade 1, typically benign tumors exhibiting slow growth. In contrast, WHO grade 2 and 3 meningiomas display increased histopathological aggressiveness, characterized by increased growth, recurrence rates, invasiveness and poorer prognoses. Emerging evidence suggests that tumor microenvironment (TME) plays a critical role in tumor aggressiveness; however, its contribution to meningioma pathobiology is poorly understood. This study aims to establish a patient derived orthotopic xenograft (PDOX) meningioma model to elucidate the role of the TME in driving tumor aggressiveness and to identify novel therapeutic targets. To this end, primary tumor cells derived from meningiomas of various WHO grades were isolated and grown as monolayer cultures, spheroids, and organoids. Additionally, the immortalized meningioma WHO Grade 3 IOMM-Lee cell line was used to represent high-grade tumors. To investigate tumor formation and TME, these cells were implanted intracranially into NMRI nude mice. Tumor development was confirmed via MRI. Subsequently, whole brains were collected either 6-9 months post-inoculation or upon symptom onset, followed by sectioning and analysis through immunohistochemistry. Moreover, tissue samples are being collected for spatial transcriptomics to further investigate molecular profiles. Preliminary analysis of brains implanted with IOMM-Lee cells revealed a 100% tumor take rate, with high Ki67 expression, indicative of significant proliferation. A notable presence of microglia and macrophages (IBA1, CD204) was observed, suggesting involvement of the TME. Current efforts focus on expanding the cohort by including more patients with different grades of meningiomas. This model will provide valuable insights into the cellular composition, heterogeneity, and tumor-immune cell interactions. Ultimately, it will uncover potential therapeutic targets for meningioma and enable their functional validation. This project is supported by the European Union’s Horizon 2020 research and innovation programme under the Marie Sklodowska-Curie grant agreement No. 945322 (LEAD) and Lundbeck Foundation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3b2bddbecba83931920b06f1f448eed8ac49c4e" target='_blank'>
              TMIC-74. A patient-derived orthotopic meningioma model for interrogation of the tumor microenvironment
              </a>
            </td>
          <td>
            Pelin Gulizar Ersan, Claes Ottzen Laurentiussen, J. Haslund-Vinding, Josephine Budde, J. Lange, Mikael Håndstad, Shania Lashkari, N. Hammouda, Karina Stræde, Nadia Hajib, M. Andersen, Bo Halle, F. Poulsen, Andreas Kjaer, Frederik Vilhardt, T. Mathiesen, B. Kristensen
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Bladder cancer (BC) is among the ten most common malignant tumors worldwide. Tumor stem cells contribute significantly to postoperative recurrence and disease progression. Understanding stem cell interactions with other tissue cells and developing a prognostic model may improve BC management. Through single-cell RNA sequencing, we identified tumor stem cells in BC tissue. We identified 91 genes specifically upregulated in the stem cell cluster versus other clusters. Of these, 67 whose high expression correlated with poorer patient survival were defined as high-risk stemness genes (HRSGs). Further analysis showed that the MDK-LRP1 axis is the primary communication pathway between stem cells and other cell types, involving 31 of these HRSGs. Based on these 31 HRSGs, patients were stratified into two stemness clusters (ST cluster A and B), with ST cluster B associated with poorer prognosis. We further selected prognosis-related genes from differentially expressed genes between the clusters to construct a risk model. Patients in ST cluster B exhibited higher risk scores, aligning with clinical outcomes. Among the HRSGs, ACTN1 emerged as a key gene, showing elevated expression in patients with poor survival and advanced disease stages. Immunohistochemistry confirmed significantly increased ACTN1 protein levels in BC tissues. Additionally, protein interface analysis indicated that the Cys104 residue of Midkine potentially interacts with both LRP1 and ACTN1 within a 5 Å distance, suggesting a critical interaction site. These findings provide novel insights into stem cell–mediated BC progression and offer potential prognostic and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a252bfa991c01d37bb86d8900bab209b05cf3f1" target='_blank'>
              Single-cell stemness analysis highlights Midkine-LRP1 pathway and identifies new bladder cancer subtypes
              </a>
            </td>
          <td>
            Cheng Qian, Jin Qin, Zixian Lin, Shenggan Wang, Anmei Deng, Tengda Li
          </td>
          <td>2025-10-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Cancer arises from genetic and epigenetic alterations that reprogram gene expression networks, driving unchecked proliferation. Glioblastoma (GBM), the most common and lethal primary brain tumor, has a median survival of about one year and a dismal 5% five-year survival rate. Despite extensive research, effective treatments remain elusive, underscoring the need for a deeper understanding of GBM biology. Although several genetic alterations have been implicated in gliomagenesis, driving genetic factors responsible for GBM are still obscure, and how these genetic changes interact with epigenetic dysregulation is poorly understood. Here, we identify CHD5, a gene located in the frequently deleted 1p36 chromosomal region, as recurrently deleted in over 20% of GBM cases and further downregulated epigenetically. To investigate the role of CHD5 in gliomagenesis, we generated a glioma-prone mouse model carrying conditional alleles of Pten, Trp53, and Chd5, and employed engineered neural stem cells (NSCs)—a major GBM cell-of-origin—to induce tumor formation. Loss of Chd5 impaired NSC differentiation, enhanced proliferation, and drove in vivo gliomagenesis. Mechanistically, Chd5 forms a NuRD complex that binds the Myc promoter and super-enhancers to regulate its expression. Chd5 loss leads to chromatin accessibility changes, Myc upregulation, and aggressive tumor development. Importantly, reactivating CHD5 in human GBM xenografts significantly extended survival. These findings establish CHD5 as a tumor suppressor in gliomagenesis, offer therapeutic insights for cancers with 1p36 deletions, and present a robust NSC-based platform for functional genetic studies and modeling GBM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccecf93321da1d6d66641061e1225be0f52f0874" target='_blank'>
              EPCO-68. CHD5 IS A TUMOR SUPPRESSOR IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Xueqin Sun, A. Mills
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Glioblastoma (GB), IDH-wildtype, exhibits increased incidence and poorer prognosis in elderly patients, yet the biological underpinnings of this age-related disparity remain poorly understood. Since previous studies have suggested only a limited increase in potential GB-driving genetic alterations in the human brain with age, we hypothesize that non-genetic factors may present aging-dependent contributors to GB pathobiology. In particular, age-associated microenvironmental drivers have been revealed in other cancer types, but their relevance in GB remains underexplored. Moreover, recent GB mouse model studies have suggested that poorer survival in older animals is driven by factors distant from the tumor bulk (“extratumoral”), but this has not been examined in humans due to limited analyses of matched non-tumor brain tissue. To address this gap, we performed single-cell RNA sequencing (scRNA-seq) on formalin-fixed paraffin-embedded tumor samples from 36 GB patients, comprising 18 elderly (65–88, median 75 years) and 18 younger (35–55, median 47 years) individuals. Cohorts were balanced by sex and by the three major epigenetic GB subtypes RTK I, RTK II, mesenchymal, as defined by DNA methylation profiling. Whole-exome sequencing was performed to control for mutational events. For half of the patients, tumor-adjacent normal-appearing cortex was histologically selected, and additionally profiled by scRNA-seq to assess potential age-dependent extratumoral changes. Our spatially stratified approach enabled the detection of diverse malignant cell populations within the tumor core, as well as enrichment of non-malignant cell types often underrepresented in tumor-core analyses, including astrocytes, neurons, and OPCs. Preliminary analyses indicate a region-specific increase in potentially tumor-promoting cellular signatures across immune and glial compartments in older patients, such as more reactive myeloid signatures at the tumor border. Ongoing analyses include integration of these findings with published normal brain datasets and correlation with clinical outcomes. Our study thereby provides a comprehensive framework for understanding aging-associated drivers in human GB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5116a94bf42e3504a7df9be18b8c861fc00afa09" target='_blank'>
              TMIC-76. Deciphering aging-associated drivers of glioblastoma biology at the tumor core and border
              </a>
            </td>
          <td>
            Laurens Zaschke, S. Schmid, D. Teichmann, Peggy Wolkenstein, Jakob Nückles, Christin Siewert, Brendan Osberg, Huy Philip Dao Trong, J. Onken, A. Koch, Matthias Endres, Leif Ludwig, David Capper, Ilon Liu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 For glioblastoma (GBM), like other solid tumors, intratumoral heterogeneity is likely a critical factor in promoting tumor growth, homeostasis, and therapeutic response. Recent applications of single-cell genomics techniques have revealed the scope and nature of GBM cell diversity. However, these techniques do not allow for the experimental manipulation of these same tumor subpopulations, raising a critical barrier to understanding tumor biology and developing effective therapies. To overcome this barrier, we developed endogenous gene reporter systems to enable the isolation, characterization, and manipulation of key GBM subpopulations directly in patient tumor isolates. This approach utilizes streamlined techniques for gene editing in GBM stem-like cells (GSCs), which eliminate the need for the cloning of individual cells. As proof of concept, we applied this approach to two genes in hGSC isolates: OLIG1 (GBM-OPC marker) and SOX9 (GSC/NSC stem cell homeostasis). We targeted the 3’ exon of each gene so that EGFP or RFP are in-frame with the upstream exon but separated by a 2A ribosome skip sequence. We then employed these GSC reporter isolates to comprehensively screen human transcription factors using a CRISPR-Cas9 library and LV-ORF library (1836 genes and 3548 isoforms) to identify transcription factors that activate or repress OLIG1 and SOX9. From these screens we identified 15 genes necessary and sufficient for regulating OLIG1 promoter activity, including known and novel regulators. These included SOX9 itself, which we find is a key factor linking OPC identify with self-renewal of GSCs. In addition, we identified dozens of transcription factor isoforms (validated by long read RNA-seq) that impact reporter activity, as well as ectopic factors not normally expressed in our GSCs which, nonetheless, significantly alter OLIG1 and SOX9 expression. At this meeting, we report results of these screens and present a functional genetic map of factors affecting the GBM-OPC cell identity, self-renewal, and plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9966806d724ae473e21ea19603f70a36860465" target='_blank'>
              STEM-26. COMPREHENSIVE FUNCTIONAL GENOMIC SCREENS REVEAL HUMAN TRANSCRIPTION FACTORS REQUIRED TO MAINTAIN GLIOBLASTOMA OLIGODENDROCYTE PROGENITOR CELL-LIKE IDENTITY
              </a>
            </td>
          <td>
            Anca B. Mihalas, Kelly Mitchell, Sonali Arora, Colette Felton, Mikey Witkowski, Aiden Abadilla, Daniel A. Kuppers, Angela Brooks, Alice Berger, Patrick J. Paddison
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 Cancer associated fibroblasts (CAFs) are a prevalent cell population in the microenvironment of pancreatic cancer. The contribution of cell of origin to CAF heterogeneity is still incompletely understood. Expression of the transcription factor Wilm’s Tumor 1 (Wt1) marks mesothelial cells, as well as a transcriptionally distinct population of fibroblasts in the normal pancreas. WT1/Wt1 expression also defines a population of CAFs in both human and mouse pancreatic cancer. We employed a previously described tamoxifen-inducible model of lineage tracing wherein cells expressing Wt1 and their progeny are labelled with tdTomato. To determine the functional role of these cells, we generated a diphtheria-toxin inducible model to deplete Wt1-expressing cells in orthotopic tumors. By lineage tracing, we found that Wt1 + cells expand in pancreatic cancer, where they give rise to a population of inflammatory pSTAT3+ CAFs. Depletion of Wt1 +stromal cells resulted in smaller tumors with decreased pSTAT3+ cells. Contrary to our expectations, these tumors also had increased immunosuppressive macrophage activation and reduced infiltration of CD8+ and FOXP3+ T cells. Notably, the reduction in tumor weights observed with Wt1 + cell depletion was independent of CD8+ and CD4+ T cells. Instead,Wt1 + cancer-associated fibroblasts expressed high levels of known tumor-promoting ligands that likely interact directly with the tumor epithelium to drive progression. Accordingly, Wt1-expressing cell-depleted tumors had reduced epithelial MAPK activation. In vitro analyses also demonstrated the ability of fibroblast-conditioned media to augment epithelial, KRAS-dependent MAPK activation. Together, our data show that Wt1 + stromal cells represent a tumor-promoting CAF population; while this population might constitute a potential therapeutic target, caution will be needed to avoid exacerbating immune suppression.



 Allison C. Bischoff, Kristee Brown, Emily L. Lasse-Opsahl, Hannah R. Watkoske, Carlos E. Espinoza, Jude Ogechukwu. Okoye, Alberto C. Olivei, Leah M. Green, Ridesh Rai, Stephanie The, Wei Yan, Aaron D. denDekker, Eileen S. Carpenter, Jiaqi Shi, Filip Bednar, Timothy L. Frankel, Yaqing Zhang, Marina Pasca di Magliano. Wilm’s Tumor 1-Expressing Stromal Cells Promote Pancreatic Cancer Growth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A066.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a500aea62251f056d56004630d00f34999c8e117" target='_blank'>
              Abstract A066: Wilm’s Tumor 1-Expressing Stromal Cells Promote Pancreatic Cancer Growth
              </a>
            </td>
          <td>
            Allison C Bischoff, Kristee L Brown, Emily L. Lasse-Opsahl, Hannah R Watkoske, Carlos E Espinoza, J. Okoye, Alberto C Olivei, Leah M Green, Ridesh Rai, Stephanie The, Wei Yan, Aaron D denDekker, Eileen S. Carpenter, Jiaqi Shi, Filip Bednar, Timothy L. Frankel, Yaqing Zhang, Marina Pasca di Magliano
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains a highly lethal malignancy, in part due to its heterogeneous tumor microenvironment. Most PDAC tumors are poorly vascularized and embedded within densely fibrotic stroma; however, in prior computational analyses, we identified a subset of PDAC tumors with hyper-vascular features. In this study, we sought to determine the extent of vascular heterogeneity in PDAC and its functional consequences. Analysis of human tumor specimens revealed that approximately 10-20% of human tumors display features of hyper-vascularity. Using a panel of clonal murine cell lines that recapitulate this heterogeneity, we demonstrate that VEGFA-mediated angiogenesis does not account for the increased vessel density. Functionally, we further show that hyper-vascular tumors display enhanced sensitivity to both chemotherapy and anti-angiogenic therapy. These findings help explain the limited efficacy of VEGFA-targeted therapies in unselected PDAC cohorts and highlight vessel density as a potential biomarker for patient stratification.



 Samantha B. Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo I. Orlen, Samuel D. Zwernik, Kailee M. Hartway, Nina G. Steele, Agnieszka K. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger. Vascular heterogeneity influences treatment response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05cec4b9af7cf61e156104d3e634ae5d707e4e9c" target='_blank'>
              Abstract A127: Vascular heterogeneity influences treatment response in pancreatic cancer
              </a>
            </td>
          <td>
            Samantha B Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo Orlen, Samuel D Zwernik, Kailee M Hartway, Nina G. Steele, A. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Merging evidence suggests that cellular fitness competition between malignant cells and the tumor microenvironment (TME) dictates tumor progression. We recently identified Flower isoforms as molecular “fitness fingerprints,” where cancer cells expressing Flower-Win gain dominance over surrounding cells marked with Flower-Lose. In this study, we uncover that loss of stromal fitness is not an inherent trait but rather a cancer-driven event. Tumor cells secrete a specific long non-coding RNA, Tu-Stroma, via exosomes, which reprograms splicing of the Flower locus in TME cells, enforcing Flower-Lose expression and thereby reducing stromal competitiveness. This cancer-induced shift establishes a permissive niche that accelerates growth, metastatic dissemination, and ultimately shapes host survival in ovarian cancer. Therapeutically, blockade of Flower using a fully humanized monoclonal antibody (Aenya) markedly suppressed tumor expansion and peritoneal spread in patient-derived xenograft models. Importantly, prophylactic administration preserved the integrity of intraperitoneal organs even in the presence of highly aggressive tumor cells. These findings highlight a previously unrecognized mechanism of cancer-driven stromal takeover and establish Flower as a first-in-class therapeutic target with broad implications for preventing metastatic colonization.



 Rajan G. Gogna. Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0864f00ed3ab31470a0c2e77e5565e26d50bb11a" target='_blank'>
              Abstract IA008: Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer
              </a>
            </td>
          <td>
            Rajan Gogna
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Therapeutic resistance remains a critical barrier in oncology, frequently leading to cancer relapse after initial treatment response. Growing evidence suggests the presence of drug-tolerant persisters (DTPs), a rare subpopulation of cancer cells that survives chemotherapy by entering a reversible specific adaptation. Unlike classical cell resistance, the DTP phenotype is independent of genetic changes and maintained through dynamic regulatory mechanisms. DTPs are phenotypically heterogeneous and can exhibit stem-like and quiescent cell phenotypes, non- or slow proliferation, and remarkable plasticity due to a di-pause-like state and executing epithelial–mesenchymal transition (EMT) or transdifferentiation processes. Despite advances in research, the molecular mechanisms underlying DTPs’ biology and their role in cancer relapse remain only partially understood. The review summarizes the current progress in processes that lead to the acquisition of cellular persistence status, which, in turn, constitute areas of vulnerability that can be exploited in cancer therapy. We highlight anti-DTP therapeutic strategies, including epigenetic modification, cell signaling and transcriptional regulation, metabolic reprogramming, and modification of cell interactions within the tumor microenvironment. Furthermore, we focus on the potential role of nanomaterials in the combat against DTPs. Nanoparticles not only act as part of the drug delivery process, enabling precise DTP targeting and enhancing intracellular drug accumulation, but their intrinsic properties can also be used to eradicate DTPs directly or by enhancing the effectiveness of other therapeutic strategies. The integrated approach offers strong potential to eliminate tumor persistence, prevent recurrence, and improve long-term patient outcomes beyond conventional therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495b98c4c7bc3661dae9aa58d4006e54132b7ae3" target='_blank'>
              Targeting Drug-Tolerant Persister Cancer Cells: Can Nanomaterial-Based Strategies Be Helpful for Anti-DTP Therapies?
              </a>
            </td>
          <td>
            Prachi Ghoderao, Eliza Kwiatkowska-Borowczyk, Sanjay Sahare, H. Dams-Kozlowska
          </td>
          <td>2025-11-04</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Pre-invasive breast cancer, or ductal carcinoma in situ (DCIS), shares many morphologic and genomic features with invasive breast cancer, yet most DCIS tumors remain indolent over decades. In this study, we performed spatial analyses of somatic mutation patterns in 18 DCIS tumors to infer the underlying growth dynamics. Experimental data combined with mathematical models and Bayesian computation revealed a growth model that mimics the cellular dynamics of branching morphogenesis, which governs normal mammary gland development during puberty. Unlike a traditional model of clonal evolution, this non-competitive growth model allowed for extensive DCIS spread without the selective pressures of subclone competition that promote the emergence of invasive phenotypes. This finding provides a framework to understand the biological basis for the limited progression of DCIS and adds rationale for ongoing clinical efforts to reduce DCIS overtreatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56eaf0d13e80db0fb1f5ee2e2242518bc8cf64f7" target='_blank'>
              Spatially Discontinuous Mutation Topographies in Ductal Carcinoma in Situ Reveal Non-Competitive Growth Dynamics.
              </a>
            </td>
          <td>
            Marc D. Ryser, Matthew A Greenwald, Daniel L. Monyak, I. Sorribes, Lorraine M. King, Allison Hall, J. Geradts, Donald L Weaver, D. Mallo, Shannon T. Holloway, Ethan Wu, Kevin A. Murgas, Robert B. West, Lars J. Grimm, Carlo C. Maley, Jeffrey R. Marks, Darryl Shibata, E. S. Hwang
          </td>
          <td>2025-10-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Renal cell carcinoma (RCC) evolves within a chronic inflammatory renal niche, where angiogenesis, metabolism, and immune programs synergize to reshape the tumor immune microenvironment (TIME). Recurrent renal damage and tissue hypoxia sustain NF-κB/STAT3 and HIF-VEGF signaling, while a sustained IFN-γ response enhances antigen presentation while inducing inhibitory checkpoints, promoting a state of “inflammation but constrained.” Single-cell and spatial studies reveal early microenvironment heterogeneity and the chemokine-checkpoint paradox: regions enriched in CXCL9/10 coexist with endothelial inertia, cancer-associated fibroblasts (CAFs)-mediated stromal barriers, and metabolic stress, which collectively exclude functional CD8
 +
 T cells. In advanced clear cell RCC, immuno-inflammatory, immuno-excluded, and immuno-desert phenotypes often coexist and undergo transitions during treatment, leading to heterogeneity in response to immune checkpoint blockade (ICB). We propose a modular perspective-the NF-κB/STAT3, HIF-VEGF, IFN-γ circuits and auxiliary regulatory factors-to link stage-specific biology with treatment matching. Integrative biomarkers couple IFN-γ characteristics with angiogenesis/stromal modules and spatial indicators, offering superior predictive power compared to single tests. These insights support the adoption of a combined strategy: integrating vascular normalization or stromal/myeloid cell reprogramming on the basis of ICB, and encouraging the use of longitudinal “immune snapshots” to guide intervention and precision immunotherapy for renal cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bd0a59026e4807122514617a7c05a2fad79de65" target='_blank'>
              From chronic inflammation to immune escape: mapping the tumor microenvironment evolution in renal cell carcinoma
              </a>
            </td>
          <td>
            Hanjun Xu, Baojun Tu, Hui Li, Yong Shan
          </td>
          <td>2025-11-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4a57be54cf710876a27f9e5a47455e094a8d69" target='_blank'>
              Tumors mimic the niche to inhibit neighboring stem cell differentiation
              </a>
            </td>
          <td>
            Yang Zhang, Yuejia Wang, Jinqiao Song, Lizhong Yan, Ziguang Wang, Dongze Song, Yudi Zhao, Shaowei Zhao
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD), a subtype of non-small cell lung cancer (NSCLC), has high incidence and poor prognosis. Although anti-programmed cell death protein 1 (PD1) therapy and epidermal growth factor receptor (EGFR) inhibitors benefit some patients, many remain unresponsive. Genome-wide association studies (GWAS) can identify risk loci, while single-cell transcriptomics enables exploration of genetic variations and mechanisms in LUAD. We integrated GWAS data from FinnGen, IEU OpenGWAS, and GWAS Catalog with single-cell and spatial transcriptomics from GEO to investigate LUAD genetics and pathology. GWAS analyses identified SNP loci, risk factors, LD Score, and SMR results, revealing a novel apoptosis-related locus, Serine/Threonine Kinase 24 (STK24) (SMR P < 0.05). Single-cell analysis identified STK24-positive epithelial cells (STK24posEpi) and their transcription factors (SCENIC), developmental trajectory (Monocle), and tumor microenvironment roles (CellCall). Spatial mapping (RCTD) and trajectory analysis (stlearn) were combined with cell communication (CellChat), spatial signal flow (Commot), and co-localization (Misty). Bayesian deconvolution linked STK24posEpi to TCGA-LUAD bulk transcriptomes, assessing overall survival (OS), immune infiltration, and clinical traits. A prognostic model (STK24SuperPC, mean c-index 0.61) was built using multiple machine learning algorithms to predict survival and immunotherapy response. Laboratory experiments tested STK24’s effects on LUAD cell proliferation and apoptosis. STK24 was identified as a LUAD risk factor (pSMR < 0.05). STK24posEpi communicated with fibroblasts, endothelial cells, and others via MIF and PDGF pathways, and was associated with apoptosis (Importance > 0.5), metastasis (Importance > 2), DNA repair (Importance > 1), and DNA damage (Importance > 2). Located at the origin of the trajectory, STK24posEpi correlated with poor prognosis (HR > 0, P < 0.05) and reduced lymphocyte infiltration (Pearson’s r < 0, P < 0.05). STK24 overexpression enhanced LUAD cell proliferation, migration, and colony formation, while reducing apoptosis, increasing BCL-2 and Caspase-3, and decreasing Cleaved-Caspase-3 expression. STK24, an apoptosis-related gene, promotes LUAD progression by enhancing proliferation and inhibiting apoptosis, with STK24posEpi interacting in the tumor microenvironment through PDGF and MIF pathways. These findings offer potential therapeutic targets and support personalized LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038883aea606036c62cbeb3f18e89605908569d" target='_blank'>
              Genome-wide association, single-cell, and spatial transcriptomics analyses reveal the role of the STK24-expressing positive cells in LUAD progression and the tumor microenvironment, identifying STK24 as a potential therapeutic target
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Yuheng Jiao, Meng Hou
          </td>
          <td>2025-10-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Ovarian cancer is a heterogeneous disease with multiple molecular subtypes that influence treatment response and prognosis. Identifying these subtypes from high-dimensional genomic data remains challenging due to the complexity and noise inherent in transcriptomic datasets. In this pilot study, we developed a multi-stage computational framework to reduce dimensionality and uncover biologically meaningful subgroups. Starting from over 63,000 mRNA features, variance-based filtering, correlation pruning, and supervised selection narrowed the dataset to 80 highly discriminative transcripts. Using these features, we constructed co-expression similarity networks, and visual inspection of the network topology revealed four distinct groups among seven ovarian cancer cell lines. These subtypes aligned with known biological and mutational profiles including TP53-driven high-grade serous groups, PI3K/AKT- and ARID1A-associated clear cell/endometrioid groups, a drug-resistant OVCAR-3 group, and a hybrid IGROV-1 profile. This integrative approach demonstrates that combining feature selection with network modeling provides a powerful method for resolving ovarian cancer heterogeneity and improving subtype-specific insights. Abstract Background/Objectives: Ovarian cancer is a heterogeneous malignancy with molecular subtypes that strongly influence prognosis and therapy. High-dimensional mRNA data can capture this biological diversity, but its complexity and noise limit robust subtype characterization. Furthermore, current classification approaches often fail to reflect subtype-specific transcriptional programs, underscoring the need for computational strategies that reduce dimensionality and identify discriminative molecular features. Methods: We designed a multi-stage feature selection and network analysis framework tailored for high-dimensional transcriptomic data. Starting with ~65,000 mRNA features, we applied unsupervised variance-based filtering and correlation pruning to eliminate low-information genes and reduce redundancy. The applied supervised Select-K Best filtering further refined the feature space. To enhance robustness, we implemented a hybrid selection strategy combining recursive feature elimination (RFE) with random forests and LASSO regression to identify discriminative mRNA features. Finally, these features were then used to construct a gene co-expression similarity network. Results: This pipeline reduced approximately 65,000 gene features to a subset of 83 discriminative transcripts, which were then used for network construction to reveal subtype-specific biology. The analysis identified four distinct groups. One group exhibited classical high-grade serous features defined by TP53 mutations and homologous recombination deficiency, while another was enriched for PI3K/AKT and ARID1A-associated signaling consistent with clear cell and endometrioid-like biology. A third group displayed drug resistance-associated transcriptional programs with receptor tyrosine kinase activation, and the fourth demonstrated a hybrid profile bridging serous and endometrioid expression modules. Conclusions: This pilot study shows that combining unsupervised and supervised feature selection with network modeling enables robust stratification of ovarian cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7f756c19c8351ff216d426702a2f26787540f1a" target='_blank'>
              Subtype Characterization of Ovarian Cancer Cell Lines Using Machine Learning and Network Analysis: A Pilot Study
              </a>
            </td>
          <td>
            R. Thelagathoti, Dinesh S. Chandel, Chao Jiang, Wesley A. Tom, Gary D. Krzyzanowski, Appolinaire A. Olou, M. R. Fernando
          </td>
          <td>2025-10-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Gliomas are highly heterogeneous tumors composed of dense cores and diffusely infiltrating margins, where glioma cells interact with non-neoplastic cells, including microglia—the brain’s resident immune cells. To investigate spatial microglial states, we isolated live microglia from paired patient glioma core and margin samples, and performed single-cell RNA sequencing. We found that margin-associated microglia displayed distinct transcriptional profiles from those in the core. Specifically, purinergic signaling pathways were upregulated at the margin, while chemokine signaling, interferon response, and antigen presentation pathways were enriched in the core. Using Monocle pseudotime analysis, we constructed microglial activation trajectories and identified P2RY12 as the gene most negatively correlated with inflammatory progression, suggesting it may play a role in early-stage activation at the tumor margin. To model this in vitro, we used human iPSC-derived microglia, which recapitulate microglial activation states seen in vivo. ATP stimulation of iPSC-microglia, mimicking increased levels of extracellular ATP (eATP) in the glioma microenvironment, significantly upregulated inflammatory genes seen in the late-stage pseudotime signatures, and downregulated genes in the early-stage pseudotime signature. This indicates that ATP stimulation induces a shift along the inflammatory trajectory towards later stage activation phenotypes. To probe therapeutic relevance, we treated microglia with the purinergic receptor inhibitor PPADS, which attenuated ATP-induced transcriptional programs, including interferon signaling. These results suggest that P2RY12 activation by eATP modulates early microglial activation. We are currently testing the effects of P2RY12 knockdown in the iPSC-microglia model to assess responses to ATP and glioma co-culture. Future directions include acute brain slices and in vivo mouse glioma models to evaluate how targeting P2RY12 affects microglial behavior and tumor progression, potentially informing novel immunomodulatory strategies for glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8293ff64f0c760324290f880f6d68ca70039fdf" target='_blank'>
              TMIC-105. P2RY12 as a potential therapeutic target for disrupting tumor microenvironment– microglia cross talk
              </a>
            </td>
          <td>
            Nkechime Ifediora, Nadine Khoury, Alexander R. Goldberg, A. Mahajan, N. Humala, J. Furnari, Clara Stucke, Kristin R Swanson, Gunnar Hargus, Andrew A. Sproul, Vilas Menon, Marta Olah, Peter Canoll
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Single-cell technologies have transformed the understanding of disease heterogeneity, but linking cell-level phenotypic alterations to clinical outcomes becomes increasingly challenging as single-cell datasets continue to expand. This is further complicated by the lack of interpretability in existing methods and the difficulty of detecting interaction effects-nonlinear dependencies between factors like sex, age, and disease. To address this, a novel explainable machine learning method, CellPhenoX, is developed to identify cell-specific phenotypes and interaction effects linked to clinical outcomes. CellPhenoX integrates classification models, explainable artificial intelligence (AI) techniques, and a statistical framework to generate interpretable, cell-specific scores to uncover condition-associated cell populations. Extensive benchmarking and applications demonstrate the efficacy of CellPhenoX across diverse single-cell study designs, including the dedicated and disease-motivated simulations, binary disease-control comparisons, and severity-stratified patient cohorts. Notably, CellPhenoX identifies an activated monocyte phenotype in COVID-19, with expansion correlated with disease severity after adjusting for covariates and interactive effects. It also uncovers a fibroblast-specific state transition gradient predicting tissue inflammation in chronic diseases, and identifies therapy-induced T cell changes and biomarkers linked to the tumor microenvironment. By integrating interpretability into clinical classification, CellPhenoX offers a powerful framework for translating single-cell findings into clinical impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9804ebab4ea48d9af38e70f8f35d67c13d52c204" target='_blank'>
              CellPhenoX: An Explainable Machine Learning Method for Identifying Cell Phenotypes To Predict Clinical Outcomes from Single-Cell Multi-Omics.
              </a>
            </td>
          <td>
            Jade Young, Jun Inamo, Zachary Caterer, Revanth Krishna, Fan Zhang
          </td>
          <td>2025-09-23</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Glioblastomas are highly aggressive, treatment-resistant malignancies with a grave prognosis. Glioma cells hijack the brain’s native immune surveillance, primarily led by resident myeloid cells or microglia, to create an immunosuppressive tumor microenvironment (TME), making GBM extremely difficult to treat. These glioma cells subvert the anti-tumorigenic functions of microglia and reprogram them to create a TME that facilitates diffuse infiltration and disease progression. However, the mechanisms by which glioma cells reprogram microglia from anti-tumorigenic to pro-tumorigenic remain unclear, primarily due to (i) our inability to isolate human tumor-associated microglia; (ii) our inability to model tumor-microglia interactions within the TME; and (iii) the lack of models that include or allow manipulation of microglia while reliably recapitulating human disease. To overcome these critical challenges, we have developed a novel human induced pluripotent stem cell (hiPSC)-derived 3D assembloid model. This engineered model allows us to study the complex interactions between human glioma and microglia, as well as their crosstalk with other CNS cell types within the TME. In this model, hiPSC-derived microglia (iMG), astrocytes (iAstro), and neurons (iN) are combined to generate “mini-brains,” which are co-cultured with patient-derived GBM spheroids to create 3D assembloids. Using single-cell transcriptomics, high-resolution 3D microscopy, and functional analyses, we show that 3D assembloids reliably recapitulate the tumor-immune interactions within GBM. The presence of GBM in 3D assembloids reprogram microglia from a “resting” state to a “reactive” state and induce microglia migration from the mini-brain towards the glioma cells to create an immunosuppressive TME, similar to patient tissue samples. Additionally, we identified cytokine signaling pathways that are necessary for microglia recruitment from the mini-brain to the TME. The transformative potential of this work is not limited to GBM. To demonstrate the broad utility of this model, we also developed a 3D assembloid model to investigate tumor-immune crosstalk in brain metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77776f3af7a9709ffe5511a1950f895c6ebf1093" target='_blank'>
              TMIC-112. Modeling tumor immune microenvironment in GBM using 3D assembloids
              </a>
            </td>
          <td>
            Shruti Jain, Sofia Tosoni, T. Trinh, Wanhua Li, M. Wernig, M. Gephart
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Brain metastases often originate from circulating tumor cells (CTCs) that become arrested within cerebral microvessels. Recent findings indicate that this process induces localized hypoxia, which in turn upregulates angiogenic mediators such as Angiopoietin-2 (Ang-2), thereby promoting vascular remodeling and facilitating extravasation of tumor cells into the brain parenchyma. Nevertheless, the early transcriptional landscape of the adjacent brain tissue—the so-called pre-metastatic niche—remains poorly defined.



 To simulate early metastatic seeding, syngeneic murine lung and breast carcinoma cells were injected into the right carotid artery of immunocompetent mice. The contralateral, non-injected hemisphere served as an internal control to allow intra-animal comparisons, reduce biological variability, and minimize animal usage. Brain tissue was collected at 1 and 4 days post-injection, processed via formalin fixation and paraffin embedding, and subjected to histological evaluation. Comparative bulk RNA sequencing of tumor-bearing and control hemispheres was performed, complemented by high-plex spatially resolved single-cell transcriptomic profiling to detect localized gene expression patterns.



 Bulk RNA sequencing revealed no significant transcriptional differences between hemispheres. In contrast, spatial transcriptomic analysis uncovered robust, spatially restricted gene expression changes in the tumor-bearing hemisphere. These alterations involved genes associated with vascular remodeling, hypoxia-responsive pathways, and neuronal connectivity. Notably, spatial mapping demonstrated that the expression of several key transcripts peaked near sites of arrested tumor cells and declined with increasing distance, indicative of a highly localized, tumor-induced host response.



 This study reveals distinct, spatially confined transcriptional alterations that define the early pre-metastatic niche in the brain. By leveraging advanced spatial single-cell transcriptomics, we provide a powerful framework to dissect the molecular cues underlying early metastatic interactions within the native tissue microenvironment. These insights not only deepen our understanding of early brain metastasis biology but also inform the development of innovative strategies for therapeutic intervention at the pre-metastatic stage.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6ab71d6adf66b6036a2c3fb3715feb43793308" target='_blank'>
              P18.25.B SPATIALLY CONFINED TRANSCRIPTIONAL RESPONSES CHARACTERIZE THE EARLY PRE-METASTATIC NICHE IN BRAIN METASTASIS
              </a>
            </td>
          <td>
            W. R. Hube, F. Schallerer, L. Bergmayr, R. Piecyk, T. Eska, K. Mueller, E. Nicolaishvili, L. Sevenich, K. Unger, P. Jurmeister, L. von Baumgarten, P. Harter
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal tumor. PDAC also shows poor response to many conventional treatments and is refractory to immunotherapy. We hypothesize that these challenges arise from the highly heterogeneous and immunosuppressive tumor microenvironment (TME) in PDAC. Therefore, to comprehend the players behind this immunosuppressive TME, we integrated 10 publicly available datasets into a robust PDAC scRNA atlas of 941.608 cells covering 201 patients (159 PDAC, 24 non-cancer, and 18 metastatic PDAC). The TME showed distinct composition for each tissue, predominating a fibro-inflammatory phenotype with infiltrating Tregs and dysfunctional CD8 T cells in primary PDACs. In contrast, adjacent normal tissue exhibited enrichment of tissue-resident memory CD8 T cells (p < 0.05). Metastatic PDAC was characterized by a reduction in stromal cells combined with myeloid and naive T cells infiltration. Comparison between treated and untreated PDAC revealed the enrichment of cancer-associated fibroblasts (CAFs) in treated patients. Besides, CD8 T cells from untreated patients exhibited higher levels of genes previously linked to immunotherapy response compared to treated patients. Non-negative matrix factorization (NMF) of the malignant cells unraveled 8 distinct metaprograms (MP). MP3 and MP7 overlapped with the well-known Classical subtype, whereas MP5 correlated with the Basal subtype. MP2 exhibited high expression of MHC-II genes and was enriched in metastatic lesions. Next, we investigated the tumor architecture using public spatial transcriptomic datasets. Spot-level deconvolution revealed 12 distinct cellular niches. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications.



 Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, Alessandro Bersch Osvaldt, Simone Marcia Dos Santos Machado, Eduardo Cremonese Filippi-Chiela, Cristina Bonorino. Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B126.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a30afbd64e95a911caaa3a76b2513c14671d0c3e" target='_blank'>
              Abstract B126: Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastatic glioblastoma multiforme (GBM) is known for its dismal prognosis due to the dissemination of single cells throughout the brain parenchyma and along white matter tracts, resulting in heightened resistance to therapies. Understanding the intricate relationship between cell migration, physical confinement, and chemotherapeutic resistance in GBM is imperative for advancing treatment strategies. In this study, we employed G55, a representative migratory GBM cell line, to investigate this phenomenon. We generated three distinct cell populations: (1) cells migrating without confinement, assessed via the Scratch assay; (2) cells migrating a short distance (10 μm) under confinement, examined through the Transwell assay; and (3) cells migrating long distances (>100 μm) under confinement, studied using the Microchannel assay. Comparative analyses of protein expression profiles and chemotherapy sensitivity among these groups revealed that migration combined with physical confinement plays a pivotal role in augmenting chemotherapeutic resistance in interstitial invasive cancer cells. Moreover, we demonstrate the utility of the microchannel device, which facilitates controlled cell migration under physical confinement, as an effective in vitro tool for investigating metastatic cancer and associated treatment resistance. This study sheds light on the mechanisms underlying GBM progression and highlights potential avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9ad0e81447d500e0c440ddcca92f5191cac6d51" target='_blank'>
              Physical confinement and distance of migration cooperatively enhance chemotherapeutic resistance in migratory GBM cells
              </a>
            </td>
          <td>
            Qionghua Shen, Adam Germain, Calvin Kong, Young-Tae Kim
          </td>
          <td>2025-09-30</td>
          <td>Physical Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 The molecular landscapes of epithelial ovarian cancers (EOCs) have largely been profiled using bulk sequencing techniques. Consequently, biologically relevant signals from the tumor and its microenvironment (hereafter stroma) are often conflated, dampening both tumor and stroma-specific observations.



 We profiled the carcinoma and stroma compartments separately across clear cell (CCOC), endometrioid (ENOC), and high-grade serous ovarian cancer (HGSC) using laser capture microdissection followed by sample-matched whole genome bisulfite sequencing and RNA-seq.



 Histotype designations drove sample segregation in the carcinoma compartment yet were not sufficient to explain top expression or methylation variations in the stroma. Unsupervised clustering of the top variable genes in the stroma identified four stromal subtypes (S1-S4) shared across the three histotypes. Of these, S1, S2, and S4 were the most distinguishable and could be recovered in external datasets. S1 was enriched for RNA splicing processes (SFSWAP, RBM25), S2 for metabolism (GATM, ALDH1A1) and S4 for remodeling of the extracellular matrix (FN1, POSTN, SFRP2). While the characteristics of S4 echo previous reports of the desmoplastic/highly fibrous stroma reported in HGSC, we show that S4 is also present in CCOC and ENOC tumors. Using HGSC samples from TCGA, we observed a strong correlation of S4 markers with the Mesenchymal subtype and an inverse correlation of S4 and S2 markers, suggesting mutual exclusivity of these stromal subtypes. Examining the TCGA-derived subtype markers in our microdissected data revealed that the canonical Immunoreactive and Mesenchymal subtypes were driven by signals from the stroma, rather than the carcinoma. Conversely, Differentiated and Proliferative markers were enriched in the carcinoma. Clustering of pure carcinomas revealed a Proliferative versus Differentiated dichotomy within the carcinomas, particularly within ENOC and HGSC tumors. CCOC tumors were systematically depleted of the Proliferative subtype, consistent with their indolent nature. We next investigated if genetic, epigenetic or transcriptional cues in the carcinoma compartment are associated with the characteristic of its TME but failed to detect differences at 5% FDR. This suggests that the stromal subtype may not be shaped solely by instructions from the cancer cells and that a more complex, reciprocal cross-talk exists between the carcinoma and its TME.



 Epigenetic and transcriptional differences in CCOC, ENOC and HGSC carcinomas can be largely attributed to histotype, yet this is not the case for their associated TMEs. With the growing importance of TME in influencing patient outcomes, and response to treatment, understanding the nature of a tumor’s TME is crucial. Thus, coupling the stromal subtypes (S1-S4) with the well-established histotype designations of EOCs may provide a more robust way to classify EOCs and inform optimal treatment strategies that account for the characteristic of the tumor stroma.



 Svetlana Djirackor, Karolin Heinze, Ian Beddows, David Sokol, Bianca Ribeiro de Souza, Martin Koebel, Marie Adams, Michael Anglesio, Hui Shen. Parallel epigenetic and transcriptomic profiling of carcinoma and stroma compartments across major ovarian cancer histotypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B007.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67d3fdc66021784b9ea5f2df0bff0c3026e49fb0" target='_blank'>
              Abstract B007: Parallel epigenetic and transcriptomic profiling of carcinoma and stroma compartments across major ovarian cancer histotypes
              </a>
            </td>
          <td>
            Svetlana Djirackor, Karolin Heinze, Ian Beddows, David Sokol, Bianca Ribeiro de Souza, M. Koebel, Marie Adams, M. Anglesio, Hui Shen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 We aim to develop clinically relevant human tumor organoid models of glioblastoma for optimizing chimeric antigen receptor (CAR) T cell therapy. We therefore have developed three-dimensional (3D) organoid cultures derived from patient tumors to preserve spatial cell-cell and cell-matrix interactions, offering advantages over traditional two-dimensional (2D) culture models. Although Matrigel is commonly used in 3D cultures, its batch variation, xenogeneic origin and non-brain relevant components raise concerns for modeling brain tumors and evaluating CAR T therapies. To address this, we evaluated brain-relevant hydrogels that contain hyaluronic acid, a major component of brain tumor extracellular matrix. We tested a panel of conditions and found one promising candidate, a mixture of PhoGel and PhotoHA (termed PGPH). PGPH allowed reliable tumor sphere migration, stable droplet formation, efficient sample recovery, and cost-effectiveness. Comparable to Matrigel, PGPH supports migration and growth of several patient-derived brain tumor spheroid cell lines. Transcriptomic analyses revealed similarities and differences between Matrigel and PGPH when culturing tumor cells or T cells. Notably, we demonstrated that T cells could migrate and kill organoid tumors in PGPH, and by incorporating soluble immune regulators, we were able to assess how these reconstructed tumor microenvironment (TME) compartments interact with CAR Ts and influence their killing potency. Moreover, PGPH enabled efficient and rapid tumor-derived explant culture (PDE) that closely resembled the native tumor ecosystem by preserving parental tumor heterogeneity and TME features, allowing for drug testing only 7 days after surgery. Lastly, we were able to evaluate CAR T cytotoxicity in PDE and are investigating the correlation of ex vivo CAR T responses with clinical responses in ongoing trials. In summary, this chemically defined, brain-relevant hydrogel offers opportunities to preserve the native or reconstructed TME, which could facilitate the translational success of CAR T treatments for brain tumors from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989f6d74db56907e7da73213b1e77e7bc2a920ba" target='_blank'>
              TMOD-27. Patient-derived glioblastoma organoid cultures for CAR T cell evaluation
              </a>
            </td>
          <td>
            Xiujian Ma, Eric Ma, Franklin Chen, Sean Sepulveda, Hernan Reza, Michael Rothe, M. Gutova, B. Badie, Christine Brown
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="


 KRAS^G12D mutations represent a significant oncogenic driver, prevalent in highly aggressive malignancies such as pancreatic ductal adenocarcinoma (PDAC) and colorectal cancer (CRC). The recent development of direct KRAS^G12D inhibitors has ushered in a new era of targeted therapy, offering unprecedented opportunities to address these previously intractable cancers. This study aimed to explore and optimize therapeutic strategies by thoroughly evaluating the efficacy of MRTX1133, a novel selective non-covalent KRAS^G12D inhibitor, in conjunction with various immunotherapeutic modalities, particularly those involving natural killer (NK) cells.



 We utilized patient-derived xenograft and syngeneic murine cancer models to systematically characterize alterations of immune landscapes within tumor microenvironment (TME) upon KRAS^G12D inhibition. Comprehensive and in-depth assessment of immune cell function and single-cell level transcriptional activity was performed using flow cytometry and single-cell RNA sequencing analysis.



 Our findings reveal that MRTX1133 not only suppresses tumor progression but also profoundly enhances the activation of infiltrated natural killer (NK) cells within the tumor microenvironment. Functional depletion of NK cells partially, yet significantly, attenuated the anti-tumor activity of MRTX1133, underscoring the contribution of NK cells to the observed therapeutic benefit. Notably, KRAS^G12D inhibition led to a reduction in tumor-infiltrating Gr-1^+ myeloid-derived suppressor cells (MDSCs) and increased the proportion of effector NK cells. Furthermore, MRTX1133 facilitated greater intratumoral accumulation of adoptively transferred NK cells and counteracted tumor-induced systemic immunosuppression on NK cells. Combined treatment with MRTX1133 and NK cell immunotherapies produced synergistic anti-tumor effects and significantly extended survival in preclinical models. Mechanistically, ex vivo analyses demonstrated that MRTX1133 upregulates NK cell-activating ligands, including MICA/B and ULBPs, and suppresses the production of cytokines such as IL-6 and GM-CSF by cancer cells, thereby enhancing tumor cell susceptibility to NK cell-mediated cytotoxicity and reducing MDSC proliferation.



 Collectively, our results illuminate the interplay between KRAS^G12D inhibition and immuno-microenvironmental remodeling, highlighting rational combinations of targeted and NK cell-based immunotherapies as promising strategies to potentiate therapeutic responses in KRAS^G12D-driven cancers, potentially improving clinical outcomes for patients.



 Hongyuan Chen, Tuo Hu, Chao Xu; Liangjie Chi, Fangqin Xue, Chunbo He. KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aa888d3ef7c73bc5df1193380be3a3238a9fcd" target='_blank'>
              Abstract A016: KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity
              </a>
            </td>
          <td>
            Hongyuan Chen, Tuo Hu, Chao Xu, Liangjie Chi, Fangqin Xue, Chunbo He
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Medulloblastoma is the most common malignant brain tumor in children and includes several molecular subtypes with distinct clinical outcomes. Because genetic mutations are relatively rare in this tumor, our study focused on understanding how epigenetic regulation—the chemical and structural changes that control gene activity without altering DNA—contributes to disease progression. Using data from large patient cohorts and single-cell sequencing, we identified specific epigenetic factors that distinguish tumor subtypes and correlate with prognosis. We also developed an epigenetic score that predicts patient survival and is particularly elevated in aggressive Sonic Hedgehog (SHH) tumors. These results suggest that dysregulated epigenetic programs affecting RNA metabolism and cell proliferation play a key role in medulloblastoma aggressiveness, highlighting new potential targets for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733b0f36e6a075919ebc25ace84fd8a8035e0b6e" target='_blank'>
              Single-Cell Heterogeneity of Epigenetic Factor Regulation Deciphers Alteration of RNA Metabolism During Proliferative SHH-Medulloblastoma
              </a>
            </td>
          <td>
            R. Francés, Jenny Bonifacio-Mundaca, Íñigo Casafont, Christophe Desterke, Jorge Mata-Garrido
          </td>
          <td>2025-10-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Single-cell molecular profiling has emerged as a cutting-edge technology in biological research, playing a pivotal role in elucidating fundamental life processes and cellular heterogeneity. However, traditional methods generally provide end point measurements and require cell disruption, posing challenges for long-term monitoring of living cell events, e.g., cancer therapy. Tumor Treating Fields (TTFields), a noninvasive therapeutic modality, exert inhibitory effects on tumor growth and invasion by applying low-intensity intermediate-frequency alternating electric fields to tumor regions, whose molecular mechanism on cancer physiology remains poorly understood. In this study, we propose a multifunctional microchip capable of continuous in situ monitoring of molecular signatures in living single cells. The platform integrates spatial cell positioning, single-cellular multimolecule tracking with surface-enhanced Raman scattering (SERS), and TTFields treatment functionalities on a microchip. Using multiplexed analysis of liver cancer cells treated with TTFields and chemotherapy drugs, we implement spatial-temporal multiomics profiling for cancer therapy. Temporal tracking of critical molecular events associated with glycolysis, genetic stability, and membrane integrity reveals significant differences between the groups with and without TTFields treatment. Causal network analysis of molecular profiles uncovers the underlying mechanisms through which TTFields modulate oncogenic pathways. Our work demonstrates the effectiveness of the proposed microchip platform for real-time monitoring of living cell molecular dynamics, offering a powerful tool for precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889ada64ff2d00892ebd5cde7eb385bb8b31694b" target='_blank'>
              Spatial-Temporal Multiomics Profiling of Living Single Cells on Chip for Cancer Therapy.
              </a>
            </td>
          <td>
            Kunru Yu, Sheng-Zong Chen, Rong Zhu
          </td>
          <td>2025-10-12</td>
          <td>ACS sensors</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains one of the most difficult solid tumors to treat, owing in part to its remarkable cell state plasticity - a property that enables tumor cells to resist therapy, evade differentiation, and adapt to selective pressures. While targeted therapies have succeeded in subsets of patients, most tumors persist and eventually progress. To pursue a fundamentally new therapeutic strategy, we conducted an unbiased chemical screen using a custom-designed dual reporter system designed to broadcast cell state dynamics in real time by fluorescently tagging the endogenous SOX9 (aberrant stem-like) and KRT20 (differentiated) loci. This screen targeted epigenetic regulators capable of supporting CRC plasticity and preventing differentiation. We discovered that selective inhibition of histone deacetylases HDAC1/2 promotes differentiation and reduces tumor growth across human and mouse CRC models. This effect was not a consequence of cytotoxicity but rather epigenetic reprogramming. Through mass spectrometry-based histone modification profiling, we identified H3K27ac and H3K9ac as key epigenetic marks enriched at HDAC1/2-bound loci. These marks correlated with enhanced chromatin accessibility and the upregulation of differentiation-associated genes. Degradation of EP300, a major acetyltransferase, reversed the phenotype, confirming the functional relevance of histone acetylation in mediating this differentiation response. Furthermore, a focused CRISPR screen uncovered DAPK3 as a previously unrecognized downstream effector, providing a mechanistic foothold for future biomarker development. Our findings introduce a paradigm shift in CRC therapy: rather than targeting plastic tumor cells for elimination, we aim to force them into a differentiated, less adaptive state, making them more vulnerable to conventional and targeted treatments. This reprogramming effect is trackable via specific histone acetylation patterns, which we propose as candidate predictive biomarkers of response. Based on these insights, we hypothesize that HDAC1/2 inhibitors, when combined with KRAS or EZH2 inhibitors and/or chemotherapy, could yield synergistic therapeutic benefit. We are actively evaluating these combinations in preclinical models. In sum, this work positions HDAC1/2 inhibition as a precision epigenetic strategy to suppress CRC plasticity through enforced differentiation. By integrating multiomic profiling, functional genomics, and novel reporter technologies, we offer a blueprint for targeting the epigenetic underpinnings of therapy resistance in gastrointestinal malignancies, one that prioritizes reprogramming over destruction.



 Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, Sandor Spisak, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James Cleary, Jun Qi, Nilay Sethi. Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1992691ab8c82d8cb17a698db10f779b389e986a" target='_blank'>
              Abstract C031: Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains largely refractory to therapy, due in part to the complex interplay between tumor cells and their microenvironment. Human antigen R (HuR/ELAVL1), a ubiquitously expressed RNA-binding protein, is emerging as an important regulator both of tumor-intrinsic and tumor-extrinsic pathways that govern PDAC progression. While the role of HuR in promoting cancer cell survival under stress is well established, recent studies reveal its broader role in shaping the tumor microenvironment (TME), including metabolic rewiring, stromal activation, angiogenesis, and immune modulation. In this review, we examine how tumor-intrinsic HuR drives epithelial-mesenchymal transition, stabilizes key transcripts involved in metabolic adaptation, and alters the tumor secretome to influence extracellular matrix deposition and fibroblast behavior. We further explore the role of HuR in regulating immune cell function and the immune landscape of PDAC. Notably, HuR-driven TME remodeling reinforces environmental stressors that further activate HuR, establishing a feed-forward loop that drives disease progression. These findings underscore HuR as a central regulator of the PDAC TME and therapeutic resistance, and thus, highlight its potential as a target in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be1b261ccac3070698258b8350c58e4ccb52e4c4" target='_blank'>
              The Role of Tumor-Intrinsic HuR in Modeling the Pancreatic Tumor Microenvironment: Molecular Mechanisms and Therapeutic Opportunities.
              </a>
            </td>
          <td>
            Katherine R. Pelz, Kyle P Gribbin, Tess Sevetson, Jonathan R. Brody
          </td>
          <td>2025-10-13</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Renal cancer, particularly clear cell renal cell carcinoma (ccRCC), is characterized by significant intratumoral heterogeneity, which poses challenges for diagnosis and treatment. Single-cell sequencing (SCS) provides unprecedented insights into the cellular landscape of renal cancer, allowing for detailed characterization of tumor heterogeneity at the single-cell level. This review highlights how SCS has been instrumental in elucidating the origins of different renal cancer subtypes, understanding mechanisms of tumor initiation and progression, and dissecting the complex tumor microenvironment (TME). It discusses the identification of novel biomarkers and therapeutic targets, as well as the potential of SCS to inform personalized treatment strategies. The review also explores the integration of SCS with spatial omics technologies, which enhances the understanding of cellular interactions within their spatial context. Moreover, it addresses the challenges and future directions in applying SCS to clinical practice, emphasizing its significance in advancing renal cancer biology and improving clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03a5f0a7b414a81333643cfcf0b78e2ab4708d98" target='_blank'>
              Single-cell sequencing technology in renal cancer: insights into tumor biology and clinical application
              </a>
            </td>
          <td>
            Hanzhong Zhang, Ying Liu, Wenqiang Liu, Anqi Lin, Yu Fang, Le Qu, Xu Zhang, Peng Luo, Linhui Wang, Aimin Jiang
          </td>
          <td>2025-10-01</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The central nervous system (CNS) represents a uniquely immune-privileged environment, with inflammatory responses involving several resident CNS-specific cell types. While stereotyped cellular and transcriptional responses recur across varied diseases, relevant signaling pathways and regulatory networks are not fully understood. Here, we investigate multi-modal inflammatory gene networks at large scale by developing a high-throughput RNA-seq screening and analysis workflow. As proof-of-concept, we investigate genetically heterogeneous mice from a large-scale chemical mutagenesis screen to identify novel functionally relevant variants in six genes previously linked to human CNS disorders: Nrros, Ctsd, Smpd1, Idua, Nlrp1a, and Inpp5d. We leverage the readily interpretable data from our large-scale study to demarcate distinct inflammatory states arising from each mutation. In all, our work provides a validated analysis framework for identifying discrete gene expression modules that are engaged divergently across disease contexts, which can be used to discover novel regulators of CNS neuroimmune homeostasis. Brain bulk-tissue RNA-seq database from a large-scale ENU-mutagenized mouse cohort. Variants in Nrros, Ctsd, Smpd1, Idua, Nlrp1a, and Inpp5d alter microglia homeostasis. An Nlrp1a gain-of-function mutation causes widespread astrocyte and microglia activation. Cross-comparison of six disease and inflammatory states facilitates interpretation of bulk tissue transcriptomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a51f54829abe0e8b61931cb5f83b202b5c060ee" target='_blank'>
              Modular inflammation network discovery from large-scale phenotypic screening in genetically heterogeneous mouse brains
              </a>
            </td>
          <td>
            Monica Xiong, L. Miosge, Carolina Correa-Ospina, Claudia M. Y. Yan, Tiffany Cripps, Stefan Bauernfried, Yuanyuan Wang, Maggie Crow, Lucy X Morris, T. D. Andrews, Andrew Trujillo, Mitchell G. Rezzonico, Yuxin Liang, Qixin Bei, Z. Modrusan, Kimberly L Stark, Tracy J Yuen, Brad A Friedman, Jesse E Hanson, Edward M Bertram, Christopher J Bohlen
          </td>
          <td>2025-09-29</td>
          <td>Journal of Neuroinflammation</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a crucial role in colorectal cancer by sustaining intratumoral heterogeneity, therapeutic resistance, and metastatic potential. CD133 (PROM1) is among the most frequently used surface markers for CSC identification, whereas TRIM28, a versatile epigenetic regulator, has been implicated in controlling CD133 expression and stem-like features. In this study, we performed a detailed molecular and functional analysis of Caco2 colorectal cancer cell clones with individual knockouts of CD133 or TRIM28. Elimination of CD133 neither altered global gene expression, as confirmed by transcriptome profiling, nor affected key cellular properties. In contrast, loss of TRIM28 led to a marked reduction in CD133 protein abundance and induced extensive molecular and phenotypic remodeling. TRIM28 knockout was associated with broad transcriptomic changes involving more than 500 differentially expressed genes, decreased proliferative activity monitored by time-lapse imaging, and reduced sensitivity to paclitaxel, cisplatin, and curcumin. Furthermore, immune evasion molecules CD24 and CD47 (“don’t eat me” signals) were strongly upregulated in TRIM28-deficient cells, consistently confirmed by both RNA-Seq and flow cytometry analyses. At the same time, imaging flow cytometry and mitochondrial activity assays indicated that these effects were not due to major shifts in mitotic index or bioenergetic status. Altogether, our results demonstrate that TRIM28, rather than CD133, functions as a central regulator of CSC-associated phenotypes in colorectal cancer. These findings highlight the importance of epigenetic context in CSC biology and may inform the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3d3eb05c9db6b57d94b54e8bdad26455b3ea6" target='_blank'>
              Contrasting Impacts of Targeted Disruption of the Cancer Stem Cell Marker CD133 and Its Epigenetic Regulator TRIM28 in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            I. Kholodenko, A. Saidova, Daria M Potashnikova, V. A. Arzumanian, D. Romashin, A. Tvorogova, E. Poverennaya, K. Yarygin, Yan S. Kim
          </td>
          <td>2025-11-09</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Glioblastoma progression features extensive dissemination of cancer cells throughout the brain. Limited by neuroimaging resolution and lack of specific markers, malignant cell infiltration is clinically undetectable. This makes complete tumor removal impossible and relapse inevitable. During their exodus from the tumor core, glioblastoma cells are traversing distinct brain territories influencing their phenotypic cell state. To better understand the cellular and molecular dynamics of glioblastoma invasion, we profiled glioblastoma patient avatars and clinical samples using total, single-cell and spatial transcriptomics. In addition to gene expression data, active intracellular pathways of glioblastoma invasion were screened with high throughput activated kinase assays (Pamgene). We identified that human glioblastoma stem cells exhibiting proneural (PN) and neuron-progenitor like (NPC) transcriptomic traits in vitro formed highly infiltrative and connected xenografts in patient avatars. Consistent with this, PN/NPC glioblastoma stem cells cultured on primary hippocampal neurons exhibited higher connectivity and motility compared to mesenchymal-like cells. Leveraging single-cell and spatial transcriptome analytics of avatar and patient samples, we performed cell clustering, annotation, pseudotime and RNA velocity analyses to associate glioblastoma phenotypic cell states to cellular functions. We found that invasive glioblastoma cells re-activated genes and pathways related to neuron progenitor maturation when transitioning from the tumor core to the brain microenvironment. To verify whether invasive PN/NPC glioblastoma cell integration would alter brain functionality, we performed preclinical live cerebral blood volume recordings using noninvasive brain functional ultrasounds (fUS). As glioblastomas progressed, we detected a gradual alteration of brain functional connectivity. Modifications happened in brain areas featuring minimal, yet consistent glioblastoma cell colonization, identified by post-mortem pathology analyses. Altogether, our data shed light on the importance of glioblastoma cell adaptability and brain integration during the metastatic progression. They also provide a framework for detecting invisible clinical features such as tumor invasion, focusing on brain functional connectivity assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6de15ca728123a2849468be8f7f288fcf64e2b6b" target='_blank'>
              ANGI-14. GLIOBLASTOMA PROGRESSION FROM TRANSCRIPTOMIC REGULATION TO BRAIN FUNCTIONAL CONNECTIVITY
              </a>
            </td>
          <td>
            Madeleine Lackman, C. Bergström, Anaïs Virenque, Kato Herman, Aarti Venkatesan, Juli Udayani, Takashi Namba, Analiz Rodriguez, E. Castrén, P. Laakkonen, Sinem Karaman, V. Le Joncour
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1fe8757b9db03b1f80d6119b9cc7fd8b56ac919" target='_blank'>
              Pan-cancer analysis reveals that tumor microbiomes generate specific epitopes through transcriptional reprogramming
              </a>
            </td>
          <td>
            Jinman Fang, Yuxin Zhang, Wei Wang, Xiao Zuo, Tao Chen, Kecheng Li, Huiya Yang, Tianyang Zhu, Yuanliang Zhang, Dandan Wei, Jianfeng Zhang, Jialong Cui, Zhen Tang, Shouwu Zhang, Lirong Wang, Zhilei Li, Dushan Ding, Ruixia Sun, Dandan Zhu, Yin He, Yanfei Zhou, Chenkun Ye, Dandan Chen, Xueting Lang, Long Jiang, Qian Zhao, Lu Zhang, Hongzhi Wang, Yuanwei Zhang
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the most common and aggressive primary brain tumor in adults, marked by extensive inter- and intratumoral heterogeneity and pronounced therapy resistance. Microglia is a major contributor to this complexity and plays a pivotal role in shaping tumor behavior and modulating responses to therapy. Yet its precise phenotypic and functional states across distinct tumor regions and model systems remain poorly defined. In this study, we employ patient-derived organoids (PDOs) alongside matched primary tissue (PT) to characterize microglial diversity and investigate how microglia contributes to the cellular inter- and intratumoral heterogeneity of GBM.



 Fresh tumor tissue was obtained from 13 GBM patients and sampled from six spatially distinct regions per tumor. PDOs were generated from these regions and compared to matched PT to evaluate microglial presence, distribution, and morphology. To assess intertumoral heterogeneity, we compared microglial characteristics across patients, while intratumoral heterogeneity was examined by analyzing differences between regions within individual tumors. Subsequently, PDOs were treated with temozolomide (TMZ), and microglia was visualized and quantified using Iba1 immunofluorescence to evaluate treatment-induced morphological changes and shifts in microglial abundance.



 Microglia in PT exhibited a microglia morphology more similar to PDOs than to healthy brain tissue. Distinct microglial distribution patterns emerged across PDOs, with microglia typically arranged in one of four configurations: (1) perinuclear, (2) ring-like, (3) at the PDO margin, or (4) diffusely dispersed throughout the PDO. In 5/13 PDOs we found significantly higher microglial abundance compared to PT (PPDO 4 = 0.0013, PPDO 7 = 0.0424, PPDO 8 = 0.0153, PPDO 10 = 0.0162, PPDO 12 = 0.0083). Analysis across patients revealed significant intertumoral heterogeneity in microglial characteristics (p = 0.0405). In terms of intratumoral heterogeneity, regional differences in microglial abundance and morphology were observed within individual PDOs, with statistically significant variation only found in PDO 10 (p = 0.0211). Following TMZ treatment, 4/13 PDOs showed a significant recline in microglial abundance (PDO 2: p < 0.0001; PDO 4: p = 0.0041; PDO 7: p = 0.0015; PDO 10: p = 0.0297).



 Our findings reveal significant inter- and intratumoral microglial heterogeneity in PDOs. Distinct microglial distribution patterns and variable responses to TMZ highlight the complex and patient-specific nature of microglial involvement in the GBM microenvironment. These results underscore the relevance of PDOs as a model system for studying tumor-immune interactions and therapeutic responses in a personalized context.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a796a165fe58659df81ee201e3cb5e0d22f92cdf" target='_blank'>
              P02.05.B MAPPING MICROGLIAL DIVERSITY AND TUMOR HETEROGENEITY IN PRIMARY GLIOBLASTOMA AND PATIENT-DERIVED ORGANOIDS
              </a>
            </td>
          <td>
            J. Mazzoni, G. Haug, E. Roth, J. Hiebl, J. Messinger, K. Mehling, E. Meller, R. Nickl, C. Monoranu, A. Appelt-Menzel, C. Hagemann, F. Liesche-Starnecker, V. Nickl
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Understanding the functional impact of bacterial genetic diversity is crucial for linking pathogen variants to clinical outcomes. Here, we introduce a high-throughput cytological profiling pipeline optimized for Mycobacterium tuberculosis (Mtb) clinical strains, integrating OD-calibrated feature analysis and high-content microscopy. Our system quantifies single-bacterium morphological and physiological traits related to DNA replication, redox state, carbon metabolism, and cell envelope dynamics. Applied to 64 Mtb clinical isolates from lineages 1, 2, and 4, the approach revealed that cytological phenotypes recapitulate genetic relationships and exhibit both lineage- and density-dependent dynamics. Notably, we identified a link between a convergent “small cell” phenotype and a convergent ino1 mutation that is associated with the presence of an antisense transcript, suggesting a potential non-canonical regulatory mechanism under selection. In summary, we present a resource-efficient approach for mapping Mtb’s phenotypic landscape, uncovering cellular traits that underlie its evolution and providing new insights into the functional consequences of bacterial genetic diversity. IMPORTANCE Understanding how genetic variation in Mycobacterium tuberculosis (Mtb) shapes its physical traits is essential to unraveling the evolution of this global pathogen. Here, we introduce a systematically optimized, high-throughput imaging platform for the comprehensive characterization of Mtb clinical strains. We demonstrate that Mtb’s phenotypic manifestation is shaped by both genetic background and culture density. By accounting for these factors, our analysis linked distinct cellular dynamics to specific lineages, sublineages, and even single nucleotide variations. Notably, we linked a recurring mutation to a unique cell-shortening phenotype, finding that it potentially acts by creating a cryptic antisense transcript. This platform provides a powerful framework for systematically dissecting the physiological dynamics underlying Mtb evolution and identifying new therapeutic vulnerabilities of this deadly pathogen. Understanding how genetic variation in Mycobacterium tuberculosis (Mtb) shapes its physical traits is essential to unraveling the evolution of this global pathogen. Here, we introduce a systematically optimized, high-throughput imaging platform for the comprehensive characterization of Mtb clinical strains. We demonstrate that Mtb’s phenotypic manifestation is shaped by both genetic background and culture density. By accounting for these factors, our analysis linked distinct cellular dynamics to specific lineages, sublineages, and even single nucleotide variations. Notably, we linked a recurring mutation to a unique cell-shortening phenotype, finding that it potentially acts by creating a cryptic antisense transcript. This platform provides a powerful framework for systematically dissecting the physiological dynamics underlying Mtb evolution and identifying new therapeutic vulnerabilities of this deadly pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56fa61f881d5174f374abc12d00931b99537639c" target='_blank'>
              High-throughput cytological profiling uncovers genotype-phenotype associations in Mycobacterium tuberculosis clinical isolates
              </a>
            </td>
          <td>
            Qingyun Liu, Yue J. Liu, Ruiyuan Liu, Peter H. Culviner, Xin Wang, I. D. Wolf, Ken Chen, Yiwang Chen, Yi Xiao, Guiming Zhang, Rongfeng Sun, Shoko Wakabayashi, Nicole C Howard, Mingyu Gan, Eric J. Rubin, Sarah M. Fortune, Junhao Zhu
          </td>
          <td>2025-10-09</td>
          <td>mSystems</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Objectives FAM49B has been shown to promote proliferation and metastasis of colorectal cancer (CRC) by stabilizing MYC through phosphorylation of NEK9; however, its role in shaping the immune suppressive tumor microenvironment (TME), particularly in macrophage polarization, remains unclear. Methods We applied multi-omics approaches to study CRC by integrating 33 scRNA-seq samples from 16 CRC patients, 2 paired spatial transcriptomics (ST) samples, and bulk RNA data to characterize malignant epithelial cells (High_FAM49B_EP) and tumor-associated macrophages (TAMs). Functional validation of FAM49B was conducted via knockdown experiments and proteomics analysis. Results A High_FAM49B_EP subpopulation was identified in primary tumors (PT) and liver metastases (LM), exhibiting elevated MYC signaling and association with poor prognosis. TAMs showed spatial heterogeneity: M1-like CXCL3+ TAMs predominated in PT, whereas M2-like SPP1+ TAMs were enriched in LM. CellChat analysis revealed that High_FAM49B_EP activated macrophage polarization through the MDK–NCL signaling axis. Pseudotime trajectory analysis confirmed differentiation from CXCL3+ to SPP1+ TAMs driven by upregulation of NCL. Spatial mapping showed co-localization of MDK+ epithelial cells with NCL+ TAMs in the immunosuppressive microenvironment. FAM49B knockdown significantly inhibited MDK expression and disrupted ECM–receptor interactions. Conclusions FAM49B promotes immunosuppressive TME formation by mediating TAM polarization via the MDK–NCL axis, suggesting the FAM49B–MDK–NCL pathway as a potential therapeutic target for CRC metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8878570ce8b97d296e4580e17e3836990c936b1" target='_blank'>
              Single-cell and spatial transcriptomics integration reveals FAM49B promotes tumor-associated macrophages polarization in colorectal cancer via the MK pathway
              </a>
            </td>
          <td>
            Tianyu Liu, Quchen Ding, Jin Gou, Chen Lu, Xingming Lu, Jiatong Chen, Y. E, Lianhong Li, Chongguo Zhang, Xiaojuan Zhu, Chunzhao Yu, Xiagang Luo
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Glioblastoma (GB), the most aggressive and lethal primary brain tumor, remains a formidable challenge, with a median survival of approximately 8 months despite current standard-of-care treatments, which typically involve surgical resection followed by concurrent chemoradiation. While the effects of adjuvant therapies on GB progression have been extensively investigated, the direct impact of surgery itself on tumor behavior remains poorly understood. This study aims to elucidate the role of surgical resection in shaping the microenvironment via multi-omics integration.



 We developed a syngeneic neurosurgical mouse model. Implantations of GB cell lines followed by size-matched surgical resection were performed to mimic human GB resection. Residual tumors were then collected one day after surgery to capture the early effects of surgical damage on the tumor ecosystem. The robustness of this model facilitated the generation of a comprehensive multi-omics dataset delineating postsurgical molecular alterations in both GB cells and their microenvironment. This dataset encompassed proteomic, phosphoproteomic, and single-cell RNA-sequencing data.



 The integration of these datasets using both data-driven and prior knowledge-based approaches shed light on the complex molecular pathways underlying the response of residual tumors to resection. HIF1a, JUN, and SP1 emerged from this analysis as critical drivers of molecular response to surgery. Finally, using single-cell resolution from the RNA-seq data, we delineated the contribution of each cell type to overall changes. This methodology led us to identify hypoxic macrophages as a central communication hub driving both tumor microenvironment reshaping and residual tumor plasticity after surgery.



 In conclusion, multi-omics study of the postsurgical effects on residual GBM cells and the microenvironment allowed us to discover new potential vulnerabilities in an unexplored phase of tumor progression. This may allow for the development of novel therapeutic strategies aimed at reducing post-surgery GBM aggressiveness and sensitizing it to subsequent therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1495342d99db473c2866de185ae22233537a5fff" target='_blank'>
              P04.12.A MULTI-OMICS INTEGRATION IN POST-SURGERY RESIDUAL TUMOR UNVEILS INTENSE MICROENVIRONMENT RESHAPING
              </a>
            </td>
          <td>
            A. Ballestín, J. Reveilles, A. Thomas, C. Pichol-Thievend, O. Anezo, G. Bourmeau, V. Ribecco, B. Blanchard, A. Chemkhi, T. Daubon, H. Isambert, F. Cavalli, G. Seano
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="BACKGROUND
Changes in the cyst microenvironment in Polycystic Kidney Disease (PKD) may drive progressive cyst formation. Bulk- and single-cell RNA sequencing have advanced our understanding of altered signaling; however, the lack of spatial information has limited our insights into local gene expression and cellular communication near cysts.


METHODS
We used wild-type and Pkd1-deficient mouse kidneys to generate 10x Genomics Visium Spatial Gene Expression datasets. Utilizing our single-cell mouse kidney atlas and single cell sequencing data for spot deconvolution, we enhanced resolution and estimated enriched cell types. We analyzed spatial gene expression patterns and used a cyst-centered analysis to identify cyst-associated gene signature. Cell communication near cysts was investigated, identifying key ligand-receptors. Prioritized key factors were validated in tissues.


RESULTS
We observed enrichment of fibroblasts, injury-repair-related cell types, and diverse immune populations in PKD. Injury-repair-related cells were exclusively observed in PKD, predominantly localized within immune cell-dense regions near cysts. These cells collectively contributed to the altered gene expression profile in PKD, including cyst-associated signature genes related to inflammatory processes. Analysis of cellular communication in less-inflamed regions around cysts revealed the involvement of multiple cell types. Key ligand-receptor interactions were associated with cytokine signaling, fibrosis, cellular development, and repair. These included Angpt2, C3, Csf1, Cxcl12, Il34, Gas6, Il16, Mdk, Mif, Ptn, Sfrp2, Spp1, Sdc1, Tnc, Tnfsf12, Wnt5a. In addition, ECM proteins implicated in immune response, ECM remodeling, cell adhesion, and cell signaling were identified, such as Adam9, Adam10, Col1a1, Col3a1, Col4a2, Lamb2, Lamc1, Efnb1, Efnb2, Thbs1, Thbs2, and Vcam1. IHC confirmed expression of Syndecan-1-Collagen IV, Midkine-Integrin β1, CSF-1, Pleiotrophin, and Tenascin-C in cystic kidneys.


CONCLUSIONS
Spatial transcriptomics in PKD revealed enrichment of (myo)fibroblasts, immune, and injury-repair-related cells near cysts, creating a (pro)inflammatory and (pro)fibrotic niche. Key ligand-receptor and ECM interactions were identified and validated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecdcf5e98138976e4b0b6f2eafc40e1b84148be0" target='_blank'>
              Spatial Transcriptomics Reveals Injured Cells, Signature Genes, and Communication Patterns in the Cyst Microenvironment of Polycystic Kidney Disease.
              </a>
            </td>
          <td>
            Sevtap A Yasinoglu, Claudio Novella-Rausell, L. Wisse, K. Dijkstra, A. Mahfouz, Hans J. Baelde, DJM. Peters
          </td>
          <td>2025-10-10</td>
          <td>Journal of the American Society of Nephrology : JASN</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Resistance to targeted cancer therapies is a significant barrier to favorable treatment outcomes. Malignant cells can tolerate and resist drug treatments due to their biological flexibility. Specifically, slow-cycling drug-resistant cells may achieve permanent resistance to the treatment or restore sensitivity upon cessation of therapy. Enhancing cancer treatment methodologies necessitates a deeper understanding of the adaptability of tumor cells. Drug resistance and cellular heterogeneity are closely associated with cancer cell adaptability. Alterations in cellular signaling, interactions with the tumor microenvironment, and genetic and epigenetic alterations are all implicated. Analyzing these pathways will enhance our understanding of how cancer cells evolve and evade treatment. Two effective strategies to address cancer cell adaptability are to target specific biological pathways and to employ combination therapies. The progression of cancer therapy methodologies relies on comprehending and exploring the concept of cancer cell adaptability. Understanding tumor heterogeneity and drug resistance necessitates identifying the cellular, molecular, and genetic processes that govern cancer cell plasticity. This understanding enables the development of more personalized and effective cancer therapies, leading to improved treatment outcomes. Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/714e22df09170482d5d0d601474075162f529a30" target='_blank'>
              Cancer cell plasticity and therapeutic resistance: mechanisms, crosstalk, and translational perspectives
              </a>
            </td>
          <td>
            S. Ghorbian
          </td>
          <td>2025-09-26</td>
          <td>Hereditas</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="BACKGROUND
Neoadjuvant treatment has demonstrated clinical benefits in advanced gallbladder cancer (GBC). However, the overall response rate remains suboptimal, and the underlying mechanisms driving treatment efficacy are not fully understood.


MATERIALS AND METHODS
This study aimed to evaluate the therapeutic effects of neoadjuvant chemo-immunotherapy (NAT) combining gemcitabine, nab-paclitaxel, and anti-PD-1 immunotherapy in advanced GBC, and to investigate the associated tumor microenvironment (TME) alterations. Single-cell RNA sequencing and multiplex immunohistochemistry were utilized to analyze the cellular composition of the TME in patients who successfully underwent downstaging.


RESULTS
NAT significantly reshaped the tumor-immune landscape, characterized by an expansion of follicular helper T (Tfh) cells and the formation of tertiary lymphoid structures (TLSs) in 50% of treated tumors. Inflammatory cancer-associated fibroblasts (CAFs) increased and exhibited upregulation of CCL19 and CXCL12, potentially promoting Tfh cell recruitment and TLS formation. Additionally, NAT led to an expansion of GZMB+ cytotoxic CD8+ T cells with an exhausted phenotype in GBC, but not in adjacent normal tissues. The treatment also increased the number and effector functions of natural killer cells while reducing tumor-promoting macrophages and angiogenesis-related CAFs. Furthermore, NAT decreased the cancer stem cell-like subpopulation while increasing a cancer cell subset with enhanced antigen-presenting capacity.


CONCLUSION
This study suggested the potential efficacy of NAT in advanced GBC and revealed alterations in the TME following treatment. The findings provided insights into the mechanisms underlying NAT responses and offered valuable directions for optimizing therapeutic strategies in GBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/363d50ed22da82003f2b54ca7ab634be84e764af" target='_blank'>
              Single-cell profiling unveils the immuno-favorable tumor microenvironment remodeling after successful neoadjuvant therapy for advanced gallbladder cancer.
              </a>
            </td>
          <td>
            Qiuyi Tang, Yi-Jian Zhang, Ziyou Wu, Tian-Yu Zhang, Xiao-ling Song, Yi-di Zhu, Zi-Yi Yang, Cheng Zhao, Jing-wei Zhao, Hua-kai Wang, Yue Yang, Yu Zhang, Wen-Ting Dai, Lin Jiang, Yang-Yang Zhai, Chuan-Xin Huang, , Wei Gong
          </td>
          <td>2025-11-06</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Approximately 40% of patients with pancreatic ductal adenocarcinoma (PDAC) receive no benefit from surgical resection and systemic chemotherapy and experience metastatic disease recurrence within 12 months. The extreme heterogeneity of the cells that populate PDAC tumors has been a significant impediment to identifying the cells that likely seed early recurrence. Previous studies show that metastatic spread is driven by specific cells with hybrid epithelial-mesenchymal characteristics, but despite the identification of numerous molecular features associated with epithelial, hybrid, and mesenchymal PDAC cancer cells, it has not been possible to specifically identify the cells that seed metastatic recurrence in human tissue. We have previously shown that PDAC cell glycan signatures are altered as a direct result of the metabolic and transcription changes that occur as PDAC cancer cells undergo EMT. Now we integrated the glycan signatures with known protein markers and morphological analyses using our pipeline for multiplexed immunofluorescence analysis to identify distinct subpopulations of epithelial, hybrid, and mesenchymal PDAC cells. In cell-level analyses of immunofluorescence images of 24 PDAC cell lines, these subpopulations revealed epithelial to hybrid to mesenchymal conversions along 3 distinct trajectories, which were defined by transcription factors and proteins previously associated with epithelial or mesenchymal states—TP53, TP63, GATA6, and MYC—and further distinguished by specific glycan signatures and protein markers of metabolism and cellular differentiation. The integrated transcription factor, glycan, and protein signatures identified the previously described classical and squamous PDAC subpopulations and defined a new signature for a fully mesenchymal subpopulation. The signatures differentiated between PDAC subpopulations in primary human tumors and in metastatic liver lesions and showed a patient-matched correspondence between the primary tumors and metastatic lesions in specific hybrid PDAC cells within distinct trajectories, potentially revealing the PDAC cells seeding metastatic recurrence. These results suggest that the development of mesenchymal and metastatic PDAC cells occurs through distinct, stepwise trajectories with divergent morphological, metabolic, molecular, and glycan-based signatures. These findings will/have the potential to inform the development of treatments that specifically target the PDAC cells that seed early metastatic recurrence.



 ChongFeng Gao, Zachary Klamer, Ali Moursy, Galen Hostetter, Paul M. Grandgenett, Peter Allen, Brian Haab. Resolving PDAC cell subpopulations that seed metastatic recurrence [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B125.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246925305285b4b397bb54f4914db32c1466cbe8" target='_blank'>
              Abstract B125: Resolving PDAC cell subpopulations that seed metastatic recurrence
              </a>
            </td>
          <td>
            Chongfeng Gao, Zachary L. Klamer, Ali Moursy, G. Hostetter, P. Grandgenett, Peter Allen, Brian Haab
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy driven by oncogenic KRAS and inflammation-induced cellular heterogeneity, yet the mechanisms underlying tumor-initiating cells (TICs) emergence and maintenance remain unclear. Our study establishes LY6D as a marker of persistent TICs that orchestrate pancreatic cancer progression across all disease stages. Single-cell RNA sequencing of inflammation-driven PDAC models reveals that LY6D+ TICs specifically originate from KRAS-mutant acinar-to-ductal metaplasia (ADM) lesions under inflammatory conditions, maintaining conserved stemness properties and exhibiting a distinctive oxidative phosphorylation (OXPHOS) dependency throughout tumor evolution. Moreover, genetic ablation of Ly6d in Kras G12D pancreata delays tumorigenesis, while forced Ly6d expression enhances tumorigenic potential and metastatic capability. Mechanistically, LY6D-despite lacking intracellular domains-scaffolds lipid raft-associated kinase networks and FOSL1-dependent epigenetic reprogramming to establish a ​stable pro-tumorigenic state. Clinically, LY6D+ cells are enriched in human PDAC and exhibit conserved stemness and epithelial-mesenchymal transition (EMT) properties. Strikingly, LY6D expression levels demonstrate PDAC-restricted prognostic power. Our work defines LY6D as pan-stage TICs marker linking cellular plasticity to PDAC initiation and progression, offering new avenues for early detection and interception of this lethal malignancy.



 Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue. LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B119.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f74adf609c9f9d814e1f1f768c348943a8c10b" target='_blank'>
              Abstract B119: LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis
              </a>
            </td>
          <td>
            Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Analyses of patient-derived cell lines have greatly enhanced discovery of molecular biomarkers and therapeutic targets. However, characterization of cellular morphological properties is limited. We studied cell morphologies of human pancreatic adenocarcinoma (PDAC) cell lines and their associations with drug sensitivity, gene expression, and functional properties. By integrating live cell and spatial messenger RNA imaging, we identified KRAS inhibitor–induced morphological changes specific for drug-resistant cells that correlated with gene expression changes. We then categorized a large panel of patient-derived PDAC cell lines into morphological and organizational subtypes and found differences in gene expression, therapeutic targeting potential, and metastatic proclivity. Patterns of cancer cell organization in human PDAC tissues stratified distinct gene expression signatures with clinical significance. In summary, we highlight the potential of cell morphological information in rapid, cost-effective assays to aid precision oncology efforts leveraging patient-derived in vitro models and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb047a3ab41c4e28c6535b371221906a6c927ab" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are increasingly recognized as critical contributors to the limited efficacy of immunotherapy, particularly through the establishment of immune-excluded tumor microenvironments. However, the functional heterogeneity and spatial ecological roles of CAFs in melanoma remain poorly characterized. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq data to systematically dissect CAF molecular programs in melanoma using weighted gene co-expression network analysis (WGCNA). Spatial distribution, CellChat, and ligand–receptor interaction analyses were applied to construct CAF-centered immune exclusion networks. A functional scoring model (riskScore) based on the CAF-M1 module was developed to evaluate clinical relevance. CAF was spatially co-localized with endothelial cells, and engaged in strong crosstalk with multiple immune cell types via pathways such as COLLAGEN–integrin and THBS1–CD47. The derived riskScore model demonstrated robust prognostic value across TCGA-SKCM cohort. Through spatial aggregation and multifaceted immune signaling, CAF constructs a multilayered immune-exclusion network in melanoma, linking stromal remodeling to immune evasion. These findings offer novel insights into CAF-driven immune resistance and may inform future stratified immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca83751e1c8ec5203b2f4b9c36b4cadfbfd1df19" target='_blank'>
              Exploration of the roles of CAFs in melanoma based on single-cell transcriptomics and spatial transcriptomics
              </a>
            </td>
          <td>
            Sheng Hong, Yuhan Zhao, Anni Hu, Yizhong Zhang, Yuming Li, Yan Rong, Zukai Li, Wenzhuo Su, Lude Zhu
          </td>
          <td>2025-11-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor matrix stiffness plays a critical role in cancer progression, metastasis, and therapy resistance. Although traditional biophysical methods have shed light on the impact of matrix stiffness on tumor behavior, these techniques are confined to measuring the physical properties of the tumors. In this study, we leveraged RNA-seq data to predict tumor matrix stiffness, aiming to reveal mechanical properties by molecular signatures across various cancer types. To this end, we systematically analyzed RNA-seq data from tumors of varying stiffness levels to identify stiffness-associated gene signatures. With these molecular signatures, we developed a computational model for predicting tumor matrix stiffness and further applied it to The Cancer Genome Atlas (TCGA) dataset. Our analysis revealed significant differences in the tumor microenvironment as well as immune response between soft and stiff tumor samples, suggesting that tumor rigidity impacts not only cellular behavior but also characteristics of the tumor microenvironment. These findings underscore the potential of RNA-based stiffness models to enhance our comprehension of tumor mechanics and cancer biology, thereby facilitating the development of innovative targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd0a56130a8bfa1d6781b487c86488f3edefb603" target='_blank'>
              Pan-cancer analysis reveals molecular signatures for predicting matrix stiffness in solid tumors.
              </a>
            </td>
          <td>
            Gongyu Tang, Xinyi Liu, Yuanxiang Li, Yunfei Ta, Minsu Cho, Hua Li, Xiaowei Wang
          </td>
          <td>2025-09-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 As cancer therapies improve and patients live longer, the incidence of brain metastases (BrMets) continues to rise. Among women with metastatic breast cancer, 10-15% develop BrMets, with rates as high as 30% in HER2+ and 50% in triple-negative. BrMets, represent highly evolved, therapy-resistant tumors with a median survival of 10 months and accompanied by severe neurologic decline. BrMets often emerge years after initial diagnosis, suggesting that metastatic cells must acquire brain-specific traits to survive. This process may involve transcriptional convergence—where tumor cells adopt brain-like programs, and brain-resident cells exhibit tumor-associated changes. Despite its clinical relevance, the molecular basis of this convergence remains poorly defined. Decoding, and targeting, these shared gene programs could reveal new therapeutic vulnerabilities.



 We performed spatial transcriptomic profiling on 235 patient tissue cores (BrMets, adjacent normal brain, primary breast tumors, and non-cancer brain). Using NanoString GeoMx, DSP, we measured the expression of 18,677 RNAs across 450 spatially-defined regions enriched for tumor, immune, and brain-resident cells. To interrogate transcriptional convergence, we developed the Equivalent Expression Index, a novel statistical tool to detect genes with biologically meaningful expression similarity across distinct cell populations.



 Our integrated analysis revealed two prognostic gene signatures. The Metastasis-Induced Brain Shared (MIBS-9) signature, comprising brain-like genes upregulated in tumor cells, was associated with longer patient survival. Conversely, the Adjacent brain-Resident Cancer Shared (ARCS-81) signature, defined by tumor-like gene expression in brain-resident cells, correlated with shorter survival. These signatures were externally validated and enriched for pathways related to neuronal remodeling, immune evasion, cellular communication, and cancer/metastasis.



 By coupling spatial transcriptomics with a novel equivalence-based algorithm, we uncover distinct, clinically relevant gene programs that mediate bidirectional adaptations between tumor and brain-resident cells. These findings illuminate key mechanisms of metastatic outgrowth and offer new targets for disrupting the breast cancer BrMet microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4c6b99afe4369f11ea5994698b0e8b275a5df02" target='_blank'>
              TMIC-89. Spatial transcriptomics reveals tumor–brain expression convergence and identifies prognostic gene signatures in breast cancer brain metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, Christine Yeh, Pablo Nunez Perez, Giuseppe Barisano, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia Plevritis, M. Gephart
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Intraductal papillary mucinous neoplasm (IPMN) is one of the two most common precursor lesions leading to the development of pancreatic ductal adenocarcinoma (PDA). IPMNs comprise a heterogeneous group of tumors with a wide range of grades and histotypes, and the emergence of single-cell RNA sequencing (RNA-seq) and multiplex digital spatial profiling have characterized unique cell populations, including dysplastic epithelial and immune cells, within the heterogeneous tumor microenvironment that carry signature gene expressions, which could be used as markers for disease progression. However, it is not clear on the heterogeneity of the epithelial and immune response of the evolution during progression of IPMN to PDA.



 Combining nuclear GSK-3β with oncogenic KRas mice (referred to as KNGC – KRas, nuclear GSK-3β, Cre) resulted in the development of IPMN. We have found that 4-week-old KNGC mice show progressive desmoplasia and ductal lesion development with increased M2-like macrophage infiltration. Interestingly, CX3CL1 (fractalkine), a chemokine involved in the recruitment of CX3CR1-expressing monocytes and M2-like macrophage conversion in various disease models is expressed by IPMN progenitor ductal cells. We aim to investigate expression and contribution of CX3CL1-CX3CL1 axis in the development and progression of IPMN.



 We utilized a newly generated CX3CL1-mCherry and well-established CX3CR1-GFP knockin/knockout mouse for crossbreeding with KNGC and KGC (KRas, GNas, Cre) for lineage tracing and function analysis by immunofluorescent staining, flow cytometry and CyTOF.



 Both KNGC- and KGC-3LmCherry/+ mice harbor a large population of CX3CL1-mCherry/EPCAM+ epithelial cells as compared to 3LmCherry/+ and wildtype (WT) control mice. In KNGC and KGC mice, we show that there was a unique population of macrophages (CD45+ CD11b+ Ly6G- F4/80+ Ly6c- ) with significantly higher expression of CX3CR1 (CX3CR1Hi) that was not seen in WT mice. Significantly, Knockout of CX3CR1 in KNGC-3RGFP/GFP animals had a paucity of M2-like macrophages, reduced T regulatory cells (Tregs), substantially reduced desmoplasia and impaired development of IPMN.



 The study provides important knowledge regarding the role of CX3CL1-CX3CR1 axis in recruitment and generation of M2-like macrophages, and their impact on the initiation of the immune/stroma microenvironment facilitating IPMN development. Further investigations using these models have the potential to identify new biomarkers for risk stratification and therapeutic intervention.



 Li Ding, Kaely Roeck, Easton Maeder, Cheng Zhang, Taylor Weiskittel, Esther Rodman, Mark Maynes, Bibo Zhu, Aaron Johnson, Jie Sun, Li Hu, Daniel Billadeau. Uncoupling CX3CL1-CX3CR1 Axis in IPMN Development Through Transgenic Mouse Models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A064.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94351cad4fa5650594e9ca02f146727e9c878075" target='_blank'>
              Abstract A064: Uncoupling CX3CL1-CX3CR1 Axis in IPMN Development Through Transgenic Mouse Models
              </a>
            </td>
          <td>
            Li Ding, Kaely Roeck, Easton Maeder, Cheng Zhang, Taylor Weiskittel, Esther Rodman, M. Maynes, Bibo Zhu, Aaron Johnson, Jie Sun, Li Hu, Daniel D. Billadeau
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Brain metastases (BrMs) represent a significant clinical challenge with increasing incidence and poor prognosis. Here, we present a comprehensive proteogenomic characterization of pan-cancer BrMs (n=923) with patient-matched primary tumors (n=82) and glioblastomas (n=20), representing the largest cohort to date. Through consensus clustering of transcriptomic data, we identified and validated four distinct molecular subtypes (CMS1-4) with unique biological features. Integrated analysis of the transcriptomic and proteomic revealed that CMS1 was activated with neural system-associated pathways, CMS2 was characterized by extracellular matrix remodeling and increased immune cell infiltration, CMS3 exhibited a more epithelial phenotype along with metabolic reprogramming and CMS4 showed upregulation of cell cycle-related genes with elevated stemness scores. These molecular features were further validated through immunohistochemistry and multiplex immunofluorescence. Genomic analysis revealed subtype-specific mutational signatures and copy number alterations, with CMS1 exhibiting lower tumor purity and CMS4 displaying the highest chromosomal instability. Survival analysis indicates that CMS4 was associated with the worst prognosis, while CMS1 and CMS2 demonstrated more favorable responsiveness to radiotherapy and immunotherapy, respectively. Using patient-derived organoid models, we identify subtype-specific therapeutic vulnerabilities, notably metabolic inhibition sensitivity in CMS3 and CDK4/6 inhibition response in CMS4. Single-nucleus RNA sequencing further elucidates the complex cellular ecosystems and intercellular communication patterns within each subtype. This study creates a multi-omics resource for understanding the metastatic adaptation to the brain microenvironment and exploring personalized treatment strategies in BrMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8325830d999701778469a9b0a790181d9230e305" target='_blank'>
              EPCO-01. PAN-CANCER PROTEOGENOMIC AND SPATIAL CHARACTERIZATION OF 1000 METASTATIC BRAIN TUMORS
              </a>
            </td>
          <td>
            Gao Zhang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Diffuse midline glioma (DMG) is a highly lethal brain tumor in children with limited treatment options. The blood-brain-barrier (BBB) remains a key challenge for therapeutics, preventing most drugs from entering brain tissue. Characterization of the blood-brain-tumor-barrier (BBTB) - the interaction between tumors and BBB vasculature - remains largely unexplored for DMG. We have established an ex vivo model of the DMG-BBTB, comprised of a co-culture of patient-derived DMG cells, primary human brain microvascular endothelial cells, primary human brain vascular pericytes, and primary human astrocytes, embedded in a fibrin matrix within a microfluidic chip. To probe the fidelity of our model and examine the impact of pediatric DMG on vasculature, we employed single cell RNA sequencing (scRNAseq).



 scRNAseq was performed on devices containing normal BBB and DMG-BBTB, as well as BBB devices dosed with tumor necrosis factor alpha (TNFa) as a positive inflammation control. Devices for each condition were seeded with cells and growth factors to form self-assembled microvascular networks within 7 days. On day 8, cells were extracted from the gel compartment to generate a single cell suspension, from which RNA was isolated to construct sequencing libraries. The prepared libraries were sequenced on the Illumina NovaSeq 6000 platform. After standard pre-processing and quality control, cells were clustered and annotated for downstream gene enrichment and pathway analyses as well projection onto reference datasets.



 scRNAseq of our model revealed both expected and unexpected cell types, consistent with on-chip differentiation of a brain vascular model. We identified significant differential gene expression across all cell populations between BBB and BBTB conditions, with signatures distinct from TNFa-induced inflammation. When comparing our data to established scRNAseq atlases of vascular development, our model recapitulates key aspects of developmental biology such as heterogeneous vascular endothelial and perictye subtypes. For BBTB conditions, we classified the patient-derived DMG cells in our model according to published developmental hierarchies established from patient samples.



 We developed a high fidelity ex vivo model to evaluate the microvasculature of the DMG-BBTB. Using scRNAseq, we demonstrate that our model recapitulates key features of the DMG tumor microenvironment. This representative model of the DMG-BBTB will provide an avenue for future preclinical testing of new therapies and exploring modalities for targeted drug delivery.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ea9304e631b710c842eb9a371d087d788737d19" target='_blank'>
              P10.05.B RECAPITULATING PATIENT BIOLOGY IN EX VIVO MODELS: A SCRNASEQ ANALYSIS OF THE BLOOD-BRAIN-TUMOR BARRIER IN PEDIATRIC DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            K. Bennett, Y. Vanrobaeys, C. Whittaker, P. T. Hammond, J. P. Straehla
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Tumor cell metabolome intricately shapes its identity and therapeutic vulnerabilities. Cancer cells dynamically adapt to the tumor microenvironment (TME) – ‘Warburg effect’, a classic example of metabolic plasticity wherein cancer cells preferentially convert glucose to lactate despite availability of oxygen to thrive challenging environments. While metabolic profiling has advanced tumor biology, spatially resolved rewiring of metabolome remains poorly defined. Neuroblastoma (NB), a pediatric cancer with high propensity for hematogenous metastasis exhibits diverse underscored metabolic phenotypes. Our recent studies uniquely identified the predictive, prognostic relevance and function of Retinal Degeneration protein 3 (RD3) in NB pathogenesis including its role in shaping the tumor immune environment. Herein, we investigated whether and how RD3 affects metabolic plasticity in metastatic NB, offering new therapeutic avenues for high-risk (HR) NB. For this, global untargeted metabolomics was performed in bone marrow (BM) derived clones (CHLA-90, CHLA-140) from HR-NB patients and compared with their RD3 stably reinstated counterparts. Data preprocessing, mass detection, EIC building, feature resolving and grouping, ion identity networking and spectral library search was performed using Mzmine4.7 followed by compound identification using Compound discoverer. RD3-driven shift in metabolites were identified using univariate analysis with stringent criteria (log2fold change) coupled with false discovery rate (FDR, log10PadJ). Pathway enrichment analysis was acquired using MetaboAnalyst6.0. Functional validation of RD3-dependent metabolic shift was performed by assessing mitostress (Seahorse Xfe Analyzer) in RD3 expressing clone (CHLA-15) derived from primary (retroperitoneal) site, RD3-null CHLA-90 (BM) and compared with their RD3 silenced and expressed clones respectively. Compared to RD3-/- clones, we observed definitive conserved (cell-line independent) rearrangement of 289 metabolites (288, downregulated; 1, upregulated) in RD3+/+ clones. Predominantly, the metabolites were highly associated in amino acid metabolism (p= 6.61*10-3; FDR, 2.1*10-1) followed by fatty acid and steroid metabolism (p= 3.7*10-2; FDR, 3.02*10-1). Notably, metabolites involved in aerobic glycolysis (Warburg effect, Enrichment score, 1.953) were among the least active in RD3+/+ reinstated clones indicating a significant shift in cell energetics. Consistently, mitostress test further revealed that RD3-loss in HR-NB favors a glycolytic phenotype whereas RD3 expression promoted a more quiescent phase. Our findings explicitly demonstrate that RD3 expression in metastatic NB suppresses glycolysis and reprograms cellular metabolism. Furthermore, the conserved shifts in amino acid and fatty acid metabolism highlight RD3 as a potential metabolic gatekeeper. These RD3-driven alterations could unveil novel therapeutic vulnerabilities in HR-NB, offering new avenues for targeted intervention.



 This work was funded by the DoD CA-210339; OCAST-HR19-045; and the NIH P20GM103639



 Poorvi Subramanian, Natarajan Aravindan, Sivaroopan Aravindan. Breaking the Warburg code: RD3 redefines energetics in metastatic Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdda285b7679b38ef5248455c327701b43a9d61b" target='_blank'>
              Abstract B045: Breaking the Warburg code: RD3 redefines energetics in metastatic Neuroblastoma
              </a>
            </td>
          <td>
            Poorvi Subramanian, N. Aravindan, Sivaroopan Aravindan
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 TERT promoter mutations are the most frequent single nucleotide variants (SNVs) in glioblastoma and many other cancers. These hotspot mutations provide novel sites for transcription factor binding, activating expression of TERT gene and leading to telomere elongation and cancer cell immortalization. Moreover, these SNVs disrupt the highly structured G-quadruplex region of TERT promoter, impacting chromatin organization more broadly. Since epigenetic cell state is highly dynamic and regulated by cues from the tumor microenvironment, studying these changes in freshly resected tumor tissue is highly relevant. To enable such studies in glioblastoma, we developed STAR-FACS, a method allowing for cell sorting based on an SNV, that can be applied to freshly dissociated human tumor tissue. The single cell suspension is fixed, and a mutation-specific PCR is performed in the intact cells. Incubation of the cells with a labeled probe that hybridizes to the PCR product renders the cells fluorescent and amenable for cell sorting. Using TERTp WT and MUT glioblastoma cell lines and neurospheres, we demonstrated that STAR-FACS protocol is compatible with downstream profiling of the sorted cells by bulk and single cell transcriptomics and epigenetic profiling with CUT&Tag. We also show that TERT promoter mutation-based cancer cell enrichment can be performed on freshly dissociated glioblastoma tissue. STAR-FACS allowed us to compare expression profiles of TERT promoter mutant glioblastoma cells accumulating 5-ALA with 5-ALA- cancer cells from the same tumor, which would be challenging with standard methods. Our mutation-based cell sorting method provides a novel, inexpensive tool for studies linking the effect of subclonal non-coding SNVs on transcriptomic and epigenetic heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297a12bdc543566d0e93d050acddeb7d7c6644de" target='_blank'>
              EPCO-50. TERT PROMOTER MUTATION-BASED CELL SORTING FOR SINGLE CELL TRANSCRIPTOMIC AND EPIGENETIC PROFILING OF GLIOBLASTOMA CELLS
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, O. Szentirmai, M. Janiszewska
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="OBJECTIVE
Large-scale CRISPR screens have identified essential genes across cancer cell lines, but links between tumor functional properties and specific dependencies require investigation to reveal the mechanisms underlying dependencies and broaden understanding of targeted therapy.


METHODS
We selected 47 breast cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) with multi-omics data including gene dependency; somatic mutations; copy number alterations; and transcriptomic, proteomic, metabolomic, and methylation data. We established a dependency marker association (DMA) analytic pipeline by using linear regression modeling to assess associations between 3,874 representative gene dependencies and multi-omics markers. Additionally, we conducted non-negative matrix factorization clustering, to stratify breast cancer cell lines according to gene dependency features, and investigated cluster-specific DMAs.


RESULTS
We interpreted valuable DMAs according to two primary aspects. First, dependencies associated with gain-of-function alterations revealed addiction to lactate transporter SLC16A3, thus suggesting a promising therapeutic target. Second, dependencies associated with loss-of-function alterations included synthetic lethality (SL), collateral SL, and prioritized metabolic SL, encompassing paralog SL (e.g., IMPDH1 and IMPDH2), single pathway SL (e.g., GFPT1 and UAP1), and alternative pathway SL (e.g., GPI and PGD). DMA analysis of the two clusters with divergent dependency signatures demonstrated that cluster1 cell lines exhibited extensive metabolism with mitochondrial protein dependencies, whereas cluster2 displays enhanced cell signaling, and reliance on DNA replication and membrane organelle regulators.


CONCLUSIONS
We established a DMA analysis pipeline linking the gene dependencies of breast cancer cell lines to multi-omics characteristics, thus elucidating the underpinnings of tumor dependencies and offering a valuable resource for developing novel precision treatment strategies incorporating relevant markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/753908ccf816b6d23f787d4f37906f4b14d28c45" target='_blank'>
              Comprehensive investigation of the molecular basis of cancer dependencies suggests therapeutic options for breast cancer.
              </a>
            </td>
          <td>
            Rui Ding, Zhimin Shao, Tianjian Yu
          </td>
          <td>2025-10-23</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/906f1e267f0452865e325381ce2fb991c017ccf2" target='_blank'>
              Are Different Populations Fairly Represented in Single-Cell Omic Atlases?
              </a>
            </td>
          <td>
            Catrina Yang, Kavitharini Saravanan, Aryan Saharan, Kuan-lin Huang
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Diverse fusions of EML4 and ALK are oncogenic drivers in lung adenocarcinomas. EML4-ALK variants have distinct breakpoints within EML4, but their functional differences remain poorly understood. Here, we use somatic genome editing to generate autochthonous mouse models of EML4-ALK-driven lung tumors and show that V3 is more oncogenic than V1. By employing multiplexed genome editing and quantifying the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth, we show that many tumor suppressor genes have variant-specific effects on tumorigenesis. Pharmacogenomic analyses further suggest that tumor genotype can influence therapeutic responses. Analysis of human EML4-ALK-positive lung cancers also identified variant-specific differences in their genomic landscapes. These findings suggest that EML4-ALK variants behave more like distinct oncogenes rather than a uniform entity and highlight the dramatic impact of oncogenic fusion partner proteins and coincident tumor suppressor gene alterations on the biology of oncogenic fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c8a78e3c9e4fd68a0441eaf2f2edc5f59a5afae" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis.
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Karen Y. Shih, Óscar González-Velasco, Mulham Najajreh, Stewart Kim, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, Ilse Hofmann, S. Sivakumar, S. Sisoudiya, Ethan S Sokol, Hongchen Cai, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2025-09-23</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma (PDA) has poor response to chemotherapy with a strong likelihood of cancer recurrence. While previous studies have examined patient samples collected before and after chemotherapy, few have analyzed matched samples from the same patients that allow us to deconvolute interpatient heterogeneity, a large limitation to PDA studies. Here, we study the effects of chemotherapy on tumor cells and their surrounding microenvironment using scRNA sequencing of matched pre- and post- treatment tissue biopsies from 5 patients. We utilized the Seurat v4 pipeline to observe transcriptomic differences and corresponding putative cellular interactions in both treatment states. Unsurprisingly, we found insufficient tumor epithelial cells post-treatment due to the intended cytotoxic effects of chemotherapy, leading us to focus on the tumor microenvironment where we had sufficient cells to analyze. We identified that immune cells in the TME are heterogeneously affected by chemotherapy treatment with patient specific differences, but we did identify a unanimous increase in CXCR4 in CD4 and CD8 positive T cells after treatment. Analysis of CAFs revealed that a subset of fibroblasts lose myofibroblastic features and increasingly acquire 'neural'-like features after treatment. This suggests that chemotherapy-altered fibroblasts may contribute to a pro-tumor microenvironment that facilitates recurrence through neuron–CAF–tumor interactions. Through inferred ligand-receptor pairs identified with the CellChat v2 pipeline, we observed that increased CXCL12 in these neural CAFs interacts with the increased CXCR4 on T-cells. We validated the expression of these genes in patients by staining human patient PDAC tissues (both treated and treatment naïve tissues) for neural genes that were highly enriched post-treatment in our single cell sequencing dataset. We have also begun inhibiting these genes of interest in CAFs and are exploring the effects of these ‘neural’-CAFs on patient-derived tumor organoids in culture.



 Aylin Z. Henstridge, Padma Kadiyala, Ahmed Elhoissiny, Jianhua Liu, Georgina Branch, Nandini Arya, Vaibhav Sahai, Nicole Peterson, Jorge Machicado, Richard Kwon, Allison Schulman, Erik Wamsteker, George Phillips, Martin Fernandez-Zapico, Mark Truty, Timothy Frankel, Fillip Bednar, Marina Pasca di Magliano, Costas Lyssiotis, Eileen S. Carpenter. An increase in neural cancer associated fibroblasts following exposure to chemotherapy in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B079.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13e666941a86feb629e0cf1155d415645c6c810d" target='_blank'>
              Abstract B079: An increase in neural cancer associated fibroblasts following exposure to chemotherapy in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Aylin Z. Henstridge, Padma Kadiyala, Ahmed Elhoissiny, Jianhua Liu, Georgina Branch, Nandini Arya, Vaibhav Sahai, Nicole Peterson, Jorge D. Machicado, Richard S Kwon, Allison Schulman, Erik Wamsteker, George Phillips, Martín E Fernandez-Zapico, M. Truty, Timothy L. Frankel, Fillip Bednar, Marina Pasca di Magliano, C. Lyssiotis, Eileen S. Carpenter
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5bbf3daa9e6f41e402a48c9bc2bd5b7f13849c" target='_blank'>
              Systematic elucidation and pharmacologic targeting on non-oncogene dependencies in imatinib-resistant gastrointestinal stromal tumor
              </a>
            </td>
          <td>
            Prabhjot S. Mundi, A. Grunn, A. Kojadinovic, Charles Karan, Ronald B. Realubit, Cristina I. Caescu, H. Hibshoosh, Mahalaxmi Aburi, Mariano J. Alvarez, M.A. Ingham, D. Evans, S. Rothschild, Gary K. Schwartz, Andrea Califano
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="


 High-grade gliomas (HGGs) are aggressive, treatment-resistant brain tumors marked by extensive intratumoral heterogeneity, complicating formulation of effective adjuvant therapy. Standard of care treatment is unimodal and fails to eradicate the tumor in the majority of patients. Traditional models like spheroids and patient-derived xenografts (PDXs) offer biological insights but lack feasibility for clinical translation. Patient-derived glioma tumor organoids (GTOs), by contrast, are rapidly scalable, preserve native tumor architecture, and enable high-throughput patient-specific drug testing.



 To establish a reproducible, high-throughput GTO platform that faithfully models glioma biology and enables individualized therapeutic screening within a clinically actionable timeframe.



 Multi-regional tumor samples from IDH-wildtype, IDH-mutant, and H3K27M-mutant gliomas were collected during clinically-indicated tumor resection, minced, and cultured for 2–4 weeks in specialized medium. Organoids were characterized using multimodal microscopy and bulk RNA sequencing to assess fidelity to native tumor as well as inter-organoid heterogeneity in oncogenic drivers and glioma cell state. High-throughput drug screening was performed to evaluate intra- and inter-patient treatment response variability.



 Histological analysis revealed heterogeneity in cellular architecture and cell state (OPC-like, MES-like, and AC-like) marker expression across organoids. RNA-sequencing confirmed preservation of transcriptional diversity in cell state proportions and highlighted differential activation of oncogenic pathways (WNT, MAPK, JAK-STAT). Drug screening revealed significant variability in therapeutic sensitivity across regions and patients, underscoring the importance of personalized treatment strategies and multi-drug regimens.



 This study demonstrates feasibility of a rapid, scalable GTO platform for precision oncology in HGG, capable of informing patient-specific treatment selection. While limitations remain—including absence of immune microenvironment and delivery across the blood-brain barrier—GTOs represent a promising tool for nominating personalized multi-drug treatment regimens and for refining patient selection in clinical trials.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aa285381bb9dfd17256ce64b1d6d93936974c5c" target='_blank'>
              TMOD-21. Precision Oncology for High-Grade Gliomas: A Tumor Organoid Platform for Rapid, Patient-Specific Drug Screening
              </a>
            </td>
          <td>
            Arushi Tripathy, Sunjong Ji, Yazen Alomary, Reka Chakravarthy Raghunathan, S. Syed, V. Ravikumar, Sunita Shankar, H. Serhan, Dah-Luen Huang, Toshiro Hara, N. Clarke, C. Koschmann, Nathan M. Merrill, S. Merajver, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) is the most common form of kidney cancer, leading to 179 000 cancer-related deaths annually. ccRCC initiating cells (CICs) are thought to drive tumor initiation, growth, therapy resistance, and metastasis, yet their molecular characteristics remain poorly defined. This study aims to identify putative CICs and their essential genes using a cell marker-agnostic strategy. Methods We performed a comprehensive analysis of ccRCC transcriptomes at single-cell resolution, developed patient-derived xenograft (PDX) and 3D patient-derived organoids (PDO) models of ccRCC, and conducted functional examinations in these models to investigate our findings. Results Computational modeling of tumor formation using single-cell RNA velocity analysis of five primary and metastatic ccRCC-PDXs revealed a minor cell population as the origin of other tumor cells, representing putative CICs. Pathway and network analyses suggested that a core network of proteins, conventionally known to regulate mitosis, are highly active in CICs and may be essential for their function. These proteins were expressed in PDOs and PDX-derived spheroids established following a CIC enrichment protocol. Spheroid cells exhibited higher tumorigenicity and colony formation ability than parental tumor cells, as confirmed by in-vivo injection in nude mice and in-vitro colony formation assays. Successive in vivo passaging confirmed the self-renewal capacity of spheroid-derived tumors. Pharmacological blockade of candidate proteins elicited dose-dependent inhibitory effect on spheroid and colony formation, with in-vivo validation showing that blocking these proteins significantly delayed tumor growth and more efficiently prevented tumor formation in mice. These results highlight the importance of these proteins in the cancer-initiating abilities of malignant cells. Interestingly, our results suggest that the identified target proteins may elicit their CIC-related function independently from their mitosis regulatory roles. Conclusions We identified and validated essential proteins in RCC-CICs, supported by single-cell transcriptome data and RCC spheroids and PDX models. Targeting the vulnerabilities of RCC-CICs, given their role in tumor initiation, progression, and therapy resistance, offers significant potential for developing new anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edac148fb846da9cb774550aaf710d22fbf68ef2" target='_blank'>
              64defining cancer initiating cells and their vulnerabilities in renal cell carcinoma
              </a>
            </td>
          <td>
            Zohreh Mehrjoo, H. Kuasne, Ariel Madrigal Aguirre, A. Shahini, M. Annis, Anne-Marie Fortier, Tianyuan Lu, Larisa Morales Soto, Hong Zhao, D. Zuo, V. Pilon, M. Dankner, Tamiko Nishimura, Kevin Petrecca, Jonathan Spicer, Peter Siegel, S. Tanguay, H. Najafabadi, Morag Park, Yasser Riazalhosseini
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Understanding the complex molecular mechanisms driving drug resistance in glioblastoma (GBM) is crucial to develop effective therapeutic strategies. While prior studies have identified resistance mechanisms tied to specific drugs or pathways, a multi-modal molecular analysis of resistance across a range of drug classes is lacking. We identified highly drug-resistant (n = 5) and drug-sensitive (n = 4) cultures from a cohort of 32 patient-derived glioblastoma stem cell (GSC) cultures screened against a broad panel of ~ 500 anti-cancer drugs. To elucidate the key drivers of drug resistance, we performed integrative profiling of stemness, differentiation capacity, global gene expression, mutation profiles, and DNA methylation patterns between the two groups. Despite heterogeneous gene expression profiles, drug-resistant GSCs showed consistent upregulation of ATP-binding cassette (ABC) drug efflux transporters, stemness, and extracellular matrix (ECM)-related genes. Compared to drug-sensitive GSCs, drug-resistant GSCs exhibited more pronounced stem-like properties and reduced differentiation capacity. Notably, genes linked to axonogenesis displayed significant CpG island hypomethylation in drug-resistant GSCs. This study suggests a pivotal role for GSC plasticity, stemness maintenance, and ECM-mediated drug evasion in GBM treatment resistance. Our findings highlight the adaptive and dynamic nature of resistance mechanisms in GSCs, emphasizing the need for comprehensive molecular insights to inform targeted therapeutic strategies in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39de902ec6cfcaa0e52251d953e6e404ad13d38" target='_blank'>
              Drug resistant glioblastoma stem cells exhibit enriched stemness signatures and share extracellular matrix overexpression
              </a>
            </td>
          <td>
            Lance L Estabillo, E. Skaga, S. Halldorsson, Einar O Vik-Mo, C. Sandberg
          </td>
          <td>2025-10-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cce6be4d706b4653ff7064a56c5297b13de1df65" target='_blank'>
              Travelling Waves in Gene Expression: A Mathematical Model of Cell-State Dynamics in Melanoma
              </a>
            </td>
          <td>
            Charlotte Taylor Barca, Rotem Salmi-Leshem, Vishaka Gopalan, Sarah Woolner, Kerrie L. Marie, Gareth Wyn Jones, Oliver E. Jensen
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Metastatic castration-resistant prostate cancer/PCa (mCRPC) is a clinically advanced form of PCa that is associated with increased aggressiveness, cancer stemness, morbidity, and the risk of developing resistance to taxanes, currently the first-line chemotherapy for mCRPC. Clofazimine (CLF) is a potential immunomodulator drug that is FDA-approved for the treatment of leprosy. Recently, using in vitro, in vivo, and ex vivo models, we established the efficacy of CLF in chronic myeloid leukemia and multiple myeloma. Here, we demonstrate that CLF is effective as a single agent and in combination with taxanes in a panel of cell lines representing the diversity of CRPC patients. Using a microfluidic assay, we showed the impact of CLF on cancer cell migration and metastatic potential. Further, we also found that CLF reduces ALDH activity—a marker for cancer ‘stem-like’ cells (CSCs), a subtype of cancer cells with self-renewal and differentiation capacities (epithelial-to-mesenchymal transdifferentiation/EMT). Bulk and single-cell RNAseq followed by functional validation and in silico analysis showed that CLF treatment is associated with apoptosis, ER stress, oxidative phosphorylation, and mitochondrial dysfunction. Most importantly, CLF modulates the expression of several non-coding RNAs, including MALAT1 and NEAT1, that are linked to tumor cell proliferation, cell migration, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b0a0423c975cc7befb26cd26f25ee60d9061c02" target='_blank'>
              Clofazimine Treatment Modulates Key Non-Coding RNAs Associated with Tumor Progression and Drug Resistance in Lethal Prostate Cancer
              </a>
            </td>
          <td>
            Sarah Batten, Harish Kumar, Jeremiah Pfitzer, Daniel Chinedu Nweze, Suman Mazumder, Robert D. Arnold, P. Mistriotis, Taraswi MITRA GHOSH, A. Mitra
          </td>
          <td>2025-11-10</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="PURPOSE
Medulloblastoma (MB) is a common pediatric brain tumor with distinct molecular subgroups, of which, Group 3 MB is associated with increased recurrence, metastatic potential and poor patient outcomes. Small molecule inhibitors targeting BMI1 have been shown to be efficacious against several types of malignant tumors, including pediatric MB. While our previously published in vivo study provides a promising proof-of-concept for the therapeutic targeting of BMI1 in Group 3 MB with small molecule inhibitor, it is not sufficient to eradicate the tumour.


EXPERIMENTAL DESIGN
In this study, following preclinical validation of BMI1 inhibitor PTC-596, DNA barcoding technology was leveraged to profile in vivo clonal dynamics of Group 3 MB in response to the established chemoradiotherapy regimen alone and in combination with PTC-596. Following demonstration of a small number of treatment-refractory clones we sought to identify potential druggable molecular vulnerabilities by utilizing phosphoproteomic profiling and genome-wide CRISPR screening.


RESULTS
By comparing the changes in phosphorylation pattern of key signaling kinases post PTC-596 treatment with the list of sensitizer genes from in vitro genome-wide CRISPR/Cas9 screen and to the essential genes in human neural stem cells (hNSCs), we identified several context-specific regulators of mTOR, AKT and PLK1 pathways. Subsequently, targeting the PI3K pathway with Enzastaurin was shown to be most meanable to synergistic targeting alongside BMI1 inhibition.


CONCLUSION
This work provides the foundation for clinical validation of small-molecule inhibitors synergistic with PTC-596 to improve the durability of remissions and extend survival of patients with treatment-refractory Group 3 MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254ad36a007c276e881f281d38c624b50e4a39ec" target='_blank'>
              Leveraging Medulloblastoma Clonal Dynamics to Overcome Treatment Resistance.
              </a>
            </td>
          <td>
            D. Bakhshinyan, Stefan Custers, Laura Escudero, Yujin Suk, Kevin R. Brown, Hardikkumar Patel, A. Adile, C. Chokshi, M. Shaikh, D. McKenna, M. Qazi, Kui Zhai, D. Tieu, Katherine Chan, Marla Weetal, C. Venugopal, Jason Moffat, Sheila K Singh
          </td>
          <td>2025-10-15</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Notch is a contact-dependent signaling pathway that plays critical roles in spatiotemporal patterning during embryogenesis. Beyond its role in development, Notch signaling regulates tissue patterning in adult organs in response to tissue injury or cancer. In the context of cancer, the role of Notch signaling has been extensively characterized in cancer cells, T cells, and endothelial cells. Our previous work has implicated Notch signaling in the myeloid and fibroblast compartments of the pancreatic tumor microenvironment (TME) in the tumor immune response, an area that has been previously understudied. In our current work we use genetically engineered mouse models to inhibit canonical Notch activation in fibroblasts (Pdgfra CreERT2/+, LSL-DNMAML) and myeloid cells (LysM Cre/+, LSL-DNMAML) separately. In both an orthotopic transplant and genetic model (Ptf1a FlpO/+;FSF-Kras G12D/+) of pancreatic cancer progression, inhibition of fibroblast-specific Notch activation had limited effects on disease progression and the transcriptional profile of the fibroblasts themselves. This is consistent with previous evidence of more limited Notch activation in pancreatic cancer-associated fibroblasts and suggests a primary role for fibroblasts in providing Notch ligand (JAG1) to neighboring cells. In contrast, inhibition of myeloid-specific Notch activation resulted in smaller orthotopic tumors and slowed progression in a genetic model of pancreatic neoplasia. These samples had reduced overall macrophage infiltration, and a secondary alteration of the fibroblast phenotypes was also seen. Collectively, these results suggest that Notch signaling in the myeloid compartment supports tumor progression and regulates multiple components of the pancreatic TME. Future studies will further interrogate how myeloid-specific Notch activation promotes disease progression and affects Notch signaling activation in adjacent compartments.



 Allison Bischoff, Wei Yan, Carlos Espinoza, Emily Lasse-Opsahl, Nur Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar. Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A080.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df0eac24a706ec3b2888b93172ea892748b9176" target='_blank'>
              Abstract A080: Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment
              </a>
            </td>
          <td>
            Allison C Bischoff, Wei Yan, Carlos E Espinoza, Emily L. Lasse-Opsahl, Nur M Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Bladder cancer (BLCA) exhibits considerable heterogeneity, and research into its tumor microenvironment and prognostic biomarkers remains insufficient. This study aims to identify key prognostic genes and immune microenvironment factors to enhance clinical assessment. Primary tumor (PT) and lymph node metastasis (LNM) samples were obtained from patients with BLCA, and single-cell RNA sequencing (scRNA-seq) was conducted to analyze cellular composition and functional variations. Prognostic genes were identified based on integrated bulk transcriptomic data, leading to the development of a risk model. Functional enrichment analyses further elucidated the molecular characteristics of high- and low-risk cohorts. The scRNA-seq analysis revealed significantly elevated metabolic activity in epithelial cells of LNM. A subpopulation of epithelial cells, defined by 133 characteristic genes, was identified as pivotal in the lymphatic metastasis of BLCA. The prognostic model, derived from nine key genes (APOL1, CAST, DSTN, SPINK1, JUN, S100A10, SPTBN1, HES1, and CD2AP), demonstrated robust predictive performance. Functional enrichment analysis indicated that the high-risk group predominantly activated extracellular matrix (ECM) receptor interactions and the complement pathway, while the low-risk group was primarily associated with carbohydrate metabolism pathways. This study provides insights into the tumor heterogeneity and immune microenvironment of BLCA, introduces a high-precision prognostic model, and establishes a novel theoretical basis for personalized BLCA treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b5f162b6ba0c04c311dbcacd7e486bc5b330efc" target='_blank'>
              Integration of Single-cell and bulk RNA sequencing data uncovers lymphatic metastasis-related prognostic genes and a predictive model in bladder cancer
              </a>
            </td>
          <td>
            Bin Yang, Yangbing Wei, Shuqin Liu, Jinyao Wang, Xuwei Zhao, Jingyu Wang, Yongqiang Fu, Changxi Wang, Xiaofeng Yang
          </td>
          <td>2025-11-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01b5dde4ed4b216cb7cf78338e646166d480ca06" target='_blank'>
              Spatial proteomics of ovarian cancer precursors delineates early disease changes and drug targets.
              </a>
            </td>
          <td>
            Anuar Makhmut, M. Dragomir, Sonja Fritzsche, Markus Moebs, W. Schmitt, Eliane T. Taube, Fabian Coscia
          </td>
          <td>2025-11-13</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most malignant primary brain tumor, characterized by extensive heterogeneity, invasiveness, infiltrating behavior, and resistance to standard therapies, including radiation and temozolomide (TMZ). Despite considerable efforts in investigating its pathophysiology, GBM represents one of the most challenging cancers to treat, with a median survival rate under 15 months and a 5-year survival rate below 5%. A major barrier to progress in GBM therapy development is the lack of reliable preclinical models that faithfully recapitulate the tumor’s molecular heterogeneity, invasive behavior, and complex microenvironment. Traditional cell lines and xenograft models often fail to reflect the key pathological features of human GBM, including immune suppression, vascular abnormalities, and treatment resistance. In recent years, attention has focused on the development of numerous clinically relevant GBM models based on brain organoids as a powerful “disease-in-a-dish” model. They strongly mimic GBM key histopathological and molecular features, such as the tumor’s cellular heterogeneity, genetic landscape, and microenvironment, enabling more accurate studies of tumor biology, invasion, and therapeutic response in a controlled in vitro setting. Notably, research in microgravity offers a unique and promising platform to study cancer biology under conditions that enhance tissue self-organization, mimic aspects of tumor growth, and potentially unveil novel therapeutic vulnerabilities. This review compares organoids to conventional preclinical models, tracing their historical development and salient features, focusing on the preparation and use of organoids in GBM research. We also introduce a novel and promising field of organoid application: space-based organoid brain research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/060d09698aac599adbbb8839f1b67125def63fa8" target='_blank'>
              Modeling Glioblastoma with Brain Organoids: New Frontiers in Oncology and Space Research
              </a>
            </td>
          <td>
            Laura Begani, L.G. Remore, Stefania Ragosta, M. D. Rizzaro, L. Guarnaccia, Giovanni Alotta, Laura Riboni, Monica R. Miozzo, Emanuela Barilla, Chiara Gaudino, Marco Locatelli, E. Garzia, G. Marfia, S. Navone
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178ca173745db4823628df69a4e845e3404906a8" target='_blank'>
              Epigenetic modulation of CTCs as a novel therapeutic target in oncology.
              </a>
            </td>
          <td>
            Husni Farah, M. Abosaoda, Hayjaa Mohaisen Mousa, S. Jyothi, Priya Priyadarshini Nayak, J. Janney, Gurjant Singh, Ashish Singh Chauhan
          </td>
          <td>2025-10-11</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="T-cell leukemias and lymphomas (TCLs) form a heterogeneous group of rare and often aggressive malignancies. Due to the rarity and heterogeneity of TCL subtypes, clinical trials are challenging to conduct, making pharmacogenomic studies in cell line panels critical for the discovery of targeted therapeutics. The scarcity of data repositories with integrated multi-omics and drug screening data hinders the preclinical evaluation of drug vulnerabilities and the identification of molecular markers predictive of responses to monotherapies and combinations. To address this gap, we conducted comprehensive pharmacogenomic profiling on a panel of 38 TCL cell lines, representing major clinical TCL subtypes to capture the molecular and phenotypic diversity. The TCL-38 multi-omics data resource includes harmonized genetic, molecular, and epigenetic profiles, with comprehensive annotations, and standardized drug response assessment of each cell line. This resource, together with machine learning predictions, was leveraged to identify TCL subtype-specific therapeutic vulnerabilities, including single-agent sensitivities and synergistic drug combinations, which were linked to genetic or epigenetic features as potential predictive biomarkers. This integrated and openly available resource (https://aittokallio.group/tcl38) could help advance the currently limited treatment options for patients with TCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df15d5c0443d2d2b30f24f2a00bf9e3203f8ea6d" target='_blank'>
              Multi-omics Profiling of T-cell Leukemia and Lymphoma Enables Targeted Therapeutic Discovery.
              </a>
            </td>
          <td>
            A. Ianevski, Kristen Nader, Julia Nguyen, H. Sorger, S. Timonen, Edith Julia, Daniel Pölöske, Katrin Spirk, Christina Wagner, Dennis Jungherz, Minoru Nakano, Sisira Kadambat Nair, Philipp Ianevski, M. Kankainen, Diogo Dias, A. Cichońska, T. Pemovska, C. Pirker, Walter Berger, T. Braun, Richard Moriggl, Emmannuel Bachy, S. Mustjoki, Marco Herling, H. Neubauer, Benjamin Haibe-Kains, T. Aittokallio
          </td>
          <td>2025-10-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84a5395a951267c1cd8002a59d2c110d6faa6ffd" target='_blank'>
              Machine Learning-Driven Discovery of Synergistic Protein Interactions Identifies ATL3 as a Putative Biomarker for Cancer Drug Response
              </a>
            </td>
          <td>
            Priya Ramarao-Milne, A. Kaphle, R. Reguant, Anne Klein, Michael J Kuiper, B. Hosking, Julika Wenzel, H. Al-Mamun, Lujain Elazab, Qing Zhong, R. Reddel, Laurence Wilson, Yatish Jain, N. Twine, Denis C. Bauer
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, A. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Background HOXA5 (homeobox A5) exhibits context-dependent roles in cancer, but its pan-cancer spatial immune regulatory functions and therapeutic potential remain poorly understood. Methods We integrated multi-omics data from 33 cancer types (TCGA, n=11,096; GTEx, n=7,469; TISCH2; spatial transcriptomics) to characterize HOXA5 expression, genomic alterations, and immune interactions. Functional validation was performed in AML cell lines (U937, KG-1; n=3 biological replicates per experiment). Results HOXA5 was significantly dysregulated across cancers, with elevated expression in AML and GBM, and reduced expression in BRCA and LUAD. In AML, high HOXA5 expression predicted poor overall survival (HR = 2.80, 95% CI: 1.60–4.89, p < 0.001) and was associated with FLT3/NPM1 mutations. Spatial transcriptomics revealed HOXA5+ malignant cells enhance fibroblast/endothelial crosstalk via IGFBP3-TMEM219. HOXA5 knockdown suppressed proliferation (p < 0.01) and induced G0/G1 arrest. Mechanistically, HOXA5 maintained AML through cholesterol biosynthesis and ECM remodeling. Mercaptopurine was identified as a potential therapeutic agent, and molecular docking predicted a potential stable interaction with HOXA5. Conclusions HOXA5 plays a dual role in solid versus hematologic malignancies and serves as a key spatial immune regulator. It is a robust prognostic biomarker and therapeutic target in AML, with mercaptopurine representing a promising repurposing candidate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbaf2f57507e416c0e787238787727a2c2564a0c" target='_blank'>
              HOXA5-mediated spatial remodeling of tumor-immune interfaces across cancers promotes AML pathogenesis
              </a>
            </td>
          <td>
            Changling Zhang, Ping Wen, Yan-yan Zeng, Tao Chen, Qulian Guo, Chunyan Liu, Fangfang Zhong
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Li-Fraumeni syndrome (LFS) has recently been redefined as a ‘spectrum’ cancer predisposition disorder to reflect its broad phenotypic heterogeneity. This variability is thought to stem in part from the diverse functional impacts of TP53 variants, although the underlying mechanisms remain poorly understood and there is an unmet clinical need for effective risk stratification. Here, we apply unsupervised clustering to functional datasets and identify distinct TP53 variant groups with clinical relevance, including a monomeric subgroup enriched in osteosarcoma cases. In cellular validation assays, dermal fibroblasts from carriers of more functionally impaired variants exhibit increased metabolic growth rates, mirroring trends observed in cluster-stratified clinical outcomes. These findings demonstrate the feasibility of developing diagnostic assays to guide personalized cancer risk assessment. More broadly, our results show that nuances in TP53 dysfunction shape the germline TP53-related cancer susceptibility spectrum and provide a framework for functionally delineating variant carriers. Li-Fraumeni syndrome is a cancer predisposition disorder caused by TP53 variants, but the way different TP53 variants contribute remains unclear. Here, the authors analyse TP53 mutagenesis datasets and identify five TP53 variant clusters that show associations with specific cancer patterns as well as potential clinical strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/569376cd7c0cf442a0e7aeae9ad9a43e8cf04a08" target='_blank'>
              TP53 variant clusters stratify phenotypic diversity in germline carriers and reveal an osteosarcoma-prone subgroup
              </a>
            </td>
          <td>
            N. Fischer, N. Ong, B. Laverty, P. Psarianos, Camilla M. Giovino, N. Alon, E. Montellier, P. Hainaut, K. Maxwell, Christian P. Kratz, R. Kafri, David Malkin
          </td>
          <td>2025-09-29</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6971b52e8812bcb489dfce1326931d0b1e99d8ab" target='_blank'>
              Cell villages and Dirichlet modeling map human cell fitness genetics
              </a>
            </td>
          <td>
            Chloe Hanson, Tim Derebenskiy, Ana Rodriguez Vega, Yashika S. Kamte, Rachel G. Fox, Hannah Lambing, Patrick Allard, Harold Pimentel, Michael F. Wells
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Lymph node (LN) lymphatic endothelial cells (LEC) actively acquire and archive foreign antigens. Here, we address questions of how LECs achieve durable antigen archiving and whether LECs with high levels of antigen express unique transcriptional programs. We use single cell sequencing in dissociated LN tissue and spatial transcriptomics to quantify antigen levels in LEC subsets and dendritic cell populations at multiple time points after immunization and determine that ceiling and floor LECs archive antigen for the longest duration. We identify, using spatial transcriptomics, antigen positive LEC-dendritic cell interactions. Using a prime-boost strategy we find increased antigen levels within LECs after a second immunization demonstrating that LEC antigen acquisition and archiving capacity can be improved over multiple exposures. Using machine learning we define a unique transcriptional program within archiving LECs that predicts LEC archiving capacity in independent mouse and human data sets. We test this modeling, showing we can predict lower levels of LEC antigen archiving in chikungunya virus-infected mice and demonstrate in vivo the accuracy of our prediction. Collectively, our findings establish unique properties of LECs and a defining transcriptional program for antigen archiving that can predict antigen archiving capacity in different disease states and organisms. Dendritic cells are supplied antigens by other cells such as lymphatic endothelial cells (LEC) at late time points after immunization. Here the authors show antigen archiving is defined by a transcriptional program that can predict antigen archiving depending on the priming pathogen, and that boosting of immune responses increases the archiving.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/382f59662225a03a1be36e91e9b0ab93cd6fd5d9" target='_blank'>
              A specific gene expression program underlies antigen archiving by lymphatic endothelial cells in mammalian lymph nodes
              </a>
            </td>
          <td>
            Ryan M. Sheridan, Thu A Doan, C. Lucas, Tadg S Forward, Ira Fleming, Valerie M Olsen, Abrianna M Qvale, Bennett J. Davenport, Kristen Zarrella, Michael G. Harbell, Aspen Uecker-Martin, Thomas E. Morrison, Jay R. Hesselberth, B. A. Tamburini
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Gene expression heterogeneity generates subpopulations of tumor cells that can evade therapeutic pressure. This heterogeneity has been observed in both primary Estrogen Receptor alpha positive (ERα+) breast tumors and cell lines. Therefore, understanding the mechanisms regulating expression heterogeneity is critical towards developing effective therapies. A key contributor to gene expression variability is the stochastic nature of transcription. Transcription occurs in a probabilistic, burst-like manner, in which gene activation occurs intermittently, producing RNA in pulses and interspersed with transcriptional off-periods. The estrogen-responsive gene TFF1 is expressed in the majority of ERα⁺ breast tumors and exemplifies such heterogeneity, with transcriptional inactivity ranging from minutes to several days. Here, we identify the molecular mechanism underlying the wide range in TFF1 expression by analyzing cells sorted based on their TFF1 activity levels. We observed that TFF1 inactive (TFF1low) cells exhibit a repressive chromatin state marked by H3K27me3 at the TFF1 promoter and enhancer. Despite global similarity in ERα binding, occupancy at the TFF1 regulatory elements was selectively reduced in TFF1low cells, resulting in fewer active alleles and diminished transcriptional bursting frequency. Conversely, TFF1high cells exhibited more active TFF1 alleles and hyperbursting. These cells also retained sensitivity to endocrine therapy, while TFF1low cells displayed reduced drug responsiveness. Genome-wide, differentially enriched H3K27me3 regions correlated with variable expression of estrogen-responsive genes, highlighting a broader regulatory mechanism that links chromatin state to expression variability. Together, our findings establish how repressive chromatin dynamics contribute to gene expression heterogeneity and endocrine resistance in ERα⁺ breast cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa89e16be6e3a720a965e4891206ee026e2fe10" target='_blank'>
              H3K27me3 chromatin heterogeneity reveals variable cell responses to estrogen and endocrine treatment
              </a>
            </td>
          <td>
            Fleur Chapus, Christopher R. Day, Laura G Kammel, Pelin Yasar, Brian D. Bennett, Erica Scappini, Charles J Tucker, Maria Sifre, Celyn Bregio, Joseph Rodriguez
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a highly aggressive brain cancer with limited treatment options and poor long-term survival. Adoptive cellular therapy (ACT) has shown promise in targeting GBM, but current strategies often rely on tumor-derived antigens, limiting their applicability in patients with inoperable tumors. Moreover, GBM recurrence is driven by tumor heterogeneity and glioma stem cells (GSCs) that evade standard therapies. Radial glial cells (RGCs), neurodevelopmental progenitors with transcriptional similarity to GSCs, express developmental antigens (DAs) also found in GBM, suggesting their potential as a tumor-independent, renewable antigen source. To explore this, we used single-cell RNA sequencing to profile RGCs from postnatal day 4 murine cortices and identified a subset of antigens shared with tumors from the KR158B GBM model. Using our O.R.A.N. antigen prediction platform, we screened these for immunogenicity and developed an RGC-derived cDNA antigen library for use in ACT. Polyclonal T cells generated from dendritic cells pulsed with RGC RNA showed robust IFNγ and TNFα release in response to cognate antigen and selectively lysed tumor cells in vitro. Further, RGC T cells were significantly better than total tumor RNA (ttRNA) T cells at killing CD133-enriched tumor cells, indicating a preference of RGC T cells for stemness markers. In GBM-bearing mice, ACT using RGC T cells significantly improved survival (p = 0.0058), achieving outcomes comparable to ttRNA ACT, without overt neurotoxicity or systemic inflammation. In parallel human studies, RGC-derived antigens were tested against patient-derived GBM xenografts. Although RGC T cells targeted a smaller fraction of tumor cells compared to ttRNA T cells, they showed enriched recognition of GSC-like populations in 3D bioprinted immune-tumor assays. These findings support the utility of RGCs as a tumor-independent antigen source capable of driving anti-GBM responses. This renewable library enables ACT even in cases where tumor resection is not possible and provides a strategy for targeting treatment-resistant GSC subpopulations early in disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa6bde2a5f253652f6585d9d9a61d9726319ee48" target='_blank'>
              CNSC-02. BORN TO KILL: NEURODEVELOPMENTAL CELLS AS RENEWABLE TARGETS IN T CELL THERAPY FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            C. Love, John Figg, D. Jin, A. Reid, Illeana West, M. Englebart, C. Francis, Jeffery Drake, D. Mitchell, C. Flores
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Prostate cancer (PCa) is a leading cause of male cancer mortality, with metastasis and immune evasion posing major therapeutic challenges. DOCK3, a guanine nucleotide exchange factor implicated in cytoskeletal dynamics, is poorly characterized in PCa. This study investigates DOCK3’s role in PCa metastasis and tumor immune microenvironment (TIME) remodeling. Methods Multi-omics analyses integrated bulk RNA-seq from TCGA-PRAD (499 tumors/52 normals), scRNA-seq from GEO (45,325 cells), and genomic data. We performed: Differential expression analysis (DESeq2), Immune deconvolution (CIBERSORT,ssGSEA, xCell), WGCNA co-expression networks, Tumor mutational burden (TMB) assessment, Distant metastasis (M1 vs. M0) association studies, scRNA-seq clustering (Harmony/UMAP) and DE testing. Statistical significance thresholds: |log2FC|>1, padj<0.05. Results DOCK3 expression was found to be significantly elevated in metastatic (M1) tumors compared to primary (M0) tumors (p<0.05) and demonstrated a strong positive correlation with a higher tumor mutational burden (TMB) in metastatic samples (p<0.001). Cellular specificity analysis revealed that DOCK3 was exclusively and highly enriched within malignant epithelial and stromal cells, specifically in Cluster 6, where it exhibited a log2 fold-change of 9.13 (padj<1e-200) and was expressed in 54% of cells, compared to a negligible presence in all other clusters. In the tumor microenvironment, elevated DOCK3 expression was associated with a significant increase in cytotoxic immune infiltration, notably of CD8+ T and Natural Killer cells, a finding consistently supported by multiple computational algorithms (all p<0.05). Clinically, a high level of DOCK3 was significantly associated with metastatic status (p<0.01), whereas high expression of CDKN3 was correlated with advanced disease features, including higher Gleason scores (3-5) and T-stage (T2-T4) (p<0.01). Furthermore, significant differences in immune infiltration patterns were observed between clusters. Pathway enrichment analysis of genes co-expressed with DOCK3, identified through the WGCNA Green Module, indicated significant involvement in biological processes such as cytoskeletal reorganization, muscle contraction, and metabolic pathways (FDR<0.01). Conclusion DOCK3 drives PCa metastasis through cytoskeletal dynamics while paradoxically promoting an immunologically active microenvironment. Its tumor-specific expression and association with aggressive clinical features nominate DOCK3 as a novel biomarker for risk stratification and a promising therapeutic target for combinatorial immunotherapy in immunologically “cold” PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fbd5c9d8d606fe242b51bce0cd05f9f0b2729c1" target='_blank'>
              DOCK3 orchestrates metastasis and immune microenvironment in prostate cancer
              </a>
            </td>
          <td>
            Jiaxue Han, Ming Zhang, Haipeng Zhou, Qiao Xiong, Xin Zhong, Ping Tan
          </td>
          <td>2025-10-22</td>
          <td>Frontiers in Urology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Solid tumor immune suppression requires cooperation of tumor cells, local immune cells, peripheral circulating immune cells, and evolution of immune cell trajectories between peripheral and local environments. This study addresses a significant knowledge gap by characterizing peripheral and local immune environments in IDH Mutant (IDH-Mut) and IDH wildtype (IDH-WT) gliomas and defines novel immunological states with prognostic relevance across the glioma landscape.


METHODS
We analyzed local and peripheral immune phenotypes in a cohort of 18 (6 IDH-Mut and 12 IDH-WT) gliomas with distinct genetic characteristics using paired human tumor and peripheral blood mononuclear cells (PBMCs) with single-cell RNA-sequencing (scRNA-seq).


RESULTS
Our analyses revealed unique intratumoral and peripheral immune cellular ontogenies, including naïve CD4+ T cell enrichment in the IDH-Mut peripheral immune compartment, monocyte enrichment in IDH-WT glioma PBMCs, and emergence of a unique population of GZMH+ CD8+ T cells preferentially in the IDH-Mut microenvironment. Additionally, we found upregulation of TNF-α signaling and inflammatory response pathways in IDH-Mut-glioma-associated peripheral lymphoid cells versus IDH-WT tumors and identified a novel population of microglia-like cells in the peripheral blood of glioma patients with complement-interfacing characteristics. Applying intratumoral transcriptomic deconvolution via The Cancer Genome Atlas revealed genotype-independent, prognostic immune signatures across the malignant glioma landscape.


CONCLUSIONS
This study reveals variable expression of immune phenotypes in adult gliomas stratified by IDH status and characterizes immune compartment and genotype-dependent differences in the immunologic glioma landscape. These genotype-dependent, tumor and circulating immune ontogenies should guide future diagnostic and immunotherapeutic considerations in malignant glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a447b143602837a31ca2cf328c82cdb349e4b0c5" target='_blank'>
              Single-cell profiling of peripheral and local immune compartments reveal unique genotype-independent prognostic immune signatures across IDH-stratified glioma.
              </a>
            </td>
          <td>
            Jonathan H Sussman, A. Cillo, Sajeev Kohli, Carly Cardello, Jonathan Patterson, Ebrar Akca, Angelo Angione, David Moon, K. Krueger, A. Ghamari, Emily Xu, Jessica Xu, A. C. Tarbay, Alex Li, B. Budihal, Xiaoran Zhang, Omar Elghawy, Wojciech K Panek, P. Rajappa, Kai Tan, Dario A A Vignali, Tullia C. Bruno, N. Amankulor
          </td>
          <td>2025-09-20</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Glioblastoma is the most aggressive primary malignant brain tumor with dismal survival. Despite surgical resection and adjuvant therapy, the tumor almost always recurs. Tumor recurrence is enriched in treatment-resistant mesenchymal-like glioblastoma stem cells (GSCs), which represent a major obstacle to treatment. GSCs are resistant to many chemotherapies and ionizing radiation; however, recent evidence suggests that they are particularly sensitive to death by lipid peroxidation, a phenomenon known as ferroptosis. However, we found that not all GBM GSCs are sensitive to ferroptosis. Here, we propose an innovative therapeutic strategy centered on eradicating these ferroptosis-resistant GSCs by targeting their metabolic dependencies. We leveraged patient-derived GSCs as well as an ex vivo GBM slice culture model to test a new therapeutic strategy and obtained in-depth insights into GSCs and the tumor microenvironment using single-nucleus RNAseq and spatial transcriptomics. We discovered that ferroptosis sensitivity requires secreted factors from tumor microenvironment astrocytes, and have identified the mechanisms by which tumor-associated astrocytes can reverse ferroptosis resistance. This multi-faceted approach offers a promising and novel avenue for combating GBM recurrence and holds potential for treating other refractory tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62a89bb99a4c9a35ea866ec8e996dca3caa4e93" target='_blank'>
              TMIC-100. Astrocytes Orchestrate Sensitivity to Ferroptosis in Glioblastoma
              </a>
            </td>
          <td>
            Shengnan Xia, Maxwell Ma, Fahad Paryani, O. Al-Dalahmah
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 The advent of RAS inhibitors in the clinic offers the possibility of revolutionized care for patients with pancreatic ductal adenocarcinoma (PDAC). While RAS(ON) multi-selective inhibitors like the investigational agent daraxonrasib (RMC-6236) and the related preclinical tool compound RMC-7977 are designed to target the KRAS-mutant malignant epithelium, their effects on the surrounding tumor stroma remain underexplored. Prior studies using genetically engineered mouse models (GEMMs) have shown that aberrant oncogenic KRAS signaling is critical for the establishment and maintenance of the tumor microenvironment. In recent years, multiple CAF subtypes have been identified based on gene expression profiles, spatial distribution, and cellular origin. Myofibroblastic CAFs (myCAFs), which are located near the malignant epithelium, are marked by active Sonic Hedgehog (SHH) signaling, while inflammatory CAFs (iCAFs) and antigen-presenting CAFs (apCAFs) reside more peripherally. Here, we investigate how pan-RAS-GTP inhibition by RMC-7977 reshapes the CAF landscape in PDAC and its impact on the extracellular matrix (ECM). We profiled CAF subtype prevalence and transcriptional programs using single-cell RNA sequencing (scRNA-seq) of GEMM-derived tumors treated in vivo with RMC-7977 across multiple timepoints. These findings were corroborated by immunostaining of murine and patient-derived tumor explants treated ex vivo for 2 to 4 days. Tumor stiffness following RMC-7977 treatment was assessed using nanoindentation, and mechanistic studies were performed in vitro using reporter fibroblasts. Preliminary data reveal that short-term RMC-7977 treatment reduces myCAF abundance and proliferation, while iCAF and apCAF populations expand. Notably, supplementation with exogenous SHH restores the original myCAF-rich phenotype even in the presence of RMC-7977. Pan-RAS-GTP inhibition also remodels the ECM, resulting in decreased tumor stiffness. These preclinical findings suggest that systemic pan-RAS-GTP inhibition alters not only the malignant epithelium but also the broader PDAC architecture through effects on stromal components. Understanding these stromal adaptations will enable improved assessment of tumor behavior and the development of more effective combination therapies.



 Allison C. Hess, Lorenzo Tomassoni, Kevin Muñoz Forti, Hun Jin Jeong, Deanna J. Besart, Tanner C. Dalton, Carmine F. Palermo, Stephen A. Sastra, Chang H. Lee, Andrea Califano, Simon Schwörer, Kenneth P. Olive, Marie C. Hasselluhn. RAS(ON) multi-selective inhibition remodels cancer-associated fibroblast subtypes and extracellular matrix in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B098.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80ccb92666321bba79e7b5a3b723ec7cf1794a5" target='_blank'>
              Abstract B098: RAS(ON) multi-selective inhibition remodels cancer-associated fibroblast subtypes and extracellular matrix in pancreatic cancer
              </a>
            </td>
          <td>
            Allison C. Hess, Lorenzo Tomassoni, Kevin Muñoz Forti, Hun Jin Jeong, Deanna J. Besart, Tanner C. Dalton, Carmine F. Palermo, Stephen A. Sastra, Chang H. Lee, Andrea Califano, Simon Schwörer, K. Olive, Marie C. Hasselluhn
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma is one of the deadliest solid tumors, with a two-year survival rate below 5%. Progress in developing effective treatments has been limited, largely due to the absence of physiologically relevant disease models that accurately recapitulate the tumor microenvironment. Such models are critical for understanding disease mechanisms and evaluating therapeutic strategies. Organoids have recently emerged as promising platforms for modeling various diseases, including pancreatic cancer, as they retain the structural and functional features of their tissue of origin. Moreover, CRISPR-Cas9 gene editing allows precise introduction of cancer-associated mutations, providing a powerful approach for generating genetically defined models.



 We established normal pancreatic organoids from human pancreatic tissue samples and validated their identity through qPCR and histological analyses, confirming their resemblance to native pancreatic epithelium. These organoids demonstrated stable growth over several passages and retained viability after cryopreservation and thawing. To model pancreatic cancer, we used CRISPR-Cas9 technology to introduce TP53 mutations, a key tumor suppressor gene altered in over 70% of pancreatic ductal adenocarcinoma (PDAC) cases. Following Nutlin-3a selection, successful TP53 knockout was confirmed by Sanger sequencing. The engineered TP53-knockout (KO) organoids were expanded and characterized to assess cancer-like phenotypes.



 Human-derived pancreatic organoids retained their structural and functional identity across passages and after thawing. Following TP53 gene editing, the TP53 KO organoids exhibited increased proliferation and morphological changes consistent with early tumorigenesis. Importantly, qPCR analysis revealed significant upregulation of cancer-associated markers, including MUC5AC, KRT19, and CEACAM5, in TP53-mutated organoids compared to controls, further supporting their malignant transformation. These findings represent a critical step toward building a genetically defined pancreatic cancer model that closely reflects human disease.



 We successfully generated a pancreatic cancer organoid model by editing TP53 in normal human pancreatic organoids. The TP53 KO organoids showed enhanced growth, morphological transformation, and elevated expression of key cancer markers, recapitulating early features of pancreatic cancer. This model mimics critical aspects of pancreatic cancer biology and provides a robust platform for studying tumor initiation, progression, and drug response. Unlike conventional animal models or stem-cell-derived systems, our approach offers a human-based, physiologically relevant, and reproducible system for pancreatic cancer research and therapeutic screening.



 Eden Demere Amare, Soraya SALAS-SILVA, Ji Hyun Shin, Dongho Choi. CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59cdfa7b3d8511b93672f4beff88185a825b61c7" target='_blank'>
              Abstract A013: CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation
              </a>
            </td>
          <td>
            Eden Demere Amare, Soraya SALAS-SILVA, Jihyun Shin, Dongho Choi
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a235a990b95cad3b6488f79eeb25d061d834e198" target='_blank'>
              Triangulating cross-carcinoma GWAS delineates shared programs and disease-specific drivers
              </a>
            </td>
          <td>
            Y.-F. Huang, K.-L. Huang
          </td>
          <td>2025-09-22</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Malignant gliomas follow two distinct natural histories: de novo high grade tumors such as glioblastoma, or lower grade tumors with a propensity to transform into high grade disease. Despite differences in tumor genotype, both entities converge on a common histologically aggressive phenotype, and the basis for this progression is unknown. Glioma associated macrophages (GAM) have been implicated in this process, however GAMs are ontologically and transcriptionally diverse, rendering the identification and isolation of pathogenic subpopulations challenging.



 Since macrophage contextual gene programs are orchestrated by transcription factors acting on cis-acting promoters and enhancers in gene regulatory networks (GRN), we hypothesized that functional subpopulations of GAMs can be resolved through GRN inference. Method IDH mutant/wildtype gliomas and control normal brain samples were collected for single cell RNA +ATAC sequencing to derive GRNs. Transcription factor network analysis identified cell surface markers to isolate subpopulations for functional assays, with tissue, in vitro and in vivo correlates.



 A subpopulation of human GAMs can be defined by a GRN centered around the Activator Protein-1 transcription factor FOSL2 preferentially enriched in high grade tumors. Using this GRN we nominate ANXA1 and HMOX1 as surrogate cell surface markers for activation, thus permitting prospective isolation and functional validation in human GAMs. These cells, termed malignancy associated GAMs (mGAMs) are pro-invasive, pro-angiogenic, pro-proliferative, possess intact antigen presentation but skew T-cells towards a CD4+FOXP3+ phenotype under hypoxia. Ontologically, mGAMs share somatic mitochondrial mutations with peripheral blood monocytes, and their presence correlates with high grade disease irrespective of underlying tumor mutation status. Furthermore, spatio-temporally mGAMs occupy distinct metabolic niches; mGAMs directly induce proliferation and mesenchymal transition of low grade glioma cells and accelerate tumor growth in vivo upon co-culture. Finally mGAMs are preferentially enriched in patients with newly transformed regions in human gliomas.



 mGAMs play a pivotal role in glioma progression and represent a potential therapeutic target in human high-grade glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7408d85300870de072bd9687aac588be6dfef1cd" target='_blank'>
              TMIC-37. A pathogenic subpopulation of human glioma associated macrophages linked to glioma progression
              </a>
            </td>
          <td>
            Kenny Kwok Hei Yu, Z. Abou-Mrad, Kristof Törkenczy, Isabell Schulze, Jennifer Gantchev, Gerard Baquer, Kelsey Hopland, Evan D. Bander, Umberto Tosi, Cameron Brennan, Nelson Moss, Pierre-Jacques Hamard, R. Koche, Caleb A. Lareau, Nathalie Y R Agar, T. Merghoub, Viviane Tabar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e71e7e279842a698e66d7ba3f958e4bde7276e0" target='_blank'>
              Supervised Factorization to Associate Spatial Transcriptomics with Complementary Molecular Readouts
              </a>
            </td>
          <td>
            Faria Binta Awal, R. Pautler, Md. Abul Hassan Samee, M. S. Rahman
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Hypoxia is a recognized driver of transcriptional heterogeneity in high-grade glioblastoma (GBM), yet its functional significance in low-grade gliomas (LGGs) remains poorly understood. We hypothesized that hypoxia orchestrates a reversible quiescence-activation axis in LGGs via EGFR-dependent transcriptional reprogramming, thereby influencing tumor progression.



 We conducted integrative analyses of single-cell RNA sequencing (n=7 patients) and spatial transcriptomics (n=8 samples) across molecularly defined IDH-mutant LGGs to delineate hypoxia-associated transcriptional states. EGFR mutation, and clinical correlations with overall survival, were assessed in The Cancer Genome Atlas (TCGA) bulk RNA-seq cohorts. To functionally validate microenvironmental contributions, we established innovative co-culture models of LGG cells with hypoxia-conditioned primary microglia and astrocytes.



 We identified a hypoxia-associated, LGG-specific ribosomal-enriched (RE) transcriptional program characterizing a non-cycling subpopulation enriched in quiescence and activation stages. Quiescent RE cells exhibited glioma stem cell (GSC)-like features, with EGFR signaling dependency for transition to an activated phenotype. Spatial transcriptomics revealed distinct co-localization of hypoxia, RE program expression, and quiescent states, whereas EGFR-activated cells formed discrete micro-niches. Hypoxia-conditioned microglia significantly enhanced RE program activation and EGFR signaling, establishing a functional microenvironmental driver of phenotypic plasticity.



 Our findings delineate a novel hypoxia-EGFR signaling axis sustaining a quiescent yet activation-capable subpopulation in IDH-mutant LGGs. These insights nominate hypoxia-modulated EGFR pathways as potential therapeutic targets to disrupt tumor cell plasticity. Future studies will interrogate vulnerabilities of RE-programmed LGG cells under hypoxic conditions to inform next-generation therapeutic development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/748507ebfe3412d3babe3bb777f9dae226192bc1" target='_blank'>
              P02.22.A HYPOXIA-DRIVEN EGFR SIGNALING PROMOTES A QUIESCENT-ACTIVATION AXIS IN IDH-MUTANT LOW-GRADE GLIOMAS
              </a>
            </td>
          <td>
            D. Das, V. Jariwala, A. Pandele, A. Mohammed, T. Das, N. Ghariwala, C. Krolla, S. Arosh, N. Hossain, I. Hosen, S. Smith, J. Beck, J. Grauvogel, I. Vasilikos, R. Roelz, R. Rahman, M. Börries, K. Joseph, V. Ravi, G. Andrieux, S. Chakraborty
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal malignancy characterized by a profoundly immunosuppressive tumor microenvironment. Within PDAC, tumor-associated macrophages (TAMs) are the most abundant immune cell population. Here, TAMs orchestrate the characteristic immunosuppression of the pancreatic tumor microenvironment, drive disease progression, and mediate therapeutic resistance in PDAC. Accordingly, successful development of approaches that reprogram or eliminate these pro-tumor immune populations have potential to improve patient outcomes. One of the most challenging aspects of developing approaches to modulate TAMs programming is the ability to perform mechanistic experiments in a system the recapitulates the complex pancreatic tumor microenvironment. We and others have leveraged primary bone marrow-derived macrophages (BMDMs) polarized by PDAC conditioned media as a discovery platform that has yielded translational findings. However, this system fails to account for the dynamic and reciprocal interactions between myeloid and cancer cells that occur within tumors. Accordingly, we sought to address if this model could be built into progressively more complex systems to assess if these better recapitulate TAMs using a systems biology approach. Using transcriptomic profiling, we compared different in vitro strategies, from direct co-cultures to conditioned media polarization, against TAMs harvested from orthotopic PDAC tumors. Our preliminary data show that allowing PDAC cells to engage in direct cellular contact and sustained paracrine signaling most closely resemble the transcriptomic profile of bona fide TAMs. Our further efforts to use these PDAC-macrophage culture models will be key to identifying relevant signaling pathways that can be mechanistically examined prior to more cumbersome in vivo validation. Collectively, our data can be used to facilitate more accurate and insightful investigations into PDAC immunology and the development of therapeutic targeting strategies.



 Cavina K. Lee, Cecily Anaraki, Austin D. Silva, Tiffany Lai, Hillary Tran, Arjun Sapru. Dissecting myeloid - cancer communication networks in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B104.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad71226813c7f5d3d378a55b84523f40aab08ce1" target='_blank'>
              Abstract B104: Dissecting myeloid - cancer communication networks in pancreatic cancer
              </a>
            </td>
          <td>
            Cavina K. Lee, Cecily Anaraki, Austin D. Silva, Tiffany Lai, Hillary Tran, Arjun Sapru
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer (PC) remains a leading cause of cancer-related mortality in men, with recurrence contributing significantly to poor outcomes. Its molecular heterogeneity complicates effective risk stratification. We evaluated Sig27, a novel 27-gene panel, across 13 bulk RNA-seq datasets (n=3,133 tumors) and 6 single-cell RNA-seq (scRNA-seq) datasets (n=53 patients). Sig27 expression was elevated in PC compared to normal tissue and further increased in high-grade Gleason tumors, node-positive, and recurrent tumors. Sig27 demonstrated recurrence prediction comparable to Oncotype DX, with strong enrichment in immune regulatory pathways. To further investigate immune associations, we developed SigIC, a 22-gene immune checkpoint panel. Sig27 showed strong correlations with SigIC and individual immune checkpoints (e.g., HAVCR2, CD96, TIGIT) in both primary and metastatic PC. In scRNA-seq data, Sig27 was enriched in tumor-associated monocytes/macrophages (TAMs) and endothelial cells. We identified five key Sig27 genes-TFEC, FPR3, NOD2, LAMP3, and MCTP1-and constructed Sig27IMG, a multigene panel formed by these five genes, and demonstrated their robust correlations with immune checkpoints and their strong enrichment in TAMs and endothelial cells. Sig27IMG strongly predicted PC recurrence and was dominantly expressed in TAMs, dendritic cells, and endothelial cells across 26 cancer types (n=386 patients) in scRNA-seq studies and 17 cancer types (n≈5,672 patients) in bulk RNA-seq investigations. Notably, Sig27IMG stratified patients with a poor prognosis risk in these 17 cancer types. In summary, Sig27 and its derivative panel, Sig27IMG, offer a robust assessment of PC recurrence, highlighting immunosuppressive features mediated by TAMs, dendritic cells, and endothelial cells across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6567137b89da0ce653868c768b7cb4430fa5d323" target='_blank'>
              Sig27 stratifies prostate cancer recurrence via assessing tumor's immunosuppressive properties.
              </a>
            </td>
          <td>
            Sandra Vega Neira, Ying Dong, Tao Zhang, Damu Tang
          </td>
          <td>2025-11-04</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [10],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>