<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-10-27 09:32:50 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Kaposi’s sarcoma (KS) is a highly inflammatory, angiogenic tumor driven by Kaposi’s sarcoma-associated herpesvirus (KSHV), yet the origins of tumor cells and mechanisms of progression remain unclear. Here, we present the first spatial single-cell atlas of KS, profiling 256 samples across patch, plaque, and nodular lesions and normal controls. We identify CD34⁺ progenitor lymphatic endothelial cells (LECs) as the primary targets of KSHV, whose clonal expansion drives tumor growth. KSHV infection induces widespread cellular reprogramming across the tumor microenvironment, including LECs, vascular endothelial cells, fibroblasts, and macrophages, generating hybrid phenotypes that support angiogenesis, inflammation, and immune modulation. KSHV⁺ macrophages are enriched in tumor-proximal niches, further promoting a proangiogenic, immunosuppressive environment. Spatial analysis reveals evolving tumor-associated niches, with a core-to-periphery gradient correlating with infection, immune modulation, and cellular remodeling. We identify disease progression predictive signatures, offering mechanistic insights into KS pathogenesis and potential new therapeutic strategies by reprogramming the tumor microenvironment. Highlights ➣ CD34+ progenitor lymphatic endothelial cells are the primary KSHV target cells, with their clonal expansion driving Kaposi’s sarcoma growth. ➣ KSHV infection reprograms broad cell types into hybrid identities, and drives tumor-specific niche and vascular remodeling, endothelial plasticity, and immune modulation. ➣ KSHV-reprogrammed macrophages drive inflammation, angiogenesis, and immune modulation. ➣ Spatially resolved molecular and cellular signatures predict Kaposi’s sarcoma progression, offering novel therapeutic strategies targeting the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/835b859927d90c9b0c5454b990e0f2840ed6f867" target='_blank'>
              Spatial Single-Cell Atlas Reveals KSHV-Driven Broad Cellular Reprogramming, Progenitor Expansion, Immune and Vascular Remodeling in Kaposi’s Sarcoma
              </a>
            </td>
          <td>
            Wen Meng, Arun Das, Harsh Sinha, Rana Naous, P. Bracci, Mike McGrath, Yufei Huang, Shou-Jiang Gao
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728733c32d8e17fbc34a5070c8b658170e0a8a22" target='_blank'>
              Identifying tissue states by spatial protein patterns related to chemotherapy response in triple-negative breast cancer
              </a>
            </td>
          <td>
            L.M. Luque, A. Khan, G. Torrisi, Tessa D. Green, D. Hardman, C. Owczarek, T. A. Phillips, D. Marks, M. Parsons, C. Sander, L. Schumacher
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Aggressive cancers, such as pancreatic ductal adenocarcinoma (PDAC), are often characterized by a complex and desmoplastic tumor microenvironment, a stroma rich supportive connective tissue composed primarily of extracellular matrix (ECM) and non-cancerous cells. Desmoplasia, a dense deposition of stroma, is a major reason for therapy resistance, acting both as a physical barrier that interferes with drug penetration and as a supportive niche that protects cancer cells through diverse mechanisms. Precise understanding of spatial cell interactions in stroma-rich tumors is essential for optimizing therapeutic responses. It enables detailed mapping of stromal-tumor interfaces, comprehensive cell phenotyping, and insights into changes in tissue architecture, improving assessment of drug responses. Recent advances in multiplexed immunofluorescence imaging have enabled the acquisition of large batches of whole-slide tumor images, but scalable and reproducible methods to analyze the spatial distribution of cell states relative to stromal regions remain limited. To address this gap, we developed an open-source computational pipeline that integrates QuPath, StarDist, and custom Python scripts to quantify biomarker expression at a single- and sub-cellular resolution across entire tumor sections. Our workflow includes: (i) automated nuclei segmentation using StarDist, (ii) machine learning-based cell classification using multiplexed marker expression, (iii) modeling of stromal regions based on fibronectin staining, (iv) sensitivity analyses on classification thresholds to ensure robustness across heterogeneous datasets, and (v) distance-based quantification of the proximity of each cell to the stromal border. To improve consistency across slides with variable staining intensities, we introduce a statistical strategy that translates classification thresholds by propagating a chosen reference percentile across the distribution of marker-related cell measurement in each image. We apply this approach to quantify spatial patterns of distribution of the phosphorylated form of the N-Myc downregulated gene 1 (NDRG1), a novel DNA repair protein that conveys signals from the ECM to the nucleus to maintain replication fork homeostasis, and a known cell proliferation marker Ki67 in fibronectin-defined stromal regions in PDAC xenografts. The pipeline is applicable for the analysis of markers of interest in stroma-rich tissues and is publicly available.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b6327fb55ba3776af18a8a1265a4ec0c3c69a0b" target='_blank'>
              An image analysis pipeline to quantify the spatial distribution of cell markers in stroma-rich tumors
              </a>
            </td>
          <td>
            Antoine A. Ruzette, Nina Kozlova, Kayla A. Cruz, Taru Muranen, S. Nørrelykke
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) presents diverse histomorphological features within the tumor microenvironment (TME) that influence prognosis and response to immunotherapy. Leveraging contrastive learning, we developed an unbiased atlas of cell neighborhoods to systematically explore the LUAD microenvironment at the cellular scale and investigate how these neighborhoods combine to form histologic patterns. This multiscale approach enables a comprehensive understanding of both cell-specific interactions and broader histologic architecture in LUAD. Our analysis identified distinct histomorphological phenotype clusters of cellular neighborhoods (cn-HPCs) with prognostic significance. For example, cn-HPC 0 was associated with immune activation and favorable survival, while cn-HPC 23 was enriched in necrotic, immune-excluded regions and aligned with poorer outcomes. Through associations with immunophenotypes, co-expressed gene modules, and pathway enrichment, we found that cn-HPCs reflect underlying processes of immune modulation, cellular growth, and inflammation, offering insights into the functional landscape of LUAD. In an independent LUAD immunotherapy cohort, we found that cn-HPC 23 showed promising predictive value for treatment response. These findings suggest that fine-grained spatial profiling of the TME may offer complementary value for identifying patients more likely to benefit from immunotherapy. Taken together, our study highlights the potential of cn-HPCs to bridge histopathological patterns and clinical outcomes, supporting their relevance for prognosis and treatment stratification in LUAD. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17117-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732a50286d99e4889f11b9dfeef8dfb5de086217" target='_blank'>
              Contrastive learning uncovers cellular interactions and morphologies in the tumor microenvironment of lung adenocarcinoma linked to immunotherapy response
              </a>
            </td>
          <td>
            H. Le, N. Coudray, A. Yeaton, Sitharam Ramaswami, A. Karimkhan, Wei-Yi Cheng, James Cai, Tai-Hsien Ou Yang, S. Punekar, Luis Chiriboga, Vamsidhar Velcheti, Kwok-Kin Wong, D. Sterman, Andre L. Moreira, Harvey I. Pass, A. Tsirigos
          </td>
          <td>2025-10-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/235c8c18e964369ee970e129fd3fe5bd3b1a1aa6" target='_blank'>
              In silico reconstruction of primary and metastatic tumor architecture using GIS-augmented spatial transcriptomics
              </a>
            </td>
          <td>
            Jin Young Yoo, Sabrina Akter, Qianying Zuo, Amir Kazemi, Wenjun Wu, Mahima Goel, Audrey Lam, Debapriya Dutta, Betsy Barnick, Melike Pekmezci, Maria Grosse Perdekamp, Aiman Soliman, Zeynep Madak-Erdogan
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Tumors acquire a set of functional capabilities, known as the Hallmarks of Cancer, to thrive and grow. While extensive research has revealed how gene and protein expression networks contribute to these traits, the relationships between tumor cell states, spatial organization of immune and stromal cells in the tumor microenvironment, and tumor genetics, remains poorly understood. To address this, we constructed a cohort of primary tumor tissues from 117 pancreatic ductal adenocarcinoma (PDAC) patients. Using Multiplexed Ion-Beam Imaging (MIBI) with a 40-antibody panel, we mapped the spatial organization of major cell types and their phenotypes. We also conducted targeted genomic and bulk proteomic analyses to explore how spatial organization patterns align with tumor genetics and proteome profiles. We identified phenotypic subtypes of tumor epithelium that shape distinct microenvironmental niches: (i) Glycolytic tumors expressing proteins such as GLUT1, LDHA, MCT1 and HIF1α reside in hypoxic niches. They are encased by dense fibrotic stroma and lacking vasculature and immune cell infiltration. (ii) Tumor and metaplastic epithelium that exhibit an oxidative proteome profile, are highly enriched for ECAD and PD-L1 expression, and are situated near pancreatic epithelium and vasculature, with significant immune infiltration. (iii) Myeloid-interacting tumors expressing CXCL5, heavily infiltrated with neutrophils and characterized by low-collagen and high FAP positive fibroblasts. Notably, neutrophils accumulate within the lumen of tumor ducts where they exhibit increased apoptosis, DNA damage, and netosis. These highly neutrophil-infiltrated tumors, associate with poor disease outcome. The genetic landscape of PDAC influences these spatial and phenotypic patterns. Patients with TGFβ/SMAD pathway mutations, demonstrate an increased proportion of basal tumor cells accompanied by reduced lymphocytic infiltration. Similarly, distinct TP53 variants shape immune-cell infiltration: patients harboring missense mutations show increased lymphocytic infiltration compared to those with truncating mutations. These findings underscore the strong link between tumor genetics, phenotypic heterogeneity and cellular architecture of the PDAC microenvironment, offering new insights into collective tumor behavior.



 Ofer Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, Inti Zlobec, Tamar Geiger, Leeat Keren. Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B123.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb5d6211d2e33680561c9558492cf85ff16b33" target='_blank'>
              Abstract B123: Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization
              </a>
            </td>
          <td>
            O. Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, I. Zlobec, Tamar Geiger, Leeat Keren
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9ca9b266d6e3d48dee0fe7d9f0b768bc883a83" target='_blank'>
              Functional interrogation uncovers a critical role for a high-plasticity cell state in lung adenocarcinoma
              </a>
            </td>
          <td>
            , , Jonathan Rub, Klavdija Krause, Emma Brown, Gary Guzman, Zeda Zhang, Hannah C Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Cancer cell plasticity, defined as the ability of tumor cells to reversibly adopt distinct functional states, plays a central role in tumor heterogeneity, therapy resistance, and disease relapse. This process enables cells to enter stem-like, dormant, or drug-tolerant persister states in response to treatment or environmental stress without undergoing genetic changes. Such reversible transitions complicate and limit current treatments. Conventional cancer models often fail to capture the complexities of these adaptive states. In contrast, patient-derived tumor organoids (PDOs), which retain the cellular diversity and structure of primary tumors, provide a unique system for investigating plasticity. This review describes how PDOs can model cellular plasticity, such as the emergence of drug-tolerant persister cells and the interconversion between cancer stem cell states across multiple tumor types. We particularly focused on colorectal cancer organoids, for which research on the mechanism of plasticity is the most advanced. Combined with single-cell analysis, lineage tracing, and functional assays, PDOs can help identify the molecular pathways that control plasticity. Understanding these mechanisms is important for developing therapies to prevent treatment failure and control disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd4e7e497e61cac197e5ff80d3e09c68cd18937b" target='_blank'>
              Patient-Derived Tumor Organoids to Model Cancer Cell Plasticity and Overcome Therapeutic Resistance
              </a>
            </td>
          <td>
            R. Coppo, Masahiro Inoue
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract The heterogeneous nature of cell populations in human tumors is a major contributor to tumor evolution, including and perhaps most importantly in response to treatment. Here, we review current knowledge on tumor heterogeneity and cell state plasticity in small cell lung cancer (SCLC), a fast growing and highly metastatic form of lung cancer which develops rapid resistance to therapy. There is a pressing need to expand treatment options for patients with SCLC, which requires a better understanding of the mechanisms by which this disease is able to rapidly grow and evolve in response to therapy. Our current understanding points to epigenetic rather than genetic factors in defining major aspects of inter- and intra-tumoral heterogeneity in SCLC. SCLC is overall considered to be a neuroendocrine (NE) cancer type but SCLC tumors harbor a wide diversity of cancer cell states, including both NE and non-neuroendocrine (non-NE) states, defined by their mutually exclusive expression of a set of transcription factors such as ASCL1, NEUROD1, and POU2F3. The immune microenvironments of SCLC tumors also contain a great deal of heterogeneity. Here, we discuss the different SCLC cell states associated with their defining transcription factors, as well as the epigenetic mechanisms regulating the ability of SCLC cells to switch from one state to another. We further discuss how the composition of SCLC tumors and the surrounding immune cells may affect the response to chemotherapy and immunotherapy. Being able to control plasticity and heterogeneity in SCLC may in the future offer unique opportunities to improve treatment efficacy in this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f9c75cb099eaa87426731186cf92ba20e24f46a" target='_blank'>
              Tumor Heterogeneity and Plasticity in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Gina Duronio, Julien Sage
          </td>
          <td>2025-10-01</td>
          <td>Lung Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The spatial organization and composition of the tumor-immune microenvironment (TME) play a critical role in shaping the progression of many solid cancers, but the organization of the TME in primary prostate cancer (PCa) remains poorly characterized. We therefore profiled the abundance and spatial distributions of major cell types involved in adaptive immunity in 29 radical prostatectomy specimens stratified into high (HGG; n=14) and low Gleason-grade (LGG; n=15). Compared to LGG, HGG PCa exhibited significantly greater B and T cell infiltration with many immune cells organized into clusters, some of which resembled tertiary lymphoid structures (TLSs). In HGG tumors, these clusters were dense, symmetric, rich in PD-1+ T cells, and frequently proximate to the tumor compartment. LGG clusters were less well organized, and T cell depleted. Thus, a subset of high-grade PCa harbor organized immune clusters that may play a role in tumor control and contain therapeutically targetable T and B cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e5e762dad0168906d48c0967522a47db6d0ac" target='_blank'>
              Spatially organized lymphocytic microenvironments in high grade primary prostate tumors
              </a>
            </td>
          <td>
            Ali Amiryousefi, Jeremiah Wala, Jia-Ren Lin, Brian Labadie, A. Atmakuri, Z. Maliga, Eamon Toye, K. Chaudagar, Madeleine S. Torcasso, Shannon Coy, G. Fanelli, Brigette Kobs, F. Socciarelli, Andréanne Gagné, Eliezer Van Allen, Akash Patnaik, P. Sorger
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Spontaneously transformed mouse ovarian surface epithelial cell line ID8 generates a syngeneic mouse model, recapitulating the primary and metastatic stages of the malignancy hence, frequently used in pre-clinical evaluation. Due to inherent phenotypic plasticity, primary and metastatic tumors show contrasting phenotypic features dictated by the tissue niche. To address the tumor heterogeneity within the model we stratified it into a specific molecular subtype, which would provide an accurate setting for interpretation of experimental data. Stratification was performed on ID8 cell line and ID8 generated xenografts based on immunostaining, live cell imaging, histopathology, transcriptomics and proteomics. Tumor microenvironment (TME) investigation for each xenograft was done through flow cytometry analysis. ID8 cells present epithelial/mesenchymal (E/M) phenotype. Phenotypic plasticity retention in them causes class switching of ID8 cells in response to niche alteration, from E/M (in culture) to intermediate epithelial (iE) in orthotopic tumors while intermediate mesenchymal (iM) in subcutaneous tumors. Metastasis of ID8 primary tumor resulted into differentiated mesenchymal phenotype in ascites while differentiated epithelial phenotype in secondary intestinal tumors. Immune profiling of primary and secondary xenografts revealed differences in immune repertoire of tumors, highlighting subcutaneous tumor model to be inappropriate as compared to orthotopic model. Local tissue niche modulates the phenotype of tumors generated, transitioning through intermediate states towards a final commitment to either an epithelial or mesenchymal state that recapitulates the reported class switching in patient tumors. During disease progression, such heterogeneity of tumor cells at different niche(s) and their cross-talks with discrete TME at each site, will pose challenges for effective therapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18264-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20023e2da4c4317c3adb1e410526c772109c27da" target='_blank'>
              ID8 cells manifest phenotypic plasticity and molecular heterogeneity of high-grade serous ovarian cancer in response to the local tissue niche
              </a>
            </td>
          <td>
            Ritika Gupta, S. Shravan, Swapna Balaji Agwane, R. M. Bankar, S. Bapat
          </td>
          <td>2025-10-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Quantitative investigation of how the spatial organization of cells within the tumor microenvironment associates with disease progression, patient outcomes, and that cell’s phenotypic state remains a key challenge in cancer biology. High-dimensional multiplexed imaging offers an opportunity to explore these relationships at single-cell resolution. Methods We developed a computational pipeline to quantify and analyze the neighborhood profiles of individual cells in multiplexed immunofluorescence images. The pipeline characterizes spatial co-localization patterns within the tumor microenvironment and applies interpretable supervised machine learning models, specifically orthogonal partial least squares analysis (OPLS), to identify spatial relationships predictive of cell states and clinical phenotypes. Results We applied this framework to a previously published non-small cell lung cancer (NSCLC) cohort across four applications. At the cellular level, we identified neighborhood features associated with lymphocyte activation states. At the tumor-immune interface, we demonstrated that the immune cell composition surrounding major histocompatibility complex class I-expressing (MHC I+) tumor cells could distinguish adenocarcinoma from squamous cell carcinoma. At the patient level, spatial features predicted tumor grade. Discussion By integrating cell-segmented imaging data with interpretable modeling, our pipeline reveals key spatial determinants of tumor biology. These findings generate testable mechanistic hypotheses about intercellular interactions and support the development of spatially informed prognostic and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a735226da4130f5ff72da463b9cc63eeddebebdb" target='_blank'>
              A flexible systems analysis pipeline for elucidating spatial relationships in the tumor microenvironment linked with cellular phenotypes and patient-level features
              </a>
            </td>
          <td>
            Gabriel F. Hanson, Kate A. Goundry, Remziye E. Wessel, Michael G Brown, Timothy N. J. Bullock, Sepideh Dolatshahi
          </td>
          <td>2025-09-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has emerged as an advanced biological technology capable of resolving the complexity of cancer landscapes at single-cell resolution. Spatial transcriptomics(ST), as an innovative complementary approach, effectively compensates for the lack of spatial information inherent in scRNA-seq data. This review explores the rapidly evolving integration of scRNA-seq and ST and their transformative role in deciphering the tumor microenvironment (TME). We highlight how these technologies jointly uncover cellular heterogeneity, stromal-immune interactions, and spatial niches driving tumor progression and therapy resistance. Moving beyond previous reviews, we emphasize emerging computational strategies for data integration—including deconvolution and mapping approaches—and evaluate their applications in characterizing immune evasion, fibroblast diversity, and cell-cell communication networks. Ultimately, this review provides a forward-looking perspective on how spatial multi-omics are poised to advance precision oncology through spatially-informed biomarkers and diagnostic tools. We conclude that the full clinical potential of these technologies relies on closing the gap between analytical innovation and robust clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84801b1d17cdb40a9473ba25f6bac709b5c4c887" target='_blank'>
              Single-cell and spatial transcriptomics integration: new frontiers in tumor microenvironment and cellular communication
              </a>
            </td>
          <td>
            Wenxin Shi, Zhiqiang Zhang, Xiaotong Xu, Yanpeng Tian, Li Feng, Xianghua Huang, Yanfang Du, Zhongkang Li
          </td>
          <td>2025-10-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="The heterogeneity and dynamic evolution of the tumor microenvironment (TME) are key drivers of tumor progression, immune evasion, and therapeutic resistance. Traditional transcriptomic approaches are limited by their reliance on population-averaged data and the lack of spatial context, making it challenging to fully dissect the complex interactions among tumor cells, immune infiltrates, and stromal components within the TME. In recent years, the synergistic advancement of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has revolutionized our ability to construct high-resolution cellular atlases and delineate spatially organized functional modules within the TME. These technologies have enabled the identification of novel cell subtypes, cell-cell communication pathways, and spatial niches associated with disease progression and therapy response. This review systematically compiles recent technological breakthroughs in the integration of single-cell and spatial multi-omics, highlights key discoveries in TME research, and discusses their transformative impact on the development of precision oncology. Additionally, it addresses current technical limitations and explores emerging strategies and future directions for advancing this rapidly evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90a0c36ae523e1a4fc23b5c8bc8013f692b5b483" target='_blank'>
              Refining Tumor Microenvironment Insights via Single-Cell and Spatial Transcriptomics for Precision Medicine
              </a>
            </td>
          <td>
            Xianbing Wang
          </td>
          <td>2025-09-05</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c74617e8894674076d6153fb81e22ba9cb23b96" target='_blank'>
              Comparison of Human Melanoma Single Cell Profiles to Evolutionary Medicine Model Xiphophorus Provides Insights in Disease Control
              </a>
            </td>
          <td>
            Yanting Xing, Rachel Carrol, X. Maggs, Wes C Warren, Manfred Schartl, Yuan Lu
          </td>
          <td>2025-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Adenoid cystic carcinoma of the salivary gland (SGACC) is a highly aggressive malignancy characterized by poor patient survival outcomes. While several studies have analyzed the transcriptome of the salivary gland at the bulk and single-cell level, no spatial transcriptomic analyses of this tissue have been published. Most of the existing publications on SGACC have predominantly relied on bulk and single cell RNA sequencing approaches, which do not resolve the spatially localized transcriptional heterogeneity nor have the resolution for defining molecular markers within tumor subpopulations. SGACC is clinically notable for the presence of multiple tumor clones, distinct spatial phenotypes, and its indolent yet invasive nature coupled with a high propensity for distant metastasis. These features may reflect co-expression of tumor-associated markers across diverse cellular niches, and a resultant biological complexity which causes standard treatment such as surgical resection, radiation therapy, and chemotherapy to be largely ineffective in significantly improving long-term survival, and highlights the need for more precise, targeted therapeutic strategies. Herein, we analyzed single cell (n = 4) and high-resolution spatial transcriptomics samples (n = 5) to characterize cancer cell populations in MYB- and non-MYB-expressing cell states, delineated gene expression signatures, and identified critical molecular interactions specific to SGACC. We used Visum HD to obtain spatial transcriptomics data at 2µm squared high resolution. This allowed a multi-omics approach comprising single cell and spatial transcriptomic methods to enable the discovery of novel transcriptional signatures and microenvironmental features not captured by conventional methods. Spatial mapping revealed marked cellular heterogeneity and demonstrated how tissue environments influence cellular transcriptomics. To tumor heterogeneity, we focused on tumorigenic cell populations, profiled plasma and T cell enrichment within the tumor microenvironment and identified key pathways and transcriptional drivers including the MYB-NFIB fusion underlying the tumor cluster formation. Our findings indicate an upregulation of genes involved in extracellular matrix remodeling, autophagy, and reactive stromal cell populations. We further found evidence of partial epithelial-mesenchymal transition (P-EMT) programming within MYB-expressing tumor clusters. Pathway analysis revealed that mutations in the spatial query sample prominently affect the PI3K-AKT and IL-17 signaling pathways, together with a downregulation of canonical Wnt signaling in some regions of the tissue architecture adjacent to immune cells. Collectively, these results underscore the complex regulatory landscape of SGACC and offer insights into its cellular dynamics and possible therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d936540aedb35ad533bb62a60043e3d08a9252f" target='_blank'>
              Gene-Expression Programs in Salivary Gland Adenoid Cystic Carcinoma Analyzed Using Single-Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ifeoma Ebinumoliseh, Gopikrishnan Bijukumar, Kendall Hoff, K. Brayer, Elaine L. Bearer, Scott A. Ness, Jeremy S. Edwards
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is a molecularly diverse form of lung cancer characterized by distinct oncogenic driver mutations that influence both tumour biology and clinical outcomes. Understanding the interplay between these oncogenic drivers and the tumour microenvironment (TME) is crucial for improving therapeutic strategies and patient management. Here, we investigate the impact of driver mutations on the composition and spatial architecture of the TME in LUAD. Using imaging mass cytometry (IMC), we analyse tumour samples from 157 LUAD patients, integrating genomic and clinical data to link specific mutations with tumour characteristics. Unique patterns are associated with mutated KRAS and EGFR tumours with TP53 co-mutations, suggesting these co-mutations reshape the TME and promote resistance to tyrosine kinase inhibitors (TKIs). Overall, our findings highlight the complex interplay between oncogenic driver mutations and the TME in LUAD, underscoring the importance of integrating genomic and cellular data to understand the underlying tumour behaviour and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05eac26faa3f5fa852d857cbc84d4c9c3228ae87" target='_blank'>
              Oncogenic driver mutations underlie the spatial tumour immune landscape of non-small cell lung cancer
              </a>
            </td>
          <td>
            Saskia Hartner, H. Abolfathi, M. Rezanejad, Bridget Liu, E. Karimi, Dakota Rogers, Mark Sorin, Samuel Doré, Lysanne Desharnais, M. Orain, William Enlow, A. Gagné, Yuhong Wei, Y. Bossé, D. Quail, Philippe Joubert, L. Walsh
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719b808c693c9788562c4b889cdeaf7fc68e251b" target='_blank'>
              Spatial transcriptomics of glioblastoma defines biologically and clinically significant reprogramming patterns across unique spatial microenvironments
              </a>
            </td>
          <td>
            V. Ravikumar, Reva Kulkarni, A. Maddox, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98126a0a8ef060e1442a8189ca2cc50b5b3d5b2e" target='_blank'>
              Integrated cell atlas and tumoroids chart pancreatic cancer therapeutic targets
              </a>
            </td>
          <td>
            Quan Xu, Ryo Okuda, Bruno Gjeta, Christoph Harmel, Marina Signer, Matilde Lucioli, Małgorzata Santel, M. Seimiya, Cinzia Esposito, Karolina Guja-Jarosz, Ashley Maynard, S. Morinaga, Y. Miyagi, Tomoyuki Yamaguchi, Y. Ueno, Salvatore Piscuoglio, Daniel J. Müller, Hideki Taniguichi, Barbara Treutlein, J. G. Camp
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af62a3f15720a1c9a956b0f31df3cb0beb418ea" target='_blank'>
              Cell migration sculpts evolutionary dynamics favouring therapy resistance in lung cancer
              </a>
            </td>
          <td>
            A. Bhargava, Xiao Fu, Sasha Bailey, Yutaka Naito, Hamid Mohammadi, R. Hynds, S. Hooper, D. Biswas, A. Le Marois, Sunil Kumar, Yuriy Alexandrov, Paul French, James Mcginty, Paul A. Bates, C. Swanton, E. Sahai
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a highly heterogeneous cancer with complex tumor–immune interactions. This heterogeneity, particularly in tumor and immune cells, complicates treatment and prognostic evaluation. Although recent studies have revealed distinct tumor cell states and immune dysfunction in HCC, the molecular basis underlying tumor aggressiveness remains poorly understood. A deeper understanding of the molecular and functional diversity of both tumor and immune cell populations, especially the identification of stem-like tumor subpopulations and immunosuppressive mechanisms, along with the development of robust prognostic biomarkers, is essential for advancing precision oncology and improving clinical outcomes. Methods We integrated three publicly available single-cell RNA sequencing (scRNA-seq) datasets from GEO to delineate the cellular architecture of the HCC tumor microenvironment. Unsupervised clustering and dimensionality reduction were employed to identify major cell types and tumor subpopulations. Functional annotation was performed using canonical markers, Monocle, CytoTRACE, and AUCell scoring. H2AFZ was identified as a candidate oncogene and validated through in vitro knockdown experiments. The interaction between T cell subsets and tumor subpopulations were further characterized. A prognostic risk model was constructed using LASSO regression. Results Six major cell types were identified in HCC TME. Tumor cells were subdivided into three distinct clusters: Tumor_C0, Tumor_C1 and Tumor_C2. Tumor_C2 showed the highest stemness, pro-metastatic activity and immunogenic cell death signatures. H2AFZ was highly expressed in Tumor_C2 and associated with poor prognosis. The knockdown of H2AFZ reduced H2A.Z protein levels, inhibited proliferation, invasion, and induced apoptosis. T cell analysis revealed five subpopulations. It was found that Tumor_C2 interacts with the proliferative and exhausted T cell subpopulations, suggesting a potential functional relationship between them. The prognostic model based on tumor_C2 transcriptomic features effectively stratified patient survival across multiple cohorts, with robust AUCs and Kaplan-Meier survival distinctions. Conclusion We identified a proliferative, stem-like tumor cell subpopulation (Tumor_C2) in HCC characterized by high H2AFZ expression, which drives tumor aggressiveness. T cell analysis revealed significant interactions with Tumor_C2. Moreover, a prognostic model based on Tumor_C2 features effectively stratified patient survival across multiple cohorts. Together, these findings highlight potential therapeutic targets for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7024320dd03972a4da66c548f19e56393e35ae99" target='_blank'>
              Single-cell transcriptomics identifies an H2AFZ-driven proliferative tumor subpopulation associated with poor prognosis in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jihui Huo, Tao Yang, Kai Lei, Zeyao Wang, Zebin Chen, Qi Zhou
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd7292fed094fd219af6750590d28db3dc5a363b" target='_blank'>
              Spatial organization of the tumor-immune microenvironment in ER-positive breast cancer: remodeling during treatment and associations with clinical response
              </a>
            </td>
          <td>
            Maria Aanesland Dahle, Jan-Lukas Førde, E. V. Egeland, A. Creason, Cameron Watson, Ø. Garred, L. Prasmickaite, G. Mælandsmo, Gordon B. Mills, O. Engebraaten, M. H. Haugen
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed2e2b6e5867f00f570022e3faf5971a5955554" target='_blank'>
              Highly multiplexed imaging recovers immune and metabolic niches in multiple myeloma associated with disease progression and bone involvement
              </a>
            </td>
          <td>
            Ingrid Aass Roseth, Lukas Hatscher, Chiara Schiller, Håkon Skjalg Selland Johnstuen, T. Slørdahl, Håkon Hov, D. Schapiro, T. Standal
          </td>
          <td>2025-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Objectives Small cell lung cancer (SCLC) accounts for approximately 15% of lung tumors and is marked by aggressive growth and early metastatic spread. In this study, we used two SCLC mouse models with differing tumor mutation burdens (TMB). To investigate tumor composition, spatial architecture, and interactions with the surrounding microenvironment, we acquired multiplexed images of mouse lung tumors using imaging mass cytometry (IMC). These data build upon a previously published characterization of the mouse model. Data description After tumor detection, mice were assigned to one of five treatment groups. Lung tumor tissues were imaged with a 37-marker IMC panel designed to identify major cell types—tumor, immune, and structural—as well as their functional states. When possible, each tumor was sampled both at its center and border regions. Tumor masks in the form of binary images are provided to delineate tumor areas. Additional metadata include tumor onset and endpoint dates to support downstream correlation or predictive analyses based on the image data. This dataset offers a valuable resource for studying the histological and cellular complexity of SCLC in a genetically controlled mouse model across multiple therapeutic conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597aa5e8dfd6901b69ffa28e7837ffa96623cc96" target='_blank'>
              Imaging mass cytometry dataset of small-cell lung cancer tumors and tumor microenvironments
              </a>
            </td>
          <td>
            France Rose, O. Ibruli, Luca Lichius, M. Kiljan, Gokcen Gozum, Manoela Iannicelli Caiaffa, Jiali Cai, Li‐na Niu, Jan M. Herter, Holger Grüll, Reinhard Büttner, F. Beleggia, Graziella Bosco, Julie George, G. Herter-Sprie, H. C. Reinhardt, Katarzyna Bożek
          </td>
          <td>2025-09-08</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Stem-like gastric cancer (GC) is an aggressive molecular subtype marked by poor prognosis and limited response to immune checkpoint blockade (ICB). The spatial mechanisms driving this resistance remain unclear. Methods We conducted spatially resolved single-cell transcriptomic profiling of diffuse-type GC tissues to uncover the spatial architecture and functional diversity of tumor and stromal populations. Cellular heterogeneity and region-specific signaling pathways were characterized using integrative bioinformatics analyses. Results We identified transcriptionally diverse, high-entropy cell populations predominantly localized in the deep tumor regions. These included unique endothelial and fibroblast subsets enriched for pro-tumorigenic and immune-regulatory signaling. A notable finding was the engagement of deep-region endothelial cells in VISFATIN (extracellular NAMPT) signaling through the ITGA5–ITGB1 integrin axis, associated with immune evasion and poor prognosis. This endothelial signaling program is distinct from and functionally independent of cancer-associated fibroblast (CAF)-mediated pathways. Elevated expression of the NAMPT–ITGA5–ITGB1 axis was observed in ICB non-responders and correlated with reduced overall survival. Conclusions Our study delineates spatially defined cellular programs that contribute to immune escape in stem-like GC, highlighting a novel VISFATIN–integrin signaling axis as a potential biomarker and therapeutic target in immunotherapy-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dd6bcdbe10e93d5e57b978f2adeff6ec26c9091" target='_blank'>
              Spatially resolved endothelial signaling via nampt-itga5 drives immune evasion in stem-like gastric cancer
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jae-Ho Cheong, Eui Tae Kim
          </td>
          <td>2025-09-29</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696e778534089692f9510070ab5b7ccf3c2a11f2" target='_blank'>
              Single-cell spatial mapping reveals dynamic bone marrow microarchitectural alterations and enhances clinical diagnostics in MDS
              </a>
            </td>
          <td>
            Ryan Nachman, Aleksandra Kopacz, Caitlin Unkenholz, Jian Chai, Arvin Ruiz, Itzel Valencia, Jeanne Jiang, F. Socciarelli, Jiwoon Park, Christopher E. Mason, Ling Zhang, D. Sallman, Gail J. Roboz, Pinkal M. Desai, Justin Kaner, Joshua A Fein, Monica L Guzman, Neal Lindeman, Amy Chadburn, Madhu M Ouseph, Paul Simonson, Julia Geyer, Giorgio Inghirami, Shahin Rafii, David Redmond, Sanjay Patel
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df773895d5d3bb014e2dd4dc6276d432f2994491" target='_blank'>
              YAP1 defines an emergent, plastic population of relapsed small cell lung cancer
              </a>
            </td>
          <td>
            C. Stewart, K. Ramkumar, Runsheng Wang, Yuanxin Xi, Alexa J. Halliday, L. Diao, Qi Wang, Alejandra G Serrano, Sarah M. Groves, Simon Heeke, Azusa Tanimoto, L. Kaiser, Whitney E Lewis, Mukulika Bose, Pedro Da Rocha, L. Karacosta, Vito Quaranta, Jing Wang, Julie George, Luisa Maren, Solis Soto, Bingnan Zhang, J. Heymach, L. Byers, C. Gay
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Background Owing to the complexity of TME components and the heterogeneity of cancer cells, the relationship between the niches of TME and prognosis in breast ductal carcinoma remains unknown. The staged characteristics of corresponding cancer cell behaviors are unclear. Our study aims to reveal spatial structures and specific cellular information of TME and cancer cells subgroups during the progression from DCIS to IDC and lymph node metastasis. Methods Single-cell sequencing, spatial transcriptomics, bulk RNA sequencing datasets were used to explore the changes in microenvironmental components and transcriptional programs of tumor cells during the progression of breast ductal carcinoma. Immunohistochemistry, multiplex immunofluorescence, flow cytometry cell cycle detection, invasion migration experiments, and WB imprinting were employed for validation. Results Analysis of TME cell type subsets revealed the accumulation of TEX, iTreg, and stress-phenotype TAM in the mammary gland in situ during the invasion process. Lymphatic metastases exhibited enrichment of nTregs and a more naïve-like CD8 T cell population. Spatial analysis and survival analysis showed that the spatial niches of CD4 TN and phagocytic-phenotype macrophages were associated with a favorable prognosis, and these niches were lost during disease progression. The proliferative subpopulation of breast ductal carcinoma was enriched in lymphatic metastatic tissues, expressing high levels of FAM111B and exhibiting intense TCA and oxidative phosphorylation metabolism. Silencing FAM111B led to cell cycle arrest, decreased invasion and migration abilities, and downregulation of core mediator genes for cuproptosis and disulfidptosis. Conclusions The stage-specific microenvironmental characteristics of breast ductal carcinoma correspond to some extent to the behavior of tumor cells. During the progression of ductal carcinoma in breast tissue, the establishment of an immunosuppressive microenvironment occurs. The microenvironmental spectrum at lymph node metastases differs somewhat, corresponding to a more enriched turnover of cancer cell proliferation and death. Inhibitors of FAM111B and inducers of cuproptosis and disulfidptosis may serve as potential therapeutic targets for proliferative subgroups. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07010-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfce9a8ff47f521350ed382190ba1004f756bb5c" target='_blank'>
              Spatiotemporal microenvironment landscape and malignant epithelial pattern transition in breast ductal carcinoma progression
              </a>
            </td>
          <td>
            Xifu Cheng, Wenjuan Zeng, Bingzhe Yin, Jiawei Gui, Hengbin Zhang, Zhenxing Lv, Simin Zhang, Yao Zhou
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03cc2aa2a287dcfe2c1cd1643687b026d41744b" target='_blank'>
              Single-cell and Spatial Omics Reveals Region-Specific Plasticity and Therapeutic Vulnerabilities in Metastatic High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Lai Man, Natalie Wu, Janet L. Oblinger, Dazhuan E Xin, Rohit Rao, Feng Zhang, Mike Adam, Oscar Lopez-Nunez, Sara Szabo, D. Allmen, , Brian D Weiss, 
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality, with tumor heterogeneity and drug resistance posing significant challenges to treatment. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq deconvolution to analyze BRCA samples. Our analysis identified 15 major cell clusters, including neoplastic epithelial, immune, stromal, and endothelial populations. Notably, low-grade tumors showed enriched subtypes, such as CXCR4+ fibroblasts, IGKC+ myeloid cells, and CLU+ endothelial cells, with distinct spatial localization and immune-modulatory functions. These subtypes were paradoxically linked to reduced immunotherapy responsiveness, despite their association with favorable clinical features. High-grade tumors exhibited reprogrammed intercellular communication, with expanded MDK and Galectin signaling. Bulk RNA-seq deconvolution further supported the prognostic significance of low-grade-enriched subtypes. Our findings highlight the heterogeneity of the tumor microenvironment and provide new insights into immune evasion and therapeutic resistance in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57925849e8c34466fe3132e9855985812e402540" target='_blank'>
              Machine learning analysis reveals tumor heterogeneity and stromal-immune niches in breast cancer
              </a>
            </td>
          <td>
            Junjie Kuang, Guofang Zhong, Linfeng Zhao, Xia Yuan, Yundong Zhou, Jun Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0d7b578f97dbff41a41bb90de3b031320cfe41" target='_blank'>
              Unravelling drug resistant proteotypes through phenotype-resolved proteomics of single-cell derived colonies
              </a>
            </td>
          <td>
            Di Qin, Ridhima Das, S. Schallenberg, K. Riecken, I. Tinhofer, Fabian Coscia
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Purpose Microsatellite instability-high colorectal cancer is characterized by hypermutated genomes and high neoantigen loads, yet a significant proportion of patients exhibit resistance to immune checkpoint blockade. This study aims to investigate tumor cell functional heterogeneity and its role in immune evasion. Patients and Methods We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA sequencing from microsatellite instability-high colorectal cancer patients. After quality control, normalization, and clustering, malignant epithelial subpopulations were identified through copy number variation analysis and non-negative matrix factorization. Functional characterization employed gene set enrichment analysis. Spatial transcriptomics clarified immune cell and tumor subpopulation localization, and survival analyses assessed prognostic implications. Results We identified a glycolysis-enriched tumor subpopulation (MP2) that co-localized with immunosuppressive niches marked by Treg accumulation, effector T-cell depletion, and FOLR2+ tumor-associated macrophages. MP2-high tumors were associated with immune checkpoint blockade resistance and poor prognosis. Mechanistically, MP2 cells secreted lactate, promoting Treg differentiation and macrophage polarization toward an immunosuppressive phenotype. Spatial transcriptomics revealed the precise organization of these lactate-rich, immune-excluded niches within tumors. Conclusion These findings establish tumor cell-intrinsic glycolysis as a key driver of immune evasion in microsatellite instability-high colorectal cancer and propose metabolic targeting as a strategy to overcome immune checkpoint blockade resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773cdb1598a2f7975562a7cc2f1fda279ad89676" target='_blank'>
              Glycolysis-Driven Immune Evasion in Microsatellite Instability-High Colorectal Cancer: An Integrated Single-Cell and Spatial Transcriptomics Study
              </a>
            </td>
          <td>
            Chenchen Li, Peicong Cai, Hengda Zeng, Jianxia Li, Huabin Hu, Jianwei Zhang, Zehua Wu, Ge Qin, Yanhong Deng
          </td>
          <td>2025-09-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is highly heterogeneous, with five-year survival rates dropping from $\sim$90% in localized disease to $\sim$15% with distant metastases. Disease progression is shaped not only by tumor-intrinsic alterations but also by the reorganization of the tumor microenvironment (TME). Metabolic, compositional, and spatial changes contribute to this progression, but considered individually they lack context and often fail as therapeutic targets. Understanding their coordination could reveal processes to alter the disease course. Here, we combined multiplexed ion beam imaging (MIBI) with machine learning to profile metabolic, functional and spatial states of 522 colorectal lesions with single-cell resolution. We observed recurrent stage-specific remodeling marked by a lymphoid-to-myeloid shift, stromal-cancer cooperation, and malignant metabolic shifts. Spatial organization of epithelial, stromal, and immune compartments provided stronger stratification of disease stage than tumor-intrinsic changes or bulk immune infiltration alone. To systematically model these coordinated changes, we condensed multimodal features into 10 latent factors of TME organization. These factors tracked disease progression, were conserved across cohorts, and revealed frequent multicellular metabolic niches and distinct, non-exclusive TME trajectories. Our framework MuVIcell exposes the elements that together drive CRC progression by grouping co-occurring changes across cell types and feature classes into coordinated multicellular programs. This creates a rational basis to therapeutically target TME reorganization. Importantly, the framework is scalable and flexible, offering a resource for studying multicellular organization in other solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884e760c0de1e86aa10844e17508931cd572069b" target='_blank'>
              Robust multicellular programs dissect the complex tumor microenvironment and track disease progression in colorectal adenocarcinomas
              </a>
            </td>
          <td>
            Loan Vulliard, Teresa Glauner, Sven Truxa, Miray Cetin, Yu-Le Wu, R. Simon, Laura Behm, Jovan Tanevski, Julio Saez-Rodriguez, G. Sauter, Felix J. Hartmann
          </td>
          <td>2025-10-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab36a15bd0fb0bbb24a4b0020944cbb88eef13f0" target='_blank'>
              OSDR2.0 infers microenvironment-driven cell-state transitions and population dynamics from a single spatial biopsy
              </a>
            </td>
          <td>
            Itay Ben Shalom, Jonathan Somer, Shoval Miyara, Avi Mayo, Shie Mannor, U. Alon
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75be2c8e54a33a7fe17a0e30ce626e667d1c6218" target='_blank'>
              Revealing Shared Tumor Microenvironment Dynamics Related to Microsatellite Instability Across Different Cancers Using Cellular Social Network Analysis
              </a>
            </td>
          <td>
            N. Zamanitajeddin, M. Jahanifar, M. Eastwood, G. Gunesli, M. J. Arends, N. Rajpoot
          </td>
          <td>2025-09-29</td>
          <td>None</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/637f216aefe87456deedf3795555bf830c51467e" target='_blank'>
              A transcriptional map of human tonsil architecture: beyond the sum of (single cell) parts
              </a>
            </td>
          <td>
            Helena L. Crowell, L. Llaó-Cid, G. Frigola, Samuel Gunz, Irene Ruano, Patricia Lorden, Max Ruiz, Marta Kulis, J. Martín-Subero, Holger Heyn, E. Campo, A. Pascual-Reguant
          </td>
          <td>2025-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3145b8048f1a33f4e0cf44307241622724c18a05" target='_blank'>
              Mapping CD4+ T Cell Landscapes in Glioblastoma Reveals Effectors and Bystanders
              </a>
            </td>
          <td>
            Cameron M. Hill, Anthony Z. Wang, Jose A. Maldonado, T. Johanns, Allegra A. Petti, Gavin P. Dunn
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35fce186f7fddf3cfc3ee6a30b68015192ce168e" target='_blank'>
              Exploiting TGF-β-mediated Stromal Programming in Homologous Recombination-Deficient Pancreatic Cancer
              </a>
            </td>
          <td>
            E. Roger, Hannah M Mummey, E. Zimmer, Dharini Srinivasan, A. Härle, Loic Moubri, Alica K Beutel, Rima Singh, M. Ekizce, M. K. Melzer, Yun Lee, Austin Silva, Luisa Härle, T. Engleitner, F. Arnold, M. Morawe, Benjamin K. Naggay, Juliane Schneider, Leonard Gilberg, Julia P. Mosler, Christina Ludwig, Chen Meng, M. Hirschenberger, V. Hunszinger, K. Kluck, Martina Kirchner, A. Volckmar, Mareike Wirth, M. S. Alhamdani, Jörg D. Hoheisel, J. Löhr, Thomas Seufferlein, Alireza Abaei, Ralf Kemkemer, Roland Rad, J. Budczies, M. Mulaw, Patrick C. Hermann, Mark Hänle, K. Sparrer, Christopher J. Halbrook, K. J. Gaulton, K. Steinestel, A. Stenzinger, Nelson Dusetti, L. Perkhofer, J. Gout, A. Kleger
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Background Metastatic renal cell carcinoma (mRCC) still lacks tissue-based biomarkers that predict benefit from first-line immune checkpoint inhibitor (ICI) regimens. High-resolution spatial transcriptomics can dissect the tumor microenvironment (TME) and reveal prognostic niches invisible to bulk assays, offering a path toward precision immunotherapy. Methods Formalin-fixed, paraffin-embedded samples from 12 therapy-naïve patients with mRCC treated with ICI-based combinations were profiled with 10x Genomics Visium and NanoString COSMx. Six patients contributed matched metastatic lesions. Cell-type deconvolution used robust cell type decomposition with a public single-cell RNA-seq reference. Clinical outcomes (mean follow-up 16.75 months) were correlated with spatial features. Validation employed Phase 2/3 JAVELIN Renal 101 as well as Checkmate 010/025 transcriptomic data. Results Spatial transcriptomics uncovered pronounced divergences between primary tumors and metastases in cellular composition and microarchitectural organization. Canonical RCC molecular subtypes displayed marked intrapatient and interpatient spatial heterogeneity, underscoring limitations of bulk classifications. Five recurrent immune niches emerged across samples. One niche—rich in macrophages and CD8+ T cells and defined by chemokine signaling—was closely associated with objective response in metastatic lesions yet absent or infrequent in matched primaries. A gene signature reflecting this niche stratified responders within the immuno-oncology (IO) -treated arms of JAVELIN 101 and CheckMate 010/025 trials. On the other hand, signatures derived from angiogenic tumor cell-rich niches defined responders within the comparator arms (tyrosine kinase inhibitor/everolimus) in these trials. Conclusion Metastasis-specific immune niches, rather than primary-tumor characteristics, appear decisive for ICI efficacy in mRCC. Spatial transcriptomic profiling can identify these clinically relevant microenvironments and generate transferable gene signatures, supporting biomarker-driven patient selection and trial design. Integrating spatial TME analysis into future studies may accelerate personalized immunotherapy strategies for mRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bedf8d3b886332325de79302aa00547b1e48f37" target='_blank'>
              Spatial transcriptomic profiling of metastatic renal cell carcinoma identifies chemokine-driven macrophage and CD8+ T-cell interactions predictive of immunotherapy response
              </a>
            </td>
          <td>
            Charis Kalogirou, M. Krebs, Andreas S Kunz, Oliver Hahn, Hubert Kübler, Marcel Schwinger, Arndt Hartmann, Astrid Schmieder, Markus Eckstein
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) incidence is rising, and treatment remains challenging unless surgery is curative. Tumour heterogeneity contributes to resistance against both chemotherapy and immune checkpoint inhibitors, underscoring the need to better understand the complex tumour microenvironment (TME). While tumour models derived from cancer tissue from patients have advanced cancer research, they often fail to capture functional RCC heterogeneity and key TME components. We developed a 3D model system with a high success rate from resected tumour, retaining cancer, stromal, and immune cell populations. This system is fully compatible with advanced imaging technologies, including mass spectrometry imaging (MSI) and live-cell multiplex imaging. By integrating static spatial analysis with dynamic live-cell visualisation, our system provides unique insights into tumour heterogeneity, microenvironment metabolic crosstalk, and real-time cellular responses. Phenotypic characterization of the tumoroids showed strong histological resemblance to the original resected tissue, indicating that the tumoroids are reflective of the tumour in vivo and suitable as a representative model system. Additionally, DESI-MSI revealed distinct lipidomic profiles within patient-derived ccRCC tumoroids, capturing spatial metabolic heterogeneity reflective of the primary tissue. Lipid signatures varied across tumour regions, with phospholipid subclasses distinguishing epithelial, endothelial, and highly proliferative cell populations. Notably, non-clear cell regions exhibited reduced lipid droplet and fatty acid content, aligning with aggressive tumour phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc004e66880e4588257a07a21abe175bda8bb5d0" target='_blank'>
              Kidney tumoroid characterisation by spatial mass spectrometry with same-section multiplex immunofluorescence uncovers tumour microenvironment lipid signatures associated with aggressive tumour phenotypes
              </a>
            </td>
          <td>
            Hazem Abdullah, Greice M Zickuhr, I. Um, Alexander Laird, Peter Mullen, David J. Harrison, A. Dickson
          </td>
          <td>2025-09-16</td>
          <td>npj Imaging</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Human pancreatic ductal adenocarcinomas (PDAC) are composed of sparse, heterogeneous tumour cell populations embedded within a dense, desmoplastic stroma and are highly resistant to existing therapies. Guided by underlying genomic aberrations, cancer cell states and phenotypes interact with their microenvironment to shape disease trajectory and therapeutic response. To quantify the cellular phenotypic heterogeneity of these complex tumour-microenvironment interactions we designed custom multiplexed histopathology imaging panels against single cell RNA sequencing-defined tumour, immune, and stromal cell compartments. Using imaging mass cytometry (IMC), we profiled the co-localization and organization of 83 cell types and their functional states in 221 resected pancreatic tumours. This identified extensive inter- and intra-patient heterogeneity including the presence of both classical and basal cell types in most patients and a gradient of pancreatic epithelial identity confirmed by scRNAseq. This single cell content defined an expanded spectrum of classifiable intermediate tumour subtypes between classical and basal which have specific associations to neighboring immune and stroma cellular content, ploidy, established broad transcriptional subtypes, heterogeneity, and patient outcome. Eight reproducible cancer microenvironments were quantified, empowering cross-tumour comparisons. Matched whole genome sequencing (WGS) identified aberrations associated with specific tumour phenotypes and microenvironments. These results support asynchronous tumour and microenvironment genotype-phenotype axes which subdivide basal to classical tumour polarization and categorize co-existing microenvironments from stiff, ECM-rich, immune suppressed to immune infiltrated regions alongside fibrovascularized CD105+ stroma. This is based upon spatially heterogeneous co-occurring microenvironment niches and correlated tumour phenotypes which are associated with KRAS or MYC amplification and CDKN2A or LATS2 deletion on one axis, mutation of epigenetic modifiers in intermediate and hybrid subtypes, and RNF43 deletion on the other axis. To deeply profile the signalling pathways and enriched biological processes characteristic of each tissue state, we performed deep proteomics measurements of each microenvironment and tumor phenotype through whole-slide IMC informed laser capture microdissection and mass spectrometry of 273 micro-regions from a subset of patients. Finally, to identify a robust, minimized set of outcome-relevant features, we used a multi-modal technology optimized machine learning model trained to predict overall survival. We compare the predictive potential of WGS and spatial proteomics and show that a combination of genomic and cellular content outperforms clinical features and platform specific models thereby demonstrating the synergistic benefit of integrated multi-modal data. Our findings classify the spatial organization of human PDAC to identify an intertwined tumour/microenvironment axis that is influenced by specific genomic aberrations.



 Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Beth Sunnucks, Jennifer Gorman, Chengxin Yu, Michael Geuenich, Sheng Ben Liang, Barbara Gruenwald, Ayelet Borgida, Cassandra Wong, Brendon Seale, Zhen Yuan Lin, Edward Chen, Golnaz Abazari, Miralem Mrkjonic, Julie Wilson, Kieran Campbell, Anne-Claude Gingras, Rob Grant, Grainne O'Kane, Faiyez Notta, Steve Gallinger, Hartland Jackson. Spatial proteomics extend tumour subtype and microenvironment classifications across genotype-phenotype axes of pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B128.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61e2ac6f788886b4d295a71549b94c0503bd2fb" target='_blank'>
              Abstract B128: Spatial proteomics extend tumour subtype and microenvironment classifications across genotype-phenotype axes of pancreatic cancer
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Beth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng Ben Liang, Barbara Gruenwald, A. Borgida, Cassandra J. Wong, Brendon Seale, Z. Lin, Edward L. Chen, Golnaz Abazari, Miralem Mrkjonic, Julie M. Wilson, Kieran R. Campbell, A. Gingras, Robert C. Grant, G. O’Kane, Faiyez Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Current colorectal cancer mouse models either lack colon specificity, limiting progression towards more advanced disease, or preclude evaluation of resident stem cells as cancer origins. Here we report the identification of NOX1 and NPY1R as cell-surface markers enriched in LGR5+ stem cells predominantly within the caecum and exclusively within the middle and distal colorectum, respectively. Selective dysregulation of Wnt signalling in NOX1+ or NPY1R+ stem cells using CreERT2 mouse lines drives colon cancer initiation, predominantly within the caecum and rectum respectively, establishing these stem cell populations as important sources of colon cancer. Selective conditional activation of Wnt signalling and oncogenic Kras in combination with loss of TRP53 in these stem cell compartments resulted in the development of advanced, invasive cancers. This study establishes CreERT2 drivers as valuable tools for studying stem cell contributions to colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77d17a800b951a14fec481a0422bbf1125414c43" target='_blank'>
              NOX1 and NPY1R mark regional colon stem cell populations that serve as cancer origins in vivo
              </a>
            </td>
          <td>
            Maxime Gasnier, Tanysha Chi-Ying Chen, Swathi Yada, Sowmya Sagiraju, Yusuke Yoshikawa, Stefano Perna, H. Lim, Bernett Lee, Nick Barker
          </td>
          <td>2025-09-02</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d4f1e5ea74f6620f6b4fbf1988331e79868e36" target='_blank'>
              Interpretable Deep Learning Reveals Biologically Relevant Spatial Gene Expression Patterns in Lung Tumors and their Microenvironment
              </a>
            </td>
          <td>
            Vibha R. Rao, Adrienne A Workman, Liang Lu, Xiaoying Liu, Shrey S. Sukhadia
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="We introduce hybrid BAG-seq: a high-throughput, multi-omic method that simultaneously captures DNA and RNA from single nuclei. We apply this protocol to 65,499 single nuclei from samples of five uterine cancer patients and validate the clustering using RNA-only and DNA-only protocols from the same tissues. Multiple tumor genome or expression clusters are often present within a patient, with different tumor clones projecting into distinct or shared expression states, demonstrating nearly all possible genome-transcriptome correlations. We also identify mutant stroma with significant X chromosome loss in various cell types and patient-specific stromal subtypes exhibiting aberrant expression patterns. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03790-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee7a2488108131e2e266c468dcd1d8f8943443a1" target='_blank'>
              Hybrid BAG-seq: DNA and RNA from the same single nucleus reveals interactions between genomic and transcriptomic landscapes in human tumor samples
              </a>
            </td>
          <td>
            Siran Li, Joan Alexander, J. Kendall, Peter A. Andrews, Elizabeth Rose, Hope Orjuela, Sarah Park, Craig Podszus, Liam Shanley, Nissim Ranade, Patrick Morris, Danielle Stauder, Daniel Bradford, Zachary Laster, M. Ronemus, Arvind Rishi, M. Frimer, Rong Ma, David L Donoho, Gary L. Goldberg, Michael Wigler, D. Levy
          </td>
          <td>2025-09-27</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c81035972b65c627d1b19e7302e769d3ddd0b086" target='_blank'>
              A Single-Cell Atlas of the Breast Cancer Microenvironment Identifies Subtype-Specific Tumor-Immune Landscapes and Vulnerabilities
              </a>
            </td>
          <td>
            Liron Zisman Schachter, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease with diverse outcomes, and intra-tumoral heterogeneity plays a significant role in both diagnosis and treatment. Despite its importance, the spatial distribution of intra-tumoral heterogeneity is not fully elucidated. Spatial transcriptomics has emerged as a promising tool to study the molecular mechanisms behind many diseases. It offers accurate measurements of RNA abundance, providing powerful tools to correlate the morphologies of cellular neighborhoods with localized gene expression patterns. However, the spot-based spatial transcriptomic tools, including the most widely used platform, Visium, do not achieve single-cell resolution readouts, which hinders data interpretability. In this study, we present a computational pathology image analysis pipeline (i.e., computational tissue annotation, CTA) that utilizes machine learning algorithms to accurately map tumor, stroma, and immune compartments within Visium-assayed tumor sections. Using a cohort of 23 breast tumor sections from four patients, we demonstrate that CTA can provide high-resolution annotations on the hematoxylin-and-eosin-stained images alongside the paired sequencing data, support the evaluation of deconvolution methods, deepen insights into intra-tumoral heterogeneity by increasing data analysis resolution, assist with spatially resolved intrinsic subtyping, and enhance the visualization of lymphocyte clones at single-cell resolution. The proposed pipeline provides valuable insights into the complex spatial architecture of breast cancer, contributing to more personalized diagnostics and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5fa100fe1b095f296661cac47143cc93bae15b" target='_blank'>
              Computational pathology annotation enhances the resolution and interpretation of breast cancer spatial transcriptomics data
              </a>
            </td>
          <td>
            Tianyi Li, Qiao Yang, B. Ács, E. Sifakis, Hosein Toosi, Camilla Engblom, K. Thrane, Qirong Lin, Jeff E Mold, Wenwen Sun, Ceren Boyaci, Sanna Steen, Jonas Frisén, Jens Lagergren, J. Lundeberg, Xinsong Chen, Johan Hartman
          </td>
          <td>2025-09-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Understanding tumor heterogeneity at the resolution of individualized cell–cell communication networks (CCCNs) remains a major computational challenge in precision oncology. Existing inference methods largely rely on population-level correlation and thus fail to capture patient-specific signaling patterns across diverse cell types. To address this limitation, we developed an integrative computational framework combining the nested hierarchical Dirichlet process (nHDP) model for identifying hierarchically structured gene expression modules, with instance-specific Greedy Fast Causal Inference (iGFCI) for inferring individualized CCCNs (iCCCNs) in colorectal cancer (CRC). Applied to large-scale single-cell RNA-seq data from over 625,000 cells, our model successfully decomposed complex gene expression modules GEMs, potentially representing the cell lineage and cellular signaling states, and uncovered iCCCNs across detailed cell subtypes. We further used TCGA bulk RNA-seq data with survival data to validate the clinical relevance of these individualized gene expression module causal interactions, demonstrating their potential as robust prognostic signatures in CRC. Finally, we used principled causal inference methods to search for ligand-receptor pairs that mediate cell-cell communication. This framework enables mechanistic insights into immune evasion. Our computational method represents a significant advance toward realizing personalized oncology, enabling precise patient stratification and identification of actionable biomarkers for improved therapeutic targeting in cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4e906c5fe00a93c967af7c5aabe8332c82ec34" target='_blank'>
              Inferring Personalized Cell-Cell Communication Networks in Colorectal Cancer with Individualized Causal Discovery
              </a>
            </td>
          <td>
            Aodong Qiu, Binfeng Lu, Gregory F. Cooper, Xinghua Lu, Lujia Chen
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b038428041ef5d754c8d9cbe5398a0911b1f648" target='_blank'>
              Fusion-driven oncogenic programs shape the immune landscape in translocation renal cell carcinoma
              </a>
            </td>
          <td>
            Prathyusha Konda, Cary N. Weiss, Yantong Cui, Sayed Matar, Jinyu Wang, Yasmin L Nabil, Riva Deodhar, Jiao Li, J. Horst, Sabrina Y. Camp, A. Sheshdeh, Jonathan L. Hecht, David J. Einstein, Yashika Rustagi, Anwesha Nag, A. Thorner, Cheng-Zhong Zhang, E. V. Van Allen, S. Signoretti, T. Choueiri, Srinivas R Viswanathan
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038a04e70ede8fe2ebdfee87bc265630d9451dc" target='_blank'>
              Characterization of quiescent subpopulations and proliferative compartments in glioblastoma
              </a>
            </td>
          <td>
            Anca B. Mihalas, Kelly Mitchell, Sonali Arora, Samantha A. O’Connor, , Christopher L. Plaisier, Patrick J. Paddison
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe40c75d4c6f7843c947dbae95bbc2601c4d486" target='_blank'>
              Multi-modal spatial characterization of tumor immune microenvironments identifies targetable inflammatory niches in diffuse large B cell lymphoma.
              </a>
            </td>
          <td>
            Yibo Dai, A. Kizhakeyil, Dai Chihara, Xubin Li, Yunhe Liu, T. P. Sainz Zuniga, Ashley Wilson, Jared Henderson, Daniil Vibe, Arman Petrosyants, Connor Jacobson, Alexander Sarachakov, K. Nomie, K. Kryukov, Aleksander Bagaev, Ayushi Chauhan, J. Westin, Christopher R. Flowers, F. Vega, Linghua Wang, Michael R Green
          </td>
          <td>2025-10-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9fa61bcf35b5d6c234bc5ce442e119522c741a" target='_blank'>
              Basal cell of origin resolves neuroendocrine-tuft lineage plasticity in cancer.
              </a>
            </td>
          <td>
            A. Ireland, Daniel A Xie, Sarah B Hawgood, Margaret W Barbier, Lisa Y. Zuo, Benjamin E Hanna, Scarlett Lucas-Randolph, Darren R Tyson, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C. Moser, Anish Thomas, Sonam Puri, C. Rudin, Joseph M Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2025-09-17</td>
          <td>Nature</td>
          <td>2</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617cbae0aef811bf0bd717218d2f691f2de87645" target='_blank'>
              Germline genetic risk converges through intercellular crosstalk in tumor microenvironment
              </a>
            </td>
          <td>
            Xiaomeng Bai, Yang Liu, Shuai He, Chun-Ling Luo, G.-W. Lin, Dong-Mei Chi, Shuang-Yan Ye, Xiu-Zhi Wang, , You Zhou, Shu-Qiang Liu, P.-P. Wei, Xin-Yuan Guan, Hai-Qiang Mai, Soon Thye Lim, Jian-Jun Liu, C. Khor, Melvin Lee, Kiang Chua, Jin-Xin Bei, Yanni Zeng
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/386c9a2a1e499f753a4ef278dadc12b1134f2bac" target='_blank'>
              Disrupted astrocyte-endothelial crosstalk drives hemangioblastoma lesions in VHL disease
              </a>
            </td>
          <td>
            Macarena de Andrés-Laguillo, Irene Garcia-Gonzalez, Susana F Rocha, Aroa Garcia-Cabero, Sandra Ruiz-García, Luis Diago-Domingo, Aimane Danana, Lorena Cussó, Katrien de Bock, J. A. Enríquez, Rui Benedito
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d4b0eb929a81f46df1d1e2e57abece7f236b787" target='_blank'>
              PDAC tumor Immune system escape architecture: Analysis of transcriptomics of mechanisms that TME is tackling immune system in Pancreatic Ductal Adenocarcinoma Tumors
              </a>
            </td>
          <td>
            Abbas Zareei, Hossein Allahdadi
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="It has become evident from decades of clinical trials that multimodal therapeutic approaches with focus on cell intrinsic and microenvironmental cues are needed to improve understanding and treat the rare, inoperable, and ultimately fatal diffuse intrinsic pontine glioma (DIPG), now categorized as a diffuse midline glioma. In this study we report the development and characterization of an in vitro system utilizing 3D Tumor Tissue Analogs (TTA), designed to replicate the intricate DIPG microenvironment. The innate ability of fluorescently labeled human brain endothelial cells, microglia, and patient-derived DIPG cell lines to self-assemble has been exploited to generate multicellular 3D TTAs that mimic tissue-like microstructures, enabling an in- depth exploration of the spatio-temporal dynamics between neoplastic and stromal cells. The 3D-TTA model recapitulates clinical patterns of DIPG growth, evidenced by resistance to chemotherapy, HDAC and proteasome inhibitors, as well as sensitization to the antibody-activated innate immune microenvironment including complement proteins and surrounding microglia. Multimodal fluorescence imaging platforms integrated with high-throughput omics revealed that alterations in tumor cell motility and growth in the 3D-TTA model compared to tumor cell only spheroids correlated with specific transcriptomic and proteomic changes. STAT3, ITGA5, LGALS1, SOD2, MVP, and CLIC1, associated with microenvironment signaling, DNA replication, and immune regulation, were identified as potential novel targets in the 3D model. The results indicate that the 3D TTA platform developed here represents a powerful tool for preclinical studies, paving the way for identification/validation of tissue specific biomarkers and novel drug targets, thus advancing disease management strategies for DIPG in children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b07b858c6fd1cb8139cba6229ad20ebbdfab9d" target='_blank'>
              Multicellular tumor-stromal interactions recapitulate aspects of therapeutic response and human oncogenic signaling in a 3D disease model for H3K27M-altered DIPG
              </a>
            </td>
          <td>
            Meenakshi Upreti, Astgik Petrosyan, M. Thornton, A. Hovsepyan, G. E. Fernandez, David S. Koos, S. Byrum, Samuel G Mackintosh, Jacob K Al-Husseini, Tania Porras, Joseph Ha, A. Tackett, Miqin Zhang, M. Johal, Anat Erdreich-Epstein, Susan Durham, Mark D Krieger, Ashley S Margol, Brendan H. Grubbs, T. Chambers, S. Asgharzadeh, Rex A. Moats, Peter A Chiarelli
          </td>
          <td>2025-09-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Abstract The tumor microenvironment (TME) is a critical determinant of tumor progression and therapeutic response in gliomas. While pediatric gliomas have historically been treated using strategies derived from the management of adult gliomas, emerging evidence reveals that pediatric gliomas possess a unique TME. The pediatric TME is distinct, characterized not only by differences in cellular composition but a lower mutational burden, diminished neoantigen presentation, and heightened immunosuppressive activity. The unique immune landscape, developmental trajectories, and immune escape mechanisms in the pediatric TME create barriers to effective therapy. Recent studies show promising results in novel and advanced therapeutic strategies, highlighting the potential for innovative immunotherapeutic approaches. Advances in methodologies for modeling the TME, including computational approaches and animal-based models, provide new insights into pediatric glioma biology. Utilization of computational models may provide opportunities to predict tumor response to specific therapies and tailor immunotherapy regimes to individuals, allowing for personalized care. Leveraging the unique features of the pediatric TME offers an opportunity to overcome current barriers to immunotherapy and develop more effective, age- and tumor-specific treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d12c4d7c16cb4f880257014b09a937ace0732f5a" target='_blank'>
              Targeting the tumor microenvironment in pediatric gliomas: Advances and future directions in immunotherapy
              </a>
            </td>
          <td>
            Cheyenne Ahamed, Lam Nguyen, Cayley S Brock, Ayla Farzamnia, Pierrick Millet, Keisaku Sato, Kevin K Kumar
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT The immune landscape of healthy prostate and its alterations during prostate cancer (PCa) progression remain poorly characterized. Using scRNA-sequencing and multiparametric flow-cytometry analysis, we comprehensively characterized immune cells in wild-type and PTEN(i)pe−/− mouse prostates, a model that closely recapitulates human PCa. PCa in PTEN(i)pe−/− is marked by the recruitment of tumor-associated neutrophils (TANs), which represent the dominant immune cell population and resolved into eight distinct states, Trem2+ tumor-associated-macrophages (TAMs), and exhausted CD8+ T cells. Trem2+ TAMs differ from the three main resident macrophage populations in the healthy prostate, exhibiting a strong metabolic and immunosuppressive signature, likely driven by the MIF/HIF1A-signaling axis. This study provides the first detailed characterization of immune cells in the healthy mouse prostate and reveals changes in the immune landscape associated with prostate cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb0c0592aa197777e78a6d1020c20a4157e6fc" target='_blank'>
              Characterization of the immune landscape in healthy mouse prostate and during prostate cancer progression
              </a>
            </td>
          <td>
            Despoina Pervizou, Joanna De Chiara, L. Spinelli, Maia Nestor-Martin, Lionel Chasson, Kateryna Len-Tayon, Darya Yanushko, Frédéric Fiore, Marc Bajénoff, Bernard Malissen, Daniel Metzger, Gilles Laverny, S. Henri
          </td>
          <td>2025-09-26</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract The ability to interrogate changes within the tumor microenvironment (TME) before, during and following therapeutic intervention could yield important understanding of treatment response and causes for disease progression. Yet, the role of investigational tissue analysis faces key challenges in the clinical setting and the value of integrating longitudinal biopsies with emerging multimodal molecular analyses (“Multi-omics”) remains to be defined. In this study, we conducted a multicenter phase 2 clinical trial examining the effect of a novel cytotoxic T-lymphocyte-associated protein 4/programmed death-ligand 1 bispecific antibody in combination with a dual-epitope blocking anti-human epidermal growth factor receptor 2 antibody in treatment-resistant metastatic breast cancer. We performed longitudinal sampling of patient tumor tissues before and following treatment. Single-cell RNA and T cell receptor sequencing from 334,183 cells from site-matched tumors reveals significant temporal shift of various immune cell populations and phenotypes within the TME associated with treatment responses. Conversely, regulatory T cells were activated while effector T cells, natural killer cells, and dendritic cells were significantly depleted in non-responding tumors. Taken together, these results support that longitudinal analysis of TME to generate multiomics data that can lead to rich insight into disease process and to provide clinical value in evaluating treatment responses. Trial registration number NCT04521179.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97d0a38c945abea56c7fcf08591e230c989d914" target='_blank'>
              Longitudinal tissue analysis reveals microenvironmental changes correlate with combined immunotherapy and targeted therapy response in metastatic breast cancer
              </a>
            </td>
          <td>
            Jianyou Liao, Jien-Ren Wang,, Hengyu Li, Zhijun Liu, Z. Tian, Xinying Lv, Jianjian Peng, Chuangui Song, Jieqiong Liu
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Heterogeneity within the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment (TME), across tumor locations, and between patients largely contributes to poor treatment response and PDAC’s status as the fourth leading cause of cancer death in the United States. In order to identify suitable targets for innovative therapeutic interventions, it is necessary to understand and mechanistically test TME constituents, their infiltration, and their interaction dynamics in the context of primary and metastatic tumor locations using pre-clinical modeling. Therefore, we designed a novel 46-marker multiplex immunofluorescent staining panel for murine formalin-fixed paraffin-embedded (FFPE) tissues. This panel enables the spatial quantification of immune cell populations and activation/exhaustion states, epithelial-to-mesenchymal phenotypes, cancer-associated fibroblast heterogeneity, hematologic and lymphatic vessel infiltration, neuronal infiltration, and extracellular matrix deposition in the TME. We quantified the composition and spatial infiltration of the TME on matched murine primary and liver tumor samples from tandem orthotopic and hemi-spleen transplants of three distinct primary murine PDAC cell lines derived from the KPC model (KrasG12D; TP53R172H; PDX1-Cre; RosaEYFP). The resulting primary tumors from each cell line showed inter-tumoral heterogeneity as their distinct levels of immune infiltration had varying clustering patterns. Spatial analysis further revealed heterogeneous cancer cell MHC-I expression, immune cell exhaustion, and myeloid and lymphoid populations correlating with cell line. However, all liver nodules were encased by a barrier of immune cells separating the tumor from normal liver parenchyma irrespective of cell line. This observation reflects our recent spatial transcriptomic analysis of patient-derived matched liver and primary tumors, supporting the use of these pre-clinical models for future mechanistic investigations. Altogether, these data highlight the complexity of PDAC TMEs and the site-specific differences between primary and metastatic tumor composition.



 Christina R. Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert Welner, Julienne L. Carstens. High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B110.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335836491342c8b5df5a982f9e0790e87ffd1d16" target='_blank'>
              Abstract B110: High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity
              </a>
            </td>
          <td>
            Christina R Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) evolves within a highly interactive tumor microenvironment (TME) that shapes therapeutic response. We utilized mass cytometry to analyze over 10 million cells from 25 ESCC tumors, 24 adjacent nontumor tissues, and 23 peripheral blood samples, employing an extensive panel of 42 immune markers. The resulting atlas reveals a compartmentalized landscape with a reproducible paucity of CD4⁺ and CD8⁺ central memory T cells (TCM) in tumor sites. Reintroduction of patient-derived TCMs restored antitumor immunity in coculture assays, demonstrating their cytotoxic capacity in vitro and suggesting their potential relevance for future therapeutic exploration. Myeloid profiling identified PD-L1⁺ tumor-associated macrophages (TAMs) as correlates of clinical benefit; ex vivo PD-L1 blockade reprogrammed TAMs toward proinflammatory states, indicating pharmacological malleability. Notably, CD39⁺ tumor-infiltrating T cells were consistently associated with favorable prognosis and increased responsiveness to PD-1 blockade across cancer types. The functional inhibition of CD39 impaired cytotoxic T-cell activity, underscoring its dual role as a marker of immune dysfunction and a promising therapeutic target. Collectively, our findings provide a comprehensive immune landscape of ESCC, highlighting key immunological deficits and opportunities for targeted interventions. The insights gained underscore the potential of tailoring immunotherapies to the specific immune profiles of the TME, potentially revolutionizing treatment paradigms for ESCC patients. This study sets the stage for a more nuanced understanding and manipulation of the immune elements critical for optimizing cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c74ee06dce5a8743106e3e0c5ec6ea39727b2973" target='_blank'>
              Single-cell atlas of the esophageal squamous cell carcinoma immune ecosystem to predict immunotherapy response
              </a>
            </td>
          <td>
            Xiankai Chen, Yahui Zhao, Yuhao Wang, Xiliang Wang, Yuhao Liu, Zhihua Liu, Yin Li
          </td>
          <td>2025-10-20</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly heterogeneous malignancy, characterized by low tumor cellularity, a dense stromal response, and intricate cellular and molecular interactions within the tumor microenvironment (TME). Although bulk omics technologies have enhanced our understanding of the molecular landscape of PDAC, the specific contributions of non-malignant immune and stromal components to tumor progression and therapeutic response remain poorly understood. Methods: We explored genome-wide DNA methylation and transcriptomic data from the Cancer Genome Atlas Pancreatic Adenocarcinoma cohort (TCGA-PAAD) to profile the immune composition of the TME and uncover gene co-expression networks. Bioinformatic analyses included DNA methylation profiling followed by hierarchical deconvolution, epigenetic age estimation, and a weighted gene co-expression network analysis (WGCNA). Results: The unsupervised clustering of methylation profiles identified two major tumor groups, with Group 2 (n = 98) exhibiting higher tumor purity and a greater frequency of KRAS mutations compared to Group 1 (n = 87) (p < 0.0001). The hierarchical deconvolution of DNA methylation data revealed three distinct TME subtypes, termed hypo-inflamed (immune-deserted), myeloid-enriched, and lymphoid-enriched (notably T-cell predominant). These immune clusters were further supported by co-expression modules identified via WGCNA, which were enriched in immune regulatory and signaling pathways. Conclusions: This integrative epigenomic–transcriptomic analysis offers a robust framework for stratifying PDAC patients based on the tumor immune microenvironment (TIME), providing valuable insights for biomarker discovery and the development of precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec3422e79e1fa464a39c9ceade9afd8d95e43c98" target='_blank'>
              Deciphering the Heterogeneity of Pancreatic Cancer: DNA Methylation-Based Cell Type Deconvolution Unveils Distinct Subgroups and Immune Landscapes
              </a>
            </td>
          <td>
            Barbara Mitsuyasu Barbosa, Alexandre Todorovic Fabro, Roberto da Silva Gomes, C. Rainho
          </td>
          <td>2025-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is an aggressive ovarian cancer with a heterogeneous tumor microenvironment (TME). Advances in single-cell RNA sequencing (scRNA-seq) and spatially-resolved transcriptomics have enabled the study of complex TME. This study explores connections between molecular subtypes described from bulk transcriptomes and spatial domains characterized by distinct gene expression in HGSC and their variability between patients. We explored both intraand inter-tumor heterogeneity across 2D space and identified differing spatial patterns of gene expression pertaining to immune pathways and vasculature development. Functional characterization of tumor spaces revealed potentially shared cell states across molecular subtypes, while correlation analysis underscored subtype-specific spatial anti-colocalization between spots exhibiting antigen-presenting functions and B cell-mediated immunity. Lastly, we performed spatially-aware cell-cell communication analysis on the spatial samples and identified a molecular subtype specific difference in total signaling activity and heterogeneity in Midikine signaling between the differentiated subtype. Our results suggest that generating multiple tissue slices per patient might be necessary to enable comprehensive characterization of HGSC spatial transcriptomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12564a0c02e3362085c018e2a380154da3a01959" target='_blank'>
              Characterizing intra- and inter-tumor heterogeneity in Ovarian high-grade serous carcinoma subtypes using single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Weishan Li, Laurie Grieshober, Jason Gertz, Adriana Ivich, J. Doherty, Casey S. Greene, SC Hicks
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Intratumoral heterogeneity in pancreatic ductal adenocarcinoma (PDAC) is characterized by the coexistence of classical epithelial, basal, and mesenchymal cancer cell states. Targeting cancer cell plasticity—the ability of tumor cells to adapt to extrinsic pressures via non-genetic mechanisms—has emerged as a promising therapeutic strategy. In parallel, the immunosuppressive tumor microenvironment remains a major barrier to effective treatment in PDAC. Yet, whether specific cell states possess distinct functional properties that drive key tumor behaviors—such as phenotypic plasticity or immune evasion—remains poorly understood. Using genetically engineered mouse models driven by oncogenic Kras G12D and loss of Trp53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We find that the KPC model harbors a basal state, which harbors transcriptional signature that aligns with the "basal" signature in human PDAC and is linked to poor prognosis and chemoresistance. The basal state co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. To functionally interrogate the basal state in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the basal state has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the basal state is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the basal state over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we find that the basal state is a key driver of therapeutic resistance: rather than being eliminated by treatment, it persists through phenotypic adaptation in response to both KRAS-targeted therapies and cytotoxic chemotherapy. Co-elimination of the basal state alongside these therapies leads to deeper and more durable responses, underscoring its functional role in tumor persistence under therapy. In parallel, we observe that the basal state plays a central role in enforcing immunosuppression in PDAC tumors. Basal state ablation rapidly reshapes the tumor immune microenvironment, and we observe loss of cancer-associated fibroblasts, polarization of PDAC-resident macrophages toward a pro-inflammatory phenotype, and increased infiltration of T and NK cells. Cytokine profiling of basal-ablated tumors revealed a dramatic reduction in multiple immune-modulatory and immunosuppressive cytokines. Taken together, our data indicate that PDAC harbors a striking dependency on the basal state for both therapeutic resistance and immune evasion, and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.



 Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C. Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Zeynep C. Tarcan, Olca Basturk, Rona Yaeger, James G. Christensen, Yan Yan, Elisa de Stanchina, Dana Pe'er, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B108.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6879ea5062e58b2315581060e5992acf02efdbb" target='_blank'>
              Abstract B108: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Z. Tarcan, O. Basturk, R. Yaeger, James G Christensen, Yan Yan, E. de Stanchina, D. Pe’er, Tuomas Tammela
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is classified into epithelial and mesenchymal subtypes based on transcriptional profiling, and these subtypes are associated with varied patient outcomes. Differences in the tumor microenvironment (TME), particularly in cancer-associated fibroblasts (CAFs), may contribute to this variation. scRNA-seq studies have observed heterogeneous CAF populations in PDAC, although the relationship between tumor subtype and CAF phenotype is not well understood. To investigate this relationship, we utilized epithelial and mesenchymal PDAC cell lines in a xenograft model to examine whether these subtypes influence CAF phenotypes.



 Xenograft models were generated by subcutaneously injecting BALB/c nude mice with PDAC cell lines of either the epithelial subtype (BxPC-3) or the mesenchymal subtype (MIA PaCa-2). Tumors were then harvested and dissociated for scRNA-seq to investigate the TME. Human tumor (graft) and murine stromal (host) cells were computationally separated using the Xenocell tool. Subsequent analyses focused on fibroblasts within the TME, which were subclustered to assess CAF heterogeneity between PDAC subtypes. To characterize transcriptional programs and identify CAF features, we performed high-dimensional weighted gene co-expression network analysis (hdWGCNA) and analyzed differentially expressed genes. Finally, spatial transcriptomics was conducted on tumor sections from BxPC-3 and MIA PaCa-2 xenografts to validate CAF clusters observed in scRNA-seq.



 Distinct CAF clusters were identified in the xenograft tumors, including LRRC15+ CAFs, RGS5+ CAFs, and inflammatory CAFs (iCAFs). LRRC15+ CAFs were predominant in tumors derived from the epithelial subtype (BxPC-3), while RGS5+ CAFs were enriched in tumors from the mesenchymal subtype (MIA PaCa-2). LRRC15+ CAFs exhibited a myofibroblastic CAF (myCAF) phenotype characterized by extracellular matrix production, with high expression of Lrrc15, Postn, and Fbln1. RGS5+ CAFs expressed canonical pericyte markers such as Rgs5 and Cox4i2, along with CAF-associated genes including Acta2. These CAF clusters also showed distinct expression patterns of collagens and secreted factors. In particular, Col11a1, Col12a1, and Wnt5a were upregulated in LRRC15+ CAFs, while Col4a1, Col10a1, and Tgfb1 were upregulated in RGS5+ CAFs. Spatial transcriptomics confirmed that gene modules associated with Lrrc15 and Rgs5, inferred from hdWGCNA, were differentially expressed between Acta2-positive stromal regions of epithelial and mesenchymal PDACs.



 These results suggest that epithelial and mesenchymal PDAC subtypes directly shape distinct CAF populations. Notably, the enrichment of RGS5+ CAFs in mesenchymal tumors may underlie their metastatic potential and poor clinical outcomes.



 Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, Jung Hyun Jo. Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B105.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020656bad62fdcf52b16552c5f7369030a0e1205" target='_blank'>
              Abstract B105: Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model
              </a>
            </td>
          <td>
            Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo-Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, J. Jo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7502a80e605765bec32ff6d8c49daea675b88242" target='_blank'>
              Stromal subsets modulate T-cell infiltration in early breast cancer
              </a>
            </td>
          <td>
            Julia Chen, Hanyun Zhang, Travis Ruan, Sunny Z. Wu, I. Slapetova, Ewan K. A. Millar, Peter Graham, Jodi Lynch, Lois Browne, Elgene Lim, Alexander Swarbrick
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Biliary tract cancer (BTC) is a highly heterogeneous and aggressive gastrointestinal malignancy, marked by a high mortality rate and limited treatment efficacy. The primary contributing factors include the absence of reliable early detection methods, the anatomical intricacy of the biliary system, the inherently aggressive tumor biology, and the restricted effectiveness of systemic therapies. A profound understanding of molecular characteristics and clinically relevant emerging biomarkers is essential for advancing BTC treatment strategies. Recent developments in single-cell multi-omics technologies have enabled the analysis of genetic, transcriptomic, proteomic, and metabolomic data at the single-cell resolution, thereby uncovering the heterogeneity and complexity of tumor biology. These techniques provide critical insights into the diversity of immune cell populations within the tumor microenvironment (TME) and offer novel perspectives on tumor progression and potential therapeutic interventions. While single-cell technologies have significantly advanced the study of solid tumors, their application in BTC remains nascent, with a paucity of comprehensive reviews. This review systematically integrates single-cell genomics, transcriptomics, and epigenomics data to construct a cross-omics molecular atlas of BTC. It highlights the utility of single-cell multi-omics technologies in elucidating tumor heterogeneity, microenvironment remodeling, and clonal evolution in biliary tumors, while thoroughly analyzing their implications for clinical outcomes. Furthermore, this review explores personalized treatment strategies informed by single-cell technologies and underscores the significance of these technologies as indispensable tools for unraveling the complexity of BTC and fostering mechanism-based therapeutic innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c48afe626e59e1df497d34aa627d14168d0c25d2" target='_blank'>
              Single-cell multi-omics in biliary tract cancers: decoding heterogeneity, microenvironment, and treatment strategies
              </a>
            </td>
          <td>
            Nannan Tang, Jiatong Li, Ao Gu, Meng‐Yao Li, Yingbin Liu
          </td>
          <td>2025-10-15</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53fdc35836d59f5a9b6ad6d520636c7ede83639b" target='_blank'>
              Single-Cell Analysis of NF1-Expressing and NF1-Deficient Schwann and Fibroblast Cells Reveals Divergent Neurofibroma Programs
              </a>
            </td>
          <td>
            Grace M Swanson, Marcia Arenas-Hernandez, Katherine Gurdziel
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb5a38dd74bce74d21cd64c9a442cf5e375b46f4" target='_blank'>
              Targeting Glioblastoma Cell State Plasticity for Enhanced Therapeutic Efficacy
              </a>
            </td>
          <td>
            S. Cirigliano, R. Singhania, James G Nicholson, Isha Monga, Yushan Wan, C. Haywood, Ashlesha Muley, Skylar A. Giacobetti, Howard A. Fine
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Despite therapeutic advances, metastatic melanoma, and particularly brain metastasis (MBM), remains a lethal burden for patients. Existing single-cell studies offer a more detailed view of melanoma and its microenvironment, which is crucial to improve diagnosis and treatment. Results We here present a computational reanalysis of single-nucleus data comparing 15 MBM and 10 extracranial melanoma metastases (ECM), considering recent best practice recommendations. We used cell type-specific pseudobulking and omit imputation during patient integration to gain complementary insights. Interestingly, our analysis revealed high homogeneity in tumor cell expression profiles within and between MBM and ECM. However, MBM displayed even higher homogeneity but a more flexible energy metabolism, suggesting a specific metastatic adaptation to the putatively more restricted brain microenvironment. While tumor cells were homogeneous, the metastasis microenvironment, especially lymphocytes and related immune-tumor interaction pathways, exhibited greater divergence between MBM and ECM. Overall, this suggests that major differences between MBM and ECM are potentially driven by variations in their microenvironment. Finally, a comparison of single-cell data to previous bulk studies, including their deconvoluted putative cell types, showed significant differences, potentially causing divergent conclusions. Conclusion Our study contributed to refine the understanding of differences between MBM and ECM, suggesting these are potentially more influenced by their local microenvironments. Future research and therapies could possibly focus on the metabolic flexibility of melanoma brain metastases and patient-specific immune pathway alterations. Supplementary Information The online version contains supplementary material available at 10.1186/s13062-025-00691-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e755d874cd5c0ea8aa714bc1a90cc86adaa7c4ba" target='_blank'>
              Single-cell transcriptome analysis suggests cells of the tumor microenvironment as a major discriminator between brain and extracranial melanoma metastases
              </a>
            </td>
          <td>
            Konrad Grützmann, Michael Seifert
          </td>
          <td>2025-09-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In most solid tumors, hypoxia constitutes a defining microenvironmental feature that reprograms malignant cells into a highly metastatic state by driving cellular plasticity and exacerbating chromosomal instability (CIN). However, the mechanisms by which cancer cells concurrently co-opt these elements of hypoxic adaptation to promote metastasis remains poorly understood. Here, we report that hypoxia promotes metastasis by suppressing the JmjC-containing histone lysine demethylase Kdm8. CRISPR/Cas9-mediated targeting of Kdm8 in a Kras;Trp53-driven mouse model of pancreatic ductal adenocarcinoma (PDA) robustly rewires the malignant cell transcriptomic programs, leading to a profound loss of the epithelial morphology and widespread metastatic disease. In PDA patients, a high KDM8-induced gene signature is associated with reduced metastatic burden and better survival in advanced disease. Notably, Kdm8 suppression in normoxia recapitulates key aspects of the global epigenetic and transcriptomic rewiring, mitotic spindle defects, and CIN induced by hypoxia. Moreover, disruption of Kdm8’s demethylase activity phenocopies Kdm8 loss, whereas expression of hypermorphic Kdm8 variants resistant to hypoxic suppression markedly reduces metastasis beyond the levels achieved by the wildtype protein. Through the suppression of Kdm8 demethylase function, hypoxia unleashes a potent metastatic program by simultaneously advancing cellular plasticity and CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ead3df1c0ef44aa86f0839ae810b8a1fead329" target='_blank'>
              Hypoxia-induced metastatic heterogeneity in pancreatic cancer
              </a>
            </td>
          <td>
            Pradeep Moon Gunasekaran, Qianqian Wang, Yoke-Chen Chang, Polina Guseva, Rajika Chauhan, Alexander Kley, Gene Lee, Siddharth Ghosh Roy, Yousef Masoudpoor, Arthur Roberts, Kelly Watkins Walton, Lucyann Franciosa, Shafiq Bhat, Emmanuel Zachariah, Kishan Patel, Zhongren Zhou, Wenjin Chen, J. Ni, S. Gu, Cristina Montagna, Shin-Heng Chiou
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of breast cancer is a complex ecosystem, in which cancer-associated fibroblasts (CAFs), as the most abundant stromal cell type, meticulously construct an ecological niche that supports tumor growth through mechanisms including extracellular matrix (ECM) remodeling, secretion of bioactive factors, and interactions with neighboring cells. High-resolution technologies, including single-cell sequencing and spatial transcriptomics, have revealed the high heterogeneity, functional diversity, and spatial distribution within the CAF population. Significant differences exist in the interactions between distinct CAF subpopulations and immune cells. Through complex crosstalk with the immune system, they collaboratively establish an immunosuppressive network, becoming a core driving force for tumor immune escape. This review focuses on the latest research advances in heterogeneous subpopulations of CAFs within the breast cancer microenvironment, delves into how the complex bidirectional crosstalk between different CAF subpopulations and immune cells collaboratively shapes the tumor immune microenvironment (TIME), and summarizes various CAF-based therapeutic strategies for breast cancer, aiming to provide critical theoretical basis and novel therapeutic perspectives for the clinical translation of CAF heterogeneity research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e74899699742a2ae95fa97ca3627d6bdb366035" target='_blank'>
              Crosstalk between heterogeneous cancer-associated fibroblast subpopulations and the immune system in breast cancer: key players and promising therapeutic targets
              </a>
            </td>
          <td>
            Lu Liu, Weijun Wan, Yilin Chang, Luoquan Ao, Yan Xu, Xiang Xu
          </td>
          <td>2025-09-01</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Intra-tumoral heterogeneity (ITH) is common in malignant tumors and arises from dynamic variations across genetic, epigenetic, transcriptomic, proteomic, metabolic, and microenvironmental factors. This complexity drives tumor evolution and treatment resistance, undermining the accuracy of clinical diagnosis, prognosis, and treatment planning. Despite recent advances in multi-omics technologies that enable comprehensive mapping of ITH across molecular layers, major challenges remain in clinical translation. This review outlines the principles and clinical applications of eight major omics modalities in the context of ITH: genomics, single-cell genomics, transcriptomics, epigenomics, proteomics, radiomics, microbiome, and metabolomics. We highlight the unique contributions of each omics platform to tumor profiling and emphasize how their integration enhances biological interpretation, patient stratification, and biomarker discovery. Furthermore, we will focus more extensively on the limitations of applying these approaches to ITH analysis. Instead of providing an exhaustive catalog, this review highlights major controversies, technical hurdles, and conceptual gaps that impede the clinical translation of multi-omics-based ITH analysis, with the aim of addressing ITH-related clinical challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3a8e32658c6f97ddac24dad4088bab77b9c897d" target='_blank'>
              Integration of multi-omics approaches in exploring intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            Mengmeng Dong, Liping Wang, Ning Hu, Yueli Rao, Zhen Wang, Yu Zhang
          </td>
          <td>2025-08-29</td>
          <td>Cancer Cell International</td>
          <td>2</td>
          <td>2</td>
        </tr>

        <tr id="Metastasis remains the principal cause of cancer-related mortality, yet its distribution across organs is far from random. Instead, tumor cells exhibit organotropism, a consistent preference for colonizing specific distant tissues, a phenomenon shaped by anatomical constraints, molecular crosstalk, and microenvironmental compatibility. Far beyond mere mechanical entrapment in vascular beds, metastatic dissemination reflects a coordinated interplay between tumor-intrinsic programs and organ-specific niches. Tumor-derived extracellular vesicles, cytokines, and matrix-remodeling enzymes actively precondition distant sites through pre-metastatic niche formation, creating permissive microenvironments primed for colonization. Simultaneously, tissue-specific immune landscapes, stromal compositions, and mechanical architectures determine the fate of disseminated tumor cells, whether they are eliminated, enter dormancy, or form macrometastases. Phenotypic plasticity, metabolic reprogramming, and immune evasion mechanisms equip subclones with the capacity to exploit these unique niches. Across cancer types, reproducible patterns of organotropic metastasis not only guide clinical surveillance and therapeutic stratification but also reveal vulnerabilities in the metastatic cascade. This review synthesizes emerging mechanistic insights across anatomical, immunological, and molecular domains to construct a comprehensive framework of organotropism, highlighting therapeutic opportunities to intercept metastasis at organ-specific checkpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7663dab540302726b6f7b6337c8bf27a6cf336" target='_blank'>
              Deciphering organotropism reveals therapeutic targets in metastasis
              </a>
            </td>
          <td>
            Jhommara Bautista, Andrés López-Cortés
          </td>
          <td>2025-10-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a fundamental role in tumor progression. Cancer cells interact with their surroundings to establish a supportive niche through structural changes and paracrine signaling. Cells around transformed tumor cells contribute to cancer development, while infiltrating immune cells in this aggressive TME often become exhausted. Solid tumors, especially the most invasive types such as pancreatic ductal adenocarcinoma, are notably stiff mechanically, with cross-linking enzymes significantly affecting the survival of cancer cells in both primary tumors and metastatic sites. In this review, we highlight recent key contributions to the field, focusing on single-cell sequencing of stromal cells, which are increasingly seen as highly heterogeneous yet classifiable into distinct subtypes. These new insights enable the development of effective co-treatment approaches that could significantly enhance current and novel therapies against the most aggressive cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5047b2e8e1a1022055daa398bbad8bc791fe908c" target='_blank'>
              Microenvironment of Solid Tumors.
              </a>
            </td>
          <td>
            T. Korneenko, Nikolay B. Pestov, M. Shakhparonov, Nickolai A. Barlev
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Single‐cell RNA sequencing (scRNA‐seq) has transformed our understanding of tumours by enabling high‐resolution profiling of their cellular composition. Traditionally perceived as masses of homogeneous cancer cells, tumours are now recognised as complex ecosystems shaped by the tumour microenvironment (TME), which includes diverse immune cells, cancer‐associated fibroblasts and extracellular matrix components. scRNA‐seq has revealed remarkable heterogeneity within the TME, identifying novel or rare immune cell subsets and delineating their dynamic functional states. In particular, it has illuminated intercellular signalling networks and temporal cell‐state transitions that drive tumour progression and immune evasion. Moreover, the integration of scRNA‐seq data with clinical information has highlighted its potential in improving patient stratification, biomarker discovery and therapeutic target identification. Here, we systematically summarise recent advances in applying scRNA‐seq to dissect the TME, discuss the implications of these findings for immunotherapy resistance and precision oncology, and outline future opportunities for integrating scRNA‐seq with emerging technologies to develop more effective and personalised cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7bf7b9a7f8397821baf87f91e3e07f64fb492a" target='_blank'>
              Tumour Single‐Cell Bioinformatics: From Immune Profiling to Molecular Dynamics
              </a>
            </td>
          <td>
            Jane Siu Fan Li, Zoey Zeyuan Ji, Aaron Qi Zhang, C. Ng, Guibin Qiao, Dongmei Zhang, Chunjie Li, Qing Zhang, Ka-Fai To, P. M. Tang
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastasis remains the leading cause of cancer-related death, yet the biological determinants that enable tumor cells to disseminate and colonize distant organs are incompletely understood. Emerging evidence identifies the microbiome, not merely as a bystander, but as an active architect of the metastatic cascade. Microbial communities residing in the gut, mucosal barriers, and within tumors shape metastatic progression by modulating immune surveillance, stromal remodeling, oncogenic signaling, and therapy response. Intratumoral and even intracellular microbes regulate epithelial–mesenchymal transition, angiogenesis, and immune escape, while gut-derived metabolites condition pre-metastatic niches and alter systemic immunity. Technological advances in spatial transcriptomics, single-cell multi-omics, and metagenomics have revealed a spatially organized, functionally integrated microbial ecosystem within tumors, challenging long-held assumptions of sterility in cancer biology. This review synthesizes five converging dimensions of this paradigm: microbial interactions in the metastatic tumor microenvironment; microbiome-mediated immunoediting and metastatic escape; the role of intratumoral and intracellular bacteria in dissemination; spatial-multi-omic approaches to map microbial niches; and microbial biomarkers predictive of metastasis and therapy outcomes. Collectively, these findings recast the microbiome as a critical and targetable determinant of metastasis. Deciphering the tumor–microbe–host triad holds transformative potential for biomarker development, therapeutic innovation, and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15720d6be643a5b1bcd7bdc88b245af42661e241" target='_blank'>
              Microbial signatures in metastatic cancer
              </a>
            </td>
          <td>
            Jhommara Bautista, María Paula Fuentes-Yépez, Joseth Adatty-Molina, Andrés López-Cortés
          </td>
          <td>2025-09-30</td>
          <td>Frontiers in Medicine</td>
          <td>2</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8df178816a6957002d637941c2a82d771063a818" target='_blank'>
              Extracellular Matrix-Guided Tumor Stratification and Network Models Reveal Clinical Molecular Grades
              </a>
            </td>
          <td>
            Aslı Dansık, Sevgi Sarıca, E. Öztürk, Nurcan Tunçbağ
          </td>
          <td>2025-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Diffuse midline glioma (DMG) is an aggressive pediatric brain tumor with no curative treatment. The tumor microenvironment (TME) of DMG, shaped by its unique anatomical location and developmental context, poses significant therapeutic hurdles. While the TME has been characterized by a predominance of tumor-associated microglia and sparse infiltration of blood-derived immune cells, a comprehensive understanding of immune-tumor cell interactions and how to effectively target them with immunotherapies remains challenging.



 To dissect the DMG TME, we employed a multi-omics approach that integrated single-nucleus RNA sequencing, spatial transcriptomics, and high-dimensional imaging on both patient samples and an immunocompetent in utero electroporation murine DMG model. We characterized the spatial and phenotypic architecture of the tumors and validated a novel immune target using syngeneic mouse models and ex vivo co-culture experiments.



 Our analysis revealed pronounced intratumoral heterogeneity, with two spatially distinct patterns of cancer-immune interaction. A mesenchymal-like niche promoted the recruitment of blood-derived immune cells, whereas another pattern exhibited an immunologically cold phenotype, reminiscent of the immune-privileged healthy brain. Spatially resolved analysis of cell-cell communication within the TME identified the VISTA immune checkpoint pathway as a key regulator of these patterns. Targeting of this axis in a syngeneic mouse model led to significant tumor reduction and 100% survival in mice, highlighting its therapeutic potential. Further validation demonstrated that microglia play a critical role in orchestrating immune cell recruitment and anti-tumor activity, and that modulating the VISTA pathway can reshape the immune response.



 Our findings uncover the spatial and cellular complexity of the DMG immune microenvironment and identify the VISTA pathway as a critical modulator of immune suppression. By demonstrating robust therapeutic efficacy upon VISTA blockade, this study lays a strong foundation for developing targeted immunotherapies in DMG and opens new avenues for overcoming immune resistance in pediatric brain tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3d8c86dfadea2de1a37fb282c8875e761a9bc8" target='_blank'>
              JS07.7.A VISTA IS A CRITICAL AND TARGETABLE IMMUNE CHECKPOINT PATHWAY IN DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            R. Collot, C. Ruiz-Moreno, C. Honhoff, A. Wezenaar, H. Ariese, H. Johnson, B. M. Vervoort, J. Bunt, E. Bokobza, H. G. Rebel, B. T. Te Pas, P. Robe, M. Kool, M. Kranendonk, D. V. van Vuurden, E. Hulleman, E. J. Wehrens, A. Zomer, H. Stunnenberg, A. Rios
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal and immunologically "cold" cancer, with a 5-year survival rate of 13%. Despite the success of immunotherapy in other malignancies, its efficacy in PDAC remains limited due to complex immune evasion mechanisms and a pro-tumorigenic immune microenvironment. Several lines of evidence suggest that PDAC progresses from acinar-to-ductal metaplasia (ADM) to pancreatic intraepithelial neoplasia (PanIN) lesions to invasive cancer, accompanied by increasing immunosuppression. We hypothesize that understanding the spatial and temporal transitions of immune cell states during tumor evolution is key to uncovering mechanisms of immune dysfunction and guiding new therapeutic strategies. Using single-cell RNA sequencing (scRNA-seq) and multispectral imaging, we aim to uncover comprehensive, large-scale, and longitudinal changes in the immune microenvironment during PDAC progression. Here, we utilized the KrasG12D/+; Trp53R172H/+; Ptf1a-Cre (KPC) genetically engineered mouse model, and through longitudinal histological analysis of pancreas, we delineated the trajectory of disease progression, with PanIN I-II lesions appearing by 8–9 weeks, PanIN III by 11–12 weeks, and locally invasive PDAC by 16 weeks of age. To study the temporal evolution of immune cell-state transitions, we performed scRNA-seq on CD45+ enriched immune cells isolated from KPC mouse pancreas collected at 4, 8, 12, and 16 weeks using the 10x Genomics Chromium platform. To examine the spatial dynamics of immune cells, we performed multispectral imaging at defined time points using the Lunaphore COMET platform with a panel of over 15 antibodies, enabling simultaneous visualization of several markers on a single tissue section. In our analysis to date, we identified subsets of pro-tumorigenic macrophages emerging during PDAC development, along with a marked increase in myeloid-derived suppressor cells (MDSCs) in late-stage tumors. We also observed dynamic shifts in other immune cell populations evolving alongside tumor progression. Notably, we uncovered distinct subsets of monocytes, macrophages, and T cells exhibiting tumor-promoting potential, progressively enriched during the advanced stages of PDAC. Using multispectral imaging, we generated a comprehensive spatial map of the immune microenvironment across PDAC progression, capturing patterns of immune exclusion, infiltration, and cell–cell neighborhood interactions. Our study highlights the multi-layered immune dysfunction that evolves from early to late stages of PDAC - features that are challenging to capture in human samples. CD45+ enrichment improved the detection of rare and transient immune subsets by mitigating the masking effects caused by abundant cancer and stromal cells. Together, these findings provide an unbiased and detailed view of the evolving immune landscape in PDAC, laying the foundation for identifying microenvironmental and cell-intrinsic factors that either constrain or promote malignant progression, as well as for developing immune-based therapeutic strategies.



 Atul Verma, Doreen Klingler, Linda Bergmayr, Tanvi V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug. Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0f49d549158981070f5d866ac858fc605d13b" target='_blank'>
              Abstract A106: Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression
              </a>
            </td>
          <td>
            A. Verma, Doreen Klingler, Linda Bergmayr, T. V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="


 Most meningiomas are benign, slow-growing tumors. However, CNS WHO grades 2 and 3 meningiomas display significantly higher aggressiveness and poorer prognosis. Recent investigations into the molecular biology of meningiomas have greatly increased our understanding of these tumors. Furthermore, emergent spatial transcriptomic sequencing methods now provide an unprecedented opportunity to study molecular mechanisms of tumor development within a spatial context and on a single-cell level. This study aims to uncover specific microenvironmental features, cell-cell interactions and transcriptional regulatory mechanisms present in different molecular classes and subclones of meningiomas.



 We employed single-nuclei RNA sequencing and spatial transcriptomic profiling on a matching set of meningiomas (n=22). We developed a method based on non-negative matrix factorization (NMF) to analyze intra-tumor heterogeneity from both single-cell and spatial transcriptional profiles across samples. Trajectory analysis was performed to screen for expression signatures selectively up-regulated around specific morphological features of the tumors. We clustered cells into types of niches based on compositions of their spatial neighborhoods.



 We annotated cell types in the tumor microenvironment (TME), and identified cellular states of meningioma tumor cells in single-cell and spatial transcriptomic data. Tumors of different subclasses display drastic differences in spatial cell state compositions, suggesting distinct modes of microenvironmental organization. We identified conserved patterns of tumor-TME cell state colocalization and interaction. Benign NF2-mutant meningiomas are characterized by extensive infiltration of myeloid cells which trigger a series of immune and stress response pathways in neighboring tumor cells. Among more malignant tumors, a decrease of immune cell populations is coupled with a marked shift in the TME toward tumor-supportive phenotypes. Moreover, we observed reduced tumor-infiltrative capabilities among tumor-associated myeloid cells and the formation of compartmentalized spatial niches of interacting tumor cells. Additionally, various tumor transcriptional signatures related to extracellular matrix remodeling, fibroblastic markers, and meningothelial signatures were mapped to distinct morphological structures across samples.



 Our results reveal that the meningioma TME is heterogenous at the cell state level, with tumor cells and normal cells forming niches that have varying functions in tumor pathophysiology and characterize molecular classes of meningiomas. A series of corresponding RNA markers and spatial signatures were identified, which could potentially be used to increase precision in diagnosis and stratification of meningiomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb7c99c73e50db41072f3f2a1080df073fd8ef43" target='_blank'>
              OS13.6.A A SINGLE-CELL AND SPATIAL ATLAS OF MENINGIOMA HETEROGENEITY ACROSS TUMOR CLASSES
              </a>
            </td>
          <td>
            Y. Tang, R. Banan, S. Maas, D. Calafato, R. Michael, J. Kada Benotmane, R. Yan, G. Rukhovich, Z. Ferdinand, V. Venkataramani, A. Suwala, I. Bludau, S. Krieg, W. Wick, A. von Deimling, D. Heiland, M. Gerstung, B. Patel, F. Sahm
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Spatial tumour heterogeneity, which denotes the changes in cellular and molecular attributes across distinct locations within a tumour, significantly influences cancer diagnosis and treatment resistance. The heterogeneity of tumour cells inside a singular mass facilitates tumour development, metastasis, and the ineffectiveness of standard therapy. Comprehending the geographical distribution of tumour cells is crucial for formulating more efficient treatment regimens. Diverse methodologies are employed to investigate spatial heterogeneity, encompassing modern imaging techniques such as MRI, PET, and multiplexed imaging, alongside omics approaches including genomes, transcriptomics, and proteomics. These instruments offer insights into the tumour microenvironment and facilitate the identification of resistant subpopulations. The amalgamation of imaging and genomic data via radiogenomics has emerged as a viable methodology, providing an extensive perspective on the spatial and molecular intricacies of tumours. Principal findings reveal that spatial heterogeneity fosters medication resistance by establishing microenvironments characterised by varying oxygen levels, immunological infiltration, and genetic alterations, hence complicating the efficacy of monotherapy strategies. Hypoxic environments and immunological evasion significantly contribute to treatment resistance. Addressing geographical heterogeneity has the potential to enhance cancer treatments. By analysing the molecular and geographical characteristics of tumours, physicians can customise therapies more efficiently, minimising resistance and improving therapeutic results. This methodology signifies a vital advancement in precision medicine, providing more individualised and efficacious cancer therapies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/691858c0b45e6f3db66c28fcbc5355f8258e6fea" target='_blank'>
              Spatial Tumor Heterogeneity: The Next Frontier in Understanding Cancer Resistance
              </a>
            </td>
          <td>
            Kwesi Egyin Taylor, Hycent Jacob, Tosin Ayodeji Oladosu, Godwin Kenechukwu Nwajiugo, Motunrayo Victoria Adigun, Markus Saerimam Nzunde, C. Ugo
          </td>
          <td>2025-10-18</td>
          <td>Oncology, Nuclear Medicine and Transplantology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad84fe59ecf17ba1d72e183c30f1361e3c24153f" target='_blank'>
              SMART: A Spatio-Molecular Atlas of Response Trajectories in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            I. Wall, Anthony Baptista, J. Quist, Holly Rafique, Mengyuan Li, Lucy Ryan, G. Verghese, Stefania Marcotti, T. A. Phillips, C. Owczarek, Signe Clausen, Trine Tramm, Graham Booker, Anca Mera, Jasmine Timbres, Elinor Sawyer, Cheryl Gillet, Sheeba Irshad, Sarah Pinder, M. Parsons, Anita Grigoriadis
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="


 The tumour microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC) is comprised of heterogenous malignant epithelium and diverse immune infiltrates within a dense, fibrotic stroma. This multi-compartmental TME includes myeloid and lymphoid lineages, cancer-associated fibroblasts (CAFs), mural and endothelial cells, and tumour cells embedded in a rich extracellular matrix (ECM). Efforts to profile and perturb compartments or cellular phenotypes in isolation have often led to unintended adverse effects on neighbouring cells, compromising drug efficacy. This drives the need for an integrated, spatially-resolved understanding of the architecture and molecular features underlying the organization of the TME. To address this, we leveraged single-cell RNA sequencing (scRNAseq) data from eight distinct datasets to design three custom multiplexed imaging mass cytometry (IMC) panels against tumour, immune, and stromal targets. These were applied to three serial sections of a tissue microarray of 221 PDAC patients. Segmentation and clustering of 7.9 million cells revealed 83 distinct cell types and their functional states, including biophysical responses. Advanced image alignment across serial sections enabled the incorporation of TME-wide features and resolved eight spatially recurrent PDAC microenvironments neighbouring phenotypically distinct epithelial ducts. Two microenvironments, ECM-rich and immune-suppressed, were associated with worse overall survival and frequently co-occurred with a third, stiff matrix, characterized by stroma enriched in phospho-myosin light chain 2 expressing CAFs. In contrast, the immune infiltrated stroma microenvironment was linked to improved clinical outcomes and often co-occurred with CD105+ fibrovascular and immune infiltrated microenvironments. Matched whole-genome sequencing (n = 182/221) revealed that patients dominant in ECM-rich regions harboured 12p11.21 (KRAS) and 8q24.21 (MYC and POU5F1B) amplifications, and 17q22 (RNF43) deletions, and stiff matrix were associated with 13q33.3 (LATS2) deletion. To link spatial niches to proteomic signatures, we performed whole-slide IMC-guided laser capture microdissection and mass spectrometry, and further integrated differentially expressed proteins with transcriptomes from an external scRNAseq cohort (n = 163). ECM-rich and immune suppressed regions showed signatures of mechanically induced immune suppression, including NETosis and innate immune surveillance via pattern recognition receptors. Contrastingly, immune infiltrated microenvironments were enriched for cytotoxic activity and immune trafficking programs, while immune infiltrated stroma displayed signatures associated with oxidative stress responses. These findings define unique spatial ecosystems within the PDAC TME that are linked to specific genomic alterations, functional states and clinical outcomes, highlighting their therapeutic relevance and potential as targets to overcome the challenges of treating PDAC.



 Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Cassandra J. Wong, Brendon Seale, Zhen Yuan Lin, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steven Gallinger, Hartland W. Jackson. Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B115.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e942fb4c97db1a6398873aeae226bf3656a085" target='_blank'>
              Abstract B115: Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, A. Borgida, Cassandra J. Wong, Brendon Seale, Z. Lin, Edward L. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Metastatic non-small cell lung cancer (mNSCLC) cells carry heterogeneity, not only among different subtypes but also within a single tumor. Most evidence suggests that mNSCLC exploits specific molecular drivers and mechanisms to maintain physiology, metabolism, and immune evasion during tumorigenesis. Genome-wide association studies also revealed particular mutations in the oncogenic drivers supporting tumor cell proliferation and survival, resulting in aggressive and drug-resistant phenotypes of mNSCLC. While significant progress has been made in understanding mNSCLC at the genetic and molecular levels, a considerable gap remains in understanding the dynamic interplay between intrinsic factors—particularly key tumor-associated cells—and tumor immune microenvironment (TIME) during metastasis. Hence, this review highlights histological and genetic characteristics, emphasizes the clinical relevance of metastasis, and the roles of tumor-associated cells in shaping the immunosuppressive tumor microenvironment (TME) in mNSCLC. Understanding these intricate features and mechanisms is crucial for identifying novel therapeutic targets and improving strategies to combat mNSCLC progression in diagnosed patients. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/931263caf4b3388e1caa012daaf5acb9be29f764" target='_blank'>
              Unraveling the molecular-pathological characteristics and cellular complexity of the tumor immune microenvironment in metastatic non-small cell lung cancer
              </a>
            </td>
          <td>
            S. Bharadwaj, J. M. Mierzwicka, Lucie Vaňková, Petr Malý
          </td>
          <td>2025-09-29</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 PDAC is the most hypovascular solid tumor, imposing harsh constraints on tumor metabolism. Cancer cells cope with poor access to vascular-derived nutrients and O2 by cooperating with abundant stromal cells, receiving alanine, proline, glycosaminoglycans, branched chain ketoacids, lysophosphatidic acid, and unsaturated fatty acids from fibroblasts. However, the extent of metabolic support provided by other cell types in the PDAC TME is unexplored. We reasoned that metabolically cooperative cell types would be spatially positioned near cancer cells in these tumors, and that cooperative spatial arrangements would be enriched under conditions of relative hypovascularity. We first performed Xenium spatial transcriptomics on a cohort of 8 treatment-naive human PDAC resections and generated a spatial atlas of nearly 700,000 cells across a range of expected cell types. Niche analysis uncovered 17 recurrent cellular neighborhoods in the data, including an endothelial-enriched niche. We find that tumor-associated neutrophils form dense aggregates within malignant glands in tumors with low endothelial content, and form in tumor regions which are most distal from vasculature. In a separate cohort of 4 additional PDAC tumors, the expression of neutrophil chemokines CXCL1/2/3/5 was highest in a subpopulation of malignant cells distant from endothelium. These data suggest metabolic stress secondary to poor vascularity drives recruitment of neutrophils via the CXCL1/2/3/5 axis. Indeed, when exposed to Tumor Interstitial Fluid Media (TIFM), cancer cells upregulated the transcription and/or secretion of these cytokines and G-CSF. Since the formation of neutrophil aggregates is driven by cancer cell nutrient derangement, we hypothesized that cancer cells could cooperate metabolically with neutrophils to alleviate nutrient stress driven by poor vascularity. Neutrophil co-culture promoted cancer cell proliferation under TIFM, but not RPMI, while under lethal combination TIFM and serum starvation, co-culture enabled enhanced survival of cancer cells. Lastly, to understand what metabolic processes could be fueled by neutrophil aggregates in pancreatic cancer, we designed a fully custom spatial transcriptomics panel targeting ∼360 metabolic enzymes and profiled 4 additional PDAC cases, performing dispersed whole-transcriptome scFFPEseq on an adjacent tissue section. We integrated data across the spatial and dispersed modalities to enable spatial projection of whole-transcriptome-derived information. Cancer cells located near neutrophil aggregates had significantly enhanced expression of vitamin B6 salvage pathway, suggesting that one critical shared metabolite between neutrophils and cancer cells is the B6 precursor pyridoxine. In total, we have deployed multiple spatial and single cell RNA profiling technologies to uncover a novel node of metabolic crosstalk in pancreatic cancer between neutrophils and cancer cells. Future work is needed to confirm transfer of specific metabolites and the downstream sequelae of their provision for tumor growth.



 Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M Celeste. Simon. Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec70e5707c093ceceed418e17d0f9e9a3b6c60c" target='_blank'>
              Abstract B127: Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer
              </a>
            </td>
          <td>
            Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M. C. Simon
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e2642fd5880574a79a83aa30d56f3b2134c4d69" target='_blank'>
              Applications of single-cell transcriptomics: updated insights in endometrial cancer.
              </a>
            </td>
          <td>
            Shuyue Xiao, Huixin Li, Jianyao Liu, Xinyi Xie, Hanzi Xu, Zhen Gong, Shanliang Zhong
          </td>
          <td>2025-09-03</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f3009a3214b83a250848d6ef5905a4fb9e1a3db" target='_blank'>
              Unraveling the single cell spatial landscapes of melanoma brain metastases
              </a>
            </td>
          <td>
            A. Manukyan, K. Peters, H. Radbruch, F. Roßner, A. Lehmann, K. Kleo, T. Conrad, J. Altmueller, I. Plumbon, A. Akalin, K. Juerchott, E. Wyler, M. Landthaler, J. Radke, T. Redmer
          </td>
          <td>2025-09-02</td>
          <td>None</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background The tumor microenvironment (TME) is instrumental in facilitating immune evasion and promoting tumor progression in non-small cell lung cancer (NSCLC). However, the spatial heterogeneity and molecular interactions between immune and non-immune cells are not yet fully understood. This research amalgamated single-cell RNA sequencing (scRNA-seq) and spatial transcriptomic data to examine the cellular composition, ligand-receptor (LR) interactions, and immunosuppressive mechanisms of NSCLC. Methods ScRNA-seq was performed on NSCLC tumors and paired adjacent normal tissues to analyze immune and stromal cell heterogeneity. Spatial transcriptomics was applied to consecutive tissue sections to map the distribution of major cell types and their interactions. Cell clustering, a differential gene expression analysis, and LR interaction modeling were used to identify key immune regulatory pathways. In vitro co-culture experiments were performed for functional validation. Results A total of 28,496 high-quality single cells were analyzed, and significant differences in immune cell composition between the tumor and normal tissues were found. The NSCLC tumors exhibited a notable increase in tumor-associated macrophages (TAMs) and fibroblasts, as well as a decrease in the presence of cytotoxic T cells and natural killer (NK) cells. Notably, spatial transcriptomics revealed that the TAMs were predominantly localized in the tumor cores, where they interacted with T cells via immunosuppressive LR pairs. The SPP1-CD44 and NECTIN2-TIGIT axes were identified as major immunosuppressive pathways, with the SPP1 secreted by the TAMs contributing to immune evasion. Additionally, the HLA-E-CD8B interactions were significantly upregulated, suggesting a potential mechanism of antigen presentation modulation. The functional validation showed that SPP1 overexpression (OE) enhanced PDCD1 and CD160 expression, further confirming its role in shaping the immunosuppressive microenvironment. Conclusions This study established a high-resolution immune atlas of NSCLC, identifying novel macrophage-T cell interactions that drive immune suppression. Our findings suggest that targeting the SPP1-CD44, NECTIN2-TIGIT, and HLA-E-CD8B pathways may improve immune responses in NSCLC. These findings provide novel therapeutic targets and highlight the potential for combination immunotherapies to overcome macrophage-mediated immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8150d6dfc08e946211f3a587c3e84a45e521b42" target='_blank'>
              Single-cell and spatial analysis reveals macrophage-T cell crosstalk in non-small cell lung cancer immunosuppression
              </a>
            </td>
          <td>
            Pengyuan Zhu, Zhenchuan Liu, Hatim Husain, Qian Wang, Yongxin Zhou
          </td>
          <td>2025-09-01</td>
          <td>Translational Lung Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5994676ec22291e594d0de0ef39f5ccc905b6ab" target='_blank'>
              Integrative analysis of spatiotemporal transcriptomics delineates dynamic cell states in squamous tumorigenesis
              </a>
            </td>
          <td>
            Varsha Sreekanth, Timur Rusanov, Farah Yousef, Mark A Sanborn, Su Yeon Yeon, Vijayalakshmi Ananthanarayanan, Olga Karginova, Jalees Rehman, A. Salahudeen
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Cancer-associated fibroblasts (CAFs) are critical mediators of tumor-stroma crosstalk in pancreatic ductal adenocarcinoma (PDAC). While bulk and single-cell transcriptomic studies have uncovered CAF heterogeneity, the spatial organization of CAF subtypes and their physical interactions with tumor epithelial subtypes remain incompletely understood. Here, we integrate bulk, single-cell, and spatial transcriptomic datasets to systematically interrogate the molecular, cell, and spatial interactions between tumor-promoting CAFs (proCAFs), tumor-restraining CAFs (restCAFs), and basal-like versus classical tumor subtypes across multiple biological scales. Ligand-receptor analysis of single-cell RNA-seq data revealed robust signaling between proCAFs and basal-like tumor cells, particularly through integrin-mediated (COL11A1–ITGA11) pathways, suggesting subtype-specific stromal-epithelial crosstalk within distinct tumor microenvironments. Histopathologic validation via immunofluorescence staining confirmed that proCAF regions are preferentially located adjacent to tumor epithelium and are characterized by reactive, myxoid stroma, whereas restCAF regions exhibit a fibrous, hypocellular morphology. To further elucidate spatial relationships, we performed 10x Genomics Visium spatial transcriptomics on seven primary PDAC tumors. Application of the DeCAF classifier to Visium spots demonstrated a distinct radial gradient of proCAF expression, peaking at the tumor-stroma interface and diminishing with distance—indicative of spatial colocalization with basal-like programs. Notably, proCAF gene signatures were specifically enriched in stromal regions adjacent to basal-like tumor zones, while classical tumor regions were spatially associated with heterogeneous CAF compositions. This subtype-specific spatial organization and paracrine signaling may underlie the clinical observation that basal-like tumors are significantly more likely to co-occur with proCAF stroma (64.5%, p = 2.17e–5). Importantly, patients harboring both basal-like and proCAF subtypes exhibited the poorest clinical outcomes (median overall survival: 11.0 months), compared to those with classical/restCAF profiles (median overall survival: 30.4 months, p < 0.001). Collectively, these multidimensional data define spatially organized and functionally interactive tumor-stroma niches in PDAC. Our findings highlight a co-evolutionary relationship between proCAFs and basal-like tumor cells that contributes to an aggressive disease phenotype, and underscore the potential of spatially resolved tumor-stroma profiling for improving patient stratification and guiding therapeutic targeting of the tumor microenvironment.



 Xianlu Laura. Peng, Ian McCabe, Elena Kharitonova, Ryan Zhao, Changfei Luan, Arthi Hariharan, Jaewon Lee, Nancy Kren, Yi Xu, Joseph Kearney, Justin Su, Michelle LaBella, Susan Tsai, Brian Belt, Roheena Panni, David Linehan, Yuliya Pylayeva-Gupta, Alina Iuga, Naim Rashid, Jen Jen Yeh. Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B100.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af6682e5a69c7a44f5aef0626bbff15428b91a16" target='_blank'>
              Abstract B100: Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications
              </a>
            </td>
          <td>
            X. Peng, Ian C McCabe, Elena V Kharitonova, Ryan Zhao, C. Luan, Arthi Hariharan, Jaewon J Lee, Nancy P. Kren, Yi Xu, Joseph F Kearney, Justin Su, M. LaBella, Susan Tsai, Brian Belt, R. Panni, David Linehan, Y. Pylayeva-Gupta, Alina C Iuga, Naim U Rashid, J. J. Yeh
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d74e30f87b2d3e252ed12332d86f09f04feb4c91" target='_blank'>
              CTDP: Identifying cell types associated with disease phenotypes using scRNA-seq data
              </a>
            </td>
          <td>
            Chonghui Liu, Murong Zhou, Zhidong Wang, Chen Yang, Yan Zhang, Zhongjun Jiang, , Zeyu Luo, Tianjiao Zhang, Guohua Wang, Lei Yuan
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cardiac myxoma, the most common primary heart tumor, remains poorly understood at the molecular level. Here, we combined single-nucleus RNA sequencing, third-generation transcriptomics, and untargeted metabolomics to dissect its origin and pathology. Single-cell analyses demonstrate an endothelial origin driven by aberrant endothelial-to-mesenchymal transition (EndMT), with pseudotime and RNA-velocity tracing a continuum from endothelial-like to mesenchymal-like and metabolically active states. We identify two distinct myxoma subtypes: Subtype 1, marked by MAPK/WNT/EGFR pathway activation, and Subtype 2, characterized by ribosomal and oxidative phosphorylation signatures alongside immune-evasive programs. Third-generation data highlight extracellular matrix remodeling and endothelial signaling, while metabolomics reveal dysregulated purine, nicotinic acid, and nicotinamide metabolism. Notably, MET–PTK2 signaling emerges as a potential driver of tumor initiation and progression. These integrated findings define the cellular architecture and metabolic adaptations of cardiac myxoma and lay the foundation for future interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0702495e091268eccb0b45edf83576207a5dbb0c" target='_blank'>
              Multi-omics data reveal the origin of cardiac myxoma
              </a>
            </td>
          <td>
            Shengzhong Liu, Wanfeng Zhang, Huajun Sun, Chenqing Zheng, Keli Huang, Chengming Fan, Rensheng Lai, Mingzhu Yin, Jie Lan, Xiushan Wu, Longke Ran, Xiaoping Li
          </td>
          <td>2025-09-26</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in shaping immune responses and therapeutic outcomes in lung cancer, yet the diversity and functional specialization of tumor-associated macrophages (TAMs) remain poorly resolved. Here, we present a refined classification of TAM subtypes across large cohorts of cancer datasets using integrative analysis of single-cell RNA sequencing, spatial transcriptomics, and clinical datasets from lung adenocarcinoma and lung squamous cell carcinoma. By combining cell-based gene scoring with hierarchical classification, we defined 7 macrophage subtypes-each with distinct transcriptional programs and abundances. Notably, lipid-associated TAMs expand with disease progression and exhibit immunosuppressive and protumorigenic features, whereas tissue-resident macrophages decline. Spatial and survival analyses reveal that an increased lipid-associated to tissue-resident TAM ratio correlates with advanced disease and poor prognosis. Given that spatial transcriptomic assays rely on deconvolution techniques to infer cell type compositions, accurate gene expression signatures are essential, especially for fine-grained subpopulations of TAMs. Our refined subtype-specific signatures address this bottleneck and enhance the resolution of spatial mapping efforts. These findings offer new insights into macrophage heterogeneity and highlight lipid-associated TAMs as potential biomarkers and therapeutic targets in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e49c52a60b0a9da9a722e72cb8391f58dc458b" target='_blank'>
              Transcriptional states of lung cancer microenvironment reveal macrophage subtype dynamics linked to disease progression.
              </a>
            </td>
          <td>
            Duygu Keremitçi, Özlem Tuna, Aissa Houdjedj, Hilal Kazan, Yasin Kaymaz
          </td>
          <td>2025-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/781db8d3b9951c3d4f119253d55c69b0ec7522ab" target='_blank'>
              Hybrid In Vivo Breast Cancer Model Reveals Transcriptomic Insights into Cancer Progression with Age
              </a>
            </td>
          <td>
            Lauren Hawthorne, Gokhan Bahcecioglu, Jun Yang, Emilija Aleksandrovic, Erin Howe, Siyuan Zhang, P. Zorlutuna
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous malignancy, with its tumor immune microenvironment (TIME) playing a crucial role in tumor progression, immune evasion, and treatment response. Despite insights from single-cell technologies, challenges such as small samples and inconsistent annotations have limited a comprehensive understanding. Thus, a large-scale, high-resolution single-cell atlas is needed to fully explore the heterogeneity of CRC TIME and guide personalized treatment. Methods We analyzed single-cell transcriptomic data from 100 CRC samples (371,223 cells). Immune, stromal, and epithelial cells in the TIME were clustered, annotated, and functionally analyzed. Transcription factor networks were inferred using SCENIC, differentiation states assessed with CytoTRACE, and cell–cell communication analyzed via CellPhoneDB. TIME subtypes were constructed from bulk RNA-seq data, followed by survival analysis. SCRN1 was further validated through in vitro experiments. Results This study presents a comprehensive single-cell CRC atlas, revealing significant heterogeneity in T cells, B cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells within the TIME. We defined two immune ecological subtypes: subtype 1 (enriched in metabolic and motility pathways with poor prognosis) and subtype 2 (enriched in immune response pathways, with better prognosis and greater immunotherapy potential). High SCRN1 expression in malignant epithelial cells correlated with poor prognosis and metastasis. In vitro experiments confirmed the role of SCRN1 in promoting CRC cell proliferation and migration. Conclusions This study provides a comprehensive characterization of the CRC TIME, links TIME subtypes to prognosis, and identifies SCRN1 as a potential therapeutic target, offering insights for CRC precision subtyping and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04173-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c5be539fad2c0073ae6e62159eaef1e83a1ef" target='_blank'>
              The single-cell atlas of colorectal cancer reveals cellular heterogeneity and key molecular features of the tumor immune microenvironment
              </a>
            </td>
          <td>
            JunHu Li, ZhiQiang Liu, BoYu Zhang, Liuping You, Yuenan Huang
          </td>
          <td>2025-10-10</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy with significant cellular heterogeneity, which poses a significant challenge for treatment. We integrated three single-cell RNA sequencing (scRNA-seq) datasets to construct an expression matrix, identify cell types, and assess copy number variation (CNV) scores, with a particular focus on ductal cells. Our study found that ductal cells are the primary source of malignant cells in PDAC and identified a high-malignancy subtype—ductal cell-3—where RACK1 was the most significantly differentially expressed gene. Through the analysis of public and clinical data, along with RT-qPCR and immunohistochemistry validation, we confirmed the high expression of RACK1 in PDAC tumors, which correlated with lymph node metastasis, clinical staging, histological differentiation, and prognosis. Further investigation revealed that RACK1 promotes PDAC cell proliferation, migration, and invasion. Sequencing analysis indicated that RACK1 inhibits the FOXO signaling pathway, promotes epithelial-mesenchymal transition (EMT), and facilitates the progression and metastasis of PDAC. This study provides new insights into the role of RACK1 in PDAC and its underlying mechanisms. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03679-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fac327fd52a6595ab14fc1ae00b1f2b7a0bcb1e" target='_blank'>
              Molecular mechanisms of RACK1-driven metastasis in pancreatic ductal adenocarcinoma revealed by single-cell RNA sequencing
              </a>
            </td>
          <td>
            Hangyu Liu, Tingxin Wang, Yuying Cui, Biao Zhang, Dong Shang, Huiyi Song
          </td>
          <td>2025-10-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Differentiating between the two main subtypes of Inflammatory Bowel Disease (IBD): Crohn’s disease (CD) and ulcerative colitis (UC) is a persistent clinical challenge due to overlapping presentations. This study introduces a novel computational framework that employs spatial transcriptomics (ST) to create an explainable machine learning model for IBD classification. We analyzed ST data from the colonic mucosa of healthy controls (HC), UC, and CD patients. Using Non-negative Matrix Factorization (NMF), we first identified four recurring cellular niches, representing distinct functional microenvironments within the tissue. From these niches, we systematically engineered 44 features capturing three key aspects of tissue pathology: niche composition, neighborhood enrichment, and niche-gene signals. A multilayer perceptron (MLP) classifier trained on these features achieved an accuracy of 0.774 ± 0.161 for the more challenging three-class problem (HC, UC, and CD) and 0.916±0.118 in the two-class problem of distinguishing IBD from healthy tissue. Crucially, model explainability analysis revealed that disruptions in the spatial organization of niches were the strongest predictors of general inflammation, while the classification between UC and CD relied on specific niche-gene expression signatures. This work provides a robust, proof-of-concept pipeline that transforms descriptive spatial data into an accurate and explainable predictive tool, offering not only a potential new diagnostic paradigm but also deeper insights into the distinct biological mechanisms that drive IBD subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7e36d179c516512e6f46f88d6a580c0cc5c9f06" target='_blank'>
              Engineering Spatial and Molecular Features from Cellular Niches to Inform Predictions of Inflammatory Bowel Disease
              </a>
            </td>
          <td>
            M. J. T. Tan, Maria Kapetanaki, P. Benos
          </td>
          <td>2025-09-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ovarian cancer represents a typically immune “cold” tumor, where obvious immunosuppression, spatial T-cell exclusion, and cellular dysfunction collectively limit immunotherapy effectiveness. Especially in high-grade serous ovarian carcinoma (HGSOC), the immune low-response state is driven by complex interactions among tumor-associated macrophages (TAMs), suppressive stromal networks, and the T-cell compartment (regulatory T cells, Tregs, and exhausted effector T cells). Emerging multi-omics technologies—particularly single-cell RNA sequencing and spatial transcriptomics—have showed the heterogeneity and spatial immune organization underlying this suppressed state. Here, we integrate these datasets to describe TAM phenotypes and spatial niches, T-cell exhaustion, Tregs accumulation, NK-cell dysfunction, and stromal barriers that enforce exclusion. We then derive phenotype-guided combination strategies to remodel the tumor microenvironment and improve responsiveness to immune checkpoint blockade. This synthesis provides a concise, multi-dimensional framework for precision immunotherapy and for overcoming resistance in immune-low ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880f6c405f773eb38fd078c68256074438417fa2" target='_blank'>
              Decoding immune low-response states in ovarian cancer: insights from single-cell and spatial transcriptomics for precision immunotherapy
              </a>
            </td>
          <td>
            Yalan Yan, Jiaan Lu, Huanyu Luo, Zizhang Wang, Ke Xu, Lexin Wang, Qin Wang
          </td>
          <td>2025-09-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566fc9fd9c1a3c438b8fdc4f08cbb7f2ae09af61" target='_blank'>
              Joint analysis of expression and variation at single cell resolution by scVar
              </a>
            </td>
          <td>
            Wei Zhao, Fei Yang, Wenting Zong, Xinchang Zheng, Yiming Bao
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer characterized by a lack of hormone receptors, making it challenging to treat. Methods We generated a comprehensive spatial cell atlas of TNBC using a multi-omics integration approach that combined single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics. This integration allowed us to characterize the spatial microenvironment and map the cell-type-specific distributions in TNBC tissues. Results Our analysis revealed significant heterogeneity in cell types and spatial distribution, with normal regions enriched in insulin resistance functions, whereas cancerous regions displayed diverse cell populations, including immune cells, cancer-associated fibroblasts (CAFs), and mesenchymal cells. By constructing transcription factor (TF) regulatory networks, we identified TFF3, RARG, GRHL1, RORC, and KLF5 as critical regulators of epithelial cells, whereas EMX2, TWIST1, TWIST2, NFATC4, and HOXC6 were found to play essential roles in mesenchymal cells. Immunohistochemical validation supported the involvement of these TFs in TNBC. Further analysis of receptor-ligand interactions highlighted the roles of KNG1_BDKRB2 and NRG1_ERBB4 signaling in promoting tumor aggression, suggesting potential therapeutic targets. GO enrichment analysis revealed overlapping pathways between epithelial and mesenchymal cells, focusing on migration, signaling, and development, indicating that the shared regulatory mechanisms contribute to cancer progression. Conclusion Our findings provide new insights into the TNBC microenvironment, emphasizing the complex spatial interactions between different cell types and highlighting key regulatory pathways that could be targeted for future therapeutic interventions. This spatial cell atlas lays the foundation for further exploration of tumor microenvironment dynamics and precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8697e1d4f457ce1a990f5f037a0dd30fbd923ee5" target='_blank'>
              Multi-omics integration uncovers key transcriptional regulators in triple-negative breast cancer spatial heterogeneity
              </a>
            </td>
          <td>
            Ning Zhang, Ye Zhang, Rui-Fei Yang, Min Tan, Hai-Qing Chu, Abdur Rehman, Lu-Yu Yang, Ya-Yu Li, Fahad M. Alshabrmi, Xin Zhou, Feili Xu, Shou-Ping Gong, Hui-Ling Cao
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Purpose The dysregulated immune microenvironment represents a key pathogenic driver in psoriatic lesions. However, the intricate cellular and molecular interactions underlying psoriasis remain incompletely elucidated. Therefore, we aim to employ integrated multi-omics approaches to characterize the immune microenvironment and pathogenic niche in psoriasis, thereby elucidating the cellular and molecular mechanisms of disease pathogenesis. Methods Integrated Single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and bulk RNA sequencing (RNA-seq) data to explore the heterogeneity of stromal cells and immune cells in psoriatic lesions and the complex spatial niches formed between them. Enrichment analysis, intercellular communication analysis, and spatial co-localization analysis were used to investigate the transcriptional changes and distribution characteristics of each cell type in the lesions of psoriasis patients. Results Using scRNA-seq, we identified a novel CD4+ tissue-resident memory T cell (TRM) subset that is exclusively present in lesional skin of psoriasis patients but absent in healthy skin. These cells exhibit elevated expression of genes including IL17RA, IL22, PD1 (PDCD1), CXCR6, ITGAE, CD69, TNFRSF9, TNFRSF4, IL7R, CD4, and STAT3. Additionally, we discovered a novel microvascular endothelial cell subset, designated Venous endo2, which highly expresses CD93, ACKR1, ICAM1, VCAM1, IL15, SELE, and SELP, while also overlapping with high endothelial venule (HEV)-associated transcriptional signatures. Integrated analysis of scRNA-seq and spatial transcriptomics further revealed strong spatial co-localization of Venous endo2 with fibroblast activation protein-positive fibroblasts (FAP+ Fbs), T cells, and antigen-presenting cells (APCs) in Psoriasis lesions—a pattern not observed in healthy control skin. Conclusion Through integrated multi-omics analysis, we identified a potential pathogenic niche in psoriasis patients, composed of Venous endo2, FAP+ Fbs, T cells, and APCs. This structure resembles tertiary lymphoid structures (TLS), suggesting a functional parallel in disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d8eeb3b234a229f3a9f78e22b42dca0befb2a94" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals a Pathological Niche Formed by FAP+ Fibroblasts, Immune, and Endothelial Cells in Psoriatic Lesions
              </a>
            </td>
          <td>
            Wuwei Zhuang, Qi Zhang, Qingtao Kong, Yun Hui, Jie Shen, Chen Zhang, Hong Sang, Qiao Ye
          </td>
          <td>2025-09-01</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f4d390efc07199143e937f1fd95a94ae1c92efa" target='_blank'>
              Resolving cDC2 heterogeneity across human cancer atlases
              </a>
            </td>
          <td>
            Nikita Rosendahl, Xiaohan Xu, Norman Teik-Wei Yap, Z. Tuong, Kristen J Radford
          </td>
          <td>2025-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38ea6290df028815fb911512fc1b477390936835" target='_blank'>
              Pan-Cancer Single-Cell Profiling Uncovers the Biological Characteristics of Cancer-Testis Genes
              </a>
            </td>
          <td>
            Chunyang Fu, Ke Liu
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05542f754678927320ac35e36879b2df7142998c" target='_blank'>
              scXpand: Pan-cancer detection of T-cell clonal expansion from single-cell RNA sequencing without paired single-cell TCR sequencing
              </a>
            </td>
          <td>
            Ofir Shorer, Ron Amit, Keren Yizhak
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02937-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08665b0e37751f05ffbba7515282ba6d54a5cdc8" target='_blank'>
              Multiomic profiling of a unique in-transit melanoma cohort identifies melanoma differentiation as predictor of tumor progression and therapy response
              </a>
            </td>
          <td>
            Giuseppe Tarantino, Anne Zaremba, Tuulia Vallius, M. Woodnorth, Yichao Hua, R. Pelletier, Mariana Lopez Leon, Yingxiao Shi, Z. Maliga, Samira Makhzami, Tyler J. Aprati, Bojan Karlaš, V. Glutsch, Bastian Schilling, J. Hassel, Carola Berking, Jochen Utikal, F. Meier, F. Meiss, Lucie M. Heinzerling, Katharina Kähler, Jiajia Chen, L. Zimmer, A. Sucker, E. Livingstone, E. Hadaschik, Christine Lian, George Murphy, Yevgeniy R. Semenov, Genevieve M Boland, P. Sorger, Florian Rambow, David Liu, Dirk Schadendorf
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Tumor resident microbes are a well-recognized component of the tumor microenvironment. Microbial subcellular location within different tumor types along with their functionality remains to be determined. Bulk microbial profiling techniques, however, lack subcellular and spatial transcriptomic resolution. To address these limitations, we developed advanced quantitative fluorescent imaging methodologies that allowed for visualization of microbial cellular compartmentalization across different tumor types. Furthermore, we performed spatial microbial transcriptomics at the regional and single cell levels to interrogate microbial regulation of signaling within pancreatic cancer cells. To confirm presence of viable microbes within tumors, we performed multiplexed culturomics of patient tissue samples followed by Whole Genomic Sequencing (WGS) analysis. We used in vitro co-culture assays together with in vivo animal experiments for functional biological validation. Overall, our results identified several pathways under microbial regulation within cancer cells that can drive immune evasion through impaired antigen presentation. In summary, using multiple complimentary novel methodologies we characterize the microbial niche of tumors (MiNT) that uncovers microbial regulation of host cells signaling and reshaping of the surrounding tumor microenvironment.



 Vidhi Chandra, Le Li, Seyda Baydogan, Haoyue Liu, Javier Gomez, Steven B. Maron, Daniel Carvajal-Hausdorf, Erick M. Riquelme, Mark W. Hurd, Anirban Maitra, Luis Diaz, Ismet Sahin, Jared Burks, Michael P. Kim, Florencia McAllister. Functional interrogation of the pancreatic intratumoral microbial niche reveals tumor modulation and reshaping of the microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B011.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9d435ae53244dd8930ff7ec47bc24ea3f53880a" target='_blank'>
              Abstract B011: Functional interrogation of the pancreatic intratumoral microbial niche reveals tumor modulation and reshaping of the microenvironment
              </a>
            </td>
          <td>
            Vidhi Chandra, Le Li, Seyda Baydogan, Haoyue Liu, Javier A Gomez, S. Maron, Daniel Carvajal-Hausdorf, Erick Riquelme, M. Hurd, Anirban Maitra, Luis A. Diaz, I. Sahin, Jared K. Burks, Michael P Kim, Florencia McAllister
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Spatial variable genes (SVGs) reveal critical information about tissue architecture, cellular interactions, and disease microenvironments. As spatial transcriptomics (ST) technologies proliferate, accurately identifying SVGs across diverse platforms, tissue types, and disease contexts has become both a major opportunity and a significant computational challenge. Here, we present a comprehensive benchmarking study of 20 state-of-the-art SVG detection methods using human slides from STimage-1K4M, a large-scale resource of ST data comprising 662 slides from more than 18 tissue types. We evaluate each method across a range of biologically and technically meaningful criteria, including recovery of pathologist-annotated domain-specific markers, cross-slide reproducibility, scalability to high-resolution data, and robustness to technical variation. Our results reveal marked differences in performance depending on tissue type, spatial resolution, and study design. Beyond benchmarking, we construct the first cross-tissue atlas of SVGs, enabling comparative analysis of spatial gene programs across cancer and normal tissues. We observe similarities between pairs of tissues that reflect developmental and functional relationships, such as high overlap between thymus and lymph node, and uncover spatial gene programs associated with metastasis, immune infiltration, and tissue-of-origin identity in cancer. Together, our work defines a framework for evaluating and interpreting spatial gene expression and establishes a reference resource for the ST community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ff70f866316ad376e66f985978ce8e7239e3c0" target='_blank'>
              Large-scale spatial variable gene atlas for spatial transcriptomics
              </a>
            </td>
          <td>
            Jiawen Chen, Jinwei Zhang, Dongshen Peng, Yutong Song, Aitong Ruan, Yun Li, Didong Li
          </td>
          <td>2025-10-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genetically engineered mouse models have advanced cancer research but often fail to capture key features of certain human tumors. Rats, with distinct physiology and tumor biology, offer a powerful alternative, yet their use has been constrained by technical barriers to genome editing. Here, we report efficient somatic genome editing in rats, enabling both Indel and substitution mutations. We then apply this approach to model estrogen receptor (ER)-positive breast cancer, which accounts for ∼70% of human cases but remains poorly represented in mice. The resulting rat tumors reproduce hallmarks of human ER+ breast cancer, including ductal histology, hormone responsiveness, and immune-microenvironmental features. By contrast, identical genetic alterations in mice failed to yield ER+ tumors, underscoring critical species differences in tumorigenesis. Together, this work establishes a versatile platform for rapid generation of clinically relevant rat tumor models, opening new avenues to dissect tumor biology, therapeutic response, and immune interactions in previously inaccessible cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19abed2134c7009bf0720ace89b27aa8404fff98" target='_blank'>
              Rat somatic genome editing enables ER+ breast cancer modeling
              </a>
            </td>
          <td>
            Wen Bu, Tobie Lee, M. Anurag, Yunfeng Ding, Ruixin Xu, Lillian He, Alexandria Z. Bu, C. Nagi, Carolina Gutierrez, S. Hilsenbeck, Chonghui Cheng, Bing Zhang, Shixia Huang, Jianming Xu, C. K. Osborne, Arun Sreekumar, Eric Chang, Chad J. Creighton, X. Zhang, Yi Li
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Pancreatic cancer, specifically pancreatic ductal adenocarcinoma (PDAC), is notorious for its aggressive nature and dismal prognosis, ranking as a leading cause of cancer-related mortality worldwide. Despite advancements in surgical techniques, chemotherapies, and novel therapeutic approaches, the overall survival rates remain bleak, primarily due to the complex and dynamic tumor microenvironment (TME) which fosters cancer progression and therapeutic resistance. Central to the TME are cancer-associated fibroblasts (CAFs), which exhibit significant heterogeneity and play multifaceted roles in tumor progression, ranging from remodeling the extracellular matrix to modulating immune responses. Recent breakthroughs in single-cell sequencing and spatial omics have unraveled the diverse functional states and spatial organization of various cell types within the TME, including distinct subtypes of CAFs which differ vastly in their influence on pancreatic cancer pathophysiology. This review delves into the intricate interactions between CAFs and pancreatic cancer cells, highlighting how metabolic reprogramming and immune regulation by CAFs contribute to the high malignancy of PDAC. By integrating single-cell and spatial resolution data, we offer new insights into the metabolic crosstalk and immune landscape remodeling driven by CAFs, establishing a foundation for targeting these stromal interactions in therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2add2bd80d02d372a76aab262dddc13647852fe6" target='_blank'>
              Metabolic and immune crosstalk between cancer-associated fibroblasts and pancreatic cancer cells
              </a>
            </td>
          <td>
            Qiyao Zhang, Z. Cao, Shangcheng Yan, Bingrong Chen, Hao Wu, Hongtao Cao, Chen Lin, Ziwen Liu
          </td>
          <td>2025-10-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic cancer remains a lethal disease despite advances in surgery and systemic treatment in the last two decades, underscoring the urgent need to better understand its biological underpinnings. Despite remarkable advances in the molecular characterization of pancreatic ductal adenocarcinoma (PDAC), clinically actionable biomarkers remain scarce, and current treatment remains empiric. Transcriptomic subtypes such as “classical” and “basal-like” offer some prognostic value, but their ability to guide real-time treatment decisions is limited. In this review, we explore the limitations of current biomarker strategies, in particular subtype-based classifications, and argue for a functional reframing of biomarker development in PDAC, centered on patient-derived organoids (PDOs). We explore four key domains in which PDOs deepen our understanding of therapeutic response and resistance, namely, drug response phenotyping, modeling chemoresistance, incorporating tumor microenvironmental complexity through co-culture systems, and more functional profiling through proteomic and metabolomic approaches. Together, these applications move PDOs beyond static avatars of the tumor to dynamic platforms capable of capturing clinically relevant biology. As functional precision medicine gains traction, PDOs may offer a path to more tailored, responsive treatment strategies in a cancer where new options are urgently needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243db33c2c4b51c3c745a2973296c1c4099759e6" target='_blank'>
              Functional Precision in Pancreatic Cancer: Redefining Biomarkers with Patient-Derived Organoids
              </a>
            </td>
          <td>
            C.A. Chew, Cheng Mun Wun, Yi Fang Lee, C.E. Chee, Khek Yu Ho, Glenn Kunnath Bonney
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Uterine leiomyosarcoma (ULSA) is a highly aggressive gynecologic malignancy characterized by early metastasis, profound immunosuppression, and resistance to conventional therapies, including immune checkpoint blockade (ICB). The intricate tumor microenvironment (TME) and cellular heterogeneity driving its progression and therapy resistance remain poorly defined. Methods We performed single-cell RNA sequencing (scRNA-seq) on metastatic lesions (pelvic cavity, rectum, peritoneum, bladder) from a treatment-naïve ULSA patient and compared them to normal uterine myometrium, MMM (n=5). Integrated analyses included cellular composition mapping, copy number variation (CNV) assessment, pseudotemporal trajectory reconstruction, cell-cell communication inference, functional enrichment, and validation via multiplex immunofluorescence (mpIF). Survival correlations were assessed using the TCGA-SARC cohort. Results In this study, the main finding is that the tumor microenvironment (TME) has a strong immunosuppressive effect. Firstly, its characteristic is exhausted CD8+T cells. This study found that as time progresses, the initial cell markers (CCR7, MAL) gradually disappear, while the exhaustion markers (LAG3, HAVCR2, TIGIT) are enriched. This is associated with poor prognosis. Secondly, the M2-polarized macrophages are mainly composed of M2-like tumor-associated macrophages (TAMs) with tumor-promoting characteristics (CD163, FTH1, FTL, TIMP1), and there is a polarization from M1 to M2. Finally, the immature, tumor-promoting N2 neutrophils (CD15+EDARADD+) enriched in the metastatic foci are associated with poor prognosis. The cell communication involves the interaction of MIF-(CD74+CD44) between T/B cells, as well as the role of the CXCL8 signaling axis in promoting angiogenesis, TAM polarization, and immunosuppression. Conclusion For the first time, a comprehensive single-cell map of ULSA was constructed, depicting a metastasis-susceptible cell subset (U11-EDARADD) and an extremely immunosuppressed tumor microenvironment dominated by depleted CD8+T cells, M2 macrophages and N2 neutrophils. These features shed light on the underlying mechanisms of chemotherapy resistance and immunotherapy failure. The biomarkers identified here (EDARADD, CLDN10, TMIGD2) as well as the dysregulated pathways (TGF-β, angiogenesis, MIF signaling) provide possible targets for future development of combined immunotherapy strategies against this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15b01f016db1d6adc66dea5efac6b72384978f8f" target='_blank'>
              Single-cell profiling delineates the tumor microenvironment and immunological networks in patient-derived uterine leiomyosarcoma
              </a>
            </td>
          <td>
            Yi Guo, Dongsheng Shen, Yuhang Xiao, Chenghao Wu, Meiyi Chen, Lina Yang, Huaifang Li, Xiaowen Tong, Rujun Chen, Fang Li
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality worldwide. However, the spatial and temporal dynamics underlying its development remain poorly characterized. This study employs spatial transcriptomics (ST) to investigate the progression of intestinal tumors in APC Min/+ mice across multiple time points. We identified distinct transcriptional profiles between tumor and normal tissues, resolving six major cell types through integrated dimensionality reduction and pathological annotation. Pseudo-time trajectory analysis revealed increased expression of MMP11 and MYL9 in later stages of tumor progression. Analysis of human CRC cohorts from the TCGA database further confirmed that high expression of these genes is associated with advanced clinical stages and promotes tumor proliferation and invasion. Temporal gene expression dynamics indicated enrichment of cancer-related pathways concurrent with suppression of lipid and amino acid metabolism. Notably, genes in the DEFA family were significantly upregulated in normal tissues compared to tumor tissues. Functional validation showed that DEFA3 inhibits colon cancer cell migration and proliferation in vitro. These demonstrate the value of ST in resolving spatiotemporal heterogeneity in CRC and identify both MMP11/MYL9 and DEFA3 as potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a441608f72da7375d3ddfdc034535ffa428b9ffc" target='_blank'>
              Investigating the Development of Colorectal Cancer Based on Spatial Transcriptomics
              </a>
            </td>
          <td>
            Zhaoyao Qi, Guoqing Gu, Huanwei Huang, Beile Lyu, Yibo Liu, Wei Wang, Xu Zha, Xicheng Liu
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is characterized by profound intratumoral heterogeneity and cellular plasticity, which drive therapeutic resistance. While molecular subtyping into classical and basal-like states has provided a framework, the biology of cells transitioning between these phenotypes remains poorly understood. This study aimed to characterize and validate a transitional, co-expressor cell state at the single-cell, spatial, and proteomic levels to elucidate mechanisms of subtype plasticity. We employed an integrative, multi-platform approach combining single-cell RNA sequencing (scRNA-seq), multiplex immunohistochemistry (mIHC), and laser microdissection with mass spectrometry-based proteomics (LMD-MS) to investigate tumor cells co-expressing the classical marker GATA6 and the basal-like marker KRT17. scRNA-seq analysis revealed that a significant population of tumor cells co-expresses GATA6 and KRT17, existing along a transcriptional continuum between the classical and basal-like poles. Pseudotime trajectory analysis identified a path from the GATA6+ classical state towards the KRT17+/GATA6+ transitional state, governed by the upregulation of pathways including MYC signaling and epithelial-mesenchymal transition (EMT). To validate these findings spatially, mIHC confirmed the presence of KRT17+/GATA6+ double-positive cells, frequently located at the interface between well-differentiated, GATA6-rich glandular structures and poorly differentiated, KRT17-rich infiltrative fronts. Furthermore, LMD-MS was used to provide proteomic confirmation. By isolating pure populations of GATA6+ and double-positive cells, we confirmed that transitional cells are enriched in proteins associated with metabolic reprogramming and cellular motility. Our findings demonstrate that PDAC plasticity is not a simple switch but a dynamic continuum involving a tangible transitional cell state. These co-expressor cells, with their unique metabolic and signaling dependencies, represent a key driver of tumor progression and a potential reservoir for therapeutic resistance. Targeting the specific vulnerabilities of this transitional phenotype offers a promising strategy to overcome plasticity, prevent relapse, and improve therapeutic outcomes in PDAC.



 Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, Natalia Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth Shroyer. Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B122.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a10f45ec5eb895ff99dbba41e30b63f2ad6188" target='_blank'>
              Abstract B122: Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, N.Ye Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth R. Shroyer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive disease with one of the lowest five-year survival rates of all cancers. Low grade, premalignant pancreatic intraepithelial neoplasias (PanINs) are common in the general population, but despite expression of mutant KRAS, the vast majority of these lesions will never progress to cancer. Understanding how additional genetic or epigenetic changes as well as changes to the tumor microenvironment contribute to PanIN to PDAC progression is critical and has the potential to reveal mechanisms of interception for this deadly disease. Our previous work identified the prolyl isomerase PIN1 as a critical factor in the tumor microenvironment for growth of a subset of pancreatic cancer cell lines. Loss of PIN1 in the tumor microenvironment restricted in vivo growth of multiple KPC cell lines. We identified distinct fibroblast phenotypes in the absence of microenvironmental PIN1 expression or function. These changes in fibroblast state drive significant changes in the extracellular matrix and growth factor expression, both of which impact the crosstalk to and behavior of the neoplastic cells. We are continuing to use conditional PIN1 knockout mice to interrogate the specific cell types responsible for the observed tumor-restrictive phenotype and are using single cell sequencing to better understand how changes to individual cell types affect multiple cell populations to inhibit tumor growth. In addition, we are comparing the fibroblast phenotypes observed in our mouse models to those found in tumor-promoting versus tumor-restrictive microenvironments in human tissues. These tissues include autopsy pancreata with low grade PanIN lesions as well as PDAC resection specimens with low and high grade PanINs. We are creating organoid and cell line models of epithelial and stromal cells from normal human pancreata to support complex co-culture engineered models of early disease progression. By integrating across mouse and human model systems, we strive to understand the crosstalk that permits progression of pancreatic cancer and to identify targets to intercept this process.



 Tana Gazdik, Madeline Kuhn, Chloe Bowman, Eric Carlson, Kyra Lindley, Ellen M. Langer. Interrogating microenvironmental changes that impact early pancreatic cancer development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4475acff7b0cbed38743a8484c9c3b1b2adb82c" target='_blank'>
              Abstract A031: Interrogating microenvironmental changes that impact early pancreatic cancer development
              </a>
            </td>
          <td>
            Tana Gazdik, Madeline R Kuhn, Chloe L Bowman, Eric J Carlson, Kyra A Lindley, Ellen M. Langer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The ability of metastatic cancer cells to invade distant tissues requires them to cross a variety of tissue boundaries, each posing distinct structural and biochemical challenges. In particular, the boundary between dense, extracellular matrix (ECM)-rich tumor tissue and surrounding stromal tissue is associated with phenotypic changes in MDA-MB-231 breast cancer cells following transmigration, including increased invasiveness and aggressiveness. It remains unclear whether this transition arises from selective, permissive filtering of pre-existing subpopulations, such as cancer stem cells, or an instructive response of the entire cell population. Here, by combining single-cell migration analysis, heterogeneity analysis of cell proliferation, and computational modeling, it is demonstrated that tumor-tissue boundaries act as instructive interfaces. Using an established 3D fibrillar collagen I matrix model of interfaces, it is shown that all cells can transmigrate the interface with no evidence of selective filtering of subpopulations. Proliferation heterogeneity remains unchanged between transmigrated and non-transmigrated cells, further supporting an instructive mechanism. Simulations confirm that the interface instructively modulates cell behavior. These results indicate that tissue boundaries can reprogram cancer cell phenotypes, representing a potentially targetable mechanism in metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aace3b21099f4dc0398d90f3d8cd70a0bc5a03df" target='_blank'>
              Tumor-Tissue Boundaries as Instructive Interfaces in Breast Cancer Cell Invasion.
              </a>
            </td>
          <td>
            Cornelia Clemens, T. Zerjatke, Andrine Frank, Hannah Trampert, Nataliia Kotsiuba, I. Glauche, T. Pompe
          </td>
          <td>2025-09-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Chemotherapy resistance remains a formidable challenge to the treatment of high-grade serous ovarian cancer (HGSOC). The drug-tolerant cells may originate from a small population of inherently resistant cancer stem cells (CSCs) in primary tumors. In contrast, sufficient evidence suggests that drug tolerance can also be transiently acquired by nonstem cancer cells. Regardless of the route, key regulators of this plastic process are poorly understood. Here, we utilized multiomics, tumor microarrays, and epigenetic modulation to demonstrate that SOX9 is a key chemo-induced driver of chemoresistance in HGSOC. Epigenetic upregulation of SOX9 was sufficient to induce chemoresistance in multiple HGSOC lines. Moreover, this upregulation induced the formation of a stem-like subpopulation and significant chemoresistance in vivo. Mechanistically, SOX9 increased transcriptional divergence, reprogramming the transcriptional state of naive cells into a stem-like state. Supporting this, we identified a rare cluster of SOX9-expressing cells in primary tumors that were highly enriched for CSCs and chemoresistance-associated stress gene modules. Notably, single-cell analysis showed that chemo treatment results in rapid population-level induction of SOX9 that enriches for a stem-like transcriptional state. Altogether, these findings implicate SOX9 as a critical regulator of early steps of transcriptional reprogramming that lead to chemoresistance through a CSC-like state in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016a8c0db4a1babe91a08a91f054acbb57a8720c" target='_blank'>
              SOX9 drives a stem-like transcriptional state and platinum resistance in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            A. Duval, Fidan Seker-Polat, Magdalena Rogozinska, Meric Kinali, A. Walts, Ozlem Neyisci, Yaqi Zhang, Zhonglin Li, Edward Tanner, Allison E. Grubbs, Sandra Orsulic, Daniela Matei, M. Adli
          </td>
          <td>2025-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d62c6090571c67ea1645ed1df6849f539cf798b" target='_blank'>
              Molecular and cellular dynamics of measurable residual disease progression in myelodysplastic syndromes
              </a>
            </td>
          <td>
            Yu-Hsiang Chen, Timothy Bi, Minhang Xu, Jaehyun Lee, , Mengchuan Zhuang, A. Velarde, D. McCurry, Hui Yang, G. Al-Atrash, Gabriela Rondon, E. Shpall, F. Jelloul, F. N. Cervoni-Curet, Sassine Ghanem, Celine Kerros, Priya Koppikar, J. Molldrem, G. Garcia-Manero, Ankit Patel, Pavan Bachireddy
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The tumor microenvironment drives many malignant features of pancreatic ductal adenocarcinoma (PDAC). The fibroblasts within pancreatic tumors promote tissue remodeling, immune suppression, and resistance to therapy. However, the interactions between stromal populations and pancreatic cancer cells are less understood in the liver, the most frequent site of PDAC metastasis. To address this, we employ single cell transcriptomics to compare primary pancreatic vs. liver PDAC lesions. Here, we identify the expression of hepatocyte growth factor (HGF) in fibroblasts and its receptor MET in cancer cells are both markedly increased in the PDAC liver niche. Using functional assays, we validate that mitogenic MET signaling is activated in PDAC cells by liver-derived fibroblasts. Importantly, the inhibition of MET signaling leads to reduced tumor growth in immune competent mouse models. Collectively, our data demonstrates that liver stromal-epithelial crosstalk networks engage in signaling pathways distinct from primary pancreatic tumors, highlighting opportunities to develop new treatments for metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aa94d2601cca690cf0f25d219abda02e990ca8c" target='_blank'>
              Comparative Analysis of Primary and Liver Fibroblasts Reveals MET as a Potent Target in Pancreatic Cancer Metastasis
              </a>
            </td>
          <td>
            Rima Singh, Natalie Yousefian, Walker M. Allen, Cecily Anaraki, Alica K Beutel, Sabrina E. Calderon, Ian M Loveless, Oliver McDonald, Jennifer P Morton, D. Imagawa, Zeljka Jutric, Thomas F. Martinez, Nina G. Steele, Christopher J. Halbrook
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Liver metastases represent a leading cause of mortality in pancreatic ductal adenocarcinoma (PDAC). Recent clinical evidence has revealed distinct immune ecosystems in the liver tumor microenvironment, yet mechanistic insight into the cellular and molecular mechanisms driving those diverse immune landscapes remains unclear. Here we report, computational cellular signaling analysis of single-cell RNA sequencing data from matched primary and liver metastatic samples isolated at early and late time points from the spontaneous genetically engineered mouse model KPC (Pdx1Cre; KrasLSL-G12D; Tp53LSL-R172H). This analysis uncovered several immunomodulatory axes between cancer cells-T cells and cancer cells- macrophages. Validation of these findings were performed using primary tumor cell lines, isolated from the same or littermate tumors as the scRNA-seq dataset. Bulk RNA-seq analysis of these cell lines showed a retention of cancer-derived ligands in culture. In vivo validation using the hemisplenic liver metastatic transplant model confirmed that cell lines isolated from primary tumors, which metastasized spontaneously to the liver, retained liver metastatic outgrowth potential. Flow cytometry and multiplex cytokine profiling of isolated cell lines, non-involved liver, and liver tumors revealed distinct immune population shifts within the metastatic niche and confirmed cancer cell derived immunomodulatory cytokines. This study highlights the interplay between cancer-intrinsic signaling and the immune landscape in PDAC liver metastases and supports future mechanistic exploration.



 Ayushi Mandloi, Jace Baines, Meet Patel, Christina R. Larson, Robert S. Welner, Julienne L. Carstens. Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A081.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fce5d9cdc25fe3d365bf9aa1c45eddedfbe9968" target='_blank'>
              Abstract A081: Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth
              </a>
            </td>
          <td>
            Ayushi Mandloi, Jace Baines, Meet Patel, Christina R Larson, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e57eb64936bf914f410576d3851ca5828a633294" target='_blank'>
              The proteomics and phosphoproteomics landscape of melanoma under T cell attack
              </a>
            </td>
          <td>
            G. Franciosa, Agnete W. P. Jensen, Ana Martínez-Val, Ilaria Piga, Marco Donia, J. Olsen
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4baec95e1144f9633cc21379cc331adaff249" target='_blank'>
              Integrated single cell analysis identifies CD39+ tumor-associated NK cells with cytotoxic potential in lung cancer
              </a>
            </td>
          <td>
            Clara Serger, Lucas Rebuffet, M. Sandholzer, Irene Fusi, Nicole Oelgarth, Sofia Tundo, Thuy T Luu, A. Hojski, D. Lardinois, Marcel P. Trefny, N. Kirchhammer, Marina Natoli, Matthias Matter, Karin Schaeuble, Eric Vivier, Andrea Romagnani, A. Zippelius
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Breast cancer brain metastases are associated with a poor prognosis, and there is a high demand for improved personalized treatment strategies. It is still poorly understood which breast cancer brain metastases patients benefit from immune checkpoint blockade.



 Here, we multimodally characterized the immune niche of breast cancer brain metastases at single-cell resolution in a clinically annotated patient cohort (n = 156), combining multiplexed imaging and cell type quantification by tissue cytometry, single-nuclei RNA-sequencing, flow cytometry, spatial transcriptomics, and functional studies in patient-derived organoids.



 Overall, infiltration of breast cancer brain metastases by CD8 T cells was strongly heterogeneous between patients, and high numbers were prognostically favorable. In-depth characterization of cell types and integrative analyses including clinical data revealed distinct immune landscapes across patients, two of which independently predicted prolonged overall survival: i) Breast cancer brain metastases containing T and B cells accumulating together with characteristic cell communities in stromal tertiary lymphoid structures, and ii) breast cancer brain metastases characterized by a high content of tissue resident-like CD8 memory T cells (CD8 TRM-like cells) spatially enriched within tumor islands. CD8 TRM-like cells exhibited high levels of activation, checkpoint, and functional markers and showed tumor cell killing capacity ex vivo, suggesting them to be critical players of tumor immune control.



 Together, our work provides novel insights into anti-tumor immunity in breast cancer brain metastases and reveals prognostic immune biomarkers with potential translational relevance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb6c7f193a5bf8137e26dc814e6139f92f208a4" target='_blank'>
              KS03.5.A DISSECTING THE CELLULAR ARCHITECTURE OF BREAST CANCER BRAIN METASTASES REVEALS PROGNOSTICALLY DISTINCT IMMUNE LANDSCAPES
              </a>
            </td>
          <td>
            L. Jassowicz, F. Feng, R. Warta, F. Sor, J. Soller, A. Wöllner, C. Lotsch, J. Wong, J. Schlegel, L. Frank, P. Dao Trong, G. Jungwirth, A. Nohman, K. Maaß, M. Barthel, A. Unterberg, S. Krieg, K. Rippe, A. Abdollahi, A. von Deimling, J. Malm, P. Lichter, M. Zapatka, M. Seiffert, C. Herold-Mende
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="


 Emerging clinical evidence highlights the importance of tumor infiltration of activated immune cells, particularly T and B lymphocytes, for the efficacy of immune-based cancer therapies. However, many patients show limited immune cell presence in the tumor microenvironment. High endothelial venules (HEVs) are specialized blood vessels that enable lymphocyte trafficking from circulation into lymph nodes or tumors. Histological analyses of clinical tumor samples indicate that HEVs are positioned near dense aggregates of B and T lymphocytes, known as tertiary lymphoid structures (TLS), underscoring their role in immune cell recruitment and anti-tumor immunity. The abundant development of TLS and HEVs is associated with clinical response to immune checkpoint inhibition therapy and cancer survival. Recently, we found that simultaneous activation of innate immune effectors, STING and lymphotoxin-β receptor (LTβR) using their agonists induces the formation of TLS and HEVs in TLS/HEV-free tumors. In the current study, mouse pancreatic ductal adenocarcinoma (KPC) tumors were grown in the dorsal skin fold window chamber (DSWC) and received STING/LTβR agonist combination therapy to monitor the real-time TLS/HEV development and dynamic tumor microenvironment in intravital microscopy (IVM). Transgenic mice with fluorescent red B cells and green fluorescent CD31+ blood vessels were infused with MECA-79 antibody (HEV marker) to visualize B cell trafficking through HEVs. Four-color confetti mice crossed with B cell-specific Cre mice, allowed us to analyze the clonal expansion of B cells expressing one of four colors, with daughter cells retaining their parent color. Our findings demonstrated that tumoral HEVs closely resemble those found in lymph nodes, characterized by fewer branches, longer segment lengths, and increased diameters compared to non-HEV tumor blood vessels, which typically exhibit chaotic patterns. Circulating B cells traveling through tumoral HEVs displayed significantly slower rolling velocities than non-HEV tumor vessels, likely due to enhanced integrin and L-selectin expression induced by the combination therapy. By repeated intravital imaging, we were also able to observe a quadruple increase in B cell number in TLS during the day 4-7 treatment time frame. Following adoptive transfer of splenic leukocytes from naïve donor mice, we found that these leukocytes preferentially homed to TLS-rich areas, with B cells constituting the majority of the recruited cells. Importantly, analyses using the confetti mouse model indicate that tumor infiltrating B cells underwent clonal expansion in TLS, with different TLS having different colors. Overall, our study demonstrates that HEVs are the gateways for B cell entry to tumors, and the intratumoral TLS formation is driven by localized clonal expansions of these B cells, resembling lymph node germinal center activities, which drive the development of humoral immunity against tumors.



 Gabrielle Rowe-Brown, Masanobu Komatsu. B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36afed9d0acad24d10324fa459b1bfa3a197ed1" target='_blank'>
              Abstract A010: B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy
              </a>
            </td>
          <td>
            Gabrielle Rowe-Brown, Masanobu Komatsu
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f365bcb6620aee945a6c17c383200bd9aed47f" target='_blank'>
              Single-Cell Transcriptomics Reveals Dynamics of NK Cell Expansion in a Feeder Cell-Free Culture of PBMCs - Implications for Immunotherapy
              </a>
            </td>
          <td>
            Brian Ladd, Markella Zacharouli, Per-Henrik Holmqvist, Stefanie Renken, Pontus Blomberg, V. Chotteau
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/026590dbba50780b99ff756d96db0d1b563f7477" target='_blank'>
              Lineage-Specific Epigenomic Mapping of Proximal and Distal Fallopian Tube Epithelium Reveals Mesenchymal/Wnt-Driven Tumor Phenotype Diversification
              </a>
            </td>
          <td>
            Liudan Wei, Huarong Wang, Wenshu Li, Jingyao Lian, Lijun Huang, Yifan Wang, Tianqi Li, Shiying Li, Zhenhua Hu, Zhaowei Tu, Wei Sun, Shuo Chen, Shuang Zhang
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Retinoblastoma is the most common intraocular malignant tumor in childhood. Although current treatments offer a high survival rate, treatment toxicity, tumor relapse, and treatment resistance require a deeper understanding of the disease mechanisms to develop adapted therapies. Microscopically, this tumor is characterized by different states of differentiation and proliferation, ranging from poorly differentiated to well-differentiated retinoblastoma. Retinocytoma, on the other hand, is a benign non-proliferative tumor. Recent next-generation multi-omics analyses classified retinoblastoma tumors in subtypes 1 or 2, subtype 2 presenting a later age of onset, more genetic alterations, and higher metastatic potential. In parallel, several single-cell transcriptomics studies demonstrated intratumoral heterogeneity. However, mapping the different cell populations directly on the tumor and comparing histological features and molecular subtypes remains an unmet need. Methods Spatial transcriptomics was used to characterize a primary enucleated case with two histologically distinct areas. The whole transcriptomic profile of sixteen regions of interest, covering the two differentiation patterns of the tumor and the non-tumoral retina, was analyzed. Results The clustering of the regions of interest based on whole transcriptome data correlated with the histological description: cluster 1 (6 regions of interest) corresponded to highly differentiated areas and cluster 2 (6 regions of interest) to poorly differentiated areas. They showed enrichment for phototransduction and proliferation respectively, confirmed by immunohistochemistry for markers of these pathways. The publicly available molecular signatures of the two retinoblastoma subtypes categorized our regions of interest into two groups, which correlated perfectly with histological observation and transcriptomic profiles. Further analysis of the expression of specific senescence markers in the well-differentiated area did not support the diagnosis of retinocytoma as it did not confirm the expected up-regulation seen in this tumor type. Conclusions This study demonstrates a strong correlation between histological observation and molecular profiling, representing the first mapping of retinoblastoma composed of adjacent molecular subtypes 1 and 2. It also highlights the diagnostic ambiguity between retinocytoma and well-differentiated non-proliferative subtype 1 retinoblastoma and emphasizes the need for specific biomarkers to differentiate these two tumor types. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14814-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/599058c9510122a1b04d9c9b60c21f33e8817b4e" target='_blank'>
              Spatial transcriptomics of retinoblastoma: a visual window on intra-patient heterogeneity
              </a>
            </td>
          <td>
            A. Moulin, J. Thevenet, L. Mazzeo, S. Tissot, C. Stathopoulos, F. L. Munier, A. Berger
          </td>
          <td>2025-09-02</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/466091ce59415bf8e8d644d95fdb8949a53fb96b" target='_blank'>
              Cancer-associated fibroblasts drive metabolic heterogeneity in KRAS-mutant colorectal cancer cells
              </a>
            </td>
          <td>
            Elizabeth Elton, Niki Tavakoli, Handan Cetin, Stacey D. Finley
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) arises from precursor lesions, the most common of which are Pancreatic Intraepithelial Neoplasia (PanIN). PanINs are microscopic and cannot be detected in live individuals; as such, PanIN characterization has occurred prevalently in the context of cancer-bearing pancreata (adjacent normal) rather than in healthy tissue. Recently, through a unique collaboration with Gift of Life Michigan, we showed that PanIN prevalence is over 60% across age groups in donor healthy tissues, with high prevalence even in young individuals. Given PanIN prevalence and the rarity of pancreatic cancer, we hypothesized that the majority of PanIN is not fated to progress to malignancy. We postulated that a combination of neoplastic cell-intrinsic factors and components of the microenvironment controls progression to malignancy. We utilized spatial transcriptomics using the Visium platform (n = 14) and scRNASeq (n = 45) to dissect gene expression profiles and characterize spatial domains in donor healthy, adjacent-normal, and tumor tissues. We developed a spatial domain identification workflow that integrates spatial transcriptomics with the scRNASeq atlas, leveraging cell-type deconvolution, spatially informed clustering and pathology annotation. This approach allowed us to study the co-evolution of epithelial and stromal cells over the progression of pancreatic cancer. We were able to delineate distinct epithelial spatial domains—including Acinar, Ductal, and PanIN structures—in both donor and tumor tissues, as well as tumor-specific regions such as glandular tumor and poorly differentiated tumor. Additionally, we identified key stromal niches, including immune-rich regions such as tertiary lymphoid structures (TLS), plasma cell clusters, and macrophage-enriched zones, alongside non-immune fibrotic domains. We began by analyzing the epithelial domains and observed that while all acinar and ductal domains formed transcriptionally distinct clusters, neoplastic cells formed a continuum from sporadic healthy donor-associated PanINs to well- and poorly-differentiated tumors. We identified key genes that exhibit expression changes throughout the progression trajectory. Secondly, we analyzed characteristics of the stroma surrounding sporadic lesions, well-differentiated, and poorly-differentiated tumors. We observed key differences in fibroblasts between healthy donor-PanIN and tumorigenic regions, as fibroblasts surrounding donor-associated PanIN lacked known cancer-associated fibroblast (CAF) markers such as Alpha-Smooth Muscle Actin (α-SMA). We also observed distinct differences in immune cells, where there were more abundant plasma and CD8+ T-cells surrounding donor-associated PanIN, while tumorigenic regions showed macrophage enrichment. We then performed immunostaining and confocal imaging to validate the defined markers and specific cell populations. Our work defines the PanIN microenvironment as distinct from that of PDAC and maps the co-evolution of cancer cells and stroma over the course of carcinogenesis.



 Ahmed M. Elhossiny, Jude Okoye, Padma Kadiyala, Alexander Bray, Jamie Mills, Hannah Watkoske, Yaqing Zhang, Jiaqi Shi, Arvind Rao, Elana J. Fertig, Eileen S. Carpetner, Timothy Frankel, Marina Pasca di Magliano. Spatially Resolved Insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A052.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8f4f9e15e43361f1e76f4bf1e3cc3e1d04354d0" target='_blank'>
              Abstract A052: Spatially Resolved Insights into epithelial-stromal co-evolution in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, J. Okoye, Padma Kadiyala, Alexander Bray, Jamie N. Mills, Hannah R Watkoske, Yaqing Zhang, Jiaqi Shi, Arvind Rao, E. Fertig, Eileen S. Carpetner, Timothy L. Frankel, Marina Pasca di Magliano
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf8e063a68a88a96c8de0cef9efb9254b78c4b6" target='_blank'>
              A cellular and spatial atlas of TP53-associated tissue remodeling defines a multicellular tumor ecosystem in lung adenocarcinoma.
              </a>
            </td>
          <td>
            William Zhao, Thinh Nguyen, Atharva Bhagwat, Akhil Kumar, Bruno Giotti, Benjamin Kepecs, Jason L. Weirather, N. Mahadevan, Åsa Segerstolpe, Komal Dolasia, Jamshid Abdul-Ghafar, Naomi Besson, Stephanie M Jones, Brian Y. Soong, Chendi Li, S. Vigneau, M. Slyper, Isaac Wakiro, Mei-ju Su, K. Helvie, Allison Frangieh, Judit Jané-Valbuena, Orr Ashenberg, Mark Awad, Asaf Rotem, Raphael Bueno, O. Rozenblatt-Rosen, K. Pfaff, Scott J. Rodig, Aaron N. Hata, Aviv Regev, B. Johnson, A. Tsankov
          </td>
          <td>2025-10-07</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da9274332e90da211d1dc7074d7c247b79d5167" target='_blank'>
              The Pdgfd-Pdgfrb axis orchestrates tumor-nerve crosstalk in pancreatic cancer
              </a>
            </td>
          <td>
            Peter L. Wang, N. Lester, Ella Perrault, J. Su, Dennis Gong, C. Shiau, Jingyi Cao, Phuong T. T. Nguyen, Jung Woo Bae, Deniz G Olgun, H. Hoffman, Ashley Lam, Jean Huang-Gao, Saifur Rahaman, Jimmy A. Guo, J. Barth, Nicholas J Caldwell, P. Divakar, Jason Reeves, Arya Bahrami, Shanshan He, Michael Patrick, Eric M. Miller, M. Ganci, G. C. Jaramillo, Theodore Hong, Jennifer Wo, Hannah J Roberts, Ralph Weissleder, Hongyoon Choi, Carlos Fernández-del Castillo, Kathy Cormier, David T Ting, Tyler Jacks, Lei Zheng, M. Hemberg, Mari Mino-Kenudson, William L Hwang
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The cellular plasticity of neuroblastoma is defined by a mixture of two major cell states, adrenergic and mesenchymal, which may contribute to therapy resistance. However, how neuroblastoma cells switch cellular states during therapy remains largely unknown, and how to eradicate neuroblastoma regardless of its cell state is a clinical challenge. To better understand the cellular plasticity of neuroblastoma in chemoresistance, we define the transcriptomic and epigenetic map of adrenergic and mesenchymal types of neuroblastomas using human and murine models treated with indisulam, a selective RBM39 degrader. We show that cancer cells not only undergo a bidirectional switch between adrenergic and mesenchymal states, but also acquire additional cellular states, reminiscent of the developmental pliancy of neural crest cells. These cell state alterations are coupled with epigenetic reprogramming and dependency switching of cell state–specific transcription factors, epigenetic modifiers, and targetable kinases. Through targeting RNA splicing, indisulam induces an inflammatory tumor microenvironment and enhances the anticancer activity of natural killer cells. The combination of indisulam with anti-GD2 immunotherapy results in a durable, complete response in high-risk transgenic neuroblastoma models, providing an innovative, rational therapeutic approach to eradicate tumor cells regardless of their potential to switch cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8bfb0658591ed4d276e6df844046df62c0e809" target='_blank'>
              RBM39 degrader invigorates innate immunity to eradicate neuroblastoma despite cancer cell plasticity
              </a>
            </td>
          <td>
            Shivendra Singh, Jie Fang, Hongjian Jin, Lee-Ann Van de Velde, Andrew Cortes, Jiani Chen, Sivaraman Natarajan, E. Poon, Qiong Wu, Christopher L. Morton, Mary A. Woolard, Waise Quarni, Jacob A Steele, Jon P Connelly, Liusheng He, Rebecca Thorne, Gregory Turner, Thomas Confer, Melissa Johnson, W. Caufield, Burgess Freeman, Timothy Lockey, A. Murphy, P. J. Murray, Takashi Owa, S. Pruett-Miller, Ruoning Wang, L. Chesler, Julie R. Park, A. Davidoff, John Easton, Xiang Chen, Paul G. Thomas, Jun Yang
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Background The recent clinical success of tumor-infiltrating lymphocyte (TIL) therapy in immune checkpoint inhibitor-refractory melanoma has renewed interest in applying this approach to other malignancies. Renal cell carcinoma (RCC) is characterized by prominent CD8+ T-cell infiltration and elevated expression of cytolytic genes, suggesting the presence of tumor-reactive T-cells that could be harnessed for personalized cell therapy. TIL products are polyclonal and may recognize a diverse repertoire of tumor antigens, which could reduce the likelihood of immune escape through antigen loss. Despite this potential, past efforts to implement TIL therapy in RCC have been largely unsuccessful, likely due to the dominance of TILs exhibiting an exhausted phenotype, and the inability to effectively rejuvenate these cells during ex vivo expansion. Thus, the development of strategies that can selectively isolate and expand tumor-specific T-cells with minimal off-target toxicity and durable cytotoxic function will likely be key to the successful development of TIL therapy for RCC. We recently demonstrated that highly expanded clonal CD8+ T-cell populations in RCC tumors exhibit an antigen-experienced phenotype, indicative of chronic stimulation and consistent with tumor antigen recognition. These clonotypes represent a key feature of RCC TIL populations. Based on this, we hypothesize that the selection of non-exhausted, clonally expanded T-cells using specific surface markers will enhance the therapeutic potency and specificity of RCC-derived TIL products. Methods To identify surface markers that distinguish non-exhausted, clonally expanded tumor-reactive T-cells, we compiled a comprehensive single-cell RNA sequencing (scRNAseq) and paired TCR sequencing dataset. This includes tumor, normal adjacent tissue (NAT), peripheral blood mononuclear cells (PBMC), and lymph nodes (LN) from 23 clear cell RCC (ccRCC) patients, combining seven published and in-house studies. We are applying both differential gene expression (DGE) analysis and multi-instance-based machine learning (ML) to identify candidate surface markers linked to non-exhausted, clonally expanded phenotypes. These models are designed to accommodate the inherent noise and sparsity of single-cell data and to uncover combinatorial marker sets. Results In RCC tumors, we observed significant clonal expansion of T-cells with a memory-like, antigen-experienced phenotype. Tumor samples showed significantly higher T-cell clonality compared to matched normal tissues. Cells sharing the same TCR clonotype displayed heterogeneous transcriptional states, including both exhausted and non-exhausted phenotypes. Markers of tumor reactivity identified for other cancers (eg, CD39, CD103) labeled mainly exhausted subsets of expanded clonotypes, suggesting a need for improved marker combinations for RCC. Clonality tended to decline with increasing tumor stage, possibly indicating a link between exhaustion and tumor progression. DGE analysis revealed several surface markers associated with expanded TCR clonotypes, for which commercial antibodies are available for flow sorting. To overcome limitations of DGE alone, we are applying ML models that integrate gene expression, TCR identity, and clonal frequency to predict tumor reactivity. We successfully trained and validated multiple machine learning models, including scFormer (transformer-based), attention_MIL (attention-based deep multi-instance learning), miBoost (tree-based), and miSVM (support vector classifier), on sparse and heterogeneous scRNAseq data using a local GPU platform. All models showed consistent improvement in validation accuracy during training. Feature analysis revealed gene sets predictive of large versus small clonotypes, offering candidate markers for isolating tumor-reactive, non-exhausted T cells for downstream applications. Conclusions T-cells sharing the same T-cell receptor can display both terminally exhausted and progenitor exhausted states. Selecting clonally expanded, non-exhausted T cells may enhance the tumor reactivity of TIL products. Our ongoing analyses, including model hyperparameter optimization and gene pathway comparison, focus on identifying surface markers that are antibody-compatible, broadly expressed across patients, and applicable to CD8+ and CD4+ T-cells while excluding Tregs. These insights will support a refined TIL manufacturing strategy for improved RCC therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3870d4f63b8bc5e4814aaf86df832a9b45daa983" target='_blank'>
              30Meta-analysis of single-cell RNA sequencing of tumor-infiltrating T-cells in clear cell renal cell carcinoma reveals distinct phenotypes and prediction markers of expanded T-cell clonotypes
              </a>
            </td>
          <td>
            Yuexin Xu, Christopher P. Miller, S. Tykodi, E. Warren
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="


 Medullary thyroid carcinoma (MTC) is a rare neuroendocrine tumor arising from parafollicular C-cells in the thyroid. Unlike other neuroendocrine tumors, knowledge about the MTC tumor microenvironment and its role in therapy resistance remains limited, mainly due to the lack of reliable in vitro models. We hypothesized that, as observed in other cancer types, MTC cells may exhibit a degree of phenotypic plasticity that could be modulated by various microenvironmental factors.



 The primary aim of this study was to establish patient-derived MTC in vitro models capable of growing under different three-dimensional (3D) culture conditions while retaining the ability to shift toward a more mesenchymal, stem-like phenotype.



 We successfully established eight patient-derived primary MTC cell lines from tumors with different genetic backgrounds and variable degrees of aggressiveness. These lines were characterized in terms of phenotype and responses to different microenvironmental stimuli and targeted therapies through Western Blot, confocal microscopy, ELISA assays, ELDA, RT-qPCR, and proliferation assays.



 Notably, the cells maintained their plasticity over extended passaging when cultured under stem-promoting conditions. In differentiation media, they preserved the ability to secrete high levels of classic MTC markers, such as calcitonin and chromogranin A. Under 3D culture conditions, all cell lines showed a significant increase in stem and neuroendocrine precursor markers, including SOX2, OCT4, TBB3 and FOXA1, along with spatially dependent shifts in Epithelial-to- Mesenchymal Transition (EMT) markers E-cadherin and Vimentin. These phenotypic variations partially recapitulated the molecular alterations observed in MTC tissue samples. Moreover, changes in culture conditions—such as inducing a pseudo- hypoxic state or culturing in non-adherent environments—profoundly affected both cell phenotype and sensitivity to therapeutic agents currently available for MTC treatment, including Pralsetinib, Selpercatinib, Cabozantinib, Lenvatinib, and Vandetanib.



 In conclusion, we present here the generation of a new valid patient- derived in vitro model that can provide new opportunities to explore MTC biology complexity and will allow future translational studies, including those involved in uncovering the mechanisms underlying the insurgence of therapy resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0c689353bda0dac4479e679a592360fdc7492a4" target='_blank'>
              OC48 - Generation of patient-derived in vitro models to unravel medullary thyroid cancer microenvironment and therapy resistance
              </a>
            </td>
          <td>
            V. Ghiandai, V. Cirello, G. Gazzano, G. Dionigi, C. Colombo, L. Fugazzola, L. Persani, E. S. Grassi
          </td>
          <td>2025-09-01</td>
          <td>European Journal of Endocrinology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec585f40c1c2e8296bee75c248183641228fbd0" target='_blank'>
              Macrophages display spatiotemporal specificity in intrahepatic cholangiocarcinoma and drive tumour progression via OSM and THBS1
              </a>
            </td>
          <td>
            Tingjie Wang, Ruitao Long, Zhongyu Cheng, Mi Zhang, Bing Wei, Cong Lyu, Cuiyun Zhang, Guangbo Tang, Zihang Li, Lei Li, Yida Li, Xiaofei Zhu, Chao Shi, Ke Yang, Chuanrui Xu
          </td>
          <td>2025-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Cutaneous T-cell Lymphomas (CTCLs) refer to a group of rare cancers that affect immune cells in the skin and often the blood. Their exact cause is not entirely understood, and they are difficult to diagnose because they can appear similar to common, non-malignant skin conditions. Recent advances in single-cell RNA sequencing have made it possible for researchers to explore these tumors at an unprecedented level of detail, revealing important differences between CTCL subtypes in how they interact with the immune system. By analyzing these patterns, researchers can begin to explain differences in disease behavior and identify targets for new therapies. This summary highlights how single-cell technologies are transforming our understanding of CTCL and may pave the way for more precise diagnosis and treatment strategies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/504bc3b3f8515fbfaaeba0942215a380977f71da" target='_blank'>
              An Update on Single-Cell RNA Sequencing in Illuminating Disease Mechanisms of Cutaneous T-Cell Lymphoma
              </a>
            </td>
          <td>
            Sara Suhl, Alexander Kaminsky, Caroline Chen, B. Lapolla, Maggie H. Zhou, Joshua Kent, Abigail Marx, I. Nebo, G. Ramush, Sophia Luyten, Yoni Sacknovitz, Julie Sung, Christina M. Bear, Celine M. Schreidah, Alejandro A. Gru, L. J. Geskin
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive disease with a dense fibrotic stroma and is often accompanied by chronic inflammation. Peritumoral inflammation is typically viewed as a reaction to nearby tumor growth. Here, we report that the inflamed pancreatic lobules are frequently invaded by tumor cells, forming a distinct, non-fibrotic tumor niche. Using a semi-supervised machine learning approach for annotations of clinical samples and multiplex protein profiling, we show that tumor cells at the invasion front are closely associated with acinar cells undergoing damage-induced changes, and with activated fibroblasts expressing markers of injury. The invaded lobules are linked to classical tumor phenotypes, in contrast to fibrotic areas where tumor cells display a more basal profile, highlighting microenvironment-dependent tumor subtype differences. In female mice, lobular invasion similarly aligns with the classical tumor phenotype. Together, our data reveal that pancreatic tumors colonize injured lobules, creating a unique niche that shapes tumor characteristics and contributes to disease biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33fdbbbab057fd0e654aba5735a7c83d83362e84" target='_blank'>
              An injury-associated lobular microniche is associated with the classical tumor cell phenotype in pancreatic cancer
              </a>
            </td>
          <td>
            Sara Söderqvist, Annika Viljamaa, Natalie Geyer, Anna-Lena Keller, Kseniya Ruksha, Carina Strell, Neda Hekmati, Alexandra Niculae, J. Engstrand, E. Sparrelid, Caroline Salmén, Tânia D. F. Costa, Miao Zhao, Staffan Strömblad, Argyro Zacharouli, P. Ghorbani, Sara Harrizi, Yousra Hamidi, Olga Khorosjutina, Stefina Milanova, Bernhard Schmierer, Béla Bozóky, C. Fernández Moro, Marco Gerling
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a39d99b88fdac73c609342d555fba7b0f41448d" target='_blank'>
              Regulatory network and spatial modeling reveal cooperative mechanisms of resistance and immune escape in ER+ breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Sarthak Sahoo, M. Jolly, Jason T. George
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Organoid technology has emerged as a revolutionary tool in cancer research, offering physiologically accurate, three-dimensional models that preserve the histoarchitecture, genetic stability, and phenotypic complexity of primary tumors. These self-organizing structures, derived from adult stem cells, induced pluripotent stem cells, or patient tumor biopsies, recapitulate critical aspects of tumor heterogeneity, clonal evolution, and microenvironmental interactions. Organoids serve as powerful systems for modeling tumor progression, assessing drug sensitivity and resistance, and guiding precision oncology strategies. Recent innovations have extended organoid capabilities beyond static culture systems. Integration with microfluidic organoid-on-chip platforms, high-throughput CRISPR-based functional genomics, and AI-driven phenotypic analytics has enhanced mechanistic insight and translational relevance. Co-culture systems incorporating immune, stromal, and endothelial components now permit dynamic modeling of tumor–host interactions, immunotherapeutic responses, and metastatic behavior. Comparative analyses with conventional platforms, 2D monolayers, spheroids, and patient-derived xenografts emphasize the superior fidelity and clinical potential of organoids. Despite these advances, several challenges remain, such as protocol variability, incomplete recapitulation of systemic physiology, and limitations in scalability, standardization, and regulatory alignment. Addressing these gaps with unified workflows, synthetic matrices, vascularized and innervated co-cultures, and GMP-compliant manufacturing will be crucial for clinical integration. Proactive engagement with regulatory frameworks and ethical guidelines will be pivotal to ensuring safe, responsible, and equitable clinical translation. With the convergence of bioengineering, multi-omics, and computational modeling, organoids are poised to become indispensable tools in next-generation oncology, driving mechanistic discovery, predictive diagnostics, and personalized therapy optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c3effe5e0ade80f8d6b3e4aa8e2c7994edd86eb" target='_blank'>
              Organoids as Next-Generation Models for Tumor Heterogeneity, Personalized Therapy, and Cancer Research: Advancements, Applications, and Future Directions
              </a>
            </td>
          <td>
            Ayush Madan, Ramandeep Saini, Nainci Dhiman, Shu-Hui Juan, M. K. Satapathy
          </td>
          <td>2025-10-08</td>
          <td>Organoids</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The metastatic tumor microenvironment (TME) is a highly dynamic and heterogeneous ecosystem that plays a critical role in promoting cancer cell colonization, immune escape, and resistance to therapy. Recent advances in multi-omics technologies—including genomics, transcriptomics, epigenomics, proteomics, and metabolomics—have enabled a systems-level understanding of the molecular reprogramming that occurs in the TME following metastasis. In this review, we systematically summarize emerging findings from recent multi-omics studies that dissect cellular composition, signaling pathways, immune landscape, and metabolic rewiring within the metastatic TME. We highlight key molecular signatures and intercellular interactions that drive metastatic progression and therapy resistance. Furthermore, we discuss how integrative multi-omics data are being leveraged to identify actionable targets and to design novel immunotherapeutic and molecular precision strategies tailored to the metastatic niche. These insights provide a scientific rationale for the development of TME-targeted approaches in the treatment of advanced-stage cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1202cb3e5acfc52d10d5fc247d4fca996a2d6f2e" target='_blank'>
              Multi-omics analysis of the tumor microenvironment after metastasis: advancing toward personalized immunotherapy and molecular targeted strategies
              </a>
            </td>
          <td>
            Qing Gao, Huqiang Wu, Li Duan, Guanhua Lv, Dongmei Quan
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Hepatoblastoma (HB), the most common childhood liver cancer, often shows aberrant expression and/or mutations of the β-catenin (B), YAP (Y) and/or NRF2 (N) transcription factors and can be modeled in mouse livers by overexpressing combinations of these (e.g., BY, BN, YN and BYN). Because very few human or murine HB cell lines exist, genetically defined murine HB cell lines would be a significant advance. We describe here the generation of cell lines from primary BN and YN tumors. Unexpectedly, one BN line shows a remarkable ability to trans-differentiate into endothelial cells and can accelerate in vivo tumorigenesis and angiogenesis. These and previously derived BY and BYN lines have similar sensitivities to four drugs used to treat HB. Our generic approach for cell line derivation should allow for the generation of additional lines driven by less common drivers. A collection of such genetically defined and well-characterized cell lines will facilitate studies that are impractical to perform in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31da89c86be409fe01133803515bcf28ea7e0273" target='_blank'>
              Derivation of Genetically Defined Murine Hepatoblastoma Cell Lines with Angiogenic Potential
              </a>
            </td>
          <td>
            Keyao Chen, Ahmet Toksoz, Colin Henchy, Jessica Knapp, Jie Lu, S. Ranganathan, Huabo Wang, E. V. Prochownik
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d523903df00c8d74634f25362c4c022807a0ad86" target='_blank'>
              Spatial proteomics for investigating solid tumor resistance mechanisms.
              </a>
            </td>
          <td>
            X. M. M. Zhou, A. D'Amiano, Charles Lu, Vrinda Madan, Sara Khoshniyati, J. Kollings, Noah E Sunshine, S. Surwase, Joel C. Sunshine
          </td>
          <td>2025-10-08</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="PURPOSE
Clonal hematopoiesis of indeterminate potential (CHIP) has been associated with adverse outcomes in some solid tumor settings, but its impact on breast cancer remains unclear. We sought to investigate the genotype-specific effects of CHIP on breast cancer outcomes and the tumor microenvironment.


EXPERIMENTAL DESIGN
We examined a retrospective cohort of 125 patients with breast cancer, using targeted sequencing to identify CHIP. Metastatic events were recorded, and distant metastasis-free survival probability was analyzed. In parallel, we developed chimeric mouse models of the two most mutated CHIP genes, DNMT3A and TET2. CHIP and control mice were orthotopically injected with syngeneic breast cancer cells. Tumor growth was measured, and immune infiltrate was profiled via mass cytometry.


RESULTS
CHIP was present in 18.4% of patients. High-burden CHIP and non-DNMT3A CHIP were associated with significantly shorter distant metastasis-free survival. In vivo, mice with Tet2-CHIP developed larger primary tumors and were more likely to experience lung metastasis, while Dnmt3a-CHIP did not differ from controls. The general immune subsets observed in both CHIP models were similar, but immunophenotyping revealed clonal expansion and immune cell subset skewing specific to the Tet2-CHIP model.


CONCLUSIONS
Our findings demonstrate a genotype-specific impact of CHIP on breast cancer across human and mouse data. Further, the chimeric mouse models we generated offer a clinically relevant tool to study solid tumors in a CHIP background. This work underscores the need for further functional studies and personalized risk assessment to clearly define the impact of various CHIP genotypes on breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ebb88f9db19ddf675e686f40d924b6b992289a8" target='_blank'>
              Clonal hematopoiesis of indeterminate potential influences breast cancer outcomes in a genotype-specific manner.
              </a>
            </td>
          <td>
            S. C. Reed, Chad R Potts, Leo Y. Luo, Brad A Davidson, Riley E. Bergman, Justin D J Kemp, Ethan K Fox, Bryce A Thomas, Lauren Ha, Vani Arora, Justin Cartailler, M. E. Sanders, Violeta Sanchez, Paula I. Gonzalez-Ericsson, S. Croessmann, Paula J Hurley, Alexander G. Bick, P. Ferrell, B. H. Park
          </td>
          <td>2025-09-19</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Papillary thyroid carcinoma (PTC) is the most prevalent form of thyroid cancer, yet its cellular heterogeneity and prognostic determinants remain poorly defined. Here, we integrate two single-cell RNA sequencing datasets comprising 20 human thyroid samples to construct a high-resolution cellular atlas of PTC. We identify 29 distinct cellular subpopulations and delineate their composition, dynamics, and interactions in healthy versus tumor tissues. Notably, epithelial and monocyte populations were markedly expanded in PTC, whereas adaptive immune subsets such as B and T cells were diminished. Cell–cell communication analysis revealed enhanced intercellular signaling in the tumor microenvironment, with epithelial and endothelial cells receiving the strongest inputs. Among monocyte-specific transcriptional signatures, we identified 65 prognostic genes via univariate Cox analysis. A LASSO-derived 14-gene risk score robustly stratified patient outcomes, with CSGALNACT1 emerging as a key epithelial-specific, independent prognostic gene. Pseudotime analysis further supported its role in epithelial cell differentiation. Functional validation demonstrated that CSGALNACT1 promotes proliferation in PTC cell lines, suggesting a potential oncogenic function. Immune deconvolution across risk groups revealed substantial divergence in innate and adaptive immune infiltration, indicating a close interplay between tumor-intrinsic transcriptional programs and immune microenvironment remodeling. Collectively, our study provides a comprehensive single-cell framework for PTC, identifies a clinically relevant risk model, and highlights CSGALNACT1 as a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62fcd940763c0c7abf39d3fd346b8a069b2e5dc" target='_blank'>
              Comprehensive analysis of the papillary thyroid carcinoma identifies CSGALNACT1 as a proliferation driver and prognostic biomarker
              </a>
            </td>
          <td>
            Yu Zhao, Juan Zhang, Yong Yao, Linna Yu
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea7d79855ee47627896db43f29cfa10fcc98ce7" target='_blank'>
              Single-cell multiomic modelling of early metastatic events promoted by the extracellular matrix.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Jun Hyeong Lee, Junho Kang, Kyeonghui Kim, Youngwon Cho, Jihyun Park, Sang-Hyun Song, J. Kwon, Young-Joon Kim, Woong-Yang Park, Tae-You Kim, Jong-Eun Park, Pilnam Kim, Jung Kyoon Choi
          </td>
          <td>2025-09-15</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is one of the most aggressive and deadly brain tumors, with a median survival of ~14 months. Its poor prognosis is largely due to its ability to infiltrate healthy brain tissue both as single cells and in collectives. A deeper understanding of both individual and collective GBM cell motility could reveal new insights for patient-specific therapies. We applied live-cell imaging and mathematical modeling to characterize migration in patient-derived GBM cells, aiming to identify stratification markers and therapeutic vulnerabilities.



 We developed Single-Cell Behavior Live Imaging (ScBLI), an approach integrating live imaging with computational analysis to study the dynamics of GBM and healthy astrocyte (HA) cultures. We analyzed 21 GBM cell lines and one HA culture. Using TrackMate and Cellpose for segmentation and tracking, we extracted data on trajectories, velocities, distances, and morphological features (e.g., volume, circularity). Delta scaling was applied to classify trajectories as subdiffusive, Brownian, or Levy-like. Functional assays evaluated responses to migration stimuli and therapeutics. We also assessed collective migration behavior and integrated these data with single-cell analyses. All findings were correlated with clinical outcomes and transcriptomic profiling.



 We tracked 100 cells per sample, analyzing 2200 trajectories. Each culture contained three distinct migratory subpopulations based on metrics such as displacement, velocity, confinement ratio, and cellular organization. GBM samples were grouped into three behavioral clusters. Delta scaling values (0.28-0.80) identified corresponding migration modes. Higher delta values indicated more structured, Levy-like migration and were associated with shorter patient survival, suggesting a relationship between organized motility and increased invasiveness. Transcriptomic analysis revealed differential gene expression signatures among the clusters, potentially linked to migration phenotypes.



 Our study presents a robust framework to characterize GBM migration dynamics across patient-derived cultures. By integrating individual and collective motility data with clinical and transcriptomic information, we identified three distinct behavioral profiles with prognostic value. Delta scaling proved to be an effective metric for stratifying tumors based on migratory efficiency and aggressiveness. These insights highlight how structured motility may facilitate tumor spread and serve as a functional biomarker. Moreover, the identification of migration-associated gene signatures opens avenues for the development of targeted therapies aimed at disrupting invasive behavior. Ultimately, our findings support the use of dynamic phenotyping in guiding personalized treatment strategies for GBM patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53502cc74a0ab473690a657868f9d225b25d02d6" target='_blank'>
              P12.34.A TRACKING INDIVIDUAL AND COLLECTIVE CELLULAR DIFFUSION IN PATIENT-DERIVED PRIMARY GBM CELL LINES: EXPLORING CELL DIFFUSION PROPERTIES AS A POTENTIAL VULNERABILITY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            M. Morelli, G. Ferri, T. Hohmann, F. Lessi, F. Di Lorenzo, A. Pastore, S. Franceschi, F. Pieri, C. Gambacciani, O. Santonocito, Y. Shah, L. Tonello, A. D. Di Stefano, P. Grigolini, L. Palatella, C. Mazzanti
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d321c41a41d8fe601602ca61c440794fe8b77a07" target='_blank'>
              Bayesian inference of tissue-migration histories in metastatic cancer from cell-lineage tracing data
              </a>
            </td>
          <td>
            Stephen J. Staklinski, Armin Scheben, Lise Brault, Rebecca Hassett, Ryan N Serio, Jiawei Xing, Dawid G. Nowak, Adam Siepel
          </td>
          <td>2025-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Drug tolerant persisters (DTPs) drive cancer therapy resistance by temporarily evading drug action, allowing multiple routes to eventual permanent resistance. Despite clear evidence for DTPs, the timing of their emergence, proliferative nature, and how their population dynamics arise from measured single-cell kinetics remain poorly understood. Here we use time-lapse microscopy data from two cancer cell lines, integrating single-cell and population measurements, to develop a quantitative description of drug persistence. Contrary to the expectation that increasing levels of genotoxic stress should lead to slower times to division and faster times to death, we observe minor changes in the single-cell intermitotic and death time distributions upon increasing cisplatin concentration. Yet, population decay rates increase 3-fold, suggesting a surprising independence of the overall dynamics from the measured birth and death rates. To explain this phenomenon, we argue that the observed lineage correlations and concentration-dependent decay rates imply cell-state dependent fate choices made both pre and post-cisplatin as opposed to just post-drug birth/death rate-based competitive fate choices. We demonstrate that these cell-states, present in the ancestors of DTP and sensitive cells, exhibit no difference in cycling speed and are inherited across 2-3 cellular generations. A stochastic model implementing these rules simultaneously recapitulates the observed decay rates and cell-fate correlations, also explaining how pre-drug fate decisions are consistent with barcoding experiments where barcode diversity remains unchanged after drug administration. Our results provide a powerful perspective on drug tolerance based on general arguments, without requiring knowledge of the underlying molecular architecture of the heterogeneous cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b499c056c673d79d24c4d2c5acdf399c0d8012aa" target='_blank'>
              Inheritable cell-states shape drug-persister correlations and population dynamics in cancer cells
              </a>
            </td>
          <td>
            Anton Iyer, Adrián E. Granada, Shaon Chakrabarti
          </td>
          <td>2024-11-03</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b5111159e92c7f129ff30372c9b9934bfcdb2c" target='_blank'>
              Single-cell RNA sequencing and large-scale bulk combination with machine learning reveal gastric cancer-related macrophage heterogeneity
              </a>
            </td>
          <td>
            Saisai Gong, Yu peng Zhao, Sheng Yang, Tong Liu, Chengyun Li, Yun Zhou, Geyu Liang
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Chimeric antigen receptor T-cell (CAR-T) therapy has emerged as a breakthrough immunotherapy for multiple cancers, yet its efficacy remains limited in glioblastoma due to tumor heterogeneity, immune-evasion of the microenvironment, and rapid T-cell dysfunction. The aim of our study is to investigate the spatially differential expression of CAR-T targets in glioblastoma. We want to focus on differential expression of CAR-T targets in tumor cell subtypes, differences between tumor core and infiltration zone as well as reactive immune zones.



 We analyzed spatial transcriptomic data from 62 glioblastoma specimens using SPATA2 software, augmented by advanced computational tools such as scVI (data integration), Cell2Location (single-cell deconvolution), graph neural networks, gene set enrichment analysis (GSEA), and spatial annotation screening (SAS). CAR-T antigens were chosen from existing published clinical and pre-clinical CAR-T cell trials.



 Spatial mapping revealed a pronounced gradient of NPC-like cells with a higher abundance of NPC-like cells in the infiltration zone and resection margins—regions critical for residual disease persistence after resection. Clinical CAR-T targets (e.g., ERBB2, PTPRZ1) were highly expressed in AC-like and OPC-like dominated niches while nearly absent in NPC-rich areas, suggesting a blind spot for current CAR-T therapies. Single-cell deconvolution further correlated NPC-like dominance to increased T-cell infiltration, while GSEA and clonal dynamics analysis of the T-cell compartment suggested downregulated functioning subclones in NPC-rich spatial neighborhoods, underscoring potential immune evasion mechanisms in these regions.



 Our study exposes a fundamental limitation of current CAR-T designs: their inefficacy against NPC-like cells that dominate the infiltration zone and therefore the post-surgical glioblastoma ecosystem. By pinpointing spatial and phenotypic resistance hubs, we provide a roadmap for refining CAR-T targets to address this gap, potentially enhancing adjuvant immunotherapy for glioblastoma. Protein-level validation of key findings is currently ongoing to further substantiate the results.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe7508f42aeb973a2f44bba592abe40968ca576d" target='_blank'>
              P04.13.B SPATIAL HETEROGENEITY OF CAR-T ANTIGEN EXPRESSION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Y. Kong, J. Zhang, F. Sahm, G. Kastner, M. Mühlbauer, E. Grabis, P. Heinrichs, J. Straehle, R. Sankowski, R. Roelz, J. Beck, D. Heiland, N. Neidert
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="B7 homolog 3 (B7-H3, CD276) has emerged as a promising target for chimeric antigen receptor (CAR) T cell therapy, with limited expression in normal tissues and high level cell-surface expression across various tumor types. Clinical studies are ongoing, with a focus on pediatric cancers. As an immune checkpoint molecule of the B7-CD28 family, B7-H3 has a proposed immune-modulatory role, though the precise nature of B7-H3-mediated cell interactions and functional contributions to immune responses are contradictory and likely context-dependent. Within tumors, B7-H3 is expressed also on non-tumor cell types in the tumor microenvironment (TME), including myeloid immune cells, endothelial cells of abnormal vasculature and cancer-associated fibroblasts. Consequently, CAR T cells directed against B7-H3 will not only target tumor cells but also components of the TME, which will affect the nature and outcome of B7-H3-targeted therapeutic immune responses. Here we review the expression of B7-H3 protein in pediatric solid tumors and in various cell types known to infiltrate the TME of solid tumors. On this background, we discuss the potential of B7-H3-targeted CAR T cells to reshape the TME and the key challenges and future directions to improve B7-H3-targeted CAR T cell therapy for pediatric patients with solid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/649b83099536525155f2e82b6d45425c1ee80892" target='_blank'>
              B7-H3 in the tumor microenvironment: Implications for CAR T cell therapy in pediatric solid tumors
              </a>
            </td>
          <td>
            Lena Jansen, J. Wienke, Ronja Molkenbur, Claudia Rossig, Ramona Meissner
          </td>
          <td>2025-10-11</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) chemotherapy regimens are largely uniform, tailored mainly to patient fitness or rare genomic alterations. This strategy fails to capitalize on the disease’s substantial molecular heterogeneity, including variations in copy number alterations, immune infiltration, stromal composition, and cancer cell phenotypes. These features are not independent; they reflect interconnected biological processes and co-evolving pathways that shape tumour progression. Multi-omic integration is therefore essential—not only to capture the complexity of PDAC biology, but also to clarify the molecular basis of tumours classified as ‘intermediate’ by current stromal and cancer taxonomies, thereby enabling rational therapeutic targeting. To this end, we performed imaging mass cytometry on three serial sections of a PDAC tissue microarray (221 resected tumours, ∼4 cores each), generating >800 multiplexed images (40–43 channels) focused on epithelial, immune, or stromal biomarkers. We profiled 76 immune and stromal cell types and states (hypoxic, proliferative, apoptotic, under tension), as well as six tumour phenotypes defined by expression of epithelial transcription factors (GATA6, FOXA2, PDX1), classical markers (AGR2, TFF1, CEACAM6), basal markers (TP63, KRT5, S100A2, CAV1), and other PDAC-associated proteins (S100A4, MMP7, MUC16). These six cancer cell types captured the classical and basal PDAC signatures, along with four discrete “intermediate” states with distinct associations to stromal heterogeneity, RNA subtype (n = 92), tumour ploidy (n = 182), and patient outcome. These phenotypes are also detectable in unmatched single-cell RNA-seq data (n = 163), though with more overlap in marker expression. Using matched 30X whole genome sequencing (n = 182), we identified mutations and copy number alterations linked to shifts in cancer and stromal cell phenotypes, raising the question of which molecular axis best informs clinical prognosis. To address this, we applied a modified version of Stabl, a Lasso-based machine learning approach, to compare across omic layers and identify features most strongly associated with overall survival. Per-modality analysis showed that models incorporating omics outperformed those based on clinical features alone, with imaging data slightly outperforming genomics. Cross-omic integration revealed several prognostically relevant copy number aberrations, fibroblast phenotypes, and cancer cell states. By consolidating information on tumour phenotypes, stromal niches, and genomic alterations, this work aims to focus future drug discovery on the most clinically impactful molecular features in PDAC.



 Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steve Gallinger, Hartland W. Jackson. Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B116.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5cd553302915b28eaa98522e24327f9aa96cbdc" target='_blank'>
              Abstract B116: Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival
              </a>
            </td>
          <td>
            Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, A. Borgida, Edward L. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, Hartland W. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Despite the availability of numerous approved immunotherapies for various cancers, durable progression-free survival remains relatively uncommon among patients with advanced cancer. As research into immunotherapy intensifies, the heterogeneity and complexity of the tumor microenvironment (TME) have emerged as critical determinants of treatment response and a major obstacle to understanding tumor resistance mechanisms. Recent advances in spatially resolved transcriptomics (SRT) enable transcriptome-wide measurement of gene expression while preserving essential spatial information, which supports the characterization of the features of the TME. This review outlines key challenges in cancer immunotherapy and emphasizes the importance of the TME in determining therapeutic efficacy. SRT strategies suitable is systematically introduced for TME profiling, comparing their relative advantages and limitations. Additionally, bioinformatics tools and methods used to reconstruct spatial tissue patterns are discussed from SRT data. Furthermore, how SRT is explored can enhance the understanding of the complexity and heterogeneity of the TME, facilitate the identification of novel therapeutic targets, and advance biomarker discovery to refine personalized treatment strategies for diverse cancer patients. Finally, future perspectives on the application of SRT in cancer immunotherapy and addressing the associated challenges are provided.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53e788cfc2c6f7fa7f419ffa32b647da6da2a8f3" target='_blank'>
              Spatially Resolved Transcriptomics: Revealing Tumor Microenvironment Heterogeneity to Advance Cancer Immunotherapy.
              </a>
            </td>
          <td>
            Dan Liu, Lu Xiao, Yuyuan Wu, Chensirong Yue, Mengmeng Li, Yuanpei Sun, Chao Yang
          </td>
          <td>2025-09-04</td>
          <td>Small methods</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/584838fc3d0ebaa26117e1c12ed3d657762b5f85" target='_blank'>
              A histology foundation model for high-resolution spatial omics prediction, tumor detection, and, clustering of spatial transcriptomics
              </a>
            </td>
          <td>
            Ning Zhang, Yahui Long, Sibo Xia, Pengzhen Jia, Yujie Jin, Siqi Chen, Jinmiao Chen, Min Li
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Understanding tumor heterogeneity is essential for advancing cancer treatment. Clonal reconstruction methods play a pivotal role in deciphering this heterogeneity. Our goal is to develop a clonal reconstruction approach that is clinically applicable, easy to implement, and capable of delivering both high-speed performance and excellent reconstruction accuracy. Results We present MyClone, a probabilistic method designed to reconstruct the clonal composition of tumors using deep sequencing genomic data. MyClone processes read counts and copy number information of single nucleotide variants derived from deep sequencing data, enabling it to determine the mutational composition of clones and the cancer cell fractions of these mutations. Compared to existing clonal reconstruction methods, MyClone enhances clustering accuracy and cancer cell fraction prediction when applied to deep-targeted sequencing data and bulk tumor sequencing data with deep sequencing coverage. Additionally, MyClone achieves a substantial improvement in computational speed. We rigorously validated MyClone’s performance using both simulated and real clinical datasets and applied it to analyze a circulating tumor DNA sequencing dataset from 139 metastatic breast cancer patients. In this analysis, we explored mechanisms of drug resistance in metastatic breast cancer and identified 10 mutated genes potentially associated with drug resistance or sensitivity. Conclusions For deeply sequenced data, MyClone outperforms existing methods on both targeted sequencing and bulk tumor data. Its high computational efficiency and reconstruction accuracy position MyClone as a promising tool for broad clinical application in cancer treatment. The source code is publicly available at https://github.com/Hansen0413/myclone-code. Supplementary Information The online version contains supplementary material available at 10.1186/s12859-025-06252-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/008b5a4c97ae8500318c69c4b98363d3ad251446" target='_blank'>
              MyClone: rapid and precise reconstruction of clonal population structures for tumors
              </a>
            </td>
          <td>
            Qihan Guo, Dan Lv, Bo Lan, Haili Qian, Fei Ma, Ting Chen
          </td>
          <td>2025-10-08</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Phenotypic heterogeneity - distinct molecular and behavioral variations within a population -significantly influences collective invasion and tumor progression. Here, we employ a molecular approach to explore how the underlying metabolic heterogeneity in non-small cell lung cancer (NSCLC) influences invasion and pack patterning. Assessing three-dimensional (3D) pack patterning revealed invasive heterogeneity across NSCLC cell lines and patient-derived samples. Flow cytometry identified IL13RA2 as a biomarker for invasive potential, enabling isolation of subpopulations with distinct invasive characteristics. By integrating a cell surface biomarker (IL13RA2±) with mitochondrial membrane potential (TMRM), we identified and isolated three distinct subpopulations. Two-dimensional (2D) analyses revealed differences in mitochondrial polarity and transcriptional programs associated with migration and oxygen-sensitivity. In 3D, these subpopulations invaded with distinct patterns, from contiguous circular packs to structured chains. Assessments under varied oxygen tension demonstrated that oxygen availability and subpopulation metabolism together influence collective invasion patterning. When recombined at ratios recapitulating the original population, both stochastic and opportunistic cooperative dynamics emerged, dependent on subpopulation composition and oxygen levels. Our molecular approach, integrating cell surface and metabolic characteristics, enables isolation of unique subpopulations and demonstrates that phenotypic and metabolic heterogeneity, population composition, and oxygen availability collectively pattern invasion packs and drive collective invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14710731ead2715e9cfc57e6f8afb7982d2087db" target='_blank'>
              Metabolic programming defines oxygen sensitive subpopulation hierarchies and patterning in collective invasion.
              </a>
            </td>
          <td>
            Veronika Y. Matsuk, Tala O. Khatib, Landon J. Marcus, Isaac E. Robinson, Yuan Liu, Janani K Pasupathy, Mala Shanmugam, Janna K Mouw, Adam I. Marcus
          </td>
          <td>2025-09-24</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Summary The rapid increase in volume, diversity, and quality of single-cell omics profiling opens new opportunities for drug and target discovery. While there are already many workflows developed for analysis and visualization of data collected with single-cell RNA-seq, few workflows output ranked drugs and targets specific for subpopulation of single cells. Here, we present the single cells to drugs and targets (sc2DAT) workflow, a web-based software application for predicting cell surface targets and therapeutic compounds tailored to target-specific cell types automatically identified from scRNA-seq and bulk RNA-seq datasets. sc2DAT can be used to develop hypotheses about selectively eliminating malignant subpopulation of cells in cancer, or reprogram disease tissues toward a healthier phenotype using compounds from the LINCS L1000 dataset. Such compounds are hypothesized to either reverse or mimic the direction of changes in gene expression signatures, restoring the subpopulation of cells towards a healthier phenotype. Availability and implementation sc2DAT is available from: https://sc2dat.maayanlab.cloud; the source code is available from: https://github.com/MaayanLab/sc2DAT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a760160659802bae9045967c03f5b8ce394f5a9" target='_blank'>
              sc2DAT: workflow for targeting tumor subpopulations of single cells
              </a>
            </td>
          <td>
            Giacomo B. Marino, Anna I. Byrd, Nasheath Ahmed, Daniel J. B. Clarke, Avi Ma’ayan
          </td>
          <td>2025-09-26</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1de125e5b8e76ca9fbd244fb7a42b1eef22672" target='_blank'>
              Mass cytometry data integration methods reveal rural-urban gradient of immune profiles across geography
              </a>
            </td>
          <td>
            Koen A. Stam, Jeremia J Pyuza, M. Manurung, D. Tahapary, T. Supali, Marloes M A R van Dorst, A. Mahfouz, M. Charrout, S. Jochems, Marion H. König, Y. Kruize, Marouba Cisse, Ibrahima Diallo, Moustapha Mbow, M. Yazdanbakhsh, W. A. de Steenhuijsen Piters
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="BRCA1 mutation is closely linked to triple-negative breast cancer (TNBC), a subtype characterized by early onset, aggressive clinical course, and marked genomic instability. Although poly ADP ribose polymerase (PARP) inhibitors have expanded treatment options, resistance driven by cancer stem cells (CSCs) and incomplete interception of premalignant progression remain pressing challenges. In a recent study, Liu et al. developed a human induced pluripotent stem cell (iPSC)-based breast cancer model that follows BRCA1‑driven tumorigenesis from mammary differentiation to invasive disease and identify S100P as a BRCA1‑repressed effector associated with cancer stemness. These findings provide a human genetic context to probe early oncogenesis and nominate the S100P–RAGE axis as a precautionary—but promising—target for risk interception.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfe5bbd7fe04bb4260117969d5b0d5e59d56a2ae" target='_blank'>
              Deciphering BRCA1-driven tumorigenesis through an iPSC-based breast cancer model
              </a>
            </td>
          <td>
            Gaoxiang Zhu, Bing Zhao
          </td>
          <td>2025-10-01</td>
          <td>Cell Transplantation</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db2e75b459eeec6bf6fbad0671d859c2a4ceef4c" target='_blank'>
              Targeting tumor-intrinsic TAK1 triggers anti-tumor immunity and sensitizes pancreatic cancer to checkpoint blockade
              </a>
            </td>
          <td>
            Sapana Bansod, , Vikas K. Somani, Lin Li, Suneeta Modekurty, Brett L. Knolhoff, Ryan C. Fields, M. Ruzinova, David G. DeNardo, kian-Haut Lim
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) cancer associated fibroblasts (CAFs) are a complex and heterogenous group of cells and are a key component of the tumour microenvironment (TME). Our understanding of CAF functional diversity has increased in recent years, however certain aspects of their biology remain elusive, such as the spatial organisation of CAFs, their interactions with other TME compartments, and their contribution to global TME architecture. Better understanding of these will significantly accelerate therapeutic development.



 We performed multiplex immunofluorescence on tissue microarrays of resected human pancreatic cancer (n=227) with an 8-plex CAF panel and a molecular subtyping panel. After assigning bulk cell type (CAF – FAP+CK-, Epithelium – FAP-CK+, Other – FAP-CK-), we integrated CAF morphology, spatial pseudotime analysis, and TME architectural scoring in a hierarchical manner (pixel > cell > neighborhood > tissue module > patient) to define architectural subtypes of PDAC with internal validation.



 Architectural subtypes are defined as follows (name, dominant CAF, median disease-specific survival): Cohesive subtype (iCAF, 40 months) Rich in dense, cohesive, and tightly-packed epithelium. Sparse, loose stroma with a heterogenous population of iCAF-like CAFs which exhibit migration away from epithelium on spatial pseudotime modelling. Inflammatory subtype (roundCAF, 31 months) Dominated by a rich infiltrate of presumed immune cells (FAP-CK-). Highly organised with low spatial entropy. Associated with the classical subtype. CAFs have “cold” marker expression, are round in shape with a smooth surface, and exhibit migration towards epithelium on spatial pseudotime modelling. Activated subtype (Enlarged myCAF, 17 months) Dense stroma infiltrated by large myCAF-like CAFs. High spatial entropy with loss of TME compartmentalisation. Immune excluded. Strongly associated with the squamous/basal-like subtype. Fragmented subtype (Stellate myCAF, 13 months) Loose, dis-cohesive epithelium infiltrated by a CAF-dense stroma rich in myCAF-like CAFs with stellate morphology. High CAF-epithelial cell spatial contact. Associated with poor histological differentiation and poorest prognosis. Hypothesised to represent impending or established (occult) metastasis.



 Morphospatial profiling reveals striking associations between CAF shape, function, and spatial niche, and adds crucial perspective to our understanding of CAF biology beyond current methods. Combined with spatial pseudotime analysis, it offers exciting insight into CAFs in transitory states between subtypes, allowing maximum leverage of datasets which lack a true temporal dimension. In addition, architectural subtyping reveals the complex and heterogenous spatial organisation of the pancreatic TME, and the co-evolution of CAFs with these organisational changes. Integrating this with histopathological analysis lays foundations for the development of novel spatial biomarkers of prognosis, therapeutic development, and treatment response.



 Adam Bryce, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, Leonor Schubert Santana, Fiona Ballantyne, Catherine Ficken, Ian Powley, Australian Pancreatic Cancer Genome Initiative (APGI), John Le Quesne, Fieke Froeling, Stephan Dreyer, David Chang. Morphospatial profiling of cancer-associated fibroblasts reveals architectural subtypes of pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B121.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76e85ad242be0ed691b8aeea89c3fdd7c71569d" target='_blank'>
              Abstract B121: Morphospatial profiling of cancer-associated fibroblasts reveals architectural subtypes of pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            A. Bryce, Silvia Martinelli, R. Pennie, Leah Officer-Jones, Leonor Schubert Santana, Fiona Ballantyne, C. Ficken, Ian R Powley, Australian Pancreatic Cancer Genome Initiative (APGI), John Le Quesne, F. Froeling, S. Dreyer, David Chang
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6b221cace1b09faa76330c09ce1ca8391e4017e" target='_blank'>
              GLIS3 Marks a Neural-like Progenitor Cell State that Drives Metastasis in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Dennis Gong, Jimmy A. Guo, Jennifer Su, Sueda Cetinkaya, Connor J Hennessey, Patrick Yu, Ananya D. Jambhale, Peter L. Wang, Nicholas J Caldwell, Ashley Lam, Junning Wang, C. Shiau, Kevin S. Kapner, Laleh Abbassi, Seema Chugh, Shatavisha Dasgupta, Jonathan O Nowak, B. Wolpin, M. L. Zhang, Mari Mino-Kenudson, T. Jacks, Andrew J. Aguirre, William L. Hwang
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The COVID-19 pandemic has highlighted long-term health concerns of viral pneumonia, yet its potential impact on cancer development and growth remains poorly understood. Here, we demonstrate that prior infection with SARS-CoV-2 or influenza virus promoted lung tumor progression by reprogramming the local immune landscape. Retrospective clinical analysis revealed that patients hospitalized with COVID-19 exhibited increased lung cancer incidence. Using multiple murine lung cancer models, we show that prior severe respiratory viral infections accelerated tumor growth and reduced survival. Mechanistically, prior viral pneumonia epigenetically remodeled the lung to establish a pro-tumor microenvironment, including the local accumulation of SiglecFhi tumor-associated neutrophils, a transcriptionally reprogrammed, immunosuppressive population whose signature predicted poor prognosis in human lung adenocarcinoma. In parallel, epithelial compartments exhibited altered differentiation trajectories, with persistence of injury-associated alveolar intermediates positioned along tumorigenic lineages. We observe sustained chromatin remodeling at key cytokine loci in immune and structure cells, linking inflammatory memory to persistent immune suppression. Therapeutically, combined inhibition of neutrophil recruitment via CXCR2 and PD-L1 signaling restored CD8⁺ T cell infiltration and suppressed tumor growth. Together, our findings establish a direct causal relationship between viral pneumonia, including COVID-19, and lung tumorigenesis, highlighting the urgent need to monitor survivors for elevated cancer risk and to develop targeted interventions and therapies aimed at preventing potential cancer bursts in COVID-19 convalescents. Graphic Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c02dc28314c51c85a1830d8f386c201cf18563" target='_blank'>
              Respiratory viral infections prime accelerated lung cancer growth
              </a>
            </td>
          <td>
            Wei Qian, Xiaoqin Wei, Andrew J Barros, Xiangyu Ye, Qing Yu, Samuel Young, Eric V Yeatts, Yury Park, Chaofan Li, Gislane Almeida-Santos, Jinyi Tang, H. Narasimhan, Nicole Kirk, Ying Li, Li Li, Peter Chen, Jeffrey M Sturek, Kwon-Sik Park, Wei Chen, I. Cheon, Jie Sun
          </td>
          <td>2025-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feaa39ba952819de056b43cb230b39c7adfa9028" target='_blank'>
              Human iPSC-derived prostate organoids with germline BRCA2 mutation undergo tumorigenic transformations
              </a>
            </td>
          <td>
            Bipul R. Acharya, George Lawless, Pablo Avalos, Prince Anand, Yesai S Fstkchyan, Shaughn Bell, Maria G Otero, Samuel Guillemette, Zachary Myers, Michael Workman, William J. Catalona, Dan Theodorescu, Clive N. Svendsen
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a common neoplasm that carries the risk of transformation into Richter’s syndrome (RS), a highly aggressive B cell lymphoma with poor prognosis. Limited availability of animal models and cell lines hinders understanding of transformation mechanisms. Addressing this gap, we established the first in silico dynamic model of the disease. Our methodology integrates mathematical logic modeling with experimental data to identify disease drivers, mechanisms, and potential therapeutic targets. We validated the model by comparing the model’s readout with experimental data from different biological levels, such as single-cell RNA sequencing analyses and a CLL/RS patient formalin-fixed paraffin-embedded (FFPE) tissue cohort. Our analyses identified BMI1 proto-oncogene and TP53 loss as key RS progression regulators. In addition, we performed an in silico target screening to identify promising target combinations in a personalized fashion. Through the synergy of mathematical modeling with experimental readouts, our model provides a complementary approach to investigate the process of CLL transformation to RS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0084b1cbd03c2692e1100a97e2d57fe76c34d4" target='_blank'>
              CLL to Richter syndrome: Integrating network strategies with experiments elucidating disease drivers and personalized therapies
              </a>
            </td>
          <td>
            Julia Maier, J. Schwab, Silke D. Werle, Ralf Marienfeld, S. Stilgenbauer, Peter Möller, Nensi Ikonomi, Hans A. Kestler
          </td>
          <td>2025-09-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 We present a novel integration of data-driven single-cell analysis with mechanistic mathematical modeling to predict the impact of cancer-associated fibroblasts (CAF) on pancreatic ductal adenocarcinoma (PDAC) invasion. Bioinformatic analyses of high-throughput single-cell and spatial molecular assays give us access to human tumor gene expression at the time of measurement, enabling direct characterization of the human tumor microenvironment (TME), complementing our understanding of cell behavior gained from preclinical models. Notably, applying our Bayesian non-negative matrix factorization algorithm CoGAPS to single-cell RNA-seq data from PDAC allowed us to associate a gene program of concurrent epithelial-to-mesenchymal transition (EMT) and inflammation in epithelial cells with the presence of CAFs, validated through organoid co-culture experiments. We hypothesize that this gene program represents a mechanism by which CAFs promote invasive behavior in neoplastic cells over time. Testing this hypothesis requires inference of temporal changes, which is not possible with single-timepoint data collection in genomics data. In contrast to data-driven bioinformatics, mechanistic mathematical modeling can capture processes that happen over time and make predictions about a system. Agent-based models (ABMs) are one form of such mechanistic mathematical models that are well suited for temporal modeling of cellular phenotypes. We developed new software for ABMs that abstract cells into software agents with individual states and their own rules of behavior from single-cell and spatial molecular assays for temporal inference. We apply this technique to understand how CAFs in the PDAC microenvironment support the series of phenotype transitions comprising tumor progression and invasion, by encoding the CAF-induced ability of tumor cells to migrate and a cessation of proliferation in tumor cells, induced by CAF-secreted, non-diffusable substrate as inferred from our genomics analysis. While this provides a virtual framework to test the impact of varying CAF density on tumor cell invasion, it initially lacked integration with the spatial architecture of human tumors. We then apply the ABM to human PDAC spatial transcriptomics data to forecast tumor behavior over time in a spatially-informed setting, varying the hypothesized mechanisms. This revealed that a substrate-driven mechanism results in a broad mesenchymal-like zone surrounding epithelial-like outgrowths, while a cell contact-mediated mechanism results in a single-cell-wide boundary surrounding more significant outgrowth. When we compared these to real tumor compositions, this favors the explanation that secreted CAF substrate drives EMT in PDAC neoplastic cells, which we then validate in organoids with CAF conditioned media. Our work demonstrates how integrating ABMs with genomic and spatial data enables testing of mechanistic hypotheses about tumor-stroma interactions and provides deeper insights into PDAC microenvironments, potentially informing strategies to intercept disease progression.



 Jeanette I. Johnson, Daniel R. Bergman, Tamara Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Yoseph W. Dance, Jacob T . Mitchell, Elizabeth M. Jaffee, Neeha Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, Ashley L. Kiemen, Luciane T. Kagohara, Young H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O'Brien, Elana J. Fertig. Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d57a26173bbcb8bbded169c7c20718d2482d35" target='_blank'>
              Abstract A025: Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion
              </a>
            </td>
          <td>
            J. Johnson, Daniel R Bergman, T. Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Y. Dance, Jacob T. Mitchell, Elizabeth M. Jaffee, N. Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, A. Kiemen, Luciane T. Kagohara, Y. H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O’Brien, E. Fertig
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Background Penile squamous cell carcinoma (PSCC) is a rare yet potentially lethal malignancy, often resulting in devastating disfigurement, with a 5-year survival rate of only ∼50%. Human papillomavirus (HPV) infection is implicated in approximately half of PSCC cases and is associated with improved clinical outcomes; however, the underlying mechanisms remain poorly understood. Methods To elucidate HPV-associated differences in the tumor microenvironment, we performed single-cell RNA sequencing on tumors from 11 treatment-naïve PSCC patients, analyzing a total of 52 980 single cells. Unsupervised clustering identified 49 distinct cellular clusters across immune and stromal compartments. Results HPV-positive tumors exhibited an increased abundance of mast cells and a reduction in the proliferative macrophages subpopulation compared to HPV-negative tumors. Notably, CD8+ T cells in HPV-positive PSCC expressed lower levels of immune checkpoint molecules, suggesting a less exhausted immune state. Conversely, TIGIT and its ligands were significantly enriched in HPV-negative tumors, potentially fostering an immunosuppressive niche. Conclusion Collectively, our study delineates the single-cell landscape of PSCC and highlights distinct tumor microenvironment remodeling associated with HPV status, suggesting that the reduced immunosuppression in HPV-positive tumors may underlie their more favorable prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d170eff0ce2ddd0035f0702533123e38640014" target='_blank'>
              Single-cell transcriptomic profiling reveals distinct tumor microenvironments in HPV-associated penile squamous cell carcinoma
              </a>
            </td>
          <td>
            Lianbang Zhu, Deyun Shen, Jiahao Zhou, Chen Cheng, Zhiyao Xu, Yong Liang, Wen Pan, Tao Tao
          </td>
          <td>2025-09-01</td>
          <td>Precision Clinical Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) refer to ectopic lymphoid aggregates that form in non-lymphoid tissues at sites of chronic inflammation including cancers. TLSs have been recognized as significant predictors of the efficacy of immune checkpoint blockade (ICB) therapies and have the potential to elicit robust anti-tumor immune response. However, recent studies have revealed substantial heterogeneity in TLSs across different individuals and cancer types, which directly impacts the effectiveness of anti-tumor immunity. Concretely, the maturation status, localization, and density of TLSs profoundly influence the dynamic interactions among immune cells within these structures, potentially leading to adverse effects. This review provides an in-depth exploration of how the heterogeneity of TLSs influences cellular composition and immune dynamics, with the objective of influencing the efficacy of ICB therapies and modulating prognostic prediction accuracy. Additionally, the potential of combining TLSs with other biomarkers for predicting anti-tumor immunity outcomes is further investigated, alongside the introduction of advanced technologies for evaluating TLS heterogeneity. Collectively, these analyses aim to advance the understanding of TLS heterogeneity and facilitate its translation into clinical and translational medicine applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3182b4ad88a2ceacb5b488328272b01f2977fe0" target='_blank'>
              Dissecting Tertiary Lymphoid Structures in Cancer: Maturation, Localization and Density
              </a>
            </td>
          <td>
            Guang-Liang Su, Meng‐Jie Zhang, Hao Li, Zhijun Sun
          </td>
          <td>2025-08-30</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Pancreatic intraepithelial neoplasms (PanINs), precursor lesions of pancreatic ductal adenocarcinoma (PDAC) that are abundant healthy individuals, rarely progress to cancer despite universally possessing many of the driver alterations associated with malignancy. Recent single cell profiling efforts of PanIN progression in vivo have described marked heterogeneity in chromatin accessibility and transcriptional identity within these neoplasias, but it remains unclear how discrete epigenetic factors play a role in shaping tumorigenesis, or what distinguishes rare neoplastic cells capable of seeding invasive carcinomas from their benign neighbors. Given that more than 80% of patients with PDAC present with unresectable or metastatic disease, further studies on the molecular features of PanIN progression are needed to enable methods of early detection and intervention. Unfortunately, owing to their microscopic size and challenges associated with the isolation of PanINs from surrounding normal epithelium and stroma, it is particularly difficult to address these knowledge gaps in vivo. By generating complex murine derived organoid models, we have built a system that allows us to faithfully reconstitute the evolution from benign PanIN to PDAC in vitro, as well as in vivo through orthotopic implantation into syngeneic mice. As they are easily perturbable, readily generate large amounts of biomass, and closely resemble the stages of PDAC initiation in vivo, these models can be leveraged to directly assay how chromatin remodelers influence malignant transformation. Further, pairing these organoids with our highly multiplexed, transcribable DNA barcoding system and single cell epi multiomics will enable us to study the evolutionary dynamics, chromatin landscapes, and transcriptional identities of clonal populations of PanIN organoids as they progress to cancer. Through these studies, we will gain unprecedented insights into the underpinnings of PDAC initiation, and thus may open up new avenues for the early detection of tumorigenesis.



 Connor J. Hennessey, Minh T. Than, Ben Z. Stanger, Sydney M. Shaffer. Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90b593e2ad7a725999b9d873ea37a620072eeba" target='_blank'>
              Abstract B106: Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models
              </a>
            </td>
          <td>
            Connor J. Hennessey, Minh T Than, Ben Z. Stanger, Sydney M. Shaffer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a055b2099d39e961c79dc682be1c20375d7bba" target='_blank'>
              Tumor microenvironment response to neoadjuvant therapy in breast cancer: insights from single-cell and spatial omics.
              </a>
            </td>
          <td>
            Qingzhong Wu, Jiawei Yang, Da Zhang, Haowei Xu, Yuhang Yu, Yu Zheng
          </td>
          <td>2025-09-13</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Vascular mimicry (VM) refers to the formation of vessel-like structures by tumor cells independent of endothelial cells. These VM channels connect to the host’s vascular network and are associated with aggressive tumors and poor patient prognosis. Most VM research has been conducted on melanoma, relying on patient-derived and mouse cell lines. In other solid tumors, VM studies rely on human cell lines, which have certain limitations for in vivo studies. Specifically, most in vivo VM research involving human cells uses subcutaneous mouse models that fail to recapitulate organ-specific tumor microenvironments. As the microenvironment is an essential driver of tumor vascularization, including VM, murine cell lines could facilitate VM investigations in syngeneic mouse models. Here, we present CT26 and KPC, well-characterized murine colorectal and pancreatic cancer cell lines, as cell models for VM investigations. Using in vitro cell-based assays, we demonstrate that CT26 and KPC undergo VM, a cell-intrinsic process that is enhanced by serum deprivation and exposure to hypoxia and is independent of tumor-secreted growth factors. Additionally, we demonstrate the importance of YAP/TAZ signaling in VM formation, as inhibition at non-cytotoxic concentrations attenuated VM formation. Remarkably, CA3, the most potent of the two inhibitors, significantly reduced cell proliferation in both cell lines at the IC50 concentration. This reduction in cell proliferation was associated with the induction of apoptosis in CT26 cells and changes in the cell cycle in both CT26 and KPC cells. Finally, dual YAP/TAZ knockdown in both cell lines significantly abrogated VM formation, validating our initial findings using inhibitors. These results show that CT26 and KPC cells undergo VM, and given their extensive use in cancer research, can be used to investigate VM in vivo using syngeneic models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec4fb2cfcbdb76b0840a4ef6bfa2d3738d1e6f07" target='_blank'>
              Murine Cell Line Models for Vascular Mimicry: The Role of YAP/TAZ Signaling
              </a>
            </td>
          <td>
            Matilde Righetti, Ana-Maria Primorac, J. T. Erler, V. Oria
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cced907718bd3a02f9c4423bb5be0323614fd5" target='_blank'>
              Long-term clonal analysis using stochastic models reveals heterogeneity and quiescence of hematopoietic stem cells
              </a>
            </td>
          <td>
            Yuri Garcia Vilela, L. Thielecke, A. C. Fassoni, I. Glauche
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is an epithelial cell cancer of the biliary tract. CCA can be further classified into intrahepatic cholangiocarcinoma (iCCA), perihilar cholangiocarcinoma (pCCA), and distal cholangiocarcinoma (dCCA) depending on the anatomical location. Until recently, the treatment for advanced CCA has remained highly reliant on chemotherapy, with Gemcitabine plus Cisplatin used in first-line treatment. Recent developments have led to the addition of immune checkpoint blockade (ICB) to the chemotherapy regimen, highlighting the promising potential of immunotherapies for CCA treatment. Despite these developments, most patients still do not benefit from current treatments, and response rates to ICB monotherapy remain modest. This underscores the need to develop more effective immunotherapeutic strategies. A major obstacle to this is the highly heterogenous nature of the disease. CCA tumors exhibit high inter-tumor heterogeneity in terms of anatomical locations, driver mutations, etiologies, and tumor microenvironment (TME) composition, making each patient immunologically distinct and difficult to benefit from a one-fit-all approach. There is a need to stratify patients according to individual disease status to identify immunotherapies and combination therapies that are most beneficial to them. Here we describe the different ways inter-tumor heterogeneity may arise in CCA, including stromal cell abundance, anatomical location, driver mutations, etiologies, TME profile, and tertiary lymphoid structure (TLS) presence. We also discuss what these factors mean to the immune microenvironment and their potential to be used as biomarkers. Careful stratification of patients is crucial in designing personalized medicine to improve survival outcomes and treatment efficacy for CCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb66e953d9482069c69df6565960e118d42cf3b5" target='_blank'>
              Stratifying Cholangiocarcinoma: Tumor Microenvironment, Molecular Drivers, and Novel Immunotherapeutic Approaches.
              </a>
            </td>
          <td>
            Cindy Xinqi Liu, Carmen Chak-Lui Wong
          </td>
          <td>2025-10-14</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Objective
Advanced gastric cancer remains highly refractory to therapy, with limited immunotherapy efficacy due to tumor microenvironment heterogeneity. Primary cilia, microtubule-based organelles involved in tumor progression, remain insufficiently explored in gastric cancer. This study aimed to define primary cilia subtypes and establish prognostic signatures for personalized treatment strategies.


Methods
Bulk transcriptomic data from over 1,500 gastric cancer samples were integrated to define distinct primary cilia subtypes. A primary ciliary phenotype-associated signature (PCS) was established using a multi-machine learning survival framework incorporating ten algorithms. The prognostic predictive value and immunotherapy response prediction capability of PCS were validated across multiple independent cohorts. Single-cell RNA sequencing analysis was performed to identify cellular populations associated with high-PCS phenotype. Causal weighted gene co-expression network analysis (WGCNA) was employed to identify driving factors, followed by functional validation through cell culture experiments and xenograft models.


Results
Two distinct primary cilia subtypes were identified and validated across all cohorts, with C2 patients exhibiting significantly worse overall survival compared to C1 patients. PCS demonstrated robust predictive value for both prognosis and immunotherapy response, with superior accuracy compared to existing models across multiple validation cohorts. High-PCS patients showed reduced tumor purity, increased stromal cell infiltration, and poor response to immunotherapy. Single-cell analysis revealed that fibroblasts had the highest PCS scores and identified a novel secreted modular calcium-binding protein 2 (SMOC2)high myofibroblastic cancer-associated fibroblast (mCAF) population as the key driver of high-PCS phenotype. Functional experiments confirmed that SMOC2 knockdown significantly suppressed gastric cancer cell proliferation, migration, and invasion, while promoting mCAF-to-inflammatory cancer-associated fibroblasts (iCAF) transition.


Conclusions
PCS serves as a robust prognostic biomarker for gastric cancer patients. Additionally, targeting SMOC2 high mCAFs represents a potential therapeutic strategy for patients with high-PCS gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8325aa5e13eb71b8edb3f16c8e7ba4db0f0ba8" target='_blank'>
              SMOC2 high myofibroblastic cancer-associated fibroblast drives primary cilia-associated tumor microenvironment remodeling and poor prognosis in gastric cancer.
              </a>
            </td>
          <td>
            Qiqi Xu, Changjiang Yang, Jingyuan Ning, Yunze Niu, Xuesong Zhao, Long Zhao, Caihong Wang, Shan Wang, Y. Ye, Zhanlong Shen
          </td>
          <td>2025-08-30</td>
          <td>Chinese journal of cancer research = Chung-kuo yen cheng yen chiu</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="T cells recognize antigens and induce specialized gene expression programs (GEPs), enabling functions like proliferation, cytotoxicity and cytokine production. Traditionally, different T cell classes are thought to exhibit mutually exclusive responses, including TH1, TH2 and TH17 programs. However, single-cell RNA sequencing has revealed a continuum of T cell states without clearly distinct subsets, necessitating new analytical frameworks. Here, we introduce T-CellAnnoTator (TCAT), a pipeline that improves T cell characterization by simultaneously quantifying predefined GEPs capturing activation states and cellular subsets. Analyzing 1,700,000 T cells from 700 individuals spanning 38 tissues and five disease contexts, we identify 46 reproducible GEPs reflecting core T cell functions including proliferation, cytotoxicity, exhaustion and effector states. We experimentally demonstrate new activation programs and apply TCAT to characterize activation GEPs that predict immune checkpoint inhibitor response across multiple tumor types. Our software package starCAT generalizes this framework, enabling reproducible annotation in other cell types and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdba0ab80686667b3dcec48b4fa56304b52cf36e" target='_blank'>
              Reproducible single-cell annotation of programs underlying T cell subsets, activation states and functions
              </a>
            </td>
          <td>
            Dylan Kotliar, Michelle Curtis, Ryan Agnew, Kathryn Weinand, A. Nathan, Y. Baglaenko, Kamil Slowikowski, Yu Zhao, P. Sabeti, Deepak A. Rao, Soumya Raychaudhuri
          </td>
          <td>2025-09-03</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) represent a small but critical subpopulation within breast tumors that drive tumor initiation, progression, metastasis, therapeutic resistance, and recurrence. BCSCs are characterized by cell surface markers such as CD44⁺/CD24⁻ and high aldehyde dehydrogenase 1 (ALDH1) activity, and are regulated by pluripotency-associated transcription factors including POU class 5 homeobox 1 (OCT4), SRY-Box transcription factor 2 (SOX2), and Nanog homeobox (NANOG). They exhibit remarkable plasticity and heterogeneity across different breast cancer subtypes, which is further enhanced by epithelial-mesenchymal transition and dynamic interactions with the tumor microenvironment, enabling immune evasion and adaptation to hostile conditions such as hypoxia. Functionally, BCSCs sustain tumorigenesis through self-renewal, contribute to metastatic dissemination as circulating tumor cells, and can enter dormancy in distant niches, leading to late recurrences. Their intrinsic resistance to chemotherapy, radiotherapy, and targeted therapies arises from quiescence, enhanced DNA damage repair, overexpression of drug efflux transporters, and metabolic reprogramming, posing major clinical challenges. Recent advances have improved BCSC identification and isolation through marker-based cell sorting, dye exclusion assays, mammosphere culture, and functional reporter systems. Conventional therapies often spare or enrich BCSCs, highlighting the need for novel treatments. Targeted strategies focusing on stemness-related signaling pathways such as Wnt, Notch, and Hedgehog, as well as regulators including NANOG, ALDH1 and programmed death-ligand 1 (PD-L1), are under investigation. Emerging approaches incorporating natural compounds, nanotechnology, and microenvironmental modulation show promise in overcoming therapeutic resistance. This review integrates current understanding of BCSC biology, heterogeneity, and microenvironmental interactions, providing valuable insights for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0824bd9fb688f1edb70aeb158a043b8788637c" target='_blank'>
              Breast cancer stem cells: Biological roles, microenvironmental interactions, and therapeutic targeting
              </a>
            </td>
          <td>
            Mou Yan
          </td>
          <td>2025-10-11</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer mortality, with minimal improvements in 5-year survival over the past decade. While PDAC originates from premalignant lesions, only a minority of these lesions advance further into malignancy. Distinguishing which lesions are likely to progress would transform enabling timely preventative surgical intervention, but remains a key challenge. In addition to the more well-characterized genetic mutations associated with PDAC onset, recent studies suggest that inflammation in the pancreas microenvironment promotes epithelial chromatin accessibility remodeling to increase their plasticity. This plasticity is thought to enhance cellular sensitivity in response to local signaling cues, which varies drastically across PDAC tumors. We propose that the cell lineage fate of the premalignant epithelial cells is heavily influenced by their surrounding microenvironment. To investigate this, we are using the CosMx spatial transcriptomics platform to profile cellular composition heterogeneity across pancreatic tissue cores from KrasLSL-G12D; Trp53LSL-R172H; Pdx1-Cre (KPC) mouse model of PDAC. By analyzing spatial heterogeneity within the tumor microenvironment, we aim to delineate how local microenvironmental differences might influence epithelial lineage fate decisions into PDAC. Our findings will help understand the factors driving progression from early lesions to invasive cancer, supporting efforts in early PDAC detection and informing clinical strategies for surgical resection.



 Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P. Morton, Crispin J. Miller. Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B118.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb056397ddb7c8e4e0421ffdba57d9fda0a6959d" target='_blank'>
              Abstract B118: Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P Morton, Crispin J. Miller
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053a87c896e7b2228f019eac1392494ac26a840e" target='_blank'>
              A spatial map of MAPK-activated immunosuppressive myeloid populations in pediatric low-grade glioma.
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Romain Sigaud, Evan Puligandla, Bridget Liu, E. Karimi, Alva Annett, M. Rezanejad, Wajih Jawhar, Robert Taylor, Yi Cao, S. Schmid, F. Selt, P. Hernáiz Driever, Svea Horn, P. Sievers, M. Prinz, M. Glatzel, Christian Mawrin, C. Hartmann, C. Monoranu, Liza Konnikova, F. Sahm, Stefan M. Pfister, Olaf Witt, David T. W. Jones, A. Koch, Claudia L Kleinman, David Capper, L. Walsh, N. Jabado, T. Milde
          </td>
          <td>2025-09-15</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="


 Alaska Native peoples experience high colorectal cancer (CRC) incidence and mortality rates, which Alaska tribal health organizations are working to address. To reduce the high rates, it is important to understand the biological mechanisms that drive risk and survival outcomes. The tumor microenvironment (TME) comprises diverse cell types, including cancer, stromal, and immune cells, whose composition and spatial organization may impact prognosis and disease progression. We performed spatial proteomics profiling on tissue microarrays from 97 Alaska Native patients with CRC using Akoya Biosciences’ PhenoCycler-Fusion system. We included 35 patients who died of CRC and 62 matched patients who did not die of CRC and lived at least as long as the patient they were matched to. Patients were matched on age at diagnosis, sex, tumor site, tumor stage, and year at diagnosis. We classified 2.90 million cells into 16 distinct cell types. We quantified immune and stromal cells as proportions of non-epithelial cells and epithelial cells as proportions of total cells. For spatial analysis, we identified eight cellular neighborhoods with distinct local cell composition and calculated spatial proximities between different cell types. We performed multivariable-adjusted logistic regression with generalized estimating equations to investigate associations of CRC-specific death with proportions of cell types and spatial proximities. We estimated odds ratios (OR) with 95% confidence intervals (CI) and calculated the false discovery rate (FDR) to account for multiple comparison using the Benjamin-Hochberg method. We found that higher proportions of stromal cells (OR for doubling of proportion=2.29, 95%CI=1.00-5.26) and higher proportion of M2 macrophages at the tumor center (OR=2.90, 95%CI=1.14-8.22) were associated with increased risks of CRC-specific death among 76 patients with microsatellite stable (MSS) tumors. We observed higher proportions of podoplanin (PDPN)-expressing stromal cells (OR=2.97 for doubling of proportion, 95%CI=1.28-7.08) and M1 macrophages (OR=2.42, 95%CI=1.05-5.69) within cellular neighborhoods predominantly composed of tumor epithelial cells among patients with CRC deaths compared to survivors, although these associations did not remain significant at FDR 0.05. Additionally, we observed that colocalization of PDPN-expressing stromal cells with CD8+ T cells, and colocalization of antigen-presenting cells with plasma cells, was associated with increased risk of CRC-specific death. In contrast, colocalization of B cells with stromal cells was associated with decreased risk (adjusted P=0.04 for all associations). Those associations remained significant among patients with MSS (adjusted P=0.03). This is the first study focusing on the TME cellular landscape among Alaska Native patients with CRC. Our results highlight the prognostic relevance of cellular composition and spatial organization within the TME, suggesting specific stromal and immune cell interactions as potential biomarkers for CRC outcomes.



 Hang Yin, Diana Redwood, Kimberly Smythe, Kristin L. Robinson, Daniel Jones, McGarry Houghton, Kevin C. Barry, Amanda L. Koehne, Elizabeth Donato, Cecilia Yeung, Minggang Lin, James J. Tiesinga, Tabitha A. Harrison, Sushma S. Thomas, Li Hsu, Jane C. Figueiredo, Li Li, Christopher I. Li, Ulrike Peters, Jeroen R. Huyghe. Spatial-resolved single-cell analysis of the tumor microenvironment in Alaska Native colorectal cancer patients reveals novel biomarkers for prognosis [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr A087.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fe434533ff8a8160b5fcdfdf886e32d3184d6f4" target='_blank'>
              Abstract A087: Spatial-resolved single-cell analysis of the tumor microenvironment in Alaska Native colorectal cancer patients reveals novel biomarkers for prognosis
              </a>
            </td>
          <td>
            Hang Yin, Diana Redwood, K. Smythe, Kristin L. Robinson, Daniel Jones, Mcgarry Houghton, K. C. Barry, Amanda L Koehne, Elizabeth Donato, Cecilia Yeung, MingGang Lin, James Tiesinga, T. Harrison, Sushma Thomas, Li Hsu, Jane C Figueiredo, Li Li, Christopher I Li, Ulrike Peters, J. Huyghe
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="MYCN amplification without concurrent RB1 mutations characterizes a rare yet highly aggressive subtype of retinoblastoma; however, its precise developmental origins and therapeutic vulnerabilities remain incompletely understood. Here, we modeled this subtype by lentiviral-mediated MYCN overexpression in human pluripotent stem cell-derived retinal organoids, revealing a discrete developmental window (days 70–120) during which retinal progenitors showed heightened susceptibility to transformation. Tumors arising in this period exhibited robust proliferation, expressed SOX2, and lacked CRX, consistent with origin from primitive retinal progenitors. MYCN-overexpressing organoids generated stable cell lines that reproducibly gave rise to MYCN-driven tumors when xenografted into immunodeficient mice. Transcriptomic profiling demonstrated that MYCN-overexpressing organoids closely recapitulated molecular features of patient-derived MYCN-amplified retinoblastomas, particularly through activation of MYC/E2F and mTORC1 signaling pathways. Pharmacological screening further identified distinct therapeutic vulnerabilities, demonstrating distinct subtype-specific sensitivity of MYCN-driven cells to transcriptional inhibitors (THZ1, Flavopiridol) and the cell-cycle inhibitor Volasertib, indicative of a unique oncogene-addicted state compared to RB1-deficient retinoblastoma cells. Collectively, our study elucidates the developmental and molecular mechanisms underpinning MYCN-driven retinoblastoma, establishes a robust and clinically relevant human retinal organoid platform, and highlights targeted transcriptional inhibition as a promising therapeutic approach for this aggressive pediatric cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7456982cd84b09cab50502a3b5ab5c8e7a74c4cd" target='_blank'>
              Human Retinal Organoid Modeling Defines Developmental Window and Therapeutic Vulnerabilities in MYCN-Amplified Retinoblastoma
              </a>
            </td>
          <td>
            Jinkyu Park, Gang Cui, Jiyun Hong, H. Jeong, Minseok Han, Min Seok Choi, Jeong Ah Lim, Sanguk Han, Christopher Seungkyu Lee, Min Kim, Sangwoo Kim, Junwon Lee, S. Byeon
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f94703bbaec5cc0e9f11f2c621c2b6ffe36e730" target='_blank'>
              Epithelial-Mesenchymal Plasticity is Associated with Immunosuppressive Features in Canine Mammary Carcinomas
              </a>
            </td>
          <td>
            Kimaya M. Bakhle, Sophie Nelissen, Lynna Li, Ioannis Ntekas, Iwijn De Vlaminck, Santiago Peralta, , Anushka Dongre
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background/Aims
Patient-derived organoids (PDOs) are promising preclinical models that replicate critical tumor features. However, intratumoral heterogeneity challenges the clinical utility of PDOs, especially in capturing diverse tumor cell subpopulations.


Methods
Single-cell transcriptomics was used to analyze PDOs from distinct sites within a single gastric cancer tumor, aiming to assess their ability to reflect intratumoral heterogeneity.


Results
The PDOs displayed similarities in gene expression but also exhibited distinct profiles. Single-cell analysis of PDOs revealed upregulation of markers for neuroendocrine tumors, which was validated via immunohistochemistry staining of neuron-specific enolase in the primary tumor. Notably, heat shock proteins showed significant variability among the PDOs, impacting immune responses. Tumors with abundant heat shock proteins are reported to have increased cytotoxic T cell activity.


Conclusions
Intratumoral heterogeneity poses challenges for PDO-based models, highlighting the need for comprehensive assessment. Despite their limitations, PDOs offer valuable insights into precision medicine for patients with gastric cancer, aiding in the development of therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9154970e5ea83aed48cb2a2d344b23752aade163" target='_blank'>
              Patient-Derived Organoids from Multiple Sites of a Single Tumor Recapitulates Intratumoral Heterogeneity in Patients with Gastric Cancer.
              </a>
            </td>
          <td>
            Bo Kyung Yoon, Y. Bae, Yeonjin Je, Seyeon Joo, Yuna Kim, Su-Jin Shin, Sungsoon Fang, Jie-Hyun Kim
          </td>
          <td>2025-09-05</td>
          <td>Gut and liver</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b4810245a3c13ba422508c74f83c676e4a47c4" target='_blank'>
              High-resolution spatial analysis reveals pregnenolone metabolism as a potential target for immuno-suppressive microenvironment of BRAF mutant colorectal cancer
              </a>
            </td>
          <td>
            Xicheng Wang, Zihao Zhao, Lian Liu, Jie Cheng, Yi Ba
          </td>
          <td>2025-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance.
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b89b91a85a77397158e35e2d66976c843568dbb" target='_blank'>
              Studying a human genetic model of lung squamous cell carcinoma with organotypic cultures and xenografts uncovers distinct advantages of each system and implicates NOTCH1 loss in tumour development
              </a>
            </td>
          <td>
            Julia Ogden, Robert Sellers, Anthony Oojageer, Sudhakar Sahoo, Caroline Dive, Carlos López-García
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Interpatient tumor heterogeneity manifests as multimodal distributions across genomic, transcriptomic, and microenvironmental profiles. This fundamentally violates the unimodal assumption of conventional machine learning models, impairing immune checkpoint blockade (ICB) response prediction. To resolve this limitation, we propose a heterogeneity-optimized framework that applies K-means clustering to stratify patients into biologically distinct hot-tumor and cold-tumor subgroups, demonstrating superiority over hierarchical/DBSCAN clustering. Subsequently, heterogeneity-optimized predictive models–a support vector machine for hot-tumor subtypes and a random forest for cold-tumor subtypes–were developed utilizing seven heterogeneity-associated biomarkers to circumvent unimodal constraints. The proposed model significantly enhances ICB response prediction in melanoma, NSCLC, other cancer types, and pan-cancer datasets, achieving a mean accuracy gain of at least 1.24% compared to 11 baseline methods. This performance improvement was consistently validated in an independent external cohort. Furthermore, the approach enables biologically interpretable precision immunotherapy by explicitly modeling multimodal heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea2e7a884fb718694551d16dff86f3e29cf9498" target='_blank'>
              Heterogeneity-optimized method for predicting immune checkpoint blockade response
              </a>
            </td>
          <td>
            Juan Liang, Qihang Guo, Shan Xiang, Yun Wang, Juntao Li
          </td>
          <td>2025-09-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Uveal melanoma (UM), the most common primary intraocular malignancy in adults, presents significant clinical challenges due to its high metastatic potential, pronounced hepatic tropism, and poor prognosis upon systemic dissemination. Despite established local therapies, nearly half of patients develop distant metastases, highlighting an urgent need for more effective systemic strategies. Recent advances in single-cell omics technologies (e.g., scRNA-seq, scATAC-seq, spatial transcriptomics) have revolutionized our understanding of UM pathobiology. These approaches have meticulously delineated the complex tumor heterogeneity, immunosuppressive microenvironment, and key molecular drivers—including novel macrophage subsets (e.g., immunosuppressive MΦ-C4), senescent endothelial cells, and non-canonical immune checkpoint expression—providing unprecedented resolution for identifying actionable therapeutic targets. Concurrently, innovations in materials science and biomedical engineering offer transformative opportunities for precision therapy. Engineered nanocarriers, biodegradable implants, and advanced gene therapy vectors (e.g., tropism-enhanced AAVs, CRISPR-Cas9 systems) enabled targeted drug delivery, sustained release, and genetic modulation tailored to the eye’s unique anatomy and immune privilege. This review synthesizes these converging frontiers, outlining how the integration of multi-omics insights with smart biomaterials can overcome current therapeutic limitations. We catalog emerging material-based platforms applicable to UM and summarize validated molecular targets (e.g., GNAQ/GNA11, YAP/TAZ, BAP1, c-Met, CXCR4). Furthermore, we propose an interdisciplinary paradigm spanning combinatorial targeted therapies, immunomodulation, minimally invasive devices (e.g., robotic radiosurgery), and engineered delivery systems. By bridging mechanistic discovery with translational engineering, this synergy holds significant promise for advancing precision medicine and improving clinical outcomes in UM, ultimately facilitating the transition from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0323cee7d6c89a49fe569ae452832b7a890d38bd" target='_blank'>
              Integrating single-cell omics and materials science for uveal melanoma: from mechanistic insights to precision therapeutics
              </a>
            </td>
          <td>
            Shouyong Fu, Changfei Li
          </td>
          <td>2025-09-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite significant advances in early detection and therapeutic interventions, breast cancer persists as the most frequently diagnosed malignancy and the leading cause of cancer-related deaths among women globally. Although multiple prognostic signatures have been proposed, their predictive power and clinical applicability remain limited. In this study, we utilized an integrated approach combining single-cell multi-omics analysis with machine learning to comprehensively examine the clinical relevance of mitochondrial-related gene sets in TCGA-BRCA and developed a mitochondrial gene set scoring system, termed MitoScore. Based on the median of MitoScore, BRCA patients were classified into high-risk and low-risk groups. Our multi-omics analysis revealed that BRCA patients with higher Mitoscore exhibited poorer prognoses compared to those with lower MitoScore. The predictive ability of the model was successfully validated using an external GEO dataset. Immune infiltration analysis further indicated that high-risk group contributed to an immunosuppressive tumor microenvironment, marked by a decrease in CD8+ T cells. This finding was corroborated by cell–cell communication analysis, which revealed abnormal activation of the MIF-CXCR4 signaling axis. Notably, carbamoyl-phosphate synthetase 1 (CPS1) emerged as the most critical factor in the MitoScore model. CPS1 was significantly upregulated in malignant BRCA cells and was closely associated with tumor aggressiveness and poor prognosis. Both in vitro and in vivo experiments confirmed the role of CPS1 in BRCA. In this study, CPS1 was identified as a metabolism-related oncogene, suggesting its involvement in enhancing glycolysis and mediating immune evasion in breast cancer cells. Knockdown of CPS1 markedly improved the efficacy of anti-PD-1 therapy in immunocompetent mice, prolonged survival, and ameliorated the immunosuppressive tumor microenvironment. In summary, this study underscores the clinical significance of mitochondrial genes in BRCA, proposes Mitoscore for precise patient stratification and personalized treatment, and highlights the therapeutic potential of targeting CPS1 to simultaneously modulate glycolysis and immune evasion mechanisms. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-19890-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61ef12279d51b95d8689db9b307e77c627791d92" target='_blank'>
              Single-cell multi-omics uncovers CPS1 as a breast cancer immune evasion therapeutic target
              </a>
            </td>
          <td>
            Jian Yue, Fang Wen, Dele He, Sheng Chen, Guoxing Huang
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d8b0d319f8fcb0e9c8819010ea4b2ab4abcd49" target='_blank'>
              HIF-1α and RhoA Drive Enhanced Motility and Aerotaxis of Polyaneuploid Prostate Cancer Cells in Hypoxia
              </a>
            </td>
          <td>
            Noreen Hosny, Shengkai Li, S. Amend, Robert A. Gatenby, Kenneth J. Pienta, Joel Brown, Junle Qu, Robert H. Austin
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Ovarian cancer (OvCa) remains largely refractory to immune checkpoint blockade (ICB), owing to a profoundly immunosuppressive tumor microenvironment (TME). Single-cell RNA sequencing (scRNA-seq) of patient tumors has revealed a TME enriched in immunosuppressive macrophages and characterized by exclusion of T cells from the tumor core, features strongly associated with ICB resistance. However, the molecular mechanisms underlying the establishment of this immune landscape remain poorly understood. To identify regulators of OvCa immunity, we employed Perturb-map, a first-of-its-kind spatial functional genomics platform that enables multiplexed CRISPR/Cas9 knockout of candidate genes while preserving the spatial context of tumor–TME interactions in situ. We targeted 35 genes, prioritized through ligand–receptor interaction analysis of human OvCa scRNA-seq data, in an immunocompetent OvCa mouse model. Among these, IL-4 emerged as a key driver of ICB resistance. While IL-4 knockout did not affect tumor growth in untreated animals, it substantially remodeled the local TME and rendered tumors responsive to anti-PD-1 therapy. Unexpectedly, we found that both mouse and human ovarian cancer cells are a source of IL-4, a cytokine traditionally restricted to immune cells. Analysis of patient tumors revealed that tumor-associated macrophages exhibit a robust IL-4, induced transcriptional signature, implicating IL-4 in shaping the immunosuppressive TME in human disease, consistent with our preclinical findings (Cell, 2024). Furthermore, we observed that standard-of-care chemotherapy regimens, including carboplatin, paclitaxel, and cisplatin—induce a dose-dependent increase in IL-4 expression in both murine and human OvCa cell lines. This chemotherapy-induced IL-4 upregulation may contribute to immune evasion and recurrence following initial therapeutic response. Therapeutically, combined blockade of IL-4Rα and PD-1 significantly delayed tumor progression in vivo compared to monotherapy, suggesting that IL-4 signaling is both a necessary and targetable mechanism of immune resistance. These findings are clinically actionable, as Dupilumab, an FDA-approved anti-IL-4Rα antibody, is already in clinical use and well-tolerated. Our results uncover a previously unrecognized, chemotherapy-induced immunoregulatory axis in ovarian cancer and provide a strong rationale for therapeutic strategies that combine IL-4 pathway inhibition with ICB to overcome resistance and improve patient outcomes.



 Gurkan Mullaoglu, Hunter Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian Brown, Alessia Baccarini. Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462188b4ccc770fdbf7b28c6a3fda711e0408972" target='_blank'>
              Abstract PR015: Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Gurkan Mullaoglu, Hunter T. Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian D. Brown, Alessia Baccarini
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38e3e2e1b934125c90539e7b6826f0bb1928fb1d" target='_blank'>
              Image-based phenotypic sorting of synthetic cells
              </a>
            </td>
          <td>
            Marijn van den Brink, Marlena Stam, Nico J. Claassens, C. Danelon
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Glioblastoma is the most common malignant primary brain tumor in adults and associated with a poor prognosis and a median survival of approximately 15 months, largely due to the lack of effective therapies and inevitable recurrence. A key challenge is the spatial and cellular heterogeneity, evident on MRI as a necrotic center, a contrast-enhancing tumor core, and a non-enhancing, T2/FLAIR-hyperintense infiltration zone. The diffuse infiltration of tumor cells into surrounding brain tissue impedes complete surgical resection and drives relapse from persisting residual cells. While regional molecular differences have been well characterized, their functional implications - particularly drug sensitivities - remain insufficiently understood.



 To address this, we performed MRI- and 5-aminolevulinic acid (5-ALA)-guided tissue sampling from the tumor core and from the infiltration zone of patients with glioblastoma. We utilized pharmacoscopy—a single-cell, microscopy-based drug screening platform—to evaluate region- and tumor cell type-specific drug responses. Cancer cell types were defined based on differential expression of nestin (marking stem-like cells) and S100B (marking more differentiated tumor cells).



 Profiling a library of 59 drugs across region-specific tumor samples from 23 patients revealed pronounced heterogeneity in drug responses across patients, tumor regions and tumor cell states. To overcome this heterogeneity, we computationally predicted complementary drug pairs and validated 20 drug combinations in an independent cohort. The combinations demonstrated enhanced efficacy compared to single agents and effectively addressed glioblastoma’s spatial, cellular, and inter-patient heterogeneity. Notably, drug combinations of oncology and neuroactive drugs outperformed those involving agents from the same drug class (oncology + oncology or neuroactive + neuroactive). Ongoing mechanistical explorations reveal first insights into mechanisms of the complementary drug efficacy.



 Our findings underscore the limitations of monotherapies for patients with glioblastoma and highlight the potential of rationally designed drug combinations tailored to the tumor’s spatial and functional complexity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60062d1c4566c5af52ee0940de47f3ed99103b0" target='_blank'>
              OS12.5.A SPATIALLY RESOLVED FUNCTIONAL PROFILING FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Buck, S. Lee, I. Sakic, R. Wegmann, J. Mena, F. Vasella, J. Friesen, L. Regli, E. Le Rhun, A. Zeitlberger, N. Tatari, G. Hutter, M. Neidert, M. Weller, B. Snijder, T. Weiss
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background/Objectives: Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with poor prognosis. T cell exhaustion (TEX) is a key factor in tumor immune evasion and therapeutic resistance. In this study, we integrated single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data to characterize TEX-related transcriptional features in HCC. Methods: We first computed TEX scores for each sample using a curated 65-gene signature and classified them into high-TEX and low-TEX groups by the median score. Differentially expressed genes were identified separately in scRNA-seq and bulk RNA-seq data, then intersected to retain shared candidates. A 26-gene prognostic signature was derived from these candidates via univariate Cox and LASSO regression analysis. Results: The high-TEX group exhibited increased expression of immune checkpoint molecules and antigen presentation molecules, suggesting a tumor microenvironment that is more immunosuppressive but potentially more responsive to immunotherapy. Functional enrichment analysis and protein–protein interaction (PPI) network construction further validated the roles of these genes in immune regulation and tumor progression. Conclusions: This study provides a comprehensive characterization of the TEX landscape in HCC and identifies a robust gene signature associated with prognosis and immune infiltration. These findings highlight the potential of targeting TEX-related genes for personalized immunotherapeutic strategies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a1103280627c731b735d281df63ab3e636c5d0" target='_blank'>
              Bulk and Single-Cell Transcriptomes Reveal Exhausted Signature in Prognosis of Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Ruixin Chun, Haisen Ni, Ziyi Zhao, Chunlong Zhang
          </td>
          <td>2025-08-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="


 Multi-modal chemotherapy, surgical resection, and radiation therapy (if applicable) make up the current standard of care for Atypical Teratoid Rhabdoid Tumors (ATRT), but they are insufficient. Up to 50% of patients experience relapse, but no standard treatment options have been established for ATRT recurrence. The malignant cells that persist after chemotherapy treatment may significantly contribute to this tumor rebound. Our principal goal is to track shifts in cell states and phenotypes under the selection of chemotherapy by combining CloneSifter barcoding technology with single cell RNA-seq. Patient derived ATRT cells, barcoded with sgRNA identifiers, are placed under the selection of a sub-lethal drug dose, and barcodes are tracked over 7 days. After treating the cells with the following chemotherapy agents - Cisplatin, Etoposide, Doxorubicin, Methotrexate, and Cyclophosphamide - barcode abundance profiles are compared between pre and post drug treated samples using hierarchical clustering. This analysis reveals that replicates subjected to the same drug treatment tend to show the most similar patterns of barcode abundance, suggesting that this drug persistence is intrinsic and pre-determined prior to treatment rather than stochastically acquired. Furthermore, a one-sided paired Wilcoxon test is leveraged to identify significant barcode enrichment in each of the drug treated replicates relative to their vehicle controls, differentiating between chemo-persistent versus chemo-sensitive barcodes. Competitive barcodes are identified under Etoposide, Cisplatin, and Methotrexate treatments and mapped to the single cell RNA-seq data, where a unique transcriptome profile can be generated for each barcode. Comparisons between transcriptome profiles of enriched versus depleted barcodes under drug treatment will reveal the underlying factors that provide a fitness advantage for chemo-persistent populations. Investigating the dynamics of chemotherapy-persistence in ATRT cell lines can help uncover vulnerabilities that may be exploited to begin overcoming chemotherapy-induced ATRT recurrence.



 Kelly Y Cai, Shriya Rangaswamy, April Apfelbaum, Jared Collins, Molina Das, Joelle Straehla, David Root, Tom Rosenberg, Susan Chi, Rameen Beroukhim, Pratiti Bandopadhayay. Factors Driving Chemo-persistence in Atypical Teratoid Rhabdoid Tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/032d4c5b96b8782c64d45452f34ed6f6fe663882" target='_blank'>
              Abstract B016: Factors Driving Chemo-persistence in Atypical Teratoid Rhabdoid Tumors
              </a>
            </td>
          <td>
            Kelly Y Cai, Shriya Rangaswamy, April A Apfelbaum, Jared Collins, Molina Das, Joelle Straehla, David E. Root, Tom Rosenberg, Susan Chi, R. Beroukhim, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>151</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e8d2f1aaaa7f0e435a0bcfed4046ad25f2a01b" target='_blank'>
              Deciphering macrophage heterogeneity and factors driving M2 polarization in lung adenocarcinoma through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Meiling Sheng, Beiwei Yu, Qunzhi Wang, Yuanchao Xiao, Ming Ling, Xiaoming Wu
          </td>
          <td>2025-10-16</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern and a high rate of relapse. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, i.e. a hypoxic niche, a leading-edge niche, and a peri-vascular niche, in which malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment, including immune cells, astrocytes, and vascular cells.



 By analysis of single-cell transcriptomic data of human GBM and transgenic mouse models of GBM, we unexpectedly identified pericytes intimately associated with the endothelium as the most active paracrine signaling hub within the tumor parenchyma. Exclusive signaling axes emanating from pericytes were received by endothelial cells, malignant cells, astrocytes, and immune cells. Depletion of pericytes through genetic engineering in several different transgenic and orthotopic mouse models of GBM demonstrated accelerated tumor progression, a disrupted blood-brain-barrier, and premature death of pericyte-poor mice. Mechanistic studies revealed that pericyte deficiency altered the cellular composition of GBM, remodeled the endothelium, and impacted on the immune cell landscape. Specifically, endothelial cells deprived of pericyte association upregulated Colony Stimulating Factor (CSF)-1, which in turn attracted peri-vascular tumor-associated macrophages polarized towards an immune-suppressive phenotype. In the absence of pericytes, the recruited macrophages expressed Hepatocyte Growth Factor (HGF), which reinforced activation of its receptor tyrosine kinase c-Met on a subset of GBM cells harboring an extreme mesenchymal-like state. Indeed, orthotopic implantation of isolated c-Met-expressing GBM cells corroborated their superior tumor-initiating capability and invasive phenotype, compared to cells negative for c-Met.



 We infer that the pericyte represents a critical modulator of GBM development by orchestrating a tumor-suppressive microenvironment; our findings thus highlight the importance of pericyte preservation in the face of current and future GBM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a67e2a70cd46f28d9a303424953a15720ec17e" target='_blank'>
              P02.01.B PERICYTES ORCHESTRATE A TUMOR-SUPPRESSIVE MICROENVIRONMENT BY IMPINGING ON THE CROSSTALK BETWEEN MACROPHAGES AND TUMOR-INITIATING CELLS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, M. Safaee Talkhoncheh, B. Phung, E. Cordero, E. Johansson, R. Rosberg, G. B. Jönsson, A. Pietras, K. Pietras
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most common brain cancers diagnosed in children, often leading to life-long neurological impairments. Whilst targeted inhibitors exist, their effects are often short-lived, highlighting the urgent need for innovative treatments. Notably, some pLGG tumors spontaneously regress with age, suggesting that changes in the tumor microenvironment can arrest tumor growth. In this work, we aim to characterize which signaling interactions promote the growth of pLGG cells and how these interactions change dynamically with time. To achieve this, we have optimized a protocol for introducing pLGG mutations into developing mouse brains via in utero electroporation. This study revealed that brains harboring the common KIAA1549::BRAF fusion mutation have a striking phenotype characterized by increased astrocyte reactivity and myeloid cell infiltration. To better understand how signals derived from immune cells may impact pLGG cell fitness, we are additionally performing a high-throughput cytokine screen. We are using barcoded cytokine constructs tethered to the cell membrane to comprehensively screen the impact of cytokine signaling on the growth of neural stem cells engineered to overexpress common pLGG mutations. Together this work aims to unravel which signaling factors govern the age-dependent growth of pLGG tumors, which may reveal targetable aspects of the tumor microenvironment amenable to therapeutic intervention.



 Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy Chang, Ria Kedia, Michelle Boisvert, David Jones, Timothy Phoenix, Pratiti Bandopadhayay. Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f149e2a5701b9900d3e20ea1c251a5956b1b56d2" target='_blank'>
              Abstract B037: Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas
              </a>
            </td>
          <td>
            Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy H. Chang, Ria Kedia, Michelle Boisvert, David Jones, T. Phoenix, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Abstract We introduce consensus MSClustering, an unsupervised hierarchical network approach that integrates multi-omics data to identify molecular subtypes and conserved pathways across diverse cancers. Using a novel heterogeneity index, we selected 167 key genes with functionally coherent roles validated through Gene Ontology analysis. Applied to 2439 tumors spanning 10 cancer types—and successfully extended to 2675 tumors (12 types) including cases with incomplete molecular data—MSClustering demonstrated: (i) precise classification of major cancer types and breast cancer molecular subtypes; (ii) discovery of novel pan-cancer squamous metaplastic signatures; (iii) exceptional prognostic stratification (log-rank P = 2.3 × 10−46); and (iv) superior performance over existing methods (COCA/SNF) in classification accuracy, cluster robustness, and computational efficiency. The method’s multi-scale architecture uniquely resolves breast cancer heterogeneity across biological resolution levels. Pathway analysis further revealed four key oncogenic programs—proteoglycan signaling, chromosomal stability, VEGF-mediated angiogenesis, and drug metabolism—along with disruptions in immune and digestive system functions. This integrative framework marks a significant advancement in cancer genomics by enabling more refined molecular classification, enhanced prognostic insights, and deeper understanding of disease mechanisms. These results highlight the potential of MSClustering to inform the development of clinically relevant biomarkers and support more personalized strategies in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84e7ad2ae2f844d0a7fbfd4f72ca4d504d9f4a62" target='_blank'>
              Mapping cancer heterogeneity: a consensus network approach to subtypes and pathways
              </a>
            </td>
          <td>
            G. Hu, Hsin-Wei Chen, Chi-Ming Chen
          </td>
          <td>2025-08-31</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Capillary malformation (CM) is a congenital vascular anomaly affecting the skin, mucosa, and brain, yet the understanding of its vascular pathogenesis remains limited. Methods We applied spatial whole-transcriptome profiling (GeoMx) and gene set enrichment analysis within CM lesions at single vasculature level. Differentially expressed genes were validated by immunofluorescence staining. Phosphoproteomics was profiled to uncover lesion- wide phosphorylation sites on proteins. Single-cell RNA sequencing was performed on CM- derived induced pluripotent stem cells (iPSCs) to determine differentiation trajectories of lesional vascular lineages. In silico gene perturbation was used to predict candidate genes for modulating vascular pathological progression, followed by functional validation in CM iPSC-derived endothelial cells (ECs) using a Tet-on system. Results A spatial transcriptomic atlas was constructed, and pathological landscape of individual CM vasculature was delineated. CM vessels exhibited hallmarks of endothelial-to-mesenchymal transition (EndMT), including disruption of adherens junctions (AJs), vascular identity transitions, and metabolic remodeling. Phosphoproteomics confirmed that differentially phosphorylated proteins were enriched in EndMT- and AJ-related pathways. Aberrant expression of venous transcriptional factor NR2F2 was observed in lesional ECs and correlated with progressive enlargement from capillaries to larger-caliber vessels containing multiple layers of smooth muscle cells (SMCs). In CM iPSCs, differentiation course yielded reduced ECs but increased SMCs. In silico knockout simulation predicted NR2F2 as a crucial regulator of facilitating SMC phenotype in CM. Consistently, enforced NR2F2 expression during iPSC differentiation suppressed endothelial markers while inducing SMC-associated genes. Conclusions Single CM vasculature displays pathological hallmarks characterized by EndMT and AJ disruption, leading to progressive vascular remodeling. NR2F2 functions as a central regulatory factor orchestrating the acquisition of the SMC phenotype, thereby representing a potential therapeutic target in CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1fa98536c86180270a8a6d151af2637cfd03767" target='_blank'>
              Single-vessel transcriptome map pathological landscapes and reveal NR2F2-mediated smooth muscle cell phenotype acquisition in capillary malformations
              </a>
            </td>
          <td>
            Vi Nguyen, Irving Mao, Siwuxie He, Isabella Castellanos, Mackenzie Azuero, Marcelo L Hochman, Yin Rong, L. Pernomian, Elliott H Chen, Harold I Friedman, Yan-Hua Chen, Qun Lu, Daping Fan, Camilla F Wenceslau, Dong-bao Chen, Stuart Nelson, Anil G Jegga, Yunguan Wang, Wenbin Tan
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains a highly lethal malignancy, in part due to its heterogeneous tumor microenvironment. Most PDAC tumors are poorly vascularized and embedded within densely fibrotic stroma; however, in prior computational analyses, we identified a subset of PDAC tumors with hyper-vascular features. In this study, we sought to determine the extent of vascular heterogeneity in PDAC and its functional consequences. Analysis of human tumor specimens revealed that approximately 10-20% of human tumors display features of hyper-vascularity. Using a panel of clonal murine cell lines that recapitulate this heterogeneity, we demonstrate that VEGFA-mediated angiogenesis does not account for the increased vessel density. Functionally, we further show that hyper-vascular tumors display enhanced sensitivity to both chemotherapy and anti-angiogenic therapy. These findings help explain the limited efficacy of VEGFA-targeted therapies in unselected PDAC cohorts and highlight vessel density as a potential biomarker for patient stratification.



 Samantha B. Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo I. Orlen, Samuel D. Zwernik, Kailee M. Hartway, Nina G. Steele, Agnieszka K. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger. Vascular heterogeneity influences treatment response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05cec4b9af7cf61e156104d3e634ae5d707e4e9c" target='_blank'>
              Abstract A127: Vascular heterogeneity influences treatment response in pancreatic cancer
              </a>
            </td>
          <td>
            Samantha B Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo Orlen, Samuel D Zwernik, Kailee M Hartway, Nina G. Steele, A. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Merging evidence suggests that cellular fitness competition between malignant cells and the tumor microenvironment (TME) dictates tumor progression. We recently identified Flower isoforms as molecular “fitness fingerprints,” where cancer cells expressing Flower-Win gain dominance over surrounding cells marked with Flower-Lose. In this study, we uncover that loss of stromal fitness is not an inherent trait but rather a cancer-driven event. Tumor cells secrete a specific long non-coding RNA, Tu-Stroma, via exosomes, which reprograms splicing of the Flower locus in TME cells, enforcing Flower-Lose expression and thereby reducing stromal competitiveness. This cancer-induced shift establishes a permissive niche that accelerates growth, metastatic dissemination, and ultimately shapes host survival in ovarian cancer. Therapeutically, blockade of Flower using a fully humanized monoclonal antibody (Aenya) markedly suppressed tumor expansion and peritoneal spread in patient-derived xenograft models. Importantly, prophylactic administration preserved the integrity of intraperitoneal organs even in the presence of highly aggressive tumor cells. These findings highlight a previously unrecognized mechanism of cancer-driven stromal takeover and establish Flower as a first-in-class therapeutic target with broad implications for preventing metastatic colonization.



 Rajan G. Gogna. Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0864f00ed3ab31470a0c2e77e5565e26d50bb11a" target='_blank'>
              Abstract IA008: Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer
              </a>
            </td>
          <td>
            Rajan Gogna
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Treatment of locally advanced unresectable pancreatic cancer remains a major clinical challenge due to pronounced heterogeneity and resistance to standard regimens. Increasing evidence highlights the critical role of the tumor microenvironment (TME) in shaping therapeutic response and driving drug resistance. In this minireview, we summarize recent advances in TME phenotyping and its potential to guide precision therapy. A four-dimensional framework integrating stromal, immune, genomic, and metabolic features has been proposed to better characterize TME heterogeneity. Preclinical and clinical studies indicate that strategies targeting the stroma, modulating immunity, or exploiting genomic vulnerabilities such as homologous recombination deficiency may enhance the efficacy of chemotherapy, immunotherapy, and targeted agents. Dynamic biomarkers, including circulating tumor DNA and carbohydrate antigen 19-9, also show promise for real-time therapy adaptation, although their clinical application remains limited. By synthesizing current evidence, we emphasize the importance of individualized treatment strategies that account for TME complexity. While encouraging, the translation of multiomics phenotyping and biomarker monitoring into routine clinical practice requires standardization, prospective validation, and integration of novel technologies. Future research should focus on establishing reproducible TME-guided models to enable dynamic and personalized therapy for patients with unresectable pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9ebed50c75b679e8298f471ae2f8ee5933e4197" target='_blank'>
              Tumor microenvironment phenotyping guides precision therapy in unresectable pancreatic cancer
              </a>
            </td>
          <td>
            Kai Zhao, Ming-Ming Xiao, Yong-Sheng Yang, Xiao Xiao
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The molecular landscapes of epithelial ovarian cancers (EOCs) have largely been profiled using bulk sequencing techniques. Consequently, biologically relevant signals from the tumor and its microenvironment (hereafter stroma) are often conflated, dampening both tumor and stroma-specific observations.



 We profiled the carcinoma and stroma compartments separately across clear cell (CCOC), endometrioid (ENOC), and high-grade serous ovarian cancer (HGSC) using laser capture microdissection followed by sample-matched whole genome bisulfite sequencing and RNA-seq.



 Histotype designations drove sample segregation in the carcinoma compartment yet were not sufficient to explain top expression or methylation variations in the stroma. Unsupervised clustering of the top variable genes in the stroma identified four stromal subtypes (S1-S4) shared across the three histotypes. Of these, S1, S2, and S4 were the most distinguishable and could be recovered in external datasets. S1 was enriched for RNA splicing processes (SFSWAP, RBM25), S2 for metabolism (GATM, ALDH1A1) and S4 for remodeling of the extracellular matrix (FN1, POSTN, SFRP2). While the characteristics of S4 echo previous reports of the desmoplastic/highly fibrous stroma reported in HGSC, we show that S4 is also present in CCOC and ENOC tumors. Using HGSC samples from TCGA, we observed a strong correlation of S4 markers with the Mesenchymal subtype and an inverse correlation of S4 and S2 markers, suggesting mutual exclusivity of these stromal subtypes. Examining the TCGA-derived subtype markers in our microdissected data revealed that the canonical Immunoreactive and Mesenchymal subtypes were driven by signals from the stroma, rather than the carcinoma. Conversely, Differentiated and Proliferative markers were enriched in the carcinoma. Clustering of pure carcinomas revealed a Proliferative versus Differentiated dichotomy within the carcinomas, particularly within ENOC and HGSC tumors. CCOC tumors were systematically depleted of the Proliferative subtype, consistent with their indolent nature. We next investigated if genetic, epigenetic or transcriptional cues in the carcinoma compartment are associated with the characteristic of its TME but failed to detect differences at 5% FDR. This suggests that the stromal subtype may not be shaped solely by instructions from the cancer cells and that a more complex, reciprocal cross-talk exists between the carcinoma and its TME.



 Epigenetic and transcriptional differences in CCOC, ENOC and HGSC carcinomas can be largely attributed to histotype, yet this is not the case for their associated TMEs. With the growing importance of TME in influencing patient outcomes, and response to treatment, understanding the nature of a tumor’s TME is crucial. Thus, coupling the stromal subtypes (S1-S4) with the well-established histotype designations of EOCs may provide a more robust way to classify EOCs and inform optimal treatment strategies that account for the characteristic of the tumor stroma.



 Svetlana Djirackor, Karolin Heinze, Ian Beddows, David Sokol, Bianca Ribeiro de Souza, Martin Koebel, Marie Adams, Michael Anglesio, Hui Shen. Parallel epigenetic and transcriptomic profiling of carcinoma and stroma compartments across major ovarian cancer histotypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B007.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67d3fdc66021784b9ea5f2df0bff0c3026e49fb0" target='_blank'>
              Abstract B007: Parallel epigenetic and transcriptomic profiling of carcinoma and stroma compartments across major ovarian cancer histotypes
              </a>
            </td>
          <td>
            Svetlana Djirackor, Karolin Heinze, Ian Beddows, David Sokol, Bianca Ribeiro de Souza, M. Koebel, Marie Adams, M. Anglesio, Hui Shen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a696fa9baeadfe94d2a764547f5853c287c52816" target='_blank'>
              Hedgehog signaling drives glial cell plasticity and oncogenic reprogramming in gastroenteropancreatic neuroendocrine neoplasms
              </a>
            </td>
          <td>
            Suzann Duan, AnneLeigh Twer, Ateh Zinkeng, Ricky A. Sontz, Juanita L. Merchant
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b1a443d05c1334b2d0a0dff8460a523dfdb2d5" target='_blank'>
              A three-dimensional ex vivo model recapitulates in vivo features and drug resistance phenotypes in childhood acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Magdalini Kanari, Iria Jimenez Garcia, Fabio Steffen, L. Krattiger, Charles Bataclan, Wangjie Liu, Benjamin R. Simona, Bart Deplancke, O. Naveiras, M. Ehrbar, B. Bornhauser, Jean-Pierre Bourquin
          </td>
          <td>2025-09-10</td>
          <td>Leukemia</td>
          <td>1</td>
          <td>46</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal tumor. PDAC also shows poor response to many conventional treatments and is refractory to immunotherapy. We hypothesize that these challenges arise from the highly heterogeneous and immunosuppressive tumor microenvironment (TME) in PDAC. Therefore, to comprehend the players behind this immunosuppressive TME, we integrated 10 publicly available datasets into a robust PDAC scRNA atlas of 941.608 cells covering 201 patients (159 PDAC, 24 non-cancer, and 18 metastatic PDAC). The TME showed distinct composition for each tissue, predominating a fibro-inflammatory phenotype with infiltrating Tregs and dysfunctional CD8 T cells in primary PDACs. In contrast, adjacent normal tissue exhibited enrichment of tissue-resident memory CD8 T cells (p < 0.05). Metastatic PDAC was characterized by a reduction in stromal cells combined with myeloid and naive T cells infiltration. Comparison between treated and untreated PDAC revealed the enrichment of cancer-associated fibroblasts (CAFs) in treated patients. Besides, CD8 T cells from untreated patients exhibited higher levels of genes previously linked to immunotherapy response compared to treated patients. Non-negative matrix factorization (NMF) of the malignant cells unraveled 8 distinct metaprograms (MP). MP3 and MP7 overlapped with the well-known Classical subtype, whereas MP5 correlated with the Basal subtype. MP2 exhibited high expression of MHC-II genes and was enriched in metastatic lesions. Next, we investigated the tumor architecture using public spatial transcriptomic datasets. Spot-level deconvolution revealed 12 distinct cellular niches. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications.



 Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, Alessandro Bersch Osvaldt, Simone Marcia Dos Santos Machado, Eduardo Cremonese Filippi-Chiela, Cristina Bonorino. Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B126.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a30afbd64e95a911caaa3a76b2513c14671d0c3e" target='_blank'>
              Abstract B126: Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell omics technologies are transforming the landscape of biomarker discovery by offering unprecedented insights into the cellular and molecular complexity of health and disease. Unlike bulk profiling approaches that obscure subtle but critical signals through averaging, single-cell platforms capture distinct cell states, rare subpopulations, and transitional dynamics that are essential for precision diagnostics. This review presents strategies for translating single-cell data into clinically actionable biomarkers by leveraging advances in transcriptomic, proteomic, epigenomic, and spatial profiling. In addition, emerging perturbation-based approaches and lineage-informed datasets are enabling deeper mechanistic insights into disease processes. Artificial intelligence, particularly foundation models and stability-driven feature selection, now allows these complex datasets to be interpreted in ways that prioritize robustness and clinical relevance. By linking high-resolution cellular mapping to real-world diagnostic applications, single-cell omics is redefining how biomarkers are discovered, validated, and implemented in precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b789b8c44968a4c8991d63e2ac1c2253dc00a58" target='_blank'>
              From single-cell maps to diagnostics: enabling biomarker discovery in precision medicine
              </a>
            </td>
          <td>
            Jung Hyun Lee
          </td>
          <td>2025-08-29</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions that drive progression and pathobiology. The aim of this study was to understand how these spatial patterns and interactions evolve with treatment.


METHODS
To explore these relationships, we employed imaging mass cytometry to measure the expression of 34 protein markers, enabling the identification of GBM-specific cell types and their interactions at single-cell protein level in paired primary (pre-treatment) and recurrent (post-treatment) GBM samples from five patients.


RESULTS
We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes.


CONCLUSIONS
These findings provide insight into treatment-induced shifts in GBM's cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fcab07cdc7f1fdc819d108cb3fdc13fdcffb099" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment.
              </a>
            </td>
          <td>
            Shoaib Ajaib, Joshua Winter-Luke, Richard J Digby, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F Stead
          </td>
          <td>2025-09-01</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Late recurrence of breast cancer poses a considerable threat to patient survival, often attributed to breast cancer dormancy. Dormancy, characterized by cancer cells remaining quiescent for extended periods, is influenced by genetic factors and modifications that directly impact cellular phenotype. Alterations in gene expression dynamically shape cellular behavior, often mediated through receptor signaling pathways. Moreover, interactions within the tumor microenvironment play a pivotal role, fostering either cancer cell dormancy or promoting their escape from dormancy. This review endeavors to provide a comprehensive synthesis of recent advancements in understanding breast cancer dormancy across genetic, receptor molecular, and cellular dimensions. By elucidating the intricate mechanisms underlying dormancy, we aim to shed light on potential therapeutic strategies to prevent late recurrences. Furthermore, we anticipate future research directions that may uncover novel insights into this complex phenomenon, ultimately improving patient outcomes and refining clinical management strategies for breast cancer recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a6a18864245f8cad5e8cc957c1b2db50aef88c" target='_blank'>
              Unraveling the Mystery of Breast Cancer Dormancy: Insights into Genetic, Receptor, and Cellular Interactions Driving Late Recurrence
              </a>
            </td>
          <td>
            Haochen Ma, Bingqiang Zhang, Mengmeng Chen, Zhendi Song, Yi Zhao
          </td>
          <td>2025-09-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background This study aimed to characterize measurable morphological features such as size, shape, presence of protrusions, and micronuclei in circulating tumor cells (CTCs) in prostate cancer (PCa) patients and assess their clinical relevance. Methods Peripheral blood (PB) and tumor-draining vein blood (TDVB) samples were collected from 118 PCa patients at the Medical University of Gdańsk between 2018 and 2023. CTCs were phenotyped using epithelial (pan-keratins, K) and mesenchymal (vimentin, V) markers, with exclusion of non-tumor cells using CD45, CD31, αSMA, and CD29 markers. Imaging flow cytometry and QuPath software were used for the acquisition of CTC images, phenotypic and morphological analysis. Statistical analysis was performed with IBM SPSS Statistics. Representative images of CTCs were shared via CTC Atlas (www.CTCAtlas.org). Results A total of 1,437 CTCs in TDVB and 578 in PB were identified across 4 phenotypes: K + V−, K + V+, K − V+, and K − V−. CTCs were larger than leukocytes, with K + V− and K − V− cells being the largest, and K − V+ the smallest. Epithelial-to-mesenchymal transition (EMT) CTCs were significantly smaller in PB. Cytoplasmic protrusions were detected in a small subpopulation of CTCs. When detected in TDVB-derived CTCs, protrusions were significantly more frequent in epithelial CTCs compared to EMT-CTCs (P < .001). Micronuclei were observed in < 1% of CTCs in TDVB (limited to K + V− and K + V+), increasing to 3% in epithelial CTCs in PB. Small CTCs were associated with the shorter time to biochemical recurrence (P = .041, HR 2.198, CI95% 1.034-4.669). PB-derived CTCs were significantly smaller (P < .0001) and more irregular (P < .0001) in patients with newly diagnosed metastatic PCa (n = 13) than in those with localized disease (n = 556) (median diameter: 9.9 µm, range 7-23 µm vs 16.5 µm, range 7-62 µm; circularity: 0.67 vs 0.76). Smaller CTCs detected in TDVB and/or PB indicated patients’ shorter time-to-biochemical recurrence (P = .041, HR 2.198, CI95% 1.034-4.669). Conclusions CTCs display morphological heterogeneity across phenotypes. This is the first report to quantify EMT-CTC morphology and identify features such as protrusions and micronuclei. Smaller and morphologically irregular CTCs appear to correlate with PCa aggressiveness and poorer clinical outcomes, suggesting that even such simple CTC characteristics might identify CTCs associated with disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee17dbd2d4b8133dbdc0eb919926013f490dd3ab" target='_blank'>
              IUC24390-83 Morphology of circulating tumor cells: how it might relate to biology and clinics in prostate cancer
              </a>
            </td>
          <td>
            R. Wenta, J. Richert, A. Muchlińska, B. Pieczyńska-Uziębło, K. Miszewski, M. Matuszewski, A. Żaczek, N. Bednarz-Knoll
          </td>
          <td>2025-08-29</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/841868dae6381828134e074133437d8b76fe3212" target='_blank'>
              Bleb-based extravasation: conserved morphodynamics, divergent calcium control
              </a>
            </td>
          <td>
            Mizuki Morita, Manami Morimoto, Junichi Ikenouchi, Bertrand Pain, Yuji Atsuta, Yoshiki Hayashi, Takayuki Teramoto, Daisuke Saito
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 KRAS^G12D mutations represent a significant oncogenic driver, prevalent in highly aggressive malignancies such as pancreatic ductal adenocarcinoma (PDAC) and colorectal cancer (CRC). The recent development of direct KRAS^G12D inhibitors has ushered in a new era of targeted therapy, offering unprecedented opportunities to address these previously intractable cancers. This study aimed to explore and optimize therapeutic strategies by thoroughly evaluating the efficacy of MRTX1133, a novel selective non-covalent KRAS^G12D inhibitor, in conjunction with various immunotherapeutic modalities, particularly those involving natural killer (NK) cells.



 We utilized patient-derived xenograft and syngeneic murine cancer models to systematically characterize alterations of immune landscapes within tumor microenvironment (TME) upon KRAS^G12D inhibition. Comprehensive and in-depth assessment of immune cell function and single-cell level transcriptional activity was performed using flow cytometry and single-cell RNA sequencing analysis.



 Our findings reveal that MRTX1133 not only suppresses tumor progression but also profoundly enhances the activation of infiltrated natural killer (NK) cells within the tumor microenvironment. Functional depletion of NK cells partially, yet significantly, attenuated the anti-tumor activity of MRTX1133, underscoring the contribution of NK cells to the observed therapeutic benefit. Notably, KRAS^G12D inhibition led to a reduction in tumor-infiltrating Gr-1^+ myeloid-derived suppressor cells (MDSCs) and increased the proportion of effector NK cells. Furthermore, MRTX1133 facilitated greater intratumoral accumulation of adoptively transferred NK cells and counteracted tumor-induced systemic immunosuppression on NK cells. Combined treatment with MRTX1133 and NK cell immunotherapies produced synergistic anti-tumor effects and significantly extended survival in preclinical models. Mechanistically, ex vivo analyses demonstrated that MRTX1133 upregulates NK cell-activating ligands, including MICA/B and ULBPs, and suppresses the production of cytokines such as IL-6 and GM-CSF by cancer cells, thereby enhancing tumor cell susceptibility to NK cell-mediated cytotoxicity and reducing MDSC proliferation.



 Collectively, our results illuminate the interplay between KRAS^G12D inhibition and immuno-microenvironmental remodeling, highlighting rational combinations of targeted and NK cell-based immunotherapies as promising strategies to potentiate therapeutic responses in KRAS^G12D-driven cancers, potentially improving clinical outcomes for patients.



 Hongyuan Chen, Tuo Hu, Chao Xu; Liangjie Chi, Fangqin Xue, Chunbo He. KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aa888d3ef7c73bc5df1193380be3a3238a9fcd" target='_blank'>
              Abstract A016: KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity
              </a>
            </td>
          <td>
            Hongyuan Chen, Tuo Hu, Chao Xu, Liangjie Chi, Fangqin Xue, Chunbo He
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive and lethal brain tumor with very poor prognosis despite recent progress in multimodal treatments. Within glioma tissue, various niche cells such as macrophages and neutrophils form a unique glioma immune microenvironment (GIME) by interacting with heterogenous cancer cells, and this has been implicated in disease progression and responsiveness to immunomodulatory therapies. This study explores novel potential prognostic markers associated with the GIME using integrated bioinformatics analyses, including single-cell RNA-sequencing (scRNA-seq), and spatial transcriptome (ST) datasets of clinical GBM specimens. We first identified 42 genes as being associated with poor prognosis in GBM from 5 different cohorts, GBM vs. nontumor tissue, grade IV vs. grade II gliomas, isocitrate dehydrogenase (IDH)-wild-type vs. IDH-mutant variants, mesenchymal vs. proneural and classical subtypes, and hazard ratio for overall survival. Among these, 32 genes were positively correlated with ESTIMATEScore, infiltration of various immune cell types, expression of known immune-related genes, and representative immune-associated biological signals. On scRNA-seq analysis, 7 genes were relatively concentrated in tumor-associated macrophages rather than in malignant cells. ST analysis revealed that Collagen beta(1-O)galactosyltransferase 1 (COLGALT1), Integrin subunit beta 2 (ITGB2), and Myosin light chain 12A (MYL12A) were distributed in the interface between the tumor and the peritumoral area, overlapping with the expression of representative immune-related genes. These findings support the potential of COLGALT1, ITGB2 and MYL12A as biomarkers for predicting the prognosis and immune responses of GBM, which can help in the development of potential immunotherapeutic strategies for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76a83bcafe86bab51686dcebc0d98499aabea491" target='_blank'>
              Potential Prognostic Markers for Glioblastoma Associated with the Glioma Immune Microenvironment.
              </a>
            </td>
          <td>
            Kazuya Tokumura, E. Hinoi
          </td>
          <td>2025-09-01</td>
          <td>Biological & pharmaceutical bulletin</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="ABSTRACT Background Colorectal cancer (CRC) remains a major challenge in oncology, necessitating the identification of new therapeutic targets. This study aimed to explore the CRC microenvironment and uncover potential targets using single‐cell RNA sequencing (scRNA‐seq). Methods Single‐cell RNA sequencing data from GEO (GSE164522, GSE132465, and GSE144735) were integrated and stratified based on CRC clinical stages and tumor grades. Cell clustering analysis identified distinct cellular subsets, while survival and functional enrichment analyses evaluated the prognostic significance and biological activity of differentially expressed genes. Additionally, transcription factor activity and pseudotime trajectory analyses elucidated cellular dynamics during CRC progression. Cell–cell communication analysis identified key ligand‐receptor interactions involved in immune regulation. Finally, multiplex immunofluorescence was used to validate the expression of key markers and cells in CRC tissues from patients. Results In the low differentiated colorectal adenocarcinoma (LDCA) grade of CRC, the proportion of T cells significantly increased, accompanied by the enrichment of Treg cells and the upregulation of genes associated with immunosuppression, proliferation, and differentiation. Notably, TGFβ1+ Tregs significantly expanded in LDCA, accumulating in late tumor grades with enhanced immunosuppressive capacity and self‐proliferation. Pseudotime analysis confirmed their dominance in late CRC, reinforcing immunosuppression through signal amplification and sustained expansion. Concurrently, CD8+ T cells in the LDCA grade exhibited a progressive loss of cytotoxic function, transitioning into an exhausted state while concurrently activating apoptotic pathways, leading to a profound impairment of antitumor immunity. Cell–cell communication analysis further demonstrated that TGFβ1+ Treg exhibits the strongest interactions with CD8+ T cells, with KLRB1‐CLEC2D, LGALS9‐HAVCR2, and TNFSF10‐TNFRSF10B emerging as pivotal ligand‐receptor pairs displaying significantly enhanced signaling in the LDCA grade. Conclusion Collectively, TGFβ1+ Treg may mediate CD8+ T cell dysfunction through these ligand‐receptor interactions, accelerating T cell exhaustion and apoptosis, thereby fostering a profoundly immunosuppressive tumor microenvironment that ultimately drives CRC immune evasion and malignant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b126f27787b7f75c899e3f95b02d5186b9c098e5" target='_blank'>
              Integrated Single‐Cell RNA‐Seq Reveals Immunosuppressive Mechanisms of Treg Cell Differentiation and Tumor Microenvironment Interactions in Colorectal Cancer
              </a>
            </td>
          <td>
            Dongwei Li, Jingya Liu, Lixian Yu, Xina Luo, Xiaomei Wu, Ronghui Chen
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Breast cancer biology is influenced by both genetic ancestry and the environment with implications for disease outcomes. However, we lack the knowledge how non-cancerous cells within tumors are functionally altered by these factors at single-cell resolution. Here, we created a single-nucleus (sn) dataset from 33 African American, 25 Kenyan, and 24 European American women with breast cancer using combined RNA and ATAC sequencing. We successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue using an optimized combination of enzymatic digestion and automated tissue homogenization. SnMultiome sequencing of 82 tumors was performed using the 10x Genomics platform. Following filtering, normalization, peak calling, and integration, our dataset includes a total of 292,458 nuclei. Cancerous (163,419 nuclei) and non-cancerous (129,039 nuclei) cells were distinguished based on DNA copy number. Within the microenvironment, 11 major immune, epithelial, and stromal cell types were identified. CD45+, EPCAM- cells were further analytically extracted, clustered, and manually annotated based on known marker genes, detecting 26 distinct immune subpopulations across our samples. After adjustment for potential confounders (age, body mass index, and tumor molecular subtype) in all analyses, we found that women of African descent possessed distinct, activated mesenchymal and endothelial cell subpopulations within the tumor vasculature that promote inflammation, angiogenesis, and cancer cell invasion. Women of African descent exhibited a more immune-suppressive tumor microenvironment, with elevated expression of immune checkpoint markers (PD-1, TIGIT, CTLA-4, and LAG-3), functional impairment of dendritic cells, and enrichment of regulatory and tolerogenic lymphocytic subpopulations. Further, malignant cells from African descent patients expressed a discrete set of CAR-T targets with potential implications for precision immunotherapy. Lastly, we developed three gene signatures within the microenvironment that are predictive of survival with external validation in both tumor-adjacent normal and cancerous tissues from the TCGA. Together, we uncovered molecular breast tumor characteristics of clinical significance in women of African ancestry.



 Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H. Dorsey, Eytan Ruppin, Shahin Sayed, Francis Makokha, Gretchen L. Gierach, Stefan Ambs. Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b4c16fead2784b8ef235041a0b9c641f582e01" target='_blank'>
              Abstract C109: Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H Dorsey, E. Ruppin, Shahin Sayed, Francis W. Makokha, Gretchen L. Gierach, S. Ambs
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="


 Tumor clonal evolution represents a dynamic ecosystem underpinned by genetic alterations and Darwinian selection, posing major challenges due to intratumoral heterogeneity and therapy resistance. Although computational methods have advanced significantly, current tools often focus on single data modalities, leaving important gaps in modeling spatial and non-genetic evolution. This review systematically surveys and assesses algorithmic progress across diverse study designs to identify key limitations and future directions.



 We systematically evaluate over 20 computational tools across four study designs—cross-sectional, regional bulk, single-cell, and lineage tracing—and perform benchmarking of seven comparable tools. Multi-omics integration approaches are shown to improve phylogenetic inference, yet challenges remain in mutation ordering and polyclonal detection. A novel spatiotemporal framework is proposed to link phylogenetic branch lengths with spatial transcriptomic gradients. Future efforts should prioritize multi-modal data integration, scalable computational architectures, and clinically applicable models to translate evolutionary insights into precision oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d2b14817118e93c14f870881445bd16f81118f7" target='_blank'>
              Computational Strategies in Tumor Phylogenetics: Evaluating Multi-Modal Integration and Methodological Trade-Offs Across Study Designs
              </a>
            </td>
          <td>
            Chenghan Jiang, Zhe Wang, Ruoyu Wang, Shanshan Liang, Shuai Tao
          </td>
          <td>2025-10-01</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumour microenvironment (TME) is a complex and dynamic environment containing diverse cellular, stromal and soluble factors, that collectively influence cancer progression, immune evasion and therapeutic resistance. Among the immune components of the TME, macrophages and natural killer (NK) cells are key players, whose interactions, particularly their crosstalk, critically shape anti-tumour immunity. The macrophage–NK cell interplay can either promote or suppress immune responses depending on the context, representing both a challenge and a therapeutic opportunity. NK cells are key effectors capable of recognising and eliminating malignant cells without prior sensitisation, whereas macrophages exhibit remarkable plasticity, functioning as either promoters or suppressors of tumour immunity depending on their activation state. This review focuses on current strategies to harness macrophages in cancer therapy, including phenotype repolarisation, selective depletion, and disruption or enhancement of the macrophage-NK cell crosstalk to enhance NK cell-mediated tumour surveillance. Finally, we highlight emerging technologies, such as single-cell RNA sequencing, spatial transcriptomics, and proteomics, as powerful tools to elucidate the dynamic interplay between macrophages and NK cells and inform the next generation of immunotherapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9dc33b989f5afc184d0d48029ff121f57f3e749" target='_blank'>
              Enhancing natural killer cell anti-tumour activity through macrophage manipulation
              </a>
            </td>
          <td>
            Natasha Palmer, Salim I Khakoo, Tilman Sanchez-Elsner, Andres F. Vallejo
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5bbf3daa9e6f41e402a48c9bc2bd5b7f13849c" target='_blank'>
              Systematic elucidation and pharmacologic targeting on non-oncogene dependencies in imatinib-resistant gastrointestinal stromal tumor
              </a>
            </td>
          <td>
            Prabhjot S. Mundi, A. Grunn, A. Kojadinovic, Charles Karan, Ronald B. Realubit, Cristina I. Caescu, H. Hibshoosh, Mahalaxmi Aburi, , M.A. Ingham, D. Evans, S. Rothschild, Gary K. Schwartz, Andrea Califano
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="The awakening of dormant disseminated cancer cells appears to be responsible for the clinical relapses of patients whose primary tumors have been successfully cured months and even years earlier. In the present study, we demonstrate that dormant breast cancer cells lodged in the lungs reside in a highly mesenchymal, nonproliferative phenotypic state. The awakening of these cells is not triggered by a cancer cell-autonomous process. Instead, lung inflammation induced by the chemotherapeutic agent bleomycin effectively awakens dormant cancer cells, providing useful models for studying metastatic awakening. Mechanistically, the awakened cells shift from a highly mesenchymal to a quasi-mesenchymal phenotypic state in which they acquire tumorigenicity and proliferative ability. Once awakened, these cells can stably reside in this quasi-mesenchymal state and maintain their tumor-initiating ability, doing so without ongoing heterotypic signaling from the lung microenvironment. Epidermal growth factor receptor ligands released by the cells of the injured tissue microenvironment, including notably M2 type macrophages, promote dormant cancer cells to move toward this quasi-mesenchymal state, a transition that is critical for the awakening process. An understanding of the mechanisms of metastatic awakening may lead in the future to treatment strategies designed to prevent such awakening and resulting metastatic relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0978c66611eb319088a607dd5da99f325c7f34ff" target='_blank'>
              Inflammation awakens dormant cancer cells by modulating the epithelial-mesenchymal phenotypic state.
              </a>
            </td>
          <td>
            Jingwei Zhang, Jingwen Zhang, Longfei Han, Shiyi Wu, Jie Li, E. Eaton, Bingbing Yuan, Ferenc Reinhardt, Hao Li, Patrick C Strasser, Sunny Das, Joana Liu Donaher, Md Imtiaz Khalil, Haiping Jiang, Alexander Deuschel, Danni Lin, Carolin Sebastiany, Mariana Maranga, Salomé Shubitidze, Xiaofei Liu, Arthur W Lambert, Yun Zhang, Yana Liu, L. Sui, Sarah Elmiligy, Umberto Pozza, Rauf Günsay, Ranjan Mishra, José Velarde, Sonia Iyer, Whitney S. Henry, K. Weiskopf, Guihai Feng, Tobiloba E. Oni, R. Watnick, Xin Li, Robert A. Weinberg
          </td>
          <td>2025-09-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4146020be8bb663490d3534c28c485f753647c10" target='_blank'>
              Machine learning integration of bulk and single-cell RNA-seq data reveals glycolytic heterogeneity in colorectal cancer.
              </a>
            </td>
          <td>
            Yuanyuan Du, Zefeng Miao, Peng Li, Dan Feng, Mulin Liu, Aifang Ji, Shijun Li
          </td>
          <td>2025-08-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the most common and aggressive primary brain tumor in adults, marked by extensive inter- and intratumoral heterogeneity and pronounced therapy resistance. Microglia is a major contributor to this complexity and plays a pivotal role in shaping tumor behavior and modulating responses to therapy. Yet its precise phenotypic and functional states across distinct tumor regions and model systems remain poorly defined. In this study, we employ patient-derived organoids (PDOs) alongside matched primary tissue (PT) to characterize microglial diversity and investigate how microglia contributes to the cellular inter- and intratumoral heterogeneity of GBM.



 Fresh tumor tissue was obtained from 13 GBM patients and sampled from six spatially distinct regions per tumor. PDOs were generated from these regions and compared to matched PT to evaluate microglial presence, distribution, and morphology. To assess intertumoral heterogeneity, we compared microglial characteristics across patients, while intratumoral heterogeneity was examined by analyzing differences between regions within individual tumors. Subsequently, PDOs were treated with temozolomide (TMZ), and microglia was visualized and quantified using Iba1 immunofluorescence to evaluate treatment-induced morphological changes and shifts in microglial abundance.



 Microglia in PT exhibited a microglia morphology more similar to PDOs than to healthy brain tissue. Distinct microglial distribution patterns emerged across PDOs, with microglia typically arranged in one of four configurations: (1) perinuclear, (2) ring-like, (3) at the PDO margin, or (4) diffusely dispersed throughout the PDO. In 5/13 PDOs we found significantly higher microglial abundance compared to PT (PPDO 4 = 0.0013, PPDO 7 = 0.0424, PPDO 8 = 0.0153, PPDO 10 = 0.0162, PPDO 12 = 0.0083). Analysis across patients revealed significant intertumoral heterogeneity in microglial characteristics (p = 0.0405). In terms of intratumoral heterogeneity, regional differences in microglial abundance and morphology were observed within individual PDOs, with statistically significant variation only found in PDO 10 (p = 0.0211). Following TMZ treatment, 4/13 PDOs showed a significant recline in microglial abundance (PDO 2: p < 0.0001; PDO 4: p = 0.0041; PDO 7: p = 0.0015; PDO 10: p = 0.0297).



 Our findings reveal significant inter- and intratumoral microglial heterogeneity in PDOs. Distinct microglial distribution patterns and variable responses to TMZ highlight the complex and patient-specific nature of microglial involvement in the GBM microenvironment. These results underscore the relevance of PDOs as a model system for studying tumor-immune interactions and therapeutic responses in a personalized context.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a796a165fe58659df81ee201e3cb5e0d22f92cdf" target='_blank'>
              P02.05.B MAPPING MICROGLIAL DIVERSITY AND TUMOR HETEROGENEITY IN PRIMARY GLIOBLASTOMA AND PATIENT-DERIVED ORGANOIDS
              </a>
            </td>
          <td>
            J. Mazzoni, G. Haug, E. Roth, J. Hiebl, J. Messinger, K. Mehling, E. Meller, R. Nickl, C. Monoranu, A. Appelt-Menzel, C. Hagemann, F. Liesche-Starnecker, V. Nickl
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains one of the most difficult solid tumors to treat, owing in part to its remarkable cell state plasticity - a property that enables tumor cells to resist therapy, evade differentiation, and adapt to selective pressures. While targeted therapies have succeeded in subsets of patients, most tumors persist and eventually progress. To pursue a fundamentally new therapeutic strategy, we conducted an unbiased chemical screen using a custom-designed dual reporter system designed to broadcast cell state dynamics in real time by fluorescently tagging the endogenous SOX9 (aberrant stem-like) and KRT20 (differentiated) loci. This screen targeted epigenetic regulators capable of supporting CRC plasticity and preventing differentiation. We discovered that selective inhibition of histone deacetylases HDAC1/2 promotes differentiation and reduces tumor growth across human and mouse CRC models. This effect was not a consequence of cytotoxicity but rather epigenetic reprogramming. Through mass spectrometry-based histone modification profiling, we identified H3K27ac and H3K9ac as key epigenetic marks enriched at HDAC1/2-bound loci. These marks correlated with enhanced chromatin accessibility and the upregulation of differentiation-associated genes. Degradation of EP300, a major acetyltransferase, reversed the phenotype, confirming the functional relevance of histone acetylation in mediating this differentiation response. Furthermore, a focused CRISPR screen uncovered DAPK3 as a previously unrecognized downstream effector, providing a mechanistic foothold for future biomarker development. Our findings introduce a paradigm shift in CRC therapy: rather than targeting plastic tumor cells for elimination, we aim to force them into a differentiated, less adaptive state, making them more vulnerable to conventional and targeted treatments. This reprogramming effect is trackable via specific histone acetylation patterns, which we propose as candidate predictive biomarkers of response. Based on these insights, we hypothesize that HDAC1/2 inhibitors, when combined with KRAS or EZH2 inhibitors and/or chemotherapy, could yield synergistic therapeutic benefit. We are actively evaluating these combinations in preclinical models. In sum, this work positions HDAC1/2 inhibition as a precision epigenetic strategy to suppress CRC plasticity through enforced differentiation. By integrating multiomic profiling, functional genomics, and novel reporter technologies, we offer a blueprint for targeting the epigenetic underpinnings of therapy resistance in gastrointestinal malignancies, one that prioritizes reprogramming over destruction.



 Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, Sandor Spisak, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James Cleary, Jun Qi, Nilay Sethi. Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1992691ab8c82d8cb17a698db10f779b389e986a" target='_blank'>
              Abstract C031: Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Single-cell molecular profiling has emerged as a cutting-edge technology in biological research, playing a pivotal role in elucidating fundamental life processes and cellular heterogeneity. However, traditional methods generally provide end point measurements and require cell disruption, posing challenges for long-term monitoring of living cell events, e.g., cancer therapy. Tumor Treating Fields (TTFields), a noninvasive therapeutic modality, exert inhibitory effects on tumor growth and invasion by applying low-intensity intermediate-frequency alternating electric fields to tumor regions, whose molecular mechanism on cancer physiology remains poorly understood. In this study, we propose a multifunctional microchip capable of continuous in situ monitoring of molecular signatures in living single cells. The platform integrates spatial cell positioning, single-cellular multimolecule tracking with surface-enhanced Raman scattering (SERS), and TTFields treatment functionalities on a microchip. Using multiplexed analysis of liver cancer cells treated with TTFields and chemotherapy drugs, we implement spatial-temporal multiomics profiling for cancer therapy. Temporal tracking of critical molecular events associated with glycolysis, genetic stability, and membrane integrity reveals significant differences between the groups with and without TTFields treatment. Causal network analysis of molecular profiles uncovers the underlying mechanisms through which TTFields modulate oncogenic pathways. Our work demonstrates the effectiveness of the proposed microchip platform for real-time monitoring of living cell molecular dynamics, offering a powerful tool for precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889ada64ff2d00892ebd5cde7eb385bb8b31694b" target='_blank'>
              Spatial-Temporal Multiomics Profiling of Living Single Cells on Chip for Cancer Therapy.
              </a>
            </td>
          <td>
            Kunru Yu, Sheng-Zong Chen, Rong Zhu
          </td>
          <td>2025-10-12</td>
          <td>ACS sensors</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Analyses of patient-derived cell lines have greatly enhanced discovery of molecular biomarkers and therapeutic targets. However, characterization of cellular morphological properties is limited. We studied cell morphologies of human pancreatic adenocarcinoma (PDAC) cell lines and their associations with drug sensitivity, gene expression, and functional properties. By integrating live cell and spatial messenger RNA imaging, we identified KRAS inhibitor–induced morphological changes specific for drug-resistant cells that correlated with gene expression changes. We then categorized a large panel of patient-derived PDAC cell lines into morphological and organizational subtypes and found differences in gene expression, therapeutic targeting potential, and metastatic proclivity. Patterns of cancer cell organization in human PDAC tissues stratified distinct gene expression signatures with clinical significance. In summary, we highlight the potential of cell morphological information in rapid, cost-effective assays to aid precision oncology efforts leveraging patient-derived in vitro models and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb047a3ab41c4e28c6535b371221906a6c927ab" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Renal cancer, particularly clear cell renal cell carcinoma (ccRCC), is characterized by significant intratumoral heterogeneity, which poses challenges for diagnosis and treatment. Single-cell sequencing (SCS) provides unprecedented insights into the cellular landscape of renal cancer, allowing for detailed characterization of tumor heterogeneity at the single-cell level. This review highlights how SCS has been instrumental in elucidating the origins of different renal cancer subtypes, understanding mechanisms of tumor initiation and progression, and dissecting the complex tumor microenvironment (TME). It discusses the identification of novel biomarkers and therapeutic targets, as well as the potential of SCS to inform personalized treatment strategies. The review also explores the integration of SCS with spatial omics technologies, which enhances the understanding of cellular interactions within their spatial context. Moreover, it addresses the challenges and future directions in applying SCS to clinical practice, emphasizing its significance in advancing renal cancer biology and improving clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03a5f0a7b414a81333643cfcf0b78e2ab4708d98" target='_blank'>
              Single-cell sequencing technology in renal cancer: insights into tumor biology and clinical application
              </a>
            </td>
          <td>
            Hanzhong Zhang, Ying Liu, Wenqiang Liu, Anqi Lin, Yu Fang, Le Qu, Xu Zhang, Peng Luo, Linhui Wang, Aimin Jiang
          </td>
          <td>2025-10-01</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains largely refractory to therapy, due in part to the complex interplay between tumor cells and their microenvironment. Human antigen R (HuR/ELAVL1), a ubiquitously expressed RNA-binding protein, is emerging as an important regulator both of tumor-intrinsic and tumor-extrinsic pathways that govern PDAC progression. While the role of HuR in promoting cancer cell survival under stress is well established, recent studies reveal its broader role in shaping the tumor microenvironment (TME), including metabolic rewiring, stromal activation, angiogenesis, and immune modulation. In this review, we examine how tumor-intrinsic HuR drives epithelial-mesenchymal transition, stabilizes key transcripts involved in metabolic adaptation, and alters the tumor secretome to influence extracellular matrix deposition and fibroblast behavior. We further explore the role of HuR in regulating immune cell function and the immune landscape of PDAC. Notably, HuR-driven TME remodeling reinforces environmental stressors that further activate HuR, establishing a feed-forward loop that drives disease progression. These findings underscore HuR as a central regulator of the PDAC TME and therapeutic resistance, and thus, highlight its potential as a target in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be1b261ccac3070698258b8350c58e4ccb52e4c4" target='_blank'>
              The Role of Tumor-Intrinsic HuR in Modeling the Pancreatic Tumor Microenvironment: Molecular Mechanisms and Therapeutic Opportunities.
              </a>
            </td>
          <td>
            Katherine R. Pelz, Kyle P Gribbin, Tess Sevetson, Jonathan R. Brody
          </td>
          <td>2025-10-13</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract In its first decade, The Journal of Pathology: Clinical Research has become a leading source of translational studies advancing molecular diagnostics in cancer, particularly in urologic pathology. This commentary highlights recent contributions that collectively place precision oncology at the forefront of pathology research. One review examines cancer stem cells in renal cell carcinoma, emphasizing the complexity of cellular plasticity and the tumor microenvironment in driving resistance and recurrence. In prostate cancer, epithelial‐to‐mesenchymal transition (EMT) regulators, including Twist, Slug, and Snail, are identified as synergistic markers of poor prognosis, linked to hypoxia and invasiveness. Another review details the integration of homologous recombination repair gene testing into clinical workflows, supporting targeted treatment strategies with poly (ADP‐ribose) polymerase inhibitors. In pediatric oncology, TP53 alterations in Wilms tumor are shown to occur beyond anaplastic cases, expanding their prognostic significance. Advances in molecular subtyping are also demonstrated in bladder cancer, where transcriptomic profiling could enable tailored neoadjuvant therapy. In clear cell renal cell carcinoma, re‐evaluation of a prognostic model revealed that while necrosis or sarcomatoid differentiation correlated with poor outcomes, only DNA methylation markers improved prognostic accuracy, underscoring their utility for biopsy‐based risk stratification. Finally, digital spatial profiling of sarcomatoid urothelial carcinoma reveals an immunosuppressive microenvironment with CD163‐positive cells, implicating them in EMT and aggressive phenotype. Together, these studies highlight the transformative role of integrated molecular diagnostics in guiding individualized therapies and improving outcomes in urologic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9269338c6f4616fe82344276fa3a9645f1fc73c2" target='_blank'>
              Translational milestones in urologic pathology: integrating molecular diagnostics across cancer types
              </a>
            </td>
          <td>
            Andres Matoso, A. Acosta
          </td>
          <td>2025-09-01</td>
          <td>The Journal of Pathology: Clinical Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Intraductal papillary mucinous neoplasm (IPMN) is one of the two most common precursor lesions leading to the development of pancreatic ductal adenocarcinoma (PDA). IPMNs comprise a heterogeneous group of tumors with a wide range of grades and histotypes, and the emergence of single-cell RNA sequencing (RNA-seq) and multiplex digital spatial profiling have characterized unique cell populations, including dysplastic epithelial and immune cells, within the heterogeneous tumor microenvironment that carry signature gene expressions, which could be used as markers for disease progression. However, it is not clear on the heterogeneity of the epithelial and immune response of the evolution during progression of IPMN to PDA.



 Combining nuclear GSK-3β with oncogenic KRas mice (referred to as KNGC – KRas, nuclear GSK-3β, Cre) resulted in the development of IPMN. We have found that 4-week-old KNGC mice show progressive desmoplasia and ductal lesion development with increased M2-like macrophage infiltration. Interestingly, CX3CL1 (fractalkine), a chemokine involved in the recruitment of CX3CR1-expressing monocytes and M2-like macrophage conversion in various disease models is expressed by IPMN progenitor ductal cells. We aim to investigate expression and contribution of CX3CL1-CX3CL1 axis in the development and progression of IPMN.



 We utilized a newly generated CX3CL1-mCherry and well-established CX3CR1-GFP knockin/knockout mouse for crossbreeding with KNGC and KGC (KRas, GNas, Cre) for lineage tracing and function analysis by immunofluorescent staining, flow cytometry and CyTOF.



 Both KNGC- and KGC-3LmCherry/+ mice harbor a large population of CX3CL1-mCherry/EPCAM+ epithelial cells as compared to 3LmCherry/+ and wildtype (WT) control mice. In KNGC and KGC mice, we show that there was a unique population of macrophages (CD45+ CD11b+ Ly6G- F4/80+ Ly6c- ) with significantly higher expression of CX3CR1 (CX3CR1Hi) that was not seen in WT mice. Significantly, Knockout of CX3CR1 in KNGC-3RGFP/GFP animals had a paucity of M2-like macrophages, reduced T regulatory cells (Tregs), substantially reduced desmoplasia and impaired development of IPMN.



 The study provides important knowledge regarding the role of CX3CL1-CX3CR1 axis in recruitment and generation of M2-like macrophages, and their impact on the initiation of the immune/stroma microenvironment facilitating IPMN development. Further investigations using these models have the potential to identify new biomarkers for risk stratification and therapeutic intervention.



 Li Ding, Kaely Roeck, Easton Maeder, Cheng Zhang, Taylor Weiskittel, Esther Rodman, Mark Maynes, Bibo Zhu, Aaron Johnson, Jie Sun, Li Hu, Daniel Billadeau. Uncoupling CX3CL1-CX3CR1 Axis in IPMN Development Through Transgenic Mouse Models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A064.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94351cad4fa5650594e9ca02f146727e9c878075" target='_blank'>
              Abstract A064: Uncoupling CX3CL1-CX3CR1 Axis in IPMN Development Through Transgenic Mouse Models
              </a>
            </td>
          <td>
            Li Ding, Kaely Roeck, Easton Maeder, Cheng Zhang, Taylor Weiskittel, Esther Rodman, M. Maynes, Bibo Zhu, Aaron Johnson, Jie Sun, Li Hu, Daniel D. Billadeau
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Approximately 40% of patients with pancreatic ductal adenocarcinoma (PDAC) receive no benefit from surgical resection and systemic chemotherapy and experience metastatic disease recurrence within 12 months. The extreme heterogeneity of the cells that populate PDAC tumors has been a significant impediment to identifying the cells that likely seed early recurrence. Previous studies show that metastatic spread is driven by specific cells with hybrid epithelial-mesenchymal characteristics, but despite the identification of numerous molecular features associated with epithelial, hybrid, and mesenchymal PDAC cancer cells, it has not been possible to specifically identify the cells that seed metastatic recurrence in human tissue. We have previously shown that PDAC cell glycan signatures are altered as a direct result of the metabolic and transcription changes that occur as PDAC cancer cells undergo EMT. Now we integrated the glycan signatures with known protein markers and morphological analyses using our pipeline for multiplexed immunofluorescence analysis to identify distinct subpopulations of epithelial, hybrid, and mesenchymal PDAC cells. In cell-level analyses of immunofluorescence images of 24 PDAC cell lines, these subpopulations revealed epithelial to hybrid to mesenchymal conversions along 3 distinct trajectories, which were defined by transcription factors and proteins previously associated with epithelial or mesenchymal states—TP53, TP63, GATA6, and MYC—and further distinguished by specific glycan signatures and protein markers of metabolism and cellular differentiation. The integrated transcription factor, glycan, and protein signatures identified the previously described classical and squamous PDAC subpopulations and defined a new signature for a fully mesenchymal subpopulation. The signatures differentiated between PDAC subpopulations in primary human tumors and in metastatic liver lesions and showed a patient-matched correspondence between the primary tumors and metastatic lesions in specific hybrid PDAC cells within distinct trajectories, potentially revealing the PDAC cells seeding metastatic recurrence. These results suggest that the development of mesenchymal and metastatic PDAC cells occurs through distinct, stepwise trajectories with divergent morphological, metabolic, molecular, and glycan-based signatures. These findings will/have the potential to inform the development of treatments that specifically target the PDAC cells that seed early metastatic recurrence.



 ChongFeng Gao, Zachary Klamer, Ali Moursy, Galen Hostetter, Paul M. Grandgenett, Peter Allen, Brian Haab. Resolving PDAC cell subpopulations that seed metastatic recurrence [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B125.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246925305285b4b397bb54f4914db32c1466cbe8" target='_blank'>
              Abstract B125: Resolving PDAC cell subpopulations that seed metastatic recurrence
              </a>
            </td>
          <td>
            Chongfeng Gao, Zachary L. Klamer, Ali Moursy, G. Hostetter, P. Grandgenett, Peter Allen, Brian Haab
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c7c66e7f054de7552ccd4ec8d0186447c75c2a" target='_blank'>
              Single-cell Genomic Copy Number Evolution Reveals Frequent Loss of the Y chromosome in Esophageal Adenocarcinoma
              </a>
            </td>
          <td>
            Shiwei Yin, Na Zhong, Swati Agrawal, Bingru Feng, Sneha Pradhan, Nathan K. Jobalia, Maryam Vaziripour, Sampath K Poreddy, L. Kresty, Erin A Gillaspie, Nicholas E Dietz, W. Al-Refaie, K. Nandipati, Yusi Fu, Jun Xia
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background: Antibody-drug conjugates (ADCs) have ushered in a new era of precision oncology by combining the targeting specificity of monoclonal antibodies with the potent cytotoxicity of chemotherapeutic drugs. However, the cellular and molecular mechanisms underlying their dose-limiting ocular toxicity remain unclear. Elahere™, the first FDA-approved ADC targeting folate receptor α (FRα), demonstrates remarkable efficacy in platinum-resistant ovarian cancer but causes keratitis and other ocular toxicities in some patients. Notably, FRα is not expressed in the corneal epithelium—the primary site of damage—highlighting the urgent need to elucidate its underlying mechanisms. The aim of this study was to identify the cell-type-specific molecular mechanisms underlying Elahere-induced ocular toxicity. Methods: Sprague-Dawley rats were treated with intravenous Elahere (20 mg/kg) or vehicle weekly for five weeks. Ocular toxicity was determined by clinical examination and histopathology. Corneal single-cell suspensions were analyzed using the BD Rhapsody single-cell RNA sequencing (scRNA-seq) platform. Bioinformatic analyses to characterize changes in corneal cell populations, gene expression, and signaling pathways included cell clustering, differential gene expression, pseudotime trajectory inference, and cell-cell interaction modeling. Results: scRNA-seq profiling of 47,606 corneal cells revealed significant damage to the ocular surface and corneal epithelia in the Elahere group. Twenty distinct cell types were identified. Elahere depleted myeloid immune cells; in particular, homeostatic gene expression was suppressed in phagocytic macrophages. Progenitor populations (limbal stem cells and basal cells) accumulated (e.g., a ~2.6-fold expansion of limbal stem cells), while terminally differentiated cells decreased in corneal epithelium, indicating differentiation blockade. Endothelial cells exhibited signs of injury and inflammation, including reduced angiogenic subtypes and heightened stress responses. Folate receptor alpha, the target of Elahere, was expressed in endothelial and stromal cells, potentially driving stromal cells toward a pro-fibrotic phenotype. Fc receptor genes were predominantly expressed in myeloid cells, suggesting a potential mechanism underlying their depletion. Conclusions: Elahere induces complex, multi-compartmental ocular toxicity characterized by initial perturbations in vascular endothelial and immune cell populations followed by the arrest of epithelial differentiation and stromal remodeling. These findings reveal a cascade of cellular disruptions and provide mechanistic insights into mitigating Elahere-associated ocular side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f4b07f86256bbe595f586072659c1ac39cff834" target='_blank'>
              Single-Cell Transcriptomics Reveals a Multi-Compartmental Cellular Cascade Underlying Elahere-Induced Ocular Toxicity in Rats
              </a>
            </td>
          <td>
            Jialing Zhang, Meng Li, Yuxuan Yang, Peng Guo, Weiyu Li, Hongxin An, Yongfei Cui, Luyun Guo, Maoqin Duan, Ye Lu, Chuanfei Yu, Lan Wang
          </td>
          <td>2025-10-04</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The thymidine analog EdU (5-ethynyl-2-deoxyuridine) is incorporated into DNA during replication and has traditionally been used as a marker of S-phase cells. In this study, we discovered that EdU fluorescence images display substantial cell-to-cell variability, which could be classified into multiple clusters by unsupervised machine learning. This suggests that seemingly random EdU patterns contain reproducible, computationally recognizable features. Building on our observation that distinct patterns emerged in response to radiation stress, we investigated whether radioresistant cancer cells exhibit specific EdU signatures. Analysis of PLK1-overexpressing cells, which acquire radioresistance through altered DNA replication, revealed radiation-induced EdU patterns distinct from control cells. Prompted by the observation that these cells displayed markedly enlarged and intensified γ-H2AX foci, a marker of DNA damage, we employed a supervised machine learning model based on γ-H2AX patterns to isolate the radioresistant cell subpopulation. We then extracted the EdU signals from these isolated cells and, through further unsupervised machine learning, successfully identified a characteristic pattern specific to radioresistance. This establishes a machine learning framework capable of extracting universal rules from the dynamic networks that vary among individual cells, which provides a novel platform for a screening system to identify molecules involved in radioresistance, focusing on cancer heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0447884e0a398eddbbba5abf08e4dfe806602b07" target='_blank'>
              Machine Learning Extracts Radiation Resistant-Specific EdU Fluorescence Pattern in Cancer Cells.
              </a>
            </td>
          <td>
            M. Ikura, Tsuyoshi Ikura, Kanji Furuya
          </td>
          <td>2025-09-01</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6971b52e8812bcb489dfce1326931d0b1e99d8ab" target='_blank'>
              Cell villages and Dirichlet modeling map human cell fitness genetics
              </a>
            </td>
          <td>
            Chloe Hanson, Tim Derebenskiy, Ana Rodriguez Vega, Yashika S. Kamte, Rachel G. Fox, Hannah Lambing, Patrick Allard, Harold Pimentel, Michael F. Wells
          </td>
          <td>2025-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="OBJECTIVE
Large-scale CRISPR screens have identified essential genes across cancer cell lines, but links between tumor functional properties and specific dependencies require investigation to reveal the mechanisms underlying dependencies and broaden understanding of targeted therapy.


METHODS
We selected 47 breast cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) with multi-omics data including gene dependency; somatic mutations; copy number alterations; and transcriptomic, proteomic, metabolomic, and methylation data. We established a dependency marker association (DMA) analytic pipeline by using linear regression modeling to assess associations between 3,874 representative gene dependencies and multi-omics markers. Additionally, we conducted non-negative matrix factorization clustering, to stratify breast cancer cell lines according to gene dependency features, and investigated cluster-specific DMAs.


RESULTS
We interpreted valuable DMAs according to two primary aspects. First, dependencies associated with gain-of-function alterations revealed addiction to lactate transporter SLC16A3, thus suggesting a promising therapeutic target. Second, dependencies associated with loss-of-function alterations included synthetic lethality (SL), collateral SL, and prioritized metabolic SL, encompassing paralog SL (e.g., IMPDH1 and IMPDH2), single pathway SL (e.g., GFPT1 and UAP1), and alternative pathway SL (e.g., GPI and PGD). DMA analysis of the two clusters with divergent dependency signatures demonstrated that cluster1 cell lines exhibited extensive metabolism with mitochondrial protein dependencies, whereas cluster2 displays enhanced cell signaling, and reliance on DNA replication and membrane organelle regulators.


CONCLUSIONS
We established a DMA analysis pipeline linking the gene dependencies of breast cancer cell lines to multi-omics characteristics, thus elucidating the underpinnings of tumor dependencies and offering a valuable resource for developing novel precision treatment strategies incorporating relevant markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/753908ccf816b6d23f787d4f37906f4b14d28c45" target='_blank'>
              Comprehensive investigation of the molecular basis of cancer dependencies suggests therapeutic options for breast cancer.
              </a>
            </td>
          <td>
            Rui Ding, Zhimin Shao, Tianjian Yu
          </td>
          <td>2025-10-23</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a crucial role in cancer progression, metastasis, immune evasion, and treatment resistance. However, the current literature often studies its components separately. This review offers an integrated view of the dynamic interactions among fibroblasts, immune and vascular cells, the extracellular matrix, cytokines, exosomes, and microbiota within the TME. It discusses classical mechanisms such as epithelial-mesenchymal transition, stromal remodeling, and metabolic rewiring alongside emerging paradigms like microbiome-driven immunomodulation and exosome-mediated therapy resistance. Spatial heterogeneity and the temporal evolution of the tumor niche are examined using recent advances in single-cell and spatial transcriptomics, 3D bioprinting, and patient-derived organoid models. Key findings emphasize the microbiome's influence on immune responses and the role of exosomes in transferring resistance traits and regulating intercellular signaling. By connecting molecular insights with clinical perspectives, the review explores translational strategies targeting the TME, including checkpoint inhibitors, stromal modulators, anti-angiogenic agents, and engineered CAR-T therapies. This comprehensive view highlights the importance of considering cancer as a complex, evolving ecosystem rather than just a cell-autonomous disease and provides a foundational framework for precision oncology approaches aimed at disrupting harmful TME interactions to improve therapeutic effectiveness and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0229215ccf5f4775676d3f8ba3362cab276b8a" target='_blank'>
              Tumor Microenvironment and Its Role in Cancer Progression: An Integrative Review
              </a>
            </td>
          <td>
            Khalaf Mohamed Almazrouei, Vartika Mishra, Hetal Pandya, Kumar Sambhav, Sainath Narayan Bhavsar
          </td>
          <td>2025-09-01</td>
          <td>Cureus</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) are one of the predominant cell components within the colorectal cancer (CRC) microenvironment, which display prominent and multifaceted roles in modulating tumor biology. The systematic subtypes and function analysis of TAMs remain elusive in CRC. Methods We performed an integrated single-cell atlas of three American studies (GSE178341, GSE200997, and GSE231559) to establish the TAM subtypes in the CRC microenvironment. We adopted unsupervised clustering and manual cell annotation to categorize macrophage subtypes accurately, performed enrichment analysis and developmental trajectory analysis for three TAM subsets, and explored the cell-to-cell crosstalk of different TAM subsets with other cell types. Finally, we further performed an integrated single-cell atlas of five Asian studies (GSE132257, GSE132465, GSE144735, GSE221575, and GSE245552) to validate these characteristics of different TAM subtypes. Results We identified three TAM subtypes, including CCL20+ TAMs with proinflammatory and anti-tumor properties, APOE+ TAMs with lipid-metabolism and pro-tumor properties, and SLC40A1+ TAMs with immunosuppressive and pro-tumor properties. APOE+ TAMs might be intermediate state during macrophage polarization and foster a desmoplastic niche by the fibroblast-derived collagen pathways. SLC40A1+ TAMs might play an immunosuppressive role by the fibroblast-derived MIF pathways and the high expression of LGALS9. CRC with enriched CCL20+ and APOE+ TAMs were characterized by high prevalence of deficient-mismatch repair (27.9%) and might achieve more benefit from immunotherapy. Conclusion This single-cell study established an accurate classification system of CRC TAMs, unveiling their diverse roles in modulating tumor biology and assisting in treatment options of CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee82a442edb4540e343e66f1260d69c23d5c6af1" target='_blank'>
              Integrated Single-Cell Analysis Reveals the Heterogeneity of Tumor-Associated Macrophages and Their Implications for Immunotherapy in Colorectal Cancer
              </a>
            </td>
          <td>
            Guozeng Xu, Binglan Fang, Xiaobi Tang, Qingqing Wei, Jing Li
          </td>
          <td>2025-09-01</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0643ed748b04336ec828931abc9d5fbd2911609" target='_blank'>
              [Progress of scRNA-seq technology in nasopharyngeal carcinoma research].
              </a>
            </td>
          <td>
            Bin Zheng, Guanqiao Jin
          </td>
          <td>2025-09-01</td>
          <td>Lin chuang er bi yan hou tou jing wai ke za zhi = Journal of clinical otorhinolaryngology head and neck surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy driven by oncogenic KRAS and inflammation-induced cellular heterogeneity, yet the mechanisms underlying tumor-initiating cells (TICs) emergence and maintenance remain unclear. Our study establishes LY6D as a marker of persistent TICs that orchestrate pancreatic cancer progression across all disease stages. Single-cell RNA sequencing of inflammation-driven PDAC models reveals that LY6D+ TICs specifically originate from KRAS-mutant acinar-to-ductal metaplasia (ADM) lesions under inflammatory conditions, maintaining conserved stemness properties and exhibiting a distinctive oxidative phosphorylation (OXPHOS) dependency throughout tumor evolution. Moreover, genetic ablation of Ly6d in Kras G12D pancreata delays tumorigenesis, while forced Ly6d expression enhances tumorigenic potential and metastatic capability. Mechanistically, LY6D-despite lacking intracellular domains-scaffolds lipid raft-associated kinase networks and FOSL1-dependent epigenetic reprogramming to establish a ​stable pro-tumorigenic state. Clinically, LY6D+ cells are enriched in human PDAC and exhibit conserved stemness and epithelial-mesenchymal transition (EMT) properties. Strikingly, LY6D expression levels demonstrate PDAC-restricted prognostic power. Our work defines LY6D as pan-stage TICs marker linking cellular plasticity to PDAC initiation and progression, offering new avenues for early detection and interception of this lethal malignancy.



 Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue. LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B119.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f74adf609c9f9d814e1f1f768c348943a8c10b" target='_blank'>
              Abstract B119: LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis
              </a>
            </td>
          <td>
            Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor matrix stiffness plays a critical role in cancer progression, metastasis, and therapy resistance. Although traditional biophysical methods have shed light on the impact of matrix stiffness on tumor behavior, these techniques are confined to measuring the physical properties of the tumors. In this study, we leveraged RNA-seq data to predict tumor matrix stiffness, aiming to reveal mechanical properties by molecular signatures across various cancer types. To this end, we systematically analyzed RNA-seq data from tumors of varying stiffness levels to identify stiffness-associated gene signatures. With these molecular signatures, we developed a computational model for predicting tumor matrix stiffness and further applied it to The Cancer Genome Atlas (TCGA) dataset. Our analysis revealed significant differences in the tumor microenvironment as well as immune response between soft and stiff tumor samples, suggesting that tumor rigidity impacts not only cellular behavior but also characteristics of the tumor microenvironment. These findings underscore the potential of RNA-based stiffness models to enhance our comprehension of tumor mechanics and cancer biology, thereby facilitating the development of innovative targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd0a56130a8bfa1d6781b487c86488f3edefb603" target='_blank'>
              Pan-cancer analysis reveals molecular signatures for predicting matrix stiffness in solid tumors.
              </a>
            </td>
          <td>
            Gongyu Tang, Xinyi Liu, Yuanxiang Li, Yunfei Ta, Minsu Cho, Hua Li, Xiaowei Wang
          </td>
          <td>2025-09-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/906f1e267f0452865e325381ce2fb991c017ccf2" target='_blank'>
              Are Different Populations Fairly Represented in Single-Cell Omic Atlases?
              </a>
            </td>
          <td>
            Catrina Yang, Kavitharini Saravanan, Aryan Saharan, Kuan-lin Huang
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic cancer (PC) is a highly aggressive malignancy with a dismal 5-year survival rate of only 12%. Currently, no effective therapeutic strategies can improve the prognosis of pancreatic cancer. The activation of quiescent pancreatic stellate cells (PSCs) and their crosstalk with pancreatic cancer cells (PCCs) lead to the formation of a fibrotic physical barrier and an immunosuppressive tumor microenvironment (TME), which severely impede drug delivery and penetration. To improve PC treatment, a dual-targeting strategy capable of simultaneously acting on PCCs and PSCs is urgently needed to revert activated PSCs (aPSCs) to their quiescent state and suppress the proliferation of PCCs. KPT-6566, a small-molecule inhibitor, shifts PSCs from an activated to a quiescent state, reprograms the tumor immune microenvironment (TIME) to enhance anti-PD-L1 immunotherapy efficacy, suppresses PCCs proliferation, and induces PD-L1 expression on PCC surfaces, thereby increasing sensitivity to PD-L1-targeted therapies. This study designs two amphiphilic peptides, C16-LLGG-COO-DPPA-1(DPPA-1@PA) and DSPE-PEG-NHS-CGKRK(CGKRK@PA), which can co-assemble to encapsulate KPT-6566 and finally form a peptide-based nanoparticle termed DP-KPT-CG@PA. This nanoparticle precisely targets PCCs and PSCs to co-deliver KPT-6566, while synergizing with a PD-L1-blocking peptide to initiate immune checkpoint blockade therapy, thereby enhancing response rates to pancreatic cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a26fa041300ac88b5d5967aad970db3f1c2709b5" target='_blank'>
              Targeting PIN1 in Pancreatic Tumor Microenvironment with Peptide Amphiphiles Unleashes Immune Checkpoint Therapy.
              </a>
            </td>
          <td>
            Yuanyuan Wu, Jianan Guo, Yuwei Zhang, Ruizhi Zhao, Wenhao Luo, Chihua Fang
          </td>
          <td>2025-09-24</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma (PDA) has poor response to chemotherapy with a strong likelihood of cancer recurrence. While previous studies have examined patient samples collected before and after chemotherapy, few have analyzed matched samples from the same patients that allow us to deconvolute interpatient heterogeneity, a large limitation to PDA studies. Here, we study the effects of chemotherapy on tumor cells and their surrounding microenvironment using scRNA sequencing of matched pre- and post- treatment tissue biopsies from 5 patients. We utilized the Seurat v4 pipeline to observe transcriptomic differences and corresponding putative cellular interactions in both treatment states. Unsurprisingly, we found insufficient tumor epithelial cells post-treatment due to the intended cytotoxic effects of chemotherapy, leading us to focus on the tumor microenvironment where we had sufficient cells to analyze. We identified that immune cells in the TME are heterogeneously affected by chemotherapy treatment with patient specific differences, but we did identify a unanimous increase in CXCR4 in CD4 and CD8 positive T cells after treatment. Analysis of CAFs revealed that a subset of fibroblasts lose myofibroblastic features and increasingly acquire 'neural'-like features after treatment. This suggests that chemotherapy-altered fibroblasts may contribute to a pro-tumor microenvironment that facilitates recurrence through neuron–CAF–tumor interactions. Through inferred ligand-receptor pairs identified with the CellChat v2 pipeline, we observed that increased CXCL12 in these neural CAFs interacts with the increased CXCR4 on T-cells. We validated the expression of these genes in patients by staining human patient PDAC tissues (both treated and treatment naïve tissues) for neural genes that were highly enriched post-treatment in our single cell sequencing dataset. We have also begun inhibiting these genes of interest in CAFs and are exploring the effects of these ‘neural’-CAFs on patient-derived tumor organoids in culture.



 Aylin Z. Henstridge, Padma Kadiyala, Ahmed Elhoissiny, Jianhua Liu, Georgina Branch, Nandini Arya, Vaibhav Sahai, Nicole Peterson, Jorge Machicado, Richard Kwon, Allison Schulman, Erik Wamsteker, George Phillips, Martin Fernandez-Zapico, Mark Truty, Timothy Frankel, Fillip Bednar, Marina Pasca di Magliano, Costas Lyssiotis, Eileen S. Carpenter. An increase in neural cancer associated fibroblasts following exposure to chemotherapy in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B079.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13e666941a86feb629e0cf1155d415645c6c810d" target='_blank'>
              Abstract B079: An increase in neural cancer associated fibroblasts following exposure to chemotherapy in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Aylin Z. Henstridge, Padma Kadiyala, Ahmed Elhoissiny, Jianhua Liu, Georgina Branch, Nandini Arya, Vaibhav Sahai, Nicole Peterson, Jorge D. Machicado, Richard S Kwon, Allison Schulman, Erik Wamsteker, George Phillips, Martín E Fernandez-Zapico, M. Truty, Timothy L. Frankel, Fillip Bednar, Marina Pasca di Magliano, C. Lyssiotis, Eileen S. Carpenter
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most frequent brain tumors in children and comprise a heterogeneous group of tumors with different locations, histologic subtypes, ages at presentation, and clinical behavior. Tumors frequently respond to treatment with chemotherapy or radiation therapy, but they can regrow after a period of quiescence, requiring further therapy. Thus, a deeper understanding of the molecular processes involved in these tumors is required to develop therapeutic strategies that are effective against their disease mechanisms. To better understand the cellular behaviors of this heterogenous group of tumors, we have employed single-cell and single-nuclei RNA sequencing technologies to analyze a large-scale dataset (>250,000 cells) of pLGGs. Analysis of this data identified a heterogenous population of cell types and cell states, detecting mature and progenitor-like astrocytes and oligodendrocytes, as well as cells exhibiting senescence or cycling programs. Moreover, we identify a significant immune infiltrate, comprised primarily of microglia. In addition to heterogeneity within pLGG tumors, heterogeneity between LGG subtypes represents another layer that stratifies pLGG biology. We performed a compositional analysis of the cell types present in these tumors and compared transcription signatures and gene expression programs across shared cellular populations of histologically and genetically distinct pLGGs. Finally, we optimized our integration and batch correction analyses by using external 293T cells as spike in controls during our single-cell and single-nuclei data generation steps to determine the most suitable method for batch-effect removal. Our analysis of human pLGGs at the single-cell and single-nuclei resolution provides critical insight into the heterogenous biological activities that constitute these tumors.



 Michelle Boisvert, Ashwyn Perera, Alexandra L. Condurat, John Jeang, Jessica W. Tsai, Dana Novikov, Kevin Zhou, Madison Chacon, Jeromy DiGiacomo, Rushil Kumbhani, Dayle Wang, Michael D. Taylor, Jordan R. Hansford, Louise Ludlow, Nada Jabado, Keith L. Ligon, Rameen Beroukhim, Pratiti Bandopadhayay, David T.W. Jones. Understanding the transcriptional heterogeneity of pediatric low-grade gliomas and its implication for tumor pathophysiology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87fff9552d8445c379c8cd3282dd22472d8f4d25" target='_blank'>
              Abstract A014: Understanding the transcriptional heterogeneity of pediatric low-grade gliomas and its implication for tumor pathophysiology
              </a>
            </td>
          <td>
            Michelle Boisvert, Ashwyn A. Perera, Alexandra L. Condurat, John Jeang, Jessica W Tsai, Dana Novikov, Kevin N Zhou, Madison S. Chacon, Jeromy J. Digiacomo, Rushil Kumbhani, Dayle K. Wang, Michael D. Taylor, J. Hansford, Louise E. Ludlow, Nada Jabado, Keith L. Ligon, R. Beroukhim, P. Bandopadhayay, David Jones
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>151</td>
        </tr>

        <tr id="PURPOSE
Medulloblastoma (MB) is a common pediatric brain tumor with distinct molecular subgroups, of which, Group 3 MB is associated with increased recurrence, metastatic potential and poor patient outcomes. Small molecule inhibitors targeting BMI1 have been shown to be efficacious against several types of malignant tumors, including pediatric MB. While our previously published in vivo study provides a promising proof-of-concept for the therapeutic targeting of BMI1 in Group 3 MB with small molecule inhibitor, it is not sufficient to eradicate the tumour.


EXPERIMENTAL DESIGN
In this study, following preclinical validation of BMI1 inhibitor PTC-596, DNA barcoding technology was leveraged to profile in vivo clonal dynamics of Group 3 MB in response to the established chemoradiotherapy regimen alone and in combination with PTC-596. Following demonstration of a small number of treatment-refractory clones we sought to identify potential druggable molecular vulnerabilities by utilizing phosphoproteomic profiling and genome-wide CRISPR screening.


RESULTS
By comparing the changes in phosphorylation pattern of key signaling kinases post PTC-596 treatment with the list of sensitizer genes from in vitro genome-wide CRISPR/Cas9 screen and to the essential genes in human neural stem cells (hNSCs), we identified several context-specific regulators of mTOR, AKT and PLK1 pathways. Subsequently, targeting the PI3K pathway with Enzastaurin was shown to be most meanable to synergistic targeting alongside BMI1 inhibition.


CONCLUSION
This work provides the foundation for clinical validation of small-molecule inhibitors synergistic with PTC-596 to improve the durability of remissions and extend survival of patients with treatment-refractory Group 3 MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254ad36a007c276e881f281d38c624b50e4a39ec" target='_blank'>
              Leveraging Medulloblastoma Clonal Dynamics to Overcome Treatment Resistance.
              </a>
            </td>
          <td>
            D. Bakhshinyan, Stefan Custers, Laura Escudero, Yujin Suk, Kevin R. Brown, Hardikkumar Patel, A. Adile, C. Chokshi, M. Shaikh, D. McKenna, M. Qazi, Kui Zhai, D. Tieu, Katherine Chan, Marla Weetal, C. Venugopal, Jason Moffat, Sheila K Singh
          </td>
          <td>2025-10-15</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Tumor cell metabolome intricately shapes its identity and therapeutic vulnerabilities. Cancer cells dynamically adapt to the tumor microenvironment (TME) – ‘Warburg effect’, a classic example of metabolic plasticity wherein cancer cells preferentially convert glucose to lactate despite availability of oxygen to thrive challenging environments. While metabolic profiling has advanced tumor biology, spatially resolved rewiring of metabolome remains poorly defined. Neuroblastoma (NB), a pediatric cancer with high propensity for hematogenous metastasis exhibits diverse underscored metabolic phenotypes. Our recent studies uniquely identified the predictive, prognostic relevance and function of Retinal Degeneration protein 3 (RD3) in NB pathogenesis including its role in shaping the tumor immune environment. Herein, we investigated whether and how RD3 affects metabolic plasticity in metastatic NB, offering new therapeutic avenues for high-risk (HR) NB. For this, global untargeted metabolomics was performed in bone marrow (BM) derived clones (CHLA-90, CHLA-140) from HR-NB patients and compared with their RD3 stably reinstated counterparts. Data preprocessing, mass detection, EIC building, feature resolving and grouping, ion identity networking and spectral library search was performed using Mzmine4.7 followed by compound identification using Compound discoverer. RD3-driven shift in metabolites were identified using univariate analysis with stringent criteria (log2fold change) coupled with false discovery rate (FDR, log10PadJ). Pathway enrichment analysis was acquired using MetaboAnalyst6.0. Functional validation of RD3-dependent metabolic shift was performed by assessing mitostress (Seahorse Xfe Analyzer) in RD3 expressing clone (CHLA-15) derived from primary (retroperitoneal) site, RD3-null CHLA-90 (BM) and compared with their RD3 silenced and expressed clones respectively. Compared to RD3-/- clones, we observed definitive conserved (cell-line independent) rearrangement of 289 metabolites (288, downregulated; 1, upregulated) in RD3+/+ clones. Predominantly, the metabolites were highly associated in amino acid metabolism (p= 6.61*10-3; FDR, 2.1*10-1) followed by fatty acid and steroid metabolism (p= 3.7*10-2; FDR, 3.02*10-1). Notably, metabolites involved in aerobic glycolysis (Warburg effect, Enrichment score, 1.953) were among the least active in RD3+/+ reinstated clones indicating a significant shift in cell energetics. Consistently, mitostress test further revealed that RD3-loss in HR-NB favors a glycolytic phenotype whereas RD3 expression promoted a more quiescent phase. Our findings explicitly demonstrate that RD3 expression in metastatic NB suppresses glycolysis and reprograms cellular metabolism. Furthermore, the conserved shifts in amino acid and fatty acid metabolism highlight RD3 as a potential metabolic gatekeeper. These RD3-driven alterations could unveil novel therapeutic vulnerabilities in HR-NB, offering new avenues for targeted intervention.



 This work was funded by the DoD CA-210339; OCAST-HR19-045; and the NIH P20GM103639



 Poorvi Subramanian, Natarajan Aravindan, Sivaroopan Aravindan. Breaking the Warburg code: RD3 redefines energetics in metastatic Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdda285b7679b38ef5248455c327701b43a9d61b" target='_blank'>
              Abstract B045: Breaking the Warburg code: RD3 redefines energetics in metastatic Neuroblastoma
              </a>
            </td>
          <td>
            Poorvi Subramanian, N. Aravindan, Sivaroopan Aravindan
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Diverse fusions of EML4 and ALK are oncogenic drivers in lung adenocarcinomas. EML4-ALK variants have distinct breakpoints within EML4, but their functional differences remain poorly understood. Here, we use somatic genome editing to generate autochthonous mouse models of EML4-ALK-driven lung tumors and show that V3 is more oncogenic than V1. By employing multiplexed genome editing and quantifying the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth, we show that many tumor suppressor genes have variant-specific effects on tumorigenesis. Pharmacogenomic analyses further suggest that tumor genotype can influence therapeutic responses. Analysis of human EML4-ALK-positive lung cancers also identified variant-specific differences in their genomic landscapes. These findings suggest that EML4-ALK variants behave more like distinct oncogenes rather than a uniform entity and highlight the dramatic impact of oncogenic fusion partner proteins and coincident tumor suppressor gene alterations on the biology of oncogenic fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c8a78e3c9e4fd68a0441eaf2f2edc5f59a5afae" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis.
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Karen Y. Shih, Óscar González-Velasco, Mulham Najajreh, Stewart Kim, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, Ilse Hofmann, S. Sivakumar, S. Sisoudiya, Ethan S Sokol, Hongchen Cai, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2025-09-23</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4ae2d5f1f6f184ad5a2b71dcc7e35b4feeb422f" target='_blank'>
              Functional biomaterials and machine learning approaches for phenotyping heterogeneous tumor cells and extracellular vesicles.
              </a>
            </td>
          <td>
            Rutwik Joshi, R. Ahmad, Karl Gardner, Hesaneh Ahmadi, Chau-Chyun Chen, Shannon Stott, Wei Li
          </td>
          <td>2025-09-18</td>
          <td>Biomaterials science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) is the most common form of kidney cancer, leading to 179 000 cancer-related deaths annually. ccRCC initiating cells (CICs) are thought to drive tumor initiation, growth, therapy resistance, and metastasis, yet their molecular characteristics remain poorly defined. This study aims to identify putative CICs and their essential genes using a cell marker-agnostic strategy. Methods We performed a comprehensive analysis of ccRCC transcriptomes at single-cell resolution, developed patient-derived xenograft (PDX) and 3D patient-derived organoids (PDO) models of ccRCC, and conducted functional examinations in these models to investigate our findings. Results Computational modeling of tumor formation using single-cell RNA velocity analysis of five primary and metastatic ccRCC-PDXs revealed a minor cell population as the origin of other tumor cells, representing putative CICs. Pathway and network analyses suggested that a core network of proteins, conventionally known to regulate mitosis, are highly active in CICs and may be essential for their function. These proteins were expressed in PDOs and PDX-derived spheroids established following a CIC enrichment protocol. Spheroid cells exhibited higher tumorigenicity and colony formation ability than parental tumor cells, as confirmed by in-vivo injection in nude mice and in-vitro colony formation assays. Successive in vivo passaging confirmed the self-renewal capacity of spheroid-derived tumors. Pharmacological blockade of candidate proteins elicited dose-dependent inhibitory effect on spheroid and colony formation, with in-vivo validation showing that blocking these proteins significantly delayed tumor growth and more efficiently prevented tumor formation in mice. These results highlight the importance of these proteins in the cancer-initiating abilities of malignant cells. Interestingly, our results suggest that the identified target proteins may elicit their CIC-related function independently from their mitosis regulatory roles. Conclusions We identified and validated essential proteins in RCC-CICs, supported by single-cell transcriptome data and RCC spheroids and PDX models. Targeting the vulnerabilities of RCC-CICs, given their role in tumor initiation, progression, and therapy resistance, offers significant potential for developing new anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edac148fb846da9cb774550aaf710d22fbf68ef2" target='_blank'>
              64defining cancer initiating cells and their vulnerabilities in renal cell carcinoma
              </a>
            </td>
          <td>
            Zohreh Mehrjoo, H. Kuasne, Ariel Madrigal Aguirre, Ali Shahini, M. Annis, Anne-Marie Fortier, Tianyuan Lu, Larisa Morales Soto, Hong Zhao, D. Zuo, V. Pilon, M. Dankner, Tamiko Nishimura, Kevin Petrecca, Jonathan Spicer, Peter Siegel, Simon Tanguay, H. Najafabadi, Morag Park, Yasser Riazalhosseini
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) exhibits profound treatment resistance attributed to complex tumor-stroma interactions within the desmoplastic microenvironment. Cancer-associated fibroblasts (CAFs) play critical roles in chemoresistance, yet current drug screening models predominantly utilize single organoid systems that fail to recapitulate authentic tumor-stroma crosstalk. We established patient-matched organoid-CAF co-culture systems from endoscopic ultrasound-guided fine needle biopsy (EUS-FNB) specimens. Primary organoids and CAFs were isolated from identical patient samples, with drug sensitivity assays comparing monoculture versus co-culture responses. CAF characterization employed flow cytometry analysis of alpha-smooth muscle actin, fibroblast activation protein, and vimentin, while multiplex cytokine profiling characterized secretory networks. Single-cell RNA sequencing and expanded functional validation studies are ongoing. Co-culture dramatically altered chemosensitivity profiles compared to organoid monoculture, revealing tumor-stroma interactions driving therapeutic resistance. Importantly, we discovered distinct patient-specific cytokine resistance profiles. Some patients exhibited tumor necrosis factor-alpha-centered resistance mechanisms, while others displayed interleukin-8 or interleukin-6-mediated inflammatory response patterns. These patient-specific cytokine networks correlated with unique resistance patterns, suggesting individualized secretory circuits mediating treatment failure. Three-way comparative analysis demonstrated emergent interactive responses exceeding additive effects, while flow cytometry revealed remarkable patient-specific heterogeneity in CAF activation marker expression. This study establishes a clinically-relevant platform that recapitulates patient-specific tumor-stroma interaction patterns in pancreatic cancer. Identification of patient-specific cytokine resistance signatures provides therapeutic targets for individualized combination therapy strategies. Ongoing single-cell transcriptomics and expanded molecular profiling will further define stromal subtypes and resistance signatures, advancing precision medicine-based therapeutic development that considers patient-specific resistance mechanism heterogeneity in pancreatic cancer.



 Kyung Min Lee, Yooyeon Kim, Eunhye Hwang, Jin Ho Choi, Woo Hyun Paik, , Sang Hyub Lee. Patient-matched EUS-FNB-derived organoid-CAF co-culture platform reveals novel resistance mechanisms in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ae9c6ba7486165ec670214f902de50a83b424de" target='_blank'>
              Abstract B050: Patient-matched EUS-FNB-derived organoid-CAF co-culture platform reveals novel resistance mechanisms in pancreatic cancer
              </a>
            </td>
          <td>
            Kyung-Min Lee, Yooyeon Kim, Eunhye Hwang, J. Choi, W. Paik, Sang Hyub Lee
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="This editorial discusses Alpsoy et al’s significant study of prognosis of pancreatic ductal adenocarcinoma (PDAC), which lacks histopathological markers. This study evaluated the synergistic prognolymphocytes. Peritumoral budding is significantly correlated with tumor volume, while intratumoral budding is closely related to lymph node metastasis. Peritumoral budding and intratumoral budding are confirmed as independent adverse prognostic factors, and their high levels of expression are associated with immature stromal phenotypes, suggesting the key role of epithelial-mesenchymal transition. These breakthrough findings provide a new multidimensional biomarker system for the prognostic assessment of PDAC, and promote the clinical transformation process of incorporating tumor budding indicators into the pathological reporting process. However, the complexity and spatiotemporal heterogeneity of the tumor microenvironment require us to go beyond traditional morphological analysis and move towards multiomics integration and dynamic monitoring. Through standardized pathological assessment, innovative treatment strategies and interdisciplinary collaboration, it is expected to transform tumor microenvironment-related markers into clinically applicable indicators, ultimately improving the treatment predicament of PDAC. This editorial intended to summarize relevant studies and share some of our views, in order to offer perspectives for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25a304affe959bbde58f99c98545a1534cc99601" target='_blank'>
              Microenvironmental code of pancreatic ductal adenocarcinoma: The prognostic symphony of budding, matrix and lymphocytes
              </a>
            </td>
          <td>
            Qing-Qing Zhou, Yi-Meng Ren, Su-Ming Shi, Teng-Fei Liu
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Melanoma shows heterogeneity across body sites like skin, acral skin, and the uvea, driven by molecular characteristics and genetic variations. However, comparative studies exploring the heterogeneity of melanoma across different anatomical sites remain limited, hindering a comprehensive understanding of its underlying biology. We proposed a research framework through bioinformatics to analyze the tumor ecosystems of cutaneous, acral, and uveal melanoma, from molecular characteristics to genetic variations at single-cell resolution. We found that oxidative phosphorylation (OXPHOS) is a critical driver of tumor cell evolution, with abnormal ribosomal gene and tumor suppressor expression observed in uveal melanoma (UM). Additionally, we screened for potential drug targets and drugs against tumor cells. In the immune microenvironment, acral melanoma (AM) and UM exhibit stronger immunosuppressive characteristics compared to cutaneous melanoma (CM). OXPHOS contributes to T cell cytotoxicity dysregulation in CM and AM, while interferon-γ is crucial in UM. Tumor cells may also induce T cell dysfunction through biological signals such as MIF-CD74 and HLA-E-NKG2A. This study offers valuable insights into melanoma heterogeneity, providing a comprehensive research framework for understanding the distinct molecular and immune characteristics of CM, AM, and UM, and potentially guiding the development of therapeutic strategies tailored to each melanoma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da658ce35ad37afa8afa0fc4adb1c52869ddb7f4" target='_blank'>
              Dissecting Melanoma Ecosystem Heterogeneity from Molecular Characteristics to Genetic Variation at Single-Cell Resolution
              </a>
            </td>
          <td>
            Congxue Hu, Liyuan Li, Tengyue Li, Baobin Qi, Wanqi Mi, He Yu, Kaiyue Yang, Qi Ou, Xia Li, Yunpeng Zhang
          </td>
          <td>2025-10-13</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a235a990b95cad3b6488f79eeb25d061d834e198" target='_blank'>
              Triangulating cross-carcinoma GWAS delineates shared programs and disease-specific drivers
              </a>
            </td>
          <td>
            Y.-F. Huang, K.-L. Huang
          </td>
          <td>2025-09-22</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="For over sixty years, blood researchers have been counting clones with every tool at their disposal. Inspired by phage and fly geneticists, Till and McCulloch irradiated mice to induce chromosomal aberrations. Using this labeling strategy, they demonstrated that different types of blood cells shared the same mutation in every spleen colony, thereby proving the existence of hematopoietic stem cells. Since their breakthrough, technological advances have enabled researchers to quantify hematopoiesis at single-cell resolution in increasingly complex samples across both mice and humans. With these modern sophisticated lineage tracing methods, our foundational understanding of the blood system is being reshaped. For instance, we now interpret hematopoietic architecture as arising from stem and progenitor cells of diverse developmental origins, each with distinct fate biases encoded by unique regulatory states. Interacting with this regulatory layer, genetic mutations and epimutations arise, expanding clonally and becoming pervasive with age. Together, clonal heterogeneity and age-driven clonal selection may underlie the perplexing diversity of therapy responses in cancer and beyond. As these paradigm-shifting insights gain traction, clonal tracing is being adopted across dozens of biological and clinical studies. Here, we review the modern toolbox of clonal tracking technologies, with a focus on next-generation sequencing-based approaches, and provide a practical guide for matching specific research questions with optimal experimental strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca9cbce80016ba1b72cbad0fc7cc9413794f150" target='_blank'>
              Clonal tracing of blood stem cells across mouse and human lifespans.
              </a>
            </td>
          <td>
            Alejo E. Rodriguez-Fraticelli
          </td>
          <td>2025-09-22</td>
          <td>Blood</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Notch is a contact-dependent signaling pathway that plays critical roles in spatiotemporal patterning during embryogenesis. Beyond its role in development, Notch signaling regulates tissue patterning in adult organs in response to tissue injury or cancer. In the context of cancer, the role of Notch signaling has been extensively characterized in cancer cells, T cells, and endothelial cells. Our previous work has implicated Notch signaling in the myeloid and fibroblast compartments of the pancreatic tumor microenvironment (TME) in the tumor immune response, an area that has been previously understudied. In our current work we use genetically engineered mouse models to inhibit canonical Notch activation in fibroblasts (Pdgfra CreERT2/+, LSL-DNMAML) and myeloid cells (LysM Cre/+, LSL-DNMAML) separately. In both an orthotopic transplant and genetic model (Ptf1a FlpO/+;FSF-Kras G12D/+) of pancreatic cancer progression, inhibition of fibroblast-specific Notch activation had limited effects on disease progression and the transcriptional profile of the fibroblasts themselves. This is consistent with previous evidence of more limited Notch activation in pancreatic cancer-associated fibroblasts and suggests a primary role for fibroblasts in providing Notch ligand (JAG1) to neighboring cells. In contrast, inhibition of myeloid-specific Notch activation resulted in smaller orthotopic tumors and slowed progression in a genetic model of pancreatic neoplasia. These samples had reduced overall macrophage infiltration, and a secondary alteration of the fibroblast phenotypes was also seen. Collectively, these results suggest that Notch signaling in the myeloid compartment supports tumor progression and regulates multiple components of the pancreatic TME. Future studies will further interrogate how myeloid-specific Notch activation promotes disease progression and affects Notch signaling activation in adjacent compartments.



 Allison Bischoff, Wei Yan, Carlos Espinoza, Emily Lasse-Opsahl, Nur Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar. Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A080.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df0eac24a706ec3b2888b93172ea892748b9176" target='_blank'>
              Abstract A080: Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment
              </a>
            </td>
          <td>
            Allison C Bischoff, Wei Yan, Carlos E Espinoza, Emily L. Lasse-Opsahl, Nur M Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb94fc6f75bdfeda4079ffba83972fab9d967bdb" target='_blank'>
              Radiotherapy drives lung metastasis in PDAC via a RhoGTPase signaling shift towards MRCK-dependency
              </a>
            </td>
          <td>
            KM McLay, M. Tesson, L. Dutton, Y. Sun, JD Misquitta, K. Stevenson, R. Corbyn, S. Lilla, L. McGarry, R. Shaw, CJ Miller, Leo M. Carlin, MF Olson, JP Morton, JL Birch
          </td>
          <td>2025-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is characterized by stromal remodelling andstiffening, a process involving extracellular matrix (ECM) depositionand cross-linking, which forms a dense physical barrier that limits the penetration of chemotherapeutic agents and restricts immune cell-mediated cytotoxicity. In addition, the tumor stroma modulates nutrients availability and fosters cancer cell invasion and metastasis. Therefore, therapeutic strategies aimed at modulating the tumor stroma hold promise for overcoming some of the major challenges in pancreatic cancer treatment. However, most clinical trials of stromal-targeted therapies have not demonstrated meaningful improvements in overall survival, underscoring an urgent need to identify novel stromal targets and approaches that modulate the tumor microenviroment in ways that enhance therapeutic response and improve patient outcomes. Methods Transcriptomic datasets of PDAC sampleswere obtained from The Cancer Genome Atlas (TCGA). By integrating immune infiltration algorithms with weighted gene co-expression network analysis (WGCNA), this study aims to identify hub genes associated with cancer-associated fibroblasts (CAFs) infiltration in PDAC, explore the prognostic relevance of CAF infiltration, and construct a gene‑expression-based prognostic mode. Furthermore, we will employ a multi-modal approach incorporating magnetic resonance elastography, atomic force microscopy, single-cell RNA sequencing (scRNA-seq), histological assays, cell-based functional experiments, and animal models to identify candidate core genes involved in CAF infiltration and stromal stiffening in pancreatic cancer, and to assess the potential therapeutic relevance of targeting these genes. Results In PDAC, increased stromal stiffness and poor prognosis are both strongly linked to activation of CAFs. WGCNA identified six hub genes—ANTXR2, ANXA1, BCAT1, PLAU, TGFβ1, and ITGB5—which were used to construct a predictive model for estimating CAF abundance and forecasting PDAC prognosis. Single-cell RNA sequencing analysis showed that ITGB5 is highly expressed in CAFs. Both in vitro and in vivo experiments demonstrated that targeting ITGB5 in CAFs reduces tumor stiffness, thereby inhibiting the malignant progression of PDAC. These findings suggest a potential therapeutic benefit, although impact on clinical prognosis remains to be established. Conclusion The integration of magnetic resonance elastography, atomic force microscopy, and single-cell sequencing allows detailed research characterization of the biomechanical properties of PDAC. ITGB5 has been identified as an important regulator of the biomechanical features of PDAC. Targeting ITGB5 in CAFs may reduce the excessive stiffness of PDAC tissue and mitigate gemcitabine-associated stromal fibrosis, suggesting a potential strategy to improve treatment response. The effect on patient prognosis, however, requires confirmation in clinical studies. Trial registration This study is a prospective single-center clinical trial, which has been registered in the U.S. Clinical Trials Database (https://clinicaltrials.gov/study/NCT06526442) with the trial registration number NCT06526442. The registration date was October 9th, 2020. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07119-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303c7c686718b4d13d9fc453421d878d4dcfcaa1" target='_blank'>
              ITGB5-mediated biomechanical regulation in pancreatic ductal adenocarcinoma stroma impacts tumor progression and prognosis
              </a>
            </td>
          <td>
            Aoran Yang, Chengli Gu, Yining Liu, Baihe Luo, Yu Zeng, Minghui Zhou, Chunli Li, Chen Pan, Linxuan Wu, Zhiwen Hu, Xin Zhang, Feng Gao, Yu Shi
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma is one of the deadliest solid tumors, with a two-year survival rate below 5%. Progress in developing effective treatments has been limited, largely due to the absence of physiologically relevant disease models that accurately recapitulate the tumor microenvironment. Such models are critical for understanding disease mechanisms and evaluating therapeutic strategies. Organoids have recently emerged as promising platforms for modeling various diseases, including pancreatic cancer, as they retain the structural and functional features of their tissue of origin. Moreover, CRISPR-Cas9 gene editing allows precise introduction of cancer-associated mutations, providing a powerful approach for generating genetically defined models.



 We established normal pancreatic organoids from human pancreatic tissue samples and validated their identity through qPCR and histological analyses, confirming their resemblance to native pancreatic epithelium. These organoids demonstrated stable growth over several passages and retained viability after cryopreservation and thawing. To model pancreatic cancer, we used CRISPR-Cas9 technology to introduce TP53 mutations, a key tumor suppressor gene altered in over 70% of pancreatic ductal adenocarcinoma (PDAC) cases. Following Nutlin-3a selection, successful TP53 knockout was confirmed by Sanger sequencing. The engineered TP53-knockout (KO) organoids were expanded and characterized to assess cancer-like phenotypes.



 Human-derived pancreatic organoids retained their structural and functional identity across passages and after thawing. Following TP53 gene editing, the TP53 KO organoids exhibited increased proliferation and morphological changes consistent with early tumorigenesis. Importantly, qPCR analysis revealed significant upregulation of cancer-associated markers, including MUC5AC, KRT19, and CEACAM5, in TP53-mutated organoids compared to controls, further supporting their malignant transformation. These findings represent a critical step toward building a genetically defined pancreatic cancer model that closely reflects human disease.



 We successfully generated a pancreatic cancer organoid model by editing TP53 in normal human pancreatic organoids. The TP53 KO organoids showed enhanced growth, morphological transformation, and elevated expression of key cancer markers, recapitulating early features of pancreatic cancer. This model mimics critical aspects of pancreatic cancer biology and provides a robust platform for studying tumor initiation, progression, and drug response. Unlike conventional animal models or stem-cell-derived systems, our approach offers a human-based, physiologically relevant, and reproducible system for pancreatic cancer research and therapeutic screening.



 Eden Demere Amare, Soraya SALAS-SILVA, Ji Hyun Shin, Dongho Choi. CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59cdfa7b3d8511b93672f4beff88185a825b61c7" target='_blank'>
              Abstract A013: CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation
              </a>
            </td>
          <td>
            Eden Demere Amare, Soraya SALAS-SILVA, Jihyun Shin, Dongho Choi
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 The advent of RAS inhibitors in the clinic offers the possibility of revolutionized care for patients with pancreatic ductal adenocarcinoma (PDAC). While RAS(ON) multi-selective inhibitors like the investigational agent daraxonrasib (RMC-6236) and the related preclinical tool compound RMC-7977 are designed to target the KRAS-mutant malignant epithelium, their effects on the surrounding tumor stroma remain underexplored. Prior studies using genetically engineered mouse models (GEMMs) have shown that aberrant oncogenic KRAS signaling is critical for the establishment and maintenance of the tumor microenvironment. In recent years, multiple CAF subtypes have been identified based on gene expression profiles, spatial distribution, and cellular origin. Myofibroblastic CAFs (myCAFs), which are located near the malignant epithelium, are marked by active Sonic Hedgehog (SHH) signaling, while inflammatory CAFs (iCAFs) and antigen-presenting CAFs (apCAFs) reside more peripherally. Here, we investigate how pan-RAS-GTP inhibition by RMC-7977 reshapes the CAF landscape in PDAC and its impact on the extracellular matrix (ECM). We profiled CAF subtype prevalence and transcriptional programs using single-cell RNA sequencing (scRNA-seq) of GEMM-derived tumors treated in vivo with RMC-7977 across multiple timepoints. These findings were corroborated by immunostaining of murine and patient-derived tumor explants treated ex vivo for 2 to 4 days. Tumor stiffness following RMC-7977 treatment was assessed using nanoindentation, and mechanistic studies were performed in vitro using reporter fibroblasts. Preliminary data reveal that short-term RMC-7977 treatment reduces myCAF abundance and proliferation, while iCAF and apCAF populations expand. Notably, supplementation with exogenous SHH restores the original myCAF-rich phenotype even in the presence of RMC-7977. Pan-RAS-GTP inhibition also remodels the ECM, resulting in decreased tumor stiffness. These preclinical findings suggest that systemic pan-RAS-GTP inhibition alters not only the malignant epithelium but also the broader PDAC architecture through effects on stromal components. Understanding these stromal adaptations will enable improved assessment of tumor behavior and the development of more effective combination therapies.



 Allison C. Hess, Lorenzo Tomassoni, Kevin Muñoz Forti, Hun Jin Jeong, Deanna J. Besart, Tanner C. Dalton, Carmine F. Palermo, Stephen A. Sastra, Chang H. Lee, Andrea Califano, Simon Schwörer, Kenneth P. Olive, Marie C. Hasselluhn. RAS(ON) multi-selective inhibition remodels cancer-associated fibroblast subtypes and extracellular matrix in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B098.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80ccb92666321bba79e7b5a3b723ec7cf1794a5" target='_blank'>
              Abstract B098: RAS(ON) multi-selective inhibition remodels cancer-associated fibroblast subtypes and extracellular matrix in pancreatic cancer
              </a>
            </td>
          <td>
            Allison C. Hess, Lorenzo Tomassoni, Kevin Muñoz Forti, Hun Jin Jeong, Deanna J. Besart, Tanner C. Dalton, Carmine F. Palermo, Stephen A. Sastra, Chang H. Lee, Andrea Califano, Simon Schwörer, K. Olive, Marie C. Hasselluhn
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178ca173745db4823628df69a4e845e3404906a8" target='_blank'>
              Epigenetic modulation of CTCs as a novel therapeutic target in oncology.
              </a>
            </td>
          <td>
            Husni Farah, M. Abosaoda, Hayjaa Mohaisen Mousa, S. Jyothi, Priya Priyadarshini Nayak, J. Janney, Gurjant Singh, Ashish Singh Chauhan
          </td>
          <td>2025-10-11</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary People with systemic sclerosis (a chronic disease that causes skin and organ scarring) have a higher risk of developing certain cancers soon after diagnosis, but the reasons remain unclear. To study systemic sclerosis in the laboratory, researchers are using tight skin (TSK) mice, which develop excess scar tissue and immune changes similar to systemic sclerosis. We combined the TSK mouse model with mouse models of skin, breast, and ovarian cancers to test whether the unique environment in TSK mice promotes cancer progression. In all cancer types, TSK mice developed more aggressive tumors. Detailed single-cell analyses showed that TSK tumors had more immune imbalance and more scar-promoting cell types. These findings suggest that the TSK mouse is a powerful tool for uncovering how systemic sclerosis-related tissue changes and immune shifts may fuel cancer growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fece107070df629a4a56795a06c184ea38a1b613" target='_blank'>
              Cancer Growth and Invasion Are Increased in the Tight Skin (TSK) Mouse
              </a>
            </td>
          <td>
            M. Recouvreux, Barbie Taylor-Harding, Amy C. Rowat, Beth Y Karlan, Sandra Orsulic
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Polycystic kidney disease (PKD) arises from mutations in cilia-associated genes, such as PKD1 and PKD2, expressed in renal epithelial cells, leading to progressive kidney dysfunction and end- stage kidney disease (ESKD). PKD patients exhibit significant heterogeneity in disease progression, largely due to genetic and environmental modifiers. Like patients, mouse models of PKD also exhibit significant heterogeneity with regards to the gene mutated, age of disease onset, and rate of disease progression. To elucidate the cellular and molecular consequences of these variables, we constructed an integrated single-cell and spatial transcriptomics atlas across mouse models of PKD, mapping changes in cell type composition, gene expression, and intercellular signaling networks across the whole atlas and within individual models. Consistently across models, single cell RNA sequencing (scRNAseq) data revealed increased Spp1 (osteopontin) expression and signaling from PKD-enriched clusters to Ly6clo monocytes. Global deletion of Spp1 in Pkd1RC/RC mice resulted in reduced cyst severity, improved kidney function, and reduced Ly6clo monocyte numbers, suggesting that SPP1 signaling to Ly6clo monocytes promotes PKD progression. We also created a freely available, searchable website (https://bmblx.bmi.osumc.edu/scPKD/) that can be used to identify cross- and intra-model specific changes in gene expression, guiding researchers to new therapeutic targets for treating PKD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c0ef30fff916db6f0ffbd15d9f601a5f6423bd0" target='_blank'>
              A cross model spatial and single-cell atlas reveals the conserved involvement of osteopontin in polycystic kidney disease
              </a>
            </td>
          <td>
            S. Miller, Hua Zhong, Weidong Wu, Audrey Cordova, Morgan E. Smith, A. Yashchenko, Zhang Li, D. Jafree, Chelsea N. Zimmerman, Christa I. DeVette, Vicki Do, Maya Hignite, Isabella G. Darby, Khalecha Bintha Ahmed, Yohan Park, Fariha Nusrat, Bibi Maryam, Sizhao Lu, Xiaoyan Li, Jenny R. Gipson, Xiaogang Li, David A. Long, M. Weiser-Evans, Bradley K. Yoder, Benjamin D. Cowley, Katharina Hopp, Jason R Stubbs, Qin Ma, A. Ma, Kurt A Zimmerman
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal malignancy characterized by a profoundly immunosuppressive tumor microenvironment. Within PDAC, tumor-associated macrophages (TAMs) are the most abundant immune cell population. Here, TAMs orchestrate the characteristic immunosuppression of the pancreatic tumor microenvironment, drive disease progression, and mediate therapeutic resistance in PDAC. Accordingly, successful development of approaches that reprogram or eliminate these pro-tumor immune populations have potential to improve patient outcomes. One of the most challenging aspects of developing approaches to modulate TAMs programming is the ability to perform mechanistic experiments in a system the recapitulates the complex pancreatic tumor microenvironment. We and others have leveraged primary bone marrow-derived macrophages (BMDMs) polarized by PDAC conditioned media as a discovery platform that has yielded translational findings. However, this system fails to account for the dynamic and reciprocal interactions between myeloid and cancer cells that occur within tumors. Accordingly, we sought to address if this model could be built into progressively more complex systems to assess if these better recapitulate TAMs using a systems biology approach. Using transcriptomic profiling, we compared different in vitro strategies, from direct co-cultures to conditioned media polarization, against TAMs harvested from orthotopic PDAC tumors. Our preliminary data show that allowing PDAC cells to engage in direct cellular contact and sustained paracrine signaling most closely resemble the transcriptomic profile of bona fide TAMs. Our further efforts to use these PDAC-macrophage culture models will be key to identifying relevant signaling pathways that can be mechanistically examined prior to more cumbersome in vivo validation. Collectively, our data can be used to facilitate more accurate and insightful investigations into PDAC immunology and the development of therapeutic targeting strategies.



 Cavina K. Lee, Cecily Anaraki, Austin D. Silva, Tiffany Lai, Hillary Tran, Arjun Sapru. Dissecting myeloid - cancer communication networks in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B104.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad71226813c7f5d3d378a55b84523f40aab08ce1" target='_blank'>
              Abstract B104: Dissecting myeloid - cancer communication networks in pancreatic cancer
              </a>
            </td>
          <td>
            Cavina K. Lee, Cecily Anaraki, Austin D. Silva, Tiffany Lai, Hillary Tran, Arjun Sapru
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="As a common malignant tumor, the heterogeneity of colorectal cancer plays an important role in tumor progression and treatment response. In recent years, the rapid development of single-cell transcriptomics and spatial transcriptomics technologies has provided new perspectives for resolving the heterogeneity of colorectal cancer. These techniques can reveal the complexity of cellular composition and their interactions in the tumor microenvironment, and thus facilitate a deeper understanding of tumor biology. However, in practical applications, researchers still face technical challenges such as data processing and result interpretation. The aim of this paper is to explore how to use artificial intelligence (AI) technology to enhance the research efficiency of single-cell and spatial transcriptomics, analyze the current research progress and its limitations, and explore how combining AI approaches can provide new ideas for decoding the heterogeneity of colorectal cancer, and ultimately provide theoretical basis and practical guidance for the clinical precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48a5aab5c189afed70eaf2f7feb72d37acadd46e" target='_blank'>
              Multidimensional decoding of colorectal cancer heterogeneity: Artificial intelligence-enabled precision exploration of single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Wen-Yu Luan, Qi Zhao, Zheng Zhang, Zhen-Xi Xu, Si-Xiang Lin, 
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Immunotherapies have shown significant clinical benefit in cancers with high unmet medical need. As next-generation bispecific antibodies are entering the clinic, early identification of patients likely to respond to immunotherapy remains both critical and challenging. We have previously demonstrated that our ex vivo 3D micro-tumor platform, which preserves the native tumor microenvironment (TME), is predictive for platinum-based chemotherapy and correlates with clinical immune biomarkers. Here, we show that the ex vivo platform also enables patient-specific profiling of drug sensitivity and immune activity for bispecific antibody ubamatamab.



 This study analyzed 14 tissue samples obtained from 12 ovarian cancer patients with primary, recurrent or progressive disease. Micro-tumors including the native TME were isolated from fresh ascites (n=12) or pleural fluid (n=2), seeded, and treated with pembrolizumab (single dose) and dose ranges of chemotherapeutics and ubamatamab (CD3xMUC16 bispecific T cell engager in Phase 1/2 clinical trials). Drug response was assessed through morphological features extracted from high-throughput 3D imaging. Sensitivity was defined by statistically significant immune cell expansion and micro-tumor killing. Supernatants collected after treatment were analyzed for secreted cytokines, chemokines and cytotoxic mediators using proximity extension assay (Olink) immune biomarker profiling for 5 patients. Quantification of immune cell populations was performed via immunofluorescence staining.



 Treatment with ubamatamab resulted in a significant expansion of immune cells in 43% (6/14) of ex vivo patient samples. Of these, 33% (2/6) also showed significant tumor killing, demonstrating complete functional responses. Responses to ubamatamab were reproducible in successive samples from the same patients, showing assay consistency. Baseline cytokine profiling confirmed substantial inter-patient heterogeneity, and patient samples with low baseline levels of CD8+ T cell cytokines showed ex vivo resistance to ubamatamab. Notably, responders with effective tumor killing exhibited robust and simultaneous induction of effector T cell cytokines and cytotoxic mediators (IFN-γ, IL-2, granzyme B, perforin) as well as IFN-γ-induced chemokines CXCL9 and CXCL11 upon ubamatamab treatment. Importantly, ex vivo sensitivity to ubamatamab did not align with responses to pembrolizumab or chemotherapy, indicating that this therapy may be effective in a distinct subset of patients.



 This study demonstrates the classification and functional characterization of patient-specific sensitivity to ubamatamab, a bispecific T cell engager in clinical development for ovarian cancer. Our ex vivo 3D micro-tumor platform facilitates quantification of immune responses and the discovery of predictive biomarkers to support patient stratification for emerging immunotherapies.



 Esmee Koedoot, Felix Behr, Timothy Sijsenaar, Léa Le Large, Cor D. de Kroon, Anne van Altena, Sander Bins, Dieudonné van der Meer, Willemijn Vader, Ingrid Boere, Nelleke Ottevanger, Judith Kroep. Bispecific antibody Ubamatamab induces personalized anti-tumor immune responses in an ex vivo platform for ovarian cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46429fd9066f1bc883507000c8c774ce1eb7d096" target='_blank'>
              Abstract A129: Bispecific antibody Ubamatamab induces personalized anti-tumor immune responses in an ex vivo platform for ovarian cancer
              </a>
            </td>
          <td>
            E. Koedoot, Felix Behr, Timothy J. Sijsenaar, Léa Le Large, C. D. de Kroon, A. V. van Altena, Sander Bins, Dieudonné J. van der Meer, W. Vader, Ingrid Boere, N. Ottevanger, J. Kroep
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) has been suggested as a key factor of tumor heterogeneity at the genomic and transcriptional levels, via unequal segregation and ecDNA hub formation, respectively. In this issue of Cancer Discovery, two studies further expand our understanding of the roles of ecDNA in enhancing tumor heterogeneity at the phenotypic and spatiotemporal levels, as well as in providing a strong selective advantage during tumor evolution. See related article by Montuori et al., p. 2054 See related article by Noorani et al., p. 2078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cba883c44a904170e7fcec05ae3700dab4a7298" target='_blank'>
              Extrachromosomal DNA: A Trusted Path to Tumor Heterogeneity.
              </a>
            </td>
          <td>
            Emmanuel E Korsah, Noah A Dusseau, Eunhee Yi
          </td>
          <td>2025-10-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d00dfc7c40a88ff7fd7b30582e0102312a0c30c" target='_blank'>
              Knowledge-informed multimodal cfDNA analysis improves sensitivity and generalization in cancer detection
              </a>
            </td>
          <td>
            Antonio De Falco, Piera Grisolia, Raffaella Giuffrida, Clara Iannarone, Cinzia Graziano, M. Scrima, A. Cossu, Rossella Tufano, Maria V Yow, Chloe Marissa Brown, Palak Bajaj, Marco Bocchetti, P. Nuzzo, F. Morgillo, Francesco Caraglia, Maria Carminia Delle Corte, G. di Guida, T. Troiani, Fortunato Ciardiello, Maria Rosaria Rizzo, Alfonso Fiorelli, N. Giorgiano, D. Arcaniolo, G. D. Rosa, Marco De Sio, A. Covre, A. D. Di Giacomo, Luana Calabrò, Paolo Fontana, Marzia Mare, O. Landgren, Damian J. Green, Alex Lesokhin, D. Coffey, Nipun B Merchant, J. Datta, Stefano Forte, M. Maio, M. Caraglia, M. Ceccarelli
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a hematologic cancer marked by clonal expansion of plasma cells in the bone marrow. Although its genomic landscape has been extensively characterized, the transcriptional mechanisms that govern malignant progression and long-term tumor cell survival remain incompletely understood. Methods We integrated single-cell RNA sequencing (scRNA-seq) data from healthy donors (HD), monoclonal gammopathy of undetermined significance (MGUS), smoldering multiple myeloma (SMM), and newly diagnosed multiple myeloma (NDMM) patients, retrieved from GEO datasets GSE124310 and GSE271107, to construct a comprehensive transcriptional landscape of plasma cell differentiation. Pseudotime and enrichment analyses identified PDIA4 as a terminal-state–associated gene. The prognostic significance of PDIA4 was validated using the multiple myeloma research foundation (MMRF) CoMMpass cohort. Functional analyses were performed in vitro and in vivo to validate the role of PDIA4 in MM cell survival and therapeutic response. Results Pseudotime trajectory analysis revealed progressive upregulation of genes involved in protein processing in the endoplasmic reticulum (ER), with PDIA4 identified as a top candidate in terminal-stage plasma cells. Survival analysis in the MMRF CoMMpass cohort further demonstrated that high PDIA4 expression correlated with poor overall survival. In RPMI-8226 cells, PDIA4 knockout activated the IRE1α/XBP1s branch of the unfolded protein response (UPR), impaired proliferation, and induced G1-phase arrest. PDIA4 depletion also sensitized cells to bortezomib. In vivo, sg-PDIA4 suppressed tumor growth in RPMI-8226 xenografts. Conclusions PDIA4 is a key regulator of the unfolded protein response and MM cell survival. Targeting PDIA4 may enhance the efficacy of proteasome inhibitors and offers a potential strategy to overcome therapeutic resistance in multiple myeloma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07098-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/881fab1a35c430a7450d4d3fe7ea84a60cfe6f7c" target='_blank'>
              Single-cell transcriptomics identifies PDIA4 as a marker of progression and therapeutic vulnerability in multiple myeloma
              </a>
            </td>
          <td>
            Wenjie Yu, Jiamin Zhan, Yanxia Wang, Xu Cao, Aoyang Yu, Yuexuan Rao, Can Huang, Xuan Zhang, Shunv Lu, Yanhua Liu, Chong Chen, Zhenyu Li
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb8107340798d9682d8c6977b5d27a13471e822" target='_blank'>
              Integrated single-cell and spatial transcriptomics uncover the prognostic, epigenetic, and immunological roles of FANCC in low-grade glioma.
              </a>
            </td>
          <td>
            Zongye Zhang, Zhi Sha, Han Liu, Yusheng Chen, Zhendong Liu, Xingbo Cheng, Sujie Gu, Yanzheng Gao
          </td>
          <td>2025-09-09</td>
          <td>Neurological research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="In prostate cancer, it is recognized that adenocarcinoma can transdifferentiate into neuroendocrine prostate cancer (NEPC) owing to lineage plasticity; however, transdifferentiation into other histological types remains uncertain. We present a case of a patient who underwent surgery for adenocarcinoma, which later recurred as prostate carcinosarcoma. Genomic analysis revealed a TMPRSS2-ERG fusion, confirming a common clonal origin and transdifferentiation from adenocarcinoma to carcinosarcoma. Additionally, we identified a frameshift mutation in TP53 and the loss of PTEN and RB1. Transcriptome analysis revealed enriched epithelial-mesenchymal transition and immune-related pathways, a pattern distinct from both adenocarcinoma and NEPC. To our knowledge, this is the first report that comprehensively evaluated the clonal origin of the rare prostate carcinosarcoma and characterized it using genomic and transcriptomic sequencing. It enhances our understanding of prostate cancer lineage plasticity and highlights the importance of developing novel therapies specifically targeted at prostate carcinosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc3dc165d184c74305476d6dbd5e449b98486ca" target='_blank'>
              Case Report: Genomic insights into prostate adenocarcinoma transdifferentiation to carcinosarcoma due to lineage plasticity
              </a>
            </td>
          <td>
            Tomohiro Fukui, Arinobu Fukunaga, Yuki Teramoto, Maki Fujiwara, Kensuke Hikami, T. Sunada, Kei Mizuno, Yuki Kita, Takayuki Sumiyoshi, Takayuki Goto, R. Saito, Takashi Kobayashi, S. Akamatsu
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Mitotic catastrophe (MC) is a well-recognized endogenous mechanism of tumor cell death, characterized as a delayed cell death process associated with aberrant mitosis. However, its prognostic significance in the context of intratumoral heterogeneity in esophageal squamous cell carcinoma (ESCC) remains largely unexplored. Methods We performed an in-depth analysis of single-cell RNA sequencing (scRNA-seq) data from ESCC obtained from the Gene Expression Omnibus (GEO) database. MC scores for individual cells were calculated using the AddModuleScore function, and T cell specific gene modules were identified via the high-dimensional weighted gene co-expression network analysis (hdWGCNA) framework. To further elucidate the developmental trajectories and intercellular interactions of T cells, pseudotime analysis and cell-cell communication inference were conducted. A prognostic risk model was then constructed using three machine learning algorithms combined with multivariate Cox regression analysis. Following risk stratification, we performed immune infiltration profiling, drug sensitivity analysis, and molecular docking to comprehensively assess the functional implications of the risk model in ESCC. Based on preliminary results from quantitative Real-time PCR (qRT-PCR) and Western blotting (WB), we selected the hub gene SLF2 for functional validation using wound healing, Cell Counting Kit-8 (CCK-8) assay, Transwell, and colony formation assays. Results Based on T cell mitotic catastrophe associated genes (MCAGs) and utilizing machine learning algorithms, we established a robust prognostic risk model for ESCC. The model demonstrated excellent stratification capability in predicting patient outcomes and effectively revealed the heterogeneity of the tumor immune microenvironment (TIME) and drug sensitivity. Furthermore, functional experiments confirmed that knockdown of the hub gene SLF2 significantly inhibited the migration, invasion, and proliferation of ESCC cells. Conclusion The prognostic model based on MCAGs we developed serves as an effective tool for predicting outcomes in ESCC.T cell-specific MCAGs drive intratumoral heterogeneity in ESCC, serving as potential prognostic biomarkers and therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00815-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4482ff5484bb99ce98963d648871eb4d8f4fb2bb" target='_blank'>
              Integrative single-cell and bulk transcriptomic analysis reveals the landscape of T cell mitotic catastrophe associated genes in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Shuang Li, Zheng Tao, Nan Wang, Yazhou Liu, Kai Xie, Haitao Ma
          </td>
          <td>2025-08-29</td>
          <td>Human Genomics</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="


 Despite the clear genetic evidence linking serous tubal intraepithelial carcinoma (STIC) and high-grade serous ovarian carcinoma (HGSOC), the specific conditions and events that promote the progression of STIC lesions into invasive disease remain poorly understood.



 As a critical initial step, we have assembled a cohort of incidental p53 signatures (p53.I), incidental STIC (STIC.I), and STIC with concurrent HGSOC (STIC.C). We have performed extensive multi-modal analysis using multiplexed tissue imaging and spatial transcriptomics to identify features of the immune system that play a vital role in the early steps of HGSOC development. We have processed 43 specimens using highly multiplexed tissue imaging at single-cell resolution (cyclic immunofluorescence, CyCIF), and 35 specimens for micro-regional spatial whole transcriptomics using the GeoMx (Nanostring) on over 500 pathologist-annotated regions of interest.



 We showed a temporal evolution of HGSOC, including the activation of the interferon (IFN) signaling pathway underpinning the progression from the very early stage, p53.I. Additional cancer hallmark pathways, such as Transforming Growth Factor Receptor-β, Hypoxia, were induced in the tumor microenvironment (TME), including STIC.C. One of the mechanisms of the early induction of IFN might be induced by DNA damage response and chromosomal instability, potentially activating the cGAS-STING pathway. We validated IFN activation at the early stage of HGSOC by tissue imaging. For instance, more epithelial cells expressing both HLA-A as well as HLA-E, p-TBK1 and p-STAT3 with disease progression than FT regardless of BRCA status (p<0.001, GLMMs). We found IFN signaling to be persistent at STIC.C and further upregulated in the tumor. To model the evolution of the cancer ecosystem, we then looked into immune composition at both protein and RNA levels. Our data suggested active immune surveillance and potential elimination for p53 signatures. Immune surveillance was potentially further increased at STIC.I by an increased activated cDC1, antigen-presenting cells (APCs), and >30% activated (PD1+) cytotoxic (CD8+ CD103-) and TRM with some level of exhaustion (LAG3+). STIC.C, presumably resembling late clonal expansion, had less intra-tumoral NK and cDC1 in both protein and RNA levels. There was a gradual decline in total CD8+ T cells with HGSOC progression, while 30-40% expressed Ki67/PD1 or LAG3 in cancer, indicating immune editing. Exhaustion state was further shown with HAVCR2 expression (TIM3). We found that one of the contributors of T cells exhaustion was chronic IFN.



 Epithelial cells with IFN activation seem to be under a natural selection with HSGSOC progression. One of the selective advantages might be the upregulation of MHC-class I. HLA-E upregulation by tumor cells was shown to be immune suppressive by reducing NK cell surveillance. Along with the gradual accumulation of cancer hallmark pathways, an emergence of immune suppressive mechanisms seems to arise at the very early precursor stage.



 Tanjina Kader, Jia-Ren Lin, Clemens Hug, Shannon Coy, Yu-An Chen, Ino de. Bruijin, Clarence Yapp, Jeremy L. Muhlich, Nikolaus Schultz, Charles W. Drescher, Peter K. Sorger, Ronny Drapkin, Sandro Santagata. Immune suppression in the fallopian tube: Insights from the pre-cancer atlas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA009.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bfa669bc19f1cf6d070d242569736aa782cd91a" target='_blank'>
              Abstract IA009: Immune suppression in the fallopian tube: Insights from the pre-cancer atlas
              </a>
            </td>
          <td>
            Tanjina Kader, Jia-Ren Lin, Clemens Hug, S. Coy, Yu-An Chen, Ino de. Bruijin, Clarence Yapp, J. Muhlich, N. Schultz, Charles W Drescher, P. Sorger, Ronny Drapkin, S. Santagata
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy characterized by a dense desmoplastic stroma and a nutrient-deprived tumor microenvironment (TME). These harsh conditions drive metabolic symbiosis, where cancer cells, stromal cells, and immune cells exchange metabolites to sustain tumor growth, therapy resistance, and immune evasion. Although MRTX1133 (KRASG12D inhibitor) has demonstrated promise in targeting the KRAS/MAPK pathway, responses differ throughout PDAC subtypes, and resistance has been reported. We treated PDAC cell lines with KRASG12D inhibitor (KRASi) and found a paradoxical increase in proliferation in some cell lines while suppressing growth in others, indicating the activation of compensatory survival mechanisms. In the sensitive cell lines, we found that KRASi disrupted growth, altered the metabolomic profile, and nutrient transport. We have identified SLC20A1, a phosphate transporter, as a novel KRAS-driven transporter. CRISPR/Cas9-mediated knockout of SLC20A1 disrupted the Warburg effect in PDAC cells. In conclusion, our data suggests that KRAS/MAPK inhibition exert robust control on nutrient transport as a mechanism to impair metabolism in KRAS-reliant PDAC cells.



 Harshita Nivrutti. Khedkar, Laiba Naz. Shiekh, Matthew Cheung, Don-Gerard Conde, Zeribe C. Nwosu. Oncogenic KRAS drives nutrient transport to support growth in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B020.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f683e1c06482d7e87be84283ced7cefbff45c117" target='_blank'>
              Abstract B020: Oncogenic KRAS drives nutrient transport to support growth in pancreatic cancer
              </a>
            </td>
          <td>
            H. N. Khedkar, Laiba Shiekh, Matthew Cheung, Don-Gerard Conde, Z. Nwosu
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Lineage plasticity, a critical hallmark of cancer progression, enables tumor cells to evade inhibition of primary oncogenic pathways through histologic transformation. This adaptive process, driven by stemness-associated features and epigenetic reprogramming, poses significant challenges in treatment. Using non-small cell lung cancer and prostate cancer as models, we examine the utility of tissue and liquid biopsies in detecting histologic transformations and tailoring treatments to specific subtypes, which has profound clinical implications, potentially improving outcomes in patients with advanced, therapy-resistant disease. We also discuss emerging therapeutic strategies, including novel molecular targets, and address ongoing clinical challenges in managing treatment-emergent histologic transformation.


SIGNIFICANCE
The advent of highly effective molecularly targeted therapies results in increased recognition of treatment-emergent histologic transformation. This review not only summarizes current evidence on diagnosis and management of lineage plasticity but also explores therapeutic strategies under study, outlining a framework for clinical translation and successful drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49dc4debf7b809a5797b2dd12ac58569636b53ca" target='_blank'>
              Histologic Transformation in Cancer: The Path for Clinical Translation.
              </a>
            </td>
          <td>
            Ioannis A. Vathiotis, Avisek Banerjee, Utsav Sen, Felix C Saalfeld, Debdatta Halder, A. Charpidou, Konstantin Syrigos, H. Horinouchi, Ashutosh K Tewari, Mark A Dawson, M. Wermke, Triparna Sen
          </td>
          <td>2025-09-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Colorectal laterally spreading tumors (LSTs) are defined as non-protruding neoplasms exceeding 10 mm in diameter that grow primarily along the intestinal wall. The morphogenetic mechanisms and evolutionary trajectories of granular-type LSTs (LST-Gs) towards colorectal carcinoma remain unclear. Methods In this study, we investigate the transcriptional features of LST-Gs using single-cell RNA sequencing technology by comparing them with protruded-type adenomas (PAs) and normal mucosal tissues. Results Adenomatous LST-Gs harbor an epithelial cell population with metaplastic differentiation, which are almost absent in PAs. Cells with a high degree of differentiation in LST-G demonstrate enhanced immunogenicity and robust adhesion/junction interactions. Furthermore, LST-Gs show upregulated expression of molecular chaperones and metallothioneins compared to PAs, reflecting a more hostile microenvironment similar to that observed in carcinoma stages. These characteristics suggest that LST-G exhibits greater heterogeneity compared to the earliest colorectal adenomas, mirroring the progression from precancerous states to cancer. Notably, the Arp2/3 complex is significantly upregulated in highly differentiated LST-G cell populations, potentially facilitating cell migration along the basement membrane, which highlights the similarities in cell motility between adenomatous LST-G and normal mucosal epithelium as well as serrated polyps (SERs). Conclusion The differentiation state of cells within LST-G exhibits a close correlation with their diverse characteristics. Metaplastic differentiation, as a prominent feature at the transcriptional level, demonstrates significant associations with the genomic features, morphogenesis, and tumor progression of LST-G.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21cfb702cab41be0ca69fc6ec0389c21e705f623" target='_blank'>
              Single-cell RNA profiling of colorectal granular-type laterally spreading tumor uncovers progression trajectory toward carcinoma and transcriptional signatures favoring lateral morphogenesis
              </a>
            </td>
          <td>
            Yueqing Gong, Yuxin Zhang, Xun Liu, Rongli Cui, Jingjing Lu, Jun Li, Fang Gu, Jing Zhang, Shigang Ding, Weiwei Fu
          </td>
          <td>2025-09-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Acquired treatment resistance in homologous recombination (HR) deficient high-grade serous ovarian cancer (HGSOC) commonly involves restoration of HR, often through secondary, somatic reversion mutations in HR genes. Our recent analysis of HR deficient, end-stage HGSOC found that most resistance mechanisms, including reversion mutations, are subclonal. The aim of this study is to investigate how the subclonal, reversion-negative cells persist during treatment. We are investigating single cell sequencing approaches that enable genotyping of individual cells, such that the phenotype of cells without reversions can be studied. In a pilot experiment we have used two HGSOC cell lines with known reversion mutations. To genotype individual cells and thus overcome the limitations of standard single-cell RNA sequencing (scRNA-seq), we combined short-read whole-transcriptome scRNA-seq with long-read, targeted single-cell cDNA sequencing on an Oxford Nanopore MinION using a 44-gene hybridization capture panel. A custom bioinformatics pipeline, incorporating Flexiplex for multiallelic codon resolution, annotated cell barcodes with their HR driver, reversion and TP53 mutations. Genotypes were then transferred to the short-read dataset following outlier and doublet filtering, and transcriptomic differences between cells with and without reversions were identified through differential gene expression analysis. We have repeated this protocol on a patient ascites sample. Prior short-read DNA sequencing from the cell lines and matched tumor samples defined the driver BRCA1 mutation, reversion events, pathogenic TP53 mutations, and whether there was loss of heterozygosity (LOH) at the BRCA1 loci. We successfully genotyped BRCA1 driver and reversion alleles in both cell lines in the long-read data. Rare cells heterozygous wildtype for the BRCA1 driver mutation were detected, suggesting either back-reversion events or subclones that escaped LOH - cells that were undetectable by bulk short-read methods. Cell genotypes were mapped to the short-read whole transcriptome data, and transcriptional programs that may support cell survival were identified in the non-reverted cells. Our approach demonstrates the utility of unbiased long-read sequencing for detecting BRCA1/2 reversions at single-cell resolution, including potential back-reversion subclones that are challenging to identify through bulk sequencing approaches. The differential gene expression results have identified potential resistance mechanisms. Ongoing work includes incorporation of fusion detection, structural-variant calling, and splice isoform analysis in our bioinformatics pipeline to further elucidate resistance pathways. We anticipate this work will result in an increased understanding of the resistance mechanisms that arise in HGSOC patients with HR deficiency, including identification of subclonal resistance mechanisms that cannot be detected through bulk sequencing approaches.



 Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne Willis, Joy Hendley, AOCS Study Group, Timothy Semple, Alicia Oshlack, Elizabeth L. Christie. Characterizing resistance at single cell resolution in HR deficient HGSOC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc7b444080721cb95cb0b3c57900c2232de880b" target='_blank'>
              Abstract PR019: Characterizing resistance at single cell resolution in HR deficient HGSOC
              </a>
            </td>
          <td>
            Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne O. Willis, J. Hendley, Aocs Study Group, T. Semple, Alicia Oshlack, Elizabeth L. Christie
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Understanding the functional impact of bacterial genetic diversity is crucial for linking pathogen variants to clinical outcomes. Here, we introduce a high-throughput cytological profiling pipeline optimized for Mycobacterium tuberculosis (Mtb) clinical strains, integrating OD-calibrated feature analysis and high-content microscopy. Our system quantifies single-bacterium morphological and physiological traits related to DNA replication, redox state, carbon metabolism, and cell envelope dynamics. Applied to 64 Mtb clinical isolates from lineages 1, 2, and 4, the approach revealed that cytological phenotypes recapitulate genetic relationships and exhibit both lineage- and density-dependent dynamics. Notably, we identified a link between a convergent "small cell" phenotype and a convergent ino1 mutation that is associated with the presence of an antisense transcript, suggesting a potential non-canonical regulatory mechanism under selection. In summary, we present a resource-efficient approach for mapping Mtb's phenotypic landscape, uncovering cellular traits that underlie its evolution and providing new insights into the functional consequences of bacterial genetic diversity.


IMPORTANCE
Understanding how genetic variation in Mycobacterium tuberculosis (Mtb) shapes its physical traits is essential to unraveling the evolution of this global pathogen. Here, we introduce a systematically optimized, high-throughput imaging platform for the comprehensive characterization of Mtb clinical strains. We demonstrate that Mtb's phenotypic manifestation is shaped by both genetic background and culture density. By accounting for these factors, our analysis linked distinct cellular dynamics to specific lineages, sublineages, and even single nucleotide variations. Notably, we linked a recurring mutation to a unique cell-shortening phenotype, finding that it potentially acts by creating a cryptic antisense transcript. This platform provides a powerful framework for systematically dissecting the physiological dynamics underlying Mtb evolution and identifying new therapeutic vulnerabilities of this deadly pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56fa61f881d5174f374abc12d00931b99537639c" target='_blank'>
              High-throughput cytological profiling uncovers genotype-phenotype associations in Mycobacterium tuberculosis clinical isolates.
              </a>
            </td>
          <td>
            Qingyun Liu, Yue J Liu, Ruiyuan Liu, Peter H. Culviner, Xin Wang, I. D. Wolf, Ken Chen, Yiwang Chen, Yi Xiao, Guiming Zhang, Rongfeng Sun, Shoko Wakabayashi, Nicole C Howard, Mingyu Gan, Eric J. Rubin, Sarah M. Fortune, Junhao Zhu
          </td>
          <td>2025-10-09</td>
          <td>mSystems</td>
          <td>0</td>
          <td>12</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 12],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>