<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-10 09:26:43 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Immune checkpoint inhibition (ICI) has fundamentally changed cancer treatment. However, only a minority of patients with metastatic triple negative breast cancer (TNBC) benefit from ICI, and the determinants of response remain largely unknown. To better understand the factors influencing patient outcome, we assembled a longitudinal cohort with tissue from multiple timepoints, including primary tumor, pre-treatment metastatic tumor, and on-treatment metastatic tumor from 117 patients treated with ICI (nivolumab) in the phase II TONIC trial. We used highly multiplexed imaging to quantify the subcellular localization of 37 proteins in each tumor. To extract meaningful information from the imaging data, we developed SpaceCat, a computational pipeline that quantifies features from imaging data such as cell density, cell diversity, spatial structure, and functional marker expression. We applied SpaceCat to 678 images from 294 tumors, generating more than 800 distinct features per tumor. Spatial features were more predictive of patient outcome, including features like the degree of mixing between cancer and immune cells, the diversity of the neighboring immune cells surrounding cancer cells, and the degree of T cell infiltration at the tumor border. Non-spatial features, including the ratio between T cell subsets and cancer cells and PD-L1 levels on myeloid cells, were also associated with patient outcome. Surprisingly, we did not identify robust predictors of response in the primary tumors. In contrast, the metastatic tumors had numerous features which predicted response. Some of these features, such as the cellular diversity at the tumor border, were shared across timepoints, but many of the features, such as T cell infiltration at the tumor border, were predictive of response at only a single timepoint. We trained multivariate models on all of the features in the dataset, finding that we could accurately predict patient outcome from the pre-treatment metastatic tumors, with improved performance using the on-treatment tumors. We validated our findings in matched bulk RNA-seq data, finding the most informative features from the on-treatment samples. Our study highlights the importance of profiling sequential tumor biopsies to understand the evolution of the tumor microenvironment, elucidating the temporal and spatial dynamics underlying patient responses and underscoring the need for further research on the prognostic role of metastatic tissue and its utility in stratifying patients for ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1bdaa11baefcf7560def6afb49fb7f0020d09d" target='_blank'>
              Temporal and spatial composition of the tumor microenvironment predicts response to immune checkpoint inhibition
              </a>
            </td>
          <td>
            N. Greenwald, I. Nederlof, Cameron Sowers, Daisy Yi Ding, Seongyeol Park, Alex Kong, Kathleen E. Houlahan, Sricharan Reddy Varra, Manon de Graaf, Veerle Geurts, Candace C. Liu, Jolene S Ranek, L. Voorwerk, M. de Maaker, Adam Kagel, E. Mccaffrey, Aziz Khan, C. Yeh, C. Fullaway, Zumana Khair, Yunhao Bai, Hadeesha Piyadasa, Tyler Risom, A. Delmastro, Felix J. Hartmann, Lise Mangiante, Cristina Sotomayor-Vivas, Ton N. Schumacher, Zhicheng Ma, M. Bosse, Marc J van de Vijver, R. Tibshirani, Hugo M. Horlings, Christina Curtis, Marleen Kok, Michael Angelo
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>157</td>
        </tr>

        <tr id="While recent innovations in spatial biology have driven new insights into how tissue organization is altered in disease, interpreting these datasets in a generalized and scalable fashion remains a challenge. Computational workflows for discovering condition-specific differences in tissue organization typically rely on pairwise comparisons or unsupervised clustering. In many cases, these approaches are computationally expensive, lack statistical rigor, and are insensitive to low-prevalence cellular niches that are nevertheless highly discriminative and predictive of patient outcomes. Here, we present QUICHE – an automated, scalable, and statistically robust method that can be used to discover cellular niches differentially enriched in spatial regions, longitudinal samples, or clinical patient groups. In contrast to existing methods, QUICHE combines local niche detection with interpretable statistical modeling using graph neighborhoods to detect differentially enriched cellular niches, even at low prevalence. Using in silico models and spatial proteomic imaging of human tissues, we demonstrate that QUICHE can accurately detect condition-specific cellular niches occurring at a frequency of 0.5% in fewer than 20% of patient samples, outperforming the next best method which required a patient prevalence of 60% for detection. To validate our approach and understand how tumor structure influences recurrence risk in triple negative breast cancer (TNBC), we used QUICHE to comprehensively profile the tumor microenvironment in a multi-center, spatial proteomics cohort consisting of primary surgical resections, analyzing over 2 million cells from 314 patients across 5 medical centers. We discovered cellular niches that were consistently enriched in key regions of the tumor microenvironment, including the tumor-immune border and extracellular matrix remodeling regions, as well as niches statistically-associated with patient outcomes, including recurrence status and recurrence-free survival. The majority of differential niches (74.2%) were specific to patients that did not relapse and formed a robust interconnected network enriched in monocytes, macrophages, APCs, and CD8T cells with tumor and stroma cells. In contrast, the interaction network for patients that relapsed was notably sparse and enriched in B cells, CD68 macrophages and neutrophils. We validated these findings using two independent cohorts, observing similar cellular interactions and predictive power. Collectively, these results suggest that salient, generalized profiles of productive anti-tumor immune responses are defined by a network of structural engagement between innate and adaptive immunity with tumor and stromal cells, rather than by any single specific cell population. We have made QUICHE freely available as a user-friendly open-source Python package at https://github.com/jranek/quiche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cda789c041e695c92206356373353eb906709b05" target='_blank'>
              QUICHE reveals structural definitions of anti-tumor responses in triple negative breast cancer
              </a>
            </td>
          <td>
            Jolene S Ranek, Noah F. Greenwald, Mako Goldston, C. Fullaway, Cameron Sowers, Alex Kong, S. Mourón, M. Quintela-Fandino, Robert West, Michael Angelo
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions driving progression and pathobiology. How these spatial patterns and interactions evolve with treatment remains unclear. Here, we apply imaging mass cytometry to analyse protein-level changes in paired pre- and post-treatment GBM samples from five patients. We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes. These findings provide insight into treatment-induced shifts in GBM’s cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets. Key points Spatial organisation in primary GBM consist of layers driven by the presence of hypoxia The layers in recurrent GBM appear are driven more by the presence of reactive astrocytes Increased cellular cross-talk in recurrent GBM presents novel therapeutic targets">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3061625679fbc2f861a2180efc0a648894c17487" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment
              </a>
            </td>
          <td>
            Shoaib Ajaib, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F. Stead
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Given the successful graft-versus-leukemia cell treatment effect observed with allogeneic hematopoietic stem cell transplant for patients with refractory or relapsed acute myeloid leukemia, immunotherapies have also been investigated in the nontransplant setting. Here, we use a multi-omic approach to investigate spatiotemporal interactions in the bone marrow niche between leukemia cells and immune cells in patients with refractory or relapsed acute myeloid leukemia treated with a combination of the immune checkpoint inhibitor pembrolizumab and hypomethylating agent decitabine. We derived precise segmentation data by extensively training nuclear and membrane cell segmentation models, which enabled accurate transcript assignment and deep learning-feature-based image analysis. To overcome read-depth limitations, we integrated the single-cell RNA sequencing data with single-cell-resolution spatial transcriptomic data from the same sample. Quantifying cell-cell distances between cell edges rather than cell centroids allowed us to conduct a more accurate downstream analysis of the tumor microenvironment, revealing that multiple cell types of interest had global enrichment or local enrichment proximal to leukemia cells after pembrolizumab treatment, which could be associated with their clinical responses. Furthermore, ligand-receptor analysis indicated a potential increase in TWEAK signaling between leukemia cells and immune cells after pembrolizumab treatment. Highlights Spatial transcriptomic analysis of R-AML bone marrow niches provides detailed information about intercellular interactions in the tumor microenvironment. Immunotherapy shifts the cell composition of the leukemia neighborhood. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79158e9c810aea3af6c5a5e81cece5524c3196f" target='_blank'>
              Single Cell Spatial Transcriptomics Reveals Immunotherapy-Driven Bone Marrow Niche Remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R. Herzog, , Laura W Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine Calvo, Christopher S Hourigan, Kasper Hansen, Chen Zhao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Imaging-based spatial transcriptomics (ST) is evolving rapidly as a pivotal technology in studying the biology of tumors and their associated microenvironments. However, the strengths of the commercially available ST platforms in studying spatial biology have not been systematically evaluated using rigorously controlled experiments. In this study, we used serial 5-m sections of formalin-fixed, paraffin-embedded surgically resected lung adenocarcinoma and pleural mesothelioma tumor samples in tissue microarrays to compare the performance of the single cell ST platforms CosMx, MERFISH, and Xenium (uni/multi-modal) platforms in reference to bulk RNA sequencing, multiplex immunofluorescence, GeoMx Digital Spatial Profiler, and hematoxylin and eosin staining data for the same samples. In addition to objective assessment of automatic cell segmentation and phenotyping, we performed pixel-resolution manual evaluation of phenotyping to carry out pathologically meaningful comparison between ST platforms. Our study detailed the intricate differences between the ST platforms, revealed the importance of parameters such as tissue age and probe design in determining the data quality, and suggested reliable workflows for accurate spatial profiling and molecular discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17805d448b42a38f2a1b50b0eb2fd17aaa264ce4" target='_blank'>
              Comparison of imaging-based single-cell resolution spatial transcriptomics profiling platforms using formalin-fixed, paraffin-embedded tumor samples
              </a>
            </td>
          <td>
            Nejla Ozirmak Lermi, Max Molina Ayala, Sharia D Hernandez, Wei Lu, Khaja B. Khan, Alejandra Serrano, Idania Lubo, Leticia Hamana, K. Tomczak, Sean Barnes, Jinzhuang Dou, Qingnan Liang, Rti Team, M. Gabriela Raso, Ximing Tang, Mei Jiang, B. Sanchez-Espiridion, A. Weissferdt, J. Heymach, Jianjun Zhang, B. Sepesi, T. Cascone, Anne Tsao, Mehmet Altan, Reza J Mehran, Don L Gibbons, Ignacio I. Wistuba, C. Haymaker, Ken Chen, L. S. Solis Soto
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to randomly generated nucleotide barcodes. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple-negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3ba3313bee111d463805c5f2fb966ea7f8b649e" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="To date, few studies have detailed the tumor microenvironment (TME) of Ewing sarcoma (EwS). The TME has a vital role in cancer survival and progression with implications in drug resistance and immune escape. By performing spatially resolved transcriptomic analysis of primary treatment-naïve EwS samples, we discovered greater stromal enrichment in localized EwS tumors compared to metastatic EwS tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumorigenic role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling, suggesting a potential immune-evasive mechanism and immunotherapy target. Our findings provide insight into tumor cell/stromal cell interactions in EwS and serve as a valuable resource for further investigations in the tumor immune microenvironment of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245c7e47704099e340bfbb4047050883a6a92e6d" target='_blank'>
              Tumor-associated stroma shapes the spatial tumor immune microenvironment of primary Ewing sarcomas
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) carries an extremely poor prognosis, in part resulting from cellular heterogeneity that supports overall tumorigenicity. Cancer associated fibroblasts (CAF) are key determinants of PDAC biology and response to systemic therapy. While CAF subtypes have been defined, the effects of patient-specific CAF heterogeneity and plasticity on tumor cell behavior remain unclear. Here, multi-omics was used to characterize the tumor microenvironment (TME) in tumors from patients undergoing curative-intent surgery for PDAC. In these same patients, matched tumor organoid and CAF lines were established to functionally validate the impact of CAFs on the tumor cells. CAFs were found to drive epithelial-mesenchymal transition (EMT) and a switch in tumor cell classificiaton from classical to basal subtype. Furthermore, we identified CAF-specific interleukin 8 (IL-8) as an important modulator of tumor cell subtype. Finally, we defined neighborhood relationships between tumor cell and T cell subsets. Statement of Significance This multidimensional analysis highlights the diverse role CAFs have in influencing other cell types in the TME, including epithelial-derived tumor and infiltrating immune cells. Our methods provide a platform for evaluating emerging therapeutic approaches and for studying mechanisms that dictate tumor behavior in a manner that reflects patient-specific heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d173d5090e9fb969bbd5c7fddf327d5b9643a1d" target='_blank'>
              Cancer associated fibroblasts drive epithelial to mesenchymal transition and classical to basal change in pancreatic ductal adenocarcinoma cells with loss of IL-8 expression
              </a>
            </td>
          <td>
            Samantha Guinn, Brayan Perez, Joseph A Tandurella, Mili Ramani, Jae W. Lee, Daniel J. Zabransky, Emma Kartalia, Jignasha Patel, H. Zlomke, Norman G Nicolson, Sarah M. Shin, Benjamin Barrett, Nicholas Sun, Alexei Hernandez, Erin M Coyne, Courtney D Cannon, Nicole E. Gross, S. Charmsaz, Yeonju Cho, James Leatherman, Melissa R Lyman, Jacob T. Mitchell, Luciane T. Kagohara, Michael G. Goggins, Kelly J. Lafaro, Jin He, Christopher R Shubert, William R Burns, Lei Zheng, E. Fertig, Elizabeth M. Jaffee, Richard A Burkhart, W. Ho, Jacquelyn W. Zimmerman
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Spatial profiling of tissues promises to elucidate tumor-microenvironment interactions and generate prognostic and predictive biomarkers. We analyzed single-cell, spatial data from three multiplex imaging technologies: cyclic immunofluorescence (CycIF) data we generated from 102 breast cancer patients with clinical follow-up, and publicly available imaging mass cytometry and multiplex ion-beam imaging datasets. Similar single-cell phenotyping results across imaging platforms enabled combined analysis of epithelial phenotypes to delineate prognostic subtypes among estrogen-receptor positive (ER+) patients. We utilized discovery and validation cohorts to identify biomarkers with prognostic value. Increased lymphocyte infiltration was independently associated with longer survival in triple-negative (TN) and high-proliferation ER+ breast tumors. An assessment of ten spatial analysis methods revealed robust spatial biomarkers. In ER+ disease, quiescent stromal cells close to tumor were abundant in good prognosis tumors, while tumor cell neighborhoods containing mixed fibroblast phenotypes were enriched in poor prognosis tumors. In TN disease, macrophage/tumor and B/T lymphocyte neighbors were enriched and lymphocytes were dispersed in good prognosis tumors, while tumor cell neighborhoods containing vimentin-positive fibroblasts were enriched in poor prognosis tumors. In conclusion, we generated comparable single-cell spatial proteomic data from several clinical cohorts to enable prognostic spatial biomarker identification and validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e86a33275ed2b41ff7e598b0d0e6585ea48b22" target='_blank'>
              Highly multiplexed imaging reveals prognostic immune and stromal spatial biomarkers in breast cancer.
              </a>
            </td>
          <td>
            Jennifer Eng, Elmar Bucher, Zhi Hu, Cameron R Walker, Tyler Risom, Michael Angelo, Paula I. Gonzalez-Ericsson, M. E. Sanders, A. B. Chakravarthy, Jennifer A Pietenpol, Summer L. Gibbs, Rosalie C. Sears, Koei Chin
          </td>
          <td>2025-01-14</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Dedifferentiation programs are commonly enacted during breast cancer progression to enhance tumor cell fitness. Increased cellular plasticity within the neoplastic compartment of tumors correlates with disease aggressiveness, often culminating in greater resistance to cytotoxic therapies or the augmented ability to metastasize to distant organs. Here we report that subpopulations of dedifferentiated neoplastic breast epithelial cells express canonical leukocyte cell surface receptor proteins and have thus named this cellular program “immune mimicry.” We document neoplastic cells engaging in immune mimicry within public human breast tumor single-cell RNA-seq datasets, histopathological breast tumor specimens, breast cancer cell lines, as well as in murine transgenic and cell line-derived mammary cancer models. Immune-mimicked neoplastic cells harbor hallmarks of dedifferentiation and appear enriched in treatment-resistant and high-grade breast tumors. We corroborated these observations in aggressive breast cancer cell lines where growth-arresting cytotoxic chemotherapies drove epithelial cells toward immune mimicry. Moreover, in subsequent proof-of-concept studies, we demonstrate that expression of the CD69 leukocyte activation marker by neoplastic cells confers a proliferative advantage that facilitates early tumor growth. We conclude neoplastic breast epithelial cells that upregulate leukocyte surface receptors potentiate malignancy. Moving forward, neoplastic immune mimicry should be evaluated for prognostic utility in breast cancer to determine its stratification potential for patients with increased risks of tumor recurrence, metastasis, and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f218f65369b11c4451676b1863ebb98feeb72b72" target='_blank'>
              Neoplastic immune mimicry is a generalizable phenomenon in breast cancer and epithelial CD69 enables early tumor progression
              </a>
            </td>
          <td>
            Eric B. Berens, Sokchea Khou, Elaine Huang, Amber Hoffman, Briana Johnson, Nell Kirchberger, Shamilene Sivagnanam, Nicholas L. Calistri, Daniel Derrick, T. Liby, Ian C. Mclean, Aryn A. Alanizi, F. Ozmen, T. Y. Ozmen, Gordon B. Mills, E. S. Hwang, P. Schedin, Hugo Gonzalez, Z. Werb, Laura M. Heiser, L. Coussens
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="The impact of cancer driving mutations in regulating immunosurveillance throughout tumor development remains poorly understood. To better understand the contribution of tumor genotype to immunosurveillance, we generated and validated lentiviral vectors that create an epi-allelic series of increasingly immunogenic neoantigens. This vector system is compatible with autochthonous Cre-regulated cancer models, CRISPR/Cas9-mediated somatic genome editing, and tumor barcoding. Here, we show that in the context of KRAS-driven lung cancer and strong neoantigen expression, tumor suppressor genotype dictates the degree of immune cell recruitment, positive selection of tumors with neoantigen silencing, and tumor outgrowth. By quantifying the impact of 11 commonly inactivated tumor suppressor genes on tumor growth across neoantigenic contexts, we show that the growth promoting effects of tumor suppressor gene inactivation correlate with increasing sensitivity to immunosurveillance. Importantly, specific genotypes dramatically increase or decrease sensitivity to immunosurveillance independently of their growth promoting effects. We propose a model of immunoediting in which tumor suppressor gene inactivation works in tandem with neoantigen expression to shape tumor immunosurveillance and immunoediting such that the same neoantigens uniquely modulate tumor immunoediting depending on the genetic context. One Sentence Summary Here we uncover an under-appreciated role for tumor suppressor gene inactivation in shaping immunoediting upon neoantigen expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a2622766d9e8241bb1972eff5c441baddd14069" target='_blank'>
              Tumor suppressor genotype influences the extent and mode of immunosurveillance in lung cancer
              </a>
            </td>
          <td>
            Keren M. Adler, Haiqing Xu, Amy C. Gladstein, Valerie M. Irizarry-Negron, Maggie R. Robertson, Katherine R. Doerig, D. Petrov, M. Winslow, D. Feldser
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="High-risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single-cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this gap, we profiled untreated and chemotherapy-treated high-risk neuroblastomas from 2 patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p-gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells and we experimentally demonstrated that CCL18 increases neuroblastoma cell migration. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical-like cells expressed the ALK ligand ALKAL2 and were predicted to communicate with neighboring ALK expressing cancer cells. We demonstrated a unique developmental pattern of adrenal medulla-specific expression of ALK and adrenocortical-specific expression of ALKAL2, suggesting a role of these signaling interactions in neuroblastoma carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cacb0dcc803b6c58db82e59f6d5ec533dad1808" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment unveils novel paracrine interactions
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlson, A. Claeys, Eva Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Osteosarcoma is the most common malignant bone tumor in young patients and remains a significant clinical challenge, particularly in the context of metastatic disease. Despite extensive documentation of genomic alterations in osteosarcoma, studies detailing the immunosuppressive mechanisms within the metastatic osteosarcoma microenvironment are lacking. Our objective was to characterize the spatial transcriptional landscape of metastatic osteosarcoma to reveal these immunosuppressive mechanisms and identify promising therapeutic targets. Here, we performed spatial transcriptional profiling on a cohort of osteosarcoma pulmonary metastases from pediatric patients. We reveal a conserved spatial gene expression pattern resembling a foreign body granuloma, characterized by peripheral inflammatory signaling, fibrocollagenous encapsulation, lymphocyte exclusion, and peritumoral macrophage accumulation. We also show that the intratumoral microenvironment of these lesions lack inflammatory signaling. Additionally, we identified CXCR4 as an actionable immunomodulatory target that bridges both the intratumoral and extratumoral microenvironments and highlights the spatial heterogeneity and complexity of this pathway. Collectively, this study reveals that metastatic osteosarcoma specimens are comprised of multiple regionally distinct immunosuppressive microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2ee7e019ae5fcae794fe1bb25f818cd218fc98" target='_blank'>
              Spatial profiling identifies regionally distinct microenvironments and targetable immunosuppressive mechanisms in pediatric osteosarcoma pulmonary metastases
              </a>
            </td>
          <td>
            Jason Eigenbrood, Nathan Wong, Paul Mallory, Janice Pereira, Douglass W Morris, Jessica A. Beck, James C Cronk, Carly M Sayers, Monica Mendez, Linus Kaiser, Julie Galindo, Jatinder Singh, Ashley Cardamone, M. Pore, Michael Kelly, Amy K LeBlanc, Jennifer Cotter, Rosandra N Kaplan, T. McEachron
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="A key challenge in cancer research is to identify the secreted factors that contribute to tumor cell survival. Nowhere is this more evident than in Hodgkin lymphoma, where malignant Hodgkin Reed Sternberg (HRS) cells comprise only 1-5% of the tumor mass, the remainder being infiltrating immune cells that presumably are required for the survival of the HRS cells. Until now, there has been no way to characterize the complex Hodgkin lymphoma tumor microenvironment at genome scale. Here, we performed genome-wide transcriptional profiling with spatial and single-cell resolution. We show that the neighborhood surrounding HRS cells forms a distinct niche involving 31 immune and stromal cell types and is enriched in CD4+ T cells, myeloid and follicular dendritic cells, while being depleted of plasma cells. Moreover, we used machine learning to nominate ligand-receptor pairs enriched in the HRS cell niche. Specifically, we identified IL13 as a candidate survival factor. In support of this hypothesis, recombinant IL13 augmented the proliferation of HRS cells in vitro. In addition, genome-wide CRISPR/Cas9 loss-of-function studies across more than 1,000 human cancer cell lines showed that IL4R and IL13RA1, the heterodimeric partners that constitute the IL13 receptor, were uniquely required for the survival of HRS cells. Moreover, monoclonal antibodies targeting either IL4R or IL13R phenocopied the genetic loss of function studies. IL13-targeting antibodies are already FDA-approved for atopic dermatitis, suggesting that clinical trials testing such agents should be explored in patients with Hodgkin lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0453e5a36d23742e3f6af668adec8ef2f3f547e8" target='_blank'>
              Genome-scale spatial mapping of the Hodgkin lymphoma microenvironment identifies tumor cell survival factors
              </a>
            </td>
          <td>
            Vignesh Shanmugam, Neriman Tokcan, Daniel Chafamo, Sean Sullivan, Mehdi Borji, Haley Martin, Gail A Newton, Naeem M. Nadaf, Saoirse Hanbury, Irving Barrera, Dylan Cable, Jackson Weir, Orr Ashenberg, Geraldine Pinkus, Scott J Rodig, Caroline Uhler, Evan Z. Macosko, Margaret M. Shipp, A. Louissaint, Fei Chen, Todd R. Golub
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Understanding cell-cell communication and its dependence on spatial organization is critical for unraveling tissue complexity and organ function. This study integrates single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics (ST) to systematically assess how spatial niches influence gene expression and intercellular communication. Using breast cancer, brain cortex, and heart datasets, our analyses reveal limited global transcriptional changes in cells depending on their spatial microenvironment, with differential gene expression observed in less than half the samples explored. Moreover, cell-cell communication predictions, derived from ligand-receptor pairs, exhibit minimal correlation with spatial colocalization of cell types. Overall, our study underscores the limitations of using scRNA-seq data to capture niche-specific molecular interactions, even when spatial information is leveraged, and it highlights the need for novel strategies to refine our understanding of intercellular communication dynamics at molecular level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790c5632749fda7ad54ef1434d0ac14122b25196" target='_blank'>
              Systematic assessment of microenvironment-dependent transcriptional patterns and intercellular communication
              </a>
            </td>
          <td>
            Elena Pareja-Lorente, Patrick Aloy
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Spatially non-resolved transcriptomic data identified functionally distinct populations of fibroblasts in health and disease. However, in-depth transcriptional profiling in situ at single-cell resolution has not been possible so far. Here, we studied fibroblast populations in the skin of SSc patients and healthy individuals using cyclic in situ hybridization (cISH) as a novel approach for spatially-resolved transcriptional phenotyping with subcellular resolution. cISH deconvoluted the heterogeneity of 20,979 cells including 3,764 fibroblasts (FB). BANKSY-based spatially-informed clustering identified nine FB subpopulations, with SFRP2+ RetD FB and CCL19+ nonPV FB as novel subpopulations that reside in specific cellular niches and display unique gene expression profiles. SFRP2+ RetD FB and CCL19+ nonPV FB as well as COL8A1+ FB, display altered frequencies in SSc skin and play specific, disease-promoting roles for extracellular matrix release and leukocyte recruitment as revealed by their transcriptional profile, their cellular interactions and ligand-receptor analyses. The frequencies of COL8A1+ FB and their interactions with monocytic cells and B cells are associated with progression of skin fibrosis in SSc. In summary, our spatially-resolved transcriptomic approach identified novel fibroblast subpopulations deregulated in SSc skin with specific pathogenic roles, some of which may potentially serve as biomarkers for progression of skin fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c095f7397487cd95a22547f9e4b6ee14b870509e" target='_blank'>
              Spatially informed phenotyping by cyclic-in-situ-hybridization identifies novel fibroblast populations and their pathogenic niches in systemic sclerosis
              </a>
            </td>
          <td>
            Yi-Nan Li, Tim Filla, A. Györfi, Minrui Liang, Veda Devakumar, Alexandru Micu, Hongtao Chai, C. Bergmann, A. Pecher, Jörg Henes, P. Moinzadeh, Thomas Krieg, Alexander Kreuter, G. Schett, Bernhard Homey, Sascha Dietrich, J. Distler, A. Matei
          </td>
          <td>2024-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The intra- and inter-tumoral heterogeneity of glioblastoma represents a significant therapeutic challenge, as well as difficulty in generating reliable models for in vitro studies. Historical 2D adherent cell lines do not recapitulate this complexity, whereas both patient-derived neurospheres (PDN) and organoids (PDO) demonstrate intra-tumoral heterogeneity. Here, we quantify the tumor cell composition from matched models established from the same primary tumor using a series of multi-omic interrogations. We find that both patient-derived models recapitulate the genomic, epigenomic and tumor cell heterogeneity of the primary tissue. Furthermore, single-nuclei RNA sequencing revealed a subset of organoids containing small numbers of non-malignant cells from neuron and immune cell lineages. Harnessing the intra-tumoral heterogeneity of PDN models, we reveal the impact of temozolomide chemotherapy on individual cell states, altering composition of tumors over time in response to treatment. Our data confirms that both patient-derived models recapitulate patient intra-tumoral heterogeneity providing a platform for tumor cell state refined therapeutic studies. Key Points Generation of matched patient-derived neurosphere and organoid models from resected GBM tissue Neurosphere models exhibit greater proliferative signatures Both patient-derived models recapitulate genomic and epigenomic features of the primary tissue Single-nuclei RNA sequencing reveals both models recapitulate intra-tumoral heterogeneity of the primary tissue Neurosphere models enable interrogation of therapeutic responses in the context of heterogeneity Importance of study Patient derived models can be powerful tools when they faithfully recapitulate the tumor tissue from which they are derived. In glioblastoma, patient derived neurospheres (PDN) and organoids (PDO) have both been used in studies, however the differences between these models and the recapitulation of patient heterogeneity remain to be fully characterized. To address this, we performed multi-omic profiling of PDN and PDO models generated from the same tumor tissue. We find that that across a range of data modalities, both model systems exhibit a high level of resemblance to tissue, and critically, maintain heterogeneity and tumor cell composition. The importance of modeling heterogeneity was demonstrated in PDN models, where temozolomide treatment specifically alters the abundance of MES-like and AC-like tumor cells. Our findings demonstrate that neurosphere and organoid models effectively preserve cellular heterogeneity, genomic alterations, methylation signatures and transcriptomic features, both highly suitable to model glioblastoma’s complex cellular landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c3caa2d109062b5ff4f7f9f68c99c0d6be111c" target='_blank'>
              Three-dimensional patient-derived models of glioblastoma retain intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            Zachery Moore, Claire Storey, Daniel V. Brown, A. Valkovic, Montana Spiteri, Jasmine F. Pignatelli, Shannon J Oliver, Alana Fakhri, Kate Drummond, Seth Malinowski, Keith L. Ligon, Oliver M. Sieber, J. Whittle, S. Freytag, Sarah A. Best
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Recent advancements in spatial transcriptomics technologies have significantly enhanced resolution and throughput, underscoring an urgent need for systematic benchmarking. To address this, we collected clinical samples from three cancer types – colon adenocarcinoma, hepatocellular carcinoma, and ovarian cancer – and generated serial tissue sections for systematic evaluation. Using these uniformly processed samples, we generated spatial transcriptomics data across five high-throughput platforms with subcellular resolution: Stereo-seq v1.3, Visium HD FFPE, Visium HD FF, CosMx 6K, and Xenium 5K. To establish ground truth datasets, we profiled proteins from adjacent tissue sections corresponding to all five platforms using CODEX and performed single-cell RNA sequencing on the same samples. Leveraging manual cell segmentation and detailed annotations, we systematically assessed each platform’s performance across key metrics, including capture sensitivity, specificity, diffusion control, cell segmentation, cell annotation, spatial clustering, and transcript-protein alignment with adjacent CODEX. The uniformly generated, processed, and annotated multi-omics dataset is valuable for advancing computational method development and biological discoveries. The dataset is accessible via SPATCH, a user-friendly web server for visualization and download (http://spatch.pku-genomics.org/).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a092f418d2d0937b601a2e81131e13ca731653a" target='_blank'>
              Systematic Benchmarking of High-Throughput Subcellular Spatial Transcriptomics Platforms
              </a>
            </td>
          <td>
            Pengfei Ren, Rui Zhang, Yunfeng Wang, Peng Zhang, Ce Luo, Suyan Wang, Xiaohong Li, Zongxu Zhang, Yanping Zhao, Yufeng He, Haorui Zhang, Yufeng Li, Zhidong Gao, Xiuping Zhang, Yahui Zhao, Zhihua Liu, Yuanguang Meng, Zhe Zhang, Zexian Zeng
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background & Aims Epithelial tumors generally resemble the cellular architecture of their tissue of origin. However, this link remains largely unexplored in the pancreas. Methods Using Nanostring GeoMx DSP®, Resolve Molecular Cartography® and Nanostring CosMx®, and integration with single cell RNAseq datasets, we mapped the human pancreatic ductal epithelium in non-neoplastic pancreas, and compared it to pancreatic cancer subtypes. Results Groups of Keratin-5+ cells among the Pan-Cytokeratin+ cells in the duct have a gene signature reminiscent of stem cells and (supra)basal cells from other tissues. In spatial analysis at single cell resolution, the pancreatic duct manifests as a stratified epithelium comprising a basal and four luminal populations; In large ducts, KRT5+ basal (BAS) cells express ΔNp63 while KRT5+ luminal (LUM)-B cells reside supra-basally and are distinct from the common KRT5- LUM-A cells. LUM-C and LUM-D cells pertain to intercalated ducts and ductal glands, respectively. LUM-A and -C cells express gel-forming mucins while LUM-B cells have membrane-bound MUC4 and MUC16. In cancer, BAS and LUM-B signatures associate with the basal-like pancreatic ductal adenocarcinoma (PDAC) and correlate with lower survival but exhibit a mixed spatial pattern with a diffracted gene signature. In contrast, adenosquamous cancers of the pancreas (ASCP) preserve the normal spatially unmixed identity of LUM-B cells and BAS cells that is regulated by ΔNp63. Next to ΔNp63, conserved drug targets were identified for both populations. Conclusion This study offers a refined pancreatic tumor classification based on the native ductal architecture, with better preservation of the LUM-B and BAS cell population identity in ASCP than in PDAC, including the conserved drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a2f6f8e9909be212c547b03063b014e38a8855" target='_blank'>
              Spatial map of native duct cell populations in human pancreas and their representation in pancreatic cancers
              </a>
            </td>
          <td>
            JL Van den Bossche, M. Van der Vliet, E. Michiels, O. A. Senar, Z. Madran, K. Coolens, S. Van Lint, M. Nacher, T. Arsenijevic, N. Messaoudi, P. Lefesvre, E. Montanya, M. Rovira, J. Van Laethem, J. Baldan, I. Houbracken, I. Rooman
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Inadequate response to androgen deprivation therapy (ADT) frequently arises in prostate cancer, driven by cellular mechanisms that remain poorly understood. Here, we integrated single-cell RNA sequencing, single-cell multiomics, and spatial transcriptomics to define the transcriptional, epigenetic, and spatial basis of cell identity and castration response in the mouse prostate. Leveraging these data along with a meta-analysis of human prostates and prostate cancer, we identified cellular orthologs and key determinants of ADT response and resistance. Our findings reveal that mouse prostates harbor lobe-specific luminal epithelial cell types distinguished by unique gene regulatory modules and anatomically defined androgen-responsive transcriptional programs, indicative of divergent developmental origins. Androgen-insensitive, stem-like epithelial populations - resembling human club and hillock cells - are notably enriched in the urethra and ventral prostate but are rare in other lobes. Within the ventral prostate, we also uncovered two additional androgen-responsive luminal epithelial cell types, marked by Pbsn or Spink1 expression, which align with human luminal subsets and may define the origin of distinct prostate cancer subtypes. Castration profoundly reshaped luminal epithelial transcriptomes, with castration-resistant luminal epithelial cells activating stress-responsive and stemness programs. These transcriptional signatures are enriched in tumor cells from ADT-treated and castration-resistant prostate cancer patients, underscoring their likely role in driving treatment resistance. Collectively, our comprehensive cellular atlas of the mouse prostate illuminates the importance of lobe-specific contexts for prostate cancer modeling and reveals potential therapeutic targets to counter castration resistance. Significance Statement Androgen deprivation therapy is a mainstay in prostate cancer treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer new directions for prostate cancer modeling and pave the way toward novel therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47e36c2b5c9a5688d815f8fbf4d281fd5d7c15cd" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>167</td>
        </tr>

        <tr id="In the field of digestive system tumor research, spatial transcriptomics technologies are used to delve into the spatial structure and the spatial heterogeneity of tumors and to analyze the tumor microenvironment (TME) and the inter-cellular interactions within it by revealing gene expression in tumors. These technologies are also instrumental in the diagnosis, prognosis, and treatment of digestive system tumors. This review provides a concise introduction to spatial transcriptomics and summarizes recent advances, application prospects, and technical challenges of these technologies in digestive system tumor research. This review also discusses the importance of combining spatial transcriptomics with single-cell RNA sequencing (scRNA-seq), artificial intelligence, and machine learning in digestive system cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbdf975c83e9f742bc7e19f83183283693f91eae" target='_blank'>
              Application of Spatial Transcriptomics in Digestive System Tumors
              </a>
            </td>
          <td>
            Bowen Huang, Yingjia Chen, S. Yuan
          </td>
          <td>2024-12-27</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) exhibits significant intra-tumoral heterogeneity (ITH) at both morphological and genetic levels, complicating treatment and contributing to disease progression1,2. Among these, rhabdoid ccRCCs stand out as highly aggressive tumors distinguished by cells with unique morphological features and poor clinical outcomes3. However, the correlation between distinct morphological phenotypes, specific molecular alterations, and their influence on tumor behavior remains poorly understood. Understanding the link between clinically relevant morphological features and their molecular underpinnings will be critical for developing more effective treatments targeting key cellular subsets responsible for tumor progression. In this study, we integrated advanced AI-based image analysis with single-cell isolation and multi-omics profiling to dissect the link between clinically relevant morphological and molecular features of ccRCC cells. Using a novel digital pathology workflow, we quantified low-grade, high-grade, and rhabdoid morphologies in ccRCC diagnostic images with unprecedented precision. Subsequently, isolation of two sets of 1,000 morphologically distinct cells for detailed mRNA and protein expression analyses, revealed significant increasing dysregulation associating with higher histopathological grades. Rhabdoid ccRCC cells (grade 4) demonstrated distinct molecular profiles, including upregulated FOXM1-driven proliferation, disrupted cell-matrix interactions, and enhanced immune evasion pathways. Despite high T-cell infiltration in rhabdoid areas, we identified a rhabdoid-specific immunosuppressive network driven by cytokines, IFN-beta, and integrin signaling, likely contributing to T-cell exhaustion. These findings provide a basis for novel therapeutic strategies targeting these pathways in combination with immunotherapy to improve outcomes for patients with aggressive rhabdoid ccRCC. Key Points ccRCC is characterized by well-established morphological heterogeneity but the correlation with the underlying molecular aberrations remained elusive. By integrating AI-based image analysis with single cell isolation and deep multi-omics profiling, we dissect the molecular intricacies of ccRCC, from targeted collection of 1,000 morphologically distinct cells. Our results demonstrate significant dysregulation of gene and protein expression correlating with higher histopathological grades in ccRCC. Aggressive ccRCC cells with rhabdoid differentiation (grade 4) display distinct molecular profiles, as they upregulate FOXM1-mediated proliferation, ECM remodeling and the immune evasion responses, suggesting new therapeutic avenues enhancing ICI efficacy in these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7461a1f413e55baf162ee7fd7309ee3cb97ce7a" target='_blank'>
              Deep visual multi-omics profiling reveals mechanisms that underly cancer cell differentiation and aggressiveness in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            H. A. Bolck, Ede Migh, A. Kriston, Natalia Zajac, Susanne Kreutzer, Tiberiu Totu, Peter Leary, Ferenc Kovács, D. Rutishauser, Sybille Pfammatter, Jonas Grossmann, Cassandra Litchfield, Marija Buljan, Niels J Rupp, Péter Horváth, Holger Moch
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Uncontrolled proliferation is a hallmark of cancer, yet tumour cells can enter G0 arrest by halting the cell cycle reversibly (quiescence) or irreversibly (senescence) to survive under stress and in hostile tumour microenvironments (TME). G0 arrested cells contribute to drug tolerance, metastasis and recurrence, but their identification remains challenging due to their rarity and elusive regulatory pathways. Here, we quantify G0 arrest and proliferation decisions in single-cell and spatially profiled breast primary tumours to unveil adaptive responses driving immune compartmentalisation. We identify a genomically-constrained and prolonged G0 arrest state resembling that of dormant precursors of cancer progression. This state featured adaptive transcriptional reprogramming, including unfolded protein response stress, reduced mitochondrial translation and epithelial-mesenchymal plasticity enabled by semaphorin signalling. Spatial transcriptomics revealed G0 arrest pockets encapsulated by APOE+ lipid-associated macrophages, myofibroblastic CAFs and immature perivascular cells, suggestive of an immunosuppressive niche contrasting with cytotoxic environments in proliferative hotspots and displaying distinct drug sensitivities. To facilitate future research, we provide a foundation model capturing G0 arrest and proliferation with 94% accuracy in single cell data, available at https://github.com/secrierlab/G0-LM. Our findings provide new insights into the spatial organisation of cell cycle arrest in breast cancer, highlighting the role of G0 states in tumour heterogeneity and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5d540e388314aece2599568858b5b0ad8004482" target='_blank'>
              Balancing tumour proliferation and sustained cell cycle arrest through proteostasis remodelling drives immune niche compartmentalisation in breast cancer
              </a>
            </td>
          <td>
            Cenk Celik, Eloise Withnell, Shi Pan, Tooki Chu, Johnathan Labbadia, M. Secrier
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Microbes have been found in various tumors and the research of tumor microbiome has been garnering increased attention. However, it remains challenge to investigate the microbiome in cancer at spatial resolution. The rapid advent of spatially resolved transcriptomics techniques has given rise to map transcripts at single-cell resolution in various types of cancer. Here, we constructed a comprehensive spatial meta-transcriptome resource by manually curating 203 fresh frozen (FF) slices from 20 cancers encompassing 334,253 spots and 1,908,646 cells. SMTdb (http://bio-bigdata.hrbmu.edu.cn/SMTdb/) was constructed to provide comprehensive insights into the abundance, distribution and enriched TME regions of 1218 microbiota in spatial tissue slices. SMTdb enables vast interactive data exploration of spatial distribution and expression of microbiota, host gene modules associated with certain microbiota and co-occurrence between microbiota and immune cells within tumor microenvironment. The atlas resource serves as a one-stop and time-effective platform to investigate the interactions among microbial ecosystems and hosts in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803a0dfe5a2e579dc8f160c9acd2466a2dcd43" target='_blank'>
              SMTdb: A comprehensive spatial meta-transcriptome resource in cancer
              </a>
            </td>
          <td>
            Weiwei Zhou, Qingyi Yang, Jiyu Guo, Si Li, Minghai Su, Feng Leng, Tingyu Rong, Jingyi Shi, Yueying Gao, Tiantongfei Jiang, Juan Xu, Yongsheng Li
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="MAPK pathway inhibitors (MAPKi) are increasingly used in the treatment of advanced colorectal cancer, but often produce short-lived responses in patients. Although acquired resistance by de novo mutations in tumors have been found to reduce response in some patients, additional mechanisms underlying the limited response durability of MAPK targeting therapy remain unknown. Here, we denote new contributory tumor biology and provide insight on the impact of tumor plasticity on therapy response. Analysis of MAPKi treated patients revealed activation of stemness programs and increased ASCL2 expression, which are associated with poor outcomes. Greater ASCL2 with MAPKi treatment was also seen in patient-derived CRC models, independent of driver mutations. We find ASCL2 denotes a distinct cell population, arising from phenotypic plasticity, with a proliferative, stem-like phenotype, and decreased sensitivity to MAPKi therapy, which were named adaptive plasticity tumor (APT) cells. MAPK pathway suppression induces the APT phenotype in cells, resulting in APT cell enrichment in tumors and limiting therapy response in preclinical and clinical data. APT cell depletion improved MAPKi treatment efficacy and extended MAPKi response durability in mice. These findings uncover a cellular program that mitigates the impact of MAPKi therapies and highlights the importance of addressing tumor plasticity to improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f697113c6b546482f0f42def83352f20ea919c" target='_blank'>
              Adaptive Plasticity Tumor Cells Modulate MAPK-Targeting Therapy Response in Colorectal Cancer
              </a>
            </td>
          <td>
            O. Villarreal, Yixin Xu, Ha Tran, Annette A. Machado, Dionne Prescod, Amanda Anderson, R. Minelli, Mike Peoples, Alejandro Hernandez Martinez, H. Lee, Chi Wut Wong, Natalie W Fowlkes, Preeti Kanikarla, Alexey Sorokin, J. Alshenaifi, Oluwadara Coker, Kangyu Lin, Chris Bristow, A. Viale, J. P. Shen, C. Parseghian, Joseph R Marszalek, Ryan Corcoran, S. Kopetz
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Glioma is an aggressive brain tumor with a poor prognosis. Establishing an in vitro culture model that closely replicates the cellular composition and microenvironment of the original tumor has been challenging, limiting its clinical applications. Here, we present a novel approach to generate glioma organoids with a microenvironment (GlioME) from patient-derived glioma tissue. These organoids maintain the genetic and epigenetic characteristics of the primary tumor and preserve cell-to-cell interactions within the tumor microenvironment, including resident immune cells. Bulk RNA sequencing, whole exome sequencing, and DNA methylation analysis were used to confirm the molecular similarities between the organoids and primary glioma tissues. Immunofluorescence and flow cytometry were used to assess immune cell viability, comparing GlioME with floating glioma organoids. GlioME exhibited high responsiveness to chemotherapy and targeted therapy, demonstrating its potential for therapeutic screening applications. Notably, GlioME accurately predicted patient response to the recently approved MET inhibitor, vebreltinib. Thus, this organoid model provides a reliable in vitro platform for glioma microenvironment-related research and clinical drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076fe0fc7a2a2145464370151771e2e7246f87bc" target='_blank'>
              GlioME: A Novel Organoid Model Retaining the Glioma Microenvironment for Personalized Drug Screening and Therapeutic Evaluation
              </a>
            </td>
          <td>
            Chengjun Zheng, Qiaodong Chen, Peng Wang, Delong Zhang, Zheng Fang, Yutong Feng, Jie Chen, Jiahong Chen, Yiwen Fu, Bao Yang, Qian Zhao, Fei Sun, Ying Zhang, Tao Jiang, Lemin Zheng, Zhaoshi Bao
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell omics analysis has unveiled the heterogeneity of various cell types within tumors. However, no methodology currently reveals how this heterogeneity influences cancer patient survival at single-cell resolution. Here, we introduce scSurv, combining a Cox proportional hazards model with a deep generative model of single-cell transcriptome, to estimate individual cellular contributions to clinical outcomes. The accuracy of scSurv was validated using both simulated and real datasets. This method identifies cells associated with favorable or adverse prognoses and extracts genes correlated with their contribution levels. In melanoma, scSurv reproduces known prognostic macrophage classifications and facilitates hazard mapping through spatial transcriptomics in renal cell carcinoma. We also identified genes consistently associated with prognosis across multiple cancers and demonstrated the applicability of this method to infectious diseases. scSurv is a novel framework for quantifying the heterogeneity of individual cellular effects on clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5139bf1027dffb0022f2f770e24762808926adf5" target='_blank'>
              scSurv: a deep generative model for single-cell survival analysis
              </a>
            </td>
          <td>
            Chikara Mizukoshi, Yasuhiro Kojima, Shuto Hayashi, Ko Abe, Daisuke Kasugai, Teppei Shimamura
          </td>
          <td>2024-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b78a9a8e3280a5c16b1080da5f7e2a241cd45990" target='_blank'>
              Diffusion Smart-seq3 of breast cancer spheroids to explore spatial tumor biology and test evolutionary principles of tumor heterogeneity
              </a>
            </td>
          <td>
            Antony Cougnoux, Loay Mahmoud, Per A Johnsson, Alper Eroglu, Louise Gsell, Jakob Rosenbauer, R. Sandberg, Jean Hausser
          </td>
          <td>2025-01-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d97774290c1f515233beb8849c220d76ed9e57" target='_blank'>
              Unsupervised clustering reveals noncanonical myeloid cell subsets in the brain tumor microenvironment
              </a>
            </td>
          <td>
            Ismaïl Hermelo, Tuomo Virtanen, I. Salonen, R. Nätkin, Sofia Keitaanniemi, A. Tiihonen, Suvi Lehtipuro, Laura Kummola, Ella Raulamo, K. Nordfors, H. Haapasalo, M. Rauhala, J. Kesseli, M. Nykter, J. Haapasalo, K. Rautajoki
          </td>
          <td>2025-01-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Hepatic arterial infusion chemotherapy (HAIC) has emerged as a promising treatment strategy for hepatocellular carcinoma (HCC), but a detailed understanding of the multicellular ecosystem after HAIC treatment is lacking. Here, we collected tumor samples from treatment‐naïve primary and post‐HAIC HCC, and integrated single‐nucleus RNA sequencing with spatial transcriptomics to characterize the tumor ecosystem in the post‐HAIC HCC. Increased fractions and enhanced cellular communication of CD4+ T, CD20+ B, and dendritic cell subtypes were identified in post‐HAIC tumors. Moreover, it is substantiated that HAIC promoted tertiary lymphoid structures (TLS) formation, and addressed the roles of TLSs as spatial niches of cellular communication. Specifically, intermediate exhausted CD8+ T cells expressing Granzyme‐K and PD‐1 (PD‐1+CD8+ Tex‐int) expanded following HAIC and exhibited a functionally antitumor phenotype. PD‐1+CD8+ Tex‐int accumulated in the TLS vicinity and disseminated throughout the tumor microenvironment, demonstrating potential as an effective biomarker for HAIC‐based treatment in HCC. This study provides valuable resources and biological insights in the cellular underpinnings of HAIC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62624d4b04f46d42e65710fe0c98f6d6afaa46d" target='_blank'>
              Single‐Nucleus and Spatial Transcriptome Profiling Delineates the Multicellular Ecosystem in Hepatocellular Carcinoma After Hepatic Arterial Infusion Chemotherapy
              </a>
            </td>
          <td>
            YeXing Huang, Zefeng Du, Zhicheng Lai, DongSheng Wen, Lichang Huang, Minke He, ZiChao Wu, Huifang Li, Hanyue Ouyang, Wenchao Wu, Anna Kan, Ming Shi
          </td>
          <td>2024-12-16</td>
          <td>Advanced Science</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Upper aerodigestive squamous cell carcinoma (UASCC) is an aggressive and lethal neoplasm, with its early neoplastic transformation mechanisms remaining poorly understood. Here, we characterize over 25 genetically-defined organoid models derived from murine and human oral/esophageal tissues harboring key driver mutations. Double knockout of TP53 and CDKN2A induced morphological dysplasia, hyperproliferation, loss of squamous differentiation, and tumorigenicity, which were further exacerbated by additional driver mutations (e.g., PIK3CA, NOTCH1, KMT2C). Single-cell analysis revealed an expansion of quiescent basal cells and proliferative squamous cells, alongside a loss of differentiated squamous cells during malignant transformation. A distinct senescence program, regulated by ANXA1, was markedly diminished during early neoplastic evolution. Mechanistically, the ANXA1-SMAD3-p27KIP1 pathway was identified as a critical regulator of this senescence program, acting to suppress neoplastic features in organoid models. Lastly, our high-throughput, single-organoid-resolution drug screens unexpectedly revealed PIK3CA-driven organoids exhibited sensitivity to Mitomycin C and Onalespib. This study provides novel mechanistic insights into early neoplastic evolution and underscores the value of genetically-defined organoid models for investigating cancer biology and identifying targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7127b7b7361c88e16950c954ef2bf66d966c5299" target='_blank'>
              Genetically Defined Organoid Models Reveal Mechanisms Driving Squamous Cell Neoplastic Evolution and Identify Potential Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Hua Zhao, Young Min Park, Yueyuan Zheng, Qiong Mao, Casey Collet, Boyan Hu, Tianming Zhou, Luda Lin, Stephanie Wong, Yuhao Pan, Anette Vistoro Monreal, Uttam Sinha, Parish Sedghizadeh, A. Soragni, De-Chen Lin
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary Clear cell renal cell carcinoma (ccRCC) is the most common type of kidney cancer, but predicting its behavior remains challenging using standard histopathologic examination. This study introduces a novel approach to predict ccRCC aggressiveness by analyzing the spatial distribution of tumor cells in H&E-stained images. The researchers found that spatial analysis outperformed traditional tumor grading in predicting metastasis, particularly for intermediate-grade tumors. They identified two distinct patient groups based on spatial characteristics, with one group showing greater spatial randomness and a higher association with metastasis. Furthermore, the study revealed a gene expression signature related to the extracellular matrix (matrisome) that correlated with the spatial patterns and aggressive tumor behavior. These findings suggest that analyzing the spatial distribution of ccRCC tumor cells could provide valuable insights into tumor behavior and metastatic potential, potentially improving prognostication and personalized treatment strategies for patients with ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b1af154f3c69daea89b74218f36407c18e3de7d" target='_blank'>
              Spatial Distribution of Tumor Cells in Clear Cell Renal Cell Carcinoma Is Associated with Metastasis and a Matrisome Gene Expression Signature
              </a>
            </td>
          <td>
            Prahlad Bhat, P. Tamboli, K. Sircar, Kasthuri Kannan
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90bace8a9791e95b3eff7c16fd892d053802f89b" target='_blank'>
              Single-cell transcriptomes of dissecting the intra-tumoral heterogeneity of breast cancer microenvironment
              </a>
            </td>
          <td>
            Peixian Chen, Kaifeng Liang, Xiaofan Mao, Qiuyuan Wu, Zhiyan Chen, Yabin Jin, Kairong Lin, Tiancheng He, Shuqing Yang, Huiqi Huang, Guolin Ye, Juntao Gao, Dan Zhou, Zhihao Zeng
          </td>
          <td>2024-12-26</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Dysregulation of enhancer-promoter communication in the context of the three-dimensional (3D) nucleus is increasingly recognized as a potential driver of oncogenic programs. Here, we profiled the 3D enhancer-promoter networks of primary patient-derived glioblastoma stem cells (GSCs) in comparison with neuronal stem cells (NSCs) to identify potential central nodes and vulnerabilities in the regulatory logic of this devastating cancer. Specifically, we focused on hyperconnected 3D regulatory hubs and demonstrated that hub-interacting genes exhibit high and coordinated expression at the single-cell level and strong association with oncogenic programs that distinguish IDH-wt glioblastoma patients from low-grade glioma. Epigenetic silencing of a recurrent 3D enhancer hub—with an uncharacterized role in glioblastoma—was sufficient to cause concordant downregulation of multiple hub-connected genes along with significant shifts in transcriptional states and reduced clonogenicity. By integrating published datasets from other cancer types, we also identified both universal and cancer type-specific 3D regulatory hubs which enrich for varying oncogenic programs and nominate specific factors associated with worse outcomes. Genetic alterations, such as focal duplications, could explain only a small fraction of the detected hyperconnected hubs and their increased activity. Overall, our study provides computational and experimental support for the potential central role of 3D regulatory hubs in controlling oncogenic programs and properties. HIGHLIGHTS - 3D regulatory “hubs” in glioblastoma enrich for highly coregulated genes at a single-cell level and expand oncogenic regulatory networks. - Targeted perturbation of a highly recurrent 3D regulatory hub in GSCs results in altered transcriptional states and cellular properties. - 3D regulatory hubs across cancer types associate with tumor-specific and universal oncogenic programs and worse outcomes. - The majority of hyperconnected hubs do not overlap with structural variants, suggesting epigenetic mechanisms. eTOC Here we profile the 3D enhancer connectomes of primary patient-derived human glioblastoma stem cells (GSCs), identify hyperconnected 3D regulatory “hubs”, and examine the impact of 3D hub perturbation on the transcriptional program and oncogenic properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef46a4bf29419cbc8b345c6cf8f3bc9f4287626" target='_blank'>
              Three-dimensional regulatory hubs support oncogenic programs in glioblastoma
              </a>
            </td>
          <td>
            Sarah L. Breves, Dafne Campigli Di Giammartino, James Nicholson, S. Cirigliano, Syed Raza Mahmood, UkJin Lee, Alexander Martínez-Fundichely, Johannes Jungverdorben, R. Singhania, Sandy Rajkumar, Raphael Kirou, L. Studer, Ekta Khurana, A. Polyzos, Howard A. Fine, E. Apostolou
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Fate-restricted cells can acquire stem cell-like properties through dedifferentiation, enabling them to gain the plasticity required for differentiation into multiple lineages. Tumour plasticity is prominently observed in brain cancers, where transient cell state changes are linked to resistance to conventional therapies. In this study, we demonstrate that a sub-population of dedifferentiated tumour neural stem cells (NSCs) in Drosophila, induced by the knockdown of prospero (pros), can generate its own glial niche. Temporal patterning, known to influence oncogenic competence and tumour malignancy, plays a key role in this process. Specifically, we show that de novo gliogenesis occurs in the more differentiated Syncrip+ (Syp+) NSC population. Modulating Syp levels alters the size of the glial niche, subsequently affecting tumour size. Furthermore, the tumour-associated glial niche expands through cell division and fails to cease proliferation on time due to dysregulated ecdysone signalling, contributing to niche expansion. Our findings reveal that tumours arising via dedifferentiation establish their own supportive glial microenvironment, which sustains tumour growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d30097dbfea218ae747551091d256416f71d4db" target='_blank'>
              Tumour-derived gliogenesis sustains dedifferentiation-dependent tumour growth in the Drosophila CNS
              </a>
            </td>
          <td>
            Edel Alvarez-Ochoa, Qian Dong, Hannah Truong, Louise Y Cheng
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Brain metastases (BrMets), common for advanced-stage breast cancer patients, are associated with poor median survival and accompanied by severe neurologic decline. Halting the progression of breast cancer brain metastases (BCBMs) may require modulation of the tumor microenvironment (TME), yet little is known about the impact of the primary breast TME on brain tropism, or how, once there, metastatic breast cancer cells coexist with brain-resident cells (e.g., neurons and glia). Traditionally, studies in this space have focused on differential expression analysis, overlooking potential insights gained from investigating genes with equivalent expression between groups. This is particularly crucial in distant metastasis, where tumor cells may co-opt the transcriptional programs of the host organ (e.g., brain) to facilitate successful seeding and outgrowth. Prior to our work, no computational framework existed to determine biologically-relevant equivalent gene expression. To resolve molecular mechanisms of BCBM enabled by metastatic cancer cells and/or resident brain cells, we leveraged Nanostring GeoMx to perform spatially-resolved transcriptomic profiling on 235 patient-derived tissue cores from BCBM (including adjacent normal brain), primary invasive breast cancers, and normal (non-cancer) brain; analyzing 18,677 RNAs in 450 areas of interest (AOIs). We introduce the “Equivalent Expression Index” a highly specific and accurate algorithm that identifies statistically significant “Equivalently-Expressed Genes”. This method facilitated the identification of molecular remodeling and mimicry genes within tissue-specific TMEs. By integrating differential expression analysis with the Equivalent Expression Index, we discovered multiple novel gene signatures associated with BCBM and primary tumor brain-metastatic potential. We demonstrate that the Equivalent Expression Index is a powerful tool to uncover shared gene expression programs representing the adaptation of metastatic cells and brain-resident cells to the BCBM microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/015422dee0790a994280a3f1f594af35182af70e" target='_blank'>
              Spatial Profiling Reveals Equivalence-Derived Molecular Signatures of Brain Mimicry and Adaptation in Breast Cancer Brain Metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, C. Yeh, Bryanna Godfrey, Pablo Nunez Perez, Giuseppe Barisano, Sushama Varma, Saman Ahmadian, A. Toland, M. Granucci, Thy T. H. Trinh, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia K Plevritis, M. Gephart
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The kidney is a complex organ requiring tightly coordinated interactions between epithelial, endothelial, and mesenchymal cells during development. Congenital kidney defects can result in kidney disease and renal failure, highlighting the importance of understanding kidney formation mechanisms. Advances in RNA sequencing have revealed remarkable cellular heterogeneity, especially in the kidney stroma, though relationships between stromal, epithelial, and endothelial cells remain unclear. This study presents a comprehensive gene expression atlas of embryonic and postnatal kidneys, integrating single-nucleus and in situ RNA sequencing data. We developed the Kidney Spatial Transcriptome Analysis Tool (KSTAT), enabling researchers to identify cell locations, predict cell-cell communication, and map gene pathway activity. KSTAT revealed significant heterogeneity among embryonic kidney pericytes, providing a critical resource for hypothesis generation and advancing knowledge of kidney development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df4312b6dec5fab934d14ad6ad0cc39d8a07d71" target='_blank'>
              Integration of spatial and single nucleus transcriptomics to map gene expression in the developing mouse kidney
              </a>
            </td>
          <td>
            Christopher P. Chaney, Alexandria N. Fusco, Elyse D. Grilli, Jane N. Warshaw, Peter M. Luo, Ondine Cleaver, Denise K. Marciano, Thomas J. Carroll
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The renal vasculature consists of highly specialized blood vessels that carry out distinct physiological functions. Defining their transcriptional signatures and tracing their developmental ontogeny has thus far been challenging due to a lack of regionally specific endothelial biomarkers. Here, we performed single nuclear RNA sequencing (snucRNA-Seq) to interrogate embryonic renal endothelial cells. We identified nine different endothelial cell types, and validated expression of novel candidates using multiplex RNAScope. We interrogate biological characteristics, including cell cycle stage, of developmental blood vessels with a variety of in vivo tools. Using genetic tools based on expression of tip cell marker Esm1, we performed detailed lineage tracing, demonstrating multi-clonal and multi-directional endothelial contribution to glomerular vasculature. Together, this study provides the first validated, tool-focused developmental atlas of the murine renal vasculature and elucidates several novel cellular mechanisms of nephron vascularization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e404560bedd479bc032952e4bd3bd630933dfdc5" target='_blank'>
              Zonal endothelial cell heterogeneity underlies murine renal vascular development
              </a>
            </td>
          <td>
            Peter M. Luo, Neha Ahuja, Christopher P. Chaney, Danielle Pi, Aleksandra Cwiek, Zaneta Markowska, C. Hiremath, Denise Marciano, Karen K. Hirschi, M. L. Iruela-Arispe, Thomas J. Carroll, Ondine Cleaver
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCCs) constitute one of the few cancer indications for which mortality rates continue to rise. While Notch signaling dictates a key progenitor lineage choice during development, its role in HCC has remained controversial. Using therapeutic antibodies targeting Notch ligands and receptors to screen over 40 patient-derived xenograft models, we here identify progenitor-like HCCs that crucially depend on a tumor-intrinsic JAG1-NOTCH2 signal. Inhibiting this signal induces tumor regressions by triggering progenitor-to-hepatocyte differentiation, the same cell fate-switch that Notch controls during development. Transcriptomic analysis places the responsive tumors within the well-characterized progenitor subclass, a poor prognostic group of highly proliferative tumors, providing a diagnostic method to enrich for Notch-dependent HCCs. Furthermore, single-cell RNA sequencing uncovers a heterogeneous population of tumor cells and reveals how Notch inhibition shifts cells from a mixed cholangiocyte-hepatocyte lineage to one resembling mature hepatocytes. Analyzing the underlying transcriptional programs brings molecular detail to this process by showing that Notch inhibition de-represses expression of CEBPA, which enables the activity of HNF4α, a hepatocyte lineage factor that is otherwise quiescent. We thus describe a compelling and targetable dependency in a poor-prognosis class of HCCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3764bfcf8bf124425bfb62c588b7e271cb0186df" target='_blank'>
              Notch signaling maintains a progenitor-like subclass of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kerstin Seidel, Robert Piskol, Thi Thu Thao Nguyen, Amy Shelton, Charisa L. Cottonham, Cecile C. de la Cruz, Joseph Castillo, Jesse Garcia, Udi Segal, Mark Merchant, Yeqing Angela Yang, Jasmine Chen, Musa Ahmed, Alexis Scherl, Rajesh Vij, Lluc Mosteiro, Yan Wu, Z. Modrusan, C. Metcalfe, Christopher W. Siebel
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Among the extensive genomic alterations in prostate cancer, the genomic deletion of PTEN stands out as one of the most consistently observed and confirmed alteration. PTEN loss in prostate tumors is primarily associated with cancer cell proliferation and survival through the activation of the PI3K-AKT-mTOR signalling pathway. However, its use as a robust biomarker in the clinical practice is hampered by the complex epigenetic, transcriptional and post-translational regulation. Here, we undertook an approach that combined in situ assessment of PTEN protein with transcriptional surrogates of its activity to gain insights into the downstream functional effects of PTEN loss in primary tumors. Our extensive bioinformatic analyses, including the integration with single-cell RNA-Seq approaches in a new clinical cohort, highlighted stroma remodeling as the major cancer cell-extrinsic process associated with PTEN loss. By applying similar computational strategy on the transcriptomic data generated from primary prostate tumors of genetically engineered Pten knock-out mouse models, we validated the causal role of Pten in the stromal reaction observed in clinical specimens. Mechanistically, we provide evidence for the activation of a paracrine program that encompasses enhanced TGF-β signalling and that is compatible with the secretome of PTEN-deficient senescent cancer cells. Our study provides relevant biological context to the cellular and molecular alterations unleashed upon PTEN protein loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2b98ad80d259dc7a130daab0e45ee7b8d7d990" target='_blank'>
              Transcriptional network analysis of PTEN protein-deficient prostate tumors reveals robust stromal reprogramming and signs of senescent paracrine communication
              </a>
            </td>
          <td>
            Ivana Rondon-Lorefice, Jose I. Lopez, Aitziber Ugalde-Olano, Maite Zufiaurre, I. Astobiza, Saioa Garcia-Longarte, Amaia Zabala, Sofía Rey, Aida Santos-Martín, Miguel Unda, Ana Loizaga-Iriarte, M. Graupera, Paolo Nuciforo, A. Carracedo, Isabel Mendizabal
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The interface between the brain surface and the adjacent meninges is a selective barrier regulating fluid, protein and immune cell exchange between the CNS and periphery. However, the cell types that form this important interface are not yet fully defined. To address this limitation, we have used single cell RNA-sequencing (scRNA-seq) and single cell spatial transcriptomics together with morphological lineage tracing and immunostaining to analyze the adult murine cortex. We show that the cortical interface is comprised of three major cell types, leptomeningeal cells, border astrocytes and tissue-resident macrophages. On the peripheral side the interface is comprised of transcriptionally-distinct PDGFRα-positive leptomeningeal mesenchymal cells that are intermingled with macrophages. This leptomeningeal pial layer is lined by a population of transcriptionally-distinct border astrocytes. The interface neighborhood is rich in growth factor mRNAs, including many leptomeningeal ligands predicted to act on both the border astrocytes and macrophages. On the CNS side of the interface is the relatively cell-sparse cortical layer one containing interneurons, microglia, parenchymal astrocytes, oligodendrocyte precursor cells and oligodendrocytes. Except for the border astrocytes, layer one cells are not closely-associated with the interface, suggesting that secreted ligands may be the major way the brain interface communicates with the underlying cortical parenchyma. Thus, our data provide a molecular/cellular resource describing the brain interface cell types and their interactions, thereby enabling future studies asking how this distinct cellular compartment regulates CNS:periphery interactions. Significance Statement Recent years have seen significant progress in identifying the diverse cell types within the meningeal space. However, the mechanisms by which these cells interact with glial and neuronal cells in layer one of the adult murine cortex remain poorly understood. During development, communication between radial precursors and meningeal layers is crucial for proper brain formation, but the role of this interaction in adulthood is still unclear. Additionally, how resident immune cells in the leptomeningeal space signal to layer one cortical cells or meningeal mesenchymal cells during homeostasis remains an open question. Understanding the identity, location, and interactions of these cells is essential for unraveling the complex dynamics at this critical brain interface.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f262d488faaf9a1e5d2dd142c2fdfc378e1c41e" target='_blank'>
              Single cell approaches define the murine leptomeninges:cortical brain interface as a distinct cellular neighborhood comprised of neural and nonneural cell types
              </a>
            </td>
          <td>
            Sarah N. Ebert, Christine Eisner, Konstantina Karamboulas, L. Bernier, David R. Kaplan, Brian Macvicar, F. Miller
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The existence of cancer stem cells (CSCs) in various tumors has become increasingly clear in addition to their prominent role in therapy resistance, metastasis, and recurrence. For early diagnosis, disease progression monitoring, and targeting, there is a high demand for clinical-grade methods for quantitative measurement of CSCs from patient samples. Despite years of active research, standard measurement of CSCs has not yet reached clinical settings, especially in the case of solid tumors. This is because detecting this plastic heterogeneous population of cells is not straightforward. This review summarizes various techniques, highlighting their benefits and limitations in detecting CSCs from patient samples. In addition, methods designed to detect CSCs based on secreted and niche-associated signaling factors are reviewed. Spatial and single-cell methods for analyzing patient tumor tissues and noninvasive techniques such as liquid biopsy and in vivo imaging are discussed. Additionally, methods recently established in laboratories, preclinical studies, and clinical assays are covered. Finally, we discuss the characteristics of an ideal method as we look toward the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a142badfcf7c429680b2b941eddecbf41e3d46a6" target='_blank'>
              Detection of Cancer Stem Cells from Patient Samples
              </a>
            </td>
          <td>
            Sofia Hakala, Anna Hämäläinen, Sanne Sandelin, Nikolaos Giannareas, Elisa Närvä
          </td>
          <td>2025-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cellular senescence features a durable exit from the cell cycle triggered by stress or carcinogens. The INK4 locus is inactivated in various cancers, yet in senescence, p16Ink4a is activated. Whether senescence is tumor-suppressing or -promoting remains a conundrum. We discovered an evolutionally-conserved Vertebrata INK4-homolog. This ink4ab triggers senescence upon oxidative- and/or carcinogenic-stress. Adult Ink4ab-deficient animals failed to activate senescence and developed spontaneous cancers. Combined Ink4ab and Tp53 deficiency revealed a reciprocal senescence and apoptosis regulation, controlling tumorigenesis, including retinoblastoma. INK4-hematopoietic-deficient mice exhibited p19Arf-dependent enhanced senescence-like phenotypes, uncontrolled cell proliferation, defective stem cell differentiation, and splenomegaly, with single-splenocytes spatially-enriched in senescence-associated secretory profiles. Our studies reveal the evolutionary origin of paradigms co-regulating senescence and tumor suppression and offer strategies to exploit these reciprocal pathways for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9243e085293669a418053332a11c9df19226137a" target='_blank'>
              Evolutionary conserved reciprocal senescence and tumor suppressor signals limit lifetime cancer
              </a>
            </td>
          <td>
            Liqiong Liu, Stephani Davis, Shamila Yusuff, Ann Strange, Yonghua Zhuang, Prasanna Vaddi, Kathleen Flaherty, Kelly Jara, Joseph Kramer, Christine Archer, Schuyler Lee, Bifeng Gao, A. van Bokhoven, Wei Wang, Sharon R. Pine, Tetsuya Nakamura, Hatim E. Sabaawy
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Hyperactivation of WNT signaling is a well-established hallmark of cancer. Various epithelial cancers express high levels of WNT7B and WNT10A that are not commonly expressed during tissue homeostasis, but rather associate with tissue development and regeneration. Although increased WNT7B/10A expression correlates with aggressive disease and lower patient survival rates, the mechanism by which these WNTs influence cancer progression remains unknown. Here, we use patient-derived organoids to show that tumor-intrinsic expression of WNT7B/10A drives survival and growth of advanced pancreatic ductal adenocarcinoma (PDAC). Bulk and single-cell profiling reveal that WNT7B drives proliferation and promotes expression of a poor prognosis basal-like state by preventing expression of a more differentiated, classical PDAC signature. By generating WNT7B reporter organoids, we show that heterogeneously distributed WNT-high PDAC cells are shifted towards a more basal-like phenotype and stably co-exist with WNT-low/negative lineages. Furthermore, hybrid co-cultures of WNT7B-proficient and -knockout PDAC organoids demonstrate that WNT-sending cells drive survival and proliferation of neighboring WNT-negative cells within the cancer epithelium via short range, cell contact-dependent signaling. In summary, our work uncovers a prominent role of WNT7B/10A in driving PDAC subtype heterogeneity and argues that WNT inhibition may be applied to force a ‘class switch’ to a more differentiated, less aggressive cancer subtype that correlates with improved therapeutical response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dedf8f2a45b81078975062b7798dffcff4606bf1" target='_blank'>
              WNT7B drives a program for pancreatic cancer subtype switching and progression
              </a>
            </td>
          <td>
            J. Sprangers, J. Bugter, D. Xanthakis, M. Boekhout, R. Kok, L. Brosens, F. Dijk, M.J. Rodríguez Colman, J. van der Vaart, M. Maurice
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="During tumorigenesis, the extracellular matrix (ECM), which constitutes the structural scaffold of tissues, is profoundly remodeled. While the impact of such remodeling on tumor growth and invasion has been extensively investigated, much less is known on the consequences of ECM remodeling on tumor infiltration by immune cells. By combining tissue imaging and machine-learning, we here show that the localization of T lymphocytes and neutrophils, which orchestrate antitumor immune responses, can be predicted by defined topographical features of fibrillar collagen networks. We further show that these collagen topographies result from the activation of a fibrotic pathway controlled by the transcription factor Tcf4 upon depletion of tumor-associated macrophages at late tumor stages. This pathway promotes the deposition of collagen 3 by both tumor and stromal cells, resulting in intermingled collagen networks that favor intra-tumoral T cell and neutrophil localization. Importantly, analysis of human colorectal cancer public bulk RNAseq databases showed a strong correlation between Tcf4 and collagen 3, as well as between the expression of these genes and tumor infiltration by T lymphocytes and neutrophils, attesting the clinical relevance of our findings. This study highlights the key structural role of macrophages on the tumor extracellular matrix and identifies collagen network topographies as a major regulator of tumor infiltration by immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5894e6b84c18eb6409dfa2fcb90416bb17621a9" target='_blank'>
              Macrophages restrict tumor permissiveness to immune infiltration by controlling local collagen topography through a Tcf4-Collagen3 fibrotic axis
              </a>
            </td>
          <td>
            Zoé Fusilier, Franck Simon, Isabel Calvente, Lou Crestey, Alexandra Clément, Mathilde Mathieu, Roude Jean-Marie, Florence Piastra-Facon, Jeyani George Clément, Enola Lumineau, Mattia Tonani, Valeria Manriquez, Livia Lacerda, Perrine de Villemagne, Eliane Piaggio, Vincent Semetey, Sylvie Coscoy, Emanuele Martini, G. Scita, Jean-Christophe Gelly, Johanna Ivaska, Hervé Isambert, C. Goudot, Paolo Pierobon, A. Lennon-Duménil, Hélène D. Moreau
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Genetic studies of haematological cancers have pointed out the heterogeneity of leukaemia in its different subpopulations, with distinct mutations and characteristics, impacting the treatment response. Next-generation sequencing (NGS) and genome-wide analyses, as well as single-cell technologies, have offered unprecedented insights into the clonal heterogeneity within the same tumour. A key component of this heterogeneity that remains unexplored is the intracellular metabolome, a dynamic network that determines cell functions, signalling, epigenome regulation, immunity and inflammation. Understanding the metabolic diversities among cancer cells and their surrounding environments is therefore essential in unravelling the complexities of leukaemia and improving therapeutic strategies. Here, we describe the currently available methodologies and approaches to addressing the dynamic heterogeneity of leukaemia progression. In the second section, we focus on metabolic leukaemic vulnerabilities in acute myeloid leukaemia (AML) and acute lymphoblastic leukaemia (ALL). Lastly, we provide a comprehensive overview of the most interesting clinical trials designed to target these metabolic dependencies, highlighting their potential to advance therapeutic strategies in leukaemia treatment. The integration of multi-omics data for cancer identification with the metabolic states of tumour cells will enable a comprehensive “micro-to-macro” approach for the refinement of clinical practices and delivery of personalised therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0cc7b64900de2dfae5390d10959b2a1e62e8bf1" target='_blank'>
              Unlocking the Heterogeneity in Acute Leukaemia: Dissection of Clonal Architecture and Metabolic Properties for Clinical Interventions
              </a>
            </td>
          <td>
            Martina Maria Capelletti, Orsola Montini, Emilio Ruini, Sarah Tettamanti, Angela Maria Savino, J. Sarno
          </td>
          <td>2024-12-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Computerized chest tomography (CT)-guided screening in populations at risk for lung cancer has increased the detection of preinvasive subsolid nodules, which progress to solid invasive adenocarcinoma. Despite the clinical significance, there is a lack of effective therapies for intercepting the progression of preinvasive to invasive adenocarcinoma. To uncover determinants of early disease emergence and progression, we used integrated single-cell approaches, including scRNA-seq, multiplexed imaging mass cytometry and spatial transcriptomics, to construct the first high-resolution map of the composition, lineage/functional states, developmental trajectories and multicellular crosstalk networks from microdissected non-solid (preinvasive) and solid compartments (invasive) of individual part-solid nodules. We found that early disease initiation and subsequent progression are associated with the evolution of immune-suppressive cellular phenotypes characterized by decreased cytotoxic CD8 T and NK cells, increased T cell exhaustion and accumulation of immunosuppressive regulatory T cells (Tregs) and M2-like macrophages expressing TREM2. Within Tregs, we identified a unique population of 4-1BB+ Treg subset enriched for the IL2-STAT5 suppressive pathway with transcription profiles supporting discrete metabolic alterations. Spatial analysis showed increased density of suppressive immune cells around tumor cells, increased exhaustion phenotype of both CD4 and CD8 T cells expressing chemokine CXCL13, and spatial micro-complex of endothelial and lymphocyte interactions within tertiary lymphoid structures. The single-cell architecture identifies determinants of early disease emergence and progression, which may be developed not only as diagnostic/prognostic biomarkers but also as targets for disease interception. Additionally, our dataset constitutes a valuable resource for the preinvasive lung cancer research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f08ee8028223311574c20ea0561992916e40482b" target='_blank'>
              Acquisition of discrete immune suppressive barriers contributes to the initiation and progression of preinvasive to invasive human lung cancer
              </a>
            </td>
          <td>
            L. Yoffe, B. Bhinder, Sung Wook Kang, Haoran Zhang, Arshdeep Singh, H. Ravichandran, G. Markowitz, Mitchell T. Martin, Junbum Kim, Chen Zhang, O. Elemento, Wesley Tansey, Stewart Bates, T. McGraw, Alain Borczuk, Hyun-Sung Lee, Nasser K Altorki, Vivek Mittal
          </td>
          <td>2025-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Tissue regeneration relies on the ability of cells to undergo de novo patterning. While tissue patterning has been viewed as the transition from initially identical un-patterned cells to an arrangement of different cell types, recent evidence suggests that initial heterogeneities between cells modulate tissue-scale pattern formation. Yet, how such heterogeneities arise and, thereafter, regulate cell type emergence in a population of cells is poorly understood. Using in vivo and in vitro mouse regenerative systems, we identify a critical tissue density that is required to induce heterogeneous inactivation of the mechanosensor YAP1. Experimental and biophysical approaches demonstrate that YAP1 cell-to-cell heterogeneity pre-patterns the first cell fate decision, via both chromatin remodelling and a supracellular feedback between FOXA1 and Delta-Notch signalling. This feedback motif induces cell fate bistability endowing memory to the system and the maintenance of patterns during homeostasis. These findings reveal a generalisable framework in which transient cell-to-cell heterogeneity, regulated by tissue-scale properties, serves as a critical control parameter for the emergence of cell fate and stable patterning during regeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4e71565210ea1e9031332a79fd8f2875c11cbbd" target='_blank'>
              Cell heterogeneity and fate bistability drive tissue patterning during intestinal regeneration
              </a>
            </td>
          <td>
            C. Schwayer, S. Barbiero, D. B. Brückner, C. Baader, N. A. Repina, O. E. Diaz, L. C. Meylan, V. Kalck, S. Suppinger, Q. Yang, J. Schnabl, U. Kilik, J. G. Camp, B. Stockinger, M. Bühler, M. B. Stadler, É. Hannezo, P. Liberali
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Breast cancer (BC) is the second leading cause of cancer-related death among women in the U.S. Organoid models of solid tumors have been shown to faithfully recapitulate aspects of cancer progression such as proliferation and invasion. Although patient-derived organoids (PDOs) and patient-derived xenograft organoids (PDXOs) are pathophysiologically relevant, they are costly to propagate, difficult to manipulate and comprised primarily of the most proliferative cell types within the tumor microenvironment (TME). These limitations prevent their use for elucidating cellular mechanisms of disease progression that depend upon tumor-associated stromal cells which are found within the TME and known to contribute to metastasis and therapy resistance. Here, we report on methods for cultivating epithelial-stromal multicellular 3D cultures. Advantages of these methods include a cost-effective system for rapidly generating organoid-like 3D cultures within scaffold-free environments that can be used to track invasion at single-cell resolution within hydrogel scaffolds. Specifically, we demonstrate how to generate these hetero-multicellular 3D cultures using BT-474 breast cancer cells in combination with fibroblasts (BJ-5ta), monocyte-like cells(THP-1) and/or endothelial cells (EA.hy926). Additionally, differential fluorescent labeling of cell populations enables time-lapse microscopy to define 3D culture assembly and invasion dynamics. Notably, the addition of any two stromal cell combinations to 3D cultures of BT-474 cells significantly reduces circularity of the 3D cultures, consistent of the presence of organoid-like or secondary spheroid structures. In tracker dye experiments, fibroblasts and endothelial cells co-localize in the peripheral organoid-like protrusions and are spatially segregated from the primary BT-474 spheroid. Finally, hetero-multicellular 3D cultures of BT-474 cells have increased hydrogel invasion capacity. Since we observed these protrusive structures in hetero-multicellular 3D cultures of both non-tumorigenic and tumorigenic breast epithelial cells, this work provides an efficient and reproducible method for generating organoid-like 3D cultures in a scaffold-free environment for subsequent analyses of phenotypes associated with solid tumor progression. SUMMARY There is a critical need for 3D cancer models that capture hetero-cellular crosstalk to study cancer metastasis. Our study presents the generation of hetero-multicellular stromal-epithelial in a scaffold and scaffold free environment that can be used to study invasion and cellular spatial distributions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a55555e37181d8e4de2e978898726ac03e16ca" target='_blank'>
              Hetero-multicellular stromal cells incorporate into scaffold-free 3D cultures of epithelial cancer cells to drive invasion
              </a>
            </td>
          <td>
            Elizabeth Ortiz, Kyaw Hsu Thway, Gabriela Ortiz-Soto, Paulina Yao, Jonathan A Kelber
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fbdae748bda0a4cdbd87979473a945cde8f436f" target='_blank'>
              Comprehensive single-cell atlas of colorectal neuroendocrine tumors with liver metastases: unraveling tumor microenvironment heterogeneity between primary lesions and metastases
              </a>
            </td>
          <td>
            Yiqiao Deng, Qichen Chen, Chengyao Guo, Jinghua Chen, Xin Li, Zhi-yu Li, Ye‐fan Zhang, Jian‐jun Zhao, Jianguo Zhou, Jianqiang Cai, T. Yan, Xiaobing Wang, X. Bi, Zhen Huang, Hong Zhao
          </td>
          <td>2025-01-21</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Understanding the phenotypic transitions of cancer cells is crucial for elucidating tumor progression mechanisms, particularly the transition from a non-invasive spheroid phenotype to an invasive network phenotype. We developed an agent-based model (ABM) using Compucell3D, an open-source biological simulation software, to investigate how varying biophysical and biochemical parameters influence emerging properties of cellular communities, including cell growth, division, and migration. Our focus was on cell-cell contact adhesion and matrix remodeling effects on cancer cell migration. We simplified enzymatic remodeling of the extracellular matrix and the subsequent enhancements to cellular chemotaxis or durotaxis as a combined effect of localized cellular secretion of a chemoattractant. By varying the chemoattractant secretion rate and contact adhesion energy, we simulated their effects on cellular behavior and driving the transition from a spheroid phenotype to a network phenotype. The model serves as a digital twin for 3D cancer cell culture, simulating cancer cell growth, division, and invasion over 1 week, validated against published data. The simulations track the emergent morphological and collective phenotype changes using key metrics such as cell circularity and invasion. Our findings indicate that increased chemoattractant secretion enhances the invasiveness of the collective cells, promoting the transition to a network phenotype. Additionally, changing cell-cell contact energy from a strong cell-cell adhesion to a weak cell-cell adhesion affects the compactness of the spheroids, resulting in lower circularity and increased collective cell invasion. Our work advances the understanding of tumor progression by providing insights into the biophysical mechanisms behind invasive cancer cell phenotypic transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6e2c13239072839fce8a0eaeadba76fadb7701" target='_blank'>
              Agent-based modeling reveals impacts of cell adhesion and matrix remodeling on cancer collective cell migration phenotypes
              </a>
            </td>
          <td>
            Temitope O. Benson, Mohammad Aminul Islam, Kailei Liu, Ashlee N. Ford Versypt
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Identifying spatial domains is critical for understanding breast cancer tissue heterogeneity and providing insights into tumor progression. However, dropout events introduces computational challenges and the lack of transparency in methods such as graph neural networks limits their interpretability. This study aimed to decipher disease progression-related spatial domains in breast cancer spatial transcriptomics by developing the three graph regularized non-negative matrix factorization (TGR-NMF). A unitization strategy was proposed to mitigate the impact of dropout events on the computational process, enabling utilization of the complete gene expression count data. By integrating one gene expression neighbor topology and two spatial position neighbor topologies, TGR-NMF was developed for constructing an interpretable low-dimensional representation of spatial transcriptomic data. The progressive lesion area that can reveal the progression of breast cancer was uncovered through heterogeneity analysis. Moreover, several related pathogenic genes and signal pathways on this area were identified by using gene enrichment and cell communication analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744a8472d128321ec52aed7c2a2480cf5c10946b" target='_blank'>
              Deciphering progressive lesion areas in breast cancer spatial transcriptomics via TGR-NMF
              </a>
            </td>
          <td>
            Juntao Li, Shan Xiang, Dongqing Wei
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. A better understanding of the temporal dynamics and specific pathways leading into and out of the persister state is needed to identify strategies to prevent treatment failure. Using spatial transcriptomics in patient-derived xenograft models, we captured clonal lineage evolution during treatment. The persister state showed increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients. Phylogenetic tracing identified intrinsic and acquired resistance mechanisms (e.g., dual specific phosphatases, reticulon-4, and CDK2) and suggested specific temporal windows of potential therapeutic susceptibility. Deep learning-enabled analysis of histopathological slides revealed morphological features correlating with specific cell states, demonstrating that juxtaposition of transcriptomics and histological data enabled identification of phenotypically distinct populations from using imaging data alone. In summary, this study defined state change and lineage selection during melanoma treatment with spatiotemporal resolution, elucidating how choice and timing of therapeutic agents will impact the ability to eradicate resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d694b41718972dca1b5b8aa26a4498984d65d307" target='_blank'>
              Spatiotemporal Profiling Defines Persistence and Resistance Dynamics During Targeted Treatment of Melanoma.
              </a>
            </td>
          <td>
            Jill C. Rubinstein, Sergii Domanskyi, Todd B Sheridan, Brian Sanderson, SungHee Park, Jessica Kaster, Haiyin Li, Olga Anczuków, M. Herlyn, Jeffrey H. Chuang
          </td>
          <td>2024-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="Purpose Eyelid infiltrative basal cell carcinoma (iBCC) is the most common malignant tumor affecting the ocular adnexa, but studies on metabolic changes within its microenvironment and heterogeneity at the tumor invasive area are limited. This study aims to analyze metabolic differences among iBCC cell types using single-cell and spatial metabolomics analysis and to examine metabolic environment at the tumor invasive area. Methods Single-cell transcriptomic data of human basal cell carcinoma (BCC) were clustered and visualized using Uniform Manifold Approximation and Projection. Metabolic reprogramming was analyzed with single-cell flux estimation analysis. Spatial metabolomics data were obtained with the Timstof Flex MALDI 2 system, and Bruker software was used for region selection. Results Eight cell types were identified within the iBCC microenvironment. Differences between inflammatory cancer-associated fibroblasts and myofibroblastic cancer-associated fibroblasts were analyzed. Metabolic flux analysis showed increased glycolysis, glutamine, heme, and glutathione fluxes in the iBCC microenvironment. Spatial metabolomics revealed high levels of taurine, deoxy-GMP, O-phosphoethanolamine, and pyrithione. Both tumor and invasive regions had significant upregulation of fatty acid pathways, with marked increases in oleic and arachidonic acids at the invasive area. Specific upregulation of UDP-glucuronic acid and high UDP-glucose 6-dehydrogenase (UGDH) expression in the tumor region suggest UXS1 as a potential therapeutic target for iBCC. Conclusions This study establishes a metabolic microenvironment atlas of iBCC, revealing significant metabolic differences and the dominance of lipid and lysosome metabolism. Potential metabolic markers and characteristic substances in the invasive area offer new insights for immunotherapy and the exploration of BCC’s metabolic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/322ca4366508114a20cf4530c76f41a9f0ca02cf" target='_blank'>
              Metabolic Atlas of Human Eyelid Infiltrative Basal Cell Carcinoma
              </a>
            </td>
          <td>
            Yanjing Huang, Chengjie He, Qiuling Hu, Zhong Liu, Xingyi Li, Wuyou Gao, Xuanwei Liang, Rongxin Chen, Zhen Mao, Xianchai Lin
          </td>
          <td>2025-01-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af37e793e7291e5ec4163cfb7481755bd374769" target='_blank'>
              Spatial transcriptomics reveals unique metabolic profile and key oncogenic regulators of cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Limin Zhou, Jiejie Liu, Peipei Yao, Xing Liu, Fei Chen, Yu Chen, Li Zhou, Chao Shen, You Zhou, Xin Du, Junbo Hu
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3852f7c6358b758c264d481b7403f1d4972dcd97" target='_blank'>
              Immunotherapy in ovarian cancer: spatial functional genomics to unravel resistance mechanisms
              </a>
            </td>
          <td>
            Martina Rausch, Karlotta Bartels, Josef Leibold
          </td>
          <td>2025-01-22</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Liposarcoma (LPS) is the most prevalent soft tissue sarcoma. The most common biological subtypes are well-differentiated (WDLPS), a low-grade disease that can evolve to high-grade dedifferentiated LPS (DDLPS), with increased rates of recurrence and metastasis and low response rates to chemotherapy and targeted therapies. Preclinical testing of immunotherapeutics for LPS has been held back by the lack of an immunocompetent mouse model. Here, we present a spontaneous immunocompetent LPS mouse model, ACPP, with targeted deletion of Trp53 and Pten in adipocytes to mimic signaling alterations observed in human LPS. Similar to human LPS, tumors arising in ACPP mice produce WDLPS and DDLPS, along with tumors that exhibit both WD and DD components. Murine and human DDLPS tumors possess transcriptional similarities, including increased expression of oncogenes Cdk4 and Hmga2 and reduced expression of the tumor suppressor Cebpa; further, both mouse and human DDLPS exhibit either high or low T cell infiltration. Syngeneic cell lines derived from spontaneous ACPP DDLPS reliably produce tumors following orthotopic injection, each with distinct growth patterns, aggressiveness and tumor infiltrating lymphocyte profiles. These models provide much needed tools to understand the complex immunobiology of LPS and greatly accelerate the pace of preclinical studies to uncover new therapies for patients with this aggressive malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/534b0303f4645ca052b41ef2e2a1a4b7d31ee268" target='_blank'>
              An immunocompetent mouse model of liposarcoma
              </a>
            </td>
          <td>
            Amanda Shafer, Emma Kenna, Lexi F Golden, Ahmed M. Elhossiny, Kyle Perry, Jodi Wilkowski, Wei Yan, Brynn Kaczkofsky, J. McGue, Scott C. Bresler, Adam H. Courtney, Jessie M. Dalman, C. Galbán, Wei Jiang, Carlos E. Espinoza, Rashmi Chugh, Matthew K. Iyer, Timothy L. Frankel, M. Pasca di Magliano, Andrzej A. Dlugosz, Christina V. Angeles
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 227


 Background:
 Colorectal cancer (CRC) is surgically resectable and eminently screenable yet remains a lethal entity with high affinity to metastasise synchronously and metachronously. It has been proposed that cellular and extracellular components of the microenvironment contribute to metastatic potential. Dissociative profiling techniques such as bulk transcriptomic and single-cell RNA sequencing have contributed insight but limited therapeutic progress. Spatial transcriptomic (ST) assessment allows molecular profiling of tissue while preserving tissue architecture. Here we employ ST approaches to interrogate tumor compartments of primary resectable and matched synchronous CRC demonstrating stromal signatures with distinct collagen expression associated with outcome.
 Methods:
 25 patients with primary resectable CRC and 4 patients with matched primary CRC and liver metastasis (CRLM) underwent single-cell spatial transcriptomics using the Nanostring CosMx Spatial Molecular Imager (SMI, 1000plex gene panel) (Discovery cohort). 71681 epithelial and 88806 stromal cells with intact spatial resolution were analysed. The spatial single-cell signatures were reconstituted in 89 patients using the Nanostring GeoMx Digital Spatial Profiler (Validation cohort). 3 GeoMx compartments were analysed: Epithelial (PanCK+); Stroma (aSMA+); Immune (PanCK-aSMA-).
 Results:
 CosMx demonstrated 2 distinct collagen signatures: COL1A1,COL1A2,COL3A1 associated with normal fibroblasts; COL9A2 associated with cancer-related stroma and lethal subtypes of epithelial cell. In the GeoMx validation cohort, 42 patients expressed the COL1A1 signature in the aSMA compartment and 47 did not (5 year recurrence free survival: 0.88 vs 0.49, p < 0.005) with different morphological patterns of each distinct aSMA group. In patients with favourable prognosis, chemokine high epithelial subtypes expressed CXCL5 and recruited COL9A2-, IL6+ neutrophils to the surrounding microenvironment. In contrast, epithelial cells in patients with poor prognosis expressed CXCL8 and recruited COL9A2+ neutrophils which were frequently in contact with SPP1+ macrophages.
 Conclusions:
 We have used ST approaches to interrogate cellular compartments of CRC demonstrating stromal subtypes that impact outcome. These insights could be used in the clinical setting to quantify COL9A2 expression in aSMA+ cells to prognosticate patients. Targeting of tumor collagen has been proposed to augment existing anti-cancer therapies and this work has demonstrated COL9A2 as a potential target in need of further investigation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ead468254f12c6dc5934a33a000887d505597fb4" target='_blank'>
              Spatial transcriptomics and lethality-associated stromal remodeling of colorectal cancer.
              </a>
            </td>
          <td>
            C. Wood, João Da Silva Filho, Tengyu Zhang, Campbell Roxburgh, Paul G. Horgan, J. Edwards, Matthias Marti, Colin W. Steele, Nigel B Jamieson
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a529afcdab161e3ac0972bc45b1449999ddb519" target='_blank'>
              EMT-driven plasticity prospectively increases cell–cell variability to promote therapeutic adaptation in breast cancer
              </a>
            </td>
          <td>
            Lauriane Muller, Frédérique Fauvet, Christelle Chassot, Francesca Angileri, Angèle Coutant, Cyril Dégletagne, Laurie Tonon, P. Saintigny, Alain Puisieux, A. Morel, Maria Ouzounova, Pierre Martinez
          </td>
          <td>2025-02-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="In most cancers, including endometrial cancer, tumor suppressor genes harboring inactivating mutations have been systematically cataloged. However, locus-specific epigenetic alterations contributing to cancer initiation and progression remain only partly described, creating knowledge gaps about functionally significant tumor suppressors and underlying mechanisms associated with their inactivation. Here, we show that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event not associated with promoter hypermethylation. PAX2 is expressed throughout normal endometrial glands; however, microscopic clones with PAX2 protein loss arise spontaneously in adulthood and progress to endometrial precancers and cancers. Here we show that PAX2 protein loss occurs via transcriptional silencing in 80% of endometrial cancers. Mechanistically, transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that silencing is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive features (H3K27me3). The spread of the H3K27me3 signal is restrained by cohesin loops, preventing transcriptional dysregulation of neighboring genes. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. Genetically engineered mouse models together with organoid and human cell line studies established Pax2 as an in vivo endometrial tumor suppressor that cooperates with Pten to produce lethal cancers. Our discovery of a specific and recurring epigenetic alteration that transcriptionally silences PAX2 to initiate most endometrial cancers opens new lines of investigation into the origins of endometrial cancer with diverse implications for its diagnosis and treatment. One Sentence Summary We demonstrate that PAX2 is silenced by a highly recurrent epigenetic mechanism to initiate most endometrial cancers, establishing a novel mechanism of tumor suppressor inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c481e61973d015ee3841a58286f99ac21dc81f" target='_blank'>
              PAX2 is Transcriptionally Silenced by a Distinct Mechanism of Epigenetic Reprogramming to Initiate Endometrial Carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction Effective treatment of breast cancer remains a formidable challenge, partly due to our limited understanding of the complex microenvironmental factors that contribute to disease pathology. Among these factors are tissue-resident perivascular cells, which play crucial roles in shaping vascular basement membranes, maintaining vessel integrity, and communicating with adjacent endothelial cells. Despite their essential functions, perivascular cells have been relatively overlooked. Identifying them by immunostaining has been challenging due to their low abundance, inherent heterogeneity, and shared marker expression with other cell types. These challenges have hindered efforts to purify pericytes and generate primary cell models for studying their biology. Methods Using a recently developed FACS method, we successfully identified and purified each cell type from breast tissues, allowing us to deep-sequence their transcriptomes and generate primary cell models of each cell type—including pericytes. Here, we used these data to analyze cell-type-specific gene expression in tumors, which revealed a strong association between pericyte-specific genes and breast cancer patient mortality. To explore this association, we defined the heterogeneity of breast pericytes using single-cell RNA sequencing and identified a broad marker for visualizing perivascular cells in breast tumors. Results Remarkably, we discovered perivascular cells dissociated from vessels and emerged as a dominant mesenchymal cell type in a subset of breast tumors that contrasted with their normal perivascular location. Moreover, when we purified pericytes from the breast and cultured them alongside breast tumor cells, we discovered that they induced rapid tumor cell growth significantly greater than isogenic fibroblast controls. Discussion These findings identify perivascular cells as a key microenvironmental factor in breast cancer, highlighting the critical need for further research to explore their biology and identify specific stimulatory mechanisms that could be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920638149619a80dd215db038826c980f92e7f53" target='_blank'>
              Breast pericytes: a newly identified driver of tumor cell proliferation
              </a>
            </td>
          <td>
            Katelyn Del Toro, Y. Licon-Munoz, William Crabtree, Tristan Oper, Christine Robbins, William C. Hines
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Background: Spatial integration of metabolic pathways with single cell-level transcriptomic data in human tumor and normal tissue may elucidate mechanisms supporting tumorigenesis for therapeutic targeting. The suppressive adenosine signaling pathway is an emerging potential target for combination with radiotherapy. This study aims to create an in situ map of the pathway in non-small cell lung cancer (NSCLC) tumor and adjacent normal tissue. We seek to define how distinct cellular compartments cooperate and contribute to adenosine signaling within the tumor microenvironment (TME). Methods: Using Matrix-Assisted Laser Desorption/Ionization mass spectrometry imaging, we examined the spatial distribution of metabolites in paired human NSCLC compared to normal tissue from the same patient (n=15). In situ metabolite quantification and pathway enrichment analysis were performed. Second, we performed single nuclear RNA-sequencing (snRNA-seq) on NSCLC patient samples (n=4) to characterize the transcriptomic landscape of the tumors, including quantification of key members of the adenosine signaling pathway: ADORA2A (A2AR), ADORA2B (A2BR), ENTPD1 (CD39), NT5E (CD73), and ENPP1. Leveraging the average gene expression per cell subtype and pathologist-annotated H&E-stained tissue sections, we are integrating transcriptomic data with the spatial metabolomics to generate an in situ pathway map. Results: In NSCLC tissue, UMP, AMP, and ADP levels were significantly higher in tumor compared to normal tissue (p<0.0001), with mean percentage increases of 79%, 89%, and 73%, respectively. Metabolites downstream of adenosine, adenine and inosine, showed no significant differences (p>0.05). Single nuclear RNA-seq analyses revealed expression of the adenosine 2B receptor, ADORA2B, was highest in fibroblasts, lower in tumor, and absent in the immune compartment. ENTPD1 was expressed by all immune cells and tumor. NT5E was absent in T cells and ENPP1 was absent in macrophages, but expressed in all other immune compartments and tumor. Integration of snRNA-seq data with spatial metabolomics is underway to correlate metabolite abundance and gene expression within distinct cellular compartments to define cell type-specific contribution to adenosine signaling. Conclusion: Our study offers a multimodal approach to spatially resolve metabolic and transcriptomic data in situ relying on frozen NSCLC and normal lung tissue. This has significant translational implications, enabling analysis of biospecimens from multi-institutional clinical trials that precludes processing fresh tissue for single-cell analyses. Herein, we are able to localize key metabolites in untreated tumor and normal tissue and quantify the expression of pathway members at the cellular level. Through spatial integration of these data, we aim to decipher the relationship between distinct cell types within the TME and their contribution to adenosine-driven suppressive changes, then extend to radiotherapy-treated tumors.
 Citation Format: Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick J. McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z. Obradovic, Ramon C. Sun, Catherine S. Spina. {Abstract title} [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69f29862a6e18da7c70e3c9d22cd4b7cabc18a3" target='_blank'>
              Abstract B028: Multimodal approach to characterization of metabolism and immune modulation with insights from spatial metabolomics and transcriptomics
              </a>
            </td>
          <td>
            Chelsea L. Rahiman, Roberto Ribas, Manan Vij, Julia An, Patrick McCann, Jason Cham, Chang Liu, Parin Shah, Rosa Martin, Ariel Chen, Simon Cheng, Aleksandar Z Obradovic, Ramon C. Sun, Catherine S. Spina
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a central role in tumor progression, recurrence, and resistance to conventional therapies, making them a critical focus in oncology research. This review provides a comprehensive analysis of CSC biology, emphasizing their self-renewal, differentiation, and dynamic interactions with the tumor microenvironment (TME). Key signaling pathways, including Wnt, Notch, and Hedgehog, are discussed in detail to highlight their potential as therapeutic targets. Current methodologies for isolating CSCs are critically examined, addressing their advantages and limitations in advancing precision medicine. Emerging technologies, such as CRISPR/Cas9 and single-cell sequencing, are explored for their transformative potential in unraveling CSC heterogeneity and informing therapeutic strategies. The review also underscores the pivotal role of the TME in supporting CSC survival, promoting metastasis, and contributing to therapeutic resistance. Challenges arising from CSC-driven tumor heterogeneity and dormancy are analyzed, along with strategies to mitigate these barriers, including novel therapeutics and targeted approaches. Ethical considerations and the integration of artificial intelligence in designing CSC-specific therapies are discussed as essential elements of future research. The manuscript advocates for a multi-disciplinary approach that combines innovative technologies, advanced therapeutics, and collaborative research to address the complexities of CSCs. By bridging existing gaps in knowledge and fostering advancements in personalized medicine, this review aims to guide the development of more effective cancer treatment strategies, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99151138c6fc7abf05753313b772a7f6497b1128" target='_blank'>
              Deciphering the Role of Cancer Stem Cells: Drivers of Tumor Evolution, Therapeutic Resistance, and Precision Medicine Strategies
              </a>
            </td>
          <td>
            Mohamed El-Tanani, S. Rabbani, S. Satyam, I. Rangraze, A. Wali, Yahia El-Tanani, Alaa A. A. Aljabali
          </td>
          <td>2025-01-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Bone is a multifaceted tissue requiring orchestrated interplays of diverse cells within specialized microenvironments. Although significant progress has been made in understanding cellular and molecular mechanisms of component cells of bone, revealing their spatial organization and interactions in native bone tissue microenvironment is crucial for advancing precision medicine, as they govern fundamental signaling pathways and functional dependencies among various bone cells. In this study, we present the first integrative high-resolution map of human bone and bone marrow, using spatial and single-cell transcriptomics profiling from femoral tissue. This multi-modal approach discovered a novel bone formation-specialized niche enriched with osteoblastic lineage cells and fibroblasts and unveiled critical cell–cell communications and co-localization patterns between osteoblastic lineage cells and other cells. Furthermore, we discovered a novel spatial gradient of cellular composition, gene expression and signaling pathway activities radiating from the trabecular bone. This comprehensive atlas delineates the intricate bone cellular architecture and illuminates key molecular processes and dependencies among cells that coordinate bone metabolism. In sum, our study provides an essential reference for the field of bone biology and lays the foundation for advanced mechanistic studies and precision medicine approaches in bone-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de859f69ecf65ef803fd91ebf44b36784f4b9786" target='_blank'>
              Mapping the spatial atlas of the human bone tissue integrating spatial and single-cell transcriptomics
              </a>
            </td>
          <td>
            Weiqiang Lin, Yisu Li, Chuan Qiu, Binghao Zou, Yun Gong, Xiao Zhang, Di Tian, William Sherman, Fernando Sanchez, Di Wu, Kuan-Jui Su, Xinyi Xiao, Zhe Luo, Qing Tian, Yiping Chen, Hui Shen, Hongwen Deng
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Antibody-drug conjugates (ADCs) are gaining increasing traction in the treatment of oncological diseases; however, many clinical failures have also been observed. One key factor limiting ADC effectiveness is the heterogeneous expression of their target antigen. While the vast majority of ADCs in clinical development target antigens on cancer cells (cancer-targeting), they can also target antigens expressed on non-cancerous stromal cells in the tumor microenvironment (stroma-targeting). It remains unclear whether targeting cancer cells or stromal cells is more effective in suppressing tumor growth. Here, we present three related mathematical models to evaluate: (1) cancer-targeting ADCs with homogeneous target antigen expression, (2) cancer-targeting ADCs with heterogeneous target antigen expression, and (3) stroma-targeting ADCs. Our simulations suggest that cancer-targeting ADCs can achieve high efficacy when their target antigen is homogenously expressed. However, in cases of heterogeneous antigen expression, cancer-targeting ADCs may lead to an initial reduction in tumor size, followed by regrowth due to the elimination of antigen-positive cells and expansion of antigen-negative cells. This limitation could potentially be overcome by stroma-targeting ADCs, as antigen-positive stromal cells may continue to be recruited into the tumor by the oncogenic factors produced by the remaining cancer cells. Furthermore, we demonstrate that ADCs with more permeable payloads and less stable linkers may offer improved efficacy in the context of heterogeneous target expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cecec004a693ebfc55213673dc24b88d447d0ee" target='_blank'>
              Mechanistic modeling suggests stroma-targeting antibody-drug conjugates as an alternative to cancer-targeting in cases of heterogeneous target expression
              </a>
            </td>
          <td>
            N. E. Wood, Anil Cengiz, Ming Gao, Alexander V Ratushny, Ronny Straube
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Understanding how radiotherapy induces metabolic reprogramming and reshapes the cytoarchitecture of the tumor microenvironment (TME) is essential for improving treatment outcomes and overcoming resistance. Despite its importance, no studies have used computational tools to quantitatively assess these cytoarchitectural rearrangements in the TME following radiotherapy. In previous work, we developed a quantitative method called “colocatome analysis,” a spatial -omic approach that catalogs statistically significant colocalizations between pairs of cell subpopulations or states. Through a normalization process, this method enables direct comparisons of spatial features across different conditions, including comparisons between in vitro models and tissue samples. We applied colocatome analysis to multiplexed immunofluorescence images of lung adenocarcinoma specimens, examining differences between naïve and recurrent tumors following radiotherapy using a panel of markers that represent primary tumor compartments (endothelial, malignant, fibroblast, and immune) and include markers for the hexosamine biosynthesis pathway (HBP), glycans, key metabolic enzymes, proliferation, cell death, DNA damage, hypoxia, and EMT. Our preliminary results indicate significant cytoarchitectural differences: tumors that recurred after radiotherapy exhibited a lower epithelial-to-mesenchymal cancer cell ratio and were enriched in FAP+ cancer-associated fibroblasts (CAFs). Although the overall proportions of macrophages and other immune cells remained consistent across samples, their spatial organization differed. In naïve tumors, immune cells were largely segregated from cancer cells, while recurrent tumors showed a more homogeneous mixture of cancer and immune cells. Additionally, CAFs were markedly increased in recurrent tumors. Lastly, cancer cells in naïve tumors displayed high levels of O-GlcNAc, which significantly decreased in recurrent tumors. These findings provide insights into the impact of metabolic reprogramming towards HBP in the TME and highlight its potential role in mediating radioresistance.
 Citation Format: Gina Bouchard.Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P019">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b0ad7d9e2bbe3827fd09e473275b031d3cd8d0" target='_blank'>
              Abstract P019: Spatiometabolic mapping of cytoarchitectural changes in lung adenocarcinoma following radiotherapy
              </a>
            </td>
          <td>
            Gina Bouchard
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b347c8724e3b90166bfc42ee1abaa89c4ab82766" target='_blank'>
              A spheroid whole mount drug testing pipeline with machine-learning based image analysis identifies cell-type specific differences in drug efficacy on a single-cell level
              </a>
            </td>
          <td>
            Mario Vitacolonna, Roman Bruch, Richard Schneider, Julia Jabs, Mathias Hafner, Markus Reischl, Rüdiger Rudolf
          </td>
          <td>2024-12-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The advent of spatial transcriptomics and spatial proteomics have enabled profound insights into tissue organization to provide systems-level understanding of diseases. Both technologies currently remain largely independent, and emerging same slide spatial multi-omics approaches are generally limited in plex, spatial resolution, and analytical approaches. We introduce IN-situ DEtailed Phenotyping To High-resolution transcriptomics (IN-DEPTH), a streamlined and resource-effective approach compatible with various spatial platforms. This iterative approach first entails single-cell spatial proteomics and rapid analysis to guide subsequent spatial transcriptomics capture on the same slide without loss in RNA signal. To enable multi-modal insights not possible with current approaches, we introduce k-bandlimited Spectral Graph Cross-Correlation (SGCC) for integrative spatial multi-omics analysis. Application of IN-DEPTH and SGCC on lymphoid tissues demonstrated precise single-cell phenotyping and cell-type specific transcriptome capture, and accurately resolved the local and global transcriptome changes associated with the cellular organization of germinal centers. We then implemented IN-DEPTH and SGCC to dissect the tumor microenvironment (TME) of Epstein-Barr Virus (EBV)-positive and EBV-negative diffuse large B-cell lymphoma (DLBCL). Our results identified a key tumor-macrophage-CD4 T-cell immunomodulatory axis differently regulated between EBV-positive and EBV-negative DLBCL, and its central role in coordinating immune dysfunction and suppression. IN-DEPTH enables scalable, resource-efficient, and comprehensive spatial multi-omics dissection of tissues to advance clinically relevant discoveries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec2a15d874ba82ad119e2c09f851c2a8a6c7876e" target='_blank'>
              Same-Slide Spatial Multi-Omics Integration Reveals Tumor Virus-Linked Spatial Reorganization of the Tumor Microenvironment
              </a>
            </td>
          <td>
            Yao Yu Yeo, Yuzhou Chang, Huaying Qiu, Stephanie Pei, Tung Yiu, Hendrik A. Michel, Wenrui Wu, Xiaojie Jin, Shoko Kure, Lindsay Parmelee, Shuli Luo, Precious Cramer, Jia Le Lee, Yang Wang, Jason Yeung, N. Ahmar, Berkay Simsek, R. Mohanna, McKayla Van Orden, Wesley Lu, K. Livak, Shuqiang Li, J. Shahryari, Leandra Kingsley, Reem N Al-Humadi, Sahar Nasr, Dingani Nkosi, Sam Sadigh, Philip Rock, Leonie Frauenfeld, Louisa Kaufmann, Bokai Zhu, Ankit Basak, Nagendra Dhanikonda, Chi Ngai Chan, Jordan Krull, Ye Won Cho, Chia-Yu Chen, Jia Ying, Joey Lee, Hongbo Wang, Bo Zhao, L. Loo, David M Kim, V. Boussiotis, Baochun Zhang, Alex K. Shalek, Brooke Howitt, S. Signoretti, Christian M. Schürch, Stephan Hodi, R. Burack, Scott J Rodig, Qin Ma, Sizun Jiang
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>100</td>
        </tr>

        <tr id="Motivation Spatially resolved omics technologies are enhancing our understanding of tissues architecture. Despite major technological improvements, gaining in spatial resolution becomes experimentally expensive, while generating spatial landscapes at moderate resolution combined with computational methods for depixelating data represent a cost-effective strategy allowing to enlarge the number of experiments to be performed. Results We have developed a computational strategy able to gain several-folds of resolution by inferring pseudo-interstitial pixels from their closest neighbors. This strategy has been validated in the context of public spatial transcriptomics data issued from melanoma, and human brain cortex tissue sections, by improving the identification of distinct tissue substructures. Furthermore, this methodology has been used for enhancing the resolution of consecutive sections collected from human brain organoids, as a way to demonstrate that a moderate resolution technology, combined with spatial depixelation processing allows to properly discern molecular tissue structures even in small tissues. Contact mmendoza@genoscope.cns.fr">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83223293873480f4821a0f7502d4baa16a5d8116" target='_blank'>
              Enhancing spatial omics resolution by pseudo-interstitial pixels inference
              </a>
            </td>
          <td>
            M. Mendoza-Parra, Maximilien Duvina, Ariel Galindo-Albarrán
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Invasion is a prerequisite for metastasis formation. During tumor development, the extracellular matrix (ECM) is remodeled in part through overexpression of type I collagen, increasing tumor microenvironment stiffness, and facilitating cancer dissemination. During breast cancer cell migration, we observed membrane debris left behind, attached to the collagen fibrils, along the migration path. We named these structures collagen-tracks. These collagen-tracks can be deposited in 3D matrices in vitro and in vivo and their formation is stimulated by the interaction between the ECM and matrix receptors, such as the discoidin-domain receptor (DDR1). However, they are different from structures already known to be involved in cell-cell communication such as exosomes and migrasomes, due to their specific nucleic acid and protein contents. When deposited by highly invasive breast cancer cells, internalized collagen-tracks reprogram non-invasive cells into highly pro-metastatic ones by inducing a partial epithelial–mesenchymal transition (EMT). This cell reprogramming is dependent on specific miRNAs present in the collagen-tracks, that are necessary to promote ECM degradation, increase cell motility and invasiveness. Collagen-tracks thus represent a new form of cell-cell communication important for driving tumor invasion that could be targeted to prevent metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607709bacd540192f2a53b41a2c4eccacb7b266e" target='_blank'>
              Cancer cells transfer invasive properties through microRNAs contained in collagen-tracks
              </a>
            </td>
          <td>
            Léa Normand, L. Rouyer, Elodie Richard, Nathalie Allain, Julie Giraud, Sylvaine Di-Tommaso, C. Dourthe, A. Raymond, Jean-William Dupuy, Kevin Moreau, Richard Iggo, Gaetan McGrogan, Nathalie Dugot-Senant, Anthony Bouter, Sisareuth Tan, Alexandre Favereaux, Violaine Moreau, R. Luco, M. Ros, F. Saltel
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Many tumors evolve through cellular mutation and selection, where subpopulations of cells (subclones) with shared ancestry compete for dominance. Introduction of next generation sequencing enables subclone identification using small somatic. However, there are several advantages to marking subclones with structural variants: they have greater functional impact, play a crucial role in late-stage tumors, and provide a more complete view of genomic instability driving tumor evolution. Here, we present SVCFit, a scalable method to estimate the cellular prevalence of somatic deletions, duplications and inversions. We demonstrate that cellular prevalence estimation can be improved by incorporating distinct read patterns for each structural variant type. Additionally, this improvement is achieved without prior knowledge of tumor purity, which is often inaccurate. Using a combination of simulated data and patient-derived metastatic samples with known mixture proportions, we show that our algorithm achieves significantly greater accuracy than state-of-the-art in estimating the structural variants cellular prevalence (p<0.05). The speed of SVCFit estimation from cost-effective bulk whole-genome sequencing (WGS) makes it well-suited for analyzing large cohorts of sequenced tumor samples, enhancing the accessibility of SV-based clonal reconstruction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77e71e2d20d9e52879b8c2d8cc5014dd16b7a335" target='_blank'>
              SVCFit: Inferring structural variant cellular fraction in tumors
              </a>
            </td>
          <td>
            Yunzhou Liu, Jiaying Lai, Laura D. Wood, Rachel Karchin
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Atherosclerosis involves complex interactions between lipids, immune cells, vascular smooth muscle cells (VSMCs), and fibroblasts within the arterial wall. While significant advances in single-cell technologies have shed light on the roles of immune cells and VSMCs in plaque development, fibroblasts remain underexplored, leaving critical gaps in understanding their contributions to disease progression and plaque stability. Comprehensive characterization of fibroblast phenotypes in atherosclerosis is essential to unravel their diverse functions and to distinguish between subsets that may play protective versus pathogenic roles in the disease process. Methods Here, we utilized CITE-seq (Cellular Indexing of Transcriptomes and Epitopes by Sequencing) to comprehensively profile fibroblast diversity in a mouse model of atherosclerosis. Mice were fed an atherogenic diet for 0, 8, 19, and 26 weeks, representing distinct stages of disease progression, enabling a detailed phenotypic characterization of fibroblasts throughout the course of atherosclerosis development. Results We identified four distinct fibroblast subpopulations, including a myofibroblast population closely resembling VSMC-derived chondromyocytes. The proportions of these fibroblast subsets exhibited a modest decline as atherosclerosis progressed. Through multimodal analysis, we identified CD26 as a highly expressed and specific marker for one of these fibroblast subpopulations, distinguishing it from other subsets. Using a combination of flow cytometry and immunohistochemistry, we demonstrated that CD26+ fibroblasts predominantly reside in the adventitia of healthy arteries. During atherosclerosis progression, these cells expand into the intima and primarily localize within the fibrous cap of the lesion. Conclusions Our multi-omic analysis highlights the phenotypic diversity and dynamic changes of fibroblasts during atherosclerosis progression. Among these, CD26+ fibroblasts emerge as a distinct subpopulation that expands within atherosclerotic lesions and may play a critical role in promoting plaque stability through their migration into the fibrous cap.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcefa069d1b01c8875c90b650f37b9a970f0c89d" target='_blank'>
              Single-Cell Multimodal Profiling Reveals a Novel CD26+ Fibroblast Subpopulation in Atherosclerosis
              </a>
            </td>
          <td>
            A. Bashore, Johana Coronel, Chenyi Xue, Lucie Y. Zhu, Muredach P. Reilly
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Peripheral nerve sheath tumors (PNSTs) encompass entities with different cellular differentiation and degrees of malignancy. Spatial heterogeneity complicates diagnosis and grading of PNSTs in some cases. In malignant PNST (MPNST) for example, single cell sequencing data has shown dissimilar differentiation states of tumor cells. Here, we aimed at determining the spatial and biological heterogeneity of PNSTs.


METHODS
We performed spatial transcriptomics on formalin-fixed paraffin-embedded diseased peripheral nerve tissue. We used spatial clustering and weighted correlation network analysis to construct niche-similarity networks and gene expression modules. We determined differential expression in primary pathologies, analysed pathways to investigate the biological significance of identified meta-signatures, integrated the transcriptional data with histological features and existing single cell data, and validated expression data by immunohistochemistry.


RESULTS
We identified distinct transcriptional signatures differentiating PNSTs. Immune cell infiltration, APOD and perineurial fibroblast marker expression highlighted the neurofibroma component of hybrid PNSTs (HPNSTs). While APOD was evenly expressed in neurofibromatous tumor tissue in both, HPNST and pure neurofibromas, perineurial fibroblast markers were evenly expressed in HPNST, but restricted to the periphery in plexiform neurofibromas. Furthermore, we provide a spatial cellular differentiation map for MPNST, locating Schwann cell precursors, neural crest-like cells and those with mesenchymal transition.


CONCLUSIONS
This pilot study shows that applying spatial transcriptomics to PNSTs provides important insight into their biology. It helps establishing new markers and provides spatial information about cellular composition and distribution of cellular differentiation states. By integrating morphological and high-dimensional molecular data it can improve PNSTs classification in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2233f7b03c50daacd9edb143f0325a91952a1c6" target='_blank'>
              Spatially resolved transcriptomics of benign and malignant peripheral nerve sheath tumors.
              </a>
            </td>
          <td>
            J. Bremer, Pamela Franco, J. A. Menstell, S. Tey, K. Zajt, Klimentina Popzhelyazkova, Kay Nolte, Jürgen Schlegel, Maria Teresa Pedro, Anja Osterloh, D. Delev, M. Hohenhaus, Christoph Scholz, O. Schnell, J. Beck, Joachim Weis, D. Heiland
          </td>
          <td>2025-01-23</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ultraviolet (UV)-induced DNA mutations produce genetic drivers of cutaneous melanoma initiation and numerous neoantigens that can trigger anti-tumor immune responses in the host. Consequently, melanoma cells must rapidly evolve to evade immune detection by simultaneously modulating cell-autonomous epigenetic mechanisms and tumor-microenvironment interactions. Angiogenesis has been implicated in this process; although an increase of vasculature initiates the immune response in normal tissue, solid tumors manage to somehow enhance blood flow while preventing immune cell infiltration. By comparing the expression of transcription factors (TFs) across early-stage melanoma, naevi, and other cancer types, we found the homeodomain-containing TF HOXD13 drives a melanoblast-like developmental program, which is upregulated in melanoma and strongly correlated with angiogenesis and immune cell exclusion. Using transcriptomics, 3D chromatin profiling, and in vivo models, we demonstrate that HOXD13 upregulation promotes tumor growth in vivo by concomitantly enhancing angiogenesis and suppressing T-cell infiltration. HOXD13 orchestrates 3D chromatin contacts between distal enhancers and promoters, simultaneously activating VEGFA, SEMA3A, and CD73. VEGFA and SEMA3A remodel the tumor vasculature and CD73 elevates extracellular levels of adenosine, a vasodilator and immune suppressor that binds adenosine receptors (AdR) on endothelial and T cells. In line with these findings, HOXD13-induced growth advantage in vivo was significantly reversed by the concomitant administration of VEGFR and AdR inhibitors. By revealing a dual pro-angiogenic and immunosuppressive HOXD13-CD73/VEGF gene regulatory axis, we identify a subset of patients who might benefit from combinations of AdR and VEGFR inhibitors which are both currently being tested in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65e09ae66ed07e79880f3fb8e2a1de2aa1f2113" target='_blank'>
              A targetable developmental program co-regulates angiogenesis and immune evasion
              </a>
            </td>
          <td>
            P. Berico, Amanda Flores Yanke, Fatemeh Van Rajabpour, Catherine Do, Irving Simonin Wilmer, Theodore Sakellaropoulos, Ines Delclaux, Robert Stagnitta, Estefania Vázquez-Cruz, Iman Osman, Jane A Skok, C. D. Robles-Espinoza, Amanda W. Lund, Markus Schober, Eva Hernando
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer is a significant health challenge worldwide, and disproportionately affects women of African ancestry (AA) who experience higher mortality rates relative to other racial/ethnic groups. Several studies have pointed to biological factors that affect breast cancer outcomes. A recently discovered stromal cell population that expresses PROCR, ZEB1 and PDGFRα (PZP cells) was found to be enriched in normal healthy breast tissue from AA donors, and only in tumor adjacent tissues from donors of European ancestry (EA). Here, we investigated the effect of PZP cells on the acquisition of tumorigenic phenotypes in AA and EA human breast epithelial cells to determine its contribution to the biological basis of outcome disparities. Using 3D cell models of tumor invasion, we find that PZP cells confer invasive capacity to epithelial cells independent of genetic ancestry, wherein cells exhibit leader-follower behaviors during extracellular matrix invasion. Enhanced epithelial invasion stems from a combination of AKT activation and fibronectin deposition by the PZP cells. Although activation of AKT in epithelial cells alone is insufficient to induce invasive behaviors, blocking AKT activation markedly reduces invasive capacity. These findings point to the germaneness of differences in breast biology and the multi-faceted roles and enrichment in PZP cells in AA breast tissue, in furthering current understanding of the molecular basis for worse prognosis for patients of African descent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe60dd766ac1428c496c998ec8bd07c1a84a6947" target='_blank'>
              Ancestry-linked stromal variations impact breast epithelial cell invasion
              </a>
            </td>
          <td>
            M. Schmidtmann, Victoria Elliott, James W. Clancy, H. Nakshatri, Crislyn D'Souza-Schorey
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Novel multiplexed spatial proteomics imaging platforms expose the spatial architecture of cells in the tumor microenvironment (TME). The diverse cell population in the TME, including its spatial context, has been shown to have important clinical implications, correlating with disease prognosis and treatment response. The accelerating implementation of spatial proteomic technologies motivates new statistical models to test if cell-level images associate with patient-level endpoints. Few existing methods can robustly characterize the geometry of the spatial arrangement of cells and also yield both a valid and powerful test for association with patient-level outcomes. We propose a topology-based approach that combines persistent homology with kernel testing to determine if topological structures created by cells predict continuous, binary, or survival clinical endpoints. We term our method TopKAT (Topological Kernel Association Test) and show that it can be more powerful than statistical tests grounded in the spatial point process model, particularly when cells arise along the boundary of a ring. We demonstrate the properties of TopKAT through simulation studies and apply it to two studies of triple negative breast cancer where we show that TopKAT recovers clinically relevant topological structures in the spatial distribution of immune and tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286f79d159b2d5895452d1adff9339095baad447" target='_blank'>
              Detecting Clinically Relevant Topological Structures in Multiplexed Spatial Proteomics Imaging Using TopKAT
              </a>
            </td>
          <td>
            Sarah Samorodnitsky, Katie Campbell, Amarise Little, Wodan Ling, Ni Zhao, Yen-Chi Chen, Michael C. Wu
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Purpose
In glioblastoma, the therapeutically intractable and resistant phenotypes can be derived from glioma stem cells, which often have different underlying mechanisms from non-stem glioma cells. Aberrant signaling across the EGFR-PTEN-AKT-mTOR pathways have been shown as common drivers of glioblastoma. Revealing the inter and intra-cellular heterogeneity within glioma stem cell populations in relations to signaling patterns through these pathways may be key to precision diagnostic and therapeutic targeting of these cells.


Materials and Methods
Single cell parallel proteomic heterogeneity profiling of the EGFR-PTEN-AKT-mTOR pathways was conducted in a panel of fifteen glioma stem cell models derived from patient glioblastoma biopsies.


Results
The analysis included 59,464 data points from 14,866 cells and identified forty-nine molecularly distinct signaling phenotypes. High content bioinformatics resolved two unique patient clusters diverging on EGFR expression and AKT/TORC1 activation. Phenotypic validation indicated drug responsive phenotypes to EGFR blocking in the high EGFR expressing cluster with lower tumor initiating potential in comparison to the AKT/TORC1 activated cluster. High EGFR expression trended with improved patient prognosis while AKT/TORC1 activated samples trended with poorer patient outcomes. Genetic heterogeneity was observed in both clusters with proneural, classical and mesenchymal subtypes observed.


Conclusion
Quantitative single cell heterogeneity profiling reveals divergent EGFR-PTEN-AKT-mTOR pathways of patient derived glioma stem cells, which would inform future research and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0984500cc5c038ee7f954e1d13785419da86e8c6" target='_blank'>
              Single-Cell Heterogeneity of EGFR Pathway Is Linked to Unique Signatures of Drug Response and Malignancy in Patient Derived Glioblastoma Stem Cells.
              </a>
            </td>
          <td>
            Michael D. Masterman-Smith, Nicholas A. Graham, E. Panosyan, J. Mottahedeh, Eric R. Samuels, Araceli Nunez, Tiffany Phillips, Meeryo Choe, T. Cloughesy, Jorge A. Lazareff, Linda M. Liau, William H. Yong, Thomas G. Graeber, H. Kornblum, Ming-Fei Lang, Yanzhao Li, Jing Sun
          </td>
          <td>2025-01-07</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Muscle-invasive bladder cancer (MIBC) is a prevalent cancer characterized by molecular and clinical heterogeneity. Assessing the spatial heterogeneity of the MIBC microenvironment is crucial to understand its clinical significance. Methods In this study, we used imaging mass cytometry (IMC) to assess the spatial heterogeneity of MIBC microenvironment across 185 regions of interest in 40 tissue samples. We focused on three primary parameters: tumor (T), leading-edge (L), and nontumor (N). Cell gating was performed using the Cytobank platform. We calculated the Euclidean distances between cells to determine cellular interactions and performed single-cell RNA sequencing (scRNA-seq) to explore the molecular characteristics and mechanisms underlying specific fibroblast (FB) clusters. scRNA-seq combined with spatial transcriptomics (ST) facilitated the identification of ligand–receptor (L–R) pairs that mediate interactions between specific FB clusters and endothelial cells. Machine learning algorithms were used to construct a prognostic gene signature. Results The microenvironments in the N, L, and T regions of MIBC exhibited spatial heterogeneity and regional diversity in their components. A distinct FB cluster located in the L region—identified as S3—is strongly associated with poor prognosis. IMC analyses demonstrated a close spatial association between S3 and endothelial cells, with S3-positive tumors exhibiting increased blood vessel density and altered vascular morphology. The expression of vascular endothelial growth factor receptor and active vascular sprouting were significant in S3-positive tumors. scRNA-seq and ST analyses indicated that the genes upregulated in S3 were associated with angiogenesis. NOTCH1–JAG2 signaling pathway was identified as a significant L–R pair specific to S3 and endothelial cell interactions. Further analysis indicated that YAP1 was a potential regulator of S3. Machine learning algorithms and Gene Set Variation Analysis were used to establish an S3-related gene signature that was associated with the poor prognosis of tumors including MIBC, mesothelioma, glioblastoma multiforme, lower-grade glioma, stomach adenocarcinoma, uveal melanoma, kidney renal clear cell carcinoma, kidney renal papillary cell carcinoma, and lung squamous cell carcinoma. Conclusions We assessed the spatial landscape of the MIBC microenvironment and revealed a specific FB cluster with prognostic potential. These findings offer novel insights into the spatial heterogeneity of the MIBC microenvironment and highlight its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fcfdce6314523f58922cfd8b6a6e220595a1f7" target='_blank'>
              Spatially-resolved analyses of muscle invasive bladder cancer microenvironment unveil a distinct fibroblast cluster associated with prognosis
              </a>
            </td>
          <td>
            Chao Feng, Yaobang Wang, Wuyue Song, Tao Liu, Han Mo, Hui Liu, Shulin Wu, Zezu Qin, Zhenxing Wang, Yuting Tao, Liangyu He, Shaomei Tang, Yuanliang Xie, Qiuyan Wang, Tianyu Li
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff90b5963386b78b07e8f87307689a97c4a0bc10" target='_blank'>
              iFlpMosaics enable the multispectral barcoding and high-throughput comparative analysis of mutant and wild-type cells.
              </a>
            </td>
          <td>
            Irene Garcia-Gonzalez, Stefano Gambera, Susana F Rocha, Alvaro Regano, Lourdes García-Ortega, Mariya Lytvyn, Luis Diago-Domingo, Maria S. Sanchez-Muñoz, Aroa Garcia-Cabero, Ivana Zagorac, Wen Luo, Macarena De Andrés-Laguillo, Macarena Fernández-Chacón, Verónica Casquero-García, F. F. Lunella, C. Torroja, F. Sánchez-Cabo, Rui Benedito
          </td>
          <td>2024-12-13</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) tumor cells and their interactions with the immune microenvironment, particularly at the leading-edge area, have been underexplored. This study employs single-cell RNA sequencing (scRNA-seq) and spatial transcriptome (ST) analysis on samples from the tumor core, adjacent non-tumorous tissue, and the leading-edge area of nine ICC patients. These findings indicate that tumor cells at the leading-edge area demonstrate enhanced proliferation and are tightly associated with the stroma, including endothelial cells and POSTN+ FAP+ fibroblasts. Notably, CD8+ T cells in this region exhibit a naive phenotype with low cytotoxicity and signs of exhaustion, likely due to compromised antigen presentation by antigen-presenting cells (APCs). The predominant CD8+ T cell subset, mucosal-associated invariant T (MAIT) cells, recruits SPP1+ macrophages within the stroma. This interaction, along with the presence of POSTN+ cancer-associated fibroblasts (CAFs) and endothelial cells, forms a unique "triad structure" that fosters tumor growth and ICC progression. The research highlights the intricate characteristics and interactions of ICC tumor cells in the leading-edge area, offering insights into potential therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4692f475931cf20c24dd17b389c955de7ede39" target='_blank'>
              Single-Cell and Spatial Transcriptomics Delineate the Microstructure and Immune Landscape of Intrahepatic Cholangiocarcinoma in the Leading-Edge Area.
              </a>
            </td>
          <td>
            Li Zuyin, Li Zhao, Cheng Qian, Changkun Zhang, Ma Delin, Jialing Hao, Chen Zhuomiaoyu, Li Yuzi, Zheng Jiaxi, Gao Jie, Zhu Jiye
          </td>
          <td>2024-12-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gamma-delta (γδ) T cells are a crucial component of the tumor immune microenvironment which are considered a promising potential therapeutic strategy and target. Increasing evidence suggests that these unique immune cells play significant roles across various cancers. However, γδ T cells are often regarded as having dual roles in tumors, and their influence on lung adenocarcinoma (LUAD) remains controversial. In this research, we employed a wild-ranging approach using multi-omics data to investigate the function of γδ T cells in LUAD. The abundance of γδ T cell infiltration is linked to a positive prognosis in lung adenocarcinoma. The tumor-inhibiting role of γδ T cells was played through intrinsic lineage evolution, acquiring cytotoxic functions and engaging in signal transduction with antigen-presenting cells. Furthermore, patients with higher γδ T cells infiltration abundance might be more favorable for immunotherapy. Lastly, we established a predictive model using CT images based on radiomics, providing a non-invasive strategy to assess γδ T cells infiltration in LUAD patients. These findings offer new insights and perspectives the personalized therapies of γδ T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ada3c2d9795ca814751eeb983c8b55cf90854b2" target='_blank'>
              Integrating of Radiomics and Single-cell Transcriptomics Uncovers the Crucial Role of γδ T Cells in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Ziyi Chen, Changqing Yang, Jiajing Wang, Baichuan Wang, Long He, Zhaoyi Liu, Yingxi Li, Tingting Qin, Peng Chen
          </td>
          <td>2025-01-06</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="The adaptation of metastatic cells to their host tissue critically determines the pathogenicity of a cancer and therefore patient survival. Yet, it remains elusive to what extent the host environment drives gene expression programs in metastatic cells. Here we identify adaptive mechanisms that enable metastases to establish themselves in a novel tissue context. We performed single-cell RNA-sequencing on malignant and benign tissue samples from untreated donors with colorectal adenocarcinoma and liver metastasis to deduce tissue adaptive expression patterns. A novel computational approach identified genes and pathways that consistently adapted to the host tissue at the transition from the primary tumor to the paired metastasis across donors. This analysis revealed that the majority of expression changes in the metastasis reflect an expression signature reminiscent of benign liver epithelial cells. Cellular processes adapting to the liver environment include basic cellular functions such as energy metabolism, as well as tissue-specific pathways such as the regulation of lipid metabolism by PPAR-α. These adaptations potentially increase the pathogenicity of the metastatic cells and may provide new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189b4d35fab487ddefe0f36a861c9c03b2e18b5a" target='_blank'>
              Gene expression adaptation of metastases to their host tissue
              </a>
            </td>
          <td>
            L. Nagel, Marten C. Wenzel, Sascha Hoppe, Patrick S. Plum, Mohammad Karimpour, Marek Franitza, Roger Wahba, Marc Bludau, Christiane J. Bruns, Alexander Quaas, A. Beyer, Axel M. Hillmer
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8afd5989cd7e1d2eeffdf0537754b1eeb0da52a8" target='_blank'>
              SMORE: spatial motifs reveal patterns in cellular architecture of complex tissues
              </a>
            </td>
          <td>
            Zainalabedin Samadi, Kai Hao, Amjad Askary
          </td>
          <td>2025-01-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) account for 0.01 to 2% of the total tumor mass; however, they play a key role in tumor progression, metastasis and resistance to current cancer therapies. The generation and maintenance of CSCs are usually linked to the epithelial–mesenchymal transition (EMT), a dynamic process involved in reprogramming cancer cells towards a more aggressive and motile phenotype with increased stemness potential. Cells that undergo an EMT process have shown to be more resistant to conventional chemo/radiotherapies. In this context, aldehyde dehydrogenase (ALDH) enzymes, known for their role in the cellular detoxification of aldehydes and enhancement of cell survival, are often upregulated in cancer cells, promoting their resistance to conventional cancer treatments. Indeed, high ALDH levels have become a hallmark biomarker of CSCs and are often used to isolate this sub-population from the more abundant cancer cell populations. Herein, we isolated human breast cancer epithelial cells with higher ALDH abundance (ALDHHigh) and compared them to those with low ALDH abundance (ALDHLow). ALDHHigh sub-populations exhibited more characteristic EMT biomarkers by adopting a more mesenchymal phenotype with increased stemness and enhanced migratory potential. Furthermore, ALDHHigh sub-populations displayed elevated senescent markers. Moreover, these cells also demonstrated higher levels of mitochondria DNA/mass, as well as greater mitochondrial and glycolytic metabolic function. Conversely, ALDHLow sub-populations showed a higher efficiency of mammosphere/colony formation and an increased proliferative capacity. Therefore, we demonstrated that these ALDH sub-populations have distinct characteristics, underscoring their role in EMT, the formation of tumors and the mechanisms of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c656cc9a0663a21dc7738f32f6803ef9c5938a5" target='_blank'>
              ALDHHigh Breast Cancer Stem Cells Exhibit a Mesenchymal–Senescent Hybrid Phenotype, with Elevated Metabolic and Migratory Activities
              </a>
            </td>
          <td>
            Luis Larrea Murillo, Conor J. Sugden, Bela Ozsvari, Zahra Moftakhar, Ghada S. Hassan, F. Sotgia, Michael P Lisanti
          </td>
          <td>2024-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1332adb107deebc84becc80e062e225cebcbced2" target='_blank'>
              Turning attention to tumor–host interface and focus on the peritumoral heterogeneity of glioblastoma
              </a>
            </td>
          <td>
            Fang Wang, Jiawei Dong, Yuyun Xu, Jiaqi Jin, Yan Xu, Xiuwei Yan, Zhihui Liu, Hongtao Zhao, Jiheng Zhang, Nan Wang, Xueyan Hu, Xin Gao, Lei Xu, Chengyun Yang, Shuai Ma, Jianyang Du, Ying Hu, Hang Ji, Shaoshan Hu
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The tumor microenvironment is composed of tumor cells and various stromal cell types, such as immune cells, fibroblasts, and vascular cells. Signaling interactions between tumor and stromal cells orchestrate the tumor microenvironment's contribution to tumor progression. Angiopoietin-like protein 2 (ANGPTL2) is a secreted glycoprotein homologous to angiopoietins. Previous studies indicate that tumor cell-derived ANGPTL2 serves as a tumor promoter. However, recent studies suggest that tumor stroma-derived ANGPTL2 shows tumor-suppressive activity by enhancing anti-tumor immune responses, supporting a dual function for ANGPTL2 in cancer pathology. Such complexity can complicate development of effective therapeutic strategies targeting ANGPTL2. In this Review, we focus on ANGPTL2 activity in the tumor microenvironment and its function in anti-cancer immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a90f8b24d69e3991005e6c2b8b108915c7b64989" target='_blank'>
              The Two Faces of Angiopoietin-Like Protein 2 in Cancer.
              </a>
            </td>
          <td>
            Haruki Horiguchi, T. Kadomatsu, Yuichi Oike
          </td>
          <td>2024-12-17</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="L1TD1 is a pluripotency factor required for embryonic stem cell self-renewal; its expression has also been detected in solid tumors, including embryonal tumors of the central nervous system (CNS). Previously, we showed that L1TD1 expression correlates with metastasis formation and shorter overall survival of medulloblastoma patients. Here, we used affinity purification coupled to mass spectrometry to map the L1TD1 interactome, and global proteomics to assess proteins differentially regulated by L1TD1 expression in patient-derived embryonal CNS tumor cell lines. We identified novel L1TD1 interactors and differentially expressed proteins related to cell proliferation, death and motility. Finally, we demonstrated that L1TD1-overexpressing tumor cells have distinct cell morphology with enhanced filopodial formation, higher cell motility, greater proliferation capability, and reduced sensitivity to cisplatin treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8af0488d6a2ccbb080a6ebfdc837df22774e9cf9" target='_blank'>
              Integrated interactome, proteomic and functional analyses reveal molecular pathways driving L1TD1-induced aggressiveness in CNS embryonal tumor cells.
              </a>
            </td>
          <td>
            T. Mitsugi, Matt Sherwood, Elisa Helena Farias Jandrey, Amanda Faria Assoni, Joseph Bell, Brandon Coke, Paul Skipp, Rob M. Ewing, Oswaldo Keith Okamoto
          </td>
          <td>2025-02-06</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Resistance to tumor necrosis factor inhibitor (TNFi) therapy remains a major obstacle in managing ulcerative colitis (UC), affecting up to 50% of patients.1 The complex interplay of immune cells and their spatial organization in the gut mucosa is poorly understood but likely contributes to therapeutic outcomes.1,2 This study utilized advanced multiplexed imaging to construct a spatial atlas of UC, providing insights into cellular interactions driving TNFi resistance. Novel spatial omic technologies show to be able to add additional thus far unknown insights to this question.



 A total of 42 tissue regions from 29 UC patients and 5 healthy controls were analyzed using CODEX multiplexed imaging. Spatially resolved single-cell data, encompassing over 1.7 million cells, were used to identify distinct cell types and map cellular neighborhoods (CNs). Pairwise cell-cell contacts and CN-level functional states were analyzed in TNFi-treated and untreated patients, correlating findings with clinical outcomes.



 TNFi therapy led to normalization of adaptive immune CNs, such as lymphoid aggregates, with reductions in T-cell frequencies and associated contacts. However, innate immune CNs, including neutrophil-enriched granulocyte niches and inflamed stroma, remained unchanged despite treatment. Notably, neutrophil-dominated CNs exhibited persistent expression of pro-inflammatory markers, such as TNFR2 and CD137, implicating these niches in sustaining mucosal inflammation and therapy resistance. Additionally, sex-dependent differences were observed, with higher TNFi responsiveness in females potentially linked to adaptive immune modulation.3



 Neutrophil-dominated inflammatory niches represent a potential mechanism of TNFi resistance in UC.3,4 These findings highlight the importance of targeting innate immune pathways alongside TNFi therapy to improve outcomes. By integrating spatial insights with functional data, this study paves the way for precision medicine approaches in UC management.



 1.R. Ungaro, S. Mehandru, P. B. Allen, L. Peyrin-Biroulet, J.-F. Colombel, Ulcerative colitis.Lancet 389, 1756–1770 (2017).
 2.R. Gaujoux, E. Starosvetsky, N. Maimon, F. Vallania, H. Bar-Yoseph, S. Pressman, R. Weisshof,I. Goren, K. Rabinowitz, M. Waterman, H. Yanai, I. Dotan, E. Sabo, Y. Chowers, P. Khatri,S. S. Shen-Orr; Israeli IBD research Network (IIRN), Cell-centred meta-analysis revealsbaseline predictors of anti-TNFα non-response in biopsy and blood of patients with IBD.Gut 68, 604–614 (2019).
 3.A.T. Mayer, D.R. Holman, A. Sood, U. Tandon, S. S. Bhate, S. Bodapati, G. L. Barlow, J. Chang, S. Black, E. C. Crenshaw, A. N. Koron, S. E. Streett, S. S. Gambhir SS, W. J. Sandborn, B. S. Boland, T. Hastie, R. Tibshirani, J. T. Chang, G. P. Nolan, C. M. Schürch, S. Rogalla “A tissue atlas of ulcerative colitis revealing evidence of sex-dependent differences in disease-driving inflammatory cell types and resistance to TNF inhibitor therapy” Sci Adv. 2023 Jan 20;9(3):eadd1166. doi: 10.1126/sciadv.add1166. Epub 2023 Jan 20.
 4.S. T. Eshghi, J. M. Gubatan, P. Mazrooei, L. Quintanilla, A. Nguyen, A. Au-Yeung, D. R. Holman, C. Takashi, C. Schiffman, B. O’Gorman, M. Keir, S. Ramanujan, S. Rogalla, J. A. Hackney, J. M McBride “Molecular characterization of response to Etrolizumab and anti-TNF reveals treatment resistance in ulcerative colitis is associated with abundance of residual neutrophil subsets and inflammatory fibroblast populations” https://doi.org/10.1101/2024.07.02.601267

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391837f71bfd5dab1136ff60f107c81956417fbb" target='_blank'>
              OP23 Neutrophil Niches and Therapy Resistance in Ulcerative Colitis: Spatial Atlas Insights
              </a>
            </td>
          <td>
            D. Holman, S. Rogalla
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1ebbd60130879f07ccf62c91d033d068147d46a" target='_blank'>
              A safe haven for cancer cells: tumor plus stroma control by DYRK1B
              </a>
            </td>
          <td>
            Miriam Ems, Anna Brichkina, Matthias Lauth
          </td>
          <td>2025-01-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed580d09e5d8a50632bdd53ac296d7f45feff0d1" target='_blank'>
              The microenvironment cell index is a novel indicator for the prognosis and therapeutic regimen selection of cancers
              </a>
            </td>
          <td>
            Xian-Yan Yang, Nian Chen, Qian Wen, Yu Zhou, Tao Zhang, Ji Zhou, Cheng-Hui Liang, Li-Ping Han, Xiao-Ya Wang, Qingmei Kang, Xiaomei Zheng, Xue-Jia Zhai, Hong-Ying Jiang, Tian-Hua Shen, Jin-Wei Xiao, Yu-Xin Zou, Yun Deng, Shuang Lin, Jiang-jie Duan, Jun Wang, Shicang Yu
          </td>
          <td>2025-01-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33631c265f68921d79fbd004eb59ad3a5ea83b81" target='_blank'>
              Spatially mapping the tumour immune microenvironments of non-small cell lung cancer
              </a>
            </td>
          <td>
            Lysanne Desharnais, Mark Sorin, M. Rezanejad, Bridget Liu, E. Karimi, Aline Atallah, Anikka M Swaby, Miranda W. Yu, Samuel Doré, Saskia Hartner, B. Fiset, Yuhong Wei, Baharak Kadang, R. Rayes, Philippe Joubert, Sophie Camilleri-Broët, P. Fiset, D. Quail, Jonathan D Spicer, L. Walsh
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Metastatic cancer, a major cause of mortality, has been understudied compared to primary tumors, leaving gaps in our understanding of how cancer genes adapt between these states. We analyzed the association between mutations and copy-number alterations in 25,000 tumor samples from both primary and metastatic cancers. Our findings show that cancer genes display distinct interaction strengths across these states, with 27.45% of genes, including ARID1A, FBXW7, and SMARCA4, shifting between one-hit and two-hit drivers. Interaction strengths varied by cancer state and treatment conditions, revealing seven state-specific interactions. We also identified 38 primary-specific and 21 metastatic-specific high-order interactions, enriched in cancer hallmarks, indicating unique tumor progression mechanisms. These findings highlight dynamic tumor progression mechanisms and underscore the importance of considering cancer state in research and treatment strategies for precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968f72ea3f3d388ad8546b594ad1735637aaf86d" target='_blank'>
              Mapping cancer gene dynamics through state-specific interactions
              </a>
            </td>
          <td>
            Adrián Maqueda-Real, Laia Ollé-Monràs, Solip Park
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLS) are lymphoid formations that develop in non-lymphoid tissues during chronic inflammation, autoimmune diseases, and cancer. Accurate identification and quantification of TLS in tissue can provide crucial insights into the immune response of several disease processes including antitumor immune response. TLS are defined as aggregates of T cells, B cells and dendritic cells. In histological tissue sections stained with Hematoxylin and Eosin they are identified as aggregates of 50 or more lymphoid cells, however immunohistochemical analysis are required to confirm presence of distinct immune cell patterns. Assessment of lymphoid aggregates can be done in H&E slides or in slides stained with single or multiplex immunohistochemistry or other tissue based high-plex approaches with key biomarkers such as CD3 (T cells) and CD20 (B cells), however manual assessment of them is time consuming thus limiting its full evaluation. To our knowledge, published algorithms that identify TLS within a tissue are based on histological assessment of H&E slides or through use of ML algorithms trained on images that show the TLS presence in tissues; and quantification and spatial analysis of TLS still remains a challenge. This study aims to develop a robust algorithm to recognize TLS using the spatial coordinates of immune cells in any given tissue. The algorithm uses X and Y coordinates of T and B cells in tissues identified by a pathologist-supervised digital image analysis of multiplex chromogenic immunohistochemistry of tissues stained with CD3 and CD20. The algorithm is flexible to be used for detailed analysis of TLS stages; including other cell types within the definition of TLS; such as dendritic cells (DC) and high endothelial venules (HEV); to assess different stages of TLS formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5237511ad45e3ecdaa3252f9aa2a014ac80b4418" target='_blank'>
              TLS_Finder: An algorithm for Identifying Tertiary Lymphoid Structures Using Immune Cell Spatial Coordinates
              </a>
            </td>
          <td>
            Ayse A Koksoy, Maria E Salvatierra, L. S. Solis Soto, C. Haymaker
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tissue homeostasis relies on a precise balance of fate choices between renewal and differentiation, which is dysregulated during tumor initiation. Although much progress has been done over recent years to characterize the dynamics of cellular fate choices at the single cell level, their underlying mechanistic basis often remains unclear. In particular, although physical forces are increasingly characterized as regulators of cell behaviors, a unifying description of how global tissue mechanics interplays with local cellular fate choices is missing. Concentrating on skin epidermis as a paradigm for multilayered tissues with complex fate choices, we develop a 3D vertex-based model with proliferation restrained in the basal layer, showing that mechanics and competition for space naturally gives rise to homeostasis and neutral drift dynamics that are seen experimentally. We then explore the effect of introducing mechanical inhomogeneities, whereby subpopulations have differential tensions. We uncover that relatively small mechanical disparities can be sufficient to heavily tilt cellular towards symmetric renewal and exponential growth. Importantly, the simulations predict that such mechanical inhomogeneities are reflected by distinct morphological changes in single-cell shapes. This led us to derive a master relationship between two very different experimentally measurable parameters, cell shape and long-term clonal dynamics, which we validated using a model of basal cell carcinoma (BCC) consisting in clonal Smoothened overexpression in mouse tail epidermis. Altogether, we propose a theoretical framework to link mechanical forces, quantitative cellular morphologies and cellular fate outcomes in complex tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/215257d8ed053f55e3fe3327a06377e6d6a6e712" target='_blank'>
              Biomechanics of stem cell fate decisions in multilayered tissues
              </a>
            </td>
          <td>
            Preeti Sahu, Sara Canato, Raquel Soares, Adriana Sánchez-Danés, É. Hannezo
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and aggressively progressing malignancy that presents considerable clinical challenges.Although chemotherapy can effectively manage symptoms during the earlystages of SCCE, its long-term effectiveness is notably limited, with theunderlying mechanisms remaining largely undefined. In this study, weemployed single-cell RNA sequencing (scRNA-seq) to analyze SCCE samplesfrom a single patient both before and after chemotherapy treatment. Our analysisrevealed significant cellular plasticity and alterations in the tumormicroenvironment’s cellular composition. Notably, we observed an increase intumor cell diversity coupled with reductions in T cells, B cells, and myeloid-likecells. The pre-treatment samples predominantly featured carcinoma cells in amiddle transitional state, while post-treatment samples exhibited an expandedpresence of cells in terminal, initial-to-terminal (IniTerm), and universally alteredstates. Further analysis highlighted dynamic interactions between tumor cells andimmune cells, with significant changes detected in key signaling pathways, suchas TIGIT-PVR and MDK-SDC4. This study elucidates the complex dynamics of cellplasticity in SCCE following chemotherapy, providing new insights and identifyingpotential therapeutic targets to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae84357d27c8ef8315335bde0ededde5ffcf7d8" target='_blank'>
              Single-cell RNA sequencing elucidates cellular plasticity in esophageal small cell carcinoma following chemotherapy treatment
              </a>
            </td>
          <td>
            Qinkai Zhang, Ziyu Gao, Ru Qiu, Jizhao Cao, Chunxiao Zhang, Wei Qin, Meiling Yang, Xinyue Wang, Ciqiu Yang, Jie Li, Dongyang Yang
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The metastasis of solid tumors hinges on cancer cells navigating through complex three-dimensional tissue environments, characterized by mechanical heterogeneity and biological diversity. This process is closely linked to the dynamic migration behavior exhibited by cancer cells, which dictates the invasiveness of tumors. In our study, we investigate tumor spheroids composed of breast cancer cells embedded in three-dimensional (3D) collagen matrices. Through a combination of quantitative experiments, artificial-intelligence-driven image processing, and mathematical modeling, we uncover rapid transitions in cell phenotypes and phenotype-dependent motility among disseminating cells originating from tumor spheroids. Persistent invasion leads to continuous remodeling of the extracellular matrix surrounding the spheroids, altering the landscape of migration phenotypes. Consequently, filopodial cells emerge as the predominant phenotype across diverse extracellular matrix conditions. Our findings unveil the complex mesoscale dynamics of invading tumor spheroids, shedding light on the complex interplay between migration phenotype plasticity, microenvironment remodeling, and cell motility within 3D extracellular matrices.




 Published by the American Physical Society
 2024


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afedef8827a246c2a67d180498da9a907634ab61" target='_blank'>
              Coupled Dynamics in Phenotype and Tissue Spaces Shape the Three-Dimensional Cancer Invasion
              </a>
            </td>
          <td>
            Austin Naylor, Maximilian Libmann, Izabel Raab, W.-J. Rappel, Bo Sun
          </td>
          <td>2024-12-24</td>
          <td>PRX Life</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The prevalent intra- and intertumoral heterogeneity results in undesirable prognosis and therapy failure of pancreatic cancer, potentially resulting from cellular senescence. Herein, integrated analysis of bulk and single-cell RNA-seq profiling was conducted to characterize senescence-based heterogeneity in pancreatic cancer. Publicly available bulk and single-cell RNA sequencing from pancreatic cancer patients were gathered from TCGA-PAAD, PACA-AU, PACA-CA, and GSE154778 datasets. The activity of three senescence-related pathways (cell cycle, DNA repair, and inflammation) was scored utilizing ssGSEA algorithm. A series of functional verifications of crucial genes were accomplished in patient tissue and pancreatic cancer cells. Based upon them, unsupervised clustering analysis was executed to classify pancreatic cancer samples into distinct senescence-based clusters at the bulk and single-cell levels. For single-cell transcriptome profiling, cell clustering and annotation were implemented, and malignant cells were recognized utilizing infercnv algorithm. Two senescence-based clusters were established and highly reproducible at the bulk level, with the heterogeneity in prognosis, clinicopathological features, genomic CNVs, oncogenic pathway activity, immune microenvironment and immune checkpoints. Senescence-relevant gene CHGA, UBE2C and MCM10 were proved to correlate with the migration and prognosis of pancreatic cancer. At the single-cell level, seven cell types were annotated, comprising ductal cells 1, ductal cells 2, fibroblasts, macrophages, T cells, stellate cells, and endothelial cells. The senescence-based classification was also proven at the single-cell level. Ductal cells were classified as malignant cells and non-malignant cells. In the tumor microenvironment of malignant cells, hypoxia and angiogenesis affected senescent phenotype. The heterogeneity in senescence was also observed between and within cell types. Altogether, our findings unveil that cellular senescence contributes to intra- and intertumoral heterogeneity in pancreatic cancer, which might facilitate the development of therapeutics and precision therapy in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2742fe352ef25d18eeda7d59fe2c6a8314ffc13" target='_blank'>
              Deciphering the senescence-based tumoral heterogeneity and characteristics in pancreatic cancer: Results from parallel bulk and single-cell transcriptome data.
              </a>
            </td>
          <td>
            Yeting Lu, Shuo Han, Jing Hu, Kaiji Lv, Y. Ruan, Gong Cheng, Jing Zhang, Xiang Wu, Zeming Weng, Xinhua Zhou
          </td>
          <td>2025-01-01</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb406f40baccfe449ee7910f107c5384df35a006" target='_blank'>
              Spatially resolved transcriptomics reveals gene expression characteristics in uveal melanoma
              </a>
            </td>
          <td>
            Jing-Ying Xiu, Yu-Ning Chen, Ya-Li Mao, Jingqiu Luo, Hao-Wen Li, Yang Li, Wenbin Wei
          </td>
          <td>2025-01-15</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b4799cdd384521f6d3933fff3e76a51c122a658" target='_blank'>
              Differential somatic coding variant landscapes between laser microdissected luminal epithelial cells from canine mammary invasive ductal solid carcinoma and comedocarcinoma
              </a>
            </td>
          <td>
            Vivi Deckwirth, S. Hundi, M. Hytönen, Sari Hannula, Pekka Ellonen, Pia Björkenheim, A. Sukura, H. Lohi
          </td>
          <td>2024-12-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="BACKGROUND
Growth hormone-secreting pituitary adenomas (GHPA) display diverse biological behaviors and clinical outcomes, necessitating the identification of tumor heterogeneity and prognostically relevant markers.


METHODS
In this study, we performed single-cell RNA sequencing (scRNA-seq) on 10 GHPA samples, four of which also underwent spatial transcriptome sequencing, and used scRNA-seq data from four normal pituitary samples as controls. Cell subtype characterization in GHPA was analyzed using multiple algorithms to identify malignant bias regulators, which were then validated using a clinical cohort.


RESULTS
We constructed the first single-cell and spatial transcriptome profiles of GHPA, which contained 87,862 cells and revealed 16 tumor cell subtypes. Among the tumor cells, we identified distinct developmental trajectories and three malignant-biased subtypes (PIT1_C05, PIT1_C06, and PIT1_C10). The spatial distribution characteristics of these malignant-biased cells may influence the growth characteristics and prognosis of GHPA. We screened specific regulatory transcription factors, including FOXO1, GTF2IRD1, and MAX. Clinical cohort validation indicated that FOXO1 might be associated with tumor invasion and progression, while high expression of MAX could result in poor endocrine outcomes.


CONCLUSION
GHPA exhibits rich heterogeneity and diverse cell subtypes, with specific transcription factors potentially regulating cell malignant bias, thereby influencing tumor characteristics and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a105784b2d156549c656072efae7d8dac904126" target='_blank'>
              Single-cell and spatial transcriptomic analyses reveal heterogeneity characteristics and specific cell subtype regulators in growth hormone-secreting pituitary adenomas.
              </a>
            </td>
          <td>
            Yu Zhang, Lifeng Wang, Xingcheng Yi, Xinyu Ma, Hongyu Wu, Mingzhao Zhang, Zhenqi Yang, Lizhen Ma, Zenghua Mi, Wei-jia Zhi, Cong Fu, Pinan Liu, Zhijun Yang
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="A comprehensive understanding of tumor heterogeneity, tumor microenvironment and the mechanisms of drug resistance is fundamental to advancing breast cancer research. While single-cell RNA sequencing has resolved the issue of “temporal dynamic expression” of genes at the single-cell level, the lack of spatial information still prevents us from gaining a comprehensive understanding of breast cancer. The introduction and application of spatial transcriptomics addresses this limitation. As the annual technical method of 2020, spatial transcriptomics preserves the spatial location of tissues and resolves RNA-seq data to help localize and differentiate the active expression of functional genes within a specific tissue region, enabling the study of spatial location attributes of gene locations and cellular tissue environments. In the context of breast cancer, spatial transcriptomics can assist in the identification of novel breast cancer subtypes and spatially discriminative features that show promise for individualized precise treatment. This article summarized the key technical approaches, recent advances in spatial transcriptomics and its applications in breast cancer, and discusses the limitations of current spatial transcriptomics methods and the prospects for future development, with a view to advancing the application of this technology in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9484c611ac13b7e4460cc6b1b5eb2c6b25e4a9" target='_blank'>
              Spatial transcriptomics in breast cancer: providing insight into tumor heterogeneity and promoting individualized therapy
              </a>
            </td>
          <td>
            Junsha An, Yajie Lu, Yuxi Chen, Yuling Chen, Zhaokai Zhou, Jianping Chen, Cheng Peng, Ruizhen Huang, Fu Peng
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The Imaging and Molecular Annotation of Xenografts and Tumors Cancer Grand Challenges team was set up with the objective of developing the "next generation" of pathology and cancer research by using a combination of single-cell and spatial omics tools to produce 3D molecularly annotated maps of tumors. Its activities overlapped, and in some cases catalyzed, a spatial revolution in biology that saw new technologies being deployed to investigate the roles of tumor heterogeneity and of the tumor micro-environment. See related article by Stratton et al., p. 22 See related article by Bhattacharjee et al., p. 28 See related article by Goodwin et al., p. 34.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9c9c61637383cf6c8f3f96d10f0ca3cac35ac4d" target='_blank'>
              Cancer Research in the Age of Spatial Omics: Lessons from IMAXT.
              </a>
            </td>
          <td>
            D. Bressan, Nicholas Walton, Gregory J. Hannon
          </td>
          <td>2025-01-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be893eec80147faf7d7cca1af35e6472e40e7cc" target='_blank'>
              Statistical identification of cell type-specific spatially variable genes in spatial transcriptomics
              </a>
            </td>
          <td>
            Lulu Shang, Peijun Wu, Xiang Zhou
          </td>
          <td>2025-01-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Summary Cellular stemness is a cells ability to self-renew and differentiate into specialized cell types. In spatial transcriptomics (ST), researchers can use gene activity to quantify stemness and the associated spatial locations, allowing for the investigation of cellular profiles in their natural context. However, many ST platforms provide data with low resolution, and each data spot (i.e., spatial location) contains multiple cell types, which hinders the investigation of cell type–specific (CTS) behaviors, such as cell stemness, at different spatial locations. We developed a bivariate kernel-weighted regression method with constrained optimization to estimate CTS stemness as a function of spatial location and developed accompanying visualization tools. Through simulation studies and an application to real breast cancer ST data from the 10x Genomics Visium platform, we demonstrated that our method can accurately estimate CTS stemness and help shed light on the interplay among cell type, tissue structure, and stemness. Availability and implementation An R Shiny app implementing the proposed method is available at GitHub (https://github.com/ericli0480/stemness-deconvolution/). Supplementary information Available in a separate PDF file">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72f608c82ad15c7c6476ead9f76633ad0bf3a052" target='_blank'>
              Delineating the Spatial Patterns of Cell Type–Specific Stemness Using a Statistical Deconvolution Model
              </a>
            </td>
          <td>
            Eric J. Li, Liang Li, Ziyi Li
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Although invasiveness is one of the major determinants of the poor glioblastoma (GBM) outcome, the mechanisms of GBM invasion are only partially understood. Among the intrinsic and environmental processes promoting cell-to-cell interaction processes, eventually driving GBM invasion, we focused on the pro-invasive role played by Extracellular Vesicles (EVs), a heterogeneous group of cell-released membranous structures containing various bioactive cargoes, which can be transferred from donor to recipient cells.


METHODS
EVs isolated from patient-derived GBM cell lines and surgical aspirates were assessed for their pro-migratory competence by spheroid migration assays, calcium imaging, and PYK-2/FAK phosphorylation. Brain invasiveness was investigated in human cortical organoids-based assembloids and in vivo orthotopic xenografts. EV molecular features were specified by multiplex bead-based flow cytometry.


RESULTS
Results unveil a self-sustaining mechanism triggering migration through autocrine release and engagement of a specific population of EVs of large size (L-EVs), isolated from either patient-derived cell lines or surgical aspirates. L-EVs act through modulation of calcium transients via Connexin 43-Gap Junctions (Cx43-GJ) and phospho-activation of PYK2. Pre-incubation with blocking antibodies targeting Cx43 hemichannels demonstrated a dose-dependent inhibition of the L-EV-mediated GBM migration. By exploiting patients' surgical aspirates, we show that only L-EVs deriving from tumoral cells, and not those with immune origin, promote tumor migration, impacting more prominently the tumoral cells with mesenchymal subtype.


CONCLUSIONS
We demonstrate that L-EVs released by GBM cells, but not by the immune cells of the tumor microenvironment, represent a relevant and unique autocrine pro-migratory input for the tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f87a08f8bbc4e6c1b435e86c434ceec17830431" target='_blank'>
              Extracellular Vesicles Released by Glioblastoma Cancer Cells Drive Tumor Invasiveness via Connexin-43 Gap Junctions.
              </a>
            </td>
          <td>
            M. Tamborini, Valentino Ribecco, Elisabetta Stanzani, Arianna Sironi, Monica Tambalo, Davide Franzone, Elena Florio, Edoardo Fraviga, Chiara Saulle, M. Gagliani, Marco Pizzocri, M. Mattioli, Katia Cortese, Jean X Jiang, Giuseppe Martano, L. Politi, Marco Riva, Federico Pessina, Davide Pozzi, Simona Lodato, L. Passoni, Michela Matteoli
          </td>
          <td>2025-01-30</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc23d3d27f11aa0ad92c216de43831771b9dae4" target='_blank'>
              Differential cell architecture and microenvironmental responses of pretumoral and tumoral cellular models exposed to coverslip-induced hypoxia.
              </a>
            </td>
          <td>
            Magdalena Millán, Felipe Parietti, Florencia Lamela, M. C. De Rossi, Belén Benítez, Valeria Levi, Manoela Domingues, R. Bologna-Molina, M. Arocena, J. Hochmann
          </td>
          <td>2025-01-03</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, Kevin Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f22ff573111b3740a2cb886527997d19806de8b" target='_blank'>
              Single-cell RNA sequencing reveals tumor heterogeneity in small cell neuroendocrine cervical carcinoma
              </a>
            </td>
          <td>
            Xuesong Xiang, Xiang Tao, Keqin Hua, Hua Jiang, Jingxin Ding
          </td>
          <td>2025-02-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The spatial organization of cells plays a pivotal role in shaping tissue functions and phenotypes in various biological systems and diseased microenvironments. However, the topological principles governing interactions among cell types within spatial patterns remain poorly understood. Here, we present the Triangulation Cellular Community Motif Neural Network (TrimNN), a graph-based deep learning framework designed to identify conserved spatial cell organization patterns, termed Cellular Community (CC) motifs, from spatial transcriptomics and proteomics data. TrimNN employs a semi-divide-and-conquer approach to efficiently detect over-represented topological motifs of varying sizes in a triangulated space. By uncovering CC motifs, TrimNN reveals key associations between spatially distributed cell-type patterns and diverse phenotypes. These insights provide a foundation for understanding biological and disease mechanisms and offer potential biomarkers for diagnosis and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c80ec316e2753068ced3597143b442f6a178a7b" target='_blank'>
              TrimNN: Characterizing cellular community motifs for studying multicellular topological organization in complex tissues
              </a>
            </td>
          <td>
            Yang Yu, Shuang Wang, Jinpu Li, Meichen Yu, Kyle McCrocklin, Jing-Qiong Kang, A. Ma, Qin Ma, Dong Xu, Juexin Wang
          </td>
          <td>2024-12-22</td>
          <td>Research Square</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Breast cancer heterogeneity has been well understood based solely on tumor components. However, the tumor microenvironment, which plays a crucial role in cancer by regulating tumor cells’ differentiation, maturation, and malignant potential, remains under-explored. This study conducted high-throughput RNA-Seq analysis on fresh breast cancer tissues from each molecular subtype and a pure population of cancer-associated fibroblasts (CAFs) isolated from cancer tissue and adjacent normal parts. Based on their functions, we identified three populations of CAFs: a) immune response-related, b) ECM remodeling, and c) calcium/protein binding. Validation of differentially expressed genes among these CAF populations revealed subtype-specific correlations. For instance, CAFs expressing immune response-related genes were significantly enriched in Luminal A/B and TNBC (log +1-fold; p =0.00047), compared to Her-2 positive (log -2.3-fold; p =0.0026). ECM remodeling genes exhibited greater expression in Her-2 positive and TNBC (log +4 to +11-fold; p =7.87E-08) when compared to Luminal subtypes (log +1.5-fold to +6-fold; p =8.66E-05). Calcium binding genes displayed overall similar expression across breast cancer subtypes. Thus, this study identified and underscored the functional heterogeneity of CAFs within the tumor microenvironment across the molecular subtypes of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e870b934402b9a2b10afe0f5e6b9ec8fa60b89" target='_blank'>
              Deciphering Functional Heterogeneity of Cancer-Associated Fibroblasts Across Molecular Subtypes of Breast Cancer
              </a>
            </td>
          <td>
            D. Kashyap
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Transgenic mice and organoid models, such as three-dimensional tumoroid cultures, have emerged as powerful tools for investigating cancer development and targeted therapies. Yet, the extent to which these preclinical models recapitulate the cellular identity of heterogeneous malignancies, like neuroblastoma (NB), remains to be validated. Here, we characterized the transcriptional landscape of TH-MYCN tumors by single-cell RNA sequencing (scRNA-seq) and developed ex vivo tumoroids. Integrated analysis with murine fetal adrenal samples confirmed that both TH-MYCN tumors and tumoroids closely mirror the cellular profiles of normal embryonic sympathoblasts and chromaffin cells. Comprehensive comparison between tumors from NB patients and TH-MYCN mice demonstrated similarities in adrenergic tumor cell composition. Ex vivo tumoroid cultures displayed histological resemblance and shared transcriptional profiles with the originating TH-MYCN tumors and human NB. Importantly, subpopulations within tumoroids exhibited gene expression associated with poor NB patient survival. Notably, recurrent observations of a low-proliferative chromaffin phenotype connected to the highly proliferative sympathetic phenotype suggested that pushing sympathoblasts into a chromaffin-like state may offer an interesting therapeutic strategy for NB. Together, this study not only deepens our understanding of a widely used transgenic mouse NB model but also introduces an ex vivo model that maintains critical adrenergic cell state identity, thereby enhancing its translational potential for NB research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5619b05ffd717eb9d8928b0e83ee66ca9cda8b" target='_blank'>
              Comparative Single-Cell Transcriptomics of Human Neuroblastoma and Preclinical Models Reveals Conservation of an Adrenergic Cell State.
              </a>
            </td>
          <td>
            Bethel Tesfai Embaie, Hirak Sarkar, A. Alchahin, Jörg Otte, T. Olsen, Conny Tümmler, Polina Kameneva, Artem V Artemov, N. Akkuratova, I. Adameyko, J. Stukenborg, Malin Wickström, P. Kogner, J. Johnsen, Shenglin Mei, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="
 Introduction: The molecular pathways involved in the response to radiation therapy (RT) in pancreatic ductal adenocarcinoma (PDAC) remain poorly understood. Attempts to combine RT with immunotherapies in PDAC have yielded inconsistent outcomes, highlighting a critical gap in our understanding of immune dynamics and mechanisms influencing RT response. Our study aims to uncover molecular pathways and cellular subtypes associated with radioresponse. To do so, we developed a novel cell-cell interaction platform, "SpaCCI", to map ligand-receptor (LR) interactions, essential for understanding cellular communication within the TME and its contribution to therapeutic resistance. Methods: Single-cell RNA sequencing (scRNA-seq) was performed on 16 matched pre- and post-treatment PDAC biopsies from a Phase I/II dose-escalation clinical trial of stereotactic body radiation therapy. Additionally, spatial transcriptomics (ST) was conducted on resected PDAC (20 treated with neoadjuvant chemoradiation, 8 untreated). We integrated scRNA-seq and ST datasets using cell type decomposition and developed SpaCCI to enhance LR interaction detection in tissues with complex cell mixtures. Results: ScRNA-seq identified five molecular subpopulations of cancer cells associated to RT response, with PDAC_3 enriched post-RT and marked by genes involved in lipid metabolism and ferroptosis defense. Post-RT depleted subpopulations, such as PDAC_0 and PDAC_prolif, saw upregulation of pathways related to rapid proliferation and oxidative phosphorylation. In a murine model, PDAC_3 was similarly enriched in post-RT tumors, with persister cell lines from these tumors showing overexpression of ferroptosis defense genes and resistance to ferroptosis inducers including Erastin and RSL3. ST analysis also revealed spatial immune changes post-RT, with increased cytotoxic and effector memory T cells balanced by regulatory T cells and M2-like macrophages, indicating immune escape. NK and memory B cells correlated with improved survival post-RT, while plasma IgG cells localized outside the tumor bed, suggesting compartmentalized immune responses. Using SpaCCI, we observed differential enrichment of ligand-receptor interactions in RT-treated tumors, particularly among lipid-associated tumor macrophages, cancer cells, myofibroblast-like cancer-associated fibroblasts, and endothelial cells. Persister PDAC_3 subpopulations showed enriched LR SEMA3 signaling with CAFs, myeloid cells, and endothelial cells consistent with axon guidance pathways. Conclusions: This study presents a molecular and cellular atlas of PDAC response to RT, revealing persister cells populations with adaptations in lipid metabolism and ferroptosis defense that could be targeted to enhance RT efficacy. ST highlighted complex immune dynamics within the TME demonstrating a balance between inflammatory and immunosuppressive populations. Our findings provide a foundation for combinatorial therapeutic strategies and suggest that axon guidance pathways may also play an important role in the response to RT in PDAC.
 Citation Format: Vincent Bernard, Li-Ting Ku, Tianyu Wang, Kimal Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, Manoop S. Bhutani, Paola A. Guerrero, Ethan B. Ludmir, Albert C. Koong, Cullen M. Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay.Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P018">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd18fd5a66ac51a4015b7b7d9504296b9a16919" target='_blank'>
              Abstract P018: Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, K. Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, M. Bhutani, Paola A. Guerrero, E. Ludmir, Albert C Koong, Cullen M Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Lung adenocarcinoma, the most common subtype of lung cancer, is genomically complex, with tumors containing tens to hundreds of non-synonymous mutations. However, little is understood about how genes interact with each other to enable the evolution of cancer in vivo, largely due to a lack of methods for investigating genetic interactions in a high-throughput and quantitative manner. Here, we employed a novel platform to generate tumors with inactivation of pairs of ten diverse tumor suppressor genes within an autochthonous mouse model of oncogenic KRAS-driven lung cancer. By quantifying the fitness of tumors with every single and double mutant genotype, we show that most tumor suppressor genetic interactions exhibited negative epistasis, with diminishing returns on tumor fitness. In contrast, Apc inactivation showed positive epistasis with the inactivation of several other genes, including synergistic effects on tumor fitness in combination with Lkb1 or Nf1 inactivation. Sign epistasis was extremely rare, suggesting a surprisingly accessible fitness landscape during lung tumorigenesis. These findings expand our understanding of the evolutionary interactions that drive tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e03ac1e8a6ee59b9db5be527feeaddbf8b397192" target='_blank'>
              Combinatorial in vivo genome editing identifies widespread epistasis and an accessible fitness landscape during lung tumorigenesis.
              </a>
            </td>
          <td>
            Jess D. Hebert, Yuning J Tang, Márton Szamecz, Laura Andrejka, Steven S. Lopez, D. Petrov, Gábor Boross, M. Winslow
          </td>
          <td>2025-02-05</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Immunologically inert or cold tumors pose a substantial challenge to the effectiveness of immunotherapy. The use of oncolytic viruses (OVs) to induce immunogenic cell death (ICD) in tumor cells is a well-established strategy for initiating the cancer immunity cycle (CIC). This process promotes the trafficking and infiltration of CD8+ T cells into tumors, thereby eliciting a tumor-specific immune response. Despite the potential of OVs for handling cold tumors, clinical outcomes have fallen short of expectations. To better understand the obstacles faced by oncolytic virus immunotherapy (OVI), we would like to revisit the OV issue. Growing evidence indicates that limited intratumoral penetration and inadequate intratumoral distribution of OVs are critical factors contributing to the suboptimal response to OVI. Aberrant expressions of matrix proteins by cancer-associated fibroblasts (CAFs) alter the mechanical properties of the tumor extracellular matrix (ECM). This results in increased ECM desmoplasia and elevated intratumoral interstitial fluid pressure (IFP), creating physical barriers that impede the penetration and dissemination of OVs within tumors. This review explores the latest advancements in strategies designed to improve the intratumoral penetration of OVs to facilitate the penetration of tumor-infiltrating lymphocytes (TILs) into cold tumors. Additionally, we investigated current clinical trials and challenges associated with translating these strategies into clinical practice to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bbdf79e7c5140e773e4af9072a8e1b4cc30733a" target='_blank'>
              Extracellular matrix re-normalization to improve cold tumor penetration by oncolytic viruses
              </a>
            </td>
          <td>
            Geofrey F. Soko, Benson K. Kosgei, Stephene S. Meena, Ying Jing Ng, Huihui Liang, Bing Zhang, Qingjun Liu, Tielong Xu, Xinju Hou, Ray P. S. Han
          </td>
          <td>2025-01-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary Melanoma is an aggressive malignancy defined by significant intratumoral heterogeneity, driving its capacity for therapeutic resistance and recurrence. This study adopts a systems-level approach to dissect the melanoma microenvironment, focusing on intricate interactions between malignant cells and immune infiltrates. We identified critical regulatory networks and intercellular communication pathways that appear to influence disease progression. These findings highlight the dynamic interplay between tumor-intrinsic factors and the surrounding microenvironment, revealing potential mechanisms underlying immune evasion and therapy resistance. By mapping these complex interactions, the present study builds on our foundation for precision-based therapeutic strategies tailored to the unique biological landscape of melanoma, offering promise for improved clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f383353b5d7dd45f21551edcc7dd626f8651941" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA Data Reveals Complexity and Significance of the Melanoma Interactome
              </a>
            </td>
          <td>
            Michael J Diaz, J. T. Tran, Arthur M. Samia, M. Forouzandeh, Jane M. Grant-Kels, Marjorie E. Montanez-Wiscovich
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaacb86999aeed003c707b4000adb56d67bb6547" target='_blank'>
              Spatial transcriptome profiling identifies DTX3L and BST2 as key biomarkers in esophageal squamous cell carcinoma tumorigenesis
              </a>
            </td>
          <td>
            Rutao Li, Na Li, Qianqian Yang, Xing Tong, Wei Wang, Chang Li, Jun Zhao, Dong Jiang, Haitao Huang, Chen Fang, Kai Xie, Jiamin Yuan, Shaomu Chen, Guangbin Li, Haitao Luo, Zhibo Gao, Dongfang Wu, Xiaoli Cui, Wei Jiang, Lingchuan Guo, Haitao Ma, Yu Feng
          </td>
          <td>2024-12-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The evolution of metastasis, the spread of cancer to distal sites within the body, represents a lethal stage of cancer progression. Yet, the evolutionary dynamics that shape the emergence of metastatic disease remain unresolved. Here, using single-cell lineage tracing data in combination with phylogenetic statistical methods, we show that the evolutionary trajectory of metastatic disease is littered with bursts of rapid molecular change as new cellular subpopulations appear, a pattern known as punctuational evolution. Next, by measuring punctuational evolution across the metastatic cascade, we show that punctuational effects are concentrated within the formation of secondary tumours at distal metastatic sites, suggesting that qualitatively different modes of evolution may drive primary and metastatic tumour progression. Taken as a whole, our findings provide empirical evidence for distinct patterns of molecular evolution at early and late stages of metastatic disease and our approach provides a framework to study the evolution of metastasis at a more nuanced level than has been previously possible.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19eea86b0534290bd81992d00ef5caf3aad45d15" target='_blank'>
              Punctuational evolution is pervasive in distal site metastatic colonization
              </a>
            </td>
          <td>
            George Butler, Sarah R. Amend, C. Venditti, Kenneth J Pienta
          </td>
          <td>2025-01-01</td>
          <td>Proceedings of the Royal Society B: Biological Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The resistance of tumors to immune checkpoint inhibitors (ICI) may be intricately linked to cellular senescence, although definitive clinical validation remains elusive. In this study, comprehensive pan-cancer scRNA-seq analyses identify fibroblasts as exhibiting the most pronounced levels of cellular senescence among tumor-associated cell populations. To elucidate this phenomenon, a fibroblast senescence-associated transcriptomic signature (FSS), which correlated strongly with protumorigenic signaling pathways and immune dysregulation that fosters tumor progression, is developed. Leveraging the FSS, the machine learning (ML) framework demonstrates exceptional accuracy in predicting ICI response and survival outcomes, achieving superior area under curve (AUC) values across validation, testing, and in-house cohorts. Strikingly, FSS consistently outperforms established signatures in predictive robustness across diverse cancer subtypes. From an integrative analysis of 17 CRISPR/Cas9 libraries, CDC6 emerges as a pivotal biomarker for pan-cancer ICI response and prognostic stratification. Mechanistically, experimental evidence reveals that CDC6 in tumor cells orchestrates fibroblast senescence via TGF-β1 secretion and oxidative stress, subsequently reprogramming the tumor microenvironment and modulating ICI response. These findings underscore the translational potential of targeting fibroblast senescence as a novel therapeutic strategy to mitigate immune resistance and enhance antitumor efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83eb29659bd661ae208e4642398f8b3b7069e404" target='_blank'>
              A Distinguished Roadmap of Fibroblast Senescence in Predicting Immunotherapy Response and Prognosis Across Human Cancers.
              </a>
            </td>
          <td>
            Dongjie Chen, Pengyi Liu, Jiayu Lin, Longjun Zang, Yihao Liu, Shuyu Zhai, Xiongxiong Lu, Yuanchi Weng, Hongzhe Li
          </td>
          <td>2024-12-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 218


 Background:
 Peritoneal metastases (PM) in colorectal cancer (CRC) portend a poor prognosis. We sought to elucidate spatially resolved molecular features facilitating transcoelomic dissemination and progression.
 Methods:
 50 PT, 116 PM and 37 primary normal samples from 96 patients were retrieved and profiled with digital spatial profiling (GeoMx DSP, Nanostring Technologies). Through unsupervised clustering, we characterized the microenvironment of tumor-stroma compartments and studied the roles of stromal phenotypes in promulgating tumorigenesis. These findings were orthogonally validated through spatial proteomics (multiplex IHC [mIHC] - COMET, Lunaphore Technologies) of 10 paired PT-PM samples retrieved from 6 patients.
 Results:
 A median of 37 region of interests (ROIs) (IQR: 32-38) were selected from a total of 9 tissue microarray (TMA) slides. A total of 269 ROIs (181 tumor, 88 stroma) from PM samples, 45 ROIs (34 tumor, 11 stroma) from PT samples and 28 (all stroma) ROIs from primary normal samples were retrieved and profiled. Through consensus clustering of stroma ROIs, we identify a fibro-collagenous and immune infiltrated stromal phenotype (stromal cluster [SC] 2) characterized by increased cancer associated fibroblasts (CAFs), memory B cells, M2 macrophages and T-cell exhaustion. Patients with SC2 stroma had poorer survival (p=0.036). SC2 stroma was also observed to support adjacent tumor compartments with enriched oncogenic pathways such as TGF-beta, TNF-alpha, hypoxia and JAK-STAT. Through discriminatory gene expression profiles, we developed a 20-gene composite bidirectional signature of SC2. The prognostic significance of the SC2-signature was externally validated through several cohorts including TCGA and MSI-H only cohorts. Next, by inspecting mIHC retrieved immune cell type densities across ROIs, we confirmed a similar phenomenon in which two distinct stromal clusters were found – one being fibroblast infiltrated but T-cell depleted (SC2-like, n=67 ROI) and another which is T-cell infiltrated but fibroblast depleted (SC1-like, n=104 ROI). Diverging spatial distributions of T-cells and fibroblasts between SC1 and SC2-like stroma was appreciated as well. Between both PT and PM, we find close clustering of T cells in SC1-like stromal compartments and fibroblasts in SC2-like stromal compartments.
 Conclusions:
 We describe SC2, a pro-tumorigenic stromal phenotype characterized by increased CAFs, T cell exhaustion and is associated with poor prognosis in CRC PM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76074158017484a4495c691d7f74783fe9c4e65" target='_blank'>
              Diverse stromal phenotypes at spatial resolution in colorectal cancer peritoneal metastasis.
              </a>
            </td>
          <td>
            Joseph J. Zhao, J. Ong, Ying Liu, Supriya Srivastava, Melissa Ching Ching Teo, Joey Jia-Ying Lee, Robert Walsh, Lit-Hsin Loo, Qiu Xuan Tan, Gillian Ng, J. W. Tan, Haoran Ma, Xuewen Ong, S. Tay, T. Sheng, Jeffrey H Y Lum, W. Yong, F. Pietrantonio, P. Tan, R. Sundar
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is the most common and aggressive type of primary brain cancer, known for its rapid growth, invasion into surrounding brain tissue, and strong resistance to current therapies. Despite intensive treatment efforts, patients have a median survival of just 15 months. One major treatment challenge is the tumor’s complex vascular network which causes the limited success of antiangiogenic therapies in extending patients’ survival. Pericytes are cells essential for maintaining the integrity of this vascular network. Additional to their structural functions, they play diverse roles that support tumor progression and treatment resistance. This review aims to understand the multifaced functions of pericytes in the glioblastoma microenvironment; the comprehension of those roles may guide new therapeutic strategies to improve outcomes for patients affected with this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/835c71d63c5567604bb678c8f9634fc408771476" target='_blank'>
              Pericytes in Glioblastoma: Hidden Regulators of Tumor Vasculature and Therapy Resistance
              </a>
            </td>
          <td>
            Irene Salazar-Saura, María Pinilla-Sala, J. Megías, Lara Navarro, Esther Roselló-Sastre, T. San-Miguel
          </td>
          <td>2024-12-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54d1904727e65e93a89095c3025ebc1f384c120" target='_blank'>
              A genome-wide atlas of human cell morphology.
              </a>
            </td>
          <td>
            Meraj Ramezani, Erin Weisbart, J. Bauman, Avtar Singh, John Yong, Maria Lozada, Gregory P. Way, Sanam L. Kavari, Celeste Diaz, Eddy Leardini, Gunjan Jetley, Jenlu Pagnotta, Marzieh Haghighi, Thiago M. Batista, Joaquín Pérez-Schindler, M. Claussnitzer, Shantanu Singh, B. Cimini, P. Blainey, A. Carpenter, Calvin H. Jan, James T Neal
          </td>
          <td>2025-01-27</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Aneuploidy, the major cause of miscarriages in humans, is pervasive in early embryos but robustly dampened during development, allowing for healthy births. Later in life, aneuploidy correlates with pathological conditions including cancer. Identification of the mechanisms underlying the elimination of aneuploid cells is relevant in development and disease. Here we generated cells carrying molecularly defined segmental monosomies and trisomies and characterized their immediate impact on cellular behavior. Our data reveal signs of out-competition of cells carrying monosomies in genomic regions devoid of previously known haploinsufficient genes. Dose-dependent effects of single genes or a discrete number of genes contribute to the observed cellular behaviors. By simultaneously inducing cells carrying monosomies and trisomies of the same genomic location, we present evidence that segmental trisomies potentiate or alleviate the negative effects of the monosomy on growth, thus revealing a key role of cell interactions in defining the in vivo elimination of aneuploid cells. Highlights Segmental monosomies cause growth impairment and are out-competed. Growth impairment relies on dose-dependent effects of single genes or a discrete number of genes. Segmental trisomies of up to 1500 genes do not have a major impact on proliferation and survival. Cell competition is modulated by the presence of cells trisomic for the same genomic region. eTOC Blurb Aneuploidy has a negative impact on the growth and proliferation of all animal cells analyzed so far. Fusari et al. unravel a role of cell interactions in defining the in vivo elimination of aneuploid cells through cell competition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc128d47b336584a75037f0f1e1bbbd450a62027" target='_blank'>
              Depletion of aneuploid cells is shaped by cell-to-cell interactions
              </a>
            </td>
          <td>
            Elena Fusari, M. Muzzopappa, Juliette Gracia, Marco Milán
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Despite the vast diversity of life forms and living histories, it appears that all branches of Metazoa face the challenge of tumor growth. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we first identified the rise of a glycolytic cell population over time, resembling the Wartburg effect observed in human tumors. We further identified two key metabolic changes in these fly tumors. First, a systemic depletion of acetyl-CoA leads to a reduction in histone acetylation levels and stochastic silencing of actively-transcribed genes. Second, defects in the methionine cycle cause a systemic depletion of S-Adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process strongly inhibits tumor growth. Finally, to understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors, and identified human tumors that exhibit metabolic signatures similar to those of fly tumors. We found that human tumors with high metabolic similarity to fly tumors have a lower mutational load, younger patient age, and lower DNA methylation levels. This study suggests that tumorigenesis processes have a deep evolutionary origin and highlights that depletion of key metabolites is an evolutionarily-ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe6bf208acb7d71c056481ed083b94ace1ace601" target='_blank'>
              Epigenetic reprogramming induced by Acetyl-CoA and SAM depletion is an evolutionarily-ancient path to malignant growth
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Despite the pivotal role of cytotoxic T lymphocytes (CTLs) in anti-tumor immunity, a substantial proportion of CTL-rich hepatocellular carcinoma (HCC) patients experience early relapse or immunotherapy resistance. However, spatial immune variations impacting the heterogeneous clinical outcomes of CTL-rich HCCs remain poorly understood. Here, we compared the single-cell and spatial landscapes of 20 CTL-rich HCCs with distinct prognoses using multiplexed in situ staining and validated the prognostic value of myeloid spatial patterns in a cohort of 386 patients. Random forest and Cox regression models identified macrophage aggregation as a distinctive spatial pattern characterizing a subset of CTL-rich HCCs with an immunosuppressive microenvironment and poor prognosis. Integrated analysis of single-cell and spatial transcriptomics, combined with in situ staining validation, revealed that spatial aggregation enhanced pro-tumoral macrophage reprogramming in HCCs, marked by lipid metabolism orientation, M2-like polarization, and increased adjacent CTL exhaustion. This spatial effect on macrophage reprogramming was replicated in HCC-conditioned human macrophage cultures, which showed an enhanced capability to suppress CTLs. Notably, increased macrophage aggregation was associated with higher response rates to anti-PD-1 immunotherapy. These findings suggest that the spatial distribution of macrophages is a biomarker of their functional diversities and microenvironment status, which holds prognostic and therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0a3e561fbd74941c09e71f727074ba04c903f0f" target='_blank'>
              Spatial Organization of Macrophages in CTL Rich Hepatocellular Carcinoma Influences CTL Antitumor Activity.
              </a>
            </td>
          <td>
            Yulan Weng, Lu Wang, Yuting Wang, Junyu Xu, Xiaoli Fan, Shufeng Luo, Qiaomin Hua, Jing Xu, Gaoteng Liu, Kai-Bo Zhao, Chang-An Zhao, Dong-Ming Kuang, Chong Wu, Limin Zheng
          </td>
          <td>2025-01-02</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd3cbd55a53252e9bb3dfd532de1bccedf85f35d" target='_blank'>
              Tumor-associated-fibrosis and active collagen-CD44 axis characterize a poor-prognosis subtype of gastric cancer and contribute to tumor immunosuppression
              </a>
            </td>
          <td>
            Yingqi Yang, Haohan Sun, Hongkai Yu, Luyao Wang, Chang Gao, Haokun Mei, Xiaomeng Jiang, Minghui Ji
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58121f921b0bd9812f510ea0edf4c0cb4c87aa52" target='_blank'>
              Spatial transcriptomic analysis drives PET imaging of tight junction protein expression in pancreatic cancer theranostics
              </a>
            </td>
          <td>
            James Wang, J. Seo, A. Kare, Martin Schneider, Mallesh Pandrala, Spencer K. Tumbale, M. Raie, Gokce Engudar, Nisi Zhang, Yutong Guo, Xiaoxu Zhong, Sofia Ferreira, Bo Wu, Laura D Attardi, G. Pratx, Andrei Iagaru, Ryan L. Brunsing, Gregory W Charville, Walter G. Park, Katherine W Ferrara
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in neoplastic initiation and progression. Epigenetic machinery, governing the expression of core oncogenes and tumor suppressor genes in transformed cells, significantly contributes to tumor development at both primary and distant sites. Recent studies have illuminated how epigenetic mechanisms integrate external cues and downstream signals, altering the phenotype of stromal cells and immune cells. This remolds the area surrounding tumor cells, ultimately fostering an immunosuppressive microenvironment. Therefore, correcting the TME by targeting the epigenetic modifications holds substantial promise for cancer treatment. This review synthesizes recent research that elucidates the impact of specific epigenetic regulations—ranging from DNA methylation to histone modifications and chromatin remodeling—on stromal and immune cells within the TME. Notably, we highlight their functional roles in either promoting or restricting tumor progression. We also discuss the potential applications of epigenetic agents for cancer treatment, envisaging their ability to normalize the ecosystem. This review aims to assist researchers in understanding the dynamic interplay between epigenetics and the TME, paving the way for better epigenetic therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea83192c32e6b6228a3b5ff261efd56060404967" target='_blank'>
              Epigenetic Regulation of Stromal and Immune Cells and Therapeutic Targets in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Kang Liu, Yue Li, Minmin Shen, Wei Xu, Shanshan Wu, Xinxin Yang, Bo Zhang, Nengming Lin
          </td>
          <td>2025-01-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Circulating tumor cells (CTCs) and CTC-clusters are pivotal in the metastatic process of breast cancer (BC). Owing to their low frequency, models replicating their biology should provide a robust platform for investigating the molecular mechanisms driving metastasis and identifying new biomarkers. We established and characterized a CTC-derived cell model from a mouse xenograft to explore its metastatic behavior and molecular profile, which allowed us to investigate the expression and prognostic significance of a set of genes associated with the metastatic potential of CTCs. Methods: The CTC line (mCTC) derived from a MDA-MB-231 mouse xenografts was used in comparative functional analyses including cell cycle evaluation, colony formation, invasion, adhesion, and metastatic competency in zebrafish models. Transcriptomic profiling and functional assays were conducted to identify candidate genes and understand their roles in metastasis. Moreover, publicly available gene expression datasets of CTCs, CTC-clusters, and tumor tissue, from GEO and TCGA, were analyzed for the identification of a gene signature that was correlated with survival data. The signature was validated in an independent cohort. Results: Compared with MDA-MB-231 cells, mCTC cells presented enhanced colony formation, invasion, and adhesion, and increased dissemination and survival in zebrafish. Transcriptomic analysis revealed that SPARC was significantly upregulated. Functional assays showed that SPARC overexpression was correlated with increased invasion and migration. Analysis of public datasets confirmed the high expression of SPARC in BC CTCs and CTC-clusters. Additionally, a 4-gene signature involving SPARC, THBS1, VCL, and HSP90AB1 was identified that demonstrated strong prognostic value, predicting shorter overall and distant metastasis-free survival in the primary tumor setting. Validation cohorts confirmed its ability to distinguish high-risk patients. Elevated expression of the 4-gene signature in CTCs was also indicative of increased mortality risk. Conclusion: mCTC exhibit distinct metastatic traits and molecular characteristics, highlighting a possible role of SPARC in CTC biology and its potential as a prognostic marker in BC metastasis. The identified 4-gene signature provides a robust prognostic tool for assessing patient risk and guiding therapeutic strategies. Further investigations into the mechanistic role of SPARC may reveal new therapeutic targets for managing BC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785682c178d882b31b862712e1108b5022d056bd" target='_blank'>
              A CTC Model Uncovers Metastatic Drivers and Prognostic Markers in Breast Cancer
              </a>
            </td>
          <td>
            C. Fernández-Santiago, I. Martínez-Peña, M. Paramés, M. Rodriguez, P. Hurtado, C. Abuín, C. Costa, A. B. Dávila Ibáñez, R. López-López, R. Pineiro
          </td>
          <td>2025-01-17</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/846ec2188a7971fef054ab5147a97b56d594c528" target='_blank'>
              Crosstalk between cancer-associated fibroblasts and non-neuroendocrine tumor cells in small cell lung cancer involves in glycolysis and antigen-presenting features
              </a>
            </td>
          <td>
            Yuanhua Lu, Hui Li, P. Zhao, Xinyue Wang, Wenjun Shao, Yan Liu, L. Tian, Rui Zhong, Haifeng Liu, Ying Cheng
          </td>
          <td>2024-12-25</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer deaths in the United States. New targeted therapies against the once-deemed undruggable oncogenic KRAS are changing current therapeutic paradigms. However, resistance to targeted KRAS inhibitors almost inevitably occurs; resistance can be driven by tumor cell–intrinsic changes or by changes in the microenvironment. Here, we utilized a genetically engineered mouse model of KRASG12D-driven lung cancer that allows for inducible and reversible expression of the oncogene: activation of oncogenic KRASG12D induces tumor growth; conversely, inactivation of KRASG12D causes tumor regression. We showed that in addition to regulating cancer cell growth and survival, oncogenic KRAS regulated the transcriptional status of cancer-associated fibroblasts and macrophages in this model. Utilizing ex vivo approaches, we showed that secreted factors from cancer cells induced the expression of multiple cytokines in lung fibroblasts, and in turn drove expression of immunosuppressive factors, such as arginase 1, in macrophages. In summary, fibroblasts emerged as a key source of immune regulatory signals, and a potential therapeutic target for improving the efficacy of KRAS inhibitors in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdae42d5132ec7ce05cdb9d5dd6bd5ef386a7a36" target='_blank'>
              KRASG12D drives immunosuppression in lung adenocarcinoma through paracrine signaling
              </a>
            </td>
          <td>
            Emily L. Lasse-Opsahl, Ivana Barravecchia, Elyse McLintock, Jen-Mei Lee, Sarah F Ferris, Carlos E Espinoza, Rachael Hinshaw, Sophia Cavanaugh, Marzia Robotti, Lily Rober, Kristee L Brown, Kristena Y. Abdelmalak, C. Galbán, Timothy L. Frankel, Yaqing Zhang, M. Pasca di Magliano, Stefanie Galbán
          </td>
          <td>2025-01-09</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Spine metastases are the most common bone site for breast cancer, with evolving surgery and multidisciplinary care improving outcomes. Current treatments, including chemotherapy and invasive surgery, may damage healthy tissue and may leave residual tumors that lead to recurrence. Cold atmospheric plasma (CAP) offers a non-invasive alternative by delivering reactive oxygen and nitrogen species (RONS) locally to tumor sites, selectively targeting cancer cells while sparing healthy tissue. To assess the impact and selectivity toward tumor cells adjacent to bone-like tissue, we develop a 3D bioprinted tumor-stroma model using a 1% alginate and 7% gelatin cell-laden hydrogel to mimic a bone-like microenvironment. The model co-cultures triple-negative MDA-MB-231 human breast cancer cells with primary human bone marrow mesenchymal stromal cells to simulate tumor-stroma interactions. The effects of CAP treatments are assessed through metabolic activity and viability assays over three days. Results show significant selectivity for cancer cells in both 2D and 3D cultures. CAP minimizes damage to healthy cells, offering the potential for localized treatment over systemic chemotherapies such as doxorubicin. Our novel bioprinted model, combined with a plasma source controlling RONS composition, enables detailed studies of redox-based cancer cell inactivation and highlights CAP as a personalized, non-invasive treatment for bone metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db15f717a397933ff6371b23deebeffcaae60439" target='_blank'>
              Cold Atmospheric Plasma Selectively Disrupts Breast Cancer Growth in a Bioprinted 3D Tumor Microenvironment Model
              </a>
            </td>
          <td>
            Laura M. Bouret, Jean-Baptiste Billeau, Michael H. Weber, Derek H. Rosenzweig, Stephan Reuter
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Spatially resolved transcriptomics enable comprehensive measurement of gene expression at subcellular resolution while preserving the spatial context of the tissue microenvironment. While deep learning has shown promise in analyzing SCST datasets, most efforts have focused on sequence data and spatial localization, with limited emphasis on leveraging rich histopathological insights from staining images. We introduce GIST, a deep learning-enabled gene expression and histology integration for spatial cellular profiling. GIST employs histopathology foundation models pretrained on millions of histology images to enhance feature extraction and a hybrid graph transformer model to integrate them with transcriptome features. Validated with datasets from human lung, breast, and colorectal cancers, GIST effectively reveals spatial domains and substantially improves the accuracy of segmenting the microenvironment after denoising transcriptomics data. This enhancement enables more accurate gene expression analysis and aids in identifying prognostic marker genes, outperforming state-of-the-art deep learning methods with a total improvement of up to 49.72%. GIST provides a generalizable framework for integrating histology with spatial transcriptome analysis, revealing novel insights into spatial organization and functional dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eddeb329dda1b1dbe9b8fc2b6a0d8ccb68b0e17" target='_blank'>
              Deep Learning-Enabled Integration of Histology and Transcriptomics for Tissue Spatial Profile Analysis
              </a>
            </td>
          <td>
            Yongxin Ge, Jiake Leng, Ziyang Tang, Kanran Wang, Kaicheng U, Sophia Meixuan Zhang, Sen Han, Yiyan Zhang, Jin-Peng Xiang, Sen Yang, Xiang Liu, Yi Song, Xiyue Wang, Yuchen Li, Junhan Zhao
          </td>
          <td>2024-12-17</td>
          <td>Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72ace1bf5ccc7bc1a835b1fd333db7f3f67e0b96" target='_blank'>
              Lessons learned from spatial transcriptomic analyses in clear-cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Jesper Jespersen, Cecilie Lindgaard, Laura Iisager, J. Ahrenfeldt, I. Lyskjær
          </td>
          <td>2025-01-09</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) have an overall poor prognosis with many high-risk cases co-opting stem cell gene regulatory programs, yet the mechanisms through which this occurs remain poorly understood. Increased expression of the stem cell transcription factor, MECOM, underlies one key driver mechanism in largely incurable AMLs. How MECOM results in such aggressive AML phenotypes remains unknown. To address existing experimental limitations, we engineered and applied targeted protein degradation with functional genomic readouts to demonstrate that MECOM promotes malignant stem cell-like states by directly repressing pro-differentiation gene regulatory programs. Remarkably and unexpectedly, a single node in this network, a MECOM-bound cis-regulatory element located 42 kb downstream of the myeloid differentiation regulator CEBPA, is both necessary and sufficient for maintaining MECOM-driven leukemias. Importantly, targeted activation of this regulatory element promotes differentiation of these aggressive AMLs and reduces leukemia burden in vivo, suggesting a broadly applicable differentiation-based approach for improving therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c42a2a7df7281b8834a5ccf4f4a2d74201a60" target='_blank'>
              CEBPA repression by MECOM blocks differentiation to drive aggressive leukemias
              </a>
            </td>
          <td>
            Travis J. Fleming, Mateusz Antoszewski, Sander Lambo, Michael Gundry, Riccardo Piussi, Lara Wahlster, Sanjana Shah, Fiona E. Reed, Kevin Dong, Joao A. Paulo, Steve P Gygi, C. Mimoso, Seth R Goldman, Karen Adelman, Jennifer A. Perry, Y. Pikman, K. Stegmaier, Maria N. Barrachina, Kellie R. Machlus, Volker Hovestadt, A. Arruda, Mark D. Minden, Richard A. Voit, V. G. Sankaran
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Dendritic cells (DCs) are promising targets for cancer immunotherapies because of their central role in the initiation and control of immune responses. The rare cDC1 population is of particular interest because of its remarkable ability to cross-present antigens (Ag) to CD8+ T cells, to promote Th1 cell polarization and NK cell activation and recruitment. However, the spatial organization and specific functions of cDC1s in response to immunotherapy remain to be clearly characterized in human tumors. Here, we implemented a multiplexed immunofluorescence analysis pipeline coupled with computational image analysis to determine the spatial organization of the cDC1 subset in a cohort of skin lesions from advanced melanoma patients treated with immune checkpoint inhibitors (ICI). For this, we performed a whole-slide image analysis of cDC1 infiltration and distribution as well as their spatial interactions with key immune partners such as CD8+ T cells and pDC according to the response of patients to ICI. We also analyzed LAMP3+-DC, which correspond to a mature subset of tumor-infiltrating DCs. Distance and cell network analyses demonstrated that cDC1s exhibited a scattered distribution compared to tumor-infiltrating pDCs and LAMP3+-DCs, which were preferentially organized in dense areas with high homotypic connections. Interestingly, the proximity and interactions between CD8+ T cells and cDC1s were positively associated with the response to ICI. In conclusion, our study unravels the complex spatial organization of cDC1s and their interactions with CD8+ T cells in melanoma patient lesions, shedding light on their pivotal role in shaping the response to ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f10d121483bbab07523097b63187cdc94830056d" target='_blank'>
              The spatial organization of cDC1 with CD8+ T cells is critical for the response to immune checkpoint inhibitors in melanoma patients.
              </a>
            </td>
          <td>
            Elisa Gobbini, Margaux Hubert, A. Doffin, Anaïs Eberhardt, Léo Hermet, Danlin Li, Pierre Duplouye, Sarah Ghamry-Barrin, Justine Berthet, Valentin Benboubker, Maxime Grimont, C. Sakref, Jimmy Perrot, Garance Tondeur, Olivier Harou, Jonathan Lopez, Bertrand Dubois, Stéphane Dalle, Christophe Caux, J. Caramel, J. Valladeau-Guilemond
          </td>
          <td>2025-01-07</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The Chromosome-Centric Human Proteome Project (C-HPP) is an international initiative. It aims to create a protein list expressed in human cells by each chromosomal and mitochondrial DNA to enhance our understanding of disease mechanisms, akin to the gene list generated by the Human Genome Project. Transmembrane protein 160 (TMEM160) is a member of the transmembrane proteins (TMEM) family. TMEM proteins have been implicated in cancer-related processes, including cell proliferation, migration, epithelial-mesenchymal transition, metastasis, and resistance to chemotherapy and radiotherapy. This study aimed to investigate the role of TMEM160 in non-small cell lung cancer and cervical cancer using cell lines, clinical samples, and xenograft studies. Our findings demonstrated that TMEM160 knockdown decreased the proliferation of lung and cervical cancer cell lines. We observed that TMEM160 is localized in the nucleus and cytoplasm and dynamic localization during mitosis of cancer cells and discovered a novel interaction between TMEM160 and nuclear proteins such as NUP50. Furthermore, the TMEM160 interactome was enriched in processes associated with apical junctions, xenobiotic metabolism, glycolysis, epithelial-mesenchymal transition, reactive oxygen species, UV response DNA, the P53 pathway, and the mitotic spindle. This study provides an initial understanding of the function of TMEM160 in lung and cervical cancer progression and clarifies the need to continue investigating the participation of TMEM160 in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e13c463e1e4f1ddbb09b31ff1b7171d0bdfe61c" target='_blank'>
              TMEM160 Promotes Tumor Growth in Lung Adenocarcinoma and Cervical Adenocarcinoma Cell Lines
              </a>
            </td>
          <td>
            Gloria Angelina Herrera-Quiterio, H. A. Valencia-González, K. G. de la Cruz-López, D. L. Fernández-Coto, Jeovanis Gil, Gyorgy Marko-Varga, Josué Morales-Gálvez, Nilda C. Sánchez, Rubén Rodríguez-Bautista, A. Avilés-Salas, Oscar Arrieta, Alejandro García-Carrancá, S. Encarnación-Guevara
          </td>
          <td>2025-01-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Mouse models that faithfully represent the biology of human brain tumors are critical tools for unraveling the underlying tumor biology and screening for potential precision therapies. This is especially true of rare tumor types, many of which have correspondingly few xenograft or cell lines available. Although our understanding of the specific biological pathways driving cancer has improved significantly, identifying the appropriate progenitor populations to drive oncogenic processes represents a significant barrier to efficient mouse model production. In this issue of Cancer Research, Jessa and colleagues developed an innovative transcription factor fingerprinting method to map the cellular origin of central nervous system neuroblastoma, FOXR2-activated to medial ganglionic eminence-derived interneurons, which could then be efficiently targeted in the developing mouse brain using in utero electroporation. This approach serves as a blueprint for investigating other rare pediatric brain tumors, potentially accelerating progress toward the development of mouse models and identification of effective therapies. See related article by Jessa et al., p. 231.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b36b1d9ff773da8434bf45b0c13f61a915472b5f" target='_blank'>
              Transcription Factor Fingerprint Provides Clues for Brain Tumor Cell of Origin.
              </a>
            </td>
          <td>
            Brent A. Orr
          </td>
          <td>2024-01-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
The tumour microenvironment significantly influences the clinical response of patients to therapeutic immune checkpoint inhibition (ICI), but a comprehensive understanding of the underlying immune-regulatory proteome is still lacking.


OBJECTIVES
To decipher targetable biologic processes that determine tumour-infiltrating lymphocytes (TiLs) as a cellular equivalent of clinical response to ICI.


METHODS
We mapped the spatial distribution of proteins in TiL-enriched vs. TiL-low compartments in melanoma by combining microscopy, matrix-assisted laser desorption mass spectrometry imaging and liquid chromatography-mass spectrometry, as well as computational data mining. Pharmacological modulation of sirtuin 1 (SIRT1) activity in syngeneic mouse models was used to evaluate the efficacy of pharmacological SIRT1 activation in two syngeneic melanoma mouse models, one known to be α-programmed cell death protein 1 (PD-1) sensitive and the other α-PD-1 resistant.


RESULTS
Spatial proteomics and gene ontology-based enrichment analysis identified > 145 proteins enriched in CD8high tumour compartments, including negative regulators of mammalian target of rapamycin signalling such as SIRT1. Multiplexed immunohistochemistry confirmed that SIRT1 protein was expressed more in CD8high than in CD8low compartments. Further analysis of bulk and single-cell RNA sequencing data from melanoma tissue samples suggested the expression of SIRT1 by different lymphocyte subpopulations (CD8+ T cells, CD4+ T cells and B cells). Furthermore, we showed in vivo that pharmacological SIRT1 activation increased the immunological effect of α-PD-1 ICI against melanoma cells in mice, which was accompanied by an increase in T-cell infiltration and T-cell-related cytokines, including interferon (IFN)-γ, CCL4, CXCL9, CXCL10 and tumour necrosis factor-α. In silico analysis of large transcriptional data cohorts showed that SIRT1 was positively associated with the proinflammatory T-cell chemokines CXCL9, CXCL10 and IFN-γ, and prolonged overall survival of patients with melanoma.


CONCLUSIONS
Our study deciphers the proteomics landscape in human melanoma, providing important information on the tumour microenvironment and identifying SIRT1 as having important prognostic and therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3e28ba69768cc37ada6cc67883351db03f9a038" target='_blank'>
              Spatial proteomics reveals sirtuin 1 to be a determinant of T-cell infiltration in human melanoma.
              </a>
            </td>
          <td>
            J. Placke, Jenny Bottek, R. Váraljai, B. Shannan, Sarah Scharfenberg, Christoph Krisp, Philippa Spangenberg, Camille Soun, Devon Siemes, Lars Borgards, Franziska Hoffmann, Fang Zhao, Anette Paschen, Hartmut Schlueter, Ferdinand von Eggeling, Iris Helfrich, Florian Rambow, Selma Ugurel, A. Tasdogan, Dirk Schadendorf, D. R. Engel, Alexander Roesch
          </td>
          <td>2024-12-31</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa374ada53bb9d0014e3752d56d972702923921" target='_blank'>
              Genome-scale modeling identifies dynamic metabolic vulnerabilities during the epithelial to mesenchymal transition
              </a>
            </td>
          <td>
            Rupa Bhowmick, Scott Campit, Shiva Krishna Katkam, V. Keshamouni, Sriram Chandrasekaran
          </td>
          <td>2024-12-27</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="The growth and progression of a neoplasm is a multi–stage process mediated by interactions between tumor cells and their surrounding stroma including various cell types (fibroblasts, endothelial cells, immune cells), cell-free components (cytokines, extracellular vesicles and extracellular matrix), blood and lymphatic vessels that form the tumor microenvironment. During the progression of neoplasms, tumor cells can evade the immune response by suppressing or losing tumor antigens releasing immunosuppressive extracellular vesicles and exosomes, secreting various cytokines, chemokines and other factors, and begin to reprogram and remodel the surrounding stroma. The most relevant areas of TME study are cells and molecules of innate and acquired immunity exhibited both suppressive and stimulating activity against tumor cells due to their high heterogeneity and plasticity. Special attention is paid to immune control points, tumor microbiota, components of intercellular interaction as well as immunotherapy of tumor diseases.
In this paper, a systematic review of scientific publications in PubMed database devoted to the study of TME and published in the period from 2019 to 2024 was carried out. 2547 sources were studied and 68 articles were selected during the screening process. Search queries and keywords included «tumor microenvironment», «immune cells», «tumor», «oncogenesis», «metastasis». The formed group of scientific articles will allow to study in more detail the cellular and molecular
composition of the tumor microenvironment, as well as the main targets for the development of new approaches for malignant neoplasms diagnosis and therapy involving TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/568399709b478d3282ff1a127c4698c1e28c9300" target='_blank'>
              Tumor microenvironment: systematic review
              </a>
            </td>
          <td>
            A. V. Vialichka, D. B. Nizheharodava, Elizaveta M. Nazaranka, Bogdan A. Muzychenka, M. Zafranskaya
          </td>
          <td>2024-12-27</td>
          <td>Journal of the Belarusian State University. Ecology.</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are crucial stromal cells in the tumor microenvironment, affecting cancer growth, angiogenesis, and matrix remodeling. Developing an effective in vitro tumor model that accurately recapitulates the dynamic interplay between tumor and stromal cells remains a challenge. In this study, a 3D bioprinted fibroblast - mediated heterogeneous breast tumor model was created, with tumor cells and fibroblasts in a bionic matrix. The impact of transforming growth factor-β （TGF-β） on the dynamic transformation of normal fibroblasts into CAFs and its subsequent influence on tumor cells is further investigated. These findings reveales a profound correlation between CAFs and several critical biological processes, including epithelial-mesenchymal transition (EMT), extracellular matrix (ECM) remodeling, gene expression profiles, and tumor progression. Furthermore, tumor models incorporating CAFs exhibits reduced drug sensitivity compared to models containing tumor cells alone or models co-cultured with normal fibroblasts. These results underscore the potential of the in vitro fibroblast-mediated heterogeneous tumor model to simulate real-life physiological conditions, thereby offering a more effective drug screening platform for elucidating tumor pathogenesis and facilitating drug design prior to animal and clinical trials. This model's establishment promotes the understanding of tumor-stromal interactions and their therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62b9a94b40703abae6a8d97875379a7a33a03aa7" target='_blank'>
              Bioprinted Fibroblast Mediated Heterogeneous Tumor Microenvironment for Studying Tumor-Stroma Interaction and Drug Screening.
              </a>
            </td>
          <td>
            You Chen, Yifan Xue, Cong Yan, Jinlong Jin, Yadong Liu, Jing Li, Shuai Han, Jie Liu
          </td>
          <td>2025-01-22</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88ba70c1f7c9e0c6817db8a69bc543b13dfed9df" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics of esophageal squamous cell carcinoma with lymph node metastases
              </a>
            </td>
          <td>
            Wei Guo, Bolun Zhou, Lizhou Dou, Lei Guo, Yong Li, J. Qin, Zhen Wang, Qilin Huai, Xuemin Xue, Yin Li, Jianming Ying, Qi Xue, Shugeng Gao, Jie He
          </td>
          <td>2025-01-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The lung tumor microenvironment (TME) or stroma is a dynamic space of numerous cells and their released molecules. This complicated web regulates tumor progression and resistance to different modalities. Lung cancer cells in conjunction with their stroma liberate a wide range of factors that dampen antitumor attacks by innate immunity cells like natural killer (NK) cells and also adaptive responses by effector T cells. These factors include numerous growth factors, exosomes and epigenetic regulators, and also anti-inflammatory cytokines. Understanding the intricate interactions between tumor cells and various elements within the lung TME, such as immune and stromal cells can help provide novel strategies for better management and treatment of lung malignancies. The current article discusses the complex network of cells and signaling molecules, which mediate communications in lung TME. By elucidating these multifaceted interactions, we aim to provide insights into potential therapeutic targets and strategies for lung cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92f952599eb5955ba5c54def492a216230a18af6" target='_blank'>
              Decoding the complex web: Cellular and molecular interactions in the lung tumor microenvironment.
              </a>
            </td>
          <td>
            Bahjat Saeed Issa, Ayat Hussein Adhab, Morug Salih Mahdi, Ashishkumar Kyada, Subbulakshmi Ganesan, Deepak Bhanot, K. S. Naidu, Sharnjeet Kaur, A. S. Mansoor, Usama Kadem Radi, Nasr Saadoun Abd, Muthena Kariem
          </td>
          <td>2024-12-21</td>
          <td>Journal of drug targeting</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c65e0e056080142eb716f1c5270f62643243964" target='_blank'>
              From ductal carcinoma in situ to invasive breast cancer: the prognostic value of the extracellular microenvironment
              </a>
            </td>
          <td>
            Taylor S. Hulahan, Peggi M. Angel
          </td>
          <td>2024-12-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The progression of colorectal cancer is strongly influenced by environmental and genetic conditions. One of the key factors is tumor heterogeneity which is extensively studied by cfDNA and bulk sequencing methods; however, we lack knowledge regarding its effects at the single-cell level. Motivated by this, we aimed to employ an end-to-end single-cell sequencing workflow from tissue-derived sample isolation to exome sequencing. Our main goal was to investigate the heterogeneity patterns by laser microdissecting samples from different locations of a tissue slide. Moreover, by studying healthy colon control, tumor-associated normal, and colorectal cancer tissues, we explored tissue-specific heterogeneity motifs. For completeness, we also compared the performance of the whole-exome bulk, cfDNA, and single-cell sequencing methods based on variation at the level of a single nucleotide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9266548dfc591e54ed5a1417777c06bd133c4150" target='_blank'>
              Investigation of Exome-Wide Tumor Heterogeneity on Colorectal Tissue-Based Single Cells
              </a>
            </td>
          <td>
            Nikolett Szakállas, A. Kalmár, B. Barták, Z. Nagy, Gábor Valcz, Tamás Richárd Linkner, Kristóf Róbert Rada, István Takács, Béla Molnár
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35c843d34d2321464470845817a613dfc68c436a" target='_blank'>
              Integrin α8 is a useful cell surface marker of alveolar lipofibroblasts
              </a>
            </td>
          <td>
            A. Fukada, Yasunori Enomoto, R. Horiguchi, Y. Aoshima, S. Meguro, H. Kawasaki, I. Kosugi, T. Fujisawa, N. Enomoto, Naoki Inui, T. Suda, T. Iwashita
          </td>
          <td>2025-01-13</td>
          <td>Respiratory Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSCs) tend to acquire genetic aberrations upon culture in vitro. Common aberrations are mutations in the tumor suppressor TP53, suspected to confer a growth-advantage to the mutant cells. However, their full impact in the development of malignant features and safety of hPSCs for downstream applications is yet to be elucidated. Here, TP53 is knocked out in hPSCs using CRISPR-Cas9 and compared them with isogenic wild-type hPSCs and human germ cell tumor lines as models of malignancy. While no major changes in proliferation, pluripotency, and transcriptomic profiles are found, mutant lines display aberrations in some of the main chromosomal hotspots for genetic abnormalities in hPSCs. Additionally, enhanced clonogenic and anchorage-free growth, alongside resistance to chemotherapeutic compounds is observed. The results indicate that common TP53-depleting mutations in hPSCs, although potentially overlooked by standard analyses, can impact their behavior and safety in a clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b157fa230ecf42b51895b57dbccc944884bf0d8" target='_blank'>
              Depletion of TP53 in Human Pluripotent Stem Cells Triggers Malignant-Like Behavior.
              </a>
            </td>
          <td>
            Joaquin Montilla-Rojo, T. F. Eleveld, Marnix van Soest, Sanne Hillenius, Dennis M. Timmerman, A. Gillis, B. Roelen, C L Mummery, Leendert H J Looijenga, Daniela C. F. Salvatori
          </td>
          <td>2025-01-06</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa45e1efe02786e2e4534cf1ae408e65c7ffadc" target='_blank'>
              To describe the subsets of malignant epithelial cells in gastric cancer, their developmental trajectories and drug resistance characteristics
              </a>
            </td>
          <td>
            Tingting Xu, Tianying Zhang, Yan Sun, Sijia Wu
          </td>
          <td>2025-01-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab8b68e7b5a2f581513237acff3c4e88418f66b" target='_blank'>
              Haploinsufficient phenotypes promote selection of PTEN and ARID1A-deficient clones in human colon.
              </a>
            </td>
          <td>
            Nefeli Skoufou-Papoutsaki, Sam Adler, Shenay Mehmed, Claire Tume, Cora Olpe, Edward Morrissey, Richard Kemp, Anne-Claire Girard, Elisa B. Moutin, C. Chilamakuri, Jodi L Miller, Cecilia Lindskog, Fabian Werle, Kate Marks, Francesca Perrone, Matthias Zilbauer, D. Tourigny, Doug Winton
          </td>
          <td>2025-02-07</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background and Aims Total abdominal colectomy (TAC) with a staged ileal pouch-anal anastomosis (IPAA) is a common surgical treatment for ulcerative colitis (UC). However, a significant percentage of patients experience pouch failure, leading to morbidity. This retrospective case-control study identified histopathological features of the TAC specimen associated with pouch failure and investigated the molecular mechanisms of this susceptibility using single-cell spatial transcriptomics. Methods We analyzed a cohort of 417 patients who underwent IPAA between 2000-2010 at the University of Chicago Medical Center for up to 18 years. Histological examination of TAC specimens focused on disease activity, depth of inflammation, and specific features, including granulomas and deep ulcers. A subset of patients was profiled using single-cell spatial transcriptomics to map gene expression and immune cell interactions in relation to the risk of pouch failure. Results The 18-year pouch failure risk was 23%, with post-procedure diagnosis of CD as a major risk factor (HR = 4.3, 95% CI: 2.3–8.1) as well as high-risk histologic features, including deep chronic inflammation (HR = 21, 95% CI: 11-41) and severe disease activity (HR = 14, 95% CI: 5.7-32) in TAC specimens. Spatial transcriptomics showed immune infiltration of T and myeloid cells, reduced myocyte-glial interactions, and cytokine signaling pathways such as IL-10, IL-1β, and type I/II interferons, associated with an increased risk of pouch failure. Conclusion Histological features and spatial molecular profiling are predictive of IPAA failure. These findings support the use of histologic evaluation and targeted molecular analysis of the TAC specimen to identify high-risk patients and improve IPAA outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a586fb545c9fdbb6b8896dfb9ebdcb9f9f1dc37" target='_blank'>
              Histopathologic Evaluation and Single-Cell Spatial Transcriptomics of the Colon Reveal Cellular and Molecular Abnormalities Linked to J-Pouch Failure in Patients with Inflammatory Bowel Disease
              </a>
            </td>
          <td>
            Andrea D. Olivas, Paul Chak Mou Ngai, Emily Schahrer, K. Olortegui, John F. Cursio, Shintaro Akiyama, Eugene B. Chang, Le Shen, Konstantin Umanskiy, David T. Rubin, David Zemmour, Christopher R. Weber
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Components of normal tissue architecture serve as barriers to tumor progression. Inflammatory and wound-healing programs are requisite features of solid tumorigenesis, wherein alterations to immune and non-immune stromal elements enable loss of homeostasis during tumor onset. The precise mechanisms by which normal stromal cell states limit tissue plasticity and tumorigenesis, and which are lost during tumor progression, remain largely unknown. Here we show that healthy pancreatic mesenchyme expresses the paracrine signaling molecule KITL, also known as stem cell factor, and identify loss of stromal KITL during tumorigenesis as tumor-promoting. Genetic inhibition of mesenchymal KITL in the contexts of health, injury, and cancer together indicate a role for KITL signaling in maintenance of pancreas tissue architecture, such that loss of the stromal KITL pool increased tumor growth and reduced survival of tumor-bearing mice. Together, these findings implicate loss of mesenchymal KITL as a mechanism for establishing a tumor-permissive microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b05f1579f2dda835515d867e6c5a5ccd6f715258" target='_blank'>
              Stromal KITL/SCF maintains pancreas tissue homeostasis and restrains tumor progression.
              </a>
            </td>
          <td>
            M. K. Oñate, Chet Oon, Sohinee Bhattacharyya, V. Low, Canping Chen, Xiaofan Zhao, F. Arnold, Ziqiao Yan, Sneha Pramod, Yan Hang, Yu-jui Ho, Scott W. Lowe, Seung K Kim, Zheng Xia, Mara H Sherman
          </td>
          <td>2025-02-10</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M. Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Patient-derived organoids represent a novel platform to recapitulate the cancer cells in the patient tissue. While cancer heterogeneity has been extensively studied by a number of omics approaches, little is known about the spatiotemporal kinase activity dynamics. Here we applied a live imaging approach to organoids derived from 10 pancreatic ductal adenocarcinoma (PDAC) patients to comprehensively understand their heterogeneous growth potential and drug responses. By automated wide-area image acquisitions and analyses, the PDAC cells were non-selectively observed to evaluate their heterogeneous growth patterns. We monitored single-cell ERK and AMPK activities to relate cellular dynamics to molecular dynamics. Furthermore, we evaluated two anti-cancer drugs, a MEK inhibitor, PD0325901, and an autophagy inhibitor, hydroxychloroquine (HCQ), by our analysis platform. Our analyses revealed a phase-dependent regulation of PDAC organoid growth, where ERK activity is necessary for the early phase and AMPK activity is necessary for the late stage of organoid growth. Consistently, we found PD0325901 and HCQ target distinct organoid populations, revealing their combination is widely effective to the heterogeneous cancer cell population in a range of PDAC patient-derived organoid lines. Together, our live imaging quantitatively characterized the growth and drug sensitivity of human PDAC organoids at multiple levels: in single cells, single organoids, and individual patients. This study will pave the way for understanding the cancer heterogeneity and promote the development of new drugs that eradicate intractable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824e882598ac674f498c3ffe96037afbee14d231" target='_blank'>
              Quantitative Live Imaging Reveals Phase Dependency of PDAC Patient-Derived Organoids on ERK and AMPK Activity.
              </a>
            </td>
          <td>
            Shoko Tsukamoto, Ye Huaze, Weisheng Zhang, Akihito Machinaga, N. Kakiuchi, Seishi Ogawa, Hiroshi Seno, Shigeki Higashiyama, Michiyuki Matsuda, Toru Hiratsuka
          </td>
          <td>2024-12-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background The role of cancer-associated pericytes (CAPs) in tumor microenvironment (TME) suggests that they are potential targets for cancer treatment. The mechanism of CAP heterogeneity in esophageal squamous cell carcinoma (ESCC) remains unclear, which has limited the development of treatments for tumors through CAPs. Therefore, a comprehensive understanding of the classification, function, cellular communication and spatial distribution of CAP subpopulations in ESCC is urgently needed. Methods This study used large-sample single-cell transcriptome sequencing (scRNA-seq) data to investigate pericytes’ subpopulation characteristics, functions, upstream and downstream regulation and interactions with other components of the TME in the ESCC, and analyzed prognostically in conjunction with Bulk RNA-seq data. In addition, pericyte subpopulations were validated and their spatial distribution in the ESCC TME was observed by multiplex immunofluorescence. Drug prediction and molecular docking was further used to validate the medicinal value of drug targets. Results CAPs in the ESCC TME were found to be highly heterogeneous, and we identified six pericyte subtypes: c1_ARHGDIB, c2_BCAM, c3_LUM, c4_SOD2, c5_TYMS, and c6_KRT17, which have commonality in a part of their functions, and each of them has a major function to play, by having different strengths of interaction with different components in the TME. In addition, we found that c4_SOD2 was negatively correlated with prognosis, conversely, c5_TYMS was positively correlated with prognosis. The drug with a better effect on c5_TYMS was docetaxel (binding energy = -8.1, -8.7 kcal/mol); raloxifene may be more effective against c4_SOD2, although raloxifene has a slightly lower binding energy to SOD2 (-6.4 kcal/mol), it has a higher binding energy to PDGFRβ (-8.1 kcal/mol). Conclusion The present study identified and discovered pericyte subpopulations that were significantly associated with prognosis, which provides new biomarkers for predicting patient prognosis and adds usable targets for immunotherapy, and it is also important for gaining insights into the composition of the TME in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e295485d04626d24ad4217e848a133499f9df967" target='_blank'>
              Single-cell sequencing analysis reveals cancer-associated pericyte subgroup in esophageal squamous cell carcinoma to predict prognosis
              </a>
            </td>
          <td>
            Kai Xiong, Bing Pan, Hao Fang, Ziyou Tao
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) is the most common histologic type of RCC. However, the spatial and functional heterogeneity of immunosuppressive cells and the mechanisms by which their interactions promote immunosuppression in the ccRCC have not been thoroughly investigated. Methods To further investigate the cellular and regional heterogeneity of ccRCC, we analyzed single-cell and spatial transcriptome RNA sequencing data from four patients, which were obtained from samples from multiple regions, including the tumor core, tumor-normal interface, and distal normal tissue. On the basis, the findings were investigated in vitro using tissue and blood samples from 15 patients with ccRCC and validated in the broader samples on tissue microarrays. Results In this study, we revealed previously unreported subsets of both stromal and immune cells, as well as mapped their spatial location at finer resolution. In addition, we validated the clusters of tumor cells after removing batch effects according to six characterized gene sets, including epithelial-mesenchymal transitionhigh clusters, metastatic clusters and proximal tubulehigh clusters. Importantly, we identified a special regulatory T (Treg) cell subpopulation that has the molecular characteristics of terminal effector Treg cells but expresses multiple cytokines, such as interleukin (IL)-1β and IL-18. This group of Treg cells has stronger immunosuppressive function and was associated with a worse prognosis in ccRCC cohorts. They were colocalized with MRC1+FOLR2+ tumor-associated macrophages (TAMs) at the tumor-normal interface to form a positive feedback loop, maintaining a synergistic procarcinogenic effect. In addition, we traced the origin of IL-1β+ Treg cells and revealed that IL-18 can induce the expression of IL-1β in Treg cells via the ERK/NF-κB pathway. Conclusions We demonstrated a novel cancer-promoting Treg cell subset and its interactions with MRC1+FOLR2+TAMs, which provides new insight into Treg cell heterogeneity and potential therapeutic targets for ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2214f0f0d02f003fb8cfcf936f4f2873d564786b" target='_blank'>
              Spatial and single-cell transcriptomics reveal cellular heterogeneity and a novel cancer-promoting Treg cell subset in human clear-cell renal cell carcinoma
              </a>
            </td>
          <td>
            Xiyu Song, Yumeng Zhu, Wenwen Geng, J. Jiao, Hongjiao Liu, Ruo Chen, Qian He, Lijuan Wang, Xiuxuan Sun, Weijun Qin, Jiejie Geng, Zhinan Chen
          </td>
          <td>2025-01-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bc9bd11ddbadc141e7404167a9d1237fb1c3a74" target='_blank'>
              Encompassing view of spatial and single-cell RNA sequencing renews the role of the microvasculature in human atherosclerosis.
              </a>
            </td>
          <td>
            Tore Bleckwehl, Anne Babler, Merel Tebens, Sidrah Maryam, Michael Nyberg, Markus Bosteen, Maurice Halder, Isaac Shaw, Susanne Fleig, Charles Pyke, Henning Hvid, Louise M. Voetmann, J. V. van Buul, J. Sluimer, Vivek Das, Simon J. Baumgart, R. Kramann, Sikander Hayat
          </td>
          <td>2024-12-23</td>
          <td>Nature cardiovascular research</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Fibroblasts are critical cells that shape the architecture and cellular ecosystems in multiple tissues. Understanding fibroblast heterogeneity and their spatial context in health and disease has enormous clinical relevance. In this study, we constructed a spatially-resolved atlas of human skin fibroblasts from healthy skin and 23 skin disorders. We define 6 major skin fibroblast populations in health and a further three skin disease-specific fibroblast subtypes, and demonstrate the fibroblast composition in different types of skin disease. We characterise a human-specific fibroblastic reticular cell (FRC)-like subtype in the skin perivascular niche and postulate their origin from prenatal skin lymphoid tissue organiser (LTo)-like cells. We also show that inflammatory myofibroblasts (IL11+MMP1+CXCL5+IL7R+) are a conserved fibroblast subtype in inflammatory disorders and cancers across multiple human tissues. We provide a harmonised nomenclature for skin fibroblasts that integrates previous findings from human skin and other tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8105c94d4aa143c539f0558370814becb076a8c6" target='_blank'>
              A single cell and spatial genomics atlas of human skin fibroblasts in health and disease
              </a>
            </td>
          <td>
            Lloyd Steele, Chloe Admane, Keerthi Priya Chakala, A. Foster, N. Gopee, Simon Koplev, Pavel Mazin, B. Olabi, Kenny Roberts, Catherine Tudor, E. Winheim, Karl Annusver, D. Correa-Gallegos, A. Forsthuber, L. Francis, S. Frech, Clarisse Gânier, T. Layton, Yingzi Liu, Hao Yuan, J. Gudjonsson, B. Lichtenberger, S. Mahil, J. Nanchahal, Edel A O'Toole, Maksim Plikus, Y. Rinkevich, Emanuel Rognoni, Catherine Smith, Sarah A. Teichmann, Maria Kasper, M. Lotfollahi, M. Haniffa
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Abstract Spatial transcriptomics (ST) offers enormous potential to decipher the biological and pathological heterogeneity in precious archival cancer tissues. Traditionally, these tissues have rarely been used and only examined at a low throughput, most commonly by histopathological staining. ST adds thousands of times as many molecular features to histopathological images, but critical technical issues and limitations require more assessment of how ST performs on fixed archival tissues. In this work, we addressed this in a cancer‐heterogeneity pipeline, starting with an exploration of the whole transcriptome by two sequencing‐based ST protocols capable of measuring coding and non‐coding RNAs. We optimised the two protocols to work with challenging formalin‐fixed paraffin‐embedded (FFPE) tissues, derived from skin. We then assessed alternative imaging methods, including multiplex RNAScope single‐molecule imaging and multiplex protein imaging (CODEX). We evaluated the methods’ performance for tissues stored from 4 to 14 years ago, covering a range of RNA qualities, allowing us to assess variation. In addition to technical performance metrics, we determined the ability of these methods to quantify tumour heterogeneity. We integrated gene expression profiles with pathological information, charting a new molecular landscape on the pathologically defined tissue regions. Together, this work provides important and comprehensive experimental technical perspectives to consider the applications of ST in deciphering the cancer heterogeneity in archived tissues. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0ac2d361f4c48551940640dc92bcb661d02f42d" target='_blank'>
              Assessing spatial sequencing and imaging approaches to capture the molecular and pathological heterogeneity of archived cancer tissues
              </a>
            </td>
          <td>
            Tuan Vo, P. Prakrithi, Kahli Jones, Sohye Yoon, P. Y. Lam, Y. Kao, Ning Ma, Samuel X. Tan, Xinnan Jin, Chenhao Zhou, Joanna Crawford, S. Walters, Ishaan Gupta, Peter H Soyer, K. Khosrotehrani, M. Stark, Quan H Nguyen
          </td>
          <td>2025-01-23</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) play pivotal roles in solid tumor initiation, growth, and immune evasion. However, the optimal biomimetic modeling conditions remain elusive. In this study, we investigated the effects of 2D and 3D culturing conditions on human primary CAFs integrated into a modular tumor microenvironment (TME). Using single-nucleus RNA sequencing (snRNAseq) and Proteomics’ Proximity Extension Assays, we characterized CAF transcriptomic profiles and cytokine levels. Remarkably, when cultured in 2D, CAFs exhibited a myofibroblast (myCAF) subtype, whereas in 3D tumor spheroid cultures, CAFs displayed a more inflammatory (iCAF) pathological state. By integrating single-cell gene expression data with functional interrogations of critical TME-related processes [natural killer (NK)-mediated tumor killing, monocyte migration, and macrophage differentiation], we were able to reconcile form with function. In 3D TME spheroid models, CAFs enhance cancer cell growth and immunologically shield cells from NK cell-mediated cytotoxicity, in striking contrast with their 2D TME counterparts. Notably, 3D CAF-secreted proteins manifest a more immunosuppressive profile by enhancing monocyte transendothelial migration and differentiation into M2-like tumor-associated macrophages (TAMs). Our findings reveal a more immunosuppressive and clinically relevant desmoplastic TME model that can be employed in industrial drug discovery campaigns to expand the cellular target range of chemotherapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43b61425d19bdd4e9908dd233e505b35ad2127a3" target='_blank'>
              Unveiling immune cell response disparities in human primary cancer-associated fibroblasts between two- and three-dimensional cultures
              </a>
            </td>
          <td>
            Jian-Ping Yang, Nikhil Nitin Kulkarni, Masashi Yamaji, Tsubasa Shiraishi, Thang Pham, Han Do, Nicole Aiello, Michael Shaw, Toshihiro Nakamura, Akiko Abiru, N. Gavva, Shane R. Horman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Ductal carcinoma in situ (DCIS) harbors genomic copy number changes reflecting early events in advancing invasive breast cancer, and changes in specific cell states related to cancer initiation and progression. DNA methylation is a covalent modification to cytosines with cell type specificity, is stable in archival samples, and provides most of the specificity in multiple noninvasive pan-cancer screens. However, the creation of single-cell DNA methylomes is technically challenging and analysis is immature, leaving cell type and state level interpretation understudied. This has led to a gap in knowledge: it is not understood how the methylome reacts to gene dosage changes brought on by early copy number events, and it is not known how long these potential methylation changes persist. To this end, we have created and optimized a high throughput assay to measure these methylation changes at single-cell level, pairing our analysis with a multimodal view of these early copy number events. Here we describe three major steps in our work on elucidating methylome response to early breast cancer events: 1) We have generated methylome libraries for breast cancer cell lines with known large scale copy number alterations and validated our copy number calling in methylomes with those of gold-standard single-cell whole genome libraries. 2) We have generated thousands of single cell methylomes on normal breast samples attained from reduction surgeries paired with our groups’ work on the Human Breast Cell Atlas project. We were able to identify all major cell types of the human breast using methylation data for the first time. We used this data set to generate high-quality pseudobulk cell type specific methylomes. 3) We have profiled 5 DCIS and 1 synchronous invasive ductal carcinoma sample with subclonal copy number events, paired with single-cell transcriptomics. Through this we identified subclones in our methylome libraries and validated these through inferred copy number changes in our single-cell transcriptome libraries. We have uncovered thousands of hypomethylated regions enriched for specific transcription factor motifs, and widespread methylation changes across DCIS subclones. Notably many differential methylation changes occur outside of the genomic boundaries of copy number effects, suggesting distal changes by transcription factor gene dosage, chromatin perturbation, or tumor microenvironment interactions. These data will serve as a basis for future analyses into long-term follow through of archival samples and expand our understanding of this understudied epigenomic mark and its importance in early cancer evolution.
 Citation Format: Ryan M Mulqueen, Mariam Mosaad, Shanshan Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin. Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa4f976dba878ba71cab253948818a9b0e43066" target='_blank'>
              Abstract A006: Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer
              </a>
            </td>
          <td>
            Ryan M Mulqueen, Mariam Mosaad, S. Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Single-cell transcriptomes is advantageous in characterizing cell types and states, but not sufficient in describing cell phenotypes due to small cohorts in single-cell datasets. Thus, inferring single-cell phenotypes with the leverage of bulk data is practical. Here we proposed an algorithm for Single Cells’ Phenotype Prediction based on the expression profiles of phenotype-associated marker genes in bulks and single cells (ScPP). ScPP first analyzes bulk data to identify phenotype-associated marker genes. Next, ScPP evaluates the enrichment scores of the phenotype-associated marker gene sets in single cells using the AUCell algorithm. The single cells with the phenotype are defined as the intersection of the single cells with top α ranks according to the phenotype-associated AUC values and the single cells with bottom α ranks according to the opposite phenotype-associated AUC values. Finally, all single cells are determined as phenotype+, phenotype- or background. We demonstrated that ScPP could effectively recognize cell subpopulations with specific phenotypes, including tumor malignancy, estrogen receptor status, microsatellite instability, copy number variation, survival and immunotherapy response. Compared to the established algorithms (Scissor and scAB), ScPP displays more excellent predictive performance and needs less running time. Thus, ScPP is an effective and competitive tool for inferring cell phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e382f0266dec158343a7ae5f61a148cc4c9c6f78" target='_blank'>
              Inferring phenotypes of single cells based on the expression profiles of phenotype-associated marker genes in bulks and single cells
              </a>
            </td>
          <td>
            Yin He, Rongzhuo Long, Li Zhao, Xiaosheng Wang
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55057507874b2d1db614aaeb55e2719a4aaa942d" target='_blank'>
              Metabolic rewiring in skin epidermis drives tolerance to oncogenic mutations.
              </a>
            </td>
          <td>
            Anupama Hemalatha, Zong-Hu Li, David G. Gonzalez, C. Matte-Martone, Karen Tai, Elizabeth Lathrop, Daniel Gil, S. Ganesan, Lauren E Gonzalez, Melissa C. Skala, Rachel J. Perry, Valentina Greco
          </td>
          <td>2025-01-06</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4bbd7ae49e10933acda8e20cbd7e3bb473c50d" target='_blank'>
              Endosomal pH is an evolutionarily conserved driver of phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            Hari Prasad, H. Bv, A. R. Subbalakshmi, S. Mandal, M. Jolly, Sandhya S. Visweswariah
          </td>
          <td>2024-12-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Disseminated tumor cells (DTCs) contribute to metastasis formation by adapting to the hostile environmental conditions encountered at the seeded secondary organs. This phenotypic plasticity is supported by the epigenetic reprogramming of cis-regulatory elements (CREs), representing a hallmark of metastatic cells. DTCs can enter a reversible state of cancer dormancy, resulting from their ability to enter quiescence after mitosis and escape immune surveillance. In this respect, mitosis may represent a window of opportunity to increase DTCs’ fitness by priming CREs, ensuring the reactivation of a dormancy-related transcriptional program while preserving the epigenetic information. By profiling the chromatin changes occurring during the early steps of mitosis, we found that DTCs experience fluctuations in chromatin accessibility at CREs, which are bookmarked by pro-metastatic TFs. Among these, the pro-dormancy TF SOX9 remained associated with mitotic chromatin, facilitating transcriptional reactivation upon exiting mitosis. By dissecting the consequences of degrading SOX9 before mitosis, we found that its priming of dormancy-related CREs, combined with the recruitment of the Mediator kinase module, is instrumental for metastatic cells to enter quiescence and escape NK-mediated immune surveillance. These findings indicate that cancer dormancy is primed during mitosis by pro-metastatic TFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The development of treatment resistance remains universal for patients with metastatic prostate cancer, driven by AR alterations and lineage state transitions. Identifying the evolution of lineage transitions in treatment resistance has been limited by the challenges of collecting serial tissue biopsies on treatment, which can be overcome using blood-based liquid biopsies. Utilizing a novel circulating tumor cell (CTC) isolation approach, we collected 273 CTC samples from 117 patients with metastatic prostate cancer for RNA sequencing. 146 samples from 70 patients had tumor purity comparable to tissue biopsies. We identified four CTC transcriptional phenotypes, mirroring lineage states identified in tissue. Patients with a luminal-B-like CTC phenotype defined by persistent AR signaling and high proliferation, as well as those with a neuroendocrine CTC phenotype, had significantly shorter survival than patients with luminal-A-like and low proliferation phenotypes. In a prospective substudy, pre-treatment CTC luminal-B-like phenotype was associated with early progression on 177Lu-PSMA-617.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166ffe8ca69e378ce5cd3ea06cb57373f1b4e28e" target='_blank'>
              High-purity CTC RNA sequencing identifies prostate cancer lineage phenotypes prognostic for clinical outcomes.
              </a>
            </td>
          <td>
            Marina N. Sharifi, J. Sperger, Amy K Taylor, Katharine E Tippins, Shannon R Reese, Viridiana Carreno, K. Kaufmann, Alex H Chang, Luke A Nunamaker, Charlotte Linebarger, Leilani Mora-Rodriguez, J. Schehr, H. Krause, K. Helzer, M. Bootsma, G. Blitzer, John M Floberg, C. Kyriakopoulos, H. Emamekhoo, Elizabeth I. Heath, Meghan Wells, S. Tagawa, M. Sjöström, A. Choudhury, Menggang Yu, Andrew J. Armstrong, D. Rathkopf, H. Beltran, Peter S Nelson, Felix Y. Feng, S. Dehm, D. Kosoff, Xiao X Wei, Rana R Mckay, Shuang G Zhao, Joshua M Lang
          </td>
          <td>2025-02-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffc5afe786b47a61ca4c6c86b698a4b6f0aa1df3" target='_blank'>
              Characterizing tertiary lymphoid structures associated single-cell atlas in breast cancer patients
              </a>
            </td>
          <td>
            Xiaokai Fan, Daqin Feng, Donggui Wei, Anqi Li, Fangyi Wei, Shufang Deng, Muling Shen, Congzhi Qin, Yongjia Yu, Lun Liang
          </td>
          <td>2025-01-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0871f3b2bbb54a6ef4ad184b9e9c10fad8f88b51" target='_blank'>
              Spatial single-cell maps reveal ST6GAL1 promoting ovarian cancer metastasis.
              </a>
            </td>
          <td>
            Lan-hui Qin, Zijian Jiang, Chongze Yang, Rui Song, Pei-Yin Chen, Weihui Xu, Guanzhen Zeng, , Liling Long
          </td>
          <td>2025-01-30</td>
          <td>Glycoconjugate journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The molecular non-genetic changes of resistance to sotorasib are currently uncertain. The aim of this study was to generate a sotorasib-resistant cell line via selective pressure and systematically examine the molecular and phenotypic alterations caused by resistance.Mutant NCI-H358 (KRASG12C) were exposed to incremental doses (2–512 nM) of sotorasib. Then, resistant clones were separated by single-cell sorting. Proliferation was analyzed in real-time by xCELLigence; protein profiles were quantified by protein arrays; and mRNA expression profile was measured using the PanCancer Pathways panel by NanoString. In silico analyses were conducted from a database comprising patient-derived xenograft (PDX) models and cell lines resistant to sotorasib. AKT and p38. The synergistic effect of combining AKT, p38, and EGFR inhibitors was assessed using the SynergyFinder platform. Additionally, AKT and p38 genes were silenced using esiRNA.Sotorasib-resistant H358-R cell line displayed markers of the mesenchymal-epithelial transition and loss of cell adhesion. Were identified 30 overexpressed genes in the resistance model, implicating in signaling pathways that leads to AKT activation and heightened protein expression levels of phosphorylated AKT and p38. To identify potential therapeutic strategies for overcoming sotorasib resistance, we investigated the combination of AKT and p38 inhibitors. Notably, combined inhibition of AKT (MK2206) and p38 (adezmapimod) restored sensitivity to sotorasib in resistant cell lines, as did silencing AKT expression.These findings underscore the importance of adaptive mechanisms in sotorasib resistance in NSCLC cells contributing by EMT activation and demonstrates synergic combination with AKT and p38 inhibitors to restore sotorasib sensitivity in KRASG12C cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d2b84647241a84761deba81b95c6e8effb443d" target='_blank'>
              Sotorasib resistance triggers epithelial-mesenchymal transition and activates AKT and P38-mediated signaling
              </a>
            </td>
          <td>
            R. A. Megid, G. G. Ribeiro, I. Gomes, A. C. Laus, L. Ferro Leal, Luciane Sussuchi da Silva, A. A. Ariwoola, J. M. Dias, R. Reis, Renato Jose da Silva-Oliveira
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8124e4116b706bf406200aaca77b811cd619c61b" target='_blank'>
              Metabolically activated and highly polyfunctional intratumoral VISTA+ regulatory B cells are associated with tumor recurrence in early-stage NSCLC
              </a>
            </td>
          <td>
            D. Lo Tartaro, Beatrice Aramini, V. Masciale, Nikolaos Paschalidis, F. Lofaro, Anita Neroni, R. Borella, Elena Santacroce, Alin Liviu Ciobanu, A. V. Samarelli, Federica Boraldi, Daniela Quaglino, Alessandra Dubini, Michele Gaudio, G. Manzotti, F. Reggiani, Federica Torricelli, A. Ciarrocchi, A. Neri, Federica Bertolini, Massimo Dominici, Pier Luigi Filosso, F. Stella, L. Gibellini, S. De Biasi, Andrea Cossarizza
          </td>
          <td>2025-01-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659f72bed8c9bc066ac845266c3179b755922c46" target='_blank'>
              Epithelial cell diversity and immune remodeling in bladder cancer progression: insights from single-cell transcriptomics
              </a>
            </td>
          <td>
            Jianpeng Li, Yunzhong Jiang, Minghai Ma, Lu Wang, Minxuan Jing, Zezhong Yang, Mengzhao Zhang, Ke Chen, Jinhai Fan
          </td>
          <td>2025-01-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="As the mechanisms underlying tumorigenesis become better understood, the dynamic roles of cellular components of the tumor microenvironment, and their cross-talk with tumor cells, have come to light as key drivers of disease progression and have emerged as important targets of new cancer therapies. In the field of oncolytic virus (OV) therapy, stromal cells have been considered as potential barriers to viral spread, thus limiting virus replication and therapeutic outcome. However, new evidence indicates that intratumoral fibroblasts could support virus replication. We have demonstrated in a rat model of stromal-rich intrahepatic cholangiocarcinoma (CCA) that vesicular stomatitis virus (VSV) can be localized within intratumoral hepatic stellate cells (HSCs), in addition to tumor cells, when the virus was applied via hepatic arterial infusion. Furthermore, VSV was shown to efficiently kill CCA cells and activated HSCs, and co-culture of CCA and HSCs increased viral titers. Interestingly, this effect is also observed when each cell type is cultured alone in a conditioned medium of the other cell type, indicating that secreted cell factors are at least partially responsible for this phenomenon. Partial reduction in sensitivity to type I interferons was observed in co-culture systems, providing a possible mechanism for the increased viral titers. Together, the results indicate that targeting activated HSCs with VSV could provide an additional mechanism of OV therapy, which, until now has not been considered. Furthermore, these findings suggest that VSV is a potentially powerful therapeutic agent for stromal-rich tumors, such as CCA and pancreatic cancer, both of which are very difficult to treat with conventional therapy and have a very poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33986145e26fb775120df782b699210631812efb" target='_blank'>
              Cross-Talk Between Tumor Cells and Stellate Cells Promotes Oncolytic VSV Activity in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Victoria Neumeyer, Purva Chavan, K. Steiger, O. Ebert, J. Altomonte
          </td>
          <td>2025-02-04</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="CNS-resident microglia and recruited myeloid cells populate the haematoma in intracerebral haemorrhage (ICH), but the contribution of these cell populations to injury and repair has been controversial. To better characterise myeloid cell changes over time, we employed single-cell RNA sequencing to generate a unique paired dataset of live haematoma and peripheral blood samples from 10 ICH patients. This work presents a temporal atlas of ICH myeloid populations and identifies population-specific in-situ transcriptional drivers. In addition to distinct populations of activated microglia and TNF-low microglia, we found a unique and highly activated population of CD14+ monocytes in the haematoma. Perturbation analysis comparing haematoma-associated monocytes to shared populations in blood and haematomas identified TNF signalling as the primary driver of their activation. Employing a custom temporal trajectory analysis based on the embeddings of a single-cell foundation model, we found that this TNF response in monocytes was transient, peaking early after haemorrhage and decreasing over the ensuing 48 hours. Our analysis further identified a transient population of highly activated microglia as the likely source of TNF during the acute stage of ICH. Acute TNF signalling in CD14+ monocytes associated with better adjusted outcomes both in our and external cohorts. Overall, our results suggest that acute TNF signalling between transient populations of activated microglia and unique haematoma-associated monocytes could potentially be beneficial in recovery after ICH. One sentence summary We present a single-cell temporal atlas of myeloid cells in intracerebral haemorrhage and identify TNF signalling as a driver of haematoma dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/538d657ea30d9d88287a93e6069de4c4ded02c18" target='_blank'>
              Single-Cell Temporal Atlas of Myeloid Cells in the Live Haemorrhagic Brain
              </a>
            </td>
          <td>
            Y. Kawamura, Conor W. Johnson, Jonathan DeLong, Lucas Paulo de Lima Camillo, Munetomo Takahashi, H. Beatty, Ryan Herbert, B. Cord, C. Matouk, Michael H. Askenase, L. H. Sansing
          </td>
          <td>2024-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38fe8b7dd282acd4da49fd77a60ef2b609353e0" target='_blank'>
              S100A13-driven interaction between pancreatic adenocarcinoma cells and cancer-associated fibroblasts promotes tumor progression through calcium signaling
              </a>
            </td>
          <td>
            Liuyuan Xia, Xin Guo, Dong Lu, Yixin Jiang, Xiaohui Liang, Yiwen Shen, Jia-Yi Lin, Lijun Zhang, Hongzhuan Chen, Jinmei Jin, Xin Luan, Weidong Zhang
          </td>
          <td>2025-01-28</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed731d817cd7c367b8f1abe8184ac2fe55d110b6" target='_blank'>
              Mapping the developmental trajectory of human astrocytes reveals divergence in glioblastoma.
              </a>
            </td>
          <td>
            Caitlin Sojka, Hsiao-Lin V. Wang, Tarun N. Bhatia, Yangping Li, Pankaj Chopra, Anson Sing, Anna Voss, Alexia King, Feng Wang, Kevin Joseph, V. Ravi, Jeffrey Olson, Kimberly Hoang, E. Nduom, Victor G. Corces, Bing Yao, Steven A. Sloan
          </td>
          <td>2025-01-08</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In this study, we delve into the intrinsic mechanisms of cell communication in hepatocellular carcinoma (HCC). Initially, employing single-cell sequencing, we analyze multiple malignant cell subpopulations and cancer-associated fibroblast (CAF) subpopulations, revealing their interplay through receptor-ligand interactions, with a particular focus on SPP1. Subsequently, employing unsupervised clustering analysis, we delineate two clusters, C1 and C2, and compare their infiltration characteristics using various tools and metrics, uncovering heightened cytotoxicity and overall invasion abundance in C1. Furthermore, our gene risk scoring model indicates heightened activity of the immune therapeutic pathway in C1. Lastly, employing a formulated scoring system, we stratify patients into high and low-risk groups, revealing notably poorer outcomes in the high-risk cohort on Kaplan-Meier curves. Risk scores exhibit a negative correlation with model genes and immune cell infiltration scores, indicating poor prognosis in the high-risk group. Further characterization elucidates the regulatory landscape of the high and low-risk groups across various signaling pathways. In addition, we used wet lab experiments to prove that ABCA1 plays a pro-oncogenic role in hepatocellular carcinoma cells by promoting proliferation, invasion, migration, and reducing apoptosis. In summary, these findings provide crucial insights, offering valuable clues and references for understanding HCC pathogenesis and patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ebd5bca77500ddcf823c450cf7ed50074d3c6d" target='_blank'>
              ABCA1 promote tumor environment heterogeneity via epithelial mesenchymal transition in Huh7 and HepG2 liver cancer cell
              </a>
            </td>
          <td>
            Dinglai Yu, Fang Guo, Qiyu Zhang, Huajun Yu, Wenmin Wang, Yunzhi Chen
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde0e9abf6b50983ec6394f87b77c188d773d609" target='_blank'>
              YAP-driven malignant reprogramming of oral epithelial stem cells at single cell resolution
              </a>
            </td>
          <td>
            F. Faraji, Sydney I. Ramirez, Lauren M. Clubb, Kuniaki Sato, Valeria Burghi, Thomas S. Hoang, Adam Officer, Paola Y. Anguiano Quiroz, William M. G. Galloway, Zbigniew Mikulski, Kate Medetgul-Ernar, P. Marangoni, Kyle B Jones, Yuwei Cao, A. Molinolo, Kenneth Kim, Kanako Sakaguchi, Joseph A. Califano, Quinton Smith, Alon Goren, Ophir D. Klein, Pablo Tamayo, J. S. Gutkind
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="ABSTRACT Background Metastasis is the major cause of cancer‐related mortality. The premetastatic niche is a promising target for its prevention. However, the generality and cellular dynamics in premetastatic niche formation have remained unclear. Aims This study aimed to elucidate the generality and cellular dynamics in premetastatic niche formation. Materials and Methods We performed comprehensive flow cytometric analysis of lung and peripheral immune cells at three time points (early premetastatic, late premetastatic, and micrometastatic phases) for mice with subcutaneous implants of three types of cancer cells (breast cancer, lung cancer, or melanoma cells). The immuno‐cell profiles were then used to predict the metastatic phase by machine learning. Results We found a common pattern of changes in both lung and peripheral immune cell profiles across the three cancer types, including a decrease in the proportion of eosinophils in the early premetastatic phase, an increase in that of regulatory T cells in the late premetastatic phase, and an increase in that of polymorphonuclear myeloid‐derived suppressor cells and a decrease in that of B cells in the micrometastatic phase. Machine learning using immune cell profiles could predict the metastatic phase with approximately 75% accuracy. Discussion Validation of our findings in humans will require data on the presence or absence of micrometastases in patients and the accumulation of comprehensive and temporal information on immune cells. In addition, blood proteins, extracellular vesicles, DNA, RNA, or metabolites may be useful for more accurate prediction. Conclusion The discovery of generalities in premetastatic niche formation allow prediction of metastatic phase and provide a basis for the development of methods for early detection and prevention of cancer metastasis in a cancer type‐independent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f02612a5b53055050e50dcce6e173f615e1f3275" target='_blank'>
              Immune Cell Profiling Reveals a Common Pattern in Premetastatic Niche Formation Across Various Cancer Types
              </a>
            </td>
          <td>
            Shigeaki Sugiyama, Kanae Yumimoto, Keiichi I Nakayama
          </td>
          <td>2024-12-30</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a multifactorial and heterogeneous disease, ranking among the most prevalent malignancies in men. In 2020, there were 1,414,259 new cases of PCa worldwide, accounting for 7.3% of all malignant tumors. The incidence rate of PCa ranks third, following breast cancer and lung cancer. Patients diagnosed with high-grade PCa frequently present with existing or developing metastases, complicating their treatment and resulting in poorer prognoses, particularly for those with bone metastases. Utilizing single-cell RNA sequencing (scRNA-seq), we identified specific malignant cell subtypes that are closely linked to high-grade PCa. By investigating the mechanisms that govern interactions within the tumor microenvironment (TME), we aim to offer new theoretical insights that can enhance the prevention, diagnosis, and treatment of PCa, ultimately striving to improve patient outcomes and quality of life. Methods Data on scRNA-seq was obtained from the GEO database. The gene ontology and gene set enrichment analysis were employed to analyze differential expression genes. Using inferCNV analysis to identify malignant epithelial cells. We subsequently employed Monocle, Cytotrace, and Slingshot packages to infer subtype differentiation trajectories. The cellular communication between malignant cell subtypes and other cells was predicted using the CellChat package. Furthermore, we employed pySCENIC to analyze and identify the regulatory networks of transcription factors (TFs) in malignant cell subtypes. The MDA PCa 2b and VCap cell lines were employed to validate the analysis results through cellular functional experiments. In addition, a risk scoring model was developed to assess the variation in clinical characteristics, prognosis, immune infiltration, immune checkpoint, and drug sensitivity. Results A malignant cell subtype in PCa with high expression of NEFH was identified through scRNA-seq analysis. This subtype was situated at the differentiation terminal, exhibited a higher level of malignancy, and exhibited characteristics that were more prone to advanced tumor lesions. In addition, our research underscored the intricate interactions that exist within the TME, particularly the interaction between PTN secreted by this subtype and fibroblasts via the NCL receptor. This interaction may be closely associated with cancer-associated fibroblasts and tumor progression. Subsequently, we determined that the NEFH+ malignant cell subtype was significantly correlated with the TF IRX4. This TF is linked to a worse prognosis in PCa and may affect disease progression by regulating gene transcription. Our conclusions were additionally verified through cellular experiments. Furthermore, the prognostic model we developed demonstrated satisfactory predictive performance, with gene sets from the high NmRS group facilitating tumor progression and deterioration. The analysis of immune infiltration was instrumental in the development of clinical intervention strategies and patient prognosis. Conclusion By examining the cellular heterogeneity of a unique NEFH+ malignant cell subtype within the PCa microenvironment, we were able to disclose their reciprocal interaction with disease progression. This offers a novel viewpoint on the diagnosis and treatment of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/850edd1c421faf9a184e4ab73c61663fd59239af" target='_blank'>
              Unveiling the NEFH+ malignant cell subtype: Insights from single-cell RNA sequencing in prostate cancer progression and tumor microenvironment interactions
              </a>
            </td>
          <td>
            Jieya Wang, Fu Zhao, Qiang Zhang, Zhoujie Sun, Zhikai Xiahou, Changzhong Wang, Yan Liu, Zongze Yu
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) tumors exhibit extensive genomic, epigenomic, and transcriptional diversity, with significant intratumoral heterogeneity, complicating standard treatment approaches involving radiation (RT) and the DNA-alkylating agent temozolomide (TMZ). In this study, we employed an integrative multi-omics approach, including targeted proteomics, transcriptomics, genomics, and DNA methylation profiling, to investigate the response of a representative panel of GBM patient-derived cancer stem cells (CSCs) to astrocytic differentiation and RT and TMZ treatments. Differentiated CSC progenies retained the expression of key stemness genes and survival pathways, while activating the BMP-Smad signaling pathway and upregulating extracellular matrix components. This was associated with increased resistance to TMZ, though not to RT, across all models. We identified TP53 status as a critical determinant of transcriptional response to both RT and TMZ, which was also modulated by the differentiation state and treatment modality in wildtype (wt) p53 GBM cells. Both mutant and wt p53 models exhibited significant activation of the DNA-damage associated interferon (IFN) response in CSCs and differentiated cells, implicating this pathway in the GBM response to therapy. We observed that activation of NF-κB was positively correlated with the levels of O-6-methylguanine-DNA methyltransferase (MGMT) protein, a direct DNA repair enzyme leading to TMZ resistance, regardless of MGMT promoter methylation status, further supporting the clinical potential for inhibition of NF-kB signaling in GBM treatment. Our integrative analysis of the impact of GBM cell developmental states, in the context of genomic and molecular diversity of patient-derived models, provides valuable insights for pre-clinical studies aimed at optimizing treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a242e45dbcf0e04c69a3eb415e835a6082edb4c" target='_blank'>
              Impact of developmental state, p53 status, and interferon signaling on glioblastoma cell response to radiation and temozolomide treatment
              </a>
            </td>
          <td>
            A. Berezovsky, O. Nuga, Indrani Datta, Kimberly Bergman, Thais S Sabedot, Katherine Gurdziel, S. Irtenkauf, L. Hasselbach, Yuling Meng, Claudius Mueller, Emanuel F. Petricoin, Stephen Brown, N. Purandare, Sidhesh Aras, Tom Mikkelsen, Laila M Poisson, H. Noushmehr, Douglas Ruden, A. deCarvalho
          </td>
          <td>2025-02-07</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90df80a01384ee300b159e30e515e05880476d0f" target='_blank'>
              Artificial intelligence-based spatial analysis of tertiary lymphoid structures and clinical significance for endometrial cancer
              </a>
            </td>
          <td>
            Haruka Suzuki, Kohei Hamada, J. Hamanishi, Akihiko Ueda, R. Murakami, Mana Taki, Rin Mizuno, Koichi Watanabe, Hanako Sato, Y. Hosoe, Hiroaki Ito, Koji Yamanoi, Hiroyuki Yoshitomi, N. Kakiuchi, Ken Yamaguchi, N. Matsumura, Seishi Ogawa, Hideki Ueno, Masaki Mandai
          </td>
          <td>2025-02-01</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAM) enhance cancer progression by promoting angiogenesis, extracellular matrix (ECM) remodeling, and immune suppression. Nerve infiltration is a hallmark of various cancers and is known to directly contribute to tumor growth. However, the role of TAM in promoting intratumoral nerve growth remains poorly understood. In this study, we demonstrate that TAM expressed a distinct “neural growth” gene signature. TAM actively enhance neural growth within tumors and directly promote neurites outgrowth. We identify secreted phosphoprotein 1 (Spp1) as a key mediator of TAM-driven neural growth activity, which triggers neuronal mTORC2 signaling. Leveraging this new neural growth function, which added to the TAM wound healing properties, we explored TAM potential to repair central nervous system. Adoptive transfer of in vitro-generated TAM in a severe complete-compressive-contusive spinal cord injury (scSCI) model, not only repaired the damaged neural parenchyma by improving tissue oxygenation, ECM remodeling, and dampening chronic inflammation, but also resulted in neural regrowth and partial functional motor recovery. Proteomic analysis and subsequent functional validation confirmed that TAM-induced spinal cord regeneration is mediated through the activation of neural mTORC2 signaling pathways. Collectively, our data unveil a previously unrecognized role of TAM in tumor innervation, neural growth, and neural tissue repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3f5750c452f318a87922f48e8ac5ee92b9f34f" target='_blank'>
              Tumor-associated macrophages enhance tumor innervation and spinal cord repair
              </a>
            </td>
          <td>
            S. Dolci, Loris Mannino, Alessandra Campanelli, Eros Rossi, E. Bottani, F. Ciarpella, I. Karkossa, E. Setten, B. Savino, Giulia Pruonto, Nicola Piazza, Stefano Gianoli, A. Amenta, G. Busetto, A. Pezzotta, M. Di Chio, A. Castagna, Nicolò Martinelli, Ilaria Barone, Federico Boschi, A. Doherty, M. Scupoli, C. Cavallini, Giorgio Malpeli, Zulkifal Malik, L. Sagripanti, V. Silani, Patrizia Cristofori, Eugenio Scanziani, Marco Sandri, Anna Pistocchi, P. Bossolasco, M. Endrizzi, Kristin Schubert, G. Fumagalli, Massimo Locati, F. Bifari, I. Decimo
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd1722df9025221021ddfd3a6fcbf675ba9705b" target='_blank'>
              OLIG2 mediates a rare targetable stem cell fate transition in sonic hedgehog medulloblastoma
              </a>
            </td>
          <td>
            Kinjal Desai, Siyi Wanggou, Erika Luis, H. Whetstone, Chunying Yu, R. Vanner, Hayden J. Selvadurai, Lilian Lee, Jinchu Vijay, J. E. Jaramillo, Jerry J. Fan, P. Guilhamon, M. Kushida, Xuejun Li, Gregory Stein, Santosh Kesari, B. D. Simons, Xi Huang, Peter B. Dirks
          </td>
          <td>2025-02-04</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Cell and tissue movement in development, cancer invasion, and immune response relies on chemical or mechanical guidance cues. In many systems, this behavior is locally guided by self-generated signaling gradients rather than long-range, pre-patterned cues. However, how heterogeneous mixtures of cells interact non-reciprocally and navigate through self-generated gradients remains largely unexplored. Here we introduce a theoretical framework for the self-organized chemotaxis of heterogeneous cell populations. We find that relative chemotactic sensitivity of cell populations controls their long-time coupling and co-migration dynamics, with boundary conditions such as external cell- and attractant reservoirs substantially influencing the migration patterns. We further predict an optimal parameter regime that leads to robust and colocalized migration. We test our theoretical predictions with in vitro experiments demonstrating the co-migration of different immune cell populations, and quantitatively reproduce observed migration patterns under wild-type and perturbed conditions. Interestingly, immune cell co-migration appears close to the predicted optimal regime. Finally, we incorporate mechanical interactions into our framework, revealing a phase diagram that illustrates non-trivial interplays between chemotactic and mechanical non-reciprocity in driving collective migration. Together, our findings suggest that self-generated chemotaxis is a robust strategy for emergent multicellular navigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1f57ee1bc013b72202f77f0ea69bbd4c759c18c" target='_blank'>
              Self-generated chemotaxis of mixed cell populations
              </a>
            </td>
          <td>
            Mehmet Can Uçar, Zane Alsberga, Jonna Alanko, M. Sixt, É. Hannezo
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Epithelial cancers are typically heterogeneous with primary prostate cancer being a typical example of histological and genomic variation. Prior studies of primary prostate cancer tumour genetics revealed extensive inter and intra-patient genomic tumour heterogeneity. Recent advances in machine learning have enabled the inference of ground-truth genomic single-nucleotide and copy number variant status from transcript data. While these inferred SNV and CNV states can be used to resolve clonal phylogenies, however, it is still unknown how faithfully transcript-based tumour phylogenies reconstruct ground truth DNA-based tumour phylogenies. We sought to study the accuracy of inferred-transcript to recapitulate DNA-based tumour phylogenies. We first performed in-silico comparisons of inferred and directly resolved SNV and CNV status, from single cancer cells, from three different cell lines. We found that inferred SNV phylogenies accurately recapitulate DNA phylogenies (entanglement = 0.097). We observed similar results in iCNV and CNV based phylogenies (entanglement = 0.11). Analysis of published prostate cancer DNA phylogenies and inferred CNV, SNV and transcript based phylogenies demonstrated phylogenetic concordance. Finally, a comparison of pseudo-bulked spatial transcriptomic data to adjacent sections with WGS data also demonstrated recapitulation of ground truth (entanglement = 0.35). These results suggest that transcript-based inferred phylogenies recapitulate conventional genomic phylogenies. Further work will need to be done to increase accuracy, genomic, and spatial resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79d56f0a1a843a737c66bbddbc1e662cc71b163e" target='_blank'>
              Clonal phylogenies inferred from bulk, single cell, and spatial transcriptomic analysis of epithelial cancers
              </a>
            </td>
          <td>
            Andrew Erickson, Sandy Figiel, Timothy Rajakumar, Srinivasa Rao, Wencheng Yin, D. Doultsinos, Anette Magnussen, Reema Singh, Ninu Poulose, Richard J. Bryant, Olivier Cussenot, Freddie C Hamdy, Dan Woodcock, I. Mills, Alastair D. Lamb
          </td>
          <td>2025-01-03</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e7693290b0f56e7e4e347ce73c837cd2d47cd1" target='_blank'>
              Stromal architecture and fibroblast subpopulations with opposing effects on outcomes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yifei Cheng, Xiaofang Chen, Li Feng, Zhicheng Yang, Liyun Xiao, Bin Xiang, Xiaodong Wang, Dongbin Liu, Penghui Lin, Jieyi Shi, Guohe Song, Wulei Qian, Boan Zhang, Yanan Xu, Zheng Gao, Lv Chen, Y. Wu, Jiaqiang Ma, Youpei Lin, Haichao Zhao, Lihua Peng, Xuebin Mao, Yang Liu, Hao Hou, Mingyu Yang, Yuan Ji, Xiaoying Wang, Jian Zhou, Xun Xu, Xiyang Liu, Wu Wei, Xiaoming Zhang, Qiang Gao, Hu Zhou, Yidi Sun, Kui Wu, Jia Fan
          </td>
          <td>2025-01-28</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The role of glioma-associated myeloid cells in tumor growth and immune evasion remains poorly understood. We performed single-cell RNA sequencing of immune and tumor cells from 33 gliomas, identifying two distinct myeloid-derived suppressor cell (MDSC) populations in isocitrate dehydrogenase-wild-type (IDT-WT) glioblastoma: an early progenitor MDSC (E-MDSC) population with up-regulation of metabolic and hypoxia pathways and a monocytic MDSC (M-MDSC) population. Spatial transcriptomics demonstrated that E-MDSCs geographically colocalize with metabolic stem-like tumor cells in the pseudopalisading region. Ligand-receptor analysis revealed cross-talk between these cells, where glioma stem-like cells produce chemokines attracting E-MDSCs, which in turn produce growth factors for the tumor cells. This interaction is absent in IDH-mutant gliomas, associated with hypermethylation and repressed gene expression of MDSC-attracting chemokines. Our study elucidates specific MDSCs that may facilitate glioblastoma progression and mediate tumor immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c3594c9527f59638cf5b92d7a7853cdf4cfa76" target='_blank'>
              Distinct myeloid-derived suppressor cell populations in human glioblastoma.
              </a>
            </td>
          <td>
            Christina Jackson, Chris C. Cherry, Sadhana Bom, Arbor G. Dykema, Rulin Wang, Elizabeth A Thompson, Ming Zhang, Runzhe Li, Zhicheng Ji, Wenpin Hou, Wentao Zhan, Hao Zhang, John Choi, A. Vaghasia, Landon Hansen, W. Wang, Brandon Bergsneider, Kate M Jones, Fausto Rodriguez, Jon D. Weingart, Calixto-Hope G Lucas, Jonathan Powell, Jennifer H. Elisseeff, Srinivasan Yegnasubramanian, Michael Lim, C. Bettegowda, Hongkai Ji, Drew M Pardoll
          </td>
          <td>2025-01-17</td>
          <td>Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d017ae1208a1788cb6dfdbf710e4c68d672f7f57" target='_blank'>
              Network-based transfer of pan-cancer immunotherapy responses to guide breast cancer prognosis
              </a>
            </td>
          <td>
            Xiaobao Ding, Lin Zhang, M. Fan, Lihua Li
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92e2f5cabd9e2a4c286349a54228e7bf25b75d61" target='_blank'>
              E2F1-driven CENPM expression promotes glycolytic reprogramming and tumorigenicity in glioblastoma
              </a>
            </td>
          <td>
            Zhiqiang Yi, Yanfei Jia, Runchun Lu, Chunwei Li, Long Wen, Xiangdong Yin, Junfei Yi, Liang Li
          </td>
          <td>2024-12-21</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Personalized precision medicine (PPPM) in cancer immunology and oncology is a rapidly advancing field with significant potential. Gliomas, known for their poor prognosis, rank among the most lethal brain tumors. Despite advancements, there remains a critical need for precise, individualized treatment strategies.We conducted a comprehensive analysis of RNA-seq and microarray data from the TCGA and GEO databases, supplemented by single-cell RNA sequencing (scRNA-seq) data from glioma patients. By integrating single-cell sequencing analysis with foundational experiments, we investigated the molecular variations and cellular interactions within neural glioma cell subpopulations during tumor progression.Our single-cell sequencing analysis revealed distinct gene expression patterns across glioma cell subpopulations. Notably, differentiation trajectory analysis identified NUSAP1 as a key marker for the terminal subpopulation. We found that elevated NUSAP1 expression correlated with poor prognosis, prompting further investigation of its functional role through both cellular and animal studies.NUSAP1-based risk models hold potential as predictive and therapeutic tools for personalized glioma treatment. In-depth exploration of NUSAP1’s mechanisms in glioblastoma could enhance our understanding of its response to immunotherapy, suggesting that targeting NUSAP1 may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5625f39e4872fec67e5df1a449a03f506dae3b79" target='_blank'>
              Single-cell sequencing elucidates the mechanism of NUSAP1 in glioma and its diagnostic and prognostic significance
              </a>
            </td>
          <td>
            Meng-Yu Zhao, , Hongzhen Dai, Wan-Yan Xu, Li-Na Wang, Yu- Gu, Jie-Hui Zhao, , Cun-Zhi Wang, Jia-feng Xu, Guan-Jun Chen, , Wen-Ming Hong, Fan Zhang
          </td>
          <td>2025-02-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The three most prevalent malignant skin tumors are basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and cutaneous melanoma (CM). While BCC and SCC generally exhibit low levels of malignancy, some tumors can become invasive and metastasize. In contrast, CM is the most malignant and lethal form of skin cancer among these three. In recent years, the increasing incidence of skin tumors has garnered significant attention within the medical community. The advent of single-cell RNA sequencing (scRNA-seq) technology has revolutionized the analysis of the tumor microenvironment (TME) in skin tumors, offering novel insights for clinical research. In this review, we provide a concise introduction to scRNA-seq technology and its application in delineating the heterogeneity of the TME in skin tumors, elucidating the mechanisms of tumor progression and pathogenicity, and uncovering new therapeutic targets. Our review offers a comprehensive overview for researchers, offering insights that may advance the understanding of the skin TME in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c806552e713787b83ea0933cd31a52d109ce5cf9" target='_blank'>
              Advances and applications of single-cell RNA sequencing in deciphering the tumor microenvironment of malignant skin tumors
              </a>
            </td>
          <td>
            Yiting Feng, Lanlan Liu, Chunlan Hu, Jie Sun, Mingzhu Yin, Xin Li
          </td>
          <td>2025-01-10</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Intracellular signaling pathways form networks through which information is transmitted, often in the form of kinase-mediated phosphorylation events, to interpret extracellular signals and elicit appropriate cellular responses. Yet, even isogenic cells in a homogenous environment show heterogeneity in their intracellular “cell-state”, as well as their response to extracellular signals. Here, we aimed to better understand this relation between these phenomena by investigating how information flows through the EGF-receptor centered network upon targeted drug treatment, and how this is affected by cell-to-cell-state differences. Using single-cell ID-seq, we profiled the cell-state and signaling activity of primary human epidermal stem cells by measuring 69 (phospho-)proteins upon inhibition of the Erk/MAPK (p90RSK) and Akt/mTOR (p70S6K) routes downstream of the EGF pathway. We found that the effects of drug treatment propagated from the EGF-signaling pathway to other connected parts of the cellular signaling network, indicating altered signaling flow. We identified nine distinct cell-states that show pervasive state-dependent drug-responses for many (phospho-)proteins. Computational modeling of the signaling network using single-cell Comparative Network Reconstruction showed that many interactions between phospho-proteins (i.e. network wiring) were quantitatively different between cell-states. Furthermore, (phospho-)proteins with a cell-state dependent drug response, were more likely to be involved in interactions that showed a cell-state dependent strength. Overall, our results indicate that drug treatment response and signaling interactions between proteins are closely related and modulated by cell-state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5aafd958806435b041f6f614bd7ef297f3ec839a" target='_blank'>
              Cell-state specific drug-responses are associated with differences in signaling network wiring
              </a>
            </td>
          <td>
            Niels Krämer, Roderick van Eijl, Tim Stohn, Sabine E. J. Tanis, Lodewyk F. A. Wessels, Evert Bosdriesz, Klaas W. Mulder
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c52c76852d3ba58984faf618de957dc0aff431e" target='_blank'>
              Spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis.
              </a>
            </td>
          <td>
            A. Vannan, Ruqian Lyu, Arianna L Williams, N. Negretti, E. D. Mee, J. Hirsh, S. Hirsh, Niran Hadad, David Nichols, C. Calvi, Chase J Taylor, V. Polosukhin, A. P. Serezani, A. S. McCall, J. Gokey, Heejung Shim, Lorraine B. Ware, M. Bacchetta, C. Shaver, Timothy S. Blackwell, Rajat Walia, J. Sucre, JA Kropski, Davis J. McCarthy, N. Banovich
          </td>
          <td>2025-02-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The tumor microenvironment (TME) milieu directs a plethora of tumor-modulating functions. Recent years have seen pivotal breakthroughs in our understanding of the TME's role in tumor initiation and progression, with tangible clinical applications. Individual components of the TME exert their function predominantly by cell-cell crosstalk, both in the form of physical interaction and secreted factors. Notably, different spatial niches represent exclusive signaling hubs in the TME, propagating pro- or antitumoral functions. The exploration of these interactions has been vastly facilitated by novel molecular technologies, each of which provides a different perspective on this intricate intercellular communication network. Together, these complementary methods paint a detailed, high-resolution map of the TME's interaction landscape. In this viewpoint, we explore how cellular interactions can unlock a new level of understanding of TME complexity, and discuss the promises and challenges of characterizing tumors based on their cellular interaction footprint.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/072aea9d500a6665bd7f3f737b00c34169b4d84a" target='_blank'>
              Decoding multicellular interaction networks-a new horizon in tumor microenvironment research.
              </a>
            </td>
          <td>
            Roi Balaban, Merav Cohen
          </td>
          <td>2025-02-05</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Purpose Papillary renal cell carcinoma (pRCC), the second most common subtype of renal cancer, exhibits heterogeneity in molecular features and response to targeted therapies, including antiangiogenic drugs. Discovering molecular biomarkers able to stratify pRCC patients into clinically relevant subgroups and understanding the underlying mechanisms are urgently needed to advance precision medicine. Here, we molecularly dissect a large pRCC series through the expression of the targets of hypoxia inducible factors (HIF), master regulators of angiogenesis, metabolic reprogramming and immune microenvironment. Experimental Design We merged and analyzed multi-omic and clinical data of 346 patients derived from two pRCC series (TCGA-KIRP and a Spanish metastatic series). Altered pathways and differences in tumor microenvironment were identified through tumor transcriptomic analyses. Tumor metabolome analysis was performed in selected cases. Results Molecular revision of driver mutations classified 302 patients as pRCC, while uncovering misclassified cases. Analysis of HIF targets gene expression identified a subset of pRCC tumors with increased HIF activity (31%; “HIF-active”). These tumors were characterized by high hypoxia scores, increased angiogenesis, low expression of Krebs cycle genes and mitochondrial activity, high immune infiltration and increased epithelial-mesenchymal transition (P<0.0001; each feature) and they were associated with increased metastasis risk and worse overall survival (P<0.005). Metabolome analyses revealed that HIF-active tumors accumulated L-2-hydroxyglutarate (L-2-HG), an oncometabolite that leads to pseudohypoxia by inhibiting HIF prolyl hydroxylases and preventing HIFα degradation. L-2-HG-accumulation agreed with diminished L2HGDH expression, associated with losing one copy of the gene and low expression of PPARGC1A, which regulates its transcription. Conclusions HIF-pathway overactivation defines a pseudohypoxic pRCC molecular subgroup characterized by high angiogenesis, immune infiltration and aggressive features. These findings advance our understanding of pRCC diversity, revealing potential clues for the variable response of patients to targeted therapies and paving the way to more personalized pRCC treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b17c0ff397a2a18624c6810af2df85c7a525f01" target='_blank'>
              Identification of a novel papillary renal cell carcinoma molecular subtype characterized by HIF-pathway over-activation
              </a>
            </td>
          <td>
            Javier de Nicolás-Hernández, Alicia Arenas, Carlos Valdivia, María Santos, Javier Lanillos, Nicole Bechmann, M. Peitzsch, Eduardo Gil-Vilariño, María Monteagudo, Sara Mellid, C. Montero‐Conde, A. Cascón, L. Leandro-García, G. de Velasco, Ignacio Durán, Javier Puente, J. García-Donás, Eduardo Caleiras, Mercedes Robledo, Cristina Rodríguez-Antona
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The kinetic parameters of cancer population dynamics are critical for developing reliable predictors of tumour growth patterns, extracting metrics for patient stratification and creating algorithms that can forecast clinically significant events. Here, we introduce a model-based Bayesian framework that leverages longitudinal phenotypic (e.g., tumour volume, cell counts) or genotypic (e.g., mutation frequency) data to infer critical parameters of tumour progression within a single patient. Our models uses population genetics to estimate probability distributions for tumour growth rates, initiation and extinction times, pinpointing abrupt shifts in tumour dynamics due to treatment response and revealing associations between drug resistance and pre-existing cancer cell populations. We apply our framework to address pivotal clinical questions across three major cancer types. In colorectal cancer, we use tumour markers data to identify extensive pre-existing RAS-linked resistance to cetuximab. In lung cancer, we use somatic mutation frequencies in citculating tumour DNA to determine prognostic growth rates and develop a test for monitoring minimal residual disease. In chronic lymphocytic leukaemia, we use white blood cell counts to stratify patients by growth patterns and predict time to treatment, advancing adaptive monitoring strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e324d640559e2fae6f8be8e484eb8d56fee86230" target='_blank'>
              Model-based Bayesian inference of cancer dynamics from heterogenous longitudinal data
              </a>
            </td>
          <td>
            Giovanni Santacatterina, R. Bergamin, A. Zucchetto, Roberta Laureana, M. D. del Principe, V. Gattei, Leonardo Egidi, G. Caravagna
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4843e60a35dd8061e986640d8a344bfa3d97b50" target='_blank'>
              Cancer Clones Revised
              </a>
            </td>
          <td>
            L. Laplane
          </td>
          <td>2024-12-23</td>
          <td>Biological Theory</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CAR-T) cell therapy has emerged as a promising treatment for hematological malignancies, offering a targeted approach to cancer treatment. Understanding the complexities of CAR-T cell therapy within solid tumors poses challenges due to the intricate interactions within the tumor microenvironment. Mathematical modeling may serve as a valuable tool to unravel the dynamics of CAR-T cell therapy and improve its effectiveness in solid tumors. This study aimed to investigate the impact of spatial aspects in CAR-T therapy of solid tumors, utilizing cellular automata for modeling purposes. Our main objective was to deepen our understanding of treatment effects by analyzing scenarios with different spatial distributions and varying the initial quantities of tumor and CAR-T cells. Tumor geometry significantly influenced treatment efficacy in-silico, with notable differences observed between tumors with block-like arrangements and those with sparse cell distributions, leading to the concept of immune suppression due to geometrical effects. This research delves into the intricate relationship between spatial dynamics and the effectiveness of CAR-T therapy in solid tumors, highlighting the relevance of tumor geometry in the outcome of cellular immunotherapy treatments. Our results provide a basis for improving the efficacy of CAR-T cell treatments by combining them with other ones reducing the density of compact tumor areas and thus opening access ways for tumor killing T-cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b56368d398e6734b66e55599843f0e9cefd36d23" target='_blank'>
              Geometric immunosuppression in CAR-T cell treatment: Insights from mathematical modeling
              </a>
            </td>
          <td>
            Silvia Bordel-Vozmediano, S. Sabir, Luc'ia Benito-Barca, B. Weigelin, V'ictor M. P'erez-Garc'ia Mathematical Oncology Laboratory, Instituto de Matem'atica Aplicada a la Ciencia y la Ingenier'ia, U. C. Mancha, Ciudad Real, Spain., Departamento de Matem'aticas, Escuela T'ecnica Superior de Ingenieros Industriales, Facultad de Ciencias Experimentales, Universidad Francisco de Vitoria, Pozuelo de Alarc'on, Department of Functional Imaging, Radiopharmacy, Multiscale Immunoimaging, E. University, Tubingen, Germany, Cluster of Excellence iFIT ''Image-Guided, Functionally Instructed Tumor Therapies''
          </td>
          <td>2025-01-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da415d9954dd04158cfcfaa83dc1880ca38c0523" target='_blank'>
              Single-cell and spatial transcriptomics reveal SPP1-CD44 signaling drives primary resistance to immune checkpoint inhibitors in RCC
              </a>
            </td>
          <td>
            Junfeng Zhang, Qingyan Peng, Jin Fan, Fuzhong Liu, Hongbo Chen, X. Bi, Shuai Yuan, Wei Jiang, Ting Pan, Kailing Li, Sihai Tan, Peng Chen
          </td>
          <td>2024-12-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung cancer remains the primary cause of cancer-related mortality, with factors such as postoperative tumor recurrence, metastasis, and therapeutic drug resistance exacerbating patient outcomes. Immunotherapy has emerged as a transformative approach, challenging conventional treatment paradigms for lung cancer. Consequently, advancing research in lung cancer immunotherapy is imperative. Recent studies indicate that numerous regulators within the tumor microenvironment (TME) drive tumor angiogenesis and epithelial-mesenchymal transition (EMT); these processes are interdependent, reciprocal, and collectively contribute to tumor progression. Tumor angiogenesis not only supplies adequate oxygen and nutrients for cellular proliferation but also establishes pathways facilitating tumor metastasis and creating hypoxic regions that foster drug resistance. Concurrently, EMT enhances metastatic potential and reinforces drug-resistance genes within tumor cells, creating a reciprocal relationship with angiogenesis. This interplay ultimately results in tumor invasion, metastasis, and therapeutic resistance. This paper reviews key regulators of angiogenesis and EMT, examining their impact on lung cancer immunotherapy and progression, and investigates whether newly identified regulators could influence lung cancer treatment, thus offering valuable insights for developing future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dc8e9c9813786849cd1f6fcf265781fb899c57a" target='_blank'>
              Angiogenesis and EMT regulators in the tumor microenvironment in lung cancer and immunotherapy
              </a>
            </td>
          <td>
            Taotao Yan, Jiahai Shi
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c214c85284c0f7c6d4f85e6f9096b58eba34d42" target='_blank'>
              Dissecting tumor transcriptional heterogeneity from single-cell RNA-seq data by generalized binary covariance decomposition.
              </a>
            </td>
          <td>
            Yusha Liu, P. Carbonetto, Jason Willwerscheid, Scott A. Oakes, Kay F Macleod, Matthew Stephens
          </td>
          <td>2025-01-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="Epithelial cell collectives migrate through tissue interfaces and crevices to orchestrate development processes, tumor invasion, and wound healing. Naturally, the traversal of cell collective through confining environments involves crowding due to narrowing spaces, which seems tenuous given the conventional inverse relationship between cell density and migration. However, the physical transitions required to overcome such epithelial densification for migration across confinements remain unclear. Here, in a system of contiguous microchannels of varying confinements, we show that epithelial (MCF10A) monolayers accumulate higher cell density and undergo fluid-like shape transitions before entering narrower channels. However, overexpression of breast cancer oncogene ErbB2 did not require such accumulation of cell density to migrate across matrix confinement. While wild-type MCF10A cells migrated faster in narrow channels, this confinement sensitivity was reduced after +ErbB2 mutation or with constitutively active RhoA. This physical interpretation of collective cell migration as density and shape transitions in granular matter could advance our understanding of complex living systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a4a64576188154027b2ea1715663784bbe17d31" target='_blank'>
              Gradients in cell density and shape transitions drive collective cell migration into confining environments
              </a>
            </td>
          <td>
            Wan-Jung Lin, Hongsheng Yu, Amit Pathak
          </td>
          <td>2024-12-30</td>
          <td>Soft Matter</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Liver cancer exhibits diverse molecular characteristics and distinct immune cell infiltration patterns, which significantly influence patient outcomes. In this study, we thoroughly examined the liver cancer tumor environment by analyzing data from 419,866 individual cells across nine datasets involving 99 patients. By categorizing patients into different groups based on their immune cell profiles, including immune deficiency, B cells-enriched, T cells-enriched and macrophages-enriched, we better understood how these cells change in various patient subgroups. Our investigation of liver metastases from intestinal cancer uncovered a group of mast cells that might promote metastasis through pathways like inositol phosphate metabolism. Using genomic and clinical data from The Cancer Genome Atlas, we identified specific cell components linked to tumor characteristics and genetics. Our detailed study of cancer-associated fibroblasts (CAFs) revealed how they adapt and acquire new functions in the tissue environment, highlighting their flexibility. Additionally, we found a significant connection between CAF-related genes and the prognosis of hepatocellular carcinoma patients. This research provides valuable insights into the makeup of the liver cancer tumor environment and its profound impact on patient outcomes, offering fresh perspectives for managing this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dff3872da8e8b83984c596296a89bdd2fda835d8" target='_blank'>
              Deciphering the heterogeneity and plasticity of the tumor microenvironment in liver cancer provides insights for prognosis
              </a>
            </td>
          <td>
            Yihao Sun, Guojuan Shi, Jian Yang, Chun-Zhong Zhou, Chuhan Peng, Yu-Hong Luo, Ying Pan, Rui-Qi Wang
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) techniques hold great value in evaluating the heterogeneity and spatial characteristics of hematopoietic cells within tissues. These two techniques are highly complementary, with scRNA-seq offering single-cell resolution and ST retaining spatial information. However, there is an urgent demand for well-organized and user-friendly toolkits capable of handling single-cell and spatial information. Here, we present HemaScope, a specialized bioinformatics toolkit featuring modular designs to analyze scRNA-seq and ST data generated from hematopoietic cells. It enables users to perform quality control, basic analysis, cell atlas construction, cellular heterogeneity exploration, and dynamical examination on scRNA-seq data. Also, it can perform spatial analysis and microenvironment analysis on ST data. Meanwhile, HemaScope takes into consideration hematopoietic cell-specific features, including lineage affiliation evaluation, cell cycle prediction, and marker gene collection. To enhance the user experience, we have deployed the toolkit in user-friendly forms: HemaScopeR (an R package), HemaScopeCloud (a web server), HemaScopeDocker (a Docker image), and HemaScopeShiny (a graphical interface). In case studies, we employed it to construct a cell atlas of human bone marrow, analyze age-related changes, and identify acute myeloid leukemia cells in mice. Moreover, we characterized the microenvironments in angioimmunoblastic T cell lymphoma and primary central nervous system lymphoma, elucidating tumor boundaries. HemaScope is freely available at https://zhenyiwangthu.github.io/HemaScope_Tutorial/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93bd105f231e9996b6a3ab55c7d2018c2946edbd" target='_blank'>
              HemaScope: A Tool for Analyzing Single-cell and Spatial Transcriptomics Data of Hematopoietic Cells.
              </a>
            </td>
          <td>
            Zhenyi Wang, Yuxin Miao, Hongjun Li, Wenyan Cheng, Minglei Shi, Lv Gang, Yating Zhu, Junyi Zhang, Tingting Tan, Jin Gu, Michael Q. Zhang, Jianfeng Li, Hai Fang, Zhu Chen, Saijuan Chen
          </td>
          <td>2025-01-25</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma multiforme (GBM) is one of the deadliest and most heterogeneous forms of brain cancer, characterized by its resistance to conventional therapies. Within GBM, a subpopulation of slow-cycling cells, often linked to quiescence and stemness, plays a crucial role in treatment resistance and tumor recurrence. This study aimed to identify novel biomarkers associated with these slow-cycling GBM cells.


METHODS
We utilized The Cancer Genome Atlas (TCGA)-GBM dataset and presented the reproducible bioinformatics analysis for our results.


RESULTS
Our analysis highlighted Membrane-Associated Protein 17 (MAP17) as strongly associated with the slow-cycling phenotype. We found that the protein cargo MAP17 expression is related to mesenchymal signatures and stem cell-related pathways. Also, MAP17 was linked to a distinct metabolic profile, characterized by significant enrichment in pathways related to folate, zinc, and fatty acids. Moreover, the immune cell distribution analysis revealed that MAP17 correlates with key molecular immune processes, including interferon-gamma (IFN-γ) signaling and antigen presentation, as well as immunosuppressive cells like myeloid-derived suppressor cells (MDSCs) and macrophages. MAP17-high tumors also showed elevated expression of several immune checkpoint inhibitors, indicating an immunosuppressive microenvironment.


CONCLUSION
These findings provide insight into the role of MAP17 in quiescence, stemness, and immune evasion, positioning it as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15fc0b51c9baf70fb9322e4160d56c406b168bfc" target='_blank'>
              Glioblastoma Stem Cells: MAP17 as a Novel Predictive Biomarker and Therapeutic Target Associated with Quiescence and Immune Evasion.
              </a>
            </td>
          <td>
            S. Aghamiri, Rada Amin
          </td>
          <td>2025-01-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59e5b41f611d8bd26c1558060c6ef330913ca3b" target='_blank'>
              Pan-cancer drivers of metastasis
              </a>
            </td>
          <td>
            Ryan Lusby, Engin Demirdizen, Mohammed M A Inayatullah, Paramita Kundu, O. Maiques, Ziyi Zhang, Mikkel G Terp, Victoria Sanz-Moreno, Vijay K. Tiwari
          </td>
          <td>2025-01-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Understanding how intratumoral immune populations coordinate antitumor responses after therapy can guide treatment prioritization. We systematically analyzed an established immunotherapy, donor lymphocyte infusion (DLI), by assessing 348,905 single-cell transcriptomes from 74 longitudinal bone marrow samples of 25 patients with relapsed leukemia; a subset was evaluated by both protein- and transcriptome-based spatial analysis. In acute myeloid leukemia (AML) DLI responders, we identified clonally expanded ZNF683+ CD8+ cytotoxic T lymphocytes with in vitro specificity for patient-matched AML. These cells originated primarily from the DLI product and appeared to coordinate antitumor immune responses through interaction with diverse immune cell types within the marrow microenvironment. Nonresponders lacked this cross-talk and had cytotoxic T lymphocytes with elevated TIGIT expression. Our study identifies recipient bone marrow microenvironment differences as a determinant of an effective antileukemia response and opens opportunities to modulate cellular therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dc256708884cd08679466f87592f86695fa33d1" target='_blank'>
              Coordinated immune networks in leukemia bone marrow microenvironments distinguish response to cellular therapy.
              </a>
            </td>
          <td>
            K. Maurer, Cameron Y Park, Shouvik Mani, M. Borji, Florian Raths, Kenneth H Gouin, L. Penter, Yinuo Jin, Jia Yi Zhang, Crystal Shin, James R Brenner, Jackson Southard, Sachi Krishna, Wesley S Lu, Haoxiang Lyu, Domenic Abbondanza, Chanell Mangum, Lars Rønn Olsen, Michael J Lawson, Martin M. Fabani, D. Neuberg, P. Bachireddy, Eli N Glezer, Samouil L. Farhi, Shuqiang Li, K. Livak, Jerome Ritz, Robert J. Soiffer, Catherine J. Wu, Elham Azizi
          </td>
          <td>2025-01-24</td>
          <td>Science immunology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background We conducted an investigation into the characteristics of single-cell differentiation data in gliomas, with a focus on developing DAPK1-based prognostic markers to predict patient outcomes. Dysregulated expression of DAPK1 has been associated with the invasive behavior of various malignancies, including gliomas. However, the precise role and underlying mechanisms of DAPK1 in gliomas remain inadequately understood. Methods We performed analyses on RNA-seq and microarray datasets from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO), in addition to single-cell RNA sequencing (scRNA-seq) data from glioma patients available in GEO. Utilizing the Seurat R package, we identified gene clusters associated with survival from the scRNA-seq data. Prognostic models were developed using LASSO and stepwise regression algorithms. Furthermore, we assessed the predictive potential of these genes within the immune microenvironment and their relevance in immunotherapy contexts. Results Our scRNA-seq data analysis revealed 32 distinct cell clusters corresponding to 10 cell types. Through dimensionality reduction and clustering, we identified three glial cell subpopulations based on their differentiation trajectories. DAPK1, serving as a marker gene for the terminal subpopulation, exhibited an association with poor prognosis. Conclusions DAPK1-based prognostic models show promise for accurately predicting outcomes in glioblastoma and glioma. An in-depth examination of DAPK1’s specific mechanisms in glioblastoma could elucidate its role in immunotherapy response. Targeting the DAPK1 gene may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fd67f17577e9d1b4649b5fe0ecd703f94585d" target='_blank'>
              Single-cell sequencing uncovers the mechanistic role of DAPK1 in glioma and its diagnostic and prognostic implications
              </a>
            </td>
          <td>
            Tian-Hang Yu, Yan-Yu Ding, Si-Guo Zhao, Jie-Hui Zhao, Yu- Gu, Dong-Hui Chen, Fan Zhang, Wen-Ming Hong
          </td>
          <td>2025-01-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The molecular features determining the risk of metachronous metastases in clear cell renal cell carcinoma (ccRCC) are poorly defined. Using a systematic tumor transcriptome deconvolution approach, we investigated the genomic and transcriptomic profiles of 192 ccRCC primary tumors with extended clinical follow-up to identify cancer and stromal cell molecular features associated with metastatic risk. At the genomic level, we identified a significantly higher frequency of copy number loss at 1p31-36 in primary tumors that later progressed with metastases. Tumor transcriptome deconvolution identified significant down-regulation of epithelial cell polarity, including PATJ (1p31), and fatty acid metabolism, including CYP4A11 (1p33), in cancer cells of tumors that developed metastatic progression. We developed and benchmarked a compact 5-feature predictive model (5G) that demonstrated improved accuracy over existing ccRCC gene signatures in the prediction of metachronous metastasis risk. Overall, our study highlights convergent genomic and transcriptomic alterations in chromosome 1p, driving dysregulation of epithelial cell polarity and fatty acid metabolism, as putative novel risk factors of metachronous metastasis in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b85898b20987df3b296ae9761d05ca0764e6d312" target='_blank'>
              Convergent genomic and molecular features predict risk of metachronous metastasis in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            M. Naeini, Mengyuan Pang, Neha Rohatgi, Sinem Kadioglu, Umesh Ghoshdastider, R. DiNatale, Roy Mano, A. Hakimi, A. Skanderup
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Understanding early tumor formation offers critical insights for cancer prevention at the earliest stages. However, most in vivo studies focus on later stages, providing limited information on initial tumorigenesis. Hypoxia, a common physiological condition, is a primary microenvironmental factor in cancer. While molecular studies indicate significant yet sometimes conflicting roles of hypoxia in cancer progression, experimental data on hypoxia’s impact on tumor formation from a single cell, especially regarding cellular dynamics, remain scarce. To address this gap, we developed an integrated platform for live-cell imaging under hypoxic conditions, enabling, for the first time, observation of 3D tumor spheroid development from a single cell. Our results show that tumorigenesis is markedly accelerated under hypoxia. Beyond linear growth, we demonstrate that directional movement enhances fusion events, leading to larger spheroids that, in turn, stimulate further proliferation — a positive feedback loop between spheroid dynamics and growth. Actin polymerization and cell-cell adhesion are optimized to promote spheroid fusion. Additionally, hypoxia induces cells with outreaching F-actin co-localized with smaller nuclei to bridge gaps between spheroids, facilitating fusion, while the appearance of tiny nuclei suggests genetic instability. These findings provide direct insights into hypoxia’s role in early tumorigenesis, highlighting the potential of modulating dynamic, mechanical, and physiological factors for cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0153fc66d9ff4cbf61d2a83aea2c920a18ecaa1" target='_blank'>
              Hypoxia-Induced Active Dynamics Promotes Early Tumorigenesis
              </a>
            </td>
          <td>
            Zhengduo Wang, Li Tian, Bo Li
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97569fe84c28d5e70fd4d5d44460cfdc4508b03d" target='_blank'>
              Quantifying cell divisions along evolutionary lineages in cancer.
              </a>
            </td>
          <td>
            Martin Blohmer, David M Cheek, Wei-Ting Hung, Maria Kessler, Foivos Chatzidimitriou, Jiahe Wang, William Hung, , Alexander N Gorelick, Emma Ce Wassenaar, Ching-Yeuh Yang, Yi-Chen Yeh, H. Ho, Dorothee Speiser, Maria M Karsten, Michael Lanuti, Sara I Pai, O. Kranenburg, J. Lennerz, Teh-Ying Chou, Matthias Kloor, Kamila Naxerova
          </td>
          <td>2025-02-04</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d479e3aee16692284dd5875a578cce134082060b" target='_blank'>
              Cell signaling communication within papillary craniopharyngioma revealed by an integrated analysis of single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Xiaoyue Zhu, Yanfei Jia, Zicheng Zhao, Xiaoyu Zhang, Yunlong Zhao, Songbai Gui, Xiu-An Yang
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62a0b0314fc694a0d5ac1d3e0908ca758f9f880c" target='_blank'>
              Cancer-associated fibroblasts, tumor and radiotherapy: interactions in the tumor micro-environment
              </a>
            </td>
          <td>
            Kris Raaijmakers, G. Adema, Johan Bussink, M. Ansems
          </td>
          <td>2024-12-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="Background Adenocarcinoma of the esophagogastric junction (AEGJ) is a highly aggressive tumor that frequently metastasizes to the liver. Understanding the cellular and molecular mechanisms that drive this process is essential for developing effective therapies. Methods We employed single-cell RNA sequencing to analyze the tumor heterogeneity and microenvironmental landscape in patients with AEGJ liver metastases. This approach enabled us to characterize the diverse cell populations involved in the liver metastatic process. Results Our analysis revealed a significant involvement of fibroblasts and mural cells in AEGJ liver metastasis. We identified a specific fibroblast type in AEGJ liver metastasis and observed distinct gene expression patterns between adenocarcinoma of the esophagogastric junction and other stomach adenocarcinomas. Our study demonstrated high expression of the SFRP2 gene in pericyte cells during the liver metastasis of AEGJ. The incorporation of GEO, TCGA, and immunofluorescence staining of SFRP2 expression enhanced our study. High expression of SFRP2 in pericytes may influence vascular stability and angiogenesis through the Wnt pathway. Conclusion Our study provides novel insights into the cellular interactions and molecular mechanisms that underlie AEGJ liver metastasis. Targeting the identified subtype of fibroblasts or influencing SFRP2 gene expression in pericytes may offer new therapeutic strategies for combating this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76cf1003bffe0097e28dee22754d6f4e46b48db" target='_blank'>
              Integrated single-cell RNA sequencing reveals the tumor heterogeneity and microenvironment landscape during liver metastasis in adenocarcinoma of esophagogastric junction
              </a>
            </td>
          <td>
            Junrui Xu, Ussama Sadiq, Wangruizhi Zhao, Hengbo Xia, Yiwei Liu, Renquan Zhang, Aman Xu
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="While optimizing cancer cell growth conditions, we discovered that cancer stem cells can generate second membranes outside the plasma membranes forming compartments separated from the extracellular matrix. The encapsulating membranes can extend and generate long cytocapsular tubes, wherein multiple cells can migrate. SILAC proteomics of the second cytocapsular membranes identified 400 membrane proteins, and a small subset of them are highly upregulated in cytocapsular cancers compared to normal tissues. The ATP-dependent calcium pump PMCA2 is one of the highest upregulated factors of the cytocapsular membrane, and antibodies serve as biomarkers for malignant tumors, as checked for 293 subtypes of cancers. Cytocapsular tumors have not been described before, possibly because the CC membranes do not exhibit epitopes targeted by conventional methods, and no efforts have been made to search for new cancer specific organelles. Antibodies against PMCA2 can now be used to map cancer evolution pathways in human bodies by comparisons of more than 12 000 annotated specimens from tissue banks worldwide. The current research reveals that the native malignant cancer cell is enclosed in a cytocapsular membrane. With the PMCA2 cancer biomarker available, the development of human cancers can be studied from cancer tissue banks and clinical cancer biopsies with a previously unknown diversity. The emerging knowledge on cancer-driving biomarkers opens doors for new routes in cancer diagnosis, surgery, therapy, and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4e5b795160ba02ecb2832797cada359373d1df" target='_blank'>
              Discovery of the Cytocapsular Membrane as Hallmark of Malignant Tumors.
              </a>
            </td>
          <td>
            Tingfang Yi, G. Wagner
          </td>
          <td>2024-12-16</td>
          <td>Biochemistry</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f1f8dfa9377169d8a09f717c0d18ed8c6b2867" target='_blank'>
              Prospective validation of ORACLE, a clonal expression biomarker associated with survival of patients with lung adenocarcinoma
              </a>
            </td>
          <td>
            D. Biswas, Yun-Hsin Liu, Javier Herrero, Yin Wu, David A Moore, Takahiro Karasaki, Kristiana Grigoriadis, Wei-Ting Lu, S. Veeriah, C. Naceur-Lombardelli, Neil Magno, S. Ward, A. Frankell, M. Hill, Emma C Colliver, Sophie de Carné Trécesson, Philip East, A. Malhi, Daniel M. Snell, Olga O’Neill, Daniel Leonce, J. Mattsson, A. Lindberg, P. Micke, Judit Moldvay, Z. Megyesfalvi, B. Dome, J. Fillinger, J. Nicod, Julian Downward, Zoltan Szallasi, A. Hackshaw, M. Jamal-Hanjani, N. Kanu, Nicolai J. Birkbak, C. Swanton
          </td>
          <td>2025-01-09</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The vast majority of cancers exhibit Somatic Copy Number Alterations (SCNAs)—gains and losses of variable regions of DNA. SCNAs can shape the phenotype of cancer cells, e.g. by increasing their proliferation rates, removing tumor suppressor genes, or immortalizing cells. While many SCNAs are unique to a patient, certain recurring patterns emerge as a result of shared selectional constraints or common mutational processes. To discover such patterns in a robust way, the size of the dataset is essential, which necessitates combining SCNA profiles from different cohorts, a non-trivial task. To achieve this, we developed CNSistent, a Python package for imputation, filtering, consistent segmentation, feature extraction, and visualization of cancer copy number profiles from heterogeneous datasets. We demonstrate the utility of CNSistent by applying it to the publicly available TCGA, PCAWG, and TRACERx cohorts. We compare different segmentation and aggregation strategies on cancer type and subtype classification tasks using deep convolutional neural networks. We demonstrate an increase in accuracy over training on individual cohorts and efficient transfer learning between cohorts. Using integrated gradients we investigate lung cancer classification results, highlighting SOX2 amplifications as the dominant copy number alteration in lung squamous cell carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f94ed57d2a2d247c1baada02015edbbd8a64f902" target='_blank'>
              CNSistent integration and feature extraction from somatic copy number profiles
              </a>
            </td>
          <td>
            Adam Streck, Roland F. Schwarz
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer treatment is evolving, and the role of senescent macrophages in tumor immune evasion has become a key focus. This study explores how senescent macrophages interact with lung cancer cells, contributing to tumor progression and immune dysfunction. As aging impairs macrophage functions, including phagocytosis and metabolic signaling, it promotes chronic inflammation and cancer development. p16INK4a-positive macrophages are common in aged mice, and their clearance slows tumor growth, suggesting these cells support tumor proliferation and immune evasion. Targeting the senescence-associated secretory phenotype (SASP) and reprogramming senescent macrophages offers potential therapeutic benefits, including reversing immune aging and boosting anti-tumor immunity. However, translating these findings into clinical practice requires further molecular understanding and rigorous clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79ceab5100ddc38b90c7c820490175e64e8754c0" target='_blank'>
              Senescent Macrophages and the Lung Cancer Microenvironment: A New Perspective on Tumor Immune Evasion.
              </a>
            </td>
          <td>
            Lexin Qin, Tingting Liang, Xinyu Zhu, Wentao Hu, Bo Li, Meidan Wei, Jiaxin Zhang, Jianxiang Li, Jin Wang
          </td>
          <td>2024-12-31</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neuroblastoma (NB) remains associated with high mortality and low initial response rate, especially for high-risk patients, thus warranting exploration of molecular markers for precision risk classifiers. Through integrating multiomics profiling, we identified a range of hub genes involved in cell cycle and associated with dismal prognosis and malignant cells. Single-cell transcriptome sequencing revealed that a subset of malignant cells, subcluster 1, characterized by high proliferation and dedifferentiation, was strongly correlated with the hub gene signature and orchestrated an immunosuppressive tumor microenvironment (TME). Furthermore, we constructed a robust malignant subcluster 1 related signature (MSRS), which was an independent prognostic factor and superior to other clinical characteristics and published signatures. Besides, TME differences conferred remarkably distinct therapeutic responses between high and low MSRS groups. Notably, polo-like kinase-1 (PLK1) was one of the most crucial contributors to MSRS and remarkably correlated with malignant subcluster 1, and PLK1 inhibition was effective for NB treatment as demonstrated by in silico analysis and in vitro experiments. Overall, our study constructs a novel molecular model to further guide the clinical classification and individualized treatment of NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33949844ac45f5e41b1ac34112dc1705f6129d2" target='_blank'>
              Multiomics Analysis Reveals Neuroblastoma Molecular Signature Predicting Risk Stratification and Tumor Microenvironment Differences.
              </a>
            </td>
          <td>
            Xing Zhou, Zhaokai Zhou, Xiaohan Qin, Jian Cheng, Yongcheng Fu, Yuanyuan Wang, Jingyue Wang, Pan Qin, Da Zhang
          </td>
          <td>2025-01-06</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="This study aimed to explore the impact of intratumoral microorganisms in conjunction with hormone receptors on the tumor microenvironment and their potential role in predicting patient prognosis. Significant bacterial variations were identified within ER, PR, HER2, and triple-negative breast cancer subtypes. Kaplan-Meier survival analysis was employed to identify bacteria associated with patient survival. Further, a humanized immune system mouse model bearing breast cancer xenografts was used to evaluate the effects of Stenotrophomonas maltophilia (SMA) on tumor growth and CD8+ T cell infiltration. Additional validation experiments included fluorescence in situ hybridization for SMA, CD8+ T cell chemotaxis, and intracellular cytokine detection. Lawsonella clevelandensis-A, Diaphorobacter nitroreducens, and SMA were identified as significant prognostic species. Notably, tumor-infiltrating immune cells, particularly CD8+ T cells, exhibited a positive association with the presence of SMA. Experimental validation with clinically isolated SMA further confirmed its positive correlation with CD8+ T cell activation. In vivo findings demonstrated that SMA inhibited tumor growth and promoted CD8+ T cell infiltration, highlighting the complex interactions between intratumoral microbiota and tumor immunity in breast cancer. These insights contribute to the understanding of microbial influences on the tumor microenvironment and suggest potential pathways for improving patient prognosis through microbiota modulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13fce5a50e9f9799a232183eb87230d236d9adfd" target='_blank'>
              Intratumoral Stenotrophomonas Maltophilia in Breast Cancer: Unraveling the Interplay with Hormone Receptors and Impact on Tumor Immunity
              </a>
            </td>
          <td>
            Qian Zhang, Dujuan Wang, Guangzheng Zhuo, Shilin Wang, Yufen Yuan, Liping Wang, Lili Ji, Yuhang Wan, Guohong Liu, Yunbao Pan
          </td>
          <td>2025-01-06</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="High-Grade Serous Ovarian Cancer (HGSOC) originates from fallopian tube (FT) precursors. However, the molecular changes that occur as precancerous lesions progress to HGSOC are not well understood. To address this, we integrated high-plex imaging and spatial transcriptomics to analyze human tissue samples at different stages of HGSOC development, including p53 signatures, serous tubal intraepithelial carcinomas (STIC), and invasive HGSOC. Our findings reveal immune modulating mechanisms within precursor epithelium, characterized by chromosomal instability, persistent interferon (IFN) signaling, and dysregulated innate and adaptive immunity. FT precursors display elevated expression of MHC-class I, including HLA-E, and IFN-stimulated genes, typically linked to later-stage tumorigenesis. These molecular alterations coincide with progressive shifts in the tumor microenvironment, transitioning from immune surveillance in early STICs to immune suppression in advanced STICs and cancer. These insights identify potential biomarkers and therapeutic targets for HGSOC interception and clarify the molecular transitions from precancer to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b491f761ec399a1258abdf4e675e3bc81f53e5e" target='_blank'>
              Multimodal Spatial Profiling Reveals Immune Suppression and Microenvironment Remodeling in Fallopian Tube Precursors to High-Grade Serous Ovarian Carcinoma.
              </a>
            </td>
          <td>
            Tanjina Kader, Jia-Ren Lin, C. Hug, S. Coy, Yu-An Chen, Ino de Bruijn, Natalie Shih, Euihye Jung, R. Pelletier, Mariana Lopez Leon, Gabriel Mingo, Dalia K Omran, Jong Suk Lee, Clarence Yapp, B. Satravada, Ritika Kundra, Yilin Xu, Sabrina Chan, Juliann B. Tefft, J. Muhlich, Sarah H Kim, Stefan M. Gysler, Judith Agudo, James R. Heath, N. Schultz, Charles Drescher, P. Sorger, Ronny Drapkin, S. Santagata
          </td>
          <td>2024-12-20</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Despite recent advances, improvements to long-term survival in metastatic carcinomas, such as pancreatic or ovarian cancer, remain limited. Current therapies suppress growth-promoting biochemical signals, ablate cells expressing tumor-associated antigens, or promote adaptive immunity to tumor neoantigens. However, these approaches are limited by toxicity to normal cells using the same signaling pathways or expressing the same antigens, or by the low frequency of neoantigens in most carcinomas. Here, we report a fundamentally different strategy for designing safer and more effective anti-cancer therapies through the sensing of cancer-driving biochemical signals and their rewiring to virotherapeutic activation. Specifically, we rationally engineer a RNA vector to self-replicate and cause cytotoxicity in cancer cells exhibiting hyperactive HER2 (ErbB2), but not in normal cells with normal HER2 signaling. Compared to a widely tested virotherapeutic from the same vector family, our hyperactive ErbB2-restricted RNA vector (HERV) exhibits lower toxicity and greater activity against metastatic HER2-positive ovarian cancer in mice, extending survival independently of tumor antigenicity. Most importantly, HERV synergizes with standard-of-care chemotherapy against ovarian cancer metastases in vivo, with 43% of combination-treated subjects surviving for months beyond subjects treated with either therapy alone. Taken together, these results introduce rewiring of cancer-driving signaling pathways to virotherapeutic activation as a strategy for more specific and effective cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519eb18e674bcbdbaf122e51b6002c650afbd522" target='_blank'>
              Rewiring oncogenic signaling to RNA vector replication for the treatment of metastatic cancer
              </a>
            </td>
          <td>
            Xinzhi Zou, Cynthia Zhao, Kevin T. Beier, Chil-Yong Kang, Michael Z. Lin
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Human cancers are heterogeneous. Their genomes evolve from genetically diverse germlines in complex and dynamic environments, including exposure to potential carcinogens. This heterogeneity of humans, our environmental exposures, and subsequent tumours makes it challenging to understand the extent to which cancer evolution is predictable. Addressing this limitation, we re-ran early tumour evolution hundreds of times in diverse, inbred mouse strains, capturing genetic variation comparable to and beyond that found in human populations. The sex, environment, and carcinogenic exposures were all controlled and tumours comprehensively profiled with whole genome and transcriptome sequencing. Within a strain, there was a high degree of consistency in the mutational landscape, a limited range of driver mutations, and all strains converged on the acquisition of a MAPK activating mutation with similar transcriptional disruption of that pathway. Despite these similarities in the phenotypic state of tumours, different strains took markedly divergent paths to reach that state. This included pronounced biases in the precise driver mutations, the strain specific occurrence of whole genome duplication, and differences in subclonal selection that reflected both cancer susceptibility and tumour growth rate. These results show that interactions between the germline genome and the environment are highly deterministic for the trajectory of tumour genome evolution, and even modest genetic divergence can substantially alter selection pressures during cancer development, influencing both cancer risk and the biology of the tumour that develops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caebf07ac0a60cd920fb19088ba70b1c3f57d1ba" target='_blank'>
              Genetic background sets the trajectory of cancer evolution
              </a>
            </td>
          <td>
            Sarah J. Aitken, Frances Connor, Christine Feig, Tim F. Rayner, M. Lukk, Juliet Luft, Stuart Aitken, Claudia Arnedo-Pac, James F. Hayes, Michael D. Nicholson, Vasavi Sundaram, Jan C. Verburg, John Connelly, Craig J. Anderson, Mikaela Behm, Susan Campbell, Maëlle Daunesse, Ailith Ewing, Vera B. Kaiser, Elissavet Kentepozidou, O. Pich, A. Redmond, Javier Santoyo-Lopez, Inés Sentís, Lana Talmane, Paul Flicek, Núria López-Bigas, Colin A. Semple, Martin S. Taylor, D. T. Odom
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="
 Background: Locally-advanced rectal cancer (LARC) is traditionally treated with neoadjuvant chemoradiotherapy before surgery, however, response heterogeneity necessitates the development of novel treatment combinations. Immune checkpoint blockade (ICB) with or without radiotherapy (RT) elicits clinical benefit in various solid tumors, yet ICB’s effectiveness so far is limited to colorectal tumors with microsatellite instability and the role of the tumor microenvironment (TME) is unclear. We present a novel orthotopic immunocompetent mouse model of LARC, where combinations of RT and ICB are tested to elucidate the role of TME in the response to treatment. Methods: Autochthonous murine tumor-derived organoids expressing common driver mutations (Apcfl/fl; KrasG12D/+; p53fl/fl; Tgfbr1fl/fl: ‘AKPT’) were orthotopically injected into syngeneic C57Bl/6 immunocompetent mice. Tumor response to RT-ICB was studied on timepoint and aging cohorts of mice with tumors growing in different TME settings: (a) in the rectal submucosa of wildtype mice, (b) subcutaneously in wildtype mice, and (c) in the rectal submucosa of mice lacking the iCCR locus (CCR1, CCR2, CCR3, and CCR5: ‘iCCRKO ’), which inhibits inflammatory monocyte recruitment into the TME. Radiation was delivered as 15Gy in 3# (Mon-Wed-Fri) using a Small Animal Radiation Research Platform on day 13 following tumor engraftment for the orthotopic models and on day 7 for the subcutaneous. PD-1 inhibition (PD-1i) was administered intraperitoneally (10mg/kg twice weekly) 3 days pre-RT until endpoint. Mice in the timepoint experiments were sampled 5 days following RT, while aging mice were sampled at clinical endpoint or 100 days, whichever came first. Results: RT and PD-1i offered significant survival extension and tumor control in the orthotopic wildtype setting (a). Transcriptomic characterization at 5 days revealed that untreated and irradiated tumors harbored a stroma-rich microenvironment, low in immune cells, which PD-1i partly reversed. This phenotypic ‘switch’ was also evident in an aging cohort, suggesting that RT + PD-1i have long-lasting effects on the stroma. In the subcutaneous setting (b), RT significantly increased tumor control but PD-1 blockade offered little additional advantage. Interestingly, subcutaneous AKPT tumors had significantly more CD8+ cells at baseline than orthotopic ones, highlighting the inherent immune permissiveness of the subcutaneous TME. Finally, tumors growing in the iCCRKO setting (c) were more radio-sensitive, leading to 3 instances of tumor clearance: one in the RT group and two in the RT + PD-1i. Conclusion: Our findings demonstrate the synergistic potential of RT and PD-1i to sensitize an inherently radio-resistant orthotopic mouse model of LARC. Tumor location and inflammatory monocytes emerged as key regulators of the immune response to RT-ICB, which could have important implications for LARC modeling. Importantly, we highlight that TME components can facilitate treatment resistance and give rise to targets that can be therapeutically exploited.
 Citation Format: Lydia Melissourgou-Syka, Michael A. Gillespie, Lily V.S. Hillson, Kathryn Gilroy, Katrina Stevenson, Susanti Susanti, Tamsin R.M. Lannagan, Eoghan Mulholland, Philip D. Dunne, Joanne Edwards, Sean M. O'Cathail, Colin W. Steele, Owen J. Sansom, Campbell S.D. Roxburgh.Investigating the role of the tumor microenvironment in the rectal cancer response to preclinical radio-immunotherapy combinations.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P022">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b958d65c1fd0bd54499a71c321762f474a07901" target='_blank'>
              Abstract P022: Investigating the role of the tumor microenvironment in the rectal cancer response to preclinical radio-immunotherapy combinations
              </a>
            </td>
          <td>
            Lydia Melissourgou-Syka, Michael A Gillespie, L. Hillson, Kathryn Gilroy, Katrina Stevenson, Susanti Susanti, T. Lannagan, Eoghan Mulholland, Philip D. Dunne, J. Edwards, S. O'Cathail, Colin W. Steele, Owen J. Sansom, Campbell S D Roxburgh
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5675284f659a316a50b22f64df578d9b883b676a" target='_blank'>
              Single-cell RNA sequencing highlights the unique tumor microenvironment of small cell neuroendocrine cervical carcinoma
              </a>
            </td>
          <td>
            Tianyou Wang, Li Zhang, Song Mei, Bo Wang, Jiaqi Liu, Weiping Yang, Jiongbo Liao, Chao Wang
          </td>
          <td>2025-01-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Thyroid Hormones (THs) critically impact human cancer. Although endowed with both tumor-promoting and inhibiting effects in different cancer types, excess of THs has been linked to enhanced tumor growth and progression. Breast cancer depends on the interaction between bulk tumor cells and the surrounding microenvironment in which mesenchymal stem cells (MSCs) exert powerful pro-tumorigenic activities.


METHODS
Primary human MSCs from healthy female donors were co-cultured with DIO2 knock out (D2KO) and wild type (WT) MCF7 breast cancer cells to assess cell growth, migration, invasion and the expression of known epithelial-mesenchymal transition (EMT)- and inflammation-related markers. Furthermore, a surgery-free intraductal delivery model, i.e., the Mouse-INtraDuctal (MIND) injection method, was used as a tool for in vivo characterization of breast tumor formation and progression.


RESULTS
In this study, we uncovered a novel role of THs in regulating the tumor-stroma crosstalk. MCF7 cells enhanced the intracellular activation of THs through the TH-activating enzyme, D2, fostering their EMT properties and the dialogue with MSCs. D2 inactivation reduced the invasiveness of MCF7 cells and their responsiveness to the pro-tumorigenic induction via MSCs, both in vivo and in vitro.


CONCLUSIONS
Thus, we argue that intracellular activation of THs via D2 is a critical requirement for invasive and metastatic conversion of breast cancer cells, advising the blocking of D2 as a potential therapeutic tool for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b5bee8d823debda4075760c0534546bb56c40c" target='_blank'>
              Thyroid Hormone Activation Regulates the Crosstalk between Breast Cancer and Mesenchymal Stem Cells.
              </a>
            </td>
          <td>
            A. Nappi, V. D'Esposito, C. Miro, A. Parascandolo, A. G. Cicatiello, S. Sagliocchi, L. Acampora, Sepehr Torabinejad, F. Restolfer, M. Raia, Melania Murolo, E. Di Cicco, Pietro Formisano, M. Dentice
          </td>
          <td>2025-01-20</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a00464522656fb99b3714169ed11d185f90a82" target='_blank'>
              A hybrid epithelial-mesenchymal transition program enables basal epithelial cells to bypass stress-induced stasis and contributes to a metaplastic breast cancer progenitor state
              </a>
            </td>
          <td>
            Joseph A. Caruso, Chira Chen-Tanyolac, T. Tlsty
          </td>
          <td>2024-12-18</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Osteosarcoma is the most common primary malignant bone tumor, predominantly affecting children and adolescents. Despite advances in conventional therapies like chemotherapy and surgical resection, the survival rate has remained stagnant due to therapeutic resistance and high rates of metastasis. The tumor microenvironment (TME), a complex network of cellular and non-cellular components, plays a pivotal role in osteosarcoma progression, metastasis, and treatment resistance. Understanding the dynamics within the TME is crucial for developing novel therapeutic strategies that can overcome these challenges. This review explores the key elements of the osteosarcoma TME, including immune cells, endothelial cells, cancer-associated fibroblasts (CAFs), and extracellular matrix (ECM). It examines the roles of cytokines, growth factors, and exosomes secreted by osteosarcoma cells in modifying the TME to foster tumor growth, evade immune surveillance, and promote angiogenesis. Furthermore, the review critically assesses current therapeutic approaches that target TME components, with a focus on disrupting the interactions between the tumor and its microenvironment. This analysis includes a review of clinical trials and preclinical studies evaluating TME-targeting therapies. The findings highlight that the TME actively contributes to osteosarcoma progression by promoting immunosuppression, angiogenesis, and metastasis, while also enhancing resistance to standard treatments such as chemotherapy and immune checkpoint inhibitors. Various therapeutic strategies targeting the TME, such as inhibiting angiogenesis, modulating immune responses, and disrupting CAF and ECM interactions, have shown promise in preclinical models. However, clinical outcomes remain variable, underscoring the complexity of the TME and the need for more comprehensive approaches. Targeting the TME represents a promising pathway to overcome therapeutic resistance in osteosarcoma. While significant progress has been made in understanding the role of the TME in tumor development and resistance mechanisms, further research is required to optimize TME-targeting therapies. A deeper comprehension of the intricate interactions between osteosarcoma cells and their microenvironment may lead to more effective, personalized treatments, improving clinical outcomes for patients with osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de846221faa142b04c72a554ab42e7526bfe9854" target='_blank'>
              Targeting the Tumor Microenvironment in Osteosarcoma: A Pathway to Overcome Therapeutic Resistance
              </a>
            </td>
          <td>
            Kaniga Pandi, B. Cheriyan, Rithika Sooriyaprakash, Pradhosh Sakthivel, Rajeshkumar Thirupathi
          </td>
          <td>2024-12-30</td>
          <td>Biomedical and Pharmacology Journal</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Oncogenes are typically overexpressed in tumor tissues and often linked to poor prognosis. However, recent advancements in bioinformatics have revealed that many highly expressed genes in tumors are associated with better patient outcomes. These genes, which act as tumor suppressors, are referred to as “paradoxical genes.” Analyzing The Cancer Genome Atlas (TCGA) confirmed the widespread presence of paradoxical genes, and KEGG analysis revealed their role in regulating tumor metabolism. Mechanistically, discrepancies between gene and protein expression-affected by pre- and post-transcriptional modifications-may drive this phenomenon. Mechanisms like upstream open reading frames and alternative splicing contribute to these inconsistencies. Many paradoxical genes modulate the tumor immune microenvironment, exerting tumor-suppressive effects. Further analysis shows that the stage- and tumor-specific expression of these genes, along with their environmental sensitivity, influence their dual roles in various signaling pathways. These findings highlight the importance of paradoxical genes in resisting tumor progression and maintaining cellular homeostasis, offering new avenues for targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab8741264f274ad63584e4f394278cc1295e34" target='_blank'>
              Discovery of paradoxical genes: reevaluating the prognostic impact of overexpressed genes in cancer
              </a>
            </td>
          <td>
            Dequan Liu, Lei Liu, Xiangyu Che, Guangzhen Wu
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Pediatric osteosarcoma, the most prevalent primary malignant bone tumor in children, is marked by aggressive progression and a generally poor prognosis. Despite advances in treatment, including multi-agent chemotherapy, survival rates remain suboptimal, with metastasis, particularly to the lungs, contributing significantly to mortality. The tumor microenvironment plays a crucial role in osteosarcoma progression, with immune cells such as tumor-associated macrophages and T lymphocytes significantly influencing tumor behavior. The immunosuppressive environment, dominated by M2 macrophages, contributes to immune evasion and poor therapeutic outcomes, though recent findings suggest the potential for reprogramming these cells to enhance immune responses. This review provides a comprehensive overview of the immune landscape in pediatric osteosarcoma, with a focus on the role of immune cells and their interactions within the tumor microenvironment (TME). It examines the impact of immune checkpoints, genetic mutations, and inflammatory pathways on osteosarcoma progression, highlighting their contribution to tumor immune evasion and disease advancement. Additionally, emerging immunotherapeutic strategies, such as immune checkpoint inhibitors, macrophage reprogramming, and antibody-based therapies, are summarized in detail, showcasing their potential to improve therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6762ab58ee075be8502ca8dec282f12b58d84c17" target='_blank'>
              Progress in immune microenvironment, immunotherapy and prognostic biomarkers in pediatric osteosarcoma
              </a>
            </td>
          <td>
            Lin Zhang, Hao-rong Jiang, Haichao Ma
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common primary liver cancer, with microvascular invasion (MVI) identified as a major predictor of early recurrence. However, the intratumor cellular heterogeneity of MVI, the identification of pertinent biomarkers, and the role of intercellular signalling interactions in MVI progression are unclear. This study aims to explore these aspects using single-cell transcriptomic analysis.The present study utilized single-cell transcriptomic data from public databases to conduct an in-depth transcriptome analysis of tumour tissues and adjacent nontumor tissues from five patients with hepatocellular carcinoma, with a particular focus on samples from three patients exhibiting microvascular invasion. Bioinformatics tools were employed to analyze gene expression patterns and signalling pathways.The findings indicated that MVI-positive malignant cells activate multiple signalling pathways to facilitate invasion and metastasis. Specific malignant cell subtypes strongly associated with MVI were identified, exhibiting distinctive gene expression patterns related to proliferation, invasion, and metabolic reprogramming of tumour cells. Further analysis revealed that the laminin and VEGF signalling pathways are crucial for remodelling the tumour microenvironment and angiogenesis associated with MVI. The MARCKSL1 gene was predominantly expressed in MVI-positive malignant cells and may contribute to MVI progression by interacting with the PTN signalling network. Additionally, MARCKSL1 is linked to tumour resistance to multiple anticancer drugs.This study sheds light on the molecular characteristics and functional heterogeneity of MVI-associated malignant cell subpopulations. The single-cell transcriptome and bioinformatics analyses provided insights into the mechanisms driving MVI, potentially aiding the development of targeted diagnostic and therapeutic strategies. Future research should further validate the role of MARCKSL1 in MVI progression and explore its potential clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e75f99f0ab352845523f0f1e55263a27b76fefc5" target='_blank'>
              Single-cell sequencing reveals cell heterogeneity and aberrantly activated pathways associated with microvascular invasion in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jianwei Cui, Fanyi Zeng, Ming Tang, Shiwu Yin
          </td>
          <td>2025-01-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd21edea6ef2a85bfcda6c2395a58850a218d8d2" target='_blank'>
              Patient-derived response estimates from zero-passage organoids of luminal breast cancer
              </a>
            </td>
          <td>
            Róża K. Przanowska, Najwa Labban, Piotr Przanowski, Russell B Hawes, Kristen A Atkins, S. Showalter, Kevin A. Janes
          </td>
          <td>2024-12-31</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Introduction Ovarian cancer (OC) is the sixth most common malignancy in women and the poor 5-year survival emphasises the need for novel therapies. NK cells play an important role in the control of malignant disease but the nature of tumour-infiltrating and peripheral NK cells in OC remains unclear. Methods Using flow cytometric analysis, we studied the phenotype and function of NK cells in blood, primary tumour and metastatic tissue in 80 women with OC. The cell type contexture of metastatic OC tissue was explored utilising scRNAseq analysis, with a focus on portraying an immunogenic tumour microenvironment and determining the characteristics of a dysfunctional NK cell population. Results The proportion of peripheral NK cells was markedly elevated with a highly activated profile and increased cytotoxicity. In contrast, NK cell numbers in primary tumour and metastasis were substantially reduced, with downregulation of activatory receptors together with elevated PD-1 expression. scRNA-Seq identified 5 NK cell subpopulations along with increased exhausted and immature NK cells within tumour tissue compared to normal tissue. These features were attenuated following chemotherapy where higher levels of activated and cytotoxic NK cells associated with improved disease-free survival. Correlation of NK cell phenotype with clinical outcomes revealed high levels of DNAM-1 expression on tissue-localised and peripheral NK cells to be associated with reduced survival. Expression of PVR, the DNAM-1 ligand, was significantly increased on tumours and DNAM-1 mediated NK cell lysis of primary tumour tissue was observed in vitro. Discussion These findings reveal profound modulation of the tumour tissue and systemic profile of NK cells which likely contributes to the high rates of local progression and metastasis seen with OC. Immunotherapeutic approaches that overcome local immune suppression and enhance DNAM-1-targeted lysis of OC offer the potential to improve disease control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec3263f862b7dea4d3ddaf31f9b8ccd7a6ea974" target='_blank'>
              The emergence of DNAM-1 as the facilitator of NK cell-mediated killing in ovarian cancer
              </a>
            </td>
          <td>
            R. Pounds, W. Croft, Hayden Pearce, Tasnia Hossain, Kavita Singh, J. Bálega, D. Jeevan, S. Sundar, S. Kehoe, J. Yap, Paul Moss, J. Zuo
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="When confronted with an undesired cell population, such as bacterial infections or tumors, we seek the most effective treatment, designed to eliminate the population as rapidly as possible. A common practice is to monitor the cells short-term response to the treatment, and from that, extrapolate the eventual treatment outcome, i.e. will it eradicate the cells, and if yes at what timescales. Underlying this approach is the assumption that the cells exhibit a homogeneous response to the treatment, and hence the early response patterns can be naturally extended to later times. Recent experiments on cancer cell populations, however, indicate a significant level of cellular heterogeneity, undermining this classic assessment protocol of treatment efficacy. We, therefore, develop here a stochastic framework, to analytically predict the temporal dynamics of a heterogeneous cell population. Quite often, we find, the average cellular parameters, governing the short-term response, fail to predict the actual treatment outcome. In contrast, our analysis, which also incorporates the populations variability, helps identify the relevant statistical parameters, which in turn, enable us to predict the full trajectory of the cell population, and specifically - the likelihood and typical timescales for remission.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7ef92fd76daed3e67a88fcf502b6cc330b90e29" target='_blank'>
              Efficient treatment of heterogeneous malignant cell populations
              </a>
            </td>
          <td>
            Uzi Harush, R. Straussman, B. Barzel
          </td>
          <td>2025-01-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Radiation is used to treat pancreatic ductal adenocarcinoma (PDAC) in the locally-advanced setting. Stereotactic body radiation therapy (SBRT), in particular, has shown improved outcomes against conventional RT in a number of clinical trials. One key cell type involved in the response to RT are neutrophils. These innate immune cells are the first responders to tissue damage and infiltrate irradiated tumors in high numbers to rectify injury. Neutrophils typically possess an immunosuppressive, wound healing phenotype in this scenario, which allows the tumor to recover and actively suppress immunological efforts to eradicate the disease. Here, we aimed to elucidate the role of neutrophils in a murine model of PDAC treated with SBRT. Mice harboring PDAC tumors were treated with targeted SBRT and neutrophils were determined to be significantly increased in the tissue when compared to unirradiated controls. Additionally, phenotypic analysis determined that these cells were largely immunosuppressive and depletion studies confirmed they played a key role in acquired radioresistance in our model. In order to establish whether these infiltrating cells could be re-wired to contribute to anti-tumor immunity, we utilized a novel combination therapy consisting of SBRT and microspheres containing recombinant IL-12 to attempt to repolarize these cells. Transcriptomic analysis confirmed intratumoral neutrophils underwent considerable changes indicative of an immunostimulatory, anti-tumor phenotype following treatment. Moreover, depleting these cells resulted in a loss of treatment efficacy, suggesting that neutrophil re-wiring was vital for the therapeutic outcome. This study highlights neutrophils as key players in SBRT and IL-12 treatment and confirms they can act as a double-edged sword depending on the treatment employed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e864765da0f0855b6d1fbbd6e6ee8a77f250f8a3" target='_blank'>
              Improving SBRT by re-wiring immunosuppressive neutrophils in murine pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Joseph D. Murphy, G. Hannon, Bradley N. Mills, Angela L. Hughson, Taylor P. Uccello, Nicholas W. Gavras, Jian Ye, B. Belt, Maggie L. Lesch, Sarah L. Eckl, Edith M. Lord, Minsoo Kim, Haoming Qiu, David C Linehan, Scott A. Gerber
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Triple negative breast cancers often contain higher numbers of tumour-infiltrating lymphocytes compared with other breast cancer subtypes, with their number correlating with prolonged survival. Since little is known about tumour-infiltrating lymphocyte trafficking in triple negative breast cancers, we investigated the relationship between tumour-infiltrating lymphocytes and the vascular compartment to better understand the immune tumour microenvironment in this aggressive cancer type. We aimed to identify mechanisms and signaling pathways responsible for immune cell trafficking in triple negative breast cancers, specifically of basal type, that could potentially be manipulated to change such tumours from immune “cold” to “hot” thereby increasing the likelihood of successful immunotherapy in this challenging patient population. We characterised the spatial immune environment in 10 basal breast cancers showing a range of tumour-infiltrating lymphocytes using multiplex fluorescent immunohistochemistry and quantitative digital analysis of CD3+ T cells. We examined their relationship to blood vessels and their activation status as defined by VCAM-1, ICAM-1 and PD-L1. Confirmation of the relationship between tumour-infiltrating lymphocytes and endothelial activation was performed through in silico analysis on TCGA BRCA RNA-seq data (N = 808). Significantly higher CD3+ T cell densities were observed in the stromal compartment compared with the neoplastic cell compartment (P = 0.003). ICAM-1 activated blood vessels were spatially associated with higher CD3+ T cell densities only within 30 microns of blood vessels compared with more distal activated and non-activated blood vessels (P = 0.041). In silico analysis confirmed higher numbers of tumour-infiltrating lymphocytes in basal breast cancers and that higher numbers were significantly associated with endothelial cell activation molecules, co-clustering with upregulated ICAM-1 and VCAM-1 amongst others. PD-L1 was also identified in a subset of blood vessels, suggesting an additional immune regulatory mechanism in endothelial cells. Regulating the activation status of tumour-associated vascular endothelial cells may improve T cell trafficking into basal breast tumours and enhance immunotherapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dd5404cd864380509879a211020591331396cbc" target='_blank'>
              Preliminary characterisation of the spatial immune and vascular environment in triple negative basal breast carcinomas using multiplex fluorescent immunohistochemistry
              </a>
            </td>
          <td>
            E. Takano, Metta K Jana, Luis E Lara Gonzalez, J. Pang, R. Salgado, S. Loi, Stephen B Fox
          </td>
          <td>2025-01-10</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The microvasculature within the tumor microenvironment is crucial for the invasion and dissemination of cancer cells throughout the body. Given its importance and dynamic behavior, several microfluidic models have been developed to study microvascular infiltration and its interaction with cancer cells. However, most of these models primarily focus on blood vessels and use microfluidic channels coated with endothelial cells, which fail to replicate near‐physiological conditions. To address this limitation, the MicroVasculoid‐chip is introduced, a novel human microcirculation‐on‐a‐chip model that features self‐organized 3D blood and lymphatic microvasculature alongside tumor spheroids. This innovative platform enables the exploration of interactions between multi‐cellular tumors and both microvascular networks. Using lung cancer as a case study, how tumor‐released mediators influence vessel morphology is investigated in relation to tumor invasion capacity, identifying molecular factors potentially associated with microvascular remodeling. Overall, the MicroVasculoid‐chip provides a robust tool for investigating and modeling critical events of cancer neo‐vascularization, for deciphering fundamental mechanisms of cancer cell invasion into the microvasculature, and for future drug screening applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8b262e54a669b6b604d46eaa5b0ec11118b17f1" target='_blank'>
              MicroVasculoid‐Chip: A 3D Self‐Assembled Human Microcirculation‐on‐a‐Chip Model Reveals Enhanced Lymphangiogenic Lung Cancer‐Induced Vessel Remodeling and Invasion
              </a>
            </td>
          <td>
            C. M. Abreu, Ana C. Lima, Nuno M. Neves, Subhas C. Kundu, Rui L. Reis, D. Caballero
          </td>
          <td>2025-01-15</td>
          <td>Advanced Materials Technologies</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024cdb898f44026b61ab427711870c26e06ca884" target='_blank'>
              Identification of Cell Fate Determining Transcription Factors for Generating Brain Endothelial Cells.
              </a>
            </td>
          <td>
            Roya Ramezankhani, J. De Smedt, B. Toprakhisar, Bernard K. van der Veer, Tine Tricot, Gert Vanmarcke, Bradley Balaton, Leo A. van Grunsven, Massoud Vosough, Y. C. Chai, C. Verfaillie
          </td>
          <td>2025-01-24</td>
          <td>Stem cell reviews and reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68f83d17426887d25cc6744915e4b78ee129628f" target='_blank'>
              Unsupervised self-organising map classification of Raman spectra from prostate cell lines uncovers substratified prostate cancer disease states
              </a>
            </td>
          <td>
            Daniel West, Susan Stepney, Y. Hancock
          </td>
          <td>2025-01-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Productive infections of oncogenic human papillomaviruses (HPVs) are closely linked to the differentiation of host epithelial cells, a process that the virus manipulates to create conditions favorable to produce virion progeny. This viral interference involves altering the expression of numerous host genes. Among these, proprotein convertases (PCs) have emerged as potential oncogenes due to their central role in cellular functions. Using RT-qPCR, aberrant PC gene expression was detected across the progression from early HPV infection stages to cancer. These findings demonstrated a progressive disruption of normal PC expression profiles, with FURIN consistently downregulated and other PCs upregulated, transitioning from the episomal stage to neoplastic and carcinoma phenotypes. This pattern of dysregulation was distinct from the broader trends observed in a variety of cancer types through bioinformatic analysis of publicly available transcriptomic datasets, where FURIN expression was predominantly upregulated compared to other PCs. Further bioinformatic investigations revealed a correlation between PC gene expression and cancer phenotype diversity, suggesting a potential link between the loss of normal PC gene expression patterns and the progression of HPV infections toward malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f01d2dc341bbc570cc91f7701d59ca7b10ccc365" target='_blank'>
              Dysregulation of FURIN and Other Proprotein Convertase Genes in the Progression from HPV Infection to Cancer
              </a>
            </td>
          <td>
            Gonzalo Izaguirre, Natalia Zirou, Craig Meyers
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="INTRODUCTION
Spatial biology is an emerging interdisciplinary field facilitating biological discoveries through the use of spatial omics technologies. Recent advancements in spatial transcriptomics, spatial genomics (e.g. genetic mutations and epigenetic marks), multiplexed immunofluorescence, and spatial metabolomics/lipidomics has enabled high-resolution spatial profiling of gene expression, genetic variation, protein expression, and metabolites/lipids profiles in tissue. These developments contribute to a deeper understanding of the spatial organization within tissue microenvironments at the molecular level.


AREAS COVERED
This report provides an overview of the untargeted, bottom-up mass spectrometry (MS)-based spatial proteomics workflow. It highlights recent progress in tissue dissection, sample processing, bioinformatics, and liquid chromatography (LC)-MS technologies that are advancing spatial proteomics toward cellular resolution.


EXPERT OPINION
The field of untargeted MS-based spatial proteomics is rapidly evolving and holds great promise. To fully realize the potential of spatial proteomics, it is critical to advance data analysis and develop automated and intelligent tissue dissection at the cellular or subcellular level, along with high-throughput LC-MS analyses of thousands of samples. Achieving these goals will necessitate significant advancements in tissue dissection technologies, LC-MS instrumentation, and computational tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/750f7b08fc1af849136e70bc860f56ef668a0aa0" target='_blank'>
              Spatial proteomics toward cellular resolution.
              </a>
            </td>
          <td>
            Yumi Kwon, James M Fulcher, L. Paša-Tolić, Wei-Jun Qian
          </td>
          <td>2024-12-22</td>
          <td>Expert review of proteomics</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e47a9918829003a93f398ed9e563361008b2b513" target='_blank'>
              Resolving tissue complexity by multimodal spatial omics modeling with MISO.
              </a>
            </td>
          <td>
            Kyle Coleman, Amelia Schroeder, Melanie Loth, Daiwei Zhang, Jeong Hwan Park, Ji-Youn Sung, Niklas Blank, Alexis J Cowan, Xuyu Qian, Jianfeng Chen, Jiahui Jiang, Hanying Yan, Laith Z. Samarah, Jean R. Clemenceau, Inyeop Jang, Minji Kim, Isabel Barnfather, Joshua D. Rabinowitz, Yanxiang Deng, Edward B. Lee, Alexander Lazar, Jianjun Gao, E. E. Furth, Tae Hyun Hwang, Linghua Wang, Cristoph Thaiss, Jian Hu, Mingyao Li
          </td>
          <td>2025-01-15</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Since its first conceptualization over a century ago, the mesenchymal phenotype has traditionally been viewed as either a transient phase between successive epithelial stages or as a feature of cell types primarily devoted to structural support. However, recent findings in cancer research challenge this limited view, demonstrating that mesenchymal traits and hybrid mesenchymal/epithelial states can mark cancer cells with stem cell properties. By analyzing publicly available single-cell transcriptome datasets from early embryonic stages and adult tissues, this study aims to extend this concept beyond pathological contexts, suggesting that a partial or fully mesenchymal phenotype may represent the morphological expression of undifferentiated and multipotent states in both the developing embryo and adult organs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8b86efcb43fa26b36795e33eead52234f743d4e" target='_blank'>
              Mesenchymal Traits as an Intrinsic Feature of Undifferentiated Cells
              </a>
            </td>
          <td>
            M. Galiè
          </td>
          <td>2024-12-24</td>
          <td>Journal of Developmental Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/350ae86f61f0bcf11ab3944cf0a0043eb71b585b" target='_blank'>
              Radiogenomics and machine learning predict oncogenic signaling pathways in glioblastoma
              </a>
            </td>
          <td>
            Abdul Basit Ahanger, Syed Wajid Aalam, Tariq Ahmad Masoodi, Asma Shah, Meraj Alam Khan, Ajaz A. Bhat, Assif Assad, M. Macha, M. Bhat
          </td>
          <td>2025-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The cell fate transition is a fundamental characteristic of living organisms. By introducing external perturbations, it is possible to artificially intervene in cell fate and trigger cell reprogramming. Revealing the general principle underlying the induced phenotypic reshaping of cell populations remains a central focus in the field of cell biology. In this study, we investigate the energetic and dynamic features of induced cell phenotypic transition from differentiated somatic state to pluripotent state by constructing a Boolean genetic network model. The simulation and experimental results highlight the critical role of genetic frustration in initiating cell fate transitions, although the two ending phenotypic states are typically featured by minimal frustration. In addition, the altered gene expression profiles exhibit a scale-free distribution, suggesting that there exist a small number of critical genes responsible for the cell fate transition. This study provides important insights into the dynamic principles governing effective cell reprogramming caused by artificial or exogenous interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112e17321e8898e722005a2009e4fe82616f4dc5" target='_blank'>
              Role of genetic frustrations in cell reprogramming
              </a>
            </td>
          <td>
            Yuxiang Yao, Jieying Zhu, Wenfei Li, Duanqing Pei
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff4ae0eaa9128342a0dedd747e19622d92c7f71" target='_blank'>
              FDFT1 maintains glioblastoma stem cells through activation of the Akt pathway
              </a>
            </td>
          <td>
            Hui Mo, Jiajia Shao, Zhun Li, Peiting Zeng, Xinke Yin, Yongsheng Huang, Peng Wang, Jianwei Liao
          </td>
          <td>2024-12-20</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Structural maintenance of chromosome-1A (SMC1A) is overexpressed in various malignancies including triple-negative breast cancer (TNBC). As a core component of the cohesin complex, SMC1A was initially recognized for its involvement in chromosomal cohesion and DNA-repair pathways. However, recent studies have unveiled its pivotal role in epithelial–mesenchymal transition (EMT), metastasis, and chemo- and radio-resistance in cancer cells. In hepatocellular carcinoma, aberrant phosphorylation of SMC1A has been associated with enhanced cell proliferation and migration. Despite these insights, the precise role of SMC1A phosphorylation in breast cancer remains largely unexplored. This study represents the first investigation to test the phosphorylation status and subcellular localization of SMC1A (p-SMC1A) in breast cancer and normal breast tissues. Immunohistochemical (IHC) staining was conducted using previously validated phospho-SMC1A antibodies on a histological section and tissue microarray (TMA) comprising samples from primary, invasive, and metastatic breast cancer and normal breast tissues. Our results revealed that p-SMC1A staining intensity was lower in normal breast tissues compared to invasive or metastatic breast cancer tissues (p < 0.001). Approximately 40% of breast cancer tissue exhibited cytoplasmic/membranous localization of p-SMC1A, whereas nuclear expression was observed in normal breast tissues. Moreover, elevated phosphorylation levels were significantly associated with higher tumor grade and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ac66fe9226183dafb37845efa13c23cb0dc9fe" target='_blank'>
              Association of Structural Maintenance of Chromosome-1A Phosphorylation with Progression of Breast Cancer
              </a>
            </td>
          <td>
            Sushma Yadav, C. Kowolik, Daniel Schmolze, Yuan Yuan, Min Lin, Arthur D. Riggs, David A. Horne
          </td>
          <td>2025-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, M. Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Odorant receptors (ORs), which constitute approximately 50% of all human G protein-coupled receptors, are increasingly recognized for their diverse roles beyond odor perception, including functions in various pathological conditions like brain diseases and cancers. However, the roles of ORs in glioblastoma (GBM), the most aggressive primary brain tumor with a median survival of only 15 months, remain largely unexplored. Here, we performed an integrated transcriptomic analysis combining The Cancer Genome Atlas RNA-seq and single-cell RNA sequencing data from GBM patients to uncover cell-type-specific roles of ORs within the tumor and its microenvironment. Our findings reveal that ORs display distinct expression patterns, with OR51E1 enriched in pericytes linked to vascular remodeling and angiogenesis, OR2B11 associated with tumor-associated macrophages supporting immunosuppressive phenotypes, and OR2L13 correlated with synaptic activity in recurrent tumors, potentially mediating treatment-induced neuronal adaptations. These results highlight ORs as potential therapeutic targets, offering new insights into their regulatory roles in GBM progression, immune modulation, and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0d78b05c85d74f15c35ba8438bef22107b35ac7" target='_blank'>
              Comprehensive Transcriptomic Analysis Reveals Cell-Type-Specific Roles of Human Odorant Receptors in Glioblastoma and the Tumor Microenvironment
              </a>
            </td>
          <td>
            Hee Jin Cho, Dong Jun Yeo, HeeWoong Yang, JaeHyung Koo
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Human colorectal cancer (CRC) most often arises as a result of spontaneous genetic driver mutations such as APC, BRAF, KRAS, SMAD4, and TP53. These are among the most common drivers of CRC and are thought to contribute directly to cancer initiation and growth. CRC tumors harboring BRAFV600E mutation exhibit a CpG Island Methylator Phenotype (CIMP). Other drivers are also more weakly associated with distinct DNA methylation profiles. It is not clear whether these DNA methylation profiles are a result of differences in the cells-of-origin for these cancers, with pre-existing methylation patterns or whether there is selective pressure for various DNA methylation alterations that cooperate with each driver. To address this question, we have modeled this in mice by inducing colon tumors in a controlled cell population at the same age, with either BRAFF-V600E, KRASLSLG12D, Smad4, or Trp53 drivers in combination with Apc heterozygosity. We have profiled DNA methylation in tumors arising in these mice to determine the interaction between different driver mutations and the tumor methylome. Using a multiple linear regression model, we have identified DNA methylation alterations that are shared across all tumor samples, as well as changes that are specific to each of the different genetic drivers. The DNA methylation profiles accurately recapitulate well-established findings, including high levels of hypermethylation in KRASLSLG12D and BRAF F-V600E tumors reminiscent of CpG Island Methylator Phenotype (CIMP), universal tumor hypermethylation at binding sites for transcription factors involved in intestinal epithelial differentiation, and hypomethylation at AP1 complex component binding sites. Driver-specific histology findings broadly align with those observed in human cancers, such as sessile serrated morphology in BRAFF-V600E mutated tumors. The tumor models with different driver mutations also show variation in hypermethylation of polycomb repressive complex target sites and replicative history. We are using the differences in the DNA methylation profiles between the driver tumor models in conjunction with RNA-seq analysis and conservation of the alterations in human CRC to identify epigenetic events cooperative with the drivers and functionally relevant to tumorigenesis.
 Citation Format: Manpreet Kalkat, Toshinori Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich. Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32f4b048765e136fdf9884436d3047629c3c3d9" target='_blank'>
              Abstract B025: Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer
              </a>
            </td>
          <td>
            Manpreet Kalkat, T. Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Colitis-associated colorectal cancer (CAC) is a severe complication of inflammatory bowel disease, particularly ulcerative colitis. Despite its clinical significance, the spatiotemporal dynamics of neoplastic progression in CAC remain poorly understood. This study aimed to elucidate the evolutionary trajectory of CAC using spatial transcriptomics.



 We employed spatial transcriptomics to analyze four tissue samples from three patients with ulcerative colitis at various stages of neoplastic progression. Whole-slide spatial transcriptomics RNA sequencing was performed, enabling simultaneous analysis of gene expression and tissue architecture. Integrated spatial transcriptomic data revealed recurring epithelial programs along the progression of copy number alteration.



 Our analysis using copy number variation (CNV) score and Hematoxylin and Eosin (H&E) staining image revealed progressive changes in gene expression patterns as tissue transitioned from non-tumor to pre-tumor and finally to tumor states. This progression was quantified using beta coefficient values, allowing us to track gradual alterations in gene expression across the disease continuum. We identified a core set of 48 genes that exhibited consistent changes across all four analyzed samples. Among these, 40 genes showed a progressive decrease in expression, while eight genes, including CLDN1, E2F1, and RECQL4, demonstrated a gradual increase in expression as the tissue advanced towards a tumorous state. These upregulated genes pointed to potential oncogenic drivers or compensatory mechanisms activated during tumor development. Pathway analysis of the 48 genes revealed enrichment in key cellular processes, including p53 signaling, transcriptional regulation by E2F factors, and role of PKR in interferon induction and antiviral response. Upstream Regulator analysis revealed YAP1 and TGFB1 as the most significant regulators, highlighting the importance of transcriptional control and growth factor signaling in CAC development.



 This study provides unprecedented insights into the spatiotemporal dynamics of CAC progression. By leveraging spatial transcriptomics, we have mapped the evolutionary trajectory of CAC at high resolution, revealing key molecular events and spatial features associated with disease progression. These findings not only enhance our understanding of CAC pathogenesis but also highlight potential targets for early intervention and personalized treatment strategies.


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5406d2b3a2271be7d5c35c08e3c0a5e93dd4e792" target='_blank'>
              DOP128 Spatial transcriptomic analysis reveals evolutionary dynamics in colitis-associated colorectal cancer
              </a>
            </td>
          <td>
            Y. Kim, Y. Lee, M. Lee, D. K. Yoon, J. Baek, S. Hwang, J. L. Lee, S. Park, D. H. Yang, J. Kim, S. K. Yang, B. Han, K Song, Y. Yoon, B. Ye, H. S. Lee
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22db0fa38324aae2b7eacb205354e39ad71173ca" target='_blank'>
              Organoids as a model system for researching human neuroendocrine tumor of the breast
              </a>
            </td>
          <td>
            Dongyi Zhao, Xue Bai, Shida Zhu, Zuowei Zhao, Xuelu Li
          </td>
          <td>2024-12-27</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Pancreatic cancer is a highly lethal malignancy with few effective treatment options. Connexin 31 (Cx31) is a membrane protein capable of forming hexameric channels to facilitate the exchange of metabolites and signaling molecules. Yet, the contribution of Cx31 to the onset and progression of pancreatic cancer remains to be understood. We analyzed Cx31 expression in pancreatic cancer tissues and cell lines using public databases and experimental models. The correlation between Cx31 expression and clinical outcomes was evaluated. The effects of Cx31 on pancreatic cancer cell proliferation, stemness, migration, chemoresistance, and immune infiltration were investigated. Transcriptome analysis and bioinformatics tools were employed to explore the underlying mechanisms. Cx31 was found to be upregulated in pancreatic cancer tissues compared to normal tissues, and its high expression correlated with shorter overall survival and higher mortality risk. Cx31 promoted acinar-to-ductal metaplasia (ADM), stemness, proliferation, migration, metastasis, and chemoresistance in pancreatic cancer cells. Bioinformatics analysis suggested a positive correlation between Cx31 and stemness-related genes. Cx31 knockdown altered the expression of genes involved in stemness and chemoresistance pathways, such as Wnt and Notch. Additionally, Cx31 was identified as a direct target of the transcription factor FOXM1, which upregulated its expression. Cx31 plays a multifaceted role in pancreatic cancer, influencing processes from initiation to metastasis and chemoresistance. It may serve as a potential therapeutic target to combat the aggressive nature of pancreatic cancer. The FOXM1-Cx31 axis could be a promising target for overcoming treatment resistance in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92baaa0b6ff72194944a8710fcf3d6648990069b" target='_blank'>
              FOXM1-Cx31 Axis Drives Pancreatic Cancer Stem Cell-Like Properties and Chemoresistance.
              </a>
            </td>
          <td>
            Yang Chen, Qihui Sun, Qi Zou, Xiaoqi Zhu, Tingting Wen, Xiaojia Li, Shu Li, Jie He, Fang Wei, Keping Xie
          </td>
          <td>2025-01-06</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9043774f4d71a3f413b248306070648bf9c170e4" target='_blank'>
              Multi-omics approaches reveal that diffuse midline gliomas present altered DNA replication and are susceptible to replication stress therapy
              </a>
            </td>
          <td>
            Anastasia E. Hains, Kashish Chetal, Tsunetoshi Nakatani, Joana G. Marques, Andreas Ettinger, C. B. Junior, Adriana Gonzalez-Sandoval, Renjitha Pillai, Mariella G. Filbin, M. Torres-Padilla, Ruslan I. Sadreyev, Capucine Van Rechem
          </td>
          <td>2024-12-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Compared to the concept of biobanks, which include one type of biomaterials from patients or a library of DNA samples, new types of biobanks shouldn’t be only repositories, but also infrastructure, which allows for innovative and translational research using biomaterials from patients. Also, parallel biobanking, including the simultaneous collection of various types of biomaterials (whole blood, plasma, exosomes, DNA, microRNA, leukocytes, tumor cells and their microenvironment - frozen with preservation of their viability, etc.) will allow them to be used in in-vitro models to test the effects of drugs, as well as to predict the development of treatment-resistant populations of cancer cells and cellular transformation. This biobanking approach opens new opportunities to study precancerous niches, rare forms of cancer and to develop personalized therapeutic strategies, giving researchers new opportunities for in vitro recapitulation of tissue mechanical and molecular changes, changes of signaling molecules profile and secretome.
 По сравнению с традиционным представлением о биобанках, включающих в себя один тип заготовляемых биоматериалов пациентов или коллекцию образцов ДНК, биобанки нового типа должны представлять собой не только репозитории, но и инфраструктуру, которая позволит проводить трансляционные и поисковые исследования с использованием биоматериалов от пациентов. Также, паралельное биобанкирование, включающее одновременный забор различных типов биоматериалов (цельная кровь, плазма, экзосомы, ДНК, микроРНК, лейкоциты, замороженные с сохранением своей жизнеспособности клетки опухолей и их микроокружения и т.д.) позволит использовать их в in vitro моделях для проверки действия лекарственных препаратов, а также для прогнозирования развития резистентных к лечению популяций раковых клеток и клеточной трансформации. Такой подход к биобанкированию открывает новые возможности для изучения предраковых ниш, редких форм рака и разработки персонализированных стратегий лечения, также обеспечивая возможность воспроизведения in vitro механической и молекулярной перестройки тканей, изменение профиля сигнальных молекул и секретома клеток.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbb3d987ca885908a90eb595ab47bf682fb4f746" target='_blank'>
              Parallel biobanking of post-COVID-19 cancer patient samples as an innovative route to multiomics screening
              </a>
            </td>
          <td>
            E.V. Petersen, D. Zorigt, D.A. Lifanov, E. Y. Shabalina, A. Khalil, A.A. Shiryaev, T.N. Pisareva, G. Zhemerikin, N. Sukortseva, P. Karalkin, I.V. Reshetov
          </td>
          <td>2024-12-27</td>
          <td>Head and neck. Russian Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a central role in cancer progression, therapy resistance, and disease recurrence. With the use of a quadruple-mutant mouse intestinal cancer organoid model and single-cell RNA-sequencing analysis, we have now identified glycosylphosphatidylinositol-specific phospholipase D1 (GPLD1), an enzyme that catalyzes the cleavage of glycosylphosphatidylinositol (GPI) anchors of membrane proteins, as a marker of slowly cycling CSCs. Ablation of Gpld1+ cells in combination with 5-fluorouracil treatment greatly attenuated cell viability in and regrowth of the intestinal cancer organoids. In addition, we identified serine protease 8 (PRSS8) as a key substrate of GPLD1 in human colorectal cancer cells. GPLD1 cleaves the GPI anchor of PRSS8 and thereby mediates release of the protease from the plasma membrane, resulting in the activation of Wnt signaling and promotion of the epithelial-mesenchymal transition (EMT) in the cancer cells. Pharmacological inhibition of GPLD1 suppressed Wnt signaling activity and EMT in association with upregulation of the amount of functional PRSS8 at the plasma membrane. Our findings suggest that targeting of GPLD1 in colorectal cancer might contribute to a new therapeutic strategy that is based on suppression of Wnt signaling and EMT-related cancer progression driven by CSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53a2de13be0cbce183bb9da3ac05db7fd47362af" target='_blank'>
              GPLD1+ cancer stem cells contribute to chemotherapy resistance and tumor relapse in intestinal cancer.
              </a>
            </td>
          <td>
            Taisuke Mizoo, Takeru Oka, Osamu Sugahara, Takafumi Minato, T. Higa, Keiichi I Nakayama
          </td>
          <td>2025-01-02</td>
          <td>Journal of biochemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Recurrent therapy resistance is a major limitation in clinical efficacy and for the outcome of glioblastoma (GBM) patients, positioning GBM among the tumor types with the poorest survival outcomes. In this work, we dissected resistance mechanisms in GBM, which resulted in the identification of FGFR1 pathway as a major regulator of the signaling and metabolic rewiring associated with temozolomide (TMZ) resistance in GBM. Hence, we described a mechanism of resistance that operates at two major levels. First, a p53-mediated regulation of cell cycle inducing cell cycle arrest to allow DNA repair in response to TMZ. And second, a complete metabolic rewiring promoting lipid catabolism and preventing lipid peroxidation. Both the p53-mediated response and the metabolic adaptation are controlled by FGFR1, as inhibition of the FGFR1 pathway completely abolishes this signaling and metabolic reprograming, restoring sensitivity to TMZ. Our results also indicated a correlation of FGFR1 levels with poor prognosis in GBM patients, and validated the treatment of TMZ in combination with FGFR1 inhibitors as an efficient strategy to induce tumor cell death in pre-clinical animal models. This data position the receptor FGFR1 as a very promising candidate for evaluation in future clinical approaches to limit the development of therapy resistance to TMZ in GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba711301ae1507c841a26c809bbac2e4e721e807" target='_blank'>
              FGFR1 inhibition improves therapy efficacy and prevents metabolic adaptation associated with temozolomide resistance in glioblastoma
              </a>
            </td>
          <td>
            Laura Zarzuela, Ignacio G. López-Cepero, K. Rattigan, Elena Sánchez-Escabias, Macarena Morillo-Huesca, Ana Reina-Bando, Sara S. Oltra, J. M. Sierra-Párraga, María Ceballos-Chávez, Vivian Capilla-González, Gema Moreno-Bueno, P. S. Murdoch, JC Reyes, G. V. Helgason, Mercedes Tomé, Raúl V. Durán
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e12a0e071ac585d8c8a1d6b9c8d498e4b4ab5313" target='_blank'>
              CYP3A5 promotes glioblastoma stemness and chemoresistance through fine-tuning NAD+/NADH ratio
              </a>
            </td>
          <td>
            Wentao Hu, Xiaoteng Cui, Hongyu Liu, Ze Li, Xu Chen, Qixue Wang, Guolu Zhang, Er Wen, Jinxin Lan, Junyi Chen, Jialin Liu, Chunsheng Kang, Ling Chen
          </td>
          <td>2025-01-03</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91aaef8a603816e9532df40a515bf3b479efc54f" target='_blank'>
              Cancer Stem Cell Regulation as a Target of Therapeutic Intervention: Insights into Breast, Cervical and Lung Cancer.
              </a>
            </td>
          <td>
            Sakshi Kevat, Archie Mistry, Naman Oza, Mohit Majmudar, Netra Patel, Rushabh Shah, A. V. Ramachandran, Ritu Chauhan, Shafiul Haque, N. C. Parashar, H. Tuli, Gaurav Parashar
          </td>
          <td>2025-01-23</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive brain tumor, characterized by high recurrence rates and poor patient outcomes. Treatment failure is driven by multiple factors, including complex tumor heterogeneity, the presence of cancer stem cells, the immunosuppressive tumor microenvironment (TME), and many others. GBM’s heterogeneity underlines its ability to resist therapies and adapt to the TME. The TME, which is highly immunosuppressive and shaped by hypoxia, impairs anti-tumor immunity and limits the efficacy of immunotherapy. The blood–brain barrier (BBB) remains a major obstacle to delivering sufficient drug concentrations to the tumor by restricting the penetration of therapeutic agents. Another problem is the lack of reliable biomarkers to perform better patient stratification or even guide personalized treatments, resulting in generalized therapeutic approaches that do not adequately address GBM complexities. This review highlights the multifactorial nature of GBM treatment failure and highlights the need for a paradigm shift and innovative, personalized strategies. A deeper understanding of tumor biology and advances in translational research will be crucial to developing effective therapies and improving patient outcomes in this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bfc84661a41258315c39a781e7c959f046327fe" target='_blank'>
              Why Do Glioblastoma Treatments Fail?
              </a>
            </td>
          <td>
            Alen Rončević, Nenad Koruga, Anamarija Soldo Koruga, Robert Rončević
          </td>
          <td>2025-02-01</td>
          <td>Future Pharmacology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are large, acentric, circular DNA molecules that occur pervasively across many human cancers. EcDNA can drive tumor formation and evolution, contribute to drug resistance, and associate with poor patient survival outcomes. Beyond mediating high copy numbers, the circular topology and dynamic conformational changes of ecDNA disrupt topological domains and rewire regulatory networks, thereby conferring an important role in the transcriptional regulation of oncogenes. Here, we develop ec3D, a computational method for reconstructing the three-dimensional structures of ecDNA and analyzing significant interactions from high-throughput chromatin capture (Hi-C) data. Given a candidate ecDNA sequence and the corresponding whole-genome Hi-C as input, ec3D reconstructs the spatial structure of ecDNA by maximizing the Poisson likelihood of observed interactions. Ec3D’s performance was validated using both simulated ecDNA structures with varying conformations, and Hi-C data from previously-characterized cancer cell lines. Our reconstructions reveal that ecDNAs occupy spherical configurations and mediate unique long-range interactions involved in gene regulation. Through algorithmic innovations, ec3D can resolve complex ecDNA structures with duplicated copies of large genomic segments, identify multi-way interactions, distinguish between interactions arising from direct spatial proximity and secondary interactions resulting from alternative folding patterns or intermolecular (trans) contacts of ecDNA molecules. Our findings provide insights into how the spatial organization of ecDNA may influence gene regulation and contribute to increased oncogene expression. Code availability https://github.com/AmpliconSuite/ec3D">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebb6e832c80fde421190f885801636b3438e629" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachro-mosomal DNA with ec3D
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, J. Luebeck, Owen S. Chapman, Katerina Kraft, Shu Zhang, Lukas Chavez, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Early-onset head and neck squamous cell carcinoma (HNSCC) has been increasingly observed in recent years, exhibiting distinct tumor behavior and a unique tumor microenvironment (TME) compared to older age groups. Studies suggest that early-onset HNSCC is associated with specific risk factors and prognostic outcomes, while the underlying mechanisms driving these age-related differences remain unclear. In this review, we systematically examined original studies involving young HNSCC patient samples, focusing on the characteristics of the TME and potential for personalized immunotherapy. While further evidence is needed, our findings indicate that the TME in early-onset HNSCC often exhibits higher aggressiveness and immune suppression. Consequently, tailored immunotherapy may offer a promising therapeutic strategy for this distinct patient population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b24c9659704e306326c563cf26bcbe9c6395a46" target='_blank'>
              Understanding the tumor microenvironment for personalized immunotherapy in early-onset head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Yidan Shan, Di He, Fengguo Yan, Weijia Huang
          </td>
          <td>2025-01-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell states evolve through the combined activity of signaling pathways and gene regulatory networks. While transcription factors can direct cell fate, these factors rely on a cell state that is receptive to transitions in cell identity. How signaling levels contribute to the emergence of receptive cell states remains poorly defined in primary cells. Using a well-defined model of direct conversion, we examined how levels of the MAPK-activating oncogene HRASG12V influence direct conversion of primary fibroblasts to induced motor neurons. We demonstrate that an optimal ‘Goldilocks’ level of MAPK signaling efficiently drives cell-fate programming. Rates of direct conversion respond biphasically to increasing HRASG12V levels. While intermediate HRASG12V levels increase the rate of conversion, high levels of HRASG12V induce senescence. Through chemogenetic tuning, we set optimal MAPK activity for high rates of conversion in the absence of HRAS mutants. As MAPK pathways influence cell-fate transitions in development and disease, our results highlight the need to tune therapeutic interventions within a non-monotonic landscape that is shaped by genetics and levels of gene expression. Highlights MAPK signaling drives proliferation and conversion of fibroblasts to motor neurons Cell-fate programming responds biphasically to HRASG12V expression High HRASG12V expression induces senescence, which reduces conversion Chemogenetic tuning of MAPK activity increases conversion rates A small-molecule MAPK inducer drives high rates of conversion in the absence of HRASG12V">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa4a4e600cf88c5b704f193ed57b3b5cfc188f0" target='_blank'>
              Chemogenetic tuning reveals optimal MAPK signaling for cell-fate programming
              </a>
            </td>
          <td>
            Brittany A. Lende-Dorn, Jane C. Atkinson, Yunbeen Bae, Kate E. Galloway
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [3, 5],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>