<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-10-13 09:31:59 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Aggressive cancers, such as pancreatic ductal adenocarcinoma (PDAC), are often characterized by a complex and desmoplastic tumor microenvironment, a stroma rich supportive connective tissue composed primarily of extracellular matrix (ECM) and non-cancerous cells. Desmoplasia, a dense deposition of stroma, is a major reason for therapy resistance, acting both as a physical barrier that interferes with drug penetration and as a supportive niche that protects cancer cells through diverse mechanisms. Precise understanding of spatial cell interactions in stroma-rich tumors is essential for optimizing therapeutic responses. It enables detailed mapping of stromal-tumor interfaces, comprehensive cell phenotyping, and insights into changes in tissue architecture, improving assessment of drug responses. Recent advances in multiplexed immunofluorescence imaging have enabled the acquisition of large batches of whole-slide tumor images, but scalable and reproducible methods to analyze the spatial distribution of cell states relative to stromal regions remain limited. To address this gap, we developed an open-source computational pipeline that integrates QuPath, StarDist, and custom Python scripts to quantify biomarker expression at a single- and sub-cellular resolution across entire tumor sections. Our workflow includes: (i) automated nuclei segmentation using StarDist, (ii) machine learning-based cell classification using multiplexed marker expression, (iii) modeling of stromal regions based on fibronectin staining, (iv) sensitivity analyses on classification thresholds to ensure robustness across heterogeneous datasets, and (v) distance-based quantification of the proximity of each cell to the stromal border. To improve consistency across slides with variable staining intensities, we introduce a statistical strategy that translates classification thresholds by propagating a chosen reference percentile across the distribution of marker-related cell measurement in each image. We apply this approach to quantify spatial patterns of distribution of the phosphorylated form of the N-Myc downregulated gene 1 (NDRG1), a novel DNA repair protein that conveys signals from the ECM to the nucleus to maintain replication fork homeostasis, and a known cell proliferation marker Ki67 in fibronectin-defined stromal regions in PDAC xenografts. The pipeline is applicable for the analysis of markers of interest in stroma-rich tissues and is publicly available.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b6327fb55ba3776af18a8a1265a4ec0c3c69a0b" target='_blank'>
              An image analysis pipeline to quantify the spatial distribution of cell markers in stroma-rich tumors
              </a>
            </td>
          <td>
            Antoine A. Ruzette, Nina Kozlova, Kayla A. Cruz, Taru Muranen, S. Nørrelykke
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) presents diverse histomorphological features within the tumor microenvironment (TME) that influence prognosis and response to immunotherapy. Leveraging contrastive learning, we developed an unbiased atlas of cell neighborhoods to systematically explore the LUAD microenvironment at the cellular scale and investigate how these neighborhoods combine to form histologic patterns. This multiscale approach enables a comprehensive understanding of both cell-specific interactions and broader histologic architecture in LUAD. Our analysis identified distinct histomorphological phenotype clusters of cellular neighborhoods (cn-HPCs) with prognostic significance. For example, cn-HPC 0 was associated with immune activation and favorable survival, while cn-HPC 23 was enriched in necrotic, immune-excluded regions and aligned with poorer outcomes. Through associations with immunophenotypes, co-expressed gene modules, and pathway enrichment, we found that cn-HPCs reflect underlying processes of immune modulation, cellular growth, and inflammation, offering insights into the functional landscape of LUAD. In an independent LUAD immunotherapy cohort, we found that cn-HPC 23 showed promising predictive value for treatment response. These findings suggest that fine-grained spatial profiling of the TME may offer complementary value for identifying patients more likely to benefit from immunotherapy. Taken together, our study highlights the potential of cn-HPCs to bridge histopathological patterns and clinical outcomes, supporting their relevance for prognosis and treatment stratification in LUAD. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17117-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732a50286d99e4889f11b9dfeef8dfb5de086217" target='_blank'>
              Contrastive learning uncovers cellular interactions and morphologies in the tumor microenvironment of lung adenocarcinoma linked to immunotherapy response
              </a>
            </td>
          <td>
            H. Le, N. Coudray, A. Yeaton, Sitharam Ramaswami, A. Karimkhan, , James Cai, Tai-Hsien Ou Yang, S. Punekar, Luis Chiriboga, Vamsidhar Velcheti, Kwok-Kin Wong, D. Sterman, Andre L. Moreira, Harvey I. Pass, A. Tsirigos
          </td>
          <td>2025-10-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Cancer immunotherapy has revolutionized clinical oncology; however, the inherent complexity and heterogeneity of cancer present substantial challenges to achieving broad therapeutic efficacy. Tumor heterogeneity manifests not only among different patients but also within individual tumors, further complicating personalized treatment approaches. Single-cell sequencing technologies encompassing genomics, transcriptomics, epigenomics, proteomics, and spatial omics have significantly enhanced our ability to dissect tumor heterogeneity at single-cell resolution with multi-layered depth. These approaches have illuminated tumor biology, immune escape mechanisms, treatment resistance, and patient-specific immune response mechanisms, thereby substantially advancing precision oncology strategies. This review systematically examines recent advances in single-cell multi-omics technologies across various cancer research areas, emphasizing their transformative impacts on understanding tumor heterogeneity, immunotherapy, minimal residual disease monitoring, and neoantigen discovery. Additionally, we discuss current technical and analytical limitations and unresolved questions associated with single-cell technologies. We anticipate single-cell multi-omics technologies will become central to precision oncology, facilitating truly personalized therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3005de124c813e8dd9ae4675ad8451e32f8f1a34" target='_blank'>
              Single-cell multi-omics in cancer immunotherapy: from tumor heterogeneity to personalized precision treatment
              </a>
            </td>
          <td>
            Jiayuan Le, Yating Dian, Deze Zhao, Ziyu Guo, Zehao Luo, Xiang Chen, Furong Zeng, Guangtong Deng
          </td>
          <td>2025-08-25</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74b1a84299cf4bd86e16928f5b0911b15d09c6d9" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Yuntian Fu, D. Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z Lin, Dylan Schaff, Sydney M Shaffer, Drew M Pardoll, Christina Jackson, Nancy R Zhang
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0987fea6e48fd7cd3a2b34fbb805d81b9c4a798" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Nancy R Zhang, Yuntian Fu, D. Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z Lin, Dylan Schaff, Sydney M Shaffer, Drew Pardoll, Christina Jackson
          </td>
          <td>2025-09-24</td>
          <td>Research Square</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Tumors acquire a set of functional capabilities, known as the Hallmarks of Cancer, to thrive and grow. While extensive research has revealed how gene and protein expression networks contribute to these traits, the relationships between tumor cell states, spatial organization of immune and stromal cells in the tumor microenvironment, and tumor genetics, remains poorly understood. To address this, we constructed a cohort of primary tumor tissues from 117 pancreatic ductal adenocarcinoma (PDAC) patients. Using Multiplexed Ion-Beam Imaging (MIBI) with a 40-antibody panel, we mapped the spatial organization of major cell types and their phenotypes. We also conducted targeted genomic and bulk proteomic analyses to explore how spatial organization patterns align with tumor genetics and proteome profiles. We identified phenotypic subtypes of tumor epithelium that shape distinct microenvironmental niches: (i) Glycolytic tumors expressing proteins such as GLUT1, LDHA, MCT1 and HIF1α reside in hypoxic niches. They are encased by dense fibrotic stroma and lacking vasculature and immune cell infiltration. (ii) Tumor and metaplastic epithelium that exhibit an oxidative proteome profile, are highly enriched for ECAD and PD-L1 expression, and are situated near pancreatic epithelium and vasculature, with significant immune infiltration. (iii) Myeloid-interacting tumors expressing CXCL5, heavily infiltrated with neutrophils and characterized by low-collagen and high FAP positive fibroblasts. Notably, neutrophils accumulate within the lumen of tumor ducts where they exhibit increased apoptosis, DNA damage, and netosis. These highly neutrophil-infiltrated tumors, associate with poor disease outcome. The genetic landscape of PDAC influences these spatial and phenotypic patterns. Patients with TGFβ/SMAD pathway mutations, demonstrate an increased proportion of basal tumor cells accompanied by reduced lymphocytic infiltration. Similarly, distinct TP53 variants shape immune-cell infiltration: patients harboring missense mutations show increased lymphocytic infiltration compared to those with truncating mutations. These findings underscore the strong link between tumor genetics, phenotypic heterogeneity and cellular architecture of the PDAC microenvironment, offering new insights into collective tumor behavior.



 Ofer Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, Inti Zlobec, Tamar Geiger, Leeat Keren. Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B123.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb5d6211d2e33680561c9558492cf85ff16b33" target='_blank'>
              Abstract B123: Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization
              </a>
            </td>
          <td>
            O. Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, I. Zlobec, Tamar Geiger, Leeat Keren
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Intrinsic genetic alterations and dynamic transcriptional changes contribute to the heterogeneity of solid tumors. Lung adenocarcinoma (LUAD) is characterized by its significant histological, cellular and molecular heterogeneity. The present study aimed to study the spatial transcriptomics of primary LUAD with initial hopes to decipher molecular characteristics of subtype transitions in LUAD progression, offering new insights for novel therapeutic strategies. Spatial transcriptomics libraries were first generated from tumor samples collected from patients with LUAD who underwent surgical resection in The Fourth Hospital of Hebei Medical University in 2022 and were sequenced using Illumina NovaSeq 6000 system. The processed data were analyzed for differential gene expressions and networks, and were annotated according to cell type, spatial ligand-receptor interaction and trajectory inference. Our analysis revealed 34 annotated cell types, with cancer-associated fibroblasts (CAFs) being the most abundant, playing a crucial role in tumor microenvironment remodeling and prognosis. We noted significant spatial correlations between various immune cells and found that different histological subtypes displayed unique cell composition profiles, particularly in the micropapillary subtype, which exhibited higher macrophage proportions and distinct gene expression pathways related to extracellular matrix organization and receptor tyrosine kinase signaling. Additionally, we explored the dedifferentiation states within these subtypes, identifying that region with higher dedifferentiation scores corresponded to increased tumor invasiveness and potential drug resistance. Our findings demonstrate dynamic biological changes and dedifferentiation states of tumor subtypes during the progression process. This study reveals important biological processes in tumor development and may offer valuable guidance for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a252a3c50c2377c8fca1733740eaa8f4897e893" target='_blank'>
              Spatial transcriptomics uncovers immune-cell plasticity and dedifferentiation signatures in aggressive lung adenocarcinoma subtypes
              </a>
            </td>
          <td>
            Huiyan Deng, Qingyi Liu, Ziqiang Tian, Guan-Yuan Liu, Shize Wang, Di Liang, Yun Wang, Yaqing Han, Shaonan Xie
          </td>
          <td>2025-08-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cell plasticity, defined as the ability of tumor cells to reversibly adopt distinct functional states, plays a central role in tumor heterogeneity, therapy resistance, and disease relapse. This process enables cells to enter stem-like, dormant, or drug-tolerant persister states in response to treatment or environmental stress without undergoing genetic changes. Such reversible transitions complicate and limit current treatments. Conventional cancer models often fail to capture the complexities of these adaptive states. In contrast, patient-derived tumor organoids (PDOs), which retain the cellular diversity and structure of primary tumors, provide a unique system for investigating plasticity. This review describes how PDOs can model cellular plasticity, such as the emergence of drug-tolerant persister cells and the interconversion between cancer stem cell states across multiple tumor types. We particularly focused on colorectal cancer organoids, for which research on the mechanism of plasticity is the most advanced. Combined with single-cell analysis, lineage tracing, and functional assays, PDOs can help identify the molecular pathways that control plasticity. Understanding these mechanisms is important for developing therapies to prevent treatment failure and control disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd4e7e497e61cac197e5ff80d3e09c68cd18937b" target='_blank'>
              Patient-Derived Tumor Organoids to Model Cancer Cell Plasticity and Overcome Therapeutic Resistance
              </a>
            </td>
          <td>
            R. Coppo, Masahiro Inoue
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Spontaneously transformed mouse ovarian surface epithelial cell line ID8 generates a syngeneic mouse model, recapitulating the primary and metastatic stages of the malignancy hence, frequently used in pre-clinical evaluation. Due to inherent phenotypic plasticity, primary and metastatic tumors show contrasting phenotypic features dictated by the tissue niche. To address the tumor heterogeneity within the model we stratified it into a specific molecular subtype, which would provide an accurate setting for interpretation of experimental data. Stratification was performed on ID8 cell line and ID8 generated xenografts based on immunostaining, live cell imaging, histopathology, transcriptomics and proteomics. Tumor microenvironment (TME) investigation for each xenograft was done through flow cytometry analysis. ID8 cells present epithelial/mesenchymal (E/M) phenotype. Phenotypic plasticity retention in them causes class switching of ID8 cells in response to niche alteration, from E/M (in culture) to intermediate epithelial (iE) in orthotopic tumors while intermediate mesenchymal (iM) in subcutaneous tumors. Metastasis of ID8 primary tumor resulted into differentiated mesenchymal phenotype in ascites while differentiated epithelial phenotype in secondary intestinal tumors. Immune profiling of primary and secondary xenografts revealed differences in immune repertoire of tumors, highlighting subcutaneous tumor model to be inappropriate as compared to orthotopic model. Local tissue niche modulates the phenotype of tumors generated, transitioning through intermediate states towards a final commitment to either an epithelial or mesenchymal state that recapitulates the reported class switching in patient tumors. During disease progression, such heterogeneity of tumor cells at different niche(s) and their cross-talks with discrete TME at each site, will pose challenges for effective therapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18264-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20023e2da4c4317c3adb1e410526c772109c27da" target='_blank'>
              ID8 cells manifest phenotypic plasticity and molecular heterogeneity of high-grade serous ovarian cancer in response to the local tissue niche
              </a>
            </td>
          <td>
            Ritika Gupta, S. Shravan, Swapna Balaji Agwane, R. M. Bankar, S. Bapat
          </td>
          <td>2025-10-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is a molecularly diverse form of lung cancer characterized by distinct oncogenic driver mutations that influence both tumour biology and clinical outcomes. Understanding the interplay between these oncogenic drivers and the tumour microenvironment (TME) is crucial for improving therapeutic strategies and patient management. Here, we investigate the impact of driver mutations on the composition and spatial architecture of the TME in LUAD. Using imaging mass cytometry (IMC), we analyse tumour samples from 157 LUAD patients, integrating genomic and clinical data to link specific mutations with tumour characteristics. Unique patterns are associated with mutated KRAS and EGFR tumours with TP53 co-mutations, suggesting these co-mutations reshape the TME and promote resistance to tyrosine kinase inhibitors (TKIs). Overall, our findings highlight the complex interplay between oncogenic driver mutations and the TME in LUAD, underscoring the importance of integrating genomic and cellular data to understand the underlying tumour behaviour and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05eac26faa3f5fa852d857cbc84d4c9c3228ae87" target='_blank'>
              Oncogenic driver mutations underlie the spatial tumour immune landscape of non-small cell lung cancer
              </a>
            </td>
          <td>
            Saskia Hartner, H. Abolfathi, M. Rezanejad, Bridget Liu, E. Karimi, Dakota Rogers, Mark Sorin, Samuel Doré, Lysanne Desharnais, M. Orain, William Enlow, A. Gagné, Yuhong Wei, Y. Bossé, D. Quail, Philippe Joubert, L. Walsh
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Simple Summary Spatial transcriptomics is an advanced technology that allows for the study of gene activity in cancer tissues while preserving the original tissue structure. Unlike traditional methods that either analyze samples without location information or examine individual cells removed from their natural environment, spatial transcriptomic technology maps where genes are active within intact tumor tissue. This approach enables a greater understanding of how different areas of a tumor vary, how cancer cells interact with surrounding healthy tissue, and how cells are organized spatially within the tumor at high resolution. As spatial technology becomes more widely used, it may improve cancer diagnosis and treatment by helping to create more personalized and effective therapies for individual patients. This review outlines the current and future role of spatial transcriptomics methodologies in the context of genitourinary cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa8ee089aa47992263f8b69f0d5705b49d03f797" target='_blank'>
              Current Role and Future Frontiers of Spatial Transcriptomics in Genitourinary Cancers
              </a>
            </td>
          <td>
            Firas Hatoum, Adnan Fazili, Justin W Miller, Xuefeng Wang, Xiaoqing Yu, Xin Lu, Jeffrey S. Johnson, Philippe E. Spiess, J. Chahoud
          </td>
          <td>2025-08-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Adult gliomas are a diverse class of tumors with substantial molecular and pathological variation. While prior single-cell omics studies have characterized intratumoral heterogeneity, they often examined tumors in isolation, thus overlooking the recurrence of tumor states across subtypes. Given their shared environment, understanding the conservation of tumor states across gliomas is essential.



 To address this, we analyzed 310 tumor cores from 284 patients spanning oligodendroglioma, low- and high-grade astrocytoma, glioblastoma, and ependymoma, using single-cell spatial transcriptomics with a 344-gene panel (10x Xenium). This approach yielded a dataset of 2.8 million cells, of which 84% were tumor and 16% represented 18 non-malignant cell types from the tumor microenvironment (TME). To further investigate local cellular interactions, we applied an extended spatial transcriptomics panel (5,096 genes) alongside multiplexed immunofluorescence (57 proteins) to a validation cohort of low-grade astrocytomas.



 Our analysis revealed tumor states recurring across gliomas, albeit in varying proportions, organized into neighborhoods that reflect local cell-cell interactions. For instance, proliferating tumor cells tended to co-localize with oligodendro- and neuro-progenitor-like tumor states, suggesting that these programs may give rise to proliferating cells. Spatial neighborhoods were primarily subtype-specific, indicating that tissue organization and not cell composition alone, underlies glioma heterogeneity. We next assessed the relationship between tumor and immune cells, revealing entity-specific immune microenvironments: astrocytoma and ependymoma exhibited distinct immune responses (T cell- versus myeloid-driven inflammation), whereas oligodendroglioma displayed a low inflammatory landscape. Notably, microglia were consistently observed in proximity to tumor cells across all glioma subtypes, highlighting their conserved role in modulating anti-tumor immunity. Methylation profiling stratified these tumors into 13 molecular subtypes, each associated with distinct neighborhoods. Glioblastoma subtypes RTK1, RTK2, and MES, in particular, displayed unique tissue architectures consistent with their subtype-specific methylation profiles.



 Collectively, these findings link microanatomical features to glioma heterogeneity by connecting tumor states to their microenvironments. Regional differences, such as those between the tumor core and infiltrating edge, shape cellular composition with implications to treatment response. By capturing such spatially dependent patterns, our work provides a framework for advancing the understanding of glioma biology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34fbb58728bc44ec20e86f909612c57947c86652" target='_blank'>
              KS03.6.A THE RECURRENT TISSUE STATES ACROSS ADULT GLIOMAS AS REVEALED BY SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            A. Mathioudaki, Z. Seferbekova, M. Ritter, D. Calafato, G. Rukhovich, F. Hinz, F. Ippen, E. Popova, N. Koeberer, P. Sant, S. Schinkewitsch, J. Malm, S. Dietrich, C. Herold-Mende, N. Etminan, A. Wick, S. Krieg, M. Platten, A. von Deimling, F. Sahm, A. Suwala, M. Gerstung
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="SUMMARY Metastasis is an emergent continuum, driven by evolving reciprocal adaptations between continuously disseminating tumor cells (DTCs) and the specialized metastatic niches of distant organs. The interplay between intrinsic and niche-driven mechanisms that enables DTCs to survive and home to distant organs remains incompletely understood. Here, using MetTag, a single-cell barcoding and transcriptome profiling approach with time-stamped batch identifiers (BC.IDs), we mapped temporal, clonal dynamics of DTCs and the immune cell landscape across ovarian cancer metastatic niches. Deep sequencing of barcodes revealed preferred enrichment of early-disseminated clones across metastatic niches. Mechanistically, single-cell RNA sequencing (scRNA-seq) coupled with velocity analyses in ascites and metastasis-bearing omenta uncovered an emergent, distinct interferon-gamma (IFNγ) centric transcriptional trajectory, enriched among early seeding clones. Moreover, in vivo CRISPR/Cas9 screening of metastatic niche-specific signatures demonstrated that genes belonging to the ascites IFNγ signature, including Marco, Gbp2b, and Slfn1, are functionally important for peritoneal metastasis. Knockout of IFNγ receptor 1 (Ifngr1) in tumor cells significantly reduced metastatic burden and extended survival, underscoring the importance of tumor cell intrinsic IFNγ signaling in ovarian cancer metastasis. Furthermore, we identified that the tumor intrinsic IFNγ response and ascites-derived tumor-associated macrophages (TAMs) protect cancer cells from anoikis-mediated death within the IFNγ-rich ascites environment. Our study resolves temporal dynamics of disseminating tumor cells and highlights an ascites-driven, IFNγ program as a necessary pro-metastatic adaptation in the ovarian metastasis cascade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95e0d9496ced48f75294cbc20541bbba656bbf" target='_blank'>
              Temporal Clonal Tracing and Functional Perturbation Reveal Niche-Adaptive and Tumor-Intrinsic IFNγ Dependencies Driving Ovarian Cancer Metastasis
              </a>
            </td>
          <td>
            Emilija Aleksandrovic, Shaneann Fross, Samantha M. Golomb, Xiyu Liu, Zhuo Zhao, Nikitha M. Das, T. Reese, Wei Ma, Jacqueline Lopez, M. S. Stack, Min Zhao, Siyuan Zhang
          </td>
          <td>2025-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Objectives Small cell lung cancer (SCLC) accounts for approximately 15% of lung tumors and is marked by aggressive growth and early metastatic spread. In this study, we used two SCLC mouse models with differing tumor mutation burdens (TMB). To investigate tumor composition, spatial architecture, and interactions with the surrounding microenvironment, we acquired multiplexed images of mouse lung tumors using imaging mass cytometry (IMC). These data build upon a previously published characterization of the mouse model. Data description After tumor detection, mice were assigned to one of five treatment groups. Lung tumor tissues were imaged with a 37-marker IMC panel designed to identify major cell types—tumor, immune, and structural—as well as their functional states. When possible, each tumor was sampled both at its center and border regions. Tumor masks in the form of binary images are provided to delineate tumor areas. Additional metadata include tumor onset and endpoint dates to support downstream correlation or predictive analyses based on the image data. This dataset offers a valuable resource for studying the histological and cellular complexity of SCLC in a genetically controlled mouse model across multiple therapeutic conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597aa5e8dfd6901b69ffa28e7837ffa96623cc96" target='_blank'>
              Imaging mass cytometry dataset of small-cell lung cancer tumors and tumor microenvironments
              </a>
            </td>
          <td>
            France Rose, O. Ibruli, Luca Lichius, M. Kiljan, Gokcen Gozum, Manoela Iannicelli Caiaffa, Jiali Cai, Li‐na Niu, Jan M. Herter, Holger Grüll, Reinhard Büttner, F. Beleggia, Graziella Bosco, Julie George, G. Herter-Sprie, H. C. Reinhardt, Katarzyna Bożek
          </td>
          <td>2025-09-08</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Stem-like gastric cancer (GC) is an aggressive molecular subtype marked by poor prognosis and limited response to immune checkpoint blockade (ICB). The spatial mechanisms driving this resistance remain unclear. Methods We conducted spatially resolved single-cell transcriptomic profiling of diffuse-type GC tissues to uncover the spatial architecture and functional diversity of tumor and stromal populations. Cellular heterogeneity and region-specific signaling pathways were characterized using integrative bioinformatics analyses. Results We identified transcriptionally diverse, high-entropy cell populations predominantly localized in the deep tumor regions. These included unique endothelial and fibroblast subsets enriched for pro-tumorigenic and immune-regulatory signaling. A notable finding was the engagement of deep-region endothelial cells in VISFATIN (extracellular NAMPT) signaling through the ITGA5–ITGB1 integrin axis, associated with immune evasion and poor prognosis. This endothelial signaling program is distinct from and functionally independent of cancer-associated fibroblast (CAF)-mediated pathways. Elevated expression of the NAMPT–ITGA5–ITGB1 axis was observed in ICB non-responders and correlated with reduced overall survival. Conclusions Our study delineates spatially defined cellular programs that contribute to immune escape in stem-like GC, highlighting a novel VISFATIN–integrin signaling axis as a potential biomarker and therapeutic target in immunotherapy-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dd6bcdbe10e93d5e57b978f2adeff6ec26c9091" target='_blank'>
              Spatially resolved endothelial signaling via nampt-itga5 drives immune evasion in stem-like gastric cancer
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jae-Ho Cheong, Eui Tae Kim
          </td>
          <td>2025-09-29</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Owing to the complexity of TME components and the heterogeneity of cancer cells, the relationship between the niches of TME and prognosis in breast ductal carcinoma remains unknown. The staged characteristics of corresponding cancer cell behaviors are unclear. Our study aims to reveal spatial structures and specific cellular information of TME and cancer cells subgroups during the progression from DCIS to IDC and lymph node metastasis. Methods Single-cell sequencing, spatial transcriptomics, bulk RNA sequencing datasets were used to explore the changes in microenvironmental components and transcriptional programs of tumor cells during the progression of breast ductal carcinoma. Immunohistochemistry, multiplex immunofluorescence, flow cytometry cell cycle detection, invasion migration experiments, and WB imprinting were employed for validation. Results Analysis of TME cell type subsets revealed the accumulation of TEX, iTreg, and stress-phenotype TAM in the mammary gland in situ during the invasion process. Lymphatic metastases exhibited enrichment of nTregs and a more naïve-like CD8 T cell population. Spatial analysis and survival analysis showed that the spatial niches of CD4 TN and phagocytic-phenotype macrophages were associated with a favorable prognosis, and these niches were lost during disease progression. The proliferative subpopulation of breast ductal carcinoma was enriched in lymphatic metastatic tissues, expressing high levels of FAM111B and exhibiting intense TCA and oxidative phosphorylation metabolism. Silencing FAM111B led to cell cycle arrest, decreased invasion and migration abilities, and downregulation of core mediator genes for cuproptosis and disulfidptosis. Conclusions The stage-specific microenvironmental characteristics of breast ductal carcinoma correspond to some extent to the behavior of tumor cells. During the progression of ductal carcinoma in breast tissue, the establishment of an immunosuppressive microenvironment occurs. The microenvironmental spectrum at lymph node metastases differs somewhat, corresponding to a more enriched turnover of cancer cell proliferation and death. Inhibitors of FAM111B and inducers of cuproptosis and disulfidptosis may serve as potential therapeutic targets for proliferative subgroups. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07010-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfce9a8ff47f521350ed382190ba1004f756bb5c" target='_blank'>
              Spatiotemporal microenvironment landscape and malignant epithelial pattern transition in breast ductal carcinoma progression
              </a>
            </td>
          <td>
            Xifu Cheng, Wenjuan Zeng, Bingzhe Yin, Jiawei Gui, Hengbin Zhang, Zhenxing Lv, Simin Zhang, Yao Zhou
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background The presence of tertiary lymphoid structures (TLS) in solid tumors, including Merkel cell carcinoma (MCC), is associated with a better prognosis and a better response to immunotherapy with immune checkpoint inhibition (ICI). The detailed mechanisms by which TLS influence antitumor immune responses are only partially understood. Methods Clinically annotated tumor tissues of 27 patients with MCC were obtained prior to ICI therapy. Tumor samples were subjected to transcriptomic and multiplex immuno-visual profiling, T-cell receptor (TCR) clonotype mapping, as well as—in selected cases—spatial transcriptomics to comprehensively characterize the tumor immune microenvironment. Results Weighted gene co-expression network analysis (WGCNA) of transcriptomic data in combination with topological overlap measures indicated a higher abundance of TLS in tumors of patients with MCC responding to ICI therapy. This concept was substantiated through immunomorphological analyses, revealing mature B-cell follicle-like structures characterized by high endothelial venules (HEVs). Further supporting HEVs as critical entry points for naïve T cells, the presence of TLS was correlated with a pronounced infiltration of CD4+ and CD8+ T cells, exhibiting both naïve and central memory phenotypes. The TCR repertoire of these infiltrates exhibited enhanced richness and diversity with a pronounced reactivity toward Merkel cell polyomavirus-derived T-cell epitopes. Spatially resolved RNA and V(D)J sequencing revealed the expression of genes associated with T-cell recruitment within TLS, alongside the presence of naïve and central memory T-cell markers. Notably, individual clonally expanded TCR transcripts were detected both within TLS and among tumor-infiltrating lymphocytes. The latter were associated with low expression of memory cell markers and high expression of effector cell markers. Additionally, a spatial gradient in the expression of genes linked to immune stress in MCC cells—such as those involved in the interferon-γ response and antigen processing and presentation machinery—originated in proximity to the TLS. Conclusion Our findings are consistent with a key role of TLS in shaping immune interactions within the MCC microenvironment, driving the recruitment of diverse tumor-reactive T cells. These insights hold promise for advancing immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13de7f3d541927495612436f916863e4107c4925" target='_blank'>
              Tertiary lymphoid structures in Merkel cell carcinoma facilitate naïve and central memory T-cell infiltration linked to immunotherapy response
              </a>
            </td>
          <td>
            Nalini Srinivas, Ivelina Spassova, Kuan Cheok Lei, Jiwei Gao, María José Pino, Giovanni Giglio, S. Kitanovski, Mazdak Dalkoohi, E. Livingstone, Ulrike Leiter, Peter Mohr, T. Gambichler, Ingo Stoffels, Selma Ugurel, Phyllis Fung-Yi Cheung, Camilla Engblom, Weng-Onn Lui, Jürgen C. Becker
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality, with tumor heterogeneity and drug resistance posing significant challenges to treatment. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq deconvolution to analyze BRCA samples. Our analysis identified 15 major cell clusters, including neoplastic epithelial, immune, stromal, and endothelial populations. Notably, low-grade tumors showed enriched subtypes, such as CXCR4+ fibroblasts, IGKC+ myeloid cells, and CLU+ endothelial cells, with distinct spatial localization and immune-modulatory functions. These subtypes were paradoxically linked to reduced immunotherapy responsiveness, despite their association with favorable clinical features. High-grade tumors exhibited reprogrammed intercellular communication, with expanded MDK and Galectin signaling. Bulk RNA-seq deconvolution further supported the prognostic significance of low-grade-enriched subtypes. Our findings highlight the heterogeneity of the tumor microenvironment and provide new insights into immune evasion and therapeutic resistance in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57925849e8c34466fe3132e9855985812e402540" target='_blank'>
              Machine learning analysis reveals tumor heterogeneity and stromal-immune niches in breast cancer
              </a>
            </td>
          <td>
            Junjie Kuang, Guofang Zhong, Linfeng Zhao, Xia Yuan, Yundong Zhou, Jun Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Meningiomas represent the most common primary intracranial tumors, accounting for approximately one-third of adult CNS tumors. Despite their frequency, model systems that accurately replicate the complex tumor-microenvironment interactions across meningioma grades remain underdeveloped, limiting translational progress. This study aimed to establish and characterize human meningioma tumor models to interrogate grade-specific cellular behaviors and microenvironmental dynamics



 Patient-derived meningioma cell lines were established through mechanical and enzymatic dissociation of surgically resected tumors, followed by expansion in specialized culture media. GFP transfection was performed to enable live-cell tracking. For ex vivo modeling, human neocortical slices — obtained from access tissue from non-meningioma surgeries, either 2 cm distal to the glioma core or from epilepsy surgeries under neuronavigation guidance — were injected with GFP-labeled meningioma cells. Tumor progression, cluster formation, and tissue integration were assessed over six days using longitudinal live imaging and immunofluorescence staining. Single-cell RNA sequencing (scRNA-seq) was employed to resolve cellular heterogeneity and tumor-microenvironmental interactions. Machine learning-based image segmentation combined with manual curation enabled precise quantification of tumor cell distribution and interaction patterns within the host tissue.



 Grade 3 meningioma cells demonstrated significantly enhanced cluster formation, tissue invasion, and microenvironmental remodeling compared to lower-grade counterparts. Ex vivo models recapitulated patient tumor behaviors, with scRNA-seq revealing upregulation of genes associated with mesenchymal transition, extracellular matrix remodeling, and immune modulation in Grade 3 tumors. Microenvironmental analysis indicated that higher-grade tumors established more extensive cell-cell interaction networks and induced greater astrocytic and microglial activation relative to Grade 1 and 2 models. Furthermore, immune profiling revealed a more immunosuppressive environment, with myeloid-derived suppressor cells, reducing immune surveillance and facilitating tumor evasion. Additionally, there was increased activation of astrocytes and microglia, contributing to a supportive tumor microenvironment.



 Our patient-derived meningioma models reveal grade-specific differences in tumor behavior and TME interactions. Grade 3 meningiomas show aggressive clustering and microenvironmental remodeling, offering insights into malignant progression. These models provide a high-fidelity platform for studying tumor biology and preclinical evaluation of therapies targeting tumor-stroma interactions
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b180fe0ce6bc7373d74452191faf8369d3bd9e2c" target='_blank'>
              P13.17.B HUMAN MENINGIOMA TUMOR MODELS REVEAL GRADE-SPECIFIC MICROENVIRONMENTAL INTERACTIONS
              </a>
            </td>
          <td>
            A O Mohammed, K. Joseph, D. Depro, G. Nishita, V. Jariwala, A. Boecke, M. Al Shhab, I. Vasilikos, J. M. Nakagawa, T. Juratli, R. Roelz, M Shah, J. Beck, J. Grauvogel, V. Ravi
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Current cancer therapies often fail due to tumor heterogeneity and rapid resistance evolution. A new evolutionary framework, ‘selection for function,’ proposes that tumor progression is driven by group phenotypic composition (GPC) and its interaction with the microenvironment, not by individual cell traits. This perspective opens new therapeutic avenues: targeting the tumor’s functional networks rather than individual cells. Real-time tracking of GPC changes could inform adaptive treatments, delaying progression and resistance. By integrating evolutionary and ecological principles with conventional therapies, this strategy aims to transform cancer from a fatal to a manageable chronic disease. Crucially, it does not necessarily require new drugs but offers a way to repurpose existing therapies to impair a tumor’s evolutionary potential. By steering tumor evolution toward less aggressive states, this approach could improve prognosis and long-term patient survival compared to current methods. We argue that leveraging GPC dynamics represents a critical, yet underexplored, opportunity in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53fd3b887f1a92efe08b517e2a13e55225513a1" target='_blank'>
              Leveraging selection for function in tumor evolution: System-level cancer therapies
              </a>
            </td>
          <td>
            F. Thomas, Jean-Pascal Capp, A. Dujon, A. Marusyk, K. Asselin, M. Campone, Pascal Pujol, Catherine Alix-Panabières, Benjamin Roche, B. Újvári, Robert Gatenby, A. Nedelcu
          </td>
          <td>2025-08-18</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Purpose Microsatellite instability-high colorectal cancer is characterized by hypermutated genomes and high neoantigen loads, yet a significant proportion of patients exhibit resistance to immune checkpoint blockade. This study aims to investigate tumor cell functional heterogeneity and its role in immune evasion. Patients and Methods We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA sequencing from microsatellite instability-high colorectal cancer patients. After quality control, normalization, and clustering, malignant epithelial subpopulations were identified through copy number variation analysis and non-negative matrix factorization. Functional characterization employed gene set enrichment analysis. Spatial transcriptomics clarified immune cell and tumor subpopulation localization, and survival analyses assessed prognostic implications. Results We identified a glycolysis-enriched tumor subpopulation (MP2) that co-localized with immunosuppressive niches marked by Treg accumulation, effector T-cell depletion, and FOLR2+ tumor-associated macrophages. MP2-high tumors were associated with immune checkpoint blockade resistance and poor prognosis. Mechanistically, MP2 cells secreted lactate, promoting Treg differentiation and macrophage polarization toward an immunosuppressive phenotype. Spatial transcriptomics revealed the precise organization of these lactate-rich, immune-excluded niches within tumors. Conclusion These findings establish tumor cell-intrinsic glycolysis as a key driver of immune evasion in microsatellite instability-high colorectal cancer and propose metabolic targeting as a strategy to overcome immune checkpoint blockade resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773cdb1598a2f7975562a7cc2f1fda279ad89676" target='_blank'>
              Glycolysis-Driven Immune Evasion in Microsatellite Instability-High Colorectal Cancer: An Integrated Single-Cell and Spatial Transcriptomics Study
              </a>
            </td>
          <td>
            Chenchen Li, Peicong Cai, Hengda Zeng, Jianxia Li, Huabin Hu, Jianwei Zhang, Zehua Wu, Ge Qin, Yanhong Deng
          </td>
          <td>2025-09-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is highly heterogeneous, with five-year survival rates dropping from $\sim$90% in localized disease to $\sim$15% with distant metastases. Disease progression is shaped not only by tumor-intrinsic alterations but also by the reorganization of the tumor microenvironment (TME). Metabolic, compositional, and spatial changes contribute to this progression, but considered individually they lack context and often fail as therapeutic targets. Understanding their coordination could reveal processes to alter the disease course. Here, we combined multiplexed ion beam imaging (MIBI) with machine learning to profile metabolic, functional and spatial states of 522 colorectal lesions with single-cell resolution. We observed recurrent stage-specific remodeling marked by a lymphoid-to-myeloid shift, stromal-cancer cooperation, and malignant metabolic shifts. Spatial organization of epithelial, stromal, and immune compartments provided stronger stratification of disease stage than tumor-intrinsic changes or bulk immune infiltration alone. To systematically model these coordinated changes, we condensed multimodal features into 10 latent factors of TME organization. These factors tracked disease progression, were conserved across cohorts, and revealed frequent multicellular metabolic niches and distinct, non-exclusive TME trajectories. Our framework MuVIcell exposes the elements that together drive CRC progression by grouping co-occurring changes across cell types and feature classes into coordinated multicellular programs. This creates a rational basis to therapeutically target TME reorganization. Importantly, the framework is scalable and flexible, offering a resource for studying multicellular organization in other solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884e760c0de1e86aa10844e17508931cd572069b" target='_blank'>
              Robust multicellular programs dissect the complex tumor microenvironment and track disease progression in colorectal adenocarcinomas
              </a>
            </td>
          <td>
            Loan Vulliard, Teresa Glauner, Sven Truxa, Miray Cetin, Yu-Le Wu, R. Simon, Laura Behm, J. Tanevski, Julio Saez-Rodriguez, G. Sauter, Felix J. Hartmann
          </td>
          <td>2025-10-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Background Metastatic renal cell carcinoma (mRCC) still lacks tissue-based biomarkers that predict benefit from first-line immune checkpoint inhibitor (ICI) regimens. High-resolution spatial transcriptomics can dissect the tumor microenvironment (TME) and reveal prognostic niches invisible to bulk assays, offering a path toward precision immunotherapy. Methods Formalin-fixed, paraffin-embedded samples from 12 therapy-naïve patients with mRCC treated with ICI-based combinations were profiled with 10x Genomics Visium and NanoString COSMx. Six patients contributed matched metastatic lesions. Cell-type deconvolution used robust cell type decomposition with a public single-cell RNA-seq reference. Clinical outcomes (mean follow-up 16.75 months) were correlated with spatial features. Validation employed Phase 2/3 JAVELIN Renal 101 as well as Checkmate 010/025 transcriptomic data. Results Spatial transcriptomics uncovered pronounced divergences between primary tumors and metastases in cellular composition and microarchitectural organization. Canonical RCC molecular subtypes displayed marked intrapatient and interpatient spatial heterogeneity, underscoring limitations of bulk classifications. Five recurrent immune niches emerged across samples. One niche—rich in macrophages and CD8+ T cells and defined by chemokine signaling—was closely associated with objective response in metastatic lesions yet absent or infrequent in matched primaries. A gene signature reflecting this niche stratified responders within the immuno-oncology (IO) -treated arms of JAVELIN 101 and CheckMate 010/025 trials. On the other hand, signatures derived from angiogenic tumor cell-rich niches defined responders within the comparator arms (tyrosine kinase inhibitor/everolimus) in these trials. Conclusion Metastasis-specific immune niches, rather than primary-tumor characteristics, appear decisive for ICI efficacy in mRCC. Spatial transcriptomic profiling can identify these clinically relevant microenvironments and generate transferable gene signatures, supporting biomarker-driven patient selection and trial design. Integrating spatial TME analysis into future studies may accelerate personalized immunotherapy strategies for mRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bedf8d3b886332325de79302aa00547b1e48f37" target='_blank'>
              Spatial transcriptomic profiling of metastatic renal cell carcinoma identifies chemokine-driven macrophage and CD8+ T-cell interactions predictive of immunotherapy response
              </a>
            </td>
          <td>
            Charis Kalogirou, M. Krebs, Andreas S Kunz, Oliver Hahn, Hubert Kübler, Marcel Schwinger, Arndt Hartmann, Astrid Schmieder, Markus Eckstein
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Repeated oncogenic mutations and polyclonal proliferation are evident in cancers. However, little is known about the polyclonal principles governing the systemic cancerous lineage during immunotherapy. Here, we examine a unique autopsy case of metastatic urothelial carcinoma that exhibits different treatment responses to anti-PD-1 therapy at each tumor site. By performing in-depth analyses of different multiregional bulk tumor masses, we reveal that subsets of subclones acquire potential driver mutations under treatment selection pressure. Spatial transcriptomics analysis reveals that subclones resistant to immunotherapy form distinct immunosuppressive environments consistent with their habitats. Furthermore, different cancer hallmarks are identified in each of the subclones that expand under immunotherapy at single-cell level; for example, one subclone is more proliferative, and another is more stem-cell-like. In summary, this study provides an overall picture of the polyclonal competition and changes in the immune microenvironment that are related to resistance to immunotherapy in patients with malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f357f9c7e57b9e95e8ee48a6378f3d690f3275bd" target='_blank'>
              Clonal diversity shapes the tumour microenvironment leading to distinct immunotherapy responses in metastatic urothelial carcinoma
              </a>
            </td>
          <td>
            Takashi Kamatani, K. Umeda, T. Iwasawa, Fuyuki Miya, K. Matsumoto, S. Mikami, Kensuke Hara, Masayuki Shimoda, Yutaka Suzuki, Jo Nishino, Mamoru Kato, Kazuhiro Kakimi, Nobuyuki Tanaka, M. Oya, T. Tsunoda
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease with diverse outcomes, and intra-tumoral heterogeneity plays a significant role in both diagnosis and treatment. Despite its importance, the spatial distribution of intra-tumoral heterogeneity is not fully elucidated. Spatial transcriptomics has emerged as a promising tool to study the molecular mechanisms behind many diseases. It offers accurate measurements of RNA abundance, providing powerful tools to correlate the morphologies of cellular neighborhoods with localized gene expression patterns. However, the spot-based spatial transcriptomic tools, including the most widely used platform, Visium, do not achieve single-cell resolution readouts, which hinders data interpretability. In this study, we present a computational pathology image analysis pipeline (i.e., computational tissue annotation, CTA) that utilizes machine learning algorithms to accurately map tumor, stroma, and immune compartments within Visium-assayed tumor sections. Using a cohort of 23 breast tumor sections from four patients, we demonstrate that CTA can provide high-resolution annotations on the hematoxylin-and-eosin-stained images alongside the paired sequencing data, support the evaluation of deconvolution methods, deepen insights into intra-tumoral heterogeneity by increasing data analysis resolution, assist with spatially resolved intrinsic subtyping, and enhance the visualization of lymphocyte clones at single-cell resolution. The proposed pipeline provides valuable insights into the complex spatial architecture of breast cancer, contributing to more personalized diagnostics and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5fa100fe1b095f296661cac47143cc93bae15b" target='_blank'>
              Computational pathology annotation enhances the resolution and interpretation of breast cancer spatial transcriptomics data
              </a>
            </td>
          <td>
            Tianyi Li, Qiao Yang, B. Ács, E. Sifakis, Hosein Toosi, Camilla Engblom, K. Thrane, Qirong Lin, Jeff E Mold, Wenwen Sun, Ceren Boyaci, Sanna Steen, Jonas Frisén, Jens Lagergren, J. Lundeberg, Xinsong Chen, Johan Hartman
          </td>
          <td>2025-09-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="The organizational principles of nephronal segments are based on anatomical and physiological attributes that are linked to the homeostatic functions of the kidney. Recent molecular approaches have uncovered layers of deeper signatures and states in tubular cells that arise at various time points on the disease trajectory. Here, we introduce an analytical pipeline of multiplexed spatial protein imaging integrated with RNA expression to characterize proximal tubular subpopulations and neighborhoods in human kidney tissue. We demonstrate that, in reference tissue, a large proportion of S1 proximal tubular epithelial cells expresses thymus antigen 1 (THY1), a mesenchymal stromal and stem cell marker that regulates differentiation. Kidney disease is associated with loss of THY1 and transition toward expression of prominin 1 (PROM1), another stem cell marker recently linked to failed repair. Our data support a model in which the interplay between THY1 and PROM1 expression in proximal tubules associates with their regenerative potential and marks the timeline of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1923d134d598f27cb8990ba3881ccadfe262fa12" target='_blank'>
              Integration of spatial protein imaging and transcriptomics in the human kidney tracks the regenerative potential of proximal tubules
              </a>
            </td>
          <td>
            Mahla Asghari, Angela R. Sabo, Daria Barwinska, Ricardo Melo Ferreira, M. Ferkowicz, William S. Bowen, Ying-Hua Cheng, D. Gisch, Connor J. Gulbronson, Carrie L. Phillips, Katherine J. Kelly, T. Sutton, James C. Williams, Miguel A Vazquez, John O'Toole, Paul M. Palevsky, Sylvia E. Rosas, S. Waikar, Krzysztof Kiryluk, C. Parikh, J. Hodgin, Pinaki Sarder, Ian H. de Boer, Jonathan Himmelfarb, Matthias Kretzler, Sanjay Jain, Michael T. Eadon, S. Winfree, T. El-Achkar, P. Dagher
          </td>
          <td>2025-08-15</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Understanding tumor heterogeneity at the resolution of individualized cell–cell communication networks (CCCNs) remains a major computational challenge in precision oncology. Existing inference methods largely rely on population-level correlation and thus fail to capture patient-specific signaling patterns across diverse cell types. To address this limitation, we developed an integrative computational framework combining the nested hierarchical Dirichlet process (nHDP) model for identifying hierarchically structured gene expression modules, with instance-specific Greedy Fast Causal Inference (iGFCI) for inferring individualized CCCNs (iCCCNs) in colorectal cancer (CRC). Applied to large-scale single-cell RNA-seq data from over 625,000 cells, our model successfully decomposed complex gene expression modules GEMs, potentially representing the cell lineage and cellular signaling states, and uncovered iCCCNs across detailed cell subtypes. We further used TCGA bulk RNA-seq data with survival data to validate the clinical relevance of these individualized gene expression module causal interactions, demonstrating their potential as robust prognostic signatures in CRC. Finally, we used principled causal inference methods to search for ligand-receptor pairs that mediate cell-cell communication. This framework enables mechanistic insights into immune evasion. Our computational method represents a significant advance toward realizing personalized oncology, enabling precise patient stratification and identification of actionable biomarkers for improved therapeutic targeting in cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4e906c5fe00a93c967af7c5aabe8332c82ec34" target='_blank'>
              Inferring Personalized Cell-Cell Communication Networks in Colorectal Cancer with Individualized Causal Discovery
              </a>
            </td>
          <td>
            Aodong Qiu, Binfeng Lu, Gregory F. Cooper, Xinghua Lu, Lujia Chen
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is characterized by exceedingly high rates of metastatic progression, with the liver representing the most common site of distant spread. Here, we established a platform for multisite immune profiling of human PDAC encompassing the tumor, peripheral circulation, and premetastatic liver, to more comprehensively study how various immune subsets might contribute to patient outcomes. Methods Tumor, liver, and blood samples were obtained from patients undergoing resection for non-metastatic PDAC. Derived immune cells underwent paired single-cell RNA and TCR sequencing. Immune composition, cell-type functional profiles, and T cell clonal expansion patterns were evaluated across tissue sites. Results In total, 106,539 immune cells were sequenced, of which 85,748 met criteria for analysis. We identified 32 cell populations, of which seven demonstrated significant enrichment within a particular tissue, highlighting that this workflow possesses the granularity needed for identifying potential future biomarkers. Functional profiling revealed tissue-specific differences in cell phenotypes. This included terminally differentiated exhausted CD8 T cells within the tumor, highly active Tregs within the premetastatic liver and tumor, and M1 versus M2 polarization of liver and tumor macrophage populations, respectively. Within the tumor, expanded Treg clones were uniquely abundant, and while expanded clones could be tracked to the blood and premetastatic liver, many of these mapped back to known viral antigens. Leveraging previously validated gene sets, we show how these can be applied to predict the tumor reactivity of intratumoral T cells using transcriptional signatures. We demonstrated a high degree of concordance between multiple independent signatures and tracked high-priority TCRs within the blood and liver. Conclusion This study demonstrates the feasibility of a platform, which has already been implemented into ongoing clinical protocols, for immune profiling of human PDAC across the sites most relevant to metastatic progression. Future applications of this work can monitor immune populations throughout metastatic progression to build a temporal database of immune phenotypes and track association with clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04146-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9fc3615c15a691adb382d776e1bcc8d09f65d7" target='_blank'>
              A platform for multisite immune profiling of premetastatic pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Elishama N. Kanu, Ashley Fletcher, Jiayin Bao, Ethan S Agritelley, Julia Button, A.M. Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, M. Lidsky, S. Zani, D. Blazer, Peter J Allen, Zhicheng Ji, Frank J. Lowery, Sri Krishna, Nicholas D. Klemen, Daniel P. Nussbaum, Erika J. Crosby
          </td>
          <td>2025-08-23</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9fa61bcf35b5d6c234bc5ce442e119522c741a" target='_blank'>
              Basal cell of origin resolves neuroendocrine-tuft lineage plasticity in cancer.
              </a>
            </td>
          <td>
            A. Ireland, Daniel A Xie, Sarah B Hawgood, Margaret W Barbier, Lisa Y. Zuo, Benjamin E Hanna, Scarlett Lucas-Randolph, Darren R Tyson, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C. Moser, Anish Thomas, Sonam Puri, C. Rudin, Joseph M Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2025-09-17</td>
          <td>Nature</td>
          <td>2</td>
          <td>36</td>
        </tr>

        <tr id="


 Tumor-infiltrating lymphocyte (TIL) therapy has demonstrated efficacy in melanoma and is emerging as a promising modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors like glioblastoma is limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas to identify molecular and spatial features associated with successful TIL expansion.



 We conducted comprehensive profiling of 18 TIL products using spectral flow cytometry and T Cell Receptor sequencing (TCRseq). Additionally, 4 tumors yielding ≥10⁸ viable TILs per 0.5 g tumor within two weeks of IL2 Rapid Expansion Protocol cultures, classified as TIL⁺, were analyzed and compared to 4 non-yielding tumors, classified as TIL⁻. Expanded TILs were subjected to spectral flow cytometry and TCRseq. These TIL⁺ and TIL⁻ tumors underwent single-cell RNA-seq (scRNAseq), in situ Xenium spatial transcriptomics using a customized 480-gene panel, and CODEX spatial proteomics.



 The TIL⁺ tumors yielded 0.5-3.2 × 10⁹ cells with 100-1,000-fold expansion. Flow cytometry showed CD4⁺ and CD8⁺ effector memory phenotypes with minimal exhaustion; Tregs were <10% and γδ T cells were detected. TCRseq revealed high frequency αβ clones and diverse γδ repertoires. Expanded γδ T cells had higher D50 and diversity indices, suggesting a broad, non-dominant repertoire. TIL⁺ tumors exhibited elevated IL7R and ACSS3 expression, enriched Th1/pro-inflammatory gene signatures, and lower Treg-associated transcripts. Spatial analysis showed perivascular immune clusters in TIL⁺ tumors, enriched for CD8⁺ T cells and antigen-presenting cells, with local upregulation of IL7R, MHC II, and CXCL13. These areas were associated with vascular markers (e.g., VCAM1) and T cell retention signatures. In contrast, TIL⁻ tumors displayed immune exclusion, neuronal lineage enrichment, and higher expression of immune suppressive genes like TOX and FERMT1. Tumor-connected TAMs expressing SOX2, NES, and AQP4 were more abundant in TIL⁻ tumors, suggesting a barrier to infiltration or active immune suppression.



 TIL⁺ tumors exhibit distinct molecular and spatial features associated with successful TIL expansion, including IL7R-driven T cell responsiveness, mesenchymal metabolic programs, and structured perivascular immune niches. These findings define biomarkers of therapeutic TIL production and inform strategies to stratify patients for TIL⁻ based therapies in glioblastoma and other immune-excluded tumors. Incorporating these biomarkers into patient selection and workflows could improve clinical trial design, product consistency, and therapeutic response rates.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98b47fb61580f74a05d0a598bf26243ed8f703f3" target='_blank'>
              KS03.4.A SPATIAL MULTI-OMIC PROFILING REVEALS DETERMINANTS OF TIL EXPANDABILITY IN HIGH-GRADE GLIOMAS
              </a>
            </td>
          <td>
            K. Hotchkiss, K. Zhang, A. Corcoran, E. Owens, P. Noldner, C. Railton, K. Van Batavia, Y. Zhou, J. Jepson, K. Singh, R. E. McLendon, K. Batich, A P Patel, K. Ayasoufi, M. Brown, E. Calabrese, J. Xie, J. Conejo-Garcia, B. Shaz, J. Hickey, M. Khasraw
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="High-grade serous carcinoma (HGSC) is the most common ovarian cancer subtype, typically diagnosed at late stages with poor prognosis. Understanding early molecular events driving HGSC progression is crucial for timely detection and development of effective treatment strategies. We performed and integrated spatial cell-type resolved proteomics and paired transcriptomics across 25 women with precursor lesions of the fallopian tube and/or HGSC. Epithelial cell signatures revealed early activation of SUMOylation machinery, increased ATR and Wnt signaling, and enhanced MHC-I antigen presentation along the disease trajectory. The stroma exhibited extracellular matrix remodeling and interferon-mediated inflammation. Serous tubal intraepithelial carcinomas (STICs) in cancer patients contained a pro-coagulative signature and reduced APOA1/2 compared to STICs in individuals without cancer. We functionally established important roles of epithelial-derived TRIP13 and SUMOylation, and cancer-associated fibroblast-derived SULF1 and BGN in HGSC progression. These findings provide unique molecular insights into HGSC pathogenesis and identify potential new therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44488614a89cae7d4f29ffc136362eb7dd32e0db" target='_blank'>
              Integration of cell-type resolved spatial proteomics and transcriptomics reveals novel mechanisms in early ovarian cancer
              </a>
            </td>
          <td>
            Andreas Metousis, Hilary A Kenny, Aasa Shimizu, L. Schweizer, S. Ben-Moshe, A. Bilecz, Rahul Krishnan, Jingwen Zhang, Isabel Alcazar, Lucy Kelliher, Mallika Ravi, Tejas Samantaray, Sabrina Richter, Yan Li, Jiying Wang, Sophia Steigerwald, F. Theis, Florian A. Rosenberger, T. Nordmann, S. Yamada, Ricardo R. Lastra, M. Mann, E. Lengyel
          </td>
          <td>2025-08-28</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="It has become evident from decades of clinical trials that multimodal therapeutic approaches with focus on cell intrinsic and microenvironmental cues are needed to improve understanding and treat the rare, inoperable, and ultimately fatal diffuse intrinsic pontine glioma (DIPG), now categorized as a diffuse midline glioma. In this study we report the development and characterization of an in vitro system utilizing 3D Tumor Tissue Analogs (TTA), designed to replicate the intricate DIPG microenvironment. The innate ability of fluorescently labeled human brain endothelial cells, microglia, and patient-derived DIPG cell lines to self-assemble has been exploited to generate multicellular 3D TTAs that mimic tissue-like microstructures, enabling an in- depth exploration of the spatio-temporal dynamics between neoplastic and stromal cells. The 3D-TTA model recapitulates clinical patterns of DIPG growth, evidenced by resistance to chemotherapy, HDAC and proteasome inhibitors, as well as sensitization to the antibody-activated innate immune microenvironment including complement proteins and surrounding microglia. Multimodal fluorescence imaging platforms integrated with high-throughput omics revealed that alterations in tumor cell motility and growth in the 3D-TTA model compared to tumor cell only spheroids correlated with specific transcriptomic and proteomic changes. STAT3, ITGA5, LGALS1, SOD2, MVP, and CLIC1, associated with microenvironment signaling, DNA replication, and immune regulation, were identified as potential novel targets in the 3D model. The results indicate that the 3D TTA platform developed here represents a powerful tool for preclinical studies, paving the way for identification/validation of tissue specific biomarkers and novel drug targets, thus advancing disease management strategies for DIPG in children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b07b858c6fd1cb8139cba6229ad20ebbdfab9d" target='_blank'>
              Multicellular tumor-stromal interactions recapitulate aspects of therapeutic response and human oncogenic signaling in a 3D disease model for H3K27M-altered DIPG
              </a>
            </td>
          <td>
            Meenakshi Upreti, Astgik Petrosyan, M. Thornton, A. Hovsepyan, G. E. Fernandez, David S. Koos, S. Byrum, Samuel G Mackintosh, Jacob K Al-Husseini, Tania Porras, Joseph Ha, A. Tackett, Miqin Zhang, M. Johal, Anat Erdreich-Epstein, Susan Durham, Mark D Krieger, Ashley S Margol, Brendan H. Grubbs, T. Chambers, S. Asgharzadeh, Rex A. Moats, Peter A Chiarelli
          </td>
          <td>2025-09-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="ABSTRACT The immune landscape of healthy prostate and its alterations during prostate cancer (PCa) progression remain poorly characterized. Using scRNA-sequencing and multiparametric flow-cytometry analysis, we comprehensively characterized immune cells in wild-type and PTEN(i)pe−/− mouse prostates, a model that closely recapitulates human PCa. PCa in PTEN(i)pe−/− is marked by the recruitment of tumor-associated neutrophils (TANs), which represent the dominant immune cell population and resolved into eight distinct states, Trem2+ tumor-associated-macrophages (TAMs), and exhausted CD8+ T cells. Trem2+ TAMs differ from the three main resident macrophage populations in the healthy prostate, exhibiting a strong metabolic and immunosuppressive signature, likely driven by the MIF/HIF1A-signaling axis. This study provides the first detailed characterization of immune cells in the healthy mouse prostate and reveals changes in the immune landscape associated with prostate cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb0c0592aa197777e78a6d1020c20a4157e6fc" target='_blank'>
              Characterization of the immune landscape in healthy mouse prostate and during prostate cancer progression
              </a>
            </td>
          <td>
            Despoina Pervizou, Joanna De Chiara, L. Spinelli, Maia Nestor-Martin, Lionel Chasson, Kateryna Len-Tayon, Darya Yanushko, Frédéric Fiore, Marc Bajénoff, Bernard Malissen, Daniel Metzger, Gilles Laverny, S. Henri
          </td>
          <td>2025-09-26</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract The ability to interrogate changes within the tumor microenvironment (TME) before, during and following therapeutic intervention could yield important understanding of treatment response and causes for disease progression. Yet, the role of investigational tissue analysis faces key challenges in the clinical setting and the value of integrating longitudinal biopsies with emerging multimodal molecular analyses (“Multi-omics”) remains to be defined. In this study, we conducted a multicenter phase 2 clinical trial examining the effect of a novel cytotoxic T-lymphocyte-associated protein 4/programmed death-ligand 1 bispecific antibody in combination with a dual-epitope blocking anti-human epidermal growth factor receptor 2 antibody in treatment-resistant metastatic breast cancer. We performed longitudinal sampling of patient tumor tissues before and following treatment. Single-cell RNA and T cell receptor sequencing from 334,183 cells from site-matched tumors reveals significant temporal shift of various immune cell populations and phenotypes within the TME associated with treatment responses. Conversely, regulatory T cells were activated while effector T cells, natural killer cells, and dendritic cells were significantly depleted in non-responding tumors. Taken together, these results support that longitudinal analysis of TME to generate multiomics data that can lead to rich insight into disease process and to provide clinical value in evaluating treatment responses. Trial registration number NCT04521179.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97d0a38c945abea56c7fcf08591e230c989d914" target='_blank'>
              Longitudinal tissue analysis reveals microenvironmental changes correlate with combined immunotherapy and targeted therapy response in metastatic breast cancer
              </a>
            </td>
          <td>
            , Jien Wang, Hengyu Li, Zhijun Liu, Z. Tian, Xinying Lv, Jianjian Peng, Chuangui Song, Jieqiong Liu
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Heterogeneity within the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment (TME), across tumor locations, and between patients largely contributes to poor treatment response and PDAC’s status as the fourth leading cause of cancer death in the United States. In order to identify suitable targets for innovative therapeutic interventions, it is necessary to understand and mechanistically test TME constituents, their infiltration, and their interaction dynamics in the context of primary and metastatic tumor locations using pre-clinical modeling. Therefore, we designed a novel 46-marker multiplex immunofluorescent staining panel for murine formalin-fixed paraffin-embedded (FFPE) tissues. This panel enables the spatial quantification of immune cell populations and activation/exhaustion states, epithelial-to-mesenchymal phenotypes, cancer-associated fibroblast heterogeneity, hematologic and lymphatic vessel infiltration, neuronal infiltration, and extracellular matrix deposition in the TME. We quantified the composition and spatial infiltration of the TME on matched murine primary and liver tumor samples from tandem orthotopic and hemi-spleen transplants of three distinct primary murine PDAC cell lines derived from the KPC model (KrasG12D; TP53R172H; PDX1-Cre; RosaEYFP). The resulting primary tumors from each cell line showed inter-tumoral heterogeneity as their distinct levels of immune infiltration had varying clustering patterns. Spatial analysis further revealed heterogeneous cancer cell MHC-I expression, immune cell exhaustion, and myeloid and lymphoid populations correlating with cell line. However, all liver nodules were encased by a barrier of immune cells separating the tumor from normal liver parenchyma irrespective of cell line. This observation reflects our recent spatial transcriptomic analysis of patient-derived matched liver and primary tumors, supporting the use of these pre-clinical models for future mechanistic investigations. Altogether, these data highlight the complexity of PDAC TMEs and the site-specific differences between primary and metastatic tumor composition.



 Christina R. Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert Welner, Julienne L. Carstens. High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B110.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335836491342c8b5df5a982f9e0790e87ffd1d16" target='_blank'>
              Abstract B110: High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity
              </a>
            </td>
          <td>
            Christina R Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296f18d57dd6ad5d3ff3f75155cdc24fcd7506bf" target='_blank'>
              Cross-species analysis identifies genotype-driven vulnerabilities in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Angeliki Karatza, Maayan Pour, Jaylen M Powell, Yuan Hao, Dimitris Nasias, Yi Jer Tan, Alfonso Lopez, M. Ciampricotti, Chengwei Peng, Álvaro Quintanal Villalonga, Yeuan Ting Lee, Ruveyda Ayasun, Muhammad Elnaggar, Priyanka Sahu, Igor Dolgalev, Hai Hu, Luis Chiriboga, Cynthia Loomis, Daniel Andrussier, Magdalena A Ploszaj, A. Tsirigos, Peter Meyn, Fred P. Davis, Nicolas Stransky, Gromoslaw A. Smolen, John T. Poirier, Jonathan So, Mark Philips, Harvey I. Pass, Dafna Bar Sagi, Itai Yanai, Kwok-Kin Wong
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly heterogeneous malignancy, characterized by low tumor cellularity, a dense stromal response, and intricate cellular and molecular interactions within the tumor microenvironment (TME). Although bulk omics technologies have enhanced our understanding of the molecular landscape of PDAC, the specific contributions of non-malignant immune and stromal components to tumor progression and therapeutic response remain poorly understood. Methods: We explored genome-wide DNA methylation and transcriptomic data from the Cancer Genome Atlas Pancreatic Adenocarcinoma cohort (TCGA-PAAD) to profile the immune composition of the TME and uncover gene co-expression networks. Bioinformatic analyses included DNA methylation profiling followed by hierarchical deconvolution, epigenetic age estimation, and a weighted gene co-expression network analysis (WGCNA). Results: The unsupervised clustering of methylation profiles identified two major tumor groups, with Group 2 (n = 98) exhibiting higher tumor purity and a greater frequency of KRAS mutations compared to Group 1 (n = 87) (p < 0.0001). The hierarchical deconvolution of DNA methylation data revealed three distinct TME subtypes, termed hypo-inflamed (immune-deserted), myeloid-enriched, and lymphoid-enriched (notably T-cell predominant). These immune clusters were further supported by co-expression modules identified via WGCNA, which were enriched in immune regulatory and signaling pathways. Conclusions: This integrative epigenomic–transcriptomic analysis offers a robust framework for stratifying PDAC patients based on the tumor immune microenvironment (TIME), providing valuable insights for biomarker discovery and the development of precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec3422e79e1fa464a39c9ceade9afd8d95e43c98" target='_blank'>
              Deciphering the Heterogeneity of Pancreatic Cancer: DNA Methylation-Based Cell Type Deconvolution Unveils Distinct Subgroups and Immune Landscapes
              </a>
            </td>
          <td>
            Barbara Mitsuyasu Barbosa, Alexandre Todorovic Fabro, Roberto da Silva Gomes, C. Rainho
          </td>
          <td>2025-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Immunotherapy has been used in the clinical management of TNBC. While BRCA1 mutations are associated with immunotherapy response, the therapeutic outcomes in TNBC patients are not promising. Methods This study integrated spatial, single-cell, and bulk RNA-seq data to explore the role of BRCA1 in reshaping the TNBC microenvironment. Through multi-scale analysis, phenotype changes and potential biomarkers in cancer-associated fibroblasts (CAF) were identified. To validate these findings at the protein level, we employed high-resolution, label-free proteomics sequencing in our in-house cohort, providing critical real-world validation. A predictive system for response to ICIs was constructed through the step-by-step machine learning pipeline. Results Compared to BRCA1 mutant patients, BRCA1 wild-type patients experienced increased T-cell exhaustion and dendritic cell tolerance. We identified a MEG3+ pre-CAF subgroup via pseudo-time analysis. Moreover, ISG15 may serve as an immunoregulatory biomarker, and the proposed predictive model demonstrated potential in forecasting immunotherapy response, although further validation is needed. Conclusions This study highlighted the cellular heterogeneity of TNBC and identified ISG15 as a candidate biomarker potentially associated with treatment response. The ISG15-based predictive system might provide a robust framework for predicting ICI response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125312e82f26c9a44c81f1fdf19e7985af609a32" target='_blank'>
              Resolving tumor microenvironment heterogeneity to forecast immunotherapy response in triple-negative breast cancer through multi-scale analysis
              </a>
            </td>
          <td>
            Shihao Sun, Shuang Chen, Kaiyuan Li, Ge Zhang, Nan Wang, Yijia Xu, Xinxing Wang, Jiangrui Chi, Lin Li, Yi Sun
          </td>
          <td>2025-08-19</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Intratumoral heterogeneity in pancreatic ductal adenocarcinoma (PDAC) is characterized by the coexistence of classical epithelial, basal, and mesenchymal cancer cell states. Targeting cancer cell plasticity—the ability of tumor cells to adapt to extrinsic pressures via non-genetic mechanisms—has emerged as a promising therapeutic strategy. In parallel, the immunosuppressive tumor microenvironment remains a major barrier to effective treatment in PDAC. Yet, whether specific cell states possess distinct functional properties that drive key tumor behaviors—such as phenotypic plasticity or immune evasion—remains poorly understood. Using genetically engineered mouse models driven by oncogenic Kras G12D and loss of Trp53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We find that the KPC model harbors a basal state, which harbors transcriptional signature that aligns with the "basal" signature in human PDAC and is linked to poor prognosis and chemoresistance. The basal state co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. To functionally interrogate the basal state in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the basal state has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the basal state is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the basal state over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we find that the basal state is a key driver of therapeutic resistance: rather than being eliminated by treatment, it persists through phenotypic adaptation in response to both KRAS-targeted therapies and cytotoxic chemotherapy. Co-elimination of the basal state alongside these therapies leads to deeper and more durable responses, underscoring its functional role in tumor persistence under therapy. In parallel, we observe that the basal state plays a central role in enforcing immunosuppression in PDAC tumors. Basal state ablation rapidly reshapes the tumor immune microenvironment, and we observe loss of cancer-associated fibroblasts, polarization of PDAC-resident macrophages toward a pro-inflammatory phenotype, and increased infiltration of T and NK cells. Cytokine profiling of basal-ablated tumors revealed a dramatic reduction in multiple immune-modulatory and immunosuppressive cytokines. Taken together, our data indicate that PDAC harbors a striking dependency on the basal state for both therapeutic resistance and immune evasion, and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.



 Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C. Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Zeynep C. Tarcan, Olca Basturk, Rona Yaeger, James G. Christensen, Yan Yan, Elisa de Stanchina, Dana Pe'er, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B108.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6879ea5062e58b2315581060e5992acf02efdbb" target='_blank'>
              Abstract B108: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Z. Tarcan, O. Basturk, R. Yaeger, James G Christensen, Yan Yan, E. de Stanchina, D. Pe’er, Tuomas Tammela
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is classified into epithelial and mesenchymal subtypes based on transcriptional profiling, and these subtypes are associated with varied patient outcomes. Differences in the tumor microenvironment (TME), particularly in cancer-associated fibroblasts (CAFs), may contribute to this variation. scRNA-seq studies have observed heterogeneous CAF populations in PDAC, although the relationship between tumor subtype and CAF phenotype is not well understood. To investigate this relationship, we utilized epithelial and mesenchymal PDAC cell lines in a xenograft model to examine whether these subtypes influence CAF phenotypes.



 Xenograft models were generated by subcutaneously injecting BALB/c nude mice with PDAC cell lines of either the epithelial subtype (BxPC-3) or the mesenchymal subtype (MIA PaCa-2). Tumors were then harvested and dissociated for scRNA-seq to investigate the TME. Human tumor (graft) and murine stromal (host) cells were computationally separated using the Xenocell tool. Subsequent analyses focused on fibroblasts within the TME, which were subclustered to assess CAF heterogeneity between PDAC subtypes. To characterize transcriptional programs and identify CAF features, we performed high-dimensional weighted gene co-expression network analysis (hdWGCNA) and analyzed differentially expressed genes. Finally, spatial transcriptomics was conducted on tumor sections from BxPC-3 and MIA PaCa-2 xenografts to validate CAF clusters observed in scRNA-seq.



 Distinct CAF clusters were identified in the xenograft tumors, including LRRC15+ CAFs, RGS5+ CAFs, and inflammatory CAFs (iCAFs). LRRC15+ CAFs were predominant in tumors derived from the epithelial subtype (BxPC-3), while RGS5+ CAFs were enriched in tumors from the mesenchymal subtype (MIA PaCa-2). LRRC15+ CAFs exhibited a myofibroblastic CAF (myCAF) phenotype characterized by extracellular matrix production, with high expression of Lrrc15, Postn, and Fbln1. RGS5+ CAFs expressed canonical pericyte markers such as Rgs5 and Cox4i2, along with CAF-associated genes including Acta2. These CAF clusters also showed distinct expression patterns of collagens and secreted factors. In particular, Col11a1, Col12a1, and Wnt5a were upregulated in LRRC15+ CAFs, while Col4a1, Col10a1, and Tgfb1 were upregulated in RGS5+ CAFs. Spatial transcriptomics confirmed that gene modules associated with Lrrc15 and Rgs5, inferred from hdWGCNA, were differentially expressed between Acta2-positive stromal regions of epithelial and mesenchymal PDACs.



 These results suggest that epithelial and mesenchymal PDAC subtypes directly shape distinct CAF populations. Notably, the enrichment of RGS5+ CAFs in mesenchymal tumors may underlie their metastatic potential and poor clinical outcomes.



 Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, Jung Hyun Jo. Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B105.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020656bad62fdcf52b16552c5f7369030a0e1205" target='_blank'>
              Abstract B105: Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model
              </a>
            </td>
          <td>
            Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo-Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, J. Jo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Despite therapeutic advances, metastatic melanoma, and particularly brain metastasis (MBM), remains a lethal burden for patients. Existing single-cell studies offer a more detailed view of melanoma and its microenvironment, which is crucial to improve diagnosis and treatment. Results We here present a computational reanalysis of single-nucleus data comparing 15 MBM and 10 extracranial melanoma metastases (ECM), considering recent best practice recommendations. We used cell type-specific pseudobulking and omit imputation during patient integration to gain complementary insights. Interestingly, our analysis revealed high homogeneity in tumor cell expression profiles within and between MBM and ECM. However, MBM displayed even higher homogeneity but a more flexible energy metabolism, suggesting a specific metastatic adaptation to the putatively more restricted brain microenvironment. While tumor cells were homogeneous, the metastasis microenvironment, especially lymphocytes and related immune-tumor interaction pathways, exhibited greater divergence between MBM and ECM. Overall, this suggests that major differences between MBM and ECM are potentially driven by variations in their microenvironment. Finally, a comparison of single-cell data to previous bulk studies, including their deconvoluted putative cell types, showed significant differences, potentially causing divergent conclusions. Conclusion Our study contributed to refine the understanding of differences between MBM and ECM, suggesting these are potentially more influenced by their local microenvironments. Future research and therapies could possibly focus on the metabolic flexibility of melanoma brain metastases and patient-specific immune pathway alterations. Supplementary Information The online version contains supplementary material available at 10.1186/s13062-025-00691-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e755d874cd5c0ea8aa714bc1a90cc86adaa7c4ba" target='_blank'>
              Single-cell transcriptome analysis suggests cells of the tumor microenvironment as a major discriminator between brain and extracranial melanoma metastases
              </a>
            </td>
          <td>
            Konrad Grützmann, Michael Seifert
          </td>
          <td>2025-09-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Gliosarcoma, a rare and aggressive variant of IDH-wildtype glioblastoma, is characterized by striking intra-tumoral heterogeneity with alternating glial and mesenchymal differentiation. The mesenchymal component is often enriched at recurrence and associated with therapy resistance, suggesting treatment-driven clonal selection. Despite its poor prognosis and distinct histology, gliosarcoma remains underexplored at the molecular level, with no systematic profiling of its cellular or spatial architecture - limiting progress toward targeted therapies.



 Single-nucleus RNA sequencing (snRNA-seq) was performed on eight FFPE gliosarcoma samples, complemented by spatial transcriptomic profiling (10X Visium) in four matched cases. Multimodal integration enabled high-resolution mapping of malignant and non-malignant compartments. To enhance spatial context extraction and to model tumor-microenvironment interactions, advanced computational approaches, including graph neural networks (GNNs) and convolutional neural networks (CNNs), are being developed.



 Integration of snRNA-seq and spatial transcriptomic data revealed a distinct population of stromal-like malignant cells enriched for mesenchymal signatures. Compared to conventional glioblastoma, gliosarcoma samples exhibited a reduced presence of neural progenitor cell (NPC-like) and oligodendrocyte progenitor cell (OPC-like) states, suggesting early dominance or selective expansion of the mesenchymal compartment. Spatial annotation of the tumor microenvironment revealed immune-rich niches populated by disease-associated microglia (DIMs), damage-associated macrophages (DAMs), and macrophages (MACs). Preliminary spatial modeling indicates potential interactions between immune population and mesenchymal-like tumor cells that may support therapy-resistant niches.



 This ongoing study integrates single-nucleus and spatial transcriptomic profiling to characterize mesenchymal transition and resistance in gliosarcoma. Early findings highlight a distinct mesenchymal-like malignant population and associated immune environments that may drive resistance. Further computational modeling aims to uncover key cellular interactions and molecular programs, that could inform novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47d9bc10df09a19173093cb2520a4af4dc46d0d5" target='_blank'>
              P14.25.B DECODING MORPHO-MOLECULAR DIVERSITY IN GLIOSARCOMA AND INSIGHTS INTO RESISTANCE USING SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            D. Savran, N. Schoebe, N. Berghaus, D. Reuss, M. Ratliff, C. Herold-Mende, S. Krieg, W. Wick, A. von Deimling, D. Heiland, F. Sahm, P. Sievers
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of breast cancer is a complex ecosystem, in which cancer-associated fibroblasts (CAFs), as the most abundant stromal cell type, meticulously construct an ecological niche that supports tumor growth through mechanisms including extracellular matrix (ECM) remodeling, secretion of bioactive factors, and interactions with neighboring cells. High-resolution technologies, including single-cell sequencing and spatial transcriptomics, have revealed the high heterogeneity, functional diversity, and spatial distribution within the CAF population. Significant differences exist in the interactions between distinct CAF subpopulations and immune cells. Through complex crosstalk with the immune system, they collaboratively establish an immunosuppressive network, becoming a core driving force for tumor immune escape. This review focuses on the latest research advances in heterogeneous subpopulations of CAFs within the breast cancer microenvironment, delves into how the complex bidirectional crosstalk between different CAF subpopulations and immune cells collaboratively shapes the tumor immune microenvironment (TIME), and summarizes various CAF-based therapeutic strategies for breast cancer, aiming to provide critical theoretical basis and novel therapeutic perspectives for the clinical translation of CAF heterogeneity research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e74899699742a2ae95fa97ca3627d6bdb366035" target='_blank'>
              Crosstalk between heterogeneous cancer-associated fibroblast subpopulations and the immune system in breast cancer: key players and promising therapeutic targets
              </a>
            </td>
          <td>
            Lu Liu, Weijun Wan, Yilin Chang, Luoquan Ao, Yan Xu, Xiang Xu
          </td>
          <td>2025-09-01</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Intra-tumoral heterogeneity (ITH) is common in malignant tumors and arises from dynamic variations across genetic, epigenetic, transcriptomic, proteomic, metabolic, and microenvironmental factors. This complexity drives tumor evolution and treatment resistance, undermining the accuracy of clinical diagnosis, prognosis, and treatment planning. Despite recent advances in multi-omics technologies that enable comprehensive mapping of ITH across molecular layers, major challenges remain in clinical translation. This review outlines the principles and clinical applications of eight major omics modalities in the context of ITH: genomics, single-cell genomics, transcriptomics, epigenomics, proteomics, radiomics, microbiome, and metabolomics. We highlight the unique contributions of each omics platform to tumor profiling and emphasize how their integration enhances biological interpretation, patient stratification, and biomarker discovery. Furthermore, we will focus more extensively on the limitations of applying these approaches to ITH analysis. Instead of providing an exhaustive catalog, this review highlights major controversies, technical hurdles, and conceptual gaps that impede the clinical translation of multi-omics-based ITH analysis, with the aim of addressing ITH-related clinical challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3a8e32658c6f97ddac24dad4088bab77b9c897d" target='_blank'>
              Integration of multi-omics approaches in exploring intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            Mengmeng Dong, Liping Wang, Ning Hu, Yueli Rao, Zhen Wang, Yu Zhang
          </td>
          <td>2025-08-29</td>
          <td>Cancer Cell International</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5047b2e8e1a1022055daa398bbad8bc791fe908c" target='_blank'>
              Microenvironment of Solid Tumors.
              </a>
            </td>
          <td>
            T. Korneenko, Nikolay B. Pestov, M. Shakhparonov, Nickolai A. Barlev
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Single‐cell RNA sequencing (scRNA‐seq) has transformed our understanding of tumours by enabling high‐resolution profiling of their cellular composition. Traditionally perceived as masses of homogeneous cancer cells, tumours are now recognised as complex ecosystems shaped by the tumour microenvironment (TME), which includes diverse immune cells, cancer‐associated fibroblasts and extracellular matrix components. scRNA‐seq has revealed remarkable heterogeneity within the TME, identifying novel or rare immune cell subsets and delineating their dynamic functional states. In particular, it has illuminated intercellular signalling networks and temporal cell‐state transitions that drive tumour progression and immune evasion. Moreover, the integration of scRNA‐seq data with clinical information has highlighted its potential in improving patient stratification, biomarker discovery and therapeutic target identification. Here, we systematically summarise recent advances in applying scRNA‐seq to dissect the TME, discuss the implications of these findings for immunotherapy resistance and precision oncology, and outline future opportunities for integrating scRNA‐seq with emerging technologies to develop more effective and personalised cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7bf7b9a7f8397821baf87f91e3e07f64fb492a" target='_blank'>
              Tumour Single‐Cell Bioinformatics: From Immune Profiling to Molecular Dynamics
              </a>
            </td>
          <td>
            Jane Siu Fan Li, Zoey Zeyuan Ji, Aaron Qi Zhang, C. Ng, Guibin Qiao, Dongmei Zhang, Chunjie Li, Qing Zhang, Ka-Fai To, P. M. Tang
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Summary Kaposi’s sarcoma (KS) is a highly inflammatory, angiogenic tumor driven by Kaposi’s sarcoma-associated herpesvirus (KSHV), yet the origins of tumor cells and mechanisms of progression remain unclear. Here, we present the first spatial single-cell atlas of KS, profiling 256 samples across patch, plaque, and nodular lesions and normal controls. We identify CD34+ progenitor lymphatic endothelial cells (LECs) as the primary targets of KSHV, whose clonal expansion drives tumor growth. KSHV infection induces widespread cellular reprogramming across the tumor microenvironment, including LECs, vascular endothelial cells, fibroblasts, and macrophages, generating hybrid phenotypes that support angiogenesis, inflammation, and immune modulation. KSHV+ macrophages are enriched in tumor-proximal niches, further promoting a proangiogenic, immunosuppressive environment. Spatial analysis reveals evolving tumor-associated niches, with a core-to-periphery gradient correlating with infection, immune modulation, and cellular remodeling. We identify disease progression predictive signatures, offering mechanistic insights into KS pathogenesis and potential new therapeutic strategies by reprogramming the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/835b859927d90c9b0c5454b990e0f2840ed6f867" target='_blank'>
              Spatial Single-Cell Atlas Reveals KSHV-Driven Broad Cellular Reprogramming, Progenitor Expansion, Immune and Vascular Remodeling in Kaposi’s Sarcoma
              </a>
            </td>
          <td>
            Wen Meng, Arun Das, Harsh Sinha, Rana Naous, P. Bracci, Mike McGrath, Yufei Huang, Shou-Jiang Gao
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Abstract Immunotherapy, particularly immune checkpoint inhibitors (ICIs), has revolutionized cancer treatment, yet durable responses are achieved in only a subset of patients. The tumor microenvironment (TME) plays a key role in cancer progression and immune modulation, critically influencing the efficacy of ICIs. Recent advances in single‐cell technologies have enabled high‐resolution profiling of the TME, particularly tumor‐infiltrating immune cells, across diverse cancer types. However, our understanding of how immune cells shape ICI responses and how they are dynamically altered during treatment remains incomplete. In this review, we summarize recent progress in characterizing TME features associated with ICI responsiveness, highlighting key immune cell subsets involved in ICI therapy and emphasizing their phenotypic plasticity and functional adaptability following ICIs. Additionally, we outline the spatial architecture of the TME in terms of its effects on immune cell behavior and interactions, and discuss the critical role of stromal components and the microbiota in modulating the immune landscape and influencing ICI responsiveness. By integrating these insights, we aim to deepen our understanding of the cellular and molecular mechanisms underlying ICI responses, elucidate determinants of therapeutic sensitivity and resistance, and inform the development of more effective immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa3d10e168326d4a4d7bd5875ce8bcf47cc231e7" target='_blank'>
              Insights into the Mechanisms of Immune‐Checkpoint Inhibitors Gained from Spatiotemporal Dynamics of the Tumor Microenvironment
              </a>
            </td>
          <td>
            Yuanyuan Zhang, Zhihua Liu
          </td>
          <td>2025-08-28</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling.
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Diffuse midline glioma (DMG) is an aggressive pediatric brain tumor with no curative treatment. The tumor microenvironment (TME) of DMG, shaped by its unique anatomical location and developmental context, poses significant therapeutic hurdles. While the TME has been characterized by a predominance of tumor-associated microglia and sparse infiltration of blood-derived immune cells, a comprehensive understanding of immune-tumor cell interactions and how to effectively target them with immunotherapies remains challenging.



 To dissect the DMG TME, we employed a multi-omics approach that integrated single-nucleus RNA sequencing, spatial transcriptomics, and high-dimensional imaging on both patient samples and an immunocompetent in utero electroporation murine DMG model. We characterized the spatial and phenotypic architecture of the tumors and validated a novel immune target using syngeneic mouse models and ex vivo co-culture experiments.



 Our analysis revealed pronounced intratumoral heterogeneity, with two spatially distinct patterns of cancer-immune interaction. A mesenchymal-like niche promoted the recruitment of blood-derived immune cells, whereas another pattern exhibited an immunologically cold phenotype, reminiscent of the immune-privileged healthy brain. Spatially resolved analysis of cell-cell communication within the TME identified the VISTA immune checkpoint pathway as a key regulator of these patterns. Targeting of this axis in a syngeneic mouse model led to significant tumor reduction and 100% survival in mice, highlighting its therapeutic potential. Further validation demonstrated that microglia play a critical role in orchestrating immune cell recruitment and anti-tumor activity, and that modulating the VISTA pathway can reshape the immune response.



 Our findings uncover the spatial and cellular complexity of the DMG immune microenvironment and identify the VISTA pathway as a critical modulator of immune suppression. By demonstrating robust therapeutic efficacy upon VISTA blockade, this study lays a strong foundation for developing targeted immunotherapies in DMG and opens new avenues for overcoming immune resistance in pediatric brain tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3d8c86dfadea2de1a37fb282c8875e761a9bc8" target='_blank'>
              JS07.7.A VISTA IS A CRITICAL AND TARGETABLE IMMUNE CHECKPOINT PATHWAY IN DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            R. Collot, C. Ruiz-Moreno, C. Honhoff, A. Wezenaar, H. Ariese, H. Johnson, B. M. Vervoort, J. Bunt, E. Bokobza, H. G. Rebel, B. T. Te Pas, P. Robe, M. Kool, M. Kranendonk, D. V. van Vuurden, E. Hulleman, E. J. Wehrens, A. Zomer, H. Stunnenberg, A. Rios
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal and immunologically "cold" cancer, with a 5-year survival rate of 13%. Despite the success of immunotherapy in other malignancies, its efficacy in PDAC remains limited due to complex immune evasion mechanisms and a pro-tumorigenic immune microenvironment. Several lines of evidence suggest that PDAC progresses from acinar-to-ductal metaplasia (ADM) to pancreatic intraepithelial neoplasia (PanIN) lesions to invasive cancer, accompanied by increasing immunosuppression. We hypothesize that understanding the spatial and temporal transitions of immune cell states during tumor evolution is key to uncovering mechanisms of immune dysfunction and guiding new therapeutic strategies. Using single-cell RNA sequencing (scRNA-seq) and multispectral imaging, we aim to uncover comprehensive, large-scale, and longitudinal changes in the immune microenvironment during PDAC progression. Here, we utilized the KrasG12D/+; Trp53R172H/+; Ptf1a-Cre (KPC) genetically engineered mouse model, and through longitudinal histological analysis of pancreas, we delineated the trajectory of disease progression, with PanIN I-II lesions appearing by 8–9 weeks, PanIN III by 11–12 weeks, and locally invasive PDAC by 16 weeks of age. To study the temporal evolution of immune cell-state transitions, we performed scRNA-seq on CD45+ enriched immune cells isolated from KPC mouse pancreas collected at 4, 8, 12, and 16 weeks using the 10x Genomics Chromium platform. To examine the spatial dynamics of immune cells, we performed multispectral imaging at defined time points using the Lunaphore COMET platform with a panel of over 15 antibodies, enabling simultaneous visualization of several markers on a single tissue section. In our analysis to date, we identified subsets of pro-tumorigenic macrophages emerging during PDAC development, along with a marked increase in myeloid-derived suppressor cells (MDSCs) in late-stage tumors. We also observed dynamic shifts in other immune cell populations evolving alongside tumor progression. Notably, we uncovered distinct subsets of monocytes, macrophages, and T cells exhibiting tumor-promoting potential, progressively enriched during the advanced stages of PDAC. Using multispectral imaging, we generated a comprehensive spatial map of the immune microenvironment across PDAC progression, capturing patterns of immune exclusion, infiltration, and cell–cell neighborhood interactions. Our study highlights the multi-layered immune dysfunction that evolves from early to late stages of PDAC - features that are challenging to capture in human samples. CD45+ enrichment improved the detection of rare and transient immune subsets by mitigating the masking effects caused by abundant cancer and stromal cells. Together, these findings provide an unbiased and detailed view of the evolving immune landscape in PDAC, laying the foundation for identifying microenvironmental and cell-intrinsic factors that either constrain or promote malignant progression, as well as for developing immune-based therapeutic strategies.



 Atul Verma, Doreen Klingler, Linda Bergmayr, Tanvi V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug. Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0f49d549158981070f5d866ac858fc605d13b" target='_blank'>
              Abstract A106: Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression
              </a>
            </td>
          <td>
            Atul Verma, Doreen Klingler, Linda Bergmayr, T. V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Most meningiomas are benign, slow-growing tumors. However, CNS WHO grades 2 and 3 meningiomas display significantly higher aggressiveness and poorer prognosis. Recent investigations into the molecular biology of meningiomas have greatly increased our understanding of these tumors. Furthermore, emergent spatial transcriptomic sequencing methods now provide an unprecedented opportunity to study molecular mechanisms of tumor development within a spatial context and on a single-cell level. This study aims to uncover specific microenvironmental features, cell-cell interactions and transcriptional regulatory mechanisms present in different molecular classes and subclones of meningiomas.



 We employed single-nuclei RNA sequencing and spatial transcriptomic profiling on a matching set of meningiomas (n=22). We developed a method based on non-negative matrix factorization (NMF) to analyze intra-tumor heterogeneity from both single-cell and spatial transcriptional profiles across samples. Trajectory analysis was performed to screen for expression signatures selectively up-regulated around specific morphological features of the tumors. We clustered cells into types of niches based on compositions of their spatial neighborhoods.



 We annotated cell types in the tumor microenvironment (TME), and identified cellular states of meningioma tumor cells in single-cell and spatial transcriptomic data. Tumors of different subclasses display drastic differences in spatial cell state compositions, suggesting distinct modes of microenvironmental organization. We identified conserved patterns of tumor-TME cell state colocalization and interaction. Benign NF2-mutant meningiomas are characterized by extensive infiltration of myeloid cells which trigger a series of immune and stress response pathways in neighboring tumor cells. Among more malignant tumors, a decrease of immune cell populations is coupled with a marked shift in the TME toward tumor-supportive phenotypes. Moreover, we observed reduced tumor-infiltrative capabilities among tumor-associated myeloid cells and the formation of compartmentalized spatial niches of interacting tumor cells. Additionally, various tumor transcriptional signatures related to extracellular matrix remodeling, fibroblastic markers, and meningothelial signatures were mapped to distinct morphological structures across samples.



 Our results reveal that the meningioma TME is heterogenous at the cell state level, with tumor cells and normal cells forming niches that have varying functions in tumor pathophysiology and characterize molecular classes of meningiomas. A series of corresponding RNA markers and spatial signatures were identified, which could potentially be used to increase precision in diagnosis and stratification of meningiomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb7c99c73e50db41072f3f2a1080df073fd8ef43" target='_blank'>
              OS13.6.A A SINGLE-CELL AND SPATIAL ATLAS OF MENINGIOMA HETEROGENEITY ACROSS TUMOR CLASSES
              </a>
            </td>
          <td>
            Y. Tang, R. Banan, S. Maas, D. Calafato, R. Michael, J. Kada Benotmane, R Yan, G. Rukhovich, Z. Ferdinand, V. Venkataramani, A. Suwala, I. Bludau, S. Krieg, W. Wick, A. von Deimling, D. Heiland, M. Gerstung, B. Patel, F. Sahm
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Background and objective Melanoma exhibits profound biological complexity, driven by immune evasion, phenotypic plasticity, and resistance to therapy. While programmed cell death (PCD) shapes tumor–immune interactions, its mechanistic landscape in melanoma remains incompletely defined. This study aims to comprehensively characterize PCD-related signatures and their associations with tumor heterogeneity, prognosis, and immunotherapeutic outcomes. Methods Single-cell RNA sequencing data from melanoma cohorts (cutaneous and acral subtypes) were used to assess PCD activity via AUCell-based scoring across major cell types. Cell-type–specific analyses examined heterogeneity, metabolic dependencies, and pathway correlations. Intercellular communication was analyzed using CellChat. Bulk RNA sequencing data were then integrated to identify PCD-related gene signatures, and machine learning models (LASSO, Ridge, XGBoost) were applied to develop a prognostic model. Immune infiltration, immunogenomic correlations, and immunotherapy responses were further evaluated using ESTIMATE, CIBERSORT, TMB, IPS, and external ICB-treated cohorts. Results Among all cell types, melanoma cells exhibited the highest PCD activation, with disulfidptosis, immunogenic cell death (ICD), and autosis being the most prominent. High PCD activity was linked to advanced clinical stage, lymphatic metastasis, and poor prognosis. Melanoma subpopulations with hyperactivated PCD displayed elevated copy number variation (CNV) burden, enhanced fibroblast/endothelial interactions, and invasive transcriptional profiles. A 15-gene prognostic signature was developed, effectively stratifying survival and immunotherapy response across multiple cohorts. Low-risk tumors demonstrated favorable immune infiltration (CD8+ T cells, M1 macrophages), higher tumor mutational burden (TMB), and greater immunogenicity, while high-risk tumors exhibited immune exclusion, cancer-associated fibroblast (CAF) enrichment, and adverse mutations. Conclusion This study highlights the functional and clinical significance of PCD heterogeneity in melanoma and provides a validated prognostic model for patient stratification and therapeutic decision-making. These findings underscore the potential of targeting PCD dynamics as a novel approach in melanoma management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2d7ead94be26e504d9669468ee3ae02d2938b23" target='_blank'>
              Programmed cell death-driven remodeling of the melanoma microenvironment enables prognostic stratification and therapeutic prediction
              </a>
            </td>
          <td>
            Bo Hu, Shengnan Chai, Xuan Li, Qiang Zhang, Mei Jin, Long Zhang
          </td>
          <td>2025-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Mixed neuroendocrine–nonneuroendocrine neoplasms (MiNEN) are usually highly aggressive tumors characterized by marked histological heterogeneity, most commonly represented by mixed adenocarcinoma and poorly differentiated neuroendocrine carcinoma (NEC). However, beyond morphological observations, the biological basis and implications of this heterogeneity remain incompletely understood. In this study, we combined component-specific next-generation sequencing and spatial transcriptomics to investigate three mixed adenocarcinoma-NEC cases from different anatomical sites (ileocecal, ovarian, gastric), tracing tumor progression from precursor lesions to invasive NEC. Genomic analyses revealed a shared trunk of driver mutations across all tumor compartments, confirming their clonal origin, while also uncovering additional compartment-specific alterations. Spatial transcriptomics, together with gene set enrichment analysis (GSEA), revealed distinct transcriptional profiles aligned with histologically annotated compartments (e.g., adenocarcinoma, NEC, precursor). In NECs, GSEA consistently showed downregulation of immune-related pathways and upregulation of proliferation-associated pathways compared to non-neuroendocrine tumor areas. Moreover, distinct transcriptomic subclusters were identified within morphologically homogeneous NEC regions in two of the three cases. These subclusters exhibited significant differences in immune regulation, proliferation signaling, and cell-cycle control, and were associated with divergent predicted chemotherapy-response signatures, suggesting clinically relevant implications for treatment sensitivity and resistance. In summary, our findings indicate that despite a shared clonal origin, MiNEN develop distinct genetic and transcriptomic features across tumor compartments. The inconsistent presence of transcriptomic subclusters within morphologically similar regions underscores the complexity of intratumoral heterogeneity in these aggressive neoplasms. By connecting morphological and molecular layers of tumor architecture, spatial profiling may aid in translating biological complexity into more targeted clinical strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12022-025-09869-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d940265ab2a62d1a11097a7fc51ef43cc25274" target='_blank'>
              Exploring Intratumoral Heterogeneity in Mixed Neuroendocrine-Nonneuroendocrine Neoplasms with Spatial Transcriptomics: Even More Diverse Than Anticipated
              </a>
            </td>
          <td>
            Annika Weiß, Julia Teply-Szymanski, Maxime Schmitt, S. Foersch, P. Jank, Joscha Griger, Uwe Wagner, Detlef K Bartsch, Carsten Denkert, Moritz Jesinghaus
          </td>
          <td>2025-08-20</td>
          <td>Endocrine Pathology</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="


 The tumour microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC) is comprised of heterogenous malignant epithelium and diverse immune infiltrates within a dense, fibrotic stroma. This multi-compartmental TME includes myeloid and lymphoid lineages, cancer-associated fibroblasts (CAFs), mural and endothelial cells, and tumour cells embedded in a rich extracellular matrix (ECM). Efforts to profile and perturb compartments or cellular phenotypes in isolation have often led to unintended adverse effects on neighbouring cells, compromising drug efficacy. This drives the need for an integrated, spatially-resolved understanding of the architecture and molecular features underlying the organization of the TME. To address this, we leveraged single-cell RNA sequencing (scRNAseq) data from eight distinct datasets to design three custom multiplexed imaging mass cytometry (IMC) panels against tumour, immune, and stromal targets. These were applied to three serial sections of a tissue microarray of 221 PDAC patients. Segmentation and clustering of 7.9 million cells revealed 83 distinct cell types and their functional states, including biophysical responses. Advanced image alignment across serial sections enabled the incorporation of TME-wide features and resolved eight spatially recurrent PDAC microenvironments neighbouring phenotypically distinct epithelial ducts. Two microenvironments, ECM-rich and immune-suppressed, were associated with worse overall survival and frequently co-occurred with a third, stiff matrix, characterized by stroma enriched in phospho-myosin light chain 2 expressing CAFs. In contrast, the immune infiltrated stroma microenvironment was linked to improved clinical outcomes and often co-occurred with CD105+ fibrovascular and immune infiltrated microenvironments. Matched whole-genome sequencing (n = 182/221) revealed that patients dominant in ECM-rich regions harboured 12p11.21 (KRAS) and 8q24.21 (MYC and POU5F1B) amplifications, and 17q22 (RNF43) deletions, and stiff matrix were associated with 13q33.3 (LATS2) deletion. To link spatial niches to proteomic signatures, we performed whole-slide IMC-guided laser capture microdissection and mass spectrometry, and further integrated differentially expressed proteins with transcriptomes from an external scRNAseq cohort (n = 163). ECM-rich and immune suppressed regions showed signatures of mechanically induced immune suppression, including NETosis and innate immune surveillance via pattern recognition receptors. Contrastingly, immune infiltrated microenvironments were enriched for cytotoxic activity and immune trafficking programs, while immune infiltrated stroma displayed signatures associated with oxidative stress responses. These findings define unique spatial ecosystems within the PDAC TME that are linked to specific genomic alterations, functional states and clinical outcomes, highlighting their therapeutic relevance and potential as targets to overcome the challenges of treating PDAC.



 Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Cassandra J. Wong, Brendon Seale, Zhen Yuan Lin, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steven Gallinger, Hartland W. Jackson. Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B115.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e942fb4c97db1a6398873aeae226bf3656a085" target='_blank'>
              Abstract B115: Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J Geuenich, , Barbara Gruenwald, A. Borgida, , Brendon Seale, Zhen Yuan Lin, Edward L. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, H. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Metastatic non-small cell lung cancer (mNSCLC) cells carry heterogeneity, not only among different subtypes but also within a single tumor. Most evidence suggests that mNSCLC exploits specific molecular drivers and mechanisms to maintain physiology, metabolism, and immune evasion during tumorigenesis. Genome-wide association studies also revealed particular mutations in the oncogenic drivers supporting tumor cell proliferation and survival, resulting in aggressive and drug-resistant phenotypes of mNSCLC. While significant progress has been made in understanding mNSCLC at the genetic and molecular levels, a considerable gap remains in understanding the dynamic interplay between intrinsic factors—particularly key tumor-associated cells—and tumor immune microenvironment (TIME) during metastasis. Hence, this review highlights histological and genetic characteristics, emphasizes the clinical relevance of metastasis, and the roles of tumor-associated cells in shaping the immunosuppressive tumor microenvironment (TME) in mNSCLC. Understanding these intricate features and mechanisms is crucial for identifying novel therapeutic targets and improving strategies to combat mNSCLC progression in diagnosed patients. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/931263caf4b3388e1caa012daaf5acb9be29f764" target='_blank'>
              Unraveling the molecular-pathological characteristics and cellular complexity of the tumor immune microenvironment in metastatic non-small cell lung cancer
              </a>
            </td>
          <td>
            S. Bharadwaj, J. M. Mierzwicka, Lucie Vaňková, Petr Malý
          </td>
          <td>2025-09-29</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 PDAC is the most hypovascular solid tumor, imposing harsh constraints on tumor metabolism. Cancer cells cope with poor access to vascular-derived nutrients and O2 by cooperating with abundant stromal cells, receiving alanine, proline, glycosaminoglycans, branched chain ketoacids, lysophosphatidic acid, and unsaturated fatty acids from fibroblasts. However, the extent of metabolic support provided by other cell types in the PDAC TME is unexplored. We reasoned that metabolically cooperative cell types would be spatially positioned near cancer cells in these tumors, and that cooperative spatial arrangements would be enriched under conditions of relative hypovascularity. We first performed Xenium spatial transcriptomics on a cohort of 8 treatment-naive human PDAC resections and generated a spatial atlas of nearly 700,000 cells across a range of expected cell types. Niche analysis uncovered 17 recurrent cellular neighborhoods in the data, including an endothelial-enriched niche. We find that tumor-associated neutrophils form dense aggregates within malignant glands in tumors with low endothelial content, and form in tumor regions which are most distal from vasculature. In a separate cohort of 4 additional PDAC tumors, the expression of neutrophil chemokines CXCL1/2/3/5 was highest in a subpopulation of malignant cells distant from endothelium. These data suggest metabolic stress secondary to poor vascularity drives recruitment of neutrophils via the CXCL1/2/3/5 axis. Indeed, when exposed to Tumor Interstitial Fluid Media (TIFM), cancer cells upregulated the transcription and/or secretion of these cytokines and G-CSF. Since the formation of neutrophil aggregates is driven by cancer cell nutrient derangement, we hypothesized that cancer cells could cooperate metabolically with neutrophils to alleviate nutrient stress driven by poor vascularity. Neutrophil co-culture promoted cancer cell proliferation under TIFM, but not RPMI, while under lethal combination TIFM and serum starvation, co-culture enabled enhanced survival of cancer cells. Lastly, to understand what metabolic processes could be fueled by neutrophil aggregates in pancreatic cancer, we designed a fully custom spatial transcriptomics panel targeting ∼360 metabolic enzymes and profiled 4 additional PDAC cases, performing dispersed whole-transcriptome scFFPEseq on an adjacent tissue section. We integrated data across the spatial and dispersed modalities to enable spatial projection of whole-transcriptome-derived information. Cancer cells located near neutrophil aggregates had significantly enhanced expression of vitamin B6 salvage pathway, suggesting that one critical shared metabolite between neutrophils and cancer cells is the B6 precursor pyridoxine. In total, we have deployed multiple spatial and single cell RNA profiling technologies to uncover a novel node of metabolic crosstalk in pancreatic cancer between neutrophils and cancer cells. Future work is needed to confirm transfer of specific metabolites and the downstream sequelae of their provision for tumor growth.



 Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M Celeste. Simon. Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec70e5707c093ceceed418e17d0f9e9a3b6c60c" target='_blank'>
              Abstract B127: Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer
              </a>
            </td>
          <td>
            Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M. C. Simon
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f3009a3214b83a250848d6ef5905a4fb9e1a3db" target='_blank'>
              Unraveling the single cell spatial landscapes of melanoma brain metastases
              </a>
            </td>
          <td>
            A. Manukyan, K. Peters, H. Radbruch, F. Roßner, A. Lehmann, K. Kleo, T. Conrad, J. Altmueller, I. Plumbon, A. Akalin, K. Juerchott, E. Wyler, M. Landthaler, J. Radke, T. Redmer
          </td>
          <td>2025-09-02</td>
          <td>None</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Cancer-associated fibroblasts (CAFs) are critical mediators of tumor-stroma crosstalk in pancreatic ductal adenocarcinoma (PDAC). While bulk and single-cell transcriptomic studies have uncovered CAF heterogeneity, the spatial organization of CAF subtypes and their physical interactions with tumor epithelial subtypes remain incompletely understood. Here, we integrate bulk, single-cell, and spatial transcriptomic datasets to systematically interrogate the molecular, cell, and spatial interactions between tumor-promoting CAFs (proCAFs), tumor-restraining CAFs (restCAFs), and basal-like versus classical tumor subtypes across multiple biological scales. Ligand-receptor analysis of single-cell RNA-seq data revealed robust signaling between proCAFs and basal-like tumor cells, particularly through integrin-mediated (COL11A1–ITGA11) pathways, suggesting subtype-specific stromal-epithelial crosstalk within distinct tumor microenvironments. Histopathologic validation via immunofluorescence staining confirmed that proCAF regions are preferentially located adjacent to tumor epithelium and are characterized by reactive, myxoid stroma, whereas restCAF regions exhibit a fibrous, hypocellular morphology. To further elucidate spatial relationships, we performed 10x Genomics Visium spatial transcriptomics on seven primary PDAC tumors. Application of the DeCAF classifier to Visium spots demonstrated a distinct radial gradient of proCAF expression, peaking at the tumor-stroma interface and diminishing with distance—indicative of spatial colocalization with basal-like programs. Notably, proCAF gene signatures were specifically enriched in stromal regions adjacent to basal-like tumor zones, while classical tumor regions were spatially associated with heterogeneous CAF compositions. This subtype-specific spatial organization and paracrine signaling may underlie the clinical observation that basal-like tumors are significantly more likely to co-occur with proCAF stroma (64.5%, p = 2.17e–5). Importantly, patients harboring both basal-like and proCAF subtypes exhibited the poorest clinical outcomes (median overall survival: 11.0 months), compared to those with classical/restCAF profiles (median overall survival: 30.4 months, p < 0.001). Collectively, these multidimensional data define spatially organized and functionally interactive tumor-stroma niches in PDAC. Our findings highlight a co-evolutionary relationship between proCAFs and basal-like tumor cells that contributes to an aggressive disease phenotype, and underscore the potential of spatially resolved tumor-stroma profiling for improving patient stratification and guiding therapeutic targeting of the tumor microenvironment.



 Xianlu Laura. Peng, Ian McCabe, Elena Kharitonova, Ryan Zhao, Changfei Luan, Arthi Hariharan, Jaewon Lee, Nancy Kren, Yi Xu, Joseph Kearney, Justin Su, Michelle LaBella, Susan Tsai, Brian Belt, Roheena Panni, David Linehan, Yuliya Pylayeva-Gupta, Alina Iuga, Naim Rashid, Jen Jen Yeh. Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B100.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af6682e5a69c7a44f5aef0626bbff15428b91a16" target='_blank'>
              Abstract B100: Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications
              </a>
            </td>
          <td>
            X. Peng, Ian C McCabe, Elena V Kharitonova, Ryan Zhao, C. Luan, Arthi Hariharan, Jaewon J Lee, Nancy P. Kren, Yi Xu, Joseph F Kearney, Justin Su, M. LaBella, Susan Tsai, Brian Belt, R. Panni, David Linehan, Y. Pylayeva-Gupta, Alina C Iuga, Naim U Rashid, J. J. Yeh
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Adenoid cystic carcinoma of the salivary gland (SGACC) is a highly aggressive malignancy characterized by poor patient survival outcomes. While several studies have analyzed the transcriptome of the salivary gland at the bulk and single-cell level, no spatial transcriptomic analyses of this tissue have been published. Most of the existing publications on SGACC have predominantly relied on bulk and single cell RNA sequencing approaches, which do not resolve the spatially localized transcriptional heterogeneity nor have the resolution for defining molecular markers within tumor subpopulations. SGACC is clinically notable for the presence of multiple tumor clones, distinct spatial phenotypes, and its indolent yet invasive nature coupled with a high propensity for distant metastasis. These features may reflect co-expression of tumor-associated markers across diverse cellular niches, and a resultant biological complexity which causes standard treatment such as surgical resection, radiation therapy, and chemotherapy to be largely ineffective in significantly improving long-term survival, and highlights the need for more precise, targeted therapeutic strategies. Herein, we analyzed single cell (n = 4) and high-resolution spatial transcriptomics samples (n = 5) to characterize cancer cell populations in MYB- and non-MYB-expressing cell states, delineated gene expression signatures, and identified critical molecular interactions specific to SGACC. We used Visum HD to obtain spatial transcriptomics data at 2μm squared high resolution. This allowed a multi-omics approach comprising single cell and spatial transcriptomic methods to enable the discovery of novel transcriptional signatures and microenvironmental features not captured by conventional methods. Spatial mapping revealed marked cellular heterogeneity and demonstrated how tissue environments influence cellular transcriptomics. To tumor heterogeneity, we focused on tumorigenic cell populations, profiled plasma and T cell enrichment within the tumor microenvironment and identified key pathways and transcriptional drivers including the MYB-NFIB fusion underlying the tumor cluster formation. Our findings indicate an upregulation of genes involved in extracellular matrix remodeling, autophagy, and reactive stromal cell populations. We further found evidence of partial epithelial-mesenchymal transition (P-EMT) programming within MYB-expressing tumor clusters. Pathway analysis revealed that mutations in the spatial query sample prominently affect the PI3K-AKT and IL-17 signaling pathways, together with a downregulation of canonical Wnt signaling in some regions of the tissue architecture adjacent to immune cells. Collectively, these results underscore the complex regulatory landscape of SGACC and offer insights into its cellular dynamics and possible therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d936540aedb35ad533bb62a60043e3d08a9252f" target='_blank'>
              Gene-Expression Programs in Salivary Gland Adenoid Cystic Carcinoma Analyzed Using Single-Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ifeoma Ebinumoliseh, Gopikrishnan Bijukumar, Kendall Hoff, K. Brayer, Elaine L. Bearer, Scott A. Ness, Jeremy S. Edwards
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in shaping immune responses and therapeutic outcomes in lung cancer, yet the diversity and functional specialization of tumor-associated macrophages (TAMs) remain poorly resolved. Here, we present a refined classification of TAM subtypes across large cohorts of cancer datasets using integrative analysis of single-cell RNA sequencing, spatial transcriptomics, and clinical datasets from lung adenocarcinoma and lung squamous cell carcinoma. By combining cell-based gene scoring with hierarchical classification, we defined 7 macrophage subtypes-each with distinct transcriptional programs and abundances. Notably, lipid-associated TAMs expand with disease progression and exhibit immunosuppressive and protumorigenic features, whereas tissue-resident macrophages decline. Spatial and survival analyses reveal that an increased lipid-associated to tissue-resident TAM ratio correlates with advanced disease and poor prognosis. Given that spatial transcriptomic assays rely on deconvolution techniques to infer cell type compositions, accurate gene expression signatures are essential, especially for fine-grained subpopulations of TAMs. Our refined subtype-specific signatures address this bottleneck and enhance the resolution of spatial mapping efforts. These findings offer new insights into macrophage heterogeneity and highlight lipid-associated TAMs as potential biomarkers and therapeutic targets in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e49c52a60b0a9da9a722e72cb8391f58dc458b" target='_blank'>
              Transcriptional states of lung cancer microenvironment reveal macrophage subtype dynamics linked to disease progression.
              </a>
            </td>
          <td>
            Duygu Keremitçi, Özlem Tuna, Aissa Houdjedj, Hilal Kazan, Yasin Kaymaz
          </td>
          <td>2025-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous malignancy, with its tumor immune microenvironment (TIME) playing a crucial role in tumor progression, immune evasion, and treatment response. Despite insights from single-cell technologies, challenges such as small samples and inconsistent annotations have limited a comprehensive understanding. Thus, a large-scale, high-resolution single-cell atlas is needed to fully explore the heterogeneity of CRC TIME and guide personalized treatment. Methods We analyzed single-cell transcriptomic data from 100 CRC samples (371,223 cells). Immune, stromal, and epithelial cells in the TIME were clustered, annotated, and functionally analyzed. Transcription factor networks were inferred using SCENIC, differentiation states assessed with CytoTRACE, and cell–cell communication analyzed via CellPhoneDB. TIME subtypes were constructed from bulk RNA-seq data, followed by survival analysis. SCRN1 was further validated through in vitro experiments. Results This study presents a comprehensive single-cell CRC atlas, revealing significant heterogeneity in T cells, B cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells within the TIME. We defined two immune ecological subtypes: subtype 1 (enriched in metabolic and motility pathways with poor prognosis) and subtype 2 (enriched in immune response pathways, with better prognosis and greater immunotherapy potential). High SCRN1 expression in malignant epithelial cells correlated with poor prognosis and metastasis. In vitro experiments confirmed the role of SCRN1 in promoting CRC cell proliferation and migration. Conclusions This study provides a comprehensive characterization of the CRC TIME, links TIME subtypes to prognosis, and identifies SCRN1 as a potential therapeutic target, offering insights for CRC precision subtyping and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04173-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c5be539fad2c0073ae6e62159eaef1e83a1ef" target='_blank'>
              The single-cell atlas of colorectal cancer reveals cellular heterogeneity and key molecular features of the tumor immune microenvironment
              </a>
            </td>
          <td>
            JunHu Li, ZhiQiang Liu, BoYu Zhang, Liuping You, Yuenan Huang
          </td>
          <td>2025-10-10</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background Renal Cell Carcinoma (RCC) has been characterized as being amongst the most immune infiltrated solid tumors with a highly heterogenous immune landscape. Within spatially organized cellular networks (CNs) of the tumor immune microenvironment (TIME), key immune cell-cell interactions (CCIs) impact immune cell function and organization ultimately impacting the patient’s overall response. Multiple studies have observed an association of tertiary lymphoid structures, a commonly observed spatial CN, with positive clinical outcomes in RCC, however additional CNs and the CCIs that control these networks need to be identified to better harness and potentially reprogram immune responses to improve patient outcomes. The recent explosion of interest in the heterogeneity of the immune landscape in RCC has led to numerous publications using the latest technologies in spatial transcriptomics, proteomics, and metabolomics. However, many of these studies use this data in isolation and therefore, may be hindered by the technological biases inherent in each method. Here, we have developed a novel approach to integrate spatial and single cell multi-omic data harnessing the strengths of each technology to better interrogate CNs that exist in the RCC TIME. Methods Fresh surgical samples were procured at the University Health Network (Toronto, Canada) through the REnal cancer MicroEnvironment DiscoverY (REMEDY) study. Bulk RNA sequencing (RNA-seq), single cell RNA sequencing (scRNA-seq), single cell suspension mass cytometry (SMC), whole transcriptome digital spatial profiling (DSP), and imaging mass cytometry (IMC) was performed on spatially concordant tumor regions across 54 patients. scRNA-seq enabled the identification of immune, stromal, and malignant high-resolution cell states, which informed a tailored antibody panel design for SMC and IMC and served as a reference framework for harmonized cell-type annotation across modalities. This enabled the integration of our transcriptomic and proteomic data to delineate RCC-specific CNs enriched for defined CCIs across unique biological pathways. Results Using this integrative approach, we identified seven high-resolution patient immunophenotypes. To evaluate their prognostic and predictive relevance, we derived representative gene signatures using a linear mixed model (Flash-MM) to interrogate publicly available bulk RNA-seq datasets, including TCGA, JAVELIN, and IMmotion151. This analysis revealed immunophenotype-specific associations with survival following surgery or systemic therapy in univariate models. To explore potential biological mechanisms associated with these divergent clinical outcomes, we incorporated spatial information into traditional CCI analyses and performed pathway analyses to define functional relationships. This revealed that patient subtypes with high lymphoid infiltration exhibit greater spatial heterogeneity, potentially reflecting the coexistence of multiple activated immune pathways. In contrast, patient subtypes with low immune infiltration were enriched in more developmental signaling pathways. In addition to our biological observations, we were also able to assess the technological differences between patient matched samples and compare the ability of each technology to capture inter-patient and intra-patient heterogeneity. Conclusions Collectively, this work identifies clinically distinct subgroups defined by CNs and details the interpatient cellular heterogeneity that exists in RCC, providing the foundation for future personalized therapeutic interventions against this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e76f0bf068f59ffeffc899012c151c0a8ca8df5b" target='_blank'>
              37Integrative multi-omic characterization of the immune landscape in renal cell carcinoma
              </a>
            </td>
          <td>
            Jennifer Pfeil, Shirley Hui, Daniel Stueckmann, Xiaoyu Zhang, Lisa Martin, Jennifer L. Gorman, Eddy Chen, Somi Afiuni, Jalna Meens, M. Komisarenko, Stéphane Chevrier, Sujana Sivapatham, Julia Szusz, Zhihui Liu, Susan Prendiville, Laurie Ailles, Philip Jonsson, Fred Davis, Cristina Penaranda, Ryan Newton, N. Stransky, Piotr Bielecki, Gromek Smolen, Masoom Haider, Bernd Bodenmiller, Sarah Crome, Gary Bader, Anthony Finelli, H. Jackson, Keith A. Lawson
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="SUMMARY Ovarian high-grade serous carcinoma (HGSC) is characterized by extensive intra-peritoneal dissemination and tumor heterogeneity. In the metastatic cascade, tumors utilize several transcriptional programs to translocate and survive in distant tissues. Here, we analyzed multi-modal, real-world data from 350 tumor samples across 160 patients with HGSC to identify transcriptional programs that drive intra-peritoneal metastasis and heterogeneity. We identified nine transcriptional programs, including those regulating epithelial-mesenchymal transition and immune modulation and cytoskeletal reorganization, which shape distinct metastatic trajectories to solid and ascitic environments and are associated to treatment response. Our results reveal pronounced intra-patient transcriptional heterogeneity, which in some cases surpassed inter-patient heterogeneity, highlighting the importance of multi-site sampling for accurate prognostication and combinatorial treatments in HGSC. Our extensive characterization offers novel insights into intra-peritoneal metastasis with significant prognostic implications, reveals histomorphological biomarkers for patient stratification and paves the way for innovative therapeutic strategies aimed at impairing cancer cell adaptability and limiting metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32c6dc18825437bb12a5a64bda09f91c2c6d329" target='_blank'>
              Multi-modal characterization of transcriptional programs that drive metastatic cascades to solid sites and ascites in ovarian cancer
              </a>
            </td>
          <td>
            Kaiyang Zhang, Essi Kahelin, G. Marchi, Oskari Lehtonen, S. Salloum, K. Lavikka, Yilin Li, Felix Dietlein, A. Lahtinen, J. Oikkonen, S. Hietanen, J. Hynninen, Antti Häkkinen, A. Virtanen, S. Hautaniemi
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Ovarian cancer represents a typically immune “cold” tumor, where obvious immunosuppression, spatial T-cell exclusion, and cellular dysfunction collectively limit immunotherapy effectiveness. Especially in high-grade serous ovarian carcinoma (HGSOC), the immune low-response state is driven by complex interactions among tumor-associated macrophages (TAMs), suppressive stromal networks, and the T-cell compartment (regulatory T cells, Tregs, and exhausted effector T cells). Emerging multi-omics technologies—particularly single-cell RNA sequencing and spatial transcriptomics—have showed the heterogeneity and spatial immune organization underlying this suppressed state. Here, we integrate these datasets to describe TAM phenotypes and spatial niches, T-cell exhaustion, Tregs accumulation, NK-cell dysfunction, and stromal barriers that enforce exclusion. We then derive phenotype-guided combination strategies to remodel the tumor microenvironment and improve responsiveness to immune checkpoint blockade. This synthesis provides a concise, multi-dimensional framework for precision immunotherapy and for overcoming resistance in immune-low ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880f6c405f773eb38fd078c68256074438417fa2" target='_blank'>
              Decoding immune low-response states in ovarian cancer: insights from single-cell and spatial transcriptomics for precision immunotherapy
              </a>
            </td>
          <td>
            Yalan Yan, Jiaan Lu, Huanyu Luo, Zizhang Wang, Ke Xu, Lexin Wang, Qin Wang
          </td>
          <td>2025-09-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer characterized by a lack of hormone receptors, making it challenging to treat. Methods We generated a comprehensive spatial cell atlas of TNBC using a multi-omics integration approach that combined single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics. This integration allowed us to characterize the spatial microenvironment and map the cell-type-specific distributions in TNBC tissues. Results Our analysis revealed significant heterogeneity in cell types and spatial distribution, with normal regions enriched in insulin resistance functions, whereas cancerous regions displayed diverse cell populations, including immune cells, cancer-associated fibroblasts (CAFs), and mesenchymal cells. By constructing transcription factor (TF) regulatory networks, we identified TFF3, RARG, GRHL1, RORC, and KLF5 as critical regulators of epithelial cells, whereas EMX2, TWIST1, TWIST2, NFATC4, and HOXC6 were found to play essential roles in mesenchymal cells. Immunohistochemical validation supported the involvement of these TFs in TNBC. Further analysis of receptor-ligand interactions highlighted the roles of KNG1_BDKRB2 and NRG1_ERBB4 signaling in promoting tumor aggression, suggesting potential therapeutic targets. GO enrichment analysis revealed overlapping pathways between epithelial and mesenchymal cells, focusing on migration, signaling, and development, indicating that the shared regulatory mechanisms contribute to cancer progression. Conclusion Our findings provide new insights into the TNBC microenvironment, emphasizing the complex spatial interactions between different cell types and highlighting key regulatory pathways that could be targeted for future therapeutic interventions. This spatial cell atlas lays the foundation for further exploration of tumor microenvironment dynamics and precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8697e1d4f457ce1a990f5f037a0dd30fbd923ee5" target='_blank'>
              Multi-omics integration uncovers key transcriptional regulators in triple-negative breast cancer spatial heterogeneity
              </a>
            </td>
          <td>
            Ning Zhang, Ye Zhang, Rui-Fei Yang, Hai-Qing Chu, Abdur Rehman, Lu-Yu Yang, Ya-Yu Li, Xin Zhou, Shou-Ping Gong, Hui-Ling Cao
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose The dysregulated immune microenvironment represents a key pathogenic driver in psoriatic lesions. However, the intricate cellular and molecular interactions underlying psoriasis remain incompletely elucidated. Therefore, we aim to employ integrated multi-omics approaches to characterize the immune microenvironment and pathogenic niche in psoriasis, thereby elucidating the cellular and molecular mechanisms of disease pathogenesis. Methods Integrated Single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and bulk RNA sequencing (RNA-seq) data to explore the heterogeneity of stromal cells and immune cells in psoriatic lesions and the complex spatial niches formed between them. Enrichment analysis, intercellular communication analysis, and spatial co-localization analysis were used to investigate the transcriptional changes and distribution characteristics of each cell type in the lesions of psoriasis patients. Results Using scRNA-seq, we identified a novel CD4+ tissue-resident memory T cell (TRM) subset that is exclusively present in lesional skin of psoriasis patients but absent in healthy skin. These cells exhibit elevated expression of genes including IL17RA, IL22, PD1 (PDCD1), CXCR6, ITGAE, CD69, TNFRSF9, TNFRSF4, IL7R, CD4, and STAT3. Additionally, we discovered a novel microvascular endothelial cell subset, designated Venous endo2, which highly expresses CD93, ACKR1, ICAM1, VCAM1, IL15, SELE, and SELP, while also overlapping with high endothelial venule (HEV)-associated transcriptional signatures. Integrated analysis of scRNA-seq and spatial transcriptomics further revealed strong spatial co-localization of Venous endo2 with fibroblast activation protein-positive fibroblasts (FAP+ Fbs), T cells, and antigen-presenting cells (APCs) in Psoriasis lesions—a pattern not observed in healthy control skin. Conclusion Through integrated multi-omics analysis, we identified a potential pathogenic niche in psoriasis patients, composed of Venous endo2, FAP+ Fbs, T cells, and APCs. This structure resembles tertiary lymphoid structures (TLS), suggesting a functional parallel in disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d8eeb3b234a229f3a9f78e22b42dca0befb2a94" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals a Pathological Niche Formed by FAP+ Fibroblasts, Immune, and Endothelial Cells in Psoriatic Lesions
              </a>
            </td>
          <td>
            Wuwei Zhuang, Qi Zhang, Qingtao Kong, Yun Hui, Jie Shen, Chen Zhang, Hong Sang, Qiao Ye
          </td>
          <td>2025-09-01</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0c307663da680bc20dfe3893c95285de2664ca" target='_blank'>
              Spatiotemporal analyses of the pan-cancer single-cell landscape reveal widespread profibrotic ecotypes associated with tumor immunity.
              </a>
            </td>
          <td>
            Ya Han, Lele Zhang, Dongqing Sun, Guang-xu Cao, Yuting Wang, Jiali Yue, Junjie Hu, Zhonghua Dong, Fang Li, Taiwen Li, Peng Zhang, Qiu Wu, Chenfei Wang
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Tumor heterogeneity, a hallmark of cancer, frequently leads to treatment failure and relapse. However, the intricate communication between various cell types within the tumor microenvironment and their roles in tumor progression in vivo remain poorly understood. Here we establish a novel tumor heterogeneity model in the Drosophila larval eye disc epithelium and dissect the in vivo mechanisms by combining sophisticated genetics with single-cell RNA sequencing. We found that mutation of the tricellular junction protein M6 in cells surrounding RasV12 benign tumors promotes their malignant transformation. Mechanistically, early RasV12//M6−/− tumors secrete Pvf1, which activates the Pvr receptor on hemocytes, facilitating their recruitment to the tumor site. These tumor-associated hemocytes secrete the Spätzle (Spz) ligand to activate the Toll receptor within the RasV12 tumors. This enhanced activation of the Toll pathway synergizes with RasV12 to promote malignant transformation through the JNK-Hippo signaling cascade. In summary, our study elucidates the complex interplay between genetically distinct oncogenic cells and between tumors and hemocytes, highlighting how hemocytes exploit the ancient innate immune system to coordinate tumor heterogeneity and drive tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d320162041d03e1b05ee82602e945a36e317508" target='_blank'>
              Hemocytes facilitate interclonal cooperation-induced tumor malignancy by hijacking the innate immune system in Drosophila
              </a>
            </td>
          <td>
            Sihua Zhao, Yifan Guo, Xiaoyu Kuang, Xiaoqin Li, Chenxi Wu, Peng Lin, Qi Xie, Zongzhao Zhai, D. Kong, Xianjue Ma
          </td>
          <td>2025-08-22</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02937-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic cancer remains a lethal disease despite advances in surgery and systemic treatment in the last two decades, underscoring the urgent need to better understand its biological underpinnings. Despite remarkable advances in the molecular characterization of pancreatic ductal adenocarcinoma (PDAC), clinically actionable biomarkers remain scarce, and current treatment remains empiric. Transcriptomic subtypes such as “classical” and “basal-like” offer some prognostic value, but their ability to guide real-time treatment decisions is limited. In this review, we explore the limitations of current biomarker strategies, in particular subtype-based classifications, and argue for a functional reframing of biomarker development in PDAC, centered on patient-derived organoids (PDOs). We explore four key domains in which PDOs deepen our understanding of therapeutic response and resistance, namely, drug response phenotyping, modeling chemoresistance, incorporating tumor microenvironmental complexity through co-culture systems, and more functional profiling through proteomic and metabolomic approaches. Together, these applications move PDOs beyond static avatars of the tumor to dynamic platforms capable of capturing clinically relevant biology. As functional precision medicine gains traction, PDOs may offer a path to more tailored, responsive treatment strategies in a cancer where new options are urgently needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243db33c2c4b51c3c745a2973296c1c4099759e6" target='_blank'>
              Functional Precision in Pancreatic Cancer: Redefining Biomarkers with Patient-Derived Organoids
              </a>
            </td>
          <td>
            C.A. Chew, Cheng Mun Wun, Yi Fang Lee, C.E. Chee, Khek Yu Ho, Glenn Kunnath Bonney
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor evolution is driven by various mutational processes, ranging from single-nucleotide variants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alterations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational framework for harmonization and joint analysis of different variant types in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by independent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e007da3f913b3e2ad1860533d9c4d9c4f0893dfe" target='_blank'>
              Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories
              </a>
            </td>
          <td>
            Yuelin Liu, Anton Goretsky, A. Keskus, S. Malikić, Tanveer Ahmad, Farid E. Michael Gertz, Rashidi Mehrabadi, Michael Kelly, Maria Hernandez, Charlie Seibert, J. M. Caravaca, Kayla Kline, Yongmei Zhao, Ying Wu, Biraj Shrestha, Bao Tran, Arindam Ghosh, Xiwen Cui, Antonella Sassano, Lakshay Malik, Breeana Baker, Cornelis Blauwendraat, Kimberley J. Billingsley, Eva Pérez-Guijarro, Glenn Merlino, Erin K. Molloy, S. C. Sahinalp, Chi-Ping Day, M. Kolmogorov, Contents
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Uterine leiomyosarcoma (ULSA) is a highly aggressive gynecologic malignancy characterized by early metastasis, profound immunosuppression, and resistance to conventional therapies, including immune checkpoint blockade (ICB). The intricate tumor microenvironment (TME) and cellular heterogeneity driving its progression and therapy resistance remain poorly defined. Methods We performed single-cell RNA sequencing (scRNA-seq) on metastatic lesions (pelvic cavity, rectum, peritoneum, bladder) from a treatment-naïve ULSA patient and compared them to normal uterine myometrium, MMM (n=5). Integrated analyses included cellular composition mapping, copy number variation (CNV) assessment, pseudotemporal trajectory reconstruction, cell-cell communication inference, functional enrichment, and validation via multiplex immunofluorescence (mpIF). Survival correlations were assessed using the TCGA-SARC cohort. Results In this study, the main finding is that the tumor microenvironment (TME) has a strong immunosuppressive effect. Firstly, its characteristic is exhausted CD8+T cells. This study found that as time progresses, the initial cell markers (CCR7, MAL) gradually disappear, while the exhaustion markers (LAG3, HAVCR2, TIGIT) are enriched. This is associated with poor prognosis. Secondly, the M2-polarized macrophages are mainly composed of M2-like tumor-associated macrophages (TAMs) with tumor-promoting characteristics (CD163, FTH1, FTL, TIMP1), and there is a polarization from M1 to M2. Finally, the immature, tumor-promoting N2 neutrophils (CD15+EDARADD+) enriched in the metastatic foci are associated with poor prognosis. The cell communication involves the interaction of MIF-(CD74+CD44) between T/B cells, as well as the role of the CXCL8 signaling axis in promoting angiogenesis, TAM polarization, and immunosuppression. Conclusion For the first time, a comprehensive single-cell map of ULSA was constructed, depicting a metastasis-susceptible cell subset (U11-EDARADD) and an extremely immunosuppressed tumor microenvironment dominated by depleted CD8+T cells, M2 macrophages and N2 neutrophils. These features shed light on the underlying mechanisms of chemotherapy resistance and immunotherapy failure. The biomarkers identified here (EDARADD, CLDN10, TMIGD2) as well as the dysregulated pathways (TGF-β, angiogenesis, MIF signaling) provide possible targets for future development of combined immunotherapy strategies against this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15b01f016db1d6adc66dea5efac6b72384978f8f" target='_blank'>
              Single-cell profiling delineates the tumor microenvironment and immunological networks in patient-derived uterine leiomyosarcoma
              </a>
            </td>
          <td>
            Yi Guo, Dongsheng Shen, Yuhang Xiao, Chenghao Wu, Meiyi Chen, Lina Yang, Huaifang Li, Xiaowen Tong, Rujun Chen, Fang Li
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality worldwide. However, the spatial and temporal dynamics underlying its development remain poorly characterized. This study employs spatial transcriptomics (ST) to investigate the progression of intestinal tumors in APC Min/+ mice across multiple time points. We identified distinct transcriptional profiles between tumor and normal tissues, resolving six major cell types through integrated dimensionality reduction and pathological annotation. Pseudo-time trajectory analysis revealed increased expression of MMP11 and MYL9 in later stages of tumor progression. Analysis of human CRC cohorts from the TCGA database further confirmed that high expression of these genes is associated with advanced clinical stages and promotes tumor proliferation and invasion. Temporal gene expression dynamics indicated enrichment of cancer-related pathways concurrent with suppression of lipid and amino acid metabolism. Notably, genes in the DEFA family were significantly upregulated in normal tissues compared to tumor tissues. Functional validation showed that DEFA3 inhibits colon cancer cell migration and proliferation in vitro. These demonstrate the value of ST in resolving spatiotemporal heterogeneity in CRC and identify both MMP11/MYL9 and DEFA3 as potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a441608f72da7375d3ddfdc034535ffa428b9ffc" target='_blank'>
              Investigating the Development of Colorectal Cancer Based on Spatial Transcriptomics
              </a>
            </td>
          <td>
            Zhaoyao Qi, Guoqing Gu, Huanwei Huang, Beile Lyu, Yibo Liu, Wei Wang, Xu Zha, Xicheng Liu
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is characterized by profound intratumoral heterogeneity and cellular plasticity, which drive therapeutic resistance. While molecular subtyping into classical and basal-like states has provided a framework, the biology of cells transitioning between these phenotypes remains poorly understood. This study aimed to characterize and validate a transitional, co-expressor cell state at the single-cell, spatial, and proteomic levels to elucidate mechanisms of subtype plasticity. We employed an integrative, multi-platform approach combining single-cell RNA sequencing (scRNA-seq), multiplex immunohistochemistry (mIHC), and laser microdissection with mass spectrometry-based proteomics (LMD-MS) to investigate tumor cells co-expressing the classical marker GATA6 and the basal-like marker KRT17. scRNA-seq analysis revealed that a significant population of tumor cells co-expresses GATA6 and KRT17, existing along a transcriptional continuum between the classical and basal-like poles. Pseudotime trajectory analysis identified a path from the GATA6+ classical state towards the KRT17+/GATA6+ transitional state, governed by the upregulation of pathways including MYC signaling and epithelial-mesenchymal transition (EMT). To validate these findings spatially, mIHC confirmed the presence of KRT17+/GATA6+ double-positive cells, frequently located at the interface between well-differentiated, GATA6-rich glandular structures and poorly differentiated, KRT17-rich infiltrative fronts. Furthermore, LMD-MS was used to provide proteomic confirmation. By isolating pure populations of GATA6+ and double-positive cells, we confirmed that transitional cells are enriched in proteins associated with metabolic reprogramming and cellular motility. Our findings demonstrate that PDAC plasticity is not a simple switch but a dynamic continuum involving a tangible transitional cell state. These co-expressor cells, with their unique metabolic and signaling dependencies, represent a key driver of tumor progression and a potential reservoir for therapeutic resistance. Targeting the specific vulnerabilities of this transitional phenotype offers a promising strategy to overcome plasticity, prevent relapse, and improve therapeutic outcomes in PDAC.



 Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, Natalia Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth Shroyer. Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B122.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a10f45ec5eb895ff99dbba41e30b63f2ad6188" target='_blank'>
              Abstract B122: Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, N.Ye Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth R. Shroyer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="High-grade serous tubo-ovarian cancer (HGSC) is marked by substantial inter- and intra-tumor heterogeneity. The tumor microenvironments (TME) of HGSC show pronounced variability in cellular make-up across metastatic sites, which is linked to poorer patient outcomes. The influence of cellular composition on therapy sensitivity, including chemotherapy and targeted treatments, has not been thoroughly investigated. In this study, we examined the premise that the variations in cellular composition can forecast drug efficacy. Using a high-throughput 3D in vitro tumoroid model, we assessed the drug responses of 23 distinct cellular configurations of tumoroids comprised of OVCAR3 HGSC cells, mesenchymal stem cells, HUVEC endothelial cells, and U937 monocytes to an assortment of five therapeutic agents, including carboplatin and paclitaxel. We identified that the overall pooled viability in response to these five drugs was highest among tumoroid compositions that contained a large number of myeloid cells, whereas the most sensitive tumoroids to these agents were comprised of only cancer cells. Additionally, we found that the “mesenchymal tumoroids” containing 400 or more mesenchymal stem cells were more sensitive to carboplatin than paclitaxel. By amalgamating our experimental findings with random forest machine learning algorithms, we assessed the influence of TME cellular composition on treatment reactions. Our findings reveal notable disparities in drug responses correlated with tumoroid composition, underscoring the significance of cellular diversity within the TME as a predictor of therapeutic outcomes. This research establishes a foundation for employing human tumoroids with varied cellular composition as a method to delve into the roles of stromal, immune, and other TME cell types in enhancing cancer cell susceptibility to various treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49e1984d98981fa778612856dadda5f531075bc3" target='_blank'>
              Cracking the code: predicting tumor microenvironment enabled chemoresistance with machine learning in the human tumoroid models
              </a>
            </td>
          <td>
            Michael Bregenzer, P. Mehta, Kathleen M Burkhard, Geeta Mehta
          </td>
          <td>2025-08-19</td>
          <td>Npj Biomedical Innovations</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Cervical cancer (CC) is a highly heterogeneous malignancy primarily driven by persistent infection with high-risk human papillomavirus (HPV). However, comprehensive analyses of heterogeneity in the immune microenvironment, particularly its spatial heterogeneity, between HPV-positive and HPV-negative CC remain limited, despite their critical clinical significance. Methods We performed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) sequencing on collected cervical cancer samples, integrating scRNA-seq, ST, and bulk RNA-seq to analyze distinct cell subtypes and characterize their spatial distribution. Multiplex immunofluorescence analysis was further utilized to validate HPV status-specific expression patterns. Cox regression and LASSO regression analyses were used to identify the prognostic signature on the TCGA dataset. Results Through integrative analysis, we found that HPV-positive samples demonstrated elevated proportions of CD4+ T cells and cDC2s, whereas HPV-negative samples exhibited increased CD8+ T cell infiltration. In HPV-positive CC, epithelial cells acted as primary regulators of cDC2s via the ANXA1-FPR1/3 pathway, with cDC2s subsequently modulating CD4+ T cells and interferon-related CD8+ T cell subtypes. In contrast, HPV-negative CC featured epithelial cells predominantly influencing monocytes and macrophages, which then interacted with CD8+ T cells. Notably, the MDK-LRP1 ligand-receptor interaction emerged as a potential key mechanism for recruiting immunosuppressive cells into CC tumors, fostering an immunosuppressive microenvironment. Further, we constructed a risk score model based on an epithelial cell-related signature (ERS), which was significantly associated with patient survival. Noteworthy variations were observed in immune cell infiltration and immune microenvironment among distinct risk groups. Conclusion Based on integrated multi-omics data, we precisely delineated the spatial transcriptional features of the tumor microenvironment in CC with different HPV statuses, including identifying distinct CD8+ T cell states and cell-cell communication. In addition, we developed an ERS closely associated with the immune environment and prognosis of CC. These results increase our understanding of the molecular mechanisms of cervical cancer under different HPV statuses and provide assistance for the precise treatment of cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5152b47b483e0040237d8137dfb459c7f501aef2" target='_blank'>
              Integrated multi-omics analysis of single-cell and spatial transcriptomics reveals distinct hpv-associated immune microenvironment features and prognostic signatures in cervical cancer
              </a>
            </td>
          <td>
            Qiuyue Su, Xiangdong Tian, Fucheng Li, Xi Yu, Wenchen Gong, Yurong Chen, Jianan Wang, Siqi Yang, Shaojun Zhang, Qian Zhang, Shanshan Yang
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has emerged as an advanced biological technology capable of resolving the complexity of cancer landscapes at single-cell resolution. Spatial transcriptomics(ST), as an innovative complementary approach, effectively compensates for the lack of spatial information inherent in scRNA-seq data. This review explores the rapidly evolving integration of scRNA-seq and ST and their transformative role in deciphering the tumor microenvironment (TME). We highlight how these technologies jointly uncover cellular heterogeneity, stromal-immune interactions, and spatial niches driving tumor progression and therapy resistance. Moving beyond previous reviews, we emphasize emerging computational strategies for data integration—including deconvolution and mapping approaches—and evaluate their applications in characterizing immune evasion, fibroblast diversity, and cell-cell communication networks. Ultimately, this review provides a forward-looking perspective on how spatial multi-omics are poised to advance precision oncology through spatially-informed biomarkers and diagnostic tools. We conclude that the full clinical potential of these technologies relies on closing the gap between analytical innovation and robust clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84801b1d17cdb40a9473ba25f6bac709b5c4c887" target='_blank'>
              Single-cell and spatial transcriptomics integration: new frontiers in tumor microenvironment and cellular communication
              </a>
            </td>
          <td>
            Wenxin Shi, Zhiqiang Zhang, Xiaotong Xu, Yanpeng Tian, Li Feng, Xianghua Huang, Yanfang Du, Zhongkang Li
          </td>
          <td>2025-10-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background: Primary hepatocellular carcinoma (HCC) and liver metastases differ in origin, progression, and therapeutic response, yet a direct high-resolution spatial comparison of their tumor microenvironments (TMEs) within the liver has not previously been performed. Methods: We applied high-definition spatial transcriptomics to fresh-frozen specimens of one HCC and one liver metastasis (>16,000 genes per sample, >97% mapping rates) as a proof-of-principle two-specimen study, cross-validated in human proteomics and patients’ survival datasets. Transcriptional clustering revealed spatially distinct compartments, rare cell states, and pathway alterations, which were further compared against an independent systemic dataset. Results: HCC displayed an ordered lineage architecture, with transformed hepatocyte-like tumor cells broadly dispersed across the tissue and more differentiated hepatocyte-derived cells restricted to localized zones. By contrast, liver metastases showed two sharply compartmentalized domains: an invasion zone, where proliferative stem-like tumor cells occupied TAM-rich boundaries adjacent to hypoxia-adapted tumor-core cells, and a plasticity zone, which formed a heterogeneous niche of cancer–testis antigen–positive germline-like cells. Across both tumor types, we detected a conserved metabolic program of “porphyrin overdrive,” defined by reduced cytochrome P450 expression, enhanced oxidative phosphorylation gene expression, and upregulation of FLVCR1 and ALOX5, reflecting coordinated rewiring of heme and lipid metabolism. Conclusions: In this pilot study, HCC and liver metastases demonstrated fundamentally different spatial architectures, with metastases uniquely harboring a germline/neural-like plasticity hub. Despite these organizational contrasts, both tumor types converged on a shared program of metabolic rewiring, highlighting potential therapeutic targets that link local tumor niches to systemic host–tumor interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb3ca762aa10606e657e9aeaed628e45b17550f" target='_blank'>
              Spatial Transcriptomics Reveals Distinct Architectures but Shared Vulnerabilities in Primary and Metastatic Liver Tumors
              </a>
            </td>
          <td>
            S. Adapa, Sahanama Porshe, Divya Priyanka Talada, T. Nywening, Mattew L. Anderson, Timothy I. Shaw, Rays H. Y. Jiang
          </td>
          <td>2025-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive disease with one of the lowest five-year survival rates of all cancers. Low grade, premalignant pancreatic intraepithelial neoplasias (PanINs) are common in the general population, but despite expression of mutant KRAS, the vast majority of these lesions will never progress to cancer. Understanding how additional genetic or epigenetic changes as well as changes to the tumor microenvironment contribute to PanIN to PDAC progression is critical and has the potential to reveal mechanisms of interception for this deadly disease. Our previous work identified the prolyl isomerase PIN1 as a critical factor in the tumor microenvironment for growth of a subset of pancreatic cancer cell lines. Loss of PIN1 in the tumor microenvironment restricted in vivo growth of multiple KPC cell lines. We identified distinct fibroblast phenotypes in the absence of microenvironmental PIN1 expression or function. These changes in fibroblast state drive significant changes in the extracellular matrix and growth factor expression, both of which impact the crosstalk to and behavior of the neoplastic cells. We are continuing to use conditional PIN1 knockout mice to interrogate the specific cell types responsible for the observed tumor-restrictive phenotype and are using single cell sequencing to better understand how changes to individual cell types affect multiple cell populations to inhibit tumor growth. In addition, we are comparing the fibroblast phenotypes observed in our mouse models to those found in tumor-promoting versus tumor-restrictive microenvironments in human tissues. These tissues include autopsy pancreata with low grade PanIN lesions as well as PDAC resection specimens with low and high grade PanINs. We are creating organoid and cell line models of epithelial and stromal cells from normal human pancreata to support complex co-culture engineered models of early disease progression. By integrating across mouse and human model systems, we strive to understand the crosstalk that permits progression of pancreatic cancer and to identify targets to intercept this process.



 Tana Gazdik, Madeline Kuhn, Chloe Bowman, Eric Carlson, Kyra Lindley, Ellen M. Langer. Interrogating microenvironmental changes that impact early pancreatic cancer development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4475acff7b0cbed38743a8484c9c3b1b2adb82c" target='_blank'>
              Abstract A031: Interrogating microenvironmental changes that impact early pancreatic cancer development
              </a>
            </td>
          <td>
            Tana Gazdik, Madeline R Kuhn, Chloe L Bowman, Eric J Carlson, Kyra A Lindley, Ellen M. Langer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The ability of metastatic cancer cells to invade distant tissues requires them to cross a variety of tissue boundaries, each posing distinct structural and biochemical challenges. In particular, the boundary between dense, extracellular matrix (ECM)-rich tumor tissue and surrounding stromal tissue is associated with phenotypic changes in MDA-MB-231 breast cancer cells following transmigration, including increased invasiveness and aggressiveness. It remains unclear whether this transition arises from selective, permissive filtering of pre-existing subpopulations, such as cancer stem cells, or an instructive response of the entire cell population. Here, by combining single-cell migration analysis, heterogeneity analysis of cell proliferation, and computational modeling, it is demonstrated that tumor-tissue boundaries act as instructive interfaces. Using an established 3D fibrillar collagen I matrix model of interfaces, it is shown that all cells can transmigrate the interface with no evidence of selective filtering of subpopulations. Proliferation heterogeneity remains unchanged between transmigrated and non-transmigrated cells, further supporting an instructive mechanism. Simulations confirm that the interface instructively modulates cell behavior. These results indicate that tissue boundaries can reprogram cancer cell phenotypes, representing a potentially targetable mechanism in metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aace3b21099f4dc0398d90f3d8cd70a0bc5a03df" target='_blank'>
              Tumor-Tissue Boundaries as Instructive Interfaces in Breast Cancer Cell Invasion.
              </a>
            </td>
          <td>
            Cornelia Clemens, T. Zerjatke, Andrine Frank, Hannah Trampert, Nataliia Kotsiuba, I. Glauche, T. Pompe
          </td>
          <td>2025-09-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Chemotherapy resistance remains a formidable challenge to the treatment of high-grade serous ovarian cancer (HGSOC). The drug-tolerant cells may originate from a small population of inherently resistant cancer stem cells (CSCs) in primary tumors. In contrast, sufficient evidence suggests that drug tolerance can also be transiently acquired by nonstem cancer cells. Regardless of the route, key regulators of this plastic process are poorly understood. Here, we utilized multiomics, tumor microarrays, and epigenetic modulation to demonstrate that SOX9 is a key chemo-induced driver of chemoresistance in HGSOC. Epigenetic upregulation of SOX9 was sufficient to induce chemoresistance in multiple HGSOC lines. Moreover, this upregulation induced the formation of a stem-like subpopulation and significant chemoresistance in vivo. Mechanistically, SOX9 increased transcriptional divergence, reprogramming the transcriptional state of naive cells into a stem-like state. Supporting this, we identified a rare cluster of SOX9-expressing cells in primary tumors that were highly enriched for CSCs and chemoresistance-associated stress gene modules. Notably, single-cell analysis showed that chemo treatment results in rapid population-level induction of SOX9 that enriches for a stem-like transcriptional state. Altogether, these findings implicate SOX9 as a critical regulator of early steps of transcriptional reprogramming that lead to chemoresistance through a CSC-like state in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016a8c0db4a1babe91a08a91f054acbb57a8720c" target='_blank'>
              SOX9 drives a stem-like transcriptional state and platinum resistance in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            A. Duval, Fidan Seker-Polat, Magdalena Rogozinska, Meric Kinali, A. Walts, Ozlem Neyisci, Yaqi Zhang, Zhonglin Li, Edward Tanner, Allison E. Grubbs, Sandra Orsulic, Daniela Matei, M. Adli
          </td>
          <td>2025-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The tumor microenvironment drives many malignant features of pancreatic ductal adenocarcinoma (PDAC). The fibroblasts within pancreatic tumors promote tissue remodeling, immune suppression, and resistance to therapy. However, the interactions between stromal populations and pancreatic cancer cells are less understood in the liver, the most frequent site of PDAC metastasis. To address this, we employ single cell transcriptomics to compare primary pancreatic vs. liver PDAC lesions. Here, we identify the expression of hepatocyte growth factor (HGF) in fibroblasts and its receptor MET in cancer cells are both markedly increased in the PDAC liver niche. Using functional assays, we validate that mitogenic MET signaling is activated in PDAC cells by liver-derived fibroblasts. Importantly, the inhibition of MET signaling leads to reduced tumor growth in immune competent mouse models. Collectively, our data demonstrates that liver stromal-epithelial crosstalk networks engage in signaling pathways distinct from primary pancreatic tumors, highlighting opportunities to develop new treatments for metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aa94d2601cca690cf0f25d219abda02e990ca8c" target='_blank'>
              Comparative Analysis of Primary and Liver Fibroblasts Reveals MET as a Potent Target in Pancreatic Cancer Metastasis
              </a>
            </td>
          <td>
            Rima Singh, Natalie Yousefian, Walker M. Allen, Cecily Anaraki, Alica K Beutel, Sabrina E. Calderon, Ian M Loveless, Oliver McDonald, Jennifer P Morton, D. Imagawa, Zeljka Jutric, Thomas F. Martinez, Nina G. Steele, Christopher J Halbrook
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Liver metastases represent a leading cause of mortality in pancreatic ductal adenocarcinoma (PDAC). Recent clinical evidence has revealed distinct immune ecosystems in the liver tumor microenvironment, yet mechanistic insight into the cellular and molecular mechanisms driving those diverse immune landscapes remains unclear. Here we report, computational cellular signaling analysis of single-cell RNA sequencing data from matched primary and liver metastatic samples isolated at early and late time points from the spontaneous genetically engineered mouse model KPC (Pdx1Cre; KrasLSL-G12D; Tp53LSL-R172H). This analysis uncovered several immunomodulatory axes between cancer cells-T cells and cancer cells- macrophages. Validation of these findings were performed using primary tumor cell lines, isolated from the same or littermate tumors as the scRNA-seq dataset. Bulk RNA-seq analysis of these cell lines showed a retention of cancer-derived ligands in culture. In vivo validation using the hemisplenic liver metastatic transplant model confirmed that cell lines isolated from primary tumors, which metastasized spontaneously to the liver, retained liver metastatic outgrowth potential. Flow cytometry and multiplex cytokine profiling of isolated cell lines, non-involved liver, and liver tumors revealed distinct immune population shifts within the metastatic niche and confirmed cancer cell derived immunomodulatory cytokines. This study highlights the interplay between cancer-intrinsic signaling and the immune landscape in PDAC liver metastases and supports future mechanistic exploration.



 Ayushi Mandloi, Jace Baines, Meet Patel, Christina R. Larson, Robert S. Welner, Julienne L. Carstens. Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A081.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fce5d9cdc25fe3d365bf9aa1c45eddedfbe9968" target='_blank'>
              Abstract A081: Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth
              </a>
            </td>
          <td>
            Ayushi Mandloi, Jace Baines, Meet Patel, Christina R Larson, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Malignant rhabdoid tumors (MRTs) are aggressive pediatric cancers with poor outcomes. MRTs exhibit low tumor mutational burden, yet recent studies reported immune cell infiltration. Here, we used spatial transcriptomics and multi-omic profiling to study immune cell infiltration in MRT samples. We observed a diverse repertoire of candidate tumor antigens (TAs), IRF1 signaling activity and elevated expression of antigen processing and presentation genes, strongly associated with a “hot” tumor immune microenvironment (TIME). Upregulation of factors involved in skeletal muscle development was observed in some MRTs with higher CD8 + T cell infiltration and the ratio of M1/M2 macrophages was positively correlated with cytotoxic lymphocyte infiltration. We identified genes, such as SPP1, preferentially expressed in tumor-associated macrophages (TAMs) and noted the MITF regulatory network appeared active in the M2-like TAMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/248c19c84a73e2abe2ce78fd660b38edcf7c75f9" target='_blank'>
              Spatial and multi-omic profiling reveals genes and pathways associated with cytotoxic lymphocyte infiltration in malignant rhabdoid tumor
              </a>
            </td>
          <td>
            Marco A. Marra, Dan Jin, Joshua Dubland, E. Mullen, James Geller, J. Bush
          </td>
          <td>2025-08-19</td>
          <td>Research Square</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous cancer with complex tumor–immune interactions. This heterogeneity, particularly in tumor and immune cells, complicates treatment and prognostic evaluation. Although recent studies have revealed distinct tumor cell states and immune dysfunction in HCC, the molecular basis underlying tumor aggressiveness remains poorly understood. A deeper understanding of the molecular and functional diversity of both tumor and immune cell populations, especially the identification of stem-like tumor subpopulations and immunosuppressive mechanisms, along with the development of robust prognostic biomarkers, is essential for advancing precision oncology and improving clinical outcomes.We integrated three publicly available single-cell RNA sequencing (scRNA-seq) datasets from GEO to delineate the cellular architecture of the HCC tumor microenvironment. Unsupervised clustering and dimensionality reduction were employed to identify major cell types and tumor subpopulations. Functional annotation was performed using canonical markers, Monocle, CytoTRACE, and AUCell scoring. H2AFZ was identified as a candidate oncogene and validated through in vitro knockdown experiments. The interaction between T cell subsets and tumor subpopulations were further characterized. A prognostic risk model was constructed using LASSO regression.Six major cell types were identified in HCC TME. Tumor cells were subdivided into three distinct clusters: Tumor_C0, Tumor_C1 and Tumor_C2. Tumor_C2 showed the highest stemness, pro-metastatic activity and immunogenic cell death signatures. H2AFZ was highly expressed in Tumor_C2 and associated with poor prognosis. The knockdown of H2AFZ reduced H2A.Z protein levels, inhibited proliferation, invasion, and induced apoptosis. T cell analysis revealed five subpopulations. It was found that Tumor_C2 interacts with the proliferative and exhausted T cell subpopulations, suggesting a potential functional relationship between them. The prognostic model based on tumor_C2 transcriptomic features effectively stratified patient survival across multiple cohorts, with robust AUCs and Kaplan-Meier survival distinctions.We identified a proliferative, stem-like tumor cell subpopulation (Tumor_C2) in HCC characterized by high H2AFZ expression, which drives tumor aggressiveness. T cell analysis revealed significant interactions with Tumor_C2. Moreover, a prognostic model based on Tumor_C2 features effectively stratified patient survival across multiple cohorts. Together, these findings highlight potential therapeutic targets for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7024320dd03972a4da66c548f19e56393e35ae99" target='_blank'>
              Single-cell transcriptomics identifies an H2AFZ-driven proliferative tumor subpopulation associated with poor prognosis in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jihui Huo, Tao Yang, Kai Lei, Zeyao Wang, Zebin Chen, Qi Zhou
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Breast cancer brain metastases are associated with a poor prognosis, and there is a high demand for improved personalized treatment strategies. It is still poorly understood which breast cancer brain metastases patients benefit from immune checkpoint blockade.



 Here, we multimodally characterized the immune niche of breast cancer brain metastases at single-cell resolution in a clinically annotated patient cohort (n = 156), combining multiplexed imaging and cell type quantification by tissue cytometry, single-nuclei RNA-sequencing, flow cytometry, spatial transcriptomics, and functional studies in patient-derived organoids.



 Overall, infiltration of breast cancer brain metastases by CD8 T cells was strongly heterogeneous between patients, and high numbers were prognostically favorable. In-depth characterization of cell types and integrative analyses including clinical data revealed distinct immune landscapes across patients, two of which independently predicted prolonged overall survival: i) Breast cancer brain metastases containing T and B cells accumulating together with characteristic cell communities in stromal tertiary lymphoid structures, and ii) breast cancer brain metastases characterized by a high content of tissue resident-like CD8 memory T cells (CD8 TRM-like cells) spatially enriched within tumor islands. CD8 TRM-like cells exhibited high levels of activation, checkpoint, and functional markers and showed tumor cell killing capacity ex vivo, suggesting them to be critical players of tumor immune control.



 Together, our work provides novel insights into anti-tumor immunity in breast cancer brain metastases and reveals prognostic immune biomarkers with potential translational relevance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb6c7f193a5bf8137e26dc814e6139f92f208a4" target='_blank'>
              KS03.5.A DISSECTING THE CELLULAR ARCHITECTURE OF BREAST CANCER BRAIN METASTASES REVEALS PROGNOSTICALLY DISTINCT IMMUNE LANDSCAPES
              </a>
            </td>
          <td>
            L. Jassowicz, F. Feng, R. Warta, F. Sor, J. Soller, A. Wöllner, C. Lotsch, J K Wong, J. Schlegel, L. Frank, P. Dao Trong, G. Jungwirth, A. Nohman, K. Maaß, M. Barthel, A. Unterberg, S. Krieg, K. Rippe, A. Abdollahi, A. von Deimling, J. Malm, P. Lichter, M. Zapatka, M. Seiffert, C. Herold-Mende
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The tumor microenvironment (TME) consists of tumor cells, stromal cells, infiltrating immune cells and non-cellular components such as extracellular matrix, blood vessels and a wide variety of secreted proteins. Evidence shows that beyond supporting tumor growth, the TME also promotes tumor cell proliferation and invasion and contributes to treatment resistance, ultimately affecting patient prognosis. Cell-to-cell communication within the TME is driven by secreted proteins such as cytokines, chemokines, growth factors and interferons, which are produced not only by tumor cells but also by various stromal cells and immune cells. These proteins form a complex signaling network that promotes tumor cell proliferation and invasion and enables tumors to evade innate and adaptive immune responses. Antibody arrays are a technology that can simultaneously screen hundreds of secreted proteins in complex biological samples, aiding in the exploration of this complex signaling network. By combining high-throughput multiplex immunoassays such as antibody arrays with cellular and molecular biology techniques, researchers have uncovered complex regulatory mechanisms of cytokine networks within the TME. The present review summarized recent findings on the communication between tumor cells and the TME, as well as key secreted proteins essential for tumor progression and the development of therapeutic resistance. In addition, it discusses how high-throughput antibody arrays contribute to our understanding of regulatory networks of secreted proteins in the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b84b1200ba5d18e4e2abf23d107f5f3c2ed9e0f6" target='_blank'>
              Unraveling tumor cell-tumor microenvironment crosstalk through antibody array technologies (Review)
              </a>
            </td>
          <td>
            Yanlin Wang, Shuhong Luo, Hua Dong, Ruo-Pan Huang
          </td>
          <td>2025-08-18</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Emerging clinical evidence highlights the importance of tumor infiltration of activated immune cells, particularly T and B lymphocytes, for the efficacy of immune-based cancer therapies. However, many patients show limited immune cell presence in the tumor microenvironment. High endothelial venules (HEVs) are specialized blood vessels that enable lymphocyte trafficking from circulation into lymph nodes or tumors. Histological analyses of clinical tumor samples indicate that HEVs are positioned near dense aggregates of B and T lymphocytes, known as tertiary lymphoid structures (TLS), underscoring their role in immune cell recruitment and anti-tumor immunity. The abundant development of TLS and HEVs is associated with clinical response to immune checkpoint inhibition therapy and cancer survival. Recently, we found that simultaneous activation of innate immune effectors, STING and lymphotoxin-β receptor (LTβR) using their agonists induces the formation of TLS and HEVs in TLS/HEV-free tumors. In the current study, mouse pancreatic ductal adenocarcinoma (KPC) tumors were grown in the dorsal skin fold window chamber (DSWC) and received STING/LTβR agonist combination therapy to monitor the real-time TLS/HEV development and dynamic tumor microenvironment in intravital microscopy (IVM). Transgenic mice with fluorescent red B cells and green fluorescent CD31+ blood vessels were infused with MECA-79 antibody (HEV marker) to visualize B cell trafficking through HEVs. Four-color confetti mice crossed with B cell-specific Cre mice, allowed us to analyze the clonal expansion of B cells expressing one of four colors, with daughter cells retaining their parent color. Our findings demonstrated that tumoral HEVs closely resemble those found in lymph nodes, characterized by fewer branches, longer segment lengths, and increased diameters compared to non-HEV tumor blood vessels, which typically exhibit chaotic patterns. Circulating B cells traveling through tumoral HEVs displayed significantly slower rolling velocities than non-HEV tumor vessels, likely due to enhanced integrin and L-selectin expression induced by the combination therapy. By repeated intravital imaging, we were also able to observe a quadruple increase in B cell number in TLS during the day 4-7 treatment time frame. Following adoptive transfer of splenic leukocytes from naïve donor mice, we found that these leukocytes preferentially homed to TLS-rich areas, with B cells constituting the majority of the recruited cells. Importantly, analyses using the confetti mouse model indicate that tumor infiltrating B cells underwent clonal expansion in TLS, with different TLS having different colors. Overall, our study demonstrates that HEVs are the gateways for B cell entry to tumors, and the intratumoral TLS formation is driven by localized clonal expansions of these B cells, resembling lymph node germinal center activities, which drive the development of humoral immunity against tumors.



 Gabrielle Rowe-Brown, Masanobu Komatsu. B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36afed9d0acad24d10324fa459b1bfa3a197ed1" target='_blank'>
              Abstract A010: B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy
              </a>
            </td>
          <td>
            Gabrielle Rowe-Brown, Masanobu Komatsu
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Retinoblastoma is the most common intraocular malignant tumor in childhood. Although current treatments offer a high survival rate, treatment toxicity, tumor relapse, and treatment resistance require a deeper understanding of the disease mechanisms to develop adapted therapies. Microscopically, this tumor is characterized by different states of differentiation and proliferation, ranging from poorly differentiated to well-differentiated retinoblastoma. Retinocytoma, on the other hand, is a benign non-proliferative tumor. Recent next-generation multi-omics analyses classified retinoblastoma tumors in subtypes 1 or 2, subtype 2 presenting a later age of onset, more genetic alterations, and higher metastatic potential. In parallel, several single-cell transcriptomics studies demonstrated intratumoral heterogeneity. However, mapping the different cell populations directly on the tumor and comparing histological features and molecular subtypes remains an unmet need. Methods Spatial transcriptomics was used to characterize a primary enucleated case with two histologically distinct areas. The whole transcriptomic profile of sixteen regions of interest, covering the two differentiation patterns of the tumor and the non-tumoral retina, was analyzed. Results The clustering of the regions of interest based on whole transcriptome data correlated with the histological description: cluster 1 (6 regions of interest) corresponded to highly differentiated areas and cluster 2 (6 regions of interest) to poorly differentiated areas. They showed enrichment for phototransduction and proliferation respectively, confirmed by immunohistochemistry for markers of these pathways. The publicly available molecular signatures of the two retinoblastoma subtypes categorized our regions of interest into two groups, which correlated perfectly with histological observation and transcriptomic profiles. Further analysis of the expression of specific senescence markers in the well-differentiated area did not support the diagnosis of retinocytoma as it did not confirm the expected up-regulation seen in this tumor type. Conclusions This study demonstrates a strong correlation between histological observation and molecular profiling, representing the first mapping of retinoblastoma composed of adjacent molecular subtypes 1 and 2. It also highlights the diagnostic ambiguity between retinocytoma and well-differentiated non-proliferative subtype 1 retinoblastoma and emphasizes the need for specific biomarkers to differentiate these two tumor types. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14814-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/599058c9510122a1b04d9c9b60c21f33e8817b4e" target='_blank'>
              Spatial transcriptomics of retinoblastoma: a visual window on intra-patient heterogeneity
              </a>
            </td>
          <td>
            A. Moulin, J. Thevenet, L. Mazzeo, S. Tissot, C. Stathopoulos, F. L. Munier, A. Berger
          </td>
          <td>2025-09-02</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background Chromophobe renal cell carcinoma (ChRCC) is the second most common non-clear cell renal cell carcinoma (ncRCC). A series of 496 ChRCC from Memorial Sloan Kettering Cancer Center demonstrated that sarcomatoid features, which were present in about 1.2% of the cases, are strongly correlated with metastatic disease and reduced overall survival. Little is known about the biological and immunological characteristics of the sarcomatoid variant of ChRCC and how it differs from its classic, non-sarcomatoid form. Methods Eight ChRCC tumor samples containing both classic and sarcomatoid components were selected. Xenium Prime 5K assay (5000 curated genes related to oncoimmunology) was used for high-resolution spatial transcriptomic profiling. Data were analyzed using Seurat and custom scripts to annotate cell types and conduct detailed phenotypic analysis. Pathway enrichment and transcription factor regulatory networks were analyzed using the SENIC package. Cell-cell communication was inferred using CellChat, and spatial proximity of immune and tumor cells (“cellular neighborhoods”) was analyzed, which model spatial gene expression patterns and intercellular distances. Results Spatial transcriptomic analyses revealed a significantly more immune-inflamed tumor microenvironment in sarcomatoid regions compared to their classic counterparts. We identified distinct spatial immune “neighborhoods” in sarcomatoid areas that were enriched in CD8+ T cells, polarized macrophages, and dendritic cells. These neighborhoods exhibited organized proximity to tumor cells expressing immune evasion markers such as PD-L1. In contrast, classic regions were relatively immune-deserted. Analysis of transcription factor activity showed upregulation of IRF1 and STAT1-related programs in the immune infiltrates of sarcomatoid regions, consistent with inflammatory activation. Importantly, comparison of matched sarcomatoid and classic regions within the same tumors enabled us to define a “sarcomatoid-enriched gene signature” composed of immune-related, pro-metastatic, and epithelial-to-mesenchymal transition (EMT) genes. This signature was validated in bulk RNA-seq data from The Cancer Genome Atlas (TCGA), where it correlated with poor prognosis in ChRCC patients. Conclusions This study presents the first spatial transcriptomic atlas of sarcomatoid ChRCC in comparison to matched non-sarcomatoid (classic) from the same patient. Profound immune microenvironmental differences were identified between tumor compartments. The sarcomatoid variant exhibits hallmarks of an immune-inflamed but potentially immune-evasive phenotype, including enriched immune infiltration, pro-inflammatory signaling, and EMT-associated transcriptional programs. These findings highlight the biological differences between sarcomatoid and classic ChRCC and uncover specific immune pathways and spatial cell-cell interactions that may be targeted in future immunotherapeutic strategies. Our work also establishes a foundational spatial map for ChRCC that offers new insight into the disease heterogeneity and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aba1786fb42a662d7a501d0a751ee46eaa895c9" target='_blank'>
              52Spatial transcriptomic mapping reveals immune-inflamed niches in sarcomatoid chromophobe renal cell carcinoma
              </a>
            </td>
          <td>
            Yan Tang, Tiegang Han, Hadi Mansour, M. Hirsch, Katrina Collins, Elizabeth P Henske
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background The tumor microenvironment (TME) is a complex network driving endometrioid endometrial cancer (EC) progression. Previous analyses of the EC TME were often limited by a lack of detailed cellular annotations. Integrating single-cell RNA sequencing (scRNA-seq) data offers a comprehensive approach to reconstruct the TME, aiming to uncover novel mechanisms and therapeutic targets. Methods We integrated public scRNA-seq data from 15 EC and 5 normal endometrium samples. Through detailed cell annotation, we performed comprehensive bioinformatic analyses, including pseudotime trajectories, copy number variation, and cell communication, to investigate mechanisms of EC development. Results We identified nine cell types and characterized subpopulations of epithelial, macrophage, lymphocyte, and stromal fibroblast cells. The SOX9+LGR5- epithelial subtype showed elevated malignancy and NFKB pathway enrichment. The M2_like2 macrophage subtype played a critical role, engaging in robust MIF-(CD74+CD44) mediated communication with SOX9+LGR5- cells. Experimental validation confirmed MIF co-expression with E-cadherin in EC tissues. Furthermore, the transcription factor NFKB2 was found to mediate MIF’s effect on the CD44 receptor in malignant epithelial cells. A pericyte-to-fibroblast transition in stromal cells may also support tumor growth, while an increase in CD8 exhausted/Treg cells and a decrease in cytotoxic CD8 cells suggest potential immune evasion. Conclusion Our single-cell analysis details the EC TME landscape, revealing robust communication between M2_like2 macrophages and SOX9+LGR5- epithelial cells. We highlight a key mechanism where NFKB2 mediates MIF’s pro-tumorigenic effects via the CD44 receptor, offering new insights into EC progression and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84d976acc17e07f02157c06df3f07e8d3258236f" target='_blank'>
              Single-Cell Transcriptomic Analysis Highlights the Impact of NFKB2-Mediated MIF-CD44 Signaling Axis in Endometrioid Endometrial Cancer
              </a>
            </td>
          <td>
            Lu Zhang, Mengjie Yang, Quan Zhang, Yiqian Zhang, Qiyuan Li, Qionghua Chen
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Women's Health</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The cellular plasticity of neuroblastoma is defined by a mixture of two major cell states, adrenergic and mesenchymal, which may contribute to therapy resistance. However, how neuroblastoma cells switch cellular states during therapy remains largely unknown, and how to eradicate neuroblastoma regardless of its cell state is a clinical challenge. To better understand the cellular plasticity of neuroblastoma in chemoresistance, we define the transcriptomic and epigenetic map of adrenergic and mesenchymal types of neuroblastomas using human and murine models treated with indisulam, a selective RBM39 degrader. We show that cancer cells not only undergo a bidirectional switch between adrenergic and mesenchymal states, but also acquire additional cellular states, reminiscent of the developmental pliancy of neural crest cells. These cell state alterations are coupled with epigenetic reprogramming and dependency switching of cell state–specific transcription factors, epigenetic modifiers, and targetable kinases. Through targeting RNA splicing, indisulam induces an inflammatory tumor microenvironment and enhances the anticancer activity of natural killer cells. The combination of indisulam with anti-GD2 immunotherapy results in a durable, complete response in high-risk transgenic neuroblastoma models, providing an innovative, rational therapeutic approach to eradicate tumor cells regardless of their potential to switch cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8bfb0658591ed4d276e6df844046df62c0e809" target='_blank'>
              RBM39 degrader invigorates innate immunity to eradicate neuroblastoma despite cancer cell plasticity
              </a>
            </td>
          <td>
            Shivendra Singh, Jie Fang, Hongjian Jin, Lee-Ann Van de Velde, Andrew Cortes, Jiani Chen, Sivaraman Natarajan, E. Poon, Qiong Wu, Christopher L. Morton, Mary A. Woolard, Waise Quarni, Jacob A Steele, Jon P Connelly, Liusheng He, Rebecca Thorne, Gregory Turner, Thomas Confer, Melissa Johnson, W. Caufield, Burgess Freeman, Timothy Lockey, A. Murphy, P. J. Murray, Takashi Owa, S. Pruett-Miller, Ruoning Wang, L. Chesler, Julie R. Park, A. Davidoff, John Easton, Xiang Chen, Paul G. Thomas, Jun Yang
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Background The recent clinical success of tumor-infiltrating lymphocyte (TIL) therapy in immune checkpoint inhibitor-refractory melanoma has renewed interest in applying this approach to other malignancies. Renal cell carcinoma (RCC) is characterized by prominent CD8+ T-cell infiltration and elevated expression of cytolytic genes, suggesting the presence of tumor-reactive T-cells that could be harnessed for personalized cell therapy. TIL products are polyclonal and may recognize a diverse repertoire of tumor antigens, which could reduce the likelihood of immune escape through antigen loss. Despite this potential, past efforts to implement TIL therapy in RCC have been largely unsuccessful, likely due to the dominance of TILs exhibiting an exhausted phenotype, and the inability to effectively rejuvenate these cells during ex vivo expansion. Thus, the development of strategies that can selectively isolate and expand tumor-specific T-cells with minimal off-target toxicity and durable cytotoxic function will likely be key to the successful development of TIL therapy for RCC. We recently demonstrated that highly expanded clonal CD8+ T-cell populations in RCC tumors exhibit an antigen-experienced phenotype, indicative of chronic stimulation and consistent with tumor antigen recognition. These clonotypes represent a key feature of RCC TIL populations. Based on this, we hypothesize that the selection of non-exhausted, clonally expanded T-cells using specific surface markers will enhance the therapeutic potency and specificity of RCC-derived TIL products. Methods To identify surface markers that distinguish non-exhausted, clonally expanded tumor-reactive T-cells, we compiled a comprehensive single-cell RNA sequencing (scRNAseq) and paired TCR sequencing dataset. This includes tumor, normal adjacent tissue (NAT), peripheral blood mononuclear cells (PBMC), and lymph nodes (LN) from 23 clear cell RCC (ccRCC) patients, combining seven published and in-house studies. We are applying both differential gene expression (DGE) analysis and multi-instance-based machine learning (ML) to identify candidate surface markers linked to non-exhausted, clonally expanded phenotypes. These models are designed to accommodate the inherent noise and sparsity of single-cell data and to uncover combinatorial marker sets. Results In RCC tumors, we observed significant clonal expansion of T-cells with a memory-like, antigen-experienced phenotype. Tumor samples showed significantly higher T-cell clonality compared to matched normal tissues. Cells sharing the same TCR clonotype displayed heterogeneous transcriptional states, including both exhausted and non-exhausted phenotypes. Markers of tumor reactivity identified for other cancers (eg, CD39, CD103) labeled mainly exhausted subsets of expanded clonotypes, suggesting a need for improved marker combinations for RCC. Clonality tended to decline with increasing tumor stage, possibly indicating a link between exhaustion and tumor progression. DGE analysis revealed several surface markers associated with expanded TCR clonotypes, for which commercial antibodies are available for flow sorting. To overcome limitations of DGE alone, we are applying ML models that integrate gene expression, TCR identity, and clonal frequency to predict tumor reactivity. We successfully trained and validated multiple machine learning models, including scFormer (transformer-based), attention_MIL (attention-based deep multi-instance learning), miBoost (tree-based), and miSVM (support vector classifier), on sparse and heterogeneous scRNAseq data using a local GPU platform. All models showed consistent improvement in validation accuracy during training. Feature analysis revealed gene sets predictive of large versus small clonotypes, offering candidate markers for isolating tumor-reactive, non-exhausted T cells for downstream applications. Conclusions T-cells sharing the same T-cell receptor can display both terminally exhausted and progenitor exhausted states. Selecting clonally expanded, non-exhausted T cells may enhance the tumor reactivity of TIL products. Our ongoing analyses, including model hyperparameter optimization and gene pathway comparison, focus on identifying surface markers that are antibody-compatible, broadly expressed across patients, and applicable to CD8+ and CD4+ T-cells while excluding Tregs. These insights will support a refined TIL manufacturing strategy for improved RCC therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3870d4f63b8bc5e4814aaf86df832a9b45daa983" target='_blank'>
              30Meta-analysis of single-cell RNA sequencing of tumor-infiltrating T-cells in clear cell renal cell carcinoma reveals distinct phenotypes and prediction markers of expanded T-cell clonotypes
              </a>
            </td>
          <td>
            Yuexin Xu, Chris Miller, S. Tykodi, E. Warren
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="


 Medullary thyroid carcinoma (MTC) is a rare neuroendocrine tumor arising from parafollicular C-cells in the thyroid. Unlike other neuroendocrine tumors, knowledge about the MTC tumor microenvironment and its role in therapy resistance remains limited, mainly due to the lack of reliable in vitro models. We hypothesized that, as observed in other cancer types, MTC cells may exhibit a degree of phenotypic plasticity that could be modulated by various microenvironmental factors.



 The primary aim of this study was to establish patient-derived MTC in vitro models capable of growing under different three-dimensional (3D) culture conditions while retaining the ability to shift toward a more mesenchymal, stem-like phenotype.



 We successfully established eight patient-derived primary MTC cell lines from tumors with different genetic backgrounds and variable degrees of aggressiveness. These lines were characterized in terms of phenotype and responses to different microenvironmental stimuli and targeted therapies through Western Blot, confocal microscopy, ELISA assays, ELDA, RT-qPCR, and proliferation assays.



 Notably, the cells maintained their plasticity over extended passaging when cultured under stem-promoting conditions. In differentiation media, they preserved the ability to secrete high levels of classic MTC markers, such as calcitonin and chromogranin A. Under 3D culture conditions, all cell lines showed a significant increase in stem and neuroendocrine precursor markers, including SOX2, OCT4, TBB3 and FOXA1, along with spatially dependent shifts in Epithelial-to- Mesenchymal Transition (EMT) markers E-cadherin and Vimentin. These phenotypic variations partially recapitulated the molecular alterations observed in MTC tissue samples. Moreover, changes in culture conditions—such as inducing a pseudo- hypoxic state or culturing in non-adherent environments—profoundly affected both cell phenotype and sensitivity to therapeutic agents currently available for MTC treatment, including Pralsetinib, Selpercatinib, Cabozantinib, Lenvatinib, and Vandetanib.



 In conclusion, we present here the generation of a new valid patient- derived in vitro model that can provide new opportunities to explore MTC biology complexity and will allow future translational studies, including those involved in uncovering the mechanisms underlying the insurgence of therapy resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0c689353bda0dac4479e679a592360fdc7492a4" target='_blank'>
              OC48 - Generation of patient-derived in vitro models to unravel medullary thyroid cancer microenvironment and therapy resistance
              </a>
            </td>
          <td>
            V. Ghiandai, V. Cirello, G. Gazzano, G. Dionigi, C. Colombo, L. Fugazzola, L. Persani, E. S. Grassi
          </td>
          <td>2025-09-01</td>
          <td>European Journal of Endocrinology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Metastasis remains the principal cause of cancer-related mortality, yet its distribution across organs is far from random. Instead, tumor cells exhibit organotropism, a consistent preference for colonizing specific distant tissues, a phenomenon shaped by anatomical constraints, molecular crosstalk, and microenvironmental compatibility. Far beyond mere mechanical entrapment in vascular beds, metastatic dissemination reflects a coordinated interplay between tumor-intrinsic programs and organ-specific niches. Tumor-derived extracellular vesicles, cytokines, and matrix-remodeling enzymes actively precondition distant sites through pre-metastatic niche formation, creating permissive microenvironments primed for colonization. Simultaneously, tissue-specific immune landscapes, stromal compositions, and mechanical architectures determine the fate of disseminated tumor cells, whether they are eliminated, enter dormancy, or form macrometastases. Phenotypic plasticity, metabolic reprogramming, and immune evasion mechanisms equip subclones with the capacity to exploit these unique niches. Across cancer types, reproducible patterns of organotropic metastasis not only guide clinical surveillance and therapeutic stratification but also reveal vulnerabilities in the metastatic cascade. This review synthesizes emerging mechanistic insights across anatomical, immunological, and molecular domains to construct a comprehensive framework of organotropism, highlighting therapeutic opportunities to intercept metastasis at organ-specific checkpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7663dab540302726b6f7b6337c8bf27a6cf336" target='_blank'>
              Deciphering organotropism reveals therapeutic targets in metastasis
              </a>
            </td>
          <td>
            Jhommara Bautista, Andrés López-Cortés
          </td>
          <td>2025-10-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Immunogenic cell death (ICD) triggers antitumor immune responses and plays a critical role in shaping the tumor microenvironment (TME). However, its specific contribution to lung adenocarcinoma (LUAD) progression and immunotherapy response remains insufficiently explored. Method We integrated single-cell RNA sequencing with machine learning to characterize ICD-related transcriptional features in LUAD. ICD activity was quantified across cell types using five scoring algorithms. To develop a robust prognostic model, we evaluated over 100 machine learning algorithm combinations and selected the CoxBoost + SuperPC approach based on the highest concordance index (C-index). The resulting ICD-related gene signature (ICDRS) was validated in six external cohorts. Downstream analyses included immune infiltration, mutation profiling, drug sensitivity, and immunotherapy response. SLC2A1 was selected for functional validation using qRT-PCR, CCK-8, Transwell, colony formation, and xenograft assays. Results Single-cell analysis revealed that macrophages exhibited the highest ICD activity and contributed significantly to intercellular communication. Based on ICD-associated genes, the ICDRS model consisting of 11 core genes was constructed and showed superior prognostic performance over 112 published LUAD signatures across multiple cohorts. The ICDRS stratified patients into distinct risk groups with significant differences in overall survival, immune infiltration patterns, and immunotherapy benefit. Low-risk patients exhibited higher levels of CD8⁺ T cells, dendritic cells, and immune function scores, along with greater sensitivity to standard chemotherapeutics and immune checkpoint inhibitors. Functional experiments confirmed that SLC2A1 was upregulated in LUAD tissues and cell lines. Silencing SLC2A1 suppressed proliferation and invasion in vitro and inhibited tumor growth in xenograft models, supporting its oncogenic role. Conclusion This study highlights the crucial role of ICD in LUAD immune regulation and prognosis. The ICDRS serves as a robust biomarker for risk stratification and immunotherapy guidance, while SLC2A1 emerges as a potential therapeutic target to augment immunotherapeutic efficacy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06889-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62932e2fbe663bdc8582669ef1ad54852c730dc6" target='_blank'>
              Integrative single-cell and machine learning approach to characterize immunogenic cell death and tumor microenvironment in LUAD
              </a>
            </td>
          <td>
            Han Zhang, Qiuqiao Mu, Yuhang Jiang, Xiaojiang Zhao, Xiaoteng Jia, Kai Wang, Xin Li, Daqiang Sun
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Cutaneous T-cell Lymphomas (CTCLs) refer to a group of rare cancers that affect immune cells in the skin and often the blood. Their exact cause is not entirely understood, and they are difficult to diagnose because they can appear similar to common, non-malignant skin conditions. Recent advances in single-cell RNA sequencing have made it possible for researchers to explore these tumors at an unprecedented level of detail, revealing important differences between CTCL subtypes in how they interact with the immune system. By analyzing these patterns, researchers can begin to explain differences in disease behavior and identify targets for new therapies. This summary highlights how single-cell technologies are transforming our understanding of CTCL and may pave the way for more precise diagnosis and treatment strategies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/504bc3b3f8515fbfaaeba0942215a380977f71da" target='_blank'>
              An Update on Single-Cell RNA Sequencing in Illuminating Disease Mechanisms of Cutaneous T-Cell Lymphoma
              </a>
            </td>
          <td>
            Sara Suhl, Alexander Kaminsky, Caroline Chen, B. Lapolla, Maggie H. Zhou, Joshua Kent, Abigail Marx, I. Nebo, G. Ramush, Sophia Luyten, Yoni Sacknovitz, Julie Sung, Christina M. Bear, Celine M. Schreidah, Alejandro A. Gru, L. J. Geskin
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive disease with a dense fibrotic stroma and is often accompanied by chronic inflammation. Peritumoral inflammation is typically viewed as a reaction to nearby tumor growth. Here, we report that the inflamed pancreatic lobules are frequently invaded by tumor cells, forming a distinct, non-fibrotic tumor niche. Using a semi-supervised machine learning approach for annotations of clinical samples and multiplex protein profiling, we show that tumor cells at the invasion front are closely associated with acinar cells undergoing damage-induced changes, and with activated fibroblasts expressing markers of injury. The invaded lobules are linked to classical tumor phenotypes, in contrast to fibrotic areas where tumor cells display a more basal profile, highlighting microenvironment-dependent tumor subtype differences. In female mice, lobular invasion similarly aligns with the classical tumor phenotype. Together, our data reveal that pancreatic tumors colonize injured lobules, creating a unique niche that shapes tumor characteristics and contributes to disease biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33fdbbbab057fd0e654aba5735a7c83d83362e84" target='_blank'>
              An injury-associated lobular microniche is associated with the classical tumor cell phenotype in pancreatic cancer
              </a>
            </td>
          <td>
            Sara Söderqvist, Annika Viljamaa, Natalie Geyer, Anna-Lena Keller, Kseniya Ruksha, Carina Strell, Neda Hekmati, Alexandra Niculae, J. Engstrand, E. Sparrelid, Caroline Salmén, Tânia D. F. Costa, Miao Zhao, Staffan Strömblad, Argyro Zacharouli, P. Ghorbani, Sara Harrizi, Yousra Hamidi, Olga Khorosjutina, Stefina Milanova, Bernhard Schmierer, Béla Bozóky, C. Fernández Moro, Marco Gerling
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Breast cancer progression is a consequence of intricate dynamics between cells and their matrix in the tumor microenvironment. However, most in vitro models are not amenable to studying the behavior of multiple cell types within a defined matrix architecture. In this study, we demonstrate a microporous matrix where breast cancer cells and adipose derived stromal cells are integrated to evaluate crosstalk between matrix parameters and heterotypic cell populations. To do this, we leveraged two biofabrication techniques, granular suspension matrices and drop-on-demand bioprinting, to deposit multiple cell types in a reproducible format amenable to high-throughput screening. 3D gelatin-methacryloyl microgels were used to create a yield stress granular suspension bath with tunable interstitial volume to mimic the porosity and densities of healthy and fibrotic microenvironments. Invasive and non-invasive breast cancer cells (MCF-7 and MDA-MB-231) were bioprinted at the interface of the ADSC-laden granular matrix to probe invasive processes and heterotypic crosstalk as a tumor–stroma model. We focused on cancer cell migration through model fibrotic tissue and ADSC transformations into cancer associated fibroblasts. α-smooth muscle actin expression indicated that the high density microgel matrices are more conducive to ADSC-CAF transformations, which in turn influenced the expression of molecular markers associated with tumorigenicity and chemoresistance in the resident cancer cells. Treatment with doxorubicin supported increased tumorigenicity in the co-cultures. Together, this work demonstrates how defined microengineered matrices can serve as platforms to evaluate cell behavior, with scope for translation to in vitro assays for biological discovery and drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbed15b3d1cf440dd524e75b1029862768a9edc2" target='_blank'>
              A biofabricated 3D cancer-stroma tumor microenvironment model
              </a>
            </td>
          <td>
            S. Romanazzo, Peilin Tian, Gagan K Jalandhra, Riddhesh Doshi, J. Gooding, K. Kilian
          </td>
          <td>2025-09-23</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Organoid technology has emerged as a revolutionary tool in cancer research, offering physiologically accurate, three-dimensional models that preserve the histoarchitecture, genetic stability, and phenotypic complexity of primary tumors. These self-organizing structures, derived from adult stem cells, induced pluripotent stem cells, or patient tumor biopsies, recapitulate critical aspects of tumor heterogeneity, clonal evolution, and microenvironmental interactions. Organoids serve as powerful systems for modeling tumor progression, assessing drug sensitivity and resistance, and guiding precision oncology strategies. Recent innovations have extended organoid capabilities beyond static culture systems. Integration with microfluidic organoid-on-chip platforms, high-throughput CRISPR-based functional genomics, and AI-driven phenotypic analytics has enhanced mechanistic insight and translational relevance. Co-culture systems incorporating immune, stromal, and endothelial components now permit dynamic modeling of tumor–host interactions, immunotherapeutic responses, and metastatic behavior. Comparative analyses with conventional platforms, 2D monolayers, spheroids, and patient-derived xenografts emphasize the superior fidelity and clinical potential of organoids. Despite these advances, several challenges remain, such as protocol variability, incomplete recapitulation of systemic physiology, and limitations in scalability, standardization, and regulatory alignment. Addressing these gaps with unified workflows, synthetic matrices, vascularized and innervated co-cultures, and GMP-compliant manufacturing will be crucial for clinical integration. Proactive engagement with regulatory frameworks and ethical guidelines will be pivotal to ensuring safe, responsible, and equitable clinical translation. With the convergence of bioengineering, multi-omics, and computational modeling, organoids are poised to become indispensable tools in next-generation oncology, driving mechanistic discovery, predictive diagnostics, and personalized therapy optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c3effe5e0ade80f8d6b3e4aa8e2c7994edd86eb" target='_blank'>
              Organoids as Next-Generation Models for Tumor Heterogeneity, Personalized Therapy, and Cancer Research: Advancements, Applications, and Future Directions
              </a>
            </td>
          <td>
            Ayush Madan, Ramandeep Saini, Nainci Dhiman, Shu-Hui Juan, M. K. Satapathy
          </td>
          <td>2025-10-08</td>
          <td>Organoids</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits profound spatial heterogeneity driving therapeutic resistance, while the role of cancer-associated fibroblasts (CAFs) in orchestrating immunosuppressive niches remains incompletely defined. This study integrates single-nucleus RNA sequencing (snRNA-seq) and spatial transcriptomics (stRNA-seq) to map the cellular and molecular landscape of HCC. snRNA-seq identifies key cell populations-including fibroblasts, T_NK cells, and endothelial cells-using canonical marker genes. Spatial transcriptomics maps gene expression across tumor regions (core, invasive front, stroma) via the robust cell type decomposition (RCTD) algorithm. Immunofluorescence validates collagen deposition and POSTN spatial distribution, confirming T-cell exclusion patterns. The analysis identifies hypoxic metabolic myofibroblasts (hmmyCAFs) as central regulators of the tumor microenvironment. hmmyCAFs enrich at the invasive front, forming collagen-rich barriers that physically exclude CD8⁺ T cells. Simultaneously, they secrete POSTN to suppress immune checkpoint signaling and drive hypoxia-mediated glycolytic reprogramming of T-cell metabolism. Clinically, hmmyCAF activity and POSTN expression correlate with reduced progression-free survival and immunotherapy resistance. This multimodal study defines hmmyCAFs as triple architects of physical immunosuppression, molecular regulation, and metabolic remodeling. By linking collagen remodeling, POSTN-mediated checkpoint inhibition, and hypoxia-driven metabolic reprogramming to clinical outcomes, hmmyCAFs and POSTN may serve as potential indicators for evaluating the efficacy of immunotherapy in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2aba5ba124e6d224e40973627e744d5081476427" target='_blank'>
              Spatial Transcriptomics and snRNA-seq Expose CAF Niches Orchestrating Dual Stromal-Immune Barriers in Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Yingxue Li, Changxiang Huan, Haoting Sun, Wei Zhang, Zhen Guo, Chuanyu Li, Jia Yao, Zhiqi Zhang, Shizhe Yu, Qiongzhu Dong, Lunxiu Qin, Jinze Li, Lianqun Zhou
          </td>
          <td>2025-10-07</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Emerging clinical evidence highlights the importance of tumor infiltration of activated immune cells, particularly T and B lymphocytes, for the efficacy of immune-based cancer therapies. However, many patients show limited immune cell presence in the tumor microenvironment. High endothelial venules (HEVs) are specialized blood vessels that enable lymphocyte trafficking from circulation into lymph nodes or tumors. Histological analyses of clinical tumor samples indicate that HEVs are positioned near dense aggregates of B and T lymphocytes, known as tertiary lymphoid structures (TLS), underscoring their role in immune cell recruitment and anti-tumor immunity. The abundant development of TLS and HEVs is associated with clinical response to immune checkpoint inhibition therapy and cancer survival. Recently, we found that simultaneous activation of innate immune effectors, STING and lymphotoxin-β receptor (LTβR) using their agonists induces the formation of TLS and HEVs in TLS/HEV-free tumors. In the current study, mouse pancreatic ductal adenocarcinoma (KPC) tumors were grown in the dorsal skin fold window chamber (DSWC) and received STING/LTβR agonist combination therapy to monitor the real-time TLS/HEV development and dynamic tumor microenvironment in intravital microscopy (IVM). Transgenic mice with fluorescent red B cells and green fluorescent CD31+ blood vessels were infused with MECA-79 antibody (HEV marker) to visualize B cell trafficking through HEVs. Four-color confetti mice crossed with B cell-specific Cre mice, allowed us to analyze the clonal expansion of B cells expressing one of four colors, with daughter cells retaining their parent color. Our findings demonstrated that tumoral HEVs closely resemble those found in lymph nodes, characterized by fewer branches, longer segment lengths, and increased diameters compared to non-HEV tumor blood vessels, which typically exhibit chaotic patterns. Circulating B cells traveling through tumoral HEVs displayed significantly slower rolling velocities than non-HEV tumor vessels, likely due to enhanced integrin and L-selectin expression induced by the combination therapy. By repeated intravital imaging, we were also able to observe a quadruple increase in B cell number in TLS during the day 4-7 treatment time frame. Following adoptive transfer of splenic leukocytes from naïve donor mice, we found that these leukocytes preferentially homed to TLS-rich areas, with B cells constituting the majority of the recruited cells. Importantly, analyses using the confetti mouse model indicate that tumor infiltrating B cells underwent clonal expansion in TLS, with different TLS having different colors. Overall, our study demonstrates that HEVs are the gateways for B cell entry to tumors, and the intratumoral TLS formation is driven by localized clonal expansions of these B cells, resembling lymph node germinal center activities, which drive the development of humoral immunity against tumors.



 Gabrielle Rowe-Brown, Masanobu Komatsu. B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr PR-11.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c11ba138005b19c5e9d04c0e07daa4c79af69fe8" target='_blank'>
              Abstract PR-11: B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy
              </a>
            </td>
          <td>
            Gabrielle Rowe-Brown, Masanobu Komatsu
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gliomas represent the most frequent primary brain tumors in adults and are marked by considerable molecular and clinical heterogeneity. The 2021 WHO classification of central nervous system tumors identifies three major adult-type diffuse glioma subtypes: astrocytoma IDH-mutant, oligodendroglioma IDHmutant with 1p/19q-codeletion, and glioblastoma IDH-wildtype.1 Despite ongoing advances, treatment resistance and poor prognosis remain major challenges, largely due to intratumoral heterogeneity and the complexity of the tumor microenvironment.2 A promising yet underexplored avenue for patient-centered glioma profiling is the tumor secretome, which includes soluble factors, extracellular vesicles, and metabolites released by both neoplastic and stromal cells.3 Existing studies rely on in vitro models that fail to recapitulate the complexity of tumor-host interactions. To bridge this gap, we established an innovative, standardized ex vivo protocol for secretome isolation directly from glioma tissue. The protocol preserves microenvironmental cues from the operating room to laboratory analysis, ensuring spontaneous secretome release under controlled conditions. Feasibility and performance were assessed in a preliminary cohort of seven adult diffuse glioma patients. Transmission electron microscopy confirmed tissue structural integrity. Proteomic profiling identified 84 cancer-associated proteins within the secretome. Additionally, metabolomic analysis revealed a distinctive signature of small molecules. Notably, glioblastoma IDH-wildtype samples exhibited increased lactate levels, consistent with hypoxia-driven metabolism, and decreased N-acetylaspartate, indicating neuronal loss. Overall, this ex vivo secretome protocol offers a robust and physiologically relevant platform for glioma characterization, with potential applications in biomarker discovery, patient stratification, and the development of personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea6096ea6e8da3b1feda45c80168b298c559e43" target='_blank'>
              P64 | EX VIVO EXTRACTION OF GLIOMA SECRETOME: A NOVEL MICROENVIRONMENT-BASED PROTOCOL FOR METABOLIC AND MOLECULAR PROFILING
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Background and Aims The relationship between chronic liver disease and liver cancer remains poorly understood, and treatment options for advanced liver disease remain limited. This study aims to elucidate the dynamic evolution of cellular and molecular alterations from normal liver to diseased liver. Methods Single‐cell RNA sequencing was performed to profile the dynamic cellular variations from normal liver to diseased liver. Bioinformatics analyses were employed to assess tumour heterogeneity and the evolution of tumour microenvironment. Molecular biology validations were performed to investigate the findings. Results Through single‐cell RNA sequencing, we constructed a human liver landscape consisting of more than 130 000 single cells, from normal, cirrhotic tissue, primary liver cancer and paired adjacent tissues. We profiled both inter‐ and intra‐tumour heterogeneity of hepatocellular carcinoma and intrahepatic cholangiocarcinoma. A dual role of the pro‐fibrosis and pro‐carcinogenesis FABP5high macrophage subset was shown to be conserved throughout the progression of liver fibrosis and carcinogenesis; markers, spatial localization, origin, and functional pathways associated with FABP5high macrophages were identified through in vitro and in vivo validation. Metabolic reprogramming of these conserved FABP5high macrophages is implicated in the formation of cirrhotic and cancerous niches. Additionally, we also identified a subset of RGCC+COL4A1+ endothelial cells enriched in HCC tissues, which might experience endothelial‐mesenchymal transition and orchestrate the proliferation and invasion of cancer cells via angiogenesis. Conclusions Our findings provide insights into the evolution from chronic liver disease to liver cancer. These insights will contribute to the development of novel cell subset‐based therapeutics based on immunotherapy and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec8b08f4e01c5f59d84102994dc20c75fd98f8a8" target='_blank'>
              A Conserved FABP5high Macrophage Subset Promotes Fibrosis and Carcinogenesis in Advanced Liver Disease
              </a>
            </td>
          <td>
            Wenhao Ge, Lifeng Ma, Yu Huang, W. E., Shujing Lai, Bijun Cui, Jingjing Wang, Yutian Ji, Daiyuan Liu, Song Ye, Yang Kong, Jiarong Zhou, Chengxuan Yu, Yingsheng Wu, Qiang Sun, Dongkai Zhou, Wei Zhu, Yuan Ding, Xiaoping Han, Guoji Guo, Weilin Wang
          </td>
          <td>2025-08-28</td>
          <td>Liver International</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Simple Summary Hepatoblastoma (HB), the most common childhood liver cancer, often shows aberrant expression and/or mutations of the β-catenin (B), YAP (Y) and/or NRF2 (N) transcription factors and can be modeled in mouse livers by overexpressing combinations of these (e.g., BY, BN, YN and BYN). Because very few human or murine HB cell lines exist, genetically defined murine HB cell lines would be a significant advance. We describe here the generation of cell lines from primary BN and YN tumors. Unexpectedly, one BN line shows a remarkable ability to trans-differentiate into endothelial cells and can accelerate in vivo tumorigenesis and angiogenesis. These and previously derived BY and BYN lines have similar sensitivities to four drugs used to treat HB. Our generic approach for cell line derivation should allow for the generation of additional lines driven by less common drivers. A collection of such genetically defined and well-characterized cell lines will facilitate studies that are impractical to perform in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31da89c86be409fe01133803515bcf28ea7e0273" target='_blank'>
              Derivation of Genetically Defined Murine Hepatoblastoma Cell Lines with Angiogenic Potential
              </a>
            </td>
          <td>
            Keyao Chen, Ahmet Toksoz, Colin Henchy, Jessica Knapp, Jie Lu, S. Ranganathan, Huabo Wang, E. V. Prochownik
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Metastatic pancreatic adenocarcinoma is the dominant clinical presentation with a grim 3% 5-year survival rate. Over 80% of metastatic disease occurs in the liver and has the poorest outcomes. Overall, there is little mechanistic understanding of what promotes liver metastatic outgrowth and organotropism. Our recent work using spatial transcriptomics on unique matched primary tumors and metastases revealed distinct cellular ecosystems, noting reduced desmoplasia, high proliferation, spatially constrained metabolism, and heightened pro-tumorigenic myeloid infiltration and T-cell dysfunction at the invasive border of liver metastases. However, moving beyond these observations requires establishing causal links between molecular drivers and metastatic competence. Causal studies are limited by current model systems, the spontaneous genetically engineered mouse models (GEMM) that form the basis of pre-clinical studies do not sufficiently model the clinical metastatic reality with robust matched pancreas and liver tumors, and are further hampered with inconsistent metastatic rates and unpredictable progression for timed analysis. Syngeneic transplants of GEMM-derived cancer lines into wild-type mice provide a rapid pre-clinical model system of liver disease. However, the liver-metastatic rates varies between cell lines, even with identical driving mutations. We suspected that the unexplored mechanisms driving these differences in metastatic outgrowth present an opportunity to understand critical biology. Here we report the development a consistent model system of matched pancreatic and liver tumors using syngeneic cell lines with high and low tropism for liver metastatic outgrowth in C57Bl/6 mice. These lines are transplanted at low cell numbers to better allow the evolution of the site-specific tumor microenvironment and provide a reliable model system to examine both cancer-cell intrinsic and site-specific microenvironmental factors dictating liver outgrowth. Our observations of comparable pancreatic growth, successful metastatic growth in other organs (peritoneum or lung), and micro-metastatic lesions in the liver at early time points, suggests the liver-tropic differences fall within the ability of these cells to successfully outgrow in the liver microenvironment, rather than in vivo proliferative or extravasation differences. Comparison of gene expression between high and low liver-tropic cell lines identified several immune regulatory genes and a general increase in lipid metabolism, consistent with our published patient data. Finally, spatial quantifications of these lesions using a novel 46-plex murine immunotyping panel on FFPE tissues show similar suppressive immune cells at the interface of the tumor and normal liver as observed in our patient data, but evolves across differing lesion size, suggestive of a spatiotemporal progression. Altogether, this model system provides a robust, efficient pre-clinical platform to dissect spatiotemporal drivers of liver metastatic disease.



 Christina R. Larson, Jace Baines, Ayushi Mandloi, Meet Patel, Tuan Tran, Nailah Jones, Ateeq M. Khaliq, Christopher A. Risley, Robert S. Welner, Satwik Acharyya, Julienne L. Carstens. Robust Pancreatic Cancer Liver Metastatic Model System Reveals Cancer Cell Dependent Organotropism and Site-specific Tumor Microenvironment Regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288b7e540e23f2b8aa282e16a6f01e2dc86d80d0" target='_blank'>
              Abstract A048: Robust Pancreatic Cancer Liver Metastatic Model System Reveals Cancer Cell Dependent Organotropism and Site-specific Tumor Microenvironment Regulation
              </a>
            </td>
          <td>
            Christina R Larson, Jace Baines, Ayushi Mandloi, Meet Patel, Tuan Tran, Nailah Jones, A. Khaliq, Christopher A. Risley, Robert S. Welner, Satwik Acharyya, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background Mucosa-associated invariant T (MAIT) cells represent a unique population of innate-like T lymphocytes capable of detecting non-peptide antigens in the context of monomorphic antigen-presenting molecules. Due to their abundance in barrier tissues, reactivity to local inflammatory cues, and cytotoxic and regulatory functions, MAIT cells are poised to shape the dynamics of various tumor microenvironments. Growing evidence suggests that MAIT cells can exert protumor and/or antitumor effects in cancers arising from or metastasizing to mucosal tissues. However, MAIT cell roles in bladder cancer (BCa) remains unclear. Methods To begin to identify MAIT cells in BCa, we stained bladder tumor biopsies for T cell receptor (TCR) Vα7.2+ cells. We then refined a human MAIT cell signature, which enabled us to interrogate a bulk RNA sequencing dataset and conduct correlation analyses linking intratumoral MAIT cell abundance and mortality from BCa. To extend our work to an in vivo setting, we employed a clinically relevant mouse model in which Mr1+/+ B6-MAITCAST (MAIT-sufficient) and Mr1−/− B6-MAITCAST (MAIT-deficient) mice were exposed to N-butyl-N-(4-hydroxybutyl)nitrosamine, a chemical carcinogen associated with tobacco smoke. In additional experiments, MAIT cells were functionally removed through acetyl-6-formylpterin (Ac-6-FP) administration. Effector and regulatory cell types were phenotyped by flow cytometry, and BCa tumor burden and progression were assessed by MRI and/or H&E and Ki67 staining. Results TCR Vα7.2+ cells were readily detectable in several BCa biopsies, and our bioinformatic analyses correlated heavier MAIT cell presence in BCa tumors with poorer overall survival. Similarly, we found higher tumor burdens in Mr1+/+ B6-MAITCAST mice than in Mr1−/− or Ac-6-FP-treated animals. Bladder MAIT cells from tumor-bearing mice exhibited phenotypic MAIT17 bias based on transcription factors they harbored along with increased interleukin-17A and tumor necrosis factor-α production capacities upon stimulation. Finally, FoxP3+ regulatory T (Treg) cell frequencies were elevated in Mr1+/+ mouse bladder tumors, likely contributing to an immunosuppressive tumor microenvironment, a finding that could be recapitulated in our transcriptomic studies on human BCa. Conclusions MAIT cells are abundant in BCa tumor microenvironments where they potentiate Treg cell accumulation and play protumor roles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/904ce037c95ea168f087ef993f969eaddff0121a" target='_blank'>
              MAIT cells promote cancer progression and regulatory T cell accumulation in bladder tumor microenvironment
              </a>
            </td>
          <td>
            Inbar A Habaz, Xuejin Ou, Nicole I Wang, Angela Schincaglia, Valeryia Shydlouskaya, Agetha Mahendran, F. Martinez-Santiesteban, Dylan Doolabi, Yige Bao, Ahmad R Movasseghi, Julius Haruna, Nicholas E. Power, Timothy J Scholl, Melissa J Huynh, Alp Sener, S. M. Haeryfar, Dr S M Mansour Haeryfar
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Papillary thyroid carcinoma (PTC) is the most prevalent form of thyroid cancer, yet its cellular heterogeneity and prognostic determinants remain poorly defined. Here, we integrate two single-cell RNA sequencing datasets comprising 20 human thyroid samples to construct a high-resolution cellular atlas of PTC. We identify 29 distinct cellular subpopulations and delineate their composition, dynamics, and interactions in healthy versus tumor tissues. Notably, epithelial and monocyte populations were markedly expanded in PTC, whereas adaptive immune subsets such as B and T cells were diminished. Cell–cell communication analysis revealed enhanced intercellular signaling in the tumor microenvironment, with epithelial and endothelial cells receiving the strongest inputs. Among monocyte-specific transcriptional signatures, we identified 65 prognostic genes via univariate Cox analysis. A LASSO-derived 14-gene risk score robustly stratified patient outcomes, with CSGALNACT1 emerging as a key epithelial-specific, independent prognostic gene. Pseudotime analysis further supported its role in epithelial cell differentiation. Functional validation demonstrated that CSGALNACT1 promotes proliferation in PTC cell lines, suggesting a potential oncogenic function. Immune deconvolution across risk groups revealed substantial divergence in innate and adaptive immune infiltration, indicating a close interplay between tumor-intrinsic transcriptional programs and immune microenvironment remodeling. Collectively, our study provides a comprehensive single-cell framework for PTC, identifies a clinically relevant risk model, and highlights CSGALNACT1 as a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62fcd940763c0c7abf39d3fd346b8a069b2e5dc" target='_blank'>
              Comprehensive analysis of the papillary thyroid carcinoma identifies CSGALNACT1 as a proliferation driver and prognostic biomarker
              </a>
            </td>
          <td>
            Yu Zhao, Juan Zhang, Yong Yao, Linna Yu
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea7d79855ee47627896db43f29cfa10fcc98ce7" target='_blank'>
              Single-cell multiomic modelling of early metastatic events promoted by the extracellular matrix.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Jun Hyeong Lee, Junho Kang, Kyeonghui Kim, Youngwon Cho, Jihyun Park, Sang-Hyun Song, J. Kwon, Young-Joon Kim, Woong-Yang Park, Tae-You Kim, Jong-Eun Park, Pilnam Kim, Jung Kyoon Choi
          </td>
          <td>2025-09-15</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma (PDAC) exhibits a huge range of genomic plasticity due to prevalent genomic instability across tumors. Longitudinal whole genome sequencing of patients at primary diagnosis and metastatic disease progression have shown KRAS copy number amplification and allelic imbalance as drivers of classical to basal-like phenotypic switch, resulting in propensity for whole genome duplication, genomic instability and aggressive metastasis. High-plex spatial proteomics have revealed that primary tumors are phenotypically mixed, but the relationship between intratumorally distinct phenotypes and their genomics remain unclear as existing genomics relies on bulk captured tumors. Therefore, this drives the need to spatially resolve the intra-tumoral genomic heterogeneity associated with distinct tumor phenotypes. We leveraged whole-slide high-plex Imaging Mass Cytometry (IMC) to stain a phenotypically mixed primary tumor with 41 different markers covering the classical (AGR2, TFF1, CEACAM6) and basal (TP63, KRT5, S100A2, CAV1) phenotype, pan-immune markers, epithelial transcription factors (GATA6, FOXA2, PDX1) and stroma. Bulk whole genome sequencing (30X) reveals that this tumor had undergone losses of chr 1p, 6p, 9, 17 and 8, in addition to 12p focal amplification and whole genome doubling. To address the association of spatial phenotypes to regional genomics, IMC guided laser capture microdissection (LCM) was performed on a serial section aligned to 4 classical (C1-4), 4 basal (B1-4) and 4 hybrid-staining (H1-4) regions, followed by independent shallow whole genome sequencing (sWGS) at 2X coverage for each captured region (n = 12). Quality control, alignment and variant calling was performed utilizing standard whole-genome sequencing workflow. Integer copy number was determined through 100kb bins and segmentation. A full range of ploidy solutions (2N, 3N and 4N) was fit for each region using Absolute Copy Number Estimation (ACE) and the best fit was used for each region. We observed that in region C2, losses in many chromosomes (1p, 6, 9, 17, 18) but had no arm level amplifications, indicating a mitotic defect. At region C3, we observed a small increase in the 12p segment harboring the KRAS locus, suggesting that a sub-clonal event had occurred resulting in amplification. At region H3, a phenotypically mixed region, a strong 12p amplification was observed along with widespread genomic instability. Finally, at region B1, the whole genome duplication is observed. Our initial pilot reveals a stepwise, orchestrated genome evolution process in a phenotypically mixed primary tumor which was not obvious in bulk whole genome sequencing, highlighting the need for spatially informed regional capture of tumors to uncover phenotypically and potentially clinically important genomic alterations.



 Tiak Ju Tan, Deni Van. Sechie, Ferris Nowlan, Noor Shakfa, Elizabeth Sunnucks, Sibyl Drissler, Jenn Gorman, Michael J. Geuenich, Chengxin Yu, Grannie O. Kane, Faiyaz Notta, Rob Grant, Julie Wilson, Steve Gallinger, Kieran R. Campbell, Hartland W. Jackson. Imaging mass cytometry guided shallow whole genome sequencing for phenotype informed regional copy number profiling in a pancreatic tumor [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B113.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e0d82bf993f8cb23419e03138a40de013a09be1" target='_blank'>
              Abstract B113: Imaging mass cytometry guided shallow whole genome sequencing for phenotype informed regional copy number profiling in a pancreatic tumor
              </a>
            </td>
          <td>
            Tiak Ju Tan, Deni Van. Sechie, Ferris Nowlan, Noor Shakfa, Elizabeth Sunnucks, Sibyl Drissler, Jennifer L. Gorman, Michael J Geuenich, Chengxin Yu, Grannie O. Kane, F. Notta, Robert C. Grant, Julie M. Wilson, Stephen Gallinger, Kieran R. Campbell, H. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Glioblastoma relapse remains inevitable, placing these tumors among the most difficult to treat. Treatment challenges include inherent drug resistance, tumour’s plastic nature, and suppressive tumour microenvironment (TME). In other cancers, treatment-induced plasticity has been associated with the emergence of drug-tolerant persister cells, which survive therapy by undergoing reversible phenotypic changes towards resistant states. The role of persisters in glioblastoma remains to be unravelled. It is unclear to what extent the responses are patient- and treatment-specific, and how much they depend on the tumor microenvironment.



 We applied a cohort of molecularly characterized patient-derived models to investigate patient-specific responses to chemotherapeutics (Temozolomide, VAL-083). Treatment efficacy and its correlation with molecular features were assessed ex vivo using organoids and stem-like cultures as well as in vivo through orthotopic xenografts. To mimic clinical treatment schedules, we assessed functional responses in time, incorporating prolonged exposure to treatment and drug holiday period over several weeks. Analysis included tumor cell viability, cell cycle, DNA damage and repair, senescence and quiescence features. Phenotypic adaptation of glioblastoma and microenvironmental cells in vivo were analysed with MRI, single-cell RNA-seq, spatial transcriptomics and IHC.



 Assessment of drug efficacy across a cohort of patient-derived models revealed resistant versus adaptive responder groups, which were treatment-specific. While resistant cells did not show functional changes in cell viability and cell cycle upon prolonged drug exposure, sensitive cells show a biphasic growth curve characteristic of persister cells. The adaptation towards the least proliferative phenotype was reversible during drug holiday period, further confirming a role of a transient drug-tolerant state. Chemotherapeutic treatment in orthotopic xenografts partially reduced tumor growth in vivo and activated apoptosis and DNA repair pathways in glioblastoma cells. These changes coincided with an increased infiltration of tumor-associated macrophages.



 Our findings highlight the critical roles of both tolerance and intrinsic resistance mechanisms. The responder groups appear to be specific to the type of treatment and the unique characteristics of each patient’s tumor, emphasizing the need to evaluate treatment-induced plasticity in a patient-specific context. Glioblastoma cells are further supported by the adaptation of the microenvironment towards a tumor-supportive crosstalk.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ea2a43463b7a7c67d446cbd650a87fb74e2953" target='_blank'>
              P01.18.B MODELING GLIOBLASTOMA TREATMENT RESPONSES: INSIGHTS INTO TUMOR PLASTICITY AND THE ROLE OF THE TUMOR MICROENVIRONMENT
              </a>
            </td>
          <td>
            M. de Lucas Sanz, H. Allahverdi, B. Nosirov, P. Moreno Sánchez, M. Baroni Milan, A. Lipsa, M. Rezaeipour, C. Dording, D. Brown, J. M. Langlands, K. Grzyb, Y. Yabo, J. Zhang, D. Heiland, A. Michelucci, P. Nazarov, A. Gołȩbiewska
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Chimeric antigen receptor T-cell (CAR-T) therapy has emerged as a breakthrough immunotherapy for multiple cancers, yet its efficacy remains limited in glioblastoma due to tumor heterogeneity, immune-evasion of the microenvironment, and rapid T-cell dysfunction. The aim of our study is to investigate the spatially differential expression of CAR-T targets in glioblastoma. We want to focus on differential expression of CAR-T targets in tumor cell subtypes, differences between tumor core and infiltration zone as well as reactive immune zones.



 We analyzed spatial transcriptomic data from 62 glioblastoma specimens using SPATA2 software, augmented by advanced computational tools such as scVI (data integration), Cell2Location (single-cell deconvolution), graph neural networks, gene set enrichment analysis (GSEA), and spatial annotation screening (SAS). CAR-T antigens were chosen from existing published clinical and pre-clinical CAR-T cell trials.



 Spatial mapping revealed a pronounced gradient of NPC-like cells with a higher abundance of NPC-like cells in the infiltration zone and resection margins—regions critical for residual disease persistence after resection. Clinical CAR-T targets (e.g., ERBB2, PTPRZ1) were highly expressed in AC-like and OPC-like dominated niches while nearly absent in NPC-rich areas, suggesting a blind spot for current CAR-T therapies. Single-cell deconvolution further correlated NPC-like dominance to increased T-cell infiltration, while GSEA and clonal dynamics analysis of the T-cell compartment suggested downregulated functioning subclones in NPC-rich spatial neighborhoods, underscoring potential immune evasion mechanisms in these regions.



 Our study exposes a fundamental limitation of current CAR-T designs: their inefficacy against NPC-like cells that dominate the infiltration zone and therefore the post-surgical glioblastoma ecosystem. By pinpointing spatial and phenotypic resistance hubs, we provide a roadmap for refining CAR-T targets to address this gap, potentially enhancing adjuvant immunotherapy for glioblastoma. Protein-level validation of key findings is currently ongoing to further substantiate the results.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe7508f42aeb973a2f44bba592abe40968ca576d" target='_blank'>
              P04.13.B SPATIAL HETEROGENEITY OF CAR-T ANTIGEN EXPRESSION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Y Kong, J. Zhang, F. Sahm, G. Kastner, M. Mühlbauer, E. Grabis, P. Heinrichs, J. Straehle, R. Sankowski, R. Roelz, J. Beck, D. Heiland, N. Neidert
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2affd9966e7d74eae7ebff40360fcc4663c6bf18" target='_blank'>
              Clonal expansion of cytotoxic CD8âº T cells in lecanemab-associated ARIA
              </a>
            </td>
          <td>
            Lance A. Johnson, Kai Saito, Akhil Pallerla, Jessica L Funnell, Ashley R. Ezzo, Chelsea L. Song, Douglas A Harrison, Noah J. Norton, Lauren C. Moore, L. Eldik, D. Fardo, Greg E. Cooper, Josh M Morganti
          </td>
          <td>2025-09-21</td>
          <td>None</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) chemotherapy regimens are largely uniform, tailored mainly to patient fitness or rare genomic alterations. This strategy fails to capitalize on the disease’s substantial molecular heterogeneity, including variations in copy number alterations, immune infiltration, stromal composition, and cancer cell phenotypes. These features are not independent; they reflect interconnected biological processes and co-evolving pathways that shape tumour progression. Multi-omic integration is therefore essential—not only to capture the complexity of PDAC biology, but also to clarify the molecular basis of tumours classified as ‘intermediate’ by current stromal and cancer taxonomies, thereby enabling rational therapeutic targeting. To this end, we performed imaging mass cytometry on three serial sections of a PDAC tissue microarray (221 resected tumours, ∼4 cores each), generating >800 multiplexed images (40–43 channels) focused on epithelial, immune, or stromal biomarkers. We profiled 76 immune and stromal cell types and states (hypoxic, proliferative, apoptotic, under tension), as well as six tumour phenotypes defined by expression of epithelial transcription factors (GATA6, FOXA2, PDX1), classical markers (AGR2, TFF1, CEACAM6), basal markers (TP63, KRT5, S100A2, CAV1), and other PDAC-associated proteins (S100A4, MMP7, MUC16). These six cancer cell types captured the classical and basal PDAC signatures, along with four discrete “intermediate” states with distinct associations to stromal heterogeneity, RNA subtype (n = 92), tumour ploidy (n = 182), and patient outcome. These phenotypes are also detectable in unmatched single-cell RNA-seq data (n = 163), though with more overlap in marker expression. Using matched 30X whole genome sequencing (n = 182), we identified mutations and copy number alterations linked to shifts in cancer and stromal cell phenotypes, raising the question of which molecular axis best informs clinical prognosis. To address this, we applied a modified version of Stabl, a Lasso-based machine learning approach, to compare across omic layers and identify features most strongly associated with overall survival. Per-modality analysis showed that models incorporating omics outperformed those based on clinical features alone, with imaging data slightly outperforming genomics. Cross-omic integration revealed several prognostically relevant copy number aberrations, fibroblast phenotypes, and cancer cell states. By consolidating information on tumour phenotypes, stromal niches, and genomic alterations, this work aims to focus future drug discovery on the most clinically impactful molecular features in PDAC.



 Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steve Gallinger, Hartland W. Jackson. Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B116.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5cd553302915b28eaa98522e24327f9aa96cbdc" target='_blank'>
              Abstract B116: Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival
              </a>
            </td>
          <td>
            Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, , Barbara Gruenwald, A. Borgida, Edward L. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, H. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Introduction A comprehensive characterization of immune cells within the tumor microenvironment (TME) of cervical squamous cell carcinoma (CSCC) is essential to advance understanding of tumor biology and guide immunotherapy development. Methods Using single-cell RNA sequencing, we analyzed 14,441 immune cells isolated from tumor tissues and paratumor tissues of three CSCC patients. By integrating data on tumor suppressors, oncogenic factors, cytokines, and chemokines, we performed differential gene expression analyses across immune populations. Results This analysis identified nine major immune cell subsets, including CD8+ T cells, regulatory T cells (Tregs), natural killer/T cells (NK/T cells), B cells, plasmacytoid dendritic cells (pDCs), and macrophages. Notably, elevated CCR7 expression in exhausted CD8+ T cells was associated with improved prognosis, suggesting it as a preliminary candidate for an immunoregulatory target that warrants further investigation. Additionally, CCL5 levels were elevated in Tregs, indicating a possible involvement in their recruitment to the tumor site that requires further validation. Furthermore, differential gene expression analysis identified candidate genes for further mechanistic investigation. Discussion This systematic comparison of the TME between tumor and paratumor tissues reveals dynamic changes in the immune landscape, providing insights and suggesting potential targets for further study to enhance understanding and treatment of CSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37a843f84193a8f20739341bcfd74ef25632d521" target='_blank'>
              Profiling of immune cell subsets and functional characteristics of cervical cancer based on single cell RNA sequencing
              </a>
            </td>
          <td>
            Yue Yuan, Dejun Sun, Mingyue Yang, Ying Xia, Chunli Yuan, Xiaosong Wang
          </td>
          <td>2025-09-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is characterized by extensive intratumoral heterogeneity and complex interactions within the bone marrow microenvironment, yet the cellular and molecular drivers of treatment resistance remain poorly defined. Protein tyrosine phosphatase receptor gamma (PTPRG) has emerged as a candidate tumor suppressor in various malignancies by antagonizing proliferative and survival signaling, but its functional and prognostic relevance in MM has not been established. Methods We analyzed 103,171 single‐cell transcriptomes from 18 MM samples (10 optimal responders [OR] and 8 suboptimal responders [SOR] to bortezomib–melphalan–prednisone) to investigate cell‐type composition, malignant plasma cell subclusters, and tumor–microenvironment crosstalk. InferCNV was used to distinguish malignant plasma cells, which were further reclustered and correlated with bulk prognostic phenotypes. Differential expression, pathway enrichment, transcription‐factor activity, pseudotime trajectory, and ligand–receptor interaction analyses were performed. Finally, bulk datasets (GSE9782, GSE2658, MMRF-CoMMpass) and in vitro knockdown assays in U266 and NCI-H929 cells were used to validate the prognostic and functional role of PTPRG. Results Eleven major cell types were annotated, with plasma cells, T/NK cells, and CD14+ monocytes predominating; SOR samples exhibited an expanded plasma‐cell fraction and reduced T/NK, CD14+ monocyte, pre-B, and HSPC populations. Among 35,944 malignant plasma cells, five subclusters were defined; one subcluster (MalPlasma3) was enriched in SOR samples and harbored 93.1% of cells associated with poor survival. MalPlasma3 and “worse‐survival” cells showed activation of stemness, E2F/MYC targets, and G2M checkpoint pathways, driven by transcription factors E2F8, E2F7, FOXM1, E2F1, and TIMELESS. Pseudotime analysis revealed a bifurcating differentiation toward a resistant phenotype, accompanied by upregulation of cell‐cycle and proliferation modules. In the OR group, enhanced cytotoxic features in NK, effector, and naïve T cells, along with IGF1–IGF1R and IFNG–IFNGR signaling, suggested a supportive microenvironment. In contrast to the known role as a tumor suppressor in solid and hematologic cancers, our integrative analyses identified PTPRG among seven stemness‐related genes upregulated in MalPlasma3 and poor‐survival cells, which was echoed in the observed reduced cell viability and increased apoptosis in MM cell lines following siRNA‐mediated PTPRG knockdown. Conclusions This single‐cell multi‐omic dissection implicates a proliferative, stem‐like MalPlasma3 subcluster and identified PTPRG as a key mediator of drug resistance and poor outcome in MM, offering novel prognostic biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b1aa46cace7effec1d51940ecccc2bdb09fa3e" target='_blank'>
              Single-cell profiling uncovers PTPRG-driven stemness in malignant plasma cells and signatures of treatment failure in multiple myeloma
              </a>
            </td>
          <td>
            Jiewen Tan, Jinman Zhong, Yueping He, Yunman Xu, Chang Chen, Dan Xiong
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite the availability of numerous approved immunotherapies for various cancers, durable progression-free survival remains relatively uncommon among patients with advanced cancer. As research into immunotherapy intensifies, the heterogeneity and complexity of the tumor microenvironment (TME) have emerged as critical determinants of treatment response and a major obstacle to understanding tumor resistance mechanisms. Recent advances in spatially resolved transcriptomics (SRT) enable transcriptome-wide measurement of gene expression while preserving essential spatial information, which supports the characterization of the features of the TME. This review outlines key challenges in cancer immunotherapy and emphasizes the importance of the TME in determining therapeutic efficacy. SRT strategies suitable is systematically introduced for TME profiling, comparing their relative advantages and limitations. Additionally, bioinformatics tools and methods used to reconstruct spatial tissue patterns are discussed from SRT data. Furthermore, how SRT is explored can enhance the understanding of the complexity and heterogeneity of the TME, facilitate the identification of novel therapeutic targets, and advance biomarker discovery to refine personalized treatment strategies for diverse cancer patients. Finally, future perspectives on the application of SRT in cancer immunotherapy and addressing the associated challenges are provided.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53e788cfc2c6f7fa7f419ffa32b647da6da2a8f3" target='_blank'>
              Spatially Resolved Transcriptomics: Revealing Tumor Microenvironment Heterogeneity to Advance Cancer Immunotherapy.
              </a>
            </td>
          <td>
            Dan Liu, Lu Xiao, Yuyuan Wu, Chensirong Yue, Mengmeng Li, Yuanpei Sun, Chao Yang
          </td>
          <td>2025-09-04</td>
          <td>Small methods</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="


 Glioblastoma tumors are characterized by a prominent infiltration of tumor-associated macrophages (TAMs), constituting up to 30-40% of the cellular make-up. These cells typically harbor immunosuppressive features which, in conjunction with T cell paucity, contribute to a tumor microenvironment (TME) uniquely refractory to immunotherapy response. Therefore, due to their critical role in tumor progression and therapy response, TAMs represent a promising therapeutic target within the brain TME.



 To address the challenge of macrophage heterogeneity in glioblastoma, we conducted comprehensive multi-omics analyses including scRNAseq, spatial transcriptomics, ATACseq and lipidomics, that identified distinct TAM subsets within tissue-resident microglia and infiltrating monocyte-derived macrophage (MDM) pools.



 We discovered a pro-tumorigenic and immunosuppressive subset common in both cell lineages characterized by high lipid metabolic activity which we termed lipid-laden macrophages (LLMs). These cells engulf and recycle lipid-rich myelin debris to fuel cancer cell proliferation and thereby promote malignancy. Beyond these functions, LLMs also exhibit immunosuppressive features that are induced by myelin debris uptake. Next, we aim to investigate the relationship between myelin-induced metabolic pathways and the subsequent immunosuppressive phenotype of LLMs. To accomplish this, we will employ a genetic screen on primary murine macrophages using a CRISPR knockout library. This approach will target the cellular metabolome and surfaceome, with the goal of rewiring TAMs toward a more immunogenic phenotype. To enable selective targeting of LLMs, we developed lipid nanoparticle (LNP)-based approaches. Our findings demonstrate that LNPs efficiently deliver cargo to macrophages in the TME, with formulations that can be tailored for enhanced cell type-specific uptake.



 We identified myelin as a key molecule fostering a tumor-permissive brain TME. We are currently harnessing genetic screening together with the development of cell type-specific LNPs which will enable interventions targeting the immuno-metabolic axis, potentially advancing new treatment strategies for glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8404fd7f21668b88fc96f4132337aef2254ebb3" target='_blank'>
              P02.04.A UNCOVERING AND TARGETING METABOLIC REGULATORS OF IMMUNOSUPPRESSIVE LIPID-LADEN MACROPHAGES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            M. Farber, T. van Schaik, D. Aschmann, M. Boon, D. Kloosterman, J. Erbani, L. Akkari
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="PURPOSE
The Tumor Microenvironment (TME) has a vital role in cancer survival and progression, and may play roles in drug resistance and immune escape. To date, few studies have detailed the TME of Ewing sarcoma (EwS).


EXPERIMENTAL DESIGN
We performed spatially resolved transcriptomics of primary treatment-naïve EwS tumor biopsies from patients with or without clinical metastasis, complemented by high-plex spatial proteomic analysis.


RESULTS
We discovered greater stromal enrichment in localized EwS primary tumors compared to metastasis-associated EwS primary tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumor role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling connectivity, suggesting a potential immune-evasive mechanism.


CONCLUSIONS
In addition to the immune recruitment role of tumor-associated stroma, our findings provide spatial insight into the TME of EwS and provide a rationale for the preclinical investigation of MIF as a potential target for Ewing sarcoma immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f9c77e0153cfc111b2db6ef0b69632696638603" target='_blank'>
              Spatial profiling identifies tumor-associated stroma enrichment and MIF as potential immunotherapy targets in primary Ewing sarcomas.
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-09-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Cholangiocarcinoma (CCA) is one of the most lethal cancers, characterized by molecular heterogeneity and treatment resistance. To uncover new biological signals and therapeutic opportunities, we employed multiomic characterization combined with machine learning.


APPROACH AND RESULTS
We profiled all anatomical CCA subtypes using whole exome sequencing, mRNA sequencing, and proteome/phosphoproteome analysis. Integrative dimensional reduction revealed RNA, protein, and phosphoprotein features driving tumor heterogeneity, enabling clustering. Machine learning algorithms identified molecular features for each cluster and mapped external datasets and patient-derived xenograft (PDX) models onto these clusters. Kinase enrichment analysis highlighted targetable kinases active in each cluster. In vivo validation was performed in cluster-specific PDX models using the selective TNK1 inhibitor, TP-5801. We identified three molecular clusters with distinct pathway characterization: immunomodulatory (cluster 1), metabolic (cluster 2), and gene regulation/cellular fate (cluster 3). Cluster assignment was independent of anatomic subtype but correlated with overall survival following curative-intent resection. We also identified multiomic features and pathways linked to overall survival and lymph node metastases, crucial for patient treatment selection. Kinase enrichment analysis pinpointed TNK1 as a highly active kinase in the metabolic cluster. Treatment with TP-5801 significantly reduced tumor growth in a metabolic PDX model, but not in models representing the other clusters. Combining internal data with publicly available datasets, we identified the immunomodulatory cluster as most responsive to gemcitabine/cisplatin therapy, confirmed in vivo using cluster-specific PDX models.


CONCLUSIONS
Integrated multiomic characterization provides translational insights by defining unique molecular subtypes associated both with therapeutic response and overall clinical outcomes. This approach identified TNK1 as a previously unrecognized therapeutic target in a defined subset of CCA tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c4fb881773081058b9b306ba40063572d97487b" target='_blank'>
              Multiomics combined with machine learning defines unique molecular subtypes of cholangiocarcinoma and identifies TNK1 as a therapeutic target.
              </a>
            </td>
          <td>
            Dong-Gi Mun, Erik Jessen, Jennifer Tomlinson, D. Carlson, R. Budhraja, R. Alva-Ruiz, A. Abdelrahman, Ryan Watkins, Lindsey Gregory, Chantal McCabe, Chen Wang, Rondell P. Graham, Kathryn Woods, Martin Golkowski, Marshall Baker, Greg J. Gores, S. Ilyas, Caitlin B Conboy, Ellen L. Larson, Jack W Sample, Enis H Ozmert, Richard K Kandasamy, M. Borad, Lewis R Roberts, Joshua Andersen, Akhilesh Pandey, R. Smoot
          </td>
          <td>2025-09-29</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Introduction Single-cell multi-omics has transformed T-cell biology by enabling the simultaneous analysis of T-cell receptor (TCR) sequences, transcriptomes, and surface proteins at the resolution of individual cells. These capabilities are critical for identifying antigen-specific T-cells and accelerating the development of TCR-based immunotherapies. Methods Here, we introduce TCRscape, an open-source Python 3 tool designed for high-resolution T-cell receptor clonotype discovery and quantification, optimized for BD Rhapsody™ single-cell multi-omics data. Results TCRscape integrates full-length TCR sequence data with gene expression profiles and surface protein expression to enable multimodal clustering of αβ and γδ T-cell populations. It also outputs Seurat-compatible matrices, facilitating downstream visualization and analysis in standard single-cell analysis environments. Discussion By bridging clonotype detection with immune cell transcriptome, proteome, and antigen specificity profiling, TCRscape supports rapid identification of dominant T-cell clones and their functional phenotypes, offering a powerful resource for immune monitoring and TCR-engineered therapeutic development. TCRscape can be found at https://github.com/Perik-Zavodskii/TCRscape/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eefe820a4c406c3e6561963250df919c0c97ebc" target='_blank'>
              TCRscape: a single-cell multi-omic TCR profiling toolkit
              </a>
            </td>
          <td>
            R. Perik-Zavodskii, O. Perik-Zavodskaia, M. Volynets, S. Alrhmoun, Sergey Sennikov
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 We present a novel integration of data-driven single-cell analysis with mechanistic mathematical modeling to predict the impact of cancer-associated fibroblasts (CAF) on pancreatic ductal adenocarcinoma (PDAC) invasion. Bioinformatic analyses of high-throughput single-cell and spatial molecular assays give us access to human tumor gene expression at the time of measurement, enabling direct characterization of the human tumor microenvironment (TME), complementing our understanding of cell behavior gained from preclinical models. Notably, applying our Bayesian non-negative matrix factorization algorithm CoGAPS to single-cell RNA-seq data from PDAC allowed us to associate a gene program of concurrent epithelial-to-mesenchymal transition (EMT) and inflammation in epithelial cells with the presence of CAFs, validated through organoid co-culture experiments. We hypothesize that this gene program represents a mechanism by which CAFs promote invasive behavior in neoplastic cells over time. Testing this hypothesis requires inference of temporal changes, which is not possible with single-timepoint data collection in genomics data. In contrast to data-driven bioinformatics, mechanistic mathematical modeling can capture processes that happen over time and make predictions about a system. Agent-based models (ABMs) are one form of such mechanistic mathematical models that are well suited for temporal modeling of cellular phenotypes. We developed new software for ABMs that abstract cells into software agents with individual states and their own rules of behavior from single-cell and spatial molecular assays for temporal inference. We apply this technique to understand how CAFs in the PDAC microenvironment support the series of phenotype transitions comprising tumor progression and invasion, by encoding the CAF-induced ability of tumor cells to migrate and a cessation of proliferation in tumor cells, induced by CAF-secreted, non-diffusable substrate as inferred from our genomics analysis. While this provides a virtual framework to test the impact of varying CAF density on tumor cell invasion, it initially lacked integration with the spatial architecture of human tumors. We then apply the ABM to human PDAC spatial transcriptomics data to forecast tumor behavior over time in a spatially-informed setting, varying the hypothesized mechanisms. This revealed that a substrate-driven mechanism results in a broad mesenchymal-like zone surrounding epithelial-like outgrowths, while a cell contact-mediated mechanism results in a single-cell-wide boundary surrounding more significant outgrowth. When we compared these to real tumor compositions, this favors the explanation that secreted CAF substrate drives EMT in PDAC neoplastic cells, which we then validate in organoids with CAF conditioned media. Our work demonstrates how integrating ABMs with genomic and spatial data enables testing of mechanistic hypotheses about tumor-stroma interactions and provides deeper insights into PDAC microenvironments, potentially informing strategies to intercept disease progression.



 Jeanette I. Johnson, Daniel R. Bergman, Tamara Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Yoseph W. Dance, Jacob T . Mitchell, Elizabeth M. Jaffee, Neeha Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, Ashley L. Kiemen, Luciane T. Kagohara, Young H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O'Brien, Elana J. Fertig. Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d57a26173bbcb8bbded169c7c20718d2482d35" target='_blank'>
              Abstract A025: Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion
              </a>
            </td>
          <td>
            J. I. Johnson, Daniel Bergman, T. Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Y. Dance, Jacob T. Mitchell, Elizabeth M. Jaffee, N. Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, A. Kiemen, Luciane T. Kagohara, Y. H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O’Brien, E. Fertig
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and highly aggressive malignancy with limited therapeutic options and poor prognosis. The paucity of clinical specimens and lack of established experimental models have hindered a comprehensive understanding of its cellular heterogeneity and tumor microenvironment.We performed single-cell RNA sequencing on SCCE samples, and integrated them with publicly available scRNA-seq datasets from esophageal squamous cell carcinoma (ESCC), esophageal adenocarcinoma (EAC), and adjacent normal tissues (NT) from ESCC and EAC cases. An integrative transcriptomic analysis was conducted to identify cell types, infer malignant states, reconstruct differentiation trajectories, evaluate immune landscapes, and investigate fibroblast subtypes and cell–cell communication networks.SCCE tumors were characterized by a predominance of malignant epithelial cells and exhibited a profoundly immunosuppressed phenotype, with reduced immune infiltration and widespread downregulation of immune checkpoint genes. Malignant epithelial cells showed pronounced chromosomal instability and were classified into three transcriptionally distinct subtypes with divergent differentiation trajectories. The tumor microenvironment featured a complex stromal compartment, with enrichment of extracellular matrix fibroblasts (eCAFs) characterized by elevated ELF3 regulatory activity, and collagen-driven signaling predominantly mediated by inflammatory CAFs (iCAFs). SCCE also showed the most intricate cell–cell communication network among esophageal cancer subtypes.Our single-cell atlas offers a detailed view of the cellular heterogeneity and microenvironmental complexity of SCCE, highlighting its distinct tumor architecture, immune exclusion, and stromal reprogramming. These findings provide a valuable resource for understanding SCCE biology and form a basis for future mechanistic and exploratory biological investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db8bbf656956c2fda0265e682ad202bf970eea8" target='_blank'>
              Single-cell transcriptomic analysis reveals epithelial and microenvironmental heterogeneity in small cell carcinoma of the esophagus
              </a>
            </td>
          <td>
            Xiaolei Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Tumor heterogeneity impacts invasive behaviors, treatment response, and clinical outcomes in triple-negative breast cancer (TNBC). However, this heterogeneity remains incompletely characterized. This study aims to utilize multi-scale data to investigate inter-tumoral heterogeneity and identify potential TNBC biomarkers. Methods Single-cell RNA expression profiles were analyzed using R packages. Specifically, the infercnv, Pyscenic, GeneNMF, SCP, Vector, CellChat, and hdWGCNA packages were employed to identify malignant cells and characterize heterogeneity in transcription factors, metaprograms, lineage evolution, developmental trajectories, cell–cell interactions, and co-expression networks. Bulk RNA datasets were incorporated to assess the prognostic value of cell clusters and candidate genes. G Protein Subunit Alpha 15 (GNA15) expression was determined via reverse transcription–quantitative PCR (RT–qPCR) and immunohistochemistry. Cell functional assays were performed to evaluate proliferation, migration, and invasion capabilities. Results A total of 14,335 malignant cells were isolated from epithelial cells across 15 single-cell RNA samples. Six tumor cell clusters were identified, which exhibited distinct prognoses, biological functions, driver transcription factors, and co-expression networks. Notably, the S2 cluster demonstrated association with multiple malignancy-related pathways and inferior survival outcomes. GNA15 emerged as the S2 cluster hub gene. In vitro experiments confirmed that GNA15 knockdown significantly attenuated proliferation, migration, and invasion in TNBC cell lines. Conclusions Our study comprehensively delineated TNBC tumor cell heterogeneity and established the critical role of GNA15 in TNBC progression. These findings enhance the understanding of TNBC heterogeneity and provide a theoretical foundation for TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e149422e4d37b7c64ca2e5cbee9d7c8957b702e6" target='_blank'>
              Multiomics profiling identifies the poor prognostic role of a tumor cluster with GNA15 overexpression in triple-negative breast cancer
              </a>
            </td>
          <td>
            Guixin Wang, Jun Cao, Chenglu Lu, Yu-Peng Cao, Shuo Wang, Ziyi Chen, Zhaohui Chen, Yingxi Li, Yue Yu, Yao Tian, Xin Wang
          </td>
          <td>2025-09-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Background Penile squamous cell carcinoma (PSCC) is a rare yet potentially lethal malignancy, often resulting in devastating disfigurement, with a 5-year survival rate of only ∼50%. Human papillomavirus (HPV) infection is implicated in approximately half of PSCC cases and is associated with improved clinical outcomes; however, the underlying mechanisms remain poorly understood. Methods To elucidate HPV-associated differences in the tumor microenvironment, we performed single-cell RNA sequencing on tumors from 11 treatment-naïve PSCC patients, analyzing a total of 52 980 single cells. Unsupervised clustering identified 49 distinct cellular clusters across immune and stromal compartments. Results HPV-positive tumors exhibited an increased abundance of mast cells and a reduction in the proliferative macrophages subpopulation compared to HPV-negative tumors. Notably, CD8+ T cells in HPV-positive PSCC expressed lower levels of immune checkpoint molecules, suggesting a less exhausted immune state. Conversely, TIGIT and its ligands were significantly enriched in HPV-negative tumors, potentially fostering an immunosuppressive niche. Conclusion Collectively, our study delineates the single-cell landscape of PSCC and highlights distinct tumor microenvironment remodeling associated with HPV status, suggesting that the reduced immunosuppression in HPV-positive tumors may underlie their more favorable prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d170eff0ce2ddd0035f0702533123e38640014" target='_blank'>
              Single-cell transcriptomic profiling reveals distinct tumor microenvironments in HPV-associated penile squamous cell carcinoma
              </a>
            </td>
          <td>
            Lianbang Zhu, Deyun Shen, Jiahao Zhou, Chen Cheng, Zhiyao Xu, Yong Liang, Wen Pan, Tao Tao
          </td>
          <td>2025-09-01</td>
          <td>Precision Clinical Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) refer to ectopic lymphoid aggregates that form in non-lymphoid tissues at sites of chronic inflammation including cancers. TLSs have been recognized as significant predictors of the efficacy of immune checkpoint blockade (ICB) therapies and have the potential to elicit robust anti-tumor immune response. However, recent studies have revealed substantial heterogeneity in TLSs across different individuals and cancer types, which directly impacts the effectiveness of anti-tumor immunity. Concretely, the maturation status, localization, and density of TLSs profoundly influence the dynamic interactions among immune cells within these structures, potentially leading to adverse effects. This review provides an in-depth exploration of how the heterogeneity of TLSs influences cellular composition and immune dynamics, with the objective of influencing the efficacy of ICB therapies and modulating prognostic prediction accuracy. Additionally, the potential of combining TLSs with other biomarkers for predicting anti-tumor immunity outcomes is further investigated, alongside the introduction of advanced technologies for evaluating TLS heterogeneity. Collectively, these analyses aim to advance the understanding of TLS heterogeneity and facilitate its translation into clinical and translational medicine applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3182b4ad88a2ceacb5b488328272b01f2977fe0" target='_blank'>
              Dissecting Tertiary Lymphoid Structures in Cancer: Maturation, Localization and Density
              </a>
            </td>
          <td>
            Guang-Liang Su, Meng‐Jie Zhang, Hao Li, Zhijun Sun
          </td>
          <td>2025-08-30</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Pancreatic intraepithelial neoplasms (PanINs), precursor lesions of pancreatic ductal adenocarcinoma (PDAC) that are abundant healthy individuals, rarely progress to cancer despite universally possessing many of the driver alterations associated with malignancy. Recent single cell profiling efforts of PanIN progression in vivo have described marked heterogeneity in chromatin accessibility and transcriptional identity within these neoplasias, but it remains unclear how discrete epigenetic factors play a role in shaping tumorigenesis, or what distinguishes rare neoplastic cells capable of seeding invasive carcinomas from their benign neighbors. Given that more than 80% of patients with PDAC present with unresectable or metastatic disease, further studies on the molecular features of PanIN progression are needed to enable methods of early detection and intervention. Unfortunately, owing to their microscopic size and challenges associated with the isolation of PanINs from surrounding normal epithelium and stroma, it is particularly difficult to address these knowledge gaps in vivo. By generating complex murine derived organoid models, we have built a system that allows us to faithfully reconstitute the evolution from benign PanIN to PDAC in vitro, as well as in vivo through orthotopic implantation into syngeneic mice. As they are easily perturbable, readily generate large amounts of biomass, and closely resemble the stages of PDAC initiation in vivo, these models can be leveraged to directly assay how chromatin remodelers influence malignant transformation. Further, pairing these organoids with our highly multiplexed, transcribable DNA barcoding system and single cell epi multiomics will enable us to study the evolutionary dynamics, chromatin landscapes, and transcriptional identities of clonal populations of PanIN organoids as they progress to cancer. Through these studies, we will gain unprecedented insights into the underpinnings of PDAC initiation, and thus may open up new avenues for the early detection of tumorigenesis.



 Connor J. Hennessey, Minh T. Than, Ben Z. Stanger, Sydney M. Shaffer. Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90b593e2ad7a725999b9d873ea37a620072eeba" target='_blank'>
              Abstract B106: Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models
              </a>
            </td>
          <td>
            Connor J. Hennessey, Minh T Than, Ben Z. Stanger, Sydney M. Shaffer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Endothelial cells (ECs) play complex roles across tissues and vessel types. Yet, systematic investigations of EC heterogeneity in the combined context of vessel type and tissue microenvironment are still largely lacking. We integrated over three million single cells of scRNA-seq datasets in 15 human tissues and found that ECs in some tissues (e.g., heart and kidney) exhibited greater tissue specificity, while others displayed more substantial vessel specificity. We developed a computational pipeline to analyze cell-cell communications (CCC) mediated by metabolites or proteins to explore microenvironmental regulation. Interestingly, our results showed that CCC events involving ECs varied vastly across tissues, highlighting tissue-specific EC interactions. Using topic modeling, we identified CCC patterns, termed CCC topics, representing metabolite- and protein-mediated interactions between ECs and other tissue-resident cells. Most CCC topics exhibited high tissue specificity, potentially explaining microenvironmental regulations for EC heterogeneity. The work systematically investigates EC heterogeneity and provides insights into how EC heterogeneity was regulated across diverse tissue microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c584c5af5601779d13578366342256b73ed6c81" target='_blank'>
              Human single-cell atlas analysis reveals heterogeneous endothelial signaling
              </a>
            </td>
          <td>
            Zimo Zhu, Rongbin Zheng, Yang Yu, Lili Zhang, Kaifu Chen
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Thyroid cancer (TC) is now among the fastest-growing solid tumours, yet therapeutic gains remain limited for poorly differentiated, anaplastic and medullary variants whose median survivals are measured in months. Once guided chiefly by histology and single-gene assays, immunotherapy is being reshaped by single-cell profiling, which exposes the cellular mosaics that arbitrate response and resistance. Dissection of more than 150–000 tumour- and immune-cell transcriptomes has uncovered follicular-like, partial EMT-like and dedifferentiated thyrocyte states embedded within ‘hot’ (CD8hi IFN-γhi), ‘cold’ (CD8lo) and ‘excluded’ (stroma-walled) immune niches; these phenotypes correlate with PD-1/LAG-3 expression, macrophage polarisation and radio-iodine refractoriness. Functional studies reveal that SPP1–CD44 and GAS6–AXL crosstalk licenses epithelial–mesenchymal transition while VSIG4+ macrophages blunt cytotoxic T-cell activity, collectively undermining checkpoint blockade. Spatial transcriptomics corroborates these insights, mapping PD-L1-high tumour islets millimetres from CXCL13-rich tertiary lymphoid structures, whereas CITE-seq quantifies actionable checkpoints and cytokine receptors across patient biopsies. Emerging therapeutics mirror this granular knowledge: combinatorial PD-1 + LAG-3 inhibition, CSF-1R-directed macrophage re-programming and TSH-receptor-targeted CAR-T cells are advancing through early-phase trials, while ex-vivo single-cell pharmacotyping aligns drug cocktails with an individual’s tumour ecosystem. Early lenvatinib-pembrolizumab or selpercatinib-nivolumab trials show ~40% ORR but grade-3 hypertension >60%, prompting staggered-start designs. These advances sharpen pathogenetic resolution, refine patient selection and accelerate translational pipeline design. By integrating single-cell biology, immunology and endocrine oncology, this review identifies diagnostic blind spots, spotlights drug-repurposing opportunities and charts a roadmap toward personalised immunotherapeutic strategies capable of improving outcomes across the diverse spectrum of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6d6beff9a786418bc13757a09189c54eb993ef" target='_blank'>
              Personalised immunotherapy strategies informed by single cell profiling in thyroid cancer: a mini review
              </a>
            </td>
          <td>
            Ruyu Chen, Zhimin Wang, Xueying Chen, Yangling Huang, Fengnuan Zhang, Tianshu Gao
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a055b2099d39e961c79dc682be1c20375d7bba" target='_blank'>
              Tumor microenvironment response to neoadjuvant therapy in breast cancer: insights from single-cell and spatial omics.
              </a>
            </td>
          <td>
            Qingzhong Wu, Jiawei Yang, Da Zhang, Haowei Xu, Yuhang Yu, Yu Zheng
          </td>
          <td>2025-09-13</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Midkine (MDK), a multifunctional growth factor, has been implicated in promoting tumor progression, yet its role in glioblastoma (GBM) remains insufficiently characterized. To investigate MDK’s function in glioma, we integrated four RNA-Seq datasets into a harmonized cohort of 1,017 adult gliomas, including 256 GBM samples. We complemented this with freshly collected human GBM tissues and matched primary cell cultures to evaluate MDK expression and secretion patterns, further contextualized using single-cell RNA-Seq. Finally, we tested the impact of GBM-derived MDK on macrophage secretome composition to validate our in silico observations. We found that MDK expression increases with tumor grade in IDHwildtype gliomas, accompanied by a shift in isoform proportions favoring the canonical MDK transcript. High MDK expression was associated with poor prognosis specifically in GBM, where the MDKhigh subgroup comprised 75% of cases. MDKhigh GBMs exhibited a distinctive multiomic signature, including elevated chemokine and cytokine expression. Functionally, GBM-derived MDK induced macrophages to secrete multiple cytokines and chemokines, suggesting its role in reshaping the tumor microenvironment. Our findings reveal MDK’s previously underappreciated role in GBM aggressiveness and immune modulation, underscoring its potential as a biomarker and actionable therapeutic target for most GBM patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-16253-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5822ef3a00105861887610dcb7da2c42fc7f87" target='_blank'>
              Coordinated changes in midkine expression and midkine-associated multiomic profile in glioma microenvironment
              </a>
            </td>
          <td>
            Mieszko Lachota, K. Zielniok, Agata Gozdz, Patrycja Szpak, Ilona Kalaszczyńska, Radosław Zagożdżon
          </td>
          <td>2025-08-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f63086cbccde700980caecaca85ce93b724792a" target='_blank'>
              Neoadjuvant immunotherapy promotes the formation of mature tertiary lymphoid structures in a remodeled pancreatic tumor microenvironment.
              </a>
            </td>
          <td>
            Dimitri N. Sidiropoulos, Sarah M. Shin, Meredith Wetzel, Alexander Girgis, Daniel Bergman, Ludmila Danilova, Susheel Perikala, Daniel H. Shu, Janelle M. Montagne, Atul Deshpande, J. Leatherman, Lucie Dequiedt, Victoria Jacobs, Aleksandra Ogurtsova, Guanglan Mo, Xuan Yuan, Dmitrijs Lvovs, Genevieve L. Stein-O’Brien, M. Yarchoan, Qingfeng Zhu, Elizabeth I. Harper, A. Weeraratna, A. Kiemen, E. Jaffee, Lei Zheng, W.J. Ho, Robert A. Anders, E. Fertig, L. T. Kagohara
          </td>
          <td>2025-08-15</td>
          <td>Cancer immunology research</td>
          <td>1</td>
          <td>47</td>
        </tr>

        <tr id="Objective
Advanced gastric cancer remains highly refractory to therapy, with limited immunotherapy efficacy due to tumor microenvironment heterogeneity. Primary cilia, microtubule-based organelles involved in tumor progression, remain insufficiently explored in gastric cancer. This study aimed to define primary cilia subtypes and establish prognostic signatures for personalized treatment strategies.


Methods
Bulk transcriptomic data from over 1,500 gastric cancer samples were integrated to define distinct primary cilia subtypes. A primary ciliary phenotype-associated signature (PCS) was established using a multi-machine learning survival framework incorporating ten algorithms. The prognostic predictive value and immunotherapy response prediction capability of PCS were validated across multiple independent cohorts. Single-cell RNA sequencing analysis was performed to identify cellular populations associated with high-PCS phenotype. Causal weighted gene co-expression network analysis (WGCNA) was employed to identify driving factors, followed by functional validation through cell culture experiments and xenograft models.


Results
Two distinct primary cilia subtypes were identified and validated across all cohorts, with C2 patients exhibiting significantly worse overall survival compared to C1 patients. PCS demonstrated robust predictive value for both prognosis and immunotherapy response, with superior accuracy compared to existing models across multiple validation cohorts. High-PCS patients showed reduced tumor purity, increased stromal cell infiltration, and poor response to immunotherapy. Single-cell analysis revealed that fibroblasts had the highest PCS scores and identified a novel secreted modular calcium-binding protein 2 (SMOC2)high myofibroblastic cancer-associated fibroblast (mCAF) population as the key driver of high-PCS phenotype. Functional experiments confirmed that SMOC2 knockdown significantly suppressed gastric cancer cell proliferation, migration, and invasion, while promoting mCAF-to-inflammatory cancer-associated fibroblasts (iCAF) transition.


Conclusions
PCS serves as a robust prognostic biomarker for gastric cancer patients. Additionally, targeting SMOC2 high mCAFs represents a potential therapeutic strategy for patients with high-PCS gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8325aa5e13eb71b8edb3f16c8e7ba4db0f0ba8" target='_blank'>
              SMOC2 high myofibroblastic cancer-associated fibroblast drives primary cilia-associated tumor microenvironment remodeling and poor prognosis in gastric cancer.
              </a>
            </td>
          <td>
            Qiqi Xu, Changjiang Yang, Jingyuan Ning, Yunze Niu, Xuesong Zhao, Long Zhao, Caihong Wang, Shan Wang, Y. Ye, Zhanlong Shen
          </td>
          <td>2025-08-30</td>
          <td>Chinese journal of cancer research = Chung-kuo yen cheng yen chiu</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="T cells recognize antigens and induce specialized gene expression programs (GEPs), enabling functions like proliferation, cytotoxicity and cytokine production. Traditionally, different T cell classes are thought to exhibit mutually exclusive responses, including TH1, TH2 and TH17 programs. However, single-cell RNA sequencing has revealed a continuum of T cell states without clearly distinct subsets, necessitating new analytical frameworks. Here, we introduce T-CellAnnoTator (TCAT), a pipeline that improves T cell characterization by simultaneously quantifying predefined GEPs capturing activation states and cellular subsets. Analyzing 1,700,000 T cells from 700 individuals spanning 38 tissues and five disease contexts, we identify 46 reproducible GEPs reflecting core T cell functions including proliferation, cytotoxicity, exhaustion and effector states. We experimentally demonstrate new activation programs and apply TCAT to characterize activation GEPs that predict immune checkpoint inhibitor response across multiple tumor types. Our software package starCAT generalizes this framework, enabling reproducible annotation in other cell types and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdba0ab80686667b3dcec48b4fa56304b52cf36e" target='_blank'>
              Reproducible single-cell annotation of programs underlying T cell subsets, activation states and functions
              </a>
            </td>
          <td>
            Dylan Kotliar, Michelle Curtis, Ryan Agnew, Kathryn Weinand, A. Nathan, Y. Baglaenko, Kamil Slowikowski, Yu Zhao, P. Sabeti, Deepak A. Rao, Soumya Raychaudhuri
          </td>
          <td>2025-09-03</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Glioblastoma (GB) is a highly aggressive brain tumor with poor prognosis and limited response to immunotherapies. Tumor intrinsic features like low mutational burden, and extrinsic factors such as T-cell dysfunction and an immunosuppressive microenvironment, contribute to resistance. To better study immune-tumor interactions in a clinically relevant context, we have developed patient-derived tumor organoids (PDTOs) that support co-culture with autologous and CAR-engineered immune cells, enabling real-time functional analysis.



 PDTOs were generated from freshly resected GB tissue, preserving tumor-specific architecture and molecular features. Patient-matched PBMCs were isolated via density gradient centrifugation, and immune cells were enriched using negative selection with MACS sorting. T cells were expanded ex vivo and characterized by flow cytometry for activation and exhaustion markers, although not yet tested in co-culture. Cytokine secretion was analyzed via ELISA to complement phenotypic profiling. For functional studies, EGFR/EGFRvIII-retargeted CAR-NK cells were generated via lentiviral transduction of primary PBMC-derived NK cells and expanded ex vivo in the presence of (low levels of) IL-15. These cells were co-cultured with 3D tumor spheroids at defined effector-to-target ratios. Immune infiltration and tumor cell killing were assessed using real-time live-cell imaging and fluorescence-based readouts.



 As proof of concept, we co-cultured EGFR-diploid and EGFR-amplified 3D GB spheroids with naïve and CAR-NK cells targeting EGFR derived from a healthy donor. CAR-NK cells showed enhanced cytotoxicity against EGFR-amplified spheroids, demonstrating receptor-dependent immune activation. These results confirm the feasibility of functional immune co-culture assays and support the ongoing integration of this approach into our organoid-based platforms.



 Our organoid-immune cell co-culture system offers a physiologically relevant platform to investigate GB immunobiology. Future applications include drug screening and CRISPR-based perturbation to identify key modulators of immune response, with the goal of informing personalized immunotherapeutic strategies. Observations from the CAR2BRAIN trial (NCT03383978) have shown CD4⁺ and CD8⁺ T cell recruitment following CAR-NK therapy, highlighting the clinical relevance of combinatorial immune approaches and supporting the use of our model to explore such mechanisms ex vivo.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb41d2ad01808f179f77775b88ffca7888d1d44" target='_blank'>
              P18.15.A IMMUNOCOMPETENT GLIOBLASTOMA ORGANOIDS FOR PERSONALIZED IMMUNE PROFILING
              </a>
            </td>
          <td>
            T. Koenig, L. M. Reindl, P. Zeiner, K. Weber, A. Rombach, M. Armbrust, P. Cakmak, W. Wels, T. Broggini, K. Plate, M. Czabanka, J. Steinbach, M. Burger, E. Ullrich, T. Buhlmann, A. Hau
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer mortality, with minimal improvements in 5-year survival over the past decade. While PDAC originates from premalignant lesions, only a minority of these lesions advance further into malignancy. Distinguishing which lesions are likely to progress would transform enabling timely preventative surgical intervention, but remains a key challenge. In addition to the more well-characterized genetic mutations associated with PDAC onset, recent studies suggest that inflammation in the pancreas microenvironment promotes epithelial chromatin accessibility remodeling to increase their plasticity. This plasticity is thought to enhance cellular sensitivity in response to local signaling cues, which varies drastically across PDAC tumors. We propose that the cell lineage fate of the premalignant epithelial cells is heavily influenced by their surrounding microenvironment. To investigate this, we are using the CosMx spatial transcriptomics platform to profile cellular composition heterogeneity across pancreatic tissue cores from KrasLSL-G12D; Trp53LSL-R172H; Pdx1-Cre (KPC) mouse model of PDAC. By analyzing spatial heterogeneity within the tumor microenvironment, we aim to delineate how local microenvironmental differences might influence epithelial lineage fate decisions into PDAC. Our findings will help understand the factors driving progression from early lesions to invasive cancer, supporting efforts in early PDAC detection and informing clinical strategies for surgical resection.



 Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P. Morton, Crispin J. Miller. Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B118.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb056397ddb7c8e4e0421ffdba57d9fda0a6959d" target='_blank'>
              Abstract B118: Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P. Morton, Crispin J. Miller
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Gliomas are the most common malignant brain tumors, characterized by their aggressive nature and poor prognosis. Despite advancements in radiation and chemotherapy, the five-year survival rate for patients with glioma remains low. A significant challenge is the lack of experimental models that accurately replicate the tumor microenvironment and cellular heterogeneity of human gliomas, limiting the understanding of disease mechanisms and the development of effective therapies.



 Here, we present a novel approach to generate glioma organoids with a microenvironment (GlioME) from patient-derived glioma tissue. These organoids maintain the genetic and epigenetic characteristics of the primary tumor and preserve cell-to-cell interactions within the tumor microenvironment, including resident immune cells. Bulk RNA sequencing, whole exome sequencing, and DNA methylation analysis were used to confirm the molecular similarities between the organoids and primary glioma tissues. Immunofluorescence and flow cytometry were used to assess immune cell viability, comparing GlioME with floating glioma organoids.



 Immunofluorescence, flow cytometry, and gene sequencing revealed that the organoids retained an immune microenvironment consistent with that of the original tumor. Unlike traditional cell lines and animal models, GlioMEs retain the immune microenvironment, allowing faster and more accurate immunotherapy screening. GlioME exhibited high responsiveness to chemotherapy and targeted therapy, demonstrating its potential for therapeutic screening applications. Notably, GlioME accurately predicted patient response to the recently approved MET inhibitor, vebreltinib.



 In summary, our results demonstrate that GlioME can retain the parental glioma microenvironment in vitro and reliably test various therapeutic drugs. This organoid model provides a reliable in vitro platform for glioma microenvironment-related research and clinical drug screening.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4662e63a3116553f2fb291a5b7c518e12a2b2372" target='_blank'>
              P12.31.B A NOVEL ORGANOID MODEL RETAINING THE GLIOMA MICROENVIRONMENT FOR PERSONALIZED DRUG SCREENING AND THERAPEUTIC EVALUATION
              </a>
            </td>
          <td>
            C Zheng, Q. Chen, Z Fang, P. Wang, D. Zhang, Y. Feng, Y. Zhang, T. Jiang, L. Zheng, Z. Bao
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053a87c896e7b2228f019eac1392494ac26a840e" target='_blank'>
              A spatial map of MAPK-activated immunosuppressive myeloid populations in pediatric low-grade glioma.
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Romain Sigaud, Evan Puligandla, Bridget Liu, E. Karimi, Alva Annett, M. Rezanejad, Wajih Jawhar, Robert Taylor, Yi Cao, S. Schmid, F. Selt, P. Hernáiz Driever, Svea Horn, P. Sievers, M. Prinz, M. Glatzel, Christian Mawrin, C. Hartmann, C. Monoranu, Liza Konnikova, F. Sahm, Stefan M. Pfister, Olaf Witt, David T. W. Jones, A. Koch, Claudia L Kleinman, David Capper, L. Walsh, N. Jabado, T. Milde
          </td>
          <td>2025-09-15</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="


 Alaska Native peoples experience high colorectal cancer (CRC) incidence and mortality rates, which Alaska tribal health organizations are working to address. To reduce the high rates, it is important to understand the biological mechanisms that drive risk and survival outcomes. The tumor microenvironment (TME) comprises diverse cell types, including cancer, stromal, and immune cells, whose composition and spatial organization may impact prognosis and disease progression. We performed spatial proteomics profiling on tissue microarrays from 97 Alaska Native patients with CRC using Akoya Biosciences’ PhenoCycler-Fusion system. We included 35 patients who died of CRC and 62 matched patients who did not die of CRC and lived at least as long as the patient they were matched to. Patients were matched on age at diagnosis, sex, tumor site, tumor stage, and year at diagnosis. We classified 2.90 million cells into 16 distinct cell types. We quantified immune and stromal cells as proportions of non-epithelial cells and epithelial cells as proportions of total cells. For spatial analysis, we identified eight cellular neighborhoods with distinct local cell composition and calculated spatial proximities between different cell types. We performed multivariable-adjusted logistic regression with generalized estimating equations to investigate associations of CRC-specific death with proportions of cell types and spatial proximities. We estimated odds ratios (OR) with 95% confidence intervals (CI) and calculated the false discovery rate (FDR) to account for multiple comparison using the Benjamin-Hochberg method. We found that higher proportions of stromal cells (OR for doubling of proportion=2.29, 95%CI=1.00-5.26) and higher proportion of M2 macrophages at the tumor center (OR=2.90, 95%CI=1.14-8.22) were associated with increased risks of CRC-specific death among 76 patients with microsatellite stable (MSS) tumors. We observed higher proportions of podoplanin (PDPN)-expressing stromal cells (OR=2.97 for doubling of proportion, 95%CI=1.28-7.08) and M1 macrophages (OR=2.42, 95%CI=1.05-5.69) within cellular neighborhoods predominantly composed of tumor epithelial cells among patients with CRC deaths compared to survivors, although these associations did not remain significant at FDR 0.05. Additionally, we observed that colocalization of PDPN-expressing stromal cells with CD8+ T cells, and colocalization of antigen-presenting cells with plasma cells, was associated with increased risk of CRC-specific death. In contrast, colocalization of B cells with stromal cells was associated with decreased risk (adjusted P=0.04 for all associations). Those associations remained significant among patients with MSS (adjusted P=0.03). This is the first study focusing on the TME cellular landscape among Alaska Native patients with CRC. Our results highlight the prognostic relevance of cellular composition and spatial organization within the TME, suggesting specific stromal and immune cell interactions as potential biomarkers for CRC outcomes.



 Hang Yin, Diana Redwood, Kimberly Smythe, Kristin L. Robinson, Daniel Jones, McGarry Houghton, Kevin C. Barry, Amanda L. Koehne, Elizabeth Donato, Cecilia Yeung, Minggang Lin, James J. Tiesinga, Tabitha A. Harrison, Sushma S. Thomas, Li Hsu, Jane C. Figueiredo, Li Li, Christopher I. Li, Ulrike Peters, Jeroen R. Huyghe. Spatial-resolved single-cell analysis of the tumor microenvironment in Alaska Native colorectal cancer patients reveals novel biomarkers for prognosis [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr A087.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fe434533ff8a8160b5fcdfdf886e32d3184d6f4" target='_blank'>
              Abstract A087: Spatial-resolved single-cell analysis of the tumor microenvironment in Alaska Native colorectal cancer patients reveals novel biomarkers for prognosis
              </a>
            </td>
          <td>
            Hang Yin, Diana Redwood, Kimberly Smythe, Kristin L. Robinson, Daniel Jones, Mcgarry Houghton, K. C. Barry, Amanda L Koehne, Elizabeth Donato, Cecilia Yeung, MingGang Lin, James Tiesinga, T. Harrison, Sushma Thomas, Li Hsu, Jane C Figueiredo, Li Li, Christopher I Li, Ulrike Peters, J. Huyghe
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Intratumor heterogeneity is a hallmark of cancer, enabling subpopulations of cells to evade therapy, adapt to immune attack, and thrive in diverse microenvironments. While retrospective genomic and epigenomic analyses have mapped the large-scale histories of tumor evolution, they cannot capture the rapid, dynamic changes in cell state that occur as individual cells divide. Panagopoulos and colleagues recently developed a cellular platform to monitor the role of transient replication stress in real time, tracking sister cells as they divide and replicate. The authors use these techniques to show that daughter and granddaughter cells can inherit very different states, often leading to further cellular instability. This work broadens our understanding of how diverse cell states arise from oncogenic stress and how cellular heterogeneity emerges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beadadba41d03604ec2c3cef748b8a0844e1482c" target='_blank'>
              Generations in Flux: Tracking the Birth of Cellular Heterogeneity in Cancer.
              </a>
            </td>
          <td>
            Aaron McKenna
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Neuroendocrine carcinomas (NECs) represent a notoriously aggressive family of lethal malignancies arising across diverse anatomical sites. Molecular subtyping based on key transcription factors ASCL1, NEUROD1, POU2F3, and YAP1 has significantly advanced understanding of tumor heterogeneity in small cell lung cancer (SCLC). Beyond SCLC, extrapulmonary NECs demonstrate analogous heterogeneity, similarly governed by these transcriptional determinants. Recent studies have further identified a fifth subtype driven by the lineage-specifying factor HNF4A. This review aims to propose a unified pan-NEC classification framework for consistent molecular subtyping across pulmonary, gastro-entero-pancreatic (GEP), and genitourinary systems. We delineate the distinct lineage hallmarks of the ANHPY subtypes (neuroendocrine, neuronal, GEP-like, tuft-like, and epithelial-mesenchymal transition phenotypes) and explore their connections to defining mechanisms, genetic alterations, clinicopathological features, and therapeutic vulnerabilities. This unified framework serves as a molecular roadmap for precise NEC research and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ab879bfc552e7093082fd146a38645bd6bb9b5d" target='_blank'>
              Molecular Subtypes of Neuroendocrine Carcinoma: From Chaos to Consensus.
              </a>
            </td>
          <td>
            Zhanyu Wang, Nan Sun, Jie He, E. Redin, Charles M. Rudin
          </td>
          <td>2025-10-08</td>
          <td>Annual review of pathology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="MYCN amplification without concurrent RB1 mutations characterizes a rare yet highly aggressive subtype of retinoblastoma; however, its precise developmental origins and therapeutic vulnerabilities remain incompletely understood. Here, we modeled this subtype by lentiviral-mediated MYCN overexpression in human pluripotent stem cell-derived retinal organoids, revealing a discrete developmental window (days 70–120) during which retinal progenitors showed heightened susceptibility to transformation. Tumors arising in this period exhibited robust proliferation, expressed SOX2, and lacked CRX, consistent with origin from primitive retinal progenitors. MYCN-overexpressing organoids generated stable cell lines that reproducibly gave rise to MYCN-driven tumors when xenografted into immunodeficient mice. Transcriptomic profiling demonstrated that MYCN-overexpressing organoids closely recapitulated molecular features of patient-derived MYCN-amplified retinoblastomas, particularly through activation of MYC/E2F and mTORC1 signaling pathways. Pharmacological screening further identified distinct therapeutic vulnerabilities, demonstrating distinct subtype-specific sensitivity of MYCN-driven cells to transcriptional inhibitors (THZ1, Flavopiridol) and the cell-cycle inhibitor Volasertib, indicative of a unique oncogene-addicted state compared to RB1-deficient retinoblastoma cells. Collectively, our study elucidates the developmental and molecular mechanisms underpinning MYCN-driven retinoblastoma, establishes a robust and clinically relevant human retinal organoid platform, and highlights targeted transcriptional inhibition as a promising therapeutic approach for this aggressive pediatric cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7456982cd84b09cab50502a3b5ab5c8e7a74c4cd" target='_blank'>
              Human Retinal Organoid Modeling Defines Developmental Window and Therapeutic Vulnerabilities in MYCN-Amplified Retinoblastoma
              </a>
            </td>
          <td>
            Jinkyu Park, Gang Cui, Jiyun Hong, H. Jeong, Minseok Han, Min Seok Choi, Jeong Ah Lim, Sanguk Han, Christopher Seungkyu Lee, Min Kim, Sangwoo Kim, Junwon Lee, S. Byeon
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT Pediatric high‐grade gliomas (pHGG) were first defined as a distinct entity in the 2021 fifth edition of the WHO classification of tumors of the central nervous system. These tumors, designated primarily as Grade 4, include the following subtypes: (1) diffuse midline glioma with H3‐K27 alterations (DMG, H3‐K27M), (2) diffuse hemispheric glioma with H3‐G34 mutations (DHG, H3G34M), and (3) diffuse pediatric‐type high‐grade glioma with wild‐type H3 and isocitrate dehydrogenase (pHGG, H3‐WT/IDH WT). Clinically, pHGGs are known for their poor outcomes and marked tumor heterogeneity. Despite this, the characteristics of the tumor microenvironment (TME) and the processes governing tumor cell lineage transitions remain incompletely understood. In this study, we used single‐cell RNA sequencing (scRNA‐seq) to analyze pHGG tumor cells (excluding infant‐type hemispheric gliomas). Through comprehensive bioinformatic approaches—including cell proportion analysis, Gene Ontology (GO) enrichment, metabolic activity inference via scMetabolism, proliferation gene scoring, stemness assessment by CytoTRACE2, SCENT, and pseudotime trajectory analysis with Monocle2—we thoroughly investigated the TME features and heterogeneity of these aggressive brain tumors. Our findings highlight the presence of oligodendrocyte precursor cell (OPC)‐like subpopulations, with epidermal growth factor receptor (EGFR)‐expressing OPC‐like cells emerging as a potential tumorigenic origin in diffuse midline gliomas due to their distinct stemness properties. Notably, platelet‐derived growth factor receptor alpha (PDGFRA)‐positive cells exhibit high specificity in DMG, suggesting greater diagnostic and therapeutic potential than EGFR. Next‐generation sequencing (NGS) and multiplex immunofluorescence analyses confirmed their distinct expression pattern, supporting PDGFRA as a key molecular marker. Moreover, OPC‐like cells at different differentiation states may drive lineage transitions in DMG. Together, these findings enhance our understanding of pHGG—especially DMG—and point to new avenues for targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/746a8877d1a92b076c1950fd12d0bc8dc2480909" target='_blank'>
              Single‐Cell Profiling of Pediatric High‐Grade Gliomas Reveals OPC‐Like Subpopulations Driving Tumorigenic Lineage Transitions
              </a>
            </td>
          <td>
            Tian Tian, Chong Huang, Lusheng Li
          </td>
          <td>2025-09-01</td>
          <td>Pediatric Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Non‐small cell lung cancer (NSCLC) is the most common lung cancer type and one of the deadliest neoplasias worldwide. NSCLC is histologically classified into adenocarcinoma, squamous cell carcinoma, and other less frequent subtypes. Both subtypes and other solid tumors are increasingly regarded as abnormal organs, highlighting the critical role of the desmoplastic tumor stroma rich in cancer‐associated fibroblasts (CAFs) in driving tumor progression and therapeutic resistance. This tumor stroma resembles a chronic fibrotic wound and is largely formed by activated/myofibroblast‐like α‐SMA+ CAFs (myCAFs), which are strongly associated with immunosuppression and poor prognosis. Despite the dominance of the myCAF phenotype, we reported a decade ago phenotypic alterations in NSCLC with a strong dependence on the histologic subtype. Subsequent studies using functional assays, single‐cell techniques, and in vivo models have refined these initial observations, enhancing our understanding of the biology of both normal fibroblasts/myofibroblasts and CAFs in NSCLC and other cancer types, including their origins, subclassification, and physiopathologic functions. Notably, increasing evidence supports that CAFs can exhibit tumor‐restraining or tumor‐promoting effects, and current therapeutic efforts aim to shift the balance towards tumor‐restraining phenotypes. Here, we review major advances in our understanding of tumor stromagenesis and CAF heterogeneity in both primary tumors and metastasis, including emerging consensus, with a special focus on NSCLC and its frequent dissemination to the brain. We also highlight the critical role of smoking through epigenetic reprogramming of the TGF‐β/SMAD3 pathway. These advances are beginning to delineate how CAF heterogeneity depends on the stage and histologic subtype in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bdccf1258eb4984f4511abb46e8045c6f47cd1e" target='_blank'>
              Stromagenesis and cancer‐associated fibroblast heterogeneity in primary tumors and metastasis: focus in non‐small cell lung cancer
              </a>
            </td>
          <td>
            Alejandro Bernardo, N. Díaz-Valdivia, Patricia Fernández-Nogueira, J. Alcaraz
          </td>
          <td>2025-09-04</td>
          <td>FEBS Open Bio</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance.
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background Tumour suppressor genes, exemplified by TP53 (encoding the human p53), function as critical guardians against tumourigenesis. Germline TP53‐inactivating mutations underlie Li‐Fraumeni syndrome, a hereditary cancer predisposition disorder characterised by early‐onset pan‐tissue malignancies. However, the context‐dependent tumour‐suppressive mechanisms of p53 remain incompletely elucidated. This study aims to investigate the disruption of cellular homeostasis and tumourigenic mechanisms following p53 inactivation across distinct cell lineages. Methods Trp53 (encoding mouse p53) knockout mouse model was employed to study molecular alterations under p53‐deficient conditions. Multi‐omics analyses – including single‐cell transcriptomics, single‐cell ATAC‐seq, spatial transcriptomics, whole genome sequencing, and CUT&Tag – were integrated to construct a Trp53 functional cell landscape. Deep learning‐based gene network models were employed to reconstruct p53 regulatory networks and simulate in silico perturbations caused by p53 loss. Results Our analyses revealed transitional dynamics in immune, stromal, and epithelial cells from normal physiology to p53‐deficient states and subsequent tumourigenesis. These transitions implicated critical pathways such as cell cycle regulation, stress response, metabolic reprogramming, and immune modulation, displaying both lineage‐conserved and lineage‐specific features. Tumour‐prone cell populations exhibiting elevated differentiation plasticity were identified across lineages within tumourigenic trajectories. Spatial transcriptomic profiling confirmed the emergence of thymic tumour‐initiating T‐cell clusters characterised by deterministic chromatin architectural disruptions under p53‐loss pressure. Notably, we uncovered a recurrent upregulation signature of ribosomal protein genes as an early pivotal molecular event preceding malignant transformation in p53‐deficient oncogenesis. Finally, we decoded the p53 downstream regulatory network and computationally evaluated the perturbation effects of genetic inactivation at single‐cell resolution. Conclusions Our results elucidate the multiscale consequences of p53 inactivation while providing valuable resources for understanding tumour predisposition associated with p53‐inactivating mutations and developing clinical interception strategies. Key points Construction of a Trp53 functional cell landscape utilising single‐cell multi‐omics and spatial omics technologies. Reconstruction of p53 downstream regulatory relationships with lineage heterogeneity via machine learning‐based gene network modelling. Dissection of shared and lineage‐specific features during cross‐lineage tumourigenesis under p53 deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc305d29bbc69cb0bead2f7a5b3cb8fa23e3cf53" target='_blank'>
              Dissecting cross‐lineage tumourigenesis under p53 inactivation through single‐cell multi‐omics and spatial transcriptomics
              </a>
            </td>
          <td>
            Xinru Wang, Yuqing Mei, Xueyi Wang, Hanyu Wu, Renying Wang, Peijing Zhang, Guodong Zhang, Jiaqi Li, Mengmeng Jiang, Xing Fang, Lifeng Ma, Yuan Liao, Danmei Jia, Haofu Niu, E. Weigao, Haide Chen, Lei Yang, Shuang Zhang, Tingyue Zhang, Yincong Zhou, Qi Zhang, He Huang, H. Ouyang, Ming Chen, Tingbo Liang, Jinrong Peng, Jingjing Wang, Guoji Guo, Xiaoping Han
          </td>
          <td>2025-08-31</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICI) have shown impressive performance in treating several types of solid tumors. However, they have been ineffective in glioblastoma (GBM), in part due to the immunosuppressive tumor microenvironment (TME) created by GBM-associated macrophages (GAMs). To uncover MHC-I peptide antigens for targeted immunotherapy, we performed cell type specific immunopeptidome analysis on primary macrophages and GBM tumor cells in a co-culture system to profile MHC-I associated antigen presentation at the tumor-macrophage interface. Co-culturing tumor cells and macrophages induced increased presentation of peptides derived from proteins associated with cytokine signaling pathways on macrophages and from proteins associated with the Rho GTPase pathway on GBM tumor cells. In vivo expression was validated for a cohort of co-culture-induced GAM or GBM associated peptides selected as potential immunotherapy targets, and an mRNA vaccine was developed encoding six peptides from GAMs and GBM tumor cells. Two doses of vaccination generated an antigen specific immune response, significantly delayed GBM tumor growth, and in some cases eradicated tumors. These results demonstrate the translational potential of co-culture induced MHC peptide antigens as therapeutic targets for GBM/GAM targeting vaccines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e57b6e506268ec6421bc4255cbb602c5178df36c" target='_blank'>
              Quantitative Cell Type Specific Immunopeptidome Analysis of Macrophage and Tumor Co-evolution Reveals Therapeutic MHC-I Peptides in Glioblastoma.
              </a>
            </td>
          <td>
            Yufei Cui, Kien Phuong, Nouran S Abdelfattah, Heidi M Temple, L. Maiorino, B. J. Kim, Jonathan Dye, K. Yu, Stefani Spranger, Forest M. White
          </td>
          <td>2025-09-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a highly heterogeneous malignancy characterized by significant variations in gene expression among distinct cell populations. Traditional sequencing technologies often obscure these differences, necessitating advanced approaches for a comprehensive understanding. Single-cell RNA sequencing (scRNA-seq) and Mendelian randomization (MR) analysis have emerged as powerful tools for unraveling the intricacies of PCa, enabling the identification of unique biological features at the individual cell level. In this study, droplet-based scRNA-seq analysis was performed on 4 PCa samples and 4 normal adjacent to tumor samples. The dataset, obtained from the NCBI GEO database, comprised 13,139 cells, facilitating the identification of diverse cell types and their respective gene expression patterns. Robust analysis pipelines, including pseudotemporal ordering, regional plot analysis, MR analysis, and comprehensive gene expression dynamics, were employed to investigate the tumor microenvironment, immune landscape, and potential causal associations with PCa susceptibility. Our scRNA-seq analysis revealed altered immune landscapes in PCa, underscoring the significance of T cells, particularly mucosal-associated invariant T (MAIT) cells. The study unveiled heterogeneity within the tumor microenvironment and identified key genes, such as ZFP36L2, associated with PCa susceptibility. MR analysis provided insights into the potential causal effects of MAIT cells on PCa, supported by analyses of gene expression dynamics and metabolic landscapes. ZFP36L2 emerged as a pivotal regulator, validated through various experimental approaches, including real-time quantitative polymerase chain reaction, western blot, and immunohistochemistry. Our findings underscore the significance of MAIT cells, presenting a novel diagnostic modality for advancing the diagnosis and treatment strategies of PCa. The identified genes, particularly ZFP36L2, offer potential avenues for precision medicine and targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121d713d54f78ea6b18adcb73c968f4917204d87" target='_blank'>
              Decoding the association between mucosal-associated invariant T cells and prostate cancer via multi-omics analysis
              </a>
            </td>
          <td>
            Lichuan Lai, Rensheng Huang, Keke Si, Junming Huang, Lianli Yin
          </td>
          <td>2025-10-03</td>
          <td>Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708353dcfdc8ca7c4e2b434fced2427c746e282e" target='_blank'>
              Tracing the evolution of single-cell 3D genomes in Kras-driven cancers.
              </a>
            </td>
          <td>
            Miao Liu, Shengyan Jin, Sherry S Agabiti, Tyler B Jensen, Tianqi Yang, Jonathan S. D. Radda, Christian F. Ruiz, Gabriel Baldissera, Moein Rajaei, Fangyong Li, Jeffrey P. Townsend, M. Muzumdar, Siyuan Wang
          </td>
          <td>2025-08-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Malignant peripheral nerve sheath tumors (MPNST) are aggressive sarcomas with limited therapeutic options. Here, we present a novel sporadic murine model of Nf1 wild-type MPNST, driven by conditional expression of oncogenic BrafV600E and Pten loss in the glial lineage using the Plp1::CreERT2 driver. This model allows for highly penetrant and rapid tumor induction through spontaneous formation, localized initiation, or cell transplantation. Comparative analysis with Tyr::CreERT2-driven melanoma revealed striking phenotypic divergence despite shared genetic alterations, underscoring the importance of the cell of origin in shaping tumor identity. In this system, MPNST cells show refractory capacities to induce melanocytic trans-differentiation upon melanoma-promoting signaling cues such as canonical Wnt signaling gain of function (GOF) or increased of levels of the epigenetic mark H3K27Me3 upon Ezh2 GOF. Our findings emphasize the significance of lineage context in tumor initiation and provide a foundation for future mechanistic and therapeutic studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1b40c771fae54171400436daa93f306b770ba99" target='_blank'>
              A novel murine model for sporadic, malignant peripheral nerve sheath tumors, driven by BrafV600E and Pten loss.
              </a>
            </td>
          <td>
            Julien Debbache, Myriam Gwerder, Elisabeth Rushing, Lukas Sommer
          </td>
          <td>2025-10-09</td>
          <td>Disease models & mechanisms</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Neuroblastomas encompass malignant cells with varying degrees of differentiation, ranging from adrenergic (adr) cells resembling the sympathoadrenal lineage to undifferentiated, stem-cell-like mesenchymal (mes) cancer cells. Relapsed neuroblastomas, which often have mesenchymal features, have a poor prognosis and respond less to anticancer therapies, necessitating the development of novel treatment strategies. To identify novel treatment options, we analyzed the sensitivity of 91 pediatric cell models, including patient-derived tumoroid cultures, to a drug library of 76 anti-cancer drugs at clinically relevant concentrations. This included 24 three-dimensionally cultured neuroblastoma cell lines representing the range of mesenchymal to adrenergic subtypes. High-throughput ATP-based luminescence measurements were compared to high-content confocal imaging. With machine learning-supported imaging analysis, we focused on changes in the lysosomal compartment as a marker for therapy-induced senescence and assessed the basal lysosomal levels in a subset of untreated mesenchymal versus adrenergic cells. We correlated these findings with pathway activity signatures based on bulk RNA and scRNAseq. Comprehensive image-based synergy screens with spheroid cultures validated the combined effects of selected drugs on proliferation and cytotoxicity. Mesenchymal models presented high basal lysosomal levels correlating with senescence-associated secretory phenotype (SASP) and sphingolipid metabolism pathways. Chemotherapy treatment further increased lysosome numbers, indicative of therapy-induced senescence. Furthermore, the mesenchymal subtypes correlated with MAPK activity and sensitivity to MAPK pathway inhibitors. Lysosomal and SASP signaling is druggable by inhibitors of lysosomal acid sphingomyelinase (SLMi) or senolytics, including BCL2-family inhibitors. Especially the sequential combination of MEK inhibitors (MEKi) with BCL2-family inhibitors was the most effective on relapsed neuroblastoma cell lines. Gene expression analysis of 223 patient samples, drug sensitivity profiling of five patient-derived fresh tissue cultures, and in vivo zebrafish embryo neuroblastoma xenograft models confirmed these findings. Inhibition of MAPK signaling in combination with BCL2-family inhibitors is a novel treatment option for patients suffering from relapsed neuroblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0577cfd3155a7a66d89cff76645a273955a12c7e" target='_blank'>
              High content-imaging drug synergy screening identifies specific senescence-related vulnerabilities of mesenchymal neuroblastomas
              </a>
            </td>
          <td>
            Sonja Herter, Marta Emperador, Kyriaki Smyrilli, Daniela Kocher, Simay Celikyürekli, Constantia Zeiser, Xenia F. Gerloff, Sina Kreth, K. Henrich, K. Maass, Johanna Rettenmeier, T. Grünewald, H. Peterziel, Frank Westermann, A. Hamacher-Brady, Olaf Witt, Ina Oehme
          </td>
          <td>2025-08-25</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background Despite notable advances with immune checkpoint inhibitors (ICIs) in esophageal squamous cell carcinoma (ESCC), their clinical efficacy remains limited, largely due to CD8⁺T cell exhaustion. Among these, progenitor exhausted T cells (Tpex) represent a key subset with stem cell–like features that sustain durable anti-tumor immunity. Methods We applied multi-color immunohistochemistry (mIHC) to determine the spatial distribution and clinical significance of Tpex cells within the tumor microenvironment (TME) of ESCC. Publicly available single-cell RNA sequencing (scRNA-seq) datasets were further analyzed to characterize Tpex cell phenotypes, differentiation trajectories, and intercellular communication networks. Results Tpex cells constituted a distinct subset of infiltrating CD8⁺T cells and represented a transitional stage of the exhaustion continuum. A higher degree of Tpex infiltration was significantly associated with improved overall survival in ESCC patients. Moreover, scRNA-seq data from patients treated with PD-1 blockade revealed that responders harbored markedly enriched Tpex populations compared with non-responders. Conclusion Our findings identify Tpex cells as a critical prognostic and immunologically active CD8⁺T cell subset in ESCC. Their abundance and functional engagement are closely associated with favorable clinical outcomes and response to PD-1 blockade. Furthermore, their stem cell-like properties may be pivotal in shaping durable anti-tumor immunity and could provide novel therapeutic targets to enhance the efficacy of PD-1-based immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08c5eac76128741742f8a923e04cbd692a6d7928" target='_blank'>
              Progenitor-exhausted T cell as prognostic indicator in esophageal squamous cell carcinoma: illuminating their key contribution to tumor immunity
              </a>
            </td>
          <td>
            Yi Liu, Hongwei Jiang, Zhang Fang, Bin Xu, Junjun Chen, Xiao Zheng, Renhao Geng, Lujun Chen
          </td>
          <td>2025-09-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Meibomian glands (MGs) are an integral component of the ocular defense system, as their secretion product, meibum, is essential for protecting the eye surface. To characterize the transcriptional program underlying meibum production, we employed spatial transcriptomics (ST) analysis of the human eyelid from a sample from a 60‐year‐old male. We resolved 18 distinct eyelid clusters, representing structures such as the conjunctiva, epidermis, hair‐associated sebaceous glands, and MGs. Focusing on the MG, we distinguished basal (MEI‐B cluster) and differentiating (MEI‐DIFF cluster) meibocytes, as well as a third, duct‐related cluster (MEI‐DUCT). Self‐organizing maps (SOM) portrayal of ST images and pseudotime analysis confirmed progress from MEI‐B to MEI‐DIFF and further to MEI‐DUCT, as the latter turned out to include terminally differentiated meibocytes. Accordingly, gene set enrichment analysis associated early/intermediate meibocyte maturation with energy and lipid metabolism, and later stages with barrier functions. We also identified significant differences between the MG and sebaceous gland transcriptomes. The MG‐specific signature included transcripts such as AQP9, MMP3, and PITX1, and selective expression of PITX1 in the MG compared to the sebaceous gland was confirmed by immunohistochemistry on the same sample and samples from three other elderly adults. We deliver the first spatial portrait of the human MG transcriptional landscape. Besides enhancing our understanding of MG physiology, our study identifies novel targets for regulating MG homeostasis in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4c5cea1e03825887de23da452694669d56e573" target='_blank'>
              Spatial transcriptome analysis of the human eyelid depicts meibomian gland cell differentiation: A pilot study
              </a>
            </td>
          <td>
            Hans Binder, Ulrike Hampel, H. Loeffler-Wirth, Florian Hansmann, Helga Pfannkuche, Maria Schmidt, Marlon R. Schneider
          </td>
          <td>2025-09-01</td>
          <td>Physiological Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pediatric high grade gliomas are lethal evolutionary disorders with stalled developmental trajectories and disrupted differentiation hierarchies. We integrate transcriptional and algorithmic network complexity based perturbation analysis to elucidate gene expression patterns and molecular divergence between Diffuse Midline Gliomas and glioblastoma, revealing shared developmental programs steering cell fate decision making. Our complex systems approach supports the emerging paradigm that pediatric high grade gliomas are neurodevelopmental disorders with hybrid lineage identities and disrupted patterning. We identify dysregulated neurodevelopmental and morphogenetic signatures, alongside bioelectric and neurotransmitter signaling programs that alter synaptic organization, neuronal fate commitment, and phenotypic plasticity, regulating glioma phenotypic switching. Causal drivers and regulators of plasticity were predicted as both biomarkers and therapeutic targets, reinforcing the view that pediatric gliomas are disorders of cell fate decisions and collective cellular identity. Decoded plasticity signatures indicate a teleonomic bias toward neural progenitor or neuron like identities, while synaptic transmission gene enrichment supports neuron glioma interactions shaping fate trajectories. These findings advance differentiation therapy as a systems medicine strategy, discovering plasticity regulators to reprogram malignant fates toward stable lineages, offering complex systems based targets for cancer reversion and predictive, preventive, precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c0c46a19437a856f0c3acd3168c8bbabd5c0a3a" target='_blank'>
              Complexity-Informed Causal Modeling of Neurodevelopmental Trajectories in Pediatric High-Grade Gliomas: Divergences from Neural Stem Cell Signatures
              </a>
            </td>
          <td>
            Abicumaran Uthamacumaran, Hector Zenil
          </td>
          <td>2025-08-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Uveal melanoma (UM), the most common primary intraocular malignancy in adults, presents significant clinical challenges due to its high metastatic potential, pronounced hepatic tropism, and poor prognosis upon systemic dissemination. Despite established local therapies, nearly half of patients develop distant metastases, highlighting an urgent need for more effective systemic strategies. Recent advances in single-cell omics technologies (e.g., scRNA-seq, scATAC-seq, spatial transcriptomics) have revolutionized our understanding of UM pathobiology. These approaches have meticulously delineated the complex tumor heterogeneity, immunosuppressive microenvironment, and key molecular drivers—including novel macrophage subsets (e.g., immunosuppressive MΦ-C4), senescent endothelial cells, and non-canonical immune checkpoint expression—providing unprecedented resolution for identifying actionable therapeutic targets. Concurrently, innovations in materials science and biomedical engineering offer transformative opportunities for precision therapy. Engineered nanocarriers, biodegradable implants, and advanced gene therapy vectors (e.g., tropism-enhanced AAVs, CRISPR-Cas9 systems) enabled targeted drug delivery, sustained release, and genetic modulation tailored to the eye’s unique anatomy and immune privilege. This review synthesizes these converging frontiers, outlining how the integration of multi-omics insights with smart biomaterials can overcome current therapeutic limitations. We catalog emerging material-based platforms applicable to UM and summarize validated molecular targets (e.g., GNAQ/GNA11, YAP/TAZ, BAP1, c-Met, CXCR4). Furthermore, we propose an interdisciplinary paradigm spanning combinatorial targeted therapies, immunomodulation, minimally invasive devices (e.g., robotic radiosurgery), and engineered delivery systems. By bridging mechanistic discovery with translational engineering, this synergy holds significant promise for advancing precision medicine and improving clinical outcomes in UM, ultimately facilitating the transition from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0323cee7d6c89a49fe569ae452832b7a890d38bd" target='_blank'>
              Integrating single-cell omics and materials science for uveal melanoma: from mechanistic insights to precision therapeutics
              </a>
            </td>
          <td>
            Shouyong Fu, Changfei Li
          </td>
          <td>2025-09-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c0faec4d04ddee86bb924ee79c08f0dc4ced64" target='_blank'>
              Cancer-associated fibroblasts drive lung adenocarcinoma progression via THBS2-mediated epithelial-mesenchymal transition.
              </a>
            </td>
          <td>
            Yuqing Ren, Ruijie Ming, Anning Zuo, Shutong Liu, Yuhao Ba, Yuyuan Zhang, Yukang Chen, Teng Pan, Peng Luo, Q. Cheng, Jinhai Deng, Yi Yue, Hui-Xian Xu, Siyuan Weng, Xinwei Han, Dongdong Zhou, Zaoqu Liu
          </td>
          <td>2025-09-19</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Ovarian cancer (OvCa) remains largely refractory to immune checkpoint blockade (ICB), owing to a profoundly immunosuppressive tumor microenvironment (TME). Single-cell RNA sequencing (scRNA-seq) of patient tumors has revealed a TME enriched in immunosuppressive macrophages and characterized by exclusion of T cells from the tumor core, features strongly associated with ICB resistance. However, the molecular mechanisms underlying the establishment of this immune landscape remain poorly understood. To identify regulators of OvCa immunity, we employed Perturb-map, a first-of-its-kind spatial functional genomics platform that enables multiplexed CRISPR/Cas9 knockout of candidate genes while preserving the spatial context of tumor–TME interactions in situ. We targeted 35 genes, prioritized through ligand–receptor interaction analysis of human OvCa scRNA-seq data, in an immunocompetent OvCa mouse model. Among these, IL-4 emerged as a key driver of ICB resistance. While IL-4 knockout did not affect tumor growth in untreated animals, it substantially remodeled the local TME and rendered tumors responsive to anti-PD-1 therapy. Unexpectedly, we found that both mouse and human ovarian cancer cells are a source of IL-4, a cytokine traditionally restricted to immune cells. Analysis of patient tumors revealed that tumor-associated macrophages exhibit a robust IL-4, induced transcriptional signature, implicating IL-4 in shaping the immunosuppressive TME in human disease, consistent with our preclinical findings (Cell, 2024). Furthermore, we observed that standard-of-care chemotherapy regimens, including carboplatin, paclitaxel, and cisplatin—induce a dose-dependent increase in IL-4 expression in both murine and human OvCa cell lines. This chemotherapy-induced IL-4 upregulation may contribute to immune evasion and recurrence following initial therapeutic response. Therapeutically, combined blockade of IL-4Rα and PD-1 significantly delayed tumor progression in vivo compared to monotherapy, suggesting that IL-4 signaling is both a necessary and targetable mechanism of immune resistance. These findings are clinically actionable, as Dupilumab, an FDA-approved anti-IL-4Rα antibody, is already in clinical use and well-tolerated. Our results uncover a previously unrecognized, chemotherapy-induced immunoregulatory axis in ovarian cancer and provide a strong rationale for therapeutic strategies that combine IL-4 pathway inhibition with ICB to overcome resistance and improve patient outcomes.



 Gurkan Mullaoglu, Hunter Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian Brown, Alessia Baccarini. Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462188b4ccc770fdbf7b28c6a3fda711e0408972" target='_blank'>
              Abstract PR015: Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Gurkan Mullaoglu, Hunter T. Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian D. Brown, Alessia Baccarini
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="ABSTRACT Multiple myeloma (MM) orchestrates a profound disruption of immune balance within the bone marrow (BM) microenvironment, driving disease progression and therapeutic resistance. To better understand these complex immune dynamics, we used high-dimensional mass cytometry (CyTOF) profiling to comprehensively characterize the immune landscape of the BM across different stages of myeloma progression, including MGUS (n = 16), smoldering MM (SMM; n = 25), and active MM, both newly diagnosed (n = 43) and relapsed/refractory (n = 104). Our analysis revealed substantial immune remodeling during disease progression, characterized by adaptive immune suppression and extensive infiltration of innate immune populations. Transformation from MGUS to SMM was marked by significant alterations in central and effector memory T helper cells, effector cytotoxic T cells, and an enrichment of monocytic and neutrophil subsets. Active MM stages were further distinguished by increased expansion of myeloid and monocytic lineages, alongside a pronounced reduction in progenitor and transitional B cells. Correspondence analysis demonstrated that specific immune profiles were significantly associated with clinical outcomes, including progression-free survival and overall survival. This study highlights the potential of CyTOF-based molecular profiling to unravel the intricate immune dynamics of the BM microenvironment across MM disease stages, enhancing our understanding of MM pathogenesis and providing a foundation for identifying prognostic biomarkers and tailoring precision immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7c28c48facee0fbd69c0cc410926fbb87e89302" target='_blank'>
              CyTOF profiling of bone marrow immune dynamics across myeloma stages
              </a>
            </td>
          <td>
            Danka Cholujová, G. Beke, Ľuboš Kľučár, Ľ. Drgoňa, Zuzana Valuskova, M. Leiba, E. Kastritis, David M. Dorfman, Kenneth C. Anderson, J. Jakubikova
          </td>
          <td>2025-08-18</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="


 Glioblastoma is the most common malignant primary brain tumor in adults and associated with a poor prognosis and a median survival of approximately 15 months, largely due to the lack of effective therapies and inevitable recurrence. A key challenge is the spatial and cellular heterogeneity, evident on MRI as a necrotic center, a contrast-enhancing tumor core, and a non-enhancing, T2/FLAIR-hyperintense infiltration zone. The diffuse infiltration of tumor cells into surrounding brain tissue impedes complete surgical resection and drives relapse from persisting residual cells. While regional molecular differences have been well characterized, their functional implications - particularly drug sensitivities - remain insufficiently understood.



 To address this, we performed MRI- and 5-aminolevulinic acid (5-ALA)-guided tissue sampling from the tumor core and from the infiltration zone of patients with glioblastoma. We utilized pharmacoscopy—a single-cell, microscopy-based drug screening platform—to evaluate region- and tumor cell type-specific drug responses. Cancer cell types were defined based on differential expression of nestin (marking stem-like cells) and S100B (marking more differentiated tumor cells).



 Profiling a library of 59 drugs across region-specific tumor samples from 23 patients revealed pronounced heterogeneity in drug responses across patients, tumor regions and tumor cell states. To overcome this heterogeneity, we computationally predicted complementary drug pairs and validated 20 drug combinations in an independent cohort. The combinations demonstrated enhanced efficacy compared to single agents and effectively addressed glioblastoma’s spatial, cellular, and inter-patient heterogeneity. Notably, drug combinations of oncology and neuroactive drugs outperformed those involving agents from the same drug class (oncology + oncology or neuroactive + neuroactive). Ongoing mechanistical explorations reveal first insights into mechanisms of the complementary drug efficacy.



 Our findings underscore the limitations of monotherapies for patients with glioblastoma and highlight the potential of rationally designed drug combinations tailored to the tumor’s spatial and functional complexity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60062d1c4566c5af52ee0940de47f3ed99103b0" target='_blank'>
              OS12.5.A SPATIALLY RESOLVED FUNCTIONAL PROFILING FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Buck, S. Lee, I. Sakic, R. Wegmann, J. Mena, F. Vasella, J. Friesen, L. Regli, E. Le Rhun, A. Zeitlberger, N. Tatari, G. Hutter, M. Neidert, M. Weller, B. Snijder, T. Weiss
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77d17a800b951a14fec481a0422bbf1125414c43" target='_blank'>
              NOX1 and NPY1R mark regional colon stem cell populations that serve as cancer origins in vivo.
              </a>
            </td>
          <td>
            Maxime Gasnier, Tanysha Chi-Ying Chen, Swathi Yada, Sowmya Sagiraju, Yusuke Yoshikawa, Stefano Perna, H. Lim, Bernett Lee, Nick Barker
          </td>
          <td>2025-09-02</td>
          <td>Nature cell biology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) represent the main immune cell population in various brain malignancies, but there is rare knowledge on the functional and, in particular, the prognostic role of TAMs in the meningioma (MGM) microenvironment. Here, we investigated TAM frequencies, activation state, survival-associated changes, and their association with tumor-infiltrating T lymphocytes (TILs) in two independent study samples comprising altogether 680 MGMs. To this end, we performed tissue cytometry analyses, quantified tissue cytokine levels, and integrated previously published TIL infiltration and microarray datasets in the discovery cohort comprising n = 195 clinically well-annotated cases. This was complemented by a DNA methylation-based deconvolution approach to predict TAM and TIL infiltration rates using immune cell-specific CpG sites as well as survival associations in an independent validation cohort of n = 485 MGMs. Our findings revealed substantial but heterogeneous TAM infiltration in newly diagnosed MGMs, with increased numbers of pro-tumoral TAMs in clinically aggressive tumors. Additional cytokine and transcriptome analyses corroborated the presence of an immunosuppressive niche in TAM-enriched MGMs. Importantly, a high frequency of pro-tumoral TAMs was associated with poor patient outcome, and high TAM infiltration was further identified as an independent prognostic factor for inferior survival, counteracting the beneficial prognostic effect of TILs. Moreover, methylation-based deconvolution analyses confirmed the opposing prognostic roles of TAMs and TILs in the validation cohort. Altogether, higher numbers of TAMs appear to be a hallmark of clinically aggressive behavior in newly diagnosed and recurrent MGMs. Unlike TILs, immunosuppressive TAMs seem to play a dominant role in the immunological landscape of MGMs with a significant negative impact on patient outcome, highlighting pro-tumoral TAMs to be an attractive treatment target in MGMs. Furthermore, our deconvolution approach presents a pipeline to computationally determine TAM and TIL infiltrates in the MGM microenvironment, which might be highly valuable for patient stratification for future immunotherapeutic treatments. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02948-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9eb1c2486a92614ef9d3bad76049f5fefd9924e" target='_blank'>
              Tumor-associated macrophages in meningiomas: a novel biomarker for poor survival outperforming the benefits of T cells
              </a>
            </td>
          <td>
            C. Lotsch, R. Warta, Fang Liu, G. Jungwirth, C. Rommel, M. Barthel, K. Lamszus, A. Kessler, Niels Grabe, Mario Loehr, R. Ketter, C. Senft, S. Maas, P. Sievers, Manfred Westphal, Sandro M Krieg, A. Unterberg, M. Simon, Andreas von Deimling, F. Sahm, D. Raleigh, C. Herold-Mende
          </td>
          <td>2025-10-09</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Patient-derived cancer cells (PDCCs) have emerged as a key strategy for advancing personalized cancer treatment. Unlike traditional cancer cell lines, PDCCs retain the genetic and phenotypic characteristics of the patient’s original tumor and can more accurately reflect tumor biology. This review explores recent advances in methods for culturing PDCCs, highlighting the role of these models in drug discovery and high-throughput screening of personalized therapeutic options. By establishing living models directly from patient tumors, PDCCs can more faithfully recapitulate tumor heterogeneity and microenvironmental features than traditional cell lines. These cultures bridge laboratory research and clinical reality, allowing functional testing of patients' cancer cells. Despite the promise of PDCCs, their culture remains fraught with challenges, including the extremely low number of cancer cells that can be obtained, difficulty maintaining tumor heterogeneity, low culture initiation success rates, and ethical considerations for using patient tissues. In addition, controversy remains regarding the reproducibility of results between different laboratories and patient samples. By examining the field’s current state, this review identifies gaps in the application of PDCCs, such as limited modeling capabilities for specific tumor types and the lack of comprehensive, scalable protocols for broad clinical use. This article discusses future directions, including integration with advanced microengineering and AI-driven analysis, which have the potential to overcome existing limitations and optimize PDCCs-based therapeutic strategies. PDCCs are expected to transform the future of cancer treatment as they ultimately provide more accurate drug testing and personalized medicine models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/350a1202d8af5d4a9d6b419293e7569e23ebec7c" target='_blank'>
              Advancements and challenges in culturing patient-derived cancer cells for personalized therapeutics
              </a>
            </td>
          <td>
            Yatian Fu, B. L. Khoo, Chwee Teck Lim
          </td>
          <td>2025-09-08</td>
          <td>Frontiers in Lab on a Chip Technologies</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background/Objectives: Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with poor prognosis. T cell exhaustion (TEX) is a key factor in tumor immune evasion and therapeutic resistance. In this study, we integrated single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data to characterize TEX-related transcriptional features in HCC. Methods: We first computed TEX scores for each sample using a curated 65-gene signature and classified them into high-TEX and low-TEX groups by the median score. Differentially expressed genes were identified separately in scRNA-seq and bulk RNA-seq data, then intersected to retain shared candidates. A 26-gene prognostic signature was derived from these candidates via univariate Cox and LASSO regression analysis. Results: The high-TEX group exhibited increased expression of immune checkpoint molecules and antigen presentation molecules, suggesting a tumor microenvironment that is more immunosuppressive but potentially more responsive to immunotherapy. Functional enrichment analysis and protein–protein interaction (PPI) network construction further validated the roles of these genes in immune regulation and tumor progression. Conclusions: This study provides a comprehensive characterization of the TEX landscape in HCC and identifies a robust gene signature associated with prognosis and immune infiltration. These findings highlight the potential of targeting TEX-related genes for personalized immunotherapeutic strategies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a1103280627c731b735d281df63ab3e636c5d0" target='_blank'>
              Bulk and Single-Cell Transcriptomes Reveal Exhausted Signature in Prognosis of Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Ruixin Chun, Haisen Ni, Ziyi Zhao, Chunlong Zhang
          </td>
          <td>2025-08-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K J Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Macrophages are highly plastic immune cells that adopt diverse functional states in response to the local microenvironment. The traditional M1/M2 polarization model that has long been used to describe macrophage activation is insufficient to capture the full spectrum of macrophage diversity observed in vivo. Advances in single-cell RNA sequencing (scRNA-seq) have revealed that macrophages exist in a continuum of transcriptional states formed by tissue-specific and disorder-specific cues. This insight has led to the recognition of disorder-specific macrophages, defined as macrophage subpopulations that emerge in response to pathological stimuli and play unique roles in disease progression. These macrophages exhibit distinct transcriptional signatures, epigenetic modifications, and functional properties shaped by their ontogeny and microenvironmental signals, arising from the reprogramming of resident macrophages or the differentiation of bone marrow–derived progenitors. Notable examples include macrophages in chronic infections (e.g., tuberculosis), immunosuppressive tumor-associated macrophages, lipid-associated macrophages in obesity, and disease-associated microglia in neurodegeneration. These subsets exhibit unique regulatory mechanisms, including enhancer remodeling driven by histone H3 lysine 27 acetylation in non-alcoholic steatohepatitis, CCAAT enhancer binding protein α-mediated differentiation in obesity, and Jmjd3-IRF4 axis control in allergic inflammation. Additionally, their function and fate are strongly influenced by their subtissular niche, as evidenced by crown-like structures in adipose tissue, tumor microenvironments, fibrotic lesions, and granulomas, where distinct microenvironmental cues shape macrophage behavior. Furthermore, interindividual heterogeneity in macrophage function, driven by genetic polymorphisms, is increasingly recognized, highlighting the role of host genetic background in disease susceptibility and macrophage-driven pathology. Here, we review the conceptual evolution of the disorder-specific macrophage, tracing its origins from the limited M1/M2 model to its refinement through scRNA-seq-based classification. We summarize the ontogeny, transcriptional regulation, and spatial heterogeneity of these macrophages across various disorders, emphasizing how the subtissular niche dictates functional specialization. Finally, we discuss potential therapeutic strategies targeting disorder-specific macrophage subsets, highlighting the need for integrative multi-omics approaches to refine their classification and functional characterization. Understanding the regulatory networks that govern disorder-specific macrophages will advance our knowledge of macrophage biology while facilitating the development of precision medicine for immune-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9feef19310b98a25e6e67c81bcaeb9daa262ec04" target='_blank'>
              Functional diversity of disorder-specific macrophages involved in various diseases
              </a>
            </td>
          <td>
            Yuichi Mitsui, Takashi Satoh
          </td>
          <td>2025-10-01</td>
          <td>Inflammation and Regeneration</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern and a high rate of relapse. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, i.e. a hypoxic niche, a leading-edge niche, and a peri-vascular niche, in which malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment, including immune cells, astrocytes, and vascular cells.



 By analysis of single-cell transcriptomic data of human GBM and transgenic mouse models of GBM, we unexpectedly identified pericytes intimately associated with the endothelium as the most active paracrine signaling hub within the tumor parenchyma. Exclusive signaling axes emanating from pericytes were received by endothelial cells, malignant cells, astrocytes, and immune cells. Depletion of pericytes through genetic engineering in several different transgenic and orthotopic mouse models of GBM demonstrated accelerated tumor progression, a disrupted blood-brain-barrier, and premature death of pericyte-poor mice. Mechanistic studies revealed that pericyte deficiency altered the cellular composition of GBM, remodeled the endothelium, and impacted on the immune cell landscape. Specifically, endothelial cells deprived of pericyte association upregulated Colony Stimulating Factor (CSF)-1, which in turn attracted peri-vascular tumor-associated macrophages polarized towards an immune-suppressive phenotype. In the absence of pericytes, the recruited macrophages expressed Hepatocyte Growth Factor (HGF), which reinforced activation of its receptor tyrosine kinase c-Met on a subset of GBM cells harboring an extreme mesenchymal-like state. Indeed, orthotopic implantation of isolated c-Met-expressing GBM cells corroborated their superior tumor-initiating capability and invasive phenotype, compared to cells negative for c-Met.



 We infer that the pericyte represents a critical modulator of GBM development by orchestrating a tumor-suppressive microenvironment; our findings thus highlight the importance of pericyte preservation in the face of current and future GBM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a67e2a70cd46f28d9a303424953a15720ec17e" target='_blank'>
              P02.01.B PERICYTES ORCHESTRATE A TUMOR-SUPPRESSIVE MICROENVIRONMENT BY IMPINGING ON THE CROSSTALK BETWEEN MACROPHAGES AND TUMOR-INITIATING CELLS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, M. Safaee Talkhoncheh, B. Phung, E. Cordero, E. Johansson, R. Rosberg, G. B. Jönsson, A. Pietras, K. Pietras
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most common brain cancers diagnosed in children, often leading to life-long neurological impairments. Whilst targeted inhibitors exist, their effects are often short-lived, highlighting the urgent need for innovative treatments. Notably, some pLGG tumors spontaneously regress with age, suggesting that changes in the tumor microenvironment can arrest tumor growth. In this work, we aim to characterize which signaling interactions promote the growth of pLGG cells and how these interactions change dynamically with time. To achieve this, we have optimized a protocol for introducing pLGG mutations into developing mouse brains via in utero electroporation. This study revealed that brains harboring the common KIAA1549::BRAF fusion mutation have a striking phenotype characterized by increased astrocyte reactivity and myeloid cell infiltration. To better understand how signals derived from immune cells may impact pLGG cell fitness, we are additionally performing a high-throughput cytokine screen. We are using barcoded cytokine constructs tethered to the cell membrane to comprehensively screen the impact of cytokine signaling on the growth of neural stem cells engineered to overexpress common pLGG mutations. Together this work aims to unravel which signaling factors govern the age-dependent growth of pLGG tumors, which may reveal targetable aspects of the tumor microenvironment amenable to therapeutic intervention.



 Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy Chang, Ria Kedia, Michelle Boisvert, David Jones, Timothy Phoenix, Pratiti Bandopadhayay. Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f149e2a5701b9900d3e20ea1c251a5956b1b56d2" target='_blank'>
              Abstract B037: Dynamic characterization of tumor-microenvironment factors that drive pediatric low-grade gliomas
              </a>
            </td>
          <td>
            Jenna Robinson, Sarah Reel, Joohee Lee, Marilyn Cobb, Shriya Rangaswamy, Timothy H. Chang, Ria Kedia, Michelle Boisvert, David Jones, T. Phoenix, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is currently the 3rd leading cause of cancer-related deaths in the United States. While studying tissue from human patients holds promise for developing new therapies, this approach is challenging due to the low neoplastic cell content of pancreatic cancer and the availability of small biopsy specimens as the only tissue source for patients with advanced disease. To maximize the amount of information that can be obtained from such specimens, we constructed a high-plex spatial proteomics assay specifically designed for PDAC and developed a matched computational analysis workflow that delivers rapid results suitable for correlative science studies.



 PULSAR (Pancreatic cancer Unified Landscape Spatial Architecture Report) is a comprehensive 61-plex protein assay designed for the PhenoCycler platform (Akoya Biosciences) that provides highly granular, single-cell, spatially-resolved characterization of PDAC, including tumor basal/classical subtype, driver gene status, immune cell infiltration and immune checkpoint expression, stromal fibroblast composition, and cell proliferation status. Antibodies (including 21 pre-conjugated and 40 custom-conjugated) were selected for their known relevance to PDAC biology and were optimized using primary PDAC specimens. To process PULSAR digital image data, we used machine learning-based algorithms as implemented in the QuPath software package to perform tissue segmentation, cell segmentation, and initial cell phenotyping. Additional downstream processing was performed using CELLestial, a novel end-to-end pipeline that performs cell phenotyping using automated thresholding, downstream analyses and data visualization in a custom-scripted interface.



 Across 30 primary pancreatic cancer specimens, the average time for tissue preparation, processing, and image collection was 63 hours, with four cases processed during a week and 100% of images yielding analyzable data. Overall, the per marker staining success rate was 90%, with 90% of all cases having >90% of all markers successfully imaged. Average image size was 58.7 GB, and the number of phenotyped cells ranged from 510,474—957,964 per case. Image processing using QuPath and CELLestial required 2 days on average, including initial quality control review, algorithm training, and refinement of cell phenotypes. Across the 61 markers, more than 75 cell phenotypes were automatically identified, and more than 500 specimen-level features, such as density for a given immune cell type and tumor basal/classical subtype abundance, are calculated.



 By combining a purpose-built assay with a customized bioinformatic workflow, we were able to conduct 61-plex protein-based profiling of pancreatic cancer specimens with a high per-marker success rate in short time frame. Our approach provides a comprehensive assessment of pancreatic cancer and its microenvironment and can accelerate biomarker discovery and therapeutic innovation for PDAC.



 Andressa Dias Costa, Emma C. Coleman, Audrey P. Lacy, Carla Rujana, Dalia Elganainy, Micaela Morgado, Cibelle F. Lima, Canh Hiep Nguyen, C.Sloane Furniss, Andrew J. Aguirre, Brian M. Wolpin, Simona Cristea, Jonathan A. Nowak. Rapid, high-plex spatial proteomic profiling of pancreatic cancer using a pancreas-focused-assay and customized analysis workflow [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B124.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42f83a024f8dc0117588f191784ff14f50dfd88a" target='_blank'>
              Abstract B124: Rapid, high-plex spatial proteomic profiling of pancreatic cancer using a pancreas-focused-assay and customized analysis workflow
              </a>
            </td>
          <td>
            Andressa Dias Costa, Emma C Coleman, Audrey P. Lacy, Carla Rujana, Dalia Elganainy, Micaela Morgado, Cibelle Lima, C. Nguyen, C.Sloane Furniss, A. Aguirre, B. Wolpin, Simona Cristea, Jonathan A. Nowak
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="T follicular helper (Tfh) cells are central to the adaptive immune response and exhibit remarkable functional diversity and plasticity. The complex nature of Tfh cell populations, inconsistent findings across experimental systems and potential differences across species have fueled ongoing debate regarding core regulatory pathways that govern Tfh differentiation. Many studies have experimentally investigated individual proteins and circuits involved in Tfh differentiation in limited contexts, each providing only a partial understanding of the process. To address this, we adopted a novel multi-scale network systems approach that incorporates both regulatory and protein-protein interactions. Our approach integrates diverse data types, captures regulation across multiple levels of immune system organization, and recapitulates known drivers. Further, we discover a core Tfh gene set that is conserved across tissue types and disease contexts, and is consistent across data modalities - bulk, single-cell and spatial. While components of this set have been individually reported, a novel aspect of our work lies in the discovery, characterization, and connectivity of this core signature using a single unbiased approach. Using this method, we also uncover a novel function of IL-12, a molecule with reported conflicting functions, in the regulation of Tfh differentiation. Notably, we find that, in both humans and mice, IL-12 is permissive for the differentiation of Tfh precursors, but blocks subsequent differentiation into GC Tfh cells. Overall, this work elucidates novel networks with unexplored roles in governing Tfh cell differentiation across species and tissues, paving the way for novel -therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cdef08e00798c3da4a3f940c2d60360ad82720" target='_blank'>
              A unified network systems approach uncovers a core novel program underlying T follicular helper cell differentiation
              </a>
            </td>
          <td>
            Alisa A. Omelchenko, Syed A. Rahman, Vinayak V. Viswanadham, Grace J. Yuen, Perla M. Del Rio Estrada, Valentino D’Onofrio, Yijia Chen, Na Sun, Hamid Mattoo, Chinmay G. Varma, Gonzalo Salgado, Maribel S. Nava, Lady C. Ruiz, Dafne D. Rivera, S. A. Rios, Sudhir P. Kasturi, Susan P. Ribeiro, M. Shlomchik, Amanda C. Poholek, Shiv S. Pillai, Rebecca A. Elsner, V. Mahajan, Jishnu Das
          </td>
          <td>2025-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Despite significant efforts, Diffuse Midline Gliomas (DMGs) remain incurable. Although promising results are obtained in preclinical studies, most approaches have failed to improve survival in these young patients. In this context, by integrating elements of the biophysical tumor microenvironment known to regulate the response to therapies, a new preclinical tool is developed to better evaluate the efficacy of antitumoral strategies. For this purpose, a novel DMG-on-Chip (DoC) is engineered composed of a 3D dense tumor disk embedded in an extracellular matrix, which is accessible for real-time monitoring using wide-field phase contrast or confocal fluorescence microscopy. By driving the oxygen supply within the chip solely in a radial manner, a hypoxia gradient is established that can be associated with changes in DMG cell phenotype, proliferation, and rewiring of metabolic and stress transcriptomic signatures. Finally, DoC is used to analyze the spatial heterogeneity of the response of DMG cell lines and patient-derived 3D cultures to treatments through cell segmentation. Altogether, these interdisciplinary characterizations validate the new tool as an interesting healthcare solution to understand how cell responses are modulated by biophysical or biochemical cues in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a25e403c641a7729cb13c93c687243726c53a1" target='_blank'>
              Novel Diffuse Midline Glioma-on-Chip Recapitulating Tumor Biophysical Microenvironment to Assess the Heterogeneity of Response to Therapies.
              </a>
            </td>
          <td>
            Lisa Terrassoux, Calypso Hazard, Agathe Laratte, Joanne Balsamelli, Mélanie Arcicasa, Elisa Fong, Léo Baland, Fabrice Soncin, Giovanni Cappello, C. Maurage, Sebastien Janel, Antony Bazir, Fabrizio Cleri, Anthony Treizebré, Eric Lartigau, Yousr Rekik, Samuel Meignan, Alessandro Furlan
          </td>
          <td>2025-09-29</td>
          <td>Small</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains a highly lethal malignancy, in part due to its heterogeneous tumor microenvironment. Most PDAC tumors are poorly vascularized and embedded within densely fibrotic stroma; however, in prior computational analyses, we identified a subset of PDAC tumors with hyper-vascular features. In this study, we sought to determine the extent of vascular heterogeneity in PDAC and its functional consequences. Analysis of human tumor specimens revealed that approximately 10-20% of human tumors display features of hyper-vascularity. Using a panel of clonal murine cell lines that recapitulate this heterogeneity, we demonstrate that VEGFA-mediated angiogenesis does not account for the increased vessel density. Functionally, we further show that hyper-vascular tumors display enhanced sensitivity to both chemotherapy and anti-angiogenic therapy. These findings help explain the limited efficacy of VEGFA-targeted therapies in unselected PDAC cohorts and highlight vessel density as a potential biomarker for patient stratification.



 Samantha B. Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo I. Orlen, Samuel D. Zwernik, Kailee M. Hartway, Nina G. Steele, Agnieszka K. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger. Vascular heterogeneity influences treatment response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05cec4b9af7cf61e156104d3e634ae5d707e4e9c" target='_blank'>
              Abstract A127: Vascular heterogeneity influences treatment response in pancreatic cancer
              </a>
            </td>
          <td>
            Samantha B Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo Orlen, Samuel D Zwernik, Kailee M Hartway, Nina G. Steele, A. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Merging evidence suggests that cellular fitness competition between malignant cells and the tumor microenvironment (TME) dictates tumor progression. We recently identified Flower isoforms as molecular “fitness fingerprints,” where cancer cells expressing Flower-Win gain dominance over surrounding cells marked with Flower-Lose. In this study, we uncover that loss of stromal fitness is not an inherent trait but rather a cancer-driven event. Tumor cells secrete a specific long non-coding RNA, Tu-Stroma, via exosomes, which reprograms splicing of the Flower locus in TME cells, enforcing Flower-Lose expression and thereby reducing stromal competitiveness. This cancer-induced shift establishes a permissive niche that accelerates growth, metastatic dissemination, and ultimately shapes host survival in ovarian cancer. Therapeutically, blockade of Flower using a fully humanized monoclonal antibody (Aenya) markedly suppressed tumor expansion and peritoneal spread in patient-derived xenograft models. Importantly, prophylactic administration preserved the integrity of intraperitoneal organs even in the presence of highly aggressive tumor cells. These findings highlight a previously unrecognized mechanism of cancer-driven stromal takeover and establish Flower as a first-in-class therapeutic target with broad implications for preventing metastatic colonization.



 Rajan G. Gogna. Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0864f00ed3ab31470a0c2e77e5565e26d50bb11a" target='_blank'>
              Abstract IA008: Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer
              </a>
            </td>
          <td>
            Rajan Gogna
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Intra-tumoral heterogeneity is a prominent characteristic of hepatocellular carcinoma (HCC). However, it remains unexplored whether intra-tumoral transcriptomic differences can capture crucial information regarding HCC evolution and be utilized to derive a predictive signature for patient’s clinical trajectories. Methods We quantified transcriptomic heterogeneity using four multiregional HCC cohorts comprising 172 samples from 37 patients, and validated transcriptomic heterogeneity and spatial dynamics using multiregional single-cell transcriptomic profiling of 110,817 cells from 34 liver specimens. The HCC evolutionary signature (HCCEvoSig) was developed and assessed across six cross-platform HCC cohorts. Results Genes exhibiting high intra- and inter-tumor expression variation were significantly enriched in a gene set associated with HCC prognosis, from which we developed and validated a reproducible and robust transcriptomic signature, HCCEvoSig. Multiregional single-cell data confirmed the high intra- and inter-tumoral heterogeneity of HCCEvoSig genes across different cell types, and importantly, demonstrated that the dysregulation of HCCEvoSig genes exhibited a geospatially gradual transition from the non-tumor region to the tumor border and tumor core, as well as from non-malignant to malignant epithelial cells. HCCEvoSig showed significant positive associations with adverse features of HCC, and a high HCCEvoSig risk score predicted increased risks of disease progression and mortality, independent of established clinicopathological indices. Furthermore, HCCEvoSig outperformed 15 published signatures in discriminative ability and prognostic accuracy, particularly regarding 1-year survival rates. Notably, HCCEvoSig demonstrated predictive utility for responses to immunotherapy and trans-arterial chemoembolization. Additionally, we established a well-calibrated predictive nomogram that integrates HCCEvoSig and TNM stage to generate an individualized numerical probability of mortality. Conclusion Our study reveals that regional transcriptional heterogeneity within tumors is substantial enough to capture survival signals, and the constructed and validated HCCEvoSig provides reliable prognostic information for HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3efeaa7fe73401eb801ace896c0e22c461a57e69" target='_blank'>
              Spatial heterogeneity reveals an evolutionary signature predicting therapeutic response and clinical outcomes in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Shangyi Luo, Li Liu, Yang Sun, Jian Shi, Yajing Zhang
          </td>
          <td>2025-08-18</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Capillary malformation (CM) is a congenital vascular anomaly affecting the skin, mucosa, and brain, yet the understanding of its vascular pathogenesis remains limited. Methods: We applied spatial whole-transcriptome profiling (GeoMx) and gene set enrichment analysis within CM lesions at single vasculature level. Differentially expressed genes were validated by immunofluorescence staining. Phosphoproteomics was profiled to uncover lesion-wide phosphorylation sites on proteins. Single-cell RNA sequencing was performed on CM-derived induced pluripotent stem cells (iPSCs) to determine differentiation trajectories of lesional vascular lineages. In silico gene perturbation was used to predict candidate genes for modulating vascular pathological progression, followed by functional validation in CM iPSC-derived endothelial cells (ECs) using a Tet-on system. Results: A spatial transcriptomic atlas was constructed, and pathological landscape of individual CM vasculature was delineated. CM vessels exhibited hallmarks of endothelial-to-mesenchymal transition (EndMT), including disruption of adherens junctions (AJs), vascular identity transitions, and metabolic remodeling. Phosphoproteomics confirmed that differentially phosphorylated proteins were enriched in EndMT- and AJ-related pathways. Aberrant expression of venous transcriptional factor NR2F2 was observed in lesional ECs and correlated with progressive enlargement from capillaries to larger-caliber vessels containing multiple layers of smooth muscle cells (SMCs). In CM iPSCs, differentiation course yielded reduced ECs but increased SMCs. In silico knockout simulation predicted NR2F2 as a crucial regulator of facilitating SMC phenotype in CM. Consistently, enforced NR2F2 expression during iPSC differentiation suppressed endothelial markers while inducing SMC-associated genes. Conclusions: Single CM vasculature displays pathological hallmarks characterized by EndMT and AJ disruption, leading to progressive vascular remodeling. NR2F2 functions as a central regulatory factor orchestrating the acquisition of the SMC phenotype, thereby representing a potential therapeutic target in CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1fa98536c86180270a8a6d151af2637cfd03767" target='_blank'>
              Single-vessel transcriptome map pathological landscapes and reveal NR2F2-mediated smooth muscle cell phenotype acquisition in capillary malformations
              </a>
            </td>
          <td>
            Vi Nguyen, Irving Mao, Siwuxie He, Isabella Castellanos, Mackenzie Azuero, Marcelo L Hochman, Yin Rong, L. Pernomian, Elliott H Chen, Harold I Friedman, Yan-Hua Chen, Qun Lu, Daping Fan, Camilla F Wenceslau, Dong-bao Chen, Stuart Nelson, Anil G Jegga, Yunguan Wang, Wenbin Tan
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Brain metastases often originate from circulating tumor cells (CTCs) that become arrested within cerebral microvessels. Recent findings indicate that this process induces localized hypoxia, which in turn upregulates angiogenic mediators such as Angiopoietin-2 (Ang-2), thereby promoting vascular remodeling and facilitating extravasation of tumor cells into the brain parenchyma. Nevertheless, the early transcriptional landscape of the adjacent brain tissue—the so-called pre-metastatic niche—remains poorly defined.



 To simulate early metastatic seeding, syngeneic murine lung and breast carcinoma cells were injected into the right carotid artery of immunocompetent mice. The contralateral, non-injected hemisphere served as an internal control to allow intra-animal comparisons, reduce biological variability, and minimize animal usage. Brain tissue was collected at 1 and 4 days post-injection, processed via formalin fixation and paraffin embedding, and subjected to histological evaluation. Comparative bulk RNA sequencing of tumor-bearing and control hemispheres was performed, complemented by high-plex spatially resolved single-cell transcriptomic profiling to detect localized gene expression patterns.



 Bulk RNA sequencing revealed no significant transcriptional differences between hemispheres. In contrast, spatial transcriptomic analysis uncovered robust, spatially restricted gene expression changes in the tumor-bearing hemisphere. These alterations involved genes associated with vascular remodeling, hypoxia-responsive pathways, and neuronal connectivity. Notably, spatial mapping demonstrated that the expression of several key transcripts peaked near sites of arrested tumor cells and declined with increasing distance, indicative of a highly localized, tumor-induced host response.



 This study reveals distinct, spatially confined transcriptional alterations that define the early pre-metastatic niche in the brain. By leveraging advanced spatial single-cell transcriptomics, we provide a powerful framework to dissect the molecular cues underlying early metastatic interactions within the native tissue microenvironment. These insights not only deepen our understanding of early brain metastasis biology but also inform the development of innovative strategies for therapeutic intervention at the pre-metastatic stage.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6ab71d6adf66b6036a2c3fb3715feb43793308" target='_blank'>
              P18.25.B SPATIALLY CONFINED TRANSCRIPTIONAL RESPONSES CHARACTERIZE THE EARLY PRE-METASTATIC NICHE IN BRAIN METASTASIS
              </a>
            </td>
          <td>
            W. R. Hube, F. Schallerer, L. Bergmayr, R. Piecyk, T. Eska, K. J. Mueller, E. Nicolaishvili, L. Sevenich, K. Unger, P. Jurmeister, L. von Baumgarten, P. Harter
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R. Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Brain metastases (BM) critically reduce breast cancer (BC) patients’ survival. Extravasation is pivotal for BM development, but the underlying events at the blood-brain barrier (BBB) remain elusive. We aimed to unravel the players and mechanisms governing BC cells (BCCs)-BBB interaction. For that, mixed cultures of human brain microvascular endothelial cells (HBMECs), mimicking the BBB, and brain-tropic triple-negative BCCs (MDA-MB-231 Br4), or non-brain-tropic (MDA-MB-231) or non-metastatic cells (MCF-7) were established. Temporal and spatial analysis of BCCs-BBB interactions (live-cell imaging automated microscopy), and assessments of endothelial-to-mesenchymal transition (EndMT) markers, transcription factors, cytoskeletal proteins, and morphology (immunocytochemistry) were performed. BBB integrity (permeability, transendothelial electrical resistance) and endothelial migration (wound-healing) were also assessed. Our results revealed that contrasting with non-metastatic and non-brain-tropic cells, BCCs quickly developed an invasive, migratory phenotype, characterized by invadopodium formation and reduced roundness. Spatial analysis showed different positioning of BCCs relative to the BBB endothelium over time, with 14% of BCCs transmigrated after 3 h, compromising BBB integrity through endothelial holes, reduced tightness, and increased permeability. Prior to transmigration, alterations in adhesion markers (E-selectin, ICAM-1, CD24, CD34, β3-integrin, Sialyl-Lewis X) were observed. EndMT was also evident by decreased endothelial (β-catenin and pan cytokeratin) and increased mesenchymal (vimentin, neuronal-cadherin, Slug, ZEB1) markers, elongation (RhoA, α-SMA), nuclear deformation, and migratory capacity. Caveolin-1 silencing in HBMEC decreased BCCs transmigration. This study reveals significant BBB phenotypic and structural changes, facilitating both paracellular and transcellular BCCs transmigration. These findings provide advanced understanding of BCCs trafficking across the BBB, aiding strategy development to prevent extravasation and BM. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-01985-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58375842dd28bad8d863cc4ae660178921a26a5b" target='_blank'>
              Unveiling the intricate dynamics of the interplay between triple-negative breast cancer cells and the blood-brain barrier endothelium
              </a>
            </td>
          <td>
            Ana Rita Garcia, Ana Rita Vaz, Rui Malhó, Hugo M. Botelho, Inês Figueira, M. A. Brito
          </td>
          <td>2025-08-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b1a443d05c1334b2d0a0dff8460a523dfdb2d5" target='_blank'>
              A three-dimensional ex vivo model recapitulates in vivo features and drug resistance phenotypes in childhood acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Magdalini Kanari, Iria Jimenez Garcia, Fabio Steffen, L. Krattiger, Charles Bataclan, Wangjie Liu, Benjamin R. Simona, Bart Deplancke, O. Naveiras, M. Ehrbar, B. Bornhauser, Jean-Pierre Bourquin
          </td>
          <td>2025-09-10</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The spatial organization and composition of the tumor-immune microenvironment (TME) play a critical role in shaping the progression of many solid cancers, but the organization of the TME in primary prostate cancer (PCa) remains poorly characterized. We therefore profiled the abundance and spatial distributions of major cell types involved in adaptive immunity in 29 radical prostatectomy specimens stratified into high (HGG; n=14) and low Gleason-grade (LGG; n=15). Compared to LGG, HGG PCa exhibited significantly greater B and T cell infiltration with many immune cells organized into clusters, some of which resembled tertiary lymphoid structures (TLSs). In HGG tumors, these clusters were dense, symmetric, rich in PD-1+ T cells, and frequently proximate to the tumor compartment. LGG clusters were less well organized, and T cell depleted. Thus, a subset of high-grade PCa harbor organized immune clusters that may play a role in tumor control and contain therapeutically targetable T and B cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e5e762dad0168906d48c0967522a47db6d0ac" target='_blank'>
              Spatially organized lymphocytic microenvironments in high grade primary prostate tumors
              </a>
            </td>
          <td>
            Ali Amiryousefi, Jeremiah Wala, Jia-Ren Lin, Brian Labadie, A. Atmakuri, Z. Maliga, Eamon Toye, K. Chaudagar, Madeleine S Torcasso, Shannon Coy, G. Fanelli, Brigette Kobs, F. Socciarelli, Andréanne Gagné, Eliezer Van Allen, Akash Patnaik, P. Sorger
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Lysosomes are critical organelles that act as degradation centers and signaling hubs within cells, playing a significant role in various cellular processes and human diseases, including cancer. However, the extent to which they influence the heterogeneity and clinical outcomes of ovarian cancer (OC) remains inadequately understood. In this study, we used consensus clustering to identify two distinct lysosome-related clusters (LCs) in OC by analyzing the expression profiles of OC patients from The Cancer Genome Atlas (TCGA) database. Further analyses revealed the functional characteristics and immune landscapes of these subgroups, providing valuable insights into the tumor immune microenvironment (TIME) and tumor responses to immunotherapy. Additionally, we developed and validated a prognostic model based on differentially expressed genes (DEGs) between the two LCs, demonstrating its effectiveness in predicting patient prognosis, TIME characteristics, and immunotherapy potential in OC. A further investigation explored the relationship between lysosome-associated risk scores, IC50 values of standard antitumor agents, and the expression levels of prognostic genes. Finally, in vitro experiments showed that inhibiting CRHR1, a lysosome-associated prognostic gene, significantly reduced OC cell proliferation, invasion, and migration. In conclusion, our study establishes a novel lysosome-based classification and prognostic framework for OC, offering a practical tool to predict clinical outcomes and guide personalized immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18271-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a65a1574799d7c11e4a3b68fa4b0f39f93a926" target='_blank'>
              Lysosome-related molecular subtypes reveal prognostic signatures and immunotherapeutic implications in ovarian cancer
              </a>
            </td>
          <td>
            Zhou Fang, Yunqing Liu, Yujie Cui, Ke Cao, Yiling Han, Zhijie Gao
          </td>
          <td>2025-09-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal tumor. PDAC also shows poor response to many conventional treatments and is refractory to immunotherapy. We hypothesize that these challenges arise from the highly heterogeneous and immunosuppressive tumor microenvironment (TME) in PDAC. Therefore, to comprehend the players behind this immunosuppressive TME, we integrated 10 publicly available datasets into a robust PDAC scRNA atlas of 941.608 cells covering 201 patients (159 PDAC, 24 non-cancer, and 18 metastatic PDAC). The TME showed distinct composition for each tissue, predominating a fibro-inflammatory phenotype with infiltrating Tregs and dysfunctional CD8 T cells in primary PDACs. In contrast, adjacent normal tissue exhibited enrichment of tissue-resident memory CD8 T cells (p < 0.05). Metastatic PDAC was characterized by a reduction in stromal cells combined with myeloid and naive T cells infiltration. Comparison between treated and untreated PDAC revealed the enrichment of cancer-associated fibroblasts (CAFs) in treated patients. Besides, CD8 T cells from untreated patients exhibited higher levels of genes previously linked to immunotherapy response compared to treated patients. Non-negative matrix factorization (NMF) of the malignant cells unraveled 8 distinct metaprograms (MP). MP3 and MP7 overlapped with the well-known Classical subtype, whereas MP5 correlated with the Basal subtype. MP2 exhibited high expression of MHC-II genes and was enriched in metastatic lesions. Next, we investigated the tumor architecture using public spatial transcriptomic datasets. Spot-level deconvolution revealed 12 distinct cellular niches. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications.



 Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, Alessandro Bersch Osvaldt, Simone Marcia Dos Santos Machado, Eduardo Cremonese Filippi-Chiela, Cristina Bonorino. Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B126.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a30afbd64e95a911caaa3a76b2513c14671d0c3e" target='_blank'>
              Abstract B126: Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Pre-invasive breast cancer, or ductal carcinoma in situ (DCIS), shares many morphologic and genomic features with invasive breast cancer, yet most DCIS tumors remain indolent over decades. In this study, we performed spatial analyses of somatic mutation patterns in 18 DCIS tumors to infer the underlying growth dynamics. Experimental data combined with mathematical models and Bayesian computation revealed a growth model that mimics the cellular dynamics of branching morphogenesis, which governs normal mammary gland development during puberty. Unlike a traditional model of clonal evolution, this non-competitive growth model allowed for extensive DCIS spread without the selective pressures of subclone competition that promote the emergence of invasive phenotypes. This finding provides a framework to understand the biological basis for the limited progression of DCIS and adds rationale for ongoing clinical efforts to reduce DCIS overtreatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56eaf0d13e80db0fb1f5ee2e2242518bc8cf64f7" target='_blank'>
              Spatially Discontinuous Mutation Topographies in Ductal Carcinoma in Situ Reveal Non-Competitive Growth Dynamics.
              </a>
            </td>
          <td>
            Marc D. Ryser, Matthew A Greenwald, Daniel L. Monyak, I. Sorribes, Lorraine M. King, Allison Hall, J. Geradts, Donald L Weaver, D. Mallo, Shannon T Holloway, Ethan Wu, Kevin A. Murgas, Robert B. West, Lars J. Grimm, Carlo C. Maley, Jeffrey R. Marks, Darryl Shibata, E. S. Hwang
          </td>
          <td>2025-10-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Ovarian cancer remains the most lethal gynecologic malignancy and a leading cause of cancer-related deaths among women. Adoptive cell therapy (ACT) using tumor-infiltrating lymphocytes (TILs) has shown success in melanoma and correlates with survival in ovarian cancer. However, its application remains limited, partly due to inefficient expansion of functional, tumor-specific T cells.



 We developed a novel ex vivo expansion platform for ovarian TILs, integrating metabolic and phenotypic reprogramming with selective enrichment for tumor-reactive populations. Tissue from nine patients with stage III/IV high-grade epithelial ovarian carcinoma was collected for banking, NGS, and TIL expansion. Four had matched TILs expanded pre- and post-neoadjuvant chemotherapy. The initial priming phase employed our published cytokine cocktail designed to induce Th1/17 and Th17/Tc17 polarization, to promote effector differentiation while preserving stemness. TILs were cultured for five days in this cocktail to enhance metabolic fitness and anti-tumor potential. CD137+ (4-1BB+) T cells were isolated based on activation-induced co-stimulatory expression and subjected to rapid expansion using a G-Rex-based bioreactor system with irradiated feeder cells, anti-CD3, and low-dose IL-2. Expanded TILs were characterized by multicolor flow cytometry, CD107a degranulation, IFN-γ ELISA, and single-cell RNA sequencing (scRNA-seq), and benchmarked against TILs expanded using the conventional protocol from NCI.



 Our approach reliably produced 15–20 billion viable TILs per patient within 30 days, achieving >95% viability and ∼2500-fold expansion. CD137+ enrichment enhanced tumor reactivity, as shown by increased IFN-γ secretion and degranulation with autologous tumor and high-grade serous ovarian cancer cell lines. scRNA-seq revealed enriched cytotoxic CD8+ T cells and mucosal-associated invariant T (MAIT) cells, with reduced FOXP3+ regulatory and non-cytotoxic CD4+ subsets, compared to TILs from matched blood, tumor, and those expanded with high-dose IL-2 in the NCI protocol. CD8+ proliferative and cytotoxic T cell subsets were enriched under the hybrid CD137-sorted protocol. Transcriptomic profiling showed that CD8+ proliferating T cells from the hybrid group downregulated key metabolic genes, including SLC7A5 and LDHB, suggesting a rewired metabolic state that may support persistence. Similarly, CD8+ cytotoxic T cells showed chromatin remodeling, with upregulation of histone demethylases KDM5A and KDM7A, and downregulation of CHD7 and JARID2, indicating transcriptional plasticity. MAIT cells also showed increased activation markers (IFNG, CCL20). Expanded TILs exhibited reduced expression of exhaustion markers PD-1 and CD38 at the protein level.



 Metabolic reprogramming combined with CD137-based enrichment enables robust expansion of metabolically reprogrammed, tumor-specific TILs from ovarian tumors. This strategy offers a promising foundation for advancing TIL-based ACT in ovarian cancer.



 Paramita Chakraborty, Stephanie Mills, Jadan Zavestoski, Satyajit Das, Suganya Subramanian, Nicolina Mascia, Jian Liu, Stefano Berto, Brian Orr, Shikhar Mehrotra, Stephanie Mills. Metabolic reprogramming of ovarian CD137+ TILs for treatment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b37662ad71037e3dead97ad855c982dd8d046b2" target='_blank'>
              Abstract PR006: Metabolic reprogramming of ovarian CD137+ TILs for treatment
              </a>
            </td>
          <td>
            Paramita Chakraborty, Stephanie Mills, Jadan Zavestoski, Satyajit Das, S. Subramanian, Nicolina Mascia, Jian Liu, S. Berto, Brian Orr, Shikhar Mehrotra
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) is a common characteristic of human cancers, which occurs most frequently in ovarian and breast cancers. The unique genetic vulnerabilities, coupled with the synthetic lethality effect of Poly-(ADP-ribose) polymerase (PARP) inhibitors (PARPi), provide a promising opportunity for targeting HRD cancers. However, only a few HRD patients benefit from PARPi monotherapy, and it is therefore imperative to explore effective combination therapies for HRD tumors. Growing evidence has underscored the distinct tumor microenvironment (TME) landscape of HRD cancers. Immune activation and immune suppression co-exist in a dynamic balance during the development of HRD cancers. At the late stage, however, negative immune regulation predominates, resulting in the formation of an immunosuppressive microenvironment. This intricate network reprograms cancer biology in multiple aspects and serves as a potential target for cancer treatment. In this review, we briefly outline the current HRD tests and genetic characteristics of HRD cancers, focusing on breast, ovarian, pancreatic, and prostate cancers. We then summarize the interactions and crosstalk between immune cells and cancer cells, as well as various signaling pathways within the TME of HRD cancers. Additionally, we highlight recent advances in combining PARP inhibitors with immunotherapies in preclinical models and clinical trials of HRD cancers. This review provides valuable insights and perspectives into the distinct landscape of TME in HRD cancers, and offers a rationale for expanding the application of this combined therapeutic approach to a broader range of HRD cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befa4ba7c7bbd20ccb4cc8148dd06bbf9c57c776" target='_blank'>
              The distinct landscape of tumor immune microenvironment in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Qiuyang Xu, Yuanjia Wen, Taiyuan Huang, Huayi Li, Xingzhe Liu, Shen-nan Shi, Wenjian Gong, Gordon B. Mills, Ding Ma, Qing-cui Gao, Yong Fang
          </td>
          <td>2025-08-20</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions that drive progression and pathobiology. The aim of this study was to understand how these spatial patterns and interactions evolve with treatment.


METHODS
To explore these relationships, we employed imaging mass cytometry to measure the expression of 34 protein markers, enabling the identification of GBM-specific cell types and their interactions at single-cell protein level in paired primary (pre-treatment) and recurrent (post-treatment) GBM samples from five patients.


RESULTS
We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes.


CONCLUSIONS
These findings provide insight into treatment-induced shifts in GBM's cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fcab07cdc7f1fdc819d108cb3fdc13fdcffb099" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment.
              </a>
            </td>
          <td>
            Shoaib Ajaib, Joshua Winter-Luke, Richard J Digby, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F Stead
          </td>
          <td>2025-09-01</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Metastasis remains the leading cause of cancer-related death, yet the biological determinants that enable tumor cells to disseminate and colonize distant organs are incompletely understood. Emerging evidence identifies the microbiome, not merely as a bystander, but as an active architect of the metastatic cascade. Microbial communities residing in the gut, mucosal barriers, and within tumors shape metastatic progression by modulating immune surveillance, stromal remodeling, oncogenic signaling, and therapy response. Intratumoral and even intracellular microbes regulate epithelial–mesenchymal transition, angiogenesis, and immune escape, while gut-derived metabolites condition pre-metastatic niches and alter systemic immunity. Technological advances in spatial transcriptomics, single-cell multi-omics, and metagenomics have revealed a spatially organized, functionally integrated microbial ecosystem within tumors, challenging long-held assumptions of sterility in cancer biology. This review synthesizes five converging dimensions of this paradigm: microbial interactions in the metastatic tumor microenvironment; microbiome-mediated immunoediting and metastatic escape; the role of intratumoral and intracellular bacteria in dissemination; spatial-multi-omic approaches to map microbial niches; and microbial biomarkers predictive of metastasis and therapy outcomes. Collectively, these findings recast the microbiome as a critical and targetable determinant of metastasis. Deciphering the tumor–microbe–host triad holds transformative potential for biomarker development, therapeutic innovation, and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15720d6be643a5b1bcd7bdc88b245af42661e241" target='_blank'>
              Microbial signatures in metastatic cancer
              </a>
            </td>
          <td>
            Jhommara Bautista, María Paula Fuentes-Yépez, Joseth Adatty-Molina, Andrés López-Cortés
          </td>
          <td>2025-09-30</td>
          <td>Frontiers in Medicine</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) are ectopic lymphoid aggregates often found in chronic inflammatory conditions, including cancer. These structures, which share many cellular and functional features with secondary lymphoid organs, can profoundly influence the tumor microenvironment by promoting local anti-tumor immune activation. TLSs have been observed in various cancers, including melanoma, and are associated with improved responses to immunotherapy and clinical outcomes. However, our understanding of the molecular mechanisms underlying TLS formation and function remains incomplete. This review summarizes the current findings on TLSs in human melanoma, drawing from multiple studies to provide an updated overview. We discuss the cellular composition, spatial distribution, and genetic signatures of TLSs at different stages of melanoma pathogenesis and in subtypes including acral, uveal, and desmoplastic melanoma. Additionally, we examine the influence of tumor mutational burden (TMB) and complement activation on TLS formation, as well as the role of TLSs in immune checkpoint inhibitor therapy. We also highlight the potential of TLSs as indicators for disease progression and treatment response, and review preclinical strategies aimed at inducing TLSs to improve therapeutic outcomes. This synthesis aims to support ongoing research into the role of TLSs in melanoma immunobiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cd5d327c6ce7e444f0890b04b9badc06dcf384c" target='_blank'>
              Tertiary Lymphoid Structures in Human Melanoma: Molecular Mechanisms and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Gelare Ghajar-Rahimi, Ishika Patel, Nabiha Yusuf
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the most frequent and aggressive primary brain tumor in adults, with a high recurrence rate even after standard treatment, which includes surgery, radiotherapy and chemotherapy. This recurrence is associated with the tumor’s ability to infiltrate brain tissue beyond the margins visible on neuroimaging, making complete resection nearly impossible. In a prospective study, conducted in collaboration with Hospital Universitario 12 de Octubre, we analyzed a cohort of radiologically-guided biopsies, including regions of tumor infiltration, vasogenic edema and even apparently normal areas. Using DAB immunohistochemistry for MIB-1, we detected proliferative tumor cells not only in the tumor core but also in the non-infiltrated peripheral zone. This finding prompted us to further characterize the tumor microenvironment in these areas.



 We developed a multiplex sequential immunofluorescence (mIF) protocol based on the cyclic elimination of antibodies using 2-mercaptoethanol and SDS. Although pre-designed commercially panels exist for this type of approach, we designed custom panels focused on studying peripheral GBM regions, including specific markers for tumor, immune, glial and endothelial cells. This method enables detailed spatial characterization using confocal microscopy. In parallel, and with the aim of increasing accessibility of this technology, we developed a complementary protocol based on multiplex sequential immunohistochemistry with chromogens (mIC), compatible with conventional light microscopy. In this case, chromogen removal between rounds is performed by washing in absolute ethanol, and antibody stripping is achieved using citrate buffer pH 6. This process allows for the complete removal of previous markers and the application of new rounds of chromogenic staining on the same sample.



 These two approaches complement each other depending on the analytical needs. While mIF enables higher multiplexing capacity and spatial resolution, its implementation requires specialized equipment. In contrast, mIC offers a simpler and more affordable solution, suitable for standard laboratory settings, though with reduced capacity for simultaneous labeling.



 This combined approach allows for a more precise study of peripheral GBM areas, providing key insights into its biology and potential mechanisms of recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c97a6d86c3a3fa9f7b4a456443488a43ca7d849e" target='_blank'>
              P02.10.A CHARACTERIZATION OF THE TUMOR MICROENVIRONMENT IN PERIPHERAL AREAS OF GLIOBLASTOMA USING MULTIPLEX SEQUENTIAL PROTOCOLS.
              </a>
            </td>
          <td>
            M. Ramírez González
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are cells shed from tumors into the bloodstream, providing a unique source of information on tumor biology. CTCs can be collected through liquid biopsy, offering a noninvasive option to track cancer progression and treatment responses. Whereas ctDNA gives valuable insights into genomic alterations, CTCs allow for the direct analysis of cellular protein and RNA expression, essential for identifying targetable markers and resistance mechanisms. Resistance to cancer therapies is often due to genetic and phenotypic adaptations within tumor cells. CTC analysis can help track these adaptations, providing real-time insights into evolving resistance mechanisms and suggesting alternative therapies.


SIGNIFICANCE
Enumeration and molecular characterization of CTCs provide unique insights into the biology of cancer metastasis as well as clinically relevant information on tumor evolution, risk assessment, and therapy response in individual patients with solid malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80509a0d0dd4086ca6871a4474226159831e914e" target='_blank'>
              From Discovery to Diagnosis: A Perspective for Circulating Tumor Cells in Personalized Oncology.
              </a>
            </td>
          <td>
            C. Alix-Panabières, Klaus Pantel
          </td>
          <td>2025-08-18</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="


 KRAS^G12D mutations represent a significant oncogenic driver, prevalent in highly aggressive malignancies such as pancreatic ductal adenocarcinoma (PDAC) and colorectal cancer (CRC). The recent development of direct KRAS^G12D inhibitors has ushered in a new era of targeted therapy, offering unprecedented opportunities to address these previously intractable cancers. This study aimed to explore and optimize therapeutic strategies by thoroughly evaluating the efficacy of MRTX1133, a novel selective non-covalent KRAS^G12D inhibitor, in conjunction with various immunotherapeutic modalities, particularly those involving natural killer (NK) cells.



 We utilized patient-derived xenograft and syngeneic murine cancer models to systematically characterize alterations of immune landscapes within tumor microenvironment (TME) upon KRAS^G12D inhibition. Comprehensive and in-depth assessment of immune cell function and single-cell level transcriptional activity was performed using flow cytometry and single-cell RNA sequencing analysis.



 Our findings reveal that MRTX1133 not only suppresses tumor progression but also profoundly enhances the activation of infiltrated natural killer (NK) cells within the tumor microenvironment. Functional depletion of NK cells partially, yet significantly, attenuated the anti-tumor activity of MRTX1133, underscoring the contribution of NK cells to the observed therapeutic benefit. Notably, KRAS^G12D inhibition led to a reduction in tumor-infiltrating Gr-1^+ myeloid-derived suppressor cells (MDSCs) and increased the proportion of effector NK cells. Furthermore, MRTX1133 facilitated greater intratumoral accumulation of adoptively transferred NK cells and counteracted tumor-induced systemic immunosuppression on NK cells. Combined treatment with MRTX1133 and NK cell immunotherapies produced synergistic anti-tumor effects and significantly extended survival in preclinical models. Mechanistically, ex vivo analyses demonstrated that MRTX1133 upregulates NK cell-activating ligands, including MICA/B and ULBPs, and suppresses the production of cytokines such as IL-6 and GM-CSF by cancer cells, thereby enhancing tumor cell susceptibility to NK cell-mediated cytotoxicity and reducing MDSC proliferation.



 Collectively, our results illuminate the interplay between KRAS^G12D inhibition and immuno-microenvironmental remodeling, highlighting rational combinations of targeted and NK cell-based immunotherapies as promising strategies to potentiate therapeutic responses in KRAS^G12D-driven cancers, potentially improving clinical outcomes for patients.



 Hongyuan Chen, Tuo Hu, Chao Xu; Liangjie Chi, Fangqin Xue, Chunbo He. KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aa888d3ef7c73bc5df1193380be3a3238a9fcd" target='_blank'>
              Abstract A016: KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity
              </a>
            </td>
          <td>
            Hongyuan Chen, Tuo Hu, Chao Xu, Liangjie Chi, Fangqin Xue, Chunbo He
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Breast cancer biology is influenced by both genetic ancestry and the environment with implications for disease outcomes. However, we lack the knowledge how non-cancerous cells within tumors are functionally altered by these factors at single-cell resolution. Here, we created a single-nucleus (sn) dataset from 33 African American, 25 Kenyan, and 24 European American women with breast cancer using combined RNA and ATAC sequencing. We successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue using an optimized combination of enzymatic digestion and automated tissue homogenization. SnMultiome sequencing of 82 tumors was performed using the 10x Genomics platform. Following filtering, normalization, peak calling, and integration, our dataset includes a total of 292,458 nuclei. Cancerous (163,419 nuclei) and non-cancerous (129,039 nuclei) cells were distinguished based on DNA copy number. Within the microenvironment, 11 major immune, epithelial, and stromal cell types were identified. CD45+, EPCAM- cells were further analytically extracted, clustered, and manually annotated based on known marker genes, detecting 26 distinct immune subpopulations across our samples. After adjustment for potential confounders (age, body mass index, and tumor molecular subtype) in all analyses, we found that women of African descent possessed distinct, activated mesenchymal and endothelial cell subpopulations within the tumor vasculature that promote inflammation, angiogenesis, and cancer cell invasion. Women of African descent exhibited a more immune-suppressive tumor microenvironment, with elevated expression of immune checkpoint markers (PD-1, TIGIT, CTLA-4, and LAG-3), functional impairment of dendritic cells, and enrichment of regulatory and tolerogenic lymphocytic subpopulations. Further, malignant cells from African descent patients expressed a discrete set of CAR-T targets with potential implications for precision immunotherapy. Lastly, we developed three gene signatures within the microenvironment that are predictive of survival with external validation in both tumor-adjacent normal and cancerous tissues from the TCGA. Together, we uncovered molecular breast tumor characteristics of clinical significance in women of African ancestry.



 Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H. Dorsey, Eytan Ruppin, Shahin Sayed, Francis Makokha, Gretchen L. Gierach, Stefan Ambs. Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b4c16fead2784b8ef235041a0b9c641f582e01" target='_blank'>
              Abstract C109: Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H Dorsey, E. Ruppin, Shahin Sayed, Francis W. Makokha, Gretchen L. Gierach, S. Ambs
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="


 The glioblastoma (GBM) tumor immune microenvironment (TIME) is an immunosuppressive barrier to therapeutic innovation, and unsurprisingly GBM responses to immune checkpoint inhibition (ICI) are rare. Here, we define therapeutic vulnerabilities and mechanisms underlying GBM resistance to ICI therapy.



 Integration of spatial and single-cell approaches from human IDH-wildtype GBMs before and after ICI treatment with preclinical genetic, immunologic, and efficacy studies in immunocompetent mouse GBM models.



 Human GBMs with radiographic responses to ICI had lower pre-treatment IL6 levels on spatial profiling than GBMs resistant to ICI. Multiplexed immunofluorescence revealed ICI-responders were enriched in T-cell proteins (CD3, CD4, CD8) and activation markers (CD25) at the time of salvage surgery; meanwhile, non-responders were enriched in myeloid proteins (CD68, CD163, CD11c). Single-cell RNA sequencing analysis of 32,877 cells from human GBMs showed IL6 is predominantly produced by radial glial like cancer stem cells. Interestingly, mice bearing intracranial SB28 GBM allografts universally suppressed intratumor IL6 levels when treated with immunostimulatory gene therapies and mouse genetic experiments confirmed prolonged survival in animals with an IL6-/- background. Mouse survival was improved by combination systemic treatment with anti-PD1 and anti-IL6 but was not improved by either antibody alone or by local anti-IL6 delivery. Mass cytometry revealed combination therapy reprogrammed the GBM TIME by increasing MHCII+ monocytes, CD103+ migratory dendritic cells (DCs), CD11b+ conventional DCs, and effector CD8+ T cells, and by decreasing immunosuppressive Tregs.



 Systemic anti-IL6 in combination with systemic ICI reprograms the GBM TIME to sensitize preclinical models to ICI, shedding light on a therapeutic strategy for GBM treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2360ab571d81347b402e726648495085f63cb772" target='_blank'>
              P02.02.A IL6 UNDERLIES MICROENVIRONMENT IMMUNOSUPPRESSION AND RESISTANCE TO THERAPY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Young, N. Cho, C. G. Lucas, K. Seo, H. Najem, K. Mirchia, W. C. Chen, N. Zakimi, V. Daggubati, T. Casey-Clyde, M P Nguyen, A Chen, J. J. Phillips, T. Ozawa, M. Aghi, J. Taylor, J. DeRisi, A. Bhaduri, A. Heimberger, M. Berger, N. Butowski, M. H. Spitzer, D. Raleigh
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="The awakening of dormant disseminated cancer cells appears to be responsible for the clinical relapses of patients whose primary tumors have been successfully cured months and even years earlier. In the present study, we demonstrate that dormant breast cancer cells lodged in the lungs reside in a highly mesenchymal, nonproliferative phenotypic state. The awakening of these cells is not triggered by a cancer cell-autonomous process. Instead, lung inflammation induced by the chemotherapeutic agent bleomycin effectively awakens dormant cancer cells, providing useful models for studying metastatic awakening. Mechanistically, the awakened cells shift from a highly mesenchymal to a quasi-mesenchymal phenotypic state in which they acquire tumorigenicity and proliferative ability. Once awakened, these cells can stably reside in this quasi-mesenchymal state and maintain their tumor-initiating ability, doing so without ongoing heterotypic signaling from the lung microenvironment. Epidermal growth factor receptor ligands released by the cells of the injured tissue microenvironment, including notably M2 type macrophages, promote dormant cancer cells to move toward this quasi-mesenchymal state, a transition that is critical for the awakening process. An understanding of the mechanisms of metastatic awakening may lead in the future to treatment strategies designed to prevent such awakening and resulting metastatic relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0978c66611eb319088a607dd5da99f325c7f34ff" target='_blank'>
              Inflammation awakens dormant cancer cells by modulating the epithelial-mesenchymal phenotypic state.
              </a>
            </td>
          <td>
            Jingwei Zhang, Jingwen Zhang, Longfei Han, Shiyi Wu, Jie Li, E. Eaton, Bingbing Yuan, Ferenc Reinhardt, Hao Li, Patrick C Strasser, Sunny Das, Joana Liu Donaher, Md Imtiaz Khalil, Haiping Jiang, Alexander Deuschel, Danni Lin, Carolin Sebastiany, Mariana Maranga, Salomé Shubitidze, Xiaofei Liu, Arthur W Lambert, Yun Zhang, Yana Liu, L. Sui, Sarah Elmiligy, Umberto Pozza, Rauf Günsay, Ranjan Mishra, José Velarde, Sonia Iyer, Whitney S. Henry, K. Weiskopf, Guihai Feng, Tobiloba E. Oni, R. Watnick, Xin Li, Robert A. Weinberg
          </td>
          <td>2025-09-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="


 TCR-engineered T cell (TCR-T) therapy holds significant promise in cancer treatment, allowing precise targeting of intracellular antigens. However, its application in neuro-oncology, particularly in glioblastoma (GB), faces unique challenges. In addition to the restrictive nature of the blood-brain barrier and a profoundly immunosuppressive tumor microenvironment, the comparatively low mutational burden of GB limits the availability of immunogenic targets, collectively hampering the success of immunotherapy in primary brain tumors. Despite these challenges, recent vaccine studies have demonstrated that both tumor-specific neoantigens and non-mutated, overexpressed glioma-associated antigens (GAAs) can induce robust and durable T cell responses in patients with GB. In tumors exhibiting high clonal heterogeneity and a limited number of shared mutations, the ability to exploit a broader antigenic repertoire may be critical for therapeutic success. To broaden the clinical applicability of TCR-T cell therapy for GB, there is an urgent need to develop TCRs targeting diverse GAAs across multiple HLA backgrounds.



 To this end, we aimed to identify and functionally validate GAA-specific TCRs from vaccinated GB patients. PBMCs from a single patient enrolled in the GAPVAC-101 vaccine trial were expanded in vitro using peptide pools spanning immunogenic GAA epitopes. Following peptide-specific expansion, GAA-reactive T cells were enriched by FACS and subjected to single-cell V(D)J sequencing to retrieve paired TCR α/β chain information. Dominant clonotypes were selected based on frequency, and corresponding TCRs were cloned into a lentiviral backbone via Golden Gate Assembly. The cloned TCRs were expressed in CD4⁺/CD8⁺ Jurkat reporter cells for functional evaluation. Following TCR expression, Jurkat cells were co-cultured overnight with patient-derived B-lymphoblastoid cell lines (B-LCLs) to assess TCR reactivity. Reactivity was evaluated the next day by flow cytometry through staining for CD69 surface-expression and analysis of NFAT-GFP reporter activity.



 Analysis of the TCR repertoire from peptide-expanded, FACS-sorted CD4⁺ and CD8⁺ T cells revealed an oligoclonal pattern, consistent with antigen-driven expansion. Utilizing material from the GAPVAC-101 trial, we identified both CD4⁺ and CD8⁺ TCRs reactive to GAAs. Selected TCRs were successfully cloned and lentivirally expressed in Jurkat reporter cells, where they exhibited antigen-specific activation in vitro, confirming functionality.



 These findings demonstrate the feasibility of isolating functional, GAA-specific TCRs from vaccinated GB patients. While further validation in vitro and in vivo is necessary, this approach provides a foundation for expanding the repertoire of TCRs available for personalized TCR-T cell therapy in GB.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c62c351184571592a0c8f085fde0a7f6110ba1f" target='_blank'>
              P08.13.A IDENTIFICATION OF VACCINE-INDUCED TCRS AGAINST GLIOBLASTOMA-ASSOCIATED ANTIGENS
              </a>
            </td>
          <td>
            P. Koopmann, Y. Chih, A. Dietsch, K. Lindner, S. Stange, I. Poschke, T. Boschert, E. Green, L. Bunse, W. Wick, M. Platten
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains one of the most lethal primary brain tumors, characterized by profound immunosuppression and resistance to conventional immunotherapy. A hallmark of the GBM microenvironment is the prevalence of functionally exhausted T cells, marked by sustained expression of inhibitory receptors such as PD-1 and impaired effector function. These exhausted T cells represent a major barrier to durable anti-tumor immunity and effective immunotherapeutic intervention. To overcome this, we investiagated the therapeutic potential of PD1-IL2v, a novel immunocytokine designed to selectively deliver IL-2 signaling to PD1-1+ cells while avoiding expansion of regulatory T cells (Tregs).



 We employed two murine models of glioblastoma: GL261, a commonly used immunogenic model, and SB28, an aggressive, poorly immunogenic model that is resistant to immune checkpoint blockade. PD1-IL2v was administered either systemically or intratumorally. Flow cytometry was used to phenotypically characterize tumor-infiltrating lymphocytes (TILs), with a focus on exhaustion markers and effector phenotypes. In the SB28 model, we additionally evaluated the combination of PD1-IL2v with radiotherapy and temozolomide to reflect the current clinical standard of care.



 Local administration of PD1-Il2v, but not systemic delivery, effectively reactivated CD8⁺ T cells within the tumor microenvironment. Treated mice exhibited enhanced cytokine production, increased proliferation, and restoration of cytotoxic activity in T cells. Phenotypic profiling revealed an expansion of a stem-like subset of exhausted T cells (PD-1⁺TCF1⁺), alongside the emergence of a novel, differentiated population with superior effector functions—termed “better-effector” T cells. Therapeutic intervention with PD1-IL2v led to prolonged survival, complete tumor rejection in a subset of animals, and increased infiltration of functional T cells into the tumor core. Furthermore, we evaluated the efficacy of PD1-IL2v in combination with radiotherapy and temozolomide—the standard of care for glioblastoma—in the SB28 model.



 Our findings underscore T cell exhaustion as an untapped reservoir of tumor-reactive T cells that can be therapeutically reinvigorated using PD1-IL2v. This approach not only restores anti-tumor immunity but also induces superior effector populations capable of driving tumor regression.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3ff9cf91d6af3eacd96055961331fe5ae9fe0e3" target='_blank'>
              P08.21.A REINVIGORATING EXHAUSTED T CELLS IN GLIOBLASTOMA VIA PD1-IL2V IMMUNOCYTOKINE THERAPY
              </a>
            </td>
          <td>
            U. Salazar, B. Taskoparan, L. Codarri Deak, V. Nicolini, C. Klein, H. Okada, J. vom Berg
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC), the most prevalent and lethal ovarian cancer subtype, is often diagnosed at an advanced stage due to the lack of effective early detection methods. Current evidence suggests that HGSOC originates from the fallopian tube (FT), specifically from secretory epithelial cells in the fimbriated region, based on the detection of early serous lesions and clonal genetic relationships linking them to HGSOC tumors. However, efforts to improve early diagnosis and interception are hindered by an incomplete understanding of HGSOC’s cellular and molecular origin. To address this, we established a diverse, clinically and genetically annotated living FT organoid biobank at the Mayo Clinic, encompassing specimens from over 200 patient donors. Using robust, multi-step protocols, we enriched FT epithelial stem/progenitor cells from tissue samples and Tao brushings, generating bi-lineage differentiated organoids independent of anatomical site (ampulla, fimbria) or laterality (right, left). To gain deeper insight into FT epithelial lineage dynamics, we applied multi-omic modalities—including single-cell RNA sequencing (scRNA-seq), single-nucleus RNA and ATAC sequencing (snMultiome), and bulk RNA-seq—to fresh cells and pooled organoids, integrating findings with public FT datasets to generate the largest known integrated single-cell atlas of normal and high-risk FTs. This enabled the generation and validation of gene expression-derived signatures for secretory and multiciliated lineages, identification of key lineage-defining transcription factors and their regulatory networks, and validation using immunohistochemistry (IHC) data from the Human Protein Atlas. Subcluster-level analysis of epithelial cells revealed transcriptionally distinct populations with unique biological functions, deepening insight into epithelial heterogeneity and FT lineage differentiation. Further, analysis of FT epithelial subclusters identified a rare population with transcriptomic and proteomic association with the mesenchymal-like HGSOC subtype. These cells retained epithelial identity while expressing mesenchymal and ECM-related genes, forming a hybrid phenotype. Spatial profiling via IHC localized them to a rare, basal, peg-like position within the FT epithelium. Developmental analysis showed shared transcriptional features with fetal mesonephric cells, suggesting both developmental plasticity and potential vulnerability to transformation. Pooled organoid scRNA-seq confirmed that they are preserved in culture, indicating they are stably maintained ex-vivo. Together, these findings suggest that this cell cluster represents a rare, multipotent epithelial population with mesenchymal features, offering a candidate cellular origin for the mesenchyme-like subtype of HGSOC. In conclusion, this living FT organoid biobank serves as a powerful, clinically annotated platform for dissecting epithelial lineage hierarchies, identifying rare cell populations, and uncovering mechanisms that may underlie subtype-specific ovarian carcinogenesis.



 Megan Ritting, Syed Mohammed Musheer Aalam, Mihai Dumbrava, Wazim Mohammed Ismail, Alexandre Gaspar-Maia, Mark Sherman, Jamie Bakkum-Gamez, Nagarajan Kannan. Tubal organoid biobanking and multi-omics framework uncovers rare basal peg cells as a source of epithelial stemness and a candidate for high-grade serous ovarian cancer origin [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3ee6c648a1ddd232c6228d410b714c8c6567b27" target='_blank'>
              Abstract A078: Tubal organoid biobanking and multi-omics framework uncovers rare basal peg cells as a source of epithelial stemness and a candidate for high-grade serous ovarian cancer origin
              </a>
            </td>
          <td>
            Megan L. Ritting, Syed Mohammed Musheer Aalam, Mihai G. Dumbrava, W. Ismail, Alexandre Gaspar-Maia, Mark Sherman, Jamie N. Bakkum-Gamez, Nagarajan Kannan
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Renal cancer, particularly clear cell renal cell carcinoma (ccRCC), is characterized by significant intratumoral heterogeneity, which poses challenges for diagnosis and treatment. Single-cell sequencing (SCS) provides unprecedented insights into the cellular landscape of renal cancer, allowing for detailed characterization of tumor heterogeneity at the single-cell level. This review highlights how SCS has been instrumental in elucidating the origins of different renal cancer subtypes, understanding mechanisms of tumor initiation and progression, and dissecting the complex tumor microenvironment (TME). It discusses the identification of novel biomarkers and therapeutic targets, as well as the potential of SCS to inform personalized treatment strategies. The review also explores the integration of SCS with spatial omics technologies, which enhances the understanding of cellular interactions within their spatial context. Moreover, it addresses the challenges and future directions in applying SCS to clinical practice, emphasizing its significance in advancing renal cancer biology and improving clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03a5f0a7b414a81333643cfcf0b78e2ab4708d98" target='_blank'>
              Single-cell sequencing technology in renal cancer: insights into tumor biology and clinical application
              </a>
            </td>
          <td>
            Hanzhong Zhang, Ying Liu, Wenqiang Liu, Anqi Lin, Yu Fang, Le Qu, Xu Zhang, Peng Luo, Linhui Wang, Aimin Jiang
          </td>
          <td>2025-10-01</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Macrophages are key components of the tumor microenvironment and play a critical role in melanoma progression. However, the dynamics of different macrophage subsets, particularly CD68+ and CD163+ populations, in relation to tumor thickness and stage remain insufficiently characterized. Objective: This study aimed to quantitatively assess intratumoral and peritumoral CD68+ and CD163+ macrophages in cutaneous melanoma and to investigate their associations with Breslow thickness, pT stage, and AJCC stage. Methods: We conducted a retrospective cohort study of 126 patients with cutaneous melanoma (AJCC stages IA–IIID). Tumor samples were examined histologically and immunohistochemically for CD68+ and CD163+ macrophages, and quantitative analysis was performed in intratumoral and peritumoral regions. Results: This study included 126 patients with cutaneous melanoma, ranging in stage from IA to IIID. Histopathological analysis revealed that melanoma tissues were primarily composed of irregular clusters of atypical melanocytic cells infiltrating the dermis and subcutaneous fat. Immunohistochemical staining identified CD68+ and CD163+ macrophages both within the tumor and in the surrounding stroma. Macrophage infiltration significantly increased with tumor thickness (Breslow) and progression to more advanced stages. Specifically, at Breslow thickness <1.0 mm, the mean number of CD68+ macrophages in the intratumoral zone was 29.7 ± 4.3 cells, increasing to 70.3 ± 6.4 cells in tumors >4.0 mm. CD163+ macrophages showed similar trends, with a rise from 15.6 ± 2.8 cells at <1.0 mm to 39.8 ± 4.6 cells at >4.0 mm in the intratumoral zone. Additionally, macrophage density was higher in tumors with ulceration, and both CD68+ and CD163+ macrophage numbers increased progressively with tumor stage, particularly in advanced stages. The number of CD68+ macrophages at stage IA in the intratumoral zone was 21.6 ± 3.1 cells and increased to 56.4 ± 6.8 cells at stage IIID, while CD163+ macrophages rose from 13.8 ± 3.2 cells at stage IA to 36.4 ± 4.6 cells at stage IIID. This suggests that macrophage infiltration, particularly CD163+ cells, correlates with melanoma progression. Conclusions: Our findings highlight distinct spatial and phenotypic patterns of macrophage infiltration in melanoma progression. The prominent increase in CD68+ and CD163+macrophages suggests their important role in tumor-associated immunomodulation. Further studies are warranted to elucidate macrophage polarization states and their prognostic and therapeutic implications in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad4969416cee26205b7a0763813b82a39e29b3d3" target='_blank'>
              Spatial Distribution and Phenotypic Profiling of Cd68+ and Cd163+ Macrophages in Melanoma Progression: Insights into Tumor Microenvironment Dynamics
              </a>
            </td>
          <td>
            G. Demyashkin, D. Atiakshin, K. Silakov, V. Shchekin, Maxim Bobrov, M. Vadyukhin, Tatyana Borovaya, Ekaterina Blinova, Petr Shegay, Andrei Kaprin
          </td>
          <td>2025-09-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Glioma intratumoral heterogeneity remains a critical barrier to effective treatment, driving recurrence and resistance to therapy. Emerging evidence suggests that glioma cells acquire neural-like features through paracrine and synaptic communication with neural cells, fostering functional diversity within tumors. While neuro-cancer interactions are implicated in glioma heterogeneity, their precise roles remain incompletely synthesized. Here, by consolidating these discoveries, a conceptual framework is proposed for understanding the glioma intratumoral heterogeneity shaped by neuro-cancer interactions: cellular phenotypes, spanning neurogliomal synapses, glioma networks, and neuronal-like motility; and transcriptional states, which exhibit remarkable resemblance to neural cells. These diverse cellular phenotypes and transcriptional states synergistically fuel glioma progression, invasion, and resistance. By emphasizing the converging phenotypical and transcriptional evidence with spatial context within them, an underexplored but critical role of neuro-cancer interactions are proposed in glioma intratumoral heterogeneity. This provides potential strategies to explore and disrupt these neuro-cancer interactions, offering new insights to address glioma intratumoral heterogeneity for improved therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7eca8bd6e6e1b76e76ba8a64aea76418740245" target='_blank'>
              Neuro-Cancer Interactions Shape Glioma Intratumoral Heterogeneity.
              </a>
            </td>
          <td>
            Tong Wu, Peiran Zhao, Zheng Chen, Tianlei Ying, Zhihong Nie, Wei Hua, Huiyan Li, Ying Mao, Wenbo Bu
          </td>
          <td>2025-08-21</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Glioma represents the most prevalent primary tumors of the central nervous system, originating from glial cells. Cancer stem cells have the ability to extensively proliferate, self-renew, and form colonies, which contribute to tumorigenesis. Studies have found a population of cells within glioblastoma exhibiting characteristics similar to those of cancer cells, termed glioma stem cells (GSCs). GSCs have two distinct phenotypes: mesenchymal subtype (MES) and proneural subtype (PN). Despite the vital role of these subtypes in glioma biology, there is a significant lack of comprehensive reviews focusing on the regulatory mechanisms underlying each phenotype.This review integrates emerging insights into the regulatory mechanisms underlying GSCs plasticity, with dedicated analysis of novel pathways governing PN and MES phenotypes and their dynamic transition. By examining these critical elements, we aim to contribute to the development of novel therapeutic strategies in the ongoing fight against gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50da6e5130555aeaeba55f2b83f42650df5fbf68" target='_blank'>
              Phenotypic variations in glioma stem cells: regulatory mechanisms and implications for therapeutic strategies
              </a>
            </td>
          <td>
            Guofeng Tian, Yifu Song, Yaochuan Zhang, Liang Kan, Ana Hou, Sheng Han
          </td>
          <td>2025-09-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfc80059f8767909d0c8bd930a6c33c0803bbb" target='_blank'>
              Preservation and Clonal Behavior of Extrachromosomal DNA in Patient-Derived Xenograft Models of Childhood Cancers.
              </a>
            </td>
          <td>
            Rishaan Kenkre, Jon D. Larson, Owen S. Chapman, J. Luebeck, Yan Yuen Lo, Megan Paul, Wenshu Zhang, V. Bafna, Robert J Wechsler-Reya, Lukas Chavez
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="


 Approximately 40% of patients with pancreatic ductal adenocarcinoma (PDAC) receive no benefit from surgical resection and systemic chemotherapy and experience metastatic disease recurrence within 12 months. The extreme heterogeneity of the cells that populate PDAC tumors has been a significant impediment to identifying the cells that likely seed early recurrence. Previous studies show that metastatic spread is driven by specific cells with hybrid epithelial-mesenchymal characteristics, but despite the identification of numerous molecular features associated with epithelial, hybrid, and mesenchymal PDAC cancer cells, it has not been possible to specifically identify the cells that seed metastatic recurrence in human tissue. We have previously shown that PDAC cell glycan signatures are altered as a direct result of the metabolic and transcription changes that occur as PDAC cancer cells undergo EMT. Now we integrated the glycan signatures with known protein markers and morphological analyses using our pipeline for multiplexed immunofluorescence analysis to identify distinct subpopulations of epithelial, hybrid, and mesenchymal PDAC cells. In cell-level analyses of immunofluorescence images of 24 PDAC cell lines, these subpopulations revealed epithelial to hybrid to mesenchymal conversions along 3 distinct trajectories, which were defined by transcription factors and proteins previously associated with epithelial or mesenchymal states—TP53, TP63, GATA6, and MYC—and further distinguished by specific glycan signatures and protein markers of metabolism and cellular differentiation. The integrated transcription factor, glycan, and protein signatures identified the previously described classical and squamous PDAC subpopulations and defined a new signature for a fully mesenchymal subpopulation. The signatures differentiated between PDAC subpopulations in primary human tumors and in metastatic liver lesions and showed a patient-matched correspondence between the primary tumors and metastatic lesions in specific hybrid PDAC cells within distinct trajectories, potentially revealing the PDAC cells seeding metastatic recurrence. These results suggest that the development of mesenchymal and metastatic PDAC cells occurs through distinct, stepwise trajectories with divergent morphological, metabolic, molecular, and glycan-based signatures. These findings will/have the potential to inform the development of treatments that specifically target the PDAC cells that seed early metastatic recurrence.



 ChongFeng Gao, Zachary Klamer, Ali Moursy, Galen Hostetter, Paul M. Grandgenett, Peter Allen, Brian Haab. Resolving PDAC cell subpopulations that seed metastatic recurrence [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B125.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246925305285b4b397bb54f4914db32c1466cbe8" target='_blank'>
              Abstract B125: Resolving PDAC cell subpopulations that seed metastatic recurrence
              </a>
            </td>
          <td>
            Chongfeng Gao, Zachary L. Klamer, Ali Moursy, G. Hostetter, P. Grandgenett, Peter Allen, Brian Haab
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The thymidine analog EdU (5-ethynyl-2-deoxyuridine) is incorporated into DNA during replication and has traditionally been used as a marker of S-phase cells. In this study, we discovered that EdU fluorescence images display substantial cell-to-cell variability, which could be classified into multiple clusters by unsupervised machine learning. This suggests that seemingly random EdU patterns contain reproducible, computationally recognizable features. Building on our observation that distinct patterns emerged in response to radiation stress, we investigated whether radioresistant cancer cells exhibit specific EdU signatures. Analysis of PLK1-overexpressing cells, which acquire radioresistance through altered DNA replication, revealed radiation-induced EdU patterns distinct from control cells. Prompted by the observation that these cells displayed markedly enlarged and intensified γ-H2AX foci, a marker of DNA damage, we employed a supervised machine learning model based on γ-H2AX patterns to isolate the radioresistant cell subpopulation. We then extracted the EdU signals from these isolated cells and, through further unsupervised machine learning, successfully identified a characteristic pattern specific to radioresistance. This establishes a machine learning framework capable of extracting universal rules from the dynamic networks that vary among individual cells, which provides a novel platform for a screening system to identify molecules involved in radioresistance, focusing on cancer heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0447884e0a398eddbbba5abf08e4dfe806602b07" target='_blank'>
              Machine Learning Extracts Radiation Resistant-Specific EdU Fluorescence Pattern in Cancer Cells.
              </a>
            </td>
          <td>
            M. Ikura, Tsuyoshi Ikura, Kanji Furuya
          </td>
          <td>2025-09-01</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is a highly lethal brain cancer with limited treatment options and poor long-term survival. Adoptive cellular therapy (ACT) has shown promise in targeting GBM, but many ACT strategies rely on patient tumor tissue to derive tumor-specific antigens, limiting their use in inoperable tumors. Further, tumor heterogeneity—primarily driven by glioma stem cells (GSCs) that survive standard treatments—complicates recurrence. Radial glial cells (RGCs), neurodevelopmental progenitors transcriptionally similar to a subpopulation of GSCs, express developmental antigens (DAs) also found in GBM. We hypothesized that RGCs harbor immunogenic antigens shared with GBM and can serve as a renewable, tumor-independent antigen source for ACT to target GSC cells.



 Using single-cell RNA sequencing, we profiled RGCs cultured from post-natal day 4 (P4) murine cortices and compared their transcriptomes with tumors from murine GBM model KR158B. Shared antigens were screened for immunogenicity using our O.R.A.N. antigen prediction and validation platform. We developed a cDNA antigen library from identified RGC immunogens and used this to generate in vitro transcribed RNA for ACT. T cells were expanded ex vivo using dendritic cells (DCs) pulsed with RGC RNA or total tumor RNA (ttRNA) and tested for cytokine production and tumor-specific killing. These RGC T cells were incorporated into our ACT platform alongside hematopoietic stem cell rescue and serial DC vaccinations. Survival was assessed in GBM-bearing mice. We also tested human RGC antigens against patient-derived xenograft (PDX) GBM models using human T cells and 3D bioprinted immune-tumor interaction assays.



 scRNA-seq analysis revealed a subset of immunogenic antigens shared between RGCs and GBM tumors. RGC T cells demonstrated robust IFNγ and TNFα release, and selectively killed tumor cells in vitro. In vivo, ACT incorporating RGC T cells significantly prolonged survival (p=0.0058) in KR158B GBM murine model, matching the efficacy of ttRNA ACT. Preliminary toxicity studies revealed no overt signs of neurotoxicity or systemic inflammation. In human models, RGC T cells recognized and killed a smaller subset of GBM PDX cells when compared to ttRNA T cells, but showed enriched targeting of GSC-like subpopulations in 3D bioprinted assays.



 RGC-derived antigens represent a potent source of targets for ACT in GBM that can be used to generate a renewable antigen library. This library bypasses the need for tumor resection while maintaining therapeutic efficacy. It offers a novel approach to overcoming initiation of tumor heterogeneity through targeting of neurodevelopmental cell types.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa2fac66b4517361b730352a54b7bf3ec8852fff" target='_blank'>
              OS05.5.A RADIAL GLIAL CELLS AS SOURCE ANTIGEN FOR ADOPTIVE CELLULAR THERAPY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            C. Love, J. W. Figg, I. West, M. Englebart, A. Reid, D. Jin, D. Mitchell, C. Flores
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor draining lymph nodes (TDLNs), as secondary lymphoid organs, are pivotal in initiating and regulating adaptive immune responses. Historically, TDLNs were recognized primarily as metastasis gateways in cancer, promoting radical dissection to prevent recurrence. However, emerging preclinical studies reveals their critical role in orchestrating systemic anti-tumor immune responses during cancer therapy, highlighting the dilemma of balancing lymph nodes (LNs) preservation with metastasis control. This review traces the evolving understanding of TDLN biology in oncology, from the era of radical LN dissection to multi-omics-driven insights, and synthesizes their dual roles as immune hubs and metastatic niches across first-line clinical therapies (e.g., immunotherapy, radiotherapy, chemotherapy, targeted therapy, etc.). We further propose the concept of “Lymph Node Multi-modal Protective Research (LNMPR)”, emphasizing the prospective value of integrating multi-omics technologies, including spatial transcriptomics, single-cell profiling, and imaging, to decode LN immune dynamics and optimize therapeutic responses. By bridging mechanistic insights with clinical strategies, LN-centric immune modulation may open up a new path for precise tumor treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ddb9cb8a82e08359fde743e98ea142d35dd6bc8" target='_blank'>
              From trash to treasure: tumor draining lymph nodes as a multi-omics goldmine in cancer therapy
              </a>
            </td>
          <td>
            Yina Li, Zihan Chen, Zhikun Guo, Jiangnan Yu, Jianan Lu, Lei Wang, Qian Zhou
          </td>
          <td>2025-08-19</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lung cancer in smokers (LCIS) and lung cancer in never-smokers (LCINS) are different entities with distinct molecular features. However, their cellular heterogeneity still requires further investigation. Through an integrated analysis of single-cell RNA sequencing (scRNA-seq) and bulk sequencing data, we identified cell subpopulations associated with smoking and non-smoking patients. Downstream transcriptomic analyses were then performed to reveal differences in cell function and tumor microenvironment. We observed that smoking-associated cancer cells exhibited a higher degree of aggressiveness, which may correlate with an adverse prognosis in smoking patients. Additionally, immunosuppressive CXCL10+ macrophages may be involved in their tumorigenesis in smokers, and the immunoregulatory LGALS9-HAVCR2 axis could be a potential immunotherapeutic target. In non-smokers, the inflammatory microenvironment may be involved in their tumor formation. Moreover, the decreased anti-tumor cytotoxicity could be associated with their suboptimal immunotherapeutic response. Our study uncovered differences in oncogenic and immune escape mechanisms between LCIS and LCINS patients and suggests potential immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6369e8d4f2741d426c67fe24a46863f596640138" target='_blank'>
              Integrating bulk and single-cell sequencing reveals cellular heterogeneity between lung adenocarcinoma in smokers and never-smokers.
              </a>
            </td>
          <td>
            Zihuan Zhao, Pan Yang, Yuhan Liu, Kai Wang, Xianfeng Xu, Yuzhuo Wang, Meng Zhu, N. Qin, Cheng Wang, Weimin Li, Hongxia Ma, Zhoufeng Wang, Hongbing Shen
          </td>
          <td>2025-08-25</td>
          <td>Journal of biomedical research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) in the tumor microenvironment (TME) act as local hubs for antigen presentation and lymphocyte activation, reinforcing antitumor immunity. They act as ectopic, lymph node-like niches that orchestrate local antitumor immunity through well-defined B-cell follicles, T-cell zones, follicular dendritic cells (FDCs), and high endothelial venules (HEVs). Although various cancer treatment methods can impact TLSs, thereby changing the state of the TME, the dynamic changes in TLSs caused by treatment and the mechanisms therein have not been comprehensively summarized to date. This review highlights the mechanisms of different cancer treatment effects on TLSs dynamic change, including the number and functionality of TLSs within tumors. We also summarize the value of TLSs in predicting prognosis and therapeutic effect, as well as the deficiencies in current research and future development directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b10ea5ef7732254210365008058829408071ec8" target='_blank'>
              Dynamic remodeling of tertiary lymphoid structures in response to cancer therapy: a recent review
              </a>
            </td>
          <td>
            Chongyu Tan, Jinliang Huang, Ning Gao, Bingquan Wu, Matsika Juliet, Jiatong Xiao, Jiao Hu, Ping Liu, Jinbo Chen
          </td>
          <td>2025-09-29</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0643ed748b04336ec828931abc9d5fbd2911609" target='_blank'>
              [Progress of scRNA-seq technology in nasopharyngeal carcinoma research].
              </a>
            </td>
          <td>
            Bin Zheng, Guanqiao Jin
          </td>
          <td>2025-09-01</td>
          <td>Lin chuang er bi yan hou tou jing wai ke za zhi = Journal of clinical otorhinolaryngology head and neck surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy driven by oncogenic KRAS and inflammation-induced cellular heterogeneity, yet the mechanisms underlying tumor-initiating cells (TICs) emergence and maintenance remain unclear. Our study establishes LY6D as a marker of persistent TICs that orchestrate pancreatic cancer progression across all disease stages. Single-cell RNA sequencing of inflammation-driven PDAC models reveals that LY6D+ TICs specifically originate from KRAS-mutant acinar-to-ductal metaplasia (ADM) lesions under inflammatory conditions, maintaining conserved stemness properties and exhibiting a distinctive oxidative phosphorylation (OXPHOS) dependency throughout tumor evolution. Moreover, genetic ablation of Ly6d in Kras G12D pancreata delays tumorigenesis, while forced Ly6d expression enhances tumorigenic potential and metastatic capability. Mechanistically, LY6D-despite lacking intracellular domains-scaffolds lipid raft-associated kinase networks and FOSL1-dependent epigenetic reprogramming to establish a ​stable pro-tumorigenic state. Clinically, LY6D+ cells are enriched in human PDAC and exhibit conserved stemness and epithelial-mesenchymal transition (EMT) properties. Strikingly, LY6D expression levels demonstrate PDAC-restricted prognostic power. Our work defines LY6D as pan-stage TICs marker linking cellular plasticity to PDAC initiation and progression, offering new avenues for early detection and interception of this lethal malignancy.



 Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue. LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B119.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f74adf609c9f9d814e1f1f768c348943a8c10b" target='_blank'>
              Abstract B119: LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis
              </a>
            </td>
          <td>
            Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor matrix stiffness plays a critical role in cancer progression, metastasis, and therapy resistance. Although traditional biophysical methods have shed light on the impact of matrix stiffness on tumor behavior, these techniques are confined to measuring the physical properties of the tumors. In this study, we leveraged RNA-seq data to predict tumor matrix stiffness, aiming to reveal mechanical properties by molecular signatures across various cancer types. To this end, we systematically analyzed RNA-seq data from tumors of varying stiffness levels to identify stiffness-associated gene signatures. With these molecular signatures, we developed a computational model for predicting tumor matrix stiffness and further applied it to The Cancer Genome Atlas (TCGA) dataset. Our analysis revealed significant differences in the tumor microenvironment as well as immune response between soft and stiff tumor samples, suggesting that tumor rigidity impacts not only cellular behavior but also characteristics of the tumor microenvironment. These findings underscore the potential of RNA-based stiffness models to enhance our comprehension of tumor mechanics and cancer biology, thereby facilitating the development of innovative targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd0a56130a8bfa1d6781b487c86488f3edefb603" target='_blank'>
              Pan-cancer analysis reveals molecular signatures for predicting matrix stiffness in solid tumors.
              </a>
            </td>
          <td>
            Gongyu Tang, Xinyi Liu, Yuanxiang Li, Yunfei Ta, Minsu Cho, Hua Li, Xiaowei Wang
          </td>
          <td>2025-09-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most frequent brain tumors in children and comprise a heterogeneous group of tumors with different locations, histologic subtypes, ages at presentation, and clinical behavior. Tumors frequently respond to treatment with chemotherapy or radiation therapy, but they can regrow after a period of quiescence, requiring further therapy. Thus, a deeper understanding of the molecular processes involved in these tumors is required to develop therapeutic strategies that are effective against their disease mechanisms. To better understand the cellular behaviors of this heterogenous group of tumors, we have employed single-cell and single-nuclei RNA sequencing technologies to analyze a large-scale dataset (>250,000 cells) of pLGGs. Analysis of this data identified a heterogenous population of cell types and cell states, detecting mature and progenitor-like astrocytes and oligodendrocytes, as well as cells exhibiting senescence or cycling programs. Moreover, we identify a significant immune infiltrate, comprised primarily of microglia. In addition to heterogeneity within pLGG tumors, heterogeneity between LGG subtypes represents another layer that stratifies pLGG biology. We performed a compositional analysis of the cell types present in these tumors and compared transcription signatures and gene expression programs across shared cellular populations of histologically and genetically distinct pLGGs. Finally, we optimized our integration and batch correction analyses by using external 293T cells as spike in controls during our single-cell and single-nuclei data generation steps to determine the most suitable method for batch-effect removal. Our analysis of human pLGGs at the single-cell and single-nuclei resolution provides critical insight into the heterogenous biological activities that constitute these tumors.



 Michelle Boisvert, Ashwyn Perera, Alexandra L. Condurat, John Jeang, Jessica W. Tsai, Dana Novikov, Kevin Zhou, Madison Chacon, Jeromy DiGiacomo, Rushil Kumbhani, Dayle Wang, Michael D. Taylor, Jordan R. Hansford, Louise Ludlow, Nada Jabado, Keith L. Ligon, Rameen Beroukhim, Pratiti Bandopadhayay, David T.W. Jones. Understanding the transcriptional heterogeneity of pediatric low-grade gliomas and its implication for tumor pathophysiology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87fff9552d8445c379c8cd3282dd22472d8f4d25" target='_blank'>
              Abstract A014: Understanding the transcriptional heterogeneity of pediatric low-grade gliomas and its implication for tumor pathophysiology
              </a>
            </td>
          <td>
            Michelle Boisvert, Ashwyn A. Perera, Alexandra L. Condurat, John Jeang, Jessica W Tsai, Dana Novikov, Kevin N Zhou, Madison S. Chacon, Jeromy J. Digiacomo, Rushil Kumbhani, Dayle K. Wang, Michael D. Taylor, J. Hansford, Louise E. Ludlow, Nada Jabado, Keith L. Ligon, R. Beroukhim, P. Bandopadhayay, David Jones
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>150</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4ae2d5f1f6f184ad5a2b71dcc7e35b4feeb422f" target='_blank'>
              Functional biomaterials and machine learning approaches for phenotyping heterogeneous tumor cells and extracellular vesicles.
              </a>
            </td>
          <td>
            Rutwik Joshi, R. Ahmad, Karl Gardner, Hesaneh Ahmadi, Chau-Chyun Chen, Shannon L Stott, Wei Li
          </td>
          <td>2025-09-18</td>
          <td>Biomaterials science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL)-therapy has received FDA approval for the treatment of advanced melanoma and shows potential for broader applications in solid tumors, including glioblastoma. In this study, tumor-reactive TILs (tr-TILs) are isolated and enriched for CD137 expression from cavitron ultrasonic aspirator (CUSA) emulsions of 161 adult patients diagnosed with diffuse gliomas. Tr-TILs are successfully expanded in 87 out of the 161 patients, reflecting an expansion rate of 54%. Notably, the presence of IDH1 mutation and the cumulative dose of steroids are identified as significant negative predictors of expansion efficacy. The expanded tr-TILs exhibit distinct phenotypic and molecular dysfunctional features yet show upregulated expression of progenitor/memory-like markers and polyclonal T-cell receptors. Importantly, these tr-TILs demonstrate specific antitumor reactivity against autologous tumor cells in both in vitro and in vivo xenograft models. These findings provide a compelling background for a personalized immunotherapeutic approach while tackling one of the most significant challenges in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fd8a91ad8ccb8e7a67febcd47df6b08cd98dc4" target='_blank'>
              Polyclonal expansion of functional tumor-reactive lymphocytes infiltrating glioblastoma for personalized cell therapy
              </a>
            </td>
          <td>
            M. Maffezzini, S. Musio, N. Di Ianni, Agnese Rumolo, M. Patanè, Andrea Galluzzo, Irene Sambruni, Arianna Berlendis, Domenico Aquino, Giacomo Baso, Manuela Zingarelli, M. Facciolla, L. Maddaloni, Rossella Valentino, R. Paterra, F. Agistri, M. Farinotti, Luca Mattei, Paola Coluccia, Francesco Acerbi, Francesco DiMeco, Bianca Pollo, Antonio Silvani, M. Eoli, Catia Traversari, Daniela Montagna, S. Pellegatta
          </td>
          <td>2025-08-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains a devastating malignancy characterized by profound lethality, aggressive local invasion, dismal prognosis, and significant resistance to existing therapies. Two critical biological features underpin the challenges in treating PDAC: extensive perineural invasion (PNI), the process by which cancer cells infiltrate and migrate along nerves, and a profoundly immunosuppressive, or “cold,” tumor microenvironment (TME). PNI is not only a primary route for local tumor dissemination and recurrence but also a major contributor to the severe pain often experienced by patients. Concurrently, the PDAC TME is typified by a dense desmoplastic stroma, hypoxia, and an abundance of immunosuppressive cells—including cancer-associated fibroblasts (CAFs), tumor-associated macrophages (TAMs), myeloid-derived suppressor cells (MDSCs), and regulatory T cells (Tregs)—while lacking sufficient infiltration of effector T cells, rendering it largely unresponsive to immunotherapies like checkpoint inhibitors. Although historically studied as separate entities, accumulating evidence reveals a deep-seated and complex bidirectional crosstalk between the neural components involved in PNI and the immune and stromal cells constituting the TME. Key cellular mediators, such as CAFs and TAMs, and shared signaling pathways, including the CXCL12/CXCR4 axis, TGF-β signaling, and neurotrophin pathways (e.g., NGF/TrkA), appear to act as critical nodes, coordinating the progression of PNI while simultaneously shaping and maintaining the immunosuppressive TME. This review synthesizes the current understanding of these intricate neuro-immune interactions in PDAC. We delineate the molecular and cellular mechanisms governing this crosstalk and explore how targeting these shared regulatory networks presents novel therapeutic opportunities, potentially disrupting PNI while concurrently “heating” the cold TME to overcome immunotherapy resistance. Elucidating this interplay is crucial not only for a deeper comprehension of PDAC’s invasive and metastatic mechanisms but also for uncovering new therapeutic vulnerabilities to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6006a1bf0a36f45add16447e3ef30da17be2aa8b" target='_blank'>
              Perineural invasion and the “cold” tumor microenvironment in pancreatic cancer: mechanisms of crosstalk and therapeutic opportunities
              </a>
            </td>
          <td>
            Jianbiao Xu, Hong Yao, Junfeng Wang, Yun Jin, Wei Chang, Lanjiang Li, Lei Zou
          </td>
          <td>2025-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Background: Extracellular matrix proteins are tightly linked to cancer progression. HCC frequently arises from chronic liver diseases with varying degrees of parenchymal fibrosis. Herein, we aimed to investigate the roles of secreted lumican, an extracellular matrix proteoglycan, in HCC. Methods: Lumican expression in clinical liver tissue samples was analyzed. In vitro and in vivo functional assays were performed with cell lines. Co-culture systems were adopted to examine the roles of lumican in the interaction between HCC cells and liver fibroblasts. Downstream mechanisms were interrogated by transcriptomic and proteomic profiling. Results: Analyses of single-cell RNA-sequencing datasets collectively revealed high lumican expression in liver fibroblasts. Lumican expression was elevated in liver tissues with advanced fibrosis, and a higher lumican level in the non-tumor liver tissue was a poor prognosticator of HCC. Functionally, recombinant human lumican (rhLUM) promoted migration, invasion, and self-renewal of HCC cells, and enhanced angiogenesis in vitro. These effects were abrogated by anti-lumican antibody. The paracrine actions of lumican in the interplay between HCC cells and liver fibroblasts were supported with co-culture models, in which lumican was manipulated by genetic or antibody approaches. In vivo, recombinant lumican promoted neovascularization and tumor incidence. Profiling results revealed the enrichment of Wnt signaling, and mechanistic dissection uncovered the crosstalk between PI3K/AKT and Wnt/β-catenin pathways in rhLUM-treated HCC cells. Conclusions: Secreted lumican promotes HCC self-renewal, tumor initiation, and progression by activating the AKT/GSK3β/β-catenin signaling cascade. Targeting secreted lumican is a potential therapeutic strategy for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30ce1ca512612e4b38fb465cf3b9bae86d579a18" target='_blank'>
              Secreted lumican from the tumor microenvironment potentiates HCC stemness and progression
              </a>
            </td>
          <td>
            Kristy Kwan-Shuen Chan, Cheuk-Yan Wong, K. Au, Long-Hin Suen, Wai-Wai Yip, Jing-Mian Zhang, Eva Yi-Man Fung, T. Lee, Irene Oi-lin Ng, T. Cheung, Regina C-L Lo
          </td>
          <td>2025-08-15</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Glioblastoma, IDH-wildtype, WHO Grade 4 (GB), is the most common primary malignant brain tumor, accounting for approximately 50% of all gliomas. The current standard of care for GB consists of maximal safe surgical resection, with the aim of gross total or supramarginal resection whenever possible. Even when gross total resection (GTR) is achieved, a heterogeneous population of residual tumor cells typically persists along the infiltrative margins. These cells, characterized by distinct genetic, epigenetic, and microenvironmental profiles, serve as a reservoir for treatment-resistant subclones that can drive tumor recurrence. Although some studies have indicated that unresected residual cells in the peritumoral brain zone display distinct alterations compared to cells within the central tumor mass, only limited molecular analyses have been conducted. This study aims to analyze molecular profiles of distinct cell populations in the peritumoral infiltration zone using next-generation sequencing (NGS) to identify molecular features associated with early disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2477cc761f52aa09ce8251efee75bbaa2c524e0" target='_blank'>
              EP12.13 TARGETING THE INFILTRATIVE EDGE OF GLIOBLASTOMA: BIOMARKER INSIGHTS INTO EARLY TUMOR RECURRENCE
              </a>
            </td>
          <td>
            M. Barak, M. Hendrych, J. Brychta, T. Kazda, P. Pospíšil, M. Hermanova, J. Šána, R. Jancalek
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, , Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Notch is a contact-dependent signaling pathway that plays critical roles in spatiotemporal patterning during embryogenesis. Beyond its role in development, Notch signaling regulates tissue patterning in adult organs in response to tissue injury or cancer. In the context of cancer, the role of Notch signaling has been extensively characterized in cancer cells, T cells, and endothelial cells. Our previous work has implicated Notch signaling in the myeloid and fibroblast compartments of the pancreatic tumor microenvironment (TME) in the tumor immune response, an area that has been previously understudied. In our current work we use genetically engineered mouse models to inhibit canonical Notch activation in fibroblasts (Pdgfra CreERT2/+, LSL-DNMAML) and myeloid cells (LysM Cre/+, LSL-DNMAML) separately. In both an orthotopic transplant and genetic model (Ptf1a FlpO/+;FSF-Kras G12D/+) of pancreatic cancer progression, inhibition of fibroblast-specific Notch activation had limited effects on disease progression and the transcriptional profile of the fibroblasts themselves. This is consistent with previous evidence of more limited Notch activation in pancreatic cancer-associated fibroblasts and suggests a primary role for fibroblasts in providing Notch ligand (JAG1) to neighboring cells. In contrast, inhibition of myeloid-specific Notch activation resulted in smaller orthotopic tumors and slowed progression in a genetic model of pancreatic neoplasia. These samples had reduced overall macrophage infiltration, and a secondary alteration of the fibroblast phenotypes was also seen. Collectively, these results suggest that Notch signaling in the myeloid compartment supports tumor progression and regulates multiple components of the pancreatic TME. Future studies will further interrogate how myeloid-specific Notch activation promotes disease progression and affects Notch signaling activation in adjacent compartments.



 Allison Bischoff, Wei Yan, Carlos Espinoza, Emily Lasse-Opsahl, Nur Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar. Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A080.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df0eac24a706ec3b2888b93172ea892748b9176" target='_blank'>
              Abstract A080: Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment
              </a>
            </td>
          <td>
            Allison C Bischoff, Wei Yan, Carlos E Espinoza, Emily L. Lasse-Opsahl, Nur M Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="PURPOSE
With surgery being the only potential cure for pancreatic cancer, high-risk premalignant pancreatic lesions often go unnoticed by palpation or white light visualization, leading to recurrence. We asked whether near-infrared fluorescence imaging of tumor-associated inflammation could identify high-risk premalignant lesions, leveraging the tumor microenvironment (TME) as a sentinel of local disease and, thus, enhance surgery outcomes.


EXPERIMENTAL DESIGN
Fluorescence-guided surgery was performed on genetically engineered mice (Ptf1a-Cre; LSL-KrasG12D/+; Smad4flox/flox [KSC]) at discrete stages of disease progression, histologically confirmed high-risk, premalignant lesions in postnatal mice to locally advanced pancreatic tumors in adults, using the imaging agent V-1520, a translocator protein (TSPO) ligand. Age-matched wild-type littermates were used as controls, while Ptf1a-Cre; LSL-KrasG12D/+(KC) mice modeled pancreatitis and precursors of low penetrance. Localization of V-1520 and tumor-associated macrophages amongst the TME was detected by immunofluorescence imaging.


RESULTS
V-1520 exhibited robust accumulation in the pancreata of KSC mice from the early postnatal stage. Increased accumulation was observed in the pancreata of adolescent- and adult-aged mice with greater ductal lesion and stromal burden. Confocal microscopy of ex vivo pancreas specimens co-localized V-1520 accumulation primarily with CD68-expressing macrophage in KSC mice. Unlike the pancreata of KSC mice, accumulation of V-1520 did not exceed background levels in the pancreata of KC mice with pancreatitis.


CONCLUSION
V-1520 exhibited differential accumulation in pancreatic cancer-associated inflammation compared to pancreatitis. Given the robust tracer uptake in tissues associated with early yet high-risk lesions, we envision V-1520 could enhance surgical resection and reduce the potential for recurrence from residual disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e1713383f0cb9b8f04471c53342e9b58b0edaa9" target='_blank'>
              Preclinical Fluorescence-Guided Imaging Leveraging Surrounding Sentinel Tumor Microenvironment Identifies High-Risk Premalignant Pancreatic Lesions.
              </a>
            </td>
          <td>
            Shilpa Sharma, Xiaoxia Wen, Jianbo Wang, Beibei Huang, Denise Hernandez, Cong-Dat Pham, Zhiwen Liu, S. J. Lin, Aiko Yamaguchi, Dimitra K. Georgiou, R. P. Coll, H. C. Manning
          </td>
          <td>2025-08-25</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Pediatric high-grade gliomas (pHGGs) are diffusely invasive and rapidly growing brain malignancies and are among the leading causes of cancer-related deaths in children. However, current treatments for pHGGs are largely ineffective. Thus, alternative therapeutic approaches are urgently needed. Genetic mutations observed in these gliomas include H3 oncohistone mutations and alterations in receptor tyrosine kinases (RTKs), including amplification, mutation, and gene fusions. However, directly targeting RTK variants with tyrosine kinase inhibitors (TKIs) has shown only partial responses in patients, due in part to the molecular and cellular heterogeneity within these tumors. Therefore, we sought to develop new experimental model systems that recapitulate inter- and intra-tumor heterogeneity to enable therapeutic discovery.



 To explore new therapeutic approaches, we created human brain organoid-tumor models of pHGG, in which patient-derived tumor cells are engrafted into ex vivo human cortical brain organoids that mimic the neonatal human brain. These organoid models provide an improved system for examining how the neural CNS microenvironment shapes cellular and molecular heterogeneity among tumor cells over time. We tested these models for responses to small molecule inhibitors and targeted therapies, including approaches that target a range of developmental and oncogenic effector pathways downstream of RTKs. Candidate targets included the YAP and TAZ transcription factors, which cooperate with TEAD co-factors to promote target gene expression, orchestrate stem cell fate decisions, and drive proliferation during embryonic neural development. We examined responses in both tumor and non-tumor cells in the neural microenvironment following YAP/TAZ pharmacologic and genetic inhibition, using single-cell RNA sequencing and immunofluorescence to visualize cellular responses.



 We observed that YAP-TEAD function is required for the growth and progression of pHGGs with oncogenic RTK mutations and H3 oncohistone mutations. Depletion of YAP reduced proliferation, inhibited pHGG stem cell self-renewal, and induced apoptosis in pHGG cells. Treatment with small molecule inhibitors targeting YAP/TAZ and TEAD mimicked the effects of genetic YAP inhibition. Further preclinical work in primary tumor cell cultures and mouse genetic models of pHGG correlated with our findings from organoid models, supporting the potential therapeutic relevance of YAP/TAZ-TEAD inhibition for pHGG treatment.



 Our results show that YAP/TAZ-TEAD inhibition may represent a promising therapeutic avenue for pHGGs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca2fbad46fe24df52bf242a29e83b7ce9c58fbe" target='_blank'>
              P01.31.A THE YAP/TAZ-TEAD AXIS PROMOTES TUMOR STEM CELL FUNCTIONALITY AND CELLULAR HETEROGENEITY IN PEDIATRIC HIGH-GRADE GLIOMAS
              </a>
            </td>
          <td>
            R. Read, J. Luis Jimenez Macias, S. Oh, Z. Hu, K. Yin, N. Sele, N. Boon, A. Sing, S. Sloan
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cabe0cbcb8c826f9af8109abd06e0ae3e7cf0a6a" target='_blank'>
              Patient-derived organoids reveal hypoxia-driven plasticity and therapeutic vulnerabilities in pheochromocytomas and paragangliomas.
              </a>
            </td>
          <td>
            Xiaojing Wang, Maite Calucho, Cynthia M Estrada-Zuniga, Mehwish Noureen, Hector Gonzalez-Cantú, Samantha Mossuto, Jonathan N Levi, Bethany Landry, Qianjin Guo, Kailee A Rutherford, Zi-Ming Cheng, Andreanne Sannajust, Sapna Raghunathan, Sanskriti Balaji, Viviane Nascimento da Conceicao, Summer Norris, S. Y. Munawar, H. Nguyen, Jacob De Rosa, G. Tomlinson, James X Wu, Michael W. Yeh, M. Livhits, Anand Vaidya, James Powers, Ron Lechan, Art Tischler, Miao Zhang, Paul Graham, Camilo Jimenez, M. Peitzsch, Nicole Bechmann, Graeme Eisenhofer, Yanli Ding, Mio Kitano, A. Soragni, P. Dahia
          </td>
          <td>2025-08-23</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma is one of the deadliest solid tumors, with a two-year survival rate below 5%. Progress in developing effective treatments has been limited, largely due to the absence of physiologically relevant disease models that accurately recapitulate the tumor microenvironment. Such models are critical for understanding disease mechanisms and evaluating therapeutic strategies. Organoids have recently emerged as promising platforms for modeling various diseases, including pancreatic cancer, as they retain the structural and functional features of their tissue of origin. Moreover, CRISPR-Cas9 gene editing allows precise introduction of cancer-associated mutations, providing a powerful approach for generating genetically defined models.



 We established normal pancreatic organoids from human pancreatic tissue samples and validated their identity through qPCR and histological analyses, confirming their resemblance to native pancreatic epithelium. These organoids demonstrated stable growth over several passages and retained viability after cryopreservation and thawing. To model pancreatic cancer, we used CRISPR-Cas9 technology to introduce TP53 mutations, a key tumor suppressor gene altered in over 70% of pancreatic ductal adenocarcinoma (PDAC) cases. Following Nutlin-3a selection, successful TP53 knockout was confirmed by Sanger sequencing. The engineered TP53-knockout (KO) organoids were expanded and characterized to assess cancer-like phenotypes.



 Human-derived pancreatic organoids retained their structural and functional identity across passages and after thawing. Following TP53 gene editing, the TP53 KO organoids exhibited increased proliferation and morphological changes consistent with early tumorigenesis. Importantly, qPCR analysis revealed significant upregulation of cancer-associated markers, including MUC5AC, KRT19, and CEACAM5, in TP53-mutated organoids compared to controls, further supporting their malignant transformation. These findings represent a critical step toward building a genetically defined pancreatic cancer model that closely reflects human disease.



 We successfully generated a pancreatic cancer organoid model by editing TP53 in normal human pancreatic organoids. The TP53 KO organoids showed enhanced growth, morphological transformation, and elevated expression of key cancer markers, recapitulating early features of pancreatic cancer. This model mimics critical aspects of pancreatic cancer biology and provides a robust platform for studying tumor initiation, progression, and drug response. Unlike conventional animal models or stem-cell-derived systems, our approach offers a human-based, physiologically relevant, and reproducible system for pancreatic cancer research and therapeutic screening.



 Eden Demere Amare, Soraya SALAS-SILVA, Ji Hyun Shin, Dongho Choi. CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59cdfa7b3d8511b93672f4beff88185a825b61c7" target='_blank'>
              Abstract A013: CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation
              </a>
            </td>
          <td>
            Eden Demere Amare, Soraya SALAS-SILVA, Jihyun Shin, Dongho Choi
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) display remarkable functional heterogeneity, yet the molecular mechanisms driving their diverse phenotypes remain elusive. Using CRISPR screens in primary macrophages, we identified tumor-derived factors, including lactic acid, PGE2, and GM-CSF, as key modulators of TAM polarization. These factors interact synergistically and antagonistically to shape distinct TAM phenotypes that are highly conserved across human cancers. Mechanistically, lactic acid and PGE2 jointly induce angiogenic gene programs while suppressing GM-CSF-driven MHC-II expression at the chromatin level, creating mutually exclusive distributions of proangiogenic and MHC-II+ TAMs, which are differentially localized to specific spatial niches in the tumor microenvironment. Furthermore, we showed that shifting TAMs to an interferon-responsive phenotype, triggered by Adar inactivation, significantly promotes the infiltration of effector CD8+ T cells through specific receptor-ligand interactions. These findings uncover a conserved mechanism of TAM polarization and offer insights into therapeutic strategies for TAM reprogramming to potentiate cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cb6bdb7d04c92080704c1d38a2b724c54141d29" target='_blank'>
              Functional genetic screens reveal key pathways instructing the molecular phenotypes of tumor-associated macrophages.
              </a>
            </td>
          <td>
            Youxue Lu, Ce Luo, Lanxiang Huang, Genyi Wu, Lihan Zhong, Jieyu Chu, Fubing Wang, Zexian Zeng, Deng Pan
          </td>
          <td>2025-09-04</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (iCCA) is a highly aggressive and heterogeneous malignancy characterized by marked resistance to standard chemotherapy and poor prognosis. While the advent of immunotherapy has revolutionized the management of several solid tumors, including melanoma, breast cancer, and non-small cell lung cancer, its efficacy in iCCA remains limited. Recent clinical trials have demonstrated the efficacy of durvalumab in combination with chemotherapy for iCCA, leading to its approval as a first-line treatment. However, overall response rates remain low, largely due to its immunosuppressive tumor immune microenvironment (TIME). The immune-cold nature of iCCA is typified by a dominant presence of immunosuppressive cell populations, including M2-polarized tumor-associated macrophages, myeloid-derived suppressor cells, and T regulatory cells. In addition, traditional biomarkers such as PD-L1 expression, tumor mutational burden, and microsatellite instability have shown limited predictive value in iCCA, highlighting the need for novel biomarkers and immunotherapeutic strategies. Emerging approaches aimed at reprogramming the TIME, including combination therapies targeting suppressive cells, stromal remodeling, and novel immune effectors like CAR-T and cancer vaccines, hold significant promise for enhancing therapeutic efficacy. This review summarizes the distinct features of iCCA TIME, key mechanisms of immune evasion, current challenges, and future directions to overcome immune resistance, with the aim of developing personalized immunotherapies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1167c3442bd04894cf43de06826ab2ab3325aec4" target='_blank'>
              Immune Landscape of Intrahepatic Cholangiocarcinoma: Evasion and Therapeutic Insights
              </a>
            </td>
          <td>
            Nunzia Porro, Elena Spínola-Lasso, F. Marra, A. Gentilini
          </td>
          <td>2025-09-17</td>
          <td>Immuno</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 The advent of RAS inhibitors in the clinic offers the possibility of revolutionized care for patients with pancreatic ductal adenocarcinoma (PDAC). While RAS(ON) multi-selective inhibitors like the investigational agent daraxonrasib (RMC-6236) and the related preclinical tool compound RMC-7977 are designed to target the KRAS-mutant malignant epithelium, their effects on the surrounding tumor stroma remain underexplored. Prior studies using genetically engineered mouse models (GEMMs) have shown that aberrant oncogenic KRAS signaling is critical for the establishment and maintenance of the tumor microenvironment. In recent years, multiple CAF subtypes have been identified based on gene expression profiles, spatial distribution, and cellular origin. Myofibroblastic CAFs (myCAFs), which are located near the malignant epithelium, are marked by active Sonic Hedgehog (SHH) signaling, while inflammatory CAFs (iCAFs) and antigen-presenting CAFs (apCAFs) reside more peripherally. Here, we investigate how pan-RAS-GTP inhibition by RMC-7977 reshapes the CAF landscape in PDAC and its impact on the extracellular matrix (ECM). We profiled CAF subtype prevalence and transcriptional programs using single-cell RNA sequencing (scRNA-seq) of GEMM-derived tumors treated in vivo with RMC-7977 across multiple timepoints. These findings were corroborated by immunostaining of murine and patient-derived tumor explants treated ex vivo for 2 to 4 days. Tumor stiffness following RMC-7977 treatment was assessed using nanoindentation, and mechanistic studies were performed in vitro using reporter fibroblasts. Preliminary data reveal that short-term RMC-7977 treatment reduces myCAF abundance and proliferation, while iCAF and apCAF populations expand. Notably, supplementation with exogenous SHH restores the original myCAF-rich phenotype even in the presence of RMC-7977. Pan-RAS-GTP inhibition also remodels the ECM, resulting in decreased tumor stiffness. These preclinical findings suggest that systemic pan-RAS-GTP inhibition alters not only the malignant epithelium but also the broader PDAC architecture through effects on stromal components. Understanding these stromal adaptations will enable improved assessment of tumor behavior and the development of more effective combination therapies.



 Allison C. Hess, Lorenzo Tomassoni, Kevin Muñoz Forti, Hun Jin Jeong, Deanna J. Besart, Tanner C. Dalton, Carmine F. Palermo, Stephen A. Sastra, Chang H. Lee, Andrea Califano, Simon Schwörer, Kenneth P. Olive, Marie C. Hasselluhn. RAS(ON) multi-selective inhibition remodels cancer-associated fibroblast subtypes and extracellular matrix in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B098.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80ccb92666321bba79e7b5a3b723ec7cf1794a5" target='_blank'>
              Abstract B098: RAS(ON) multi-selective inhibition remodels cancer-associated fibroblast subtypes and extracellular matrix in pancreatic cancer
              </a>
            </td>
          <td>
            Allison C. Hess, Lorenzo Tomassoni, Kevin Muñoz Forti, Hun Jin Jeong, Deanna J. Besart, Tanner C. Dalton, Carmine F. Palermo, Stephen A. Sastra, Chang H. Lee, Andrea Califano, Simon Schwörer, K. Olive, Marie C. Hasselluhn
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Gastric-type adenocarcinoma (GAS) of the uterine cervix is a rare and aggressive subtype of cervical adenocarcinoma characterized by intrinsic resistance to chemotherapy and poor clinical outcomes due to the lack of effective treatment options. To address this critical unmet need, we established a novel GAS-derived cell line, KGAS, from ascitic fluid collected from a patient with recurrent GAS. Short tandem repeat (STR) analysis confirmed the genetic identity between the primary tumor and the cell line. Upon transplantation into immunocompromised mice, KGAS cells formed tumors that expressed Claudin-18 and MUC6, clinically recognized markers of GAS. Furthermore, KGAS cells exhibited marked resistance to paclitaxel and carboplatin, showing significantly reduced growth inhibition compared to HeLa cells. We also established a paclitaxel- and carboplatin-resistant subline, rKGAS, and performed microRNA (miRNA) sequencing to explore the molecular basis of acquired chemoresistance. Seventeen differentially expressed miRNAs were identified between KGAS and rKGAS cells. Upregulated miRNAs in rKGAS were predicted to target oncogenes such as BCL2, MET, SIRT1, and VEGFA, whereas downregulated miRNAs were associated with tumor suppressor genes, including IGF1R, TNFAIP3, and MTOR. The KGAS and rKGAS cell lines represent valuable preclinical models for elucidating the molecular mechanisms of chemoresistance and malignant progression in cervical GAS, and may contribute to the development of novel therapeutic strategies for this challenging cancer subtype. Supplementary Information The online version contains supplementary material available at 10.1007/s13577-025-01286-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e177230889c8d8ae625607ead94690aa8c28b9" target='_blank'>
              Establishment of KGAS, a cell line derived from gastric-type adenocarcinoma of the uterine cervix
              </a>
            </td>
          <td>
            Hiroaki Yamada, Akira Yokoi, Eri Asano-Inami, Masami Kitagawa, Kosuke Yoshida, Kazuhiro Suzuki, Shin Nishio, Hiroaki Kajiyama, Naotake Tsuda
          </td>
          <td>2025-09-15</td>
          <td>Human Cell</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/034c8133be4ba672c54b97e9a66fb7c40de89b73" target='_blank'>
              Single-cell transcriptomic analysis reveals a metastasis-associated PCSK1+ neuroendocrine subpopulation in pancreatic neuroendocrine tumors.
              </a>
            </td>
          <td>
            X. Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-08-16</td>
          <td>Journal of neuroendocrinology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Brain metastases are a significant complication of non-small-cell lung cancer (NSCLC), contributing to high morbidity and mortality rates. The introduction of immune checkpoint inhibitors (ICIs) has opened new therapeutic avenues for patients with NSCLC, including those with brain metastases. However, the distinct microenvironment of the brain presents unique challenges to the effectiveness of these treatments. This review examines the mechanisms by which ICIs impact brain metastases from NSCLC, with particular focus on immune cell trafficking across the blood–brain barrier (BBB), tumor microenvironment modulation, and transcriptomic evolution of brain-tropic tumor clones. Unlike prior reviews, we integrate emerging data from single-cell and spatial transcriptomic studies, BBB disruption mechanisms, and the tumor-supportive role of brain-resident glia. We also critically evaluate key clinical trials and real-world evidence, highlighting differences in ICI efficacy across patient subgroups and therapeutic contexts. Additionally, we address the evolving role of surgical resection, stereotactic radiosurgery, and cerebrospinal-fluid-based biomarkers in optimizing ICI-based treatment strategies. This synthesis provides a comprehensive, mechanistic, and clinically relevant framework for improving outcomes in patients with NSCLC brain metastases treated with immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c6ed65bc2f6ca5f07b82a4924c158eebc0c9a9" target='_blank'>
              Immune Checkpoint Inhibition in Patients with Brain Metastases from Non-Small-Cell Lung Cancer: Emerging Mechanisms and Personalized Clinical Strategies
              </a>
            </td>
          <td>
            Nicola J. Nasser, K. Sindhu, Loor Nasser, Z. Shafaee, Joshua Li, Lucas Resende Salgado, Baoqing Li
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background We previously defined a Genomic Dedifferentiation Signature (GDS) derived from differential gene expression analysis of sarcomatoid renal cell carcinoma (sRCC) tumor clones compared to clear cell RCC (ccRCC; Salgia et al. KCRS. 2024). GDS expression is negatively prognostic across clinical RCC cohorts, yet carries predictive capacity for identifying patients likely to achieve benefit from immune checkpoint blockade (ICB). Furthermore, we have also demonstrated that sRCCs are enriched for expression of tertiary lymphoid structure (TLS) signatures. Crucially, TLS signatures have been associated with response to ICB in ccRCC (Jammihal et al. Nature Cancer. 2025). Therefore, we sought to investigate if an interplay between expression of the GDS—a tumor-intrinsic gene signature—and TLS signatures—characterizing the inflammatory immune microenvironment—could best identify ICB sensitivity in patients with metastatic RCC. Methods Transcript-per-million normalized gene expression counts matrices and clinical metadata were collected from three trials of ICB regimens in patients with metastatic RCC: IMmotion 151 (Motzer et al. Cancer Cell. 2020), Javelin101 (Motzer et al. Nature Medicine. 2020), and HCRN-GU16-260 (Hugaboom et al. Cancer Discovery. 2025). Gene expression scores for the GDS and the Imprint TLS signature (Meylan et al. Immunity. 2022) were assigned to cases using single sample gene set enrichment analysis. Correlation between GDS and TLS scores was performed via Spearman’s rank-order correlation. Cases were stratified into -high and -low signature expression (TLShigh-GDShigh, TLShigh-GDSlow, TLSlow-GDShigh, and TLSlow-GDSlow) based on median signature expression within trial cohorts. Progression-free survival (PFS) was compared between groups and trial arms within each cohort using Cox proportional hazards models, while objective response rates (ORR) were compared using Fisher’s exact tests. Results The correlation between GDS & Imprint TLS expression was moderate across IMmotion151 (n = 822; R = 0.44; p < 2.2x10-16), Javelin101 (n = 723; R = 0.43; p < 2.2x10-16) and HCRN-GU16-260 (n = 72; R = 0.53; p = 2.6x10-6). Within IMmotion151, patients classified as TLShigh-GDShigh experienced prolonged PFS with the ICB-containing atezolizumab/bevacizumab regimen (median PFS: 10.9 months) compared to sunitinib (median PFS: 6.9 months; Hazard Ratio [HR]=0.68; P = .009; Figure 1), whereas no significant differences between atezolizumab/bevacizumab and sunitinib PFS were observed in TLShigh-GDSlow (HR = 1.18; P = .5), TLSlow-GDShigh (HR = 0.71; P = .09), or TLSlow-GDSlow (HR = 0.93; P = .6) subgroups. ORR to atezolizumab/bevacizumab within the TLShigh-GDShigh population (47.5%) was also significantly higher compared to atezolizumab/bevacizumab ORR within the TLSlow-GDSlow population (30.6%; P = .008) and compared to ORR with sunitinib in the TLShigh-GDShigh population (31.0%; P = .009). Conversely, ORR with sunitinib (43.6%) was significantly greater than with atezolizumab/bevacizumab (30.6%; P = .004) in the GDSlow-TLSlow population. Consistent results were observed in Javelin101: the TLShigh-GDShigh population experienced prolonged PFS with the ICB-containing axitinib/avelumab regimen (median PFS: 12.5 months) compared to sunitinib (median PFS: 6.9 months; HR = 0.47; P < .001). Again, no significant differences between axitinib/avelumab and sunitinib PFS were observed in TLShigh-GDSlow (HR = 1.04; P = .9), TLSlow-GDShigh (HR = 0.81; P = .4), or TLSlow-GDSlow (HR = 0.81; P = .3) subgroups. In the HCRN-GU16-260 trial of front-line nivolumab in metastatic RCC, TLShigh-GDShigh status was associated with prolonged PFS, albeit non-significantly, compared to TLShigh-GDSlow (HR = 2.02; P = .10), TLSlow-GDShigh (HR = 1.99; P = .13) and TLSlow-GDSlow (HR = 1.72; P = .09). Additionally, ORR to nivolumab was significantly enriched in the TLShigh-GDShigh cohort (76.9%) compared to those classified as TLShigh-GDSlow (23.1%; P = .0009), TLSlow-GDShigh (25.0%; P = .01), and TLSlow-GDSlow (18.5%; P < .0001) within HCRN-GU16-260. Conclusions Across metastatic RCC patient cohorts, simultaneous enrichment of tumor-intrinsic and TLS gene expression best predict response to ICB versus anti-angiogenic therapy. This suggests that a pro-inflammatory milieu is not sufficient for effective immune sensitivity, but that a biologic interplay between tumor cells and their surrounding microenvironment is necessary for effective immune surveillance and ICB responsiveness in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b4535064bf0393b2610743ad10a87355b938d65" target='_blank'>
              32Interplay of Tumor-Intrinsic and microenvironmental gene expression predicts immune checkpoint blockade response in metastatic renal cell carcinoma
              </a>
            </td>
          <td>
            Nicholas J. Salgia, Eric C. Kauffman, J. Muhitch
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb5a38dd74bce74d21cd64c9a442cf5e375b46f4" target='_blank'>
              Targeting Glioblastoma Cell State Plasticity for Enhanced Therapeutic Efficacy.
              </a>
            </td>
          <td>
            S. Cirigliano, R. Singhania, James G Nicholson, Isha Monga, Yushan Wan, C. Haywood, Ashlesha Muley, Skylar A. Giacobetti, Howard A. Fine
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary People with systemic sclerosis (a chronic disease that causes skin and organ scarring) have a higher risk of developing certain cancers soon after diagnosis, but the reasons remain unclear. To study systemic sclerosis in the laboratory, researchers are using tight skin (TSK) mice, which develop excess scar tissue and immune changes similar to systemic sclerosis. We combined the TSK mouse model with mouse models of skin, breast, and ovarian cancers to test whether the unique environment in TSK mice promotes cancer progression. In all cancer types, TSK mice developed more aggressive tumors. Detailed single-cell analyses showed that TSK tumors had more immune imbalance and more scar-promoting cell types. These findings suggest that the TSK mouse is a powerful tool for uncovering how systemic sclerosis-related tissue changes and immune shifts may fuel cancer growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fece107070df629a4a56795a06c184ea38a1b613" target='_blank'>
              Cancer Growth and Invasion Are Increased in the Tight Skin (TSK) Mouse
              </a>
            </td>
          <td>
            M. Recouvreux, Barbie Taylor-Harding, Amy C. Rowat, Beth Y Karlan, Sandra Orsulic
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Pancreatic β-cells contain insulin secretory granules (ISGs), organelles where proinsulin is converted into insulin. As ISGs mature, they undergo extensive biophysical remodeling, producing a spectrum of subpopulations with heterogeneous molecular and spatial characteristics. However, systematic methods to define ISG subpopulations remain underdeveloped. To address this gap in knowledge, we employed soft x-ray tomography (SXT), which can quantitatively measure the biochemical density of ISGs within whole β-cells. Using unsupervised clustering, we classified subpopulations based on molecular density, size, and spatial positioning. Across different insulin secretory stimuli, we observed shifts towards mature and releasable subtypes, demonstrating that exogenous signals can dynamically remodel ISG subpopulation distributions. We extended this methodology to primary β-cells characterized using volume electron microscopy (vEM). Integrating subpopulations from SXT and vEM uncovered insights inaccessible by a single method in isolation. This strategy establishes a framework for defining therapeutic approaches aimed at enriching physiologically beneficial ISG subpopulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5df3a912eecd0bc834b9b470f634387d21062df4" target='_blank'>
              Classifying Biophysical Subpopulations of Insulin Secretory Granules using Quantitative Whole Cell Structure Analysis
              </a>
            </td>
          <td>
            Kevin Chang, Aneesh Deshmukh, Riva Verma, Valentina Loconte, Kate L. White
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da9274332e90da211d1dc7074d7c247b79d5167" target='_blank'>
              The Pdgfd-Pdgfrb axis orchestrates tumor-nerve crosstalk in pancreatic cancer.
              </a>
            </td>
          <td>
            Peter L. Wang, N. Lester, Ella Perrault, J. Su, Dennis Gong, C. Shiau, Jingyi Cao, Phuong T T Nguyen, Jung Woo Bae, Deniz G Olgun, H. Hoffman, Ashley Lam, Jean Huang-Gao, Saifur Rahaman, Jimmy A. Guo, J. Barth, Nicholas J Caldwell, P. Divakar, Jason Reeves, Arya Bahrami, Shanshan He, Michael Patrick, Eric M. Miller, M. Ganci, Grissel Cervantes-Jaramillo, Theodore Hong, Jennifer Wo, Hannah Roberts, Ralph Weissleder, Hongyoon Choi, C. Fernández-del-Castillo, Kathy Cormier, David T Ting, Tyler Jacks, Lei Zheng, M. Hemberg, Mari Mino-Kenudson, William L Hwang
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumor–immune interactions are central to cancer progression and treatment response, driving cell death through immune-mediated killing and resource-limited competition. In early-stage disease or following effective treatment, cancer populations are often small and difficult to observe directly. Disease monitoring therefore relies on the detection of biomarkers such as circulating tumor DNA (ctDNA) as noisy proxies to cancer size. However, existing approaches lack robust frameworks to infer tumor burden from these signals when populations approach detection thresholds. To address this, we present a coupled deterministic–stochastic framework that links tumor–immune dynamics to biomarker release. A two-prey, one-predator Lotka–Volterra model captures interactions between immune cells and competing tumor subpopulation under shared resource constraints. Biomarker production is modeled using a linear stochastic differential equation, incorporating ctDNA release from both apoptotic death (immune-mediated) and necrotic death (due to intra-tumor competition). We derive analytical solutions for the resulting biomarker trajectories, including mean detection time under a minimal detection threshold. These results show how volatility in measured biomarker signal, disease heterogeneity, and immune pressure jointly shape signal emergence and persistence. Finally, to model situations in which the observer has access to future information—such as the terminal biomarker signal or sampling time in retrospective studies—we adopt an anticipative theoretic perspective. Using anticipative stochastic calculus, we derive path solutions to the resulting anticipating stochastic differential equation, capturing how future observations influence the inferred biomarker dynamics. This approach links the dynamics of underlying tumor–immune interactions to the corresponding detectable biomarker levels, with implications for early detection, immune monitoring, and retrospective reconstruction of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2813d28434da5ea45b29f5ad951e2cc1a36e6259" target='_blank'>
              Coupled SDE-ODE Modeling of Tumor-Immune Dynamics to Infer Biomarker Release
              </a>
            </td>
          <td>
            Pujan Shrestha, Yijia Fan, J. T. George
          </td>
          <td>2025-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal malignancy characterized by a profoundly immunosuppressive tumor microenvironment. Within PDAC, tumor-associated macrophages (TAMs) are the most abundant immune cell population. Here, TAMs orchestrate the characteristic immunosuppression of the pancreatic tumor microenvironment, drive disease progression, and mediate therapeutic resistance in PDAC. Accordingly, successful development of approaches that reprogram or eliminate these pro-tumor immune populations have potential to improve patient outcomes. One of the most challenging aspects of developing approaches to modulate TAMs programming is the ability to perform mechanistic experiments in a system the recapitulates the complex pancreatic tumor microenvironment. We and others have leveraged primary bone marrow-derived macrophages (BMDMs) polarized by PDAC conditioned media as a discovery platform that has yielded translational findings. However, this system fails to account for the dynamic and reciprocal interactions between myeloid and cancer cells that occur within tumors. Accordingly, we sought to address if this model could be built into progressively more complex systems to assess if these better recapitulate TAMs using a systems biology approach. Using transcriptomic profiling, we compared different in vitro strategies, from direct co-cultures to conditioned media polarization, against TAMs harvested from orthotopic PDAC tumors. Our preliminary data show that allowing PDAC cells to engage in direct cellular contact and sustained paracrine signaling most closely resemble the transcriptomic profile of bona fide TAMs. Our further efforts to use these PDAC-macrophage culture models will be key to identifying relevant signaling pathways that can be mechanistically examined prior to more cumbersome in vivo validation. Collectively, our data can be used to facilitate more accurate and insightful investigations into PDAC immunology and the development of therapeutic targeting strategies.



 Cavina K. Lee, Cecily Anaraki, Austin D. Silva, Tiffany Lai, Hillary Tran, Arjun Sapru. Dissecting myeloid - cancer communication networks in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B104.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad71226813c7f5d3d378a55b84523f40aab08ce1" target='_blank'>
              Abstract B104: Dissecting myeloid - cancer communication networks in pancreatic cancer
              </a>
            </td>
          <td>
            Cavina K. Lee, Cecily Anaraki, Austin D. Silva, Tiffany Lai, Hillary Tran, Arjun Sapru
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In most solid tumors, hypoxia constitutes a defining microenvironmental feature that reprograms malignant cells into a highly metastatic state by driving cellular plasticity and exacerbating chromosomal instability (CIN). However, the mechanisms by which cancer cells concurrently co-opt these elements of hypoxic adaptation to promote metastasis remains poorly understood. Here, we report that hypoxia promotes metastasis by suppressing the JmjC-containing histone lysine demethylase Kdm8. CRISPR/Cas9-mediated targeting of Kdm8 in a Kras;Trp53-driven mouse model of pancreatic ductal adenocarcinoma (PDA) robustly rewires the malignant cell transcriptomic programs, leading to a profound loss of the epithelial morphology and widespread metastatic disease. In PDA patients, a high KDM8-induced gene signature is associated with reduced metastatic burden and better survival in advanced disease. Notably, Kdm8 suppression in normoxia recapitulates key aspects of the global epigenetic and transcriptomic rewiring, mitotic spindle defects, and CIN induced by hypoxia. Moreover, disruption of Kdm8’s demethylase activity phenocopies Kdm8 loss, whereas expression of hypermorphic Kdm8 variants resistant to hypoxic suppression markedly reduces metastasis beyond the levels achieved by the wildtype protein. Through the suppression of Kdm8 demethylase function, hypoxia unleashes a potent metastatic program by simultaneously advancing cellular plasticity and CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ead3df1c0ef44aa86f0839ae810b8a1fead329" target='_blank'>
              Hypoxia-induced metastatic heterogeneity in pancreatic cancer
              </a>
            </td>
          <td>
            Pradeep Moon Gunasekaran, Qianqian Wang, Yoke-Chen Chang, Polina Guseva, Rajika Chauhan, Alexander Kley, Gene Lee, Siddharth Ghosh Roy, Yousef Masoudpoor, Arthur Roberts, Kelly Watkins Walton, Lucyann Franciosa, Shafiq Bhat, Emmanuel Zachariah, Kishan Patel, Zhongren Zhou, Wenjin Chen, J. Ni, S. Gu, Cristina Montagna, Shin-Heng Chiou
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumor-immune interactions are shaped by both antigenic heterogeneity and stochastic perturbations in the tumor microenvironment, yet the mathematical mechanisms underlying immune phase transitions remain poorly understood. We propose a four-compartment dynamical model that incorporates antigen accumulation and immune escape mutations. Bifurcation analysis reveals bistability between immune surveillance and immune escape states, providing a mechanistic explanation for heterogeneous immune outcomes during tumor progression. In the multistable regime, the stable manifold of a saddle point partitions the state space into distinct basins of attraction, determining the long-term fate of the system. We further analyze how stochastic fluctuations in the tumor microenvironment perturb these separatrices, potentially triggering irreversible state transitions. By characterizing the critical noise intensity and estimating the tipping time, we establish a mathematical framework for assessing noise-induced transitions. The model further predicts that increasing tumor cell death can improve system resilience to stochastic perturbations, whereas stronger immune pressure may facilitate immune escape-highlighting the nonlinear and non-monotonic nature of tumor-immune dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/358f26ec76382702bbe9f0bdc46b3b599d87dd6f" target='_blank'>
              Bistability and Noise-Induced Evasion in Tumor-Immune Dynamics with Antigen Accumulation and Immune Escape
              </a>
            </td>
          <td>
            Mengfan Tan, Shaoqing Chen, Chunjin Wei, Da Zhou
          </td>
          <td>2025-09-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Glioblastoma remains a formidable challenge due to its complex intra- and intertumoral heterogeneities and immunosuppressive microenvironment, which have contributed to the limited progress in treatment over the past decades. A comprehensive approach, involving the strategic combination of therapies, is essential to effectively address the challenges associated with glioma treatment. L19-TNF, a tumor-targeted cytokine-based therapy that triggers anti-tumor immunity, is currently being tested in clinical trials for newly diagnosed and recurrent glioblastoma patients. To enhance its efficacy, we sought to identify synergistic or additive drug targets using unbiased in vivo screens in immunocompetent glioma mouse models.



 To achieve this, we employed a novel genome-wide in vivo CRISPR screening approach to uncover druggable vulnerabilities in glioma under the selective pressure of L19-TNF immunotherapy in the CT-2A orthotopic murine glioblastoma model. This platform overcomes challenges, such as the bottleneck during cell engraftment, by enabling timed sgRNA activation and incorporating an in vivo tumor-internal reference library. This strategy mitigates off-target effects and enhances the precision of genetic perturbations. Subsequently, we refined our findings with a second, targeted in vivo CRISPR screen to validate the most promising therapeutic candidates.



 Using this approach, we discovered previously unrecognized vulnerabilities of glioblastoma that converge to ferroptosis. To validate its therapeutic potential, we tested small molecules triggering ferroptosis in combination with L19-TNF immunotherapy in preclinical glioblastoma mouse models. This dual therapy significantly prolonged survival in mice. We also confirmed target expression in glioblastoma patient samples by RNA sequencing, supporting its clinical relevance. Additionally, we validated the hits using orthogonal ex vivo drug-screens in freshly obtained patient-derived glioblastoma tissue samples.



 Our research marks a considerable advancement in systematically uncovering gene candidates using in vivo CRISPR screening. These findings lay the foundation for the rational design of synergistic approaches, offering exciting prospects for more effective combination therapies for glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f0235edf09717257e0b5183dd6de6e8b2dbaa92" target='_blank'>
              KS01.7.A IN VIVO CRISPR SCREENING UNCOVERS VULNERABILITIES FOR CANCER IMMUNOTHERAPY AGAINST GLIOBLASTOMA
              </a>
            </td>
          <td>
            I. Sakic, E. Uijttewaal, A. Buck, B. Snijder, U. Elling, M. Weller, T. Weiss
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb8107340798d9682d8c6977b5d27a13471e822" target='_blank'>
              Integrated single-cell and spatial transcriptomics uncover the prognostic, epigenetic, and immunological roles of FANCC in low-grade glioma.
              </a>
            </td>
          <td>
            Zongye Zhang, Zhi Sha, Han Liu, Yusheng Chen, Zhendong Liu, Xingbo Cheng, Sujie Gu, Yanzheng Gao
          </td>
          <td>2025-09-09</td>
          <td>Neurological research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="In prostate cancer, it is recognized that adenocarcinoma can transdifferentiate into neuroendocrine prostate cancer (NEPC) owing to lineage plasticity; however, transdifferentiation into other histological types remains uncertain. We present a case of a patient who underwent surgery for adenocarcinoma, which later recurred as prostate carcinosarcoma. Genomic analysis revealed a TMPRSS2-ERG fusion, confirming a common clonal origin and transdifferentiation from adenocarcinoma to carcinosarcoma. Additionally, we identified a frameshift mutation in TP53 and the loss of PTEN and RB1. Transcriptome analysis revealed enriched epithelial-mesenchymal transition and immune-related pathways, a pattern distinct from both adenocarcinoma and NEPC. To our knowledge, this is the first report that comprehensively evaluated the clonal origin of the rare prostate carcinosarcoma and characterized it using genomic and transcriptomic sequencing. It enhances our understanding of prostate cancer lineage plasticity and highlights the importance of developing novel therapies specifically targeted at prostate carcinosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc3dc165d184c74305476d6dbd5e449b98486ca" target='_blank'>
              Case Report: Genomic insights into prostate adenocarcinoma transdifferentiation to carcinosarcoma due to lineage plasticity
              </a>
            </td>
          <td>
            Tomohiro Fukui, Arinobu Fukunaga, Yuki Teramoto, Maki Fujiwara, Kensuke Hikami, T. Sunada, Kei Mizuno, Yuki Kita, Takayuki Sumiyoshi, Takayuki Goto, R. Saito, Takashi Kobayashi, S. Akamatsu
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Despite the clear genetic evidence linking serous tubal intraepithelial carcinoma (STIC) and high-grade serous ovarian carcinoma (HGSOC), the specific conditions and events that promote the progression of STIC lesions into invasive disease remain poorly understood.



 As a critical initial step, we have assembled a cohort of incidental p53 signatures (p53.I), incidental STIC (STIC.I), and STIC with concurrent HGSOC (STIC.C). We have performed extensive multi-modal analysis using multiplexed tissue imaging and spatial transcriptomics to identify features of the immune system that play a vital role in the early steps of HGSOC development. We have processed 43 specimens using highly multiplexed tissue imaging at single-cell resolution (cyclic immunofluorescence, CyCIF), and 35 specimens for micro-regional spatial whole transcriptomics using the GeoMx (Nanostring) on over 500 pathologist-annotated regions of interest.



 We showed a temporal evolution of HGSOC, including the activation of the interferon (IFN) signaling pathway underpinning the progression from the very early stage, p53.I. Additional cancer hallmark pathways, such as Transforming Growth Factor Receptor-β, Hypoxia, were induced in the tumor microenvironment (TME), including STIC.C. One of the mechanisms of the early induction of IFN might be induced by DNA damage response and chromosomal instability, potentially activating the cGAS-STING pathway. We validated IFN activation at the early stage of HGSOC by tissue imaging. For instance, more epithelial cells expressing both HLA-A as well as HLA-E, p-TBK1 and p-STAT3 with disease progression than FT regardless of BRCA status (p<0.001, GLMMs). We found IFN signaling to be persistent at STIC.C and further upregulated in the tumor. To model the evolution of the cancer ecosystem, we then looked into immune composition at both protein and RNA levels. Our data suggested active immune surveillance and potential elimination for p53 signatures. Immune surveillance was potentially further increased at STIC.I by an increased activated cDC1, antigen-presenting cells (APCs), and >30% activated (PD1+) cytotoxic (CD8+ CD103-) and TRM with some level of exhaustion (LAG3+). STIC.C, presumably resembling late clonal expansion, had less intra-tumoral NK and cDC1 in both protein and RNA levels. There was a gradual decline in total CD8+ T cells with HGSOC progression, while 30-40% expressed Ki67/PD1 or LAG3 in cancer, indicating immune editing. Exhaustion state was further shown with HAVCR2 expression (TIM3). We found that one of the contributors of T cells exhaustion was chronic IFN.



 Epithelial cells with IFN activation seem to be under a natural selection with HSGSOC progression. One of the selective advantages might be the upregulation of MHC-class I. HLA-E upregulation by tumor cells was shown to be immune suppressive by reducing NK cell surveillance. Along with the gradual accumulation of cancer hallmark pathways, an emergence of immune suppressive mechanisms seems to arise at the very early precursor stage.



 Tanjina Kader, Jia-Ren Lin, Clemens Hug, Shannon Coy, Yu-An Chen, Ino de. Bruijin, Clarence Yapp, Jeremy L. Muhlich, Nikolaus Schultz, Charles W. Drescher, Peter K. Sorger, Ronny Drapkin, Sandro Santagata. Immune suppression in the fallopian tube: Insights from the pre-cancer atlas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA009.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bfa669bc19f1cf6d070d242569736aa782cd91a" target='_blank'>
              Abstract IA009: Immune suppression in the fallopian tube: Insights from the pre-cancer atlas
              </a>
            </td>
          <td>
            Tanjina Kader, Jia-Ren Lin, Clemens Hug, S. Coy, Yu-An Chen, Ino de. Bruijin, Clarence Yapp, J. Muhlich, N. Schultz, Charles W Drescher, P. Sorger, Ronny Drapkin, S. Santagata
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) is a highly vascularized and immunogenic malignancy with a complex tumor microenvironment (TME) that shapes disease progression and therapeutic resistance. Despite advances in immune checkpoint inhibitors (ICIs) and targeted therapies, clinical responses remain heterogeneous, underscoring the need for a deeper understanding of RCC immunobiology. This review comprehensively examines the immunosuppressive TME of RCC, emphasizing the roles of cytotoxic and immunosuppressive immune cells, carcinoma-associated fibroblasts (CAFs), abnormal vasculature, and extracellular matrix (ECM) remodeling in fostering immune evasion. This review summarized emerging biomarkers—including PD-L1 expression, tumor mutational burden (TMB), gene mutations, and immune-based subtypes—that may predict ICI response. Furthermore, we evaluate current immunotherapeutic strategies, such as ICIs, combination therapies, and novel approaches targeting immunosuppressive cells and metabolic pathways. While combination therapies have improved outcomes, challenges like toxicity and resistance persist, necessitating biomarker-driven patient stratification and optimized treatment sequencing. Future directions should focus on deciphering TME heterogeneity and developing precision immunotherapy strategies to enhance clinical efficacy in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c635f92429f0a0ad81f2c837971a966305d3f506" target='_blank'>
              Characteristics of the tumor microenvironment and potential immunotherapy strategies in renal cell carcinoma
              </a>
            </td>
          <td>
            Hui Wen, Shi Zheng, Xiaoqin Zhu, Ling Wang, Dongping Chen
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b038428041ef5d754c8d9cbe5398a0911b1f648" target='_blank'>
              Fusion-driven oncogenic programs shape the immune landscape in translocation renal cell carcinoma.
              </a>
            </td>
          <td>
            Prathyusha Konda, Cary N. Weiss, Yantong Cui, Sayed Matar, Jinyu Wang, Yasmin L Nabil, Riva Deodhar, Jiao Li, J. Horst, Sabrina Y. Camp, A. Sheshdeh, Jonathan L. Hecht, David J. Einstein, Yashika Rustagi, Anwesha Nag, A. Thorner, Cheng-Zhong Zhang, E. V. Van Allen, S. Signoretti, T. Choueiri, Srinivas R Viswanathan
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive and lethal malignant brain tumor. Cell–cell interactions (CCIs) in the tumor microenvironment, mediated by ligand–receptor (LR) pairs, are known to contribute to its poor prognosis. However, the prognostic influence of CCIs on patients with GBM and the spatial expression profiles of such LR pairs within tumor tissues remain incompletely understood. This study aimed to identify prognostic LR pairs in GBM and their intratumoral localization via multitranscriptomic analysis. The CCIs among GBM cells as well as between GBM and niche cells were comprehensively evaluated using 40,958 cells in single-cell RNA sequencing datasets. They were found to form intercellular networks in GBM by specific LR pairs, which were mainly implicated in extracellular matrix (ECM)-related biological processes. Survival analysis revealed that 13 LR pairs related to ECM biological processes contributed to poor prognosis (p < 0.05, and 95% confidence intervals > 1). Notably, our spatial transcriptomic analysis using three independent GBM cohorts revealed that the identified poor prognostic LR pairs were localized in specific regions within GBM tissues. Although the clinical importance of these LR pairs requires further investigation, our findings suggest potential therapeutic targets for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/102a7c96b1dcd02ce81762c455b5d0717c9c4822" target='_blank'>
              Integrated Spatial and Single-Cell Transcriptomics Reveals Poor Prognostic Ligand–Receptor Pairs in Glioblastoma
              </a>
            </td>
          <td>
            Makoto Yoshimoto, Kengo Sugihara, Kazuya Tokumura, Shohei Tsuji, E. Hinoi
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Lineage plasticity, a critical hallmark of cancer progression, enables tumor cells to evade inhibition of primary oncogenic pathways through histologic transformation. This adaptive process, driven by stemness-associated features and epigenetic reprogramming, poses significant challenges in treatment. Using non-small cell lung cancer and prostate cancer as models, we examine the utility of tissue and liquid biopsies in detecting histologic transformations and tailoring treatments to specific subtypes, which has profound clinical implications, potentially improving outcomes in patients with advanced, therapy-resistant disease. We also discuss emerging therapeutic strategies, including novel molecular targets, and address ongoing clinical challenges in managing treatment-emergent histologic transformation.


SIGNIFICANCE
The advent of highly effective molecularly targeted therapies results in increased recognition of treatment-emergent histologic transformation. This review not only summarizes current evidence on diagnosis and management of lineage plasticity but also explores therapeutic strategies under study, outlining a framework for clinical translation and successful drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49dc4debf7b809a5797b2dd12ac58569636b53ca" target='_blank'>
              Histologic Transformation in Cancer: The Path for Clinical Translation.
              </a>
            </td>
          <td>
            Ioannis A. Vathiotis, Avisek Banerjee, Utsav Sen, Felix C Saalfeld, Debdatta Halder, A. Charpidou, Konstantin Syrigos, H. Horinouchi, Ashutosh K Tewari, Mark A Dawson, M. Wermke, Triparna Sen
          </td>
          <td>2025-09-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Colorectal laterally spreading tumors (LSTs) are defined as non-protruding neoplasms exceeding 10 mm in diameter that grow primarily along the intestinal wall. The morphogenetic mechanisms and evolutionary trajectories of granular-type LSTs (LST-Gs) towards colorectal carcinoma remain unclear. Methods In this study, we investigate the transcriptional features of LST-Gs using single-cell RNA sequencing technology by comparing them with protruded-type adenomas (PAs) and normal mucosal tissues. Results Adenomatous LST-Gs harbor an epithelial cell population with metaplastic differentiation, which are almost absent in PAs. Cells with a high degree of differentiation in LST-G demonstrate enhanced immunogenicity and robust adhesion/junction interactions. Furthermore, LST-Gs show upregulated expression of molecular chaperones and metallothioneins compared to PAs, reflecting a more hostile microenvironment similar to that observed in carcinoma stages. These characteristics suggest that LST-G exhibits greater heterogeneity compared to the earliest colorectal adenomas, mirroring the progression from precancerous states to cancer. Notably, the Arp2/3 complex is significantly upregulated in highly differentiated LST-G cell populations, potentially facilitating cell migration along the basement membrane, which highlights the similarities in cell motility between adenomatous LST-G and normal mucosal epithelium as well as serrated polyps (SERs). Conclusion The differentiation state of cells within LST-G exhibits a close correlation with their diverse characteristics. Metaplastic differentiation, as a prominent feature at the transcriptional level, demonstrates significant associations with the genomic features, morphogenesis, and tumor progression of LST-G.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21cfb702cab41be0ca69fc6ec0389c21e705f623" target='_blank'>
              Single-cell RNA profiling of colorectal granular-type laterally spreading tumor uncovers progression trajectory toward carcinoma and transcriptional signatures favoring lateral morphogenesis
              </a>
            </td>
          <td>
            Yueqing Gong, Yuxin Zhang, Xun Liu, Rongli Cui, Jingjing Lu, Jun Li, Fang Gu, Jing Zhang, Shigang Ding, Weiwei Fu
          </td>
          <td>2025-09-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer with a dismal five-year survival rate of 13%. Most patients present with metastatic disease, and 80% undergoing surgery relapse within two years. Research on PDAC has traditionally focused on primary tumors, thus most treatment derives from primary tumor biology. Recent evidence unveiled significant differences in the tumor immune microenvironment (TIME) between primary and metastatic lesions. Understanding these differences is crucial, as immunotherapies are predominantly administered to patients with refractory metastatic PDAC, yet traditional treatment fails to account for organ specificity and patient TIME signatures. In this study, we wanted to know why the same cancer cell type in different organs would create different microenvironments in the pancreas, liver, and lungs. To do this, we investigated the TIME in early and late-stage metastatic cohorts by multiplex immunohistochemistry (mIHC), transcriptomic assays and spatial analysis. To profile the TIME of different immune and stromal cells, we employed spatial protein expression using mIHC platform for the early cohort. In total, we applied a 30-plex panel to 56 samples. Our analysis revealed a higher density of COL1A1+ fibroblasts and CD163+ macrophages in the liver compared to other sites. Additionally, liver metastases exhibited significantly fewer CD8+ T cells compared to other organs within the tumor core. These results confirmed the earlier studies regarding the TIME of metastatic lesions, while also showing more macrophage and fibrosis in the liver. In the late-stage cohort, similar trends of decreased immune cell presence and exclusion phenotypes were observed using mIHC and Visium platforms. To explore the underlying mechanisms, we performed transcriptomic assays on the immune compartment. Our findings indicated an enrichment of interferon-positive macrophage subsets, particularly in liver metastases compared to primary PDAC. Additionally, we observed a higher presence of exhaustive CD8+ T cells and regulatory T cells in primary sites compared to liver and lung metastases, suggesting that both primary PDAC and liver metastases possess an immunosuppressive environment. Furthermore, spatial proximity analysis demonstrated that CD8+ T cells were closer to tumor cells in lung metastases compared to liver and primary tumors, reinforcing the notion that lung metastases are more immune-active. Pathway analysis revealed downregulation of antigen presentation and lymphocyte proliferation signaling in liver T cells compared to those in the lungs. Notably, immune infiltration was observed in some liver samples, but the percentage was significantly lower compared to lung samples. This divergence in TIME modulation between organs suggests a significant impact on disease progression in PDAC patients. The significant difference in immuno-response across organ shows why it is critical to consider distinct immune landscapes of different organs and patient histories when developing tailored therapies for advanced PDAC patients.



 Yu-Lan Kao, Olivia M. Dres, Blake G. Sells, Varun Shenoy, Xiang Li, Kapur Dhami, Paul M. Grandgenett, Li Ding, Liang-I Kang, David G. DeNardo. Dynamics of tumor immune environment across organs in early and late-stage metastatic pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A084.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/846e100bfc6ae1b16ca4e9e6b72859a2dc02155f" target='_blank'>
              Abstract A084: Dynamics of tumor immune environment across organs in early and late-stage metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Yu-Lan Kao, Olivia M Dres, Blake Sells, Varun V Shenoy, Xiang Li, Kapur B. Dhami, P. Grandgenett, Li Ding, Liang-I Kang, D. DeNardo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background Immunotherapy has progressively gained prominence as a cornerstone therapeutic modality across diverse oncological contexts, with its clinical efficacy intricately linked to the dynamic interactions between the tumor microenvironment (TME) and neoplastic cells. Central to this paradigm is angiogenesis—a quintessential hallmark of cancer—which not only sustains tumor growth but also orchestrates immunomodulatory networks within the TME, thereby profoundly influencing therapeutic responsiveness. However, in the field of bladder cancer (BC), the relationship among angiogenesis and prognosis, immunotherapy response, and immune cell infiltration remains to be further explored. Methods To systematically uncover this relationship, we carried out an exhaustive assessment of 36 genes linked to angiogenesis (AAGs) and explored the relationship between angiogenesis and transcript, prognostic outcomes, as well as the infiltration of immune cells. By constructing an AAG_score, we quantified the angiogenic subtype characteristics of each patient. Subsequently, we evaluated the value of these characteristics in foreseeing BC prognosis and the response of treatment, and concurrently analyzed the performance of AAGs in diffuse large B-cell lymphoma for comparative study. Through RT-qPCR, CCK8 and other experiments, we verified the role of NID2 in bladder cancer. Results This study explored different types of AAGs mutations in BC samples at the genetic level and elucidated the expression patterns of AAGs. Through in-depth analysis, we identified two distinct molecular subpopulations and found significant associations between AAG mutations and patients’ clinicopathological features, prognosis, and invasive TME. We found that patients with low AAG_score showed increased microsatellite instability, high mutation tendency, and immune motivation, and had a better prognosis. NID2 plays a role in promoting proliferation in bladder cancer and evaluate in diffuse large B-cell lymphoma. In addition, our study found a significant correlation between index of cancer stem cell and AAG_score in drug sensitivity. Conclusion In summary, our study successfully identified prognosis-related AAG characteristics in BC patients. These characteristics not only contribute to a clearer understanding of TME properties but also provide an important basis for exploring more effective immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e0c3a38e058744faab2e7b9e1bb3c40f92cc4d" target='_blank'>
              Angiogenesis-related gene NID2 profiling and immune infiltration in bladder cancer: prognostic implications and immunotherapy response
              </a>
            </td>
          <td>
            Jinxia Cao, Wang Zheng
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background With immune checkpoint inhibitors (ICIs) now widely used as a primary treatment for metastatic RCC, understanding the composition and function of the tumor microenvironment (TME) has become more important. Numerous studies have focused on T cells to explain both treatment response and resistance, and single-cell RNA sequencing (scRNA-seq) has revealed the complex heterogeneity of T cell states. However, the contribution of myeloid cells (particularly tumor-associated macrophages (TAMs), which are known to suppress antitumor immunity) remains incompletely understood in the context of ICI therapy. In this study, we aimed to investigate TAM populations associated with ICI resistance in RCC using single-cell transcriptomic profiling. Methods We analyzed 70 tumor samples (58 clear cell and 12 non-clear cell) obtained from 63 patients with advanced RCC. The cohort included 9 untreated patients, 10 who received non-ICI systemic therapies, and 44 who were treated with ICI-based regimens. From the ICI-based therapy group, we excluded 17 patients who had stable disease and focused on 29 tumor samples from 27 patients with either pre-treatment (n = 15) or post-treatment (n = 14) samples. These patients received various ICI regimens, including mono-ICI (n = 11), ICI plus ICI (n = 11), ICI plus VEGF inhibitor (n = 6), and ICI plus IDO1 inhibitor (n = 1). We performed scRNA-seq (10x Genomics) on all samples and applied non-negative matrix factorization (NMF) to identify transcriptional programs within TAMs. We then compared these programs between responders (n = 18, complete or partial response) and non-responders (n = 11, progressive disease) based on RECIST criteria. Statistical significance was assessed using the Wilcoxon signed-rank test. Results A total of 443 337 high-quality viable cells were analyzed and classified into major cell types, including lymphoid, myeloid, tumor, endothelial, and fibroblast compartments. Within the TAM compartment, NMF uncovered 8 gene expression programs, such as “antigen presentation”, “S100A8/9 inflammation”, “stress response”, “C1Q/APOE/TREM2 signaling”, “CD163/MRC1-M2-like”, “hypoxia-related signaling”, “interferon-stimulated response”, and a distinct “LILRB/SIGLEC10” immunosuppressive program. This LILRB/SIGLEC10-enriched TAM subcluster was significantly more abundant in non-responders than in responders (P = .005). Importantly, this difference was also observed in pre-treatment samples alone (P = .014), suggesting it may be involved in primary resistance. These TAMs expressed higher expression levels of immunosuppressive LILRB1/2/3 genes, the inhibitory receptor SIGLEC10 (a recently characterized “don’t eat me” signal), and the immune checkpoint molecule VISTA, compared to other TAM subclusters (p < 2.22E-16 for each gene). Conclusions Our scRNA-seq-based analysis identified a distinct population of TAMs characterized by immunosuppressive transcriptional programs associated with poor response to ICI therapy in RCC. These findings provide insight into potential mechanisms of resistance and suggest that targeting this TAM subset may improve therapeutic efficacy. This study also demonstrates the utility of single-cell transcriptomics for uncovering key immunoregulatory populations in large clinical cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c5a6a9e73d777a1a4cc9b52941635d122ce502" target='_blank'>
              41Single-cell dissection of immunosuppressive myeloid subclusters driving resistance to immune checkpoint therapy in renal cell carcinoma (RCC)
              </a>
            </td>
          <td>
            S. Kashima, Rishabh Rout, M. Hugaboom, Zhaochen Ye, N. Schindler, A. Dighe, Maxine Sun, G. M. Lee, Wenxin Xu, S. Signoretti, Bradley A. McGregor, Rana McKay, Toni Choueri, D. Braun
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="


 The dynamic evolution of the immune tumor microenvironment (TME) during targeted therapy is a critical yet poorly understood determinant of treatment response and resistance. While most studies compare immune states before and after treatment, temporal immune changes during therapy remain largely uncharacterized, limiting development of effective combination strategies. Here, we investigated immune dynamics throughout targeted therapy using mouse melanoma models that recapitulate human therapeutic responses. Single-cell RNA sequencing (scRNA-seq) identified an inflection point where the inflamed TME during tumor regression, characterized by robust NK cell infiltration, transitions to an immune-excluded state upon onset of drug-tolerant residual disease. We uncovered a unique macrophage subset that orchestrates NK cell recruitment during regression. Depletion of these macrophages using LysM-cre;iDTR mice significantly reduced NK cell infiltration. Extending these findings to human cancer, longitudinal scRNA-seq analysis of melanoma and lung cancer patient samples revealed dynamic NK cell infiltration during targeted therapy, establishing a direct link between innate immune remodeling and treatment outcome. Unlike prior prognostic studies assessing immune states at single time points, our results provide mechanistic evidence of a temporal relationship between NK cell infiltration and therapeutic efficacy. Together, these findings position immune evolution as a driver of acquired resistance and identify macrophage–NK cell crosstalk as a therapeutically actionable axis to overcome immune exclusion and improve targeted therapy.



 Chia-Hsin Hsu, Andrew C. White. Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064d31f044dc7aac267bdcec71d797f7f005e82d" target='_blank'>
              Abstract A012: Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions
              </a>
            </td>
          <td>
            Chia-Hsin Hsu, Andrew C White
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Transforming Growth Factor (TGF) and Bone Morphogenetic Protein (BMP) signalling critically influence pancreatic ductal adenocarcinoma (PDAC) progression, with TGF-B paradoxically exerting both tumour-promoting and -suppressive effects. Parallel to this observation, the specific context-dependent, spatial dynamics of these pathways and their interaction with the tumour microenvironment (TME) remain poorly understood. Methods We performed a spatially resolved analysis of PDAC on a multi-region tissue microarray cohort of 117 curatively resected PDAC specimens consisting of tumour centre (TC), tumour front (TF), and stromal(-predominant) tissue cores each. Protein (ID1, pSMAD2) and mRNA (TGF-A, TGF-B1/2, BMP4, GREM1) expression were assessed in each tissue compartment by immunohistochemistry and in situ hybridization, respectively, quantified by digital image analysis, and correlated with clinicopathologic features. Results ID1 was significantly overexpressed in PDAC cells compared to associated stroma (p < 0.01), while pSMAD2 was largely absent in PDAC cells, but preserved among associated stroma compartments, particularly in TF cores (p = 0.04). Higher stromal GREM1 signal correlated with reduced overall tumoural ID1 protein expression (p = 0.02), and TGF-B2high/TGF-Alow stroma was significantly associated with worse survival (p < 0.01). Intratumoural TGF-B2 was inversely correlated with stromal pSMAD2 expression (p = 0.03) and was associated with lymph node involvement (p = 0.02). FOXP3+ regulatory T-cells were significantly reduced in TGF-B2high tumours (p = 0.04), while higher tumoural TGF-B1 exhibited a trend towards increased FOXP3+ cells (p = 0.08). Conclusions Our spatial analysis reveals intratumoural heterogeneity of TGF/BMP signalling and its significance for PDAC progression. Notably, stromal TGF-B2 emerges as a prognostic biomarker, while TGF-B1 and ID1 are implicated in adverse clinical and pathologic features. These findings highlight the importance of TGF/BMP signalling niches in the TME with implications for PDAC biology and can inform the development of future therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14751-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0085a0bcef90131114a7aad5d7931cab1f83ce7d" target='_blank'>
              Spatially resolved analysis of TGF/BMP signalling in pancreatic ductal adenocarcinoma by digital pathology identifies patient subgroups with adverse outcome
              </a>
            </td>
          <td>
            Konstantin Bräutigam, Philipp Zens, Stefan Reinhard, Jessica L. Rohrbach, Simon J Leedham, A. Wenning, B. Gloor, V. H. Koelzer, Martin Wartenberg
          </td>
          <td>2025-08-18</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Intratumoral heterogeneity remains a major barrier to durable cancer therapies, largely driven by the persistence of cancer stem cells (CSCs). In this study, we employed an integrated, multi-scale approach to investigate how melanoma CSCs respond to siRNA-mediated silencing of three key regulatory genes: KLF4, SHH, and HIF1α. Using a combination of morphological, molecular, spectroscopic, and elemental analyses, we explored structural and biochemical consequences of gene knockdown. Gene silencing resulted in significant changes in cell shape and size, reduced F-actin organization, and decreased PFN1 expression, indicating a loss of stem-like properties. ATR-FTIR spectroscopy revealed shifts in biomolecular composition, notably a reduction in amide III intensity and an increase in lipid ester content. SEM-EDS point-based elemental analysis revealed SEM-EDS point-based elemental analysis revealed relative differences in carbon and nitrogen levels between selected central and peripheral regions of silenced and control cells, at the micron-scale working depth, reflecting broader elemental distribution trends rather than precise subcellular compartmentalization. XPS analysis further confirmed these differences, providing additional insights into the elemental composition of the cellular surface. The integration of FTIR spectroscopy into this study highlights the potential of infrared spectroscopy as a powerful tool in cancer research. These findings demonstrate that targeting critical regulatory pathways induces cytoskeletal and biochemical remodelling in melanoma CSCs, offering a multi-dimensional perspective on cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5835a6c3f340e01657d8039c9562c5a0b45cfc" target='_blank'>
              Integrated spectroscopic and morphological analyses reveal cellular shifts in gene-silenced melanoma CSCs
              </a>
            </td>
          <td>
            Berrin Ozdil, Günnur Güler, E. Ataman, H. Aktug
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Despite decades of progress in ovarian cancer treatment, therapeutic resistance remains a persistent and lethal challenge. Emerging evidence suggests polyploid giant cancer cells (PGCCs) as key drivers of this resistance. PGCCs are distinguished by multinucleation or a single oversized nucleus containing multiple chromosome sets. Polyploid cell populations are present in premalignant lesions and expand with disease progression and therapeutic stress. PGCCs demonstrate heightened resistance to treatment relative to non-PGCCs and can regenerate non-PGCCs via budding once therapeutic pressure is alleviated, leading to relapse. Supported by extensive preclinical and clinical evidence, we hypothesize that elimination of PGCCs may reduce therapeutic resistance in ovarian cancer. Given their significant role in resistance and relapse, PGCCs remain largely untargeted due to a lack of scalable detection methods. Standard viability assays such as MTT, XTT, and ATP-based screens fail to capture the rare but critical PGCC subpopulation. Current gold-standard methods, fluorescence-activated cell sorting (FACS) coupled with manual microscopy, while accurate, are time-intensive and unsuitable for high-throughput drug screening. Thus, a rapid and accurate technique for studying PGCCs is essential. In this pilot study, we have developed a high-throughput single-cell morphological analysis pipeline capable of precisely quantifying PGCC numbers and proportions. The pipeline accurately quantifies PGCCs based on nuclear and cellular features, including size and DNA content. Leveraging this platform, we screened a library of 2,726 FDA-approved compounds to identify agents that effectively kill PGCCs. Hits from this screen were validated in 3D spheroid cultures, revealing compounds with potent anti-PGCC activity. Complementary transcriptomic analysis (RNA-Seq) of flow-sorted PGCCs versus non-PGCCs unveiled profound dysregulation of the FOXM1 signaling axis, implicating this pathway as a key regulator of PGCC formation and survival. Consistent with this, the high-throughput screen identified Thiostrepton, a known FOXM1 inhibitor, as a selective anti-PGCC agent. Across multiple ovarian cancer cell lines, FOXM1 inhibitors outperformed standard-of-care agents (Paclitaxel, Cisplatin, Olaparib) in eliminating PGCCs. siRNA knockdown of FOXM1 significantly reduced PGCC numbers, particularly under Paclitaxel stress. Moreover, FOXM1 inhibition triggered apoptosis in PGCCs, an effect reversible with the pan-caspase inhibitor Z-VAD-FMK. Together, these findings underscore PGCCs as a therapeutically actionable cell state driving resistance in ovarian cancer. Our integrative single-cell morphological and transcriptomic platform enables rapid discovery of anti-PGCC agents and establishes FOXM1 inhibition as a compelling strategy to eradicate this resilient subpopulation. This pipeline offers broad utility for overcoming therapy resistance across diverse malignancies.



 Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen. Combating ovarian cancer resistance via polyploid cancer cell targeted screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62f27ef6b0280aa103e9f9c96579bbfa985df28c" target='_blank'>
              Abstract A023: Combating ovarian cancer resistance via polyploid cancer cell targeted screening
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSC) typically originates in the Fallopian tube epithelium (FTE) and is characterized by genomic instability and near-universal TP53 defects. HGSC is typically diagnosed at an advanced stage and rapid progression once it becomes invasive. To date, immunotherapies have had limited efficacy in HGSC. Most current HGSC research focuses on tumor and immune profiling of fully fledged HGSC, where in the tumor microenvironment (TME) has already been established. Much less is known about the microenvironment of pre-neoplastic lesions and the role of the immune system in controlling HGSC emergence. Hence, there is an urgent need for in-depth research on the molecular interactions between precancerous cells and immune cells to better understand the process of malignant transformation and escape from host immunity.



 By using our genetically defined, syngeneic organoid platform, FTE organoids were engineered to Trp53 -/- ,Ccne1-overexpressing FTE organoids (“PC” organoids) and Brca1 +/- ;Trp53 -/- (PBrca1) and Brca2 -/- ;Trp53 -/- (PBrca2) to model Homologous recombination (HR)-deficient (HRD) and HR-Proficient (HRP) HGSC, respectively. Pre-tumor growth was monitored using an in vivo imaging system (IVIS) following luciferase-labeled precancerous cell ovarian bursal injection. In addition, adaptive transfer and antibody deletion were performed to determine which immune cell type(s) mediate immune surveillance.



 In a previous study, our laboratory found that PC organoids are not tumorigenic in wild-type (WT) C57BL6 mice. In this study, I found that PC cells were eliminated around 1 month after bursal injection in WT mice. Surprisingly, however, they kept growing and eventually formed tumors in Rag2 -/- mice, which lack B and T lymphocytes. Then, T cells were adaptive transfer from WT mice bearing PC cells to Rag2 -/- mice. I observed a marked reduction in PC cell signal as early as 24 hours post-transfer, which suggests T cells play a critical role in the clearance of precancerous cells. I also found compared to PC cell, PBrca1 and PBrca2 FTE organoids, both of which are non-tumorigenic, are eliminated much earlier (in 24 hours) and was completed in rapidly (∼15 days) following bursal injection into both WT and Rag2 -/- mice. Following NK cells deletion in Rag2 -/- mice, I observed that PBrca1 cells successfully established growth, whereas PBrca2 cells exhibited significantly delayed clearance compared to the IgG control group. Additionally, compared to PC cells, the expression of mouse MHC-Ib Qa-1b, which inhibits NK cell–mediated cytotoxicity, was markedly reduced in PBrca1/2 cells. Although other explanations are possible, these data raise the possibility that PBrca1/2 cells elicit a strong NK cell mediated innate immune response.



 This study represents a novel and highly feasible approach to interrogating the pre-cancer TME. My findings demonstrate that HRD (PBrca1/2) and HRP (PC) trigger markedly divergent immune surveillance mechanisms during the pre-malignant phase.



 Jianmei Hou, Benjamin Neel. Organoid models reveal distinct mechanisms of pre-tumor immune surveillance in high grade serous ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/263093496afb42b25c3ac1b6f0bdab9faf970516" target='_blank'>
              Abstract PR003: Organoid models reveal distinct mechanisms of pre-tumor immune surveillance in high grade serous ovarian cancer
              </a>
            </td>
          <td>
            Jianmei Hou, Benjamin G Neel
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the deadliest diseases, with a 5-year survival rate of only 13%. While rare mutations have been identified to drive PDAC therapy resistance, undiscovered potent, post-transcriptional regulatory mechanisms could be key to understanding this resistance. The RNA-binding protein HuR (ELAVL1) is known to play a critical role in PDAC oncogenesis by stabilizing mRNA transcripts involved in key oncogenic pathways. Our lab has shown that HuR is overexpressed in PDAC tumor and promotes multiple pro-survival signaling pathways. To study HuR-mediated PDAC progression in immune competent models that reflect the tumor microenvironment, we deleted ELAVL1 via CRISPR-Cas9 in Kras-p53 mutant-driven (KPC) murine PDAC cells and found that HuR-knockout (KO) KPC tumors grew more slowly compared to KPC wild-type (WT) tumors in orthotopic implantation settings. Using flow cytometry and multiplex immunohistochemistry, we found that HuR-WT tumors had significantly less T cell infiltration and greater endothelial cell infiltration compared to HuR-KO tumors. Deep transcriptomic and proteomic profiling revealed that HuR mediates the expression of multiple metabolic pathways and determines mRNA and protein cargo loading in secreted extracellular vesicles (EVs). In fact, when mice bearing HuR-KO tumors were injected with purified EVs from HuR-WT cells, both tumor size and endothelial cell function in leukocyte trafficking were rescued. In the context of T cell function, HuR directly enriched transcripts in the mTOR pathway, and inhibition of mTOR activity in HuR-WT tumor cells rescued T cell function in co-culture. Consistent with these observations linking tumor-intrinsic HuR to dysregulated endothelial and anti-tumor immune function, HuR-KO tumors were significantly more responsive to immune checkpoint blockade therapy as compared to WT-HuR tumors. Similarly, in a Kras-MYC driven mouse model, HuR-WT tumor cells metastasized at significantly higher rates to the liver compared to HuR-KO tumors. Finally, given the recent advancements in KRAS inhibitors, we tested their efficacy in these pre-clinical models, and surprisingly, we observed that HuR-KO tumors were up to >100-fold more resistant to KRAS inhibition than WT tumors. These observations could be explained by HuR expression and activity being closely linked to aberrant KRAS signaling, where loss of HuR leads to the reliance of compensatory pathways and drives PDAC cells away from a KRAS addiction phenotype, thus providing a setting of KRAS inhibitor resistance. Accordingly, surviving PDAC cells post-KRAS inhibitor treatment in vivo lost HuR expression compared to untreated tumors. Together, these findings highlight that HuR is a global post-transcriptional regulator that: 1) reshapes the tumor microenvironment to be immune evasive; 2) is involved in a feed-forward loop enhancing KRAS pathway activation; and 3) facilitates PDAC metastatic potential. Strategies to target HuR activity and use it as a potential biomarker could lead to improved therapeutic outcomes.



 Miffy Guo, Jennifer Finan, Hen Halamish, Katherine Pelz, Robert Eil, Rosalie Sears, Jonathan Brody. HuR drives immune evasion, metastases and KRAS addiction in pancreatic ductal adenocarcinoma: implications for IO-based and KRAS inhibitor therapies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9ebd5486262a2dcbf20ec9a786ef9c644e158e4" target='_blank'>
              Abstract A027: HuR drives immune evasion, metastases and KRAS addiction in pancreatic ductal adenocarcinoma: implications for IO-based and KRAS inhibitor therapies
              </a>
            </td>
          <td>
            Miffy Guo, Jennifer M. Finan, Hen Halamish, Katherine R. Pelz, Robert Eil, Rosalie C. Sears, Jonathan R. Brody
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Wilms tumor (WT), the most common pediatric renal malignancy, exhibits a relatively low mutational burden compared to adult cancers, which hinders the development of targeted therapies. To elucidate the molecular landscape of WT, we perform integrative proteomic, phosphoproteomic, transcriptomic, and whole-exome sequencing analyses of WT and normal kidney tissue adjacent to tumor. Our multi-omics approach uncovers prognostic genetic alterations, distinct molecular subgroups, immune microenvironment features, and potential biomarkers and therapeutic targets. Proteome- and transcriptome-based stratification identifies three molecular subgroups with unique signatures, correlating with different histopathological subtypes and putative cellular origins at different stages of embryonic kidney development. Notably, we identify EHMT2 as a promising prognostic biomarker and therapeutic target associated with epigenetic regulation and Wnt/β-catenin pathway. In this work, we provide a comprehensive molecular characterization of WT, offering valuable insights into its pathogenesis and a foundational resource for future therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3be681b81ddbc1d74bdc72b4011124ef4e90944" target='_blank'>
              Integrative proteogenomic characterization of Wilms tumor
              </a>
            </td>
          <td>
            Cheng Cheng, Li Zhang, Xiaofeng Chang, Kai Chen, Tian He, Jia Shi, Fan Lv, Lijia Pan, Yangkun Wu, Qianqian Cheng, Dong Ren, Y. Guo, Weiping Zhang, Huanmin Wang, Tieliu Shi, Jing Li, Xinxin Ni, Yeming Wu, Yaqiong Jin, Zhixiang Wu
          </td>
          <td>2025-08-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cytotoxic chemotherapy remains the standard-of-care treatment for patients with pancreatic ductal adenocarcinoma (PDAC). However, chemotherapy only has modest effects at improving patient survival due to primary or rapidly acquired chemoresistance. The biological underpinnings of PDAC therapy resistance are incompletely defined, but the tumor microenvironment is known to be a major contributor to chemoresistance. We have found chemoresistance is imprinted on PDAC cells by the tumor microenvironment and persists for a period of days after PDAC cells are removed from tumors. However, PDAC chemoresistance is lost upon long term culture in standard laboratory conditions. Interestingly, culture of PDAC cells in Tumor Interstitial Fluid Medium (TIFM), a culture medium we developed to recapitulate the nutrient availability of the tumor microenvironment, maintains PDAC cells in a chemo- and targeted therapy resistant state even after long term culture ex vivo. These findings suggest that microenvironmental metabolic stress keeps PDAC cells in a physiologically relevant, therapy resistant cell state that standard culture models fail to maintain. Using TIFM culture, we sought to understand how PDAC cells in this state resist therapeutic challenge. We found that chemo- and targeted therapies largely retain on-target activity within TIFM medium but fail to activate cell death, enabling a “chemotolerant” cell state, which is also observed in PDAC tumors. This chemotolerant state is driven by suppression of apoptotic priming and can be overcome by targeting the anti-apoptotic regulator BCL-XL. Taken together, these findings suggest that reprogramming of cell death mechanisms by the PDAC nutrient microenvironment is a key contributor to therapy resistance in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bea129494e792fbbf0176ec1d7ce93ff6acb889" target='_blank'>
              Tumor nutrient stress gives rise to a drug tolerant cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Colin Sheehan, Lyndon Hu, Guillaume Cognet, Grace Croley, Thao Trang Nguyen, Anika Thomas-Toth, Darby Agovino, Patrick B. Jonker, Mumina Sadullozoda, Leah M. Ziolkowski, James K. Martin, Alica K Beutel, Ranya Dano, Mohammed A. Khan, Christopher J Halbrook, Kay F. Macleod, Christopher R. Weber, James L LaBelle, Alexander Muir
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of chronic inflammation-associated cancers (CIACs) is shaped by cycles of injury and maladaptive repair, yet the principles organizing fibrotic stroma in these tumors remain unclear. Here, we applied the concept of hot versus cold fibrosis, originally credentialed in non-cancerous fibrosis of heart and kidney, to lung squamous cell carcinoma (LUSC), a prototypical CIAC. Single-cell transcriptomics of matched tumor and adjacent-normal tissue from 16 treatment-naive LUSC patients identified a cold fibrotic architecture in the LUSC TME: cancer-associated fibroblasts (CAFs) expanded and adopted myofibroblast and stress-response states, while macrophages were depleted. This macrophage-poor, CAF-rich stroma was maintained by CAF autocrine growth factor loops, including TIMP1, INHBA, TGFB1, and GMFB. In parallel, the immune compartment exhibited a hot tumor phenotype with abundant T and B cells, forming spatially distinct but molecularly engaged networks with CAFs. CAF gene programs typifying cold fibrosis in LUSC were conserved in other CIACs, including esophageal and gastric adenocarcinomas. These results redefine desmoplastic regions of tumors through the lens of a non-cancer fibrosis model, demonstrating that conserved stromal circuits constitute therapeutic vulnerabilities in CIACs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ed36e6ea62839a2c887c4f2b23f5bf7ed74c7d" target='_blank'>
              A fibroblast-centric network drives cold fibrosis in the tumor microenvironment of lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Shoval Miyara, Shachaf Frenkel, Avi Mayo, Philippe Gascard, Michael K. Strasser, David Gibbs, Eviatar Weizman, Itay Ben Shalom, Yan Stein, Deng Pan, Joseph A. Caruso, V. Sangwan, Nicholas Bertos, Julie Bérubé, Sophie Camilleri-Broet, Spyridon Oikonomopoulos, H. Djambazian, K. Umansky, Jacob Elkahal, Shimrit Mayer, Pierre Fiset, Jiannis Ragoussis, Miri Adler, Eldad Tzahor, Sui Huang, Lorenzo Ferri, T. Tlsty, Ruth Scherz-Shouval, U. Alon
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are malignant cells that detach from primary or metastatic tumors and enter the bloodstream. Organoids, as three‐dimensional in vitro models, can mimic the tumor microenvironment and histopathological characteristics, thereby serving as valuable tools in tumor research. CTC‐derived organoids retain tumor heterogeneity and metastatic potential, which provides a unique model for the study of metastatic cascade mechanisms, individualized drug screening, and precision therapy. However, the current research on CTC‐derived organoids faces challenges, such as the scarcity of CTCs and the high technical difficulty in their isolation and enrichment, which leads to a low success rate in constructing organoid models. Moreover, most existing studies focus on a single cancer type and lack systematic integration of full‐process standardization as well as cross‐cancer applicability. In this paper, we review the isolation and enrichment strategies of CTC‐derived organoids along with the techniques for optimizing in vitro culture systems, and discusses their potential applications. This review summarizes the existing results, analyzes the technical bottlenecks, and provides a theoretical basis for the standardized construction and application of CTC‐derived organoids, while promoting their application in tumor precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/971e10876cd9eda4f902688d2d69a778549269d1" target='_blank'>
              Circulating Tumor Cell‐Derived Organoids: Current Progress, Applications, and Future
              </a>
            </td>
          <td>
            Tiantian Li, Biao Deng, Shenglong Li, Yanxia Wu, Zhenghao Lu, Zhu Liang
          </td>
          <td>2025-09-01</td>
          <td>MedComm – Future Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d43ce8f81cf0d90a92d73349eebd57717f9bf249" target='_blank'>
              Decoding adenomyosis pathogenesis using an assembloid model.
              </a>
            </td>
          <td>
            Yiliang Xu, Tao Cheng, Jianzhang Wang, Zhiruo Qiu, Xiaohong Guan, Yayuan Yu, Jiacheng Shen, Fangfang Xu, Xiaohong Jiang, Dandan Bai, Mingzhu Wang, Shuyan Mei, Hong Wang, Xiaocui Xu, Li Wu, Shaorong Gao, Xuan Che
          </td>
          <td>2025-08-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f06d8c3c1ce1c0e2c42d69d2aefb0ff5769089e" target='_blank'>
              STAT3 sustains tumorigenicity following mutant KRAS ablation.
              </a>
            </td>
          <td>
            S. D'Amico, V. Kirillov, Jingxuan Liu, Zhijuan Qiu, Xinyuan Lei, Hong Qin, Brian S. Sheridan, Nancy C. Reich
          </td>
          <td>2025-08-26</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Diffuse midline gliomas (DMGs) are universally fatal pediatric brain tumors with a median survival of just 11 months. Despite radiation therapy offering temporary benefit, there are no effective long-term treatments, and targeted therapies have shown minimal impact. The infiltrative nature of DMGs, combined with their critical brainstem location, underscores the urgent need to identify novel mechanisms driving tumor progression and resistance. Cellular senescence is a state of permanent cell-cycle arrest induced by stress and has traditionally been viewed as tumor-suppressive. However, recent evidence in adult cancers implicates senescent cells as active contributors to tumor growth, treatment resistance, and relapse through the secretion of pro-tumorigenic factors known as the senescence-associated secretory phenotype (SASP). While senescence has emerged as a hallmark of cancer in adults, its role in pediatric tumors such as DMGs remains largely unexplored. Using single-cell RNA sequencing of treatment-naïve DMG specimens, we identified rare populations of senescent glioma cells and microglia that would be undetectable using conventional approaches. Computational analyses revealed predicted paracrine interactions between senescent and non-senescent cells including pathways previously linked to tumor invasiveness and resistance. This study aims to define how senescent cells and their secretory programs shape the tumor microenvironment, promote DMG progression, and contribute to therapy resistance. By elucidating these mechanisms, we hope to uncover new therapeutic vulnerabilities in DMGs and expand our broader understanding of senescence as a driver of malignancy in pediatric cancers.



 Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya M. Rangaswamy, Claudia Kleinman, Pratiti Bandopadhayay. Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785f69efbe0f5fccc4e2b6c0500f58472565b0dc" target='_blank'>
              Abstract B040: Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance
              </a>
            </td>
          <td>
            Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya Rangaswamy, Claudia L Kleinman, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Glioblastoma multiforme represents the most common primary malignant brain tumor in adults, with a 90% 2-year recurrence rate despite standard therapy of surgical resection followed by chemo- and radiotherapy. Yet, the cellular mechanisms underlying therapeutic resistance remain poorly understood. By integrating analysis of single-cell RNA sequencing results of 41 primary and 43 recurrent tumor specimens with experimental validation, we identify neuro-oligodendrocyte precursors as the predominant cellular population driving tumor recurrence through establishment of protective microenvironmental niches. We discover that CTNNA3 and MBP exhibit paradoxical roles in promoting survival in primary tumors while conferring chemoresistance in recurrent GBM, revealing that these factors determine treatment efficacy rather than directly driving relapse. Functional assays confirmed their role as mediators of extracellular matrix remodeling, establishing a mechanistic link between tumor microenvironmental remodeling and therapeutic resistance. By revealing how recurrent tumors engineer self-protective niches, our study provides a new framework for developing combination therapies that disrupt these resistance mechanisms and provides foundation for overcoming treatment failure in this universally fatal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec11a04512f1a7cb7a16592e9fd8fa7748da070" target='_blank'>
              CTNNA3 and MBP Mediate Treatment Resistance Through Tumor Microenvironment Remodeling in Recurrent Glioblastoma
              </a>
            </td>
          <td>
            Wenna Mao
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intratumoral microbiota, a critical component of the tumor microenvironment, notably impacts tumor progression through various complex mechanisms such as metabolic regulation, immune system remodeling and genotoxicity. The present review focuses on eight prevalent solid tumors (breast, colorectal, lung, pancreatic, gastric, ovarian, prostate cancers and melanoma), detailing the intratumoral microbial compositional heterogeneity within these malignancies. The present review analyzes the heterogeneous carcinogenic mechanisms mediated by these microorganisms, including gene instability, immune microenvironment remodeling and metabolic intervention. The diagnostic value of microbial markers in liquid biopsy and in situ tissue detection is explored, and the potential for synergistic strategies combining microbial-targeted therapy and immunotherapy is discussed. Finally, the review suggests future research directions, such as spatiotemporal dynamic analysis and organoid-microorganism co-culture, offering new insights for precise cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09bce21033da16d36e409d313fbe6c3d34814bec" target='_blank'>
              Tumor-associated microbiota: Multi-cancer landscape, mechanistic insights and clinical translation (Review)
              </a>
            </td>
          <td>
            Jindi Yang, Yige Dong, Yi Chen, Hong Liang, Shengyu Rong, Zhe Liu, Qiulei Lang
          </td>
          <td>2025-09-16</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75865e4fe743320d35269c6c38da63e48d3392a1" target='_blank'>
              Lineage plasticity and histological transformation: tumor histology as a spectrum.
              </a>
            </td>
          <td>
            Xiaoling Li, Eric E. Gardner, Sonia Molina-Pinelo, Clare Wilhelm, Ping Mu, Á. Quintanal-Villalonga
          </td>
          <td>2025-09-30</td>
          <td>Cell research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Glioblastoma, the most common and lethal primary brain tumor, creates profound immunosuppression that has resisted clinical interventions. The mechanisms behind immunotherapy failures in glioblastoma remain unclear. We conducted a systematic comparison of different immunotherapeutic approaches in mouse models to better understand treatment responses.



 Using multiomic single-cell analysis and spatial profiling, we evaluated GL261 glioma-bearing mice treated with anti-PD-L1 antibodies, NKG2D-CAR-expressing T cells, natural killer (NK) cells, or macrophages.



 All treatment models induced immune cell migration through chemotaxis. Additionally, PD-L1 and CAR T cell treatments led to an exhaustion-associated T cell signature alongside increased cytotoxic T cell presence. Further, we observed higher numbers of immunomodulatory lymphocytes and macrophages associated with these two immunotherapies. CAR macrophages and CAR NK cells exhibited enhanced cytotoxic gene signatures without increasing T cell numbers. Also, the immunomodulatory macrophage response appeared attenuated in CAR NK cells and macrophages. Notably, CAR NK cells demonstrated an enhanced antiviral response with interferon-beta expression that was not observed in other models.



 While PD-L1 and CAR cell treatments successfully polarized the lymphoid compartment toward cytotoxic phenotypes, these approaches alone appear insufficient as they were accompanied by pronounced immunomodulatory cell responses. CAR NK cells and macrophages were characterized by attenuated immunomodulatory responses. Our findings suggest that cytotoxic lymphocytes alone fail to override tumor-associated immune suppression. This is in line with recent evidence on T helper cell roles in CAR therapy. Future immunotherapy approaches should balance cytotoxic and helper lymphocyte populations while enhancing inflammatory macrophages to overcome the immunosuppressive environment of glioblastoma
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde65cb7289ab4952a15cc4d954df652b303b749" target='_blank'>
              P08.22.B SYSTEMATIC COMPARISON OF IMMUNOTHERAPIES IN GLIOBLASTOMA REVEALS DISTINCT IMMUNE RESPONSE PATTERNS
              </a>
            </td>
          <td>
            Y. Liu, T. Look, J Fang, A. Dalmau Gasull, M. Prinz, T. Weiss, R. Sankowski
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive solid malignancy with poor overall prognosis and limited response to standard treatments. Growing interest in the modulation of DNA repair mechanisms, including the homologous recombination (HR) repair pathway, has opened new avenues for therapeutic development. BARD1 (BRCA1-Associated RING Domain 1) plays a complex role in tumor biology, functioning either as a tumor suppressor or as an oncogenic driver, depending on isoform expression, cellular context, and regulatory environment. In this review, we examine the dual roles of BARD1, focusing on its regulation and paradoxical activities in PDAC. We summarize evidence that BARD1 and BARD1 isoforms differentially affect DNA repair, apoptosis, and drug resistance and evaluate the therapeutic potential of targeting BARD1 and other DNA damage response (DDR) proteins. We also review ongoing clinical trials and investigational agents designed to exploit DDR vulnerabilities in PDAC. Together, these insights highlight BARD1’s context-dependent roles in PDAC and support continued efforts to profile BARD1 isoforms, clarify their functions, and leverage DDR pathways through precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c070f9d1ec5a160a1dcacf90475e9772e358bc" target='_blank'>
              BARD1: A Friend or Foe in Pancreatic Ductal Adenocarcinoma?
              </a>
            </td>
          <td>
            Lily Zekavat, Aditi Jain
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b63ca88458c114713519c6871fd49fc0e1a4b977" target='_blank'>
              Mechanical confinement governs phenotypic plasticity in melanoma.
              </a>
            </td>
          <td>
            Miranda V. Hunter, Eshita Joshi, Sydney Bowker, Emily D. Montal, Yilun Ma, Young Hun Kim, Zhifan Yang, Laura Tuffery, Zhuoning Li, Eric Rosiek, A. Browning, Reuben Moncada, Itai Yanai, Helen M Byrne, Mara Monetti, Elisa de Stanchina, Pierre-Jacques Hamard, R.P. Koche, Richard M. White
          </td>
          <td>2025-08-27</td>
          <td>Nature</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Compared to sun-exposed melanomas, acral melanomas are genetically diverse and occur in areas with low sun exposure and high mechanical loads. During metastatic growth, melanomas invade from the epidermis to the dermis layers through dense tumor stroma and are exposed to fibrillar collagen architectures and mechanical stresses. However, the role of these signals during acral melanoma pathogenesis is not well understood. In this study, a novel 3D in vitro platform comprising heterogeneous, bundled collagen architectures recapitulates mechanical and architectural signals from the melanoma tumor microenvironment. YUSEEP patient-derived human acral melanoma and B16F10 mouse melanoma single cells and spheroids are embedded in collagen or bundled collagen hydrogels and mechanically compressed to quantitatively profile cellular responses to these cues, including viability, DNA damage and repair, proliferation, invasion, and nuclear and cellular morphologies. Spatial confinement of cells in a microfluidic platform, solid mechanics simulations, and pharmacological inhibition studies lend further mechanistic insights into these cues. Results reveal mechanical compression induces DNA damage and repair, while interactions with bundled collagen promote a malignant, protrusive phenotype. The findings further suggest that actin polymerization and contractility inhibitors may rescue DNA damage and mitigate malignancy upon compression, thereby potentially paving the way for novel therapeutic targets against acral melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a50277034a8bc6fee3365fef5db114e494ef3ed6" target='_blank'>
              Impact of Mechanical and Architectural Signals in the Tumor Microenvironment on Melanoma.
              </a>
            </td>
          <td>
            Z. M. Khan, Alejandro Rossello-Martinez, Michael Mak
          </td>
          <td>2025-09-07</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Surrogate selection is an experimental design that without sequencing any DNA can restrict a sample of cells to those carrying certain genomic mutations. In immunological disease studies, this design may provide a relatively easy approach to enrich a lymphocyte sample with cells relevant to the disease response because the emergence of neutral mutations associates with the proliferation history of clonal subpopulations. A statistical analysis of clonotype sizes provides a structured, quantitative perspective on this useful property of surrogate selection. Our model specification couples within-clonotype birth-death processes with an exchangeable model across clonotypes. Beyond enrichment questions about the surrogate selection design, our framework enables a study of sampling properties of elementary sample diversity statistics; it also points to new statistics that may usefully measure the burden of somatic genomic alterations associated with clonal expansion. We examine statistical properties of immunological samples governed by the coupled model specification, and we illustrate calculations in surrogate selection studies of melanoma and in single-cell genomic studies of T cell repertoires.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db201d19838bab76eccd97ad0f817ca6a6bc83d" target='_blank'>
              SURROGATE SELECTION OVERSAMPLES EXPANDED T CELL CLONOTYPES.
              </a>
            </td>
          <td>
            Peng Yu, Yumin Lian, Elliot Xie, Cindy L. Zuleger, Richard J Albertini, Mark R Albertini, Michael A. Newton
          </td>
          <td>2025-09-01</td>
          <td>The annals of applied statistics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Glioblastoma multiforme (GBM) is one of the most aggressive and lethal forms of brain cancer, characterized by cellular heterogeneity, poor prognosis and resistance to conventional therapies. One of the major challenges in treating GBM is its diffuse nature, which complicates surgical resection and contributes to high recurrence rates. Our research focuses on the role of biomolecular condensates - membraneless compartments formed by phase separation - in driving oncogenic processes. These condensates organize cellular biochemical processes by compartmentalizing proteins and nucleic acids, thereby regulating gene expression, signal transduction, and metabolism. We are particularly interested in the formation of aberrant condensates in glioblastoma cells and their contribution to malignant transformation, metabolic reprogramming and treatment resistance.



 Using patient-derived GBM models, CRISPR/Cas9 technology, super-resolution microscopy, and proteomic analyses, we are investigating the structural and functional properties of these condensates.



 Our goal is to determine how their dysregulation facilitates cancer progression and to identify novel vulnerabilities that could be therapeutically targeted.



 This research advances our understanding of the pathophysiological roles of biomolecular condensates in cancer and opens new avenues for innovative treatment strategies in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc0d78352f5d6b9af0c7f40eba7db59a9e99146d" target='_blank'>
              P01.03.A SPATIOTEMPORAL ORGANIZATION OF BIOMOLECULES IN GLIOBLASTOMA MULTIFORME CELLS BY PHASE SEPARATION
              </a>
            </td>
          <td>
            M. Sztacho, A. Chytła, C. Carrera Bravo, S. Abbasi Sani, E. Herencia Lagunar, R. Gönül
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 To identify novel drivers of pancreatic ductal adenocarcinoma (PDAC) malignancy, we employed a systems biology approach called regulatory network analysis, which accurately infers the activity of transcription factors and other regulatory proteins based on the integrated expression of their positive and negative target genes. This highly validated approach enables the identification of the most hyper-activated and hyper-repressed regulatory proteins (“master regulators”) that drive phenotypic distinctions. We applied this technique to a set of 200 laser capture microdissected human PDAC samples as well as 45 low-grade precursors for which we had matched histopathological, clinical, and epidemiological annotation. We identified the master regulators associated with 4 malignancy phenotypes: tumor initiation, tumor progression, survival post resection, and association with KRAS activity. Integrating across these phenotypes, the top master regulator was BMAL2, a member of the PAS family. Although BMAL2 is classically associated to the circadian rhythm, gene set enrichment analysis highlighted a potential role in hypoxia response, a key phenotype in PDAC evolution. We previously demonstrated that PDAC in humans and in the genetically engineered “KPC” mouse model is hypovascularized and hypoperfused, and we have since shown that KPC tumors are profoundly hypoxic. Sequence conservation analysis of BMAL2 demonstrated close analogy to hypoxia-related genes, its activity is induced in response to it and inhibited following treatment with multiple RAF, MEK, and ERK inhibitors, validating its computationally inferred association with RAS activity. Strikingly, knockout of BMAL2 in human PDAC cells led to defects in viability and invasion in the setting of hypoxia. Metabolically, loss of BMAL2 impairs the ability to induce glycolysis - key metabolic pathway unregulated during hypoxia response - upon exposure to severe hypoxia. Surprisingly, knockout of BMAL2 in multiple PDAC cell lines led to a complete loss of HIF1A stabilization in response to hypoxia. By contrast, HIF2A was further upregulated under hypoxia in the setting BMAL2 loss. Additionally, metabolomic studies showed an overrepresentation of glycolysis metabolites and related metabolic pathways in hypoxic control conditions but to a lesser degree in BMAL2 knockout cells. Finally, preliminary in vivo experiments in cell line derived xenograft models displayed lower tumor growth rates in BMAL2 knock out cells. Together, our data indicates that BMAL2 loss blunts hypoxia response providing also a potential answer to the long-standing question of how these proteins are differentially regulated. Certainly, a connection between RAS signaling and glycolysis has long been apparent, but our findings provide a novel mechanistic understanding of the relationship between RAS and hypoxic metabolism.



 Alvaro Curiel Garcia, Carlo H. Maurer, Sam Holmstrom, Cristina Castillo, Pasquale Laise, Carmine F. Palermo, Steven A. Sastra, Anthony Andren, Li Zhang, Tessa Le Large, Irina Sagalovsky, Daniel R. Ross, Vilma Rosario, Kate Lu, Ethan Ferraiuolo, Nicholas Spinosa, Winston Wong, Kaitlin Shaw, John A. Chabot, Jeanine Genkinger, Hanina Hibshoosh, Gulam A. Manji, Alina C. Iuga, Roland M. Schmid Schmid, Michael A. Badgley, Kristen C. Johnson, Andrea Califano, Yatrik M. Shah, Costas A. Lyssiotis, Kenneth P. Olive. Ras-dependent activation of BMAL2 regulates hypoxic metabolism in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A047.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b200421d854e96ea48e923482c3749c3e10676" target='_blank'>
              Abstract A047: Ras-dependent activation of BMAL2 regulates hypoxic metabolism in pancreatic cancer
              </a>
            </td>
          <td>
            Á. C. García, Carlo Maurer, Sam Holmstrom, Cristina Castillo, Pasquale Laise, Carmine F. Palermo, Stephen A. Sastra, A. Andren, Li Zhang, Tessa Y. Le Large, Irina Sagalovsky, Daniel R. Ross, V. Rosario, K. Lu, Ethan Ferraiuolo, Nicholas Spinosa, W. Wong, K. Shaw, John A. Chabot, Jeanine Genkinger, H. Hibshoosh, G. Manji, Alina C Iuga, Roland M. Schmid Schmid, Michael A. Badgley, , Andrea Califano, Yatrik M. Shah, C. Lyssiotis, K. Olive
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL) therapy is a promising approach, earning U.S. Food and Drug Administration approval in patients with anti-PD-1-resistant melanoma. Extending TIL therapy to patients with cholangiocarcinoma (CCA), an aggressive and largely immune-refractory cancer, is an emerging area of interest. However, cost and manufacturing complexity constrain clinical scalability of TIL therapy for CCA, underscoring the need for a murine model to optimize efficacy. Here, we established a novel orthotopic model of TIL therapy for CCA and tested a new ex vivo expansion strategy. We first characterized the immune landscape of orthotopic CCA and then compared 2 TIL expansion methods: (1) a conventional protocol using CD3 agonist stimulation (CD3 TILs) and (2) a tumor antigen-based protocol using irradiated autologous CCA cells to enrich for tumor-reactive TILs (Tumor Ag TILs). Tumor Ag TILs displayed superior tumor lysis in vitro compared to CD3 TILs. While both TIL products engrafted in vivo, Tumor Ag TILs showed enhanced persistence. Despite this, monotherapy with either TIL product alone had only a modest impact on tumor growth rate, and infused cells had upregulation of inhibitory checkpoint receptors, including PD-1. Further investigations demonstrated that the in vivo antitumor efficacy of both Tumor Ag TILs and CD3 TILs was enhanced when combined with PD-L1 inhibitor therapy. Altogether, our study establishes a preclinical platform for modeling CCA TIL therapy, identifies a rational combination strategy that potentiates TIL efficacy, and provides the field with a foundation to advance adoptive T-cell transfer development for CCA and related solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c94988fa5b70c12c198d5f39ec050ac02e9b35b8" target='_blank'>
              Development of a murine tumor-infiltrating lymphocyte therapy model for cholangiocarcinoma.
              </a>
            </td>
          <td>
            Megen C. Wittling, Frances J Bennett, Emilie A K Warren, K. M. Oppat, M. Wyatt, Jacklyn Hammons, Yuan Liu, Shishir K Maithel, Chrystal M. Paulos, Gregory B. Lesinski
          </td>
          <td>2025-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Background Metastatic melanoma is a highly aggressive disease with poor survival rates despite recent therapeutic advancements with immunotherapy. The proteomic landscape of advanced melanoma remains poorly understood, especially regarding proteomic heterogeneity across metastases within patients. Methods We collected 83 melanoma metastases from 19 different metastatic sites in 24 patients with advanced metastatic melanoma almost exclusively from the pre‐immunotherapy era, using semi‐rapid autopsies. The metastases were subjected to histopathological evaluation, RNA‐sequencing and mass spectrometry‐based proteomics for protein quantitation and non‐reference peptide (NRP) sequence detection using a proteogenomic data integration approach. Results NRPs associated with mutations frequently occurred in proteins related to focal adhesion, vesicle‐mediated transport, MAPK signalling and immune response pathways across the cohort. Intrapatient heterogeneity was negligible when considering morphology and driver gene mutation status but was substantial at the proteogenomic level. This heterogeneity was not driven by metastasis location, albeit liver metastases exhibited distinct proteogenomic patterns, including upregulation of metabolic pathways. Cluster analysis outlined four proteomic clusters (C1–4) of the metastases, characterised by the upregulation of cell cycle and RNA‐splicing (C1), mitochondrial processes (C3), extracellular matrix (ECM) and immune pathways (C2) and ECM and vesicle‐mediated transport pathways (C4). Around two‐thirds of patients had metastases that had strongly distinct phenotypes. Patients in our cohort whose metastases were primarily assigned to clusters C1 and C3 exhibited shorter overall survival than patients whose metastases were categorised mainly into the C2 and C4 clusters. Conclusion Our unique multi‐metastasis cohort captured the proteogenomic heterogeneity of immunotherapy‐naïve melanoma distant metastases, establishing a foundation for future studies aimed at identifying novel therapeutic targets to complement current immunotherapies. Key points Comprehensive proteogenomic profiling of post‐mortem melanoma metastases, collected primarily before the immunotherapy era. Description of 1177 protein sequence variants predicted by RNA‐Seq and validated via mass spectrometry‐based proteomics. Empirical evidence of prominent intrapatient heterogeneity, driven by heterogeneous protein expression related to cell cycle‐ and mitochondrial processes, immune system and extracellular matrix organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c47896dcec6e305a1baa3208d22fb00cf053050b" target='_blank'>
              Distant metastases of melanoma exhibit varying extent of intrapatient proteogenomic heterogeneity
              </a>
            </td>
          <td>
            B. Szeitz, Y. Hagemeijer, Zoltán G Páhi, Zsuzsanna Ujfaludi, M. Kuras, Jimmy Rodriguez, V. Doma, R. Mohácsi, Magdolna Herold, Zoltán Herold, Z. Horváth, I. Pla, Y. Sugihara, Bo Baldetorp, H. Lindberg, Henriett Oskolás, M. Rezeli, Jeovanis Gil, R. Appelqvist, L. Kemény, Jéssica Guedes, Johan Malm, Aniel Sanchez, Imre M Boros, I. Németh, Victor Guryev, Tibor Pankotai, Krzysztof Pawłowski, E. Wieslander, A. M. Szasz, David Fenyö, Peter Horvatovich, József Timár, Gyorgy Marko-Varga, L. Betancourt
          </td>
          <td>2025-09-28</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Confident identification of human leukocyte antigen (HLA)-presented peptides is crucial for advancing cancer immunotherapy. We present MHCquant2, a scalable and modular Nextflow pipeline integrated into nf-core as a reproducible, portable, and standardized workflow for immunopeptidomics. This integration allows a community-driven, robust solution for high-throughput analyses across operating systems and cloud infrastructures. MHCquant2 integrates open-source tools including OpenMS, DeepLC, and MS2PIP, improving peptide identifications by up to 27% across diverse MS platforms, particularly enriching low-abundant peptides. MHCquant2 demonstrates state-of-the-art performance on our novel benignMHCquant2 dataset (n = 92) and expands the benign human immunopeptidome by over 160,000 unique naturally presented HLA peptides. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03763-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e678695db6815730d0674dcf746e931d4d6d4de2" target='_blank'>
              MHCquant2 refines immunopeptidomics tumor antigen discovery
              </a>
            </td>
          <td>
            Jonas Scheid, Steffen Lemke, Naomi Hoenisch-Gravel, Anna Dengler, Timo Sachsenberg, Arthur Declerq, R. Gabriels, J. Bauer, Marcel Wacker, Leon Bichmann, Lennart Martens, M. Dubbelaar, Sven Nahnsen, J. Walz
          </td>
          <td>2025-09-22</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>29</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [12],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>