<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-08 09:37:18 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>103</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/629f37baf89cd9a5cc6f98228b4de5cf8f891b4c" target='_blank'>
              ROSIE-Enabled Spatial Mapping Reveals Architectural Fragmentation and Immune Reprogramming in Lung Adenocarcinoma Evolution
              </a>
            </td>
          <td>
            Beibei Huang, Bo Zhu
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d2c494a1cd657524f51f9577b21d3baa396ea7" target='_blank'>
              Spatiotemporal regulation of cell cycle states within the complex tumor microenvironment
              </a>
            </td>
          <td>
            Gianni Zanardelli, Olivier Tassy, Maulik K. Nariya, Nacho Molina
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, Ivan Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Spatial transcriptomics has emerged as a transformative technology in biomedical research, offering unprecedented insights into gene and protein expression within their native tissue context. Unlike conventional bulk or single-cell sequencing approaches, spatial omics has the advantage of preserving the spatial structure of tissues, allowing researchers to directly map molecular information onto histological structures. This review provides an overview of the current state of spatial omics technologies, highlighting their application in cancer research. Spatial omics has enabled detailed characterization of the tumor microenvironment (TME), revealing spatial heterogeneity, immune cell infiltration patterns, and complex mechanisms of tumor progression and therapy resistance across various cancer types. The review covers future directions, including artificial intelligence–driven analytics, improved standardization, and cost reduction to accelerate clinical translation. Ultimately, spatial omics is poised to play a central role in precision oncology, enabling a deeper understanding of tumor biology and informing more effective individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a77ce136fd5fd93d240b66bc2895fcb841db1de" target='_blank'>
              Spatial omics: applications and utility in profiling the tumor microenvironment
              </a>
            </td>
          <td>
            Ji-Eun See, Sarah Barlow, Wani Arjumand, Hannah DuBose, Felipe Segato Dezem, Jasmine T. Plummer
          </td>
          <td>2025-12-01</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Protein spatial imaging, rooted in spatial proteomics, enables the localization of proteins within their native environment, thus preserving structural and functional context. Its integration into cancer research has provided insights into tumor progression, especially in breast cancer, where ductal carcinoma in situ (DCIS) may transition into invasive breast cancer (IBC). Multiplexed ion beam imaging by time-of-flight (MIBI-TOF) has emerged as a powerful method to visualize protein distribution, tumor–stroma interactions, and immune cell dynamics. Evidence indicates that the tumor microenvironment (TME)—including stromal architecture, extracellular matrix (ECM), and immune cell infiltration—plays a central role in malignant progression. Here, we highlight recent advances in protein spatial imaging, discuss optimization strategies, and examine its implications for breast cancer progression and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a7eef8cc6668ecc3d6de49f721002ee2756192" target='_blank'>
              Spatial Proteomics Imaging to Decode Disease Progression: A Perspective on Tumor Microenvironments
              </a>
            </td>
          <td>
            Haonan Zhang
          </td>
          <td>2025-11-30</td>
          <td>Clinical Medicine and Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f7dcdd3510de0385797859e77f339a9cbb0192e" target='_blank'>
              Multidimensional Single-Cell Transcriptomic Profiling of Uterine Leiomyosarcomas Identifies Molecular Subtypes with Distinct Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Maria Korah, J. Agolia, Biren Reddy, R. Flojo, Amanda Gonçalves, Lilin Wang, Wesley Bobst, Brooke Liang, Kaylin A. Yip, Rosyli Reveron-Thornton, Chuner Guo, Robert Gruener, Andrea E. Delitto, Peter Y. Xie, Angela D. Tabora, Beatrice J. Sun, Sara K. Daniel, Babak Litkouhi, Oliver Dorigo, Nam Q. Bui, Minggui Pan, E. Moding, A. Kalbasi, Amanda R. Kirane, George A. Poultsides, Jeffrey A. Norton, Aaron M. Newman, G. Charville, K. Ganjoo, Deshka S. Foster, R. Stephanie Huang, Byrne Lee, Michael T. Longaker, Daniel J. Delitto
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/235c8c18e964369ee970e129fd3fe5bd3b1a1aa6" target='_blank'>
              In silico reconstruction of primary and metastatic tumor architecture using GIS-augmented spatial transcriptomics
              </a>
            </td>
          <td>
            Jin Young Yoo, Sabrina Akter, Qianying Zuo, Amir Kazemi, Wenjun Wu, Mahima Goel, Audrey Lam, Debapriya Dutta, Betsy Barnick, M. Pekmezci, Maria Grosse Perdekamp, Aiman Soliman, Zeynep Madak-Erdogan
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Integration of scRNA-seq data from millions of cells revealed a high diversity of cell types in the healthy and diseased human lung. In a large and complex organ, constantly exposed to external agents, it is crucial to understand the influence of lung tissue topography or external factors on gene expression variability within cell types. Here, we apply three spatial transcriptomics approaches, to: (i) localize the majority of lung cell types, including rare epithelial cells within the tissue topography, (ii) describe consistent anatomical and regional gene expression variability within and across cell types, and (iii) reveal distinct cellular neighborhoods in specific anatomical regions and examine gene expression variations in them. We thus provide a spatially resolved tissue reference atlas in three representative regions of the healthy human lung. We further demonstrate its utility by defining previously unknown imbalances of epithelial cell type compositions in chronic obstructive pulmonary disease lungs. Our topographic atlas enables a precise description of characteristic regional cellular responses upon experimental perturbations or during disease progression. Using multiplexed spatially resolved transcriptomic approaches, the authors generate a topographic atlas of the healthy adult lung with location-related gene expression variability or neighbourhood changes as exemplified by COPD patient sample analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe80d855be30beaf44c2810f98505d59172a3d0" target='_blank'>
              Spatial single-cell atlas reveals regional variations in healthy and diseased human lung
              </a>
            </td>
          <td>
            A. Firsova, Sergio Marco Salas, L. B. Kuemmerle, X. Abalo, A. Sountoulidis, L. Larsson, Krishnaa T. Mahbubani, Jonas Theelke, Žaneta Andrusivová, Leire Alonso Galicia, A. Liontos, Tamás Balassa, Ferenc Kovács, Peter Horvath, Yuexin Chen, Janine Gote-Schniering, M. Stoleriu, Jürgen Behr, Kerstin B. Meyer, W. Timens, Herbert B. Schiller, Malte D. Luecken, F. Theis, J. Lundeberg, Mats Nilsson, M. Nawijn, Christos Samakovlis
          </td>
          <td>2025-11-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Melanoma plasticity drives immune evasion and therapy resistance through dynamic cell-state transitions beyond genetic alterations. Epigenetic remodeling critically influences such processes, yet its role in reshaping the tumor ecosystem under therapeutic pressure remains unresolved. Here, we profiled longitudinal biopsies from melanoma patients treated in the phase Ib NIBIT-M4 epi-immunotherapy clinical trial (NCT02608437), testing the combination of a DNMT1 inhibitor with anti-CTLA4 using single-cell multiome and high-resolution spatial transcriptomics. Integrated analyses resolved seven malignant meta-programs, including a rare Wnt/β-catenin–driven melanocytic state and a de-differentiated neural crest–like state enriched in non-responders. Spatial modeling revealed that homotypic clustering stabilizes resistant programs, with neural crest–like cells forming compact, centrally localized niches, whereas Wnt/β-catenin subpopulations displayed a bimodal architecture, either cohesive clusters sustained by adhesion or dispersed, transcriptionally plastic cells. Responders exhibited progressive enrichment of an antigen presentation/interferon program and coordinated remodeling of the tumor microenvironment with T and B cell expansion, whereas tumors from non-responder patients maintained stable composition of neural crest–like clusters. Epigenetic therapy reactivated transposable elements, providing both regulatory signals that prime innate immunity within microenvironment and generating antigens that drive immunoediting and immunogenicity of Antigen presentation/interferon cell states in responders. Finally, NFATC2 emerged as a master regulator of neural crests–like transcriptional phenotypes and promoter of resistance to therapeutic interventions in melanoma patients. NFATC2 perturbation was able to shift tumor cells towards more differentiated and immunogenic states. These findings reveal how epigenetic-based immunotherapy reshapes melanoma ecosystems, provide mechanistic insights into how multiple transcriptional programs promote tumor plasticity and resistance to both combinatorial therapies and immune checkpoint blockade, identify spatial clustering as a principle stabilizing resistant niches, and highlight β-catenin and NFATC2 as actionable vulnerabilities to overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9320984fb7ca80afa8b2d49d724dd86390a378" target='_blank'>
              Transposable Elements and Homotypic Niches Drive Immune Dynamics and Resistance in Melanoma Epigenetic-based immunotherapy
              </a>
            </td>
          <td>
            Erika Ciervo, Francesco Ceccarelli, A. D. Di Giacomo, Piera Grisolia, A. Covre, Z. M. Besharat, Antonio De Falco, F. Caruso, Luigi Laezza, Luigi Ferraro, Gloria Mas Martin, M. F. Lofiego, Tommaso Sani, E. Ferretti, Yan Guo, Sean B. Holden, R. Mortarini, A. Anichini, M. Maio, T. Noviello, M. Ceccarelli
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7c241dbb9b8047ecee9d971f81cea0a1e193e3" target='_blank'>
              Multi-omic profiling provides insights into the heterogeneity, microenvironmental features, and biomarker landscape of small-cell lung cancer.
              </a>
            </td>
          <td>
            Mingchao Xie, Miljenka Vuko, Shashank Saran, Siyu Liu, Andrew G Chambers, Hana Baakza, Helen K. Angell, Felicia Ng, C. Gay, R. Cardnell, F. Segerer, Alma Andoni, Jaime Rodriguez-Canales, Paul M Waring, Markus Schick, J. C. Barrett, L. Byers, Giulia Fabbri
          </td>
          <td>2025-12-02</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf21fbcae37403e9935121738032d46b9b3bab3c" target='_blank'>
              ProMPt: A modular preclinical platform for functional modelling of prostate cancer heterogeneity and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Nicole Pandell, Jacob Househam, Matteo Tartagni, Archana Thankamony, Florian Gabel, Roberto Rota, Elizabeth Flittner, B. Gurel, I. Figueiredo, G. Seed, A. Neeb, Mary Chol, Wei Yuan, J. Clohessy, C. Tape, A. Sharp, J. D. De Bono, Marco Bezzi
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes TP53 and RB1. However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yueli Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a891d2740ad2db98b44c1bca0f35faa07a82811" target='_blank'>
              Spatial and Single-Cell Transcriptomics Decipher the Crosstalk Environment of DEFB1+ Cancer Cells and IFI30+ Macrophages in Intrahepatic Cholangiocarcinoma
              </a>
            </td>
          <td>
            Guoliang Wang, Hang Meng, Meng Li, Xiangdong Fang, Weilong Zou, H. Qu
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern, and a high relapse rate. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, where malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, we identify pericytes as the most active paracrine signaling hub within the tumor parenchyma. Their depletion through genetic engineering results in accelerated tumor progression and shortened survival. Mechanistic studies reveal that pericyte deficiency remodels the endothelium and impacts the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, the pericyte-deprived endothelium recruits perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages express Hepatocyte Growth Factor, which reinforces activation of its receptor tyrosine kinase MET on GBM cells harboring a pronounced mesenchymal subtype driven by the key phenotypic regulator Fosl1. Indeed, orthotopic implantation of MET-expressing GBM cells corroborates their superior tumor-initiating and invasive capabilities. Thus, pericytes represent critical modulators of GBM development by orchestrating a tumor-suppressive microenvironment, highlighting the importance of their preservation in therapy. Improved understanding of the tumor ecosystem in glioblastoma is critical for developing new treatment strategies for the disease. Here, the authors identify pericytes as an active paracrine signaling hub within the tumor parenchyma that orchestrates a tumor-suppressive microenvironment, suggesting pericyte preservation as a key feature of future therapeutic regimens for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c09a05790107e087a632df06714ceec3176e20b4" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, Mehrnaz Safaee Talkhoncheh, B. Phung, E. Cordero, R. Rosberg, E. Johansson, G. Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5df09ad47e15df1d3b2fa365bedcbec8d739cdeb" target='_blank'>
              Unveiling Oligodendrocyte-Lineage Differentiation in the Glioblastoma Infiltrative Zone Through Spatial Transcriptomics
              </a>
            </td>
          <td>
            C. M. Argento, Enrico Sebastiani, E. F. Rosatti, D. Sighel, S. Longhi, A. Soldano, T. Tebaldi, Alessandro Quattrone, F. Corsini, Luciano Annicchiarico, M. Barbareschi, L. Pecciarini, P. L. Poliani
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The human somatic genome evolves throughout our lifespan, producing mosaic individuals comprising clones harboring different mutations across tissues. While clonal expansions in the hematopoietic system have been extensively characterized and reported to be nearly ubiquitous, clonal mosaicism (CM) has more recently also been described across multiple solid tissues. However, outstanding questions remain about the parameters and processes of human somatic evolution in non-cancerous solid human tissues, including when clones arise, how they evolve over time, and what mechanisms lead to their expansion. Questions of timing and clonal dynamics can be addressed through phylogenetic reconstruction, which serves as a ‘temporal microscope’, while uncovering the mechanisms of expansion necessitates simultaneous phenotypic profiling. To address this gap, here we develop Single-cell Miniaturized Automated Reverse Transcription and Primary Template-directed Amplification (SMART-PTA) for joint single-cell whole-genome and whole-transcriptome sequencing for large scale and cost efficient interrogation of solid tissue CM. We established a workflow that generates hundreds of matched single-cell whole genome and transcriptome libraries within a week. We profiled phenotypically normal esophagus tissue from four aged donors and used somatic variants to build high-resolution single-cell lineages from >2,700 cells with accompanying transcriptomic information, reconstructing >70 years of somatic evolution. T cell expansions identified from T cell receptor (TCR) sequences validated the clonal structure of the single-nucleotide variant (SNV)-based phylogenies and phylogenetic cross-correlation analysis showed that epithelial cells had higher degrees of shared ancestry by spatial location compared to immune cells. Mapping mutation signatures to the phylogenetic tree revealed the emergence of tobacco/alcohol exposure-related signatures later in life, consistent with the donors’ exposure histories. We identified variants in driver genes that were previously reported in the phenotypically normal esophagus, detecting clonal expansions harboring mutations in genes including TP53 and FAT1. We mapped the evolution of clones with both monoallelic as well as biallelic TP53 loss, including a clone associated with high expression of cell cycling genes and higher chromosome instability. Leveraging the matched transcriptome data, we uncovered cell type biases in mutant clones, with a higher proportion of TP53 or FAT1-mutant cells in an earlier basal epithelial cell state compared to wild-type cells. We further observed copy-neutral loss of heterozygosity (CNLOH) events on chromosome 9q that spanned the NOTCH1 locus in up to ∼35% of epithelial cells. Mapping CNLOH events to the phylogenetic tree revealed a striking pattern in which CNLOH was separately acquired many times, reflecting convergent evolution. Cells with CNLOH events were biased towards the earlier basal epithelial state, suggestive of a selective advantage that leads to prevalent recurrence of chr9q CNLOH. Together, we demonstrate that SMART-PTA is an efficient, scalable approach for single-cell whole-genome and whole-transcriptome profiling to build phenotypically annotated single-cell phylogenies with enough throughput and power for application to normal tissue somatic evolution. Moreover, we reconstruct the evolutionary history of the esophageal epithelium at high scale and resolution, providing a window into the dynamics and processes that shape clonal expansions in phenotypically normal tissues throughout a lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a93b5a70ed710c9aacd9cdfba3788cae5ad699" target='_blank'>
              Large-scale single-cell phylogenetic mapping of clonal evolution in the human aging esophagus
              </a>
            </td>
          <td>
            Tamara Prieto, Dennis J. Yuan, John Zinno, Clayton Hughes, Nicholas Midler, Sheng Kao, Jani Huuhtanen, Ramya Raviram, Foteini Fotopoulou, Neil Ruthen, Srinivas Rajagopalan, Joshua S. Schiffman, Andrew R. D’Avino, Sang-Ho Yoon, Jesus Sotelo, Nathaniel D. Omans, Noelle Wheeler, Alejandro Garces, Barun Pradhan, A. P. Cheng, N. Robine, Catherine Potenski, Katharine Godfrey, N. Kakiuchi, Akira Yokoyama, Seishi Ogawa, Julian A. Abrams, Ivan Raimondi, D. Landau
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2a93ffa8be7e0912bd5d9f29072cbe15c529994" target='_blank'>
              Inferring cellular communication through mapping cells in space using Tangram 2
              </a>
            </td>
          <td>
            Hejin Huang, Alma Andersson, Sunny Z. Wu, Gabriele Scalia, Jenna Collier, K. H. Hoi, Sören Müller, Graham Heimberg, Hector Corrada-Bravo, Shreya Gaddam, Jan-Christian Huetter, Shannon J. Turley, David Richmond, Aicha Bentaieb, Tommaso Biancalani
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 A major barrier in understanding adult diffuse gliomas lies in the inability to track tumor evolution in real time. To address this, we employed a novel method known as continuous lineage tracing, which integrates CRISPR/Cas9-based expressed DNA barcoding with single-cell RNA sequencing, enabling a phylogenetic approach to studying tumor development. Patient-derived glioma-initiating cell lines were engineered with expressed barcodes targeted by CRISPR/Cas9 and implanted into mice to create intracranial xenografts. Tumors underwent single-cell RNA sequencing; expressed barcodes were used to infer clonal relationships, and transcriptomic profiles enabled cell state classification. Phylogenetic lineage trees were reconstructed using lineage inference algorithms to characterize cell fitness, expansion, and plasticity. Our analysis uncovered extensive intra-clonal cell state heterogeneity, indicating active phenotype switching prior to therapy. We identified consistent transcriptional programs associated with tumor engraftment and in vivo clonal advantage. Lineage tracing revealed gene expression signatures linked to key phenotypes: fitness, enriched for neural-mesenchymal and injury-response pathways; expansion, associated with RNA splicing; and plasticity, correlated with cell cycle and DNA repair programs. Glioma stem cells appeared to span a transcriptional continuum from undifferentiated, high-fitness states to more differentiated, low-fitness states, with high-fitness cells potentially representing transitional phenotypes. We then validated these findings in a cohort of 185 surgically resected gliomas with matched bulk RNA and proteomic data. Phylogenetic gene signatures differed markedly between IDH-mutant and IDH-wildtype tumors. When stratified by tumor type, both fitness and expansion signatures were significantly associated with overall survival in GBM (median 12.6 months, p=0.041) and oligodendrogliomas (median 66 months, p=0.027). This study demonstrates the utility of continuous lineage tracing to reconstruct tumor evolution and identify transcriptional programs linked to tumor growth and prognosis. Our approach provides a powerful framework for dissecting glioma biology and identifying potential therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d67c2559a40c369b3a39bc8fe0aad62196164a" target='_blank'>
              STEM-24. SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS, EXPANSION AND PLASTICITY AXES IN ADULT DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Ajisebutu, Xiaojun Fan, Chloe Gui, Jeff Liu, Vikas Patil, A. Landry, Y. Ellenbogen, L. Yefet, Phooja Persaud, F. Nassiri, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor heterogeneity drives drug resistance and relapse, influencing immune evasion and tumor progression. While intratumor heterogeneity has been extensively studied at the genomic level, its functional outcomes and interactions with the tumor microenvironment remain underexplored. In contrast, the functional outcome of heterogeneity and the interplay with the tumor microenvironment have not been addressed. In this study, we integrate multi-region spatial MS-based proteomics of 280 tumor regions, exome sequencing, and imaging to investigate spatial proteomic heterogeneity in breast cancer. Our findings reveal increased proteomic heterogeneity with tumor progression, independent of genomic heterogeneity but closely associated with microenvironmental differences. Integration with immune and stromal imaging highlighted a dynamic interplay where low-grade tumors exhibit constrained immune infiltration, and upon progression to higher grades, macrophages and T cells infiltrate. However, anti-inflammatory pathways involving kynurenine and prostaglandins are more highly expressed in infiltrated regions, suggesting that anti-tumorigenic activities are inhibited. Integration with the global protein network provides potential targetable mediators of immune evasion in breast cancer that can serve as the basis for future development of personalized breast cancer therapies. Intratumor heterogeneity (ITH) has been mainly studied at the genomic level. Here, the authors integrate multi-region proteomics of 280 tumor regions from 33 patients, exome sequencing, and imaging-based immune and stromal profiling to investigate functional spatial proteomic ITH in breast cancer and the interactions of cancer cells and the tumor microenvironment during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d48d5d89f33d4d9425895ba820536d951749d4" target='_blank'>
              Integrated spatial proteomic analysis of breast cancer heterogeneity unravels cancer cell phenotypic plasticity
              </a>
            </td>
          <td>
            M. Mardamshina, Shiri Karagach, Vishnu Mohan, Gali Arad, Daniela Necula, O. Golani, Liat Fellus-Alyagor, Anjana Shenoy, Kateryna Krol, Daniel Pirak, Nitay Itzhacky, I. Marin, Bruria Shalmon, Y. Addadi, Roded Sharan, E. Gal-Yam, Iris Barshack, Tamar Geiger
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and lethal primary malignant tumor of the central nervous system. Advances in therapy are hindered by the complex intratumoral heterogeneity of GBM, where distinct malignant and non-malignant cellular states and interactions exist in spatially defined niches of the tumor microenvironment (TME), shaping both tumor behavior and treatment response. In this work, we define GBM biological reprogramming, TME recomposition, and cell-cell interactions in relation to spatially well defined Ivy Glioblastoma Atlas Project regions. Further, we apply a novel spatially informed approach to integrate biological reprogramming, as identified through gene co-expression network modules, with specific changes in cell-cell communications. Our results validate prior findings and offer novel insights into the spatial patterning of transcription factor regulation, cellular interactions, and biological pathway activity, in addition to informing rational combination therapies targeting spatial niche specific vulnerabilities. In addition, our work contributes a novel Visium spatial transcriptomics dataset of 14 samples for use by the GBM research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719b808c693c9788562c4b889cdeaf7fc68e251b" target='_blank'>
              Spatial transcriptomics of glioblastoma defines biologically and clinically significant reprogramming patterns across unique spatial microenvironments
              </a>
            </td>
          <td>
            V. Ravikumar, A. Maddox, Reva Kulkarni, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Merkel cell carcinoma (MCC) is an aggressive skin cancer with neuroendocrine differentiation marked by high cellular plasticity, often manifesting as rapid therapy resistance. Although the cell-of-origin is presumed to be epithelial, epidermal localization of MCC is rarely observed, largely because in situ MCC is typically an incidental finding. Nevertheless, a subset of MCC tumors exhibits epidermotropism, wherein tumor cells are present in the epidermis. The behavior of cancer cells is profoundly influenced by the tumor microenvironment and interactions with neighboring cells. Notably, the normal counterparts of the cancer’s cell-of-origin have been shown to attenuate tumor aggressiveness. Thus, epidermotropic MCC presents a unique opportunity to explore the potential role of epidermal microenvironment in modulating tumor cell behavior. While the epidermotropic tumor nests share histological resemblance with their dermal counterparts, their transcriptomic profiles remain unexplored. Here, we employed high-definition spatial and single-cell transcriptomics to dissect the gene expression profiles of epidermotropic MCC cells, comparing them to MCC cells in the tumor core and those adjacent to blood vessels. Notably, epidermotropic MCC cells exhibit a transcriptomic signature reminiscent of cutaneous squamous cell carcinoma, characterized by upregulation of genes encoding keratins, S100A proteins, as well as calmodulin-like proteins 3 and 5. Mechanistically, this keratinocytic differentiation is associated with enhanced p63 activity, leading to the upregulation of PERP. Collectively, our study demonstrates that MCC cells can adopt a keratinocytic differentiation program in response to microenvironmental cues, underscoring the remarkable phenotypic plasticity of this malignancy and the importance of the microenvironment for tumor cell characteristics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4113c46dcadcc5e605fb92cfb97a958b41a5f2" target='_blank'>
              High-resolution spatial transcriptomics uncover epidermal-dermal divergences in Merkel cell carcinoma: spatial context reshapes the gene expression landscape
              </a>
            </td>
          <td>
            Kuan Cheok Lei, Nalini Srinivas, Mitalee Chandra, Vahan Serobyan, Selma Ugurel, Daniel Hoffmann, T. Kervarrec, Weng-Onn Lui, Jürgen C. Becker
          </td>
          <td>2025-10-23</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="ecDNA contributes to cancer genetic heterogeneity through random segregation during mitosis. Emerging evidence links ecDNA to immune evasion, but the mechanism remains elusive. Using genetically engineered mouse models of pancreatic ductal adenocarcinoma (PDAC), we show that Kras and Myc oncogenes are amplified either on ecDNAs or as homogeneously staining regions (HSRs) on chromosomes. ecDNA-driven tumors are more aggressive in immunocompetent mice. Single-cell transcriptomic and histological analyses reveal that ecDNA-driven tumors rapidly establish an immunoevasive tumor microenvironment (TME), marked by increased myofibroblastic cancer-associated fibroblasts (myCAFs) and reduced T cell infiltration. Mechanistically, ecDNA heterogeneity generates a subset of cancer cells with extremely high Kras expression, termed super-expressors, which secrete amphiregulin to promote myCAF expansion and suppress T cell infiltration. Clonally organized super-expressors establish an immunoevasive niche in the TME from patients with PDAC. Our findings demonstrate a causal role of ecDNA in TME remodeling, offering insights into cancer heterogeneity and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e61660d8b517da35a1b97a2f0dd25e3588ab32" target='_blank'>
              ecDNA-driven oncogene super-expressors shape immunoevasive tumor microenvironment
              </a>
            </td>
          <td>
            Kailiang Qiao, Qing-Lin Yang, Tuo Li, Xiongfeng Chen, Zeynep Yazgan, Yoon Jung Kim, C. Gilbreath, Jun Yi Stanley Lim, Yipeng Xie, Xiaohui Sun, Yang Liu, Yiyue Jia, Zhijian J. Chen, Huocong Huang, Sihan Wu
          </td>
          <td>2025-11-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bdb3fc96051ec080fed6bcf88bde02391e145c9" target='_blank'>
              Integrative spatial multi-omics reveals prognostic tumor niches in female genital tumors
              </a>
            </td>
          <td>
            Tian Xu, Siyu Lin, Yuling Wu, Yufeng He, Zongxu Zhang, Yanping Zhao, Peng Zhang, Luyang Zhao, Wen Yang, M. Ye, Rui Zhang, Liang Wen, Pengfei Ren, Ce Luo, Zhe Zhang, Zexian Zeng, Yuanguang Meng
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fa5a90466cca365ac23cb5111624676a9f4bea9" target='_blank'>
              Deciphering Cellular Ecosystems Driving Tumor Progression and Immune Escape from Spatial Transcriptomics and Single-Cell with COMPOTES
              </a>
            </td>
          <td>
            L. Herpin, Anaïs Chossegros, Roberta Codato, Josep Monserrat Sanchez, Jean El Khoury, Simon Grouard, V. Ducret, Alex Cornish, Baptiste Gross, Elodie Pronier, C. Hoffmann, Alberto Romagnoni, Eric Y. Durand, Almudena Espin Perez, Quentin Bayard
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0d7b578f97dbff41a41bb90de3b031320cfe41" target='_blank'>
              Unravelling drug resistant proteotypes through phenotype-resolved proteomics of single-cell derived colonies
              </a>
            </td>
          <td>
            Di Qin, Ridhima Das, S. Schallenberg, K. Riecken, I. Tinhofer, Fabian Coscia
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Spatial context is a fundamental driver of intra-tumor heterogeneity in glioblastoma (GBM), shaping the molecular landscape, cellular composition, and therapeutic vulnerabilities. While single-cell profiling has illuminated the diversity of GBM cell states, a critical gap remains in understanding how local microenvironments orchestrate gene expression programs and cellular plasticity within intact tumors. To address this, we analyzed spatial transcriptomics data from 44 GBM specimens, comprising over 128,000 tissue spots, to systematically map the biological programs and cellular architecture of distinct tumor niches. Leveraging machine learning, we derived a unified batch-adjusted data embedding that captured the local tissue context of each spot across all slides, allowing the study of spatially continuous rewiring of molecular programs. This approach enabled us to identify ligand–receptor interactions and transcription factors (TF) with spatially localized activity across tissue slides. Using cross-correlation metrics, we identified multiple TF modules specifically active in certain environments. Notably, the perivascular region exhibited the most diverse transcriptional activity, with multiple TF modules associated with tumor growth, division, and migration processes. Additional modules were uniquely enriched at the leading edge and hypoxic regions, as well as at the tumor–necrosis interface. Importantly, we identified a unique module active at the tumor core–leading edge interface, with corresponding spots enriched for alternative splicing processes, suggesting this could be a core driver of the invasive tumor phenotype. Dynamic gene network inference along a continuous hypoxic gradient uncovered context-specific hub genes and TFs that facilitate tumor metabolic adaptation and cell state shifts. These findings underscore the immense spatial heterogeneity of GBM, characterized by context-specific molecular rewiring of tumor cells driven by complex cellular interactions. Our study not only reveals new vulnerabilities but also highlights the need for spatially targeted combination therapies to disrupt the adaptive spectrum of cancer cell states in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fddeac5d22fe2de68ff3fa47ac318949e6c8ee0" target='_blank'>
              TMIC-118. Identifying spatially-localized biological programs and molecular vulnerabilities in glioblastomas
              </a>
            </td>
          <td>
            V. Ravikumar, Reva Kulkarni, A. Maddox, Arushi Tripathy, Emre Kiziltug, Dah-Luen Huang, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cervical cancer ranks as the fourth most common malignancy among women worldwide, with approximately 604,000 new cases and 342,000 deaths annually. Despite advances in HPV vaccination and screening programs, significant challenges remain in diagnostic accuracy, therapeutic efficacy, and prognostic assessment. Traditional bulk RNA sequencing methods average gene expression data across entire tissue specimens, masking cellular diversity and spatial architectural patterns that influence tumor progression. Telomere maintenance mechanisms play crucial roles in cervical cancer development, with over 90% of cervical cancers exhibiting telomerase reactivation. This study integrated single-cell RNA sequencing and spatial transcriptomics technologies to construct the first comprehensive spatial molecular atlas of cervical cancer. Single-cell RNA sequencing analysis utilized public datasets from the GEO database with systematic quality control through a multidimensional assessment framework. Spatial transcriptomics analysis employed the 10x Genomics Visium platform, successfully identifying 38 distinct cellular neighborhoods (N1-N38). Spatial expression patterns of 10 key genes were analyzed, including epithelial markers (MUC1, CDH1, KRT16), stromal markers (COL1A1, COMP, DCN), and immune markers (CD3G, FCGR1A). In vitro validation was performed using HeLa and SiHa cell lines. Multiple distinct cell subpopulations and 38 cellular neighborhoods with unique molecular characteristics were successfully identified. MKI67 demonstrated spatial heterogeneity with proliferative “hotspots”; COMP was primarily expressed in stromal regions and participated in tumor-stroma interactions; KRT16 exhibited patterns reflecting epithelial differentiation gradients. Different neighborhoods showed enrichment for various cell types, including immune hotspots, stromal-rich regions, and epithelial-dominant areas. Immunoglobulin-related genes (IGLC2, IGHG1, IGHG2, etc.) displayed unique spatial expression characteristics. Quantitative PCR validation confirmed differential expression patterns between cell lines. This study established the first comprehensive spatial transcriptomic atlas of cervical cancer, revealing unprecedented insights into tumor microenvironment organization and cellular spatial relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a6a98dd06253f9d1bf4fd4787fb52f0b0c3d2f1" target='_blank'>
              Spatial transcriptomic landscape and cellular neighborhood heterogeneity in cervical cancer: integrative single-cell and spatial RNA sequencing analysis
              </a>
            </td>
          <td>
            Dieyi Mo, Huana Cao, Shaojuan Lai, Jinding Luo, Weijie Ye
          </td>
          <td>2025-11-05</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Immune checkpoint inhibitors (ICI) have demonstrated prolonged efficacy in certain melanoma patients, yet a significant portion of patients do not experience clinical improvement, with the mechanisms underlying this resistance still not fully understood. Using established cell markers, we partitioned the single‐cell transcriptome into clusters, finding a notable link between NK cells and patient response to immunotherapy. We further identified four distinct subpopulations of NK cells, profiling marker gene sets and unique biological functions associated with each subpopulation. This analysis provides insights into the trajectory of NK cell development and differentiation, along with identifying the transcription factors driving these processes. The study pinpointed NK cluster 01 as pivotal in influencing patient sensitivity and prognosis during immunotherapy. Single‐cell transcriptome and spatial transcriptomics (ST) analysis revealed the proximity of NK cluster 01 cells to melanoma cells, hinting at a potential regulation of cell‐cell interaction via the IFN‐II signaling pathway network. ST analysis revealed the spatial arrangement and interaction of NK cluster 01 cells with melanoma cells. This study explores the feasibility of targeting NK cluster 01 cells with small molecule drugs via molecular docking, offering a promising approach to bolster the clinical utility of NK cell therapy. We comprehensively analyze the heterogeneity of NK cells within melanoma, elucidate the potential regulatory interactions between NK cells and other microenvironmental components, and establish a basis for the future clinical utilization of distinct NK cell subsets as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fbdc1aaba32b7600dd3221ae61c5e31ee96bd3" target='_blank'>
              Integrating Single‐Cell and Spatial Transcriptomics Reveals NK Cell Subpopulations Associated With Immunotherapy for Melanoma
              </a>
            </td>
          <td>
            Zhicheng Hu, Yongfei Chen, Hao Yang, Qiuming Pan, Hongrui Li, Shuting Li, Junxi Wang, Yudi Huang, Guanglong Huang, Shanqiang Qu
          </td>
          <td>2025-12-01</td>
          <td>Smart Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7811a13e858225580004f20ab45830293f49cac3" target='_blank'>
              Tissue architecture and immune niches govern ctDNA release in colorectal cancer
              </a>
            </td>
          <td>
            Stefan Kühberger, Katja Sallinger, , Maria Escriva Conde, Sergio Marco Salas, Silvia Andaloro, C. Beichler, Ricarda Graf, Karin Pankratz, Julia Enzi, Sarah Binder, Martina Scheiber, Lilli Bonstingl, J. Blatterer, S. Uranitsch, G. Moitzi, Hannes Schmölzer, Hubert Hauser, Karin Strohmeyer, Mats Nilsson, Sigurd Lax, Antonia Syrnioti, Rudolf Oehler, Gerald Höfler, F. Aigner, Amin El-Heliebi, Ellen Heitzer
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, Michal R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Cancer is not composed of a single cell type but rather a complex community of cancer cells, immune cells, and supporting stromal cells that communicate with each other. These cellular conversations shape how each cancer grows, spreads, and responds to treatment. In this study, we compared single-cell sequencing data from seven different human cancers to explore how these cell interactions differ among tumor types. We found that pancreatic cancer contains many neutrophils, a type of immune cell that interacts mainly with other immune cells, while liver cancer lacks the usual fibroblast support cells. In contrast, esophageal and breast cancers were rich in fibroblasts that send growth signals to tumor cells, and thyroid cancer retained genes that may slow tumor progression. These differences help explain why some cancers behave more aggressively than others. Our findings provide a clearer picture of how the tumor environment varies among cancers and may guide the development of new strategies to treat solid tumors by targeting their surrounding cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95ce9b5897ed438ba40418ac1eb1c9c140e7a59a" target='_blank'>
              Comparative Oncology: Cross-Sectional Single-Cell Transcriptomic Profiling of the Tumor Microenvironment Across Seven Human Cancers
              </a>
            </td>
          <td>
            Riku Okamoto, Kota Okuno, Akiko Watanabe, Kanako Naito, Hiroyuki Minoura, Shumpei Shibaki, Kyonosuke Ikemura, Keiko Oki, Yu Kuroda, Shiori Fujino, Yusuke Nie, Nobuyuki Nishizawa, Eiichiro Watanabe, Mariko Kikuchi, Koshi Kumagai, Takahiro Yamanashi, Hiroshi Katoh, Hajime Takayasu, Takeo Sato, Takafumi Sangai, Yusuke Kumamoto, T. Naitoh, Naoki Hiki, Keishi Yamashita
          </td>
          <td>2025-10-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCLs) are a heterogeneous group of malignancies that can arise in lymph nodes or extranodal locations, including immune-privileged sites. Here, we applied highly multiplexed spatial transcriptomics and proteomics together with genomic profiling to characterize the immune microenvironment architecture of 78 DLBCL tumors. We define seven distinct cellular niches, each characterized by unique cellular compositions, spatial organizations and patterns of intercellular communication associated with niche-specific phenotypes of both T cells and tumor B cells. Among these, DLBCLs from immune-privileged sites showed abundant T cell infiltration into diffuse niches, where immune cells were intermixed with tumor B cells and bore transcriptional hallmarks of activation and effector function, suggesting that they may be primed for anti-tumor immunity. Spatial characterization of the DLBCL immune microenvironment, therefore, reveals cellular niches that foster divergent patterns of cell–cell interactions contributing to the phenotypic heterogeneity of both niche-resident tumor and immune cells. Analysis of the immune microenvironment of diffuse B cell lymphomas using spatial transcriptomics, proteomics and genomics highlights discrete cellular niches with divergent patterns of cell–cell communication that contribute to the phenotypic heterogeneity of both tumor and immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe40c75d4c6f7843c947dbae95bbc2601c4d486" target='_blank'>
              Multi-modal spatial characterization of tumor immune microenvironments identifies targetable inflammatory niches in diffuse large B cell lymphoma
              </a>
            </td>
          <td>
            Yibo Dai, A. Kizhakeyil, Dai Chihara, Xubin Li, Yunhe Liu, T. P. Sainz Zuniga, A. Wilson, Jared Henderson, Daniil Vibe, Arman Petrosyants, Connor Jacobson, Alexander Sarachakov, K. Nomie, K. Kryukov, Aleksander Bagaev, Ayushi Chauhan, J. Westin, Christopher R. Flowers, Francisco Vega, Linghua Wang, Michael R Green
          </td>
          <td>2025-10-21</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Cancer progression and treatment failure are profoundly influenced by the complex ecosystem of tumors, characterized by extensive cellular heterogeneity and a spatially organized microenvironment. Single-cell sequencing has revealed unprecedented diversity in malignant, immune, and stromal tumor populations, but lacks anatomical context. In contrast, spatial transcriptomics can map gene expression within tissue architecture, but typically at low cellular resolution. Integrative approaches now merge these advantages, enabling precise characterization of clonal dynamics, cellular communication, and the ecological niches that underlie metastasis and therapeutic resistance. This integrative approach provides a multidimensional view of tumor biology, highlighting context-dependent mechanisms of drug vulnerability and immune evasion. In this review, we explore how single-cell and spatial technologies are reshaping our understanding of the cancer ecosystem, integrating these two advanced technologies and advancing our progress toward spatially based precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e319db97a0be1c872a8ca8ea98199f6dcdc2cfc0" target='_blank'>
              Single-cell sequencing and spatial transcriptomics: a new paradigm for understanding the cancer ecosystem
              </a>
            </td>
          <td>
            Jasper Morelli, Lachlan Kerr
          </td>
          <td>2025-10-15</td>
          <td>Clinical and translational reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Leukemia and lymphoma represent a significant burden in Nigeria, characterized by distinct molecular profiles and treatment responses compared to global patterns. This uniqueness can be attributed to several factors including genetic diversity within the Nigerian population, environmental exposures specific to the region, and healthcare access challenges that affect early diagnosis and treatment adherence. The comprehensive understanding of these hematological malignancies has been revolutionized by the advent of single-cell technologies, which have uncovered unprecedented resolution into tumor heterogeneity, evolution, and microenvironmental interactions. Unlike traditional bulk sequencing methods that average signals across thousands of cells, single-cell approaches allow researchers to examine individual cells, revealing rare but critical subpopulations that drive disease progression and therapy resistance.
This review synthesizes findings from Nigerian studies integrated with global advances in single-cell research, highlighting how clonal architecture, cellular ecosystems, and molecular signatures contribute to pathogenesis and treatment outcomes. We explore how single-cell multi-omics approaches—including transcriptomics, genomics, and proteomics—have identified novel therapeutic targets and resistance mechanisms in leukemia and lymphoma. Additionally, we discuss the potential for implementing these technologies in Nigeria to advance precision medicine, considering both the opportunities and challenges specific to the Nigerian healthcare context. The integration of single-cell analyses into clinical practice promises to improve risk stratification, therapy selection, and overall management of these malignancies in resource-limited settings through more targeted and effective treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca65a47e4dd6298d29b7f5a8f9777fdb3fa1474c" target='_blank'>
              Unraveling Clonal Architecture and Microenvironmental Complexity of Leukemia and Lymphoma Using Singlecell Multi-Omics in Nigeria
              </a>
            </td>
          <td>
            Leon Oluci Kochukgu
          </td>
          <td>2025-11-06</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) remains a leading cause of cancer mortality, with liver metastasis being the principal determinant of poor prognosis, but the spatial mechanisms orchestrating metastatic niches remain elusive. Method To dissect the molecular and spatial dynamics of CRC progression, we constructed an integrative atlas using 35 single-cell RNA-seq datasets and spatial transcriptomics from primary tumors, liver metastases, and matched normal tissues. Malignant epithelial subpopulations were stratified via inferCNV and CytoTRACE analyses. Stromal-tumor interactions were dissected using CellChat and NicheNet, with functional validation through in vitro co-culture and immunohistochemistry. Result We identified a transcriptionally distinct epithelial subpopulation, termed high-malignancy CRC (High-M CRC), enriched in metastatic lesions and characterized by enhanced stemness, MYC-driven transcriptional activity, and glycolytic reprogramming. Stromal-tumor interaction analyses revealed that cancer-associated fibroblasts (CAFs), particularly matrix CAFs (mCAFs), promote malignant progression via the HGF-MET-MYC signaling axis. Spatial transcriptomic mapping confirmed the physical proximity and molecular co-localization of High-M CRC cells and mCAFs, along with enriched glycolysis and MYC expression at the cell-cell interface. In vitro functional validation demonstrated that CAF-derived HGF activates MET-MYC signaling in CRC cells, enhancing their invasion and proliferation—effects reversible by MET knockdown. Conclusion We unveil a spatially organized metabolic niche driven by stromal-tumor HGF-MET-MYC signaling. These findings offer novel insights into the stromal-tumor interaction and suggest actionable targets for therapeutic intervention in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dc67b8fc38aa04a6e00b35ef87b6cb65aaaec9" target='_blank'>
              A spatially resolved single-cell landscape of colorectal cancer liver metastasis reveals a stromal-tumor glycolytic signaling interaction
              </a>
            </td>
          <td>
            Jiahui Chen, Zukai Wang, Bingwang Zhu, Guoxian Guan
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="With the advancement of novel technologies such as whole-genome sequencing, single-cell sequencing, and spatial transcriptomics, single-omics analyses have already promoted the research of tumorigenesis as well as development and have partly elucidated the evolutionary processes of lung cancer. However, it is still difficult to distinguish these confounding features via single dimensional approaches due to the complexity, heterogeneity and cell-cell interactions with the immune microenvironment in lung cancer. Multi-omics approaches provide a holistic framework for constructing detailed tumor ecosystem landscapes, thereby facilitating the development of a more robust classification system for precision diagnosis and treatment, and aiding in the discovery of novel cancer biomarkers. In this review, we summarize the potential and applications of multi-omics approaches in characterizing intratumor heterogeneity and the tumor microenvironment throughout the course of lung cancer development. By further discussing the discovery and application of diagnostic and therapeutic biomarkers across precancerous lesions, early-stage lung cancer, tumor progression, metastasis, and therapy resistance, we outline the current challenges and future prospects of using multi-omics to identify reliable biomarkers. Moreover, we emphasize that integrative multi-omics models hold great promise for elucidating the complex interactions within the lung cancer ecosystem, thereby contributing to improved diagnostic accuracy, optimized therapeutic strategies, and better patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb5432eb94e92ff0ae0b03efad123a651887abd5" target='_blank'>
              Harnessing multi-omics approaches to decipher tumor evolution and improve diagnosis and therapy in lung cancer
              </a>
            </td>
          <td>
            Yicong Cheng, Ling Bai, Jiuwei Cui
          </td>
          <td>2025-11-05</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Intratumour heterogeneity (ITH) is one of the key characteristics of cancer and is closely associated with patient prognosis, treatment resistance, and tumor metastasis. Nevertheless, the study of ITH in hepatocellular carcinoma (HCC) remains limited. Methods: The present study elucidated the influence of ITH on the tumor microenvironment (TME) in HCC. We applied Non-negative Matrix Factorization (NMF) analysis to a cohort of 78 single-cell RNA sequencing (scRNA-seq) HCC samples to systematically characterize ITH. Furthermore, by integrating spatial transcriptomics (ST) data from five HCC patients, we comprehensively analyzed the spatial organization and functional properties of distinct niches within HCC. We conducted a detailed analysis of the cell-type co-localization relationships within the TME and constructed a comprehensive atlas of HCC spatial organization. Results: We observed a co-localization relationship between hypoxia tumor cells, plasmalemma vesicle-associated protein (PLVAP+) endothelial cells (EC), and vascular endothelial growth factor A (VEGFA+) cancer-associated fibroblasts (CAF), suggesting a key role for hypoxia tumor cells in VEGFA+ CAF transformation and tumor angiogenesis. We identified a unique boundary region enriched with dendritic cells1 (DC1), interferon-expressing tumor cells, lymphatic EC, C–X–C Motif Chemokine Ligand 10 (CXCL10+) macrophages (Mac), and secreted phosphoprotein 1 (SPP1+) Mac located between the tumor-infiltrating immune cells and tumor regions. Furthermore, we found that CXCL10+ Mac and SPP1+ Mac, despite co-localizing in the boundary region, exhibit distinct functions, which may be attributed to their unique spatial locations, with the former being closer to the immune-infiltrated region and the latter more proximal to the tumor area. Conclusions: Our study highlights the critical role of spatial interactions between tumor cells and the microenvironment in HCC. The findings offer new insights into ITH and underscore the importance of spatial organization in understanding cancer biology and designing future precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e826ac6739dc6ba19b8b336aa7292a375efd8d3e" target='_blank'>
              Integrative Analysis of Single-Cell and Spatial Transcriptomics Reveals Intratumor Heterogeneity Shaping the Tumor Microenvironment in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Yue Liu, Guoping Dong, Jie Yu, Ping Liang
          </td>
          <td>2025-12-01</td>
          <td>Livers</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0ba0d723e0604e91072fa9745f53e634a7a5814" target='_blank'>
              Evaluation of single-cell RNA profiling technologies using FFPE, fresh, and frozen ccRCC tumor specimens
              </a>
            </td>
          <td>
            Sherin Xirenayi, Sabrina Y. Camp, Amanda E. Garza, Yashika Rustagi, Erica Pimenta, Erin Shannon, Anwesha Nag, Adam Betherman, Anthony Anselmo, Rachel Trowbridge, Aaron R. Thorner, Jihye Park, Kevin Bi, E. V. Van Allen
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="In recent years, spatial transcriptomics (ST) has emerged as a groundbreaking technology with the potential to transform and accelerate our understanding of cellular crosstalk. While single-cell approaches have uncovered an unexpected level of cellular heterogeneity in both healthy and diseased tissues, they remain limited in their ability to capture cellular interactions in the native microenvironment. ST techniques bridge this gap by preserving anatomical information, enabling a direct investigation of spatially defined cellular interactions. This feature is particularly relevant in tissues such as skeletal muscle, where syncytial myofibers coexist with a heterogeneous set of interstitial cell populations. Spatial localization is a key factor during muscle regeneration, particularly as stem cell progression is driven by complex interactions between resident and recruited cell populations. Understanding these spatial dynamics is therefore critical to better characterize the fundamental mechanisms of muscle repair and identify aberrant signaling pathways of chronic injury or impaired regeneration. In this review, we will explore the various types of ST techniques, provide a brief summary of the available analytical tools, and highlight recent advancements in the skeletal muscle field enabled by the application of ST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1289a819036649b0e45823c7b193207d283e3d73" target='_blank'>
              Across the space: applications of spatial transcriptomic technology in healthy and diseased muscle
              </a>
            </td>
          <td>
            L. Virtanen, C. D’Ercole, L. Giordani
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT The metabolic programs of immune cells influence their activation, differentiation, and effector functions. While much of immunometabolism has focused on cell‐intrinsic regulation, it is now clear that metabolic activity is profoundly influenced by the surrounding tissue environment. In tumors and other inflammatory settings, immune cells are shaped by nutrient gradients, hypoxia, and immunoregulatory metabolites, factors that are spatially heterogeneous and often poorly captured by traditional methods. This review highlights recent technological advances that enable spatially resolved analysis of immune metabolism, with an emphasis on multimodal integration and cancer as a model system. Mass spectrometry imaging (MALDI, DESI), high‐resolution platforms like SIMS, and vibrational imaging approaches such as Raman microscopy enable direct visualization of metabolites in tissue. Transcriptomic and proteomic data can be used to infer metabolic states, and computational models are being developed to integrate these diverse data layers. Together, these technologies are transforming the study of immunometabolism from dissociated cells to the intact tissue context. Key challenges remain in resolution, annotation, and data integration, but spatial immunometabolism holds particular promise for illuminating mechanisms of immune regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4ef9dd08ead570796d0b86fc9dc291d59a2835" target='_blank'>
              Spatial Immunometabolism: Integrating Technologies to Decode Cellular Metabolism in Tissues
              </a>
            </td>
          <td>
            F. Hartmann
          </td>
          <td>2025-11-01</td>
          <td>European Journal of Immunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Penile squamous cell carcinoma (PSCC) is a malignancy characterized by a poor prognosis and lack of reliable biomarkers, presenting considerable therapeutic challenges. Tertiary lymphoid structures (TLSs) are critical modulators of antitumor immunity; however, the immunological dynamics, particularly the roles of B cells and their interactions with naive T cells in PSCC tissues, remain inadequately understood. This study integrates transcriptomic approaches, including spatial transcriptomics, single‐cell sequencing (scRNA‐seq), and bulk RNA sequencing (bulk RNA‐seq), and immunohistochemistry to elucidate the immune architecture of and functional mechanisms within TLSs. The results reveal a positive correlation between TLS density and patient survival, with CD74⁺ B cells tending to be enriched during early TLS formation. These cells exhibit strong immune activation and a propensity to differentiate into plasma cells. By engaging with naive T cells through HLA‐DRA via ligand–receptor interactions, CD74⁺ B cells activate transcription factors, including NFKB1, NFKB2, NFATC1, NFATC2, FOS, and RUNX1, in naive T cells, thereby enhancing the immune response. Consequently, CD74⁺ B cells represent a compelling biomarker for and therapeutic target of PSCC, offering profound insights into the immunological mechanisms that drive PSCC progression and response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f775dcc4d8e646b4cd53d88e405d27443da1f7f" target='_blank'>
              Spatial Transcriptional Dynamics of CD74⁺ B Cells in Tertiary Lymphoid Structures Drive Immune Evolution in Penile Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Ting Xue, Chuangzhong Deng, Jingya Liu, Ru Yan, Jing Li, Xiheng Hu, Xueying Li, Xiao Xiao, Jietian Jin, Hongzhen Tang, Desi Chen, Zihan Zuo, Yujie Liang, Dongbin Wang, Bonan Chen, Hui Han, Zaishang Li
          </td>
          <td>2025-10-20</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617cbae0aef811bf0bd717218d2f691f2de87645" target='_blank'>
              Germline genetic risk converges through intercellular crosstalk in tumor microenvironment
              </a>
            </td>
          <td>
            X. Bai, Yang Liu, Shuai He, Chun-Ling Luo, G.-W. Lin, Dong-Mei Chi, Shuang-Yan Ye, Xiu-Zhi Wang, , You Zhou, Shu-Qiang Liu, P.-P. Wei, Xin-Yuan Guan, Hai-Qiang Mai, Soon Thye Lim, Jian-Jun Liu, C. Khor, Melvin Lee, Kiang Chua, Jin-Xin Bei, Yanni Zeng
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Thyroid cancer, the most common endocrine malignancy, is characterized by a unique and complex tumor microenvironment (TME). To unravel the high tumor heterogeneity and molecular mechanisms driving cancer progression, we performed single-cell RNA sequencing (scRNA-seq) analysis, enabling a comprehensive exploration of cellular diversity and molecular dynamics at single-cell resolution. We employed Principal Component Analysis (PCA) and Uniform Manifold Approximation and Projection (UMAP) for dimensionality reduction and subsequent identification of cellular clusters. Differential gene expression analysis across subclusters was conducted using the FindAllMarkers function, while the DoHeatmap function was utilized to visualize the distribution of differentially expressed genes. The AUCell algorithm was applied to evaluate pathway enrichment within specific cell subtypes. To decipher cellular communication networks, we integrated the CellChat and NicheNet algorithms, which revealed intricate intercellular signaling interactions. Finally, multiplex immunohistochemistry (mIHC) was performed to validate key cellular interactions identified in silico. By analyzing 405,077 single cells from 50 thyroid cancer samples (including papillary, anaplastic, and metastatic tumors) and 14 normal thyroid tissues, we identified four major cellular subpopulations through unbiased clustering based on gene expression patterns and representative cellular markers. The TME was found to encompass diverse immune, endothelial, and mesenchymal cell subtypes, including novel populations such as CD4 + HSPA1A + T cells. Functional pathway enrichment analysis highlighted the roles of abundant cell types in tumor progression. Cell-cell communication analysis uncovered potential immunotherapeutic targets and revealed critical crosstalk among hub niche cells, including APOE+ macrophages, EMT-like cancer-associated fibroblasts (CAFs), and RBP7+ endothelial cells. These findings were further validated by multiplex immunohistochemistry, confirming the spatial organization and interactions of these cell populations within the TME. Our study provides a comprehensive single-cell transcriptomic atlas of thyroid cancer, offering profound insights into tumor heterogeneity, the functional roles of key niche cells, and potential biomarkers for anticancer therapy. These findings not only enhance our understanding of thyroid cancer biology but also pave the way for the development of novel therapeutic strategies targeting the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b009366b4ed17ce1b669fc0314c882a610b52822" target='_blank'>
              Comprehensive single-cell RNA analysis reveals intertumoral microenvironment heterogeneity and hub niche of carcinogenesis in thyroid cancer
              </a>
            </td>
          <td>
            Hao Xu, Junjie Ma, Nanjun Li, Haoxiang Xuan, Shijie Lin, Yunuo Shen, Lu Yin, Ping Wang, Qianhui Xu
          </td>
          <td>2025-11-26</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Recent single-cell studies implicate the glutamatergic lineage of the rhombic lip (RL) as the origin of group (G) 3/4 medulloblastomas (MBs) and a stalled differentiation program as underlying their malignant transformation. It is unknown, however, if G3/4 MBs exhibit cellular plasticity as has been observed in glioma. The effect of therapy on cellular hierarchies in G3/4 MBs is unknown. We profiled N=57 G3/4 MB frozen specimens obtained from the University of California, San Francisco Brain Tumor Center and the Children’s Brain Tumor Network biobanks via single-nucleus RNA and epigenetic sequencing. This cohort included 43 diagnostic and 14 patient-matched recurrent specimens. We profiled N=32 FFPE specimens via Xenium single-cell spatial transcriptomics. We performed genetic lineage tracing in intracranial implants of G3 MB cell lines derived from primary and metastatic disease using engineered barcodes, and in clinical specimens using endogenous mutations. Remarkably, xenografts from primary tumors produced similar cell types, in similar proportions to those found in treatment-naive human MBs. Moreover, the distribution of cell types found in xenografts was conserved at the level of individual clones, indicating cell-intrinsic regulation of tumor composition. Single-cell phylogenetic analysis demonstrated significant rates of dedifferentiation events from unipolar brush-like cells to RL-progenitor phenotypes. This plasticity was increased in recurrent specimens and in metastatic models compared to models and samples derived from primary disease. Samples with higher plasticity exhibited greater spatial heterogeneity, while samples with lower rates of dedifferentiation had more structured spatial organization. These studies show that G3/4 MB cellular composition is homeostatic in treatment-naive disease and conserved at the level of individual clones. Yet, these tumors exhibit plasticity that is exacerbated by therapy and affects spatial organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7150b972b545d28e4de15c0584606a43a9a4dc" target='_blank'>
              STEM-18. GENETIC LINEAGE TRACING OF MEDULLOBLASTOMA UNDER THERAPY WITH SPATIAL CONTEXT
              </a>
            </td>
          <td>
            Bohyeon Yu, Jangham Jung, Abhinav Jain, Emanuele Rosatti, Joanna Phillips, Aaron A. Diaz
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor heterogeneity is a fundamental property of leukemia and lymphoma that drives disease progression, relapse, and therapy resistance. For decades, bulk genomic and transcriptomic analyses have provided a foundational understanding of these malignancies but have obscured the cellular diversity within the tumor ecosystem. The advent of high-throughput single-cell technologies, particularly single-cell RNA sequencing (scRNA-seq), has revolutionized our capacity to deconvolute this complexity. This review synthesizes how single-cell analyses are reshaping our understanding of tumor heterogeneity in hematological cancers. We explore the role of intra-clonal genetic diversity in Acute Myeloid Leukemia (AML) and B-cell Acute Lymphoblastic Leukemia (B-ALL), the functional and transcriptional plasticity of subpopulations, and the dynamic interplay between malignant cells and the immune microenvironment in both leukemia and lymphoma. We further highlight how the tumor microenvironment in lymphomas fosters heterogeneity and immune evasion. Furthermore, the integration of single-cell data with spatial transcriptomics and computational models is beginning to predict evolutionary trajectories, offering a window into future clinical behaviors. Critically, we discuss how single-cell approaches have identified pre-existing or emergent resistant subclones under therapeutic pressure, providing a mechanistic basis for treatment failure. The clinical implications are profound, paving the way for single-cell diagnostics for minimal residual disease (MRD) monitoring and the identification of novel therapeutic targets. We conclude that single-cell multi-omics is an indispensable tool for unraveling the multifaceted nature of tumor heterogeneity, with the potential to propel the field toward more precise and effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0baf5679635e6fcbb86a03a048b12087069db02" target='_blank'>
              Tumor Heterogeneity in Leukemia and Lymphoma: Insights from Single-Cell Analyses
              </a>
            </td>
          <td>
            Pang Hou, Wen
          </td>
          <td>2025-11-21</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c21e29753393c78c9a7147383291954089d02fd" target='_blank'>
              KDM5-driven transcriptional noise fuels plasticity-led awakening and relapse in paediatric cancer
              </a>
            </td>
          <td>
            Alejandro Allo Anido, Cecilia Roux, Emilia Chen, Siân Hamer, Abigail Shea, Ayeh Sadat Sadr, Christie English, Charlotte Butterworth, Harvey Che, Angella Bellini, Birgit Geoerger, Gudrun Schleiermacher, L. Chesler, Michael David Morgan, Alejandra Bruna
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Deciphering the spatial biology of the tumor microenvironment (TME) can inform researchers about the likelihood of therapeutic success. Of particular interest is the spatial organization of immune cells, especially T cells, which can undergo exhaustion resulting in reduced effector functions due to expression of inhibitory markers such as PD-1. Imaging Mass Cytometry™ (IMC™) technology is a quantitative multiplexed spatial imaging technique for assessment of 40-plus biomarkers simultaneously on the same tissue slide without signal amplification or autofluorescence interference. We used IMC technology to investigate the spatial distribution and phenotypic characteristics of immune cells, focusing on exhausted T cells in the TME of multiple cancers. Various IMC imaging modes revealed striking heterogeneity of the TME, including clusters of lymphocytes at the tumor margins, indicative of immune surveillance. Pixel-clustering and single-cell analyses delineated distinct tumor areas based on extent of T cell exhaustion, immune cell density, cell proliferation and stromal components. These findings illustrate the power of IMC technology to elucidate the complex spatial landscape of the TME, which may aid in developing prognostic assessments and guide personalized therapeutic strategies for cancer treatments. For Research Use Only. Not for use in diagnostic procedures.



 Tumor Immunology: Checkpoints, Prevention, and Treatment (TIPT)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b22f9c7f1b4911c7dde0700548e2e19989a2ee" target='_blank'>
              High-plex spatial analysis of T cell exhaustion using Imaging Mass Cytomerty for immuno-oncology studies 2622
              </a>
            </td>
          <td>
            Qanber Raza, T. Pfister, Yun-Jyh Chwee, Nikesh Parsotam, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ffa550baca76e714d9b657ce67086b4b77a9db" target='_blank'>
              Deciphering Tumor Microenvironment Dynamics in Tumorigenesis and Lymph Node Metastasis of Esophageal Squamous Cell Carcinoma using Single-cell RNA Sequencing
              </a>
            </td>
          <td>
            Hansoll Na, T. Hong, Chung Lee, Y. Yang, Ha Eun Kim, Byung Jo Park, Hyun Ki Kim, Dae Joon Kim
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="

 Breast cancer (BC) prognosis is fundamentally dictated by its pronounced molecular and cellular heterogeneity. While single-cell RNA sequencing (scRNA-seq) has detailed the tumor microenvironment (TME) cellular composition, a critical knowledge gap persists regarding the vascular heterogeneity of tumor endothelial cells (TECs) across different BC subtypes. Current TEC classification remains conventional, obscuring whether subtype-specific TECs possess unique transcriptomic signatures that influence treatment response, particularly to anti-angiogenic therapies. Given the pivotal role of tumor vasculature in proliferation and metastasis, and the existing research bias towards Triple-Negative BC (TNBC), a comprehensive investigation into the complexity of the vascular system is warranted. Furthermore, as lymph node metastasis (LNMT) is a major determinant of cancer mortality, distinguishing the endothelial and immune cell subtypes within LNMT microenvironments from primary tumors is essential for clarifying pathogenesis.



 To address this, we integrated scRNA-seq data from primary BC tumors, ER_LN and healthy tissues, generating a robust dataset of 98,000 cells (~ 12 samples). Employing single-cell transcriptomics, spatial transcriptomics, and immunohistochemistry, we precisely delineated the endothelial cell (EC) heterogeneity within matched tumor and peri-tumoral tissues and scrutinized their intricate interactions with immune populations.



 We identified two previously uncharacterized, tumor-enriched endothelial cell subtypes, designated EC4 and EC5, which demonstrate subtype-specific functional adaptations and prognostic significance. EC4 cells, highly prevalent across BC, are principally characterized by antigen presentation, immune cell recruitment, and pro-inflammatory signaling. Conversely, EC5 cells, also enriched in BC, exhibit robust extracellular matrix (ECM) remodeling and potent tumor angiogenesis. We further characterized the functional divergence of EC4 and EC5 within ER tumors relative to HER2 tumors and ER_LN metastases. Our analysis reveals conserved endothelial programming mechanisms across BC subtypes, coexisting with distinct TME-driven transcriptional adaptations. Importantly, interactome analysis highlighted novel and subtype-specific communications between these novel EC subsets and immune cells, particularly CD8+T cells and macrophages. Experimental validation demonstrates that ECs overexpressing APP can mediate the M2 polarization of macrophages, underscoring diverse immunomodulatory roles for EC subsets across different BC contexts.



 These findings offer critical and granular insights into the complex interplay between novel EC subtypes and the immune microenvironment in BC progression and metastasis. We establish that ECs are active and heterogeneous modulators of the TME, identifying specific therapeutic vulnerabilities within the tumor vasculature and providing a foundational blueprint for developing future precision immunotherapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67936270be24af1f65608417190d3a4e352d6fe3" target='_blank'>
              Single-cell profiling identifies heterogeneity of the immune microenvironment in healthy, primary and lymph node metastatic BC
              </a>
            </td>
          <td>
            Chuyu Liu, Feng Zeng, Shanshan Gao, Nengying Zhang, Yulan Cai
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is the 4th leading cause of cancer death in the United States as of 2024. Its tumor microenvironment (TME) contains heterogenous inflammatory and immunosuppressive areas due to varying immunophenotypes relating to immune cell infiltration and exhaustion. CD4+ T helper cells can contribute to these dynamics, but defined roles of some subsets in PDAC remain debatable. We hypothesize that effector CD4+ T cell subsets spatially influence levels of inflammation in the PDAC TME to promote tumor metastasis. To investigate these dynamics, we designed a novel sequential multiplex immunofluorescent panel for murine FFPE slides consisting of 50 markers to spatially capture immune cells with activation/exhaustion states, epithelial-to-mesenchymal phenotypes, vessel infiltration, and stromal deposition in the TME. Using FFPE slides also improves preservation of structural morphology and translation to human tumor samples. Here, we applied the panel to primary murine PDAC tumors and liver metastases compared to normal tissues, revealing heterogenous MHC-I presentation from primary tumor cells, variations in CD4+ subsets in the primary tumor, and increased lymphocyte infiltration in liver metastases, validating our panel with recent spatial and dissociative multimodal publications. Ultimately, we will apply the panel to mice deficient in specific CD4+ subsets to better elucidate the spatial role of effector T helper cells in PDAC metastases.



 Supported by UAB Hematology & Oncology Startup Fund; Hirshberg Foundation for Pancreatic Cancer Reasearch



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d46f7f6b06d7c343cfa41f8977a3f99ebd086c73" target='_blank'>
              High-dimensional spatial analysis of murine tumor microenvironment highlights the heterogeneity of primary and metastatic pancreatic cancer immune landscape 3822
              </a>
            </td>
          <td>
            Christina R Larson, Ayushi Mandloi, Jace Baines, Richard Laube, Alana M. Franceski, R. Welner, Julienne Carstens
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background : Despite the use of immunotherapy in esophageal squamous cell carcinoma (ESCC), treatment failure occurs occasionally in patients, yet the underlying mechanisms remain poorly understood. Methods : We conducted large-scale single-cell RNA sequencing (scRNA-seq) data analysis, which integrated seven independent datasets from 192 ESCC patients to yield over 440,000 high-quality single cells, to systematically characterize the tumor microenvironment (TME) landscape during ESCC progression and immunotherapy response. Additionally, we performed high-resolution spatial transcriptomics (stRNA-seq) using the 10x Visium HD platform on paired pre-and post-treatment tissues from two patients (one immunotherapy responder and one non-responder), which enhanced the findings from the scRNA-seq data and mapped therapy-induced TME at the spatial level. Multiplex immunohistochemistry was employed based on seven patients to confirm distinct patterns of intercellular crosstalk underlying differential therapeutic outcomes. Results : In scRNA-seq data, we found that B lineage cells were reduced during ESCC progression but were enriched in immunotherapy-resistant patients. Further analysis of malignant ESCC cells suggested that immunotherapy resistance might be associated with a subpopulation of tumor cells exhibiting aberrantly elevated cholesterol biosynthesis. Cell communication analysis of scRNA-seq and stRNA-seq data collectively revealed that immunotherapy resistance was linked to cellular crosstalk between cholesterol-biosynthetic tumor cells and germinal center (GC) B cells within tertiary lymphoid structures. Notably, single-cell, spatial data, and multiplex immunohistochemistry demonstrated that cholesterol biosynthesis-associated ESCC cells express elevated levels of MIF. This disrupts GC reactions by competing with the CXCL12-CXCR4 signaling axis via MIF-CXCR4 interactions, thereby impairing B cell-mediated immunity. Conclusions : MIF + tumor cells in GCs may be a biomarker for predicting immunotherapy resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8a1ee6ff72d48f3e16c31d484505ab587cac16" target='_blank'>
              MIF-expressing tumor cells mediate immunotherapeutic resistance in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Jing Song, Xiaomei Song, Yue Xie, Hong Guo, Yupeng Cun
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Somatic evolution drives cancer progression and therapy resistance. Underlying phenotypic progressions is the development of marked clonal heterogeneity in most cancers. However, deciphering the downstream molecular effects of clonal heterogeneity in primary cancer has been limited due to the inability to isolate subclones by standard methods. While single-cell RNA-sequencing (scRNA-seq) provides high-resolution mapping of heterogeneous cell states, it cannot link transcriptional programs to cancer-driving mutations in these individual cells. We and others developed methods to detect genotypes and their transcriptional outputs in single cells (e.g., Genotyping of Transcriptomes; Nature, 2019). However, these approaches have technical limitations to broadly profile numerous mutations, which is nonetheless required for clonally complex neoplasms. The existing approaches are also restricted to fresh or frozen samples, limiting the use of the widely available pathology formalin fixed paraffin embedded (FFPE) tissues.
 To address these challenges, we developed Genotyping of Transcriptomes for Multiple Targets and Sample Types, GoT-Multi, a next generation single-cell multi-omics method compatible with FFPE tissues that integrates multiplexed genotyping with transcriptomic profiling. GoT-Multi adapts a probe-based scRNA-seq method (Flex Fixed RNA Profiling, 10x Genomics) by including custom mutation-specific probes along with the standard transcriptional probes. Furthermore, we developed an ensemble-based machine learning pipeline, GoT-Multi-ML, that optimizes the genotyping calls by denoising technical artifacts. GoT-Multi-ML leverages multiple machine learning models to learn the non-linear patterns of the genotyping data features (e.g., reads per genotyping transcript, GC content of probes), agnostic to the whole transcriptomic data. We validated GoT-Multi via a cell line-mixing experiment using SK-BR-3 and MCF-7 cell lines. GoT-Multi detected the targeted mutations (HIST1H1C A24T and S100A10 A77V) in the expected cell line (MCF-7) with an average genotyping rate of 72% and accuracy of 97%.To define the impact of clonal evolution on phenotypic progression, we focused on a prototype of cancer evolution, the progression of chronic lymphocytic leukemia (CLL) to therapy-resistant large B-cell lymphoma (LBCL) called Richter Transformation (RT). We applied GoT-Multi to a unique cohort of primary nodal cryopreserved samples containing both CLL and LBCL components (n = 5). We targeted 18 mutations (1-6 targets per sample) with a 98% median accuracy. We observed heterogeneous cancer cell states, including proliferating, stress response and inflammatory (n = 51,465 cells). LBCL was enriched in proliferating and cyclin D2-elevated states, whereas CLL was associated with a stress response signature. GoT-Multi enabled us to reconstruct the clonal architectures, up to 4 distinct subclones per lymphoma, and their associated cell states. Differential gene expression between mutated and wildtype cells revealed that genetically heterogenous subclones, including those with SRRM2 and JUNB mutations and therapy resistance-associated mutations in PLCG2 and BTK, displayed enrichment in inflammatory cell states, upregulating TNF and interferon signaling genes while suppressing cell cycle-related genes. Conversely, other subclones with mutations in, e.g., IRF8, POU2F2, PRKDC were associated with MYC activation and/or enhanced proliferation. These findings suggested that heterogeneous genotypes may converge on similar downstream transcriptional cell states. In addition, TNF itself was overexpressed in the proliferative subclones, suggesting that TNF production from these subclones may enhance inflammatory signaling of the TNF-responsive subclones, indicative of crosstalk between subclones. Finally, we demonstrated the applicability of GoT-Multi to pathology archived FFPE tissues. We profiled 9 mutations with an accuracy of 98% in an FFPE RT sample (n = 2,140 nuclei), allowing us to detect 5 subclones and their associated cell states. In summary, co-mapping of clonal and cell state heterogeneity at single cell resolution through GoT-Multi suggested that heterogeneous subclonal genotypes may converge on similar downstream oncogenic pathways to enhance overall tumor fitness. We envision that the broad applicability of GoT-Multi may help uncover the molecular underpinnings of cancer progression and therapy-resistance across oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/954b694c778f5e54689dba8e5834b825525f28c2" target='_blank'>
              Co-mapping clonal and transcriptional heterogeneity in somatic evolution via got-multi.
              </a>
            </td>
          <td>
            Minwoo Pak, M. Saurty-Seerunghen, Kellie Wise, Tsega-Ab Abera, Chhiring Lama, Neelang Parghi, Xiaotian Sun, Qi Gao, Liming Bao, Mikhail Roshal, John N Allan, Richard Furman, Luciano G. Martelotto, Anna S. Nam
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/106e152b7c5daaf95009a3cfc0dacddd50f5bb8d" target='_blank'>
              Detecting Somatic Mutations in Rare Clones using Single Cell Multi-Omics
              </a>
            </td>
          <td>
            R. Gillman, Sonam Dudka, Jerome Samir, Raymond Louie, Chris Goodnow, Fabio Luciano, Mandeep Singh, Matt A. Field
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/300bf1ce4707bea01a2973482ed7fc72841204e2" target='_blank'>
              Genomic heterogeneity drives distinct infiltration patterns in glioblastoma.
              </a>
            </td>
          <td>
            D. Harwood, S. Artzi, V. Pedersen, A. Locallo, M. Lü, David Scheie, D. Nørøxe, N. Hammouda, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-11-29</td>
          <td>Acta neuropathologica communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Phenotypic plasticity is a major factor in tumor heterogeneity and treatment resistance. In particular, cancer stem cells (CSCs) represent a small subpopulation within tumors that possesses self-renewal and tumor-forming capabilities. Understanding reprogramming, maintenance, and lineage properties of CSCs requires dedicated tools to disentangle the respective influences of phenotypic inheritance and cell-cell interactions. Here, we set up ultra-wide field microscopy to image breast cancer cell lines expressing a stemness fluorescent reporter over several days. The fluorescent reporter distinguishes three phenotypes: CSCs, cancer differentiated cells (CDCs), and intermediate/transiting cancer cells (iCCs). Spatial statistics indicate significant zonation in which CSCs cluster together and are spatially separated from CDCs, forming patterns resembling niches. Surprisingly, single-cell time series reveal spontaneous reprogramming events from CDC to CSC even in unperturbed populations. We identify that such transitions are prone to arise during the cell cycle. Moreover, lineage analysis shows that the phenotype is partially inherited from ancestor cells. However, such heredity is not sufficient to explain the spatial properties of the cell population, which also depend on cell-cell interactions. Indeed, we find that phenotypic transitions of cancer cells are influenced by the phenotypic state of neighboring cells. Reprogramming into CSCs is respectively promoted and inhibited by the presence of CSCs and CDCs in the neighborhood. Altogether, our results disentangle how phenotypic inheritance and intercellular interactions orchestrate the spatio-temporal self-organization of cancer cell heterogeneity, maintaining a subpopulation of CSCs within niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0816aa85e620c96aaa89badbcea20ed0e370fd6d" target='_blank'>
              Spatial self-organization of cancer stem cell niches revealed by live single-cell imaging
              </a>
            </td>
          <td>
            Mathilde Brulé, Anaïs Horochowska, Emeline Fontaine, Raoul Torero-Ibad, Flavie Woesteland, Marie Denoulet, Jean Pesez, E. Adriaenssens, R. Toillon, X. Le Bourhis, Benjamin Pfeuty, Chann Lagadec, F. Anquez
          </td>
          <td>2025-10-27</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) invasion is shaped by dynamic clonal evolution within the spatial and cellular architecture of the brain. To investigate this process, we integrated multiparametric MRI, whole-exome sequencing (WES), bulk RNA-seq and single-nucleus RNA-seq (snRNA-seq) from 69 IDH-wildtype GBM patients, using spatially resolved biopsies from contrast-enhancing (CE) core tumor and non-enhancing (NE) invasive regions. Phylogenetic analysis revealed distinct clonal architectures. CE-enriched clones carried truncal mutations in conanical drivers of cell cycle and RTK signaling (TP53, PTEN, EGFR, PDGFRA/B), and private alterations in DNA repair genes (CDH11, PAX7, RAD51B). In contrast, NE-enriched clones exhibited private mutations in MAPK and Hippo pathways components (STAG2, BRAF, MAP2K3, CTNNA1), suggesting region-specific evolutionary trajectories. Furthermore, snRNAseq cell-cell communication analysis uncovered enriched ligand-receptor signaling in NE regions between malignant cells and brain-resident populations, including PTN—PTPRZ1 (myeloid), NRG-ERBB4 and SLIT-ROBO (neuronal, astrocytic, and oligodendrocytic). Bulk RNAseq deconvolution confirmed increased representation of neurons and glial cells in biopsies harboring NE-specific clones. These findings define a unique population of GBM clones localized at the infiltrative margin, marked by private MAPK/Hippo mutations and enhanced intracellular communication with the surrounding brain microenvironment. The recurrent engagement of neural-glial signaling pathways suggests that these NE-associated clones co-opt normal cellular networks to support invasion, immune evasion, and persistence following surgical resection. This study establishes a mechanistic link between spatial clonal evolution and tumor-host communication at the GBM infiltrative region and provide a framework for targeting cell-cell signaling circuits that sustain invasive tumor ecosystems and escape therapeutic treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee4b73c1f939dc48d12b3b70a8dbe5ab434d2210" target='_blank'>
              CSIG-31. REDEFINING THE GLIOBLASTOMA RESECTION MARGIN THROUGH SPATIAL CLONAL AND CELLULAR ECOSYSTEM ANALYSIS
              </a>
            </td>
          <td>
            Angad S. Beniwal, Junlin Wang, Xiaoling Lu, C. Sereduk, S. Migliozzi, L. Garofano, C. Plaisier, Leland Hu, R. Barajas, Junwen Wang, N. Tran
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Understanding the evolutionary dynamics of clonal populations is essential for uncovering the principles of development, disease progression, and therapeutic resistance. Recent advances in single-cell lineage tracing and transcriptomics enable such analyses by combining heritable barcodes with cell-state information. Here, we present SCOUT (single-cell Ornstein–Uhlenbeck trees), a framework that models gene expression dynamics along single-cell lineage trees using Ornstein–Uhlenbeck processes to distinguish neutral drift from selective pressure. Using simulations, we demonstrate that SCOUT accurately classifies genes based on their underlying evolutionary models. We further validate SCOUT in Caenorhabditis elegans development, identifying biological processes under selection across distinct developmental contexts. Finally, we apply SCOUT to a lung adenocarcinoma xenograft model, revealing key regulators of metastatic progression and tumor microenvironmental adaptation. By integrating lineage and transcriptomic data, SCOUT provides a powerful evolutionary lens for dissecting the forces that shape cell fate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b1a07c3c29336ff660b3f4e1f28115198c90579" target='_blank'>
              SCOUT: Ornstein–Uhlenbeck modelling of gene expression evolution on single-cell lineage trees
              </a>
            </td>
          <td>
            Hannah Stuart, Aaron McKenna
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Resistance to immune checkpoint blockade (ICB) remains a major challenge in lung adenocarcinoma (LUAD), with stromal mechanisms underlying CD8⁺ T cell exhaustion still poorly understood. By integrating single-cell, bulk, and spatial transcriptomic datasets using EcoTyper, we identified a distinct immunosuppressive ecotype, EC10, enriched for TGF-β signaling and epithelial–mesenchymal transition. EC10 exhibited spatial co-localization of fibroblasts, malignant epithelial cells, and exhausted CD8+ T cells, and was consistently associated with immune exclusion, poor progression-free survival, and elevated TIDE scores across four ICB-treated LUAD cohorts. Cell–cell communication analyses revealed a dominant TGFB1–SERPINE1 signaling axis originating from fibroblasts, linking stromal remodeling to T cell dysfunction. In contrast, EC12 represented an inflamed, ICB-responsive state enriched in interferon signaling. These findings define EC10 as a spatially organized, fibroblast-driven immunosuppressive ecosystem predictive of ICB resistance, and highlight the therapeutic potential of targeting the TGF-β axis in LUAD. This research identifies a fibroblast-driven immunosuppressive ecosystem in lung adenocarcinoma, linking TGF-β signaling to immune resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130525b051ffbdab87e5edbec0e75b95564de9c8" target='_blank'>
              Fibroblast TGF-β signaling defines spatial tumor ecosystems linked to immune checkpoint blockade resistance
              </a>
            </td>
          <td>
            Seo-Young Lee, Youngjoo Lee
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Background Angioimmunoblastic T cell lymphoma (AITL) is a subtype of peripheral T-cell lymphoma with a poor prognosis. Pathologically, AITL is characterized by the proliferation of diverse stromal cells (SCs), including blood endothelial cells (BECs) and follicular dendritic cells (FDCs). However, the tumor histology exhibits considerable heterogeneity, both between and within patients. It is hypothesized that these SCs form a supportive tumor microenvironment, and their histological heterogeneity may influence the clinical course. We previously reported that single-cell RNA sequencing (scRNA-seq) of SCs in lymph nodes (LNs) identified 30 distinct SC subclusters (Nat Cell Biol, 2022). However, the specific SC subclusters involved in AITL tumorigenesis remain undefined.
 Aims We aimed to characterize the SC heterogeneity in AITL microenvironments by employing scRNA-seq and spatial multi-omics at single-cell resolution.
 Methods Non-hematopoietic cells were isolated from 10 AITL and 5 normal LNs using magnetic- and fluorescence-activated cell sorting, followed by scRNA-seq. Based on the scRNA-seq data of SCs and immune cells, we designed a 289-gene panel for spatial transcriptomics (ST) using Xenium in situ, and a 50-protein panel for spatial proteomics (SP) using PhenoCycler fusion targeting immune cell and SC subclusters. These assays were applied to 27 formalin-fixed paraffin-embedded AITL samples. Whole exome sequencing (WES) was successfully performed on 26 of the 27 samples. We analyzed a bulk RNA-seq dataset of 97 AITL to validate the prognostic potential of gene expression patterns.
 Results First, we analyzed scRNA-seq data from 49,363 stroma cells and identified 30 SC subclusters consistent with our prior findings. Notably, transitional BECs between capillary BECs and activated high endothelial venules (C-aHEVs) expressed markers of both capillary BECs and HEVs, and were further subdivided into C-aHEV1 and C-aHEV2. Both tip cells, which contribute to angiogenesis, and C-aHEV2 were increased in AITL samples. Differentially expressed genes analysis revealed that both C-aHEV1 and C-aHEV2 highly expressed ACKR1. Notably, C-aHEV1 exhibited high expression of chemokines such as CXCL10 and CXCL12, whereas C-aHEV2 expressed genes encoding extracellular matrix proteins. Interaction analysis revealed that the CCL5-ACKR1 axis, which is known as a regulator of leukocyte migration, was activated between CD8-positive T cells and C-aHEVs. Furthermore, the COL15A1-integrin and SPARC-ENG axis, both of which have been reported to promote angiogenesis, were specifically enriched between C-aHEV2 and other HEVs.
 ST analysis of 2,020,192 cells identified 10 BEC subclusters, 5 non-endothelial SC subclusters, and 12 immune/tumor cell subclusters. Spatial proximity measurement revealed that C-aHEV1 was the BEC subcluster most closely associated with tumor cells. Spatial niche analysis revealed that C-aHEV1, together with FDCs, formed a distinct tumor-associated niche (CFT niche). On the other hand, C-aHEV2 was localized farther from the tumor than C-aHEV1, and formed a niche in conjunction with other HEV subclusters (HEV niche). The CFT niche-dense area and the HEV niche-dense area were often in close proximity, and the densities of CXCR4, encoding areceptor for CXCL12, and LAG3 and CXCR6, serving as T-cell exhaustion markers, were high in the CFT niche.
 SP analysis detected 6,172,072 cells and identified SCs containing 5 BEC components (artery, capillary, C-aHEV, HEV, and vein), tumor cells, and immune cell subtypes. C-aHEVs tended to be located closer to tumor cells than other BECs and formed a unified vascular niche in conjunction with HEV.
 WES analysis revealed the recurrent G17V RHOA mutations in 19 samples. Notably, samples harboring G17V RHOA mutations exhibited significantly higher densities of C-aHEV2 cells.
 Bulk RNA-seq analysis revealed that patients with high expression of C-aHEVs and other HEV-related signature exhibited poorer survival outcomes.
 Summary/ Conclusion By integrating scRNA-seq and spatial data, we identified C-aHEVs as key components in shaping the AITL microenvironment through the recruitment of immune cells. As the gene expression level, C-aHEVs can be subdivided into C-aHEV1 and C-aHEV2, which exhibited distinct properties. Further investigation is needed to validate the mechanisms by which these BEC clusters interact with tumor cells and contribute to tumorigenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/087d4af2ea5c2629eb9fea9276a3b1bcc2d2dd51" target='_blank'>
              Distinct blood endothelial cells shape spatial niches in angioimmunoblastic T-cell lymphoma
              </a>
            </td>
          <td>
            Yuki Shimizu, Manabu Fujisawa, Yoshiaki Abe, K. Makishima, S. Suma, Y. Suehara, K. Hattori, Tatsuhiro Sakamoto, Kyohei Sano, A. Tsubosaka, Miwako Kakiuchi, D. Kaji, Ayako Suzuki, Kensuke Usuki, Kosei Matsue, Yasunori Ota, Shigeru Chiba, S. Ishikawa, Yutaka Suzuki, Christian Steidl, M. Sakata-Yanagimoto
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Hematopoiesis occurs in the complex microenvironment of the bone marrow. This includes interactions between hematopoietic cells themselves as well as interactions with non-hematopoietic cells such as stromal cells, endothelial cells, adipocytes, bone cells, and neural cells, as well as extracellular matrix materials. Given this complex milieu, it is critical to study hematopoietic regulation in the context of its native tissue architecture. However, a lack of methods to simultaneously identify cell types at high granularity and examine their spatial organization, inter-cellular signaling, and cellular states has impeded efforts to further understand hematopoietic regulation in health and disease. Recently, a suite of spatial transcriptomic technologies has emerged enabling such an analysis. Among these, Multiplex Error Robust Fluorescence In Situ Hybridization (MERFISH) is a leading platform. MERFISH is a massively multiplexed single-molecule RNA imaging method built upon single-molecule fluorescence in situ hybridization (smFISH). MERFISH multiplexes smFISH by assigning to each RNA a unique binary barcode physically encoded into a library of nucleic acid encoding probes, which is read-out through sequential rounds of smFISH measurements on the same sample. Critically, MERFISH utilizes error-robust and error-correcting encoding methods which allow this technique to image hundreds to thousands of RNAs with a high detection efficiency, a low false positive rate, and a large dynamic range. MERFISH has been applied to multiple tissues such as the brain and intestine yielding novel insights into tissue organization and signaling. A major obstacle to applying MERFISH, or other RNA-based methods, to the blood system has been the high RNase activity of hematopoietic tissues, resulting in poor RNA integrity. Here we present a new method of bone marrow preparation that prevents RNA degradation and enables high quality bone marrow MERFISH data acquisition. With this method and a 1,022 gene library we spatially mapped ~1.5 million cells from 28 mouse femoral cross sections, identifying 53 distinct cell clusters. This dataset embeds well with published dissociated single cell RNAseq datasets, verifying capture of all expected cell types. Importantly, it also includes cell types that do not dissociate well such as mesenchymal stromal cells, osteoclasts, eosinophils, mature megakaryocytes, and rare hematopoietic stem cells. With this dataset, we quantified the frequency and distance of all cell-cell contacts and find that bone marrow microenvironments are well described by a statistical model of cell contact frequency. We find that two cell types—sinusoidal endothelial cells (SECs) and mesenchymal stromal cells (MSCs)—exist in a spectrum of transcriptional states that can be defined by a pseudotime axis. The spatial intersection of high pseudotime SECs and MSCs coincides with HSC niches, indicating that specific signaling from a subset of these cells facilitates hematopoiesis. Conversely, the intersection of low pseudotime SECs and MSCs coincides with increased density of natural killer, dendritic, T-, and plasma B-cells, indicating that subsets of these cells facilitate immune microenvironments. To understand the factors that regulate these microenvironments, we interrogated our MERFISH measurements and integrated scRNAseq datasets and find a series of candidate factors secreted by MSCs and SECs that may facilitate bone marrow niche formation. To understand the restructuring of bone marrow during aging, we compared MERFISH measurements of femoral cross sections from 8-, 40-, and 90-week mice. In aged mice, we find a structural breakdown of hematopoietic organization correlating with mouse age. These results are consistent with age-associated chronic inflammation and provide a step toward understanding aging related bone marrow dysfunction. Overall, this study provides new insights into bone marrow single cell spatial organization and a new tool that should greatly facilitate further understanding bone marrow niches in health and disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638434b03a57ff13fc4e0f106290fada913449e3" target='_blank'>
              Single cell spatial transcriptomics reveals age-dependent cellular organization of mouse bone marrow
              </a>
            </td>
          <td>
            Christian Reardon-Lochbaum, Paolo Cadinu, Hao Zhang, Jeffrey R. Moffitt, Alan B. Cantor
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="HPV status is a key determinant of prognosis and treatment response in head and neck squamous cell carcinoma (HNSCC). To investigate how HPV influences the tumor-immune-stromal landscape, we performed high-dimensional spatial profiling, including its impact on spatial organization, tertiary lymphoid structures (TLSs), and spatially organized cellular neighborhoods. Tumor biopsies from HNSCC patients (n = 16; 7 HPV-positive, 9 HPV-negative) were stained with a multiplex immunofluorescence (mIF) panel focused on immune profiling. A deep learning-based analysis pipeline enabled the identification and phenotypic state profiling of 14 cell types. Tissues were segmented into four distinct tumor regions, and spatial neighborhoods and TLSs were identified and analyzed for differential cellular composition, activation states, and spatial interactions between HPV-positive and HPV-negative tumors. HPV-positive and HPV-negative tumors differ in their tumor microenvironment (TME) composition, tumor cell state and spatial organization. The TME of HPV-positive tumors exhibited a greater abundance of activated lymphocytes, B- and T-cell-enriched spatial neighborhoods, and PD-1–PD-L1 interactions within the tumor area, whereas HPV-negative tumors were dominated by fibroblast- and macrophage-rich niches. T- cells in HPV-positive tumors showed greater activation across neighborhoods and areas, while in HPV-negative tumors T- cells demonstrated enrichment of exhaustion and terminal differentiation markers such as PD-1 and CD57. HPV-positive tumor cells had increased IDO1, HLA-DR, and Ki67 positivity, whereas HPV-negative tumor cells were more frequently CD44 positive, reflecting a more stem-like phenotype. Importantly, TLSs in HPV-positive tumors were located closer to the tumor area and enriched in activated immune cells, including ICOS+ CD4 T- cells, memory T- cells, and CD21+ B- cells. In contrast, TLSs in HPV-negative tumors were more distant and enriched for immunosuppressive populations such as PD-1+/PD-L1+ Tregs and macrophages. HPV status defines distinct spatial immune architectures in HNSCC. HPV-positive tumors harbor immune-activating TLSs and cellular neighborhoods that support antitumor immunity, whereas HPV-negative tumors exhibit suppressive niches and stromal dominance. These findings highlight TLSs, particularly their proximity and composition, as key features of the HPV-stratified TME and potential biomarkers for immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec5a252b8e081f817dffff9e37e6b264cbbcf5" target='_blank'>
              Spatial profiling of HPV-stratified head and neck squamous cell carcinoma reveals distinct immune niches and microenvironmental architectures
              </a>
            </td>
          <td>
            E. Markovits, Dmytro Klymyshyn, Roni Froumine, Hailing Zong, M. Mints, Sangeetha Mahadevan, Kenneth Bloom, Jamie G. Bates, Gareth J. Thomas, Lauri Diehl, Oscar Puig, Abhishek Aggarwal
          </td>
          <td>2025-11-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Metastasis remains the principal cause of cancer-related mortality, yet its distribution across organs is far from random. Instead, tumor cells exhibit organotropism, a consistent preference for colonizing specific distant tissues, a phenomenon shaped by anatomical constraints, molecular crosstalk, and microenvironmental compatibility. Far beyond mere mechanical entrapment in vascular beds, metastatic dissemination reflects a coordinated interplay between tumor-intrinsic programs and organ-specific niches. Tumor-derived extracellular vesicles, cytokines, and matrix-remodeling enzymes actively precondition distant sites through pre-metastatic niche formation, creating permissive microenvironments primed for colonization. Simultaneously, tissue-specific immune landscapes, stromal compositions, and mechanical architectures determine the fate of disseminated tumor cells, whether they are eliminated, enter dormancy, or form macrometastases. Phenotypic plasticity, metabolic reprogramming, and immune evasion mechanisms equip subclones with the capacity to exploit these unique niches. Across cancer types, reproducible patterns of organotropic metastasis not only guide clinical surveillance and therapeutic stratification but also reveal vulnerabilities in the metastatic cascade. This review synthesizes emerging mechanistic insights across anatomical, immunological, and molecular domains to construct a comprehensive framework of organotropism, highlighting therapeutic opportunities to intercept metastasis at organ-specific checkpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7663dab540302726b6f7b6337c8bf27a6cf336" target='_blank'>
              Deciphering organotropism reveals therapeutic targets in metastasis
              </a>
            </td>
          <td>
            Jhommara Bautista, Andrés López-Cortés
          </td>
          <td>2025-10-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="Biliary tract cancer (BTC) is a highly heterogeneous and aggressive gastrointestinal malignancy, marked by a high mortality rate and limited treatment efficacy. The primary contributing factors include the absence of reliable early detection methods, the anatomical intricacy of the biliary system, the inherently aggressive tumor biology, and the restricted effectiveness of systemic therapies. A profound understanding of molecular characteristics and clinically relevant emerging biomarkers is essential for advancing BTC treatment strategies. Recent developments in single-cell multi-omics technologies have enabled the analysis of genetic, transcriptomic, proteomic, and metabolomic data at the single-cell resolution, thereby uncovering the heterogeneity and complexity of tumor biology. These techniques provide critical insights into the diversity of immune cell populations within the tumor microenvironment (TME) and offer novel perspectives on tumor progression and potential therapeutic interventions. While single-cell technologies have significantly advanced the study of solid tumors, their application in BTC remains nascent, with a paucity of comprehensive reviews. This review systematically integrates single-cell genomics, transcriptomics, and epigenomics data to construct a cross-omics molecular atlas of BTC. It highlights the utility of single-cell multi-omics technologies in elucidating tumor heterogeneity, microenvironment remodeling, and clonal evolution in biliary tumors, while thoroughly analyzing their implications for clinical outcomes. Furthermore, this review explores personalized treatment strategies informed by single-cell technologies and underscores the significance of these technologies as indispensable tools for unraveling the complexity of BTC and fostering mechanism-based therapeutic innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c48afe626e59e1df497d34aa627d14168d0c25d2" target='_blank'>
              Single-cell multi-omics in biliary tract cancers: decoding heterogeneity, microenvironment, and treatment strategies
              </a>
            </td>
          <td>
            Nannan Tang, Jiatong Li, Ao Gu, Meng‐Yao Li, Yingbin Liu
          </td>
          <td>2025-10-15</td>
          <td>Molecular Biomedicine</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cf856a29d7dc82e41e89d6c3e46ad0a67ee3c3a" target='_blank'>
              Cell-stereotyped DNA repair outcomes are widespread during genome editing
              </a>
            </td>
          <td>
            Moritz F. Schlapansky, Markus S. Schröder, Antonio J. Santinha, Tanja Rothgangl, Eleonora I Ioannidi, G. Cullot, Bohdan Lewków, G. C. Ortega, Ahmed Nouraiz, Dominic Mailänder, Louisa Selbert, Simon Lam, Aitana Egea, Jiyung J. Shin, Matteo Bordi, Mark DeWitt, Ana Gvozdenovic, Stephen P. Jackson, T. Schroeder, Helmuth Gehart, Gerald Schwank, Randall J. Platt, Jacob E. Corn
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 H3K27 altered diffuse midline gliomas (DMG) are aggressive brain tumors that arise centrally in the brain. These tumors are not amenable to surgery and no curative therapy exists. DMG exhibits profound heterogeneity, with tumor cells displaying dynamic and interchangeable cellular states that recapitulate aspects of normal development. In parallel, the tumor microenvironment (TME) is highly diverse, further contributing to the complexity of the disease. A better understanding of the DMG ecosystem is needed for therapeutic target identification. In this study, publicly available single cell RNA seq data of DMG (Jessa et al. Nat. Genet. 2022) were analyzed to define key molecular characteristics of DMG. Xenium spatial transcriptomics and multiplex immunohistochemistry were performed on FFPE patient samples collected at diagnosis to study the tumor and TME cells in their native spatial context. In DMG, both the glioma cellular composition and the heterogeneity of the TME varied based on the anatomical location of the tumor. Tumors located in the pons exhibited a high proportion of astrocyte-like glioma cells. In contrast, thalamic DMG showed a greater abundance of oligodendrocyte-like tumor cells, suggesting a regional bias in cellular differentiation states. Furthermore, the TME in thalamic tumors was notably enriched with immunotolerant macrophage populations, expressing molecules associated with an anti-inflammatory, pro-tumoral phenotype. With spatial profiling, distinct tumor niches were identified, showing heterogeneity among different samples and underscoring the complex and spatially organized nature of both tumor cell states and the surrounding microenvironment in DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/408581e61f93671e8e7b1c9fc7f6b7a6669a068a" target='_blank'>
              EPCO-24. DISSECTING THE ECOSYSTEM OF PEDIATRIC H3K27 ALTERED DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            Julie Morscio, Chiara Caprioli, C. González-Blas, Pierluigi Calò, K. Vandereyken, Thierry Voet, R. Sciot, Sandra Jacobs, F. De Smet
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Deep learning and large language models can integrate complex datasets to uncover biological insights that are often undetectable through conventional analyses. With application to translational cancer research, these computational tools have positioned 3D patient-derived tumor avatars front and center as crucial data input sources. However, a major challenge remains: the lack of standardization in media composition in 3D patient-derived tumor models unpredictably affects cell behavior and limit the utility beyond predicting treatment responses. To address this unmet need, we developed a simple, reproducible perfusion circuit system to approximate in vivo physiology using autologous patient plasma. With peritoneal metastases and core needle biopsies across multiple tumor histologies, we demonstrate preservation of the tumor microenvironment for up to 48 hours using multi-modal interrogation techniques. With proof-of-concept experiments, we display the system’s ability to unveil complex drug-dependent biology within this time window. Standardizable, physiologically relevant platforms for 3D patient-derived tumor avatars will yield unprecedented insights through the integration of data from broad groups of patients and the use of an expanding armamentarium of artificial intelligence capabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7a2b5548999bc3e5693c3a46fe302fa33927e3b" target='_blank'>
              A simple circuit to sustain intact tumor microenvironments for complex drug interrogations
              </a>
            </td>
          <td>
            A. Sarvestani, Samantha Ruff, K. Remmert, Martha E Teke, Emily A Verbus, A. Saif, Marcial Garmendia-Cedillos, Alex Rossi, Ashley Rainey, K. Nomie, Surajit Sinha, Michael M. Wach, Carrie E Ryan, Stephanie N. Gregory, Priyanka P Desai, A. Panda, Emily Smith, Reed Ayabe, James D McDonald, Shreya Gupta, Tracey Pu, Tahsin M. Khan, Jacob T Lambdin, K. Luberice, Stephanie C Lux, Hanna Hong, Imani A Alexander, Sarfraz R. Akmal, Shahyan Rehman, Sophia Xiao, Jason Ho, Justine F. Burke, C. Hannah, Alexandra Livanova, Andrey Tyshevich, Stanislav Kurpe, Andrey Kravets, Basim Salem, Daniil Vibe, Arina Tkachukv, Anna Belozerova, A. Sarachakov, Evgeny Barykin, V. Kushnarev, E. Postovalova, A. Bagaev, Noemi Kedei, Maria Hernandez, Baktiar Karim, Donna Butcher, Jennifer Matta, Michael Kelly, Jatinder Singh, Todd D Prickett, James W Hodge, T. Andresson, Michael B Yaffe, Ethan M Shevach, David E. Kleiner, Jeremy L. Davis, Andrew M. Blakely, Jonathan M. Hernandez
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Tumors are composed of cellular populations with distinct genotypes and phenotypes, which dynamically evolve over time and during treatment. This process is known as clonal evolution, and it is difficult to reveal fine-scale clonal structure with traditional bulk sequencing. Although single-cell genome sequencing could enable reconstruction of tumor clonal evolution, it remains technically challenging and the number of single cells profiled is generally insufficient due to high cost. To address this issue, we developed scClone, a computational toolkit that integrates variant detection and genotype inference for single-cell RNA-seq (scRNA-seq) and spatial transcriptomic data. It further provides interactive visualization of clonal structure and dynamic evolution. scClone addresses key limitations inherent to scRNA-seq, such as expression drop-out and allelic imbalance, and incorporates cell type or state annotation with mutational signature analysis to enable comprehensive profiling of tumor heterogeneity. scClone demonstrated robust performance across multiple datasets—generated from both full-length and fragmented RNA sequencing—by accurately reproducing mutation profiles and resolving clonal mixtures in myeloma, hepatocellular carcinoma and pancreatic cancer. Additionally, scClone has been applied to spatial transcriptomics, enabling the delineation of clonal structures within histological sections from ovarian cancer and cutaneous squamous cell carcinoma. In summary, our results demonstrate that scClone can extract genetic information from scRNA-seq datasets, thereby successfully establishing genotype–phenotype associations at the single-cell level and providing insights into the clonal evolution of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d470f131086238e06c10a786c9bfc7fb86a41116" target='_blank'>
              De Novo Detection of Clonal Structure and Evolution in Single-Cell and Spatial Transcriptomes
              </a>
            </td>
          <td>
            Shihao Bai, Xianbin Su, Ziyao Chen, Ze-Guang Han
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance.
              </a>
            </td>
          <td>
            , Si-Jia He, Xiao-yu Jing, , Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ab2b82b250c7fe0872534fbb6e35f7aaf3d36d" target='_blank'>
              Cell State Chaos Underpins the Evolution of SMARCA4-Deficient Dedifferentiated Endometrial Cancer
              </a>
            </td>
          <td>
            M. Brandon-Coatham, Julia Vassalakis, Hannah Plummer, Jasleen Kaur, Farzaneh Afzali, Zhihua Xu, Guihua Zhang, Jiahui Liu, Wei Wang, Einav Wajsbrot Renert, Tyler T. Cooper, Alan Dimitriev, Cierra Perron, Andrew Garven, Simatsidk Haregu Abebe, David M Berman, Amber L Simpson, G. Wong, Gilles A. Lajoie, Felix K.F. Kommos, Andreas von Diemlin, Sheela A. Abraham, David J.H.F. Knapp, Alan Underhill, M. Koebel, Cheng Han-Lee, Lynne-Marie Postovit
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02530b52b1f076b640bb9d3473208058e5838dbf" target='_blank'>
              Self-organization of tumor heterogeneity and plasticity
              </a>
            </td>
          <td>
            Carlos Pérez-González, D. Brückner, M. Di-Luoffo, Sophie Richon, R. Goswami, Meryem B Baghdadi, Florence Piastra-Facon, Neta Felsenthal, Réda Bouras, A. Fumagalli, Mirjam C van der Net, M. Gloerich, Salvatore Girardo, Jochen Guck, J. van Rheenen, J. Guillermet-Guibert, É. Hannezo, D. Vignjevic
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Spatial tumour heterogeneity, which denotes the changes in cellular and molecular attributes across distinct locations within a tumour, significantly influences cancer diagnosis and treatment resistance. The heterogeneity of tumour cells inside a singular mass facilitates tumour development, metastasis, and the ineffectiveness of standard therapy. Comprehending the geographical distribution of tumour cells is crucial for formulating more efficient treatment regimens. Diverse methodologies are employed to investigate spatial heterogeneity, encompassing modern imaging techniques such as MRI, PET, and multiplexed imaging, alongside omics approaches including genomes, transcriptomics, and proteomics. These instruments offer insights into the tumour microenvironment and facilitate the identification of resistant subpopulations. The amalgamation of imaging and genomic data via radiogenomics has emerged as a viable methodology, providing an extensive perspective on the spatial and molecular intricacies of tumours. Principal findings reveal that spatial heterogeneity fosters medication resistance by establishing microenvironments characterised by varying oxygen levels, immunological infiltration, and genetic alterations, hence complicating the efficacy of monotherapy strategies. Hypoxic environments and immunological evasion significantly contribute to treatment resistance. Addressing geographical heterogeneity has the potential to enhance cancer treatments. By analysing the molecular and geographical characteristics of tumours, physicians can customise therapies more efficiently, minimising resistance and improving therapeutic results. This methodology signifies a vital advancement in precision medicine, providing more individualised and efficacious cancer therapies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/691858c0b45e6f3db66c28fcbc5355f8258e6fea" target='_blank'>
              Spatial Tumor Heterogeneity: The Next Frontier in Understanding Cancer Resistance
              </a>
            </td>
          <td>
            Kwesi Egyin Taylor, Hycent Jacob, T. Oladosu, Godwin Kenechukwu Nwajiugo, Motunrayo Victoria Adigun, Markus Saerimam Nzunde, C. Ugo
          </td>
          <td>2025-10-18</td>
          <td>Oncology, Nuclear Medicine and Transplantology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d38817463882bea59fae7ae417ac7b28c60ff93" target='_blank'>
              Multi-omics evaluation of cell lines as models for metastatic prostate cancer
              </a>
            </td>
          <td>
            Xueying Liu, Weixing Yu, Xiuyuan Jin, Yugang Wang, Ke Liu
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The diffusely infiltrative growth pattern of glioblastoma (GBM) is a major obstacle to effective therapy. A complex network of tumor intrinsic and microenvironmental features interact to shape the landscape of this growth pattern, but deciphering the molecular substrates of these interactions remains challenging. While single cell RNA sequencing has reshaped our understanding of GBM heterogeneity and uncovered cellular phenotypes of many cellular populations in these tumors, the spatial context of this heterogeneity could not be assessed with earlier technologies. By contrast, spatial transcriptomics platforms have the potential to map this transcriptome-wide information in situ. We recently reported the use of this technology to develop a layered model of spatial architecture in GBM and showed that hypoxia represents a key organizing feature of GBM heterogeneity. However, the limited resolution of this platform precluded assessment of whole-transcriptome features at truly cellular resolution. In this work, we now present our findings utilizing the next generation of this technology and showcase the ability to map whole transcriptome data in spatial context at true single cell resolution in primary patient GBM samples. We explore heterogeneous GBM phenotypes in the context of GBM spatial microarchitecture. Using copy number inference, we demonstrate the ability to assess clonal heterogeneity in situ and map the pattern of invasion of unique GBM clones. Finally, we discuss our progress to discover organizing principles underlying distinct patterns of GBM cell invasion. Taken together, we demonstrate the potential of single cell spatial transcriptomics to decipher the molecular heterogeneity of GBM microenvironment in situ and uncover the biology driving the aggressive infiltrative growth of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad815e1440b141545b8a8eee62737b92edef994" target='_blank'>
              TMIC-68. Deciphering the spatial architecture of transcriptional heterogeneity in glioblastoma at single cell resolution
              </a>
            </td>
          <td>
            C. Mount, Zebin Wen, Alissa C. Greenwald, S. Kovatsis, Jack Lu, D. Gerovasilis, I. Tirosh, M. Suvà
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Current cancer therapies often fail due to a highly immunosuppressive tumor microenvironment (TME), which impedes immune cells from effectively targeting and eliminating cancer cells. Tumor-associated macrophages (TAMs) have been shown to compose up to 50% of a tumor’s mass and have been shown to exist on a polarization spectrum from anti-tumoral to pro-tumoral depending on the direct and indirect effects they have on tumors. The objective of this project is to establish pathways that control TAM tumor phenotypes. To study which TAM pathways are most amenable to manipulation to drive anti-tumor efficacy, we utilized a multi-omics approach which combines single cell RNA sequencing (scRNA-seq), spatial transcriptomics, and imaging mass spectrometry. We integrated publicly available datasets with datasets we developed from our mouse model of orthotopic pancreatic ductal adenocarcinoma (PDAC). These techniques allowed the identification of alterations in TAM metabolic networks associated with markers known to be a part of macrophage reprogramming such as Arg1, Spp1, and C1q, but the mechanism of how this shift can drive the phenotype is still to be elucidated. Future work will focus on identifying key genes involved in TAM metabolic reprogramming and assessing whether targeting these genes can modulate tumor growth by altering TAM activity. Ultimately, this research seeks to define actionable pathways that can be manipulated to enhance TAM-mediated anti-tumor responses.



 NIH K00CA274682-03



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d07758d1aaf12c832cc03f655246443f1a51f16" target='_blank'>
              Characterization of anti-tumor networks in tumor-associated macrophages 2377
              </a>
            </td>
          <td>
            Stephen Sakuma, Ming Sun, S. Kaech
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype; however, clinically approved prognostic biomarkers and therapeutic options remain limited. This study aimed to investigate tumor dependency genes to identify novel therapeutic targets for TNBC. Methods Tumor dependency genes for TNBC were identified using the The Cancer Dependency Map (DEPMAP) database. The TCGA-BRCA dataset was utilized to analyze the expression, survival associations, and pathway enrichment of these genes. Single-cell datasets were employed to explore cellular trajectories and biological functions within tumor dependency gene-associated cell subpopulations. Genomic sequencing was used to investigate the somatic mutational landscape influencing the infiltration abundance of the tumor dependency-associated subpopulation. The METABRIC dataset assessed the impact of the tumor dependency-associated subpopulation on radiotherapy, chemotherapy, and combination therapy outcomes. Potential drugs were identified using the Connectivity Map (CMAP). Colony formation experiment and the CCK-8 experiment were performed to validate the biological function of gene. Results Four tumor dependency genes (TDGs) were identified. These genes were highly expressed in TNBC and associated with poor prognosis. Enrichment analysis revealed their significant involvement in cell cycle-related pathways. Single-cell analysis demonstrated that the tumor dependency-associated subpopulation (TDAS), defined by these four genes, resided at the differentiation terminus of epithelial/tumor cells and was linked to energy metabolism and cell proliferation pathways. Crucially, patients with high TDAS infiltration abundance were found to be unsuitable for surgery alone and should receive combined radiotherapy or chemotherapy. Potential therapeutic agents targeting the TDAS were screened. And in vitro experiments confirmed the cell proliferation role of candidate genes. Conclusion This study identifies four potential TNBC biomarkers for assessing TDAS abundance, providing novel insights and strategies for personalized TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c8aa1f75b0d7e0c49aabd5f19b6bf7360c1cf" target='_blank'>
              Integrating single-cell transcriptomics and whole-genome CRISPR CAS9 screen identifies a cell cluster associated with tumor dependency in triple-negative breast cancer
              </a>
            </td>
          <td>
            Su Liu, Shuo Wang, Guixin Wang, Yingxi Li, Zhigang Zhao, Yao Tian, Jun Cao
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) poses significant challenges to precision oncology due to its pronounced molecular heterogeneity and complex tumor microenvironment (TME). Traditional two-dimensional (2D) cultures and animal models fall short in capturing patient-specific tumor biology, limiting their translational relevance. In contrast, three-dimensional (3D) organoid platforms offer structurally and functionally representative models that retain key pathological and pharmacological features of RCC. Notably, RCC organoids enable pharmacokinetic assessment, including drug penetration, metabolic activation, and off-target toxicity in a spatially organized context. This review outlines current strategies for RCC organoid construction-including patient-derived organoids (PDOs), air–liquid interface (ALI) cultures, scaffold-based matrices, and microfluidic systems-and evaluates their applications in drug screening, resistance modeling, and immunotherapy prediction. We further discuss technical and biological limitations such as phenotypic drift, inter-sample variability, and TME reconstruction, alongside emerging solutions in synthetic scaffolds, immune co-cultures, and multi-omics integration. In conclusion, RCC organoids are rapidly evolving into clinically actionable platforms, offering a scalable and predictive approach to personalized therapy in renal oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec8f5e11dac4c8c235f8ea9f0538afb3e72cf307" target='_blank'>
              Renal cell carcinoma organoids for precision medicine: bridging the gap between models and patients
              </a>
            </td>
          <td>
            Jian Gao, Huijiu Luo, Shiyu Wang, Chao Zhou, Zhenhao Li, Chea Kimhoy, Guobiao Liang, Shulian Chen
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a formidable clinical challenge due to therapy resistance, metastasis, and relapse. Central to these processes are colorectal cancer stem cells (CCSCs), a dynamic subpopulation endowed with self-renewal capacity, plasticity, and heterogeneity. This review synthesizes recent advancements in understanding how CCSCs orchestrate tumor progression through intricate bidirectional crosstalk with the tumor immune microenvironment (TIME). We begin by elucidating the cellular origins of CCSCs, their profound intratumoral heterogeneity, and their remarkable phenotypic plasticity—driven by genetic, epigenetic, and metabolic reprogramming—which collectively serve as the root cause of therapeutic failure. A significant portion of our discussion is dedicated to deconstructing the immunosuppressive niche co-opted by CCSCs. We detail mechanisms of immune evasion and tolerance, highlighting how CCSCs modulate innate and adaptive immune cells—including NK cells, Tregs, dendritic cells, macrophages, neutrophils, and myeloid-derived suppressor cells—to foster an environment that supports stemness and suppresses cytotoxic attack. This reciprocal interaction forms a vicious cycle that perpetuates tumor survival and progression. Finally, we critically evaluate emerging therapeutic strategies that concurrently target CCSC-specific vulnerabilities and counteract immunosuppression. We explore the limitations of conventional chemotherapy and the promise of targeted therapies (e.g., Wnt inhibitors), immunotherapies (e.g., CAR-T, bispecific antibodies), and combination regimens designed to remodel the TIME and eradicate the CCSC reservoir. By integrating insights from single-cell omics and spatial biology, this review provides a comprehensive framework for overcoming therapy resistance and proposes novel precision medicine approaches for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b2fada714432c2d56f0013d3edefb668b69269" target='_blank'>
              Colorectal cancer stem cells crosstalk in tumor immune microenvironment and targeted therapeutic strategies
              </a>
            </td>
          <td>
            Yuxuan Duan, Anshu Li, Daojia Miao, Diaoyi Tan, Keshan Wang, Jian Shi
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01df2c37dabe824163d0ae6c9689244efcafc8f6" target='_blank'>
              Multi-scale Network Topology Analysis Reveals Metabolic Reprogramming and Therapeutic Vulnerabilities in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Handan Cetin, P. Bogdan, S. Mumenthaler, Stacey D. Finley
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cancer‐associated fibroblasts (CAFs) are pivotal stromal components that shape the tumor microenvironment through their remarkable heterogeneity and dynamic functions. This review comprehensively examines the biology of CAFs, starting with their diverse cellular origins and induction mechanisms via key signaling pathways. We highlight the spatial‐temporal heterogeneity of CAF subsets and propose an updated classification system that encompasses eight functionally distinct subtypes based on recent single‐cell transcriptomic studies. The tumor‐promoting roles of CAFs are explored in depth, including their contributions to tumor behavior through novel mechanisms such as neutrophil extracellular traps formation, and multidrug resistance via metabolic reprogramming. A major focus is placed on CAF‐mediated immunotherapy resistance through immune checkpoint regulation, recruitment of immunosuppressive cells, and metabolite secretion. This review also highlights key technologies that are advancing CAF research, including patient‐derived 3D models, artificial intelligence‐enhanced spatial multiomics, clustered regularly interspaced short palindromic repeats (CRISPR)‐engineered mouse systems, and machine learning approaches for analyzing CAF heterogeneity and function. Finally, we evaluate therapeutic strategies targeting CAF activation, secreted factors, and immune crosstalk, while critically analyzing ongoing clinical trials. Overall, this review not only clarifies the biology of CAFs but also provides a translational framework for developing next‐generation stromal‐targeted therapies to overcome treatment resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1dae86be5576422570b1acdcb11b97738bdec4b" target='_blank'>
              Cancer‐Associated Fibroblasts: Origin, Classification, Tumorigenicity, and Targeting for Cancer Therapy
              </a>
            </td>
          <td>
            Chang Fan, Wenlong Zhu, Yuan Chen, Wanwan Zhu, Jin Ding
          </td>
          <td>2025-10-28</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy, poised to become the second leading cause of cancer-related deaths in North America. The paucity of neoantigen presentation in PDAC has been hypothesized to promote the development of an “immunologically-cold” tumor microenvironment (TME), driving poor clinical responses to immunotherapy. Despite the abundant immune infiltrate observed in PDAC, T cells display a limited cytotoxic capacity to kill malignant tumor cells which further diminishes throughout disease progression. Utilizing single cell RNA-sequencing coupled to T cell receptor profiling, we have characterized T cell populations in clinical specimens of inflammatory pre-malignant intraductal papillary mucinous neoplasia (IPMN) lesions and advanced stage PDAC tumors. Striking phenotypic and clonal dynamic differences were observed in both CD8 and CD4 T cell populations between IPMN and PDAC specimens which underly their distinct functional capabilities in the evolving TME. The spatial interactions and microenvironmental neighborhoods of neoantigen-reactive T cells in PDAC have yet to be explored in-depth and hold unexploited therapeutic potential. Utilizing the 10X Genomics Xenium platform with custom probes designed to hybridize with neo-antigen-specific T cell clones, we are interrogating the cellular interactions which drive T cell clonal expansion to identify therapeutically relevant vulnerabilities in PDAC malignant progression.



 Supported by National Insitutes of Health, Canadian Institutes of Health Research, and Princess Margaret Cancer Center Catalyst Grant.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f6400ccbc9c89325532880266eae31b90968879" target='_blank'>
              Defining molecular determinants of T cell spatial localization and clonal response throughout PDAC malignant progression 3030
              </a>
            </td>
          <td>
            Matthew Bianca, Sara Lamorte, Teresa Ciudad, Cristiane Naffah de Souza, Rene Quevedo, Tracy Mcgaha
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29217e0d8ddcbbf053576ea9783b9b547cccc698" target='_blank'>
              Cell surface vimentin-positive circulating tumor cells: developments, and clinical applications
              </a>
            </td>
          <td>
            Jie Zhong, Meiling Du, Hongyi Yi, Yu-Peng Zheng, Mian Yang, Jiazi Yu
          </td>
          <td>2025-12-02</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01b5dde4ed4b216cb7cf78338e646166d480ca06" target='_blank'>
              Spatial proteomics of ovarian cancer precursors delineates early disease changes and drug targets.
              </a>
            </td>
          <td>
            Anuar Makhmut, M. Dragomir, Sonja Fritzsche, M. Moebs, W. Schmitt, Eliane T. Taube, Fabian Coscia
          </td>
          <td>2025-11-13</td>
          <td>Molecular systems biology</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="Cancer cell plasticity, the dynamic ability of tumor cells to switch phenotypic states, drives tumor progression, metastasis, and therapy resistance, posing a formidable challenge to effective cancer management. This review comprehensively explores the genetic and epigenetic mechanisms underlying plasticity, highlighting how oncogenic mutations, chromosomal instability, and pathways like PI3K/AKT and RAS/MAPK enable adaptive behaviors such as epithelial-mesenchymal transition and stemness. Epigenetic regulators, including DNA methylation, histone modifications, and non-coding RNAs, amplify this adaptability, allowing rapid responses to therapeutic pressures. We examine the profound clinical implications, with plasticity fueling relapse in breast, lung, and prostate cancers, exacerbated by tumor microenvironment interactions. Challenges in detecting plastic states due to intratumor heterogeneity underscore the need for advanced diagnostics like single-cell genomics. Emerging interventions, from CRISPR-based gene editing and epigenetic therapies to patient-derived organoids and AI-driven combination strategies, offer transformative potential to disrupt plasticity and overcome resistance. This review illuminates the intricate genetic landscape of cancer cell plasticity, advocating for integrated approaches to revolutionize personalized oncology and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56a74b7b33fc882b2d862736314940182017fc5" target='_blank'>
              The Genetic Landscape of Cancer Cell Plasticity: Mechanisms, Implications for Therapy Resistance and Emerging Interventions
              </a>
            </td>
          <td>
            I. N. Monago, Nyerovwo Charity Okei, Chibuike Stephen Nzereogu, Taiwo Emmanuel Olowodasa, Tamasanni Omeiza Salawu, Ahmad Muhammad Garba, Doris Ada Ojum
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Biological and Pharmaceutical Sciences Archive</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Introduction Leiomyosarcoma (LMS) is a rare, aggressive cancer with limited treatment options at the metastatic stage. The response to immune checkpoint inhibitors (ICIs) is inconsistent, likely due to intratumoral heterogeneity, which is more pronounced in large tumours such as retroperitoneal LMS. Methods This study examined heterogeneity in four large treatment-naive LMS tumours (ten samples per tumour) by analysing immune cells, tertiary lymphoid structures (TLSs), checkpoint molecules, and cytokine secretion across different tumour regions. Results Significant region-dependent differences were observed in immune components, with TLSs present only at tumour margins and inconsistently across samples from the same tumour. Expression levels of programmed cell death 1 (PD-1) and programmed death ligand 1 (PD-L1) varied within individual tumours, and shared immune patterns were identified in specific regions, including elevated indoleamine 2,3-dioxygenase 1, absence of a particular macrophage subpopulation, and reduced PD-1 and lymphocyte activation gene 3 (LAG-3) expression at organ-adjacent margins. Anti-LAG-3 blockade altered cytokine and checkpoint molecule levels in a region-specific manner. Conclusion These findings highlight substantial intratumoral heterogeneity, which may contribute to the variable response to ICI therapy. As immune checkpoint molecule expression influences treatment eligibility, multiple biopsies from different tumour regions may be necessary to assess immune infiltration accurately and guide therapy decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f47a854ab32ca2eceb50d1b8d1469818b11328c" target='_blank'>
              Spatial immunoprofiling of retroperitoneal leiomyosarcomas reveals intratumoral heterogeneity in immune cell infiltration, checkpoint molecule expression, and tertiary lymphoid structures
              </a>
            </td>
          <td>
            Iva Benesova, Jan Balko, V. Tovazhnianska, M. Rataj, R. Lischke, J. Bartůňková, K. Kopeckova, T. Buchler, W. V. van Houdt, Yvonne Schrage, David Moura, J. Broto, Zuzana Ozaniak Střížová, A. Ozaniak
          </td>
          <td>2025-10-13</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The tumor immune microenvironment plays a critical role in tumor progression and responses to immunotherapy. Nevertheless, its cellular complexity and heterogeneity remain incompletely understood. In this study, we employed high-resolution single-cell RNA sequencing on CD45+ immune cells isolated from ten syngeneic murine tumor models, representing seven distinct cancer types under treatment-naïve conditions, thereby enabling a comprehensive profiling of tumor-infiltrating immune cells. We identified seven principal immune cell populations and provided an in-depth characterization of T cells, NK/innate lymphoid cells, dendritic cells, monocytes/macrophages, and neutrophils. Cross-species analyses further delineated conserved immune cell states and transcriptomic features within the T cell and monocyte/macrophage compartments that are shared across syngeneic models and human tumors. To investigate the functional relevance of the predominant monocyte/macrophage compartment and the notable presence of neutrophils in syngeneic tumors, we evaluated responses to anti-PD-1 therapy across various models and analyzed the enrichment of monocyte/macrophage subsets in tumors that responded to treatment. Furthermore, we conducted neutrophil depletion experiments using anti-Ly6G antibodies, administered both as monotherapy and in combination with PD-1 blockade. Remarkably, an interferon-stimulated gene-high (ISGhigh) monocyte subset was significantly enriched in models responsive to anti-PD-1 therapy. Neutrophil depletion resulted in variable antitumor effects across models but failed to enhance the efficacy of PD-1 blockade. In summary, our single-cell profiling offered a detailed atlas of the immune microenvironment across multiple syngeneic mouse tumor models, thereby enabling rational model selection for immuno-oncology studies. We uncovered an ISGhigh monocyte subset enriched in anti-PD-1 responsive models, and showed the context-dependent effects of neutrophil depletion on tumor immunity and immunotherapy, underscoring the heterogeneity and functional divergence of immune cell sublineages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f925e5b4fe351a72672a38a0505d78e0be84e52" target='_blank'>
              Single-cell atlas of the tumor immune microenvironment across syngeneic murine models
              </a>
            </td>
          <td>
            Jia Wang, Bin Jiang, Minjuan Deng, Han Yan, Peilin Zhang, Wei Jin, Zhirong Shen
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05c188c0c6e2f128d8772ff9b6be6a1353be8fc1" target='_blank'>
              A Hybrid Multiscale Model for Predicting CAR-T Therapy Outcomes in Solid Tumors
              </a>
            </td>
          <td>
            M. Nikmaneshi, Lance L. Munn
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Endometrial cancer is regarded as one of the most prevalent malignancies in women globally. Despite the advancements brought by multiple therapeutic strategies, the efficacy and effectiveness of treatment still appear to be diminished. Therefore, this accentuates the importance of understanding the tumour microenvironment (TME). Conventional methods, including immunohistochemistry (IHC), along with recently emerged techniques like single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST), have rejuvenated the notion of deciphering TME. From deconstructing tumour heterogeneity, identifying cell populations that play significant roles in treatment response, to discovering key biomarkers and therapeutic targets, these technologies lay the base for innovations. Importantly, this is the first comprehensive review that brings together scRNA-seq, ST, and IHC in Endometrial cancer. Merging these methods is geared towards creating a better grasp of the dynamics and interactions between diverse cells and the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bef0a0a3a7a3aad177138a9aca3da36e10bf85b" target='_blank'>
              Navigating tumour microenvironment in endometrial carcinoma: a comprehensive review integrating immunohistochemistry, single-cell RNA-sequencing and spatial transcriptomics
              </a>
            </td>
          <td>
            Oishee Mondal, Blessy Kiruba, Sajitha Lulu Sudhakaran, V. Sundararajan
          </td>
          <td>2025-11-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The spatial organization of the tumor microenvironment (TME) encodes the selective pressures that drive cancer evolution, but many tools focus on local cell contacts or single-modality readouts. We developed Tumor Landscape Analysis (TLA), a modular framework that borrows from landscape ecology and spatial statistics to quantify heterogeneity, fragmentation, and diversity across scales, from single cells to whole-slide ecosystems.



 TLA ingests whole-cell or nuclear centroids and region-level segmentations to compute (i) point statistics such as nearest-neighbor distance, Ripley’s H, and Morisita-Horn colocalization; (ii) kernel-smoothed local spatial profiles including Getis-Ord Gi* hot and cold spots; and (iii) fragmentation, shape, and diversity metrics such as patch density, contagion, shape index, and Shannon entropy. We also introduce Local Microenvironments (LMEs), unsupervised ecological niches defined by joint cell-type abundance and spatial mixing. Pilot analyses were performed in ductal carcinoma in situ (DCIS; n=353 samples, 175 patients) with cell-level maps of epithelial, lymphocyte, and fibroblast populations, and in pancreatic ductal adenocarcinoma (PDAC; n=203 resected cases after neoadjuvant therapy) using cancer and stroma segmentations.



 In DCIS, spatial relationships rather than raw abundance associated with recurrence risk. Shorter lymphocyte-to-fibroblast minimum nearest-neighbor distance and lower Ripley’s H, indicating greater intermixing, correlated with earlier recurrence independent of cell counts. This highlights immune–stromal proximity as a candidate eco-evolutionary driver of persistence. In PDAC, tissue-level fragmentation captured prognostic architecture after therapy. Two shape-based signatures emerged: (1) stromal uniformity, where lower standard deviation of stromal patch shape index associated with improved disease-free survival, consistent with a more constrained, less reactive stroma; and (2) tumor geometric complexity, where higher mean cancer patch shape index, reflecting more irregular or dendritic nests, associated with lower recurrence risk, whereas compact nests predicted worse outcomes. Across datasets, LME maps localized niche phenotypes, such as mixed-immune or segmented tumor habitats, that align with hypothesized selection landscapes and provide inputs for fragmentation analysis.



 TLA operationalizes an eco-evolutionary view of cancer by unifying interpretable, multiscale spatial metrics in a single workflow that works with any rasterized data input. By quantifying how habitats, edges, and mixing patterns are organized, TLA reveals niche structures and architectural motifs linked to clinical outcomes and complements foundation models and modality-specific pipelines with theory-grounded, clinically translatable descriptors. Ongoing work integrates TLA features with molecular data and outcomes to derive robust biomarkers and to inform adaptive, niche-targeted strategies in precision oncology.



 Ryan M. Carr, Luis Cisneros, Merih Deniz. Toruner, Martin E. Fernandez-Zapico, Carlo Maley. Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f5d8ed56157756f83dcf8a730cae9e8b632c46" target='_blank'>
              Abstract A015: Tumor landscape analysis: An ecologically informed framework to understand tumor microenvironments
              </a>
            </td>
          <td>
            Ryan M. Carr, L. Cisneros, M. D. Toruner, Martín E Fernandez-Zapico, Carlo C. Maley
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pleomorphic adenoma (PA), the most common benign salivary gland tumor, harbors unpredictable risks of recurrence and malignant transformation into carcinoma ex pleomorphic adenoma (CXPA), posing significant clinical challenges. To better delineate the tumor transformation trajectory, we performed single-cell RNA sequencing of normal salivary gland, primary PA, recurrent PA (rPA), and CXPA. Cell trajectory reconstruction, differential expression gene identification, and key gene network analysis were integrated to characterize molecular transitions and intercellular crosstalk driving PA recurrence and malignant transformation. Immunohistochemistry was used to validate key findings. GALNT13 + myoepithelial cells were identified as CXPA-specific malignant progenitors, delineating early malignant conversion. Concurrently, MIF + myoepithelial cells exhibited enhanced tissue-destructive capabilities. Fibroblasts enforced fibrotic restraint in primary PA and drove extracellular matrix degradation in CXPA. The tumor microenvironment exhibited stage-specific adaptations, with CXPA favoring pro-inflammatory MIF-CD74/CD44 signaling and rPA adopting immunosuppressive traits. Stromal reprogramming and immune-editing dynamics collectively orchestrated microenvironmental adaptation, linking cellular heterogeneity to clinical aggressiveness. This study provides the first comprehensive molecular atlas of PA-to-CXPA transformation, revealing malignant specialization of the myoepithelial subpopulation, fibroblast-mediated stromal reprogramming, and immune-editing driven microenvironmental adaptation. These findings provide a framework for precision stratification of the malignant potential of PA, while positioning microenvironmental intervention as a cornerstone of future clinical strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/110371b54a7af37b631e968b70c953daf9f4053b" target='_blank'>
              Single-cell transcriptomics reveals cellular evolution underlying pleomorphic adenoma recurrence and malignant transformation
              </a>
            </td>
          <td>
            Yike Li, Zhixuan Li, Guile Zhao, Qi Han, Zhangfan Ding, Cheng Wang, Chunjie Li
          </td>
          <td>2025-10-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Melanoma shows heterogeneity across body sites like skin, acral skin, and the uvea, driven by molecular characteristics and genetic variations. However, comparative studies exploring the heterogeneity of melanoma across different anatomical sites remain limited, hindering a comprehensive understanding of its underlying biology. We proposed a research framework through bioinformatics to analyze the tumor ecosystems of cutaneous, acral, and uveal melanoma, from molecular characteristics to genetic variations at single-cell resolution. We found that oxidative phosphorylation (OXPHOS) is a critical driver of tumor cell evolution, with abnormal ribosomal gene and tumor suppressor expression observed in uveal melanoma (UM). Additionally, we screened for potential drug targets and drugs against tumor cells. In the immune microenvironment, acral melanoma (AM) and UM exhibit stronger immunosuppressive characteristics compared to cutaneous melanoma (CM). OXPHOS contributes to T cell cytotoxicity dysregulation in CM and AM, while interferon-γ is crucial in UM. Tumor cells may also induce T cell dysfunction through biological signals such as MIF-CD74 and HLA-E-NKG2A. This study offers valuable insights into melanoma heterogeneity, providing a comprehensive research framework for understanding the distinct molecular and immune characteristics of CM, AM, and UM, and potentially guiding the development of therapeutic strategies tailored to each melanoma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da658ce35ad37afa8afa0fc4adb1c52869ddb7f4" target='_blank'>
              Dissecting Melanoma Ecosystem Heterogeneity from Molecular Characteristics to Genetic Variation at Single-Cell Resolution
              </a>
            </td>
          <td>
            Congxue Hu, Liyuan Li, Tengyue Li, Baobin Qi, Wanqi Mi, He Yu, Kaiyue Yang, Qi Ou, Xia Li, Yunpeng Zhang
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566fc9fd9c1a3c438b8fdc4f08cbb7f2ae09af61" target='_blank'>
              Joint analysis of expression and variation at single cell resolution by scVar
              </a>
            </td>
          <td>
            Wei Zhao, Fei Yang, Wenting Zong, Xinchang Zheng, Yiming Bao
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous malignancy, with its tumor immune microenvironment (TIME) playing a crucial role in tumor progression, immune evasion, and treatment response. Despite insights from single-cell technologies, challenges such as small samples and inconsistent annotations have limited a comprehensive understanding. Thus, a large-scale, high-resolution single-cell atlas is needed to fully explore the heterogeneity of CRC TIME and guide personalized treatment. We analyzed single-cell transcriptomic data from 100 CRC samples (371,223 cells). Immune, stromal, and epithelial cells in the TIME were clustered, annotated, and functionally analyzed. Transcription factor networks were inferred using SCENIC, differentiation states assessed with CytoTRACE, and cell–cell communication analyzed via CellPhoneDB. TIME subtypes were constructed from bulk RNA-seq data, followed by survival analysis. SCRN1 was further validated through in vitro experiments. This study presents a comprehensive single-cell CRC atlas, revealing significant heterogeneity in T cells, B cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells within the TIME. We defined two immune ecological subtypes: subtype 1 (enriched in metabolic and motility pathways with poor prognosis) and subtype 2 (enriched in immune response pathways, with better prognosis and greater immunotherapy potential). High SCRN1 expression in malignant epithelial cells correlated with poor prognosis and metastasis. In vitro experiments confirmed the role of SCRN1 in promoting CRC cell proliferation and migration. This study provides a comprehensive characterization of the CRC TIME, links TIME subtypes to prognosis, and identifies SCRN1 as a potential therapeutic target, offering insights for CRC precision subtyping and immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c5be539fad2c0073ae6e62159eaef1e83a1ef" target='_blank'>
              The single-cell atlas of colorectal cancer reveals cellular heterogeneity and key molecular features of the tumor immune microenvironment
              </a>
            </td>
          <td>
            JunHu Li, ZhiQiang Liu, BoYu Zhang, Liuping You, Yuenan Huang
          </td>
          <td>2025-10-10</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08fb4029b669b488e3f7a2bf959803989f3d7dae" target='_blank'>
              Single-Cell RNA and ATAC Sequencing Reveal Cellular Heterogeneity and Chromatin Accessibility Dynamics in Young Chinese Breast Cancer Patients across Pre- and Post-Neoadjuvant Therapy
              </a>
            </td>
          <td>
            Xue Han, Bin Wang, Wenting Xiang, Fan Lin, Bo Sun, Beibei Yang, Zhihan Ruan, Shasha Liu, Shuo Li, Hong Liu, Jian Liu
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Tumor progression is driven not only by the complex milieu of transcriptional cell states of distinct lineages that populate the tumor microenvironment (TME) but also by the interactions these states forge with one another. Systematically charting the landscape of such prognostic cell-state interactions (CSIs) remains difficult as single-cell datasets are small and rarely linked to clinical outcomes, whereas bulk profiles obscure cellular context. We developed CSI-TME, a computational pipeline that adapts the concept of synthetic lethality to infer prognostic cell-state interactions by directly utilizing the large cohorts of bulk transcriptomic datasets. We applied CSI-TME to IDH-mutant gliomas, deriving a highly reproducible cell-state interaction network (CSIN) that is predominantly pro-tumor and differentially activated in IDH-mut astrocytoma versus oligodendroglioma. Malignant cells within the CSIN resemble multiple neuronal lineages including, astrocyte-like and oligodendrocyte-progenitor-like program, and reveal key interactions between glioma stem cells and T cells. CSIN stratifies patient response to immune-checkpoint blockade therapy. Roughly 20% of interactions involve direct ligand-receptor communication, and their participating cell states significantly co-localize in spatial-transcriptomic datasets, most notably for a pro-tumorigenic interaction between tip-like endothelial cells and hypoxic malignant cells involving multiple ligand-receptor interactions. Integration with somatic-mutation profiles reveals that many interactions in CSIN exert anti-tumor effect early-on in glioma but shift toward pro-tumor roles as the disease progresses. Taken together, CSI-TME provides a unique way of identifying the prognostic and clinically relevant cell state interactions without the need for extensive single cell datasets and provides a framework for prioritizing therapeutic ligand-receptor targets, patient stratification, while providing new insight into how interactions among the cell states shape the TME in IDH-mutant glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3146cb53ad109ab0b9473e62c6d93d13f03288e3" target='_blank'>
              TMIC-77. Identification of Clinically Relevant Cell State Interactions in the Tumor Microenvironment of IDH-Mut Glioma
              </a>
            </td>
          <td>
            Arashdeep Singh, Bharati Mehani, Vishaka Gopalan, Ken Aldape, Sridhar Hannenhalli
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Understanding the immune microenvironment requires precise delineation of immune cell populations and their spatial interactions within tissues. Advanced spatial-omics technologies enable simultaneous analysis of cellular biomarkers, spatial contexts, and tissue architecture. However, current analytical tools fall short in integrating this multidimensional data, limiting insights into immune cell functionality and heterogeneity. To address this, we present CellSNAP (Cell Spatio- and Neighborhood-informed Annotation and Patterning), a self-supervised computational framework that integrates molecular expression, spatial neighborhood contexts, and local histological features using graph neural networks. CellSNAP enables the de novo discovery of fine-grained immune cell subpopulations, overcoming the constraints of predefined cell atlases. Applying CellSNAP to spatial-omics datasets across various tissues, including spleen and lymphoma, revealed immune subpopulations with distinct spatial distributions and functional states, such as tumor-associated macrophages and spatially stratified B and T cells. These findings underscore the utility of integrating spatial and molecular data to advance immunological research, offering a robust tool for unraveling immune cell dynamics in health and disease.



 1P01AI177687-01, 75N93019C00071, DP2AI171139, R01AI149672, U24CA224331, NSF 2345215, NSF 2245575.



 Technological Innovations in Immunology (TECH)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6caeae1dc09bc627a0349265ceed8d4f08e8abb" target='_blank'>
              Cross-domain information fusion for enhanced immune cell population delineation in single-cell spatial-omics data 2975
              </a>
            </td>
          <td>
            Bokai Zhu
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d4e23f0e2eb9b451cd70e81ca9b082a2f8b6469" target='_blank'>
              Advancements in single-cell sequencing for cervical cancer research.
              </a>
            </td>
          <td>
            Congli Pu, Biyuan Xing, Shujie Wang, Zhao Liu, Yingchao Zhao
          </td>
          <td>2025-11-06</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Subclones within a patient tumor can differ in their growth rates within the tumor microenvironment. Characterizing this intra-tumor-heterogeneity (ITH) in growth rates offer insights into the tumor’s evolutionary trajectories and the specific aberrations that underpin adaptive success. Recent advancement in single cell sequencing technique provides enhanced resolution in genomic architecture. We developed ith.Fitness, a mathematical framework that leverages the topology of reconstructed single cell phylogenetic trees to infer the relative fitness of individual cells at the leaves and quantify growth-rate variation among subclones. In contrast to existing evolutionary fitness ranking tools originally designed for pathogens such as influenza, our method assumes an evolutionary scenario characterized by potentially large fitness effects from rare driver aberrations and expanding population, more closely reflecting the dynamics of cancer evolution. In addition, ith.Fitness incorporates a tree-rescaling strategy to adjust for variable mutation rates along specific branches, such as those following the event of whole genome doubling (WGD). Simulation demonstrates the accuracy of our method in both non-spatial and spatial tumor growth models. Applied to single cell whole-genome-sequencing data from ovarian cancer, ith.Fitness identified both slow-cycling and fast-growing subclones. Interestingly, subclones arising with WGD do not show elevated growth rates relative to sibling lineages without WGD at the time of sampling. However, WGD-positive cells may gain fitness advantages over time and ultimately dominate tumor evolution.



 Ruping Sun, Chenyu Wu, Zicheng Wang. Inferring subclonal fitness landscapes from single-cell tumor phylogenies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aafde700b6f96c2584ca7470fd535dd0e4d3c559" target='_blank'>
              Abstract B041: Inferring subclonal fitness landscapes from single-cell tumor phylogenies
              </a>
            </td>
          <td>
            Ruping Sun, Chenyu Wu, Zicheng Wang
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Low-grade gliomas (LGGs) exhibit low overall mutational burden and an immunosuppressive microenvironment, contributing to immunotherapy resistance. Intratumoral heterogeneity (ITH) further complicates the targeting of tumor-specific antigens (TSAs), yet its influence on the antigenic and immune landscapes in LGG remains understudied, particularly due to reliance on single-biopsy analyses.



 We performed exome and transcriptome sequencing on 70 spatially mapped biopsies from seven IDH-mutant Grade II astrocytoma patients. Tumor purity, immune deconvolution, and neoantigen prediction were conducted to assess regional immune variation and antigenic burden. In one patient (P375), neoantigen-specific CD8+ T cells were isolated using barcoded peptide-HLA multimers. Reactive T-cell receptor (TCR) clonotypes were identified via single-cell TCR sequencing and functionally validated in Jurkat76 and CD8+ T cells.



 Spatial profiling of whole tumors revealed most mutations were biopsy-specific, with a steep drop in mutations shared across multiple regions. RNA-seq–based unsupervised clustering with xCell and DESeq2 revealed two distinct immune microenvironments: immune-hot regions with elevated immune infiltration, and immune-cold regions with minimal immune presence. Notably, mutation-derived n-mer peptides from immune-hot regions showed enhanced predicted immunogenicity relative to those from immune-cold areas. In one patient (P375), neoantigen-specific CD8⁺ TCR clonotypes were identified targeting PRMT5 mutations found in immune-hot regions. A mutant PRMT5-specific TCR demonstrated robust, antigen-dependent activation and dose-dependent cytotoxicity in vitro. This TCR showed high specificity toward the mutant peptide with minimal cross-reactivity to the wild-type sequence. Furthermore, engineered CD8⁺ T cells effectively killed glioma cells endogenously expressing the full-length mutant PRMT5, supporting its candidacy as a viable immunotherapeutic target.



 Our findings reveal significant 3D spatial and immunologic heterogeneity in low-grade gliomas and demonstrate that neoantigens such as mutant PRMT5 can be selectively targeted by highly specific, cytotoxic TCR-engineered CD8⁺ T cells. These results support the potential of personalized TCR-based immunotherapy for low-grade glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fa3c48068f63d55f21f844232ecea3f66c8ffb8" target='_blank'>
              EXTH-52. Spatially Resolved Antigenic Landscapes In Low-grade Glioma Reveal Targets For Personalized Immunotherapy
              </a>
            </td>
          <td>
            Darwin W. Kwok, Michael Zhang, Cliff Wang, Maggie Colton Cove, Chibo Hong, Nicholas O. Stevers, Benjamin Lerman, S. Lakshmanachetty, Tyler Borrman, Zheng Pan, Benjamin Yuen, Songming Peng, Diana Nguyen, Michael Martin, S. Hilz, Joanna Phillips, Anny Shai, N. Bush, S. Hervey-Jumper, Michael McDermott, S. Mandl, H. Okada, Joseph F Costello
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6c4f2ec4971acec7d7b9addcd8348e71b0ec1ff" target='_blank'>
              Decoding epithelial-T cell interactions in colorectal cancer through single-cell and spatial transcriptomics.
              </a>
            </td>
          <td>
            Rentao Yu, Xinyu Lu, Zhuangzhuang Yue
          </td>
          <td>2025-11-28</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 MAPK pathway alterations occur in 10-20% of diffuse midline glioma, H3K27-altered (DMG-H3K27), resulting in a proposed novel subtype of the disease with distinct clinical features. In a previous co-clinical trial platform, we showed an increased sensitivity to MEK inhibitors in DMG-MAPK patient-derived models, though resistance occurred in vitro, in vivo and clinically. To understand the nature of these resistance mechanisms, we used sophisticated coupling of retrievable high-complexity barcoding and single cell RNA sequencing in DMG-MAPK cells. Lineage tracing conducted in several technical replicate experiments showed a subset of clones is consistently enriched across replicates when treated with a GI90 concentration of trametinib for 11 weeks. Interrogation of clonal dynamics using mathematical modelling showed barcodes commonly enriched across replicates exhibit significantly higher clonal fitness compared to replicate-specific enriched barcodes, suggesting a deterministic pattern of resistance. Serial single cell RNA sequencing analysis at weeks 5, 8 and 11 post-treatment initiation showed some resistant H3.3K27M/BRAF-mutant cells adopted astrocyte-like and neural progenitor-like metaprograms in which barcodes were commonly enriched across two cell states, suggesting a subset of clones circumvent trametinib treatment by phenotypic switching; simultaneously, a late-emerging resistant population of cells activate a distinct mesenchymal program. Furthermore, we determine the previously identified MAP2K1 (MEK1) mutations to arise specifically from a small population of radial glia-like stem cells that appear acutely predisposed to the acquisition of these mutations. Together, our data suggest two clonally and mechanistically distinct resistance mechanisms can be active in DMG-MAPK in response to MEK inhibition, involving both transcriptional plasticity and a progenitor-like state primed for the acquisition of genetic determinants which circumvent drug response. With trametinib-resistant cells being highly sensitive to selective SRC inhibition, evolutionary-guided combinatorial treatment schedules may provide for strategies to significantly lengthen time taken for resistance development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c54926c3fb10216fcf5c70e35738abc3a296f32" target='_blank'>
              DDDR-39. Untangling MEK inhibitor resistance in DMG-MAPK using coupled lineage tracing and single cell RNA sequencing
              </a>
            </td>
          <td>
            Molina Das, Y. Grabovska, Haider Tari, A. Mackay, Rebecca F Rogers, Jared Collins, Jiin Song, Laura Bevington, A. Burford, Drenusha Sejdiu, R. Beroukhim, P. Bandopadhayay, Chris Jones
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>152</td>
        </tr>

        <tr id="


 Despite the remarkable success of targeted therapy, the emergence of acquired resistance remains a major obstacle to cure acute myeloid leukemia (AML) patients. While existing studies have predominantly focused on genetic heterogeneity as a driver for therapy resistance, recent evidence suggests that non-genetic persister state is equally important, but such studies are lacking in AML. Here, we define the temporal transcriptome and chromatin landscape along with lineage trajectory of individual genetically identical AML persister clones to existing anti-AML therapeutics.
 To model persister state in AML, we utilized watermelon library (Oren et al., 2021), that expresses lineage barcode with inducible Histone-2B-mCherry to trace clonal evolution and cell division. We first evaluated whether slow-cycling, quiescent persisters emerge in AML lines after exposure to standard therapy. Our data showed the emergence of slow cycling persisters with cytarabine (AraC) in all 4 cell lines, and gilteritinib (GLT) in MOLM-13 and MV4-11 cells while BH3 mimetics (venetoclax (VEN), S63845 and navitoclax) had fast-cycling persisters. Integrating proliferative-index profiles with cell cycle profiles depicted extensive heterogeneity in AML persisters imposed by drug target rather than genetic state. Drug holiday experiments exhibited a reversible drug tolerance state in both slow and fast-cycling persisters. Longitudinal drug sensitivity and BH3 profiling assays showed that persisters emerged with adaptation to reduced mitochondrial priming that reversed after drug holiday. However, transitioning from persister to stable resistant cells resulted in a sustained loss in apoptotic priming, mirroring observations made in PDX models and primary tumors.
 To understand trajectory of persister emergence and underlying mechanisms, we performed single-cell multiomics on drug-naïve and persister populations (day 9) in MV4-11 (GLT), and OCI-AML2 (VEN and AraC). We expanded 10,000 unique barcoded cells and administered the drug treatment to derive persisters. Using lineage barcode analysis, we report significant reduction in the dominant lineage of day 0 and survival of multiple small lineages indicating an acquired state of persistence than selection of pre-existing resistant clone in both the lines. Further trajectory analysis defined three fates of the lineages as: expanding (log2FC>1), shrinking (log2FC <1) and stable. AraC persisters showed higher expanding and stable lineages, while VEN and GLT persisters resulted in higher shrinking and stable clones, revealing heterogeneity in lineage expansion. We next asked whether persisters resulting from different therapies arised from the same parental lineage. To test this, we performed lineage tracing on VEN and AraC persisters in OCI-AML2 and compared surviving lineages at Day 9 vs Day 0 cells. This analysis identified 52 parental lineages that were able to produce both VEN and AraC persisters in AML2, providing evidence of branched evolution of persisters from the same parental clone.
 We finally defined transcriptional and epigenetic heterogeneity in baseline and persister cells by performing UMAP on single-cell RNAseq and ATACseq data. We identified persisters to all 3 drugs clusters distinctly from their drug naïve counterparts at both transcriptome and chromatin accessibility. Next, we investigated differentially expressed pathways in persisters and observed higher inflammatory response pathways such as TNFα signaling in GLT, enrichment of MYC targets and oxidative phosphorylation in VEN and high mTORC1 signalling in AraC persisters. We observed higher chromatin accessibility in GLT and VEN whereas condensed chromatin in AraC persisters. We next asked which motifs are highly enriched in each persister and observed the enrichment of various KLF and SP family of motifs in each persister driving multiple gene signatures in each persister type. We also observed enrichment of LSC-primed and cDC like cells in GLT persisters but with AraC and VEN persisters a uniform pattern of all states was observed, indicated stemness may not be common mechanism to persister evolution.
 Collectively, our integrated single-cell multiomics and lineage tracing approach reveals that AML persisters emerge through stochastic drug-specific, non-genetic adaptive mechanisms characterized by distinct transcriptional and chromatin remodeling programs rather than deterministic Darwinian selection of pre-existing resistant cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b912c83b14f52bda36c8d81fafb30ac8e86a906c" target='_blank'>
              Dissection of non-genetic resistance mechanisms by lineage tracing and single cell multiomics reveals drug-specific persister programs in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            K. Bhatia, Xiao Xian Lin, Ankit Jana, Emanuelle I. Grody, Ryan Jeremy Lim Jing Wei, Shawn Koh Jia Jun, Elisha Teh Jia Yu, Cin thet Kyi, Prasad Iyer, Greg Kellogg, Yogesh Goyal, Lawrence Boise, Yaara Oren, Shruti Bhatt
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Genomic and epigenetic profiling, particularly DNA methylation analysis, have refined the molecular classification of meningiomas and revealed marked intratumoral heterogeneity. To further characterize heterogeneity within and between patients, we analyzed meningiomas using single-cell RNA sequencing (n=11), whole-exome sequencing (n=9), and spatial transcriptomics (n=3). Single-cell analysis revealed six transcriptionally distinct tumor cell states that corresponded to unique biological processes. Integration of the single-cell data with published exome and bulk RNA sequencing data from a large cohort revealed significant associations among somatic variants, tumor cell states, and immunological signatures. Notably, NF2-altered tumors were enriched for an epithelial-to-mesenchymal transition (EMT) cell state and immune cells, whereas NF2-intact tumors were enriched for a sterol metabolism cell state. Spatial transcriptomic analysis confirmed co-localization of immune cells and EMT tumor cells. Comparisons with immune cells from other brain tumors and peripheral tissues highlighted immunological cell states specific to meningioma. Collectively, these findings refine our genetic and molecular understanding of meningioma heterogeneity and underscore the link between genotype and molecular phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9c0fd6ae388efaa1d2256b78e87826f557e365" target='_blank'>
              PATH-95. Single-cell analysis of meningiomas reveals mutation-associated tumor and immune cell gene expression programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus Anzaldua-Campos, Gregory Zipfel, Albert Kim, Joshua Dowling, Eric C. Leuthardt, Joshua W. Osbun, Ananth K. Vellimana, Michael Chicoine, Gavin Dunn, Allegra S Petti
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Pediatric cancers are often driven by unique alterations suggesting mechanistic ties to underlying developmental programs. Ependymoma (EPN) is an example of a brain cancer subtype driven by gene fusions involving ZFTA-RELA (ZR) that are exclusive to this disease. This leads to our hypothesis that specific chromatin states in developmental lineage programs are at risk of oncogenic transformation, and that these chromatin signatures persist during tumor development and govern tumor cell heterogeneity. To address this question, we constructed an integrated single-nucleus Multiome (snRNA-seq and snATAC-seq) atlas of the developing mouse forebrain and compared it with snMultiome analysis of ZR-driven mouse and human EPN. We identified specific developmental lineage programs present in glial progenitor cells enriched with the Plagl family transcription factors; the precise DNA binding sites of ZFTA fusion oncoproteins. In both mouse and human EPN, ZR expression maintains accessibility of developmental epigenomic programs leading to persistent cell proliferation and tumorigenicity. Cross-species analysis of mouse and human EPN revealed significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that established a putative tumor cell hierarchy. These findings were expanded by applying in vivo lineage barcode tracing methods in mouse EPN, which revealed dominant cell lineages that aggressively expanded and recapitulated the entire tumor cell diversity. Our findings unravel the intersection between developmental and oncogenic epigenomic states critical for ZR driven brain cancer. These findings shape our understanding of the earliest stages of epigenetic programs in brain cancer, and how cancer drivers such as ZR intersect with developmental programs to establish tumor cell heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76bc0f8a27b22de067c5d347cc54b71b6de69b8d" target='_blank'>
              EPCO-12. DOMINANT MALIGNANT LINEAGE LEVERAGE RESTRICTED EPIGENOMIC PROGRAMS TO DRIVE EPENDYMOMA DEVELOPMENT
              </a>
            </td>
          <td>
            Hua Sun, A. Kardian, S. Ippagunta, N. Laboe, Hsiao-Chi Chen, Jiangshan Zhan, Erik Emanus, S. Varadharajan, T. Zheng, B. Holcomb, Jon P. Connelly, Yong-Dong Wang, K. Lowe, Hyun Kyoung Lee, S. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background Gastric cancer progression involves complex interactions among tumor cells, immune components, and stromal elements within the tumor microenvironment. However, a comprehensive understanding of cellular heterogeneity, spatial organization, and cell-cell communication in gastric cancer remains incomplete. Methods Single-cell RNA sequencing was performed on 252, 399 cells from six tissue types, spanning gastritis, intestinal metaplasia, primary tumors, adjacent normal tissue, and metastatic lesions. Integration with spatial transcriptomics enabled spatial mapping of cellular interactions. Pseudotime, cell-cell communication, and transcriptional heterogeneity analyses were conducted. Tumor stage-associated gene modules were identified using Weighted Gene Co-expression Network Analysis (WGCNA) of The Cancer Genome Atlas (TCGA) data. Finally, a deep learning-based prognostic model was developed and externally validated. Results Our analysis revealed dynamic remodeling of the tumor microenvironment during gastric cancer progression, characterized by the expansion of dysfunctional CD8+ T cells, pro-tumorigenic fibroblasts (e.g., ITGBL1+, PI16+, and ITLN1+), and altered myeloid populations. Stromal-immune crosstalk, particularly fibroblast-driven immunosuppressive signaling, was prominent. Spatial transcriptomics revealed the colocalization of immune and stromal cells, supporting spatially organized cellular interactions. WGCNA identified a gene module (657 genes) associated with T cell, myeloid, and stromal alterations, as well as tumor stage. A deep learning model based on this gene set accurately stratified patients according to survival in both TCGA and independent validation cohorts. Risk scores were correlated with clinical features, including tumor stage and therapeutic response. Conclusions Our integrative single-cell, spatial, and computational analysis provides a high-resolution map of gastric cancer microenvironment remodeling. We identified key stromal and immune subpopulations, extensive cellular communication networks, and spatial structures that collectively drive tumor progression and metastasis. The derived gene signature and prognostic model have the potential for clinical risk stratification and therapeutic targeting in gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f147ce2dccf2934265aaef857f3e2b638aa838fb" target='_blank'>
              Integrative multi-omics analysis of gastric cancer evolution from precancerous lesions to metastasis identifies a deep learning-based prognostic model
              </a>
            </td>
          <td>
            Yulin Ren, Xiaoyan Zhang, Ke Li, Shuning Xu, L. Qiao, Qun Li, Cheng Zhang, Ying Liu
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11151ef60d8479a0c0f55b160c123009b32bf93" target='_blank'>
              Single-cell transcriptomic analysis reveals the heterogeneity and functional characteristics of macrophage subpopulations in colon cancer.
              </a>
            </td>
          <td>
            Jiazheng Zhao, Baochun Wang, Xiao Li, Chunting Wei, Yi Min, Dayong Wang
          </td>
          <td>2025-11-23</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b990cf440846339c231eccc76316041249b0f0d" target='_blank'>
              UHRF1 Overexpression Generates Distinct Senescent States with Different Tp53 Dependencies
              </a>
            </td>
          <td>
            Elena Magnani, Charlene Chen, F. Macchi, Bhavani P Madakashira, Tijana Randic, Ian McBain, Kirsten C. Sadler
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Yurie Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, Reo Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Colorectal carcinoma exhibits high heterogeneity, comprising subtypes that show poor efficacy of T cell-based immunotherapies, such as programmed cell death protein 1 (PD-1) checkpoint inhibitors. Although natural killer (NK) cells are considered a promising approach for cancer immunotherapy, it remains unclear what molecular mechanisms drive NK cell activation or suppression within the tumor microenvironment. Moreover, limitations in human tumor models that reflect the diversity of individual patient tumors hinder the ability to effectively select patients who would benefit most from NK cell-based therapies. Methods Here, we established a co-culture platform of genetically diverse colorectal cancer (CRC) patient-derived organoids (PDOs) with primary allogeneic NK cells. We performed bulk RNA sequencing analysis of sorted NK cells after exposure to PDOs and aligned gene expression signatures derived from our findings with publicly available single-cell RNA sequencing data of NK cells from peripheral blood and CRC tissues of patients. Moreover, we evaluated identified pathways using flow cytometry and IncuCyte live-cell imaging analysis to quantify phenotypic alterations and NK cell-mediated killing of PDOs over time, respectively. Ultimately, we tested CRISPR-Cas9-edited NK cells and PDOs, small molecule compounds, and clinically relevant monoclonal antibodies (mAbs) to increase NK cell potency. Results On co-culture, NK cells acquired common transcriptional signatures related to hypoxia and transforming growth factor-beta (TGF-β), similar to NK cells infiltrating CRC tissues of patients. In addition, we observed patient-specific differential PDO susceptibilities to NK cell-mediated lysis. Major histocompatibility complex class I deficiency and natural killer group 2, member D (NKG2D)-ligand expression on PDOs facilitated NK cell-mediated cytotoxicity, and induced phenotypic NK cell diversification related to activation and the acquisition of inflammation and tissue-residency-related transcriptional signatures. Genetic or pharmaceutical targeting of hypoxia-inducible factors HIF1A/EPAS1 or TGF-βR1, or the addition of anti-CEACAM1 mAbs, enhanced NK cell-mediated PDO killing or activation, respectively. Conclusions The NK cell/PDO co-culture platform allows the identification of both common and patient-specific impacts of the tumor microenvironment on NK cell function and can aid the development of patient-tailored immunotherapies. The majority of CRC (CMS2/CMS3) PDOs from our cohort were susceptible to NK cell-mediated killing and induced NK cell activation, highlighting the potential of NK cells for CRC immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bfdbb13391ad7497b66effc8a4643668f2be84a" target='_blank'>
              Colorectal cancer organoids drive hypoxia, TGF-β, and patient-specific diversification of NK cell activation programs
              </a>
            </td>
          <td>
            Andreas von Kries, Irene Garcés-Lázaro, Bianca M Balzasch, Carsten Sticht, Indra A Shaltiel, Kim E. Boonekamp, Annika Sams, Alessia Triassi, Tomáš Hofman, Elke Burgermeister, J. Betge, Matthias P. Ebert, Michael Boutros, Laura Helming, A. Stojanovic, A. Cerwenka
          </td>
          <td>2025-12-01</td>
          <td>Journal for ImmunoTherapy of Cancer</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Cancer metastasis, the process by which tumor cells spread from the primary site to other organs, is one of the leading causes of death in cancer patients. However, due to its complexity and unpredictability, the study of cancer metastasis remains a significant challenge in medicine. In recent years, with the advancement of high-throughput sequencing technologies and single-cell transcriptomics, we could gain a deeper understanding of the molecular mechanisms and cellular heterogeneity underlying cancer metastasis. There is an accumulating volume of publicly available cancer metastasis research data. Nevertheless, these resources lack proper organization, hindering systematic analysis. In this study, we developed an integrated resource named Panmim (http://www.gdwk-bioinfo.com/pan_metastasis/home) to investigate the immune microenvironment of metastatic tumors. The database currently encompasses 90 single-cell RNA-seq datasets from metastatic cancers of diverse origins, encapsulating 14 distinct metastatic sites and 36 primary cancer types. Panmim facilitates a cellular-level comparison of similarities and differences between primary and metastatic cancers, encompassing pathway analysis, alterations in cellular metabolic pathways, cellular distribution preferences, and various aspects of intercellular communication. Panmim presents the analysis results through an intuitive interactive graphical interface, enabling users to explore and understand the complex biological phenomena during cancer metastasis more conveniently. This resource will become a valuable tool for biologists and bioinformaticians to study the mechanisms of cancer metastasis, providing important data support and scientific basis for the optimization of cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9f11b0d110baaaf542f6916b80b13cbf907d57" target='_blank'>
              Panmim: a resource of pan-cancer metastasis immune microenvironment
              </a>
            </td>
          <td>
            Xuelu Zhang, Simeng Hu, Huanjing Hu, Yongkun Li, Pengyin Nie, Xichen Hu, Ruidong Xue, Xiao Xiang, Lu Zheng
          </td>
          <td>2025-10-28</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer (BC) is the most common malignancy among women, with its progression and prognosis significantly influenced by the tumor microenvironment (TME). Age-related differences in TME composition lead to distinct tumor behaviors: young patients (≤ 40 years) exhibit aggressive tumors, while elderly patients (> 70 years) experience immunosenescence and reduced therapy responses. We performed single-cell RNA sequencing (scRNA-seq) analysis on tumors from 10 breast cancer patients (5 ≤ 40 years, 5 ≥ 70 years), encompassing 33,664 high-quality cells. After cell annotation and batch correction, malignant epithelial cells were identified using inferCNV. We applied pseudotime trajectory analysis, pathway enrichment, and cell–cell communication profiling to investigate age-specific TME dynamics. Survival relevance was assessed using a GEO cohort (GSE20685) of young breast cancer patients, and immunohistochemical staining was performed on clinical tumor and fibroadenoma tissues to validate protein-level expression of key ISGs. In young patients, malignant epithelial cells showed gradual upregulation of interferon-stimulated genes (ISGs) such as IFI44, IFI44L, IFIT1, and IFIT3 along the pseudotime trajectory, suggesting their involvement in early tumorigenesis. High expression of these ISGs was significantly associated with poor overall survival in a young BC cohort (GSE20685). Immunohistochemical validation further confirmed elevated IFIT3 protein levels in young tumor tissues. In contrast, elderly patients had a TME enriched in macrophages and fibroblasts, with activation of immunosuppressive pathways (e.g., SPP1, COMPLEMENT). Our integrative analysis identifies ISGs as key transcriptional drivers of tumorigenesis in young breast cancer, with potential prognostic and therapeutic value. Despite limited sample size, the combination of single-cell transcriptomics, clinical survival data, and protein-level validation provides robust evidence of age-specific TME remodeling. These findings support the development of age-tailored immunotherapy strategies targeting interferon signaling in young patients and immune checkpoint pathways (e.g., LAG3, CTLA4) in elderly individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81afa54ab4f065b06519a2062a7ca2a21beb87a4" target='_blank'>
              Single-cell RNA sequencing reveals age-related heterogeneity in the tumor microenvironment of breast cancer patients
              </a>
            </td>
          <td>
            Zihao Li, Ji Feng, Mengyang An, Shuming Kou, Zengren Zhao, Jingjing Yu, Bo Liu, Fangjian Shang
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Introduction Although the non-covalent BTK inhibitor Pirtobrutinib (PBN) has demonstrated clinical efficacy in treating relapsed and refractory mantle cell lymphoma (MCL) patients, resistance emerges. Inter-patient genetic variability and tumor intrinsic plasticity give rise to diverse resistant phenotypes under therapeutic pressure, posing a major challenge to understanding and overcoming drug resistance.
 Methods To dissect the molecular dynamics driving PBN resistance, we performed integrative single-cell multi-omic profiling on pre-treatment and relapsed primary MCL patient samples. Our approach combined single-cell RNA sequencing (scRNA-seq) to measure gene expression, single-cell assay for transposase-accessible chromatin using sequencing (scATAC-seq) to assess chromatin accessibility, and single-cell DNA sequencing (scDNA-seq) to define genomic alterations, including copy number variations and clonal architecture.
 Results We analyzed 24 peripheral blood or apheresis samples collected from 13 MCL patients at both drug-sensitive and resistant timepoints, yielding a high-quality dataset of 113,143 single cells. Malignant B cells, marked by high levels of CD79A, CCND1, MS4A1, and IGHM, comprised 50.28% of the dataset (n= 56,889). Our analyses revealed both genetic and non-genetic routes to resistance. In some patients, resistance involved stepwise acquisition of copy number gains, with 2p and 8q gains detected in early resistant clones and additional 1q gains emerging in late-stage resistant clones (adjusted p < 0.05). Polygenetic tree reconstructed from scDNA-seq data confirmed this clonal evolution, and metaphase karyotyping further validated these findings.
 In contrast, some patients showed no large-scale genomic alterations and developed resistance through transcriptional and epigenetic reprogramming alone, reflecting non-genetic adaptation to therapeutic pressure. Pathway analysis revealed enrichment of NF-κB signaling and gene programs associated with chromosome segregation, mitosis, and protein localization to chromosomes (enrichment ratio >0, adjusted p < 0.05).
 Integration of scATAC-seq and scRNA-seq enabled gene regulatory network inference. RAD21, a core component of the cohesin complex, was identified as a key regulator by binding to 987 resistance-specific DNA regions and regulating 816 genes, potentially driving PBN resistance (adjusted p < 0.05). Simulated in silico knockout of RAD21 using a random forest model shifted resistant cells toward a sensitive-like transcriptional state, supporting RAD21 as a candidate therapeutic target for overcoming PBN resistance.
 Additionally, we identified a stem-like malignant B cell population enriched in resistant samples, displaying features of metabolic reprogramming and epithelial-mesenchymal transition. Within this population, two distinct subclusters emerged. Cluster 1 demonstrated epithelial-like, progenitor-associated, while cluster 2 showed mesenchymal-like features and metabolic dysregulation (enrichment ratio >0, adjusted p < 0.05). This aligns with a recently proposed model published in Nature describing a resistance continuum, in which treatment-resistant tumors are not static but contain a range of stem-like states with varying degrees of lineage plasticity and metabolic flexibility.
 Conclusion In summary, our study presents the first longitudinal single-cell multi-omic characterization of PBN resistance in primary MCL patients. We identified coordinated transcriptional and epigenetic remodeling during resistance evolution across patients. Our integrative analyses highlight RAD21 and the cohesin complex, along with stem-like heterogeneity, as potential drivers of therapeutic resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd26da811ab0e01e4e319098f4c0cd592f67ff0a" target='_blank'>
              Dissecting pirtobrutinib resistance in Mantle Cell Lymphoma through single-cell multi-omics
              </a>
            </td>
          <td>
            Fangfang Yan, Yang Liu, , Wei Wang, J. McIntosh, Yijing Li, J. Vargas, Yue Fei, Lukas M. Simon, Michael Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The tumor microenvironment (TME) constitutes a highly complex and dynamic ecosystem wherein diverse leukocyte subsets engage in intricate cross-talk that ultimately dictates the balance between tumor progression and suppression. This review provides a comprehensive analysis of the dualistic functions of key immune cell populations-including natural killer (NK) cells, effector T cells, regulatory T cells (Tregs), myeloid-derived suppressor cells (MDSCs), and tumor-associated macrophages (TAMs)-within this milieu. Particular emphasis is placed on the Nigerian context, where unique genetic backgrounds, such as the high prevalence of sickle cell trait, and environmental factors, including endemic infections, significantly shape immune phenotypes and cancer pathogenesis. We explore how cutting-edge single-cell technologies are revolutionizing our understanding of the spatial architecture, functional plasticity, and cellular heterogeneity of these leukocytes in patient cohorts, revealing mechanisms of both immune surveillance and tumor-promoting immunosuppression. The paradoxical roles of these cells elucidate the variable clinical outcomes observed with immunotherapies and underscore the critical need for precision medicine strategies that are tailored to individual immune landscapes. Finally, we discuss emerging therapeutic modalities aimed at selectively depleting pro-tumor subsets like Tregs or MDSCs while activating anti-tumor effectors such as NK cells, which hold substantial promise for reshaping the TME and improving oncology outcomes both in Nigeria and across the globe.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d4138591bf3dda987df758be78f378fdcee1f4" target='_blank'>
              Leukocyte Subset Dynamics in the Tumor Microenvironment: Dual Roles, Single-Cell Insights, and Implications for Cancer Immunotherapy in Nigeria
              </a>
            </td>
          <td>
            Okluchi Hartumu
          </td>
          <td>2025-11-25</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d62c6090571c67ea1645ed1df6849f539cf798b" target='_blank'>
              Molecular and cellular dynamics of measurable residual disease progression in myelodysplastic syndromes
              </a>
            </td>
          <td>
            Yu-Hsiang Chen, Timothy Bi, Minhang Xu, Jaehyun Lee, Wesley S. Lu, Mengchuan Zhuang, A. Velarde, D. McCurry, Hui Yang, G. Al-Atrash, Gabriela Rondon, E. Shpall, F. Jelloul, F. Cervoni-Curet, Sassine Ghanem, Celine Kerros, Priya Koppikar, J. Molldrem, G. Garcia-Manero, Ankit Patel, Pavan Bachireddy
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe18ec72d329e63e66769b08f8cae7347edc672" target='_blank'>
              Prognosis prediction using autophagy gene expression in osteosarcoma
              </a>
            </td>
          <td>
            Qingzhu Liu, Cong Zhou, Ke Xu, Qikui Zhu, Junqin Lu, Chun Zhang, Wukun Xie, Guojiu Fang, Xue Zhou Chen, Dasheng Tian, Juehua Jing, Yize Li, Hongsheng Wang, Wenxiu Duan, Yihui Bi
          </td>
          <td>2025-11-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Introduction CAR-T cell therapy efficacy in B-cell lymphoma remains limited by treatment resistance. To address this fundamental challenge and elucidate underlying mechanisms, we undertook a comprehensive single-cell analysis of the CAR-T-tumor microenvironment interface. By leveraging five public scRNA-seq datasets spanning 1.48 million cells from CD19-targeted CAR-T patients, this study establishes an advanced computational framework to decipher the cellular ecosystem governing therapeutic resistance.
 Methods Methodologically, we developed three integrated innovations: (1) A biologically informed hierarchical graph convolutional network (GCN) modeling patient-specific tumor microenvironments as cell-cell correlation graphs for response prediction; (2) A novel Grad-CAM extension enabling quantitative interpretation of GCN outputs at single-cell resolution to establish gene-outcome relationships and resistance biomarkers; (3) GeneTransformer—a hybrid transformer architecture predicting transcriptional responses to genetic perturbations while resolving cross-platform heterogeneity via stochastic triple-masking during finetuning.
 Results Analysis revealed system-wide transcriptional dysregulation and extensive cell-subset-specific alterations distinguishing non-responders. Functional integration identified four core resistance modules: pro-survival signaling (BCL2/XIAP), metabolic rewiring (AMPK exhaustion), immune dysfunction (PD-1/TIGIT), and homeostatic collapse (Hippo/YAP-mediated stemness). Crucially, we exposed mechanistic crosstalk revealing actionable combinatorial targets—notably PI3K inhibition synergizing with anti-PD-1 blockade. Cross-population validation confirmed model robustness, with 84.65% (p<0.05) of reverse-engineered resistance genes exhibiting differential expression. The GCN achieved superior predictive performance (AUC=0.900).
 Conclusion This study establishes a transformative computational framework that deciphers the single-cell architecture of CAR-T resistance, providing a mechanism-guided blueprint for designing combinatorial therapeutic strategies. The GeneTransformer platform further enables optimization of individualized therapies through perturbation response prediction. Our findings illuminate fundamental resistance pathways while delivering clinically actionable targets for overcoming treatment failure in B-cell malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a8a729b90dea4325ceebae0fe0354baffbb43f1" target='_blank'>
              Deep learning-based tumor microenvironment profiling predicts CAR-t efficacy and guides combinatorial immunotherapy in B-cell lymphoma
              </a>
            </td>
          <td>
            Shaojie Wu, Jiatong Wang, Jiaxiu Li
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The spatial dynamics of B cells are crucial during immune response, shaping clonal evolution and affinity maturation. B cell evolution can by studied by reconstructing phylogenetic lineage trees, which represent B cell receptor (BCR) mutations among sampled cells. SpatialVDJ sequencing is a recently developed technique that pairs spatial RNA-sequencing with long-read, spatially resolved BCR data on a 2D slice of tissue. Currently, no methods exist to analyze the movement of B cell lineages across 2D coordinates. We introduce GIBLE (Geographic Inference of B Cell Lineage Evolution), a novel computational framework that reconstructs spatial locations of unobserved, ancestral B cells and quantifies movement across the tissue slice. We used GIBLE to understand B cell maturation in ectopic germinal centers in thymic tissue from patients with myasthenia gravis, an autoimmune disorder in which autoreactive B cells produce antibodies against the neuromuscular junction. GIBLE provides a robust and widely applicable framework for integrating spatial and phylogenetic data to trace B cell evolution during immune responses using SpatialVDJ data.



 Supported by NIH/NCI 5T32CA134286.



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171685d301f0d9a0ae908798444d8439b632feea" target='_blank'>
              Resolving B cell migration from SpatialVDJ sequencing with GIBLE 3612
              </a>
            </td>
          <td>
            Hunter J. Melton, Chinmay M. Vaidya, Jessie J. Fielding, Sherry Wu, Fred W. Kolling, Kevin C. O’Connor, Kenneth B. Hoehn
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8a1f1ccbeb7cda31923d2bd498d2b7cb3a690e5" target='_blank'>
              Metabolic zonation and characterization of tissue slices with spatial transcriptomics
              </a>
            </td>
          <td>
            Oier Etxezarreta Arrastoa, A. Pirona, Allon Wagner, N. Yosef
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) represents a significant global health challenge due to its molecular heterogeneity and the immunosuppressive nature of its tumor microenvironment (TME). Tumor-associated macrophages (TAMs), particularly the M2-polarized subset, are crucial in HCC progression, contributing to immune evasion, angiogenesis, and metastasis. However, the mechanisms driving TAM reprogramming and their metabolic interactions within the TME remain poorly understood. In this study, we applied single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to identify a novel metabolic-immune axis in HCC. Our findings revealed increased oxidative phosphorylation (OXPHOS) in tumor cells, which correlated with enhanced ribosomal activity in M2 macrophages, suggesting a metabolic coupling between the two. Tumor cells, under elevated OXPHOS, secrete factors that recruit macrophages and promote M2 polarization, thereby reinforcing immune suppression. Further analysis of The Cancer Genome Atlas (TCGA) dataset led to the development of a prognostic model based on OXPHOS-related genes, showing strong predictive power across multiple cohorts. These results suggest a novel OXPHOS-ribosome-M2 polarization axis and offer potential therapeutic targets for immunometabolic therapies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d98d705964e6108c751cf9dce67e7948e856f2d" target='_blank'>
              M2 macrophages and tumor cells engage in a metabolic feedback loop to drive HCC progression
              </a>
            </td>
          <td>
            Yimin Nong, Xiaolong Chen, Baoxin Zhang, Zhenying Lin, Wuriniletu, Chengyou Du, Qiang Liu
          </td>
          <td>2025-11-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f4df7fa2e4800fd55af9dfd1c2bbdecb131c8f" target='_blank'>
              A guide to transcriptomic deconvolution in cancer.
              </a>
            </td>
          <td>
            Yaoyi Dai, Shuai Guo, Yidan Pan, C. Castignani, Matthew D. Montierth, P. van Loo, Wenyi Wang
          </td>
          <td>2025-12-02</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Glioblastoma (GBM) is a highly aggressive brain tumor with a median survival of only 15 months. A major challenge in GBM management is the pronounced inter- and intratumoral heterogeneity, which complicates prognosis and therapy. Radiomics, the quantitative extraction of features from medical images, can capture this heterogeneity across the entire tumor volume, but the biological basis of radiographic phenotypes remains poorly understood.



 We integrated preoperative MRI-based radiomic stratification with multi-platform transcriptomics (bulk RNA-seq, single-cell RNA-seq, and spatial transcriptomics) in IDH-wildtype GBM patients. Unsupervised clustering of radiomic features identified four imaging subtypes.



 Group 4 emerged as a high-risk subtype associated with significantly worse survival and a distinctive MRI pattern of peripheral contrast enhancement. Transcriptomic analyses revealed that Group 4 tumors were enriched in cell-cycle and proliferation markers and exhibited neural stem cell–like gene expression signatures. Single-cell profiling confirmed an elevated proportion of stem-like malignant cells in this subtype. Spatial transcriptomics further demonstrated that these proliferative, stem-like programs were localized predominantly to the tumor periphery, corresponding to the rim-enhancing regions on MRI. Finally, we identified the developmental transcription factor VAX2 as a candidate driver of the Group 4 gene network; functional assays showed that VAX2 promotes GBM cell proliferation in vitro.



 Our findings link a radiomics-defined MRI phenotype to specific molecular programs and cell populations in GBM, suggesting that radiomic subtypes can serve as noninvasive biomarkers of tumor biology and highlighting potential therapeutic targets in aggressive, stem-like tumor cell populations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63325a32d6106e3ca9f5bff7f21292a347c5da7b" target='_blank'>
              MRI-based radiomic clustering identifies a glioblastoma subtype enriched for neural stemness and proliferative programs
              </a>
            </td>
          <td>
            Zhongyi Zhang, Yang Liu, Zhicong Zhang, Tian Gui, Youhui Chen, Qian Chen, Xiufu Wu, Li Sun, Sujie Li, Shuyang Wei
          </td>
          <td>2025-11-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed06b707f53d2e1de85e352e5c2852fcf93afdb7" target='_blank'>
              Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer
              </a>
            </td>
          <td>
            Maria Vidal-Sabanés, R. Navaridas, Núria Bonifaci, Ada Gay-Rua, Damià Ortega-Peinado, Joaquim Egea, Mario Encinas, Xavier Matias-Guiu, David Llobet-Navás, Xavier Dolcet
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chemotherapy resistance remains a major clinical challenge in ovarian cancer treatment, limiting therapeutic efficacy and patient survival. Understanding the molecular mechanisms underlying chemoresistance at the single-cell level is crucial for developing targeted therapeutic strategies. We performed comprehensive single-cell RNA sequencing analysis on primary and chemoresistant ovarian cancer samples to characterize cellular heterogeneity and identify resistance-associated molecular signatures. Cell type identification, differential gene expression analysis, functional enrichment analysis, and immune microenvironment characterization were conducted to elucidate chemoresistance mechanisms. Computational predictions were validated by RT-qPCR analysis of four key resistance genes (BIRC5, ABCB1A, ABCG2, BCL2) in SKOV3 and SKOV3-CDDP cell lines. Our analysis identified nine distinct cell populations within the ovarian cancer microenvironment, revealing extensive transcriptional reprogramming in chemoresistant tumors. Key chemoresistance genes including Abcb1a, Abcg2, Bcl2, Birc5, Nek2, Inhba, Irs1, and Prc1 showed differential expression patterns across cell types. Epithelial cells and proliferating cells exhibited elevated expression of multiple resistance markers, particularly Birc5. Functional analysis revealed enrichment of ribosome biogenesis, protein synthesis machinery, and amoeboid-type cell migration pathways in resistant cells. Macrophage polarization states were significantly altered, and immune checkpoint molecules showed coordinated upregulation, indicating comprehensive immune microenvironment reprogramming. RT-PCR validation confirmed single-cell predictions with significant upregulation in resistant cells: BIRC5 (4.2-fold, p < 0.001), ABCB1A (3.8-fold, p < 0.001), ABCG2 (2.9-fold, p < 0.01), and BCL2 (2.1-fold, p < 0.05). This study provides a comprehensive single-cell atlas of chemoresistance in ovarian cancer, revealing cell type-specific resistance mechanisms and immune evasion strategies. The identified molecular signatures offer potential therapeutic targets for overcoming chemotherapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f229e22beb03e0ca3194ad4df8f24784e9b8331" target='_blank'>
              Single cell transcriptomic analysis reveals molecular mechanisms of chemoresistance and immune microenvironment remodeling in ovarian cancer
              </a>
            </td>
          <td>
            Xia Hu, Caifang Du, Dongjie Du, Yonghua Zhang, Haijian Gao, Minxian Tao
          </td>
          <td>2025-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Human papillomavirus (HPV)-associated oropharyngeal cancer (OPC) is increasing in prevalence, but the drivers of metastasis remain poorly understood, which impacts the ability to personalise management decisions. Much of the genomic research to date focuses on the HPV-negative population. Here, we utilise single-cell and spatial single-cell techniques to understand the drivers of metastasis. Patients with HPV-positive OPC and cervical lymph node metastases treated with curative surgery had matched samples from the primary and lymph nodes collected for research. Single-cell RNA sequencing, single-cell spatial sequencing (Visium) and in-situ spatial platforms were performed. Cancer clones were delineated using inferred copy number variation. Expression phenotypes and interactions with the tumour microenvironment were compared between the metastasising and non-metastasising cancer clones. Individual cancer clones have varied abilities to metastasise and undergo clonal expansion in the lymph node, with only a subset of clones present in the primary expanding in the lymph node. Four mechanisms were identified as defining the metastatic phenotype, including protein translation adaptation, immunoproteasome dysfunction and immune evasion, suppression of the IFN immune response and cap-independent protein translation. This research elucidates multiple mechanisms driving the expansion of cancer clones in HPV-positive oropharyngeal cancer. By detailing the roles of translational adaptation, immunoproteasome dysfunction, suppression of the interferon immune response and cap-independent protein translation, we provide insights into how these processes contribute to immune evasion and tumour survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fda352f2dd3f74c2b6cb392920d418565101f0" target='_blank'>
              Protein translation dysregulation and immune cell evasion mediated by IFN and immunoproteasome downregulation define metastatic clones in HPV-related cancer of the oropharynx
              </a>
            </td>
          <td>
            Venessa T. Chin, W. Muskovic, Rachael A. McCloy, Drew R. Neavin, José Alquicira-Hernández, Himanshi Arora, A. Senabouth, K. Sudhakar, Patricia Keith, Ellie Spenceley, Dominik Kaczorowski, Peter Floros, Brett Leavers, Julia Crawford, Richard Gallagher, Joseph E. Powell
          </td>
          <td>2025-10-14</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocytes (TILs) play a critical role in the ability of the immune system to combat cancer, offering a foundation for personalized immunotherapies. However, the effectiveness of TILs is often reduced by problems like becoming less active, the tumor making the immune system weaker, and not lasting long in the tumor environment. Recent advancements in single-cell technologies, including single-cell RNA sequencing (scRNA-seq), single-cell T-cell receptor sequencing (scTCR-seq), and mass cytometry (CyTOF), have revolutionized our understanding of TIL heterogeneity and dynamics. These tools offer new perspectives on the diverse phenotypes, functional states, and spatial organization of TILs, enabling the identification of key exhaustion markers, regulatory pathways, and neoantigen-specific clones. Concurrently, genetic reprogramming strategies have emerged to address TIL limitations by reversing exhaustion, enhancing metabolic resilience, and improving persistence in vivo. This review explores the synergistic integration of single-cell technologies and genetic engineering in refining TIL-based therapies. We talk about how spatial transcriptomics can help us understand how TILs work in different areas of the body and how changing their epigenetics can help them become more effective at fighting cancer. Additionally, we highlight emerging approaches to overcome immunosuppressive barriers in the tumor microenvironment (TME), including targeting regulatory immune cells, neutralizing suppressive cytokines, and enhancing antigen presentation. Together, these strategies promise to unlock the full therapeutic potential of TILs, paving the way for more effective and durable cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8024b4f684c7b9b9c99e6af201613f98c8efa0" target='_blank'>
              Probing and enhancing tumor-infiltrating lymphocytes: insights from single-cell technologies and genetic reprogramming
              </a>
            </td>
          <td>
            Chaitanya Kumar, Ravi Verma, Ashok Sharma, V. V. Priya
          </td>
          <td>2025-11-17</td>
          <td>Exploration of Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87fbfbc883a34e550ce8b95d1c7f558541bbd9cd" target='_blank'>
              Clonal Dynamics in the Male Germline from Development to Next Generation
              </a>
            </td>
          <td>
            Tatsuro Ikeda, Maurice Langhinrichs, Tamar Nizharadze, Chieko Koike, Yuzuru Kato, K. Yamaguchi, Shuji Shigenobu, Kana Yoshido, Shinnosuke Suzuki, Toshinori Nakagawa, A. Maruyama, Seiya Mizuno, Satoru Takahashi, Hans-Reimer Rodewald, Thomas Höfer, Shosei Yoshida
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Metastatic outgrowth requires that cancer cells delaminate from the primary tumor, intravasate, survive in circulation, extravasate, migrate to, and proliferate at a distal site. Recurrent genetic drivers of metastasis remain elusive, suggesting that unlike the early steps of oncogenesis, metastasis drivers may be variable. We develop a framework for identifying metastasis regulators using CRISPR/Cas9-based screening in a genetically defined organoid model of colorectal adenocarcinoma. We conduct in vitro screens for invasion and migration alongside orthotopic, in vivo screens for gain of metastasis in a syngeneic mouse model. We identify CTNNA1 and BCL2L13 as bona fide metastasis-specific suppressors which do not confer any selective advantage in primary tumors. CTNNA1 loss promotes cell invasion and migration, and BCL2L13 loss promotes anchorage-independent survival and non-cell-autonomous changes to macrophage polarization. This study demonstrates proof of principle that large-scale genetic screening can be performed in tumor-organoid models in vivo and identifies novel regulators of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33c9f8937cc2d0d69608a2acde03a6c6afb3fab8" target='_blank'>
              Forward genetic screening in engineered colorectal cancer organoids identifies regulators of metastasis.
              </a>
            </td>
          <td>
            Xin Wang, Zvi Cramer, N. A. Leu, Keara Monaghan, K. Durning, S. Adams-Tzivelekidis, Joshua Rhoades, Jonathan Heintz, Yuhua Tian, Joshua Rico, Diego Mendez, R. Petroni, Austin King, Melissa S Kim, Rina Matsuda, Olivia Hanselman, Alice E. Shin, María F Carrera Rodríguez, Igor E. Brodsky, Anil Rustgi, Ning Li, Christopher Lengner, M. Andres Blanco
          </td>
          <td>2025-11-11</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acaf4b9f2872701e5c44aad69b0ee83629b9228" target='_blank'>
              Interconnected axes of phenotypic plasticity drive coordinated cellular behaviour and worse clinical outcomes in breast cancer
              </a>
            </td>
          <td>
            Ritesh Kumar Meena, Yijia Fan, Soundharya Ramu, Anjaney J. Pandey, Abhinav Kannan, Yashvita Subramanian, Yukta Subramanian, Jason T. George, M. Jolly
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have revolutionized cancer treatment and show notable success in some cancer types such as non-small cell lung cancer, melanoma and colorectal cancers, while they demonstrate relatively low response rate in others, such as esophageal cancers. Due to the heterogeneous nature of the tumor microenvironment and patient-to-patient variability, there remains a need to improve ICI response rates. Combining ICIs with therapies that can overcome resistance is a promising strategy. Compared to de novo drug development, drug repurposing offers a faster and more cost-effective approach to identifying such combination candidates. A variety of computational drug repurposing tools leverage genomics and/or transcriptomic data. As single-cell RNA sequencing (scRNA-seq) technology becomes available, it enables precise targeting of cancer-driving cellular components. In this review, we highlight current computational drug repurposing tools utilizing scRNA-seq data and demonstrate the application of two such tools, scDrug and scDrugPrio, on an esophageal squamous cell carcinoma dataset to identify potential drug candidates for combination with ICI therapy to enhance treatment response. scDrug focuses on predicting tumor cell-specific cytotoxicity, while scDrugPrio prioritizes drugs by reversing gene signatures associated with ICI non-responsiveness across diverse tumor microenvironment cell types. Together, this review underscores the importance of a multi-faceted approach in computational drug repurposing and highlights its potential for identifying drugs that enhance ICI treatment. Future work can expand the application of these strategies to multi-omics and spatial transcriptomics datasets, as well as personalized patient samples, to further refine drug repurposing involving ICI therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b33c93cda41e1064ce2cfb630c9ed87d2c20b17" target='_blank'>
              Harnessing Single-Cell RNA-Seq for Computational Drug Repurposing in Cancer Immunotherapy
              </a>
            </td>
          <td>
            Olivia J. Cheng, T.T.T. Tran, Y. A. Chen, A. Tan
          </td>
          <td>2025-11-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The spatial organization of the tumor microenvironment (TME) profoundly influences cancer biology. However, the cell types and spatial distribution driving lung adenocarcinoma (LUAD) invasion remain poorly understood. By integrating spatially resolved transcriptomics with scRNA-seq data, we identify a novel secretory leukocyte protease inhibitor (SLPI)-expressing AT2-like subpopulation that localizes specifically at the invasive tumor margin, which drives the transition from minimally invasive (MIA) to invasive (IA) LUAD. Functional characterization reveals that SLPI-expressing AT2-like cells upregulated Dickkopf-1(DKK1), enhancing tumor cell stemness and epithelial-mesenchymal transition (EMT). Furthermore, spatially co-localized MRC1-expressing resident tissue macrophages (RTM-TAMs) secrete pro-tumor cytokines upon interaction with SLPI+ tumor cells, alongside cancer-associated myofibroblasts (myo-CAFs) exhibiting reduced type I collagen production. These TME components establish a pro-tumorigenic niche and engaged in synergistic interactions with SLPI+ AT2-like cells to facilitate LUAD invasion. These findings reveal the specific cellular composition, spatial architecture, and functional crosstalk between tumor cells and TME subpopulations that orchestrated LUAD progression. The frontier molecular targets at the tumor invasive identified in this study can serve as a basis for developing novel therapeutic targets in the future and assist pathologists in accurately assessing patients' disease progression and survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef641f6801c52b34b7967192c024cc7cd0f87f78" target='_blank'>
              SLPI⁺ AT2-Like Cells Orchestrate Lung Adenocarcinoma Invasion via Wnt Pathway Activation and Stromal Crosstalk in a Spatially Defined Margin Niche.
              </a>
            </td>
          <td>
            Zhoufeng Wang, Guonian Zhu, Pan Tang, Yan Wang, Wenxin Luo, Wenpeng Song, Zhikang Pan, Bingjie Zheng, YongChao Jiang, Defu Xiao, Xin Jin, Yong Bai, Guowei Che, Weimin Li
          </td>
          <td>2025-11-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Genomic alterations in glioblastoma (GB) drive distinct transcriptional subtypes, yet how they shape the tumour microenvironment (TME) architecture remains poorly understood. Here, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss, and PDGFB overexpression in Tp53−/− Nestin-tv-a mice, that recapitulated classical, mesenchymal, and proneural transcriptomic GB subtypes. We integrated single-cell RNA sequencing with regional (NanoString GeoMx) and single cell (10x Visium HD) spatial transcriptomics and proteomics data to acquire an in-depth characterization of the TME organization. At single cell level, NF1- and EGFRvIII-driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human GBs that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1043a078420592595b843047c99776c531341d4b" target='_blank'>
              TMIC-04. Genetic drives promote distinct microenvironment organization in glioblastoma
              </a>
            </td>
          <td>
            Junyi Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is the most aggressive and lethal primary brain tumor in adults, characterized by rapid proliferation, an immunologically inert microenvironment, diffuse tumor cell infiltration and therapy resistance. Progress in developing effective glioblastoma treatments is impeded by a lack of mechanistic insight into how GBM cells and the surrounding microenvironment respond to therapeutic interventions. To address this prominent gap, we developed an innovative on-therapy, longitudinal, and multi-regional tissue sampling approach in patients receiving rQNestin34.5v.2, an engineered oncolytic biologic, based on herpes simplex virus 1, to investigate the dynamic evolution of tumor and immune ecosystems during treatment. Using targeted spatial proteomics (CyCIF) alongside spatial metabolomics (MALDI-MSI), we analyzed over 100 samples from six patients, profiling immune, stromal, and malignant cells across both space and time. Our CyCIF data revealed that T and B cells infiltrate and colocalize within rQnestin34.5v.2–positive regions, forming clusters that reflect active immune interactions. These immune-rich zones were inversely correlated with SOX2⁺ and OLIG2⁺ tumor cells. Notably, patients with longer survival showed a progressive increase in CD3⁺ T cells, driven by expanding cytotoxic T cell populations alongside a reduction in CD4⁺ T cells. Despite increased immune infiltration, post-trial tissues displayed an immunosuppressive environment characterized by few cytotoxic T cells and reduced tumor cells. Integrating spatial proteomics and metabolomics, we observed elevated purine metabolites—including ATP, GTP, and xanthine—in regions of heightened immune infiltration. Extracellular ATP and GTP likely facilitate T cell activation and migration through purinergic signaling and GTPase pathways. These metabolic patterns suggest that purine metabolism plays a key role in shaping immunologically active tumor microenvironments following treatment. Together, these findings provide novel mechanistic insights into the evolving tumor-immune landscape during rQnestin34.5v.2 therapy in glioblastoma, highlighting purine metabolism as a potential driver of immune activation within the tumor microenvironment. This integrated spatial approach offers a powerful framework to guide the development of more effective immunotherapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82f2a6552b2a8d376cecc37c28c7caffa62bc8f2" target='_blank'>
              TMIC-70. Integrative spatiotemporal proteomic and metabolomic characterization of immune infiltration following intracranial injection of oncolytic immunotherapy in glioblastoma patients
              </a>
            </td>
          <td>
            Jennifer Gantchev, Gerard Baquer, Peter Zheng, Alicia D’Souza, Alexander L. Ling, Michael S Regan, G. Teamlab, Atul Deshpande, Forest M. White, Kenny K. H. Yu, E. Chiocca, Charles Couturier, Nathalie Y R Agar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Understanding how small molecules affect cellular morphology is essential for exploring their mechanisms of action (MoA) and identifying new therapeutic candidates. In this study, the Cell Painting Assay (CPA) was used to profile 196 small molecules in HCT116 colorectal cancer cells. By applying t-distributed stochastic neighbor embedding (t-SNE) followed by density-based clustering, 18 distinct phenotypic clusters were identified based on similarities in the quantitative morphological profiles generated from Cell Painting data. Although it was initially hypothesized that clustering would reflect known MoAs, most clusters showed only partial overlap with target-based classifications. Instead, compounds from different MoA classes converged on similar cellular phenotypes, suggesting common downstream effects or shared stress responses. Notably, compounds affecting DNA replication, mitosis, or transcriptional control appeared across multiple clusters, indicating functional diversity within morphologically similar groups. Clusters enriched with mTOR/PI3K inhibitors, spindle poisons, or transcriptional CDK blockers exhibited well-defined phenotypes, supporting the robustness of the assay. In contrast, compounds inducing more subtle phenotypes formed distinct micro-clusters, highlighting the method’s sensitivity. Overall, this study demonstrates that CPA can capture convergent phenotypic signatures that extend beyond target-based classification. These findings underscore the value of phenotype-driven screening for the functional annotation of chemical compounds and may help uncover unexpected relationships among molecules with diverse biological activities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b69630fa6518b96d44a14fa030997f1e5bc439" target='_blank'>
              Phenotypic profiling of small molecules using cell painting assay in HCT116 colorectal cancer cells
              </a>
            </td>
          <td>
            Velemir Lavrinenko, Ilona Donskaya, Vladimir Popov, Ekaterina Litau, Varvara Petrova, Stanislav Tyazhelnikov
          </td>
          <td>2025-10-29</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Over 95% of primary central nervous system lymphoma (PCNSL) cases are histologically classified as diffuse large B-cell lymphoma (DLBCL), but present markedly poorer prognosis than systemic DLBCL. This discrepancy may stem from its high molecular heterogeneity, immune privilege, inhibitory tumor microenvironment (TME) and dilemma in drug delivery. Current understanding in the tumor ecosystem of PCNSL remains limited. This study employed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomic (ST) analyses to decode tumor microenvironmental features of PCNSL.
 We performed scRNA-seq of PCNSL tumor samples and reactive lymphoid hyperplasia samples, and additionally conducted ST on PCNSL tumor samples. Unsupervised clustering of B cells identified 4 malignant B cell (mBc) subpopulations with apparently transcriptional and spatial discrepancy. Pseudotime trajectory analysis revealed that mBc3 represented the origin, which evolved into mBc4 and mBc1 and ultimately differentiated into mBc2. To assess the clinical relevance of these subsets, we applied BayesPrism for deconvolution of bulk RNA-seq data from DLBCL cohorts, presenting mBc2 subset was significantly correlated to poor prognosis (p< 0.001). Kyoto Encyclopedia of Genes and Genomes pathway analysis and scMetabolism analyses indicated that mBc2 exhibited a strong oxidative phosphorylation (OXPHOS) metabolic signature. ST analyses of tumor specimens further confirmed the overlap of mBc2 localization and the regions with elevated OXPHOS activity.
 Based on its high OXPHOS characteristics, we further explored its molecular expression features in an attempt to identify potential therapeutic targets. COX7B, a structural subunit of mitochondrial respiratory chain complex IV, was identified as an mBc2 marker through Venn analysis, specifically overexpressed in the mBc2 subset, the terminal stage of developmental trajectory. In vitroassays demonstrated that COX7B expression is upregulated in activated B-cell-like DLBCL cell lines, and its knockdown significantly reduced cell viability and induced apoptosis, supporting its potential as a therapeutic target.
 The previous studies have shown that clonally expanded CD8+T cells in PCNSL undergo a dynamic transition from a pre-exhausted state to an exhausted state, exhibiting more pronounced T cell exhaustion features compared to systemic DLBCL, suggesting the presence of an immunosuppressive TME in PCNSL. However, the specific mechanisms driving T cell exhaustion and immune suppression remain incompletely understood. Recognizing the critical role of T cells in the TME and in determining therapeutic response, we further investigated the immunosuppressive landscape of T cells and regulatory mechanisms mediated by tumor cells. T cell cluster analysis demonstrated a substantial enrichment of exhausted T cells (exhTc) and regulatory T cells (TregCD4) in PCNSL. Key exhausted molecules (TIGIT, HAVCR2/TIM-3, LAG3, CTLA4, and PDCD1/PD-1) were notably upregulated. ST confirmed high expression of HAVCR2/TIM-3 in T-cell-enriched regions. Importantly, the ligands for HAVCR2/TIM-3, CTLA4 and CEACAM1 were found to be upregulated in malignant B cells, with CD80 (CTLA4 ligands) particularly enriched in B-cell regions of spatial slices. These findings suggested active involvement of mBc in shaping the immunosuppressive TME. To explore intercellular signaling, we used CellChat to construct the communication network within the PCNSL microenvironment. mBc subsets showed stronger interactions with exhTc and TregCD4 compared to other TME cell types, primarily mediated by the macrophage migration inhibitory factor (MIF) signaling pathway. CD74+CXCR4 and CD74+CD44 were identified as key ligand-receptor pairs in this pathway. Analysis of spatial slices revealed elevated expression of CXCR4 and CD74 in tumor regions, further indicating widespread activation of the MIF pathway. These findings suggest that mBc subsets mediate the immunosuppression of exhTc and TregCD4 via MIF signaling.
 In conclusion, this study reveals the intratumoral heterogeneity and immunosuppressive microenvironment of PCNSL, identifies the high-risk COX7B⁺ mBc2 subset, and indicates COX7B as a potential therapeutic target. Additionally, we clarify the key role of MIF signaling in immune suppression, offering new directions for targeted and immune therapies in PCNSL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f0ba610cbed6b065d81ef84a88bd658adbfc55a" target='_blank'>
              Single-cell and spatial transcriptomics analyses decipher malignant B cell heterogeneity and immunosuppressive microenvironment in primary central nervous system lymphoma
              </a>
            </td>
          <td>
            Cong Wang, Tiange Lu, Xiangrong Zheng, Ran Kong, Mei Ding, Xiaohui Sui, Xiangxiang Zhou
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 The World Health Organization provides clear criteria for diagnosing IDH1-wildtype glioblastoma (GBM), relying on traditional approaches to define pathoetiological features, genomic mutations, copy number variation, and aberrant expression of prognostic genes. While these methods guide diagnosis, they may lack the resolution to fully capture tumor heterogeneity, therapeutic response, or outcomes. To address this, we applied leading-edge technologies to 14 primary and recurrent tumors to add granularity to current diagnostic metrics and clinical classification. Aiming to better inform tumor subtyping and improve patient care, this integrative approach enabled us to identify malignant gene programs and localize prognostically relevant features at both single-cell and spatial levels. We analyzed 14 GBM tumors from 6 patients using spatial in situ sequencing (ISS) and single-nuclei RNA-sequencing. A custom ISS gene panel enabled spatial mapping of tumor-associated cell types and state markers, while matched snRNA-seq from adjacent tumor tissue provided transcriptome-wide profiles. One patient contributed 5 serially collected recurrent tumors, enabling longitudinal analysis of tumor evolution. Profiling these tumors using snRNA-seq identified canonical copy number alterations (+7/–10) and ligand-receptor interaction analysis showed an evocative ANXA1 signaling axis progress across recurrent tumors, which suggests possible therapeutic opportunities. Finally, the dataset enabled subcellular and spatial localization of the prognostic GBM biomarker, EGFRvIII, being enriched in oligodendrocyte progenitor (OPC)-like tumor cells, which provides exciting evidence of these cancer cell subtypes as the putative cell of origin for GBM. By extending beyond conventional diagnostic benchmarks, our molecular profiling reveals new insight into GBM biology that may support more refined tumor subtyping and prognostic assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb079e9e9eb8a8e1b502483557634499fe4e750" target='_blank'>
              BIOM-72. INTEGRATIVE SINGLE-NUCLEI SPATIAL TRANSCRIPTOMIC PROFILING OF PRIMARY AND RECURRENT HUMAN IDH-WILDTYPE GLIOBLASTOMA REVEALS CELL-SPECIFIC EGFRVIII EXPRESSION AND ANXA1 SIGNALING AXIS
              </a>
            </td>
          <td>
            Lauren Powers, Vaibhav Jain, Michael Aksu, Emily Hocke, Alan Smith, Stephanie Arvai, Ellora Haukefrers, Kevin Stevenson, Khooshbu Patel, Diane Satterfield, Elizabeth Thomas, Karen Abramson, Giselle López, R. McLendon, David M. Ashley, Simon Gregory
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Uveal melanoma (UM) is the most common intraocular malignancy, with poor prognosis in metastatic cases and limited response to conventional therapies. Despite advances in genetic stratification, the immunological landscape of primary UM remains poorly understood. Secondary data generation of single-cell RNA sequencing (scRNA-seq) of primary class 1 and class 2 (loss of BAP1) UM tumours and flow cytometric analysis of 8 primary UM tumour biopsies were used to characterize the tumour microenvironment, cellular composition, tumour–immune cell interactions, and stromal marker expression associated with tumour progression and immune infiltration. scRNA-seq analysis revealed 16 distinct cell clusters, including melanocytes, T cells, macrophages, and stromal cells. Class 2 tumours contained unique melanocyte subpopulations exhibiting chromosome 8 copy number variations and enriched in hypoxia, PI3K-Akt, and MAPK signalling pathways. Ligand–receptor analysis identified extensive interactions between these aggressive melanocytes and pericytes/macrophages. Flow cytometric analysis confirmed two distinct immune infiltrate profiles: low-infiltrate tumours dominated by CD14⁺ cells, and high-infiltrate tumours with CD8⁺ memory-like T cells expressing PD-1 and CD27. Stromal marker analysis revealed elevated expression of CD81 and NGFR in immune-excluded tumours, implicating them in metastatic potential. Our study reveals cellular and immunological heterogeneity within primary UM tumours. The identification of immunologically distinct tumour types, along with aggressive melanocyte subpopulations and stromal interactions, provides insight into UM pathogenesis and supports stratified immunotherapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddff61a0a880b01ec6e97ff402716c2ca58aa31b" target='_blank'>
              Identification of two distinct uveal melanoma subtypes based on the immune cell infiltration of the primary tumour
              </a>
            </td>
          <td>
            Konstantinos Panagiotidis, S. Owens, Gráinne Finegan, Noel Horgan, Stefanos Tsavdaridis, S. Kennedy, Achilleas Floudas
          </td>
          <td>2025-10-23</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769ab003240fead51b76ade414628f11f7033a1a" target='_blank'>
              Dedifferentiation-Driven Oncogenic Stemness Promotes Tumor-Sustaining Adaptability in the Intestinal Epithelium
              </a>
            </td>
          <td>
            Kylee Zgeib, Thomson Hui, Simon Garcia, Zahra Hashemi, Shima Nejati, Crystal Lim, Dahlia Matouba, Atharva Inamdar, Christina Li, Hossein Khiabanian, Binfeng Lu, Ansu Perekatt
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Epithelial compartments play a central role in the progression and metastasis of colorectal cancer (CRC), yet the mechanisms underlying their transcriptional reprogramming across disease stages remain incompletely understood. We constructed comprehensive single-cell and spatial transcriptomic atlases encompassing normal colorectal tissue, primary CRC, and liver metastases (LMs), profiling over 200,000 high-quality cells. Specific coexpression modules and transcriptional regulators were identified and spatially mapped. Additionally, the clinical and functional roles of SCAND1 were investigated using western blot, RT-qPCR, and immunohistochemistry in clinical samples, as well as functional assays in CRC cell lines. Distinct coexpression modules were selectively enriched in normal epithelial cells and associated with vesicle transport, ion homeostasis, and barrier function, whereas LM-specific modules were linked to Wnt signalling, ribosome biogenesis, and cell cycle processes. High-CNV epithelial cells exhibited progressive activation of key transcription factors such as CEBPB, E2F, and the AP-1 family, with concurrent suppression of ARGLU1 and MTNR2L8, indicating coordinated remodelling of transcriptional and metabolic programs during metastasis. These changes were spatially validated and correlated with alterations in epithelial composition within the tumour core. Consistently, multiplex immunofluorescence (mIHC) confirmed the expression patterns of CEBPB, ARGLU1, S100A4 and FOSL1, across normal epithelium, CRC, and LM. Notably, SCAND1 was identified as a potential therapeutic target for malignant epithelial plasticity. We demonstrated for the first time that SCAND1 expression is significantly upregulated in CRC patients and is associated with superior diagnostic and prognostic value. Functional experiments revealed that SCAND1 promotes tumour cell proliferation, inhibits apoptosis, and enhances metastatic potential. Furthermore, co-culture of SCAND1-overexpressing tumour cells with T cells showed that SCAND1 overexpression attenuates T cell–mediated cytotoxicity, supporting an additional immune-evasive role. Our study elucidates the molecular logic of epithelial state transitions during CRC metastasis and underscores the promise of SCAND1 as a novel biomarker and therapeutic target in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/914a9adf8fcce69d1ec7290317d881a8d12ed794" target='_blank'>
              Dynamic remodelling of epithelial plasticity in colorectal cancer from single-cell and spatially resolved perspectives
              </a>
            </td>
          <td>
            Chengyuan Xu, Yun Pan, Siqi Zhang, Moubin Lin, Hailong Liu, Zicheng Yu
          </td>
          <td>2025-11-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Muscle-invasive (MI) urothelial carcinoma (UC) is a clinically challenging malignancy with a poor prognosis. Understanding the cellular dynamics that drive UC progression is critical for the development of optimized therapeutic strategies. Through integrative analysis of large-scale single-cell transcriptomic datasets from non-muscle-invasive (NMI) and MI tumours and validation with spatial transcriptomic datasets, we systematically characterized immune cell dynamics and cancer cell plasticity during UC progression. Our analysis revealed an immunosuppressive tumour microenvironment and a subset of cancer cells with upregulated major histocompatibility complex II (MHC-II) expression in MI tumours. Notably, MHC-II⁺ cancer cells were induced by interferon-γ signalling, as confirmed through in vitro experiments, and exhibited phenotypic alterations characterized by enhanced proliferative and migratory capacities. Furthermore, MHC-II⁺ cancer cells spatially colocalized with CD8⁺ T cells, regulatory T cells, and SPP1⁺ macrophages, where they engaged with inhibitory receptors on these immune cells, promoted CD8⁺ T cell exhaustion and facilitated immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27b9542e0889bc085781e9b9db8fbe77cfa48b4c" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveal cellular heterogeneity and cancer cell-intrinsic major histocompatibility complex II expression in urothelial carcinoma
              </a>
            </td>
          <td>
            S. Xiong, Jian Fan, Cong Huang, Shi-ming He, Yuan Liang, Qi Zhang, Bin Guo, Yucai Wu, Mancheng Xia, Fangzhou Zhao, W. Ci, Liqun Zhou, Yanqing Gong, Xuesong Li
          </td>
          <td>2025-10-20</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Resistance to chemotherapy, targeted agents, and particularly immunotherapy remains the principal challenge in the management of gastrointestinal malignancies. This review aims to comprehensively delineate the molecular and microenvironmental drivers of resistance, with emphasis on mechanisms impacting immunotherapy response, and evaluate emerging, mechanism−guided interventions (including immunotherapeutic combinations) for precision therapy. We first examine intrinsic mechanisms—including drug−target alterations, dysregulated drug metabolism and efflux, hyperactivation of DNA damage repair pathways, and epigenetic remodeling—and extrinsic influences stemming from the tumor microenvironment and extracellular matrix remodeling. We then highlight epithelial–mesenchymal transition (EMT) as a critical nexus that integrates stromal cues with cell−intrinsic survival programs, thereby promoting drug efflux and immune evasion. Next, we discuss how single−cell and spatial omics, liquid biopsy, patient−derived organoids, and AI−enabled analytics facilitate subclone−level mapping of resistance networks and real−time tracking of clonal evolution. Finally, we review mechanism−based strategies—including KRAS G12C inhibitors, efflux−pump antagonists, apoptosis reactivators, and epigenetic/autophagy modulators—and propose an integrated, multimodal regimen leveraging immunotherapy where appropriate, informed by real-time drug sensitivity data (e.g., from liquid biopsy), dynamic biomarkers and AI−driven optimization to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebd9ae366f20576b8816fc8c01cf3154ec38755" target='_blank'>
              Integrated molecular and microenvironmental drivers of drug resistance in gastrointestinal cancers: mechanisms, immunotherapy challenges, and precision strategies
              </a>
            </td>
          <td>
            Heng Xu, Jiaan Lu, Jiangying Wu, Kangling Zhang, Xuancheng Zhou, Ziqi Gao, Jingqi Feng, Ziye Zhuang, Xiaolin Zhong
          </td>
          <td>2025-10-28</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Introduction
 Classical Hodgkin Lymphoma (cHL) is one of the most manageable human cancers. The early identification of patients at risk of relapse following front-line therapy, currently represents an unsolved clinical need. cHL is a complex ecosystem characterized by an intricate and mutual interaction between an extensively dominant tumor microenvironment (TME) and malignant Hodgkin Reed-Sternberg (HRS) cells; disease progression likely reflects some innate features of this organization. Nevertheless, even the most advanced prognostic models fail to fully account for the clinical heterogeneity of the disease. A major limitation lies in the inherent difficulty of profiling HRS cells, given their rarity and unique distribution within the tumor parenchyma. In this study, we employed an integrated multidimensional transcriptomic analysis to explore the peculiar architecture of the cHL tumor ecosystem and its potential relationship with clinical progression and patient outcomes.
 Methods
 Patient cohort: A retrospective training cohort of untreated cHL patients (N=155), age < 65 years and including all disease stages, was selected. Patients were classified as either relapsed/refractory (R/R+) (N=31) or non-relapsed (R/R-) (N=124) based on a minimum follow-up of 3 years.
 A morphology-guided spatial transcriptomics approach was used to analyze the transcriptional profile of HRS and TME components in surgical lymph node resections of a matched R/R-:R/R+ case-control subset of the cohort. A total of 118 Areas of Interest (AOIs) were collected, 56 from the TME (CD45+/CD30-) and 62 from HRS-rich regions (CD30+). Digital barcoding profiling was used to assess the expression of 770 immune-related genes across the entire training cohort. Cox regression analysis was used to identify genes significantly associated with progression-free survival (PFS). IHC was used to confirm the validity of the derived model.
 Results
 Morphology-guided profiling of HRS cells showed significant differences between R/R+ and R/R- cHLs. Unsupervised clustering based on the transcriptional profile of HRS cells strongly segregated AOIs according to clinical classification. Topographical organization of HRS within the tumor parenchyma, along with paracrine signaling, accounted for HRS differential properties in the two groups. The TME in R/R+ cHLs exhibited an immunosuppressed phenotype, defined by the exclusion of non-tumoral B-cells and overrepresentation of immunosuppressive populations (M2 Macrophages, Tregs and Mast cells). This phenotype was associated with altered immune-modulatory signaling by HRS cells, involving inhibition of stimulatory mediators such as chemotactic cytokines (CXCL9, CXCL10, etc) and IFN signaling and the upregulation of inhibitory molecules including CCL17 (encoding for TARC) and IL13. An inverse correlation was observed between the presence of non-tumoral B-cells and immunosuppressive populations. Digital barcoding gene expression analysis on the entire cohort (N=155) confirmed this model and identified a non-tumoral B-cell specific gene signature (N=18) that significantly correlated with improved PFS and stratified patients according to risk of progression (HR 0.23 95% CI 0.06-0.87; p=0.03)
 Conclusion
 This study provides, one of the first molecular characterization of HRS cells and of their interplay with the surrounding TME, uncovering mechanisms that drive immune escape and disease progression. Notably, our findings reveal a previously unrecognized protective role of non-malignant B cells residing in the TME of cHL, highlighting their potential utility as biomarkers for risk stratification in cHL. Finally, our data demonstrate that integrating high-resolution transcriptional profiling with morphology enables a more precise understanding of cHL complexity, capturing not only intrinsic variability but also the reciprocal spatial organization between cell populations, which significantly shape the biological properties of the entire tumor ecosystem.
 REFERENCES
 1. Scott, DW &Gascoyne, RD The tumour microenvironment in B cell lymphomas. Nat. Rev. Cancer 14, 517–534 (2014)
 2. Aoki, T. et al. Single-cell transcriptome analysis reveals disease-defining t-cell subsets in the tumor microenvironment of classic hodgkin lymphoma. Cancer Discov.10, 406–421 (2020)
 3. Aoki T. et al. Spatially Resolved Tumor Microenvironment Predicts Treatment Outcomes in Relapsed/Refractory Hodgkin Lymphoma. J Clin Oncol. doi: 10.1200/JCO.23.01115. (2024)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e25acab9e1dc7d742c876b8b8b809c99b9c1c2e9" target='_blank'>
              Morphology guided spatial transcriptomics to study topography of Hodgkin reed sternberg cells interaction with tumor microenvironment and its impact in defining risk of progression in classical Hodgkin lymphoma.
              </a>
            </td>
          <td>
            B. Donati, T. Lazic, Riccardo Valli, R. Durmo, Attilio Gennaro, Cristian Ascione, Pierluigi Di Chiaro, A. Ruffini, M. Gotti, Luca Arcaini, Erica Travaglino, Marco Paulli, Francesco Merli, A. Ciarrocchi, S. Luminari
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Primary prostate cancer presents with multifocal lesions and unpredictable clinical behavior, posing significant challenges for effective prognosis. To address this, we investigate the epigenomic landscape of prostate tumor biopsies from 25 treatment-naïve male patients by analyzing chromatin compartmentalization patterns. Our analysis reveals two distinct molecular subtypes: one with a Low Degree of Decompartmentalization (LDD) and another with a High Degree of Decompartmentalization (HDD). Here we show that the HDD subgroup exhibits extensive chromatin reorganization associated with diminished oncogenic potential. This subtype shows repression of molecular pathways involved in extracellular matrix remodeling and cellular plasticity. From this distinction, we derive an 18-gene transcriptional signature capable of differentiating HDD from LDD cases. Importantly, we validate the prognostic relevance of this signature in multiple independent cohorts totaling over 900 patients. Our findings suggest that epigenetic-derived signature at the time of diagnostic biopsy can offer a powerful tool for risk stratification in prostate cancer. Prostate cancer (PCa) evaluation remains challenging due to its heterogenous and multifocal nature. Here, the authors reconstruct the nuclear chromatin compartmentalization of PCa patients-derived biopsies distinguishing two epigenetic subtypes and deriving an 18-gene signature with prognostic value on TCGA samples that is further validated with multiple independent PCa cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696cb38bd331720bed277dcfa9f4701c78f389c7" target='_blank'>
              Chromatin remodeling restrains oncogenic functions in prostate cancer
              </a>
            </td>
          <td>
            V. Rosti, Giovanni Lembo, C. Petrini, Francesca Gorini, Roberto Quadri, C. Cordiglieri, Margherita Mutarelli, Elisa Salviato, Elisabetta Casari, Emanuele Di Patrizio Soldateschi, Emanuele Montanari, G. Albo, Francesco Ripa, Alessandra Fasciani, Mariacristina Crosti, E. De Lorenzis, M. Maggioni, V. Vaira, Maria Vivo, Francesco Ferrari, Chiara Lanzuolo
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Plasticity, or the ability to rapidly and reversibly change phenotypes, may help explain how a single progenitor cell eventually generates a tumor with many different cell phenotypes. Normal colon plasticity is characterized by a conserved and broadly permissive epigenome, where expression and phenotype are determined by the microenvironment instead of epigenetic remodeling. To determine whether this stem-like plasticity is retained during progression, gene expression was measured with spatial transcriptomics and compared with gene-level DNA methylation in two colorectal cancers (CRCs). Like normal colon, genes that were differentially expressed between regions, subclones, and phenotypes (superficial, invasive, and metastatic) tended to have lower DNA methylation variability. We propose a quantitative signal of plasticity that correlates gene epigenetic variability with gene expression variability. In this framework, negative correlation implies phenotypic plasticity, as more variably expressed genes tend to have less epigenetic variability. We verify the presence of this signal in multiple external single-cell RNA-Seq datasets, in both normal colon and CRC samples. Therefore, the plasticity of normal colon appears to be retained during progression. A CRC progenitor with a preconfigured plastic phenotype is poised for rapid growth because it expresses, as needed, transcripts required for progression with minimal epigenetic remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86cc1dfed608d8a0144edef3180374f9ebee40c0" target='_blank'>
              Epigenetic conservation infers that colorectal cancer progenitors retain the phenotypic plasticity of normal colon
              </a>
            </td>
          <td>
            Kelly Street, Yifan Zhang, Kimberly Siegmund, Darryl Shibata
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Single-cell & MR link ESCC immunotherapy to immune cells. Combined single-cell datasets yield more credible results. ESCC-immune cell discovery aids subsequent experimental validation. Esophageal cancer has a low 5-year survival rate despite treatments. scRNA-seq and MR offer insights into neoadjuvant treatment efficacy for precision medicine. Three post-neoadjuvant treatment datasets underwent QC for Seurat analysis. Marker genes identified cell subsets. MR analyzed eQTL data from GWAS cohorts for causal links. Single-cell and MR-derived genes intersected to reveal PLTP in CD4⁺T cells. Single-cell analysis of 16 samples found 40,198 genes and 120,102 cells. CD4⁺T cell numbers differed significantly between groups after therapy. Differentially expressed genes were immune-related. Pseudotime and cell-cell communication varied. PLTP, linked to esophageal cancer, co-expressed with genes involved in cell cycle processes. The study highlights CD4⁺T cells’ predictive role in therapy efficacy via scRNA-seq and MR. PLTP emerges as a key gene, offering new precision medicine strategies for esophageal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cc89377f616ee4461240aafc742e5dad712a1f6" target='_blank'>
              Integrated single-cell and Mendelian randomization analyses: dissecting underlying causes of varied efficacy in immune neoadjuvant therapy for esophageal carcinoma
              </a>
            </td>
          <td>
            Jiaxin Li, Sibo Meng, Ying Zhou, Yufeng Cheng
          </td>
          <td>2025-10-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intratumoral heterogeneity supports cancer progression and metastasis while limiting therapeutic efficacy. Comprehensive biological characterization of heterogeneous subclones in breast cancer may hold promise in battling against this deadly disease. Using single-cell transcriptomic analysis and in situ profiling of primary tumors, metastases, and circulating tumor cells from multiple patients with breast cancer, we identified a unique EPCAM+ CD68+ TREM2+ tumor subpopulation, likely resulting from the fusion of tumor cells and lipid-associated macrophages (LAM). The presence of these tumor-LAM fusion cells in the blood or in distinct metastatic sites was significantly correlated with metastatic progression. Stable fusion clonal lines established in vitro exhibited substantially enhanced proliferation, tumor initiation, and metastasis formation in mice. Integrative molecular and functional analyses revealed a critical role for SNX10 in mediating tumor-LAM fusion. Mechanistically, SNX10 physically interacted with the phospholipid scramblase ANO6 and maintained its protein stability by suppressing proteasome-mediated degradation. Furthermore, the transcription factor CBX3 directly targeted SNX10 promoter and modulated its expression. Fusion cells accumulated abundant intracellular lipid droplets and were highly sensitive to simvastatin treatment in vitro and in vivo. Together, this study uncovered that CBX3-SNX10-ANO6 signaling facilitates generation of an aggressive tumor-LAM fusion cell subpopulation that promotes metastasis, revealing an alternative metastatic mechanism and exposing putative therapeutic vulnerabilities.


SIGNIFICANCE
Single-cell transcriptomic profiling combined with functional and clinical validation identifies fusion of tumor cells and lipid-associated macrophages mediated by the CBX3-SNX10-ANO6 axis as a potentially targetable mechanism driving cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4271e1aeed14ff3c28d665da96c160262a128c0a" target='_blank'>
              Fusion of Tumor Cells with Lipid-Associated Macrophages Drives Metastatic Progression of Breast Cancer.
              </a>
            </td>
          <td>
            Yixin Cheng, Guanyin Huang, Xuefei Liu, Chun Wu, Jingru Lian, Jing Cao, Lianhui Duan, Boxi Zhao, Yufan Yang, Siqi Chen, Guanqiao Li, Liping Wu, Shuqian Zheng, Wenjing Wang, Tianyu Zha, Yang Yang, Yilin Lu, Jian Zhang, Hao Yu, Jun Tan, Feiqiu Wen, Zhenyu He, Xin Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background: Intrahepatic cholangiocarcinoma (ICC) has a poor prognosis due to late-stage presentation and ineffective systemic therapies. Targeting the tumor microenvironment (TME) in ICC offers new therapeutic possibilities, particularly through tumor-associated macrophages (TAM), which can both promote and inhibit tumor progression. The current study utilized multi-omics analysis to characterize the gene signature of TAM and explore its therapeutic potential in ICC. Methods: Public GEO datasets provided the basis for analysis. Single-cell RNA sequencing (scRNA-seq) data from five ICCs, three adjacent non-tumorous tissues (ANTs), and four healthy liver samples were examined with Python. To validate scRNA-seq findings, bulk RNA-seq data from 27 ICC and 27 matched ANT samples were assessed using R. Differentially expressed genes were identified with adjusted p-values <0.01 and log2-fold changes >1 or <−1. CIBERSORT pipeline analyzed 22 immune cell subtypes in bulk RNA-seq data. STRING database analyzed the contribution of unique TAM-related genes to networks of protein–protein interactions. Results: TAM population demonstrated phenotypic heterogeneity exhibiting partial gene signatures of inflammatory (MS1) and anti-inflammatory (MS2) macrophages. Unique TAM-associated markers, TREM2, CD9, and PRMT10, showed variable expression within the TAM subpopulation. Bulk RNAseq analysis confirmed the scRNA-seq results, highlighting overexpression of TREM2 and CD9 in most ICC samples versus ANT. Immune cell deconvolution revealed decreased MS1 and MS2 macrophages in ICC, and alterations in adaptive immune profile, suggesting immunotolerant TME. STRING database defined TREM2-LGALS3 axis as a potential target for anti-tumor therapies. Conclusions: TAM represents a unique heterogenous population which is primarily found in ICC TME versus ANT or healthy liver tissue The non-uniform expression of unique gene signature demonstrates additional heterogeneity in the TAM subpopulation and suggests that TREM2+ TAM may be desirable targets for anti-TREM2-LGALS3 immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be67d95f17acd65abd3873af222e463c89ca98c4" target='_blank'>
              Defining Gene Signature of Tumor-Associated Macrophages in Intrahepatic Cholangiocarcinoma as Target for Immunotherapy Using Single Cell and Bulk RNA Sequencing
              </a>
            </td>
          <td>
            J. S. Badshah, Ryan M. Lee, Andrea Reitsma, M. Melcher, Olivia M. Martinez, Sheri M. Krams, Daniel J. Delitto, Varvara A. Kirchner
          </td>
          <td>2025-10-29</td>
          <td>Livers</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5bf5750b51fd180863b176de6d4f229a2cae4e" target='_blank'>
              Single-cell and spatial transcriptomics reveal lactylation-associated tumor cell clusters and define a prognostic risk model in glioblastoma.
              </a>
            </td>
          <td>
            Rui Han, Guangfan Chi, Dongjie Sun, Ziran Xu, Liangfu Zhou, Kan Xu
          </td>
          <td>2025-11-22</td>
          <td>BMC cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b5111159e92c7f129ff30372c9b9934bfcdb2c" target='_blank'>
              Single-cell RNA sequencing and large-scale bulk combination with machine learning reveal gastric cancer-related macrophage heterogeneity
              </a>
            </td>
          <td>
            Saisai Gong, Yu peng Zhao, Sheng Yang, Tong Liu, Chengyun Li, Yun Zhou, Geyu Liang
          </td>
          <td>2025-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce05a3966abc2d6663f2f9fc06215e6d78446ea" target='_blank'>
              Tumor microenvironment-mediated immune evasion and resistance in prostate cancer: mechanisms, cross-talk, and therapeutic opportunities.
              </a>
            </td>
          <td>
            Mingfeng Li, Louis Boafo Kwantwi, Cuimei Wang, Qin Xiao
          </td>
          <td>2025-11-26</td>
          <td>Clinical and experimental medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Brain-invasion of meningioma is a hallmark of malignant behavior and linked to higher risk of surgical resection and poor clinical outcome. Here, we hypothesized that meningioma tumor ecosystems with invasive behavior enhance cellular communication with residual brain cells allowing an integration of tumor cells into neuronal circuits. To this end, we profiled 42 tumors with and without brain invasive pattern leveraging single cell (Xenium) and array-based (Visium) spatially resolved transcriptomics followed by a spatial graph attention model (sGAT) trained to identify the distance of tumor ecosystems to the brain-tumor boarder. Leveraging explainable AI techniques (integrated gradients and attention) we identified enhanced SPP1+ myeloid cells enriched at the infiltrative boarder. Cellular neighborhood analysis identified enriched proximity of pro-inflammatory myeloid cells, activated astrocytes and excitatory neurons within a distance of 50-100µm to the infiltrative boarder followed by OPC- and oligodendrocytes enhancement peaking at at 200µm distance. To further investigate the tumor-neuronal interaction we leveraged tumor neuronal co-culture systems with genetically diverse meningioma cell models followed by rabies-ΔG-GFP retrograde tracing model demonstrating high degree of neuronal connectivity after 5 days co-culture. To confirm electrically synaptic transmission, patch-clamp and ca2+ imaging revealed functional synaptic contact predominantly activated by acetylcholine (puff). Cross-validation with spatial and single cell transcriptomic data revealed a meningioma neuron-like cell population enriched for acetylcholine receptor expression and localized mainly in the invasive front. Using super-resolution microscopy in brain-invasive meningioma specimens, we demonstrated putative neuron-to-tumor synapses at brain-tumor interfaces. In summary, invasive meningioma are marked by inflammatory reshape activating microglia and astrocytes and accumulation neuronal input on meningioma tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22186b7a6b175f66d08a9c29db91c360e9425639" target='_blank'>
              TMIC-86. Meningioma brain-invasion is linked to inflammatory induced tumor-neuronal connectivity
              </a>
            </td>
          <td>
            J. Benotmane, Yiheng Tang, B. Patel, D. Savran, M. Ratliff, C. Herold-Mende, Sandro M. Krieg, Wolfgang Wick, A. von Deimling, M. Gerstung, I. Bludau, Ekin Reyhan, Svenja K. Tetzlaff, Nina Drewa, Elijah Keifert, Mahammad Bashirov, N. Layer, Y. Yabo, L. van Hijfte, Sybren L. N. Maas, V. Venkataramani, D. Heiland, F. Sahm
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba29f656998508469f49c53f9437fe122dadd98" target='_blank'>
              DYRK1A regulates cancer cell and cancer-associated fibroblast secretomes to foster an immunosuppressive microenvironment in pancreatic cancer
              </a>
            </td>
          <td>
            Silvia Pascual-Sabater, Sushmita Varhadi, Chiara Di Vona, Marta Celma, Cristina Fillat, S. de la Luna
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Gliomas and brain metastases are associated with poor prognosis, necessitating a deeper understanding of brain tumor biology and the development of effective therapeutic strategies. Although our group and others have demonstrated microbial presence in various tumors, recent controversies regarding cancer-type-specific intratumoral microbiota emphasize the importance of rigorous, orthogonal validation. This prospective, multi-institutional study included a total of 243 samples from 221 patients, comprising 168 glioma and brain metastases samples and 75 non-cancerous or tumor-adjacent tissues. Using stringent fluorescence in situ hybridization, immunohistochemistry and high-resolution spatial imaging, we detected intracellular bacterial 16S rRNA and lipopolysaccharides in both glioma and brain metastases samples, localized to tumor, immune and stromal cells. Custom 16S and metagenomic sequencing workflows identified taxa associated with intratumoral bacterial signals in the tumor microenvironment; however, standard culture methods did not yield readily cultivable microbiota. Spatial analyses revealed significant correlations between bacterial 16S signals and antimicrobial and immunometabolic signatures at regional, neighborhood and cellular levels. Furthermore, intratumoral 16S bacterial signals showed sequence overlap with matched oral and gut microbiota, suggesting a possible connection with distant communities. Together, these findings introduce microbial elements as a component of the brain tumor microenvironment and lay the foundation for future mechanistic and translational studies. A combination of high-resolution spatial imaging, spatial proteomics and transcriptional data reveals sparse and heterogeneous bacterial signals in gliomas and brain metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae0d1782181d2f2125dbd26294df4180d5bf9d82" target='_blank'>
              Microbial signals in primary and metastatic brain tumors
              </a>
            </td>
          <td>
            Golnaz Morad, Ashish V Damania, Brenda Melendez, Bharat Singh, Fabiana J. Veguilla, Rebecca A. Soto, Yasmine M. Hoballah, P. Sahasrabhojane, M. Wong, Mona M. Ahmed, Rene N. Rico, Kaitlyn N. Lewis, Khalida M Wani, D. Shamsutdinova, Rossana N. Lazcano Segura, D. Ingram, Eric A. Goethe, Abderrahman Day, Ivonne I Flores, Lauren K McDaniel, M. Chelvanambi, Sarah B. Johnson, Florentia Dimitriou, Pravesh Gupta, Shivangi Oberai, M. Zal, Phoebe Doss, Mohamed A Jamal, Eiko Hayase, Chetna Wathoo, Lisa M. Norberg, Stephanie L. Jenkins, Sara Nass, J. Gumin, L. Long, Jing Yang, Gina R. Bradley, M. Bekal, A. Dono, P. Pichardo-Rojas, Samuel W. Andrewes, Leomar Ballester, J. Losh, Jiyong Liang, Longfei Huo, Douglas Nielsen, Brittany C Parker Kerrigan, Priscilla K. Brastianos, N. Fowlkes, Chia-Chi Chang, Robert R. Jenq, C. Gomez-Manzano, Jason Huse, Michael A. Davies, A. Lazar, Krishna P. Bhat, Nitin Tandon, Y. Esquenazi, Christine B. Peterson, Vinay K. Puduvalli, Frederick F. Lang, Christopher D. Johnston, Susan Bullman, N. Ajami, S. Ferguson, J. Wargo
          </td>
          <td>2025-11-01</td>
          <td>Nature Medicine</td>
          <td>1</td>
          <td>49</td>
        </tr>

        <tr id="


 Pancreatic neuroendocrine tumors (panNETs) undergo high-grade (HG) transformation often, with the liver being a common site for metastasis​. Surgical resection being considered as long-term outcomes for selected patients. However, recurrence and grade transformation remain potential significant challenges, with recurrence rates approaching 50–95%.1,2 Despite extensive whole exome analysis and construction of phylogenetic trees, we were unable to identify either a consistent genetic driver or any specific patterns of tumor transformation.2 We are also concerned that some of our treatments may inadvertently contribute to grade transformation. To assess the treatment effects, we performed pre- and post- trans arterial embolization bulk-transcriptome analysis, which revealed cell population shifts in panNETs that may be linked to grade transformation.4To explore cellular shifts within the liver tumor microenvironment during panNET progression, we performed single-cell RNA sequencing. The resulting data was processed using Seurat.v5. The clusters of panNET cells, fibroblasts, hepatocytes, and immune cells were annotated by known markers. Additionally, we distinguished malignant cells from a reference of immune cells, fibroblasts, and hepatocytes by InferCNV. To understand the tumor progression, we performed trajectory analysis using monocle3. Differential metabolic activity between the panNETs grades were assessed using the scMetabolism package. Cell-cell communication and related ligand-receptor pairs were inferred using CellChatDB. Our results revealed distinct transcriptional trajectories, with pseudotime analysis demonstrating a trajectory of tumor evolution from low-grade (LG) to HG states. HG tumors were enriched for proliferative and aggressive subclusters compared to LG cells. Metabolic profiling further highlighted grade-specific adaptations, with HG tumors showing enhanced oxidative phosphorylation and TCA cycle across tumor cells, whereas LG tumors displayed relatively lower metabolic activity. Notably, immune cells were found metabolically active in LG via glycolysis. Conversely, HG panNETs exhibit glucose depletion in immune cells and less active microenvironment. The CellChat analysis revealed distinct cellular communication patterns in panNET grade. For instance, in HG tumors, signaling was primarily mediated between fibroblasts and panNET cells, with Glu-(SLC1A2+GLS)-GRM7 as the dominant ligand–receptor pair. In LG tumors, major interactions occurred among immune cells, hepatocytes, and panNET cells, with Glu-(SLC1A3+GLS)-GRIK2 representing the highest contributor. The results suggest that metabolic rewiring, altered intercellular signaling, and transcriptional plasticity underlie progression from LG to HG disease.
 Reference:
 1. Spolverato etal J Gastrointest Surg. 2017, 21(12):2039-2047
 2. Mayo etal Ann Surg Onc. 2011, 18(13):3657-65.
 3. Singh et al. Early ... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium. 2023.
 4. Singh et al. Post... Margie and Robert E. Peterson Neuroendocrine Tumor Research Symposium.2023



 Himanshu N. Singh, Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jungeui Hong, Nitya Raj, Olca Basturk, Zeynep Tarcan, Christine A. Iacobuzio, Etay Ziv. Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f19a2d46016dbf4a94424ac84c3f39682a28b2d" target='_blank'>
              Abstract B025: Metabolic rewiring and cellular crosstalk may drive grade transformation in pancreatic neuroendocrine tumors
              </a>
            </td>
          <td>
            , Chenyang Zhan, Jessica C. Puzzuoli, Elias-Ramzey Karnoub, Jung-Pyo Hong, Nitya Raj, O. Basturk, Z. Tarcan, Christine A. Iacobuzio, E. Ziv
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Therapeutic resistance remains a critical barrier in oncology, frequently leading to cancer relapse after initial treatment response. Growing evidence suggests the presence of drug-tolerant persisters (DTPs), a rare subpopulation of cancer cells that survives chemotherapy by entering a reversible specific adaptation. Unlike classical cell resistance, the DTP phenotype is independent of genetic changes and maintained through dynamic regulatory mechanisms. DTPs are phenotypically heterogeneous and can exhibit stem-like and quiescent cell phenotypes, non- or slow proliferation, and remarkable plasticity due to a di-pause-like state and executing epithelial–mesenchymal transition (EMT) or transdifferentiation processes. Despite advances in research, the molecular mechanisms underlying DTPs’ biology and their role in cancer relapse remain only partially understood. The review summarizes the current progress in processes that lead to the acquisition of cellular persistence status, which, in turn, constitute areas of vulnerability that can be exploited in cancer therapy. We highlight anti-DTP therapeutic strategies, including epigenetic modification, cell signaling and transcriptional regulation, metabolic reprogramming, and modification of cell interactions within the tumor microenvironment. Furthermore, we focus on the potential role of nanomaterials in the combat against DTPs. Nanoparticles not only act as part of the drug delivery process, enabling precise DTP targeting and enhancing intracellular drug accumulation, but their intrinsic properties can also be used to eradicate DTPs directly or by enhancing the effectiveness of other therapeutic strategies. The integrated approach offers strong potential to eliminate tumor persistence, prevent recurrence, and improve long-term patient outcomes beyond conventional therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495b98c4c7bc3661dae9aa58d4006e54132b7ae3" target='_blank'>
              Targeting Drug-Tolerant Persister Cancer Cells: Can Nanomaterial-Based Strategies Be Helpful for Anti-DTP Therapies?
              </a>
            </td>
          <td>
            Prachi Ghoderao, Eliza Kwiatkowska-Borowczyk, Sanjay Sahare, H. Dams-Kozlowska
          </td>
          <td>2025-11-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Background: We conducted an open-label, randomized, multicenter trial (NCT06496308) enrolling previously untreated patients with intermediate- or high-risk mantle-cell lymphoma (MCL) who were ineligible for autologous stem-cell transplantation (Order Trail).
 Aims: Using single-cell RNA-seq, we will dissect the microenvironmental mechanisms of resistance in responders versus non-responders, identify biomarkers for efficacy, and characterize the tumor and microenvironmental features of blastoid, refractory patients.
 Methods: Single-cell RNA sequencing (scRNA-seq) were performed on 33 enrolled patients with available samples (25 classical, 6 blastoid, and 2 pleomorphic), yielding 370,392 high-quality cells.
 Results: UMAP clustering identified 21 distinct cell populations, which were further annotated into 13 B cell subclusters, 5 T cell subclusters, 1 myeloid cell population and 1 epithelial population, based on the canonical markers, BCR/TCR profiling, and functional signatures.
 RNA velocity analysis revealed that MCL tumor cells originated from the MCL-Precursor populations and subsequently differentiated into heterogeneous subclusters, including two patient-unique subclusters (Patient1-unique B cells and Patient21-unique B cells). Cell cycle analysis showed elevated S-phase fractions in MCL-6, MCL-7, and MCL-8 subclusters. BCR repertoire analysis revealed that Memory B-like and Plasma cell-like subclusters were enriched in monoclonal B cells, with IgH heavy chains expressing IGHG/IGHA, indicating that these two subclusters have undergone class switching and were closer to normal B cells.
 The immunue microenvironment characterization identified 10 T-cell subclusters, including three CD4+ subclusters, three CD8+ subclusters, 1 NKT population, and 3 TOX-high DNT populations. Concurrently, the cytotoxic score, exhausted score and regulatory score for each T cell subclusters were calculated. Myeloid compartments were predominantly composed of DCs, M1/M2 macrophages, neutrophils and mast cells.
 Subsequently, we compared the characteristics of tumor B cell and immune microenvironment among classical, blastic, and pleomorphic MCL. For B cells, classical MCL showed predominant MCL-3 and MCL-4 B-cell populations, while blastoid cases exhibited MCL-7 expansion. Pleomorphic samples demonstrated exceptional heterogeneity with patient-specific clusters. Pathway analysis showed that cell cycle-related pathways were significantly activated in blastoid B-cells, while pleomorphic cases exhibited B cell differentiation and stemness maintenance signatures. Regarding the immune microenvironment, classical MCL presented higher proportion of CD4+ naïve T cells, while blastoid/pleomorphic subtypes displayed elevated proportions of CD8+ CTLs compared to the classical subtype. Gene Ontology analysis revealed that the activated genes of T cells in blastic MCL were mainly enriched in cytokine secretion pathways, contrasting with the enriched immune-regulation pathways in pleomorphic cases. Myeloid compartments in pleomorphic cases exhibited distinct proportional differences compared to classical and blastic subtypes, with significantly increased mast cells and neutrophils, and reduced macrophages and DCs.
 To investigate the underlying molecular mechanisms between patients with different treatment responses, we compared the proportions of the cell populations between patients achieving CR/PR and those with PD/SD. Compared to CR/PR responders, PD/SD patients had higher MCL-Precursor, MCL-4, DNT-2/3 proportions, but lower MCL-1/5 frequncies, alongside increased DCs and decreased neutrophils and mast cells. Pathway enrichment analysis demonstrated that upregulated genes in B cells from the PD/SD group were significantly enriched in cell cycle-related pathways, compared to the CR/PR group. Additionally, MCL-Precursor of PD/SD responders showed activation of the NF-κB and JAK-STAT signaling pathways, suggesting stronger malignant proliferation and earlier malignant transformation of tumor cells in patients with poor prognosis.
 Summary and conclusion: This is a preliminary micro-environment-based efficacy analysis derived from the ORDER study. We will further interrogate the RNA transcriptomes of responders and non-responders—integrated with single-cell data—to uncover resistance mechanisms and identify novel therapeutic targets for progressive disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06e501ffe60ee00560e8204d50b839bbe699bd32" target='_blank'>
              Single-cell transcriptomic profiling reveals tumor-intrinsic heterogeneity and immune-microenvironment dynamics in the order trial for mantle-cell lymphoma
              </a>
            </td>
          <td>
            Lu Jiang, Xiaozhong Shen, Weili Zhao, Li Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc219a64ec3d466c88c4dfd11649123286e2032b" target='_blank'>
              Cross-Species Insights from ART-D to Uncover Evolutionarily Conserved Oncogenic Mechanisms
              </a>
            </td>
          <td>
            Yifan Guo, Jiadong Zheng, Yixin Wu, Kun Zhao, Dian Lv, Wenhan Liu, Mengling Wang, Jin-Yu Lu, Wenyan Xu, Xianping Wang, Xianjue Ma
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC) continues to represent a major cause of cancer-related mortality worldwide, with its progression and therapeutic outcomes strongly shaped by the complexity and heterogeneity of the tumor immune microenvironment (TME). This review critically examines the cellular and molecular mechanisms driving immune evasion in CRC, emphasizing the dual roles of immune cell populations-including tumor-associated macrophages, neutrophils, dendritic cells, T cells, B cells, and natural killer cells-as well as non-cellular elements such as the extracellular matrix and extracellular vesicles.


OBJECTIVE
A key objective is to evaluate recent developments in immunotherapeutic approaches, including immune checkpoint inhibitors, tumor vaccines, adoptive cell transfer, and novel combinatorial regimens, while addressing their therapeutic promise and inherent limitations, especially in microsatellite-stable (MSS) tumors that exhibit primary resistance to standard immunotherapies. Further analysis integrates perspectives on metabolic reprogramming within the TME, epigenetic alterations, and advances in engineered cellular therapies, thereby providing a comprehensive framework for overcoming immunosuppressive mechanisms.


DISCUSSION AND CONCLUSION
Special consideration is directed toward the translational value of targeting immune-metabolic interactions and spatial dynamics within the TME. Ultimately, this work synthesizes current knowledge and outlines forward-looking strategies to advance personalized, multi-target immunotherapy, with the potential to reshape clinical paradigms in CRC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03fb3a1b140db8c8ae9afa2ff97c1f40f6b8cd5d" target='_blank'>
              Cellular characteristics of the immune microenvironment of colorectal cancer and progress in immunotherapy research.
              </a>
            </td>
          <td>
            Chunbaixue Yang, Jianchun Fan, Yixuan Zhang, Yixiao Zhang, Lei Xia, Xinran Cao, Xueliang Wu
          </td>
          <td>2025-11-30</td>
          <td>Annals of medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Human bone marrow stromal cells (BMSC) include skeletal stem cells with ground-breaking the-rapeutic potential. However, BMSC colonies have very heterogeneous in vivo behaviour, due to their different potency; this unpredictability is the greatest hurdle to the development of skeletal regeneration therapies. Colony-level heterogeneity urges a fundamental question: how is it possible that one colony as a collective unit behaves differently from another one? If cell-to-cell variability were just an uncorrelated random process, a million cells in a transplant-bound colony would be enough to yield statistical homogeneity, hence washing out any colony-level traits. A possible answer is that the differences between two originating cells are transmitted to their progenies and collectively persist through an hereditary mechanism. But non-genetic inheritance remains an elusive notion, both at the experimental and at the theoretical level. Here, we prove that heterogeneity in the lineage topology of BMSC clonal colonies is determined by heritable traits that regulate cell-cycle exit. The cornerstone of this result is the definition of a novel entropy of the colony, which measures the hereditary ramifications in the distribution of inactive cells across different branches of the proliferation tree. We measure the entropy in 32 clonal colonies, obtained from single-cell lineage tracing experiments, and show that in the greatest majority of clones this entropy is decisively smaller than that of the corresponding non-hereditary lineage. This result indicates that hereditary epigenetic factors play a major role in determining cycle exit of bone marrow stromal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b8b3d836cf1a27dbadc5ff0c8ab8409486bfd3" target='_blank'>
              Inheritance entropy quantifies epigenetic regulation of cell-cycle exit in human bone marrow stromal cells
              </a>
            </td>
          <td>
            Alessandro Allegrezza, Riccardo Beschi, Domenico Caudo, A. Cavagna, A. Corsi, A. Culla, Samantha Donsante, Giuseppe Giannicola, I. Giardina, Giorgio Gosti, T. Grigera, S. Melillo, B. Palmisano, L. Parisi, L. Postiglione, Mara Riminucci, Francesco Saverio Rotondi
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa8b0639992b1a58eec77b9ba628d8cad1cb0d2" target='_blank'>
              Somatic copy number mutations contribute to fitness in transplantation models of spontaneous human breast cancer metastasis
              </a>
            </td>
          <td>
            Hoa Tran, Gurdeep Singh, Hakwoo Lee, Damian Boon Siew Yap, Eric Lee, William Daniels, Farhia Kabeer, C. O’Flanagan, Vinci Au, Michael Van Vliet, Daniel Lai, Elena Zaikova, Sean Beatty, Esther Kong, Shuyu Fan, Jessica Chan, Hoang Quan Dang, Viviana Cerda, Teresa Ruiz de Algaza, Andrew Roth, Sam Aparicio
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/759c03413ef4b1106fa2a25c75713885026e6884" target='_blank'>
              Single-cell transcriptomic analysis reveals key cell types and pathogenic genes associated with metastasis in pancreatic adenocarcinoma.
              </a>
            </td>
          <td>
            Shijun Shen, Zhiqiang Li, Hong Yang, Xinwei Zhang, Gang Chen, Chengzhou Pa
          </td>
          <td>2025-11-29</td>
          <td>Clinical and experimental medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite advances in understanding systemic immune responses to Mycobacterium tuberculosis (Mtb), the localized immune dynamics within infected lymph nodes, particularly cell-type-specific transcriptional reprogramming and intercellular crosstalk, remain poorly defined, impeding targeted therapy development. By means of single-cell transcriptomics, the objective was to dissect the immune microenvironment and map intercellular crosstalk in Mtb-infected cervical lymph nodes, with the purpose of uncovering the mechanisms of localized immunity and immunopathology. Paired Mtb-positive cervical swollen lymph nodes (SLN) and adjacent Mtb-negative normal-appearing lymph nodes (NLN) from five cervical LNTB patients were analyzed using single-cell RNA sequencing (scRNA-seq). Computational network modeling (CellChat) and flow cytometry validation were employed to map immune cell heterogeneity and cell–cell communication. ScRNA-seq identified ten T cell subsets, ten B cell subsets, and six myeloid subsets, revealing conserved frequencies but substantial transcriptional reprogramming in SLN. The SLN exhibited extensive upregulation of pro-inflammatory pathways across T, B, and myeloid cells, accompanied by minimal alterations in subset frequencies. IL1B + macrophages in SLN showed an enrichment of genes associated with oxidative phosphorylation, antigen presentation, and inflammasome-related genes. The SLN demonstrated an increased cell–cell communication driven by the crosstalk between macrophage and CD8 + T/NKT cells. Validation through flow cytometry confirmed comparable proportions of immune subsets between the SLN and the NLN, which was consistent with the findings of scRNA-seq. This study delineates a spatially coordinated immune strategy in cervical LNTB. In this context, Mtb infection induces transcriptional and metabolic reprogramming rather than subset redistribution. The strengthened interaction between macrophages and T cells emphasizes that cellular immunity serves as the main impetus for bacterial containment. Nevertheless, there are trade-offs between inflammation and tissue integrity. These insights provide a framework for developing therapies that target intercellular networks to achieve a balance between immunity and pathology in LNTB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f7d9d58e6ee25e924ad6baff06b4c1c3d3fa38" target='_blank'>
              Single-cell transcriptomics of cervical lymph node tuberculosis reveals cellular heterogeneity and enhanced cellular communication
              </a>
            </td>
          <td>
            Guohui Xiao, Haijiang Wang, Su Zhang, Xuefeng Zhou, M. Ou, Xiangmei Zhang, Guobao Li, Aimei Liu, Shimin Shuai, Guoliang Zhang
          </td>
          <td>2025-10-21</td>
          <td>BMC Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) represent a small but critical subpopulation within breast tumors that drive tumor initiation, progression, metastasis, therapeutic resistance, and recurrence. BCSCs are characterized by cell surface markers such as CD44⁺/CD24⁻ and high aldehyde dehydrogenase 1 (ALDH1) activity, and are regulated by pluripotency-associated transcription factors including POU class 5 homeobox 1 (OCT4), SRY-Box transcription factor 2 (SOX2), and Nanog homeobox (NANOG). They exhibit remarkable plasticity and heterogeneity across different breast cancer subtypes, which is further enhanced by epithelial-mesenchymal transition and dynamic interactions with the tumor microenvironment, enabling immune evasion and adaptation to hostile conditions such as hypoxia. Functionally, BCSCs sustain tumorigenesis through self-renewal, contribute to metastatic dissemination as circulating tumor cells, and can enter dormancy in distant niches, leading to late recurrences. Their intrinsic resistance to chemotherapy, radiotherapy, and targeted therapies arises from quiescence, enhanced DNA damage repair, overexpression of drug efflux transporters, and metabolic reprogramming, posing major clinical challenges. Recent advances have improved BCSC identification and isolation through marker-based cell sorting, dye exclusion assays, mammosphere culture, and functional reporter systems. Conventional therapies often spare or enrich BCSCs, highlighting the need for novel treatments. Targeted strategies focusing on stemness-related signaling pathways such as Wnt, Notch, and Hedgehog, as well as regulators including NANOG, ALDH1 and programmed death-ligand 1 (PD-L1), are under investigation. Emerging approaches incorporating natural compounds, nanotechnology, and microenvironmental modulation show promise in overcoming therapeutic resistance. This review integrates current understanding of BCSC biology, heterogeneity, and microenvironmental interactions, providing valuable insights for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0824bd9fb688f1edb70aeb158a043b8788637c" target='_blank'>
              Breast cancer stem cells: Biological roles, microenvironmental interactions, and therapeutic targeting
              </a>
            </td>
          <td>
            Mou Yan
          </td>
          <td>2025-10-11</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Classical models of cancer focus on tumour-intrinsic genetic aberrations and immune dynamics and often overlook how the metabolic environment of healthy tissues shapes tumour development and immune efficacy. Here, we propose that tissue-intrinsic metabolic intensity and waste-handling capacity act as an upstream gatekeeper of anti-tumour immunity, determining whether immune infiltration translates into effective immune function and safeguards the tissue from tumourigenesis. Across human cancers, tumours arising in high-metabolism tissues - like kidney, brain, and eye - tend to show high T cell infiltration but poor prognosis, suggesting pre-existing metabolic environments prior to malignant transformation may undermine immune function. This pattern is mirrored across species: large mammals with lower mass-specific metabolic rates (e.g., elephants, whales) accumulate fewer metabolic byproducts and show lower cancer incidence (Peto's paradox), while long-lived small mammals like bats and naked mole-rats resist tumourigenesis via suppressed glycolysis or altered hypoxia responses leading to lower metabolic rates and/or byproduct accumulation. Through integrative synthesis spanning human single-cell expression data and cross-species comparisons, we outline a framework of 'immunometabolic gatekeeping', where tissues with high metabolic rate and poor waste clearance foster immune-exhausting niches even before transformation. This unifying framework reconciles multiple paradoxes in cancer biology: Peto's paradox, T cell infiltration non-prognosticity, tissue tropisms, and sex-based inequalities, and suggests new principles for identifying high-risk patients and metabolic-immune combination strategies for prevention and treatment. By shifting focus from tumour-intrinsic mutations to host-tissue metabolism, this work offers a novel, integrative lens on cancer vulnerability and immune failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/debe635e8c376165545fae9b98889da043230849" target='_blank'>
              Immunometabolic Gatekeeping: Reconciling Peto's & the T-cell Infiltration Prognostic Paradox
              </a>
            </td>
          <td>
            Naomi Iris van den Berg, Matouš Elphick, Kevin Mulder, Omar Bouricha, O. Sadeghi-Alavijeh, Xiao Fu, S. Turajlic
          </td>
          <td>2025-11-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="


 Introduction : Chronic graft-versus-host disease (GVHD) is a leading cause of late non-relapse mortality following allogeneic hematopoietic cell transplantation, yet spatial biomarkers capable of predicting outcomes remain lacking. Minor salivary glands (MSGs) are anatomically accessible organs that can be easily accessed by fast and minor invasive biopsies and reflect systemic immune perturbations, making them ideal for spatial profiling of immune diseases. We applied STARComm, a spatial multiomics framework that identifies Multicellular Interaction Modules (MCIMs), which represent tissue microenvironments where multiple cell types engage in coordinated signaling. MCIMs capture clinically relevant hubs of immune communication and tissue remodeling, enabling spatial resolution of pathogenic processes in GVHD.
 Methods : Thirty-six MSG tissues were collected from patients enrolled between 2018 and 2022 at the Hospital das Clínicas, University of São Paulo. All patients had undergone allogeneic bone marrow transplantation and subsequently received MSG biopsies as part of clinical follow-up for chronic GVHD assessment. The cohort included healthy controls (n=8), GVHD survivors (n=18), and patients who died from GVHD-related complications (n=10), with detailed clinical metadata. Spatial multiomic profiling was performed on the same histological section using a dual-modality approach: spatial transcriptomics with Xenium (10x Genomics) using a targeted 280-gene panel, and spatial proteomics with Phenocycler Fusion (Akoya Biosciences) using a 36-antibody multiplex panel. MCIMs were computed via STARComm by integrating local communication density maps across spatial grids. Cell types were annotated with TACIT, and differences in MCIM composition were compared across groups. Prognostic associations were modeled using Cox regression. To prioritize therapeutic targets, we applied SpatialDrug2Cell, a single-cell-based pipeline for inferring druggable dependencies, to spatially segmented cell populations, adapting it for high-resolution in situ data.
 Results:A total of 842,154 spatially resolved cells were analyzed across all samples. Receptor–ligand interactions were inferred from 17 curated gene pairs selected for biological relevance to GVHD and compatibility with both transcriptomic and proteomic detection, based on expression specificity and spatial proximity. Nine MCIMs were identified. In healthy MSGs, MCIMs reflected structured stromal, epithelial, and vascular signaling. GVHD samples exhibited marked reorganization, with MCIMs-0, -2, and -6 significantly enriched (p<0.001), particularly within TLS-like periductal niches. MCIM-0, dominant in fatal cases, exhibited dense B and T cell, T and T cell, and stromal interactions, with high spatial density of CXCL12-CXCR4 and CCL5-SDC4 signaling, primarily transcribed by fibroblasts and pericytes. In contrast, MCIM-3, enriched in fibroblast–endothelial interactions and B cell regulatory networks, was more abundant in survivors. In multivariate analysis, MCIMs-0 and -6 were independently associated with increased mortality (HR=2.08 and 2.83), while MCIM-3 showed a protective trend (HR=0.51). SpatialDrug2Cell, applied to MCIM-localized cell populations and spatially restricted therapeutic targets, highlighting candidate agents such as rituximab, atezolizumab, and plerixafor.Conclusions:Our analysis establishes spatially resolved MCIMs as prognostic biomarkers in chronic GVHD. The emergence of high-risk communication modules in MSGs, particularly in patients with fatal outcomes, reveals tissue-localized immune architectures associated with disease progression. By integrating STARComm with adapted drug inference tools like SpatialDrug2Cell, we identify clinically relevant, spatially anchored therapeutic targets in routinely sampled tissues. This framework may enable mechanism-based stratification and therapeutic guidance in GVHD and other immune-mediated diseases
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1daff8734fed12743fa31dcf5eea76ecab05128e" target='_blank'>
              Spatial communication modules identify prognostic cell-cell signaling hubs in chronic graft-versus-host disease
              </a>
            </td>
          <td>
            Bruno Fernandes Matuck, Khoa L. A. Huynh, Deiziane de Souza, Giancarlo Fatobene, Luiz Valente Soares Junior, L. F. Ferraz da Silva, Marisa Dolhnikoff, V. Rocha, Jinze Liu, Kevin Byrd
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer is a highly heterogeneous disease, with many cancers containing multiple distinct subclones. While subclones are often seen as competitors (survival of the fittest), intratumor heterogeneity can also offer direct benefits to the tumor through cooperation between different clones. This has important clinical implications, as interdependent populations may present therapeutic vulnerabilities. Here, we review existing evidence for clonal cooperativity to address key questions and outline future developments based on six overarching principles: (a) secreted factors are important mediators of clonal cooperation; (b) (very) small subclones can significantly affect tumor behavior; (c) both genetic and nongenetic heterogeneity are substrates for cooperation; (d) nonmalignant cells from the tumor microenvironment can act as cooperating partners; (e) clonal cooperation occurs throughout different stages of cancer, from premalignancy to metastasis; and (f) clonal cooperation can promote therapy resistance by protecting otherwise sensitive populations. Together, these principles suggest clonal cooperation as an important mechanism in cancer. Lastly, we discuss how novel technological developments could address remaining gaps to open up new therapeutic strategies that exploit clonal cooperativity by targeting the tumor's weakest link.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb39ce21ba07d76bffe91c932b77c8c8a7e6988d" target='_blank'>
              Strength through diversity: how cancers thrive when clones cooperate.
              </a>
            </td>
          <td>
            Marije C. Kuiken, Maartje Witsen, E. Voest, K. Dijkstra
          </td>
          <td>2025-11-09</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Endometriosis (EM) is a chronic gynecological disorder that affects 5% to 10% of women of childbearing age, often causing pelvic pain and infertility. Fibrosis is a hallmark of EM progression, yet its underlying molecular drivers remain poorly understood. Emerging progress in single-cell and spatial transcriptomic technologies offer new opportunities to unravel the cellular heterogeneity and intercellular interactions driving fibrotic and immune remodeling in EM lesions. Methods We performed an integrative multi-omics analysis combining single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to dissect fibroblast heterogeneity and cell–cell communication networks in EM. ScRNA-seq data from 15 EM patients (GSE213216) were processed to identify transcriptionally distinct fibroblast subpopulations. Functional enrichment (GO, GSEA), stemness estimation (CytoTRACE), and trajectory inference were applied to reveal lineage plasticity. CellChat was used to infer intercellular signaling networks, and spatial transcriptomic data from two ectopic lesions (GSM6690475, GSM6690476) were analyzed to validate the spatial distribution of key ligand–receptor interactions. Results We identified 35 cell clusters across EM lesions, with Fibroblast and T/NK cells as dominant populations. Fibroblast were divided into five subtypes, which were associated with extracellular matrix remodeling, immune interactions, and metabolic regulation. Notably, the C2 CXCR4 + Fibroblast subpopulation exhibited high proliferative capacity and stemness characteristics, and mediated signaling pathways involved in immune and fibrotic responses through FN1. Spatial transcriptomic analysis confirmed the local enrichment of these Fibroblast in ectopic lesions, where they were associated with regions of active signaling. Conclusion This study revealed the transcriptional and spatial heterogeneity of Fibroblast in EM syndrome, and identified the C2 CXCR4 + Fibroblast subpopulation as a may represent key driver of fibrosis and immune regulation. Our integrated omics approach provided new mechanistic insights into the pathogenesis of EM and pointed out new targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ccad044a665115ac5faba149f5925920f3e3b05" target='_blank'>
              Fibroblast heterogeneity and FN1-mediated signaling in endometriosis revealed by single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Wenwen Shao, Hongmei Ju, Zhikai Xiahou, Sheng Fang, Rugen Yan, Chunyan Li, Yuan Xu, Pingping Cai
          </td>
          <td>2025-10-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Pineoblastoma (PB) is a clinically aggressive embryonal central nervous system (CNS) tumor composed of distinct molecular subgroups. The cellular and biological basis of PB remains poorly defined, limiting the development of more effective therapeutic strategies. To identify the cellular composition, origins, and drivers of PB, we derived single-cell transcriptomes from patient-derived pineal tumors (n=38) and integrated malignant cells with a novel transcriptional atlas of mouse pineal gland development (E13-P21). Application of a series of computational strategies pinpointed transient, cycling pinealocyte progenitors as the likely cellular origin of PB, irrespective of molecular subgroup. To functionally validate these findings, we generated lineage-specific genetically engineered mouse (GEM) models representing distinct PB subgroups with high temporal, anatomic, and phenotypic fidelity. Transcriptomic analysis further substantiated the accuracy of novel GEM models which converged on cell-cycle dysregulation, highlighting a core pathogenic mechanism across PB subgroups. Relative to the developing pineal gland, photoreceptor transcription factors were aberrantly activated in PB GEM models, suggesting photoreceptor involvement in tumor initiation and/or maintenance. Multi-omic analysis across a large range of diverse CNS cancers revealed an oncogenic photoreceptor program specific to PB, retinoblastoma (RETB), and Group 3 medulloblastoma (G3-MB). This shared transcriptional program was active within respective cellular origins, establishing a unified mechanistic basis across these anatomically distinct CNS tumors. CRISPR-based functional studies confirmed selective dependencies of photoreceptor transcription factors (NRL, CRX, OTX2) in PB, RETB and G3-MB, demonstrating the essentiality of this photoreceptor program. These discoveries not only resolve longstanding uncertainties regarding PB pathogenesis but also highlight conserved molecular signatures and therapeutic vulnerabilities spanning different pediatric CNS malignancies. Our results underscore the potential of targeting developmentally regulated programs and master transcription factors as novel therapeutic strategies for PB, RETB, and G3-MB, offering critical insights and preclinical models for future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cee7231cd5056fbbc7fcab0e2a1e377781c0c82" target='_blank'>
              EPCO-05. AN ONCOGENIC PHOTORECEPTOR PROGRAM UNIFIES DISTINCT CENTRAL NERVOUS SYSTEM TUMORS
              </a>
            </td>
          <td>
            B. Gudenas, S. Ahmad, Bernhard Englinger, Anthony P Y Liu, Miao-yun Zhao, Leena Paul, Jennifer L. Hadley, Hong Lin, Yiran Li, Melissa Batts, Priya Mittal, Stephanie C Wu, Sara Lewis, Katie Han, T. Soliman, Laure Janke, David Meredith, E. Pfaff, J. Gojo, Jennifer Cotter, P. Klimo, F. Boop, Amar Gajjar, G. Robinson, G. Rosén, S. Alexandrescu, David T. W. Jones, Brent A. Orr, F. Swartling, M. Filbin, P. Northcott
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Abstract Summary Whereas transcriptomic and spatial profiling offer static snapshots of tissue structure, mechanistic models use biological rules to predict how tissues evolve. We present the BioInformatics WalkThrough (BIWT) software to directly initialize spatial agent-based models from single-cell and spatial molecular data. We demonstrate how initialization strategies affect tumor–immune dynamics and spatial clustering, positioning BIWT as a software suite to generate data-driven virtual cells representing both experimental and clinical contexts. Availability and implementation The BIWT software is available at https://github.com/PhysiCell-Tools/PhysiCell-Studio. The sample dataset for running the BIWT is available at https://zenodo.org/records/16365625. The code and instructions for reproducing the use case example is available at https://github.com/drbergman/BIWT-Paper.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de85f70990a72aa732cef216c202f1c4a4e48e9" target='_blank'>
              BIWT: a bioinformatics walkthrough for embedding spatial multiomics in agent-based models for virtual cells
              </a>
            </td>
          <td>
            Daniel R. Bergman, Jeanette Johnson, Marwa Naji, Max Booth, Heber Lima da Rocha, Atul Deshpande, Dimitri N. Sidiropoulos, T. Lopez-Vidal, Randy Heiland, Luciane T. Kagohara, Robert A Anders, Lei Zheng, Elizabeth M. Jaffee, Genevieve Stein-O'Brien, Paul Macklin, E. Fertig
          </td>
          <td>2025-10-16</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="


 Introduction:Plasma cell leukemia (PCL) is a rare yet highly aggressive form of plasma cell dyscrasia, defined by high tumor burden, frequent extramedullary dissemination, and rapid clinical progression. Compared with multiple myeloma (MM), PCL exhibits greater genomic instability, immune escape, and a more hostile bone marrow microenvironment, resulting in poor therapeutic responses and short survival. While prior bulk sequencing and cytogenetic studies have revealed select features of PCL biology, a high-resolution, single-cell dissection of its malignant and immune landscape remains lacking. In particular, how PCL reshapes its tumor microenvironment (TME) to support immune evasion and systemic dissemination has not been fully elucidated.
 Aim: This study aims to characterize the transcriptional, genomic, and immunologic features of PCL at single-cell resolution. We sought to identify PCL-specific oncogenic drivers, immune evasion mechanisms, and intercellular communication circuits that differentiate PCL from MM and healthy bone marrow, with a particular focus on the ERBB signaling axis and T/NK cell dysfunction.
 Methods:Bone marrow samples were collected from three newly diagnosed primary PCL patients and one MM patient from our center, These specimens were subjected to 10x Genomics 3' single-cell sequencing. Additionally, three normal bone marrow and three MM bone marrow samples retrieved from the Gene Expression Omnibus (GEO) database were incorporated for integrative analysis. Comparative analyses were conducted to assess clonal architecture, transcriptional programs, and immune landscape. Key findings were validated using qPCR and publicly available clinical and transcriptomic datasets.
 Results:After quality control filtering, we obtained single-cell transcriptome data from 49,059 high-quality mononuclear cells. Based on canonical marker gene expression, we identified eight major cell populations: plasma cells, myeloid cells, T cells, NK cells, NKT cells , B cells , dendritic cells, and megakaryocytes. This included 11,485 PCs, 3,466 B cells, and 33,603 other immune cells.
 PCL plasma cells demonstrated significantly higher CNV burden than MM, with recurrent gains on chr3q, chr9q, chr15q, and chr18q. Amplification of oncogenes including BCL2, MCL1, PIK3CA, KRAS, and BRAF suggested enhanced apoptotic resistance, proliferative signaling, and immune evasion.
 Transcriptomic profiling revealed PCL-specific upregulation of the ERBB signaling axis, including ERBB4, EPHA6, and downstream effectors KRAS, SOS1/2, and MAPK1/8. ERBB pathway enrichment was confirmed by GSEA and pseudotime analysis revealed persistent upregulation along the malignant differentiation trajectory. High expression of ERBB pathway genes correlated with significantly worse overall survival in the CoMMpass cohort. qRT-PCR validation in patient samples confirmed significant overexpression of ERBB-related genes in PCL.
 Analysis of over 23,000 T/NK lineage cells identified 21 subtypes. In PCL, cytotoxic subsets such as CD8⁺ naive, CD56dim NK, and NKT cells were notably reduced or exhibited elevated exhaustion scores. NKT cells showed elevated IFN-response genes but downregulation of cytotoxic effectors, indicating a paradoxical “hyperactivated-yet-dysfunctional” state.
 CellChat analysis revealed extensive reprogramming of ligand-receptor interactions in PCL. Compared to ND and MM, PCL plasma cells engaged in enriched immunosuppressive and adhesive signaling via MIF, TGF-β, CD86, NOTCH, and APRIL axes. Enhanced communication with CD8⁺ T and NKT cells may promote chronic stimulation and immune exhaustion, supporting tumor immune escape and dissemination.
 Conclusion: This study presents the first high-resolution single-cell atlas of primary PCL, revealing that enhanced ERBB signaling, elevated CNV burden, and profound immune dysfunction collectively drive its aggressive clinical behavior. Malignant plasma cells in PCL not only acquire stem-like features and oncogenic pathway activation but also actively remodel the immune microenvironment via suppressive signaling, leading to T/NK cell exhaustion and impaired surveillance. These findings uncover actionable targets such as ERBB4/PI3K/MAPK and immune exhaustion pathways, offering a molecular framework for future precision therapy in PCL. Further functional validation and clinical translation of these discoveries are warranted.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2c5eed7a02f279167bbebc35b4990c64fcc8584" target='_blank'>
              Single-cell transcriptomics reveals erbb-driven clonal evolution and immune evasion in plasma cell leukemia
              </a>
            </td>
          <td>
            Tingting Lei, Luyao Shi, Xinli Han, Chi Li, Zongyang Li, Xiaotong Zhang, Kairan Li, Saisai Sun, Lu Yang, Chong Wang, Zhilei Bian, Shujuan Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background/Objectives: High-grade gliomas (HGGs), including glioblastomas, are among the most aggressive brain tumors due to their high intratumoral heterogeneity and extensive infiltration. Glioma stem-like cells (GSCs) frequently invade along white matter tracts such as the corpus callosum, but the molecular programs driving this region-specific invasion remain poorly defined. The aim of this study was to identify transcriptional signatures associated with GSC infiltration into the corpus callosum. Methods: We established an orthotopic xenograft model by implanting fluorescently labeled human GSCs into nude mouse brains. Tumor growth and invasion patterns were assessed using tissue clearing, light-sheet fluorescence microscopy, and histological analyses. To characterize region-specific molecular profiles, we performed microfluidic-based single-cell RNA expression analysis of 48 invasion- and stemness-related genes in cells isolated from the tumor bulk (TB) and corpus callosum (CC). Results: By six weeks post-implantation, GSCs displayed marked tropism for the corpus callosum, with distinct infiltration patterns captured by three-dimensional imaging. Single-cell gene expression profiling revealed significant differences in 7 of the 48 genes (14.6%) between TB- and CC-derived GSCs. These genes—NES, CCND1, GUSB, NOTCH1, E2F1, EGFR, and TGFB1—collectively defined a “corpus callosum invasion signature” (CC-Iv). CC-derived cells showed a unimodal, high-expression profile of CC-Iv genes, whereas TB cells exhibited bimodal distributions, suggesting heterogeneous transcriptional states. Importantly, higher CC-Iv expression correlated with worse survival in patients with low-grade gliomas. Conclusions: This multimodal approach identified a corpus callosum-specific invasion signature in glioma stem-like cells, revealing how local microenvironmental cues shape transcriptional reprogramming during infiltration. These findings provide new insights into the spatial heterogeneity of gliomas and highlight potential molecular targets for therapies designed to limit tumor spread through white matter tracts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561590a9b7ca738409c5933b00e64b421edbb629" target='_blank'>
              Innovative In Vivo Imaging and Single Cell Expression from Tumor Bulk and Corpus Callosum Reveal Glioma Stem Cells with Unique Regulatory Programs
              </a>
            </td>
          <td>
            Natália dos Santos, Aline Aquino, Friedrich Preusser, Fabio Rojas Rusak, E. H. Jandrey, Miyuki Uno, T. K. Furuya, C. Lancellotti, M. Maldaun, Roger Chammas, Stephan Preibisch, A. Camargo, C. Masotti, Erico Tosoni Costa
          </td>
          <td>2025-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05709be045bd3ac955b0ab18505a25294de0610f" target='_blank'>
              Optimized protocol for processing murine tumor-bearing lung tissue for flow cytometry and single cell RNA-sequencing
              </a>
            </td>
          <td>
            Sicong Wang, Anastasiia Ivanova, Helen P. Makarenkova, Katja A. Lamia
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a185da472d5e884ad916a2af2cce445855a3e882" target='_blank'>
              Single-cell lineage trajectory defines CDK inhibitor-sensitive cells-of-origin in esophageal squamous cell cancer
              </a>
            </td>
          <td>
            Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/999c9f7dc52086f050a2aa67ed25e42b93fae763" target='_blank'>
              Single-cell RNA sequencing in osteosarcoma: applications in diagnosis, prognosis, and treatment
              </a>
            </td>
          <td>
            Christèle Asmar, Guy Awad, Marc Boutros, Simon Daccache, A. Chebly, C. Alix‐Panabières, H. Kourie
          </td>
          <td>2025-11-13</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) serve as a promising biomarker for tracking cancer metastasis, progression, and recurrence. Liquid biopsy techniques centered on CTC detection have demonstrated considerable potential due to their non-invasive nature and ability to provide real-time monitoring of tumor dynamics. However, conventional bulk CTC analyses fail to capture the intrinsic heterogeneity among CTC populations, obscuring crucial insights into tumor biology. Single-cell RNA sequencing (scRNA-seq) enables high-resolution characterization of CTC heterogeneity, offering new opportunities for precision oncology and mechanistic studies of tumor progression. Despite these advantages, the existing methodologies for single-CTC sequencing tend to suffer from inefficiencies, including low recovery rates, labor-intensive workflows, and contamination risks associated with multiple manual handling steps. To address these limitations, we present an integrated microfluidic protocol that consolidates CTC enrichment, purification, and single-cell sequencing into a unified workflow. The method employs dynamically controlled magnetic capture within a herringbone-structured chip, where vortex mixing and cumulative immunomagnetic bead binding enable robust, high-throughput CTC isolation with minimal cell damage. Subsequent purification using a leukocyte antibody-coated microfluidic chip effectively removes the non-target cells, further enhancing CTC purity through negative selection. Finally, a high-precision single-cell sequencing chip, designed based on differential flow resistance principles, facilitates efficient single-cell capture and pairing with uniquely barcoded microbeads. This novel platform overcomes the limitations of Poisson distribution-based methods, improving CTC utilization while minimizing microbead consumption and sequencing costs. Our integrated protocol significantly enhances CTC capture efficiency, purity, and single-cell sequencing throughput, making it well-suited for clinical applications and large-scale cancer research. By enabling a more precise and scalable analysis of CTC heterogeneity, this method has the potential to refine early cancer diagnosis, treatment monitoring, and mechanistic studies of metastasis, ultimately advancing the field of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9babafe8a10523678f47d2a96e859b16fafc3f98" target='_blank'>
              Microfluidics-based High-throughput Circulating Tumor Cell Sorting and Single-cell Sequencing Technology.
              </a>
            </td>
          <td>
            Zheyu Liu, Ye Tian, Zian Zhang, Lingling Wu, Liang Dong, Wei Xue
          </td>
          <td>2025-11-14</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In this study, we analyzed publicly available scRNA-seq data from primary tumor tissues of 10 Retinoblastoma (RB) patients to explore tumor microenvironment (TME) heterogeneity. Our findings revealed distinct subpopulations of cone precursor (CP) cells, with higher proportions in invasive RB. Differential gene expression and pathway analysis highlighted functional diversity among CP subpopulations, with CP4 showing elevated TGF-β signaling in invasive RB. Cell–cell interaction analysis further identified rewiring of communication networks, with increased fibroblast–CP interactions in invasive tumors. Bulk RNA-seq identified two molecular subtypes, with subtype 1 showing an immunosuppressive TME. Finally, DOK7 was identified as a key gene associated with invasion, with functional assays confirming its role in promoting tumor progression. These results provide valuable insights into RB progression and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0641facdea378a09ac72da42fc0862a4720147" target='_blank'>
              Comprehensive analysis of single-cell and bulk RNA sequencing uncover tumor microenvironment diversity in invasive Retinoblastoma
              </a>
            </td>
          <td>
            Wei Wan, Xiaoqian Chen, Hongli Liu, Ling Yang, Pingping Huang
          </td>
          <td>2025-11-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Bladder cancer is a significant malignancy, for which prognostic prediction and understanding of the tumor immune microenvironment are crucial. B cells play a key regulatory role in this environment, making their study essential for advancing bladder cancer research. Methods In this study, a multi-omics analysis strategy combining single-cell RNA sequencing (scRNA-seq) and bulk RNA-seq was used to establish single-cell transcriptome profiles of tumor tissues from bladder cancer patients, focusing on B-cell populations and their interactions with other cell types in the tumor microenvironment. Large public databases were used to screen for key prognostic genes associated with bladder cancer B cells, and their biomarker expression was verified by in vitro experiments. Results Based on tumor samples from eight patients with bladder cancer and four normal samples, we selected 84, 967 cells for single-cell sequencing analysis. From these, we identified 10, 967 B cells and identified 508 key genes associated with B cells in bladder cancer from five different B cell subtypes. By integrating a large amount of RNA sequencing data, we identified VCL, FLNA, TAGLN, ACTA2, COL6A2, and CALD1 as potential biomarkers for B-cell-associated bladder cancer, and experimentally verified that these markers were significantly lower in bladder cancer patients than in the normal group, and were effective in predicting the survival rate of the patients and the status of the tumor immune microenvironment. Conclusions Using a combination of transcriptomic and experimental validation at single-cell and batch levels, this study provides insights into the key gene signatures of B cells from patients with bladder cancer and their roles in regulating the tumor immune microenvironment, providing new biomarkers and potential therapeutic targets for predicting patient’ prognosis and immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c8bfbe519c06579f4740a61199654ca1af067bc" target='_blank'>
              Comprehensive analysis of single-cell and bulk transcriptomes reveals key B-cell genes and immune microenvironment regulation in bladder cancer
              </a>
            </td>
          <td>
            Lijun Wang, Juan Yang, Zhangxiao Xu, Bo Tao, Yunpeng He, Yuan Zhao, Jian Wu, Yira Ma, Zitao Zhong, Lin Ye
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal malignancy marked by extensive genomic complexity, early dissemination, and resistance to therapy. APOBEC3A (A3A), a cytidine deaminase, has emerged as a candidate driver of chromosomal instability (CIN) and intertumoral heterogeneity (ITH), yet its role in PDAC evolution remains incompletely defined. To address this gap, we have assembled a comprehensive dataset that integrates genetically engineered mouse models, human PDAC tissues, and established cancer cell lines. The engineered mouse models were specifically designed to represent distinct biological scenarios of A3A activity, providing a systematic framework to dissect its impact on tumor evolution. Complementary data from human tissues and cell lines will validate these findings in clinically relevant contexts, ensuring direct translational significance. In addition, we have optimized and tested a laser-capture microdissection (LCM) approach that enables spatially resolved sampling of multiple regions within human tumors, thereby facilitating high-resolution analyses of clonal dynamics and their relationship to A3A activity. With these resources in place, we will apply bulk whole-genome sequencing (WGS) and advanced computational pipelines to identify A3A-associated copy number signatures. By integrating cross-species models with high-resolution human sampling, we aim to reveal how A3A mediates the transition from pre-neoplastic lesions to advanced PDAC and to uncover potential vulnerabilities that could be therapeutically targeted. This work will establish a systematic and clinically validated framework for evaluating A3A as a driver of genomic instability in PDAC, ultimately guiding strategies to mitigate heterogeneity and improve patient outcomes.



 Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, Vince Bernard. Pagan, Alexander Semaan, Anirban Maitra, Fredrik Thege, Mia Petljak, Sonja M. Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe98cd4acb4288fdaef0ded37a1cd6fefa76ca4" target='_blank'>
              Abstract A003: APOBEC3A drives chromosomal instability and tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Ashwath Seetharaman, Norbert Kreisz, Abhijnan Chakraborty, V. B. Pagan, Alexander Semaan, A. Maitra, F. Thege, Mia Petljak, Sonja Wörmann, P. van Loo
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="

 The most aggressive type of skin cancer, melanoma, has a high prevalence of metastases and a poor prognosis. Despite advancements in treatment, drug resistance and tumor microenvironment heterogeneity, especially involving neuro-immune interactions, continue to exist. The goal of this study is to uncover the cellular heterogeneity of melanoma in order to pinpoint molecular targets and tumor-promoting subtypes.



 Melanoma single-cell RNA sequencing data came from GEO. Twelve cell types were discovered after Harmony batch effect adjustment and Seurat quality control; melanoma cells were subtyped. Functional expression of differential genes was examined using gene ontology and gene set enrichment. Cytotrace measured subtype differentiation potency. PySCENIC revealed transcription factor regulatory networks, and CellChat predicted intercellular communication between malignant cell subtypes and other cell types. Functional experiments with A375 and MEWo cell lines—lentiviral shRNA knockdown, CCK-8 proliferation, wound healing, transwell migration, and flow cytometry apoptotic assays—confirmed the analysis.




 Single-cell RNA sequencing was utilized to analyze melanoma cell subtypes and their interactions in the tumor microenvironment. C2
 IGFBP3
 +, which had the lowest CytoTRACE2 score and was enriched in late tumor stages, affected melanoma development. This subtype expressed prominent immunomodulatory pathways. The C2 subtype mediate signaling to fibroblasts/pericytes via the PROS pathway and myeloid/plasmacytoid dendritic cells via the MHC-II pathway. The C2 subtype was strongly linked with FOSL1 expression, which may affect gene transcription and illness progression. FOSL1 knockdown significantly increased apoptosis and decreased melanoma cell motility and proliferation
 in vitro
 .





 We identified immunoregulatory C2
 IGFBP3
 + melanoma cell subtypes in our investigation, and FOSL1 was a crucial transcription factor that aided in cell migration, proliferation, and survival. Because the C2 subtype involves the MHC-II and PROS signaling pathways, which have been shown to have roles in neuroimmunology, neuroinflammation, and pain regulation, it may serve as a hub for neuro-immune interactions in the tumor microenvironment. Precision treatments for melanoma may be advanced by focusing on the FOSL1 or C2 subtype pathways, which may assist in overcoming immunotherapy resistance.

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d34e116063cb6903b41d6e715ba8448ec0e1c090" target='_blank'>
              Single-cell transcriptomics identifies FOSL1-regulated IGFBP3+ melanoma subtype as a neuro-immunoregulatory signaling hub facilitating tumor progression
              </a>
            </td>
          <td>
            Wenjia Ge, Ziwei Zhang, Wenjie Chen, Zhijie Zhao, Huabao Cai, Yantao Ding, Jin Xu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The c-MYC transcription factor is aberrantly expressed in most human cancers to enhance expression of proliferative gene programs. Owing to its pseudo-ordered structure and reliance on extensive and dynamic protein-protein interactions in distinct transcriptional regulatory complexes, defining context-specific MYC interactors has remained challenging. Therefore, mapping MYC-centered complex topologies in disease relevant models could identify components critical for its function which may serve as therapeutic targets in MYC-driven cancers. Here, we present a matched pair of photoproximity probes coupled with quantitative proteomics which enable context-dependent mapping of protein complex topology inside cells. We applied this spatially resolved, intracellular photoproximity (siPROX) profiling workflow to map MYC interactomes across temporal, spatial and disease-relevant contexts. Basal and inhibitor-treated profiles confirmed interactions with a wide range of known chromatin-associated transcriptional regulatory factors that define the extended MYC transcriptional bubble in live cells. Time-resolved mapping of inhibitor treated cells identified dynamic remodeling of numerous transcriptional regulatory factors and identified several BAF complex members (e.g., PBRM1 and SMARCC1)1 that persist in the presence of bromodomain inhibition. Furthermore, spatial MYC topology maps in small cell lung cancer cells confirmed the presence of BAF complex members under conditions where MYC induced target gene expression, altered cell morphology and enhanced proliferation. Lastly, loss of BAF function via inhibition of SMARCA2/4 ATPase activity resulted in rapid loss of chromatin-bound and nuclear MYC levels, downregulation of MYC-dependent transcripts and MYC-specific cell growth in several cancer cell models. Together, these data highlight the potential for siPROX to identify spatially resolved, dynamic TF interactors and highlight MYC-proximal BAF interactions as a targetable liability to regulate MYC-dependent transcription and proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f78a06fd992b15a6520a4d4a8e38c9a04a5e3b92" target='_blank'>
              Spatially-resolved Photoproximity Profiling of MYC Identifies a MYC-BAF Liability in Cancer Cells
              </a>
            </td>
          <td>
            Anthony J. Carlos, Shuyuan Huang, Dongbo Yang, Colin Swenson, Pratyasha Chakraborty, Shaopeng Yu, Benjamin D. Stein, Raymond E. Moellering
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastasis is the primary cause of cancer-related mortality worldwide. This narrative review integrates recent advances in the molecular circuits orchestrating metastatic progression, encompassing epithelial–mesenchymal transition (EMT), organotropism, extracellular matrix remodeling, angiogenesis, hypoxia-inducible signaling, tumor-cell migration modes, and tumor–immune interactions through expert-guided literature selection. We examined therapeutic innovations that disrupt these pathways, including EMT modulators, matrix metalloproteinase inhibitors, VEGF/VEGFR-targeted regimens, hypoxia-activated prodrugs, and next-generation immunotherapies such as immune checkpoint blockade and chimeric antigen receptor T cells. Additionally, we discuss established nanotechnology-based delivery systems, advancing multi-omics integration, evolving single-cell analyses, and emerging CRISPR-Cas9 gene-editing applications as tools for improving metastasis detection, monitoring, and treatment. Despite this progress, translational obstacles persist, particularly regarding intratumoral heterogeneity, adaptive resistance, and limited preclinical model fidelity. Addressing these challenges requires biomarker-guided, multi-target therapeutic combinations, interdisciplinary collaboration, and globally inclusive clinical trials. This evidence underscores the importance of integrated strategies that simultaneously target intrinsic tumor plasticity and microenvironmental support to transform metastatic cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d7d3de1a04e69b49fb547b3a441611b4041aca5" target='_blank'>
              Deciphering molecular pathways driving cancer invasion and metastasis: advances and therapeutic prospects
              </a>
            </td>
          <td>
            Aziza Alshahrani, Arwa Alsubait, Zahrah Asiri, Sahar Alghamdi, Sarah Bin Saqyah, Tariq Alqahtani, Rawan Fitaihi, Njoud Altuwaijri, Yahya F. Jamous
          </td>
          <td>2025-12-02</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a crucial role in colorectal cancer by sustaining intratumoral heterogeneity, therapeutic resistance, and metastatic potential. CD133 (PROM1) is among the most frequently used surface markers for CSC identification, whereas TRIM28, a versatile epigenetic regulator, has been implicated in controlling CD133 expression and stem-like features. In this study, we performed a detailed molecular and functional analysis of Caco2 colorectal cancer cell clones with individual knockouts of CD133 or TRIM28. Elimination of CD133 neither altered global gene expression, as confirmed by transcriptome profiling, nor affected key cellular properties. In contrast, loss of TRIM28 led to a marked reduction in CD133 protein abundance and induced extensive molecular and phenotypic remodeling. TRIM28 knockout was associated with broad transcriptomic changes involving more than 500 differentially expressed genes, decreased proliferative activity monitored by time-lapse imaging, and reduced sensitivity to paclitaxel, cisplatin, and curcumin. Furthermore, immune evasion molecules CD24 and CD47 (“don’t eat me” signals) were strongly upregulated in TRIM28-deficient cells, consistently confirmed by both RNA-Seq and flow cytometry analyses. At the same time, imaging flow cytometry and mitochondrial activity assays indicated that these effects were not due to major shifts in mitotic index or bioenergetic status. Altogether, our results demonstrate that TRIM28, rather than CD133, functions as a central regulator of CSC-associated phenotypes in colorectal cancer. These findings highlight the importance of epigenetic context in CSC biology and may inform the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3d3eb05c9db6b57d94b54e8bdad26455b3ea6" target='_blank'>
              Contrasting Impacts of Targeted Disruption of the Cancer Stem Cell Marker CD133 and Its Epigenetic Regulator TRIM28 in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            I. Kholodenko, A. Saidova, D. Potashnikova, V. A. Arzumanian, D. Romashin, A. Tvorogova, E. Poverennaya, K. Yarygin, Yan S. Kim
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Extensive studies have demonstrated the relationship between metabolic reprogramming and the tumor microenvironment. Here, we characterized the head and neck squamous cell carcinoma (HNSCC) evolutionary landscape using spatial metabolomics/transcriptomics, single-cell transcriptomics, and bulk multi-omics. Metabolic heterogeneity during HNSCC malignant transformation was identified, with significant enrichment in the purine metabolism. Integrating single-cell and bulk data, we developed a robust ligand-receptor-based signature (LRS) linked to NT5E, a key upstream regulator of purine metabolism, which served as an independent prognostic indicator. The low LRS subtype was associated with a high proportion of immune cell infiltration and improved response to immunotherapy. Notably, in vitro and in vivo experiments demonstrated that AMIGO2, a core molecule within the LRS, regulates tumor-associated purine metabolism, and that its downregulation suppresses tumor cell invasion and migration, inhibits myofibroblast differentiation, and promotes immune effector cell infiltration. Moreover, combining AMIGO2 targeting with anti-PD-1 therapy yielded superior efficacy. Consistent validation was also obtained in a clinical cohort of HNSCC and premalignancy patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca668a38f0d67d558807d21d034edd4de991a2" target='_blank'>
              Metabolomic and transcriptomic profiling of HNSCC identifies AMIGO2 as a therapeutic target modulating tumor microenvironment
              </a>
            </td>
          <td>
            Gan Liu, Xinfeng Yao, Yuchen Hou, Wentao Deng, Liyu Zhang, Shutong Li, Panpan Huang, Wenjiao Chang, Haiyan Huang, Lan Shi, Qianqian Zhou, Yanqing Wang, Jia Chen, Wenchao Gu, Xiaoling Ma, Yu Zhou, Honghong Liu, Shanshan Zhang, Zongcheng Yang
          </td>
          <td>2025-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Renal cell carcinoma (RCC) is responsive to immunotherapy despite having a low tumor mutational burden. Interestingly, the degree of CD8+ T cell infiltration in RCC negatively correlates with outcomes, suggesting that a deeper understanding of the tumor microenvironment may provide critical insight to help improve clinical outcomes.



 We conducted a multi-omics approach including flow cytometry, multiplex immunohistochemistry, and single-cell RNA-seq to compare immune cell composition, gene expression, spatial distribution, and cell-cell interactions among a cohort of primary breast cancer, head and neck squamous cell carcinoma, non-small cell lung cancer, and RCC specimens.



 Flow cytometry analysis of tumor-infiltrating lymphocytes (TILs) revealed increased exhausted CD8 + (Tex) and CD4+CD8+ double positive (DP) T cells and decreased CD4+ T cells including regulatory T (Treg) cells in RCC samples compared to other tumor types. scRNA-seq analysis of TILs confirmed these results and showed Tex cells as a major population of CD8+ T cells. Using biomarkers for tumor-reactive cells including 4-1BB, CD39, and CD103 along with a gene signature to identify neo-antigen reactive T cells, we found that the Tex population contained putative tumor-reactive T cells and were not merely bystander cells.



 Exhausted CD8+ and DP T cells in RCC contain tumor-reactive T cells and may represent a novel cell population capable of targeting RCC.



 Funding: Funding was provided by a sponsored research grant from GlaxoSmithKline, the Providence Portland Medical Foundation, and the M. J. Murdock Charitable Trust.



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3623ddf9190f1765df9cf05747187e1bf4ab0a38" target='_blank'>
              Multi-cancer comparative multi-omics approach reveals a permissive tumor microenvironment in renal cell carcinoma through accumulation of exhausted CD8+ and CD4+CD8+ double positive T cells 2661
              </a>
            </td>
          <td>
            M. Janghorban, Y. Koguchi, Noah D Simons, Wesley Rosales, V. Rajamanickam, Tanisha L. Christie, William L. Miller, Iliana Gonzalez, Zhaoyu Sun, Yaping Wu, Lakhvir Kaur, Brendan D. Curti, N. Yanamandra, Johanna Kaufmann, B. Bernard, Thomas Duhen, Eric Tran, William L Redmond
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is an aggressive primary brain cancer with few effective therapies. Standard endpoints such as overall survival, progression free survival or radiology are coarse markers of treatment response, and there is a strong need to improve the current understanding of how the tumor, host immune system and tumor microenvironment respond to therapies. Currently, few opportunities exist to obtain additional tissue samples to understand tumor responses apart from initial surgery. This lack of visibility into tumor and immune co-evolution under treatment is a fundamental limitation in our ability to develop and advance new GBM therapies. Stereotactic needle biopsies are routinely used for diagnosis; however, the feasibility and utility of investigative biopsies to monitor treatment response remains ill-defined.



 The goal of the present study was the investigate the feasibility of performing highly resolved multi-omics analyses on routine stereotactic biopsy samples obtained during surgery.



 Single-cell RNA sequencing, spatial transcriptomics, metabolomics, proteomics, phosphoproteomics, T-cell clonotype analysis, and MHC Class I immunopeptidomics was performed on standard biopsy tissue obtained intra-operatively. We also examine biopsies taken from different locations and provide a framework for measuring spatial and genomic heterogeneity. Finally, we investigate the utility of stereotactic biopsies as a method for generating patient-derived xenograft (PDX)models



 Optimized workflows resulted in sufficient yield to perform single cell RNA sequencing of biopsy samples and matched spatial proteomic profiling. MHC Class I immunopeptidomics and phosphoproteomics was generated and cross referenced to single cell gene expression data. Spatial proteomics data was integrated with single cell RNA sequencing and spatial metabolomics data. Multi-modal dataset integration highlights spatially mapped immune cell-associated metabolic pathways and validates inferred cell-cell ligand-receptor interactions. Intra- and inter-regional biopsy variance was also quantified. Finally, patient derived xenografts were generated using needle core biopsies.



 In conclusion, investigative biopsies provide data-rich insight into disease processes and will likely play an increasingly important role in evaluating treatment responses going forward.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3010146c38dd35467e336aa62e520ff0972396d" target='_blank'>
              INNV-33. Investigative needle core biopsies support multimodal deep-data generation in glioblastoma
              </a>
            </td>
          <td>
            Kenny K. H. Yu, Sreyashi Basu, Gerard Baquer, Ryuhjin Ahn, Jennifer Gantchev, Sonali Jindal, Michael S Regan, Z. Abou-Mrad, Marc J. Williams, Alicia D’Souza, Yuval Elhanati, Alexei Stortchevoi, Charles Couturier, The Gbm TeamLab, Vinay K. Puduvalli, J. Huse, R. Beroukhim, Betty Y. S. Kim, Sangeeta Goswami, A. Boire, M. J. Cima, Matthias Holdhoff, Calixto-Hope G. Lucas, C. Bettegowda, Stuart S. Levine, T. Bale, Cameron W. Brennan, D. Reardon, Frederick F. Lang, E. Chiocca, Keith L. Ligon, Forest M. White, Padmanee Sharma, Viviane Tabar, Nathalie Y R Agar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>152</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61684838fc3c53a5cd2329af114682e03471ba75" target='_blank'>
              Single-Cell Mendelian Randomization Identifies Cell-Type Specific Genetic Drivers of Lung Cancer Subtypes
              </a>
            </td>
          <td>
            Q. Yang, C. Yan, X.-f. Wang
          </td>
          <td>2025-12-02</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast Cancer is one of the most prevalent malignancies and the cause of mortality among women. Despite technological advancements, diseases have inherent heterogeneity, especially in the tumor microenvironment (TME), which poses a significant challenge for effective therapy. The study explores the spatial heterogeneity of two key molecular markers, EPCAM (Epithelial Cell Adhesion Molecule) and ERBB2 (HER2).  The research was conducted based on spatial transcriptomics and a graph-based clustering method. EPCAM plays a crucial role in maintaining epithelial integrity, and its dysregulation facilitates tumor invasion and metastasis. On the other hand, ERBB2 is a well-characterized oncogene that drives proliferation in HER2-positive breast cancer. The study leveraged 10x Genomics Visium data and a high-resolution histopathology image to analyze gene expression and spatially variable genes (SVGs) within the tissue. The study identified eight functional spatial domains. ERBB2 expression was localized to tumor cores, which reflects the role as an oncogenic driver. On the other hand, EPCAM exhibited a wider distribution, highlighting the involvement in tumor-stroma interactions and epithelial-mesenchymal transition (EMT). The findings demonstrate the utility of spatial transcriptomics in two aspects. The first is resolving tumor heterogeneity, and the second is to underscore the importance of spatial context in understanding the roles of EPCAM and ERBB2 for cancer progression. The insights contribute to the development of precision medicine approaches that target a specific tumor region for improving the therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b117f399cc68a98eef4707a09f6308f49ac08645" target='_blank'>
              Spatial Heterogeneity of EPCAM and ERBB2 Expression in Breast Cancer: Insights from Spatial Transcriptomics
              </a>
            </td>
          <td>
            Muhammad Afrizal Amrustian, Bita Parga Zen, Aminatus Sa'adah
          </td>
          <td>2025-10-27</td>
          <td>International Journal on Advanced Science, Engineering and Information Technology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Transgenic mouse models are excellent tools to model human antibody response against antigens of choice, given their ability to undergo diversification, in vivo affinity maturation, clonal selection and clonal expansion in their natural immune environment. Current antibody discovery methods lack the scale required to fully screen the antibody repertoire of transgenic mouse models to uncover the maximum number of desirable antibody hits per discovery campaign. Here, we demonstrate our novel combinatorial barcoding approach to enable high-throughput single cell BCR and whole transcriptome profiling of transgenic mouse samples with unprecedented sensitivity, scalability, and flexibility. As a proof of concept, we profiled over 1 million cells from multiple transgenic mice that were either naive or immunized with antigens of interest to obtain hundreds of thousands of paired BCR clonotypes. Using the whole transcriptome data we detected all B cell subpopulations including plasma cells at single cell resolution. We then used the paired heavy and light chain BCR data to detect productive chains in the majority of B cells revealing both unique and expanded clonotypes. In the expanded clonotype pool we found specific patterns of VDJ utilization and amino acid motifs compared to the naive pool. This method paves the way for high throughput screening of antibody repertoires from transgenic mice to allow for deep mining of antibodies.



 Technological Innovations in Immunology (TECH)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec58ff71103d49c869b14288e5b231dd3bddda07" target='_blank'>
              Deep-mining the antibody repertoires of transgenic mouse models with high-throughput single cell BCR and whole transcriptome sequencing 3955
              </a>
            </td>
          <td>
            Ajay Sapre, Alec Salvino, Guillermo Gallareta-Olivares, Efthymia Papalexi, Jose Jacob, Ashwath Kumar, Charles Roco, Alexander Rosenberg
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) evolves within a chronic inflammatory renal niche, where angiogenesis, metabolism, and immune programs synergize to reshape the tumor immune microenvironment (TIME). Recurrent renal damage and tissue hypoxia sustain NF-κB/STAT3 and HIF-VEGF signaling, while a sustained IFN-γ response enhances antigen presentation while inducing inhibitory checkpoints, promoting a state of “inflammation but constrained.” Single-cell and spatial studies reveal early microenvironment heterogeneity and the chemokine-checkpoint paradox: regions enriched in CXCL9/10 coexist with endothelial inertia, cancer-associated fibroblasts (CAFs)-mediated stromal barriers, and metabolic stress, which collectively exclude functional CD8+T cells. In advanced clear cell RCC, immuno-inflammatory, immuno-excluded, and immuno-desert phenotypes often coexist and undergo transitions during treatment, leading to heterogeneity in response to immune checkpoint blockade (ICB). We propose a modular perspective-the NF-κB/STAT3, HIF-VEGF, IFN-γ circuits and auxiliary regulatory factors-to link stage-specific biology with treatment matching. Integrative biomarkers couple IFN-γ characteristics with angiogenesis/stromal modules and spatial indicators, offering superior predictive power compared to single tests. These insights support the adoption of a combined strategy: integrating vascular normalization or stromal/myeloid cell reprogramming on the basis of ICB, and encouraging the use of longitudinal “immune snapshots” to guide intervention and precision immunotherapy for renal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bd0a59026e4807122514617a7c05a2fad79de65" target='_blank'>
              From chronic inflammation to immune escape: mapping the tumor microenvironment evolution in renal cell carcinoma
              </a>
            </td>
          <td>
            Hanjun Xu, Baojun Tu, Hui Li, Yong Shan
          </td>
          <td>2025-11-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Soft tissue leiomyosarcoma (STLMS) is an aggressive malignancy lacking validated molecular subclassification and effective targeted treatments. We performed comprehensive proteogenomic analysis of primary and metastatic STLMS to uncover biological traits and therapeutic weaknesses. Integrative proteomic and phosphoproteomic analyses using non-negative matrix factorization identified three subtypes. Subtype P1 shows genomic stability, low proliferation, and enrichment of FGFR2 and PDK signaling pathways. Subtype P2 exhibits chromosomal instability, inflammatory programs, activation of CDK-AURKA/B-mTOR/ERK kinome with IGF1R/PDGFRA gene alterations, and poorest survival outcomes. Subtype P3 is highly proliferative, with E2F/DNA-repair programs, elevated NCOR1, and shift towards non-homologous end joining with upregulation of PARP1. Homologous recombination deficiency (HRD) analysis distinguishes HRD-low P1 from HRD–high P2/P3. Paired analyses suggest HRD increases in metastases within P3. Immune profiling shows P2 as immunosuppressive, characterized by LGALS9 and M2 macrophages. Our proteogenomic analyses provide a molecular landscape of LMS, revealing biological insights, patient outcome stratification, and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7db5af2738185f0160cf0a8521442be786b57d" target='_blank'>
              Proteogenomic profiling of soft tissue leiomyosarcoma reveals distinct molecular subtypes with divergent outcomes and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Atsushi Tanaka, Makiko Ogawa, Yusuke Otani, Ronald C. Hendrickson, Zhuoning Li, Narasimhan P. Agaram, D. Klimstra, Julia Y. Wang, Michael H. Roehrl
          </td>
          <td>2025-11-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Recent advances in single-cell multi-omics technologies have revolutionized cellular analysis, enabling comprehensive exploration of cellular heterogeneity, developmental trajectories, and disease mechanisms at unprecedented resolution. Foundation models, originally developed for natural language processing, are now driving transformative approaches to high-dimensional, multimodal single-cell data analysis. Frameworks such as scGPT and scPlantFormer excel in cross-species cell annotation, in silico perturbation modeling, and gene regulatory network inference. Multimodal integration approaches, including pathology-aligned embeddings and tensor-based fusion, harmonize transcriptomic, epigenomic, proteomic, and spatial imaging data to delineate multilayered regulatory networks across biological scales. Federated computational platforms facilitate decentralized data analysis and standardized, reproducible workflows, fostering global collaboration. Challenges persist, including technical variability across platforms, limited model interpretability, and gaps in translating computational insights into clinical applications. Overcoming these hurdles demands standardized benchmarking, multimodal knowledge graphs, and collaborative frameworks that integrate artificial intelligence with human expertise. This review synthesizes recent technological advancements and proposes actionable strategies to bridge single-cell multi-omics innovations with mechanistic biology and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd39e97dd01c5b51454dd49ad3ed59f55d8cf50a" target='_blank'>
              Transformative advances in single-cell omics: a comprehensive review of foundation models, multimodal integration and computational ecosystems
              </a>
            </td>
          <td>
            T. Yiu, Bin Chen, Haoyu Wang, Genyi Feng, Qiangqiang Fu, Huijing Hu
          </td>
          <td>2025-10-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f5dbf13eb9670595afcc13d733cac2504d66e7a" target='_blank'>
              Single-cell aggrephagy landscape delineates intercellular communication of tumor microenvironment associated with ovarian cancer progression and immunotherapy.
              </a>
            </td>
          <td>
            Feng Fang, Jiemiao Chen, Xinran Li, Cao Zhou, Yilan Dai, Yuanyuan Chen, Xin Yang, Yong Xie, Fujie Qin, Huijiao Fu
          </td>
          <td>2025-11-24</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic cancer, recognized as a refractory tumor, has an overall survival rate of less than 10%, and its mortality rate continues to rise. Due to the low immune activity induced by its unique tumor microenvironment, pancreatic cancer is classified as a “cold” tumor and is insensitive to current immunotherapies. However, little is still known about the identification and functional mechanisms of key regulatory molecules in the formation of “cold” tumors. In this study, we identified the pancreatic ductal cell niche through single-cell sequencing. By calculating CNV scores using inferCNV to distinguish malignant from non-malignant cells and analyzing differences in transcriptional levels, we constructed a 15-gene model. The inhibitory effect of UBE2H on the immune microenvironment was studied through single-cell sequencing analysis, including the inhibition of T cell function, impairment of antigen presentation in macrophages and neutrophils, and activation of neutrophil extracellular traps (NETs). Our results revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche, primarily observed in pancreatic cancer samples. Notably, single-cell sequencing analysis of UBE2H overexpression revealed an enhanced T cell PD-L1 interaction axis and a downregulation of antigen presentation pathways in macrophages and neutrophils (antigen processing and presentation, macrophage activation, and neutrophil activation). Further in vivo experiments confirmed that high UBE2H expression promotes tumor progression, leading to increased T cell exhaustion (PD1) and decreased activation (CD69, GZMB, IFNR) in the immune microenvironment, impaired antigen presentation of macrophages and neutrophils (H2Kb, I-Ab, I-A/E), and increased neutrophil extracellular trap formation (MPO, NE, CITH3). This study revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche and analyzed the inhibition of UBE2H on the immune microenvironment of pancreatic cancer through single-cell sequencing and in vivo experiments, providing important clues for the formation of “cold” tumors in pancreatic cancer and opening new directions for exploring new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ecc8251593af0fcef99c0ebc48949b3c7fba9f" target='_blank'>
              Integrated scRNA-seq and transcriptome analyses uncover the effects of UBE2H on the immune microenvironment regulation in pancreatic cancer
              </a>
            </td>
          <td>
            Fuxin Huang, Zhongyan Zhang, Jike Fang, Yue Chen, Jinhui Wei, Quanzhang Li, Chuanzhao Zhang, Shanzhou Huang, B. Hou
          </td>
          <td>2025-11-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Lineage plasticity enables cancer cells to bypass therapeutic pressure, but the underlying chromatin mechanisms remain incompletely defined. A recent study by Lu et al. uncovers how the transcription factors NKX2-1 and FOXA2 reprogram 3D genome architecture to activate neuroendocrine gene networks during the transition from castration-resistant prostate cancer (CRPC) to neuroendocrine prostate cancer (NEPC) (Lu et al., 2025). Importantly, pharmacological inhibition of CBP/p300 markedly suppresses NEPC growth in vitro and in vivo, revealing a promising therapeutic avenue for advanced disease. These findings illuminate how spatial chromatin reorganization can drive cell fate transitions, positioning 3D genome dynamics as a key determinant of cellular identity in both development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c9ebfa025117c138be3767acc9d75dcb26318e" target='_blank'>
              3D chromatin dynamics in cellular neuroendocrine transformation
              </a>
            </td>
          <td>
            Jianhua Xiong
          </td>
          <td>2025-10-17</td>
          <td>Protein &amp; Cell</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Over 75% of PTCL-NOS patients fail to achieve long-term control with current therapies, highlighting the need for novel therapeutic targets. Genetic and non-genetic drivers in cancer cells require external cues from other cancer cells and from the lymphoma microenvironment (LME), which enable cancer cell adaptation to metabolic stress, immune mediated stress and treatment exposure. The resulting signaling crosstalk propels the co-evolution of cancer and LME cells with emergence of diverse cancer-supporting cell states. Identifying these and their maintenance mechanisms may therefore reveal novel therapeutic targets.
 We characterized the LME of PTCL using transcriptional signature deconvolution and multiparametric imaging in 845 patient samples. We identified four biologically and clinically distinct categories named Depleted (DP), B-cell rich (BR), Mesenchymal (MS) and Inflammatory (IN) based on the proportion of different LME cells. RNA-seq analysis showed that SMA+ cancer associated fibroblasts (CAF) are present in DP, MS and IN LMEs each with distinct phenotypes. To functionally analyze the contribution of CAFs to PTCL maintenance, we established multiple patient-derived tumor xenografts (PDXs) that retain the same LME categories of primary tumors. We then tested lymphoma cell survival in co-cultures of CAFs and PTCL cells from matched (e.g., IN-CAF + IN-PTCL) and unmatched PDXs (e.g., IN-CAF + DP-PTCL). Matched co-cultures sustained lymphoma cell proliferation to a higher extent than unmatched, independently of LME subtype. This suggests that CAF are distinctly reprogrammed within each LME.
 We hypothesize that pro-tumoral CAF phenotypes require adaptive gene expression in coordination with lymphoma cells' phenotypes. RNA-Seq of matched IN-CAF co-cultured with IN-PTCL revealed activation of pathways like “Transcription”, “DNA replication” and “Translation” including the upregulation of CDK7. Although CDK7 plays a minor role in cell cycle progression, its phosphorylation of the RNA polymerase II as part of the TFIIH complex is critical in activating and coordinating “cell state”-defining transcription.
 To assess CDK7's role in establishing lymphoma-supportive CAF states, IN-PTCL PDX mice were treated with the covalent selective CDK7 inhibitor YKL-5-124 for 12 h followed by transcriptomic and proteomic profiling of IN-PTCL and IN-CAFs. In IN-CAFs, CDK7 inhibition altered gene and protein expression of several collagens and cytokines, including COL1A1, COL1A2, MMP8, and CXCL12. Ligand-receptor analysis identified CXCL12-CXCR4 as the key CDK7-dependent interaction between IN-CAFs and IN-PTCL cells. IN-PTCL cells showed activation of DNA damage repair pathways, including upregulation of ALYREF and EIF4E, likely as compensatory mechanisms for decreased CDK7-dependent transcription. Since these may represent acquired therapeutic vulnerabilities, we conducted viability screenings in co-cultures of IN-PTCL and IN-CAF isolated from YKL-5-124 (and vehicle) tretated PDX mice. Cells were subsequently exposed for 48 h, individually and in co-cultures, to a drug library containing 40 clinical-phase compounds. This approach led to the identification of 14 compounds with synergistic anti-lymphoma effects. Among these, ten drugs including the XPO1 inhibitor Selinexor, the HSP90 inhibitor PU-H71 and the translation inhibitor Omacetaxine exhibited enhanced anti-lymphoma activity specifically in the co-culture setting, whereas four compounds showed reduced efficacy. This was not seen in mismatched co-cultures and indicates a potential protective role of CAFs and, importantly, that this mechanism is CDK7 dependent.
 To address the limitations of PDX models established in immunodeficient mice with limited ability to capture the immune-mediated effects of drug treatments, we developed an IN-PTCL syngeneic model by intrasplenic implantation of the murine T-cell lymphoma cell line EL4. Treatment with YKL-5-124 significantly reduced the tumor burden compared to vehicles. Cytokine profiling (immunoblotting array) and ELISA of culture medium of YKL-5-124 treated EL4-CAFs showed reduced levels of pro-inflammatory cytokines such as CXCL12, IL6, CCL2, ICAM-1, and CXCL1 indicating that CDK7 is necessary in establishing pro-lymphomagenic IN-CAF phenotypes.
 In conclusion, CDK7 activity is required to sustain specific pro-tumoral crosstalk interactions of matched CAFs and PTCL cells, which can be exploited therapeutically.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/386e7a67f6fb5e2e6ca873d1aaf4d401efdcd89c" target='_blank'>
              CDK7 enables adaptive gene transcription required by stromal cells to support distinct T-cell lymphoma phenotypes and represents a novel microenvironment-directed therapeutic target
              </a>
            </td>
          <td>
            K. Kouidri, D. Fiore, M. Revuelta, N. Di Siervi, Federico Ruiz Moreno, Subhashini Joshi, C. Kayembe, Abigail Taylor, G. Inghirami, L. Cerchietti, N. Zamponi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 For glioblastoma (GBM), like other solid tumors, intratumoral heterogeneity is likely a critical factor in promoting tumor growth, homeostasis, and therapeutic response. Recent applications of single-cell genomics techniques have revealed the scope and nature of GBM cell diversity. However, these techniques do not allow for the experimental manipulation of these same tumor subpopulations, raising a critical barrier to understanding tumor biology and developing effective therapies. To overcome this barrier, we developed endogenous gene reporter systems to enable the isolation, characterization, and manipulation of key GBM subpopulations directly in patient tumor isolates. This approach utilizes streamlined techniques for gene editing in GBM stem-like cells (GSCs), which eliminate the need for the cloning of individual cells. As proof of concept, we applied this approach to two genes in hGSC isolates: OLIG1 (GBM-OPC marker) and SOX9 (GSC/NSC stem cell homeostasis). We targeted the 3’ exon of each gene so that EGFP or RFP are in-frame with the upstream exon but separated by a 2A ribosome skip sequence. We then employed these GSC reporter isolates to comprehensively screen human transcription factors using a CRISPR-Cas9 library and LV-ORF library (1836 genes and 3548 isoforms) to identify transcription factors that activate or repress OLIG1 and SOX9. From these screens we identified 15 genes necessary and sufficient for regulating OLIG1 promoter activity, including known and novel regulators. These included SOX9 itself, which we find is a key factor linking OPC identify with self-renewal of GSCs. In addition, we identified dozens of transcription factor isoforms (validated by long read RNA-seq) that impact reporter activity, as well as ectopic factors not normally expressed in our GSCs which, nonetheless, significantly alter OLIG1 and SOX9 expression. At this meeting, we report results of these screens and present a functional genetic map of factors affecting the GBM-OPC cell identity, self-renewal, and plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9966806d724ae473e21ea19603f70a36860465" target='_blank'>
              STEM-26. COMPREHENSIVE FUNCTIONAL GENOMIC SCREENS REVEAL HUMAN TRANSCRIPTION FACTORS REQUIRED TO MAINTAIN GLIOBLASTOMA OLIGODENDROCYTE PROGENITOR CELL-LIKE IDENTITY
              </a>
            </td>
          <td>
            Anca B. Mihalas, Kelly Mitchell, Sonali Arora, Colette Felton, Mikey Witkowski, Aiden Abadilla, Daniel A. Kuppers, Angela Brooks, Alice Berger, Patrick J. Paddison
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a8fc3f37c54ae77fa1e33d4bf2768934a457fd9" target='_blank'>
              Single-cell transcriptomic profiling of bam mutant tumor reveals germline heterogeneity and gcrf1 as a modulator in Drosophila germ cells
              </a>
            </td>
          <td>
            Zhipeng Sun, Yujun Zeng, T. Nystul, Guohua Zhong
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/128cca3d1edd8a7877ad22a89e79521ff40bea75" target='_blank'>
              Single-cell sequencing in bladder cancer: new insights from tumor cell diversity to individualized treatment strategies.
              </a>
            </td>
          <td>
            Cen Liufu, Shuai Ye, Cong Yin, Tao Zhu, Jiahao Jiang, Yan Wang, Bentao Shi
          </td>
          <td>2025-12-03</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a leading male malignancy worldwide, with metabolic reprogramming being a critical hallmark of its progression. Extracellular vesicles (EVs) derived from tissues directly reflect the tumor microenvironment, offering unique insights into cancer pathophysiology that are unattainable through cell line or biofluid-derived EVs. However, the functional roles of tissue-derived EVs in PCa metabolism remain poorly understood. Leveraging our expertise in murine PCa model establishment and EV isolation from prostate tissue, this study aimed to characterize functional differences between PCa and normal prostate tissue via proteomic analysis of tissue-derived sEVs. We orthotopically implanted luciferase-labeled PCa cells into nude mice to establish an in situ PCa model, confirmed tumor formation via in vivo imaging, and harvested tissues after 4 weeks. sEVs were isolated using ultracentrifugation combined with an iodixanol density cushion and characterized by transmission electron microscopy, nanoparticle tracking analysis, and protein marker profiling. Proteomic analysis identified 28 upregulated and 24 downregulated proteins in PCa-derived sEVs compared to normal controls. Subcellular localization revealed enrichment in the cytoplasm, while pathway analysis highlighted significant involvement in metabolic processes, particularly glycolysis, amino acid biogenesis, carbon metabolism, and pyruvate metabolism. Our study establishes a robust method for isolating prostate tissue sEVs and provides the first evidence that PCa tissue-derived sEVs exhibit profound metabolic pathway alterations. These findings enhance our understanding of PCa progression mechanisms and may facilitate the development of novel diagnostic biomarkers and therapeutic strategies targeting metabolic dysregulation in PCa. SUMMARY: In this study, we created a method to isolate prostate tissue small EVs, based on our knowledge of the murine prostate cancer model building. Our data suggested that prostate tissue small EVs proteins significantly changed in many metabolism pathways, such as Glycolysis, Biogenesis of amino acids, Carbon metabolism and Pyruvate metabolism. In this study, we are the first to report prostate tissue-derived EVs proteins enriched in alterations of cancer metabolism. These differential proteins in PCa tissue EVs reflect metabolic changes in PCa and may provide insights into the development of early diagnostic biomarkers or novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c976599295f2bbc1baf63bb2a61d05088e5e38" target='_blank'>
              Proteomics of Prostate Cancer Tissue Small Extracellular Vesicles Reveal Alteration of Metabolism.
              </a>
            </td>
          <td>
            Lijuan Yu, Ting Ding, R. Olofsson Bagge, Lei Zheng, Xiaoke Hao
          </td>
          <td>2025-11-17</td>
          <td>Proteomics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) is a highly aggressive malignancy with a heterogeneous molecular landscape. This study investigates the interplay between mRNA subtypes and the tumor immune microenvironment (TIME) in HGSOC. We analyzed transcriptomic data to characterize four mRNA subtypes: immunoreactive, differentiated, proliferative, and mesenchymal. Using DESeq2 and WGCNA, we identified gene modules associated with clinical traits and immune cell infiltration. The immunoreactive subtype exhibited significant enrichment of M1/M2 macrophages, Th1 cells, and B cells, along with LAIR-1, FCRL5, and IRF4 expression. The mesenchymal subtype was associated with fibroblasts, while the proliferative subtype showed strong ovarian stromal cell involvement. Our findings reveal distinct immune landscapes associated with each mRNA subtype, suggesting potential therapeutic targets and prognostic markers for HGSOC.



 NIH/NCI U54CA118638; Chan Zuckerberg Initiative CZIF2002-007045



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aa36312916ae96eb8236af900b44415c56ff2fa" target='_blank'>
              Transcriptomic subtypes and immune infiltration in high-grade serous ovarian cancer 4644
              </a>
            </td>
          <td>
            Kaylin M. Carey, C. Young, , Eric D Dammer, Rajesh Singh, J. Lillard
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) exhibits significant transcriptomic diversity, limiting treatment efficacy. This study aims to identify conserved molecular pathways that remain stable in heterogeneous TNBC, providing reliable therapeutic and prognostic targets. Genomic diversity was assessed via Mutant-Allele Tumor Heterogeneity (MATH) in TCGA-TNBC cohorts. Single-cell RNA sequencing from TNBC samples (GSE176078) underwent non-negative matrix factorization (NMF) to identify intratumoral heterogeneity (ITH) metaprograms. Transcriptomic diversity was quantified using PCA-based cell-cell Euclidean distances. CellChat analyzed cellular communication in high- and low-diversity groups. Conserved pathways were identified using KEGG enrichment, ssGSEA, and spatial transcriptomics. And we confirmed the feasibility of targeting sphingolipid metabolism through in vitro and in vivo experiments. TNBC showed elevated genomic diversity, inversely correlated with immune infiltration and pathway activity. Single-cell RNA sequencing revealed ITH metaprograms across malignant subtypes, with Cancer Cycling cells showing proliferative enrichment. High transcriptomic diversity tumors exhibited enhanced Cancer Basal SC interactions with CAFs, promoting stemness and EMT. Multi-region TNBC dataset analysis confirmed sphingolipid metabolism as the most conserved pathway (high interpatient and low intratumoral heterogeneity), enriched in malignant regions and negatively correlated with immune cells. A prognostic model incorporating sphingolipid genes (SPTLC2, SGMS1, SGPP2) stratified high-risk patients. Inhibiting SPHK1 via PF-543 induced dose-dependent cell death. Sphingolipid metabolism could be a transcriptionally conserved pathway in TNBC. Its inhibition may overcome heterogeneity-driven resistance, and its prognostic value enables precise patient stratification, offering a potential strategy against TNBC plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/273a42854b1bca75f9d7c92c18142aeda8983ebb" target='_blank'>
              Conserved sphingolipid metabolism under transcriptomic diversity: a prognostic and therapeutic target in triple-negative breast cancer
              </a>
            </td>
          <td>
            Jian Li, Rui Chen, Xiaowu Wang, Qing Zheng, Ruijie Dai
          </td>
          <td>2025-11-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd9ada955f131727b146cbf93779ae18c5727fa" target='_blank'>
              Comparative multi-omics evaluation of the tumor microenvironment in dog and human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Y. Nussbaum, Jeffrey N. Bryan, Guangfu Li, Alana R. Rodney, William T. N. Culp, Peter J. Dickinson, Anna Koutoulas, Kate Megquier, Hong An, Skyler T. Kramer, Edward S Ricemeyer, Jonathan B. Mitchem, Wesley C. Warren
          </td>
          <td>2025-10-23</td>
          <td>Veterinary Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="CIC::DUX4 sarcoma (CDS) is a highly aggressive malignancy with limited therapeutic options. Here, we present a doxycycline-inducible CIC::DUX4 chimeric mouse model and a cancer line derived from it, imChCDS, that faithfully recapitulates the molecular, histological, and immunological features of human CDS. We demonstrate that CIC::DUX4 expression alone is sufficient to drive tumorigenesis in permissive lineages of soft connective tissues. The imChCDS cell line retains the transcriptional footprint of its mesenchymal cell of origin, develops metastatic tumors in immunocompetent hosts, and exhibits a clear dependency on the P300/CBP transcriptional co-activators. Notably, we identify CIC::DUX4/P300/CBP-mediated suppression of MHC class I (MHCI) as a key mechanism of CDS immune evasion. Genetical inactivation of CIC::DUX4 or pharmacological inhibition of P300/CBP induces cancer cell cycle arrest, restores MHCI expression, and triggers robust anti-tumor immune responses, thereby transforming the immunologically “cold” CDS microenvironment into a “hot” one and driving tumor regression. Together, these models offer a versatile and physiologically relevant platform to investigate CDS pathogenesis, unravel immune evasion mechanisms, and evaluate emerging therapeutic strategies, including those targeting CIC::DUX4/P300/CBP oncogenic axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9305f441fceeff970adfae523dd3b3c2cd1aa05" target='_blank'>
              Modeling CIC::DUX4 sarcoma reveals oncogene-mediated MHCI-dependent immune evasion
              </a>
            </td>
          <td>
            Ajay Ram Vachanaram, Erdong Wei, Ana Mitanoska, William Bassett, Michael Kyba, D. Bosnakovski
          </td>
          <td>2025-11-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Deciphering patient-specific mechanisms of cancer cell reprogramming remains a crucial challenge in systems oncology, as it is key to improving patient diagnosis and treatment. For this reason, comprehensive and patient-specific multi-omic characterization of tumor specimens has become increasingly common in clinical practice. Here, we developed PatientProfiler, a computational workflow that integrates proteogenomic data with curated causal interaction networks to generate mechanistic models of signal transduction for individual patients. PatientProfiler allows multi-omic data analysis and standardization, generation of patient-specific mechanistic models of signal transduction, and extraction of network-based prognostic biomarkers. We successfully benchmarked the tool on proteogenomic and clinical data derived from 122 biopsies of treatment-naïve breast cancer, available through the CPTAC portal. We identified patient-specific mechanistic models that recapitulate oncogenic signaling pathways. In-depth topological exploration of these networks revealed seven subgroups of patients, associated with unique transcriptomic signatures and distinct prognostic values. We identified well-known Basal-like 1 and 2 subtypes, while also highlighting distinct mechanistic drivers such as the MYC–CDK4/6 axis or NF-kappaB-mediated inflammatory programs. Beyond breast cancer, PatientProfiler offers a generalizable framework to transform cohort-level multi-omic data into interpretable mechanistic models, making it applicable across diverse cancer types and other complex diseases. PatientProfiler is a new workflow that analyzes multiomics data derived from cohorts of patients to generate personalized models of signal transduction at phosphorylation-resolution and to extract signalling-based transcriptomic signatures. Models recapitulate signalling events dysregulated in each patient to accelerate therapy prioritization. Application of PatientProfiler to breast cancer data produced a collection of 122 models that were used to identify prognostic biomarkers. PatientProfiler systematically captured molecular differences across the classical breast cancer subtypes. Models recapitulate signalling events dysregulated in each patient to accelerate therapy prioritization. Application of PatientProfiler to breast cancer data produced a collection of 122 models that were used to identify prognostic biomarkers. PatientProfiler systematically captured molecular differences across the classical breast cancer subtypes. PatientProfiler is a new workflow that analyzes multiomics data derived from cohorts of patients to generate personalized models of signal transduction at phosphorylation-resolution and to extract signalling-based transcriptomic signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0abc30f7b635f5bf66730a8c3d3f3c3975f11d5" target='_blank'>
              PatientProfiler: building patient-specific signaling models from proteogenomic data
              </a>
            </td>
          <td>
            Veronica Lombardi, Lorenzo Di Rocco, Eleonora Meo, Veronica Venafra, Elena Di Nisio, Valerio Perticaroli, Mihail Lorentz Nicolaeasa, C. Cencioni, Francesco Spallotta, Rodolfo Negri, Francesca Sacco, Livia Perfetto
          </td>
          <td>2025-10-10</td>
          <td>Molecular Systems Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Introduction: Subclonal diversity in multiple myeloma (MM)—defined by genetically distinct populations of malignant plasma cells—plays a pivotal role in disease progression, therapeutic resistance, and relapse. Characterizing subclone-specific transcriptomic profiles, copy number alterations, somatic variants, and their interactions with the immune microenvironment is critical for understanding tumor heterogeneity and identifying novel therapeutic targets. In this study, we analyzed smoldering multiple myeloma (SMM) and newly diagnosed multiple myeloma (NDMM) samples to explore the genomic landscape of subclones and their crosstalk with immune cells.
 Methods: CD138+ scRNA-seq and scATAC-seq multiomic data from SMM patients (n=10) and NDMM patients (n=21) were obtained from the published study dbGAP phs003220 (PMC11099140). Matched CD138- scRNA-seq data of immune cells were sourced from Sudha et al., IMS 2023. Subclones for each tumor sample were identified using inferCNV (v1.8.1) and a custom-built integration pipeline that utilized both scRNA-seq and scATAC-seq data. Cell-cell communications between subclones and immune cell types were identified using CellChat (v1.6.0) using the ligand-receptor interactions from CellChatDB. Copy Number Abnormality (CNA) profiles of subclones from single cell were compared with whole genome sequencing (WGS) data (phs003220) and similarity scores were computed. Due to differing technologies (RNA-seq vs WGS) and sequencing depth (single cell vs. bulk), this score reflects how well single-cell multiomics captures major chromosomal changes seen in WGS. Subclone-to-immune cell interactions were quantified to identify statistically significant chromosomal abnormalities that change their prevalence. Survival analysis was performed on the most common MM abnormalities to identify if subclonal co-occurrence influences prognosis. For each subclone, BAM files were generated, and somatic variants (SNVs + Indels) were called using Strelka2 with WGS control samples from the same patient.
 Results: We identified 23 subclones in SMM with a mean of 2.30 (min= 1; max=4); and 79 subclones in NDMM with a mean of 3.59 (min=1; max=8). Subclone and WGS CNAs were comparable to one another with mean Jaccard index (JI) of 0.57 for SMM and 0.69 for NDMM. Samples with low JI tended to have more subclones and more complex CNAs as measured through ploidy (correlation coefficient (r) = -0.069 in SMM and -0.177 in NDMM). NDMM subclones tended to have more CNAs than in SMM. From the IU cohort, Del13 was observed in 60% of the SMM and 45% of the NDMM patients. Gain1q was identified in 50% of the SMM and NDMM patients. Patients with co-occurring Gain1q and Del13q in the same subclone had significantly shorter progression-free survival (PFS) than patients with at least one of these alterations (median PFS 285 days vs 1194 days; P = 0.004). Subclones with Gain1q had a lower average percentage of interactions with CD8T cells (Gain1q=25.435, Norm1q=33.26; P=0.005). In NDMM subclones with Gain1q, there was a lower percentage of MHC-I pathway interactions with CD8T cells (P=0.001) and MHC-II pathway interactions with CD4T cells (P=0.001). Subclones with Del13q showed a higher percentage of MIF pathway (MIF-CD74/CXCR4) interactions in NDMM. SMM subclones with Del13p tended to have a lower percentage of MHC-I interactions in CD8T, CD4T and NK cells. Gain6p subclones had a higher percentage of MK pathway (MDK-NCL/ITGA4+ITGB1) interactions in SMM subclones. Somatic variants were identified for each subclone from the scATAC-seq. When compared with WGS somatic variants in NDMM, a median of 253 variants was found both subclones and WGS while 2752 were unique to subclones. Numerous pathogenic somatic variants were identified in the myeloma driver genes from subclones.Conclusions: The identification of subclone-specific immune interaction networks provides new opportunities for targeted therapeutic intervention. These results establish subclonal CNA profiling as both a prognostic tool and guide for precision immunotherapy in multiple myeloma, particularly for high-risk disease characterized by specific genomic alterations co-occurring in subclones. This study underscores the importance of single-cell analysis to fully capture the clinically relevant genomic heterogeneity that drives disease progression and treatment resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4387fcdd106b6a1f94375e3c3e641c2dbb1f9a" target='_blank'>
              Subclonal genomic heterogeneity shapes immune crosstalk and clinical outcomes in multiple myeloma
              </a>
            </td>
          <td>
            Parvathi Sudha, Sylvia Robertson, Sihong Li, Sean D. McCabe, Kelvin Lee, R. Abonour, Brian Walker, A. Suvannasankha, Travis Johnson
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Abstract Tumor tissues are composed of malignant subclones with diverse genetic profiles. Reconstructing the evolutionary trajectory of these subclones is crucial for understanding how tumors acquire malignant traits. However, current approaches to subclonal tree reconstruction are limited either by their reliance on single-cell DNA sequencing (scDNA-seq) that involve a small number of cells and thus yield low-resolution results, or using single-cell RNA sequencing (scRNA-seq) data, which despite including larger cell populations, remain susceptible to bias from high dropout rates and technical noise. Here, we introduce CluVar, an autoencoder-based framework for inferring the phylogeny of cancer subclones from scRNA-seq data using mutation profile analysis. To address the extensive missing variant information inherent in scRNA-seq datasets, CluVar incorporates a customized loss function and multiple hidden layers optimized for clustering. CluVar demonstrated superior performance in reconstructing phylogenetic trees of cancer subclones under a range of erroneous conditions. When applied to cancer scRNA-seq data, the phylogenetic tree predicted using CluVar aligned well with the transcriptomic profiles. These findings highlight its utility for tracing evolutionary trajectories and identifying novel variants associated with cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68f982657d8bf3e302b1cdc665cd1c0c69419b27" target='_blank'>
              CluVar: clustering of variants using autoencoder for inferring cancer subclones from single cell RNA sequencing data
              </a>
            </td>
          <td>
            Chae Won Kim, Heewon Park, Dohyeon Kim, Yuchang Seong, Minhae Kwon, Junil Kim
          </td>
          <td>2025-11-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Ovarian cancer is a heterogeneous disease with multiple molecular subtypes that influence treatment response and prognosis. Identifying these subtypes from high-dimensional genomic data remains challenging due to the complexity and noise inherent in transcriptomic datasets. In this pilot study, we developed a multi-stage computational framework to reduce dimensionality and uncover biologically meaningful subgroups. Starting from over 63,000 mRNA features, variance-based filtering, correlation pruning, and supervised selection narrowed the dataset to 80 highly discriminative transcripts. Using these features, we constructed co-expression similarity networks, and visual inspection of the network topology revealed four distinct groups among seven ovarian cancer cell lines. These subtypes aligned with known biological and mutational profiles including TP53-driven high-grade serous groups, PI3K/AKT- and ARID1A-associated clear cell/endometrioid groups, a drug-resistant OVCAR-3 group, and a hybrid IGROV-1 profile. This integrative approach demonstrates that combining feature selection with network modeling provides a powerful method for resolving ovarian cancer heterogeneity and improving subtype-specific insights. Abstract Background/Objectives: Ovarian cancer is a heterogeneous malignancy with molecular subtypes that strongly influence prognosis and therapy. High-dimensional mRNA data can capture this biological diversity, but its complexity and noise limit robust subtype characterization. Furthermore, current classification approaches often fail to reflect subtype-specific transcriptional programs, underscoring the need for computational strategies that reduce dimensionality and identify discriminative molecular features. Methods: We designed a multi-stage feature selection and network analysis framework tailored for high-dimensional transcriptomic data. Starting with ~65,000 mRNA features, we applied unsupervised variance-based filtering and correlation pruning to eliminate low-information genes and reduce redundancy. The applied supervised Select-K Best filtering further refined the feature space. To enhance robustness, we implemented a hybrid selection strategy combining recursive feature elimination (RFE) with random forests and LASSO regression to identify discriminative mRNA features. Finally, these features were then used to construct a gene co-expression similarity network. Results: This pipeline reduced approximately 65,000 gene features to a subset of 83 discriminative transcripts, which were then used for network construction to reveal subtype-specific biology. The analysis identified four distinct groups. One group exhibited classical high-grade serous features defined by TP53 mutations and homologous recombination deficiency, while another was enriched for PI3K/AKT and ARID1A-associated signaling consistent with clear cell and endometrioid-like biology. A third group displayed drug resistance-associated transcriptional programs with receptor tyrosine kinase activation, and the fourth demonstrated a hybrid profile bridging serous and endometrioid expression modules. Conclusions: This pilot study shows that combining unsupervised and supervised feature selection with network modeling enables robust stratification of ovarian cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7f756c19c8351ff216d426702a2f26787540f1a" target='_blank'>
              Subtype Characterization of Ovarian Cancer Cell Lines Using Machine Learning and Network Analysis: A Pilot Study
              </a>
            </td>
          <td>
            R. Thelagathoti, Dinesh S. Chandel, Chao Jiang, Wesley A. Tom, Gary D. Krzyzanowski, Appolinaire A. Olou, M. R. Fernando
          </td>
          <td>2025-10-31</td>
          <td>Cancers</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Epithelial–mesenchymal transition (EMT) is linked to an immunosuppressive tumor microenvironment (TME). However, direct comparisons of the TME in paired tumor regions with and without EMT in primary tumors—essential for elucidating EMT’s impact on the TME—are still lacking. Using Digital Spatial Profiler (DSP) assay and NanoString nCounter Digital Profiling, we analyzed immune-oncology-related markers in the TME of no special type (NST) and paired spindle carcinomatous (SPS) components, the later considered the EMT counterpart of the former, in nine cases of metaplastic breast carcinoma, a type of mammary carcinosarcoma. We identified macrophage markers as consistently and significantly enriched immune-oncology-related proteins within the TME profiles of SPS components compared to their paired NST counterparts. Additionally, we observed notable enrichment of macrophage-related signatures, M2 macrophage phenotypes, M2 macrophage-inducing cytokines, and M2 macrophage-related genes, as key distinctions in the TME profiles of SPS components relative to NST counterparts. Immunohistochemistry and multiplex immunofluorescence confirmed the presence of M2 macrophages, predominantly characterized by CD14+/CD68+/CD163+ phenotypes, in the SPS components. The in vivo findings were supported by in vitro analysis, which revealed that primary breast cancer cells undergoing spontaneous EMT exhibit enhanced induction of M2 macrophage polarization, with CSF1 contributing to this effect. Despite some heterogeneity, the upregulation of CSF1 and CCL2 likely contributed to the increased presence of M2 macrophages in the SPS components in MpBC cases. This EMT-driven M2 macrophage-enriched TME was also observed in sarcomatous versus carcinomatous components in nonmammary carcinosarcomas across organs, as well as in vimentin-positive versus vementin-negative triple-negative breast cancer, where vimentin expression is associated with EMT. Given the immunosuppressive role of M2 macrophages, our findings suggest that an EMT-driven, M2 macrophage-enriched TME—a consistent phenomenon across organs—can contribute to immune suppression. This underscores the therapeutic potential of macrophage-targeting strategies and immune checkpoint inhibition in treating aggressive carcinosarcomas characterized by EMT-mediated, M2 macrophage-rich TMEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f22a3edfc1ec4b0450a68532734e2d9ca5cb62" target='_blank'>
              Carcinomas exhibiting epithelial–mesenchymal transition manifest an M2 macrophage-enriched tumor immune microenvironment
              </a>
            </td>
          <td>
            H. Lien, Yu-Chia Li, Ruby Yun-Ju Huang, Ko-Chen Chen, Tom Wei-Wu Chen, I. Chen, Li-Ping Hsiao, Ling-Chun Yeh, Yen-Shen Lu
          </td>
          <td>2025-10-14</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Colorectal cancer (CRC) exhibits profound molecular heterogeneity between left-sided and right-sided tumors with distinct therapeutic responses that current static genomic analyses incompletely explain. We developed Dynamic Functional Influence Computation (DynaFIC), a computational framework modeling time-resolved signal propagation through biological networks to quantify functional influence beyond static expression levels. Using the GSE39582 dataset comprising 583 primary CRC samples, we performed confounder-adjusted differential expression analysis controlling for microsatellite instability status, BRAF mutations, Tumor Node Metastasis (TNM) stage, age, and sex, identifying 105 laterality-associated genes that underwent DynaFIC temporal network analysis. Right-sided tumors exhibited dramatically higher network connectivity density despite fewer nodes, creating distributed vulnerability patterns with HOXC6 as the dominant regulator, achieving 200-fold influence through network amplification. Left-sided tumors showed compartmentalized, hierarchical organization with PRAC1 as the primary regulator and predictable expression-influence scaling. Temporal clustering revealed distinct propagation kinetics: right-sided tumors demonstrated rapid signal saturation requiring early intervention, while left-sided tumors exhibited sustained propagation permitting sequential approaches. Stability Volatility Index analysis showed right-sided tumors maintain significantly higher systemic vulnerability. These findings establish anatomical location as a fundamental network organizational principle, suggesting that incorporating temporal dynamics into cancer analysis reveals therapeutically relevant differences for precision medicine applications in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d7c01b60bd996bd83a24952c12899281177b924" target='_blank'>
              Confounder-Adjusted Differentiation of Colorectal Cancer via Dynamic Propagation of Pathway Influence
              </a>
            </td>
          <td>
            L. Batrancea, Ömer Akgüller, M. A. Balcı, Gizem Çalıbaşı Koçal, L. Găban
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e8d2f1aaaa7f0e435a0bcfed4046ad25f2a01b" target='_blank'>
              Deciphering macrophage heterogeneity and factors driving M2 polarization in lung adenocarcinoma through single-cell RNA sequencing
              </a>
            </td>
          <td>
            Meiling Sheng, Beiwei Yu, Qunzhi Wang, Yuanchao Xiao, Ming Ling, Xiaoming Wu
          </td>
          <td>2025-10-16</td>
          <td>Apoptosis</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a high-grade glioma marked by high intratumoral molecular heterogeneity, resistance to therapy and poor prognosis. Leveraging a multisampling approach, we investigated the relation between genomic and physical distance, and whether this information was captured by imaging phenotypes and patient outcomes. We profiled 79 spatially distinct tumor regions from 24 GBM patients (F/M = 0.60; median age = 65.5 years) using deep whole-exome sequencing (>300x coverage). Genomic distance was quantified by pairwise Euclidean distance of somatic mutational profiles and integrated with 3D neuronavigation-based spatial coordinates and progression-free survival (PFS). Two distinct growth patterns emerged: expansive, defined by strong correlation between molecular and spatial distance (RE = 0.6), and stochastic, characterized by molecular divergence uncoupled from spatial proximity (RS = -0.2). High molecular distance correlated with reduced PFS (R = -0.5538, p = 0.026), and stochastic expansion predicted unfavorable outcome (p = 0.035), frequently localized to the frontal lobe. Radiomic analysis from contrast-enhanced T1-weighted MRI revealed that molecular distance was positively correlated with heterogeneous texture features (e.g., GLCM entropy, NGTDM complexity; p < 0.05). Stochastic tumors were enriched for radiomic heterogeneity (e.g.), while expansive tumors displayed homogeneous imaging textures, suggesting convergent phenotypic adaptation in genomically diverse tumors. Functional annotation using neuromaps demonstrated that tumors with high molecular distance exhibited positive correlations with differentiation- and angiogenesis-related metrics (SA_axis, genePC1, CBV), and negative associations with expression of neurotransmitter receptors (acetylcholine, serotonin, dopamine). Stochastic tumors correlated with increased Magnetoencephalography (MEG) derived timescales, implicating higher neural network engagement and spatial fluctuations. These findings suggest a link between type of clonal expansion, radiomic-based structural imaging phenotype, and functional neuroanatomy in GBM, providing a multidimensional framework to interpret tumor behavior and identify clinically relevant radiogenomic signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/111ccc1f311e4f7d5b04bf8582bc8726a2476d94" target='_blank'>
              CNSC-23. MULTIDIMENSIONAL SPATIAL AND MOLECULAR MODELING OF REFRACTORY GLIOBLASTOMA EVOLUTION
              </a>
            </td>
          <td>
            Youjin Song, Y. Miller-Michlits, K. Nenning, L. Greutter, B. Kiesel, D. Lötsch-Gojo, J. Gesperger, J. Breu, D. Nam, G. Widhalm, J. K. Sa, A. Woehrer
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD), a subtype of non-small cell lung cancer (NSCLC), has high incidence and poor prognosis. Although anti-programmed cell death protein 1 (PD1) therapy and epidermal growth factor receptor (EGFR) inhibitors benefit some patients, many remain unresponsive. Genome-wide association studies (GWAS) can identify risk loci, while single-cell transcriptomics enables exploration of genetic variations and mechanisms in LUAD. We integrated GWAS data from FinnGen, IEU OpenGWAS, and GWAS Catalog with single-cell and spatial transcriptomics from GEO to investigate LUAD genetics and pathology. GWAS analyses identified SNP loci, risk factors, LD Score, and SMR results, revealing a novel apoptosis-related locus, Serine/Threonine Kinase 24 (STK24) (SMR P < 0.05). Single-cell analysis identified STK24-positive epithelial cells (STK24posEpi) and their transcription factors (SCENIC), developmental trajectory (Monocle), and tumor microenvironment roles (CellCall). Spatial mapping (RCTD) and trajectory analysis (stlearn) were combined with cell communication (CellChat), spatial signal flow (Commot), and co-localization (Misty). Bayesian deconvolution linked STK24posEpi to TCGA-LUAD bulk transcriptomes, assessing overall survival (OS), immune infiltration, and clinical traits. A prognostic model (STK24SuperPC, mean c-index 0.61) was built using multiple machine learning algorithms to predict survival and immunotherapy response. Laboratory experiments tested STK24’s effects on LUAD cell proliferation and apoptosis. STK24 was identified as a LUAD risk factor (pSMR < 0.05). STK24posEpi communicated with fibroblasts, endothelial cells, and others via MIF and PDGF pathways, and was associated with apoptosis (Importance > 0.5), metastasis (Importance > 2), DNA repair (Importance > 1), and DNA damage (Importance > 2). Located at the origin of the trajectory, STK24posEpi correlated with poor prognosis (HR > 0, P < 0.05) and reduced lymphocyte infiltration (Pearson’s r < 0, P < 0.05). STK24 overexpression enhanced LUAD cell proliferation, migration, and colony formation, while reducing apoptosis, increasing BCL-2 and Caspase-3, and decreasing Cleaved-Caspase-3 expression. STK24, an apoptosis-related gene, promotes LUAD progression by enhancing proliferation and inhibiting apoptosis, with STK24posEpi interacting in the tumor microenvironment through PDGF and MIF pathways. These findings offer potential therapeutic targets and support personalized LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038883aea606036c62cbeb3f18e89605908569d" target='_blank'>
              Genome-wide association, single-cell, and spatial transcriptomics analyses reveal the role of the STK24-expressing positive cells in LUAD progression and the tumor microenvironment, identifying STK24 as a potential therapeutic target
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Yuheng Jiao, Meng Hou
          </td>
          <td>2025-10-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is an aggressive cancer thought to arise from impaired myogenesis. This can be substantially overcome by increasing the levels of pannexin 1 (PANX1), a critical component of the myogenic program, but the mechanism involved is unknown. Using RNA-seq, we have previously found that overexpression of PANX1 dramatically reshapes the transcriptomic landscape of RMS including downregulation of a myogenic modulator, APOBEC2 (apolipoprotein B mRNA editing enzyme catalytic subunit 2). Following this clue, we investigated the role of APOBEC2 in the PANX1-mediated suppression of RMS malignancy. Here we show that, using a panel of patient-derived RMS cell lines and tumor specimens, APOBEC2 is expressed in RMS, but that its levels are lower than those in both differentiating myoblasts and skeletal muscle. In most RMS cell lines examined, APOBEC2 accumulates during proliferation and sustains their stem-like characteristics, as evidenced by its ability to promote the growth of spheroids upon increased expression. Yet, ectopic PANX1 expression led to a marked downregulation of APOBEC2 across a large proportion of RMS cell lines assessed. Strikingly, these were the same cells in which PANX1 triggers multinucleation. We further reveal that, like healthy myoblasts progressing through myogenesis, the multinucleation observed here in RMS cells results from cell fusion. Importantly, in RMS cells engineered to overexpress APOBEC2, PANX1 no longer enhances cell fusion, but its other anti-tumorigenic properties are still preserved. Collectively, our data indicate that PANX1 promotes RMS cell fusion by downregulating APOBEC2 expression, driving these tumor cells further into the myogenic program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/398de031cdf1c2189823064d4c0ea5c0fa3f3f02" target='_blank'>
              Pannexin 1 induces Rhabdomyosarcoma cell fusion by downregulating APOBEC2
              </a>
            </td>
          <td>
            Alexandra Welten, Amit Bera, S. Langlois, X. Xiang, Keshav Gupta, Emily Freeman, Kyle N. Cowan
          </td>
          <td>2025-11-18</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7a9a257557fe4b01c761e5ab2bf1e53a00ef799" target='_blank'>
              TIF1γ drives oral cancer recurrence by the transcriptional regulation of self-renewal genes, such as HES1
              </a>
            </td>
          <td>
            K. P. Padmaja, A. Mohan, G. M. Nair, Jyothy S Prabhu, Hafsa Shabeer, V. P. Snijesh, P. G. Balagopal, A. Thameem, Alan Jose, N. George, Jiss Maria Louis, Parvathy Visakh, Gayathri V Nair, Anna P Joseph, V. S. Jaikumar, Riya Ann Paul, Jackson James, T. T. Maliekal
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Therapeutic resistance remains a defining challenge in oncology, limiting the durability of current therapies and contributing to disease relapse and poor patient outcomes. This review systematically integrates recent progress in understanding the molecular, cellular, and ecological foundations of drug resistance across chemotherapy, targeted therapy, and immunotherapy. We delineate how genetic alterations, epigenetic reprogramming, post-translational modifications, and non-coding RNA networks cooperate with metabolic reprogramming and tumor microenvironment remodeling to sustain resistant phenotypes. The influence of the microbiome is highlighted as an emerging determinant of therapeutic response through immune modulation and metabolic cross-talk. By summarizing key regulatory circuits, We establishe a unified framework linking clonal evolution, metabolic adaptability, and tumor ecological dynamics. We further synthesizes novel therapeutic strategies that convert resistance mechanisms into therapeutic vulnerabilities, including synthetic lethality approaches, metabolic targeting, and disruption of stem cell and stromal niches. Advances in single-cell and spatial omics, liquid biopsy, and artificial intelligence are emphasized as transformative tools for early detection and real-time prediction of resistance evolution. This review also identifies major translational gaps in preclinical modeling and proposes precision oncology frameworks guided by evolutionary principles. By bridging mechanistic understanding with adaptive clinical design, this work provides an integrated roadmap for overcoming therapeutic resistance and achieving sustained, long-term cancer control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5732b68dc3b5afcc6ec00f3b3bd38479653a277" target='_blank'>
              Drug resistance in cancer: molecular mechanisms and emerging treatment strategies
              </a>
            </td>
          <td>
            Jinxin Li, Jiatao Hu, Yiren Yang, Hanzhong Zhang, Ying Liu, Yu Fang, Le Qu, Anqi Lin, Peng Luo, Aimin Jiang, Linhui Wang
          </td>
          <td>2025-11-17</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Gliomas carry a high disease burden, particularly the aggressive IDH-wildtype subtype. However, some IDH-mutant gliomas can transition from a less aggressive G-CIMP-High (GCH) state to a more malignant G-CIMP-Low (GCL) phenotype, mimicking IDH-wildtype tumors. Understanding the cellular and molecular mechanisms behind this progression is vital for developing effective therapies.



 We propose that single-nuclei resolution analysis can reveal distinct neoplastic cell subtypes driving the GCH-to-GCL transition through activation of stemness- and proliferation-related gene programs.



 We analyzed 22 glioma specimens from 15 patients (including matched primary and recurrent tumors) using single-nuclei RNA- and ATAC-sequencing. After quality control and neoplastic cell identification via somatic CNVs, we applied non-negative matrix factorization (NMF) to identify cell subclusters based on gene expression patterns. We further characterized these clusters using gene set enrichment, protein-protein interaction networks, developmental trajectory analysis, and integration with external RNA-seq datasets. Chromatin accessibility was assessed via snATAC-seq to identify regulatory elements and transcription factor activity.



 We identified six major neoplastic cell subtypes: Astrocytic-Support, Pro-neural Housekeepers, Proteo-synthetic Cells, Mitotic Proliferators, Neural Guidance & Growth Cells, and Mesenchymal-mutant Cells. Each showed distinct gene expression signatures, pathway enrichments, and survival associations. Two subtypes—Mitotic Proliferators and Mesenchymal-mutant Cells—were linked to high-grade tumors and poor patient outcomes. Chromatin accessibility patterns corroborated transcriptional findings, highlighting differential TF regulation in aggressive subtypes.



 This study reveals two aggressive GCL-related neoplastic cell subtypes with distinct stem-like and proliferative features. These findings provide insight into glioma progression mechanisms and uncover potential molecular targets for therapeutic intervention in IDH-mutant gliomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd05421cc91e3cb0411e5625733c706631851341" target='_blank'>
              EPCO-21. SINGLE-NUCLEI ANALYSIS IDENTIFIES NOVEL NEOPLASTIC IDH MUTANT ASTROCYTOMA PROLIFERATIVE AND STEMNESS PHENOTYPES DRIVING POOR PATIENT OUTCOMES
              </a>
            </td>
          <td>
            H. Noushmehr, G. Herrgott, L. Garofano, Ian Lee, J. Snyder, T. Walbert, A. Castro, A. Iavarone
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Secreted phosphoprotein 1 (SPP1
 +
 ) macrophages are a recurrent and functionally critical immune cell subset across multiple cancer types. They drive adverse clinical outcomes by promoting immunosuppression, tumor invasion, metastasis, and therapy resistance. Given their prevalence and pivotal role, SPP1
 +
 macrophages have become a major focus in cancer immunology and a promising target for therapeutic development. SPP1
 +
 macrophages have been identified in a wide range of human malignancies through single-cell RNA sequencing and spatial transcriptomics studies. Their differentiation and maintenance are strongly influenced by reciprocal cellular interactions and hypoxic conditions within the tumor microenvironment (TME). Within the tumor microenvironment (TME), SPP1
 +
 macrophages promote tumor progression by interacting with cancer-associated fibroblasts (CAFs) and helping to form a physical barrier that restricts immune cell infiltration into the tumor core. Specifically, they impair the recruitment of CD8
 +
 T cells and promote T cell exhaustion (TEX). In this review, we focus on recent advances in understanding the differentiation of SPP1 macrophages in hypoxic tumor microenvironment and the role of SPP1
 +
 macrophages in immunosuppression and their therapeutic implications in cancer. Targeting this subset of macrophages has emerged as a highly promising therapeutic strategy, with several approaches demonstrating encouraging results in preclinical models.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7c7489149a391c5b3e442dad42f10334718f488" target='_blank'>
              SPP1+ macrophages in tumor immunosuppression: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Juanjuan Wang, Ya Wang, Yuqing Liu, Rongcun Yang
          </td>
          <td>2025-12-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/394995c0fcb10ae0448160919d2f62f9453d3170" target='_blank'>
              Comprehensive single-cell analysis reveals mast cells’ roles in cancer immunity
              </a>
            </td>
          <td>
            Hao Qian, Wenbo Wang, Jiaomeng Pan, Ming Zhao, Li Gao, Jia Zeng, Hang Gong, Guangyan Zhangyuan, Zi Li, Mao Zhang, Xia Shen, Yuanchi Weng, Xiaxing Deng, Yingying Huang
          </td>
          <td>2025-10-10</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88201947fac2d05b37de4b53710cc1d1179ec932" target='_blank'>
              MEF2D-expressing cancer precursors reprogram tissue-resident macrophages to support liver tumorigenesis.
              </a>
            </td>
          <td>
            Junyu Xiang, Y. Ni, Jiajun Yu, Hui Sun, Zhiyun Gu, Ziyong Li, Tongwang Yang, Juan Feng, Li Su, Limei Liu, Jiatao Li, Jesús Prieto, Matías A Ávila, Juanjuan Shan, Na Zhuang, Cheng Qian
          </td>
          <td>2025-10-23</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f662f4b181b3395cd096885302fa365b2511567e" target='_blank'>
              Exploring tumor heterogeneity and drug resistance in cervical cancer through single-cell and bulk transcriptomics.
              </a>
            </td>
          <td>
            Yinxia Yang, Sirong Cheng, Yuxuan Zhong, Pengyu Sun, Xiaoru Deng, Bohui Zhou
          </td>
          <td>2025-11-27</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d14996c871db81cbd57850e173c87363704962d" target='_blank'>
              Spatially varying cell-specific gene regulation network inference
              </a>
            </td>
          <td>
            Yurui Li, Jin Y Chen, Haohan Wang
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gliomas, especially IDH‑wildtype astrocytomas, remain lethal despite multimodal therapy. Collagen type XXII alpha-1 (COL22A1) exhibits oncogenic activity in other cancers but remains underexplored in brain tumors. We asked whether its expression, cellular origin, and spatial context add prognostic and translational value in diffuse glioma. We reprocessed TCGA/CGGA RNA‑seq to transcripts‑per‑million and re‑annotated tumours under the WHO 2021 taxonomy. Differential expression, Kaplan–Meier, and multivariable Cox models assessed survival associations; gene‑set enrichment profiled pathway context. Cell‑type sources were mapped by deconvolution and single‑cell datasets (GSE131928, GSE89567), and spatial distribution was examined using Ivy GAP and 10 × Visium. Protein abundance was validated by quantitative immunohistochemistry in 75 surgical specimens, with orthogonal proteomic corroboration from the TCGA Proteomic Data Commons. COL22A1 mRNA was up‑regulated in glioma and increased with grade. High expression predicted shorter overall survival and remained an independent hazard factor after adjusting for age, grade, Karnofsky score, radiotherapy, and temozolomide. Transcriptome deconvolution and Single‑cell analyses localized COL22A1 predominantly to mesenchymal-like tumour subpopulations with negligible signal in lymphoid, reactive-glial, or myeloid compartments. Spatial transcriptomics showed marked enrichment in peri‑necrotic and microvascular proliferation niches. Gene Set Enrichment Analysis (GSEA) linked COL22A1‑high tumours to epithelial–mesenchymal transition, cell-cycle, and inflammatory (IL6/JAK-STAT3, TNF–NFκB) programmes, consistent with a proliferative and hypoxic phenotype. Immunohistochemistry confirmed higher protein abundance in tumours versus normal brain, a stepwise increase with grade, and a trend association with Ki‑67; mass‑spectrometry data corroborated increased protein abundance and adverse survival at high expression. A parsimonious nomogram combining COL22A1 with clinical covariates improved the prediction of 1-, 3-, and 5-year survival. Across bulk, single‑cell, and spatial layers, COL22A1 integrates molecular, cellular, and microenvironmental hallmarks of glioma aggressiveness. Its tumour‑centric expression, peri‑necrotic localisation, and independent prognostic value support COL22A1 as a robust biomarker and a tractable candidate for imaging or therapeutic strategies in treatment‑refractory disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cbb933838cac0aea9d168b156f392bccc834d07" target='_blank'>
              COL22A1 expression identifies aggressive glioma and independently predicts survival through integrated multiomics and clinical validation
              </a>
            </td>
          <td>
            Wei-Wen Hsu, Wen-Shin Song, Pei-Chi Chang, Dueng-Yuan Hueng, Yao-Feng Li
          </td>
          <td>2025-10-21</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The biological heterogeneity between left- and right-sided colorectal cancer (CRC) poses a significant clinical challenge and the underlying regulatory mechanisms remain elusive. As an emerging hallmark of cancer, the contribution of circadian rhythm disruption (CRD) to this side-specific heterogeneity is unclear. By integrating single-cell and spatial transcriptomic analyses, this study shows that tumor cells in right-sided CRC exhibit significantly higher CRD scores and identifies the emergence of a NONO-positive tumor-cell subpopulation (NONO⁺ TC) as a key molecular feature of this phenotype. Spatial analysis further confirms that this NONO⁺ TC forms a tightly co-localized microenvironment with fibroblasts in situ. Notably, cell–cell communication analyses indicate that NONO expression does not augment the signal-sending capacity of tumor cells but instead reprograms them into highly efficient signal receivers. These augmented incoming signals predominantly originate from the more pro-tumorigenic myofibroblastic cancer-associated fibroblast (myCAF) subtype, indicating that NONO⁺ TC are particularly sensitive to malignant stromal inputs. In conclusion, our study delineates the dual role of NONO as both a circadian regulator and a pro-tumorigenic signaling hub. By enhancing tumor-cell reception of CAF-derived signals, NONO links CRD to the formation of NONO⁺ TC niches and a malignant microenvironment in right-sided CRC, providing new mechanistic insight into the spatial heterogeneity of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241b9e1e30a34229f91641257f9f4e9436ac45df" target='_blank'>
              NONO links circadian rhythm disruption and enhanced tumor-fibroblast crosstalk in right-sided colorectal cancer
              </a>
            </td>
          <td>
            Zhi-Hao Shang, Qin-Chang Zhang, Song-yang Xi, Kai Chen, Shao-Bo Guo, Zhou Zhou, Xin-Zhuo Zhan, Yun-Xia Wu, Xin-Yi Li, Hai-Bo Cheng, Xue-Jun Song, Gui-Hua Tian
          </td>
          <td>2025-10-31</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains one of the most difficult solid tumors to treat, owing in part to its remarkable cell state plasticity - a property that enables tumor cells to resist therapy, evade differentiation, and adapt to selective pressures. While targeted therapies have succeeded in subsets of patients, most tumors persist and eventually progress. To pursue a fundamentally new therapeutic strategy, we conducted an unbiased chemical screen using a custom-designed dual reporter system designed to broadcast cell state dynamics in real time by fluorescently tagging the endogenous SOX9 (aberrant stem-like) and KRT20 (differentiated) loci. This screen targeted epigenetic regulators capable of supporting CRC plasticity and preventing differentiation. We discovered that selective inhibition of histone deacetylases HDAC1/2 promotes differentiation and reduces tumor growth across human and mouse CRC models. This effect was not a consequence of cytotoxicity but rather epigenetic reprogramming. Through mass spectrometry-based histone modification profiling, we identified H3K27ac and H3K9ac as key epigenetic marks enriched at HDAC1/2-bound loci. These marks correlated with enhanced chromatin accessibility and the upregulation of differentiation-associated genes. Degradation of EP300, a major acetyltransferase, reversed the phenotype, confirming the functional relevance of histone acetylation in mediating this differentiation response. Furthermore, a focused CRISPR screen uncovered DAPK3 as a previously unrecognized downstream effector, providing a mechanistic foothold for future biomarker development. Our findings introduce a paradigm shift in CRC therapy: rather than targeting plastic tumor cells for elimination, we aim to force them into a differentiated, less adaptive state, making them more vulnerable to conventional and targeted treatments. This reprogramming effect is trackable via specific histone acetylation patterns, which we propose as candidate predictive biomarkers of response. Based on these insights, we hypothesize that HDAC1/2 inhibitors, when combined with KRAS or EZH2 inhibitors and/or chemotherapy, could yield synergistic therapeutic benefit. We are actively evaluating these combinations in preclinical models. In sum, this work positions HDAC1/2 inhibition as a precision epigenetic strategy to suppress CRC plasticity through enforced differentiation. By integrating multiomic profiling, functional genomics, and novel reporter technologies, we offer a blueprint for targeting the epigenetic underpinnings of therapy resistance in gastrointestinal malignancies, one that prioritizes reprogramming over destruction.



 Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, Sandor Spisak, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James Cleary, Jun Qi, Nilay Sethi. Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1992691ab8c82d8cb17a698db10f779b389e986a" target='_blank'>
              Abstract C031: Disarming colorectal plasticity: Dual HDAC1/2 inhibition reprograms tumor cell fate via H3K27ac-dependent differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Kishore Raghawan. Akhouri, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, Pranshu Sahgal. Sethi, Daulet Aitymbayev, Jennifer Susan . Thalappillil, Malvina Papanastasiou, William Hawkins, Steven Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Single-cell molecular profiling has emerged as a cutting-edge technology in biological research, playing a pivotal role in elucidating fundamental life processes and cellular heterogeneity. However, traditional methods generally provide end point measurements and require cell disruption, posing challenges for long-term monitoring of living cell events, e.g., cancer therapy. Tumor Treating Fields (TTFields), a noninvasive therapeutic modality, exert inhibitory effects on tumor growth and invasion by applying low-intensity intermediate-frequency alternating electric fields to tumor regions, whose molecular mechanism on cancer physiology remains poorly understood. In this study, we propose a multifunctional microchip capable of continuous in situ monitoring of molecular signatures in living single cells. The platform integrates spatial cell positioning, single-cellular multimolecule tracking with surface-enhanced Raman scattering (SERS), and TTFields treatment functionalities on a microchip. Using multiplexed analysis of liver cancer cells treated with TTFields and chemotherapy drugs, we implement spatial-temporal multiomics profiling for cancer therapy. Temporal tracking of critical molecular events associated with glycolysis, genetic stability, and membrane integrity reveals significant differences between the groups with and without TTFields treatment. Causal network analysis of molecular profiles uncovers the underlying mechanisms through which TTFields modulate oncogenic pathways. Our work demonstrates the effectiveness of the proposed microchip platform for real-time monitoring of living cell molecular dynamics, offering a powerful tool for precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889ada64ff2d00892ebd5cde7eb385bb8b31694b" target='_blank'>
              Spatial-Temporal Multiomics Profiling of Living Single Cells on Chip for Cancer Therapy.
              </a>
            </td>
          <td>
            Kunru Yu, Sheng-Zong Chen, Rong Zhu
          </td>
          <td>2025-10-12</td>
          <td>ACS sensors</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Understanding the mechanisms underlying key agricultural traits remains a central challenge in crop research, but recent advances in technologies are providing powerful tools to address this issue. Among these, single-cell and spatial transcriptomics have revealed tissue heterogeneity and spatial organization, offering unique insights into cellular gene expression dynamics and the coordinated activity of multiple cell types. These approaches help uncover how specific cell types contribute to agricultural traits and refine candidate loci lists through integration with trait-associated loci. Additionally, single-cell and spatial transcriptomics have the potential to serve as cell-level readout platforms integrating cellular perturbations, enabling high-throughput discovery of causal relationships between genotype and gene expression at the cellular level in plants. Successful implementation will accelerate the identification of key genetic variants for crop improvement. Here we review lessons learned from application of single-cell screening in mammalian cells, highlight major technical and biological barriers to its use in plants, and outline potential strategies to overcome these challenges. Together, the widespread application and integration of single-cell and spatial transcriptomics with other technologies enable not only the descriptive cataloging of cell states but also the causal interrogation of sequence functions and regulatory networks at cell-type resolution, ultimately advancing gene function studies and accelerating crop improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bff7551fbd396347f03bdd7b21419c8e91773900" target='_blank'>
              Decoding crops one cell at a time: from cell atlases to single-cell genetics.
              </a>
            </td>
          <td>
            Hao Zhang, Robert J. Schmitz
          </td>
          <td>2025-11-25</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edde3e6259df0a2e4e7c324d76c028ecfb0de34e" target='_blank'>
              TMPRSS11B promotes an acidified microenvironment and immune suppression in squamous lung cancer
              </a>
            </td>
          <td>
            Hari Shankar Sunil, Jean R. Clemenceau, Anthony Grichuk, Isabel Barnfather, S. R. Nakkireddy, Luke Izzo, Q. Feng, William Hartnett, Bret M. Evers, Lisa Thomas, I. Subramaniyan, Li Li, William C Putnam, Steven Hepensteil, Jingfei Zhu, Barrett L. Updegraff, John D. Minna, R. DeBerardinis, Tae Hyun Hwang, Jinming Gao, T. Oliver, Kathryn A. O’Donnell
          </td>
          <td>2025-11-10</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1fe8757b9db03b1f80d6119b9cc7fd8b56ac919" target='_blank'>
              Pan-cancer analysis reveals that tumor microbiomes generate specific epitopes through transcriptional reprogramming
              </a>
            </td>
          <td>
            Jinman Fang, Yuxin Zhang, Wei Wang, Xiao Zuo, Tao Chen, Kecheng Li, Huiya Yang, Tianyang Zhu, Yuanliang Zhang, Dandan Wei, Jianfeng Zhang, Jialong Cui, Zhen Tang, Shouwu Zhang, Lirong Wang, Zhilei Li, Dushan Ding, Ruixia Sun, Dandan Zhu, Yin He, Yanfei Zhou, Chenkun Ye, Dandan Chen, Xueting Lang, Long Jiang, Qian Zhao, Lu Zhang, Hongzhi Wang, Yuanwei Zhang
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Microbiomes within the tumor microenvironment (TME) are intricately linked to the modulation of cancer cell properties and the prognoses of various types of cancer, including esophageal squamous cell carcinoma (ESCC). However, their contribution to the onset and advancement of ESCC is still not fully elucidated. We adopted single‐cell RNA sequencing to investigate the effects of esophageal commensal bacteria during the development of mice's spontaneous ESCCs. By analyzing the single‐cell transcriptomes of 54,562 cells across different ESCC pathological stages, commensal bacteria were observed to display a cellular heterogeneous distribution in the ESCC TME. These bacteria were associated with the malignant transformation of epithelial cells and the formation of inflammatory fibroblasts, contributing to a chronic inflammatory microenvironment. However, after depriving commensal bacteria with an antibiotic cocktail (ABX), infiltrations of inflammatory fibroblasts and anti‐bacteria macrophages were lower than those in the control H2O group, while the infiltration of regulatory Cd4+ T (Treg) cells was higher. Interactions between inflammatory fibroblasts and Treg cells were intensified in the ABX group, which exacerbated the immunosuppressive state in the TME and developed more invasive carcinomas. This study demonstrates the significant role of commensal bacteria in ESCC tumorigenesis and sheds light on the potential clinical applications of adjusting commensal bacteria for the prevention and treatment of ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d454a6586fa201921140053c53f75c4969d9bdaa" target='_blank'>
              Single‐Cell RNA‐Seq Reveals the Effect of Commensal Bacteria on the Multistep Development of Esophageal Cancer
              </a>
            </td>
          <td>
            Xuan Xie, Shuyue Zhang, Zihang Mai, Jing Zhan, Sheng Lei, Zhiqiang Ouyang, Zelin Weng, Xiuying Xie, Jingjing Han, Jing Wen
          </td>
          <td>2025-10-13</td>
          <td>Cancer Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Objectives FAM49B has been shown to promote proliferation and metastasis of colorectal cancer (CRC) by stabilizing MYC through phosphorylation of NEK9; however, its role in shaping the immune suppressive tumor microenvironment (TME), particularly in macrophage polarization, remains unclear. Methods We applied multi-omics approaches to study CRC by integrating 33 scRNA-seq samples from 16 CRC patients, 2 paired spatial transcriptomics (ST) samples, and bulk RNA data to characterize malignant epithelial cells (High_FAM49B_EP) and tumor-associated macrophages (TAMs). Functional validation of FAM49B was conducted via knockdown experiments and proteomics analysis. Results A High_FAM49B_EP subpopulation was identified in primary tumors (PT) and liver metastases (LM), exhibiting elevated MYC signaling and association with poor prognosis. TAMs showed spatial heterogeneity: M1-like CXCL3+ TAMs predominated in PT, whereas M2-like SPP1+ TAMs were enriched in LM. CellChat analysis revealed that High_FAM49B_EP activated macrophage polarization through the MDK–NCL signaling axis. Pseudotime trajectory analysis confirmed differentiation from CXCL3+ to SPP1+ TAMs driven by upregulation of NCL. Spatial mapping showed co-localization of MDK+ epithelial cells with NCL+ TAMs in the immunosuppressive microenvironment. FAM49B knockdown significantly inhibited MDK expression and disrupted ECM–receptor interactions. Conclusions FAM49B promotes immunosuppressive TME formation by mediating TAM polarization via the MDK–NCL axis, suggesting the FAM49B–MDK–NCL pathway as a potential therapeutic target for CRC metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8878570ce8b97d296e4580e17e3836990c936b1" target='_blank'>
              Single-cell and spatial transcriptomics integration reveals FAM49B promotes tumor-associated macrophages polarization in colorectal cancer via the MK pathway
              </a>
            </td>
          <td>
            Tianyu Liu, Quchen Ding, Jin Gou, Chen Lu, Xingming Lu, Jiatong Chen, Y. E, Lianhong Li, Chongguo Zhang, Xiaojuan Zhu, Chunzhao Yu, Xiagang Luo
          </td>
          <td>2025-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) recurs frequently as the most common primary malignant brain tumor. Microvascular proliferation is a hallmark of GBM. Endothelial cells contribute to the tumor microenvironment (TME) in primary GBM and promote malignancy. However, their role in recurrent GBM (rGBM) remains unclear. Methods An analysis was made of single-cell RNA data from GSE182109. Endothelial cells were isolated. Subclusters underwent differential expression and pseudotime trajectory analysis for the identification of signature genes. TCGA data confirmed the expression patterns and prognostic significance of signature genes. Immunofluorescence was performed on matched primary-recurrent GBM samples. Signature genes were knocked down in human umbilical vein endothelial cells (HUVECs). Co-culture experiments with THP-1 and U87 cells explored crosstalk mechanisms and their potential upstream and downstream pathways. Results Two endothelial cell clusters were identified in single-cell analysis. Cluster 1 showed an elevation in rGBM and was associated with a poorer prognosis. Pseudotime analysis identified intercellular adhesion molecule-1 (ICAM1) as a signature gene. ICAM1 was up-regulated in rGBM, which was likely due to hypoxia. The knockdown of ICAM1 in endothelial cells reduced the adhesion of bone marrow-derived macrophages (BMDMs). Meanwhile, adrenomedullin (ADM) expression was decreased by lowering CCL2. Co-culture experiments demonstrated that macrophage-secreted ADM facilitated angiogenesis and the proliferation and migration of tumor cells. Conclusion This study suggests that hypoxia induces endothelial cells to upregulate the expression of ICAM1 during GBM recurrence and facilitates the recruitment of BMDMs. Concurrently, endothelial-derived CCL2 induces the expression of ADM in BMDMs. This macrophage-derived ADM, in turn, accelerates angiogenesis in endothelial cells while enhancing the proliferation and migration of tumor cells. These results elucidate a feedforward loop in the endothelial-BMDM-tumor cell axis, and provide mechanistic insights into the TME of rGBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/211b2af9797b0a57830e4b01b29ab35c7795b74e" target='_blank'>
              ICAM1-Mediated Endothelial Cells and Macrophage Interactions in Modulating GBM Malignant Transformation
              </a>
            </td>
          <td>
            Junlin Li, Tingyu Liang, Bo Liang, Xiaoman Kang, Wenbo Wu, Hai Wang, Huiyu Yang, Yu Wang, Wenbin Ma
          </td>
          <td>2025-11-01</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
Changes in the cyst microenvironment in Polycystic Kidney Disease (PKD) may drive progressive cyst formation. Bulk- and single-cell RNA sequencing have advanced our understanding of altered signaling; however, the lack of spatial information has limited our insights into local gene expression and cellular communication near cysts.


METHODS
We used wild-type and Pkd1-deficient mouse kidneys to generate 10x Genomics Visium Spatial Gene Expression datasets. Utilizing our single-cell mouse kidney atlas and single cell sequencing data for spot deconvolution, we enhanced resolution and estimated enriched cell types. We analyzed spatial gene expression patterns and used a cyst-centered analysis to identify cyst-associated gene signature. Cell communication near cysts was investigated, identifying key ligand-receptors. Prioritized key factors were validated in tissues.


RESULTS
We observed enrichment of fibroblasts, injury-repair-related cell types, and diverse immune populations in PKD. Injury-repair-related cells were exclusively observed in PKD, predominantly localized within immune cell-dense regions near cysts. These cells collectively contributed to the altered gene expression profile in PKD, including cyst-associated signature genes related to inflammatory processes. Analysis of cellular communication in less-inflamed regions around cysts revealed the involvement of multiple cell types. Key ligand-receptor interactions were associated with cytokine signaling, fibrosis, cellular development, and repair. These included Angpt2, C3, Csf1, Cxcl12, Il34, Gas6, Il16, Mdk, Mif, Ptn, Sfrp2, Spp1, Sdc1, Tnc, Tnfsf12, Wnt5a. In addition, ECM proteins implicated in immune response, ECM remodeling, cell adhesion, and cell signaling were identified, such as Adam9, Adam10, Col1a1, Col3a1, Col4a2, Lamb2, Lamc1, Efnb1, Efnb2, Thbs1, Thbs2, and Vcam1. IHC confirmed expression of Syndecan-1-Collagen IV, Midkine-Integrin β1, CSF-1, Pleiotrophin, and Tenascin-C in cystic kidneys.


CONCLUSIONS
Spatial transcriptomics in PKD revealed enrichment of (myo)fibroblasts, immune, and injury-repair-related cells near cysts, creating a (pro)inflammatory and (pro)fibrotic niche. Key ligand-receptor and ECM interactions were identified and validated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecdcf5e98138976e4b0b6f2eafc40e1b84148be0" target='_blank'>
              Spatial Transcriptomics Reveals Injured Cells, Signature Genes, and Communication Patterns in the Cyst Microenvironment of Polycystic Kidney Disease.
              </a>
            </td>
          <td>
            Sevtap A Yasinoglu, Claudio Novella-Rausell, L. Wisse, K. Dijkstra, A. Mahfouz, Hans J. Baelde, DJM. Peters
          </td>
          <td>2025-10-10</td>
          <td>Journal of the American Society of Nephrology : JASN</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Analyses of patient-derived cell lines have greatly enhanced discovery of molecular biomarkers and therapeutic targets. However, characterization of cellular morphological properties is limited. We studied cell morphologies of human pancreatic adenocarcinoma (PDAC) cell lines and their associations with drug sensitivity, gene expression, and functional properties. By integrating live cell and spatial messenger RNA imaging, we identified KRAS inhibitor–induced morphological changes specific for drug-resistant cells that correlated with gene expression changes. We then categorized a large panel of patient-derived PDAC cell lines into morphological and organizational subtypes and found differences in gene expression, therapeutic targeting potential, and metastatic proclivity. Patterns of cancer cell organization in human PDAC tissues stratified distinct gene expression signatures with clinical significance. In summary, we highlight the potential of cell morphological information in rapid, cost-effective assays to aid precision oncology efforts leveraging patient-derived in vitro models and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb047a3ab41c4e28c6535b371221906a6c927ab" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253b52aabc9f910e8e84f8a906624aeebc79b3e8" target='_blank'>
              Single-cell transcriptomic profiling reveals liver fibrosis in colorectal cancer liver metastasis.
              </a>
            </td>
          <td>
            Yiqiao Deng, Chengyao Guo, Xiaomeng Liu, Xin Li, Jianmei Liu, Wenjie Liu, Jinghua Chen, Zhen Huang, Ye‐fan Zhang, Xinyu Bi, Jian‐jun Zhao, Jianguo Zhou, Zhi-yu Li, Hongliang Wu, Baocai Xing, Qichen Chen, Hong Zhao
          </td>
          <td>2025-11-14</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are increasingly recognized as critical contributors to the limited efficacy of immunotherapy, particularly through the establishment of immune-excluded tumor microenvironments. However, the functional heterogeneity and spatial ecological roles of CAFs in melanoma remain poorly characterized. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq data to systematically dissect CAF molecular programs in melanoma using weighted gene co-expression network analysis (WGCNA). Spatial distribution, CellChat, and ligand–receptor interaction analyses were applied to construct CAF-centered immune exclusion networks. A functional scoring model (riskScore) based on the CAF-M1 module was developed to evaluate clinical relevance. CAF was spatially co-localized with endothelial cells, and engaged in strong crosstalk with multiple immune cell types via pathways such as COLLAGEN–integrin and THBS1–CD47. The derived riskScore model demonstrated robust prognostic value across TCGA-SKCM cohort. Through spatial aggregation and multifaceted immune signaling, CAF constructs a multilayered immune-exclusion network in melanoma, linking stromal remodeling to immune evasion. These findings offer novel insights into CAF-driven immune resistance and may inform future stratified immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca83751e1c8ec5203b2f4b9c36b4cadfbfd1df19" target='_blank'>
              Exploration of the roles of CAFs in melanoma based on single-cell transcriptomics and spatial transcriptomics
              </a>
            </td>
          <td>
            Sheng Hong, Yuhan Zhao, Anni Hu, Yizhong Zhang, Yuming Li, Yan Rong, Zukai Li, Wenzhuo Su, Lude Zhu
          </td>
          <td>2025-11-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 As cancer therapies improve and patients live longer, the incidence of brain metastases (BrMets) continues to rise. Among women with metastatic breast cancer, 10-15% develop BrMets, with rates as high as 30% in HER2+ and 50% in triple-negative. BrMets, represent highly evolved, therapy-resistant tumors with a median survival of 10 months and accompanied by severe neurologic decline. BrMets often emerge years after initial diagnosis, suggesting that metastatic cells must acquire brain-specific traits to survive. This process may involve transcriptional convergence—where tumor cells adopt brain-like programs, and brain-resident cells exhibit tumor-associated changes. Despite its clinical relevance, the molecular basis of this convergence remains poorly defined. Decoding, and targeting, these shared gene programs could reveal new therapeutic vulnerabilities.



 We performed spatial transcriptomic profiling on 235 patient tissue cores (BrMets, adjacent normal brain, primary breast tumors, and non-cancer brain). Using NanoString GeoMx, DSP, we measured the expression of 18,677 RNAs across 450 spatially-defined regions enriched for tumor, immune, and brain-resident cells. To interrogate transcriptional convergence, we developed the Equivalent Expression Index, a novel statistical tool to detect genes with biologically meaningful expression similarity across distinct cell populations.



 Our integrated analysis revealed two prognostic gene signatures. The Metastasis-Induced Brain Shared (MIBS-9) signature, comprising brain-like genes upregulated in tumor cells, was associated with longer patient survival. Conversely, the Adjacent brain-Resident Cancer Shared (ARCS-81) signature, defined by tumor-like gene expression in brain-resident cells, correlated with shorter survival. These signatures were externally validated and enriched for pathways related to neuronal remodeling, immune evasion, cellular communication, and cancer/metastasis.



 By coupling spatial transcriptomics with a novel equivalence-based algorithm, we uncover distinct, clinically relevant gene programs that mediate bidirectional adaptations between tumor and brain-resident cells. These findings illuminate key mechanisms of metastatic outgrowth and offer new targets for disrupting the breast cancer BrMet microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4c6b99afe4369f11ea5994698b0e8b275a5df02" target='_blank'>
              TMIC-89. Spatial transcriptomics reveals tumor–brain expression convergence and identifies prognostic gene signatures in breast cancer brain metastases
              </a>
            </td>
          <td>
            Maxine Umeh Garcia, Christine Yeh, Pablo Nunez Perez, Giuseppe Barisano, Hannes Vogel, Robert West, Michael Angelo, Lu Tian, Sylvia Plevritis, M. Gephart
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Small-cell lung carcinoma (SCLC) remains one of the most aggressive lung cancers and continues to pose a major challenge for precision oncology. Despite its morphological uniformity, SCLC exhibits marked molecular heterogeneity with recurrent, potentially targetable genomic alterations. Comprehensive profiling is often hindered by limited tissue availability and the need for rapid therapeutic intervention. We performed genomic profiling of 55 primary and metastatic SCLC samples using a 324-gene hybrid-capture next-generation sequencing panel. Consistent with prior reports, nearly all tumors exhibited biallelic TP53 and RB1 inactivation. Recurrent alterations involved the PI3K/Akt/mTOR pathway (62%), chromatin regulators (42%), and NOTCH signaling genes (15%). PTEN mutations were enriched in brain metastases. Frequent copy-number gains affected SOX2, NKX2-1, MYC-family genes, and CCNE1. Two novel recurrent amplifications of potential clinical significance were identified: TYRO3 (33%) and SDHA (13%). TYRO3, a TAM family receptor tyrosine kinase, and SDHA, a mitochondrial enzyme involved in succinate metabolism, may contribute to tumor progression and represent emerging therapeutic vulnerabilities. These findings underscore the genomic diversity of SCLC and highlight the potential utility of broad next-generation sequencing in uncovering new molecular targets for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c257de36865932aa9b4ccc1af139b71a5ba02b7" target='_blank'>
              Comprehensive Genomic Profiling of Small-Cell Lung Cancer Reveals Frequent Potentially Targetable Alterations
              </a>
            </td>
          <td>
            Dániel Schmalz, Zoltán Krabóth, Veronika Czoma, Péter Urbán, Attila Gyenesei, István Ruzsics, Veronika Sárosi, Á. Boronkai, Emőke Papp, B. Kajtár
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fb5388d0bd0362161f91d2111550a5aa12010f" target='_blank'>
              Inference of clonal hematopoiesis using the collective behaviour of DNA methylation states
              </a>
            </td>
          <td>
            Samuel. J. C. Crofts, Caleb M. Grenko, Neil A. Robertson, Kristina Kirschner, Tamir Chandra
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Brain metastases (BrMs) represent a significant clinical challenge with increasing incidence and poor prognosis. Here, we present a comprehensive proteogenomic characterization of pan-cancer BrMs (n=923) with patient-matched primary tumors (n=82) and glioblastomas (n=20), representing the largest cohort to date. Through consensus clustering of transcriptomic data, we identified and validated four distinct molecular subtypes (CMS1-4) with unique biological features. Integrated analysis of the transcriptomic and proteomic revealed that CMS1 was activated with neural system-associated pathways, CMS2 was characterized by extracellular matrix remodeling and increased immune cell infiltration, CMS3 exhibited a more epithelial phenotype along with metabolic reprogramming and CMS4 showed upregulation of cell cycle-related genes with elevated stemness scores. These molecular features were further validated through immunohistochemistry and multiplex immunofluorescence. Genomic analysis revealed subtype-specific mutational signatures and copy number alterations, with CMS1 exhibiting lower tumor purity and CMS4 displaying the highest chromosomal instability. Survival analysis indicates that CMS4 was associated with the worst prognosis, while CMS1 and CMS2 demonstrated more favorable responsiveness to radiotherapy and immunotherapy, respectively. Using patient-derived organoid models, we identify subtype-specific therapeutic vulnerabilities, notably metabolic inhibition sensitivity in CMS3 and CDK4/6 inhibition response in CMS4. Single-nucleus RNA sequencing further elucidates the complex cellular ecosystems and intercellular communication patterns within each subtype. This study creates a multi-omics resource for understanding the metastatic adaptation to the brain microenvironment and exploring personalized treatment strategies in BrMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8325830d999701778469a9b0a790181d9230e305" target='_blank'>
              EPCO-01. PAN-CANCER PROTEOGENOMIC AND SPATIAL CHARACTERIZATION OF 1000 METASTATIC BRAIN TUMORS
              </a>
            </td>
          <td>
            Gao Zhang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) represent a complex ecosystem of intertwined interconnections between various cell types. Through diverse mechanisms of intercellular communication, GBM cells can modulate the behavior of both malignant and non-tumor cells, promoting cancer progression. One of the key drivers of GBM plasticity and adaptability is extrachromosomal DNA (ecDNA). ecDNA allows GBM cells to rapidly and reversibly modify their genome and by thousands of folds increase the expression of critical oncogenes. However, virtually nothing is known about the role of ecDNA in intercellular communication. Using qPCR and immunofluorescent staining combined with fluorescent in situ hybridization, we demonstrated that ecDNA can be transported between different populations of GBM cells via tumor microtubes in vitro. The efficiency of this transfer substantially differs for various ecDNAs and for GBM cells derived from different patients. Our results suggest that recipient GBM cells can maintain exogenous ecDNA for up to five weeks. Further experiments with iPSC-derived cerebral organoids implanted with ecDNA-containing GBM cells revealed ecDNA transfer between GBM and non-tumor cells. To confirm that this process occurs in patients, we performed single-cell RNA sequencing and whole-genome sequencing of malignant and adjacent non-transformed brain tissues. Integrated analysis of mutations in DNA and RNA, copy number variations, and gene expression patterns confirmed that a substantial fraction of non-cancer cells within the tumor microenvironment harbor ecDNA. We hypothesize that ecDNA transfer between GBM cells can accelerate cancer evolution by disseminating the most beneficial ecDNA molecules within the tumor, thereby bypassing the bottleneck of clonal competition. On the other hand, ecDNA transfer from GBM to non-tumor cells can theoretically have a profound impact on the properties of recipient normal cells, altering their phenotypes to promote further tumor progression or even turning them into seeds for tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2992457556b240d36672234fd746fe0938a62d29" target='_blank'>
              TMIC-26. Intercellular transfer of extrachromosomal DNA in glioblastoma
              </a>
            </td>
          <td>
            Marat Pavlyukov, Anastasia Kazakova, Lindsey A Dudley, Yuji Nakano, Soniya Bastola, Kunal S. Patel, Harley Kornblum
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7d18180720ca9386d53478462a05bc6c11edd5b" target='_blank'>
              Inferring EMT dynamics from cell cycle profiles using a hidden Markov framework
              </a>
            </td>
          <td>
            Annice Najafi
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Introduction: The prevalence of epigenetic regulator mutations in clonal hematopoiesis and hematologic malignancies highlights the necessity of understanding how mutations in these regulators affect the fate potential of hematopoietic stem and progenitor cells (HSPCs) at a fundamental level. Utilizing single-cell multi-omics data, including paired transcriptome and chromatin accessibility (10X Multiome) and paired histone post-translational modification profiles and RNA polymerase II profiling (sciCUT&Tag + sciCUTAC) of human bone marrow mononuclear cells from healthy donors, we have identified regulatory dynamics of cell fate gain and loss unique to single lineages and quantify the loss of the stem cell program over a given fate transition.These tools will unravel the complex regulatory landscape of priming and lineage biases in healthy hematopoiesis and provide a pathway for investigating the specific mechanisms by which clinically observed epigenetic regulator mutations disrupt HSPC fate decisions in hematologic diseases.
 Methods: To construct a combined low-dimensional representation of chromatin accessibility scATAC-seq and sciCUTAC data were processed using shared genomic features using Seurat (Hao et al. 2021) and Signac (Stuart et al. 2021), then integrated using Seurat IntegrateData(). A shared pseudotime axis could then be computed using Palantir (Setty et al. 2019). To quantify epigenetic mark enrichment and chromatin accessibility at the gene level, gene scores were computed using ArchR (Granja et al. 2021). Using these scores, regulatory activity trends are fit for individual genes using Gaussian process regression (Otto et al. 2024).
 Results: Co-occurring H3K27me3 depletion, transcription, chromatin accessibility, and H3K27ac enrichment are restricted to the B cell lineage for PAX5 and EBF1, both critical master regulators of B lymphopoiesis. Comparable patterns of repression and de-repression at key transcription factor loci are detectable in the myeloid and erythroid lineages. The pseudotime ordering constructed from combined scATAC-seq and sciCUTAC data allows elucidation of the relative ordering of changes in histone mark enrichment, chromatin accessibility, and gene expression throughout hematopoiesis.
 Conclusions: Preliminary results from this work demonstrate the potential of co-analyzing more than two single-cell data modalities simultaneously, without the need for the development of novel techniques that can measure three or more modalities paired in single cells. The observed epigenetic activity at key master regulators, including PAX5 and EBF1 in B lymphopoiesis, defines a signature of lineage-specific gene activation during hematopoiesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f39fbc2f6b34120c4aaa60d8bbea50ff4cc6bd5" target='_blank'>
              Detecting lineage-specific regulatory activity and stemness retention in single-cell multiomics of human hematopoiesis
              </a>
            </td>
          <td>
            Cailin Jordan, Dominik J. Otto, Jacob Greene, S. Henikoff, Manu Setty
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Phenotypic plasticity plays a pivotal role in cancer, enabling tumor cells to adapt to environmental pressures and evade therapeutic interventions by transitioning between distinct cellular states. However, the contribution of phenotypic plasticity to adaptive drug resistance in glioblastoma (GBM), one of the most lethal of all cancers, remains poorly understood. In this study, we identify that GBM tumor-initiating cells resembling normal radial glia (RG), which occupy the apex of normal neurodevelopment, are driven by aberrant epidermal growth factor receptor (EGFR) signaling. Using a suite of patient-derived GBM models, we demonstrate through global proteomics and single-cell RNA sequencing that pharmacological inhibition of EGFR triggers a lineage transition toward neuronal and oligodendrocyte progenitor (OPC)-like states. This shift is accompanied by activation of oncogenic RAS-MAPK signaling – despite robust and durable inhibition of EGFR activation – and is further modulated by brain microenvironmental cues, including synaptic and calcium-mediated signaling programs. Dual inhibition of EGFR and RAS-MAPK with novel, tumor-selective small molecules blocks these phenotypic transitions and enhances GBM cell death in EGFR-mutated GBM models. To determine the subpopulation dynamics of RAS-MAPK signaling within GBM neurodevelopmental lineages, we develop DENALI (Dual-Expression Nuclear reporter of ERK Activity and Lineage Identity) – a novel, high-complexity barcoded lentiviral vector and integrative fluorescence reporter system. Using DENALI, we investigate the clonal mechanisms driving lineage plasticity in GBM following oncogenic EGFR inhibition and couple adaptive RAS signaling programs to the emergence of neuronal and OPC-like states under EGFRi therapy. Together, our findings establish neurodevelopmental lineage plasticity as a key driver of adaptive resistance in GBM and support dual-inhibition strategies to improve therapeutic outcomes in patients with GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a2a811b34aed9bceff625fc38dcd958e60df8" target='_blank'>
              STEM-23. GBM NEURODEVELOPMENTAL LINEAGE PLASTICITY DRIVES RAPID ADAPTATION TO ONCOGENE TARGETED THERAPY
              </a>
            </td>
          <td>
            Dimitri Cadet, Toby Harris, Elisa Fazzari, Cassidy Andrasz, Nicholas A. Bayley, Thomas G. Graeber, T. Cloughesy, A. Bhaduri, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Colorectal cancer (CRC) often occurs in combination with sarcopenia (SP), which affects the prognosis of CRC, but the histopathological mechanisms underlying this process are unclear. Single-cell transcriptome technology provides a tool for resolving multi-organ pathological interactions, and previous studies have lacked direct molecular evidence of a trans-tissue microenvironment. To address this, we applied an integrated analytical approach. The integration of single-cell transcriptome data from 16 CRC tumor tissues (scRNA-seq) and 17 SP muscle tissues (snRNA-seq) was used to systematically resolve heterogeneity across tissue microenvironments using Harmony to correct for batch effects and combining cellular annotations, differential expression analyses, and CellChat communication modeling. Our analysis revealed that CRC and SP share 3 core cell types, namely, endothelial cells, fibroblasts and monocytes, but they show distinct transcriptional and functional differentiation in different tissues. CRC cells are enriched in the inflammatory, angiogenic and matrix remodeling pathways, accompanied by the upregulation of hypoxia-associated genes, whereas SP is biased toward muscle structural maintenance, metabolic regulation and repair functions. The MIF-CD74, COLLAGEN and THBS-CD47 pathways were active in CRC, whereas SDC4 and CD36-related signals were dominant in SP. VWF, PDGFRA and FCN1 were stably expressed in both groups, suggesting the existence of a conserved homeostatic regulatory network. In conclusion, this study reveals the cross-tissue functional heterogeneity of cell types shared by CRC and SP, suggests a set of potential pervasive regulators, and provides new perspectives for understanding the systemic effects of tumors and the microenvironmental mechanisms of sarcopenia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28aac551ab88d08b24ed6a70bb80afdcde46ff4f" target='_blank'>
              Single-cell transcriptome analysis elucidates the microenvironmental interactions between colorectal cancer and sarcopenia
              </a>
            </td>
          <td>
            Tihong Qiu, Xiaoying Wang, Jie Li, Yiping Guan, Hongyou Wen
          </td>
          <td>2025-11-28</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) play crucial and important role in cancer dynamics by affecting homeostasis, immunosuppression, and angiogenesis within the tumor microenvironment. Using single-cell transcriptomics, we constructed a comprehensive atlas of myeloid cell populations across healthy and pan-cancer tissues that revealed heterogeneity. Our analysis suggested that TAMs may arise from two distinct origins: C1QC+ TAMs, which likely are derived from resident tissue macrophages, SPP1+ TAMs and ISG15+ TAMs, which appear to originate from circulating monocytes. Regarding immature myeloid-derived suppressor cells (MDSCs), we highlighted THBS1+ MDSCs and their descendants, SPP1+ TAMs, as key contributors to tumor progression, immunosuppression, and angiogenesis. We proposed a dichotomous model for TAMs, in which C1QC+ TAMs are associated with better patient outcomes, whereas the THBS1+ MDSCs - SPP1+ TAMs lineage correlates with poorer survival and unfavorable response to immunotherapy. This study offers insight into the complex interactions among monocyte-macrophage subtypes and sheds light on TAM heterogeneity and its implications for cancer progression and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a07f8d41863dfbb5389df0779f0c2e6e40c36fc" target='_blank'>
              Pan-cancer single-cell RNA sequencing analysis refines multi-origin monocyte and macrophage lineages.
              </a>
            </td>
          <td>
            Truc Do Thanh Nguyen, Andrew J Lee, Hyun Jung Park, Nameeta Shah, Bayrta Mandzhieva, Dong-Sup Lee, Inkyung Jung, Woong-Yang Park
          </td>
          <td>2025-11-13</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cce6be4d706b4653ff7064a56c5297b13de1df65" target='_blank'>
              Travelling Waves in Gene Expression: A Mathematical Model of Cell-State Dynamics in Melanoma
              </a>
            </td>
          <td>
            Charlotte Taylor Barca, Rotem Salmi-Leshem, Vishaka Gopalan, Sarah Woolner, Kerrie L. Marie, Gareth Wyn Jones, Oliver E. Jensen
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast cancer remains the most prevalent malignancy among women, necessitating the development of novel therapeutic strategies. Experimental animal models that closely mimic human breast cancer are crucial for advancing these therapies. This study utilized the criteria of the tumour, node, metastasis (TNM) staging system and variations in metabolic rates to develop models representing stages II and IV of human breast cancer, using the MBL-6 mouse breast cancer cell line. We assessed tumor growth curves in vivo and investigated distant metastasis to organs such as the liver, lungs, lymph nodes, and spleen. Carcinoma-associated fibroblasts (CAFs) were isolated, and their proliferation rates, inflammatory enzyme expression, and matrix metalloproteinase levels were compared between stages II and IV. By analyzing tumor kinetics and metabolic differences, we were able to predict tumor size and progression at each stage. Our results revealed that CAFs isolated from both stages exhibited similar phenotypic characteristics. However, CAFs from stage II tumors showed higher expression of indoleamine 2,3-dioxygenase 1 (IDO1), while those from stage IV tumors had higher levels of inducible nitric oxide synthase (iNOS). These distinct expression patterns suggest unique microenvironmental features at different stages of tumor progression. Further investigation of the cancer microenvironment may provide valuable insights for selecting targeted therapies and improving disease management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d06703a306e6912c38523fa9c498e367782abad" target='_blank'>
              Stage-specific tumor microenvironment dynamics and cancer-associated fibroblast profiling in MBL-6 mouse models of breast cancer
              </a>
            </td>
          <td>
            L. Langroudi, Maryam Iranpour, Mojtaba Mollaei, Masoud Soleimani, Seyed Mahmoud Hashemi, Zuhair M. Hasan
          </td>
          <td>2025-10-10</td>
          <td>Veterinary Science Development</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding the complex molecular mechanisms driving drug resistance in glioblastoma (GBM) is crucial to develop effective therapeutic strategies. While prior studies have identified resistance mechanisms tied to specific drugs or pathways, a multi-modal molecular analysis of resistance across a range of drug classes is lacking. We identified highly drug-resistant (n = 5) and drug-sensitive (n = 4) cultures from a cohort of 32 patient-derived glioblastoma stem cell (GSC) cultures screened against a broad panel of ~ 500 anti-cancer drugs. To elucidate the key drivers of drug resistance, we performed integrative profiling of stemness, differentiation capacity, global gene expression, mutation profiles, and DNA methylation patterns between the two groups. Despite heterogeneous gene expression profiles, drug-resistant GSCs showed consistent upregulation of ATP-binding cassette (ABC) drug efflux transporters, stemness, and extracellular matrix (ECM)-related genes. Compared to drug-sensitive GSCs, drug-resistant GSCs exhibited more pronounced stem-like properties and reduced differentiation capacity. Notably, genes linked to axonogenesis displayed significant CpG island hypomethylation in drug-resistant GSCs. This study suggests a pivotal role for GSC plasticity, stemness maintenance, and ECM-mediated drug evasion in GBM treatment resistance. Our findings highlight the adaptive and dynamic nature of resistance mechanisms in GSCs, emphasizing the need for comprehensive molecular insights to inform targeted therapeutic strategies in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39de902ec6cfcaa0e52251d953e6e404ad13d38" target='_blank'>
              Drug resistant glioblastoma stem cells exhibit enriched stemness signatures and share extracellular matrix overexpression
              </a>
            </td>
          <td>
            Lance L Estabillo, E. Skaga, S. Halldorsson, Einar O Vik-Mo, C. Sandberg
          </td>
          <td>2025-10-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11c95fbbadbecbdf667b52410258455d109e138e" target='_blank'>
              A comparative review of single-cell atlases: mapping cellular diversity across species and tissues.
              </a>
            </td>
          <td>
            Keyun Look, Pei En Goh, Long Chiau Ming, Shi Hui Law, Sang Loon Tan, Ee Wern Tan, Hooi‐Leng Ser, Bey-Hing Goh
          </td>
          <td>2025-12-02</td>
          <td>Cellular and molecular life sciences : CMLS</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Reconstructing and understanding intra-tumor heterogeneity, the coexistence of multiple genetically distinct subclones within the tumor of a patient, and tumor development is essential for resolving carcinogenesis and for identifying mechanisms of therapy resistance. While bulk sequencing can provide a broad view on tumoral complexity/heterogeneity of a patient, single-cell analysis remains essential to identify rare subclones that might drive chemotherapy resistance. In this study, we performed an integrated analysis of bulk and single-cell DNA sequencing data of core-binding factor acute myeloid leukemia patients, defined by the presence of a RUNX1::RUNX1T1 or CBFB::MYH11 fusion gene. By single-cell sequencing, we inferred tumor phylogenies for 8 patients at diagnosis including patient-specific somatic variants, somatic copy-number alterations and fusion genes, and studied clonal evolution under the pressure of chemotherapy for 3 patients. As a result, we developed an approach to reliably integrate subclonal somatic copy number alterations into phylogenetic trees and clonal evolution analysis, obtaining unprecedented resolution of intra-tumor heterogeneity in CBF AML. We were able to show that the fusion gene is among the earliest events of leukemogenesis at single-cell level. We identified remaining tumor clones in 6 patients with complete remission samples indicating incomplete eradication of the tumor clones. Here, we show that identifying the order of mutation acquisition can provide valuable insights into evolutionary history, offering a framework to improve drug selection in the era of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79dc8d9b0526060e06d2b2413c27f6af3f0f4c7" target='_blank'>
              Resolving intra-tumor heterogeneity and clonal evolution of core-binding factor acute myeloid leukemia patients with single-cell resolution
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina M. Strzelecka, Klara Kopp, Natalia Estrada, A. Dolnik, Marlon Tilgner, Coral Fustero-Torre, Felicitas R. Thol, F. Heidel, M. Heuser, L. Haghverdi, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2025-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 INTRODUCTION Monoclonal gammopathies –including MGUS, smoldering multiple myeloma (SMM), and multiple myeloma (MM)– are driven by clonal expansion of malignant plasma cells. Understanding their clonal architecture and the genomic landscape at single-cell resolution is essential to unravel disease progression and guide treatment strategies, particularly in the context of novel immunotherapies. In this study, single-cell DNA sequencing, was combined with surface protein profiling, to comprehensively dissect somatic mutations, copy number variations (CNVs), clonotypes, and immunophenotypes across disease stages and treatment responses. This novel single cell multi-omics approach provides a powerful perspective on the genomic and functional heterogeneity of plasma cell disorders, offering key insights for precision medicine and therapeutic stratification.
 METHODS CD138+ were isolated from bone marrow aspirates of 11 patients (21 samples) with MGUS, SMM, or MM using AutoMACS sorting. Patient samples included diagnostic, progression, relapse, and treatment-resistant stages. Single-cell DNA sequencing was performed using a panel of 839 amplicons, combined with surface protein profiling via a 20-antibody cocktail. Clonal architecture was inferred by integrating somatic mutations, clonotyping and CNVs. Surface protein expression was used to correlate genotypic features with targetable phenotypes at single cell resolution.
 RESULTS Clonal complexity and genomic architecture showed substantial heterogeneity across disease stages and therapeutic contexts. In early disease, such as MGUS and SMM, some cases already exhibited complex CNV profiles or early driver mutations, indicating subclinical clonal evolution. Several patients displayed stable CNV landscapes over time, while mutational profiles fluctuated, suggesting a selection or suppression of cell-specific subclones rather than wholesale genomic shifts.
 In relapsed/refractory multiple myeloma, branched clonal evolution was frequently observed, with sequential acquisition of mutations affecting genes such as TP53, FRG1, and TRAF2 in specific cells. In some cases, dual-clone coexistence with distinct genomic and immunoglobulin rearrangement profiles was evident, reflecting convergent tumor evolution. Furthermore, certain patients exhibited a distinct pattern of surface antibody expression, indicating the presence of plasma cells co-expressing both major therapeutic targets in MM, GPRC5D and BCMA, while other plasma cell populations expressed only one or neither of these targets.
 A key finding emerged when integrating genomic data with clinical outcomes: patients harboring monoallelic deletions and/or mutations in GPRC5D or BCMA (TNFRSF17) consistently showed limited or no clinical response to corresponding targeted therapies. As an example, one patient with biallelic inactivation of GPRC5D in near 90% of the cells prior to TALQUETAMAB treatment, failed to respond, correlating with absent membrane expression of the target. Conversely, a separate patient who progressed on anti-GPRC5D therapy demonstrated a favorable clinical response to BCMA-directed CAR-T cells, consistent with preserved BCMA expression and copy number in all the cells.
 These findings indicate that the genomic integrity and surface expression of immunotherapeutic targets strongly influence treatment efficacy. Importantly, discrepancies between gene copy number and surface protein levels — such as cases with BCMA amplification but heterogeneous expression — highlight post-transcriptional regulation as an additional layer of resistance.
 CONCLUSIONS Single-cell DNA and protein co-profiling in monoclonal gammopathies provides critical insights into clonal evolution, therapeutic resistance, and the dynamic expression of immunotherapeutic targets. Specifically, GPRC5D genomic alterations and reduced protein expression may compromise response to targeted therapies, highlighting the need for functional biomarker screening. Our data support the clinical relevance of integrating single-cell genomics in the management of plasma cell dyscrasias to tailor therapy and anticipate resistance. Our results suggest that performingsingle-cell genomic and proteomic profiling prior to treatment initiation may improve therapeutic selection and predict response, particularly in the era of targeted immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b66dd32e338c623da193862d4d414630dc6f82e" target='_blank'>
              Single-cell DNA profiling unveils clonal evolution and identifies mechanisms of therapeutic resistance and progression in monoclonal gammopathies
              </a>
            </td>
          <td>
            B. Ariceta, P. Aguirre-Ruiz, Andrea Castro, D. Alignani, Saioa Vicente, A. Lopez, A. Alfonso-Piérola, L. Tamariz-Amador, Jesús F San-Miguel, P. Rodríguez-Otero, Felipe Prósper
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72837cc909b2a1ccbe8a1449f9aafc89b0a2cfa3" target='_blank'>
              Epigenetic reprogramming as the nexus of cancer stemness and therapy resistance: implications for biomarker discovery.
              </a>
            </td>
          <td>
            Chu Xin Ng, Shin Yuh Lee, Xin Yi Yap, Yong Hui Wong, Jian Sheng Loh, Kuan Ping Ang, Wen Hwei Lee, Wei Xu, Phelim Voon Chen Yong, Sau Har Lee
          </td>
          <td>2025-11-16</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite the emergence of new therapies such as, immunotherapy, the treatment and diagnosis of Hepatocellular Carcinoma (HCC) still face many challenges, and the therapeutic outcomes for patients remain less than satisfactory. The study identified differentially expressed genes in HCC through differential analysis and then used univariate cox regression analysis to identify genes associated with prognosis. The intersection of these two sets of genes was used to obtain differentially expressed prognostic genes in HCC, which were then subjected to enrichment analysis. We analyzed two single-cell RNA sequencing (scRNA-seq) datasets from HCC patients, comprising 24,637 cells. Non-negative Matrix Factorization (NMF) clustering was used to identify cell cycle regulation in HCC tumor microenvironment (TME) cells, including three cell subpopulations: proliferating cells (PC), dendritic cells (DC), and macrophages (MAC). We employed the CellChat package to analyze cell–cell communication, the Monocle package for pseudotime trajectory analysis, and the SCENIC software package to study gene regulatory networks. Survival analysis was also performed using cell cycle-related features. A total of 26 clusters, including 15 major cell types, were identified in the HCC samples. Complex cell–cell communication networks were observed among these cell types. Enrichment analysis revealed that these cells were mainly enriched in pathways related to the cell cycle. The expression of cell cycle-related genes was elevated in tumor samples, and changes in cell cycle-related genes in specific subtypes were associated with different overall survival rates. The study focused on single-cell level data analysis of the cell cycle. The bubble plot results showed that the cell cycle scores were significantly upregulated in the PC, DC, and MAC subpopulations. Further subtyping revealed that these subtypes exhibited distinct biological states, cell–cell communication, and metabolic pathways. This study demonstrates that cell cycle regulation and cell–cell communication within the HCC tumor microenvironment impact tumor progression and patient prognosis. Cell cycle dysregulation in TME cells correlates with poor prognosis and immunotherapy efficacy, suggesting cell cycle targeting as a therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc3531834465d59ab5f37ade029201ae0f13d144" target='_blank'>
              Impact of single-cell cell cycle regulation of intercellular communication on the prognosis of hepatocellular carcinoma in the tumor microenvironment
              </a>
            </td>
          <td>
            Cong Hu, Rui Deng, Shuxiong Nong, Xinglang Mou
          </td>
          <td>2025-11-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 While single-cell profiling has been extensively explored in various tumor types, pituitary adenomas, particularly somatotroph tumors (ST), remain understudied. These growth hormone-secreting adenomas are clinically classified into two subtypes, Densely Granulated (DG) and Sparsely Granulated (SG), which have distinct prognostic implications. However, the cellular and molecular differences between these subtypes at the single-cell level are poorly understood. In this study, we performed single-cell RNA sequencing (scRNA-seq) on 12 somatotroph tumor samples, comprising 52,310 cells prospectively collected from acromegaly patients (10 DG and 2 SG). For the first time, we delineated tumor subclustering within ST, identifying two major populations: classic somatotroph (classic ST) and neuronal-like clusters (neuronal ST). A comparison of DG and SG subtypes revealed significant differences in immune cell composition, with DG tumors harboring higher proportions of myeloid and lymphoid lineage cells. Gene set enrichment analysis (GSEA) further highlighted the upregulation of immune-related pathways, including TNF-alpha signaling, in the DG subtype. These findings were validated by immunohistochemistry (IHC). Additionally, prolactin (PRL) expression was markedly elevated in DG tumors compared to SG tumors. A distinct population of cells co-expressing growth hormone (GH1) and PRL was identified, exclusively present in the DG subtype, and corroborated by IHC. Finally, copy number variation (CNV) analysis revealed substantial genomic instability within ST. Notably, most ST cells exhibited high CNV in the chromosome 20p arm within the classic ST cluster, while the neuronal ST cluster of the SG subtype displayed more prominent high CNV in the 20q arm compared to the DG subtype. This study represents the first single-cell transcriptomic analysis of somatotroph tumors, uncovering key cellular and molecular features that differentiate DG and SG subtypes. These findings provide new insights into tumor classification and hold significant potential for the development of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d4981aca3364216be263083130840bd242b07fd" target='_blank'>
              PATH-46. Single-cell Transcriptomic Profiling of Somatotroph Tumors Reveals Subtype-Specific Cellular and Molecular Characteristics
              </a>
            </td>
          <td>
            Euihyun Kim, H. Oh, Cheol Ryoung Ku
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose Monocytes and macrophages are recognized as predominant immune populations in human glioblastoma, where they play vital roles in tumor progression. Despite their established significance, the heterogeneity of these cells—particularly within the monocyte compartment—remains incompletely characterized. Methods We comprehensively used scRNA-seq, spatial transcriptome sequencing combined with immunofluorescence and T cell co-culture assays to illuminate the heterogeneity and function of monocyte in glioblastoma. Results In this study, from the perspective of ligand-receptor networks, we have identified and characterized three distinct glioblastoma subtypes. Single-cell RNA-seq analysis further revealed that the C3 subtype with bad prognosis exhibited a higher proportion of S100A9high monocytes. Spatial transcriptomics combined with immunofluorescence assays demonstrated that these S100A9high monocytes were spatially adjacent to M2 macrophages, exhausted CD8+ T cells, and endothelial cells. In vitro T cell co-culture assays revealed S100A9high monocyte produced elevated levels of the immunosuppressive cytokine IL-10, reactive oxygen species (ROS) and inducible nitric oxide synthase (iNOS), all of which might impair T cell function and immune response. Notably, elevated abundance of S100A9high monocyte correlated with poor patient prognosis. Conclusion In summary, our results deciphered the heterogeneity of monocytes in glioblastoma and identified a novel poor prognosis-associated monocyte subset, S100A9high monocytes, which foster an immunosuppressive, pro-tumorigenic microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b56f421b008f9fd171284dfd7fddcbd5bc0f429e" target='_blank'>
              Decoding the Prognosis-Related S100A9high Monocyte in Glioblastoma Using Single-Cell and Spatial Transcriptome Sequencing
              </a>
            </td>
          <td>
            Xiucan Li, Ying Qin, Pengfei Gao, Huixue Wang, Yanling Liu, Lian Ren, Dongdong Wu, Xueyuan Heng
          </td>
          <td>2025-11-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR)-T cell therapy has achieved significant progress in the treatment of hematologic cancers but continues to face major obstacles in solid tumors, including antigen heterogeneity, limited infiltration, and an immunosuppressive tumor microenvironment (TME). Oncolytic viruses (OVs) have emerged as promising tools to reshape the TME and improve CAR-T cell activity, yet many OVs encounter translational hurdles due to human seroprevalence and safety concerns. Newcastle disease virus (NDV), a naturally tumor-selective avian paramyxovirus, offers unique advantages as a non-integrating, non-pathogenic platform with a longstanding veterinary safety record and minimal pre-existing immunity in humans. NDV mediates direct oncolysis and immunogenic cell death, while simultaneously activating dendritic cells, repolarizing macrophages, and enhancing immune cell recruitment, thereby creating a TME that is more permissive to CAR-T cell therapy. Recent advances have enabled NDV to deliver immunostimulatory payloads locally within tumors, offering synergistic combinations to address CAR-T cell exhaustion and persistence. Looking ahead, further engineering efforts may expand the potential of this combined approach. This review summarizes the biological rationale, preclinical evidence, and translational prospects for integrating NDV with CAR-T cell therapy to improve outcomes in solid tumors. Newcastle disease virus (NDV) offers low human seroprevalence and strong safety as an oncolytic platform. NDV kills tumor cells and remodels the microenvironment to improve CAR-T cell efficacy. Engineered NDV can deliver local cytokines, chemokines, and checkpoint inhibitors to overcome CAR-T cell exhaustion. Synthetic biology may further expand NDV–CAR-T cell combination strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24757618f894f679c2162b5de9b2d0b9a4a2ccd2" target='_blank'>
              Engineering the tumor microenvironment: oncolytic NDV to facilitate CAR-T cell therapy
              </a>
            </td>
          <td>
            Mei Wang, Ke Jiang, A. Aicher, Christopher Heeschen
          </td>
          <td>2025-11-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly lethal malignancy with poor prognosis due to its resistance to conventional chemotherapy and limited number of targetable mutations. The recurrent loss-of-function mutation of the tumor suppressor ARID1A represents a key vulnerability potentially exploitable via synthetic lethality. To advance research in this area, we successfully established and characterized two novel CCA cell lines, SiSP-K01 and SiSP-K05, derived from moderately differentiated intrahepatic CCA. Both models were confirmed to carry ARID1A heterozygous loss and complete protein depletion, authentically reproducing the target genotype. Comprehensive molecular and phenotypic profiling (including karyotype, growth kinetics, 100% in vivo tumorigenesis, and tumor marker expression) confirmed their fidelity to the patient tissue and CCA lineage. Importantly, exome and transcriptome analysis not only validated the cell lines but also revealed an associated derangement of the PI3K/AKT signaling pathway. Based on this molecular finding, we performed drug screening and demonstrated that these ARID1A-deficient CCA cell lines are hypersensitive to inhibition by both PI3K/AKT and PARP inhibitors. The SiSP-K01 and SiSP-K05 cell lines are therefore critical new preclinical models suitable for functional studies of ARID1A deficiency and the development of targeted therapies for CCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef0e9dfbc0a68397e20306d980742627a07920fa" target='_blank'>
              Establishment of novel cholangiocarcinoma cell lines with ARID1A deficiency and preclinical validation of synthetic lethality therapies
              </a>
            </td>
          <td>
            Sunisa Prasopporn, G. Sittithumcharee, Jantappapa Chanthercrob, Somchai Limsrichamrern, Arada Hirankitti, Pimkanya More-krong, Sakda Sathirareuangchai, A. Chaiboonchoe, Somponnat Sampattavanich, Seiji Okada, Siwanon Jirawatnotai
          </td>
          <td>2025-11-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a00c830ef8ab043112aca0d9f13cae9a2beec912" target='_blank'>
              Met-Vision stratifies single cell metabolism revealing coexisting energetic states in tissue macrophages redistributed by inflammation
              </a>
            </td>
          <td>
            Adriana Lecourieux, Margot Bardou, Zacarias Garcia, Philippe Bousso
          </td>
          <td>2025-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This study addresses the limited mechanistic understanding behind medical imaging for tumor microenvironment (TME) assessment. We developed a novel framework that analyzes tumor imaging heterogeneity index (TIHI)-correlated genes to uncover underlying TME biology and therapeutic vulnerabilities. DCE-MRI and mRNA data from 987 high-risk breast cancer patients in the I-SPY2 trial, together with mRNA data from 508 patients in GSE25066, were analyzed. TIHI-associated genes were identified via Pearson correlation, clustered via weighted gene co-expression network analysis (WGCNA), and subgroups were defined via non-negative matrix factorization (NMF). The clinical relevance of the image-to-gene comprehensive (I2G-C) subtype defined by subgroups was assessed using logistic regression and Cox analysis. I2G-C comprised four clusters with distinct immune and replication/repair functions. It further stratified receptor, PAM50, and RPS5 subtypes. The “immune+/replication+” was more likely to achieve pathological complete response (pCR) (OR = 2.587, p < 0.001), while the “immune−/replication−” was the least likely to achieve pCR (OR = 0.402, p < 0.001). The “immune+/replication+” showed sensitivity to pembrolizumab (OR = 10.192, p < 0.001) and veliparib/carboplatin (OR = 5.184, p = 0.006), while “immune-/replication-” responded poorly to pembrolizumab (OR = 0.086, p < 0.001). Additionally, “immune+/replication-” had the best distant recurrence-free survival (DRFS), whereas “immune-/replication+” had the worst (log-rank p = 6 × 10−4, HR = 5.45). By linking imaging heterogeneity directly to molecular subtypes and therapeutic response, this framework provides a robust, non-invasive surrogate for genomic profiling and a strategic tool for personalized neoadjuvant therapy selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaac41c7947552aff84e1e324fe7769979665178" target='_blank'>
              Tumor Imaging Heterogeneity Index-Inspired Insights into the Unveiling Tumor Microenvironment of Breast Cancer
              </a>
            </td>
          <td>
            Qingpei Lai, Xinzhi Teng, Jiang Zhang, Xinyu Zhang, Yufeng Jiang, Yao Pu, Peixin Yu, Wen Li, Tian Li, Jing Cai, Ge Ren
          </td>
          <td>2025-11-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Anaplastic thyroid cancer (ATC) and papillary thyroid carcinoma (PTC) exhibit significant differences in clinical behavior and immune microenvironments, particularly concerning the mechanisms underlying CD8+ T cell dysfunction. However, these specific mechanisms have yet to be thorThe original blots and abbreviations are presented in Supplemeoughly investigated. The present study utilized single-cell RNA sequencing (scRNA-seq) data to conduct a comprehensive analysis of CD8+ T cells in the thyroid tissues of patients diagnosed with ATC and PTC. The results of the study indicate that CD8+ T cells in ATC display disruptions in energy supply and marked signs of exhaustion. Conversely, CD8+ T cells in PTC are more prone to maintaining a stable expression of immunosuppression-related membrane proteins through posttranslational modifications. This study highlights the distinct mechanisms of CD8+ T cell exhaustion in two types of thyroid cancer, offering valuable insights into the regulation of their immune microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6d7073dd0f41d411330fa169f596b64ce85797" target='_blank'>
              Single-cell transcriptome analysis reveals the potential heterogeneous mechanism of CD8+ T cell immune dysfunction in thyroid cancer
              </a>
            </td>
          <td>
            Qinling Zhang, Kaiyu Song, Chaolin Li, Xicheng Song, Jin Zhou
          </td>
          <td>2025-10-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="As a common malignant tumor, the heterogeneity of colorectal cancer plays an important role in tumor progression and treatment response. In recent years, the rapid development of single-cell transcriptomics and spatial transcriptomics technologies has provided new perspectives for resolving the heterogeneity of colorectal cancer. These techniques can reveal the complexity of cellular composition and their interactions in the tumor microenvironment, and thus facilitate a deeper understanding of tumor biology. However, in practical applications, researchers still face technical challenges such as data processing and result interpretation. The aim of this paper is to explore how to use artificial intelligence (AI) technology to enhance the research efficiency of single-cell and spatial transcriptomics, analyze the current research progress and its limitations, and explore how combining AI approaches can provide new ideas for decoding the heterogeneity of colorectal cancer, and ultimately provide theoretical basis and practical guidance for the clinical precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48a5aab5c189afed70eaf2f7feb72d37acadd46e" target='_blank'>
              Multidimensional decoding of colorectal cancer heterogeneity: Artificial intelligence-enabled precision exploration of single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Wen-Yu Luan, Qi Zhao, Zheng Zhang, Zhen-Xi Xu, Si-Xiang Lin, Yan-Dong Miao
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Brain tumors such as glioblastoma remain among the most lethal and immunologically resistant cancers, in large part due to epigenetic programs that sculpt the tumor–immune microenvironment. DNA methylation, histone modifications, and chromatin remodeling do not merely drive tumor-intrinsic changes; they also profoundly reprogram immune responses, shaping antigen presentation, cytokine signaling, and immune cell recruitment. At the center of this regulation are T cells, whose effector functions are suppressed through promoter hypermethylation of antigen-processing genes, silencing of interferon pathways, and the establishment of exhaustion-specific chromatin states. Mutations such as IDH1/2 and H3K27M further reinforce these epigenetic barriers, fostering immune-cold microenvironments that disable cytotoxic T-cell activity. Emerging evidence highlights both CNS-specific adaptations, including microglial and astrocytic epigenetic programs that reinforce immune privilege, and conserved features of T cell exhaustion that mirror those in peripheral cancers. This duality underscores the need for therapeutic strategies that dismantle CNS-specific barriers while leveraging shared exhaustion programs across tumor types. Epigenetic drugs, ranging from DNA methyltransferase and EZH2 inhibitors to BET degraders and CRISPR-based epigenome editors, are beginning to restore antigenicity, reverse T cell dysfunction, and sensitize tumors to checkpoint blockade. Yet these approaches carry the paradoxical risk of disrupting CNS immune tolerance, potentially triggering harmful neuroinflammation or autoimmunity. To our knowledge, this is among the first comprehensive reviews to integrate CNS-specific immune privilege mechanisms with peripheral exhaustion pathways, providing a unified perspective on how epigenetic regulation orchestrates immune dysfunction across central and peripheral contexts. By mapping the continuum between immune evasion and global immunosuppression, we propose a conceptual framework for tailoring epigenetic-immunotherapy combinations to achieve durable antitumor immunity in the CNS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d7202ed1c4ffc9d0ee982c0f1ad42403d845276" target='_blank'>
              Beyond the genome: epigenetic regulation of immune responses and T cells in brain tumors
              </a>
            </td>
          <td>
            Shuo Sun, Yu Han, Haiying Li, Chengyan Wang, Shu Zhou, Xiaowei Zhang, Shuhong Dai, Yao Peng, Zhuoqun Wang
          </td>
          <td>2025-11-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immunotherapy remains ineffective in many solid tumors due to poor T-cell infiltration and a metabolically suppressive tumor microenvironment. A dual strategy combining low-level light (LLL) therapy with a nanoscale stimulator of interferon genes (STING) agonist formulation (nanoSTING@Mn) is presented to enhance immune activation and metabolic fitness for durable tumor immunity against T-cell lymphoma (EL4) model. NanoSTING@Mn, composed of ADU-S100 complexed with Mn2⁺ and encapsulated in biomimetic liposomes, potently activates the cGAS-STING pathway, induces a type I interferon response, and promotes lymphocyte infiltration. These monocytes polarize into M1 macrophages, suppressing regulatory T cells. Simultaneously, LLL photo-biomodulation reprograms mitochondrial metabolism in tumor-infiltrating CD8⁺ T and natural killer cells, restoring their durability and leading to complete local tumor eradication. This combination expands a distinct CD8⁺ T-cell subset with Tcf-1⁺ progenitor-exhausted features and elevated memory/effector gene expression, enhancing proliferation and cytotoxicity, as shown by single-cell RNA sequencing. Intranasal nanoSTING@Mn delivery mobilizes these LLL-revived T cells to the lung, where they differentiate into resident memory T cells and establish systemic antitumor immunity. Upon intravenous rechallenge, disseminated tumor cells are eliminated, preventing metastasis and ensuring long-term protection. This synergistic approach offers a scalable platform to boost immunotherapy efficacy and redefines immune-based metastasis prevention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e9ba5699caa736e818a9807d89458c43c8a2b62" target='_blank'>
              Light Metabolically Reprograms CD8+ T Cells to Potentiate STING-Driven Tumor Eradication and Prevent Metastasis.
              </a>
            </td>
          <td>
            Asmita Banstola, Shilin Gao, ZhengKun Zhang, Yan Dong, Prabhat Upadhyay, Quanwei Zhang, Yongli Li, Zuan-Tao Lin, Zhilong Wang, Mei X. Wu
          </td>
          <td>2025-10-23</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Malignant gliomas follow two distinct natural histories: de novo high grade tumors such as glioblastoma, or lower grade tumors with a propensity to transform into high grade disease. Despite differences in tumor genotype, both entities converge on a common histologically aggressive phenotype, and the basis for this progression is unknown. Glioma associated macrophages (GAM) have been implicated in this process, however GAMs are ontologically and transcriptionally diverse, rendering the identification and isolation of pathogenic subpopulations challenging.



 Since macrophage contextual gene programs are orchestrated by transcription factors acting on cis-acting promoters and enhancers in gene regulatory networks (GRN), we hypothesized that functional subpopulations of GAMs can be resolved through GRN inference. Method IDH mutant/wildtype gliomas and control normal brain samples were collected for single cell RNA +ATAC sequencing to derive GRNs. Transcription factor network analysis identified cell surface markers to isolate subpopulations for functional assays, with tissue, in vitro and in vivo correlates.



 A subpopulation of human GAMs can be defined by a GRN centered around the Activator Protein-1 transcription factor FOSL2 preferentially enriched in high grade tumors. Using this GRN we nominate ANXA1 and HMOX1 as surrogate cell surface markers for activation, thus permitting prospective isolation and functional validation in human GAMs. These cells, termed malignancy associated GAMs (mGAMs) are pro-invasive, pro-angiogenic, pro-proliferative, possess intact antigen presentation but skew T-cells towards a CD4+FOXP3+ phenotype under hypoxia. Ontologically, mGAMs share somatic mitochondrial mutations with peripheral blood monocytes, and their presence correlates with high grade disease irrespective of underlying tumor mutation status. Furthermore, spatio-temporally mGAMs occupy distinct metabolic niches; mGAMs directly induce proliferation and mesenchymal transition of low grade glioma cells and accelerate tumor growth in vivo upon co-culture. Finally mGAMs are preferentially enriched in patients with newly transformed regions in human gliomas.



 mGAMs play a pivotal role in glioma progression and represent a potential therapeutic target in human high-grade glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7408d85300870de072bd9687aac588be6dfef1cd" target='_blank'>
              TMIC-37. A pathogenic subpopulation of human glioma associated macrophages linked to glioma progression
              </a>
            </td>
          <td>
            Kenny K. H. Yu, Z. Abou-Mrad, Kristof Törkenczy, Isabell Schulze, Jennifer Gantchev, Gerard Baquer, Kelsey Hopland, Evan D. Bander, Umberto Tosi, Cameron W. Brennan, Nelson Moss, Pierre-Jacques Hamard, R. Koche, Caleb A. Lareau, Nathalie Y R Agar, T. Merghoub, Viviane Tabar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="We present a reproducible protocol for isolating and analyzing natural killer (NK) cells from freshly resected human glioblastoma multiforme (GBM) tissue using flow cytometry. The workflow yields viable single‐cell suspensions suitable for downstream applications, including cell sorting, immunophenotyping, functional assays, and single‐cell RNA‐seq.© 2025 Wiley Periodicals LLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06f1df545a0cd99ba6068b96f7eb4984ee839068" target='_blank'>
              Isolation and Flow Cytometric Analysis of Natural Killer Cells from Human Glioblastoma Multiforme (GBM) Tissues
              </a>
            </td>
          <td>
            Soumyajit Das, Kyle B. Lupo, Veronika Slivova, Aaron A. Cohen-Gadol, Sandro Matosevic
          </td>
          <td>2025-11-01</td>
          <td>Current Protocols</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Relapsed/refractory (R/R) acute myeloid leukemia (AML) remains a major clinical challenge, particularly in patients previously exposed to hypomethylating agents (HMAs). Despite the widespread use of HMAs in combination with venetoclax, the mechanisms of action and resistance remain poorly defined, and predictive biomarkers are lacking. To address this, we applied the Proximity Network Assay (PNA) (Karlsson et al. 2025. bioRxiv doi:10.1101/2025.06.19.660329), a next-generation single-cell spatial proteomics platform, to longitudinal bone marrow biopsies from 10 R/R AML patients treated with decitabine – 5 complete responders (CR) and 5 non-responders (NR). PNA constructs nanoscale protein interaction maps by combining barcoded antibodies, rolling circle amplification, and proximity-dependent ligation, enabling sequencing-based reconstruction of single-cell protein networks. Each cell yields multiplexed spatial nodes across 155 proteins, allowing simultaneous quantification of protein abundance, colocalization, and polarity with ~50 nm resolution.
 We employed this approach to capture the dynamic remodeling of the tumor interface and stem cell compartment during therapy. Despite similar blast count prior to treatment, patients who responded to decitabine exhibited higher proportions of monocytes and NK cells. Blasts from responding patients showed increased abundance and colocalization of immune recognition molecules (HLA, B2M), adhesion proteins (CD18, CD43, CD54), and reduced CD45RB. In contrast, CD44 abundance and clustering was increased in blasts from non-responders, with increased abundance of different adhesion proteins (CD62P, CD82, CD26), suggesting differential membrane organization and interaction with the microenvironment and immune effectors.
 Following decitabine, CR patients maintained lower blast counts, and persistence of higher numbers of immune effector cell frequencies were observed. Early upregulation of CD71 and CD36 on CD34+ cells with a distinct immunophenotypic “proxiome” likely indicates early erythroid recovery and metabolic activation in healthy HSC recovery. CD90 remained elevated in remission marrows, consistent with physiologic HSC expansion. In contrast, non-responding patients retained high CD34 clustering and adhesion molecule abundance, potentially representing a marker of persistent leukemic stem-like cells. Using proximity and abundance features, we trained a classifier to predict response. Prediction modeling utilizing the random forest algorithm was able to discriminate between healthy versus leukemic HSC based on PNA protein interactions (AUC 0.889), with CD36/CD82, CD71/CD71, and B2M/HLA-ABC colocalization emerging as top predictive features based on importance scores. These signatures enabled tracking of clonal evolution across timepoints. This study represents the first longitudinal spatial proteomics analysis of AML response to HMA therapy. PNA reveals dynamic remodeling of the tumor interface and identifies predictive signatures of response and resistance. Integration with methylation-aware Oxford Nanopore Technology sequencing is underway to correlate chromatin accessibility with proteomic remodeling and stem cell fate.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e77531367efd62a536d7049f2833bf23bea0a560" target='_blank'>
              Longitudinal single-cell spatial proteomics reveals predictive immune and stem cell signatures in Relapsed/Refractory AML
              </a>
            </td>
          <td>
            Varun S. Sudunagunta, Stefan Petkov, Rongjia Deng, Yi Chen, Sebastian Fernando, Edna Stewart, Michele Simonetti, S. Iyer, Monica Kasbekar, Daniel Lee, Mark Frattini, Simon Fredriksson, J. Jurcic, T. Rosenblat, Aaron Viny
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db2469c6632fab23ebab1e0e71bc53bea256e7dd" target='_blank'>
              CDKN2A homozygous deletions and TSC2 somatic mutations in metastatic pancreatic neuroendocrine tumors.
              </a>
            </td>
          <td>
            Tito Teles Jesus, Lorenzo Ferrando, Lia Rodrigues, Rui Sousa Martins, Luís Cardoso, , Paula Soares, Arnaud Da Cruz Paula, J. Vinagre
          </td>
          <td>2025-12-05</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAM) exert both pro- and anti-tumoral functions that influence cancer progression and patient prognosis. However, single-cell RNA sequencing (scRNA-seq) studies have revealed that TAM heterogeneity remains incompletely characterized. By performing an unbiased, integrated in silico analyses of publicly available scRNA-seq datasets, comprising samples from blood, tumor, and non-tumoral mammary tissue (NTMT) from both breast cancer (BRCA) patients and healthy individuals, we identified seven transcriptional signatures corresponding to distinct TAM subsets, exhibiting unique functional profiles, including heightened interferon responses, scavenging, and matrix remodeling, the latter two being characteristic of tissue repair. Notably, none of these subsets aligned with the M1/M2 classification of macrophage (Mφ) polarization. Interferon-associated genes were predominantly enriched in blood monocytes, whereas tissue-repair-associated-signatures were more abundant in tissue resident Mφ, suggesting that TAMs bearing these signatures resemble monocyte-derived or tissue-resident Mφs, respectively. Importantly, TAM subsets expressing interferon-associated genes were associated with improved survival compared to tissue-repairing TAMs in a BRCA cohort from The Cancer Genome Atlas (TCGA). Additionally, one signature was heightened in peripheral monocytes from BRCA patients compared with healthy individuals, which was experimentally validated in a pilot study of Mexican BRCA patients. We concluded that these signatures are a closer description of TAM heterogeneity in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769e9d0c6f0718581121627c404c58103ae137e7" target='_blank'>
              Single cell analysis identifies inflammatory and tissue remodeling tumor associated macrophages distinct from M1/M2 paradigm.
              </a>
            </td>
          <td>
            Eric López-Huerta, Diana P Portales-Pérez, M. E. Martínez-Leija, Vianney Ortiz-Navarrete, E. Fuentes-Pananá
          </td>
          <td>2025-12-05</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Large B-cell lymphoma (LBCL) is an aggressive hematological malignancy characterized by significant clinical and genetic heterogeneity, resulting in variable prognosis and treatment response.
 It has become increasingly evident that the elements of the lymphoma tumor microenvironment (TME) impact on cancer progression and drug resistance. The TME features may be heterogeneous in LBCL and related to specific molecular subtypes, such as those characterized by the dark-zone (DZ) gene signature. MicroRNAs (miRNAs), small non-coding RNAs that regulate gene expression, play a pivotal and multifarious role in cancer biology by directly influencing tumor cell behavior, regulating immune evasion, and shaping TME.
 In this study, we performed a transcriptome analysis by total RNA sequencing on tumor samples from 39 de novo LBCL patients treated with R-CHOP first-line treatment, 20 refractory/relapsed (R/R) patients, and 19 therapy-responders. Using the Molecular Functional Portraits bioinformatic tool, we classified tumors according to gene expression profiles into prognostically relevant TME subtypes. Notably, R/R patients were enriched for the “immune-desert” (ID) subtype, which is associated with the poorest clinical outcomes.
 Given that the high-risk DZ-signature subtype has been previously characterized by an immunosuppressive TME, we examined the overlap between the gene expression profiles of our ID samples and the DZ signature. Remarkably, 84 out of 104 genes composing the DZ signature showed similar modulation in ID samples, underscoring a strong correlation between the DZ signature and the ID phenotype.
 To investigate the role of miRNAs in shaping TME subtypes, a small-RNA sequencing was performed on samples from the same patient cohort. We found 124 miRNAs differentially expressed between ID patients and those from other TME groups. We observed a significant positive correlation (Spearman's R = 0.35; p=0.00012) between miRNA expression changes linked to TME subtypes (ID type vs other TME types) and those associated with treatment response (R/R vs responding patients), highlighting the connection between miRNA-mediated TME remodeling and clinical outcomes. Of note, four of the identified TME-related miRNAs—miR-130b-3p (upregulated in ID and R/R patients), miR-100-5p, miR-99b-5p, and miR-125a-5p (all downregulated in ID and R/R patients)— were also able to accurately discriminate R/R from responsive patients with an area under the ROC curve >0.67. These miRNAs have previously been implicated in modulating the tumor immune microenvironment across various solid and hematologic malignancies.
 Moreover, an anticorrelated target and pathways analysis shows that genes targeted by the identified TME-related miRNA signature were enriched in pathways critical for TME regulation and tumor immune evasion, such as the Notch and IL6-JAK-STAT3 signaling pathways. Deconvolution analyses are ongoing to identify which TME components differ between tumors with high versus low expression of this four-miRNA signature. This will allow a better understanding of the molecular mechanisms by which these miRNAs influence the tumor microenvironment and treatment response. We will further validate all these data in a larger cohort of LBCL patients as well as in cellular models of resistance to R-CHOP.
 Altogether, our results represent a comprehensive, multi-layered approach to understanding how microRNAs contribute to drug resistance by reshaping the TME in LBCL and may pave the way for novel combined therapeutic strategies, holding significant promise for advancing precision medicine.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd5e42361d62ed7960e9e705935b24b001e66c4e" target='_blank'>
              An immune desert phenotype underlies therapy resistance in LBCL through distinct microenvironment-related miRNA profiles
              </a>
            </td>
          <td>
            Giulia Regazzo, G. Vari, F. Marchesi, F. Palombi, Andrea Sacconi, Giulia Orlandi, M. Novello, E. Papa, Martina Tomassi, Francesco Bertoni, E. Maiolo, A. Mengarelli, S. Hohaus, M. Rizzo
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73602df6625b44848edf1084a701c143743994eb" target='_blank'>
              Tumors Located in the Brain Impair the Frequency and Phenotype of Dendritic Cells in Blood and Tumor
              </a>
            </td>
          <td>
            B. Gardam, T. Gargett, E. Nam, Sidra Khan, Rebecca J Ormsby, Santosh I Poonnoose, Julie M Bracken, Anupama Pasam, S. Lenin, Briony L Gliddon, Melinda N. Tea, C. Shard, Stuart M. Pitson, Guillermo A Gomez, Katherine A. Pillman, Shahneen Sandhu, Michael P Brown, LM Ebert
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Intratumoral genomic heterogeneity (ITGH), the existence of genotypic and phenotypic variation within an individual tumor, is known to be a key mechanism in treatment resistance. Deviating gene Expression Profiling Tumor Heterogeneity 2 (DEPTH2) algorithm was developed to estimate ITGH using solely RNA expression data unlike the others that require both DNA- and RNA-expression data. A total of 7508 breast cancer patients from multiple independent cohorts were analyzed using DEPTH2. High DEPTH2 score patients were associated with worse overall survival consistently across all subtypes in SCAN-B, but not in TCGA and METABRIC cohort. Higher scores were also linked to increased cell proliferation, as evidenced by elevated Nottingham histological grades and Ki67 gene expression, as well as enrichment of the cell proliferation-related gene sets, and immune cell infiltrations. DEPTH2 score was significantly higher in triple negative breast cancer among the subtypes but did not reflect with lymph node and distal metastasis. DEPTH2 scores decreased in two but showed no change in another two cohorts after neoadjuvant chemotherapy (NAC). This score was not associated with pathological complete response after NAC in any subtypes across 3 cohorts. DEPTH2 score may not capture the entire biological aspects of ITGH in breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/674d3dd6eefd604ca7935c900aea2b2b00942a09" target='_blank'>
              DEPTH2 score was associated with cell proliferation and immune cell infiltrations but not with systemic treatment response in breast cancer
              </a>
            </td>
          <td>
            Kohei Chida, Rongrong Wu, A. M. Roy, Takashi Ishikawa, Kenichi Hakamada, K. Takabe
          </td>
          <td>2025-11-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/319c8e142d34b2dd372b0b3c7e925d3decc67da5" target='_blank'>
              An Immuno-Linguistic Transformer for Multi-Scale Modeling of T-Cell Spatiotemporal Dynamics
              </a>
            </td>
          <td>
            Linyu Tan, Jiarong Xun
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT This study contributes to the characterization of human macrophages in normal and pathological conditions such as cancer. We characterized a macrophage population expressing membrane-associated IL-18 (mIL-18) that shows peculiar proteomic, phenotypic, ultrastructural, and functional properties. mIL-18+ macrophages exhibit increased levels of key proteins involved in pathogen recognition, activation, migration, and endocytosis. They also display specialized functions in vesicle and actin filament transport and lipid metabolism, and have typical mitochondrial traits. Importantly, mIL-18+ cells dominate the peritoneal fluid of adult cancer patients and are present in the bone marrow of children with neuroblastoma. They express high levels of TREM2 but display heterogeneous FOLR2 expression, distinguishing distinct cell subsets with possibly different functions. Accordingly, in primary neuroblastomas, transcriptional signatures associated with mIL-18 expression show different prognostic values. Our data show that mIL-18+ macrophages, which are predominant across the tumor microenvironment, exhibit previously undetected heterogeneity, potentially impacting tumor progression in a variable manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c8c541abb85c0a0f0a0ff2fa75573a315a92f36" target='_blank'>
              Membrane IL-18 identifies a human macrophage subset with distinct proteomic and functional traits
              </a>
            </td>
          <td>
            Chiara Vitale, Andrea Petretto, Katia Cortese, Sonia Carta, A. Dondero, Chiara Lavarello, Davide Cangelosi, Martina Morini, F. Bellora, Pietro Arnaldi, Fabrizio Loiacono, Santina Bruzzone, Francesco Piacente, Silvia Bruno, Martina Serra, Annamaria Pessino, S. Mammoliti, Alberto Garaventa, Massimo Conte, Massimo Locati, G. Norata, Marco Colonna, Eric Vivier, Cristina Bottino, R. Castriconi
          </td>
          <td>2025-11-05</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related mortality, representing a highly coordinated, multistep process in which malignant cells gain the ability to invade, survive in the circulation, and establish secondary tumors at distant sites. While genetic mutations initiate oncogenesis, accumulating evidence shows that epigenetic and epitranscriptomic regulators, encompassing DNA methylation, RNA modifications, and noncoding RNAs (ncRNAs), reshape metastatic phenotypes. This review integrates current insights into these mechanisms and their crosstalk, with a primary focus on their methylation modification. Given their plasticity and potential reversibility, these regulators are attractive targets for therapeutic intervention. Defining the dynamic interplay between DNA and RNA modifications and ncRNAs provides a coherent framework for controlling metastasis and guides the development of precision epigenetic strategies and biomarkers. Future research that integrates multi-omics approaches and spatial transcriptomics will be essential for revealing the epigenetic and epitranscriptomic layers of the metastatic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32e97223f0c41298eec4d68679c66b5c3087095" target='_blank'>
              Rewriting the Fate of Cancer: Epigenetic and Epitranscriptomic Regulators in the Metastatic Cascade
              </a>
            </td>
          <td>
            Toshifumi Hara, M. Subramanian
          </td>
          <td>2025-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 (G3) MBs have poor overall survival at <50%, higher frequency of metastasis, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling are frequent in G3MB. Remarkably, some tumors have no reported mutations, suggesting roles for chromatin-driven mechanisms in this disease. We have generated novel humanized models for G3MB by transducing neuroepithelial stem cells (NESC) derived from human induced pluripotent stem cells with TGFβ effectors alone and/or in combination with MYC, prioritizing combinations observed in patients. We determined that both MYC and TGFβ effectors drove tumor formation in vivo with the combination driving more aggressive tumors and higher frequency of metastasis. Importantly, NESC-derived tumors clustered with human G3MB, indicating our models recapitulated human disease. To decipher mechanisms of resistance, we integrated CUT&RUN for MYC genomic localization and histone PTMs with RNA-sequencing and discovered MYC-bound neural developmental genes repressed in MYC and TGFβR1-driven lines. These gene signatures were associated with the Polycomb Repressive Complex (PRC) and demarcated with H3K27me3; the histone mark directly regulated by PRC. Further interrogation identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of these repressed neural genes. This is further reinforced by the observation of higher H3K27me3 levels concurrent with increased JARID1B expression in human G3MB patient samples, compared to other subtypes. We postulate that deregulation of MYC and context-specific targeting to the chromatin template culminates in transcriptional alterations that promote aggressive G3MB. These studies support a critical role of MYC in repressing target genes to drive tumor formation and JARID1B as a new target for therapy. Collectively, we present a critical framework for understanding mechanisms of resistance and the chromatin landscape of MYC- and TGFβ-driven G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c126def35a2e55a28669f74c51bd28ad931871" target='_blank'>
              EPCO-47. DISSECTING MYC-DRIVEN EPIGENETIC MECHANISMS THAT PROMOTE GROUP 3 MEDULLOBLASTOMA UTILIZING HUMANIZED IPSC MODELS
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, W. Ismail, Linyu Wang, M. Beytagh, Ronald Phua, Liam D. Hendrikse, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael Taylor, William A. Weiss
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c362676e5f313e967da9a02b557ed5fbcda507b0" target='_blank'>
              Single-cell and spatial transcriptome-based metabolism-immunity interaction network and therapeutic target discovery of matrine in cervical cancer.
              </a>
            </td>
          <td>
            Hong Zou, Wang Xiong, Yumei He
          </td>
          <td>2025-11-24</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b332eb177c3231afc269f28608f60924be826cc" target='_blank'>
              Pan-cancer analysis of RNA expression signatures associated with cancer tissue architecture.
              </a>
            </td>
          <td>
            Megumi Mogi, Katsushige Kawase, Hiroyoshi Y. Tanaka, Suguru Miyata, Satoko Takemoto, Hideki Ikeda, Eri Katayama, Yuki Nakamura, Ryuta Kojima, Takao Morinaga, Shihori Tanabe, S. Yonekura, M. Kano, Toyoyuki Hanazawa, Masahito Kawazu
          </td>
          <td>2025-10-17</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Aneuploidy is a hallmark of cancer and particularly prevalent in glioblastoma (GBM), where it is frequently observed alongside early tumorigenic mutations. Inferred tumor lineage trajectories from patient derived samples suggest that aneuploidy may act as a genetic driver of gliomagenesis; however, its role in initiating GBM remains poorly understood and has never been experimentally validated. To address this, we developed a novel platform that integrates engineered chromosomal aneuploidy with foundational genetic alterations using human cerebral organoids. Leveraging karyotype CRISPR-engineered aneuploidy technology (KaryoCreate), we introduced chromosome 7 gains and chromosome 10 losses—hallmark aneuploidies specific to GBM—into human embryonic stem cells (hESCs) that were differentiated into cerebral organoids alongside karyotype normal hESCs. This resulted in tumor-like clonal expansion of engineered populations validated via fluorescent lineage tracing, inferred copy number variation (CNV) events from single-cell transcriptomics, and fluorescent in situ hybridization (FISH) of Chr 7 and Chr 10 centromeric probes. To investigate alternate cell-of-origin candidates, we introduced doxycycline-inducible aneuploidy machinery into a hESC safe harbor locus and later induced Chr 7 gains and Chr 10 losses throughout organoid neurodevelopment. We further incorporated C228T TERT promoter mutations and CDKN2A deletions to recapitulate genetic alterations commonly found in patients. Over time, immunofluorescent staining and integrated single cell transcriptomic analyses show these models span the diverse transcriptomic cell states characteristic of GBM. This modular, temporally controlled platform enables systematic dissection of how early genomic instability and driver mutations converge to shape cellular fitness, lineage dynamics, and tumor-initiation in a developing human brain context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35eb8b0304fb71fa2998d4110eb943846e6b6fc" target='_blank'>
              TMOD-44. Modeling gliomagenesis via engineered chromosomal aneuploidy and foundational genetic events throughout neurodevelopment of cerebral organoids
              </a>
            </td>
          <td>
            E. Blandford, Aditya Mohan, Eliese M. Moelker, Bhairavy J Puviindran, Ariel Gonzalez, Andrew Hardigan, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181bda14a56f6f64f5ac4fffa1ec57650a41f5a9" target='_blank'>
              Cooperative and antagonistic interactions between sub-clones favour the co-existence of multiple resistance mechanisms in melanoma
              </a>
            </td>
          <td>
            Karin Schlegelmilch, Viola Hollek, S. Hooper, Giovanni Giangreco, Sasha Bailey, Sarah C. Macfarlane, Alexandrine Carminati, Amy Bowes, S. Strohbuecker, Xiao Fu, E. Sahai
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="BACKGROUND: The spatial organization of immune cells within tumors is an important area of investigation in studies of interactions between tumor cells and the tumor microenvironment. This is particularly relevant because some immune cells function through direct contact with their target cells, whereas others communicate over a distance via paracrine signaling involving cytokines. Thus, the topography of tumor cells and microenvironmental cells may determine the possibility and nature of intercellular interactions and thereby influence the functional effects of immune cells. AIM: This study aimed to compare the spatial transcriptomic profiles of tumor and stromal regions enriched in macrophages in triple-negative breast cancer. METHODS: Eight patients with triple-negative breast cancer were included. Spatial transcriptomic analysis was performed on formalin-fixed, paraffin-embedded tissue sections using high-throughput RNA sequencing with the 10x Visium platform. The annotated spots enriched in intraepithelial and stromal macrophages were used for downstream bioinformatic analysis. RESULTS: A total of 437 differentially expressed genes were identified between the two groups of spots containing macrophages with distinct spatial localization. Spots with intraepithelial macrophages were characterized by activation of processes related to cytokine and chemokine signaling, regulation of regulatory T-cell differentiation, organization of cell – cell contacts, wound healing, and inhibition of viral activity. Spots enriched in stromal macrophages demonstrated activation of biological processes associated with the regulation of angiogenesis, cell migration and recruitment, cell adhesion, and stromal remodeling. CONCLUSION: Macrophage topography within primary tumors of triple-negative breast cancer is associated with their functional characteristics. These fundamental findings may be useful for developing prognostic criteria and therapeutic approaches aimed at modulating the tumor microenvironment to improve long-term outcomes in patients with triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1996d432009ab729b97a00ec19d18afda8c70949" target='_blank'>
              Spatial Characterization of Macrophage-Enriched Tumor Regions in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            I. Patskan, A. Kalinchuk, Elisaveta Tsarenkova, Evgeniia Grigoryeva, I. Larionova, Natalia Popova, L. Tashireva
          </td>
          <td>2025-12-02</td>
          <td>Genes &amp; Cells</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Colon adenocarcinoma (COAD) is influenced by the tumor microenvironment, including immune cell dynamics, particularly M2 macrophages. Understanding these dynamics can offer insights into tumor progression and potential therapies. We analyzed TCGA COAD data to quantify immune infiltration using TIMER, CIBERSORT, QUANTISEQ, MCPcounter, xCell, and EPIC methods. The ESTIMATE algorithm provided stromal and immune scores. WGCNA identified key gene modules correlated with immune infiltration. Prognostic significance of these genes was assessed using univariate Cox regression, Kaplan-Meier analysis, and nomogram construction. Single-cell RNA sequencing data from TISCH2 was used to explore gene expression correlations across cell types. We also examined correlations of LDOC1 with TMB, MSI, NEO, and tumor purity, and predicted immunotherapy responses using TIDE and Submap analyses. Drug sensitivity was analyzed using GDSC data. Significant variations in M2 macrophage levels were found between tumor and normal tissues. WGCNA identified three key gene modules. Univariate Cox regression highlighted DMPK and LDOC1 as significant prognostic genes. Pan-cancer analysis showed LDOC1 as a crucial prognostic marker across various cancers, with strong correlations to multiple pathways. Single-cell analysis revealed distinct expression patterns of DMPK and LDOC1. In COAD, LDOC1 expression correlated significantly with immune checkpoints and tumor microenvironment scores. TIDE and Submap analyses suggested better immunotherapy response in low LDOC1 groups. Drug sensitivity analysis identified eight potential therapeutic agents. This study underscores the importance of M2 macrophages and LDOC1 in COAD, highlighting their potential as prognostic markers and therapeutic targets, paving the way for personalized treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11e05862de6b52dfc8fd8583cebee00c133e3ea2" target='_blank'>
              Immune cell dynamics and prognostic role of M2 macrophages and LDOC1 in colon adenocarcinoma
              </a>
            </td>
          <td>
            Laibin Luo, Zhimin Xu, Hongwu Lin, Rong Huang
          </td>
          <td>2025-10-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="F-box proteins have been well-established as important regulators of cancer development after decades of exploration. As the understanding of cancer biology continues to deepen, novel biological characteristics of cancer cells have been incorporated, and the perspective has expanded from cancer cells themselves to a more complex tumor microenvironment (TME). In this context, F-box proteins form an extensive and multi-layered regulatory network to fine-tune diverse biological events, not only within cancer cells, but also in the TME. Therefore, in this review, we summarize the functional and mechanistic paradigms of F-box proteins in both cancer cells and the TME, and underscore their adaptability and wide coverage on the basis of a sophisticated regulatory hierarchy, presenting the clinical prospects of F-box proteins as potential targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/950db0086d31d060f586b6c69912e1bbecc0d101" target='_blank'>
              F-box proteins in cancer: from cancer cells to the tumor microenvironment
              </a>
            </td>
          <td>
            Jingyi Cheng, Ousheng Liu, Xin Bin, Zhangui Tang
          </td>
          <td>2025-10-13</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="

 CD47 is a “don’t eat me” signal that is overexpressed in tumors to evade phagocytosis by tumor associated macrophages (TAM). Investigational agents targeting CD47, such as magrolimab, aim to induce phagocytosis of tumor cells by TAMs. Previously, two key TAM subsets have been identified: C1QC TAMs, which display pro-phagocytic activity, and SPP1 TAMs that express pro-angiogenic markers. We characterize CD47 expression and its relationship with tumor macrophages in solid tumor samples.



 Resectable tumors from head and neck squamous cell carcinoma (n=36) (HNSCC), breast cancer (n=37) (BC), and colorectal cancer (n=36) (CRC) were evaluated for CD47 expression by immunohistochemistry (IHC), two multiplex immunofluorescence panels were used to characterize TAM markers and T cell markers. RNA sequencing was also performed.



 CD47 protein expression was higher on tumor cells compared to stromal cells across tumor indications tested. Although CRC had the lowest prevalence for CD47 expression in primary tumors, we observed a marked increase in CD47 expression in CRC liver metastases. We developed an SPP1 TAM gene signature and validated a C1QC TAM gene signature to estimate TAM abundances from bulk RNA-Seq data. In the TAM mIF analysis, HNSCC had the highest macrophage density of the indications tested. We observed a positive correlation between a higher C1QC: SPP1 TAM ratio and macrophage phenotype and tumor T cell density. C1QC macrophage signatures correlate with tumor CD47 protein expression in BC and HNSCC samples, suggesting interplay between them.



 We characterized CD47 expression across key solid tumor indications being evaluated clinically using anti-CD47 blockade agents: HNSCC, breast cancer and CRC. Using a CD47 IHC assay, we identified HNSCC as an indication with the highest CD47 expression. In addition, we quantified tumor macrophages using multiplex immunofluorescence (mIF) and determined that HNSCC had the highest density of TAMs. Compared to relatively low CD47 expression in primary CRC tumors, CRC liver metastases had very high CD47 expression. Quantification of TAM signatures and CD47 expression represent key biomarkers to monitor in patient samples during exploration of CD47-blockade agents in the clinic.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31725c03ca6311073138a838325de55dcf4fe468" target='_blank'>
              CD47 expression in solid tumors correlates with phagocytic tumor-associated macrophage gene signature
              </a>
            </td>
          <td>
            Nicholas van Buuren, Mengshu Xu, Yi Zhang, Paola Correa, Shiva Zaboli, Azadeh Madjidi, Christina Moon, Szu-Wen Liu, Ruidong Li, Kai Sun, Shahed Iqbal, Abhishek Aggarwal, Min Wang, Li Li, Jared M Odegard, Kelli Boyd
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3257e07d3eed982df5f02daf4c23bc618d1ee9c3" target='_blank'>
              A quantitative approach to discover nonlinear signaling linking combinatorial environments with cellular responses
              </a>
            </td>
          <td>
            Dmitry Kuchenov, Frederik Ziebell, Florian Salopiata, Alix Thomas, Mevlut Citir, Ursula Klingmueller, Wolfgang Huber, Carsten Schultz
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95a82ca455a15911dad7d392761fde3e786fd270" target='_blank'>
              Genetic dissection of tissue composition in genetically diverse mouse populations
              </a>
            </td>
          <td>
            G. R. Keele, M. Moor
          </td>
          <td>2025-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 The advent of cancer immunotherapy, particularly immune checkpoint inhibitors, has revolutionised the treatment landscape for many malignancies. However, a significant proportion of patients exhibit primary, adaptive, or acquired resistance to these therapies, limiting their broad clinical benefit. Tumour microenvironment (TME), a complex and dynamic ecosystem of cancer cells, immune cells, stromal cells and extracellular matrix, has emerged as a critical determinant of immunotherapy response and a major driver of resistance.



 This review provides a comprehensive overview of the intricate mechanisms by which the TME fosters an immunosuppressive milieu, thereby impeding effective anti‐tumour immunity. We delineate the key cellular and molecular components of the TME that contribute to immunotherapy resistance, including tumour‐associated macrophages, myeloid‐derived suppressor cells, regulatory T cells, and cancer‐associated fibroblasts. We then discuss the diverse and innovative strategies being developed to reprogram the TME and overcome these resistance mechanisms. These emerging approaches include targeting immunosuppressive cell populations, modulating tumour metabolism, epigenetic reprogramming, normalising the tumour vasculature and employing engineered cell therapies such as chimeric antigen receptor T cells and oncolytic viruses. The potential of combination therapies to synergistically dismantle the immunosuppressive TME is also explored.



 Reprogramming the TME represents a paradigm shift in cancer treatment, moving beyond direct tumour cell targeting to modulate the entire tumour ecosystem. While significant challenges remain, including TME heterogeneity and the need for predictive biomarkers, the strategies discussed herein hold immense promise for overcoming immunotherapy resistance and improving patient outcomes. A deeper understanding of the complex interplay between the tumour and its microenvironment will be crucial for the development of next‐generation, personalised immunotherapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab5e861ba23ee597d8c992f1eb0a4c50564cbedf" target='_blank'>
              Reprogramming the tumour microenvironment: Emerging strategies to overcome immunotherapy resistance
              </a>
            </td>
          <td>
            Mohammed Elmujtba Adam Essa, Shahd Elbadri, Abdelkareem A. Ahmed, Hamid Noori
          </td>
          <td>2025-11-09</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7be6c7d53c98a122e4526c54f30b287f477e9f3a" target='_blank'>
              Interrogation of the cellular hierarchies reveals neoplastic evolution and therapeutic vulnerability in craniopharyngioma.
              </a>
            </td>
          <td>
            Bin Zhang, Jing Zhang, Zhidan Li, Hui Sheng, Haotai Li, Yu Lu, Xixi Liu, Zhongwen Xu, Yunhang Huang, Chenyan Zhu, Yi Wen, Xuelian He, Liguo Zhang
          </td>
          <td>2025-10-24</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 The standard of care for glioblastoma (GBM) has not changed since 2005, which is due, in part, to its hallmark heterogeneity. Oncogene amplifications are key mediators of GBM cellular states. Extrachromosomal DNA (ecDNA), large circular DNA elements harboring oncogenes, are a class of amplifications with oncogenic properties due to their structure. Without centromeres, ecDNA segregate randomly into daughter cells, promoting genetic heterogeneity, exceptionally high copy number, and aberrantly high transcription. ecDNA are present in over 50% of GBM tumors, more than any other cancer type, and the most prevalent genes found on ecDNA are well-characterized cellular state drivers like EGFR. ecDNA studies have traditionally relied on bulk sequencing or low-throughput, low-resolution imaging. However, given the intratumoral and possible intermolecular heterogeneity of ecDNA, mechanistic studies of ecDNA would ideally visualize its higher order chromatin structure, multi-scale organization, and RNA expression at a single-molecule resolution in a high-throughput manner. Several multi-modal Multiplex Error-Robust Fluorescence in-situ Hybridization (mm-MERFISH) strategies were leveraged to investigate the multi-scale organization of ecEGFR: from their 3D conformation to their subcellular localization to their distribution across the higher order tumor architecture. In an orthotopic mouse model, there were distinct differences in ecDNA copy number, expression, and localization. Several cell populations had decreased ecEGFR copy number and expression and were localized to the tumor rim. One population diffusively infiltrated cortical neurons, expressed a Proneural transcriptional program, contained low ecEGFR copy number, and expressed the lowest levels of EGFR in the entire tumor. This spatial organization was also observed in multi-region primary GBM samples. Testing whether the microenvironment can impact ecDNA phenotype, GBM39 cells grown with neurons had dramatic morphological changes and decreased ecDNA copy number. These results suggest that ecEGFR dosage and expression are spatially organized and that the unique properties of ecDNA contribute to transcriptional heterogeneity in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a69ae9f36b8ddbe254640b32b7e2cdf7c9422b43" target='_blank'>
              EPCO-54. DEFINING THE MULTI-SCALE ORGANIZATION AND REGULATION OF EGFR EXTRACHROMOSOMAL DNA IN GLIOBLASTOMA WITH MULTI-MODAL MERFISH
              </a>
            </td>
          <td>
            Brett Taylor, Weixiu Dong, Zhaoning Wang, Yohei Miyake, Zane A. Gibbs, Daisuke Kawauchi, Yang Xie, Lei Chang, Kseniya Malukhina, Sahana Kashyap, Tula Keal, Brandon Jones, Bing Ren, Frank B Furnari, Bogdan Bintu
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Traditional population sequencing methods struggle to directly correlate gene perturbations with transcriptome changes at the single-cell level, due to limitations such as long cycle times and significant batch effects. Single-cell sequencing (SCS) can reveal the diversity and dynamics of cell populations, while CRISPR gene editing and screening technologies enable large-scale functional gene discovery. The combination of these two approaches has led to the development of ScCRISPR-seq. This article reviews the principles of CRISPR screening and SCS technologies, introduces the development of ScCRISPR-seq, and, drawing on research on lung cancer and intrahepatic cholangiocarcinoma (ICC), summarizes its key applications in cancer research, including deciphering tumor heterogeneity, elucidating mechanisms of drug resistance, and modulating the immune microenvironment. This article further explores challenges with this technology, including experimental workflow, data sparsity, and technical noise. Solutions are proposed, including data completion using AI algorithms, eliminating amplification bias using UMI tags, and optimizing gene-phenotype associations through network modeling. However, clinical translation is limited by unclear mechanisms of action and high barriers to entry in drug development. Future research may consider combining ScCRISPR-seq with network pharmacology and molecular docking techniques to improve the efficiency of new cancer drug development and advance precision immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1262cf91f44f77bd99618ed1bc278657dd28dee" target='_blank'>
              The Application and Prospect of Single-cell CRISPR Screening Technology in Tumor Research
              </a>
            </td>
          <td>
            Zineng Zhou
          </td>
          <td>2025-10-23</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Malignant glioblastoma exhibits cellular senescence characterized by changing tumor microenvironment. Solute carrier family 6 member 6 (SLC6A6), a multichannel transmembrane protein, plays a crucial role in in regulating cell proliferation, apoptosis, differentiation, and cellular osmolality. However, the molecular mechanism of SLC6A6 in the pathogenesis of glioblastoma remains unknown. We explored the architecture complexity of GBM through spatial transcriptomics and we found that SLC6A6 emerged as a particularly important gene of interest. We examined expression profiling of SLC6A6 in tumor samples derived from 50 patients with glioblastoma, and evaluated associations between SLC6A6 expression levels and survival outcome using Kaplan–Meier survival and Cox regression analyses. We applied short hairpin RNA (shRNA) and overexpression (OE) vectors to construct SLC6A6-knockdown (KD) and -OE glioblastoma cell lines, respectively. We established an orthotopic mouse model of glioblastoma. Clinically, patients with high expression of SLC6A6 had a worse prognosis (reduced overall survival [OS]). Downregulation of SLC6A6 protein inhibited malignant phenotypes of glioblastoma cells in vitro. SLC6A6 affected tumor senescence by directly binding to CSK with its N-terminal cytoplasmic domain, thereby enhancing AKT phosphorylation. Furthermore, SLC6A6 KD inhibited tumor growth and extended survival in a xenograft mouse model. Collectively, SLC6A6 can promote malignant progression and inhibit cellular senescence of glioblastoma by affecting the CSK/AKT/FOXO1 signaling pathway. Therefore, SLC6A6 might be a valuable target for therapy of glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af842a193cc393f298396ce6e2048f68fd482c7d" target='_blank'>
              PATH-17. Spatial Transcriptomics Uncovers the Functional Role of SLC6A6 in Cellular Senescence and Its Pathological Significance in Glioblastoma
              </a>
            </td>
          <td>
            BO Wang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Background: Despite recent advancements in treatment for multiple myeloma (MM), most patients eventually relapse due to the emergence of resistant tumor subclones. Bulk DNA sequencing has identified recurrent resistance-associated mutations and can infer subclonal architecture, but it lacks the resolution to determine whether multiple mutations co-exist within the same malignant cell or arise in separate subclones. This distinction is critical, as multiple somatic variants within individual cells may confer resistance that does not arise from any single alteration alone. We hypothesized that targeted single-cell DNA sequencing of myeloma tumor cells would reveal the co-occurrence of mutations within individual cells that are missed by conventional bulk sequencing approaches.
 Methods: Bone marrow aspirates were collected from 46 patients, including 9 with newly diagnosed and 37 with relapsed or refractory MM. Two patients were sampled at multiple time points, yielding a total of 48 samples. CD138⁺ plasma cells were enriched using magnetic-activated cell sorting and subjected to single-cell encapsulation for simultaneous genomic and proteomic profiling. Libraries were prepared using the Mission Bio Tapestri Single-Cell Multiple Myeloma Multiomics Assay, which includes amplicons targeting single-nucleotide variants, insertions and deletions, and focal copy number aberrations commonly altered in the disease. In parallel, immunoglobulin heavy- and light-chain V(D)J clonotypes were identified, and cells were labeled with a 20-plex oligo-conjugated antibody panel to quantify surface protein markers associated with cell lineage and immunotherapy targets. Tumor cells were defined as those harboring the dominant immunoglobulin V(D)J sequence within each sample. Reference cells were identified as VDJ-negative and expressing high levels of non–plasma cell markers (CD3, CD19, or CD33). Somatic mutations were identified by comparing variant allele frequencies between tumor and reference cell populations.
 Results: Across all 48 samples, 146,946 cells were sequenced, with 68,052 of those being tumor cells. Hallmark MM copy number alterations, such as 17p deletion, 1q amplification, and 13q deletion, were identified using the copy number assay. The most frequent mutations identified were in KRAS, TP53, TENT5C, and CRBN. The two patients with CRBN mutations underwent 27 and 34 prior cycles of immunomodulatory (IMiD) therapy, respectively, compared to a median of 8.0 cycles in patients without CRBN mutations, raising the possibility of treatment-driven selection. In another case, we identified three mutually exclusive KRAS mutations (Q61H, G12V, and G12D) in distinct subclones within a single patient, consistent with convergent evolution toward activation of the MAPK pathway. We also resolved a true “double hit” in another patient, characterized by 58% of tumor cells carrying a deletion of one allele and a somatic mutation in the other, consistent with biallelic inactivation. Finally, we observed CDKN2A more frequently co-mutated with TP53, PTPRT, and TENT5C than as isolated events, suggesting multi-hit inactivation of key cell-cycle checkpoints and DNA-damage response pathways that may drive selective growth advantage or therapeutic resistance. CDKN2A and TP53 are canonical tumor suppressors that regulate G1/S transition and apoptosis, respectively, while PTPRT encodes a phosphatase that negatively regulates STAT3 signaling, and TENT5C is a tumor suppressor involved in mRNA stability and plasma cell homeostasis. These recurrent co-mutations may act synergistically by simultaneously disrupting transcriptional and signaling regulation, thereby conferring a selective tumor cell growth advantage under therapeutic pressure.
 Conclusions: Targeted single-cell DNA sequencing reveals co-occurring mutations within individual myeloma cells that are obscured by bulk approaches, providing critical insight into the cooperative and competitive genomic events driving disease progression and resistance. These findings support a model in which myeloma subclones accumulate synergistic alterations within the same cell. Work is ongoing to provide functional validation of key mutation pairs and their integration with transcriptomic data to refine the mechanistic understanding and identify candidates for combination therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98368199e493cbd0fcaadb2df0e1e6f8e200c1a5" target='_blank'>
              Single-cell DNA sequencing uncovers synergistic co-mutations in multiple myeloma
              </a>
            </td>
          <td>
            Ava Amidei, B. Diamond, Elizabeth Lyubchenko, Stephanie Fernandes, Baharak G. Fard, L. Nousheen, Marc Arribas-Layton, Adam Sciambi, D. Kazandjian, James Hoffman, Abhishek Pandey, Marcella Kaddoura, Brian Walker, O. Landgren, D. Coffey
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>32</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [11, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>