<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-30 09:32:06 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c056a7bc97d38004639e645962dc9ff9c3cfad" target='_blank'>
              Modeling cellular influence delineates functionally relevant cellular neighborhoods in primary and metastatic pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yeonju Cho, , Sarah M. Shin, Alexei Hernandez, Xuan Yuan, Jowaly Schneider, Jody E. Hooper, Laura D. Wood, Elizabeth M. Jaffee, Atul Deshpande, W. Ho
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Over the past years, cancer research has transitioned from a ‘cancer cell-centered’ focus to a more integrative view of tumors as dynamic ecosystems. This paradigm shift emphasizes the tumor microenvironment (TME) as a complex network of interacting cellular and acellular components, where tumor cells orchestrate a supportive environment that facilitates progression, metastasis, and immune evasion. Understanding the spatial organization of these components within the TME is crucial, as the positioning and interactions between cancerous and non-cancerous cells significantly influence tumor behavior and therapy response. Spatial proteomics has emerged as a powerful tool for TME analysis, enabling the detection and quantification of proteins within intact tissue architecture at subcellular resolution. This approach provides insights into cellular interactions, signaling pathways, and functional states, facilitating the discovery of novel biomarkers and therapeutic targets linked to specific tissue regions and cellular contexts. Translating spatial proteomics into clinical practice requires overcoming challenges related to technology refinement, standardization of workflows, and adaptation to routine pathology settings. Melanoma is an aggressive, highly immunogenic malignancy with variable response rates to existing immunotherapies. Given that over half of patients treated with immune checkpoint inhibitors (ICIs) fail to respond or experience disease progression, the identification of novel biomarkers and therapeutic targets to enhance current therapies is urgently required. Spatial imaging technologies are increasingly being utilized to dissect the complex interplay between stroma, melanoma, and immune cell types within the TME to address this need. This review examines key spatial proteomics methods, their applications in melanoma biology, and associated image analysis pipelines. We highlight the current limitations, and future directions, emphasizing the potential for clinical translation to guide personalized treatment strategies, inform prognosis, and predict therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1eb1624eaa05c3beff7c464020ea5f0d816374c" target='_blank'>
              Spatial proteomics of the tumor microenvironment in melanoma: current insights and future directions
              </a>
            </td>
          <td>
            Chiara Bungaro, Michele Guida, Benedetta Apollonio
          </td>
          <td>2025-05-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bc89e4de14d8f34f2814e7cf735899a3c934813" target='_blank'>
              Spatial determinants of tumor cell dedifferentiation and plasticity in primary cutaneous melanoma
              </a>
            </td>
          <td>
            Tuulia Vallius, Yingxiao Shi, Edward Novikov, Shishir M Pant, R. Pelletier, Yu-An Chen, Juliann B. Tefft, Ajit Nirmal Johnson, Z. Maliga, Guihong Wan, George F. Murphy, S. Santagata, Yevgeniy R. Semenov, David Liu, C. G. Lian, P. Sorger
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a7baab96b7e0361b2e10254d2ab098fe969a9" target='_blank'>
              Tumor Cells Enriched for Interferon and Inflammatory Programs Pre-Exist in High Grade Serous Ovarian Cancer and are Proportionately Significantly Increased Post Chemotherapy
              </a>
            </td>
          <td>
            B. Winterhoff, Nomeda Girnius, Muyi Liu, Jaeyoung Kang, Mihir Shetty, M. A. Martinez-Gakidis, Weiyi Xu, Mayura Thomas, Linna Lahmadi, Aylin Z. Henstridge, Pilar Baldominos, Shobhana Talukdaer, Z. Chang, Jordan Mattson, Morgan Gruner, Sally A. Mullany, Peter Argenta, Gottfried E. Konecny, Dennis J. Slamon, Ronny Drapkin, Andrew C. Nelson, Timothy K. Starr, Joan S Brugge
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="A comprehensive understanding of cellular behavior and response to the tumor microenvironment (TME) in colorectal cancer (CRC) remains elusive. Here, we introduce the high-definition Visium spatial transcriptomic technology (Visium HD) and investigate formalin-fixed paraffin-embedded human CRC samples (n = 5). We demonstrate the high sensitivity, single-cell-scale resolution and spatial accuracy of Visium HD, generating a highly refined whole-transcriptome spatial profile of CRC samples. We identify transcriptomically distinct macrophage subpopulations in different spatial niches with potential pro-tumor and anti-tumor functions via interactions with tumor and T cells. In situ gene expression analysis validates our findings and localizes a clonally expanded T cell population close to macrophages with anti-tumor features. Our study demonstrates the power of high-resolution spatial technologies to understand cellular interactions in the TME and paves the way for larger studies that will unravel mechanisms and biomarkers of CRC biology, improving diagnosis and disease management strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f42a49dad6477b432ac875de1bc87a28316b76b" target='_blank'>
              High-definition spatial transcriptomic profiling of immune cell populations in colorectal cancer
              </a>
            </td>
          <td>
            Michelli F Oliveira, J. P. Romero, Meii Chung, Stephen R. Williams, Andrew D. Gottscho, Anushka Gupta, Susan E. Pilipauskas, Seayar Mohabbat, Nandhini Raman, David Sukovich, David Patterson, Sarah E B Taylor
          </td>
          <td>2025-06-05</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Identification of the consensus molecular subtypes (CMS) opened significant potential for understanding the tumor biology and intertumoral heterogeneity of colorectal cancer (CRC). However, molecular subtyping in CRC traditionally relies on bulk transcriptomics, therefore, lacks spatial and single-cell level aspect. Methods We constructed tissue microarrays using tumor cores from 222 CRC patients. Arrays were stained and imaged using 54 cell identity and cancer hallmark markers, delivering spatially resolved protein profiles of >2 million cells. RNA sequencing data and CMS classification were also available for these patients. After segmentation of cancer, stromal and immune cells, we investigated intratumoral heterogeneity within CMS subtypes using spatially resolved single-cell protein profiling (>2 million cells). We compared cell types, their spatial organization and their expression of cancer hallmark-related proteins in CMS 1-4 subtypes. Results We revealed tissue atlases illustrating the cell types/states, spatial heterogeneity, cellular neighborhoods, cellular network, and single-cell protein profiles of CMS tumors. CMS1 tumors had more CD3+, CD8+, and PD1+ immune cells that were found in the epithelial layer frequently. CMS1 was also associated with higher levels of metabolic reprogramming markers such as upregulated glycolysis. CMS2 showed immune segregation, reactive stroma patterns and higher levels of apoptotic and proliferative signaling proteins. CMS3 exhibited clustered cancer cells with high RIP3 levels, suggesting a pro-inflammatory microenvironment. CMS4 displayed stromal-centric and immune-evasive tumors characterized by decreased HLA-1 levels and regulatory T-cell exclusion from epithelium. Conclusion We present a spatial protein atlas of CRC at single-cell resolution and demonstrate novel aspects of CMS tumour structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd0aeac5c1de86e47bfdfaa33c8809067d3de4e" target='_blank'>
              High-Resolution Spatial Proteomics Characterises Colorectal Cancer Consensus Molecular Subtypes
              </a>
            </td>
          <td>
            Batuhan Kısakol, Anna Matveeva, Sanghee Cho, Mohammadreza Azimi, A. Lindner, M. Salvucci, E. McDonough, Joanna Fay, Tony O’Grady, N. McCawley, John P Burke, Deborah A. McNamara, John Graf, Simon McDade, Daniel B. Longley, Fiona Ginty, J. Prehn
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer cells display highly heterogeneous and plastic states in glioblastoma, an incurable brain tumour. However, how these malignant states arise and whether they follow defined cellular trajectories across tumours is poorly understood. Here, we generated a deep single cell and spatial multi-omic atlas of human glioblastoma that pairs transcriptomic, epigenomic and genomic profiling of 12 tumours across multiple regions. We identify that glioblastoma heterogeneity is driven by spatially-patterned transitions of cancer cells from developmental-like states towards those defined by a glial injury response and hypoxia. This cellular trajectory regionalises tumours into distinct tissue niches and manifests in a molecularly conserved manner across tumours as well as genetically distinct tumour subclones. Moreover, using a new deep learning framework to map cancer cell states jointly with clones in situ, we show that tumour subclones are finely spatially intermixed through glioblastoma tissue niches. Finally, we show that this cancer cell trajectory is intimately linked to myeloid heterogeneity and unfolds across regionalised myeloid signalling environments. Our findings define a stereotyped trajectory of cancer cells in glioblastoma and unify glioblastoma tumour heterogeneity into a tractable cellular and tissue framework.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a25f85ea327b0edecce4b687c0e133b5fa0f6d5" target='_blank'>
              A spatiotemporal cancer cell trajectory underlies glioblastoma heterogeneity
              </a>
            </td>
          <td>
            Grant de Jong, Fani Memi, Tannia Gracia, Olga Lazareva, Oliver Gould, Alexander Aivazidis, Manas Dave, Qianqian Zhang, Melanie Jensen, A. Rifaioglu, Joao D. Barros-Silva, Sabine Eckert, Di Zhou, Yvette Wood, Elizabeth Tuck, Sezgin Er, Henry Marshall, Kenny Roberts, Andrew L. Trinh, Shreya Rai, Tyler Shaw, Agnes Oszlanczi, Hayden Powell, R. Petryszak, Zoi Katsirea, Irfan Mamun, I. Mulas, Annelies Quaegebeur, Mayen Briggs, Stanislaw Makarchuk, Jessica Cox, Jimmy Tsz Hang Lee, Laura Rueda, Manu Saraswat, Harry J C J Bulstrode, Adam Young, Minal Patel, Tarryn Porter, E. Prigmore, Moritz Mall, Julio Saez-Rodriguez, James Briscoe, David H. Rowitch, Richard Mair, Sam Behjati, O. Stegle, O. Bayraktar
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4028f5dcc0e86189184980c13bb56279010a29ab" target='_blank'>
              Clonal lineage tracing and parallel multiomics profiling reveal transcriptional diversification induced by ARID1A deficiency
              </a>
            </td>
          <td>
            Kenichi Miyata, Liying Yang, Yuxiao Yang, Kohei Kumegawa, R. Maruyama
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d556d25908485193c4e548584bab10bfa70daa1" target='_blank'>
              Oncogenic drivers shape the tumor microenvironment in human gliomas
              </a>
            </td>
          <td>
            Bo Zhao, Chae Yun Cho, Lingqun Ye, Tula Keal, Thomas Mitchell, Inés Martín-Barrio, C. A. Faraj, Mesut Unal, Jessica Alsing, Nora M Lawson, Ellie Rahm Kim, Francisco Chavez, Kristy Mendoza Rangel, Sanjay K Singh, Joseph H. McCarthy, D.M. Ashley, W. Yung, Vinay K. Puduvalli, Guy Nir, Jeffrey S. Weinberg, Linghua Wang, Keith L. Ligon, G. Genovese, Peter Van Loo, P. Futreal, Jason T. Huse, Jesse R. Dixon, Frederick F. Lang, Kadir C. Akdemir
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Background Breast cancer is a highly heterogeneous disease where variations of biomarker expression may exist between individual foci of a cancer (intra-tumoral heterogeneity). The extent of variation of biomarker expression in the cancer cells, distribution of cell types in the local tumor microenvironment and their spatial arrangement could impact on diagnosis, treatment planning and subsequent response to treatment. Methods Using quantitative multiplex immunofluorescence (MxIF) imaging, we assessed the level of variations in biomarker expression levels among individual cells, density of cell cluster groups and spatial arrangement of immune subsets from regions sampled from 38 multi-focal breast cancers that were processed using whole-mount histopathology techniques. Molecular profiling was conducted to determine the intrinsic molecular subtype of each analysed region. Results A subset of cancers (34.2%) showed intra-tumoral regions with more than one molecular subtype classification. High levels of intra-tumoral variations in biomarker expression levels were observed in the majority of cancers studied, particularly in Luminal A cancers. HER2 expression quantified with MxIF did not correlate well with HER2 gene expression, nor with clinical HER2 scores. Unsupervised clustering revealed the presence of various cell clusters with unique IHC4 protein co-expression patterns and the composition of these clusters were mostly similar among intra-tumoral regions. MxIF with immune markers and image patch analysis classified immune niche phenotypes and the prevalence of each phenotype in breast cancer subtypes was illustrated. Conclusions Our work illustrates the extent of spatial heterogeneity in biomarker expression and immune phenotypes, and highlights the importance of a comprehensive spatial assessment of the disease for prognosis and treatment planning. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02038-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185f08263c4f5dcd967322d5656a07ab8cc0083f" target='_blank'>
              Intra-tumoral spatial heterogeneity in breast cancer quantified using high-dimensional protein multiplexing and single cell phenotyping
              </a>
            </td>
          <td>
            Alison M. Cheung, Dan Wang, M. Quintayo, Yulia Yerofeyeva, Melanie Spears, John M. S. Bartlett, Lincoln Stein, Jane Bayani, Martin J Yaffe
          </td>
          <td>2025-05-21</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="IDH-wildtype glioblastoma (GBM) is an aggressive brain tumor with poor survival and few therapeutic options. Transcriptionally-defined cell states coexist in GBM and occupy defined regions of the tumor. Evidence indicates that GBM cell states are plastic, but it remains unclear if they are determined by the underlying epigenetic state and/or by microenvironmental factors. Here, we present spatially-resolved epigenomic profiling of human GBM tissues that implicate chromatin structure as a key enabler of cell plasticity. We report two epigenetically-defined and spatially-nested tumor niches. Each niche activates short-range molecular signals to maintain its own state and, surprisingly, long-range signals to reinforce the state of the neighboring niche. The position of a cell along this gradient-like system of opposing signals determines its likelihood to be in one state or the other. Our results reveal an intrinsic system for cell plasticity that is encoded in the chromatin profiles of two adjacent niches that dot the topological architecture of GBM in cartesian space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc7a8f38a5d53281f1873999c12438505ab4d573" target='_blank'>
              Spatial epigenomic niches underlie glioblastoma cell state plasticity
              </a>
            </td>
          <td>
            Sam Kint, S. T. Younes, Shuozhen Bao, Gretchen Long, David Wouters, Erin Stephenson, Xing Lou, Mei Zhong, Di Zhang, Graham Su, Archibald Enninful, Mingyu Yang, Huey-Miin Chen, K. Ellestad, Colleen Anderson, Jennifer Moliterno, John J. Kelly, Jennifer A. Chan, Alejandro Sifrim, Jiangbing Zhou, Ana Nikolic, Rong Fan, Marco Gallo
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b6a64a6d2ad28ecff7ab0f31663d8f3fd11b6a" target='_blank'>
              A spatial atlas of chemoradiation therapy in pancreatic cancer identifies cellular and microenvironmental determinants of persister populations
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, Ariana Acevedo-Diaz, K. Rajapakshe, Galia Jacobson, Daniela Tovar, Jimin Min, Guangsheng Pei, Candise Tat, Ayush Suresh, Ching-Wei Tzeng, Matthew HG Katz, M. Bhutani, Huaming Wang, Robert A. Wolff, C. Haymaker, E. Ludmir, Huocong Huang, Xiongfeng Chen, Liang Li, Albert C Koong, Linghua Wang, N. Navin, Dadi Jiang, Ziyi Li, Anirban Maitra, Eugene J. Koay
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61a2da83ec8f696482ebba50d8d961251109e469" target='_blank'>
              MONTAGE: A Computation Framework to Identify Spatially Resolved Functional Enrichment Gradients in the Tissue Microenvironment via Spatial Communities
              </a>
            </td>
          <td>
            Weiruo Zhang, Zinaida Good, M. Baertsch, Guolan Lu, John W. Hickey, Rachel Hildebrand, Serena Chang, Andrew J. Gentles, John B. Sunwoo, Quynh-Thu Le, Christina S. Kong, Garry P. Nolan, S. Plevritis
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83adb5995a58165fece89d2aea822be3362a5535" target='_blank'>
              Epigenetic Subtypes of High-Grade T1 Bladder Cancer Reveal Intra-Tumor Heterogeneity and Distinct Interactions with Tumor Microenvironment
              </a>
            </td>
          <td>
            J. Bellmunt, Yingtian Xie, N. Juanpere, M. Munoz, Shweta Kukreja, Sonsoles Liria Veiga, Rong Li, Xintao Qiu, Yijia Jiang, Alba Font-Tello, Marie Nunez Duarte, Ilana B. Epstein, Silvia Hernández-Llodrà, Marta Lorenzo, Silvia Menendez, T. Choueiri, Myles Brown, H. Long, Paloma Cejas
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1dc109a3a473a99af4a77b675644d925cf4104" target='_blank'>
              Oncogenic and tumor-suppressive forces converge on a progenitor-orchestrated niche to shape early tumorigenesis
              </a>
            </td>
          <td>
            José Reyes, Isabella S. Del Priore, Andrea C. Chaikovsky, Nikhita Pasnuri, Ahmed M. Elhossiny, Tobias Krause, Andrew Moorman, Catherine Snopkowski, Meril Takizawa, Cassandra Burdziak, Nalin Ratnayeke, Ignas Masillioni, Yu-jui Ho, R. Chaligné, P. Romesser, A. Filliol, T. Nawy, John P. Morris, Zhen Zhao, M. P. di Magliano, Direna Alonso-Curbelo, D. Pe’er, Scott W. Lowe
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Cells are the fundamental structural and functional units of living organisms and the study of these entities has remained a central focus throughout the history of biological sciences. Traditional cell research techniques, including fluorescent protein tagging and microscopy, have provided preliminary insights into the lineage history and clonal relationships between progenitor and descendant cells. However, these techniques exhibit inherent limitations in tracking the full developmental trajectory of cells and elucidating their heterogeneity, including sensitivity, stability and barcode drift. In developmental biology, nucleic acid barcode technology has introduced an innovative approach to cell lineage tracing. By assigning unique barcodes to individual cells, researchers can accurately identify and trace the origin and differentiation pathways of cells at various developmental stages, thereby illuminating the dynamic processes underlying tissue development and organogenesis. In cancer research, nucleic acid barcoding has played a pivotal role in analyzing the clonal architecture of tumor cells, exploring their heterogeneity and resistance mechanisms and enhancing our understanding of cancer evolution and inter-clonal interactions. Furthermore, nucleic acid barcodes play a crucial role in stem cell research, enabling the tracking of stem cells from diverse origins and their derived progeny. This has offered novel perspectives on the mechanisms of stem cell self-renewal and differentiation. The present review presented a comprehensive examination of the principles, applications and challenges associated with nucleic acid barcode technology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d5cf40687295751922bb2f7725d94b4cd317977" target='_blank'>
              Barcodes based on nucleic acid sequences: Applications and challenges (Review)
              </a>
            </td>
          <td>
            Ying Hong Wei, Faquan Lin
          </td>
          <td>2025-05-02</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ba430604b98f8f47f7bf5f84f30b7530579a6d" target='_blank'>
              Spatial Biology and Organoid Technologies Reveal a Potential Therapy-Resistant Cancer Stem Cell Population in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Pritha Adhikary, Jayati Chakrabarti, Jiang Wang, Ugonna Ezuma-Igwe, Xi Sun, Willian C. Bastian, Nicolas Braissand, Megan P. Corbett, Eugene F. Douglass, Sangmin Kim, Rohan Kanakamedala, Abigail G. Branch, Saptarshi Mallick, Meenu Priya Resmi, Payton Stevens, Marco Padilla-Rodriguez, Maga Sanchez, Gregory Lawrence Beatty, James Griffin, Petros G. Nikolinankos, Kellen Chen, Taylor Riall, Syed A. Ahmad, Rachna Shroff, Frédéric Hollande, Davendra P. S. Sohal, Yana Zavros
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9da57971dc8950b12d4ca5ccbc0969a63be6c27" target='_blank'>
              Comparison of QuPath and HALO platforms for analysis of the tumor microenvironment in prostate cancer
              </a>
            </td>
          <td>
            Wei Zhang, Qin Zhou, Jonathan V. Nguyen, Erika Egal, Qian Yang, Michael R. Freeman, S. Hu-Lieskovan, Gita Suneja, Anna Coghill, Beatrice S Knudsen
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Single-cell RNA-sequencing (scRNA-seq) coupled with robust computational analysis facilitates the characterization of phenotypic heterogeneity within tumors. Current scRNA-seq analysis pipelines are capable of identifying a myriad of malignant and non-malignant cell subtypes from single-cell profiling of tumors. However, given the extent of intra-tumoral heterogeneity, it is challenging to assess the risk associated with individual cell subpopulations, primarily due to the complexity of the cancer phenotype space and the lack of clinical annotations associated with tumor scRNA-seq studies. To this end, we introduce SCellBOW, a scRNA-seq analysis framework inspired by document embedding techniques from the domain of Natural Language Processing (NLP). SCellBOW is a novel computational approach that facilitates effective identification and high-quality visualization of single-cell subpopulations. We compared SCellBOW with existing best practice methods for its ability to precisely represent phenotypically divergent cell types across multiple scRNA-seq datasets, including our in-house generated human splenocyte and matched peripheral blood mononuclear cell (PBMC) dataset. For tumor cells, SCellBOW estimates the relative risk associated with each cluster and stratifies them based on their aggressiveness. This is achieved by simulating how the presence or absence of a specific cell subpopulation influences disease prognosis. Using SCellBOW, we identified a hitherto unknown and pervasive AR−/NElow (androgen-receptor-negative, neuroendocrine-low) malignant subpopulation in metastatic prostate cancer with conspicuously high aggressiveness. Overall, the risk-stratification capabilities of SCellBOW hold promise for formulating tailored therapeutic interventions by identifying clinically relevant tumor subpopulations and their impact on prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7e092ade35e70219652d97def0ea91976347a8d" target='_blank'>
              Artificial intelligence driven tumor risk stratification from single-cell transcriptomics using phenotype algebra
              </a>
            </td>
          <td>
            Namrata Bhattacharya, A. Rockstroh, S. Deshpande, S. Thomas, Anunay Yadav, Chitrita Goswami, Smriti Chawla, Pierre Solomon, Cynthia Fourgeux, Gaurav Ahuja, Brett Hollier, Himanshu Kumar, A. Roquilly, Jérémie Poschmann, M. Lehman, Colleen C. Nelson, Debarka Sengupta
          </td>
          <td>2025-06-13</td>
          <td>eLife</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Live-cell imaging (LCI) of modified T cells co-cultured with cancer cells is commonly used to quantify T cell anti-cancer function. Videos captured by LCI show complex multi-cell behavioral phenotypes that go beyond simple cancer cell fluorescence measurements. Here, we develop an unsupervised analysis workflow to characterize LCI data generated using the Incucyte imaging platform. Unlike most LCI analyses, we avoid cell segmentation due to the low spatiotemporal resolution of the LCI videos and high levels of cell-cell contact. Instead, we develop methods that identify global aggregation patterns and local cellular keypoints to characterize the multicellular interactions that determine cancer cell sensitivity to, or escape from, T cell surveillance. We demonstrate our segmentation-free live-cell behavioral analysis (SF-LCBA) methods on TCR T cells from four donors with varying proportions of cells with a beneficial RASA2 knockout and effector-to-target initial concentrations in a co-culture with A375 melanoma cells. We find that different T cell modifications affect the spatiotemporal dynamics of multicellular aggregate formation. In particular, we show that fewer and smaller cancer cell aggregates form at high ratios of effector T cells to target cancer cells and high titrations of T cells with RASA2 knockouts. Our SF-LCBA method identifies, characterizes, and tracks cellular aggregate formation in datasets that are unsuitable for cell segmentation and tracking, opening the door to more therapeutically-relevant measurements of modified T cell therapy cell behavioral phenotypes from LCI data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3384915d0e9331905d28acb564a6fd8addad93fa" target='_blank'>
              Live-cell analyses with unsegmented images to study cancer cell response to modified T cell therapy
              </a>
            </td>
          <td>
            Leo Epstein, A. Weiner, Archit Verma, Mozhgan Saedi, J. Carnevale, Alex Marson, Barbara E. Engelhardt
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Circulating tumor cell (CTC) clusters are highly efficient metastatic seeds in various cancers. Yet, their genetic heterogeneity and clonal architecture is poorly characterized. Using whole-exome sequencing coupled with phylogenetic inference from CTC clusters of patients with breast and prostate cancer, as well as mouse cancer models alongside barcode-mediated clonal tracking in vivo, we demonstrate oligoclonal composition of individual CTC clusters. These results improve our understanding of metastasis-relevant clonal dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03c2f424fa808947c7f2df2ddf86dcfb503a1bd5" target='_blank'>
              Phylogenetic inference reveals clonal heterogeneity in circulating tumor cell clusters
              </a>
            </td>
          <td>
            David Gremmelspacher, Johannes Gawron, B. Szczerba, Katharina Jahn, Francesc Castro-Giner, Jack Kuipers, Jochen Singer, Francesco Marass, Ana Gvozdenovic, Selina Budinjas, Heike Pueschel, Cyril Rentsch, Alfred Zippelius, V. Heinzelmann-Schwarz, Christian Kurzeder, W. Weber, Christoph Rochlitz, Marcus Vetter, N. Beerenwinkel, Nicola Aceto
          </td>
          <td>2025-06-02</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd09b27d0d8fe6fa80c372bc8924f459f7a4d33" target='_blank'>
              MOSAIC: Intra-tumoral heterogeneity characterization through large-scale spatial and cell-resolved multi-omics profiling
              </a>
            </td>
          <td>
            Caroline Hoffmann
          </td>
          <td>2025-05-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f703b1371a985c2aa75a6f16b3b87971f363e77b" target='_blank'>
              Spatial analysis reveals a novel inflammatory tumor transition state which promotes a macrophage-driven induction of sarcomatoid renal cell carcinoma
              </a>
            </td>
          <td>
            Allison M. May, Suguru Kadomoto, Claire Williams, Alex C. Soupir, Stephanie The, J. McGue, Tyler Robinson, G. Shelley, Mitchell T Hayes, Brooke L. Fridley, Jodi A. Balasi, Paolo M. Ramos Echevarria, J. Dhillon, S. Nallandhighal, Satwik Acharyya, Liying Chen, Jessica Aldous, Nathan Schurman, Veera Baladandayuthapani, Timothy L. Frankel, S. Salami, Brandon J. Manley, R. Mehra, Aaron M. Udager, Evan T. Keller
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d980820189344192705dd2c34f2a4d48d18931d" target='_blank'>
              An atlas of gene programs in non-small cell lung cancer delineates novel patterns of lineage vacillation, plasticity, and aggressiveness
              </a>
            </td>
          <td>
            Rui Hong, Pascal Klöckner, Sarah A. Mazzilli, Avrum E. Spira, J. Beane, Masanao Yajima, Joshua D. Campbell
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6bec286991f2affb3f9522a83843f9469ba7b89" target='_blank'>
              Developmental Plasticity and Stromal Co-option Shape a Pituitary Neuroendocrine Tumor Transcriptional Continuum
              </a>
            </td>
          <td>
            Robert C. Osorio, , Jangham Jung, Alexander J. Ehrenberg, A. Saha, Meeki K. Lad, Harmon Khela, Nicole Brennick, P. Giannikopoulos, William W. Seeley, Lea T. Grinberg, Aaron Diaz, M. Aghi
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 e20040


 Background:
 Immunotherapy has revolutionized cancer treatment, particularly in non-small cell lung cancer (NSCLC), yet a large fraction of patients remain refractory to treatment. Understanding treatment response requires characterization of immune cell populations and their spatial organization, as cellular neighborhoods critically influence immune cell recruitment, activation, and function. Current imaging-based and NGS-based methods struggle to achieve clear separation of individual cells in densely packed regions. Hence, we developed STAMP-seq, a novel method tagging single nuclei with cleavable spatial barcodes using high-density DNA sequencing chips, to uncover immunotherapy response-related modules and dynamic changes in NSCLC.
 Methods:
 STAMP-seq employs spatially-encoded DNA arrays to label individual nuclei with cleavable barcodes. Following validation in mouse brain tissue, we analyzed surgically resected NSCLC samples from immunotherapy-treated patients, integrating spatial transcriptomics with BCR profiling to characterize treatment response features.
 Results:
 STAMP-seq achieves precise spatial barcoding, detecting over 30,000 cells per adult mouse brain section with reliable cell subtype assignment. In immunotherapy-treated NSCLC samples, spatial analysis revealed distinct cellular communities associated with treatment response. Responding patients showed enrichment of tertiary lymphoid structures and specialized plasma cell niches, with tumor cells exhibiting elevated TNF-α signaling and reduced proliferation. These tumor cells showed proximity to

 ROBO2

 + CAFs while remaining distant from

 ACTA2

 + CAFs. BCR clonotype and pseudotime analysis revealed that

 IGHG1

 + plasma cells are recruited from TLS or vasculature by stromal-rich "transit stops" containing antigen-presenting capabilities CAFs through

 HMGB1

 -

 CXCR4

 interaction.
 Conclusions:
 STAMP-seq enables high-resolution spatial transcriptomics at single-nucleus level. Our characterization of NSCLC spatial architecture revealed previously unrecognized immune response organization, particularly how specialized stromal-rich "transit stops" coordinate cancer-suppressing plasma cell recruitment. These findings demonstrate how spatial information can advance diagnostic and prognostic module development for human diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/023df8c02dd2a154737c443a9b87fee23263bac2" target='_blank'>
              Use of STAMP-seq to enable spatial transcriptomics through single-nucleus in situ tagging and exploration of relationship of dynamic plasma cell modules and immunotherapy response in non-small cell lung cancer.
              </a>
            </td>
          <td>
            Yitong Pan, Huan Yan, Jinhuan Han, Rui Wu, Xingyong Ma, Ying Guan, Keyu Li, Qingquan Wei, Guangxin Zhang, Shaozhuo Jiao, Yongchang Zhang, Chenxi Tian
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/432419f2032b46caaaf3eb1aa445ee0f4fa38314" target='_blank'>
              Clonal tracing with somatic epimutations reveals dynamics of blood ageing.
              </a>
            </td>
          <td>
            Michael Scherer, Indranil Singh, M. Braun, Chelsea Szu-Tu, Pedro Sanchez Sanchez, Dominik Lindenhofer, N. A. Jakobsen, Verena Körber, Michael Kardorff, Lena Nitsch, Pauline Kautz, Julia Rühle, Agostina Bianchi, Luca Cozzuto, R. Frömel, Sergi Beneyto-Calabuig, Caleb A Lareau, Ansuman T. Satpathy, Renée Beekman, Lars M Steinmetz, S. Raffel, Leif S. Ludwig, Paresh Vyas, Alejo E. Rodriguez-Fraticelli, Lars Velten
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="
 8540


 Background:
 The liver is a frequent site of metastasis and carries a poor prognosis in patients with non-small cell (NSCLC) and small cell lung cancer (SCLC). Patients with liver metastases (LM) derive limited benefit from immune checkpoint inhibitors (ICI), due to hepatic myeloid derived suppressor cell (MDSC) mediated T cell elimination. Here, we used imaging mass cytometry (IMC) to perform single cell, highly multiplexed, analysis of LM and primary lung tumors to investigate how vascular endothelial growth factor (VEGF) influences T cell depletion within the tumor immune microenvironment (TIME) of LM.
 Methods:
 We comprehensively characterizedthe TIME in LM and primary lung tumors in 21 patients with NSCLC or SCLC using IMC. A panel of 40 antibodies was assembled to interrogate immune subsets and VEGF pathway markers. Each antibody was conjugated to a unique metal isotope. After validation, the antibody cocktail was used to stain the biopsies. Tissue images were segmented using Mesmer, and hierarchical clustering was applied to single-cell expression data to identify phenotypes. Similar clustering of cell neighbor profiles was applied to obtain spatial motifs. Phenotypic and motif frequencies, together with functional expression across phenotypes, were compiled from all samples and compared across conditions.
 Results:
 Initial visualization of the raw, unsegmented data revealed higher infiltration of CD8
 +
 and CD4
 +
 T cells in LM compared to the lung TIME. After segmentation, marker expression heatmaps uncovered complex cell-cell interaction ecosystems. The liver samples were enriched with M2 macrophages (CD163
 +
 ), MDSC (CD11b
 +
 ), and proliferative endothelial cells (CD105
 +
 ) whereas the lung samples were enriched in tumor cells (TTF1
 +
 for NSCLC and INSM1
 +
 /synaptophysin
 +
 for SCLC) and T cells (CD4
 +
 , CD8
 +
 ). Spatial neighborhood profiling of NSCLC liver tissues identified 12 neighborhood types, showing a general trend of mutual exclusivity between MDSCs and CD4
 +
 /CD8
 +
 T cells across neighborhoods. Notably, CD8
 +
 T cells in MDSC-enriched neighborhoods exhibited consistently higher FAS expression, a key apoptotic marker. Heterogenous FAS and VEGF signaling across neighborhoods suggested a mixed immune-suppressive and vascularized response in the liver TIME, supporting VEGF’s role in mediating MDSC-driven hepatic CD8
 +
 T cell depletion in patients with LM.
 Conclusions:
 Our findings highlight significant differences in the TIME between LM and primary lung tumors, with LM demonstrating a more immunosuppressive and VEGF-enriched milieu. The spatial association of MDSCs with CD8
 +
 T cells, along with elevated FAS expression and VEGF signaling suggests a mechanistic role for VEGF in driving immune evasion within liver tumors. These results underscore the potential of VEGF blockade as a therapeutic strategy to overcome T cell suppression and improve ICI efficacy in patients with lung cancer metastatic to liver (NCT05588388, PI Sankar).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59e054bf8906de7ff5d9b26427c709d2d7fa221a" target='_blank'>
              Immune landscape of liver metastases in advanced lung cancer.
              </a>
            </td>
          <td>
            K. Sankar, Simeon Mahov, A. Villamejor, A. Walts, Akil A. Merchant, Karen L. Reckamp, Neil A. Bhowmick
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5b1b0582e2b72f3132558990be0699903a9036" target='_blank'>
              CLONEID: A Framework for Monitoring and Steering Subclonal Dynamics
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Vural Tagal, Tao Li, Saeed S. Alahmari, Jackson Cole, Daniel Hannaby, Xiaoqing Yu, Konstantin Maksin, Andrew Schultz, Hojoon Lee, Issam ElNaqa, S. Eschrich, Janine M Lupo, Hanlee P. Ji, Aaron A Diaz, Noemi Andor
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3c75d8fc63308ed8697666b49a18333c902334" target='_blank'>
              A Single-cell and Spatially Resolved Cell Atlas of Human Esophageal Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Yong Shi, Ke An, Yu Qi, Xinhan Zhang, Yueqin Wang, Xuran Zhang, Shaoxuan Zhou, Ouwen Li, Yanan Song, Jiayi Zhou, Yue Du, Mingyang Hou, Yun-Gui Yang, QuanC. Kan, Xin Tian
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Image-based single-cell transcriptomics can identify diverse cell types within intact tissues. However, in adaptive immunity, V(D)J recombination generates unique immune receptors within cells of the same type, leading to important functional variation that is not yet defined by these methods. Here we introduce B-cell-receptor multiplexed error robust fluorescence in situ hybridization (BCR-MERFISH), which distinguishes plasma cell clones based on V-gene usage in combination with transcriptome profiling. We demonstrate that BCR-MERFISH accurately identifies V-gene usage in cell culture and in mice with restricted or native plasma cell diversity. We then use BCR-MERFISH to reveal the microbiota-dependent changes in plasma cell abundance, clonal diversity, and public clonotype usage in the mouse gut and the non-uniform distribution of plasma cell clones along the mouse ileum. As tissue context is an essential modulator of plasma cell dynamics, we anticipate that BCR-MERFISH may offer new insights into a wide range of immunological questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04bc662f55dd7766902cd268f9ec1ec3c80d4c7" target='_blank'>
              In situ profiling of plasma cell clonality with image-based single-cell transcriptomics
              </a>
            </td>
          <td>
            Evan Yang, Jose Aceves-Salvador, C. Castrillon, Uli Herrmann, E. Akama‐Garren, M. Carroll, Jeffrey R. Moffitt
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pituitary neuroendocrine tumors (PitNETs) can be invasive or aggressive, yet the mechanisms behind these behaviors remain poorly understood, impeding treatment advancements. Here, we integrat single-cell RNA sequencing and spatial transcriptomics, analyzing over 177,000 cells and 35,000 spots across 57 tissue samples. This comprehensive approach facilitates the identification of PitNETs tumor populations and characterizes the reconfiguration of the tumor microenvironment (TME) as PitNETs progress and invade. We trace the trajectory of TPIT-lineage PitNETs and identify an aggressive tumor cluster marked by elevated p53-mediated proliferation and a higher Trouillas classification, both associated with tumor progression. Additionally, we document the heterogeneity of immune stromal cells within PitNETs, particularly noting the enrichment of SPP1+ tumor associated macrophages (TAMs) in invasive tumors. These TAMs facilitate tumor invasion through the SPP1-ITGAV/ITGB1 signaling pathway. Our in-depth single-cell and spatial analysis of PitNETs uncovers the molecular dynamics within the TME, suggesting potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b1b01507bfc7f0aa4eb306817808e84f0ce1133" target='_blank'>
              Single-cell and spatial transcriptome analyses reveal tumor heterogeneity and immune remodeling involved in pituitary neuroendocrine tumor progression
              </a>
            </td>
          <td>
            Wan Su, Zhang Ye, Jifang Liu, K. Deng, Jinghua Liu, Huijuan Zhu, Lian Duan, Chen Shi, Linjie Wang, Yuxing Zhao, Fengying Gong, Yi Zhang, Bo Hou, Hui You, Feng Feng, Qing Ling, Yu Xiao, Yongdong Guo, Wenyi Fan, Sumei Zhang, Zixin Zhang, Xiaomin Hu, Yong Yao, Chunhong Zheng, Lin Lu
          </td>
          <td>2025-05-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90f49812ba9bf54efb86922eb7c7255926faf91" target='_blank'>
              Mapping and reprogramming microenvironment-induced cell states in human disease using generative AI
              </a>
            </td>
          <td>
            Amir Akbarnejad, Lloyd Steele, D. Jafree, Sebastian Birk, M. Sallese, K. Rademaker, Adam Boxall, Ben Rumney, Catherine Tudor, Minal Patel, M. Prete, Stanislaw Makarchuk, Tong Li, Heather Stanley, A. Foster, Kenny Roberts, Andrew L. Trinh, C. E. Villa, G. Testa, S. Mahil, Arash Mehrjou, Catherine Smith, Sattar Vakili, M. Clatworthy, Thomas Mitchell, O. Bayraktar, M. Haniffa, M. Lotfollahi
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1554b6f0212ba05651f81096c5d01da1508bdc40" target='_blank'>
              Cell simulation as cell segmentation.
              </a>
            </td>
          <td>
            Daniel C Jones, Anna E. Elz, Azadeh Hadadianpour, H. Ryu, David R. Glass, Evan W. Newell
          </td>
          <td>2025-05-22</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90517c7621a56cfd44f51891279305813cf3e121" target='_blank'>
              Notch3 regulates pericyte phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            N. Chalkidi, Athanasia Stavropoulou, V. Arvaniti, Christina Paraskeva, Artemis Monogyiou, Maria Sakkou, Christoforos Nikolaou, V. Koliaraki
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328ab9ab4a1194c8da958fc1fd33834df82031cb" target='_blank'>
              Context-dependent spatial multicellular network motifs for single-cell spatial biology
              </a>
            </td>
          <td>
            Amos Zamir, Yael Amitay, Yuval Tamir, Leeat Keren, A. Zaritsky
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Simple Summary Spatial transcriptomics (ST) is an innovative technology that maps gene expression while preserving tissue structure, offering new perspectives on disease mechanisms in the lung. This review explores how ST has advanced our understanding of lung cancer and other pulmonary diseases. In lung cancer, ST helps reveal tumor heterogeneity, immune evasion mechanisms, and spatially distinct cellular subtypes that influence therapy response. In diseases such as pulmonary fibrosis, COPD, and asthma, ST identifies localized immune niches, fibroblast diversity, and vascular remodeling that drive disease progression. The review also compares major ST platforms like GeoMx, Visium, and Xenium and highlights current clinical trials and future directions. By integrating molecular insights with spatial context, ST holds great promise for improving diagnosis, treatment, and precision medicine in respiratory diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d6db631a991c94025cd8ea23579c8fe3e08735" target='_blank'>
              Spatial Transcriptomics in Lung Cancer and Pulmonary Diseases: A Comprehensive Review
              </a>
            </td>
          <td>
            D. Kang, Yoonjoo Kim, Ji Hyeon Lee, Hyeong Seok Kang, Chaeuk Chung
          </td>
          <td>2025-06-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) is emerging as a powerful technology that transforms our understanding of thyroid cancer by offering a spatial context of gene expression within the tumor tissue. In this review, we synthesize the recent applications of ST in thyroid cancer research, with a particular focus on the heterogeneity of the tumor microenvironment, tumor evolution, and cellular interactions. Studies have leveraged the spatial information provided by ST to map distinct cell types and expression patterns of genes and pathways across the different regions of thyroid cancer samples. The spatial context also allows a closer examination of invasion and metastasis, especially through the dysregulation at the tumor leading edge. Additionally, signaling pathways are inferred at a more accurate level through the spatial proximity of ligands and receptors. We also discuss the limitations that need to be overcome, including technical limitations like low resolution and sequencing depth, the need for high-quality samples, and complex data handling processes, and suggest future directions for a wider and more efficient application of ST in advancing personalized treatment of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3b3022cb800cbef42ede8e65012209b885afa4" target='_blank'>
              Spatial Transcriptomics in Thyroid Cancer: Applications, Limitations, and Future Perspectives
              </a>
            </td>
          <td>
            Chaerim Song, Hye-Ji Park, Man S. Kim
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a445923377e45e9715876b4b50114b16dab283f8" target='_blank'>
              Integrated single cell spatial multi-omics landscape of WHO grades 2-4 diffuse gliomas identifies locoregional metabolomic regulators of glioma growth
              </a>
            </td>
          <td>
            Yanxia Ma, Shamini Ayyadhury, Sanjay K Singh, Yashu Vashishath, Cagri Ozdemir, Trevor D McKee, Nhat Nguyen, Akshay V. Basi, Duncan Mak, Javier A. Gomez, Jason T. Huse, Sana Noor, Daniel G. Winkowski, Regan Baird, Jeffrey S. Weinberg, Frederick F. Lang, Jared K. Burks, S. Bozdag, Erin H. Seeley, C. Ene
          </td>
          <td>2025-05-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Developing high-resolution reference maps of disease-susceptible spatial niches is a critical step to mitigating the profound effects of lung disease. Here, we present an integrated multimodal single-nucleus human lung atlas (snHLA) profiling 746,047 nuclei from 49 mapped lung blocks spanning clinically relevant distal airways, alveoli, and interstitium across 11 healthy adults. Integrating snRNA-seq and SNARE-seq2, which co-assays chromatin accessibility and gene expression from the same nucleus, we resolved 70 molecularly distinct populations and captured 332,846 accessible chromatin regions, nominating new transcriptional regulators of human lung cell diversity. Spatial transcriptomics using MERFISH mapped 25 cell populations across 7 structural neighborhoods and multiplexed immunofluorescence localized cell subtypes and distal airway-defining protein markers, expanding and validating distinct lung structure-specific cell populations. This open access snHLA and companion Cell Type and Marker Gene Dictionary with anatomically aligned nomenclature delivers a foundational resource at an unprecedented resolution to interrogate the origins of lung pathophysiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/085ae835dd14cf9157e86d76e8875d20b81277fe" target='_blank'>
              A Multimodal Spatial and Epigenomic Atlas of Human Adult Lung Topography
              </a>
            </td>
          <td>
            T. Duong, D. Diep, Kimberly Y. Conklin, Indy Bui, J. Purkerson, Eric Boone, Jacqueline Olness, Sahil Patel, Beverly Peng, Colin Kern, Zoey Zhao, Ravi S. Misra, H. Huyck, J. Verheyden, Zea Borok, Yun Zhang, R. Scheuermann, Quan Zhu, Gail Deutsch, Jim Hagood, Xin Sun, Kun Zhang, Gloria S. Pryhuber
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c75666ffc8a05530da0c0fa726109747688e9c42" target='_blank'>
              Single-cell chromatin landscape and DNA methylation patterns reveal shared molecular programs in human tumor and non-tumor tissue CCR8+ Treg cells
              </a>
            </td>
          <td>
            K. Braband, Tamara Kaufmann, Niklas Beumer, Malte Simon, Sara Salome Helbich, Delia Mihaela Mihoc, Morten Voss, A. Nedwed, K. Bauer, Jan-Philipp Mallm, Tanja Ziesmann, Ute Distler, Stephan Tenzer, Christoph Eckert, Michael Volkmar, D. Weichenhan, Marion Bähr, C. Plass, Matthias Linke, S. Diederich, Matthias Klein, T. Bopp, Hansjoerg Schild, Andreas Henkel, Edoardo Filippi, Federico Marini, B. Brors, Matthias M Gaida, Michael Delacher
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Recent technological advances enable mapping of tissue spatial organization at single-cell resolution, but methods for analyzing spatially continuous microenvironments are still lacking. We introduce ONTraC, a graph neural network-based framework for constructing spatial trajectories at niche-level. Through benchmarking analyses using multiple simulated and real datasets, we show that ONTraC outperforms existing methods. ONTraC captures both normal anatomical structures and disease-associated tissue microenvironment changes. In addition, it identifies tissue microenvironment-dependent shifts in gene expression, regulatory network, and cell–cell interaction patterns. Taken together, ONTraC provides a useful framework for characterizing the structural and functional organization of tissue microenvironments. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03588-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e479cda2baa828dc88acb0a9487d930981c05cbb" target='_blank'>
              ONTraC characterizes spatially continuous variations of tissue microenvironment through niche trajectory analysis
              </a>
            </td>
          <td>
            Wen Wang, Shiwei Zheng, Sujung Crystal Shin, J. C. Chávez‐Fuentes, Guo-Cheng Yuan
          </td>
          <td>2025-05-08</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="In the progression from Inflammatory Bowel Disease to associated cancer, the clonal mutational landscape shifts from selection of mutations in inflammatory genes to selection of cancer-driver mutations1–4. How prevalence and expansion of either type of mutated clones could be impacted by the cellular environment in which they arise, and how this affects the neoplastic outcome of colitis is unknown. Here, we combine in vivo lineage tracing, in-silico modelling, mutational profiling and spatial transcriptomics in a mouse model of colitis-associated tumorigenesis to capture clone fates associated with chronic inflammation. We identify epithelial- and immune-enriched neighbourhoods and propose a model in which establishment of a reparative tissue environment facilitates tumours initiation by promoting the selection and expansion of pro-oncogenic clones, reducing the span of inflammation-resistant neighbourhoods containing non-oncogenic clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b08041bfbc2e608a3daf26d2af9f4a68fced7b6" target='_blank'>
              Paired mutation calling and spatial transcriptomics identify cellular neighbourhoods dictating the neoplastic outcome of colitis
              </a>
            </td>
          <td>
            Doug Winton, Elisa B. Moutin, Linus Chang, Giada Giavara, Shenay Mehmed, Mathilde Colombé, Filipe C Lourenço, Maria-Nefeli Skoufou-Papoutsaki, Richard Kemp, Philippe Gascard, T. Tlsty, David Tourigny
          </td>
          <td>2025-05-08</td>
          <td>Research Square</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e14574


 Background:
 Understanding cellular interactions within the tumor microenvironment (TME) is essential for elucidating disease progression and advancing immunotherapy. The TME is a complex ecosystem composed of cells with dysregulated metabolism immune response and signaling pathways, which in turn influence tumor development and treatment response. Multiplexed assessment serves as an important tool for clinical oncology. The simultaneous readout of multiple processes can provide biological insights and elucidate disease mechanism. Imaging Mass Cytometry (IMC) is a spatial biology imaging technique that utilizes CyTOF technology and enables deep characterization of the diversity and complexity of the TME. IMC technology offers scalable, high-throughput acquisition while generating high-quality data with true dynamic range of signal without amplification or fluorescence-based limitations such as spectral overlap and autofluorescence.
 Methods:
 To study cellular processes and their roles in tumor progression, we utilized the Human Cell Metabolism and Human Cell Signaling Panels to investigate energy production, cellular homeostasis and mitogenic signaling pathways. We then mapped these processes to the types of cells in the TME by using the Human Immuno-Oncology IMC Panel and the Human T Cell Exhaustion IMC Panel or the Maxpar Neuro Phenotyping IMC Panel Kit to characterize immune cell and neurological phenotypes in detail. We first acquired data using Preview Mode to assess the whole tissue, followed by higher-resolution imaging of selected regions of interest using Cell Mode or of the whole tissue section using Tissue Mode.
 Results:
 Our data analysis revealed significant insights into the spatial organization and metabolic profile of cells across cancer tissues. Elevated glycolysis and mTOR pathway activation suggested adaptations to hypoxia and anabolic growth in tumor areas, while interactions between fibroblasts and immune cells highlighted crosstalk within the TME. Our Neuro Phenotyping Panel was used to reveal potential for immune response in glioblastoma. Unsupervised pixel clustering and hierarchical clustering using MCD SmartViewer highlighted metabolic activity and activation of signaling pathways within tumor regions.
 Conclusions:
 Comprehensive spatial biology profiling using the IMC approach highlights the interconnected roles these pathways play in promoting tumor survival and resistance to therapies. These findings, which illuminate the metabolic and signaling heterogeneity of the TME, are crucial for developing future prognostic assessments and have the potential to guide more effective, personalized cancer therapies. For Research Use Only. Not for use in diagnostic procedures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b16079bb313271e366d31125c0f7c2617d3a11a" target='_blank'>
              Spatial profiling of solid tumor microenvironment using imaging mass cytometry with high-plex panels.
              </a>
            </td>
          <td>
            T. Pfister, J. Y. Chwee, Nick Zabinyakov, Qanber Raza, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Solid cancers are complex 'ecosystems' comprised of diverse cell types and extracellular molecules, where heterotypic interactions significantly influence disease etiology and therapeutic response. However, our current understanding of tumor microenvironments (TMEs) remains incomplete, hindering the development and implementation of novel TME-targeted drugs. To maximize cancer therapeutic development we require a systems-level understanding of the malignant, stromal, and immune states that define the tumor and determine treatment response. In their recent study, Liu and colleagues took a new approach to resolving the complexity of stromal heterogeneity. They leveraged extensive single-cell spatial multi-omics datasets across various cancer types and platforms to identify four conserved spatial Cancer-Associated Fibroblast (CAF) subtypes, classified by their spatial organization and cellular neighborhoods. Their work expands upon prior efforts to develop a CAF taxonomy, which primarily relied on single-cell RNA-Sequencing (scRNA-Seq) and yielded a multitude of classification systems. This study advances our understanding of CAF biology by establishing a link between spatial context and CAF identity across diverse tumor types. Departing from recent single-cell transcriptomics studies that employed a marker-based approach for sub-state identification, Liu and colleagues conducted de novo discovery of CAF subtypes using spatial neighborhood information alone. By positioning spatial organization as the defining axis of CAF heterogeneity, this research redefines CAF classification and provides a new framework for exploring the rules governing tumor ecosystems and developing novel ecosystem-based therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940389ee957141f2833e0d9de53711c0ead3a30c" target='_blank'>
              Pan-Cancer Spatial Profiling Reveals Conserved Subtypes and Niches of Cancer-Associated Fibroblasts.
              </a>
            </td>
          <td>
            Hani Jieun Kim, Travis Ruan, Alex Swarbrick
          </td>
          <td>2025-05-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Disease heterogeneity across a diverse patient cohort poses challenges to cancer drug development due to inter-patient variability in treatment responses. However, current preclinical models fail to depict inter-patient tumor heterogeneity, leading to a high failure rate when translating preclinical leads into clinical successes. Methods: We integrated the expression profiles of prostate cancer (PC) lines and castration-resistant PC (CRPC) patient tumors to identify cell-lines that transcriptomically match distinct tumor subtypes in a clinical cohort. Representative cell-lines were co-cultured to create “mixed-cell” models depicting inter-patient heterogeneity in CRPC, which were employed to assess drug combinations. Results: When drug combinations previously tested in CRPC clinical cohorts, were assessed to establish proof-of-concept, in-vitro responses measured in our models concurred with their known clinical efficacy. Additionally, novel drug combinations computationally predicted to be efficacious in heterogeneous tumors, were evaluated. They demonstrated preclinical efficacy in the mixed-cell models, suggesting they will likely benefit heterogeneous patient cohorts. Furthermore, we showed that the current practice of screening cell-lines/xenografts separately and aggregating their responses, failed to detect their efficacy. Conclusions: We believe that the application of our models will enhance the accuracy of preclinical drug assessment, thereby improving the success rate of subsequent clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4d639da8c86d125f39c0d18727d1ba6b4439891" target='_blank'>
              Novel mixed cancer-cell models designed to capture inter-patient tumor heterogeneity for accurate evaluation of drug combinations
              </a>
            </td>
          <td>
            Sampreeti Jena, Daniel Kim, Adam Lee, Weijie Zhang, Kevin Zhan, Yingming Li, S. Dehm, R. S. Huang
          </td>
          <td>2025-05-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e526c65686bd001de0faa1340763d5a1c493080f" target='_blank'>
              Cancer systems immunology reveals myeloid—T cell interactions and B cell activation mediate response to checkpoint inhibition in metastatic breast cancer
              </a>
            </td>
          <td>
            Edgar Gonzalez, Jesse Kreger, Yingtong Liu, Xiaojun Wu, Arianna Barbetta, A. Baugh, Batul Al-zubeidy, Julie Jang, Sarah M. Shin, Matthew Jacobo, Vered Stearns, Roisin Connolly, Won Ho, Juliet Emamaullee, Adam L. MacLean, Evanthia T. Roussos Torres
          </td>
          <td>2025-06-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity. GBM tumors contain neoplastic cells within a spectrum of different transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we developed a novel framework, ISOSCELES (Inferred cell Sensitivity Operating on the integration of Single-Cell Expression and L1000 Expression Signatures), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identified compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states. We validated the significance of these findings in vitro, ex vivo, and in vivo, and identified a novel combination of an OLIG2 inhibitor and Depatux-M for GBM. Our studies suggest that ISOSCELES identifies cell states sensitive and resistant to targeted therapies in GBM and that it can be applied to identify new synergistic combinations. Highlights Integration of GBM single-cell RNA sequencing data with L1000-derived drug response signatures facilitates clustering of tumor cells and small molecules on cell-drug connectivity. Cell-drug connectivity predicts the identities of drug-sensitive and resistant cell states. In silico perturbation analysis using cell-drug connectivity predicts drug-induced changes in the cell-drug connectivity landscape in vivo. In silico perturbation analysis to predict drug-induced changes in the tumor cell-drug connectivity landscape predicts drug combinations that synergize in vivo to extend survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25613f3a85e6691e66bbf8705e07ffcebf8c6cbf" target='_blank'>
              DRUG AND SINGLE-CELL GENE EXPRESSION INTEGRATION IDENTIFIES SENSITIVE AND RESISTANT GLIOBLASTOMA CELL POPULATIONS
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, Ricardo J Komotar, M. I. De La Fuente, Gregory Stein, Alexander Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, N. Ayad
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83b5ca4a7a4ab998ed122fece632645d240cae73" target='_blank'>
              Decoding Plasticity Regulators and Transition Trajectories in Glioblastoma with Single-cell Multiomics
              </a>
            </td>
          <td>
            Manu Saraswat, Laura Rueda-Gensini, Elisa Heinzelmann, Tannia Gracia, Fani Memi, Grant de Jong, Jannes Straub, Cedar Schloo, D. Hoffmann, E. Jung, Tim Kindinger, Bettina Weigel, Bryce Lim, S. Weil, Oliver Gould, Richard Mair, Katharina Mikulik, Martin Rohbeck, W. Wick, F. Winkler, O. Bayraktar, O. Stegle, Moritz Mall
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Lymphomas are highly heterogeneous malignancies characterized by diverse genetic, epigenetic, and phenotypic profiles, which complicate diagnosis, prognosis, and treatment. Traditional bulk sequencing methods mask the complexity of intratumoral variation, often overlooking rare subclones that may drive disease progression and therapeutic resistance. Single-cell genomics has emerged as a transformative approach to decipher tumor heterogeneity at unprecedented resolution. This technology enables the dissection of individual cellular populations within lymphomas, offering insights into clonal evolution, transcriptional diversity, and microenvironmental interactions. By applying single-cell RNA sequencing (scRNA-seq), chromatin accessibility assays (scATAC-seq), and single-cell DNA sequencing (scDNA-seq), researchers can unravel lineage relationships, identify resistant subpopulations, and track dynamic changes in response to therapy. In lymphomas such as diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL), and mantle cell lymphoma (MCL), single-cell approaches have revealed distinct malignant and non-malignant cell states that correlate with treatment outcomes. Moreover, integrating single-cell data with spatial transcriptomics and immune profiling enhances the understanding of the tumor microenvironment, including immune evasion mechanisms. These insights can inform personalized treatment strategies, identify novel therapeutic targets, and enable early detection of relapse. Despite technical challenges such as data complexity, sample viability, and cost, the application of single-cell genomics in lymphoma research is rapidly advancing. Future directions include multi-omics integration, real-time patient monitoring, and clinical translation of predictive biomarkers. This review underscores the pivotal role of single-cell genomics in resolving tumor heterogeneity and predicting therapeutic resistance, positioning it as a cornerstone for next-generation precision oncology in lymphomas,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/438856abbab164e715372bfe3f2a6c07df9594ec" target='_blank'>
              Role of mutational signatures and clonality assessments in tailoring targeted therapies for lymphoma
              </a>
            </td>
          <td>
            Andoh Sheikh Atta-ullah, Uche Philip, Sampson Janice Candy
          </td>
          <td>2025-05-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4036914e070de84fceb3196161df4ff74c7602d" target='_blank'>
              Predicting targeted- and immunotherapeutic response outcomes in melanoma with single-cell Raman Spectroscopy and AI
              </a>
            </td>
          <td>
            Kai Chang, Mamatha Serasanambati, Baba Ogunlade, Hsiu-Ju Hsu, James Agolia, Ariel Stiber, Jeffrey Gu, Saurabh Sharma, Jay Chadokiya, Amanda Gonçalves, Fareeha Safir, Nhat Vu, Daniel Delitto, Amanda R. Kirane, Jennifer A. Dionne
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Nasopharyngeal carcinoma (NPC) is a common malignancy with a complex pathogenesis and diverse cellular composition. This study aimed to systematically analyze the transcriptional characteristics of different cell types in NPC and their roles in tumor development using single-cell transcriptomics. Methods The research team collected NPC and normal tissue samples, and performed in-depth single-cell sequencing analysis of the transcriptomes. Single-cell RNA sequencing was performed on the collected samples to obtain high-resolution transcriptional profiles of individual cells. This allowed the researchers to identify and characterize the diverse cell populations present within the NPC tumor microenvironment. Results The results showed that NPC samples contained multiple distinct cell subpopulations, including epithelial cells, immune cells (such as macrophages and T cells), endothelial cells, and stromal cells. These cell types exhibited marked differences in spatial distribution and transcriptional profiles, reflecting the high degree of heterogeneity within the tumor microenvironment. Further functional analysis revealed significant dysregulation of mitochondrial-related pathways, extracellular matrix-receptor interactions, as well as Wnt, Notch, and other signaling cascades in NPC. Conclusion This study employed single-cell transcriptomics to comprehensively elucidate the complexity of the NPC tumor microenvironment, providing new insights into the underlying mechanisms of disease pathogenesis and suggesting potential therapeutic targets. These findings lay the groundwork for the development of precision medicine approaches for NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9793611f5914170a344f624509f87b1fa27aa37" target='_blank'>
              Single-cell transcriptomic landscape reveals complex cellular heterogeneity and dysregulated signaling pathways in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Xiaoyan Zhang, Zuojia Guan, Zhan Xu
          </td>
          <td>2025-05-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spatial transcriptomics technologies have emerged as powerful tools for understanding cellular identity and function within the natural spatial context of tissues. Traditional transcriptomics techniques, such as bulk and single-cell RNA sequencing, lose this spatial information, which is critical for addressing many biological questions. Here, we present a protocol for high-resolution spatial transcriptomics using fixed frozen mouse lung sections mounted on 10X Genomics Xenium slides. This method integrates multiplexed fluorescent in situ hybridization (FISH) with high-throughput imaging to reveal the spatial distribution of mRNA molecules in lung tissue sections, allowing detailed analysis of gene expression changes in a mouse model of pulmonary hypertension (PH). We compared two tissue preparation methods, fixed frozen and fresh frozen, for compatibility with the Xenium platform. Our fixed frozen approach, utilizing a free-floating technique to mount thin lung sections onto Xenium slides at room temperature, preserved tissue integrity and maximized the imaging area, resulting in high-fidelity spatial transcriptomics data. Using a predesigned 379-gene mouse panel, we identified 40 major lung cell types. We detected key cellular changes in PH, including an increase in arterial endothelial cells (AECs) and fibroblasts, alongside a reduction in capillary endothelial cells (CAP1 and CAP2). Through differential gene expression analysis, we observed markers of endothelial-to-mesenchymal transition and fibroblast activation in PH lungs. High-resolution spatial mapping further confirmed increased arterialization in the distal microvasculature. These findings underscore the utility of spatial transcriptomics in preserving the native tissue architecture and enhancing our understanding of cellular heterogeneity in disease. Our protocol provides a reliable method for integrating spatial and transcriptomic data using fixed frozen lung tissues, offering significant potential for future studies in complex diseases such as PH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/399fddf9db865d1e3c47fe6ffcc8cfaa55ffe72d" target='_blank'>
              Enhanced Spatial Transcriptomics Analysis of Mouse Lung Tissues Reveals Cell-Specific Gene Expression Changes Associated with Pulmonary Hypertension
              </a>
            </td>
          <td>
            Hanqiu Zhao, Xiaokuang Ma, Peng Chen, Bin Liu, Jing Wei, John Zhang, Ankit A. Desai, Andrea L. Frump, Olga Rafikova, Michael B Fallon, Shenfeng Qiu, Zhiyu Dai
          </td>
          <td>2025-05-15</td>
          <td>Journal of respiratory biology and translational medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy with extremely poor prognosis. SCLC cells exhibit high plasticity and can progress from neuroendocrine (NE) to non-NE phenotypes. This dynamic evolution promotes treatment resistance and relapses, representing a challenge for targeted therapies in this elusive disease. Here we identify the transcription factor ONECUT2 (OC2) as a driver of plasticity in SCLC, leading to non-NE transcriptional states. OC2 is highly expressed in SCLC tumors compared to normal lung tissue and its expression is associated with heightened clinical stage and lymph node metastasis. We show that OC2 is a repressor of ASCL1, the NE master regulator transcription factor. In addition, OC2 upregulates non-NE programs through activation of c-MYC and Notch signaling. We also demonstrate that OC2 is required for growth and survival of SCLC cells and that it can be targeted with a small molecule inhibitor that acts synergistically with the standard combination of cisplatin and etoposide, providing a novel therapeutic strategy for OC2 active SCLC tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01267-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736f50c299b3fe9c13bfbd3d67e9601f1567eb2" target='_blank'>
              ONECUT2 reprograms neuroendocrine fate and is an actionable therapeutic target in small cell lung cancer
              </a>
            </td>
          <td>
            Mirian Gutiérrez, Irene Zamora, Raquel Iriarte, M. Pajares, Qian Yang, Chen Qian, N. Otegui, J. Fernández-Irigoyen, Enrique Santamaria, N. Alcala, A. Sexton-Oates, L. Fernandez-Cuesta, Miguel Barajas, Alfonso Calvo, Luis M. Montuenga, Beatrice S. Knudsen, Sungyong You, Michael R. Freeman, Ignacio J. Encío, Mirja Rotinen
          </td>
          <td>2025-06-11</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Breast cancer brain metastases (BCBM) affect nearly 90,000 patients annually in the United States and carry a significant risk of mortality. As metastatic lesions develop, the unique milieu of the brain microenvironment shapes disease progression and therapeutic response. Among resident brain cells, astrocytes are both the most common, and are increasingly recognized as key regulators of this process, yet their precise role remains poorly defined. Here, we present a hybrid agent-based model (ABM) to simulate tumor–astrocyte interactions on a two-dimensional lattice. In our model, metastatic tumor cells induce phenotypic reprogramming of astrocytes from an antito a pro-metastatic state, thereby enhancing tumor proliferation. We systematically evaluate how variations in astrocyte density, spatial distribution, and chemotherapy impact tumor expansion and spatial morphology, quantified by fractal dimension, lacunarity, and eccentricity. Our simulations reveal that astrocyte reprogramming accelerates tumor progression and contributes to increased morphological complexity and chemotherapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331a7ee541e9213b60f3df2d93320f1bf057f75f" target='_blank'>
              Astrocyte reprogramming drives tumor progression and chemotherapy resistance in agent-based models of breast cancer brain metastases
              </a>
            </td>
          <td>
            Rupleen Kaur, R. Barker-Clarke, Andrew Dhawan
          </td>
          <td>2025-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Organotropism results from the functional versatility of metastatic cancer cells to survive and proliferate in diverse microenvironments. This adaptivity can originate in clonal variation of the spreading tumor and is often empowered by epigenetic and molecular reprogramming of cell regulatory circuits. Related to organotropic colonization of metastatic sites are environmentally-sensitive, differential responses of cancer cells to therapeutic attack. Accordingly, understanding the organotropic profile of a cancer and probing the underlying driver mechanisms are of high clinical importance. However, determining systematically the organotropism of one cancer versus the organotropism of another cancer, potentially with the granularity of comparing the same cancer type between patients or tracking the evolution of a cancer in a single patient for the purpose of personalized treatment, has remained very challenging. It requires a host organism that allows observation of the spreading pattern over relatively short experimental times. Moreover, organotropic patterns often tend to be statistically weak and superimposed by experimental variation. Thus, an assay for organotropism must give access to statistical powers that can separate ‘meaningful heterogeneity’, i.e., heterogeneity that determines organotropism, from ‘meaningless heterogeneity’, i.e., heterogeneity that causes experimental noise. Here we describe an experimental workflow that leverages the physiological properties of zebrafish larvae for an imaging-based assessment of organotropic patterns over a time-frame of 3 days. The workflow incorporates computer vision pipelines to automatically integrate the stochastic spreading behavior of a particular cancer xenograft in tens to hundreds of larvae allowing subtle trends in the colonization of particular organs to emerge above random cell depositions throughout the host organism. We validate our approach with positive control experiments comparing the spreading patterns of a metastatic sarcoma against non-transformed fibroblasts and the spreading patterns of two melanoma cell lines with previously established differences in metastatic propensity. We then show that integration of the spreading pattern of xenografts in 40 – 50 larvae is necessary and sufficient to generate a Fish Metastatic Atlas page that is representative of the organotropism of a particular oncogenotype and experimental condition. Finally, we apply the power of this assay to determine the function of the EWSR1::FLI1 fusion oncogene and its transcriptional target SOX6 as plasticity factors that enhance the adaptive capacity of metastatic Ewing sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5025875828f543583bc1d12169d95499fa95592" target='_blank'>
              Robust statistical assessment of Oncogenotype to Organotropism translation in xenografted zebrafish
              </a>
            </td>
          <td>
            D. Saucier, Xuexia Jiang, Divya Rajendran, Roshan Ravishankar, Erin Butler, A. Marchetto, R. Kurmasheva, T. G. Grünewald, James F. Amatruda, G. Danuser
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Advances in cancer biology have been achieved by the identification of oncogenes and tumor suppressor genes through the remarkable progression of next‐generation sequencing. New techniques, such as single‐cell analysis, help uncover cancer progression and heterogeneity. Reverse genetic screenings, including methods like random mutagenesis via retroviruses, transposons, RNA interference, and CRISPR, are useful for exploring gene functions and their roles in cancer. Especially in random mutagenesis, CRISPR screening and its modifications have recently emerged as powerful tools due to their comprehensiveness and simplicity in inducing genetic mutations. Initially, CRISPR screening focused on analyzing biological phenotypes in a cell population. It has since evolved to incorporate advanced techniques, such as combining single‐cell and spatial analyses. These developments enable the investigation of cell–cell and spatial interactions, which more closely mimic In Vivo microenvironments, offering deeper insights into complex biological processes. These approaches allow for the identification of essential genes involved in cancer survival, drug resistance, and tumorigenesis. Together, these technologies are advancing cancer research and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d93feca0cdb57db69fce7d9acc441dc2942da5" target='_blank'>
              Identification of Target Genes Using Innovative Screening Systems
              </a>
            </td>
          <td>
            Keisuke Sugita, Morita Kurata
          </td>
          <td>2025-05-05</td>
          <td>Pathology International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acab5ca78a81e16038185b4373a584725e52ba21" target='_blank'>
              Low-branching vessel architecture shapes immune cell niches and predicts immune responses in renal cancer
              </a>
            </td>
          <td>
            M. Toma, Yangping Li, Melina Kehl, Tim Kempchen, L. Esser, Katharina Baschun, S. Leonardelli, Thomas Pinetz, R. Turiello, M. Yong, Natalie Pelusi, Guillermo Altamirano-Escobedo, Eduardo Bayro-Corrochano, Sebastian Kadzik, M. Eckstein, Lukas Flatz, Fabian Hörst, J. Kleesiek, J. Saal, Glen Kristiansen, Manuel Ritter, J. Ellinger, Viktor Grünwald, Niklas Klümper, Alexander Effland, M. Hölzel
          </td>
          <td>2025-06-06</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c327cba87285dc890d1f7b106ff92f090a08a250" target='_blank'>
              Single-Cell Analysis of Meningiomas Reveals Mutation-Associated Tumor and Immune Cell Gene Expression Programs
              </a>
            </td>
          <td>
            Jose A. Maldonado, Bryce L. Mashimo, Anthony Z. Wang, Rupen Desai, Saad M. Khan, Ngima D Sherpa, Markus I Anzaldua-Campos, Gregory J Zipfel, Albert H. Kim, J. Dowling, Eric C. Leuthardt, Joshua W. Osbun, A. Vellimana, Michael R. Chicoine, Gavin P. Dunn, Allegra A. Petti
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Chromosomal instability (CIN), a characteristic feature of esophageal adenocarcinoma (EAC), drives tumor aggressiveness and therapy resistance, presenting an intractable problem in cancer treatment. CIN leads to constitutive stimulation of the innate immune cGAS–STING pathway, which has been typically linked to anti-tumor immunity. However, despite the high CIN burden in EAC, the cGAS– STING pathway remains largely intact. To address this paradox, we developed novel esophageal cancer models, including a CIN-isogenic model, discovering myeloid-attracting chemokines – with the chemokine CXCL8 (IL-8) as a prominent hit – as conserved CIN-driven targets in EAC. Using high-resolution multiplexed immunofluorescence microscopy, we quantified the extent of ongoing cGAS-activating CIN in human EAC tumors by measuring cGAS-positive micronuclei in tumor cells, validated by orthogonal whole-genome sequencing-based CIN metrics. By coupling in situ CIN assessment with single-nucleus RNA sequencing and multiplex immunophenotypic profiling, we found tumor cell-intrinsic innate immune activation and intratumoral myeloid cell inflammation as phenotypic consequences of CIN in EAC. Additionally, we identified increased tumor cell-intrinsic CXCL8 expression in CINhigh EAC, accounting for the inflammatory tumor microenvironment. Using a novel signature of CIN, termed CINMN, which captures ongoing CIN-associated gene expression, we confirm poor patient outcomes in CINhigh tumors with signs of aberrantly rewired cGAS–STING pathway signaling. Together, our findings help explain the counterintuitive maintenance and expression of cGAS–STING pathway components in aggressive, CINhigh tumors and emphasize the need to understand the contribution of CIN to the shaping of a pro-tumor immune landscape. Therapeutic strategies aimed at disrupting the cGAS-driven inflammation axis may be instrumental in improving patient outcomes in this aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dfd8088150f447fa336f3b6ae36838000571864" target='_blank'>
              Chromosomal instability shapes the tumor microenvironment of esophageal adenocarcinoma via a cGAS–chemokine–myeloid axis
              </a>
            </td>
          <td>
            Bruno Beernaert, Rose L Jady-Clark, Parin Shah, Erik Ramon‐Gil, Nora M Lawson, Zack D. Brodtman, Somnath Tagore, Frederik Stihler, Alfie S Carter, Shannique Clarke, Tong Liu, Winston Zhu, Erkin Erdal, Alistair Easton, Leticia Campo, Molly Browne, Stephen Ash, Nicola Waddell, Thomas Crosby, Simon R. Lord, Derek A Mann, Ignacio Melero, Carlos E. De Andrea, Andrea E. Tijhuis, F. Foijer, Ester M. Hammond, Kadir C Akdemir, J. Leslie, Benjamin Izar, Eileen E Parkes
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Uveal melanoma (UM) is a highly aggressive intraocular malignancy with limited therapeutic options for metastatic disease. Existing transgenic UM mouse models inadequately recapitulate human disease progression, while transplant models lack immune competence for studying the tumor immune microenvironment and therapeutic interventions. To address these limitations, we developed a genetically engineered mouse model incorporating stepwise genetic alterations implicated in human UM progression. Spatiotemporally controlled expression of mutant GNAQQ209L from the endogenous locus induced choroidal nevi with limited penetrance. Concomitant BAP1 deletion enhanced nevus formation, while further MYC activation led to fully penetrant intraocular tumors with metastatic potential. Single-cell RNA sequencing revealed malignant cells segregated into Melanocytic and Neural Crest-like subpopulations characterized by distinct transcriptional and biosynthetic programs. Trajectory analyses inferred dedifferentiation from the Melanocytic toward the Neural Crest-like state during tumor progression. Comparison to human UM revealed commonalities with highly aggressive Class 2 UM, including gene expression signatures and copy number gains affecting genes that map to human chromosome 8q beyond the activated MYC allele, suggesting cooperative effects of multiple drivers in this chromosomal region. The tumor microenvironment featured immunosuppressive macrophage populations and exhausted T cells, closely resembling human UM. This physiologically relevant, immune-competent model provides a platform for investigating UM biology, functionally characterizing candidate driver genes, and developing immune-based therapeutic strategies. SIGNIFICANCE STATEMENT We developed a mouse model that resembles the genetic progression and phenotypic plasticity of human UM. This spatially controlled model confirms the critical role of driver mutations in GNAQ and BAP1, proposes MYC as a promoter of malignant transformation in coordination with other chromosome 8q genes, and reveals UM progression through distinct cellular states. This model offers an urgently needed preclinical platform for understanding the immunogenomics of UM and for testing immune and targeted treatments for this lethal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35df183db757e5d1e691fe22c0b60fdf40bee0e5" target='_blank'>
              A multi-step immune-competent genetic mouse model reveals phenotypic plasticity in uveal melanoma
              </a>
            </td>
          <td>
            Xiaonan Xu, Xiaoxian Liu, J. Dollar, Xiao Liu, Neel Jasani, Benjamin Posorske, S. Sriramareddy, Vinesh Jarajapu, J. Kuznetsoff, John Sinard, Richard L. Bennett, Jonathan D. Licht, K. Smalley, J. W. Harbour, Xiaoqing Yu, F. Karreth
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d1c1b28cd57c3d7a9126fb0efbcbda9de4833e" target='_blank'>
              The genomic and transcriptional landscape of the spinal cord H3K27-altered diffuse midline glioma suggests the potential therapeutic strategy
              </a>
            </td>
          <td>
            Yiwei Xiao, Mengyao Li, Qiang Gao, Zhihong Qian, Yukui Shang, Linkai Jing, Zili Zhen, Yong Ai, Guihuai Wang, Kehkooi Kee, Wei Zhang
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e285847d1a624eba0863eb2e7a290a733cac7918" target='_blank'>
              Phenotypic and Functional Characterization of Oncohistone Mutations in Breast Cancers
              </a>
            </td>
          <td>
            Andrea D. Edwards, Yangyang Dai, Siddharth Singh, Micah Thornton, T. Nandu, Ralf Kittler, C. Camacho, Dan Huang, W. Kraus
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is one of the most lethal cancers for humans. HCC is highly heterogeneous. In this study, we performed ultra-depth (∼1 million reads per spot) sequencing of 6,320 spatial transcriptomes on a case of HCC. Sixteen distinct spatial expression clusters were identified. Each of these clusters was spatially contiguous and had distinct gene expression patterns. In contrast, benign liver tissues showed minimal heterogeneity in terms of gene expression. Numerous immune cell-enriched spots were identified in both HCC and benign liver regions. Cells adjacent to these immune cell-enriched spots showed significant alterations in their gene expression patterns. Interestingly, the responses of HCC cells to the nearby immune cells were significantly more intense and broader, while the responses of benign liver cells to immune cells were somewhat narrow and muted, suggesting an innate difference in immune cell activities towards HCC cells in comparison with benign liver cells. However, cell-cell interaction analyses showed significant immune evasion by HCC cancer cells. When standard-depth sequencing was performed, significant numbers of genes and pathways that were associated with these changes disappeared. Qualitative differences in some pathways were also found. These results suggest that deep spatial sequencing may help to uncover previously unidentified mechanisms of liver cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa11a255e72a72322fc9f248cef1483a72dd27ef" target='_blank'>
              Deep spatial sequencing revealing differential immune responses in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yan-Ping Yu, Caroline Obert, Baoguo Ren, Marielle Krivit, Kyle Metcalfe, Jia-Jun Liu, Tuval Ben-Yehezkel, Silvia Liu, Jian-Hua Luo
          </td>
          <td>2025-06-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract The most common cause of death in neurofibromatosis type 1 (NF1) is the development of malignant peripheral nerve sheath tumor (MPNST), a deadly sarcoma that can transform from benign plexiform neurofibromas (PN) or premalignant atypical neurofibromas (AN). We built a single-cell dataset of 55 NF1-associated PN, AN, and MPNST to define cellular changes in neurofibroma at-risk of malignant transformation. Integrative analysis of changes in the tumor microenvironment revealed the emergence of malignant tumor cells, regulatory T cells, and loss of activated macrophages in MPNST. Using this reference dataset, we validated findings using anchor-based label transfer in an additional 19 NF1 nerve sheath tumors profiled with single cell sequencing, and public datasets. We then defined protein biomarkers of malignant transformation from high-throughput proteomic analysis of plasma samples collected from 45 NF1 patients that correlated to mRNAs specific to MPNST cell populations. Fifty plasma proteins accurately and non-invasively distinguished patients with MPNST from those with premalignant tumors. These markers should improve the ability to identify high-risk neurofibromas for improved cancer surveillance and enable early detection of malignant transformation in NF1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea5d7e2095827b87208bdc18119d43a501ce00d" target='_blank'>
              Single-cell tumor microenvironment profiling informs a circulating proteome test for the interception of malignant transformation in NF1 nerve sheath tumors
              </a>
            </td>
          <td>
            Jack Shern
          </td>
          <td>2025-06-18</td>
          <td>Research Square</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ca2ada71e9dda717c7f2fb533cc95887911976c" target='_blank'>
              SMARCA4 is essential for early-stage tumor development but its loss promotes late-stage cancer progression in small-cell lung cancer
              </a>
            </td>
          <td>
            Nicole Kirk, Jin Ng, Kate-Lin Ly, Young Ho Ban, Godfrey Dzhivhuho, Jinho Jang, Kyung-Pil Ko, M. Kareta, Jae-Ll Park, Anthony N. Karnezis, Anish Thomas, K. Sutherland, Kwon-Sik Park
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 8539


 Background:
 Immune checkpoint inhibitors have greatly improved outcomes in advanced non-small cell lung cancer (NSCLC). However, patients with EGFR mutant NSCLC have a poor response to immune checkpoint inhibitor therapy. Emerging evidence suggests that an immunosuppressive tumor microenvironment plays an important role in this setting, however, we still lack fundamental knowledge about tumor endothelial cell biology. We hypothesized that oncogene specific changes in the expression of immune-related genes in tumor endothelial cells account for differences in the tumor microenvironment and efficacy of immune checkpoint inhibitor therapy.
 Methods:
 We utilized spatial transcriptomics (GeoMx Digital Spatial Profiling) on resected tumor tissue of EGFR mutant (n = 5) and KRAS mutant (n = 5) NSCLC patients to investigate the transcriptional signature of tumor, endothelial and stromal cells. Additionally, we used NicheNet to explore intercellular communication between tumor, endothelial, and stromal cells. Immune gene set enrichment analysis scores were calculated using ESTIMATE. Immune cell type proportions were estimated using CIBERSORT.
 Results:
 Using spatial transcriptomics, we dissected the tumor microenvironment into tumor, stromal and endothelial compartments. By analyzing predicted cellular communication, we found that tumor and stromal cells primarily affect an interferon-related gene signature in tumor endothelial cells. Notable differentially expressed interferon-related genes, that were significantly up-regulated in KRAS mutant and down-regulated in EGFR mutant NSCLC patients, included CXCL9, STAT1, WARS1, IRF1 and ICAM1. In the stromal compartment, immune gene set enrichment analysis scores were significantly lower in EGFR mutant than KRAS mutant NSCLC (median, 355 vs. 713, P = 0.026) indicating an immunosuppressive tumor microenvironment. We observed substantial heterogeneity while exploring the cellular landscape of the stromal compartment in EGFR mutant and KRAS mutant NSCLC. Notably, we found a significantly decreased proportion of pro-inflammatory macrophages in the stromal compartment of EGFR compared to KRAS mutant NSCLC (P = 0.006).
 Conclusions:
 We identified distinct interferon-related gene signatures in tumor endothelial cells of patients with EGFR and KRAS mutantNSCLC associated with variations in the cellular composition of the tumor microenvironment. This may provide a better understanding for the development of spatial biomarkers to identify which oncogene-driven NSCLC patients are most likely to benefit from immune checkpoint inhibitor therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3c21a018e08000029baaeb3ba8f18554c44ba0f" target='_blank'>
              Spatial transcriptomic profiling of the tumor microenvironment in EGFR and KRAS mutant non-small cell lung cancer.
              </a>
            </td>
          <td>
            F. J. Bolte, Nick Natale, Samantha DiBenedetto, Margaret Moore, E. Stelow, Richard Hall, D. Gioeli
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background: Gallbladder cancer (GBC) is a rare but aggressive malignancy, often detected late due to early asymptomatic stages. Understanding cellular and molecular changes from normal tissue to high-grade intraepithelial neoplasia (HGIN) and invasive GBC is vital for identifying early biomarkers and therapeutic targets. Methods: We performed single-cell RNA sequencing on 98,113 cells derived from 2 normal adjacent tissues (NAT), 2 HGIN, and 6 GBC samples. The cellular diversity and heterogeneity, particularly within epithelial and immune cell populations in NAT–HGIN–GBC, were investigated utilizing single-cell RNA sequencing, bulk RNA sequencing (bulk RNA-seq), and 10 machine learning methodologies. Furthermore, the intercellular crosstalk between epithelial cells and tumor immune microenvironment cells was examined and validated through multiplex immunofluorescence staining. Results: The constructed cell atlas elucidated alterations in the immune landscape across various states of NAT–HGIN–GBC, highlighting a more pronounced inhibitory immune microenvironment in GBC. The epithelial subtype TOP2A+ Epi is markedly elevated in GBC and is correlated with a poor prognosis. Key genes associated with this subtype may include GMNN, CYTOR, KLK6, and BIRC5. Similarly, immunosuppressive macrophages, identified as TOP2A+ Macro, also increase along the NAT–HGIN–GBC sequence and are linked to reduced patient survival. Furthermore, TOP2A+ Macro and CD8+ exhausted T cells (CD8+ Tex) engage in intercellular communication with epithelial TOP2A+Epi cells via the TWEAK/FN14 signaling pathway, thereby promoting tumor progression and immune evasion in GBC. The findings were further corroborated through multiplex immunofluorescence staining conducted on specimens from patients. Conclusions: This study elucidates significant alteration in the cellular ecosystems and intercellular signaling within the tumor immune microenvironment across the NAT–HGIN–GBC sequence. It identifies TOP2A, TWEAK, and FN14 as potential biomarkers and therapeutic targets for GBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f8ecb5a2d68898a868d59ee319a91eaeca48bb" target='_blank'>
              Single-cell analyses unravel ecosystem dynamics and intercellular crosstalk during gallbladder cancer malignant transformation
              </a>
            </td>
          <td>
            Zhaobin He, Jianqiang Cao, Xiqiang Wang, Shengbiao Yang, Huijie Gao, Yongzhe Yu, Zequn Di, Cheng Peng
          </td>
          <td>2025-05-16</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14d1b8aeaba5a454b04d6690efe6420aea37c43e" target='_blank'>
              Stochastic modelling of prostate progenitor architecture
              </a>
            </td>
          <td>
            Christo Morison, Esther Baena, Weini Huang
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intra-tumor heterogeneity impacts disease progression and therapeutic resistance but remains poorly characterized by conventional histologic, immunophenotypic, and molecular approaches. Single-cell biophysical properties distinguish functional phenotypes complementary to these approaches, providing additional insight into cellular diversity. Here we link both buoyant mass and stiffness to gene expression to identify clinically relevant phenotypes within primary mantle cell lymphoma (MCL) cells, employing MCL as a model of biological and clinical diversity in human cancer. Linked measurements reveal that buoyant mass and stiffness characterize B-cell development states from naïve to plasma cell and correlate with expression of oncogenic B-cell receptor signaling genes such as BLK and CD79A. Additionally, changes in cell buoyant mass within primary patient specimens ex vivo correlate with sensitivity to Bruton’s Tyrosine Kinase inhibitors in vivo in MCL and chronic lymphocytic leukemia, another B-cell malignancy. These findings highlight the value of biophysical properties as biomarkers of response in pursuit of future precision therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/222e5440bee40a1cfc03b8b975d31d717ee09614" target='_blank'>
              Integrating Single-Cell Biophysical and Transcriptomic Features to Resolve Functional Heterogeneity in Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            Ye Zhang, Lydie Debaize, Adam Langenbucher, Jenalyn Weekes, Ioulia Vogiatzi, Teemu P. Miettinen, Mingzeng Zhang, Emily Sumpena, Huiyun Liu, Sarah M. Duquette, L. Hackett, Jeremy Zhang, Sona Baghiyan, R. Redd, M. Aryee, M. Davids, Austin I. Kim, C. Ryan, David M. Weinstock, S. Manalis, M. Murakami
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1ca3901ac698b79cb29b7a3967fd687d4414a1" target='_blank'>
              Multiomic profiling of hypoxic glioblastoma stem cells reveals expansion of subpopulations with distinct epigenetic and CNV profiles
              </a>
            </td>
          <td>
            Adrianne Corseri, Travis Moore, Nicole Szczepanski, Hyeyeon Hwang, Alan Zdon, Gurkan Yardimci Galip, Nikos Tapinos
          </td>
          <td>2025-05-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GB), particularly IDH-wildtype, is the most aggressive brain malignancy with a dismal prognosis. Despite advances in molecular profiling, the complexity of its tumor microenvironment and spatial organization remains poorly understood. This study aimed to create a comprehensive single-cell and spatial atlas of GB to unravel its cellular heterogeneity, spatial architecture, and clinical relevance.


METHODS
We integrated single-cell RNA sequencing data from 26 datasets, encompassing over 1.1 million cells from 240 patients, to construct GBmap, a harmonized single-cell atlas. High-resolution spatial transcriptomics was employed to map the spatial organization of GB tissues. We developed the Tumor Structure Score (TSS) to quantify tumor organization and correlated it with patient outcomes.


RESULTS
We showcase the applications of GBmap for reference mapping, transfer learning, and biological discoveries. GBmap revealed extensive cellular heterogeneity, identifying rare populations such as tumor-associated neutrophils and homeostatic microglia. Spatial analysis uncovered seven distinct tumor niches, with hypoxia-dependent niches strongly associated with poor prognosis. The TSS demonstrated that highly organized tumors, characterized by well-defined vasculature and hypoxic niches, correlated with worse survival outcomes.


CONCLUSIONS
This study provides a comprehensive resource for understanding glioblastoma heterogeneity and spatial organization. GBmap and the TSS provide an integrative view of tumor architecture in GB, highlighting hypoxia-driven niches that may represent avenues for further investigation. Our resource can facilitate exploratory analyses and hypothesis generation to better understand disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f160a16e8e12a7fc63680546f0557a806342875" target='_blank'>
              Charting the Single-Cell and Spatial Landscape of IDH-Wildtype Glioblastoma with GBmap.
              </a>
            </td>
          <td>
            Cristian Ruiz-Moreno, Sergio Marco Salas, Erik R Samuelsson, Mariia Minaeva, Ignacio L Ibarra, Marco Grillo, S. Brandner, Ananya Roy, K. Forsberg-Nilsson, M. Kranendonk, Fabian J. Theis, Mats Nilsson, H. Stunnenberg
          </td>
          <td>2025-05-02</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The effectiveness of colorectal cancer (CRC) therapy is limited owing to the absence of treatments targeting drug-tolerant residual cancer cells. Although neoadjuvant therapy is effective, pathological examination of residual tumors has revealed the presence of small clusters of LGR5-positive cancer cells in the fibrous tissue. Here, we established a colorectal cancer cell line-derived organoid (CCD-organoid) regrowth model using a patient-derived cell line with cancer stem cell properties and demonstrated that it displayed the morphological characteristics of small clusters in clinical tissues. Time course analysis of single-cell RNA sequencing of the CCD-organoid regrowth model revealed various states and dynamic alterations within non-cycling cells. We identified subpopulations highly expressing protein translation-related genes RPL17 and EEF1G. To identify key signals for the transition of residual cancer cells to regrowth, we evaluated inhibitors targeting pathways such as the Wnt pathway, reactive oxygen species pathway, and RNA polymerase I pathway, highlighted in the single-cell RNA sequencing analysis. Only the polymerase I-inhibitor BMH-21 significantly reduced tumor growth both in vitro and in vivo, indicating the critical cell subpopulation driving recurrence. Our results demonstrate the possibility of a unique therapeutic target for CRC treatment targeting drug-tolerant residual cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa9b58cf763b8a32b666aa1599214c1a009f1d2" target='_blank'>
              Colorectal cancer cell line-derived organoid model with stem cell properties captures the regrowing state of residual cancer cells after neoadjuvant chemotherapy
              </a>
            </td>
          <td>
            Kiyotaka Nakano, E. Oki, Masaki Yamazaki, Masami Suzuki, S. Kawai, T. Fujita, Atsuhiko Kato, Y. Zaitsu, Tomoko Jogo, Chie Kato, Takeshi Watanabe, Eri Hashimoto, Chiyoko Nishime, E. Fujii, K. Ando, G. Nagae, N. Harimoto, Mitsuhiko Ota, H. Saeki, Hiroyuki Aburatani, Yoshihiko Maehara, Tatsumi Yamazaki
          </td>
          <td>2025-06-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Fibroblasts play essential roles in cancer progression, exhibiting activation states that can either promote or inhibit tumor growth. Understanding these differential activation states is critical for targeting the tumor microenvironment (TME) in cancer therapy. However, traditional molecular markers used to identify cancer-associated fibroblasts are limited by their co-expression across multiple fibroblast subtypes, making it difficult to distinguish specific activation states. Morphological and motility characteristics of fibroblasts reflect their underlying gene expression patterns and activation states, making these features valuable descriptors of fibroblast behavior. This study proposes an artificial intelligence-based classification framework to identify and characterize differentially activated fibroblasts by analyzing their morphodynamic and motile features. We extract these features from label-free live-cell imaging data of fibroblasts co-cultured with breast cancer cell lines using deep learning and machine learning algorithms. Our findings show that morphodynamic and motile features offer robust insights into fibroblast activation states, complementing molecular markers and overcoming their limitations. This biophysical state-based cellular classification framework provides a novel, comprehensive approach for characterizing fibroblast activation, with significant potential for advancing our understanding of the TME and informing targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1693652b380d377e2983bb1084b665960d61177d" target='_blank'>
              Classification of differentially activated groups of fibroblasts using morphodynamic and motile features
              </a>
            </td>
          <td>
            Minwoo Kang, Chanhong Min, Somayadineshraj Devarasou, Jennifer H. Shin
          </td>
          <td>2025-05-15</td>
          <td>APL Bioengineering</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd1f3cec19d8b2ce70a881702528c91746b1cae" target='_blank'>
              CellLENS enables cross-domain information fusion for enhanced cell population delineation in single-cell spatial omics data.
              </a>
            </td>
          <td>
            Bokai Zhu, Sheng Gao, Shuxiao Chen, Yuchen Wang, Jason Yeung, Yunhao Bai, Amy Y. Huang, Y. Yeo, Guanrui Liao, Shulin Mao, Z. G. Jiang, Scott J Rodig, Ka-Chun Wong, Alex K. Shalek, GP Nolan, Sizun Jiang, Zongming Ma
          </td>
          <td>2025-05-22</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Presented here is an emerging DNA-barcode-based multiplex imaging technique based on Co-Detection-by-indEXing that analyzes the spatial proteomics of tissue microenvironments. Successful imaging requires a repertoire of well-designed and properly validated antibody panels, but very few currently exist for formalin-fixed paraffin-embedded (FFPE) samples. FFPE offers several advantages over fresh-frozen specimens, such as widespread availability, ease of handling and storage, and the ability to make tissue microarrays (TMAs). Here, we present a protocol to develop an antibody panel for visualizing and analyzing FFPE tissues from a murine melanoma model treated with nanoparticles, which deliver plasmid DNA encoding immunologic signals for tumor microenvironment reprogramming. We also describe an image analysis pipeline using open-source computational tools for annotating tissues, segmenting cells, processing proteomics data, phenotyping cell populations, and quantifying spatial metrics. The protocol offers applications for designing antibody panels in murine FFPE and generating novel insights into the spatial proteomics of complex tissue microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b66f89429f0aec5217bc5c1e61248f703eaaf39" target='_blank'>
              DNA-barcode-based Multiplex Immunofluorescence Imaging to Analyze FFPE Specimens from Genetically Reprogrammed Murine Melanoma.
              </a>
            </td>
          <td>
            S. Surwase, Xin Ming M. Zhou, K. Luly, Qingfeng Zhu, Niki Talebian, Robert A. Anders, Jordan J. Green, S. Tzeng, Joel C. Sunshine
          </td>
          <td>2025-06-06</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Pancreatic adenosquamous cancer (PASC) is an extremely rare subtype of pancreatic cancer characterized by a poorer prognosis and higher likelihood of metastasis compared with the more prevalent pancreatic ductal adenocarcinoma (PDAC). Although genomic changes during PASC tumorigenesis have been documented, the corresponding alterations in the tumor immune microenvironment (TIME) remain inadequately elucidated. Therefore, this study aims to analyze the immune landscape of PASC by employing multiplex immunohistochemistry (mIHC) and digital image analysis. Methods In this study, we analyzed four independent cohorts comprising 120 patients with PASC and 386 patients with PDAC. We employed mIHC to quantify three in situ panels of immuno-oncology-related biomarkers at subcellular resolution. We then used five samples to perform laser capture microdissection, RNA sequencing, and whole-exome sequencing to explore the underlying mechanisms of the compartment-specific immune phenotypes in PASC. Results Our findings revealed a more immunosuppressive TIME in PASC compared with PDAC, characterized by a decreased abundance of T cells. Immune cell types indicative of enhanced immune surveillance, including cytotoxic and memory T cells and antigen-experienced T cells, were present at significantly lower densities in PASC compared with PDAC. Conversely, some immunosuppressive macrophage phenotypes exhibited increased distribution in PASC. Immunosuppressive immune cells (ICs) were abundant, enriched within stromal regions, highly heterogeneous across tumors, and exhibited distinct distributions between squamous cell (SQC) and adenocarcinoma (ADC) compartments in PASC. Notably, the TIME of SQC compartments harbored more exhausted T cells compared with synchronous ADC compartments, indicating attenuated immune surveillance during squamous transformation. Transcriptomic profiling of microdissected SQC and ADC regions revealed immune exhaustion signatures and downregulated T-cell differentiation pathways in SQC compartments, alongside altered antigen presentation machinery and elevated tumor mutational burden, suggesting squamous-specific tumor-associated antigens with potential immunotherapeutic relevance. Beyond differences in IC density, we observed closer spatial proximity of CD45RO+ and PD-1+CD3+CD8+ T cells to tumor cells within 10, 20 and 30 µm ranges in PASC compared with PDAC, with variations by histological subregion. Furthermore, we found distinct expression patterns of the programmed cell death protein-1 (PD-1)/programmed death ligand 1 (PD-L1) and T-cell immunoreceptor with immunoglobulin and the ITIM domain (TIGIT)/CD155 axes in the PASC TIME associated with survival outcomes. Notably, TIGIT+CD8+ T cells and CD155+ CD68+macrophages, along with their proximity to tumor cells, served as independent prognostic indicators. These findings were validated in an independent cohort study. Conclusion Our study advances the understanding of PASC by providing updated insights into its immunoenvironmental features. These findings underscore the potential of targeting immune checkpoint pathways, particularly the TIGIT/CD155 and PD-1/PD-L1 axes, as a therapeutic strategy for PASC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d936da9d89fc333e1aeb534f40e042c28a5208e" target='_blank'>
              Attenuated immune surveillance during squamous cell transformation of pancreatic adenosquamous cancer defines new therapeutic opportunity for cancer interception
              </a>
            </td>
          <td>
            Xinyuan Chen, Shanyue Sun, Shuofeng Li, Shuangni Yu, Jie Chen, Xian-liang Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35658443146621f3cb5a98587b659629d87dcfdc" target='_blank'>
              Longitudinal multi-omic profiling uncovers immune escape and predictors of response in multiple myeloma
              </a>
            </td>
          <td>
            Denis Ohlstrom, William C. Pilcher, Marina E. Michaud, Chaitanya R. Acharya, Sarthak Satpathy, Edgar Gonzalez-Kozlova, R. Jayasinghe, Katherine E. Ferguson, Hope L. Mumme, Shivani Nanda, Yizhe Song, Sowmitri Karthikeya Siddhartha Mantrala, D. Karagkouni, Jessica Schulman, Nick Pabustan, Junia Vieira Dos Santos, Daniel W. Sherbenou, J. Keats, Alex Gout, S. Foltz, A. Laganà, T. Kourelis, Ravi Vij, M. Dhodapkar, D. Avigan, H. Cho, L. Baughn, Ajay Nooka, S. Lonial, Shaji K. Kumar, Mehmet K Samur, Ioannis S. Vlachos, Li Ding, Sacha Gnjatic, George Mulligan, Manoj K. Bhasin
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Quiescence (reversible cell-cycle arrest) and senescence (irreversible arrest) are challenging to distinguish due to a lack of specific biomarkers, yet both arise simultaneously after chemotherapy. While senescence suppresses tumors by limiting proliferation and recruiting the immune system, quiescent cancer cells evade future therapies and may resume proliferation. Here, we pair time-lapse imaging of cell-cycle dynamics with single-cell RNA-sequencing after etoposide treatment to differentiate these states, linking heterogeneous cell-cycle phenotypes to the transcriptomic landscape. We identify diverse senescent types (senotypes) and link them to two arrest pathways – a gradual path arising after a standard mitosis-to-G0 transition, and a rarer but direct path driven by a mitotic slip. Using pseudotime trajectory analysis, we find that senescent phenotypes begin to manifest early and gradually along the first trajectory, even in shallow quiescent cells. These data support a model wherein, following chemotherapy, quiescence and senescence exist on a continuum of cell-cycle withdrawal at a transcriptome-wide level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e448f12e8844e81bc7406643cd94547023bd4f4c" target='_blank'>
              Single-cell RNA sequencing reveals distinct senotypes and a quiescence-senescence continuum at the transcriptome level following chemotherapy
              </a>
            </td>
          <td>
            Brianna Fernandez, Victor J. Passanisi, Humza M. Ashraf, Sabrina L. Spencer
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac70776dbfeffce80a568047c1e5e8a2e5f399d9" target='_blank'>
              In vivo nuclear envelope adaptation during cell migration across confining embryonic tissue environments
              </a>
            </td>
          <td>
            H. Häkkinen, Soraya Villaseca, Zain Alhashem, Szymon Chomiczewski, Maxime Desevedavy, Solene Leleux, Atrin Hamidzadeh, Filomena Gallo, Dina El-Zohiry, Victor Petre, Elena Scarpa
          </td>
          <td>2025-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78beb6714803e0c6118158e2cf27d989400224a0" target='_blank'>
              Single Cell RNA Sequencing Reveals Gene Expression Continuums Along the Spatial Hierarchy of the Pulmonary Circulation
              </a>
            </td>
          <td>
            S. Sveiven, C. Knutsen, Fabio Zanini, David N Cornfield, C. Alvira
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Chondrosarcoma, a rare and heterogeneous malignant bone tumor, presents significant clinical challenges due to its complex molecular underpinnings and limited treatment options. In this study, we employ single-cell RNA sequencing (scRNA-seq) and bioinformatics analyses to delineate cell subtypes, decipher signaling networks, and identify gene expression patterns, thereby providing novel insights into potential therapeutic targets and their implications in cancer biology. Methods scRNA-seq was performed on both clinical and experimental chondrosarcoma samples. Dimensionality reduction techniques (UMAP/t-SNE) were used to cluster cell subtypes, followed by Gene Ontology (GO) and pathway analyses to elucidate their biological functions. Cell–cell interaction networks, including the MIF signaling network, were reconstructed to map intercellular communications. Pseudotime analysis charted differentiation trajectories, while machine learning models evaluated the classification accuracy of gene expression patterns. GSEA was conducted to identify state-specific differential expression profiles. Results Over ten distinct cell subtypes were identified, including endothelial cells, fibroblasts, and epithelial cells. Key signaling pathways, such as TGF-beta signaling, focal adhesion, and actin cytoskeleton regulation, were found to mediate intercellular interactions. The MIF signaling network underscored the critical roles of immune cells within the tumor microenvironment. Pseudotime analysis revealed dynamic differentiation states, while state-specific gene expression patterns emerged from GSEA. Machine learning models demonstrated robust classification performance across training and external validation datasets. Conclusions This comprehensive analysis uncovers the cellular heterogeneity and complex intercellular networks in chondrosarcoma, elucidating critical molecular pathways and identifying novel therapeutic targets. By integrating gene expression, signaling networks, and advanced computational methods, this study contributes to the broader understanding of cancer biology and highlights the potential for precision medicine strategies in treating chondrosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9ceffe30e0b675aea772415730fccdf8b446ff" target='_blank'>
              Multidimensional bioinformatics analysis of chondrosarcoma subtypes and TGF-β signaling networks using big data approaches
              </a>
            </td>
          <td>
            Shengke Li, Junteng Chen, Fuping He, Maosheng Wang, Jun Liu, Hui Xie
          </td>
          <td>2025-06-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c2fd63e082e26fba9172752b056b5282ca5810" target='_blank'>
              Barcoded viral tracing identifies immunosuppressive astrocyte-glioma interactions.
              </a>
            </td>
          <td>
            Brian M Andersen, Camilo Faust Akl, Michael A. Wheeler, Zhaorong Li, M. Diebold, Michael Kilian, Joseph M Rone, Aditya Misra, Jessica E. Kenison, Joon-Hyuk Lee, , C. M. Polonio, David Merrell, J. Weiss, Lillie Godinez, Gavin Piester, Tomer Illouz, Jessica J Ye, Arianna Ghia, Jazmin Martinez, Elizabeth N. Chung, Léna Srun, Daniel Farrenkopf, Lucas E. Flausino, Anton M Schüle, L. Sanmarco, Federico Giovannoni, Luca Fehrenbacher, M. Charabati, Cristina Gutiérrez-Vázquez, Margaret M Cusick, Prem Prabhakar, Connor C Bossi, Emily Lapinskas, Roni Nowarski, G. Getz, Keith L. Ligon, M. Prinz, E. Chiocca, David A. Reardon, Francisco J. Quintana
          </td>
          <td>2025-06-25</td>
          <td>Nature</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Solid tumors are spatially heterogeneous in their genetic, molecular, and cellular composition, but recent spatial profiling studies have mostly charted genetic and RNA variation in tumors separately. To leverage the potential of RNA to identify copy number alterations (CNAs), we develop SlideCNA, a computational tool to extract CNA signals from sparse spatial transcriptomics data with near single cellular resolution. SlideCNA uses expression-aware spatial binning to overcome sparsity limitations while maintaining spatial signal to recover CNA patterns. We test SlideCNA on simulated and real Slide-seq data of (metastatic) breast cancer and demonstrate its potential for spatial subclone detection. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03573-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/871ec83d7b32f772090863f7cf126ce76bd95862" target='_blank'>
              SlideCNA: spatial copy number alteration detection from Slide-seq-like spatial transcriptomics data
              </a>
            </td>
          <td>
            Diane Zhang, Åsa Segerstolpe, M. Slyper, Julia Waldman, Evan Murray, Robert Strasser, Jan Watter, Ofir Cohen, Orr Ashenberg, D. Abravanel, Judit Jané-Valbuena, Simon Mages, Ana Lako, K. Helvie, O. Rozenblatt-Rosen, Scott J Rodig, Fei Chen, Nikhil Wagle, Aviv Regev, Joanna Klughammer
          </td>
          <td>2025-05-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afbfafb6e752991df103b290fb3be622c923aa5e" target='_blank'>
              Immune Spatial Organization Predicts Metastasis Risk in Aggressive Localized Prostate Cancer
              </a>
            </td>
          <td>
            David D. Yang, Aya Abdelnaser, Jeremiah Wala, Alexander J Haas, Alfred A Barney, Eddy Saad, Jett Crowdis, Cora A. Ricker, Jihye Park, Martin T King, P. L. Nguyen, T. Choueiri, A. Tewari, Keyan Salari, M. Taplin, Chin-Lee Wu, E. V. Van Allen
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="The tumor microenvironment (TME) significantly influences cancer prognosis and therapeutic outcomes, yet its composition remains highly heterogeneous, and currently, no highly accessible, high-throughput method exists to define it. To address this complexity, the TMEclassifier, a machine-learning tool that classifies cancers into three distinct subtypes: immune Exclusive (IE), immune Suppressive (IS), and immune Activated (IA), is developed. Bulk RNA sequencing categorizes patient samples by TME subtype, and in vivo mouse model validates TME subtype differences and differential responses to immunotherapy. The IE subtype is marked by high stromal cell abundance, associated with aggressive cancer phenotypes. The IS subtype features myeloid-derived suppressor cell infiltration, intensifying immunosuppression. In contrast, the IA subtype, often linked to EBV/MSI, exhibits robust T-cell presence and improved immunotherapy response. Single-cell RNA sequencing is applied to explore TME cellular heterogeneity, and in vivo experiments demonstrate that targeting IL-1 counteracts immunosuppression of IS subtype and markedly improves its responsiveness to immunotherapy. TMEclassifier predictions are validated in this prospective gastric cancer cohort (TIMES-001) and other diverse cohorts. This classifier could effectively stratify patients, guiding personalized immunotherapeutic strategies to enhance precision and overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e59035a4145d3d05ce3897eefa444405fb68dd" target='_blank'>
              Immunotyping the Tumor Microenvironment Reveals Molecular Heterogeneity for Personalized Immunotherapy in Cancer.
              </a>
            </td>
          <td>
            D. Zeng, Yunfang Yu, Wenjun Qiu, Q. Ou, Qianqian Mao, Luyang Jiang, Jianhua Wu, Jiani Wu, Huiyan Luo, Peng Luo, Wenchao Gu, N. Huang, S. Zheng, Shaowei Li, Yonghong Lai, Xiatong Huang, Yiran Fang, Qiongzhi Zhao, R. Zhou, Huiying Sun, Wei Zhang, J. Bin, Yulin Liao, Masami Yamamoto, Tetsuya Tsukamoto, Sachiyo Nomura, Min Shi, Wangjun Liao
          </td>
          <td>2025-05-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 e13119


 Background:
 The tumor microenvironment (TME) of primary breast tumors (PTs) and breast-to-liver metastases shows heterogeneous infiltration and interactions between tumor, immune, and stromal cells. Hepatic metastases exhibit a more immunosuppressive TME with fewer cytotoxic T cells (CTLs) relative to PTs, which may contribute to increased tumor growth and resistance to systemic therapies. To investigate the immune cell infiltration and immune-tumor interactions in PT and liver metastases, we utilized whole transcriptome analysis and immunofluorescence-based protein expression as part of spatial TME profiling.
 Methods:
 We identified 13 chemo-naïve PTs and 3 breast-to-liver metastases. Formalin-fixed paraffin-embedded (FFPE) samples were used for hematoxylin and eosin (H&E) staining and spatial transcriptomics (ST) profiling with the 10X Genomics Visium CytAssist and Visium HD platforms. The H&E slides were manually annotated in collaboration with a board-certified pathologist, and ST data were deconvoluted using breast-specific scRNA-sequenced references and canonical marker labeling.
 Results:
 Our analysis of the 13 PTs and 3 breast-to-liver metastases revealed significant heterogeneity in tumor clusters, immune cell infiltration, gene set enrichments, and gene signatures between primary and visceral TMEs. Unsupervised clustering of hormone receptor-positive (HR+) HER2-negative (HER2) samples demonstrated considerable intratumoral and intertumoral variability, with differences in the expression of tumor markers (EPCAM, ER, PR, ERBB2, KRT14, KRT18), hypoxia-related signatures (HIF1a), and pro-angiogenic drivers (VEGF). Breast-to-liver metastases showed higher expression of pathways involved in proliferation, angiogenesis, and anti-apoptosis compared to primary tumors. Hepatic metastases were enriched for HALLMARK gene sets related to proliferation and metabolic pathways, including PI3K/AKT/mTOR, mTORC1 signaling, E2F targets, MYC targets, G2M checkpoint, mitotic spindle, glycolysis, oxidative phosphorylation, and fatty acid metabolism when compared to primary tumors. In contrast, primary tumors were enriched for immune-related pathways (TNF-α, interferon-γ, inflammatory response, IL-6 JAK/STAT3), epithelial-mesenchymal transition (EMT), myogenesis, and angiogenesis gene signatures, which are consistent with their metastatic potential. Cellular profiling of liver lesions revealed higher myeloid suppressor cells (TAMs and MDSCs): CD8
 +
 ratio with shorter cell-cell distances relative to PTs suggesting a key role in immune evasion, tumor progression, and therapeutic resistance.
 Conclusions:
 ST data from primary and metastatic breast cancer samples revealed notable tumor heterogeneity, characterized by highly active metabolic and proliferative pathways, along with a higher abundance of myeloid cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/389d6dee563fb2269c25d3de12654b70850dddf7" target='_blank'>
              Spatial transcriptomics and functional mapping of the suppressive TME in primary breast tumor and breast-to-liver metastases.
              </a>
            </td>
          <td>
            Batul Al-zubeidy, Edgar Gonzalez, A. Baugh, Dominic Zavala, Cheol Park, Matthew Jacobo, Marqus Dela Cruz, Michelle Li, Sabrina Carrel, Elexa Rallos, Michael Press, Evanthia T. Roussos Torres
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mitochondrial (MT) mutations serve as natural genetic markers for inferring clonal relationships using single cell sequencing data. However, the fundamental challenge of MT mutation-based lineage tracing is automated identification of informative MT mutations. Here, we introduced an open-source computational algorithm called “MitoTracer”, which accurately identified clonally informative MT mutations and inferred evolutionary lineage from scRNA-seq or scATAC-seq samples. We benchmarked MitoTracer using the ground-truth experimental lineage sequencing data and demonstrated its superior performance over the existing methods measured by high sensitivity and specificity. MitoTracer is compatible with multiple single cell sequencing platforms. Its application to a cancer evolution dataset revealed the genes related to primary BRAF-inhibitor resistance from scRNA-seq data of BRAF-mutated cancer cells. Overall, our work provided a valuable tool for capturing real informative MT mutations and tracing the lineages among cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b015ab9e9a2db2d9420366fceee8ad8d28fa830" target='_blank'>
              MitoTracer facilitates the identification of informative mitochondrial mutations for precise lineage reconstruction
              </a>
            </td>
          <td>
            Xuexin Yu, Jing Hu, Yuhao Tan, Mingyao Pan, Hongyi Zhang, Bo Li
          </td>
          <td>2025-06-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Objective To investigate the metabolic remodeling and functional specialization of immune cells within the tumor microenvironment (TME) of bone tumors, including Ewing’s sarcoma, osteosarcoma, and giant cell tumor of bone, through high-resolution single-cell RNA sequencing (scRNA-seq) analysis. Methods Immune cells were isolated from 13 bone tumor samples and profiled via scRNA-seq to delineate cellular compositions, metabolic adaptations, and intercellular communication networks. Differential gene expression analysis, metabolic pathway enrichment, and pseudotime trajectory inference were employed to characterize functional states and differentiation processes of immune cell subsets. Results We identified 12 major immune cell clusters with distinct functional and metabolic characteristics. Naïve T cells exhibited amino acid metabolism-dependent activation potential, whereas NK cells relied on lipid metabolism and the TCA cycle for cytotoxic activity. Macrophage subsets demonstrated functional divergence: C06 macrophages adopted lipid metabolism to facilitate immunosuppression and tissue repair, while C04 macrophages displayed pro-inflammatory characteristics associated with complement activation. Intercellular signaling analysis revealed FN1 as a central regulator of immune coordination, governing cell adhesion, migration, and homeostasis within the TME. Conclusion This study provides novel insights into the metabolic and functional plasticity of immune cells in bone tumor TMEs, underscoring the critical role of metabolic remodeling in immune regulation. Our findings highlight potential therapeutic targets for modulating immune cell function and offering new avenues to improve treatment outcomes for patients with bone tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06346-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e0d19e0f285642428880ab85bf21db7c2bbae62" target='_blank'>
              Single-cell transcriptomics reveals metabolic remodeling and functional specialization in the immune microenvironment of bone tumors
              </a>
            </td>
          <td>
            Jun Chen, Na Cui, Shaokai He, Chun-Yan Xia, Wei-Qing Li
          </td>
          <td>2025-05-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Resistance to chemotherapy of pancreatic ductal adenocarcinoma (PDAC) is largely driven by intratumoral heterogeneity (ITH) due to tumor cell plasticity and clonal diversity. In order to develop novel strategies to overcome this defined mechanism of resistance, tools to monitor and quantify ITH in a rapid and scalable fashion are needed urgently. Here, we employed label-free digital holographic microscopy (DHM) to characterize ITH in PDAC. We established a robust experimental and machine learning analysis pipeline to perform single cell phenotyping based on DHM-derived phase images of PDAC cells in suspension. Importantly, we are able to detect dynamic changes in tumor cell differentiation and heterogeneity of distinct PDAC subtypes upon induction of epithelial-to-mesenchymal transition and under treatment-imposed pressure in murine and patient-derived model systems. This platform allows us to assess phenotypic ITH in PDAC on a single cell level in real-time. Implementing this technology into the clinical workflow has the potential to fundamentally increase our understanding of tumor heterogeneity during evolution and treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b97958c7f42c79b39377ba5cf9b12106537163c" target='_blank'>
              Label-free single cell phenotyping to determine tumor cell heterogeneity in pancreatic cancer in real-time.
              </a>
            </td>
          <td>
            Katja Wittenzellner, M. Lengl, S. Röhrl, Carlo Maurer, C. Klenk, A. Papargyriou, Laura Schmidleitner, Nicole Kabella, Akul R Shastri, D. Fresacher, Farid Harb, Nawal Hafez, S. Bärthel, Daniele Lucarelli, Carmen Escorial-Iriarte, F. Orben, R. Öllinger, Ellen Emken, L. Fricke, Joanna Madej, Patrick Wustrow, I. Demir, Helmut Friess, Tobias Lahmer, Roland M. Schmid, Roland Rad, Günter Schneider, Bernhard Kuster, Dieter Saur, O. Hayden, Klaus Diepold, Maximilian Reichert
          </td>
          <td>2025-05-27</td>
          <td>JCI insight</td>
          <td>1</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac2fa84e64df92a9b47c54f45baf0d93294fea97" target='_blank'>
              Substrate heterogeneity promotes cancer cell dissemination
              </a>
            </td>
          <td>
            Zuzana Dunajova, Saren Tasciyan, Juraj Májek, J. Merrin, E. Sahai, M. Sixt, É. Hannezo
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="
 e20030


 Background:
 Tumoral heterogeneity has been widely acknowledged to influence clinical outcome of lung adenocarcinoma (LUAD). This study investigated spatial gene profiling of different pathological subtypes of early-stage invasive LUAD with Stereo-seq transcriptomics technique. The tumor microenvironment (TME) was decoded with cellular and molecular events at perspectives of tumoral activities and immune activities.
 Methods:
 We collected spatial RNA profiles of nine Stereo-seq chips (captured area of 1 x1 cm) from 4 invasive LUAD patients. Pathologist identified histological subtypes including lepidic adenocarcinoma (LP), acinar adenocarcinoma (AC) and other invasive adenocarcinoma like papillary adenocarcinoma (IA) on adjacent H&E images. Spatial data matrix was obtained based on pathological annotation accordingly. Spatially resolved gene expression of each subtypes was obtain based on pathological annotation to conduct high resolution analyses. TME functional units were calculated by analysis of cellular neighborhoods (CNs) based on cell compositions within resolution of bin 100 (window of 50um x 50um). Sub-populations of tumoral cells were stratified by molecular functions (GSEA and GSVA score) and stemness (CytoTRACE).
 Results:
 All pathological subtypes shared the identical 10 heterogeneous CNs and 5 tumoral clusters (TM) with various propotions. Tumor cluster 0 (TM0) and TM3 were identified as the malignant population with significant high stemness and significant high activities of MTORC1 signaling, TGF-BETA signaling, PI3K-ATK signaling and hypoxia. AC and IA were mainly composed of TM0 and TM3. Compared to TM0, TM3 had higher EMT activity. We further identified two immune-related CNs (CN4 and CN8) with significant high GSVA score of Interferon-Alpha response, Interferon-Gamma response and Inflammatory response. Spatial association showed that the lymphoid-cells-dominated CN4 was enriched in LP and IA and nearly absent in AC. Meanwhile, the myeloid-cells-dominated CN8 was enriched in AC. GSEA score showed that immune activity of CN4 was more intense in IA when compared to LP. Notably, TM0 mainly present in CN8, and TM3 mainly present in CN4. Therefore, the LP was speculated to reach an inert and statical status. TME of AC was featured as colocalization of myeloid cells and tumor cells with high stemness. TME of IA was featured as colocalization of lymphoid cells and tumor cells with high EMT activity.
 Conclusions:
 This is the first study to delineate the homogeneity of tumor cells and heterogeneity of immune cells among different pathological types of NSCLC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344f597f0ff9f98416e718719f81b290a2438fe0" target='_blank'>
              Decoding spatial transcriptomics-based heterogeneity of TME in early-stage invasive pulmonary adenocarcinoma.
              </a>
            </td>
          <td>
            Xin Liu, Jingyi Tian, Z. Shang, C. Xiang, Fei Teng, Jiajun Zhang, Yuchen Han
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="We are pleased to present this special section of Small Methods , which highlights the rapidly advancing ﬁelds of single-cell and spatial transcriptomics. Single-cell transcriptomics and spatial transcriptomics have emerged as transformative tools for high-resolution proﬁling of gene expression. Single-cell approaches reveal cellular diversity at unprecedented resolution, while spatial transcriptomics preserves the spatial context of gene activity, enabling precise mapping of tissue architecture. Together, these technologies provide complementary insights into biological systems, uncovering cellular heterogeneity, dynamic interactions, and spatially driven molecular processes across diverse ﬁelds including developmental biology, cancer biology, immunology, and neuroscience. This collection includes 3 reviews and 10 research articles from prominent scientists with their valuable insights. In this special section, we explore the transformative potential of spatial transcriptomics data analysis. Gao et al. (smtd. 2401451) introduce PASSAGE, a deep learning framework for identifying phenotype-associated signatures across heterogeneous spatial slices. Yang et al. (smtd.2400975) propose a novel cell segmentation method UCS, optimized for large-scale subcellular spatial transcriptomics data. Ishaque et al. (smtd. 2401123) present Sainsc, a cell-segmentation-free approach for transcriptome-wide, nanoscale-resolution spatial data. Yuan et al. (smtd.2401199) present a kernel-based strategy to model spatially continuous variations of the tissue microenvironment us-ing a new kernel-based strategyFei et al. (smtd.2401056) intro-S">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9aa8e9995463c05e1c6e24838fae6f8d6bcbd7" target='_blank'>
              Unraveling Tissue Complexity Through Single‐Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shiquan Sun, Chaoyong Yang, Lulu Shang, Rong Fan
          </td>
          <td>2025-05-01</td>
          <td>Small Methods</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9faa1273d62a3ded4dd4ed42a59e3c9f4c050ab5" target='_blank'>
              Spatial omics technology potentially promotes the progress of tumor immunotherapy.
              </a>
            </td>
          <td>
            Zhen Lan, Yuanyuan Yang, Lingling Li, Chaoguan Wang, Zhenqiang Sun, Qiming Wang, Yang Liu
          </td>
          <td>2025-06-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common malignancies worldwide, and its complex pathogenesis and significant tumor cell heterogeneity remain major challenges. With the rapid development of single-cell sequencing technology, we can now delve deeper into the cellular composition and dynamic changes within the tumor microenvironment, revealing cellular interactions and their potential roles in tumorigenesis. Method In this study, we systematically analyzed comprehensive single-cell RNA sequencing data from 25 colorectal cancer and 10 adjacent normal tissue samples. We explored the characteristics and biological significance of tumor cell subpopulations, performed quality control, dimensionality reduction, and cell type identification, and further investigated epithelial cell copy number variations, cell communication, and pseudotime analysis. Subsequently, Boruta feature selection algorithm was combined to identify prognosis related genes. The expression patterns, clinical significance and biological effects of PRSS22 were validated in vitro. Results Our analysis found an epithelial cell subcluster with high expression of PRSS22 exhibited high proliferation and migration abilities, and it was also associated with the dysregulated immune microenvironment. After further experimental verification, we proved the high expression patterns and clinical significance of PRSS22. Downregulation of PRSS22 in CRC cells resulted in a reduction of proliferation, migration and invasion. Conclusion Our study has identified a cell subcluster that is closely linked to progression, immune dysregulation and prognosis in CRC, and we have also identified PRSS22 as its hub gene that has great potential to become a new immunotherapeutic targets target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background The dynamics of single-nucleotide variants (SNVs) play a critical role in understanding tumor development, yet their influence on shaping tumor microenvironments remains largely unexplored. Spatial transcriptomics offers an opportunity to map SNVs within the tumor context, potentially uncovering new insights into tumor microenvironment dynamics. Results This study developed SpatialSNV for identifying effective SNVs across tumor sections using multiple spatial transcriptomics platforms. The analysis revealed that SNVs reflect regional tumor evolutionary traces and extend beyond RNA expression changes. The tumor margins exhibited a distinct mutational profile, with novel SNVs diminishing in a distance-dependent manner from the tumor boundary. These mutations were significantly linked to inflammatory and hypoxic microenvironments. Furthermore, spatially correlated SNV groups were identified, exhibiting distinct spatial patterns and implicating specific roles in tumor–immune system crosstalk. Among these, critical SNVs such as S100A11L40P in colorectal cancer were identified as tumor region–specific mutations. This mutation, located within exonic nonsynonymous regions, may produce neoantigens presented by HLAs, marking it as a potential therapeutic target. Conclusions SpatialSNV represents a promising framework for unraveling the mechanisms underlying tumor–immune crosstalk within the tumor microenvironment by leveraging spatial transcriptomics and SNV-based tissue domain characterization. This approach is designed to be scalable, integrative, and adaptable, making it accessible to researchers aiming to explore tumor heterogeneity and identify therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32eb52bad2a391e12368299093f1326ca08dc55e" target='_blank'>
              SpatialSNV: A novel method for identifying and analyzing spatially resolved SNVs in tumor microenvironments
              </a>
            </td>
          <td>
            Yi Liu, Fan Zhu, Xinxing Li, Xiangyu Guan, Yong Hou, Yu Feng, Xuan Dong, Young Li
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastatic prostate cancer (mPC) is a clinically and molecularly heterogeneous disease. While there is increasing recognition of diverse tumor phenotypes across patients, less is known about the molecular and phenotypic heterogeneity present within an individual. In this study, we aimed to define the patterns, extent, and consequences of inter- and intra-tumoral heterogeneity in lethal prostate cancer. By combining and integrating in situ tissue-based and sequencing approaches, we analyzed over 630 tumor samples from 52 mPC patients. Our efforts revealed phenotypic heterogeneity at the patient, metastasis, and cellular levels. We observed that intra-patient, inter-tumoral molecular subtype heterogeneity was common in mPC and showed associations with genomic and clinical features. Additionally, cellular proliferation rates varied within a given patient across molecular subtypes and anatomic sites. Single-cell sequencing studies revealed features of morphologically and molecularly divergent tumor cell populations within a single metastatic site. These data provide a deeper insight into the complex patterns of tumoral heterogeneity in mPC with implications for clinical management and the future development of diagnostic and therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb925628a7789a4c45633ccff7c788aadf4bd5ff" target='_blank'>
              Patterns of intra- and inter-tumor phenotypic heterogeneity in lethal prostate cancer.
              </a>
            </td>
          <td>
            Martine P. Roudier, Roman Gulati, Erolcan Sayar, Radhika A Patel, Micah Tratt, Helen Richards, Paloma Cejas, Miguel Muñoz Gomez, Xintao Qiu, Yingtian Xie, B. Hanratty, Samir Zaidi, Jimmy L. Zhao, M. Adil, Chitvan Mittal, Yibai Zhao, R. Dumpit, Ilsa Coleman, J. Low, Thomas W Persse, Patricia Galipeau, John K. Lee, Maria S Tretiakova, Meagan Chambers, F. Vakar‐López, Lawrence D. True, Marie E Perrone, Hung-Ming Lam, Lori Kollath, Chien-Kuang C. Ding, Stephanie Harmon, Heather H. Cheng, Evan Y. Yu, R. Montgomery, Jessica E. Hawley, Daniel W. Lin, Eva Corey, M. Schweizer, Manu Setty, G. Ha, Charles Sawyers, C. Morrissey, Henry Long, Peter S Nelson, Michael C. Haffner
          </td>
          <td>2025-06-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Abstract Background Pancreatic angiosarcoma is a rare and highly aggressive tumor originating from lymphatic or vascular endothelial cells, with poor prognosis and few effective treatments. In this study, we aimed to characterize the tumor ecosystem of metastatic pancreatic angiosarcoma, along with its potential treatment strategies. Methods Single-cell RNA-sequencing and bioinformatics analysis were performed on samples obtained from one patient, including at total of 16,841 cells from pancreatic angiosarcoma liver metastasis and adjacent normal liver tissue. Results Pancreatic angiosarcoma cells exhibited marked upregulation of nuclear factor kappa-B (NF-κB), hypoxia-inducible factor 1 (HIF-1), and myelocytomatosis oncogene (MYC) proto-oncogene signaling pathways, while presenting limited upregulation of actionable therapeutic targets except for cyclin-dependent kinase 4 (CDK4) and epidermal growth factor receptor. Several immune checkpoint genes, including cytotoxic T-lymphocyte-associated protein 4 (CTLA4), lymphocyte-activation gene 3 (LAG3), programmed cell death protein 1 (PDCD1), and cluster of differentiation 86 (CD86), were upregulated in tumor-infiltrating T cells, natural killer (NK) cells, and myeloid cells. Furthermore, intercellular interaction profiling demonstrated enhanced activity of the programmed death-ligand 1 (PD-L1) and CD86 signaling pathways within the tumor microenvironment. The gene-set scores of T/NK-cell exhaustion, regulatory T cell, and macrophage angiogenesis were significantly higher in tumor tissues compared with adjacent normal tissues. However, the phagocytosis scores of macrophages within the tumor-infiltrating region were significantly lower than those in the adjacent normal tissues. Conclusions Our findings outlined an immunosuppressive and angiogenic tumor ecosystem in pancreatic angiosarcoma liver metastasis, suggesting that pancreatic angiosarcoma may be insensitive to most targeted therapies. Conversely, immunotherapies targeting LAG3, PD-L1, and CD86 (e.g. isatuximab, Opdualag, and abatacept) and anti-angiogenic agents may be therapeutically effective and worthy of subsequent exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fffbe9dc01181c5c05c578e4d7eb55a7b801a4e" target='_blank'>
              Single-cell transcriptomic landscape indicates the potential role of immunotherapy in metastatic pancreatic angiosarcoma
              </a>
            </td>
          <td>
            Yizhen Yang, Luohai Chen, Man Liu, Xiaoxuan Lin, Sui Peng, Yubin Xie, Zhirong Zeng, Minhu Chen, Ning Zhang
          </td>
          <td>2025-06-16</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 11508


 Background:
 Leiomyosarcoma (LMS) is a smooth muscle-derived tumor with significant heterogeneity and limited treatment options for recurrent/metastatic disease. A lack of targetable driver mutations and prognostic and predictive biomarkers have hampered the care of patients with LMS. There is a great need to better understand the biology of LMS and develop novel therapeutics. Advances in single cell RNA sequencing (scRNA-seq) have allowed for better understanding of intratumoral heterogeneity in diverse cancer subtypes. However, tissue dissociation during this process leads to loss of spatial context. Spatial gene expression analysis builds upon scRNA-seq and has the potential to yield information about tissue organization, cell-cell interactions, niches, and cell states. To date there have been limited application of spatial transcriptomics to sarcoma.
 Methods:
 We performed single nucleus multiome (snRNA-seq and snATAC-seq) on a cohort of 16 primary, untreated LMS samples including 12 soft tissue (STLMS) and 4 uterine (ULMS) tumors. We then designed a custom 480-gene panel using the differentially expressed genes from clusters identified in snRNA-seq data to be able to identify spatial relationships between these clusters and to assess these clusters on a larger scale. We utilized the 10x Genomics Xenium platform. This was applied to LMS tissue microarrays (TMAs) comprising a total of 326 tissue cores from 127 unique patients. Matched primary and metastatic samples from the same patient were available for 33 patients.
 Results:
 Analysis of scRNAseq data identified 2 distinct subtypes: a dedifferentiated subtype with mesenchymal features (MES) and a differentiated subtype with enrichment of smooth muscle cell markers (SMC). Integration of chromatin accessibility data from snATACseq showed enrichment of nuclear factor I (NFI) transcription factor (TF) motifs in the MES and AP-1 motifs in the SMC group. Whole genome sequencing did not reveal an obvious genomic etiology for these subtypes. Spatial transcriptomics was able to identify these 2 subtypes in a larger cohort of tumors. Consistent with snRNAseq data, we find that most tumors had almost exclusively either MES or SMC cells. We assessed spatial relationships between these subtypes and infiltrating immune cells. This revealed an enrichment in immunosuppressive macrophages and exhausted T cells in MES tumors compared to SMC tumors. Analysis of matched primary and metastatic tumors demonstrated that the subtype (MES or SMC) generally remains consistent between different sites of disease.
 Conclusions:
 We identify 2 novel LMS subtypes (MES and SMC) driven by distinct TFs. Spatial transcriptomic analysis confirmed the presence of these 2 subtypes in a larger cohort and demonstrated that MES tumors are associated with a more immunosuppressive tumor microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/170994843a3a0b5e571cb27c29ebdaf0b414317f" target='_blank'>
              Spatial transcriptomic profiling from over 300 leiomyosarcoma samples.
              </a>
            </td>
          <td>
            Ryan A Denu, Zhao Zheng, Veena Kochat, Yingda Jiang, William I. Padron, D. Ingram, Khalida M Wani, Larissa A Meyer, P. Soliman, R. Ratan, Alexander J. Lazar, E. Keung, Kunal Rai, E. N. Nassif Haddad
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="To investigate the early, poorly understood events driving metastatic progression, we searched for the earliest detectable disseminated cancer cells (DCCs), also often referred to as disseminated tumor cells (DTCs), in sentinel lymph node (SLN) biopsies of 492 patients with stage I–III melanoma. Using micromanipulator-assisted isolation of rare DCCs, single-cell mRNA and DNA sequencing, codetection by indexing immunofluorescence imaging and survival analysis, we identified melanoma-associated chondroitin sulfate proteoglycan (MCSP)+ melanoma cells as metastasis founder cells (MFCs). We found that DCCs entering SLNs predominantly exhibited a transitory phenotype that, upon interferon-γ exposure triggered by CD8 T cells, dedifferentiated into a neural-crest-like phenotype. This was accompanied by increased production of small extracellular vesicles (sEVs) carrying the immunomodulatory proteins CD155 and CD276 but rarely programmed cell death protein 1 ligand 1. The sEVs suppressed CD8 T cell proliferation and function, facilitating colony formation. Targeting MCSP+ MFCs or their immune escape mechanisms could be key to curing melanoma early by preventing manifestation of metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a62c093eb484f4f584e506a0255fa28c4bd064b8" target='_blank'>
              MCSP+ metastasis founder cells activate immunosuppression early in human melanoma metastatic colonization
              </a>
            </td>
          <td>
            Severin Guetter, Courtney König, Huiqin Koerkel-Qu, Aleksandra Markiewicz, S. Scheitler, Marie Katzer, Mark Berneburg, Philipp Renner, Beatrix Cucuruz, Leonhard Guttenberger, Veronika Naimer, K. Weidele, S. Treitschke, C. Werno, Hanna Jaser, Tonia Bargmann, Armin Braun, Florian Weber, Katja Evert, Alexander Rochwarger, Christian M. Schürch, Katharina Limm, Peter J Oefner, Reinhard Rachel, Felix Baumann, J. Warfsmann, Lisa Schmidleithner, Kathrin Guetter, Parvaneh Mohammadi, Anja Ulmer, S. Haferkamp, Chistoph A Klein, M. Werner-Klein
          </td>
          <td>2025-05-16</td>
          <td>Nature Cancer</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2702678fe4bf9d99269192e19243ce5bd9b25ff1" target='_blank'>
              Patient-Derived Three-Dimensional Lung Tumor Models to Evaluate Response to Immunotherapy
              </a>
            </td>
          <td>
            Kayla F. Goliwas, Kenneth P. Hough, Sruti Sivan, Sierra L. Single, Sameer S. Deshmukh, Joel L. Berry, Maya Khalil, Benjamin Wei, Yanis Boumber, Mohammad Athar, Aakash Desai, S. Ponnazhagan, James M. Donahue, J. Deshane
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Pancreatic cancer remains as one of the most challenging malignancies to diagnose and treat due to the late development of symptoms and limited early diagnostic options. Intraductal papillary mucinous neoplasms (IPMNs) are non-invasive precursors to invasive pancreatic ductal adenocarcinoma (PDAC)and an understanding of the changes in patterns of protein expression that accompany the progression from normal ductal (ND) cell, to IPMN to PDAC may provide avenues for improved earlier detection. In this study, we present an optimized spatial tissue proteomics workflow, termed SP-Max (Spatial Proteomics Optimized for Maximum Sensitivity and Reproducibility in Minimal Sample), designed to maximize protein recovery and quantification from limited laser micro dissected (LMD) samples. Our workflow enabled the identification of more than 6,000 proteins and the quantification of over 5,200 protein groups from FFPE tissue contours of pancreatic tissues. Comparative analyses across ND, IPMN, and PDAC revealed critical molecular differences in protein pathways and potential markers of progression. SP-Max provides a systematic, reproducible approach that significantly enhances our ability to study precancerous lesions and cancer progression in pancreatic tissues at unprecedented resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d82c5add7d35195aeb6805be33f9404740748724" target='_blank'>
              High-Sensitive Spatial Proteomics for Pancreatic Cancer Progression Analysis
              </a>
            </td>
          <td>
            Jongmin Woo, Zhenyu Sun, Yingwei Hu, Trung Hoàng, Decapite Christine, Smith Katelyn, Singhi Aatur, Randall Brand, Daniel W. Chan, Qing Kay Li, Ralph H. Hruban, Hui Zhang
          </td>
          <td>2025-05-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="This study explores the inflammatory response observed in pancreata and pancreatic lymph node (pLN) samples obtained throughout the natural history of type 1 diabetes (T1D) including non-diabetic individuals and non-diabetic autoantibody positive individuals with high susceptibility using spatial transcriptomics (ST). Integration of ST with public single-cell RNA sequencing data enabled interrogation of transcriptional alterations in T1D pathogenesis across both tissues and cellular scales. In the T1D pancreas, we observed global upregulation of multiple inflammation-associated transcripts, including regenerating islet-derived (REG) family genes, complement factor 3 (C3), SOD2, and OLFM4, and highlighted cellular candidates potentially contributing to these signatures. Within the T1D pLN, we observed spatially restricted upregulation of lymphotoxin-β (LTB) alongside follicular dendritic cell (FDC)-associated transcripts including FDCSP, CLU, and FCER2. Collectively, these findings highlight distinct inflammation signatures in the pancreas and regional pLN which can help inform the development of future therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d86fa210fd0a0e42f81c1f4633973c64cff969b8" target='_blank'>
              Spatial transcriptomics from pancreas and local draining lymph node tissue reveals a lymphotoxin-β signature in human type 1 diabetes
              </a>
            </td>
          <td>
            Miguel A. Medina-Serpas, Maigan A. Brusko, Gregory J. Golden, Martha Campbell-Thompson, Trevor Rogers, A. Posgai, Eline T. Luning Prak, Chengyang Liu, Klaus H. Kaestner, Ali Naji, Michael R. Betts, Lauren M McIntyre, M. Atkinson, Todd M. Brusko
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2373b7e16ed6b42c705e005f4504fde1168b7ff8" target='_blank'>
              Do PGCCs in Solid Tumors Appear Due to Treatment-related Stress or Clonal Expansion of CSCs that Survive Oncotherapy?
              </a>
            </td>
          <td>
            D. Bhartiya, Nripen Sharma, Anish Tripathi, Ashish Tripathi
          </td>
          <td>2025-05-08</td>
          <td>Stem cell reviews and reports</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86e8972816206093759eba24f787234eeba5b95a" target='_blank'>
              Analysis of Microsatellite Instability Intensity in Single-Cell Resolution with scMnT reveals Tumor Heterogeneity in Colorectal Cancer
              </a>
            </td>
          <td>
            Gyumin Park, Jihwan Park
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/104badcf54cd0ff8c28f3f2bf4d66c298f91b54a" target='_blank'>
              CSFeatures improves identification of cell type-specific differential features in single-cell and spatial omics data
              </a>
            </td>
          <td>
            Rufeng Li, Yongkang Li, Heyang Hua, Zhen Li, Yu Zhang, Yufei Yao, Shengquan Chen, Yungang Xu
          </td>
          <td>2025-05-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumor microenvironment is heterogeneous, structurally complex, and continually evolving, making it difficult to fully capture. Common dissociative techniques thoroughly characterize the heterogeneity of cellular populations but lack structural context. The recent boom in spatial analyses has exponentially accelerated our understanding of the structural complexity of these cellular populations. However, to understand the dynamics of cancer pathogenesis, we must assess this heterogeneity across space and time. In this review, we provide an overview of current dissociative, spatial, and temporal analysis strategies in addition to existing and prospective spatiotemporal techniques to illustrate how understanding the tumor microenvironment, focusing on dynamic immune-cancer cell interactions, across four dimensions will advance cancer research and its diagnostic and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1f3339cce25c83c98dd45cea90c67e2a48bc606" target='_blank'>
              The tumor microenvironment across four dimensions: assessing space and time in cancer biology
              </a>
            </td>
          <td>
            Christina R. Larson, Ayushi Mandloi, Satwik Acharyya, Julienne L. Carstens
          </td>
          <td>2025-06-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Angiosarcoma, a rare and highly aggressive malignancy originating from vascular endothelial cells, is characterized by its rapid progression, high invasiveness, and poor prognosis. Due to the limited understanding of its tumor microenvironment (TME) and the absence of effective treatments, further research is essential to elucidate its pathogenic mechanisms and improve therapeutic strategies. Objective This study aims to characterize the cellular heterogeneity and unique TME of primary breast angiosarcoma using single-cell RNA sequencing (scRNA-seq), to identify potential therapeutic targets and improve clinical outcomes. Methods Tumor samples were obtained from a patient with bilateral primary breast angiosarcoma and two patients with invasive breast cancer. Single-cell RNA sequencing (scRNA-seq) was conducted to capture the transcriptomic profiles of individual cells within the tumor samples. Following stringent quality control, a total of 31,771 cells were analyzed using comprehensive bioinformatics approaches. Cell populations were identified and classified into distinct cell types, and differential gene expression analysis was performed to explore key signaling pathways. Functional enrichment analysis was used to identify pathways related to tumor progression and immune evasion. Additionally, cell-cell communication networks were mapped to understand interactions within the TME, with a focus on pathways that may serve as therapeutic targets. Results The scRNA-seq analysis revealed significant differences in the distribution of perivascular cells, fibroblasts, T cells, endothelial cells, and myeloid cells in breast angiosarcoma compared to invasive breast cancer. Key pathways enriched in angiosarcoma samples included growth factor binding, platelet-derived growth factor binding, and ribosome biogenesis, with abnormal expression of several ribosomal proteins. Notably, genes such as FAT4, KDR, FN1, and KIT were highly expressed in angiosarcoma endothelial cells, correlating with poor prognosis. Cell communication analysis highlighted the CXCL12-CXCR4 axis as a crucial mediator of the TME in angiosarcoma. Conclusion This study provides critical insights into the TME of primary breast angiosarcoma, highlighting potential molecular targets and pathways for therapeutic intervention. These findings may inform the development of more effective treatment strategies for this rare and challenging tumor type. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02022-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61afd778df17e939f7a978e09a799d8e740668a2" target='_blank'>
              Dissecting the tumor microenvironment in primary breast angiosarcoma: insights from single-cell RNA sequencing
              </a>
            </td>
          <td>
            Peikai Ding, Shengbin Pei, Yi Zhai, Zheng Qu, Yazhe Yang, Xiaolong Feng, Qiang Liu, Xiangyu Wang, Wenxiang Zhang, Zhongzhao Wang, Xiangyi Kong, Jing Wang, Yi Fang
          </td>
          <td>2025-06-05</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61913974e99e15ccb9e61a055d93cfcb623fd1c0" target='_blank'>
              Matrix structure and microenvironment dynamics correlate with chemotherapy response in ovarian cancer
              </a>
            </td>
          <td>
            Florian Laforêts, Panoraia Kotantaki, Samar Elorbany, Joseph J Hartlebury, Joash D Joy, Beatrice Malacrida, Rachel C Bryan-Ravenscroft, Chiara Berlato, Erica Di Federico, John F Marshall, R. Manchanda, Wolfgang Jarolimek, Lara Perryman, E. Maniati, Frances R Balkwill
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a biologically heterogeneous malignancy of clonal plasma cells, often progressing from MGUS or smoldering MM. It causes anemia, bone lesions, and immune dysfunction due to abnormal plasma cell expansion in the bone marrow. Neuroinflammatory and neurotrophic factors may influence MM progression by affecting immune cells and the bone marrow niche. Growing evidence points to a role for neuroimmune regulation in tumor immunity. Despite therapeutic progress, disease heterogeneity and resistance highlight the need for new strategies targeting the tumor microenvironment and neuroimmune axis. Methods This investigation exploited single-cell RNA sequencing (scRNA-seq) to analyze MM and high-risk smoldering multiple myeloma (SMMh) samples, identifying 11 distinct cell types. We examined their transcriptional signatures, stemness, proliferative properties, and metabolic pathways, with particular attention to neuroimmune interactions in the tumor microenvironment. Using trajectory inference tools such as CytoTRACE, Monocle2, and Slingshot, we traced the differentiation paths of MM cell subpopulations and identified key signaling pathways that may influence immune responses and tumor progression. Results The analysis identified four distinct subpopulations of myeloma cells, with the C0 IGLC3+ myeloma cells representing the least differentiated and most proliferative subset. These cells played a critical role in MM progression and may contribute to immune evasion mechanisms. Additionally, receptor-ligand interactions within the tumor microenvironment were identified, which may be influenced by neuroinflammatory and neurotrophic factors. These findings suggest that the nervous system and immune modulation significantly affect tumor biology, highlighting potential therapeutic targets that could be exploited to overcome resistance to conventional therapies. Conclusion This single-cell analysis provided new insights into the cellular diversity and differentiation trajectories in MM, offering a deeper understanding of the complex neuroimmune interactions that drive tumor progression and resistance. By incorporating the role of neuroinflammation and immune modulation, our study suggested novel therapeutic strategies targeting the neuroimmune axis in oncology, ultimately contributing to the development of more effective, personalized treatment approaches for MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb42682593bbacba35775b457e4d73a57648a9" target='_blank'>
              Decoding multiple myeloma: single-cell insights into tumor heterogeneity, immune dynamics, and disease progression
              </a>
            </td>
          <td>
            Zhenzhen Zhao, Zhijie Zhao, Zhiheng Lin, Lu Fan, Zhikai Xiahou, Yujiang Dong, Weiying Bao
          </td>
          <td>2025-05-08</td>
          <td>Frontiers in Immunology</td>
          <td>3</td>
          <td>9</td>
        </tr>

        <tr id="Background Breast cancer progression and metastasis involve the action of multiple transcription factors in tumors and in the cells of the tumor microenvironment (TME) and understanding how these transcription factors are coordinated can guide novel therapeutic strategies. Myocardin-related transcription factors A and B (MRTFA/B also known as MKL1/2) are two related transcription factors that redundantly control cancer cell invasion and metastasis in mouse models of breast cancer, but their roles in human cancer are incompletely understood. Here, we investigated the expression and activation of these transcription factors to better assess their tumorigenic and metastatic impact on breast cancer and cells of the tumor microenvironment. Methods We used a multiplexed immunofluorescence approach to label MRTFA, MRTFB, tumor cells by using pan Cytokeratin, endothelial cells by using CD31, and antigen presenting cells (APCs) by using HLA-DRA on two different breast cancer tissue microarrays (TMA): The breast cancer progression TMA provided by the Cooperative Human Tissue Network (CHTN_BrCaProg3) and the University of Illinois Breast Cancer Working Group (TMA BCWG UIC-001-TMA) that included primary tumor and lymph node metastases from patients residing in the West Side and South Side of Chicago. We also used bioinformatics analyses of the TCGA and METABRIC databases and the Broad Institute’s single-cell RNA sequencing portal to investigate MRTFA/B expression patterns in the cells of the tumor microenvironment (TME). Results We found that in human tumors, MRTFA/B are concurrently activated in cancer cells, but they show distinct patterns of expression across different histological subtypes and in the cells of the TME. Importantly, MRTFA expression was elevated in metastatic tumors of African American patients, who disproportionately die from breast cancer. Interestingly, in contrast to publicly available mRNA expression data, MRTFA was similarly expressed across estrogen receptor (ER) positive and negative breast tumors, while MRTFB expression was highest in ER+ breast tumors. Furthermore, MRTFA was specifically expressed in the perivascular antigen-presenting cells (APCs), which has been previously associated with immune suppression and breast cancer progression. We also found that MRTFA expression correlated with the expression of the immune checkpoint protein V-set immunoregulatory receptor (VSIR) in the TCGA data and found that MRTFA activity promotes VSIR expression in THP-1 monocytes and cultured HEK293 cells. Conclusions Our results provide unique insights into how MRTFA and MRTFB promote metastasis in human cancer, the differences of their expression patterns, and their immune suppressive function within the breast cancer TME. Our results will guide future studies on targeting MRTFA/B transcriptional activity and the resulting immune suppression in breast cancer. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06559-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9b888786c13a4f36a60caad28e0f780e4db8c77" target='_blank'>
              Multiplex imaging reveals novel patterns of MRTFA/B activation in the breast cancer microenvironment
              </a>
            </td>
          <td>
            Stephanie M. Wilk, Kihak Lee, Caitlyn C. Castillo, Mohamed Haloul, Alexa M. Gajda, Virgilia Macias, Elizabeth L. Wiley, Zhengjia Chen, Xinyi Liu, Xiaowei Wang, Maria Sverdlov, Kent F Hoskins, E. E. Er
          </td>
          <td>2025-05-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Introduction Medullary thyroid carcinoma (MTC) is a rare and aggressive thyroid cancer with a challenging prognosis. While the immune microenvironment plays a crucial role in cancer progression, its role in MTC remains underexplored compared to more common thyroid cancers. Methods this study investigates the immune landscape of MTC by systematically evaluating immune cell infiltration and expression of immune markers across various tissue topographies. We utilized advanced immunohistochemical techniques to analyze tissue samples from 24 MTC patients, focusing on the tumor core, interface with healthy tissue, adjacent normal thyroid tissue, and lymph node metastases. Results our findings reveal a distinct immune profile with increased CD3+, CD4+, CD8+ and CD20+ lymphocytes in normal tissues adjacent to tumors and a notable presence of granzyme B+ cells in the tumor interface, particularly in patients with structural disease. Additionally, we observed a significant enrichment of mast cells in metastatic tissues. Discussion these results highlight the complex and spatially dependent immune landscape of MTC, suggesting implications for targeted immunotherapy. This study provides novel insights into the immune microenvironment of MTC and emphasizes the need for further research to elucidate its impact on disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8ab2d28852a497483bb6cba642f6c18c52966ea" target='_blank'>
              Spatial immune profiling reveals distinct microenvironments in medullary thyroid carcinoma
              </a>
            </td>
          <td>
            Maria Eduarda de Castro, Gustavo Forlin de Siqueira, Jean Ferrante Mariano, M. Kizys, L. Ceolin, Fernando Augusto Soares, Rodrigo Natal, Humberto Carneiro, Rodrigo Nalio Ramos, Laura Sterian Ward, Niels Olsen Saraiva Camara, Cleber Pinto Camacho, Flavia de Oliveira Facuri Valente, Susan C. Lindsey, Diego Dias dos Santos, Cristiane Damas Gil, João Roberto Maciel Martins, Rui Monteiro de Barros Maciel, L. L. Cunha
          </td>
          <td>2025-05-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2fc80562262fc23fc64cea425e831c1c998be8" target='_blank'>
              Functional Spatial Mapping of the Tumour Immune Microenvironment In Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Sam Legg, Emma Wagner, Christopher J. Applebee, Amanda R. Kirane, Julian Padget, Banafshé Larijani
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cervical cancer represents a significant global health challenge, with complex cellular and molecular mechanisms driving its progression from HPV infection to invasive malignancy. This study employed an integrated approach combining single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (stRNA-seq) to comprehensively characterize the tumor microenvironment (TME) across different stages of cervical cancer development. Through analysis of samples from normal cervix, HPV-infected normal cervix, high-grade squamous intraepithelial lesions (HSIL), and invasive cervical cancer, we identified distinct cellular populations and their dynamic changes during disease progression. Our findings revealed significant heterogeneity in immune cell populations, particularly highlighting the role of SPP1+ macrophages that were substantially enriched in cervical cancer compared to precancerous and normal tissues. Cell-cell communication networks and spatial mapping demonstrated that SPP1+ macrophages interact extensively with immune cells through the SPP1-CD44 signaling axis. This interaction contributes to an immunosuppressive microenvironment through modulation of T cell function and promotion of tumor cell survival. Furthermore, high expression of SPP1 correlated with advanced tumor stages and poor overall survival in cervical cancer patients, highlighting its potential as a prognostic biomarker. Our comprehensive characterization of the cellular landscape and intercellular communication networks in cervical cancer progression provides valuable insights for the development of targeted therapeutic strategies aimed at modulating the TME, particularly through disruption of the SPP1-CD44 axis. These findings establish a foundation for more effective personalized approaches to improve clinical outcomes in cervical cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87cbb4dd12be6f1664aacd1df3e0b1efa6fe30d7" target='_blank'>
              Deciphering the cellular and molecular landscape of cervical cancer progression through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Peng Xia, Juanhong Zhou, Rong Shen, Degui Wang
          </td>
          <td>2025-05-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Basic and translational cancer biology research requires model systems that recapitulate the features of human tumors. While two-dimensional (2D) cell cultures have been foundational and allowed critical advances, they lack the organizational complexity, cellular interactions, and extracellular matrix present in vivo. Mouse models have thus remained the gold standard for studying cancer. In addition to high cost and low throughput, mouse models can also suffer from reduced tumor heterogeneity and species-specific differences. Three-dimensional (3D) culture models have emerged as a key intermediary between 2D cell lines and mouse models, with lower cost and greater flexibility than mouse models and a more accurate representation of the tumor microenvironment than 2D cell lines. In neuroblastoma, an aggressive childhood cancer, 3D models have been applied to study drug responses, cell motility, and tumor–matrix interactions. Recent advances include the integration of immune cells for immunotherapy studies, mesenchymal stromal cells for tumor–stroma interactions, and bioprinted systems to manipulate matrix properties. This review examines the use of 3D culture systems in neuroblastoma, highlighting their advantages and limitations while emphasizing their potential to bridge gaps between in vitro, preclinical, and clinical applications. By improving our understanding of neuroblastoma biology, 3D models hold promise for advancing therapeutic strategies and outcomes in this childhood cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f6d28651a5e98e6b7e3247c16a4fa5871e08763" target='_blank'>
              Three-Dimensional Culture Systems in Neuroblastoma Research
              </a>
            </td>
          <td>
            Piotr Jung, A. Wolpaw
          </td>
          <td>2025-05-08</td>
          <td>Organoids</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="One of the key challenges in defeating advanced tumors is the ability of cancer cells to evade the selective pressure imposed by chemotherapy, targeted therapies, immunotherapy and cellular therapies. Both genetic and epigenetic alterations contribute to the development of resistance, allowing cancer cells to survive initially effective treatments. In this narration, we explore how genetic and epigenetic regulatory mechanisms influence the state of tumor cells and their responsiveness to different therapeutic strategies. We further propose that an altered balance between cell growth and cell death is a fundamental driver of drug resistance. Cell death programs exist in various forms, shaped by cell type, triggering factors, and microenvironmental conditions. These processes are governed by temporal and spatial constraints and appear to be more heterogeneous than previously understood. To capture the intricate interplay between death-inducing signals and survival mechanisms, we introduce the concept of Death-ision. This framework highlights the dynamic nature of cell death regulation, determining whether specific cancer cell clones evade or succumb to therapy. Building on this understanding offers promising strategies to counteract resistant clones and enhance therapeutic efficacy. For instance, combining DNMT inhibitors with immune checkpoint blockade may counteract YAP1-driven resistance or the use of transcriptional CDK inhibitors could prevent or overcome chemotherapy resistance. Death-ision aims to provide a deeper understanding of the diversity and evolution of cell death programs, not only at diagnosis but also throughout disease progression and treatment adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24390bbe04bc2b2ed371e1dae568af6c9f084554" target='_blank'>
              Death-ision: the link between cellular resilience and cancer resistance to treatments
              </a>
            </td>
          <td>
            Gustavo Baldassarre, Ivana L. de la Serna, François M Vallette
          </td>
          <td>2025-05-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dc9b5c052f897d56a8e0a52da8575d2a449f473" target='_blank'>
              DR5 CAR-T cells target solid tumors and suppress MDSCs with minimal toxicity
              </a>
            </td>
          <td>
            Huaishan Wang, Shujing Liu, Prithvi Sinha, Fang Liu, Xiaogang Zhang, Yeye Guo, Beatriz Goncalves, Qiuxian Zheng, Haiwei Mou, Jingbo Yang, Lili Huang, John Scholler, Fei Miao, Tingting Zeng, G. Karakousis, Alexander C Huang, Tara C. Mitchell, Ravi Amaravadi, Lynn M Schuchter, M. Milone, Wei Guo, Carl H. June, M. Herlyn, Yi Fan, Xiaowei Xu
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="The gastric mucosa is characterized by continuous innate immune surveillance and inflammatory signaling, yet a high proportion of gastric carcinomas (GCs) are recalcitrant to immune-directed therapies. The mechanisms by which GCs evade adaptive immune surveillance within the highly antigenic microenvironment of the gastric mucosa remains unknown. To address this, we collected patient-matched tumor tissue, distant normal tissue, metastasis, and draining lymph nodes to generate a large-scale single-cell immune profiling dataset from 64 patients (n=179 samples, >150,000 cells). From single cell analysis, we identified two distinct sources of impaired tumor surveillance within tumor draining lymph nodes. First, we observed that a significant fraction of tumor draining lymph nodes had undergone cytokine-driven reprogramming, leading to reduced dendritic cell homing and limited T cell priming. Second, T cells undergoing successful activation exhibited limited expansion and constrained differentiation, marked by expression of the quiescence-associated transcription factor Kruppel-like Factor 2 (KLF2). Overexpression of KLF2 in primary T cells limited both their differentiation and cytotoxic capacity. These findings implicate both impaired T cell priming and KLF2-dependent T cell quiescence in limiting T cell immunity in gastric adenocarcinoma. We suggest these findings represent an emerging model for immune silencing in tumors developing from tissues with chronic inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6466661e6877ba6916df85de1c24be3277d5306" target='_blank'>
              Disrupted priming within draining lymph nodes drives immune quiescence in gastric cancer
              </a>
            </td>
          <td>
            Sohrab Salehi, Emily Stroobant, Hannah Lees, Ya-Hui Lin, Shoji Shimada, Miseker Abate, Matthew Zatzman, Nicholas Ceglia, Samuel S. Freeman, Monika Laszkowska, S. Maron, Andrew McPherson, Nicole Rusk, Eliyahu Havasov, Harrison Drebin, Ping Gu, Laura H. Tang, Y. Janjigian, Ruslan Soldatov, R. Chaligné, Sohrab P. Shah, V. Strong, S. Vardhana
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3294fa089aae1c04ac4218c49906b4bc0f6ea71" target='_blank'>
              NF2 loss malignantly transforms human pancreatic acinar cells and enhances cell fitness under environmental stress
              </a>
            </td>
          <td>
            Yi Xu, Michael H Nipper, Angel A. Dominguez, Chenhui He, Francis E Sharkey, Sajid Khan, Han Xu, Daohong Zhou, Lei Zheng, Yu Luan, Jun Liu, Pei Wang
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The tumor microenvironment plays a crucial role in shaping the tumor phenotype, yet replicating its complexity in vitro remains challenging. Traditional two-dimensional culture models lack physiologic relevance, and three-dimensional models often fail to fully capture native extracellular matrix (ECM) composition and cellular heterogeneity. In this issue of Cancer Research, Buckenmeyer and colleagues bridged this gap by integrating decellularized porcine-derived small intestinal submucosa ECM into monocellular spheroids, resulting in the formation of "MatriSpheres" with in vivo-like cancer cell heterogeneity. Although small intestinal submucosa ECM proved to be beneficial, several routes remain unexplored, such as incorporating other cell types (immune cells and cancer-associated fibroblasts) or species-, organ- or pathology-specific ECM or leveraging patient-derived tumor material. Pursuing these avenues of investigation to further develop Self-Matrix-Assembly to Recapitulate a Tumor (SMART) three-dimensional models could provide a powerful tool for cancer research, drug discovery, and personalized medicine. See related article by Buckenmeyer et al., p. 1577.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c194b106b13ab4f43c268e1b9699aab011f64" target='_blank'>
              Launching SMART 3D Cancer Models.
              </a>
            </td>
          <td>
            Sam Ernst, O. De Wever
          </td>
          <td>2025-05-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Retinoblastoma is a rare pediatric eye cancer caused by mutations in the RB1 gene, which regulates retinal cell growth. Early detection and treatment are critical for preventing vision loss and improving survival outcomes. This study aimed to perform an integrated proteotranscriptomic characterization of human retinoblastoma to provide a deeper understanding of disease biology and to identify novel therapeutic targets. Methods Paired tumor and adjacent retinal tissue samples were dissected from seven eyes. RNA sequencing and liquid chromatography-mass spectrometry were performed on the same samples. The spatially resolved cellular landscape was assessed using Imaging Mass Cytometry (IMC). Results The correlation between RNA and protein level was moderate with variations across different pathways, underscoring the value of an integrated proteotranscriptomic approach. IMC identified more than 67,000 single cells in 11 distinct clusters, including antigen presenting cells, T cells, stroma cells, vascular cells and two clusters of proliferating and CD44/c-Myc positive tumor cells. Antigen presenting cells expressed higher levels of CD68 in retinoblastoma compared to controls. Conclusions CD44+ and high-c-Myc-expressing tumor cells may represent cancer stem cells with possible involvement in metastasis, warranting further validation. Our multilayered approach could pave the way for enhanced molecular assessments and novel targeted therapies for human retinoblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39cca8d9fff8120c4da3617b30c0c9a1d7e7644" target='_blank'>
              A proteotranscriptomic approach to dissect the molecular landscape of human retinoblastoma
              </a>
            </td>
          <td>
            Julian Wolf, R. I. Hajdú, Stefaniya Boneva, Ira Godbole, Lucas Stürzbecher, Claudia Auw-Haedrich, W. A. Lagrèze, Hansjürgen Agostini, Thomas Reinhard, S. Tholen, Oliver Schilling, Günther Schlunck, Bertram Bengsch, Clemens Lange
          </td>
          <td>2025-05-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Cancer treatment options including small molecules, antibody-drug conjugates (ADCs), CAR T-cells, and combination therapies are rapidly expanding to improve cancer patient survival. However, efficacy of targeted therapies is limited by interpatient and intratumoral heterogeneity in breast cancer, and may be further exacerbated by dysregulated tumor microenvironments. For example, studies suggested that changes in tumor and breast tissue microenvironment in obesity (e.g. inflammation, increased growth factors) could lead to worsened prognosis that could be related to therapy resistance. Systematic characterization of the cellular effects of emerging drug candidates, particularly across different tumor microenvironments, is challenging, as it needs to be surveyed in diverse cancer contexts beyond just a handful of cancer cell lines. To overcome this challenge, we created PRISM (Profiling Relative Inhibition Simultaneously in Mixtures), an effective drug discovery screening platform with over 900 genetically distinct cancer cell lines of diverse lineages, to rapidly test therapeutics in a high-throughput manner. These cell lines, including 33 different breast cancer cell lines covering diverse cell types and subgroups, carry a specific 24 nucleotide “barcode” tag and have rich baseline genomic and functional characterization. They are first pooled into 20-25 cell pools of different lineages with similar growth rates, then subsequently cultured together with or without drugs. The relative barcode quantity read out by NextGen sequencing determines cell line responses to drugs. This pooled approach significantly reduces the time and the resources required for drug screening. Importantly, utilizing the rich cell line characterization to interpret viability profiles enables the identification of drivers of differential sensitivity and potential biomarkers of compound response. Furthermore, we created a breast cancer cell specific pool to investigate how media conditions mirroring different tumor microenvironments [e.g. media filtration, addition of insulin (0.01 mg/mL) or EGF (0.01 ug/mL)] can impact drug effects on breast cancer cells. In the PRISM 900+ cell panel treated with ADCs conjugated to different payloads that target HER2, there was a clear association between cell line sensitivity and ERBB2 (encoding HER2) gene expression, with the strongest effects on viability observed in a subset of cell lines overexpressing ERBB2. Upon further comparison of biomarker profiles across different ADCs, we established relationships between payload resistance and ABCB1 efflux pumps emerged for a subset of ADC payloads. Using the breast cancer cell specific pool, our data showed that estrogen receptor (ER) degraders fulvestrant and vepdegestrant (ARV-471) had significantly greater effects in reducing viability of ER+ versus ER- breast cancer cell lines. Media filtering reduced the drug effects of fulvestrant but not ARV-471, suggesting that ARV-471 efficacy may be less sensitive to the tumor microenvironment than fulvestrant. Lapatinib (HER2/neu and EGF receptor inhibitor) was more effective in killing HER2+ than HER2- negative breast cancer cells, with cells cultured in EGF-spiked media being more sensitive to lapatinib, whereas media filtration diminished its effects. We did not observe breast cancer subgroup specific killing by alpelisib (PI3K inhibitor) or olaparib (PARP inhibitor), but breast cancer cells became less sensitive to alpelisib when cultured in either insulin- or EGF-spiked media, and less sensitive to olaparib in EGF-spiked media. Overall, our study highlights how the PRISM 900+ cell panel can provide insights into drug specificity, cancer subtype selectivity, and to uncover clinically relevant targets. The breast cancer specific pool can further increase the throughput of understanding how drug responses may be altered in different tumor microenvironments in breast cancer.
 Citation Format: Blanche C. Ip, Jillian N. Eskra, Tenzin Sangpo, Ashish Bino George, Melissa Ronan, Matthew G. Rees, Jennifer A. Roth. PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-10-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5ec03b1ee18de91282ff62b13d5cb24a1236003" target='_blank'>
              Abstract P4-10-26: PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells
              </a>
            </td>
          <td>
            Blanche C. Ip, Jillian N. Eskra, Tenzing Sangpo, Ashish Bino George, M. Ronan, M. Rees, Jennifer A. Roth
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 2520


 Background:
 Despite the transformational impact of immune checkpoint blockade, many cancer patients do not experience long-term survival. T cells with innate immune signatures can secrete inflammatory cytokines/chemokines and deliver potent cytotoxic signals potentially ideal for tumor immunity. The novel double-stranded RNA sensor RIG-I agonist SLR14 improved the control of murine melanoma. We tested the hypothesis that SLR14 transforms T cells to a cytotoxic state in immunologically “cold” human tumor specimens.
 Methods:
 We developed an approach, called PERCEPT, to directly test the response of patient tumor and immune samples to novel and established therapies
 ex vivo
 using perturbational single-cell RNA sequencing (Table). We obtained 9 surgical resections from primary or metastatic melanoma and Merkel Cell Carcinoma (MCC) tumors and lymph node metastases and made suspension replicates of tumor and infiltrating immune cell co-cultures. We stimulated for 42-48 hours (Table). We Flourescently Activated Cell Sorted live cells and then barcoded for multiplexed single-cell sequencing using 10x scRNAseq. We used CINEMA-OT to identify factors associated with response and resistance to the perturbations tested. We developed and validated CRISPR-KO MCC and small cell lung cancer (SCLC) cell lines, and co-cultured with CD14+ monocytes or monocyte-derived DCs.
 Results:
 Stimulation with RIG-I agonist SLR14 induced expression beyond canonical IFN-stimulated genes in tumor cells, NK cells, and T cells. SLR14 stimulates tumor-infiltrating T cells into antiviral states in tumor-immune co-cultures and primes
 in vitro
 T-cell production of IFNγ. However, MCC immune infiltrate responsiveness to IFN or SLR14 was notably decreased compared to the melanoma samples, and perturbational computational analyses with CINEMA-OT identified the cytokine midkine (MDK) associated with nonresponse in MCC. Knockout of MDK restored response to IFN and SLR14 by MCC and SCLC tumor cell lines, as well as co-cultured CD14+ monocytes or monocyte-derived DCs.
 Conclusions:
 Our approach revealed that midkine, a multifunctional cytokine, suppresses innate immune sensing of IFN and SLR14 in both tumor and immune cells, disrupting the tumor immunity cycle at multiple points. We show that this effect, while comparatively infrequent in melanoma, is pronounced in MCC and SCLC. Our study thus uses a direct assessment of patient tumor and immune samples to identify a novel resistance mechanism enriched in neuroendocrine tumors MCC and SCLC.




 Therapeutic class
 Stimulation
 Target
 Clinical development




 Immune checkpoint inhibitor
 αPD-1
 PD-1
 Standard of care


 Cytokine
 IFNγIFNβ
 IFNGRIFNAR
 Early-phase clinical trials orPre-clinical


 Innate immune agonist
 Poly(I:C)-NTPoly(I:C)-TADU-S100SLR14
 TLR3MDA5/TIG-I/TLR3STINGRIG-I


 Combination
 αPD-1 + IFNβ
 PD-1/IFNAR




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/404b41f4f104c4cf7c170db6748ecf8ec9a5f2f1" target='_blank'>
              Perturbational single-cell RNA sequencing of patient tumors in Merkel cell and small cell lung carcinomas.
              </a>
            </td>
          <td>
            Curtis J. Perry, Alexander Frey, Yuewei Fei, P. Costa, Jason Z Wang, Sina Ghadermarzi, Daniel Levine, Asuka Koda, Amin Nassar, Min Ding, Yunan Nie, Therese Cordero-Dumit, Arnaud Augert, David van Djik, K. Olino, J. Ishizuka
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec6ace55f209d3279f18b935c41cea6d1dea4c0c" target='_blank'>
              Cell Type-Specific Effects of miR-21 Loss Attenuate Tumor Progression in MYC-Driven Prostate Cancer
              </a>
            </td>
          <td>
            Kenji Zennami, M. Graham, Shireen Chikara, Polina Sysa-Shah, Fatima Rafiqi, Rulin Wang, Bulouere Abel, Qizhi Zeng, Timothy E. G. Krueger, W. Brennen, Sudipto Ganguly, Jelani C. Zarif, B. Simons, Theodore L. DeWeese, Angelo De Marzo, Srinivasan Yegnasubramanian, Shawn E Lupold
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background: Disease modeling is vital for our understanding of disease mechanisms and for developing new therapeutic strategies. Accurately modeling the intact tumor microenvironment (TME) is increasingly recognized as essential for gaining insights into cancer biology and therapeutic response. Preclinical mouse models have provided utility for studying the evolving TME, but these models are costly and can lead to animal suffering and the discontinuation of drug investigations. To address these limitations, particularly in hepatocellular carcinoma (﻿HCC), we have developed an ex vivo model using tumor precision-cut slices (TPCS) derived from orthotopic liver tumors. Methods: Murine HCC tumors were generated via intrahepatic injection of Hep-53.4 cells, providing a source of tumor tissue for TPCS generation. Subsequent scaling to a 96-well format and modification to include a secreted luciferase enabled longitudinal ex vivo screening of 26 drugs applied at 2 doses over an 8-day period, using just 5 tumors. One drug identified in the screen, salinomycin, was then validated in vivo via intraperitoneal injection of mice with orthotopic liver tumors. Results: Histological characterization determined that TPCS maintain the architecture, cellular complexity, and drug responsiveness of the original HCC-TME under simplified culture conditions that preserve viability and metabolic activity. In addition to typical HCC therapies, sorafenib and anti-PD1 immunotherapy, the screen identified 2 drugs as potent anticancer agents capable of impacting the viability of TPCS: salinomycin and rottlerin. Salinomycin was further validated in vivo, significantly reducing tumor burden without evidence of toxicity. Conclusions: We present a 3Rs (Reduction, Refinement, Replacement) approach for studying HCC biology and performing 96-well-scale drug screening within an intact, metabolically active TME, offering a more ethical and effective platform for drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0fc9ab2a0c87013cf6a5b53d9f8f813f5001d3b" target='_blank'>
              Precision-cut tumor slices for modeling hepatocellular carcinoma enable at-scale drug screening
              </a>
            </td>
          <td>
            Amy L. Collins, Keara Kirkness, Erik Ramon‐Gil, Eleni Tzortzopoulou, D. Geh, Jack Dishington, Eleanor Graham, Rhys Muir, Rainie Cameron, Saimir Luli, Eman Khurram, Daniel Storey, Hannah L. Paish, Glyn Nelson, David McDonald, Andrew Filby, Lee A Borthwick, Fiona Oakley, Derek A Mann, J. Leslie
          </td>
          <td>2025-05-16</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Emerging spatial profiling technologies have revolutionized our understanding of how tissue architecture shapes disease progression, yet the contribution of cellular diversity remains underexplored. Here, Ding and colleagues introduce multiomics and ecological spatial analysis (MESA), an ecology-inspired framework that integrates spatial and single-cell expression data to quantify tissue diversity across multiple scales. MESA both identifies distinct cellular neighborhoods and computes a variety of diversity metrics alongside the identification of diversity "hotspots". Applied to human tonsil tissue, MESA revealed previously undetected germinal center organization, while in spleen tissue of a murine lupus model, MESA highlights increasing cellular diversity with disease progression. Importantly, diversity hotspots do not correspond to conventional compartments identified by existing methods, presenting an orthogonal metric of spatial organization. In colorectal cancer, MESA's diversity metrics outperformed established subtypes at predicting patient survival, while in hepatocellular carcinoma, multi-omic integration identified significantly more ligand-receptor interactions between immune cells compared to single-modality analysis. This work establishes cellular diversity within tissues as a critical correlate of disease progression and underscores the value of multi-omic integration in spatial biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eb774cd8ef924c70b07f117ccf0af3bd80ea8f6" target='_blank'>
              Cells keep diverse company in diseased tissues.
              </a>
            </td>
          <td>
            Kieran R. Campbell, A. Goeva
          </td>
          <td>2025-05-16</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background The immunological landscape of metabolic dysfunction-associated steatohepatitis (MASH)-driven hepatocellular carcinoma (HCC) is not well understood. Herein, we aim to delineate the immunological landscape in the MASH-to-HCC transition and to identify the critical genes that contribute to the pathogenesis of MASH-related HCC. Methods A well-established MASH-driven HCC mouse model, STAM model, was first constructed. Thereafter, we applied single-cell RNA sequencing (scRNA-seq) analysis of CD45+ cells sorted from livers of mice with normal chow or MASH, as well as paired paracancerous and cancer tissues from mice with HCC. Flow cytometry and multiplexed immunohistochemistry were performed to validate the analysis results of scRNA-seq. Finally, STAM model was applied between apolipoprotein E (ApoE)-deficient mice and wild type controls. Results We identified 23 major clusters corresponding to nine populations among 31,822 cells. Obviously, immunosuppressive and exhausted CD4+ T (IKZF2+OX40+FOXP3+CD4+ and GZMK+LAG-3+PD-1+CD4+), CD8+ T (LY49I+LY49G+IKZF2+FOXP3−CD8+, IKZF2+FOXP3+CD8+ and GZMK+LAG-3+PD-1+CD8+) and γδ T cells (γδ Treg and exhausted γδ T cells) were induced in the MASH-to-HCC transition. As MASH-related HCC progressed, B cells matured and differentiated into immunosuppressive cells. Natural killer cells (NKs) were found to be strikingly reduced at HCC stage. Particularly, the activation of liver-infiltrated NK cells was inhibited, leading to attenuation of anti-tumor capacity in the MASH-to-HCC transition. Moreover, tumor-associated macrophages were increased in MASH-related HCC. Importantly, multiple immune cells highly expressed ApoE in HCC, and ablation of ApoE impeded MASH-driven hepatocarcinogenesis by disrupting both ApoE-PI3K-AKT-NF-κB and ApoE-PI3K-AKT-c-Jun/c-Fos signaling pathways. Conclusions We illustrate the profound reprogramming of the liver immune microenvironment in the MASH-to-HCC transition and clarify the role of ApoE in MASH-driven HCC, implying that ApoE may serve as a potential therapeutic target for MASH-related HCC. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02370-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d33d0cf3ae24c19bf9ea4aeaf77109fca34bff" target='_blank'>
              Single-cell transcriptome reveals the reprogramming of immune microenvironment during the transition from MASH to HCC
              </a>
            </td>
          <td>
            Yu Huang, Ying Xie, Yuqing Zhang, Zhemian Liu, Weihua Jiang, Yingying Ye, Jiale Tang, Zhenhua Li, Zhinan Yin, Xue-Jia Lin
          </td>
          <td>2025-06-11</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer drug resistance is multi-factorial, driven by heritable (epi)genetic changes but also phenotypic plasticity. Here, we dissected drivers of resistance by perturbing colorectal cancer patient-derived organoids longitudinally with drugs in sequence. Combined longitudinal lineage tracking, single cell multi-omics analysis, evolutionary modelling, and machine learning revealed that different targeted drugs select for distinct subclones, supporting rationally designed drug sequences. The cellular memory of drug resistance was encoded as a heritable epigenetic configuration from which multiple transcriptional programs could run, supporting a one-to-many (epi)genotype-to-phenotype map that explains how clonal expansions and plasticity manifest together. This epigenetic landscape may ensure drug resistant subclones can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding for their selective advantage. Chemotherapy resistance was instead entirely driven by transient phenotypic plasticity rather than stable clonal selection. Inducing further chromosomal instability before drug application changed clonal evolution but not convergent transcriptional programs. Collectively, these data show how genetic and epigenetic alterations are selected to engender a "permissive epigenome" that enables phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83269ee41c4604edc0996e3b22f4bf8776d7c6" target='_blank'>
              Epigenetic Heritability of Cell Plasticity Drives Cancer Drug Resistance through a One-to-Many Genotype-to-Phenotype Paradigm.
              </a>
            </td>
          <td>
            É. Oliveira, Salvatore Milite, J. Fernández-Mateos, George D. Cresswell, Erika Yara-Romero, G. Vlachogiannis, Bingjie Chen, Chela James, Lucrezia Patruno, Gianluca Ascolani, Ahmet Acar, Timon Heide, Inma Spiteri, Alex Graudenzi, G. Caravagna, Andrea Bertotti, Trevor A. Graham, Luca Magnani, Nicola Valeri, A. Sottoriva
          </td>
          <td>2025-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CAR‐T) cell therapies, a cornerstone of immunotherapy, have demonstrated remarkable efficacy in treating hematological malignancies and have more recently expanded into applications for solid tumors and autoimmune diseases. Emerging multidimensional profiling technologies offer promising solutions for enhancing CAR‐T efficacy, overcoming resistance, and facilitating the development of novel CAR‐T constructs. The integration of genomics, epigenomics, transcriptomics, proteomics, metabolomics, and microbiomics enables a comprehensive understanding of the intrinsic mechanisms underlying CAR‐T therapy, while single‐cell and spatial omics significantly improve data resolution and analytical depth. Coupled with advances in biomedical engineering, visualization technologies form the foundation for omics data generation by bridging microscopic and macroscopic scales and enabling dynamic, 3D in vivo monitoring of CAR‐T behavior. Artificial intelligence (AI) further supports this framework by enabling the analysis of complex, high‐dimensional datasets. This review highlights recent advances in the integration of multidimensional omics within CAR‐T therapy and explores cutting‐edge developments in visualization technologies and AI applications. The full convergence of multi‐omics, visualization tools, and AI is poised to deliver transformative insights into the mechanisms governing CAR‐T cell therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec20ecc46258adb912c1ad15d62f51a53d851ab" target='_blank'>
              From Multi‐Omics to Visualization and Beyond: Bridging Micro and Macro Insights in CAR‐T Cell Therapy
              </a>
            </td>
          <td>
            Yuting Gong, Peng Fei, Yicheng Zhang, Yang Xu, Jia Wei
          </td>
          <td>2025-05-01</td>
          <td>Advanced Science</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Cell plasticity is a hallmark of cancer, enabling tumor cells to acquire multiple phenotypes responsible for tumor progression, metastasis, and therapy resistance. In this issue of the JCI, Kawai and colleagues leveraged genetically engineered mouse models (GEMM) of pancreatic ductal adenocarcinoma (PDAC) to demonstrate that loss of Pbrm1, a member of the SWI/SNF complex, drives dedifferentiation and aggressive tumor features. Pbrm1 loss activated a program of epithelial-to-mesenchymal transition (EMT) and allowed the emergence of poorly differentiated histologies that are commonly associated with high recurrence rate and dismal prognosis. These findings reveal the role of the SWI/SNF complex during PDAC evolution in maintaining cell identity and restraining the progression of this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51cf4e0b352b84ce0ee22909d350d6fe7531e126" target='_blank'>
              Switching on the evolutionary potential of pancreatic cancer: the tumor suppressor functions of PBRM1
              </a>
            </td>
          <td>
            Luigi Perelli, G. Genovese
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The study of circulating tumor cells (CTCs) provides critical insights into the biological mechanisms underlying metastasis. This study aims to demonstrate the applicability of an integrated DEPArray-based phenotypic analysis combined with transcriptomic characterization to investigate the biology of CTCs isolated from 10 patients with metastatic breast cancer (MBC). The transcriptional profiles of CTCs were consistent with both the cancer type and epithelial characteristics. Gene set enrichment analysis identified pathways associated with synapse organization and calcium channel activity. Furthermore, distinct gene expression profiles were linked to specific metastatic sites, particularly bone metastases. We also report a rare and understudied population of CTCs, characterized by the co-expression of epithelial and leukocyte markers, observed exclusively in patient-derived samples and not in blood samples from healthy volunteers spiked with SKBR-3 and MCF-7 cell lines. This suggests that these double-positive CTCs (dpCTCs) may have a specific role in the metastatic process. The transcriptomic characterization of these rare CTCs enhances our understanding of their biology and potential involvement in metastasis. As a pilot study, our findings underscore the potential of CTC-based transcriptomics as a valuable tool for advancing clinical and biological understanding of MBC, particularly regarding metastatic mechanisms and organotropism. Moreover, it provides preliminary insights into dpCTCs, a poorly characterized population that may play a pivotal role in metastasis but remains largely unexplored. These findings could pave the way for developing targeted therapies aimed at the pathways driving metastasis and for improving patient monitoring through CTC profiling. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00659-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5c8d5b6cac842e5c06154018a6fe136a8682dd8" target='_blank'>
              Transcriptomic profiling of circulating tumor cells from metastatic breast cancer patients reveals new hints in their biological features and phenotypic heterogeneity
              </a>
            </td>
          <td>
            T. Rossi, Sara Bandini, Michele Zanoni, M. Cortesi, M. Palleschi, E. Bandini, Andrea Rocca, Giulia Gallerani, Ivan Vannini, Meropi Plousiou, L. Gerratana, A. Musolino, G. Tallini, Giovanni Martinelli, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-05-06</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d55bd04a082dd960e72d401069d912b22df3a0ed" target='_blank'>
              Autocrine TGFβ2 enforces a transcriptionally hybrid cell state in Ewing sarcoma
              </a>
            </td>
          <td>
            Emma D. Wrenn, Jacob C. Harris, April A. Apfelbaum, Jonah R. Valenti, Patricia A. Lipson, Stephanie I. Walter, Nicolas M Garcia, Aya Miyaki, Steven C. Chen, Jim M. Olson, Jason S. Price, Kelly M. Bailey, Elizabeth R. Lawlor
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Non–small cell lung cancer (NSCLC) constitutes over 80% of lung cancer cases and remains a leading cause of cancer-related mortality worldwide. Despite the advent of immune checkpoint inhibitors, their efficacy is limited to 27 to 45% of patients. Identifying likely treatment responders is essential for optimizing healthcare and improving quality of life. We generated multiplex immunofluorescence (mIF) images, histopathology, and RNA sequencing data from human NSCLC tissues. Through the analysis of mIF images, we characterized the spatial organization of 1.5 million cells based on the expression levels for 33 biomarkers. To enable large-scale characterization of tumor microenvironments, we developed NucSegAI, a deep learning model for automated nuclear segmentation and cellular classification in histology images. With this model, we analyzed the morphological, textural, and topological phenotypes of 45.6 million cells across 119 whole-slide images. Through unsupervised phenotype discovery, we identified specific lymphocyte phenotypes predictive of immunotherapy response. Our findings can improve patient stratification and guide selection of effective therapeutic regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebcac8871256527456ef7fc38ff937674e4c6cc4" target='_blank'>
              Single-cell multimodal analysis reveals tumor microenvironment predictive of treatment response in non–small cell lung cancer
              </a>
            </td>
          <td>
            Yuanning Zheng, Christoph Sadée, Michael Ozawa, B. Howitt, Olivier Gevaert
          </td>
          <td>2025-05-23</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/938950b1958bbfaef501d56fa236f86c6e2c6141" target='_blank'>
              Multi-omic analysis of gallbladder cancer identifies distinct tumor microenvironments associated with disease progression.
              </a>
            </td>
          <td>
            Tao Zhou, Yanhong Wu, Shuai Li, Xinyao Qiu, Erdong Liu, Zhihua Xie, Xuebing Shi, Yani Zhang, Guosheng Ma, Wenbo Guo, Xiang Wang, Kaiting Wang, Xiaomeng Yao, Ji Hu, Siyun Shen, Shuai Yang, Xiaoqing Jiang, Jing Fu, Hongyang Wang, Jin Gu, Lei Chen
          </td>
          <td>2025-06-26</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f4e542b3c6d086c581d428766c82b0185e38fd" target='_blank'>
              Intratumoral heterogeneity in microsatellite instability status at single cell resolution
              </a>
            </td>
          <td>
            Harrison Anthony, C. Seoighe
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 e14555


 Background:
 Natural killer (NK) cells are innate lymphocytes that engage in direct cytotoxic responses against malignant cells and promote pro-inflammatory tumor microenvironments. While studies have predominantly focused on conventional NK cell subsets, recent findings uncovered adaptive and tissue-resident (TR) subsets with unique functional properties. The influence of distinct NK cell subsets on tumor immunobiology and clinical outcomes remains unknown.
 Methods:
 Single-cell RNA-sequencing (scRNA-seq) of primary, therapy-naïve tumors from 276 patients across 14 cancer types, as well as 39 patients treated with immune checkpoint blockade (ICB), was accessed through publicly available datasets. We employed systematic computational analyses to phenotype and examine intra-tumoral NK cells. Gene signatures for NK cell subsets were applied to bulk RNA-sequencing from 212,105 tumor samples to delineate their associations with clinical outcomes using the Caris CODEai database.
 Results:
 Six main subsets of NK cells were identified in solid tumors, including conventional CD56
 dim
 and CD56
 bright
 NK cells, alongside adaptive, TR adaptive, TR CD56
 bright
 , and TR CD56
 bright
 /adaptive subsets. Across cancer, TR adaptive NK cells emerged as the most prognostic subset, with high infiltration conferring significantly longer overall survival (HR[95% CI] = 0.68[0.66-0.69], p < 0.001) than conventional CD56
 dim
 and CD56
 bright
 subsets. This was also observed in cohorts of head and neck, non-small cell lung, and esophageal cancers, particularly those treated with ICB. Notably, elevated TR adaptive NK cell infiltration was more prognostic than high tumor mutational burden in these settings. This was most apparent in ICB-treated head and neck cancers (HR = 0.48[0.41-0.57], p < 0.001 vs HR = 0.90[0.76-1.04], p = 0.16). Additionally, TR adaptive NK cells were strongly correlated to the infiltration of CD8
 +
 T cells and B cells in tumors. Underlying these observations, our scRNA-seq analyses demonstrated that TR adaptive NK cells exhibit the highest immunomodulatory activity of all subsets, expressing high levels of pro-inflammatory cytokines (
 CCL3
 ,
 CXCL13
 , and
 IFNG
 ) and low levels of immunosuppressive cytokines (
 TGFB1
 ). Mechanistically, this activity was linked to the upregulation of transcription factors associated with NK cell activation, function, and persistence and downregulation of genes related to stress response and exhaustion. Interestingly, scRNA-seq from paired pre- and post-ICB-treated tumors revealed a unique expansion of cytotoxic
 GZMB
 + TR adaptive NK cells after treatment.
 Conclusions:
 TR adaptive NK cells represent a novel subset of NK cells that may drive anti-tumor immune responses, particularly in the context of ICB treatment. Features of this unique population may be harnessed to design more effective NK cell-based therapies against solid tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdbcf1a77672e37211fd7cd09d5cd12367404c3a" target='_blank'>
              Identification of tissue-resident adaptive NK cells and their association with reduced mortality across diverse solid tumors.
              </a>
            </td>
          <td>
            John R Lozada, A. Ali, Christine Luo, Rosemary N. Plagens, A. Elliott, Ella Boytim, Nicholas A Zorko, W. El-Deiry, Elisabeth I. Heath, Akash Patnaik, A. Vanderwalde, Jeffrey S Miller, E. Antonarakis, Frank Cichocki, Justin H Hwang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background The heterogeneity of colorectal cancer (CRC) and its complex immune microenvironment pose significant challenges for treatment. Understanding the cellular composition and dynamic changes is essential for uncovering mechanisms of tumour progression. Methods To investigate the cellular heterogeneity and immune microenvironment of CRC, identifying critical subpopulations, functional pathways, and prognostic biomarkers, single-cell transcriptomic data from 41 CRC samples across four datasets were integrated. Bioinformatic analyses identified cellular subpopulations, cell communication networks, and prognostic biomarkers. The expression patterns, clinical significance and biological function of TUBB were validated in vitro. Results A distinct epithelial subpopulation with proliferative and invasive features was identified, promoting tumour progression by resisting apoptosis and remodelling the extracellular matrix. ActMono, a terminal state of myeloid cells, was enriched in tumours and linked to disease progression. Cell communication analysis highlighted galectin signalling in immune regulation. A prognostic model (CRS) based on secretory immune cell-related genes identified TUBB as a key molecule influencing the cell cycle and extracellular matrix remodelling, with its expression patterns, clinical significance and biological effects validated in vitro. Conclusion This study reveals critical subpopulations, signalling pathways, and biomarkers in CRC, providing insights into tumour progression and potential therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2e4b3d5d75559211564431db384708f82801552" target='_blank'>
              Single-cell sequencing reveals dysregulated cell type perturbations and critical mediator communication remodelling in colorectal cancer
              </a>
            </td>
          <td>
            Chengyuan Xu, Siqi Zhang, Zhouyu Zhang, Luoqiu Zhang, Bin Sun, Zicheng Yu, Hailong Liu
          </td>
          <td>2025-06-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="During cancer development, mutations promote gene expression changes that cause transformation. Leukemia is frequently associated with aberrant HOXA expression driven by translocations in nucleoporin genes or KMT2A, and mutations in NPM1. How disparate mutations converge on this regulatory pathway is not understood. Here we demonstrate that mutant NPM1 (NPM1c) forms nuclear condensates in multiple human cell lines, mouse models, and primary patient samples. We show NPM1c phase separation is necessary and sufficient to coordinate the recruitment of NUP98 and KMT2A to condensates. Through extensive mutagenesis and pharmacological destabilization of phase separation, we find that NPM1c condensates are necessary for regulating gene expression, promoting in vivo expansion, and maintaining the undifferentiated leukemic state. Finally, we show that nucleoporin and KMT2A fusion proteins form condensates that are biophysically indistinguishable from NPM1c condensates. Together, these data define a new condensate underlying leukemias that we term coordinating bodies (C-bodies), and propose C-bodies as a therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0742ebc47874d0a681b5ec4cb75157b2b57840b5" target='_blank'>
              Nuclear Phase Separation Drives NPM1-mutant Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Gandhar K. Datar, Elmira Khabusheva, Archish Anand, Joshua Beale, Marwa Sadek, Chun-Wei Chen, Evdokiia Potolitsyna, Nayara Alcantara-Contessoto, Guangyuan Liu, Josephine De La Fuente, Christina Dollinger, A. Guzman, Alejandra G Martell, Katharina Wohlan, A. Maiti, N. Short, S. S. Yi, V. Andresen, Bjørn T. Gjertsen, B. Falini, R. E. Rau, L. Brunetti, Nidhi Sahni, M. Goodell, Joshua A. Riback
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="
 8091


 Background:
 Small cell lung cancer (SCLC) is a highly aggressive neuroendocrine malignancy with a median survival of 9–11 months. Despite the addition of immune checkpoint blockade (ICB) to frontline chemotherapy, improvements remain limited. SCLC evades anti-tumor immunity by suppressing major histocompatibility complex class I (MHC-I) and antigen presentation (AP). The variability of MHC repression across molecular subtypes of SCLC and high-grade neuroendocrine tumors is not fully understood. This study analyzed clinical SCLC and neuroendocrine tumor samples to identify molecular subtypes and regulators of MHC repression, enhancing the understanding of immune evasion.
 Methods:
 Molecular profiling was conducted on 944 SCLC and 5056 non-small cell lung cancer (NSCLC) tumors using next-generation DNA (592-gene panel/whole exome), RNA (whole transcriptome) sequencing, and immunohistochemistry. Transcriptomic profiling of 40 SCLC cell lines and genetically engineered mouse models (GEMMs) was also performed.
 Results:
 Key regulators and gene networks driving MHC-I suppression in SCLC and other high-grade neuroendocrine tumors were identified. Canonical (MHC-I and II) and non-canonical AP expression scores characterized the spectrum of MHC-I repression across subtypes. Low MHC-I scores correlated with reduced immune signatures (e.g., T cell-inflamed, NK cell, and STING pathway signatures; Spearman = 0.87, 0.58, 0.62, respectively) and predicted poor OS and PFS to immunotherapy. Enhancer network and gene expression analyses identified actionable regulators of MHC-I repression, including interferon signaling. Knockout studies of top MHC-I regulators and single cell analyses revealed non-homologous end-joining (NHEJ) genes (like DNAPKCs) inversely correlated with HLA-A/B/C expression. DNAPKCs inhibition upregulated MHC-I, TAP1, and TAP2, enhanced antigen-specific T-cell-mediated cytolysis in SCLC. This inhibition activated the STING pathway, increased CD8+ T-cell infiltration, and enhanced MHC Class-I. Combining DNAPKCs inhibitor NU1774 with anti-PD-L1 reactivated MHC-I and led to significant tumor regression in aggressive murine SCLC models, supporting new combination therapies.
 Conclusions:
 We identified molecular subtypes and actionable pathways to restore MHC-I expression in SCLC, advancing understanding of immune evasion mechanisms. DNAPKCs is established as a key therapeutic target, enhancing PD-L1 blockade by reactivating antigen presentation. The developed MHC-I expression scores and associated biomarkers offer predictive tools for identifying patients likely to benefit from immunotherapy. These findings support biomarker-driven, personalized approaches to improve treatment for SCLC, addressing a critical unmet need in this devastating cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cef19343d2cbcd91f40c5103e3abe3dd630690e8" target='_blank'>
              Unveiling drivers of MHC repression and therapeutic strategies to counter immune evasion in small cell lung cancer.
              </a>
            </td>
          <td>
            Triparna Sen, Subhamoy Chakraborty, A. Elliott, V. Jethalia, A. Vanderwalde, B. Halmos, D. Demircioglu, D. Hasson
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of prostate cancer that strongly correlates with metastatic burden and appears prominently in both primary cancer and metastatic disease. Low Gleason score primary prostate tumors display pervasive centrosome loss, a known mechanistic driver of CIN, that disrupts normal spindle assembly and increases mitotic errors. Previously, we found that transient depletion of centrosomes in immortalized, non-tumorigenic prostate epithelial cells (PrEC) induced a burst of CIN, generating cell lines capable of forming xenograft tumors. Here, we use a multi-omics approach to identify the oncogenic alterations caused by transient centrosome loss. By integrating genomic and transcriptomic data of the prostate lines, we identified a consensus set of focal copy-number variations (CNVs) induced by centrosome loss in cultured cells that are also detectable within a subset of samples from a prostate cancer patient cohort. Using this CNV signature, we were able to derive a unique transcriptomic signature from prostate cancer patient samples that showed strong predictive value for adverse clinical outcomes. In summary, our experimental system uses centrosome loss to promote a punctuated burst of genomic crisis that is characteristic of genome evolution during prostate cancer progression. Consequently, this prostate cancer model produced recurrent structural variations that are detectable in patient samples and associate with worse outcomes. Graphic Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7263ae00a48684dacddf5d2cc713b37a7dad7c68" target='_blank'>
              Transient centrosome loss in cultured prostate epithelial cells induces chromosomal instability to produce an oncogenic genotype that correlates with poor clinical outcomes
              </a>
            </td>
          <td>
            Jiawen Yang, Diogo de Oliveira Pessoa, John M. Ryniawec, Matthew R. Coope, Daniel W. Buster, Emily Loertscher, Mengdie Wang, Chen Chen, A. Cress, Gregory C Rogers, Megha Padi
          </td>
          <td>2025-05-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Esophageal squamous cell carcinoma (ESCC) is a highly heterogeneous malignancy. Traditional clinical staging systems have limited value in prognostic evaluation and treatment guidance, often failing to capture the profound impact of intratumoral diversity on patient outcomes. Single-cell RNA sequencing (scRNA-seq) provides new perspectives on the cellular makeup and conditions in tumor tissues, with promising implications for functional classification and personalized therapeutic strategies in esophageal cancer. Methods In this study, we integrated scRNA-seq data with bulk RNA-seq profiles from esophageal cancer tissues to construct a comprehensive cellular atlas, focusing on the transcriptional characteristics of epithelial cells. Malignant epithelial cells were identified based on copy number variation (CNV) features using inferCNV analysis. Their developmental states and regulatory mechanisms were further characterized via transcription factor activity inference (SCENIC) and pseudotime trajectory analysis (Monocle). Based on marker genes of malignant epithelial subpopulations, we developed a multi-gene risk scoring model using data from the TCGA and GEO (GSE53624) cohorts. The model’s predictive value for immune landscape, mutational features, and drug sensitivity was also evaluated. Additionally, qRT-PCR was conducted to quantify the expression levels of model genes in ESCC tissue samples, further evaluating their biological relevance. Functional roles of the key gene HMGB3 were validated in vitro through CCK-8 proliferation assays, Transwell invasion assays, and colony formation assays following gene knockdown in ESCC cell lines. Results At the single-cell level, we identified ten major cell types and six distinct malignant epithelial subclusters, which exhibited pronounced heterogeneity in cell cycle states, transcriptional regulatory networks, and differentiation trajectories. High CNV scores and the enrichment of specific transcription factors (e.g., FOXC1, E2F1, RUNX1) suggested a proliferative and immune-evasive phenotype. A six-gene prognostic model (HMGB3, CHORDC1, CTSD, BTG2, MT1E, PHYHD1) showed strong predictive accuracy for overall survival in the TCGA and GSE53624 cohorts. Furthermore, the risk score showed a significant correlation with an immunosuppressive tumor microenvironment, increased tumor purity, and the activation of certain immune-related pathways. Analysis of drug sensitivity suggests that patients classified as low-risk could respond better to various targeted therapies and chemotherapeutic agents, underscoring their potential clinical relevance. Functional assays revealed that HMGB3 knockdown markedly suppressed ESCC cell proliferation, invasion, and colony formation, supporting its oncogenic role in esophageal cancer progression. Conclusion This study systematically characterized epithelial cell heterogeneity in esophageal cancer at single-cell resolution and established a risk model based on malignant epithelial markers that effectively predicts prognosis, immune status, and potential drug response. Combined with experimental validation, our findings highlight the pivotal role of HMGB3 in ESCC progression and provide a novel theoretical and practical framework for functional tumor classification and individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70d92eb766fe9a932637e37c8c15e4650793e3d8" target='_blank'>
              Malignant epithelial cell marker–driven risk signature enables precise stratification in esophageal cancer
              </a>
            </td>
          <td>
            Hao Zhang, Shizhao Cheng, Yijun Xu
          </td>
          <td>2025-05-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0e9134dab7b134dd165787785bd9a7f1f9b1843" target='_blank'>
              Deep learning inference of cell type-specific gene expression from breast tumor histopathology
              </a>
            </td>
          <td>
            Andrew Wang, S. R. Dhruba, Kun Wang, Emma M. Campagnolo, E. Shulman, E. Ruppin
          </td>
          <td>2025-05-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background
 The tumor microenvironment and biomarkers play a pivotal role in breast cancer research, yet there remains a pressing need for effective biomarkers. This study focuses on identifying a novel IGKC+ T Cell subpopulation and its related biomarkers to pave the way for innovative targeted therapies and improved clinical outcomes.


Methods
We first performed single-cell RNA sequencing (scRNA-seq) analysis to characterize immune cell heterogeneity within the tumor microenvironment, leading to the identification of series cell subpopulation. Then, by performing univariate analysis to correlate cell proportions with patient prognosis, we identified a novel IGKC+ T cell subpopulation. Next, we applied bulk RNA-seq deconvolution algorithms to estimate the abundance of this subpopulation across breast cancer cohorts. Differential expression analysis and weighted gene co-expression network analysis (WGCNA) were employed to identify genes associated with the IGKC+ T cell population. To pinpoint key regulatory genes, we applied machine learning algorithms. Based on the hub genes identified, we constructed a prognostic risk model and developed a nomogram to aid clinical decision-making. Immune infiltration patterns were further assessed in high- vs low-risk groups defined by the model. Finally, functional validation was performed through overexpression of BCL2L14 in vitro, and downstream signaling pathways were examined.


Results
We identified the novel IGKC+ T cell subpopulation and core genes. Machine learning pinpointed BCL2L14, IGHD, MAPT-AS1, NT5DC4, and TNIP3 as key regulators of breast cancer progression in this subpopulation. The model stratified patients into high- and low-risk groups, with high-risk patients showing worse prognosis and weaker immune infiltration. Overexpression of BCL2L14 was experimentally demonstrated to accelerate breast cancer progression, linked to enhanced phosphorylation of the NF-κB pathway.


Conclusion
Our results underscore BCL2L14 as a potential driver within the novel T-cell subpopulation and a critical biomarker for breast cancer diagnosis. These findings provide a basis for developing advanced diagnostic tools and targeted therapies, which may ultimately enhance patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/899ab08619ae94e923463082f4db3808d873da0a" target='_blank'>
              Integration of Bulk and Single-Cell Transcriptomics Reveals BCL2L14 as a Novel IGKC+ T Cell-Associated Therapeutic Target in Breast Cancer.
              </a>
            </td>
          <td>
            Jiaming He, Aiman Akhtar, Jing Li, Qiang Wei, Yang Yuan, Jianhua Ran, Yongping Ma, Dilong Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of inflammation research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gliomas, particularly glioblastomas, are highly malignant brain tumors with high recurrence rates and poor prognosis. Despite advances in treatment, recurrence remains a major challenge. Epithelial-mesenchymal transition (EMT) plays a key role in tumor invasion and recurrence. This study explores the transcriptional and regulatory mechanisms driving glioma recurrence, focusing on mesenchymal-like (MES-like) subpopulations. Single-nucleus RNA sequencing was performed on 52 IDH wild-type GBM specimens, including 26 primary and 26 recurrent tumors. Spatial transcriptomics data were also incorporated. Tumor subpopulations were identified through gene regulatory network analysis, copy number variation detection, and nonnegative matrix factorization. Functional validation was conducted using gene knockdown experiments, followed by xenograft studies. We discovered novel MES-like subpopulations in recurrent GBM enriched with EMT-related genes like EGR1 and SERPINE1. These subpopulations exhibited increased transcriptional activity and were associated with poor prognosis and invasiveness. Knockdown of SERPINE1 significantly reduced cell proliferation and migration. Spatial transcriptomics showed MES-like cells concentrated at the tumor margins, highlighting their role in invasion and recurrence. MES-like subpopulations, driven by EGR1 and SERPINE1, are critical in GBM. Targeting these regulators could offer new therapeutic strategies to reduce glioma recurrence and improve outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02036-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f7432f6a50e6a5d830f429844d915dd10fcd2" target='_blank'>
              Single-cell transcriptome sequencing reveals new epithelial-stromal associated mesenchymal-like subsets in recurrent gliomas
              </a>
            </td>
          <td>
            Jinwei Li, Shengrong Long, Yang Zhang, Shuangqi Yu, Hongyu Xu, Rui Liang, Quan Liu, Jinnan Zhang, Xiang Li, Yixin Fu, Tao Xin, Yinyan Wang
          </td>
          <td>2025-06-07</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The role of the microbiome in cancer metastasis has emerged as a critical area of research, with growing evidence suggesting that microbial composition and interactions within the tumour microenvironment may significantly influence metastatic progression. This review explores the role of the microbiome in cancer metastasis, as well as potential key bacteria and their mechanisms through which they could impact tumour dissemination, seeding and growth. Biological models used to study metastasis are discussed to provide context for the further investigation of these interactions. In order to answer unresolved questions regarding the microbiome’s involvement in metastatic dissemination, recent advancements in spatial biology techniques are examined, including spatial genomics, transcriptomics, proteomics and metabolomics, which enable the spatial mapping of microbial interactions within the tumour microenvironment. Additionally, multimodal-omics imaging approaches are highlighted for their potential to integrate multiple molecular layers, offering comprehensive insights into the microbiome’s role in cancer metastasis. The review also addresses the challenges and limitations of these techniques, underscoring the complexity of studying microbiome-tumour interactions and offering directions for future research to better explore and target the microbiological landscape in metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bd737e751891e52e56b56db2c8e1ef16eeffa30" target='_blank'>
              Microbiome in cancer metastasis: biological insights and emerging spatial omics methods
              </a>
            </td>
          <td>
            Marianne Meyers, Charlotte B. A. Stoffels, G. Frache, Elisabeth Letellier, Maureen Feucherolles
          </td>
          <td>2025-06-04</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb25978847c5553a988089ffa28339e8dff51f1" target='_blank'>
              Single-Cell profiling reveals targetable malignant T-cell subtypes and immune evasion pathways in Sezary Syndrome
              </a>
            </td>
          <td>
            M.S.F Beth A. Childs, Ph.D Eslam A. Elghonaimy, Praveen, Ramakrishnan Geethakumari, Kiran A. Kumar, Joseph F. Merola, M. D. Heather W. Goff, M. Todd A Aguilera, Ph.D Todd
          </td>
          <td>2025-05-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastasis is a hallmark of malignancy and poses a formidable challenge for oncologists. This complex process begins when the primary malignant cells start to proliferate unlimitedly. Once the tumor reaches a certain size, cancer cells detach from the primary tumor mass and the basal lamina to which they are anchored, eventually invading nearby blood vessels. Within the bloodstream, these tumor cells have to survive and attach to the endothelium at distant sites. Cell‐to‐cell and cell‐to‐matrix interactions play an important role during these stages, in which integrins—a family of cell adhesion molecules (CAMs)—stand out as functionally centrally involved. Their expression on the tumor cell surface needs to be dynamically regulated throughout the process of metastasis. During the attachment phase to the endothelium, another group of CAMs, such as E‐/P‐selectins—primarily expressed on endothelial cells—may also play a critical role in certain malignancies. Additionally, the interplay between integrins and selectins may influence the tumor microenvironment. This review focuses on the role of integrins and their interplay with selectins in metastasis, emphasizing findings from in vivo studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9e434e2ef5f3a3703e0c8f6c7c1cc5f839b38f1" target='_blank'>
              Interplay of integrins and selectins in metastasis
              </a>
            </td>
          <td>
            Diana Maltseva, Ashot Nersisyan, Alexander Tonevitsky
          </td>
          <td>2025-05-06</td>
          <td>Molecular Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Patients with high-grade serous ovarian cancer (HGSOC) typically present with widespread metastasis, obscuring a temporal understanding of tumor-immune dynamics. To address this, we perform multi-site global proteomics alongside matched immunohistochemistry (IHC) for CD4⁺ and CD8⁺ tumor-infiltrating lymphocytes (TILs) in patient samples. We order the protein expression profiles using an unbiased pseudotime analysis, recapitulating clinical observations of metastatic progression, and providing a framework to explore tumor-immune dynamics from localized to metastatic disease. Metastatic progression correlates with immune cell infiltration, the recruitment of regulatory T cells (Tregs) to counterbalance γδ T cell abundance, and an increased abundance of exhausted CD8⁺ T cells. The accumulation of Tregs at metastatic sites correlates with SNX8 expression, a critical regulator of the STING pathway. In early-stage tumors, keratin-expressing cancer cells recruit Tregs via MHC class II, fostering an inflammatory phenotype with limited IFNγ production and non-clonally expanded T cells. Together, our findings reveal novel mechanisms of immune escape associated with both localized disease and metastatic progression in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5284b73444d2932c5ad492644667337328f9e74" target='_blank'>
              A temporal model of tumor-immune dynamics during the metastatic progression of high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Donagh Egan, Kate Glennon, Ann Treacy, Aurelie Fabre, Janet Mccormack, Salisha Hill, Ryan Morrison, Vadim Zhernovkov, Daniel C Liebler, Walter Kolch, Donal J Brennan
          </td>
          <td>2025-06-16</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="As the most aggressive primary brain tumor, glioblastoma (GBM) is considered incurable due to its molecular heterogeneity and therapy resistance. Identifying key regulatory factors in GBM is critical for developing effective therapeutic strategies. Based on the analysis of TCGA data, we confirmed a robust co-expression and correlation of OLIG1 and OLIG2 in human GBM. However, their roles in the astrocytic GBM subtype remain unclear. In this study, we first establish an astrocytic-featured GBM mouse model by introducing PiggyBac-driven hEGFRvIII plasmids and demonstrate that both OLIG1 and OLIG2 are highly expressed within this context. Next, using CRISPR/Cas9 technology to knockout Olig1/2, we found that astrocyte differentiation markers such as GFAP, SOX9, and HOPX were preserved, but tumor cell proliferation was significantly diminished. Mechanistically, CUT&Tag-seq revealed that OLIG1/2 directly binds to the promoter region of various cyclins (Cdk4, Ccne2, Ccnd3, and Ccnd1), where an enrichment of the active histone marker H3K4me3 was observed, indicating transcriptional activation of the genes. Notably, Olig1/2 knockout did not suppress tumor initiation or migration, suggesting that their primary role is to amplify proliferation rather than to drive tumorigenesis. This study defines Olig1 and Olig2 as master regulators of GBM proliferation through various cyclins, thereby offering a novel therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ba56851ac668f875e426e5e567cb95a07c19f0" target='_blank'>
              Olig1/2 Drive Astrocytic Glioblastoma Proliferation Through Transcriptional Co-Regulation of Various Cyclins
              </a>
            </td>
          <td>
            Yu Tian, Ziwu Wang, Mengge Sun, Jialin Li, Wenhui Zheng, Feihong Yang, Zhuangzhi Zhang
          </td>
          <td>2025-05-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/193dde17ef6934508de69e6a33e445fcab88581e" target='_blank'>
              Impaired T cell and neoantigen retention in time-serial analysis of metastatic non-small cell lung cancer in patients unresponsive to TIL cell therapy.
              </a>
            </td>
          <td>
            Chao Wang, Xiaoqing Yu, Jamie K Teer, Jiqiang Yao, Dongliang Du, Xiaoxian Liu, Zachary J Thompson, Min Hsuan Wang, Eric A. Welsh, Danish Memon, Timothy A Chan, Vladimir Makarov, C. Anadon, Lamees Saeed, Theresa A. Boyle, Bin Fang, John M. Koomen, Cheryl A Cox, Ana M Landin, Sean J Yoder, Sungjune Kim, Dung-Tsa Chen, Shari Pilon-Thomas, Jose R Conejo-Garcia, S. Antonia, E. Haura, Ben C Creelan
          </td>
          <td>2025-05-08</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Although the clinical significance of plasma cells within tumors has been recognized, studies on the development of plasma cells and the characteristics of the antibodies they secrete within the tumor microenvironment remain limited. We investigated the properties of plasma cells within cancer tissues using single-cell RNA and single cell B cell receptor sequencing. We characterized plasma cells exhibiting clonal expansion and synthesized the antibodies produced by these cells, confirming the clinical relevance of immunoglobulin H (IGH) isotypes. Plasma cells comprised approximately 5% of the total immune cell population within the tumor; clonal expansion was more prevalent in plasma cells than in B cells. Among plasma cells, the most frequent immunoglobulin isotype was IGHG1 and IGKC. We synthesized six recombinant antibodies, including those from the largest clonal plasma cells. Two antibodies that formed clones showed membranous staining in cancer cells. The cancer cells that metastasized to the lymph node showed a loss of expression as observed by immunohistochemistry. Analysis of bulk RNA sequencing data from 1078 patients with breast cancer revealed that tumor-infiltrating plasma cells expressing IGHG1 were associated with favorable prognoses. These tumors exhibited increased B cell receptor diversity, immunogenic mutation, and intratumoral heterogeneity. This study suggests the potential for discovering cancer-associated antibodies derived from intratumoral plasma cells. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04045-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54a09d3573d6929df29bef067a1cea30d57f996f" target='_blank'>
              Recombinant antibodies from clonally expanded cancer-associated plasma cells
              </a>
            </td>
          <td>
            Wonkyung Kim, Ji-Hye Oh, Chae Won Park, Hyori Kim, Young Gwang Kang, Da Eun Oh, H. Lee, Ji Hun Kim, Chang Ohk Sung
          </td>
          <td>2025-05-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 7070


 Background:
 Follicular lymphoma (FL) exhibits considerable variability in biological features and clinical trajectories, compounded by a complex tumor microenvironment (TME) populated with nonmalignant immune cells. In this study, we comprehensively characterized FL by integrating multi-omics data that considered both tumor cells and microenvironmental components. This analysis revealed four subtypes of FL, each exhibiting distinct biological characteristics and differing clinical behaviors.
 Methods:
 Our study included 1,203 samples, 53 bulk transcriptomic samples from our center, 1,150 samples from public databases, and five single-cell RNA sequencing (scRNA-seq) cohorts. The Ecotyper algorithm was employed to identify immune cell states. Unsupervised clustering analysis using non-negative matrix factorization (NMF) was performed to identify distinct FL subtypes based on cellular infiltration.
 Results:
 A total of 30 unique cell states were identified from nine annotated cell populations (B cells, plasma cells, CD4 T cells, CD8 T cells, Tregs, NK cells, Tfh, monocytes/macrophages, and dendritic cells). Validation was performed across three independent scRNA-seq and bulk transcriptomic cohorts to ensure robustness. Four distinct B cell states were identified, each exhibiting unique gene expression profiles and varying prognostic implications. B cell S2 and S3 were associated with adverse outcomes. Notably, PRDM15 in B cell S2 had the most significant prognostic impact, remaining an independent adverse prognostic factor even after adjustment for m7-FLIPI. Mutational profile analysis demonstrated distinct mutation patterns across B cell states: mutations in CCND3, SPEN, and ARID1A were enriched in B cell S1; CREBBP and STAT6 mutations were prominent in B cell S2; and IRF8 and BCL2 mutations were observed in B cell S4, while no significant mutation enrichment was detected in B cell S3. Similarly, immune cells of the same type yet in different states exhibited functional and clinical heterogeneity. Four subtypes of follicular lymphoma characterized by distinct tumor cell states and varying immune infiltration were identified through unsupervised clustering analysis. FLE1 exhibits characteristics of a "cold tumor" with a high abundance of B cell S2. In contrast, FLE2 and FLE3 show moderate immune infiltration, while FLE4 is marked by abundant immune infiltration and elevated expression of immunosuppressive checkpoint molecules, resembling an "inflammatory" tumor phenotype. Prognostic analysis indicates that FLE1 has the most unfavorable prognosis, followed by FLE4.
 Conclusions:
 Our study stratifies FL patients based on the heterogeneity of tumor cells and the immune microenvironment, proposing an immune-based classification for FL. Targeting CREBBP and PRDM15 may offer promising new strategies for the clinical management of patients exhibiting a "cold tumor" phenotype.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0fb822abee7b71f9392d723669dd1a087479f12" target='_blank'>
              Clinical and biological subtypes of follicular lymphoma revealed by tumor and immune cell states.
              </a>
            </td>
          <td>
            Hengqi Liu, Yingfang Feng, Ning Zhang, Cong Sun, Ning Zhang, Xianhuo Wang, Huilai Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (ICC) is characterized by high malignancy, and its global incidence is predicted to continue to increase over the past decades. However, the mechanisms underlying ICC pathogenesis and progression remain unclear.


APPROACH AND RESULTS
The training cohort consisted of single-cell sequencing of 12 treatment-naïve ICC samples and spatial transcriptomics of four ICC samples. The validation cohort comprised of RNA-seq data from 87 ICC tumor samples. Finally, we validated our findings via multiplex immunofluorescence, organoids, and mice models both in vivo and in vitro. We found significant heterogeneity within the tumor microenvironment (TME) of ICC patients. ICC cells were classified into five molecular subtypes, and we revealed that aspartate beta-hydroxylase (ASPH) was a marker gene for invasion subtypes. We then selected cepharanthine (CEP) as an ASPH inhibitor that effectively suppressed tumor progression. Regarding the ICC lymphatic metastasis mechanism, we found that tumor cells in N1 lymph nodes highly expressed tumor-specific MHC-II molecules but lacked co-stimulatory factors CD80/CD86, inducing a state of anergy in CD4+ T cells, which might facilitate ICC immune evasion.


CONCLUSIONS
The TME of ICC was heterogeneous. ASPH markedly enhanced ICC invasion The ASPH inhibitor CEP significantly inhibits ICC progression and may serve as a targeted therapeutic drug for ICC. Tumor cells in N1 lymph nodes demonstrate high expression of tumor-specific MHC-II molecules, but silencing of co-stimulatory factors such as CD80/CD86 induces CD4+ T cells into an anergic state. Our study indicated that ASPH and MHC-II may serve as novel therapeutic targets for ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2012ee56dc72ff219a9edcce829b5a1b74d80284" target='_blank'>
              Tumor microenvironment heterogeneity and progression mechanisms in intrahepatic cholangiocarcinoma: A study based on single-cell and spatial transcriptomic sequencing.
              </a>
            </td>
          <td>
            Fengwei Li, Yao Li, Lishan Wang, Lei Xu, Hui Xue, Wenxin Wei, Yong Xia, Lei Wang, Feng Shen, Kui Wang
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/525bc26ccdba95190ad1d88938dd65bd9ec49925" target='_blank'>
              Cell painting in activated cells illuminates phenotypic dark space and uncovers novel drug mechanisms of action
              </a>
            </td>
          <td>
            Matylda Zietek, Akshar Lohith, Derfel Terciano, Beverley Marie Rabbitts, Aswad S. Khadilkar, John B MacMillan, Scott Lokey
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Accumulating research suggests that Epstein-Barr Virus-positive Diffuse Large B-cell Lymphoma (EBV+DLBCL) is associated with immune dysfunction and tumor microenvironment (TME) heterogeneity. While the prognostic role of the TME in EBV-DLBCL is established, its impact on EBV+DLBCL survival remains unclear. Here, we integrated 10X Visium spatial transcriptomics (ST) with single-cell RNA sequencing (scRNA-seq) to map TME heterogeneity in EBV+DLBCL. Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses identified PD-1/PD-L1 signaling as a hallmark of EBV+DLBCL’s immunosuppressive TME. Functional validation using the PD-1/PD-L1 inhibitor BMS202 revealed dose-dependent suppression of proliferation and enhanced apoptosis in EBV+Farage cells, with TLR4 emerging as a downstream effector showing EBV-status-dependent regulation. These findings not only link TME-driven PD-1/PD-L1 activation to EBV+DLBCL’s poor prognosis but also provide preclinical evidence for PD-1/PD-L1 blockade as a therapeutic strategy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00410-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1796a1ec9b1ce6b656d31aa53096b6765637e098" target='_blank'>
              Spatial transcriptomics reveals tumor microenvironment heterogeneity in EBV positive diffuse large B cell lymphoma
              </a>
            </td>
          <td>
            Mei-Yao He, Meng Liu, Jiayin Yuan, Jin Lv, Wei Li, Qianwen Yan, Yujiao Tang, Luyi Wang, Li Guo, Fang Liu
          </td>
          <td>2025-05-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell heterogeneity is a universal feature of life. Although biological processes affected by cell-to-cell variation are manifold, from developmental plasticity to tumour heterogeneity and differential drug responses, the sources of cell heterogeneity remain largely unclear1,2. Mutational and epigenetic signatures from cancer (epi)genomics are powerful for deducing processes that shaped cancer genome evolution3–5. However, retrospective analyses face difficulties in resolving how cellular heterogeneity emerges and is propagated to subsequent cell generations. Here, we used multigenerational single-cell tracking based on endogenously labelled proteins and custom-designed computational tools to elucidate how oncogenic perturbations induce sister cell asymmetry and phenotypic heterogeneity. Dual CRISPR-based genome editing enabled simultaneous tracking of DNA replication patterns and heritable endogenous DNA lesions. Cell lineage trees of up to four generations were tracked in asynchronously growing cells, and time-resolved lineage analyses were combined with end-point measurements of cell cycle and DNA damage markers through iterative staining. Besides revealing replication and repair dynamics, damage inheritance and emergence of sister cell heterogeneity across multiple cell generations, through combination with single-cell transcriptomics, we delineate how common oncogenic events trigger multiple routes towards polyploidization with distinct outcomes for genome integrity. Our study provides a framework to dissect phenotypic plasticity at the single-cell level and sheds light onto cellular processes that may resemble early events during cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc8a5871308fcb63b1e32d25ce00cce571108d0" target='_blank'>
              Multigenerational cell tracking of DNA replication and heritable DNA damage
              </a>
            </td>
          <td>
            Andreas Panagopoulos, Merula Stout, Sinan Kılıç, Peter Leary, Julia Vornberger, Virginia Pasti, A. Galarreta, Aleksandra Lezaja, Kyra Kirschenbühler, Ralph Imhof, H. Rehrauer, Urs Ziegler, Matthias Altmeyer
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>42</td>
        </tr>

        <tr id="
 e16452


 Background:
 Liver metastasis is a leading cause of death amongst localized pancreatic ductal adenocarcinoma (PDAC) patients following surgical resection. Yet, there are no adjuvant therapies currently available targeting liver colonization in PDAC. We are hindered in our ability to develop such therapies by an incomplete understanding of PDAC liver metastasis' molecular mechanisms.
 Methods:
 To address this challenge, we labeled primary mouse PDAC subclones with DNA barcodes to characterize their pre-metastatic state using ATAC-seq and RNA-seq and determine their relative
 in vivo
 liver metastasis potential in competition assays in immunocompetent hosts.
 Results:
 We identified a gene signature separating metastasis-high and metastasis-low subclones orthogonal to the normal-to-PDAC and classical-to-basal axes. The metastasis-high subclones feature activation of inflammation-related genes and high NF-κB and Zeb/Snail family activity and the metastasis-low subclones feature activation of neuroendocrine, motility, and Wnt pathway genes and high CDX2 and HOXA13 activity. In a functional screen, we validated novel mediators of PDAC liver metastasis related to inflammation, including the NF-κB targets
 Fos
 and
 Il23a
 , and beyond inflammation including
 Myo1b
 and
 Tmem40
 . We scored human PDAC tumors for our signature of metastatic potential from mouse and found that metastases have higher scores than primary tumors. Moreover, primary tumors with higher scores are associated with worse prognosis. We also found that our metastatic potential signature is enriched in other human carcinomas, suggesting that it is conserved across epithelial malignancies.
 Conclusions:
 This study establishes a model of clonal evolution during PDAC liver metastasis under immune selective pressure, revealing novel functional regulators amenable to therapeutic intervention in the adjuvant setting. Furthermore, it provides a method for scoring human carcinomas based on metastatic potential that may have prognostic and predictive value across multiple cancer subtypes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d54beefd4d0e886101bb6db8b7a938594843dac" target='_blank'>
              Effect of clonal competition model of PDAC liver metastasis on a pan-carcinoma gene signature of metastatic potential.
              </a>
            </td>
          <td>
            Jesse S Handler, Zijie Li, Rachel K Dveirin, Jessica Lin, Weixiang Fang, James Forsmo, Hani Goodarzi, E. Fertig, Reza Kalhor
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy; however, no validated treatments are currently available. Clinically, the tumor position, head and uncinate process (HU), or body and tail (BT) of the pancreas are vital for surgical strategies; however, fundamental research has seldom revealed the heterogeneous tumor microenvironment (TME) among different types of PDAC. Here, we applied multicohort single-cell and spatial RNA-seq methods together with patient-derived organoid models to reveal the TME heterogeneity between HU and BT PDAC. Osteopontin, encoded by SPP1, is secreted by vessel endothelial cells in BT PDAC and is associated with increased tumor burden. The number of tumor cells marked by CRABP2 was lower in BT PDAC and was identified as a prognostic marker of overall survival, as well as CD8+ T-cell infiltration. The expression of CRABP2 was also validated to be downregulated in BT PDAC in patient-derived organoid models. Overall, we profiled the heterogeneous PDAC TME between HU and BT PDAC, which could provide novel insight into the relationships between clinical characteristics and TME molecular research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077490657a2feb3d7ea606c6fed7a402a475e6c0" target='_blank'>
              SPP1+ Venous Endothelial Cells and CRABP2+ Tumor Cells Contribute to Favorable Body and Tail Pancreatic Ductal Adenocarcinoma Tumor Microenvironment.
              </a>
            </td>
          <td>
            Yueze Liu, Yifan Fu, Tao Liu, Jun Wang, Zeyu Zhang, Yanan Shi, Zhe Cao, Gang Yang, Hao Chen, W. Luo, Jinxin Tao, Yuanyang Wang, Guihu Weng, Menggang Zhang, Liyuan Ye, Jianchun Xiao, J. Qiu, Taiping Zhang, Hua Huang
          </td>
          <td>2025-06-11</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Alterations in glycolysis play a crucial role in cancer cells, influencing tumor aggressiveness and therapeutic effect, particularly in pancreatic adenocarcinoma (PAAD). However, the specific glycolysis-related genes involved in PAAD progression remain poorly understood. This study established glycolysis-related molecular subtypes with distinct survival outcomes using TCGA datasets. The favorable prognosis subtype exhibited enhanced immune infiltration and an activated tumor microenvironment. A glycolysis prognostic model effectively predicted PAAD survival, correlating with global glycolytic pathways, and AUCell evaluated neutrophil communication networks of models. Functional validation demonstrated that ENO1/PGM2L1 co-expression promoted tumor proliferation, migration, invasion, and glycolytic flux in vitro, while accelerating xenograft growth in vivo. Conversely, their knockdown suppressed malignancy. Our study demonstrated that the glycolytic prognostic risk model serves as a reliable tool for prognosis and prediction of PAAD progression. ENO1 and PGM2L1 emerge as key risk factors promoting the malignant progression of PAAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbf7a9ccf491c81be3110e0f15ea439fc392789" target='_blank'>
              Integrative Single-Cell and Bulk RNA Sequencing Identifies a Glycolysis-Related Prognostic Signature for Predicting Prognosis in Pancreatic Cancer
              </a>
            </td>
          <td>
            Nan Wu, Chong Zhou, Xu Yan, Ziang Liu, Ruohan Jiang, Yuzhou Luo, Ping Jiang, Yu Mu, Shan Xiao, Xien Huang, Yun-Yun Zhou, Donglin Sun, Yan Jin
          </td>
          <td>2025-05-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 9562


 Background:
 The groundbreaking neoadjuvant ICB clinical trials have established beyond doubt that ICB before surgery will become the new standard of care for metastatic melanoma. Importantly, the shift from radiographic to pathologic response scoring offers an unprecedented window of opportunity to interrogate a goldmine of ‘on-treatment’ biospecimens. To date, single-cell profiling and region-based spatial transcriptomics (ST) have highlighted the importance of tertiary lymphoid structures (TLS) and stem-like T-cells as positive predictors of ICB response. However, the molecular mechanisms by which tumor infiltrating lymphocytes communicate and organize multicellular immune hubs within the spatial context of the tumor ecosystem remains poorly understood.
 Methods:
 To identify robust biomarkers of response to neoadjuvant ICB, we assembled a cohort of 58 stage III melanoma patients treated with neoadjuvant ICB [24 ipilimumab-nivolumab (IPI-NIVO), 21 NIVO-relatlimab (RELA), 13 PD1 mono] and deployed transformative technologies and computational methods, including single-cell FFPE sequencing, multiplexed FISH single-cell ST (MERFISH 305 genes), digital pathology, 3D open-top light-sheet imaging and AI-based computational pathology, to decipher the complex neoadjuvant ICB tumor ecosystem in response to therapy. We also developed 2 critical computational tools, SCIRA (Spatial Cellular Interaction and Receptor Activation), to compute receptor-ligand (R-L) interactions in whole slide images, and GC-SCAN (graph-based spatial clustering against noise), a graph-based algorithm that quantifies locally clustered structures from spatial -omics data.
 Results:
 Our results showed that the quantity and size of hyper-expanded germinal center/TLS, with increased GC: non-GC B-cell ratio, plasma cells and spatially resolved stem-like T-cells are strongly associated with response. IPI-NIVO elicited significantly stronger GC proliferation compared to NIVO-RELA and PD1 monotherapy, suggesting anti-CTLA4 can robustly induce germinal center/TLS proliferation. SCIRA spatial R-L analyses revealed the critical chemokine R-L interactions that organize the GC- and T-cell zones in response to therapy. Lastly, 3D light-sheet imaging revealed remarkable morphologic heterogeneity in 3D, with interconnected GC-TLS networks that are indicative of long-range molecular gradients.
 Conclusions:
 Our investigations herein have provided a comprehensive characterization of the immune architectures, cellular communications and 3D large-scale morphologic organizations of the TME that drive response to neoadjuvant ICB therapy. We believe the results of this study will enable the development of robust predictive biomarkers to guide the design of next generation combination ICB therapies in the clinical trial setting for melanoma and other cancer types.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb50bf439bc9a3d8fb630c6a0899f89e6769df7" target='_blank'>
              Multidimensional, spatially resolved immunologic hallmarks of response to neoadjuvant immune checkpoint blockade (ICB) therapies.
              </a>
            </td>
          <td>
            Zichao Liu, Xiaofei Song, Shikhar Dhingra, Wei-Shen Chen, Justin He, Jodi A. Balasi, Jonathan Nguyen, Carlos M. Morán-Segura, Joseph Johnson, Chaomei Zhang, Zena Sayegh, Douglas C. Marchion, Sean Yoder, Nicholas Reder, Jeffrey H. Chuang, Pei-Ling Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b812a28564555f6b348829df1f2e0c57b249620f" target='_blank'>
              Quasi-spatial single-cell transcriptome based on physical tissue properties defines early aging associated niche in liver.
              </a>
            </td>
          <td>
            Kwon Yong Tak, Juyeon Kim, Myungsun Park, Wooseok Kim, Seoyeong Lee, Narae Park, Min Jeong Kim, Ju-Bin Kang, Yongjun Koh, Hae Young Yang, Min Kyu Yum, Injune Kim, Yong Ryoul Yang, Won-Il Jeong, Jinsung Yang, Cheolju Lee, Chuna Kim, Jong-Eun Park
          </td>
          <td>2025-05-05</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Locally advanced esophageal adenocarcinoma remains difficult to treat and the ecological and evolutionary dynamics responsible for resistance and recurrence are incompletely understood. Here, we performed longitudinal multiomic analysis of patients with esophageal adenocarcinoma in the MEMORI trial. Multi-region multi-timepoint whole-exome and paired transcriptome sequencing was performed on 27 patients before, during and after neoadjuvant treatment. We found major transcriptomic changes during treatment with upregulation of immune, stromal and oncogenic pathways. Genetic data revealed that clonal sweeps through treatment were rare. Imaging mass cytometry and T cell receptor sequencing revealed remodeling of the tumor microenvironment during treatment. The presence of genetic immune escape, a less-cytotoxic T cell phenotype and a lack of clonal T cell expansions were linked to poor treatment response. In summary, there were widespread transcriptional and environmental changes through treatment, with limited clonal replacement, suggestive of phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286e0cfb0095dd021b03614405167d3960808941" target='_blank'>
              Evolutionary and immune microenvironment dynamics during neoadjuvant treatment of esophageal adenocarcinoma
              </a>
            </td>
          <td>
            M. Barroux, J. Househam, E. Lakatos, T. Ronel, Ann-Marie Baker, Henrike Salié, M. Mossner, Kane Smith, C. Kimberley, S. Nowinski, Alison Berner, V. Gunasri, Martin Borgmann, S. Liffers, Marnix Jansen, G. Caravagna, K. Steiger, J. Slotta-Huspenina, Wilko Weichert, Luis Zapata, Eleftheria Giota, Sylvie Lorenzen, M. Alberstmeier, Benjamin M Chain, H. Friess, B. Bengsch, Roland M. Schmid, J. Siveke, M. Quante, Trevor A. Graham
          </td>
          <td>2025-05-14</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, Muhammad T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Brain tumors, in particular gliomas and glioblastoma multiforme (GBM), are thought to originate from different cells facing specific founding insults, a feature that partly justifies the complexity and heterogeneity of these severe forms of cancer. However, gliomas and GBM are usually reproduced in animal models by inducing molecular alterations in mature glial cells, which, though being part of the puzzle, do not represent the whole picture. To fill this conceptual gap, we previously developed a neurogenic model of brain cancer in Drosophila, demonstrating that the loss of cell polarity in neural stem cells (called neuroblasts in the fruit fly) is sufficient to promote the formation of malignant masses that continue to grow in the adult, displaying several phenotypic traits typical of human GBM. Here, we expand on previous work by restricting polarity disruption to Drosophila type II neuroblasts, whose self-renewal is comparable to that of mammalian neural progenitors, with the aim to capture the molecular signature of the resulting cancers in a specific and reproducible context. A comparison of the most deregulated transcripts with those found in human primary GBMs confirmed that our model can be proficiently used to delve into the roots of human brain tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/655c1526e0548f12b64a21fe0105b8cfbd172571" target='_blank'>
              Transcriptome-Wide Analysis of Brain Cancer Initiated by Polarity Disruption in Drosophila Type II Neuroblasts
              </a>
            </td>
          <td>
            Simona Paglia, P. Morciano, Dario de Biase, F. Giorgi, A. Pession, Daniela Grifoni
          </td>
          <td>2025-05-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are the most abundant non-cancerous cell type in glioblastoma (GBM) and heavily influence GBM biology, contributing to tumor progression, therapeutic resistance, immune evasion, and neovascularization. Current in vitro models that utilize IL-4/IL-13 stimulation fail to capture the transcriptional and functional heterogeneity of TAMs observed in vivo. In this study, we utilize a serum-free indirect co-culture model with patient-derived xenolines to polarize primary human macrophages and characterize their molecular and functional phenotypes. We demonstrate that xenoline-polarized macrophages diverge from classical M1/M2 states and instead adopt transcriptional signatures reflective of TAM subsets identified from patients. Notably, macrophages polarized with the radiation-therapy selected xenoline, JX14P-RT, exhibited gene expression patterns enriched for interferon response and hypoxia, mirroring recurrent GBM samples. In contrast, JX14P TAMs showed enrichment in phagocytic gene sets. Functional validation of these phenotypes revealed discrepancies between the transcriptionally predicted and observed phenotypes, emphasizing the importance of integrating phenotypic validation in sequencing studies. Altogether, our findings establish xenoline-polarized macrophages as a more physiologically relevant alternative to traditional models, offering a useful model for studying tumor-immune interaction in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e59e91c3e7e604dd7d80dd0333a4e13f653a5b4e" target='_blank'>
              Xenoline-polarized macrophages as a physiologically relevant in vitro model of tumor-associated macrophages in glioblastoma
              </a>
            </td>
          <td>
            Hasan Alrefai, Benjamin Lin, Amr Elkohly, Manoj Kumar, Taylor L. Schanel, Kevin J. Lee, Patricia H. Hicks, Joshua C Anderson, Gao Guo, Eun-Young Erin Ahn, C. R. Miller, Christopher D. Willey
          </td>
          <td>2025-05-09</td>
          <td>Research Square</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Colorectal cancer (CRC), the third most common cancer worldwide, often shows limited responsiveness to immunotherapy due to its predominantly immune-excluded phenotype. Despite increasing insights into the complex tumor microenvironment (TME), the metabolic heterogeneity of CRC cells and their interactions with tumor-infiltrating immune cells remain poorly understood. Methods We analyzed 46,374 epithelial cells from 17 CRC patients treated with PD-1 blockade to develop an amino acid (AA) metabolism score using the AUCell algorithm. This score was applied to a separate single-cell RNA sequencing (scRNA-seq) dataset from 23 CRC patients to investigate cell-cell interactions and functions of tumor-infiltrating immune cells, revealing distinct immune TME landscapes shaped by tumor metabolism. An in vitro co-culture assay of CRC cells and CD8+ T cells was performed to validate the findings. Additionally, LASSO and Cox regression analyses were conducted to construct an AA metabolism-related risk score for predicting prognosis and drug sensitivity across multiple bulk transcriptome cohorts. Results This study identified a link between elevated amino acid metabolism in CRC epithelial cells and resistance to PD-1 blockade therapy. A 31-gene AA score was developed by intersecting differentially expressed genes between responders and non-responders to PD-1 blockade with amino acid metabolism-related genes from the Molecular Signature Database (MSigDB). Using this score, 23 additional CRC samples were classified into high and low AA score groups. Comparative analysis revealed that the low AA group exhibited a more robust immune response, characterized by a greater number and stronger cell-cell interactions. Tumor-infiltrating immune cells in this group demonstrated enhanced activation and anti-tumor functions. Furthermore, CD8+ T cells showed increased Granzyme B levels when co-cultured with CRC cells in which Psat1 or Shmt2 was knocked down. Finally, a machine learning-derived risk score based on six genes was established to translate single-cell findings to bulk transcriptomes. This risk score was found to correlate with immune checkpoint expression and immune cell infiltration, with potential implications for predicting prognosis and drug sensitivity. Conclusion Our findings highlight the role of elevated epithelial amino acid metabolism in shaping an immune-suppressive microenvironment, offering insights for patient stratification and therapeutic decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b9bda18d8cb73c7c5646b13e67e456908ec19a2" target='_blank'>
              Single-cell and bulk transcriptome analyses reveal elevated amino acid metabolism promoting tumor-directed immune evasion in colorectal cancer
              </a>
            </td>
          <td>
            Tianyue Sun, Yan Chen, Ying-Xuan Chen
          </td>
          <td>2025-05-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="PURPOSE Colorectal cancer (CRC) ranks third in global incidence and second in mortality, with rates increasing among younger populations. The enteric nervous system (ENS) is crucial for gastrointestinal function, and its dysfunction is associated with the progression of CRC. Neurodegeneration may disrupt ENS function and facilitate tumor growth although the mechanisms remain unclear. METHODS AND MATERIALS This study integrated single-nucleus, single-cell RNA sequencing, spatial transcriptomics (ST), and bulk transcriptomics data. Summary data-based Mendelian randomization (SMR) analysis identified neurodegeneration-related genes associated with CRC progression. Cell-cell communication was analyzed using CellCall and Cytotalk, and trajectory analysis performed using VECTOR. ST analysis was conducted using robust cell type decomposition, and spatial dependencies were assessed through MISTy and cell neighborhood network analyses. A deep learning–based prognostic model was developed using DeepSurv. Multiplex immunofluorescence was used to validate the functions of cells. RESULTS SMR analysis identified seven neurodegeneration-related genes, including MAPK9, associated with the development of CRC. High-neurodegeneration neurons were associated with poorer prognoses and infiltrated deeper into tumor niches. MISTy analysis revealed stronger spatial dependencies between high-neurodegeneration neurons and epithelial cells. Homotypic and heterotypic cell neighborhood network analyses revealed tighter neuron networks in high-degeneration groups, with enhanced neuron-epithelial communication. MAPK9-positive neurons were highly expressed in sensory neurons and facilitated tumor growth via EGFR signaling. The DeepSurv model demonstrated strong prognostic performance, with high-risk patients exhibiting worse outcomes and elevated expression of immune checkpoint genes. CONCLUSION Neurodegeneration, particularly involving MAPK9-positive neurons, promotes CRC progression through enhanced neuron-tumor interactions and EGFR signaling. This study underscores the significance of targeting neurodegeneration in CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ebbbaa1c310c2d03f5809c332bdce3f1b5993a" target='_blank'>
              Neurodegeneration Promotes Tumorigenesis in Colorectal Cancer: Insights From Single-Cell and Spatial Multiomics
              </a>
            </td>
          <td>
            Shaocong Mo, Bochen Pan, Liming Qin, Sheng Wang, Yulin Wang, Midie Xu, Fan Xia, Wenchao Gu
          </td>
          <td>2025-06-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e010b52ec17520834defd15d1777a5ca8afbac" target='_blank'>
              A systematic approach to analyze T cell migration: application to mouse melanoma tumors
              </a>
            </td>
          <td>
            Nikolaos Memmos, Jason S. Mitchell, B. Fife, D. Masopust, David J. Odde
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="
 e15533


 Background:
 Colorectal cancer (CRC) remains a major global health concern due to its high prevalence and mortality, especially in advanced stages. Despite advancements such as immune checkpoint inhibitors, resistance and tumor heterogeneity remain significant obstacles. Recent research highlights the β2-adrenergic receptor (ADRB2) signaling pathway as a crucial factor in T cell exhaustion, which weakens the immune system’s anti-tumor response. Inhibiting this pathway could restore immune function against CRC. Single-cell RNA sequencing (scRNA-seq) has enhanced understanding of the tumor microenvironment, offering new opportunities for discovering biomarkers and therapeutic targets for personalized treatment strategies.
 Methods:
 We applied scRNA-seq and bulk RNA sequencing to examine adrenergic receptor signaling in CRC, particularly before and after neoadjuvant therapy. Bioinformatics analyses identified T cells expressing ADRB2, confirmed by immunofluorescence and flow cytometry. Copy number variation (CNV) analysis detected malignant cells in the tumor microenvironment, and intercellular communication studies were conducted to explore interactions between ADRB2+ T cells and other immune cells. RNA scope was employed to map ADRB2 distribution in tissue samples. Additionally, deconvolution of TCGA data revealed important immune cell populations, helping to build a prognostic model based on adrenergic receptor signaling and therapy-induced gene expression changes.
 Results:
 The adrenergic receptor signaling pathway was notably enriched in immune cells, especially T cells. After neoadjuvant therapy, scRNA-seq revealed a significant increase in ADRB2 expression in T cells. RNA scope validated ADRB2 co-localization with T cells in CRC tissues, while flow cytometry showed increased ADRB2 expression alongside markers of T cell exhaustion and reduced immune function. CNV analysis identified malignant populations within the tumor microenvironment. Studies of intercellular interactions suggested that ADRB2+ T cells contribute to immune suppression. TCGA data deconvolution indicated that ADRB2+ T cells and CD44+ tumor cells are critical for prognosis. The resulting prognostic model, integrating adrenergic signaling and post-treatment gene expression changes, demonstrated strong predictive power for patient outcomes.
 Conclusions:
 Increased ADRB2 expression in T cells after neoadjuvant therapy in CRC correlates with poor prognosis. This positions ADRB2 as a promising prognostic biomarker and a potential therapeutic target to enhance immunotherapy effectiveness following neoadjuvant treatment. Targeting ADRB2 could offer novel approaches to alleviate immune exhaustion and improve patient outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65e2be39dd18963a3eba9d28b1c8ce6324042919" target='_blank'>
              ADRB2-driven immunosuppression as a novel prognostic indicator and therapeutic avenue following neoadjuvant treatment in colorectal cancer.
              </a>
            </td>
          <td>
            Songbing He, Tianliang Liu, Yingyi Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor diseases are characterized by high interindividual and intratumoral heterogeneity (ITH). The development and progression of neoplasms outline complex networks of extracellular and cellular signals that have yet to be fully elucidated. This narrative review provides a comprehensive overview of the literature related to the cellular and molecular mechanisms underlying the heterogeneity of the tumor mass. Furthermore, it examines the possible role of the tumor microenvironment in the development and support of the neoplasm, in order to highlight its potential in the construction of a diagnostic–therapeutic approach to precision medicine. Many authors underline the importance of the tumor microenvironment (TME) as it actively takes part in the growth of the neoplastic mass and in the formation of metastases and in the acquisition of resistance to anticancer drugs. In specific body districts, the ideal conditions occur for the TME establishment, particularly the inflammatory state, the recruitment of cell types, the release of specific cytokines and growth factors, hypoxic conditions. These components actively intervene by enabling tumor progression and construction of physical barriers shaped by the extracellular matrix that contribute to forming peripheral tolerance by intervention of myeloid precursors and the polarization of M2 macrophages. In recent years, ITH and the TME have assumed an important position in cancer research and pharmacology as they enable understanding the dense network of communication existing between the neoplasm and the surrounding environment, and to monitor and deepen the effects of drugs with a view to develop increasingly precise and effective therapies. In the last decade, knowledge of TME has been exploited to produce targeted molecular agents (inhibitory small molecules, monoclonal antibodies, gene therapy). Nonetheless, the bibliography shows the need to study ITH through new prognostic and predictive biomarkers (e.g., ctDNA and CTCs) and to increase its basic biology knowledge. Precision medicine is a new opportunity in the treatment of oncological diseases that is transforming the development of new drug approaches and their clinical use. Biology and biotechnologies are providing the bases for this revolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9eef0c89f58575a3286340faed6565d85cc9b7" target='_blank'>
              Microenvironment and Tumor Heterogeneity as Pharmacological Targets in Precision Oncology
              </a>
            </td>
          <td>
            S. Tonello, Roberta Rolla, Paolo Amedeo Tillio, PP. Sainaghi, Donato Colangelo
          </td>
          <td>2025-06-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Breast cancer exhibits heterogeneity at various levels, from subtype differences between patients (inter-patient heterogeneity) to diverse malignant cell compositions, variability in hormone receptor (HR) expression, and cellular makeup within single breast cancer samples (intra-tumour heterogeneity). Despite the availability of numerous effective therapies, many patients still experience incomplete treatment responses and subsequent relapse. These adverse outcomes are often due to the critical but frequently overlooked factor of cellular heterogeneity.
 To gain deeper insights into intra-tumour heterogeneity, we applied single-cell technologies to a cohort of 250 primary, untreated breast cancers. We optimized tissue cryopreservation methods, eliminating the need for fresh sample processing, and developed multiplex tissue profiling techniques. These cost-efficient processes improve handling of small tissue sizes, such as biopsies, and reduce batch effects. To ensure accurate and reliable data processing, we created a scalable computational workflow that includes benchmarked methods for sample SNP-demultiplexing, doublet detection, high-resolution cell annotation, and cellular integration. Additionally, we extended our existing methods to study the cellular heterogeneity of breast cancers.
 Our method, scSubtyper, examines the phenotypic differences between malignant cells within tumours by comparing each single cell to features associated with different molecular subtypes and assigning each cell to one of these subtypes. Our previous study and preliminary results from this project revealed that over 90% of samples exhibit a mix of malignant cells of different subtypes, and 50% of samples contain cells with characteristics of all subtypes, demonstrating that cellular heterogeneity exists not only between malignant cells but also within malignant cells of a tumour. Our second approach, known as ecotyping, assesses patterns of cell type frequencies across samples and groups them based on the similarity of these co-occurrences. Preliminary results have revealed the existence of five ecotypes that lack significant associations with clinical subtypes. Application of the same approach exclusively within the HR-positive samples identified four ecotypes characterized by distinct abundances of immune and stromal cells. This analysis revealed that ecotypes are not simple surrogates for clinical and molecular subtypes, but their presence could influence different responses to treatment.
 Together, our high-throughput tissue processing and computational approaches to study intra-tumour heterogeneity are now being applied to our large, well-annotated clinical cohort. Supported by preliminary results, we hypothesize that this study will play a vital role in optimizing breast cancer patient stratification to improve treatment management and outcomes.
 Citation Format: Beata Kiedik, Daniel L. Roden, Kate Harvey, Ghamdan Al-Eryani, Sunny Z. Wu, Mun N. Hui, Sandra O’Toole, Elgene Lim, Charles M. Perou, Alexander Swarbrick. Exploring cellular heterogeneity of localised breast cancers [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-09-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/355a116d2a0ae027b0d9081aa9023a8465a9971c" target='_blank'>
              Abstract P4-09-19: Exploring cellular heterogeneity of localised breast cancers
              </a>
            </td>
          <td>
            Beata Kiedik, D. Roden, K. Harvey, G. Al-Eryani, Sunny Z. Wu, M. Hui, Sandra O’Toole, Elgene Lim, C. Perou, Alex Swarbrick
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Simple Summary Single-label immunohistochemistry (sIHC) is used in infectious disease studies to detect etiological agents, and to quantify and spatially localize antigens in lesions. While sIHC is limited to staining one antigen per tissue section, multiplex immunohistochemistry (mIHC) offers significantly more detailed information and is more cost- and time-efficient than repeated sIHC. Given that swine lymph node pathology involves variations in T and B lymphocytes and macrophages, this study aims to apply mIHC to visualize simultaneously and on the same slide their spatial distribution and variation across the three main regions of the swine lymph node: follicles, interfollicular areas and medulla-like tissue. We demonstrated the use of bright-field mIHC for in situ analysis of spatial cellular distribution in swine lymph nodes during PCV2-SD and PRRSV lymphadenopathy, revealing distinct lymphocyte distributions compared to reactive lymphoid hyperplasia. The study highlights the potential of mIHC as a valuable tool for understanding and studying disease dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7c47de3fe690520b75f0437e0eae36becf35b80" target='_blank'>
              Bright-Field Multiplex Immunohistochemistry in Swine PCV2 and PRRSV Lymphadenopathies
              </a>
            </td>
          <td>
            G. D’Annunzio, L. V. Muscatello, Chiara Tugnoli, Stefano Pesaro, Andrea Luppi, Michelangelo Fiorentino, T. Franceschini, Alessia Grillini, G. Rugna, Giuseppe Sarli, L. Mandrioli
          </td>
          <td>2025-06-01</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA), a neoplasm arising from biliary epithelial cells, is particularly widespread in Southeast Asia, with northeastern Thailand exhibiting the greatest prevalence attributed to Opisthorchis viverrini infection. This malignancy exhibits considerable molecular heterogeneity, leading to therapeutic resistance and recurrence. Comprehending its molecular mechanisms is essential for enhancing diagnostic and treatment approaches. Our research utilized multi-region LC-MS/MS proteomic analysis to investigate intratumor heterogeneity (ITH) in CCA. We examined 52 tumor areas and 13 neighboring tissues from 13 patients, concentrating on protein profiling, pathway analysis, differential protein expression, and the identification of shared and unique protein signatures. The findings indicated considerable inter-patient proteome variability, characterized by markedly distinct protein expressions among individuals, aligning with prior cancer research. Intra-tumor heterogeneity was apparent, with merely 18 proteins common to all tumor areas and patients, underscoring the intricacy of CCA. Significantly, the common proteins were associated with metabolic reprogramming and oxidative stress pathways, indicating possible indicators and therapeutic targets. This work highlights the significant proteome variability in CCA at both intra-tumor and inter-patient levels, underscoring the necessity for customized therapeutic approaches to tackle the disease’s complexity and improve treatment outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-02713-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afe6a3a1d14e85f2bd0571da7c1e456fa657e91b" target='_blank'>
              Proteomic profiling reveals common and region-specific protein signatures underlying tumor heterogeneity in cholangiocarcinoma
              </a>
            </td>
          <td>
            Sirinya Sitthirak, S. Roytrakul, Arporn Wangwiwatsin, N. Namwat, Poramate Klanrit, Hasaya Dokduang, P. Sa-Ngiamwibool, Attapol Titapan, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2025-05-18</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a4637e215d477abe1d7339165f5ea3e33b4aff9" target='_blank'>
              Transcriptional Roadmap of the Human Airway Epithelium Identifying HLF as a Novel Regulator of Basal Stem Cell Function
              </a>
            </td>
          <td>
            P. Prabhala, Sofia Freiman, N. Gvazava, Jiten Sharma, S. Wijk, K. Kanzenbach, Stefan Lang, Rebecka Cattani, Jenny Wigén, D. Bryder, S. Soneji, Johan Flygare, E. Tufvesson, Leif Bjermer, Darcy E. Wagner, G. W. Thorsson, Mattias Magnusson
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Evolution and outgrowth of drug-resistant cancer cells is a common cause of treatment failure. Patients with leukemic cutaneous T-cell lymphoma (L-CTCL) have a poor prognosis due to development of drug-resistance and severe bacterial infections. Here, we show that most L-CTCL patients harbor multiple genetically distinct subclones that express an identical clonal antigen receptor but display distinct phenotypes and functional properties. These co-existing malignant subclones exhibit differences in tissue homing, metabolism, and cytokine expression, and respond differently to extrinsic factors like Staphylococcus (S.)aureus and cancer drugs. Indeed, while S. aureus toxins selectively enhance activation and proliferation of certain subclones, these responsive subclones are also the most intrinsically sensitive to cancer drugs when the stimuli are removed. Consequently, although divergent evolution of malignant subclones drives aggressiveness, adaptability and drug-resistance, by removing extrinsic stimuli and mapping malignant subclones, we can expose inherent vulnerabilities that can be exploited in the treatment of these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9c75ce389008c94fe5703552a6c93e0ffef4c7" target='_blank'>
              Divergent Evolution of Malignant Subclones Maintains a Balance Between Induced Aggressiveness and Intrinsic Drug Resistance in T Cell Cancer.
              </a>
            </td>
          <td>
            T. B. Buus, C. K. Vadivel, Maria Gluud, Martin R J Namini, Ziao Zeng, S. Hedebo, Menghong Yin, A. Willerslev-Olsen, Emil M H Pallesen, Lang Yan, E. Blümel, E. U. Ewing, S. Ahmad, Lara P Sorrosal, Carsten Geisler, C. Bonefeld, A. Woetmann, M. Andersen, T. Mustelin, Claus Johansen, Marion Wobser, M. Kamstrup, E. Guenova, Jürgen C. Becker, S. Koralov, R. Bech, N. Ødum
          </td>
          <td>2025-06-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Metastasis remains the primary cause of mortality in gastric cancer patients; however, the underlying mechanisms driving this process remain incompletely understood. Here, we performed an integrated single‐cell analysis of gastric cancer primary tumours and their corresponding liver and lymph node metastases to identify critical intercellular communication networks driving the metastatic process. Notably, gene expression analysis of metastatic tissues showed significant upregulation of cholesterol metabolism and PPAR signalling pathway (a nuclear receptor–mediated regulatory system that orchestrates lipid metabolism, adipogenesis and energy homeostasis) genes compared to primary tumours. Our analysis revealed that myeloid cell–derived Galectin‐9 (LGALS9) and its receptor beta‐subunit of prolyl 4‐hydroxylase (P4HB) on epithelial cells constitute a previously uncharacterized ligand–receptor interaction involved in gastric cancer metastasis. Functional experiments confirmed that the activation of P4HB by LGALS9 significantly enhanced proliferation, epithelial‐mesenchymal transition (EMT) and lipid metabolism in gastric cancer cells, while pharmacological inhibition of P4HB reversed these effects. Collectively, our findings establish the myeloid‐derived LGALS9‐P4HB interaction as a crucial mediator of gastric cancer metastatic colonisation through modulation of lipid metabolism, suggesting a potential therapeutic target for metastatic gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f83d8f3206990f4a7210f3e831becc7c62823f" target='_blank'>
              Myeloid‐Derived LGALS9‐P4HB Immune Interaction Promotes Metastasis in Gastric Cancer Through Enhanced Cell Proliferation and Lipid Metabolism
              </a>
            </td>
          <td>
            Xiaobin Zhu, Yating Zhang, Aiping Yu, Xiao Xiao
          </td>
          <td>2025-06-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, has emerged as a pivotal tumor suppressor altered in a broad range of human malignancies. Its frequent inactivation across diverse cancer types has revealed pleiotropic roles that intersect multiple Hallmarks of Cancer. In this review, we integrate current knowledge on how ARID1A loss influences cellular processes including proliferative signaling, resistance to cell death, genomic instability, metabolic reprogramming, immune evasion, and more. We discuss the context-specific consequences of ARID1A deficiency, its cooperation with other oncogenic events, and its implications for therapeutic vulnerability—particularly in the realm of synthetic lethality and immune modulation. By mapping ARID1A’s functional impact onto the established hallmarks framework, we highlight its centrality in cancer biology and underscore opportunities for biomarker-driven strategies and targeted interventions. Understanding ARID1A’s multifaceted roles offers a compelling lens through which to explore chromatin dysregulation in cancer and guide translational advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017483c65324200e0adc9a76376f32832abb8dbc" target='_blank'>
              ARID1A and Its Impact Across the Hallmarks of Cancer
              </a>
            </td>
          <td>
            Bridger Kearns, Andralyn McKell, Isaac Steveson, Peyton Worley, Braeden Barton, Jordan Bennett, DeLaney Anderson, Jacob Harris, James Christensen, Jared Barrott
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Hepatocellular carcinoma is a highly aggressive and heterogeneous malignancy with limited understanding of its heterogeneity. Methods In this study, we applied ten multi-omics classification algorithms to identify three distinct molecular subtypes of HCC (C1–C3). To further explore the immune microenvironment of these molecular subtypes, we leveraged single-cell transcriptomic data and employed CIBERSORTx to deconvolute their immune landscape. Results Among them, C3 exhibited the worst prognosis, whereas C1 and C2 were associated with relatively better clinical outcomes. Patients in the C3 group exhibited a high burden of copy number variations, mutation load, and methylation silencing. Our results revealed that compared to C1 and C2, C3 had a lower proportion of hepatocytes but a higher proportion of cholangiocytes and macrophages. Through analyses of hepatocyte, cholangiocyte, and macrophage subpopulations, we characterized their functional states, spatial distribution preferences, evolutionary relationships, and transcriptional regulatory networks, ultimately identifying cell subpopulations significantly associated with patient survival. Furthermore, we identified key ligand-receptor interactions, such as APOA1-TREM2 and APOA2-TREM2 in hepatocyte-macrophage crosstalk, and VTN-PLAUR in cholangiocyte-macrophage communication. Discussion Finally, we employed machine learning methods to construct a prognostic model for HCC patients and identified novel potential compounds for high risk patients. In summary, our novel multi-omics classification of HCC provides valuable insights into tumor heterogeneity and prognosis, offering potential clinical applications for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64917cbcdeb277a734016cb34d2bfe7542a8101f" target='_blank'>
              Multi-omics and single-cell approaches reveal molecular subtypes and key cell interactions in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Xueqing Zou, Yongmei Wang, Mingyuan Luan, Yizheng Zhang
          </td>
          <td>2025-05-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer remains a significant global health burden due to its high morbidity and mortality. Oncogene-targeted therapy and immunotherapy have markedly improved the 5-year survival rate in the patients with advanced or metastatic tumors compared to outcomes in the era of chemotherapy/radiation. Nevertheless, the majority of patients remain incurable. Initial therapies eliminate the bulk of tumor cells, yet residual populations termed drug-tolerant persister cells (DTPs) survive, regenerate tumor and even drive distant metastases. Notably, DTPs frequently render tumor cross-resistance, a detrimental phenomenon observed in the patients with suboptimal responses to subsequent therapies. Analogous to species evolution, DTPs emerge as adaptative products at the cellular level, instigated by integrated intracellular stress responses to therapeutic pressures. These cells exhibit profound heterogeneity and adaptability shaped by the intricate feedforward loops among tumor cells, surrounding microenvironments and host ecology, which vary across tumor types and therapeutic regimens. In this review, we revisit the concept of DTPs, with a focus on their generation process upon targeted therapy or immunotherapy. We dissect the critical phenotypes and molecule mechanisms underlying DTPs to therapy from multiple aspects, including intracellular events, intercellular crosstalk and the distant ecologic pre-metastatic niches. We further spotlight therapeutic strategies to target DTP vulnerabilities, including synthetic lethality approaches, adaptive dosing regimens informed by mathematical modeling, and immune-mediated eradication. Additionally, we highlight synergistic interventions such as lifestyle modifications (e.g., exercise, stress reduction) to suppress pro-tumorigenic inflammation. By integrating mechanistic insights with translational perspectives, this work bridges the gap between DTP biology and clinical strategies, aiming for optimal efficacy and preventing relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a66d9e3af1fc508d0ed83941eccffc587ef2ea" target='_blank'>
              Drug-induced tolerant persisters in tumor: mechanism, vulnerability and perspective implication for clinical treatment
              </a>
            </td>
          <td>
            Shujie Liu, Anfeng Jiang, Faqing Tang, Minghao Duan, Bin Li
          </td>
          <td>2025-05-24</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Background Melanoma is a highly heterogeneous malignancy with diverse molecular and clinical behaviors. A precise molecular classification is critical for improving prognostic assessment and guiding personalized therapy. Methods We performed an integrative multi-omics analysis of skin cutaneous melanoma using data from The Cancer Genome Atlas (TCGA) and validated our findings in independent cohorts. Multi-layered data, including transcriptomic, genomic, epigenetic, and immune landscape profiles, were analyzed using unsupervised clustering and machine learning approaches to define molecular subtypes. Functional assays and in silico drug screening were employed to explore subtype-specific vulnerabilities. Results Three robust molecular subtypes (CS1, CS2, CS3) were identified, each with distinct genomic alterations, tumor microenvironment characteristics, and clinical outcomes. The CS2 subtype was immunologically “hot,” characterized by high tumor mutational burden (TMB), elevated neoantigen load, strong immune infiltration, and activated IFN-γ signaling. CS2 tumors showed significant enrichment of immune checkpoint gene expression and were associated with favorable response to anti-PD-1 therapy in external validation cohorts. In contrast, CS1 and CS3 were immunologically “cold” with immune exclusion, high chromosomal instability, and activation of oncogenic pathways linked to immune evasion. Transcriptomic drug sensitivity modeling suggested that CS1 and CS3 may benefit from HSP90 or MEK inhibitors. Moreover, COL11A2 was identified as a subtype-enriched oncogenic driver predominantly expressed in CS1/CS3, and its silencing impaired tumor cell proliferation, invasion, and epithelial–mesenchymal transition (EMT) features. Conclusions This study presents a refined multi-omics classification of melanoma that reveals biologically and clinically distinct subtypes with divergent immune and therapeutic profiles. It offers a framework for subtype-specific treatment strategies, and identifies COL11A2 as a potential target in immune-cold melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47ce450bfa8af83563781e72095e944188c7b621" target='_blank'>
              Multi-omics-based subtyping of melanoma suggests distinct immune and targeted therapy strategies
              </a>
            </td>
          <td>
            Changchang Li, Xiaoqiong Lin, Jinhui Wang, Qiaochu Zhou, Fangfang Feng, Jie Xu
          </td>
          <td>2025-06-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e17615


 Background:
 Uterine carcinosarcoma (UCS) is a rare gynecologic malignancy that has poor clinical outcomes and few targeted therapies. This aggressive variant of endometrial carcinoma has a uniquely biphasic appearance and morphology of metastases from these tumors can be carcinoma, sarcoma, or carcinosarcoma. The tumor immune microenvironment (TIME) plays a crucial role in the behavior of cancers. Recent advances in single cell spatial transcriptomics allow for greater insight into the TIME of these aggressive tumors.
 Methods:
 Ten samples from four patients with multiple matched longitudinal samples underwent single-cell spatial transcriptomic (scST) profiling (Nanostring CosMx 6k). Standard segmentation and QC filters were applied. Cell typing was performed with confirmation by expert pathologist annotation of tissue regions. Normalized gene expression was compared within the clusters based on clinicopathologic characteristics.
 Results:
 scST profiling of 80,843 high-quality cells revealed 38 distinct clusters (11 carcinoma, 12 sarcoma, 9 immune, 7 benign mesenchymal). Tumor clusters were largely patient-specific with tumors demonstrating diverse transcriptional heterogeneity while metastatic tumors were composed of several less diverse clones. While immune cell clusters represented cells from all patients, there was significant patient and sample heterogeneity in the proportions of immune cell subsets observed. Macrophages were the most abundant immune cell type in all samples, plasma cells were common in one patient's samples, and rare T cells were found only in specific cellular niches. Transcriptomic features of intra-tumoral macrophages were assessed based on site (primary versus distant metastasis) and temporal (naïve versus treated with chemotherapy) groupings. Compared to tumor associated macrophages (TAM) in primary tumors, TAMs in distant metastases were more likely to have M2-like features with higher expression of
 CXCL2, CD163, CD209,
 and
 CD44
 (P<0.005). TAMs residing in chemotherapy-treated tumors had higher expression of
 IGF2, CXCL2, CXCL1, CD163, CD209, CXCL16, CD24
 ,
 CD44, CD93
 (P<0.005). Upregulation of pro-angiogenic pathways (i.e.
 VEGFA
 ) was common in both distant metastases and post-chemotherapy UCS TAMs.
 Conclusions:
 Uterine carcinosarcoma is a highly heterogeneous tumor both with respect to tumor cell and immune microenvironment features. This work highlights transcriptomic characteristics in macrophages in the UCS microenvironment. Notably, macrophages in distant metastases and in patients with prior chemotherapy were found to have more M2 TAM-like macrophages than primary untreated tumors. These results provide rationale for both TAM-based immunotherapy and anti-angiogenic approaches in UCS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ce5dd582063cc9e8b6f92d0391909010dfa4c1" target='_blank'>
              Single cell characterization of uterine carcinosarcoma and analysis of transcriptomic shifts in tumor-associated macrophages in metastatic and chemotherapy-treated tumors.
              </a>
            </td>
          <td>
            C. Nief, Sabrina Zdravkovic, Sahar Nasr, Phoebe Hammer, Brooke Howitt
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beed6f709c8c4efc3e5e432a352d4565e72fee8a" target='_blank'>
              The Curated Cancer Cell Atlas provides a comprehensive characterization of tumors at single-cell resolution.
              </a>
            </td>
          <td>
            Michael Tyler, Avishai Gavish, Chaya Barbolin, R. Tschernichovsky, Rouven Hoefflin, M. Mints, Sidharth V. Puram, I. Tirosh
          </td>
          <td>2025-05-08</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="
 e15185


 Background:
 Glioblastoma (GBM), a highly aggressive brain tumor, is challenging to treat due to its resistance to conventional therapies and the restrictive nature of the blood-brain barrier (BBB). Loss or dysfunction of the tumor suppressor PTEN, prevalent in approximately 40% of GBM cases, contributes to tumor progression via activation of the PI3K/AKT/mTOR pathway. Newcastle disease virus (NDV), an intrinsic oncolytic virus, demonstrates the unique ability to penetrate the BBB and selectively target tumor cells.
 Methods:
 A recombinant NDV encoding the human PTEN gene (rNDV-PTEN) was developed to enhance therapeutic efficacy. PTEN was strategically inserted between the NP and P genes of the NDV genome for optimal expression. In vitro and in vivo studies evaluated rNDV-PTEN’s capacity to restore PTEN expression, inhibit tumor growth, and cross the BBB. Human GBM cell lines (e.g., U87-MG) and an orthotopic mouse model were used for functional analyses, including proliferation, migration, apoptosis, and tumor burden assessments.
 Results:
 rNDV-PTEN successfully delivered the PTEN gene to GBM cells, resulting in significant PTEN protein expression and restoration of its tumor suppressor function. This restoration inhibited tumor growth by suppressing the PI3K/AKT/mTOR pathway. In vitro, rNDV-PTEN treatment reduced GBM cell proliferation and migration while inducing apoptosis. In vivo, intravenous administration of rNDV-PTEN demonstrated efficient BBB penetration, leading to localized PTEN delivery, significant tumor size reduction, and improved survival in GBM-bearing mice. MRI and histopathological evaluations confirmed these effects. Notably, rNDV-PTEN exhibited specificity for tumor cells, leaving normal astrocytes unaffected.
 Conclusions:
 This study highlights the potential of rNDV as a delivery vehicle capable of crossing the BBB and selectively targeting GBM. The combination of NDV’s intrinsic oncolytic properties and PTEN’s tumor-suppressive functions offers a promising therapeutic strategy for treating PTEN-deficient GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26629d7f80057c6506f472b38566ab9152ba5a9e" target='_blank'>
              Enhancing glioblastoma therapy with a recombinant Newcastle disease virus harboring the PTEN gene.
              </a>
            </td>
          <td>
            Hyun Jang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Natural killer (NK) cells in mice and humans are key effectors of the innate immune system with complex immunoregulatory functions, and diverse subsets have been identified with distinct characteristics and roles. Companion dogs with spontaneous cancer have been validated as models of human disease, including cancer immunology and immunotherapy, and greater understanding of NK cell heterogeneity in dogs can inform NK biology across species and optimize NK immunotherapy for both dogs and people. Methods Here, we assessed canine NK cell populations by single-cell RNA sequencing (scRNAseq) across blood, lung, liver, spleen, and placenta with comparison to human NK cells from blood and the same tissues to better characterize the differential gene expression of canine and human NK cells regarding ontogeny, heterogeneity, patterns of activation, inhibition, and tissue residence. Results Overall, we observed tissue-specific NK cell signatures consistent with immature NK cells in the placenta, mature and activated NK cells in the lung, and NK cells with a mixed activated and inhibited signature in the liver with significant cross-species homology. Discussion Together, our results point to heterogeneous canine NK populations highly comparable to human NK cells, and we provide a comprehensive atlas of canine NK cells across organs which will inform future cross-species NK studies and further substantiate the spontaneous canine model to optimize NK immunotherapy across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9999f23f52fd97aebbc710ad85c0ac308c757fa" target='_blank'>
              Single cell atlas of canine natural killer cells identifies distinct circulating and tissue resident gene profiles
              </a>
            </td>
          <td>
            Aryana M. Razmara, M. Lammers, Sean J Judge, William J Murphy, Cameron E. Gaskill, William T. N. Culp, Alicia A. Gingrich, Zachary Morris, R. Rebhun, C. T. Brown, David Vail, Michael S Kent, Robert J. Canter, Tetsuro Kobayashi, Riken Yokohama, Steven Dow
          </td>
          <td>2025-05-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 DNA methylation, an essential epigenetic modification, plays a critical role in establishing and maintaining cellular identities by regulating gene expression. In cancer, aberrant DNA methylation patterns contribute to tumor development and progression, yet the cell-type-specific methylation alterations within the tumor microenvironment (TME) remain poorly understood. In breast cancer, a highly heterogeneous malignancy, endothelial cells within the TME (TECs) are key players in tumor angiogenesis and progression. Methylation changes detected in bulk tumor analyses, however, reflect signals from multiple cell types present in the TME, making it challenging to pinpoint cell-type-specific epigenetic changes. Recently, methods have been developed to using DNA methylation to deconvolute cell type proportions which can in turn be used to attribute methylation alterations to specific cell types in the TME. A reference-based cell-type deconvolution algorithm, the Hierarchical Tumor Immune Microenvironment Epigenetic Deconvolution (HiTIMED) method, allows accurate quantification of cell type proportions within the TME. We accessed DNA methylation array data from 609 breast tumors and 331 nontumor normal breast tissues from the Gene Expression Omnibus (GEO) and the Genotype-Tissue Expression (GTEx) portal. To these DNA methylation profiles, we applied HiTIMED and measured tumor and non-tumor cell proportions. Then, applying a statistical interaction model called CellDMC we leveraged the cell type proportion data to identify differential methylation specific to cell types in the breast cancer TME, including TECs and tumor cells. We identified 1897 tumor-cell-specific and 180 TEC-specific differentially methylated cytosines (DMCs) in cancer compared to normal breast tissue (FDR ≤ 1e-10). Importantly, in the cell-type unadjusted differential analysis, tumors exhibited hypermethylation compared to normal tissues; however, in the tumor-cell-specific analysis, tumor cells were hypomethylated relative to normal samples. By applying these advanced techniques to breast cancer samples, our study unveils cell-type-specific DNA methylation changes that are pivotal in driving the pro-tumorigenic environment. These findings not only enhance our understanding of the molecular mechanisms underlying breast cancer progression but also highlight potential therapeutic targets within the TME. The integration of HiTIMED and CellDMC represents a significant methodological advancement in cancer epigenetics, offering a more precise and insightful approach to studying the complex interplay between different cell types within the TME.
 Citation Format: Barbara Barbara, Lucas A. Salas, Brock C. Christensen. Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-29.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77ae964d40b279e042709ea22a740c1da93f0fc" target='_blank'>
              Abstract P5-06-29: Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution
              </a>
            </td>
          <td>
            Barbara Barbara, Lucas A Salas, Brock C. Christensen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e14635


 Background:
 Recent advances in computational pathology have revolutionized the analysis of histopathology images, enabling precise and rapid clinical outcome predictions. However, a significant challenge persists in translating computational insights into biologically meaningful data that informs clinical decisions. Human-interpretable image features (HIFs) have emerged as a solution, offering a detailed view of the tumor microenvironment (TME). In this study, we explored how HIFs derived from high-resolution histopathology images can predict the expression of genes associated with poor survival outcomes in stomach adenocarcinoma (STAD), aiming to bridge the gap between computational pathology and actionable clinical insights.
 Methods:
 Whole-slide histopathology images of STAD from The Cancer Genome Atlas were analyzed. Expert pathologists annotated labels were used to identify tissue types like cancer, stroma, necrosis, and normal tissue and cell types, including cancer cells, lymphocytes, macrophages, plasma cells, and fibroblasts. Using these annotations, convolutional neural networks were trained to classify these tissue and cell types, from which HIFs were extracted reflecting the biological composition of the TME, such as cell density ratios relative to surrounding tissues. We then identified genes with strong correlations to HIFs (ρ > 0.5) for further analysis.
 Results:
 Our results revealed high expression of four genes
 ABCA6, ABCA8, ADAM33,
 and
 ADAMTS10
 that were significantly associated with poor survival outcomes in STAD (p < 0.01). Each gene demonstrated a strong correlation (ρ > 0.5 and p = < 0.01) with specific HIFs.
 ABCA6
 and
 ABCA8
 , genes involved in lipid transport, are linked to a high fibroblast-to-stroma density ratio. This stromal-dominant environment is a hallmark of therapy resistance and poor outcomes.
 ADAM33
 and
 ADAMTS10
 , proteases involved in extracellular matrix remodeling, were strongly correlated (ρ > 0.5a and p = < 0.01) with a high macrophage-to-stroma density ratio. This feature represents an immunosuppressive TME, often seen in aggressive and invasive cancers.
 Conclusions:
 These findings underscore the power of HIFs to reveal key TME characteristics. A stromal-dominant TME, defined by elevated
 ABCA6
 and
 ABCA8
 , and an immunosuppressive, macrophage-rich environment associated with increased
 ADAM33
 and
 ADAMTS10
 , serve as markers of disease aggressiveness and poor prognosis. By integrating these insights into AI-driven models, clinicians may identify high-risk patients earlier in their treatment journey, enabling personalized treatment strategies to improve outcomes for STAD patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebad59e7c9b25a675a77d956eb6f9b7c2d45acd0" target='_blank'>
              Linking gene expression to tumor microenvironment using H&E features in stomach adenocarcinoma.
              </a>
            </td>
          <td>
            S. H. Bukhari, F. Vempalli, J. D. Warren, Shyam Aggarwal, Aditya Sarin, Mandeep Singh Malhotra, Rakesh K. Yadav, M. Macha, A. Bhat, Harry Lander, Tariq Masoodi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background Diverse cell types and cellular states in the tumor microenvironment (TME) are drivers of biological and therapeutic heterogeneity in ovarian cancer (OV). Characterization of the diverse malignant and immunology cellular states that make up the TME and their associations with clinical outcomes are critical for cancer therapy. However, we are still lack of knowledge about the cellular states and their clinical relevance in OV. Methods We manually collected the comprehensive transcriptomes of OV samples and characterized the cellular states and ecotypes based on a machine-learning framework. The robustness of the cellular states was validated in independent cohorts and single-cell transcriptomes. The functions and regulators of cellular states were investigated. Meanwhile, we thoroughly examined the associations between cellular states and various clinical factors, including clinical prognosis and drug responses. Results We depicted and characterized an immunophenotypic landscape of 3,099 OV samples and 80,044 cells based on a machine learning framework. We identified and validated 32 distinct transcriptionally defined cellular states from 12 cell types and three cellular communities or ecotypes, extending the current immunological subtypes in OV. Functional enrichment and upstream transcriptional regulator analyses revealed cancer hallmark-related pathways and potential immunological biomarkers. We further investigated the spatial patterns of identified cellular states by integrating the spatially resolved transcriptomes. Moreover, prognostic landscape and drug sensitivity analysis exhibited clinically relevant immunological subtypes and therapeutic vulnerabilities. Conclusion Our comprehensive analysis of TME helps leveraging various immunological subtypes to highlight new directions and targets for the treatment of cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06521-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d1ebac5d2f15e31777d9394d795738342be5a0" target='_blank'>
              Multi-dimensional characterization of cellular states reveals clinically relevant immunological subtypes and therapeutic vulnerabilities in ovarian cancer
              </a>
            </td>
          <td>
            Can Zhang, Si Li, Jiyu Guo, Tao Pan, Ya Zhang, Yueying Gao, Jiwei Pan, Meng Liu, Qingyi Yang, Jinyang Yu, Juan Xu, Yongsheng Li, Xia Li
          </td>
          <td>2025-05-08</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c490b5674b718496318b2118c9f55620a929ba49" target='_blank'>
              Dictionary of human intestinal organoid responses to secreted niche factors at single cell resolution
              </a>
            </td>
          <td>
            Meghan M. Capeling, Bob Chen, Kazeera Aliar, Elisa Penna, Veronica Ibarra Lopez, Conrad Foo, Sandra Rost, Loryn Holokai, Xinming Tong, Devan Phillips, Caden W. Sweet, Jing Li, Sharmila Chatterjee, Elizabeth Skippington, Z. Modrusan, Lisa M. McGinnis, Runmin Wei, M. Keir, O. Rozenblatt-Rosen, Michelle B. Chen
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Spatial transcriptomics is a technology that captures gene expression levels at different spatial locations, widely used in tumor microenvironment analysis and molecular profiling of histopathology, providing valuable insights into resolving gene expression and clinical diagnosis of cancer. Due to the high cost of data acquisition, large-scale spatial transcriptomics data remain challenging to obtain. In this study, we develop a contrastive learning-based deep learning method to predict spatially resolved gene expression from whole-slide images. Evaluation across six different disease datasets demonstrates that, compared to existing studies, our method improves Pearson Correlation Coefficient (PCC) in the prediction of highly expressed genes, highly variable genes, and marker genes by 6.27%, 6.11%, and 11.26% respectively. Further analysis indicates that our method preserves gene-gene correlations and applies to datasets with limited samples. Additionally, our method exhibits potential in cancer tissue localization based on biomarker expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d3b24ff57bbd859a313eed33222a927cb01b31b" target='_blank'>
              Spatial Transcriptomics Expression Prediction from Histopathology Based on Cross-Modal Mask Reconstruction and Contrastive Learning
              </a>
            </td>
          <td>
            Junzhuo Liu, Markus Eckstein, Zhixiang Wang, Friedrich Feuerhake, D. Merhof
          </td>
          <td>2025-06-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 2042


 Background:
 Glioblastoma (GBM) is the most lethal primary brain tumor in adults, with a median survival of ~15 months despite current therapies (surgery, radiation, temozolomide). Advances like immune checkpoint inhibitors, anti-angiogenic agents, and tumor vaccines have shown suboptimal results. The 2021 WHO classification highlights molecular markers (e.g., IDH, MGMT) for better stratification, but these fail to fully capture tumor microenvironmental dynamics. Using an AI-driven transcriptomic approach, we identified novel prognostic subtypes in IDH-wildtype GBM, aiming to refine stratification, enhance understanding of tumor biology, and guide personalized therapeutic strategies.
 Methods:
 We accessed microarray data from The Cancer Genome Atlas (TCGA) (n=353 newly diagnosed, IDH-WT GBM) for a training set and RNA-seq data from the Chinese Glioma Genome Atlas (CGGA) (n=170 primary and n=106 recurrent tumors) for validation. A proprietary SphereBio machine learning–based algorithm was used to derive transcriptomic signatures with prognostic relevance. Subtypes were assessed via Kaplan–Meier analyses in the training cohort and tested in both primary and recurrent validation cohorts. Immune/stromal infiltration was quantified using a tumor deconvolution tool (DA_505), and pathway enrichment (GAGE) was performed on the validation sets.
 Results:
 AI-driven clustering revealed three transcriptomic subtypes with significant survival differences in both the training (p<0.0001) and primary validation (p=0.0004) cohorts. In the recurrent cohort, a similar survival trend by subtype was observed, though significance was diminished (p=0.12), likely due to limited sample size and therapy-related changes. Immune/stromal deconvolution showed distinct infiltration patterns: subtypes enriched for CD4+ and CD8+ T cells correlated with prolonged survival. Pathway enrichment analysis in both primary and recurrent tumors highlighted potential targets involving embryogenesis, immune modulation, cell cycle, and stress response. The persistence of a consistent survival trend and comparable microenvironment and pathway patterns suggest that these transcriptomic subtypes remain biologically relevant even after standard treatment.
 Conclusions:
 Our integrated transcriptomic and microenvironment-focused approach identified three prognostically distinct GBM subtypes, validated across independent cohorts. These findings underscore the utility of AI-driven transcriptomic signatures for personalized stratification, with the potential to guide targeted therapeutic strategies and inform clinical trial design in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff54807210ca2a362e060a895b6f6e635241ee" target='_blank'>
              AI-driven transcriptomic classification of glioblastoma: Associations with survival and tumor microenvironment.
              </a>
            </td>
          <td>
            J. Fernandez-Muñoz, Román Oberti, D. Reardon, Francisco Quintana, Guido Nicolas Molina, M. E. Guerrero-Gimenez
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The metabolism of tumors is a complex network process involving multiple metabolic species and exhibits metabolic spatial heterogeneity, which poses challenges for the development of effective therapeutic strategies. Patient‐derived tumor organoids, recently developed, hold significant potential in elucidating novel aspects of cancer biology, expediting the discovery of oncology drugs, and personalizing cancer treatment. However, tracing metabolic reactions in the tumor organoid network presents a technological challenge. In this study, we propose methodologies that utilize enzymatically derived tissue debris (TD) to generate the biobank patient‐derived lung cancer organoids (LCOs) that accurately replicate the histological characteristics, stemness of lung cancer stem cells, cellular heterogeneity, and mutational patterns observed in their respective original tumors. Importantly, our LCO model can be integrated with mass spectrometry imaging to analyze tumor metabolic heterogeneity and the spatial distributions of fingerprint lipids associated with extracellular matrix homeostasis including lysophosphatidic acid (LPA), phosphatidic acid (PA), phosphatidylserine (PS), phosphatidylglycerol (PG), diacylglycerol (DAG). Furthermore, we demonstrate the effectiveness of LCOs in evaluating personalized treatment by establishing a connection between their mutational profiles and reactions to specific drugs. Our findings highlight that LCOs from TD retain crucial metabolic characteristics of lung cancer and can be readily utilized to investigate individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4eb756e3a5aa845f598bc861225d3c569477eee" target='_blank'>
              Integrating construction and spatial imaging of organoid for tumor metabolic heterogeneity and interactions
              </a>
            </td>
          <td>
            Ling Guo, Zhaohua Xia, Jiaxin Wang, Xiaoling Yu, Chuanyue Wu, Qian Luo, Chao Zhao
          </td>
          <td>2025-05-04</td>
          <td>VIEW</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/129265cf9a8d9036012e909d0315f3224ece47d6" target='_blank'>
              The epigenetic regulator TRIM24 controls melanoma cell dedifferentiation and resistance to treatment in melanoma
              </a>
            </td>
          <td>
            Simon Durand, Félix Boivin, Roxane M. Pommier, L. Barbollat-Boutrand, Maxime Grimont, Félix Pham, Marion Dufeu, Eric Cumunel, Raphaël Schneider, Anthony Ferrari, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Tumor development is mainly marked by the gradual transformation of cells that acquire capacities such as sustained growth signaling, evasion of growth suppression, resistance to cell death, and induction of angiogenesis, achieving replicative immortality and activating invasion and metastasis. How different epigenetic alterations like m1A, m5C, and m6A contribute to tumor development is a field that still needs to be investigated. The immune modulators, CD70, CD80, and TIGIT, mainly regulate T-cell activation and consequently the immune evasion of tumors. Here, we explored the presence and the potential consequences of RNA modifications in these regulators in pan-cancer. Our findings highlight the critical role of the m6A, m5C, and m1A in regulating CD70, CD80, and TIGIT across multiple solid tumors. By combining epitranscriptomics data with functional enrichment and survival modeling, we show that RNA modification enzymes not only modulate immune-related gene expression but also serve as potential biomarkers for patient prognosis. By constructing a robust four-gene prognostic signature involving YTHDF3, RBM15B, IGF2BP2, and TRMT61A, we demonstrate that RNA modification profiles can accurately stratify patients into risk groups with distinct overall survival outcomes. The performance of this model across eight cancer types underscores the translational promise of epitranscriptomic markers in both mechanistic understanding and personalized oncology. Altogether, our study bridges the gap between the mechanistic regulation of immune checkpoints and their clinical utility, offering novel insights into how the epitranscriptome can be leveraged to improve cancer prognosis and potentially enhance immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d03e20bb526dd9e6903a8377f001f578b9d1d1d" target='_blank'>
              Epitranscriptomics Regulation of CD70, CD80, and TIGIT in Cancer Immunity
              </a>
            </td>
          <td>
            C. Rigopoulos, Marios Gkoris, Ilias Georgakopoulos-Soares, I. Boulalas, A. Zaravinos
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fb1152ca394367d1b31e6130a4964a702584963" target='_blank'>
              Multiscale Modeling Uncovers Macrophage Infiltration and TNF-α Signaling Networks for Targeting in Inflammatory Breast Cancer Tumor Emboli
              </a>
            </td>
          <td>
            Pritha Pai, Christophe Van Berckelaer, S. V. van Laere, Alexandra Bennion, Theresa Charity, Jinming Yang, F. Bertucci, Peter van Dam, Gregory M Palmer, Shannon McCall, Luc Y. Dirix, Naoto T. Ueno, G. Devi
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/011f8b8526e8e1a0af0fe6dc9a2da7b6532fccfb" target='_blank'>
              scCoBench: Benchmarking single cell RNA-seq co-expression using promoter-reporter lines
              </a>
            </td>
          <td>
            Tran N. Chau, K. Ryu, Razan Alajoleen, Bastiaan O. R. Bargmann, John Schiefelbein, Song Li
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in shaping immunometabolism in prostate cancer, influencing disease progression and therapeutic response. This review examines the dynamic interactions between tumor cells and immune cells within the prostate cancer TME, focusing on how metabolic reprogramming of both tumor and immune cells drives immunosuppression. Key immune players, including T-cells, macrophages, and myeloid-derived suppressor cells, undergo metabolic adaptations influenced by hypoxia, nutrient deprivation, and signaling from tumor cells. Additionally, we discuss the metabolic pathways involved, such as glycolysis and oxidative phosphorylation, and how these processes are exploited by cancer cells to evade immune surveillance. Furthermore, this review highlights potential therapeutic strategies targeting immunometabolism, including metabolic inhibitors and their combination with immunotherapies. A deeper understanding of the complex role of immunometabolism in prostate cancer will not only provide insights into the tumor’s immune evasion mechanisms but also facilitate the development of novel treatment approaches that enhance the efficacy of current therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd6031bfb5363d860488050852357492fe22b41c" target='_blank'>
              Role of Tumor Microenvironment in Prostate Cancer Immunometabolism
              </a>
            </td>
          <td>
            Yutao Wang, Yiming Chen, Jianfeng Wang
          </td>
          <td>2025-06-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer progression and therapeutic resistance are propelled by the remarkable plasticity of signaling networks, which dynamically rewire under selective pressures to maintain proliferation, enable immune evasion and promote metastasis. Despite advances in precision oncology, the dynamic crosstalk between tumor cells, non-coding genomes and the microenvironment continues to undermine treatment efficacy. This call for submissions, Revolutionizing Cancer Treatment: Navigating the Intricate Landscape of Cellular Signaling Networks, seeks cutting-edge research that dissects these adaptive mechanisms through innovative technologies - from single-cell multi-omics and spatial transcriptomics to AI-powered network modeling. We welcome studies leveraging physiomimetic models (e.g., organoids, 3D-bioprinted ecosystems) to decode tumor heterogeneity, as well as translational work targeting emergent vulnerabilities at the intersection of epigenetics, metabolic reprogramming and stromal interactions. By integrating systems biology with computational and experimental approaches, this collection aims to catalyze the design of adaptive therapies that outmaneuver cancer's evolutionary resilience.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba13faeaf95516c4bb5cbd68aeea6df92e61f127" target='_blank'>
              Revolutionizing cancer treatment: Navigating the intricate landscape of cellular signaling networks.
              </a>
            </td>
          <td>
            Hao Zhang
          </td>
          <td>2025-05-26</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) presents significant clinical challenges owing to its dense stroma and complex tumor microenvironment (TME). In this study, large-scale single-cell transcriptomics and spatial transcriptomics (ST) were integrated to dissect the heterogeneity of fibroblasts and their crosstalk with epithelial cells, with a focus on key ligand-receptor interactions. Eight distinct fibroblast subpopulations were identified, among which extracellular matrix (ECM)-remodeling fibroblasts were particularly enriched in tumor tissues and associated with poor prognosis. ECM-remodeling fibroblasts were located at the terminal stage of the fibroblast pseudotime trajectory, and SOX11 was identified as a key transcription factor in this subpopulation. Further analyses revealed that ECM-remodeling fibroblasts can interact with epithelial cells through the POSTN-ITGAV/ITGB5 ligand-receptor axis, a critical pathway that promotes tumor progression. Clinical analyses demonstrated a strong correlation between POSTN expression and poor prognosis in patients with PDAC. Mechanistically, POSTN interacts with integrin ITGAV/ITGB5 on tumor cells, activating the PI3K/AKT/β-catenin pathway and promoting epithelial-mesenchymal transition (EMT) phenotype. Pharmacological inhibition of the POSTN-integrin axis partially reversed these malignant traits, highlighting its potential as a therapeutic target. This study provides new insights into fibroblast heterogeneity and its role in PDAC progression, emphasizing the POSTN-ITGAV/ITGB5 axis as a promising target for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e1d45940647abe0d0d6b914abc94702cee2e0ff" target='_blank'>
              Integrative Single-Cell and Spatial Transcriptomics Analysis Reveals ECM-remodeling Cancer-associated Fibroblast-Derived POSTN as a Key Mediator in Pancreatic Ductal Adenocarcinoma Progression
              </a>
            </td>
          <td>
            Yifan Wu, Shuquan Li, Hao Yu, Shan Zhang, Lianghua Yan, Xiao-ya Guan, Wei Xu, Zhen Wang, Ang Lv, Xiuyun Tian, Chunyi Hao, Jianhui Wu
          </td>
          <td>2025-05-27</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Human skin is not a uniform organ but a mosaic of anatomically distinct niches, with each site finely tuned to unique environmental demands and immune pressures. Yet, the molecular determinants that define these regional identities and their relationship to site-specific vulnerability to inflammatory disease remain poorly understood. Here, we generate a high-resolution single-cell atlas of human skin, profiling 274,834 cells from 96 healthy samples across 7 anatomically distinct sites (acral, arm, axilla, back, face, leg and scalp). Our analysis reveals striking region-specific transcriptional and cellular networks, uncovering how local immune-stromal crosstalk governs tissue homeostasis and underpins anatomical susceptibility to distinct inflammatory diseases such as such as systemic lupus erythematosus (SLE), atopic dermatitis (AD), and psoriasis. These findings illuminate the tissue-intrinsic foundations of regional immune identity and provide a blueprint/resource for the development of precision therapies tailored to the distinct immunological microenvironments of specific anatomical skin sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0efad5522b30037ec342a6195a031267736175c" target='_blank'>
              A Transcriptomic Atlas of Healthy Human Skin Links Regional Identity to Inflammatory Disease
              </a>
            </td>
          <td>
            Sahiti Marella, Rachael Bogle, Jennifer Fox, L. Tsoi, Xianying Xing, Yiqian Gu, J. Kirma, M. Sarkar, Vincent van Drongelen, Benjamin Klein, Jeff H. Kozlow, Paul W. Harms, Katherine A. Gallagher, S.Chakravarthy Naik, Vito W. Rebecca, Bogi Andersen, Mio Nakamura, J. Kahlenberg, R. Modlin, A. Billi, Johann E. Gudjonsson
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bb1b241042af13953303a559606bd29dd3c0c78" target='_blank'>
              Differentiation status determines tumorigenicity and immunogenicity of cancer cells
              </a>
            </td>
          <td>
            Yang Liu, Chen Wang, Jing Li, Ying Cao
          </td>
          <td>2025-06-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Embryonal tumor with multilayered rosettes (ETMR) is a pediatric brain tumor with dismal prognosis. Characteristic alterations of the chromosome 19 microRNA cluster (C19MC) are observed in most ETMR; however, the ramifications of C19MC activation and the complex cellular architecture of ETMR remain understudied. Here we analyze 11 ETMR samples from patients using single-cell transcriptomics and multiplexed spatial imaging. We reveal a spatially distinct cellular hierarchy that spans highly proliferative neural stem-like cells and more differentiated neuron-like cells. C19MC is predominantly expressed in stem-like cells and controls a transcriptional network governing stemness and lineage commitment, as resolved by genome-wide analysis of microRNA-mRNA binding. Systematic analysis of receptor–ligand interactions between malignant cell types reveals fibroblast growth factor receptor and Notch signaling as oncogenic pathways that can be successfully targeted in preclinical models and in one patient with ETMR. Our study provides fundamental insights into ETMR pathobiology and a powerful rationale for more effective targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02046d3dbdf78b8dbf1ae4df369d0858d2ab96f2" target='_blank'>
              Cellular hierarchies of embryonal tumors with multilayered rosettes are shaped by oncogenic microRNAs and receptor–ligand interactions
              </a>
            </td>
          <td>
            Alexander Beck, Lisa Gabler-Pamer, G. Alencastro Veiga Cruzeiro, S. Lambo, Bernhard Englinger, McKenzie L. Shaw, Olivia A Hack, Ilon Liu, Rebecca D. Haase, Carlos A O de Biagi, A. Baumgartner, Andrezza Do Nascimento Silva, M. Klenner, Pia Freidel, Jochen Herms, Louisa von Baumgarten, Joerg-Christian Tonn, N. Thon, K. Bruckner, S. Madlener, L. Mayr, D. Senfter, A. Peyrl, I. Slavc, D. Lötsch, C. Dorfer, Rene Geyregger, N. Amberg, C. Haberler, Norman L Mack, Benjamin Schwalm, Stefan M. Pfister, Andrey Korshunov, Lissa C Baird, Edward Yang, Susan N Chi, S. Alexandrescu, J. Gojo, Marcel Kool, Volker Hovestadt, Mariella G. Filbin
          </td>
          <td>2025-05-26</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 4155


 Background:
 The tumor microenvironment (TME) that surrounds pancreatic ductal adenocarcinoma (PDAC) is a multi-faceted and dynamic ecosystem in which stromal fibroblasts communicate with cancer cells to mediate tumor growth, metastasis, and chemotherapy resistance. It is well recognized that both cancer-associated fibroblasts (CAFs) and its non-cellular, fibrotic components within the TME can foster a protumorigenic environment for PDAC. However, we still lack a comprehensive understanding of the precise mechanisms in which this dense, fibrotic matrix can help drive malignant behaviors. Here, we reveal a novel mechanism in which the zinc-finger transcription factor GLI2 regulates type 1 collagen expression within CAFs and how the soluble variant of this collagen promotes irinotecan chemoresistance.
 Methods:
 We leveraged transcriptomic data from The Cancer Genome Atlas, International Cancer Genome Consortium, and Clinical Proteomic Tumor Analysis Consortium to evaluate GLI2 expression and stromal content of human PDAC tumors through bulk RNA-sequencing deconvolution. Using single-nucleus RNA sequencing of human PDAC tumors, we validated the association of GLI2 expression and stromal matrix constituents in CAFs. Chromatin immunoprecipitation assays in human CAFs confirmed GLI2 binding at the COL1A1 promoter. Through RNAi-based inactivation of GLI2, we determined how loss of GLI2 impacts regulation of type 1 collagen. Additionally, we conducted MTT assays to assess tumor viability in response to irinotecan treatment.
 Results:
 Transcriptomic analysis revealed that GLI2 is highly enriched in CAFs and strongly correlated with stromal fibrosis compared to other non-tumor cell constituents within the TME. We have shown that GLI2 directly binds to the promoter of COL1A1, a key component of type 1 collagen, in CAFs and regulates its transcription in a manner dependent on TGFβ1 signaling. Interestingly, PDAC tumors exposed to type 1 collagen show increased expression of pro-tumorigenic pathways involved in inflammation, EGR signaling, cytokine-receptor interactions, and irinotecan resistance. We further validate that human PDAC cells pre-treated with collagen can confer chemoresistance to irinotecan with viability assays.
 Conclusions:
 Taken together, our study demonstrates a novel mechanism in which GLI2 regulates the secretion of collagen within CAFs, which in turn can enable PDAC to acquire resistance to standard-of-care treatments. These findings highlight not only the oncogenic functions for CAFs and their fibrotic secretome, but also open new avenues in which therapeutic targeting of the TME may provide clinical benefit for PDAC patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3a2065caeb9cc2540ae0418d70044e4a839709" target='_blank'>
              Functional role of GLI2 in cancer-associated fibroblasts for modulation of the fibrotic tumor microenvironment within pancreatic cancer.
              </a>
            </td>
          <td>
            John Y. Kwon, Renzo E Vera, Maite G. Fernadez-Barrena, Jose M. Falero, Roberto G. Chaparro, Matthew D. Ross, M. D. Toruner, Murat Toruner, Ezequiel J Tolosa, Luciana L Almada, Huocong Huang, Rolf A. Brekken, Martin E Fernandez-Zapico
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744871a7bb11281b1630d2cb920a865723043fec" target='_blank'>
              Virus-Positive Merkel Cell Carcinoma Model Supports Unified Cancer Origin
              </a>
            </td>
          <td>
            Wendy Yang, Sara Contente, Sarah Rahman
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7be43f99c39d72183ee247e245dd3934afb3150" target='_blank'>
              Decoding meningioma prognosis with multi-omics: macrophage diversity, immune-CNV interplay, and novel SPP1-targeted strategies.
              </a>
            </td>
          <td>
            Hailang Fan, Xiaojie Li, Yaqian Zhao, Lairong Song, Lanlan Sun, Zhen Wu, Junting Zhang, Haikun Zhang, Feifan Zhang, Yingdi Huang, Dake Zhang, Liang Wang
          </td>
          <td>2025-06-16</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aggressive brain tumors often exhibit immunologically ‘cold’ microenvironment, where the vascular barrier impedes effective immunotherapy in poorly understood ways. Tumor vasculature also plays a pivotal role in immunoregulation and antitumor immunity. Here, we show that small GTPase Rab27 controls the vascular morphogenesis and permeability for blood content and immune effectors. Thus, in Rab27a/b double knock out (Rab27-dKO) mice, the brain vasculature is abnormally scarce, while the blood vessels become dysmorphic and hyperpermeable in the context of brain tumors, including syngeneic glioblastoma. These defects are reflected in rearrangements of endothelial cell subpopulations with underlying diminution of venous endothelial subtype along with changes in gene and protein expression. Notably, Rab27-dKO brain endothelial cells exhibit deficient tight junctions, whereby they enable large-scale extravasation of cytotoxic T cells into the tumor mass. We show that Rab27-regulated vascular T cell infiltration can be exploited to enhance adoptive T cell therapy in syngeneic brain tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/512fe359893feca658456aa6611efa099498e729" target='_blank'>
              Immune cell infiltration into brain tumor microenvironment is mediated by Rab27-regulated vascular wall integrity
              </a>
            </td>
          <td>
            L. Adnani, Brian Meehan, Minjun Kim, Dongsic Choi, Christopher E Rudd, Yasser Riazalhosseini, J. Rak
          </td>
          <td>2025-05-23</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="In vitro hematopoiesis systems can be used to define mechanisms for blood cell formation and function, produce cell therapeutics, and model blood cell contributions to systemic disease. Hematopoietic progenitor cell (HPC) production remains inefficient, precluded by knowledge gaps related to specification and morphogenesis of specialized hemogenic endothelial cells, which undergo an endothelial-to-hematopoietic transition (EHT) to form HPCs. We elected to define changes in gene expression and chromatin organization during HPC formation to reveal regulatory mechanisms. Using paired single cell RNA/ATAC sequencing together with Hi-C, we profiled cells before and after EHT. Pathway analysis and pseudotime inferences confirmed a continuum of stromal and endothelial cells undergoing development into HE cells and lineage-based HPCs in vitro. In these cell types, we characterize cis-regulatory elements and transcriptional regulatory activities that facilitate EHT and HPC homeostasis, including for SNAI1, SOX17, TGFβ, STAT4, as well as for GFI1b and KLF1 in megakaryocyte- and erythroid-biased progenitors, respectively. We then leveraged our insights into chromatin organization among in vitro-derived cells to assess enrichments corresponding to human trait variation reported in human genome wide association studies. HPCs revealed locus enrichment for quantitative blood traits and autoimmune disease predisposition, which were particularly enriched in myeloid- and lymphoid-biased populations. Stromal and endothelial cells from our in vitro cultures were specifically enriched for accessible chromatin at blood pressure loci. Our findings reveal genes and mechanisms governing in vitro hematopoietic development and blood cell-related disease pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4774749a5a1117ef1a36110f9d8a3ed3849f1988" target='_blank'>
              Multimodal analysis of in vitro hematopoiesis reveals blood cell-specific genetic impacts on complex disease traits
              </a>
            </td>
          <td>
            Rong Qiu, Khanh B. Trang, Carson Shalaby, J. Pippin, James Garifallou, Struan F. A. Grant, Christopher S. Thom
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c087be1ee8d5c3fb0e227ebb941d840bf16ba32a" target='_blank'>
              Residual Breast Cancer Cells Co-opt SOX5-driven Endochondral Ossification to Maintain Dormancy
              </a>
            </td>
          <td>
            Amulya Sreekumar, Eric Blankemeyer, Christopher J. Sterner, Tien-Chi Pan, D. Pant, Sarah Acolatse, Hamza Turkistani, G. Belka, Sean D. Carlin, C. Assenmacher, M. Sellmyer, David A. Mankoff, Lewis A. Chodosh
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="BACKGROUND
Metabolic reprogramming in glioblastoma (GBM) is a putative determinant of GBM subtype, malignant cell state and tumor-immune crosstalk. In the present study, we investigated how polyamine metabolic rewiring contributes to the malignant cell-intrinsic and microenvironment-dependent biological processes underpinning GBM subtype classification.


METHODS
Liquid chromatography/tandem mass spectrometry (LC-MS/MS) was used for polyamine quantification in human and murine GBM tumors and cell lines. Through single-cell RNA sequencing, metabolic profiling and additional functional experiments, we dissect the malignant cell-intrinsic and paracrine signaling processes regulated by SAT1 (spermidine/spermine-N1-acetyltransferase1) and its product, N1-acetylspermidine.


RESULTS
We find that polyamine acetylation is elevated in human and murine GBM tumors and contributes to the classification of mesenchymal/plurimetabolic GBM through both regulation of tumor-cell intrinsic glucose metabolism and by facilitating metabolic crosstalk with tumor-associated macrophages/myeloid cells (TAMs). The impact of SAT1 on tumor cell metabolism is mediated, at least in part, by N1-acetylspermdine, the sole polyamine elevated in human and murine tumors. Furthermore, the relatively high levels of N1-acetylspermidine released by GBM is taken up by myeloid cells to promote intracellular polyamine flux, cellular respiration and migration. In vivo, both genetic disruption of polyamine acetylation and pharmacological inhibition of polyamine transport reduced myeloid cell infiltration and sensitized tumors to chemoradiation.


CONCLUSIONS
Collectively, the findings highlight a previously unidentified role for SAT1 and its product, N1-acetylspermidine, in bridging the metabolic activity of tumor cells and tumor-associated macrophages/myeloid cells (TAMs), together promoting mesenchymal/plurimetabolic states and therapeutic resistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d122ce83bdf79e8ba49d2105d5e47f84085f92ab" target='_blank'>
              Polyamine acetylation mediates crosstalk between cancer cells and myeloid cells to promote mesenchymal/plurimetabolic states in glioblastoma.
              </a>
            </td>
          <td>
            Ayush B Rana, Timothy M. Horton, Vijay S Thakur, Dazhi Wang, Varsha Thakur, Molly Dalzell, Juliano T Freitas, D. P. Gannamedi, Ifeanyichukwu C Ogobuiro, Barbara Bedogni, S. Gultekin, Timothy J. Garrett, Alejandro V. Villarino, Jun Lu, David B Lombard, Ashish H Shah, S. Welford
          </td>
          <td>2025-05-27</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is highly aggressive and lacks targeted therapies, posing a major challenge in oncology. Traditional two-dimensional (2D) cell cultures fail to capture the tumor microenvironment’s complexity, whereas three-dimensional (3D) cultures provide a more accurate model of tumor biology. We developed an advanced 3D culture system for TNBC cell lines BT-20 and MDA-MB-231, enhancing the hanging-drop method with Matrigel to restore essential extracellular matrix interactions. Confocal imaging showed MDA-MB-231 cells forming clusters typical of aggressive carcinoma, while BT-20 cells organized into duct-like structures. Molecular analysis of PI3K and β-catenin target genes revealed distinct expression patterns, with PI3K overexpressed and β-catenin downregulated in 3D cultures. Moreover, β-catenin distribution in the 3D cell culture closely resembles its pattern in tissue. These findings underscore the value of 3D models in understanding TNBC progression and in supporting the exploration of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847fb65a19c3a57935a901992b8fffe775547b6b" target='_blank'>
              Development of a reliable method for human triple-negative breast cancer organotypic culture: Improving imaging and genomic studies in 3D cultures
              </a>
            </td>
          <td>
            Mercedes Olvera-Valencia, Verónica García-Castillo, R. Ramos-Payán, M. Aguilar-Medina, Samuel Trujano-Camacho, Alejandro López-Saavedra, L. Marchat, C. López-Camarillo, R. Sumagin, E. Pérez-Yépez, C. Pérez-Plasencia
          </td>
          <td>2025-05-01</td>
          <td>Journal of Tissue Engineering</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Gastric cancer (GC) is among the most lethal human malignancies with limited treatment options. Cell-cell interactions within the tumor microenvironment (TME) can promote tumor growth, yet their therapeutic value has not been fully explored. Here, bulk RNA-seq, single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics (ST) were integrated to analyze the heterogeneity of GC microenvironment. Tumor-specific GREM1+ fibroblasts and SPP1+ macrophages were significantly enriched in GC tissues and were involved in immunosuppression, inflammation regulation, and tumor progression. We then indicated that GREM1+ fibroblasts and SPP1+ macrophages were positively correlated in 12 independent GC datasets and validated their close localization by multiplex immunohistochemistry staining and spatial transcriptomics. Patients with both high GREM1+ fibroblasts and SPP1+ macrophages exhibited significantly shorter OS and showed enrichment of tumor-associated pathways. Our results demonstrated the complex interactions between GREM1+ fibroblasts and SPP1+ macrophages, which may serve as a potential therapeutic target for future treatment of GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad0abacc337130966d37d3605e0104e32183302" target='_blank'>
              Multi-omics analyses reveal interactions between GREM1+ fibroblasts and SPP1+ macrophages in gastric cancer
              </a>
            </td>
          <td>
            L. Qiu, Xiao Zhao, Sheng Yao, Yang Fei, Y. Gong, Zishan Zhou, Shunchang Jiao, Jianming Xu
          </td>
          <td>2025-06-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Cancer cells achieve replicative immortality through telomere maintenance mechanisms (TMMs), primarily via telomerase activation or alternative lengthening of telomeres (ALT). Sarcomas frequently employ the ALT pathway, which traditionally correlates with adverse clinical outcomes. However, chondrosarcomas represent a unique context where the role and prognostic significance of ALT remain largely unexplored. Methods We performed comprehensive analyses of single-cell RNA-sequencing data from patients with chondrosarcoma and integrated this with 90 bulk RNA-seq datasets. This approach enabled detailed characterization of TMM at single-cell resolution, identification of ALT-specific signatures, and evaluation of the tumor microenvironment in chondrosarcomas. Results Patients with ALT-like chondrosarcomas exhibited significantly improved survival compared to those with non-ALT-like chondrosarcomas. Analysis of the tumor immune microenvironment revealed distinct metabolic and immune landscapes between the ALT-like and non-ALT-like groups. Single-cell analysis showed that high-entropy stem-like cells in high-grade chondrosarcomas predominantly adopted telomerase activation over the ALT pathway as their TMM. Additionally, we identified a 100-gene signature that reliably distinguishes ALT-like chondrosarcomas, providing a robust molecular marker for classification and prognosis. Conclusions Our study reveals ALT-like state as a marker of favorable prognosis in chondrosarcomas—contrasting with its typically adverse implications in other sarcomas. We establish a robust 100-gene signature that reliably identifies ALT-like chondrosarcomas and characterize their distinct immune microenvironment profile. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/881425c49c7bf310ad569e7f1a7bdeb0d0fff86f" target='_blank'>
              Alternative lengthening of telomeres confers favorable prognosis in chondrosarcomas
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jin-Hong Kim, Yi-Jun Kim
          </td>
          <td>2025-05-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Ewing sarcoma (ES) is a rare and aggressive pediatric bone malignancy with poor prognosis, driven by therapy-resistant tumor microenvironments (TME). The TME plays a critical role in tumor progression through a complex and dynamic network of reciprocal interactions among immune cells (dysfunctional T cells, immunosuppressive macrophages), stromal components (cancer-associated fibroblasts), and tumor cells. These interactions collectively shape the immune landscape, promote immune evasion, and contribute to therapeutic resistance. Identifying reliable prognostic markers remains a critical challenge.Here we performed an integrated single-cell RNA sequencing, WGCNA, and bulk RNA-seq analyses to investigate tumor-immune interactions. Differentially expressed genes (DEGs) intersected with T cell markers identified a total of 174 T cell-associated genes. Functional enrichment analysis and molecular subtyping were performed to explore immune-related pathways. A prognostic model based on CLEC11A, BDP1, and ID3 was constructed using Cox regression and validated in external datasets. Immune infiltration was assessed using the CIBERSORT algorithm.T cell marker analyses revealed key roles in pathways such as PI3K-Akt signaling and immune modulation. Molecular subtyping identified two clusters with distinct immune microenvironments: Cluster C1 (immunosuppressive phenotype and poorer prognosis) and Cluster C2 (functionally active immune profile associated with better prognosis). The prognostic model demonstrated high predictive accuracy for 1-, 3-, and 5-year survival (AUC: 0.85, 0.82, 0.78). Additionally, a higher tumor mutation burden (TMB) with low survival rate has been observed in High-risk group. Immune infiltration analysis showed higher CD8+ T cell and dendritic cell activity and immune checkpoint expression in low-risk groups. Experimental validation demonstrated that ID3 silencing inhibited tumor cell proliferation and induced cell cycle arrest in ES cell lines.Together, our study identified CLEC11A, BDP1, and ID3 as key T cell associated prognostic markers and developed a validated model to predict survival outcomes in ES. Insights into T cell markers and tumor-immune dynamics offer promising advances in prognostic assessment and immunotherapy for ES. Furthermore, the role of ID3 in immune evasion and tumor proliferation underscores its potential as a therapeutic target, providing new avenues for immune checkpoint regulation and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61b182bc7d5b374e78a6d8b617b48c4b427c14cb" target='_blank'>
              Integrative analysis of T cell-associated markers in Ewing sarcoma reveals prognostic signatures and immune dynamics
              </a>
            </td>
          <td>
            U. Saddozai, Chenxu Liu, Fei Yan, Zhendong Lu, M. B. Khawar, Muhammad Usman Akbar, Saadullah Khattak, Haibo Sun, Ping Yang
          </td>
          <td>2025-06-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Many studies have shown that pancreatic cancer is one of the cancers with extremely high mortality. The poor prognosis and lack of early diagnostic methods remain major challenges in the treatment of this cancer. In this article, in order to better detect the occurrence of pancreatic cancer, tools such as monocle3, singleR, harmony in R, and scanpy in Python were used to analyze the cells and genes of pancreatic cancer tissues in mice. By analyzing the data, the following results were obtained: T-cells in healthy PBMCs exhibited broader spatial dispersion than in PDAC tissues, suggesting tumor-driven immune surveillance impairment, while PDAC-associated macrophages displayed expanded distribution linked to pro-tumorigenic functions such as COL1A1-mediated ECM remodeling; Pseudotemporal trajectory analysis revealed myeloid progenitor bifurcation into monocytes/macrophages, with PDAC macrophages showing epigenetically silenced cytotoxic pathways such as suppressed GZMA/NKG7 and enhanced ribosomal biogenesis; Tissue-specific markers such as LCN2 in healthy and CTRB1/AMY2A in PDAC) and spatial co-localization of macrophages/tumor cells highlighted NOP53 as a dual-function hubinhibiting PI3K-AKT while activating p53and SPP1 as a paradoxical regulator of metastasis and antitumor immunity; Differential expression and GO enrichment analyses identified ribosomal biogenesis and cytoplasmic translation as PDAC-enriched pathways, contrasting with suppressed stress responses. Our spatial transcriptomic profiling further resolved elevated NOP53, CFB, and SPP1 expression gradients in PDAC tissues, proposing these as diagnostic biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab94aa8cc2272f4a29c06caf0dcfe1338e28a0a0" target='_blank'>
              Spatiotemporal Transcriptomic Dissection of Tumor-Associated Macrophage Heterogeneity and Dual-Function Molecular Nodes in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Mengfei Li
          </td>
          <td>2025-06-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Understanding how intricate cellular networks and signaling pathways communicate during the formation of craniofacial tissues like the palate and tooth has been the subject of intense investigation for several decades. Both organ systems undergo patterning morphogenesis and the subsequent terminal differentiation of matrix-producing cells that form biomineralized matrices like bone, enamel, dentin, and cementum. Until recently, gene expression profiles could only be assessed for a select number of cells without the context of the entire milieu of genes expressed by neighboring cells and tissues. Today, the cutting-edge field of spatial transcriptomics offers a remarkable suite of innovative technologies of multiplex gene analyses and imaging that can assess the expression of a vast library of genes that are present in situ during normal and abnormal conditions. In this review, we summarize some key technologies which have in recent years enabled an unprecedented breadth and depth of transcriptomic analyses in craniofacial development. We focus in detail on select methods that our research group has applied to better understand the cellular and molecular events that drive palate and tooth development. Our overall goal is to unravel the complexities of these unique biological systems to provide meaningful biological insights into the cellular and molecular events that drive normal development. As a work-in-progress, we strive for a deeper understanding of the temporal and spatial gene expression profiles within cells and tissues during normal and abnormal palate and tooth development. Such knowledge provides the framework for further studies that can characterize the function of new or novel genes that have the potential of serving as therapeutic targets for correcting disorders like cleft palate and tooth agenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4f61558b14fefb4992652cd1c230a8a5c04bb3f" target='_blank'>
              Leveraging Spatial Transcriptomics to Decode Craniofacial Development
              </a>
            </td>
          <td>
            J. O. Piña, R. Raju, A. Myo, Evan Stipano, Malachi Wright, R.N. D’Souza
          </td>
          <td>2025-05-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="HOX (homeobox) genes are virtually absent in healthy adult brains but are detected in malignant brain tumors, particularly gliomas. In 2021, the World Health Organization (WHO) classified adult-type diffuse gliomas into three distinct categories: astrocytomas (isocitrate dehydrogenase [IDH]-mutated), oligodendrogliomas (IDH-mutated and 1p/19q-deleted), and glioblastomas, IDH-wildtype (GBM). GBM is the most common and aggressive primary malignant tumor of the Central Nervous System (CNS), characterized by its high recurrence rate and rapid growth. Dysregulation of HOX genes is a well-established phenomenon in both solid and liquid malignancies, playing crucial roles in various fundamental characteristics of cancer, including GBM. In recent years, HOX genes have gained recognition not only as key regulators of tumor progression but also as potential biomarkers for predicting disease outcomes and as promising therapeutic targets for GBM. This review compiles the latest research on HOX genes in GBM, encompassing studies published before and after the 2021 WHO classification of CNS tumors. Our goal is to provide a comprehensive overview of key findings on the role of HOX gene clusters, which are groups of genes involved in regulating the development of the body plan along the anterior–posterior axis, in GBM initiation, progression, prognosis, and treatment response. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02684-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e4a53ead8582c3ade2d72aeb49d5ea960a31a96" target='_blank'>
              HOX gene dysregulation in glioblastoma: a narrative review of current advances
              </a>
            </td>
          <td>
            Suleyman Hekim, S. Bunda, Shaliman Dilibaerguli, Maierdan Palihati, Yasin Mamatjan
          </td>
          <td>2025-05-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a lethal brain cancer with no effective treatment; understanding how GBM cells respond to tumor microenvironment remains challenging as conventional cell cultures lack proper cytoarchitecture while in vivo animal models present complexity all at once. Developing a culture system to bridge the gap is thus crucial. Here, a multicellular approach is employed using human glia and vascular cells to optimize a 3D brain vascular niche model that enabled not only long-term culture of patient derived GBM cells but also recapitulation of key features of GBM heterogeneity, in particular invasion behavior and vascular association. Comparative transcriptomics of identical patient derived GBM cells in 3D and in vivo xenotransplants models revealed that glia-vascular contact induced genes concerning neural/glia development, synaptic regulation, as well as immune suppression. This gene signature displayed region specific enrichment in the leading edge and microvascular proliferation zones in human GBM and predicted poor prognosis. Gene variance analysis also uncovered histone demethylation and xylosyltransferase activity as main themes for gene adaption of GBM cells in vivo. Furthermore, the 3D model also demonstrated the capacity to provide a quiescence and a protective niche against chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58339daec87fc88b9de1cfb8daf1fd79e18b6ecc" target='_blank'>
              3D Brain Vascular Niche Model Captures Glioblastoma Infiltration, Dormancy, and Gene Signatures.
              </a>
            </td>
          <td>
            Vivian K. Lee, R. Tejero, Nathaniel Silvia, Anirudh Sattiraju, Aarthi Ramakrishnan, Li Shen, Alexandre Wojcinski, Santosh Kesari, R. Friedel, Hongyan Zou, Guohao Dai
          </td>
          <td>2025-06-19</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 e14585


 Background:
 Immune checkpoint blockade (ICB) targeting PD-1/PD-L1 is crucial in treating breast cancer. Despite clinical success, the mechanisms by which ICB therapies reshape the tumor microenvironment (TME) and enhance anti-tumor activity remain unclear. Recent studies reveal that anti-PD-1 therapies broadly alter TME cells beyond PD-1+ T cells, highlighting extensive cell-cell communication (CCC) as a key factor. Understanding CCC changes in response to anti-PD-1 therapy can illuminate its mechanisms of action (MOA) and TME dynamics. This study applies causal inference methodology to investigate CCC between T cells and non-T cells in the TME of breast cancer patients receiving anti-PD-1 therapy.
 Methods:
 We analyzed single-cell RNA-seq data from 31 breast cancer patients pre- and on-treatment with anti-PD-1 (pembrolizumab) reported by Bassez et al., 2021. We identified differentially expressed genes (DEGs) across cell types. We applied the instrumental variable method to identify causal relationships between signals in T-cell and non-T-cell DEGs to uncover treatment-induced CCC. We further searched for ligand-receptor pairs mediating CCC and identified gene expression modules (GEMs) regulated by these ligand-receptors. Finally, we constructed a CCC network from T to non-T cells.
 Results:
 Anti-PD1 treatment induced broad gene expression changes in diverse cell populations. Major T cell pathways influenced by anti-PD-1 therapy include NF-κB, interferon-γ and interleukins. CD4
 +
 and CD8
 +
 exhausted cells, expressing high levels of
 PDCD1
 (encoding PD-1), exhibited distinct activated pathways. We identified the CCC network from T cells to non-T cells via ligand-receptor interactions. For example, anti-PD1 treatment activated cellular stress, apoptosis, and pro-inflammatory cytokine signaling pathways in T cells, altering the expression of a GEM that included
 TSC22D3
 and
 TXNIP
 . This initiated signaling to myeloid cells via
 RPS19
 -
 C5AR1
 , leading to NF-κB activation. CD4+ exhausted T cells primarily signaled to monocytes, enhancing helper functions to recruit and activate monocytes, thereby promoting immune regulation and amplification. In contrast, CD8+ exhausted T cells primarily interacted with macrophages, intensifying cytotoxic responses that facilitated effective tumor-cell killing.
 Conclusions:
 Our analyses provide insights into the dynamic interplay of cells within the TMEs during anti-PD-1 therapy. These findings could facilitate the identification of new biomarkers for predicting heterogeneous treatment responses to anti-PD-1 regimens, potentially enhancing the design and customization of immunotherapeutic strategies for breast cancer patients. Finally, this study demonstrated the utility of causal inference methodologies for mechanistic studies of CCC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74715a58ffb2aed0b55ed5b7dec461d0fe94b624" target='_blank'>
              Use of causal modeling to uncover cell-cell communication dynamics in the tumor microenvironment during anti-PD-1 therapy in breast cancer patients.
              </a>
            </td>
          <td>
            Aodong Qiu, Han Zhang, Joseph Ramsey, Boyang Sun, Bryan Andrews, Kun Zhang, Gregory F. Cooper, Lujia Chen, Xinghua Lu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Biliary tract cancers (BTCs) are aggressive malignancies encompassing intrahepatic and extrahepatic cholangiocarcinoma, gallbladder carcinoma, and ampullary carcinoma. Here, we report integrative analysis of 63 BTC cell lines via multi-omics and genome-scale CRISPR screens. We identify widespread EGFR dependency in BTC, alongside dependencies selective to anatomic subtypes. Additionally, we delineate strategies to overcome therapeutic resistance, with combined EGFR inhibition potentiating targeting of KRAS-mutant and FGFR2-fusion-driven models, and SHP2 inhibition effective in the latter context. Clustering RNA/protein expression and dependencies data revealed functional relationships transcending single-gene alterations, with biliary, squamous, or dual biliary/hepatocyte lineage signatures stratifying BTC models. These subtypes exhibit distinct dependency profiles- including cell fate transcription factors GRHL2, TP63, and HNF1B, respectively- and demonstrate prognostic significance in patient samples. Potential subtype-specific targetable vulnerabilities include Integrin-a3 and the detoxification enzyme UXS1. This cell line atlas reveals therapeutic targets in molecularly-defined BTCs, unveils disease subtypes, and provides a resource for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/156ff64109bed8af48870c13d716792fd5799d42" target='_blank'>
              Generation of a biliary tract cancer cell line atlas identifies molecular subtypes and therapeutic targets.
              </a>
            </td>
          <td>
            Vindhya Vijay, Negin Karisani, Lei Shi, Yu-Han Hung, Phuong Vu, Prabhat Kattel, Lauren Kenney, J. Merritt, Ramzi Adil, Qibiao Wu, Yuanli Zhen, Robert Morris, J. Kreuzer, Meena Kathiresan, Xcanda Ixchel Herrera Lopez, Haley Ellis, Ilaria Gritti, Lilian Lecorgne, Ines Farag, Alexandra Popa, William Shen, Hiroyuki Kato, Qin Xu, E. Balasooriya, Meng-Ju Wu, Jinkai Wan, Hiroshi Kondo, Saireudee Chaturantabut, Srivatsan Raghavan, Matthew D. Hall, S. Patnaik, Min Shen, R. Kelley, James M Cleary, Michael S Lawrence, D. Root, Krushna C Patra, Vanessa S Silveira, Cyril H. Benes, Vikram Deshpande, D. Juric, William R. Sellers, Cristina R Ferrone, W. Haas, Francisca Vazquez, Gaddy Getz, N. Bardeesy
          </td>
          <td>2025-05-12</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>102</td>
        </tr>

        <tr id="BACKGROUND
Atherosclerosis involves complex interactions between lipids, immune cells, vascular smooth muscle cells, and fibroblasts within the arterial wall. While significant advances in single-cell technologies have shed light on the roles of immune cells and vascular smooth muscle cells in plaque development, fibroblasts remain underexplored, leaving critical gaps in understanding their contributions to disease progression and plaque stability. Comprehensive characterization of fibroblast phenotypes in atherosclerosis is essential to unravel their diverse functions and to distinguish between subsets that may play protective versus pathogenic roles in the disease process.


METHODS
Here, we utilized cellular indexing of transcriptomes and epitopes by sequencing to comprehensively profile fibroblast diversity in a mouse model of atherosclerosis. Mice were fed an atherogenic diet for 0, 8, 19, and 26 weeks, representing distinct stages of disease progression, enabling a detailed phenotypic characterization of fibroblasts throughout the course of atherosclerosis development.


RESULTS
We identified 4 distinct fibroblast subpopulations, including a myofibroblast population closely resembling vascular smooth muscle cell-derived chondromyocytes. The proportions of these fibroblast subsets exhibited a modest decline as atherosclerosis progressed. Through multimodal analysis, we identified CD26 as a highly expressed and specific marker for one of these fibroblast subpopulations, distinguishing it from other subsets. Using a combination of flow cytometry and immunohistochemistry, we demonstrated that CD26+ fibroblasts predominantly reside in the adventitia of healthy arteries. During atherosclerosis progression, these cells expand into the intima and primarily localize within the fibrous cap of the lesion.


CONCLUSIONS
Our multiomic analysis highlights the phenotypic diversity and dynamic changes of fibroblasts during atherosclerosis progression. Among these, CD26+ fibroblasts emerge as a distinct subpopulation that expands within atherosclerotic lesions and may play a critical role in promoting plaque stability through their migration into the fibrous cap.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af796b09d1dc7b9ff103a4aea099fa62c9a6120" target='_blank'>
              Single-Cell Multimodal Profiling Reveals a Novel CD26+ Fibroblast Subpopulation in Atherosclerosis.
              </a>
            </td>
          <td>
            A. Bashore, Johana Coronel, Chenyi Xue, Lucie Y. Zhu, Muredach P. Reilly
          </td>
          <td>2025-06-12</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The evolution of isocitrate dehydrogenase (IDH)-wildtype glioblastoma (GBM) after standard-of-care therapy remains poorly understood. Here we analyzed matched primary and recurrent GBMs from 59 patients using single-nucleus RNA sequencing and bulk DNA sequencing, assessing the longitudinal evolution of the GBM ecosystem across layers of cellular and molecular heterogeneity. The most consistent change was a lower malignant cell fraction at recurrence and a reciprocal increase in glial and neuronal cell types in the tumor microenvironment (TME). The predominant malignant cell state differed between most matched pairs, but no states were exclusive or highly enriched in either time point, nor was there a consistent longitudinal trajectory across the cohort. Nevertheless, specific trajectories were enriched in subsets of patients. Changes in malignant state abundances mirrored changes in TME composition and baseline profiles, reflecting the co-evolution of the GBM ecosystem. Our study provides a blueprint of GBM’s diverse longitudinal trajectories and highlights the treatment and TME modifiers that shape them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0990e85585d1a285c19cfc504ce05c954e344b" target='_blank'>
              Deciphering the longitudinal trajectories of glioblastoma ecosystems by integrative single-cell genomics
              </a>
            </td>
          <td>
            Avishay Spitzer, K. Johnson, M. Nomura, L. Garofano, Djamel Nehar-Belaid, Noam Galili Darnell, Alissa C. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, Simona Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, Jong Bae Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, Simone P Niclou, Labeeba Nusrat, Sorcha Kellet, Sunit Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, Marc Sanson, S. Tanaka, Nobuhito Saito, D.M. Ashley, Stephen T Keir, Keith L. Ligon, Jason T Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-05-09</td>
          <td>Nature Genetics</td>
          <td>3</td>
          <td>62</td>
        </tr>

        <tr id="Background In the present study, we intended to discover predictive or prognostic factors of pancreatic ductal adenocarcinoma (PDAC). We intended to investigate the differences between PDAC cases that are treated with upfront surgery (UFS) and surgery after neoadjuvant FOLFIRINOX chemotherapy (NAT), and cases with good and poor responses to NAT, using digital spatial profiling (DSP) and immunohistochemical (IHC) analysis. Methods Forty-eight PDAC cases that were surgically resected with or without NAT were included. A tissue microarray was constructed for DSP and IHC. Pathological tumor regression to NAT was graded based on the College of American Pathologists (CAP) system. Results Between the UFS and NAT groups, there were no significant differentially expressed genes in all cell types. In the NAT group, MFAP4 and EGR3 were upregulated in CAP 2 in pan CK- and CD45-negative cells. Gene set enrichment analysis of CD45-positive cells showed that genes related to B or T cell-associated pathways were enriched in CAP 2, which correlated with the IHC; higher CD3-, CD4-, and CD8-positive cell densities in CAP 2. Multivariate analysis revealed age, high monocyte infiltration, and high CD68-positive cell infiltration as independent prognostic factors for overall survival. Conclusions Increased expression of MFAP4 and EGR3 as well as high CD3-, CD4-, and CD8-positive cell infiltration may be predictive markers of the NAT response in PDAC. Additionally, high monocyte infiltration and high CD68-positive cell infiltration could serve as prognostic markers for PDAC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06599-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9befa53f83d45764fa9fa52e4929caa248a1c35f" target='_blank'>
              Spatial transcriptomics reveal high T cell and monocyte status as predictive and prognostic markers in pancreatic cancer
              </a>
            </td>
          <td>
            Yeon Bi Han, Sejoon Lee, Jin-Ok Lee, Se In Jeong, Ki Rim Lee, Soomin Ahn, K. Jung, Jong-chan Lee, Yoo-Seok Yoon, Jin-Hyeok Hwang, Ho-Seong Han, H. Na, Jaihwan Kim
          </td>
          <td>2025-05-23</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Abrogating the immunoevasive role of the tumor immune microenvironment (TIME) represents a critical yet still elusive challenge in cancer treatment. Progress in this area has been hampered by both technological limitations and incomplete understanding of TIME-dependent immunoevasion mechanisms. We hypothesize that the immune-evasive role of TIME subpopulations—including regulatory T cells, cancer-associated fibroblasts, and tumor-associated macrophages—is critically mediated by hyperconnected Master Regulator Checkpoint (MRC) modules whose aberrant activity, as induced by paracrine signals, can be abrogated or modulated either genetically or pharmacologically. MRCs are primarily composed of transcription and co-transcription factors, acting downstream of surface receptors and signal transduction cascades to control the transcriptional identity and, ultimately, the phenotype of individual TIME subpopulations. Pharmacological inhibition of subpopulation-specific MRC proteins can thus help reprogram the TIME and potentially abrogate or modulate its immunosuppressive state. This paradigm shift, away from single ligand/receptor targeting, is supported by recent algorithmic, experimental, and clinical advances allowing systematic identification of MRCs and their pharmacological modulators using systems immunology-based approaches. Refocusing the deployment of existing tools and experimental methods that have proven successful in tumor cell contexts to identify and validate MRC-targeting agents capable of remodeling the immunosuppressive cell states of the tumor microenvironment can potentially pave the road to novel combination therapy synergizing with immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/992372e5de760c711f403d3ef7aa2e616073ce3d" target='_blank'>
              Systematic identification and targeting of master regulator checkpoints (MRC) governing tumor microenvironment-mediated immune evasion
              </a>
            </td>
          <td>
            Pasquale Laise, G. Bosker, Mariana Babor, Xiaoyun Sun, Stuart Andrews, Lorenzo Tomassoni, Andrea Califano, Mariano J. Alvarez
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinoma (SCC) is known for its stepwise progression from healthy skin to premalignant actinic keratosis (AK), followed by a malignant transformation to SCC. Unfortunately, less attention has been paid to changes in gene expression in the tumour microenvironment during this process. We retrospectively selected early‐stage cutaneous SCC tissue samples containing both invasive and premalignant portions and conducted a spatial transcriptomic experiment using a NanoString GeoMx Digital Spatial Profiler (DSP). First, we selected invasive and premalignant regions of interest (ROIs) for each tissue. We then compared the gene expression patterns between the two portions (invasive versus premalignant) of the three segments: tumour cells, immune cells and fibroblasts, in each ROI. As a result, early‐stage cutaneous SCC tissue samples from 17 patients were selected for this study. We identified 29, 14 and 15 differentially expressed genes (DEGs) between the invasive and premalignant portions of the tumour cells, immune cells and fibroblasts, respectively. The top three genes with the highest absolute log2 fold‐change were CCDC88C, GJD3 and COMP in tumour cells; SVEP1, TSLP and PPP2R5C in immune cells; and SPAG6, PPP1CA and CCDC68 in fibroblasts. Notably, several genes, such as COMP, SVEP1 and SPAG6, have been linked to the development and function of cancer‐associated fibroblasts. Functional enrichment analysis revealed that several pathways were altered in tumour and immune cells. In conclusion, distinctive changes in gene expression patterns were observed as AK progressed to SCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1212cb843dc2d0cad33b53da437b7c501e408d51" target='_blank'>
              Spatial Transcriptomics Shows a Distinctive Tumour Microenvironment in the Invasive Versus Premalignant Portion of Early Cutaneous Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jeong-An Gim, Chungyeul Kim, Hyun Jyung Oh, Ko Eun Kim, J. Jeon, Aeree Kim, Y. Baek
          </td>
          <td>2025-06-01</td>
          <td>Experimental Dermatology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The tumor immune microenvironment plays a critical role in tumor progression and immunotherapy response, with the abundance and composition of infiltrating immune cells serving as key determinants of therapeutic outcomes. Given the limitations of direct experimental methods, computational deconvolution algorithms are widely applied to infer immune cell infiltration from bulk RNA-seq data. While such estimates have proven valuable for studying tumor-immune interactions, large-scale, systematic analyses across multiple cancer types and treatment contexts remain limited. To address this need, we developed TIMER3 (https://compbio.cn/timer3/), an upgraded web server that substantially extends the functionality and scope of its predecessors. TIMER3 integrates 15 state-of-the-art immune deconvolution algorithms, including both human-specific and mouse-specific methods, to improve the robustness and interpretability of immune cell estimation. It incorporates an expanded collection of public RNA-seq datasets of immunotherapy-related cohorts, enabling large-scale analyses of immune dynamics across pre-treatment and post-treatment conditions. TIMER3 introduces new analytical modules for immunotherapy response, signature-based functional profiling, and interactive visualization of cell composition, gene expression, and survival associations. Collectively, TIMER3 provides a comprehensive, user-friendly platform for dissecting the TIME across diverse datasets, facilitating translational research and precision immuno-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916473c15b853461167c6dded26a3f5465619ac1" target='_blank'>
              TIMER3: an enhanced resource for tumor immune analysis.
              </a>
            </td>
          <td>
            Hao Cui, Guile Zhao, Yiwen Lu, Siying Zuo, Dingyu Duan, Xiaobo Luo, Hang Zhao, Jing Li, Zexian Zeng, Qianming Chen, Taiwen Li
          </td>
          <td>2025-05-08</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Standing as the most aggressive form of primary malignant tumor, Glioblastoma (GBM) tumors with marked heterogeneity represents one of the enormous challenges in glioma treatment. Myeloid cells, which includes neutrophils, myeloid-derived suppressor cells, microglia, and macrophages, play a pivotal role in the tumor microenvironment of GBM. In the tumor microenvironment (TME), T cells and natural killer (NK) cells exert anti-tumor functions, whereas myeloid-derived suppressor cells (MDSCs) can promote tumor progression by suppressing these immune responses. Therefore, MDSCs play a critical role in shaping the effectiveness of immunotherapy. TME has constrained the ability of traditional GBM treatment approaches to significantly enhance prognostic outcomes for patients. This category encompasses conventional therapies like surgical resection and radiation therapy, along with cutting-edge methodologies such as immunotherapy. Through extensive investigations into the dynamic interactions between the GBM microenvironment and neoplastic cells, both targeted treatment strategies and innovative immunotherapeutic modalities have emerged, offering promising new directions for clinical intervention. This review focuses on the interactions between GBM and myeloid cells (MCs), providing novel insights into the oncogenesis and progression of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e527685c44af6c300c634e53af18ed422e9ae9" target='_blank'>
              Interactions between glioblastoma and myeloid cells
              </a>
            </td>
          <td>
            Yuting Li, Yuhong Chen, Kai Cai, Yujuan Qin, Xi Wang, Bo Zhang, Lin Shi, Zonglin He, Jiasheng Wang, Jiecun Long, Yishun Zeng, Qiong Gong
          </td>
          <td>2025-06-24</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Ovarian cancer is one of the most common malignancies of the female reproductive system and is associated with poor prognosis. This study aimed to utilize single-cell RNA sequencing to investigate the heterogeneity of malignant epithelial cells in ovarian cancer, focusing on their potential functions and the implications for treatment and prognosis. Methods Single-cell RNA sequencing data were clustered using a single-cell transcriptome clustering method, and malignant epithelial cells were identified through copy number variation analysis. The interaction patterns between different malignant subpopulations and immune/stromal cells were analyzed using cell-to-cell communication analysis. A risk score (URS) model based on the UBE2C + epithelial subpopulation was then constructed through LASSO and multivariable Cox regression. High and low URS groups were compared in terms of tumor mutational burden (TMB), survival outcomes, and drug sensitivity. Finally, the role of GTF2F2 in ovarian cancer progression was validated through gene knockdown experiments in an ovarian cancer cell line (ES-2). Results Three major malignant epithelial cell subpopulations were identified (TMSB4X + Epi, TSC22D1 + Epi, and UBE2C + Epi). The UBE2C + Epi subpopulation exhibited higher stemness and greater invasive potential. The constructed URS model effectively stratified patients into high- and low-risk groups, with the high-risk group displaying a higher TMB level (p = 0.00011). Drug sensitivity predictions indicated that osimertinib, rapamycin, and dihydrorotenone might have stronger inhibitory effects in the high-risk group, whereas ERK inhibitors were more effective in the low-risk group. Functional assays demonstrated that GTF2F2 knockdown significantly suppressed ovarian cancer cell migration and invasion. Western blot analyses further showed elevated E-cadherin and reduced N-cadherin expression, suggesting that GTF2F2 may promote epithelial-mesenchymal transition (EMT). Conclusion The risk score model established in this study offers a novel framework for patient stratification and personalized therapy. Notably, the identification of the UBE2C + Epi subpopulation and key genes such as GTF2F2 highlights potential diagnostic and therapeutic targets, shedding light on the pathogenesis of ovarian cancer and paving the way for precision medicine approaches. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01686-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519b55d3039bd908fc634a743e91a97b194c1fa0" target='_blank'>
              Single-cell RNA sequencing reveals the role of GTF2F2 in ovarian cancer oncogenesis and progression
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-05-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Tumor-associated macrophages (TAMs) play a crucial role in the tumor microenvironment, regulating immune escape and promoting cancer progression. Understanding the role of phagocytosis regulators in female-specific cancers is essential for developing effective therapeutic strategies. Methods We performed comprehensive analyses of public databases to evaluate the expression, somatic mutations, and copy number variations of phagocytosis regulators. DNA methylation patterns, biological pathways, survival outcomes, and drug sensitivity were assessed. Additionally, immune modulators, immune cell infiltration, and single-cell sequencing were used to explore alterations in phagocytosis and their cellular origins. The functional role of HMGB2 in tumor cell behavior was validated through in vitro assays. Results Phagocytosis regulators exhibited differential expression across various female-specific cancers, with key genes such as CD47 and FOXO1 playing significant roles in modulating tumor progression. High-frequency mutations were found in PTEN, ARID1A, and UBR4. Genes like COX5B and MS4A1 emerged as potential predictors of clinical outcomes and therapeutic response. HMGB2 knockdown significantly inhibited cancer cell proliferation, migration, and invasion in female-specific cancers. HMGB2 knockdown in macrophages led to a significant impairment in phagocytosis of breast, cervical, ovarian, and endometrial cancer cells. Furthermore, when HMGB2 knockdown was combined with Palbociclib treatment, a significant decrease in tumor cell proliferation was observed across multiple cancer models. Conclusion This study highlights the pivotal role of phagocytosis regulators, particularly HMGB2, in the progression of female-specific cancers. Targeting HMGB2 offers promising therapeutic opportunities, potentially enhancing precision oncology and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04106b265eee3d853f545357f94ba192188932f5" target='_blank'>
              Pan-cancer analysis of phagocytosis regulators in female-specific cancers: a focus on HMGB2
              </a>
            </td>
          <td>
            Xiaoqin Lu, Dan Ren, Panpan Zhao, Yanfang Li, Zhenhui Wang, Jingyan Zhang
          </td>
          <td>2025-05-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains a leading cause of cancer-related mortality in men worldwide, with tumor heterogeneity, therapy resistance, and lineage plasticity posing significant clinical challenges. Androgen deprivation therapy (ADT), while initially effective, often culminates in castration-resistant prostate cancer (CRPC), may fueled by castration-resistant prostate cancer stem cells (CSCs) with adaptive self-renewal and regenerative capacities. Emerging evidence implicates prostate CSCs as pivotal contributors to tumor heterogeneity, drug resistance, and disease recurrence. These stem subpopulations exhibit intrinsic adaptability through genetic, epigenetic, and microenvironmental reprogramming, enabling survival under androgen-deprived conditions and fostering clonal diversification. This review synthesizes current knowledge on the identity, regulation, and functional dynamics of prostate stem cells, emphasizing their role in shaping the tumor ecosystem. By dissecting the genetic drivers, epigenetic alteration and crosstalk with microenvironment, this review underscores critical triggers on prostate CSCs determination and differentiation. Additionally, we discuss emerging strategies to target prostate CSCs-specific vulnerabilities, including molecular drivers of stemness and plasticity, to improve therapeutic outcomes for advanced prostate malignancies. This synthesis underscores the critical need to unravel prostate stem cells biology for developing precision therapies against prostate CSCs-driven adaptation in prostate diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/749e1d03aa4d620992d251430c919ea9331b3e5d" target='_blank'>
              Prostate cancer stem cell dynamics in the evolution of drug resistance.
              </a>
            </td>
          <td>
            Xiaoyu Zhang, Xiaoqing Zhang, Xinpei Deng, Jingna Cao, Qing Bao, Huan Wang, Dong Chen, Hailin Tang
          </td>
          <td>2025-06-04</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metabolic dysfunction-associated steatotic liver disease (MASLD) has limited treatments, and cell type-specific regulatory networks driving MASLD represent therapeutic avenues. We assayed five transcriptomic and epigenomic modalities in 2.4M cells from 86 livers across MASLD stages. Integrating modalities increased annotation of the genome in liver cell types several-fold over previous catalogs. We identified cell type regulatory networks of MASLD progression, including distinct hepatocyte networks driving MASL and mild and severe fibrosis MASH. Our single cell atlas annotated 88% of MASH-associated loci, including a third affecting hepatocyte regulation which we linked to distal target genes. Finally, we characterized hepatocyte heterogeneity, including MASH-enriched populations with altered repression, localization, and signaling. Overall, our results provide high-resolution maps of liver cell types and revealed novel targets for anti-MASH therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7d8151962524268bafd46f3bbc5c2abb6789115" target='_blank'>
              Single cell multiomics reveals drivers of metabolic dysfunction-associated steatohepatitis
              </a>
            </td>
          <td>
            Weston S Elison, Lei Chang, Yang Xie, Charlene Miciano, Qian Yang, Hannah M Mummey, Ryan Lancione, Sierra Corban, Sadatsugu Sakane, J. Lucero, S. Mamde, Hyun Young Kim, Matthew J Kim, Rebecca L. Melton, Luca Tucciarone, Audrey Lie, Timothy Loe, Tanmayi Vashist, Kelsey Dang, R. Elgamal, Daofeng Li, Melissa Vu, Elie N. Farah, Chad Seng, Jovina Djulamsah, Bing Yang, Justin Buchanan, Michael Miller, Mai Tran, J. Birrueta, Neil C Chi, Ting Wang, Agnieszka D’Antonio-Chronowska, Allen Wang, T. Kisseleva, David A. Brenner, Bing Ren, K. J. Gaulton
          </td>
          <td>2025-05-11</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f23b04434bb73a39ffc142be7c6ab3896c53a0" target='_blank'>
              Longitudinal single cell RNA-sequencing reveals evolution of micro- and macro-states in chronic myeloid leukemia
              </a>
            </td>
          <td>
            D. Frankhouser, Dandan Zhao, Yu-Hsuan Fu, Anupam Dey, Ziang Chen, Jihyun Irizarry, Jennifer Rangel Ambriz, S. Branciamore, Denis O’Meally, Lucy Ghoda, Jeffery M. Trent, Stephen J Forman, Adam L. Maclean, Ya-Huei Kuo, Bin Zhang, Russell C. Rockne, Guido Marcucci
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is an aggressive and rare neuroendocrine malignancy with poor prognosis. Here, we firstly performed single-cell transcriptional profiling derived from 10 SCCE patients, with normal esophageal mucosa, adjacent non-malignant tissue and tumors from esophageal squamous cell carcinoma (ESCC) as reference. We observed enrichment of activated regulatory T cells and an angiogenesis-induced niche existed in SCCE compared with ESCC, revealing an immune suppressive and vessel-induced tumor microenvironment (TME) in SCCE. Totally, we identified five TME ecotypes (EC1 ~ 5). Notably, EC1 was highly enriched in SCCE, associating with molecular subtyping and survival outcomes. To dissecting heterogeneity of epithelium in SCCE, we constructed eight transcriptional metaprograms (MPs) that underscored significant heterogeneity of SCCE. High expression of MP5 was linked to neuroendocrine phenotype and poor clinical survival. Collectively, these results, for the first time, systematically deciphered the TME and epithelial heterogeneity of SCCE and provided evidences that SCCE patients might benefit from anti-angiogenesis therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02335-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa69c6576495960bcb58dbd800d5471d898628a" target='_blank'>
              Dissecting small cell carcinoma of the esophagus ecosystem by single-cell transcriptomic analysis
              </a>
            </td>
          <td>
            Hao-xiang Wu, Yu-Kun Chen, Yingnan Wang, Jiaying Chen, Shu-Jing Xiang, Ying Jin, Zi-Xian Wang, Chun‐yu Huang, Lu-Ping Yang, Ye He, Wen-long Guan, Long Bai, Yanxing Chen, Min Wang, Chao-Ye Wang, Runjie Huang, Yue Huang, Jin-Ling Zhang, Zhi-Da Lv, Si-Qi Yang, Rui-Hua Xu, Qi Zhao, Feng Wang
          </td>
          <td>2025-05-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 e20113


 Background:
 Small cell lung cancer (SCLC) is a highly lethal malignancy, and there is an urgent need for more effective therapies. Antibody-drug conjugates (ADCs), which combine the precision of monoclonal antibodies with the cytotoxic power of small molecules, represent a promising approach. Despite extensive research, the relationship between ADC target overexpression and the transcriptional profile of SCLC remains unclear. Our study aimed to elucidate how ADC target expression correlates with transcriptional subtypes of SCLC, potentially guiding the selection of ADCs based on molecular profiling.
 Methods:
 RNA-seq and tandem mass spectrometry data for 107 patients with matched SCLC tumors and normal adjacent lung tissue were obtained from the CPTAC SCLC cohort. SCLC subtypes were defined according to expression of four transcription factors: achaete-scute homologue 1 (ASCL1), neurogenic differentiation factor 1 (NeuroD1), yes-associated protein 1 (YAP1) and POU class 2 homeobox 3 (POU2F3). ADC target expression for DLL3, SEZ6, B7-H3 (
 CD276
 ), and Trop2 (
 TACSTD2
 ) were assessed at the transcriptomic and proteomic level. The correlation between transcriptional subtype and ADC target expression was determined using Pearson’s correlation coefficient.
 Results:
 In 107 SCLC tumors, subtypes corresponded to 66 (61.6%) ASCL1, 21 (19.6%) NeuroD1, 3 (2.8%) YAP1, and 17 (15.9%) POU2F3. The ASCL1 subtype was positively correlated with DLL3 (R
 2
 =0.93, p=9E-87) and SEZ6 (R
 2
 =0.85, p=2.9E-57), and negatively correlated with TROP2 expression (R
 2
 =-0.72, p=1.5E-34). The NEUROD1 subtype showed moderate positive correlation with DLL3 (R
 2
 =0.52, p=4.8E-14) and SEZ6 (R
 2
 =0.58, p=3.6E-18), and a moderate negative correlation with TROP2 (R
 2
 =-0.52, p=1.6E-14). The POU2F3 subtype did not demonstrate strong positive or negative correlations with ADC targets. The YAP1 subtype was negatively correlated DLL3 (R
 2
 =-0.72, p=1.1E-32) and SEZ6 (R
 2
 =-0.71, p=9.5E-33), and positively correlated with Trop2 expression (R
 2
 =0.85, P=7.5E-61).
 Conclusions:
 Our findings reveal that molecular subtypes of SCLC exhibit unique patterns of ADC target expression. These profiles are crucial for further research and could lead to the development of subtype-specific treatments that improve clinical outcomes for patients with SCLC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34167c0dda56aaaeb23c8d98646f057387899f8a" target='_blank'>
              Proteogenomic characterization of antibody-drug conjugate target expression profiles in transcriptional subtypes of small cell lung cancer.
              </a>
            </td>
          <td>
            Edwin Lin, Mitch Singstock, Sanad Alhushki, Sameer S. Deshmukh, Changde Cheng, Kayla F. Goliwas, J. Deshane, A. R. Naqash, Fawzi Abu Rous, Rajat Thawani, Aakash Desai
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="BACKGROUND AND OBJECTIVES
Field cancerisation is the process that results in a group of cells acquiring some of the phenotypic changes of cancer prior to transformation into cancer. Clinically, an important challenge remains the ability to distinguish clonal lineages and microenvironments within cancerised fields that will remain indolent from those that will progress to malignant transformation. Spatial 'omics' can help us investigate genetic, epigenetic, transcriptomic, proteomic, and cellular microenvironments that transform normal cells into a cancerised field, and subsequently into cancer. In this review, we will discuss how spatial omics techniques have expanded our understanding of field cancerisation in prostate and bladder cancer, and the challenges associated with this research.


METHODS
We identified key articles relating to field cancerisation in bladder and prostate cancer. Special emphasis was placed on studies that used modern spatial profiling technologies and studies that were designed to investigate changes within normal tissue rather than simply using it as a control for tumour tissue.


RESULTS
Spatial omics research into field cancerisation has identified interesting early findings that have informed our understanding of: transformation of the benign epithelium and mechanisms of intra-prostatic clonal expansion for prostate cancer; clonal expansion within the normal urothelium; mutations that are unique to cancerised fields within the bladder; and how field cancerisation may prime the urothelium for cancer transformation.


CONCLUSIONS
Spatial omics profiling of field cancerisation can inform risk stratification and personalised treatment options. However, there are a number of challenges associated with the technologies that must be overcome before the potential of spatial omics can be fully realised in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/957e9305f80e5b98aef1432b61f0ded66b1ee007" target='_blank'>
              What spatial omics is teaching us about field cancerisation in prostate and bladder cancer.
              </a>
            </td>
          <td>
            Matthew H V Byrne, T. Anbarasan, Lisa Browning, Dan Woodcock
          </td>
          <td>2025-06-25</td>
          <td>BJU international</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Reactivation of transposable elements (TEs) in somatic tissues, particularly of LINE-1, is associated with disease by causing gene mutations and DNA damage. Previous work has shown that the PIWI pathway is crucial for TE suppression in the germline. However, the status and function of this pathway has not been well characterized in differentiated somatic cells and there is lack of consensus on the role of the pathway in somatic tumorigenesis. To shed light on this conundrum, we examined the PIWI pathway in colon cancer through a combination of bioinformatic analyses and cell-based assays. Shifted Weighted Annotation Network (SWAN) analysis revealed that the pathway experiences significant allelic losses in colon cancer and that PIWIL2, the main catalytic component of the pathway responsible for TE silencing, experiences the highest percent deletions. PIWIL2 is downregulated in colon tumors of advanced stage, nodal metastasis, and in certain subtypes, correlating with poor survival, while it is even downregulated in ulcerative colitis, an inflammatory bowel disease that predisposes to colon cancer. Knockout studies in colon epithelial Caco2 cells show that PIWIL2 depletion leads to increased anchorage-independent growth, increased LINE-1 levels and activity, and in DNA damage, altogether highlighting a tumor-suppressing role of PIWIL2 in the colon. SUMMARY STATEMENT This study investigates the PIWI-piRNA pathway in colon cancer using a nuanced bioinformatic and cell-based approach, linking the downregulation of PIWIL2 to disease progression, transposable element activation and DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6553455e0dd4bcf44406169f133238204316d7c1" target='_blank'>
              PIWIL2 downregulation in colon cancer promotes transposon activity and pro-tumorigenic phenotypes
              </a>
            </td>
          <td>
            Alyssa Risner, J. Nair-Menon, Abhinav Cheedipudi, Joe R Delaney, Vamsi K Gangaraju, A. Kourtidis
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Isoform switching in cancer is a prevalent phenomenon with significant implications for immunotherapy, as actionable neoantigens derived from these cancer-specific events would be applicable to broad categories of patients, reducing the necessity for personalized treatments. By integrating five large-scale transcriptomic datasets comprising over 19,500 samples across 29 cancer and 54 normal tissue types, we identified cancer-associated isoform switching events common to multiple cancer types, several of which involve genes with established mechanistic roles in oncogenesis. The presence of neoantigen-containing peptides derived from these transcripts was confirmed in broad cancer and normal tissue proteome datasets and the binding affinity of predicted neoantigens to the human leukocyte antigen (HLA) complex via molecular dynamics simulations. The study presents strong evidence that isoform switching in cancer is a significant source of actionable neoantigens that have the capability to trigger an immune response. These findings suggest that isoform switching events could potentially be leveraged for broad immunotherapeutic strategies across various cancer types. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00817-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f67b4f15a19d7345c9bb37824d41a5eb5aa238" target='_blank'>
              Potential shared neoantigens from pan-cancer transcript isoforms
              </a>
            </td>
          <td>
            Jirapat Techachakrit, Aijaz Ahmad Malik, T. Pisitkun, S. Sriswasdi
          </td>
          <td>2025-05-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Compelling evidences have manifested that breast cancer cells prefer to metastasize to certain distant organs, including brain, lung, bone and liver. According to the canonical “seed and soil” theory, this prominent biological behavior, termed as metastatic organotropism, involves intricate interactions between breast cancer cells (the “seeds”) and specific residents in the tumor microenvironment (the “soil”), initiating from pre-metastatic niche formation to metastatic outgrowth. Recently, multifaceted heterogeneity of tissue-specific macrophages (TSMs) and their roles played in organotropic metastases of breast cancer are incrementally unveiled. Herein, we decipher multiple diversities of TSMs, including evolvement, profiles, functions and metabolic characteristics under different polarization states. Further, we elaborate on bidirectional effects of TSMs on metastatic organotropism of breast cancer (both to the “seeds” and “soil”), and unearth underlying signaling pathways based on updated mechanistic researches. Lastly, we compile a series of clinical trials, hoping to illuminate promising TSM-targeting therapies against breast cancer organotropic metastases. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ed79b9316a9fed3f14b3bb67e4e61ce4c40709" target='_blank'>
              Heterogeneous tissue-specific macrophages orchestrate metastatic organotropism of breast cancer: implications for promising therapeutics
              </a>
            </td>
          <td>
            Cenzhu Wang, Pinchao Fan, Yu Zhou, Meican Ma, Haowei Zhong, Lei Liu, Qin Chen, Kun Xu
          </td>
          <td>2025-06-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aneuploidy, a hallmark of cancer, leads to widespread changes in chromosome copy number, altering the abundance of hundreds or thousands of proteins. How-ever, evidence suggests that levels of proteins encoded on affected chromosomes are often buffered toward their abundances observed in diploid cells. Despite its preval-ence, the molecular mechanisms driving this protein dosage compensation remain largely unknown. It is unclear whether all proteins are buffered to a similar degree, what factors determine buffering, and whether dosage compensation varies across different cell lines or tumor types. Moreover, its potential adaptive advantage and therapeutic relevance remain unexplored. Here, we established a novel approach to quantify protein dosage buffering in a gene copy number-dependent manner, show-ing that dosage compensation is widespread but variable in cancer cell lines and in vivo tumor samples. By developing multifactorial machine learning models, we identify mean gene dependency, protein complex participation, haploinsufficiency, and mRNA decay as key predictors of buffering. We also show that dosage com-pensation can affect oncogenic potential and that higher buffering correlates with reduced proteotoxic stress and increased drug resistance. These findings highlight protein dosage compensation as a crucial regulatory mechanism and a potential therapeutic target in aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63d2bd35509beaab88a25ac6ce826723195385e" target='_blank'>
              Explainable Machine Learning Identifies Factors for Dosage Compensation in Aneuploid Human Cancer Cells
              </a>
            </td>
          <td>
            Erik Marcel Heller, Karen Barthel, M. Räschle, Klaske M. Schukken, J. Sheltzer, Zuzana Storchová
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 e15100


 Background:
 Intratumoral heterogeneity, driven by phenotypic plasticity, contributes to disease progression and therapy failures. However, a thorough understanding of its underlying mechanisms is still lacking. Breast cancer exemplifies this with its histological and molecular diversity; estrogen receptor-positive (ER+), progesterone receptor-positive (PR+), and HER2- luminal A subtypes, such as the MCF7 cell line, are not the exception. While single-cell RNA sequencing (scRNA-seq) and three-dimensional multicellular tumor spheroids (MCTS) represent initial approaches to understanding heterogeneity, integrating these tools with computational strategies offers new paths to uncover the regulatory groundwork of tumor behavior. When modeled via Boolean frameworks, transcriptional regulatory networks (TRNs) provide a dynamic platform for studying gene interactions, identifying attractors (stable cellular states corresponding to distinct phenotypes), and simulating cellular transitions. This study represents the first step in constructing such a framework for ER+ breast cancer to advance precision medicine and improve therapeutic strategies.
 Methods:
 Regulatory elements relevant to the MCF7 cell line were identified through an extensive review of NCBI literature. Interactions were strictly curated based on evidence quality (ranging from robust experimental validation to predictive hypotheses), and evaluated for type and significance, such as feedback loops, activation, coactivation, and inhibition. Logical operators (AND, OR, NOT) were used to translate regulatory connections into Boolean functions to simulate binary gene expression dynamics. The final TRN was visualized using Cytoscape (v3.10.3) with a hierarchical layout.
 Results:
 The TRN consists of 54 nodes and 81 interactions, representing the regulatory landscape of ER+, PR+, and HER2- breast cancer. Boolean equations offer a framework for simulating the evolution of binary states to attractors, enabling the observation of how perturbations, such as gene overexpression or silencing, influence heterogeneity. While simulation and attractor analyses are ongoing, the network provides a foundation for elucidating the regulatory mechanisms underlying breast cancer plasticity.
 Conclusions:
 This study highlights the potential of integrating experimental data, TRN reconstruction, and Boolean modeling to understand intratumoral heterogeneity. Our rigorously curated model establishes a foundational framework for precision oncology and the integration of experimental and computational methods.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10947cef519fbed46d1a42e2676b4ea1fd778e16" target='_blank'>
              Boolean network modeling: The first step toward a new understanding of breast cancer heterogeneity.
              </a>
            </td>
          <td>
            Lia Medina Montalvo, Thelma Escobedo-Tapia, Osbaldo Resendis-Antonio
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Natural killer (NK) cells are innate lymphocytes endowed with potent cytotoxic activity. The presence of tumor-associated NK cells has been correlated with better prognosis in several solid tumors including colorectal cancer (CRC). This malignant disease is the second cause of cancer death worldwide and is in urgent need for novel approaches to improve current immunotherapies. Since CRC microenvironment can induce NK cell dysfunction and hinder cancer control, understanding tumor-associated NK cell features is mandatory to fully unlock their immunotherapeutic potential.Our study aims at elucidating the molecular and functional characteristics of tumor-associated NK cells in CRC focusing on the expression of immune checkpoints that critically regulate NK cell function. We performed an in-depth cytofluorimetric analysis of tumor-associated NK cells obtained by tissue dissociation of samples derived from 80 CRC patients comparing tumor with matched tumor-free tissue and peripheral blood, stratifying patients by tumor stage or MSI/MSS condition. Tumor tissue was also analyzed by immunohistochemistry.NK cells expressing immune checkpoints (i.e., KIR, NKG2A and TIM-3) were significantly enriched in tumor compared to tumor-free tissue, and an increase in PD-1+ NK cells was observed in tumors compared to peripheral blood and tumor-free tissue, indicating TME-induced modulation. Notably, tumor-associated PD-1+ NK cells characterized MSI rather than MSS CRC. In addition, tumor-associated NK cells also expressed tissue residency markers (CD103 and/or CD49a) and displayed a distinct profile also including the PD-1+ NK cell subset in MSI CRC, possibly representing NK cells recruited from circulation, retained in tumors, and reconfigured by TME signals. Importantly, tissue resident NK cells adequately expressed activating NK receptors and cytotoxic molecules.These results suggest, together with an increased PD-L1 expression on MSI tumor cells, that the efficacy of immunotherapies in MSI CRC based on PD-1/PD-L1 blockade could also rely on a superior anti-tumor potential of PD-1+ NK cells. Conversely, MSS CRC, in which tumor-associated PD-1+ NK cells are scarce, could benefit more from immunotherapies blocking NKG2A and/or KIRs. Thus, novel approaches based on NK cell features related to CRC type, fully exploiting circulating and resident NK cell anti-tumor activity, could be key to next-generation therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d24c3ead36cebff454d826553750607699c7935" target='_blank'>
              Colorectal cancer-infiltrating NK cell landscape analysis unravels tissue-resident PD-1+ NK cells in microsatellite instability tumors
              </a>
            </td>
          <td>
            Valentina Obino, Chiara Giordano, S. Carlomagno, Chiara Setti, Marco Greppi, Matteo Bozzo, Silvia Pesce, Elisa Ferretti, Simona Candiani, Letizia Muccio, Enrico Ciferri, Tania Buttiron Webber, Agnese Solari, Fulvia Ortolani, L. Paleari, Matteo Clavarezza, Andrea Barberis, M. Filauro, N. Provinciali, M. Rutigliani, E. Marcenaro, A. DeCensi, M. Della Chiesa, S. Sivori
          </td>
          <td>2025-06-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) has a poor survival rate due to late detection. PDAC arises from precursor microscopic lesions, termed pancreatic intraepithelial neoplasia (PanIN), that develop at least a decade before overt disease--this provides an opportunity to intercept PanIN-to-PDAC progression. However, immune interception strategies require full understanding of PanIN and PDAC cellular architecture. Surgical specimens containing PanIN and PDAC lesions from a unique cohort of five treatment-naïve patients with PDAC were surveyed using spatial-omics (proteomic and transcriptomic). Findings were corroborated by spatial proteomics of PanIN and PDAC from tamoxifen-inducible KPC (tiKPC) mice. We uncovered the organization of lymphoid cells into tertiary lymphoid structures (TLSs) adjacent to PanIN lesions. These TLSs lacked CD21+CD23+ B cells compared to more mature TLSs near the PDAC border. PanINs harbored mostly CD4+ T cells with fewer Tregs and exhausted T cells than PDAC. Peri-tumoral space was enriched with naïve CD4+ and central memory T cells. These observations highlight the opportunity to modulate the immune microenvironment in PanINs before immune exclusion and immunosuppression emerge during progression into PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6b6b93e0faedffa7b598bebfd58275c143f057" target='_blank'>
              Spatial Proteomics and Transcriptomics Reveal Early Immune Cell Organization in Pancreatic Intraepithelial Neoplasia.
              </a>
            </td>
          <td>
            Melissa R Lyman, Jacob T. Mitchell, Sidharth Raghavan, Luciane T. Kagohara, Amanda L. Huff, Saurav D Haldar, Sarah M. Shin, Samantha Guinn, Benjamin Barrett, Gabriella Longway, Alexei Hernandez, Erin M Coyne, Xuan Yuan, Lalitya Andaloori, Jiaying Lai, Yun Zhou Liu, Rachel Karchin, Anuj Gupta, A. Kiemen, André Forjaz, Denis Wirtz, Pei-Hsun Wu, Atul Deshpande, Jae W Lee, T. Armstrong, Nilofer S. Azad, Jacquelyn W. Zimmerman, Laura D. Wood, Robert A Anders, Elizabeth D. Thompson, Elizabeth M. Jaffee, E. Fertig, W. Ho, N. Zaidi
          </td>
          <td>2025-06-26</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a crucial role in the progression of tumors, resistance to immunotherapy, and the development of thyroid carcinoma (THCA). However, the precise mechanisms involved remain poorly understood. In this study, 26 mRNAs linked to cancer stemness were identified through an analysis of 13 publicly available THCA transcriptomic datasets alongside a CRISPR dataset for thyroid cancer cell lines. Notably, we revealed an inverse relationship between the stemness of cancer cells and the efficacy of immunotherapy. Utilizing multiple machine learning approaches, we created and confirmed a model for tumor stemness cells (TSC), which included genes associated with cancer stemness. This model successfully forecasted patient outcomes and responses to treatment with Cytotoxic T-Lymphocyte Associated Protein 4 (CTLA4) immune checkpoint inhibitors (ICIs) across different types of cancer. Additionally, chromosomal amplifications in regions 1q and 8p were identified as intrinsic contributors to the onset of thyroid cancer. Specifically, based on single cell dataset, the amplification of CKS1B on chromosome 1q was revealed to advance the progression of thyroid cancer cells by boosting their proliferation. Furthermore, experimental results revealed that the heightened expression of CKS1B considerably enhanced the proliferation and invasion capabilities of THCA, suggesting it could serve as a possible therapeutic target for this condition. In summary, our study sheds light on the relationship between the traits of cancer stem cells and the efficacy of immunotherapy, providing a novel model for predicting the prognosis and individual responses to immunotherapy in patients with THCA. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02813-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/865efc03cf551caa6a2f623a0978529554bdabd9" target='_blank'>
              Multi-omics analysis reveals cancer stemness function and establishes a predictive model for thyroid cancer prognosis and immunotherapy
              </a>
            </td>
          <td>
            Zhanrong Feng, Yue-Chao Xu, Ying Ding, Xiaoxiao Sun, Qian Zhao, Jingjing Zhu, Qiang Chen, Yang Zhang, Yilai Zhang
          </td>
          <td>2025-06-03</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 e20009


 Background:
 Osimertinib resistance in patients with EGFR-driven non-small cell lung cancer (NSCLC) contributes to mortality. Targeting bypass alterations have had limited efficacy. Here, we explored the role of CD105 (endoglin) expression, a co-receptor for bone morphogenetic protein (BMP) signaling as a factor of osimertinib treatment.
 Methods:
 Sixty-six NSCLC tumors, 44 of which had EGFR mutations were subjected to imaging mass cytometry to identify CD105 expression status as a predictor of clinical outcomes. To study the effects of EGFR and CD105 disruption two osimertinib-resistant EGFR-mutant cell lines were developed for subsequent characterization by single cell RNA sequencing. The promoter chromatin status and glycolytic state of the lines were evaluated by ATAC sequencing and Single Cell ENergetIc metabolism by profiling Translation inHibition analysis (SCENITH), respectively.
 Results:
 There was an inverse correlation between the expression CD105 and overall survival in treatment-naïve patients (
 P
 = 0.0071). Modeling osimertinib resistance highlighted the emergence of a persister cell population expressing tumor dormancy features. Further, the persister cells were found to have elevated CD105 levels with responsivity to the neutralizing antibody, carotuximab (ENV105), in restoring epigenetic and metabolic reprogramming to support osimertinib sensitivity. The carotuximab mechanism of action was through the re-engagement of EGFR signaling by potentiating physical CD105/EGFR interaction. Combination of osimertinib and carotuximab significantly reduced tumor growth in two independent NSCLC mouse models (
 P
 = 0.0056 and
 P =
 0.0008), where either single agent had little benefit.
 Conclusions:
 CD105 targeting provides a novel mutation-independent strategy to overcome resistance in EGFR-mutant NSCLC. Based on the efficacy of carotuximab capacity to limit tumor dormancy and restored osimertinib sensitivity, a Phase 1 trial (NCT05401110) investigating the combination therapy is currently accruing subjects.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/403b9eb716a17d3e6aa13d9c70998a95aef3047d" target='_blank'>
              Effect of carotuximab (ENV105)-mediated CD105 blockade on osimertinib sensitivity in EGFR-mutant non-small cell lung cancer persister cells.
              </a>
            </td>
          <td>
            Neil A. Bhowmick, Sandrine Billet, B. Waissengrin, K. Sankar, E. Posadas, Sungyong You, Jean Lopategui, Karen L. Reckamp, Manish Thiruvalluvan
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d135048b3a2d9e0d80a5651add7d7f82d46f7f4a" target='_blank'>
              Innate immunity and the NF-κB pathway control prostate stem cell plasticity, reprogramming and tumor initiation.
              </a>
            </td>
          <td>
            Chen Jiang, Yura Song, S. Rorive, Justine Allard, Elisavet Tika, Zahra Zahedi, C. Dubois, Isabelle Salmon, Alejandro Sifrim, Cédric Blanpain
          </td>
          <td>2025-06-23</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 8084


 Background:
 Pleural mesothelioma (PM) is an aggressive malignancy that harbors significant inter- and intra-tumoral heterogeneity. Spatial transcriptomics enables the dissection of the tumor's molecular architecture by facilitating compartment-specific gene expression profiling. We performed high-resolution RNA-seq analysis of tumor (Tm) and stroma (St) compartments in PM samples to identify gene expression patterns and their association with clinical outcomes.
 Methods:
 Formalin-fixed paraffin-embedded (FFPE) tumor samples from untreated PM patients (pts) across three institutions were analyzed using the NanoString GeoMx Digital Spatial Profiling (DSP) platform. Regions of interest (ROIs) were selected based on histopathological features and fluorescently labeled antibodies for tumor and stromal areas. RNA expression of >1800 genes from selected ROIs was analyzed using GeoMx Cancer Transcriptome Atlas (CTA). Differential gene expression was assessed utilizing R “limma” package. A cutoff of absolute fold change ≥1 and p-value <0.05, with the Benjamini-Hochberg false discovery rate method, was applied to identify significant differentially expressed genes (DEGs). Elastic Net regression optimized through cross-validation methods was employed to identify genes associated with overall survival (OS) outcomes. Data from the TCGA PanCancer Atlas was utilized for external validation.
 Results:
 A total of 72 pts, 80.3% male, median age of 71y (range: 44-94) were identified for the analysis. Among them, 87.5% (63/72) were epithelioid (Ep) and 12.5% (9/72) non-epithelioid (NEp). After quality control, RNA data was available from 71 and 67 pts in Tm and St compartments, respectively. Across 132 ROIs in the Tm compartment, we identified 4 significantly DEGs between NEp (upregulated
 COL5A2, THBS1
 ; downregulated
 CLU, KRT19
 ) and Ep subgroups. No DEG between Ep and NEp subgroups were identified in 115 ROIs from the St compartment. Unsupervised clustering identified four molecular subgroups with distinct gene expression in the Tm compartment, of which Cluster 1 showed significantly decreased OS (6.3m vs. 16.4m; HR 3.3,
 p
 =0.001). Elastic Net regression identified 31 genes predictive of OS (R² = 0.43, Harrell’s c-index = 0.87), including nine genes (
 IFNGR2, FCER1G, MFGE8, CKLF, CBL, HLA-DRB3, HK1, PLAT, CD163
 ) associated with worse prognosis. Tumors in Cluster 1 demonstrated higher expression of these genes. External validation using the TCGA cohort confirmed four genes
 IFNGR2
 (
 p
 = 0.02),
 CBL
 (
 p
 = 0.01),
 HK1
 (
 p
 <0.001),
 PLAT
 (
 p
 <0.001), as significantly associated with decreased OS in PM.
 Conclusions:
 Spatially resolved transcriptomic profiling suggests Tm-enriched regions as the primary drivers of PM subtype and aggressiveness, identifying nine genes and a molecular subgroup associated with poorer survival outcomes. Further validation and functional studies are warranted.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e449bee363e2fcac7be12428599b558c6df5c9ff" target='_blank'>
              Digital spatial profiling for identification of prognostic genes and molecular subgroups in pleural mesothelioma.
              </a>
            </td>
          <td>
            Mercedes Herrera, David Lora, M. Peressini, Ramon Yarza, J. Gracia, Álvaro C. Ucero, Magdalena Molero Abraham, A. Enguita, David Gómez-Sánchez, Vera Adradas Rodriguez, Esther Conde, N. Carrizo, Igor Gómez-Randulfe, Roxana Reyes, Noemi Reguart, J. Baena, Helena Bote de Cabo, S. Ponce Aix, L. Paz-Ares, Jon Zugazagoitia
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Given their rarity and diversity, a fundamental understanding of the genomic underpinnings for many sarcoma subtypes is still lacking. To better define the molecular landscape of this group of diseases, we perform matched whole exome sequencing and RNA sequencing on a cohort of 1340 sarcoma tumor specimens. We identify recurrent somatic mutations and observe an increased mutational burden in metastatic vs. primary samples (p < 0.001). We observe frequent copy number alterations including whole genome doubling, with this feature being more common in metastatic tumors (p = 0.026). Estimation of immune cell abundances followed by hierarchical clustering identifies five immune subtypes ranging from low to high and we observe inferior overall survival in immune deplete clusters compared to immune enriched (p < 0.01). Interestingly, GIST predominantly form a distinct “immune intermediate” cluster that is marked by a specific enrichment for NK cells (FDR < 0.01).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09dedff6b3501b2376796d9aaef10264491ab2ba" target='_blank'>
              Genomic, transcriptomic, and immunogenomic landscape of over 1300 sarcomas of diverse histology subtypes
              </a>
            </td>
          <td>
            Alex C. Soupir, Oscar E. Ospina, Oliver Hampton, Michelle L. Churchman, Michael D. Radmacher, D. Hedges, David M. McKean, Phaedra Agius, Saman Zeeshan, Nathan D. Seligson, Raphael Pollock, D. Liebner, James L. Chen, G. Tinoco, B. Salhia, Martin D McCarter, B. Wilky, Benjamin J. Miller, M. Cavnar, John S. Groundland, Bryan P. Schneider, Gregory Riedlinger, Stephen B. Edge, C. Moskaluk, Kenneth Cardona, A. R. Naqash, Ricardo J. Gonzalez, J. Mullinax, David M Joyce, O. Binitie, G. Douglas Letson, A. Naghavi, M. Druta, Damon R Reed, E. Siegel, Jamie K Teer, Brooke L. Fridley, Andrew S Brohl
          </td>
          <td>2025-05-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Pancreatic cancer is a highly lethal malignancy with limited treatment response. Despite advancements in treatment, systemic chemotherapy remains the primary therapeutic approach for over 80% of patients, with no established biomarkers to guide drug selection. Traditional two-dimensional (2D) culture models fail to replicate the tumor microenvironment, necessitating the development of more advanced models, such as three-dimensional (3D) organoid models. Methods We established 3D organoid cultures using patient-derived conditionally reprogrammed cell (CRC) lines, originally cultured under 2D conditions. These CRC organoids were developed using a Matrigel-based platform without organoid-specific medium components to preserve the intrinsic molecular subtypes of the cells. Morphological, molecular, and drug sensitivity analyses were performed to compare the clinical responses of 3D CRC organoids with those of their 2D counterparts and clinical responses. Results The 3D CRC organoids retained the molecular characteristics, transcriptomic and mutational profiles of the parental tumors and displayed distinct morphologies corresponding to cancer stages and differentiation. Drug response profiling of gemcitabine plus nab-paclitaxel (Abraxane) and FOLFIRINOX demonstrated that the 3D organoids more accurately mirrored patient clinical responses than the 2D cultures. Notably, the IC50 values for the 3D organoids were generally higher, reflecting the structural complexity and drug penetration barriers observed in vivo. Conclusion Matrigel-based 3D organoid culture models provide a robust platform for pre-clinical drug evaluation, overcoming the limitations of 2D models. Although time- and resource-intensive, integrating both 2D and 3D platforms enables efficient initial screening and validation. This approach holds promise for identifying predictive biomarkers and advancing precision medicine in pancreatic cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02374-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b838d6d2ae555a96968c36aded77ee037334aba" target='_blank'>
              Establishing 3D organoid models from patient-derived conditionally reprogrammed cells to bridge preclinical and clinical insights in pancreatic cancer
              </a>
            </td>
          <td>
            Jin Su Kim, Chan Hee Park, Eunyoung Kim, Hee Seung Lee, Jinyoung Lee, Jeehoon Kim, Eun Hee Kam, Sanghee Nam, M. Chung, J. Park, Seung Woo Park, Sangwoo Kim, Galam Leem, Seungmin Bang
          </td>
          <td>2025-06-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Melanoma remains one of the deadliest skin cancers, despite accounting for only a small proportion of overall skin cancer diagnoses. Although immune checkpoint inhibitors (ICIs) have significantly improved outcomes in advanced melanoma, nearly half of all patients fail to respond, highlighting the urgent need for clinically actionable biomarkers. In this study, we leveraged single-cell RNA sequencing (scRNA-seq) data from a publicly available melanoma cohort to dissect the mechanisms driving therapeutic resistance. Our analyses revealed a prominent enrichment of macrophages in non-responders, prompting the development of a macrophage-focused gene signature that was strongly associated with poor overall survival across multiple independent datasets, including The Cancer Genome Atlas (TCGA). Importantly, this signature demonstrated high predictive accuracy (AUC = 0.896) for identifying patients unlikely to benefit from ICIs. Functional validation using an in vitro co-culture model showed that M2-polarized macrophages robustly suppressed T cell proliferation, reinforcing their immunosuppressive role in the tumor microenvironment. These findings offer immediate translational relevance: the macrophage-derived gene signature could be incorporated into clinical workflows to guide therapeutic decision-making, sparing patients the financial and physical burdens of ineffective treatment. Moreover, targeting the immunosuppressive M2 macrophage axis may open new avenues for combination therapies, ultimately paving the way for more personalized and cost-effective management of advanced melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5f85a48514f86ac93a7d40ef2d29e623a8d8df5" target='_blank'>
              Cellular and Molecular Biomarkers to Predict Immunotherapy Outcomes in Melanoma Patients
              </a>
            </td>
          <td>
            Tingjia Shi
          </td>
          <td>2025-05-30</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e16331


 Background:
 Cholangiocarcinoma (CCA) describes a malignancy originating from the epithelial cells of the bile ducts. While rare, this malignancy often has an unfortunate outlook given its advanced stage at presentation. Development of CCA likely depends upon a gradual accumulation of genetic mutations. These mutations in-turn, transition precursor lesions, biliary intraepithelial neoplasia and intraductal papillary mucinous neoplasm, into the cancer that is CCA. The tumor microenvironment has been increasingly studied and likely plays a big role in the pathogenesis of CCA. A variety of cells contribute to said environment, ranging from regulatory T cells(Treg) to natural killer(NK) cells. The role of the NK cell is being increasingly studied in the context of CCA as it is one of the most abundant immune cells in the tumor microenvironment and likely plays a large role in the immune response to cancer cells.
 Methods:
 Our sample set included 36 patients from a publicly available RNA sequencing dataset from MD Anderson. RNA sequences were extracted and then input into a Stanford based application, CIBERSORT. This illustrates the impact different immune cells have on a tumor microenvironment. The immune cells included are the following: B-cells naïve, B-cells memory, plasma cells, CD8 T-cells, CD4 T-cells, T-cells follicular, T-cells regulatory, natural killer activated cells, natural killer resting cells, monocytes, macrophages (M0, M1, M2), activated dendritic cells, resting dendritic cells, activated mast cells, resting mast cells, eosinophils, and neutrophils. CIBERSORT uses immune cell fractions as the marker of immune cell impact. These fractions were then analyzed in a statistical tool, Radiant by Shiny, that computes a linear regression describing the relationship between immune cells and impact on overall survival.
 Results:
 Natural killer cells in the tumor microenvironment have a strong, positive impact on overall survival (p = 0.044). The coefficient of correlation was +198.
 Conclusions:
 Results of our analysis illustrate the strong and positive relationship that NK cells have on the overall survival of CCA. This is likely due to the NK cell’s ability to promote cell death through various pathways, such as the Fas-L and TNF apoptotic pathways. More specifically, it has been shown that cholangiocytes may have the ability to present lipid antigens to NK cells thus activating an alternate killing pathway making the NK cell a crucial player in the tumor microenvironment. This, coupled with being one of the most abundant cells in the tumor microenvironment in CCA, opens the door for the NK cell as an potential target for future treatment options. Understanding these specific interactions between NK cells and tumor cells proves to be essential for developing novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a1da82ab42c7f1091a653ec8bd73313150dda1f" target='_blank'>
              The tumor microenvironment in cholangiocarcinoma: Impact of natural killer cells on overall survival.
              </a>
            </td>
          <td>
            Morgan Kiryakoza, Lorin Sallan, Paul F Hanona, Neil Jairath, Daniel Ezekwudo, George Howard, I. Jaiyesimi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Advances in single-cell and spatial omics technologies have revolutionised biology by revealing the diverse molecular states of individual cells and their spatial organization within tissues. The field of plant biology has widely adopted single-cell transcriptome and chromatin accessibility profiling and spatial transcriptomics, which extend traditional cell biology and genomics analyses and provide unique opportunities to reveal molecular and cellular dynamics of tissues. Using these technologies, comprehensive cell atlases have been generated in several model plant species, providing valuable platforms for discovery and tool development. Other emerging technologies related to single-cell and spatial omics, such as multiomics, lineage tracing, molecular recording, and high-content genetic and chemical perturbation phenotyping, offer immense potential for deepening our understanding of plant biology yet remain underutilised due to unique technical challenges and resource availability. Overcoming plant-specific barriers, such as cell wall complexity and limited antibody resources, alongside community-driven efforts in developing more complete reference atlases and computational tools, will accelerate progress. The synergy between technological innovation and targeted biological questions is poised to drive significant discoveries, advancing plant science. This review highlights the current applications of single-cell and spatial omics technologies in plant research and introduces emerging approaches with the potential to transform the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd263d7989d8d7fac3ccdc19b4b2541ba1d90c6a" target='_blank'>
              Exploring the untapped potential of single-cell and spatial omics in plant biology.
              </a>
            </td>
          <td>
            T. Nobori
          </td>
          <td>2025-05-21</td>
          <td>The New phytologist</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1093733a8ade229255066e064b631c0ae1860e" target='_blank'>
              ImmunoFusion: A Unified Platform for Investigating RNA-seq-Derived Gene Fusions in Cancer and Immunotherapy
              </a>
            </td>
          <td>
            YanKun Zhao, Shixiang Wang, Shensuo Li, Minjun Chen, Su-Han Jin, U. Gaipl, Hu Ma, Jian-Guo Zhou
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by significant heterogeneity and a poor prognosis. M2 macrophages (M2) play a crucial role in the microenvironment of PDAC, yet their exact functional significance remains poorly understood. This study sought to explore the potential communication and clinical implications of M2 in PDAC. The differentiation pathway of M2 and their interaction with malignant PDAC cells were analyzed using single-cell RNA sequencing (scRNA-seq). Additionally, a prognostic signature related to M2 marker genes was constructed and analyzed using bulk RNA sequencing. Three distinct differentiation states of M2 were characterized, each exhibiting unique transcription factor expression profiles and signaling pathway activations. Potential intercellular communication between M2, PDAC malignant cells, and fibroblasts was initially observed at the level of computer analysis. The prognostic model was identified through the LASSO algorithm. Patients stratified by risk level displayed significant variations in prognostic indicators such as signaling pathway activity, gene mutation frequency, immune cell infiltration, and drug sensitivity. These findings are of course preliminary, and future in vitro/in vivo experiments will be needed to knock out these signature genes or ligand-receptor pairs of genes in macrophages to validate their role in macrophage development. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02838-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/273ead8bc63462bd01d5d5a63b43ec9833ef2540" target='_blank'>
              Establishment of M2 macrophage-related prognostic signature in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Hui Chen, Jingyang Yin, Xianxing Wang, Renpei Xia, Fanbo Qin, Zhuo Li, Jiali Yang, Changjiang Liu, Shixiang Guo, Huaizhi Wang
          </td>
          <td>2025-06-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 8565


 Background:
 Non-small cell lung cancer (NSCLC) patients with EGFR mutations or ALK fusions often exhibit suboptimal responses to immunotherapy, yet the genomic and immunological basis for this remains poorly understood. This study aims to elucidate the genomic and immune microenvironment differences in NSCLC patients harboring EGFR mutations or ALK fusions that may contribute to their distinct immunotherapy responses.
 Methods:
 We analyzed tumor specimens from 12,528 NSCLC patients using a comprehensive next-generation sequencing (NGS) panel targeting 733 genes. Of these, 191 patients also underwent multiplex fluorescence immunohistochemistry (mIHC) analysis to assess the immune microenvironment. Subgroups included patients with EGFR mutations, ALK fusions, both EGFR mutations and ALK fusions, and those wild-type for both EGFR and ALK. We compared these groups for DNA damage response (DDR) gene mutation frequencies, tumor mutational burden (TMB), intratumoral heterogeneity (ITH), and immune microenvironment.
 Results:
 Among the cohort, 6389 (51%) harbored EGFR mutations and 300 (2.4%) had ALK fusions. Patients with EGFR mutations or ALK fusions exhibited significantly lower DDR mutation frequencies compared to their wild-type counterparts (P<0.001). TMB levels showed a gradient: wild-type > EGFR mutation > EGFR mutation with ALK fusion > ALK fusion (P< 2.2e−16), with wild-type patients having markedly higher TMB than any other group (P< 2.22e−16, P=0.0047, P< 2.22e−16). Conversely, ITH was lowest in wild-type patients and progressively higher with ALK fusions (P=0.017), EGFR mutations (P< 2.22e−16), and combined mutations (P< 2.22e−16). Lower ITH was associated with higher immunogenic neoantigen production, correlating with improved immunotherapy responses. Immunohistochemical analysis revealed that EGFR-mutant tumors had significantly fewer M1 tumor-associated macrophages (CD68+ HLA-DR+) (P<0.001)within the tumor parenchyma and reduced CD4+ T cell (P=0.013) infiltration in the tumor stroma compared to wild-type.
 Conclusions:
 Our findings suggest that low TMB, high ITH, and a suppressed immune microenvironment characterize EGFR-mutant and ALK-fusion NSCLC, potentially undermining their immunotherapy efficacy. These genomic and immunological signatures, particularly ITH and immune cell distribution, might be critical factors affecting the immunotherapy responsiveness of these patient subsets, warranting further investigation into tailored therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44f43a8981bce67c58920709ed029cc5cd201ab7" target='_blank'>
              Analysis of genomic and immune microenvironment differences in Chinese populations: Revealing potential mechanisms of poor immunotherapy outcomes in patients with EGFR mutation and ALK fusion non-small cell lung cancer.
              </a>
            </td>
          <td>
            Junhong Lü, Weixiong Huang, Yongqiang Chen, Yating Zheng, Panli Peng
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Metabolic reprogramming is a hallmark of cancer, yet its role in glioma remains poorly understood. Gliomas are characterized by a highly immunosuppressive tumor microenvironment (TME) and poor prognosis. This study systematically explores the relationship between glioma metabolomics, tumor phenotype, and the immune microenvironment. Methods Bulk RNA sequencing data were retrieved from the Chinese Glioma Genome Atlas (CGGA) and The Cancer Genome Atlas (TCGA). Single-cell gene set enrichment analysis (ssGSEA) was employed to quantify seven nutrient metabolic pathways and immune infiltration. Consensus clustering was applied to group gliomas based on metabolic gene expression, and survival analysis was performed to evaluate survival differences across these clusters. A predictive model was constructed and validated using our cohort. Finally, we knocked out G0S2 in glioma cells and performed RNA sequencing to investigate differentially activated pathways. Additionally, in vivo experiments were conducted to explore the antitumor effects of G0S2 knockout in combination with PD-1 monoclonal antibody. Results Significant metabolic differences were identified between low-grade gliomas (LGG) and glioblastomas (GBM), with consistent findings across both databases. We found that LGGs and GBMs exhibit distinct metabolic patterns. Consensus clustering revealed three metabolic subgroups, with the C3 subgroup demonstrating poor survival and enhanced infiltration of immunosuppressive cells. The predictive model showed robust performance in forecasting the survival of glioma patients. Functional analysis identified G0S2 as a key metabolic regulator highly expressed in gliomas. G0S2 knockout activated the type I interferon signaling pathway, enhanced CD8+ T cell functionality, and synergized with anti-PD-1 therapy, resulting in suppressed tumor growth and prolonged survival in vivo. Conclusion These findings provide a comprehensive analysis of glioma metabolic patterns and identify G0S2 as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/558a3d2f46f81261c799a7e02e2b5142df29476d" target='_blank'>
              Metabolic profiling of glioblastoma and identification of G0S2 as a metabolic target
              </a>
            </td>
          <td>
            Jianlei Kang, Yu-Jun Xu, Qitai Zhao, Ying Wang, Zhenyan He, Xin Xu
          </td>
          <td>2025-05-30</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lineage plasticity is recognized as a critical determinant of lethality and resistance to AR pathway inhibitors in prostate cancer. Lineage plasticity is a continuum, ranging from AR activity-low tumors, AR-null tumors that do not express a neuroendocrine prostate cancer (NEPC) program (i.e., double-negative prostate cancer [DNPC]), and AR-null NEPC tumors. Factors upregulated early in lineage plasticity are not well-characterized. The clarification of such factors is essential to identify tumors undergoing lineage plasticity or at risk of this occurring. Our integrative analysis of metastatic prostate cancer patient tumors, patient-derived xenografts, and cell models determined that PROX1 is upregulated early in the lineage plasticity continuum and progressively increases as tumors lose AR activity. We determined DNA methylation is a key regulator of PROX1 expression. PROX1 suppression in DNPC and NEPC reduces cell survival and impacts apoptosis and differentiation, demonstrating PROX1’s functional importance. PROX1 is not directly targetable with standard drug development approaches. However, affinity immunopurification demonstrated histone deacetylases (HDACs) are among the top PROX1-interacting proteins; HDAC inhibition depletes PROX1 and recapitulates PROX1 suppression in DNPC and NEPC. Altogether, our results suggest PROX1 promotes the emergence of lineage plasticity, and HDAC inhibition is a promising approach to treat tumors across the lineage plasticity continuum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b94f3206000c31e8479a7c87d320d045bdcd46f" target='_blank'>
              PROX1 is an early driver of lineage plasticity in prostate cancer
              </a>
            </td>
          <td>
            Zhi Duan, Mingchen Shi, A. Kumaraswamy, Dong Lin, Dhruv Khokhani, Yong Wang, Chao Zhang, Diana Flores, Eva Rodansky, Olivia A. Swaim, W. Storck, Hannah N. Beck, Radhika A Patel, Erolcan Sayar, B. P. Hanratty, Hui Xue, Xin Dong, Zoe R. Maylin, Rensheng Wan, David A Quigley, M. Sjöström, Ya-Mei Hu, Faming Zhao, Zheng Xia, Siyuan Cheng, Xiuping Yu, Felix Y. Feng, Li Zhang, Rahul Aggarwal, Eric J. Small, V. Ravikumar, A. Rao, Karan Bedi, John K. Lee, C. Morrissey, Ilsa M Coleman, Peter S Nelson, Eva Corey, Aaron M. Udager, Ryan J. Rebernick, M. Cieslik, A. Chinnaiyan, Joel A. Yates, Michael C. Haffner, Yuzhuo Wang, J. Alumkal
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cf265ca3e7fb2004f37001431683cd2d9283383" target='_blank'>
              The invadosome of stem-like cancer cells: metastasis machinery and targetable vulnerability.
              </a>
            </td>
          <td>
            Kelvin K Tsai
          </td>
          <td>2025-06-04</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Temozolomide (TMZ) is an important chemotherapeutic agent for glioma treatment. However, the emergence of drug resistance impedes its application. Traditional population-level studies are limited in elucidating resistance mechanisms. Advances in single-cell and spatial transcriptomics technologies provide feasible resolution for studying the cellular composition and dynamics of tumors. In this study, we investigated the heterogeneity of gliomas associated with TMZ resistance at the single-cell and spatial transcriptome levels to identify the resistance mechanisms and potential therapeutic strategies. Methods Single cell sequencing technology was utilized to identify the cellular clusters of gliomas. Drug perturbation analysis and cellular propensity analysis revealed key cluster responding to TMZ. Enrichment analysis was preformed to explore the function of clusters. Transcription factor activity analysis revealed key transcription factors contributing to tumor resistance. Spatial transcriptome data and bulk RNA-seq data validates the role of key transcription factors. Downstream targets of key transcription factors were predicted and validated using gene regulation assays. Drug sensitivity analyses were used to seek viable strategies to overcome drug resistance. Results Glioma cells from before and after temozolomide treatment samples were classified into six clusters: NPC-like cluster, OPC-like cluster, MES-like cluster, AC-like cluster, OC, and Neuron. NPC-like clusters exhibited strong stemness and DNA repair capacity. The activity of MAZ in NPC-like cluster was significantly enhanced after TMZ treatment. The proportion of MAZ( +)_NPC-like cluster was higher in TMZ treated samples. Patients with high proportion of MAZ( +)_NPC-like cluster had poorer survival. Upregulation of MAZ is able to enhance drug resistance in glioma cells, but this phenomenon disappeared when FoxM1 expression was further silenced. The combination of paclitaxel and Trametinib is a promising strategy to overcome resistance. Conclusions NPC-like cluster is prevalent in recurrent and drug-resistant gliomas. MAZ transcription factors are critical regulators that promote the development of drug resistance in NPC-like clusters by enhancing the capacity of DNA repair and stemness. Patients with high proportions of MAZ( +)_NPC-like clusters have poor TMZ sensitivity and prognosis. MAZ enhances stemness and drug resistance in glioma cells by upregulating FOXM1 expression. The combination of paclitaxel and paclitaxel is a promising therapeutic strategy for treating gliomas and overcoming drug resistance. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06706-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c31c53340b2d02203f651d970425a0fc3e3628" target='_blank'>
              Single-cell and spatial transcriptome analyses reveal MAZ(+) NPC-like clusters as key role contributing to glioma recurrence and drug resistance
              </a>
            </td>
          <td>
            Qiang Gu, Kan Wang, Tingming Lu, Yifei Xiao, Yaqing Wu, Hang Zhou, Kaijia Zhou
          </td>
          <td>2025-06-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41cd39016a4b2c876a2210fd9227cfa83893338" target='_blank'>
              Unraveling Tyrosine-Kinase Inhibitor Resistance in NSCLC Cells via Same-cell Measurement of RNA, Protein, and Morphological Responses
              </a>
            </td>
          <td>
            Vivian Dien, Connor Thompson, Tristin Rammel, Jaime Roquero, Marielle Krivit, Grace Yeo, Daniel Honigfort, Pazinah Bhadha, Colin Brown-Greeves, Mariam Dawood, Keerthana Elango, Hyukjin J. Kwon, Tyler Lopez, Alvin Lo, Meghna Manjunath, Pin Ren, Ben Song, Mitch Sundkamp, Ramreddy Tippana, Wilfred Van Ijcken, Jennifer Wong, Jane Yao, Da Zhou, Sinan Arslan, Matthew Kellinger, Amirali Kia, S. Kruglyak, Michael Previte, Molly He
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The NRF2 signaling pathway promotes tumor initiation, progression and resistance to chemotherapy, radiation therapy and immune checkpoint inhibitors. The mechanisms underlying the biology of NRF2-active tumors are varied, and include altered cellular metabolism, a reductive shift in redox state, and immunosuppression. Here we determined the molecular and phenotypic impact of NRF2 activation on two human non-small cell lung cancer (NSCLC) cell models. Inducible expression of NRF2E79Q, a common activating NRF2 mutation, in H358 lung adenocarcinoma (LUAD) cells altered cellular morphology and increased xenograft tumor growth in mice but not in 2D cell culture. In contrast, NRF2E79Q expression in H596 lung adeno-squamous cell carcinoma altered cellular morphology, increased neuroendocrine marker gene expression, but did not impact tumor growth in 2D or in xenografts. Gene expression profiling revealed shared and unique NRF2 transcriptional programs between these models, some of which were shared in primary lung tumors. Collectively, our findings reveal context-dependent effects of NRF2 activation on the growth and differentiation-state of two human NSCLC models, supporting a role for NRF2 activation in altering the differentiation of human NSCLC during tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d380f52033048aa4b6b43030a36a049f7941d3" target='_blank'>
              Activating NRF2E79Q mutation alters the differentiation of human non-small cell lung cancer
              </a>
            </td>
          <td>
            Samera Hamad, Hansa Joshi, T. Hess, Stuart Jefferys, Zena Saleh, Rani Sellers, G. Zhu, Travis Shrank, Rayvon Moore, David Corcoran, Jeremy M. Simon, Francis R. Spitz, David Shersher, M. Major, Bernard Weissman
          </td>
          <td>2025-05-15</td>
          <td>Research Square</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In isocitrate dehydrogenase wildtype glioblastoma (GBM), cellular heterogeneity across and within tumors may drive therapeutic resistance. Here we analyzed 121 primary and recurrent GBM samples from 59 patients using single-nucleus RNA sequencing and bulk tumor DNA sequencing to characterize GBM transcriptional heterogeneity. First, GBMs can be classified by their broad cellular composition, encompassing malignant and nonmalignant cell types. Second, in each cell type we describe the diversity of cellular states and their pathway activation, particularly an expanded set of malignant cell states, including glial progenitor cell-like, neuronal-like and cilia-like. Third, the remaining variation between GBMs highlights three baseline gene expression programs. These three layers of heterogeneity are interrelated and partially associated with specific genetic aberrations, thereby defining three stereotypic GBM ecosystems. This work provides an unparalleled view of the multilayered transcriptional architecture of GBM. How this architecture evolves during disease progression is addressed in the companion manuscript by Spitzer et al.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c45eaa99273988e47319564458ae28d00342e00" target='_blank'>
              The multilayered transcriptional architecture of glioblastoma ecosystems
              </a>
            </td>
          <td>
            M. Nomura, Avishay Spitzer, K. Johnson, L. Garofano, Djamel Nehar-Belaid, Noam Galili Darnell, Alissa C. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, Simona Migliozzi, L. G. Gonzalez Castro, Tamrin ChowdhFury, N. Robine, C. Reeves, Jong Bae Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, Simone P Niclou, Labeeba Nusrat, Sorcha Kellet, Sunit Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, Marc Sanson, S. Tanaka, Nobuhito Saito, D.M. Ashley, Stephen T Keir, Keith L. Ligon, Jason T Huse, W. Yung, A. Lasorella, R. Verhaak, A. Iavarone, M. Suvà, I. Tirosh
          </td>
          <td>2025-05-09</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aae76ee848421ca34df444a8b29937afca779e4" target='_blank'>
              MicroCT enables simultaneous longitudinal tracking of murine pancreatic cancer progression and cachexia
              </a>
            </td>
          <td>
            Katherine R. Pelz, Philip Jimenez, Colin J. Daniel, Samuel D. Newton, Melissa Cunningham, Rosalie C. Sears, P. J. Worth, Jonathan R. Brody, T. Zimmers
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fdc708d82395eddf5ef9f62e5ba5f1c540963b1" target='_blank'>
              Single-cell and bulk transcriptome analysis identifies B-cell subpopulations and associated cancer subtypes with distinct clinical and molecular characteristics.
              </a>
            </td>
          <td>
            Yin A He, Li Zhao, Yufen Zheng, Xiaosheng Wang
          </td>
          <td>2025-06-17</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Targeted therapy interventions using tyrosine kinase inhibitors (TKIs) provide encouraging treatment responses in patients with ALK-rearranged lung adenocarcinomas, yet resistance occurs almost inevitably. In addition to tumor cell-intrinsic resistance mechanisms, accumulating evidence suggests that cancer-associated fibroblasts (CAFs) within the tumor microenvironment contribute to therapy resistance. This study aimed to investigate CAF-driven molecular networks that shape the therapeutic susceptibility of ALK-driven lung adenocarcinoma cells. Methods Three-dimensional (3D) spheroid co-cultures comprising ALK-rearranged lung adenocarcinoma cells and CAFs were utilized to model the tumor microenvironment. Single-cell RNA sequencing was performed to uncover transcriptional differences between TKI-treated homotypic and heterotypic spheroids. Functional assays assessed the effects of CAF-conditioned medium and CAF-secreted factors on tumor cell survival, proliferation, lipid metabolism, and downstream AKT signaling. The therapeutic potential of targeting metabolic vulnerabilities was evaluated using pharmacological inhibition of lipid metabolism and by ferroptosis induction. Results CAFs significantly diminished the apoptotic response of lung tumor cells to ALK inhibitors while simultaneously enhancing their proliferative capacity. Single-cell RNA sequencing identified lipogenesis-associated genes as a key transcriptional difference between TKI-treated homotypic and heterotypic lung tumor spheroids. CAF-conditioned medium and the CAF-secreted factors HGF and NRG1 activated AKT signaling in 3D-cultured ALK-rearranged lung tumor cells, leading to increased de novo lipogenesis and suppression of lipid peroxidation. These metabolic adaptations were critical for promoting tumor cell survival and fostering therapy resistance. Notably, both dual inhibition of ALK and the lipid-regulatory factor SREBP-1, as well as co-treatment with ferroptosis inducers such as erastin or RSL3, effectively disrupted the CAF-driven metabolic-supportive niche and restored sensitivity of resistant lung tumor spheroids to ALK inhibition. Conclusions This study highlights a critical role for CAFs in mediating resistance to ALK-TKIs by reprogramming lipid metabolism in ALK-rearranged lung cancer cells. It suggests that targeting these metabolic vulnerabilities, particularly through inhibition of lipid metabolism or induction of ferroptosis, could provide a novel therapeutic approach to overcome resistance and improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40170-025-00400-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a02a6efa6497a2bcee01f9f0743c25e6ad43b8" target='_blank'>
              Cancer-associated fibroblasts promote drug resistance in ALK-driven lung adenocarcinoma cells by upregulating lipid biosynthesis
              </a>
            </td>
          <td>
            Ann-Kathrin Daum, Lisa Schlicker, M. Schneider, T. Muley, Ursula Klingmüller, Almut Schulze, Michael Thomas, P. Christopoulos, Holger Sültmann
          </td>
          <td>2025-06-16</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Clinically aggressive lactotroph pituitary neuroendocrine tumors (PitNET) are invasive tumors with an unusually rapid growth rate despite maximally tolerated doses of dopamine agonist (DA). We aimed to unravel the molecular heterogeneity of lactotroph PitNET and to identify biomarkers of aggressiveness and resistance to pharmacological treatment. A total of 13 patients harboring DA-resistant lactotroph PitNET were included in this study. Visium Spatial Transcriptomics (ST), whole transcriptome sequencing (WTS), and whole exome sequencing (WES) were performed in tumors from 4 of these patients; WTS and WES was carried out in 5; tumors from two patients underwent ST and WES and tumors from two other patients underwent only ST. Tumors were classified as null or partial responders according to their response to DA treatment. The eight PitNET analyzed by ST exhibited significant intratumoral heterogeneity, with clones showing alterations in PI3K/AKT and lipid metabolism pathways, particularly inositol phosphate, glycerophospholipid, and sphingolipid metabolism. The cell-cell communication analysis showed FGF-FGFR ligand receptor interaction whilst the transcription factors RXRA and CREM showed participation in both groups. A trajectory exploration was performed by including all PitNET together in a single analysis to determine whether there was a tendency or molecular pathway showing a differentiation pattern that would guide the transition from a partially responsive PitNET to a completely unresponsive one. We did not observe any such pattern. All of these findings were corroborated in the cohort of DA-resistant PitNETs in which only bulk WTS and WES were performed. The bulk WTS corroborated lipid metabolism and PI3K-AKT pathway alteration in PitNET, whereas the WES showed only SF3β1 and TP53 variants in one tumor each. Our work suggests that the PI3K/AKT pathway may constitute a molecular target at which to aim therapeutic strategies designed to treat aggressive and DA-resistant lactotroph PitNET. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02025-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9584b357aff2d73647fc85b3bbd6a84a02ea98" target='_blank'>
              Spatial transcriptomics reveal PI3K-AKT and metabolic alterations in aggressive, treatment-resistant lactotroph pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            F. Martinez-Mendoza, Sergio Andonegui-Elguera, E. Sosa-Eroza, Erick Gomez-Apo, Aurea Escobar-España, Carolina González-Torres, Javier Gaytán-Cervantes, Alam Palma-Guzman, Hugo Torres-Flores, Alberto Moscona-Nissan, S. Hinojosa-Alvarez, J. Hernández-Perez, R. Chavez-Santoscoy, G. Guinto, Gerardo Y. Guinto-Nishimura, B. López-Félix, Erick U. Zepeda-Fernandez, Erick M. Estrada-Estrada, V. Correa-Correa, P. A. González-Zavala, Marco A. Asenscio-Montiel, M. A. García-Vargas, Emmanuel Cantu-Chavez, Rocío Arreola-Rosales, Keiko Taniguchi-Ponciano, D. Marrero-Rodríguez, Moises Mercado
          </td>
          <td>2025-05-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Matrix metalloproteinase 1 plays a pivotal role in tumor biology and immune modulation through its enzymatic remodeling of the extracellular matrix, facilitating tumor progression. In this study, we utilized large-scale single-cell RNA sequencing and spatial transcriptomics to investigate MMP1 expression, its cellular localization, and its impact on tumor progression and immune modulation. Our findings reveal that MMP1 expression is elevated in various tumor types and is strongly correlated with metastatic potential. High MMP1 expression was associated with increased activity in epithelial–mesenchymal transition signaling and TNFα/NF-κB pathways, which are known to promote tumor progression. Furthermore, MMP1+ malignant cells exhibited significant interactions with immune cells, particularly macrophages and CD8+ T cells. MMP1 expression correlated with enhanced macrophage infiltration and impaired CD8+ T-cell function, contributing to an immunosuppressive tumor microenvironment. Notably, the CXCL16-CXCR6 and ANXA1-FPR3 signaling axes were identified as key mediators of these interactions. Inhibition of MMP1 in vitro demonstrated reduced cell invasion, stemness, and proliferation, while increasing reactive oxygen species levels and promoting apoptosis. Our findings position MMP1 as a key player in the “tumor-immune” vicious cycle and a promising therapeutic target to enhance anti-tumor responses and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d0c741845c10c07b47cb268ec3e101fb8935759" target='_blank'>
              Decoding the impact of MMP1+ malignant subsets on tumor-immune interactions: insights from single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Da-Ming Xu, Ling-Xiao Chen, Ting Xue, Xiao-Yu Zhuang, Li-Chao Wei, Hui Han, Miao Mo
          </td>
          <td>2025-05-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Addressing the growing need for organized data on tumor homing peptides (THPs), we present TumorHoPe2, a manually curated database offering extensive details on experimentally validated THPs. This represents a significant update to TumorHoPe, originally developed by our group in 2012. TumorHoPe2 now contains 1847 entries, representing 1297 unique tumor homing peptides, a substantial expansion from the 744 entries in its predecessor. For each peptide, the database provides critical information, including sequence, terminal or chemical modifications, corresponding cancer cell lines, and specific tumor types targeted. The database compiles data from two primary sources: phage display libraries, which are commonly used to identify peptide ligands targeting tumor-specific markers, and synthetic peptides, which are chemically modified to enhance properties such as stability, binding affinity, and specificity. Our dataset includes 594 chemically modified peptides, with 255 having N-terminal and 195 C-terminal modifications. These THPs have been validated against 172 cancer cell lines and demonstrate specificity for 37 distinct tumor types. To maximize utility for the research community, TumorHoPe2 is equipped with intuitive tools for data searching, filtering, and analysis, alongside a RESTful API for efficient programmatic access and integration into bioinformatics pipelines. It is freely available at https://webs.iiitd.edu.in/raghava/tumorhope2/">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4673f139ff89b98a8a9cf242e48538be8a2f0a9d" target='_blank'>
              TumorHoPe2: An updated database for Tumor Homing Peptides
              </a>
            </td>
          <td>
            Diksha Kashyap, Devanshi Gupta, Naman Kumar Mehta, G. Raghava
          </td>
          <td>2025-05-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="For over a century, scientists reported the disruption of normal nuclear shape and size in cancer. These changes have long been used as tools for diagnosis and staging of malignancies. However, to date, the mechanisms underlying these aberrant nuclear phenotypes and their biological significance remain poorly understood. Using a model of pancreatic ductal adenocarcinoma (PDAC), the major histological subtypes of pancreatic cancer, we found oncogenic mutant KRAS reduces nuclear size. Transcriptomic and protein expression analysis of mutant KRAS-expressing PDAC cells revealed differential levels of several nuclear envelope-associated genes. Further analysis demonstrated the nuclear lamina protein, Emerin (EMD), acted downstream of KRAS to mediate nuclear size reduction in PDAC. Analysis of human PDAC samples showed that increased EMD expression associates with reduced nuclear size. Finally, in vivo genetic depletion of EMD in a mutant KRAS-driven PDAC model resulted in an increased nuclear size and a reduced incidence of poorly differentiated PDAC. Thus, our data provides evidence of a novel mechanism underlying nuclear size regulation and its impact in PDAC carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fad3a87beb812ed2828574a6cdacbebbce33710" target='_blank'>
              Emerin is an effector of oncogenic KRAS-driven nuclear dynamics in pancreatic cancer.
              </a>
            </td>
          <td>
            Luis F Flores, David L Marks, Renzo E Vera, Ashley N Sigafoos, Ezequiel J Tolosa, Luciana L Almada, David R Pease, M. D. Toruner, Brian Chang, Brooke R Tader, Kayla C LaRue-Nolan, Ryan M. Carr, Rondell P Graham, Catherine E Hagen, Matthew R Brown, Aleksey V Matveyenko, Katherine L Wilson, David W Dawson, Christopher L. Pin, Kyle J Roux, Martin E Fernandez-Zapico
          </td>
          <td>2025-06-10</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Heart failure is a leading cause of morbidity and mortality; yet gene regulatory mechanisms driving cell type-specific pathologic responses remain undefined. Here, we present the cell type-resolved transcriptomes, chromatin accessibility, histone modifications and chromatin organization of 36 non-failing and failing human hearts profiled from 776,479 cells spanning all cardiac chambers. Integrative analyses revealed dynamic changes in cell type composition, gene regulatory programs and chromatin organization, which expanded the annotation of cardiac cis-regulatory sequences by ten-fold and mapped cell type-specific enhancer-gene interactions. Cardiomyocytes and fibroblasts particularly exhibited complex disease-associated cellular states, gene regulatory programs and global chromatin reorganization. Mapping genetic association data onto cell type-specific regulatory programs revealed likely causal genetic contributors to heart failure. Together, these findings provide comprehensive, multimodal gene regulatory maps of the human heart in health and disease, offering a valuable framework for designing precise cell type-targeted therapies for treating heart failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de43af11fee4de7b9c73d4ebfeedb9bf0f1db3e0" target='_blank'>
              Single cell multiomics and 3D genome architecture reveal novel pathways of human heart failure
              </a>
            </td>
          <td>
            Yang Xie, Luca Tucciarone, Elie N. Farah, Lei Chang, Qian Yang, T. Shankar, Weston S Elison, Shaina Tran, Jovina Djulamsah, Audrey Lie, Timothy Loe, Alyssa R Holman, Sierra Corban, Justin Buchanan, S. Mamde, Haowen Zhou, R. Elgamal, E. Tseliou, Vincent Huang, Zhaoning Wang, Jeffery Chiu, Rebecca L. Melton, Emily Griffin, Qingquan Zhang, J. Lucero, S. Navankasattusas, Daofeng Li, Chanrung Seng, Eugin Destici, C. Selzman, Agnieszka D’Antonio-Chronowska, Ting Wang, Allen Wang, Stavros G. Drakos, K. J. Gaulton, Bing Ren, Neil C Chi
          </td>
          <td>2025-05-09</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f88d93c45659d05d4c112ca60db918cd868e7fbf" target='_blank'>
              Signalomics for molecular tumor boards and precision oncology of breast and gynecological cancers.
              </a>
            </td>
          <td>
            Tatiana V Denisenko, Anna E. Ivanova, A. Koval, Denis N. Silachev, Lee Jia, G. Sukhikh, Vladimir L. Katanaev
          </td>
          <td>2025-06-09</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Super-enhancers (SEs) are large clusters of highly active enhancers that play key regulatory roles in cell identity, development, and disease. While conventional methods classify SEs in a binary fashion—super-enhancer or not—this threshold-based approach can overlook significant intermediate states of enhancer activity. Here, we present a dataset and accompanying framework that facilitate a more nuanced, non-binary examination of SE activation across mouse tissue types (mammary gland, lung tissue, and NMuMG cells) and various experimental conditions (normal, tumor, and drug-treated samples). By consolidating overlapping SE intervals and capturing continuous enhancer activity metrics (e.g., ChIP-seq signal intensities), our dataset reveals gradual transitions between moderate and high enhancer activity levels that are not captured by strictly binary classification. Additionally, the data include extensive functional annotations, linking SE loci to nearby genes and enabling immediate downstream analyses such as clustering and gene ontology enrichment. The flexible approach supports broader investigations of enhancer landscapes, offering a comprehensive platform for understanding how SE activation underpins disease mechanisms, therapeutic response, and developmental processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1f05b1d5f45c670b7d2814b318d5ad6d35ad996" target='_blank'>
              A Non-Binary Approach to Super-Enhancer Identification and Clustering: A Dataset for Tumor- and Treatment-Associated Dynamics in Mouse Tissues
              </a>
            </td>
          <td>
            Ekaterina D. Osintseva, G. A. Ashniev, Alexey V. Orlov, Petr I. Nikitin, Zoia G. Zaitseva, Vladimir V. Volkov, N. Orlova
          </td>
          <td>2025-05-14</td>
          <td>Data</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Small-cell lung cancer (SCLC) is the most lethal type of lung cancer, characterized by rapid evolution from chemosensitivity to chemoresistance and limited treatment options. However, the mechanisms underlying this evolution remain poorly understood. Here, we show that Retinoid X receptor γ (RXRγ) is uniquely overexpressed in chemo-resistant SCLC tumors, and that RXRγ serves as an essential factor driving chemoresistance in SCLC. RXRγ forms phase-separated droplets with LSD1 in the nucleus, which enhances RXRγ-mediated gene transcription activity and reprograms gene expression, promoting tumor stemness and metastasis, and eventually driving SCLC chemoresistance. In turn, RXRγ antagonist disrupts RXRγ-LSD1 interaction, reducing their binding to the target gene locus, markedly suppressing the expression of the RXRγ target gene network. Finally, RXRγ antagonists strongly suppress tumor growth and metastasis and restore SCLC vulnerability to chemotherapy in multiple preclinical SCLC models, resulting in a substantial extension of survival in mouse models. Thus, these results establish RXRγ as a key player in SCLC by phase separation and as a potential therapeutic target for this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc6976505863353b005498741bc1fbef13303de9" target='_blank'>
              Phase separation of RXRγ drives tumor chemoresistance and represents a therapeutic target for small-cell lung cancer.
              </a>
            </td>
          <td>
            Hong Wang, Jie Huang, Zhenhua Zhang, Yana An, Huizi Sun, Jiang-liang Chen, Weineng Feng, Hao Duan, Yonggao Mou, Yuanxiang Wang, Peiqing Liu, Huihao Zhou, Hong-Wu Chen, Jian Zhang, Xiaoyun Lu, Junjian Wang
          </td>
          <td>2025-05-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 e12602


 Background:
 Neoadjuvant dual HER2 blockade + chemotherapy (NAC) is standard of care for early-stage HER2+ breast cancer (BC), leading to pathological complete response (pCR) associated with high cure rates in ~50-60% of patients (pts). In contrast, non-pCR pts have significant relapse risk. Mechanisms of intrinsic resistance to HER2-targeted therapy remain poorly understood.
 Methods:
 We conducted a phase II trial of NAC with dual HER2 blockade (12 weeks trastuzumab + HER2 tyrosine kinase inhibitor varlitinib + paclitaxel), followed by surgery, in stage I-III HER2+ BC pts. Tumor biopsies were taken at baseline (BL) and at surgery post-NAC. We analyzed 24 paraffin-embedded tumor samples from 8 pCR (BL) and 8 non-pCR (BL, post-NAC) pts with spatial transcriptomics profiling (10X Genomics) to evaluate the multidimensional organization and contribution of cell components to treatment response. Pre-processing of the raw matrix was performed using Seurat (v5.0.3), with normalization conducted via the SCTransform function. Cell composition was deconvoluted using Cell2location (v0.1.3). Copy number alterations were inferred using SCEVAN (v1.0.1), while pathway activities were estimated with PROGENy (v3.20). Cell-to-cell and cell-to-pathway colocation analyses were carried out using mistyR (v1.10.0).
 Results:
 At BL, non-pCR pts showed significantly lower tumor HER2 expression, more heterogeneity in HER2 expression and ERBB2 gene amplification (amp), and greater dissociation between ERBB2 amp and HER2 expression vs pCR pts. Significant heterogeneity in ERBB2 amp persisted in non-pCR post-NAC tumors. Non-pCR pts showed significant suppression of immune-related pathways vs pCR pts at BL. Deconvolution analysis showed non-pCR pts to exhibit marked lack of immune cells, including T / NKT cells / plasma blasts at BL and post-NAC as well as reduced B cells post-NAC vs BL tumors of pCR pts. Spatial analysis revealed differences in colocalization of immune cells: BL tumors in pCR pts showed aggregation of memory B cells with other immune cells to form immune hotspots. In contrast, both BL and post-NAC tumors in non-pCR pts showed memory B cells to colocalize with cycling T and myeloid cells with less immune hotspots. Pathway analysis revealed significant differences in immune cell interactions with signaling pathways in pCR vs non-pCR pts. At BL, immune cells were associated with JAK-STAT and TNFα pathway activation in pCR pts, but were strongly correlated with TRAIL in non-pCR pts. Post-NAC, immune cells in non-pCR pts were significantly linked to JAK-STAT and TGFβ signaling.
 Conclusions:
 Spatial transcriptomics analyses reveal tumor and immune microenvironment heterogeneity between pCR and non-pCR pts to neoadjuvant HER2-targeted therapy. This heterogeneity may explain intrinsic resistance and could unravel potential therapeutic targets in resistant HER2+ breast cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60a55807eda1066e45530a8ab8c5c2eba46686f5" target='_blank'>
              Tumor spatial transcriptomics and potential mechanisms of intrinsic resistance to HER2-directed therapy.
              </a>
            </td>
          <td>
            Junlong Song, J. Lim, S. Ow, A. Wong, Robert Walsh, A. Datta, S. Lee
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Aberrant glycosylation is a hallmark of cancer, offering opportunities to enhance clinical decision-making and enable precise targeting of cancer cells. Nevertheless, alterations in the bladder urothelial carcinoma (BLCA) N-glycome remain poorly characterized. Here, we used in situ N-deglycosylation and mass spectrometry, revealing a marked enrichment of oligomannose-type N-glycans in non-invasive Ta tumors, which diminished with disease progression. A complementary analysis of The Cancer Genome Atlas (TCGA) transcriptomic data revealed downregulation of the key mannosidases in BLCA, suggesting a mechanistic basis for oligomannose accumulation, though this requires further validation. Then, targeted glycoproteomic profiling identified potential stage-specific carriers of oligomannoses. Exploratory functional annotation suggests stage-dependent differences among detected glycoproteins, ranging from metabolic regulation in Ta tumors to oxidative stress adaptation in muscle-invasive disease, highlighting glycosylation’s contribution to tumor progression. Furthermore, myeloperoxidase (MPO) was enriched in more aggressive stages. Spatial validation confirmed MPO overexpression in tumor-infiltrating immune cells and its correlation with oligomannose content. Importantly, high MPO expression combined with low mannosidase levels was linked to poor survival, suggesting biological relevance. This study suggests a dynamic, stage-specific N-glycome in BLCA and identifies oligomannose-bearing glycoproteins as exploratory leads for biomarker and therapeutic target discovery, providing a N-glycomic resource for further investigation towards glycan-based precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93cf2d157a31be9d833f4db699f5b4f62d9a15c1" target='_blank'>
              Glycomics and Glycoproteomics Reveal Distinct Oligomannose Carriers Across Bladder Cancer Stages
              </a>
            </td>
          <td>
            Marta Relvas-Santos, Dylan Ferreira, Andreia Brandão, Luís Pedro Afonso, Lúcio Lara Santos, André M. N. Silva, J. Ferreira
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6086eff81e816ca3205d5bed8a0f1a0d7841fa7" target='_blank'>
              Oncogene aberrations drive medulloblastoma progression, not initiation.
              </a>
            </td>
          <td>
            K. Okonechnikov, P. Joshi, Verena Körber, Anne Rademacher, Michele Bortolomeazzi, Jan-Philipp Mallm, Jan Vaillant, P. B. da Silva, Britta Statz, Mari Sepp, Ioannis Sarropoulos, Tetsuya Yamada, A. Wittmann, Kathrin Schramm, Mirjam Blattner-Johnson, Petra Fiesel, Barbara Jones, N. Jäger, T. Milde, K. Pajtler, Cornelis M. van Tilburg, Olaf Witt, Konrad Bochennek, Katharina Johanna Weber, Lisa Nonnenmacher, Christian Reimann, David R. Ghasemi, U. Schüller, M. Mynarek, S. Rutkowski, David T. W. Jones, A. Korshunov, K. Rippe, Frank Westermann, S. Thongjuea, Thomas Höfer, Henrick Kaessmann, Lena M. Kutscher, Stefan M. Pfister
          </td>
          <td>2025-05-07</td>
          <td>Nature</td>
          <td>1</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/503cca6deb2a91fe468e2c781a6fa64738c62b22" target='_blank'>
              Drug resistance and tumor heterogeneity: cells and ensembles
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-05-31</td>
          <td>Biophysical Reviews</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background Tumor resistance is the primary reason for treatment failure in patients with cancer, while oncolytic viruses (OVs), as a novel therapy, have been rapidly advancing through clinical evaluation and are typically assessed in recurrent tumors that are refractory to standard chemotherapy. However, whether the adaptive process that fosters chemotherapy resistance influences the efficacy of OV therapy is unknown. Methods We analyzed chemo-resistant colorectal cancer (CRC) using in vitro, in vivo, and patient-derived organoid models to assess sensitivity to OVs. Through RNA sequencing analysis and immunohistochemistry were performed in clinical samples that indicated TANK-binding kinase 1 (TBK1) expression. Using single-cell RNA sequencing, flow cytometry, and in vivo neutralization assays to demonstrate that the combination of TBK1 inhibitor (TBK1i) and OVs reprograms the tumor immune microenvironment, particularly by activating natural killer (NK) cells. Through RNA sequencing analysis, we identified intercellular cell adhesion molecule-1 (ICAM-1) as a potential target responsible for NK cell activation. Subsequently, we designed and conducted rescue experiments, both in vitro and in vivo, to validate the influence of ICAM-1 on NK cell activity. Results We demonstrated that chemo-resistant CRC showed decreased sensitivity to the OV in vitro, in vivo, and patient-derived organoids. Further investigation revealed aberrant activation of TBK1 in chemo-resistant CRC, which mediated the activation of the type I interferon pathway and impaired viral replication. TBK1 inhibition enhanced intratumor viral replication and direct oncolysis effect in vitro and augmented the antitumor immunity elicited by OVs in vivo. Immune cell profiles presented that OV/TBK1i combination reshaped the tumor microenvironment and especially activated the NK cell response. Immune cell depletion studies demonstrated that NK cells were required for the synergistic therapeutic activity of the OV/TBK1i combination. Mechanistically, TBK1 inhibition synergized with VSVΔ51 to increase ICAM1 expression in a RIPK1-dependent manner, promoting NK cell-mediated tumor killing. Conclusion This study presents a promising approach for treating chemo-resistant CRC by combining OVs and TBK1i.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad16bd40220efd0dd4a029d8c16e8f0ba36e528" target='_blank'>
              Targeting TBK1 potentiates oncolytic virotherapy via amplifying ICAM1-mediated NK cell immunity in chemo-resistant colorectal cancer
              </a>
            </td>
          <td>
            Xin Guo, Huolun Feng, Zhihui Xi, Ji Zhou, Zuda Huang, Jieqing Guo, Jiabin Zheng, Zejian Lyu, Yongfeng Liu, Jianlong Zhou, Yucheng Zhang, Yuhan Zhang, Yong Li, Fan Xing
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background The incidence and mortality of endometrial cancer (EC) is on the rise. Eighty-five percent of ECs depend on estrogen receptor alpha (ERα) for proliferation, but little is known about its transcriptional regulation in these tumors. Results We generate epigenomics, transcriptomics, and Hi-C datastreams in healthy and tumor endometrial tissues, identifying robust ERα reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with endometrial cancer risk single-nucleotide polymorphisms and whole-genome sequencing data from primary tumors and metastatic samples reveals a striking enrichment of risk variants and non-coding somatic mutations at tumor-enriched ERα sites. Through machine learning-based predictions and interaction proteomics analyses, we identify an enhancer mutation which alters 3D genome conformation, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. Conclusions In summary, we identify a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ERα. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03596-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79c6ec91a0f963e937bbd37603ce55880af9ede" target='_blank'>
              Endometrial tumorigenesis involves epigenetic plasticity demarcating non-coding somatic mutations and 3D-genome alterations
              </a>
            </td>
          <td>
            Sebastian Gregoricchio, Aleksandar Kojic, M. Hoogstraat, K. Schuurman, Suzan Stelloo, T. Severson, Tracy A. O’Mara, M. Droog, Abhishek A. Singh, Dylan M Glubb, L. Wessels, Michiel Vermeulen, Flora E. van Leeuwen, W. Zwart
          </td>
          <td>2025-05-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Neutrophils, accounting for 50–70% of circulating leukocytes, exhibit remarkable plasticity in tumor biology. Depending on tumor type and microenvironmental cues, they can exert either anti-tumor or pro-tumor effects. During tumor initiation, neutrophils exposed to chronic inflammation secrete cytokines and oncogenic microRNAs that promote genomic instability and malignant transformation. In tumor progression, neutrophils adopt context-dependent phenotypes and execute diverse functions, including polarization into anti-tumor (N1) or pro-tumor (N2) subsets; secretion of inflammatory and angiogenic mediators; formation of neutrophil extracellular traps (NETs); production of reactive oxygen and nitrogen species (e.g., H2O2 and nitric oxide); and modulation of immune cell infiltration and function within the tumor microenvironment. During metastasis, neutrophils facilitate cancer dissemination through three principal mechanisms: (1) promoting epithelial–mesenchymal transition (EMT) via inflammatory signaling, adhesion molecule interactions, and lipid metabolic support; (2) establishing pre-metastatic niches by remodeling distant organ stroma through NETs and matrix metalloproteinases; and (3) reactivating dormant tumor cells in response to chronic inflammation, viral infection, or stress hormones. Collectively, neutrophils function as central regulators across all stages of tumor evolution, influencing cancer growth, immune evasion, and metastatic progression. This review aims to provide a comprehensive synthesis of neutrophil-mediated mechanisms in the tumor microenvironment and highlight emerging strategies for neutrophil-targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6be2dce4f0c9adc087f698bd86c14d00e5f0275" target='_blank'>
              Neutrophil Spatiotemporal Regulatory Networks: Dual Roles in Tumor Growth Regulation and Metastasis
              </a>
            </td>
          <td>
            Pengcheng Li, Feimu Fan, Bixiang Zhang, Chaoyi Yuan, Huifang Liang
          </td>
          <td>2025-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4697eaf2cc8a0419c494eb4b94964848aca7f5d" target='_blank'>
              Nanopore Sequencing Unveils Somatic Structural Variations as Biomarkers in Laryngeal squamous cell carcinoma Genomes
              </a>
            </td>
          <td>
            Xuyan Liu, Lin Xia, Yixin Qiao, Yang Li, Yan Huang, Bingyan Yue, Xi Liang, Xin Yang, Honghui Zhang, Jiaxun Zhang, Xiao Chen, Dan Xie, Jifeng Liu
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Macrophages play a critical role in shaping the immune landscape of various diseases, with secreted phosphoprotein 1 (SPP1)-expressing macrophages emerging as a distinct subset implicated in both cancerous and non-cancerous conditions. Leveraging recent advances in single-cell RNA sequencing, numerous studies have identified SPP1+ macrophages across diverse pathological contexts, shedding light on their functional heterogeneity. In cancer, SPP1+ tumour-associated macrophages contribute to tumour growth, angiogenesis, and immune evasion, often interacting with T cells and stromal components to sustain an immunosuppressive microenvironment. Conversely, in non-cancerous diseases, these macrophages exhibit both profibrotic and disease-promoting properties, depending on the tissue context. This review provides a comprehensive synthesis of the latest findings on SPP1+ macrophages, highlighting their roles in tumour progression, immune suppression, tissue remodelling, and fibrosis. By comparing their shared traits and tissue-specific differences, we explore how SPP1+ macrophages adapt to distinct microenvironments and influence disease progression. Understanding their conserved and context-dependent functions may open new avenues for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2f870e69c0e0a9df1074bd00e100f8379d0713" target='_blank'>
              The Landscape of SPP1+ Macrophages Across Tissues and Diseases: A Comprehensive Review.
              </a>
            </td>
          <td>
            Alessandro Palma
          </td>
          <td>2025-05-21</td>
          <td>Immunology</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive brain tumor with limited treatment options and prognosis due to its highly invasive nature and therapy resistance. Investigating the nanomechanical and pathophysiological properties of GBM cells will shed light on tumor behavior. In our study, we investigated the mechanical properties, migration dynamics, and cytoskeletal organization of T98G and U87 MG glioblastoma cell lines using in vitro techniques: atomic force microscopy (AFM) and electric cell-substrate impedance sensing (ECIS). While U87 MG cells are Temozolomide (TMZ)-sensitive and exhibit increased susceptibility to cell death and growth inhibition, T98 cells exhibit improved survival and repair in response to TMZ therapy. This study found that T98G cells are rougher, stiffer, and more viscous, while U87 MG cells are smoother, more elastic, and less viscous, leading to distinct cellular migration patterns. Such differences indicate GBM cell heterogeneity and have consequences for tumor development and resistance to treatment. The key differences of the nanomechanical, viscoelastic, and migratory properties between T98G and U87 MG cells, demonstrated in this work will help us gauge the diverse effects of different dosages of radiation along with immunotherapeutic agents, identifying the ideal radioimmunotherapy option.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ba2cc0d72ec2de612a1d7d825b152f96e3640f" target='_blank'>
              Exploring the heterogeneity in glioblastoma cellular mechanics using in-vitro assays and atomic force microscopy
              </a>
            </td>
          <td>
            Nabila Masud, Md. Hasibul Hasan Hasib, Bayode Ibironke, Charlotte Block, Jayce Hughes, Andrew E. Ekpenyong, Anwesha Sarkar
          </td>
          <td>2025-06-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Spatial biology is revolutionising our understanding of cellular organisation and disease by preserving the interactions between cells within tissues. Traditional methods often disrupted these delicate structures, making it challenging to study cells in their natural environments. This article explores key technologies, such as immunohistochemistry (IHC) and in situ hybridisation, which enable precise molecular analysis while maintaining spatial context. IHC remains essential for identifying protein markers in pathology, while multiplex imaging systems significantly enhance biomarker detection and high-throughput spatial profiling. Additionally, new computational tools like MuSpAn are advancing our understanding of spatial cell organisation. Despite challenges posed by data complexity, spatial biology opens exciting new possibilities for precision medicine, facilitating targeted therapies and advancing personalised treatment strategies. As the field rapidly evolves, it continues to drive groundbreaking breakthroughs in disease research and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab2a3155c2afaaf1db4fa8380e5514a4df2b6086" target='_blank'>
              Holding space for spatial biology
              </a>
            </td>
          <td>
            EJ. Mulholland
          </td>
          <td>2025-05-07</td>
          <td>The Biochemist</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lineage plasticity is critical for tumor progression and therapy resistance, but the molecular mechanisms underlying cell identity shifts in cancer remain poorly understood. In lung adenocarcinoma (LUAD), the loss of pulmonary lineage fidelity and acquisition of alternate identity programs converge on hybrid identity (hybrid ID) states, which are postulated to be key intermediates in LUAD evolution and are characterized by the coactivation of developmentally incompatible identity programs within individual cells. Here, we uncover a previously unrecognized role for the gastrointestinal transcriptional regulator HNF4α in driving tumor growth and hybrid ID states in LUAD. In LUAD cells expressing the lung lineage specifier NKX2-1, HNF4α induces a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also disrupts NKX2-1 genomic localization and dampens pulmonary identity within hybrid ID LUAD. We show that this hybrid ID state is maintained by sustained RAS/MEK signaling. Inhibition of the RAS/MEK signaling cascade augments NKX2-1 chromatin binding at pulmonary-specific genes and induces drug resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion sensitizes LUAD cells to KRASG12D inhibition. Collectively, our data show that coexpression of opposing lineage specifiers is a novel mechanism of identity dysregulation in LUAD that influences both tumor progression and response to targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e47763b8688fbd0471d5f59c0d98d63dd2cf51e" target='_blank'>
              Opposing lineage specifiers induce a protumor hybrid identity state in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Kayla T. O'Toole, Rushmeen Tariq, Anna Waters, Katherine Gillis, Eric L. Snyder
          </td>
          <td>2025-06-12</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8625519b03e820eb6ff86341cc2c11cbf94fb11d" target='_blank'>
              Remodelling hypoxic TNBC microenvironment restores antitumor efficacy of Vγ9Vδ2 T cell therapy.
              </a>
            </td>
          <td>
            Yanyun Jing, Yangzhe Wu, Qinglin Hu, Wenfeng Wu, Dang Li, Wenhao Hu, Heming Li, Minggang Fu, Xin Huang, Yi Hu
          </td>
          <td>2025-05-28</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background The PCDH17 family has emerged as a prominent research focus in the field of oncology; however, the precise mechanism underlying the regulatory role of PCDH17 in shaping the inflammatory tumor microenvironment in pancreatic cancer remains elusive. Method A thorough examination was carried out to explore the presence of PCDH17 in cancerous tumor tissues and its association with genes related to inflammation. Additionally, we comprehensively examined the association between PCDH17 and immune cell infiltration as well as immunotherapy in pancreatic cancer. Moreover, the single-cell data of pancreatic cancer was utilized for analyzing PCDH17 expression, cell differentiation, and intercellular communication. Result PCDH17 exhibited differential expression across various tumor types. The low-expression group of PCDH17 showed reduced levels of inflammatory genes. Immunoinfiltration analysis indicated a significant association between T-cell infiltration and the expression of PCDH17. Analysis of single-cell sequencing data revealed that PCDH17 was primarily expressed in endothelial cells, with a decrease in expression observed during cellular differentiation trajectory. Notably, inflammation-related genes also displayed significant changes in their expression patterns along the endothelial cell differentiation trajectory. Cellular communication investigations unveiled multiple signaling pathways through which endothelial cells interacted with T cells. The presence of PCDH17 in endothelial cells was verified through various immunofluorescence techniques, and a simultaneous decline in its levels was observed alongside the decrease in inflammatory factors within the tumor microenvironment. Conclusion The predominant expression of PCDH17 was observed in endothelial cells, exhibiting a strong association with inflammatory genes and infiltration of immune cells. PCDH17 exhibits potential as a target for regulating the immune-suppressive tumor microenvironment in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06b73ddbf21e245d86c9b823def7478b1068908c" target='_blank'>
              Single-cell and transcriptomic analyses reveal the role of PCDH17 in the non-inflammatory tumor microenvironment of pancreatic cancer
              </a>
            </td>
          <td>
            Yong Sun, Hong Wan, Jie Xiong, Kun Cao, Dashuai Yang, Jun Huang
          </td>
          <td>2025-05-23</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>6</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [20],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>