<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-23 09:31:44 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c056a7bc97d38004639e645962dc9ff9c3cfad" target='_blank'>
              Modeling cellular influence delineates functionally relevant cellular neighborhoods in primary and metastatic pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yeonju Cho, , Sarah M. Shin, Alexei Hernandez, Xuan Yuan, Jowaly Schneider, Jody E. Hooper, Laura D. Wood, Elizabeth M. Jaffee, Atul Deshpande, W. Ho
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad839f8d8971ed34ba7a38eb79bda1b7721cc7ff" target='_blank'>
              Multiplexed Deep Visual Proteomics Unveils Spatial Heterogeneity and Rare Endocrine States in Human Adult Pancreatic Islets
              </a>
            </td>
          <td>
            M. Mardamshina, Nicolai Dorka, Marvin Thielert, Frida Björklund, Frederic Ballllosera Navarro, Anna Martinez Casals, Ferenc Kovács, Jocelyn E. Manning Fox, Peter Horvath, Patrick E. MacDonald, M. Mann, Emma Lundberg
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Tumor microenvironments (TMEs) are spatially complex and dynamic systems shaped by evolutionary pressures, tissue architecture, and cellular interactions. To capture this complexity, we developed the Tumor Landscape Analysis (TLA) pipeline, a computational framework applying principles from landscape ecology and spatial statistics to quantitatively characterize tumor spatial heterogeneity. TLA integrates spatially resolved pathology data, including whole-cell, point-based, and region-level formats, and computes multiscale metrics to assess cell distributions, neighborhood relationships, and tissue-level organization. The framework leverages ecological indices such as the Morisita-Horn index, Ripley’s H function, and Shannon diversity to quantify intercellular proximity, spatial clustering, and cellular diversity. It also uses the concept of local microenvironments (LMEs), data-driven ecological niches defined by local cell-type abundance and spatial uniformity, enabling unsupervised and reproducible classification of tumor regions. The use of fragmentation metrics, including patch density, shape complexity, and interspersion, provide further insight into spatial disorganization and emergent tissue architecture. TLA is agnostic to imaging modality and biological context, supporting broad applicability across tumor types and sample formats. By translating complex tissue architectures into interpretable spatial metrics, the pipeline enables integrative analyses that link spatial ecology to clinical and molecular phenotypes. This approach facilitates a deeper understanding of how spatial features contribute to tumor progression, therapeutic resistance, and clinical outcomes, offering new opportunities for precision oncology rooted in spatial systems biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/718b2bbad37dfa5c06ac139aad6c2d7117e031ab" target='_blank'>
              Tumor Landscape Analysis: An Ecologically Informed Framework to Understand Tumor Microenvironments
              </a>
            </td>
          <td>
            Luis Cisneros, M. D. Toruner, Martin E Fernandez-Zapico, Carlo C. Maley, Ryan M. Carr
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Over the past years, cancer research has transitioned from a ‘cancer cell-centered’ focus to a more integrative view of tumors as dynamic ecosystems. This paradigm shift emphasizes the tumor microenvironment (TME) as a complex network of interacting cellular and acellular components, where tumor cells orchestrate a supportive environment that facilitates progression, metastasis, and immune evasion. Understanding the spatial organization of these components within the TME is crucial, as the positioning and interactions between cancerous and non-cancerous cells significantly influence tumor behavior and therapy response. Spatial proteomics has emerged as a powerful tool for TME analysis, enabling the detection and quantification of proteins within intact tissue architecture at subcellular resolution. This approach provides insights into cellular interactions, signaling pathways, and functional states, facilitating the discovery of novel biomarkers and therapeutic targets linked to specific tissue regions and cellular contexts. Translating spatial proteomics into clinical practice requires overcoming challenges related to technology refinement, standardization of workflows, and adaptation to routine pathology settings. Melanoma is an aggressive, highly immunogenic malignancy with variable response rates to existing immunotherapies. Given that over half of patients treated with immune checkpoint inhibitors (ICIs) fail to respond or experience disease progression, the identification of novel biomarkers and therapeutic targets to enhance current therapies is urgently required. Spatial imaging technologies are increasingly being utilized to dissect the complex interplay between stroma, melanoma, and immune cell types within the TME to address this need. This review examines key spatial proteomics methods, their applications in melanoma biology, and associated image analysis pipelines. We highlight the current limitations, and future directions, emphasizing the potential for clinical translation to guide personalized treatment strategies, inform prognosis, and predict therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1eb1624eaa05c3beff7c464020ea5f0d816374c" target='_blank'>
              Spatial proteomics of the tumor microenvironment in melanoma: current insights and future directions
              </a>
            </td>
          <td>
            Chiara Bungaro, Michele Guida, Benedetta Apollonio
          </td>
          <td>2025-05-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae3a20489c547fc5b01d13afcfc1a7edcf2969d" target='_blank'>
              An image analysis pipeline to quantify the spatial distribution of cell markers in stroma-rich tumors
              </a>
            </td>
          <td>
            Antoine A. Ruzette, Nina Kozlova, Kayla A. Cruz, Taru Muranen, S. Nørrelykke
          </td>
          <td>2025-05-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="A comprehensive understanding of cellular behavior and response to the tumor microenvironment (TME) in colorectal cancer (CRC) remains elusive. Here, we introduce the high-definition Visium spatial transcriptomic technology (Visium HD) and investigate formalin-fixed paraffin-embedded human CRC samples (n = 5). We demonstrate the high sensitivity, single-cell-scale resolution and spatial accuracy of Visium HD, generating a highly refined whole-transcriptome spatial profile of CRC samples. We identify transcriptomically distinct macrophage subpopulations in different spatial niches with potential pro-tumor and anti-tumor functions via interactions with tumor and T cells. In situ gene expression analysis validates our findings and localizes a clonally expanded T cell population close to macrophages with anti-tumor features. Our study demonstrates the power of high-resolution spatial technologies to understand cellular interactions in the TME and paves the way for larger studies that will unravel mechanisms and biomarkers of CRC biology, improving diagnosis and disease management strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f42a49dad6477b432ac875de1bc87a28316b76b" target='_blank'>
              High-definition spatial transcriptomic profiling of immune cell populations in colorectal cancer
              </a>
            </td>
          <td>
            Michelli F Oliveira, J. P. Romero, Meii Chung, Stephen R. Williams, Andrew D. Gottscho, Anushka Gupta, Susan E. Pilipauskas, Seayar Mohabbat, Nandhini Raman, David Sukovich, David Patterson, Sarah E B Taylor
          </td>
          <td>2025-06-05</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b426c5f702b08645ac569d89b1da888e84c491" target='_blank'>
              Deciphering the cellular tumor microenvironment landscape in salivary gland carcinomas using multiplexed imaging mass cytometry
              </a>
            </td>
          <td>
            M. Mayer, L. Nachtsheim, L. Jansen, P. Wolber, M. Schmiel, A. Quaas, J. P. Klussmann, C. Arolt
          </td>
          <td>2025-05-13</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Identification of the consensus molecular subtypes (CMS) opened significant potential for understanding the tumor biology and intertumoral heterogeneity of colorectal cancer (CRC). However, molecular subtyping in CRC traditionally relies on bulk transcriptomics, therefore, lacks spatial and single-cell level aspect. Methods We constructed tissue microarrays using tumor cores from 222 CRC patients. Arrays were stained and imaged using 54 cell identity and cancer hallmark markers, delivering spatially resolved protein profiles of >2 million cells. RNA sequencing data and CMS classification were also available for these patients. After segmentation of cancer, stromal and immune cells, we investigated intratumoral heterogeneity within CMS subtypes using spatially resolved single-cell protein profiling (>2 million cells). We compared cell types, their spatial organization and their expression of cancer hallmark-related proteins in CMS 1-4 subtypes. Results We revealed tissue atlases illustrating the cell types/states, spatial heterogeneity, cellular neighborhoods, cellular network, and single-cell protein profiles of CMS tumors. CMS1 tumors had more CD3+, CD8+, and PD1+ immune cells that were found in the epithelial layer frequently. CMS1 was also associated with higher levels of metabolic reprogramming markers such as upregulated glycolysis. CMS2 showed immune segregation, reactive stroma patterns and higher levels of apoptotic and proliferative signaling proteins. CMS3 exhibited clustered cancer cells with high RIP3 levels, suggesting a pro-inflammatory microenvironment. CMS4 displayed stromal-centric and immune-evasive tumors characterized by decreased HLA-1 levels and regulatory T-cell exclusion from epithelium. Conclusion We present a spatial protein atlas of CRC at single-cell resolution and demonstrate novel aspects of CMS tumour structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd0aeac5c1de86e47bfdfaa33c8809067d3de4e" target='_blank'>
              High-Resolution Spatial Proteomics Characterises Colorectal Cancer Consensus Molecular Subtypes
              </a>
            </td>
          <td>
            Batuhan Kısakol, Anna Matveeva, Sanghee Cho, Mohammadreza Azimi, A. Lindner, M. Salvucci, E. McDonough, Joanna Fay, Tony O’Grady, N. McCawley, John P Burke, Deborah A. McNamara, John Graf, Simon McDade, Daniel B. Longley, Fiona Ginty, J. Prehn
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer cells display highly heterogeneous and plastic states in glioblastoma, an incurable brain tumour. However, how these malignant states arise and whether they follow defined cellular trajectories across tumours is poorly understood. Here, we generated a deep single cell and spatial multi-omic atlas of human glioblastoma that pairs transcriptomic, epigenomic and genomic profiling of 12 tumours across multiple regions. We identify that glioblastoma heterogeneity is driven by spatially-patterned transitions of cancer cells from developmental-like states towards those defined by a glial injury response and hypoxia. This cellular trajectory regionalises tumours into distinct tissue niches and manifests in a molecularly conserved manner across tumours as well as genetically distinct tumour subclones. Moreover, using a new deep learning framework to map cancer cell states jointly with clones in situ, we show that tumour subclones are finely spatially intermixed through glioblastoma tissue niches. Finally, we show that this cancer cell trajectory is intimately linked to myeloid heterogeneity and unfolds across regionalised myeloid signalling environments. Our findings define a stereotyped trajectory of cancer cells in glioblastoma and unify glioblastoma tumour heterogeneity into a tractable cellular and tissue framework.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a25f85ea327b0edecce4b687c0e133b5fa0f6d5" target='_blank'>
              A spatiotemporal cancer cell trajectory underlies glioblastoma heterogeneity
              </a>
            </td>
          <td>
            Grant de Jong, Fani Memi, Tannia Gracia, Olga Lazareva, Oliver Gould, Alexander Aivazidis, Manas Dave, Qianqian Zhang, Melanie Jensen, A. Rifaioglu, Joao D. Barros-Silva, Sabine Eckert, Di Zhou, Yvette Wood, Elizabeth Tuck, Sezgin Er, Henry Marshall, Kenny Roberts, Andrew L. Trinh, Shreya Rai, Tyler Shaw, Agnes Oszlanczi, Hayden Powell, R. Petryszak, Zoi Katsirea, Irfan Mamun, I. Mulas, Annelies Quaegebeur, Mayen Briggs, Stanislaw Makarchuk, Jessica Cox, Jimmy Tsz Hang Lee, Laura Rueda, Manu Saraswat, Harry J C J Bulstrode, Adam Young, Minal Patel, Tarryn Porter, E. Prigmore, Moritz Mall, Julio Saez-Rodriguez, James Briscoe, David H. Rowitch, Richard Mair, Sam Behjati, O. Stegle, O. Bayraktar
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL) therapy, recently approved by the FDA for melanoma, is an emerging modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors such as glioblastoma remains limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas using spectral flow cytometry, TCR sequencing, single-cell RNA-seq, Xenium in situ transcriptomics, and CODEX spatial proteomics. Comparative analysis of TIL-generating (TIL+) versus non-generating (TIL−) tumors revealed that IL7Rexpression, structured perivascular immune clustering, and tumor-intrinsic metabolic programs such as ACSS3 were associated with successful TIL expansion. In contrast, TIL−tumors were enriched for neuronal lineage signatures, immunosuppressive transcripts including TOX and FERMT1, and tumor-connected macrophages. This study defines spatial and molecular correlates of TIL manufacturing success and establishes a genomics-enabled selection platform for adoptive T cell therapy. The profiling approach is now being prospectively implemented in the GIANT clinical trial (NCT06816927), supporting its translational relevance and scalability across glioblastoma and other immune-excluded cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7799f3cb909d60657cf83c09eaecb6b9bad79c3" target='_blank'>
              A Spatial Multi-Omic Framework Identifies Gliomas Permissive to TIL Expansion
              </a>
            </td>
          <td>
            Mustafa Khasraw, K. Hotchkiss, Kenan Zhang, Anna M Corcoran, Elizabeth Owens, P. Noldner, C. Railton, Kyra Van Batavia, Ying Zhou, Jodie Jepson, Kirit Singh, R. McLendon, Kristen Batich, Anoop P Patel, Katayoun Ayasoufi, Michael Brown, Evan Calabrese, Jichun Xie, Jose R Conejo-Garcia, B. Shaz, John Hickey
          </td>
          <td>2025-04-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4028f5dcc0e86189184980c13bb56279010a29ab" target='_blank'>
              Clonal lineage tracing and parallel multiomics profiling reveal transcriptional diversification induced by ARID1A deficiency
              </a>
            </td>
          <td>
            Kenichi Miyata, Liying Yang, Yuxiao Yang, Kohei Kumegawa, R. Maruyama
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d556d25908485193c4e548584bab10bfa70daa1" target='_blank'>
              Oncogenic drivers shape the tumor microenvironment in human gliomas
              </a>
            </td>
          <td>
            Bo Zhao, Chae Yun Cho, Lingqun Ye, Tula Keal, Thomas Mitchell, Inés Martín-Barrio, C. A. Faraj, Mesut Unal, Jessica Alsing, Nora M Lawson, Ellie Rahm Kim, Francisco Chavez, Kristy Mendoza Rangel, Sanjay K Singh, Joseph H. McCarthy, D.M. Ashley, W. Yung, Vinay K. Puduvalli, Guy Nir, Jeffrey S. Weinberg, Linghua Wang, Keith L. Ligon, G. Genovese, Peter Van Loo, P. Futreal, Jason T. Huse, Jesse R. Dixon, Frederick F. Lang, Kadir C. Akdemir
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background Breast cancer is a highly heterogeneous disease where variations of biomarker expression may exist between individual foci of a cancer (intra-tumoral heterogeneity). The extent of variation of biomarker expression in the cancer cells, distribution of cell types in the local tumor microenvironment and their spatial arrangement could impact on diagnosis, treatment planning and subsequent response to treatment. Methods Using quantitative multiplex immunofluorescence (MxIF) imaging, we assessed the level of variations in biomarker expression levels among individual cells, density of cell cluster groups and spatial arrangement of immune subsets from regions sampled from 38 multi-focal breast cancers that were processed using whole-mount histopathology techniques. Molecular profiling was conducted to determine the intrinsic molecular subtype of each analysed region. Results A subset of cancers (34.2%) showed intra-tumoral regions with more than one molecular subtype classification. High levels of intra-tumoral variations in biomarker expression levels were observed in the majority of cancers studied, particularly in Luminal A cancers. HER2 expression quantified with MxIF did not correlate well with HER2 gene expression, nor with clinical HER2 scores. Unsupervised clustering revealed the presence of various cell clusters with unique IHC4 protein co-expression patterns and the composition of these clusters were mostly similar among intra-tumoral regions. MxIF with immune markers and image patch analysis classified immune niche phenotypes and the prevalence of each phenotype in breast cancer subtypes was illustrated. Conclusions Our work illustrates the extent of spatial heterogeneity in biomarker expression and immune phenotypes, and highlights the importance of a comprehensive spatial assessment of the disease for prognosis and treatment planning. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02038-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185f08263c4f5dcd967322d5656a07ab8cc0083f" target='_blank'>
              Intra-tumoral spatial heterogeneity in breast cancer quantified using high-dimensional protein multiplexing and single cell phenotyping
              </a>
            </td>
          <td>
            Alison M. Cheung, Dan Wang, M. Quintayo, Yulia Yerofeyeva, Melanie Spears, John M. S. Bartlett, Lincoln Stein, Jane Bayani, Martin J Yaffe
          </td>
          <td>2025-05-21</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="IDH-wildtype glioblastoma (GBM) is an aggressive brain tumor with poor survival and few therapeutic options. Transcriptionally-defined cell states coexist in GBM and occupy defined regions of the tumor. Evidence indicates that GBM cell states are plastic, but it remains unclear if they are determined by the underlying epigenetic state and/or by microenvironmental factors. Here, we present spatially-resolved epigenomic profiling of human GBM tissues that implicate chromatin structure as a key enabler of cell plasticity. We report two epigenetically-defined and spatially-nested tumor niches. Each niche activates short-range molecular signals to maintain its own state and, surprisingly, long-range signals to reinforce the state of the neighboring niche. The position of a cell along this gradient-like system of opposing signals determines its likelihood to be in one state or the other. Our results reveal an intrinsic system for cell plasticity that is encoded in the chromatin profiles of two adjacent niches that dot the topological architecture of GBM in cartesian space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc7a8f38a5d53281f1873999c12438505ab4d573" target='_blank'>
              Spatial epigenomic niches underlie glioblastoma cell state plasticity
              </a>
            </td>
          <td>
            Sam Kint, S. T. Younes, Shuozhen Bao, Gretchen Long, David Wouters, Erin Stephenson, Xing Lou, Mei Zhong, Di Zhang, Graham Su, Archibald Enninful, Mingyu Yang, Huey-Miin Chen, K. Ellestad, Colleen Anderson, Jennifer Moliterno, John J. Kelly, Jennifer A. Chan, Alejandro Sifrim, Jiangbing Zhou, Ana Nikolic, Rong Fan, Marco Gallo
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In breast cancer related to the BRCA1 mutation, luminal progenitor cells are believed to be the cells of origin, yet how these cells transform into invasive cancer cells remain poorly understood. Here, we combine single-cell epigenomic and transcriptomic data to reconstitute sequences of events in luminal cells that lead to tumorigenesis. Upon deletion of Trp53 and Brca1, we find that luminal progenitors display an extensive epigenomic disorder associated with a loss of cell identity. These cells then progress to tumor formation through a partial epithelial-to-mesenchymal transition, orchestrated by Snail and the timely activation of immunosuppressive and FGF signaling with their microenvironment. In human samples, pre-tumoral changes can be detected in early stage, basal-like tumors, which rarely recur, as well as in normal-like mammary glands of BRCA1 mutation carriers who have had cancer. Our study fills critical gaps in our understanding of BRCA1-driven tumorigenesis, opening perspectives for the early monitoring of individuals with high cancer risk. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02331-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f69d9b899da12be47c2084b992747c9f0654fa" target='_blank'>
              Epigenomic disorder and partial EMT impair luminal progenitor integrity in Brca1-associated breast tumorigenesis
              </a>
            </td>
          <td>
            Camille Landragin, Mélissa Saichi, Marthe Laisné, Adeline Durand, Pacôme Prompsy, Renaud Leclère, Jérémy Mesple, Kyra Borgman, Amandine Trouchet, Marisa M Faraldo, A. Chiche, A. Vincent-Salomon, Hélène Salmon, Céline Vallot
          </td>
          <td>2025-04-28</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61a2da83ec8f696482ebba50d8d961251109e469" target='_blank'>
              MONTAGE: A Computation Framework to Identify Spatially Resolved Functional Enrichment Gradients in the Tissue Microenvironment via Spatial Communities
              </a>
            </td>
          <td>
            Weiruo Zhang, Zinaida Good, M. Baertsch, Guolan Lu, John W. Hickey, Rachel Hildebrand, Serena Chang, Andrew J. Gentles, John B. Sunwoo, Quynh-Thu Le, Christina S. Kong, Garry P. Nolan, S. Plevritis
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Drug delivery to solid tumors is challenged by multiple physiological barriers arising from the tumor microenvironment, including dense extracellular matrix, cellular heterogeneity, hypoxic gradients, and elevated interstitial fluid pressure. These features hinder the uniform distribution and accumulation of therapeutics, reducing treatment efficacy. Despite their widespread use, conventional two-dimensional monolayer cultures fail to reproduce these complexities, contributing to the poor translational predictability of many preclinical candidates. Three-dimensional multicellular tumor spheroids have emerged as more representative in vitro models that capture essential features of tumor architecture, stromal interactions, and microenvironmental resistance mechanisms. Spheroids exhibit spatially organized regions of proliferation, quiescence, and hypoxia, and can incorporate non-tumor cells to mimic tumor–stroma crosstalk. Advances in spheroid analysis now enable detailed evaluation of drug penetration, cellular migration, cytotoxic response, and molecular gradients using techniques such as optical and confocal imaging, large-particle flow cytometry, biochemical viability assays, and microfluidic integration. By combining physiological relevance with analytical accessibility, spheroid models support mechanistic studies of drug transport and efficacy under tumor-like conditions. Their adoption into routine preclinical workflows has the potential to improve translational accuracy while reducing reliance on animal models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e55ff70454b1bfef937caf2e04dc247b0607a96" target='_blank'>
              Modeling Tumor Microenvironment Complexity In Vitro: Spheroids as Physiologically Relevant Tumor Models and Strategies for Their Analysis
              </a>
            </td>
          <td>
            Shrey Shah, Gerard G M D'Souza
          </td>
          <td>2025-05-01</td>
          <td>Cells</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1dc109a3a473a99af4a77b675644d925cf4104" target='_blank'>
              Oncogenic and tumor-suppressive forces converge on a progenitor-orchestrated niche to shape early tumorigenesis
              </a>
            </td>
          <td>
            José Reyes, Isabella S. Del Priore, Andrea C. Chaikovsky, Nikhita Pasnuri, Ahmed M. Elhossiny, Tobias Krause, Andrew Moorman, Catherine Snopkowski, Meril Takizawa, Cassandra Burdziak, Nalin Ratnayeke, Ignas Masillioni, Yu-jui Ho, R. Chaligné, P. Romesser, A. Filliol, T. Nawy, John P. Morris, Zhen Zhao, M. P. di Magliano, Direna Alonso-Curbelo, D. Pe’er, Scott W. Lowe
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a484735c9e4b45e9e9fb696fcc84a51c8d5f9b9a" target='_blank'>
              From Transcripts to Cells: Dissecting Sensitivity, Signal Contamination, and Specificity in Xenium Spatial Transcriptomics
              </a>
            </td>
          <td>
            Mariia Bilous, Daria Buszta, Jonathan Bac, Senbai Kang, Yixing Dong, S. Tissot, Sylvie André, Marina Alexandre-Gaveta, Christel Voize, Solange Peters, K. Homicsko, R. Gottardo
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cells are the fundamental structural and functional units of living organisms and the study of these entities has remained a central focus throughout the history of biological sciences. Traditional cell research techniques, including fluorescent protein tagging and microscopy, have provided preliminary insights into the lineage history and clonal relationships between progenitor and descendant cells. However, these techniques exhibit inherent limitations in tracking the full developmental trajectory of cells and elucidating their heterogeneity, including sensitivity, stability and barcode drift. In developmental biology, nucleic acid barcode technology has introduced an innovative approach to cell lineage tracing. By assigning unique barcodes to individual cells, researchers can accurately identify and trace the origin and differentiation pathways of cells at various developmental stages, thereby illuminating the dynamic processes underlying tissue development and organogenesis. In cancer research, nucleic acid barcoding has played a pivotal role in analyzing the clonal architecture of tumor cells, exploring their heterogeneity and resistance mechanisms and enhancing our understanding of cancer evolution and inter-clonal interactions. Furthermore, nucleic acid barcodes play a crucial role in stem cell research, enabling the tracking of stem cells from diverse origins and their derived progeny. This has offered novel perspectives on the mechanisms of stem cell self-renewal and differentiation. The present review presented a comprehensive examination of the principles, applications and challenges associated with nucleic acid barcode technology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d5cf40687295751922bb2f7725d94b4cd317977" target='_blank'>
              Barcodes based on nucleic acid sequences: Applications and challenges (Review)
              </a>
            </td>
          <td>
            Ying Hong Wei, Faquan Lin
          </td>
          <td>2025-05-02</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9da57971dc8950b12d4ca5ccbc0969a63be6c27" target='_blank'>
              Comparison of QuPath and HALO platforms for analysis of the tumor microenvironment in prostate cancer
              </a>
            </td>
          <td>
            Wei Zhang, Qin Zhou, Jonathan V. Nguyen, Erika Egal, Qian Yang, Michael R. Freeman, S. Hu-Lieskovan, Gita Suneja, Anna Coghill, Beatrice S Knudsen
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abfdae0d9888d7167ae3464de1089da14e882abe" target='_blank'>
              Multi-cellular phenotypic dynamics during the progression of breast tumors
              </a>
            </td>
          <td>
            Louise Gsell, Spencer S. Watson, Iva Sutevski, Matteo Massara, Klara Soukup, Alper Eroglu, Jeff E Mold, Antony Cougnoux, Johanna A. Joyce, Jean Hausser
          </td>
          <td>2025-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b492f429da8efbdf244cdd27a285c03ffb8c4208" target='_blank'>
              MARQO pipeline resolves multiparametric cellular and spatial organization in cancer tissue lesions
              </a>
            </td>
          <td>
            Mark Buckup, Igor Figueiredo, G. Ioannou, Sinem Ozbey, Rafael Cabal, Alexandra Tabachnikova, Leanna Troncoso, Jessica Le Berichel, Zhen Zhao, Stephen C. Ward, C. Hennequin, Guray Akturk, Steve Hamel, M. Fiel, R. Brody, M. Schwartz, T. Marron, S. Kim-Schulze, Edgar Gonzalez-Kozlova, V. Roudko, Pauline Hamon, Miram Merad, Sacha Gnjatic
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3867ee374678a732e8d29ff1143462f5a27ee092" target='_blank'>
              Towards interpretable molecular and spatial analysis of the tumor microenvironment from digital histopathology images with HistoTME-v2
              </a>
            </td>
          <td>
            Sushant Patkar, Timothy R. Rosean, Palak Patel, Stephanie A. Harmon, Peter L. Choyke, Tamara Jamaspishvili, B. Turkbey
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Cancer‐associated fibroblasts (CAFs) are a heterogeneous population of stromal cells, which modulate the immune system and can have both pro‐ and anti‐tumorigenic effects. In classic Hodgkin lymphoma (cHL), the role of CAFs has remained largely undefined. We applied multiplexed immunofluorescence imaging and spatial analysis on tumor samples from two independent cHL patient cohorts (n = 131 and n = 148) to study CAFs and their interactions with Hodgkin Reed–Sternberg (HRS) and tumor microenvironment (TME) cells at the single‐cell resolution. We show that higher proportions of CAFs are associated with favorable outcomes, independent of the clinical covariables. In contrast, a subset of CD45+ immune cells with strong fibroblast‐activation protein positivity, classified as macrophages, was less abundant in nodular sclerosis subtype and associated with worse outcomes. Neighborhood analysis allowed for the identification of colocalization or regional exclusion of phenotypically defined cell types and recurrent cellular neighborhoods. Despite the positive impact of CAF proportions on survival, patients with enrichment of platelet‐derived growth factor receptor beta (PDGFRB)‐positive CAFs in the vicinity of HRS cells had worse survival in both cohorts, independent of the clinical determinants. Our findings distinguish various subsets of CAFs and macrophages impacting survival in cHL and underscore the importance of the spatial arrangements in the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbaf0281c74f211aa49d27cd219fa5ad14a4a2cf" target='_blank'>
              Characterization of cancer‐associated fibroblasts and their spatial architecture reveals heterogeneity and survival associations in classic Hodgkin lymphoma
              </a>
            </td>
          <td>
            S. Leivonen, Kristiina Karihtala, T. Pellinen, M. Karjalainen-Lindsberg, Tomohiro Aoki, Christian Steidl, S. Leppä
          </td>
          <td>2025-05-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd09b27d0d8fe6fa80c372bc8924f459f7a4d33" target='_blank'>
              MOSAIC: Intra-tumoral heterogeneity characterization through large-scale spatial and cell-resolved multi-omics profiling
              </a>
            </td>
          <td>
            Caroline Hoffmann
          </td>
          <td>2025-05-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178f28f36aaf5c0de57a93091858bdbe9ce93e72" target='_blank'>
              Tumor Microenvironment in Ovarian Cancer through Spatial Transcriptomics and Identification of Key Gene Expression Profiles
              </a>
            </td>
          <td>
            K. K. Kırboğa, E. Küçüksille, M. Rudrapal, E. Aktaş, R. Achar, Gouri Deshpande, V. Stupin, E. Silina
          </td>
          <td>2025-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Merkel cell carcinoma (MCC) is an aggressive skin cancer with high propensity for metastasis, caused by Merkel-cell-polyomavirus (MCPyV), or chronic UV-light-exposure. How MCPyV spatially modulates immune responses within the tumor microenvironment and how such are linked to patient outcomes remains unknown. We interrogated the cellular and transcriptional landscapes of 60 MCC-patients using a combination of multiplex proteomics, in-situ RNA-hybridization, and spatially oriented transcriptomics. We identified a spatial co-enrichment of activated CD8+ T-cells and CXCL9+PD-L1+ macrophages at the invasive front of virus-positive MCC. This spatial immune response pattern was conserved in another virus-positive tumor, HPV+ head-and-neck cancer. Importantly, we show that virus-negativity correlated with high risk of metastasis through low CD8+ T-cell infiltration and the enrichment of cancer-associated-fibroblasts at the tumor boundary. By contrast, responses to immune-checkpoint blockade (ICB) were independent of viral-status but correlated with the presence of a B-cell-enriched spatial contexts. Our work is the first to reveal distinct immune-response patterns between virus-positive and virus-negative MCC and their impact on metastasis and ICB-response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28305c5ea574406c24e96fc8bdd4ca5602d07afe" target='_blank'>
              Spatially organized inflammatory myeloid-CD8 + T cell aggregates linked to Merkel-cell Polyomavirus driven Reorganization of the Tumor Microenvironment
              </a>
            </td>
          <td>
            Maximilian Haist, M. Matusiak, Yuqi Tan, S. Zimmer, H. Stege, Tim N. Kempchen, S. Mitschke, Pauline Chu, B. Weidenthaler-Barth, Graham L. Barlow, F. Rogall, Antonio Delgado Gonzalez, M. Baertsch, Silvia Albertini, M. Gariglio, C. Borgogna, Y. Goltsev, S. Grabbe, John W. Hickey, Garry P. Nolan
          </td>
          <td>2025-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d980820189344192705dd2c34f2a4d48d18931d" target='_blank'>
              An atlas of gene programs in non-small cell lung cancer delineates novel patterns of lineage vacillation, plasticity, and aggressiveness
              </a>
            </td>
          <td>
            Rui Hong, Pascal Klöckner, Sarah A. Mazzilli, Avrum E. Spira, J. Beane, Masanao Yajima, Joshua D. Campbell
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="MOTIVATION
Understanding the spatial architecture of tissues is essential for decoding the complex interactions within cellular ecosystems and their implications for disease pathology and clinical outcomes. Recent advances in multiplex imaging technologies have enabled high-resolution profiling of cellular phenotypes and their spatial distributions, revealing critical roles of tissue structures such as tertiary lymphoid structures (TLSs) in shaping immune responses and influencing disease progression. However, existing methods for analyzing spatial tissue structures often rely on hard clustering or adjacency-based spatial models, which are limited in capturing the nuanced and overlapping nature of cellular communities. To address these challenges, we develop a novel spatial topic modeling framework for the unsupervised discovery of spatial tissue structures in multiplex imaging data.


RESULTS
We propose TopSpace, a novel Bayesian spatial topic model that integrates Gaussian processes into latent Dirichlet allocation to flexibly model spatial dependencies in tissue microenvironments. By leveraging the Bayesian framework, TopSpace supports multicellular mixed-membership clustering and offers key inferential advantages, including robust uncertainty quantification and data-driven determination of the number of multicellular microenvironments. We demonstrate the utility of TopSpace through simulations and a case study on non-small cell lung cancer (NSCLC) data. Simulations show that TopSpace accurately recovers latent tissue microenvironments and spatial clustering patterns, outperforming existing methods in scenarios with varying spatial dependencies. Applied to NSCLC data, TopSpace successfully identifies TLS and captures their spatial probability distribution, which strongly correlates with patient survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2919fe0963b495439ad698a4bbdab1480229fe8d" target='_blank'>
              TopSpace: spatial topic modeling for unsupervised discovery of multicellular spatial tissue structures in multiplex imaging.
              </a>
            </td>
          <td>
            Junsouk Choi, Jian Kang, Veerabhadran Baladandayuthapani
          </td>
          <td>2025-04-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 e20040


 Background:
 Immunotherapy has revolutionized cancer treatment, particularly in non-small cell lung cancer (NSCLC), yet a large fraction of patients remain refractory to treatment. Understanding treatment response requires characterization of immune cell populations and their spatial organization, as cellular neighborhoods critically influence immune cell recruitment, activation, and function. Current imaging-based and NGS-based methods struggle to achieve clear separation of individual cells in densely packed regions. Hence, we developed STAMP-seq, a novel method tagging single nuclei with cleavable spatial barcodes using high-density DNA sequencing chips, to uncover immunotherapy response-related modules and dynamic changes in NSCLC.
 Methods:
 STAMP-seq employs spatially-encoded DNA arrays to label individual nuclei with cleavable barcodes. Following validation in mouse brain tissue, we analyzed surgically resected NSCLC samples from immunotherapy-treated patients, integrating spatial transcriptomics with BCR profiling to characterize treatment response features.
 Results:
 STAMP-seq achieves precise spatial barcoding, detecting over 30,000 cells per adult mouse brain section with reliable cell subtype assignment. In immunotherapy-treated NSCLC samples, spatial analysis revealed distinct cellular communities associated with treatment response. Responding patients showed enrichment of tertiary lymphoid structures and specialized plasma cell niches, with tumor cells exhibiting elevated TNF-α signaling and reduced proliferation. These tumor cells showed proximity to

 ROBO2

 + CAFs while remaining distant from

 ACTA2

 + CAFs. BCR clonotype and pseudotime analysis revealed that

 IGHG1

 + plasma cells are recruited from TLS or vasculature by stromal-rich "transit stops" containing antigen-presenting capabilities CAFs through

 HMGB1

 -

 CXCR4

 interaction.
 Conclusions:
 STAMP-seq enables high-resolution spatial transcriptomics at single-nucleus level. Our characterization of NSCLC spatial architecture revealed previously unrecognized immune response organization, particularly how specialized stromal-rich "transit stops" coordinate cancer-suppressing plasma cell recruitment. These findings demonstrate how spatial information can advance diagnostic and prognostic module development for human diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/023df8c02dd2a154737c443a9b87fee23263bac2" target='_blank'>
              Use of STAMP-seq to enable spatial transcriptomics through single-nucleus in situ tagging and exploration of relationship of dynamic plasma cell modules and immunotherapy response in non-small cell lung cancer.
              </a>
            </td>
          <td>
            Yitong Pan, Huan Yan, Jinhuan Han, Rui Wu, Xingyong Ma, Ying Guan, Keyu Li, Qingquan Wei, Guangxin Zhang, Shaozhuo Jiao, Yongchang Zhang, Chenxi Tian
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/432419f2032b46caaaf3eb1aa445ee0f4fa38314" target='_blank'>
              Clonal tracing with somatic epimutations reveals dynamics of blood ageing.
              </a>
            </td>
          <td>
            Michael Scherer, Indranil Singh, M. Braun, Chelsea Szu-Tu, Pedro Sanchez Sanchez, Dominik Lindenhofer, N. A. Jakobsen, Verena Körber, Michael Kardorff, Lena Nitsch, Pauline Kautz, Julia Rühle, Agostina Bianchi, Luca Cozzuto, R. Frömel, Sergi Beneyto-Calabuig, Caleb A Lareau, Ansuman T. Satpathy, Renée Beekman, Lars M Steinmetz, S. Raffel, Leif S. Ludwig, Paresh Vyas, Alejo E. Rodriguez-Fraticelli, Lars Velten
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5b1b0582e2b72f3132558990be0699903a9036" target='_blank'>
              CLONEID: A Framework for Monitoring and Steering Subclonal Dynamics
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Vural Tagal, Tao Li, Saeed S. Alahmari, Jackson Cole, Daniel Hannaby, Xiaoqing Yu, Konstantin Maksin, Andrew Schultz, Hojoon Lee, Issam ElNaqa, S. Eschrich, Janine M Lupo, Hanlee P. Ji, Aaron Diaz, Noemi Andor
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Image-based single-cell transcriptomics can identify diverse cell types within intact tissues. However, in adaptive immunity, V(D)J recombination generates unique immune receptors within cells of the same type, leading to important functional variation that is not yet defined by these methods. Here we introduce B-cell-receptor multiplexed error robust fluorescence in situ hybridization (BCR-MERFISH), which distinguishes plasma cell clones based on V-gene usage in combination with transcriptome profiling. We demonstrate that BCR-MERFISH accurately identifies V-gene usage in cell culture and in mice with restricted or native plasma cell diversity. We then use BCR-MERFISH to reveal the microbiota-dependent changes in plasma cell abundance, clonal diversity, and public clonotype usage in the mouse gut and the non-uniform distribution of plasma cell clones along the mouse ileum. As tissue context is an essential modulator of plasma cell dynamics, we anticipate that BCR-MERFISH may offer new insights into a wide range of immunological questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04bc662f55dd7766902cd268f9ec1ec3c80d4c7" target='_blank'>
              In situ profiling of plasma cell clonality with image-based single-cell transcriptomics
              </a>
            </td>
          <td>
            Evan Yang, Jose Aceves-Salvador, C. Castrillon, Uli Herrmann, E. Akama‐Garren, M. Carroll, Jeffrey R. Moffitt
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Single-cell sequencing has advanced our understanding of cellular heterogeneity and disease pathology, offering insights into cellular behavior and immune mechanisms. However, extracting meaningful phenotype-related features is challenging due to noise, batch effects, and irrelevant biological signals. To address this, we introduce Deep scSTAR (DscSTAR), a deep learning-based tool designed to enhance phenotype-associated features. DscSTAR identified HSP+ FKBP4+ T cells in CD8+ T cells, which linked to immune dysfunction and resistance to immune checkpoint blockade in non-small cell lung cancer. It has also enhanced spatial transcriptomics analysis of renal cell carcinoma, revealing interactions between cancer cells, CD8+ T cells, and tumor-associated macrophages that may promote immune suppression and affect outcomes. In hepatocellular carcinoma, it highlighted the role of S100A12+ neutrophils and cancer-associated fibroblasts in forming tumor immune barriers and potentially contributing to immunotherapy resistance. These findings demonstrate DscSTAR’s capacity to model and extract phenotype-specific information, advancing our understanding of disease mechanisms and therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fde632c3052d28ab7ea417def4f342e90babf978" target='_blank'>
              Deep scSTAR: leveraging deep learning for the extraction and enhancement of phenotype-associated features from single-cell RNA sequencing and spatial transcriptomics data
              </a>
            </td>
          <td>
            Lianchong Gao, Yujun Liu, Jiawei Zou, Fulan Deng, Zheqi Liu, Zhen Zhang, Xinran Zhao, Lei Chen, Henry H. Y. Tong, Yuan Ji, Huangying Le, Xin Zou, Jie Hao
          </td>
          <td>2025-05-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pituitary neuroendocrine tumors (PitNETs) can be invasive or aggressive, yet the mechanisms behind these behaviors remain poorly understood, impeding treatment advancements. Here, we integrat single-cell RNA sequencing and spatial transcriptomics, analyzing over 177,000 cells and 35,000 spots across 57 tissue samples. This comprehensive approach facilitates the identification of PitNETs tumor populations and characterizes the reconfiguration of the tumor microenvironment (TME) as PitNETs progress and invade. We trace the trajectory of TPIT-lineage PitNETs and identify an aggressive tumor cluster marked by elevated p53-mediated proliferation and a higher Trouillas classification, both associated with tumor progression. Additionally, we document the heterogeneity of immune stromal cells within PitNETs, particularly noting the enrichment of SPP1+ tumor associated macrophages (TAMs) in invasive tumors. These TAMs facilitate tumor invasion through the SPP1-ITGAV/ITGB1 signaling pathway. Our in-depth single-cell and spatial analysis of PitNETs uncovers the molecular dynamics within the TME, suggesting potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b1b01507bfc7f0aa4eb306817808e84f0ce1133" target='_blank'>
              Single-cell and spatial transcriptome analyses reveal tumor heterogeneity and immune remodeling involved in pituitary neuroendocrine tumor progression
              </a>
            </td>
          <td>
            Wan Su, Zhang Ye, Jifang Liu, K. Deng, Jinghua Liu, Huijuan Zhu, Lian Duan, Chen Shi, Linjie Wang, Yuxing Zhao, Fengying Gong, Yi Zhang, Bo Hou, Hui You, Feng Feng, Qing Ling, Yu Xiao, Yongdong Guo, Wenyi Fan, Sumei Zhang, Zixin Zhang, Xiaomin Hu, Yong Yao, Chunhong Zheng, Lin Lu
          </td>
          <td>2025-05-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb74e2d2b03a62ef097600aaf5d84f08dfc469c4" target='_blank'>
              Prognostically favorable immune responses to ovarian cancer are distinguished by self-reactive intra-epithelial plasma cells
              </a>
            </td>
          <td>
            Allyson C. Banville, Céline M. Laumont, Karanvir Singh, Breeze Gladwin, Jaden Dedora, Liam Mitchell, Jen Wang, A. Miranda, Bridget Mateyko, E. Chavez, G. Negri, Se Wing Grace Cheng, Julian Smazynski, K. Milne, Kylee Wright, Malia Lampard, Nicole S. Gierc, P. Hamilton, Sandra E. Spencer, Shreena Kalaria, Talen Oostenbroek, Victor Negrea Puskas, Christian Steidl, G. Morin, F. Nathoo, Brad H Nelson
          </td>
          <td>2025-05-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328ab9ab4a1194c8da958fc1fd33834df82031cb" target='_blank'>
              Context-dependent spatial multicellular network motifs for single-cell spatial biology
              </a>
            </td>
          <td>
            Amos Zamir, Yael Amitay, Yuval Tamir, Leeat Keren, A. Zaritsky
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Recent technological advances enable mapping of tissue spatial organization at single-cell resolution, but methods for analyzing spatially continuous microenvironments are still lacking. We introduce ONTraC, a graph neural network-based framework for constructing spatial trajectories at niche-level. Through benchmarking analyses using multiple simulated and real datasets, we show that ONTraC outperforms existing methods. ONTraC captures both normal anatomical structures and disease-associated tissue microenvironment changes. In addition, it identifies tissue microenvironment-dependent shifts in gene expression, regulatory network, and cell–cell interaction patterns. Taken together, ONTraC provides a useful framework for characterizing the structural and functional organization of tissue microenvironments. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03588-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e479cda2baa828dc88acb0a9487d930981c05cbb" target='_blank'>
              ONTraC characterizes spatially continuous variations of tissue microenvironment through niche trajectory analysis
              </a>
            </td>
          <td>
            Wen Wang, Shiwei Zheng, Sujung Crystal Shin, J. C. Chávez‐Fuentes, Guo-Cheng Yuan
          </td>
          <td>2025-05-08</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="In the progression from Inflammatory Bowel Disease to associated cancer, the clonal mutational landscape shifts from selection of mutations in inflammatory genes to selection of cancer-driver mutations1–4. How prevalence and expansion of either type of mutated clones could be impacted by the cellular environment in which they arise, and how this affects the neoplastic outcome of colitis is unknown. Here, we combine in vivo lineage tracing, in-silico modelling, mutational profiling and spatial transcriptomics in a mouse model of colitis-associated tumorigenesis to capture clone fates associated with chronic inflammation. We identify epithelial- and immune-enriched neighbourhoods and propose a model in which establishment of a reparative tissue environment facilitates tumours initiation by promoting the selection and expansion of pro-oncogenic clones, reducing the span of inflammation-resistant neighbourhoods containing non-oncogenic clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b08041bfbc2e608a3daf26d2af9f4a68fced7b6" target='_blank'>
              Paired mutation calling and spatial transcriptomics identify cellular neighbourhoods dictating the neoplastic outcome of colitis
              </a>
            </td>
          <td>
            Doug Winton, Elisa B. Moutin, Linus Chang, Giada Giavara, Shenay Mehmed, Mathilde Colombé, Filipe C Lourenço, Maria-Nefeli Skoufou-Papoutsaki, Richard Kemp, Philippe Gascard, T. Tlsty, David Tourigny
          </td>
          <td>2025-05-08</td>
          <td>Research Square</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e14574


 Background:
 Understanding cellular interactions within the tumor microenvironment (TME) is essential for elucidating disease progression and advancing immunotherapy. The TME is a complex ecosystem composed of cells with dysregulated metabolism immune response and signaling pathways, which in turn influence tumor development and treatment response. Multiplexed assessment serves as an important tool for clinical oncology. The simultaneous readout of multiple processes can provide biological insights and elucidate disease mechanism. Imaging Mass Cytometry (IMC) is a spatial biology imaging technique that utilizes CyTOF technology and enables deep characterization of the diversity and complexity of the TME. IMC technology offers scalable, high-throughput acquisition while generating high-quality data with true dynamic range of signal without amplification or fluorescence-based limitations such as spectral overlap and autofluorescence.
 Methods:
 To study cellular processes and their roles in tumor progression, we utilized the Human Cell Metabolism and Human Cell Signaling Panels to investigate energy production, cellular homeostasis and mitogenic signaling pathways. We then mapped these processes to the types of cells in the TME by using the Human Immuno-Oncology IMC Panel and the Human T Cell Exhaustion IMC Panel or the Maxpar Neuro Phenotyping IMC Panel Kit to characterize immune cell and neurological phenotypes in detail. We first acquired data using Preview Mode to assess the whole tissue, followed by higher-resolution imaging of selected regions of interest using Cell Mode or of the whole tissue section using Tissue Mode.
 Results:
 Our data analysis revealed significant insights into the spatial organization and metabolic profile of cells across cancer tissues. Elevated glycolysis and mTOR pathway activation suggested adaptations to hypoxia and anabolic growth in tumor areas, while interactions between fibroblasts and immune cells highlighted crosstalk within the TME. Our Neuro Phenotyping Panel was used to reveal potential for immune response in glioblastoma. Unsupervised pixel clustering and hierarchical clustering using MCD SmartViewer highlighted metabolic activity and activation of signaling pathways within tumor regions.
 Conclusions:
 Comprehensive spatial biology profiling using the IMC approach highlights the interconnected roles these pathways play in promoting tumor survival and resistance to therapies. These findings, which illuminate the metabolic and signaling heterogeneity of the TME, are crucial for developing future prognostic assessments and have the potential to guide more effective, personalized cancer therapies. For Research Use Only. Not for use in diagnostic procedures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b16079bb313271e366d31125c0f7c2617d3a11a" target='_blank'>
              Spatial profiling of solid tumor microenvironment using imaging mass cytometry with high-plex panels.
              </a>
            </td>
          <td>
            T. Pfister, J. Y. Chwee, Nick Zabinyakov, Qanber Raza, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Solid cancers are complex 'ecosystems' comprised of diverse cell types and extracellular molecules, where heterotypic interactions significantly influence disease etiology and therapeutic response. However, our current understanding of tumor microenvironments (TMEs) remains incomplete, hindering the development and implementation of novel TME-targeted drugs. To maximize cancer therapeutic development we require a systems-level understanding of the malignant, stromal, and immune states that define the tumor and determine treatment response. In their recent study, Liu and colleagues took a new approach to resolving the complexity of stromal heterogeneity. They leveraged extensive single-cell spatial multi-omics datasets across various cancer types and platforms to identify four conserved spatial Cancer-Associated Fibroblast (CAF) subtypes, classified by their spatial organization and cellular neighborhoods. Their work expands upon prior efforts to develop a CAF taxonomy, which primarily relied on single-cell RNA-Sequencing (scRNA-Seq) and yielded a multitude of classification systems. This study advances our understanding of CAF biology by establishing a link between spatial context and CAF identity across diverse tumor types. Departing from recent single-cell transcriptomics studies that employed a marker-based approach for sub-state identification, Liu and colleagues conducted de novo discovery of CAF subtypes using spatial neighborhood information alone. By positioning spatial organization as the defining axis of CAF heterogeneity, this research redefines CAF classification and provides a new framework for exploring the rules governing tumor ecosystems and developing novel ecosystem-based therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940389ee957141f2833e0d9de53711c0ead3a30c" target='_blank'>
              Pan-Cancer Spatial Profiling Reveals Conserved Subtypes and Niches of Cancer-Associated Fibroblasts.
              </a>
            </td>
          <td>
            Hani Jieun Kim, Travis Ruan, Alex Swarbrick
          </td>
          <td>2025-05-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Tumors are dynamic ecosystems where interactions between cancer cells and the tumor microenvironment (TME) shape malignancy. Using spatial transcriptomics, we analyzed 63 untreated tumors from 10 cancer types to investigate the spatial distribution of 13 cancer hallmarks. Our findings reveal that hallmark activity is spatially organized, with distinct compartmentalization of hallmark phenotypes across neoplastic and TME regions. Hallmarks such as "Resisting Cell Death" and "Enabling Replicative Immortality" were enriched in cancer-dominant areas, while "Inducing Angiogenesis" and "Activating Invasion and Metastasis" were primarily localized to the TME. Genomic analysis showed that subclonal diversity influences hallmark specialization, with genetically distinct clones often occupying specific ecological niches. These insights suggest that hallmark activities are not uniformly distributed but instead reflect the interplay between genetic heterogeneity and tumor spatial organization.
 We further modeled the spatial interdependence of hallmarks between the neoplastic and TME compartments, demonstrating that hallmark activities are shaped by ecological dynamics across the tumor. For example, hallmarks such as "Sustaining Proliferative Signaling" in the TME interface strongly influenced cancer cell hallmark expression. Application of this framework to 33 bladder cancer patients from the DUTRENEO trial revealed correlations between hallmark spatial patterns and treatment response. Notably, differences in the spatial localization of "Inducing Angiogenesis" and "Resisting Cell Death" were linked to neoadjuvant chemotherapy and immunotherapy sensitivity. These findings underscore the importance of spatial hallmark distribution in determining drug efficacy and suggest that spatial hallmark signatures could serve as predictive biomarkers for personalized cancer therapy.


 Eduard Porta-Pardo, Mustafa Sibai, Sergi Cervilla, Daniela Grases, Manel Esteller. The spatial landscape of cancer hallmarks reveals patterns of tumor ecological dynamics and drug sensitivity [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB265.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89052d3ed2d82ce2288e348ab845b3ae781f2935" target='_blank'>
              Abstract LB265: The spatial landscape of cancer hallmarks reveals patterns of tumor ecological dynamics and drug sensitivity
              </a>
            </td>
          <td>
            E. Porta-Pardo, Mustafa Sibai, Sergi Cervilla, D. Grases, M. Esteller
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity. GBM tumors contain neoplastic cells within a spectrum of different transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we developed a novel framework, ISOSCELES (Inferred cell Sensitivity Operating on the integration of Single-Cell Expression and L1000 Expression Signatures), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identified compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states. We validated the significance of these findings in vitro, ex vivo, and in vivo, and identified a novel combination of an OLIG2 inhibitor and Depatux-M for GBM. Our studies suggest that ISOSCELES identifies cell states sensitive and resistant to targeted therapies in GBM and that it can be applied to identify new synergistic combinations. Highlights Integration of GBM single-cell RNA sequencing data with L1000-derived drug response signatures facilitates clustering of tumor cells and small molecules on cell-drug connectivity. Cell-drug connectivity predicts the identities of drug-sensitive and resistant cell states. In silico perturbation analysis using cell-drug connectivity predicts drug-induced changes in the cell-drug connectivity landscape in vivo. In silico perturbation analysis to predict drug-induced changes in the tumor cell-drug connectivity landscape predicts drug combinations that synergize in vivo to extend survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25613f3a85e6691e66bbf8705e07ffcebf8c6cbf" target='_blank'>
              DRUG AND SINGLE-CELL GENE EXPRESSION INTEGRATION IDENTIFIES SENSITIVE AND RESISTANT GLIOBLASTOMA CELL POPULATIONS
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, Ricardo J Komotar, M. I. De La Fuente, Gregory Stein, Alexander Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, N. Ayad
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83b5ca4a7a4ab998ed122fece632645d240cae73" target='_blank'>
              Decoding Plasticity Regulators and Transition Trajectories in Glioblastoma with Single-cell Multiomics
              </a>
            </td>
          <td>
            Manu Saraswat, Laura Rueda-Gensini, Elisa Heinzelmann, Tannia Gracia, Fani Memi, Grant de Jong, Jannes Straub, Cedar Schloo, D. Hoffmann, E. Jung, Tim Kindinger, Bettina Weigel, Bryce Lim, S. Weil, Oliver Gould, Richard Mair, Katharina Mikulik, Martin Rohbeck, W. Wick, F. Winkler, O. Bayraktar, O. Stegle, Moritz Mall
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Elucidating the role of somatic mutations in cancer, healthy tissues, and aging depends on methods that can accurately characterize somatic mosaicism across different cell types, as well as assay their impact on cellular function. Current technologies to study cell-type-specific somatic mutations within tissues are low-throughput. We developed Duplex-Multiome, incorporating duplex consensus sequencing to accurately identify somatic single-nucleotide variants (sSNV) from the same nucleus simultaneously analyzed for single-nucleus ATAC-seq (snATAC-seq) and RNA-seq (snRNA-seq). By introducing strand-tagging into the construction of snATAC-seq libraries, duplex sequencing reduces sequencing error by >10,000-fold while eliminating artifactual mutational signatures. When applied to 98%/2% mixed cell lines, Duplex-Multiome identified sSNVs present in 2% of cells with 92% precision and accurately captured known sSNV mutational spectra, while revealing unexpected subclonal lineages. Duplex-Multiome of > 51,400 nuclei from postmortem brain tissue captured sSNV burdens and spectra across all major brain cell types and subtypes, including those difficult to assay by single-cell whole-genome sequencing (scWGS). This revealed for the first time that diverse neuronal and glial cell types show distinct rates and patterns of age-related mutation, while also directly discovering developmental cell lineage relationships. Duplex-Multiome identified clonal sSNVs occurring at increased rates in glia of certain aged brains, as well as clonal sSNVs that correlated with changes in expression of nearby genes, in both neurotypical and autism spectrum disorder (ASD) individuals, directly demonstrating that somatic mutagenesis can contribute to gene expression phenotypes. Duplex-Multiome can be easily adopted into the 10X Multiome protocol and will bridge somatic mosaicism to a wide range of phenotypic readouts across cell types and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c45c976013cae1b807eb9d7fbbf98792d114f7a" target='_blank'>
              Cell-type-specific patterns and consequences of somatic mutation in development and aging brain
              </a>
            </td>
          <td>
            Andrea J. Kriz, Shulin Mao, Diane D. Shao, Daniel A. Snellings, Rebecca E. Andersen, Guanlan Dong, Chanthia C. Ma, Hayley E. Cline, A. Huang, Eunjung Lee, Christopher A. Walsh
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lymphomas are highly heterogeneous malignancies characterized by diverse genetic, epigenetic, and phenotypic profiles, which complicate diagnosis, prognosis, and treatment. Traditional bulk sequencing methods mask the complexity of intratumoral variation, often overlooking rare subclones that may drive disease progression and therapeutic resistance. Single-cell genomics has emerged as a transformative approach to decipher tumor heterogeneity at unprecedented resolution. This technology enables the dissection of individual cellular populations within lymphomas, offering insights into clonal evolution, transcriptional diversity, and microenvironmental interactions. By applying single-cell RNA sequencing (scRNA-seq), chromatin accessibility assays (scATAC-seq), and single-cell DNA sequencing (scDNA-seq), researchers can unravel lineage relationships, identify resistant subpopulations, and track dynamic changes in response to therapy. In lymphomas such as diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL), and mantle cell lymphoma (MCL), single-cell approaches have revealed distinct malignant and non-malignant cell states that correlate with treatment outcomes. Moreover, integrating single-cell data with spatial transcriptomics and immune profiling enhances the understanding of the tumor microenvironment, including immune evasion mechanisms. These insights can inform personalized treatment strategies, identify novel therapeutic targets, and enable early detection of relapse. Despite technical challenges such as data complexity, sample viability, and cost, the application of single-cell genomics in lymphoma research is rapidly advancing. Future directions include multi-omics integration, real-time patient monitoring, and clinical translation of predictive biomarkers. This review underscores the pivotal role of single-cell genomics in resolving tumor heterogeneity and predicting therapeutic resistance, positioning it as a cornerstone for next-generation precision oncology in lymphomas,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/438856abbab164e715372bfe3f2a6c07df9594ec" target='_blank'>
              Role of mutational signatures and clonality assessments in tailoring targeted therapies for lymphoma
              </a>
            </td>
          <td>
            Andoh Sheikh Atta-ullah, Uche Philip, Sampson Janice Candy
          </td>
          <td>2025-05-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Nasopharyngeal carcinoma (NPC) is a common malignancy with a complex pathogenesis and diverse cellular composition. This study aimed to systematically analyze the transcriptional characteristics of different cell types in NPC and their roles in tumor development using single-cell transcriptomics. Methods The research team collected NPC and normal tissue samples, and performed in-depth single-cell sequencing analysis of the transcriptomes. Single-cell RNA sequencing was performed on the collected samples to obtain high-resolution transcriptional profiles of individual cells. This allowed the researchers to identify and characterize the diverse cell populations present within the NPC tumor microenvironment. Results The results showed that NPC samples contained multiple distinct cell subpopulations, including epithelial cells, immune cells (such as macrophages and T cells), endothelial cells, and stromal cells. These cell types exhibited marked differences in spatial distribution and transcriptional profiles, reflecting the high degree of heterogeneity within the tumor microenvironment. Further functional analysis revealed significant dysregulation of mitochondrial-related pathways, extracellular matrix-receptor interactions, as well as Wnt, Notch, and other signaling cascades in NPC. Conclusion This study employed single-cell transcriptomics to comprehensively elucidate the complexity of the NPC tumor microenvironment, providing new insights into the underlying mechanisms of disease pathogenesis and suggesting potential therapeutic targets. These findings lay the groundwork for the development of precision medicine approaches for NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9793611f5914170a344f624509f87b1fa27aa37" target='_blank'>
              Single-cell transcriptomic landscape reveals complex cellular heterogeneity and dysregulated signaling pathways in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Xiaoyan Zhang, Zuojia Guan, Zhan Xu
          </td>
          <td>2025-05-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spatial transcriptomics technologies have emerged as powerful tools for understanding cellular identity and function within the natural spatial context of tissues. Traditional transcriptomics techniques, such as bulk and single-cell RNA sequencing, lose this spatial information, which is critical for addressing many biological questions. Here, we present a protocol for high-resolution spatial transcriptomics using fixed frozen mouse lung sections mounted on 10X Genomics Xenium slides. This method integrates multiplexed fluorescent in situ hybridization (FISH) with high-throughput imaging to reveal the spatial distribution of mRNA molecules in lung tissue sections, allowing detailed analysis of gene expression changes in a mouse model of pulmonary hypertension (PH). We compared two tissue preparation methods, fixed frozen and fresh frozen, for compatibility with the Xenium platform. Our fixed frozen approach, utilizing a free-floating technique to mount thin lung sections onto Xenium slides at room temperature, preserved tissue integrity and maximized the imaging area, resulting in high-fidelity spatial transcriptomics data. Using a predesigned 379-gene mouse panel, we identified 40 major lung cell types. We detected key cellular changes in PH, including an increase in arterial endothelial cells (AECs) and fibroblasts, alongside a reduction in capillary endothelial cells (CAP1 and CAP2). Through differential gene expression analysis, we observed markers of endothelial-to-mesenchymal transition and fibroblast activation in PH lungs. High-resolution spatial mapping further confirmed increased arterialization in the distal microvasculature. These findings underscore the utility of spatial transcriptomics in preserving the native tissue architecture and enhancing our understanding of cellular heterogeneity in disease. Our protocol provides a reliable method for integrating spatial and transcriptomic data using fixed frozen lung tissues, offering significant potential for future studies in complex diseases such as PH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/399fddf9db865d1e3c47fe6ffcc8cfaa55ffe72d" target='_blank'>
              Enhanced Spatial Transcriptomics Analysis of Mouse Lung Tissues Reveals Cell-Specific Gene Expression Changes Associated with Pulmonary Hypertension
              </a>
            </td>
          <td>
            Hanqiu Zhao, Xiaokuang Ma, Peng Chen, Bin Liu, Jing Wei, John Zhang, Ankit A. Desai, Andrea L. Frump, Olga Rafikova, Michael B Fallon, Shenfeng Qiu, Zhiyu Dai
          </td>
          <td>2025-05-15</td>
          <td>Journal of respiratory biology and translational medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy with extremely poor prognosis. SCLC cells exhibit high plasticity and can progress from neuroendocrine (NE) to non-NE phenotypes. This dynamic evolution promotes treatment resistance and relapses, representing a challenge for targeted therapies in this elusive disease. Here we identify the transcription factor ONECUT2 (OC2) as a driver of plasticity in SCLC, leading to non-NE transcriptional states. OC2 is highly expressed in SCLC tumors compared to normal lung tissue and its expression is associated with heightened clinical stage and lymph node metastasis. We show that OC2 is a repressor of ASCL1, the NE master regulator transcription factor. In addition, OC2 upregulates non-NE programs through activation of c-MYC and Notch signaling. We also demonstrate that OC2 is required for growth and survival of SCLC cells and that it can be targeted with a small molecule inhibitor that acts synergistically with the standard combination of cisplatin and etoposide, providing a novel therapeutic strategy for OC2 active SCLC tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01267-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736f50c299b3fe9c13bfbd3d67e9601f1567eb2" target='_blank'>
              ONECUT2 reprograms neuroendocrine fate and is an actionable therapeutic target in small cell lung cancer
              </a>
            </td>
          <td>
            Mirian Gutiérrez, Irene Zamora, Raquel Iriarte, M. Pajares, Qian Yang, Chen Qian, N. Otegui, J. Fernández-Irigoyen, Enrique Santamaria, N. Alcala, A. Sexton-Oates, L. Fernandez-Cuesta, Miguel Barajas, Alfonso Calvo, Luis M. Montuenga, Beatrice S. Knudsen, Sungyong You, Michael R. Freeman, Ignacio J. Encío, Mirja Rotinen
          </td>
          <td>2025-06-11</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background Renal cell carcinoma (RCC) is characterized by a high degree of genomic but also functional intratumoral heterogeneity (ITH). Mutations in VHL, chromatin remodeling genes such as SETD2 and genes that regulate the PI3K/AKT/mTOR pathway have been identified as recurrent drivers despite genomic ITH. Whether and to what extent these mutations shape functional ITH including the formation of spatial niches is incompletely understood. Herein, we analyze the correlation between mutational drivers and their functional proxies in a spatially defined manner. Methods A total of 23 RCCs were analyzed by panel next-generation sequencing followed by immunohistochemistry for five functional proxies for key genetic alterations including the expression of CD31, GLUT1, phospho-mTOR S2448, H3K36me3 and Ki-67. Antibody stainings were scored semiquantitatively in the tumor periphery and the tumor center. Results Unexpectedly, the presence of a VHL mutation was not found to correlate with its functional proxies including the expression of CD31/microvessel density or the expression of the glucose transporter GLUT1. Likewise, there was no correlation between the presence of activating mutations in genes of the PI3K/AKT/mTOR pathway and the expression of activated phospho-mTOR S2448. Furthermore, mutations in the methyltransferase gene SETD2 were not found to correlate with the expression level of its downstream target H3K36me3. Lastly, there was no correlation between the expression of the proliferation marker Ki-67 and the number of driver mutations. Conclusion This proof-of-concept study adds genotype-phenotype heterogeneity as additional layer of complexity to the known genomic and functional ITH in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/455b536369691fca3d4bb3ee3d0cab12ab897412" target='_blank'>
              Extensive genotype-phenotype heterogeneity in renal cell carcinoma - a proof-of-concept study
              </a>
            </td>
          <td>
            Jakob Wieke, Christina Jurcic, A. Kaczorowski, Sarah Böning, Martina Kirchner, C. Schwab, V. Schütz, Markus Hohenfellner, A. Duensing, Albrecht Stenzinger, S. Duensing
          </td>
          <td>2025-04-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Breast cancer brain metastases (BCBM) affect nearly 90,000 patients annually in the United States and carry a significant risk of mortality. As metastatic lesions develop, the unique milieu of the brain microenvironment shapes disease progression and therapeutic response. Among resident brain cells, astrocytes are both the most common, and are increasingly recognized as key regulators of this process, yet their precise role remains poorly defined. Here, we present a hybrid agent-based model (ABM) to simulate tumor–astrocyte interactions on a two-dimensional lattice. In our model, metastatic tumor cells induce phenotypic reprogramming of astrocytes from an antito a pro-metastatic state, thereby enhancing tumor proliferation. We systematically evaluate how variations in astrocyte density, spatial distribution, and chemotherapy impact tumor expansion and spatial morphology, quantified by fractal dimension, lacunarity, and eccentricity. Our simulations reveal that astrocyte reprogramming accelerates tumor progression and contributes to increased morphological complexity and chemotherapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331a7ee541e9213b60f3df2d93320f1bf057f75f" target='_blank'>
              Astrocyte reprogramming drives tumor progression and chemotherapy resistance in agent-based models of breast cancer brain metastases
              </a>
            </td>
          <td>
            Rupleen Kaur, R. Barker-Clarke, Andrew Dhawan
          </td>
          <td>2025-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Organotropism results from the functional versatility of metastatic cancer cells to survive and proliferate in diverse microenvironments. This adaptivity can originate in clonal variation of the spreading tumor and is often empowered by epigenetic and molecular reprogramming of cell regulatory circuits. Related to organotropic colonization of metastatic sites are environmentally-sensitive, differential responses of cancer cells to therapeutic attack. Accordingly, understanding the organotropic profile of a cancer and probing the underlying driver mechanisms are of high clinical importance. However, determining systematically the organotropism of one cancer versus the organotropism of another cancer, potentially with the granularity of comparing the same cancer type between patients or tracking the evolution of a cancer in a single patient for the purpose of personalized treatment, has remained very challenging. It requires a host organism that allows observation of the spreading pattern over relatively short experimental times. Moreover, organotropic patterns often tend to be statistically weak and superimposed by experimental variation. Thus, an assay for organotropism must give access to statistical powers that can separate ‘meaningful heterogeneity’, i.e., heterogeneity that determines organotropism, from ‘meaningless heterogeneity’, i.e., heterogeneity that causes experimental noise. Here we describe an experimental workflow that leverages the physiological properties of zebrafish larvae for an imaging-based assessment of organotropic patterns over a time-frame of 3 days. The workflow incorporates computer vision pipelines to automatically integrate the stochastic spreading behavior of a particular cancer xenograft in tens to hundreds of larvae allowing subtle trends in the colonization of particular organs to emerge above random cell depositions throughout the host organism. We validate our approach with positive control experiments comparing the spreading patterns of a metastatic sarcoma against non-transformed fibroblasts and the spreading patterns of two melanoma cell lines with previously established differences in metastatic propensity. We then show that integration of the spreading pattern of xenografts in 40 – 50 larvae is necessary and sufficient to generate a Fish Metastatic Atlas page that is representative of the organotropism of a particular oncogenotype and experimental condition. Finally, we apply the power of this assay to determine the function of the EWSR1::FLI1 fusion oncogene and its transcriptional target SOX6 as plasticity factors that enhance the adaptive capacity of metastatic Ewing sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5025875828f543583bc1d12169d95499fa95592" target='_blank'>
              Robust statistical assessment of Oncogenotype to Organotropism translation in xenografted zebrafish
              </a>
            </td>
          <td>
            D. Saucier, Xuexia Jiang, Divya Rajendran, Roshan Ravishankar, Erin Butler, A. Marchetto, R. Kurmasheva, T. G. Grünewald, James F. Amatruda, G. Danuser
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Multiplex imaging technologies have developed rapidly over the past decades. The advancement of multiplex imaging has been driven in part by the recognition that the spatial organization of cells can represent important prognostic biomarkers and that simply studying the composition of cells in diseased tissue is often insufficient. There remains a lack of tools that can perform spatial analysis at the level of cellular aggregates (a common histopathological presentation) such as tumors and granulomas, with most analysis packages focusing on smaller regions of interest and potentially missing patterns in the overall lesion structure and cellular distribution. Here, we present protocols to quantitatively describe the cellular structure of entire tissue lesions, built around two novel metrics. The Total Cell Preference Index reports whether a lesion tends to change in density in its central versus peripheral areas and can indicate the extent of necrosis across the entire lesion. The Immune Cell Preference Index then reports whether each immune cell type is located more centrally or peripherally across the entire lesion. The output of both indexes is a single number readout for simple interpretation and visualization, and these indexes can be applied to lesions of any size or shape. This simplifies cross‐lesion comparison compared to traditional Euclidian distance–based analysis, which outputs multiple values for each lesion (one for output for each band used in the infiltration analysis). Additionally, this approach can be applied to any slide‐scanning multiplexed imaging system, based on either protein or nucleic acid staining. Finally, the approach uses the open‐source software QuPath and can be utilized by researchers with a basic understanding of QuPath, with the full analysis able to be applied to pre‐generated images within 1 hr. © 2025 The Author(s). Current Protocols published by Wiley Periodicals LLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21c9a3868c16731773831bf236e1e04a21a9d674" target='_blank'>
              Highly Adaptable Analysis Tools for Mapping Spatial Features of Cellular Aggregates in Tissues
              </a>
            </td>
          <td>
            Andrew Sawyer, Nick Weingaertner, Ellis Patrick, Carl G Feng
          </td>
          <td>2025-05-01</td>
          <td>Current Protocols</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosomal instability (CIN), a characteristic feature of esophageal adenocarcinoma (EAC), drives tumor aggressiveness and therapy resistance, presenting an intractable problem in cancer treatment. CIN leads to constitutive stimulation of the innate immune cGAS–STING pathway, which has been typically linked to anti-tumor immunity. However, despite the high CIN burden in EAC, the cGAS– STING pathway remains largely intact. To address this paradox, we developed novel esophageal cancer models, including a CIN-isogenic model, discovering myeloid-attracting chemokines – with the chemokine CXCL8 (IL-8) as a prominent hit – as conserved CIN-driven targets in EAC. Using high-resolution multiplexed immunofluorescence microscopy, we quantified the extent of ongoing cGAS-activating CIN in human EAC tumors by measuring cGAS-positive micronuclei in tumor cells, validated by orthogonal whole-genome sequencing-based CIN metrics. By coupling in situ CIN assessment with single-nucleus RNA sequencing and multiplex immunophenotypic profiling, we found tumor cell-intrinsic innate immune activation and intratumoral myeloid cell inflammation as phenotypic consequences of CIN in EAC. Additionally, we identified increased tumor cell-intrinsic CXCL8 expression in CINhigh EAC, accounting for the inflammatory tumor microenvironment. Using a novel signature of CIN, termed CINMN, which captures ongoing CIN-associated gene expression, we confirm poor patient outcomes in CINhigh tumors with signs of aberrantly rewired cGAS–STING pathway signaling. Together, our findings help explain the counterintuitive maintenance and expression of cGAS–STING pathway components in aggressive, CINhigh tumors and emphasize the need to understand the contribution of CIN to the shaping of a pro-tumor immune landscape. Therapeutic strategies aimed at disrupting the cGAS-driven inflammation axis may be instrumental in improving patient outcomes in this aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dfd8088150f447fa336f3b6ae36838000571864" target='_blank'>
              Chromosomal instability shapes the tumor microenvironment of esophageal adenocarcinoma via a cGAS–chemokine–myeloid axis
              </a>
            </td>
          <td>
            Bruno Beernaert, Rose L Jady-Clark, Parin Shah, Erik Ramon‐Gil, Nora M Lawson, Zack D. Brodtman, Somnath Tagore, Frederik Stihler, Alfie S Carter, Shannique Clarke, Tong Liu, Winston Zhu, Erkin Erdal, Alistair Easton, Leticia Campo, Molly Browne, Stephen Ash, Nicola Waddell, Thomas Crosby, Simon R. Lord, Derek A Mann, Ignacio Melero, Carlos E. De Andrea, Andrea E. Tijhuis, F. Foijer, Ester M. Hammond, Kadir C Akdemir, J. Leslie, Benjamin Izar, Eileen E Parkes
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e285847d1a624eba0863eb2e7a290a733cac7918" target='_blank'>
              Phenotypic and Functional Characterization of Oncohistone Mutations in Breast Cancers
              </a>
            </td>
          <td>
            Andrea D. Edwards, Yangyang Dai, Siddharth Singh, Micah Thornton, Tulip Nandu, Ralf Kittler, C. Camacho, Dan Huang, W. Kraus
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is one of the most lethal cancers for humans. HCC is highly heterogeneous. In this study, we performed ultra-depth (∼1 million reads per spot) sequencing of 6,320 spatial transcriptomes on a case of HCC. Sixteen distinct spatial expression clusters were identified. Each of these clusters was spatially contiguous and had distinct gene expression patterns. In contrast, benign liver tissues showed minimal heterogeneity in terms of gene expression. Numerous immune cell-enriched spots were identified in both HCC and benign liver regions. Cells adjacent to these immune cell-enriched spots showed significant alterations in their gene expression patterns. Interestingly, the responses of HCC cells to the nearby immune cells were significantly more intense and broader, while the responses of benign liver cells to immune cells were somewhat narrow and muted, suggesting an innate difference in immune cell activities towards HCC cells in comparison with benign liver cells. However, cell-cell interaction analyses showed significant immune evasion by HCC cancer cells. When standard-depth sequencing was performed, significant numbers of genes and pathways that were associated with these changes disappeared. Qualitative differences in some pathways were also found. These results suggest that deep spatial sequencing may help to uncover previously unidentified mechanisms of liver cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa11a255e72a72322fc9f248cef1483a72dd27ef" target='_blank'>
              Deep spatial sequencing revealing differential immune responses in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yan-Ping Yu, Caroline Obert, Baoguo Ren, Marielle Krivit, Kyle Metcalfe, Jia-Jun Liu, Tuval Ben-Yehezkel, Silvia Liu, Jian-Hua Luo
          </td>
          <td>2025-06-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 8539


 Background:
 Immune checkpoint inhibitors have greatly improved outcomes in advanced non-small cell lung cancer (NSCLC). However, patients with EGFR mutant NSCLC have a poor response to immune checkpoint inhibitor therapy. Emerging evidence suggests that an immunosuppressive tumor microenvironment plays an important role in this setting, however, we still lack fundamental knowledge about tumor endothelial cell biology. We hypothesized that oncogene specific changes in the expression of immune-related genes in tumor endothelial cells account for differences in the tumor microenvironment and efficacy of immune checkpoint inhibitor therapy.
 Methods:
 We utilized spatial transcriptomics (GeoMx Digital Spatial Profiling) on resected tumor tissue of EGFR mutant (n = 5) and KRAS mutant (n = 5) NSCLC patients to investigate the transcriptional signature of tumor, endothelial and stromal cells. Additionally, we used NicheNet to explore intercellular communication between tumor, endothelial, and stromal cells. Immune gene set enrichment analysis scores were calculated using ESTIMATE. Immune cell type proportions were estimated using CIBERSORT.
 Results:
 Using spatial transcriptomics, we dissected the tumor microenvironment into tumor, stromal and endothelial compartments. By analyzing predicted cellular communication, we found that tumor and stromal cells primarily affect an interferon-related gene signature in tumor endothelial cells. Notable differentially expressed interferon-related genes, that were significantly up-regulated in KRAS mutant and down-regulated in EGFR mutant NSCLC patients, included CXCL9, STAT1, WARS1, IRF1 and ICAM1. In the stromal compartment, immune gene set enrichment analysis scores were significantly lower in EGFR mutant than KRAS mutant NSCLC (median, 355 vs. 713, P = 0.026) indicating an immunosuppressive tumor microenvironment. We observed substantial heterogeneity while exploring the cellular landscape of the stromal compartment in EGFR mutant and KRAS mutant NSCLC. Notably, we found a significantly decreased proportion of pro-inflammatory macrophages in the stromal compartment of EGFR compared to KRAS mutant NSCLC (P = 0.006).
 Conclusions:
 We identified distinct interferon-related gene signatures in tumor endothelial cells of patients with EGFR and KRAS mutantNSCLC associated with variations in the cellular composition of the tumor microenvironment. This may provide a better understanding for the development of spatial biomarkers to identify which oncogene-driven NSCLC patients are most likely to benefit from immune checkpoint inhibitor therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3c21a018e08000029baaeb3ba8f18554c44ba0f" target='_blank'>
              Spatial transcriptomic profiling of the tumor microenvironment in EGFR and KRAS mutant non-small cell lung cancer.
              </a>
            </td>
          <td>
            F. J. Bolte, Nick Natale, Samantha DiBenedetto, Margaret Moore, E. Stelow, Richard Hall, D. Gioeli
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Objective The tumor microenvironment (TME) plays a crucial role in tumor progression. Recent advancements in single-cell sequencing technology have identified Tc17 cells, a newfound subset of CD8+ T cells with a phenotype similar to Th17, within the gastric cancer (GC) microenvironment. However, the role of Tc17 cells are unclear. Materials and method We collected 296,879 cells from the single-cell sequencing data of 65 GC samples, along with spatial transcriptomic data from 10 GC samples, to further explore the role of Tc17 cells. Multicolor immunohistochemical staining demonstrated the presence of Tc17 cells in the GC TME. SCENIC analysis was performed to identify the specific transcription factors of Tc17. Pseudotime analysis of T cells was conducted to decipher the differentiation trajectory of Tc17. Additionally, cell-cell interaction analysis revealed the interactions between Tc17 cells and tumor cells. Finally, functional validation through CCK-8 proliferation assays, wound Healing assays, and transwell assays conclusively demonstrated the tumor-promoting effects of IL-17A and IL-26 on gastric tumor cells. Results Our results indicate that Tc17 cells are absent from blood and exclusively found in tissues, with greater enrichment in tumor tissues compared to normal tissues. Among CD8+ T cells, Tc17 exhibits the weakest cytotoxic capacity and the highest anti-inflammatory profile, suggesting reduced tumor-killing ability. Pseudotime analysis revealed that Tc17 cells ultimately progress towards a state of exhaustion, losing their capacity to eliminate tumor cells. Survival analysis further correlates the presence of Tc17 cells with poor prognosis. Notably, a robust interaction between Tc17 cells and tumor cells was observed in GC. We found that tumor cells can recruit Tc17 cells via the CXCL16-CXCR6 axis, this finding was validated on spatial transcriptome data. And in turn, Tc17 cells may promote tumor progression by secreting IL-17A and IL-26, as functionally corroborated by CCK-8 Assay, wound healing, and transwell assay in vitro. Conclusions These findings reveal the immunosuppressive role of Tc17 cells in gastric cancer, provide a new perspective for understanding the immunosuppressive mechanism of the gastric cancer tumor microenvironment, and provide a potential target for the development of targeted therapeutic strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4a49cf0a193cdeef05a09af604560eb6cfc60d0" target='_blank'>
              A novel Tc17 population recruited by tumor cells promotes tumor progression in gastric cancer
              </a>
            </td>
          <td>
            Yichao Wang, Hao Yuan, Yayun Tan, Li Wu, Xinchen Zhao, Zeyuan Yang, Shaoxing Dai, Naixue Yang
          </td>
          <td>2025-05-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICI) have improved patient outcomes in hepatocellular carcinoma (HCC); however, most patients do not experience durable benefit. The non-T cell-inflamed tumor microenvironment, characterized by limited CD8+ T-cell infiltration, reduced dendritic cell function, and low interferon-γ-associated gene expression, is associated with a lower likelihood of response to ICI. To nominate new therapeutic targets for overcoming ICI resistance in HCC, we conducted a large-scale multiomic analysis on 900+human specimens (RNA sequencing (RNA-seq), proteomics) and 31 tumor single-cell (sc) RNA-seq samples, with tissue validation through imaging mass cytometry (IMC) and spatial lipidomics by matrix-assisted laser desorption/ionization (MALDI), with experimental investigation by in vitro CD8+ T-cell recruitment and macrophage polarization functional assays using three-dimensional (3D) co-culture models. We discovered 32 oncogenic pathways associated with immune exclusion, with sterol regulatory element binding protein 1 (SREBP1, encoded by SREBF1) as a hub regulator. scRNA-seq analysis showed that SREBP1 signaling is associated with enriched lipid biogenesis pathways in tumor cells, elevated immunosuppressive markers in macrophages, and diminished CD8+ T-cell infiltration. Integration of IMC and MALDI images revealed distinct lipid species differentially abundant in tumor regions with low versus high CD8+ T cell infiltration. Functional studies in a 3D in vitro tumor-immune co-culture system demonstrated that CRISPR-mediated SREBF1 knockout (KO) in HepG2 cells reduced monocyte recruitment, decreased expression of the protumorigenic CD206 marker in macrophages, and enhanced CD8+ T-cell migration compared with wild-type (WT) (p<0.0001). RNA-seq of SREBF1 KO versus WT tumor cells confirmed suppression of lipid biosynthesis genes. Our findings nominate an atlas of tumor-intrinsic drivers of immune exclusion, particularly SREBP1 via pro-tumorigenic macrophage (M2-like) reprogramming. These pathways may represent novel therapeutic targets to enhance antitumor immunity and deserve further study as targeted therapy candidates to enhance ICI in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e84d6872c644cf55b881ebad30a21a4ff2aaeed3" target='_blank'>
              Multiomics identifies tumor-intrinsic SREBP1 driving immune exclusion in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Rebekah E Dadey, Ruxuan Li, Jake Griner, Jie Chen, Arjun Singh, Brian P Isett, Sarah Newman, Ryan Augustin, Aofei Li, Joseph A Manning, S. Monga, A. Singhi, David A A Geller, Carsten Krieg, Ioannis K. Zervantonakis, J. Luke, R. Bao
          </td>
          <td>2025-06-01</td>
          <td>Journal for ImmunoTherapy of Cancer</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Intra-tumor heterogeneity impacts disease progression and therapeutic resistance but remains poorly characterized by conventional histologic, immunophenotypic, and molecular approaches. Single-cell biophysical properties distinguish functional phenotypes complementary to these approaches, providing additional insight into cellular diversity. Here we link both buoyant mass and stiffness to gene expression to identify clinically relevant phenotypes within primary mantle cell lymphoma (MCL) cells, employing MCL as a model of biological and clinical diversity in human cancer. Linked measurements reveal that buoyant mass and stiffness characterize B-cell development states from naïve to plasma cell and correlate with expression of oncogenic B-cell receptor signaling genes such as BLK and CD79A. Additionally, changes in cell buoyant mass within primary patient specimens ex vivo correlate with sensitivity to Bruton’s Tyrosine Kinase inhibitors in vivo in MCL and chronic lymphocytic leukemia, another B-cell malignancy. These findings highlight the value of biophysical properties as biomarkers of response in pursuit of future precision therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/222e5440bee40a1cfc03b8b975d31d717ee09614" target='_blank'>
              Integrating Single-Cell Biophysical and Transcriptomic Features to Resolve Functional Heterogeneity in Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            Ye Zhang, Lydie Debaize, Adam Langenbucher, Jenalyn Weekes, Ioulia Vogiatzi, Teemu P. Miettinen, Mingzeng Zhang, Emily Sumpena, Huiyun Liu, Sarah M. Duquette, L. Hackett, Jeremy Zhang, Sona Baghiyan, R. Redd, M. Aryee, M. Davids, Austin I. Kim, C. Ryan, David M. Weinstock, S. Manalis, M. Murakami
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1ca3901ac698b79cb29b7a3967fd687d4414a1" target='_blank'>
              Multiomic profiling of hypoxic glioblastoma stem cells reveals expansion of subpopulations with distinct epigenetic and CNV profiles
              </a>
            </td>
          <td>
            Adrianne Corseri, Travis Moore, Nicole Szczepanski, Hyeyeon Hwang, Alan Zdon, Gurkan Yardimci Galip, Nikos Tapinos
          </td>
          <td>2025-05-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35acfa591bf2d48662dc328bde4741026068f8bb" target='_blank'>
              Neoantigens and Stochastic Fluctuations Regulate T Cell Proliferation in Primary and Metastatic Malignant Brain Tumors
              </a>
            </td>
          <td>
            Maheshwor Poudel, William Stewart, Ciriyam Jayaprakash, Jayajit Das
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GB), particularly IDH-wildtype, is the most aggressive brain malignancy with a dismal prognosis. Despite advances in molecular profiling, the complexity of its tumor microenvironment and spatial organization remains poorly understood. This study aimed to create a comprehensive single-cell and spatial atlas of GB to unravel its cellular heterogeneity, spatial architecture, and clinical relevance.


METHODS
We integrated single-cell RNA sequencing data from 26 datasets, encompassing over 1.1 million cells from 240 patients, to construct GBmap, a harmonized single-cell atlas. High-resolution spatial transcriptomics was employed to map the spatial organization of GB tissues. We developed the Tumor Structure Score (TSS) to quantify tumor organization and correlated it with patient outcomes.


RESULTS
We showcase the applications of GBmap for reference mapping, transfer learning, and biological discoveries. GBmap revealed extensive cellular heterogeneity, identifying rare populations such as tumor-associated neutrophils and homeostatic microglia. Spatial analysis uncovered seven distinct tumor niches, with hypoxia-dependent niches strongly associated with poor prognosis. The TSS demonstrated that highly organized tumors, characterized by well-defined vasculature and hypoxic niches, correlated with worse survival outcomes.


CONCLUSIONS
This study provides a comprehensive resource for understanding glioblastoma heterogeneity and spatial organization. GBmap and the TSS provide an integrative view of tumor architecture in GB, highlighting hypoxia-driven niches that may represent avenues for further investigation. Our resource can facilitate exploratory analyses and hypothesis generation to better understand disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f160a16e8e12a7fc63680546f0557a806342875" target='_blank'>
              Charting the Single-Cell and Spatial Landscape of IDH-Wildtype Glioblastoma with GBmap.
              </a>
            </td>
          <td>
            Cristian Ruiz-Moreno, Sergio Marco Salas, Erik R Samuelsson, Mariia Minaeva, Ignacio L Ibarra, Marco Grillo, S. Brandner, Ananya Roy, K. Forsberg-Nilsson, M. Kranendonk, Fabian J. Theis, Mats Nilsson, H. Stunnenberg
          </td>
          <td>2025-05-02</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The effectiveness of colorectal cancer (CRC) therapy is limited owing to the absence of treatments targeting drug-tolerant residual cancer cells. Although neoadjuvant therapy is effective, pathological examination of residual tumors has revealed the presence of small clusters of LGR5-positive cancer cells in the fibrous tissue. Here, we established a colorectal cancer cell line-derived organoid (CCD-organoid) regrowth model using a patient-derived cell line with cancer stem cell properties and demonstrated that it displayed the morphological characteristics of small clusters in clinical tissues. Time course analysis of single-cell RNA sequencing of the CCD-organoid regrowth model revealed various states and dynamic alterations within non-cycling cells. We identified subpopulations highly expressing protein translation-related genes RPL17 and EEF1G. To identify key signals for the transition of residual cancer cells to regrowth, we evaluated inhibitors targeting pathways such as the Wnt pathway, reactive oxygen species pathway, and RNA polymerase I pathway, highlighted in the single-cell RNA sequencing analysis. Only the polymerase I-inhibitor BMH-21 significantly reduced tumor growth both in vitro and in vivo, indicating the critical cell subpopulation driving recurrence. Our results demonstrate the possibility of a unique therapeutic target for CRC treatment targeting drug-tolerant residual cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa9b58cf763b8a32b666aa1599214c1a009f1d2" target='_blank'>
              Colorectal cancer cell line-derived organoid model with stem cell properties captures the regrowing state of residual cancer cells after neoadjuvant chemotherapy
              </a>
            </td>
          <td>
            Kiyotaka Nakano, E. Oki, Masaki Yamazaki, Masami Suzuki, S. Kawai, T. Fujita, Atsuhiko Kato, Y. Zaitsu, Tomoko Jogo, Chie Kato, Takeshi Watanabe, Eri Hashimoto, Chiyoko Nishime, E. Fujii, K. Ando, G. Nagae, N. Harimoto, Mitsuhiko Ota, H. Saeki, Hiroyuki Aburatani, Yoshihiko Maehara, Tatsumi Yamazaki
          </td>
          <td>2025-06-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 e14575


 Background:
 Over the last decade, immunotherapy has revolutionized cancer treatment, offering transformative outcomes for many patients. However, its effectiveness in treating solid tumors remains limited due to challenges such as T cell exhaustion, immunosuppressive tumor microenvironment (TME), lack of tumor-specific antigens and a complex immune landscape. Solid tumors also induce systemic immune changes that evolve dynamically with tumor progression. To improve patient outcome, it is essential to understand the interplay between localized and systemic immune responses.
 Methods:
 To address these challenges, we employed an integrative approach leveraging CyTOF and Imaging Mass Cytometry (IMC) technologies. Matched samples, including peripheral blood mononuclear cells (PBMCs), tumor-derived cells (TDCs) and formalin-fixed, paraffin-embedded (FFPE) tumor tissues from cancer patients, were analyzed. A custom CyTOF panel with over 40 markers was designed to profile immune subsets, checkpoint molecules, cytokines and transcription factors. Samples were barcoded, stained and analyzed using a CyTOF XT system. For spatial immune profiling, FFPE tumor tissues were stained with a 40-plus-marker IMC panel to characterize immune, stromal and tumor cells and assess immune activation states. Data was acquired with the Hyperion XTi Imaging System, enabling high-resolution imaging of tumor microenvironments.
 Results:
 CyTOF analysis revealed heterogeneity in T cell exhaustion and activation profiles in PBMCs and TDCs, highlighting the functional diversity of immune responses. IMC technology provided spatial insights into immune cell interactions within the TME, identifying distinct patterns of immune infiltration and congregation around tumor cells and stromal elements. Comparative analysis of CyTOF and IMC data uncovered shared and divergent phenotypes between peripheral and tumor-resident immune compartments, identifying biomarkers that link systemic immune profiles with TME dynamics.
 Conclusions:
 These findings offer invaluable insights into the immunological mechanisms driving antitumor responses. By integrating functional and spatial immune profiling, our approach identifies predictive biomarkers and potential therapeutic targets, guiding the development of personalized immunotherapies. This work highlights the importance of understanding systemic and localized immune responses to improve cancer treatment outcomes and optimize treatment strategies. For Research Use Only. Not for use in diagnostic procedures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8358df8b3da76d7b7dab6d3ef70735cb54fac5bc" target='_blank'>
              Integrated immune profiling approach for immuno-oncology: Unveiling biomarkers and therapeutic targets with mass cytometry.
              </a>
            </td>
          <td>
            Ling Wang, T. Pfister, J. Y. Chwee, Stephen Li, Qanber Raza, Shakir Hasan, Michael J. Cohen, Nikesh Parsotam, David Howell, Liang Lim, C. Loh
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Fibroblasts play essential roles in cancer progression, exhibiting activation states that can either promote or inhibit tumor growth. Understanding these differential activation states is critical for targeting the tumor microenvironment (TME) in cancer therapy. However, traditional molecular markers used to identify cancer-associated fibroblasts are limited by their co-expression across multiple fibroblast subtypes, making it difficult to distinguish specific activation states. Morphological and motility characteristics of fibroblasts reflect their underlying gene expression patterns and activation states, making these features valuable descriptors of fibroblast behavior. This study proposes an artificial intelligence-based classification framework to identify and characterize differentially activated fibroblasts by analyzing their morphodynamic and motile features. We extract these features from label-free live-cell imaging data of fibroblasts co-cultured with breast cancer cell lines using deep learning and machine learning algorithms. Our findings show that morphodynamic and motile features offer robust insights into fibroblast activation states, complementing molecular markers and overcoming their limitations. This biophysical state-based cellular classification framework provides a novel, comprehensive approach for characterizing fibroblast activation, with significant potential for advancing our understanding of the TME and informing targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1693652b380d377e2983bb1084b665960d61177d" target='_blank'>
              Classification of differentially activated groups of fibroblasts using morphodynamic and motile features
              </a>
            </td>
          <td>
            Minwoo Kang, Chanhong Min, Somayadineshraj Devarasou, Jennifer H. Shin
          </td>
          <td>2025-05-15</td>
          <td>APL Bioengineering</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer (BRCA) is a complex disease influenced by the tumor microenvironment, where interactions between immune cells and cancer cells play a crucial role in tumor progression and response to therapy. Understanding the intricacies of these interactions requires detailed analysis at the single-cell level, enabling the identification of specific immune cell subpopulations and their functional roles within the tumor milieu. This study comprehensively analyzed immune cell subpopulations and macrophage subtypes in BRCA using single-cell RNA sequencing technology and various computational tools. Initially, Sc-Type software accurately identified and annotated immune cell subpopulations, followed by CNV analysis using infercnv software, revealing significant CNV variations in epithelial cells. Subsequently, macrophages were re-clustered into 5 clusters, and their biological significance and functional features were assessed. CellChat analysis elucidated potential interactions between macrophage subtypes and BRCA cells, primarily through SPP1-CD44 and LGALS9-CD44 signaling networks. Additionally, CytoTRACE and Monocle were employed to analyze cellular plasticity and differentiation trajectories of macrophage subtypes. Furthermore, efferocytosis-related gene set scoring, transcription factor analysis, and risk score development were conducted, followed by immune infiltration and tumor mutation burden analysis, revealing increased immune infiltration and higher TMB levels in the high-risk group. These findings offer crucial insights into the interaction mechanisms of immune cells and macrophage subtypes within the BRCA tumor microenvironment, aiding in the understanding of tumor progression and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61362329e16982a536f86bb6af1a06a1e1ff5d77" target='_blank'>
              Mapping immune cell dynamics and macrophage plasticity in breast cancer tumor microenvironment through single-cell analysis
              </a>
            </td>
          <td>
            Wang Chen, Siyu Zeng, Junyong Zhong, Jian Zou, Yanli Lei, Xiaohan Chen, Qinghua Mei, Qianhua Luo
          </td>
          <td>2025-04-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background Gastric cancer (GC) exhibits high heterogeneity that relies on the oncogenic properties of cancer cells and multicellular interactions in the tumour microenvironment. However, the heterogeneity of GC and their molecular characteristics are still largely unexplored. Methods We employed single‐cell and spatial transcriptomics to comprehensively map the intra‐tumoural heterogeneity within GC. Additionally, in vitro experiments, clinical sample analyses, and patient‐derived organoid models (PDOs) were conducted to validate the key interaction patterns between tumor cells and stromal cells. Results Seven robust meta‐programs (MP1–MP7) in GC were defined with distinct biological significance and spatial distributions. MP3 and MP4 were intimately associated with distinct CD8 T cells skewed toward a cytotoxic or exhaustion state, while MP7, characterised by the highest degree of malignancy, harboured an immune lockdown microenvironment around it and spatially associated with myofibroblasts (myCAFs). Notably, we clarified the interplay between the MP7 and myCAFs, where MP7 induces the chemotactic migration of fibroblasts and promoting their transformation into myCAFs via GDF15/TGFBR2, and in turn, myCAFs‐derived RSPO3 up‐regulates EGR1 to promote the transformation to MP7 in GC cells and human PDOs. Ultimately, the accumulation of myCAFs around MP7 led to fewer infiltration of CD8 T cells, resulting an immune‐deprived microenvironment and the diminished efficacy of immunotherapy. Additionally, based on the gene expression signatures of MP7 GC cells, we predicted specific drugs and verified more potent inhibitory effects of Taselisib and Lapatinib for MP7 GC cells than conventional drugs at the same concentration. Conclusion Taken together, these results deepened the understanding of GC heterogeneity and paved the way for novel therapeutic strategies by targeting MP7 GC cells and their interaction loop with myCAFs in GC treatment. Key points Seven robust meta‐programs (MP1‐MP7) were identified in gastric cancer. MP7 was strongly correlated with cancer metastasis and poor survival of gastric cancer patients. MP7 promoted fibroblast transformation into myCAFs via GDF15/TGFBR2, creating an immune lockdown microenvironment. MyCAFs induced MP7 transformation via the RSPO3/EGR1 pathway, promoting gastric cancer cell migration. Taselisib and Lapatinib were potent inhibitors of MP7 GC cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6915321c1b17b2ab2500b930b7287f803a2304b" target='_blank'>
              Deciphering of intra‐tumoural heterogeneity and the interplay between metastasis‐associated meta‐program and myofibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xiong-yan Wu, Zhijian Jin, Baolong Li, Yifan Lu, Junyi Hou, Lizhong Yao, Zhenjia Yu, Qingqing Sang, Beiqin Yu, Jian-fang Li, Chen Li, Chao Yan, Zhenggang Zhu, Kaiwen Tang, Bingya Liu, Liping Su
          </td>
          <td>2025-04-28</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Quiescence (reversible cell-cycle arrest) and senescence (irreversible arrest) are challenging to distinguish due to a lack of specific biomarkers, yet both arise simultaneously after chemotherapy. While senescence suppresses tumors by limiting proliferation and recruiting the immune system, quiescent cancer cells evade future therapies and may resume proliferation. Here, we pair time-lapse imaging of cell-cycle dynamics with single-cell RNA-sequencing after etoposide treatment to differentiate these states, linking heterogeneous cell-cycle phenotypes to the transcriptomic landscape. We identify diverse senescent types (senotypes) and link them to two arrest pathways – a gradual path arising after a standard mitosis-to-G0 transition, and a rarer but direct path driven by a mitotic slip. Using pseudotime trajectory analysis, we find that senescent phenotypes begin to manifest early and gradually along the first trajectory, even in shallow quiescent cells. These data support a model wherein, following chemotherapy, quiescence and senescence exist on a continuum of cell-cycle withdrawal at a transcriptome-wide level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e448f12e8844e81bc7406643cd94547023bd4f4c" target='_blank'>
              Single-cell RNA sequencing reveals distinct senotypes and a quiescence-senescence continuum at the transcriptome level following chemotherapy
              </a>
            </td>
          <td>
            Brianna Fernandez, Victor J. Passanisi, Humza M. Ashraf, Sabrina L. Spencer
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78beb6714803e0c6118158e2cf27d989400224a0" target='_blank'>
              Single Cell RNA Sequencing Reveals Gene Expression Continuums Along the Spatial Hierarchy of the Pulmonary Circulation
              </a>
            </td>
          <td>
            S. Sveiven, C. Knutsen, Fabio Zanini, David N Cornfield, C. Alvira
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Neoadjuvant chemo-immunotherapy has significantly improved the treatment landscape for patients with ALK/EGFR wt resectable non-small cell lung cancer (NSCLC), offering novel opportunities for translational and clinical investigations. By leveraging deep molecular profiling through spatial transcriptomics integrated with single-cell RNA-sequencing from public atlases, and serum proteomic profiling, we report a case of a patient with resectable NSCLC and a solitary synchronous brain metastasis showing a complete pathologic response after chemo-immunotherapy, and a durable event-free survival. The deep profiling of the tumor immune microenvironment of both pre-treatment brain metastasis and post-treatment primary lung tumor tissues unveiled a key role of B lymphocytes at different maturation state, spanning from naïve to plasma cells, in mediating tumor eradication. Notably, the formation of several tertiary lymphoid structures in the regression bed of post-treatment primary tumor was observed, suggesting the in situ generation of high-affinity antibody and specific immune memory response. This multimodal approach paves the way for the discovery of novel biomarkers at both tissue and systemic levels, fostering improved patient stratification and guiding clinical decisions on post-surgical treatment escalation or de-escalation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ca3e849d66be330813305765da5566fa9efc94" target='_blank'>
              Spatial transcriptomics highlights B cells as key contributors to a complete and durable response to chemo-immunotherapy in a patient with resectable NSCLC
              </a>
            </td>
          <td>
            Nicla Porciello, F. Gallina, G. Frisullo, Francesca Fusco, L. D’Ambrosio, V. Balzano, F. de Nicola, Paolo Visca, L. Landi, F. Cappuzzo, Paola Nisticò
          </td>
          <td>2025-05-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Oncogenic mutations are widespread in normal human tissues1. Similarly, in murine chimeras, cells carrying an oncogenic lesion contribute normal cells to adult tissues without causing cancer2–4. How lineages that escape cancer via normal development differ from the minority that succumb is unclear. Tumours exhibit characteristic cancer hallmarks; we therefore searched for hallmarks that differentiate cancer-prone lineages from resistant lineages. Here we show that total cell cycle duration (Tc) predicts transformation susceptibility across multiple tumour types. Cancer-prone Rb- and p107-deficient retina (Rb is also known as Rb1 and p107 is also known as Rbl1) exhibited defects in apoptosis, senescence, immune surveillance, angiogenesis, DNA repair, polarity and proliferation. Perturbing the SKP2–p27–CDK2/CDK1 axis could block cancer without affecting these hallmarks. Thus, cancer requires more than the presence of its hallmarks. Notably, every tumour-suppressive mutation that we tested increased Tc, and the Tc of the cell of origin of retinoblastoma cells was half that of resistant lineages. Tc also differentiated the cell of origin in Rb−/− pituitary cancer. In lung, loss of Rb and p53 (also known as Trp53) transforms neuroendocrine cells, whereas KrasG12D or BrafV600E mutations transform alveolar type 2 cells5–7. The shortest Tc consistently identified the cell of origin, regardless of mutation timing. Thus, relative Tc is a hallmark of initiation that distinguishes cancer-prone from cancer-resistant lineages in several settings, explaining how mutated cells escape transformation without inducing apoptosis, senescence or immune surveillance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2877ae2005e375e07a36ea5fb80465ebbd690a1" target='_blank'>
              Cell cycle duration determines oncogenic transformation capacity
              </a>
            </td>
          <td>
            Danian Chen, Suying Lu, Katherine Huang, Joel D Pearson, M. Paçal, Phillipos Peidis, S. McCurdy, Tao Yu, M. Sangwan, Angela Nguyen, Philippe P Monnier, Daniel Schramek, Liang Zhu, David Santamaria, Mariano Barbacid, Nagako Akeno, K. Wikenheiser-Brokamp, Rod Bremner
          </td>
          <td>2025-04-30</td>
          <td>Nature</td>
          <td>2</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afbfafb6e752991df103b290fb3be622c923aa5e" target='_blank'>
              Immune Spatial Organization Predicts Metastasis Risk in Aggressive Localized Prostate Cancer
              </a>
            </td>
          <td>
            David D. Yang, Aya Abdelnaser, Jeremiah Wala, Alexander J Haas, Alfred A Barney, Eddy Saad, Jett Crowdis, Cora A. Ricker, Jihye Park, Martin T King, P. L. Nguyen, T. Choueiri, A. Tewari, Keyan Salari, M. Taplin, Chin-Lee Wu, E. V. Van Allen
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="The tumor microenvironment (TME) significantly influences cancer prognosis and therapeutic outcomes, yet its composition remains highly heterogeneous, and currently, no highly accessible, high-throughput method exists to define it. To address this complexity, the TMEclassifier, a machine-learning tool that classifies cancers into three distinct subtypes: immune Exclusive (IE), immune Suppressive (IS), and immune Activated (IA), is developed. Bulk RNA sequencing categorizes patient samples by TME subtype, and in vivo mouse model validates TME subtype differences and differential responses to immunotherapy. The IE subtype is marked by high stromal cell abundance, associated with aggressive cancer phenotypes. The IS subtype features myeloid-derived suppressor cell infiltration, intensifying immunosuppression. In contrast, the IA subtype, often linked to EBV/MSI, exhibits robust T-cell presence and improved immunotherapy response. Single-cell RNA sequencing is applied to explore TME cellular heterogeneity, and in vivo experiments demonstrate that targeting IL-1 counteracts immunosuppression of IS subtype and markedly improves its responsiveness to immunotherapy. TMEclassifier predictions are validated in this prospective gastric cancer cohort (TIMES-001) and other diverse cohorts. This classifier could effectively stratify patients, guiding personalized immunotherapeutic strategies to enhance precision and overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e59035a4145d3d05ce3897eefa444405fb68dd" target='_blank'>
              Immunotyping the Tumor Microenvironment Reveals Molecular Heterogeneity for Personalized Immunotherapy in Cancer.
              </a>
            </td>
          <td>
            D. Zeng, Yunfang Yu, Wenjun Qiu, Q. Ou, Qianqian Mao, Luyang Jiang, Jianhua Wu, Jiani Wu, Huiyan Luo, Peng Luo, Wenchao Gu, N. Huang, S. Zheng, Shaowei Li, Yonghong Lai, Xiatong Huang, Yiran Fang, Qiongzhi Zhao, R. Zhou, Huiying Sun, Wei Zhang, J. Bin, Yulin Liao, Masami Yamamoto, Tetsuya Tsukamoto, Sachiyo Nomura, Min Shi, Wangjun Liao
          </td>
          <td>2025-05-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf6f030b19fe7349fb879d14d939b39089ac03e9" target='_blank'>
              Treatment Resistant Persister Cells Exploit Macrophage Lipid Metabolism to Sustain Glioblastoma Growth
              </a>
            </td>
          <td>
            Avirup Chakraborty, Changlin Yang, Aryeh Silver, Diana Feier, Guimei Tian, Avinash Pittu, C. V. von Roemeling, Nagheme Thomas, Michael Andrews, Cole Conforti, Olusegun O. Sobanjo, Nyla T. Searl, Miruna Anica, Raquel Mctiernan, Maryam Rahman, Jianping Huang, Jeffrey K Harrison, Duane A Mitchell, Matthew R. Sarkisian, L. Deleyrolle
          </td>
          <td>2025-06-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 e13119


 Background:
 The tumor microenvironment (TME) of primary breast tumors (PTs) and breast-to-liver metastases shows heterogeneous infiltration and interactions between tumor, immune, and stromal cells. Hepatic metastases exhibit a more immunosuppressive TME with fewer cytotoxic T cells (CTLs) relative to PTs, which may contribute to increased tumor growth and resistance to systemic therapies. To investigate the immune cell infiltration and immune-tumor interactions in PT and liver metastases, we utilized whole transcriptome analysis and immunofluorescence-based protein expression as part of spatial TME profiling.
 Methods:
 We identified 13 chemo-naïve PTs and 3 breast-to-liver metastases. Formalin-fixed paraffin-embedded (FFPE) samples were used for hematoxylin and eosin (H&E) staining and spatial transcriptomics (ST) profiling with the 10X Genomics Visium CytAssist and Visium HD platforms. The H&E slides were manually annotated in collaboration with a board-certified pathologist, and ST data were deconvoluted using breast-specific scRNA-sequenced references and canonical marker labeling.
 Results:
 Our analysis of the 13 PTs and 3 breast-to-liver metastases revealed significant heterogeneity in tumor clusters, immune cell infiltration, gene set enrichments, and gene signatures between primary and visceral TMEs. Unsupervised clustering of hormone receptor-positive (HR+) HER2-negative (HER2) samples demonstrated considerable intratumoral and intertumoral variability, with differences in the expression of tumor markers (EPCAM, ER, PR, ERBB2, KRT14, KRT18), hypoxia-related signatures (HIF1a), and pro-angiogenic drivers (VEGF). Breast-to-liver metastases showed higher expression of pathways involved in proliferation, angiogenesis, and anti-apoptosis compared to primary tumors. Hepatic metastases were enriched for HALLMARK gene sets related to proliferation and metabolic pathways, including PI3K/AKT/mTOR, mTORC1 signaling, E2F targets, MYC targets, G2M checkpoint, mitotic spindle, glycolysis, oxidative phosphorylation, and fatty acid metabolism when compared to primary tumors. In contrast, primary tumors were enriched for immune-related pathways (TNF-α, interferon-γ, inflammatory response, IL-6 JAK/STAT3), epithelial-mesenchymal transition (EMT), myogenesis, and angiogenesis gene signatures, which are consistent with their metastatic potential. Cellular profiling of liver lesions revealed higher myeloid suppressor cells (TAMs and MDSCs): CD8
 +
 ratio with shorter cell-cell distances relative to PTs suggesting a key role in immune evasion, tumor progression, and therapeutic resistance.
 Conclusions:
 ST data from primary and metastatic breast cancer samples revealed notable tumor heterogeneity, characterized by highly active metabolic and proliferative pathways, along with a higher abundance of myeloid cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/389d6dee563fb2269c25d3de12654b70850dddf7" target='_blank'>
              Spatial transcriptomics and functional mapping of the suppressive TME in primary breast tumor and breast-to-liver metastases.
              </a>
            </td>
          <td>
            Batul Al-zubeidy, Edgar Gonzalez, A. Baugh, Dominic Zavala, Cheol Park, Matthew Jacobo, Marqus Dela Cruz, Michelle Li, Sabrina Carrel, Elexa Rallos, Michael Press, Evanthia T. Roussos Torres
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Objective To investigate the metabolic remodeling and functional specialization of immune cells within the tumor microenvironment (TME) of bone tumors, including Ewing’s sarcoma, osteosarcoma, and giant cell tumor of bone, through high-resolution single-cell RNA sequencing (scRNA-seq) analysis. Methods Immune cells were isolated from 13 bone tumor samples and profiled via scRNA-seq to delineate cellular compositions, metabolic adaptations, and intercellular communication networks. Differential gene expression analysis, metabolic pathway enrichment, and pseudotime trajectory inference were employed to characterize functional states and differentiation processes of immune cell subsets. Results We identified 12 major immune cell clusters with distinct functional and metabolic characteristics. Naïve T cells exhibited amino acid metabolism-dependent activation potential, whereas NK cells relied on lipid metabolism and the TCA cycle for cytotoxic activity. Macrophage subsets demonstrated functional divergence: C06 macrophages adopted lipid metabolism to facilitate immunosuppression and tissue repair, while C04 macrophages displayed pro-inflammatory characteristics associated with complement activation. Intercellular signaling analysis revealed FN1 as a central regulator of immune coordination, governing cell adhesion, migration, and homeostasis within the TME. Conclusion This study provides novel insights into the metabolic and functional plasticity of immune cells in bone tumor TMEs, underscoring the critical role of metabolic remodeling in immune regulation. Our findings highlight potential therapeutic targets for modulating immune cell function and offering new avenues to improve treatment outcomes for patients with bone tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06346-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e0d19e0f285642428880ab85bf21db7c2bbae62" target='_blank'>
              Single-cell transcriptomics reveals metabolic remodeling and functional specialization in the immune microenvironment of bone tumors
              </a>
            </td>
          <td>
            Jun Chen, Na Cui, Shaokai He, Chun-Yan Xia, Wei-Qing Li
          </td>
          <td>2025-05-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac2fa84e64df92a9b47c54f45baf0d93294fea97" target='_blank'>
              Substrate heterogeneity promotes cancer cell dissemination
              </a>
            </td>
          <td>
            Zuzana Dunajova, Saren Tasciyan, Juraj Májek, J. Merrin, E. Sahai, M. Sixt, É. Hannezo
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="
 e20030


 Background:
 Tumoral heterogeneity has been widely acknowledged to influence clinical outcome of lung adenocarcinoma (LUAD). This study investigated spatial gene profiling of different pathological subtypes of early-stage invasive LUAD with Stereo-seq transcriptomics technique. The tumor microenvironment (TME) was decoded with cellular and molecular events at perspectives of tumoral activities and immune activities.
 Methods:
 We collected spatial RNA profiles of nine Stereo-seq chips (captured area of 1 x1 cm) from 4 invasive LUAD patients. Pathologist identified histological subtypes including lepidic adenocarcinoma (LP), acinar adenocarcinoma (AC) and other invasive adenocarcinoma like papillary adenocarcinoma (IA) on adjacent H&E images. Spatial data matrix was obtained based on pathological annotation accordingly. Spatially resolved gene expression of each subtypes was obtain based on pathological annotation to conduct high resolution analyses. TME functional units were calculated by analysis of cellular neighborhoods (CNs) based on cell compositions within resolution of bin 100 (window of 50um x 50um). Sub-populations of tumoral cells were stratified by molecular functions (GSEA and GSVA score) and stemness (CytoTRACE).
 Results:
 All pathological subtypes shared the identical 10 heterogeneous CNs and 5 tumoral clusters (TM) with various propotions. Tumor cluster 0 (TM0) and TM3 were identified as the malignant population with significant high stemness and significant high activities of MTORC1 signaling, TGF-BETA signaling, PI3K-ATK signaling and hypoxia. AC and IA were mainly composed of TM0 and TM3. Compared to TM0, TM3 had higher EMT activity. We further identified two immune-related CNs (CN4 and CN8) with significant high GSVA score of Interferon-Alpha response, Interferon-Gamma response and Inflammatory response. Spatial association showed that the lymphoid-cells-dominated CN4 was enriched in LP and IA and nearly absent in AC. Meanwhile, the myeloid-cells-dominated CN8 was enriched in AC. GSEA score showed that immune activity of CN4 was more intense in IA when compared to LP. Notably, TM0 mainly present in CN8, and TM3 mainly present in CN4. Therefore, the LP was speculated to reach an inert and statical status. TME of AC was featured as colocalization of myeloid cells and tumor cells with high stemness. TME of IA was featured as colocalization of lymphoid cells and tumor cells with high EMT activity.
 Conclusions:
 This is the first study to delineate the homogeneity of tumor cells and heterogeneity of immune cells among different pathological types of NSCLC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344f597f0ff9f98416e718719f81b290a2438fe0" target='_blank'>
              Decoding spatial transcriptomics-based heterogeneity of TME in early-stage invasive pulmonary adenocarcinoma.
              </a>
            </td>
          <td>
            Xin Liu, Jingyi Tian, Z. Shang, C. Xiang, Fei Teng, Jiajun Zhang, Yuchen Han
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="We are pleased to present this special section of Small Methods , which highlights the rapidly advancing ﬁelds of single-cell and spatial transcriptomics. Single-cell transcriptomics and spatial transcriptomics have emerged as transformative tools for high-resolution proﬁling of gene expression. Single-cell approaches reveal cellular diversity at unprecedented resolution, while spatial transcriptomics preserves the spatial context of gene activity, enabling precise mapping of tissue architecture. Together, these technologies provide complementary insights into biological systems, uncovering cellular heterogeneity, dynamic interactions, and spatially driven molecular processes across diverse ﬁelds including developmental biology, cancer biology, immunology, and neuroscience. This collection includes 3 reviews and 10 research articles from prominent scientists with their valuable insights. In this special section, we explore the transformative potential of spatial transcriptomics data analysis. Gao et al. (smtd. 2401451) introduce PASSAGE, a deep learning framework for identifying phenotype-associated signatures across heterogeneous spatial slices. Yang et al. (smtd.2400975) propose a novel cell segmentation method UCS, optimized for large-scale subcellular spatial transcriptomics data. Ishaque et al. (smtd. 2401123) present Sainsc, a cell-segmentation-free approach for transcriptome-wide, nanoscale-resolution spatial data. Yuan et al. (smtd.2401199) present a kernel-based strategy to model spatially continuous variations of the tissue microenvironment us-ing a new kernel-based strategyFei et al. (smtd.2401056) intro-S">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9aa8e9995463c05e1c6e24838fae6f8d6bcbd7" target='_blank'>
              Unraveling Tissue Complexity Through Single‐Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shiquan Sun, Chaoyong Yang, Lulu Shang, Rong Fan
          </td>
          <td>2025-05-01</td>
          <td>Small Methods</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef66ca808cfa075ebf0f9050468daef1d8172daf" target='_blank'>
              Classification of indeterminate and off-target cell types within human kidney organoid differentiation
              </a>
            </td>
          <td>
            Sean B. Wilson, J. Vanslambrouck, Angela Murphy, Drew R Neavin, Joseph E. Powell, S. Howden, Melissa H. Little
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Aging leads to functional decline across tissues, often accompanied by profound changes in cellular composition and cell-intrinsic molecular states. However, a comprehensive catalog of how the population of individual cell types change with age and the associated epigenomic dynamics is lacking. Here, we constructed a single-cell chromatin accessibility atlas consisting of ∼7 million cells from 21 tissue types spanning three age groups in both sexes. This dataset revealed 536 main cell types and 1,828 finer-grained subtypes, defined by unique chromatin accessibility landscapes at ∼1.3 million cis-regulatory elements. We observed widespread remodeling of immune lineages, with increases in plasma cells and macrophages, and depletion of T and B cell progenitors. Additionally, non-immune cell populations, including kidney podocytes, ovary granulosa cells, muscle tenocytes and lung aerocytes, showed marked reductions with age. Meanwhile, many subtypes changed synchronously across multiple organs, underscoring the potential influence of systemic inflammatory signals or hormonal cues. At the molecular level, aging was marked by thousands of differentially accessible regions, with the most concordant changes shared across cell types linked to genes related to inflammation or development. Putative upstream factors, such as intrinsic shifts in transcription factor usages and extrinsic cytokine signatures, were identified. Notably, around 40% of aging-associated main cell types and subtypes showed sex-dependent differences, with tens of thousands of chromatin accessibility peaks altered exclusively in one sex. Together, these findings present a comprehensive framework of how aging reshapes the chromatin landscape and cellular composition across diverse tissues, offering a comprehensive resource for understanding the molecular and cellular programs underlying aging and supporting the exploration of targeted therapeutic strategies to address age-related dysfunction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63b5f446430c3fb12a56cac7a496342a8a29945" target='_blank'>
              Organism-wide cellular dynamics and epigenomic remodeling in mammalian aging
              </a>
            </td>
          <td>
            Ziyu Lu, Zehao Zhang, Zihan Xu, Abdulraouf Abdulraouf, Wei Zhou, Junyue Cao
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) presents substantial heterogeneity and unpredictability in its progression. Despite therapeutic advancements, mortality from advanced PCa remains a significant challenge. Understanding the intercellular communication within the tumor microenvironment (TME) is critical for uncovering mechanisms driving tumorigenesis and identifying novel therapeutic targets. Methods We employed an integrative approach combining bulk RNA sequencing, single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics to investigate interactions between FAP+ fibroblasts and tumor-associated macrophages in PCa. Key findings were validated using immunohistochemical and immunofluorescence staining techniques. Results Analysis of 23,519 scRNA-seq data from 23 prostate samples revealed a pronounced accumulation of FAP+ fibroblasts in tumor tissues. Spatial transcriptomics and bulk RNA sequencing demonstrated strong associations between FAP+ fibroblasts and SPP1+ macrophages. Notably, tumor-specific intercellular signaling pathways, such as CSF1/CSF1R and CXCL/ACKR1, were identified, highlighting their potential role in fostering an immunosuppressive TME. Conclusion Our findings unveil a distinct pattern of crosstalk between FAP+ fibroblasts and SPP1+ macrophages in PCa, shedding light on potential therapeutic targets for advanced PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d000b5dbafb928d866718b2a6f1d1452be0aec5" target='_blank'>
              Intercellular communication between FAP+ fibroblasts and SPP1+ macrophages in prostate cancer via multi-omics
              </a>
            </td>
          <td>
            Tingting Wu, Xinyu Li, Fei Zheng, Hanchao Liu, Yang Yu
          </td>
          <td>2025-05-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9faa1273d62a3ded4dd4ed42a59e3c9f4c050ab5" target='_blank'>
              Spatial omics technology potentially promotes the progress of tumor immunotherapy.
              </a>
            </td>
          <td>
            Zhen Lan, Yuanyuan Yang, Lingling Li, Chaoguan Wang, Zhenqiang Sun, Qiming Wang, Yang Liu
          </td>
          <td>2025-06-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common malignancies worldwide, and its complex pathogenesis and significant tumor cell heterogeneity remain major challenges. With the rapid development of single-cell sequencing technology, we can now delve deeper into the cellular composition and dynamic changes within the tumor microenvironment, revealing cellular interactions and their potential roles in tumorigenesis. Method In this study, we systematically analyzed comprehensive single-cell RNA sequencing data from 25 colorectal cancer and 10 adjacent normal tissue samples. We explored the characteristics and biological significance of tumor cell subpopulations, performed quality control, dimensionality reduction, and cell type identification, and further investigated epithelial cell copy number variations, cell communication, and pseudotime analysis. Subsequently, Boruta feature selection algorithm was combined to identify prognosis related genes. The expression patterns, clinical significance and biological effects of PRSS22 were validated in vitro. Results Our analysis found an epithelial cell subcluster with high expression of PRSS22 exhibited high proliferation and migration abilities, and it was also associated with the dysregulated immune microenvironment. After further experimental verification, we proved the high expression patterns and clinical significance of PRSS22. Downregulation of PRSS22 in CRC cells resulted in a reduction of proliferation, migration and invasion. Conclusion Our study has identified a cell subcluster that is closely linked to progression, immune dysregulation and prognosis in CRC, and we have also identified PRSS22 as its hub gene that has great potential to become a new immunotherapeutic targets target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background The dynamics of single-nucleotide variants (SNVs) play a critical role in understanding tumor development, yet their influence on shaping tumor microenvironments remains largely unexplored. Spatial transcriptomics offers an opportunity to map SNVs within the tumor context, potentially uncovering new insights into tumor microenvironment dynamics. Results This study developed SpatialSNV for identifying effective SNVs across tumor sections using multiple spatial transcriptomics platforms. The analysis revealed that SNVs reflect regional tumor evolutionary traces and extend beyond RNA expression changes. The tumor margins exhibited a distinct mutational profile, with novel SNVs diminishing in a distance-dependent manner from the tumor boundary. These mutations were significantly linked to inflammatory and hypoxic microenvironments. Furthermore, spatially correlated SNV groups were identified, exhibiting distinct spatial patterns and implicating specific roles in tumor–immune system crosstalk. Among these, critical SNVs such as S100A11L40P in colorectal cancer were identified as tumor region–specific mutations. This mutation, located within exonic nonsynonymous regions, may produce neoantigens presented by HLAs, marking it as a potential therapeutic target. Conclusions SpatialSNV represents a promising framework for unraveling the mechanisms underlying tumor–immune crosstalk within the tumor microenvironment by leveraging spatial transcriptomics and SNV-based tissue domain characterization. This approach is designed to be scalable, integrative, and adaptable, making it accessible to researchers aiming to explore tumor heterogeneity and identify therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32eb52bad2a391e12368299093f1326ca08dc55e" target='_blank'>
              SpatialSNV: A novel method for identifying and analyzing spatially resolved SNVs in tumor microenvironments
              </a>
            </td>
          <td>
            Yi Liu, Fan Zhu, Xinxing Li, Xiangyu Guan, Yong Hou, Yu Feng, Xuan Dong, Young Li
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastatic prostate cancer (mPC) is a clinically and molecularly heterogeneous disease. While there is increasing recognition of diverse tumor phenotypes across patients, less is known about the molecular and phenotypic heterogeneity present within an individual. In this study, we aimed to define the patterns, extent, and consequences of inter- and intra-tumoral heterogeneity in lethal prostate cancer. By combining and integrating in situ tissue-based and sequencing approaches, we analyzed over 630 tumor samples from 52 mPC patients. Our efforts revealed phenotypic heterogeneity at the patient, metastasis, and cellular levels. We observed that intra-patient, inter-tumoral molecular subtype heterogeneity was common in mPC and showed associations with genomic and clinical features. Additionally, cellular proliferation rates varied within a given patient across molecular subtypes and anatomic sites. Single-cell sequencing studies revealed features of morphologically and molecularly divergent tumor cell populations within a single metastatic site. These data provide a deeper insight into the complex patterns of tumoral heterogeneity in mPC with implications for clinical management and the future development of diagnostic and therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb925628a7789a4c45633ccff7c788aadf4bd5ff" target='_blank'>
              Patterns of intra- and inter-tumor phenotypic heterogeneity in lethal prostate cancer.
              </a>
            </td>
          <td>
            Martine P. Roudier, Roman Gulati, Erolcan Sayar, Radhika A Patel, Micah Tratt, Helen Richards, Paloma Cejas, Miguel Muñoz Gomez, Xintao Qiu, Yingtian Xie, B. Hanratty, Samir Zaidi, Jimmy L. Zhao, M. Adil, Chitvan Mittal, Yibai Zhao, R. Dumpit, Ilsa Coleman, J. Low, Thomas W Persse, Patricia Galipeau, John K. Lee, Maria S Tretiakova, Meagan Chambers, F. Vakar‐López, Lawrence D. True, Marie E Perrone, Hung-Ming Lam, Lori Kollath, Chien-Kuang C. Ding, Stephanie Harmon, Heather H. Cheng, Evan Y. Yu, R. Montgomery, Jessica E. Hawley, , Eva Corey, M. Schweizer, Manu Setty, G. Ha, Charles Sawyers, C. Morrissey, Henry Long, Peter S Nelson, Michael C. Haffner
          </td>
          <td>2025-06-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) remains a leading cause of cancer-related mortality, emphasizing the need for novel therapeutic strategies. Decorin (DCN), a chondroitin sulfate proteoglycan (CSPG), has been proposed as a tumor suppressor, yet its precise role in HCC and the tumor microenvironment (TME) remains underexplored. Through integrated analyses of bulk RNA and single-cell RNA sequencing datasets, we identified a distinct tumor stromal subset highly expressing DCN and associated chondroitin sulfate (CS) synthases. Our findings revealed that DCN expression is significantly downregulated in HCC tissue, but upregulated in peri-tumor stroma, where it correlates with better prognosis and reduced capsular invasion. Western blot analysis demonstrated that CS-DCN, the glycosylated form of DCN, plays a dominant role in this context. Single-cell clustering analysis identified a unique stromal subset in HCC characterized by elevated expression of DCN, CSPGs, and CS synthases, associated with extracellular matrix (ECM) remodeling and protective barrier functions. A six-gene DCN-associated signature derived from this subset, including DCN, BGN, SRPX, CHSY3, CHST3, and CHPF, was validated as a prognostic marker for HCC. Furthermore, functional assays demonstrated that CS-DCN significantly inhibited HCC cell proliferation and invasion. Our study highlights the critical role of DCN in HCC TME and provides insights into its therapeutic potential. Modulating CSPG pathways, particularly on CS-DCN-expressing stromal cells, may offer a promising approach for improving HCC treatment and patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03811-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d6876baaea5bb11c0650c2968885e52f83ef1a4" target='_blank'>
              Revealing a novel Decorin-expressing tumor stromal subset in hepatocellular carcinoma via integrative analysis single-cell RNA sequencing
              </a>
            </td>
          <td>
            Chi Hsiao, Wen-Chieh Liao, Ju-Pi Li, Yu-Cheng Chou, Yu-Lun Chou, Jeng-Rong Lin, Chia-Hua Chen, Chiung-Hui Liu
          </td>
          <td>2025-05-26</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumour microenvironment is composed of a complex cellular network involving cancer, stromal and immune cells in dynamic interactions. A large proportion of this network relies on direct physical interactions between cells, which may impact patient responses to clinical therapy. Doublets in scRNA-seq are usually excluded from analysis. However, they may represent directly interacting cells. To decipher the physical interaction landscape in relation to clinical prognosis, we inferred a physical cell–cell interaction (PCI) network from ‘biological’ doublets in a scRNA-seq dataset of approximately 18,000 cells, obtained from 7 treatment-naive ovarian cancer patients. Focusing on cancer-stromal PCIs, we uncovered molecular interaction networks and transcriptional landscapes that stratified patients in respect to their clinical responses to standard therapy. Good responders featured PCIs involving immune cells interacting with other cell types including cancer cells. Poor responders lacked immune cell interactions, but showed a high enrichment of cancer-stromal PCIs. To explore the molecular differences between cancer-stromal PCIs between responders and non-responders, we identified correlating gene signatures. We constructed ligand-receptor interaction networks and identified associated downstream pathways. The reconstruction of gene regulatory networks and trajectory analysis revealed distinct transcription factor (TF) clusters and gene modules that separated doublet cells by clinical outcomes. Our results indicate (i) that transcriptional changes resulting from PCIs predict the response of ovarian cancer patients to standard therapy, (ii) that immune reactivity of the host against the tumour enhances the efficacy of therapy, and (iii) that cancer-stromal cell interaction can have a dual effect either supporting or inhibiting therapy responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641e58c95ba92d526c7831c22b5ab2944de7c20b" target='_blank'>
              Direct cell interactions potentially regulate transcriptional programmes that control the responses of high grade serous ovarian cancer patients to therapy
              </a>
            </td>
          <td>
            S. Hameed, Walter Kolch, Donal J. Brennan, Vadim Zhernovkov
          </td>
          <td>2025-04-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Myofibroblastic cancer-associated fibroblasts (CAF) in tumor stroma serves as an independent poor prognostic indicator, supporting higher stemness in oral cancer; however, the underlying biology is not fully comprehended. Here, we have explored the crucial role of Tunica Interna Endothelial Cell Kinase (Tie2/TEK) signaling in transition and maintenance of myofibroblastic phenotype of CAFs, and as possible link with the poor prognosis of head and neck squamous cell carcinoma (HNSCC) patients. Methods Bulk and single cell RNA-sequencing (scRNAseq) methods and in-depth bioinformatic analysis were applied for CAF and cancer cells co-culture for studying molecular relationships. In vitro 3D-spheroid-forming ability, expression of stemness markers, in vivo tumor formation ability in zebrafish embryo and syngeneic mouse allografts formation was conducted to test stemness, upon targeting CAF-specific Tie2 activity by gene silencing or with small molecule inhibitor. Immunohistochemistry analysis was performed to locate the distribution of Tie2 and αSMA in primary tumors of oral carcinoma. Prognosis in HNSCC patient cohort from The Cancer Genome Atlas (TCGA) study was analysed based on single sample gene set enrichment score (ssGSEA) and Kaplan–Meier analysis. Results Autocrine or exogenous TGFβ-induction in CAF led to the recruitment of histone deacetylase 2 (HDAC2) on the promoter of Tie2-antagonist, Angiopoietin-2 (ANGPT2), resulting in its downregulation, leading to phosphorylation of Tie2 (Y992) and subsequent activation of SRC (Y418). This led to SRC/ROCK mediated αSMA-positive stress-fiber formation with gain of myofibroblast phenotype. The CAF-specific Tie2-signaling was responsible for producing embryonic-like cell state in co-cultured cancer cells; with enhanced tumor initiating ability. Tie2 activity in CAF exerted the dynamic gene expression reprogramming, with the upregulation of ‘cell migration’ and downregulation of ‘protein biosynthesis’ related gene-regulatory-network modules in malignant cells. The AUCell scores calculated for gene signatures derived from these modules showed significant concordance in independently reported scRNAseq studies of HNSCC tumors and significant association with poor prognosis in HNSCC patient cohort. Conclusions CAF-specific Tie2 activity may serve as direct stromal-target against cancer cell plasticity leading to poor prognosis of oral cancer patients. Overall, our work has provided wider applicability of Tie2-specific functions in tumor biology, along with its known role in endothelial cell-specific function. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03405-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/973b044801c6bab58b14790142170e897c97d9cf" target='_blank'>
              Tie2 activity in cancer associated myofibroblasts serves as novel target against reprogramming of cancer cells to embryonic-like cell state and associated poor prognosis in oral carcinoma patients
              </a>
            </td>
          <td>
            Paromita Mitra, Uday Saha, Kingsly Joshua Stephen, Priyanka Prasad, Subhashree Jena, Ankit Kumar Patel, H. Bv, Santosh Kumar Mondal, S. Kurkalang, Sumitava Roy, Arnab Ghosh, Shantanu Saha Roy, J. Das Sarma, N. Biswas, Moulinath Acharya, R. Sharan, P. Arun, M. Jolly, Arindam Maitra, Sandeep Singh
          </td>
          <td>2025-05-10</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 Engineered tumor avatars provide opportunities to identify mechanisms of cancer initiation, progression, and treatment resistance. We previously reported the development of a single-cell bioprinting method, with subcellular resolution, to fabricate 2D tumor microenvironments (TME) replicating an annotated patient biopsy. Here we report the development of 3D tumor avatars that are cell dense, spatially resolved with 60 µm resolution, and heterogeneous containing up to 7 cell types. This approach overcomes current challenges in spheroid/organoid models that are limited in their ability to control the spatial arrangement of cells, geometry, and heterogeneity to dictate specific cell-cell interactions. A microfluidic dispenser (Biopixlar, Fluicell) was used to pattern cells of the ductal carcinoma in situ tumor microenvironment. The dispensed cells were supported by gelatin while they adopted tissue-like morphology and formed junctions to create an aggregated construct. The avatars contained a cancer cell line (MCF7, HCC1143, or MDA-MB-231) surrounded by an epithelial ring of either MCF10A or a combination of primary human mammary epithelial cells and myoepithelial cells. The stromal compartment included primary mammary fibroblasts along with combinations of primary human umbilical vein endothelial cells, mesenchymal stromal cells, cancer associated fibroblasts, and/or macrophages. Prints were cultured for up to two weeks, with or without secondary Cultrex matrix support, live cell imaged, fixed, and stained for markers of proliferation, cell-cell junctions, morphology, and phenotype states. We found that cells can be patterned with high precision, density, and heterogeneity while maintaining viability, proliferation, and migrational capacity. Cells spread to establish their expected morphology and cell-cell junctions with or without secondary Cultrex matrix support. By not including Cultrex, we demonstrate the epithelial cells can stabilize for 24 hours before introducing the cancer into the system. To demonstrate the high spatial precision of our method, a 1.5 mm x 1 mm region of interest was identified from an annotated breast cancer biopsy containing cancer filled mammary ducts, immune infiltrates, and surrounding stromal compartment of fibroblasts and vasculature. We extruded the 2D map 100 µm in the Z direction to create a cell dense tissue containing 7 cell types matching the size of the histological features. To assess function of the engineered avatars, we perturbed TMEs containing MCF7, MCF10A, and fibroblasts with 10 ng/mL transforming growth factor beta 1 (TGF-β1). Within 48 hours we observed greater fibroblast density, matrix remodeling, and MCF10A elongation in the treatment group relative to control. By controllably manipulating the cellular and extracellular environment in a spatially resolved manner we can begin to identify which conditions promote or inhibit cancer initiation and progression beyond what has been possible with conventional in-vivo and in-vitro models.


 Haylie R. Helms, Anthony Tahayeri, Ellen M. Langer, Alexander E. Davies, Luiz E. Bertassoni. Bioprinting 3D spatially resolved tumor avatars to mimic the native tumor microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB161.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e6b61389a616ff13ee13899ada0ff35cea9d234" target='_blank'>
              Abstract LB161: Bioprinting 3D spatially resolved tumor avatars to mimic the native tumor microenvironment
              </a>
            </td>
          <td>
            Haylie R. Helms, A. Tahayeri, Ellen M. Langer, Alexander Davies, L. Bertassoni
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 11508


 Background:
 Leiomyosarcoma (LMS) is a smooth muscle-derived tumor with significant heterogeneity and limited treatment options for recurrent/metastatic disease. A lack of targetable driver mutations and prognostic and predictive biomarkers have hampered the care of patients with LMS. There is a great need to better understand the biology of LMS and develop novel therapeutics. Advances in single cell RNA sequencing (scRNA-seq) have allowed for better understanding of intratumoral heterogeneity in diverse cancer subtypes. However, tissue dissociation during this process leads to loss of spatial context. Spatial gene expression analysis builds upon scRNA-seq and has the potential to yield information about tissue organization, cell-cell interactions, niches, and cell states. To date there have been limited application of spatial transcriptomics to sarcoma.
 Methods:
 We performed single nucleus multiome (snRNA-seq and snATAC-seq) on a cohort of 16 primary, untreated LMS samples including 12 soft tissue (STLMS) and 4 uterine (ULMS) tumors. We then designed a custom 480-gene panel using the differentially expressed genes from clusters identified in snRNA-seq data to be able to identify spatial relationships between these clusters and to assess these clusters on a larger scale. We utilized the 10x Genomics Xenium platform. This was applied to LMS tissue microarrays (TMAs) comprising a total of 326 tissue cores from 127 unique patients. Matched primary and metastatic samples from the same patient were available for 33 patients.
 Results:
 Analysis of scRNAseq data identified 2 distinct subtypes: a dedifferentiated subtype with mesenchymal features (MES) and a differentiated subtype with enrichment of smooth muscle cell markers (SMC). Integration of chromatin accessibility data from snATACseq showed enrichment of nuclear factor I (NFI) transcription factor (TF) motifs in the MES and AP-1 motifs in the SMC group. Whole genome sequencing did not reveal an obvious genomic etiology for these subtypes. Spatial transcriptomics was able to identify these 2 subtypes in a larger cohort of tumors. Consistent with snRNAseq data, we find that most tumors had almost exclusively either MES or SMC cells. We assessed spatial relationships between these subtypes and infiltrating immune cells. This revealed an enrichment in immunosuppressive macrophages and exhausted T cells in MES tumors compared to SMC tumors. Analysis of matched primary and metastatic tumors demonstrated that the subtype (MES or SMC) generally remains consistent between different sites of disease.
 Conclusions:
 We identify 2 novel LMS subtypes (MES and SMC) driven by distinct TFs. Spatial transcriptomic analysis confirmed the presence of these 2 subtypes in a larger cohort and demonstrated that MES tumors are associated with a more immunosuppressive tumor microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/170994843a3a0b5e571cb27c29ebdaf0b414317f" target='_blank'>
              Spatial transcriptomic profiling from over 300 leiomyosarcoma samples.
              </a>
            </td>
          <td>
            Ryan A Denu, Zhao Zheng, Veena Kochat, Yingda Jiang, William I. Padron, D. Ingram, Khalida M Wani, Larissa A Meyer, P. Soliman, R. Ratan, Alexander J. Lazar, E. Keung, Kunal Rai, Elise F Nassif Haddad
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSC) exhibits profound intratumoral heterogeneity, driven by dynamic interactions between malignant cells and the tumour microenvironment (TME). Using consensus non‐negative matrix factorisation (cNMF) on multi‐site HNSC single‐cell transcriptomes, we resolving conserved meta‐programs define cellular ecosystems. Six major epithelial programmes emerged, including a differentiation‐associated programme (Epi_Diff) correlated with SPDEF activity and favourable patient prognosis, and an invasive programme (Epi_pEMT) potentially controlled by TEAD4‐mediated ECM remodelling, exhibiting partial EMT markers (VIM, TGFB1). Compartment‐specific crosstalk analysis revealed Epi_pEMT cells may coordinate with mCAF1 fibroblasts and TAM(SPP1) through COL1A1‐CD44 and SPP1‐CD44 signalling, suggesting potential formation of a pro‐invasive niche. Conversely, Epi_Diff cells may interact with NK/T cells through CEACAM5‐CD8A and CCL5‐ACKR2, and may contribute to inhibit immune infiltration. Multi‐compartment correlation analysis revealed three ecosystem‐level patterns: (1) Inverse association between Epi_Diff and Epi_pEMT (Spearman R = −0.43); (2) Negative correlation between mCAF1 abundance and cCAF frequency (R = −0.48); (3) TAM(SPP1) dominance inversely correlating with both TAM(C1Q) (R = −0.43) and NK/T infiltration (R = −0.36). These axes suggest a potential hierarchical ecology framework where lineage‐specific polarisation and inter‐compartment synergies may collectively govern disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde8be024032928fa86b340cc4ae81b7272248aa" target='_blank'>
              Single‐Cell Profiling Reveals Conserved Differentiation and Partial EMT Programs Orchestrating Ecosystem‐Level Antagonisms in Head and Neck Cancer
              </a>
            </td>
          <td>
            Donghui Jiang, Xiaoguang Wu, Yuanyuan Deng, Xi Yang, Zhiqiang Wang, Yong Tang, Li He, Xiaoguang He
          </td>
          <td>2025-05-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2702678fe4bf9d99269192e19243ce5bd9b25ff1" target='_blank'>
              Patient-Derived Three-Dimensional Lung Tumor Models to Evaluate Response to Immunotherapy
              </a>
            </td>
          <td>
            Kayla F. Goliwas, Kenneth P. Hough, Sruti Sivan, Sierra L. Single, Sameer S. Deshmukh, Joel L. Berry, Maya Khalil, Benjamin Wei, Yanis Boumber, Mohammad Athar, Aakash Desai, S. Ponnazhagan, James M. Donahue, J. Deshane
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="This study explores the inflammatory response observed in pancreata and pancreatic lymph node (pLN) samples obtained throughout the natural history of type 1 diabetes (T1D) including non-diabetic individuals and non-diabetic autoantibody positive individuals with high susceptibility using spatial transcriptomics (ST). Integration of ST with public single-cell RNA sequencing data enabled interrogation of transcriptional alterations in T1D pathogenesis across both tissues and cellular scales. In the T1D pancreas, we observed global upregulation of multiple inflammation-associated transcripts, including regenerating islet-derived (REG) family genes, complement factor 3 (C3), SOD2, and OLFM4, and highlighted cellular candidates potentially contributing to these signatures. Within the T1D pLN, we observed spatially restricted upregulation of lymphotoxin-β (LTB) alongside follicular dendritic cell (FDC)-associated transcripts including FDCSP, CLU, and FCER2. Collectively, these findings highlight distinct inflammation signatures in the pancreas and regional pLN which can help inform the development of future therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d86fa210fd0a0e42f81c1f4633973c64cff969b8" target='_blank'>
              Spatial transcriptomics from pancreas and local draining lymph node tissue reveals a lymphotoxin-β signature in human type 1 diabetes
              </a>
            </td>
          <td>
            Miguel A. Medina-Serpas, Maigan A. Brusko, Gregory J. Golden, Martha Campbell-Thompson, Trevor Rogers, A. Posgai, Eline T. Luning Prak, Chengyang Liu, Klaus H. Kaestner, Ali Naji, Michael R. Betts, Lauren M McIntyre, M. Atkinson, Todd M. Brusko
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Objectives The tumor microenvironment (TME) plays a critical role in the progression of lung adenocarcinoma (LUAD). This study aims to investigate the cellular composition of the TME in LUAD and assess the role of the MDK-NCL signaling pathway. Methods We employed a multi-omics strategy to investigate LUAD, combining single-cell RNA sequencing (scRNA-seq), spatial transcriptomics (ST), and bulk RNA-seq datasets. Publicly available scRNA-seq data and ST data were utilized. scRNA-seq data underwent quality control, dimensionality reduction, and clustering to characterize cell populations and identify malignant epithelial subtypes using the Seurat and inferCNV packages. Spatial transcriptomics data facilitated the identification of distinct tumor niches, while immune infiltration and ligand-receptor interactions were analyzed using MCPcounter and Niches. Experimental validation was performed via real-time PCR and western blotting on paired LUAD and adjacent normal tissue samples. Results scRNA-seq revealed the presence of multiple immune and stromal cell populations, with malignant epithelial cells being subdivided into six clusters. The MDK-NCL axis demonstrated high activity in malignant cells, showing strong interactions with immune and stromal components. Spatial transcriptomics revealed nine distinct tumor niches, with MDK-NCL signaling notably upregulated at the tumor-immune interface, highlighting its role in establishing an immunosuppressive microenvironment. In both the TCGA-LUAD cohort and in-house cohort, MDK and NCL were significantly upregulated at the mRNA and protein levels in tumor samples compared to normal tissues. High MDK-NCL expression in the TCGA-LUAD cohort correlated with increased TMB, MSI, and reduced immune cell infiltration. Elevated levels of immune checkpoint genes, including PD-1 and CTLA-4, in patients with high MDK-NCL expression suggested a potential resistance to immune checkpoint inhibitors. Moreover, patients with high MDK-NCL expression exhibited poorer survival outcomes, underscoring the pathway’s role in tumor progression and immune evasion. Conclusion Our findings reveal that LUAD cells use the MDK-NCL signaling pathway to shape the TME, suppressing immune activity and promoting malignancy in epithelial cells. This study highlights the MDK-NCL axis as a potential therapeutic target for LUAD, particularly for patients with high MDK-NCL expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b567f020873734871af5baa9880f442a2f4775bd" target='_blank'>
              Integrative single-cell and spatial transcriptomics analysis reveals MDK-NCL pathway’s role in shaping the immunosuppressive environment of lung adenocarcinoma
              </a>
            </td>
          <td>
            Yu Fu, Song Li, Yikang Zhao, Xiran Zhang, Xiaolu Mao, Ran Xu
          </td>
          <td>2025-05-06</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) adapt to the tumor microenvironment (TME), either aiding cancer eradication or promoting tumor growth and immune evasion. To manipulate TAMs therapeutically, a deep understanding of their interaction with the TME is essential. This study explores the responsiveness of TMEs to bexmarilimab, a macrophage reprogramming therapy showing clinical benefit in various solid tumors. Methods We exploited a breast cancer patient-derived explant culture (PDEC) model to characterize bexmarilimab responses in both tumor and adjacent cancer-free tissues by RNA sequencing and multiplex cytokine profiling. Using single-cell RNA sequencing, spatial transcriptomics, and conditioned media treatment, we further investigated the effects of Clever-1+ macrophages and TME features on bexmarilimab sensitivity. Results The PDEC model captured key aspects of bexmarilimab’s mode of action and validated a gene signature for determining treatment sensitivity. We identified three distinct responses to bexmarilimab in tumors and adjacent cancer-free tissues, shaped by the local microenvironment and macrophage phenotype, origin, and localization. The inflammatory state of the TME emerged as the primary determinant of response. Immune activation occurred in immunologically cold TMEs lacking late-stage activated TAMs, whereas interferon-regulated TMEs exhibited suppressed inflammation. In cancer-free breast tissue, bexmarilimab activated B cell responses independent of treatment sensitivity in the adjacent tumor. Conclusions These findings reveal the complexity of TAM targeting in cancer and emphasize the need for patient selection to maximize bexmarilimab’s efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e5846340e5a9a1f216ee09c0f97b9fea80d0f55" target='_blank'>
              Macrophage sensitivity to bexmarilimab-induced reprogramming is shaped by the tumor microenvironment
              </a>
            </td>
          <td>
            Jenna H. Rannikko, Rita Turpin, Pia Boström, Reetta Virtakoivu, Chantal Harth, Akira Takeda, Anselm Tamminen, I. Koskivuo, M. Hollmén
          </td>
          <td>2025-05-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in tumor progression, immune evasion, and therapeutic responses. Among its key components, endothelial cells (ECs) are crucial regulators of angiogenesis, immune cell trafficking, and metabolic adaptations. This study integrates single-cell and transcriptomic analyses to identify tumor-specific endothelial cell signatures in hepatocellular carcinoma (HCC) and stratify tumors into three distinct molecular subtypes. These subtypes exhibit unique immune landscapes and biological characteristics, including pathway activation and differential responses to immunotherapy and targeted treatments. Using machine learning, we developed a robust prognostic scoring model to predict patient outcomes and therapy responsiveness, which was validated across independent cohorts. Our findings highlight the critical role of endothelial cells in modulating the TME and underscore the potential of targeting EC-specific molecular features to enhance the efficacy of immunotherapy and optimize personalized cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02543-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/727e8918f7771cb02692376a24c49897f7676656" target='_blank'>
              Unraveling the TEC-associated landscape in hepatocellular carcinoma: a comprehensive study based on multi-omics analyses
              </a>
            </td>
          <td>
            Jianwei Lan, Longhui Xie, Dekun Song, Pengpeng Liu, Quanyan Liu
          </td>
          <td>2025-05-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86e8972816206093759eba24f787234eeba5b95a" target='_blank'>
              Analysis of Microsatellite Instability Intensity in Single-Cell Resolution with scMnT reveals Tumor Heterogeneity in Colorectal Cancer
              </a>
            </td>
          <td>
            Gyumin Park, Jihwan Park
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e19dad7b139ef16ca0124619b3b70027a193368b" target='_blank'>
              Uncovering Convergent Cell State Dynamics Across Divergent Genetic Perturbations Through Single-Cell High-Content CRISPR Screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Andrew Liao, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-05-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Angiosarcoma, a rare and highly aggressive malignancy originating from vascular endothelial cells, is characterized by its rapid progression, high invasiveness, and poor prognosis. Due to the limited understanding of its tumor microenvironment (TME) and the absence of effective treatments, further research is essential to elucidate its pathogenic mechanisms and improve therapeutic strategies. Objective This study aims to characterize the cellular heterogeneity and unique TME of primary breast angiosarcoma using single-cell RNA sequencing (scRNA-seq), to identify potential therapeutic targets and improve clinical outcomes. Methods Tumor samples were obtained from a patient with bilateral primary breast angiosarcoma and two patients with invasive breast cancer. Single-cell RNA sequencing (scRNA-seq) was conducted to capture the transcriptomic profiles of individual cells within the tumor samples. Following stringent quality control, a total of 31,771 cells were analyzed using comprehensive bioinformatics approaches. Cell populations were identified and classified into distinct cell types, and differential gene expression analysis was performed to explore key signaling pathways. Functional enrichment analysis was used to identify pathways related to tumor progression and immune evasion. Additionally, cell-cell communication networks were mapped to understand interactions within the TME, with a focus on pathways that may serve as therapeutic targets. Results The scRNA-seq analysis revealed significant differences in the distribution of perivascular cells, fibroblasts, T cells, endothelial cells, and myeloid cells in breast angiosarcoma compared to invasive breast cancer. Key pathways enriched in angiosarcoma samples included growth factor binding, platelet-derived growth factor binding, and ribosome biogenesis, with abnormal expression of several ribosomal proteins. Notably, genes such as FAT4, KDR, FN1, and KIT were highly expressed in angiosarcoma endothelial cells, correlating with poor prognosis. Cell communication analysis highlighted the CXCL12-CXCR4 axis as a crucial mediator of the TME in angiosarcoma. Conclusion This study provides critical insights into the TME of primary breast angiosarcoma, highlighting potential molecular targets and pathways for therapeutic intervention. These findings may inform the development of more effective treatment strategies for this rare and challenging tumor type. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02022-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61afd778df17e939f7a978e09a799d8e740668a2" target='_blank'>
              Dissecting the tumor microenvironment in primary breast angiosarcoma: insights from single-cell RNA sequencing
              </a>
            </td>
          <td>
            Peikai Ding, Shengbin Pei, Yi Zhai, Zheng Qu, Yazhe Yang, Xiaolong Feng, Qiang Liu, Xiangyu Wang, Wenxiang Zhang, Zhongzhao Wang, Xiangyi Kong, Jing Wang, Yi Fang
          </td>
          <td>2025-06-05</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61913974e99e15ccb9e61a055d93cfcb623fd1c0" target='_blank'>
              Matrix structure and microenvironment dynamics correlate with chemotherapy response in ovarian cancer
              </a>
            </td>
          <td>
            Florian Laforêts, Panoraia Kotantaki, Samar Elorbany, Joseph J Hartlebury, Joash D Joy, Beatrice Malacrida, Rachel C Bryan-Ravenscroft, Chiara Berlato, Erica Di Federico, John F Marshall, R. Manchanda, Wolfgang Jarolimek, Lara Perryman, E. Maniati, Frances R Balkwill
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d13f61b1ac6cc61dea3334671b6a1eb6cf074a1" target='_blank'>
              Spatial μProBe: a correlative multimodal imaging approach for spatial profiling of biological micro-environments
              </a>
            </td>
          <td>
            F. C. Marques, Neashan Mathavan, Nicholas Ohs, Christopher Goenczoel, Jack J. Kendall, Amit Singh, Dilara Yılmaz, Gisela A. Kuhn, F. Schulte, E. Wehrle, Ralph Müller
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a biologically heterogeneous malignancy of clonal plasma cells, often progressing from MGUS or smoldering MM. It causes anemia, bone lesions, and immune dysfunction due to abnormal plasma cell expansion in the bone marrow. Neuroinflammatory and neurotrophic factors may influence MM progression by affecting immune cells and the bone marrow niche. Growing evidence points to a role for neuroimmune regulation in tumor immunity. Despite therapeutic progress, disease heterogeneity and resistance highlight the need for new strategies targeting the tumor microenvironment and neuroimmune axis. Methods This investigation exploited single-cell RNA sequencing (scRNA-seq) to analyze MM and high-risk smoldering multiple myeloma (SMMh) samples, identifying 11 distinct cell types. We examined their transcriptional signatures, stemness, proliferative properties, and metabolic pathways, with particular attention to neuroimmune interactions in the tumor microenvironment. Using trajectory inference tools such as CytoTRACE, Monocle2, and Slingshot, we traced the differentiation paths of MM cell subpopulations and identified key signaling pathways that may influence immune responses and tumor progression. Results The analysis identified four distinct subpopulations of myeloma cells, with the C0 IGLC3+ myeloma cells representing the least differentiated and most proliferative subset. These cells played a critical role in MM progression and may contribute to immune evasion mechanisms. Additionally, receptor-ligand interactions within the tumor microenvironment were identified, which may be influenced by neuroinflammatory and neurotrophic factors. These findings suggest that the nervous system and immune modulation significantly affect tumor biology, highlighting potential therapeutic targets that could be exploited to overcome resistance to conventional therapies. Conclusion This single-cell analysis provided new insights into the cellular diversity and differentiation trajectories in MM, offering a deeper understanding of the complex neuroimmune interactions that drive tumor progression and resistance. By incorporating the role of neuroinflammation and immune modulation, our study suggested novel therapeutic strategies targeting the neuroimmune axis in oncology, ultimately contributing to the development of more effective, personalized treatment approaches for MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb42682593bbacba35775b457e4d73a57648a9" target='_blank'>
              Decoding multiple myeloma: single-cell insights into tumor heterogeneity, immune dynamics, and disease progression
              </a>
            </td>
          <td>
            Zhenzhen Zhao, Zhijie Zhao, Zhiheng Lin, Lu Fan, Zhikai Xiahou, Yujiang Dong, Weiying Bao
          </td>
          <td>2025-05-08</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2fc80562262fc23fc64cea425e831c1c998be8" target='_blank'>
              Functional Spatial Mapping of the Tumour Immune Microenvironment In Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Sam Legg, Emma Wagner, Christopher J. Applebee, Amanda R. Kirane, Julian Padget, Banafshé Larijani
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background Pancreatic cancer (PC) is a highly aggressive pancreatic malignant tumor with poor prognosis due to its complex tumor microenvironment (TME) and metastatic potential. Fibroblasts play an important role in tumor progression and metastasis by remodeling the extracellular matrix (ECM) and influencing the immune response. This study explored migrasome-associated fibroblasts, especially TSPAN4 and ITGA5, as key regulators of pancreatic cancer metastasis, aiming to provide new ideas for therapeutic strategies targeting TME. Methods We employed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to analyze pancreatic cancer tissues. Data from the GEO and TCGA databases were integrated and processed using batch correction techniques. Single-cell data were analyzed with Seurat and Monocle for dimensionality reduction and pseudotime trajectory analysis. Cell communication was assessed using CellCall and CellChat. Spatial transcriptomic analysis was conducted with RCTD and MISTy tools to investigate cell interactions within the TME. Additionally, gene enrichment, deconvolution, and prognostic analyses were performed, alongside experimental validation through siRNA transfection, qRT-PCR, and various functional assays to investigate the role of TSPAN4 in metastasis. Results Our results underscore the critical role of TSPAN4+ fibroblasts in pancreatic cancer. These fibroblasts were predominantly located at the tumor periphery and exhibited elevated migrasome gene expression, which was associated with enhanced ECM remodeling and immune suppression. Intercellular communication analysis revealed that TSPAN4+ fibroblasts engaged in extensive interactions with immune cells, such as macrophages and endothelial cells, facilitating metastasis and immune evasion. Moreover, the high expression of immune checkpoint markers indicated their involvement in modulating the immune response. Conclusion TSPAN4+ fibroblasts are key regulators of pancreatic cancer progression, contributing to metastasis, immune suppression, and ECM remodeling. Targeting these fibroblasts represents a promising therapeutic strategy to improve clinical outcomes and enhance the effectiveness of immunotherapies in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3280d6d2c8de28d9483ff92142917bd9b0c11e4" target='_blank'>
              TSPAN4+ fibroblasts coordinate metastatic niche assembly through migrasome-driven metabolic reprogramming and stromal-immune crosstalk in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Qingwen Hu, Jiali Chen, Yang Liu, Haiqing Chen, Haotian Lai, Lai Jiang, Xuancheng Zhou, Shengke Zhang, Jinbang Huang, Hao Chi, Bo Li, Xiaolin Zhong
          </td>
          <td>2025-05-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="PURPOSE
Immunotherapy has demonstrated promise in small cell lung cancer (SCLC), but certain patients encounter limited benefits, highlighting the need for immunosuppressive biomarkers. Extrachromosomal circular DNA (ecDNA) promotes amplification of MYC-paralogs (MYC, MYCN and MYCL), driving cross-resistance in SCLC. Here, we aim to investigate whether ecDNA-mediated MYC-paralogs amplification (ecMYC+) represents immunosuppressive features in SCLC.


EXPERIMENTAL DESIGN
Bulk RNA sequencing data were retrieved from public database and paraffin-embedded samples. The overexpression and amplification of MYC-paralogs were identified using immunohistochemistry and fluorescence in situ hybridization. Imaging mass cytometry and multiplex immunohistochemistry were used to characterize spatial distribution of tumor immune microenvironment (TIME). The copy number of MYC-paralogs was investigated using real-time polymerase chain reaction. RNA-sequencing and flow cytometry were performed in SCLC cell lines.


RESULTS
The mean copy number of ecDNAs and the frequency of ecMYC+ cell lines were higher in SCLC than that in the other lineages (SCLC 22/47 vs others 15/282). In ecMYC+ SCLC, multiple immune-related pathways were downregulated while nucleotide metabolism processes were upregulated. Inhibition of nucleotide metabolism induced ecDNA elimination, along with activated antigen presenting pathways. Highly dispersed MYC-paralogs amplifications were detected in resected treatment-naïve SCLC samples. Through the resolution of 103,341 cells from 24 pathological regions, we observed higher expression of MKI67, VEGFA, FAP and FOXP3 and reduced T cell infiltration in ecMYC+ samples. Moreover, ecMYC+ samples exhibited elevated cellular neighborhoods dominated by Ki67+ tumors, with reduced spatial interaction with immune cells.


CONCLUSIONS
Extrachromosomal amplification of MYC-paralogs shapes suppressive TIME, identifying potential subgroup of immunotherapy resistant patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202c6e4f5a1bfae884ce6819555bba6dc3de7227" target='_blank'>
              Amplification of extrachromosomal MYC paralogs shapes immunosuppressive tumor microenvironment in small cell lung cancer.
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Haodong Lin, Jiaming Deng, Yan Ju, Xueyan Hu, Jianqi She, Zhijian Liang, Kongxu Dai, Mantang Qiu, Kunkun Sun, Jun Wang, Fan Yang, Jian Chen, Ence Yang, Xiao Li
          </td>
          <td>2025-04-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cervical cancer represents a significant global health challenge, with complex cellular and molecular mechanisms driving its progression from HPV infection to invasive malignancy. This study employed an integrated approach combining single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (stRNA-seq) to comprehensively characterize the tumor microenvironment (TME) across different stages of cervical cancer development. Through analysis of samples from normal cervix, HPV-infected normal cervix, high-grade squamous intraepithelial lesions (HSIL), and invasive cervical cancer, we identified distinct cellular populations and their dynamic changes during disease progression. Our findings revealed significant heterogeneity in immune cell populations, particularly highlighting the role of SPP1+ macrophages that were substantially enriched in cervical cancer compared to precancerous and normal tissues. Cell-cell communication networks and spatial mapping demonstrated that SPP1+ macrophages interact extensively with immune cells through the SPP1-CD44 signaling axis. This interaction contributes to an immunosuppressive microenvironment through modulation of T cell function and promotion of tumor cell survival. Furthermore, high expression of SPP1 correlated with advanced tumor stages and poor overall survival in cervical cancer patients, highlighting its potential as a prognostic biomarker. Our comprehensive characterization of the cellular landscape and intercellular communication networks in cervical cancer progression provides valuable insights for the development of targeted therapeutic strategies aimed at modulating the TME, particularly through disruption of the SPP1-CD44 axis. These findings establish a foundation for more effective personalized approaches to improve clinical outcomes in cervical cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87cbb4dd12be6f1664aacd1df3e0b1efa6fe30d7" target='_blank'>
              Deciphering the cellular and molecular landscape of cervical cancer progression through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Peng Xia, Juanhong Zhou, Rong Shen, Degui Wang
          </td>
          <td>2025-05-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Abstract Purpose: Peritoneal metastases (PM) in colorectal cancer portend a poor prognosis. We sought to elucidate molecular features differentiating primary tumors (PT) from PMs and actionable targets facilitating transcoelomic dissemination and progression. Experimental Design: We performed multiomic profiling of 227 samples from 136 patients, including 56 PTs and 120 synchronous PMs comprising 34 matched PT–PM pairs. Whole-exome and bulk RNA sequencing analyses were conducted to identify underlying genomic aberrations and transcriptomic differences between primary and peritoneal lesions. We spatially characterized the microenvironment of tumor–stroma compartments and studied the roles of stromal phenotypes in promulgating tumorigenesis. Results: Whole-exome sequencing found that genomic alterations and clonality patterns between PTs and PMs remain broadly similar. Transcriptomic profiles, however, suggest a transition as tumors reach the peritoneum toward a more mesenchymal tumor profile and fibrotic tumor microenvironment. Applying spatial profiling, we identify a fibro-collagenous and immune-infiltrated stromal phenotype [stromal cluster (SC) 2] characterized by increased cancer-associated fibroblasts, memory B cells, M2 macrophages, and T-cell exhaustion. These findings were orthogonally validated by multiplex IHC. Patients with SC2 stroma had poorer survival and were characterized by high SERPINE-1 (PAI-1) expression. PMs in patients with SC2 stroma were associated with enriched oncogenic pathways such as TGF-β. PAI-1 inhibition of colorectal cancer PM cell lines with a novel biologic demonstrated reduced IL2–STAT5 and TGF-β pathways and cell death. Conclusions: Our findings unveil distinctive and actionable molecular signatures, offering deeper insights into the intricate cross-talk between tumor cells and stromal microenvironments enabling PM in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e55caa0c96eda1ece0ec721a1f932b608bbc992" target='_blank'>
              Spatial Heterogeneity, Stromal Phenotypes, and Therapeutic Vulnerabilities in Colorectal Cancer Peritoneal Metastasis
              </a>
            </td>
          <td>
            J. C. Ong, Joseph J Zhao, Ying Liu, Supriya Srivastava, D. Chia, Ying En Quek, Xiaonan Fan, Haoran Ma, Kiekyong Huang, T. Sheng, Qiu Xuan Tan, Gillian Ng, J. W. Tan, J. J. Lee, L. Loo, Li Yen Chong, Xue Wen Ong, S. Tay, T. Hagihara, Angie Tan, Craig Ryan Cecil Joseph, Melissa C C Teo, Josephine Hendrikson, C. Chong, Wanyu Guo, C. Chia, Jolene S M Wong, Chin Jin Seo, Mingzhe Cai, Yvonne Tay, Kevin M S Sim, R. Tay, Robert Walsh, M. Guaglio, F. Morano, Ming Teh, Huey Yew Jeffrey Lum, Tony K H Lim, Louis Vermeulen, M. Bijlsma, K. Lenos, S. Klempner, J. Yeong, W. Yong, F. Pietrantonio, P. Tan, R. Sundar
          </td>
          <td>2025-04-29</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Glioblastoma is the most aggressive and lethal cancer of the central nervous system, presenting substantial treatment challenges. The current standard treatment, which includes surgical resection followed by temozolomide and radiation, offers limited success. While immunotherapies, such as immune checkpoint inhibitors, have proven effective in other cancers, they have not demonstrated significant efficacy in GBM. Emerging research highlights the pivotal role of tumor-associated macrophages (TAMs) in supporting tumor growth, fostering treatment resistance, and shaping an immunosuppressive microenvironment. Preclinical studies show promising results for therapies targeting TAMs, suggesting potential in overcoming these barriers. TAMs consist of brain-resident microglia and bone marrow-derived macrophages, both exhibiting diverse phenotypes and functions within the tumor microenvironment. This review delves into the origin, heterogeneity, and functional roles of TAMs in GBM, underscoring their dual roles in tumor promotion and suppression. It also summarizes recent progress in TAM-targeted therapies, which may, in combination with other treatments like immunotherapy, pave the way for more effective and personalized strategies against this aggressive malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f77ce2f24a7dfd7badc0d732e7d82ccdd3d1f004" target='_blank'>
              Exploring tumor-associated macrophages in glioblastoma: from diversity to therapy
              </a>
            </td>
          <td>
            Wenwen Zhao, Zhi Zhang, Mingyuan Xie, Feng Ding, Xiangrong Zheng, Shicheng Sun, Jianyang Du
          </td>
          <td>2025-05-02</td>
          <td>NPJ Precision Oncology</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="A molecular understanding of lung organogenesis requires delineation of the timing and regulation of the cellular transitions that ultimately form and support a surface capable of gas exchange. While the advent of single-cell transcriptomics has allowed for the discovery and identification of transcriptionally distinct cell populations present during lung development, the spatiotemporal dynamics of these transcriptional shifts remain undefined. With imaging-based spatial transcriptomics, we analyzed the gene expression patterns in 17 human infant lungs at varying stages of development and injury, creating a spatial transcriptomic atlas of ∼1.2 million cells. We applied computational clustering approaches to identify shared molecular patterns among this cohort, informing how tissue architecture and molecular spatial relationships are coordinated during development and disrupted in disease. Recognizing that all preterm birth represents an injury to the developing lung, we created a simplified classification scheme that relies upon the routinely collected objective measures of gestational age and life span. Within this framework, we have identified cell type patterns across gestational age and life span variables that would likely be overlooked when using the conventional “disease vs. control” binary comparison. Together, these data represent an open resource for the lung research community, supporting discovery-based inquiry and identification of targetable molecular mechanisms in both normal and arrested human lung development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02b0fe705f634a247d40a92ce22d360cda7da4a3" target='_blank'>
              A spatial transcriptomic atlas of acute neonatal lung injury across development and disease severity
              </a>
            </td>
          <td>
            Saahithi Mallapragada, R. Lyu, Arianna L. Williams-Katek, Brandon K. Fischer, A. Vannan, Niran Hadad, Evan D. Mee, Shawyon P. Shirazi, C. Jetter, N. Negretti, Anne Hilgendorff, Laurie C. Eldredge, Gail H. Deutsch, Davis J. McCarthy, J. Kropski, Jennifer M. S. Sucre, N. Banovich
          </td>
          <td>2025-06-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Basic and translational cancer biology research requires model systems that recapitulate the features of human tumors. While two-dimensional (2D) cell cultures have been foundational and allowed critical advances, they lack the organizational complexity, cellular interactions, and extracellular matrix present in vivo. Mouse models have thus remained the gold standard for studying cancer. In addition to high cost and low throughput, mouse models can also suffer from reduced tumor heterogeneity and species-specific differences. Three-dimensional (3D) culture models have emerged as a key intermediary between 2D cell lines and mouse models, with lower cost and greater flexibility than mouse models and a more accurate representation of the tumor microenvironment than 2D cell lines. In neuroblastoma, an aggressive childhood cancer, 3D models have been applied to study drug responses, cell motility, and tumor–matrix interactions. Recent advances include the integration of immune cells for immunotherapy studies, mesenchymal stromal cells for tumor–stroma interactions, and bioprinted systems to manipulate matrix properties. This review examines the use of 3D culture systems in neuroblastoma, highlighting their advantages and limitations while emphasizing their potential to bridge gaps between in vitro, preclinical, and clinical applications. By improving our understanding of neuroblastoma biology, 3D models hold promise for advancing therapeutic strategies and outcomes in this childhood cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f6d28651a5e98e6b7e3247c16a4fa5871e08763" target='_blank'>
              Three-Dimensional Culture Systems in Neuroblastoma Research
              </a>
            </td>
          <td>
            Piotr Jung, A. Wolpaw
          </td>
          <td>2025-05-08</td>
          <td>Organoids</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="One of the key challenges in defeating advanced tumors is the ability of cancer cells to evade the selective pressure imposed by chemotherapy, targeted therapies, immunotherapy and cellular therapies. Both genetic and epigenetic alterations contribute to the development of resistance, allowing cancer cells to survive initially effective treatments. In this narration, we explore how genetic and epigenetic regulatory mechanisms influence the state of tumor cells and their responsiveness to different therapeutic strategies. We further propose that an altered balance between cell growth and cell death is a fundamental driver of drug resistance. Cell death programs exist in various forms, shaped by cell type, triggering factors, and microenvironmental conditions. These processes are governed by temporal and spatial constraints and appear to be more heterogeneous than previously understood. To capture the intricate interplay between death-inducing signals and survival mechanisms, we introduce the concept of Death-ision. This framework highlights the dynamic nature of cell death regulation, determining whether specific cancer cell clones evade or succumb to therapy. Building on this understanding offers promising strategies to counteract resistant clones and enhance therapeutic efficacy. For instance, combining DNMT inhibitors with immune checkpoint blockade may counteract YAP1-driven resistance or the use of transcriptional CDK inhibitors could prevent or overcome chemotherapy resistance. Death-ision aims to provide a deeper understanding of the diversity and evolution of cell death programs, not only at diagnosis but also throughout disease progression and treatment adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24390bbe04bc2b2ed371e1dae568af6c9f084554" target='_blank'>
              Death-ision: the link between cellular resilience and cancer resistance to treatments
              </a>
            </td>
          <td>
            Gustavo Baldassarre, Ivana L. de la Serna, François M Vallette
          </td>
          <td>2025-05-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocytes (TILs) are a diverse population of immune cells that play a central role in tumor immunity and have emerged as critical mediators in cancer immunotherapy. This review explores the phenotypic and functional diversity of TILs—including CD8+ cytotoxic T cells, CD4+ helper T cells, regulatory T cells, B cells, and natural killer (NK) cells—and their dynamic interactions within the tumor microenvironment (TME). While TILs can drive tumor regression, their activity is often hindered by immune checkpoint signaling, metabolic exhaustion, and stromal exclusion. We highlight TIL recruitment, activation, and polarization mechanisms, focusing on chemokine gradients, endothelial adhesion molecules, and dendritic cell-mediated priming. Special emphasis is placed on preclinical models that evaluate TIL function, including 3D tumor spheroids, organoid co-cultures, syngeneic mouse models, and humanized systems. These provide valuable platforms for optimizing TIL-based therapies. Furthermore, we examine the prognostic and predictive value of TILs across cancer types, their role in adoptive cell therapy, and the challenges of translating preclinical success into clinical efficacy. Emerging technologies such as single-cell sequencing, neoantigen prediction, and biomaterial platforms are transforming our understanding of TIL biology and enhancing their therapeutic potential. Innovative strategies—ranging from genetic engineering and combination therapies to targeted modulation of the TME—are being developed to overcome resistance mechanisms and improve TIL persistence, infiltration, and cytotoxicity. This review integrates current advances in TIL research and therapy, offering a comprehensive foundation for future clinical translation. TILs hold significant promise as both biomarkers and therapeutic agents, and with continued innovation, they are poised to become a cornerstone of personalized cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f0d6ebefadb43d85bc098a73c546761ba747a24" target='_blank'>
              Tumor-infiltrating lymphocytes in cancer immunotherapy: from chemotactic recruitment to translational modeling
              </a>
            </td>
          <td>
            F. Kraja, Vladimir Jurišić, Altijana Hromić-Jahjefendić, Nafsika Rossopoulou, T. Katsila, K. Mirjačić Martinović, Javier De Las Rivas, Carmen Cristina Diaconu, Árpád Szöőr
          </td>
          <td>2025-05-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3294fa089aae1c04ac4218c49906b4bc0f6ea71" target='_blank'>
              NF2 loss malignantly transforms human pancreatic acinar cells and enhances cell fitness under environmental stress
              </a>
            </td>
          <td>
            Yi Xu, Michael H Nipper, Angel A. Dominguez, Chenhui He, Francis E Sharkey, Sajid Khan, Han Xu, Daohong Zhou, Lei Zheng, Yu Luan, Jun Liu, Pei Wang
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background The current immunotherapeutic strategies yield limited therapeutic benefits in patients with castration-resistant prostate cancer (CRPC) due to its immunologically “cold” tumor milieu. Pericytes play a pivotal role in facilitating metastatic dissemination and modulating immune response in malignancies. Our investigation is designed to decipher the biological effects of pericytes on CRPC and their interactions with the tumor microenvironment. Methods We leveraged single-cell transcriptomics and immunofluorescence staining to ascertain the presence and spatial distribution of pericytes in prostate cancer (PCa). Subsequently, we thoroughly delineated the phenotypic and functional characteristics of the CRPC-pericytes subpopulation. The clinical and prognostic significance of CRPC-pericytes was assessed by employing several bulk RNA-seq and microarray datasets. Additionally, we examined the association between the abundance of CRPC-pericytes and immunological features in the PCa microenvironment. Furthermore, the RM-1 subcutaneous tumor model was leveraged to assess the synergistic efficacy of platelet-derived growth factor (PDGF) signaling inhibition in conjunction with immunotherapeutic interventions. Results We discerned pericytes according to their marker genes and observed the α-SMA-positive pericytes encircling the vasculature in PCa and adjacent normal tissues. In the CRPC-pericytes subpopulation, a pronounced upregulation of PDGF signaling and angiogenesis was observed, whereas antitumor immunity-related pathways were suppressed. In addition, CRPC-pericytes displayed notably enhanced interactions with endothelial cells, fibroblasts, and myeloid cells, compared to PCa-pericytes. Patients with elevated prevalence of CRPC-pericytes exhibited notably reduced recurrence-free survival and unresponsiveness to immunotherapeutic interventions. Moreover, CRPC-pericytes were positively associated with immunosuppressive properties of the TME. Notably, combinatorial application of PDGFR inhibitor and anti-PD-1 therapy elicited substantial synergistic antitumor effects in murine PCa models. Conclusion Our investigation uncovers a CRPC-pericytes subpopulation implicated in cancer progression and immunosuppression, suggesting that therapies targeting the phenotypic transition of pericytes could act synergistically with immunotherapeutic regimens to improve survival rates in CRPC. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03838-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0a533b75dfdf1ea7f39068eba94e6ca4759cf00" target='_blank'>
              Pericytes in castration-resistant prostate cancer associated with disease progression and immunotherapy response: insights from single-cell analysis
              </a>
            </td>
          <td>
            Yifeng Qiu, Yuhan Wang, Jiahe Liu, Kai Sun, Shigeo Horie, Zhitong Bing, Qi Hou
          </td>
          <td>2025-06-03</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Microwave thermochemotherapy (MTC) has been applied to treat lip squamous cell carcinoma (LSCC), but a deeper understanding of its therapeutic mechanisms and molecular biology is needed. To address this, we used single-cell transcriptomics (scRNA-seq) and spatial transcriptomics (ST) to highlight the pivotal role of tumor-associated neutrophils (TANs) among tumor-infiltrating immune cells and their therapeutic response to MTC. MNDA+ TANs with anti-tumor activity (N1-phenotype) are found to be abundantly infiltrated by MTC with benefit of increased blood perfusion, and these TANs are characterized by enhanced cytotoxicity, ameliorated hypoxia, and upregulated IL1B, activating T&NK cells and fibroblasts via IL1B-IL1R. In this highly anti-tumor immunogenic and hypoxia-reversed microenvironment under MTC, fibroblasts accumulated in the tumor front (TF) can recruit N1-TANs via CXCL2-CXCR2 and clear N2-TANs (pro-tumor phenotype) via CXCL12-CXCR4, which results in the aggregation of N1-TANs and extracellular matrix (ECM) deposition. In addition, we construct an N1-TANs marker, MX2, which positively correlates with better prognosis in LSCC patients, and employ deep learning techniques to predict expression of MX2 from hematoxylin-eosin (H&E)-stained images so as to conveniently guide decision making in clinical practice. Collectively, our findings demonstrate that the N1-TANs/fibroblasts defense wall formed in response to MTC effectively combat LSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2560362b63b6e3eb203f753fb2eef7cc7ed8926f" target='_blank'>
              Single-cell and spatial transcriptomics reveals an anti-tumor neutrophil subgroup in microwave thermochemotherapy-treated lip cancer
              </a>
            </td>
          <td>
            Bingjun Chen, Huayang Fan, Xin Pang, Zeliang Shen, Rui Gao, Haofan Wang, Zhenwei Yu, Tianjiao Li, Mao Li, Yaling Tang, Xinhua Liang
          </td>
          <td>2025-05-13</td>
          <td>International Journal of Oral Science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The tumor microenvironment plays a crucial role in shaping the tumor phenotype, yet replicating its complexity in vitro remains challenging. Traditional two-dimensional culture models lack physiologic relevance, and three-dimensional models often fail to fully capture native extracellular matrix (ECM) composition and cellular heterogeneity. In this issue of Cancer Research, Buckenmeyer and colleagues bridged this gap by integrating decellularized porcine-derived small intestinal submucosa ECM into monocellular spheroids, resulting in the formation of "MatriSpheres" with in vivo-like cancer cell heterogeneity. Although small intestinal submucosa ECM proved to be beneficial, several routes remain unexplored, such as incorporating other cell types (immune cells and cancer-associated fibroblasts) or species-, organ- or pathology-specific ECM or leveraging patient-derived tumor material. Pursuing these avenues of investigation to further develop Self-Matrix-Assembly to Recapitulate a Tumor (SMART) three-dimensional models could provide a powerful tool for cancer research, drug discovery, and personalized medicine. See related article by Buckenmeyer et al., p. 1577.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c194b106b13ab4f43c268e1b9699aab011f64" target='_blank'>
              Launching SMART 3D Cancer Models.
              </a>
            </td>
          <td>
            Sam Ernst, O. De Wever
          </td>
          <td>2025-05-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) largely consists of lung squamous (LUSC) and lung adenocarcinoma (LUAD). Alterations in the TRP53 and PTEN tumor suppressors are common in both subtypes, but their relationship with SOX2 is poorly understood. We deleted Trp53 or Pten in a C57BL/6J-Sox2hi;Nkx2-1-/-;Lkb1-/- (SNL) genetic background and generated a highly metastatic LUSC cell line (LN2A; derived from a Sox2hi mouse model, followed by Trp53, Pten, and Cdkn2a deletion). Histologic and single-cell RNAseq analyses corroborated that SNL mice developed mixed tumors with both LUAD and LUSC histopathology while SNL-Trp53 and SNL-Pten mice developed LUAD and LN2A tumors retained LUSC morphology. Compared with SNL mice, additional loss of Trp53 or Pten resulted in significantly reduced survival, increased tumor burden and altered tumor mucin composition. We identified a sub-cluster of CD38+ tumor-associated inflammatory monocytes in the LN2A model that significantly enriched for activation of the classical and alternative complement pathways. Complement Factor B (CFB) is associated with poor survival in LUSC patients, and we observed the LN2A model had significantly improved survival on a Cfb-/- background. Our findings demonstrate a cooperative role of Trp53 and Pten tumor suppressors in Sox2-mediated NSCLC tumor progression, mucin production, and remodeling of the immune tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7a0ddd6a5df2598723f17fca05f2a9908bc3e24" target='_blank'>
              Tumor suppressors in Sox2-mediated lung cancers promote distinct cell intrinsic and immunologic remodeling.
              </a>
            </td>
          <td>
            Nisitha Sengottuvel, Kristina M. Whately, Jennifer L Modliszewski, Rani S Sellers, W. D. Green, Weida Gong, Allison T. Woods, Eric W Livingston, Katerina D Fagan-Solis, Gabrielle Cannon, Lincy Edatt, Hong Yuan, Aaron C. Chack, Yazmin Sanchez, Katherine Zhou, Alyaa Dawoud, Jarred M. Green, Virginia Godfrey, J. Milner, Gaorav P Gupta, Chad V. Pecot
          </td>
          <td>2025-05-06</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) immunotherapy has been revolutionized by immune checkpoint inhibitors (ICIs), yet response heterogeneity persists due to dynamic tumor-immune interactions. This review summarizes recent studies in understanding tumor-infiltrating lymphocyte (TIL) biology, highlighting CD8+ cytotoxic T cells and regulatory T cells (Tregs) as pivotal regulators of immune surveillance and suppression. We summarize emerging biomarkers such as TCR clonality, spatial distribution of tumor-infiltrating lymphocytes (TILs), and exhaustion markers including PD-1, TCF1, and TIM-3, which predict immune checkpoint inhibitor (ICI) efficacy beyond PD-L1 expression. This review specifically describes radiotherapy-induced immunogenic remodeling and peripheral T cell dynamics as innovative strategies to monitor immune response and resistance mechanisms. By integrating results from single-cell omics and AI-driven spatial analysis, we propose multidimensional frameworks of TIL in NSCLC to overcome resistance and optimize immunotherapy combinations. These insights collectively advance NSCLC immunotherapy toward precision modulation of the tumor immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/856dfef659f572b16bf42e7166c3799f87d37332" target='_blank'>
              T lymphocyte heterogeneity in NSCLC: implications for biomarker development and therapeutic innovation
              </a>
            </td>
          <td>
            Yu Liu, Denghui Qin, Jiejun Fu
          </td>
          <td>2025-05-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Disease modeling is vital for our understanding of disease mechanisms and for developing new therapeutic strategies. Accurately modeling the intact tumor microenvironment (TME) is increasingly recognized as essential for gaining insights into cancer biology and therapeutic response. Preclinical mouse models have provided utility for studying the evolving TME, but these models are costly and can lead to animal suffering and the discontinuation of drug investigations. To address these limitations, particularly in hepatocellular carcinoma (﻿HCC), we have developed an ex vivo model using tumor precision-cut slices (TPCS) derived from orthotopic liver tumors. Methods: Murine HCC tumors were generated via intrahepatic injection of Hep-53.4 cells, providing a source of tumor tissue for TPCS generation. Subsequent scaling to a 96-well format and modification to include a secreted luciferase enabled longitudinal ex vivo screening of 26 drugs applied at 2 doses over an 8-day period, using just 5 tumors. One drug identified in the screen, salinomycin, was then validated in vivo via intraperitoneal injection of mice with orthotopic liver tumors. Results: Histological characterization determined that TPCS maintain the architecture, cellular complexity, and drug responsiveness of the original HCC-TME under simplified culture conditions that preserve viability and metabolic activity. In addition to typical HCC therapies, sorafenib and anti-PD1 immunotherapy, the screen identified 2 drugs as potent anticancer agents capable of impacting the viability of TPCS: salinomycin and rottlerin. Salinomycin was further validated in vivo, significantly reducing tumor burden without evidence of toxicity. Conclusions: We present a 3Rs (Reduction, Refinement, Replacement) approach for studying HCC biology and performing 96-well-scale drug screening within an intact, metabolically active TME, offering a more ethical and effective platform for drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0fc9ab2a0c87013cf6a5b53d9f8f813f5001d3b" target='_blank'>
              Precision-cut tumor slices for modeling hepatocellular carcinoma enable at-scale drug screening
              </a>
            </td>
          <td>
            Amy L. Collins, Keara Kirkness, Erik Ramon‐Gil, Eleni Tzortzopoulou, D. Geh, Jack Dishington, Eleanor Graham, Rhys Muir, Rainie Cameron, Saimir Luli, Eman Khurram, Daniel Storey, Hannah L. Paish, Glyn Nelson, David McDonald, Andrew Filby, Lee A Borthwick, Fiona Oakley, Derek A Mann, J. Leslie
          </td>
          <td>2025-05-16</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Emerging spatial profiling technologies have revolutionized our understanding of how tissue architecture shapes disease progression, yet the contribution of cellular diversity remains underexplored. Here, Ding and colleagues introduce multiomics and ecological spatial analysis (MESA), an ecology-inspired framework that integrates spatial and single-cell expression data to quantify tissue diversity across multiple scales. MESA both identifies distinct cellular neighborhoods and computes a variety of diversity metrics alongside the identification of diversity "hotspots". Applied to human tonsil tissue, MESA revealed previously undetected germinal center organization, while in spleen tissue of a murine lupus model, MESA highlights increasing cellular diversity with disease progression. Importantly, diversity hotspots do not correspond to conventional compartments identified by existing methods, presenting an orthogonal metric of spatial organization. In colorectal cancer, MESA's diversity metrics outperformed established subtypes at predicting patient survival, while in hepatocellular carcinoma, multi-omic integration identified significantly more ligand-receptor interactions between immune cells compared to single-modality analysis. This work establishes cellular diversity within tissues as a critical correlate of disease progression and underscores the value of multi-omic integration in spatial biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eb774cd8ef924c70b07f117ccf0af3bd80ea8f6" target='_blank'>
              Cells keep diverse company in diseased tissues.
              </a>
            </td>
          <td>
            Kieran R. Campbell, A. Goeva
          </td>
          <td>2025-05-16</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 There are limited studies regarding the tumor microenvironment of aggressive prostatic adenocarcinoma (PCa), at the single-cell level. In addition, there is a paucity of RNAseq data regarding gene expression of immune infiltrates and cancer-associated fibroblasts adjacent to PCa. The Visium-HD Spatial RNAseq platform sequences formalin-fixed paraffin embedded (FFPE) tissue using spatial barcodes at up to 2-micron pixel resolution, which achieves single-cell resolution.



 We performed Visium-HD Spatial RNAseq analysis on FFPE blocks from radical prostatectomy specimens of 12 patients of which six had lymph node metastases (LNM) and six did not. All tissues analyzed by spatial RNAseq included foci of prostatic stroma adjacent to PCa and separate foci of benign glands and adjacent stroma.



 Visium-HD RNAseq could readily identify distinct tumor microenvironments such as benign glands, stroma, Gleason IV tumor, Gleason V tumor, and Tumor Necrosis. We observed that patients with LNM had more significant enrichment for gene signatures associated with inflammation and immune responses and high numbers of naïve regulatory T cells (Naïve Treg) near the invasive front of the aggressive tumor foci. The cancer-associated stroma in metastatic patients was enriched in expression of antigen presenting genes, but also in immunosuppressive gene signatures. We also observed tumor cells with neuroendocrine prostate cancer gene expression patterns in these patients who had not yet received any androgen-deprivation therapy. Tumors of patients with LNM were also more enriched in proliferative gene signatures such as E2F Targets, G2M Checkpoint genes, and Mitotic spindle genes. Finally, tumors of patients with LNM were enriched for gene expression indicative of endogenous cholesterol biosynthesis, whereas non-metastatic tumors were not.



 High-definition spatial RNAseq analysis with the Visium-HD platform enables automated identification of biologically and pathologically distinct regions of the PCa tumor microenvironment. Tumors of patients with metastatic disease had greater expression of immune gene sets, proliferative gene sets, and genes associated with cholesterol metabolism.



 Rakesh Shiradkar, Tilak Pathak, Cynthia Winham, Ismaheel O. Lawal, Olayinka A. Abiodun-Ojo, Martin G. Sanda, David M. Schuster, Adeboye O. Osunkoya, Carlos S. Moreno. Visium-HD spatial RNAseq analysis of aggressive prostatic adenocarcinoma identifies distinct RNA signatures in metastatic and non-metastatic disease [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB466.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af476913bede9308f97c975da143461f67e3a4d" target='_blank'>
              Abstract LB466: Visium-HD spatial RNAseq analysis of aggressive prostatic adenocarcinoma identifies distinct RNA signatures in metastatic and non-metastatic disease
              </a>
            </td>
          <td>
            Rakesh Shiradkar, Tilak Pathak, Cynthia L. Winham, I. Lawal, O. Abiodun-Ojo, Martin G Sanda, David M. Schuster, A. Osunkoya, Carlos S. Moreno
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822c59156ea3b1fc22260f9e8a6f1db29bc4fc3a" target='_blank'>
              Pancreatic cancer patient-derived organoids capture therapy response and tumor evolution
              </a>
            </td>
          <td>
            J. Gout, L. Perkhofer, Jessica Lindenmayer, Y. J. Resheq, J. Schwab, J. Kraus, E. Roger, Dharini Srinivasan, E. Zimmer, B. Mayer, Ralf Marienfeld, N. Gaisa, Julia K. Kühlwein, Mark Hänle, T. Ettrich, K. Danzer, Alica K Beutel, Bo Kong, Christoph Michalski, André L Mihaljević, N. Rahbari, Marko Kornmann, Hans A. Kestler, Thomas Seufferlein, Alexander Kleger
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Cancer drug resistance is multi-factorial, driven by heritable (epi)genetic changes but also phenotypic plasticity. Here, we dissected drivers of resistance by perturbing colorectal cancer patient-derived organoids longitudinally with drugs in sequence. Combined longitudinal lineage tracking, single cell multi-omics analysis, evolutionary modelling, and machine learning revealed that different targeted drugs select for distinct subclones, supporting rationally designed drug sequences. The cellular memory of drug resistance was encoded as a heritable epigenetic configuration from which multiple transcriptional programs could run, supporting a one-to-many (epi)genotype-to-phenotype map that explains how clonal expansions and plasticity manifest together. This epigenetic landscape may ensure drug resistant subclones can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding for their selective advantage. Chemotherapy resistance was instead entirely driven by transient phenotypic plasticity rather than stable clonal selection. Inducing further chromosomal instability before drug application changed clonal evolution but not convergent transcriptional programs. Collectively, these data show how genetic and epigenetic alterations are selected to engender a "permissive epigenome" that enables phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83269ee41c4604edc0996e3b22f4bf8776d7c6" target='_blank'>
              Epigenetic Heritability of Cell Plasticity Drives Cancer Drug Resistance through a One-to-Many Genotype-to-Phenotype Paradigm.
              </a>
            </td>
          <td>
            É. Oliveira, Salvatore Milite, J. Fernández-Mateos, George D. Cresswell, Erika Yara-Romero, G. Vlachogiannis, Bingjie Chen, Chela James, Lucrezia Patruno, Gianluca Ascolani, Ahmet Acar, Timon Heide, Inma Spiteri, Alex Graudenzi, G. Caravagna, Andrea Bertotti, Trevor A. Graham, Luca Magnani, Nicola Valeri, A. Sottoriva
          </td>
          <td>2025-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e4a4169aaf7d2eba5de69f52ef7120fb04fd31" target='_blank'>
              Personalized CRISPR Knock-In Cytokine Gene Therapy to Remodel the Tumor Microenvironment and Enhance CAR T Cell Therapy in Solid Tumors
              </a>
            </td>
          <td>
            Michael Launspach, Julia Macos, Shoaib Afzal, Janik Hohmann, Stefanie Beez, Maximilian Pilgram, Emily Ohlendorf, Andrii Yevchuk, C. F. T. van der Ven, Chahrazad Lachiheb, Lena Andersch, Marvin Jens, Marc Lukas Appis, Felix Zirngibl, Jonas Kath, M. Stecklum, K. Anders, D. Wagner, Ralf Kühn, Angelika Eggert, A. Künkele
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cardiomyopathy encompasses a diverse range of conditions characterised by extensive molecular heterogeneity, particularly the variations in cell–cell communication events such as ligand‐receptor interactions and downstream signalling. Understanding the common and unique features of these intercellular interactions is crucial for advancing targeted treatments. We analysed single‐cell RNA sequencing datasets from the ventricular regions of patients with arrhythmogenic cardiomyopathy (ACM), dilated cardiomyopathy (DCM), hypertrophic cardiomyopathy (HCM) and healthy donors (HD), as well as ischemic cardiomyopathy (ICM). Our analyses focused on cell type‐specific disease preferences, differential gene expression, pathway enrichment and particularly cell–cell communication. We observed that inflammatory, autoimmune, angiogenesis, lymphangiogenesis and fibrotic extracellular matrix deposition are consistently enriched across all four disease subtypes, highlighting their universal significance in disease progression through intercellular interactions. Additionally, we identified subtype‐specific pathways that reflect distinct intercellular communication patterns unique to each disease subtype: arrhythmia‐associated pathways in ACM, chronic inflammation‐related pathways in DCM, ECM remodelling pathways in HCM and ischaemic injury and recovery pathways in ICM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae0c90b3d024b71d885913796724ef2769e5aef3" target='_blank'>
              Characterising Shared and Specific Cell–Cell Communication in Cardiomyopathy Subtypes From Single‐Cell Transcriptomics Data
              </a>
            </td>
          <td>
            Wenqi Tao, Miao Gong, Zunping Ke
          </td>
          <td>2025-05-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The advent of the omics era has facilitated the identification of precise biomarkers for cancer progression, revealing a broader diversity of macrophage phenotypes beyond the traditional M1/M2 classification. Folate receptor 2 (FOLR2)-positive macrophages, co-expressing markers such as mannose receptor C-Type 1 (MRC1/CD206) and lymphatic vessel endothelial hyaluronan receptor 1(LYVE1), are an embryonically derived subset typically found around blood vessels in both tumor stroma and normal tissues. Despite FOLR2’s longstanding association with anti-inflammatory, immunosuppressive macrophages in tumors, its precise role in cancer progression remains unclear. Recent studies suggest that FOLR2+ macrophages can either promote or inhibit cancer progression, depending on their multifaceted roles in the tumor microenvironment. This review provides a comprehensive overview of the biological features, functional roles, molecular mechanisms, and therapeutic potential of FOLR2+ macrophages in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a55317e456e1c334fa4e34ea5c90a5105fdddf" target='_blank'>
              FOLR2+ macrophages in cancer: allies or enemies
              </a>
            </td>
          <td>
            Decheng Jiang, Ruiling Xiao, Jialu Bai, Xing Wang, Yuan Fang, Jianzhou Liu, Xiaohan Wang, Ming Cui, Lei You, Yupei Zhao
          </td>
          <td>2025-06-02</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Lung combined large-cell neuroendocrine carcinoma (CoLCNEC) refers to lung regions exhibiting both the features of large-cell neuroendocrine carcinoma (LCNEC) and the defined components of nonsmall cell lung cancer (NSCLC), with a relatively high mitotic rate. Diagnosing and predicting the prognosis of CoLCNEC are challenging. This study aimed to explore spatial transcriptomic expression patterns and identify crucial genes. Methods: We utilized a sample from a CoLCNEC patient containing three distinct components, namely, LCNEC, adenocarcinoma, and squamous cell carcinoma, with the former being predominant. Spatial transcriptomics (ST) technology, which employs the 10× Genomics Visium formalin-fixed paraffin-embedded ST kit, was applied along with high-throughput sequencing to obtain gene expression information and spatial locations for each spot. Subsequent analysis included differentially gene expression and functional enrichment. Finally, immunohistochemistry was employed to validate the marker protein structural maintenance of chromosomes 1A (SMC1A). Then, SMC1A was overexpressed and silenced in NCI-H661 and LTEP-a-2 cells, and the migration and invasion ability of the cells were detected by scratch assay and Transwell, respectively. The role of SMC1A in cancer cell cycle was detected by Real-time Reverse Transcription-PCR(RT-qPCR), Western blot, and flow cytometry, the apoptosis was detected by flow cytometry. Results: The results revealed that tumor tissue regions had higher unique molecular identifiers and gene counts than nontumor regions did. Unsupervised clustering identified four clusters, revealing the uniform distribution of unique transcripts, which were mapped onto slices to display apparent spatial separation. Differentially gene expression analysis revealed genes highly expressed in cancer cells. Further analysis of different regions revealed distinct cellular subgroups enriched through differentially gene expression analysis in various pathways, such as the cell cycle and DNA replication. Finally, SMC1A was chosen as a candidate gene, and immunohistochemistry confirmed its elevated expression in tumor regions. In addition, compared with oe-NC, oe-SMC1A can significantly promote the migration, invasion and G1/S phase transition of lung cancer cells, and promote the inhibition of apoptosis of cancer cells, while sh-SMC1A is completely opposite. Conclusions: In the tumor region of CoLCNEC, SMC1A is significantly upregulated. Moreover, silencing SMC1A effectively inhibits lung cancer cell invasion, migration, and G1/S phase transition, while promoting apoptosis. These findings indicate that SMC1A has the potential to be a new therapeutic target for CoLCNEC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/674f9dfd7baf6cb1f8c06b44560fb524bef8d3ed" target='_blank'>
              Investigation into the Spatial Heterogeneity of Lung Composite Large-Cell Neuroendocrine Carcinoma Spatial Transcriptomic Analysis of Combined Large-Cell Neuroendocrine Carcinoma.
              </a>
            </td>
          <td>
            Mingyu Ji, Daming Fan, Yaqi Yuan, Jing Wang, Xiaodong Feng, Weihua Yang, Xiaofei Dang, Yihui Xu, Jun Wang
          </td>
          <td>2025-04-30</td>
          <td>Cancer biotherapy & radiopharmaceuticals</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54dd2ec057c7a25df4bed262e9ae15c8f28acaf6" target='_blank'>
              Mapping the Transcriptional Landscape of Drug Responses in Primary Human Cells Using High-Throughput DRUG-seq
              </a>
            </td>
          <td>
            Lauren Baugh, Sébastien Vigneau, Srijani Sridhar, Sarah Boswell, George Pilitsis, John Bradley, Olga Allen, Laura M. Rand, Amanda Riccardi, Kelsey Roach, Kevin Lema, Ayla Ergun
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891242f777da10d08f786d978bb8ecf69ade37e9" target='_blank'>
              Replaying germinal center evolution on a quantified affinity landscape
              </a>
            </td>
          <td>
            William S DeWitt, Ashni A. Vora, Tatsuya Araki, Jared G. Galloway, T. Alkutkar, Juliana Bortolatto, Tiago B.R. Castro, Will Dumm, Chris Jennings-Shaffer, Tongqiu Jia, L. Mesin, G. Ozorowski, Juhee Pae, Duncan K. Ralph, Jesse D. Bloom, Armita Nourmohammad, Yun S. Song, A. Ward, Tyler N. Starr, Frederick A. Matsen, G. Victora
          </td>
          <td>2025-06-05</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="
 e14555


 Background:
 Natural killer (NK) cells are innate lymphocytes that engage in direct cytotoxic responses against malignant cells and promote pro-inflammatory tumor microenvironments. While studies have predominantly focused on conventional NK cell subsets, recent findings uncovered adaptive and tissue-resident (TR) subsets with unique functional properties. The influence of distinct NK cell subsets on tumor immunobiology and clinical outcomes remains unknown.
 Methods:
 Single-cell RNA-sequencing (scRNA-seq) of primary, therapy-naïve tumors from 276 patients across 14 cancer types, as well as 39 patients treated with immune checkpoint blockade (ICB), was accessed through publicly available datasets. We employed systematic computational analyses to phenotype and examine intra-tumoral NK cells. Gene signatures for NK cell subsets were applied to bulk RNA-sequencing from 212,105 tumor samples to delineate their associations with clinical outcomes using the Caris CODEai database.
 Results:
 Six main subsets of NK cells were identified in solid tumors, including conventional CD56
 dim
 and CD56
 bright
 NK cells, alongside adaptive, TR adaptive, TR CD56
 bright
 , and TR CD56
 bright
 /adaptive subsets. Across cancer, TR adaptive NK cells emerged as the most prognostic subset, with high infiltration conferring significantly longer overall survival (HR[95% CI] = 0.68[0.66-0.69], p < 0.001) than conventional CD56
 dim
 and CD56
 bright
 subsets. This was also observed in cohorts of head and neck, non-small cell lung, and esophageal cancers, particularly those treated with ICB. Notably, elevated TR adaptive NK cell infiltration was more prognostic than high tumor mutational burden in these settings. This was most apparent in ICB-treated head and neck cancers (HR = 0.48[0.41-0.57], p < 0.001 vs HR = 0.90[0.76-1.04], p = 0.16). Additionally, TR adaptive NK cells were strongly correlated to the infiltration of CD8
 +
 T cells and B cells in tumors. Underlying these observations, our scRNA-seq analyses demonstrated that TR adaptive NK cells exhibit the highest immunomodulatory activity of all subsets, expressing high levels of pro-inflammatory cytokines (
 CCL3
 ,
 CXCL13
 , and
 IFNG
 ) and low levels of immunosuppressive cytokines (
 TGFB1
 ). Mechanistically, this activity was linked to the upregulation of transcription factors associated with NK cell activation, function, and persistence and downregulation of genes related to stress response and exhaustion. Interestingly, scRNA-seq from paired pre- and post-ICB-treated tumors revealed a unique expansion of cytotoxic
 GZMB
 + TR adaptive NK cells after treatment.
 Conclusions:
 TR adaptive NK cells represent a novel subset of NK cells that may drive anti-tumor immune responses, particularly in the context of ICB treatment. Features of this unique population may be harnessed to design more effective NK cell-based therapies against solid tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdbcf1a77672e37211fd7cd09d5cd12367404c3a" target='_blank'>
              Identification of tissue-resident adaptive NK cells and their association with reduced mortality across diverse solid tumors.
              </a>
            </td>
          <td>
            John R Lozada, A. Ali, Christine Luo, Rosemary N. Plagens, A. Elliott, Ella Boytim, Nicholas A Zorko, W. El-Deiry, Elisabeth I. Heath, Akash Patnaik, A. Vanderwalde, Jeffrey S Miller, E. Antonarakis, Frank Cichocki, Justin H Hwang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88d0b651441089f49a566cbb1c515d194c73e9c4" target='_blank'>
              Transcriptional landscape of pleural mesothelioma patients in relation to NF2 gene mutational status.
              </a>
            </td>
          <td>
            Carlos Orozco-Castaño, Alejandro Mejía-García, Hsuan Megan Tsao, Diego A Bonilla, Ricardo Bruges-Maya, A. Cómbita, R. Parra-Medina
          </td>
          <td>2025-06-09</td>
          <td>Journal of the Egyptian National Cancer Institute</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0de20b198cece7e5c5f0e1be4cf88f6c32e54a75" target='_blank'>
              Learning copy number dependent variation in single tumour cell transcriptomes with deep generative models
              </a>
            </td>
          <td>
            Alina Selega, Hassaan Maan, Chengxin Yu, Tiak Ju Tan, Kieran R. Campbell
          </td>
          <td>2025-04-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Focal cortical dysplasia type II (FCDII) is a cortical malformation causing refractory epilepsy. FCDII arises from developmental somatic activating mutations in mTOR pathway genes, leading to focal cortical dyslamination and abnormal cytomegalic cells. Which cell types carry pathogenic mutations and how they affect cell-type-specific transcriptional programs remain unknown. In the present study, we combined several single-nucleus genotyping and transcriptomics approaches with spatial resolution in surgical cortical specimens from patients with genetically mosaic FCDII. Mutations were detected in distinct cell types, including glutamatergic neurons and astrocytes, and a small fraction of mutated cells exhibited cytomegalic features. Moreover, we identified cell-type-specific transcriptional dysregulations in both mutated and nonmutated FCDII cells, including synapse- and neurodevelopment-related pathways, that may account for epilepsy and dysregulation of mitochondrial metabolism pathways in cytomegalic cells. Together, these findings reveal cell-autonomous and non-cell-autonomous features of FCDII that may be leveraged for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b1c497a50538dc755451ab8718d7cd0ac62adfa" target='_blank'>
              Single-cell genotyping and transcriptomic profiling of mosaic focal cortical dysplasia
              </a>
            </td>
          <td>
            S. Baldassari, Esther Klingler, Lucia Gomez Teijeiro, Marion Doladilhe, Corentin Raoux, Sergi Roig-Puiggros, S. Bizzotto, Jeanne Couturier, Alice Gilbert, Lina Sami, Théo Ribierre, E. Aronica, Homa Adle-Biassette, M. Chipaux, D. Jabaudon, S. Baulac
          </td>
          <td>2025-04-30</td>
          <td>Nature Neuroscience</td>
          <td>2</td>
          <td>50</td>
        </tr>

        <tr id="Background Esophageal squamous cell carcinoma (ESCC) is a highly heterogeneous malignancy. Traditional clinical staging systems have limited value in prognostic evaluation and treatment guidance, often failing to capture the profound impact of intratumoral diversity on patient outcomes. Single-cell RNA sequencing (scRNA-seq) provides new perspectives on the cellular makeup and conditions in tumor tissues, with promising implications for functional classification and personalized therapeutic strategies in esophageal cancer. Methods In this study, we integrated scRNA-seq data with bulk RNA-seq profiles from esophageal cancer tissues to construct a comprehensive cellular atlas, focusing on the transcriptional characteristics of epithelial cells. Malignant epithelial cells were identified based on copy number variation (CNV) features using inferCNV analysis. Their developmental states and regulatory mechanisms were further characterized via transcription factor activity inference (SCENIC) and pseudotime trajectory analysis (Monocle). Based on marker genes of malignant epithelial subpopulations, we developed a multi-gene risk scoring model using data from the TCGA and GEO (GSE53624) cohorts. The model’s predictive value for immune landscape, mutational features, and drug sensitivity was also evaluated. Additionally, qRT-PCR was conducted to quantify the expression levels of model genes in ESCC tissue samples, further evaluating their biological relevance. Functional roles of the key gene HMGB3 were validated in vitro through CCK-8 proliferation assays, Transwell invasion assays, and colony formation assays following gene knockdown in ESCC cell lines. Results At the single-cell level, we identified ten major cell types and six distinct malignant epithelial subclusters, which exhibited pronounced heterogeneity in cell cycle states, transcriptional regulatory networks, and differentiation trajectories. High CNV scores and the enrichment of specific transcription factors (e.g., FOXC1, E2F1, RUNX1) suggested a proliferative and immune-evasive phenotype. A six-gene prognostic model (HMGB3, CHORDC1, CTSD, BTG2, MT1E, PHYHD1) showed strong predictive accuracy for overall survival in the TCGA and GSE53624 cohorts. Furthermore, the risk score showed a significant correlation with an immunosuppressive tumor microenvironment, increased tumor purity, and the activation of certain immune-related pathways. Analysis of drug sensitivity suggests that patients classified as low-risk could respond better to various targeted therapies and chemotherapeutic agents, underscoring their potential clinical relevance. Functional assays revealed that HMGB3 knockdown markedly suppressed ESCC cell proliferation, invasion, and colony formation, supporting its oncogenic role in esophageal cancer progression. Conclusion This study systematically characterized epithelial cell heterogeneity in esophageal cancer at single-cell resolution and established a risk model based on malignant epithelial markers that effectively predicts prognosis, immune status, and potential drug response. Combined with experimental validation, our findings highlight the pivotal role of HMGB3 in ESCC progression and provide a novel theoretical and practical framework for functional tumor classification and individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70d92eb766fe9a932637e37c8c15e4650793e3d8" target='_blank'>
              Malignant epithelial cell marker–driven risk signature enables precise stratification in esophageal cancer
              </a>
            </td>
          <td>
            Hao Zhang, Shizhao Cheng, Yijun Xu
          </td>
          <td>2025-05-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, J. Cook, Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="BACKGROUND
Lung adenocarcinoma (LUAD) is the most prevalent subtype of non-small cell lung cancer (NSCLC), characterized by high molecular and pathological heterogeneity. While traditional histopathology plays a key role in LUAD diagnosis, integrating computational pathology with multi-omics analysis provides novel insights into tumor microenvironment (TME) dynamics and molecular mechanisms. However, the relationship between pathological histological features and genomic instability in LUAD remains poorly understood.


METHODS
This study employed whole-slide images (WSIs) from the TCGA-LUAD dataset, which were processed into image patches for deep learning feature extraction using ResNet-50 and pathological feature selection using CellProfiler. Copy number variations (CNV) were inferred using inferCNV, and high-dimensional weighted gene co-expression network analysis (hdWGCNA) was performed to identify key regulatory modules associated with CNV-defined malignant cell populations. Additional analyses included intercellular communication using CellChat, pseudotime trajectory inference with Monocle2, and immune landscape profiling. Finally, we performed correlation analyses between the gene expression patterns of high-CNV (HCNV) cell lines and pathological image features, followed by prognostic model construction using a machine learning benchmark framework.


RESULTS
This study first identified LUAD malignant cells with high CNV scores. These cells also exhibited high stemness, and their proportion gradually increases with the progression of LUAD. CNV-driven tumor subpopulations exhibited distinct metabolic and immune signatures, with HCNV cells showing enhanced glycolysis, MYC signaling, and immune evasion. Intercellular communication analysis highlighted VEGF, MK and IGF signaling pathways as key mediators of HCNV-stroma interactions. A set of 192 imaging features significantly correlated with CNV burden in LUAD was identified, including 11 pathological features from CellProfiler and 181 deep learning features from ResNet-50. Machine learning-based prognostic modeling using deep learning and pathology features demonstrated robust survival prediction, with high-risk patients exhibiting lower immune infiltration and reduced immunotherapy responsiveness.


CONCLUSION
This study provides a comprehensive multi-dimensional framework integrating computational pathology and single-cell multi-omics to characterize LUAD heterogeneity. By identifying CNV-associated imaging features and key molecular regulators, we propose potential biomarkers for prognosis and therapeutic targeting in LUAD. However, as this study is based primarily on retrospective bioinformatics analysis, the clinical utility of these findings requires further validation through prospective cohorts and experimental studies. These results lay the groundwork for future translational applications but should be interpreted with caution in the absence of functional validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38e7c47777c82482e8a54128c75b302a913f6b6" target='_blank'>
              Integrating computational pathology and multi-transcriptomics to characterize lung adenocarcinoma heterogeneity and prognostic modeling.
              </a>
            </td>
          <td>
            Zerong Li, Wenmei Qiao, Siming Yu, Bin Fan, Ming Yang, Mingjuan Wu, Fang Qiu, Jinping Wang
          </td>
          <td>2025-06-05</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gliomas, particularly glioblastomas, are highly malignant brain tumors with high recurrence rates and poor prognosis. Despite advances in treatment, recurrence remains a major challenge. Epithelial-mesenchymal transition (EMT) plays a key role in tumor invasion and recurrence. This study explores the transcriptional and regulatory mechanisms driving glioma recurrence, focusing on mesenchymal-like (MES-like) subpopulations. Single-nucleus RNA sequencing was performed on 52 IDH wild-type GBM specimens, including 26 primary and 26 recurrent tumors. Spatial transcriptomics data were also incorporated. Tumor subpopulations were identified through gene regulatory network analysis, copy number variation detection, and nonnegative matrix factorization. Functional validation was conducted using gene knockdown experiments, followed by xenograft studies. We discovered novel MES-like subpopulations in recurrent GBM enriched with EMT-related genes like EGR1 and SERPINE1. These subpopulations exhibited increased transcriptional activity and were associated with poor prognosis and invasiveness. Knockdown of SERPINE1 significantly reduced cell proliferation and migration. Spatial transcriptomics showed MES-like cells concentrated at the tumor margins, highlighting their role in invasion and recurrence. MES-like subpopulations, driven by EGR1 and SERPINE1, are critical in GBM. Targeting these regulators could offer new therapeutic strategies to reduce glioma recurrence and improve outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02036-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f7432f6a50e6a5d830f429844d915dd10fcd2" target='_blank'>
              Single-cell transcriptome sequencing reveals new epithelial-stromal associated mesenchymal-like subsets in recurrent gliomas
              </a>
            </td>
          <td>
            Jinwei Li, Shengrong Long, Yang Zhang, Shuangqi Yu, Hongyu Xu, Rui Liang, Quan Liu, Jinnan Zhang, Xiang Li, Yixin Fu, Tao Xin, Yinyan Wang
          </td>
          <td>2025-06-07</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The role of the microbiome in cancer metastasis has emerged as a critical area of research, with growing evidence suggesting that microbial composition and interactions within the tumour microenvironment may significantly influence metastatic progression. This review explores the role of the microbiome in cancer metastasis, as well as potential key bacteria and their mechanisms through which they could impact tumour dissemination, seeding and growth. Biological models used to study metastasis are discussed to provide context for the further investigation of these interactions. In order to answer unresolved questions regarding the microbiome’s involvement in metastatic dissemination, recent advancements in spatial biology techniques are examined, including spatial genomics, transcriptomics, proteomics and metabolomics, which enable the spatial mapping of microbial interactions within the tumour microenvironment. Additionally, multimodal-omics imaging approaches are highlighted for their potential to integrate multiple molecular layers, offering comprehensive insights into the microbiome’s role in cancer metastasis. The review also addresses the challenges and limitations of these techniques, underscoring the complexity of studying microbiome-tumour interactions and offering directions for future research to better explore and target the microbiological landscape in metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bd737e751891e52e56b56db2c8e1ef16eeffa30" target='_blank'>
              Microbiome in cancer metastasis: biological insights and emerging spatial omics methods
              </a>
            </td>
          <td>
            Marianne Meyers, Charlotte B. A. Stoffels, G. Frache, Elisabeth Letellier, Maureen Feucherolles
          </td>
          <td>2025-06-04</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb25978847c5553a988089ffa28339e8dff51f1" target='_blank'>
              Single-Cell profiling reveals targetable malignant T-cell subtypes and immune evasion pathways in Sezary Syndrome
              </a>
            </td>
          <td>
            M.S.F Beth A. Childs, Ph.D Eslam A. Elghonaimy, Praveen, Ramakrishnan Geethakumari, Kiran A. Kumar, Joseph F. Merola, M. D. Heather W. Goff, M. Todd A Aguilera, Ph.D Todd
          </td>
          <td>2025-05-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c1a2a7bbd8743cf30a055215fa7fa8f8e871839" target='_blank'>
              Dynamic Landscape Analysis of Cell Fate Decisions: Predictive Models of Neural Development From Single-Cell Data
              </a>
            </td>
          <td>
            Marine Fontaine, M. Delás, M. Saez, R. Maizels, Elizabeth Finnie, J. Briscoe, D. A. Rand
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>8</td>
        </tr>

        <tr id="Background The heterogeneity of bladder cancer (BLCA) is affected by its inherent transcriptional properties and tumor microenvironment (TME). Stromal transcriptional components in the TME significantly influence the transcriptional classification of BLCA, and the intrinsic biological transcriptional characteristics of cancer cells may be obscured by the dominant, lineage-dependent transcriptional components of stromal origin. This study aimed to explore the degree and mechanisms by which cancer-intrinsic gene expression profiles contribute to the classification and prognosis of BLCA patients. Materials and methods In this study, BLCA single-cell transcriptome data from GSE135337 were used to identify pure tumor cells in BLCA and explore the different intrinsic heterogeneous cell subgroups of BLCA through pathway-based cancer transcriptome classification. Additionally, BLCA intrinsic subtypes were uncovered in the TCGA BLCA dataset based on the characteristic genes of the subgroups. Lastly, various machine learning algorithms were applied to identify novel potential targets of BLCA, following which their pro-tumorigenic effects were experimentally verified. Results Four BLCA intrinsic subtypes with different molecular, functional and phenotypic characteristics were successfully identified. Specifically, MA and DP subtypes demonstrated malignant phenotypes, accompanied by unfavorable clinical prognoses, limited involvement in cell death pathways, marked cell proliferation, and diminished immune activation. Notably, MA subtype exhibited the most favorable response to immunotherapy, potentially attributable to its distinctive tumor immune microenvironment. DSM subtype represented an immune-rich subtype with the optimal prognosis, characterized by abundant immune cells, high levels of co-stimulatory, co-inhibitory, major histocompatibility complex molecules, and a potential for immunotherapy response. On the other hand, HM subtype was associated with a high level of autophagy and necrosis and an “immune-hot” TIME. Furthermore, BLCA intrinsic subtypes effectively classified independent sets of BLCAs, with limited overlap with existing transcriptional classifications and showcasing unprecedented predictive and prognostic value. Finally, the DP subtype, associated with the worst prognosis, was further analyzed, leading to the identification of three potential target genes (DAD1, CYP1B1, and REXO2) significantly associated with metabolic disorders, as well as BLCA stage and grade. Conclusion This study identified a promising platform for understanding intrinsic tumor heterogeneity, which could offer new insights into the intricate molecular mechanisms of BLCA. Targeted therapy against BEXO2 may improve the prognosis of BLCA patients by regulating mitochondria-related metabolic disorders. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06682-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9934b8e51ca26d789e1232177b46072874a11e0" target='_blank'>
              Pathway-based cancer transcriptome deciphers a high-resolution intrinsic heterogeneity within bladder cancer classification
              </a>
            </td>
          <td>
            Zhan Wang, Zhaokai Zhou, Shuai Yang, Zhengrui Li, Run Shi, Ruizhi Wang, Kui Liu, Xiaojuan Tang, Qi Li
          </td>
          <td>2025-06-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Accurately resolving the composition of tumor-infiltrating leukocytes is pivotal for advancing cancer immunotherapy strategies. Despite the success of some clinical trials, applying these strategies remains limited due to the challenges in deciphering the immune microenvironment. In this study, we developed a streamlined, two-step workflow to address the complexity of bioinformatics processes involved in analyzing immune cell composition from transcriptomics data. Our dockerized toolkit, DOCexpress_fastqc, integrates the hisat2-stringtie pipeline with customized scripts within Galaxy/Docker environments, facilitating RNA sequencing (RNA-seq) gene expression profiling. The output from DOCexpress_fastqc is seamlessly formatted with mySORT, a web application that employs a deconvolution algorithm to determine the immune content across 21 cell subclasses. We validated mySORT using synthetic pseudo-bulk data derived from single-cell RNA sequencing (scRNA-seq) datasets. Our predictions exhibit strong concordance with the ground-truth immune cell composition, achieving Pearson’s correlation coefficients of 0.871 in melanoma patients and 0.775 in head and neck cancer patients. Additionally, mySORT outperforms existing methods like CIBERSORT in accuracy and provides a wide range of data visualization features, such as hierarchical clustering and cell complexity plots. The toolkit and web application are freely available for the research community, providing enhanced resolution for conventional bulk RNA sequencing data and facilitating the analysis of immune microenvironment responses in immunotherapy. The mySORT demo website and Docker image are free at https://mysort.iis.sinica.edu.tw and https://hub.docker.com/r/lsbnb/mysort_2022. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14089-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec31f2c8613b3a1fcfe2f00c0d12735378b94081" target='_blank'>
              Unveiling the immune microenvironment of complex tissues and tumors in transcriptomics through a deconvolution approach
              </a>
            </td>
          <td>
            Shu-Hwa Chen, Bo-Yi Yu, Wen-Yu Kuo, Ya-Bo Lin, Sheng-Yao Su, Wei-Hsuan Chuang, I-Hsuan Lu, Chung-Yen Lin
          </td>
          <td>2025-05-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Treatment-induced neuroendocrine prostate cancer (t-NEPC) is a lethal, castration-resistant subtype of prostate cancer. While t-NEPC typically arises from adenocarcinoma through neuroendocrine transdifferentiation after androgen pathway inhibition, the temporal dynamics and molecular drivers of this process remain poorly understood. Here, utilizing the first-in-field patient-derived xenograft (PDX) model of adenocarcinoma-to-NEPC transdifferentiation (LTL331/331R), we performed longitudinal single-cell transcriptomic sequencing (scRNA-seq) across seven timepoints spanning pre-castration to relapsed NEPC. Our analysis demonstrated 15 distinct cell clusters, including twelve adenocarcinoma clusters and two NEPC clusters (ASCL1high/FOXA2low and ASCL1low/FOXA2high clusters). Notably, we revealed a newly-discovered, early intermediate transitional cell state during t-NEPC development distinguished by epithelial-mesenchymal transition (EMT), stem cell-related, metabolically active, and HDAC-associated regulatory signatures. Analysis of this intermediate transitional cluster led to the identification of RUNX1T1 as a pivotal transcriptional regulator of NEPC transdifferentiation. Functionally, RUNX1T1 overexpression promoted AR pathway inhibition (ARPI) -induced NE transdifferentiation and increased resistance to ARPI treatment in prostate adenocarcinoma. RUNX1T1 knockdown reverses the NE transdifferentiation, inhibits NEPC cell proliferation and induces apoptosis, and cell cycle arrest. In summary, this study identifies a critical intermediate transitional cell state during t-NEPC development and reveals the heterogeneity of terminal NEPC, offering new insights into NEPC biology and emphasizing the importance of early intervention. Moreover, the discovery of RUNX1T1 as a key early driver active in both the initial and terminal phases of NEPC progression presents promising opportunities for therapeutic intervention. Highlights Longitudinal single-cell RNA sequencing of treatment-induced adeno-to-NE transdifferentiation model revealed an intermediate transitional cell state in NEPC development and progression. The sequential emergence of ASCL1high/FOXA2low and ASCL1low/FOXA2high NEPC subclusters indicates a temporal evolution of NEPC and highlights their role in contributing to NEPC heterogeneity. RUNX1T1, a transcriptional regulator showing increased expression in both intermediate cell state and NEPCs, is identified as an early driver of NEPC development. RUNX1T1 plays important functional roles in promoting adeno-to-NE transdifferentiation in early phase of NEPC development and maintaining NE phenotype and aggressiveness in terminal NEPC. Targeting RUNX1T1 or its complex could offer new therapeutic strategies for NEPC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3dfda20cdaed4a7ffd5a497959ce67fe25256c9" target='_blank'>
              Longitudinal single-cell analysis reveals RUNX1T1 as an early driver in treatment-induced neuroendocrine transdifferentiation
              </a>
            </td>
          <td>
            Yuchao Ni, Dong Lin, Mingchen Shi, Yen-Yi Lin, Hui Xue, Xin Dong, Liang Liu, F. Sar, Rebecca Wu, Tunc Morova, A. Haegert, Robert Bell, Xinyao Pang, Adam Classen, Yu Wang, Junru Chen, Stanislav Volik, S. L. Bihan, Nathan A. Lack, Christopher Ong, Gang Wang, H. Zeng, Colin C. Collins, Yuzhuo Wang
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Metastasis is a hallmark of malignancy and poses a formidable challenge for oncologists. This complex process begins when the primary malignant cells start to proliferate unlimitedly. Once the tumor reaches a certain size, cancer cells detach from the primary tumor mass and the basal lamina to which they are anchored, eventually invading nearby blood vessels. Within the bloodstream, these tumor cells have to survive and attach to the endothelium at distant sites. Cell‐to‐cell and cell‐to‐matrix interactions play an important role during these stages, in which integrins—a family of cell adhesion molecules (CAMs)—stand out as functionally centrally involved. Their expression on the tumor cell surface needs to be dynamically regulated throughout the process of metastasis. During the attachment phase to the endothelium, another group of CAMs, such as E‐/P‐selectins—primarily expressed on endothelial cells—may also play a critical role in certain malignancies. Additionally, the interplay between integrins and selectins may influence the tumor microenvironment. This review focuses on the role of integrins and their interplay with selectins in metastasis, emphasizing findings from in vivo studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9e434e2ef5f3a3703e0c8f6c7c1cc5f839b38f1" target='_blank'>
              Interplay of integrins and selectins in metastasis
              </a>
            </td>
          <td>
            Diana Maltseva, Ashot Nersisyan, Alexander Tonevitsky
          </td>
          <td>2025-05-06</td>
          <td>Molecular Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Patients with high-grade serous ovarian cancer (HGSOC) typically present with widespread metastasis, obscuring a temporal understanding of tumor-immune dynamics. To address this, we perform multi-site global proteomics alongside matched immunohistochemistry (IHC) for CD4⁺ and CD8⁺ tumor-infiltrating lymphocytes (TILs) in patient samples. We order the protein expression profiles using an unbiased pseudotime analysis, recapitulating clinical observations of metastatic progression, and providing a framework to explore tumor-immune dynamics from localized to metastatic disease. Metastatic progression correlates with immune cell infiltration, the recruitment of regulatory T cells (Tregs) to counterbalance γδ T cell abundance, and an increased abundance of exhausted CD8⁺ T cells. The accumulation of Tregs at metastatic sites correlates with SNX8 expression, a critical regulator of the STING pathway. In early-stage tumors, keratin-expressing cancer cells recruit Tregs via MHC class II, fostering an inflammatory phenotype with limited IFNγ production and non-clonally expanded T cells. Together, our findings reveal novel mechanisms of immune escape associated with both localized disease and metastatic progression in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5284b73444d2932c5ad492644667337328f9e74" target='_blank'>
              A temporal model of tumor-immune dynamics during the metastatic progression of high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Donagh Egan, Kate Glennon, Ann Treacy, Aurelie Fabre, Janet Mccormack, Salisha Hill, Ryan Morrison, Vadim Zhernovkov, Daniel C Liebler, Walter Kolch, Donal J Brennan
          </td>
          <td>2025-06-16</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSC) is an aggressive malignancy with poor treatment outcomes. Although transformative for some cancer types, immune checkpoint inhibitors (ICIs) have been only marginally successful at best in HGSC. Comprehensive interrogation of the cellular and molecular features of the tumor and immune microenvironment (TIME) of HGSC cases with varying responses to ICI could provide novel insights into the potential drivers of therapeutic response.



 We applied 10x Genomics Visium Spatial Transcriptomics (ST) to archival tissue sections from 10 patients treated with ICI (nivolumab or pembrolizumab) with varying degrees of response. Integrated cluster analysis of 39,147 ST features using Louvain optimization resulted in 20 integrated clusters. Molecular annotation using GSEA, ESTIMATE, and CIBERSORTx was performed on individual clusters to assess spatially resolved cellular transcriptional states within the cohort and by ICI response.



 Bioinformatics analysis of marker gene data from each group (responder or non-responder) provided insights into cell types for each cluster defined by transcriptional states. Major cell type populations by cluster included tumor, immune, fibroblast, epithelial, and smooth muscle. Cluster 1 cell type of both responders and non-responders is molecularly defined by known HGSC genes including CD24, WFDC2, SLPI, and MUC1. Non-responders also expressed CRIP1 and ELF3, which are associated with metastasis and cancer cell stemness, respectively. Pathway enrichment analysis across the entire cohort identified distinct immune-related pathways, including allograft rejection, IL6-JAK-STAT3 signaling and interferon responses, corroborating the immune cell deconvolution findings from CIBERSORTx. Interestingly, clusters 6 and 8 for both groups seem to represent a specialized population of lipid-associated macrophages (LAMs), marked by expression of LYZ, APOE, APOC1, GPNMB. However, in responders, we observed upregulation of antigen presenting genes, including LYZ, C1QA, C1QC, CD74, and CXCR4. In contrast, non-responders had high expression of immunosuppressive macrophages including MMP9 and IFI30 within the LAM compartment. Non-responders demonstrated higher immune composition scores in several clusters, predominantly for M0 and M2 macrophages, which are associated with immunosuppressive phenotypes. In contrast, responders displayed a more immune-activated profile, with increased infiltration of CD8+ T cells, activated CD4+ memory T cells, and activated NK cells. Notably, M2 macrophages were more abundant in the tumor cluster of non-responders, indicating a more immunosuppressive TME.



 These results underscore the heterogeneity within the tumor and immune microenvironments in HGSC tumors that may uncover novel markers or targets of response to ICI.



 Jing Qian, Lynda D. Roman, Rania Bassiouni, Hanim I. Ozkizilkaya, Kathy Kwock, Yuxin Jin, Javier Arias-Stella, Joseph W. Carlson, John D. Carpten. Spatial transcriptomics reveals differences in the tumor and immune microenvironment of high-grade serous ovarian cancers with differing responses to immune checkpoint inhibitors [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB076.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29faac1f752106215ff7dadafa3504675aecf0f8" target='_blank'>
              Abstract LB076: Spatial transcriptomics reveals differences in the tumor and immune microenvironment of high-grade serous ovarian cancers with differing responses to immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Jing Qian, Lynda D. Roman, Rania Bassiouni, Hanim I Ozkizilkaya, Kathy Kwock, Yuxin Jin, Javier Arias-Stella, Joseph W. Carlson, John D. Carpten
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/193dde17ef6934508de69e6a33e445fcab88581e" target='_blank'>
              Impaired T cell and neoantigen retention in time-serial analysis of metastatic non-small cell lung cancer in patients unresponsive to TIL cell therapy.
              </a>
            </td>
          <td>
            Chao Wang, Xiaoqing Yu, Jamie K Teer, Jiqiang Yao, Dongliang Du, Xiaoxian Liu, Zachary J Thompson, Min Hsuan Wang, Eric A. Welsh, Danish Memon, Timothy A Chan, Vladimir Makarov, C. Anadon, Lamees Saeed, Theresa A. Boyle, Bin Fang, John M. Koomen, Cheryl A Cox, Ana M Landin, Sean J Yoder, Sungjune Kim, Dung-Tsa Chen, Shari A Pilon-Thomas, Jose R Conejo-Garcia, S. Antonia, E. Haura, Ben C Creelan
          </td>
          <td>2025-05-08</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 e15561


 Background:
 Colorectal cancer (CRC), the third most prevalent cancer globally, remains a major cause of cancer-related mortality. Genomic and epigenomic plasticity during CRC progression, particularly metastasis, poses significant challenges to effective treatment. Patient-derived organoids (PDOs) reflect the heterogeneous genetic and morphological composition of cancer cells, enabling ex vivo disease modeling. This study investigates the epigenomic makeup of human primary and metastatic CRC to advance precision epigenetic therapies.
 Methods:
 We analyzed 45 organoid models, including 40 CRC organoids (from primary tumors and liver/lung metastases) and 5 generated from healthy colonic tissue. Using integrative epigenomic profiling, including chromatin accessibility, chromatin state mapping (via six histone modifications), and 3D genome conformation, we constructed a comprehensive CRC epigenomic landscape.
 Results:
 Our analysis of CRC patient-derived organoids revealed a core set of accessible genomic regions shared across all organoids, irrespective of tumor location or metastatic status. This conserved chromatin signature highlights common regulatory pathways in CRC during disease progression. Unsupervised clustering based on epigenomic features revealed four distinct CRC subtypes, encompassing both primary and metastatic models. Notably, organoids derived from liver and lung metastases did not cluster by metastatic site, underscoring epigenomic convergence. Subtype D comprised 57.1% (8/14) of primary tumors, while subtypes B and C contained 47.3% (9/19) of liver metastases and 42.8% (3/7) of lung metastases, respectively. KRAS mutation status further stratified subtypes, with wild-type KRAS observed exclusively in subtypes A and B, while KRAS-mutant variants were enriched in subtypes C (6/7) and D (4/10). Histone modification patterns, defined by six marks (H3K4me1, H3K4me3, H3K27me3, H3K27ac, H3K36me3, and H3K9me3) revealed subtype-specific regulatory chromatin landscapes. Three-dimensional chromatin interactions were mapped, and genome-wide interaction contacts are used to identify various chromatin structures that may influence metastatic capacity.
 Conclusions:
 This study highlights distinct and shared epigenomic signatures in CRC PDOs, offering novel insights into tumor progression and metastasis. Integrative chromatin analysis provides a framework for identifying epigenetic biomarkers and epigenome targeting strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66760aad28585c2f41842514d0645e6e13f51968" target='_blank'>
              Deciphering colorectal cancer progression through integrative epigenomic analysis for precision therapeutics.
              </a>
            </td>
          <td>
            Ghazaleh Tavallaee, Amin Nooranikhojasteh, Elias Orouji
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (ICC) is characterized by high malignancy, and its global incidence is predicted to continue to increase over the past decades. However, the mechanisms underlying ICC pathogenesis and progression remain unclear.


APPROACH AND RESULTS
The training cohort consisted of single-cell sequencing of 12 treatment-naïve ICC samples and spatial transcriptomics of four ICC samples. The validation cohort comprised of RNA-seq data from 87 ICC tumor samples. Finally, we validated our findings via multiplex immunofluorescence, organoids, and mice models both in vivo and in vitro. We found significant heterogeneity within the tumor microenvironment (TME) of ICC patients. ICC cells were classified into five molecular subtypes, and we revealed that aspartate beta-hydroxylase (ASPH) was a marker gene for invasion subtypes. We then selected cepharanthine (CEP) as an ASPH inhibitor that effectively suppressed tumor progression. Regarding the ICC lymphatic metastasis mechanism, we found that tumor cells in N1 lymph nodes highly expressed tumor-specific MHC-II molecules but lacked co-stimulatory factors CD80/CD86, inducing a state of anergy in CD4+ T cells, which might facilitate ICC immune evasion.


CONCLUSIONS
The TME of ICC was heterogeneous. ASPH markedly enhanced ICC invasion The ASPH inhibitor CEP significantly inhibits ICC progression and may serve as a targeted therapeutic drug for ICC. Tumor cells in N1 lymph nodes demonstrate high expression of tumor-specific MHC-II molecules, but silencing of co-stimulatory factors such as CD80/CD86 induces CD4+ T cells into an anergic state. Our study indicated that ASPH and MHC-II may serve as novel therapeutic targets for ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2012ee56dc72ff219a9edcce829b5a1b74d80284" target='_blank'>
              Tumor microenvironment heterogeneity and progression mechanisms in intrahepatic cholangiocarcinoma: A study based on single-cell and spatial transcriptomic sequencing.
              </a>
            </td>
          <td>
            Fengwei Li, Yao Li, Lishan Wang, Lei Xu, Hui Xue, Wenxin Wei, Yong Xia, Lei Wang, Feng Shen, Kui Wang
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/525bc26ccdba95190ad1d88938dd65bd9ec49925" target='_blank'>
              Cell painting in activated cells illuminates phenotypic dark space and uncovers novel drug mechanisms of action
              </a>
            </td>
          <td>
            Matylda Zietek, Akshar Lohith, Derfel Terciano, Beverley Marie Rabbitts, Aswad S. Khadilkar, John B MacMillan, Scott Lokey
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Accumulating research suggests that Epstein-Barr Virus-positive Diffuse Large B-cell Lymphoma (EBV+DLBCL) is associated with immune dysfunction and tumor microenvironment (TME) heterogeneity. While the prognostic role of the TME in EBV-DLBCL is established, its impact on EBV+DLBCL survival remains unclear. Here, we integrated 10X Visium spatial transcriptomics (ST) with single-cell RNA sequencing (scRNA-seq) to map TME heterogeneity in EBV+DLBCL. Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses identified PD-1/PD-L1 signaling as a hallmark of EBV+DLBCL’s immunosuppressive TME. Functional validation using the PD-1/PD-L1 inhibitor BMS202 revealed dose-dependent suppression of proliferation and enhanced apoptosis in EBV+Farage cells, with TLR4 emerging as a downstream effector showing EBV-status-dependent regulation. These findings not only link TME-driven PD-1/PD-L1 activation to EBV+DLBCL’s poor prognosis but also provide preclinical evidence for PD-1/PD-L1 blockade as a therapeutic strategy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00410-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1796a1ec9b1ce6b656d31aa53096b6765637e098" target='_blank'>
              Spatial transcriptomics reveals tumor microenvironment heterogeneity in EBV positive diffuse large B cell lymphoma
              </a>
            </td>
          <td>
            Mei-Yao He, Meng Liu, Jiayin Yuan, Jin Lv, Wei Li, Qianwen Yan, Yujiao Tang, Luyi Wang, Li Guo, Fang Liu
          </td>
          <td>2025-05-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is a debilitating disease that accounts for an estimated 890,000 new cases per year. Despite advancements in chemotherapy, radiotherapy, surgery and immunotherapy, the prognosis of HNSCC has remained relatively unchanged for more than a decade. Insight into the tumour microenvironment (TME) using spatially resolved approaches, and its association with clinical endpoints, may provide useful prognostic tools and refine current treatment outcomes. Here, we profiled 84 mucosal HNSCC tissue samples using next-generation ultra-high plex spatial protein profiling (580-proteins, Immuno-Oncology Proteome Atlas (IPA)) and spatial transcriptome mapping (18,000 mRNA, Whole Transcriptome Atlas (WTA)) from Bruker Spatial Biology. Samples were collected during tumour resection, after which patients went on to receive either chemotherapy and/or radiotherapy. Each sample was subdivided into tumour and stromal regions prior to digital spatial profiling. We found that patient survival outcomes were associated with anatomical subsite and tumour stage. Independent validation of key proteomic findings (including CD34 and CD44) was performed using single-cell protein profiling (PhenoCycler-Fusion, Akoya Biosciences). Harnessing the breadth of the 580-plex protein panel and WTA, we identified region-specific proteins and RNA that associate with patient survival. These findings include the expression of immune-specific proteins, CD3e and CXCR5, differentially expressed in the tumour compartment and indicate that the location of these immune signals is important for understanding disease progression. Taken together, this study provides a systematic workflow for the discovery and validation of high-plex protein and transcriptomic profiling in mucosal HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f4a98f332ffeb43581fc45c7b779e9477f23e65" target='_blank'>
              The development of a high-plex spatial proteomic methodology for the characterisation of the head and neck tumour microenvironment
              </a>
            </td>
          <td>
            Chin Wee Tan, Naomi Berrell, Meg L. Donovan, J. Monkman, Clara Lawler, H. Sadeghirad, V. Y. Naei, Jinjin Chen, Alyssa Rossenbloom, Shilah A Bonnett, Mark Conner, B. Filanoski, Christine Kang, Joseph M. Beechem, John F. Fraser, Catherine Barnett, Rahul Ladwa, Brett G M Hughes, A. Kulasinghe
          </td>
          <td>2025-06-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Cell heterogeneity is a universal feature of life. Although biological processes affected by cell-to-cell variation are manifold, from developmental plasticity to tumour heterogeneity and differential drug responses, the sources of cell heterogeneity remain largely unclear1,2. Mutational and epigenetic signatures from cancer (epi)genomics are powerful for deducing processes that shaped cancer genome evolution3–5. However, retrospective analyses face difficulties in resolving how cellular heterogeneity emerges and is propagated to subsequent cell generations. Here, we used multigenerational single-cell tracking based on endogenously labelled proteins and custom-designed computational tools to elucidate how oncogenic perturbations induce sister cell asymmetry and phenotypic heterogeneity. Dual CRISPR-based genome editing enabled simultaneous tracking of DNA replication patterns and heritable endogenous DNA lesions. Cell lineage trees of up to four generations were tracked in asynchronously growing cells, and time-resolved lineage analyses were combined with end-point measurements of cell cycle and DNA damage markers through iterative staining. Besides revealing replication and repair dynamics, damage inheritance and emergence of sister cell heterogeneity across multiple cell generations, through combination with single-cell transcriptomics, we delineate how common oncogenic events trigger multiple routes towards polyploidization with distinct outcomes for genome integrity. Our study provides a framework to dissect phenotypic plasticity at the single-cell level and sheds light onto cellular processes that may resemble early events during cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc8a5871308fcb63b1e32d25ce00cce571108d0" target='_blank'>
              Multigenerational cell tracking of DNA replication and heritable DNA damage
              </a>
            </td>
          <td>
            Andreas Panagopoulos, Merula Stout, Sinan Kılıç, Peter Leary, Julia Vornberger, Virginia Pasti, A. Galarreta, Aleksandra Lezaja, Kyra Kirschenbühler, Ralph Imhof, H. Rehrauer, Urs Ziegler, Matthias Altmeyer
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy; however, no validated treatments are currently available. Clinically, the tumor position, head and uncinate process (HU), or body and tail (BT) of the pancreas are vital for surgical strategies; however, fundamental research has seldom revealed the heterogeneous tumor microenvironment (TME) among different types of PDAC. Here, we applied multicohort single-cell and spatial RNA-seq methods together with patient-derived organoid models to reveal the TME heterogeneity between HU and BT PDAC. Osteopontin, encoded by SPP1, is secreted by vessel endothelial cells in BT PDAC and is associated with increased tumor burden. The number of tumor cells marked by CRABP2 was lower in BT PDAC and was identified as a prognostic marker of overall survival, as well as CD8+ T-cell infiltration. The expression of CRABP2 was also validated to be downregulated in BT PDAC in patient-derived organoid models. Overall, we profiled the heterogeneous PDAC TME between HU and BT PDAC, which could provide novel insight into the relationships between clinical characteristics and TME molecular research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077490657a2feb3d7ea606c6fed7a402a475e6c0" target='_blank'>
              SPP1+ Venous Endothelial Cells and CRABP2+ Tumor Cells Contribute to Favorable Body and Tail Pancreatic Ductal Adenocarcinoma Tumor Microenvironment.
              </a>
            </td>
          <td>
            Yueze Liu, Yifan Fu, Tao Liu, Jun Wang, Zeyu Zhang, Yanan Shi, Zhe Cao, Gang Yang, Hao Chen, W. Luo, Jinxin Tao, Yuanyang Wang, Guihu Weng, Menggang Zhang, Liyuan Ye, Jianchun Xiao, Jiangdong Qiu, Taiping Zhang, Hua Huang
          </td>
          <td>2025-06-11</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="In their recent publication in Nature, Miller et al. studied recurrent transcriptional programs across various brain tumors. The authors identi ﬁ ed four immunomodulatory expression programs in myeloid cells driven by factors within the tumor microenvironment (TME) including hypoxia, IL-1 β , TGF β , and dexamethasone treat-ment 1 . Notably, the authors link clinical use of dexamethasone with an immunosuppressive myeloid program, potentially high-lighting dexamethason ’ s compounding effect on glioblastoma-associated immunosuppression. Dexamethasone is currently a mainstay therapy used to control peritumoral edema. The identi ﬁ ed myeloid programs can — to some extent — predict immunotherapy response and survival rates, providing new insights for the regulatory myeloid cell landscape. Among many innovations, the central groundbreaking aspect of the study is that the tumor microenvironment is the main driver of myeloid states. Earlier evidence could not exclude the role of cell ontogeny (brain-resident vs. bone marrow-derived) and tumor characteristics (e.g., certain mutations). The ﬁ ndings highlight avenues towards developing novel immunotherapies that should focus on shielding immune cells from the tumor microenvironment.Glioblastomasare the most prevalent and lethal malignant brain tumor, mainly due to a unique immune TME 2 . Myeloid cells comprising about half of the total tumor cell content, create a highly immunosuppressive environment that hinders an effective immune response. While malignant cell states in glioblastoma have been extensively mapped 3 , the identity and regulatory programs of myeloid populations remain an area of active research. The complexity of brain-resident macrophage populations and blood-derived">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13e39e6d19ebd35ff88be943e7cbfdf7958542b8" target='_blank'>
              Limited but powerful toolbox: myeloid cells as critical immunomodulators in glioma progression
              </a>
            </td>
          <td>
            Yun Liu, Jincheng Fang, R. Sankowski
          </td>
          <td>2025-06-11</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) has a poor prognosis and most patients are diagnosed at a stage where the disease is unresectable, locally advanced, or has already metastasized. Invasive pancreatic cancer is believed to arise through a progression of noninvasive ductal lesions referred to as pancreatic intraepithelial neoplasia (PanIN). The mechanisms driving the transition from PanIN, to invasive PDAC are not fully understood. Moreover, extensive stromal involvement of pancreatic cancer tissue complicates bulk analysis, hindering precise tumor-specific molecular data. Methods This issue was addressed through a comprehensive in-silico analysis of laser-capture microdissected (LCM) pure tumor epithelial cells from transcriptomic PDAC cohort datasets, with survival outcomes further validated using TCGA. We employed LCM to evaluate mRNA expression in PanIN lesions, tumor epithelial, and stromal cells, using RT-qPCR in 20 PDAC patients. Immunohistochemistry (IHC) on 353 PDAC patients, including 73 PanIN lesions and 280 tumor tissues on tissue microarray (TMA) slides, provided further confirmation. Results Based on in-silico analysis, RAD51AP1 was identified as a tumor-specific marker associated with aggressive tumor behavior in captured PDAC samples. RT-qPCR validation demonstrated significantly elevated RAD51AP1 expression in tumor epithelial cells compared to tumor stromal cells, PanIN lesions, and adjacent normal tissues. Consistently, IHC findings for nucleus, cytoplasm, and membrane localization revealed higher RAD51AP1 protein expression in tumor tissues compared to PanIN lesions and normal tissues. This increased expression was correlated with tumor progression and aggressiveness in PDAC patients, as well as reduced survival and poor prognosis in patients with PanIN lesions. Conclusions RAD51AP1 is a tumor-specific molecule that associated with more aggressive behavior, advanced disease, and a worse survival rate. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c962e676353308024f8deb4e987c24145bac8841" target='_blank'>
              Transcriptomic analysis of laser-capture microdissected tumors reveals RAD51AP1 as a tumor-specific marker associated progression from pancreatic intraepithelial neoplasia to invasive pancreatic cancer
              </a>
            </td>
          <td>
            Fereshteh Rezagholizadeh, Adel Salimi, A. Sharifi-Zarchi, Farid Azmoudeh-Ardalan, K. Mousavizadeh, Fatemeh Tajik, Mahshid Panahi, Hossein Khorramdelazad, Masoud Baghai Wadji, Atefeh Kashanizadeh, Alireza Lotfalizadeh, S. Mowla, M. Joghataei
          </td>
          <td>2025-06-10</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="
 9562


 Background:
 The groundbreaking neoadjuvant ICB clinical trials have established beyond doubt that ICB before surgery will become the new standard of care for metastatic melanoma. Importantly, the shift from radiographic to pathologic response scoring offers an unprecedented window of opportunity to interrogate a goldmine of ‘on-treatment’ biospecimens. To date, single-cell profiling and region-based spatial transcriptomics (ST) have highlighted the importance of tertiary lymphoid structures (TLS) and stem-like T-cells as positive predictors of ICB response. However, the molecular mechanisms by which tumor infiltrating lymphocytes communicate and organize multicellular immune hubs within the spatial context of the tumor ecosystem remains poorly understood.
 Methods:
 To identify robust biomarkers of response to neoadjuvant ICB, we assembled a cohort of 58 stage III melanoma patients treated with neoadjuvant ICB [24 ipilimumab-nivolumab (IPI-NIVO), 21 NIVO-relatlimab (RELA), 13 PD1 mono] and deployed transformative technologies and computational methods, including single-cell FFPE sequencing, multiplexed FISH single-cell ST (MERFISH 305 genes), digital pathology, 3D open-top light-sheet imaging and AI-based computational pathology, to decipher the complex neoadjuvant ICB tumor ecosystem in response to therapy. We also developed 2 critical computational tools, SCIRA (Spatial Cellular Interaction and Receptor Activation), to compute receptor-ligand (R-L) interactions in whole slide images, and GC-SCAN (graph-based spatial clustering against noise), a graph-based algorithm that quantifies locally clustered structures from spatial -omics data.
 Results:
 Our results showed that the quantity and size of hyper-expanded germinal center/TLS, with increased GC: non-GC B-cell ratio, plasma cells and spatially resolved stem-like T-cells are strongly associated with response. IPI-NIVO elicited significantly stronger GC proliferation compared to NIVO-RELA and PD1 monotherapy, suggesting anti-CTLA4 can robustly induce germinal center/TLS proliferation. SCIRA spatial R-L analyses revealed the critical chemokine R-L interactions that organize the GC- and T-cell zones in response to therapy. Lastly, 3D light-sheet imaging revealed remarkable morphologic heterogeneity in 3D, with interconnected GC-TLS networks that are indicative of long-range molecular gradients.
 Conclusions:
 Our investigations herein have provided a comprehensive characterization of the immune architectures, cellular communications and 3D large-scale morphologic organizations of the TME that drive response to neoadjuvant ICB therapy. We believe the results of this study will enable the development of robust predictive biomarkers to guide the design of next generation combination ICB therapies in the clinical trial setting for melanoma and other cancer types.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb50bf439bc9a3d8fb630c6a0899f89e6769df7" target='_blank'>
              Multidimensional, spatially resolved immunologic hallmarks of response to neoadjuvant immune checkpoint blockade (ICB) therapies.
              </a>
            </td>
          <td>
            Zichao Liu, Xiaofei Song, Shikhar Dhingra, Wei-Shen Chen, Justin He, Jodi A. Balasi, Jonathan Nguyen, Carlos M. Morán-Segura, Joseph Johnson, Chaomei Zhang, Zena Sayegh, Douglas C. Marchion, Sean Yoder, Nicholas Reder, Jeffrey H. Chuang, Pei-Ling Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b812a28564555f6b348829df1f2e0c57b249620f" target='_blank'>
              Quasi-spatial single-cell transcriptome based on physical tissue properties defines early aging associated niche in liver.
              </a>
            </td>
          <td>
            Kwon Yong Tak, Juyeon Kim, Myungsun Park, Wooseok Kim, Seoyeong Lee, Narae Park, Min Jeong Kim, Ju-Bin Kang, Yongjun Koh, Hae Young Yang, Min Kyu Yum, Injune Kim, Yong Ryoul Yang, Won-Il Jeong, Jinsung Yang, Cheolju Lee, Chuna Kim, Jong-Eun Park
          </td>
          <td>2025-05-05</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Locally advanced esophageal adenocarcinoma remains difficult to treat and the ecological and evolutionary dynamics responsible for resistance and recurrence are incompletely understood. Here, we performed longitudinal multiomic analysis of patients with esophageal adenocarcinoma in the MEMORI trial. Multi-region multi-timepoint whole-exome and paired transcriptome sequencing was performed on 27 patients before, during and after neoadjuvant treatment. We found major transcriptomic changes during treatment with upregulation of immune, stromal and oncogenic pathways. Genetic data revealed that clonal sweeps through treatment were rare. Imaging mass cytometry and T cell receptor sequencing revealed remodeling of the tumor microenvironment during treatment. The presence of genetic immune escape, a less-cytotoxic T cell phenotype and a lack of clonal T cell expansions were linked to poor treatment response. In summary, there were widespread transcriptional and environmental changes through treatment, with limited clonal replacement, suggestive of phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286e0cfb0095dd021b03614405167d3960808941" target='_blank'>
              Evolutionary and immune microenvironment dynamics during neoadjuvant treatment of esophageal adenocarcinoma
              </a>
            </td>
          <td>
            M. Barroux, J. Househam, E. Lakatos, T. Ronel, Ann-Marie Baker, Henrike Salié, M. Mossner, Kane Smith, C. Kimberley, S. Nowinski, Alison Berner, V. Gunasri, Martin Borgmann, S. Liffers, Marnix Jansen, G. Caravagna, K. Steiger, J. Slotta-Huspenina, Wilko Weichert, Luis Zapata, Eleftheria Giota, Sylvie Lorenzen, M. Alberstmeier, Benjamin M Chain, H. Friess, B. Bengsch, Roland M. Schmid, J. Siveke, M. Quante, Trevor A. Graham
          </td>
          <td>2025-05-14</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, Muhammad T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3403a48f5780117716d0b711f94672490029280d" target='_blank'>
              Universal Dual-Target CAR-γδT Cells Targeting B7-H3 and IL13Rα2 for Glioblastoma Therapy
              </a>
            </td>
          <td>
            Yang Xu, Yunhao Ye, Yi Wang, Guangna Liu
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Brain tumors, in particular gliomas and glioblastoma multiforme (GBM), are thought to originate from different cells facing specific founding insults, a feature that partly justifies the complexity and heterogeneity of these severe forms of cancer. However, gliomas and GBM are usually reproduced in animal models by inducing molecular alterations in mature glial cells, which, though being part of the puzzle, do not represent the whole picture. To fill this conceptual gap, we previously developed a neurogenic model of brain cancer in Drosophila, demonstrating that the loss of cell polarity in neural stem cells (called neuroblasts in the fruit fly) is sufficient to promote the formation of malignant masses that continue to grow in the adult, displaying several phenotypic traits typical of human GBM. Here, we expand on previous work by restricting polarity disruption to Drosophila type II neuroblasts, whose self-renewal is comparable to that of mammalian neural progenitors, with the aim to capture the molecular signature of the resulting cancers in a specific and reproducible context. A comparison of the most deregulated transcripts with those found in human primary GBMs confirmed that our model can be proficiently used to delve into the roots of human brain tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/655c1526e0548f12b64a21fe0105b8cfbd172571" target='_blank'>
              Transcriptome-Wide Analysis of Brain Cancer Initiated by Polarity Disruption in Drosophila Type II Neuroblasts
              </a>
            </td>
          <td>
            Simona Paglia, P. Morciano, Dario de Biase, F. Giorgi, A. Pession, Daniela Grifoni
          </td>
          <td>2025-05-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="This study explores the systemic interactions between cancer stem cell markers (CSCMs) and cancer-affected organs through a hypergraph-based approach. By modeling organs as nodes and CSCMs as hyperedges, we capture multiway relationships beyond conventional pairwise analyses. Using the Louvain algorithm, we identify clusters of organs with shared CSCM profiles, revealing potential tumorigenic pathways and functional networks. Additionally, O-information quantifies redundancy and synergy, highlighting CSCMs with systemic influence on tumor progression and metastasis. These findings provide a structured framework for understanding CSCM dynamics across organ systems, offering insights that may inform targeted, multiorgan therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc0c90754467e3fb8ba0ecf9a2054d9f83ecd19d" target='_blank'>
              Decoding cancer dynamics: Hypergraph analysis of stem cell marker interactions through O-information and community detection.
              </a>
            </td>
          <td>
            D. H. Margarit, M. V. Reale, Gustavo Paccosi, Lilia M. Romanelli
          </td>
          <td>2025-05-01</td>
          <td>Physical review. E</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are the most abundant non-cancerous cell type in glioblastoma (GBM) and heavily influence GBM biology, contributing to tumor progression, therapeutic resistance, immune evasion, and neovascularization. Current in vitro models that utilize IL-4/IL-13 stimulation fail to capture the transcriptional and functional heterogeneity of TAMs observed in vivo. In this study, we utilize a serum-free indirect co-culture model with patient-derived xenolines to polarize primary human macrophages and characterize their molecular and functional phenotypes. We demonstrate that xenoline-polarized macrophages diverge from classical M1/M2 states and instead adopt transcriptional signatures reflective of TAM subsets identified from patients. Notably, macrophages polarized with the radiation-therapy selected xenoline, JX14P-RT, exhibited gene expression patterns enriched for interferon response and hypoxia, mirroring recurrent GBM samples. In contrast, JX14P TAMs showed enrichment in phagocytic gene sets. Functional validation of these phenotypes revealed discrepancies between the transcriptionally predicted and observed phenotypes, emphasizing the importance of integrating phenotypic validation in sequencing studies. Altogether, our findings establish xenoline-polarized macrophages as a more physiologically relevant alternative to traditional models, offering a useful model for studying tumor-immune interaction in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e59e91c3e7e604dd7d80dd0333a4e13f653a5b4e" target='_blank'>
              Xenoline-polarized macrophages as a physiologically relevant in vitro model of tumor-associated macrophages in glioblastoma
              </a>
            </td>
          <td>
            Hasan Alrefai, Benjamin Lin, Amr Elkohly, Manoj Kumar, Taylor L. Schanel, Kevin J. Lee, Patricia H Hicks, Joshua C Anderson, Gao Guo, Eun-Young Erin Ahn, C. R. Miller, Christopher Willey
          </td>
          <td>2025-05-09</td>
          <td>Research Square</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="PURPOSE Colorectal cancer (CRC) ranks third in global incidence and second in mortality, with rates increasing among younger populations. The enteric nervous system (ENS) is crucial for gastrointestinal function, and its dysfunction is associated with the progression of CRC. Neurodegeneration may disrupt ENS function and facilitate tumor growth although the mechanisms remain unclear. METHODS AND MATERIALS This study integrated single-nucleus, single-cell RNA sequencing, spatial transcriptomics (ST), and bulk transcriptomics data. Summary data-based Mendelian randomization (SMR) analysis identified neurodegeneration-related genes associated with CRC progression. Cell-cell communication was analyzed using CellCall and Cytotalk, and trajectory analysis performed using VECTOR. ST analysis was conducted using robust cell type decomposition, and spatial dependencies were assessed through MISTy and cell neighborhood network analyses. A deep learning–based prognostic model was developed using DeepSurv. Multiplex immunofluorescence was used to validate the functions of cells. RESULTS SMR analysis identified seven neurodegeneration-related genes, including MAPK9, associated with the development of CRC. High-neurodegeneration neurons were associated with poorer prognoses and infiltrated deeper into tumor niches. MISTy analysis revealed stronger spatial dependencies between high-neurodegeneration neurons and epithelial cells. Homotypic and heterotypic cell neighborhood network analyses revealed tighter neuron networks in high-degeneration groups, with enhanced neuron-epithelial communication. MAPK9-positive neurons were highly expressed in sensory neurons and facilitated tumor growth via EGFR signaling. The DeepSurv model demonstrated strong prognostic performance, with high-risk patients exhibiting worse outcomes and elevated expression of immune checkpoint genes. CONCLUSION Neurodegeneration, particularly involving MAPK9-positive neurons, promotes CRC progression through enhanced neuron-tumor interactions and EGFR signaling. This study underscores the significance of targeting neurodegeneration in CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ebbbaa1c310c2d03f5809c332bdce3f1b5993a" target='_blank'>
              Neurodegeneration Promotes Tumorigenesis in Colorectal Cancer: Insights From Single-Cell and Spatial Multiomics
              </a>
            </td>
          <td>
            Shaocong Mo, Bochen Pan, Liming Qin, Sheng Wang, Yulin Wang, Midie Xu, Fan Xia, Wenchao Gu
          </td>
          <td>2025-06-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The spatial arrangement of cells is vital in developmental processes and organogenesis in multicellular life forms. Deep learning models trained with spatial omics data uncover complex patterns and relationships among cells, genes, and proteins in a high-dimensional space, providing new insights into biological processes and diseases. State-of-the-art in silico spatial multi-cell gene expression methods using histological images of tissue stained with hematoxylin and eosin (H&E) to characterize cellular heterogeneity. These computational techniques offer the advantage of analyzing vast amounts of spatial data in a scalable and automated manner, thereby accelerating scientific discovery and enabling more precise medical diagnostics and treatments.
 In this work, we developed a vision transformer (ViT) framework to map histological signatures to spatial single-cell transcriptomic signatures, named SPiRiT (Spatial Omics Prediction and Reproducibility integrated Transformer). Our framework was enhanced by integrating cross validation with model interpretation during hyper-parameter tuning. SPiRiT predicts single-cell spatial gene expression using the matched histopathological image tiles of human breast cancer and whole mouse pup, evaluated by Xenium (10x Genomics) datasets. Furthermore, ViT model interpretation reveals the high-resolution, high attention area (HAR) that the ViT model uses to predict the gene expression, including marker genes for invasive cancer cells (FASN), stromal cells (POSTN), and lymphocytes (IL7R). In an apple-to-apple comparison with the ST-Net Convolutional Neural Network algorithm, SPiRiT improved predictive accuracy by 40% using human breast cancer ST-Net dataset. Cancer biomarker gene prediction and expression level are highly consistent with the tumor region annotation. In summary, our work highlights the feasibility to infer spatial single-cell gene expression using tissue morphology in multiple-species, i.e., human and mouse, and multi-organs, i.e., mouse whole body morphology. Importantly, incorporating model interpretation and vision transformer is expected to serve as a general-purpose framework for spatial transcriptomics.


 Yue Zhao, Elaheh Alizadeh, Yang Liu, Ming Xu, J Matthew Mahoney, Sheng Li. Inferring single-cell spatial gene expression with tissue morphology via explainable deep learning [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB087.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2517e56efe3b91d270006441fd26af51c37a6ee2" target='_blank'>
              Abstract LB087: Inferring single-cell spatial gene expression with tissue morphology via explainable deep learning
              </a>
            </td>
          <td>
            Yue Zhao, Elaheh Alizadeh, Yang Liu, Ming Xu, J. M. Mahoney, Sheng Li
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e010b52ec17520834defd15d1777a5ca8afbac" target='_blank'>
              A systematic approach to analyze T cell migration: application to mouse melanoma tumors
              </a>
            </td>
          <td>
            Nikolaos Memmos, Jason S. Mitchell, B. Fife, D. Masopust, David J. Odde
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="
 The cancer cell cycle possesses an incredible capacity to adapt or change in response to oncogenic mutations or drug treatment (cell cycle “plasticity”) to generate a diversity of solutions to the cell cycle (distinct cell cycle “programs”) that differ in key regulatory effectors and mechanisms. To study the extent and role of cell cycle plasticity in tumor progression and drug resistance, we developed cell cycle mapping - a single-cell proteomic imaging platform that combines highly multiplexed immunofluorescence with manifold learning to visualize cell cycle plasticity among and within tumors. Applying cell cycle mapping to a large library of hormone receptor (HR)+ breast tumors, we observed 4 distinct cell cycle programs driving tumor growth. These programs differed in the expression/activity of key cyclins and CDKs. In many cases, we observed multiple cell cycle programs within a single tumor correlating with differences in the cellular composition of the local tumor environment. These results also highlight the challenge of targeting the cancer cell cycle (e.g. with CDK4/6 inhibitors), given the heterogeneity of cell cycle programs actin within a single tumor. To further explore the role of cell cycle plasticity in targeted therapy, we used a combination of cell cycle mapping and time-lapse imaging to reveal how the breast cancer cell cycle responds to a panel of FDA-approved drugs targeting a range of cell cycle effectors, including multiple CDK inhibitors. We observed clear heterogeneity in the mechanisms of cell cycle adaptation executed across a panel of cell lines modeling distinct molecular (HR+, HER2+, basal) and histological subtypes (invasive ductal carcinoma, invasive lobular carcinoma) of breast cancer. Altered expression/activity of key cell cycle regulators provides a new “path” through the cell cycle, a molecular detour that allows cells to continue proliferating in the presence of drug to drive therapeutic resistance.


 Yian Yang, Thu Pham, Janet McLaughlin, Steffi Oesterreich, Adrian Lee, Katarzyna Kedziora, Wayne Stallaert. Cell cycle plasticity drives tumor progression and drug resistance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY22-02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05724e131a84cae0882ab8214c7527b7f572cfbc" target='_blank'>
              Abstract SY22-02: Cell cycle plasticity drives tumor progression and drug resistance
              </a>
            </td>
          <td>
            Yian Yang, Thu Pham, Janet McLaughlin, S. Oesterreich, Adrian Lee, Katarzyna Kedziora, Wayne Stallaert
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Melanoma is a highly heterogeneous malignancy with diverse molecular and clinical behaviors. A precise molecular classification is critical for improving prognostic assessment and guiding personalized therapy.We performed an integrative multi-omics analysis of skin cutaneous melanoma using data from The Cancer Genome Atlas (TCGA) and validated our findings in independent cohorts. Multi-layered data, including transcriptomic, genomic, epigenetic, and immune landscape profiles, were analyzed using unsupervised clustering and machine learning approaches to define molecular subtypes. Functional assays and in silico drug screening were employed to explore subtype-specific vulnerabilities.Three robust molecular subtypes (CS1, CS2, CS3) were identified, each with distinct genomic alterations, tumor microenvironment characteristics, and clinical outcomes. The CS2 subtype was immunologically “hot,” characterized by high tumor mutational burden (TMB), elevated neoantigen load, strong immune infiltration, and activated IFN-γ signaling. CS2 tumors showed significant enrichment of immune checkpoint gene expression and were associated with favorable response to anti-PD-1 therapy in external validation cohorts. In contrast, CS1 and CS3 were immunologically “cold” with immune exclusion, high chromosomal instability, and activation of oncogenic pathways linked to immune evasion. Transcriptomic drug sensitivity modeling suggested that CS1 and CS3 may benefit from HSP90 or MEK inhibitors. Moreover, COL11A2 was identified as a subtype-enriched oncogenic driver predominantly expressed in CS1/CS3, and its silencing impaired tumor cell proliferation, invasion, and epithelial–mesenchymal transition (EMT) features.This study presents a refined multi-omics classification of melanoma that reveals biologically and clinically distinct subtypes with divergent immune and therapeutic profiles. It offers a framework for subtype-specific treatment strategies, and identifies COL11A2 as a potential target in immune-cold melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47ce450bfa8af83563781e72095e944188c7b621" target='_blank'>
              Multi-omics-based subtyping of melanoma suggests distinct immune and targeted therapy strategies
              </a>
            </td>
          <td>
            Changchang Li, Xiaoqiong Lin, Jinhui Wang, Qiaochu Zhou, Fangfang Feng, Jie Xu
          </td>
          <td>2025-06-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The intratumoral microbiota, as an important component of the tumor microenvironment, is increasingly recognized as a key factor in regulating responses to cancer immunotherapy. Recent studies have revealed that the intratumoral microbiota is not uniformly distributed but instead exhibits significant spatial heterogeneity, with its distribution patterns influenced by factors such as tumor anatomy, local immune status, and therapeutic interventions. This spatial heterogeneity not only alters the interactions between microbes and the host immune system but may also reshape the immunogenic and immunosuppressive landscapes of tumors. The enrichment or depletion of microbiota in different tumor regions can influence immune cell infiltration patterns, metabolic pathway activities, and immune checkpoint molecule expression, thereby driving the development of resistance to immunotherapy. Moreover, certain bacterial metabolites form concentration gradients between the tumor core and margins, thereby regulating immune cell function. Therefore, understanding and manipulating the spatial distribution of intratumoral microbiota, particularly in resistant patients, holds promise for developing new strategies to overcome immunotherapy resistance. In the future, precise modulation strategies targeting microbial spatial heterogeneity, such as engineered bacterial vectors, probiotic combinations, and phage therapy, may open new avenues for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58f37c6ba2761018518110c9bed4886530b224c2" target='_blank'>
              Spatial Heterogeneity of Intratumoral Microbiota: A New Frontier in Cancer Immunotherapy Resistance
              </a>
            </td>
          <td>
            Qiwen Tan, Xiongjing Cao, Falong Zou, Hanwenchen Wang, Lijuan Xiong, Shenghe Deng
          </td>
          <td>2025-05-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Recent advancements in spatial transcriptomics (ST) have revolutionized our understanding of the lung’s cellular organization and pathological alterations. By preserving the spatial distribution of gene expression, ST reveals localized immune niches, stromal–epithelial interactions, and disease-associated transcriptional “hotspots” that cannot be captured by conventional sequencing methods alone. In lung cancer, ST-based investigations have delineated distinct tumor microenvironments between tumor cores and invasive fronts, revealing prognostically significant gene signatures and identifying subpopulations with differential responses to immunotherapy and chemotherapy. Similarly, in chronic obstructive pulmonary disease, asthma, and idiopathic pulmonary fibrosis, ST has mapped the ecosystem, including immune cells, inflammatory mediators, and fibroblast subtypes, of discrete regions within diseased lung tissue, offering mechanistic insights into disease progression and tissue remodeling. In addition, a more recent ST study provides spatial information for where drugs act within tissues. This review highlights the emerging role of spatial transcriptomics in respiratory research, demonstrating its potential to refine disease classification, elucidate mechanisms of therapeutic resistance, and inform spatially guided personalized interventions in respiratory diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d6db631a991c94025cd8ea23579c8fe3e08735" target='_blank'>
              Spatial Transcriptomics in Lung Cancer and Pulmonary Diseases: A Comprehensive Review
              </a>
            </td>
          <td>
            D. Kang, Yoonjoo Kim, Ji Hyeon Lee, Hyeong Seok Kang, Chaeuk Chung
          </td>
          <td>2025-06-09</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA), a neoplasm arising from biliary epithelial cells, is particularly widespread in Southeast Asia, with northeastern Thailand exhibiting the greatest prevalence attributed to Opisthorchis viverrini infection. This malignancy exhibits considerable molecular heterogeneity, leading to therapeutic resistance and recurrence. Comprehending its molecular mechanisms is essential for enhancing diagnostic and treatment approaches. Our research utilized multi-region LC-MS/MS proteomic analysis to investigate intratumor heterogeneity (ITH) in CCA. We examined 52 tumor areas and 13 neighboring tissues from 13 patients, concentrating on protein profiling, pathway analysis, differential protein expression, and the identification of shared and unique protein signatures. The findings indicated considerable inter-patient proteome variability, characterized by markedly distinct protein expressions among individuals, aligning with prior cancer research. Intra-tumor heterogeneity was apparent, with merely 18 proteins common to all tumor areas and patients, underscoring the intricacy of CCA. Significantly, the common proteins were associated with metabolic reprogramming and oxidative stress pathways, indicating possible indicators and therapeutic targets. This work highlights the significant proteome variability in CCA at both intra-tumor and inter-patient levels, underscoring the necessity for customized therapeutic approaches to tackle the disease’s complexity and improve treatment outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-02713-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afe6a3a1d14e85f2bd0571da7c1e456fa657e91b" target='_blank'>
              Proteomic profiling reveals common and region-specific protein signatures underlying tumor heterogeneity in cholangiocarcinoma
              </a>
            </td>
          <td>
            Sirinya Sitthirak, S. Roytrakul, Arporn Wangwiwatsin, N. Namwat, P. Klanrit, Hasaya Dokduang, P. Sa-Ngiamwibool, Attapol Titapan, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2025-05-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a4637e215d477abe1d7339165f5ea3e33b4aff9" target='_blank'>
              Transcriptional Roadmap of the Human Airway Epithelium Identifying HLF as a Novel Regulator of Basal Stem Cell Function
              </a>
            </td>
          <td>
            P. Prabhala, Sofia Freiman, N. Gvazava, Jiten Sharma, S. Wijk, K. Kanzenbach, Stefan Lang, Rebecka Cattani, Jenny Wigén, D. Bryder, S. Soneji, Johan Flygare, E. Tufvesson, Leif Bjermer, Darcy E. Wagner, G. W. Thorsson, Mattias Magnusson
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) tumors contain distinct cell lineages, including treatment-sensitive cells (TSCs) relying on aerobic glycolysis and treatment-resistant persister cells (TRPCs) dependent on lipid metabolism. Our systems biology-based, spatial multi-omics and multi-targeted approach decodes a critical node in GBM progression, lipid acquisition in TRPCs, and validates this through a retrospective analysis of statin-treated patients, offering clinical insights into the potential benefits of targeting lipid metabolism in high TRPC signature GBM patients.



 We performed spatial transcriptomics and proteomics, combined with 3D rendering of the tumor microenvironment (TME) and geospatially resolved single-cell neighborhood analysis of IHC-stained GBM tissues using our custom-developed SNAQ algorithm to investigate the immune microenvironment and lipid distribution specific to TSCs and TRPCs. To characterize lipid dynamics, we employed flow cytometry, time-lapse imaging, and holotomography to track lipid transfer from Tumor-Associated Myeloid Cells (TAMCs) to TRPCs. A critical node for GBM progression was identified through in vivo survival and tumor progression studies that assessed effects of genetic targeting of CCR2 and pharmacological targeting of CSF1R and FABP3. Retrospective analysis was conducted on 4,085 GBM patients to evaluate the effect of statin treatment on GBM outcomes based on their TRPC signature.



 Our study establishes the skewed accumulation of TAMCs within the TRPC microenvironment through upregulation of the CCL2/7-CCR2 and CSF1-CSF1R pathways. TAMCs actively transfer lipids to TRPCs to fulfill their metabolic demands. Targeting elements responsible for lipid incorporation in TRPCs, such as CCR2, CSF1R, and FABP3 in vivo, resulted in tumor immune microenvironment remodeling, significant tumor growth reduction, and improved survival. This reinforces the critical role of metabolic communications via lipid trafficking in TRPCs as an essential node in GBM progression. Our retrospective study showed that GBM patients with elevated TRPC signatures, typically associated with shorter survival, had no significant survival differences when treated with lipid-lowering statins. This suggests statins may disrupt lipid metabolism in TRPCs, reducing TAMC lipid support and improving patient outcomes.



 Our study highlights lipid trafficking in TRPCs as a key mechanism in GBM progression, suggesting that targeting TAMC recruitment and lipid acquisition in TRPCs may offer a promising therapy. Additionally, our retrospective analysis highlights the potential of transcriptomic profiling for patient stratification based on TRPC signature expression levels, enabling the identification of patients to benefit from lipid metabolism-targeted therapies.



 Avirup Chakraborty, Changlin Yang, Diana Feier, Aryeh Silver, Nyla T. Searl, Miruna Anica, Avinash Pittu, Illena West, Olusegun O. Sobanjo, Ethan D. Hodge, Mia K. Engelbart, Srabasti Sengupta, Nagheme Thomas, Christina Von Roemeling, Maryam Rahman, Matthew Sarkisian, Jianping Huang, Jeffrey Harrison, Duane A. Mitchell, Loic P. Deleyrolle. Disrupting lipid metabolism in GBM: A systems biology approach to targeting treatment-resistant persister cells and tumor associated myeloid cell-mediated support [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB082.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fa2a70e190a923893596d261326d79e9e7ada3b" target='_blank'>
              Abstract LB082: Disrupting lipid metabolism in GBM: A systems biology approach to targeting treatment-resistant persister cells and tumor associated myeloid cell-mediated support
              </a>
            </td>
          <td>
            Avirup Chakraborty, Changlin Yang, Diana Feier, Aryeh Silver, Nyla T. Searl, Miruna Anica, Avinash Pittu, Illena West, Olusegun O. Sobanjo, Ethan D. Hodge, Mia K. Engelbart, Srabasti Sengupta, Nagheme Thomas, C. V. von Roemeling, Maryam Rahman, Matthew R. Sarkisian, Jianping Huang, Jeffrey Harrison, Duane A Mitchell, L. Deleyrolle
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Simple Summary This review highlights the role of tumor-associated macrophages (TAMs) in shaping the immunosuppressive tumor microenvironment (TME) of glioblastoma (GBM), an aggressive brain tumor with limited treatment options. TAMs, which constitute a substantial portion of the TME, exhibit high phenotypic plasticity and dynamically shift between pro-inflammatory and immunosuppressive states. GBM exploits this plasticity to drive tumor growth by inducing TAM polarization toward the immunosuppressive phenotype through mechanisms involving cytokine secretion, immune checkpoint pathways, and metabolic interactions. This polarization contributes to immune evasion, tumor proliferation, angiogenesis, and treatment resistance. Therapeutic strategies targeting TAMs include depletion, reprogramming toward the pro-inflammatory phenotype, and inhibiting pro-tumor signals. Several approaches are under investigation, such as blocking CSF-1R, disrupting CCL2/CCR2 and PI3K pathways, targeting PD-L1 expression, and utilizing nanoparticle-based delivery systems for selective TAM modulation. Further research targeting TAM plasticity and polarization is essential for developing treatments that overcome GBM’s robust immunosuppressive TME and improve patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4f2380ce9e95379acbe519ba8ae82cc97186303" target='_blank'>
              The Basis for Targeting the Tumor Macrophage Compartment in Glioblastoma Immunotherapy
              </a>
            </td>
          <td>
            Thomas Eckert, Chase Walton, Marcus Bell, Coulter Small, Nathan C. Rowland, Charlotte Rivers, A. Zukas, S. Lindhorst, Peter E. Fecci, Ben A. Strickland
          </td>
          <td>2025-05-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, has emerged as a pivotal tumor suppressor altered in a broad range of human malignancies. Its frequent inactivation across diverse cancer types has revealed pleiotropic roles that intersect multiple Hallmarks of Cancer. In this review, we integrate current knowledge on how ARID1A loss influences cellular processes including proliferative signaling, resistance to cell death, genomic instability, metabolic reprogramming, immune evasion, and more. We discuss the context-specific consequences of ARID1A deficiency, its cooperation with other oncogenic events, and its implications for therapeutic vulnerability—particularly in the realm of synthetic lethality and immune modulation. By mapping ARID1A’s functional impact onto the established hallmarks framework, we highlight its centrality in cancer biology and underscore opportunities for biomarker-driven strategies and targeted interventions. Understanding ARID1A’s multifaceted roles offers a compelling lens through which to explore chromatin dysregulation in cancer and guide translational advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017483c65324200e0adc9a76376f32832abb8dbc" target='_blank'>
              ARID1A and Its Impact Across the Hallmarks of Cancer
              </a>
            </td>
          <td>
            Bridger Kearns, Andralyn McKell, Isaac Steveson, Peyton Worley, Braeden Barton, Jordan Bennett, DeLaney Anderson, Jacob Harris, James Christensen, Jared Barrott
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Hepatocellular carcinoma is a highly aggressive and heterogeneous malignancy with limited understanding of its heterogeneity. Methods In this study, we applied ten multi-omics classification algorithms to identify three distinct molecular subtypes of HCC (C1–C3). To further explore the immune microenvironment of these molecular subtypes, we leveraged single-cell transcriptomic data and employed CIBERSORTx to deconvolute their immune landscape. Results Among them, C3 exhibited the worst prognosis, whereas C1 and C2 were associated with relatively better clinical outcomes. Patients in the C3 group exhibited a high burden of copy number variations, mutation load, and methylation silencing. Our results revealed that compared to C1 and C2, C3 had a lower proportion of hepatocytes but a higher proportion of cholangiocytes and macrophages. Through analyses of hepatocyte, cholangiocyte, and macrophage subpopulations, we characterized their functional states, spatial distribution preferences, evolutionary relationships, and transcriptional regulatory networks, ultimately identifying cell subpopulations significantly associated with patient survival. Furthermore, we identified key ligand-receptor interactions, such as APOA1-TREM2 and APOA2-TREM2 in hepatocyte-macrophage crosstalk, and VTN-PLAUR in cholangiocyte-macrophage communication. Discussion Finally, we employed machine learning methods to construct a prognostic model for HCC patients and identified novel potential compounds for high risk patients. In summary, our novel multi-omics classification of HCC provides valuable insights into tumor heterogeneity and prognosis, offering potential clinical applications for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64917cbcdeb277a734016cb34d2bfe7542a8101f" target='_blank'>
              Multi-omics and single-cell approaches reveal molecular subtypes and key cell interactions in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Xueqing Zou, Yongmei Wang, Mingyuan Luan, Yizheng Zhang
          </td>
          <td>2025-05-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 e17615


 Background:
 Uterine carcinosarcoma (UCS) is a rare gynecologic malignancy that has poor clinical outcomes and few targeted therapies. This aggressive variant of endometrial carcinoma has a uniquely biphasic appearance and morphology of metastases from these tumors can be carcinoma, sarcoma, or carcinosarcoma. The tumor immune microenvironment (TIME) plays a crucial role in the behavior of cancers. Recent advances in single cell spatial transcriptomics allow for greater insight into the TIME of these aggressive tumors.
 Methods:
 Ten samples from four patients with multiple matched longitudinal samples underwent single-cell spatial transcriptomic (scST) profiling (Nanostring CosMx 6k). Standard segmentation and QC filters were applied. Cell typing was performed with confirmation by expert pathologist annotation of tissue regions. Normalized gene expression was compared within the clusters based on clinicopathologic characteristics.
 Results:
 scST profiling of 80,843 high-quality cells revealed 38 distinct clusters (11 carcinoma, 12 sarcoma, 9 immune, 7 benign mesenchymal). Tumor clusters were largely patient-specific with tumors demonstrating diverse transcriptional heterogeneity while metastatic tumors were composed of several less diverse clones. While immune cell clusters represented cells from all patients, there was significant patient and sample heterogeneity in the proportions of immune cell subsets observed. Macrophages were the most abundant immune cell type in all samples, plasma cells were common in one patient's samples, and rare T cells were found only in specific cellular niches. Transcriptomic features of intra-tumoral macrophages were assessed based on site (primary versus distant metastasis) and temporal (naïve versus treated with chemotherapy) groupings. Compared to tumor associated macrophages (TAM) in primary tumors, TAMs in distant metastases were more likely to have M2-like features with higher expression of
 CXCL2, CD163, CD209,
 and
 CD44
 (P<0.005). TAMs residing in chemotherapy-treated tumors had higher expression of
 IGF2, CXCL2, CXCL1, CD163, CD209, CXCL16, CD24
 ,
 CD44, CD93
 (P<0.005). Upregulation of pro-angiogenic pathways (i.e.
 VEGFA
 ) was common in both distant metastases and post-chemotherapy UCS TAMs.
 Conclusions:
 Uterine carcinosarcoma is a highly heterogeneous tumor both with respect to tumor cell and immune microenvironment features. This work highlights transcriptomic characteristics in macrophages in the UCS microenvironment. Notably, macrophages in distant metastases and in patients with prior chemotherapy were found to have more M2 TAM-like macrophages than primary untreated tumors. These results provide rationale for both TAM-based immunotherapy and anti-angiogenic approaches in UCS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ce5dd582063cc9e8b6f92d0391909010dfa4c1" target='_blank'>
              Single cell characterization of uterine carcinosarcoma and analysis of transcriptomic shifts in tumor-associated macrophages in metastatic and chemotherapy-treated tumors.
              </a>
            </td>
          <td>
            C. Nief, Sabrina Zdravkovic, Sahar Nasr, Phoebe Hammer, Brooke Howitt
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beed6f709c8c4efc3e5e432a352d4565e72fee8a" target='_blank'>
              The Curated Cancer Cell Atlas provides a comprehensive characterization of tumors at single-cell resolution.
              </a>
            </td>
          <td>
            Michael Tyler, Avishai Gavish, Chaya Barbolin, R. Tschernichovsky, Rouven Hoefflin, M. Mints, Sidharth V. Puram, I. Tirosh
          </td>
          <td>2025-05-08</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="
 e14635


 Background:
 Recent advances in computational pathology have revolutionized the analysis of histopathology images, enabling precise and rapid clinical outcome predictions. However, a significant challenge persists in translating computational insights into biologically meaningful data that informs clinical decisions. Human-interpretable image features (HIFs) have emerged as a solution, offering a detailed view of the tumor microenvironment (TME). In this study, we explored how HIFs derived from high-resolution histopathology images can predict the expression of genes associated with poor survival outcomes in stomach adenocarcinoma (STAD), aiming to bridge the gap between computational pathology and actionable clinical insights.
 Methods:
 Whole-slide histopathology images of STAD from The Cancer Genome Atlas were analyzed. Expert pathologists annotated labels were used to identify tissue types like cancer, stroma, necrosis, and normal tissue and cell types, including cancer cells, lymphocytes, macrophages, plasma cells, and fibroblasts. Using these annotations, convolutional neural networks were trained to classify these tissue and cell types, from which HIFs were extracted reflecting the biological composition of the TME, such as cell density ratios relative to surrounding tissues. We then identified genes with strong correlations to HIFs (ρ > 0.5) for further analysis.
 Results:
 Our results revealed high expression of four genes
 ABCA6, ABCA8, ADAM33,
 and
 ADAMTS10
 that were significantly associated with poor survival outcomes in STAD (p < 0.01). Each gene demonstrated a strong correlation (ρ > 0.5 and p = < 0.01) with specific HIFs.
 ABCA6
 and
 ABCA8
 , genes involved in lipid transport, are linked to a high fibroblast-to-stroma density ratio. This stromal-dominant environment is a hallmark of therapy resistance and poor outcomes.
 ADAM33
 and
 ADAMTS10
 , proteases involved in extracellular matrix remodeling, were strongly correlated (ρ > 0.5a and p = < 0.01) with a high macrophage-to-stroma density ratio. This feature represents an immunosuppressive TME, often seen in aggressive and invasive cancers.
 Conclusions:
 These findings underscore the power of HIFs to reveal key TME characteristics. A stromal-dominant TME, defined by elevated
 ABCA6
 and
 ABCA8
 , and an immunosuppressive, macrophage-rich environment associated with increased
 ADAM33
 and
 ADAMTS10
 , serve as markers of disease aggressiveness and poor prognosis. By integrating these insights into AI-driven models, clinicians may identify high-risk patients earlier in their treatment journey, enabling personalized treatment strategies to improve outcomes for STAD patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebad59e7c9b25a675a77d956eb6f9b7c2d45acd0" target='_blank'>
              Linking gene expression to tumor microenvironment using H&E features in stomach adenocarcinoma.
              </a>
            </td>
          <td>
            S. H. Bukhari, F. Vempalli, J. D. Warren, Shyam Aggarwal, Aditya Sarin, Mandeep Singh Malhotra, Rakesh K. Yadav, M. Macha, A. Bhat, Harry Lander, Tariq Masoodi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Non-small cell lung cancer (NSCLC) remains the leading cause of cancer-related mortality, largely due to its propensity to metastasize to distant organs. To identify new treatment strategies, there is a need to better understand the biological factors, such as metabolic reprogramming, that enable the survival of NSCLC cells throughout the metastatic cascade. However, standard cell culture media have non-physiological levels of key nutrients, limiting the ability to translate these findings in vivo. Here, we integrated physiological in-vitro and in-vivo approaches to reveal how metabolic plasticity drives NSCLC progression. Culturing NSCLC cells in a Human-Plasma-Like Medium versus conventional culture media revealed a marked reduction in glucose- and glutamine-derived carbon oxidation in the tricarboxylic acid (TCA) cycle. Next, we utilized non-adherent conditions to mimic extracellular matrix detachment and circulating tumor cells. We identified that NSCLC cells increased their reliance on lactate import and contribution to TCA cycle intermediates, demonstrating both a capacity to shift fuel sources under selective pressure and the influence of physiological nutrient environments on cancer cells. To assess how metabolic reprogramming supports NSCLC progression in-vivo, we utilized patient-derived xenografts with novel capacity for spontaneous metastasis. Inhibition of lactate import significantly reduced metastatic spread without substantially affecting primary tumor growth, suggesting a shift to lactate reliance in metastatic NSCLC cells. Together, these data suggest that circulating NSCLC cells' survival relies on metabolic flexibility, with key alterations of nutrient contribution to the TCA cycle. These data underscore the therapeutic potential of targeting metabolic dependencies to hinder NSCLC progression.


 Mayher Kaur, Massar Alsamraae, Nia G. Hammond, Robert Cameron, Brianna Chang, Brandon Faubert. Metabolic vulnerabilities in non-small cell lung cancer metastasis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB057.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc5f806522b166efc6f1a9991518f5749ab47e81" target='_blank'>
              Abstract LB057: Metabolic vulnerabilities in non-small cell lung cancer metastasis
              </a>
            </td>
          <td>
            Mayher Kaur, M. Alsamraae, Nia G. Hammond, Robert B. Cameron, Brianna K Chang, Brandon Faubert
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Diverse cell types and cellular states in the tumor microenvironment (TME) are drivers of biological and therapeutic heterogeneity in ovarian cancer (OV). Characterization of the diverse malignant and immunology cellular states that make up the TME and their associations with clinical outcomes are critical for cancer therapy. However, we are still lack of knowledge about the cellular states and their clinical relevance in OV. Methods We manually collected the comprehensive transcriptomes of OV samples and characterized the cellular states and ecotypes based on a machine-learning framework. The robustness of the cellular states was validated in independent cohorts and single-cell transcriptomes. The functions and regulators of cellular states were investigated. Meanwhile, we thoroughly examined the associations between cellular states and various clinical factors, including clinical prognosis and drug responses. Results We depicted and characterized an immunophenotypic landscape of 3,099 OV samples and 80,044 cells based on a machine learning framework. We identified and validated 32 distinct transcriptionally defined cellular states from 12 cell types and three cellular communities or ecotypes, extending the current immunological subtypes in OV. Functional enrichment and upstream transcriptional regulator analyses revealed cancer hallmark-related pathways and potential immunological biomarkers. We further investigated the spatial patterns of identified cellular states by integrating the spatially resolved transcriptomes. Moreover, prognostic landscape and drug sensitivity analysis exhibited clinically relevant immunological subtypes and therapeutic vulnerabilities. Conclusion Our comprehensive analysis of TME helps leveraging various immunological subtypes to highlight new directions and targets for the treatment of cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06521-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d1ebac5d2f15e31777d9394d795738342be5a0" target='_blank'>
              Multi-dimensional characterization of cellular states reveals clinically relevant immunological subtypes and therapeutic vulnerabilities in ovarian cancer
              </a>
            </td>
          <td>
            Can Zhang, Si Li, Jiyu Guo, Tao Pan, Ya Zhang, Yueying Gao, Jiwei Pan, Meng Liu, Qingyi Yang, Jinyang Yu, Juan Xu, Yongsheng Li, Xia Li
          </td>
          <td>2025-05-08</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 2042


 Background:
 Glioblastoma (GBM) is the most lethal primary brain tumor in adults, with a median survival of ~15 months despite current therapies (surgery, radiation, temozolomide). Advances like immune checkpoint inhibitors, anti-angiogenic agents, and tumor vaccines have shown suboptimal results. The 2021 WHO classification highlights molecular markers (e.g., IDH, MGMT) for better stratification, but these fail to fully capture tumor microenvironmental dynamics. Using an AI-driven transcriptomic approach, we identified novel prognostic subtypes in IDH-wildtype GBM, aiming to refine stratification, enhance understanding of tumor biology, and guide personalized therapeutic strategies.
 Methods:
 We accessed microarray data from The Cancer Genome Atlas (TCGA) (n=353 newly diagnosed, IDH-WT GBM) for a training set and RNA-seq data from the Chinese Glioma Genome Atlas (CGGA) (n=170 primary and n=106 recurrent tumors) for validation. A proprietary SphereBio machine learning–based algorithm was used to derive transcriptomic signatures with prognostic relevance. Subtypes were assessed via Kaplan–Meier analyses in the training cohort and tested in both primary and recurrent validation cohorts. Immune/stromal infiltration was quantified using a tumor deconvolution tool (DA_505), and pathway enrichment (GAGE) was performed on the validation sets.
 Results:
 AI-driven clustering revealed three transcriptomic subtypes with significant survival differences in both the training (p<0.0001) and primary validation (p=0.0004) cohorts. In the recurrent cohort, a similar survival trend by subtype was observed, though significance was diminished (p=0.12), likely due to limited sample size and therapy-related changes. Immune/stromal deconvolution showed distinct infiltration patterns: subtypes enriched for CD4+ and CD8+ T cells correlated with prolonged survival. Pathway enrichment analysis in both primary and recurrent tumors highlighted potential targets involving embryogenesis, immune modulation, cell cycle, and stress response. The persistence of a consistent survival trend and comparable microenvironment and pathway patterns suggest that these transcriptomic subtypes remain biologically relevant even after standard treatment.
 Conclusions:
 Our integrated transcriptomic and microenvironment-focused approach identified three prognostically distinct GBM subtypes, validated across independent cohorts. These findings underscore the utility of AI-driven transcriptomic signatures for personalized stratification, with the potential to guide targeted therapeutic strategies and inform clinical trial design in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff54807210ca2a362e060a895b6f6e635241ee" target='_blank'>
              AI-driven transcriptomic classification of glioblastoma: Associations with survival and tumor microenvironment.
              </a>
            </td>
          <td>
            J. Fernandez-Muñoz, Román Oberti, D. Reardon, Francisco Quintana, Guido Nicolas Molina, M. E. Guerrero-Gimenez
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The metabolism of tumors is a complex network process involving multiple metabolic species and exhibits metabolic spatial heterogeneity, which poses challenges for the development of effective therapeutic strategies. Patient‐derived tumor organoids, recently developed, hold significant potential in elucidating novel aspects of cancer biology, expediting the discovery of oncology drugs, and personalizing cancer treatment. However, tracing metabolic reactions in the tumor organoid network presents a technological challenge. In this study, we propose methodologies that utilize enzymatically derived tissue debris (TD) to generate the biobank patient‐derived lung cancer organoids (LCOs) that accurately replicate the histological characteristics, stemness of lung cancer stem cells, cellular heterogeneity, and mutational patterns observed in their respective original tumors. Importantly, our LCO model can be integrated with mass spectrometry imaging to analyze tumor metabolic heterogeneity and the spatial distributions of fingerprint lipids associated with extracellular matrix homeostasis including lysophosphatidic acid (LPA), phosphatidic acid (PA), phosphatidylserine (PS), phosphatidylglycerol (PG), diacylglycerol (DAG). Furthermore, we demonstrate the effectiveness of LCOs in evaluating personalized treatment by establishing a connection between their mutational profiles and reactions to specific drugs. Our findings highlight that LCOs from TD retain crucial metabolic characteristics of lung cancer and can be readily utilized to investigate individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4eb756e3a5aa845f598bc861225d3c569477eee" target='_blank'>
              Integrating construction and spatial imaging of organoid for tumor metabolic heterogeneity and interactions
              </a>
            </td>
          <td>
            Ling Guo, Zhaohua Xia, Jiaxin Wang, Xiaoling Yu, Chuanyue Wu, Qian Luo, Chao Zhao
          </td>
          <td>2025-05-04</td>
          <td>VIEW</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/129265cf9a8d9036012e909d0315f3224ece47d6" target='_blank'>
              The epigenetic regulator TRIM24 controls melanoma cell dedifferentiation and resistance to treatment in melanoma
              </a>
            </td>
          <td>
            Simon Durand, Félix Boivin, Roxane M. Pommier, L. Barbollat-Boutrand, Maxime Grimont, Félix Pham, Marion Dufeu, Eric Cumunel, Raphaël Schneider, Anthony Ferrari, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Introduction: Current FDA-approved immunotherapeutics for triple-negative breast cancer (TNBC) are only available for a subset of patients, highlighting the need to develop improved immune-targeting strategies. VISTA (V-domain Ig suppressor of T cell activation) is a single-pass transmembrane immune checkpoint receptor that is an emerging clinical target for cancer immunotherapy. It is expressed widely on lymphoid and myeloid cells in healthy individuals, is commonly expressed on cells of the tumor microenvironment and can be abnormally upregulated on tumor cells in a wide variety of tumors. Although VISTA confers quiescence to naïve T cells, molecular mechanisms responsible for this effect remain unclear. Also, the effects of VISTA expression on tumor cells remains poorly defined. Results: Through cell line multi-omics profiling and histologic examination of 250 human TNBC clinical specimens we identify a subtype of TNBC with extremely high VISTA expression (∼10% of all primary tumors). These tumors had undetectable PDL1 expression, very low infiltrating tumor lymphocytes and diminished proliferative rates (Ki67 ∼50%, compared to >75% for VISTA-low tumors). We characterized VISTA-high tumor biology by expressing VISTA in TNBC cell lines and monitoring tumor growth in vivo after mammary fat pad injections. In all tested cell lines (HCC1806, MDA-MB-231, 4T1, EO771) enforced expression of VISTA significantly diminished tumor growth, which was evident in both immunocompetent and immunosuppressed hosts. Molecular investigations revealed that VISTA could decrease tumor growth through its intracellular domain, which repressed EGFR trafficking and decreased EGF-dependent cell growth. We purified proteins bound to the VISTA intracellular domain, which identified PTB-domain-containing proteins and Rab11 effector proteins as functional VISTA interactors responsible for suppressing EGFR activity. These effects were mapped to a four amino acid (NPGF) motif in the VISTA intracellular domain, which could directly bind and recruit NUMB, a PTB-domain containing protein. Mutation of the VISTA NPGF motif completely reverted binding to NUMB, repression of EGFR activity and the slow tumor growth phenotype of VISTA+ tumors in vivo. To further demonstrate the functional significance of the VISTA NPGF motif we generated mice with VISTA NPGF mutations (homozygous NPAA) and studied their T cells, which naturally express high VISTA levels. T cells from NPAA mutant mice evinced increased proliferation in response to T cell receptor stimulation compared to control, demonstrating that the NPGF motif is a bona fide repressor of cell division in both tumors and T cells. Mice bearing VISTA NPAA mutations in their immune system also had diminished tumor growth compared to WT mice when injected with TNBC cell lines. Conclusions: We report a distinct subtype of TNBC characterized by high VISTA expression, low PDL1 expression and decreased proliferative index. This work identifies mechanisms by which VISTA can diminish cell division of tumors or T cells through recruitment of effector proteins to an NPGF motif in its intracellular domain. Clinical trials of VISTA-blocking monoclonal antibodies are ongoing and may have relevance for VISTA+ TNBCs that are unlikely to respond to PD1-targeted agents. In addition, therapeutic strategies to target the VISTA intracellular domain in T cells are predicted to improve immunotherapy responses and could be considered for VISTA-negative TNBCs.
 Citation Format: Joshua Gruber, Yan Zhao, Tina Andoh, Fatima Charles, Ishrat Durdana, Priyanka Reddy, Kristina Paul, Harsh Goar, Caiden Golder, Marisa Juntilla, Yan Peng, James Ford, Allison Kurian, Melinda Telli, Song Zhang, Robert West, Andrew Lipchik, Michael Snyder. Deciphering VISTA immunoreceptor mechanisms to target VISTA+ triple-negative breast cancers [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-02-23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a80629f8b3355a93c90c3b971b3fa534b6d2a30" target='_blank'>
              Abstract P1-02-23: Deciphering VISTA immunoreceptor mechanisms to target VISTA+ triple-negative breast cancers
              </a>
            </td>
          <td>
            Joshua Gruber, Yan Zhao, Tina Andoh, Fatima Charles, Ishrat Durdana, Priyanka Reddy, Kristina Paul, Harsh Goar, Caiden J Golder, Marisa Juntilla, Yan Peng, James M. Ford, Allison W. Kurian, Melinda L. Telli, Song Zhang, Robert B. West, Andrew Lipchik, M. Snyder
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer progression and therapeutic resistance are propelled by the remarkable plasticity of signaling networks, which dynamically rewire under selective pressures to maintain proliferation, enable immune evasion and promote metastasis. Despite advances in precision oncology, the dynamic crosstalk between tumor cells, non-coding genomes and the microenvironment continues to undermine treatment efficacy. This call for submissions, Revolutionizing Cancer Treatment: Navigating the Intricate Landscape of Cellular Signaling Networks, seeks cutting-edge research that dissects these adaptive mechanisms through innovative technologies - from single-cell multi-omics and spatial transcriptomics to AI-powered network modeling. We welcome studies leveraging physiomimetic models (e.g., organoids, 3D-bioprinted ecosystems) to decode tumor heterogeneity, as well as translational work targeting emergent vulnerabilities at the intersection of epigenetics, metabolic reprogramming and stromal interactions. By integrating systems biology with computational and experimental approaches, this collection aims to catalyze the design of adaptive therapies that outmaneuver cancer's evolutionary resilience.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba13faeaf95516c4bb5cbd68aeea6df92e61f127" target='_blank'>
              Revolutionizing cancer treatment: Navigating the intricate landscape of cellular signaling networks.
              </a>
            </td>
          <td>
            Hao Zhang
          </td>
          <td>2025-05-26</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a6170bfed9f76995f4210372388e1e3eba5ca6c" target='_blank'>
              Visualizing phenotypic heterogeneity and single-cell morphology in situ during gut infection
              </a>
            </td>
          <td>
            Nicholas V. DiBenedetto, M. L. Donnelly-Morell, Carol A. Kumamoto, A. Shen
          </td>
          <td>2025-05-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 The circadian clock profoundly influences cancer biology, including tumor progression, immune dynamics, and therapeutic efficacy. However, a comprehensive understanding of circadian regulation within the tumor microenvironment (TME) remains elusive. To address this gap and elucidate the diurnal cancer cell-immune interaction landscape of the TME, we generated a temporal single-cell epigenomic and transcriptomic atlas from mouse breast tumors sampled at four time points across a 24-hour cycle with a total of more than 100,000 cells with both modalities. We identified 6 main cell types including epithelial, mesenchymal, plasmablasts, T/NK cells, B cells and myeloid cells, and further classified into 14 cell subclasses and 36 cell subtypes. Significant time-of-day-dependent fluctuations in immune cell infiltration were observed, with T cells predominating in the TME during nighttime, accompanied by a decrease in macrophage populations. We characterized the rhythmic genes across cell subclasses including Cancer epithelial cells, Cancer-associated fibroblasts, CD4+/CD8+T cells, B cells, Macrophages/Monocytes, Neutrophils, and Natural killer T cells. Integrated analysis of circadian gene modules revealed cell subclass-specific regulatory patterns characterized by variations in amplitude and acrophase. Mechanistic analysis further highlighted a circadian misalignment between Cancer epithelial cells and immune cells, wherein the peak phases of cell cycle and antigen-processing activities in cancer epithelial cells were desynchronized with the activation, antigen recognition, and anti-tumor cytotoxicity peak phases of immune cells. These misalignments indicate a potential mechanism by which Cancer epithelial cells evade immune surveillance. Our findings provide a single-cell multiomic atlas of circadian regulation in breast cancer TME, uncovering rhythmic tumor-immune interactions and providing translational perspective on combining chronotherapy with immunotherapy.


 Chuqian Liang, Zunpeng Liu, Qi Zhang, Tao Zhang, Bingqiu Xiu, Gongwei Wu, Stuart Benjamin Fass, Li-Lun Ho, Manolis Kellis, Myles Brown. Circadian misalignment between cancer cells and immune cells drives immune escape [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB093.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d78a96ecb3fe054ebb7e70ba8edea4881f5b987" target='_blank'>
              Abstract LB093: Circadian misalignment between cancer cells and immune cells drives immune escape
              </a>
            </td>
          <td>
            Chuqian Liang, Zunpeng Liu, Qi Zhang, Tao Zhang, Bingqiu Xiu, Gongwei Wu, Stuart Benjamin Fass, Li-Lun Ho, M. Kellis, Myles Brown
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Human skin is not a uniform organ but a mosaic of anatomically distinct niches, with each site finely tuned to unique environmental demands and immune pressures. Yet, the molecular determinants that define these regional identities and their relationship to site-specific vulnerability to inflammatory disease remain poorly understood. Here, we generate a high-resolution single-cell atlas of human skin, profiling 274,834 cells from 96 healthy samples across 7 anatomically distinct sites (acral, arm, axilla, back, face, leg and scalp). Our analysis reveals striking region-specific transcriptional and cellular networks, uncovering how local immune-stromal crosstalk governs tissue homeostasis and underpins anatomical susceptibility to distinct inflammatory diseases such as such as systemic lupus erythematosus (SLE), atopic dermatitis (AD), and psoriasis. These findings illuminate the tissue-intrinsic foundations of regional immune identity and provide a blueprint/resource for the development of precision therapies tailored to the distinct immunological microenvironments of specific anatomical skin sites.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0efad5522b30037ec342a6195a031267736175c" target='_blank'>
              A Transcriptomic Atlas of Healthy Human Skin Links Regional Identity to Inflammatory Disease
              </a>
            </td>
          <td>
            Sahiti Marella, Rachael Bogle, Jennifer Fox, L. Tsoi, Xianying Xing, Yiqian Gu, J. Kirma, M. Sarkar, Vincent van Drongelen, Benjamin Klein, Jeff H. Kozlow, Paul W. Harms, Katherine A. Gallagher, S.Chakravarthy Naik, Vito W. Rebecca, Bogi Andersen, Mio Nakamura, J. Kahlenberg, R. Modlin, A. Billi, Johann E. Gudjonsson
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44df7a1666a420df006d1e08fae43d8fd1241151" target='_blank'>
              PCSK9 dependent cholesterol acquisition and utilization underlies metastatic organ preference in pancreatic cancer
              </a>
            </td>
          <td>
            Gilles Rademaker, Grace A Hernandez, Sumena Dahal, Lisa Miller-Phillips, Alexander L Li, C. Luan, Longhui Qui, M. Liegeois, Bruce Wang, K. W. Wen, Grace E. Kim, Eric A. Collisson, Stephan F. Kruger, Stefan Boeck, Steffen Ormanns, M. Guenther, Volker Heinemann, Michael Haas, J. J. Yeh, R. Zoncu, Rushika M. Perera
          </td>
          <td>2025-04-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Single-cell omics has been widely applied in oncology research for biomarker discovery, providing an in-depth understanding of cancer heterogeneity. While bulk sequencing methods lack the specificity afforded by single cell studies, single cell applications miss insights due to trade-offs for sensitivity at scale. Current high throughput single cell DNA-seq applications are limited to targeted sequencing approaches, while scaled single cell RNA-seq applications are limited to 3’ or 5’ end counting methods or only capture polyadenylated RNA transcripts. To address these challenges, we have developed a nanoliter dispensing instrument, library prep chemistries and a bioinformatics analyses suite that scales both single cell genomics and transcriptomics assays while maintaining whole genome and whole transcriptome coverage, respectively. To demonstrate the ability to scale a non-targeted single cell whole genome amplification (WGA) application, we applied our new WGA workflow to cancer cell lines and primary Clear Cell Renal Cell Carcinoma samples. The data revealed segmental aneuploidies and both germline and putative somatic variants in thousands of single cancer cells in a single day, at shallow sequencing depths of approximately 300,000 paired end reads per single cell. Addressing the limitation of scaled single-cell transcriptomic solutions, our new total RNA-seq workflow is capable of generating data on up to 100,000 single cells at a time in two days. We applied this high-throughput workflow on cancer cells treated and untreated with epigenetic therapy and selected 11,000 cells to reach a deeper sequencing depth. The results demonstrate the ability to identify new biomarkers through comprehensive profiling of both protein-coding and noncoding genes with full gene-body coverage, revealing significant expression differences across multiple RNA biotypes as well as identifying splice junction isoforms. Overall, our data highlights the advantages of complex and rich datasets generated from single-cell workflows, which, when paired with an unbiased, non-targeted approach, enable the discovery of novel genomic and transcriptomic events in oncology samples.


 Shuwen Chen, Peng Xu, Xuan Li, Joseph Liu, Yana Ryan, Kazuo Tori, Hima Anbunathan, Alan Du, Mike Covington, Raymond Mendoza, Samantha Leong, Tomoya Uchiyama, Mohammad Fallahi, Xuan Qu, Xiaoyun Xing, Bryan Bell, Patricio Espinoza, Ting Wang, Yue Yun, Andrew Farmer. Resolving tumor heterogeneity by uncovering novel genomic and transcriptomic events with a new scaled and automated workflow [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB047.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbc8689bc43dfd14400580dddce77b1d4a41ac2f" target='_blank'>
              Abstract LB047: Resolving tumor heterogeneity by uncovering novel genomic and transcriptomic events with a new scaled and automated workflow
              </a>
            </td>
          <td>
            Shuwen Chen, Peng Xu, Xuan Li, Joseph Liu, Yana Ryan, Kazuo Tori, Hima Anbunathan, Alan Du, Mike Covington, Raymond Mendoza, Samantha Leong, Tomoya Uchiyama, Mohammad Fallahi, Xuan Qu, Xiaoyun Xing, Bryan Bell, Patricio Espinoza, Ting Wang, Yue Yun, Andrew Farmer
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02046d3dbdf78b8dbf1ae4df369d0858d2ab96f2" target='_blank'>
              Cellular hierarchies of embryonal tumors with multilayered rosettes are shaped by oncogenic microRNAs and receptor-ligand interactions.
              </a>
            </td>
          <td>
            Alexander Beck, Lisa Gabler-Pamer, G. Alencastro Veiga Cruzeiro, S. Lambo, Bernhard Englinger, McKenzie L. Shaw, Olivia A Hack, Ilon Liu, Rebecca D. Haase, Carlos A O de Biagi, A. Baumgartner, Andrezza Do Nascimento Silva, M. Klenner, Pia Freidel, Jochen Herms, Louisa von Baumgarten, Joerg-Christian Tonn, N. Thon, K. Bruckner, S. Madlener, L. Mayr, D. Senfter, A. Peyrl, I. Slavc, D. Lötsch, C. Dorfer, Rene Geyregger, N. Amberg, C. Haberler, Norman L Mack, Benjamin Schwalm, Stefan M. Pfister, Andrey Korshunov, Lissa C Baird, Edward Yang, Susan N Chi, S. Alexandrescu, J. Gojo, Marcel Kool, Volker Hovestadt, Mariella G. Filbin
          </td>
          <td>2025-05-26</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fae7b7e6446983cf0fc2dedcf8371ed2cd79ef2" target='_blank'>
              Mapping the temporal and functional landscape of Sonic Hedgehog signaling reveals new insights into early human forebrain development
              </a>
            </td>
          <td>
            Veranika Panasenkava, Jules Garreau, H. Guyodo, Farah Diab, Ariane Mahieux, W. Carré, Yann Verres, Emmanuelle Jullion, S. Odent, C. Dubourg, Paul Rollier, Lauryane Dubos, Vanessa Ribes, Anne Camus, Marie de Tayrac, V. Dupé
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744871a7bb11281b1630d2cb920a865723043fec" target='_blank'>
              Virus-Positive Merkel Cell Carcinoma Model Supports Unified Cancer Origin
              </a>
            </td>
          <td>
            Wendy Yang, Sara Contente, Sarah Rahman
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Single-cell RNA sequencing (scRNA-seq) analyses of human and murine models of pulmonary fibrosis have uncovered multiple novel cell states involved in lung repair and regeneration, including an intermediate epithelial cell state identified as the KRT5-/KRT17+ aberrant basaloid cells in humans and the cytokeratin 8-positive alveolar differentiation intermediate (Krt8+ ADI) in mice. However, the specific contributions of these transitional cells to fibrogenesis remain poorly understood. In this study, we introduce a novel RNA-sensing-dependent protein translation technology that enables selective targeting and functional interrogation of Krt8+ ADI cells both in vitro and in vivo . Through transcriptomic analysis, we identified small proline-rich protein 1A ( SPRR1A ) mRNA as a highly specific marker shared by mouse Krt8+ ADI cells and human KRT5-/KRT17+ aberrant basaloid cells, effectively distinguishing them from other lung cell populations during lung injury and repair. Leveraging this transcriptomic specificity, we deployed programmable RNA sensors to drive EGFP reporter expression selectively in Krt8+ ADI cells in vivo . EGFP-labeled cells isolated from mouse lungs closely recapitulated the transcriptomic and phenotypic features of Krt8+ ADI cells, validating efficient and specific targeting. To assess the functional role of these cells, we established an RNA-sensing-dependent diphtheria toxin receptor (DTR) system to selectively ablate Sprr1a-expressing cells following diphtheria toxin (DT) administration. Targeted depletion of Sprr1a + cells significantly reduced fibrosis in bleomycin-injured mice, underscoring the pathogenic role of these transitional epithelial cells in lung fibrogenesis. Collectively, our findings define intermediate alveolar epithelial cells as key drivers of fibrosis and suggest that targeting this population may offer new therapeutic avenues for pulmonary fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc1a3489d69c50c47ed63956a0e70fda4a6d1ee" target='_blank'>
              Transcriptomic Signature-Guided Depletion of Intermediate Alveolar Epithelial Cells Ameliorates Pulmonary Fibrosis in Mice
              </a>
            </td>
          <td>
            Yong Zhou, Fei Peng, Chun-sun jiang, Zhen Zheng, Shahram Aliyari, Dan Shan, Aaryan Sabharwal, Qinyan Yin, Chao He, Joseph A Lasky, Victor Thannickal
          </td>
          <td>2025-05-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7be43f99c39d72183ee247e245dd3934afb3150" target='_blank'>
              Decoding meningioma prognosis with multi-omics: macrophage diversity, immune-CNV interplay, and novel SPP1-targeted strategies.
              </a>
            </td>
          <td>
            Hailang Fan, Xiaojie Li, Yaqian Zhao, Lairong Song, Lanlan Sun, Zhen Wu, Junting Zhang, Haikun Zhang, Feifan Zhang, Yingdi Huang, Dake Zhang, Liang Wang
          </td>
          <td>2025-06-16</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly invasive and heterogeneous malignancy, posing challenges for reproducible modeling and functional phenotypic analysis. To address these limitations, we developed a standardized 3D patternoid platform using collagen-based microcavity arrays to enhance organoid formation consistency and quantify subtype-specific invasion mechanisms. We utilized murine primary PDAC cells stratified by epithelial-mesenchymal transition (EMT) into three subtypes: epithelial (E-9591), hybrid EMT (Mlow-8028), and mesenchymal (M-16992). The platform's sensitivity was verified by a strong correlation between EMT scores and invasive phenotypes, as well as responses to physiological concentrations of the protease inhibitor batimastat. Key invasion parameters-including invasive area, maximum invasion distance, and branching complexity-were measured under both genomic and drug-induced conditions. The platform demonstrated high inter-organoid reproducibility, with precise control over initial cell numbers ensuring batch-to-batch comparability. Invasion dynamics analysis revealed that epithelial cells (E-9591) primarily relied on spatial constraints within the microcavity to invade. Batimastat drug sensitivity assays further distinguished invasion dependencies of the mesenchymal subtypes, confirming that M-16992 patternoids exhibit a stronger sensitivity towards MMP inhibition compared to Mlow-8028 patternoids. Concurrentlty, both subtypes experienced a shift towards epithelial-like spatial constraint triggered invasion morphology, reflecting the plasticity of PDAC invasiveness. This scalable and adaptable 3D patternoid platform enables high-throughput analysis of invasive behaviors and therapeutic responses, offering significant potential for preclinical cancer research and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97101c783ed99f665df3978439f230398b92fc54" target='_blank'>
              A 3D patternoid model for the reproducible characterization of invasive phenotypes and drug sensitivity in PDAC.
              </a>
            </td>
          <td>
            Sophie Christin Kurzbach, Violetta Carvajal-Heckele, T. Teshima, Maximilian Reichert, Andreas R. Bausch
          </td>
          <td>2025-06-19</td>
          <td>Lab on a chip</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0378275323bdf300f3993adc1058134f3560751f" target='_blank'>
              GFP-Free Live-Neuron Quantitative Imaging Reveals Compartmentalization and Growth Dynamics of PolyQ Aggregates
              </a>
            </td>
          <td>
            Xiaotian Bi, Li-En Lin, Kun Miao, Lu Wei
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c087be1ee8d5c3fb0e227ebb941d840bf16ba32a" target='_blank'>
              Residual Breast Cancer Cells Co-opt SOX5-driven Endochondral Ossification to Maintain Dormancy
              </a>
            </td>
          <td>
            Amulya Sreekumar, Eric Blankemeyer, Christopher J. Sterner, Tien-Chi Pan, D. Pant, Sarah Acolatse, Hamza Turkistani, G. Belka, Sean D. Carlin, C. Assenmacher, M. Sellmyer, David A. Mankoff, Lewis A. Chodosh
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is highly aggressive and lacks targeted therapies, posing a major challenge in oncology. Traditional two-dimensional (2D) cell cultures fail to capture the tumor microenvironment’s complexity, whereas three-dimensional (3D) cultures provide a more accurate model of tumor biology. We developed an advanced 3D culture system for TNBC cell lines BT-20 and MDA-MB-231, enhancing the hanging-drop method with Matrigel to restore essential extracellular matrix interactions. Confocal imaging showed MDA-MB-231 cells forming clusters typical of aggressive carcinoma, while BT-20 cells organized into duct-like structures. Molecular analysis of PI3K and β-catenin target genes revealed distinct expression patterns, with PI3K overexpressed and β-catenin downregulated in 3D cultures. Moreover, β-catenin distribution in the 3D cell culture closely resembles its pattern in tissue. These findings underscore the value of 3D models in understanding TNBC progression and in supporting the exploration of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847fb65a19c3a57935a901992b8fffe775547b6b" target='_blank'>
              Development of a reliable method for human triple-negative breast cancer organotypic culture: Improving imaging and genomic studies in 3D cultures
              </a>
            </td>
          <td>
            Mercedes Olvera-Valencia, Verónica García-Castillo, R. Ramos-Payán, M. Aguilar-Medina, Samuel Trujano-Camacho, Alejandro López-Saavedra, L. Marchat, C. López-Camarillo, R. Sumagin, E. Pérez-Yépez, C. Pérez-Plasencia
          </td>
          <td>2025-05-01</td>
          <td>Journal of Tissue Engineering</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Gastric cancer (GC) is among the most lethal human malignancies with limited treatment options. Cell-cell interactions within the tumor microenvironment (TME) can promote tumor growth, yet their therapeutic value has not been fully explored. Here, bulk RNA-seq, single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics (ST) were integrated to analyze the heterogeneity of GC microenvironment. Tumor-specific GREM1+ fibroblasts and SPP1+ macrophages were significantly enriched in GC tissues and were involved in immunosuppression, inflammation regulation, and tumor progression. We then indicated that GREM1+ fibroblasts and SPP1+ macrophages were positively correlated in 12 independent GC datasets and validated their close localization by multiplex immunohistochemistry staining and spatial transcriptomics. Patients with both high GREM1+ fibroblasts and SPP1+ macrophages exhibited significantly shorter OS and showed enrichment of tumor-associated pathways. Our results demonstrated the complex interactions between GREM1+ fibroblasts and SPP1+ macrophages, which may serve as a potential therapeutic target for future treatment of GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad0abacc337130966d37d3605e0104e32183302" target='_blank'>
              Multi-omics analyses reveal interactions between GREM1+ fibroblasts and SPP1+ macrophages in gastric cancer
              </a>
            </td>
          <td>
            L. Qiu, Xiao Zhao, Sheng Yao, Yang Fei, Y. Gong, Zishan Zhou, Shunchang Jiao, Jianming Xu
          </td>
          <td>2025-06-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Metabolic dysfunction-associated steatotic liver disease related hepatocellular carcinoma (MASLD-HCC) is an emerging malignancy. However, the identity and function of cancer stem cells (CSCs) in MASLD-HCC are largely unclear. Here, we aim to investigate the function, mechanism, and targeted therapy of CSCs in MASLD-HCC. Intravital genetic lineage tracing in mice demonstrated that CD133+ cells from high-fat high cholesterol diet (HFHC)-induced MASLD-HCC possess the characteristics of CSCs, including superior self-renewal capacity in vitro and tumorigenicity in vivo compared to CD133- counterparts. ScRNA-seq of CD133+ progenies revealed their clonal expansion into multiple lineages during tumor progression, highlighting the role of CD133+ CSCs in contributing to cellular plasticity in MASLD-HCC. Further, we showed that CD133 functionally drives CSC phenotypes in MASLD-HCC and that hepatocyte-specific Cd133 overexpression in mice accelerated MASLD-HCC formation driven by diethylnitrosamine plus choline-deficient, high-fat (CDHF) diet. Mechanistically, CD133 interacts with myosin heavy chain 9 (MYH9) to promote the stabilization of active β-catenin, thereby propagating Wnt/β-catenin signaling critical for CSC phenotypes as well as pro-tumorigenic potential. Accordingly, MYH9 knockdown abrogated the induction of Wnt/β-catenin by CD133. In terms of translational relevance, targeted ablation of CD133+ cells using a selective diphtheria toxin receptor system (DTRlslCd133-CreER) in mice sensitizes MASLD-HCC to multikinase inhibitors (Sorafenib and Lenvatinib). Moreover, targeting of CD133 by nanoparticle-siRNA in combination with Sorafenib synergized to induce regression of tumors in subcutaneous and orthotopic MASLD-HCC. In summary, our results indicate that CD133+ CSCs potentiate MASLD-HCC by activating Wnt/β-catenin signaling through MYH9 and is a therapeutic target, especially in combination with multikinase inhibitors.


 Yasi Pan, Xiang Zhang, Chi Chun Wong, Huarong Chen, Kai Yuan, Jun Yu. Visualization and targeting of CD133+ cancer stem cells in MASLD-related hepatocellular carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB177.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/993084e76a16325f31b7821cc1f5edf94e2b6579" target='_blank'>
              Abstract LB177: Visualization and targeting of CD133+ cancer stem cells in MASLD-related hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yasi Pan, Xiang Zhang, C. Wong, Huarong Chen, Kai Yuan, Jun Yu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Cancer cells alter their mechanical properties in response to the rigidity of their environment. Here, we explored the implications of this environmental mechanosensing for anti-tumor immunosurveillance using single cell biophysical profiling and metastasis models. Cancer cells stiffened in more rigid environments, a biophysical change that sensitized them to cytotoxic lymphocytes. In immunodeficient mice, this behavior manifested in the outgrowth of stiffer metastatic cells in the rigid bone than in the soft lung, while in immunocompetent hosts, it led to preferential elimination of stiffer cancer cells and suppression of bone metastasis. Environmentally-induced cell stiffening and immune sensitization both required Osteopontin, a secreted glycoprotein that is upregulated during bone colonization. Analysis of patient metastases spanning mechanically distinct tissues revealed associations between environmental rigidity, immune infiltration, and cancer cell stiffness consistent with mechanically driven immunosurveillance. These results demonstrate how environmental mechanosensing modulates anti-tumor immunity and suggest a mechanoimmunological basis for metastatic site selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36919956b3fc6d216b0503cde81fa82cac7cb9cf" target='_blank'>
              Mechanoimmunological Control of Metastatic Site Selection
              </a>
            </td>
          <td>
            Yassmin Elbanna, M. Tello-Lafoz, Aliya Holland, Ye Zhang, Jun-Goo Kwak, Zhenghan Wang, Alexandrina M. Yakimov, Myra Dada, Samuel Vayner, Sarah M. Duquette, Young Hun Kim, T. Bale, Benjamin Y. Winer, Kenny K. H. Yu, J. Massagué, Jungwoo Lee, O. Barzilai, S. Manalis, Morgan Huse
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/743a748682182612c504b30793e3105a56193c50" target='_blank'>
              TIGIT expression dictates the immunosuppressive reprogramming of myeloid cells in glioblastoma
              </a>
            </td>
          <td>
            Mohammad Asad, Julio Inocencio, Stefan Mitrasinovic, Minori Aoki, C. Crisman, Patrick Lasala, Emad Eskandar, Chandan Guha, Xingxing Zang, I. Parney, B. Himes
          </td>
          <td>2025-05-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Breast cancer is the most prevalent form of malignant tumor that frequently metastasizes to axillary lymph nodes (LNs). Nonetheless, the precise mechanisms underlying alterations in the tumor microenvironment (TME) in LN metastasis in breast cancer remain poorly understood. Single-cell RNA sequencing of 28 LN samples from 23 patients is performed, and a comprehensive landscape of the entire ecosystem is generated. Ten major cell types are identified, with the subclusters of each major cell type exhibiting diverse characteristics. Furthermore, multiple signatures are collected to evaluate the key components of the subclusters using multi-omics methodologies. This study finds that myCAFs may hasten LN metastasis, and observed a notable increase in APOE+ macrophages and a higher proportion of exhausted CD8+ T cells, contributing to the immunosuppressive TME. Moreover, cancer cells in metastatic lesions exhibited diverse expression patterns linked to proliferation, metastasis, oxidative phosphorylation, hypoxia, and interferon responses. Using multi-omics approaches and experimental validations, it determines that GLO1 can promote lymphatic angiogenesis, metastasis, and inhibit the proteasomal degradation of GSS, thereby maintaining intracellular glutathione (GSH) and reactive oxygen species (ROS) balance. Collectively, the study offers novel perspectives on the microenvironment remodeling of breast cancer LN metastases, suggesting that GLO1 may be a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f92d409a651118984d17df6fe54f30f3acdf6d65" target='_blank'>
              Paracrine Orchestration of Tumor Microenvironment Remodeling Induced by GLO1 Potentiates Lymph Node Metastasis in Breast Cancer.
              </a>
            </td>
          <td>
            Jindong Xie, Wenjian Liu, Xinpei Deng, Huan Wang, Xueqi Ou, Xin An, Min-yi Situ, An-Kui Yang, Chuan Peng, Rongfang He, Yi Xie, Hailin Tang, Yuman Chen, Jie-Ying Liang, Ruonan Shao, Weikai Xiao, Shaoquan Zheng
          </td>
          <td>2025-06-10</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Many studies have shown that pancreatic cancer is one of the cancers with extremely high mortality. The poor prognosis and lack of early diagnostic methods remain major challenges in the treatment of this cancer. In this article, in order to better detect the occurrence of pancreatic cancer, tools such as monocle3, singleR, harmony in R, and scanpy in Python were used to analyze the cells and genes of pancreatic cancer tissues in mice. By analyzing the data, the following results were obtained: T-cells in healthy PBMCs exhibited broader spatial dispersion than in PDAC tissues, suggesting tumor-driven immune surveillance impairment, while PDAC-associated macrophages displayed expanded distribution linked to pro-tumorigenic functions such as COL1A1-mediated ECM remodeling; Pseudotemporal trajectory analysis revealed myeloid progenitor bifurcation into monocytes/macrophages, with PDAC macrophages showing epigenetically silenced cytotoxic pathways such as suppressed GZMA/NKG7 and enhanced ribosomal biogenesis; Tissue-specific markers such as LCN2 in healthy and CTRB1/AMY2A in PDAC) and spatial co-localization of macrophages/tumor cells highlighted NOP53 as a dual-function hubinhibiting PI3K-AKT while activating p53and SPP1 as a paradoxical regulator of metastasis and antitumor immunity; Differential expression and GO enrichment analyses identified ribosomal biogenesis and cytoplasmic translation as PDAC-enriched pathways, contrasting with suppressed stress responses. Our spatial transcriptomic profiling further resolved elevated NOP53, CFB, and SPP1 expression gradients in PDAC tissues, proposing these as diagnostic biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab94aa8cc2272f4a29c06caf0dcfe1338e28a0a0" target='_blank'>
              Spatiotemporal Transcriptomic Dissection of Tumor-Associated Macrophage Heterogeneity and Dual-Function Molecular Nodes in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Mengfei Li
          </td>
          <td>2025-06-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79b9ccad6349bc919fa38a076a3195bd6f23bab" target='_blank'>
              Loss of colonic fidelity enables multilineage plasticity and metastasis.
              </a>
            </td>
          <td>
            Patrizia Cammareri, Michela Raponi, Yourae Hong, C. Billard, Nat Peckett, Yujia Zhu, Fausto D Velez-Bravo, Nicholas T Younger, Donnchadh S Dunican, S. Ö. Pohl, Aslıhan Bastem Akan, Nora J Doleschall, John Falconer, M. White, Jean Quinn, K. Pennel, Roberta Garau, Sudhir B. Malla, P. Dunne, Richard R Meehan, Owen J. Sansom, J. Edwards, Malcolm G Dunlop, Farhat V N Din, Sabine Tejpar, Colin W. Steele, K. B. Myant
          </td>
          <td>2025-06-04</td>
          <td>Nature</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="This study explores the spatial organization of Parkinson’s disease (PD)-related genes and neuroinflammatory markers in the brain using high-resolution MERFISH spatial transcriptomic data. We focused on spatial relationships among core cell types—dopaminergic, astrocytic, vascular, and immune—and their associations with cells expressing high levels of PD and inflammation-related genes. Neighborhood enrichment and density correlation analyses revealed strong co-localization of astrocytes and vascular cells, while dopaminergic and immune cells were spatially segregated. Spatial expression maps showed regionally restricted gene activity, and combined analysis of cell types and gene-high expressing cells highlighted spatially structured neighborhoods. Clustering of gene-high groups across three brains yielded reproducible gene modules enriched in immune response and neurotransmitter regulation pathways. These findings reveal consistent spatial interactions and transcriptional modules that may underlie PD-related neuroinflammation, offering insights into disease-associated tissue architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34e375ab758825c1a2620356d230e60142b191d9" target='_blank'>
              Spatial Transcriptomics Analysis of Neuroinflammatory Pathways Associated with Parkinson’s Disease Genes
              </a>
            </td>
          <td>
            Austin Doekyung Kim
          </td>
          <td>2025-05-09</td>
          <td>Journal of Medicine and Health Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbcccd092da4df387dc9187eeb5acec5c01b0b6" target='_blank'>
              Loss of chromosome Y shapes the immune cell fate with aging in men
              </a>
            </td>
          <td>
            A. A. Z. Dawoud, L. Green, O. Rackham
          </td>
          <td>2025-06-02</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a lethal brain cancer with no effective treatment; understanding how GBM cells respond to tumor microenvironment remains challenging as conventional cell cultures lack proper cytoarchitecture while in vivo animal models present complexity all at once. Developing a culture system to bridge the gap is thus crucial. Here, a multicellular approach is employed using human glia and vascular cells to optimize a 3D brain vascular niche model that enabled not only long-term culture of patient derived GBM cells but also recapitulation of key features of GBM heterogeneity, in particular invasion behavior and vascular association. Comparative transcriptomics of identical patient derived GBM cells in 3D and in vivo xenotransplants models revealed that glia-vascular contact induced genes concerning neural/glia development, synaptic regulation, as well as immune suppression. This gene signature displayed region specific enrichment in the leading edge and microvascular proliferation zones in human GBM and predicted poor prognosis. Gene variance analysis also uncovered histone demethylation and xylosyltransferase activity as main themes for gene adaption of GBM cells in vivo. Furthermore, the 3D model also demonstrated the capacity to provide a quiescence and a protective niche against chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58339daec87fc88b9de1cfb8daf1fd79e18b6ecc" target='_blank'>
              3D Brain Vascular Niche Model Captures Glioblastoma Infiltration, Dormancy, and Gene Signatures.
              </a>
            </td>
          <td>
            Vivian K. Lee, R. Tejero, Nathaniel Silvia, Anirudh Sattiraju, Aarthi Ramakrishnan, Li Shen, Alexandre Wojcinski, Santosh Kesari, R. Friedel, Hongyan Zou, Guohao Dai
          </td>
          <td>2025-06-19</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 e14562


 Background:
 T cell-based immunotherapies have demonstrated great success for the treatment of many solid tumors, but not all tumor types and patients respond well or durably. Reasons for these discrepancies are unclear. High grade serous ovarian cancer (HGSOC) is an example of a deadly disease that has exhibited poor responses to T cell targeted therapies, like checkpoint inhibitors. Thus, it would be an ideal model for studying other parts of the immune ecosystem that could contribute to anti-tumor immunity. Tumor infiltrating B cells (TIL-Bs) are being increasingly studied in HGSOC because of their association with improved prognosis, especially when differentiated into germinal center (GC) B cells. However, the true effector functions of B cells, including antigen presentation, cytokine production, and antibody production, in ovarian and other solid tumors remains understudied. Here, we study tumor antigen-specific B cell responses in a mouse model of HGSOC.
 Methods:
 We have engineered a murine HGSOC cell line ID8 p53 -/- brca2 -/- (ID8) to express a membrane bound mode antigen, hen egg lysozyme (mHEL), at two different affinities to primary HEL-specific B cells (MD4 B cells) from MD4 transgenic mice. We use flow cytometry to characterize HEL-specific B cell responses to ID8 mHEL-expressing cells
 in vitro
 . We use a syngeneic mouse model of HGSOC in which we intraperitoneally inject ID8 cells (with or without mHEL) and use flow cytometry to identify immune phenotypes of B and T cells in tumor tissue, spleen, lymph nodes, and ascites. We also use ELISA to measure HEL-specific antibody production in the ascites and serum of these mice.
 Results:
 We demonstrate that MD4 B cells can upregulate canonical B cell receptor signaling elements in response to stimulation with ID8 mHEL
 in vitro
 , and that these responses can be titrated based upon the affinity of the mHEL presented. We also show that the syngeneic ID8 p53 -/- brca2 -/- mouse model produces tumors in the ovaries and peritoneum that are enriched for B cells, and that the ascites is specifically enriched for antigen specific B2 B cells (with a much smaller innate like B1 compartment as compared to the peritoneal lavage of non-tumor controls). We plan to inject host mice with ID8 mHEL tumors and compare tumor progression in mice that subsequently receive intraperitoneal adoptive transfer of MD4 HEL-specific B cells versus not. We also plan to compare the phenotypes of B and T cells in the tumor, spleen, lymph nodes, and ascites of these mice, as well as the serum and ascites levels of HEL antibodies.
 Conclusions:
 We have generated a model antigen expressing ovarian cancer cell line that produces antigen-specific B cell responses
 in vitro
 , and have shown that there is an enrichment in antigen-specific B cells in these ovarian tumors and ascites; future
 in vivo
 studies will elucidate the important effector functions of TIL-Bs in ovarian cancer and other solid tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/107a1451e84aeccb51bc914d82c8c35407885897" target='_blank'>
              Dissecting B-cell responses in high grade serous ovarian cancer.
              </a>
            </td>
          <td>
            Ramya Parameswaran, James Mueller, Katherine C Fuh, J. Zikherman
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Introduction Immune checkpoint inhibitors (ICIs) have significantly improved survival for patients with metastatic melanoma, yet many experienceresistance due to immunosuppressive mechanisms within the tumor immune microenvironment (TIME). Understanding how the spatial architecture of immune and inflammatory components changes across disease stages may reveal novel prognostic biomarkers and therapeutic targets. Methods We performed high-dimensional spatial profiling of two melanoma tissue microarrays (TMAs), representing Stage III (n = 157) and Stage IV (n = 248) metastatic tumors. Using imaging mass cytometry (IMC) and multiplex immunofluorescence (mIF), we characterized the phenotypic, functional, and spatial properties of the TIME. Cellular neighborhoods were defined by inflammatory marker expression, and spatial interactions between immune and tumor cells were quantified using nearest-neighbor functions (G-cross). Associations with survival were assessed using Cox proportional hazards models with robust variance estimation. Results Stage IV tumors exhibited a distinct immune landscape, with increased CD74- and MIF-enriched inflammatory neighborhoods and reduced iNOS-associated regions compared to Stage III. Cytotoxic T lymphocytes (CTLs) and tumor cells were more prevalent in Stage IV TIME, while B cells and NK cells were depleted. Spatial analysis revealed that CTL–Th cell, NK–T cell, and B–NK cell interactions were linked to improved survival, whereas macrophage aggregation and excessive B–Th cell clustering in inflammatory regions correlated with worse outcomes. Organ-specific analyses showed that CTL infiltration near tumor cells predicted survival in gastrointestinal metastases, while NK–T cell interactions were prognostic in lymph node and skin metastases. Discussion Our results reveal stage-specific shifts in immune composition and spatial organization within the melanoma TIME. In advanced disease, immunosuppressive neighborhoods emerge alongside changes in immune cell localization, with spatial patterns of immune coordination—particularly involving CTLs, NK cells, and B cells—strongly predicting survival. These findings highlight spatial biomarkers that may refine patient stratification and guide combination immunotherapy strategies targeting the inflammatory architecture of the TIME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26ea85ba1c4980ace36f2d1785fed9a6143d63d8" target='_blank'>
              Characterizing spatial immune architecture in metastatic melanoma using high-dimensional multiplex imaging
              </a>
            </td>
          <td>
            Joel Eliason, S. Krishnan, Yasunari Fukuda, Matias A. Bustos, Daniel G. Winkowski, Sungnam Cho, Akshay V. Basi, Regan Baird, Elizabeth A. Grimm, Michael A. Davies, Dave S. B. Hoon, Arvind Rao, Jared K. Burks, S. Ekmekcioglu
          </td>
          <td>2025-04-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 e14585


 Background:
 Immune checkpoint blockade (ICB) targeting PD-1/PD-L1 is crucial in treating breast cancer. Despite clinical success, the mechanisms by which ICB therapies reshape the tumor microenvironment (TME) and enhance anti-tumor activity remain unclear. Recent studies reveal that anti-PD-1 therapies broadly alter TME cells beyond PD-1+ T cells, highlighting extensive cell-cell communication (CCC) as a key factor. Understanding CCC changes in response to anti-PD-1 therapy can illuminate its mechanisms of action (MOA) and TME dynamics. This study applies causal inference methodology to investigate CCC between T cells and non-T cells in the TME of breast cancer patients receiving anti-PD-1 therapy.
 Methods:
 We analyzed single-cell RNA-seq data from 31 breast cancer patients pre- and on-treatment with anti-PD-1 (pembrolizumab) reported by Bassez et al., 2021. We identified differentially expressed genes (DEGs) across cell types. We applied the instrumental variable method to identify causal relationships between signals in T-cell and non-T-cell DEGs to uncover treatment-induced CCC. We further searched for ligand-receptor pairs mediating CCC and identified gene expression modules (GEMs) regulated by these ligand-receptors. Finally, we constructed a CCC network from T to non-T cells.
 Results:
 Anti-PD1 treatment induced broad gene expression changes in diverse cell populations. Major T cell pathways influenced by anti-PD-1 therapy include NF-κB, interferon-γ and interleukins. CD4
 +
 and CD8
 +
 exhausted cells, expressing high levels of
 PDCD1
 (encoding PD-1), exhibited distinct activated pathways. We identified the CCC network from T cells to non-T cells via ligand-receptor interactions. For example, anti-PD1 treatment activated cellular stress, apoptosis, and pro-inflammatory cytokine signaling pathways in T cells, altering the expression of a GEM that included
 TSC22D3
 and
 TXNIP
 . This initiated signaling to myeloid cells via
 RPS19
 -
 C5AR1
 , leading to NF-κB activation. CD4+ exhausted T cells primarily signaled to monocytes, enhancing helper functions to recruit and activate monocytes, thereby promoting immune regulation and amplification. In contrast, CD8+ exhausted T cells primarily interacted with macrophages, intensifying cytotoxic responses that facilitated effective tumor-cell killing.
 Conclusions:
 Our analyses provide insights into the dynamic interplay of cells within the TMEs during anti-PD-1 therapy. These findings could facilitate the identification of new biomarkers for predicting heterogeneous treatment responses to anti-PD-1 regimens, potentially enhancing the design and customization of immunotherapeutic strategies for breast cancer patients. Finally, this study demonstrated the utility of causal inference methodologies for mechanistic studies of CCC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74715a58ffb2aed0b55ed5b7dec461d0fe94b624" target='_blank'>
              Use of causal modeling to uncover cell-cell communication dynamics in the tumor microenvironment during anti-PD-1 therapy in breast cancer patients.
              </a>
            </td>
          <td>
            Aodong Qiu, Han Zhang, Joseph Ramsey, Boyang Sun, Bryan Andrews, Kun Zhang, Gregory F. Cooper, Lujia Chen, Xinghua Lu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) continues to present a therapeutic challenge due to the fact that by definition, these cancer cells lack the expression of targetable receptors. Current treatment options include cytotoxic chemotherapy, antibody–drug conjugates (ADC), and the PD-1 checkpoint inhibitor, pembrolizumab. Due to high rates of recurrence, current guidelines for early-stage TNBC recommend either multi-agent chemotherapy or chemo–immunotherapy in all patients other than those with node-negative tumors < 0.5 cm. This approach can lead to significant long-term effects for TNBC survivors, driving a growing interest in de-escalating therapy where appropriate. Tumor infiltrating lymphocytes (TILs) represent a promising prognostic and predictive biomarker for TNBC. These diverse immune cells are present in the tumor microenvironment and within the tumor itself, and multiple retrospective studies have demonstrated that a higher number of TILs in early-stage TNBC portends a favorable prognosis. Research has also explored the potential of TIL scores to predict the response to immunotherapy. However, several barriers to the widespread use of TILs in clinical practice remain, including logistical and technical challenges with the scoring of TILs and lack of prospective trials to validate the trends seen in retrospective studies. This review will present the current understanding of the role of TILs in TNBC and discuss the future directions of TIL research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179f3f7188fc55a777be68aae10786476ade4158" target='_blank'>
              Harnessing Tumor-Infiltrating Lymphocytes in Triple-Negative Breast Cancer: Opportunities and Barriers to Clinical Integration
              </a>
            </td>
          <td>
            Cara Coleman, Tharakeswari Selvakumar, A. Thurlapati, Kevin Graf, Sushma Pavuluri, Shikhar Mehrotra, Ozgur Sahin, A. Sivapiragasam
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The evolution of isocitrate dehydrogenase (IDH)-wildtype glioblastoma (GBM) after standard-of-care therapy remains poorly understood. Here we analyzed matched primary and recurrent GBMs from 59 patients using single-nucleus RNA sequencing and bulk DNA sequencing, assessing the longitudinal evolution of the GBM ecosystem across layers of cellular and molecular heterogeneity. The most consistent change was a lower malignant cell fraction at recurrence and a reciprocal increase in glial and neuronal cell types in the tumor microenvironment (TME). The predominant malignant cell state differed between most matched pairs, but no states were exclusive or highly enriched in either time point, nor was there a consistent longitudinal trajectory across the cohort. Nevertheless, specific trajectories were enriched in subsets of patients. Changes in malignant state abundances mirrored changes in TME composition and baseline profiles, reflecting the co-evolution of the GBM ecosystem. Our study provides a blueprint of GBM’s diverse longitudinal trajectories and highlights the treatment and TME modifiers that shape them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0990e85585d1a285c19cfc504ce05c954e344b" target='_blank'>
              Deciphering the longitudinal trajectories of glioblastoma ecosystems by integrative single-cell genomics
              </a>
            </td>
          <td>
            Avishay Spitzer, K. Johnson, M. Nomura, L. Garofano, Djamel Nehar-Belaid, Noam Galili Darnell, Alissa C. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, Simona Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, Jong Bae Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, Simone P Niclou, Labeeba Nusrat, Sorcha Kellet, Sunit Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, Marc Sanson, S. Tanaka, Nobuhito Saito, D.M. Ashley, Stephen T Keir, Keith L. Ligon, Jason T Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-05-09</td>
          <td>Nature Genetics</td>
          <td>2</td>
          <td>62</td>
        </tr>

        <tr id="Endometrial polyps are the predominant structural anomalies of the endometrial mucosa observed in unexplained infertility cases, potentially compromising endometrial receptivity and suggesting shared etiological characteristics. However, their comprehensive cell atlas and immune landscape remain inadequately defined. In this study, we employed single‐cell RNA sequencing and bulk RNA‐seq to systematically analyze ectopic endometrial polyps (EPs) alongside adjacent eutopic endometrial tissues (EUs). This enabled us to delineate alterations in cell composition and transcriptional dynamics across diverse cell types associated with endometrial polyps. Notably, we observed an increase and activation of mast cells, with significant transcriptional profile variations. Through transcription regulatory network analysis, WT1 was identified as a pivotal transcriptional regulator mediating mast cell proliferation in EPs, concomitant with the dysregulation of WT1 target genes involved in cell growth. These findings provide novel insights into the cellular heterogeneity and molecular mechanisms of endometrial polyps at single‐cell resolution, presenting potential therapeutic targets for clinical intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4c7abba5ac975d82b85fc31185d462113528bc2" target='_blank'>
              Single‐cell sequencing reveals a regulatory role of WT1 in mast cell proliferation in endometrial polyps
              </a>
            </td>
          <td>
            Yu Lin, Yiwen Qi, Yutong Yao, Huan Tong, Lan Chen, Weizhou Song, Guojing Li, Hong Xu, Xiaoying He
          </td>
          <td>2025-04-30</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The tumor immune microenvironment plays a critical role in tumor progression and immunotherapy response, with the abundance and composition of infiltrating immune cells serving as key determinants of therapeutic outcomes. Given the limitations of direct experimental methods, computational deconvolution algorithms are widely applied to infer immune cell infiltration from bulk RNA-seq data. While such estimates have proven valuable for studying tumor-immune interactions, large-scale, systematic analyses across multiple cancer types and treatment contexts remain limited. To address this need, we developed TIMER3 (https://compbio.cn/timer3/), an upgraded web server that substantially extends the functionality and scope of its predecessors. TIMER3 integrates 15 state-of-the-art immune deconvolution algorithms, including both human-specific and mouse-specific methods, to improve the robustness and interpretability of immune cell estimation. It incorporates an expanded collection of public RNA-seq datasets of immunotherapy-related cohorts, enabling large-scale analyses of immune dynamics across pre-treatment and post-treatment conditions. TIMER3 introduces new analytical modules for immunotherapy response, signature-based functional profiling, and interactive visualization of cell composition, gene expression, and survival associations. Collectively, TIMER3 provides a comprehensive, user-friendly platform for dissecting the TIME across diverse datasets, facilitating translational research and precision immuno-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916473c15b853461167c6dded26a3f5465619ac1" target='_blank'>
              TIMER3: an enhanced resource for tumor immune analysis.
              </a>
            </td>
          <td>
            Hao Cui, Guile Zhao, Yiwen Lu, Siying Zuo, Dingyu Duan, Xiaobo Luo, Hang Zhao, Jing Li, Zexian Zeng, Qianming Chen, Taiwen Li
          </td>
          <td>2025-05-08</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Ovarian cancer is one of the most common malignancies of the female reproductive system and is associated with poor prognosis. This study aimed to utilize single-cell RNA sequencing to investigate the heterogeneity of malignant epithelial cells in ovarian cancer, focusing on their potential functions and the implications for treatment and prognosis. Methods Single-cell RNA sequencing data were clustered using a single-cell transcriptome clustering method, and malignant epithelial cells were identified through copy number variation analysis. The interaction patterns between different malignant subpopulations and immune/stromal cells were analyzed using cell-to-cell communication analysis. A risk score (URS) model based on the UBE2C + epithelial subpopulation was then constructed through LASSO and multivariable Cox regression. High and low URS groups were compared in terms of tumor mutational burden (TMB), survival outcomes, and drug sensitivity. Finally, the role of GTF2F2 in ovarian cancer progression was validated through gene knockdown experiments in an ovarian cancer cell line (ES-2). Results Three major malignant epithelial cell subpopulations were identified (TMSB4X + Epi, TSC22D1 + Epi, and UBE2C + Epi). The UBE2C + Epi subpopulation exhibited higher stemness and greater invasive potential. The constructed URS model effectively stratified patients into high- and low-risk groups, with the high-risk group displaying a higher TMB level (p = 0.00011). Drug sensitivity predictions indicated that osimertinib, rapamycin, and dihydrorotenone might have stronger inhibitory effects in the high-risk group, whereas ERK inhibitors were more effective in the low-risk group. Functional assays demonstrated that GTF2F2 knockdown significantly suppressed ovarian cancer cell migration and invasion. Western blot analyses further showed elevated E-cadherin and reduced N-cadherin expression, suggesting that GTF2F2 may promote epithelial-mesenchymal transition (EMT). Conclusion The risk score model established in this study offers a novel framework for patient stratification and personalized therapy. Notably, the identification of the UBE2C + Epi subpopulation and key genes such as GTF2F2 highlights potential diagnostic and therapeutic targets, shedding light on the pathogenesis of ovarian cancer and paving the way for precision medicine approaches. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01686-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519b55d3039bd908fc634a743e91a97b194c1fa0" target='_blank'>
              Single-cell RNA sequencing reveals the role of GTF2F2 in ovarian cancer oncogenesis and progression
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-05-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f23b04434bb73a39ffc142be7c6ab3896c53a0" target='_blank'>
              Longitudinal single cell RNA-sequencing reveals evolution of micro- and macro-states in chronic myeloid leukemia
              </a>
            </td>
          <td>
            D. Frankhouser, Dandan Zhao, Yu-Hsuan Fu, Anupam Dey, Ziang Chen, Jihyun Irizarry, Jennifer Rangel Ambriz, S. Branciamore, Denis O’Meally, Lucy Ghoda, Jeffery M. Trent, Stephen J Forman, Adam L. Maclean, Ya-Huei Kuo, Bin Zhang, Russell C. Rockne, Guido Marcucci
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is the complex ecosystem surrounding a tumor. It is a complex and continuously evolving entity. The TME comprises a variety of complex components, such as cancer cells, immune cells, stromal cells, and extracellular matrix, that precisely regulate the interaction of tumor cells with other components, allowing tumor cells to continue to metastasize, escape immune surveillance, resist apoptosis, and proliferate. Researchers believe that the TME is not just a silent bystander but instead an active promoter of cancer progression. At the onset of tumor growth, a dynamic and reciprocal relationship is developed between cancer cells and components of the tumor microenvironment; this relationship supports cancer cell survival, local invasion, and metastatic dissemination. In this article, we discuss the role of the various components of the TME in the progression and metastasis of cancer, the pathways influenced by the TME, and the drugs that can treat tumors by targeting the components of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9869eb44a45ce4947a2fc2cac8a08a25b3dc82" target='_blank'>
              The Role of the Tumor Microenvironment in Cancer Progression and Metastasis
              </a>
            </td>
          <td>
            Oseghale, Ikalo David, Ikokwu, Godwin Mmaduabuchi, Ugoagwu, Kingsley Ugonna, Elizabeth Anuoluwa Akintayo, Matthew Gyang, Okiki Nosiru Quadri, Catherine Rono, O. Ralph-Okhiria, Odunvbun Praise
          </td>
          <td>2025-05-29</td>
          <td>Journal of Cancer and Tumor International</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Reactivation of transposable elements (TEs) in somatic tissues, particularly of LINE-1, is associated with disease by causing gene mutations and DNA damage. Previous work has shown that the PIWI pathway is crucial for TE suppression in the germline. However, the status and function of this pathway has not been well characterized in differentiated somatic cells and there is lack of consensus on the role of the pathway in somatic tumorigenesis. To shed light on this conundrum, we examined the PIWI pathway in colon cancer through a combination of bioinformatic analyses and cell-based assays. Shifted Weighted Annotation Network (SWAN) analysis revealed that the pathway experiences significant allelic losses in colon cancer and that PIWIL2, the main catalytic component of the pathway responsible for TE silencing, experiences the highest percent deletions. PIWIL2 is downregulated in colon tumors of advanced stage, nodal metastasis, and in certain subtypes, correlating with poor survival, while it is even downregulated in ulcerative colitis, an inflammatory bowel disease that predisposes to colon cancer. Knockout studies in colon epithelial Caco2 cells show that PIWIL2 depletion leads to increased anchorage-independent growth, increased LINE-1 levels and activity, and in DNA damage, altogether highlighting a tumor-suppressing role of PIWIL2 in the colon. SUMMARY STATEMENT This study investigates the PIWI-piRNA pathway in colon cancer using a nuanced bioinformatic and cell-based approach, linking the downregulation of PIWIL2 to disease progression, transposable element activation and DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6553455e0dd4bcf44406169f133238204316d7c1" target='_blank'>
              PIWIL2 downregulation in colon cancer promotes transposon activity and pro-tumorigenic phenotypes
              </a>
            </td>
          <td>
            Alyssa Risner, J. Nair-Menon, Abhinav Cheedipudi, Joe R Delaney, Vamsi K Gangaraju, A. Kourtidis
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ae51b6818a0b9bf42d218489a880c96562774e1" target='_blank'>
              Regulatory logic of human cortex evolution by combinatorial perturbations
              </a>
            </td>
          <td>
            Adrianos Skaros, Alessandro Vitriolo, Oliviero Leonardi, Veronica Finazzi, Marlene F. Pereira, Filippo Prazzoli, Sebastiano Trattaro, Juan Moriano, Daniele Capocefalo, C. E. Villa, Michael Boettcher, Cedric Boeckx, G. Testa
          </td>
          <td>2025-05-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>165</td>
        </tr>

        <tr id="Ovarian cancer (OC) is the most common malignant gynecological tumor. Cancer cells with high stemness often exhibit resistance to anti-tumor therapies, contributing to recurrence and poor prognosis. However, stemness-related subtypes in OC and their therapeutic implications remain underexplored.We identified stemness-associated genes by comparing transcriptome profiles between adherent and sphere-forming SKOV3 cells. Unsupervised clustering was applied to define stemness-related molecular subgroups in OC patients. A prognostic model was constructed using WGCNA and LASSO regression, and a nomogram was developed by integrating clinicopathological variables. Differences in the tumor microenvironment (TME), tumor mutation burden (TMB), immune checkpoint expression, and drug sensitivities were evaluated between risk groups. Single-cell RNA sequencing was used to investigate stemness-related cell types. Functional assays were conducted to validate the role of AKAP12 in OC progression.Three distinct stemness-related subgroups were identified with significant differences in prognosis and immunological features. Fibroblasts were identified as major contributors to the maintenance of stemness traits in the TME. AKAP12 was found to be positively associated with stemness phenotypes. Knockdown of AKAP12 reduced tumor sphere formation, impaired cell migration, and enhanced cisplatin sensitivity. Immunohistochemistry in clinical samples and OC organoids confirmed the correlation between AKAP12 and the immune checkpoint molecule OX40L.This study establishes a novel stemness-related gene signature for prognosis prediction and therapeutic stratification in OC. AKAP12 was identified as a potential biomarker and therapeutic target, offering new avenues for precision treatment in stemness-driven OC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e08eeb26e9b9f23cd59841fc893c5585796e22c1" target='_blank'>
              Stemness-driven clusters in ovarian cancer: immune characteristics and prognostic implications
              </a>
            </td>
          <td>
            Xinyan Zeng, Wentian Wu, Xiaoqin Li, Xiaorui Wu, Yingying Du, Ping Li
          </td>
          <td>2025-06-11</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 AI histopathology-inferred spatial transcriptomics as a driver of next-generation fast and cost effective precision oncology.
 I will start briefly reviewing my lab’s decade long work on developing a synthetic lethality approach for predicting patients’ response to therapy from tumor transcriptomics and whole slide pathology images [Cell 2021, MED 2023, Nature Cancer 2024]. The main part of my talk will focus on Path2Space, a new generic unpublished deep learning approach that predicts spatial gene expression directly from histopathology slides. Applied to study breast cancer as an example, Path2Space robustly predicts the spatial gene expression of more than 4000 genes and accurately infers cell-type abundances in the tumor microenvironment (TME) based on the inferred ST data. Path2Space characterizes thee TME of more than a 1000 breast cancer TCGA samples on an unprecedented scale and identifies three new spatially-grounded breast cancer patients subgroups with distinct survival rates. Closing a decade long circle, Path2Space- enables more accurate predictions of patients’ response to chemotherapy and trastuzumab directly from H&E slides than those obtained by existing established sequencing-based transcriptomics biomarkers. Path2Space thus offers a transformative approach to robustly delineate the tumors microenvironment directly from their histopathology slides. It facilitates the development of a new type of spatially-grounded biomarkers and lays the basis for a new generation of fast and low cost precision oncology biomarkers.



 Eytan Ruppin. AI inferred spatial transcriptomics unlocks cancer biomarker discovery from histopathology [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY14-01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46449f3c05fb0c5777c591d2b58948a0fca84cb4" target='_blank'>
              Abstract SY14-01: AI inferred spatial transcriptomics unlocks cancer biomarker discovery from histopathology
              </a>
            </td>
          <td>
            E. Ruppin
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Compelling evidences have manifested that breast cancer cells prefer to metastasize to certain distant organs, including brain, lung, bone and liver. According to the canonical “seed and soil” theory, this prominent biological behavior, termed as metastatic organotropism, involves intricate interactions between breast cancer cells (the “seeds”) and specific residents in the tumor microenvironment (the “soil”), initiating from pre-metastatic niche formation to metastatic outgrowth. Recently, multifaceted heterogeneity of tissue-specific macrophages (TSMs) and their roles played in organotropic metastases of breast cancer are incrementally unveiled. Herein, we decipher multiple diversities of TSMs, including evolvement, profiles, functions and metabolic characteristics under different polarization states. Further, we elaborate on bidirectional effects of TSMs on metastatic organotropism of breast cancer (both to the “seeds” and “soil”), and unearth underlying signaling pathways based on updated mechanistic researches. Lastly, we compile a series of clinical trials, hoping to illuminate promising TSM-targeting therapies against breast cancer organotropic metastases. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ed79b9316a9fed3f14b3bb67e4e61ce4c40709" target='_blank'>
              Heterogeneous tissue-specific macrophages orchestrate metastatic organotropism of breast cancer: implications for promising therapeutics
              </a>
            </td>
          <td>
            Cenzhu Wang, Pinchao Fan, Yu Zhou, Meican Ma, Haowei Zhong, Lei Liu, Qin Chen, Kun Xu
          </td>
          <td>2025-06-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aneuploidy, a hallmark of cancer, leads to widespread changes in chromosome copy number, altering the abundance of hundreds or thousands of proteins. How-ever, evidence suggests that levels of proteins encoded on affected chromosomes are often buffered toward their abundances observed in diploid cells. Despite its preval-ence, the molecular mechanisms driving this protein dosage compensation remain largely unknown. It is unclear whether all proteins are buffered to a similar degree, what factors determine buffering, and whether dosage compensation varies across different cell lines or tumor types. Moreover, its potential adaptive advantage and therapeutic relevance remain unexplored. Here, we established a novel approach to quantify protein dosage buffering in a gene copy number-dependent manner, show-ing that dosage compensation is widespread but variable in cancer cell lines and in vivo tumor samples. By developing multifactorial machine learning models, we identify mean gene dependency, protein complex participation, haploinsufficiency, and mRNA decay as key predictors of buffering. We also show that dosage com-pensation can affect oncogenic potential and that higher buffering correlates with reduced proteotoxic stress and increased drug resistance. These findings highlight protein dosage compensation as a crucial regulatory mechanism and a potential therapeutic target in aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63d2bd35509beaab88a25ac6ce826723195385e" target='_blank'>
              Explainable Machine Learning Identifies Factors for Dosage Compensation in Aneuploid Human Cancer Cells
              </a>
            </td>
          <td>
            Erik Marcel Heller, Karen Barthel, M. Räschle, Klaske M. Schukken, J. Sheltzer, Zuzana Storchová
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 e15100


 Background:
 Intratumoral heterogeneity, driven by phenotypic plasticity, contributes to disease progression and therapy failures. However, a thorough understanding of its underlying mechanisms is still lacking. Breast cancer exemplifies this with its histological and molecular diversity; estrogen receptor-positive (ER+), progesterone receptor-positive (PR+), and HER2- luminal A subtypes, such as the MCF7 cell line, are not the exception. While single-cell RNA sequencing (scRNA-seq) and three-dimensional multicellular tumor spheroids (MCTS) represent initial approaches to understanding heterogeneity, integrating these tools with computational strategies offers new paths to uncover the regulatory groundwork of tumor behavior. When modeled via Boolean frameworks, transcriptional regulatory networks (TRNs) provide a dynamic platform for studying gene interactions, identifying attractors (stable cellular states corresponding to distinct phenotypes), and simulating cellular transitions. This study represents the first step in constructing such a framework for ER+ breast cancer to advance precision medicine and improve therapeutic strategies.
 Methods:
 Regulatory elements relevant to the MCF7 cell line were identified through an extensive review of NCBI literature. Interactions were strictly curated based on evidence quality (ranging from robust experimental validation to predictive hypotheses), and evaluated for type and significance, such as feedback loops, activation, coactivation, and inhibition. Logical operators (AND, OR, NOT) were used to translate regulatory connections into Boolean functions to simulate binary gene expression dynamics. The final TRN was visualized using Cytoscape (v3.10.3) with a hierarchical layout.
 Results:
 The TRN consists of 54 nodes and 81 interactions, representing the regulatory landscape of ER+, PR+, and HER2- breast cancer. Boolean equations offer a framework for simulating the evolution of binary states to attractors, enabling the observation of how perturbations, such as gene overexpression or silencing, influence heterogeneity. While simulation and attractor analyses are ongoing, the network provides a foundation for elucidating the regulatory mechanisms underlying breast cancer plasticity.
 Conclusions:
 This study highlights the potential of integrating experimental data, TRN reconstruction, and Boolean modeling to understand intratumoral heterogeneity. Our rigorously curated model establishes a foundational framework for precision oncology and the integration of experimental and computational methods.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10947cef519fbed46d1a42e2676b4ea1fd778e16" target='_blank'>
              Boolean network modeling: The first step toward a new understanding of breast cancer heterogeneity.
              </a>
            </td>
          <td>
            Lia Medina Montalvo, Thelma Escobedo-Tapia, Osbaldo Resendis-Antonio
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83fa54668a1bde6cb09830187a5dd62f51b3deb5" target='_blank'>
              Glioblastoma shift from bulk to infiltrative growth is guided by plexin-B2-mediated microglia alignment in invasive niches.
              </a>
            </td>
          <td>
            Sangjo Kang, Mary E. Ughetta, Jack Y Zhang, Valerie Marallano, Anirudh Sattiraju, Theodore Hannah, Shalaka Wahane, Aarthi Ramakrishnan, Molly E. Estill, N. Tsankova, Li Shen, A. Tsankov, R. Friedel, Hongyan Zou
          </td>
          <td>2025-05-29</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Given their rarity and diversity, a fundamental understanding of the genomic underpinnings for many sarcoma subtypes is still lacking. To better define the molecular landscape of this group of diseases, we perform matched whole exome sequencing and RNA sequencing on a cohort of 1340 sarcoma tumor specimens. We identify recurrent somatic mutations and observe an increased mutational burden in metastatic vs. primary samples (p < 0.001). We observe frequent copy number alterations including whole genome doubling, with this feature being more common in metastatic tumors (p = 0.026). Estimation of immune cell abundances followed by hierarchical clustering identifies five immune subtypes ranging from low to high and we observe inferior overall survival in immune deplete clusters compared to immune enriched (p < 0.01). Interestingly, GIST predominantly form a distinct “immune intermediate” cluster that is marked by a specific enrichment for NK cells (FDR < 0.01).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09dedff6b3501b2376796d9aaef10264491ab2ba" target='_blank'>
              Genomic, transcriptomic, and immunogenomic landscape of over 1300 sarcomas of diverse histology subtypes
              </a>
            </td>
          <td>
            Alex C. Soupir, Oscar E. Ospina, Oliver Hampton, Michelle L. Churchman, Michael D. Radmacher, D. Hedges, David M. McKean, Phaedra Agius, Saman Zeeshan, Nathan D. Seligson, Raphael Pollock, D. Liebner, James L. Chen, G. Tinoco, B. Salhia, Martin D McCarter, B. Wilky, Benjamin J. Miller, M. Cavnar, John S. Groundland, Bryan P. Schneider, Gregory Riedlinger, Stephen B. Edge, C. Moskaluk, Kenneth Cardona, A. R. Naqash, Ricardo J. Gonzalez, J. Mullinax, David M Joyce, O. Binitie, G. Douglas Letson, A. Naghavi, M. Druta, Damon R Reed, E. Siegel, Jamie K Teer, Brooke L. Fridley, Andrew S Brohl
          </td>
          <td>2025-05-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Drug resistance results in poor outcomes for cancer patients. Adaptive therapy is a potential strategy to address drug resistance that exploits competitive interactions between sensitive and resistant subclones. Here, we showed that adapting carboplatin dose according to tumor response (adaptive therapy) significantly prolonged survival of murine ovarian cancer models compared to standard carboplatin dosing, without increasing mean daily drug dose or toxicity. Platinum resistant ovarian cancer cells exhibited diminished fitness when drug was absent in vitro and in vivo, which caused selective decline of resistant populations due to reduced proliferation and increased apoptosis. Conversely fitter, sensitive cells re-grew when drug was withdrawn. Using a bioinformatics pipeline that exploits copy number changes to quantify the emergence of treatment resistance, analysis of cell-free DNA obtained longitudinally from ovarian cancer patients during treatment showed subclonal selection through therapy, and measurements of resistant population growth correlated strongly with disease burden. These preclinical findings pave the way for future clinical testing of personalized adaptive therapy regimens tailored to the evolution of carboplatin resistance in individual ovarian cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872bfbc7aed3e8f94862c1f84bd0f838583a3b38" target='_blank'>
              Adaptive Therapy Exploits Fitness Deficits in Chemotherapy-Resistant Ovarian Cancer to Achieve Long-Term Tumor Control.
              </a>
            </td>
          <td>
            H. Hockings, E. Lakatos, Weini Huang, M. Mossner, Mohammed A Khan, Nikolina Bakali, Jacqueline McDermott, Kane Smith, Ann-Marie Baker, Trevor A. Graham, M. Lockley
          </td>
          <td>2025-04-29</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4eb70f970ca5d62a053191099abf0f49d770733" target='_blank'>
              Heterogeneous therapy-resistant cancer cells have distinct and exploitable drug sensitivity profiles
              </a>
            </td>
          <td>
            Gianna T. Busch, Ryan H. Boe, Jingxin Li, Robert F. Gruener, Miles Arnett, Pavithran T. Ravindran, M. Herlyn, R. S. Huang, Arjun Raj
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="
 Melanoma, a highly aggressive form of skin cancer, presents a significant clinical challenge. The identification of BRAFV600E mutations as a driver in roughly half of all melanoma cases has led to the development of targeted therapies, exemplified by Vemurafenib (Vem), which inhibits overactivated MAPK signaling. However, frequent relapses remain a major obstacle. To investigate the complex mechanisms driving drug resistance in melanoma, we developed PerturbFate, a scalable single-cell combinatorial-indexing strategy that integrates multi-modal profiling with CRISPR perturbations. This approach simultaneously captures newly synthesized and pre-existing transcriptomes, chromatin accessibility, and CRISPR sgRNA identity from individual cells. Using this platform, we profiled BRAFV600E melanoma cells (A375) after knocking down each of 143 potential drug resistance candidate genes, both with and without Vem treatment, in a pooled manner. We identified extensive phenotypic state transitions in melanoma cells under genetic and chemical perturbations, revealing that an undifferentiated state is associated with enhanced drug resistance, while a more benign, Vem-responsive melanocytic state exists at the opposite end of the phenotypic spectrum. Temporal gene regulatory network analyses revealed that AP-1 transcription factors (TFs), particularly FOSL1, pioneer other diverse TFs to drive the melanoma dedifferentiation. In contrast, SOX10 is the key determinant of the melanocytic state. We identified MAPK pathway, which primarily promotes proliferation, and Hippo/YAP signaling, which predominantly drives the undifferentiated state transition, as convergent key pathways mediating drug resistance. Their synergistic interplay, executed through the collaborative activation of downstream TFs (e.g., FOSL1, RREB1, SMADs, KLF5)-mediated gene expression, is a critical driver of Vem resistance. To elucidate the mechanisms underlying the observation that perturbation of genes encoding the Mediator Complex components (e.g., MED12, CCNC, MED13, MED19, MED15, MED24) induced autonomous dedifferentiation and/or resistance to Vem in melanoma, we established linkages between their distinct biochemical properties and heterogeneous perturbation responses. We also identified the convergence of dysregulated inflammation, potentially mediated by VEGFC and interleukins (ILs), with Hippo/YAP-mediated transcriptional responses in driving melanoma dedifferentiation and Vem resistance following perturbation of mediator complex genes. Altogether, our study introduces a novel single-cell multi-modal CRISPR perturbation platform, which we applied to comprehensively chart the responses of melanoma cells to the perturbation of all potential Vem resistance candidate genes. These findings pave the way for the identification of precise targets for combination therapies with enhanced efficacy against melanoma drug resistance.


 Zihan Xu, Ziyu Lu, Aileen Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao. Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e1b2442ed827fe07315db2d2daebd9063c39b" target='_blank'>
              Abstract LB048: Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary Cancer initiation and early tumour growth involve not only genetic mutations, but also important changes in the tumour microenvironment. The extracellular matrix (ECM) in particular undergoes biochemical and mechanical remodelling, influencing cell polarity and abnormal vascular development to drive cancer initiation and tumorigenesis. MicroRNAs (miRNAs) further regulate these early processes by altering gene expression, promoting invasion, and contributing to tumour plasticity. Recent advances in in vitro modelling have aimed to replicate tumorigenic environments using interdisciplinary approaches to better recapitulate in vivo environments. This review discusses how ECM mechanics, cellular polarity loss, and miRNA dysregulation collectively contribute to tumour progression. It also highlights how current/emerging in vitro models, including self-assembling peptide hydrogels and bioprinting technologies, provide insight into strategies to recapitulate cancer initiation and early tumour growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bd99c12367a85c8693deebdc80eb89c3d308d6e" target='_blank'>
              Modelling Cancer Pathophysiology: Mechanisms and Changes in the Extracellular Matrix During Cancer Initiation and Early Tumour Growth
              </a>
            </td>
          <td>
            Luis Larrea Murillo, Megan Green, Niall Mahon, A. Saiani, Olga Tsigkou
          </td>
          <td>2025-05-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by significant heterogeneity and a poor prognosis. M2 macrophages (M2) play a crucial role in the microenvironment of PDAC, yet their exact functional significance remains poorly understood. This study sought to explore the potential communication and clinical implications of M2 in PDAC. The differentiation pathway of M2 and their interaction with malignant PDAC cells were analyzed using single-cell RNA sequencing (scRNA-seq). Additionally, a prognostic signature related to M2 marker genes was constructed and analyzed using bulk RNA sequencing. Three distinct differentiation states of M2 were characterized, each exhibiting unique transcription factor expression profiles and signaling pathway activations. Potential intercellular communication between M2, PDAC malignant cells, and fibroblasts was initially observed at the level of computer analysis. The prognostic model was identified through the LASSO algorithm. Patients stratified by risk level displayed significant variations in prognostic indicators such as signaling pathway activity, gene mutation frequency, immune cell infiltration, and drug sensitivity. These findings are of course preliminary, and future in vitro/in vivo experiments will be needed to knock out these signature genes or ligand-receptor pairs of genes in macrophages to validate their role in macrophage development. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02838-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/273ead8bc63462bd01d5d5a63b43ec9833ef2540" target='_blank'>
              Establishment of M2 macrophage-related prognostic signature in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Hui Chen, Jingyang Yin, Xianxing Wang, Renpei Xia, Fanbo Qin, Zhuo Li, Jiali Yang, Changjiang Liu, Shixiang Guo, Huaizhi Wang
          </td>
          <td>2025-06-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Increased stromal fibroblasts in tumors is associated with inferior radiotherapy responses, however, we lack a mechanistic understanding of how fibroblast reprogramming contributes to radiotherapy outcomes. Methods: Radiotherapy (8Gy) was applied to 5555 BRAF mutant melanoma flank tumors grown in WT and RAG1 KO Bl/6 mice. Tumors were harvested at early (1 week post radiation) and late timepoints. 1D11, which inhibits all three isoforms of TGFβ, or the relevant isotype control, was administered from 24 hours prior to irradiation for one week. Multiplex immunohistochemistry and flow cytometry evaluated biological changes following treatment. A Cre-inducible murine model of podoplanin (PDPN) knockout was used to evaluate the specific role of PDPN expression in fibroblasts. NanoString GeoMx spatial profiling was used to evaluate transcriptional changes in different tumour regions. Results: An extensive peri-tumoral expansion of PDPN+ fibroblasts was observed after radiation in WT mice but not in RAG1 KO mice where tumours grew more rapidly. Radiation induced entry of CD8+ T cells which remained sequestered at the fibroblast/tumor cell interface, with minimal entry into the tumor core. Upregulation of inflammatory signaling was identified in PDPN+ fibroblasts at this interface with tumor cells but not elsewhere. The addition of 1D11 to irradiation promoted T cell entry into the tumor core however a paradoxical increase in TGFβ signaling was observed specifically in PDPN fibroblasts with 1D11. When PDPN knockout was induced shortly before irradiation, a higher rate of spontaneous tumor rejection was observed. The PDPN+ fibroblast phenotype observed in the above pre-clinical models is also seen across a range of human tumors both pre- and post-radiation. Conclusion: Rapid stromal reprogramming after irradiation, involving PDPN+ fibroblast expansion and associated inflammatory signaling, restricts T cell entry into tumors which appears to compromise tumor control



 Anna Wilkins. Interrogating how the dialogue between immune cells and fibroblasts impacts radiotherapy outcomes [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY36-02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8b79d2a6ba6afcb9ee47e35a6d8728ff57e8ff1" target='_blank'>
              Abstract SY36-02: Interrogating how the dialogue between immune cells and fibroblasts impacts radiotherapy outcomes
              </a>
            </td>
          <td>
            Anna Wilkins
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite the advances in detection, diagnosis, and treatments, cancer remains a lethal disease, claiming the lives of more than 600,000 people in the United States alone in 2024. To accelerate the development of new therapeutic strategies with improved responses, significant efforts have been made to develop microfabricated in vitro models of tumor microenvironments (TMEs) that address the limitations of animal-based cancer models. These models incorporate several advanced tissue engineering techniques to better reflect the organ- and patient-specific TMEs. Additionally, microfabricated models integrated with next-generation single-cell omics technologies provide unprecedented insights into patient's cellular and molecular heterogeneity and complexity. This review provides an overview of the recent understanding of cancer development and outlines the key TME elements that can be captured in microfabricated models to enhance their physiological relevance. We highlight the recent advances in microfabricated cancer models that reflect the unique characteristics of their organs of origin or sites of dissemination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a86d7d422483613dc814a73531cbae554734924" target='_blank'>
              Microfabricated Organ-Specific Models of Tumor Microenvironments.
              </a>
            </td>
          <td>
            Jeong Min Oh, Yongkuk Park, Jungwoo Lee, Keyue Shen
          </td>
          <td>2025-05-01</td>
          <td>Annual review of biomedical engineering</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Temozolomide (TMZ) is an important chemotherapeutic agent for glioma treatment. However, the emergence of drug resistance impedes its application. Traditional population-level studies are limited in elucidating resistance mechanisms. Advances in single-cell and spatial transcriptomics technologies provide feasible resolution for studying the cellular composition and dynamics of tumors. In this study, we investigated the heterogeneity of gliomas associated with TMZ resistance at the single-cell and spatial transcriptome levels to identify the resistance mechanisms and potential therapeutic strategies. Methods Single cell sequencing technology was utilized to identify the cellular clusters of gliomas. Drug perturbation analysis and cellular propensity analysis revealed key cluster responding to TMZ. Enrichment analysis was preformed to explore the function of clusters. Transcription factor activity analysis revealed key transcription factors contributing to tumor resistance. Spatial transcriptome data and bulk RNA-seq data validates the role of key transcription factors. Downstream targets of key transcription factors were predicted and validated using gene regulation assays. Drug sensitivity analyses were used to seek viable strategies to overcome drug resistance. Results Glioma cells from before and after temozolomide treatment samples were classified into six clusters: NPC-like cluster, OPC-like cluster, MES-like cluster, AC-like cluster, OC, and Neuron. NPC-like clusters exhibited strong stemness and DNA repair capacity. The activity of MAZ in NPC-like cluster was significantly enhanced after TMZ treatment. The proportion of MAZ( +)_NPC-like cluster was higher in TMZ treated samples. Patients with high proportion of MAZ( +)_NPC-like cluster had poorer survival. Upregulation of MAZ is able to enhance drug resistance in glioma cells, but this phenomenon disappeared when FoxM1 expression was further silenced. The combination of paclitaxel and Trametinib is a promising strategy to overcome resistance. Conclusions NPC-like cluster is prevalent in recurrent and drug-resistant gliomas. MAZ transcription factors are critical regulators that promote the development of drug resistance in NPC-like clusters by enhancing the capacity of DNA repair and stemness. Patients with high proportions of MAZ( +)_NPC-like clusters have poor TMZ sensitivity and prognosis. MAZ enhances stemness and drug resistance in glioma cells by upregulating FOXM1 expression. The combination of paclitaxel and paclitaxel is a promising therapeutic strategy for treating gliomas and overcoming drug resistance. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06706-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c31c53340b2d02203f651d970425a0fc3e3628" target='_blank'>
              Single-cell and spatial transcriptome analyses reveal MAZ(+) NPC-like clusters as key role contributing to glioma recurrence and drug resistance
              </a>
            </td>
          <td>
            Qiang Gu, Kan Wang, Tingming Lu, Yifei Xiao, Yaqing Wu, Hang Zhou, Kaijia Zhou
          </td>
          <td>2025-06-16</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41cd39016a4b2c876a2210fd9227cfa83893338" target='_blank'>
              Unraveling Tyrosine-Kinase Inhibitor Resistance in NSCLC Cells via Same-cell Measurement of RNA, Protein, and Morphological Responses
              </a>
            </td>
          <td>
            Vivian Dien, Connor Thompson, Tristin Rammel, Jaime Roquero, Marielle Krivit, Grace Yeo, Daniel Honigfort, Pazinah Bhadha, Colin Brown-Greeves, Mariam Dawood, Keerthana Elango, Hyukjin J. Kwon, Tyler Lopez, Alvin Lo, Meghna Manjunath, Pin Ren, Ben Song, Mitch Sundkamp, Ramreddy Tippana, Wilfred Van Ijcken, Jennifer Wong, Jane Yao, Da Zhou, Sinan Arslan, Matthew Kellinger, Amirali Kia, S. Kruglyak, Michael Previte, Molly He
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="In isocitrate dehydrogenase wildtype glioblastoma (GBM), cellular heterogeneity across and within tumors may drive therapeutic resistance. Here we analyzed 121 primary and recurrent GBM samples from 59 patients using single-nucleus RNA sequencing and bulk tumor DNA sequencing to characterize GBM transcriptional heterogeneity. First, GBMs can be classified by their broad cellular composition, encompassing malignant and nonmalignant cell types. Second, in each cell type we describe the diversity of cellular states and their pathway activation, particularly an expanded set of malignant cell states, including glial progenitor cell-like, neuronal-like and cilia-like. Third, the remaining variation between GBMs highlights three baseline gene expression programs. These three layers of heterogeneity are interrelated and partially associated with specific genetic aberrations, thereby defining three stereotypic GBM ecosystems. This work provides an unparalleled view of the multilayered transcriptional architecture of GBM. How this architecture evolves during disease progression is addressed in the companion manuscript by Spitzer et al.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c45eaa99273988e47319564458ae28d00342e00" target='_blank'>
              The multilayered transcriptional architecture of glioblastoma ecosystems
              </a>
            </td>
          <td>
            M. Nomura, Avishay Spitzer, K. Johnson, L. Garofano, Djamel Nehar-Belaid, Noam Galili Darnell, Alissa C. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, Simona Migliozzi, L. G. Gonzalez Castro, Tamrin ChowdhFury, N. Robine, C. Reeves, Jong Bae Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, Simone P Niclou, Labeeba Nusrat, Sorcha Kellet, Sunit Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, Marc Sanson, S. Tanaka, Nobuhito Saito, D.M. Ashley, Stephen T Keir, Keith L. Ligon, Jason T Huse, W. Yung, A. Lasorella, R. Verhaak, A. Iavarone, M. Suvà, I. Tirosh
          </td>
          <td>2025-05-09</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>62</td>
        </tr>

        <tr id="Prostate cancer (PCa) is the second most common cancer and the second leading cause of cancer-related mortality among men. Gene expression analysis has been crucial in understanding tumor biology and providing disease progression markers. Cell surface glycoproteins and those in the extracellular matrix play significant roles in the PCa microenvironment by promoting migration, invasion, and metastasis. The molecular and histopathological heterogeneity of prostate tumors necessitates a new marker discovery to better stratify patients at risk for poor prognosis. In this study, our objectives were to investigate and characterize the localization and expression of SLIT/ROBO in PCa samples from transgenic mice and human tumor samples, aiming to identify novel prognostic markers and potential therapeutic targets. We conducted histopathological, immunohistochemical, and bioinformatics analyses on prostate tumors from two knockout mice models (Pb-Cre4/Ptenf/f and Pb-Cre4/Trp53f/f;Rb1f/f) and human prostate tumors. Transcriptomic analyses revealed special changes in the expression of genes related to the SLIT/ROBO neural signaling pathway. We further characterized the gene and protein expression of the SLIT/ROBO pathway in knockout animal samples, and protein expression in the PCa samples of patients with different Gleason scores. Public datasets with clinical data from patients (The Human Protein Atlas, cBioPortal, SurvExpress and CamcAPP) were used to validate the gene and protein expression of SLIT1, SLIT2, ROBO1, and ROBO4, correlating these alterations with the prognosis of subgroups of patients. Our findings highlight potential biomarkers of the SLIT/ROBO pathway with prognostic and predictive value, as well as promising therapeutic targets for PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434696c39815cfe10c4acf784bb41298dbd8e27e" target='_blank'>
              SLIT/ROBO Pathway and Prostate Cancer: Gene and Protein Expression and Their Prognostic Values
              </a>
            </td>
          <td>
            N. J. Santos, F. C. Mosele, C. N. Barquilha, Isabela C Barbosa, Flávio de Oliveira Lima, G. O. Barbosa, H. F. Carvalho, F. K. Delella, S. Felisbino
          </td>
          <td>2025-05-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is challenging to treat due to its immunosuppressive tumor microenvironment (TME) and resistance to immune checkpoint inhibitors. This study aims to discover new therapeutic targets and predictive biomarkers for PDAC. Methods Using Mendelian randomization, we studied causal relationships between PDAC and an array of immune cell traits, bacterial traits, inflammatory factors, and blood metabolites. We employed large genome-wide association study datasets and the two-sample MR approach for the investigation. Results Our results highlight suggestive evidence of associations between PDAC and distinct immune cell phenotypes, revealing nuanced alterations across monocytes, T-cells, B-cells, dendritic cells, and myeloid-derived suppressor cells. Our study provides a granular view of the PDAC-immune interface, identifying key immune cell traits and their associations with PDAC. For instance, our findings suggest a detrimental reduction in various monocyte traits, alongside a decrease in B-cell populations. Conversely, certain T-cell subsets showed increased associations, indicating potential targets for immunotherapeutic strategies. The bacterial trait associations, particularly with Collinsella and Ruminococcus torques, highlight the gut microbiome's influence on immune modulation and PDAC pathogenesis. Additionally, the traits concerning Interleukin-12 subunit beta levels and T-cell surface glycoprotein CD5 levels further indicate their function of this complex interaction. Conclusions This study enhances our understanding of PDAC's resistance to immunotherapies and highlights the potential of personalized immunotherapy and metabolic pathway modulation in PDAC treatment. Our findings provide supportive evidence for research and clinical translation. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02250-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c74862182c3a4e9643e24cf48c912c5e5651c" target='_blank'>
              Unveiling the immunological terrain of pancreatic ductal adenocarcinoma: strategies to prompt immunotherapy from Mendelian randomization
              </a>
            </td>
          <td>
            Peizan Ni, Lilin Li, Kunpeng Du, Pengkhun Nov, Duanyu Wang, Changqian Wang, Qianzi Kou, Ying Li, Yangfeng Zhang, Chongyang Zheng, Wen Fu, Jiqiang Li
          </td>
          <td>2025-04-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2c94a8a149ed8b89b155e50f8c13949352a8f13" target='_blank'>
              Molecular dynamics driving phenotypic divergence among KRAS mutants in pancreatic tumorigenesis
              </a>
            </td>
          <td>
            A. Grimont, David J. Falvo, Whitney J. Sisso, P. Zumbo, Christopher W. Chan, Francisco Santos, Grace Pan, Megan Cleveland, Tomer M. Yaron, Alexa Osterhoudt, Yinuo Meng, Maria Paz Zafra, William B. Fall, André F. Rendeiro, Erika Hissong, R. Yantiss, D. Betel, Mark A. Magnuson, Steven D. Leach, Anil K. Rustgi, Lukas E. Dow, R. Chandwani
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The remarkable advancement of single-cell RNA sequencing (scRNA-seq) technology has empowered researchers to probe gene expression at the single-cell level with unprecedented precision. To gain a profound understanding of the heterogeneity inherent in cell fate determination, a central challenge lies in the comprehensive analysis of the dynamic regulatory alterations that underlie transcriptional differences and the accurate inference of the differentiation trajectory. Here, we propose the method scRDEN, a robust framework that infers important cell sub-populations and differential expression networks of multiple genes along the differentiation directions of each branch by converting the unstable gene expression values in cells into relatively stable gene-gene interactions (global features) and extracting the order of differential expression (network features), and further integrating the expression features of different dimension reduction methods. When applied to five published scRNA-seq datasets from human and mouse cell differentiation, scRDEN not only successfully captures the stable cell subpopulations with potential marker genes, measures the transcriptional differences of gene pairs to identify the rank differential expression network along the differentiation direction of each branch. In addition, in multiple gene rank differential expression networks, the rank expression directly related to transcription factors/marker genes shows a significant strengthening and weakening trend along with their expression changes, and the distribution of diversity and cluster coefficient show a non-monotonic change trend, including the cases of increasing first and then decreasing or decreasing first and then increasing. This may correspond to the mechanism of cells gradually differentiating into stable functions. It is particularly noteworthy that scRDEN method yielded exceptional results when applied to the large-scale, multi-branched, double-batch mouse dentate gyrus data. This outstanding performance provides novel and valuable insights into large-scale, multi-batch trajectory inference and the study of transcriptional mechanism regulation during the processes of differentiation and development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fec63e89c70ec4c20481861c050a6fc9e78267c1" target='_blank'>
              scRDEN: single-cell dynamic gene rank differential expression network and robust trajectory inference
              </a>
            </td>
          <td>
            Hang Zhang, Wei Zhang, Xiaoying Zheng, Yuanyuan Li
          </td>
          <td>2025-05-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metabolic‐associated steatotic liver disease (MASLD) is among the most prevalent liver disorders worldwide, with many patients progressing to metabolic‐associated steatohepatitis (MASH) characterised by fibrosis and inflammation. The current lack of effective treatments for MASH highlights the urgent need to deepen our understanding of its underlying mechanisms. Examining cellular dynamics—specifically, changes in cell type proportions across disease stages—offers a promising avenue for gaining such insights. However, previous deconvolution analyses have been limited to a few cell types, and a comprehensive analysis encompassing diverse cell populations and their unique subtypes has yet to be conducted. In this study, we employed MuSiC deconvolution to analyse two bulk RNA sequencing datasets spanning the MASLD spectrum across both fibrosis staging and Non‐Alcoholic Fatty Liver Disease Activity Score (NAS) staging. Our analysis reveals distinct proportional trends in 10 different cell types, including hepatocytes, cholangiocytes, two subpopulations of hepatic stellate cells, endothelial cells, and immune cells such as kupffer cells, TREM2+ macrophages, and plasma B cells. In addition to deconvolution analysis, we integrated cell type proportion data with transcriptomic profiles, significantly enhancing the performance of random forest models in classifying fibrosis stages compared to using transcriptomic data alone. The study's findings highlight critical cellular dynamic changes across MASLD progression, advancing our understanding of the disease mechanisms and potentially informing the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc7b46b0292c8267b1eab868b1b7aa97bbed3ae3" target='_blank'>
              Cell‐Type Deconvolution Reveals Dynamic Changes in MASLD
              </a>
            </td>
          <td>
            Jeff J. H. Kim, Yang Dai
          </td>
          <td>2025-04-28</td>
          <td>Liver International Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Targeted therapy interventions using tyrosine kinase inhibitors (TKIs) provide encouraging treatment responses in patients with ALK-rearranged lung adenocarcinomas, yet resistance occurs almost inevitably. In addition to tumor cell-intrinsic resistance mechanisms, accumulating evidence suggests that cancer-associated fibroblasts (CAFs) within the tumor microenvironment contribute to therapy resistance. This study aimed to investigate CAF-driven molecular networks that shape the therapeutic susceptibility of ALK-driven lung adenocarcinoma cells. Methods Three-dimensional (3D) spheroid co-cultures comprising ALK-rearranged lung adenocarcinoma cells and CAFs were utilized to model the tumor microenvironment. Single-cell RNA sequencing was performed to uncover transcriptional differences between TKI-treated homotypic and heterotypic spheroids. Functional assays assessed the effects of CAF-conditioned medium and CAF-secreted factors on tumor cell survival, proliferation, lipid metabolism, and downstream AKT signaling. The therapeutic potential of targeting metabolic vulnerabilities was evaluated using pharmacological inhibition of lipid metabolism and by ferroptosis induction. Results CAFs significantly diminished the apoptotic response of lung tumor cells to ALK inhibitors while simultaneously enhancing their proliferative capacity. Single-cell RNA sequencing identified lipogenesis-associated genes as a key transcriptional difference between TKI-treated homotypic and heterotypic lung tumor spheroids. CAF-conditioned medium and the CAF-secreted factors HGF and NRG1 activated AKT signaling in 3D-cultured ALK-rearranged lung tumor cells, leading to increased de novo lipogenesis and suppression of lipid peroxidation. These metabolic adaptations were critical for promoting tumor cell survival and fostering therapy resistance. Notably, both dual inhibition of ALK and the lipid-regulatory factor SREBP-1, as well as co-treatment with ferroptosis inducers such as erastin or RSL3, effectively disrupted the CAF-driven metabolic-supportive niche and restored sensitivity of resistant lung tumor spheroids to ALK inhibition. Conclusions This study highlights a critical role for CAFs in mediating resistance to ALK-TKIs by reprogramming lipid metabolism in ALK-rearranged lung cancer cells. It suggests that targeting these metabolic vulnerabilities, particularly through inhibition of lipid metabolism or induction of ferroptosis, could provide a novel therapeutic approach to overcome resistance and improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40170-025-00400-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a02a6efa6497a2bcee01f9f0743c25e6ad43b8" target='_blank'>
              Cancer-associated fibroblasts promote drug resistance in ALK-driven lung adenocarcinoma cells by upregulating lipid biosynthesis
              </a>
            </td>
          <td>
            Ann-Kathrin Daum, Lisa Schlicker, M. Schneider, T. Muley, Ursula Klingmüller, Almut Schulze, Michael Thomas, P. Christopoulos, Holger Sültmann
          </td>
          <td>2025-06-16</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Clinically aggressive lactotroph pituitary neuroendocrine tumors (PitNET) are invasive tumors with an unusually rapid growth rate despite maximally tolerated doses of dopamine agonist (DA). We aimed to unravel the molecular heterogeneity of lactotroph PitNET and to identify biomarkers of aggressiveness and resistance to pharmacological treatment. A total of 13 patients harboring DA-resistant lactotroph PitNET were included in this study. Visium Spatial Transcriptomics (ST), whole transcriptome sequencing (WTS), and whole exome sequencing (WES) were performed in tumors from 4 of these patients; WTS and WES was carried out in 5; tumors from two patients underwent ST and WES and tumors from two other patients underwent only ST. Tumors were classified as null or partial responders according to their response to DA treatment. The eight PitNET analyzed by ST exhibited significant intratumoral heterogeneity, with clones showing alterations in PI3K/AKT and lipid metabolism pathways, particularly inositol phosphate, glycerophospholipid, and sphingolipid metabolism. The cell-cell communication analysis showed FGF-FGFR ligand receptor interaction whilst the transcription factors RXRA and CREM showed participation in both groups. A trajectory exploration was performed by including all PitNET together in a single analysis to determine whether there was a tendency or molecular pathway showing a differentiation pattern that would guide the transition from a partially responsive PitNET to a completely unresponsive one. We did not observe any such pattern. All of these findings were corroborated in the cohort of DA-resistant PitNETs in which only bulk WTS and WES were performed. The bulk WTS corroborated lipid metabolism and PI3K-AKT pathway alteration in PitNET, whereas the WES showed only SF3β1 and TP53 variants in one tumor each. Our work suggests that the PI3K/AKT pathway may constitute a molecular target at which to aim therapeutic strategies designed to treat aggressive and DA-resistant lactotroph PitNET. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02025-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9584b357aff2d73647fc85b3bbd6a84a02ea98" target='_blank'>
              Spatial transcriptomics reveal PI3K-AKT and metabolic alterations in aggressive, treatment-resistant lactotroph pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            F. Martinez-Mendoza, Sergio Andonegui-Elguera, E. Sosa-Eroza, Erick Gomez-Apo, Aurea Escobar-España, Carolina González-Torres, Javier Gaytán-Cervantes, Alam Palma-Guzman, Hugo Torres-Flores, Alberto Moscona-Nissan, S. Hinojosa-Alvarez, J. Hernández-Perez, R. Chavez-Santoscoy, G. Guinto, Gerardo Y. Guinto-Nishimura, B. López-Félix, Erick U. Zepeda-Fernandez, Erick M. Estrada-Estrada, V. Correa-Correa, P. A. González-Zavala, Marco A. Asenscio-Montiel, M. A. García-Vargas, Emmanuel Cantu-Chavez, Rocío Arreola-Rosales, Keiko Taniguchi-Ponciano, D. Marrero-Rodríguez, Moises Mercado
          </td>
          <td>2025-05-19</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Among the various types of breast cancer that endanger women’s lives, triple-negative breast cancer (TNBC) stands out due to its extreme heterogeneity, aggressive nature, and high likelihood of recurrence. The absence of unique targets and targeted drugs is a major factor contributing to the failure of cancer treatments and the eventual death of patients. Methods Single-cell RNA sequencing (scRNA-seq) was applied to investigate the immune microenvironment of TNBC, facilitating the detection of key cell subpopulations and regulatory genes. The mRNA expression of SLC31A1 in macrophages was measured by qPCR. Flow cytometry was utilized to ascertain the M2 macrophage proportion and cancer cell apoptosis. Transwell and scratch assays were conducted to gauge cancer cell migration and invasion. Copper ion and H2O2 detection kits were employed to evaluate the copper ion content and oxidative stress levels in macrophages. Results SLC31A1 overexpression in TNBC myeloid cells, particularly macrophage subpopulations, was identified through scRNA-seq analysis. Cluster and pseudotime analyses showed that macrophages with high SLC31A1 expression are often in advanced stages of cell growth, accompanied by notable changes in oxidative stress. Functional studies revealed that knocking down SLC31A1 in macrophages significantly reduced M2-type polarization. The conditioned medium from these macrophages markedly inhibited TNBC cell migration and invasion, while promoting apoptosis. Furthermore, SLC31A1 knockdown resulted in decreased copper ion content and H2O2 levels in macrophages. Conclusion SLC31A1 enhances the malignant phenotype of TNBC cells by inducing M2 polarization in macrophages. Supplementary Information The online version contains supplementary material available at 10.1007/s00432-025-06191-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b15126a7f2cb660d27b7680fac8fa230c8c3d5" target='_blank'>
              Single-cell RNA sequencing reveals SLC31A1-mediated M2 polarization of macrophages promotes malignant progression in triple-negative breast cancer
              </a>
            </td>
          <td>
            Xiwei Zhang, Guoqing Li, Tieyan Chen, Haohang Sun, J. Dai, Qi Chen, Mengze Chen, Meidi Yan
          </td>
          <td>2025-05-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Matrix metalloproteinase 1 plays a pivotal role in tumor biology and immune modulation through its enzymatic remodeling of the extracellular matrix, facilitating tumor progression. In this study, we utilized large-scale single-cell RNA sequencing and spatial transcriptomics to investigate MMP1 expression, its cellular localization, and its impact on tumor progression and immune modulation. Our findings reveal that MMP1 expression is elevated in various tumor types and is strongly correlated with metastatic potential. High MMP1 expression was associated with increased activity in epithelial–mesenchymal transition signaling and TNFα/NF-κB pathways, which are known to promote tumor progression. Furthermore, MMP1+ malignant cells exhibited significant interactions with immune cells, particularly macrophages and CD8+ T cells. MMP1 expression correlated with enhanced macrophage infiltration and impaired CD8+ T-cell function, contributing to an immunosuppressive tumor microenvironment. Notably, the CXCL16-CXCR6 and ANXA1-FPR3 signaling axes were identified as key mediators of these interactions. Inhibition of MMP1 in vitro demonstrated reduced cell invasion, stemness, and proliferation, while increasing reactive oxygen species levels and promoting apoptosis. Our findings position MMP1 as a key player in the “tumor-immune” vicious cycle and a promising therapeutic target to enhance anti-tumor responses and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d0c741845c10c07b47cb268ec3e101fb8935759" target='_blank'>
              Decoding the impact of MMP1+ malignant subsets on tumor-immune interactions: insights from single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Da-Ming Xu, Ling-Xiao Chen, Ting Xue, Xiao-Yu Zhuang, Li-Chao Wei, Hui Han, Miao Mo
          </td>
          <td>2025-05-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Addressing the growing need for organized data on tumor homing peptides (THPs), we present TumorHoPe2, a manually curated database offering extensive details on experimentally validated THPs. This represents a significant update to TumorHoPe, originally developed by our group in 2012. TumorHoPe2 now contains 1847 entries, representing 1297 unique tumor homing peptides, a substantial expansion from the 744 entries in its predecessor. For each peptide, the database provides critical information, including sequence, terminal or chemical modifications, corresponding cancer cell lines, and specific tumor types targeted. The database compiles data from two primary sources: phage display libraries, which are commonly used to identify peptide ligands targeting tumor-specific markers, and synthetic peptides, which are chemically modified to enhance properties such as stability, binding affinity, and specificity. Our dataset includes 594 chemically modified peptides, with 255 having N-terminal and 195 C-terminal modifications. These THPs have been validated against 172 cancer cell lines and demonstrate specificity for 37 distinct tumor types. To maximize utility for the research community, TumorHoPe2 is equipped with intuitive tools for data searching, filtering, and analysis, alongside a RESTful API for efficient programmatic access and integration into bioinformatics pipelines. It is freely available at https://webs.iiitd.edu.in/raghava/tumorhope2/">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4673f139ff89b98a8a9cf242e48538be8a2f0a9d" target='_blank'>
              TumorHoPe2: An updated database for Tumor Homing Peptides
              </a>
            </td>
          <td>
            Diksha Kashyap, Devanshi Gupta, Naman Kumar Mehta, G. Raghava
          </td>
          <td>2025-05-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Three-dimensional in vitro cell models have gained popularity as they better recapitulated key aspects of human tissues and tumor microenvironments. Forming structures with patient-derived (PD) cancer cells show great promise for translational work. These 3D models are being used for drug discovery and development, but adoption has been limited in part due to challenges related to sample handling, assay development, and the need for optimized instrumentation. Here, we present a novel automated workflow for drug safety and efficacy studies using PD 3D tumor models and show utility for translational research. Individual tumoroids comprised of patient-derived colorectal cancer (CRC) cells or patient-derived breast cancer (BC) cells were isolated using the spheroONE into individual wells of the Pu·MA System flowchip. Isolated tumoroids were treated with varying concentrations of doxorubicin, oxaliplatin, Cetuximab and 5-Flourouricil using the integrated microfluidic architecture of the Pu·MA and iDOT systems. Treated tumoroids were incubated with compounds for 48 hours and stained with viability dyes. Tumoroids were imaged using widefield fluorescence microscopy and analyzed for viability using a custom AI-assisted analysis program. Patient-derived BCs, immortalized estrogen receptor positive breast cancer cells (MCF7) and triple negative breast cancer cells (MDA-MB-231) will be used to further validate the workflow we outlined. Viability of drug treated tumoroids showed a dose dependent increase in cell death, consistent with known cellular responses seen in humans. Furthermore, phenotypic differences and responses were observed between primary CRCs and spheroids generated from immortalized colorectal cells (HTC116). The mechanisms for these differences are being investigated. These data suggest that classical drug discovery using immortalized cell lines may not accurately reflect cellular behaviors seen in CRCs and potentially other patient-derived cancer cells. This novel automated workflow allows for the potential to use tumoroids from patients to be rapidly screened for drug effects, thus providing a promising platform for future clinical validation and personalized medicine.


 Anthony Stephen Eritano, Dylan Bryda, Andreea Merloiu, Alissa Schutte, Gerri Ortiz, Barbara McIntosh, Seyoum Ayehunie, Evan Cromwell. Drug response profiling of patient-derived tumoroids and immortalized cancer cell lines using a novel automation workflow and AI-assisted image analysis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB171.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20c9fe4cd4c474d45f3ec390b54b9d741b0fab09" target='_blank'>
              Abstract LB171: Drug response profiling of patient-derived tumoroids and immortalized cancer cell lines using a novel automation workflow and AI-assisted image analysis
              </a>
            </td>
          <td>
            Anthony S. Eritano, Dylan Bryda, Andreea Merloiu, Alissa Schutte, Gerri Ortiz, Barbara McIntosh, S. Ayehunie, Evan Cromwell
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="For over a century, scientists reported the disruption of normal nuclear shape and size in cancer. These changes have long been used as tools for diagnosis and staging of malignancies. However, to date, the mechanisms underlying these aberrant nuclear phenotypes and their biological significance remain poorly understood. Using a model of pancreatic ductal adenocarcinoma (PDAC), the major histological subtypes of pancreatic cancer, we found oncogenic mutant KRAS reduces nuclear size. Transcriptomic and protein expression analysis of mutant KRAS-expressing PDAC cells revealed differential levels of several nuclear envelope-associated genes. Further analysis demonstrated the nuclear lamina protein, Emerin (EMD), acted downstream of KRAS to mediate nuclear size reduction in PDAC. Analysis of human PDAC samples showed that increased EMD expression associates with reduced nuclear size. Finally, in vivo genetic depletion of EMD in a mutant KRAS-driven PDAC model resulted in an increased nuclear size and a reduced incidence of poorly differentiated PDAC. Thus, our data provides evidence of a novel mechanism underlying nuclear size regulation and its impact in PDAC carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fad3a87beb812ed2828574a6cdacbebbce33710" target='_blank'>
              Emerin is an effector of oncogenic KRAS-driven nuclear dynamics in pancreatic cancer.
              </a>
            </td>
          <td>
            Luis F Flores, David L Marks, Renzo E Vera, Ashley N Sigafoos, Ezequiel J Tolosa, Luciana L Almada, David R Pease, M. D. Toruner, Brian Chang, Brooke R Tader, Kayla C LaRue-Nolan, Ryan M. Carr, Rondell P Graham, Catherine E Hagen, Matthew R Brown, Aleksey V Matveyenko, Katherine L Wilson, David W Dawson, Christopher L. Pin, Kyle J Roux, Martin E Fernandez-Zapico
          </td>
          <td>2025-06-10</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Hypoxia is a common feature of solid tumors and plays a critical role in cancer progression. A thorough understanding of tumor hypoxia is essential for gaining deeper insights into various aspects of cancer biology. This review examines the key factors contributing to tumor hypoxia, such as inadequate blood supply and oxygen delivery resulting from rapid tumor growth. We present a detailed classification of hypoxic regions and provide an overview of current methods used to identify these areas-from molecular techniques to imaging approaches-offering a comprehensive and multifaceted perspective. Additionally, we explore the mechanisms by which hypoxia drives tumor progression. Under low-oxygen conditions, tumor cells can alter their biological behavior, influencing processes such as cell proliferation, immune evasion, and the maintenance of tumor stem cells. By addressing these dimensions, we aim to enhance understanding of how hypoxia contributes to cancer development. Through this in-depth exploration, we hope this review will offer valuable insights to guide future research and clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b6bd6f5c8102c6066579377e2e78fca233b952" target='_blank'>
              Tumor hypoxia: Classification, detection, and its critical role in cancer progression.
              </a>
            </td>
          <td>
            Yan Zhao, Bing Zhong, Jing-Yi Cao, Qian Wang, Jie Liang, Ke Lu, Shenghua Lu
          </td>
          <td>2025-04-30</td>
          <td>Biomolecules & biomedicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/820da81d9403dbf282a8bb0fbbd5219d7649afe7" target='_blank'>
              Spatiotemporal Analysis of Remyelination Reveals a Concerted Interferon-Responsive Glial State That Coordinates Immune Infiltration
              </a>
            </td>
          <td>
            Michael John Dolan, Raphael Rakosi-Schmidt, Eric Garcia, Yuesang Lin, Naeem M. Nadaf, Jessica Dixon, Shiran Guo, Nader Morshed, Constanze Depp, Jordan L Doman, Judy Xia, Henna Jäntti, Beth Stevens, Evan Z. Macosko
          </td>
          <td>2025-04-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8aace533fb9be40cad1883d118a83d4e2257b543" target='_blank'>
              Single-cell spatial map of cis-regulatory elements for disease-related genes in the macaque cortex
              </a>
            </td>
          <td>
            Juan Meng, Ying Lei, Cheng Chen, Zhiyong Zhu, Yongkang Sun, Yiming Huang, Kaijie Hu, Yiqin Bai, Tianyi Fei, Zhen Liu, Chao Li, Zhi-jian Shen, Longqi Liu, Chengyu Li, Tao Song, Cirong Liu, Muming Poo, Shingping Liu, Yidi Sun
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f88d93c45659d05d4c112ca60db918cd868e7fbf" target='_blank'>
              Signalomics for molecular tumor boards and precision oncology of breast and gynecological cancers.
              </a>
            </td>
          <td>
            Tatiana V Denisenko, Anna E. Ivanova, A. Koval, Denis N. Silachev, Lee Jia, G. Sukhikh, Vladimir L. Katanaev
          </td>
          <td>2025-06-09</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Glioblastoma (GBM), a highly aggressive and malignant form of primary adult brain cancer, poses significant therapeutic challenges. Despite our improved understanding of the cellular and molecular mechanisms underlying tumorigenesis and the evolution of GBM, targeted molecular therapies have failed to improve patient survival outcomes. The failure of standard treatments and targeted therapies is mainly attributed to the acquisition of phenotypic plasticity of tumor cells and GBM stem-like cells. Epigenetic modifications and their mediators have emerged as crucial regulators of phenotypic plasticity, influencing tumor heterogeneity, therapy resistance and disease progression. Here, we summarize and provide insights into epigenetic regulation of GBM plasticity and specifically, focus on the roles played by DNA- and histone modifiers and non-coding RNAs in driving phenotypic plasticity and resistance. We also delve into their dynamics in response to standard therapies and the challenges for targeting them to overcome phenotypic plasticity and resistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6415f436aef41d337556c7911fe6ef8d5aa73a" target='_blank'>
              Epigenetic mechanisms of plasticity and resistance in glioblastoma: therapeutic targets and implications
              </a>
            </td>
          <td>
            Farzaneh Amirmahani, Saurav Kumar, S. Muthukrishnan
          </td>
          <td>2025-05-16</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6086eff81e816ca3205d5bed8a0f1a0d7841fa7" target='_blank'>
              Oncogene aberrations drive medulloblastoma progression, not initiation.
              </a>
            </td>
          <td>
            K. Okonechnikov, P. Joshi, Verena Körber, Anne Rademacher, Michele Bortolomeazzi, Jan-Philipp Mallm, Jan Vaillant, P. B. da Silva, Britta Statz, Mari Sepp, Ioannis Sarropoulos, Tetsuya Yamada, A. Wittmann, Kathrin Schramm, Mirjam Blattner-Johnson, Petra Fiesel, Barbara Jones, N. Jäger, T. Milde, K. Pajtler, Cornelis M. van Tilburg, Olaf Witt, Konrad Bochennek, Katharina Johanna Weber, Lisa Nonnenmacher, Christian Reimann, David R. Ghasemi, U. Schüller, M. Mynarek, S. Rutkowski, David T. W. Jones, A. Korshunov, K. Rippe, Frank Westermann, S. Thongjuea, Thomas Höfer, Henrick Kaessmann, Lena M. Kutscher, Stefan M. Pfister
          </td>
          <td>2025-05-07</td>
          <td>Nature</td>
          <td>1</td>
          <td>111</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) has one of the lowest 5-year survival rates of all cancers, and limited treatment options exist. Immunotherapy is effective in some cancer types, but the immunosuppressive tumor microenvironment (TME) of PDAC is a barrier to effective immunotherapy. CXCR2+ myeloid-derived suppressor cells (MDSCs) are abundant in PDAC tumors in humans and in mouse models. MDSCs suppress effector cell function, making them attractive targets for restoring anti-tumor immunity. In this study, we show that the most abundant soluble factors released from a genetically diverse set of human and mouse PDAC cells are CXCR2 ligands, including CXCL8, CXCL5, and CXCL1. Expression of CXCR2 ligands is at least partially dependent on mutant KRAS and NFκB signaling, which are two of the most commonly dysregulated pathways in PDAC. We show that MDSCs are the most prevalent immune cells in PDAC tumors. MDSCs expressed high levels of CXCR2, and we found that myeloid cells readily migrate toward conditioned media (CM) prepared from PDAC cultures. We designed CXCR2 ligand-Fc fusion proteins to modulate the CXCR2 chemotactic signaling axis. Unexpectedly, these fusion proteins were superior to native chemokines in binding and activation of CXCR2 on myeloid cells. These “superkines” were potent inhibitors of PDAC CM-induced myeloid cell migration and were superior to CXCR2 small-molecule inhibitors and neutralizing antibodies. Our findings suggest that CXCR2 superkines may disrupt myeloid cell recruitment to PDAC tumors, ultimately improving immunotherapy outcomes in patients with PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/999e4a695d0377bd24dd8e7bc3af0220d4292a10" target='_blank'>
              Modulating the CXCR2 Signaling Axis Using Engineered Chemokine Fusion Proteins to Disrupt Myeloid Cell Infiltration in Pancreatic Cancer
              </a>
            </td>
          <td>
            Gérald Guillaumet, S. E. Kazzouli, Ben N. Christopher, Lena Golick, Ashton Basar, L. Reyes, Reeder M Robinson, Aaron O. Angerstein, Carsten Krieg, G. A. Hobbs, Denis C. Guttridge, John P O'Bryan, Nathan G. Dolloff
          </td>
          <td>2025-04-30</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background The incidence and mortality of endometrial cancer (EC) is on the rise. Eighty-five percent of ECs depend on estrogen receptor alpha (ERα) for proliferation, but little is known about its transcriptional regulation in these tumors. Results We generate epigenomics, transcriptomics, and Hi-C datastreams in healthy and tumor endometrial tissues, identifying robust ERα reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with endometrial cancer risk single-nucleotide polymorphisms and whole-genome sequencing data from primary tumors and metastatic samples reveals a striking enrichment of risk variants and non-coding somatic mutations at tumor-enriched ERα sites. Through machine learning-based predictions and interaction proteomics analyses, we identify an enhancer mutation which alters 3D genome conformation, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. Conclusions In summary, we identify a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ERα. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03596-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79c6ec91a0f963e937bbd37603ce55880af9ede" target='_blank'>
              Endometrial tumorigenesis involves epigenetic plasticity demarcating non-coding somatic mutations and 3D-genome alterations
              </a>
            </td>
          <td>
            Sebastian Gregoricchio, Aleksandar Kojic, M. Hoogstraat, K. Schuurman, Suzan Stelloo, T. Severson, Tracy A. O’Mara, M. Droog, Abhishek A. Singh, Dylan M Glubb, L. Wessels, Michiel Vermeulen, Flora E. van Leeuwen, W. Zwart
          </td>
          <td>2025-05-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4697eaf2cc8a0419c494eb4b94964848aca7f5d" target='_blank'>
              Nanopore Sequencing Unveils Somatic Structural Variations as Biomarkers in Laryngeal squamous cell carcinoma Genomes
              </a>
            </td>
          <td>
            Xuyan Liu, Lin Xia, Yixin Qiao, Yang Li, Yan Huang, Bingyan Yue, Xi Liang, Xin Yang, Honghui Zhang, Jiaxun Zhang, Xiao Chen, Dan Xie, Jifeng Liu
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 STING agonists have emerged as a promising strategy in tumor immunotherapy. However, their overall efficacy is constrained by several limiting factors. Studies indicate that chronic activation of the STING pathway can facilitate tumor immune evasion through the secretion of pro-inflammatory factors. Pancreatic ductal adenocarcinoma (PDAC), a highly aggressive gastrointestinal malignancy, is notably resistant to immunotherapy. Thus, understanding the role of STING signaling in shaping the immune microenvironment of PDAC is crucial for evaluating its potential as a therapeutic target.



 This study utilized multiplex immunofluorescence staining (mIHC) and spatial transcriptomics to delineate the immune microenvironment in PDAC patient samples. Differential gene expression analysis was conducted using bulk RNA sequencing and single-cell RNA sequencing. Both single-cell sequencing and bulk RNA sequencing were employed in genetically engineered mouse mode (LSL-KrasG12D/+,LSL-Trp53R172H/+,McuLSL/+,Pdx-1-Cre mice) to investigate intercellular communication networks. Specific immune cell subsets were further validated through flow cytometry.



 Our findings revealed that STING signaling was significantly activated in tumor samples from PDAC patients, with activation levels strongly associated with disease stage, tumor differentiation, and prognosis. Single-cell transcriptomic analysis identified a distinct population of STING-activated ductal epithelial cells, characterized by high enrichment of mitochondrial DAMP pathways and close interactions with myeloid-derived suppressor cells (MDSCs). Multi-omics analyses identified the mitochondrial calcium uniporter (MCU) as a key regulator of mitochondrial STING activation. In vitro and in vivo experiments demonstrated that MCU-overexpressing tumor cells robustly activated the STING pathway, exhibited spatial colocalization with CCR2+MDSCs, and induced the exhaustion of CD8+T cells and antigen-presenting cells in the immune microenvironment. Mechanistically, MCU activation triggered mitochondrial permeability transition pore (mPTP) opening, leading to sustained DAMP release. Virtual drug screening identified SBI-993 as a potent MCU inhibitor. Treatment with SBI-993, in combination with AG chemotherapy and anti-PD-1 therapy, significantly reduced tumor burden in vivo.



 This study underscores the pivotal role of mitochondrial dysfunction in chronic STING activation and immune evasion in PDAC. It further highlights the detrimental accumulation of MDSCs as a consequence of sustained STING activation. Targeting MCU with specific inhibitors, in combination with immunotherapy, offers a promising therapeutic strategy for combating pancreatic cancer.



 Zekun Li, Yunzhan Li, Yongjie Xie, Mengzhe Zhang, Bo Ni, Chenyang Meng, Guangcong Shen, Boyang Fu, Yueying Shan, Jun Yu, Jihui Hao, Xiuchao Wang. Mitochondrial damage-associated molecular pattern (damp)-mediated chronic stingactivation shapes the immune microenvironment in pancreatic cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/333c35e0f9ec481140ef5d8a5a7dd860c4127644" target='_blank'>
              Abstract LB046: Mitochondrial damage-associated molecular pattern (damp)-mediated chronic stingactivation shapes the immune microenvironment in pancreatic cancer
              </a>
            </td>
          <td>
            Zekun Li, Yunzhan Li, Yongjie Xie, Mengzhe Zhang, Bo Ni, Chenyang Meng, Guangcong Shen, Boyang Fu, Yueying Shan, Jun Yu, Jihui Hao, Xiuchao Wang
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Canine oral squamous cell carcinoma (COSCC) is the second most common oral tumor in dogs and the most relevant for comparative human trials as a spontaneous large animal model of disease. Historical genomic work has focused primarily on bulk sequencing. The present study describes the complete transcriptomic landscape of COSCC with spatial distinction between the surface tumor, deep invasive tumor, peritumoral dysplastic epithelium, and tumor microenvironment compared to matched normal oral samples. Each region demonstrated distinct molecular signatures. Genes related to epithelial growth factor (EGFR) and epithelial-mesenchymal transformation (EMT) were upregulated in both peritumoral dysplasia and surface cancer. Additionally, the KRAS gene set, KRT17, and SSP1 were enriched in cancer. We identified five genes that represent dysplastic lesion with high potential for malignant transformation (FZD4, GAS1, HACD2, NOG, and SLC39A6). Also, three genes, SFRP4, FZD1, and IL34 represented a specific signature of the invasive portion of the COSCC that should be explored for prognostic value as a biomarker of malignancy. Lastly, we verified the immunomodulatory tumor microenvironment detecting an increase in macrophages and an abundance of IL-10 secretion. The other predominant leukocytes were T-cells, with CD4+ T-cells being the most prevalent. CD4+ T cells expressed transcripts for both stimulatory (Inducible T-cell Co-Stimulator (ICOS) and inhibitory molecules (CTLA4). The observed high CTLA4 suggests that this inhibitory signal may be preventing a robust antitumor immune response. Taken together, this study identified multiple targets to be explored for biomarkers of malignancy, prediction of tumor behavior, and potential targets for development of novel therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8c8aa8e67efb4b761fc68addeac746847b989f" target='_blank'>
              Spatial Transcriptomic Landscape of Canine Oral Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Stephanie L Goldschmidt, Clifford G Tepper, Jack Goon, M. Soltero-Rivera, Robert B Rebhun, Andrew C. Birkeland, Xiao-Jing Wang, Ryan R Davis, Stephenie Y Liu, Iris Rivas, Brian Murphy, Natalis Vapniarsky
          </td>
          <td>2025-06-17</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Macrophages play a critical role in shaping the immune landscape of various diseases, with secreted phosphoprotein 1 (SPP1)-expressing macrophages emerging as a distinct subset implicated in both cancerous and non-cancerous conditions. Leveraging recent advances in single-cell RNA sequencing, numerous studies have identified SPP1+ macrophages across diverse pathological contexts, shedding light on their functional heterogeneity. In cancer, SPP1+ tumour-associated macrophages contribute to tumour growth, angiogenesis, and immune evasion, often interacting with T cells and stromal components to sustain an immunosuppressive microenvironment. Conversely, in non-cancerous diseases, these macrophages exhibit both profibrotic and disease-promoting properties, depending on the tissue context. This review provides a comprehensive synthesis of the latest findings on SPP1+ macrophages, highlighting their roles in tumour progression, immune suppression, tissue remodelling, and fibrosis. By comparing their shared traits and tissue-specific differences, we explore how SPP1+ macrophages adapt to distinct microenvironments and influence disease progression. Understanding their conserved and context-dependent functions may open new avenues for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2f870e69c0e0a9df1074bd00e100f8379d0713" target='_blank'>
              The Landscape of SPP1+ Macrophages Across Tissues and Diseases: A Comprehensive Review.
              </a>
            </td>
          <td>
            Alessandro Palma
          </td>
          <td>2025-05-21</td>
          <td>Immunology</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="
 Spatial transcriptomics, an increasingly prominent technique, has been extensively utilized to examine tumors within the digestive tract (such as liver and colorectal cancers) and the nervous system. However, its application in prostate cancer research remains comparatively limited. This article provides a detailed overview of the principles and features of spatial transcriptomics, particularly highlighting its applications in studying the tumor microenvironment, heterogeneity, and clinical implications in prostate cancer. Through a systematic review and analysis of current literature, we identify the main focus areas and limitations of existing research on spatial transcriptomics and suggest potential future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0488613515f6c6175fd5e7d9f3f6457ebab77dff" target='_blank'>
              Applications and prospects of spatial transcriptomics in prostate cancer research: A narrative review
              </a>
            </td>
          <td>
            Yiling Jin, Zhiming Bai, Gang Wang, Yu Zhang, Jing Chen
          </td>
          <td>2025-05-23</td>
          <td>Current Urology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Tumor cells develop various strategies to evade immune surveillance, one of which is the modulation of the metabolic state of the tumor microenvironment (TME). In response to metabolic stress in the TME, several tumor-infiltrating immune subsets upregulate CD36 to take up lipids. This leads to impaired anti-tumor immunity, as intratumoral regulatory T cells (Tregs) exhibit increased survival and suppressive activity, while CD8+ T cells become more susceptible to ferroptosis and exhaustion. Here, we develop a humanized anti-CD36 IgG4 antibody, PLT012, against the lipid-binding domain of CD36 with excellent safety and favorable pharmacokinetic features in mice and cynomolgus monkey. PLT012 alone or in combination with PD-L1 blockade or standard-of-care immunotherapy results in robust anti-tumor immunity in both immunotherapy-sensitive and -resistant hepatocellular carcinomas (HCCs). Notably, PLT012 also reprograms immune landscape of human HCC ex vivo. Our findings provide proof-of-concept evidence that PLT012 effectively reprograms anti-tumor immunity in HCC, positioning it as a first-in-class immunotherapy targeting CD36.


 Ping-Chih Ho. Reprogramming the tumor microenvironment with a single punch- our journey from bench to bedside [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY21-01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53871640c550afc871cdc39051e8769bb453d9a8" target='_blank'>
              Abstract SY21-01: Reprogramming the tumor microenvironment with a single punch- our journey from bench to bedside
              </a>
            </td>
          <td>
            Ping-Chih Ho
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive brain tumor with limited treatment options and prognosis due to its highly invasive nature and therapy resistance. Investigating the nanomechanical and pathophysiological properties of GBM cells will shed light on tumor behavior. In our study, we investigated the mechanical properties, migration dynamics, and cytoskeletal organization of T98G and U87 MG glioblastoma cell lines using in vitro techniques: atomic force microscopy (AFM) and electric cell-substrate impedance sensing (ECIS). While U87 MG cells are Temozolomide (TMZ)-sensitive and exhibit increased susceptibility to cell death and growth inhibition, T98 cells exhibit improved survival and repair in response to TMZ therapy. This study found that T98G cells are rougher, stiffer, and more viscous, while U87 MG cells are smoother, more elastic, and less viscous, leading to distinct cellular migration patterns. Such differences indicate GBM cell heterogeneity and have consequences for tumor development and resistance to treatment. The key differences of the nanomechanical, viscoelastic, and migratory properties between T98G and U87 MG cells, demonstrated in this work will help us gauge the diverse effects of different dosages of radiation along with immunotherapeutic agents, identifying the ideal radioimmunotherapy option.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ba2cc0d72ec2de612a1d7d825b152f96e3640f" target='_blank'>
              Exploring the heterogeneity in glioblastoma cellular mechanics using in-vitro assays and atomic force microscopy
              </a>
            </td>
          <td>
            Nabila Masud, Md. Hasibul Hasan Hasib, Bayode Ibironke, Charlotte Block, Jayce Hughes, Andrew E. Ekpenyong, Anwesha Sarkar
          </td>
          <td>2025-06-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa5c65d0dab89614a43e0eef89e8b1a1f0c241ce" target='_blank'>
              Single-cell spatial transcriptomics of tertiary lymphoid organ-like structures in human atherosclerotic plaques.
              </a>
            </td>
          <td>
            Zhichao Lai, Deqiang Kong, Qingsong Li, Yue Wang, Kang Li, Xiaohan Duan, Jiang Shao, Yiyun Xie, Junye Chen, Tianjing Zhang, Yuyao Feng, Haohao Deng, Jiaxian Wang, Chao-nan Wang, Keqiang Shu, Hongmei Zhao, Hanze Du, Congwei Jia, Huanyu Dai, Lizhi Xie, Jianlin Liu, Xujiao Luo, Lin Wang, Leyin Xu, Zhan Zhu, Xiangling Lei, Yuru Wang, Yixuan Yang, Yanan Liu, Yuyu Liang, Yang Yang, Jun Xie, Bao Liu, Ziqing Deng, Xin Liu
          </td>
          <td>2025-04-28</td>
          <td>Nature cardiovascular research</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Spatial biology is revolutionising our understanding of cellular organisation and disease by preserving the interactions between cells within tissues. Traditional methods often disrupted these delicate structures, making it challenging to study cells in their natural environments. This article explores key technologies, such as immunohistochemistry (IHC) and in situ hybridisation, which enable precise molecular analysis while maintaining spatial context. IHC remains essential for identifying protein markers in pathology, while multiplex imaging systems significantly enhance biomarker detection and high-throughput spatial profiling. Additionally, new computational tools like MuSpAn are advancing our understanding of spatial cell organisation. Despite challenges posed by data complexity, spatial biology opens exciting new possibilities for precision medicine, facilitating targeted therapies and advancing personalised treatment strategies. As the field rapidly evolves, it continues to drive groundbreaking breakthroughs in disease research and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab2a3155c2afaaf1db4fa8380e5514a4df2b6086" target='_blank'>
              Holding space for spatial biology
              </a>
            </td>
          <td>
            EJ. Mulholland
          </td>
          <td>2025-05-07</td>
          <td>The Biochemist</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Different glioblastoma (GBM) subtypes have been identified based on the tumor microenvironment (TME). The discovery of new therapies for these hard-to-treat tumors requires a thorough characterization of preclinical models, including their TME, to apply preclinical results to the most similar GBM subtype. Using single-nucleus RNA sequencing (snRNA-seq), we characterized the tumor and TME in an immunocompetent mouse model with intracranially implanted GBM stem cells at different stages and treatments. Visium spatial transcriptomics confirmed the location of annotated cells. This model exhibits GBM targets related to integration into neural circuits - Grik2, Nlgn3, Gap43 or Kcnn4-, immunoevasion - Nt5e, Cd274 or Irf8- and immunosuppression - Csf1r, Arg1, Mrc1 and Tgfb1. The landscape of cytokines, checkpoint ligands and receptors uncovered Mrc1, PD-L1, TIM-3 or B7-H3, among the immunotherapy targets that can be addressed in this model. The comparison with human GBMs unveiled crucial similarities with TMEMed GBM, the most frequent subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a19bb62c27014ffabb13d4cd461bff92fe32fc4" target='_blank'>
              Single-nucleus RNA sequencing reveals a preclinical model for the most common subtype of glioblastoma
              </a>
            </td>
          <td>
            L. García-Vicente, María Martínez-Fernández, M. Borja, Vanessa Tran, Andrea Álvarez-Vázquez, Raquel Flores-Hernández, Yuxin Ding, Raúl González-Sánchez, Alejandro Granados, Erin McGeever, Yang-Joon Kim, Angela M. Detweiler, Honey Mekonen, S. Paul, A. Pisco, Norma F Neff, Arantxa Tabernero
          </td>
          <td>2025-04-28</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Lineage plasticity is critical for tumor progression and therapy resistance, but the molecular mechanisms underlying cell identity shifts in cancer remain poorly understood. In lung adenocarcinoma (LUAD), the loss of pulmonary lineage fidelity and acquisition of alternate identity programs converge on hybrid identity (hybrid ID) states, which are postulated to be key intermediates in LUAD evolution and are characterized by the coactivation of developmentally incompatible identity programs within individual cells. Here, we uncover a previously unrecognized role for the gastrointestinal transcriptional regulator HNF4α in driving tumor growth and hybrid ID states in LUAD. In LUAD cells expressing the lung lineage specifier NKX2-1, HNF4α induces a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also disrupts NKX2-1 genomic localization and dampens pulmonary identity within hybrid ID LUAD. We show that this hybrid ID state is maintained by sustained RAS/MEK signaling. Inhibition of the RAS/MEK signaling cascade augments NKX2-1 chromatin binding at pulmonary-specific genes and induces drug resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion sensitizes LUAD cells to KRASG12D inhibition. Collectively, our data show that coexpression of opposing lineage specifiers is a novel mechanism of identity dysregulation in LUAD that influences both tumor progression and response to targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e47763b8688fbd0471d5f59c0d98d63dd2cf51e" target='_blank'>
              Opposing lineage specifiers induce a protumor hybrid identity state in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Kayla O'Toole, Rushmeen Tariq, Anna Waters, Katherine Gillis, Eric L. Snyder
          </td>
          <td>2025-06-12</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The breast peritumor microenvironment (pTME) is increasingly recognized as a mediator of breast cancer progression and treatment resistance. However, if and how growth-induced tumor compressive forces (i.e., solid stresses) influence the breast pTME remains unclear. Here we show using instant fluorescence lifetime imaging microscopy (FLIM)—a frequency-domain FLIM system capable of simultaneous image acquisition and instantaneous data processing—that breast tumor-mimicking in vitro compression promotes metabolic changes in stromal cells found in the breast pTME. Namely, compression shifts NIH3T3 fibroblasts and differentiated 3T3-L1 (d3T3-L1) adipocytes toward a more glycolytic state, while it promotes increased oxidative phosphorylation in 3T3-L1 undifferentiated adipocytes. The gold-standard Seahorse extracellular flux assay fails to capture these changes, underscoring the superior sensitivity of instant FLIM in detecting metabolic shifts. We validate these phenotypic findings at the transcriptomic level via RNA sequencing, confirming that compressed fibroblasts downregulate oxidative phosphorylation and upregulate glycolysis compared to uncompressed controls. We further demonstrate that compression induces mitochondrial dysregulation in undifferentiated adipocytes, driven in part by upregulated mitophagy and disrupted fusion dynamics. Finally, we confirm that these stromal cell types recapitulate these distinct metabolic states in human breast cancer patient samples, consistent with our in vitro findings. By elucidating mechano-metabolic interactions occurring at the tumor-host interface, these results will inform the development of innovative mechano-metabolic reprogramming treatment strategies to improve breast cancer patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d792df55ee5bac5edf40597207282a8a899c162" target='_blank'>
              Instant fluorescence lifetime imaging microscopy reveals mechano-metabolic reprogramming of stromal cells in breast cancer peritumoral microenvironments
              </a>
            </td>
          <td>
            Julian Najera, Hao Chen, Bianca Batista, Frank Ketchum, Aktar Ali, Pinar Zorlutuna, Scott Howard, Meenal Datta
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8625519b03e820eb6ff86341cc2c11cbf94fb11d" target='_blank'>
              Remodelling hypoxic TNBC microenvironment restores antitumor efficacy of Vγ9Vδ2 T cell therapy.
              </a>
            </td>
          <td>
            Yanyun Jing, Yangzhe Wu, Qinglin Hu, Wenfeng Wu, Dang Li, Wenhao Hu, Heming Li, Minggang Fu, Xin Huang, Yi Hu
          </td>
          <td>2025-05-28</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background The PCDH17 family has emerged as a prominent research focus in the field of oncology; however, the precise mechanism underlying the regulatory role of PCDH17 in shaping the inflammatory tumor microenvironment in pancreatic cancer remains elusive. Method A thorough examination was carried out to explore the presence of PCDH17 in cancerous tumor tissues and its association with genes related to inflammation. Additionally, we comprehensively examined the association between PCDH17 and immune cell infiltration as well as immunotherapy in pancreatic cancer. Moreover, the single-cell data of pancreatic cancer was utilized for analyzing PCDH17 expression, cell differentiation, and intercellular communication. Result PCDH17 exhibited differential expression across various tumor types. The low-expression group of PCDH17 showed reduced levels of inflammatory genes. Immunoinfiltration analysis indicated a significant association between T-cell infiltration and the expression of PCDH17. Analysis of single-cell sequencing data revealed that PCDH17 was primarily expressed in endothelial cells, with a decrease in expression observed during cellular differentiation trajectory. Notably, inflammation-related genes also displayed significant changes in their expression patterns along the endothelial cell differentiation trajectory. Cellular communication investigations unveiled multiple signaling pathways through which endothelial cells interacted with T cells. The presence of PCDH17 in endothelial cells was verified through various immunofluorescence techniques, and a simultaneous decline in its levels was observed alongside the decrease in inflammatory factors within the tumor microenvironment. Conclusion The predominant expression of PCDH17 was observed in endothelial cells, exhibiting a strong association with inflammatory genes and infiltration of immune cells. PCDH17 exhibits potential as a target for regulating the immune-suppressive tumor microenvironment in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06b73ddbf21e245d86c9b823def7478b1068908c" target='_blank'>
              Single-cell and transcriptomic analyses reveal the role of PCDH17 in the non-inflammatory tumor microenvironment of pancreatic cancer
              </a>
            </td>
          <td>
            Yong Sun, Hong Wan, Jie Xiong, Kun Cao, Dashuai Yang, Jun Huang
          </td>
          <td>2025-05-23</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cancer cells undergo widespread changes in epigenetic patterns that mediate cancer compromised gene expression programs during cancer progression. However, the alterations in higher-order genome organization in which these changes occur and their functional implications are less well understood. To explore how chromatin structure and epigenetic parameters of genome architecture changes during cancer progression at a fine scale and genome-wide, we generated high-resolution Micro-C contact maps in non-malignant, pre-cancerous, and metastatic MCF10 breast cancer epithelial cells. Results We profiled progression-associated reorganization of chromatin compartments, topologically associated domains (TADs), and chromatin loops, and also identified invariable chromatin features. We find large-scale compartmental shifts occur predominantly in early stages of cancer development, with more fine-scale structural changes in TADs and looping accumulating during the later transition to metastasis. We related these structural features to changes in gene expression, histone marks, and potential enhancers and found a large portion of diYerentially expressed genes physically connected to distal regulatory elements. While changes in chromatin loops were relatively rare during progression, diYerential loops were enriched for progression-associated genes, including those involved in proliferation, angiogenesis, and diYerentiation. Changes in either enhancer-promoter contacts or distal enhancer activity were accompanied by diYerential gene regulation, suggesting that changes in chromatin contacts are not necessary but can be suYicient for gene regulation. Conclusions Together, our results demonstrate a functionally relevant connection between gene regulation and genome remodeling at many key genes during cancer progression. KEY FINDINGS ‒ The cancer genome is reorganized throughout cancer progression at the level of compartments, chromatin domains, and loops ‒ Compartmental shifts occur in early stages of cancer development, with more fine-scale structural changes accumulating during metastasis ‒ Chromatin domain boundaries are weakened during cancer progression ‒ Many progression-regulated genes exhibit changes in distal enhancer histone modifications that are bridged by stable chromatin loops ‒ Changes in enhancer activity or subtle changes in chromatin contacts can rewire enhancer-promoter connections to facilitate changes in gene expression ‒ Prominent changes in chromatin loops occur at a small subset of diYerentially regulated genes during progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f24eeb9036338ab76b344cd3bcd26b76e9bfec03" target='_blank'>
              Genome reorganization and its functional impact during breast cancer progression
              </a>
            </td>
          <td>
            K. S. Reed, Andrew J. Fritz, Haley Greenyer, Kerstin Heselmeyer-Haddad, Seth Frietze, Janet Stein, Gary S. Stein, Tom Misteli
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771ef0b5061a9c5213ccca9fb3801ba5c0ea8357" target='_blank'>
              Whole Exome Sequencing and Single-Cell DNA Sequencing for Assessment of Clonal Heterogeneity and Evolution in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Sadiksha Adhikari, S. Eldfors, Esa Pitkänen, Markus Vähä-Koskela, C. Heckman
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797d0b2769352db6f37defdb43fa1da2c4040fb1" target='_blank'>
              A Thiopurine-like Mutagenic Process Defines TGCT Subtypes
              </a>
            </td>
          <td>
            Kevin Brown, Jun Zhong, Adriana Morales Miranda, Mengyan Zhang, Joycelyn Williams, Jacob Williams, Haoyu Zhang, Cheng Liang, Wenbo Li, Bin Zhu, Stephen J Chanock, K. Nathanson, Tongwu Zhang
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/039177f32ef3e3647a1c5ee590da14e120ba3b6c" target='_blank'>
              Cross-tissue multicellular coordination and its rewiring in cancer.
              </a>
            </td>
          <td>
            Qiang Shi, Yihan Chen, Yang Li, Shishang Qin, Yu Yang, Yang Gao, Linna Zhu, Dongfang Wang, Zemin Zhang
          </td>
          <td>2025-05-28</td>
          <td>Nature</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="Heterogeneity in cancer gene expression is typically linked to genetic and epigenetic alterations, yet post-transcriptional regulation likely influences these patterns as well. However, the quantitative contribution of post-transcriptional mechanisms to cancer transcriptome dynamics remains unclear. Here, we systematically measured mRNA dynamics across diverse breast cancer models, revealing that mRNA stability significantly shapes gene expression variability. To decipher the regulatory grammar underlying these dynamics, we developed GreyHound, an interpretable multimodal deep-learning framework integrating RNA sequence features and RNA-binding protein (RBP) expression. GreyHound identified an extensive network of RBPs and their regulons underlying variations in mRNA stability. Among these, we uncovered a metastasis-suppressive regulatory axis centered on the RNA-binding protein RBMS3 and its post-transcriptional control of the redox regulator TXNIP. Functional and molecular analyses revealed that RBMS3 depletion resulted in targeted transcript destabilization, which was associated with poor clinical outcomes and enhanced metastatic potential in xenograft models. Using in vivo epistasis studies, we confirmed that RBMS3 regulation of TXNIP mRNA stability drives this metastasis-suppressive program. These findings position the RBMS3-TXNIP regulatory axis as a key post-transcriptional mechanism in breast cancer and illustrate how interpretable models of RNA dynamics can uncover hidden regulatory programs in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a157de4d34fa592e866b9d4ddb110508f65adde6" target='_blank'>
              Integrative analysis of mRNA stability regulation uncovers a metastasis-suppressive program in breast cancer
              </a>
            </td>
          <td>
            Heather Karner, Tabea Mittmann, Vicky W. Chen, Ashir A. Borah, Andreas Langen, Hassan Yousefi, L. Fish, Balyn W. Zaro, A. Navickas, Hani Goodarzi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Colorectal cancer (CRC) is the second leading cause of cancer-related deaths worldwide, with an increasing incidence and mortality rate among younger patients. Although various risk factors have been identified, they remain insufficient to fully explain the rise of early-onset CRC. While genetic driver mutations are necessary for tumorigenesis, they alone are not sufficient to drive malignant transformation CRC. This is evident as these mutations are often present in histologically normal tissue. A critical unresolved question is: why do some benign polyps progress to malignant CRC while others remain non-invasive? Studying this phenomenon in human patients is challenging, as the earliest stages of benign-to-malignant transition are difficult to capture and cannot be functionally interrogated. To address this, we developed a new genetically engineered mouse model that recapitulates the stepwise-progression of colon cancer by combining in vivo CRISPR-Cas9 editing in the distal colon and an inducible “split-Cre” system. This allows us to decouple tumor initiating events from those that drive malignant progression. Benign adenomas are induced via colonoscopy guided injection of lentivirus expressing single guide RNA against the Apc gene (mutated in ∼80% of human CRC). Recombination of Kras LSL-G12D and Trp53 flox/flox alleles (modeling the most common drivers of advanced CRC in humans) is spatiotemporally controlled via an inducible split Cre-recombinase system, with C-terminal Cre (CreC) delivered in the initiating lentivirus and N-terminal Cre (CreN), fused to a destabilization domain (dd) and Estrogen Receptor T2 (ERT2), expressed from an internal ribosome entry site (IRES) following the stop codon of the endogenous Epcam gene (conferring epithelial specificity). We have validated that this model can capture the full histological spectrum of CRC benign-to-malignant progression: benign adenoma, intramucosal carcinoma, early invasion, and adenocarcinoma-in-adenoma with extensive histological and transcriptional intratumoral heterogeneity. In many of these Apc knockout polyps, we also observe multiple distinct regions with Kras G12D recombination alone and in combination with Trp53 knockout, recapitulating the polyclonal mutational heterogeneity of early CRC in humans. We have performed 10x Visium HD spatial transcriptomics analysis on these samples to elucidate the earliest transcriptional changes associated with stepwise Kras G12D and Trp53 loss-of-function mutations and benign-to-malignant transition. Through comprehensive tumor kinetics and spatial transcriptomic analyses, we seek to uncover deeper insights into the earliest tumor intrinsic and microenvironmental mechanisms underlying CRC benign-to-malignant transition. This approach holds significant potential to inform strategies for CRC early detection and prevention.


 Yihan Qin, Daniel Zhang, Nikita Persaud, Nischal Bhandari, Zakeria Aminzada, Colin McLaughlin, Song Han, Alex Liu, Rodrigo Romero, Santiago Naranjo, Claire Regan, William Rideout III, Alexander Cicala, Karen Yee, Jonathan Preall, Semir Beyaz, Sepideh Gholami, Zhen Zhao, Tyler Jacks, Peter M. Westcott. Capture the early benign-to-malignant transition of colon cancer in the mouse [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB475.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00157f86b246135cbe526f81bc6d42dbb5a1bc96" target='_blank'>
              Abstract LB475: Capture the early benign-to-malignant transition of colon cancer in the mouse
              </a>
            </td>
          <td>
            Yihan Qin, Daniel Zhang, Nikita Persaud, N. Bhandari, Zakeria Aminzada, Colin McLaughlin, Song Han, Alex Liu, Rodrigo Romero, S. Naranjo, C. Regan, William Rideout, A. Cicala, Karen Yee, Jonathan Preall, Semir Beyaz, S. Gholami, Zhen Zhao, Tyler Jacks, P. Westcott
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The tumor microenvironment is a highly dynamic and complex system where cellular interactions evolve over time, influencing tumor growth, immune response, and treatment resistance. In this study, we develop a graph-theoretic framework to model the tumor microenvironment, where nodes represent different cell types, and edges denote their interactions. The temporal evolution of the tumor microenvironment is governed by fundamental biological processes, including proliferation, apoptosis, migration, and angiogenesis, which we model using differential equations with stochastic effects. Specifically, we describe tumor cell population dynamics using a logistic growth model incorporating both apoptosis and random fluctuations. Additionally, we construct a dynamic network to represent cellular interactions, allowing for an analysis of structural changes over time. Through numerical simulations, we investigate how key parameters such as proliferation rates, apoptosis thresholds, and stochastic fluctuations influence tumor progression and network topology. Our findings demonstrate that graph theory provides a powerful mathematical tool to analyze the spatiotemporal evolution of tumors, offering insights into potential therapeutic strategies. This approach has implications for optimizing cancer treatments by targeting critical network structures within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80685bafd98ff8bdecc99e2498fa961cf364fcad" target='_blank'>
              Stochastic Graph-Based Models of Tumor Growth and Cellular Interactions
              </a>
            </td>
          <td>
            J. A. Rodrigues
          </td>
          <td>2025-05-29</td>
          <td>AppliedMath</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer with high metastatic ability, poor prognosis, and resistant to treatment. Tumor microenvironment plays a major role in the complexity of PDAC. Objective of the Study: The aim of the study was to examine the role of P21 activated kinase-1 (PAK1) in the sustenance of tumor microenvironment to enable tumor growth and progression. Methodology: The effect of PAK1 in the tumor microenvironment was analyzed using a novel co-culture method involving pancreatic cancer cells and pancreatic stellate cells. The 2 cell types were grown in both direct and indirect cell culture models to facilitate the juxtracrine signaling and establish a secretome network. The established network was studied using the transcriptome sequencing of PAK1-modulated MIA PaCa-2 cells co-cultured with stellate cells. Results: The results showed that PAK1 influenced cells have increased interferon pathway when compared to PAK1 depleted cells. The levels of chemokine CCL3 was altered in PAK1-modulated cells as evidenced by the bioinformatic, QPCR, and ELISA analysis. The pathway and interactome analysis showed that CCL3 promotes interferon activation and myofibroblast differentiation in pancreatic cancer microenvironment. These results might help in identifying the PAK1 induced metastatic network in pancreatic cancer. Further investigation will provide adequate evidence of CCL3 and PAK1 in pancreatic carcinogenesis and metastasis. Conclusions: The present study provides an understanding of tumor microenvironment and involvement of inflammatory cytokines in a juxtacrine mechanism to aggravate and accelerate pancreatic adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9fc7be731cc23a00411812ade4f590104f77799" target='_blank'>
              Transcriptome Analysis of Human Pancreatic Stellate Cells Co-cultured With PAK1-Modulated Cells Revealed the Role of Cytokine Pathway in Tumor Microenvironment
              </a>
            </td>
          <td>
            Sridevi Duggirala, Vaishnavi Balasubramanian, Abirami Seetharaman, Sowmiya Murugan, Joydeep Roy, Sameer Hassan, Devi Rajeswari V, Ganesh Venkatraman, Suresh Kumar Rayala
          </td>
          <td>2025-05-01</td>
          <td>Pancreas</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Gastric Cancer Guangdun Peng, Xinlong Yan, Yanchun Wang, Min Zhang, and co‐workers meticulously conducted an in‐depth and comprehensive exploration of the single‐cell and spatial transcriptomic landscape of gastric cancer initiation and progression. They prominently highlighted the crucial role of inflammatory cancer‐associated fibroblast in regulation of stemness, drug resistance, and self‐renewal of cancer stem cells (CSCs) during gastric cancer progression. More details can be found in article number 2413019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c3aed54ade998a681aa2c709fbf80632d3ef31c" target='_blank'>
              Dissecting the Spatial and Single‐Cell Transcriptomic Architecture of Cancer Stem Cell Niche Driving Tumor Progression in Gastric Cancer (Adv. Sci. 18/2025)
              </a>
            </td>
          <td>
            Guangyu Zhang, Xin Zhang, Wenting Pan, Xizhao Chen, Lingfei Wan, Chunjie Liu, Yuting Yong, Yue Zhao, Shuli Sang, Lihua Zhang, Sheng Yao, Yushu Guo, Mingmei Wang, Xinhui Wang, Guangdun Peng, Xinlong Yan, Yanchun Wang, Min Zhang
          </td>
          <td>2025-05-01</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0036830d7ca0d3dd2d206e6880d31ee221a0279" target='_blank'>
              Advances in single-cell DNA sequencing enable insights into human somatic mosaicism.
              </a>
            </td>
          <td>
            Diane D. Shao, Andrea J. Kriz, Daniel A. Snellings, Zinan Zhou, Yifan Zhao, Liz Enyenihi, Christopher A. Walsh
          </td>
          <td>2025-04-25</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>5</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [26],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>