<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-10-21 06:33:28 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intravital microscopy (IVM) enables live imaging of animals at single-cell level, offering essential insights into cancer progression. This technique allows for the observation of single-cell behaviors within their natural 3D tissue environments, shedding light on how genetic and microenvironmental changes influence the complex dynamics of tumors. The complexity of data generated by IVM often surpasses the capabilities of conventional analyses accessible to biomedical scientists, thereby neglecting single-cell heterogeneity and limiting the exploration of microenvironmental influences on cellular behavior without bias. To address this challenge, here we introduce BEHAV3D Tumor Profiler (BEHAV3D-TP), a user-friendly computational framework designed for the comprehensive analysis of single tumor cell behaviors and their interactions with the tumor microenvironment (TME). BEHAV3D-TP facilitates unbiased profiling of cancer cell dynamics without requiring advanced computational expertise. Here, we apply BEHAV3D-TP to study diffuse midline glioma (DMG), a highly aggressive pediatric brain tumor characterized by invasive growth. Our analysis reveals that distinct migratory behaviors of DMG cells correlate with specific TME components such as tumor-associated macrophages and vasculature. This approach, initially aimed at uncovering tumor invasive patterns and their interactions with the TME, holds promise for understanding additional cancer cell behaviors like intravasation and metastasis. BEHAV3D-TP represents a significant advancement in democratizing the analysis of heterogeneous cancer cell behaviors and their TME interactions, providing accessible computational insights into tumor dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/148dca420d0f76e0533267a0df34c8b8969cf9bd" target='_blank'>
              BEHAV3D Tumor Profiler to map heterogeneous cancer cell behavior in the tumor microenvironment
              </a>
            </td>
          <td>
            Emilio Rios-Jimenez, A. Zomer, R. Collot, Mario Barrera Román, H. Ariese, Ravian L. van Ineveld, Michiel Kleinnijenhuis, Nils Bessler, H. Johnson, A. Rios, Maria Alieva
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba675eb14cd995fdc082910df7aeef7f5f1a429b" target='_blank'>
              Immune landscape of oncohistone-mutant gliomas reveals diverse myeloid populations and tumor-promoting function
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Alva Annett, E. Karimi, D. G. Topouza, Morteza Rezanejad, Yitong Liu, Michael McNicholas, Eduardo G Gonzalez Santiago, Dhana Llivichuzhca-Loja, Arne Gehlhaar, Selin Jessa, Antonella De Cola, Bhavyaa Chandarana, Caterina Russo, D. Faury, Geoffroy Danieau, Evan Puligandla, Yuhong Wei, M. Zeinieh, Qing Wu, S. Hébert, N. Juretic, Emily M. Nakada, Brian Krug, Valérie Larouche, Alexander G Weil, Roy W. R. Dudley, Jason Karamchandani, S. Agnihotri, D. Quail, B. Ellezam, Liza Konnikova, L. Walsh, Manav Pathania, Claudia L Kleinman, Nada Jabado
          </td>
          <td>2024-09-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, an infiltrative growth pattern, and a high rate of relapse. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, i.e. a hypoxic niche, a leading-edge niche, and a perivascular niche, in which malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, by analysis of single-cell transcriptomic data of human GBM and transgenic mouse models of GBM, we unexpectedly identified pericytes, mural cells intimately associated with the endothelium, as the most active paracrine signaling hub within the tumor parenchyma. Exclusive signaling axes emanating from pericytes were received by endothelial cells, malignant cells, astrocytes, and immune cells. Depletion of pericytes through genetic engineering in several different transgenic and orthotopic mouse models of GBM demonstrated accelerated tumor progression, a disrupted blood-brain-barrier, and premature death of pericyte-poor mice. Mechanistic studies revealed that pericyte deficiency altered the cellular composition of GBM, remodeled the endothelium, and impacted on the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, endothelial cells deprived of pericyte association altered their signaling programs, which in turn attracted perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages expressed Hepatocyte Growth Factor (HGF), which reinforced activation of its receptor tyrosine kinase MET on GBM cells harboring an extreme mesenchymal subtype driven by the key phenotypic regulator Fosl1 within hypoxic regions. Indeed, orthotopic implantation of isolated, MET-expressing GBM cells corroborated their superior tumor-initiating capability and invasive phenotype. In patients, low expression of a pericyte core gene signature was reduced in recurrent GBM, compared to primary tumors. Consistently, gene signatures for transcriptional programs of Fosl1+Met+ GBM cells were indicative of poor survival in human tumors, and spatial transcriptomics corroborated their superior invasive capacity. Taken together, we infer that the pericyte represents a critical modulator of GBM development by orchestrating a tumor-suppressive microenvironment; our findings thus highlight the importance of pericyte preservation in the face of current and future GBM therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1287c43b4f299f76e0c243131298cd37830d70a7" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            Sebastian Braun, Paulina Bolivar, Clara R. L. Oudenaarden, Jonas Sjölund, M. Bocci, Katja Harbst, Mehrnaz Safaee Talkhoncheh, Bengt Phung, E. Cordero, R. Rosberg, Elinn Johansson, Göran Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The characteristics of the tumor microenvironment (TME) are closely linked to tumor progression and treatment response. The TME comprises various cell types, their spatial distribution, cell-cell interactions, and their organization into cellular niches or neighborhoods. To capture this complexity, several spatial profiling technologies have been developed. However, challenges such as low throughput, high costs, and complicated data analysis have limited their widespread use in immune research. In this study, we introduce the Cyclic-multiplex TSA (CmTSA) staining platform, a high-throughput superplex staining technology based on tyramide signal amplification (TSA) immunostaining combined with an efficient fluorophore recycling method. The CmTSA platform allows for the labeling of 30-60 antigens across multiple parallel formalin-fixed paraffin-embedded (FFPE) slides. Furthermore, the automated CmTSA workflow requires only standard histological equipment and conventional immunohistochemistry (IHC) primary antibodies (Abs), significantly reducing costs. While the superplex images produced contain extensive multidimensional information, extracting the spatial features of the TME from raw pixel data can be challenging. To address this, we present a computer vision-based analysis pipeline, which begins with deep learning-based algorithms to segment individual cells and identify cell types based on defined annotation rules. It then evaluates the spatial distribution tendencies of each cell type, the interaction intensity between paired cells, and the multicellular functional niches. This comprehensive approach enables researchers to visualize and quantify the types, states, and levels of immune activities within the TME effectively, advancing tumor immunology research and precision immune medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd339bff0e8e2231f57027618c2bef4a563bab3" target='_blank'>
              Pipeline for Assessing Tumor Immune Status Using Superplex Immunostaining and Spatial Immune Interaction Analysis
              </a>
            </td>
          <td>
            Chaoxin Xiao, Ruihan Zhou, Qin Chen, Wanting Hou, Xiaoying Li, Yulin Wang, Lu Liu, Huanhuan Wang, Xiaohong Yao, Tongtong Xu, Fujun Cao, Banglei Yin, Ouying Yan, Lili Jiang, Wei Wang, Dan Cao, Chengjian Zhao
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer progression to visceral organs such as lung and liver is regarded as a dreadful event, unequivocally associated with a poor prognosis. Yet, these vital sites are characterized by highly diverse cellular microenvironments and physiological functions, suggesting that they may influence cancer cells behavior in divergent ways. Unexpectedly, we find that while the liver microenvironment fosters metastasis-boosting properties of its metastatic seeds and favors secondary spread, the lungs impose a drastic roadblock to the same processes. Using patient data and mouse models of breast cancer that spontaneously metastasize to visceral organs, coupled with genetic barcode-mediated metastasis tracing, niche labeling technology and single cell analysis of both tumor cells and their direct microenvironment at each site, we characterize this dichotomy and identify differential microenvironmental factors that contribute to liver vs lung metastasis biology. Mechanistically, we uncover pulmonary cytokines, such as Granulin, whose activity naturally blocks metastatic properties. Conversely, we identify BMP2-producing endothelial cells within the liver metastatic niche, capable to enhance further spread. Together, we reveal an unexpected contrast in the site-specific behavior of visceral metastases in triple-negative breast cancer models, highlighting microenvironmental factors that contribute to its geographical diversity, as well as organ-specific opportunities for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9abbeb3d8e83ff0a32a06d5c39fa11a5d2c21ff" target='_blank'>
              Mapping the niche of breast cancer metastases in lung and liver
              </a>
            </td>
          <td>
            Magdalena K. Sznurkowska, Francesc Castro-Giner, Yongzhan Zhang, Alexander Ring, F. Schwab, Francesca Albrecht, Kirsten D. Mertz, Karin Strittmatter, Lauren L. Ozimski, Selina Budinjas, G. Auray, C. Giachino, Verdon Taylor, Ana Gvozdenovic, I. Malanchi, Heike Frauchiger-Heuer, Andreas Wicki, M. Vetter, Nicola Aceto
          </td>
          <td>2024-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e27ecfb76604dddc8d09da4d1bbaaf7b66725bf" target='_blank'>
              Spatial multiplexed immunofluorescence analysis reveals coordinated cellular networks associated with overall survival in metastatic osteosarcoma
              </a>
            </td>
          <td>
            Ryan A Lacinski, Sebastian A. Dziadowicz, Vincent K. Melemai, Brody Fitzpatrick, John J. Pisquiy, Tanya Heim, Ines Lohse, Karen Schoedel, Nicolas J. Llosa, Kurt R Weiss, B. Lindsey
          </td>
          <td>2024-09-27</td>
          <td>Bone Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background: HCC is a highly vascular tumor, and many effective drug regimens target the tumor blood vessels. Prior bulk HCC subtyping data used bulk transcriptomes, which contained a mixture of parenchymal and stromal contributions. Methods: We utilized computational deconvolution and cell-cell interaction analyses to cell type-specific (tumor-enriched and vessel-enriched) spatial transcriptomic data collected from 41 resected HCC tissue specimens. Results: We report that the prior Hoshida bulk transcriptional subtyping schema is driven largely by an endothelial fraction, show an alternative tumor-specific schema has potential prognostic value, and use spatially paired ligand-receptor analyses to identify known and novel (LGALS9 tumor-HAVCR2 vessel) signaling relationships that drive HCC biology in a subtype-specific and potentially targetable manner. Conclusions: Our study leverages spatial gene expression profiling technologies to dissect HCC heterogeneity and identify heterogeneous signaling relationships between cancer cells and their endothelial cells. Future validation and expansion of these findings may validate novel cancer-endothelial cell interactions and related drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e5076806fef62fc7b47a9b3751a8891a984dc18" target='_blank'>
              HCC spatial transcriptomic profiling reveals significant and potentially targetable cancer-endothelial interactions
              </a>
            </td>
          <td>
            Chenyue Lu, Amaya Pankaj, Michael J. Raabe, Cole Nawrocki, Ann Liu, Nova Xu, Bidish Kumar Patel, Matthew J. Emmett, Avril K Coley, Cristina R. Ferrone, Vikram Deshpande, I. Bhan, Y. Hoshida, David T. Ting, Martin J. Aryee, Joseph W Franses
          </td>
          <td>2024-09-27</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PDAC) is a rapidly progressing cancer that responds poorly to immunotherapies. Intratumoral tertiary lymphoid structures (TLS) have been associated with rare long-term PDAC survivors, but the role of TLS in PDAC and their spatial relationships within the context of the broader tumor microenvironment remain unknown. We generated a spatial multi-omics atlas encompassing 26 PDAC tumors from patients treated with combination immunotherapies. Using machine learning-enabled H&E image classification models and unsupervised gene expression matrix factorization methods for spatial transcriptomics, we characterized cellular states within TLS niches spanning across distinct morphologies and immunotherapies. Unsupervised learning generated a TLS-specific spatial gene expression signature that significantly associates with improved survival in PDAC patients. These analyses demonstrate TLS-associated intratumoral B cell maturation in pathological responders, confirmed with spatial proteomics and BCR profiling. Our study also identifies spatial features of pathologic immune responses, revealing TLS maturation colocalizing with IgG/IgA distribution and extracellular matrix remodeling. GRAPHICAL ABSTRACT HIGHLIGHTS Integrated multi-modal spatial profiling of human PDAC tumors from neoadjuvant immunotherapy clinical trials reveal diverse spatial niches enriched in TLS. TLS maturity is influenced by tumor location and the cellular neighborhoods in which TLS immune cells are recruited. Unsupervised machine learning of genome-wide signatures on spatial transcriptomics data characterizes the TLS-enriched TME and associates TLS transcriptomes with survival outcomes in PDAC. Interactions of spatially variable gene expression patterns showed TLS maturation is coupled with immunoglobulin distribution and ECM remodeling in pathologic responders. Intratumoral plasma cell and immunoglobin gene expression spatial dynamics demonstrate trafficking of TLS-driven humoral immunity in the PDAC TME. Significance We report a spatial multi-omics atlas of PDAC tumors from a series of immunotherapy neoadjuvant clinical trials. Intratumorally, pathologic responders exhibit mature TLS that propagate plasma cells into malignant niches. Our findings offer insights on the role of TLS-associated humoral immunity and stromal remodeling during immunotherapy treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b4043873d18d3b758a620d9a4f90e32bea7034" target='_blank'>
              Spatial multi-omics reveal intratumoral humoral immunity niches associated with tertiary lymphoid structures in pancreatic cancer immunotherapy pathologic responders
              </a>
            </td>
          <td>
            Dimitri N. Sidiropoulos, Sarah M. Shin, Meredith Wetzel, Alexander Girgis, Daniel Bergman, Ludmila Danilova, Susheel Perikala, Daniel H. Shu, Janelle M. Montagne, Atul Deshpande, James Leatherman, Lucie Dequiedt, Victoria Jacobs, Aleksandra Ogurtsova, Guanglan Mo, Xuan Yuan, Dmitrijs Lvovs, G. Stein-O’Brien, M. Yarchoan, Qingfeng Zhu, Elizabeth I. Harper, A. Weeraratna, A. Kiemen, Elizabeth M. Jaffee, Lei Zheng, W. Ho, Robert A Anders, E. Fertig, Lusciane T. Kagohara
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) resides in an immune rich microenvironment, yet, immune-based therapies have faltered in eliciting durable responses. Bridging this paradox requires a comprehensive understanding of leukemic interactions within the bone marrow microenvironment. We optimized a high-throughput tissue-microarray based pipeline for high-plex spatial immunofluorescence and mass cytometry imaging on a single slide, capturing immune, tumor, and structural components. Using unbiased clustering on the spatial K function, we unveiled the presence of tertiary lymphoid-like aggregates in bone marrow which we validated using spatial transcriptomics and an independent proteomics approach. We then found validated TLS signatures predictive of outcomes in AML using an integrated public 480 patient transcriptomic dataset. By harnessing high-plex spatial proteomics, we open the possibility of discovering of novel structures and interactions that underpin leukemic immune response. Further, our study’s methodologies and resources can be adapted for other bone marrow diseases where decalcification and autofluorescence present challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1b1bad3b36f52c6886c710f3a8b1e66619ef44" target='_blank'>
              Multimodal Spatial Proteomic Profiling in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Christopher Ly, Ivo Veletic, Christopher D. Pacheco, Enes Dasdemir, F. Jelloul, Sammy Ferri-Borgongo, Akshay V Basi, Javier A. Gomez, Jessica L Root, P. Reville, Padmanee Sharma, Sreyashi Basu, Andres E. Quesada, Carlos Bueso-Ramos, T. Manshouri, Miriam Garcia, Jared K. Burks, Hussein A. Abbas
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0222f474962325ec3943ada79fb1a6a8a5016a" target='_blank'>
              Spatial resolved transcriptomics reveals distinct cross-talk between cancer cells and tumor-associated macrophages in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Zhao-Ru Dong, Meng-Ya Zhang, Ling-Xin Qu, Jie Zou, Yong-Heng Yang, Yun-Long Ma, Chuncheng Yang, Xue-Lei Cao, Li-Yuan Wang, Xiao-Lu Zhang, Tao Li
          </td>
          <td>2024-09-11</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Objective Single-cell technologies have revolutionized our understanding of the phenotypic and transcriptional diversity of aortic leukocytes in atherosclerotic humans and mice. However, enzymatically dissociated tissues lose the spatial context of plaque cells in situ. Here we utilized imaging mass cytometry (IMC) combining with single-cell RNA sequencing (scRNA-seq) to characterize the spatial distribution dynamics, phenotypic transitions, metabolic and functional phenotypes, and the intercellular interaction networks of plaque cells during atherosclerotic progression. Additionally, the dynamic immune landscape of circulating leukocytes associated with atherosclerosis was characterized using cytometry of time of flight (CyTOF). Approach and Results A highly multiplexed IMC panel with 33 metal-conjugated antibodies was designed to generate 11 highly multiplexed histology images of aortic root tissues from ApoE-/- mice on high-fat diet at different stage of atherosclerosis. Using histoCAT, we identified 8 principal cell subtypes with distinct phenotypic and geographic dynamics. Furthermore, IMC-defined cell subsets partially corresponded to scRNA-seq-annotated aortic cell subtypes, including 4 macrophage subsets, neutrophils, smooth muscle cells (SMCs) and SMC-derived SEMs (Stem cell, endothelial cell and macrophage-like cell). Activation of inflammatory pathways, increased oxidative phosphorylation and augmented osteoclast differentiation were observed in macrophage populations, SMCs and SEMs from an early stage to advanced stage of atherosclerosis. Notably, cell neighborhood analysis by IMC uncovered multifaceted cell-cell interactions within the plaque, in particular in neutrophil-mediated interactions with smooth muscle cells and macrophages, which were confirmed by ligand-receptor interactions based on scRNA-seq data. Additionally, characterization of the peripheral immune cells by CyTOF revealed an increased ratio of myeloid cells to lymphocytes, and certain neutrophil and monocyte subpopulations also exhibited enhanced lipid metabolism and glycolysis as well as activated inflammatory signaling. Conclusion This study provides a dynamic spatiotemporal landscape of atherosclerotic lesions and peripheral leukocytes. The new information based on IMC may help understand atherosclerotic pathology and develop novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47ce553643345229db62995608b4a068983d0b49" target='_blank'>
              Molecular and spatiotemporal characterization of cells in murine atherosclerotic plaques
              </a>
            </td>
          <td>
            Pengbo Hou, Zhanhong Liu, Jiankai Fang, Ziyi Wang, Shisong Liu, Shiqing Wang, Gerry Melino, Peishan Li, Yufang Shi, Changshun Shao
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Most human cancers are heterogeneous consisting of cancer cells at different epigenetic and transcriptional states and with distinct phenotypes, functions, and drug sensitivities. This inherent cancer cell heterogeneity contributes to tumor resistance to clinical treatment, especially the molecularly targeted therapies such as tyrosine kinase inhibitors (TKIs) and androgen receptor signaling inhibitors (ARSIs). Therapeutic interventions, in turn, induce lineage plasticity (also called lineage infidelity) in cancer cells that also drives therapy resistance. In this Perspective, we focus our discussions on cancer cell lineage plasticity manifested as treatment-induced switching of epithelial cancer cells to basal/stem-like, mesenchymal, and neural lineages. We employ prostate cancer (PCa) as the prime example to highlight ARSI-induced lineage plasticity during and towards development of castration-resistant PCa (CRPC). We further discuss how the tumor microenvironment (TME) influences therapy-induced lineage plasticity. Finally, we offer an updated summary on the regulators and mechanisms driving cancer cell lineage infidelity, which should be therapeutically targeted to extend the therapeutic window and improve patients’ survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea1fde8339eba5fa332bd913bbc774ac173a97e" target='_blank'>
              Treatment-induced stemness and lineage plasticity in driving prostate cancer therapy resistance
              </a>
            </td>
          <td>
            Anmbreen Jamroze, Xiaozhuo Liu, Dean G. Tang
          </td>
          <td>2024-08-25</td>
          <td>Cancer heterogeneity and plasticity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fae692b36b9cb8dfff150568e46638865d4946" target='_blank'>
              Single-cell and spatial proteo-transcriptomic profiling reveals immune infiltration heterogeneity associated with neuroendocrine features in small cell lung cancer
              </a>
            </td>
          <td>
            Ying Jin, Yuefeng Wu, Alexandre Reuben, Liang Zhu, C. Gay, Qingzhe Wu, Xintong Zhou, Haomin Mo, Qi Zheng, Junyu Ren, Zhaoyuan Fang, Teng Peng, Nan Wang, Liang Ma, Yun Fan, Hai Song, Jianjun Zhang, Ming Chen
          </td>
          <td>2024-09-04</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b2b2736cd24fffcca3410f55115541def7e515" target='_blank'>
              METI: deep profiling of tumor ecosystems by integrating cell morphology and spatial transcriptomics
              </a>
            </td>
          <td>
            Jiahui Jiang, Yunhe Liu, Jiang-Jiang Qin, Jianfeng Chen, JingJing Wu, Melissa P Pizzi, R. Lazcano, Kohei Yamashita, Zhiyuan Xu, Guangsheng Pei, K. S. Cho, Yanshuo Chu, Ansam Sinjab, F. Peng, Xinmiao Yan, Guangchun Han, Ruiping Wang, Enyu Dai, Yibo Dai, Bogdan Czerniak, Andrew Futreal, Anirban Maitra, Alexander Lazar, H. Kadara, A. Jazaeri, Xiangdong Cheng, Jaffer J. Ajani, Jianjun Gao, Jian Hu, Linghua Wang
          </td>
          <td>2023-10-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is heterogeneous with low tumor purity, prominent microenvironment and complex architecture, which preclude identification of shared tumor intrinsic biology within and across patients. We overcame these challenges by applying complementary spatial omics approaches – providing necessary resolution and context – to primary untreated PDAC tumors from 39 patients capturing 341,949 low-bulk and 531,718 single-cell spatial transcriptomes. Of these, 59,403 low-bulk profiles and 205,665 single-cells represented tumor cells. We leveraged this data to build on prior reports of tumor cell subtypes in PDAC. We identify 5 distinct malignant subtypes, excluding ductal-like and intraepithelial neoplasia cells, that span the classical-to-basal spectrum. One tumor subtype is highly proliferative and enriched for the DNA synthesis-condensation-mitosis transcriptional network suggesting this subset disproportionately contribute to tumor growth. Strikingly, the spectrum of tumor cell subtypes was present within all patients, suggesting that subtype-based treatment stratification may need refinement. We find that each subtype has its own distinct regulators and histology. The classical subtype has extensive cell intrinsic regulation, higher cell density and lower extracellular matrix content. In contrast, the basal subtype phenotype results from a combination of cell intrinsic and tumor microenvironment (TME) interactions, wherein the basal subtype has a lower cell density, is surrounded by activated stroma and dense collagen matrix, and is poised for hypoxic adaptation. We derive the composition of cellular neighborhoods providing an explanation for the complex architecture seen in PDAC. Tumor cell neighborhoods stratified based on which side of the classical-to-basal spectrum they fell. For example, cells towards the classical-side of the spectrum had more homogenous tumor neighborhoods that generally lacked intermediate and basal tumor cells. In contrast, cells towards the basal-side of the spectrum were likely to contain classical neighbors. More broadly, neighborhoods along this continuum also contained different proportions of immune and stromal cells, cell densities, collagen deposition, and TME adaptation. These observations support a model where tumor subtypes exist within disparate niches. Lastly, our atlas unifies the catalog of previously reported stromal cells (fibroblasts and stellate cells versus myofibroblastic and inflammatory cancer associated fibroblasts (CAF)) and describes a novel interferon stimulated gene (ISG) resistance signature (ISG.RS) fibroblast. We map the location of these various stromal cells relative to tumor subtypes. For example, myofibroblast CAFs and ISG.RS fibroblasts are enriched near the basal tumor subtype whereas inflammatory CAF abundance increases with distance away from tumor cells. In sum, our atlas provides a lens for stratifying tumor complexity in a cancer cell centric manner and provides a path for testing therapeutic hypothesis in the right cell subtype and context.
 Citation Format: Brian Rabe, Anna Lyubetskaya, Andrew Kavran, Yulong Bai, Andrew Fisher, Hannah Pliner, Alba Font-Tello, Anne Lewin, Ruifeng Hu, Alexandre P Alloy, Yelena Cheng, Chao Dai, Yunfan Fan, Constance Brett, Todd Brett, Lauren Giampapa, Soeren Stahlschmidt, Fayaz Seifuddin, Steven Vasquez Grinnell, Daniel Carrera, Carlos Rios, Tom Lila, Pradeep Kar, Abhishek Shukla, Rachael Bashford Rogers, Lara Heij, Mike Mason, Ryan Golhar, Isaac Neuhaus, Enas Abu Shah, Shivan Sivakumar, Jimena Trillo Tinoco, Benjamin J Chen, Konstantinos J Mavrakis, Eugene Drokhlyansky. In situ multi-modal characterization of pancreatic ductal adenocarcinoma reveals tumor cell identity as a defining factor of the surrounding microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B081.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566e9b3495ee9764c389c226c9c17e824614540e" target='_blank'>
              Abstract B081: In situ multi-modal characterization of pancreatic ductal adenocarcinoma reveals tumor cell identity as a defining factor of the surrounding microenvironment
              </a>
            </td>
          <td>
            Brian Rabe, Anna Lyubetskaya, Andrew Kavran, Yulong Bai, Andrew Fisher, Hannah Pliner, Alba Font-Tello, Anne Lewin, Ruifeng Hu, Alexandre P. Alloy, Yelena Cheng, Chao Dai, Yunfan Fan, Constance Brett, Todd Brett, Lauren Giampapa, Soeren Stahlschmidt, Fayaz Seifuddin, Steven Vasquez Grinnell, Daniel Carrera, Carlos Rios, Tom Lila, Pradeep Kar, Abhishek Shukla, Rachael Bashford Rogers, Lara R. Heij, Mike Mason, R. Golhar, Isaac Neuhaus, E. A. Shah, Shivan Sivakumar, Jimena Trillo Tinoco, Benjamin Chen, Konstantinos J Mavrakis, Eugene Drokhlyansky
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Diffuse Large B-cell Lymphoma (DLBCL) is the most prevalent subtype of non-Hodgkin’s lymphoma for which current therapeutic strategies remain insufficient. The diffuse nature of DLBCL, lacking distinct tissue structures, represents a challenge to elucidate the cellular organization and interactions within the tumor microenvironment (TME). In this study, we applied spatial transcriptomics to identify spatially-resolved gene expression profiles in 10 DLBCL tissue samples, identifying distinct immune cell infiltration and colocalization patterns. These profiles were classified into six cellular ecosystems (Cell-Eco) that differ in cellular composition, functional patterns, and neighborhood characteristics. The spatially-resolved Cell-Eco signatures provided prognostic scores that stratified patients with different overall survival rates. We also found that C1q+ tumor-associated macrophages are the primary cells interacting with malignant B cells and influencing the spatial architecture of the TME. This study provides novel biological insights into the complexity of the TME in DLBCL and highlights the potential prognostic value of its spatial organization. Graphical abstract Key findings Spatial transcriptomics classifies DLBCL tissues based on immune cell infiltration and colocalization patterns. DLBCL tumor microenvironment consists of cellular ecosystems (Cell-Eco) that differ in cellular composition, transcriptomic profiles and neighborhood characteristics. Spatially-resolved Cell-Eco signatures stratify patients with different overall survival. C1q+ tumor-associated macrophages primarily interact with malignant B cells and contribute to the spatial organization of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ebc56c31f2327f9141f086b678169242b28d1d" target='_blank'>
              Spatial transcriptomics unveils immune cellular ecosystems associated with patient survival in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Alba Díaz Herrero, Hector F Pelaez-Prestel, Lucile Massenet-Regad, Maëva Veyssiere, Julien Calvani, Caterina Cristinelli, Jacqueline Lehmann-Che, Véronique Meignin, Catherine Thieblemont, Véronique Blanc, V. Soumelis, P. Tonnerre
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background Smooth muscle cell (SMC) plasticity and phenotypic switching play prominent roles in the pathogenesis of multiple diseases, but their role in tumorigenesis is unknown. We investigated whether and how SMC diversity and plasticity plays a role in tumor angiogenesis and the tumor microenvironment. Methods and Results We use SMC-specific lineage-tracing mouse models and single cell RNA sequencing to observe the phenotypic diversity of SMCs participating in tumor vascularization. We find that a significant proportion of SMCs adopt a phenotype traditionally associated with macrophage-like cells. These cells are transcriptionally similar to ‘resolution phase’ M2b macrophages, which have been described to have a role in inflammation resolution. Computationally predicted by the ligand-receptor algorithm CellChat, signaling from BST2 on the surface of tumor cells to PIRA2 on SMCs promote this phenotypic transition; in vitro SMC assays demonstrate upregulation of macrophage transcriptional programs, and increased proliferation, migration, and phagocytic ability when exposed to BST2. Knockdown of BST2 in the tumor significantly decreases the transition towards a macrophage-like phenotype, and cells that do transition have a comparatively higher inflammatory signal typically associated with anti-tumor effect. Conclusion As BST2 is known to be a poor prognostic marker in multiple cancers where it is associated with an M2 macrophage-skewed TME, these studies suggest that phenotypically switched SMCs may have a previously unidentified role in this immunosuppressive milieu. Further translational work is needed to understand how this phenotypic switch could influence the response to anti-cancer agents and if targeted inhibition of SMC plasticity would be therapeutically beneficial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcad9e00e00b0b1e9b4d096ed61477beb04a0767" target='_blank'>
              BST2 induces vascular smooth muscle cell plasticity and phenotype switching during cancer progression
              </a>
            </td>
          <td>
            Caitlin F. Bell, Richard A. Baylis, N. G. Lopez, W. Ma, Hua Gao, Fudi Wang, Sharika Bamezai, Changhao Fu, Yoko Kojima, Shaunak S. Adkar, Lingfeng Luo, Clint L. Miller, N. Leeper
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Solid tumours exhibit a well‐defined architecture, comprising a differentiated core and a dynamic border that interfaces with the surrounding tissue. This border, characterised by distinct cellular morphology and molecular composition, serves as a critical determinant of the tumour's invasive behaviour. Notably, the invasive border of the primary tumour represents the principal site for intravasation of metastatic cells. These cells, known as circulating tumour cells (CTCs), function as ‘seeds’ for distant dissemination and display remarkable heterogeneity. Advancements in spatial sequencing technology are progressively unveiling the spatial biological features of tumours. However, systematic investigations specifically targeting the characteristics of the tumour border remain scarce. In this comprehensive review, we illuminate key biological insights along the tumour body‐border‐haematogenous metastasis axis over the past five years. We delineate the distinctive landscape of tumour invasion boundaries and delve into the intricate heterogeneity and phenotype of CTCs, which orchestrate haematogenous metastasis. These insights have the potential to explain the basis of tumour invasion and distant metastasis, offering new perspectives for the development of more complex and precise clinical interventions and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f6a6ffd5124276ada6cdb3c914a272e4284a04" target='_blank'>
              Single‐cell and spatial omics unravel the spatiotemporal biology of tumour border invasion and haematogenous metastasis
              </a>
            </td>
          <td>
            Xifu Cheng, Yuke Cao, Xiangyi Liu, Yuanheng Li, Qing Li, Dian Gao, Qiongfang Yu
          </td>
          <td>2024-09-30</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 HCC develops in the context of chronic inflammation; however, the opposing roles the immune system plays in both the development and control of tumors are not fully understood. Mapping immune cell interactions across the distinct tissue regions could provide greater insight into the role individual immune populations have within tumors.



 A 39-parameter imaging mass cytometry panel was optimized with markers targeting immune cells, stromal cells, endothelial cells, hepatocytes, and tumor cells. We mapped the immune landscape of tumor, invasive margin, and adjacent nontumor regions across 16 resected tumors comprising 144 regions of interest. X-shift clustering and manual gating were used to characterize cell subsets, and Spectre quantified the spatial environment to identify cellular neighborhoods. Ligand-receptor communication was quantified on 2 single-cell RNA-sequencing data sets and 1 spatial transcriptomic data set.



 We show immune cell densities remain largely consistent across these 3 regions, except for subsets of monocyte-derived macrophages, which are enriched within the tumors. Mapping cellular interactions across these regions in an unbiased manner identifies immune neighborhoods comprised of tissue-resident T cells, dendritic cells, and various macrophage populations around perivascular spaces. Importantly, we identify multiple immune cells within these neighborhoods interacting with VEGFA+ perivascular macrophages. VEGFA was further identified as a ligand for communication between perivascular macrophages and CD34+ endothelial cells.



 Immune cell neighborhood interactions, but not cell densities, differ between intratumoral and adjacent nontumor regions in HCC. Unique intratumoral immune neighborhoods around the perivascular space point to an altered landscape within tumors. Enrichment of VEGFA+ perivascular macrophages within these tumors could play a key role in angiogenesis and vascular permeability.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f97ce36e43e0dfde11d3ba7e6fa851a271fe9a2" target='_blank'>
              Spatial mapping of the HCC landscape identifies unique intratumoral perivascular-immune neighborhoods
              </a>
            </td>
          <td>
            Felix Marsh-Wakefield, C. Santhakumar, A. L. Ferguson, T. Ashhurst, Joo-Shik Shin, Fiona H. X. Guan, Nicholas J. Shields, Barry J. Platt, Givanna H. Putri, Ruta Gupta, Michael Crawford, Carlo Pulitano, C. Sandroussi, Jerome M Laurence, Ken Liu, Geoffrey W McCaughan, U. Palendira
          </td>
          <td>2024-10-17</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood. Methods We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles. Results We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation. Discussion Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Phenotypic diversity in tumors makes treatment difficult and leads to poor prognosis. A better understanding of what causes intratumoral heterogeneity and differences in cells that are within the same tumor will advance therapeutic options and improve patient outcomes. This review will summarize recent studies that have examined the various cancer cell phenotypes along the epithelial–mesenchymal spectrum present in tumors and how epigenetic factors and the mechanical properties of tumor tissue mediate these phenotypes. This review will also discuss the therapeutic implications of intratumoral heterogeneity, such as drug resistance and tumor recurrence, and highlight opportunities to address current limitations in cancer treatments. Abstract The tumor microenvironment comprises various cell types and experiences dynamic alterations in physical and mechanical properties as cancer progresses. Intratumoral heterogeneity is associated with poor prognosis and poses therapeutic challenges, and recent studies have begun to identify the cellular mechanisms that contribute to phenotypic diversity within tumors. This review will describe epithelial–mesenchymal (E/M) plasticity and its contribution to phenotypic heterogeneity in tumors as well as how epigenetic factors, such as histone modifications, histone modifying enzymes, DNA methylation, and chromatin remodeling, regulate and maintain E/M phenotypes. This review will also report how mechanical properties vary across tumors and regulate epigenetic modifications and E/M plasticity. Finally, it highlights how intratumoral heterogeneity impacts therapeutic efficacy and provides potential therapeutic targets to improve cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b710fff2d564553c94ffb93440614a32c0f2e22e" target='_blank'>
              Epithelial–Mesenchymal Plasticity and Epigenetic Heterogeneity in Cancer
              </a>
            </td>
          <td>
            Jessica L. Sacco, Esther W. Gomez
          </td>
          <td>2024-09-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related deaths. It is unclear how intratumor heterogeneity (ITH) contributes to metastasis and how metastatic cells adapt to distant tissue environments. The study of these adaptations is challenged by the limited access to patient material and a lack of experimental models that appropriately recapitulate ITH. To investigate metastatic cell adaptations and the contribution of ITH to metastasis, we analyzed single-cell transcriptomes of matched primary tumors and metastases from patient-derived xenograft models of breast cancer. We found profound transcriptional differences between the primary tumor and metastatic cells. Primary tumors upregulated several metabolic genes, whereas motility pathway genes were upregulated in micrometastases, and stress response signaling was upregulated during progression. Additionally, we identified primary tumor gene signatures that were associated with increased metastatic potential and correlated with patient outcomes. Immune-regulatory control pathways were enriched in poorly metastatic primary tumors, whereas genes involved in epithelial-mesenchymal transition were upregulated in highly metastatic tumors. We found that ITH was dominated by epithelial-mesenchymal plasticity (EMP), which presented as a dynamic continuum with intermediate EMP cell states characterized by specific genes such as CRYAB and S100A2. Elevated expression of an intermediate EMP signature correlated with worse patient outcomes. Our findings identified inhibition of the intermediate EMP cell state as a potential therapeutic target to block metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4040c30de029de626b99afe51b1e8ce55165671f" target='_blank'>
              Single-cell analysis of breast cancer metastasis reveals epithelial-mesenchymal plasticity signatures associated with poor outcomes
              </a>
            </td>
          <td>
            Juliane Winkler, Weilun Tan, Catherine M. M. Diadhiou, Christopher S. McGinnis, Aamna J. Abbasi, Saad Hasnain, Sophia Durney, Elena Atamaniuc, Daphne A. Superville, L. Awni, Joyce V. Lee, Johanna H. Hinrichs, Patrick S. Wagner, Namrata Singh, Marco Y. Hein, M. Borja, Angela M Detweiler, Su-Yang Liu, Ankitha Nanjaraj, Vaishnavi Sitarama, H. Rugo, Norma Neff, Zev J. Gartner, Angela Oliveira Pisco, Andrei Goga, S. Darmanis, Z. Werb
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>117</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54370344e70d12f9e093a0f96e506dd839c6df0" target='_blank'>
              Deciphering breast cancer dynamics: insights from single-cell and spatial profiling in the multi-omics era
              </a>
            </td>
          <td>
            Xin Xiong, Xin Wang, Cui-Cui Liu, Zhi-Ming Shao, Ke-Da Yu
          </td>
          <td>2024-09-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal cancer (CRC) features molecular heterogeneity and differing immune cell infiltration patterns, affecting how patients respond to therapy. However, a comprehensive understanding of immune cell phenotypes across patient subgroups is missing. Here, we dissect the CRC tumor microenvironment by integrating 4,27 million single cells from 1670 samples and 650 patients across 48 studies (76 datasets) representing 7 billion expression values. These samples encompass the full spectrum of disease progression, from normal colon to polyps, primary tumors, and metastases, covering both early and advanced stages of CRC. We present a high-resolution view of CRC with 62 major cell types or states, revealing different cell-type composition patterns in CRC subtypes. The atlas allows for refined tumor classification and patient stratification into four immune phenotypes: immune desert, myeloid, B cell, and T cell subtypes. Using bulk samples with genomic and clinical information, we identify cellular components associated with tumor histology and genotypes. These findings could have important implications for developing improved cancer immunotherapy approaches in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b83f214ca7a0f8d484f556a59979b00abd51f6" target='_blank'>
              Dissecting heterogeneity of tumor microenvironment in colorectal cancer using high-resolution single-cell atlas
              </a>
            </td>
          <td>
            Valentin Marteau, Niloofar Nemati, Kristina Handler, Deeksha Raju, Erika Kvalem Soto, G. Fotakis, Sandro Carollo, Nina Böck, A. Kirchmair, A. Scheiber, Arno Ammann, Andreas Seeber, Elisabeth Gasser, Lena Horvath, Hassan Fazilaty, Tomas Valenta, Gregor Sturm, D. Rieder, S. Sopper, A. Pircher, P. Ziegler, Markus Tschurtschenthaler, Dominik Wolf, Isabelle Arnold, S. Salcher, Z. Trajanoski
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Bone marrow (BM) is a common site for solid tumor metastasis, often causing poor outcome. Here, we define the characteristics of BM-disseminated tumor cells (DTCs) using neuroblastoma as a model. We combined single-cell RNA-sequencing (scRNA-seq) and cell-surface protein analysis using 7 paired BM and primary tumor (PT) samples and found that DTCs contain a higher percentage of cycling cells and higher expression of neurodevelopmental genes compared to corresponding PT cells. In 6 patients, the copy number variation profile differed between PT cells and DTCs, indicating spatial heterogeneity. Within the BM, we detected dormant DTCs with potentially reduced chemosensitivity; this population contained cells expressing low levels of the immunotherapeutic antigen GD2 and increased NGFR expression. In conclusion, we characterized DTCs that are particularly challenging to target, offering new avenues for developing therapeutic strategies designed to target all subpopulations within the highly complex metastatic site, thereby preventing the development of drug-resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fc1c99c4919cd41656eef13d151b98e6e5eb661" target='_blank'>
              Metastatic tumor cells in bone marrow differ from paired neuroblastoma tumor and contain subsets with therapy-resistant characteristics
              </a>
            </td>
          <td>
            Caroline Hochheuser, A. Boltjes, Kaylee M. Keller, Simon Tol, Marieke van de Mheen, Carolina Pita Barros, Zeinab van Gestel-Fadaie, André B. P. van Kuilenburg, Sander van Hooff, C. Voermans, Jan J. Molenaar, G. Tytgat, Ilse Timmerman
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9797ff0fc8da1478461b88997f4f8cb4c2bab8" target='_blank'>
              Single-cell and spatial sequencing identifies senescent and germinal tumor cells in adamantinomatous craniopharyngiomas
              </a>
            </td>
          <td>
            Xianlong Wang, Jincheng Lin, Hongxing Liu, Chuan Zhao, Zhiwei Tu, Dapeng Xu, En Zhang, Zhongqing Zhou, Xueling Qi, Xingfu Wang, Zhixiong Lin
          </td>
          <td>2024-09-02</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19dd1cc421c82853f5d16e4b6c21745d87d8c500" target='_blank'>
              Characteristics and transcriptional regulators of spontaneous epithelial–mesenchymal transition in genetically unperturbed patient-derived non-spindled breast carcinoma
              </a>
            </td>
          <td>
            H. Lien, Hui-Chieh Yu, Wen-Hsuan Yu, Su-Fang Lin, Tom Wei-Wu Chen, I. Chen, Li-Ping Hsiao, Ling-Chun Yeh, Yu-Chia Li, Chiao Lo, Yen-Shen Lu
          </td>
          <td>2024-09-10</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains an intractable cancer characterized by significant inter- and intra-patient heterogeneity. Recent research has established connections between key genetic drivers and distinct transcriptional profiles in GBM, yet the interactions between genetic changes and the transcriptional and spatial heterogeneity remain unclear. This study used genetically engineered mouse models, coupled with spatially resolved and single-cell RNA sequencing, to explore how specific genetic alterations influence the tumor microenvironment (TME) and cellular responses spatially.



 We used nestin-Tva C57BL/6 mice and generated de novo GBM using overexpression of oncogenes in combinations with short hairpin RNAs (shRNAs) or deletion to various tumor suppressors, including Tp53, Cdkn2A, and Pten. We modeled key driver mutations for GBM: PDGFB, NF1, and EGFR. We conducted single-cell RNA sequencing, combined with spatially resolved transcriptomics and proteomics. Data integration techniques, including horizontal (mutual nearest neighbors) and vertical (weighted nearest neighbors) approaches, were used and analyzed with the SPATA2 software. Advanced computational methods, such as graph neural networks and single-cell deconvolution, were applied to examine GBM genotypes and the spatial variability within tumors and their TMEs.



 Our findings revealed persistent cellular heterogeneity among tumor cells under various genetic conditions. However, specific genetic alterations influenced the cells toward dominant transcriptional states. NF1 and EGFR mutations resulted in shifts towards MES and AC-like states, respectively, unlike PDGFB, which promoted a NPC/OPC-like state. These transcriptional shifts were strongly associated with distinct microenvironmental changes, such as enhanced neuronal signaling in PDGFB-driven GBMs versus pronounced immune infiltration in NF1 and EGFR models. Utilizing weighted co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA) alongside spatial-niche deconvolution, we identified a mesenchymal transition leading to an immunosuppressive TME in NF1-driven GBMs and increased lymphocyte presence in EGFR-amplified GBMs, especially around blood vessels. Similar spatial patterns were observed in human spatially resolved transcriptomic data, highlighting the complexity and variability of genetic drivers in human GBMs.



 Our genetically engineered mouse models provide a valuable platform for investigating GBM with defined driver mutations. By leveraging sophisticated analytical techniques and computational modeling, this study underscores the potential to elucidate the interactions between genetic drivers and the tumor microenvironment. Our approach paves the way for developing targeted therapies and precision medicine strategies in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0056b4191c0de2f537409a208d3f06f5659352" target='_blank'>
              OS07.7.A DISSECTING SPATIAL DIVERSITY OF GLIOBLASTOMA GENOTYPES
              </a>
            </td>
          <td>
            J Zhang, T. Joshi, D. Hambardzumyan, D. Heiland
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Microscopy and genomics are both used to characterize cell function, but approaches to connect the two types of information are lacking, particularly at subnuclear resolution. While emerging multiplexed imaging methods can simultaneously localize genomic regions and nuclear proteins, their ability to accurately measure DNA-protein interactions is constrained by the diffraction limit of optical microscopy. Here, we describe expansion in situ genome sequencing (ExIGS), a technology that enables sequencing of genomic DNA and superresolution localization of nuclear proteins in single cells. We applied ExIGS to fibroblast cells derived from an individual with Hutchinson-Gilford progeria syndrome to characterize how variation in nuclear morphology affects spatial chromatin organization. Using this data, we discovered that lamin abnormalities are linked to hotspots of aberrant euchromatin repression that may erode cell identity. Further, we show that lamin abnormalities heterogeneously increase the repressive environment of the nucleus in tissues and aged cells. These results demonstrate that ExIGS may serve as a generalizable platform for connecting nuclear abnormalities to changes in gene regulation across disease contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e86969e327346d3a55aa56def236fca89f4284c6" target='_blank'>
              Expansion in situ genome sequencing links nuclear abnormalities to hotspots of aberrant euchromatin repression
              </a>
            </td>
          <td>
            Ajay S. Labade, Zachary D. Chiang, Caroline Comenho, Paul L. Reginato, Andrew C. Payne, Andrew S. Earl, Rojesh Shrestha, Fabiana M. Duarte, Ehsan Habibi, Ruochi Zhang, George M. Church, E. Boyden, Fei Chen, Jason D. Buenrostro
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Recent advancements in spatial biology have revolutionized our understanding of the organization and functional dynamics of cells and tissues. In this study, we applied Molecular Pixelation (MPX), a single-cell spatial proteomics assay, to investigate the modulation of the cell surface proteome in an in vitro drug screening model using the ETV6::RUNX1 acute lymphoblastic leukemia (ALL) cell line, Reh. Specifically, we focused on the in vitro response to fludarabine, a chemotherapeutic agent used prior to allogenic stem cell transplantation and chimeric antigen receptor (CAR)-T cell therapy in high-risk, refractory, or relapsed ALL patients. Using MPX, we quantified changes in protein abundance, spatial distribution, and colocalization of 76 targeted cell surface proteins in Reh cells before and after fludarabine treatment. Our analysis revealed 25 proteins with altered abundance, 24 proteins with increased polarity, and 138 protein pairs with modified colocalization following treatment. Notably, the tetraspanins CD82 and CD53, which are known for their roles in chemotherapy resistance, exhibited increased abundance, polarization, and colocalization post-treatment, suggesting their potential as a therapeutic scaffold. These findings underscore the unique ability of spatially resolved single-cell proteomics to uncover nuanced cellular responses that would otherwise remain undetected.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f5a352c544200f32c43afeccf0c1c6f723f3d81" target='_blank'>
              Mapping the Spatial Proteome of Leukemia Cells Undergoing Fludarabine Treatment
              </a>
            </td>
          <td>
            , Henrik Gezelius, Anna Pia Enblad, Anders Lundmark, Mariya Lysenkova Wiklander, O. Krali, Martin Åberg, Arja Harila, A. Raine, Claes Andersson, J. Nordlund
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The interaction of glioblastoma (GB) and microglia is critical due to its implications for tumor progression, immune response modulation, and potential therapeutic strategies. However, the role of microglia in GB pathogenesis remains unclear, especially regarding the in vivo dynamics of their interplay. Performing three-photon imaging in an autochthonous, immunocompetent mouse GB model, we examined tumor/microglia dynamics within previously inaccessible regions at the GB far infiltration zone in the corpus callosum. Initially, microglia increased tissue surveillance upon encountering GB-cells in sparsely infiltrated areas. In contrast, when GB-cell density increased, microglia reduced surveillance, suggesting a biphasic response to tumor invasion. Additionally, microglia were not uniformly attracted to infiltrating GB-cells; only a subset moved directionally toward them within a defined spatial range. This study provides insight into the heterogeneity of the immune response to tumor invasion and the dynamics of microglia-GB interactions in vivo. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39820cb3045ee584de7bf3d731fda31accb858a" target='_blank'>
              Dynamics of microglia-glioblastoma crosstalk at the far infiltration zone
              </a>
            </td>
          <td>
            F. Nebeling, Falko Fuhrmann, M. Mittag, Fabrizio Mussachio, Nala Gockel, Lea L Friker, S. Leonardelli, S. Filser, A. Deli, Miriam Stork, D. Bano, Torsten Pietsch, S. Parrinello, M. Hölzel, U. Herrlinger, P. Salomoni, M. Fuhrmann
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Background Breast cancer's complex transcriptional landscape requires an improved understanding of cellular diversity to identify effective treatments. The study of genetic variations among breast cancer subtypes at single‐cell resolution has potential to deepen our insights into cancer progression. Methods In this study, we amalgamate single‐cell RNA sequencing data from patient tumours and matched lymph metastasis, reduction mammoplasties, breast cancer patient‐derived xenografts (PDXs), PDX‐derived organoids (PDXOs), and cell lines resulting in a diverse dataset of 117 samples with 506 719 total cells. These samples encompass hormone receptor positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple‐negative breast cancer (TNBC) subtypes, including isogenic model pairs. Herein, we delineated similarities and distinctions across models and patient samples and explore therapeutic drug efficacy based on subtype proportions. Results PDX models more closely resemble patient samples in terms of tumour heterogeneity and cell cycle characteristics when compared with TNBC cell lines. Acquired drug resistance was associated with an increase in basal‐like cell proportions within TNBC PDX tumours as defined with SCSubtype and TNBCtype cell typing predictors. All patient samples contained a mixture of subtypes; compared to primary tumours HR+ lymph node metastases had lower proportions of HER2‐Enriched cells. PDXOs exhibited differences in metabolic‐related transcripts compared to PDX tumours. Correlative analyses of cytotoxic drugs on PDX cells identified therapeutic efficacy was based on subtype proportion. Conclusions We present a substantial multimodel dataset, a dynamic approach to cell‐wise sample annotation, and a comprehensive interrogation of models within systems of human breast cancer. This analysis and reference will facilitate informed decision‐making in preclinical research and therapeutic development through its elucidation of model limitations, subtype‐specific insights and novel targetable pathways. Key points Patient‐derived xenografts models more closely resemble patient samples in tumour heterogeneity and cell cycle characteristics when compared with cell lines. 3D organoid models exhibit differences in metabolic profiles compared to their in vivo counterparts. A valuable multimodel reference dataset that can be useful in elucidating model differences and novel targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b1b9e5e204010f0ff424f3f3d4b1b7c96f99e7a" target='_blank'>
              Single‐cell transcriptional atlas of human breast cancers and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X Steven Chen, Yunshun Chen, C. Perou, Brian D Lehmann, Jane E. Visvader, J. C. Harrell
          </td>
          <td>2024-10-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>161</td>
        </tr>

        <tr id="Intraductal papillary mucinous neoplasms (IPMN) is one of the known precancerous lesions. Patients’ prognosis is aggravated as IPMN transforms into invasive Pancreatic Ductal Adenocarcinoma (PDAC). The molecular mechanisms underlying this progression lack effective experimental models and urgently need to be elaborated. We performed spatial transcriptomics (ST) on fresh tissue samples from the same patient including normal pancreas, high-grade IPMN, and invasive PDAC, and described the step-by-step development of transcriptional landscape including clone evolution and adjacent TME feature variation. Our findings identified the master transcript factors and critical signaling pathways promoting IPMN progression to invasive PDAC. Additionally, both IPMN and PDAC harbored the ELF3, MYC, and KLF4 amplification. The Spatial CNV profile demonstrated significant heterogeneity among PDAC in their spatial distribution compared to IPMN, with seven distinct subclones showing diverse functions, such as hypoxia, oxidative phosphorylation, and epithelial-mesenchymal Transition. We observed a marked shift in the immune landscape, with the depletion of CD4+ T-cells and dendritic cells and an increase in immune-suppressive M2 macrophages in invasive PDAC, indicating a transition to an immune-evasive microenvironment. Additionally, cancer-associated fibroblasts (CAFs), particularly myofibroblastic CAFs, were enriched adjacent to invasive PDAC, suggesting their active role in tumor progression. By leveraging spatial transcriptomic analysis, our study provides comprehensive insights into the intricate molecular landscape that underlies the progression of IPMNs to invasive PDAC. These findings not only enhance our understanding of this complex process but also offer valuable knowledge for early diagnosis and intervention. Highlights Spatial CNV analysis reveals clonal evolution and distinct subclones in PDAC. Key drivers like ELF3, MYC, and KLF4 are amplified in both IPMN and invasive PDAC. Immune landscape shifts from pro-inflammatory in IPMN to immune-evasive in PDAC. Enrichment of myofibroblastic CAFs suggests their role in tumor progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4caecc6532101fd5cd79dde472c73a99ffa9c7b0" target='_blank'>
              Evolution of the Spatial transcriptomic landscape during the progression of high-grade pancreatic intraductal papillary mucinous neoplasms to invasive cancer
              </a>
            </td>
          <td>
            Sirui Peng, Qiangxing Chen, Zixin Chen, Mengling Yao, Yu Cai, Du He, Yu Cai, K. Cheng, Jun Li, H. Cai, P. Gao, Xiafei Gu, Xin Wang, Yongbin Li, Man Zhang, L. Meng, Qi Xia, Panpan Xu, Jin Zhou, Zhong Wu, Bing Peng
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Predicting the progression of solid cancers based solely on genetics is challenging due to the influence of the tumor microenvironment (TME). For colorectal cancer (CRC), tumors deficient in mismatch repair (dMMR) are more immune infiltrated than mismatch repair proficient (pMMR) tumors and have better prognosis following resection. Here we quantify features of the CRC TME by combining spatial profiling with genetic analysis and release our findings via a spatially enhanced version of cBioPortal that facilitates multi-modal data exploration and analysis. We find that ∼20% of pMMR tumors exhibit similar levels of T cell infiltration as dMMR tumors and that this is associated with better survival but not any specific somatic mutation. These T cell-infiltrated pMMR (tipMMR) tumors contain abundant cells expressing PD1 and PDL1 as well as T regulatory cells, consistent with a suppressed immune response. Thus, like dMMR CRC, tipMMR CRC may benefit from immune checkpoint inhibitor therapy. SIGNIFICANCE pMMR tumors with high T cell infiltration and active immunosuppression are identifiable with a mid-plex imaging assay whose clinical deployment might double the number of treatment-naïve CRCs eligible for ICIs. Moreover, the low tumor mutational burden in tipMMR CRC shows that MMR status is not the only factor promoting immune infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb91d144afd07ee13d239cb16bf5689a34f70783" target='_blank'>
              Integrating spatial profiles and cancer genomics to identify immune-infiltrated mismatch repair proficient colorectal cancers
              </a>
            </td>
          <td>
            Jeremiah Wala, Ino de Bruijn, S. Coy, Andréanne Gagné, Sabrina Chan, Yu-An Chen, J. Hoffer, J. Muhlich, N. Schultz, Sandro Santagata, P. Sorger
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bca16ff9172b241a4b47443f7584ae07d099226" target='_blank'>
              Transcriptomic and proteomic spatial profiling of pediatric and adult diffuse midline glioma H3 K27-Altered
              </a>
            </td>
          <td>
            Sudarshawn Damodharan, Jack M. Shireman, Elliot Xie, Emily Distler, Christina Kendziorski, Mahua Dey
          </td>
          <td>2024-09-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba40a48a9694556eef4d0d290c4702c1cf12dc5" target='_blank'>
              Spatially resolved analysis of pancreatic cancer identifies therapy-associated remodeling of the tumor microenvironment.
              </a>
            </td>
          <td>
            C. Shiau, Jingyi Cao, Dennis Gong, M. Gregory, Nicholas J Caldwell, Xunqin Yin, Jae-Won Cho, Peter L. Wang, J. Su, Steven Wang, J. Reeves, T. Kim, Youngmi Kim, Jimmy A. Guo, N. A. Lester, Jung Woo Bae, Ryan Zhao, Nathan Schurman, J. L. Barth, M. L. Ganci, Ralph Weissleder, Tyler Jacks, M. Qadan, Theodore S. Hong, J. Y. Wo, Hannah J Roberts, J. Beechem, Carlos Fernández-del Castillo, M. Mino-Kenudson, David T Ting, M. Hemberg, W. L. Hwang
          </td>
          <td>2024-09-03</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>50</td>
        </tr>

        <tr id="Abstract Neuroendocrine transdifferentiation (NEtD), also commonly referred to as lineage plasticity, emerges as an acquired resistance mechanism to molecular targeted therapies in multiple cancer types, predominately occurs in metastatic epidermal growth factor receptor (EGFR)‐mutant non‐small cell lung cancer treated with EGFR tyrosine kinase inhibitors and metastatic castration‐resistant prostate cancer treated with androgen receptor targeting therapies. NEtD tumors are the lethal cancer histologic subtype with unfavorable prognosis and limited treatment. A comprehensive understanding of molecular mechanism underlying targeted‐induced plasticity could greatly facilitate the development of novel therapies. In the past few years, increasingly elegant studies indicated that NEtD tumors share key the convergent genomic and phenotypic characteristics irrespective of their site of origin, but also embrace distinct change and function of molecular mechanisms. In this review, we provide a comprehensive overview of the current understanding of molecular mechanism in regulating the NEtD, including genetic alterations, DNA methylation, histone modifications, dysregulated noncoding RNA, lineage‐specific transcription factors regulation, and other proteomic alterations. We also provide the current management of targeted therapies in clinical and preclinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7693a38c0d6b0d0a5a68e9cf1e1598b353f12ed8" target='_blank'>
              Neuroendocrine transdifferentiation in human cancer: molecular mechanisms and therapeutic targets
              </a>
            </td>
          <td>
            Jun Jiang, Donghui Han, Jiawei Wang, Weihong Wen, Rui Zhang, Weijun Qin
          </td>
          <td>2024-10-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="While a subset of patients with metastatic melanoma achieves durable responses to immune checkpoint blockade (ICB) therapies, the majority ultimately exhibit either innate or acquired resistance to these treatments. However, the molecular mechanisms underlying resistance to ICB therapies remain elusive and are warranted to elucidate. Here, we comprehensively investigated the tumor and tumor immune microenvironment (TIME) of paired pre- and post-treatment tumor specimens from metastatic melanoma patients who were primary or secondary resistance to anti-CTLA-4 and/or anti-PD-1/PD-L1 therapies. Differentially expressed gene (DEG) analysis and single-sample gene set enrichment analysis (ssGSEA) with transcriptomic data identified cell cycle and c-MYC signaling as pathway-based resistance signatures. And weighted gene co-expression network analysis (WGCNA) revealed the activation of a cross-resistance meta-program involving key signaling pathways related to tumor progression in ICB resistant melanoma. Moreover, spatially-resolved, image-based immune monitoring analysis by using NanoString’s digital spatial profiling (DSP) and Cyclic Immunofluorescence (CyCIF) showed infiltration of suppressive immune cells in the tumor microenvironment of melanoma with resistance to ICB therapies. Our study reveals the molecular mechanisms underlying resistance to ICB therapies in patients with metastatic melanoma by conducting such integrated analyses of multi-dimensional data, and provides rationale for salvage therapies that will potentially overcome resistance to ICB therapies. Statement of translational relevance This study paves the way for the creation of innovative therapeutic strategies, aimed at subverting resistance to immune checkpoint blockade (ICB) therapies in metastatic melanoma patients. By unraveling the specific molecular mechanisms underlying resistance, scientists can design effective alternative treatments that target pathways such as pathways associated with cell cycle dysregulation and c-MYC signaling. Furthermore, through the application of advanced immune monitoring techniques such as NanoString Digital Spatial Profiling (DSP) and Cyclic Immunofluorescence (CyCIF), this study has significantly enriched our understanding of the tumor microenvironment. This enhanced characterization facilitates the discovery of potential biomarkers that may forecast a patient’s response to ICB treatment. Ultimately, these advancements could potentially refine patient outcomes and foster the development of more personalized cancer treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59c359686d6b4f4f12c03d81c082e61089dfa6d8" target='_blank'>
              A Comprehensive Proteogenomic and Spatial Analysis of Innate and Acquired Resistance of Metastatic Melanoma to Immune Checkpoint Blockade Therapies
              </a>
            </td>
          <td>
            Shiyou Wei, Kuang Du, Hongbin Lan, Zhenyu Yang, Yulan Deng, Zhi Wei, Dennie T. Frederick, Jinho Lee, Marilyne Labrie, Tian Tian, Tabea Moll, Yeqing Chen, Ryan J. Sullivan, Gordon B. Mills, Genevieve M. Boland, Keith T. Flaherty, Lunxu Liu, M. Herlyn, Gao Zhang
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="The abundance and biological contribution of natural killer (NK) cells in cancer are controversial. Here, we aim to uncover clinical relevance and cellular roles of NK cells in colon cancer liver metastasis (CCLM). Here, we integrated single-cell RNA-sequencing, spatial transcriptomics (ST), and bulk RNA-sequencing datasets to investigate NK cells’ biological properties and functions in the microenvironment of primary and liver metastatic tumors. Results were validated through an in vitro co-culture experiment based on bioinformatics analysis. Useing single-cell RNA-sequencing and ST, we mapped the immune cellular landscape of colon cancer and well-matched liver metastatic cancer. We discovered that GZMK+ resting NK cells increased significantly in tumor tissues and were enriched in the tumor regions of both diseases. After combining bulk RNA and clinical data, we observed that these NK cell subsets contributed to a worse prognosis. Meanwhile, KIR2DL4+ activated NK cells exhibited the opposite position and relevance. Pseudotime cell trajectory analysis revealed the evolution of activated to resting NK cells. In vitro experiments further confirmed that tumor-cell-co-cultured NK cells exhibited a decidual-like status, as evidenced by remarkable increasing CD9 expression. Functional experiments finally revealed that NK cells exhibited tumor-activating characteristics by promoting the dissociation of SCF (stem cell factor) on the tumor cells membrane depending on cell-to-cell interaction, as the supernatant of the co-culture system enhanced tumor progression. In summary, our findings revealed resting NK cells exhibited a clinical relevance with CCLM, which may be exploited for novel strategies to improve therapeutic outcomes for patients with CCLM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be15cf8ef30b7e56c663c4156f8b61fd676de622" target='_blank'>
              Resting natural killer cells promote the progress of colon cancer liver metastasis by elevating tumor-derived stem cell factor
              </a>
            </td>
          <td>
            Chenchen Mao, Yanyu Chen, Dong Xing, Teming Zhang, Yangxuan Lin, Cong Long, Jiaye Yu, Yunhui Luo, Tao Ming, W. Xie, Zheng Han, Dianfeng Mei, Dan Xiang, Mingdong Lu, Xian Shen, Xiangyang Xue
          </td>
          <td>2024-10-10</td>
          <td>eLife</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The transcriptional state of a cell provides a multi-parameter representation of gene expression programs that reflect its identity and phenotype. However, current transcriptomic profiling technologies result in destruction of the biological sample, preventing direct analysis of transcriptional dynamics in the same living cells over time. Here, we developed a synthetic RNA export system called cellular ‘self-reporting’ to address this fundamental technological limitation. We repurposed the murine leukemia virus retroviral protein Gag to enable diverse types of immortalized and primary mammalian cells to package cellular RNA molecules in virus-like particles (VLPs) for export into the extracellular environment. We applied self-reporting to interrogate the transcriptome-wide dynamics that occur during neuronal differentiation from induced pluripotent stem cells and detected gene expression profiles from individual live cells. Leveraging this genetically encodable approach, we expanded the capabilities of self-reporting through molecular engineering of VLP components. Pseudotyping VLPs with epitope-tagged envelope proteins enabled multiplexed selective live-cell readout of transcriptional states from heterogeneous co-cultures. Furthermore, structure-guided protein engineering of Gag fusions with human RNA binding domains improved the mRNA representation in self-reporting readouts and enabled the directed export of libraries of synthetic barcode transcripts. Taken together, this work establishes self-reporting as a facile and broadly enabling technology for live-cell, transcriptome-scale profiling of dynamic processes across diverse cell types and biological applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd9363ab1dcbb456eb0b3e0944d84bfdb375d0c" target='_blank'>
              Live-cell transcriptomics with engineered virus-like particles
              </a>
            </td>
          <td>
            M. Najia, Jacob Borrajo, Anna Le, FuNien Tsai, Jeremy Y. Huang, Linda G. Griffith, George Q Daley, P. Blainey
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="In clinical applications, spatial data collected under varying conditions, time points, or patients often lack discernible structural alignment. Computational tools designed to align adjacent tissue sections are unsuited for dealing with this structural heterogeneity. There is a growing demand for methods that can effectively align and compare spatial data in the absence of obvious visual correspondence. To address this challenge, we developed an interpretable cell mapping strategy by considering spatial context at various scales. Our approach outperforms existing mapping tools in dealing with heterogeneous samples and is flexible enough to map cells across samples, technologies, resolutions, developmental and regenerative time. Using our approach, we showed spatiotemporal decoupling of cells during development. We even performed alignment for a population of spatial data from cancer patients to identify sub- populations. Our interpretable mapping approach facilitates systemic comparison and analysis of heterogeneous spatial data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3efb8af02c2551620bc2f4404557c0e501a57546" target='_blank'>
              Multi-scale and multi-context interpretable mapping of cell states across heterogenous spatial samples
              </a>
            </td>
          <td>
            P. Martin, Wenqi Wang, Hyobin Kim, Henrietta Holze, Kyoung Jae Won
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="We present GBM-cRegMap, an online resource providing a comprehensive coregulatory influence network perspective on glioblastoma (GBM) heterogeneity and plasticity. Using representation learning algorithms, we derived two components of this resource: GBM-CoRegNet, a highly specific coregulatory network of tumor cells, and GBM-CoRegMap, a unified network influence map based on 1612 tumors from 16 studies. As a widely applicable closed-loop system connecting cellular models and tumors, GBM-cRegMap will provide the GBM research community with an easy-to-use web tool (https://gbm.cregmap.com) that maps any existing or newly generated transcriptomic “query” data to a reference coregulatory network and a large-scale manifold of disease heterogeneity. Using GBM-cRegMap, we demonstrated the synergy between the two components by refining the molecular classification of GBM, identifying potential key regulators, and aligning the transcriptional profiles of tumors and in vitro models. Through the amalgamation of a vast dataset, we validated the proneural (PN)-mesenchymal (MES) axis and identified three subclasses of classical (CL) tumors: astrocyte-like (CL-A), epithelial basal-like (CL-B), and cilium-rich (CL-C). We revealed the CL-C subclass, an intermediate state demonstrating the plasticity of GBM cells along the PN-MES axis under chemotherapy. We identified key regulators, such as PAX8, and NKX2.5, involved in TMZ resistance. Notably, NKX2.5, more expressed in higher-grade gliomas, negatively impacts patient survival and regulates genes involved in glucose metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd1c463b43fec2813fd1baca33de19d265544140" target='_blank'>
              A coregulatory influence map of glioblastoma heterogeneity and plasticity – From cells in vitro to tumors and back again
              </a>
            </td>
          <td>
            Chloé Bernhard, Konstantinos Geles, Geoffrey Pawlak, Wajdi Dhifli, Aurélien Dispot, Jules Dusol, Maria Kondratova, Sophie Martin, M. Messé, Damien Reita, David Tulasne, I. Seuningen, N. Entz-Werlé, S. Ciafrè, Monique Dontenwill, Mohamed Elati
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Epithelial ovarian cancer (EOC) is characterized by resistance to platinum-based therapy, resulting in rapid progression and poor survival. Here, we ask whether drug resistance and invasiveness coevolve to drive metastasis. Selection experiments involving pulsed carboplatin exposure established isogenic chemoresistant variants of lines, which typify high-grade serous ovarian carcinoma (HGSOC), the most aggressive type of EOC. Time-lapse imaging showed enhanced migration of resistant single cells and their collectives. Resistant cell spheroids spread faster on Collagen I substrata than sensitive controls. The resistant OVCAR-3 transcriptome was ontologically enriched for migration and showed overlap with previously reported markers of resistance in EOC patients and other evolved lines. Gene set enrichment predicted transition between epithelial, mesenchymal, and amoeboid states is higher in resistance compared to control lines. Lower matrix adhesion, weak focal adhesion, and highly deformable and translatory dynamics of cell collectives indicated that resistant cancer cells displayed a unique collective amoeboid-like migration. When injected intraperitoneally into immunodeficient mice, resistant cells colonized to a greater extent on parietal mucosae. Ex vivo, suspended resistant cells formed moruloids associated with quicker peritoneal adhesion, clearing human coelomic mesothelial monolayers with higher efficiency. Knockdown in resistant OVCAR-3 cells of two upregulated proteins, E-cadherin and LGALS3BP, had distinct consequences. E-cadherin knockdown partially restored sensitivity to carboplatin but did not affect invasion. In contrast, silencing LGALS3BP decreased invasion but not resistance. Our results suggest that drug resistance and invasiveness could coevolve through the upregulation of distinct trait drivers in EOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27926bd61dc910737d26dac10dc74c6a8eecd80b" target='_blank'>
              Collective amoeboid dynamics drives colonization of drug-resistant ovarian cancer cells
              </a>
            </td>
          <td>
            Shyamili Goutham, Anchita Gopikrishnan, BV Harshavardhan, Nivedhya Venas, R. Sabarinathan, M. Jolly, R. Bhat
          </td>
          <td>2024-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Astrocytes comprise ∼50% of all brain cells and present distinct morphological, molecular and functional properties in different brain regions. In glioblastoma (GBM), an aggressive primary brain tumour, tumour-associated astrocytes (TAAs) become activated and exhibit different transcriptomic profiles, morphology and functions supporting disease progression. Heterogeneity and specific roles of TAAs within various regions of tumours are poorly known. Advancements of single-cell and spatial transcriptomics allow to profile tumours at unprecedented resolution revealing cell phenotypes, hidden functionalities and spatial architecture in disease-specific context. We combined spatial transcriptomics and multiple immunofluorescent staining to visualize TAAs heterogeneity and location of various subpopulations in intracranial murine gliomas. Using distinct gene expression profiles, we identified subtypes of TAAs with distinct localization and inferred their specialized functionalities. Gene signatures associated with TAAs reflected their reprograming in the tumour microenvironment (TME), revealed their multiple roles and potential contributing factors shaping the local milieu. Using spatial correlation analysis of the spots, we inferred the interactome of Slc1a2 (encoding a glutamate transporter) with the other markers of TAAs based on segregated areas of the tumour. The designer RGD peptide blocking tumour-microglia communications, alters the spatial distribution of TAAs in experimental gliomas providing insights into potential mechanisms. Spatial transcriptomics combined with multiple staining unveils multiple functional phenotypes of TAAs and interactions within TME. It shows their distinct morphology and unveils different roles in various regions of the tumour. We demonstrate the glioma-induced heterogeneity of TAAs and their adaption to the pharmacologically-induced modification of the immunosuppressive TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2a092533402eea8774b3945747486b079a48e5e" target='_blank'>
              Integration of spatial transcriptomics with immunofluorescence staining reveals spatial heterogeneity and plasticity of astrocytes in experimental glioblastomas
              </a>
            </td>
          <td>
            Mitrajit Ghosh, P. Pilanc-Kudlek, Karol Jacek, Szymon Baluszek, Katarzyna Poleszak, P. Szadkowska, B. Gielniewski, Aleksandra Ellert Miklaszewska, Bozena Kaminska
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive brain cancer with a poor survival rate. Despite hundreds of clinical trials, there is no effective targeted therapy. Glioblastoma stem cells (GSCs) are an important GBM model system. In culture, these cells form spatial structures that share morphological aspects with their source tumors. We collected 17,000 phase contrast images of 15 patient-derived GSC lines growing to confluence. We find that GSCs grow in characteristic multicellular patterns depending on their transcriptional state. Interpretable computer vision algorithms identified specific image features that predict transcriptional state across multiple cell confluency levels. This relationship will be useful in developing GSC screens where image features can be used to identify how GSC biology changes in response to perturbations simply by imaging cultured cells on plates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1043d1e1cafd115cae393754618f9b76789fb01" target='_blank'>
              Glioblastoma stem cells show transcriptionally correlated spatial organization
              </a>
            </td>
          <td>
            Shamini Ayyadhury, Patty Sachamitr, M. Kushida, Nicole I Park, F. Coutinho, Owen Whitley, Panagiotis Prinos, Cheryl H. Arrowsmith, Peter B. Dirks, Trevor Pugh, Gary D. Bader
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8b0d44c911ccfcc30fd21e07313533937dc5a4" target='_blank'>
              Ultralow-dose irradiation enables engraftment and intravital tracking of disease initiating niches in clonal hematopoiesis
              </a>
            </td>
          <td>
            Kevin Lee, Wimeth Dissanayake, Melissa MacLiesh, Cih-Li Hong, Zi Yin, Yuko Kawano, Christina M Kaszuba, Hiroki Kawano, Emily R. Quarato, Brian Marples, Michael Becker, Jeevisha Bajaj, Laura M Calvi, Shu-Chi A. Yeh
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfb73fa1be8db27e4aafae1a4087794eca77af2e" target='_blank'>
              Spatial interactions modulate tumor growth and immune infiltration
              </a>
            </td>
          <td>
            S. Marzban, Sonal Srivastava, Sharon Kartika, Rafael Bravo, Rachel Safriel, Aidan Zarski, Alexander R. A. Anderson, Christine H Chung, Antonio L. Amelio, Jeffrey West
          </td>
          <td>2024-09-30</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667052bc6e34331d4c683bafe12d9a9b03403b8b" target='_blank'>
              Single-cell landscape of intratumoral heterogeneity and tumor microenvironment remolding in pre-nodal metastases of breast cancer
              </a>
            </td>
          <td>
            Kaidong Liu, Huiming Han, Kai Xiong, Songmei Zhai, Xiuqi Yang, Xinmiao Yu, Bo Chen, Mingyue Liu, Qi Dong, Hongxue Meng, Yunyan Gu
          </td>
          <td>2024-08-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The tumor microenvironment of brain metastases has become a focus in the development of immunotherapeutic drugs. However, countless brain metastasis patients have not experienced clinical benefit. Thus, understanding the immune cell composition within brain metastases, and how the immune cells interact with each other and other microenvironmental cell types, may be critical for optimizing immunotherapy. We applied spatial whole transcriptomic profiling with extensive multiregional sampling (19-30 regions per sample) and multiplex immunohistochemistry on formalin-fixed, paraffin-embedded lung cancer brain metastasis samples. We performed deconvolution of gene expression data to infer the abundances of immune cell populations and inferred spatial relationships from the multiplex immunohistochemistry data. We also described cytokine networks between immune and tumor cells and used a protein language model to predict drug-target interactions. Finally, we performed deconvolution of bulk RNA data to assess the prognostic significance of immune-metastatic tumor cellular networks. We show that immune cell infiltration has a negative prognostic role in lung cancer brain metastases. Our in-depth multiomics analyses further reveal recurring intratumoral immune heterogeneity and the segregation of myeloid and lymphoid cells into distinct compartments that may be influenced by distinct cytokine networks. By employing computational modeling, we identify drugs that may target genes expressed in both tumor core and regions bordering immune infiltrates. Finally, we illustrate the potential negative prognostic role of our immune-metastatic tumor cellular networks. Our findings advocate for a paradigm shift from focusing on individual genes or cell types, towards targeting networks of immune and tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488bc40a00ebddc1b6645e7141cbe9f55941284e" target='_blank'>
              Spatial multiomics reveal intratumoral immune heterogeneity with distinct cytokine networks in lung cancer brain metastases.
              </a>
            </td>
          <td>
            Gustav Christensson, M. Bocci, Julhash U. Kazi, Geoffroy Durand, Gustav Lanzing, Kristian Pietras, Hugo Gonzalez Velozo, C. Hagerling
          </td>
          <td>2024-10-14</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1374088281d4f8b66ab8cd65f25ae26c49ad31" target='_blank'>
              Spatial transcriptomics reveals tumor-derived SPP1 induces fibroblast chemotaxis and activation in the hepatocellular carcinoma microenvironment
              </a>
            </td>
          <td>
            W. Tong, Tianze Wang, Yi Bai, Xingpeng Yang, Pinsheng Han, Liuyang Zhu, Yamin Zhang, Zhongyang Shen
          </td>
          <td>2024-09-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma is the most hypovascular solid tumor, imposing harsh constraints on tumor metabolism. Cancer cells adapt to reduced access to vascular-derived nutrients and O2 by cooperating with abundant stromal cells. Specifically, critical nutrients including alanine, proline, glycosaminoglycans, branched chain ketoacids, lysophosphatidic acid, and unsaturated fatty acids can be provided by fibroblasts in vitro and in vivo. However, the extent of metabolic support provided by other cell types in the PDAC TME is relatively unexplored. We hypothesized that cell types that are cooperating metabolically with cancer cells will be spatially positioned near cancer cells in these tumors. We performed Xenium spatial transcriptomics on 8 treatment-naive PDAC resections and generated a spatial atlas of nearly 700,000 cells across a range of expected cell types. We noted striking neutrophil aggregates, nucleating in the glands of classical-like malignant cells and elaborating extraglandularly. Cellular neighborhood analysis demonstrates neutrophil aggregation is strongly correlated with malignant cell proliferation and low endothelial content, suggesting neutrophils may be an important source of pro-tumoral, non-vascular nutrients in the PDAC TME. Reanalysis of published scRNAseq data revealed 3 neutrophil metabolic states in human PDAC tumors. Metabolic cluster 7 neutrophils were found only in PDAC patient samples but not healthy controls and were enriched for Type I interferon response as well as pentose phosphate pathway and iron uptake. Metabolic cluster 2 neutrophils were strikingly depleted from PDAC tumor samples compared to blood, suggesting decreased infiltration or increased cell death. Cluster 2 neutrophils were enriched for several catabolic pathways including glycogenolysis, autophagy, and proteasome, and selectively express the NETosis gene PADI4. Multiplexed immunofluorescence demonstrated that neutrophil aggregates in PDAC are citH3+/MPO+ large neutrophil extracellular traps (NETs). We reasoned pancreatic cancer cells could proliferate under tumor-like metabolic stress by repurposing amino acids liberated by autophagy, glucose liberated by glycogenolysis, or simply consuming bulk neutrophil dead cell debris via macropinocytosis. Using tumor interstitial fluid media (TIFM) and 0.5% O2 to mimic tumor-like metabolic stress, we cocultured pancreatic cancer cells with human primary neutrophils and HL60-derived neutrophil-like cells. However, under tumor-like metabolic conditions, coculture with human primary neutrophils failed to substantially increase cancer cell proliferation and NETotic neutrophils were cytotoxic. However, we noticed enhanced neutrophil viability in tumor-like metabolic conditions, suggesting metabolic regulation of neutrophil cell death. Together, these data shed light on the metabolic phenotypes and interactions with cancer cells of neutrophils in the PDAC tumor microenvironment.
 Citation Format: Carson D Poltorack, Sydney M Shaffer, M Celeste Simon. Spatial and single cell transcriptomics uncovers metabolic reprogramming and NETosis of neutrophils in human pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A065.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e97afc3e27a38cdf1cbacb0a0b2dbe91b12facfe" target='_blank'>
              Abstract A065: Spatial and single cell transcriptomics uncovers metabolic reprogramming and NETosis of neutrophils in human pancreatic cancer
              </a>
            </td>
          <td>
            Carson D. Poltorack, Sydney M Shaffer, M. C. Simon
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0107b91d27e36a7693d55731e76f890903068027" target='_blank'>
              Single cell spatial profiling of FFPE splenic tissue from a humanized mouse model of HIV infection
              </a>
            </td>
          <td>
            Guoxin Wu, Samuel H. Keller, Luca Sardo, Brian Magliaro, Paul Zuck, C. Balibar, Claire Williams, Liuliu Pan, Mark Gregory, Kathy Ton, Jill Maxwell, Carol Cheney, Tom Rush, Bonnie J. Howell
          </td>
          <td>2024-10-08</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Multiplex imaging technologies have developed rapidly over the past decades. The advancement of multiplex imaging has been driven in part by the recognition the spatial organisation of cells can represent important prognostic biomarkers and that simply studying the composition of cells in diseased tissue is often insufficient. There remains a lack of tools that can perform spatial analysis at the level of cellular aggregates (a common histopathological presentation) such as tumors and granulomas, with most analysis packages focussing on smaller regions of interest and potentially missing patterns in the overall lesion structure and cellular distribution. Here we present a protocol to quantify the cellular structure of entire tissue lesions built around two novel metrics. The Total Cell Preference Index reports whether a lesion tends to change in density in its central vs peripheral areas and can indicate the extent of necrosis across the entire lesion. The Immune Cell Preference Index then reports whether each immune cell-type is located more centrally or peripherally across the entire lesion. The output of both indexes is a single number readout for simple interpretation and visualization, and they can be applied to lesions of any size or shape. Additionally, this protocol can be applied to any slide-scanning multiplexed imaging system, either based on protein and nucleic acid staining. Finally, the protocol uses the open-source software QuPath and can be utilized by researchers with a basic understanding of QuPath with the full protocol able to be applied on pre-generated images within one hour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd048c5276460698536d41824ce54a133b750a5e" target='_blank'>
              A highly adaptable protocol for mapping spatial features of cellular aggregates in tissues
              </a>
            </td>
          <td>
            Andrew Sawyer, Nick Weingaertner, E. Patrick, Carl G Feng
          </td>
          <td>2024-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a7e2a6359020fadce11d67094c49cba1e4ed0b" target='_blank'>
              Multi-scale transcriptomics reveals that specific tumor cells promote lung adenocarcinoma metastasis through crosstalk with the microenvironment
              </a>
            </td>
          <td>
            Qi Cai, Lin Shi, Meng-ya Zhang, Peng Chen
          </td>
          <td>2024-10-03</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Recent advances in neuro-oncology highlighted the meninges as an important immunological hub at the crossroads between systemic immunity and the central nervous system. Meningiomas are predominantly benign tumors arising from meningeal cells. However, about 20% are high-grade cases, commonly refractory to standard treatment, showing high recurrence rates. Methylation-based classification has crucially improved meningioma patient stratification. However, the biological processes that govern differences between these classes remain unknown. Here, we investigate the role of differential immune infiltration, cellular communities, and their spatial organization. In light of previously found differential immune infiltration across methylation classes, this study aims to spatially resolve the role of myeloid cells and study their influence on tumor progression and recurrence.



 We deployed multiplexed ion beam imagining and spatial transcriptomics on a retrospective cohort with extensive clinical annotations. Analyzing single-cell transcriptomic data of a subset of patients (n=22), we delineated the expression of gene and protein markers that were used to design an antibody panel comprising 41 targets for cell-type identification, phenotypic and metabolic characterization at the protein level. Additionally, we curated a targeted gene panel for spatial transcriptomics to perform granular spatial analysis. An integration of histological, spatial transcriptomic, and protein layers provides a comprehensive multi-omics approach to studying spatial cellular interactions.



 Transcriptional programs and cell populations observed in single-nuclei data were captured in the spatial context revealing differences in the myeloid compartment across meningioma methylation classes. In specific classes, myeloid cells constitute a significant part of the tumor bulk warranting further investigation into their phenotypic differences. Accordingly, our analyses revealed a distinct phenotypic macrophage landscape along disease progression. Furthermore, spatial profiling of the metabolic regulome displayed distinct niche formation within perivascular and tumor compartments.



 In this study we confirm that macrophages undergo a phenotypic switch from more tolerogenic in low-grade to inflammatory in high-grade cases of meningioma. Correspondingly, specific phenotypic and metabolic features of myeloid and tumor cells contribute to tumor progressions. Considering the limited targeted therapy options and poor responses to immunotherapy in meningioma patients, we hope that our spatial multi-omics analysis may reveal recurrent infiltration patterns of phenotypically distinct cell types that could be informative of biological processes and predictive of disease outcome.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c6aee05db85854153c8ed4386e1bdf7d8947c0" target='_blank'>
              P02.23.A SPATIAL PROFILING OF THE MYELOID COMPARTMENT IN MENINGIOMA REVEALS DISTINCT PHENOTYPIC HETEROGENEITY ACROSS METHYLATION CLASSES
              </a>
            </td>
          <td>
            D. Calafato, Y. Tang, G. Rukhovich, B. Patel, A. Suwala, F. Sahm, M. Gerstung, F. Hartmann
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Heterogeneity of tumor metabolism is an important, but still poorly understood aspect of tumor biology. Present work is focused on the visualization and quantification of cellular metabolic heterogeneity of colorectal cancer using fluorescence lifetime imaging (FLIM) of redox cofactor NAD(P)H. FLIM-microscopy of NAD(P)H was performed in vitro in four cancer cell lines (HT29, HCT116, CaCo2 and CT26), in vivo in the four types of colorectal tumors in mice and ex vivo in patients’ tumor samples. The dispersion and bimodality of the decay parameters were evaluated to quantify the intercellular metabolic heterogeneity. Our results demonstrate that patients’ colorectal tumors have significantly higher heterogeneity of energy metabolism compared with cultured cells and tumor xenografts, which was displayed as a wider and frequently bimodal distribution of a contribution of a free (glycolytic) fraction of NAD(P)H within a sample. Among patients’ tumors, the dispersion was larger in the high-grade and early stage ones, without, however, any association with bimodality. These results indicate that cell-level metabolic heterogeneity assessed from NAD(P)H FLIM has a potential to become a clinical prognostic factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d39beee686da3cb9d1fa5f224388f8f6c20fccf" target='_blank'>
              Insights into metabolic heterogeneity of colorectal cancer gained from fluorescence lifetime imaging
              </a>
            </td>
          <td>
            A. D. Komarova, S. D. Sinyushkina, Ilia D Shchechkin, I. Druzhkova, Sofia A Smirnova, Vitaliy M Terekhov, A. Mozherov, Nadezhda I. Ignatova, Elena E. Nikonova, E. Shirshin, L. Shimolina, S. V. Gamayunov, V. Shcheslavskiy, M. Shirmanova
          </td>
          <td>2024-08-28</td>
          <td>eLife</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) tumor heterogeneity impedes the development of biomarker assays suitable for early disease detection that would improve patient outcomes. The CA19-9 glycan is currently used as a standalone biomarker for PDAC. Furthermore, previous studies have shown that cancer cells may display aberrant membrane-associated glycans. We therefore hypothesized that PDAC cancer cell subpopulations could be distinguished by aberrant glycan signatures. We used multiplexed glycan immunofluorescence combined with pathologist annotation and automated image processing to distinguish between PDAC cancer cell subpopulations within tumor tissue. Using a training-set/test-set approach, we found that PDAC cancer cells may be identified by signatures comprising 4 aberrant glycans (VVL, CA19-9, sTRA, and GM2) and that there are three glycan-defined PDAC tumor types: sTRA type, CA19-9 type, and intermixed. To determine whether the aberrant glycan signatures could be detected in blood samples, we developed hybrid glycan sandwich assays for membrane-associated glycans. In both patient-matched tumor and blood samples, the proportion of aberrant glycans detected was consistent. Furthermore, our multiplexed glycan immunofluorescent approach proved to be more sensitive and more specific than CA19-9 alone. Our results provide proof of concept for a novel methodology to improve early PDAC detection and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e6b226460c28415c919ed9fc15c5b9945dc42aa" target='_blank'>
              Multiplexed Glycan Immunofluorescence Identification of Pancreatic Cancer Cell Subpopulations in Both Tumor and Blood Samples
              </a>
            </td>
          <td>
            Braelyn Binkowski, Zachary L. Klamer, Chongfeng Gao, Ben Staal, Anna Repesh, Hoang-Le Tran, David M. Brass, Pamela Bartlett, Stephen Gallinger, Maria Blomqvist, J. B. Morrow, Peter Allen, Chanjuan Shi, A. Singhi, Randall Brand, Ying Huang, G. Hostetter, Brian Haab
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Despite recent advances in cancer treatment, only 20-40% of all patients respond to immunotherapy. Further, efficacy varies highly by tumor type. Tumors with high infiltration of canonically anti-tumoral immune cells, such as lung cancers, also tend to display increased sensitivity to treatment. In contrast, those with higher infiltration of pro-tumoral immune cells, such as pancreatic cancers, are much less amenable to immunotherapy. These differences in immune infiltration have been linked to differential transcriptional regulation of immune cell recruiting factors, suggesting that perhaps tumors can intrinsically assemble different immune microenvironments in a tissue-of-origin dependent manner. Thus, it is necessary to better understand the factors that lead to heterogeneity of immune cell infiltration, how these mechanisms differ across tumor types, and how they may be perturbed to improve therapy responses. Here, we utilized two panels of clonal cell lines both driven by Kras-G12D and Tp53-R172H mutations but derived from two different tissues of origin: the lung and pancreas. These panels serve to recapitulate the immune heterogeneity of non-small cell lung cancer and pancreatic ductal adenocarcinoma, respectively, while controlling for genetic drivers that may influence immune infiltration. Analyses of subcutaneously implanted lung and pancreatic-origin tumors revealed that the overall abundance of immunosuppressive granulocytic myeloid-derived suppressor cells (gMDSCs) is significantly higher in pancreatic-origin tumors while the infiltration of dendritic cells is much lower as compared to that of lung-origin tumors, despite controlling for engraftment site. Likewise, lung, but not pancreatic-origin tumors exhibit more rapid tumor growth upon natural killer (NK) cell depletion, suggesting a lung-origin specific role of NK cells in controlling tumor burden. There were also similarities in immune archetype, with T-cell abundance negatively correlating with that of gMDSCs. However, whereas “T-cell inflamed” pancreatic tumors partially respond to immune checkpoint blockade, their lung-origin counterparts do not. We hypothesized that these differences in immune phenotype and subsequent therapy response may be due in part to somatic copy number alterations (SCNAs). We performed sparse whole genome sequencing and identified substantial genomic rearrangements in both lung and pancreatic-origin tumors. Notably, the specific regions of rearrangements differed highly between the two panels, with lung-origin clones showing more amplifications in chromosomes 2 and 11 and deletions in chromosomes 16 and 18. These results suggest that there are indeed tissue-of-origin specific differences in immune phenotype that can influence therapy outcome. Ongoing work seeks to further elucidate specific tumor cell-intrinsic factors that can be leveraged to skew the tumor immune microenvironment towards a more anti-tumoral phenotype and ultimately increase sensitivity to immunotherapy in a site-specific manner.
 Citation Format: Jane Xie, Timour Baslan, Ben Stanger. Tissue-of-origin determines immune cell heterogeneity and response to therapy via unique tumor cell-intrinsic factors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A046.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/882907c818197419b7f2d1a08e333d371fa45d1f" target='_blank'>
              Abstract A046: Tissue-of-origin determines immune cell heterogeneity and response to therapy via unique tumor cell-intrinsic factors
              </a>
            </td>
          <td>
            Jane Xie, T. Baslan, Ben Z. Stanger
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Breast cancer cells exhibit organotropism during metastasis, showing preferential homing to certain organs such as bone, lung, liver, and brain. One potential explanation for this organotropic behavior is that cancer cells gain properties that enable thriving in certain microenvironments. Such specific metastatic traits may arise from gene regulation at the primary tumor site. Spatial genome organization plays a crucial role in oncogenic transformation and progression, but the extent to which chromosome architecture contributes to organ-specific metastatic traits is unclear. This work characterizes chromosome architecture changes associated with organotropic metastatic traits. By comparing a collection of genomic data from different subtypes of localized and lung metastatic breast cancer cells with both normal and cancerous lung cells, we find important trends of genomic reorganization. The most striking differences in 3D genome compartments segregate cell types according to their epithelial vs. mesenchymal status. This EMT compartment signature occurs at genomic regions distinct from transcription-defined EMT signatures, suggesting a separate layer of regulation. Specifically querying organotropism, we find 3D genome changes consistent with adaptations needed to survive in a new microenvironment, with lung metastatic breast cells exhibiting compartment switch signatures that shift the genome architecture to a lung cell-like conformation and brain metastatic prostate cancer cells showing compartment shifts toward a brain-like state. TCGA patient data reveals gene expression changes concordant with these organ-permissive compartment changes. These results suggest that genome architecture provides an additional level of cell fate specification informing organotropism and enabling survival at the metastatic site. SIGNIFICANCE Computational analysis of a cohort of cancer cell lines reveals 3D genome spatial compartment changes are associated with transitions in cancer cell state that favor metastasis (EMT) and enable survival in a new organ context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e4758661702fdde348d0333dad4378a173eeb39" target='_blank'>
              Rearrangement of 3D genome organization in breast cancer epithelial - mesenchymal transition and metastasis organotropism
              </a>
            </td>
          <td>
            Priyojit Das, Rebeca San Martin, Tian Hong, R. McCord
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Oncogenic fusions of EML4 and ALK occur in ∼5% of lung adenocarcinomas. More than 15 EML4-ALK variants with distinct breakpoints within EML4 have been identified, but the functional differences between these variants remain poorly understood. Here we use CRISPR/Cas9 somatic genome editing to generate autochthonous mouse models of the two most common EML4-ALK variants, V1 and V3, and show that V3 is more oncogenic than V1. By integrating these models with multiplexed genome editing, we quantify the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth in vivo and show that many tumor suppressor genes have dramatically variant-specific effects on tumorigenesis. Analysis of a novel dataset representing the largest human EML4-ALK lung cancer cohort to date identified alterations in the genomic landscape depending on the EML4-ALK variant. These findings demonstrate functional heterogeneity among EML4-ALK variants, suggesting that EML4-ALK variants behave more like distinct oncogenes than a uniform entity. More broadly, these findings highlight the dramatic impact of oncogenic fusions partner proteins on tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f320bc764c0ea759a77487f2f2744f6c0fd41b3d" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Oscar Gonzalez, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, S. Sivakumar, Saumya D. Sisoudiya, Ethan S. Sokol, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Despite the crucial role of T cell clones in the anti-tumor activity, their characterization and association with clinical outcome following immune checkpoint inhibitors (ICI) is lacking. Here we analyzed paired single-cell RNA-sequencing/T-cell receptor sequencing of 767,606 T cells from 460 samples spanning 6 cancer types. We found a robust signature of response based on expanded CD8+ clones that differentiates between responders and non-responders. Analysis of persistent clones showed transcriptional changes that are differentially induced by therapy in the different response groups, suggesting an improved reinvigoration capacity in responding patients. Moreover, a gene trajectory analysis revealed changes in the pseudo-temporal state of de-novo clones that are associated with response to therapy. Lastly, we found that clones shared between tumor and blood are more abundant in non-responders and execute distinct transcriptional programs. Overall, our results highlight differences in clonal transcriptional states that are linked to patient response, offering valuable insights into the mechanisms driving effective anti-tumor immunity. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3504c21c4fc5c6f9ee1a5a18b7688458b8f59956" target='_blank'>
              Single-cell meta-analysis of T cells reveals clonal dynamics of response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            Ofir Shorer, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Uterine carcinosarcoma (UCS) is a rare gynecologic cancer with unfavorable prognosis. Its biphasic histologic components are known to associated with outcomes. Dissecting the tumor heterogeneity within UCS is crucial to discover molecular biomarkers for patient stratification. The aim of this study was to utilize spatial profiling technology to analyze immune presentation in different histological components within UCS tumors. The study included 23 UCS patients whose tumor sections were subjected to nanoString GeoMx Digital Spatial Profiling (DSP) to profile 30 immune-related proteins in 491 segments. We identified distinct cluster-labeled subtypes showing varied immune marker expressions. The immune presentations within the carcinoma and sarcoma components showed significant differences. Markers indicating M2 macrophages (CD45+CD163+) and epithelial-mesenchymal transition (EMT) (SMA, fibronectin) were highly expressed in the sarcoma components. The immune presentation subtypes within each histological component also differed between patients with recurrence in 5 years and without recurrence over 5 years. The cluster-labeled subtype with high Neutrophil/PMN-MDSC marker expression (CD66b) was associated with longer survival, whereas the subtype with high NK cell marker expressions (CD56) was linked to poor prognosis. Our result suggested that immune features within the carcinoma component of UCS tumors would determine patient outcomes and should be considered as a strategy for patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/677b6a5b9abbf69e6cc6fe6fd37eee258299da27" target='_blank'>
              Inter- and intra-tumor heterogeneous immune presentation in primary uterine carcinosarcoma determined by digital spatial profiling
              </a>
            </td>
          <td>
            Ying-Xuan Li, Wei-Chou Lin, Ko-Chen Chen, Wen-Chun Chang, Chin-Jui Wu, Lin-Hung Wei, Ruby Yun-Ju Huang, Bor-Ching Sheu
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Adrenocortical carcinoma (ACC) is a rare endocrine malignancy with a poor prognosis and limited treatment options. Bulk genomic characterization of ACC has not yielded obvious therapeutic or immunotherapeutic targets, yet novel therapies are needed. We hypothesized that elucidating the intratumoral cellular heterogeneity by single nuclei RNA sequencing analyses would yield insights into potential therapeutic vulnerabilities of this disease. In addition to characterizing the immune cell and fibroblast landscape, our analyses of single nuclei gene expression profiles identified an adrenal cortex cell cluster exhibiting a program of replication stress and DNA damage response in primary and metastatic ACC. In vitro assessment of replication stress and DNA damage response using an ACC cell line and a series of newly-derived hormonally active patient-derived tumor organoids revealed ATR sensitivity. These findings provide novel mechanistic insight into ACC biology and suggest that an underlying dependency on ATR may be leveraged therapeutically in advanced ACC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1469f57c44a466002613ce12061fcafe073cc550" target='_blank'>
              Single Nuclei Sequencing Reveals Intratumoral Cellular Heterogeneity and Replication Stress in Adrenocortical Carcinoma
              </a>
            </td>
          <td>
            Liudmila V. Popova, Elizabeth A R Garfinkle, Daniel M. Chopyk, Jaye B. Navarro, Adithe Rivaldi, Yaoling Shu, Elena Lomonosova, J. Phay, Barbra S. Miller, Swati Sattuwar, Mary Mullen, Elaine R. Mardis, Katherine E. Miller, Priya H. Dedhia
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Gliomas are aggressive brain tumors associated with a poor prognosis. Cancer stem cells (CSCs) play a significant role in tumor recurrence and resistance to therapy. This study aimed to identify and characterize glioma stem cells (GSCs), analyze their interactions with various cell types, and develop a prognostic signature. Methods Single-cell RNA sequencing data from 44 primary glioma samples were analyzed to identify GSC populations. Spatial transcriptomics and gene regulatory network analyses were performed to investigate GSC localization and transcription factor activity. CellChat analysis was conducted to infer cell-cell communication patterns. A GSC signature (GSCS) was developed using machine learning algorithms applied to bulk RNA sequencing data from multiple cohorts. In vitro and in vivo experiments were conducted to validate the role of TUBA1C, a key gene within the signature. Results A distinct GSC population was identified, characterized by high proliferative potential and an enrichment of E2F1, E2F2, E2F7, and BRCA1 regulons. GSCs exhibited spatial proximity to myeloid-derived suppressor cells (MDSCs). CellChat analysis revealed an active MIF signaling pathway between GSCs and MDSCs. A 26-gene GSCS demonstrated superior performance compared to existing prognostic models. Knockdown of TUBA1C significantly inhibited glioma cell migration, and invasion in vitro, and reduced tumor growth in vivo. Conclusion This study offers a comprehensive characterization of GSCs and their interactions with MDSCs, while presenting a robust GSCS. The findings offer new insights into glioma biology and identify potential therapeutic targets, particularly TUBA1C, aimed at improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c2780868fec467184ab80544d5944cba2384cf" target='_blank'>
              From single-cell to spatial transcriptomics: decoding the glioma stem cell niche and its clinical implications
              </a>
            </td>
          <td>
            Lei Cao, Xu Lu, Xia Wang, Hao Wu, Xiaye Miao
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute graft-versus-host disease (aGVHD) is a significant complication of allogeneic hematopoietic stem cell transplantation (aHSCT), driven by alloreactive donor T cells in the gut. However, the roles of additional donor and host cells in this process are not fully understood. We conducted multiplexed imaging on 59 biopsies from patients with gastrointestinal GVHD and 10 healthy controls, revealing key pathological changes, including fibrosis, crypt alterations, loss of Paneth cells, accumulation of endocrine cells, and disrupted immune organization, particularly a reduction in IgA-secreting plasma cells. Interestingly, CD8T cells were enriched only in a subset of patients, while others exhibited non-canonical enrichments of macrophages and neutrophils. Post-transplantation time significantly influenced immune composition, with host cells dominating plasma and T cell compartments long after transplantation. This spatial atlas of healthy duodenum and GVHD uncovers non-canonical immune dynamics, offering insights into disease pathophysiology and potential clinical applications in GVHD and other inflammatory bowel diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69689939e79be56e05c61106cc952ea2166e2eac" target='_blank'>
              A spatial atlas of human gastro-intestinal acute GVHD reveals epithelial and immune dynamics underlying disease pathophysiology
              </a>
            </td>
          <td>
            Nofar Azulay, Idan Milo, Yuval Bussi, Raz Ben Uri, T. Haran, Michal Eldar, O. Elhanani, Yotam Harnik, Oran Yakubovsky, I. Nachmany, T. Salame, Martin Wartenberg, Philippe Bertheau, D. Michonneau, Gérard Socié, Leeat Keren
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6f6c8a044b42cc74a45de30bf8885e624a6a702" target='_blank'>
              Single-cell transcriptional atlas of tumor-associated macrophages in breast cancer
              </a>
            </td>
          <td>
            Yupeng Zhang, Fan Zhong, Lei Liu
          </td>
          <td>2024-09-04</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell techniques have convincingly demonstrated that tumor tissue usually contains multiple genetically defined cell subclones with different gene mutation sets as well as various transcriptional profiles, but the spatial heterogeneity of the microenvironment and the macrobiological characteristics of the tumor ecosystem have not been described. For the past few years, spatial multi-omics technologies have revealed the cellular interactions, microenvironment, and even systemic tumor-host interactions in the tumor ecosystem at the spatial level, which can not only improve classical therapies such as surgery, radiotherapy, and chemotherapy but also promote the development of emerging targeted therapies in immunotherapy. Here, we review some emerging spatial omics techniques in cancer research and therapeutic applications and propose prospects for their future development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef38d3cdba0bda6123b389b5084be973c8790711" target='_blank'>
              Advances in spatial multi-omics in tumors.
              </a>
            </td>
          <td>
            Junyan Wang, Ahmad Alhaskawi, Yanzhao Dong, Tu Tian, S. Abdalbary, Hui Lu
          </td>
          <td>2024-08-26</td>
          <td>Tumori</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Since the approval of the first checkpoint inhibitor, ipilimumab (α-CTLA4), there continues to be enthusiasm around leveraging the immune system to eradicate solid tumors. However, unknowns about the cellular mechanism(s) that cause some patients to durably respond while others relapse, continue to stymie the field. To support ongoing efforts elucidating cell subsets responsible for differences in patient responses, we aim to evaluate the functional heterogeneity of T-cell subsets responding to checkpoint inhibitors in mouse models of solid tumors. Single-cell RNA-seq has enabled discoveries of transcriptionally defined cell subsets, but understanding their function require inference from transcription, flow-based validation biased by pre-selection of subsets with specific markers, or reliance on surrogate markers for cell-extrinsic functions (e.g., cytotoxicity). Thus, functional heterogeneity could be obscured. To address this, we leveraged a novel technology that converges live, cellular functions with transcriptomics at the resolution of single cell-cell interactions. This technology captures tens of thousands of individual cells or defined combinations of groups of cells in bio-compatible hydrogel compartments, enabling longitudinal imaging over days to weeks. Throughout this period, functions are characterized via bright-field or immunofluorescence readouts, including antibodies, cytokine capture beads, and dyes. Cells are then lysed within compartments to generate barcoded libraries for mRNA-sequencing and gene signatures from individual cells computationally deconvoluted. With this technology, we developed a novel assay that enables longitudinal, unbiased functional characterization of individual T cells mapped to single-cell transcriptomics. Single mouse primary T cells were encapsulated with one or more MC38 target cells and imaged over time. We validated combinations of different readouts can be measured from the same T-cell, including cell-counts for proliferation, CD8/3 expression to differentiate T cells from target cells, activation markers, and / or target cell death via Sytox. Thus, single T cells can be multi-functionally evaluated over time, enabling the study of attributes inaccessible at high scale with prior technologies, including kinetics of killing or the number of target cells killed by a single T-cell. For assays combining cytotoxicity and surface receptors, we generated barcoded libraries from functionally characterized T cells for sequencing and developed an analytical pipeline that deconvolutes mRNA from single T cells mapped to their multi-functional profiles, to identify novel pathways that regulate anti-tumor functions. We will leverage this workflow to functionally characterize single T cells in tumor-bearing mice treated with checkpoint inhibitors and map the data to underlying transcriptomics. In doing so, we aim to discover functional subsets, potentially with identical transcriptional profiles, to support hypothesis generation around mechanisms governing anti-tumor efficacy of checkpoint inhibitors.
 Citation Format: Jacqueline Yee, Praveer Sharma, Yunmin Li, Isaac Thomas, Pier F Gherardini, Teresa Ai, Mostafa Ronaghi, Gary Schroth, Matthew H Spitzer. Novel workflow for characterizing T-cell functional heterogeneity in response to checkpoint inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b89db5bb53ca20d5ac128fa18040b839f5041ca9" target='_blank'>
              Abstract A025: Novel workflow for characterizing T-cell functional heterogeneity in response to checkpoint inhibitors
              </a>
            </td>
          <td>
            Jacqueline Yee, Praveer Sharma, Yunmin Li, Isaac Thomas, Pier F Gherardini, Teresa Ai, Mostafa Ronaghi, Gary Schroth, Matthew H Spitzer
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Juvenile localized and systemic scleroderma are rare autoimmune diseases which cause significant disability and morbidity in children. The mechanisms driving juvenile scleroderma remain unclear, necessitating further cellular and molecular level studies. The Visium CytAssist spatial transcriptomics (ST) platform, which preserves the spatial location of cells and simultaneously sequences the whole transcriptome, was employed to profile the histopathological slides from skin lesions of juvenile scleroderma patients. (1) Spatial domains were identified from ST data and exhibited strong concordance with the pathologist’s annotations of anatomical structures. (2) The integration of paired ST data and single-cell RNA sequencing (scRNA-seq) from the same patients validated the comparable accuracy of the two platforms and facilitated the estimation of cell type composition in ST data. (3) The pathologist-annotated immune infiltrates, such as perivascular immune infiltrates, were clearly delineated by the ST analysis, underscoring the biological relevance of the findings. This is the first study utilizing spatial transcriptomics to investigate skin lesions in juvenile scleroderma patients. The validity of the ST data was corroborated by gene expression analyses and the pathologist’s assessments. Integration with scRNA-seq data facilitated the cell type-level analysis and validation. Analyses of immune infiltrates through combined ST data and pathological review enhances our understanding of the pathogenesis of juvenile scleroderma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d114ce60551b0d377557f894744cf39236d724d1" target='_blank'>
              Spatial Transcriptomics Identifies Cellular and Molecular Characteristics of Scleroderma Skin Lesions: Pilot Study in Juvenile Scleroderma
              </a>
            </td>
          <td>
            Tianhao Liu, Deren Esencan, Claudia M. Salgado, Chongyue Zhao, Ying-Ju Lai, Theresa Hutchins, Anwesha Sanyal, Wei Chen, Kathryn S Torok
          </td>
          <td>2024-08-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdce58909f9de06504a3ce2e2bc45ebf1454164f" target='_blank'>
              Spatial single-cell protein landscape reveals vimentinhigh macrophages as immune-suppressive in the microenvironment of hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Xinyao Qiu, Tao Zhou, Shuai Li, Jianmin Wu, Jing Tang, Guosheng Ma, Shuai Yang, Ji Hu, Kaiting Wang, Siyun Shen, Hongyang Wang, Lei Chen
          </td>
          <td>2024-09-26</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The development and progression of diseases in multicellular organisms unfold within the intricate three-dimensional body environment. Thus, to comprehensively understand the molecular mechanisms governing individual development and disease progression, precise acquisition of biological data, including genome, transcriptome, proteome, metabolome, and epigenome, with single-cell resolution and spatial information within the body’s three-dimensional context, is essential. This foundational information serves as the basis for deciphering cellular and molecular mechanisms. Although single-cell multi-omics technology can provide biological information such as genome, transcriptome, proteome, metabolome, and epigenome with single-cell resolution, the sample preparation process leads to the loss of spatial information. Spatial multi-omics technology, however, facilitates the characterization of biological data, such as genome, transcriptome, proteome, metabolome, and epigenome in tissue samples, while retaining their spatial context. Consequently, these techniques significantly enhance our understanding of individual development and disease pathology. Currently, spatial multi-omics technology has played a vital role in elucidating various processes in tumor biology, including tumor occurrence, development, and metastasis, particularly in the realms of tumor immunity and the heterogeneity of the tumor microenvironment. Therefore, this article provides a comprehensive overview of spatial transcriptomics, spatial proteomics, and spatial metabolomics-related technologies and their application in research concerning esophageal cancer, gastric cancer, and colorectal cancer. The objective is to foster the research and implementation of spatial multi-omics technology in digestive tumor diseases. This review will provide new technical insights for molecular biology researchers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77672e163c08fe2553e07120e8f19a354c4e1b31" target='_blank'>
              The burgeoning spatial multi-omics in human gastrointestinal cancers
              </a>
            </td>
          <td>
            Weizheng Liang, Zhenpeng Zhu, Dandan Xu, Peng Wang, Fei Guo, Haoshan Xiao, Chenyang Hou, Jun Xue, Xuejun Zhi, Rensen Ran
          </td>
          <td>2024-09-13</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Many aspects of breast cancer metastasis remain poorly understood, despite its clinical importance. In this issue of the JCI, Winkler et al. have applied an elegant patient-derived xenograft (PDX) model to map the transcriptomes of single cells in matched primary tumors and lung metastases across 13 breast cancer PDX models. They identified distinct transcriptional changes associated with metastatic evolution in lowly and highly metastatic primary tumors. Furthermore, by classifying the “epithelial-mesenchymal plasticity” (EMP) state of single cells, they revealed that considerable EMP heterogeneity exists among primary and metastatic human breast cancer cells. However, the EMP profile of a tumor does not change substantially upon metastasis. These findings give an unprecedentedly detailed view into the transcriptional heterogeneity and evolution of metastatic human breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54971b7e64340d8e740cdf1de341d57f2e78ee5a" target='_blank'>
              Mapping the transcriptional evolution of human metastatic breast cancer
              </a>
            </td>
          <td>
            Melissa Q. Reeves
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is an increasingly prevalent malignancy worldwide. This study aims to understand molecular alterations associated with lymph node metastasis of OSCC in order to improve treatment strategies. We analyzed a cohort of 46 patients with primary OSCC, including 10 with lymph node metastasis and 36 without. Using a comprehensive multi-omics approach—encompassing genomic, transcriptomic, proteomic, epigenetic, single-cell, and spatial analyses—we integrated data to delineate the molecular landscape of OSCC in the context of lymph node metastasis. Our genomic analysis identified significant mutations in key genes within the MAPK, TGF-β, and WNT signaling pathways, which are essential for tumor development. The proteogenomic analysis highlighted pathways critical for lymph node dissemination and factors contributing to an immunosuppressive tumor microenvironment. Elevated levels of POSTN were found to reorganize the extracellular matrix (ECM), interact with TGF-β, disrupt cell cycle regulation, and suppress the immune response by reducing VCAM1 activity. Integrated analyses of single-cell and spatial transcriptome data revealed that cancer-associated fibroblasts (CAFs) secrete TGF-β1/2, promoting cancer cell metastasis through epithelial-mesenchymal transition (EMT). Our integrated multi-omics analysis provides a detailed understanding of molecular mechanisms driving lymph node metastasis of OSCC. These insights could lead to more precise diagnostics and targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bdd901aa91cd3ba2711b681f62cf92ddfab3b2" target='_blank'>
              Proteogenomic Characterization of Primary Oral Squamous Cell Carcinomas Unveils the Extracellular Matrix Remodeling and Immunosuppressive Microenvironment Linked with Lymph Node Metastasis
              </a>
            </td>
          <td>
            Yu Liu, Zhenyu Yang, J. Pu, Jie Zhong, U. Khoo, Yu-Xiong Su, Gao Zhang
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/134c6a39946a95594365892da7bfa505a4ad41ab" target='_blank'>
              The prognostic effect of infiltrating immune cells is shaped by proximal M2 macrophages in lung adenocarcinoma
              </a>
            </td>
          <td>
            Jian-Rong Li, Vikram R Shaw, Yupei Lin, Xiang Wang, M. Aminu, Yong Li, Jia Wu, Jianjun Zhang, Christopher I Amos, Chao Cheng
          </td>
          <td>2024-09-04</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ab5a7fdae99ad93f99247468e27d3a8b9633164" target='_blank'>
              Spatiotemporal modeling reveals high-resolution invasion states in glioblastoma
              </a>
            </td>
          <td>
            Varsha Thoppey Manoharan, Aly O. Abdelkareem, Gurveer Gill, Samuel Brown, Aaron H. Gillmor, Courtney Hall, Heewon Seo, Kiran Narta, Sean Grewal, Ngoc Ha Dang, Bo Young Ahn, Kata Osz, X. Lun, Laura Mah, Franz Zemp, Douglas J Mahoney, Donna L Senger, Jennifer A. Chan, A. Morrissy
          </td>
          <td>2024-10-10</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab3d4f61a2577f3255d5a96a1f62094418b1629b" target='_blank'>
              A cell state-specific metabolic vulnerability to GPX4-dependent ferroptosis in glioblastoma
              </a>
            </td>
          <td>
            Matei A. Banu, Athanassios Dovas, Michael G. Argenziano, Wenting Zhao, Colin P. Sperring, Henar Cuervo Grajal, Zhouzerui Liu, Dominique Mo Higgins, Misha Amini, B. Pereira, Ling F Ye, A. Mahajan, N. Humala, J. Furnari, P. Upadhyayula, Fereshteh Zandkarimi, Trang Tt Nguyen, Damian E Teasley, Peter B Wu, Li Hai, Charles Karan, Tyrone Dowdy, Aida Razavilar, M. Siegelin, Jan Kitajewski, Mioara Larion, Jeffrey N. Bruce, Brent R. Stockwell, Peter A. Sims, Peter D Canoll
          </td>
          <td>2024-08-27</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Cancer-associated fibroblast (CAF) composition within the same organ varies across different cancer subtypes. Distinct CAF subtypes exhibit unique features due to interactions with immune cells and the tumor microenvironment. However, data on CAF subtypes in individuals with vestibular schwannoma (VS) are lacking. Therefore, we aimed to distinguish CAF subtypes at the single-cell level, investigate how stem-like CAF characteristics influence the tumor immune microenvironment, and identify CAF subtype-specific metabolic reprogramming pathways that contribute to tumor development. Data were analyzed from three patients with VS, encompassing 33,081 single cells, one bulk transcriptome cohort, and The Cancer Genome Atlas Pan-Cancer database (RNA sequencing and clinical data). Our findings revealed that antigen-presenting CAFs are linked to substantially heightened immune activity, supported by metabolic reprogramming, which differs from tumorigenesis. High expression of the stem-like CAF gene signature correlated with poor prognosis in low-grade gliomas within the pan-cancer database. This is the first study to classify CAF subtypes in VS patients and identify a therapeutic vulnerability biomarker by developing a stem-like CAF gene signature. Personalized treatments tailored to individual patients show promise in advancing precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd8e5819e3704f5476a3b3da2ab4be193ceaa21" target='_blank'>
              Cancer-Associated Fibroblast Subtypes Reveal Distinct Gene Signatures in the Tumor Immune Microenvironment of Vestibular Schwannoma
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jung Woo Lee
          </td>
          <td>2024-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is crucial in cancer development and therapeutic response. Immunotherapy is increasingly recognized as a critical component of cancer treatment. While immunotherapies have shown efficacy in various cancers, including breast cancer, patient responses vary widely. Some patients receive significant benefits, while others experience minimal or no improvement. This disparity underscores the complexity and diversity of the immune system. In this study, we investigated the immune landscape and cell–cell communication within the TME of breast cancer through integrated analysis of bulk and single-cell RNA sequencing data. We established profiles of tumor immune infiltration that span across a broad spectrum of adaptive and innate immune cells. Our clustering analysis of immune infiltration identified three distinct patient groups: high T cell abundance, moderate infiltration, and low infiltration. Patients with low immune infiltration exhibited the poorest survival rates, while those in the moderate infiltration group showed better outcomes than those with high T cell abundance. Moreover, the high cell abundance group was associated with a greater tumor burden and higher rates of TP53 mutations, whereas the moderate infiltration group was characterized by a lower tumor burden and elevated PIK3CA mutations. Analysis of an independent single-cell RNA-seq breast cancer dataset confirmed the presence of similar infiltration patterns. Further investigation into ligand–receptor interactions within the TME unveiled significant variations in cell–cell communication patterns among these groups. Notably, we found that the signaling pathways SPP1 and EGF were exclusively active in the low immune infiltration group, suggesting their involvement in immune suppression. This work comprehensively characterizes the composition and dynamic interplay in the breast cancer TME. Our findings reveal associations between the extent of immune infiltration and clinical outcomes, providing valuable prognostic information for patient stratification. The unique mutations and signaling pathways associated with different patient groups offer insights into the mechanisms underlying diverse tumor immune infiltration and the formation of an immunosuppressive tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ff73b13094e5afc913b4266529d0100e73db7ee" target='_blank'>
              Characterizing the Tumor Microenvironment and Its Prognostic Impact in Breast Cancer
              </a>
            </td>
          <td>
            Wenjuan Zhang, Alex Lee, Amit K. Tiwari, Mary Qu Yang
          </td>
          <td>2024-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4161d5e91ced9145566548eee008e1ecc6c40ac" target='_blank'>
              Revealing the crosstalk between LOX+ fibroblast and M2 macrophage in gastric cancer by single-cell sequencing
              </a>
            </td>
          <td>
            Dapeng Chen, W. Tong, Bing Ang, Yi Bai, Wenhui Dong, Xiyue Deng, Chunjiong Wang, Yamin Zhang
          </td>
          <td>2024-09-09</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Women with obesity-driven diabetes are predisposed to more aggressive breast cancers. However, patient metabolic status does not fully inform the current standard of care. We previously identified plasma exosomes as functionally critical actors in intercellular communication and drivers of tumor progression. Here, we generated patient-derived organoids (PDOs) from breast tumor resections to model signaling within the tumor microenvironment (TME). Novel techniques and a short (1-week) culture preserved native tumor-infiltrating lymphocytes for the first time in breast tumor PDOs. After 3-day exosome treatment, we measured the impact of exosomal signaling on PDOs via single-cell RNA sequencing. Exosomes derived from Type 2 diabetic patient plasma significantly upregulated pathways associated with epithelial-to-mesenchymal transition, invasiveness, and cancer stemness, compared to non-diabetic exosome controls. Intratumoral heterogeneity and immune evasion increased in the diabetic context, consistent with enhanced tumor aggressiveness and metastatic potential of these PDOs. Our model of systemic metabolic dysregulation and perturbed transcriptional networks enhances understanding of dynamic interactions within the TME in obesity-driven diabetes and offers new insights into novel exosomal communication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/493bbab60be824313a0cfb6f098ea3a69d0e3f72" target='_blank'>
              Plasma exosomes from individuals with type 2 diabetes drive breast cancer aggression in patient-derived organoids
              </a>
            </td>
          <td>
            Christina S. Ennis, Michael Seen, Andrew Chen, Heejoo Kang, A. Ilinski, Kiana Mahdaviani, Naomi Y. Ko, S. Monti, Gerald V. Denis
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/890b64940fb0066cdb995ca79773b85be943b6fd" target='_blank'>
              Integrated multiomics revealed adenosine signaling predict immunotherapy response and regulate tumor ecosystem of melanoma
              </a>
            </td>
          <td>
            Yantao Xu, Poyee Lau, Xiang Chen, Shuang Zhao, Yilang He, Zixi Jiang, Xiang Chen, Guanxiong Zhang, Hong Liu
          </td>
          <td>2024-09-15</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Whether metastasis in humans can be accomplished by most primary tumor cells or requires the evolution of a specialized trait remains an open question. To evaluate whether metastases are founded by non-random subsets of primary tumor lineages requires extensive, difficult-to-implement sampling. We have realized an unusually dense multi-region sampling scheme in a cohort of 26 colorectal cancer patients with peritoneal metastases, reconstructing the evolutionary history of on average 28.8 tissue samples per patient with a microsatellite-based fingerprinting assay. To assess metastatic randomness, we evaluate inter- and intra-metastatic heterogeneity relative to the primary tumor and find that peritoneal metastases are more heterogeneous than liver metastases but less diverse than locoregional metastases. Metachronous peritoneal metastases exposed to systemic chemotherapy show significantly higher inter-lesion diversity than synchronous, untreated metastases. Projection of peritoneal metastasis origins onto a spatial map of the primary tumor reveals that they often originate at the deep-invading edge, in contrast to liver and lymph node metastases which exhibit no such preference. Furthermore, peritoneal metastases typically do not share a common subclonal origin with distant metastases in more remote organs. Synthesizing these insights into an evolutionary portrait of peritoneal metastases, we conclude that the peritoneal-metastatic process imposes milder selective pressures onto disseminating cancer cells than the liver-metastatic process. Peritoneal metastases’ unique evolutionary features have potential implications for staging and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d18aa39a03328c1be1ca186cb7127b46d1c690b2" target='_blank'>
              A unique interplay of access and selection shapes peritoneal metastasis evolution in colorectal cancer
              </a>
            </td>
          <td>
            E. Wassenaar, A. Gorelick, Wei-Ting Hung, David M Cheek, Emre Kucukkose, I-Hsiu Lee, Martin Blohmer, Sebastian Degner, Peter Giunta, Rene MJ Wiezer, M. Raicu, I. Ubink, S. Klaasen, Nico Lansu, E. Watson, Ryan B. Corcoran, Genevieve Boland, G. Getz, Geert Jpl Kops, D. Juric, Jochen K Lennerz, D. Boerma, O. Kranenburg, Kamila Naxerova
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>195</td>
        </tr>

        <tr id="(1) Background: This study investigates the tumor immune microenvironment, focusing on immune cell distribution in lung adenocarcinoma. (2) Methods: We evaluated fifty cases of lung adenocarcinoma, and suitable areas for further studies were annotated on the histological slides. Two tumor cores per case were obtained, one from the tumor’s center and another from its periphery, and introduced into three paraffin receptor blocks for optimized processing efficiency. The 4-micrometer-thick tissue microarray sections were stained for H&E and for CD68, CD163, CD8, CD4, and PD-L1; (3) Results: Our investigation revealed significant correlations between PD-L1 expression in tumor cells and the presence of CD163+ macrophages, between CD4+ cells and CD8+, CD68+, and CD163+ cells, and also between CD8+ T cells and CD163+ cells. Additionally, while we observed some differences in cellular components and densities between the tumor center and periphery, these differences were not statistically significant. However, distinct correlations between PD-L1 and immune cells in these regions were identified, suggesting spatial heterogeneity in the immune landscape. (4) Conclusions: These results emphasize the intricate interactions between immune cells and tumor cells in lung adenocarcinoma. Understanding patient spatial immune profile could improve patient selection for immunotherapy, ensuring that those most likely to benefit are identified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebf90ba602f501858b8c8085e2f07447aa272fce" target='_blank'>
              Analyzing the Spatial Distribution of Immune Cells in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Florina Almarii, María Sajin, George Simion, Simona O. Dima, Vlad Herlea
          </td>
          <td>2024-08-30</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="
 Plasticity in tumor cells represents the ability of cancer cells to transition between different phenotypic states, including epithelial and mesenchymal states, known as the epithelial-mesenchymal transition (EMT). This transition is critical for processes such as invasion, metastasis, and resistance to therapy. Understanding the dynamics of plasticity and the extrinsic factors influencing state transitions may be crucial for improving therapeutic strategies and patient outcomes. However, deciphering the complex interactions and factors driving these state changes remains challenging. We introduce PlastiNet, a graphical attention-based network (GAT) designed to create spatially aware embeddings that enable the identification of cellular neighborhoods and the plasticity spectrum, despite gene panel limitations. PlastiNet incorporates both self-attention and neighbor-attention mechanisms in its GAT. This approach allows us to account for the influence of surrounding cells in determining the state of a target cell. We validated our model by constructing differentiation paths in healthy colon data, starting from stem cell signatures. The attention mechanisms highlighted known interactions, such as those between fibroblasts and epithelial cells. When applied to pancreatic ductal adenocarcinoma (PDAC) tumors, PlastiNet identified the SPP1-CD44 axis as a potential mediator in the classical-to-basal transition of tumor cells. To further explore the impact of CD44, particularly the isoform switch that occurs during EMT, we studied a patient-derived a PDAC cell line using single-cell RNA sequencing (scRNAseq) and Multiplexed Arrays Isoform Sequencing (Mas-Iso-Seq) to analyze cells spanning a range of the plasticity spectrum. Looping this analysis back to spatial transcriptomics data, we observed that the signature associated with CD44v8-10 or CD44s isoforms is co-localized with the SPP1 ligand, suggesting a spatially confined interaction. This finding underscores the potential of PlastiNet to generate hypotheses directly from patient-derived samples, offering new insights into the cellular mechanisms driving tumor plasticity. PlastiNet effectively constructs spatially aware embeddings that capture the dynamics of tumor cell plasticity. By identifying key interactions and extrinsic factors, this method opens new avenues for understanding and potentially targeting the mechanisms underlying tumor progression and treatment resistance.
 Citation Format: Izabella L Zamora, Milan Parikh, Samuel Wright, Lynn Bi, Bidish Patel, Dana Pe’er, David Ting, Nir Hacohen, Arnav Mehta. PlastiNet: Understanding the Epithelial - Mesenchymal Transition Through Graphical Attention in Spatial Transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C065.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b321cb8d990031a14ea0e651c0cf868b86d2e213" target='_blank'>
              Abstract C065: PlastiNet: Understanding the Epithelial - Mesenchymal Transition Through Graphical Attention in Spatial Transcriptomics
              </a>
            </td>
          <td>
            Izabella Zamora, Milan Parikh, Samuel J Wright, Lynn Bi, Bidish Kumar Patel, D. Pe’er, David T Ting, N. Hacohen, Arnav Mehta
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162a418f3caf8634ccba24bf052ce0feed7db43f" target='_blank'>
              Unraveling molecular characteristics and tumor microenvironment dynamics of neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            David Heimdörfer, N. Artamonova, Zoran Culig, Isabel Heidegger
          </td>
          <td>2024-10-16</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eadaab3d293ff876f51403f3620a7ad83b9f29" target='_blank'>
              Spatial transcriptomic analysis of primary and metastatic pancreatic cancers highlights tumor microenvironmental heterogeneity.
              </a>
            </td>
          <td>
            A. Khaliq, Meenakshi Rajamohan, Omer Saeed, Kimia Mansouri, Asif Adil, Chi Zhang, Anita Turk, Julienne L. Carstens, Michael House, Sikander Hayat, G. Nagaraju, Sam G Pappas, Y. A. Wang, N. Zyromski, Mateusz Opyrchal, Kelvin P Lee, Heather M. O’Hagan, Bassel El Rayes, Ashiq Masood
          </td>
          <td>2024-09-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc9bc6fb64c943c109f90f687d279d53c1cce3e" target='_blank'>
              Integrating microarray-based spatial transcriptomics and RNA-seq reveals tissue architecture in colorectal cancer
              </a>
            </td>
          <td>
            Zheng Li, Xiao-Jie Zhang, Chongyuan Sun, Zefeng Li, He Fei, Dongbing Zhao
          </td>
          <td>2024-09-17</td>
          <td>J. Big Data</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88e014c815070452f107fe9ff31d1f8aa2306a82" target='_blank'>
              Molecular and functional landscape of malignant serous effusions for precision oncology
              </a>
            </td>
          <td>
            R. Wegmann, L. Bankel, Y. Festl, Kate Lau, Sohyon Lee, F. Arnold, V. Cappelletti, Aaron Fehr, P. Picotti, Konstantin J Dedes, Daniel Franzen, D. Lenggenhager, Peter K. Bode, M. Zoche, H. Moch, Christian Britschgi, B. Snijder
          </td>
          <td>2024-10-02</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Somatic mutations inactivating TET2 are among the most common drivers of clonal hematopoiesis (CH). While TET2 inactivation is associated with monocyte-derived inflammation and improved chimeric antigen-receptor-T cell function, its impact on immunotherapy response is unknown. In our mouse model, hematopoietic Tet2 mutation enhanced immune checkpoint blockade (ICB) response. Enhanced ICB response with Tet2 mutation required phagocytes, CD4 and CD8 T cells. Mechanistically, in Tet2-mutant tumor-infiltrating leukocytes (TILs), ICB preferentially induced anti-tumor states and restricted cell states linked to tumor progression. Tet2-mutant monocytes activated costimulatory programs, while Tet2-mutant T cells showed enhanced T cell memory signatures, lesser exhaustion and decreased regulatory phenotype. Our murine data was clinically relevant, since we found that melanomas from patients with TET2 driver mutation-CH (TET2-CH) showed enhanced immune infiltration, T cell activation, and T cell memory programs. In melanoma patients treated with ICB, TET2-CH was associated with 6-fold greater odds of clinical benefit. Collectively, our data establishes that hematopoietic Tet2 inactivation primes leukocytes for anti-tumor states associated with immunotherapy response and provides a potential biomarker for personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3e6160e702c2224ea8ff0222f9be539e313873" target='_blank'>
              Hematopoietic Tet2 inactivation enhances the response to checkpoint blockade immunotherapy
              </a>
            </td>
          <td>
            R. Vanner, Suraj Bansal, Marco M. Buttigeig, Andy G. X. Zeng, Vincent Rondeau, Darryl Y. Chan, Michelle A. Chan-Seng-Yue, Liqing Jin, Jessica Mcleod, E. Donato, Patrick Stelmach, C. Vlasschaert, Yitong Yang, Aarushi Gupta, Sofia Genta, Enrique Sanz Garcia, LI Shlush, Mauricio Ribeiro, Marcus O Butler, Sagi Abelson, Mark D. Minden, Steven M. Chan, Michael J. Rauh, A. Trumpp, John E. Dick
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Cell morphologies represent a phenotypic integration of intrinsic cellular processes, changes in cell state, and interactions with neighboring cells. While analyses of large cell line collections such as the Cancer Dependency Map (DepMap) have greatly enhanced discovery of novel molecular biomarkers and therapeutic targets, morphological analysis of these models has not yet been performed. Hence, there is a rich opportunity to discover novel regulators and functional properties of cell morphologies by linking image-based profiling to -omics based assays. Here, we studied the transcriptional and functional properties of 16 human pancreatic cancer cell lines, associations with distinct cell morphologies, and treatment-induced remodeling. At baseline, we observed extensive morphological heterogeneity across and within cell lines, as well as differences in multicellular organization. We categorized cell line morphologies into epithelial, mesenchymal, and spheroid, and organizational patterns into tightly aggregated, multilayered, and dispersed. We performed differential expression analysis across morphological subtypes utilizing DepMap transcriptomics data and queried functional correlates including CRISPR dependency, drug sensitivity, and proclivity for metastasis. For example, we discovered that the mesenchymal morphology and multilayered organizational pattern [WH3] were associated with metastatic potential, while expressing reduced levels of tight junction and cell adhesion genes. We then explored treatment associated morphological changes and their potential as cost-effective biomarkers for cell-intrinsic resistance to chemotherapy (e.g., 5-FU and gemcitabine) and KRAS inhibitors (e.g., MRTX1133 and RMC- 6236). Cells were treated continuously and monitored with live cell imaging (Incucyte SX5) followed by Cell Painting at the three-day endpoint. We observed conserved morphological responses to therapy across cell lines including a shift towards spindle-like cell shapes with neurite-like projections in chemotherapy-treated conditions, and a cell bloating phenomena characterized by an increase in cell area in KRAS inhibitor treated conditions. To directly link transcriptional state to morphology, we performed 1000-plex spatial molecular imaging (Nanostring CosMx) at subcellular resolution to pools of cell lines, demonstrating that morphological diversity within cell lines corresponded to distinct transcriptional states. In conclusion, this study highlights the potential of harnessing cell morphological information in a rapid, cost-effective phenotyping assay to aid precision oncology efforts leveraging patient- derived in vitro models.
 Citation Format: Dennis Gong, William L Hwang. Morphological diversity predicts functional traits in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B076.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ee8c48f6e344128f81cda79e9fefc7cbd605a4f" target='_blank'>
              Abstract B076: Morphological diversity predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, W. L. Hwang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f337210da04933a9ae37820fdc6d7b7fe43d5d5" target='_blank'>
              CALB2 drives pancreatic cancer metastasis through inflammatory reprogramming of the tumor microenvironment
              </a>
            </td>
          <td>
            Jinxin Tao, Yani Gu, Zeyu Zhang, Guihu Weng, Yueze Liu, Jie Ren, Yanan Shi, J. Qiu, Yuanyang Wang, Dan Su, Ruobing Wang, Yifan Fu, Tao Liu, Liyuan Ye, W. Luo, Hao Chen, Gang Yang, Zhe Cao, Hua Huang, Jianchun Xiao, Bo Ren, Lei You, Taiping Zhang, Yupei Zhao
          </td>
          <td>2024-10-03</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Chimeric antigen receptor T cell is important because of its potential to treat various diseases. As deep learning continues to advance, using unsupervised methods to classify medical images has become a significant focus because collecting high-quality labeled data for medical images is labor-intensive and time-consuming. Beyond the need for accurate labeling, there is a desire to explore the underlying characteristics of the data, even when labels may be ambiguous or uncertain. To address these challenges, we present a novel approach that combines image clustering with an insightful explanation of how these clusters are formed. Our method employs a U-net combined with a clustering algorithm to segment the dataset into different groups. After clustering, we use various techniques to interpret and elucidate the results. Moreover, our paper introduces a unique dataset focused on cell data, specifically highlighting the developmental patterns of cancer cells and T cells under various experimental conditions. This dataset offers a rich source of information and presents a complex challenge for image classification due to the diversity of conditions and cell behaviors involved. Our study thoroughly compares different architectural models on this new dataset, demonstrating the superior performance of our proposed architecture. Through experimental analysis and ablation studies, we provide substantial evidence of the benefits offered by our architecture, not only in terms of accuracy but also in its ability to reveal deeper insights into the data. This work advances the field of image classification and opens new possibilities for understanding complex biological processes through computer vision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43b2222e0e50090ec7fb0ff692b055ed44f83f60" target='_blank'>
              High-throughput Image-based Clustering of CAR-T/Tumor Cocultures for Rapid and Facile Hit Identification
              </a>
            </td>
          <td>
            Zhi Xu, Xueqi Liu, Kirby Madden-Hennessey, Jordan Urbani, S. K. Nikkhoi, Anusuya Ramasubramanian, Kartika G. Venugopal, Qi Zhao, Eric L. Smith, Yun Fu
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acquiring detailed spatial information about intratumor microbiota in situ is challenging, which leaves 3D distributions of microbiota within entire tumors largely unexplored. Here, a modified iDISCO-CUBIC tissue clearing and D-amino acid microbiome labeling-based (miCDaL) strategy are proposed, that integrates microbiota in situ labeling, tissue clearing, and whole-mount tissue imaging to enable 3D visualization of indigenous intratumor microbiota. Leveraging whole-mount spatial resolution and centimeter-scale imaging depth, the 3D biogeography of microbiota is successfully charted across various tumors at different developmental stages, providing quantitative spatial insights in relation to host tumors. By incorporating an immunostaining protocol, 3D imaging of the immunologic microenvironment is achieved in both murine and human mammary tumors that is previously assumed to be bacteria-free. Notably, immune infiltrates, including T cells and NK cells, and tertiary lymphoid structures are conspicuously absent in bacteria-colonized regions. This 3D imaging strategy for mapping Indigenous intratumor microbiota offers valuable insights into host-microbiota interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6fdb761437877e43564b6889311f1059c2228cb" target='_blank'>
              Whole-Tumor Clearing and Imaging of Intratumor Microbiota in Three Dimensions with miCDaL Strategy.
              </a>
            </td>
          <td>
            Yuezhou Wang, Zile Jiang, Kai Zhang, Huimin Tang, Guimei Wang, Jinshan Gao, Guanghui He, Baoyue Liang, Li Li, Chaoyong Yang, Xianming Deng
          </td>
          <td>2024-10-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is one of the most lethal cancers, with a 5-year survival rate of only 13%. PDAC develops from precursor lesions, most commonly Pancreatic Intraepithelial Neoplasia (PanIN). Studying PanIN in human patients is difficult due to its microscopic nature and the lack of indication for sampling the pancreas in the absence of disease. In collaboration with Gift of Life - Michigan, a local organ and tissue donor organization, we obtained over 80 pancreata for research. Surprisingly, we found PanIN in the vast majority of the organs, regardless of the donor’s age; the prevalence of PanIN compared to the relatively low incidence of PDAC indicates that most lesions are destined not to progress. To gather insight into factors that might distinguish lesions designed to progress from lesions that are likely to remain dormant, we sought to characterize both epithelial cells and their supporting microenvironment. Using spatial transcriptomics, we determined that PanIN epithelial cells are transcriptionaally similar to pancreatic cancer; further, limited genetic characterization revealed that they contain similar oncogenic KRAS mutations as pancreatic malignancy. We then sought to determine whether factors in the microenvironment distinguish sporadic PanIN from those associated with overt malignancy. We used 10x Genomic CytAssist Visium technology and profiled an initial batch of 7 tissue sections from donor pancreata and an equal number from tumors. We employed the BayesPrism algorithm to deconvolute the Visium data (which lacked single-cell resolution), using our single-cell RNAseq atlas to define stroma- and epithelial-specific expression profiles. We then segmented the tissue into epithelial and stromal compartments. We further identified various stromal subpopulations, with macrophages and fibroblasts being the most abundant cell types. We defined subpopulations of epithelial cells, fibroblasts, and macrophages through unsupervised clustering. In both donor samples and tumors, we identified heterogeneous populations of stromal cells present in specific geographic patterns, with PanIN-adjacent stroma clearly distinct from stroma surrounding healthy ducts. Cell-cell neighborhood analysis revealed a distinct geographic arrangement of epithelial and stromal compartments, indicating potential communication patterns. Independent unsupervised statistical analysis using Non-negative Matrix Factorization (NMF) using Coordinated Gene Association in Pattern Sets (CoGAPS) R package also revealed similar patterns. Our next steps involve validating gene expression signatures that define each subpopulation and identifying tumor-specific or PanIN-specific genes. Ultimately, we aim to investigate interactions between epithelial cells, fibroblasts, and macrophages to identify potential tumor-promoting or tumor-restraining pathways.
 Citation Format: Ahmed M. Elhossiny, Jude Okoye, Imade Williams, Yaqing Zhang, Hannah Watkoske, Atul Deshpande, Elana Fertig, Arvind Rao, Eileen S. Carpenter, Timothy L. Frankel, Marina Pasca di Magliano. Spatial transcriptomics reveals heterogeneity of epithelial and stromal compartments in healthy and tumor pancreas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A046.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f587649d06619c2f34a308f750ab4e8707c4ec42" target='_blank'>
              Abstract A046: Spatial transcriptomics reveals heterogeneity of epithelial and stromal compartments in healthy and tumor pancreas
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, Jude Okoye, Imade Williams, Yaqing Zhang, Hannah R Watkoske, Atul Deshpande, E. Fertig, Arvind Rao, Eileen S Carpenter, Timothy L. Frankel, M. Pasca di Magliano
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Background: Hepatoblastoma (HB) and hepatocellular carcinoma (HCC) are the most common malignant hepatocellular tumors seen in children. Patients with vascular invasion and metastasis generally have poor outcomes. The aim of this study was to delineate the unique cellular changes that drive dissemination by performing comprehensive molecular and phenotypic profiling of samples representing disseminating cells, including primary circulating tumor cells (CTCs) from patients and orthotopic patient-derived xenograt (PDX) mouse models and cell lines grown in vitro in the presence of other cells that represent the tumor microenvironment (TME). Methods: With primary patient whole blood samples, we used a RosetteSep CD45 Depletion Cocktail (Stem Cell Technologies) to enrich CTCs. We validated that these cells were CTCs by staining for a panel of validated markers, including indocyanine green (ICG), Glypican-3 (GPC3), and DAPI. We then performed single cell RNA sequencing (scRNA-seq, 10x Genomics) to comprehensively analyze gene expression of these cells. To understand tumor dissemination mechanisms influenced by the TME, we grew HepT1 HB cells in the presence of supernatant from human umbilical vein endothelial cells (HUVECs). Specifically, HUVEC and HepT1 cells were cultured in separate plates in a combination of endothelial cell medium and minimal essential medium in a 1:1 ratio. The supernatant from each plate was then added to HepT1 cells, and we examined resulting transcriptomic changes with bulk RNA-seq and phenotypic alterations with proliferation (CCK8), scratch (Incucyte), and invasion (Boyden chamber) assays. We also worked to develop a pipeline to establish stable cell lines with primary patient- and murine-derived CTCs. Results: Using our ICG/GPC3/DAPI panel, we showed that the cells isolated from patient whole blood samples were CTCs. scRNA-seq analyses of these samples revealed that there was upregulation of key pathways in CTCs, including NRF2 activity, compared to low-risk primary HB tumors. Factors secreted by HUVECs influenced HepT1 cell phenotypes and gene expression. Specifically, we showed significantly higher migration and faster wound closure when HepT1 cells were grown in the presence of secreted factors from HUVECs, compared to control HepT1 cells. Conclusions: This study provides a comprehensive transcriptomic landscape of pediatric liver cancer CTCs. This work also underscores the importance of the interactions between endothelial cells and tumor cells in the initiation of metastasis. Taken together, this work builds a strong foundation for future studies elucidating the specific mechanisms of how pediatric liver tumor cells successfully disseminate with the overall goal of developing novel therapeutic regimens that target these mechanisms.
 Citation Format: Priyanka Rao, Andres Espinoza, Roma Patel, Mohammad J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield. Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B072.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165a476b6bcad6280ce2606b1f9945d425fc4ed9" target='_blank'>
              Abstract B072: Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer
              </a>
            </td>
          <td>
            Priyanka Rao, Andres F. Espinoza, R. Patel, M. J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Drug resistance remains a major clinical challenge in the treatment of colorectal cancer (CRC) with conventional chemotherapy. Analyzing changes within tumor cells and tumor microenvironment (TME) after treatment and in metastases is essential to understanding how resistance develops. In this study, we analyzed scRNA-seq data from 56 CRCs including treatment-naïve tumors and tumors treated with standard chemotherapy with the known response status (18 responders and 6 progressors). In our cohort, primary left-sided CRCs were associated with metastatic potential mesenchymal phenotype and with depleted B cells. In the post-treatment CRC, there was a high prevalence of dendritic cells (DC) in the TME in the response group. The DC-derived signature was associated with better survival in a large CRC cohort from the TCGA. In progressors there was an enrichment of pericyte-like fibroblasts, which appeared to be associated with poor survival in a CRC-TCGA cohort. Progressors also showed elevated fractions of exhausted CD8+ T memory cells suggesting a pro-inflammatory TME. In tumor cells of progressors group, we identified specific expression of chemo-protective markers MTRNR2L1 and CDX1; and their co-expression with stemness-related immune-checkpoint CD24. In summary, scRNA-seq provides a valuable information for the discovery of prognostic markers, and reveals distinct features potentially underlying response to chemotherapy or disease progression in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d907795ada63036c087467c94e2007531b414d" target='_blank'>
              Single-cell Transcriptome Profiling of Post-treatment and Treatment-naïve Colorectal Cancer: Insights into Putative Mechanisms of Chemoresistance
              </a>
            </td>
          <td>
            Grigory A. Puzanov, Clémence Astier, Andrey A. Yurchenko, Gerome Jules-Clement, Fabrice André, Aurélien Marabelle, A. Hollebecque, Sergey Nikolaev
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44ac4365f3d537cc727149bd2f0c2b4a261aed5" target='_blank'>
              The neuroendocrine transition in prostate cancer is dynamic and dependent on ASCL1.
              </a>
            </td>
          <td>
            Rodrigo Romero, Tinyi Chu, Tania J González Robles, Perianne Smith, Yubin Xie, Harmanpreet Kaur, Sara Yoder, Huiyong Zhao, Chenyi Mao, Wenfei Kang, Maria V. Pulina, K. Lawrence, Anuradha Gopalan, S. Zaidi, K. Yoo, Jungmin Choi, Ning Fan, Olivia Gerstner, W. Karthaus, E. deStanchina, Kelly V. Ruggles, P. Westcott, R. Chaligné, D. Pe’er, C. Sawyers
          </td>
          <td>2024-10-11</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="T cell heterogeneity is a cornerstone of the adaptive immune response, with CD8+ T cells playing a pivotal role in mediating cytotoxic responses to pathogens and tumors. While traditional studies have focused on phenotypic and functional diversity through surface marker expression and functional assays, the significance of biophysical properties in delineating T cell subpopulations remains underexplored. Here, we show that resting T cells exhibit a bimodal distribution in buoyant mass, which is not captured by surface marker based canonical immunophenotypes or physical parameters such as volume, density, dry mass, or light scattering. Functionally, we demonstrate that T cells with lower buoyant mass exhibit reduced proliferation, tend towards effector differentiation, and become more exhausted in the presence of innate tumor-secreted factors. Conversely, T cells with higher buoyant mass show greater memory potential and resistance to exhaustion. Our study not only provides insights into T cell biology but also suggests buoyant mass as a promising label-free biomarker to potentiate T cell-based therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5377b1937748d2da8a9f6d4a6c6c5712c248ef8" target='_blank'>
              Bimodal cell mass distribution separates CD8+ T cells into two distinct types with divergent differentiation dynamics
              </a>
            </td>
          <td>
            Jiaquan Yu, Ye Zhang, Sarah M. Duquette, Gianfranco L. Yee, Teresa Dinter, Thomas R. Usherwood, Weida Wu, Teemu P. Miettinen, Stefani Spranger, S. Manalis
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="The tumor microenvironment (TME) comprises a diverse array of cells, both cancerous and non-cancerous, including stromal cells and immune cells. Complex interactions among these cells play a central role in driving cancer progression, impacting critical aspects such as tumor initiation, growth, invasion, response to therapy, and the development of drug resistance. While targeting the TME has emerged as a promising therapeutic strategy, there is a critical need for innovative approaches that accurately replicate its complex cellular and non-cellular interactions; the goal being to develop targeted, personalized therapies that can effectively elicit anti-cancer responses in patients. Microfluidic systems present notable advantages over conventional in vitro 2D co-culture models and in vivo animal models, as they more accurately mimic crucial features of the TME and enable precise, controlled examination of the dynamic interactions among multiple human cell types at any time point. Combining these models with next-generation technologies, such as bioprinting, single cell sequencing and real-time biosensing, is a crucial next step in the advancement of microfluidic models. This review aims to emphasize the importance of this integrated approach to further our understanding of the TME by showcasing current microfluidic model systems that integrate next-generation technologies to dissect cellular intra-tumoral interactions across different tumor types. Carefully unraveling the complexity of the TME by leveraging next generation technologies will be pivotal for developing targeted therapies that can effectively enhance robust anti-tumoral responses in patients and address the limitations of current treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee78b792b563d1d9bcfce83a027e1da609d44dc5" target='_blank'>
              Improving tumor microenvironment assessment in chip systems through next-generation technology integration
              </a>
            </td>
          <td>
            Daniela Gaebler, Stephanie J. Hachey, Christopher C. W. Hughes
          </td>
          <td>2024-09-25</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Pancreatic cancer remains one of the most deadly cancers, with over 450,000 fatalities in 2022 alone. The most lethal variation of this cancer is Pancreatic Ductal Adenocarcinoma (PDAC), with a 5-year survival rate of under 10%. Combination chemotherapy has provided major advances in this disease but it remains challenging for many reasons including the immunosuppressive nature of the tumor microenvironment (TME). PDAC has a highly desmoplastic microenvironment that suppresses antitumor immunity and promotes tumor cell invasion, metastasis, and resistance through epithelial to mesenchymal transition (EMT) of cancer cells. Cancer cells can undergo various forms of cell death, notably apoptosis, necrosis, autophagy, pyroptosis, and ferroptosis. These mechanisms are triggered by different cellular stressors and shape the microenvironment. Through the use of 15 Visium spatial transcriptomics samples from publicly available primary PDAC tumors and paired single-cell RNA (scRNA) data, we aim to understand how cell stress and cell death shape the tumor microenvironment (TME) and epithelial-mesenchymal transition (EMT). By leveraging information from known pathways in literature, we will identify cellular programs related to stress, death, and EMT. We will explore intracellular correlations within single-cell data and extracellular interactions within the TME using spatial data to uncover how different forms of cell death and stress signatures influence the TME and EMT spectrum. The initial phase involves identifying the types of stress that lead to various forms of cell death intracellularly and validating these gene signatures. Subsequently, we will apply these validated signatures to the spatial data to correlate each score and search for significant overlapping programs. This will extend beyond stress and death, incorporating TME signatures corresponding to different cell types and the EMT spectrum. By understanding the signals that promote the death of various cell populations, we aim to identify combination treatments that target both ends of the EMT spectrum. Additionally, this research will provide insights into the inflammatory impact of cell death on the TME. Our approach aims to inform the development of therapies that disrupt resistance mechanisms, ultimately improving treatment outcomes for patients with PDAC. This research can potentially reveal new therapeutic targets and strategies, enhancing our ability to combat this deadly disease.
 Citation Format: Rahul Bansal, Izabella Zamora, Samuel Wright, Nir Hacohen, Arnav Mehta. Influence of Cell Death and Stress on the Tumor Microenvironment and EMT in Pancreatic Ductal Adenocarcinoma Through Single Cell and Spatial Transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba425dbbc0b9c870572a7ccded079a29b6e67232" target='_blank'>
              Abstract C074: Influence of Cell Death and Stress on the Tumor Microenvironment and EMT in Pancreatic Ductal Adenocarcinoma Through Single Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Rahul Bansal, Izabella Zamora, Samuel J Wright, N. Hacohen, Arnav Mehta
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Increasing evidence suggests the tumor microenvironment (TME) governs solid tumor response to immune checkpoint inhibition (ICI). Decoding the relationship between the cell compositional diversity of the tumor microenvironment (TME) and the therapeutic outcomes has been a longstanding problem in solid tumor research. In this work, we develop a systems-theoretic formalism to decipher the key mechanisms of growth, proliferation, immune evasion, and drug resistance that run common across solid tumors in the context of Immune Checkpoint Inhibitors (ICI). We reconstructed a core TME network in common across most solid tumors, containing multiple tumor and non-tumor cell types in distinct functional states, and molecular agents mediating cellular signaling and cell-cell interactions. Our analysis shows that the core TME network is sufficient to yield a multiplicity of attractors corresponding to clinically observed TME subtypes namely, immune or fibro dominated, immune or fibro desert, and immune and fibro deficient. Importantly, the reachability around the pre-ICI attractors governs the response to ICI explaining the TME subtype-specific therapy outcomes. We analyzed the attractor transition network to identify subtype-specific combination therapies that can drive unresponsive TME to a responsive subtype. We derived mathematical conditions relating TME balances to determine the limits of the efficacy of combination therapies. Our results hold for a large class of smooth biochemical kinetics with monotone and bounded interactions and (semi-)concave proliferation rules. The analytical findings have been verified with extensive simulation of different TME sub-types. Overall, we propose a generalized systems formalism that accounts for the TME properties governing ICI response and can aid in designing intervention strategies for improved tumor prognosis. Significance Statement Non-responsive therapy outcomes have been a persistent problem in cancer treatment. Predicting the possibility of non-responsiveness to a particular therapy from the pre-treatment composition of the tumor microenvironment (TME) aids in designing appropriate combination treatment strategies toward an improved prognosis. The present work develops a systems-theoretic formalism that aims to unfold the mechanisms behind solid tumors’ growth, non-responsiveness, and recurrence. Unlike a single model, the proposed formalism does not assume any particular kinetics, barring some minimal assumptions. This enables us to explain some of the relevant observations made by the recent experimental studies. Finally, the closed-form conditions obtained for responsivity and recurrence can also guide novel therapeutic strategies that may be able to restore responsiveness to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657fb6c08b911bd43c20c8cf26d56275f3634fc0" target='_blank'>
              From tumor microenvironment to immuno-therapeutic outcomes for solid tumors: A systems theoretic approach
              </a>
            </td>
          <td>
            Priyan Bhattacharya, R. Vadigepalli
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8fc05c94d1bb030c65261bd89103d4e78395190" target='_blank'>
              Caspase-1-dependent spatiality in triple-negative breast cancer and response to immunotherapy
              </a>
            </td>
          <td>
            Weiyue Zheng, Wanda Marini, K. Murakami, Valentin Sotov, Marcus O Butler, Chiara Gorrini, P. Ohashi, M. Reedijk
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee31a2e42e286fd7d20df7672d92a049ecf6880" target='_blank'>
              An integrated single-cell reference atlas of the human endometrium
              </a>
            </td>
          <td>
            Magda Marečková, L. Garcia-Alonso, Marie Moullet, Valentina Lorenzi, R. Petryszak, Carmen Sancho-Serra, Agnes Oszlanczi, Cecilia Icoresi Mazzeo, Frederick C. K. Wong, Iva Kelava, Sophie Hoffman, M. Krassowski, K. Garbutt, Kezia Gaitskell, S. Yancheva, E. Woon, V. Male, I. Granne, Karin Hellner, Krishnaa T. Mahbubani, K. Saeb‐Parsy, M. Lotfollahi, E. Prigmore, J. Southcombe, Rebecca Dragovic, Christian M. Becker, K. Zondervan, R. Vento-Tormo
          </td>
          <td>2024-08-28</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Cell biology and genetic analysis of intracellular, intercellular and inter-organ interaction studies in animal models are key for understanding development, physiology, and disease. The MARCM technique can emulate tumor development by simultaneous clonal tumor suppressor loss-of-function generation coupled with GAL4-UAS-driven oncogene and marker expression, but the utility is limited for studying tumor-host interactions due to genetic constraints. To overcome this, we introduce EyaHOST, a novel system that replaces MARCM with the QF2-QUAS binary gene expression system under the eya promoter control, unleashing the fly community genome-wide GAL4-UAS driven tools to manipulate any host cells or tissue at scale. EyaHOST generates epithelial clones in the eye epithelium similar to MARCM. EyaHOST-driven RasV12 oncogene overexpression coupled with scribble tumor suppressor knockdown recapitulates key cancer features, including systemic catabolic switching and organ wasting. We demonstrate effective tissue-specific manipulation of host compartments such as neighbouring epithelial cells, immune cells, fat body, and muscle using fly avatars with tissue-specific GAL4 drivers. Organ-specific inhibition of autophagy or stimulation of growth-signaling through PTEN knockdown in fat body or muscle prevents cachexia-like wasting. Additionally, we show that RasV12, scribRNAi tumors induce caspase-driven apoptosis in the epithelial microenvironment. Inhibition of apoptosis by p35 expression in the microenvironment promotes tumor growth. EyaHOST offers a versatile modular platform for dissecting tumor-host interactions and other mechanisms involving intercellular and inter-organ communication in Drosophila. Highlights *eyes absent, eye disc-specific enhancer drives clonal KD recombinase flip-out activated QF2 expression in the larval eye epithelium for simultaneous QUAS-driven gain and loss-of-function analysis of gene function. *Clones are visualized by QUAS-tagBFP or QUAS-eGFP facilitating analysis of existing fluorescent reporters. *The GAL4-UAS system and existing genome-wide genetic tools are released to independently manipulate any cell population in the animal for cell biology, intercellular or inter-organ analysis for developmental, physiological, or disease model analysis. *Fly avatars for tumor-host interaction studies with multiple organs allow live monitoring and manipulation of tumors and organs in translucent larva.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6d5a397edb7ccb05aea0ec451c7e0881eefa9e" target='_blank'>
              EyaHOST, a modular genetic system for investigation of intercellular and tumor-host interactions in Drosophila melanogaster
              </a>
            </td>
          <td>
            José Teles-Reis, Ashish Jain, Dan Liu, Rojyar Khezri, Sofia Micheli, Alicia Alfonso Gomez, Caroline Dillard, T. Rusten
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 Immunotherapies such as immune checkpoint inhibitors (ICIs) have greatly improved patient outcomes in multiple tumor types, but ICIs have failed to achieve responses in many patients. Meta-analyses of patient cohorts treated with ICIs have observed that worse clinical outcomes are strongly correlated with high levels of intratumoral genetic heterogeneity (ITH). ITH occurs when tumor cells mutate as they divide, typically due to oxidative stress or DNA damage from rapid proliferation and continued carcinogenic insults, generating tumor subclones. However, the mechanisms by which ITH impairs the anti-tumor immune response are not fully understood. We propose a novel mechanism for how ITH in highly T cell-infiltrated inflamed tumors can adversely affect T cell behavior. We hypothesize that strongly pro-inflammatory tumor subclones secreting robust pro-inflammatory chemokine profiles can dominate T cell trafficking within the tumor, resulting in a weakened immune response against other tumor subclones. Using a murine squamous cell skin carcinoma (SCC) model, we have demonstrated that in immune-inflamed tumors, the spatial distribution and behavior of tumor-infiltrating T cells is influenced by ITH. A strongly pro-inflammatory tumor line expressing an endogenous RFP fluorophore was mixed with a weakly pro-inflammatory tumor line expressing an endogenous YFP fluorophore at a 1:1 ratio. The resulting tumors contained distinct RFP+ and YFP+ regions. These were analyzed by immunofluorescence (IF) and by manually dissecting the tumors and analyzing each region by flow cytometry. Flow cytometry analysis determined that the strongly pro-inflammatory RFP+ regions contained significantly more total T cells and CD4+ T cells than YFP+ regions. Furthermore, FoxP3- “conventional” CD4+ T cells were significantly more likely to be “terminal effector-like” CD44+CD62L- in RFP+ regions, and significantly more likely to be “naïve-like” CD62L+CD44- in YFP+ regions. IF analysis confirmed the preferential distribution of CD4+ T cells in RFP+ regions, and observed that this spatial pattern was evident as early as 11 days after subcutaneous injection. Notably, this localization trend did not occur with CD8+ T cells, which were evenly dispersed throughout the tumors. ELISA analysis of lysates of both tumor cell lines determined that the pro-inflammatory RFP+ cell line secretes significantly higher levels of the chemokine CCL5, which has been observed by other studies to selectively mediate CD4+ T cell chemotaxis. These results demonstrate that intratumoral CD4+ trafficking is shaped by the spatial distribution of specific tumor subclones, and suggest tumor cell chemokine signaling may directly mediate T cell spatial localization and behavior within tumors.
 Citation Format: Robert Letchworth, Savannah Hughes, Abigail Keku, Melissa Reeves. Pro-inflammatory tumor cells drive CD4+ T cell localization within heterogeneous tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aeeff4ff2175551d0a146978acdaeb3ec633ea2b" target='_blank'>
              Abstract A033: Pro-inflammatory tumor cells drive CD4+ T cell localization within heterogeneous tumors
              </a>
            </td>
          <td>
            Robert Letchworth, Savannah Hughes, Abigail Keku, Melissa Q Reeves
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69fde8318f7a677932512a6597cf3b791cc10829" target='_blank'>
              B-cell intrinsic RANK signaling cooperates with TCL1 to induce lineage-dependent B-cell transformation
              </a>
            </td>
          <td>
            Lisa Pfeuffer, Viola Siegert, Julia Frede, Leonie Rieger, Riccardo Trozzo, N. de Andrade Krätzig, Sandra Ring, Shamim Sarhadi, Nicole Beck, Stefan Niedermeier, Mar Abril-Gil, Mohamed Elbahloul, Marianne Remke, Katja Steiger, Ruth Eichner, Julia Jellusova, Roland Rad, Florian Bassermann, Christof Winter, J. Ruland, M. Buchner
          </td>
          <td>2024-08-28</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbcb18aaf8922f16ad19ad9eeb7b0ec8bc2789d" target='_blank'>
              Serial single-cell RNA sequencing unveils drug resistance and metastatic traits in stage IV breast cancer
              </a>
            </td>
          <td>
            K. Otsuji, Yoko Takahashi, Tomo Osako, Takayuki Kobayashi, T. Takano, S. Saeki, Liying Yang, Satoko Baba, Kohei Kumegawa, Hiromu Suzuki, Tetsuo Noda, Kengo Takeuchi, Shinji Ohno, T. Ueno, Reo Maruyama
          </td>
          <td>2024-10-03</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Hepatocellular carcinoma (HCC) is a typical highly heterogeneous solid tumor with high morbidity and mortality worldwide, especially in China; however, the immune microenvironment of HCC has not been clarified so far. Here, we employed single‐cell RNA sequencing (scRNA‐seq) on diethylnitrosamine (DEN)‐induced mouse HCC model to dissect the immune cell dynamics during tumorigenesis. Our findings reveal distinct immune profiles in both precancerous and cancerous lesions, indicating early tumor‐associated immunological alterations. Notably, specific T and B cell subpopulations are preferentially enriched in the HCC tumor microenvironment (TME). Furthermore, we identified a subpopulation of naïve B cells with high CD83 expression, correlating with improved prognosis in human HCC. These signature genes were validated in The Cancer Genome Atlas HCC RNA‐seq dataset. Moreover, cell interaction analysis revealed that subpopulations of B cells in both mouse and human samples are activated and may potentially contribute to oncogenic processes. In summary, our study provides insights into the dynamic immune microenvironment and cellular networks in HCC pathogenesis, with a specific emphasis on naïve B cells. These findings emphasize the significance of targeting TME in HCC patients to prevent HCC pathological progression, which may give a new perspective on the therapeutics for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76c814af70ac2f24272bcadf22a4dd9e85ffd9ec" target='_blank'>
              Leveraging single‐cell RNA‐seq for uncovering naïve B cells associated with better prognosis of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Qingjia Sun, Rui Gao, Yingxin Lin, Xianchao Zhou, Tao Wang, Jian He
          </td>
          <td>2024-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Ethnic disparities in breast cancer (BC) outcomes have been reported, with lower survival rates among black women compared to white. Additionally, the unfavorable triple negative (TNBC) sub-type is more prevalent in black women and within this population, risk of mortality is again higher compared to white women. Biological contributors to this disparity are less clearly understood than socio-economic factors. Recent studies recognize different breast tumour immune microenvironments between black and white BC patients, necessitating further research into ethnicity-related differences in anti-tumor immune responses. Such responses begin in regional lymph nodes (LNs) with tumor neoantigen presentation. Here, we generate single-cell spatial maps of involved TNBC LN cores in treatment naïve black and white women. To guide design of a phenotyping panel for LNs, we profiled blood samples (n=95) using high-dimensional flow cytometry from newly diagnosed black and white BC patients. We found differential skewing of NK cell subsets, lower memory B cell (identified as IgM+CD27+IgD- & IgM+CD27+IgD+ cells) abundance and higher CD4+ T effector memory presence in black patients compared to white. Additionally, higher antigen presentation capacity was observed with increased intermediate monocytes and conventional DCs, confirmed by MS-based proteomics. Using highly multiplexed imaging with a panel of 26 markers, we performed single-cell spatial immune profiling to characterize cellular phenotypes, neighborhoods and interactions within a cohort of LN core biopsies from black patients and white patients. A total of 536,113 cells were segmented for analysis. Cell abundance analysis revealed higher CD8+ T effector memory and CD4+ T cell subsets in white patients, except for CD4+ T effector memory cells which were higher in black patients, consistent with blood profiling. Cellular neighborhood analysis revealed black patient tumor-rich regions were enriched for M2 macrophages while white patients had no such enrichment. In these white patients, a neighborhood resembling canonical follicular structures almost exclusively containing B cells was identified. Interestingly in these black patients, analogous B cell rich regions also comprised a heterogeneous array of other immune cells. Further analysis to explore B cell zones is ongoing. Preliminary interaction analysis identified immunosuppressive crosstalk in black patients with enriched Treg & CD8+ T cell and Treg & NK interactions, not observed in white patients. These initial findings suggest ethnicity-based variation in LN architectures from treatment naive patients with TNBC, supporting the hypothesis that variation in immune programming could translate into differences in pro and anti-tumor immunity, contributing to disparities in outcome amongst black and white patients.
 Citation Format: Paul R. Buckley, Melek Akay, Esme Carpenter, Rosalind Graham, Helen Kakkassery, Thanussuyah Alaguthurai, Priya Hossain, Patrycja Gazińska, Sheeba Irshad. Differential single-cell spatial immune cell profiles and immune cell interactions identified within lymph nodes metastases of black and white women diagnosed with triple negative breast cancers [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C142.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c431f2f91ac1c49d9f19dea4541265da8c85d8d6" target='_blank'>
              Abstract C142: Differential single-cell spatial immune cell profiles and immune cell interactions identified within lymph nodes metastases of black and white women diagnosed with triple negative breast cancers
              </a>
            </td>
          <td>
            Paul Buckley, Melek Akay, Esme Carpenter, Rosalind Graham, Helen Kakkassery, T. Alaguthurai, Priya Hossain, Patrycja Gazińska, S. Irshad
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Glioblastoma is the most common malignant primary brain tumor in adults with a poor survival and an unmet need for more effective therapies. Macroscopically, as observed through MR imaging and microscopy, these tumors are characterized by distinct tumor regions: a contrast-enhancing tumor core with a high cancer cell density and an infiltration zone that appears T2/FLAIR hyperintense without the uptake of contrast agent and consists of infiltrating tumor cells as well as non-malignant cells of the microenvironment. Previous genomic and transcriptomic studies revealed molecular differences between the tumor core and the infiltration zone. However, functional differences, such as drug sensitivity, between tumor regions are still largely unknown.



 To address this gap, we collected region-specific glioblastoma patient tissue samples from both the contrast-enhancing tumor core and the non-contrast-enhanced but 5-aminolevulinic acid (5-ALA) fluorescent infiltration zone directly from surgery. We utilized pharmacoscopy, a microscopy-based single-cell drug screening approach, to identify drug responses across regions and patients.



 By profiling a library of 60 drugs across regions in a cohort of 23 patients, we identified region-specific drug responses and developed a complementarity score to pinpoint drug combinations capable of targeting both the tumor core and the infiltration zone. These combination treatments exhibited highly promising anti-tumor activity across regions and patients.



 Given the high levels of functional spatial intra- and inter-tumor heterogeneity, relying on a single drug across patients may not be feasible for effective patient treatment. Thus, there is a strong rationale to administer drug combinations that not only consider regional heterogeneity but also account for patient-specific variations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fa1d38bc5a81ab129f0d7423cbbd80eb88f3cf3" target='_blank'>
              P01.13.B SPATIALLY RESOLVED FUNCTIONAL PROFILING FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Buck, S. Lee, R. Wegmann, J. Mena, F. Vasella, J. Friesen, L. Regli, E. Le Rhun, A. Zeitlberger, M. Neidert, N. Tatari, G. Hutter, M. Weller, B. Snijder, T. Weiss
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: The analysis of single-cell transcriptome profiling of tumour tissue isolates helps to identify heterogeneous tumour cells, neighbouring stromal cells and immune cells. Local metastasis of lymph nodes is the most dominant and influential biological behaviors of oral squamous cell carcinoma (OSCC) in terms of treatment prognosis. Understanding metastasis initiation and progression is important for the discovery of new treatments for OSCC and prediction of clinical responses to immunotherapy. However, the identity of metastasis-initiating cells in human OSCC remains elusive, and whether metastases are hierarchically organized is unknown. Therefore, this study was conducted to understand the cellular origins and gene expression signature of OSCC at the single-cell level. Methods: Single-cell RNA sequencing (scRNA-seq) was used to analyze cells from tissue of para-carcinoma (PCA: adjacent normal tissue not less than 2 cm from the tumour), carcinoma (CA), lymph node metastasis (LNM) from patients with OSCC and PCA and CA tissue from patients with second primary OSCC (SPOSCC) after radiotherapy of nasopharyngeal carcinoma (NPC). The cell types and their underlying functions were classified. The comparisons were then conducted between the homology and heterogeneity from cell types and both conservative and heterogeneous aspects of evolution were identified. Immunohistochemistry was performed to verify the makers of cell clusters and the expression level of novel genes. Results: A single-cell transcriptomic map of OSCC was created, including 16 clusters of PCA cells, 17 clusters of CA cells, 14 clusters of left LNM cells, and 14 clusters of right LNM cells. We also discovered two novel types of cells including CD1C-CD141-dendritic cells and CD1C+_B dendritic cells. Most of the non-cancer cells are immune cells, with two distinct clusters of T lymphocytes, B lymphocytes, CD1C-CD141-dendritic cells+ and CD1C+_B dendritic cells. We also classified cells into 15 clusters for SPOSCC after radiotherapy of NPC. Determining the upregulated expression levels of IL1RN and C15orf48 as novel markers using immunohistochemistry facilitated the correct classification of OSCC including SPOSCC after radiotherapy of NPC and the prediction of their prognosis. Conclusions: The findings provided an unprecedented and valuable view of the functional states and heterogeneity of cell populations in LNM of OSCC and SPOSCC after radiotherapy of NPC at single-cell genomic resolution. Moreover, this transcriptomic map discovered new cell types in mouth, and novel tumour cell-specific markers/oncogene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77292556c55e925fa761d1773f56e9b7ac40ff6d" target='_blank'>
              Revelation of comprehensive cell profiling of primary and metastatic tumour ecosystems in oral squamous cell carcinoma by single-cell transcriptomic analysis
              </a>
            </td>
          <td>
            Yin-han Liao, Li Chen, Bing-hua Feng, Wei Lv, Xuan-ping Huang, Hao Li, Cuiping Li
          </td>
          <td>2024-08-26</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a937fbb57d8c42791d7a325a409f71c5d3e4aa5b" target='_blank'>
              Reconstruction of cancer marker analysis with holistic anatomical precision implicates heterogeneity development during breast tumor progression
              </a>
            </td>
          <td>
            Zhicheng Ge, Jing Wang, Libing He, Meng Zhao, Yang Si, Siyuan Chang, Guoyan Zhang, Shan Cheng, Wei Ding
          </td>
          <td>2024-10-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Background Intrahepatic cholangiocarcinoma (ICC) is a disease with poor prognosis and limited therapeutic options. We investigated the tumor immune microenvironment (TIME) to identify predictors of disease outcome and to explore targets for therapeutic modulation. Methods Liver tissue samples were collected during 2008–2019 from patients (n = 139) diagnosed with ICC who underwent curative intent surgery without neoadjuvant chemotherapy. Samples from the discovery cohort (n = 86) were immunohistochemically analyzed on tissue microarrays (TMAs) for the expression of CD68, CD3, CD4, CD8, Foxp3, PD-L1, STAT1, and p-STAT1 in tumor core and stroma areas. Results were digitally analyzed using QuPath software and correlated with clinicopathological characteristics. For validation of TIME-related biomarkers, we performed multiplex imaging mass cytometry (IMC) in a validation cohort (n = 53). Results CD68+ cells were the predominant immune cell type in the TIME of ICC. CD4+high T cell density correlated with better overall survival (OS). Prediction modeling together with validation cohort confirmed relevance of CD4+ cells, PD-L1 expression by immune cells in the stroma and N-stage on overall disease outcome. In turn, IMC analyses revealed that silent CD3+CD4+ clusters inversely impacted survival. Among annotated immune cell clusters, PD-L1 was most relevantly expressed by CD4+FoxP3+ cells. A subset of tumors with high density of immune cells (“hot” cluster) correlated with PD-L1 expression and could identify a group of candidates for immune checkpoint inhibition (ICI). Ultimately, higher levels of STAT1 expression were associated with higher lymphocyte infiltration and PD-L1 expression. Conclusions These results highlight the importance of CD4+ T cells in immune response against ICC. Secondly, a subset of tumors with “hot” TIME represents potential candidates for ICI, while stimulation of STAT1 pathway could be a potential target to turn “cold” into “hot” TIME in ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/911abf7859bef4d49e76010cb034c39084622596" target='_blank'>
              Clinical prognosticators and targets in the immune microenvironment of intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Isis Lozzi, Alexander Arnold, Matthias Barone, Juliette Claire Johnson, Bruno V Sinn, Johannes Eschrich, P. Gebert, Ruonan Wang, Mengwen Hu, L. Feldbrügge, Anja Schirmeier, A. Reutzel‐Selke, Thomas Malinka, Felix Krenzien, W. Schöning, Dominik P. Modest, Johann Pratschke, I. M. Sauer, M. Felsenstein
          </td>
          <td>2024-10-01</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Even in its localized stages, pancreatic ductal adenocarcinoma (PDAC) is virtually incurable, characterized by early dissemination and disease recurrence. The liver is the most common site of metastasis, and in most cases hepatic recurrences drive survival outcomes. This underscores the critical need to target and prevent liver metastases. We leveraged a novel biobanking strategy to perform paired single-cell RNA (scRNA-seq) and TCR sequencing (scTCR-seq) to profile multisite immunity in patients with localized PDAC across the tumor, peripheral blood, and premetastatic liver. This analysis of three patients included three tumor, three blood, and nine liver (three per patient from separate segments) samples, and generated 85,748 cells for downstream analysis. We evaluated populations of tumor-expanded T cells as potential indicators of antitumoral immunity, and utilized validated computational approaches to identify tumor-reactive T cells based on key markers of dysfunction and exhaustion. Gene set enrichment analysis was performed on tumor infiltrating lymphocytes (TILs) using signatures specific to tumor-reactive CD4 and CD8 T cells (“NeoTCR4” and “NeoTCR8,” respectively), and scored these using the AUCell package in R. Overall, 153 CD4 and 22 CD8 T cells fit candidate NeoTCR4 and NeoTCR8 signature states, representing 5.6% and 0.8% of the total TIL population respectively. These nominated T cells expressed key marker genes associated with tumor specificity such as CXCL13, TIGIT, PDCD1, ENTPD1, and TOX. Thirteen of these predicted tumor-reactive T cell clones identified within the tumor were also detected within our liver samples, suggestive of a potential antitumoral immune response in the premetastatic liver space. We next performed 10x Xenium in situ spatial transcriptomic analysis of primary tumor, normal adjacent pancreas, and premetastatic liver from two of these patients, using tissue microarrays to incorporate replicate 2mm core biopsies from each of these sites. A custom designed gene panel, guided by our scRNA-seq dataset, targeted tumor-specific mutations (e.g. KRAS G12V, G12D, G12R, and Q61) and immune related genes with a specific focus on markers of T cell activity and exhaustion. Leveraging these latter marker genes, which represent a subset of the NeoTCR4 and NeoTCR8 signatures, we identified a population of exhausted TILs within the tumor microenvironment, representing potentially tumor-reactive T cells whose interactions can now be explored at spatial resolution. Future directions are aimed at experimentally validating the tumor reactivity of the predicted neoantigen-specific TILs within our dataset and characterizing the immune neighborhoods that exist in both the tumor and liver of PDAC patients. This work establishes exciting opportunities for novel T-cell therapy in the treatment of PDAC, and we have already implemented these experimental findings in a forthcoming clinical trial investigating the applicability of this predictive signature score for the generation of TCR-engineered lymphocyte therapy in patients with PDAC.
 Citation Format: Elishama N Kanu, Ashley A Fletcher, Sri Krishna, Frank J Lowery, Haotian Zhuang, Ethan S Agritelley, Jiayin Bao, Austin M Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, Michael E Lidsky, Sabino Zani, Dan G Blazer III, Peter J Allen, Nicholas D Klemen, Zhicheng Ji, Daniel P Nussbaum, Erika J Crosby. Single-cell and spatial analysis of the immune landscape unveils a subset of potentially tumor-reactive T cells in patients with localized PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb49745088796a4e1e8fb6d2e9a57e189de99728" target='_blank'>
              Abstract B052: Single-cell and spatial analysis of the immune landscape unveils a subset of potentially tumor-reactive T cells in patients with localized PDAC
              </a>
            </td>
          <td>
            Elishama N. Kanu, Ashley A. Fletcher, Sri Krishna, Frank J Lowery, Haotian Zhuang, Ethan S Agritelley, Jiayin Bao, A. Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, M. Lidsky, S. Zani, D. Blazer, Peter J. Allen, Nicholas D. Klemen, Zhicheng Ji, Daniel P. Nussbaum, Erika J. Crosby
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b28b9f10dc7dee52d960145a8bb408a48e4a08ef" target='_blank'>
              Multiparametric profiling of HER2-enriched extracellular vesicles in breast cancer using Single Extracellular VEsicle Nanoscopy
              </a>
            </td>
          <td>
            Nan Jiang, Andras Saftics, Eugenia Romano, I. Ghaeli, Cristal Resto, V. Robles, Saumya Das, K. Van Keuren-Jensen, Victoria L Seewaldt, T. Jovanović-Talisman
          </td>
          <td>2024-09-28</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSC) remains a disease of poor prognosis that is unresponsive to current immune checkpoint inhibitors. Although PI3K pathway alterations, such as PTEN loss, are common in HGSC, attempts to target this pathway have been unsuccessful. We hypothesized that aberrant PI3K pathway activation may alter the HGSC immune microenvironment and present a targeting opportunity. Single-cell RNA sequencing identified populations of resident macrophages specific to Pten-null omental tumors in murine models, which were confirmed by flow cytometry. These macrophages derived from peritoneal fluid macrophages and had a unique gene expression program, marked by high expression of the enzyme heme oxygenase-1 (HMOX1). Targeting resident peritoneal macrophages prevented the appearance of HMOX1hi macrophages and reduced tumor growth. Furthermore, direct inhibition of HMOX1 extended survival in vivo. RNA sequencing identified IL33 in Pten-null tumor cells as a likely candidate driver leading to the appearance of HMOX1hi macrophages. Human HGSC tumors also contained HMOX1hi macrophages with a corresponding gene expression program. Moreover, the presence of these macrophages correlated with activated tumoral PI3K/mTOR signaling and poor overall survival in HGSC patients. In contrast, tumors with low numbers of HMOX1hi macrophages were marked by increased adaptive immune response gene expression. These data suggest targeting HMOX1hi macrophages as a potential therapeutic strategy for treating poor prognosis HGSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d6297fbdda342e2a463b24c533bad3a26f20d1" target='_blank'>
              PTEN loss shapes macrophage dynamics in high grade serous ovarian carcinoma.
              </a>
            </td>
          <td>
            S. Spear, O. Le Saux, H. B. Mirza, Nayana Iyer, Katie Tyson, Fabio Grundland Freile, J. Walton, Chloe Woodman, Sheba Jarvis, D. Ennis, Carmen Aguirre Hernandez, Yuewei Xu, P. Spiliopoulou, James D. Brenton, A. P. Costa-Pereira, D. Cook, B. Vanderhyden, Hector C. Keun, E. Triantafyllou, James N Arnold, I. McNeish
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Summary Imagine if we could simultaneously predict spatial protein expression in tissues from their routine Hematoxylin and Eosin (H&E) stained images, and create tissue images given protein expression profiles thus enabling virtual simulations of how protein expression alterations impact histology in complex diseases like cancer. Such an approach could lead to more informed diagnostic and therapeutic decisions for precision medicine at lower costs and shorter turnaround times, more detailed insights into underlying disease pathology as well as improvement in predictive and generative performance. In this study, we investigate the intricate correlation between protein expressions obtained from Hyperion mass cytometry and histopathological microstructures in conventional H&E stained glioblastoma (GBM) samples, unveiling morphological patterns and cellular-level spatial alterations associated with protein expression changes. To model these complex relationships, we propose a novel generative-predictive framework called Ouroboros for producing H&E images from protein expressions and simultaneously predicting protein expressions from H&E images. Our comprehensive sample-independent validation over 9920 tissue spots from 4 GBM samples encompassing visual image analysis, quantitative analysis, subspace alignment and perturbation experiments shows that the proposed generative-predictive approach offers significant improvements in predicting protein expression from images in comparison to baseline methods as well as accurate generation of virtual GBM sample images. This proof of concept study can contribute to advancing our understanding of histological responses to protein expression perturbations and lays the foundations for further developments in this area. Availability and implementation Implementation and associated data for the proposed approach are available at the URL: https://github.com/Srijay/Ouroboros.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67ded4a0599c7810f90359134e15ada22ea589a" target='_blank'>
              Ouroboros: cross-linking protein expression perturbations and cancer histology imaging with generative-predictive modeling
              </a>
            </td>
          <td>
            Srijay Deshpande, Sokratia Georgaka, Michael Haley, Robert Sellers, James Minshull, J. Nallala, Martin Fergie, Nicholas Stone, N. Rajpoot, Syed-Murtuza Baker, Mudassar Iqbal, Kevin Couper, Federico Roncaroli, F. Minhas
          </td>
          <td>2024-09-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="3D bioprinting has become a valuable tool for studying the biology of solid tumors, including glioblastoma multiforme (GBM). Our analysis of publicly available bulk RNA and single-cell sequencing data has allowed us to define the chemotactic profile of GBM tumors and identify the cell types that secrete particular chemokines in the GBM tumor microenvironment (TME). Our findings indicate that primary GBM tissues express multiple chemokines, whereas spherical monocultures of GBM cells significantly lose this diversity. Subsequently, the comparative analysis of GBM spherical monocultures vs. 3D-bioprinted multicultures of cells showed a restoration of chemokine profile diversity in 3D-bioprinted cultures. Furthermore, single-cell RNA-Seq analysis showed that cells of the perivascular niche (pericytes and endocytes) express multiple chemokines in the GBM TME. Next, we 3D-bioprinted cells from two glioblastoma cell lines, U-251 and DK-MG, alone and as co-cultures with mesenchymal stromal cells (representing cells of the perivascular niche) and assessed the chemokine secretome. The results clearly demonstrated that the interaction of tumors and mesenchymal cells leads to in a significant increase in the repertoire and levels of secreted chemokines under culture in 21% O2 and 1% O2. Our study indicates that cells of the perivascular niche may perform a substantial role in shaping the chemokine microenvironment in GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aae245299abc68f53e5e3fd8e4dc327ea2b0cb63" target='_blank'>
              3D-Bioprinted Co-Cultures of Glioblastoma Multiforme and Mesenchymal Stromal Cells Indicate a Role for Perivascular Niche Cells in Shaping Glioma Chemokine Microenvironment
              </a>
            </td>
          <td>
            K. Zielniok, Kinga Rusinek, A. Słysz, Mieszko Lachota, E. Bączyńska, Natalia Wiewiórska-Krata, Anna Szpakowska, Martyna Ciepielak, B. Foroncewicz, K. Mucha, Radosław Zagożdżon, Zygmunt Pojda
          </td>
          <td>2024-08-23</td>
          <td>Cells</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Understanding the mechanisms by which cancer cells switch between different adaptive states and evade therapeutic interventions is essential for clinical management. In this study, the in vivo cellular dynamics of a new chronic myeloid leukaemia cell line displaying altered phenotype and resistance to tyrosine kinase inhibitors were investigated in correlation with their parental cells for invasiveness/metastasis, angiogenic potential and population kinetics. We showed that the cells exhibiting drug resistance and plastic phenotype possess an increased capacity for invasion compared to their parental cells, that exposure to imatinib mesylate has the potential to enhance cellular motility and that in a leukaemic cell population, even a minority of plastic cells exhibit improved migratory ability. Furthermore, we show that these plastic cells have angiogenic and extravasation potential. The present study provides significant insights into the cellular dynamics displayed by a TKI‐resistant, phenotypically plastic CML cell line, using a zebrafish (Danio rerio) xenograft model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8debdaa7aac5be72ea83fbcd5d951e3b97abea7b" target='_blank'>
              Phenotypically plastic drug‐resistant chronic myeloid leukaemia cell line displays enhanced cellular dynamics in a zebrafish xenograft model
              </a>
            </td>
          <td>
            Seda Baykal, Zeynep Yuce, G. Ozhan
          </td>
          <td>2024-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor heterogeneity often leads to substantial differences in responses to same drug treatment. The presence of pre-existing or acquired drug-resistant cell sub-populations within a tumor survive and proliferate, ultimately resulting in tumor relapse and metastasis. The drug resistance is the leading cause of failure in clinical tumor therapy. Therefore, accurate identification of drug-resistant tumor cell subpopulations could greatly facilitate the precision medicine and novel drug development. However, the scarcity of single-cell drug response data significantly hinders the exploration of tumor cell resistance mechanisms and the development of computational predictive methods. In this paper, we propose scDrugAtlas, a comprehensive database devoted to integrating the drug response data at single-cell level. We manually compiled more than 100 datasets containing single-cell drug responses from various public resources. The current version comprises large-scale single-cell transcriptional profiles and drug response labels from more than 1,000 samples (cell line, mouse, PDX models, patients and bacterium), across 66 unique drugs and 13 major cancer types. Particularly, we assigned a confidence level to each response label based on the tissue source (primary or relapse/metastasis), drug exposure time and drug-induced cell phenotype. We believe scDrugAtlas could greatly facilitate the Bioinformatics community for developing computational models and biologists for identifying drug-resistant tumor cells and underlying molecular mechanism. The scDrugAtlas database is available at: http://drug.hliulab.tech/scDrugAtlas/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c67f110b19aa753b80dffa15ad6e0a2ac3b41aaf" target='_blank'>
              scDrugAtlas: an integrative single-cell drug response atlas for unraveling tumor heterogeneity in therapeutic efficacy
              </a>
            </td>
          <td>
            Wei Huang, Xinda Ren, Yinpu Bai, Hui Liu
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Introduction This study aims at elucidating mechanisms of immune infiltration in osteosarcoma (OSA) to identify novel therapeutic strategies. Methods Here, joint analysis of spatial transcriptomics (ST, Visium) data from 26 formalin-fixed paraffin embedded OSA samples at diagnosis (n=9) and relapse (n=17, MAPPYACTS NCT02613962 & OS2006 NCT00470223) from 19 donors (including 5 diagnosis/relapse pairs) was performed using the CellsFromSpace unbiased reference-free signal deconvolution and analysis workflow. This methodology enabled the definition of the topology and the molecular signatures of the cancerous, stromal and immune components across all samples, and allowed for the phenotyping of individual tumors based on cancer subset composition. Spatial colocalization analysis was performed using Lee’s bivariate spatial association measure of different cell populations, enabling the characterization of tumor-infiltrating immune cells found across samples. Similar ST analysis of Patient-derived xenograft (PDX) samples was performed on fresh-frozen samples. Subsequent separation of human and mouse reads using the Xenome algorithm allowed for a clear dissection of the cancer vs. microenvironment compartments. Preliminary signature validation on larger cohorts was performed on bulk RNAseq samples from the OS2006 and MAPPYACTS cohorts. Results Our joint spatial transcriptomics analysis of 26 primary and relapse tumors revealed 64 transcriptomic signatures associated with cancer cells, some of which were shared across multiple samples. These signatures could be consolidated into 18 broad phenotypic categories frequently detectable in various samples, showcasing significant intratumoral heterogeneity within the cancerous compartment. By averaging the composition per sample, we found that samples clustered based on the relative intratumoral abundance of cancer phenotypes, indicating the presence of at least two main differentiation archetypes in OSA: “High intratumoral heterogeneity” and “Undifferentiated.” Additionally, spatial niche analysis of immune cells within tumors identified immune populations consistently associated with tumors across samples. This analysis revealed a specific myeloid lineage capable of infiltrating tumors in all samples, likely mediated by chemotactic signals linked to cancer differentiation, as suggested by ligand-receptor analysis. To validate these findings, PDX models of OSA were analyzed, confirming the presence of a similar phenotype of tumor-infiltrating myeloid cells and underscoring the relevance of PDX models in OSA research. Furthermore, preliminary analyses querying the transcriptomic signature of infiltrating myeloid cells in larger bulk RNAseq cohorts revealed promising associations with patient outcomes, detectable even at diagnosis. Conclusions Our research elucidates patterns of intratumoral phenotypic heterogeneity in OSA and identifies the primary immune cell populations specifically infiltrating OSA tumors. This paves the way for developing novel therapies by leveraging these cellular tropism mechanisms.
 Citation Format: Gaël Moquin-Beaudry, Maria Eugenia Marques Da Costa, Hanane Zair, Corentin Thuilliez, Pierre Khneisser, Nicolas Signolle, Jean-Yves Scoazec, Birgit Geoerger, Nathalie Gaspar, Antonin Marchais. Characterizing the spatial transcriptomic landscape of Osteosarcoma from diagnosis to relapse [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0f2ff33e6fea180aea3b91082872e0d960db848" target='_blank'>
              Abstract B033: Characterizing the spatial transcriptomic landscape of Osteosarcoma from diagnosis to relapse
              </a>
            </td>
          <td>
            G. Moquin-Beaudry, Maria Eugénia Marques da Costa, Hanane Zair, Corentin Thuilliez, Pierre Khneisser, Nicolas Signolle, J. Scoazec, Birgit Geoerger, Nathalie Gaspar, A. Marchais
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Despite efforts to understand breast cancer biology, metastatic disease remains a clinical challenge. Identifying suppressors of breast cancer progression and mechanisms of transition to more invasive phenotypes could provide game changing therapeutic opportunities. Transcriptional deregulation is central to all malignancies, highlighted by the extensive reprogramming of regulatory elements that underlie oncogenic programs. Among these, super-enhancers (SEs) stand out due to their enrichment in genes controlling cancer hallmarks. To reveal novel breast cancer dependencies, we integrated the analysis of the SE landscape with master regulator activity inference for a series of breast cancer cell lines. As a result, we identified T-helper-inducing Poxviruses and Zinc-finger (POZ)/Krüppel-like factor (ThPOK, ZBTB7B), a CD4+ cell lineage commitment factor, as a breast cancer master regulator that is recurrently associated with a SE. ThPOK expression is highest in luminal breast cancer but is significantly reduced in the basal subtype. Manipulation of ThPOK levels in cell lines shows that its repressive function restricts breast cancer cells to an epithelial phenotype by suppressing the expression of genes involved in the epithelial-mesenchymal transition (EMT), WNT/β-catenin target genes, and the pro-metastatic TGFβ pathway. Our study reveals ThPOK as a master transcription factor that restricts the acquisition of metastatic features in breast cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bcac35bb061bf9ab262a029b66da149e294b402" target='_blank'>
              Super-enhancer profiling reveals ThPOK/ZBTB7B, a CD4+ cell lineage commitment factor, as a master regulator that restricts breast cancer cells to a luminal non-migratory phenotype
              </a>
            </td>
          <td>
            Camila D. Arcuschin, Kamin Kahrizi, Rosalyn W. Sayaman, Carolina DiBenedetto, Yizhuo Shen, Pedro J Salaberry, Ons Zakroui, Cecilia Schwarzer, Alessandro Scapozza, Paola Betancur, Julie D. Saba, Jean-Philippe Coppé, M. Barcellos-Hoff, Dietmar Kappes, L. J. van ’t Veer, Ignacio E Schor, Denise P Muñoz
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22d59061a2bd287943e37e1f11ba6104aff05a2f" target='_blank'>
              Prognostic and therapeutic implications of tumor-restrictive type III collagen in the breast cancer microenvironment
              </a>
            </td>
          <td>
            Daniel C Stewart, Becky K Brisson, Bassil Dekky, Ashton C. Berger, William K. Yen, Elizabeth A Mauldin, Claudia Loebel, Deborah Gillette, C. Assenmacher, Corisa Quincey, Darko Stefanovski, M. Cristofanilli, Edna Cukierman, Jason A. Burdick, Virginia F Borges, S. Volk
          </td>
          <td>2024-10-02</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Gliomas are primary brain tumors that develop from glial cells within the central nervous system and are among the deadliest human cancers. Glioblastoma (GBM) is the most malignant form of glioma. NLRX1 is an innate immune pattern recognition receptor that exhibits tumor-suppressive and tumor-promoting effects that may be cancer or cell-type, context-dependent, aided by differences in the microenvironment. Here, we report that NLRX1 is differentially expressed in microglia, astrocytes, GBM cell lines, and glioma patient tissues. siRNA-mediated silencing of Nlrx1 decreases the ability of the GBM cell line, LN-229, to proliferate and migrate. Nlrx1-/- GBM cells exhibit attenuated ability to generate 3D spheroids and enhanced capability to form tunneling nanotubes. Moreover, Nlrx1-/- GBM cells show decreased expression of autophagy markers, suggesting that NLRX1 plays a role in maintaining autophagy in GBM. In summary, our findings indicate that NLRX1 may modulate GBM pathophysiology by regulating GBM cell proliferation, migration, and metabolism. We believe our understanding of NLRX1 in GBM pathophysiology paves the potential development of GBM-targeting therapeutics that may delay disease progression and/or improve survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95a9293ab6ceae7c379f276e26000ebf9b80a6b9" target='_blank'>
              Innate Immune Receptor NLRX1: Potential Modulator of Glioblastoma Pathophysiology
              </a>
            </td>
          <td>
            Durgesh Meena, Divya Shivakumar, Sushmita Rajkhowa, Neermita Bhattacharya, Priya Solanki, Shalini Chhipa, Vikas Janu, M. Garg, J. Gosal, Sushmita Jha
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Isocitrate dehydrogenase 1/2 mutant (IDHmt) astrocytoma is considered a T cell-deprived tumor, yet little is known regarding the exact phenotype that relates to T cell exclusion. To aid better understanding of this mechanism, we systematically characterized the immune microenvironment in IDHmt astrocytoma.



 We performed multiplex immunofluorescence (IF) stainings for CD4+ and CD8+ T cells, B cells, tumor associated macrophages (TAMs) and tumor cells on 75 IDHmt astrocytoma samples from which we collected and analyzed 1295 images. We performed bulk RNA sequencing on 158 IDHmt astrocytoma samples, accompanied with single nucleus RNA sequencing and spatial RNA and protein profiling on samples from the same cohort. We integrated and analyzed all datasets to acquire a robust characterization of the microenvironment organization.



 We demonstrated that when present, T cells were either distributed in the tumor stroma, or in the majority of cases, contained in the perivascular space (i.e. T cell cuffs). We found that the tumor stroma surrounding T cell cuffs showed significant local accumulation of gemistocytic tumor cells (GTCs), a tumor cell type that histologically shows a large eosinophilic cytoplasm and ectopically placed nucleus. GTC-high tumors (>20% of all tumor cells) demonstrated enhanced expression of genes related to immune cell migration and activation that were reliably expressed by TAMs and T cells. Notably, we found that GTCs formed a unique transcriptional cell sub-cluster that was part of the ‘astrocyte-like’ tumor cell state. GTCs expressed markers such as CD44, TNC and CRYAB that are specific for glial scarring, a central nervous system specific mechanism for immune cell exclusion. Surrounding T cell cuffs, GTCs co-localized with TAMs that transcriptomically resembled reactive microglia and showed increased expression of the chemo-attractants CCL3 and CCL4. Together, GTCs and reactive microglia expressed receptor-ligand pairs that take part in reactive astrogliosis and glial scarring, such as Osteopontin (SPP1) - CD44 and IL-1β - IL1R1. We validated this spatial pattern on protein level using whole-slide multiplex IF stainings of GTC-low and GTC-high tumors and showed specific local accumulation of CD44 positive GTCs and TAMs around T cell cuffs in GTC high tumors.



 Using different modalities of bulk, spatial and single cell profiling for RNA and protein, we uncovered a previously unknown spatial pattern of co-localizing GTCs and reactive microglia around T cell cuffs in IDHmt astrocytoma. Together, we show that IDHmt astrocytomas organize a cellular network of tumor cells and leukocytes that mimics glial scarring and can aid immune cell exclusion. Characterizations of these complex localized interactions between tumor- and immune cell subsets aids a more specific target selection for immune therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a0b251a3c8047c4a4b52d909d37695b06564f8" target='_blank'>
              OS11.6.A GEMISTOCYTIC TUMOR CELLS AND IMMUNE REACTIVE MICROGLIA PROGRAMMED FOR GLIAL SCARRING CHARACTERIZE T CELL CONTAINMENT IN IDH-MUTANT ASTROCYTOMA
              </a>
            </td>
          <td>
            L. van Hijfte, M. Geurts, I. D. de Heer, S. Ghisai, H. Balcioglu, Y. Hoogstrate, W. Vallentgoed, R. Head, B. Westerman, P. Wesseling, J. Joyce, R. Debets, P. French
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ad6a53d090caa0cff0b86e7a380a1302888db4" target='_blank'>
              Circulating tumor cells: from new biological insights to clinical practice
              </a>
            </td>
          <td>
            Xuyu Gu, Shiyou Wei, Xin Lv
          </td>
          <td>2024-09-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Incomplete understanding of metastatic disease mechanisms continues to hinder effective treatment of cancer. Despite remarkable advancements toward the identification of druggable targets, treatment options for patients in remission following primary tumor resection remain limited. Bioengineered human tissue models of metastatic sites capable of recreating the physiologically relevant milieu of metastatic colonization may strengthen our grasp of cancer progression and contribute to the development of effective therapeutic strategies. We report the use of an engineered tissue model of human bone marrow (eBM) to identify microenvironmental cues regulating cancer cell proliferation and to investigate how triple-negative breast cancer (TNBC) cell lines influence hematopoiesis. Notably, individual stromal components of the bone marrow niche (osteoblasts, endothelial cells, and mesenchymal stem/stromal cells) were each critical for regulating tumor cell quiescence and proliferation in the three-dimensional eBM niche. We found that hematopoietic stem and progenitor cells (HSPCs) impacted TNBC cell growth and responded to cancer cell presence with a shift of HSPCs (CD34+CD38-) to downstream myeloid lineages (CD11b+CD14+). To account for tumor heterogeneity and show proof-of-concept ability for patient-specific studies, we demonstrate that patient-derived tumor organoids survive and proliferate in the eBM, resulting in distinct shifts in myelopoiesis that are similar to those observed for aggressively metastatic cell lines. We envision that this human tissue model will facilitate studies of niche-specific metastatic progression and individualized responses to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c057d7a78a1831b3a99616c08933dc1aff17f82" target='_blank'>
              An engineered model of metastatic colonization of human bone marrow reveals breast cancer cell remodeling of the hematopoietic niche.
              </a>
            </td>
          <td>
            Ilaria Baldassarri, D. Tavakol, Pamela L Graney, Alan Chramiec, H. Hibshoosh, G. Vunjak‐Novakovic
          </td>
          <td>2024-10-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Metastatic breast cancer remains largely incurable, and the mechanisms underlying the transition from primary to metastatic breast cancer remain elusive. We analyzed the complex landscape of primary and metastatic breast cancer using scRNA-seq data from twenty-three female patients with either primary or metastatic disease to elucidate the genetic and molecular mechanisms underlying changes in the metastatic tumor ecosystem. We identify specific subtypes of stromal and immune cells critical to forming a pro-tumor microenvironment in metastatic lesions, including CCL2+ macrophages, cytotoxic T cells with an exhausted gene signature, and FOXP3+ regulatory T cells. Analysis of cell-cell communication highlights a marked decrease in tumor-immune cell interactions in metastatic tissues, likely strengthening the immunosuppressive microenvironment. In contrast, primary breast cancer samples displayed increased activation of the TNF- signaling pathway via NF-kB, indicating a potential therapeutic target. Our study comprehensively characterizes the transcriptional landscape encompassing primary and metastatic breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f83bcd576506ee2a8774cb0cfd1433b6936e5b1" target='_blank'>
              Single-cell RNA sequencing reveals immunosuppressive pathways associated with metastatic breast cancer
              </a>
            </td>
          <td>
            F. Ozmen, T. Y. Ozmen, A. Ors, M. Janghorban, M. J. Rames, X. Li, F. Behbod, G. B. Mills, H. Mohammed
          </td>
          <td>2024-09-26</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87d5f0ed69b574960ec0cf8e7cfd67555c4fa973" target='_blank'>
              Drug resistant pancreatic cancer cells exhibit altered biophysical interactions with stromal fibroblasts in imaging studies of 3D co-culture models
              </a>
            </td>
          <td>
            Eric Struth, M. Labaf, V. Karimnia, Yiran Liu, Gwendolyn Cramer, Joanna B. Dahl, Frank J Slack, Kourosh Zarringhalam, Jonathan P Celli
          </td>
          <td>2024-09-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Tumours are complex ecosystems comprised of tumour cells and various normal cell types. The interactions among these cells with diverse molecular characteristics lead to intratumoral heterogeneity and cancer evolution. However, little is known about the spatial organisation of the glioma tumour ecosystem, the cellular interactions therein, and their influence on patient outcomes. Spatially resolved transcriptomics, which measures the activity of hundreds of genes in situ, offers new avenues to address these long-standing questions.



 We assembled a clinically annotated cohort of glioma patients spanning 278 specimens from low- and high-grade IDH-mutant astrocytoma, IDH-mutant and 1p/19q co-deleted oligodendroglioma, ependymoma, and glioblastoma. Using tissue microarrays from archival FFPE, we performed spatial transcriptomics profiling with the 10x Xenium platform. A custom panel of 344 genes, selected based on publicly available single-cell RNA sequencing datasets of gliomas, was interrogated.



 The spatial transcriptomics data enabled annotation and spatial mapping of more than 20 tumour microenvironment (TME) cell types and various tumour cell types present across the 5 glioma entities. We observed broadly conserved transcriptional profiles of TME cell types across gliomas, but TME cell type composition and spatial organisation varied depending on tumour entity. In particular, lower amounts of macrophages and microglia as well as oligodendrocytes and oligodendrocyte progenitor cells were observed in oligodendroglioma tumours compared to other entities. High-grade astrocytoma samples contained more myeloid dendritic cells and monocytes in comparison to low-grade astrocytomas. A lower proportion of vascular cells correlated with better overall survival for glioblastomas and low- and high-grade astrocytomas. Tumour cells exhibited mostly entity-specific gene expression, but also distinct tumour cell states including AC-, OPC-, NPC-, and MES-like cell states.



 Our work demonstrates the utility of the 10x Xenium platform for high-resolution spatial profiling of expression in archival FFPE samples. Preliminary mapping of the TME and tumour cell types unravelled a complex spatial architecture of the glioma ecosystems. Distinct compositions of the TME were found in each glioma subtype, which may help understand different mechanisms of malignancy. Associations between cellular spatial patterns and clinical data available for FFPE samples have the potential to provide better insights for improved patient stratification and treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5877e1818dcb8b95627446b49e8f170fc56f766" target='_blank'>
              P01.03.A DECIPHERING THE TUMOUR ECOSYSTEM IN GLIOMAS USING SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            Z. Seferbekova, D. Calafato, G. Rukhovich, A Patel, M. Ritter, F. Ippen, A. Suwala, F. Sahm, M. Gerstung
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Isocitrate dehydrogenase (IDH) mutant gliomas are progressive, invariably fatal brain tumors affecting young and middle-age adult patients. Previous single-cell transcriptomic characterization of IDH-mutant gliomas has uncovered a heterogeneity of cell states, with a developmental hierarchy that starts with neural progenitor-like stem cells that transition to more differentiated malignant cell states. Despite this understanding, the dynamics of these cell state transitions as tumors undergo grade progression, and the factors that facilitate or restrict cell state transitions remain poorly understood.



 Here, we characterize the cell states of IDH-mutant gliomas via single-cell gene expression (scRNA-seq) and single-cell analyses of accessible chromatin (scATAC-seq) in tumors spanning all grades using fresh patient samples and compare them with transcriptional programs and chromatin accessibility profiles of normal brain cells. We perform mechanism validation experiments using IDH-mutant glioma organoids.



 We find that tumor transcriptional cell states experience a marked shift as tumors undergo progression. In low-grade tumors, predominant cell states mimic slow-cycling oligodendrocyte progenitor-like cells (OPCs). With grade progression, predominant cell states transition to resemble highly proliferative neural progenitor-like cells (NPCs). Additionally, scATAC-seq analyses revealed novel permissive chromatin states, which could facilitate this transition. Genetic analyses identified several molecular alterations in high-grade tumors, including loss of DNA methylation, oncogene amplification, and the deletion of genes associated with the interferon (IFN) response. Those IFN genes, as well as tumor suppressors, are initially repressed by DNA hypermethylation in low-grade tumors. Lastly, we validate the role of DNA hypomethylation in favoring immune evasion with grade progression using an IDH-mutant glioma organoid model.



 Overall, our findings provide evidence that tumor progression in IDH-mutant gliomas is driven by an interplay of epigenetic and genetic mechanisms, with the former being determinant in low-grade tumors and the latter in high-grade tumors. The emergence of genetically-driven mechanisms promotes more mitotically active cell states as well as decreased anti-tumoral immune activation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13fd62dc7724c0acd329992464a530f1dbcd46aa" target='_blank'>
              KS02.6.A SINGLE-CELL MULTIMODAL ANALYSIS OF IDH-MUTANT GLIOMAS UNCOVERS SHIFT FROM EPIGENETIC TO GENETIC DRIVERS WITH TUMOR PROGRESSION
              </a>
            </td>
          <td>
            L. G. Gonzalez Castro, J. Wu, S. Battaglia, C. El Farran, J. D’Antonio, T. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 Introduction: Pediatric sarcomas are the second most common solid tumor in children and adolescence. Despite advances in multimodal therapies, one-third of sarcoma patients do not survive the disease. The development of new treatments for sarcomas is hindered by a limited understanding of the tumor microenvironment (TME), especially the role of immune cells such as tumor-associated macrophages (TAMs). The abundance of TAMs significantly impacts the clinical outcomes of pediatric sarcoma patients, yet our knowledge of their molecular heterogeneity and interactions with other cell types in the TME remains limited. Methods: To characterize the transcriptomic and epigenomic heterogeneity of sarcoma TME, we profiled 32 treatment-naïve tumor samples from 5 pediatric sarcoma subtypes using the single-nuclei RNA+ATAC-Seq multiome assay. We also interrogated the spatial organization of the sarcoma TME using the multiplexed immunohistochemistry assay, Co-detection by indexing (CODEX), with a focus on the diversity and inter-cellular interactions of TAMs. We utilized in vitro co-culture models of human monocyte-derived macrophages, CD8 T cells and malignant sarcoma cells to validate immunosuppressive signaling interactions between TAMs and CD8 T cells predicted based on our multiome and CODEX data. Results: We sequenced 175,964 and 178,000 high-quality cells from RNA and ATAC modality respectively. Analysis of cell type frequencies across sarcoma subtypes showed that TAMs were the most abundant immune cells in the TME. We identified seven TAM subpopulations with distinct gene expression and epigenomic signatures. Among these, SPP1+ and C1QC+ TAM subsets exhibited higher immunosuppressive potential. By integrating gene expression and chromatin accessibility data, we identified key transcription factors that drive the transcriptional phenotypes of SPP1+/C1QC+ TAMs, including RUNX1 and ELF2. By ligand-receptor analysis using our snRNA-Seq data, we identified signaling ligand-receptor pairs that potentially mediate immunosuppression by SPP1+/C1QC+ TAMs, including SPP1-CD44 and HLA-E-KLRC1. Additionally, interactions like SEMA3D-NRP1 and LTBP3-ITGB5 between malignant cells and SPP1+/C1QC+ TAMs were observed in multiple sarcoma subtypes and significantly associated with inferior prognosis. Furthermore, our CODEX data revealed spatial co-localization between SPP1+TAMs and T cells, further corroborating the immune regulatory role of SPP1+TAMs in the sarcoma TME. Finally, our in vitro experiments revealed that macrophages co-cultured with sarcoma cell lines (RD and RH30) showed increased expression of C1QC/SPP1+ TAM markers, including CD163 and CD109, and demonstrated enhanced immunosuppressive effect on CD8+ T cells. Conclusion: This study presents a comprehensive single-cell multi-omics and spatial omics analysis of the pediatric sarcoma microenvironment. By identifying specific pro-tumorigenic TAM subsets and signaling interactions associated with inferior patient outcome, this study reveals novel insights into the role of TAMs in pediatric sarcomas.
 Citation Format: Zhan Zhang, Kyung Jin Ahn, Rumeysa Biyik-Sit, Changya Chen, Anusha Thadi, Chia-hui Chen, William Molina, Brian Lockhart, Theodore Laetsch, Lea Surrey, Malay Haldar, Kathrin Bernt, Vinodh Pillai, Kai Tan. Dissecting pediatric sarcoma microenvironment using single-cell and spatial multi-omics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B061.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea4e7ffe970f4c90d3fbc5950148fc0b9f42efbd" target='_blank'>
              Abstract B061: Dissecting pediatric sarcoma microenvironment using single-cell and spatial multi-omics
              </a>
            </td>
          <td>
            Zhan Zhang, Kyung Jin Ahn, Rumeysa Biyik‐Sit, Changya Chen, Anusha Thadi, Chia-Hui Chen, William Molina, Brian Lockhart, Theodore W Laetsch, Lea Surrey, Malay Haldar, Kathrin M Bernt, Vinodh Pillai, Kai Tan
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Hotspot mutations in the IDH genes serve as tumor-initiating events that define a subset of adult diffuse glioma that can further be divided into either oligodendroglioma or astrocytoma subtypes. Despite standard treatment, IDH-mutant gliomas inevitably recur. Previous studies suggest that therapeutic resistance may result from a combination of intratumoral cellular heterogeneity, epigenetic evolution, and acquired genetic alterations. However, how these multilayered molecular features interact to influence the evolutionary paths of IDH-mutant gliomas remains incompletely understood.



 To chart this evolution, 75 glioma samples were longitudinally collected from 35 patients (n = 13 oligodendroglioma, n = 22 astrocytoma) and profiled using single nucleus RNA sequencing, ATAC sequencing, and bulk DNA sequencing.



 Analysis of 331,016 nuclei (snRNA) revealed eight tumor microenvironment cell types and five pan-IDH-mutant malignant cellular states. The malignant states were distributed along a cellular hierarchy of stem-like populations (neural progenitor cell-like, oligodendrocyte progenitor cell-like, and undifferentiated) and more differentiated populations (mesenchymal-like and astrocyte-like) with cycling cells enriched in the stem-like populations. Joint open chromatin accessibility data (snATAC) was available for a subset of malignant cells (71,088 nuclei) and state-specific differentially accessible peaks supported that these states are epigenetically encoded. Across both subtypes, higher tumor grade was associated with reduced malignant cell differentiation and increased cycling populations. Between the two time points, we identified that there was a longitudinal increase in the cycling, undifferentiated, and mesenchymal-like populations with a corresponding decrease in the astrocyte-like population. Longitudinal genetic analysis revealed that 19 of 35 tumors acquired at least one of the following genetic alterations: treatment-associated hypermutation, cell cycle alteration, or large changes in copy number alterations. Importantly, tumors that acquired these key genetic alterations demonstrated significant shifts towards reduced differentiation and increased cycling populations while those tumors that did not had more stable malignant profiles.



 Collectively, our results suggest a common cellular hierarchy across IDH-mutant gliomas with a shift towards reduced differentiation, increased cycling populations during disease progression that is driven by acquired genetic alterations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d952023b3d2c6da79f53a908951847332e2e66b4" target='_blank'>
              KS02.7.A LONGITUDINAL SINGLE-CELL ANALYSES IDENTIFY DRIVERS OF GENETIC, EPIGENOMIC, AND CELLULAR EVOLUTION IN IDH-MUTANT GLIOMA
              </a>
            </td>
          <td>
            K C Johnson, A. Spitzer, F. Varn, M. Nomura, L. Garofano, T. Chowdhury, K J Anderson, F. D’Angelo, L. Bussema, S. Gritsch, Y Oh, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, A. Lipsa, F. Hertel, Z. Zhao, Q. Wang, T Jiang, B. Hermes, N. Sanai, A. Gołȩbiewska, S. Niclou, J. Huse, W. Yung, A. Lasorella, M. Suvà, A. Iavarone, I. Tirosh, R. Verhaak
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Background The landscape of the tumor microenvironment (TME) is intricately linked to the development of head and neck squamous cell carcinoma (HNSCC) and significantly influences immunotherapy efficacy. Recent research has underscored the pivotal role of PNCK in cancer progression, yet its relationship with immunotherapy remains elusive. Methods We leveraged sequencing data from our cohort and public databases to evaluate PNCK expression, prognostic significance, and immune efficacy prediction. In vitro and in vivo experiments explored the role of PNCK in HNSCC progression. Animal models assessed the therapeutic effects and survival benefits of PNCK knockdown combined with immune checkpoint inhibitors (ICIs). Single-cell transcriptomics analyzed the impact of PNCK on the TME, and proteomic studies elucidated the mechanisms. Results PNCK exerts multifaceted critical roles in the progression of HNSCC. Lower PNCK expression is associated with improved prognosis, enhanced immune cell infiltration, and increased responsiveness to ICIs. Conversely, PNCK promotes HNSCC cell migration, invasion, proliferation, colony formation, zebrafish angiogenesis, and tumor growth in mice. Moreover, targeting PNCK enhances sensitivity to ICIs and leads to significant alterations in the T-cell and B-cell ratios within the TME. These changes are attributed to the inhibition of nuclear transcription of PNCK-phosphorylated ZEB1, which restricts cytokine release and inflames the immune microenvironment to regulate the TME. Conclusions Inhibition of PNCK may be a potential strategy for treating HNSCC, as it may activate the immune response and improve the TME, thereby enhancing the efficacy of immunotherapy for HNSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e39710b7154239b65b699617b1705c954762a80e" target='_blank'>
              Inhibition of PNCK inflames tumor microenvironment and sensitizes head and neck squamous cell carcinoma to immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Q. Ding, Y. Weng, Ying Li, Wanzun Lin, Xiaosan Lin, Tingting Lin, Hanxuan Yang, Wenqian Xu, Jianmin Wang, Hongmei Ying, Sufang Qiu
          </td>
          <td>2024-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Purpose: Optimization of pancreatic ductal adenocarcinoma (PDAC) patient survival will depend on advancing our understanding of the mechanisms governing tumor cell plasticity and effectively mitigating their invasive and metastatic capabilities. This study aims to investigate the correlation of cancer biomarkers crucial to these processes by integrating molecular signatures with histopathological characteristics using single-cell RNA sequencing (scRNA-seq) and multiplexed IHC/IF. Methods: Single-cell RNA sequencing (scRNA-seq) data from a comprehensive dataset of over 53,000 PDAC cells and validation of lead targets through 14-plex immunofluorescence, using a customized MICS platform (Miltenyi Biotec) and dual color IHC were employed to assess protein expression of defining biomarkers of basal and classical subtypes of PDAC including Keratin 17 (K17) and GATA6. Results: scRNAseq analysis revealed distinct patterns of K17 and GATA6 expression, highlighting intratumoral heterogeneity among PDAC tumors. Specifically, populations of cells expressing K17+/GATA6+ and K17+/GATA6- exhibited significant enrichment in pathways associated with invasion and cell migration, including PI3K/Akt/mTOR, TGF-β, and epithelial-mesenchymal transition (EMT) signaling. Conversely, GATA6-/K17- cells demonstrated enrichment in biosynthetic and metabolic pathways that govern cell cycle regulation. Additionally, mIF data assessment revealed correlations between histomorphological patterns and dynamic shifts in K17 and GATA6 expression within PDAC tumors. Diffusely infiltrative tumor cells (DITCs) uniformly expressed K17, contrasting with the gland-forming components (GFCs) of PDAC, which showed enrichment in GATA6 and sporadic expression of K17. E-cadherin, a key adhesion marker, exhibited the highest expression levels in the glandular and solid components of PDAC, underscoring their epithelial characteristics. Conversely, vimentin, a marker typically associated with epithelial-to-mesenchymal transition (EMT), was significantly expressed in DITCs, suggesting a mesenchymal-like phenotype in these infiltrative cells. Dual color IHC further validated these findings, reinforcing the correlation between morphologic plasticity and the expression patterns of K17 and GATA6, emphasizing their potential to distinguish distinct tumor cell populations within PDAC. Conclusions: These insights suggest that the interaction between these markers may indicate a distinct biological state characterized by altered cellular metabolism and potentially different therapeutic vulnerabilities. The robust association between K17 expression and diffusely infiltrative tumor cells underscores its potential as a marker for the most aggressive forms of this disease. Our results highlight the heterogeneous nature of PDAC and emphasize the critical importance of understanding these molecular distinctions in histopathological contexts to develop targeted therapies aimed at improving patient survival.
 Citation Format: Lyanne A Delgado- Coka, Brian Nelson, Michael Horowitz, Shayan Sarkar, Natalia Marchenko, Scott Powers, Luisa F Escobar-Hoyos, Kenneth Shroyer. Keratin 17 and GATA6 correlate with diffusely infiltrative versus gland-forming components of PDAC: Uncovering the transitional state in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C070.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10730db85bb1ae24ebbf29f9506b2592a344ce53" target='_blank'>
              Abstract C070: Keratin 17 and GATA6 correlate with diffusely infiltrative versus gland-forming components of PDAC: Uncovering the transitional state in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A Delgado- Coka, Brian J. Nelson, Michael Horowitz, Shayan Sarkar, N.Ye Marchenko, Scott Powers, Luisa F. Escobar‐Hoyos, Kenneth R. Shroyer
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) disease progression involves complex cell state transitions in both malignant and non-malignant populations within the tumor microenvironment. To uncover PDAC cell states and epigenetic features associated with clinical outcome, we profiled 40 tumor samples (33 primary tumors and 7 metastases) from 39 patients with single cell ATAC-seq, along with 10 sample-matched single cell RNA-seq profiles. Through topic modeling of high dimensional chromatin data, we uncovered distinct cis-regulatory networks underlying the classical and basal-like molecular subtypes of PDAC, which showed differential activity in basal-classical hybrid cells. Furthermore, delineation of epigenetic alterations by basal, classical, and hybrid subtypes revealed a core set of chromatin regions opened in all PDAC subtypes relative to ductal and acinar cells. These universally open regions showed striking overlap with gastrointestinal enhancers, highlighting a common endoderm lineage infidelity process across pancreatic tumors. Within the stromal compartment, we identified novel regulators of myofibroblastic cancer-associated fibroblasts (myCAFs) and inflammatory CAFs (iCAFs) from enhancer networks active in each CAF subtype. Importantly, our data uncover a highly prognostic gene program active in only one of two distinct myCAF chromatin states found in our data, highlighting a putative tumor-promoting myCAF subpopulation. Our work provides a high-resolution description of epigenetic cell state heterogeneity in PDAC, revealing novel regulators and gene programs associated with PDAC subtypes and clinical outcome.
 Citation Format: Kevin MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, Carl Pelz, Trent Waugh, Ethan Agritelle, John Muschler, Andrew Adey, Rosalie Sears. Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55da9db59167776bfc809f1ebb52c0cfa3226d09" target='_blank'>
              Abstract C078: Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes
              </a>
            </td>
          <td>
            Kevin A MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, C. Pelz, Trent Waugh, Ethan Agritelle, John L. Muschler, Andrew C. Adey, Rosalie C. Sears
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c56d7dfef26c62111d5f106e161ec720c881a32b" target='_blank'>
              A new exploration: characterization of the differentiation trajectory of prostate cancer cells
              </a>
            </td>
          <td>
            Jiyu Yang, Changyou Wang, Xiao Ma, Jie Li, Haoran Yuan, Renzhen Tan, Ling Ling, Xianchun Zhou
          </td>
          <td>2024-09-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Esophageal cancer (EC) is a major health issue, ranking seventh in incidence and sixth in mortality worldwide. Despite advancements in multidisciplinary treatment approaches, the 5-year survival rate for EC remains low at 21%. Challenges in EC treatment arise from late-stage diagnosis, high malignancy, and poor prognosis. Understanding the tumor microenvironment is critical, as it includes various cellular and extracellular components that influence tumor behavior and treatment response. Mast cells (MCs), as tissue-resident immune cells, play dual roles in tumor dynamics. High-throughput single-cell RNA sequencing offers a powerful tool for analyzing tumor heterogeneity and immune interactions, although its application in EC is limited. Methods In this study, we investigated the immune microenvironment of EC using single-cell RNA sequencing and established a comprehensive immune profile. We also performed analysis of upstream transcription factors and downstream pathway enrichment to further comprehensively decipher MCs in EC. Besides, we performed knockdown experiments to explore the role of epidermal growth factor receptor (EGFR) signaling pathway in MCs-tumor cell interactions, highlighting its potential as a prognostic marker. Finally, we constructed a prognostic model for EC, which provided valuable suggestions for the diagnosis and prognosis of EC. Results Our analysis identified 11 major cell types, of which MCs were particularly present in pericarcinoma tissues. Further grouping of the 5,001 MCs identified 8 distinct subtypes, including SRSF7-highly expressed MCs, which showed strong tumor preference and potential tumor-promoting properties. Moreover, we identified the key signaling receptor EGFR and validated it by in vitro knockdown experiments, demonstrating its cancer-promoting effects. In addition, we established an independent prognostic indicator, SRSF7+ MCs risk score (SMRS), which showed a correlation between high SMRS group and poor prognosis. Conclusion These findings illuminate the complex interactions within the tumor microenvironment of EC and suggest that targeting specific MCs subtypes, particularly via the EGFR signaling pathway, may present novel therapeutic strategies. This study establishes a comprehensive immune map of EC, offering insights for improved treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b52619ad1abc5e3af051c9a613c06b47617866c" target='_blank'>
              Unraveling the ecological landscape of mast cells in esophageal cancer through single-cell RNA sequencing
              </a>
            </td>
          <td>
            Shengyi Zhang, Xinyi Zhang, Zhikai Xiahou, Shunqing Zuo, Jialong Xue, Yi Zhang
          </td>
          <td>2024-10-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Although the combination of immunotherapy and radiotherapy (RT) for the treatment of malignant tumors has shown rapid development, the insight of how RT remodels the tumor microenvironment to prime antitumor immunity involves a complex interplay of cell types and signaling pathways, much of which remains to be elucidated. Four tumor samples were collected from the same abdominal wall metastasis site of the patient with gastric cancer at baseline and during fractionated RT for single-cell RNA and T-cell receptor sequencing. The Seurat analysis pipeline and immune receptor analysis were used to characterize the gastric cancer metastasis ecosystem and investigated its dynamic changes of cell proportion, cell functional profiles and cell-to-cell communication during RT. Immunohistochemical and immunofluorescent staining and bulk RNA sequencing were applied to validate the key results. We found tumor cells upregulated immune checkpoint genes in response to RT. The infiltration and clonal expansion of T lymphocytes declined within tumors undergoing irradiation. Moreover, RT led to the accumulation of proinflammatory macrophages and natural killer T cells with enhanced cytotoxic gene expression signature. In addition, subclusters of dendritic cells and endothelial cells showed decrease in the expression of antigen present features in post-RT samples. More ECM component secreted by myofibroblasts during RT. These findings indicate that RT induced the dynamics of the immune response that should be taken into consideration when designing and clinically implementing innovative multimodal cancer treatment regimens of different RT and immunotherapy approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/523682bb135dcad73fb3cd605aea7d93bedb4b8c" target='_blank'>
              Dynamic single-cell sequencing unveils the tumor microenvironment evolution of gastric cancer abdominal wall metastases during radiotherapy.
              </a>
            </td>
          <td>
            Qianqian Mao, Zhenzhen Wu, Yonghong Lai, Ling Wang, Qiongzhi Zhao, Xi Xu, Xiansheng Lu, Wenjun Qiu, Zhi-Huan Zhang, Jiani Wu, Gaofeng Wang, R. Zhou, Jianhua Wu, Huiying Sun, N. Huang, Xiatong Huang, Luyang Jiang, Yiran Fang, Yuyun Kong, Li Liang, J. Bin, Yulin Liao, Min Shi, Wangjun Liao, D. Zeng
          </td>
          <td>2024-09-26</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) that is refractory to treatment frequently leads to extensive systemic and multi-organ metastases. PDAC is characterized as an immune “cold” cancer; however, the persistence of this “cold” tumor immune microenvironment (TIME) across distant metastases, and the mechanisms by which metastases influence their respective TIME, remain poorly understood. In this study, we conducted spatial protein expression profiling of the TIME using the Nanostring GeoMx Digital Spatial Profiling (DSP) platform. We employed a 43- plex panel on 54 tissue samples, including 14 primary tumors and 40 matched metastases from organs commonly affected by PDAC metastases (liver, lung, and peritoneum), obtained through a rapid (“warm”) autopsy program. We quantified protein expression in three cellular compartments: PanCK+ epithelial cells, α-SMA+ fibroblasts, and CD45+ immune cells, across 181 regions of interest (ROIs). Our analysis revealed a significant enrichment of CD45+ immune cells in lung metastases compared to other organ sites, with a notably lower presence of these cells in liver metastases. Focusing on the immune cell compartment, we analyzed the CD45 segments for further investigation. After normalization using housekeeping proteins, including Histone H3, GAPDH, and ribosomal protein S6, we found that the immune profiles were most distinct between liver and lung metastases. Specifically, liver metastases exhibited the greatest deviation in immune profile compared to primary PDAC, lung, and peritoneal metastases. We scored protein expression signatures for various immune cell subsets based on specific marker expressions. When comparing liver to lung metastases, scores for T cells, myeloid cells, checkpoint markers, macrophages, and interferon signaling were significantly lower in the liver, underscoring the immune “coldness” of the liver metastasis environment. Our cohort included six untreated patients and nine patients treated with chemotherapies. We compared protein expressions in each organ site between untreated and treated patients. Notably, treatment induced divergent expression patterns of multiple proteins between liver and lung metastases. While treatment significantly reduced the expression of several immune markers, including CD45 (total immune), CD3 (T cell), CD4 (T helper cell), CD45RO (memory T cell), CD163 (M2 macrophage), and B7-H3 (checkpoint marker), in lung metastases, these markers were significantly upregulated in liver metastases of treated patients. This finding indicates a pronounced divergence in TIME modulation in response to treatment between liver and lung metastases, potentially fostering a treatment-resistant environment in PDAC patients with multi- organ metastasis. Consequently, this urges the need to develop tailored therapeutic strategies for PDAC metastasis, taking into account the distinct immune landscapes of different metastasis sites.
 Citation Format: Jimin Min, Vittorio Branchi, Kimal I Rajapakshe, Nisha Narayanan, Guangsheng Pei, Linghua Wang, Jean L Grem, Thomas C Caffrey, Paul M Grandgenett, Michael A Hollingsworth, Paola A Guerrero, Anirban Maitra. Organ-specific diversity of tumor immune microenvironment in metastatic pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B073.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd4226ecd000fb900607928bb9c305d2b12ea81" target='_blank'>
              Abstract B073: Organ-specific diversity of tumor immune microenvironment in metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Jimin Min, Vittorio Branchi, K. Rajapakshe, Nisha Narayanan, Guangsheng Pei, Linghua Wang, Jean L. Grem, T. Caffrey, P. Grandgenett, Michael A Hollingsworth, Paola A Guerrero, Anirban Maitra
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a68f84c9fb8e36a5e9dc6d992ffaa775efbb85db" target='_blank'>
              SMAC mimetic drives microglia phenotype and glioblastoma immune microenvironment
              </a>
            </td>
          <td>
            E. Snacel-Fazy, A. Soubéran, M. Grange, Kevin Joseph, C. Colin, Philippe Morando, H. Luche, A. Pagano, S. Brustlein, F. Debarbieux, Soline Toutain, C. Siret, S. A. van de Pavert, G. Rougon, D. Figarella-Branger, V. Ravi, E. Tabouret, A. Tchoghandjian
          </td>
          <td>2024-09-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cancer is characterized by uncontrolled cell growth and spread into surrounding tissue. According to the North American Association of Central Cancer Registries (NAACCR) and the National Center for Health Statistics, there were 1,918,030 cancer cases and a total of 609,360 deaths estimated in the US in 2022. Although the direct cause of cancer is not fully understood, it is recognized that the immune system plays a major role in the development and progression of tumors. Investigating tumor-associated macrophages (TAMs) is crucial for understanding the immunosuppressive role that they may play in the tumor microenvironment, which makes them a target for immunotherapies to combat cancer. However, angiogenesis, tumor initiation and invasion promoting signals prevent gaining deeper insight into the intricate molecular mechanisms involved in TAMs and the tumor microenvironment. While researchers in the past have used population samples to analyze cells, recent efforts have incorporated single-cell technologies to study the complex mixture of cells in a tumor, monitor the effects of immunotherapies and more. With the revolutionary advent of the first single-cell RNA sequencing (scRNA-seq), researchers have since been able to gain insight into single-cell data with unprecedented accuracy and develop new immunotherapies through different single-cell technologies, such as polymethylsiloxane (PDMS) and Multiplexed Ion Beam Imaging (MIBI) multiplexing. In this review, we underline the benefits of adopting single-cell technologies to identify distinct cell types in complex environments (i.e., tumors) and highlight the potential application of these technologies along with various multiplexing techniques to immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64f7bdccd7146f521a4c249388cdbf8225adf0ff" target='_blank'>
              A Review on Single-Cell analysis of Macrophages in Cancer
              </a>
            </td>
          <td>
            Isaiah Yim, Amirali Assylkhan
          </td>
          <td>2024-09-17</td>
          <td>STEM Fellowship Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0970058b2af0e5ea082fb819477f1708b4c4db9a" target='_blank'>
              DNA demethylation triggers cell free DNA release in colorectal cancer cells
              </a>
            </td>
          <td>
            V. Pessei, M. Macagno, Elisa Mariella, Noemi Congiusta, V. Battaglieri, Paolo Battuello, Marco Viviani, Giulia Gionfriddo, S. Lamba, Annalisa Lorenzato, D. Oddo, Fariha Idrees, Alessandro Cavaliere, A. Bartolini, S. Guarrera, M. Linnebacher, Laura Monteonofrio, Luca Cardone, Michele Milella, A. Bertotti, Silvia Soddu, Elena Grassi, G. Crisafulli, A. Bardelli, L. Barault, F. Di Nicolantonio
          </td>
          <td>2024-10-09</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Tumor‐associatedmacrophages (TAMs) exhibit remarkable heterogeneity in glioblastoma. Spatially resolved single‐cell transcriptomic studies identified a monocyte‐derived TAM subset localized in the peri‐necrotic niche, driven by hypoxic cues to acquire ahypoxia response signature. These hypoxia‐TAMs destabilize endothelial adherens junctions through adrenomedullin paracrine signaling, promoting the formation of hyperpermeable neovasculature that impedes drug delivery. Blocking adrenomedullin produced by hypoxia‐TAMs restores vascular integrity, increases drug deliveryinto tumors, and provides combinatorial therapeutic benefits. Here we discuss the heterogeneity of TAMs regarding functional states and locations in glioblastomas, and propose future directions for studying the temporospatial dynamics of multifaceted TAM. Highlights Single‐cell omics reveal a functionally and spatially distinct hypoxia‐TAM subset in glioblastoma. Adrenomedullin secreted by hypoxia‐TAM destabilizes tumor vasculature and its blockade enhances vessel integrity and drug delivery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989fdc1e11579a80529898e9b0f1ec0aa652779e" target='_blank'>
              Identification of hypoxic macrophages in glioblastoma: Unveiling therapeutic insights from tumour microenvironment analysis
              </a>
            </td>
          <td>
            Zhen Qin, Xiu-Wu Bian, Yu Shi
          </td>
          <td>2024-09-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ff9444b58ab2430b3356815749e5a620be0ebab" target='_blank'>
              Tumor-associated neutrophils upregulate Nectin2 expression, creating the immunosuppressive microenvironment in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            H. Luo, Naoki Ikenaga, Kohei Nakata, Nobuhiro Higashijima, Pingshan Zhong, Akihiro Kubo, Chenyi Wu, C. Tsutsumi, Yukihiko Shimada, Masataka Hayashi, K. Oyama, S. Date, Toshiya Abe, Noboru Ideno, C. Iwamoto, Koji Shindo, K. Ohuchida, Yoshinao Oda, Masafumi Nakamura
          </td>
          <td>2024-09-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background Medulloblastoma (MB) is a pediatric brain tumor that develops in the cerebellum, representing one of the most common malignant brain cancers in children. Standard treatment includes surgery, chemotherapy, and radiation, but despite a 5-year survival rate of approximately 70%, these therapies often lead to significant neurological damage in the developing brain. This underscores the urgent need for less toxic, more effective therapeutic alternatives. Recent advancements in cancer immunotherapy, including immune checkpoint inhibitors and CAR-T cell therapy, have revolutionized cancer treatment. One promising avenue is the use of Gamma Delta (γδ) T cells, a unique T cell population with potential advantages such as non-alloreactivity, potent tumor cell lysis, and broad antigen recognition. However, their capacity to recognize and target MB cells remains underexplored. Methods To investigate the therapeutic potential of γδT cells against MB, we analyzed the proportion and status of MB-infiltrated γδ T cells within patient datasets. We next investigated the expression of γδ T cell ligands on MB cells. Finally, we generated various human γδ T cell subpopulations and evaluated their activation potential and cytotoxicity against a panel of MB cell lines and patient derived cells. Results Our findings identified Ephrin-A2 receptor and the phosphoantigen/Butyrophilin complex as key ligands, activating Vγ9Vδ1 and Vγ9Vδ2 T cells, respectively, leading to significant MB cell lysis in both monolayer and spheroid models. Importantly, preliminary safety data showed that γδT cells did not target differentiated neurons or neuroepithelial stem cells derived from induced pluripotent stem cells, underscoring the selectivity and safety of this approach. Conclusion In conclusion, γδT cells trigger an efficient and specific killing of MB, and would offer a promising novel therapeutic strategy, with potential for broader application to other pediatric brain tumors. Key messages Medulloblastoma patients often experience significant long-term side effects from current standard treatments. Immunotherapy has emerged as a promising alternative to conventional therapeutic approaches. In our study, we demonstrated that γδ T cells can efficiently and specifically target medulloblastoma cells without causing harm to healthy neuronal tissue. These findings suggest that γδ T cell therapy may provide therapeutic benefits while potentially reducing treatment-related toxicity in medulloblastoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7728d8e6eb2e9fe32658d85f902861981a9e4d1e" target='_blank'>
              Ephrin-A2 and Phosphoantigen-Mediated Selective Killing of Medulloblastoma by γδT Cells Preserves Neuronal and Stem Cell Integrity
              </a>
            </td>
          <td>
            Lola Boutin, Mingzhi Liu, Julie Déchanet Merville, Oscar Bedoya-Reina, M. Wilhelm
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) patients with tumors enriched for the basal-like molecular subtype exhibit severe resistance to standard of care treatments and have significantly worse overall survival, compared to patients with classical subtype enriched tumors. It is therefore important to develop resources that allow identification of novel targets in a statistically rigorous way to overcome therapeutic challenges. Here we compiled a single cell RNA sequencing (scRNAseq) atlas of the human pancreas with 229 individual patient samples, aggregated from raw data obtained from published studies. We mapped cell-type specific scRNAseq gene signatures in newly generated (n=12) and existing (n=13) spatial transcriptomics (ST) data from PDAC samples. Analysis of tumor cells from our scRNAseq atlas revealed nine transcriptionally distinct populations, two of which strongly align with a basal gene signature, correlating with worse overall survival in PDAC bulk RNAseq datasets. Deconvolution showed one of the basal populations identified to be the predominant tumor subtype in non- dissociated ST tissues and in vitro tumor cell and organoid PDAC lines. In bulk RNAseq datasets, we uncovered that myofibroblasts (myFb) expressing CXCL10 consistently correlated with both basal subtypes and were found near basal tumor cells using immunostaining. We also identified that besides immune cells, ductal cells also express CXCR3, the receptor for CXCL10, further suggesting a relationship between these cell types in PDAC tumor microenvironment (TME). Overall, we show that our newly built scRNAseq atlas, containing over 700,000 cells, integrated with ST data is a powerful resource, given that it corroborates numerous PDAC TME observations in the literature, with increased statistical power. Moreover, we uncovered a novel association between CXCL10+ myFb and basal tumor cells, underscoring the presence of unique spatial niches in PDAC that should be explored for the development of targeted therapies for this dismal disease.
 Citation Format: Ian M Loveless, Samantha B Kempt, Yuesong Wu, Daniel J Salas-Escabillas, Kailee Hartway, Madison George, Yetunde Makinwa, Thais Stockdale, Samuel D Zwernik, Kendyll Gartrelle, Rohit G Reddy, Daniel Long, Allison Wombell, Julie M Clark, Albert M Levin, David Kwon, Ling Huang, Ralph Francescone, Débora B Vendramini-Costa, Ben Stanger, Adam Alessio, Andrew M Waters, Yuehua Cui, Brian Theisen, Howard C Crawford, Nina G Steele. Human pancreatic cancer single cell atlas reveals association of CXCL10 positive fibroblasts and basal subtype tumor cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B068.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb2cdcff90781c8782f8a20266c95fd99d51d7f6" target='_blank'>
              Abstract B068: Human pancreatic cancer single cell atlas reveals association of CXCL10 positive fibroblasts and basal subtype tumor cells
              </a>
            </td>
          <td>
            Ian Loveless, Samantha B Kempt, Yuesong Wu, Daniel J. Salas-Escabillas, Kailee Hartway, Madison George, Yetunde Makinwa, Thais Stockdale, Samuel Zwernik, K. Gartrelle, Rohit G Reddy, Daniel W Long, Allison Wombell, Julie Clark, Albert Levin, David Kwon, Ling Huang, R. Francescone, D. Vendramini-Costa, Ben Z. Stanger, Adam Alessio, Andrew M. Waters, Yuehua Cui, Brian Theisen, Howard C. Crawford, Nina G. Steele
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Background Major histocompatibility complex class I (MHC-I) loss is frequent in non-small cell lung cancer (NSCLC) rendering tumor cells resistant to T cell lysis. NK cells kill MHC-I-deficient tumor cells, and although previous work indicated their presence at NSCLC margins, they were functionally impaired. Within, we evaluated whether NK cell and CD8 T cell infiltration and activation vary with MHC-I expression. Methods We used single-stain immunohistochemistry (IHC) and Kaplan-Meier analysis to test the effect of NK cell and CD8 T cell infiltration on overall and disease-free survival. To delineate immune covariates of MHC-I-disparate lung cancers, we used multiplexed immunofluorescence (mIF) imaging followed by multivariate statistical modeling. To identify differences in infiltration and intercellular communication between IFNγ-activated and non-activated lymphocytes, we developed a computational pipeline to enumerate single-cell neighborhoods from mIF images followed by multivariate discriminant analysis. Results Spatial quantitation of tumor cell MHC-I expression revealed intratumoral and intertumoral heterogeneity, which was associated with the local lymphocyte landscape. IHC analysis revealed that high CD56+ cell numbers in patient tumors were positively associated with disease-free survival (HR=0.58, p=0.064) and overall survival (OS) (HR=0.496, p=0.041). The OS association strengthened with high counts of both CD56+ and CD8+ cells (HR=0.199, p<1×10−3). mIF imaging and multivariate discriminant analysis revealed enrichment of both CD3+CD8+ T cells and CD3–CD56+ NK cells in MHC-I-bearing tumors (p<0.05). To infer associations of functional cell states and local cell–cell communication, we analyzed spatial single-cell neighborhood profiles to delineate the cellular environments of IFNγ+/– NK cells and T cells. We discovered that both IFNγ+ NK and CD8 T cells were more frequently associated with other IFNγ+ lymphocytes in comparison to IFNγ– NK cells and CD8 T cells (p<1×10–30). Moreover, IFNγ+ lymphocytes were most often found clustered near MHC-I+ tumor cells. Conclusions Tumor-infiltrating NK cells and CD8 T cells jointly affected control of NSCLC tumor progression. Coassociation of NK and CD8 T cells was most evident in MHC-I-bearing tumors, especially in the presence of IFNγ. Frequent colocalization of IFNγ+ NK cells with other IFNγ+ lymphocytes in near-neighbor analysis suggests NSCLC lymphocyte activation is coordinately regulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bffb606e7f7747c3416045b156baec2a760c44c" target='_blank'>
              Spatial colocalization and combined survival benefit of natural killer and CD8 T cells despite profound MHC class I loss in non-small cell lung cancer
              </a>
            </td>
          <td>
            Remziye E Wessel, Nardin Ageeb, Joseph M. Obeid, Ileana S Mauldin, Kate A. Goundry, Gabriel F. Hanson, Mahdin Hossain, Chad Lehman, R. Gentzler, Nolan A Wages, C. Slingluff, T. Bullock, Sepideh Dolatshahi, Michael G. Brown
          </td>
          <td>2024-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befebce4fc2c4b9cfeedafc2e4927bdb0544dbd9" target='_blank'>
              Mechanisms that clear mutations drive field cancerization in mammary tissue
              </a>
            </td>
          <td>
            Marta Ciwinska, H. Messal, Hristina R Hristova, C. Lutz, Laura Bornes, Theofilos Chalkiadakis, Rolf Harkes, Nathalia S M Langedijk, S. Hutten, Renée X Menezes, Jos Jonkers, S. Prekovic, Ben Simons, C. Scheele, J. van Rheenen
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>2</td>
          <td>39</td>
        </tr>

        <tr id="Background: Gastrointestinal cancer cells display both morphology and physiology diversity, thus posing a significant challenge for precise representation by a single data model. We conducted an in-depth study of gastrointestinal cancer heterogeneity by integrating and analyzing data from multiple modalities. Methods: We used a modified Canny algorithm to identify edges from tumor images, capturing intricate nonlinear interactions between pixels. These edge features were then combined with differentially expressed mRNA, miRNA, and immune cell data. Before data integration, we used the K-medoids algorithm to pre-cluster individual data types. The results of pre-clustering were used to construct the kernel matrix. Finally, we applied spectral clustering to the fusion matrix to identify different tumor subtypes. Furthermore, we identified hub genes linked to these subtypes and their biological roles through the application of Weighted Gene Co-expression Network Analysis (WGCNA) and Gene Ontology (GO) enrichment analysis. Results: Our investigation categorized patients into three distinct tumor subtypes and pinpointed hub genes associated with each. Genes MAGI2-AS3, MALAT1, and SPARC were identified as having a differential impact on the metastatic and invasive capabilities of cancer cells. Conclusion: By harnessing multimodal features, our study enhances the understanding of gastrointestinal tumor heterogeneity and identifies biomarkers for personalized medicine and targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279c68f2f427231b32eddf4811a28d710e5bb1cf" target='_blank'>
              Tumor Heterogeneity in Gastrointestinal Cancer Based on Multimodal Data Analysis
              </a>
            </td>
          <td>
            Dongmei Ai, Yang Du, Hongyu Duan, Juan Qi, Yuduo Wang
          </td>
          <td>2024-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e78446518da6c71cd38b60ad342558f8d623b65" target='_blank'>
              Medullary thyroid cancer: single-cell transcriptome and tumor evolution
              </a>
            </td>
          <td>
            Li-feng Wang, Wen-wen Zhou, Fang Yuan, Kai-wen Fu, Yongpeng He, Rui Chen
          </td>
          <td>2024-08-30</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Introduction: A complex immunosuppressive tumor microenvironment (TME) is a distinctive feature of pancreatic ductal adenocarcinoma (PDAC). The TME consists of an admixture of cell types including epithelial, endothelial, immune cells of both myeloid and lymphoid origin, and cancer-associated fibroblasts (CAFs). The traditionally defined subtypes of PDAC tumor cells include classical, associated with the retention of epithelial markers and a better prognosis, and basal, which is characterized by a quasi-mesenchymal phenotype with poorer responses to chemotherapy and worse overall survival. Additionally, CAF subtypes are associated with specific roles within the TME. These include inflammatory (iCAF), with the capacity to secrete IL-6, myofibroblastic (myCAF), defined by high levels of alpha smooth muscle actin (αSMA) expression, and antigen presenting (apCAF), defined by expression of HLA-DR and antigen processing and presentation capacity. We hypothesized that interactions between CAFs and tumor cells altered tumor cell phenotype and could be delineated. Methods: Using multi-omics platforms, we evaluated the impact of CAF populations on tumor cell phenotype. Our approach integrated multiplex immunohistochemistry, imaging mass cytometry (IMC) and RNA-scope of patient tumor tissues with functional in vitro studies from matched patient-derived organoid (PDO) and CAF cocultures. The dynamic impact of CAFs on PDOs was examined using a combination of bulk RNA sequencing, multiparametric flow cytometry and secretome analysis, in addition to RT-qPCR and western blot comparing cocultures to matched PDO monocultures. Results: IMC of 15 patient tumors revealed distinct neighborhoods of CAFs and tumor cells. iCAFs were in close proximity to classical tumor cells, while myCAFs were in close proximity to basal tumor cells. Further, in our PDO-CAF co-cultures the PDOs underwent a shift from classical to basal subtype in 42% of co-cultures, a phenomenon also inferred in patient tissues by associating collagen content with classical or basal designation. We also captured a tumor cell population expressing both basal and classical markers that may represent a transition state. The shift from classical to basal tumor cell subtype was accompanied by enhanced epithelial-mesenchymal transition and a loss of IL-8 secretion. Interestingly, when we examined immune cell distribution within tumor tissues, T cells were both the most proximal and the most abundant immune cell type in relation to tumor cells. Moreover, the highest frequency of total and activated T cells was seen in close association with basal tumor cells. Conclusions: Our study underscores the importance of understanding the complex interplay between CAFs and tumor cells in PDAC. Further, we identify a relationship between T cells and tumor cell subtypes. By leveraging high-dimensional data and patient-derived models, we provide new insights into the spatial and functional dynamics of the PDAC TME, paving the way for novel therapeutic strategies to reprogram the TME and enhance patient survival.
 Citation Format: Jacquelyn W Zimmerman, Samantha Guinn, Mili Ramani, Brayan Perez, Joseph A Tandurella, Daniel J Zabransky, Jae W Lee, Emma Kartalia, Jignasha Patel, Sarah Shin, Alexei Hernandez, Courtney Cannon, Nicole Gross, Soren Charmsaz, Elana J Fertig, Richard A Burkhart, Won J Ho, Elizabeth M Jaffee. Cancer associated fibroblasts drive classical to basal change associated with increased T cell presence in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A032.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b1b3de60c255399613042a94100d74cd1acaab3" target='_blank'>
              Abstract A032: Cancer associated fibroblasts drive classical to basal change associated with increased T cell presence in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jacquelyn W. Zimmerman, Samantha Guinn, Mili Ramani, Brayan Perez, Joseph A Tandurella, Daniel J. Zabransky, Jae W. Lee, Emma Kartalia, Jignasha Patel, Sarah M. Shin, Alexei Hernandez, Courtney D Cannon, Nicole E. Gross, Soren Charmsaz, E. Fertig, Richard A Burkhart, W. Ho, Elizabeth M. Jaffee
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="


 Glioblastoma trigger systemic immune responses, resulting in elevated white blood cell counts and elevated temperatures, as the body attempts to combat tumor cells. However, glioblastoma reprograms immune cells to evade immune-mediated destruction and promote tumor growth. While targeted immunotherapies have shown promise in some cancers like melanoma, they have not been efficacious for malignant brain tumors, leading to significant societal costs due to high morbidity and mortality rates.



 Understanding immune cell trafficking is crucial for developing effective immunotherapies. We conducted spatially resolved single-cell analyses to investigate immune cell entry into glioblastoma. Integrating these data with multi-omic sequencing analyses of anatomically dissected specimens from glioblastoma-affected human brain border regions and tumor cores revealed distinct paths of immune cell trafficking into glioblastoma.



 One path involves an ordered entry of mononuclear cells through the subarachnoid and perivascular spaces, guided by chemotactic cues. Cells in these regions display markers of immune activation, suggesting immune priming. Another path occurs at sites of vascular leakage due to hypoxia and necrosis, where a mix of mononuclear cells and neutrophils accumulate. These cells exhibit immunosuppressive transcriptomic programs, regulated by the phagocytosis-inducing transcription factor MITF and downstream metabolic changes controlled by PPAR-gamma. Ongoing analyses aim to identify pharmacological agents that can modulate immune cell reprogramming at hypoxic niches.



 In summary, we have identified distinct modes of immune cell trafficking in glioblastoma. Chemotaxis-mediated trafficking via the brain border regions appears to prime immune cells, while entry at hypoxic niches is associated with immune suppression. Pharmacological agents shielding immune cells from immunosuppressive molecular cues in the tumor microenvironment show promise for glioblastoma immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8f5d39f4dab2e338bbcd19df4cc3dd4650f4fd" target='_blank'>
              P02.20.A NICHE-DEPENDENT MODES OF IMMUNE CELL TRAFFICKING IN GLIOBLASTOMA OFFER TARGETS FOR NOVEL IMMUNOTHERAPIES
              </a>
            </td>
          <td>
            R. Sankowski, A. Dalmau Gasull, M. Prinz
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Despite the growing insights into the cellular heterogeneity of pediatric brain tumors, the transcriptional changes and spatial reorganization during disease progression remain largely unexplored. Here, we present an in-depth single-cell and 10X Xenium spatial transcriptomic characterization of 24 matched pediatric high-grade gliomas (pHGGs) in children, adolescents and young adults (AYA) profiled at both diagnosis and recurrence - constituting a total of 55 tissue samples. By stratifying the transcriptional programs activated at recurrence, we identify two distinct global response sets. Hemispheric AYA tumors predominantly exhibit upregulation of extracellular matrix pathways at recurrence, whereas midline tumors preferentially activate generic stress response programs. On a cellular level, we find that heterogeneity within the malignant tumor compartment remains largely unchanged at recurrence. However, a subset of AYA patients with hemispheric pHGG shows enrichment of the neural-progenitor-like (NPC-like) tumor cell state, and compositional changes in the immune and normal cell landscape, characterized by a global decrease in myeloid cells and an increase in oligodendrocyte fractions. Those changes lead, in turn, to enhanced NPC-like tumor cell-to-oligodendrocyte interactions. Spatial analysis of our extensive cohort at unprecedented subcellular resolution further reveals distinct cellular and spatial features associated with tumor progression, particularly in the remodeling of cell-cell interactions and tumor cell neighborhoods. Overall, this study provides a comprehensive longitudinal single-cell and spatial atlas of pHGG, uncovering the extensive cellular heterogeneity associated with tumor progression. Our findings highlight the role of intrinsic and extrinsic tumor adaptations in shaping disease progression in pHGG, offering potential novel therapeutic targets.
 Citation Format: Sara G. Danielli, Sina Neyazi, Olivia A. Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, Johannes Gojo, Keith L. Ligon, Lissa Baird, Sanda Alexandrescu, Jennifer Cotter, Michael Prados, Adam Resnick, Lin Wang, Michelle Monje, Aaron Diaz, Mariella G. Filbin. Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca1e500481d6ce9fb3ed15507a312915878f86c" target='_blank'>
              Abstract A052 Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults
              </a>
            </td>
          <td>
            Sara G Danielli, S. Neyazi, Olivia A Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob S Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, J. Gojo, Keith L. Ligon, Lissa Baird, S. Alexandrescu, Jennifer Cotter, Michael Prados, Adam C. Resnick, Lin Wang, M. Monje, Aaron A. Diaz, Mariella G. Filbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b7649961c3767077277c2ff33140a98201a7bd6" target='_blank'>
              Characterization and spatial distribution of infiltrating lymphocytes in medullary, and lymphocyte-predominant triple negative breast cancers
              </a>
            </td>
          <td>
            A. Alfaro, C. Catelain, H. El-Masri, P. Rameau, M. Lacroix-Triki, JY. Scoazec, V. Marty, F. Mosele, B. Pistilli
          </td>
          <td>2024-09-14</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de9b939e6665a85e15c06d7919e4ac22bd18c3f" target='_blank'>
              Cellular origin and clonal evolution of human dedifferentiated liposarcoma
              </a>
            </td>
          <td>
            N. Gruel, C. Quignot, Laëtitia Lesage, Sophie El Zein, S. Bonvalot, Dimitri Tzanis, Khadija Ait Rais, Fabien Quinquis, Bastien Manciot, J. Vibert, Nadine El Tannir, A. Dahmani, Héloise Derrien, Didier Decaudin, Ivan Bièche, L. Courtois, O. Mariani, Laetitia K Linares, L. Gayte, Sylvain Baulande, J. Waterfall, O. Delattre, G. Pierron, Sarah Watson
          </td>
          <td>2024-09-12</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>40</td>
        </tr>

        <tr id="


 Glioblastoma (GB) stem cells (GSCs) are described to recapitulate the expression signatures of stem and progenitor cell classes in the neurodevelopmental hierarchy and are implicated in driving therapeutic resistance. However, the heterogeneity of both the GSC subclasses and their expression of stemness markers hinders comprehensive labelling of these cells. Cellular dormancy is a key feature of GSCs that can be captured by label retention method to track slowly dividing cells. We used CellTrace (CT) dye to identify long term label retaining GB cells (LRGC) in orthotopic xenograft model. We analysed the distribution of LRGC versus the tumour bulk in histology. Additionally, we analysed the transcriptional signature by FAC-sorting LGRC for single cell RNA sequencing. Further, we demonstrated that LRCs survived through temozolomide (TMZ) treatment in vivo and re-initiated tumour proliferation in a faster pace than other surviving glioma cells, signifying its role in therapeutic relapse.



 To label LRGC, patient derived GB cell lines were stained with CT dye before transplantation into the striatum of immunodeficient mice. After 6 weeks of growth the brains were cryosectioned, histologically stained and imaged via confocal microscopy, followed by histologic AI assisted analysis with the AIVIA software. LRGC and non-label retaining tumor cells were FAC-sorted for single-cell RNA sequencing (SMART seq2). To explore the role of LGRC in therapeutic resistance, tumors were treated with TMZ. Re-initiation of tumor growth was tracked by subcutaneous surgical implantation of BrdU pumps at different timepoints after therapy. The proliferation patterns of LRGC versus other surviving tumor cells were analysed in histology again using an AI assisted software.



 LRGC could be identified in histology 6 weeks after tumor growth. LRGC were found near the centre of tumor but rarely at the invasion front. Single cell RNA-seq of LRGC showed differentially expressed pathways related to lipid metabolism and neurodevelopment. Re-initiation of tumor growth started 12-14 days after therapy. LRGC showed a more intense proliferation compared to other surviving tumor cells, indicating the ability of LRGC to not only survive TMZ treatment, but also drive GB relapse.



 LRGC could be visualized 6-8 weeks after tumor growth in an orthotopic xenograft model. On a transcriptional level these cells share similarities with stem cells in normal brain development but also show distinct features. An important role of LRGC in re-initiation of tumor growth after therapy could be confirmed.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863c1df5d02a6308a5d876cfe8995a6849735071" target='_blank'>
              P03.02.B DECIPHERING THE IDENTITY AND CHARACTERISTICS OF LONG-TERM LABEL RETAINING GLIOBLASTOMA CELLS AND THEIR ROLE IN TUMOUR RELAPSE
              </a>
            </td>
          <td>
            H. Chow, A. Kourtesakis, L. Hai, S. Schaffranek, S. Pusch, D. Hoffmann, D. Reibold, W. Wick, R. Wagener
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is the most common type of malignancy in children. ALL prognosis after initial diagnosis is generally good; however, patients suffering from relapse have a poor outcome. The tumor microenvironment is recognized as an important contributor to relapse, yet the cell-cell interactions involved are complex and difficult to study in traditional experimental models. In the present study, we established an innovative larval zebrafish xenotransplantation model, that allows the analysis of leukemic cells (LCs) within an orthotopic niche using time-lapse microscopic and flow cytometric approaches. LCs homed, engrafted and proliferated within the hematopoietic niche at the time of transplant, the caudal hematopoietic tissue (CHT). A specific dissemination pattern of LCs within the CHT was recorded, as they extravasated over time and formed clusters close to the dorsal aorta. Interactions of LCs with macrophages and endothelial cells could be quantitatively characterized. This zebrafish model will allow the quantitative analysis of LCs in a functional and complex microenvironment, to study mechanisms of niche mediated leukemogenesis, leukemia maintenance and relapse development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93d8b671ae9c63ac39a6096f61f5f4b266ff65" target='_blank'>
              In vivo monitoring of leukemia-niche interactions in a zebrafish xenograft model
              </a>
            </td>
          <td>
            A. Arner, Andreas Ettinger, Bradley Wayne Blaser, Bettina Schmid, I. Jeremias, Nadia Rostam, Vera Binder-Blaser
          </td>
          <td>2024-08-30</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and lethal primary brain tumor in adults and is driven by self-renewing glioblastoma stem cells (GSCs) that persist after therapy and seed treatment refractory recurrent tumors. GBM tumors display a high degree of intra- and inter-tumoral heterogeneity that is a prominent barrier to targeted treatment strategies. This heterogeneity extends to GSCs that exist on a gradient between two transcriptional states or subtypes termed developmental and injury-response. Drug targets for each subtype are needed to effectively target GBM. To identify conserved and subtype-specific genetic dependencies across a large and heterogeneous panel of GSCs, we designed the GBM5K targeted gRNA library and performed fitness screens in a total of 30 patient-derived GSC cultures. The focused CRISPR screens identified the most conserved subtype-specific vulnerabilities in GSCs and elucidated the functional dependency gradient existing between the developmental and injury-response states. Developmental-specific fitness genes were enriched for transcriptional regulators of neurodevelopment, whereas injury-response-specific fitness genes were highlighted by several genes implicated in integrin and focal adhesion signaling. These context-specific vulnerabilities conferred differential sensitivity to inhibitors of β1 integrin, FAK, MEK and OLIG2. Interestingly, the screens revealed that the subtype-specific signaling pathways drive differential cyclin D (CCND1 vs. CCND2) dependencies between subtypes. These data provide biological insight and mechanistic understanding of GBM heterogeneity and point to opportunities for precision targeting of defined GBM and GSC subtypes to tackle heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43d6812955301efd762e7b039babebcb9586d37" target='_blank'>
              Fitness Screens Map State-Specific Glioblastoma Stem Cell Vulnerabilities.
              </a>
            </td>
          <td>
            G. Macleod, Fatemeh Molaei, Shahan Haider, Maira P Almeida, Sichun Lin, M. Kushida, Haresh Sureshkumar, Jasmine K Bhatti, Jack Q Lu, Daniel Schramek, Peter B Dirks, Stéphane Angers
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Combined hepatocellular carcinoma–cholangiocarcinoma (cHCC-CCA) is a challenging primary liver cancer subtype with limited treatment options and a devastating prognosis. Recent studies have underscored the context-dependent roles of SOX9 in liver cancer formation in a preventive manner. Here, we revealed that liver-specific developmental Sox9 elimination using Alb-Cre;Sox9(flox/flox) (LKO) and CRISPR/Cas9-based tumor-specific acute Sox9 elimination (CKO) in SB-HDTVI-based Akt-YAP1 (AY) and Akt-NRAS (AN) cHCC-CCA models showed contrasting responses. LKO abrogates the AY CCA region while stimulating poorly differentiated HCC proliferation, whereas CKO prevents AY and AN cHCC-CCA development irrespective of tumor cell fate. Additionally, AN, but not AY, tumor formation partially depends on the Sox9-Dnmt1 cascade. SOX9 is dispensable for AY-mediated, HC-derived, LPC-like immature CCA formation but is required for their maintenance and transformation into mature CCA. Therapeutic Sox9 elimination using the OPN-CreERT2 strain combined with inducible Sox9 iKO specifically reduces AY but not AN cHCC-CCA tumors. This necessitates the careful consideration of genetic liver cancer studies using developmental Cre and somatic mutants, particularly for genes involved in liver development. Our findings suggest that SOX9 elimination may hold promise as a therapeutic approach for a subset of cHCC-CCA and highlight the need for further investigation to translate these preclinical insights into personalized clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/100cbf992f4ca1fca4c932c3b496484d8626cfa3" target='_blank'>
              Context-Dependent Distinct Roles of SOX9 in Combined Hepatocellular Carcinoma–Cholangiocarcinoma
              </a>
            </td>
          <td>
            Yoojeong Park, Shikai Hu, Minwook Kim, Michael Oertel, A. Singhi, S. Monga, Silvia Liu, Sungjin Ko
          </td>
          <td>2024-08-29</td>
          <td>Cells</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Liver cancer represents a substantial global health challenge, contributing significantly to worldwide morbidity and mortality. It has long been understood that tumors are not composed solely of cancerous cells, but also include a variety of normal cells within their structure. These tumor-associated normal cells encompass vascular endothelial cells, fibroblasts, and various inflammatory cells, including neutrophils, monocytes, macrophages, mast cells, eosinophils, and lymphocytes. Additionally, tumor cells engage in complex interactions with stromal cells and elements of the extracellular matrix (ECM). Initially, the components of what is now known as the tumor microenvironment (TME) were thought to be passive bystanders in the processes of tumor proliferation and local invasion. However, recent research has significantly advanced our understanding of the TME’s active role in tumor growth and metastasis. Tumor progression is now known to be driven by an intricate imbalance of positive and negative regulatory signals, primarily influenced by specific growth factors produced by both inflammatory and neoplastic cells. This review article explores the latest developments and future directions in understanding how the TME modulates liver cancer, with the aim of informing the design of novel therapies that target critical components of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d21309486c5cd52dddda7197045d0a0b43a28e44" target='_blank'>
              Beyond Cancer Cells: How the Tumor Microenvironment Drives Cancer Progression
              </a>
            </td>
          <td>
            Hussein Sabit, B. Arneth, Shaimaa Abdel-Ghany, Engy F. Madyan, Ashraf H. Ghaleb, Periasamy Selvaraj, Dong M. Shin, Ramireddy Bommireddy, Ahmed Elhashash
          </td>
          <td>2024-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 INTRODUCTION: Immune checkpoint inhibitors (ICI) in metastatic melanoma results in durable responses in ∼50% of patients. Our understanding of the tumour immune microenvironment (TME) features underpinning ICI response or resistance is currently incomplete. METHODS: To address this issue, we explored the TME of melanoma lymph node metastases (MLN mets) at a spatial (n = 32) and a single cell level (n=24). MLN mets were either from ICI-responder and ICI-resistant patients or untreated controls. We used multiplex IHC and our SPIAT algorithm to examine the spatial immune context of the TME, and also analyzed gene expression by RNAseq. Cells were also isolated from fresh tissue before or after ICI and analyzed by single cell RNAseq and TCRseq to derive high dimension data on T cells and their clonality, all other immune and melanoma cells. Finally, we performed cell-cell interactome and neighborhood analyses to reveal which cell and molecular interactions were associated with response. RESULTS: In patients who responded to ICI, both CD4+ and CD8+ tissue resident memory (TRM) T cell subset density were significantly increased within the tumor, and this correlated with closer distance to melanoma cells. (p<0.05). Single cell data revealed multiple clusters of immune cells (T, B, NK, Mø, DC) and melanoma cells. CD8+ T cell clustering revealed heterogeneous differentiation states. In ICI responders, we observed enrichment for TRM with an earlier transcriptional profile (TCF1+ Tpex.TRM and early TEM.TRM) and lower expression of the exhausted T cell (Texh) gene signature (p<0.05). Hyperexpanded CD8+ T cell clones were present in the TRM cell cluster and were enriched in the responder patients but not in those with resistant disease. CD4+ T cells showed both TRM and TFH clusters with shared expression of multiple genes, with CD4+ TRM also expressed a cytotoxicity gene module. Additionally, Mø and melanoma cells in resistant disease had significantly higher IFNg and IFN-I response gene expression levels compared to tumors from responders (p<0.05), suggesting that chronic activation of T cells in LN mets could lead to the observed Texh phenotype. Interactome analysis displayed putative interactions between CD4+ and CD8+ TRM via the CCL4-CCR5 axis in the ICI responders. This was confirmed by spatial analysis of neighborhood archetypes, which showed significantly increased cell-cell contact between both TRM subsets within the tumor of responders (p<0.05). Finally, conventional dendritic cells interact with both CD8+ and CD4+ TRM subsets in the tumor of responders, providing key signals for maintenance of the TRM pool within the tumor. CONCLUSIONS: MLN mets that respond to ICI are characterized by increased TRM comprising hyperexpanded TCR clones and complex TRM-containing neighborhood archetypes. Whilst these tumors show increased global inflammation, at a single cell level the resistant tumors have increased Texh and greater expression of IFN-regulated genes in melanoma cells and macrophages suggesting ineffective response to chronic antigen stimulation.
 Citation Format: Andrea Di Pietro, Lewis Au, Patrick Crock, Niko Thio, Angela Pizzolla, Marina Bacac, Ramona Schlenker, Aparna Rao, L Spain, David Gyorki, Shahneen Sandhu, Grant McArthur, Paul J Neeson. Tissue resident memory (TRM) T cells and dendritic cells form an in situ archetype for improved response to immune checkpoint therapy in metastatic melanoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e276b4c90b1ba50a0922e85cf9d1021a42b8e15c" target='_blank'>
              Abstract A006: Tissue resident memory (TRM) T cells and dendritic cells form an in situ archetype for improved response to immune checkpoint therapy in metastatic melanoma
              </a>
            </td>
          <td>
            Andrea Di Pietro, Lewis Au, Patrick Crock, N. Thio, Angela Pizzolla, M. Bacac, R. Schlenker, Aparna Rao, L. Spain, David Gyorki, S. Sandhu, Grant McArthur, Paul J Neeson
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7721878e5f49189d68fbf5ffadef4b78f908c349" target='_blank'>
              Deciphering the role of LGALS2: insights into tertiary lymphoid structure-associated dendritic cell activation and immunotherapeutic potential in breast cancer patients
              </a>
            </td>
          <td>
            Shuyu Li, Nan Zhang, Hao Zhang, Zhifang Yang, Q. Cheng, Kang Wei, Meng Zhou, Chen-shen Huang
          </td>
          <td>2024-09-30</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Serrated lesions are common precancerous pathways in colorectal cancer (CRC), but the process by which they progress to malignancy remains unclear. We aimed to elucidate this progression through a single-cell RNA landscape. We conducted single-cell RNA sequencing on three normal colonic tissues and fifteen SLs (including HPs, SSLs, SSLD, and TSAs) and integrated these data with datasets containing tumor samples. We identified three invasive malignant epithelial cell subtypes related to CRC progression: SLC1, SLC2, and tumor cell. SLC1, specific to SSLs, is involved in cell proliferation and shows a continuum of malignancy in gene expression. TSA-specific SLC2 exhibited FOXQ1 upregulation and active EMT, indicating invasiveness. The trajectory analysis showed that HPs do not progress to cancer, and different SL types are linked to the MSI status of advanced CRCs. We validated molecular drivers in premalignant lesions and later carcinogenesis. In the tumor microenvironment, CAF and pre-CAF fibroblast subtypes associated with progression were identified. During the premalignant stage, SLC1 triggered CD8+ T cell responses, while at the advanced stage, CAFs promoted tumor invasion and metastasis via FN1-CD44, influencing tumor progression and the treatment response. Our findings highlight transcriptional changes across serrated pathway stages, aiding in early CRC diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f52080cc10a19d5a436d2837e1d1e6040efb30" target='_blank'>
              Single-Cell Transcriptomics Reveals Cellular Heterogeneity and Drivers in Serrated Pathway-Driven Colorectal Cancer Progression
              </a>
            </td>
          <td>
            Jiahui Wang, Yu Zhang, Xinyi Chen, Qi Sheng, Junluo Yang, Yuyao Zhu, Yuhang Wang, Fangrong Yan, Jingya Fang
          </td>
          <td>2024-10-11</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f00e386abbf367932af2ed3eee8d073da659417c" target='_blank'>
              Multi-stage mechanisms of tumor metastasis and therapeutic strategies
              </a>
            </td>
          <td>
            Zaoqu Liu, Jingqi Chen, Yuqing Ren, Shutong Liu, Yuhao Ba, Anning Zuo, Peng Luo, Q. Cheng, Hui-Xian Xu, Xinwei Han
          </td>
          <td>2024-10-11</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 KRAS mutations are common in non-small cell lung cancer (NSCLC) and often co-occur with TP53 mutations. Although the KRAS oncogene was once considered ‘undruggable,’ FDA approval of KRASG12C inhibitors and promising pre-clinical data from KRASG12D inhibitors highlight the potential for targeted therapies. The tumor microenvironment (TME) of NSCLC comprises immune and stromal cells, including antigen-presenting cancer-associated fibroblasts (for simplicity, hereafter ‘apCAFs,’ regardless of cancer association), a population that expresses markers that interact with and affect T cell activation, exhaustion and differentiation. However, the role of apCAFs in tumorigenesis, how they interact with T cells, and their response to KRAS inhibitors remain unknown. A crucial need exists to characterize apCAFs to optimize therapeutic approaches. We used an inducible and reversible KrasG12D lung cancer mouse model (L-iKras) to perform single cell RNA sequencing analyses. Lung tissue from two each (1 male/1 female) of control mice, mice with KrasG12D turned on with doxycycline (ON), and mice with KrasG12D turned on for 20 weeks and off for 1 week via doxycycline withdrawal (OFF) were included. Fibroblasts were clustered and stratified using the Seurat package. One population resembled apCAFs expressing Pdgfra, Clec3b, Col1a1, and Col1a2 and antigen-presenting markers H2-Ab1, H2-Aa, and Cd74. This population increased in ON and OFF groups compared to controls and expressed genes required for three signals of T cell activation: engagement of the T cell receptor with the peptide- MHC complex of an antigen-presenting cell (APC); binding of costimulatory proteins CD28 or CTLA4 (T cell) with CD80/CD86 (APC); and secreted cytokines (APC) initiating T cell differentiation. This apCAF population showed increased markers for immune checkpoints Cd274 (PDL1) and Cd86, which can inhibit or exhaust T cells, upon KRAS inhibition. Further, these apCAFs upregulated markers involved with the induction of Tregs upon KRAS inhibition (Tgfb1). Treg populations were increased, and CD8 T cell populations were decreased in the OFF group compared to the ON group. These findings suggest that apCAFs are induced in the TME during tumorigenesis and interact with tumor-infiltrating T cells. Further, we hypothesize that apCAFs become immunosuppressive upon KrasG12D inhibition, activate Tregs, and induce CD8 T cell exhaustion. Interestingly, upregulation of immune checkpoints in apCAFs suggests that these cells may respond well to checkpoint therapies, which could be harnessed for future therapeutic approaches in NSCLC. The focus of our future studies will be to validate our findings in an orthotopic lung cancer mouse model developed by our lab, which utilizes KrasG12D-inducible and reversible lung cancer cell lines derived from L- iKras. With this model we will further characterize apCAF-T cell interactions, investigate the impact of KRAS inhibitors on apCAFs, and identify the role of apCAFs in tumor responses to combinations of KRAS inhibitors and checkpoint inhibitors.
 Citation Format: Rachael Hinshaw, Carlos Espinoza, Yaqing Zhang, Marina Pasca di Magliano, Stefanie Galban. Investigating the role of antigen-presenting fibroblasts in the tumor microenvironment of non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr B019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beac9e00fd1ced3575c16790fafc36abcc21b523" target='_blank'>
              Abstract B019: Investigating the role of antigen-presenting fibroblasts in the tumor microenvironment of non-small cell lung cancer
              </a>
            </td>
          <td>
            Rachael Hinshaw, Carlos E Espinoza, Yaqing Zhang, M. Pasca di Magliano, Stefanie Galbán
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Abstract Background Cell death modulates the biological behaviors of tumors. However, the comprehensive role of the multiple forms of cell death in lower-grade glioma (LGG) is unknown. Methods We collected the transcriptional data of LGG patients from public repositories to comprehensively examine six cell death patterns (autophagy, apoptosis, cuproptosis, necroptosis, ferroptosis, and pyroptosis) in LGG samples and systematically correlated these patterns with patient survival, underlying biological processes, and drug sensitivity using serial bioinformatics analysis, clinical sample validation and in vitro assays. Results We identified and independently validated three reproducible cell death-based clusters associated with distinct clinical outcomes and tumor microenvironment characteristics. The Tumor Immune Dysfunction and Exclusion algorithm was applied for predicting how these three clusters would respond to immune checkpoint blockade (ICB) therapy; we found potential resistance of cluster B to ICB therapy. We also performed drug screening to identify cluster-specific drugs. Furthermore, a scoring system, designated as the CDPM score, was developed to estimate the cell death patterns of patients with LGG; this system could predict both LGG patients’ prognosis and immunotherapy efficacy. By performing multiplex immunofluorescence staining, we validated the correlations of GNAL expression with the molecular and clinical features of LGG in an independent LGG cohort. Finally, in vitro assays showed that GNAL promoted apoptosis and inhibited the proliferation of LGG cells. Conclusion The new cell death-based subtype system indicates several features of LGG biology and reveals novel insights into the use of precision medicine for treating LGG. The CDPM score could be used to predict the immunotherapy response and prognosis of LGG patients; moreover, it could indicate a novel direction for improving LGG management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7766cfe0385ddf0c9af53006d23560398f64acd" target='_blank'>
              An integrative analysis based on multiple cell death patterns identifies an immunosuppressive subtype and establishes a prognostic signature in lower-grade glioma
              </a>
            </td>
          <td>
            Hao Lian, Jiajia Wang, Shan Yan, Kui Chen, Lilun Jin
          </td>
          <td>2024-10-10</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="PURPOSE
Immunotherapies have led to a paradigm shift in the treatment of hepatocellular carcinoma (HCC). Studies have revealed the single-cell catalogues of tumor-infiltrating immune cells and the trajectories of their differentiation. Nevertheless, the spatial distribution of these immune cells with distinct phenotypes in tumor microenvironment and their clinicopathological significance in resectable and unresectable HCCs are still largely unclear.


EXPERIMENTAL DESIGN
We analyzed the spatial dynamics of intratumoral CD4 and CD8 T cells and their association with B and plasma cells using 283 surgically resected HCCs, 58 unresectable HCC samples before combined immunotherapy (atezolizumab plus bevacizumab [Atezo+Bev]), and autopsy specimens from 50 cases of advanced-stage HCCs through multiplex immunohistochemistry combined with transcriptomic and driver gene mutation analysis. Classification based on the spatial dynamics of T and B cell responses (refined immunosubtype) was developed and its clinicopathological significance was analyzed.


RESULTS
We found that stem-like CD4 and CD8 T cells were mainly observed in T cell aggregates. Differentiation of T follicular helper cells were associated with the development of tertiary lymphoid structures, whereas differentiation of CD4 TCXCL13 cells with phenotype resembling T peripheral helper cells were associated with the development of lymphoplasmacytic microenvironment. The refined immunosubtype could predict clinical outcomes of resectable HCC after surgery and unresectable HCC after Atezo+Bev therapy. The immune microenvironment of metastatic lesions tended to reflect those of primary lesions.


CONCLUSIONS
We revealed the spatial dynamics of T and B cell response in HCC, which is closely associated with the clinical outcome after surgical resection or Atezo+Bev therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30a9933c34558056c95ceeec869060a2ac77726f" target='_blank'>
              Spatial Dynamics of T and B Cell Response Predicts Clinical Outcome of Resectable and Unresectable Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Yutaka Kurebayashi, K. Sugimoto, Hanako Tsujikawa, Kosuke Matsuda, Rui Nomura, A. Ueno, Y. Masugi, Ken Yamazaki, Kathryn Effendi, Hirohito Takeuchi, Takao Itoi, Y. Hasegawa, Yuta Abe, M. Kitago, H. Ojima, Michiie Sakamoto
          </td>
          <td>2024-10-17</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Dclk1 expression defines a rare population of cells in the normal pancreas whose frequency is increased at early stages of pancreatic tumorigenesis. The identity and the precise roles of Dclk1 expressing cells in pancreas have been matter of debate, although evidence suggests their involvement in a number of key functions, including regeneration and neoplasia. We employed a recently developed Dclk1 reporter mouse model and single cell RNAseq analysis to define Dclk1 expressing cells in normal pancreas and pancreatic neoplasia. In normal pancreas, Dclk1 epithelial expression identifies subsets of ductal, islet and acinar cells. In pancreatic neoplasia, Dclk1 expression identifies five epithelial cell populations, among which acinar-to-ductal metaplasia (ADM)-like cells and tuft-like cells are predominant. These two cell populations play opposing roles in pancreatic neoplasia, with Dclk1+ ADM-like cells sustaining tumor growth while Dclk1+ tuft-like cells restraining tumor progression. The differentiation of Kras mutant acinar cells into Dclk1+ tuft-like cells requires the activation of the transcription factor SPIB and is further supported by a cellular paracrine loop involving cancer group 2 innate lymphoid cells (ILC2) and cancer activated fibroblasts (CAFs) that provide IL13 and IL33, respectively. In turn, Dclk1+ tuft-like cells release angiotensinogen that plays protective roles against pancreatic neoplasia. Overall, our study provides novel insights on the biology of Dclk1+ cells in normal pancreas and unveils a protective axis against pancreatic neoplasia, involving CAFs, ILC2 and Dclk1+ tuft-like cells, which ultimately results in angiotensinogen release.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297c14738f38a2df2fd03a4a3c024c764c7c76b3" target='_blank'>
              Regulatory network analysis of Dclk1 gene expression reveals a tuft cell-ILC2 axis that inhibits pancreatic tumor progression
              </a>
            </td>
          <td>
            G. Valenti, P. Laise, Ryota Takahashi, Feijin Wu, Tuo Ruan, A. Vasciaveo, Zhengyu Jiang, Masaki Sunagawa, M. Middelhoff, Henrik Nienhüser, Na Fu, E. Malagola, Y. Hayakawa, Alina C. Iuga, Andrea Califano, Timothy C Wang
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Alpha-smooth muscle actin (α-SMA) expression in the stroma is linked to the presence of cancer-associated fibroblasts and is known to correlate with worse outcomes in various tumors. In this study, using a GeoMx digital spatial profiling approach, we characterized the gene expression of the tumor and α-SMA-expressing stromal cell compartments in pancreatic neuroendocrine tumors (PanNETs). The profiling was performed on tissues from eight retrospective cases (three grade 1, four grade 2, and one grade 3). Selected regions of interest were segmented geometrically based on tissue morphology and fluorescent signals from synaptophysin and α-SMA markers. The α-SMA-expressing stromal-cell-associated genes were involved in pathways of extracellular matrix modification, whereas, in tumor cells, the gene expression profiles were associated with pathways involved in cell proliferation. The comparison of gene expression profiles across all three PanNET grades revealed that the differences between grades are not only present at the level of the tumor but also in the α-SMA-expressing stromal cells. Furthermore, the tumor cells from regions with a rich presence of adjacent α-SMA-expressing stromal cells revealed an upregulation of matrix metalloproteinase-9 (MMP9) expression in grade 3 tumors. This study provides an in-depth characterization of gene expression profiles in α-SMA-expressing stromal and tumor cells, and outlines potential crosstalk mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fe06c12f5865d2f0fe11fa3ad5ad66746e58fe" target='_blank'>
              Tumor and α-SMA-expressing stromal cells in pancreatic neuroendocrine tumors have a distinct RNA profile depending on tumor grade.
              </a>
            </td>
          <td>
            H. Niedra, R. Peculis, Rihards Saksis, Ilona Mandrika, Sofija Vilisova, Jurijs Nazarovs, Austra Breikša, Aija Gerina, Julie Earl, Ignacio Ruz-Caracuel, M. Rosas, A. Puķītis, Natalja Senterjakova, V. Rovite
          </td>
          <td>2024-09-08</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Pancreatic cancer, recognized as one of the most lethal cancer types, exhibits a dismal five-year survival rate below 12%. The genetic and epigenetic heterogeneity in human cancer poses a tremendous challenge in understanding disease mechanisms and developing targeted therapeutics. Current mouse models in the field can recapitulate some features of human pathophysiology but have limitations. While tumor formation in patients is focal, most genetically engineered mouse models (GEMMs) express oncogenes throughout the pancreas. GEMMs also have limited flexibility to manipulate gene expression and model the heterogeneity of human cancer. Additionally, GEMMs require significant time for crossing and tumor formation. Tumor transplantation models offer greater flexibility, however, they often require immunodeficient mice, which limits their applications in modeling the tumor microenvironment. To overcome the limits of current mouse models, we combined the piggyBac transposon system, CRISPR/Cas9, and in vivo electroporation to engineer somatic pancreatic tissue to induce focal tumor formation in immune-competent mice. With this approach, we engineered wildtype pancreata to express oncogenic KrasG12D while knocking out multiple tumor suppressors. We optimized this protocol by generating multiplexed gRNA constructs and cell type-specific promoters to improve reproducibility. Low-grade tumors formed within 10 days after electroporation, progressed to higher grade within a month, and metastasized to peritoneal organs during disease progression. Mouse tumors also recapitulated histopathological features in human patients. Lastly, we used our novel mouse model to study how mutant Kras disrupts post-transcriptional regulation by microRNA to drive pancreatic tumorigenesis. This model offers a powerful platform for manipulating gene expression and functionality to model cancer heterogeneity, ultimately advancing precision oncology's prospects for treating human patients.
 Citation Format: Ziyue (Zoey) Yang, Mara Cardenas, Akihiko Miyauchi, Caroline Snider, Ronald Parchem. Generation of a flexible pancreatic cancer mouse model via somatic tissue engineering [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02653c15c1c4443d9e4685c7c408a134b796978a" target='_blank'>
              Abstract A006: Generation of a flexible pancreatic cancer mouse model via somatic tissue engineering
              </a>
            </td>
          <td>
            Z. Yang, Mara Cardenas, Akihiko Miyauchi, Caroline Snider, Ronald Parchem
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The bone marrow microenvironment is critical for B-cell acute lymphoblastic leukemia (B-ALL) but its cellular heterogeneity remains poorly defined. Here, we employed single-cell RNA sequencing to comprehensively characterize the stromal and hematopoietic niches in pediatric B-ALL. Our analysis revealed two distinct mesenchymal stromal cell (MSC) populations as primary leukemia-supportive niches: early mesenchymal progenitors and adipogenic progenitors. Single-cell transcriptomic analysis infers that ALL blasts use distinct cell-cell interactions to communicate with the different stromal populations. Purified adipogenic progenitors from the bone of children with ALL support survival of the leukemic blasts ex vivo and their signature is enriched in relapse samples. Our data establish adipogenic progenitors as a distinct and novel component of the ALL niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f24fbb432c05f1e5ee3ed61b3b778c9a293df20" target='_blank'>
              Distinct Stromal Cell Populations Define the B-cell Acute Lymphoblastic Leukemia Microenvironment
              </a>
            </td>
          <td>
            Mauricio N. Ferrao Blanco, Bexultan Kazybay, M. Belderbos, Olaf Heidenreich, H. J. Vormoor
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a31875ebe3aec993187c2c6c6d773a5efac6f4" target='_blank'>
              Human intermediate prostate cancer stem cells contribute to the initiation and development of prostate adenocarcinoma
              </a>
            </td>
          <td>
            Jie Mu, Ruizhi Li, Yu Zheng, Yi Lu, Lei Ma, Lin Yin, Miao Zhang, Wenyu Ma, Mengjia Chang, Aihua Liu, Jing Li, Haihui Zhu, Dong Wang
          </td>
          <td>2024-09-11</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer liver metastases (BCLM) are hypovascular lesions that resist intravenously administered therapies and have grim prognosis. Immunotherapeutic strategies targeting BCLM critically depend on the tumor microenvironment (TME), including tumor-associated macrophages (TAM). However, a priori characterization of the BCLM TME to optimize therapy is challenging since BCLM tissue is rarely collected. In contrast to primary breast tumors for which tissue is usually obtained and histological analysis performed, biopsies or resections of BCLM are generally discouraged due to potential complications. This study tested the novel hypothesis that BCLM TME characteristics could be inferred from the primary tumor tissue. Matched primary and metastatic human breast cancer samples were analyzed by imaging mass cytometry (IMC), identifying 20 shared marker clusters denoting macrophages (CD68, CD163, CD206), monocytes (CD14), immune response (CD56, CD4, CD8a), Programmed Cell Death protein 1 (PD1), Programmed Death Ligand 1 (PD-L1), tumor tissue (Ki-67, pERK), cell adhesion (E-cad), hypoxia (HIF1α), vascularity (CD31), and ECM (αSMA, collagen, MMP9). A machine learning (ML) workflow was implemented and trained on primary tumor clusters to classify each metastatic cluster density as being either above or below median values. The proposed approach achieved robust classification of BCLM marker data from matched primary tumor samples (AUROC ≥0.75, 95% CI ≥0.7, on the validation subsets). Top clusters for prediction included CD68+, E-cad+, CD8a+PD1+, CD206+, and CD163+MMP9+. We conclude that the proposed workflow using primary breast tumor marker data offers the potential to predict BCLM TME characteristics, with the longer term goal to inform personalized immunotherapeutic strategies targeting BCLM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ceb47da0148310896397e5222d84cc4c2bec57e" target='_blank'>
              Characterization of the breast cancer liver metastasis microenvironment via machine learning analysis of the primary tumor microenvironment.
              </a>
            </td>
          <td>
            Dylan A. Goodin, Eric Chau, Junjun Zheng, Cailin O'Connell, Anjana Tiwari, Yitian Xu, Polly Niravath, Shu-Hsia Chen, Biana Godin, H. Frieboes
          </td>
          <td>2024-10-07</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Spatial techniques such as spatial transcriptomes and MALDI-MSI, offering insights into both transcripts and metabolite of tissue sections. However, integrating them with high accuracy is challenge due to no shared spots or features. We present haCCA, a workflow designed to integrate spatial transcriptomes and metabolomes data using high-correlated feature pairs and modified spatial morphological alignment. This approach ensures high-resolution and accurate spot-to-spot data integration across neighbor tissue section. We applied haCCA to both publicly available 10X Visium and MALDI-MSI datasets from mouse brain tissue and a custom spatial transcriptome and MALDI-MSI dataset from an intrahepatic cholangiocarcinoma (ICC) model, exploring the metabolic alteration of NETs(neutrophil extracellular traps) on ICC, and finding a potential mechanism that NETs upregulated Scd1 to activate fatty acid metabolism. Providing new insights into the dynamic crosstalk between genes and metabolites that regulates the tumor biological behavior and drives the response to treatment. We developed and published an easy-to-use Python package to facilitate its use.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a471f5555baf62c3ebcc010fc547002f0b3c386" target='_blank'>
              haCCA: Multi-module Integrating of spatial transcriptomes and metabolomes
              </a>
            </td>
          <td>
            Xiao-Tian Shen, Jing Xu, Chen Zhang, Xiaoyun Zhang, Zhou-Qing Chen, Hu-Liang Jiang, Lu-Yu Yang
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/484efc8a4ee08b37da85a0f29cd67a40a824997b" target='_blank'>
              Spatial analysis reveals targetable macrophage-mediated mechanisms of immune evasion in hepatocellular carcinoma minimal residual disease.
              </a>
            </td>
          <td>
            Lea Lemaitre, Nia Adeniji, Akanksha Suresh, Reshma Reguram, Josephine Zhang, J. Park, Amit Reddy, Alexandro E. Trevino, Aaron T. Mayer, A. Deutzmann, Aida S Hansen, Ling Tong, Vinodhini Arjunan, N. Kambham, B.C. Visser, M. Dua, C. A. Bonham, N. Kothary, H. B. D'Angio, Ryan Preska, Yanay Rosen, James Y. Zou, Vivek Charu, Dean W Felsher, R. Dhanasekaran
          </td>
          <td>2024-09-20</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
 The complex biology of pancreatic ductal adenocarcinoma (PDAC) remains a significant challenge and limits overcoming its dismal prognosis. The tumor microenvironment (TME), shaped by dynamic interactions between cancer-associated fibroblasts (CAFs), immune and tumor cells, plays a crucial role in PDAC pathogenesis. Uncharted field lies in the impact of specific gene mutations within the tumor epithelium on orchestrating distinct oncogenic TME patterns. Our investigation into homologous recombination deficiency (HRD), caused by mutations in genes like ATM, reveals its role in promoting PDAC aggressiveness and triggering a pronounced desmoplastic reaction, suggesting a unique TME programing. Through single-cell-resolved and multiomics analyses of murine ATM and/or P53-depleted PDACs, and validation in human cases, we unveil transcriptional, translational, and secretory regulatory events, shed light on the intricate interplay between tumor genetics and PDAC microenvironment and pave the way for the elaboration of novel personalized therapies. Murine tumor analysis using single-nucleus multiomics RNA and ATAC sequencing, alongside histological validation in human tissues, revealed specific enrichment of myCAFs in ATM-deficient tumors and reduced populations of T cells and macrophages. Molecular signatures and cell-cell communication inference uncovered ATM-specific interactions within the TME, highlighting TGF-β signaling as pivotal in mediating dialog between CAFs and an aggressive EMT-like tumor cell subset prevalent in ATM-depleted tumors. Additionally, distinct communication patterns were observed between CAFs and immune cells, associated with specific epigenetic and molecular profiles. Mechanistically, transcriptomics, secretomics and proteomics investigations confirmed increased TGFβ release by ATM-deficient PDAC cells and identified a reactive oxygen species (ROS)-mediated mechanism affecting tumor cell behavior and interactions with CAFs. Consistently, a combinatorial therapy targeting TGF-β-mediated tumor-stroma dialog reversed cancer-promoting TME remodeling and exacerbated FOLFIRINOX cytotoxic effects, in vivo, exclusively in ATM-deficient HRD PDACs. Dissecting the biology of ATM-deficient HRD malignant cells, our study unveils their pivotal role in orchestrating oncogenic communication networks with CAFs and immune cells, along with specific tumor promoting stromal reprogramming. Our findings paint a comprehensive picture of how distinct tumor-stroma interactions reshape the TME, steering it towards a cancer-promoting outcome and shed light on the potential to exploit genotype-specific vulnerabilities via tailored tumor-stroma interference strategies.
 Citation Format: Elodie Roger, Hannah M. Mummey, Eleni Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, Lukas Perkhofer, Johann Gout, Alexander Kleger. Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fd93ca5c41a43eb8ee9eb151fa6f7aeb554c533" target='_blank'>
              Abstract A055: Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer
              </a>
            </td>
          <td>
            E. Roger, Hannah M. Mummey, E. Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, L. Perkhofer, J. Gout, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Background Pancreatic ductal adenocarcinoma (PDAC) is the fourth leading cause of cancer- related deaths in the United States and is often not diagnosed until it has already metastasized. Our prior work established that calcium signaling pathways facilitate a pro-metastatic phenotype through the induction of an epithelial-to-mesenchymal transition (EMT) program in PDAC tumor cells, but the downstream signaling molecules mediating this phenotype have remained elusive. MethodsUsing a novel genetically engineered model of pancreatic cancer, we demonstrate that loss of Calcium/calmodulin-dependent kinase-II B (CaMK2B), a downstream effector kinase activated by calcium, enhances cellular plasticity, and increases metastasis. To this end, we generated a genetically engineered mouse model where Camk2b was knocked out in tumor cells in a lineage-labeled Kras- and p53-driven PDAC mouse model (herein KPCY-Camk2bcKO). Results Genetic deletion of Camk2b in the KPCY genetically engineered model accelerated tumor formation and dramatically reduced the overall median survival of KPCY- Camk2b cKO mice (78 days) relative to KPCY controls (136 days). Pathological and immunofluorescence analysis of KPCY-Camk2b cKO animals showed an increased number of poorly differentiated tumor regions and a higher metastatic burden, with >85% of KPCY- Camk2b cKO animals presenting with multi-focal metastases. Further analysis of tumor cells in KPCY-Camk2b cKO mice showed enhanced expression of markers of the highly aggressive squamous PDAC subtypes. Strikingly, Camk2b deletion combined with oncogenic Kras G12D (in the absence of p53 alterations), was sufficient to form lethal metastatic PDAC tumors, while age-matched control Kras G12D tumors with intact Camk2b only progressed to the acinar-to-ductal metaplasia (ADM) or Pancreatic Intraepithelial Neoplasia (PanIN) stage. Collectively, these results establish Camk2b loss as a new oncogenic and pro-metastatic mechanism in PDAC Conclusion This work demonstrates that Camk2b loss enhances tumor cell evolution and plasticity within PDAC to promote metastasis. In this context, we speculate that loss of Camk2b rewires calcium signaling to drive an aggressive subtype of PDAC with increased metastasis.
 Citation Format: Jessica Peura, Nikita Bhalerao, Calvin Johnson, Yamini Ogoti, Faith Keller, Emma Watson, Zhen Zhao, Jason Pitarresi. Loss of CaMK2B accelerates tumor formation and enhances metastatic competency in genetically engineered mouse models of PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C061.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7848880ff2d74aa871376f23a8aeef10bd0463c1" target='_blank'>
              Abstract C061: Loss of CaMK2B accelerates tumor formation and enhances metastatic competency in genetically engineered mouse models of PDAC
              </a>
            </td>
          <td>
            Jessica Peura, Nikita Bhalerao, Calvin Johnson, Yamini Ogoti, Faith Keller, Emma Watson, Zhen Zhao, Jason R. Pitarresi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Osteosarcoma is a common type of bone cancer characterized by a poor prognosis due to its metastatic nature. The tumor microenvironment (TME) plays a critical role in tumor metastasis and therapy response. Therefore, our study aims to explore the metastatic mechanism of osteosarcoma, potentially opening new avenues for cancer treatment. Methods In this study, we collected data from the GSE152048, GSE14359, and GSE49003 datasets. Differentially expressed genes (DEGs) were identified in osteosarcoma cases with primary and metastatic features using R software and the limma package. Gene ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analyses were performed to investigate metastasis-related genes. A protein–protein interaction (PPI) network was established using the STRING database to further analyze these metastasis-associated genes. The abundances of different cell types with a mixed cell population were estimated using the CIBERSORT approach. The scRNA-seq data were analyzed by the Seurat package in R software, and intercellular communications were elucidated using the CellChat R package. Results In this study, 92 DEGs related to metastasis were identified, including 41 upregulated and 51 downregulated genes in both the GSE14359 and GSE49003 datasets. Metastasis-associated pathways were identified, including those involving the cyclin-dependent protein kinase holoenzyme complex, transferase complex, transferring phosphorus-containing groups, SCF ubiquitin ligase complex, and the serine/threonine protein kinase complex. KEGG and PPI network analyses revealed 15 hub genes, including Skp2, KIF20A, CCNF, TROAP, PHB, CKS1B, MCM3, CCNA2, TRIP13, CENPM, Hsp90AB1, JUN, CKS2, TK1, and KIF4A. Skp2 has been known as an E3 ubiquitin ligase involved in osteosarcoma progression. The proportion of CD8+ T cells was found to be higher in metastatic osteosarcoma tissues, and high expression of PHB was associated with a favorable prognosis in osteosarcoma patients. Additionally, 23 cell clusters were classified into eight cell types, including chondrocytes, MSC, T cells, monocytes, tissue stem cells, neurons, endothelial cells, and macrophages. The 15 hub genes were expressed across various cell types, and interactions between different cell types were observed. Conclusion Our study reveals the intricate communication between tumor microenvironment components and tumor metastasis in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff33ba2ff2e5acd8821d77a975c86bee16bbf2ba" target='_blank'>
              Single-cell RNA sequencing reveals the communications between tumor microenvironment components and tumor metastasis in osteosarcoma
              </a>
            </td>
          <td>
            Jiatong Li, Yang Bai, He Zhang, Ting Chen, Guanning Shang
          </td>
          <td>2024-09-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cell type identity is determined by gene regulatory networks (GRNs), comprising the expression of specific transcription factors (TFs) regulating target genes (TGs) via binding to open chromatin regions (OCRs). The regulatory logic of differentiation includes factors specific to one or multiple cell types, functioning in a combinatorial fashion. Classic approaches of GRN discovery used perturbational data to elucidate TF-TG links, but are laborious and not scalable across the tree of life. Single cell transcriptomics has emerged as a revolutionary approach to study gene expression with cell type resolution, but incorporating perturbational data is challenging. Planarians, with their pluripotent neoblast stem cells continuously giving rise to all cell types, offer an ideal model to attempt this integration. Despite extensive single cell transcriptomic studies, the transcriptional and chromatin regulation at the cell type level remains unexplored. Here, we investigate the regulatory logic of planarian stem cell differentiation by obtaining an organism-level integration of single cell transcriptomics and single cell accessibility data. We identify specific open chromatin profiles for major differentiated cell types and analyse their transcriptomic landscape, revealing distinct gene modules expressed in individual types and combinations of them. Integrated analysis unveils gene networks reflecting known TF interactions in each type and identifies TFs potentially driving differentiation across multiple cell types. To validate our predictions, we combined TF knockdown RNAi experiments with single cell transcriptomics. We focus on hnf4, a TF known to be expressed in gut phagocytes, and confirm its influence on other types, including parenchymal cells. Our results demonstrate high overlap between predicted targets and experimentally-validated differentially-regulated genes. Overall, our study integrates TFs, TGs and OCRs to reveal the regulatory logic of planarian stem cell differentiation, showcasing that the combination of single cell methods and perturbational studies will be key for characterising GRNs widely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d812fbadba7ebc9a3898e7e95eae459989657669" target='_blank'>
              The Regulatory Logic of Planarian Stem Cell Differentiation
              </a>
            </td>
          <td>
            Alberto Pérez-Posada, Helena García-Castro, Elena Emili, Virginia Vanni, Cirenia Arias-Baldrich, S. Frölich, S. V. van Heeringen, N. Kenny, Jordi Solana
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Astrocytoma is a diffuse and heterogeneous glioma that infiltrates the brain. Despite post-operative therapy including radiation and chemotherapy, 52% of patients’ tumours recur within five-years of surgery. These tumours are characterised by mutations in the IDH1 or IDH2 gene that confer metabolic alterations, most notably the production of the onco-metabolite 2-hydroxyglutarate. To reveal the metabolic alterations and their upstream regulation across the heterogeneous tumour microenvironment, we applied spatial metabolomics and spatial transcriptomics to sequential sections from resected tumour samples of patients with low grade astrocytoma.



 Spatial assessment of heterogeneity was performed on serial sections using spatial metabolomics (atmospheric pressure MALDI-TOF) and spatial transcriptomics (10X Genomics Xenium). This necessitated extensive optimisation of image registration across modalities and sequential sections to create three dimensional models to deeply interrogate heterogeneity within tumours.



 Combining spatial transcriptomics and spatial metabolomics across 8 matched sequential slices of the same patient tumour, enabled the identification of metabolic niches associated with distinct cell type composition and transcriptional regulation. Strikingly, normal domains that were present at the leading edge of tumours could be identified both metabolically and transcriptomically.



 Understanding the heterogeneity of astrocytomas on a spatial level reveals unique domains and organisation of tumour and immune cell types. Matching these transcriptomic and cell state information with metabolic insights leads to pathway insights that will be essential to the identification of druggable targets. Our pipeline combining spatial transcriptomics and metabolomics with tissue histology reveals these biologically important domains.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbf36889624e6059d6b35a03a0d2ffccf6ee4adc" target='_blank'>
              OS07.5.A MULTI-MODAL EXAMINATION OF SPATIAL HETEROGENEITY IN THE ASTROCYTOMA MICROENVIRONMENT
              </a>
            </td>
          <td>
            T. Lu, L. Freytag, J. Moffet, J. Kriel, J. Whittle, S. Freytag, S. A. Best
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Uveal melanoma (UM) is a rare type of cancer of the eye that develops from a cell type called melanocytes, which are present in the uveal tract. These patients have a high risk of liver metastasis, usually many years after the initial diagnosis, which is life threatening. We discovered a population of cells in the primary tumor called dual-nature hybrid cells (DNCs) similar to previously discovered cells in the blood (i.e., circulating hybrid cells, CHCs) with shared characteristics of both cancer cells and white blood cells. In this paper, we determined the types of white blood cells capable of participating in the formation of a hybrid cell with a cancer cell, and we re-created these DNCs in a cell culture. The study of the cells of this tumor, particularly DNCs, is crucial to better understanding dissemination and uncovering a possible target for treatment of this systemic disease. Abstract Background: Metastasis, occurring years after primary diagnosis, represents a poor prognosis in uveal melanoma (UM)-affected individuals. The nature of cells involved in this process is under debate. Circulating hybrid cells that have combined tumor and immune cell features found in blood were predictive of metastasis and may correspond to dual-nature cells (DNC) in the primary tumor. Herein, we sought to determine the presence of DNCs in primary UM tumors, the cell types involved in their genesis, and their ability to be formed in vitro. Methods: UM lesions (n = 38) were immunolabeled with HMB45 in combination with immune-cell-specific antibodies. In parallel, we co-cultured UM cells and peripheral blood mononuclear cells (PBMCs) to analyze DNC formation. Results: HMB45+/CD45+ DNCs were present in 90% (26/29) of the tumors, HMB45+/CD8+ DNCs were present in 93% (26/28), and HMB45+/CD68+ DNCs were present in 71% (17/24). DNCs formed with CD8+ and CD68+ cells were positively correlated to the infiltration of their respective immune cells. Notably, UM cells were prone to hybridize with PBMCs in vitro. Conclusions: This phenotypical characterization of DNCs in UM demonstrates that CD8+ T-cells and macrophages are capable of DNC formation, and they are important for better understanding metastatic dissemination, thus paving the path towards novel therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c17116b61c31885d971f0798b1beb4e75b2a6ef8" target='_blank'>
              The Phenotypical Characterization of Dual-Nature Hybrid Cells in Uveal Melanoma
              </a>
            </td>
          <td>
            Emily Marcotte, Alicia Goyeneche, Mohamed Abdouh, J. Burnier, MiguelN Burnier
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="BACKGROUND
Cancer-Associated Fibroblasts (CAFs) constitute a heterogeneous group of cells critical for the remodeling of the tumor microenvironment (TME). Given their significant impact on tumor progression, particularly in skin cancers, a deeper understanding of their characteristics and functions is essential.


METHODS
This study employed a single-cell transcriptomic analysis to explore the diversity of CAFs within three major types of skin cancer: basal cell carcinoma, melanoma, and head and neck squamous cell carcinoma. We applied analytical techniques, including Gene Ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), Gene Set Enrichment Analysis (GSEA), pseudotime tracking, metabolic profiling, and stemness assessment to delineate and define the functional attributes of identified CAF subgroups.


RESULTS
Our analysis successfully delineated nine distinct CAF subgroups across the studied tumor types. Of particular interest, we identified a novel CAF subtype, designated as C0, exclusive to basal cell carcinoma. This subtype exhibits phenotypic traits associated with invasive and destructive capabilities, significantly correlating with the progression of basal cell carcinoma. The identification of this subgroup provides new insights into the role of CAFs in cancer biology and opens avenues for targeted therapeutic strategies.


CONCLUSION
A pan-cancer analysis was performed on three cancers, BCC, MA, and HNSCC, focusing on tumor fibroblasts in TME. Unsupervised clustering categorized CAF into nine subpopulations, among which the C0 subpopulation had a strong correspondence with BCC-CAF and an invasive- destructive-related phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b65ae0c3c55b73c69971a741aaa77cb534354a3" target='_blank'>
              Pan-Cancer Single-Cell Analysis Revealing the Heterogeneity of Cancer-Associated Fibroblasts in Skin Tumors.
              </a>
            </td>
          <td>
            Yichi Zhang, Zhijie Zhao, Wenyi Huang, Byeong Seop Kim, Li Lin, Xin Li, Mengyuan Hou, Li Li, Yan Zhang, Wenjing Xi, G. Chai
          </td>
          <td>2024-09-25</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c790b68920aa80778ac106b1b70165f05c9d802" target='_blank'>
              Metabolic and Immunological Implications of MME+CAF-Mediated Hypoxia Signaling in Pancreatic Cancer Progression: Therapeutic Insights and Translational Opportunities
              </a>
            </td>
          <td>
            Bin Wang, Yue Pan, Yongjie Xie, Cong Wang, Yinli Yang, Haiyan Sun, Zhuchen Yan, Yameng Cui, Ling Li, Yaoyao Zhou, Weishuai Liu, Zhanyu Pan
          </td>
          <td>2024-09-28</td>
          <td>Biological Procedures Online</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) -- the expansion of somatically-mutated hematopoietic cells in blood -- is common in solid cancers. CH is associated with systemic inflammation that may lead to cancer, but its impact on tumor biology is underexplored. Here, we report the effects of CH on the tumor microenvironment (TME) using 1,550 treatment-naive patient samples from the CPTAC cohort. CH was present in 18.3% of patients, with one-third of CH mutations also detectable in tumor-derived DNA from the same individual (CH-Tum), reflecting CH-mutant leukocyte infiltration. The presence of CH-Tum was associated with worse survival across cancers, particularly for glioblastoma. Transcriptomics and proteomics revealed that CH drives inflammation in the TME in a cancer- and CH driver-specific manner, and may improve immunotherapy responses. In glioblastoma, CH associated with pronounced macrophage infiltration, inflammation, and an aggressive, mesenchymal phenotype. Our findings demonstrate that CH shapes the TME, with potential applications as a biomarker in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df335cc5550a0c8767d76bf480da6e95ae3c38e1" target='_blank'>
              Inflammatory reprogramming of the tumor microenvironment by infiltrating clonal hematopoiesis is associated with adverse outcomes in solid cancer
              </a>
            </td>
          <td>
            M. Buttigieg, C. Vlasschaert, A. G. Bick, R. Vanner, M. Rauh
          </td>
          <td>2024-08-30</td>
          <td>None</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Effective drug discovery relies on combining target knowledge with functional assays and multi-omics data to understand chemicals’ molecular actions. However, the relationship between cell morphology and gene expression over time and across cell lines remains unclear. To explore this, we analyzed Cell Painting and L1000 data for 106 compounds across three cell lines from osteoblast, lung, and breast tumors (U2OS, A549, and MCF7) at three time points (6h, 24h, 48h) using a 10µM concentration. We found significant time and cell line effects in Cell Painting data, with less pronounced effects in transcriptomics. Using Weighted Gene Co-expression Network Analysis (WGCNA) and enrichment analysis, we identified connections between cell morphology and gene deregulation for chemicals with similar biological effects (e.g., HDAC and CDK inhibitors). These findings suggest that while Cell Painting shows distinct patterns, both technologies offer complementary insights into compound-induced cellular changes, enhancing drug discovery and chemical risk assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4e8eb6d8184a42daa0a1c6534359ca9aca19c28" target='_blank'>
              Cell morphology and gene expression: tracking changes and complementarity across time and cell lines
              </a>
            </td>
          <td>
            Vanille Lejal, David Rouquié, Olivier Taboureau
          </td>
          <td>2024-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose Cancer-associated fibroblasts (CAFs) significantly contribute to tumor progression and the development of resistance to therapies across a range of malignancies, notably breast cancer. This study aims to elucidate the specific role and prognostic relevance of CALU across multiple cancer types. Patients and Methods The association between CALU expression and prognosis, along with clinical characteristics in BRCA, HNSC, KIRP, LGG, and LIHC, was analyzed using data from the TCGA, GTEx, and GEO databases. Transcriptomic analysis of TCGA BRCA project data provided insights into the interaction between CALU and epithelial-mesenchymal transition (EMT) marker genes. Using TIMER and TISCH databases, the correlation between CALU expression and tumor microenvironment infiltration was assessed, alongside an evaluation of CALU expression across various cell types. Furthermore, CALU’s influence on TNBC BRCA cell lines was explored, and its expression in tumor tissues was confirmed through immunohistochemical analysis of clinical samples. Results This study revealed a consistent upregulation of CALU across several tumor types, including BRCA, KIRP, LIHC, HNSC, and LGG, with elevated CALU expression being associated with unfavorable prognoses. CALU expression was particularly enhanced in clinical contexts linked to poor outcomes. Genomic analysis identified copy number alterations as the principal factor driving CALU overexpression. Additionally, a positive correlation between CALU expression and CAF infiltration was observed, along with its involvement in the EMT process in both CAFs and malignant cells. In vitro experiments demonstrated that CALU is highly expressed in TNBC-BRCA cell lines, and knockdown of CALU effectively reversed EMT progression and inhibited cellular migration. Immunohistochemical analysis of clinical samples corroborated the elevated expression of CALU in tumors, along with alterations in EMT markers. Conclusion This comprehensive pan-cancer analysis underscores CALU’s critical role in modulating the tumor microenvironment and facilitating cell migration via the EMT pathway, identifying it as a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dfdbccf6247f0f2af866ccb68efa9dd11a6fa74" target='_blank'>
              Pan-Cancer Screening and Validation of CALU’s Role in EMT Regulation and Tumor Microenvironment in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Shi-liang Chen, Dan Hu, Tian-zhu Chen, Si-yu Shen, Chen-Fei Zhao, Cong Wang, Shi-yuan Tong, Zhao Liu, Shao-hua Lin, Li-xia Jin, Yi-bo He, Zhe-zhong Zhang
          </td>
          <td>2024-09-01</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cellular heterogeneity in nasopharyngeal cancer (NPC) and oral cancer remains unclear. In the current study, using single-cell RNA sequencing techniques, we investigated the cellular landscape in NPC and oral cancers. We identified a diverse range of cell types within the tumor microenvironment (TME) and variations in cell infiltration between NPC and oral cancer. In oral cancer, we observed a predominant infiltration of epithelial cells, fibroblasts, and endothelial cells (ECs), while T cells were the main infiltrating cell population in NPCs. We further classified these infiltrating cells into subclusters. Additionally, we observed complex interactions among cells that led to distinct trajectories. In particular, a unique epithelial subcluster with high expression of major histocompatibility complex class II (MHC-II) molecules was correlated with a favorable outcome and infiltration of CD4+ T cells. In addition, MHC-II+ epithelial cells inhibited mouse tumor growth and promoted T-cell infiltration. Consequently, our findings provide a deep understanding of the TME showing a significant prognostic value and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9da59ab40fc418775c5958a840e96f87ebb0a0a" target='_blank'>
              Deciphering the cellular landscape and potential targets of nasopharyngeal and oral cancers using single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Yanfei Zhang, Xiaoyu Qin, Weihua Lou, Liang Wang, Wu-hao Lu, Changhui Gao, Shousen Hu
          </td>
          <td>2024-08-29</td>
          <td>Cell biology international</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The acquisition of invasive properties is a prerequisite for tumor progression and metastasis. Molecular subtypes of KRAS-driven lung cancer exhibit distinct modes of invasion that contribute to unique growth properties and therapeutic susceptibilities. Despite this, pre-clinical strategies designed to exploit growth within the context of invasion are lacking. To address this, we designed an experimental system to screen for targetable signaling pathways linked to active early 3D invasion phenotypes in different molecular subtypes of KRAS-driven lung adenocarcinoma (LUAD). Combined live-cell imaging of human bronchial epithelial cells in a 3D invasion matrix and transcriptomic profiling identified mutant LKB1-specific upregulation of BMP6. LKB1 loss increased BMP6 signaling, which induced the canonical iron regulatory hormone hepcidin. Intact LKB1 was necessary to maintain BMP6 signaling homeostasis and restrict ALK2/BMP6-fueled growth. Pre-clinical studies in a Kras/Lkb1-mutant syngeneic mouse model and in a xenograft model showed potent growth suppression by inhibiting the ALK2/BMP6 signaling axis with single agent inhibitors that are currently in clinical trials. Lastly, BMP6 expression was elevated in LKB1-mutant early-stage lung cancer patient tumors. These results are consistent with a model where LKB1 acts as a 'brake' to iron regulated growth and suggest that ALK2 inhibition can be used for patients with LKB1-mutant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6933a02c2787f96cb462a49d5a6b0f47ef53005" target='_blank'>
              Live-Cell Invasive Phenotyping Uncovers ALK2 as a Therapeutic Target in LKB1-Mutant Lung Cancer.
              </a>
            </td>
          <td>
            Junghui Koo, Chang-Soo Seong, Rebecca E. Parker, Amy Herrera, Bhakti Dwivedi, Robert A. Arthur, Ashok R. Dinasarapu, H. R. Johnston, Henry Claussen, Carol Tucker-Burden, Suresh S. Ramalingam, Haian Fu, Wei Zhou, Adam I. Marcus, Melissa Gilbert-Ross
          </td>
          <td>2024-08-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 The alteration of the glioblastoma genome by epigenetic mechanisms that share functions with normal developmental processes, such as self-renewal and fate specification of neural stem cells (NSC), is a key piece of evidence that links brain cancer pathogenesis with dysregulated stem cell functions. A patient-specific comparison of glioblastoma stem cells (GSC) with NSC, a putative cell of origin of at least a proportion of these tumours, is not feasible as patient-matched endogenous NSC are not surgically accessible and all epigenetic studies in glioblastoma have so far compared epigenetic changes of different tumours with each other or to comparators obtained from foetal brains or an unrelated donor. We reasoned that availability of syngeneic GSC and NSC pairs would allow to identify crucial epigenetic differences on a patient-specific basis and would provide essential therapeutic contrast to define disease-and patient-intrinsic biomarkers of drug response that are less confounded by germline variation.



 We have derived GSC from IDH-wildtype glioblastoma and harnessed state-of-the-art stem cell technologies to generate patient-matched fibroblast-derived EPSC, which were induced to NSC (iNSC). We generated transcriptome, DNA methylome, ChIP-Seq (H3K4me3, H3K27ac, H3K3me3, H3K27me3) and ATACSeq datasets for 10 GIC/iNSC pairs.



 Comparative integrated analysis has shown that redistribution of selected histone marks as well as shifts in chromatin states across the genome identifies differentially expressed genes, including novel druggable regulatory mechanisms in GSC, which are specific of the neoplastic context. In particular, we identified METTL7B as an essential regulator of lineage specification and plasticity in human glioblastoma, with impact on both tumour size and invasiveness in an in vivo xenograft model. Single cell transcriptomic analysis of these tumours and of cerebral organoids derived from expanded potential stem cells (EPSCs) overexpressing METTL7B reveal a regulatory role for the gene in the neural stem cells-to-astrocyte differentiation trajectory. Mechanistically, METTL7B downregulates the expression of key neuronal differentiation players, including SALL2, via post-translational modifications of histone marks.



 SYNGN is an epigenetic platform which can identify novel disease mechanisms and druggable targets in glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be96bd21ac249c6fcb9083c430d82766e4fcbf5a" target='_blank'>
              P01.14.B SYNGN: A NOVEL EPIGENETIC PLATFORM TO DISCOVER PATIENT-SPECIFIC DISEASE MECHANISMS AND DRUGGABLE TARGETS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            S. Marino, C. Vinel, M. Constantinou
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pancreatic cancer is a highly lethal and aggressive disease, commonly characterized by a unique tumor microenvironment, where mutant KRAS signaling plays a pivotal role in its pathophysiology. To deepen our understanding of KRAS-dependent signaling in earliest tumor development, oncogenic KRAS (KRASG12D) expression was induced in human stem cell-derived pancreatic ductal-like organoids (PDLOs), followed by time-resolved single-cell RNA and bulk ATAC sequencing. This unique precancerous state model facilitated a comprehensive characterization of early oncogenic events at cellular resolution. Furthermore, a machine learning approach was applied to accurately predict KRAS-dependent cell identities. Within seven days, mutated KRAS triggers distinct gene signature programs related to extracellular matrix remodeling and inflammation. Although derived from dysplastic yet non-cancerous states, these gene signatures negatively correlated with overall survival in PDAC (pancreatic ductal adenocarcinoma) patients. The KRAS-dependent signatures retrieved with the machine learning approach allowed the grouping of 24 published PDAC cases, showing differences in niche composition and communication patterns. Various co-culture systems demonstrated that KRASG12D-expressing PDLOs could activate pancreatic stellate cells, inducing a cancer- associated fibroblast (CAF)-like state. Simultaneously, these KRAS-activated organoids were protected from T cell infiltration in vitro. Furthermore, in silico approaches were employed to reconstruct a virtual pancreatic cancer space, demonstrating that our PDLOs transcriptionally bridge healthy and malignant ductal states. This also revealed potential KRAS-driven pathways mediating CAF activation and T cell shielding preceding cancer onset. In summary, our human PDLO model enables us to explore the transition from normal to cancerous states, addressing gaps in our understanding of tumor development and tumor niche preparation.
 Citation Format: Chantal Allgoewer, Markus Breunig, Meike Hohwieler, Medhanie Mulaw, Alexander Kleger. Single-cell profiling unleashes KRAS-driven redrafting of the tumor microenvironment before cancer onset [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc4900be652d49c2d242f617b8845d915d29b" target='_blank'>
              Abstract C010: Single-cell profiling unleashes KRAS-driven redrafting of the tumor microenvironment before cancer onset
              </a>
            </td>
          <td>
            Chantal Allgoewer, M. Breunig, M. Hohwieler, Medhanie M Mulaw, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3afd477a323b89fccf668eab916310c90e1a4e1" target='_blank'>
              Targeting IRE1α reprograms the tumor microenvironment and enhances anti-tumor immunity in prostate cancer
              </a>
            </td>
          <td>
            Bilal Unal, Omer F. Kuzu, Yang Jin, Daniel Osorio, W. Kildal, Manohar Pradhan, Sonia H Y Kung, H. Oo, M. Daugaard, Mikkel Vendelbo, John B Patterson, M. Thomsen, M. Kuijjer, F. Saatcioglu
          </td>
          <td>2024-10-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Prostate cancer (PC), which is a disease driven by the activity of the androgen receptor (AR), is the most commonly diagnosed malignancy and despite advances in diagnostic and treatment strategies, PC is the second most common cause of cancer mortality in men (Bray et al., 2018). Taxane-based chemotherapy is the only chemotherapy that prolongs survival in metastatic PC patients (Petrylak et al., 2004; Tannock et al., 2004). At the cellular level, taxanes bind to and stabilize microtubules (MTs) inhibiting all MT-dependent intracellular pathways. MTs are highly dynamic polymers that stochastically switch between phases of growth, shrinkage, and pause (Jordan and Wilson, 2004). Altered MT dynamics endow cancer cells with both survival and migratory advantages (Mitchison, 2012). Taxanes inhibit MT dynamics and alter the spatial organization of the MT network, thereby inhibiting intracellular trafficking of molecular cargo critical for tumor survival. In PC specifically, taxanes inhibit transcriptional activity downstream of MT stabilization (Thadani-Mulero et al., 2012) and AR nuclear accumulation (Darshan et al., 2011; Zhu et al., 2010). Different tubulin inhibitors, even from within the same structural class as the taxanes, affect distinct parameters of MT dynamics (Jordan and Wilson, 2004), yet the selection of taxane for chemotherapy is not based on the particular patterns of dynamic behavior of the MT cytoskeleton in individual patients. We envisage that systematic characterization using quantitative analysis of MT dynamics in PC patient cells expressing clinically relevant protein isoforms (Matov et al., 2024; Thoma et al., 2010), before and after treatment with each of the taxanes, will allow us to identify criteria for the selection of the most suitable drug combination at the onset of treatment. We link MT dynamics in the presence of AR variants and sensitivity/resistance to taxanes and connect fundamental research with clinically relevant concepts to elucidate cellular mechanisms of clinical response to taxanes and, thus, advance the customization of therapy. Our computational live-cell analysis addresses questions in the context of the inherent differences in MT homeostasis as a function of AR content in PC cells, the specific parameters of MT dynamics each of the taxanes affects, and how can this information be used to match endogenous patterns of MT dynamics with drug-modulated MT behavior. We investigate whether the sensitivity to taxanes, evaluated by computational analysis of MTs, can be linked to gene expression driven by AR and its variants, and whether the resistance to taxanes be linked to the presence of a specific AR splice variant, and can we identify which of the taxanes will be most effective based on the endogenous patterns of MT dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e90d5e5c268c6138c1b50b9fd253e67162113bd" target='_blank'>
              Computational Analysis of Treatment Resistant Cancer Cells
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-08-31</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Bone metastasis (BM) is a mortality-related event of late-stage cancer, with non-small cell lung cancer (NSCLC) being a common origin for BM. However, the detailed molecular profiling of the metastatic bone ecosystem is not fully understood, hindering the development of effective therapies for advanced patients. In this study, we examined the cellular heterogeneity between primary tumours and BM from tissues and peripheral blood by single-cell transcriptomic analysis, which was verified using multiplex immunofluorescence staining and public datasets. Our results demonstrate a senescent microenvironment in BM tissues of NSCLC. BM has a significantly higher infiltration of malignant cells with senescent characteristics relative to primary tumours, accompanied by aggravated metastatic properties. The endothelial-mesenchymal transition involved with SOX18 activation is related to the cellular senescence of vascular endothelial cells from BM. CD4Tstr cells, with pronounced stress and senescence states, are preferentially infiltrated in BM, indicating stress-related dysfunction contributing to the immunocompromised environment during tumour metastasis to bone. Moreover, we identify the SPP1 pathway-induced cellular crosstalk among T cells, vascular ECs and malignant cells in BM, which activates SOX18 and deteriorates patient survival. Our findings highlight the roles of cellular senescence in modulating the microenvironment of BM and implicate anti-senescence therapy for advanced NSCLC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae5ee75b17730b9ef56c88719d46b498e6167453" target='_blank'>
              Single-cell transcriptomic analysis of the senescent microenvironment in bone metastasis.
              </a>
            </td>
          <td>
            Shenglin Wang, Lu Ao, Huangfeng Lin, Hongxiang Wei, Zhaoyang Wu, Shuting Lu, Fude Liang, R. Shen, Huarong Zhang, Tongjie Miao, Xiaopei Shen, Jianhua Lin, Guangxian Zhong
          </td>
          <td>2024-09-04</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53e6452f948769bae38a66bb4f9a7addad82684" target='_blank'>
              Single-cell RNA sequencing explored potential therapeutic targets by revealing the tumor microenvironment of neuroblastoma and its expression in cell death
              </a>
            </td>
          <td>
            Lei Sun, Wenwen Shao, Zhiheng Lin, Jingheng Lin, Fu Zhao, Juan Yu
          </td>
          <td>2024-09-05</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3db9b2bc9f499477ff494631d94e30c054c0bf" target='_blank'>
              A Boolean model explains phenotypic plasticity changes underlying hepatic cancer stem cells emergence
              </a>
            </td>
          <td>
            Alexis Hernández-Magaña, Antonio Bensussen, Juan Carlos Martínez-García, E. Álvarez-Buylla
          </td>
          <td>2024-09-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) it though to develop through induction of different precursor lesions, however, the normal cell type that gives rise to these precancerous lesions and PDAC is unclear. Recent studies using mouse models and ex vivo cultured human cells suggest that the cellular origin of PDAC may be either acinar or ductal cells. Potential transcriptional markers associated with PDAC of different cellular origins have been identified. Here, we interrogate whether these markers are predictive of cellular of origin across models using mouse models where acinar and ductal cells (Ptf1aCreER or Sox9CreER) expressing oncogenic Kras in the absence of Trp53 or Pten give rise to PDAC tumors. We found that acinar- and ductal-cell-derived PDAC exhibit transcriptional differences both in vivo and in vitro when comparing within one genotype or sample source type (e.g. bulk tumor or primary cell lines). However, previously predicted markers of cellular origin had variable expression across models and sample types suggestive of differences specific to those sample sets. Using our larger sample sets and multiple genotypes and source types, we proposed a more comprehensive set of cell of origin markers for consideration. Application of these markers to our own data sets, however, failed to properly segregate acinar- and ductal-cell-derived PDAC into separate groups. This suggests that the transcriptional heterogeneity observed in our sample set was not driven by the tumors arising from different cellular origins. We also examined the relationship between cell of origin and patient PDAC transcriptional subtypes using our mouse model datasets. Our findings support previous research indicating that mouse models of acinar-cell-derived PDAC favor those of a classical subtype. However, our mouse PDAC samples all express similar or very low levels of the genes denoting the basal subtype, suggesting that the basal subtype was not strongly present in any of the datasets we examined. In summary, our data suggest that acinar and ductal cells converge into a very similar PDAC transcriptional state and that, if differences can be identified, they will likely be identified by non-transcript based readouts.
 Citation Format: Ken Chu, Alex Lee, Karnjit Sarai, Yan Dou, Wesley Hunt, Emma Croft, Atefeh Samani, Farnaz Taghizedeh, Claire Dubois, Stephane Flibotte, Janel Kopp. Cell of origin does not underlie transcription heterogeneity in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7619576b4dd9f9188da32a5b5d7af49469c5cb88" target='_blank'>
              Abstract A025: Cell of origin does not underlie transcription heterogeneity in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ken Chu, Alex Y. L. Lee, Karnjit Sarai, Yan Dou, Wesley Hunt, Emma Croft, Atefeh Samani, Farnaz Taghizedeh, Claire L. Dubois, Stephane Flibotte, Janel L. Kopp
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Background: Pancreatic ductal adenocarcinoma (PDAC) is highly resistant to chemoradiation and immune therapies. There is limited biomarker to identify subsets of PDAC that responds to immunotherapy or targeted therapy. We have identified a subset of PDACs (approximately 25%) carrying COMPASS-like complex (CLC) gene mutations. Analysis of the clinicopathologic and molecular features of this PDAC subset revealed that these tumors are an aggressive group of PDACs with poor or squamous histologic differentiation, increased rates of TP53 mutations, and poor patient survival compared to control PDACs. Our preliminary data showed that PDACs carrying mutations in two members of the CLC genes (KDM6A and KMT2D) have immunosuppressive tumor microenvironment (TME), suggesting that CLC genes are candidate biomarkers for immunotherapy. However, a comprehensive analysis of the immune composition, immune checkpoint expression, and the relationship between tumor cells and immune cells in CLC gene mutated PDACs compared to controls has not been performed. In this study, we characterized the immune composition in CLC gene mutated PDACs compared to paired control PDACs using tyramide signal amplification multiplex fluorescent immunohistochemistry (mfIHC). Methods: Human PDAC tissue slides were stained with 2 mfIHC panels and cell types were defined by different markers: panel 1 (PanCK, CD163, PD-L1, CD3, CD8, FoxP3) and panel 2 (PanCK, CD163, CD3, CD8, TIGIT, TIM3). Random image sections were selected in each sample (5∼8 sections per sample) and divided into wild-type (WT) group (n=57) and mutant group (n=41) and analyzed using InForm Cell Analysis software. Tissue segmentation training was performed to separate stromal and epithelial elements. Cell percentage, cell density, and nearest neighbor distance of each cell type were calculated using R program. Results: The CLC gene mutation is associated with elevated epithelial cells (ECs) and PDL1+ ECs and increased infiltration of exhausted cytotoxic T cells (TIGIT+/TIM3+ CD8 T cells). CLC gene mutated PDACs are characterized by spatial proximity between exhausted T cells (TIGIT+) and epithelial cells/antigen presenting cells (APCs). Higher number of epithelial cells is associated with reduced infiltration of helper T cells in the mutant PDAC tissue. We also observed that high numbers of APC in proximity to CD4 T cells were positively correlated with the proximity of EC and CD8 cytotoxic T cells as well as the proximity of CD4 T cells and cytotoxic T cells in the WT group but not in the CLC gene mutant group. Conclusion: Our results provide insight into the effects of CLC gene mutations on shaping the immune microenvironment of PDAC and suggest that CLC gene mutated PDACs have an immunosuppressive immune microenvironment than other PDACs and may respond differently to immunotherapy.
 Citation Format: Shungang Zhang, Elaina Daniels, Jake McGue, Hong Sun Kim, Dafydd Thomas, Timothy Frankel, Jiaqi Shi. Multiplex fluorescent immunohistochemistry reveals immunosuppressive tumor microenvironment in COMPASS-like complex gene mutated pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2790a085665b4a36a3c8b84d6bede9ac8c16c70" target='_blank'>
              Abstract A040: Multiplex fluorescent immunohistochemistry reveals immunosuppressive tumor microenvironment in COMPASS-like complex gene mutated pancreatic cancer
              </a>
            </td>
          <td>
            Shungang Zhang, Elaina Daniels, Jake McGue, H. Kim, Dafydd Thomas, Timothy L. Frankel, Jiaqi Shi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Today, the intratumoral composition is a relevant factor associated with the progression and aggression of cancer. Although it suggests a metabolic interdependence among the subpopulations inside the tumor, a detailed map of how this interdependence contributes to the malignant phenotype is still lacking. To address this issue, we developed a systems biology approach integrating single-cell RNASeq and genome-scale metabolic reconstruction to map the metabolic cross-feeding among the subpopulations previously identified in the spheroids of MCF7 breast cancer. By calibrating our model with expression profiles and the experimental growth rate, we concluded that the reverse Warburg effect emerges as a mechanism to optimize community growth. Furthermore, through an in silico analysis, we identified lactate, alpha-ketoglutarate, and some amino acids as key metabolites whose disponibility alters the growth rate of the spheroid. Altogether, this work provides a strategy for assessing how space and intratumoral heterogeneity influence the metabolic robustness of cancer, issues suggesting that computational strategies should move toward the design of optimized treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c54b7152ffb9e525c998dbbf7f6c9c4f3769b3" target='_blank'>
              Intratumoral Heterogeneity and Metabolic Cross-Feeding in a Three-Dimensional Breast Cancer Culture: An In Silico Perspective
              </a>
            </td>
          <td>
            Jorge E. Arellano-Villavicencio, Aarón Vázquez-Jiménez, J. J. Oropeza-Valdez, Cristian Padron-Manrique, H. Prado-García, Armando R. Tovar, O. Reséndis-Antonio
          </td>
          <td>2024-10-10</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0102937166e631177aafa289ad76311941f3d10" target='_blank'>
              Genomic and immune heterogeneity of multiple synchronous lung adenocarcinoma at different developmental stages
              </a>
            </td>
          <td>
            Yue Zhao, Jian Gao, Jun Wang, F. Fan, Chao Cheng, Danwen Qian, Ran Guo, Yang Zhang, Ting Ye, Marcellus Augustine, Yicong Lin, Jun Shang, Hang Li, Yunjian Pan, Qingyuan Huang, Haiqing Chen, Han Han, Zhendong Gao, Qiming Wang, Shiyue Zhang, Mou Zhang, F. Fu, Yueren Yan, Shanila Fernandez Patel, R. Vendramin, Hui Yuan, Yawei Zhang, Jiaqing Xiang, Hong Hu, Yihua Sun, Yuan Li, Kevin Litchfield, Zhiwei Cao, Haiquan Chen
          </td>
          <td>2024-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Galectin-9 is a multifaceted regulator of various pathophysiological processes that exerts positive or negative effects in a context-dependent manner. Here, we elucidated the distinctive functional properties of galectin-9 on myeloid cells within the brain tumor microenvironment. Galectin-9-expressing cells were abundant at the hypoxic tumor edge in the tumor-bearing ipsilateral hemisphere compared to the contralateral hemisphere in an intracranial mouse brain tumor model. Galectin-9 was highly expressed in microglia and macrophages in tumor-infiltrating cells. In primary glia, both the expression and secretion of galectin-9 were influenced by tumors. Analysis of a human glioblastoma bulk RNA-sequencing dataset and a single-cell RNA-sequencing dataset from a murine glioma model revealed a correlation between galectin-9 expression and glial cell activation. Notably, the galectin-9high microglial subset was functionally distinct from the galectin-9neg/low subset in the brain tumor microenvironment. Galectin-9high microglia exhibited properties of inflammatory activation and higher rates of cell death, whereas galectin-9neg/low microglia displayed a superior phagocytic ability against brain tumor cells. Blockade of galectin-9 suppressed tumor growth and altered the activity of glial and T cells in a mouse glioma model. Additionally, glial galectin-9 expression was regulated by Hif-2α in the hypoxic brain tumor microenvironment. Myeloid-specific Hif-2α deficiency led to attenuated tumor progression. Together, these findings reveal that galectin-9 on myeloid cells is an immunoregulator and putative therapeutic target in brain tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a40e47816f2629de4ffda7a5ee09bfdf58817c" target='_blank'>
              Galectin-9 Mediates the Functions of Microglia in the Hypoxic Brain Tumor Microenvironment.
              </a>
            </td>
          <td>
            Chanju Lee, Dahee Yu, Hyung-Seok Kim, Ki Sun Kim, Chi Young Chang, H. Yoon, Su Bin Won, Dae Yeon Kim, Eun Ah Goh, Yong Sun Lee, Jong Bae Park, Sang Soo Kim, Eun Jung Park
          </td>
          <td>2024-08-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e985501495b95f9eef3f50592dfe9ef56c04c3f" target='_blank'>
              Digital spatial profiling identifies the tumor center as a topological niche in prostate cancer characterized by an upregulation of BAD
              </a>
            </td>
          <td>
            Ann-Kathrin Huber, A. Kaczorowski, Felix Schneider, Sarah Böning, M. Görtz, David Langhoff, C. Schwab, Albrecht Stenzinger, Markus Hohenfellner, A. Duensing, Stefan Duensing
          </td>
          <td>2024-08-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Immune evasion is not only critical for tumor initiation and progression, but also determines the efficacy of immunotherapies. Through iterative in vivo CRISPR screens with seven syngeneic tumor models, we identified core and context-dependent immune evasion pathways across cancer types. This valuable high-confidence dataset is available for the further understanding of tumor intrinsic immunomodulators, which may lead to the discovery of effective anticancer therapeutic targets. With a focus on triple-negative breast cancer (TNBC), we found that Mga knock-out significantly enhances antitumor immunity and inhibits tumor growth. Transcriptomics and single-cell RNA sequencing analyses revealed that Mga influences various immune-related pathways in the tumor microenvironment. Our findings suggest that Mga may play a role in modulating the tumor immune landscape, though the precise mechanisms require further investigation. Interestingly, we observed that low MGA expression in breast cancer patients correlates with a favorable prognosis, particularly in those with active interferon-γ signaling. These observations provide insights into tumor immune escape mechanisms and suggest that further exploration of MGA's function could potentially lead to effective therapeutic strategies in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a148d1222506ce8171cce7b1609e8c01da8e2a" target='_blank'>
              In vivo CRISPR screens identify Mga as an immunotherapy target in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Xu Feng, Chang Yang, Yuanjian Huang, Dan Su, Chao Wang, L. L. Wilson, Ling Yin, Mengfan Tang, Siting Li, Zhen Chen, Dandan Zhu, Shimin Wang, Shengzhe Zhang, Jie Zhang, Huimin Zhang, Litong Nie, Min Huang, Jae-Ll Park, Traver Hart, Dadi Jiang, Kuirong Jiang, Junjie Chen
          </td>
          <td>2024-09-19</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 The tumor immune microenvironment shapes breast tumor evolution. We previously described the ductal carcinoma in situ (DCIS) to invasive breast carcinoma (IBC) transition as an evolutionary bottleneck and a key step of immune escape. Here, we dissected the cellular composition and spatial topology of DCIS and IBC patient samples and rat tumors that faithfully recapitulate human breast cancer. We identified differences in T cells as the main distinguishing feature between DCIS and IBC and discovered a novel cycling regulatory T cell population (cycTreg) as the primary orchestrator of the immune suppressive environment during invasive progression, especially enriched in triple-negative histological subtype. We showed that cycTregs are locally expanding Treg precursors in response to increased antigen presentation, and further demonstrated that modulating them by OX40 agonist in combination with anti-PD-L1 enhances CD8+ T cell infiltration and suppresses tumor growth. Characterizing therapeutic responses at the single cell level revealed a Treg and cancer-associated fibroblast (CAF) crosstalk via IL33-ST2 pathway that is critical for Treg expansion. Multimodal analysis of the patient cohorts demonstrated that cycTregs predict recurrence in low-grade DCIS and poor patient outcomes in IBC, and that increased CAF-immune-DCIS proximity is prognostic of progression to IBC. In sum, our study establishes cycTregs as key drivers of immune escape during invasive progression and implies that their targeting is essential for enhancing the efficacy of immune checkpoint inhibitors in IBC.
 Citation Format: Triet Minh Bui, Ernesto A Rojas, Pierre Foidart, Zheqi Vaciry Li, Pengze Yan, Jun Nishida, Robert West, Shelley Hwang, Simona Cristea, Kornelia Polyak. Cycling T regulatory cells architect an immune escape hub during in situ to invasive breast carcinoma transition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303a814c313c222a2e5b96739715f771fd2858ed" target='_blank'>
              Abstract A028: Cycling T regulatory cells architect an immune escape hub during in situ to invasive breast carcinoma transition
              </a>
            </td>
          <td>
            Triet Bui, Ernesto Rojas, P. Foidart, Zheqi Li, Pengze Yan, Jun Nishida, Robert West, Shelley Hwang, Simona Cristea, K. Polyak
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="The immune system exhibits spatial diversity in in vivo tissues. Immune cells are strategically distributed within tissues to maintain the organ integrity. Advanced technologies such as intravital imaging and spatial transcriptomics have revealed the spatial heterogeneity of immune cell distribution and function within organs such as the liver, kidney, intestine, and lung. In addition, these technologies visualize nutrient and oxygen environments across tissues. Recent spatial analyses have suggested that a functional immune niche is determined by interactions between immune and non-immune cells in an appropriate nutrient and oxygen environment. Understanding the spatial communication between immune cells, environment, and surrounding non-immune cells is crucial for developing strategies to control immune responses and effectively manage inflammatory diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85012b1d0f74509d0584679c5295a88b2bdb94a9" target='_blank'>
              Spatial diversity of in vivo tissue immunity.
              </a>
            </td>
          <td>
            Yu Miyamoto, Masaru Ishii
          </td>
          <td>2024-08-23</td>
          <td>International immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a6029311c7be5cdea36f9e6f715640956c9f739" target='_blank'>
              Biophysically interpretable inference of cell types from multimodal sequencing data
              </a>
            </td>
          <td>
            T. Chari, Gennady Gorin, L. Pachter
          </td>
          <td>2024-09-01</td>
          <td>Nature computational science</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6a7400a55e661d30ea0a596e56eadc3b19d76c4" target='_blank'>
              Single-cell RNA-sequencing reveals a unique landscape of the tumor microenvironment in obesity-associated breast cancer.
              </a>
            </td>
          <td>
            Guanghui Zhao, Xiaodong Zhang, Liying Meng, Ke Dong, Shipeng Shang, Tengfei Jiang, Ziqian Liu, Haidong Gao
          </td>
          <td>2024-09-16</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a lethal cancer lacking effective drugs and therefore new treatment targets are needed. Transcriptomic analysis comparing human embryonic and PDAC tissue identified a large overlap of expression profiles suggesting a re-initiation of developmental programs in pancreatic cancer. Specifically, we identified the transcription factors HOXB6 and HOXB8 as potential key regulators in PDAC. Loss of HOXB6 and HOXB8 in pancreatic cancer cells inhibited cell proliferation, induced apoptosis and senescence and enhanced gemcitabine sensitivity. Moreover, reduced HOXB6 and HOXB8 expression in pancreatic and lung adenocarcinoma cell lines affected transcription of immune response pathways which resulted in an increased sensitivity of cancer cells to anti-tumorigenic activities of macrophages suggesting that the HOXB6 and HOXB8 immune regulatory pattern is conserved in different cancer types. Additionally, naïve M0 macrophages exposed to HOXB8 deficient PDAC cells were unable to differentiate into tumor associated macrophages, suggesting that HOXB8 promotes the transition of initial anti-tumor macrophage to a tumor-promoting macrophage phenotype in pancreatic cancer. Our findings indicate that HOXB6 and HOXB8 play important roles in regulating cell proliferation, immune response and treatment resistance to promote pancreatic cancer tumorigenesis and could be useful therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36fecdcad1619bbba1d92b8c90a5731e5af6c417" target='_blank'>
              HOXB6 and HOXB8 control immune-cancer cell interactions in pancreatic cancer
              </a>
            </td>
          <td>
            Ludivine Bertonnier-Brouty, Kavya Achanta, Jonas Andersson, Sara Bsharat, Tania Singh, T. Kaprio, J. Hagström, C. Haglund, H. Seppänen, R. Prasad, Isabella Artner
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Acute myeloid leukaemia (AML) is a type of blood and bone marrow cancer characterized by the proliferation of abnormal clonal haematopoietic cells in the bone marrow leading to bone marrow failure. Over the course of the disease, angiogenic factors released by leukaemic cells drastically alter the bone marrow vascular niches resulting in observable structural abnormalities. We use a technique from topological data analysis - persistent homology - to quantify the images and infer on the disease through the imaged morphological features. We find that persistent homology uncovers succinct dissimilarities between the control, early, and late stages of AML development. We then integrate persistent homology into stage-dependent Gaussian mixture models for the first time, proposing a new class of models which are applicable to persistent homology summaries and able to both infer patterns in morphological changes between different stages of progression as well as provide a basis for prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/908fdd9f4a85fe4603f9b7353c67f06a0aab3abc" target='_blank'>
              A Topological Gaussian Mixture Model for Bone Marrow Morphology in Leukaemia
              </a>
            </td>
          <td>
            Qiquan Wang, Anna Song, A. Batsivari, Dominique Bonnet, Anthea Monod
          </td>
          <td>2024-08-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Intraductal papillary mucinous neoplasms (IPMNs), a common precursor to pancreatic ductal adenocarcinoma, present as a spectrum of benign indolent behavior and progression to invasive carcinoma, which highlights the importance of identifying biomarkers of IPMN progression. This study utilized whole transcriptome spatial transcriptomics to define the molecular characteristics of Epithelial, Immune, and Fibroblast cell populations of the three distinct histological sub-types of IPMN to understand the dynamics between the cell types and identify biomarkers of neoplastic progression. Fifty-two surgically resected IPMN cores from 21 patients with resected IPMNs comprising 24 Gastric, 14 Intestinal, and 14 Pancreaticobiliary with a distribution of low-grade dysplasia (LGD), high-grade dysplasia (HGD), and Invasive carcinoma (IC) were analyzed. Epithelial, Immune, and Fibroblast components were analyzed using the GeoMx spatial profiler. Unsupervised hierarchical clustering of epithelial areas of interest (AOI) identified 3 distinct clusters demonstrating a spectrum of molecular phenotypes in IPMN that were correlated with the three histological subtypes. Comparing the gene expression profile based on the shared genes expressed on all three subtypes, the Intestinal type tends to have an overexpression of genes whereas the Pancreaticobiliary represents a repressed phenotype relative to the Gastric phenotype. Supervised clustering using Moffitt’s classifier to identify the Epithelial-Mesenchymal transition demonstrates the Pancreaticobiliary subtype to be more Basal-like versus the Gastric and Intestinal inclined towards the classical phenotype. Supervised clustering based on the degree of dysplasia within the epithelial compartments of each of these subtypes, demonstrated a spectrum of molecular heterogeneity and identified biomarkers specific to High-Grade dysplasia in the Gastric and Intestinal subtype. Immune AOI deconvolution analysis demonstrated that Tregs were associated with Pancreaticobiliary, Macrophage enrichment associated with the Intestinal phenotype, while there is enrichment of CD8 T-cell and CD4 T-cell gene signatures associated with the Gastric subtype. Fibroblast AOI analyses demonstrated heterogenous myCAF- iCAF signatures between the subtypes. In summary, Spatial whole transcriptome analysis of IPMN identifies molecular signatures in epithelial and immune compartments that differentiate between the three histological subtypes of IPMN and helps understand their progression from Low-grade dysplasia to High-grade dysplasia.
 Citation Format: Amaya Pankaj, Michael J Raabe, Yuhui Song, Bidish K Patel, Katherine Xu, Joshua R Kocher, Nicholas J Caldwell, Maria L Ganci, Linda T Nieman, Mari Mino-Kenudson, Vikram Deshpande, Nabeel Bardeesy, Carlos Fernandez-Del Castillo, Martin J Aryee, David T Ting, Yasmin G Hernandez-Barco. Spatial Transcriptomics Unveils Intraductal Papillary Mucinous Neoplasm Heterogeneity: From Novel Clusters to Immune Dynamics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B091.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a43f4d0c8d98a15ce28e74661a5da83f45aa30c" target='_blank'>
              Abstract B091: Spatial Transcriptomics Unveils Intraductal Papillary Mucinous Neoplasm Heterogeneity: From Novel Clusters to Immune Dynamics
              </a>
            </td>
          <td>
            Amaya Pankaj, Michael J. Raabe, Yuhui Song, Bidish Kumar Patel, Katherine Xu, Joshua R. Kocher, Nicholas J Caldwell, M. L. Ganci, Linda T. Nieman, M. Mino-Kenudson, Vikram Deshpande, N. Bardeesy, C. F. Fernandez-Del Castillo, Martin J. Aryee, David T. Ting, Yasmin G Hernandez-Barco
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Bladder cancer (BCa) is one of the most challenging and costly cancers to treat, yet little progress has been made on the development of predictive biomarkers and targeted therapies. Here, we uncover a critical function of Numb as a tumor suppressor in the bladder, identifying loss of Numb expression as a causal alteration in BCa that underlies biological aggressiveness and disease progression. Through retrospective cohort studies, we established that a Numb-deficient tumor status correlates with worse overall survival in post-cystectomy muscle-invasive bladder cancer (MIBC) patients and increased risk of MIBC progression in non-muscle-invasive bladder cancer (NMIBC) patients. The prognostic value of Numb loss can be attributed to its crucial role as a determinant of aggressive bladder tumorigenesis, as demonstrated in mouse and human models. Targeted Numb ablation in the basal layer of the urothelium was alone sufficient to trigger spontaneous bladder tumorigenesis and drive progression from preneoplastic to preinvasive and, ultimately, overtly invasive tumors. Additionally, Numb ablation sensitized the urothelium to other oncogenic insults, accelerating tumor onset and progression. Using 3D-Matrigel organoid cultures to recapitulate bladder tumorigenesis in vitro, we found that Numb loss heightens the proliferative and invasive potential of both mouse and human BCa cells. Integrative transcriptomic and functional analyses revealed that downregulation of the canonical Hippo pathway, resulting in enhanced YAP transcriptional activity, underlies the biological aggressiveness of Numb-deficient BCa. These molecular events are dependent on the activation of RhoA/ROCK signaling subsequent to Numb loss. Thus, a dysfunctional Numb–RhoA/ROCK–Hippo/YAP regulatory network is at play in aggressive Numb-deficient BCa and represents a therapeutic vulnerability. A 27-gene prognostic signature capable of identifying high-risk Numb-deficient patients could provide the basis of a clinical tool to stratify patients for innovative RhoA/ROCK/YAP targeted therapies. One Sentence Summary Numb loss-directed hyperactivation of RhoA/ROCK/YAP underlies aggressive bladder cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72434e6e063f988f837149b1be62c13ce13452e2" target='_blank'>
              Loss of the tumor suppressor NUMB drives aggressive bladder cancer through hyperactivation of a RhoA/ROCK/YAP signaling circuitry
              </a>
            </td>
          <td>
            F. A. Tucci, R. Pennisi, D. Rigiracciolo, M. G. Filippone, R. Bonfanti, F. Romeo, S. Freddi, C. Soriani, S. Rodighiero, G. Jodice, F. Sanguedolce, G. Renne, N. Fusco, P. Di Fiore, G. Pruneri, G. Musi, G. Vago, D. Tosoni, S. Pece
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac67a3ed7d66d72ed2d1b0afc516f030fad809a" target='_blank'>
              Immunological hide-and-seek: epigenetically reprogrammed cancer cells and the dynamics of CD8+ T cells.
              </a>
            </td>
          <td>
            Jie-Ting Low, M. W. Chan, Cheng-Huang Shen, Kuo-Liang Wei
          </td>
          <td>2024-09-04</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Coronary atherosclerotic plaques susceptible to acute coronary syndrome have traditionally been characterized by their surrounding cellular architecture. However, with the advent of intravascular imaging, novel mechanisms of coronary thrombosis have emerged, challenging our contemporary understanding of acute coronary syndrome. These intriguing findings underscore the necessity for a precise molecular definition of plaque stability. Considering this, our study aimed to investigate the vascular microenvironment in patients with stable and unstable plaques using spatial transcriptomics.


METHODS
Autopsy-derived coronary arteries were preserved and categorized by plaque stability (n=5 patients per group). We utilized the GeoMx spatial profiling platform and Whole Transcriptome Atlas to link crucial histological morphology markers in coronary lesions with differential gene expression in specific regions of interest, thereby mapping the vascular transcriptome. This innovative approach allowed us to conduct cell morphological and spatially resolved transcriptional profiling of atherosclerotic plaques while preserving crucial intercellular signaling.


RESULTS
We observed intriguing spatial and cell-specific transcriptional patterns in stable and unstable atherosclerotic plaques, showcasing regional variations within the intima and media. These regions exhibited differential expression of proinflammatory molecules (eg, IFN-γ [interferon-γ], MHC class II, proinflammatory cytokines) and prothrombotic signaling pathways. By using lineage tracing through spatial deconvolution of intimal CD68+ (cluster of differentiation 68) cells, we characterized unique, intraplaque subpopulations originating from endothelial, smooth muscle, and myeloid lineages with distinct regional locations associated with plaque instability. In addition, unique transcriptional signatures were observed in vascular smooth muscle and CD68+ cells among plaques exhibiting coronary calcification.


CONCLUSIONS
Our study illuminates distinct cell-specific and regional transcriptional alterations present in unstable plaques. Furthermore, we characterize the first spatially resolved, in situ evidence supporting cellular transdifferentiation and intraplaque plasticity as significant contributors to plaque instability in human coronary atherosclerosis. Our results provide a powerful resource for the identification of novel mediators of acute coronary syndrome, opening new avenues for preventative and therapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a73fe256c8ad5204b69d364f2110a060834134" target='_blank'>
              Spatial Transcriptomic Approach to Understanding Coronary Atherosclerotic Plaque Stability.
              </a>
            </td>
          <td>
            M. Gastanadui, Camilla Margaroli, Silvio Litovsky, Robert P Richter, Dezhi Wang, D.D. Xing, J. Wells, Amit Gaggar, V. Nanda, Rakesh P Patel, Gregory A Payne
          </td>
          <td>2024-09-05</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Triple-negative breast cancer (TNBC) is the most aggressive subtype of breast cancer. TNBC displays exceptional transcriptional heterogeneity, both between patients and intratumorally, and disproportionately affects underserved women. Hypoxia, a state of low oxygen levels, is often present in the solid tumor microenvironment. In TNBC, hypoxia is a key driver of aggressiveness and therapeutic resistance. Past bulk RNA sequencing (RNAseq) studies in cancer research have lacked spatial resolution, failing to uncover precise transcriptional differences within tumor regions. This spatial limitation has been especially problematic for studying transcriptional heterogeneity of the cellular hypoxia response within tumors. By integrating single-cell RNA sequencing and spatial transcriptomic analyses in a TNBC xenograft tumor model, our research has uncovered a novel hypoxia gene signature, specific to cancer cells. Elevated expression of this hypoxia signature predicts poorer prognosis in breast cancer patients, especially within TNBC patients and those of African ancestry. This suggests that hypoxia may play a role in the racial disparities observed in TNBC prognosis, potentially informing treatment strategies. This workflow introduces a novel framework for translating general spatial transcriptomics data into clinically relevant and accurate gene signatures.
 Citation Format: Kent Schechter, Long C. Nguyen, Geetha P. Yerradoddi, Rania Bassiouni, John D. Carpten, Marsha R. Rosner. Novel cancer-specific hypoxia signature predicts poor prognosis and highlights racial disparities in breast cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C105.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43ba633337a4ee769f8bd92a6f7364c821869174" target='_blank'>
              Abstract C105: Novel cancer-specific hypoxia signature predicts poor prognosis and highlights racial disparities in breast cancer
              </a>
            </td>
          <td>
            Kent Schechter, Long C Nguyen, G. P. Yerradoddi, R. Bassiouni, John D. Carpten, M. Rosner
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Advances in omics technologies have allowed spatially resolved molecular profiling of single cells, providing a window not only into the diversity and distribution of cell types within a tissue but also into the effects of interactions between cells in shaping the transcriptional landscape. Cells send chemical and mechanical signals which are received by other cells, where they can subsequently initiate context-specific gene regulatory responses. These interactions and their responses shape the individual molecular phenotype of a cell in a given microenvironment. RNAs or proteins measured in individual cells, together with the cells' spatial distribution, provide invaluable information about these mechanisms and the regulation of genes beyond processes occurring independently in each individual cell. SpaCeNet is a method designed to elucidate both the intracellular molecular networks (how molecular variables affect each other within the cell) and the intercellular molecular networks (how cells affect molecular variables in their neighbors). This is achieved by estimating conditional independence relations between captured variables within individual cells and by disentangling these from conditional independence relations between variables of different cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8ef214444a5271219b33a7d73c5e44038be1f04" target='_blank'>
              Spatial Cellular Networks from omics data with SpaCeNet.
              </a>
            </td>
          <td>
            Stefan Schrod, Niklas Lück, R. Lohmayer, S. Solbrig, Dennis Völkl, Tina Wipfler, K. Shutta, Marouen Ben Guebila, Andreas Schäfer, Tim Beißbarth, Helena U. Zacharias, Peter J Oefner, John Quackenbush, M. Altenbuchinger
          </td>
          <td>2024-09-04</td>
          <td>Genome research</td>
          <td>2</td>
          <td>12</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCL) are highly aggressive tumors. Their genetic complexity and heterogeneity have hampered the development of novel approaches for precision medicine. Our study aimed to develop a personalized therapy for DLBCL by utilizing the CRISPR/Cas system to induce knockouts (KO) of driver genes, thereby causing cancer cell death while minimizing side effects. We focused on OCI-LY3 cells, modeling DLBCL, and compared them with BJAB cells as controls. Analysis of whole exome sequencing revealed significant mutations in genes like PAX5, CD79B, and MYC in OCI-LY3 cells. CRISPR/Cas9-mediated KO of these genes resulted in reduced cancer cell viability. Subsequent single and dual gRNA targeting of PAX5 mutations inhibited proliferation specifically in OCI-LY3 cells. Moreover, dual gRNA targeting of PAX5 and MYC induced chromosomal rearrangements, reducing cell proliferation substantially. However, targeting single intronic mutations did not affect cell viability, highlighting the importance of disrupting protein function. Targeting multiple mutations simultaneously addresses intra-tumoral heterogeneity, and the transient delivery of CRISPR/Cas9 allows for permanent gene disruption. While challenges such as incomplete editing efficiency and delivery limitations exist, further optimization may enhance therapeutic efficacy. Overall, our findings demonstrate the efficacy of CRISPR/Cas9 in targeting oncogenic mutations, opening avenues for precision medicine in DLBCL treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6638da8f9e731128d18c1e54c6964f784156f835" target='_blank'>
              Targeting DLBCL by mutation-specific disruption of cancer-driving oncogenes
              </a>
            </td>
          <td>
            Najmeh Heshmatpour, S. M. Kazemi, Niklas D. Schmidt, Sarita R. Patnaik, Patrick Korus, Bodo G. C. Wilkens, Arturo Macarrón Palacios
          </td>
          <td>2024-10-14</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Adult glioma patients rarely benefit from immunotherapy. This lack of efficacy has been related to gliomas immune “cold” microenvironment, which is commonly observed in these tumors and reduces lymphocyte-mediated antitumor responses. Here, we investigated the immune microenvironment of rare adult gliomas with high lymphocytic infiltration in order to evaluate the biology of this unusual immune response, its prognostic value and its potential role as predictive biomarker for immunotherapy strategies.



 Two retrospective cohorts of adult gliomas were analyzed: a supervised cohort of 206 samples enriched for tumors with high lymphocytic infiltration (including glioblastoma IDH wildtype [GBM], gemistocytic variant of astrocytoma IDH-mutant, ganglioglioma [GGG] and pleomorphic xanthoastrocytoma [PXA], and hypermutated mismatch repair deficient gliomas [MMRd]), and a consecutive cohort of 112 samples as control of the usual epidemiology. Immune microenvironment was characterized through (i) an analysis of spatial patterns and semi-automated quantification of T lymphocytes on CD3 immunolabeling and (ii) deconvolution of immune cells content from bulk RNA sequencing of FFPE tumor tissue. Cellular and spatial patterns of immune infiltration were correlated with demographic, histomolecular and outcome features. We investigated the underlying biology through multiplex immunolabeling and single-cell RNAseq of FACS-sorted intratumoral immune cells. Results: We identified three main spatial patterns (perivascular, peritumoral, diffuse intratumoral) of T cells infiltration in addition to the most usual aspect of scarce tumor-infiltrating lymphocytes. T cell proportion exhibited a wide range of distribution and was significantly associated with tumor type and IDH-status, with GBM, PXA and GGG exhibiting the highest proportions. Using transcriptomic signatures of immune cells and deconvolution of bulk RNAseq data, we observed recurrent patterns of associated immune cell types across diverse tumor types. We analyzed the correlations of these spatial and cellular immune patterns with tumor types and genetic alterations. Multiplex immunolabeling and single cell RNAseq revealed distinct T cells subpopulations, as well as potential functional states and cell-cell communication pathways driving these states. T cell infiltration had a significant prognostic impact among GBM patients.



 Adult gliomas with high lymphocytic content stands for a population of interest to better understand immune responses in the brain parenchyma and develop novel immunotherapy clinical trials. Our results suggest that therapeutic evaluation requires an adjustment due to the prognostic impact of immune microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/204034fedcd671ddafac89899df457a2d6c8c475" target='_blank'>
              OS12.7.A INTEGRATED DIGITAL PATHOLOGY AND TRANSCRIPTOMIC ANALYSIS OF ADULT GLIOMAS WITH HIGH LYMPHOCYTE INFILTRATION IDENTIFIES CELLULAR AND SPATIAL DETERMINANTS OF IMMUNE INFILTRATION WITH PROGNOSTIC VALUE
              </a>
            </td>
          <td>
            L. Salhi, J. Lerond, A. A. Nadaradjane, L. Drouiche, M. M. Otero Pizarro, L. Hanna Doumit Sakr, K. Labrèche, S. Kacimi, C. Bellamy, A. Picca, L. Stimmer, J. Brocic, Q. Letourneur, B. Barka, C. Carpentier, Y. Marie, H. Madry, J. Bhalshankar, R. Nicolle, H. Malaizé, B. Mathon, K. Mokhtari, A. Idbaih, K. Hoang-Xuan, M. Sanson, A. Alentorn, A. Duval, P. Le Rouzic, M. Dieu-Nosjean, I. Bloch, M. Touat, F. Bielle
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Ionizing radiation exposure can cause damage to diverse tissues and organs, with the hematopoietic system being the most sensitive. However, limited information is available regarding the radiosensitivity of various hematopoietic cell populations in the bone marrow due to the high heterogeneity of the hematopoietic system. In this study, we observed that granulocyte–macrophage progenitors, hematopoietic stem/progenitor cells, and B cells within the bone marrow showed the highest sensitivity, exhibiting a rapid decrease in cell numbers following irradiation. Nonetheless, neutrophils, natural killer (NK) cells, T cells, and dendritic cells demonstrated a certain degree of radioresistance, with neutrophils exhibiting the most pronounced resistance. By employing single-cell transcriptome sequencing, we investigated the early responsive genes in various cell types following irradiation, revealing that distinct gene expression profiles emerged between radiosensitive and radioresistant cells. In B cells, radiation exposure led to a specific upregulation of genes associated with mitochondrial respiratory chain complexes, suggesting a connection between these complexes and cell radiosensitivity. In neutrophils, radiation exposure resulted in fewer gene alterations, indicating their potential for distinct mechanisms in radiation resistance. Collectively, this study provides insights into the molecular mechanism for the heterogeneity of radiosensitivity among the various bone marrow hematopoietic cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb664766831a6e9ac2aeeb0e263cf03abab8ee2" target='_blank'>
              Single-Cell Transcriptomics Reveals Early Effects of Ionizing Radiation on Bone Marrow Mononuclear Cells in Mice
              </a>
            </td>
          <td>
            Yun-Qiang Wu, Ke-Xin Ding, Zhi-Chun Lv, Zheng-Yue Cao, Ke Zhao, Huiying Gao, Hui-Ying Sun, Jing-Jing Li, Si-Yu Li, Xiong-Wei Zhao, Yang Xue, Shen-Si Xiang, Xiao-Fei Zheng, Xiao-Ming Yang, Chang-Yan Li
          </td>
          <td>2024-08-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb1f558b1760a8e01aa262dc80b32d0f2f31634b" target='_blank'>
              CISD3 is a prognostic biomarker and therapeutic target in pan-cancer
              </a>
            </td>
          <td>
            Jie Li, Han Yang, Yixin Qi, Peng Yu, Xiahui Han, Z. Zhang, Kai Zhao, Xinbao Yin, Guanqun Zhu, Xuechuan Yan, Zaiqing Jiang, Xuezhen Ma, Tianzhen He, Ke Wang
          </td>
          <td>2024-10-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The concept of cancer stemness is undergoing rapid evolution, facilitated by technological advances allowing the tracking and elimination of stem cells in vivo. Following their identification in solid tumors, cancer stem cells were placed at the center of tumor initiation, growth and metastasis. However, increasing evidence suggests that stemness is a cellular state that can be acquired by differentiated cells in a process called plasticity. Here we show that CD44v6 may act as a molecular switch controlling plasticity of colorectal cancer cells. CD44v6/MET signaling controls the reappearance of Lgr5+ cells after ablation, in vitro and in vivo as demonstrated in tumor organoids derived from Lgr5DTR/eGFP mice using a CD44v6 blocking peptide. CD44v6 also affects the YAP/TAZ signaling pathway, involved in the very first steps of plasticity. In view of the essential role of plasticity for the establishment of metastases, blocking CD44v6 signaling may represent a pivotal and promising therapy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04cb2eb85c74641a51555dede7b20eb7d8aa317e" target='_blank'>
              Breaking the cycle: How targeting CD44v6/MET signaling disrupts colorectal cancer cell plasticity
              </a>
            </td>
          <td>
            S. J. Sonnentag, S. Goldbach, D. Hoch, G. Andrieux, J. Becker, H. Fromm, L. Munoz-Sagredo, Y. Heneka, D. Grimm, F. R. Greten, V. Orian-Rousseau
          </td>
          <td>2024-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a030314e31860990960812246eddebd6ed05b0d" target='_blank'>
              Immunotherapy and pan-apoptotic characterization of the tumor microenvironment in gastric cancer (STAD): a single-cell multidimensional analysis
              </a>
            </td>
          <td>
            Sheng Zhang, Jianhong Wang, Huan Zhang, Benhua Li, Shun Gao
          </td>
          <td>2024-10-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3edc3fd4bae12ec25e30e258ed04a39388899eef" target='_blank'>
              Multi-omics analysis unveils the predictive value of IGF2BP3/SPHK1 signaling in cancer stem cells for prognosis and immunotherapeutic response in muscle-invasive bladder cancer
              </a>
            </td>
          <td>
            Yaobang Wang, Wuyue Song, Chao Feng, Shulin Wu, Zezu Qin, Tao Liu, Yu Ye, Rong Huang, Yuanliang Xie, Zhong Tang, Qiuyan Wang, Tianyu Li
          </td>
          <td>2024-10-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumors in patients non-responsive to immunotherapy harbor a series of barriers that impede the efficacy of effector T-cells. Consequently, therapeutically modulating the chemotaxis machinery to enable effector T cell infiltration and function in the tumor could result in more successful therapeutic outcomes. Complex in-vitro models allow re-creation of in-vivo tumor complexities in an in-vitro setting, allowing improved translatability to patient biology at the laboratory scale. We identified a gap in available industrial scale microphysiological (MPS) assays for faster validation of targets and strategies that enable T-cell chemotaxis and effector function within tumor microenvironments. Using a commercially available, 96 -chip 2-lane microfluidic assay system, we present a novel, scalable, complex in vitro microphysiological assay to study 3D T-cell chemotaxis and function within native, extracellular matrix (ECM)-rich multicellular tumor environments. Activated or naïve CD3+ T-cells stained with far-red nuclear stain responded to the chemokine gradients generated within the matrigel-collagen ECM by migrating into the microfluidic channel (~5 mm horizontal window), in a concentration- and cell type-dependent manner. Furthermore, we observed and tracked chemotaxis and cancer cell killing function of antigen-specific CD4.CD8.CAR-T cells (chimeric antigen receptor (CAR)-T cells) that responded to CXCR3 agonist gradient built through the expansive 5 mm of cancer cell colony containing stroma. The 2-lane assay system yielded useful information regarding donor and dose-dependent differences in CAR-T cell chemotaxis and tumor killing. The scalable assay system allows a granular window into immune cell migration and function in tissue spaces beyond endothelium, addressing a missing gap in studying tissue-specific immune cell chemotaxis and function to bring forward advancements in cancer immunotherapy. .">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eca169b0affeb07236df28d0336bb6204d0626e" target='_blank'>
              A microphysiological assay for studying T-cell chemotaxis, trafficking and tumor killing.
              </a>
            </td>
          <td>
            Taraka Sai Pavan Grandhi, Makda Mebrahtu, Ryan Musso, Alexis Fullman, Brady Nifong, Katrina Wisdom, Terrence T Roh, Matthew Sender, Derek Poore, Claire E Macdougall, Ravit Oren, Sue Griffin, Aaron Cheng, Jason E. Ekert
          </td>
          <td>2024-10-08</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Background Research on circulating tumor cells (CTCs) offers the opportunity to better understand the initial steps of blood‐borne metastasis as main cause of cancer‐related deaths. Here, we have used the colon cancer CTC‐MCC‐41 and breast cancer CTC‐ITB‐01 lines, which were both established from human CTCs as permanent cell lines as models to further study CTC biology with special emphasis on anchorage‐independent survival and growth. Methods and Results Both cell lines showed a marked intrinsic plasticity to switch between suspension and adherent in vitro growth, in 2D adherent culture conditions, and established an equilibrium of both growth patterns with predominant adherent cells in the CTC‐MCC‐41 line (77%) and suspension cells in the CTC‐ITB‐01 line (85%). Western blot analysis revealed a higher expression of pERK1/2 in CTC‐ITB‐01 adherent cells compared to the suspension counterpart that suggested the involvement of kinases in this process. Subsequent functional kinome profiling identified several serine/threonine as well as tyrosine kinases that were differentially regulated in adherent and suspension CTCs. In the adherent cells of the breast cancer line CTC‐ITB‐01 the activity of MSK1, Src family kinases and the PKG family was increased compared to the suspension counterpart. In adherent cells of the colorectal CTC‐MCC‐41 line, an increased activity of TYRO3 and JAK2 was detected, whereas p38 MAPK was strongly impaired in the suspension CTC‐MCC‐41 cells. Some of the regulated kinases, which include the Src family, TYRO3, MSK1, JAK2 and p38 MAPK, have been associated with crucial cellular processes including proliferation, migration and dormancy in the past. Conclusions The investigated CTC lines exhibit a high plasticity, similar to the concept of ‘adherent‐to‐suspension transition (AST)’ that was recently suggested as a new hallmark of tumor biology by Huh et al. Moreover, we identified differentially regulated kinome profiles that may represent potential targets for future studies on therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9bc4607b04aee13d28e6301790ec63b75171c7" target='_blank'>
              Analysis of the Plasticity of Circulating Tumor Cells Reveals Differentially Regulated Kinases During the Suspension‐to‐Adherent Transition
              </a>
            </td>
          <td>
            Daniel J. Smit, K. Hoffer, Bettina Bettin, M. Kriegs, L. Cayrefourcq, Udo Schumacher, Klaus Pantel, C. Alix-Panabières, M. Jücker
          </td>
          <td>2024-10-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b34bf47262137f174937f9ce2b2938693cc40a" target='_blank'>
              Genome-wide RNA-Seq identifies TP53-mediated embryonic stem cells inhibiting tumor invasion and metastasis
              </a>
            </td>
          <td>
            Yatong Li, Yongna Fan, Yunyi Xie, Limin Li, Juan Li, Jingyi Liu, Zhengyu Jin, Huadan Xue, Zhiwei Wang
          </td>
          <td>2024-10-16</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Investigating oncogenes and the mechanisms driving oncogenic processes in human tumors is imperative for the development of efficient therapies. Peroxidasin (PXDN) has been reported to play a critical role in tissue development and homeostasis. However, the role of PXDN in the occurrence and development of Nasopharyngeal Carcinoma (NPC) remains unknown. Methods Data from multiple databases, including GEO and TCGA, were used to analyze the expression levels of PXDN. Taking nasopharyngeal carcinoma as an example, in vitro experiments were conducted to explore the biological functions of PXDN. Overexpression of stable cell lines was achieved through lentiviral infection, cell proliferation was examined using CCK8 and BrdU incorporation assays, and clone formation experiments were performed to assess cell growth. Transwell and wound healing assays were employed to evaluate cell invasion and migration abilities. Additionally, immunofluorescence staining with multiple targets was used to analyze the immune microenvironment of the tumor tissues. Co-culture experiments, followed by clone formation and CFSE incorporation assays, were conducted to observe the impact of NPC stable cell lines on T cells. Flow cytometry was performed to detect surface markers and cytokines in T cells after co-culture to assess T cell function. Results PXDN was highly expressed in multiple tumors, and its high expression and mutation profile were correlated with poor survival. Functionally, PXDN plays a crucial role in promoting oncogenic processes by enhancing NPC cell proliferation and metastasis. Mechanistically, PXDN activates extracellular matrix (ECM) signaling pathways while simultaneously inhibiting T-cell infiltration and activation, thereby facilitating cancer progression. Conclusion We characterized PXDN as a valuable biomarker for pan-cancer diagnosis and prognosis. We also uncovered new oncogenic roles for PXDN in promoting cancer progression and regulating T-cell immunosuppressive function in NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d0995530e52ad4e68be420d3100fec03cc1052" target='_blank'>
              PXDN as a pan-cancer biomarker and promotes tumor progress via immune inhibition in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Yu-Chun Li, Yan-Hong Xiao, Fei-Xiong Chen, Si-Yu Xiao, Jian-Mei Lin, Shu-Tao Cai, Cui-Lan Zeng, Xue-Yan Ye, Xu-Fa Yu, Li Yuan, Shi-Bing Li
          </td>
          <td>2024-09-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a prevalent malignant pediatric brain tumor with high mortality rates despite currentmultimodal therapies. MYC-driven Group 3 MB represents a highly aggressive subgroup with poor prognosis and limited treatment options. To systematically identify genes representing therapeutic vulnerabilities in MYC-driven MB, we performed a CRISPR-Cas9 screen with a specific focus on 1140 druggable genes across three MB cell lines. CDK8 emerged as a significantly selected gene crucial for MYC-driven MB growth, for which clinically relevant inhibitors are in early phase trials. CDK8 gene expression is high in group 3 MB (Colorado cohort) and specifically high in MYC driven MB subtypes and elevated expression of CDK8 is an adverse prognostic marker even in high-risk group 3 tumors. CDK8 is highly expressed in progenitor populations identified in our recent single-cell RNA-seq analysis and similarly expressed in cycling progenitors in murine models of MB. CDK8 depletion in MB cells using CRISPR-Cas9-induced knockout and lentivirus-mediated CDK8 shRNAs resulted in decreased i cell proliferation and MYC protein. Additionally, CDK8 depletion significantly decreased neurosphere growth in MB cells and supressed orthotopic xenograft growth in vivo. Evaluation of eight CDK8 selective inhibitors demonstrated a broad range of half-maximal inhibitory concentrations (IC50) across three G3-MB cell lines with RVU120 showing the maximum potency, RVu120 supressed colony growth in vitro in MB cells but not normal astrocytes. In vivo RVU120 potently supressed growth of orthotopic xenografts as evaluated by MRI and prolonged survival. RNA-seq analysis revelaed that CDK8 depletion leads to repression of mRNA translation and ribosome biogenesis. GSEA indicated that the loss of CDK8 resulted in strong negative enrichment in the transcriptomic profile associated with stemness gene sets and positive enrichment in differentiation gene sets. We performed a genome-wide analysis to map the occupancy of CDK8 and key histone markers using CUT&RUN in three G3-MB cell lines. We found that CDK8 functions as a transcriptional activator affecting the translation program. In vitro, inhibition of protein tranlation via mTOR supression was synergistic with CDK8 inhibition. We found that this synthetic lethality was relevant in vivo,where CDK8i and mTORi combined to supress xenograft growth and prolong survival. Taken together, these studies established the therapeutic efficacy of combination treatment with mTOR and CDK8 inhibitors in vivo and in vitro, opening an alternate path for biologically based therapeutic trials in MYC-driven MB.
 Citation Format: Dong Wang, Breunna Brunt, Bethany Veo, Milena Mazan, Tomasz Rzmyski, Sujatha Venkataraman, Nathan Dahl, Rajeev Vibhakar. Understanding the role of CDK8 in protein synthesis for treating MYC-driven medulloblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A082.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309ac06061039702b54427736670e453090c19d2" target='_blank'>
              Abstract A082 Understanding the role of CDK8 in protein synthesis for treating MYC-driven medulloblastoma
              </a>
            </td>
          <td>
            Dong Wang, Breunna Brunt, Bethany L. Veo, Milena Mazan, Tomasz Rzmyski, S. Venkataraman, N. Dahl, Rajeev Vibhakar
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Curcumin is widely recognized for its diverse antitumor properties, ranging from breast cancer to many other types of cancers. However, its role in the tumor microenvironment remains to be elucidated. In this study, we established a 3D tumor spheroids model that can simulate the growth environment of tumor cells and visualized the antitumor metabolic alteration caused by curcumin using mass spectrometry imaging technology. Our results showed that curcumin not only exerts a profound impact on the growth and proliferation of breast cancer cells but in situ multivariate statistical analysis also reveals the significant effect on the overall metabolic profile of tumor spheroids. Meanwhile, our visualization map characterized curcumin metabolic processes of reduction and glucuronidation in tumor spheroids. More importantly, abnormal metabolic pathways related to lipid metabolism and polyamine metabolism were also remodeled at the metabolite and gene levels after curcumin intervention. These insights deepen our comprehension of the regulatory mechanism of curcumin on the tumor metabolic network, furnishing powerful references for antitumor treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9708e466bdcc061f5dd608fcbfa98f940da2834e" target='_blank'>
              Spatial Metabolomics Profiling Reveals Curcumin Induces Metabolic Reprogramming in Three-Dimensional Tumor Spheroids
              </a>
            </td>
          <td>
            Zihan Zhu, Yaqi Zhang, Lei Wang, Haoyuan Geng, Min Li, Shiping Chen, Xiao Wang, Panpan Chen, Chenglong Sun, Chao Zhang
          </td>
          <td>2024-09-01</td>
          <td>Metabolites</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Tumor-associated macrophages and microglia (TAMs) are highly abundant myeloid cells in gliomas, with their phenotype and immune response determined by ontogeny and microenvironment. TAMs display distinctive transcriptional programs according to the IDH mutation status but the underlying regulatory mechanisms remain largely unknown. Herein, we uncover that CD11B+ myeloid cells in human IDHmut gliomas exhibit DNA hypermethylation predominantly at distal enhancers. This hypermethylation was linked to decreased expression of genes involved in inflammatory responses and glycolytic metabolism, and the inactivation of transcription factors that regulate microglial responses to environmental stimuli. Prolonged exposure of human primary microglia to D-2-hydroxyglutarate (D-2HG) inhibited TET-mediated 5mC oxidation, resulting in a reduced accumulation of global 5hmC levels. We confirmed high 5mC/5hmC ratios at lineage-specific enhancers, by analyzing CpGs at single-base resolution. D-2HG-treated microglia show reduced proinflammatory capacity and enhanced oxidative phosphorylation, consistent with the remodeled enhancer landscape. Conversely, depletion of D-2HG following treatment of a glioma patient with an IDH mutant inhibitor was associated with enhanced microglial responses, as assessed by snRNA-seq. Our findings provide a mechanistic rationale for the hyporesponsive state of microglia in IDHmut gliomas and support the concept that oncometabolites may disrupt the function of immune cells residing in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbc5ed6e8a9af5e3aa64ecf135e384212bae5953" target='_blank'>
              The oncometabolite D-2-hydroxyglutarate promotes DNA hypermethylation at lineage-specific enhancers controlling microglial activation in IDHmut gliomas
              </a>
            </td>
          <td>
            A. Laurenge, Pietro Pugliese, Q. Richard, B. Mathon, S. Jouannet, Y. Hayat, S. Scuderi, Laurent Capelle, P. Marijon, Karim Labreche, A. Alentorn, M. Verreault, A. Idbaih, C. Birzu, E. Huillard, N. Pottier, A. Desmons, I. Fayache, G. Kaas, P. Kingsley, Lawrence J. Marnett, E. Duplus, E. El-Habr, L. Salas, K. Mokhtari, S. Tran, M. Touat, Franck Bielle, M. Suvà, A. Iavarone, M. Ceccarelli, M. Mallat, Marc Sanson, L. Castro-Vega
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6392838cbd8fd43bfdbde70561f2b8df6cd16609" target='_blank'>
              A multidimensional analysis reveals distinct immune phenotypes and the composition of immune aggregates in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            J. Koedijk, I. van der Werf, Livius Penter, M. Vermeulen, F. Barneh, A. Perzolli, J. Meesters-Ensing, Dennis S Metselaar, T. Margaritis, M. Fiocco, H. D. de Groot-Kruseman, Rubina Moeniralam, Kristina Bang Christensen, Billie Porter, Kathleen L. Pfaff, Jacqueline S Garcia, Scott J. Rodig, Catherine J. Wu, Henrik Hasle, S. Nierkens, M. Belderbos, C. M. Zwaan, Olaf Heidenreich
          </td>
          <td>2024-08-26</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Alterations in Fibroblast growth factor receptor (FGFR)-family proteins frequently occur as oncogenes in many cancers, including a subset of pediatric gliomas. Here, we performed a genomic analysis of 11,635 gliomas across ages and found that 4.5% of all gliomas harbor FGFR alterations including structural variants (SV) and single nucleotide variants (SNV), with an incidence of almost 10% in pediatric gliomas. FGFR family members are differentially enriched by age, tumor grade, and histological subtype, with FGFR1-alterations associated with glioneuronal histologies and pediatric low-grade gliomas. Across development, we find FGFR1 expression in both neuronal and glial precursors, while FGFR3 expression is largely restricted to astrocytic lineages. Leveraging novel isogenic model systems, we confirm FGFR1 alterations to be sufficient to activate MAPK and mTOR signaling, drive gliomagenesis, activate neuronal transcriptional programs and exhibit sensitivity to MAPK pathway inhibitors, including pan-FGFR inhibitors. Models driven by FGFR1 SVs exhibited different patterns of sensitivity compared to those driven by SNVs. Finally, we performed a retrospective analysis of clinical responses in children diagnosed with FGFR-driven gliomas and found that targeted MAPK or FGFR-inhibition with currently available inhibitors is largely associated with stability of disease. This study provides key insights into the biology of FGFR1-altered gliomas, therapeutic strategies to target them and associated challenges that still need to be overcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badc242f4fb57f142e32352b142311d05135cc1f" target='_blank'>
              A diverse landscape of FGFR alterations and co-mutations defines novel therapeutic strategies in pediatric low-grade gliomas
              </a>
            </td>
          <td>
            Eric Morin, April Apfelbaum, D. Sturm, Georges Ayoub, Jeromy J. Digiacomo, Sher Bahadur, Bhavyaa Chandarana, Phoebe C Power, Margaret Cusick, Dana Novikov, Robert Jones, J. Vogelzang, Connor Bossi, Seth W. Malinowski, John Jeang, Jared Collins, Sehee Oh, Hyesung Jeon, A. Cameron, Patrick Rechter, Angela Deleon, K. Murugesan, M. Montesion, L. Albacker, S. Ramkissoon, C. M. Tilburg, Emily C Hardin, P. Sievers, Felix Sahm, K. Yeo, Tom Rosenberg, Susan N Chi, Karen D Wright, S. Hébert, Sydney Peck, A. Picca, Valérie Larouche, Samuele Renzi, T. Bale, Amy A. Smith, M. Touat, Nada Jabado, Eric S Fischer, Michael Eck, Lissa Baird, Olaf Witt, Claudia L. Kleinman, Quang-De Nguyen, S. Alexandrescu, D. Jones, Keith L. Ligon, P. Bandopadhayay
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 Background Pancreatic ductal adenocarcinoma is a devastating disease lacking targeted therapy. Basal-like pancreatic cancer is a tumor subtype with a worse clinical outcome than other subtypes. These cancers are characterized by the expression of basal keratins and mesenchymal markers. However, the genes that drive the basal-like phenotype are poorly understood. Ras homolog family member V (RhoV) is one of the Rho GTPases, and it has been demonstrated to drive the progression of lung and breast cancers. The current study aims to reveal the role of RhoV in basal-like pancreatic cancer. Design RhoV, p63, and CK5/6 were stained in sequential sections by immunohistochemistry. Expression of RhoV in pancreatic cancer and cell lines was reanalyzed based on the data from ProteinAtlas and Broad Insitute DepMap Portal. Basal cell line, BxPC3, and non-basal cell line, Panc1 were selected for functional evaluation based on their differentiated RhoV expression. RhoV gene was overexpressed and knocked down using the CRISPR/gRNA-based Lentiviral system, followed by sphere formation, colony formation, migration, and invasion assays. Results RhoV, p63, and CK5/6 were expressed in the same histologic location of basal pancreatic cancer. The gene expression level of RhoV was co-expressed with p63 (a classic basal marker) in four pancreatic cancer cell lines (broadinstitute.org). Knockout of RhoV by three runs of gRNAs in BxPC3 cells all significantly attenuated tumor sphere formation with less colony formation as compared to mock knockout. Overexpression of RhoV in non-basal-like cells, Panc 1, significantly increased the size of sphere formation, numbers of colony formation, and p38 activation compared to EV control. Cell migration and invasion assays showed the same RhoV-dependent migration and invasion in corresponding cells. Panc1 cells with RhoV knockout significantly generated smaller tumor xenografts in immunodeficient mice and RhoV overexpression led to larger tumor sizes. Conclusion Finding the driver of the basal-like phenotype helps to reveal the pathophysiology of the basal-like subtype of pancreatic cancer and facilitates the identification of new targets for therapeutic purposes. Our preliminary results indicated that RhoV, a small G protein from the Rho family of GTPases, played essential roles in tumor proliferation and progression, by activating the p38 signaling pathway. Further study will be focused on revealing Rhov's downstream pathways and exploring the role of RhoV in apoptosis/cell cycle, cell metabolism, autophagy, etc. in basel-like pancreatic cancer.
 Citation Format: Shang Wu, Jia Feng, Selina Gao, Ting Wang, Zebang Li, Yifei Liu, Chiung-Kuei Huang, Shaolei Lu. The role of RhoV in basal-like pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffc91f4069ed1467bf0a9573f938915fbe029125" target='_blank'>
              Abstract A077: The role of RhoV in basal-like pancreatic cancer
              </a>
            </td>
          <td>
            Shang Wu, Jia Feng, Selina Gao, Ting Wang, Z. Li, Yifei Liu, Chiung-Kuei Huang, Shaolei Lu
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc4ff74f564f7b2a57419b04c8a2f80af41275c3" target='_blank'>
              Multiregional transcriptomic profiling provides improved prognostic insight in localized non-small cell lung cancer
              </a>
            </td>
          <td>
            Chenyang Li, Thinh T. Nguyen, Jian-Rong Li, Xingzhi Song, J. Fujimoto, L. Little, Curtis Gumb, C. Chow, Ignacio I. Wistuba, Andrew P Futreal, Jianhua Zhang, Shawna M. Hubert, J. Heymach, Jia Wu, Christopher I Amos, Jianjun Zhang, Chao Cheng
          </td>
          <td>2024-10-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Granulomas are a key pathological feature of tuberculosis (TB), characterized by cell heterogeneity, spatial composition, and cellular interactions, which play crucial roles in granuloma progression and host prognosis. This study aims to analyze the transcriptome profiles of cell populations based on their spatial location and to understand the core transcriptome characteristics of granuloma formation and development. Methods In this study, we collected four clinical biopsy samples including Mycobacterium tuberculosis (Mtb) infected lung (MTB-L) and omentum tissues (MTB-O), as well as two lung and omentum biopsies from non-TB patients. The tissues were analyzed by spatial transcriptomics to create a spatial atlas. Utilizing cell enrichment scores and intercellular communication analysis, we investigated the transcriptome signatures of cell populations in various spatial regions and identified genes that may play a decisive role in the formation of pulmonary and omental tuberculosis granulomas. To validate our major findings, an in vitro TB model based on organoid-macrophage co-culture was established. Results Spatial transcriptomics mapped the cell composition and spatial distribution characteristics of tuberculosis granulomas in lung and omental tissues infected with Mtb. The characteristics and evolutionary relationships of major cell populations in granulomas reveal a shift in the immune microenvironment: from a predominance of B cells and fibroblasts in pulmonary granulomas to a predominance of myeloid cells and fibroblasts in omental granulomas. Furthermore, our data identified key differentially expressed genes across cell clusters and regions, showing that upregulation of collagen genes is a common feature of granulomas. Using an organoid-macrophage co-culture model, we demonstrated the notable efficacy of Thrombospondin-1 (THBS1) in reducing protein expression levels related to extracellular matrix remodeling. Conclusion These results provide insights into the pathogenesis and development of tuberculosis, enhancing our understanding of the composition and interactions of tuberculosis granuloma cells from a spatial perspective, and pave the way for novel adjuvant treatments for tuberculosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/466e12ea36c10b55f6829e91a348fd47392619ea" target='_blank'>
              Spatial transcriptomic sequencing reveals immune microenvironment features of Mycobacterium tuberculosis granulomas in lung and omentum
              </a>
            </td>
          <td>
            Xiaochen Qiu, Pengfei Zhong, Liang Yue, Chaofan Li, Zhimin Yun, Guangqian Si, Mengfan Li, Zhi Chen, Yingxia Tan, Pengtao Bao
          </td>
          <td>2024-09-23</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa2963e25bac9e3ab08f5a5b7df7ab88d90b9cc" target='_blank'>
              Multiomic single cell sequencing identifies stemlike nature of mixed phenotype acute leukemia
              </a>
            </td>
          <td>
            C. Peretz, Vanessa E. Kennedy, Anushka Walia, C. Delley, Andrew Koh, Elaine Tran, Iain C Clark, Corey E. Hayford, Chris D’Amato, Yi Xue, Kristina Fontanez, Aaron A. May-Zhang, Trinity Smithers, Y. Agam, Qian Wang, Haixin Dai, Ritu Roy, Aaron C Logan, Alexander E Perl, A. Abate, Adam Olshen, Catherine C Smith
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Data from cell viability assays, which measure cumulative division and death events in a population and reflect substantial cellular heterogeneity, are widely available. However, interpreting such data with mechanistic computational models is hindered because direct model/data comparison is often muddled. We developed an algorithm that tracks simulated division and death events in mechanistically detailed single-cell lineages to enable such a model/data comparison and suggest causes of cell-cell drug response variability. Using our previously developed model of mammalian single-cell proliferation and death signaling, we simulated drug dose response experiments for four targeted anti-cancer drugs (alpelisib, neratinib, trametinib and palbociclib) and compared them to experimental data. Simulations are consistent with data for strong growth inhibition by trametinib (MEK inhibitor) and overall lack of efficacy for alpelisib (PI-3K inhibitor), but are inconsistent with data for palbociclib (CDK4/6 inhibitor) and neratinib (EGFR inhibitor). Model/data inconsistencies suggest (i) the importance of CDK4/6 for driving the cell cycle may be overestimated, and (ii) that the cellular balance between basal (tonic) and ligand-induced signaling is a critical determinant of receptor inhibitor response. Simulations show subpopulations of rapidly and slowly dividing cells in both control and drug-treated conditions. Variations in mother cells prior to drug treatment all impinging on ERK pathway activity are associated with the rapidly dividing phenotype and trametinib resistance. This work lays a foundation for the application of mechanistic modeling to large-scale cell viability assay datasets and better understanding determinants of cellular heterogeneity in drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10d80f29cb012f285134e2b29378c29a5d6024bc" target='_blank'>
              Mechanistic modeling of cell viability assays with in silico lineage tracing
              </a>
            </td>
          <td>
            Arnab Mutsuddy, Jonah R. Huggins, Aurore Amrit, Cemal Erdem, Jon Calhoun, Marc R. Birtwistle
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Immunotherapies, such as immune checkpoint blockade, demonstrate the potential power of CD8 T cells to eliminate cancer cells, however these strategies only work in a subset of patients and tumor types. To design predictably effective immunotherapies, we must elucidate the mechanisms controlling tumor-specific T cell dysfunction and therapeutic reprogramming. TOX is a key transcription factor driving T cell differentiation in progressing tumors and required for T cell persistence. T cell receptor (TCR) signal strength (determined by the affinity of TCR for peptide-bound major histocompatibility complex (pMHC)) is known to regulate T cell differentiation during infections. However, how TCR signal strength impacts TOX expression and differentiation in tumors is less clear. Here, we developed a novel autochthonous mouse cancer model in which we can vary TCR signal strength to span the range observed for human tumor-infiltrating T cells and track tumor-specific CD8 T cells over several months from tumor initiation until end-stage cancer progression: we cloned a Sleeping Beauty transposon/transposase-based vector (SB-Cre) encoding Cre-recombinase and a fluorescent reporter protein fused to a tumor model antigen (OVA) under the control of the liver-specific albumin promoter. Through single amino acid substitutions of the native OVA sequence, we generated altered peptide ligands that are recognized by OVA-specific CD8 T cells with varying TCR signal strength. SB-vectors were delivered via hydrodynamic tail vein injection into mice with inducible activating oncogenes and silencing of tumor suppressor genes, a highly-efficient method to induce liver tumors. Over the course of tumor development we phenotypically and functionally characterized tumor-specific CD8 T cells in tumor draining lymph nodes (tdLN) and progressing liver tumors. Our data reveal how TCR signal strength determines TOX-driven differentiation and dysfunction of tumor-specific T cells starting at the earliest stages of tumorigenesis in tdLN. Ongoing efforts aim to investigate the functional impact of TCR signal strength on T cell differentiation and anti-tumor responses, including responsiveness to immunotherapeutic interventions.
 Citation Format: Sandra Carson, Andrea Schietinger. TCR signal strength drives tumor-specific CD8 T cell differentiation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr B031.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99a1dfc36748f041edc9c8f912c619db3af5d4ef" target='_blank'>
              Abstract B031: TCR signal strength drives tumor-specific CD8 T cell differentiation
              </a>
            </td>
          <td>
            Sandra Carson, A. Schietinger
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7199657c55f0905635cd26958d5762b121b0a3f2" target='_blank'>
              Integrated multi-omics assessment of lineage plasticity in a prostate cancer patient with brain and dural metastases
              </a>
            </td>
          <td>
            Megan L Ludwig, David Moline, Alec Horrmann, Ella Boytim, Gabrianne Larson, Ali T Arafa, Masooma Sayeda, John R Lozada, Hannah E Bergom, Abderrahman Day, Sandhyarani Dasaraju, S. Dehm, Paari J. Murugan, Justin H. Hwang, J. Drake, E. S. Antonarakis
          </td>
          <td>2024-09-30</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) originates from precursor lesions, the most common of which are known as pancreatic intraepithelial neoplasia (PanIN). PanIN formation and progression is accompanied by accumulation of a precursor microenvironment (PME) composed of desmoplastic stroma that plays an important role in promoting tumor growth and immune evasion. The crosstalk between tumor cells and stromal cells is an area of active investigation. While PanIN and the PME have been extensively studied in mouse models, their nature in humans have remained elusive as there is no clinical indication to sample healthy pancreas. Through a unique collaboration with the Gift of Life Organ and Tissue donation program, we performed transcriptomic analysis on normal pancreata declined for transplant. We noted that epithelial cells in the PanIN and cancer microenvironment were transcriptomically similar; however, their surrounding stromal fibroblasts were markedly distinct. Given that most precursor lesions remain quiescent while some progress to tumor, we sought out to elucidate how the transcriptional programs in epithelial cells influence fibroblast cell fate. To do so, we developed a 3D ex-vivo organoid-fibroblast coculture system using primary organoid and fibroblast cultures successfully derived from Gift of Life pancreata or PDAC patients undergoing surgical resection. Our coculture system retains histological features of heterogeneity observed in vivo. Using single cell RNA-sequencing analysis of matched fibroblast-organoid cocultures we identified factors that were highly expressed in fibroblasts in tumor cocultures, but not in normal cocultures. We are currently interrogating candidate ligand/receptor interactions in our cocultures to determine the mediators of this epithelial-mediated fibroblast polarization. This model will allow us to gain an understanding about how components of the microenvironment restrain or promote malignant transformation and serve as a great tool to study therapies targeting tumor and stromal cells in a patient specific manner.
 Citation Format: Padma Kadiyala, Ahmed Elhossiny, Jianhua Liu, Alexander Bray, Katelyn Donahue, Arvind Rao, Jiaqi Shi, Yaqing Zhang, Filip Bednar, Timothy Frankel, Eileen Carpenter, Marina Pasca Di Magliano. Investigating Fibroblast-Epithelial crosstalk in pre-malignant and cancerous microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A057.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9841941d0df90af0a793863b27fe1e8b8a9497" target='_blank'>
              Abstract A057: Investigating Fibroblast-Epithelial crosstalk in pre-malignant and cancerous microenvironment
              </a>
            </td>
          <td>
            Padma Kadiyala, Ahmed M. Elhossiny, Jianhua Liu, Alexander W Bray, Katelyn L. Donahue, Arvind Rao, Jiaqi Shi, Yaqing Zhang, Filip Bednar, Timothy L. Frankel, Eileen S Carpenter, M. P. di Magliano
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Most patients with prostate adenocarcinoma develop resistance to therapies targeting the androgen receptor (AR). Consequently, a portion of these patients develop AR-independent neuroendocrine prostate cancer (NEPC), a rapidly progressing cancer with limited therapies and poor survival outcomes. Current research to understand the progression to NEPC suggests a model of lineage plasticity whereby AR-dependent luminal-like tumors progress towards an AR-independent NEPC state. Genetic analysis of human NEPC identified frequent loss of RB1 and TP53, and the loss of both genes in experimental models mediates the transition to a neuroendocrine lineage. Transcriptomics studies have shown that lineage transcription factors ASCL1 and NEUROD1 are present in NEPC. In this study, we modeled the progression of prostate adenocarcinoma to NEPC by establishing prostate organoids and subsequently generating subcutaneous allograft tumors from genetically-engineered mouse models harboring Cre-induced loss of Rb1 and Trp53 with Myc overexpression (RPM). These tumors were heterogeneous and displayed adenocarcinoma, squamous, and neuroendocrine features. ASCL1 and NEUROD1 were expressed within neuroendocrine-defined regions, with ASCL1 being predominant. Genetic loss of Ascl1 in this model did not decrease tumor incidence, growth, or metastasis; however, there was a notable decrease in neuroendocrine identity and an increase in basal-like identity. This study provides an in vivo model to study progression to NEPC and establishes the requirement for ASCL1 in driving neuroendocrine differentiation in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b06924a4ca0fcd63effebd03ee725e6bf5e8759" target='_blank'>
              Neuroendocrine Differentiation in Prostate Cancer Requires ASCL1.
              </a>
            </td>
          <td>
            Kathia E Rodarte, Shaked Nir Heyman, Lei Guo, Lydia Flores, T. Savage, Juan Villarreal, Su Deng, Lin Xu, Rajal B Shah, T. Oliver, Jane E Johnson
          </td>
          <td>2024-09-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Progressive metastasis is the primary cause of cancer-related deaths. It has been recognized that many cancers are characterized by long periods of stability followed by subsequent progression. Genes termed metastasis progression suppressors (MPS) are functional gatekeepers of this process, and their loss leads to late-stage progression. Previously, we identified regulator of calcineurin 1, isoform 4 (RCAN1.4) as a functional MPS for several cancers, including thyroid cancer, a tumor type prone to metastatic dormancy. RCAN1.4 knockdown increases expression of the cancer-promoting transcription factor NFE2-like bZIP transcription factor (NFE2L3), and through this mechanism increases cancer cell proliferation and invasion in in vitro and in vivo and promotes metastatic potential to lungs in tail vein models. However, the mechanisms by which RCAN 1.4 regulates specific metastatic steps is incompletely characterized. Studies of the metastatic cascade are limited in mouse systems due to high cost and long duration. Here, we have shown the creation of a thyroid-to-lung metastasis-on-a-chip (MOC) model to address these limitations, allowing invasion analysis and quantification on a single cell level. We then deployed the platform to investigate RCAN1.4 knockdown in fluorescently tagged hTh74 and FTC236 thyroid cancer cell lines. Cells were circulated through microfluidic channels, running parallel to lung hydrogel constructs allowing tumor cell-lung tissue interactions. Similar to studies in mouse models, RCAN1.4 knockdown increased NFE2L3 expression, globally increased invasion distance into lung constructs and had cell line and clonally dependent variations on bulk metastatic burden. In line with previous in vivo observations, RCAN1.4 knockdown had a greater impact on hTh74 metastatic propensity than FTC236. In summary, we have developed and validated a novel MOC system evaluate and quantify RCAN1.4-regulated thyroid cancer cell lung adherence and invasion. This system creates opportunities for more detailed and rapid mechanistic studies the metastatic cascade and creates opportunities for translational assay development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef429960eab2165380cb16cd51185548a253029" target='_blank'>
              RCAN1.4 regulates tumor cell engraftment and invasion in a thyroid cancer to lung metastasis-on-a-chip microphysiological system.
              </a>
            </td>
          <td>
            Kylie G. Nairon, Akanksha Nigam, Tilak Khanal, Marco A. Rodriguez, Neel Rajan, Sydney R Anderson, Matthew D Ringel, Aleksander Skardal
          </td>
          <td>2024-10-03</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Even though epigenetic factors contribute to oncogenesis, most human cancer models still assume that disease originates from driver DNA mutations. Thus, it is still unclear if non-genetic mechanisms are sufficient to trigger malignant transformation. Special AT-rich binding protein 2 (SATB2) is a chromatin organizer that brings distal DNA elements into close proximity, thus remodeling chromatin structures to reprogram cell-specific and/or developmentally-sensitive gene networks. Here, we discover that SATB2 generates a co-expressed circ3915 RNA that is translated into a peptide and co-locates with SATB2 in the cell nucleus. Ectopic SATB2 or circ3915 over- expression rearranges global chromatin accessibility, generates KRAS- and NFE2L2-like oncogenic gene expression patterns, and transforms lung epithelial cells independent of driver mutations. Thus, oncogenic pathways can be activated in mammalian cells without pre-disposing mutations in oncogenes or epigenetic regulators. Graphical abstract:">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2809e3fd65e4f1849899e9487a4c550596b35a99" target='_blank'>
              SATB2 and circ3915 RNA chromatin dysregulation drive KRAS-like oncogenic transformation
              </a>
            </td>
          <td>
            Rebekah Eleazer, Smitha George, Luke Shoemaker, Wesley N. Saintilnord, Kin H. Lau, Darrell Chandler, Yvonne Fondufe-Mittendorf
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mechanotransduction is a collection of pathways in which the cells reprogram themselves by sensing mechanical stimuli. Cells use biological cues to interpret the physiological stresses and respond to changing conditions by modifying the cellular and ECM architecture. This feedback loop regulates a variety of cellular processes, including migration, growth, differentiation, and death, which is essential for the network stability to work together in a coordinated manner. The effect of stress on cancer progression and the role of mechanics as a critical inducer in determining the cancer cell fate has been studied. This review discusses the progression of cancer cells to epithelial to mesenchymal transitions. It examines tumor microenvironment models, such as spheroids, bio-printing, and microfluidics, and how they recapitulate the tumor microenvironment. These offer certain benefits and help replicate the fundamental behavior in vivo conditions. We further discuss mechanosensing, the associated signaling molecules, and how it modulates the cancer drug resistance and transduction pathways that implicate cancer treatment. The difficulties with the existing methods and the prospects for additional study that may be applied in this area are discussed, and how they allow for new therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c5da4c6457c9e0269895f58a987813d2f60e71" target='_blank'>
              Mechanotransduction alterations in tissue-engineered tumor models for new drug interventions
              </a>
            </td>
          <td>
            Nipun Jain, Yusuf Olatunji Waidi, Souvik Debnath, V. Vannaladsaysy, Sudipto Datta
          </td>
          <td>2024-09-03</td>
          <td>Pharmacy &amp; Pharmacology International Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Ovarian cancer is high recurrence and mortality malignant tumor. The most common ovarian cancer was High‐Grade Serous Ovarian Cancer. However, High‐Grade Serous Ovarian Cancer organoid is rare, which organoid with patient immune microenvironment and blood vessels even absence. Here, we report a novel High‐Grade Serous Ovarian Cancer organoid system derived from patient ovarian cancer samples. These organoids recapitulate High‐Grade Serous Ovarian Cancer organoids' histological and molecular heterogeneity while preserving the critical immune microenvironment and blood vessels, as evidenced by the presence of CD34 + endothelial cells. Whole exome sequencing identifies key mutations (CSMD3, TP53, GABRA6). Organoids show promise in testing cisplatin sensitivity for patients resistant to carboplatin and paclitaxel, with notable responses in cancer proteoglycans and p53 (TP53) signaling, like ACTG/ACTB1/AKT2 genes and BBC3/MDM2/PERP. Integration of immune microenvironment and blood vessels enhances potential for novel therapies like immunotherapies and angiogenesis inhibitors. Our work may provide a new detection system and theoretical basis for ovarian cancer research and individual therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dedad20ef4fac488d3d16a1328a435bc572aae5" target='_blank'>
              Patient‐derived ovarian cancer organoid carries immune microenvironment and blood vessel keeping high response to cisplatin
              </a>
            </td>
          <td>
            Yuqing Zhao, Chen Wang, Wei Deng, Lanyang Li, Jiping Liu, Yanghua Shi, Xiang Tao, Jian Zhang, Qi Cao, Chunhui Cai, Xinxin Han
          </td>
          <td>2024-08-28</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b437992e1cf50ec83b6f6a40806009d50e99add" target='_blank'>
              The genomic and transcriptomic landscape of metastastic urothelial cancer
              </a>
            </td>
          <td>
            Yohann Loriot, M. Kamal, L. Syx, R. Nicolle, C. Dupain, Naoual Mensourri, Igor Duquesne, P. Lavaud, C. Nicotra, Maud Ngocamus, Ludovic Lacroix, L. Tselikas, Gilles Crehange, L. Friboulet, Zahra Castel-Ajgal, Y. Neuzillet, E. Borcoman, Philippe Beuzeboc, G. Marret, Tom Gutman, Jennifer Wong, François Radvanyi, S. Dureau, J. Scoazec, N. Servant, Yves Allory, Benjamin Besse, F. André, Christophe Le Tourneau, Christophe Massard, I. Bieche
          </td>
          <td>2024-10-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7254d2e767a2a42779b5a71fd655bef014a5b881" target='_blank'>
              A deep learning method that identifies cellular heterogeneity using nanoscale nuclear features
              </a>
            </td>
          <td>
            Davide Carnevali, Limei Zhong, Esther González-Almela, Carlotta Viana, Mikhail Rotkevich, Aiping Wang, Daniel Franco-Barranco, Aitor González-Marfil, M. V. Neguembor, Álvaro Castells-García, I. Arganda-Carreras, M. Cosma
          </td>
          <td>2024-08-27</td>
          <td>Nature Machine Intelligence</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/723eb2d41e78a0b627e8df5f52344ae5f48397f9" target='_blank'>
              Neoantigen immunogenicity landscapes and evolution of tumor ecosystems during immunotherapy with nivolumab.
              </a>
            </td>
          <td>
            Tyler J Alban, N. Riaz, Prerana B. Parthasarathy, Vladimir Makarov, S. Kendall, Seong-Keun Yoo, Rachna Shah, Nils Weinhold, Raghvendra Srivastava, Xiaoxiao Ma, Chirag Krishna, J. Y. Mok, Wim J E van Esch, Edward Garon, Wallace Akerley, B. Creelan, Nivedita Aanur, D. Chowell, W. Geese, N. Rizvi, Timothy A Chan
          </td>
          <td>2024-09-30</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f8959f0cd1763d135183d030a6b223aefb572b8" target='_blank'>
              Therapeutic targeting of differentiation-state dependent metabolic vulnerabilities in diffuse midline glioma
              </a>
            </td>
          <td>
            Nneka E. Mbah, Amy L. Myers, Peter Sajjakulnukit, Chan Chung, Joyce K Thompson, Hanna S Hong, Heather Giza, Derek Dang, Z. Nwosu, Mengrou Shan, Stefan R. Sweha, Daniella D Maydan, Brandon Chen, Li Zhang, Brian Magnuson, Zirui Zhu, Megan D Radyk, Brooke Lavoie, Viveka N Yadav, Imhoi Koo, Andrew D. Patterson, Daniel R. Wahl, Luigi Franchi, S. Agnihotri, C. Koschmann, S. Venneti, C. Lyssiotis
          </td>
          <td>2024-10-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="The expression of PD-L1 on tumor cells (TCs) is used as an immunotherapy biomarker in lung cancer, but heterogeneous intratumoral expression is often observed. Using a Digital Spatial Profiling, we performed proteomic and whole-transcriptomic analyses of TCs and immune cells (ICs) in spatially matched areas based on tumor PD-L1 expression and the status of the immune microenvironment. Our findings were validated using immunohistochemistry, The Cancer Genome Atlas, and immunotherapy cohorts. ICs in areas with high PD-L1 expression on TCs showed more features indicative of immunosuppression and exhaustion than ICs in areas with low PD-L1 expression on TCs. TCs highly expressing PD-L1 within immune-inflamed (IF) areas show up-regulation of pro-inflammatory processes, whereas TCs highly expressing PD-L1 within immune-deficient (ID) areas show up-regulation of various metabolic processes. Using differentially expressed genes of TCs between the IF and ID areas, we identified a novel prognostic gene signature for lung cancer. In addition, a high ratio of CD8+ cells to M2 macrophages was found to predict favorable outcomes in patients with PD-L1-expressing lung cancer after immune checkpoint inhibitor therapy. This study demonstrates that TCs and ICs have distinct spatial features within the tumor microenvironment that are related to tumor PD-L1 expression and IC infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc0a9207938d739ad9fb6635845dbbb9be7ec54" target='_blank'>
              Spatially resolved whole-transcriptomic and proteomic profiling of lung cancer and its immune-microenvironment according to PD-L1 expression.
              </a>
            </td>
          <td>
            Yoon Kyung Jeon, Jaemoon Koh, Dongjoo Lee, Sehui Kim, Seung Geun Song, B. Han, Hyein Jeong, Young A Kim, B. Keam, Se-Hoon Lee, K. Kim, Doo Hyun Chung
          </td>
          <td>2024-09-05</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="
 Patient-derived organoids (PDOs) have emerged as a powerful tool for modeling human cancers due to their ability to closely mirror in vivo architecture and preserve the genetic landscape of tumors. They offer a crucial platform for studying disease progression and treatment responses. In this study, we present a comprehensive characterization of a unique collection of PDO cultures derived from pancreatic ductal adenocarcinoma (PDAC), a particularly aggressive malignancy often diagnosed in advanced stages with limited treatment options and poor prognosis. Our investigation explores morphological and genetic profiling of 10 PDAC PDOs, specifically identifying distinct drug-response patterns. The morphological characteristics of these PDO cultures were investigated by microscopy, while the mutational status was acquired by panel sequencing. Subsequently, PDOs were treated with a panel of 10 inhibitors, either individually or in combination. The response to these treatments was assessed through live-cell imaging (Incucyte®) to capture and quantify changes in organoid morphology and endpoint measurements of cellular viability using a CellTiter-Glo® assay. Our detailed in vitro drug sensitivity testing revealed a remarkable diversity in the responses of the PDAC PDOs to different chemotherapeutic agents, including those currently under clinical trials. Of notable significance is the depth of understanding we have achieved, allowing us to establish potential correlations between PDO morphology, genetic mutations, and drug reactions. This finding provides a key insight into the complex interplay of these factors and their impact on treatment outcomes, offering valuable insights into understanding individual tumor subtypes' sensitivity or resistance to treatments. The PDOs exhibited a spectrum of cancer-associated mutations, including KRAS, TP53, SMAD4, CDKN2A, and ARID1A, reflecting the heterogeneity observed in PDAC patients. Morphologically, the organoids displayed various growth patterns, ranging from cystic and regular structures to filled or unfilled, granular, dark, and light membrane structures. Drug treatments underscored the diverse and unique drug-response profiles within the PDOs, with some demonstrating sensitivity to multiple treatments. In contrast, others exhibited resistance, highlighting the intricate nature of drug responses within this model. Overall, our findings emphasize the ability of PDAC organoids to faithfully capture the complexity and heterogeneity of the disease, making them promising platforms for guiding personalized oncology approaches. The diverse drug response profiles observed in PDOs advocate leveraging this model to develop precision medicine strategies tailored to patients' characteristics, ultimately paving the way for more effective and personalized treatment regimens in the fight against PDAC.
 Citation Format: Constanza Tapia Contreras, Günter Schneider, Denise Müller, Kimia Mirzakhani, Karly Conrads, Silke Kaulfuß, Tim Beißbarth, Gabriela Salinas, Nils Beyer, Sophia Reinländer, Ulrix Sax, Elisabeth Heßmann, Volker Ellenrieder, Philipp Ströbel, Michael Ghadimi. Patient-Derived Organoid Cultures for Personalized Therapies and Targeted Drug Screening Applications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A023.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a23581700d7eba1d7151f1cd1e85577199cc66e" target='_blank'>
              Abstract A023: Patient-Derived Organoid Cultures for Personalized Therapies and Targeted Drug Screening Applications
              </a>
            </td>
          <td>
            Constanza Tapia Contreras, Günter Schneider, Denise Müller, Kimia Mirzakhani, Karly Conrads, S. Kaulfuß, Tim Beißbarth, Gabriela Salinas, Nils Beyer, Sophia Reinländer, Ulrix Sax, E. Hessmann, V. Ellenrieder, Philipp Ströbel, Michael Ghadimi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="During cellular differentiation, enhancers transform overlapping gradients of transcription factors (TFs) to highly specific gene expression patterns. However, the vast complexity of regulatory DNA impedes the identification of the underlying cis-regulatory rules. Here, we have characterized 62,126 fully synthetic DNA sequences to bottom-up dissect design principles of cell-state specific enhancers in the context of the differentiation of blood stem cells to seven myeloid lineages. Focusing on binding sites for 38 TFs and their pairwise interactions, we found that identical sites displayed both repressive and activating function, as a consequence of cellular context, site combinatorics, or simply predicted occupancy of a TF on an enhancer. Surprisingly, we found that combinations of activating sites frequently neutralized each other or even gained repressive function. These negative synergies convert quantitative imbalances in transcription factor expression into binary downstream activity patterns, a principle that can be exploited to build differentiation-state specific enhancers from scratch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab0f40a79f1a6210ac7f2c422c1cc54bfc294a6c" target='_blank'>
              Synthetic enhancers reveal design principles of cell state specific regulatory elements in hematopoiesis
              </a>
            </td>
          <td>
            R. Frömel, Julia Rühle, Aina Bernal Martinez, Chelsea Szu-Tu, Felix Pacheco Pastor, Rosa Martinez Corral, Lars Velten
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Signaling pathways induce stereotyped transcriptional changes as stem cells progress into mature cell types during embryogenesis. Signaling perturbations are necessary to discover which genes are responsive or insensitive to pathway activity. However, gene regulation is additionally dependent on cell state-specific factors like chromatin modifications or transcription factor binding. Thus, transcriptional profiles need to be assayed in single cells to identify potentially multiple, distinct perturbation responses among heterogeneous cell states in an embryo. In perturbation studies, comparing heterogeneous transcriptional states among experimental conditions often requires samples to be collected over multiple independent experiments. Datasets produced in such complex experimental designs can be confounded by batch effects. We present Design-Aware Integration of Single Cell ExpEriments (DAISEE), a new algorithm that models perturbation responses in single-cell datasets with a complex experimental design. We demonstrate that DAISEE improves upon a previously available integrative non-negative matrix factorization framework, more efficiently separating perturbation responses from confounding variation. We use DAISEE to integrate newly collected single-cell RNA-sequencing datasets from 5-hour old zebrafish embryos expressing optimized photoswitchable MEK (psMEK), which globally activates the extracellular signal-regulated kinase (ERK), a signaling molecule involved in many cell specification events. psMEK drives some cells that are normally not exposed to ERK signals towards other wild type states and induces novel states expressing a mixture of transcriptional programs, including precociously activated endothelial genes. ERK signaling is therefore capable of introducing profoundly new gene expression states in developing embryos. Significance Statement Signaling perturbations produce heterogeneous transcriptional responses that must be measured at the single-cell level. Data integration techniques allow us to model these responses which, however, can be confounded by batch effects. We present a computational tool (DAISEE) for extracting the common and perturbation-specific features of single-cell datasets representing multiple experimental conditions while achieving efficient batch effect correction. DAISEE outperforms its predecessor and will enable accurate analysis of a broad range of single-cell datasets. DAISEE applied to new single-cell RNA sequencing data from zebrafish embryos shows that gain-of-function signaling perturbations can induce novel states. Our analysis suggests that a wild type endothelial cell-specification program can be activated in abnormal developmental contexts when the extracellular signal-regulated kinase (ERK) pathway is deregulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9843baa0dda4952c851210ee811a954b61353c56" target='_blank'>
              Disrupted developmental signaling induces novel transcriptional states
              </a>
            </td>
          <td>
            Aleena Patel, Vanessa Gonzalez, Triveni Menon, Stanislav Y. Shvartsman, R. Burdine, M. Avdeeva
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/869c734206e668652b18e8e50d0afc69d2273b39" target='_blank'>
              Intratumoral microbiota of pancreatic ductal adenocarcinoma impact patient prognosis by influencing tumor microenvironment
              </a>
            </td>
          <td>
            Jingze Leng, Hengyi Xu, Xiaoyu Liu, Yufan Yang, Chun Ning, Lejia Sun, Jiangming Qu, Xindi Ke, Xun Lan
          </td>
          <td>2024-09-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/690e8f5811cd038f3703a746b9c0925f1ed139ae" target='_blank'>
              Comprehensive pan-cancer analysis reveals EPHB2 is a novel predictive biomarker for prognosis and immunotherapy response
              </a>
            </td>
          <td>
            Shengshan Xu, You-You Zheng, Min Ye, Tao Shen, Dongxi Zhang, Zumei Li, Zhu-ming Lu
          </td>
          <td>2024-08-28</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mitotic Arrest Deficient 1 (gene name MAD1L1), an essential component of the mitotic spindle assembly checkpoint, is frequently overexpressed in colon cancer, which correlates with poor disease-free survival. MAD1 upregulation induces two phenotypes associated with tumor promotion in tissue culture cells–low rates of chromosomal instability (CIN) and destabilization of the tumor suppressor p53. Using CRISPR/Cas9 gene editing, we generated a novel mouse model by inserting a doxycycline (dox)-inducible promoter and HA tag into the endogenous mouse Mad1l1 gene, enabling inducible expression of HA-MAD1 following exposure to dox in the presence of the reverse tet transactivator (rtTA). A modest 2-fold overexpression of MAD1 in murine colon resulted in decreased p53 expression and increased mitotic defects consistent with CIN. After exposure to the colon-specific inflammatory agent dextran sulfate sodium (DSS), 31% of mice developed colon lesions, including a mucinous adenocarcinoma, while none formed in control animals. Lesion incidence was particularly high in male mice, 57% of which developed at least one hyperplastic polyp, adenoma or adenocarcinoma in the colon. Notably, mice expressing HA-MAD1 also developed lesions in tissues in which DSS is not expected to induce inflammation. These findings demonstrate that MAD1 upregulation is sufficient to promote colon tumorigenesis in the context of inflammation in immune-competent mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b8f033c5cf9d7fd01adbc62c82ee49aacc06c1c" target='_blank'>
              MAD1 upregulation sensitizes to inflammation-mediated tumor formation
              </a>
            </td>
          <td>
            Sarah E Copeland, Santina M. Snow, Jun Wan, Kristina A Matkowskyj, R. Halberg, B. Weaver
          </td>
          <td>2024-10-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 The inherent adaptive capacity of GBM to metabolic stress represents a key clinical challenge, and the underlying mechanisms remain ill-defined. Our lab focuses on the emerging role of lipid metabolism in cancer cell stress adaptation. We have previously found increased scavenging of lipid particles, including extracellular vesicles (EVs) and lipoproteins, during acute stress, further resulting in a lipid droplet (LD) storage phenotype. Here, we propose a new mechanistic link between chondrotin sulfate proteoglycan (CSPG) remodelling, extracellular lipid scavenging, and LD formation that together serve to fine tune intracellular lipid metabolism to escape ferroptosis.



 We have comprehensively profiled GBM tissues and cells during different acidic stress adaptation conditions, including freshly resected GBM specimens, primary GBM cultures, and established GBM cell-lines. Methods/analyses include: Multiomics (RNA, proteome, glycome), laser capture microdissection for spatial profiling, snRNA-seq, immunophenotyping by FACS and imaging, and in vitro treatment studies (in 2D and 3D) with live imaging of cytotoxicity and ferroptosis. Mouse GBM models were employed for in vivo treatment studies.



 We provide first evidence of a functional role of cell-surface CSPGs during GBM adaptation to metabolic acidosis. Substantial CSPG induction coincides with LD formation in a process dependent on extracellular lipids and acidification. Specifically, CSPG encapsulation of stressed GBM cells acts as a shield against excessive lipid particle scavenging while LDs consitute an intracellular sink for lipid storage. Based on these findings, we introduce combined targeting of CSPGs and LDs as a strategy to challenge the stressed tumor niche. Concomitant disruption of CSPG and LD formation unleashes a metabolic vulnerability in acidic cells driven by uncontrolled lipid uptake and peroxidation, mitochondrial disintegration, and ferroptotic death.



 In conclusion, this study identifies a potentially targetable adaptative mechanism in the treatment resistant, acidic niche of GBM. We propose that cancer cell escape from acidic stress involves the induction of CSPGs - providing a shield against the lipid rich tumor microenvironment - and LDs - forming an intracellular lipid sink. Combined targeting of CSPG and LD formation to induce ferroptosis may offer new therapeutic opportunities in GBM and other solid tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94131c305e74656b4d8349f795b86e501414c82d" target='_blank'>
              OS09.5.A UNEXPECTED FUNCTIONAL ROLE OF GLYCOCALYX ENCAPSULATION DURING GBM METABOLIC ADAPTATION OPENS AVENUES FOR NEW TREATMENT CONCEPTS
              </a>
            </td>
          <td>
            A. Bång-Rudenstam, M. Cerezo-Magaña, K. Gonçalves de Oliveira, V. Governa, H. Talbot, L. Kjellen, E. Tykesson, A. Malmström, T. van Kuppevelt, K. Forsberg-Nilsson, J. Bengzon, M. Belting
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background The prognosis of colorectal cancer (CRC) is known to vary across different etiologies. Inflammatory bowel disease (IBD) is often identified as a factor contributing to poorer outcomes. However, the mechanisms that link IBD to a worse CRC prognosis remain to be elucidated. We aim to reveal the complex tumor microenvironment of inflammatory CRC and provide a weak theoretical basis for the treatment of different subtypes of CRC. Methods We conducted a bioinformatics analysis using single-cell RNA sequencing (scRNA-seq) data from 8,494 individual CRC cells derived from azoxymethane (AOM)/dextran sodium sulfate (DSS) and adenomatous polyposis coli (APC) mutant datasets. The expression of implicated genes in both tumor and adjacent normal tissues was examined via immunohistochemistry and immunofluorescence. Results CRC from AOM/DSS treatment contained fewer immune cells relative to APC-mutant CRC. However, a macrophage subcluster enriched for inflammatory factors was more prevalent in AOM/DSS datasets. This subcluster exhibited elevated expression of APOE and BNIP3. Immunofluorescence and immunohistochemistry of patient samples confirmed that the expression of APOE and BNIP3 was higher in adjacent normal tissues compared to tumors. Conclusions Our findings shed light on the heterogeneous microenvironments in IBD and APC-mutant CRC. Furthermore, we identify APOE as a potential biomarker for CRC recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9b46e5e612d7c24c6d0e3171cb3896808db60bd" target='_blank'>
              Heterogeneity of tumor microenvironment cell groups in inflammatory and adenomatous polyposis coli mutant colorectal cancer based on single cell sequencing
              </a>
            </td>
          <td>
            Liyang Liang, Chao Zhang, Jiawang Han, Zhipeng Liu, Jinzhong Liu, Suhang Wu, Hongyu Wang
          </td>
          <td>2024-09-01</td>
          <td>Translational Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Disclosure: D. Zhang: None. M. Bergsneider: None. M.B. Wang: None. W. Kim: None. H. Vinters: None. A.P. Heaney: None. Most pituitary corticotroph adenomas are benign but ∼15% are refractory to conventional surgical, medical and even radiotherapeutic management. This tumor subset invade sellar adjacent structures, and in rare cases metastasize locally within the central nervous system or to neck lymph nodes (LN) or more distant organs such as the liver. To better understand the potential interactions between tumor cells and neighboring tissues during these progressive stages, we used spatial transcriptomics (ST, Visium, 10x Genomics) to profile the genetic composition and native architecture of 4 surgically resected corticotroph tumor paraffin-embedded samples collected from the same patient over several years. Samples 1& 2 invaded the sino-nasal mucosa, sample 3 the parietal dura, sample 4 the clival bone, and sample 5 had metastasized locally to a neck LN. Quality of the 4 samples was confirmed by DV200 score (51 ± 7%). An average of 2,473 tissue spots per tissue sample were obtained after sequencing with a median of 4,893 genes detected per spot. We then used the Seurat analytic pipeline to reconstitute spatial transcriptional distribution that authentically mirrored histologically verifiable structures. These included the main tumor area itself, it’s invading tumor edge, adjacent normal epithelium, dura, and LN tissue. Further deconvolution analysis revealed a distinct cellular composition at the invading tumor edge which was tissue context specific. For instance, a proliferative cell population was noted between the central tumor area and it’s invasive margin, B cell and fibroblast cell enrichment was prominent at sino-nasal mucosal invasive margins, whereas endothelial and pericyte cells were prominent along the tumor/ normal pituitary border. . As expected, the central tumor area expressed genes associated with corticotroph hormone production including POMC, NNAT, GAL and GNAS, genes associated with mitochondrial energy metabolism such as ATP5F1E, NDUFA8 and COX7B as well as transcripts related to lipid kinase activity (GKG, CERKL, PIK3C2G, DGKK). Most intriguingly, tumor cells at the fore-front of the invading edge expressed high level of cyclin-dependent kinases, potentially reflecting higher proliferative rates of these invasive margin cells. Additional studies to validate and define the role of theses invasive margin transcripts are now warranted to translate our profiling findings into potential drug targets. In summary, this is the first use of spatial transcriptomics to delineate the evolving transcriptomic features in a series of surgically resected pituitary corticotroph tumors from the same patient that progressed over time to a locally metastatic corticotroph carcinoma. It provides unparalleled insights into the differential transcriptional topography of corticotroph tumor tissue and most importantly, its invading edge and may identify novel therapeutic targets. Presentation: 6/3/2024">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52629cfca0e770c37dfc12e4ac5b75c2f708ab6e" target='_blank'>
              8133 Spatial Transcriptomic Profiling of Pituitary Corticotroph Carcinoma: Uncovering Architectural Complexity
              </a>
            </td>
          <td>
            Dongyun Zhang, M. Bergsneider, Marilene B. Wang, Won Kim, H. Vinters, A. Heaney
          </td>
          <td>2024-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="BACKGROUND
Despite significant strides in lung cancer immunotherapy, the response rates for Kirsten rat sarcoma viral oncogene homolog (KRAS)-driven lung adenocarcinoma (LUAD) patients remain limited. Fibrinogen-like protein 1 (FGL1) is a newly identified immune checkpoint target, and the study of related resistance mechanisms is crucial for improving the treatment outcomes of LUAD patients. This study aimed to elucidate the potential mechanism by which FGL1 regulates the tumor microenvironment in KRAS-mutated cancer.


METHODS
The expression levels of FGL1 and SET1 histone methyltransferase (SET1A) in lung cancer were assessed using public databases and clinical samples. Lentiviruses were constructed for transduction to overexpress or silence FGL1 in lung cancer cells and mouse models. The effects of FGL1 and Yes-associated protein (Yap) on the immunoreactivity of cytotoxic T cells in tumor tissues were evaluated using immunofluorescence staining and flow cytometry. Chromatin immunoprecipitation and dual luciferase reporter assays were used to study the SET1A-directed transcriptional program.


RESULTS
Upregulation of FGL1 expression in KRAS-mutated cancer was inversely correlated with the infiltration of CD8+ T cells. Mechanistically, KRAS activated extracellular signal-regulated kinase 1/2 (ERK1/2), which subsequently phosphorylated SET1A and increased its stability and nuclear localization. SET1A-mediated methylation of Yap led to Yap sequestration in the nucleus, thereby promoting Yap-induced transcription of FGL1 and immune evasion in KRAS-driven LUAD. Notably, dual blockade of programmed cell death-1 (PD-1) and FGL1 further increased the therapeutic efficacy of anti-PD-1 immunotherapy in LUAD patients.


CONCLUSION
FGL1 could be used as a diagnostic biomarker of KRAS-mutated lung cancer, and targeting the Yap-FGL1 axis could increase the efficacy of anti-PD-1 immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa74574b86205c0fd05108e0df64d1fac6c4ec0e" target='_blank'>
              Yap methylation-induced FGL1 expression suppresses anti-tumor immunity and promotes tumor progression in KRAS-driven lung adenocarcinoma.
              </a>
            </td>
          <td>
            Ji Jiang, Pengfei Ye, Ningning Sun, Weihua Zhu, Mei Yang, Manman Yu, Jingjing Yu, Hui Zhang, Zijie Gao, Ningjie Zhang, Shijie Guo, Yuru Ji, Siqi Li, Cuncun Zhang, Sainan Miao, Mengqi Chai, Wenmin Liu, Yue An, Jian Hong, Wei Wei, Shihao Zhang, Huan Qiu
          </td>
          <td>2024-09-28</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0fe672a884b6c86c84e083c9b2ab93ac24f65f" target='_blank'>
              Multiplex, single-cell CRISPRa screening for cell type specific regulatory elements
              </a>
            </td>
          <td>
            Florence M. Chardon, Troy A. McDiarmid, Nicholas F. Page, R. Daza, Beth K. Martin, Silvia Domcke, Samuel Regalado, Jean-Benoît Lalanne, Diego Calderon, Xiaoyi Li, L. Starita, Stephan J. Sanders, N. Ahituv, J. Shendure
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>49</td>
        </tr>

        <tr id="
 Malignant rhabdoid tumors (MRT) are rare aggressive tumors of infancy characterized by the biallelic inactivation of SMARCB1. Their lineage of origin remains uncertain, but increasing evidence suggest a neural-crest origin. By analyzing the transcriptome of MRT occurring in peripheral and cranial nerves, we observe that their expression profiling was not distinct from any other tumor localization. We therefore investigated whether the development of Smarcb1-deficient tumors emerging from embryonic precursors of peripheral nerves in mice could recapitulate human MRT. we generated conditional Smarcb1 knockout in the Prss56-Cre mouse strain thereby targeting boundary cap cells which are neural crest derived Schwann cell precursor. All Smarcb1Flox/Flox;Prss56Cre+/+ showed ataxia within a median of two months, due to the development of massive intracranial tumor in the ponto-cerebellum angle. Smarcb1Flox/Flox;Prss56Cre+/- mainly developed paraplegia related to nerve root tumors, within a median delay of 7 months. Interestingly, transcriptome profiling confirmed the good correlation between mouse peripheral nerve tumors and human MRT; intracranial tumors in this new model correlated with MYC-ATRTs but with some specific features not observed in their human counterparts. All tumors harbored a significant immune infiltration, with some clonal T-cell expansion. Altogether, the Smarcb1Flox/Flox;Prss56-Cre mouse tumors offer a new model for human MRT and promising perspectives for immune-based innovative treatments.
 Citation Format: Zhi-Yan HAN, Mamy Andrianteranagna, Stéphanie Fitte-Duval, Valeria Manriquez, Magali Frah, Rachida Bouarich, Sandrina Turcynski, Arnault Tauziède-Espariat, Kevin Beccaria, Julien Masliah-Planchon, Christine Bourneix, Delphine Guillemot, Stéphanie Reynaud, Gaelle Pierron, Maëva Veyssiere, Owen Hoare, Céline Chauvin, Grégory Thomson, Didier Surdez, Pascale Varlet, Christelle Dufour, Volodia Dangouloff-Ros, Joshua J. Waterfall, Eliane Piaggio, Olivier Delattre, Piotr Topilko, Franck Bourdeaut. Targeting boundary cap cells leads to mouse Smarcb1-deficient peripheral nerve tumors recapitulating human peripheral rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624aa14765b3c481a1fc38ac19c8ad7677137eda" target='_blank'>
              Abstract A006 Targeting boundary cap cells leads to mouse Smarcb1-deficient peripheral nerve tumors recapitulating human peripheral rhabdoid tumors
              </a>
            </td>
          <td>
            Zhi-Yan Han, Mamy Andrianteranagna, Stéphanie Fitte-Duval, Valeria Manriquez, Magali Frah, Rachida Bouarich, Sandrina Turcynski, A. Tauziède-Espariat, K. Beccaria, J. Masliah-Planchon, C. Bourneix, D. Guillemot, S. Reynaud, G. Pierron, Maëva Veyssiere, Owen Hoare, C. Chauvin, Grégory Thomson, D. Surdez, Pascale Varlet, Christelle Dufour, V. Dangouloff-Ros, J. Waterfall, Eliane Piaggio, Olivier Delattre, Piotr Topilko, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Numerous aspects of cellular signaling are regulated by the kinome—the network of over 500 protein kinases that guides and modulates information transfer throughout the cell. The key role played by both individual kinases and assemblies of kinases organized into functional subnetworks leads to kinome dysregulation driving many diseases, particularly cancer. In the case of pancreatic ductal adenocarcinoma (PDAC), a variety of kinases and associated signaling pathways have been identified for their key role in the establishment of disease as well as its progression. However, the identification of additional relevant therapeutic targets has been slow and is further confounded by interactions between the tumor and the surrounding tumor microenvironment. In this work, we attempt to link the state of the human kinome, or kinotype, with cell viability in treated, patient-derived PDAC tumor and cancer-associated fibroblast cell lines. We applied classification models to independent kinome perturbation and kinase inhibitor cell screen data, and found that the inferred kinotype of a cell has a significant and predictive relationship with cell viability. We further find that models are able to identify a set of kinases whose behavior in response to perturbation drive the majority of viability responses in these cell lines, including the understudied kinases CSNK2A1/3, CAMKK2, and PIP4K2C. We next utilized these models to predict the response of new, clinical kinase inhibitors that were not present in the initial dataset for model devlopment and conducted a validation screen that confirmed the accuracy of the models. These results suggest that characterizing the perturbed state of the human protein kinome provides significant opportunity for better understanding of signaling behavior and downstream cell phenotypes, as well as providing insight into the broader design of potential therapeutic strategies for PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd178f02946551d2f3d6c0478dc67e5c83de2358" target='_blank'>
              Kinome state is predictive of cell viability in pancreatic cancer tumor and cancer-associated fibroblast cell lines
              </a>
            </td>
          <td>
            Matthew E. Berginski, Madison R. Jenner, Chinmaya U. Joisa, Gabriela Herrera Loeza, B. Golitz, Matthew B Lipner, J. R. Leary, Naim U. Rashid, Gary L. Johnson, J. J. Yeh, Shawn M. Gomez
          </td>
          <td>2024-08-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The prognosis for patients with brain metastasis remains dismal despite intensive therapy including surgical resection, radiotherapy, chemo-, targeted and immunotherapy. Thus, there is a high medical need for new therapeutic options. Recent advances employing high-throughput and spatially resolved single-cell analyses have provided unprecedented insights into the composition and phenotypes of the diverse immune cells in the metastatic brain, revealing a unique immune landscape starkly different from that of primary brain tumours or other metastatic sites. This review summarises the current evidence on the composition and phenotypes of the most prominent immune cells in the brain metastatic niche, along with their dynamic interactions with metastatic tumour cells and each other. As the most abundant immune cell types in this niche, we explore in detail the phenotypic heterogeneity and functional plasticity of tumour-associated macrophages, including both resident microglia and monocyte-derived macrophages, as well as the T-cell compartment. We also review pre-clinical and clinical trials evaluating the therapeutic potential of targeting the immune microenvironment in brain metastasis. Given the substantial evidence highlighting a significant role of the immune microenvironmental niche in brain metastasis pathogenesis, a comprehensive understanding of the key molecular and cellular factors within this niche holds great promise for developing novel therapeutic approaches as well as innovative combinatory treatment strategies for brain metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d0792a763e0140a3e34412a153d4422e59db4ed" target='_blank'>
              The Immune Landscape in Brain Metastasis.
              </a>
            </td>
          <td>
            Luca D Schreurs, Alexander F. vom Stein, S. Jünger, Marco Timmer, Ka-Won Noh, Reinhard Buettner, Hamid Kashkar, Volker Neuschmelting, R. Goldbrunner, P.H. Nguyen
          </td>
          <td>2024-10-15</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer death by 2030. PDAC lacks effective treatment options and innovative strategies for therapeutic intervention are urgently needed. Chimeric Antigen Receptor (CAR)-T cell therapy has shown remarkable success in treating hematologic malignancies, yet achieving efficacy in solid tumors remains a significant challenge. Early results from clinical trials show limited efficacy of CAR T-cell therapy in PDAC. To uncover tumor-intrinsic mechanisms of resistance to CAR T-cell therapy in vivo, we performed iterative CRISPR/Cas9-based pooled screens in a fully immunocompetent, orthotopic model of PDAC. We identified a variety of genes influencing CAR T-cell treatment efficacy. In particular, loss of genes involved in redox biology and oxidative stress sensitized PDAC tumors to CAR T-cell therapy in vivo. Additionally, single-cell RNA sequencing of PDAC tumors subjected to CAR T-cell therapy unveiled a distinct subset of tumor cells resilient to the treatment, characterized by changes in oxidative stress. Our findings indicate a potential avenue for therapeutic synergy by targeting pathways involved in oxidative stress to increase the efficacy of CAR T cell therapy in PDAC.
 Citation Format: Julia Fröse, Charlie Whittaker, Paul Leclerc, Adam Langenbucher, Riley Hellinger, Daniel R. Goulet, Michael T. Hemann. Unveiling resistance mechanisms to CAR T-cell therapy in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B060.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a130506e26e2da6b9c6338dc0032b746f1f4ca46" target='_blank'>
              Abstract B060: Unveiling resistance mechanisms to CAR T-cell therapy in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening
              </a>
            </td>
          <td>
            Julia Fröse, Charles A. Whittaker, Paul Leclerc, Adam Langenbucher, Riley D Hellinger, Daniel Goulet, M. Hemann
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is characterized by its aggressive behavior and complex molecular heterogeneity, posing significant challenges for treatment and prognostication. This study offers a comprehensive analysis of ccRCC by leveraging both bulk and single-cell RNA sequencing data, with a specific aim to unravel the complexities of sphingolipid metabolism and the intricate dynamics within the tumor microenvironment (TME). By examining ccRCC samples sourced from public databases, our investigation delves deep into the genetic and transcriptomic landscape of this cancer type. Employing advanced analytical techniques, we have identified pivotal patterns in gene expression and cellular heterogeneity, with a special focus on the roles and interactions of various immune cells within the TME. Significantly, our research has unearthed insights into the dynamics of sphingolipid metabolism in ccRCC, shedding light on its potential implications for tumor progression and strategies for immune evasion. A novel aspect of this study is the development of a risk score model designed to enhance prognostic predictions for ccRCC patients, which is currently pending external validation to ascertain its clinical utility. Despite its contributions, the study is mindful of its limitations, including a reliance on observational data from public sources and a primary focus on RNA sequencing data, which may constrain the depth and generalizability of the findings. The study does not encompass critical aspects, such as protein expression, posttranslational modifications, and comprehensive metabolic profiles. Moreover, its retrospective design underscores the necessity for future prospective studies to solidify these preliminary conclusions. Our findings illuminate the intricate interplay between genetic alterations, sphingolipid metabolism, and immune responses in ccRCC. This research not only enhances our understanding of the molecular foundations of ccRCC but also paves the way for the development of targeted therapies and personalized treatment modalities. The study underlines the importance of cautious interpretation of results and champions ongoing research using diverse methodologies to thoroughly comprehend and effectively combat this formidable cancer type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d290ea11babde7963cf17fade3be71816bf90d65" target='_blank'>
              Integrative analysis of bulk and single-cell RNA sequencing reveals sphingolipid metabolism and immune landscape in clear cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Dongdong Xie, Zhitao Han, Yu Wang, Haoyu Shi, Xiang Wu, Jiaqing Wu, Y. Dai
          </td>
          <td>2024-09-04</td>
          <td>Environmental toxicology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background Osteosarcoma (OS) survival rates and outcome have not improved in 50 years since the advent of modern chemotherapeutics. Thus, there is a critical need for an improved understanding of the tumor microenvironment to identify better therapies. Extracellular matrix (ECM) deposition and hypoxia are known to abrogate the efficacy of various chemical and cell‐based therapeutics. Here, we aim to mechanistically investigate the combinatorial effects of hypoxia and matrix deposition with the use of OS spheroids. Methods We use two murine OS cell lines with differential metastatic potential to form spheroids. We form spheroids of two sizes, use ascorbate‐2‐phosphate supplementation to enhance ECM deposition, and study cell response under standard (21% O2) and physiologic (5% O2) oxygen tensions. Finally, we examine chemotherapeutic responses to doxorubicin treatment. Results ECM production and oxygen tension are key determinants of spheroid size through cell organization based on nutrient and oxygen distribution. Interestingly, highly metastatic OS is more susceptible to chemotherapeutics compared to less metastatic OS when matrix production increases. Together, these data suggest that dynamic interactions between ECM production and oxygen diffusion may result in distinct chemotherapeutic responses despite inherent tumor aggressiveness. Conclusion This work establishes OS spheroids as a valuable tool for early OS tumor formation investigation and holds potential for novel therapeutic target and prognostic indicator discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e5e2fb20e2a4e399455c45af17440bdc5ed9071" target='_blank'>
              Extracellular matrix production and oxygen diffusion regulate chemotherapeutic response in osteosarcoma spheroids
              </a>
            </td>
          <td>
            Isabel S Sagheb, Thomas P Coonan, R. L. Randall, Katherine H Griffin, J. K. Leach
          </td>
          <td>2024-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Desmoplastic small round cell tumor (DSRCT) is an aggressive sarcoma subtype that is driven by the EWS-WT1 chimeric transcription factor. The prognosis for DSRCT is poor, and major advances in treating DSCRT have not occurred for over two decades. To identify effective therapeutic approaches to target DSRCT, we conducted a high-throughput drug sensitivity screen in a DSRCT cell line assessing chemosensitivity profiles for 79 small-molecule inhibitors. DSRCT cells were sensitive to PARP and ATR inhibitors (PARPi, ATRi), as monotherapies and in combination. These effects were recapitulated using multiple clinical PARPi and ATRi in three biologically distinct, clinically-relevant models of DSRCT, including cell lines, a patient-derived xenograft (PDX)-derived organoid model, and a cell line-derived xenograft mouse model. Mechanistically, exposure to a combination of PARPi and ATRi caused increased DNA damage, G2/M checkpoint activation, micronuclei accumulation, replication stress, and R-loop formation. EWS-WT1 silencing abrogated these phenotypes and was epistatic with exogenous expression of the R-loop resolution enzyme RNase H1 in reversing the sensitivity to PARPi and ATRi monotherapies. The combination of PARPi and ATRi also induced EWS-WT1-dependent cell-autonomous activation of the cGAS/STING innate immune pathway and cell surface expression of PD-L1. Taken together, these findings point towards a role for EWS-WT1 in generating R-loop-dependent replication stress that leads to a targetable vulnerability, providing a rationale for the clinical assessment of PARPi and ATRi in DSRCT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33f00435911403846b95b540256a684bbb112555" target='_blank'>
              Replication Stress is an Actionable Genetic Vulnerability in Desmoplastic Small Round Cell Tumors.
              </a>
            </td>
          <td>
            Asuka Kawai-Kawachi, Madison M Lenormand, Clémence Astier, Noé Herbel, Meritxell B Cutrona, Carine Ngo, M. Garrido, Thomas Eychenne, N. Dorvault, Laetitia Bordelet, Fei Fei Song, Ryme Bouyakoub, Anastasia Loktev, Antonio Romo-Morales, C. Henon, Léo Colmet-Daage, Julien Vibert, Marjorie Drac, R. Brough, Etienne Schwob, Oliviano Martella, G. Pinna, Janet M Shipley, Sibylle Mittnacht, Astrid Zimmermann, Aditi Gulati, Olivier Mir, Axel Le Cesne, M. Faron, Charles Honoré, Christopher J. Lord, Roman Chabanon, S. Postel-Vinay
          </td>
          <td>2024-10-16</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Demonstrating a combination of high prevalence, late detection, and low survival, Lung cancer is the leading cause of cancer related deaths worldwide (Sainz de Aja J, 2021) and Lung Adenocarcinoma (LUAD) is its most prevalent pathologic subtype (Zhou, 2021). Emerging from within the distal alveolar epithelium, LUAD presents with vast heterogeneity in histology, mutational spectra, and epigenomic dysregulation (Sainz de Aja J, 2021) (Zhou, 2021). Additionally, LUAD’s considerable intratumoral heterogeneity manifests vast genomic and phenotypic diversity across tumor cells and cellular components of the tumor microenvironment (TME) within the same tumor tissue. LUAD also reveals significant racial disparities in incidence, survival and mortality rates between White and Black men in the United States. The occurrence of LUAD is approximately 68.3 per capita for Black men and 61.5 per capita for White men, reflecting an 11% higher rate in the former. Further, survival outcomes indicate that Black men generally exhibit lower 5-year relative survival rates compared to White men, highlighting greater mortality risk and conspicuously poorer prognosis (Siegel RL et al., 2023). While addressing the social determinants of health is vital for the mitigation of health disparities, understanding the genetic and molecular foundations that underly racial health inequities is essential for developing comprehensive interventions and advancing more precise therapeutic modalities for more diverse patient populations. Single Nucleotide Polymorphisms (SNPs) can influence oncogenic transformation and may be associated with distinct molecular landscapes in tumors and adjacent TME. Furthermore, ancestry may play a pivotal role in shaping individual susceptibility and risk for mitochondrial dysfunction and ensuing oncogenic transformation. To elucidate the biological underpinnings of racial health disparities in LUAD, 10 Black and 10 White male LUAD patients’ tumor and normal corresponding adjacent tissues underwent molecular profiling for KRAS mutation status, Whole Exome Sequencing (WES) to determine ancestry, and analysis of associated LUAD induced Mitochondrial Somatic Nucleotide Variants (mtSNVs). We also employed the 10X Visium HD spatial transcriptomic platform to evaluate spatially resolved molecular and cellular differences in KRAS mediated LUAD between Black and White men. Hypothesis: Thus, we propose that global ancestry is associated with variations in intratumoral heterogeneity, TME composition, and mitochondrial dynamics, indicating a potential role in the observed disparities in susceptibility, malignancy and mortality in LUAD across diverse patient populations.
 Citation Format: Eun Kyu Sung, Yuxin Jin, Yonatan Amzaleg, John Carpten, William Dean Wallace, Dan Raz, Aaron M. Neely. Investigating the effects of ancestry in KRAS mediated lung adenocarcinoma: Exploring intratumoral heterogeneity and mitochondrial dynamics [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C169.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32fd50ebcf66bd13578e33674dfa106c7a5fd8db" target='_blank'>
              Abstract C169: Investigating the effects of ancestry in KRAS mediated lung adenocarcinoma: Exploring intratumoral heterogeneity and mitochondrial dynamics
              </a>
            </td>
          <td>
            Eun Kyu Sung, Yuxin Jin, Yonatan Amzaleg, John D. Carpten, W. D. Wallace, Dan Raz, Aaron M. Neely
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Introduction Cancer-associated fibroblasts (CAFs) are key players in the fibrotic tumour microenvironment (TME) of pancreatic cancer. Recent evidence has demonstrated the existence of multiple CAF subtypes, however there is relatively little evidence validating and corroborating these CAF subtypes in resected human pancreatic cancer. Furthermore, the spatial interactions between CAF subtypes and the epithelial and immune compartments of the pancreatic cancer TME is not well understood. Methods We designed and optimised a bespoke 8-plex immunofluorescence panel for the purposes of CAF subtype profiling and elucidation of CAF spatial biology: panCK, FAP, SMA, CTGF, S100A4, IL6, LIF, PDGFR. This was applied to a cohort of n=215 resected pancreatic cancer, well characterised with highly granular clinico-pathological and transcriptomic data through the Australian Pancreatic Cancer Genome Initiative. Spatial analysis was performed with novel computational biology techniques. Results We demonstrate seven CAF subtypes which cluster to form distinct, highly prognostic “enrichment profiles” (EPs). The cold EP represents quiescent CAFs in close proximity to immune cells (multivariate HR 0.45, 95% CI 0.30-0.68, p<0.001), and the fibrotic EP represents activated CAFs in close proximity to epithelial cells (multivariate HR 2.22, 95% CI 1.48-3.35, p<0.001). We also demonstrate the prognostic power of high spatial entropy (loss of tissue architecture, correlating with the fibrotic EP) vs low spatial entropy (preservation of tissue architecture, correlating with the cold EP) (HR 2.22, 95% CI 1.47-3.35, p<0.001). Conclusion We begin to unravel the elusive spatial biology of pancreatic CAFs in a large resected human cohort. We see changes in immune cell infiltrate, modulation of cellular relationships, and greater tissue disorganisation as a result of CAF activation. These features are highly prognostic and we hypothesise they are driven by changes in crosstalk between TME compartments. Current work focuses on application of these techniques to a large resected neoadjuvantly-treated cohort, allowing further interrogation of the notorious fibrotic post-neoadjuvant pancreatic cancer TME. If accepted for presentation we look forward to presenting both cohorts simultaneously.
 Citation Format: Adam Bryce, Stephan Dreyer, Fieke Froeling, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, John Le Quesne, David Chang. Interrogating the spatial biology of pancreatic cancer-associated fibroblasts in resected human pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A053.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179192d42336b982da176e9aa5fe870344303cba" target='_blank'>
              Abstract A053: Interrogating the spatial biology of pancreatic cancer-associated fibroblasts in resected human pancreatic cancer
              </a>
            </td>
          <td>
            A. Bryce, S. Dreyer, F. Froeling, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, John Le Quesne, David Chang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The membrane composition of extracellular vesicles (EVs) largely reflects that of the plasma membrane of the cell of origin. We therefore hypothesized that EVs could be used for immunizations to generate monoclonal antibodies against well-known tumor antigens but possibly also against hitherto unknown tumor-associated target molecules. From an immunization experiment, we obtained a monoclonal antibody specific for SRRM2, an RNA-binding protein involved in splicing and a major component of nuclear speckles. Here, we used this antibody to demonstrate that SRRM2 is exposed on the surface of most cancer cell lines from various entities and, even more important, on cancer cells in vivo. Moreover, we demonstrated that SRRM2-specific CAR-T cells are functional in vitro and in vivo. Collectively, we identified SRRM2 as a promising new target molecule exposed on the cancer cell surface and showed that our SRRM2-specific antibody can be used as a basis for the development of new targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e28b566c1806f22e74663da9c96c01cab220620e" target='_blank'>
              The Nuclear Speckles Protein SRRM2 Is Exposed on the Surface of Cancer Cells
              </a>
            </td>
          <td>
            Markus Kellner, Julia Hörmann, Susanne Fackler, Yuanyu Hu, Tielin Zhou, Lin Lu, Ibrahim Ilik, Tuğçe Aktaş, Regina Feederle, Stefanie M. Hauck, Olivier Gires, Kathrin Gärtner, Lietao Li, Reinhard Zeidler
          </td>
          <td>2024-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Glioblastoma Multiforme (GBM) is a high-grade glioma known to infiltrate the surrounding brain parenchyma inducing a direct neuronal damage. Indeed, GBM mass is not an isolated and self-sustained entity but rather participates and interacts with the surrounding neuronal tissue. Specifically, tumoral cells induce neuron hyper-excitability while peritumoral cells seem to regulate tumor proliferation and progression. GBM affects 3 in 100,000 people and has a median survival of 15 months; it remains poorly understood and lethal. Therefore, gaining a deeper understanding of the alterations occurring in peritumoral tissues during GBM progression is of paramount importance.



 We investigate the plastic rearrangements of motor cortical areas in two different glioma-bearing mouse models using optogenetic stimulation, two photon imaging and immunohistochemical analyses. Specifically, we took advantage of Thy1-ChR2 transgenic mice, injected with two syngeneic lines of murine glioma cells (i.e. GL261 and PDGF+ Trp53-/- cells) into the motor cortex. Mice were optogenetically stimulated in 120 points per session (covering the entire extent of forelimb motor representation) at 3 different time points: baseline (before glioma cells injection), 14 and 21 days after tumor implantation.



 We show that both models of glioma-bearing mice underwent a strong remapping of forelimb motor territory. Namely, an increased threshold required for eliciting a forelimb movement was coupled with a decrease of forelimb specificity in the peritumoral area with respect to naïve mice. In addition, glioma-bearing mice showed a decrease in the number of dendritic spines, along with reduced density of perineuronal nets (PNNs) and inhibitory interneuron markers in the peritumoral cortex.



 We provide important insight into the interaction between GBM and surrounding tissue that can be used for developing more effective treatments to improve patients’ quality of life and survival.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6297cf9746d4f9b221b8e8f9f648c2b95db0018f" target='_blank'>
              P02.22.A REARRANGEMENTS OF PERITUMORAL TISSUE THAT OCCUR ALONG WITH GLIOMA PROGRESSION
              </a>
            </td>
          <td>
            E. Mori, C. Spalletti, M. Scalera, S. Haddad, D. Cangi, V. Pillai, E. Parmigiani, M. Mainardi, S. Landi, M. Caleo, E. Vannini
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 18],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>