<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-25 09:30:33 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c71e23566365a6eb56245362a90fb9a248ccb88" target='_blank'>
              SpatioEv: Spatial evolution of protein and morphological features reveals development dynamics of cells and spatial neighbourhoods
              </a>
            </td>
          <td>
            Shihong Wu, Sakina Amin, Carl Lee, Jean-Baptiste Richard, Nabeel Merali, Caroline Morrell, Lauren Overend, Weijia Gao, Felicia Tucci, Alex Gordon-Weeks, Adam E. Frampton, Nicola Annels, Emma Culver, Michael Dustin, Kim S. Midwood, Rachael Bashford-Rogers
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76828c8834b0e2a09dc4aceaa26a03b54ca8e902" target='_blank'>
              Spatial tumour-immune ecosystems shape the efficacy of anti-PD1 immunotherapy in primary cutaneous melanoma
              </a>
            </td>
          <td>
            Félix Pham, Marion Dufeu, Valentin Benboubker, Maxime Grimont, Amélie Lhorisson, Justine Berthet, Marie Donzel, Raphaël Schneider, Laurie Tonon, A. Doffin, Félix Boivin, Simon Durand, Bertrand Dubois, Jonathan Lopez, Christophe Caux, J. Valladeau-Guilemond, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Metastasis is the predominant cause of cancer mortality, primarily driven by complex tumor–host interactions within specialized metastatic niches. Recent advances in single-cell technologies have provided unprecedented insights into metastatic niche formation, evolution and function, including how primary tumors precondition distant organs for metastases and how disseminated tumor cells dynamically interact with host cells to modulate their environments. Integrated single-cell studies across multiple cancer types have also revealed divergent and convergent metastatic adaptation strategies. These findings collectively highlight metastasis as a dynamic, cooperative process shaped by intricate tumor–host interactions, and provide a foundation for novel therapeutic strategies targeting components of the metastatic microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff161933eaf3d02dd3bc2d33e3ccaf6a23f9f8b" target='_blank'>
              Decoding metastatic microenvironments through single-cell omics reveals new insights into niche dynamics and tumor evolution
              </a>
            </td>
          <td>
            Fengshuo Liu, X. Zhang
          </td>
          <td>2025-07-01</td>
          <td>PLOS Biology</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Cutting-edge multiplex imaging technologies have significantly advanced our understanding of the tumor immune microenvironment (TIME), delivering nanometer-scale spatial resolution that illuminates previously inaccessible cellular interactions and organizational patterns. This mini-review discusses the latest multiplex imaging methods, including Imaging Mass Cytometry (IMC), Multiplexed Ion Beam Imaging (MIBI), Cyclic Immunofluorescence (CycIF), and Digital Spatial Profiling (DSP), emphasizing their roles in identifying spatial immune signatures predictive of immunotherapy responses. Clinical applications across various cancers—such as NSCLC, melanoma, breast cancer, colorectal cancer, and hepatocellular carcinoma—highlight how spatially resolved immune profiles can enhance patient stratification and treatment personalization. Notably, increased CD8+ T cell density and spatial colocalization with tumor cells have been broadly correlated with improved immunotherapy response and survival across multiple cancer types. Despite current technical and analytical challenges, ongoing technological advancements and integration with emerging methods like spatial transcriptomics and super-resolution imaging promise broader clinical utility, ultimately improving patient outcomes in precision immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b65c011095a02fc98758e3d9804d5f1395199b21" target='_blank'>
              Multiplex imaging analysis of the tumor immune microenvironment for guiding precision immunotherapy
              </a>
            </td>
          <td>
            Feng Liu, Guiqing Li, Yi Zheng, Yanxia Liu, Kang Liu
          </td>
          <td>2025-07-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Summary Intratumoral heterogeneity, originating from genetic, epigenetic, and phenotypic cellular diversity, is pervasive in cancer. As these heterogeneous states employ diverse mechanisms to promote tumor progression, metastasis, and therapy resistance, emergence of cancer heterogeneity is one of the most significant barriers to curative treatment. Here we leverage deep learning approaches to develop a high-throughput image-analysis paradigm with subcellular resolution that quantifies and predicts colorectal cancer (CRC) patient outcome based on tissue architecture and nuclear morphology. We further combine this approach with spatial transcriptomics, multiplex immunohistochemistry, and patient-derived organoids to uncover a dynamic, co-evolutionary relationship between tumor architecture and cell states. We identify clinically relevant architectural interfaces in CRC tissue that diversify cellular identities by favoring distinct cancer stem cell states and thus promote evolution of tumor heterogeneity. Specifically, tissue fragmentation and associated compressive forces promote loss of classic stem cell signature and acquisition of fetal/regenerative stem cell states, which initially emerges as a hybrid state with features of epithelial-to-mesenchyme (EMT) transition. Additional tumor stroma communication then diversifies these states into distinct stem cell and EMT states at the invasive margins of tumors. Reciprocally, Wnt signaling state of tumor cells tunes their responsiveness to tissue architecture. Collectively, this work uncovers a feedback loop between cell states and tissue architecture that drives cancer heterogeneity and cell state diversification. Machine learning-based analyses harness this co-dependency, independently of mutational status or cancer stage, to predict patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8d83b8caf4ac682c803318ebb577d7836c261e" target='_blank'>
              Colorectal cancer heterogeneity co-evolves with tumor architecture to determine disease outcome
              </a>
            </td>
          <td>
            Fabien Bertillot, Solène Hervé, Nelly Drobjazko, Noemi Kedei, Mona El Gendi, Maria O. Hernandez, Kai Kruse, E. Erkan, Mirjam I. Binner, Karolina Punovuori, T. Pellinen, M. Ahtiainen, Jan Böhm, J. Mecklin, Caj Haglund, Jaana Hagström, Mike Nikolaev, N. Gjorevski, Matthias Lütolf, Toni T Seppälä, Sara A. Wickström, Yekaterina A. Miroshnikova
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Simple Summary While subcutaneous colorectal cancer (CRC) models are commonly employed in biological and preclinical studies, their cell morphological organization and molecular landscape are not yet fully understood. Using spatial transcriptomics, we identified two spatially distinct tumor regions demarcated by cancer-associated fibroblasts (CAFs), each region exhibiting unique molecular signatures and immune niche. Moreover, we uncovered specific cancer–stroma cell interactions that actively remodel the tumor microenvironment to facilitate immune evasion. These findings uncover the previously unrecognized spatial heterogeneity of the CRC mouse model and highlight the critical role of specialized tumor niches in driving tumor development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acb09b0c3fbf7dabf3a2ff1e9a531d5a3277e04" target='_blank'>
              Cancer-Associated Fibroblasts Establish Spatially Distinct Prognostic Niches in Subcutaneous Colorectal Cancer Mouse Model
              </a>
            </td>
          <td>
            Zhixian Lin, Jinmeng Wang, Yixin Ma, Yanan Zhu, Yuhan Li, Zhengtao Xiao, Wei Zhao
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6cfe1f46dc56e476a3d0be0d700257cc9ae2152" target='_blank'>
              Single-Cell and Spatial Profiling of Tumor Microenvironment Heterogeneity in Human Osteosarcomas
              </a>
            </td>
          <td>
            Xuejing zheng, Jin Yuan, Peng Jin, Xuejun Wang, Runlei Xu, Helin Feng, Xinxin Zhang, Yongjie Xie, Wence Wu
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Racial disparities in the clinical outcomes of triple-negative breast cancer (TNBC) have been well-documented, but the underlying biological mechanisms remain poorly understood. To investigate these disparities, we employed a multi-omic approach integrating imaging mass cytometry and spatial transcriptomics to characterize the tumor microenvironment (TME) in self-identified Black American (BA) and White American (WA) TNBC patients. Our analysis revealed that the TME in BA patients is marked by a network of endothelial cells, macrophages, and mesenchymal-like cells, which correlates with reduced patient survival. In contrast, the WA TNBC microenvironment is enriched in T-cells and neutrophils, indicative of T-cell exhaustion and suppressed immune responses. Ligand-receptor and pathway analyses further demonstrated that BA TNBC tumors exhibit a relatively “immune-cold” profile, while WA TNBC tumors display features of an “inflamed” TME, suggesting the evolution of a unique immunosuppressive mechanism. These findings provide insight into racially distinct tumor-promoting and immunosuppressive microenvironments, which may contribute to the observed differences in clinical outcomes among BA and WA TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2139c15b76d963542dd090ab4bb098c3eb9f6a" target='_blank'>
              Integrative spatial omics reveals distinct tumor-promoting multicellular niches and immunosuppressive mechanisms in Black American and White American patients with TNBC
              </a>
            </td>
          <td>
            Qian Zhu, Akhila Balasubramanian, Jaya Ruth Asirvatham, Megha Chatterjee, Badrajee Piyarathna, Jaspreet Kaur, Nada Mohamed, Ling Wu, Stacy Wang, Niloufar Pourfarrokh, P. D. Binsol, Mahak Bhargava, Uttam Rasaily, Yitian Xu, Junjun Zheng, D. Jebakumar, Arundhati Rao, Carolina Gutierrez, A. Omilian, Carl D. Morrison, Gokul M. Das, C. Ambrosone, Erin H. Seeley, Shu-hsia Chen, Yi Li, Eric Chang, Xiaoxian Li, Elizabeth Baker, Ritu Aneja, X. Zhang, Arun Sreekumar
          </td>
          <td>2025-07-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transcriptome profiling of bladder cancer has revealed distinct basal-like and luminal-like molecular subtypes, which may be correlated with pathological subtypes of different patient outcomes. However, whether these molecular subtypes originate from the corresponding cell types in the normal urothelium and whether different cells of origin influence bladder cancer progression remain unclear. Here, we conducted cell-type-specific lineage tracing in CRISPR/Cas9-induced mouse bladder cancer models of Pten and Trp53 targeting. We show that although basal, intermediate, and superficial umbrella cells can all serve as the cell of origin for bladder cancer, transformed umbrella cells were gradually displaced by tumor cells from inner layers, particularly transformed basal cells, which had highest stemness. Histological and single cell RNA-sequencing data comparing basal- and intermediate-cell-induced bladder tumors revealed that basal-induced tumors displayed higher heterogeneity, and contained unique cell clusters including Krt14+Ki67+ highly proliferative basal cells, squamous cell carcinoma, and transitioning cells towards the Gata3+ luminal subtype. Trajectory analysis confirmed the cell lineage differentiation hierarchy uncovered in lineage tracing. Moreover, human bladder cancer molecular subtype signatures were highly enriched in mouse tumor cell clusters of the corresponding cell of origin, and a gene signature derived from the unique basal-induced clusters is predictive of worse patient outcome. Overall, our results support that the basal and luminal molecular subtypes of bladder cancer have the corresponding cells of origin as their basis, and that urothelial basal cells are intrinsically more competitive than intermediate and umbrella cells in generating aggressive bladder cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27c74441168c668a9994b2523834450566ba1b30" target='_blank'>
              Intrinsic higher potency of basal urothelial cells to intermediate and umbrella cells as the cell of origin for bladder cancer
              </a>
            </td>
          <td>
            Chuan Yu, Nicholas Chu, Alexandra Aguirre, Jonathan Green, Qing Xie, Beatrice Knudsen, Zhu A. Wang
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe2aa8fe44537c3fe83f62a4d4acf8f8b14517" target='_blank'>
              STHELAR, a multi-tissue dataset linking spatial transcriptomics and histology for cell type annotation
              </a>
            </td>
          <td>
            Félicie Giraud-Sauveur, Quentin Blampey, Hakim Benkirane, A. Marinello, PH Cournède, Stergios Christodoulidis
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91860355badc3254098a406329f26559d939957e" target='_blank'>
              Optimised dissociation and multimodal profiling of prostate cancer stroma reveal fibromuscular cell heterogeneity with clinical correlates
              </a>
            </td>
          <td>
            E. Damisch, Elena Brunner, Lukas Nommensen, Lucy Neumann, Georgios Fotakis, Z. Trajanoski, S. Sopper, Georg Schäfer, M. Puhr, Isabel Heidegger, Marianna Kruithof-de Julio, Natalie Sampson
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/963ead743bf88aafb98699a954ce89dbd9be99ad" target='_blank'>
              Decoding Lymphangioleiomyomatosis (LAM) Niche Environment via Integrative Analysis of Single Cell Multiomics and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ken Chen, Shuyang Zhao, M. Guo, Hasan Al Reza, Andrew Wagner, Adnan Cihan Cakar, Cheng Jiang, Erik Zhang, Jenna Green, Emily Martin, Kathryn A. Wikenheiser-Brokamp, A. Perl, Debora Sinner, Jane J. Yu, Yan Xu
          </td>
          <td>2025-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Tumor relapse remains a significant obstacle to successful therapy. Preclinical animal models that accurately reflect tumor relapse in patients are urgently needed. Here, we employed a dual recombinase-mediated genetic system to genetically trace and ablate proliferating cells in a polyomavirus middle T antigen (PyMT)-induced spontaneous murine breast cancer model. This system enabled the acute ablation of cells that had undergone proliferation within a defined time window, resulting in a drastic tumor shrinkage, followed by a gradual tumor relapse due to the presence of residual low-cycling cells. We then applied single-cell RNA sequencing (scRNA-seq) to unbiasedly compare the tumor ecosystems of the primary and relapsed PyMT tumors. Compared with the primary tumors, the relapsed tumors exhibited a higher proportion of cancer stem cells and pro-tumor γδ T cells, as well as co-expression of Spp1 and Vegfa in multiple myeloid cell populations – features that predict poor therapeutic response and unfavorable outcomes in human breast cancer patients. Collectively, this proliferation tracing and ablation model emulates chemotherapies that preferentially eliminate proliferating cancer cells, serving as a robust tool and a valuable resource for testing novel therapeutic strategies in relapsed tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95fdfebff6c6c3226be581651d876e8c2d584753" target='_blank'>
              Modeling tumor relapse using proliferation tracing and ablation transgenic mouse
              </a>
            </td>
          <td>
            Chuang Zhao, Xin-nan Zheng, Han-Ying Huang, Lin Tian
          </td>
          <td>2025-07-17</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47dca4509610b2f00715550942002b3b04fd28df" target='_blank'>
              Tracing colorectal malignancy transformation from cell to tissue scale
              </a>
            </td>
          <td>
            Helena L. Crowell, Irene Ruano, Zedong Hu, Yourae Hong, G. Caratù, Hubert Piessevaux, Ashley Heck, Rachel Liu, Max Walter, Megan Vandenberg, Kimberly Young, Dan McGuire, Evelyn Metzger, Margaret L Hoang, Joseph M. Beechem, Sabine Tejpar, A. Pascual-Reguant, Holger Heyn
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Xenium, a new spatial transcriptomics platform, enables subcellular-resolution profiling of complex tumor tissues. Despite the rich morphological information in histology images, extracting robust cell-level features and integrating them with spatial transcriptomics data remains a critical challenge. We introduce CellSymphony, a flexible multimodal framework that leverages foundation model-derived embeddings from both Xenium transcriptomic profiles and histology images at true single-cell resolution. By learning joint representations that fuse spatial gene expression with morphological context, CellSymphony achieves accurate cell type annotation and uncovers distinct microenvironmental niches across three cancer types. This work highlights the potential of foundation models and multimodal fusion for deciphering the physiological and phenotypic orchestration of cells within complex tissue ecosystems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0df7c5843203625e193cd9c5f8b65c48433cc0" target='_blank'>
              CellSymphony: Deciphering the molecular and phenotypic orchestration of cells with single-cell pathomics
              </a>
            </td>
          <td>
            Paul H. Acosta, Pingjun Chen, Simon P. Castillo, M. E. Salvatierra, Yinyin Yuan, Xiaoxi Pan
          </td>
          <td>2025-08-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="While pathology foundation models have transformed cancer image analysis, they often lack integration with molecular data at single-cell resolution, limiting their utility for precision oncology. Here, we present PAST, a pan-cancer single-cell foundation model trained on 20 million paired histopathology images and single-cell transcriptomes spanning multiple tumor types and tissue contexts. By jointly encoding cellular morphology and gene expression, PAST learns unified cross-modal representations that capture both spatial and molecular heterogeneity at the cellular level. This approach enables accurate prediction of single-cell gene expression, virtual molecular staining, and multimodal survival analysis directly from routine pathology slides. Across diverse cancers and downstream tasks, PAST consistently exceeds the performance of existing approaches, demonstrating robust generalizability and scalability. Our work establishes a new paradigm for pathology foundation models, providing a versatile tool for high-resolution spatial omics, mechanistic discovery, and precision cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9399a2bf1b84832c86d75a49ae3c9a7107309283" target='_blank'>
              PAST: A multimodal single-cell foundation model for histopathology and spatial transcriptomics in cancer
              </a>
            </td>
          <td>
            Changchun Yang, Haoyang Li, Yushuai Wu, Yilan Zhang, Yifeng Jiao, Yu Zhang, Rihan Huang, Yuan Cheng, Yuan Qi, Xin Guo, Xin Gao
          </td>
          <td>2025-07-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a2fe9a625c4ba7b1dd23966948293739db34623" target='_blank'>
              In Situ Inference of Copy Number Variations in Image-Based Spatial Transcriptomics
              </a>
            </td>
          <td>
            Augusta Jensen, Helena L. Crowell, Anna Pascual Reguant, Irene Ruano, Sabine Tejpar, Holger Heyn, Mats Nilsson, Sergio Marco Salas
          </td>
          <td>2025-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal cancer (CRC), including both microsatellite instability (MSI) and microsatellite stability (MSS) subtypes, frequently exhibits intrinsic resistance to immunotherapy. However, the spatial tumor microenvironment (TME) and its role in distinguishing immunotherapy responders from non-responders remain poorly understood. In this study, spatial multiomics, including imaging mass cytometry (n = 50 in-house), spatial proteomics (n = 50 in-house), and spatial transcriptomics (n = 9 in-house), were employed to elucidate the spatial TME of metastatic CRC (mCRC) patients receiving immunotherapy. These methodologies were integrated with single-cell RNA sequencing (scRNA-seq), bulk RNA-seq, and bulk proteomics for comprehensive analysis and validation. A spatial immune atlas containing 314,774 cells was constructed. We found that C1QC+ resident tissue macrophages (RTMs) were more abundant in responders regardless of microsatellite status. Co-localization of C1QC+ RTMs with CD4+ T cells was observed in responders, and MHC-II expression facilitated their interaction. In contrast, cancer-associated fibroblasts inhibited this interaction in non-responders. Moreover, whole genome screening identified key genes involved in antigen presentation in C1QC+ RTMs. Hence, our study highlights the importance of spatial immune mapping in revealing the complex spatial topology of CRC and corresponding immunotherapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcb9f523addd61c5cd3fcf55f0e488cedb552f07" target='_blank'>
              Spatially resolved C1QC+ macrophage-CD4+ T cell niche in colorectal cancer microenvironment: implications for immunotherapy response
              </a>
            </td>
          <td>
            Hangyu Zhang, Libing Hong, Zhen Dong, Shan Xin, Bo Lin, Jinlin Cheng, Weihong Tian, Bin Li, Jing Wang, Xiaoyan Liu, Chuan Liu, Yuzhi Jin, Yanzhi Feng, Ge Su, Xuqi Sun, Qiqi Liu, Xiaomeng Dai, Yang Gao, Zhou Tong, Lulu Liu, Xudong Zhu, Yi Zheng, Peng Zhao, Tiannan Guo, Weijia Fang, Xuanwen Bao
          </td>
          <td>2025-07-01</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Simple Summary The tumor microenvironment (TME) is known to influence both disease progression and treatment outcomes in cancer. In this study, we analyzed high-dimensional imaging mass cytometry data from lung adenocarcinoma and triple-negative breast cancer to examine how the spatial positioning of immune cells relates to clinical outcomes. We developed a relative distance (RD) score that quantifies the proximity of immune cell pairs to cancer cells. This metric showed stronger associations with patient survival and treatment response than conventional features such as cell fraction or density. Our results suggest that spatial immune features may serve as robust biomarkers for patient stratification and personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1da098f6c41405a345327975831d3ae20b4e8e98" target='_blank'>
              Spatial Proximity of Immune Cell Pairs to Cancer Cells in the Tumor Microenvironment as Biomarkers for Patient Stratification
              </a>
            </td>
          <td>
            Jian-Rong Li, Xingxin Pan, Yupei Lin, Yanding Zhao, Yanhong Liu, Yong Li, Christopher I Amos, Chao Cheng
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Objective To illustrate a new barcoded orthotopic patient‐derived xenograft (PDX) mouse model where one can investigate phenotypic effects of single‐cell level gene manipulation in a pooled format. To address some concerns of current PDX mouse models of head and neck squamous cell carcinoma (HNSCC): (1) genomic evolution with passage by generating high‐purity cancer cells, which can also be utilized for other downstream applications, including cell culture‐based studies, and (2) cost‐effectiveness of current PDX models. Methods Two‐millimeter tumor cubes from nine patients were implanted into immunodeficient mouse flanks subcutaneously. Purified tumor cells were obtained from subcutaneous xenografts. Various numbers of purified tumor cells were then injected into the lingual tissue of immunodeficient mice, and the lowest amount of cells needed to achieve a 100% orthotopic engraftment rate were identified. Clonal stability was tested using a lentiviral barcoding system. The orthotopic PDXs' genetic landscapes were characterized using whole exome sequencing. Results This approach yielded an overall engraftment rate of 88.9%. The purification process increased cancer cell purity from 34% to 92%. Lingual injection of 100,000 purified tumor cells achieved a 100% orthotopic engraftment rate from purified subcutaneous PDX tumor cells while maintaining clonal and genetic stability. Conclusion Our study presents a barcoded orthotopic patient‐derived xenograft model for head and neck squamous cell carcinoma with clonal stability. This model provides a way to study phenotypic effects of single cell level gene manipulation in a pooled format. The method can be adapted for in vitro work as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/770f65963b9ed4389a6e68906eb989805afcd015" target='_blank'>
              Barcoded Orthotopic Patient‐Derived Head & Neck Squamous Cell Carcinoma Model Demonstrating Clonal Stability and Maintenance of Cancer Driver Mutational Landscape
              </a>
            </td>
          <td>
            Peiran Zhou, Claire B Mills, Zhao Ming Dong, Brittany R Barber, Slobodan Beronja
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Meningioma is a common primary intracranial tumor with high recurrence and metastasis rate. Delineating the pathological ecosystem at the brain-tumor interface (BTI) of meningioma is critical for understanding the mechanisms of tumor metastasis and developing effective new therapies. Methods To identify biomarkers of early metastasis and discover potential therapeutic targets, we integrated single-cell transcriptome datasets of meningioma, and identified the cell populations and molecular signatures uniquely present at the BTI. Results A specific BTI-enriched tumor cell population with a pro-EMT (epithelial mesenchymal transition) characteristics was associated with invasion and metastasis, and ANXA2 and COL5A1 were detected as the biomarkers for these BTI-enriched tumor cells. Additionally, we characterized the BTI-specific immunosuppressive microenvironment composed of SPP1+ tumor-associated macrophages, as well as specific endothelial cells (ACKR1high) and pericytes (THY1high) promoting the highly malignant invasive state of angiogenesis. Conclusions Collectively, BTI in meningioma is a metastatic and immunosuppressive zone. We have discovered potential biomarkers that help detect early metastasis and recurrence of meningioma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06935-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dea39ae7cd89d7f388a32faad30c0efc42673529" target='_blank'>
              Single-cell transcriptomic analysis reveals metastatic and immunosuppressive characteristics in meningioma brain-tumor interface
              </a>
            </td>
          <td>
            Boya Huang, Jinlian Liang, Xiaogen Tang, Yue Zhu, Lijuan Gao, Dongwei Fu, Cheng Wei, Yifang Li, Chao Ke, Hongyi Zhang, Oscar Junhong Luo
          </td>
          <td>2025-08-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cell-in-cell (CIC) structures, in which one cell is entirely engulfed by another, have been associated with poor outcomes in cancers. However, the mechanisms underlying this association remain poorly understood. We performed multiplex imaging of 56 cell identity, cell ‘state’ and cancer ‘hallmark’ proteins to characterise CICs, map their spatial interactions, and assess clinical associations across 444 tumour cores from 148 colorectal cancer patients, which contained over one million spatially resolved cells. We found that tumour regions containing CICs were associated with lower levels of cytotoxic T cells. We identified upregulated glucose metabolism as a consistent metabolic hallmark of CICs independent of cell type. Spatial analyses revealed that T cells adjacent to CICs underwent selective remodeling with distinct apoptotic and metabolic signatures. Finally, the presence of T cells within CIC neighbourhoods identified a subset of patients with improved survival. Our findings suggest that CICs may be a feature of metabolically competitive niches and a potential factor contributing to T-cell exclusion in tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bc73acfaddb58e731cdec2701f17feb2d562ab2" target='_blank'>
              Multiplex analysis of colorectal cancer tissue describes the composition, cell biology and spatial effects of cell-in-cell events and identifies a T cell-dependent prognostic signature
              </a>
            </td>
          <td>
            E. Bozkurt, Batuhan Kısakol, Mohammadreza Azimi, H. Dussmann, Sanghee Cho, E. McDonough, Joana Fay, Tony O’Grady, John P Burke, N. McCawley, Deborah A. McNamara, Daniel B. Longley, Fiona Ginty, Jochen H M Prehn
          </td>
          <td>2025-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Chronic respiratory diseases such as asthma and COPD involve interactions between multiple resident and immune cell types within bronchial airways, resulting in structural and functional changes. Thus cellular heterogeneity, arrangements and associated neighborhoods as well as interactions between cells and matrices represent intriguing yet challenging areas of study. Spatial phenotypic profiling facilitates exploration of these issues of the cellular microenvironment and identification of context-dependent cell-cell interactions. Utilizing spatial phenotyping, we interrogated the features and cellular landscape of lungs from non-asthmatics, asthmatics, and COPD in FFPE samples by developing a 10-plex antibody panel for the Akoya PhenoCycler®-Fusion system, focused on immune cells (CD45, CD3, CD4, CD8), proliferative cells (Ki67, PCNA), angiogenesis (CD34), epithelium (E-cadherin), smooth muscle (SMA) and extracellular matrix (collagen). We performed cell segmentation on multiplex immunofluorescence images and quantified marker intensity in each cell. Phenotypes were manually identified after normalization, integration, and clustering cells across samples. The composition, cell profiling, and distribution varied significantly between asthmatics and COPD compared to non-asthmatics emphasizing disease heterogeneity. Spatially agnostic analysis revealed that the matrix cluster was more abundant in COPD compared to non-asthmatics and asthmatics, consistent with a greater role for fibrosis. However, asthmatic patients had a higher proportion of unclassified and CD8 + clusters highlighting immune responses. Co-localization analysis showed near random distribution in non-asthmatics. But strong spatial interaction between T cells and other immune or matrix cells in asthma, and a higher avoidance of smooth muscle and immune cells, and of proliferative markers in both asthmatic and COPD. Niche analysis demonstrated different recurrent cell-cell interactions in asthmatic and COPD cohorts. In COPD, the matrix cell-enriched niche was more abundant, while in asthmatics, the unclassified cell-enriched niche was more prevalent compared to non-asthmatics. These findings provide insights into differential spatial organization of cells and tissues in asthma and COPD, with immune and epithelial mechanisms suggesting active inflammation and remodeling in asthma, but fibrotic processes in COPD, and potential role for vascular processes in both conditions. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12931-025-03315-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4fb506511da56b346daca646c35cd40c21b519d" target='_blank'>
              Spatial phenotyping of human bronchial airways in obstructive lung disease
              </a>
            </td>
          <td>
            L. Khalfaoui, R.M. Moore, J. Villasboas, K. R. Whitaker, B.C. Novotny, M. Thompson, C. Pabelick, Y. Prakash
          </td>
          <td>2025-07-02</td>
          <td>Respiratory Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Recent advances in spatial transcriptomics (ST) and spatial proteomics (SP) technologies have enabled high-dimensional molecular profiling at single-cell resolution, providing deeper insights into the tumour-immune microenvironment. However, these modalities are typically applied to separate tissue sections, limiting direct comparisons across molecular layers. We developed a wet-lab and computational framework to perform and integrate ST and SP from the same tissue section, as demonstrated on human lung cancer samples. Applying ST, SP, and hematoxylin and eosin (H&E) staining from the same section ensured consistency in tissue morphology and spatial context. Computational registration using Weave software allowed accurate alignment and annotation transfer across modalities. This co-registered dataset enabled single-cell level comparisons of RNA and protein expression, revealed segmentation accuracy and transcript-protein correlation analyses within individual cells. Notably, we observed systematic low correlations between transcript and protein levels—consistent with prior findings—now resolved at cellular resolution. Our approach highlights the feasibility and utility of performing spatially-resolved multi-omics analysis on the same section without compromising data quality, facilitating concordance studies and region-specific analysis of immune and tumour markers, and ultimately advancing our understanding of disease heterogeneity at the molecular level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8aed282942d7e95e421a4594b7010065640953" target='_blank'>
              An integrated approach for analyzing spatially resolved multi-omics datasets from the same tissue section
              </a>
            </td>
          <td>
            T. Tran, Felicia Wee, C. Joseph, Wanqiu Zhang, J. Lim, Zhen Wei Neo, Li Yen Chong, Francis Hong Xin Yap, N. H. Patterson, Marc Claesen, Alice Ly, Joe Yeong
          </td>
          <td>2025-07-15</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT Background Cutaneous squamous cell carcinoma (cSCC) and basal cell carcinoma (BCC) are the most prevalent types of nonmelanoma skin cancer (NMSC) and exhibit significant inter‐ and intra‐tumor heterogeneity. cSCC has a higher metastatic potential than BCC, accompanied by a considerable mortality rate. However, the detailed mechanisms of tumor evolution in cSCC have not yet been described. Methods We performed single‐cell RNA sequencing (scRNA‐seq) and T cell receptor (TCR) clonal analysis of skin biopsies from five BCCs, three squamous cell carcinomas in situ (SCCIS), and two invasive squamous cell carcinomas (SCC). Independent SCC specimens were used for spatial transcriptomic (ST) analysis using GeoMx Digital Spatial Profiler (DSP). Result Using scRNA‐seq, we analyzed a total of 117,663 cells. We distinguished cancer cells using copy number variation and identified SCC‐specific genes that potentially contribute to tumor progression. Analysis of tumor clones revealed SCC‐specific COL6A1+/ITGA5+ carcinoma cells which produce CXCL16. We also annotated CXCR6+ regulatory T cells (Tregs) which potentially move toward the tumor site by CXCL16, shaping the immunosuppressive TME. ST analysis supported these clones were located at the invasion site of SCC. Conclusion We suggest COL6A1 and ITGA5 promote the invasive and metastatic property of SCC. We also uncovered how SCC recruits Tregs via the CXCL16/CXCR6 axis to create a TME favorable for its survival. These molecules can be used as potential therapeutic targets for treatment of SCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10a0e1cd92069735154742961ad751b08397d911" target='_blank'>
              CXCL16 Producing Tumor Clones Are Shaping Immunosuppressive Microenvironment in Squamous Cell Carcinoma via CXCR6 Regulatory T Cell
              </a>
            </td>
          <td>
            H. Choi, Sunyoung Jung, Ki-Myo Kim, Mihyun Lee, Jun Ho Park, Sanha Hwang, Seung-Min Cha, Jade N Young, D. Poplausky, Hyunsung Nam, Nicholas Gulati, Chung-Gyu Park, Hyun Je Kim, Ji-Ung Park
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains among the most lethal cancers, with metastasis as the primary driver of mortality. While metastatic mechanisms are shared across malignancies, PDAC metastasis poses unique therapeutic challenges due to the presence of extensive tumor heterogeneity, desmoplasia, and immunosuppression — features that enable diverse migratory behaviors and therapeutic resistance. Recent advances have shown that metastatic progression in PDAC emerges from dynamic interactions between tumor cell–intrinsic and microenvironmental factors, each adapting to evolving stressors throughout the metastatic cascade. In the primary tumor, genomic instability and epigenetic reprogramming generate subclones with heightened invasive potential, while dense stromal reactions and myeloid-dominated immune suppression facilitate escape. During circulation, PDAC cells employ distinctive survival strategies through homotypic clustering and heterotypic interactions with blood components. At distant sites, PDAC cells adapt to organ-specific microenvironments through context-dependent metabolic and immune modulation, resulting in phenotypes that diverge from the primary tumor. In this Review, we examine how tumor-stroma crosstalk mechanisms shape metastatic progression in PDAC, provide a framework for understanding why conventional therapies often fail against metastatic disease, and highlight emerging opportunities for stage- and site-specific therapeutic interventions that target these unique adaptations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48f2d47124253bedbc482deeccc073d8dd0b354b" target='_blank'>
              Metastatic heterogeneity in pancreatic cancer: mechanisms and opportunities for targeted intervention
              </a>
            </td>
          <td>
            R. Maddipati
          </td>
          <td>2025-07-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Cancer therapy resistance (CTR) remains a significant challenge in oncology. Traditional methods like imaging, liquid biopsies and conventional omics analyses provide valuable insights, but lack the spatial resolution to fully characterise heterogeneity of tumour and the tumour microenvironment (TME). Recent advancements in spatial omics technologies offer unprecedented insights into the spatial organisation of tumours and TME. In this review, we summarise current methodologies for CTR research and highlight how spatial omics technologies and computational methods are revolutionising our understanding of CTR mechanisms. We also summarise recent studies leveraging spatial omics to uncover novel insights into CTR across various cancer types and therapies and discuss future opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baf8daf1a76c566edbbdf0a752cb4ca39d6eea13" target='_blank'>
              Cancer therapy resistance from a spatial‐omics perspective
              </a>
            </td>
          <td>
            Yinghao Zhang, Cheng Yang, Xi Chen, Liang Wu, Zhiyuan Yuan, Fan Zhang, Bin-Zhi Qian
          </td>
          <td>2025-07-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The organizational principles of nephronal segments are based on anatomical and physiological attributes that are linked to the homeostatic functions of the kidney. Recent molecular approaches have uncovered layers of deeper signatures and states in tubular cells that arise at various time points on the disease trajectory. Here, we introduce an analytical pipeline of multiplexed spatial protein imaging integrated with RNA expression to characterize proximal tubular subpopulations and neighborhoods in human kidney tissue. We demonstrate that, in reference tissue, a large proportion of S1 proximal tubular epithelial cells expresses thymus antigen 1 (THY1), a mesenchymal stromal and stem cell marker that regulates differentiation. Kidney disease is associated with loss of THY1 and transition toward expression of prominin 1 (PROM1), another stem cell marker recently linked to failed repair. Our data support a model in which the interplay between THY1 and PROM1 expression in proximal tubules associates with their regenerative potential and marks the timeline of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1923d134d598f27cb8990ba3881ccadfe262fa12" target='_blank'>
              Integration of spatial protein imaging and transcriptomics in the human kidney tracks the regenerative potential of proximal tubules
              </a>
            </td>
          <td>
            Mahla Asghari, Angela R. Sabo, Daria Barwinska, Ricardo Melo Ferreira, M. Ferkowicz, William S. Bowen, Ying-Hua Cheng, D. Gisch, Connor J. Gulbronson, Carrie L. Phillips, Katherine J. Kelly, Timothy A Sutton, James C Williams, Miguel A Vazquez, John O’Toole, Paul M. Palevsky, Sylvia E. Rosas, S. Waikar, Krzysztof Kiryluk, C. Parikh, Jeffrey B. Hodgin, Pinaki Sarder, Ian H. de Boer, Jonathan Himmelfarb, Matthias Kretzler, Sanjay Jain, Michael T. Eadon, S. Winfree, Tarek M. El-Achkar, P. Dagher
          </td>
          <td>2025-08-15</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Background: Lung adenocarcinoma (LUAD) remains a leading cause of cancer-related mortality despite advances in treatments, necessitating more effective therapeutic strategies. Single-cell RNA sequencing (scRNA-seq) technology has revolutionized our ability to dissect the cellular complexity of cancers, which is often obscured in conventional bulk transcriptomic experiments. Methods: In this study, we performed an integrative analysis of scRNA-seq data from multiple LUAD patient cohorts to investigate cell-type-specific transcriptomic changes across disease stages. Clustering, lineage trajectory analysis, and transcriptional regulatory network reconstruction were employed to identify stage-specific gene markers and their upstream regulators. Additionally, we constructed intercellular communication networks to evaluate signaling changes within the tumor microenvironment (TME) during LUAD progression. Results: Our analysis revealed that epithelial cells from stage IV tumors exhibited a distinct transcriptional profile compared to earlier stages, a separation not observed in immune or stromal cell populations. We identified a panel of gene markers that differentiated epithelial cells across disease stages and effectively stratified patients into subgroups with distinct survival outcomes and TME compositions. Regulatory network analysis uncovered key transcription factors, including ATF3, ATF4, HSF1, KLF4, and NFIC, as potential upstream regulators of these stage-specific genes. Moreover, cell–cell communication analysis revealed a significant increase in signaling originating from epithelial cells and a concomitant decrease in immune-derived signals in late-stage LUAD. We identified several signaling pathways enriched in stage-specific crosstalk, including Wnt, PTN, and PDGF pathways, which may play critical roles in LUAD progression. Conclusions: This study provides a comprehensive single-cell resolution map of LUAD progression, highlighting epithelial-driven regulatory programs and dynamic intercellular communication within the TME. Our findings uncover novel molecular markers and regulatory mechanisms with potential prognostic and therapeutic value for more precise treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ecbc245a8fbf12aaba1f14070430ba2f845cb23" target='_blank'>
              Single-Cell Transcriptomic Analysis Unveils Key Regulators and Signaling Pathways in Lung Adenocarcinoma Progression
              </a>
            </td>
          <td>
            Jialu Ma, Caleb McQuay, John Talburt, Amit K. Tiwari, Mary Qu Yang
          </td>
          <td>2025-06-30</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Lymph node metastasis markedly worsens prognosis in lung adenocarcinoma (LUAD); however, the evolutionary dynamics and regulatory mechanisms underlying the heterogeneity of malignant epithelial cells during this process remain poorly understood and warrant comprehensive investigation. Methods We performed a comprehensive single-cell transcriptomic analysis of epithelial cells from 18 samples comprising normal lung tissue and lymph node metastases. Malignant epithelial cells were identified via inferred copy number variation (CNV) profiles. Key malignant subpopulations were further characterized through trajectory inference, cell–cell communication mapping, gene set variation analysis (GSVA), and reconstruction of transcription factor regulatory networks. To assess clinical relevance, we developed and validated a prognostic model—termed the EAS score—based on the transcriptional signatures of malignant epithelial subsets, using integrated data from multiple TCGA and GEO cohorts. The functional role of the hub gene SELENBP1 was experimentally validated through quantitative PCR (qPCR), Western blotting, immunohistochemistry (IHC), Transwell migration assays, colony formation assays, flow cytometry, ROS quantification, and subcutaneous tumorigenesis assays in vivo. Results Single-cell transcriptomic analysis identified four distinct malignant epithelial subtypes (Clusters 0–3), each characterized by unique patterns of CNV. Leveraging these defined cellular subpopulations, we constructed a highly accurate model for prognostication in LUAD, enabling reliable classification of patients based on clinical outcomes. Through detailed comparisons between groups with divergent prognostic risks, the study revealed notable differences across the tumor microenvironment (TME), including alterations in pathway activity, gene enrichment distributions, mutation profiles, and anticipated responses to immune checkpoint blockade. In addition, functional validation experiments confirmed that SELENBP1 plays a tumor-suppressive role, further supporting its relevance as a potential intervention target in LUAD. Conclusion This research provides insights into the evolutionary complexity and heterogeneity of malignant epithelial populations in lymph node metastatic sites of LUAD. It also presents a scoring system based on prognostic indicators, which serves as a reliable tool for forecasting patient survival outcomes. Moreover, the discovery of SELENBP1 as a candidate tumor suppressor emphasizes its importance in guiding both clinical risk categorization and the design of personalized treatment strategies for individuals classified as high-risk LUAD cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6a2796c042204c38d000db3a271f2db16f9cb8" target='_blank'>
              Single-cell transcriptomic profiling reveals the heterogeneity of epithelial cells in lung adenocarcinoma lymph node metastasis and develops a prognostic signature
              </a>
            </td>
          <td>
            Qiuqiao Mu, Han Zhang, Ying Shi, Mengli Xue, Jingxian Wang, Yun Ding, Lin Tan, Hui Yuan, Xin Li, Daqiang Sun
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cutaneous malignant melanoma is an extraordinarily aggressive and heterogeneous cancer that contains a small subpopulation of tumor stem cells (CSCs) responsible for tumor initiation, metastasis, and recurrence. Identification and characterization of CSCs in melanoma is challenging due to tumor heterogeneity and the lack of specific markers (CD271, ABCB5, ALDH, Nanog) and the ability of cells to dynamically change their phenotype. Phenotype-maintaining signaling pathways (Wnt/β-catenin, Notch, Hedgehog, HIF-1) promote self-renewal, treatment resistance, and epithelial–mesenchymal transitions. Tumor plasticity reflects the ability of differentiated cells to acquire stem-like traits and phenotypic flexibility under stress conditions. The interaction of CSCs with the tumor microenvironment accelerates disease progression: they induce the formation of cancer-associated fibroblasts (CAFs) and neo-angiogenesis, extracellular matrix remodeling, and recruitment of immunosuppressive cells, facilitating immune evasion. Emerging therapeutic strategies include immunotherapy (immune checkpoint inhibitors), epigenetic inhibitors, and nanotechnologies (targeted nanoparticles) for delivery of chemotherapeutic agents. Understanding the role of CSCs and tumor plasticity paves the way for more effective innovative therapies against melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624525e56a22d9cf1d979cf203e09ae4124349ea" target='_blank'>
              Cancer Stem Cells in Melanoma: Drivers of Tumor Plasticity and Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Adriana Sabău, A. Tinca, Raluca Niculescu, I. Cocuz, Andreea Raluca Cozac-Szöke, B. Lazar, D. Chiorean, C. Budin, O. S. Cotoi
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dce7c8d7d28d1cfa3f77ec4f74044ee737c8a1da" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment in archived FFPE samples unveils novel paracrine interactions.
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlsson, A. Claeys, E. Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2025-08-08</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e652ee8a36b3b3637d56b8e5667a1abc73cc2445" target='_blank'>
              Differential Assembly of Mouse and Human Tumor Microenvironments
              </a>
            </td>
          <td>
            T. Courau, Rebecca G. Jaszczak, B. Samad, Emily Flynn, Nayvin W. Chew, Gabriella C. Reeder, Jessica Tsui, A. Ray, Harrison Wismer, Daniel Bunis, Leonard C. Lupin-Jimenez, Noah V. Gavil, David Masopust, John P Graham, Daniel A. Skelly, Xavier Vesco, Edison T. Liu, G. Fragiadakis, A. Combes, Matthew F. Krummel
          </td>
          <td>2025-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 The tumor microenvironment (TME) actively contribute to tumor development and treatment response. The interplay between tumor cells, immune cells, fibroblasts and blood vessels contribute to immune escape and drug resistance. Prior to treatment, higher tumor infiltrating lymphocytes correlate with better survival, while greater stromal content is linked to poor survival. Studying the composition and dynamics of the TME is essential for improving patient stratification, however a scalable tool for addressing this question is still lacking. Spatially resolved omics technologies allow for charting tissue architecture at the individual cell level, though large-scale studies remain challenging due to high expenses. In contrast, hematoxylin and eosin (H&E) slides are a cost-effective modality that provide rich morphological information for studying spatial biology. However, their use relies on pathologist interpretation. A key area of research in digital pathology has been automating cell (type) identification, predicting nuclei location and cell type in H&E slides. Existing deep learning models are limited by the quantity and diversity of training data, which requires pathologists to carefully annotate the location and identity of large volume of cells. To date, the largest dataset comprises approximately 200,000 cells, annotated with four cell types across 19 cancer types. We propose a novel approach of automated “molecular annotation”, where cell types and location on H&E slides are annotated with the aid of spatial proteomics, in place of pathologist annotation. Specifically, samples were profiled with both modalities. From the spatial omics modality, pixels were first segmented into cells, followed by cell clustering and cluster annotation based on molecular features of the cells. The location and identity of all cells on the tissue were then identified. These information were subsequently transferred to the H&E image by alignment at single-cell resolution, forming a dataset annotated with molecular ground truth. With a spatial omics dataset of two spatial proteomics slides from colorectal cancer patients, we obtained 160,000 annotated cells including 50,000 immune cells, 25,000 tumor cells, 12,600 stroma, 7,900 endothelial cells, among others. The size of this dataset is close to the largest annotated dataset publicly available to date. We then use this dataset to benchmark existing state-of-the-art deep-learning based cell type predictions models, as well as to fine-tune existing models for predicting cells in the colorectal tumor microenvironment. This proof-of-concept study aims to demonstrate the feasibility of molecular annotation approach. By including more spatial omics data, this approach can boost the performance of existing pre-trained models and enhance generalizability to specific tumor types. It opens up the opportunity to harness millions of cells for deep learning models to predict cell types on H&E slides, make AI models a cost-effective option for studying the TME.



 Siao-Han Wong, Benedikt Brors, Sonja Loges. Empowering AI-driven prediction of the tumor microenvironment from histopathology images via molecular annotation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B053.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62b30dcab5fc423419dfc7696f7849978493fc18" target='_blank'>
              Abstract B053: Empowering AI-driven prediction of the tumor microenvironment from histopathology images via molecular annotation
              </a>
            </td>
          <td>
            Siao-Han Wong, B. Brors, Sonja Loges
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Given the graft-versus-leukemia effect observed with allogeneic hematopoietic stem cell transplantation in refractory or relapsed acute myeloid leukemia (AML), immunotherapies have been explored in nontransplant settings. We applied a multiomic approach to examine bone marrow interactions in patients with AML treated with pembrolizumab and decitabine. Using extensively trained nuclear and membrane segmentation models, we achieved precise transcript assignment and deep learning–based image analysis. To address read-depth limitations, we integrated single-cell RNA sequencing with single-cell spatial transcriptomics from the same sample. Quantifying cell-cell distances at the edge level enabled more accurate tumor microenvironment analysis, revealing global and local immune cell enrichment near leukemia cells postpembrolizumab treatment, potentially linked to clinical response. Furthermore, ligand-receptor analysis indicated potential alterations in specific signaling pathways between leukemia and immune cells following immunotherapy treatment. These findings provide insights into immune interactions in AML and may inform therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0066efa7bc3f52352d23a57bb535219a1adb0cf1" target='_blank'>
              Single-cell spatial transcriptomics reveals immunotherapy-driven bone marrow niche remodeling in AML
              </a>
            </td>
          <td>
            Gege Gui, Molly A. Bingham, Julius R Herzog, Abigail Wong-Rolle, Laura W. Dillon, Meghali Goswami, Eddie Martin, Jason Reeves, Sean Kim, Arya Bahrami, Hermann F Degenhardt, George Zaki, P. Divakar, E. Schrom, Katherine R Calvo, Christopher S. Hourigan, Kasper D Hansen, Chen Zhao
          </td>
          <td>2025-07-09</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the third most common cancer in the world. Within the tumor microenvironment (TME), cancer cells are surrounded by diverse cell types including cancer-associated fibroblasts (CAFs) and immune cells. These neighboring cells can have a profound effect on prognosis and disease progression by providing local signals that impact tumor growth and metastasis. Prior work in the lab has identified via single cell RNA sequencing novel subtypes of cancer epithelial cells and enrichment of specific CAFs within the CRC TME. However, the locations and signaling interactions of these cells within the tumors remain poorly understood due to the loss of spatial information. In this study, we adopt a multi-facet approach to try to understand the CRC TME by utilizing a combination of single cell RNA sequencing and spatial omics technology. We first comprehensively characterized all the major cell types present in the CRC TME using single cell RNA sequencing, and with the knowledge gained, we set out to investigate the spatial localization of these cell types of interests and their interactions within the tumor architecture using the 10x Xenium Analyzer and merFISH. From our data, we identified subtypes of tumor epithelial cells that are uniquely enriched in CRC patients and observed distinct spatial organization of these cells within colonic glands across patient subgroups. In addition, we observed an enrichment of CAFs and immune cell populations at the tumor invasive margin, often accompanying regions of tumor budding. Further study of these tumor epithelial, CAF and immune subtypes and their spatial interactions may reveal new insights into CRC tumor biology and assist in the development of targeted therapies for CRC.



 Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, Nirmala Arul Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar. Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2896c139d399fbdf03bf5e11753408c9bfebdc83" target='_blank'>
              Abstract P19: Profiling of Colorectal Cancer Tumor Microenvironment using Spatial Multi-Omics
              </a>
            </td>
          <td>
            Bo Li, Grace Hui Ting Yeo, Ignasius Joanito, Vairavan Lakshmanan, Yurike Laurensia, Run Bing Han, N. A. Rayan, Kok Hao Chen, Kiat Hon Lim, Iain Bee Huat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Individual cells within a given population exhibit striking variability in viral susceptibility, but it remains unknown whether this heterogeneity reflects memories encoded into the cellular lineage or true probabilistic variability. We used multi-color lineage tracing in a human primary organotypic skin model to reveal that viral resistance is encoded within specific cellular lineages. These lineages create distinct boundaries that block viral spread. Our lineage analyses in vitro confirmed that viral susceptibility exhibits strong heritability across cell generations, with siblings and cousins displaying remarkably similar infection outcomes. ATAC and proteomics profiling of resistant and susceptible clones revealed distinct epigenomic and proteomic states, with the transcription factor AP-1 emerging as a potential central regulator of lineage-encoded viral resistance. Inducing AP-1 activity with PMA rendered cells resistant to viral infection, suggesting a causative role in mediating resistance memory. Our findings demonstrate that antiviral resistance in human skin cells is encoded within cellular lineages and preserved through cell divisions, revealing how cell memory may shape infection dynamics and viral containment in tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb2bae8a5c0fba39a07b5a8201b6d06e217381d" target='_blank'>
              Lineage memory shapes viral resistance barriers in human skin
              </a>
            </td>
          <td>
            Laura C. Van Eyndhoven, Grant Kinsler, Jingchao Zhang, Aoife O’Farrell, Rim Abderrahim, Apurv Srivastav, Catherine G. Triandafillou, Todd M. Greco, Kenneth S. Zaret, I. Cristea, Megan H. Orzalli, N. Drayman, Abhyudai Singh, Arjun Raj
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Introduction Spatial transcriptomic analysis has proposed valuable insights into the behavior of tongue cancer. However, the specific cell types involved in chemically-induced carcinogenesis and the process of tumor development remain elusive. Methods We leveraged artificial intelligence (AI) algorithms and spatial transcriptomic sequencing to meticulously characterize the spatial and temporal evolution of 4-nitroquinoline-1-oxide (4NQO)-induced tongue carcinogenesis and intratumor heterogeneity. Results An AI classifier effectively categorized dysplastic tongue tissue into 13 distinct groups. Spatial transcriptomics identified 13 corresponding cellular subgroups with unique features within the lesion. Both methods successfully distinguished subtle muscle phenotype and genetic lineage variations induced by 4NQO, despite limited morphological differences. Evolutionary tree analysis revealed the dynamic appearance and disappearance of functionally and genetically diverse cell subgroups during the progression from epithelial dysplasia to in situ carcinoma and invasive cancer. Key findings include the identification of specific switch genes associated with tumor invasion and the revelation of significant intratumor heterogeneity. Discussion This spatial transcriptomic analysis of 4NQO-induced tongue cancer provides a detailed characterization of tumor evolution and heterogeneity. It elucidates critical aspects of tongue cancer cell behavior and identifies potential therapeutic targets (switch genes). These findings offer novel insights for improving the diagnosis and treatment of tongue cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679c71fb0ae1356ab18cbbcb3f0859a89370b05e" target='_blank'>
              Spatial transcriptomic analysis of 4NQO-induced tongue cancer revealed cellular lineage diversity and evolutionary trajectory
              </a>
            </td>
          <td>
            Feng Liu, Xiaojun Wu, Shaoqing Yu, Deqiang Cheng, Jun Pan, Xiaodong Wang, Guanzhen Yu, Chaofu Li
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Tumor organoids have emerged as transformative tools in cancer research, enabling the study of tumor biology and immunology in a physiologically relevant, three-dimensional in vitro environment. Derived from patient tumor samples, these self-organizing structures recapitulate the histological and genetic heterogeneity of tumors and their microenvironment, offering significant advantages over traditional two-dimensional cell cultures and animal models. This work provides a comprehensive overview of tumor organoid generation, their characteristics, and their use as models to study tumor-immune interactions. We discuss how tumor organoids faithfully recapitulate tumor heterogeneity, support immune cell infiltration, and simulate immunosuppressive environments, making them ideal platforms for investigating immunotherapy strategies. Emerging technologies, including advanced imaging and single-cell analysis, as well as gene editing tools, further enhance the utility of tumor organoids in dissecting immune-tumor interactions at unprecedented resolution. We also highlight the translational potential of tumor organoids in preclinical immunotherapy research. Organoids offer a promising approach for predicting patient response to immunotherapy and developing personalized treatment strategies. As tumor organoid technology continues to evolve, its application in clinical settings holds great promise for advancing cancer immunotherapy, improving patient outcomes, and overcoming the challenges of drug resistance. Finally, the future direction of tumor organoid development is speculated according to current challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ddc027c2d341c33035f5a68f5849e1c092dda5" target='_blank'>
              Tumor organoids in immunotherapy: from disease modeling to translational research
              </a>
            </td>
          <td>
            Qi Si, Shuhan Tao, Jizhun Wu, Jing Ma, Zhengxing Li, Xiaohang Feng, Jianping Song, Tiantian Kong, Zhuolong Zhou
          </td>
          <td>2025-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d59997cf6c8e6c97ba758d271ce928cfa49100ba" target='_blank'>
              Transcriptomic Profiles from Normal and Tumor Tissue Samples Reveal Distinct Venule Populations and Novel Tumor Endothelial Cell Markers in Breast Cancer
              </a>
            </td>
          <td>
            K. Phoenix, Vijender Singh, Patrick A. Murphy, Kevin P. Claffey
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Integrating spatial transcriptomic data with immunofluorescence image data is challenging using existing tools due to their differences in spatial resolution. Immunofluorescence provides information about protein expression at the cellular or subcellular level, whereas spatial transcriptomic platforms typically rely on multicellular “spots” for RNA profiling. Our study coupled spatial transcriptomics of irradiated glioblastoma tissues with immunofluorescence for γH2AX, a marker of DNA damage within the nuclei of cells. We then compared gene expression in γH2AX-positive and negative regions within the tissue. There was significant interobserver variability in manual annotation of γH2AX positivity in multicellular spots by three different researchers (Kappa statistic = 0.345), despite all of them being familiar with γH2AX immunofluorescence and having predefined imaging parameters for annotation. This variability led to different researchers nominating different genes as being associated with DNA repair. To overcome this problem, we have developed a new tool using MATLAB. This tool performs “spot”-wise image analysis and uses researcher-defined parameters such as immunofluorescent marker intensity threshold and number of positive cells to annotate the “spots” as γH2AX positive or negative. The tissue with the most variability in manual annotation was annotated reproducibly by our MATLAB tool, leading to reproducible downstream analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d36da1cf7d69ccada17068ecf953f32e30ee56bd" target='_blank'>
              A New Tool to Decrease Interobserver Variability in Biomarker Annotation in Solid Tumor Tissue for Spatial Transcriptomic Analysis
              </a>
            </td>
          <td>
            Sravya Palavalasa, Emily Baker, Jack Freeman, Aditri Gokul, Weihua Zhou, Dafydd Thomas, W. Al-Holou, Meredith A. Morgan, Theodore S. Lawrence, Daniel R Wahl
          </td>
          <td>2025-07-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Vestibular schwannoma (VS) is a benign tumor that can result in significant neurological and otological complications. The mechanisms underlying its development and spatial heterogeneity remain poorly understood. In this study, we analyzed single-cell RNA sequencing (scRNA-seq) data from three previously published vestibular schwannomas (VS1-3), along with spatial transcriptomics data from two additional specimens (VS_S1-2). Our results identified a VEGFA-enriched Schwann cell (SC) subtype in scRNA-seq data, which was validated by spatial transcriptomics. This subtype also exhibited a significant positive correlation with NOVhi SCs expression. These cells were centrally localized within tumor tissue. Furthermore, spatial analysis provided new insights into SC-stromal cell interactions, and we constructed a preliminary cellular atlas of VS tissues, enhancing our understanding of tumor growth dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7a7b00297ec42d2e5217a894e786882ec23b539" target='_blank'>
              Integrating single-cell and spatial transcriptomics reveals the cellular heterogeneity of vestibular schwannoma
              </a>
            </td>
          <td>
            Wenqi Dong, Yuchen Jin, Lingkang Dong, Yumeng Jiang, Zhuangzhuang Li, Maoxiang Xu, Jingjing Wang, Feng Liu, Dongzhen Yu
          </td>
          <td>2025-07-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Lung adenocarcinoma (LUAD) exhibits significant cellular heterogeneity, yet the precise interactions between epithelial and stromal cells remain unclear. This study integrates single-cell and spatial transcriptomics to delineate tumor microenvironment dynamics, aiming to uncover key cellular subpopulations and their roles in LUAD progression. Methods We analyzed single-cell RNA sequencing (scRNA-seq) data from 21 LUAD patients and performed spatial transcriptomic deconvolution. Epithelial and fibroblast subpopulations were identified using Seurat and Harmony. Cell-cell communication was inferred via CellChat, while metabolic interactions were assessed using MEBOCOST. Copy number variation (CNV) analysis distinguished malignant cells, and trajectory inference mapped differentiation states. Spatial colocalization was examined via CellTrek. Prognostic signatures were derived from Cox regression, and a six-gene MCI score was validated using survival analysis. Results We identified eight epithelial (e.g., MUC21 + Epi, ASCL1 + Epi) and nine fibroblast subpopulations (e.g., Fb_IGFBP4, Fb_COL11A1), with tumor-enriched subsets showing elevated CNVs and metabolic crosstalk. Fb_IGFBP4 correlated with poor prognosis, while MUC21 + Epi exhibited amplified COL1A1/SDC4-mediated interactions with fibroblasts. Pathway analysis highlighted tumor-specific MK and collagen signaling between fibroblasts and epithelial cells, suggesting stromal-epithelial synergy drives progression. Spatial analysis revealed colocalization of epithelial and fibroblast subclusters in tumors, contrasting with normal tissue. The MCI score, derived from six genes (e.g., ADAM10, MARVELD1), independently predicted survival and stratified high-risk patients (AUC > 0.6). Conclusion This study identifies key stromal-epithelial subset interactions in LUAD, proposing prognostic biomarkers and therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1007/s00432-025-06250-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e782303b020428f38f4b87de805bc2e91d348a09" target='_blank'>
              Decoding epithelial–fibroblast interactions in lung adenocarcinoma through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Jiajin Yang, Qiuping Xu, Yanjun Lu
          </td>
          <td>2025-07-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatoblastoma is the most common pediatric liver cancer, with the fastest rising incidence among childhood malignancies. Early genomic studies revealed that hepatoblastoma has the lowest mutational burden of any human cancer, however, recent advances in single-cell RNA-seq, multiomics, spatial transcriptomics, and functional genomics screenings have revealed substantial complexity. Diverse cellular subpopulations, divergent WNT signaling, key developmental pathways, and intricate interactions between the tumor cells and tumor immune microenvironment (TME) collectively shape tumor heterogeneity, disease progression, therapeutic responses, and genetic dependencies. Mosaic embryonic loss of heterozygosity (LOH) at chromosome 11p15.5 may be a hepatoblastoma-initiating event, as clonal expansion of 11p15.5 LOH occurs in adjacent normal liver tissue. A cholangiocyte-like subpopulation expresses FGF19, in a SOX4-dependent, paracrine manner, to drive the proliferation of neighboring embryonal hepatoblastoma cells. WNT-signaling dependent MDK promotes the immunosuppressive TME, which impairs immune cell infiltration. The TME may also be driven by islands of erythroblasts, which influence treatment resistance. Plasticity driven by changes in chromatin accessibility enables differentiation transition between hepatocytic and liver progenitor cell types, which is associated with treatment resistance. Here, we review recent findings in pediatric hepatoblastoma cells, tumor-associated cell types, and genetic dependencies that will serve to advance hepatoblastoma therapy. Clinical trial number: Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02405-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361b9ebd3fa588c3f28e25ba36c91c45f4b07fb6" target='_blank'>
              From preneoplastic lesion to heterogenous tumor: recent insights into hepatoblastoma biology and therapeutic opportunities
              </a>
            </td>
          <td>
            Jun Yang, A. Davidoff, Andrew J. Murphy
          </td>
          <td>2025-07-19</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The tumor microenvironment (TME) plays a pivotal role in shaping therapeutic outcomes in lung cancer, with cancer-associated fibroblasts (CAFs) emerging as key mediators of drug resistance. In situ sequencing enables spatial resolution of tumor architecture, offering insight into heterogeneity and the influence of CAFs on tumor cell biology. However, directly translating these microenvironmental effects on tumor cell behavior into drug response predictions remains a major challenge. In this work, we leveraged spatial transcriptomics and in situ single-cell RNA sequencing to predict spatially resolved drug responses in non-small cell lung cancer (NSCLC) patient samples and investigated the impact of fibroblast density on therapeutic efficacy. We adapted our recently published scIDUC modeling approach—which integrates bulk RNA-seq from cancer cell lines with single-cell RNA-seq data to predict drug responses—for use with in situ data from eight NSCLC patients. Our spatial analysis revealed that tumor cells in high fibroblast-density regions exhibited distinct predicted drug sensitivities compared to those in low-fibroblast contexts. Notably, lapatinib—a dual EGFR/HER2 inhibitor—was consistently predicted to be less effective in tumor cells residing within fibroblast-rich niches. These predictions were validated experimentally using a co-culture model of CALU-3 lung cancer cells and IMR-90 fibroblasts. CALU-3 cells grown with fibroblasts, either in direct contact or separated by a transwell, showed significantly increased resistance to lapatinib compared to monoculture conditions, confirming our computational findings. Our study highlights the role of CAF-rich microenvironments in therapeutic response and proposes a CAF-informed framework for refining therapeutic strategies in NSCLC. By integrating spatial transcriptomics with predictive modeling, we provide a pathway to directly link microenvironmental context with functional drug response in tumors.



 Robert F. Gruener, Lilin Wang, Adam Lee, Weijie Zhang, R. Stephanie Huang. Spatially resolved drug response predictions reveal fibroblast-associated resistances in in situ lung cancer samples [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd4918af0e969655d588afdfb7b613e252a4d08" target='_blank'>
              Abstract A023: Spatially resolved drug response predictions reveal fibroblast-associated resistances in in situ lung cancer samples
              </a>
            </td>
          <td>
            Robert F. Gruener, Lilin Wang, Adam Lee, Weijie Zhang, R. S. Huang
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) remains a lethal malignancy. Although immunochemotherapy regimens have improved patient survival rates, drug resistance still occurs in a significant subset of patients, highlighting the importance of elucidating the mechanisms within the tumor microenvironment. Here, we applied spatial single-cell transcriptomics to investigate the spatial characteristics of SCLC and their associations with immunochemotherapy resistance. By analyzing samples from 18 patients with extensive-stage SCLC, we identified two distinct epithelial cell subtypes: Epi-I and Epi-II. Epi-I exhibited high proliferative activity and was associated with treatment resistance and poor survival outcomes. In contrast, Epi-II showed more spatial contact with immune cells and was associated with treatment sensitivity. Further analysis uncovered a fascinating cellular transition paradigm, wherein Epi-I may be derived from Epi-II, with myeloid cells playing a facilitatory role in this transformation cascade. Specifically, within the spatial zone that was enriched with the Epi-II, epithelial cells may secrete MIF gene, which promoted the polarization of myeloid cells towards the M2 macrophages. The M2-polarized myeloid cells subsequently upregulated the expression of SPP1 that in turn triggered the activation of the PI3K-AKT signaling pathway in the adjacent epithelial cells, driving the conversion of Epi-II to Epi-I cells. Our findings revealed that the intricate crosstalk between epithelial and myeloid cells constitutes a pivotal resistance mechanism in SCLC, and targeting the SPP1/MIF pathway emerged as a promising strategy with the potential to enhance the treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0302ec6123f8cd6d62c8c99e2e99dcc7087718b" target='_blank'>
              Spatial transcriptomics reveals macrophage domestication by epithelial cells promotes immunotherapy resistance in small cell lung cancer
              </a>
            </td>
          <td>
            Yu Sun, Minghui Zhang, Yanbin Zhao, Yubo Yan, Lei Wang, Xuhui Liu, Songsong Xia, Bingbing Wang, Xiaoxin Zhang, Yan Wang
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background Neoadjuvant immune checkpoint blockade (nICB) has revolutionized cancer treatment, yet the underlying mechanisms of resistance in bladder cancer remain to be explored. Methods We conducted single-cell RNA sequencing (scRNA-seq) on peripheral blood mononuclear cells, tumor tissues, adjacent normal tissues, and metastatic lymph nodes from 2 nICB-naïve and 10 nICB-treated patients with bladder cancer (5 responders and 5 non-responders). Spatial RNA sequencing was performed on tumor slides from two responders and four non-responders. Findings were validated by multiplex immunohistochemistry, mice orthotopic bladder cancer model, and flow cytometry assays. Results nICB remodeled the tumor microenvironment of bladder cancer from both single-cell and spatial perspectives. scRNA-seq analysis revealed a significant increase in MYBL2hi cancer stem cells (CSCs) among non-responders. Analysis of the myeloid population showed that SPP1+ macrophages associated with angiogenesis were linked to CD8+ T cell exclusion. Further investigation into cell–cell communication revealed a propensity for bidirectional crosstalk between MYBL2hi CSCs and SPP1+ macrophages in non-responders. MYBL2hi CSCs derived CCL15, which bound to CCR1 and induced SPP1 upregulation in macrophages which reciprocally enhanced bladder cancer stemness and resistance to nICB through the SPP1-ITGα9β1 axis. Additionally, we identified an aged CCL3+ neutrophil population that interacted with SPP1+ macrophages through a positive feedback loop, contributing to nICB resistance. Finally, in vivo studies demonstrated that combined MYBL2 knockdown and SPP1 targeting synergistically enhanced ICB efficacy in bladder cancer. Conclusions Our research reveals transcriptomic characteristics associated with distinct therapeutic responses to nICB treatment, offering a foundation for optimizing personalized neoadjuvant strategies in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f561fa737d2808a2b787b34e9d1e658c2219648a" target='_blank'>
              Malevolent alliance of MYBL2hi cancer stem cell and SPP1+ macrophage confers resistance to neoadjuvant immunotherapy in bladder cancer
              </a>
            </td>
          <td>
            Hualin Chen, Zhaoheng Jin, Yueqiang Peng, Yingjie Li, Ziyi Li, Xuebin Zhang, Yi Xie, Jie Dong, Lin Ma, Zhigang Ji
          </td>
          <td>2025-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Accurately predicting which patients will respond to immune checkpoint blockade (ICB) remains a major challenge. Here, we present TIME_ACT, an unsupervised 66-gene transcriptomic signature of tumor immune activation derived from TCGA melanoma data. First, TIME_ACT scores accurately identify tumors with activated immune microenvironments across cancer types. Analysis of spatial features of the tumor microenvironment revealed that TIME_ACT-high regions exhibit dense lymphocyte infiltration near tumor cells, indicating localized immune activation. Second, in 15 anti-PD1 transcriptomic cohorts spanning six cancer types, TIME_ACT outperforms 22 established signatures and methods, achieving a mean AUC of 0.76 and a clinically meaningful mean odds ratio of 6.11. Thirdly, TIME_ACT scores can be accurately inferred from tumor histopathology slides. Finally, slide-inferred TIME_ACT scores predict ICB response across eight unseen cohorts, achieving a mean AUC of 0.72 and a mean odds ratio of 5.02. These findings establish TIME_ACT as a robust, pan-cancer, and low-cost predictor of ICB response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f083d96ace3eccacc20f8f51f00bb48faf2731" target='_blank'>
              Pan-cancer prediction of tumor immune activation and response to immune checkpoint blockade from tumor transcriptomics and histopathology
              </a>
            </td>
          <td>
            S. Mukherjee, Sumeet Patiyal, L.R. Pal, Tiangen Chang, Sumona Biswas, S. R. Dhruba, Amos Stemmer, Arashdeep Singh, Abbas Yousefi-Rad, Tien-Hua Chen, Binbin Wang, Denis Marino, Wonwoo Shon, Yuan Yuan, Mark Faries, Omid Hamid, Karen L. Reckamp, B. Waissengrin, Beatriz Ornelas, P. Chu, Salah Boudjadi, L. Ley, Dilara Akbulut, N. El Ahmar, S. Signoretti, D. Braun, Hyunjeong Joo, Hyungsoo Kim, A. Osipov, R. Figlin, J. Bar, Iris Barshack, Chi-Ping Day, Karine Sargsyan, A. Apolo, Kenneth D Aldape, Muh-Hwa Yang, Michael B. Atkins, Ze’ev A. Ronai, Danh-Tai Hoang, E. Ruppin
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Abstract Spatial transcriptomics (ST) has emerged as an efficient technology for mapping gene expression within tissue sections, offering informative spatial context for gene activities. However, most current ST techniques suffer from low spatial resolution, where each spatial location often contains cells of various types. Deconvolution methods are used to resolve the cell mixture within the spots, but conventional approaches rely on spot-by-spot analyses, which are limited by low gene expression levels and disregard spatial relationships between spots, ultimately reducing performance. Here, we introduce DECLUST, a cluster-based deconvolution method to accurately estimate the cell-type composition in ST data. The method identifies spatial clusters of spots using both gene expression and spatial coordinates, hence preserving the spatial structure of the tissue. Deconvolution is subsequently performed on the aggregated gene expression of individual clusters, mitigating the challenges associated with low expression levels in individual spots. We evaluate DECLUST on simulated ST datasets from a human breast cancer tissue and two real ST datasets from human ovarian cancer and mouse brain. We compare DECLUST to current methods including CARD, GraphST, Cell2location, and Tangram. The results indicate that DECLUST not only maintains the spatial integrity of tissues but also outperforms existing methods in terms of robustness and accuracy. In conclusion, DECLUST provides an effective and reliable approach for identifying cell-type compositions in ST data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a849f6608a52c27ce54815b06c38254be7c33009" target='_blank'>
              A cluster-based cell-type deconvolution of spatial transcriptomic data
              </a>
            </td>
          <td>
            Qingyue Wang, Parth Khatri, Huy Q. Dinh, Jian Huang, Yudi Pawitan, T. Vu
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The intrinsic complexity of biological processes often hides the role of dynamic microenvironmental cues in the development of pathological states. Microphysiological systems (MPSs) are emerging technological platforms that model in vitro dynamics of tissue-specific microenvironments, enabling a holistic understanding of pathophysiology. In our previous works, we engineered and used breast tumor MPS differing in matrix stiffness, pH, and fluid flow mimicking normal and tumor breast tissue. High-dimensional data using two distinctive human breast cell lines (i.e., MDA-MB-231, MCF-7), investigating cell proliferation, epithelial-to-mesenchymal transition (EMT), and breast cancer stem cell markers (B-CSC), were obtained from breast-specific microenvironments. Recognizing that the widespread adoption of MPS requires tailoring its complexity to application demands, we herein report an innovative machine-learning (ML)-based approach to analyze MPS data. This approach uses unsupervised k-means clustering and feature extraction to inform on key markers and specific microenvironments that distinguish invasive from non-invasive breast cell phenotypes. This data-driven approach streamlines future experimental design and emphasizes the translational potential of integrating MPS-derived insights with ML to refine prognostic tools and personalize therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d04f6b70ba36ca6bdecf1ece254a7da2cd8bdc7c" target='_blank'>
              Data-Driven Sustainable In Vitro Campaigns to Decipher Invasive Breast Cancer Features
              </a>
            </td>
          <td>
            Lekha Shah, V. Breschi, Annalisa Tirella
          </td>
          <td>2025-07-25</td>
          <td>ACS Biomaterials Science & Engineering</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer progression remains a significant clinical challenge. Phenotypic adaptation by tumour cells results in disease heterogeneity, which drives treatment resistance and immune escape. T-cell immunotherapy, while effective at treating some cancer subtypes, can also fail due to limits on tumour immunogenicity or T-cell recognition. For example, one potential contributor to immune escape involves the density and alignment of the extracellular matrix (ECM) surrounding tumours, also known as tumour-associated collagen signature (TACS). However, the specific mechanisms by which aligned fibres contribute to decreased patient survival rates have not yet been decoupled. Here, we developed EVO-ACT (EVOlutionary agent-based cancer T-cell interaction), a two-dimensional agent-based modelling framework designed to investigate how different TACS architectures impact tumour evolution and dynamic interactions with CD8 ⁣+ T cells. Our results highlight that TACS-driven modulation of T-cell dynamics, combined with phenotypic adaptation, such as epithelial-to-mesenchymal transition, underlies differences in tumour immunogenicity and the application of our model can successfully recapitulate clinically observed breast cancer survival trends.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195f55d30b50b8b9b55b6c384b5267c2b8395881" target='_blank'>
              Distinct evolutionary patterns of tumour-immune escape and elimination determined by extracellular matrix architectures
              </a>
            </td>
          <td>
            Yijia Fan, Jason T. George
          </td>
          <td>2025-07-01</td>
          <td>Journal of the Royal Society Interface</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Neoadjuvant therapy (NAT) has transformed cancer treatment by improving surgical outcomes and survival rates, yet resistance mechanisms across multiple cancer types remain unclear. This study aimed to decipher tumor ecosystem dynamics during NAT using cross-cancer single-cell sequencing data, focusing on identifying key mediators of immunosuppression and treatment resistance. Methods Single-cell RNA-sequencing (scRNA-seq) datasets from five solid tumors (esophageal squamous cell carcinoma, esophagogastric junction carcinoma, colorectal cancer, cervical cancer, and triple-negative breast cancer) were integrated. The data from these five cancer types underwent a rigorous process to standardize cell types across all datasets. Cell-cell communication analysis, Meta-Programs (MPs) via non-negative matrix factorization, and functional enrichment were performed. Immunohistochemistry (IHC) and Western blot validated S100A4 expression and PD-L1 induction in vitro. Results We constructed a single-cell map across cancer types and systematically characterized dynamic changes in tumor cells and diverse microenvironmental cell populations following neoadjuvant therapy, along with thier gene expression and pathway alterations. Our findings highlight that crosstalk between cancer-associated fibroblasts (CAFs) and tumor cells represents a critical determinant of neoadjuvant therapy resistance. Fibroblasts underwent significant state transitions post-treatment, marked by hypoxia-associated gene upregulation (e.g., S100A4) and immunosuppressive pathways. Meta-Programs (MPs) analysis identified a hypoxia-driven fibroblast state (MP5) containing S100A4 that correlated with treatment resistance. In vitro experiments, S100A4 co-localized with α-SMA + fibroblasts and directly induced PD-L1 expression in tumor cells, linking CAFs secreted S100A4 to immunosuppressive PD-L1 upregulation. Conclusion This cross-cancer single-cell atlas reveals S100A4, secreted by CAFs, as a conserved mediator of PD-L1 upregulation in tumor cells, driving immunosuppression and resistance to nICT. The atlas and mechanistic findings provide a rationale for targeting S100A4 to enhance treatment efficacy, pending validation in larger cohorts and mechanistic studies. This resource also supports the development of personalized, cross-cancer neoadjuvant strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/875f07738539c0aaf3118d5091d4cb483e59f11d" target='_blank'>
              Deciphering the tumor ecosystem dynamics undergoing immunochemotherapy therapy across multiple cancer types unveils the immunosuppressive role of S100A4 in fibroblasts by promoting PD-L1 expression in tumor cells
              </a>
            </td>
          <td>
            Bo Yang, Ruiji Chen, Mali Zu, Jie Yao, Hong Ren, Yi‐Shung Lin, Bo Zhang, Tianjiao Ji, Yang Liu
          </td>
          <td>2025-07-23</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metabolic crosstalk among diverse cellular populations contributes to shaping a competitive and symbiotic tumor microenvironment (TME) to influence cancer progression and immune responses, highlighting vulnerabilities that can be exploited for cancer therapy. Using a spatial multiomics platform to study the cell-specific metabolic spectrum in hepatocellular carcinoma (HCC), we map the metabolic interactions between different cells in the HCC TME and identify a unique tumor-immune-cancer-associated fibroblast (CAF) "interface" zone, where cell-cell interactions are enhanced and accompanied by significant upregulation of lactic acid and long-chain polyunsaturated fatty acids. Further combining single-cell mass spectrometry imaging of patient-derived tumor organoids, cocultured CAFs, and macrophages, we demonstrate that CAFs increase glycolysis and secrete lactic acid to the surrounding microenvironment to drive immunosuppressive macrophage M2 polarization. These findings facilitate the understanding of cancer-associated metabolic interactions in complex TME and provide clues for targeted clinical therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b292d4bb0787fa6280b9910112e99765dee292d" target='_blank'>
              Integrating spatial omics and single-cell mass spectrometry imaging reveals tumor-host metabolic interplay in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Panpan Chen, Haoyuan Geng, Bangzhen Ma, Yaqi Zhang, Zihan Zhu, Min Li, Shiping Chen, Xiao Wang, Chenglong Sun
          </td>
          <td>2025-07-29</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer remains a formidable global health burden owing to its persistently high incidence and mortality, highlighting the need to elucidate its underlying biological mechanisms. Such insights are crucial for refining diagnostic precision and therapeutic efficacy. Notably, human tumors represent highly heterogeneous ecosystems, comprising a dynamic interplay between malignant cells and non-malignant components, such as immune infiltrates and stromal elements. This cellular heterogeneity constitutes a major obstacle in tumor research. Recent technological advancements have enabled the development and application of novel tools for investigating tumor heterogeneity. In this context, single-cell sequencing technologies emerged as transformative tools for dissecting tumor architecture at a cellular resolution, offering unprecedented insights into the cellular diversity and molecular underpinnings of cancer. In particular, single-cell RNA sequencing (scRNA-seq) has been widely used to analyze the tumor microenvironment, particularly its immune components, thereby providing a valuable framework for understanding tumor-immune system interactions. However, the intrinsic heterogeneity of tumor cells has received comparatively less attention, with limited studies focusing on these malignant populations. Comprehensive profiling of tumor cells has become increasingly feasible as scRNA-seq technologies continue to evolve, offering higher throughput, increased cell capture efficiency, and more advanced analytical pipelines. Despite these advancements, current applications of scRNA-seq remain primarily focused on immune and stromal cell populations. Therefore, a paradigm shift is warranted, redirecting the investigative focus toward tumor cells and gaining deeper insights into tumorigenesis and therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c3598e7911b47f7291bb4a9f0c4b83bd05a9bc" target='_blank'>
              Exploration and Application of Malignant Cell Heterogeneity Analysis with Single-Cell Transcriptome Sequencing Technology.
              </a>
            </td>
          <td>
            Wen Zhang, Tianliang Liu, Ting Liu, Yu Zhang, Zhen-guo Zhao, Xinxin Zhang, Xiaoyang Li, Jiasheng Zhong, Zhicheng Li, Shifu Chen, Libin Xu
          </td>
          <td>2025-07-25</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) exhibits substantial intertumoral heterogeneity, largely attributable to multiple tumor stem‐like cell populations, whose molecular identities and clinical significance remain incompletely defined. This study delineates tumor‐intrinsic stem‐like cell diversity and its prognostic implications through single‐cell transcriptomic profiling of 171,906 tumor epithelial cells (n = 152), integrated with bulk transcriptomic (n = 1389) and genomic (n = 1077) datasets. Functional validation was conducted via in vitro assays and multiplex immunofluorescence. A previously unrecognized lysosome‐associated transmembrane protein 4B‐positive (LAPTM4B+) stem‐like cell cluster was identified, distinct from the classical leucine‐rich repeat‐containing G‐protein coupled receptor 5‐positive (LGR5+) population. LAPTM4B+ cells exhibited MYC pathway activation and 8q chromosomal gains, with preferential enrichment in microsatellite‐stable, POLE wild‐type, and left‐sided tumors. Stratification based on LAPTM4B+/LGR5+ stem‐like cell ratios defined four CRC stem‐like subtypes (CSS), with CSS2 (LAPTM4B+‐dominant) associated with the poorest prognosis (HR = 2.31, p < 0.001). The combined expression of LAPTM4B and LGR5 demonstrated superior predictive power for CRC progression compared to either marker alone (AUC = 0.820 vs. 0.715/0.699), underscoring the synergistic influence of distinct stem‐like cell populations on patient outcomes. These findings provide novel insights into CRC heterogeneity and cooperative interactions among diverse stem‐like populations shaping disease outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/870d87b3057664fbe8a51f6ca3867c7fe15a2488" target='_blank'>
              Novel Lysosomal‐Associated Transmembrane Protein 4B‐Positive Stem‐Like Cell Subpopulation Characterizes High‐Risk Colorectal Cancer Subtypes
              </a>
            </td>
          <td>
            Yangyang Fang, Tian-Hao Fu, Ziqing Xiong, Qian Zhang, Wei Liu, Kuai Yu, Aiping Le
          </td>
          <td>2025-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34269e7ccaff0901b281158ca70f685dbde49243" target='_blank'>
              Pericytes change function depending on glioblastoma vicinity: emphasis on immune regulation.
              </a>
            </td>
          <td>
            C. Buizza, Robert Carlsson, Coralie Gamper, Gayatri Chitale, J. Bengzon, Gesine Paul
          </td>
          <td>2025-07-17</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Organoid models often fail to capture the complex niche of patient-specific tumors. In this study, we present a novel methodology for generating gastric cancer assembloids composed of matched tumor organoids and stromal cell subpopulations, which closely recapitulate the cellular heterogeneity and microenvironment of primary tumors. The inclusion of autologous stromal cell subpopulations significantly influences gene expression and drug response sensitivity. By incorporating diverse stromal cell populations derived from the same tumor tissue as the organoids, these assembloids enable a more comprehensive investigation of individual tumor biology, biomarker expression, transcriptomic profiles, and cell–cell interactions. The model also supports personalized drug screening and the optimization of combination therapies. Altogether, the integration of patient-specific stromal cell subsets enhances the physiological relevance of preclinical testing, providing insights into resistance mechanisms and ultimately contributing to the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a497a2c2ec986f0ccfd70ff27fc0f7edf51baf58" target='_blank'>
              Patient-Derived Gastric Cancer Assembloid Model Integrating Matched Tumor Organoids and Stromal Cell Subpopulations
              </a>
            </td>
          <td>
            Irit Shapira-Netanelov, Olga Furman, Dikla Rogachevsky, G. Luboshits, Yael Maizels, Dmitry Rodin, Igor Koman, Gabriela A. Rozic
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Intratumoral heterogeneity is a key determinant of immunotherapy resistance, yet current biomarkers largely rely on single-modality features—such as CD8+ T cell infiltration, IFNG expression, or checkpoint ligand staining—without accounting for the spatial complexity of the tumor microenvironment. Composite biomarkers that integrate both tumor-intrinsic programs and their spatial organization remain underdeveloped. To address this, we combined machine learning–based molecular classification with spatial transcriptomics to uncover how transcriptional tumor states are physically structured within the tumor architecture and how this organization governs response or resistance to immunotherapy.



 We leveraged non-negative matrix factorization (NMF) and tumor microenvirnoment metabolism signatures with bulk RNA-seq data from >700 melanoma tumors across multiple phase I–III clinical trials. These programs were mapped onto spatial transcriptomics data using Tangram 2.0, a deep learning model that aligns single-cell or deconvolved bulk data to spatial coordinates. We analyzed cell–cell interactions, tumor-stromal patterning, and radial gene gradients to interpret functional spatial architecture.



 Spatial mapping revealed that melanoma tumors often harbor multiple transcriptional programs arranged in discrete, non-overlapping tissue regions. Notably, undifferentiated (UR) and differentiated (DC) programs—associated with distinct therapeutic responses—co-existed within the same tumor but remained compartmentalized. Only spatial domains with DC-like features exhibited MAPK inhibitor–induced MHC-I expression and immune cell infiltration, whereas UR regions remained immune-excluded and were enriched for fibroblasts and extracellular matrix remodeling. These findings suggest that spatial segregation of tumor states, rather than overall tumor composition, may predict response to immunotherapy. Conclusions: This study presents a machine learning–enabled spatial framework that links transcriptional identity to physical tissue structure in melanoma. By uncovering how spatial architecture constrains therapeutic response, our work highlights the value of AI-driven models like Tangram for interpreting spatial data and guiding precision immuno-oncology.



 Kalpit Shah. Deep learning–guided spatial dissection of melanoma uncovers compartmentalized tumor states associated with response and resistance to immunotherapy and its combination with MAPK inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2adc1103b85dcc4735f8a205c521f8d92967a09a" target='_blank'>
              Abstract B045: Deep learning–guided spatial dissection of melanoma uncovers compartmentalized tumor states associated with response and resistance to immunotherapy and its combination with MAPK inhibitors
              </a>
            </td>
          <td>
            Kalpit Shah
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mixed neuroendocrine–nonneuroendocrine neoplasms (MiNEN) are usually highly aggressive tumors characterized by marked histological heterogeneity, most commonly represented by mixed adenocarcinoma and poorly differentiated neuroendocrine carcinoma (NEC). However, beyond morphological observations, the biological basis and implications of this heterogeneity remain incompletely understood. In this study, we combined component-specific next-generation sequencing and spatial transcriptomics to investigate three mixed adenocarcinoma-NEC cases from different anatomical sites (ileocecal, ovarian, gastric), tracing tumor progression from precursor lesions to invasive NEC. Genomic analyses revealed a shared trunk of driver mutations across all tumor compartments, confirming their clonal origin, while also uncovering additional compartment-specific alterations. Spatial transcriptomics, together with gene set enrichment analysis (GSEA), revealed distinct transcriptional profiles aligned with histologically annotated compartments (e.g., adenocarcinoma, NEC, precursor). In NECs, GSEA consistently showed downregulation of immune-related pathways and upregulation of proliferation-associated pathways compared to non-neuroendocrine tumor areas. Moreover, distinct transcriptomic subclusters were identified within morphologically homogeneous NEC regions in two of the three cases. These subclusters exhibited significant differences in immune regulation, proliferation signaling, and cell-cycle control, and were associated with divergent predicted chemotherapy-response signatures, suggesting clinically relevant implications for treatment sensitivity and resistance. In summary, our findings indicate that despite a shared clonal origin, MiNEN develop distinct genetic and transcriptomic features across tumor compartments. The inconsistent presence of transcriptomic subclusters within morphologically similar regions underscores the complexity of intratumoral heterogeneity in these aggressive neoplasms. By connecting morphological and molecular layers of tumor architecture, spatial profiling may aid in translating biological complexity into more targeted clinical strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12022-025-09869-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d940265ab2a62d1a11097a7fc51ef43cc25274" target='_blank'>
              Exploring Intratumoral Heterogeneity in Mixed Neuroendocrine-Nonneuroendocrine Neoplasms with Spatial Transcriptomics: Even More Diverse Than Anticipated
              </a>
            </td>
          <td>
            Annika Weiß, Julia Teply-Szymanski, Maxime Schmitt, S. Foersch, P. Jank, Joscha Griger, Uwe Wagner, Detlef K Bartsch, Carsten Denkert, Moritz Jesinghaus
          </td>
          <td>2025-08-20</td>
          <td>Endocrine Pathology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Globally, gastric cancer (GC) stands as the fifth most prevalent form of malignant neoplasm and represents a significant contributor to mortality associated with oncological conditions. Despite advancements in therapeutic strategies for GC, the outcomes for patients with advanced stages of the disease continue to be unfavorable, largely due to tumor heterogeneity and the challenges posed by resistance to therapeutic agents. Metabolic reprogramming is pivotal in driving the advancement of GC, contributing to the development of resistance to pharmacological treatments and facilitating the cancer’s ability to evade immune surveillance. Developing multi-target comprehensive treatment strategies by integrating tumor microenvironment (TME) modulation holds promise for significantly improving therapeutic efficacy. Methods The study analyzed GC and identified key cell subtypes by integrating data derived from single-cell RNA-sequencing (scRNA-seq) alongside spatial transcriptomics information. Cell type identification was accomplished using the tool of Seurat, and the spatial distribution of cell types was revealed through the Robust Cell Type Decomposition technique. CellChat was used to analyze the interactions between key cell subtypes and other cells, and the “StLearn” package was employed to investigate spatial cell communication in depth. Additionally, the functional role of the key molecule ELK4 was validated through in vitro experiments. Results This research utilized scRNA-seq combined with spatial transcriptomics to comprehensively analyze GC, identifying the C1 NDUFAB1+ subtype, which exhibited high proliferative activity, metabolic reprogramming capabilities, and immune evasion properties. It was found that the C1 NDUFAB1+ subtype closely interacted with fibroblasts and pericytes via the PARs signaling pathway. Additionally, in vitro experiments confirmed that knockdown of ELK4 substantially curbed tumor cell proliferation, migration, and invasion. Conclusion This study revealed the main significance of the C1 NDUFAB1+ subtype in GC, elucidating its core mechanisms in tumor progression, metabolic reprogramming, and immune evasion. ELK4 was identified as a key regulatory factor that markedly enhanced the proliferation, migratory capacity, and invasive potential of tumor cells, while changes in the TME were a driving force behind immune suppression and drug resistance. The findings underscored the importance of developing specific therapeutic targets, targeting metabolic reprogramming, and overcoming immune evasion, providing new theoretical foundations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94360dc80f5310630bd93066b812e27d59cfa9fb" target='_blank'>
              Integrative single-cell and spatial transcriptomics uncover ELK4-mediated mechanisms in NDUFAB1+ tumor cells driving gastric cancer progression, metabolic reprogramming, and immune evasion
              </a>
            </td>
          <td>
            Yuwei Sun, Wenyang Nie, Zhikai Xiahou, Xiaojing Wang, Wenjia Liu, Zongkai Liu, Zhiheng Lin, Zhaidong Liu
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Motivation Fine-grained cellular characterization provides critical insights into biological processes, including tissue development, disease progression, and treatment responses. The spatial organization of cells and the interactions among distinct cell types play a pivotal role in shaping the tumor micro-environment, driving heterogeneity, and influencing patient prognosis. While computational pathology can uncover morphological structures from tissue images, conventional methods are often restricted to identifying coarse-grained and limited cell types. In contrast, spatial transcriptomics-based approaches hold promise for pinpointing fine-grained transcriptional cell types using histology data. However, these methods tend to overlook key molecular signatures inherent in gene expression data. Results To this end, we propose a cross-modal unified representation learning framework (CUCA) for identifying fine-grained cell types from histology images. CUCA is trained on paired morphology-molecule spatial transcriptomics data, enabling it to infer fine-grained cell types solely from pathology images. Our model aims to harness the cross-modal embedding alignment paradigm to harmonize the embedding spaces of morphological and molecular modalities, bridging the gap between image patterns and molecular expression signatures. Extensive results across three datasets show that CUCA captures molecule-enhanced cross-modal representations and improves the prediction of fine-grained transcriptional cell abundances. Downstream analyses of cellular spatial architectures and intercellular co-localization reveal that CUCA provides insights into tumor biology, offering potential advancements in cancer research. Availability and implementation The source code of CUCA is available in Zenodo: 10.5281/zenodo.15087256.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaf396e0261fb92d99d805f5f25e9836867512e3" target='_blank'>
              Predicting fine-grained cell types from histology images through cross-modal learning in spatial transcriptomics
              </a>
            </td>
          <td>
            Chaoyang Yan, Zhihan Ruan, Songkang Chen, Yichen Pan, Xue Han, Yuanyu Li, Jian Liu
          </td>
          <td>2025-07-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Osteosarcoma (OS) is the most common primary malignant bone tumour of childhood, yet five-year survival has plateaued at ~60–70% for localised disease and plunges below 30% once metastasis emerges. Formerly viewed as a cell-intrinsic neoplasm entombed in mineralised bone, OS is now understood as a spatially stratified ecosystem whose immune-evasion niches choreograph progression. Three-dimensional spatial transcriptomics (3-D ST) fuses barcode-based transcript capture with volumetric reconstruction, preserving x-, y- and z-axis context and exposing concentric C1QC necrotic belts, MCAM (melanoma cell-adhesion molecule, CD146) peri-vascular corridors, hypoxic glycolytic rims and therapy-induced tertiary-lymphoid islets that collectively sequester cytotoxic lymphocytes. Pre-clinical atlases link PD-L1 high SOX9 stem-like cells, LGALS3 macrophages and VEGFA-driven endothelial tips to chemoresistance and immune-checkpoint failure, while ligand–receptor inference highlights VEGFA–VEGFR2, CXCL12–CXCR4 and complement–CSF1R axes as actionable bottlenecks. Translational efforts already echo these insights: dual MCAM/VEGFR blockade collapses vascular gates, C1s or CSF1R antagonists dismantle necrotic-core “cold pockets”, and MCT1–POSTN combinations target lactate-stiffened stromal shells. By weaving methodological advances with emergent biology, this review crystallises how 3-D ST redefines OS immunopathology, sharpens biomarker discovery and accelerates spatially guided combination therapies. We aim to expose diagnostic blind spots, spotlight niche-directed interventions and chart a roadmap toward lifting the long-standing therapeutic ceiling in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c84eb34c63a5fd31899b587fca53cec9b872be8" target='_blank'>
              Deciphering spatially confined immune evasion niches in osteosarcoma with 3-D spatial transcriptomics: a literature review
              </a>
            </td>
          <td>
            Guangqiang Qiu, Yongcheng Tang, Junhui Zuo, Heng Wu, Yongxian Wan
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Triple-negative breast cancer is a highly aggressive disease subtype with limited treatment options. Despite the approval of targeted therapies such as PARP inhibitors, metastatic progression remains inevitable. Additionally, spatial arrangements, including immune exclusion zones and immune–stromal crosstalk, can profoundly influence therapeutic response. These observations underscore the need for more effective, individualized treatments that both target malignant cells and promote anti-tumor microenvironments. Toward that goal, we have developed a coordinated experimental-computational approach designed to enable identification of effective treatment strategies that consider the functional relationships between different cell types. First, to identify adaptive responses that may serve as therapeutic vulnerabilities, we treated C3TAg genetically engineered mouse model with a panel of clinically approved therapies then subjected these samples to single-cell RNAseq and multiplex tissue imaging. We observed that relatively short treatment durations lead to changes in the composition and spatial organization of tumors, suggesting adaptive responses that may be leveraged in combination treatment strategies. Notably, we observed changes in several macrophage populations, including changes to co-localization of M2-like macrophage and cancer-associated fibroblasts, highlighting stromal responses to therapeutic pressure. Second, we are using the resultant data to guide development and parameterization of an agent-based model that considers the functional relationships between different tumor components and can be used to predict effective therapeutic treatment strategies. We have developed analytical approaches to leverage our single-cell RNAseq data to estimate key model parameters operable in prioritized cell populations, including those related to proliferation, motility, antigen presentation, and cell-cell signaling. Initial model simulations suggest that spatial cell patterns are often not steady: immune cells dynamically aggregate, disperse, migrate, and reaggregate at new locations; macrophages infiltrated hypoxic and necrotic regions, where they polarized from an anti-tumor M1-like state to a pro-tumor M2-like state. These observations lay the groundwork for leveraging agent-based models for in silico testing of therapeutic combinations that can be prioritized for experimental validation. In total, our approach has the potential to rationally identify effective therapeutic strategies that lead to long-term durable control of tumors.



 Laura M. Heiser, Young Hwan Chang, Paul Macklin. Multiscale systems approach to target tumor ecosystem responses for therapeutic benefit [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3f02bb56a8eb490bc79542639fae113efb7474f" target='_blank'>
              Abstract A027: Multiscale systems approach to target tumor ecosystem responses for therapeutic benefit
              </a>
            </td>
          <td>
            Laura M. Heiser, Young Hwan Chang, Paul Macklin
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Spread through air spaces (STAS) represents a novel invasion mechanism in adenocarcinoma that considerably influences lung cancer clinical outcomes; however, studies of its mechanisms at the spatial level are lacking. Methods We used the NanoString GeoMx digital spatial profiling (DSP) technology to conduct a spatial transcriptomic analysis of surgically resected tissues from non-small-cell lung cancer (NSCLC) patients with or without STAS. Results Compared with tumor nests in non-STAS patients, HLA-DRB5 and RASGRF1 were significantly less expressed in compartments of STAS, suggesting their inhibitory roles in the occurrence of STAS. Meanwhile, an increase in CD4 T memory cells and a decrease in B cells were observed in the tumor immune microenvironment of STAS. Furthermore, distinct molecular profiles were observed between tumor cells in tumor nests and in air spaces in STAS patients, which was highlighted by the elevated ITGA2 expression in the air spaces. These results were validated in an independent cohort by multiplex immunofluorescence stainings. Conclusion This study is the first to use DSP to analyze spatial transcriptomic profiles of NSCLC tumor nests and air space tumors, and it identifies potential module features that may be used for STAS identification and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/570168f77732c756800b064b7d546cfa69566936" target='_blank'>
              Spatial deciphering of the transcriptomic heterogeneity of tumor spread through air spaces in lung cancer
              </a>
            </td>
          <td>
            Wenhao Wang, Wenhao Zhou, Jingli Fan, Tao Jiang, Guang Yang, Congcong Song, Siwei Xu, Haitao Luo, Huining Liu
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Nuclear protein in testis (NUT) carcinomas (NCs) are rare, clinically aggressive tumors with characteristic translocation involving NUTM1 and BRD4 genes. While NCs are generally characterized by undifferentiated basaloid cells with focal/abrupt squamous differentiation, tumoral heterogeneity remains unexplored. This may have therapeutic implications as NC frequently develops drug resistance and metastasis. Spatial transcriptomics (ST) sequencing technology emerged as an approach to elucidate tumoral heterogeneity by integrating cellular transcriptome and spatial distributions within tissues, providing insights into the interactions among different cell types and overall tissue genomic architecture.


METHODS
Three cases of sinonasal NC with formalin-fixed paraffin embedded tissue formed the study material. A representative region of each NC was subjected to 10X Genomics Visium Spatial Gene Expression analysis. Resulting data were analyzed in 10X Genomics Loupe Browser.


RESULTS
All cases demonstrated statistically distinct (graph-neural network based) transcriptomic clusters. These clusters were correlated with histologic characteristics by parsing all spots by region types based on tissue distribution including tumor, limited stroma, and normal epithelium. Cluster constitution and transcriptomic pattern were consistent with the morphologic features of each tissue region. NC#3 had a relatively low number of tumor clusters in comparison to NC#1 and NC#2.


CONCLUSIONS
Spatial transcriptomic profiling corresponds to histopathologic features in NC and yet highlights cell type diversity across tumors and in the case of NC#2, within a given tumor. Behavioral correlates are speculative, but NC#3 for which the patient had a rapidly lethal outcome showed a preponderance of neural gene expression which may be of interest in tumor cell progression/evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8f96bf0cc82c0a5a49b8e030eb8a17bb4a9e111" target='_blank'>
              Magnifying Glass on Sinonasal NUT Carcinoma Heterogeneity via Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Diana Bell, Garret W. Choby, M. Afkhami, E. Maghami, R. Ferrarotto, Carl Snyderman, Eric Wang, Ehab Y. Hanna, Raja Seethala
          </td>
          <td>2025-07-07</td>
          <td>Head & neck</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="ABSTRACT Intra‐tumoral heterogeneity poses a major challenge to treating and managing cancer patients. A characteristic feature of melanoma is its composition of cancer cells with typically heterogeneous content of melanin pigment, the production of which is a hallmark of normal melanocytic differentiation but of poorly understood consequence in melanoma cells, as prospective assessment of pigment heterogeneity in melanoma cells has been experimentally challenging. Here, we describe a novel flow cytometric method for high purity separation of viable melanoma cells based on their melanin content, exploiting the light scattering properties of melanin. By fluorescence‐activated cell sorting, we show that cells with low‐pigment content (LPCs) in melanoma cell lines and patient tumors are usually far more abundant than high‐pigment cells (HPCs) and have substantially increased potentials for colony formation in vitro and tumor formation in vivo. In RNAseq analysis, HPCs showed P53 activation and perturbed cell cycling, whereas LPCs displayed upregulation of MYC‐associated transcription and activated ribosome biogenesis. In proof‐of‐concept studies, the latter was targeted by topoisomerase 2 beta targeting with CX‐5461, which induced senescent HPC phenotypes and irreversible loss of clonogenic activity. These data indicate an ‘inverted pyramid’ hierarchical model of melanoma cell propagation wherein abundant LPCs frequently renew their own malignant potential to propagate disease but also infrequently generate HPCs that spontaneously lose this ability in a manner that might be exploited as an anti‐melanoma strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a7c2a2a2e8b632c604835e92877002fe7954605" target='_blank'>
              Prospective Isolation According to Melanin Pigment Content of Melanoma Cells With Heterogeneous Potentials for Disease Propagation
              </a>
            </td>
          <td>
            Clare Fedele, Gamze Kuser-Abali, Ralph Rossi, Peinan Zhao, Jason Li, M. Ameratunga, Pacman Szeto, Youfang Zhang, Miles C. Andrews, Mark Shackleton
          </td>
          <td>2025-06-30</td>
          <td>Pigment Cell & Melanoma Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Technological breakthroughs in spatial omics and artificial intelligence (AI) have the potential to transform the understanding of cancer cells and the tumor microenvironment. Here we review the role of AI in spatial omics, discussing the current state-of-the-art and further needs to decipher cancer biology from large-scale spatial tissue data. An overarching challenge is the development of interpretable spatial AI models, an activity which demands not only improved data integration, but also new conceptual frameworks. We discuss emerging paradigms, in particular data-driven spatial AI, constraint-based spatial AI, and mechanistic spatial modeling, as well as the importance of integrating AI with hypothesis-driven strategies and model systems to realize the value of cancer spatial information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2461fe4245281300586e28170c9708b8e28c1062" target='_blank'>
              Emerging AI Approaches for Cancer Spatial Omics
              </a>
            </td>
          <td>
            Javad Noorbakhsh, Ali Foroughi pour, Jeff Chuang
          </td>
          <td>2025-06-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Current approaches to estimating cell trajectories, tumor progression dynamics, and cell population diversity of tumor microenvironment often depend on single-cell RNA sequencing, which is costly and resource intensive. To address this limitation, we developed an artificial intelligence (AI) model that leverages cell morphology features and histological spatial organization to classify tumor cell differentiation status, infer cell dynamic trajectories, and quantify tumor progression from hematoxylin and eosin (H&E)–stained whole-slide images. In three independent lung adenocarcinoma cohorts, our AI-based model accurately predicted cell differential status and provided quantifiable measures of tumor progression that were prognostic of patient survival. Spatial transcriptomic integrative analyses revealed cell components and gene signatures enriched in different cell differentiation statuses. Bulk transcriptomic analyses revealed that fast-progressing tumors exhibit up-regulated cell cycle pathways, while slow-progressing tumors retain characteristics of normal lung epithelium. This cost-effective method enables large-scale analysis of tumor progression dynamics using routinely collected pathology slides and provides insights into intratumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ccd7a11dd44adb135af6900f42639f7a9eb70bd" target='_blank'>
              Image-based inference of tumor cell trajectories enables large-scale cancer progression analysis
              </a>
            </td>
          <td>
            Yang Liu, Ling Cai, Ruichen Rong, Shidan Wang, Liwei Jia, Peiran Quan, Qin Zhou, Guan‑Di Xiao, Yang Xie
          </td>
          <td>2025-07-18</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J. Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Recent advancements in transcriptomic analysis, combined with single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics, have deepened our understanding of the tumor microenvironment and cellular heterogeneity, laying the groundwork for personalized therapies. The aim of this research is to explore the heterogeneity of tumor cells in colorectal cancer (CRC) and evaluate their prognostic value in different therapeutic contexts, emphasizing the impact of tumor cell heterogeneity on disease progression. Methods scRNA-seq alongside spatial transcriptomics was employed to analyze the heterogeneity of tumor cells in CRC, the spatial distribution of tumor cells, and their interactions with the microenvironment. Results We identified nine distinct tumor cell subtypes, with MLXIPL + neoplasm prevalent in advanced CRC, while ADH1C + and MUC2 + neoplasms were more common in early-stage CRC. MLXIPL + neoplasm was significantly associated with chemotherapy and targeted therapy efficacy. Analysis of spatial transcriptomics indicated that MLXIPL + neoplasm is located in the core area of the tumor cells. We developed a 13-gene prognostic signature (PS) using machine learning algorithm (StepCox backward), which predicts the prognosis of CRC patients. Furthermore, the patients with low PS score demonstrated higher immune cell infiltration and immune regulatory factors, suggesting enhanced immune surveillance and treatment response. Conclusions The findings highlight the critical role of tumor cell heterogeneity in CRC progression and treatment response, suggesting the need for personalized therapeutic strategies targeting different subpopulations. The constructed PS demonstrates significant clinical application value in predicting patient prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00805-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8065bcb7e9315d28e942e5a0a0b33a35e0d6f402" target='_blank'>
              Spatial transcriptomics and scRNA-seq: decoding tumor complexity and constructing prognostic models in colorectal cancer
              </a>
            </td>
          <td>
            Wei Song, Yatao Wang, Min Zhou, Fengqin Guo, Yanliang Liu
          </td>
          <td>2025-08-13</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Innate immune cells constitute the majority of the tumor microenvironment (TME), where they mediate both natural anti-tumor immunity and immunotherapy responses. While single-cell T- and B-cell receptor sequencing has provided fundamental insights into the clonal dynamics of human adaptive immunity, the lack of appropriate tools has precluded similar analysis of innate immune cells. Here, we describe a method that leverages somatic mitochondrial DNA (mtDNA) mutations to reconstruct clonal lineage relationships between single cells across cell types in native human tissues. We jointly sequenced single-cell transposase-accessible chromatin and mtDNA to profile n=124,958 cells from matched tumor, non-involved lung tissue (NILT), and peripheral blood of early-stage non-small cell lung cancer (NSCLC) patients, as well as n=93,757 cells from matched tumor and peripheral blood of ovarian cancer patients. Single-cell concomitant profiling of lineage and cell states of thousands of immune cells resolved clonality across cell types, tissue sites, and malignancies. Clonal tracing of innate immune cells demonstrates that TME-resident myeloid subsets, including macrophages and type 3 dendritic cells (DC3), are clonally linked to both circulating and tissue-infiltrating monocytes. Further, we identify distinct DC-biased and macrophage-biased myeloid clones, enriched in the tumor and NILT, respectively, and find that their circulating monocyte precursors exhibit distinct epigenetic profiles, suggesting that myeloid differentiation fate may be predetermined before TME infiltration. These results delineate the clonal pathways of intratumoral myeloid cell recruitment and differentiation in human cancer and suggest that remodeling of the tumor myeloid compartment may be peripherally programmed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3aaf4843ae398b98669a86ffe5560f59ef43af0" target='_blank'>
              Clonal lineage tracing of innate immune cells in human cancer
              </a>
            </td>
          <td>
            Vincent Liu, Katalin D. Sandor, Patrick K. Yan, Max Miao, Yajie Yin, Robert R. Stickels, Andy Y. Chen, Kamir J. Hiam-Galvez, Jacob C. Gutierrez, Wenxi Zhang, Sairaj Sajjath, Raeline Valbuena, Steven Wang, Bence Dániel, Leif S. Ludwig, Brooke E Howitt, Caleb A. Lareau, Ansu Satpathy
          </td>
          <td>2025-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) lacks targeted therapies, leading to a poor prognosis. Younger patients with TNBC often present with aggressive disease and exhibit distinct tumor microenvironments (TME). We performed spatial profiling to understand the influence of menopausal status on the immune environment, tumor progression, and therapy response. Methods Eleven treatment-naive TNBC tumors were examined in epithelial and non-epithelial areas using digital spatial profiling to identify differentially expressed markers and pathways. Deconvolution methods were employed to identify immune cell subtypes, and the protein expression of T and B cells was confirmed. The prognostic utility of the identified genes and pathways was validated using the METABRIC and SCAN-B datasets. The expression of target genes was analyzed in the I-SPY 2 trial data to understand their influence on the response to specific therapies. Results Premenopausal tumors formed a distinct cluster characterized by the upregulation of cell activation and antigen presentation pathways. In contrast, T cell checkpoint, cancer antigen, and PI3K-AKT pathways were downregulated. External datasets validated these findings, showing a lower hazard and better prognosis for genes upregulated in premenopausal tumors. An enrichment of CD8 + T cells, endothelial cells, and monocytes was observed, along with increased intratumoral protein expression of CD8, CD4, and CD20. Premenopausal tumors demonstrated better responses to PARP and HSP90 inhibitors but showed lower sensitivity to Pembrolizumab and PI3K-AKT inhibitors compared to postmenopausal tumors in the I-SPY 2 trial data. Conclusion Our study underscores the importance of menopausal status in shaping the TME within TNBC, revealing distinct immune landscapes and therapy responses. These findings highlight the need for larger prospective studies to validate differential treatment strategies in younger patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06786-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c6a6cdaccc22fd9e9450a555636943aa5ed9adb" target='_blank'>
              Spatial profiling reveals unique immune microenvironment in premenopausal triple-negative breast cancer associated with therapy response
              </a>
            </td>
          <td>
            V. P. Nimbalkar, V. Snijesh, S. Rajarajan, C. Anupama, S. Mahalakshmi, Annie Alexander, Deshica Dechamma, Manju Moorthy, Gopalakrishna Ramaswamy, Rakesh Ramesh, B. Srinath, J. Prabhu
          </td>
          <td>2025-07-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78d270a21b6be735f10f280e682156618ed9ceb0" target='_blank'>
              OmniPert: A Deep Learning Foundation Model for Predicting Responses to Genetic and Chemical Perturbations in Single Cancer Cells
              </a>
            </td>
          <td>
            Farzan Taj, Lincoln D. Stein
          </td>
          <td>2025-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) was considered the most aggressive type of primary brain tumor, marked by poor clinical outcomes and a high tendency to relapse. The therapeutic efficacy of GBM was significantly compromised by tumor heterogeneity, dysregulated metabolic pathways, the formation of an immunosuppressive microenvironment, and treatment resistance. Therefore, multi-dimensional therapeutic strategies targeting GBM-specific molecular features, its intrinsic properties, and microenvironmental regulatory networks were considered to potentially provide new breakthroughs for overcoming treatment resistance in GBM. Methods We analyzed single-cell RNA sequencing (scRNA-seq) data processed with the Seurat package to accurately identify cell types. Spatial transcriptomics integrated Multimodal Intersection Analysis, TransferData, and Robust Cell Type Decomposition techniques to characterize the spatial distribution patterns of key cell subtypes. CellChat was employed to assess intercellular communication networks. Furthermore, in vitro experiments confirmed the main regulatory role of YEATS4 (key transcription factor of C2 PCLAF+ subtype) in GBM malignant progression. Results Through scRNA-seq, we identified the C2 PCLAF+ subtype in GBM and analyzed its molecular characteristics and functional role in tumor progression. This subtype exhibited a unique malignant phenotype, marked by significant proliferative activity, characteristic metabolic reprogramming, and dysregulated cell death regulation mechanisms. Spatial transcriptomics revealed its preferential localization within specific tumor niches. Furthermore, the C2 PCLAF+ subtype established a specific interaction with fibroblasts through the MDK-LRP1 ligand–receptor pair. Critically, silencing YEATS4 in vitro significantly inhibited GBM malignancy. Additionally, the prognostic risk score model based on the C2 PCLAF+ subtype demonstrated significant clinical translational value. Conclusion Our study systematically elucidated the malignant characteristics of the C2 PCLAF+ subtype and its molecular mechanisms driving GBM progression. This subtype promoted therapeutic resistance through unique metabolic reprogramming, MDK-LRP1-mediated microenvironmental interactions, and immunosuppressive properties. YEATS4 knockdown effectively suppressed malignant tumor behaviors, highlighting its therapeutic potential. These findings provided novel targeted intervention strategies to address GBM heterogeneity and treatment resistance, offering promising avenues for overcoming current therapeutic limitations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d8a996fe1a4f26a1455e98643e994edaf34d934" target='_blank'>
              Single-cell and spatial atlas of glioblastoma heterogeneity: characterizing the PCLAF+ subtype and YEATS4’s oncogenic role
              </a>
            </td>
          <td>
            Siqi Ma, Yuwei Sun, Shaowei Zheng, Yilong Fu, Liangyu Wang, Dun Liu, Henan Jiao, Xuqiang Zhu, Xueyuan Li, Dongming Yan, Di Chen, Zi Ye
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most lethal and treatment-resistant malignancies, characterized by high recurrence rates following standard-of-care therapy. While previous longitudinal studies employing whole-exome and RNA sequencing have revealed patient-specific clonal evolution, they have not identified conserved biological programs that drive recurrence or therapeutic resistance. A recent study published in Cancer Cell presents the first integrated proteogenomic analysis of matched primary and recurrent GBMs. This integrative approach reveals a striking phenotypic transition in recurrent tumors, characterized by neuronal reprogramming supported by coordinated transcriptional, proteomic, and phosphoproteomic evidence. In this review, we contextualize these findings within the broader landscape of GBM evolution, emphasizing the mechanistic contributions of WNT/planar cell polarity (PCP) signaling and BRAF kinase activation in facilitating a neuronal-like state that enhances tumor plasticity, invasion, and treatment resistance. We further discuss how these profound insights align with preclinical models of tumor-neuron synaptic crosstalk, and propose that proteogenomics offers a powerful lens through which to uncover clinically actionable vulnerabilities. By redefining the functional landscape of recurrent GBM, the current study establishes a new framework for biomarker discovery and the rational design of targeted therapies informed by tumor evolution and neuronal niche adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/038e3b3c0223704408dab5888923306203acc73b" target='_blank'>
              Proteogenomic Insights Into Glioblastoma Evolution: Neuronal Reprogramming and Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Harim Koo, Jason K. Sa
          </td>
          <td>2025-07-01</td>
          <td>Brain Tumor Research and Treatment</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of kidney renal clear cell carcinoma (KIRC) exhibits complex dynamics among immune, stromal, and malignant cells that drive immune escape (IE) mechanisms and influence clinical outcomes. Through single-cell RNA sequencing and high-dimensional weighted gene co-expression network analysis (hdWGCNA), we identified a crucial IE-related gene module most strongly associated with KIRC progression. Partitioning Around Medoids (PAM) clustering delineated two distinct IE patterns, with pattern one demonstrating prolonged patient survival. Employing advanced machine learning (ML) algorithms, we identified Cathepsin S (CTSS) as the most pivotal tumor suppressor, with elevated CTSS expression consistently predicting improved survival across multiple independent cohorts. Functional analyses revealed significant enrichment of CTSS in immune-regulatory pathways, including B/T cell activation and inflammatory activity. Mutation profiling uncovered distinct genomic alterations in 5q35.3 among CTSS-high tumors, while drug response prediction identified eight potential therapeutic agents (e.g., Navitoclax, Ibrutinib) exhibiting enhanced efficacy in these patients. Notably, CTSS expression strongly correlated with immune cell infiltration and established immunotherapy biomarkers, supporting its dual role as both a prognostic indicator and predictor of immune response. This study provides mechanistic insights into IE in KIRC and positions CTSS as a promising biomarker and therapeutic target for precision immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03267-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6364c60b20a894510d01361e9f698e8acfc1b2ab" target='_blank'>
              CTSS in the tumor microenvironment links immune escape and immunotherapy sensitivity in kidney renal clear cell carcinoma
              </a>
            </td>
          <td>
            Hanjing Zhou, Jun Ying, Xuchun Xu, Jian Huang
          </td>
          <td>2025-07-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Cervical cancer (CC) remains a significant global health challenge despite advancements in screening, HPV vaccination, and therapeutic strategies. Tumor heterogeneity, driven by epigenetic modifications, affects immune evasion, metastasis, and treatment response. Cancer-associated fibroblasts (CAFs) play a crucial role in CC progression and therapy resistance. Single-cell sequencing offers new insights but remains underutilized in CC research. This study integrates single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and deconvolution analysis to identify key genes and immunotherapy targets. By constructing a prognostic model and exploring the immune microenvironment, we aim to provide novel insights into CC pathogenesis and potential therapeutic strategies. Methods We utilized scRNA-seq, spatial transcriptomics, deconvolution analysis, and pseudotime trajectory mapping to delineate fibroblast subtypes within the tumor immune microenvironment (TIME) of CC. Functional annotations, differential gene expression profiling, cell–cell communication pathways, and transcription factor networks were systematically analyzed. A prognostic model based on bulk RNA-seq data was constructed and validated through survival analysis, with correlations to immune microenvironment characteristics. Functional experiments investigated the role of SDC1, a critical mediator of fibroblast-tumor crosstalk. Additionally, Fibroblast–tumor cell co-culture systems and functional assays were employed to investigate the paracrine role of SDC1. The CAF MYH11⁺ subpopulation was isolated via fluorescence-activated cell sorting (FACS). Multiplex immunofluorescence and immunohistochemical analyses were performed on both cultured cells and human cervical cancer tissue samples to characterize the spatial distribution and dynamic remodeling of MYH11 during stromal reorganization. Results Six distinct fibroblast subtypes were identified, including the C0 MYH11 + fibroblasts, which exhibited unique roles in stemness maintenance, metabolic activity, and immune regulation. Spatial and functional analyses revealed that the C0 subtype is central to tumor-fibroblast interactions, particularly through the MDK-SDC1 signaling axis. The prognostic model incorporating fibroblast-specific markers demonstrated robust predictive power for patient survival outcomes. Additionally, in vitro SDC1 knockdown significantly inhibited CC cell proliferation, migration, and invasion. Fibroblasts show spatially regulated heterogeneity, with activation markers enriched in the tumor zone and MYH11 highest in normal zones, indicating dynamic stromal remodeling. C0 MYH11 + CAF Promotes Tumor Cell Proliferation, Migration, and Inhibits Apoptosis via Soluble SDC1. Conclusion Our results illustrate, in some ways, the possible immunomodulatory and tumor supporting roles of CAFs in CC TIME and highlight the possibility that the MDK-SDC1 pathway is a promising therapeutic target. This study not only promotes a partially new understanding of temporal heterogeneity in CC, but also provides a possible reference base for the development of new biomarkers and immunotherapy approaches to improve clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03432-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d1387a936b1ef6d7e045f6c7bcff60d568a6f7" target='_blank'>
              Deciphering the tumor immune microenvironment: single-cell and spatial transcriptomic insights into cervical cancer fibroblasts
              </a>
            </td>
          <td>
            Zhiheng Lin, Youwei Zhou, Zhenran Liu, Wenyang Nie, Hengjie Cao, Shengnan Li, Xuanling Li, Lijun Zhu, Guangyao Lin, Yanyu Ding, Yi Jiang, Zuxi Gu, Lianwei Xu, Zhijie Zhao, Hua-Bao Cai
          </td>
          <td>2025-07-05</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>2</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Pancreatic cancer is a highly malignant tumor of the digestive system with a dismal prognosis. Despite advances in diagnosis and treatment, overall survival remains extremely low. Early diagnostic markers and an improved understanding of tumor-microenvironment interactions are essential for developing more effective therapies. Methods We analyzed 74 single-cell RNA sequencing (scRNA-seq) samples, performing unsupervised clustering and marker-gene expression profiling to define major cell types. Large-scale chromosomal copy-number variation (CNV) analysis distinguished malignant from non-malignant ductal cells. Non-negative matrix factorization (NMF) identified stage-associated gene modules, which were integrated with TCGA bulk-RNA data and machine-learning feature selection to pinpoint candidate prognostic genes. Two independent cohorts were used for validation. Regulatory network inference (pySCENIC) and ligand–receptor interaction analysis (CellPhoneDB) explored cross-talk between malignant cells and macrophages. Finally, in vitro knockdown of CTSV assessed its functional role in pancreatic cancer (PAC) cell proliferation and migration. Results Three prognosis-related genes—ANLN, NT5E, and CTSV—were selected based on their strong association with clinical stage and validated in external datasets. High expression of these genes correlated with poorer overall survival and an increased infiltration of M0 macrophages. CellPhoneDB predicted significant interactions between high-expression malignant ductal cells and M0 macrophages via CXCL14–CXCR4 and IL1RAP–PTPRF axes, with SPI1 identified as an upstream regulator of IL1RAP. In vitro CTSV knockdown significantly inhibited PAC cell proliferation and migration. Discussion Our integrative single-cell and bulk-RNA workflow identifies ANLN, NT5E, and CTSV as novel prognostic biomarkers in pancreatic cancer and highlights a pro-tumorigenic interaction between malignant ductal cells and macrophages. Targeting CTSV or disrupting CXCL14–CXCR4 and IL1RAP–PTPRF signaling may offer new therapeutic avenues for PAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50ce5676a35dfc57b1b078e8894ee25f419175d" target='_blank'>
              Integration of single-cell RNA and bulk RNA sequencing revealed malignant ductal cell heterogeneity and prognosis signatures in pancreatic cancer
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Tumor-associated macrophages (TAMs) are critical regulators of the hepatocellular carcinoma (HCC) microenvironment, yet their epigenetic heterogeneity and regulatory programs remain poorly defined. Methods: We performed integrative analysis on single-cell RNA-seq and ATAC-seq profiling of HCC patients to dissect TAM subtypes at high resolution. By correlating chromatin accessibility with gene expression, we identified cell-type-specific candidate cis-regulatory elements (CREs). TAM subsets with prognostic significance were determined through integration with HCC clinical cohorts. Pseudotime and multi-regional analyses were used to uncover regulatory trajectories underlying macrophage phenotypic transitions. The identification framework of a super-enhancer (SE) was constructed, and potential therapeutic targets were prioritized using drug–gene interaction data. Results: We delineated the regulatory landscape of TAMs in HCC, revealing cell-type-specific chromatin accessibility patterns underlying TAM heterogeneity. The 65,342 CREs linked to gene expression were identified, with distal CREs contributing most to cell-type-specific regulation. Notably, SPP1+ TAMs were found to be enriched in tumor cores and associated with poor prognosis in HCC. Liver-resident Kupffer cells showed progressive loss of the core transcription factors SPIC and MAFB, suggesting a potential transition into SPP1+ TAMs under tumor pressure. We identified 133 SPP1+ TAM-specific SEs and constructed a TF–SE–target gene regulatory network. Notably, 13 target genes showed higher drug–gene interaction effects, highlighting their therapeutic potential. Conclusions: This study provides the chromatin accessibility map of TAMs in HCC and reveals how distal CRE-driven transcriptional programs shape TAM states. Our findings lay the foundation for understanding the epigenetic regulation of TAM heterogeneity and nominate potential targets for TAM-directed immunotherapy in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e02327a10ea72222cca675d378cb3f0e78ed3b3" target='_blank'>
              Integrated Single-Cell Analysis Dissects Regulatory Mechanisms Underlying Tumor-Associated Macrophage Plasticity in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Yu Gu, Wenyong Zhu, Zhihui Zhang, Huiling Shu, Hao Huang, Xiao Sun
          </td>
          <td>2025-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e5808d86a5155048eee62dffb4fd9d18cc91c36" target='_blank'>
              AI-powered Deep Visual Proteomics reveals critical molecular transitions in pancreatic cancer precursors
              </a>
            </td>
          <td>
            Jimin Min, Lisa Schweizer, Gijs Zonderland, B. Selvanesan, Lukas Oldenburg, Seong-Woo Bae, Bong Jun Kim, Benjamin J. Swanson, Kelsey A. Klute, T. Caffrey, P. Grandgenett, Michael A. Hollingsworth, Ishani Ummat, Maximilian T. Strauss, A. Mund, Anirban Maitra
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="High-grade serous tubo-ovarian cancer (HGSC) is marked by substantial inter- and intra-tumor heterogeneity. The tumor microenvironments (TME) of HGSC show pronounced variability in cellular make-up across metastatic sites, which is linked to poorer patient outcomes. The influence of cellular composition on therapy sensitivity, including chemotherapy and targeted treatments, has not been thoroughly investigated. In this study, we examined the premise that the variations in cellular composition can forecast drug efficacy. Using a high-throughput 3D in vitro tumoroid model, we assessed the drug responses of 23 distinct cellular configurations of tumoroids comprised of OVCAR3 HGSC cells, mesenchymal stem cells, HUVEC endothelial cells, and U937 monocytes to an assortment of five therapeutic agents, including carboplatin and paclitaxel. We identified that the overall pooled viability in response to these five drugs was highest among tumoroid compositions that contained a large number of myeloid cells, whereas the most sensitive tumoroids to these agents were comprised of only cancer cells. Additionally, we found that the “mesenchymal tumoroids” containing 400 or more mesenchymal stem cells were more sensitive to carboplatin than paclitaxel. By amalgamating our experimental findings with random forest machine learning algorithms, we assessed the influence of TME cellular composition on treatment reactions. Our findings reveal notable disparities in drug responses correlated with tumoroid composition, underscoring the significance of cellular diversity within the TME as a predictor of therapeutic outcomes. This research establishes a foundation for employing human tumoroids with varied cellular composition as a method to delve into the roles of stromal, immune, and other TME cell types in enhancing cancer cell susceptibility to various treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49e1984d98981fa778612856dadda5f531075bc3" target='_blank'>
              Cracking the code: predicting tumor microenvironment enabled chemoresistance with machine learning in the human tumoroid models
              </a>
            </td>
          <td>
            Michael Bregenzer, P. Mehta, Kathleen M Burkhard, Geeta Mehta
          </td>
          <td>2025-08-19</td>
          <td>Npj Biomedical Innovations</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Intrinsically resistant glioma stem cells (GSCs) in the setting of a highly immunosuppressive tumor microenvironment (TME) remain the most predominant phenomenon leading to unfavorable therapeutic outcomes in glioblastoma (GBM). Hence there is an unmet need for novel anti-GBM therapeutic paradigms that can effectively target GSCs while simultaneously reprogramming the TME. Methods In this study, we leverage evidence from SMAC mimetic screening to evaluate and characterize the anti-tumor and immune TME modulating impacts of the lead SMAC mimetic Xevinapant at the single cell level in GBM. We utilized viability assays and orthotopic human and murine GBM models to assess the survival impacts of Xevinapant on GSCs in vitro and in vivo. Moreover, we employed single-cell RNA sequencing (scRNA-seq) to investigate the modulation impact of Xevinapant on GBM TME. Lastly, we investigated drug combination synergies to address potential mechanisms of tolerance or resistance to Xevinapant. Results According to our observations, in vitro exposure to Xevinapant induced apoptosis along with significant viability reduction in a dose-dependent manner, in both human and mouse GSCs. Moreover, Xevinapant treatment produced robust anti-tumor effects in vivo and significantly prolonged animal overall survival. Based on single-cell RNA seq analysis, Xevinapant did not only enhance GSCs apoptosis but also activated antitumor effector immune response leading to favorable reprogramming of immunosuppressive TME. Furthermore, we established and queried Xevinapant therapeutic signatures to the LINCS database in an effort to identify small molecules that could reverse treatment-induced tolerance to Xevinapant. We have identified a novel set of candidate small molecules with robust synergy when combined with Xevinapant. Conclusions In summary, Xevinapant exhibits robust anti-tumor activity on GSCs and favorable immune modulation of the TME in GBM, hence providing a rationale for further clinical investigation in GBM. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03452-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb7b9489b0db7151a32f56228f76b248fb8bf1d8" target='_blank'>
              The reprogramming impact of SMAC-mimetic on glioblastoma stem cells and the immune tumor microenvironment evolution
              </a>
            </td>
          <td>
            Qiong Wu, Jianan Chen, Anders E Berglund, Dongliang Du, Robert J Macaulay, Arnold B Etame
          </td>
          <td>2025-07-04</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Single-cell RNA sequencing captures static snapshots of gene expression but lacks the ability to track continuous gene expression dynamics over time. To overcome this limitation, we developed PROFET (Particle-based Reconstruction Of generative Force-matched Expression Trajectories), a computational framework that reconstructs continuous, nonlinear single-cell gene expression trajectories from sparsely sampled scRNA-seq data. PROFET first generates particle flows between time-stamped samples using a novel Lipschitz-regularized gradient flow approach and then learns a global vector field for trajectory reconstruction using neural force-matching. The framework was developed using synthetic data simulating cell state transitions and subsequently validated on both mouse and human in vitro datasets. We then deployed PROFET to investigate heterogeneity in treatment responses to palbociclib, a CDK4/6 inhibitor, in hormone receptor positive breast cancer. By comparing newly generated scRNA-seq data from a palbociclib-resistant breast cancer cell line with published patient-derived datasets, we identified a subpopulation of patient cells exhibiting profound phenotypic shifts in response to treatment, along with surface markers uniquely enriched in those cells. By recovering temporal information from static snapshots, PROFET enables inference of continuous single-cell expression trajectories, providing a powerful tool for dissecting the heterogeneity of cell state transitions in treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3148960cea2da72c3e81995d56cd3f9d3682ff6f" target='_blank'>
              PROFET Predicts Continuous Gene Expression Dynamics from scRNA-seq Data to Elucidate Heterogeneity of Cancer Treatment Responses
              </a>
            </td>
          <td>
            Yu-Chen Cheng, Hyemin Gu, Thomas O McDonald, Wenbo Wu, Shubham Tripathi, Cristina Guarducci, Douglas Russo, Daniel L. Abravanel, Madeline Bailey, Yue Wang, Yun Zhang, Yannis Pantazis, Herbert Levine, R. Jeselsohn, M. Katsoulakis, Franziska Michor
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr id="Insufficient infiltration of cytotoxic T cells into solid tumors remains a critical obstacle in cancer immunotherapy. Despite extensive efforts to comprehend the mechanisms governing this limited infiltration, few studies have focused on the evolution of T cell motility behavior after co-culture. In this study, we combined quantitative cell trajectory analysis, computational modeling, and bulk/single-cell RNA sequencing to systematically characterize the impact of cell interactions. We reveal that in a 2.5D co-culture system with multiple cancer-cell clusters, cancer-specific T cells exhibit increased directional persistence, which facilitates their efficient searching of cancer-cell clusters. Additionally, these T cells form prolonged interactions with cancer cells, which is the most crucial factor for their accumulation on cancer-cell clusters. Furthermore, post-interaction, a cancer-cell subpopulation displays immunosuppressive traits, reducing T cell attractant expression, and undergoing epithelial-to-mesenchymal transition. These findings offer valuable insights into improving immunotherapy efficacy and tackling T cell infiltration challenges in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0a380330210bd455d4b1fb482cbeaabdb64b1d" target='_blank'>
              Deciphering antigen-specific T cell navigation tactics and cancer immune evasion in co-cultures
              </a>
            </td>
          <td>
            Xinyue Li, Taoli Jin, Lisha Wang, Senbin Wu, Yerong Liu, Ming Li, Weijing Han, Xuefei Li
          </td>
          <td>2025-07-31</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) play dual roles in cancer, either promoting or suppressing tumor progression, complicating therapeutic approaches. TAMs include recruited macrophages (recMacs), derived from circulating monocytes, and tissue-resident interstitial macrophages (IMs). We recently identified a heterogeneous population of chemokine-expressing IMs, including subsets that support tertiary lymphoid structure (TLS) formation during lung inflammation. Here, we show that IMs can be either pro- or anti-tumorigenic, depending on the subset. Using Pf4 Cx3cr1 mice to deplete CD206hi IMs expressing Cxcl13, Cxcl9, and Cxcl10, we demonstrate their essential role in TLS formation, lymphocyte recruitment, and tumor suppression in melanoma and lung adenocarcinoma. In contrast, Ccl2-expressing IMs promote tumor growth by recruiting pro-tumorigenic recMacs. Spatial transcriptomics confirmed the distinct localization and chemokine profiles of these subsets. Finally, CCR5 blockade with the FDA-approved inhibitor Maraviroc during neoantigen vaccination improved tumor control by preventing the migration of immunosuppressive, antigen-presenting recMacs (moDCs). These findings support the development of macrophage-targeted therapies by identifying pro-tumorigenic subsets and recMac trafficking as actionable targets, while preserving macrophage populations that sustain anti-tumor immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/313d8c71525c97f65bdf0888b072625e1176bbdb" target='_blank'>
              The Dichotomy of Tumor Control by Recruited and Resident Tumor-Associated Macrophages
              </a>
            </td>
          <td>
            C. Jakubzick, Soubhik Ghosh, Xin Li, Kavita Rawat, Aishwarya Dighal, Stephanie Kalinowski, F. Iv, Carol Ringelberg
          </td>
          <td>2025-07-04</td>
          <td>Research Square</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) progression is driven by a complex interplay of immunosuppressive and inflammatory mechanisms involving various cell types, functional states, and extracellular milieu. To unravel this intricate relationship, we took a systems biology approach using high resolution spatial distributions of multiple key markers in clinically diverse CRC tumors. We employed Multiplexed Ion Beam Imaging (MIBI-TOF) to capture 65 protein markers across 3,700 fields of view from 250 CRC tumors. This allowed us to deeply characterize the phenotypes and functional states of 5.7 million cells. To incorporate extracellular geography information, we developed a computational approach named CellExt. This quantified and modeled the gradated abundance of key structural and functional proteins found extracellularly, such as fibronectin, granzymeB and vimentin, from each cell boundary to a 40-micron distance in pixelwise steps. CellExt detected a distinct immunosuppressive modality characterized by extracellular PDL1, TNF-related apoptosis-inducing ligand (TRAIL), and matrix metalloproteinase-9 (MMP9). Investigating the cell types involved led us to map the potential receivers and emitters by linking extracellular trends to cellular enrichment and gene expression. We found that Pankeratin-high epithelial cells were acting as TRAIL receivers with downstream activation of non-canonical pro-tumorigenic signaling through S6 phosphorylation. In parallel, we found that T cell suppressive CD40+ antigen-presenting B cells with PD1 expression were acting as MMP9 emitters. Additionally, CellExt was able to identify two spatially segregated inflammatory modalities: IFNgamma cytokine with mucin protein, MUC5AC and prostaglandin synthase, cyclooxygenase-2 with extracellular matrix component, Collagen1A1. This suggests a strong inflammatory role for Collagen1A1 in CRC. We then contextualized our identified modalities by genetic background, tumor stage and CMS subtype of CRC tumors. Taken together, our approach revealed novel axes of immunosuppression and inflammation and puts forward potential extracellular targets for therapeutic intervention in specific CRC patient subsets.



 Reema Baskar, Vairavan Lakshmanan, Audrey Lee, Wilfred Wong, Dane Bagaoisan, Yuhan Peng, Marc Bosse, Ferda Filiz, Christine Fullaway, Sean Bendall, Iain Beehuat Tan, Shyam Prabhakar. Spatial Profiling of Colorectal Cancer Extracellular Milieu Reveals Novel Axes of Immunosuppression and Inflammation [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P38.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18eea1ae1312b735cfb9925fbcce433bb07c964b" target='_blank'>
              Abstract P38: Spatial Profiling of Colorectal Cancer Extracellular Milieu Reveals Novel Axes of Immunosuppression and Inflammation
              </a>
            </td>
          <td>
            Reema Baskar, Vairavan Lakshmanan, Audrey Lee, Wilfred Wong, Dane Bagaoisan, Yuhan Peng, Marc Bosse, Ferda Filiz, C. Fullaway, Sean C. Bendall, Iain Beehuat Tan, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) remains one of the most prevalent malignancies. A series of differentially expressed genes (DEGs) have been identified in tumor samples and play critical roles in modulating the characteristics of tumor cells. However, some DEGs are specifically expressed in the tumor microenvironment (TME) cells. The underlying mechanisms of the functional DEGs warrant comprehensive investigation to elucidate their contributions to tumor biology of NSCLC. Therefore, the primary goal of our study is to systematically investigate TME-related DEGs using NSCLC as a model. Methods DEG analysis was performed by comparing bulk transcriptomes of adjacent and tumor samples across 7 independent NSCLC cohorts. Expression pattern of these DEGs were annotated to specific cell types using a single-cell RNA sequencing (scRNA-seq) dataset from 13 NSCLC studies. Myeloid and stromal cells were re-clustered to achieve a detailed characterization of cell-cell interactions within the TME. Spatial co-localization of distinct subpopulations was validated by immunofluorescence staining and spatial transcriptomics (ST). Finally, functional relevance of these interactions was evaluated using a conditional knockout mouse model. Results A total of 82 overlapping DEGs were screened out using bulk transcriptomes across 7 NSCLC cohorts. After clustering the integrated 547,360 cells from 217 adjacent/tumor NSCLC samples with available scRNA-seq data, we observed that most of these DEGs were specifically expressed in epithelial, myeloid, and stromal cells. Notably, SPP1 ranks the top DEG and is specifically expressed in myeloid cells. The elevated SPP1 expression in bulk transcriptome was attributed to both enriched proportion of SPP1+ macrophages and increased expression level of SPP1 in such myeloid subcluster in NSCLC tumor, which was potentially regulated by several transcriptional factors (e.g., HIF1A, ATF5, and STAT1). Cell-cell communication analysis indicated FAP+ fibroblasts exhibited the strongest interaction with SPP1+ macrophages within TME. Multiple fluorescent staining demonstrated significantly increased SPP1+ macrophage and FAP+ fibroblast interactions in tumor compared to adjacent tissues. Nichnet analysis showed VCAN-ITGB1 ligand-receptor axis mediating this cellular interaction. ST analysis revealed that this interaction established an immune-excluding barrier surrounding the tumor core. In addition, immune cells (CD4+/CD8+ T and NK cell) were recruited to tumor edge. Consistently, macrophage-specific Spp1 knockout in a mouse model resulted in increased CD3+ and CD8+ T cell infiltration and reduced subcutaneous tumor size. Clinically, patients with FAPHighSPP1High expression in tumor exhibited significantly poor prognosis for immunotherapy treatment. Conclusions SPP1 ranks as the top DEG in NSCLC tumor attributed to its specific expression in SPP1+ macrophages, proportion of which increases in tumor and interact with FAP+ fibroblasts. Their interactions contribute to the formation of a tumor barrier that inhibits immune infiltration, reducing the immunotherapy efficacy in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5040bbe7618af6ffa532070665742270d9ea5641" target='_blank'>
              Single-cell and spatial transcriptomics profile the interaction of SPP1+ macrophages and FAP+ fibroblasts in non-small cell lung cancer
              </a>
            </td>
          <td>
            Minqin Xiao, Yiqi Deng, Hang Guo, Zhixiang Ren, Yajiao He, Xia Ren, Li-Bin Huang, Wei-Han Zhang, Hai-Ning Chen, Y. Shu, Fanxin Zeng, Yan Zhang, Heng Xu, Lanlan Wang
          </td>
          <td>2025-07-01</td>
          <td>Translational Lung Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) tumors exhibit pronounced phenotypic plasticity, alternating between a treatment-sensitive classical phenotype and a more aggressive basal-like state associated with drug resistance and poor prognosis. The frequent coexistence of these phenotypes complicates patient stratification and the selection of effective therapies. Tumor-derived organoids are valuable tools for drug screening; however, their clinical relevance relies on how accurately they recapitulate the phenotypic and functional characteristics of the original tumors. In this study, we present a quantitative analysis of how hydrogel composition influences the phenotype, tissue remodeling, metabolism, and drug resistance of PDAC organoids. Organoids were cultured within three types of hydrogels: Matrigel, collagen-I, and a mixture of collagen-I and Matrigel. Our results demonstrate that: (i) PDAC organoids grown in Matrigel exhibit a classical phenotype, with metabolic and drug response profiles similar to those of low-physiological two-dimensional cultures; (ii) Organoids grown in collagen-containing hydrogels, particularly those in collagen-Matrigel composites, faithfully recapitulate basal-like tumors, characterized by epithelial-to-mesenchymal transition, tissue remodeling, metabolic activity, and drug resistance; (iii) TGFβ induces an exacerbated, highly invasive basal-like phenotype. Summarizing, our findings highlight the importance of 3D hydrogel composition in modulating PDAC organoid phenotype and behavior and suggest collagen-Matrigel hydrogels as the most suitable matrix for modeling PDAC biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24bfd79f1227aca7f9356ac5488b713b766f1a87" target='_blank'>
              Tumor Organoids Grown in Mixed-Composition Hydrogels Recapitulate the Plasticity of Pancreatic Cancers
              </a>
            </td>
          <td>
            I. Sorzabal-Bellido, Xabier Morales, I. Cortés-Domínguez, Maider Esparza, Lucía Grande, Pedro Castillo, Silvia Larumbe, María Monteserín, Shruthi Narayanan, Mariano Ponz-Sarvise, Silve Vicent, Carlos Ortiz-de-Solórzano
          </td>
          <td>2025-07-01</td>
          <td>Gels</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d66d1c34039c0624a67bfae6ffcc0194c9308ee" target='_blank'>
              Bioelectric and Epigenetic Landscapes in Lateralized Breast Tumors Reveal Distinct Tumor Microenvironment Signatures
              </a>
            </td>
          <td>
            Sebastián Real, Sergio R. Laurito, Pablo Gonzalez, Oscar Bello, Joao Carvalho, María Roqué
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tumorigenesis is a microevolutionary process in which heterogeneous tumor cells adapt within a complex microenvironment. Current tumor omics analyses often focus exclusively on tumor samples, overlooking the valuable insights that normal tissues can provide. To address this gap, we introduce VaDTN (Variational Autoencoder–Derived Tumor-to-Normal), a pan-cancer framework that integrates transcriptomic data from both tumor and normal samples into a unified latent space. By measuring each tumor’s “distance” from a normal reference within this latent space, VaDTN reveals subtle molecular shifts linked to tumor evolution and heterogeneity. We applied VaDTN to six representative cancers (SKCM, BRCA, LIHC, LUSC, STAD, and PAAD), identifying distinct subtypes characterized by unique transcriptional profiles. Notably, four cancer types (SKCM, BRCA, LIHC, and STAD) displayed significant survival stratification based on these subtype groupings, underscoring the clinical relevance of the distance-based approach. This reference-centered perspective thus provides a refined lens for dissecting intra-tumor diversity and guiding potential precision oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dc636c3386b69302e143969833b154f54e564be" target='_blank'>
              Novel cancer subtyping method guided by tumor-normal sample in latent space of transcriptomic variational autoencoder
              </a>
            </td>
          <td>
            Hongzhi Wang, Yu Zhang, Dandan Zhang, Min Luo
          </td>
          <td>2025-07-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Globally, endometrial cancer continues to impact a significant number of women. Immunotherapy provides those suffering from advanced or relapsed disease hope, but an important barrier is still the absence of trustworthy predictive biomarkers. To tackle this challenge, single-cell sequencing and spatial transcriptomics (ST) are increasingly applied. In cervical cancers of the no specific molecular profile (NSMP) subtype accompanied by p53 mutations. In many cases, the tumor microenvironment (TME) in endometrial cancer exhibits strong immunosuppression or poor immune cell infiltration, often leading to worse clinical outcomes. Single-cell sequencing reveals cellular heterogeneity and helps identify potential therapeutic targets and predict treatment responses. Conversely, ST assists in determining biomarkers that influence the effectiveness of immunotherapy by capturing the spatial organization of tumors. When combined, these technologies allow for integrated multi-omics analysis that aids in the development of immunotherapies, prognostication, and diagnosis. But there are still moral and legal issues. Clinicians may be able to improve outcomes for patients who don’t respond well to current immunotherapies by utilizing these combined approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f087dd53dee17003d5211dac4cb4360cce4847" target='_blank'>
              Single-cell technologies and spatial transcriptomics: decoding immune low - response states in endometrial cancer
              </a>
            </td>
          <td>
            Yumeng Li, Hua Qiu, Zhenzhen Zhao, Fanghua Qi, Pingping Cai
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Cancer development and progression are driven by the accumulation of somatic genetic alterations, which occur in a specific temporal order. However, how the order of mutations impacts cancer phenotypes of solid tumors remains poorly understood. To address this, we developed a novel computational framework, IMOP-Cancer (Identifying Mutation Order Pairs in Cancer), to identify mutation gene pairs whose order influences cancer phenotypes. We applied IMOP-Cancer to The Cancer Genome Atlas-Lung Adenocarcinoma (TCGA-LUAD) cohort and identified 446 key mutation order pairs, with 34 pairs significantly associated with prognosis. Mutation order impacts cancer phenotypes, as demonstrated by CSMD3 and PTPRD (tumor proliferation) and TP53 and NAV3 (immune modulation), with effects validated in four independent datasets. We further presented the impact of mutation pairs on cancer phenotypes through case studies in the TCGA cohorts of bladder urothelial carcinoma (BLCA), and colon adenocarcinoma (COAD). We extended this analysis to 33 cancer cohorts from TCGA portal, identifying 106 034 critical mutation pairs across 17 cancers, with 3036 pairs co-occurring in multiple cancers. Shared mutation pairs across cancers also showed distinct effects on cancer phenotype. Our study highlights the importance of mutation order in cancer progression and diversity, offering new insights into the temporal dynamics of co-occurring mutations and paving the way for personalized treatment strategies and improved diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348ae5004fc1076373867ea177acfe998124027" target='_blank'>
              IMOP-Cancer: identifying mutation order pairs impacting cancer phenotypes
              </a>
            </td>
          <td>
            Yijing Zhang, Shaobo Kang, Renjie Dou, Wanmei Zhang, Yuanyuan Liu, Yang Wu, Dongxue Li, Fangfang Fan, Yanyan Ping
          </td>
          <td>2025-07-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Patient-derived tumor xenograft (PDX) models serve as powerful tools in oncology research owing to their ability to capture patient-specific tumor heterogeneity and clinical behavior. However, the conventional matrices derived from murine tumors, commonly used to generate PDX models, suffer from key limitations such as lack of tissue specificity, high production costs, and inconsistent batch quality. In response, our study investigates the use of decellularized liver extracellular matrix (Liver ECM) as a biomimetic alternative that more accurately recapitulates the native hepatic microenvironment. We demonstrate that Liver ECM, enriched with liver-specific biochemical cues, enables robust engraftment, growth, and metastasis of patient-derived hepatocellular carcinoma cells in both subcutaneous and orthotopic PDX models. Notably, orthotopic models established with Liver ECM exhibited enhanced metastatic behavior, particularly to the intestine, compared to those formed using conventional matrices. Transcriptomic analysis further revealed activation of key pathways associated with cancer progression, including angiogenesis, apoptosis, migration, and inflammation. Additionally, we extend the application of Liver ECM to patient-derived organoid xenografts, which showed improved tumorigenicity and retained pathophysiological features of the original tumor tissue. Together, these findings underscore the potential of liver-specific ECM as a superior platform for generating physiologically relevant PDX models and enhancing the translational relevance of preclinical cancer studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77773de20ab5035f09de4e444e6e5d2f39f36813" target='_blank'>
              Liver-Specific Extracellular Matrix Enables High-Fidelity Patient-Derived Hepatocellular Carcinoma Xenograft Models
              </a>
            </td>
          <td>
            Su Kyeom Kim, Jungho Bae, Mi Jeong Lee, Dai Hoon Han, Seung-Woo Cho
          </td>
          <td>2025-07-23</td>
          <td>Biomaterials Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac106c26ebd96b1e15958a8ddb1a880a0c7c51a9" target='_blank'>
              Intratumoral Heterogeneity of Vimentin Modulates Nuclear Mechanotransduction, DNA Damage Response and Cancer Cell Survival
              </a>
            </td>
          <td>
            Infante Elvira, Terriac Emmanuel, Gelin Matthieu, Siegfried Hugo, Pereira David, Roca Vanessa, Varet Hugo, Khalilian Sara, Rietveld Reinier, Asnacios Atef, van Bodegraven Emma J, Etienne-Manneville Sandrine
          </td>
          <td>2025-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) continues to be a leading cause of cancer-related deaths globally, primarily due to its late diagnosis and the complex nature of its tumor microenvironment (TME). Within this environment, cancer-associated fibroblasts (CAFs) play a crucial role in regulating NSCLC progression and therapeutic resistance. Recent advancements in single-cell and spatial technologies have uncovered significant heterogeneity among CAFs, revealing distinct subpopulations with varying cellular origins, phenotypes, and functions. Besides, the role of small extracellular vesicles (sEVs) in facilitating bidirectional communication highlights functional importance of CAF derived sEVs in mediating the crosstalk between cancer and TME, implicating the potential of CAF-sEVs to be un-invasive diagnostic biomarkers for NSCLC. Despite such new insights, challenges remain exist, particularly concerning the mechanisms that drive CAF plasticity, the interactions between specific CAF subsets with cancer cells or immune cells, and the translational significance of CAF-sEVs. In this review, we summarize the latest advances in our understanding of CAF heterogeneity, the functional roles and clinical relevance of CAFs and their secreted sEVs in driving NSCLC, and the translational landscape of CAF-targeted strategies. By critically evaluating the most recent evidence, this review provides clinically relevant insights and highlights future directions for overcoming CAF-mediated barriers to therapy. Our aim is to facilitate the development of more precise, biomarker-driven, and safe approaches for targeting CAFs, ultimately improving personalized treatment and outcomes for patients with NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82d1bb104cbbe26ccaae9f632691df643466beac" target='_blank'>
              Hidden forces: the impact of cancer-associated fibroblasts on non-small cell lung cancer development and therapy
              </a>
            </td>
          <td>
            Ziheng Wu, Meilin Luo, Huiyi Hu, Zhijun Jiang, Yinan Lu, Zhi-Jie Xiao
          </td>
          <td>2025-07-25</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lineage tracing is a fundamental tool in developmental biology and cancer research, providing critical insights into cell fate decisions, tissue homeostasis and tumor initiation. The mammary gland is a highly dynamic organ with a complex cellular hierarchy, making it an ideal system for lineage-tracing studies. Classic approaches, such as tamoxifen-inducible CreER/loxP recombination, have significantly advanced our understanding of mammary epithelial cell (MEC) differentiation, homeostasis, and transformation. However, these methods have limitations, including potential effects of tamoxifen on estrogen signaling, low mammary gland specificity, and the requirement for transgenic model creation and mouse breeding. Adenovirus-Cre (Ad-Cre)-based lineage tracing has emerged as a powerful alternative, enabling rapid and organ-specific recombination. This review provides a comprehensive evaluation of the Ad-Cre approach in mammary gland biology, comparing its efficiency, specificity, and technical advantages over the CreER-based method. We discuss applications of Ad-Cre intraductal injection-based lineage tracing in mapping MEC fates, identifying the cellular origins of breast cancer, and modeling tumor progression. Additionally, we highlight its ability to induce genetic marking at a clonal level, facilitating precise investigations into MEC plasticity and tumor cell heterogeneity. Despite its advantages, Ad-Cre lineage tracing also presents challenges, such as low cell-targeting efficiency and potential effect on the mammary gland immune microenvironment. Future advancements, including the integration of CRISPR-based barcoding, may further enhance its utility for high-resolution fate mapping. By summarizing recent advancements and comparative analyses, this review underscores the significance of Ad-Cre lineage tracing as a versatile and powerful tool in mammary gland biology and breast cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4426d4775fffc4e3ce84eab2c9b5359f90d37f" target='_blank'>
              Intraductal Injection of Adenoviruses to Perform Lineage Tracing in the Mammary Gland
              </a>
            </td>
          <td>
            Xueqing Chen, Sen Han, Dongyi Zhao, Zhe Li
          </td>
          <td>2025-07-07</td>
          <td>Journal of Mammary Gland Biology and Neoplasia</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0289ee010aab237793e43a5fa9c9591075ed88ec" target='_blank'>
              A clinically relevant morpho-molecular classification of lung neuroendocrine tumours
              </a>
            </td>
          <td>
            A. Sexton-Oates, Emilie Mathian, Noah Candeli, Yuliya Lim, C. Voegele, A. Génova, Laurane Mangé, Zhaozhi Li, Tijmen van Weert, L. Hillen, R. Blázquez-Encinas, A. González-Pérez, Maike L. Morrison, Eleonora Lauricella, L. Mangiante, Lisa Bonheme, L. Moonen, G. Absenger, Janine Altmuller, Cyril Dégletagne, Vincent Cahais, G. Centonze, Amélie Chabrier, C. Cuenin, F. Damiola, V. Montpréville, J. Deleuze, Anne-Marie C. Dingemans, É. Fadel, N. Gadot, A. Ghantous, Paolo Graziano, Paul Hofman, V. Hofman, A. Ibáñez-Costa, S. Lacomme, Núria López-Bigas, M. Lund-Iversen, Massimo Milione, L. Muscarella, S. Pedraza-Arévalo, C. Perrin, G. Planchard, H. Popper, L. Roz, A. Sparaneo, W. Buikhuisen, José van den Berg, M. Tesselaar, Jaehee Kim, Ernst Jan, M. Speel, S. Tabone-Eglinger, Thomas Walter, Gavin M. Wright, Justo P Castaño, Lara Chalabreysse, Liming Chen, Christophe Caux, Marco Volante, Nicolas Girard, J. Vignaud, Esther Conde, A. Mansuet-Lupo, Luka Brčić, Giuseppe Pelosi, Mauro Papotti, Sylvie Lantuéjoul, J. Derks, Talya Dayton, N. Alcala, M. Foll, L. Fernandez-Cuesta
          </td>
          <td>2025-07-18</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Up until present day, chimeric antigen receptor (CAR)-T cell therapy has only been approved for hematological malignancies, as CAR-T cells do not show comparable efficacy in solid tumors. Therefore, understanding the features of the tumor microenvironment (TME), is key to improve efficacy of adoptive cell therapies (ACTs) against solid tumors. In this context, robust workflows, which dissect the complex interactions between CAR-T cells and the TME are still lacking. To address this need, we have established an ex vivo workflow co-culturing tissue slices from patient tumor resections with CAR-T cells. The workflow is composed of assessing several complementary attributes, such as cytokine release via flow cytometry, quantification of cell infiltration into the tumor and assessment of the regions of the tissue slice the CAR-T cell infiltrate into by using the MACSima™ imaging cyclic staining technology. Using this workflow it is possible to observe the behavior of CAR-T cells within the tumor and its TME, their infiltration into distinct tumor compartments, as well as to dissect the underlying molecular mechanisms that drive T cell migration, thanks to MACSima™ multiplexing technology and its ability to image several markers at the same time. Assessment of ovarian carcinoma tissue slices revealed substantial release of specific cytokines and increased infiltration of T cells in the tumor areas when CAR-T cells were added to the tissue slices as compared to non-engineered T cells. The establishment of this novel approach will enable researchers to better characterize the interaction between CAR-T cells and the TME. Tissue slices present an intrinsic heterogeneity, which is indeed an advantage compared to other in vitro models but can turn itself into complex results interpretation. Therefore, we recommend that any conclusion derived from this assay should be verified with complementary models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e80f902b452985b81dfcba4e7339eba4450d5adf" target='_blank'>
              Precision-cut tumor tissue slices, a novel tool to study the tumor microenvironment interactions with chimeric antigen receptor (CAR) T cells
              </a>
            </td>
          <td>
            Valeria Durante, Alina Wittwer, Benjamin Theek, Manuel Martinez-Osuna, E. Donnadieu, O. Hardt, Dominik Eckardt, A. Bosio, Sonja Schallenberg, Christoph Herbel
          </td>
          <td>2025-08-08</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract High‐grade serous ovarian carcinoma (HGSC) is a lethal malignancy characterized by high incidence, mortality, and chemoresistance. However, its molecular drivers are unknown. In this study, spatially resolved proteomics was applied to 1144 formalin‐fixed paraffin‐embedded tissue spots obtained by laser capture microdissection from 10 patients with HGSC and divergent carboplatin‐paclitaxel (CP) responses. Specific sampling revealed stroma‐driven tumour heterogeneity, identifying 642 tumour‐specific and 180 stroma‐specific proteins, with 505 CP‐responsive therapeutic targets. Most of these protein signatures represented previously unreported associations with chemoresistance in HGSCs. Two clinically significant spatial proteomic maps were generated by introducing tumour (TS) and chemical (CS) scores. TS analysis revealed conserved tissue architecture across CP response groups, whereas CS mapping revealed pretreatment metabolic reprogramming (rather than proliferation) as the defining feature of chemo‐resistant tumours, challenging current resistance paradigms. Immunohistochemical validation of HGSC tissue microarrays confirmed the spatial proteomic localization of TFRC and PDLIM3, which are linked to tumour progression, while establishing their novel role as chemotherapy resistance biomarkers through this study, with broader predictive potential observed across additional targets in the discovery cohort. This study developed a spatially resolved proteomic framework to enhance the diagnostic and therapeutic strategies for HGSC. Key points HGSC intra‐tumour heterogeneity is predominantly driven by stroma, as revealed by spatial proteomic compartmentalization (tumour/stroma). Spatial proteomics expands the therapeutic target database, enabling prediction of platinum‐based chemotherapy response. Chemo‐resistant patients exhibit pre‐treatment metabolic activation rather than proliferative signatures. TFRC (iron transport) and PDLIM3 (cytoskeletal remodelling) are spatially validated as chemo‐response biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd4110366f8665c0c5fd0609d17d3715035781c1" target='_blank'>
              Spatially resolved proteomics surveys the chemo‐refractory proteins related to high‐grade serous ovarian cancer
              </a>
            </td>
          <td>
            Linyuan Fan, Yi Liu, Haichao Zhou, Yang Feng, Guangyi Jiang, Guixue Hou, Zhihan Cao, Zhiguo Zheng, Lu Sun, Hao Chen, Yuefei Zhang, Weiran Chen, Yun Xi, Benliang Cheng, Qinghai Yang, Yan Ren, Jianqing Zhu, Siqi Liu
          </td>
          <td>2025-07-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) are functionally diverse stromal regulators that orchestrate tumor progression, metastasis, and therapy resistance through dynamic crosstalk within the tumor microenvironment (TME). Recent advances in single‐cell multiomics and spatial transcriptomics have identified conserved CAF subtypes with distinct molecular signatures, spatial distributions, and context‐dependent roles, highlighting their dual capacity to promote immunosuppression or restrain tumor growth. However, therapeutic strategies struggle to reconcile this functional duality, hindering clinical translation. This review systematically categorizes CAF subtypes by origin, biomarkers, and TME‐specific functions, focusing on their roles in chemoresistance, maintenance of stemness, and formation of immunosuppressive niches. We evaluate emerging targeting approaches, including selective depletion of tumor‐promoting subsets (e.g., fibroblast activation protein+ CAFs), epigenetic reprogramming toward antitumor phenotypes, and inhibition of CXCL12/CXCR4 or transforming growth factor‐beta signaling pathways. Spatial multiomics‐driven combinatorial therapies, such as the synergistic use of CAFs and immune checkpoint inhibitors, are highlighted as strategies to overcome microenvironment‐driven resistance. By integrating CAF biology with translational advances, this work provides a roadmap for developing subtype‐specific biomarkers and precision stromal therapies, directly informing efforts to disrupt tumor‐stroma coevolution. Key concepts include spatial transcriptomics, stromal reprogramming, and tumor‐stroma coevolution, offering actionable insights for both mechanistic research and clinical innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d8f3a8f9589a88ea165cf34934eda0905c9005" target='_blank'>
              Cancer‐Associated Fibroblasts: Heterogeneity, Cancer Pathogenesis, and Therapeutic Targets
              </a>
            </td>
          <td>
            Yutao Li, Qingyun Liu, Xilin Jing, Yuqi Wang, Xiaohua Jia, Xing Yang, Kezhong Chen
          </td>
          <td>2025-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pediatric low-grade gliomas (pLGG) comprise 35% of all brain tumors. Despite favorable survival, patients experience significant morbidity from disease and treatments. A deeper understanding of pLGG biology is essential to identify novel, more effective, and less toxic therapies. We utilized single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and cytokine analyses to characterize and understand tumor and immune cell heterogeneity of pilocytic astrocytoma (PA) and ganglioglioma (GG). scRNA-seq revealed tumor and immune cells within the tumor microenvironment (TME). Tumor cell subsets include both progenitor and mature cell populations. Immune cells included myeloid and lymphocytic cells. There was a significant difference between the prevalence of two major myeloid subclusters between PA and GG. Bulk and single-cell cytokine analyses evaluated the immune cell signaling cascade with distinct immune phenotypes among tumor samples. KIAA1549-BRAF tumors appeared more immunogenic, secreting higher levels of immune cell activators and chemokines, compared to BRAF V600E tumors. Spatial transcriptomics revealed the differential gene expression of these chemokines and their location within the TME. A multi-pronged analysis demonstrated the complexity of the PA and GG TME and differences between genetic drivers that may influence their response to immunotherapy. Further investigation of immune cell infiltration and tumor-immune interactions is warranted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dc01e4fe40878b7885262fd79ea7216b56b949" target='_blank'>
              Multi-pronged analysis of pediatric low-grade glioma and ganglioglioma reveals a unique tumor microenvironment associated with BRAF alterations.
              </a>
            </td>
          <td>
            S. Zahedi, Kent Riemondy, Tian Liu, A. Griesinger, A. Donson, April A Apfelbaum, Rui Fu, Julian Grandvallet Contreras, M. Crespo, J. DeSisto, Madeline Groat, Emil Bratbak, Adam L Green, Todd C Hankinson, Michael Handler, Rajeev Vibhakar, Nicholas Willard, Nicholas K Foreman, Tzu Phang, Jean M. Mulcahy Levy
          </td>
          <td>2025-06-30</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ef3fe6c1c7dcea2e010f5575de7254a23fff3d7" target='_blank'>
              Spatial and single-cell transcriptomics landscape of adenomyosis
              </a>
            </td>
          <td>
            X. Yang, C. Li, J. He, B. Xie, L. Lv, J. Xu, Y. Lin, C. Zhou, M. Li, Z. Zeng, J. Tan, S. Chen, G. Cui, G. Liu, S. Suo, G. Peng, X. Liang
          </td>
          <td>2025-07-07</td>
          <td>None</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Extracellular vesicles (EVs) have emerged as key mediators of intercellular communication, gaining recognition as tumor biomarkers and promising therapeutic targets. As the study of EVs advances, it has become increasingly clear that the cellular context in which they are produced significantly influences their composition and function. Traditional two-dimensional in vitro models are being progressively replaced by more advanced three-dimensional systems, such as tumor spheroids and organoids. These 3D models are particularly valuable in cancer research, providing a more accurate representation of the complex cellular and molecular heterogeneity that characterizes tumors, better mimicking the in vivo microenvironment compared to standard monolayer cultures. This review explores the role of EVs derived from tumor spheroids and organoids in key oncogenic processes, including tumor growth, metastasis, and interactions within the tumor microenvironment. We highlight how EVs contribute to the spread of cancer cells, affecting surrounding tissues, and promote immune evasion, which poses significant challenges in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/038e5b59a9368ce2c0bcde764480d3c277624905" target='_blank'>
              The 3D Language of Cancer: Communication via Extracellular Vesicles from Tumor Spheroids and Organoids
              </a>
            </td>
          <td>
            S. Campora, Alessandra Lo Cicero
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Precursor lesions like polyps and adenomas in the colon commonly precede colorectal cancer (CRC), advancing through the “normal-polyp-adenoma-carcinoma” sequence towards malignancy. Yet, the cellular heterogeneity and molecular mechanisms involved in CRC development remain inadequately characterized. Methods To understand the molecular mechanisms driving the onset and progression of CRC, we conducted a comprehensive analysis of ten clinical colorectal samples representing sequential pathological stages using single-cell RNA sequencing (scRNA-seq). Validation was performed through immunofluorescence and immunohistochemistry analyses in a separate human colorectal tissue cohort. Additional verification was carried out using bioinformatics analyses of public TCGA and GEO datasets. Results Our comprehensive analyses not only reveal the cellular diversity and transcriptomic differences throughout disease progression but also highlight the importance of leveraging ligand–receptor gene expression to distinguish various cell subtypes. Subsequent examination and validation with a larger sample cohort uncover the specific involvement of ligand–receptor genes, transcription factors, immunoglobulin genes, and heat shock genes in regulating immune responses and microenvironment changes during colorectal tumorigenesis. Conclusions Our extensive transcriptome dataset provides valuable insights and acts as a fundamental resource to deepen our understanding of the complex molecular landscape in CRC. This dataset facilitates improved diagnostic accuracy and the creation of more personalized therapeutic approaches. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06785-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e15cc889bc4d3cf181a1a8c15dbc72bc14a7b3" target='_blank'>
              A dynamic molecular landscape in colorectal cancer progression at single-cell resolution
              </a>
            </td>
          <td>
            Jianwen Sheng, Nianshuang Li, Shuai Li, Yuman Ye, Yuanhang Liao, Yupeng Zhang, Huan Chen, Mengxian Tu, Guping Zhong, Chaoliang Xiong, Pan Zheng, Yuting Lei, Zekun He, Xingxing He, Yao Zhang, Lei Wang, Xiaolin Gao, Yin Zhu, Jianping Liu, Huizhen Fan
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Traditional gene expression deconvolution methods assess a limited number of cell types, therefore do not capture the full complexity of the tumor microenvironment (TME). Here, we integrate nine deconvolution tools to assess 79 TME cell types in 10,592 tumors across 33 different cancer types, creating the most comprehensive analysis of the TME. In total, we found 41 patterns of immune infiltration and stroma profiles, identifying heterogeneous yet unique TME portraits for each cancer and several new findings. Our findings indicate that leukocytes play a major role in distinguishing various tumor types, and that a shared immune-rich TME cluster predicts better survival in bladder cancer for luminal and basal squamous subtypes, as well as in melanoma for RAS-hotspot subtypes. Our detailed deconvolution and mutational correlation analyses uncover 35 therapeutic target and candidate response biomarkers hypotheses (including CASP8 and RAS pathway genes).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6b1e17d9e760b6c7495c8a6aea0db799ec80899" target='_blank'>
              Pan-cancer immune and stromal deconvolution predicts clinical outcomes and mutation profiles
              </a>
            </td>
          <td>
            B. Bhinder, Verena Friedl, Sunantha Sethuraman, Davide Risso, Kami E. Chiotti, R. J. Mashl, Kyle Ellrott, Jordan A. Lee, Christopher K. Wong, K. Gyan, Aditya Deshpande, Marcin Imielinski, R. Bareja, Josh Stuart, Myron Peto, K. Hoadley, Alexander J. Lazar, A. Cherniack, Jingchun Zhu, S. Cao, Mark Rubin, Wenyi Wang, Oliver F. Bathe, N. Robine, Li-hua Ding, Peter W. Laird, Wanding Zhou, Hui Shen, V. Thorsson, Jen Jen Yeh, Matthew H. Bailey, D. Zhou, X. Peng, M. Goldman, Yongsheng Li, Anil Korkut, Nidhi Sahni, D. N. Hayes, Michael K. A. Mensah, Ina Felau, Anab Kemal, Samantha J. Caesar-Johnson, John A. Demchok, Liming Yang, M. Ferguson, R. Tarnuzzer, Zhining Wang, J. Zenklusen, Adam Alana Andrea Andrew Angela Benjamin Benjamin Brian Margolin Weinstein Sboner Blair Brooks Berman Raph, Adam Margolin, Alana Weinstein, A. Sboner, Andrew Blair, Angela Brooks, Benjamin Berman, Benjamin Raphael, Brian Craft, Dante S. Bortone, David Heimain, David L Gibbs, David Haan, D. Betel, Duncan McColl, Emek Demir, Faeze Brahman, F. Farshidfar, Gaddy Getz, G. Gao, Gordon Robertson, Huy Dinh, Hyo Young Choi, I. Shmulevich, Ioannis Anastopoulous, Jasmine Yang, Jeff Damrauer, Ken Chen, Lauren Sanders, Lee Cooper, Liang-Bo Wang, Matt Reyna, Mike Noble, Mohammed El-Kebir, Molly Zhang, Nicole Yeager, Paul Little, Richard Moffitt, S. Meier, Teresa Swatloski, T. Knijnenburg, Thomas Matthew, Vicky Chen, Vlado Uzunangelov, Xinghua Lu, Yige Wu, Yulia Newton, Zeya Wang, Paul T. Spellman, O. Elemento
          </td>
          <td>2025-07-04</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="Background Understanding the molecular interactions between cells, tissues or organs is key to understanding the functioning of a biological system as a whole. Results Here, we propose crossWGCNA: a co-expression-based method that identifies highly interacting genes unbiasedly and that we employ to study stroma-epithelium communication in breast cancer. CrossWGCNA can be applied to bulk, single cell and spatial transcriptomics data. We validate it both in silico and experimentally, and we provide a fully documented R package allowing users to employ it. Conclusions The wide applicability and agnostic nature of our tool make it complementary to existing methods overcoming the limitations arising from strong baseline assumptions. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11747-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b943ef5ac17c56205d2bc6ae7f9f20263cb8cd8" target='_blank'>
              Cross-tissue gene expression interactions from bulk, single cell and spatial transcriptomics with crossWGCNA
              </a>
            </td>
          <td>
            Aurora Savino, R. Iannuzzi, L. Avalle, Andrea Lobascio, Francesco Iorio, Paolo Provero, Valeria Poli
          </td>
          <td>2025-07-01</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Despite the recent implementation of immunotherapies into clinical practice of various tumor types, the immunobiology of tumors and in particular the role and clinical relevance of the different immune cell populations and their function still remained unclear. Therefore, an in depth analysis of the complex landscape of immune cell populations and their soluble mediators in the peripheral blood and tumor microenvironment of cancer patients is urgently needed. Mass cytometry has revolutionized the immune phenotyping in particular in settings where simultaneous breadth and detailed characterization of the phenotype and function of immune cell (sub)populations with limited sample size is required, such as monitoring of patients’ response to immunotherapies. Since mass cytometry is a powerful multiplex approach to decipher tumor intrinsic and tumor extrinsic effects of tumor immunotherapies, this review summarizes the use of this technology for determination of the frequency and functional status of immune cell populations within the tumor and in the blood leading to the identification of intratumoral/peripheral immune signatures that might serve as biomarkers (i) for treatment response and/or failure, (ii) for the stratification of tumor patients or (iii) for the identification of novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061718c8dca223c8af052f9bed0ef7bd95b974a1" target='_blank'>
              CyTOF as a suitable tool for stratification and monitoring of cancer patients
              </a>
            </td>
          <td>
            Barbara Seliger, C. Massa
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background: Breast cancer, the most prevalent malignancy among women worldwide, exhibits significant heterogeneity, particularly in the tumor microenvironment (TME), which poses challenges for treatment. Spatial transcriptomics (ST) has emerged as a transformative technology, enabling gene expression analysis while preserving tissue spatial architecture. This provides unprecedented insights into tumor heterogeneity, cellular interactions, and disease mechanisms, offering a powerful tool for advancing breast cancer research and therapy. This review aims to synthesize the applications of ST in breast cancer research, focusing on its role in decoding tumor heterogeneity, characterizing the TME, elucidating progression and metastasis dynamics, and predicting therapeutic responses. We also explore how ST can bridge molecular profiling with clinical translation to enhance precision therapy. The key scientific concepts of review included the following: We summarize the technological advancements in ST, including imaging-based and sequencing-based methods, and their applications in breast cancer. Key findings highlight how ST resolves spatial heterogeneity across molecular subtypes and histological variants. ST reveals the dynamic interplay between tumor cells, immune cells, and stromal components, uncovering mechanisms of immune evasion, metabolic reprogramming, and therapeutic resistance. Additionally, ST identifies spatial prognostic markers and predicts responses to chemotherapy, targeted therapy, and immunotherapy. We propose that ST serves as a hub for integrating multi-omics data, offering a roadmap for precision oncology and personalized treatment strategies in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbbaea20f0037aedc680b32a7558f9b67f94b665" target='_blank'>
              Spatial Transcriptomics Decodes Breast Cancer Microenvironment Heterogeneity: From Multidimensional Dynamic Profiling to Precision Therapy Blueprint Construction
              </a>
            </td>
          <td>
            A. Ma, L. Xiang, Jingping Yuan, Qianwen Wang, Lina Zhao, Honglin Yan
          </td>
          <td>2025-07-24</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca80c0d5c8118b6afa306ae5816ee0be1c3263a" target='_blank'>
              Single-cell transcriptomics reveals hypoxia-driven iCAF_PLAU is associated with stemness and immunosuppression in anorectal malignant melanoma.
              </a>
            </td>
          <td>
            Hao Zhang, Lin Gan, Xiaofei Duan, Baojing Tuo, Hao Zhang, Senbo Liu, Yugui Lian, Enjie Liu, Zhenqiang Sun
          </td>
          <td>2025-07-04</td>
          <td>Journal of gastroenterology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Gliomas are aggressive brain tumors whose infiltrative growth is mediated by intercellular crosstalk. Exosomes, small extracellular vesicles, play a key role in cell–cell communication but are difficult to visualize using conventional microscopy. Performing immunostaining for CD63, a known exosome marker, and using STED microscopy, we demonstrate exosome secretion in primary glioma cells. Applying mathematical deconvolution, we enhance the contrast and resolution for in-depth analysis of STED images. We identify CD63-positive cellular footprints and exosome deposits in the extracellular space. Quantitative analysis shows CD63-positive exosomes ranging 36.55-157.06 nm in size. CD63/actin co-staining demonstrates different actin polymerization states associated with exosomes. In conclusion, STED microscopy coupled with immunostaining allows exosome primary characterization at the single-vesicle level in the cellular spatial context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19403b747c6c798bf26375a3c2009af4ac5fca44" target='_blank'>
              Super-resolution microscopy reveals glioma cell footprints and exosome deposits
              </a>
            </td>
          <td>
            Stefania Petrini, Frédéric Eghiaian, Valentina Apollonio, Giulia Pericoli, Maria Vinci
          </td>
          <td>2025-07-24</td>
          <td>Cell Adhesion & Migration</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The discovery of microbial communities residing within tumors has unveiled a new dimension of cancer biology. In lung cancer, the intratumoral microbiome—comprising bacteria, fungi, and viruses—has emerged as a critical modulator of tumorigenesis, immune evasion, therapeutic response, and metastasis. This review comprehensively examines the landscape of the lung tumor microbiota, highlighting its mechanistic roles in shaping the tumor microenvironment, altering host immune responses, and reprogramming of cancer metabolism. We discuss the influence of specific microbial taxa on immunotherapeutic efficacy, including their interplay with immune checkpoints and pro-inflammatory signaling pathways. Moreover, we evaluate current evidence linking microbial signatures for diagnostic and prognostic applications, emphasizing their potential in biomarker discovery and precision oncology. By integrating findings from molecular epidemiology, multi-omics profiling, and preclinical models, this review provides a translational framework for leveraging the tumor-resident microbiota as both a within tumors, we may develop new microbiome-based strategies. These strategies could improve treatment outcomes and help overcome resistance to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bcc108fb1a40c357b4ea3f2b37ffb067b21a69e" target='_blank'>
              Dissecting the intratumoral microbiome landscape in lung cancer
              </a>
            </td>
          <td>
            Yan Zhao, Zhibo Yang, Dan Wu, Hai Zhao
          </td>
          <td>2025-07-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most aggressive and treatment-resistant brain tumors, largely due to its profound intratumoral heterogeneity and immunosuppressive microenvironment. Various classifications of GBM subtypes were created based on transcriptional and methylation profiles. This effort, followed by the development of new technology such as single-nuclei sequencing (snRNAseq) and spatial transcriptomics, led to a better understanding of the glioma cells’ plasticity and their ability to transition between diverse cellular states. GBM cells can mimic neurodevelopmental programs to resemble oligodendrocyte or neural progenitor behavior and hitchhike the local neuronal network to support their growth. The tumor microenvironment, especially under hypoxic conditions, drives the tumor cell clonal selection, which then reshapes the immune cells’ functions. These adaptations contribute to immune evasion by progressively disabling T cell and myeloid cell functions, ultimately establishing a highly immunosuppressive tumor milieu. This complex and metabolically constrained environment poses a major barrier to effective antitumor immunity and limits the success of conventional therapies. Understanding the dynamic interactions between glioma cells and their microenvironment is essential for the development of more effective immunotherapies and rational combination strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0dda0986360e9c65e565d1dc755ad16cfd3932d" target='_blank'>
              Unraveling Glioblastoma Heterogeneity: Advancing Immunological Insights and Therapeutic Innovations
              </a>
            </td>
          <td>
            Joshua H. Liu, Maksym Horiachok, Santosh Guru, Cecile L. Maire
          </td>
          <td>2025-08-02</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Spatial transcriptomics (ST) enables high-resolution mapping of gene expression across tissue architecture and can reveal the organization of cellular states in complex tissues, such as the tumor microenvironment. Using ST data from glioblastoma (GBM) patients, a large recent study1 identified robust gene programs in a spatially oblivious manner and used them to assign cell states to spatial spots and assess cell-state spatial organization. The results characterized GBM tumors as generally spatially heterogeneous, with some hypoxia-associated structured regions. However, recent computational methods support analyses that more tightly integrate spatial context and gene expression to identify continuous gradients and rare niches in ST data. We hypothesized that applying such an approach may reveal additional key aspects of the spatial organization of transcriptional programs in GBM tumors.



 Based on our previous successes with traditional topic modeling in transcriptomic analysis, we focused on the recently published method Spatial Transcriptomics Analysis with topic Modeling to uncover spatial Patterns (STAMP)2. STAMP uses a deep generative model to perform spatially aware topic modeling that can account for replicates. Using STAMP, we reanalyzed the 26 ST samples (10x Visium, 60,000+ spots) from 17 patients in the GBM dataset1 to infer spatial topics or gene programs, which we then interpreted in the context of previous studies. We also extended the analysis framework to enable quantification and statistical assessment of the spatial coherence of continuous gene program weights.



 Consistent with previous studies, we identified gene programs associated with a hypoxic niche, as well as with precursor and differentiated endothelial, oligodendrocyte and neuronal cell types. However, our study also revealed a number of new gene programs with strongly spatially organized patterns of expression occurring in multiple patient samples. These findings indicate additional spatial axes that may capture important variation in and across GBM tumor microenvironments.



 Complementing a previous hypoxia-associated model of GBM spatial organization, our study characterized additional, spatially resolved functional processes, which may be clinically and therapeutically relevant. The results suggest that a spatially aware, non-categorical approach to ST analysis may drive discovery of previously hidden, spatially coherent, recurrent gene programs in the tumor microenvironment. This framework is broadly applicable for dissecting the transcriptionally governed spatial architecture of solid tumors.



 1Greenwald, A.C. et al. Integrative spatial analysis reveals a multi-layered organization of glioblastoma. Cell 187, 2485–2501.e26 (2024). 2ZHong, C., Ang, K.S. & Chen, J. Interpretable spatially aware dimension reduction of spatial transcriptomics with STAMP. Nat. Methods 21, 2072–2083 (2024).



 Joseph J. Sifakis, Sophia Madejski, Samantha J. Riesenfeld. Spatially aware transcriptomic topic modeling reveals novel signals of spatial organization in glioblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/953df0c913ecb019b9b2578d60ef3030541a85b4" target='_blank'>
              Abstract B050: Spatially aware transcriptomic topic modeling reveals novel signals of spatial organization in glioblastoma
              </a>
            </td>
          <td>
            Joseph J. Sifakis, Sophia Madejski, Samantha J Riesenfeld
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David J Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) tumours are classified by pathology into several distinct subtypes. Gene expression profiling has revealed transcriptional heterogeneity across NEPC, but this is rarely considered in the context of variation between pathologies. Diagnosis typically relies on immunohistochemical markers (CHGA, SYP, NCAM1) and genomic alterations in RB1, PTEN and TP53. We hypothesized that NEPC pathologies have unique transcriptional features. Single-cell RNA sequencing of 18 632 tumour cells from nine patient-derived xenograft models representing five pathologies (small-cell and large-cell neuroendocrine carcinomas, focal neuroendocrine differentiation (Focal NED), low-grade neuroendocrine and amphicrine) demonstrated pathway-specific enrichment. Focal NED and amphicrine tumours exhibited cellular subpopulations enriched for KRAS, IL2-STAT5 and TNF signalling pathways, absent in small- and large-cell carcinomas, which were instead enriched for Myc and E2F pathways. Furthermore, focal NED cells exhibited minimal clonal divergence from adjacent adenocarcinoma cells, while small cell carcinoma cells were clonally distinct. These data underscore significant transcriptional variation among NEPC pathologies, highlighting focal NED's unique biological context and its clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c9094a51a0844d2931b4655b974ca910515f91f" target='_blank'>
              Single-cell transcriptomics redefines focal neuroendocrine differentiation as a distinct prostate cancer pathology.
              </a>
            </td>
          <td>
            Rosalía Quezada Urban, S. Keerthikumar, A. Clark, Hong Wang, B. Phipson, Andrew Bakshi, Andrew Ryan, Heather Thorne, R. Taylor, Mitchell G. Lawrence, G. Risbridger, Roxanne Toivanen, D. L. Goode
          </td>
          <td>2025-07-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Spatial transcriptomics has advanced our understanding of cellular heterogeneity at single-cell resolution. Here, we assess the suitability of in situ sequencing (ISS) for analyzing formalin-fixed, paraffin-embedded (FFPE) postmortem human brain tissue. A key challenge in ISS data analysis is optimizing transcript allocation while minimizing misallocation, particularly in the morphologically complex central nervous system (CNS). We compared geospatial methods using nuclear and expanded nuclear boundaries for segmentation and transcript allocation. While overall cell-type proportions remained comparable, transcript allocation methods affected specific cell types, including microglia, neurons, and neurovascular cells. To enhance specificity, we integrated fluorescent imaging data targeting 18S RNA and IBA1 protein to direct transcript allocation toward RNA-rich cells (e.g., neurons) and microglia, respectively. We demonstrate how this approach, paired with secondary allocation of transcripts outside imaging masks, improved both the number of microglia detected and the specificity of microglial transcripts assigned. Our method offers a flexible and efficient strategy for targeted transcript allocation based on cellular morphology, optimizing CNS cell segmentation in FFPE-preserved human brain tissue.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/865ca3f5d716d0f571d46550eae97052916e2053" target='_blank'>
              Beyond the nuclear border: single-cell analysis of in situ sequenced human brain tissue using cellular features
              </a>
            </td>
          <td>
            J. Kotah, Thomas Rust, H. R. V. van Weering, Janneke Bosma, Amber L. Woudstra, S. Kooistra, B. Eggen
          </td>
          <td>2025-07-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Metastasis causes about 90% of cancer deaths, but the anticipation of its occurrence from primary tumor biopsies remains an intractable clinical dilemma. Spatial organization and biochemical heterogeneity of TME are regulators of metastatic formation but are usually not interrogated by typical diagnostic approaches. To overcome this deficiency, we established a hybrid modeling approach that unites biopsy-derived proteomics, spatial statistical analysis, and machine learning to predict metastatic potential. Employing multiplexed proteomic imaging technologies like imaging mass cytometry, we profiled in situ protein expression patterns in the patient's biopsy tissue. These data were used to calculate spatial features like cell–cell interaction graphs, spatial proximity scores, and neighborhood-based immune signatures. Machine learning classifiers—graph neural networks and random forests—were trained to predict these spatial-biochemical signatures from recognized metastatic endpoints. Our model performed with excellent prediction capability (AUC ≈ 0.90) in a retrovalidation set across various solid cancers. Notably, we also found specific TME signatures that correlate with metastasis: dense foci of PD-L1⁺ macrophages and FOXP3⁺ regulatory T cells at the invasive edge were correlated with a higher risk of metastasis, and abundant CD8⁺ T cell infiltration at the invasive edge with lower progression. This integrated strategy provides a clinically feasible method for predicting metastasis from standard biopsy samples, which could potentially lead to earlier treatment and therapy stratification. Our results highlight the translational value of the combination of spatial proteomics and computational modeling towards improved prognostic precision in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc1c3111d9dc4060886bfbec6b57329f90c0fbb" target='_blank'>
              BIOCHEMICAL AND SPATIAL MODELING OF TUMOR MICROENVIRONMENT FROM BIOPSY PROTEOMICS TO PREDICT METASTATIC PROGRESSION
              </a>
            </td>
          <td>
            PEACE CHINONYEREM IKE, CHIKA GLORIA OKWUOGORI, THEOPHILUS IGWEBUIKE ILOEGBU, ONYEKA MILICENT ASUMAH, ONWUKWE OLUCHI BUCHI, AFOLAYAN OLUWATOSIN SUNDAY
          </td>
          <td>2025-07-10</td>
          <td>International Journal of Medical Biological and Pharmaceutical Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Formalin-fixed paraffin-embedded (FFPE) samples represent a vast, untapped resource for epigenomic research, yet molecular tools for deep analysis of these specimens remain limited. We introduce spatial FFPE-ATAC-seq, an approach for in situ profiling chromatin accessibility within archived tissues. This approach overcomes formalin-induced crosslinking challenges, allowing high-resolution mapping of chromatin landscapes while preserving tissue architecture. Applying spatial FFPE-ATAC-seq to mouse and human tissues, including brain and thymus, reveals intricate spatial organization and distinct cell types in alignment with tissue morphology. Integration with single-cell RNA sequencing validates the precision of our chromatin profiles in identifying key cell types and regulatory elements. We further apply this method to human melanoma, comprehensively characterizing chromatin accessibility across both tumor and non-tumor regions. This method significantly expands the toolkit for epigenomic research, unlocking the potential of an extensive collection of archived FFPE samples for studying gene regulation and disease mechanisms with spatial context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9a8ea3c46aa7a27b26189991ac0f0ce423df34a" target='_blank'>
              Spatial profiling of chromatin accessibility in formalin-fixed paraffin-embedded tissues
              </a>
            </td>
          <td>
            Pengfei Guo, Yufan Chen, Liran Mao, Angelysia Cardilla, Chin Nien Lee, Yan Cui, Dengge Jin, Yucong Hua, Xiaowei Xu, Yanxiang Deng
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Preclinical modeling of epithelial ovarian cancer in immune-competent mice progressing to orthotopic, spontaneous tumors is challenging, requiring multiple genetic modifications in the host. Transplantable models using cell lines are easier to implement than spontaneous animal models, given that they reproduce the key disease characteristics. To create new in vivo ovarian tumor models, we generated 28 murine ovarian cancer cell lines with distinct genetic traits, such as deletion of Trp53, activation of KrasG12D, or deletion of Pten or KrasG12D/Pten−/− combination. Two distinct Trp53 null cell lines recapitulate high-grade serous histology when orthotopically injected into immune-competent, syngeneic hosts. Cells with Pten deletion trigger high-grade endometrioid tumors, and cells with dual KrasG12D activation and Pten deletion model carcinosarcoma. The cells express different tumor antigens, secrete varying levels of cytokines and chemokines, and trigger tumors with diverse inflammation profiles and various intratumoral T- and B-lymphocyte infiltration patterns. RNA-sequencing data from 16 cell lines reveal the gene expression profile across distinct models with different histotypes. This versatile collection of murine cell lines supports translationally relevant studies in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a48d4b41ac26a6f4817b14a453f5343068f2e4c4" target='_blank'>
              Murine cell lines with defined mutations model different histological subtypes of epithelial ovarian cancer
              </a>
            </td>
          <td>
            Lixin Zhang, Yusi Fang, Ibrahim Uygun, Danyang Li, Mary Strange, Syed K Zaidi, Wenjia Wang, Julia Knight, M. Radolec, E. Elishaev, Joan F Brozick, Allison Edwards, George Tseng, Sandra Cascio, Ronald J Buckanovich, Robert P Edwards, A. Vlad
          </td>
          <td>2025-07-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Spatial genomics has revolutionized breast cancer research by mapping gene expression within the tumor microenvironment, offering unprecedented insights into the molecular and cellular interactions that drive tumor progression and therapeutic response across luminal A, luminal B, HER2-positive, and triple-negative subtypes. Unlike traditional genomic methods, spatial genomics preserves tissue architecture, revealing subtype-specific TME features, such as immune-rich regions in triple-negative breast cancer and stromal-dense environments in HER2-positive tumors. This review explores spatial genomics technologies, including Visium, GeoMx, and MERFISH, which enable precise TME analysis, highlighting their role in identifying biomarkers like PD-L1 and TGF-β for targeted therapies and immunotherapies. By elucidating TME-driven resistance mechanisms, such as hypoxia in triple-negative tumors or stromal remodeling in HER2-positive cases, spatial genomics informs clinical trials and combination therapies, enhancing patient stratification and treatment efficacy. Despite its promise, challenges including limited resolution, complex data analysis, and high costs hinder clinical adoption. Emerging technologies, artificial intelligence, and multi-omics integration offer solutions, promising higher precision and scalability. This review underscores spatial genomics’ transformative potential to bridge research and clinical practice, paving the way for personalized breast cancer therapies tailored to the dynamic TME, with future advancements poised to redefine treatment paradigms and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5822524fe77c324fb45f43e4adc21fca11863f06" target='_blank'>
              Unraveling tumor microenvironments with spatial genomics: Implications for precision breast cancer therapy
              </a>
            </td>
          <td>
            Amina Suleiman Kamsalem, Obianujunwa Mercy Ubah, Tobiloba Philip Olatokun, Regina Omodolapo Oluwaniyi, Olivia Ajifa Philips, Damilola Samuel Ojo-Omoniyi, Omolola Lolade Oluwasakinju
          </td>
          <td>2025-07-30</td>
          <td>International Journal of Life Science Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug resistance poses a significant challenge in cancer therapy, contributing to rapid recurrence, disease progression, and high patient mortality. Despite its critical impact, few reliable predictors for cancer drug response or failure have been established for clinical application. Tumor heterogeneity and the tumor microenvironment (TME) are pivotal factors influencing cancer drug efficacy and resistance. Tumor heterogeneity leads to variable therapeutic responses among patients, while dynamic interactions between cancer cells and the TME enhance tumor survival and proliferation, underscoring the urgent need to identify cellular hallmarks for predicting drug response and resistance. Single-cell and spatial omics technologies provide high-resolution insights into gene expression at the individual cell level, capturing intercellular heterogeneity and revealing the underlying pathologies, mechanisms, and cellular interactions. This review delves into the principles, methodologies, and workflows of single-cell and spatial omics in cancer drug research, highlighting key hallmarks involving tumor heterogeneity, TME reprogramming, cell–cell interactions, metabolic modulation, and signaling pathway regulation in drug treatment at single-cell and spatial levels. Furthermore, we synthesize predictive cellular biomarkers for cancer drug response and resistance across 25 cancer types, paving the way for advancements in cancer precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af5a8672965af056452be6aaf8b56f1ff303b62" target='_blank'>
              Application of single-cell and spatial omics in deciphering cellular hallmarks of cancer drug response and resistance
              </a>
            </td>
          <td>
            Xiaoxia Cheng, Ting Peng, Tian Chu, Yiqun Yang, Jia Liu, Qinglei Gao, C. Cao, Juncheng Wei
          </td>
          <td>2025-07-02</td>
          <td>Journal of Hematology & Oncology</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Background Although the significance of intratumoral heterogeneity (ITH) has been widely acknowledged in various cancers, its role in pancreatic cancer (PC) remains underexplored and warrants further investigation. Methods Pancreatic cancer transcriptomic data were acquired from the TCGA and GEO databases. The DEPTH2 algorithm, in combination with differential expression analysis, was used to identify genes associated with intratumoral heterogeneity (ITH). We applied univariate Cox regression analysis and multiple machine learning techniques to establish a reliable prognostic model. Patients were then stratified according to their ITH scores, and differences between subgroups were examined through pathway enrichment analysis, immune cell infiltration profiling, and drug response prediction. Furthermore, we conducted subcellular localization and differential analysis of ITH using single-cell data, followed by cell-cell communication analysis to explore interaction relationships and identify key pathways. Results Patients exhibiting lower intratumoral heterogeneity (ITH) levels demonstrated poorer clinical outcomes. The constructed 11-gene signature successfully differentiated individuals into high- and low-risk categories with significant survival differences. Immune profiling revealed notable differences in immune cell composition between the two groups, with patients in the high ITH cohort exhibiting enhanced immune activation. Drug sensitivity analysis indicated a differential response to therapies, with high-risk patients more resistant to certain drugs. Single-cell RNA sequencing identified a greater ITH score in epithelial cells, highlighting key interactions, particularly involving Galectin signaling pathways. Conclusion Our results highlight intratumoral heterogeneity’s prognostic and therapeutic relevance in pancreatic cancer, suggesting its potential utility in guiding individualized treatment approaches. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03080-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c15cd4c65cc7acb2163700609aefaa2af8b33ee" target='_blank'>
              Integrating bulk and single-cell RNA sequencing reveals intratumor heterogeneity phenotypes and immune infiltration in pancreatic cancer
              </a>
            </td>
          <td>
            Li Zhao, Yao Huang, Jiang Li, Rui Xie, Kaiming Li
          </td>
          <td>2025-07-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4879912b1737fce64c962b382e72c481f6e838" target='_blank'>
              Spatial mapping of proteins and their activity states in cancer models by multiplex in situ PLA
              </a>
            </td>
          <td>
            Liza Löf, Bo Xu, Tanay Kumar Sinha, Caroline Dahlström, J. Vennberg, Tore Larsson Forssén, Axel Klaesson, C. Clausson, Xuan Wang, Ulla Strömberg-Olsson, M. Kamali-Moghaddam, C. Avenel, C. Wählby, Agata Zieba-Wicher, Ulf Landegren
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most aggressive and lethal brain tumor in adults and poses significant challenges to patient survival. This review provides a comprehensive exploration of the molecular and genetic landscape of GBM, focusing on key oncogenic drivers, such as epidermal growth factor receptor (EGFR), platelet-derived growth factor receptor (PDGFR), and the PI3K/AKT/mTOR pathway, which are critical for tumorigenesis and progression. We delve into the role of epigenetic alterations, including DNA methylation and histone modifications, in driving therapy resistance and tumor evolution. The tumor microenvironment is known for its pivotal role in immune evasion, with tumor-associated macrophages, myeloid-derived suppressor cells, and regulatory T cells creating an immunosuppressive niche that sustains GBM growth. Emerging therapies, such as immunotherapies, oncolytic viral therapies, extracellular vesicle-based approaches, and non-coding RNA interventions, are highlighted as promising avenues to disrupt GBM pathogenesis. Advances in precision medicine and innovative technologies, including electric field therapy and locoregional treatments, are discussed for their potential to overcome the blood‒brain barrier and treatment resistance. Additionally, this review underscores the importance of metabolic reprogramming, particularly hypoxia-driven adaptations and altered lipid metabolism, in fueling GBM progression and influencing the therapeutic response. The role of glioma stem cells in tumor recurrence and resistance is also emphasized, highlighting the need for targeted therapeutic approaches. By integrating molecular targeting, immune energetics, and technological advancements, this review outlines a multidisciplinary framework for improving GBM treatment outcomes. Ultimately, the convergence of genetic, metabolic, and immune-based strategies offers transformative potential in GBM management, paving the way for increased patient survival and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1445c096bc679931586564b48bbb15167d4847f8" target='_blank'>
              Glioblastoma at the crossroads: current understanding and future therapeutic horizons
              </a>
            </td>
          <td>
            Shilpi Singh, Devanjan Dey, Debashis Barik, Iteeshree Mohapatra, Stefan Kim, Mayur Sharma, Sujata Prasad, Peize Wang, Amar Singh, Gatikrushna Singh
          </td>
          <td>2025-07-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4bb0e4b1912ff07e936c216e052a79b324c47ba" target='_blank'>
              Mechanistic modeling and machine learning identifies optimum radiotherapy schedules to prevent treatment-induced metastasis
              </a>
            </td>
          <td>
            Christopher J Graser, Zhi Zhou, Manuel Schürch, Graydon Moorhead, Justin Dean, Cindy Lin, Jamie Dean, David E. Kozono, G. Lahav, Franziska Michor
          </td>
          <td>2025-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The dysregulated and fibrotic tumor microenvironment of hepatocellular carcinoma (HCC) delays diagnosis and presents many complex signals that drive disease progression. To better recapitulate this microenvironment, we have enhanced our established protein microarray platform by integrating design of experiments (DoE) methodology with high-throughput cell microarray screening. This innovative approach systematically interrogates the intricate roles of matrix stiffness (spanning healthy and fibrotic conditions), extracellular matrix (ECM) composition, and protein concentration, while simultaneously examining their interdependent interactions. By leveraging DoE principles, we were able to explore 117 unique microenvironments on a single microscope slide, ultimately generating a comprehensive dataset of 234 different microenvironments without compromising statistical rigor. Our enhanced screening system enabled the identification of unique microenvironmental interactions critically significant in dictating cellular responses, including adhesion, survival, proliferation, epithelial-to-mesenchymal transition, and drug resistance markers. Utilizing advanced statistical techniques such as linear models and principal component analysis, we characterized phenotypic clusters defined by precise microenvironmental cues. This work presents a robust, high-throughput microarray screening system that comprehensively explores the contributions of 9 physiologically relevant extracellular matrix proteins and matrix stiffness in modulating cellular behavior and disease progression through a methodologically sophisticated and statistically sound approach. .">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d6d43b14b77f59b187769ee40b324e60d7e33b" target='_blank'>
              Enhanced combinatorial analysis of tumor cell-ECM interactions using design-of-experiment optimized microarrays.
              </a>
            </td>
          <td>
            Hannah R C Kimmel, Allison L Paxhia, Zahra Adamji, Gregory H Underhill
          </td>
          <td>2025-07-24</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/021065c3a42b66477bb5801231e24d3f3a77bd0a" target='_blank'>
              Decoding menopause-induced tissue fibrosis using pan-tissue network inference
              </a>
            </td>
          <td>
            Hirotaka Iijima, Atsushi Yamashita, J. Galloway, Nam Vo, Haksoo Choi, F. Ambrosio
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and aggressive primary malignant brain tumor in adults, with limited survival outcomes due to tumor recurrence, mainly driven by GBM cell invasion and therapy resistance. Although temozolomide (TMZ) remains the standard-of-care chemotherapeutic, its long-term efficacy is often compromised by rapid emergence of acquired resistance, largely mediated by the DNA repair enzyme, methylguanine methyltransferase (MGMT). To investigate the interplay between tumor heterogeneity, drug resistance, and the extracellular matrix (ECM) microenvironment, we adapted a 3D methacrylamide-functionalized gelatin (GelMA) hydrogel model to study the behavior of mixed populations of TMZ-sensitive and TMZ-resistant GBM cells. Using both single-cell distributions and multicellular spheroids, we report the impact of heterogeneous cell populations and TMZ dosing regimens, including physiological, supraphysiological, and metronomic TMZ schedules, on drug response and migration. We show that the combination therapy of TMZ with an MGMT inhibitor, lomeguatrib, can modulate TMZ resistance in vitro. This hydrogel model enables systematic investigation of GBM heterogeneity, “go-or-grow” phenotypic plasticity, and therapeutic resistance in an ECM-rich microenvironment, offering a valuable platform for future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbe93715d79528e499ebbccc2bfd3c31f1cbde8" target='_blank'>
              Multicellular Model of Temozolomide Resistance in Glioblastoma Reveals Phenotypic Shifts in Drug Response and Migratory Potential
              </a>
            </td>
          <td>
            Victoria Kriuchkovskaia, Ela K. Eames, Sydney A McKee, Paul J. Hergenrother, Rebecca B. Riggins, Brendan A. C. Harley
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f63086cbccde700980caecaca85ce93b724792a" target='_blank'>
              Neoadjuvant immunotherapy promotes the formation of mature tertiary lymphoid structures in a remodeled pancreatic tumor microenvironment.
              </a>
            </td>
          <td>
            Dimitri N. Sidiropoulos, Sarah M. Shin, Meredith Wetzel, Alexander Girgis, Daniel Bergman, Ludmila Danilova, Susheel Perikala, Daniel H. Shu, Janelle M. Montagne, Atul Deshpande, J. Leatherman, Lucie Dequiedt, Victoria Jacobs, Aleksandra Ogurtsova, Guanglan Mo, Xuan Yuan, Dmitrijs Lvovs, G. Stein-O’Brien, M. Yarchoan, Qingfeng Zhu, Elizabeth I. Harper, A. Weeraratna, A. Kiemen, E. Jaffee, Lei Zheng, W.J. Ho, Robert A. Anders, E. Fertig, L. T. Kagohara
          </td>
          <td>2025-08-15</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) has emerged as a powerful tool to map gene expression patterns to the local tissue structure in cancer, enabling unprecedented insights into cellular heterogeneity and tumour microenvironments. As the technology matures, developing new, spatially informed analytical frameworks will be essential to fully leverage its potential to elucidate the complex organisation and emerging properties of cancer tissues. Here, we highlight key challenges in cancer spatial transcriptomics, focusing on three emerging topics: (a) defining cell states, (b) delineating cellular niches and (c) integrating spatial data with other modalities that can pave the way towards clinical translation. We discuss multiple analytical approaches that are currently implemented or could be adapted in the future in order to tackle these challenges, including classical biostatistics methods as well as methods inherited from geospatial analytics or artificial intelligence. In the rapidly expanding landscape of ST, such methodologies lay the foundation for biological discoveries that conceptualise cancer as an evolving system of interconnected niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9031fde1ffd2ce70d2c223e1b5d79202f9418532" target='_blank'>
              Decrypting cancer's spatial code: from single cells to tissue niches.
              </a>
            </td>
          <td>
            Cenk Celik, Shi Pan, Eloise Withnell, Hou Wang Lam, M. Secrier
          </td>
          <td>2025-07-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer cell morphology reflects the dynamic interplay between genetic, epigenetic, and environmental factors that
drive tumor progression. This review explores the morphological evolution of cancer cells as a phenotypic manifestation of
underlying molecular dynamics, drawing on recent advances in evolutionary biology, cytoskeletal remodeling, computational
pathology, and tumor metabolism. Clonal evolution, both gradual and punctuated, generates morphological diversity that is
shaped by selective pressures from the tumor microenvironment and therapeutic interventions. Studies highlights the structural
and mechanical properties, such as reduced cellular stiffness and altered actin organization, are linked to metastatic potential
and cytoskeletal deregulation. Oncogenic signaling pathways (e.g., PI3K/AKT, Ras/MAPK) and metabolic reprogramming
further modulate cell shape and behavior. High-throughput imaging and machine learning have enabled quantification of both
static and dynamic morphological traits, correlating them with chromosomal instability and clinical outcomes. Morphological
heterogeneity has emerged as a powerful diagnostic and prognostic biomarker, with deep learning-derived metrics providing
scalable tools for cancer classification and risk stratification. Collectively, these insights underscore the clinical utility of
integrating morphological analysis with molecular profiling, offering new directions for personalized cancer diagnosis and
therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055067ac5bdde412ce5a95b6486dff3c99db46a4" target='_blank'>
              Morphology Matters: A Multidimensional Perspective on Cancer Cell Biology
              </a>
            </td>
          <td>
            Rose Rani
          </td>
          <td>2025-06-30</td>
          <td>International Journal for Research in Applied Science and Engineering Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutated oncogenic Kirsten rat sarcoma virus (KRAS) antigen is expressed in a large variety of cancers, including pancreatic, colorectal, and pulmonary cancers. The oncogenic KRAS mutations cause malignancies and are usually ubiquitously expressed by all cells of a tumor. The KRAS amino acid substitutions at the positions 12 or 13 are among the most frequent mutations in human cancers. Here, we developed immuno-oncotherapeutic non-integrative lentiviral vectors encoding a segment encompassing KRASG12D, either alone, or associated with antigen carriers. These carriers can improve the intracellular antigen routing to major histocompatibility complex presentation machineries or provide universal helper CD4+ epitopes. Immunotherapy with one of these vectors resulted in significant immune control of tumor growth in colorectal or pulmonary preclinical cancer models, in several murine genetic backgrounds. The antitumor effect was correlated with increased proportions of intra-tumoral hematopoietic cells and notably CD8+ T cells. Although this effect was partial, it was robust, reproducible and advantageously combinable with conventional chemotherapies and immunotherapies to improve antitumor protection. Therefore, this approach shows promise as an immuno-oncotherapy against KRAS-mediated malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61951127f6ba1a1f776a4ed744c835e44ef0de6c" target='_blank'>
              A lentiviral vector targeting a KRAS neoepitope for cancer immunotherapy
              </a>
            </td>
          <td>
            Anastasia Goloudina, Fabien le Chevalier, P. Authié, Sylvain Ciret, K. Nemirov, Ingrid Fert, F. Moncoq, B. Vesin, A. Noirat, Catherine Blanc, Yu Wei, Pierre Charneau, L. Majlessi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Despite substantial progress in discovering novel therapeutic targets and compounds, cancer hyperproliferative cells (CHCs) play a central role in driving tumor progression in gastric cancer (GC). High-resolution single-cell RNA sequencing (scRNA-seq) integrated with spatial transcriptomics sequencing (ST-seq) elucidating the role of the CHC microenvironment in driving GC progression remains lacking. Methods Through integrated analysis of 40 scRNA-seq samples and 6 ST samples from human gastric tissues spanning tumor progression, we delineate the phenotypic plasticity of tumor epithelium and characterize the transcriptional trajectory from normal gastric cells to CHCs, a process associated with RRM2 overexpression. Finally, we validated the analysis results through cellular and animal experiments. Results Our findings demonstrate that RRM2 knockdown triggers ferroptosis in GC cells by promoting lipid peroxidation and intracellular iron accumulation. We further screen a compound and RRM2 inhibitor, osalmid, which effectively suppresses tumor growth in GC cell xenografts by triggering ferroptosis, thus offering a new therapeutic option for GC. Conclusions This study elucidates the critical role of RRM2 + CHCs in gastric cancer development and progression, providing novel insights for targeted therapies. The discovery of osalmid as a potential therapeutic agent offers a promising new treatment strategy for GC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06847-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad62270a5ce806049ffe5312712ee483297a8c1" target='_blank'>
              Integration of scRNA-seq and ST-seq identifies hyperproliferative RRM2+ cells features and therapeutic targets in gastric cancer
              </a>
            </td>
          <td>
            Shuai Ping, Xiong Jia, Yanan Tian
          </td>
          <td>2025-07-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cutaneous melanoma is an aggressive cancer with an increasing incidence worldwide, highlighting the need for research into its pathogenesis. The tumor microenvironment (TME) plays a critical role in melanoma progression and consists of cellular components and an extracellular matrix (ECM) rich in cytokines and signaling molecules. The most abundant stromal cells within the TME are cancer-associated fibroblasts (CAFs), which remodel the ECM and modulate immune responses. Among immune cells, tumor-associated macrophages (TAMs) predominate, and their polarization toward the M2 phenotype supports tumor progression. Tumor-infiltrating lymphocytes (TILs) have diverse functions, including cytotoxic T-cells, helper T-cells that modulate immune response, B-cells forming tertiary lymphoid structures (TLS), and regulatory T-cells with immunosuppressive properties. Dendritic cells (DCs) also play a complex role in the TME. A notable subpopulation are mature regulatory dendritic cells (mregDCs), which contribute to immune evasion. All of these TME components may drive tumorigenesis. Advancements in melanoma treatment—including immunotherapy and targeted therapies—have significantly improved outcomes in advanced-stage disease. In parallel, emerging approaches targeting the tumor microenvironment and gut microbiome, as well as personalized strategies such as neoantigen vaccines and cell-based therapies, are under active investigation and may further enhance therapeutic efficacy in the near future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00d1c207a44eb08e471b7d10318dde99844e3005" target='_blank'>
              Tumor Microenvironment in Melanoma—Characteristic and Clinical Implications
              </a>
            </td>
          <td>
            Hubert Sikorski, M. Żmijewski, Anna Piotrowska
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is a highly malignant brain tumor with poor outcome, despite current therapies. CIITA-modified tumor cells, capable of presenting antigens via MHC-II, show promise in eliciting effective antitumor immune responses. The efficacy of the CIITA-based vaccination strategy has been previously demonstrated using the GL261 murine GBM model. This study aimed at verifying whether a different GBM model, the CT-2A cells, more closely resembles human GBM, could follow the same rule and importantly share cross immunity with GL261. Methods To assess tumor growth and immune response, immunocompetent mice were injected intracranially with either CT-2A or CIITA-modified CT-2A (CT-2A-CIITA) cells, and brain tissues were examined histologically. Additionally, mice vaccinated with GL261-CIITA cells were subsequently challenged with parental CT-2A cells in the opposite hemisphere, followed by immunohistochemical analysis of brain tissues. Results Results indicated that CT-2A-CIITA cells were either rejected or exhibited delayed tumor growth in immunocompetent mice. Furthermore, mice previously vaccinated with GL261-CIITA cells demonstrated efficient rejection of CT-2A tumors, suggesting the induction of cross-protective immunity. This immune response was associated with a shift from an immunosuppressive to an immunogenic tumor microenvironment, implying the development of adaptive immunity against shared tumor antigens. Conclusions In conclusion, these findings support the broader applicability of CIITA-based vaccination against GBM and provide the first evidence of shared immunogenic antigens between two distinct murine GBM models. This discovery opens avenues for identifying common tumor antigens through immunopeptidomics, which could inform future immunotherapeutic strategies for glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06816-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5557b03a9f988e993ff7c76c2eef46dc1300f8a6" target='_blank'>
              CIITA-modified glioblastomas vaccinate and induce cross-protection against heterologous wild-type glioblastomas
              </a>
            </td>
          <td>
            A. Gatta, A. K. Shaik, Mariam Shallak, A. Chiaravalli, Michele Cerati, Stefano La Rosa, R. Accolla, G. Forlani
          </td>
          <td>2025-07-08</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Endometriosis is often diagnosed late and presents significant challenges in clinical treatment. A comprehensive investigation of the cellular classification and composition of endometriosis is essential for studying its diagnosis and treatment. Methods This study utilized the Gene Expression Omnibus (GEO) public database and referenced single-cell RNA sequencing (scRNA-seq) atlases. The CIBERSORTx algorithm was applied to perform deconvolution on the samples and estimate the proportions of endometrial cell subtypes. A random forest model was constructed to predict the diagnosis of endometriosis. Additionally, immunohistochemical validation was performed on the marker genes of MUC5B+ epithelial cells and dStromal late mesenchymal cells, which showed high diagnostic contribution. Results Endometriosis consists of 5 major cell types, further classified into 52 distinct cell subtypes. Compared to healthy controls, these subtypes exhibited varying degrees of alterations, with MUC5B+ epithelial cells, dStromal late mesenchymal cells, and M2 macrophages showing an increasing trend. Enriched signaling pathways were primarily associated with epithelial-mesenchymal transition (EMT), cell migration, and inflammatory responses. A random forest model, based on cell-type proportions, has been shown to achieve excellent diagnostic performance (AUC = 0.932), with MUC5B+ epithelial cells identified as the top predictive feature. Immunohistochemical validation confirmed high expression of the marker genes MUC5B and TFF3. Conclusion By integrating single-cell and bulk transcriptomics, we identified MUC5B+ epithelial cells and dStromal-late mesenchymal cells as dual drivers of fibrosis and inflammation in endometriosis. Our findings revealed that MUC5B+ epithelial cells may serve as the top factor for the diagnosis of endometriosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99fd62628ca18755a361b8c7c59b2f9cf5de62a" target='_blank'>
              Integrated analysis of single-cell and bulk transcriptomic data reveals altered cellular composition and predictive cell types in ectopic endometriosis
              </a>
            </td>
          <td>
            Meihong Chen, Liqun Wang, Yuanting Chen, Ting Wang, Guanqun Jiang, Qi Chen
          </td>
          <td>2025-07-18</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common primary brain cancer. It causes death mainly by local invasion via several routes, including infiltration of white matter tracts and penetration of perivascular spaces. However, the pathways that mediate these invasion routes are only partly known. Here, we conduct an integrative study to identify cell states and central drivers of route-specific invasion in GBM. Combining single-cell profiling and spatial protein detection in patient-derived xenograft models and clinical tumor samples, we demonstrate a close association between the differentiation state of GBM cells and their choice of invasion route. Computational modeling identifies ANXA1 as a driver of perivascular involvement in GBM cells with mesenchymal differentiation and the transcription factors RFX4 and HOPX as orchestrators of growth and differentiation in diffusely invading GBM cells. Ablation of these targets in tumor cells alters their invasion route, redistributes the cell states, and extends survival in xenografted mice. Our results define a close association between GBM cell differentiation states and invasion routes, identify functional biomarkers of route-specific invasion, and point toward targeted modulation of specific invasive cell states as a therapeutic strategy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bd579d99eaa69724a2c2c47d55f912d2748b0a9" target='_blank'>
              The invasion phenotypes of glioblastoma depend on plastic and reprogrammable cell states
              </a>
            </td>
          <td>
            Milena Doroszko, Rebecka Stockgard, Irem Uppman, Josephine Heinold, Faidra Voukelatou, H. Mangukiya, T. Millner, Madeleine Skeppås, Mar Ballester Bravo, Ramy Elgendy, Maria Berglund, Ludmila Elfineh, C. Krona, Soumi Kundu, Katarzyna Koltowska, Silvia Marino, Ida Larsson, Sven Nelander
          </td>
          <td>2025-07-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer cells orchestrate the surrounding tumor microenvironment (TME) to strike a fine balance between tissue regeneration providing them with nutrients, and tissue destruction triggered by immunogenic alarm signals. At steady state, the tenuous balance favors cancer growth. Therapies aimed at enhancing the immunogenic properties of cancer cells or the reacting immune responses can, however, revert the equilibrium to clear the host of cancer. The understanding of factors that affect this balance is progressing rapidly due to advances in high throughput technologies disclosing from previously uncharted territories new biologies referred to as “dark matter”. These advances are critical for the understanding of the true mechanisms leading to immune-mediated cancer rejection. This review focuses on cancer genetic, epigenetic and metabolic derangements that approximate those caused by intra-cellular pathogen infection, a phenomenon referred to as “viral mimicry” (VM) and other aspects of cancer/host cells interactions unexplored in the past that enhance the VM effects. On the cancer side, VM prompts alterations of cancer cell metabolism leading to the generation of aberrant cellular products recognized as foreign by the host’s immune system. The latter are defined as “dark matter” to emphasize the powerful effects exerted by these obscure bioproducts on the TME as the mass of invisible particles can dictate the rotational period of galaxies. On the other side, a myriad of previously unappreciated factors can influence the host responses. Thus, here we propose an extended definition of dark matter beyond the limits of cancer cell-intrinsic biology, to a broader interpretation encompassing elements that influence the cellular networks within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a42b4f37f1c7313faa25f37bb036aac0202e1c7" target='_blank'>
              The dark matter in cancer immunology: beyond the visible– unveiling multiomics pathways to breakthrough therapies
              </a>
            </td>
          <td>
            Salvatore De Rosis, Gianni Monaco, Joyce Hu, Erick Hett, R. Lappano, Francesco M. Marincola, A. Asadi, M. Maggiolini
          </td>
          <td>2025-07-22</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="BACKGROUND
Atypical teratoid rhabdoid tumors (ATRTs) are highly aggressive pediatric central nervous system tumors defined by the inactivation of the SMARCB1 gene. Despite the identification of three distinct molecular subtypes, each defined by unique clinical and molecular characteristics, no subtype-specific therapeutic strategies are currently available. This highlights an urgent need to deepen our understanding of the cellular heterogeneity and developmental origins of ATRTs.


METHODS
We generated a comprehensive single-nucleus transcriptomic atlas of ATRT samples, integrated it with single-nucleus ATAC-seq and spatial transcriptomics data, and validated our findings experimentally using patient-derived ATRT tumoroid models.


RESULTS
Our analyses revealed distinct subtype-specific differentiation trajectories, each resembling different brain progenitor lineages. We identified key transcription factors that appear to drive these developmental pathways. Furthermore, a shared cycling, intermediate precursor cell (IPC)-like cell population, interspersed throughout tumors, was consistently present within all ATRT samples. We demonstrate that these subtype-specific differentiation pathways can be pharmacologically manipulated in patient-derived ATRT tumoroids. By directing tumor cells along their respective subtype-specific trajectories, we were able to induce a shift toward more differentiated, non-proliferative states.


CONCLUSIONS
Collectively, our findings show that ATRTs recapitulate fetal brain signaling programs in a subtype-specific manner. This work provides a framework for understanding ATRT heterogeneity and supports the feasibility of maturation-based therapeutic strategies tailored to the molecular subtype of the tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f089b7cc6f0aae306ce7c5e4e459d6bc20cd73eb" target='_blank'>
              A cycling, progenitor-like cell population at the base of atypical teratoid rhabdoid tumor subtype differentiation trajectories.
              </a>
            </td>
          <td>
            Enrique Blanco-Carmona, Irene Paassen, Jiayou He, Jeff DeMartino, Annette Büllesbach, Nadia Anderson, Juliane L Buhl, A. Federico, Monika Mauermann, Mariël Brok, Karin Straathof, S. Behjati, Rajeev Vibhakar, A. Donson, Nicholas K Foreman, McKenzie L. Shaw, M. Frühwald, A. Korshunov, M. Hasselblatt, C. Thomas, Niels Franke, M. Kranendonk, E. Hoving, N. Jäger, P. Johann, Stefan M Pfister, Mariella G. Filbin, M. Kool, J. Drost
          </td>
          <td>2025-07-28</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Gliomas, particularly IDH-wildtype ones, are associated with poor prognosis, yet their immunological landscape remains uncertain. We analyzed RNA sequencing data from 55 glioma patients, estimating immune infiltration with CIBERSORTx and immune cell states via Ecotyper. IDH-wildtype gliomas showed significantly higher immune cell infiltration (p = 0.002), notably of regulatory T cells (Tregs) and macrophages, and a greater proportion of exhausted T cells compared to IDH-mutant gliomas. Clustering based on immune profiles revealed two groups. Cluster A, enriched for IDH-wildtype cases, exhibited heightened immune infiltration but also marked immunosuppression. Cluster B, which included both IDH-wildtype and mutant cases, showed lower levels of immune infiltration. Tumor-infiltrating lymphocyte (TIL) cultured from IDH-wildtype tumors demonstrated limited expansion following anti-PD-1, a CSF1R inhibitor, or a STAT3 inhibitor treatment, without clear cluster-specific differences. Tumor-reactive TILs were mainly observed in cluster A. These findings highlight that IDH-wildtype gliomas have an immunosuppressive and heterogeneous microenvironment, potentially limiting responses to single-agent immunotherapies. A personalized, multi-targeted approach addressing multiple immunosuppressive mechanisms may be essential to improve immunotherapy outcomes in this aggressive glioma subgroup.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb950007f7a53c8e4f48f1b9d238f1ea25d6914" target='_blank'>
              Dissecting the Immunological Microenvironment of Glioma Based on IDH Status: Implications for Immunotherapy
              </a>
            </td>
          <td>
            Miyu Kikuchi, Hirokazu Takami, Yukari Kobayashi, K. Nagaoka, Yosuke Kitagawa, Masashi Nomura, Shunsaku Takayanagi, S. Tanaka, Nobuhito Saito, Kazuhiro Kakimi
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Background Colorectal cancer (CRC) ranks among the most prevalent malignant tumours of the digestive system globally and is associated with unfavourable survival outcomes. The exhaustion of CD8⁺ T cells serves a crucial role in facilitating tumour immune escape. Yet, the dynamic evolution of CD8⁺ T cell exhaustion and its impact on clinical prognosis across TNM (tumour‐node‐metastasis) stages in CRC remains incompletely characterized. Methods Tumour and adjacent tissues (20 samples total) from 6 CRC patients spanning diverse TNM stages were analyzed using integrated single‐cell transcriptomic profiling (scRNA‐seq), single‐cell T cell receptor/B cell receptor sequencing (scVDJ‐seq), and spatial transcriptomics. T cell exhaustion markers, immune clonality, gene expression profiles, and the spatial distribution of both tumour cells and immune cells were systematically profiled. Functional enrichment and intercellular communication analyses were conducted. Key findings were validated using immunofluorescence and public datasets. Results Our results illustrate how advancing TNM stages in CRC shape CD8⁺ T cell exhaustion through divergent TNFRSF18/CXCL13 dynamics and ribosomal stemness. TNFRSF18 expression was notably higher in T cells infiltrating tumour tissues relative to their counterparts in adjacent non‐tumorous areas, with high‐expressing CD8⁺ T cells exhibiting marked exhaustion features. During CRC progression, TNM‐stage‐driven remodelling of the tumour microenvironment (TME) induced progressive CD8⁺ T cell exhaustion marked by declining TNFRSF18 and rising CXCL13 expression in tumour‐infiltrating T cells elevation of both markers in the tumour compared with adjacent tissues. Moreover, we show that tumour cells displayed elevated expression of stemness‐associated ribosomal genes (RPS7, RPL8, RPL30), peaking at stage T4, which correlated with poor prognosis and immune escape. Conclusions This integrative multi‐omics study uncovers CD8⁺ T cell exhaustion dynamics and ribosomal stemness‐mediated immune evasion across CRC progression. CXCL13, TNFRSF18, and ribosomal proteins (RPS7/RPL8/RPL30) are identified as novel biomarkers with direct prognostic value and therapeutic relevance, providing therapeutic targets for precision immunotherapy in CRC. Key points   Multi‐omics analysis reveals dynamic CD8+ T cell exhaustion patterns across CRC samples with different TNM stages. TNFRSF18 is highly expressed in exhausted tumour‐infiltrating CD8+ T cells and declines with disease progression. Ribosomal stemness in tumour cells promotes immune evasion by impairing TNF‐mediated CD8+ T cell function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85694b81f9cc8075cdca1110b58ea7303ea2c1a1" target='_blank'>
              Multi‐omics profiling identifies TNFRSF18 as a novel marker of exhausted CD8⁺ T cells and reveals tumour‐immune dynamics in colorectal cancer
              </a>
            </td>
          <td>
            Tengfei Jia, Yingxi Guo, Xin Meng Cheng, Zeyang Zhou, Xiaojiang Xu, Hebin Liu, Xiaodong Yang
          </td>
          <td>2025-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common and deadly malignancies worldwide, with a particularly low 5-year survival rate in advanced patients. Immune cells in the tumor microenvironment, especially mast cells, play crucial roles in tumor initiation and progression. However, the dual role of mast cells in CRC remains poorly understood. Methods In this study, we used single-cell RNA sequencing (scRNA-seq), bulk RNA sequencing, and bioinformatics analyses to explore the heterogeneity of mast cell subpopulations in the CRC tumor microenvironment and their relationship with prognosis. We analyzed gene expression signatures associated with mast cell subpopulations derived from single-cell data of 40 CRC tumor samples and combined bulk RNA-seq data from HMU, GEO, and TCGA cohorts for prognostic prediction. Non-negative matrix factorization was used for clustering of mast cell subpopulations, followed by analysis of their specific gene markers, transcription factor activity, and biological pathways. Survival analysis and ROC curves were performed to assess their prognostic significance. Results Mast cells in the CRC tumor microenvironment were classified into three distinct subpopulations, each with unique gene markers and functional pathways. Mast cell subpopulations 1 and 3 were highly associated with pro-tumor pathways, while mast cell subpopulation 2 primarily exhibited anti-tumor immune regulatory characteristics. High expression of mast cell subpopulations 1 and 3 was associated with poor survival prognosis, while high expression of subpopulation 2 was linked to a better survival outcome. Key marker genes such as DNAJB1, SEMA7A, and XCR1 were identified as potential prognostic factors, with high expression of DNAJB1 and SEMA7A being significantly associated with poor prognosis, while high expression of XCR1 was linked to a favorable prognosis. Conclusion This study reveals the functional heterogeneity of mast cell subpopulations in the CRC tumor microenvironment and their differential roles in tumor progression. Identification of mast cell subpopulation-specific marker genes provides new molecular targets for clinical diagnosis, prognostic prediction, and personalized immunotherapy in CRC. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04119-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f36d01ec68db5890c180d0142a0a3d3d4bc0a598" target='_blank'>
              Diversity of mast cell subpopulations in the tumor microenvironment of colorectal cancer and their prognostic implications
              </a>
            </td>
          <td>
            Tian-yu Qiao, Chao Ding, Songtao Yu, Wenyang Li, Yonghou Zhao, Guiyu Wang
          </td>
          <td>2025-06-30</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Polyploid giant cancer cells (PGCCs) represent a distinct subpopulation of tumor cells characterized by enlarged or multiple nuclei and aneuploidy. PGCCs are products of genomic instability, possessing cancer stem cell properties and exhibiting significant resistance to radiotherapy and chemotherapy. They can generate highly invasive daughter cells through asymmetric division, exhibiting epithelial-mesenchymal transition characteristics, and facilitating tumor recurrence and metastasis. In vivo, PGCCs with daughter cells in tumor tissue can migrate and infiltrate into the forefront stroma to form tumor budding, which are closely related to solid tumor recurrence, metastasis, and drug resistance. Studies have shown that inhibiting sphingolipid enzyme acid ceramidase or regulating autophagy can reduce the production of PGCCs with daughter cells. Under appropriate induction conditions, PGCCs with daughter cells can be induced to differentiate into benign tissues such as adipocytes, chondrocytes, and osteocytes, inhibiting their malignant proliferation and invasive destruction. This study reviewed the recent research developments regarding PGCCs, mainly explored the endogenous mechanisms of PGCCs formation and their malignant phenotype, as well as the process of tumor budding formation in vivo and potential therapeutic strategies targeting PGCCs. The main novelty of this study lies in exploring the translation of PGCCs basic research into the clinical pathological prognostic role of tumor budding, which can reveal the potential mechanism of PGCCs/tumor budding formation at the molecular level, providing theoretical basis for prognosis assessment, monitoring of recurrence and metastasis risks, as well as improving drug resistance and targeted therapy in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a3661c8ef6e9fb0fa2a5fd9bf5fffcb0f097f8" target='_blank'>
              Polyploid giant cancer cells and tumor budding: translation from basic research to clinical application
              </a>
            </td>
          <td>
            Peng Huang, Rong Wu, Zhimou Yang, Yuwei Li, Fei Fei, Yongjun Yu
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor characterized by an immunosuppressive microenvironment that importantly contributes to treatment resistance. Monocyte-derived macrophages (MDMs), which comprise approximately 50% of the cellular population within the GBM microenvironment, represent a major subset of tumor-associated macrophages. These cells drive tumor progression by promoting angiogenesis, immune evasion, and the phenotypic transformation of tumor cells. MDM infiltration is mediated by specific signaling pathways and regulated by the disruption of the blood–brain barrier and tumor-associated hypoxia. Recent technological advances have uncovered substantial heterogeneity among macrophages, including hypoxia-induced, lipid-metabolizing, phagocytic, and interferon-activated subtypes. This functional diversity is shaped by tumor-specific genetic alterations and metabolic reprogramming. Therapeutic approaches focusing on MDMs include inhibiting their recruitment, enhancing phagocytic activity, employing genetically engineered macrophage, and modulating metabolic pathways. While preclinical studies suggest that these approaches may improve efficacy when combined with immune checkpoint inhibitors, the dynamic spatiotemporal heterogeneity and adaptability of macrophages within the tumor microenvironment remain substantial therapeutic challenges. Future development in combination therapies, integrating single-cell multi-omics, spatial metabolic profiling, and targeted interventions, will be critical to precisely modulate MDMs, overcome immune tolerance, and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e73dae7727564fcd760697c5cdc2cc02393ac584" target='_blank'>
              New Insights into Monocyte-Derived Macrophages in Glioblastoma
              </a>
            </td>
          <td>
            Xuetong Li, Wei Gao, Xinmiao Long, Minghua Wu
          </td>
          <td>2025-07-23</td>
          <td>Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Recent biological research has highlighted the relevance of myeloid-cell populations in glioma growth, with a particular role played by tumor-associated macrophages (TAMs), which comprise resident microglia and monocyte-derived macrophages. Additionally, radiation therapy, the most common treatment for gliomas, significantly alters the tumor microenvironment, affecting TAMs and contributing to tumor recurrence. Promising preclinical studies have identified and developed drugs targeting TAMs. The development and combined deployment of these therapies require in silico techniques that enable us to optimize their outcomes. To do so, we need mathematical models of glioma growth and therapy response that explicitly incorporate TAMs—an often overlooked component in existing models. Here, we present a dynamical model of glioma growth driven by tumor-immune interactions. The model was parametrized using published data from mice experiments, including responses to ionizing radiation. We used this model to investigate glioma progression under radiotherapy combined with three treatments targeting distinct aspects of TAM biology. Simulations revealed that anti-CD47 enhanced the otherwise weak phagocytic activity, extending the upper tail of the survival curve. α-CD49d, which limits monocyte trafficking after irradiation, offered consistent survival benefits across digital twins of mice. Finally, CSF-1R inhibitors, which block the primary growth factor regulating TAM function, resulted in the largest overall survival improvement in silico. Our results aligned well with experimental evidence, suggesting that the model may help inform the optimization of myeloid cell-targeted immunotherapies, including their timing, dosage, and combination with radiation therapy, with potential relevance for improving glioma treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92c930ffd6201d9f5f23b5cf400a233231307968" target='_blank'>
              Mathematical model of tumor-macrophage dynamics in glioma to advance myeloid-targeted therapies
              </a>
            </td>
          <td>
            Jesús J. Bosque, Jordan Martínez, José García Otero, Guim Aguadé-Gorgorió, Javier E. Sanchez-Galan, Juan Belmonte-Beitia
          </td>
          <td>2025-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708353dcfdc8ca7c4e2b434fced2427c746e282e" target='_blank'>
              Tracing the evolution of single-cell 3D genomes in Kras-driven cancers.
              </a>
            </td>
          <td>
            Miao Liu, Shengyan Jin, Sherry S Agabiti, Tyler B Jensen, Tianqi Yang, Jonathan S. D. Radda, Christian F. Ruiz, Gabriel Baldissera, Moein Rajaei, Fangyong Li, Jeffrey P. Townsend, M. Muzumdar, Siyuan Wang
          </td>
          <td>2025-08-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Immune checkpoint inhibitors show limited efficacy in tumors with low tumor mutational burden, partly due to insufficient neoantigen presentation. Methods We developed a novel approach for neoantigen identification using circulating tumor cells (CTCs) isolated via leukapheresis and flow cytometry. Peripheral blood mononuclear cells (PBMCs) were collected from 11 stage IV cancer patients and 2 healthy volunteers. CTCs were enriched by depleting CD45+ hematopoietic cells and selecting CD45−Vimentin+ cells, which were confirmed cytologically to contain malignant cells. Hematopoietic lineage analysis showed that over 50% of the CTC fraction consisted of non-hematopoietic cells. DNA extracted from both the CTC and normal hematopoietic fractions underwent exome sequencing. Neoantigens were identified using the Ancer® bioinformatics platform. Results In representative patients with gastric and salivary gland cancers, 94,636 and 46,423 CTCs were isolated, respectively. DNA yields were sufficient for exome sequencing without amplification or extensive cell culture. A total of 102 (patient with gastric cancer) and 108 (patient with salivary gland cancer) neoantigens were identified in each subject, including high-ranking T-cell epitopes derived from single nucleotide variants and frameshift mutations. According to the same procedures we could successfully identify a large number of neoantigens from the CTCs of all stage IV cancer patients. This confirms the feasibility of identifying individual patient-specific neoantigens from CTCs without requiring tumor biopsies. Conclusions This is the first study to demonstrate successful neoantigen identification using non-amplified CTCs isolated by apheresis and flow cytometry. The approach provides a minimally invasive, scalable alternative for neoantigen discovery and may better capture tumor heterogeneity compared to single-site biopsies. This method holds promise for enabling rapid, personalized immunotherapy strategies, including peptide vaccines, dendritic cell vaccines, and mRNA-based treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f2f611c004eda4f555a0f11de496ee09c9cb0f" target='_blank'>
              A method for identifying neoantigens through isolation of circulating tumor cells using apheresis among patients with advanced-stage cancer
              </a>
            </td>
          <td>
            Daiki Kobayashi, Takuya Kosumi, Queenie Lai Kwan Lam, Shigeharu Fujita, Yasuki Hijikata, Kaori Takeda, Tomoya Narita, Naomi Yamashita, Guilhem Richard, Anne S De Groot, Naohide Yamashita
          </td>
          <td>2025-07-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c9003a002bf9a6c00cceca68fcf223699d2693" target='_blank'>
              Single-cell multi-omic integration analysis prioritizes druggable genes and reveals cell-type-specific causal effects in glioblastomagenesis
              </a>
            </td>
          <td>
            Y.-F. Huang, K.-L. Huang
          </td>
          <td>2025-07-01</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a3488612e1158a55dd4c8b299433b0f980348d5" target='_blank'>
              Molecular signature of human endometrial stem/progenitor cells at the single cell level
              </a>
            </td>
          <td>
            Harriet C. Fitzgerald, S. Mortlock, C. Filby, Fiona L. Cousins, Elizabeth Marquez-Garcia, K. A. Wyatt, Brett McKinnon, Luk Rombauts, J. Tsaltas, G. Montgomery, Caroline E Gargett
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Single-cell ribonucleic acid sequencing (scRNA-seq) is an important tool in molecular biology, allowing transcriptomic profiling at the single-cell level. This transformative technology has provided unprecedented insights into cellular heterogeneity, lineage differentiation, and cell-type-specific gene expression patterns, significantly advancing our understanding of complex biological systems. scRNA-seq is broadly applied across various fields, including oncology, where it sheds light on intratumoral heterogeneity and precision medicine strategies, and developmental biology, where it uncovers cellular trajectories in both model and non-model organisms. Additionally, scRNA-seq has been instrumental in ecological genomics, which can help elucidate cellular responses to environmental perturbations and species interactions. Despite these advancements, several challenges remain, particularly technical and financial barriers, limiting its application to non-model organisms and tissues with complex cellular compositions. Addressing these issues will require continued innovation in single-cell isolation methods, cost-effective sequencing technologies, and sophisticated bioinformatics tools. As scRNA-seq advances, it can deepen our understanding of biological systems, with broad implications for personalized medicine, evolutionary biology, and ecological research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35562f7f2960b20968152b898a167bc09fe4badc" target='_blank'>
              Applications and techniques of single-cell RNA sequencing across diverse species
              </a>
            </td>
          <td>
            Hyunmin Woo, Seong-il Eyun
          </td>
          <td>2025-07-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Recent advances in single-cell technologies have enabled the creation of comprehensive cell atlases, reference maps of various cell types within organisms. Here we specifically focus on T cell atlases, which offer a detailed catalog of the adaptive immune system at single-cell resolution. As such, they capture cellular diversity, functional states, and spatial dynamics across tissues, developmental stages, and disease conditions. Given the central role of T cells in orchestrating immune responses, their dysregulation underpins autoimmune disorders, cancer progression and failed immunotherapies. Therefore, a unified T cell atlas is critical for decoding such disease mechanisms, identifying therapeutic targets, and advancing personalized treatments. In this article, we explore the latest advances in T cell atlases, describing breakthroughs in multi-omics technologies, spatial profiling and computational frameworks that resolve transcriptional, epigenetic and proteomic heterogeneity. We also address persistent challenges and highlight strategies to address these gaps. Finally, we discuss emerging frontiers set to reshape our understanding of T cell dynamics in both health and diseases. Together, these insights underscore the transformative potential of T cell atlases in reconstructing precision immunology and accelerating therapeutic innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d0b0e5dda9a3845db88c70250149559b1315699" target='_blank'>
              Sketching T cell atlases in the single-cell era: challenges and recommendations.
              </a>
            </td>
          <td>
            Itana Bojović, António G. G. Sousa, S. Junttila, L. Elo
          </td>
          <td>2025-06-29</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT The oral mucosa exhibits superior healing and minimal scarring. Although mouse models are widely used to study wound healing and various diseases, their translational relevance remains unclear. Here, we performed a comparative single‐cell transcriptomic analysis of human and mouse oral mucosa to identify both shared and species‐specific mechanisms. A total of 34,969 cells from human and mouse datasets were integrated using Harmony for batch effect correction, allowing us to establish a unified oral mucosa transcriptome atlas. Fibroblasts emerged as the prominent cell population in both species, displaying conserved gene expression profiles and cell communication networks, underscoring their central role in tissue homeostasis. Key pathways involved in extracellular matrix remodelling and wound healing were highly conserved, supporting the utility of mouse models for studying fibroblast‐mediated tissue regeneration. These findings suggest that mouse models can effectively replicate human fibroblast biology, offering valuable insights for developing translational therapies that target fibroblast activity and regulatory gene networks to enhance wound healing and tissue regeneration. Additionally, we identified species‐specific cell populations, including human‐specific capillary endothelial cells and melanocytes, as well as mouse‐specific salivary gland epithelial cells. Their distinct cellular composition and functional differences suggest that these subpopulations may not be directly translatable from mouse models to human contexts. Overall, our study highlights the evolutionary conservation of fibroblasts while identifying species‐specific differences that warrant consideration in translational research. These findings provide a valuable resource for researchers using mouse models to study oral mucosa‐related diseases, facilitating the translation of preclinical discoveries into clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/911d67a1442e99627fad8d7f5ca4ca5281e764ef" target='_blank'>
              Delineating Tissue‐Specific Cell Identity of Oral Mucosa in Humans and Mice From a Single‐Cell Perspective
              </a>
            </td>
          <td>
            Huanyu Luo, Xiaoyi Yu, Hongchen Sun, Zhengwen An
          </td>
          <td>2025-08-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Liver diseases involve many distinct and functionally diverse cell types, but it remains unclear which specific cellular states contribute to inherited genetic risk. In this study, we integrated genetic data from several liver diseases with single-cell gene expression profiles from human liver tissue to identify disease-associated cells at single-cell resolution. By focusing on cellular heterogeneity, we found that genetic risk is not associated with broad cell types, but with distinct subpopulations defined by their functional states. In particular, liver sinusoidal endothelial cells and cholangiocytes showed disease associations only when in immune-activated or stress-responsive states. For example, autoimmune and viral liver diseases were linked to inflammatory subsets, not their metabolically active counterparts. These findings demonstrate that disease risk is shaped by specific cell states within heterogeneous populations. Our study underscores the need to consider cell-state diversity when studying disease mechanisms and developing targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/192d524a339350f75aa0e4bcaad1a36c461732e3" target='_blank'>
              Dissecting the Cellular Heterogeneity Underlying Liver Diseases Through the Integration of GWASs and Single-Cell RNA Sequencing
              </a>
            </td>
          <td>
            Miao Zhou, Meng Liu, Chao Xue
          </td>
          <td>2025-06-27</td>
          <td>Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastasis is a hallmark of cancer and is responsible for the majority of cancer-related deaths. Evidence suggests that even a single cancer cell can spread and seed a secondary tumour. However, not all circulating tumour cells have this ability, which implies that dissemination and distal growth require adaptative mechanisms during circulation. Here we report that constriction during microcapillary transit will trigger reprogramming of melanoma cells to a tumorigenic cancer stem cell-like state. Using a microfluidic device mimicking physiological flow rates and gradual capillary narrowing, we showed that compression through narrow channels lead to cell and nuclear deformation, rapid changes in chromatin state and increased calcium handling through the mechanosensor PIEZO1. Within minutes of microcapillary transit, cells show increased regulation of transcripts associated with metabolic reprogramming and metastatic processes, which culminates in the adoption of cancer stem cell-like properties. Squeezed cells displayed elevated melanoma stem cell markers, increased propensity for trans-endothelium invasion, and characteristics of enhanced tumorigenicity in vitro and in vivo. Pharmacological disruption of channel activity inhibited the stem cell-like state, while the selective PIEZO1 activator Yoda1 primed this state irrespective of constriction. Finally, deletion of PIEZO1 led to complete abrogation of the constriction-induced stem cell-like state. Together, this work demonstrates that compressive forces during circulation can reprogram circulating cancer cells to tumorigenic stem cell-like states that are primed for extravasation and metastatic colonization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3810403f83f02e75d7eba1f1e769e5b2b7eed299" target='_blank'>
              Capillary constrictions prime cancer cell tumorigenicity through PIEZO1
              </a>
            </td>
          <td>
            G. Silvani, C. Kopecky, S. Romanazzo, V. Rodriguez, A. Das, E. Pandzic, J. G. Lock, C. Chaffer, K. Poole, K. A. Kilian
          </td>
          <td>2025-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC), a prevalent and highly lethal malignancy, is notorious for its aggressive nature and inherent tendency to metastasize, posing significant challenges in clinical management and prognosis. Hypoxia, a pivotal characteristic of the tumor microenvironment (TME) in hepatocellular HCC, is intimately linked to disease progression and unfavorable patient outcomes, underscoring its critical role in shaping the malignant behavior of this cancer. Methods Our research leveraged single-cell RNA sequencing technology to dissect the heterogeneity of the HCC TME, focusing on hypoxia-related genes. To probe the effects of hypoxia on HCC invasion, we developed the Cell Hypoxia-Related Prognostic Feature (CHPF). We analyzed transcriptome data from the The Cancer Genome Atlas (TCGA) database and the GSE149614 dataset, employing computational methods such as UMAP, Weighted correlation network analysis (WGCNA), and CellChat to identify hypoxia cells, characterize cell subsets, and elucidate intercellular communications. Results Our analysis revealed significant heterogeneity in hypoxia cell populations within the HCC TME, with distinct expression patterns of hypoxia-related genes in neoplastic and immune cells. Our analysis revealed distinct hypoxia subpopulations within HCC, with significant overexpression of genes like MEG3, KLF6 and JUN in hypoxia cells. We identified a unique hypoxia subpopulation with high invasive potential and constructed a prognostic model based on H2-specific transcription factors including LRP10、MED8、NOL10、NOP58 and REXO4. The model demonstrated significant predictive value for lifespan of patients as verified in the TCGA dataset and an external validation group. Conclusion Key transcription factors like NOP58, MED8 play pivotal roles in hypoxia-induced HCC invasion and metastasis, and a predictive model based on these factors forecasts HCC survival. Our findings provide novel molecular markers and therapeutic targets for HCC, highlighting the importance of considering the hypoxia TME in diagnostic and treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03201-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85919f351fa84b022b25ebecfed4ffc0c988bac6" target='_blank'>
              Deciphering hypoxia’s role in hepatocellular carcinoma prognosis with single-cell approaches
              </a>
            </td>
          <td>
            Jun Liang, Di Chen, Huiyu Liang
          </td>
          <td>2025-07-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains one of the most lethal malignancies, largely due to its profoundly immunosuppressive tumor microenvironment (TME) and intrinsic resistance to conventional therapies. Radiotherapy (RT), traditionally valued for its cytotoxic effects, has recently been recognized for its immunomodulatory potential. This mini-review explores the multifaceted interactions between RT and the PDAC immune microenvironment, highlighting mechanisms such as induction of immunogenic cell death, enhancement of antigen presentation, modulation of cytokine and chemokine profiles, and upregulation of immune checkpoint molecules. These effects may transform immunologically “cold” tumors into “hot” ones, providing a rationale for combination strategies with immunotherapy. However, the dense desmoplastic stroma, abundance of regulatory T cells, and myeloid-derived suppressor cells within PDAC present substantial challenges that hinder effective immune activation. Advances in single-cell and spatial transcriptomic technologies offer new opportunities to better characterize the TME and guide personalized treatment strategies. By synthesizing mechanistic insights and clinical evidence, this review underscores the potential of integrating RT with immunotherapy to overcome resistance mechanisms and improve therapeutic outcomes in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97772762d5bc3d1c76c3496afb2362f063af01af" target='_blank'>
              Radiotherapy and immune microenvironment crosstalk in pancreatic cancer: a comprehensive review of current insights and future directions
              </a>
            </td>
          <td>
            Yucheng Xu, Jingjing Chen, Yudong Qiu, Juan Du
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Advances in treating primary breast tumors and extracranial metastases have inadvertently led to a rise in brain metastases, which are severe and carry bleak prognostic outcomes. Thus, there is an urgent need to investigate the cellular and molecular drivers of metastatic central nervous system (CNS) tumors. The tumor microenvironment (TME) is critically important for metastasis, as metastatic cells must alter their new environment to survive and grow. Our laboratory's recent studies have pinpointed platelet-derived growth factor receptor beta (PDGFRβ) signaling as a critical pathway in promoting breast cancer-associated brain metastases (BCBM). Key findings from this published work involved data emanating from a genetically engineered mouse model (GEMM) with hyperactive PDGFRβ-D849V in the mesenchymal lineage driven by Fsp1-cre. These mice exhibit increased BCBM when injected with murine mammary tumor cells secreting the ligand PDGFB, making them an effective model for studying BCBM. However, hyperactivity of PDGFRβ in these mice occurs in all mesenchymal cells. To investigate cell-specific PDGFRβ functions in the CNS, we analyzed publicly available single-cell RNA-seq data of the adult murine brain. This analysis identified astrocyte populations expressing both Pdgfrb and the Fsp1/S100A4 mesenchymal lineage marker. To probe the role of PDGFRβ in reactive astrocytes, we developed a Gfap-cre GEMM model crossed with our mutant PDGFRβ hyperactivation allele (Gfap-cre;Pdgfrb-D849V). These mice exhibit increased GFAP expression in the absence of tumor and decreased overall survival upon intracardiac injection of PDGFB secreting breast cancer cells. Furthermore, flow cytometry at day 10 post intracardiac injection indicates potential differences in the immune response of the brain TME compared to controls. Current work is focused on identifying specific transcriptomic changes in astrocyte populations due to both PDGFRβ hyperactivation as well as metastatic breast cancer colonization of the brain. Overall, this work aims to identify potential therapeutic targets to treat BCBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/065fae0e35727b10f37e95370e754abb9b46417c" target='_blank'>
              BSBM-08 ASTROCYTE PDGFRβ HYPERACTIVATION IN BREAST CANCER BRAIN METASTASIS
              </a>
            </td>
          <td>
            Nathaniel Grabinski, Johnathon Scheibel, Jesse Reardon, Paul Lee, Jonathan Godbout, S. Sizemore, Jessica Winter, Gina M. Sizemore
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f09209df407344118779f936014e334d9c123d1" target='_blank'>
              Roadmap for spatial transcriptomics of HIV in tissues.
              </a>
            </td>
          <td>
            Qijie Guan, Amare Eshetu, Ya-Chi Ho
          </td>
          <td>2025-07-07</td>
          <td>Current opinion in HIV and AIDS</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction Immune-based agents, especially Immune Checkpoint Inhibitors (ICI), are standard of care therapy in non-small cell lung cancers (NSCLC); however, a significant number of patient tumors fail to respond, or develop resistance. While target expression, mutation burden and oncogenic pathways impact responses, an established mechanism contributing to ICI therapy failure is evasion of T-cell responses via downregulation of human leukocyte antigen (HLA). Conversely, natural killer (NK) cells eQector function is enhanced in the absence of HLA, making NK cellular therapies an attractive option for ICI resistant tumors. Challenges for current NK cell therapies include failure to adequately infiltrate solid tumors and long-term persistence, which may be overcome by deploying NK-derived extracellular vesicles (NKEVs) as a personalized novel adoptive cellular therapeutic with cytotoxic effects. Methods In a human NSCLC cohort (n=10), we used single cell RNAseq and antibody labeling (CITEseq) to examine the immune cell landscape in peripheral immune cells (PBMCs) and tumors. NKEVs retrieved from patient NK cells were characterized with proteomics and bulk RNAseq, and EV functionality was assessed using primary tumor organoids. Results We identified circulating NK cell subsets, describing diQerences in cell composition, gene expression and signaling, related to time point, NSCLC subtype (adenocarcinoma, squamous cell), composition and tumor grade. Next, we examined the functional capabilities of patient NKEVs in organoid structures derived from primary tumor cells, finding that exposure to patient NKEVs resulted in a 40-45% decrease in organoid viability, and significantly lowered the cisplatin dose required to elicit cytotoxicity. In Nivolumab treated PBMC co-culture experiments, NKEV addition favorably shifted the organoid infiltrating immune population to significantly fewer CD4+ T cells and more CD56+ NK cells. Finally, we used the multi-omic characterization of NKEV molecular cargo to identify RNA transcripts and proteins associated with cytotoxic and immune recruiting functions. Conclusions This work demonstrated that NKEVs can be successfully harvested from patient derived, expanded NK cells, and highlights their heterogeneous cargo, and anti-tumor properties in combination with standard-of-care therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d000c1883c73b9666be10545d77653f5089b0cd0" target='_blank'>
              NK cell-derived extracellular vesicles enhance cytotoxicity and immune cell recruitment in non-small cell lung cancer
              </a>
            </td>
          <td>
            Joanna Palade, Eric Alsop, Nanyun Tang, J. Antone, Dorothy M. Paredes, Tithi Ghosh Halder, R. Soldi, Taylor Bargenquast, Gary Schwartz, Jennifer P. Finholt, George J. Snipes, Sunil Sharma, Michael E. Berens, T. Whitsett, K. Van Keuren-Jensen, R. Kelly
          </td>
          <td>2025-07-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive primary brain tumor, characterized by high heterogeneity and resistance to standard therapies. Traditional glioma cell lines often fail to retain patient-specific genetic and phenotypic characteristics, limiting their translational relevance. To address this, we had established 50 patient-derived glioma cell lines (PDGCs) using a serum-free neural stem cell culture system. This protocol outlines the collection, processing, and long-term culture of tumor samples obtained from GBM patients, ensuring the preservation of key molecular features. Our methodology includes enzymatic tissue dissociation, red blood cell lysis, and basement membrane matrix extract-assisted adherent culture, which improves cell viability and facilitates high-throughput drug screening. Characterization of PDGCs via whole-genome and transcriptomic sequencing and immunofluorescence staining confirms their retention of common GBM genetic alterations and neural stem cell and progenitor markers. Additionally, these cells exhibit tumorigenic potential in xenograft models. By optimizing culture conditions, this protocol provides a standardized framework for generating biologically relevant GBM models to support therapeutic discovery and mechanistic studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/957e6a2f99a8ca6e228fa6b356d5f813fc4b9e69" target='_blank'>
              Optimized Workflow for Isolation and Long-term Culture of Patient-derived Glioma Cells Retaining Original Tumor Characteristics.
              </a>
            </td>
          <td>
            Tingting Wang, Jian Chen
          </td>
          <td>2025-07-08</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BrMs represent a significant clinical challenge due to its high incidence, complex biology, and poor prognosis. Astrocytes, the predominant glial cells in the central nervous system, are increasingly recognized for their central role in shaping the brain microenvironment and driving the progression of brain metastases. This review highlights the multifaceted contributions of astrocytes to the metastatic cascade, including their involvement in tumor cell invasion, modulation of the blood–brain barrier, and establishment of a pro-metastatic niche. Astrocytes interact with tumor cells through various mechanisms, such as forming gap junctions, secreting factors that enhance tumor survival, and supporting immune evasion. These interactions promote tumor growth and protect metastatic cells from chemotherapy and radiotherapy, complicating current treatment strategies. Furthermore, astrocytes help reshape the brain microenvironment to support tumor adaptation and expansion, making them critical players in both the early and late stages of metastasis. Emerging therapeutic approaches are focused on disrupting astrocyte-tumor interactions, with strategies targeting astrocyte-driven communication pathways, immune modulation, and metabolic support systems. Nanotechnology and precision medicine also offer novel opportunities to improve drug delivery across BBB and directly target the astrocyte-tumor axis. This review underscores the importance of further research to fully understand the role of astrocytes in brain metastasis and develop therapies that leverage this knowledge to improve patient outcomes. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88692860b76d7d14ada25817247bee705d2a6f44" target='_blank'>
              Astrocyte involvement in brain metastasis: from biological mechanisms to therapeutic strategies
              </a>
            </td>
          <td>
            Teng Zhou, Yuxin Yan, Mingxi Lin, C. Zeng, Xinhui Mao, Yifei Zhu, Jinlu Han, Dou-Dou Li, Jian Zhang
          </td>
          <td>2025-07-15</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Neuroblastomas are highly heterogeneous tumors originating from neural crest-derived cells destined to form the sympathetic nervous system. Nearly half of high-risk tumors present with amplification of the MYCN proto-oncogene. Here, we describe a Mycn-driven, transplantable, non-germline, genetically engineered mouse model (Mycn-nGEMM). Mycn-nGEMM tumors recapitulate the immune-evasive, macrophage-rich tumor microenvironment of high-risk, MYCN-amplified human neuroblastoma. Treatment of tumor-bearing mice with anti-PD-L1, but not anti-PD-1 or anti-CTLA-4, inhibited tumor growth, profoundly remodeling the tumor microenvironment by depleting anti-inflammatory macrophages and increasing T cell infiltration. Surprisingly, while tumor cells showed low expression of PD-L1, anti-inflammatory macrophages from both murine and human neuroblastoma expressed PD-L1. We identified cytokines, including macrophage migration inhibitory factor, secreted by the Mycn-nGEMM cancer cells that drive expression of PD-L1 on macrophages. Combining anti-PD-L1 with CD40 agonist antibodies further improved survival in Mycn-nGEMM mice, demonstrating the potential for myeloid-targeting immunotherapies to overcome inhibitory barriers in immune-evasive neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2aecad2de1b569da0f36a9e846ad42a57aa1628" target='_blank'>
              Myeloid-targeting immunotherapies overcome inhibitory barriers in immune-evasive neuroblastoma.
              </a>
            </td>
          <td>
            Marie Ménard, Hiroyuki Yoda, Nicole Nasholm, Megumi J Barata, Linyu Wang, Erin F. Simonds, Edbert D Lu, Shannon T. Wong-Michalak, Lauren McHenry, Alvin Farrel, Rebecca S. Kaufman, Vanessa Lopez, Rebekah J Kennedy, G. E. Fernandez, Hiroyuki Shimada, Liron D. Grossmann, S. Asgharzadeh, John M. Maris, W. Gustafson, William A Weiss
          </td>
          <td>2025-07-14</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Purpose Programmed cell death (PCD) mechanisms play crucial roles in cancer progression and treatment response. This study aims to develop a PCD scores prediction model to evaluate the prognosis of hepatocellular carcinoma (HCC) and elucidate the tumor microenvironment differences. Methods We analyzed transcriptomic data from 363 HCC patients in the TCGA database and 221 patients in the GEO database to develop a PCD prediction model. Single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics sequencing (ST-seq) data from HCC patients were analyzed to investigate the tumor microenvironment and functional disparities. The oncogenic role of the key gene UBE2E1 in the model was explored in HCC through various in vitro experiments. Results Seventeen PCD-related genes were identified as significant prognostic indicators, forming the basis of our PCD prediction model. High-PCD scores correlated with poorer overall survival (OS) and exhibited significant predictive capabilities. scRNA-seq analysis revealed distinct tumor cell characteristics and immune microenvironment differences between high- and low-PCD groups. High-PCD tumors showed increased cell proliferation and malignancy-associated gene expression. T cells in high-PCD patients were more likely to be exhausted, with elevated expression of exhaustion markers. ST-seq data also confirmed these results. Among the genes associated with the PCD prognostic model, UBE2E1 was identified as a key oncogenic marker in HCC. Conclusions The PCD prediction model effectively predicts prognosis in HCC patients and reveals critical insights into the tumor microenvironment and immune cell exhaustion. This study underscores the potential of PCD-related biomarkers in guiding personalized treatment strategies for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88d93e2b7ad6d106b6fbcf29fd3ec02edc2e6591" target='_blank'>
              Integrative spatial and single-cell transcriptomics elucidate programmed cell death-driven tumor microenvironment dynamics in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kai Lei, Yutong Zhao, Shumin Li, Jiawei Liu, Wenhao Chen, Caihong Zhou, Yi Zhang, Jinmei Tan, Jian Wu, Qi Zhou, Jiehui Tan
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) is a global health challenge shaped by a complex tumor microenvironment (TME), where myeloid‐driven inflammatory and immunosuppressive processes play central roles. Understanding the spatial and dynamic interactions within the TME is essential for uncovering novel regulatory mechanisms. We integrated spatial transcriptomics, single‐cell RNA‐sequencing, and network pharmacology to develop a comprehensive analytical framework for deconstructing cellular and molecular interactions across spatial domains within the TME. Using this approach, we investigated the potential mechanisms by which the Sparganii Rhizoma‐Curcumae Rhizoma (SCP) Chinese medicine (TCM) herbal pair may influence primary CRC and liver metastases. Our analysis suggests that in primary CRC, SCP may affect neutrophil‐enriched domains through modulation of CEACAM‐related signaling, while in liver metastases, it may influence macrophage‐dominated domains associated with APP signaling. These findings highlight possible domain‐specific molecular interactions and suggest that SCP may regulate intra‐ and inter‐domain signaling patterns relevant to TME remodeling. This study establishes a novel integrative methodology to investigate spatial domain‐specific mechanisms within the TME. Our findings reveal the multi‐dimensional effects of SCP, providing a precision‐targeted, TME‐centric therapeutic strategy for CRC. The framework aligns with TCM principles and offers new avenues for personalized cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64108e2457f88f143ed78e3504c4067d48fd77bb" target='_blank'>
              Spatial Domain‐Based Approach to Analyze the Mechanism of Sparganii Rhizoma‐Curcumae Rhizoma Pair in the Treatment of Colorectal Cancer
              </a>
            </td>
          <td>
            Yuan-Jie Liu, Qian-wen Ye, Yi‐lan Jin, Qian Zhang, Xi Zou, 
          </td>
          <td>2025-08-01</td>
          <td>Food Science & Nutrition</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Cervical cancer exhibits heterogeneous clinical outcomes, requiring improved prognostic tools. Single-cell RNA sequencing enables high-resolution analysis of tumor microenvironment cellular heterogeneity. This study developed a prognostic model for cervical cancer through single-cell transcriptomic analysis and immune infiltration characterization, focusing on PTK6 as a key biomarker. Methods We analyzed TCGA and GEO transcriptomic data with single-cell RNA sequencing datasets. Fifteen machine learning algorithms constructed prognostic models using immune infiltration-related genes. Single-cell analysis employed Seurat for cell clustering and annotation. PTK6 expression was validated in H8 and HeLa cell lines via RT-qPCR and siRNA knockdown experiments. Results Single-cell sequencing revealed distinct cellular populations including CD8T cells, CD4Tconv cells, and fibroblasts. The prognostic model achieved excellent performance with AUC values of 0.737–0.757 across 1–5 years. PTK6 showed significantly elevated expression in tumors and strong correlations with immune infiltration. Single-cell analysis confirmed PTK6 expression across multiple cell types. Functional validation demonstrated that PTK6 knockdown reduced HeLa cell proliferation, confirming its oncogenic role. Conclusion PTK6 emerges as a critical immune infiltration-related prognostic biomarker through single-cell transcriptomic analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea19777f2b2d221453d4adbe7a44b8371785742e" target='_blank'>
              PTK6 mediated immune signatures revealed by single cell transcriptomic and multi omics big data analysis in cervical cancer
              </a>
            </td>
          <td>
            Fen Zhao, Huanxin Zhong, Lifang You, Yi Du, Changchang Huang
          </td>
          <td>2025-08-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We recovered tumor microenvironmental features previously described to associate with therapy response. We subsequently identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.



 Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andreanne Gagne, Samantha Hoffman, Kevin Bi, Breanna Titchen, Connor Hennessey, Joshua Remland, Matthew Carnes, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, Eliezer M. Van Allen. Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b559d25313fad22976c28c00d01803263f1319a3" target='_blank'>
              Abstract A046: Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda E. Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor Hennessey, Joshua Remland, Matthew Carnes, Erin Shannon, Sabrina Y. Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve M Boland, A. Aguirre, Nilay S. Sethi, Valentina Boeva, Eliezer M Van Allen
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Innovative identification technologies for hematopoietic stem cells (HSCs) have expanded the scope of stem cell biology. Clinically, the functional quality of HSCs critically influences the safety and therapeutic efficacy of stem cell therapies. However, most analytical techniques capture only a single snapshot, disregarding the temporal context. A comprehensive understanding of the temporal heterogeneity of HSCs necessitates live-cell, real-time and non-invasive analysis. Here, we developed a prediction system for HSC diversity by integrating single-HSC ex vivo expansion technology with quantitative phase imaging (QPI)-driven machine learning. By analyzing the cellular kinetics of individual HSCs, we discovered previously undetectable diversity that snapshot analysis cannot resolve. The QPI-driven algorithm quantitatively evaluates stemness at the single-cell level and leverages temporal information to significantly improve prediction accuracy. This platform advances the field from snapshot-based identification of HSCs to dynamic, time-resolved prediction of their functional quality based on past cellular kinetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d1e09a9f61aa5ec203755a5a99580bea38ed797" target='_blank'>
              Quantitative phase imaging with temporal kinetics predicts hematopoietic stem cell diversity
              </a>
            </td>
          <td>
            Takao Yogo, Yuichiro Iwamoto, H. J. Becker, Takaharu Kimura, Reiko Ishida, Ayano Sugiyama-Finnis, Tomomasa Yokomizo, Toshio Suda, Sadao Ota, Satoshi Yamazaki
          </td>
          <td>2025-07-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background SPOCK1, a matricellular glycoprotein, has been implicated in tumor progression, metastasis, and the tumor immune microenvironment, yet its specific roles in breast cancer (BRCA) remain unclear. This study aimed to systematically explore the expression pattern, prognostic significance, mutation landscape, immune association, and spatial localization of SPOCK1 in breast cancer through integrated multi-omics analyses. Methods Transcriptomic, genomic, and clinical data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) were utilized. Bulk RNA sequencing and single-cell RNA sequencing (scRNA-seq) analyses were conducted, including functional enrichment, immune infiltration assessments, mutation profiling, and transcription factor activity analysis. Multiplex immunohistochemistry (mIHC) was performed to validate the spatial distribution of SPOCK1+ cancer-associated fibroblasts (CAFs) within the tumor microenvironment. Statistical analyses were performed using R and GraphPad Prism. Results SPOCK1 was broadly overexpressed in multiple cancer types and significantly associated with poor prognosis in BRCA. High SPOCK1 expression correlated with immune checkpoint activation, enhanced immune infiltration, and enriched metastasis-related pathways such as epithelial–mesenchymal transition (EMT) and TGF-β signaling. Single-cell analysis identified CAFs as the primary cell population expressing SPOCK1, with spatial mIHC confirming their close proximity to tumor cells. Furthermore, SPOCK1-high CAFs exhibited stronger intercellular communications with malignant cells via collagen, fibronectin, and IGFBP signaling pathways, alongside distinct transcription factor and metabolic profiles. In breast cancer CAF cell lines with knockdown of ANXA2 we found that the expression of both SPOCK1 and IGF1 was reduced. Conclusion SPOCK1 serves as a critical regulator of breast cancer progression, influencing tumor metastasis and reshaping the immune microenvironment via CAF-mediated mechanisms. These findings suggest that targeting SPOCK1+ CAFs could offer new therapeutic opportunities for breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6ec4d3a2efd20e6cfad0b54c715da9bed1f6fb" target='_blank'>
              SPOCK1 promotes the progression of breast cancer by modulating cancer-associated fibroblasts and exerts a synergistic effect with ANXA2
              </a>
            </td>
          <td>
            Yuan Jie, Xin Fei, Meng Fan
          </td>
          <td>2025-08-05</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0e616d882e7bd8159604ae82c04f4049ee875a" target='_blank'>
              Proneural-Mesenchymal hybrid glioblastoma cells are resistant to therapy and dependent on nuclear import.
              </a>
            </td>
          <td>
            G. Bourmeau, O. Anezo, Jeremy Raymond, A. Ballestín, C. Pichol-Thievend, J. Reveilles, Adrien Thomas, Lin Wang, Melanie Miranda, Eve Moutaux, S. Liva, V. Ribecco, L. Besse, F. Dingli, D. Loew, Céline Vallot, G. Gargiulo, V. Ravi, Kevin Joseph, G. Seano
          </td>
          <td>2025-07-08</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Objective: The aim of this study was to investigate the role and mechanism of CD79A+ B cells in mediating the microenvironment of breast cancer and the relationship with the prognosis of breast cancer. Methods: Single-cell RNA sequencing and bulk RNA sequencing analysis were combined to annotate breast cancer cell subtypes, perform cell communication and trajectory analysis. CD79A-related signature was constructed by LASSO and multivariate Cox analysis. CD79A+ B cell subsets in the tumor microenvironment were explored by immunoanalysis and multiple immunofluorescence analysis. Results: There were communication relationships between CD79A+ B cells and multiple cell types. A prognostic risk signature containing 6 genes was constructed by combining the TCGA dataset. The immune profile analysis showed that the low-risk group showed a higher immune response. In addition, multiple immunofluorescence analysis showed an attraction between CD79A+ B cells and tumor cells, and patients with high CD79A+ B cells expression had significantly higher survival rates. Conclusion: This study comprehensively explored the heterogeneity of CD79A+ B cells through transcriptome analysis and chromatin analysis, which contributes to an in-depth understanding of the function of CD79A+ B cells in biological processes as well as the molecular mechanism of breast carcinogenesis, providing a theoretical basis for treatment and prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/088e6783ffeb2259cf83f0eff8450fb916b96b21" target='_blank'>
              Single-Cell Sequence and Machine Learning Identify a CD79A+B Cells-Related Transcriptional Signature for Predicting Clinical Outcomes and Immune Microenvironment in Breast Cancer
              </a>
            </td>
          <td>
            Haihong Hu, Wendi Zhan, Hongxia Zhu, Bo Hao, Ting Yan, Jingdi Zhang, Siyu Wang, Taolan Zhang
          </td>
          <td>2025-07-26</td>
          <td>Cancer Informatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glucose metabolism reprogramming as a defining hallmark of cancer has become a pivotal frontier in oncology research. Recent technological advances in single-cell sequencing, spatial omics, and metabolic imaging have transformed the field from static bulk analyses to dynamic investigations of spatiotemporal heterogeneity at a single-cell resolution. This review systematically summarizes the current knowledge on tumor glucose metabolism dynamics, discussing spatial heterogeneity and temporal evolution patterns, metabolic subpopulation interactions revealed by single-cell metabolomics, the glucose metabolism–epigenetics–immunology regulatory axis, and therapeutic strategies targeting metabolic vulnerabilities. Recent technological advances in single-cell sequencing and spatial omics have transformed our understanding of tumor glucose metabolism by providing high-resolution insights into metabolic heterogeneity and regulatory mechanisms, contrasting with classical bulk analyses. Spatiotemporal heterogeneity critically influences therapeutic outcomes by enabling tumor cells to adapt metabolically under selective pressures (e.g., hypoxia, nutrient deprivation), fostering treatment resistance and relapse. Deciphering these dynamics is essential for developing spatiotemporally targeted strategies that address intratumoral diversity and microenvironmental fluctuations. By integrating cutting-edge advances, this review deepens our understanding of tumor metabolic complexity and provides a conceptual framework for developing spatiotemporally precise interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eed1c2a93a86e6df9bbaf577827b208495bf5a0" target='_blank'>
              Spatiotemporal Heterogeneity of Tumor Glucose Metabolism Reprogramming: From Single-Cell Mechanisms to Precision Interventions
              </a>
            </td>
          <td>
            Xiaoxue Chai, Qian Tao, Lili Li
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Objectives: Cell–cell communication (CCC) is a critical process within the tumor microenvironment, governing regulatory interactions between cancer cells and other cellular subpopulations. Aiming to improve the accuracy and completeness of intercellular gene-regulatory network inference, we constructed a novel spatial-resolved gene-regulatory network framework (spGRN). Methods: Firstly, the spatial multi-omics data of colorectal cancer (CRC) patients were analyzed. We precisely located the tumor boundaries and then systematically constructed the spGRN framework to study the network regulation. Subsequently, the key signaling molecules obtained by the spGRN were identified and further validated by the spatial-proteomics dataset. Results: Through the constructed spatial gene regulatory network, we found that in the communication with malignant cells, the highly expressed ligands LIF and LGALS3BP and receptors IL6ST and ITGB1 in fibroblasts can promote tumor proliferation, and the highly expressed ligands S100A8/S100A9 in plasma cells play an important role in regulating inflammatory responses. Further, validation of the key signaling molecules by the spatial-proteomics dataset highlighted the role of these genes in mediating the regulation of boundary-related cells. Furthermore, we applied the spGRN to publicly available single-cell and spatial-transcriptomics datasets from three other cancer types. The results demonstrate that ITGB1 and its target genes FOS/JUN were commonly expressed in all four cancer types, indicating their potential as pan-cancer therapeutic targets. Conclusion: the spGRN was proven to be a useful tool to select signal molecules as potential biomarkers or valuable therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec6067d180dbb66ca08258914f72ecc220328a60" target='_blank'>
              Construction of Gene Regulatory Networks Based on Spatial Multi-Omics Data and Application in Tumor-Boundary Analysis
              </a>
            </td>
          <td>
            Yiwen Du, Kun Xu, Siwen Zhang, Lanming Chen, Zhenhao Liu, Lu Xie
          </td>
          <td>2025-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract While immune-modulating therapies which enhance antitumor T cell function have revolutionized cancer therapy, including for patients with brain metastases, the antigen specificity of intratumoral T cells remains poorly characterized. We performed droplet-based 5’ single-cell RNA sequencing and T cell receptor (TCR) sequencing to characterize the transcriptional profile and clonal repertoire of tumor-infiltrating T cells in resected tumors from patients with previously untreated non-small cell lung cancer (NSCLC) brain metastases (n=6). We identified a population of clonally-expanded, CXCL13+CD8+ T cells enriched for transcriptional signatures predictive of tumor reactivity. Using a high-throughput T cell antigen identification method developed by our group, we screened 164 TCRs from clonally-expanded and/or CXCL13+CD8+ T cells against 2,289 putative neoantigens in addition to 146 tumor-associated antigens. In total, we identified reactivities for 21% (n=35) of the TCRs screened. Consistent with recent studies, we observed that the transcriptional state of the T cell was more predictive of tumor reactivity than the degree of clonal expansion. Of TCR specificities identified, 25.7% (n=9) of TCRs were reactive to neoantigens derived from somatic single-nucleotide variants specific to each patient (private antigen). Intriguingly, 54.3% (n=19) of the TCRs were reactive to unmutated peptides expressed by many patients’ lung tumors (public antigens). In summary, we present a deep characterization of the antigenic reactivities of intratumoral T cells in NSCLC brain metastases. Our data are consistent with an emerging body of literature that has theorized that “missing reactivities” from neoantigen-centric studies arise from complex mechanisms of genetic and epigenetic dysregulation within tumors. Further characterization of these “public antigens” may provide the basis for novel antigen-directed T cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a71379070ce21b871f60cf702eab6f6054459a92" target='_blank'>
              TRLS-11 UNCOVERING ANTIGEN-SPECIFICITY OF INTRATUMORAL CD8+ T CELLS IN NON-SMALL CELL LUNG CANCER BRAIN METASTASES
              </a>
            </td>
          <td>
            Benjamin Lu, Berkay Yahsi, Jianlei Gu, Jack Coburn, Mithila Kasbe, Anna Arnal-Estapé, Kayla Sohn, Kalyn Whitehead, Mashwiyat Mosharraf, Don Nguyen, Hongyu Zhao, Liliana Lucca, K. Schalper, Veronica Chiang, David Hafler, Mark N. Lee
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Glioma is a highly heterogeneous and aggressive brain tumour that demands an integrated understanding of its molecular and immunological landscape. We collected multi-omics data from 575 TCGA diffuse-glioma patients (156 IDH-wild-type WHO-grade 4 glioblastomas and 419 IDH-mutant WHO-grade 2/3 diffuse gliomas) together with two validation cohorts (CGGA n = 970; GEO n = 110). Using the MOVICS framework, we derived three integrative molecular subtypes—CS1, CS2 and CS3. Ten machine-learning algorithms in MIME were benchmarked, and the Lasso + SuperPC combination yielded an eight-gene GloMICS (Glioma Multi-Omics Consensus Signature) prognostic score. The subtypes display discrete biology: CS1 (astrocyte-like) is characterized by glial lineage features, immune-regulatory signaling, and relatively favorable prognosis; CS2 (basal-like/mesenchymal) shows epithelial-mesenchymal transition, stromal activation, and high immune infiltration, including PD-L1 expression; CS3 (proneural-like/IDH-mut metabolic) exhibits metabolic reprogramming (OXPHOS, hypoxia) and an immunologically cold tumour microenvironment (TME). CS2 is associated with the worst overall survival, whereas CS1 confers the most favourable outcome. Dual checkpoint blockade or T-cell-rejuvenation strategies may benefit CS2 tumours, while metabolic inhibitors could prove effective in CS3. The eight-gene GloMICS score outperformed 95 published prognostic models (C-index 0.74–0.66 across TCGA, CGGA and GEO). TME deconvolution, immune checkpoint profiling and TIDE analysis indicate that high-risk GloMICS tumours harbour immunosuppressive fibroblast-rich niches and exhausted CD8⁺ T cells. Connectivity-map screening nominated dabrafenib, irinotecan and three additional CTRP/PRISM compounds as candidate agents for the high-risk group. Our study establishes robust glioma subtypes and a transferable prognostic signature, offering a blueprint for biomarker-guided therapy. Future work should include single-cell and immunohistochemical validation of subtype hallmarks and prospective trials stratified by GloMICS score. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-09742-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/661f2e8ad785f396f7fe131161f6eca4562f8504" target='_blank'>
              Comprehensive multi-omics and machine learning framework for glioma subtyping and precision therapeutics
              </a>
            </td>
          <td>
            Yi Ding, Zhaiyue Xu, Wenjing Hu, Peng Deng, Mian Ma, Jiandong Wu
          </td>
          <td>2025-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Spatial transcriptomics (ST) is transforming our understanding of tumor heterogeneity by enabling high-resolution, location-specific mapping of gene expression across tumors and their microenvironment. However, the translational potential of spatial transcriptomics is still limited by its high cost, hindering the assembly of large patient cohorts needed for robust biomarker discovery. Here we present Path2Space, a deep learning approach that predicts spatial gene expression directly from histopathology slides. Trained on substantial breast cancer ST data, it robustly predicts the spatial expression of over 4,300 genes in independent validations, markedly outperforming existing ST predictors. Path2Space additionally accurately infers cell-type abundances in the tumor microenvironment (TME) based on the inferred ST data. Applied to more than a thousand breast tumor histopathology slides from the TCGA, Path2Space characterizes their TME on an unprecedented scale and identifies three new spatially-grounded breast cancer subgroups with distinct survival rates. Path2Space-inferred TME landscapes enable more accurate predictions of patients’ response to chemotherapy and trastuzumab directly from H&E slides than those obtained by existing established sequencing-based biomarkers. Path2Space thus offers a transformative, fast and cost-effective approach to robustly delineate the TME directly from their histopathology slides, facilitating the development of spatially-grounded biomarkers to advance precision oncology.



 Emma M. Campagnolo, Eldad D. Shulman, Roshan Lodha, Amos Stemmer, Peng Jiang, Carlos Caldas, Simon Knott, Danh-Tai Hoang, Kenneth Aldape, Eytan Ruppin. Path2Space: An AI approach for cancer biomarker discovery via histopathology inferred spatial transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B047.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d0f988faed3c86233b6f535df129f10c4acb357" target='_blank'>
              Abstract B047: Path2Space: An AI approach for cancer biomarker discovery via histopathology inferred spatial transcriptomics
              </a>
            </td>
          <td>
            Emma M. Campagnolo, E. Shulman, Roshan Lodha, Amos Stemmer, Peng Jiang, Carlos Caldas, Simon Knott, Danh-Tai Hoang, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) is a common characteristic of human cancers, which occurs most frequently in ovarian and breast cancers. The unique genetic vulnerabilities, coupled with the synthetic lethality effect of Poly-(ADP-ribose) polymerase (PARP) inhibitors (PARPi), provide a promising opportunity for targeting HRD cancers. However, only a few HRD patients benefit from PARPi monotherapy, and it is therefore imperative to explore effective combination therapies for HRD tumors. Growing evidence has underscored the distinct tumor microenvironment (TME) landscape of HRD cancers. Immune activation and immune suppression co-exist in a dynamic balance during the development of HRD cancers. At the late stage, however, negative immune regulation predominates, resulting in the formation of an immunosuppressive microenvironment. This intricate network reprograms cancer biology in multiple aspects and serves as a potential target for cancer treatment. In this review, we briefly outline the current HRD tests and genetic characteristics of HRD cancers, focusing on breast, ovarian, pancreatic, and prostate cancers. We then summarize the interactions and crosstalk between immune cells and cancer cells, as well as various signaling pathways within the TME of HRD cancers. Additionally, we highlight recent advances in combining PARP inhibitors with immunotherapies in preclinical models and clinical trials of HRD cancers. This review provides valuable insights and perspectives into the distinct landscape of TME in HRD cancers, and offers a rationale for expanding the application of this combined therapeutic approach to a broader range of HRD cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befa4ba7c7bbd20ccb4cc8148dd06bbf9c57c776" target='_blank'>
              The distinct landscape of tumor immune microenvironment in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Qiuyang Xu, Yuanjia Wen, Taiyuan Huang, Huayi Li, Xingzhe Liu, Shen-nan Shi, Wenjian Gong, Gordon B. Mills, Ding Ma, Qing-cui Gao, Yong Fang
          </td>
          <td>2025-08-20</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2411017a831bd32082b22e6f4221bf8c31f61827" target='_blank'>
              ETS-guided iPSC-endothelial models recapitulate malaria pathogenesis
              </a>
            </td>
          <td>
            François Korbmacher, Hannah Fleckenstein, Rory K. M. Long, Patryk Poliński, Livia Piatti, Borja López-Gutiérrez, Alina Batzilla, Dennis Crusius, Vikas Trivedi, Miki Ebisuya, Maria Bernabeu
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background DNA methylation plays a crucial role in cancer development and progression and has been linked to genetically and clinically distinct tumor classes, including IDH-mutated and IDH-wildtype adult-type diffuse gliomas. Here, we identify a CpG-island methylator phenotype (CIMP) that characterizes the receptor tyrosine kinase 2 (RTK2) subtype of IDH-wildtype glioblastoma. Results This RTK2-CIMP affects genomic locations and cell functions distinct from those of IDH mutation-associated IDH-CIMP and suppresses the expression of its target genes. The RTK2-CIMP-region chromatin is characterized by a combination of repressive and activating marks, including polycomb-associated H3K27me3 and enhancer-associated H3K4me1, consistent with DNA methylation-mediated silencing of genes with bivalent-state promoters in neural progenitor cells. Functionally, RTK2-CIMP affects neuronal lineage genes and is significantly associated with astrocyte-like glioblastoma, suggesting that RTK2-CIMP is an epigenetic signature of the astrocyte-like cell state. Furthermore, we demonstrate that RTK2-CIMP can be induced by genetic manipulation in glioblastoma cells. Conclusions Our results suggest that RTK2-CIMP is a key contributor to cell-state plasticity in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03670-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94bee397b61a1f7011617f82ad34a983c326fca3" target='_blank'>
              A new IDH-independent hypermethylation phenotype is associated with astrocyte-like cell state in glioblastoma
              </a>
            </td>
          <td>
            Ana Luisa Costa, Daria Doncevic, Yonghe Wu, Lin Yang, Ka-Hou Man, Anna-Sophie Spreng, Hannah Winter, M. Fletcher, B. Radlwimmer, Carl Herrmann
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Although immune checkpoint inhibitors (ICIs) hold promise for those diagnosed with advanced human lung adenocarcinoma (LUAD), notable heterogeneity in patient responses and the complex tumor microenvironment (TME) limits their clinical utility while providing clinical benefit. To identify new therapeutic targets to pursue in chimeric antigen receptor (CAR) T-cell therapy, we leveraged an integrative multi-omic approach. This study identified three oncogenic drivers, karyopherin α2 (KPNA2), golgi membrane protein 1 (GOLM1) and thymidine kinase 1 (TK1), that are overexpressed in LUAD, spatially enriched in malignant niches and associated with significantly reduced overall survival (log-rank P < 0.01). Furthermore, functional interrogation using RNAi-mediated gene silencing in LUAD cell lines (NCI-H1650, A549), demonstrated their essential roles in mediating protumorigenic pathways. Specifically, CRISPR interference of KPNA2 or TK1 inhibited cellular proliferation and invasion while also phenocopying a pro-apoptotic effect with cisplatin. GOLM1 depletion inhibited PD-L1 upregulation and restored CD8 + T-cell cytotoxicity in co-culture. Mechanistically, dual knockdown targets, KPNA2 and TK1, were shown to restore deregulated STAT3 signaling that promotes cell survival and also enhance MHC class I antigen presentation implicating functional roles for KPNA2 and TK1 in linked processes of intrinsic oncogenesis and immunoediting by immune evasion and suppression. In summary, KPNA2, GOLM1, and TK1 are functionally relevant molecular coordinators for tumor cell autonomous proliferation and TME mediated immunosuppression. Their membrane expression (notable in the case of PD-L1) and functional roles in immune modulation, make them reasonable targets of CAR T-cell immunotherapy. We posit that a combined strategy targeting KPNA2, GOLM1 and TK1 with or without PD-1/PD-L1 axis inhibitors could provide a better strategy to treat patients with ICI resistant LUAD and generate prolonged and stable immune remodeling. Supplementary Information The online version contains supplementary material available at 10.1186/s40001-025-02955-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb248ed2c42e77c5eb8fc8c37967d8c58decf41" target='_blank'>
              Coordinating oncogenesis and immune evasion: KPNA2, GOLM1, and TK1 as novel CAR T-cell targets in lung adenocarcinoma
              </a>
            </td>
          <td>
            Wangrui Liu, Hui Zou, Lei Guo, Zhonghua Zhou, Yahui Xie, Huaqi Guo, Gang Wei, Kai Zhang, Hui Yin, Shiyin Wei, Jiachang Chi
          </td>
          <td>2025-08-19</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Aging and obesity are associated with pro-inflammatory changes in adipose tissue. Overlapping mechanisms, such as the infiltration of inflammatory macrophages and T cells into visceral adipose tissue, have been implicated in contributing inflammation. However, a comparative analysis from both states is needed to identify distinct regulatory targets. Here, we performed single-cell RNA sequencing of stromal vascular fractions (SVF) isolated from gonadal white adipose tissue (gWAT) of young mice fed either a normal or a high-fat diet, and aged mice fed a normal diet. Our analysis revealed that physiological aging, compared to high-fat diet induced obesity, was associated with accumulation of phenotypically distinct CD8 T cells resembling virtual memory (VM) CD8 T cells. These cells expressed high levels of Cd44, Sell, Il7r, Il2rb, lacked Itga4, and exhibited elevated Fcgr2b expression which was associated with pseudotime differentiation trajectories. Flow cytometry confirmed an age-associated increase in Fcgr2b+CD49d− VM like CD8 T cells in gWAT. Notably, these Fcgr2b-expressing cells exhibited a cytotoxic profile, and expressed granzyme M. Functional analysis using recombinant granzyme M revealed its potential in inducing inflammation in mouse fibroblasts and macrophages. Together, our study has identified Fcgr2b+CD49d− VM-like CD8 T cells in the adipose tissue of aged mice with regulatory, cytotoxic and inflammatory potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01438aa6f02a393a7c465e601e6991a4c53eadcb" target='_blank'>
              Single-cell RNA sequencing identifies accumulation of Fcgr2b+ virtual memory like CD8 T cells with cytotoxic and inflammatory potential in aged mouse white adipose tissue
              </a>
            </td>
          <td>
            Archit Kumar, Martin O’Brien, Vincent B. Young, Raymond Yung
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744c450e0fe105bb7eff34d3f9f47cc3b73c8c0c" target='_blank'>
              A patient-derived organoid model captures fetal-like plasticity in colorectal cancer.
              </a>
            </td>
          <td>
            Liang Xiong, Ying Xu, Zhaoya Gao, Jingyi Shi, Yunfan Wang, Xiaodong Wang, Wensheng Huang, Ming Li, Longteng Wang, Jun Xu, Cheng Li, Jin Gu, Hongkui Deng, Molong Qu
          </td>
          <td>2025-07-04</td>
          <td>Cell research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Primary tumors constantly shed cancer cells into the circulation, yet only a fraction of these cells manages to give rise to metastatic tumors. Successful metastatic seeding and growth appears to depend on metabolic changes within cancer cells. Here, using a metabolism-focused CRISPR screen in a spontaneous metastasis model, we found that expression of the enzyme γ-butyrobetaine hydroxylase 1 (BBOX1) in a subpopulation of tumor cells in various carcinomas enables immune evasion. The metabolite carnitine produced by BBOX1 inhibited the small GTPase RhoA in natural killer (NK) cells, preventing immunological synapse formation and thereby protecting metastatic cells. Loss of BBOX1 in tumor cells promoted their destruction by NK cells in vitro and improved the efficacy of NK cell adoptive transfer therapy in vivo. These findings illustrate how BBOX1-positive tumor cells hijack carnitine production to evade immune surveillance during metastasis and propose BBOX1 as a potential metabolic checkpoint for anti-metastatic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83c565b424262f8f0a380c1e10b4fb97fa55b5b" target='_blank'>
              BBOX1 is a Metabolic Checkpoint that Mediates Metastatic Cancer Cell Evasion from Immunosurveillance by Natural Killer Cells.
              </a>
            </td>
          <td>
            Chuanjie Zhang, Hongchao He, Yi Gao, Jiawei Ding, Hai Huang, Yixun Liu, Hanqing Liu, Jun Xiao, Dan-Feng Xu, Zunguo Du, Zhenbo Zhang, Lechi Ye
          </td>
          <td>2025-07-31</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a high-mortality cancer characterized by its aggressive, treatment-resistant phenotype and a complex tumour microenvironment (TME) featuring significant hypoxia. Bulk transcriptomic analysis has identified the “classical” and “basal-like” transcriptional subtypes which have prognostic value; however, it is not well-established how microenvironmental heterogeneity contributes to these transcriptional signatures. Here, we exploited the TRACER platform to perform single cell transcriptome analysis of organoids at specific spatial locations to explore the effect of oxygen and other cell-generated microenvironmental gradients on organoid heterogeneity. We found that the microenvironmental gradients present in TRACER significantly impact the distribution of organoid transcriptional phenotypes and the enrichment of gene sets linked to cancer progression and treatment resistance. More significantly, we found that microenvironmental gradients, predominantly oxygen, drive changes in the expression of classical and basal-like transcriptional subtype gene signatures. This work suggests that hypoxia contributes to determining transcriptional subtypes in PDAC tumour cells independent of additional cells in the TME and broadly highlights the importance of considering microenvironmental gradients such as oxygen in organoid-based studies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-98344-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d7b936c3248bb437b55ed176049e4ec9a8d43e" target='_blank'>
              Analysis of an engineered organoid model of pancreatic cancer identifies hypoxia as a contributing factor in determining transcriptional subtypes
              </a>
            </td>
          <td>
            Natalie Landon-Brace, Simon Latour, Brendan T. Innes, Michelle Nurse, Jose L. Cadavid, I. Co, Cassidy M. Tan, Ferris Nowlan, Sibil Drissler, F. Notta, H. Jackson, Gary D Bader, A. McGuigan
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Objectives This study aims to elucidate the complex molecular and cellular landscape of pancreatic ductal adenocarcinoma (PDAC) by identifying key regulatory non-coding RNAs (ncRNAs), hub protein-coding genes, and Intercellular communication pathways that may serve as prognostic biomarkers and therapeutic targets. Background Pancreatic cancer remains one of the deadliest malignancies worldwide, characterized by late diagnosis, limited treatment response, and poor prognosis. Among its histological subtypes, PDAC accounts for over 80% of cases and is defined by a highly fibrotic and immunosuppressive tumor microenvironment (TME). Methods We performed a comprehensive bioinformatics analysis integrating multiple transcriptomic datasets from the NCBI Gene Expression Omnibus (GEO), including mRNA, miRNA, lncRNA, and circRNA profiles from PDAC and adjacent normal tissues. Differential expression analysis was conducted using GEO2R, followed by functional enrichment via DAVID. Hub genes were identified from protein–protein interaction (PPI) networks constructed using STRING and validated using GEPIA2. A competing endogenous RNA (ceRNA) network was developed to investigate regulatory ncRNA–mRNA axes. To refine these findings, single-cell RNA-seq (scRNA-seq) data were analyzed to resolve the cellular origin of hub genes and ncRNAs, and CellChat was employed to model intercellular communication within the TME. Results We identified several dysregulated genes and ncRNAs implicated in key oncogenic pathways, including ECM remodeling, inflammation, and immune evasion. The ceRNA network highlighted functional interactions between circRNAs, lncRNAs, and miRNAs regulating key hub genes. Single-cell analysis revealed cell-type-specific expression of hub genes—e.g., FN1 and COL11A1 in fibroblasts, CXCL8 in macrophages, and ITGA3 in ductal cells—and uncovered a macrophage–endothelial CXCL8–ACKR1 signaling axis potentially driving tumor-associated angiogenesis. Moreover, correlations with immune cell infiltration and drug sensitivity further underscored the translational relevance of the identified molecular targets. Conclusion Our analysis combining bulk and single-cell transcriptomics provides a multi-scale view of PDAC pathogenesis. The findings highlight the interplay between ncRNAs, hub genes, and cellular crosstalk in shaping the tumor ecosystem and suggest novel targets for precision therapeutic intervention and biomarker development. Supplementary Information The online version contains supplementary material available at 10.1007/s10238-025-01815-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/588c78ae7417c5606a00168a20889c995ded56f7" target='_blank'>
              Combined bulk and single-cell transcriptomic analysis reveals cell-type-specific inflammatory crosstalk in pancreatic cancer
              </a>
            </td>
          <td>
            Ahmad Golestanifar, Mana Zakeri, Sahar Gohari-Lasaki, Hengameh Khedri, Mohammadreza Saberiyan
          </td>
          <td>2025-07-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a globally prevalent malignancy. This disease often progresses rapidly, resulting in many patients being diagnosed at a late stage, making early detection and intervention a major clinical challenge. Postoperative recurrence and metastasis rates remain significantly high, and no effective prevention strategies are currently available. Cancer-associated fibroblasts (CAFs) are essential components in the reorganization of the tumor microenvironment (TME), as they can modulate cancer cell proliferation, migration, invasion, and chemoresistance through diverse mechanisms or signaling pathways, including the release of cytokines, remodeling of the extracellular matrix, and the evasion of the immune response. This review offers a detailed overview of the cellular origins, subtype diversity, and functional differences among CAFs. In addition, it depicts the expression profiles of key markers in various CAF subtypes and clarifies essential signaling pathways and mechanisms of CAFs. Additionally, we discuss current and future therapeutic strategies targeting CAFs in the context of HCC. This review provides critical insights into future studies on novel therapeutic approaches for CAFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8451f01fad30bf68417ccee67bbd624a7786b63" target='_blank'>
              Cancer-associated fibroblasts in hepatocellular carcinoma: origins, heterogeneity, and therapeutic implications
              </a>
            </td>
          <td>
            Xin Shi, Weixiong Zhu, Jianpeng Zhang, Chuanlei Fan, Jing Zhang
          </td>
          <td>2025-07-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Although tumor cell heterogeneity between the tumor center (TC) and invasion front (IF) of oral squamous cell carcinoma (OSCC) is well documented, the morphological, molecular, and functional characteristics of cancer-associated fibroblasts (CAFs) in these regions remain poorly understood. Methods We examined hematoxylin and eosin (H&E)–stained OSCC sections to assess CAF morphology and correlation with patient prognosis. We then isolated paired CAFs from the tumor center (CAFTC) and invasion front (CAFIF) of four OSCC patients and compared their ECM-remodeling activity and pro-tumorigenic effects on OSCC cells. Furthermore, RNA sequencing identified differentially expressed genes between CAFTC and CAFIF. Finally, based on RNA-seq findings, we knocked down hyaluronan synthase 1 (HAS1) in CAFIF to evaluate its role in extracellular matrix (ECM) remodeling and tumor invasion. Results Compared to CAFTC, CAFIF exhibited a plump cell morphology and were associated with shorter disease-free survival. Functionally, CAFIF showed higher ECM-remodeling activity and more effective ability for promoting OSCC invasion and lymph node metastasis than CAFTC. RNA-seq identified HAS1 was significantly upregulated in CAFIF, promoting hyaluronic acid (HA) production and ECM remodeling. HAS1 knockdown in CAFIF diminished ECM remodeling and attenuated the ability of CAFIF to promoting OSCC invasion. Conclusion CAFIF with plump cell morphology showed pro-invasive abilities, driven in part by HAS1 overexpression and ECM remodeling, suggesting that targeting HAS1-driven ECM remodeling could be a promising therapeutic strategy. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03493-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6dfa996b747be7a23ea3bc4905b2cb5a5989f62" target='_blank'>
              HAS1high cancer associated fibroblasts located at the tumor invasion front zone promote oral squamous cell carcinoma invasion via ECM remodeling
              </a>
            </td>
          <td>
            Wanyong Jin, Qiuya Yu, Liyuan Yu, Ting Zhou, Xiren Wang, Wanqiu Lu, Xiaoxin Zhang, Liang Ding, Qingang Hu, Y. Ni
          </td>
          <td>2025-08-14</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is an aggressive subtype characterized by limited treatment options and poor prognosis. Recent evidence highlights the crucial role of cancer-associated fibroblasts (CAFs) in TNBC progression, yet their molecular characteristics remain incompletely understood. In this study, we performed a comprehensive analysis combining bioinformatics approaches with experimental validation to investigate CAF-related genes in TNBC. Using weighted gene co-expression network analysis (WGCNA) of TNBC samples from TCGA and METABRIC datasets, we identified 185 CAF-related genes significantly associated with extracellular matrix organization and TGF-β signaling pathways. Through rigorous statistical modeling, we developed a 3-gene prognostic signature (CERCAM, JAM3, PLAU) that effectively stratified TNBC patients into high- and low-risk groups with distinct survival outcomes. Importantly, we validated the functional role of PLAU, one of the signature genes, through in vitro and in vivo experiments. Results showed that CAF-derived PLAU played key role in the malignant behaviors of TNBC. Our findings provide new insights into CAF-mediated TNBC progression and suggest potential stromal targets for therapeutic intervention. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03867-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1bb9fe2fe0c3e330d474111c7b5267eb0d6dbf" target='_blank'>
              Stromal PLAU mediates tumor progression and informs a novel therapeutic target in triple-negative breast cancer
              </a>
            </td>
          <td>
            Jun Zou, Jing-yao Zhang, Yu Li, Baowen Yuan, Yuanyi Wang, Yalong Qi, Qian Wang, Wan Qin, Xianglin Yuan, Binghe Xu
          </td>
          <td>2025-07-11</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Breast cancer (BC) remains a leading cause of cancer-related mortality in women, with complex mechanisms driving its initiation, progression, and resistance to therapy. In recent years, the tumor microenvironment (TME) has gained attention for its critical role in shaping tumor behavior, where small extracellular vesicles (small EVs) have emerged as key mediators of intercellular communication. These vesicles carry a diverse cargo of proteins, lipids, DNA, and various non-coding RNAs—such as miR-21, miR-155, and miR-1246—mirroring the molecular status of their originating cells. This review highlights the roles of small EVs in immune modulation, stromal remodelling, and metastatic niche formation, emphasizing their contribution to therapy resistance and immune evasion. We discuss recent updates on EV biogenesis, characterisation and isolation techniques, such as ultracentrifugation, immunoaffinity and microfluidic systems. We also critically evaluate their potential for clinical application and how well they conform to the MISEV2023 guidelines. Furthermore, we examine small EVs as diagnostic tools in liquid biopsies and compare them with conventional methods such as mammography and tissue biopsies. We also discuss organotropism mediated by small EV cargo (e.g., integrins α6β4, αvβ5) and the diagnostic potential of protein and lipid signatures (e.g., PD-L1, CD63, and exosomal lipidomics). Therapeutically, we explore engineered small EVs for drug delivery, gene modulation, and immune activation, addressing challenges of targeting efficiency, in vivo stability, immunogenicity, and clinical scalability. The review discusses ongoing clinical trials involving small EVs in BC and highlights key translational gaps between preclinical advances and clinical implementation. Finally, we explores how integrating artificial intelligence, single-cell transcriptomics, and multi-omics approaches can help overcome major challenges such as small EV heterogeneity and tracking limitations. Crucially, this integration enables a more tailored understanding of each patient’s tumor biology, reducing therapeutic failures by guiding more personalized and effective treatment strategies. Overall, small EVs represent a transformative tool in precision oncology, contingent on resolving key challenges in their clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbe25e9e7298ee7befa5cf726da413903d9b701b" target='_blank'>
              Small extracellular vesicles: messengers at the service of breast cancer agenda in the primary and distant microenvironments
              </a>
            </td>
          <td>
            Angela Galardi, Valentina Fogazzi, Claudia Tottone, M. Giussani, S. Pupa, G. Cosentino, M. Iorio
          </td>
          <td>2025-07-21</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="ABSTRACT In humans, cell-cell communication orchestrates tissue organization, immune coordination, and repair, yet spatially mapping these interactions remains a challenge for biology. We introduce STARComm, a scalable-interpretable computational method that identifies Multicellular Communication Interaction Modules (MCIMs) by detecting spatially co-located receptor-ligand activity from high-plex spatial transcriptomics in 2D and 3D. Applied to an atlas of >14million cells across 8 cancers, STARComm revealed 24 conserved and tumor-specific MCIMs, including a fibro-immune module with targetable axes linked to immune exclusion and immunotherapy resistance. In chronic graft-versus-host disease, STARComm identified three salivary gland MCIMs predictive of patient death and two druggable axes ( CXCL12-CXCR4 , CCL5-SDC4 ), both with FDA-approved therapeutics. STARComm demonstrated that peripheral tissue profiling can forecast fatality nearly 3 years in advance using minor salivary glands. By enabling scalable biomarker discovery, drug targeting, and spatially resolved precision profiling, STARComm bridges the gap between spatial biology and clinical translation, advancing the field of spatial medicine. SUMMARY Despite major advances in spatial biology, no framework has yet linked spatially resolved intercellular communication networks, independent of cell types, to clinical outcomes in human disease. Here, we present STARComm, a scalable method that identifies Multicellular Interaction MCIMs (MCIMs). Applying STARComm to minor salivary gland biopsies from patients with chronic graft-versus-host disease (GVHD), we identify MCIMs that not only distinguish healthy from diseased tissue but also stratify patient survival. High-risk MCIMs are enriched for actionable immune and stromal pathways, including those targetable with existing therapies. These findings establish the first outcome-linked spatial communication framework in any human disease and highlight the translational potential of oral tissues as minimally invasive platforms for real-time immune diagnostics, prognostic modeling, and therapeutic screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bde44317cab9ca6c4ffa3edf62ce03cec0cdae" target='_blank'>
              STARComm Scalably Detects Emergent Modules of Spatial Cell-Cell Communication in Inflammation and Cancer
              </a>
            </td>
          <td>
            Khoa Huynh, B. Matuck, Deiziane de Souza, XiuYu Zhang, Giancarlo Fatobene, Luiz Alberto Valente Soares Júnior, L. F. Ferraz da Silva, V. Rocha, K. Byrd, Jinze Liu
          </td>
          <td>2025-08-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Oncogene amplification is a key driver of cancer pathogenesis and is often mediated by extrachromosomal DNA (ecDNA). EcDNA amplifications are associated with increased pathogenicity of cancer and poorer outcomes for patients. EcDNA can be detected accurately using fluorescence in situ hybridization (FISH) when cells are arrested in metaphase. However, the majority of cancer cells are non-mitotic and must be analyzed in interphase, where it is difficult to discern extrachromosomal amplifications from chromosomal amplifications. Thus, there is a need for methods that accurately predict oncogene amplification status from interphase cells. Here, we present interSeg, a deep learning-based tool to cytogenetically determine the amplification status as EC-amp, HSR-amp, or not amplified from interphase FISH images. We trained and validated interSeg on 652 images (40,446 nuclei). Tests on 215 cultured cell and tissue model images (9,733 nuclei) showed 89% and 97% accuracy at the nuclear and sample levels, respectively. The neuroblastoma patient tissue hold-out set (67 samples and 1,937 nuclei) also revealed 97% accuracy at the sample level in detecting the presence of focal amplification. In experimentally and computationally mixed images, interSeg accurately predicted the level of heterogeneity. The results showcase interSeg as an important method for analyzing oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a539ca1a35b8049ad46a136d79392bb325111564" target='_blank'>
              Accurate Prediction of ecDNA in Interphase Cancer Cells using Deep Neural Networks
              </a>
            </td>
          <td>
            Utkrisht Rajkumar, Gino Prasad, Ellis J Curtis, I. Wong, Xiaowei Yan, Shu Zhang, Lotte Brückner, Kristen Turner, Julie Wiese, Justin Wahl, Sihan Wu, J. Theißen, Matthias Fischer, Howard Y. Chang, A. Henssen, P. Mischel, V. Bafna
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8261c73bf75d788913526d1e389ee8b4f740f28c" target='_blank'>
              Inhibition of ACVR1 in Cancer-Associated Fibroblasts Suppresses Colorectal Cancer Cell Growth.
              </a>
            </td>
          <td>
            Shinya Kato, N. Miyoshi, S. Fujino, Masafumi Horie, Shinichi Yachida, Mitsunobu Takeda, Y. Sekido, Tsuyoshi Hata, Atsushi Hamabe, T. Ogino, Mamoru Uemura, Hirofumi Yamamoto, Masayoshi Yasui, M. Ohue, Y. Doki, H. Eguchi
          </td>
          <td>2025-07-31</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Surgical ablation or broad radiation of tumor-draining lymph nodes can eliminate the primary tumor response to immunotherapy, highlighting the crucial role of these nodes in mediating the primary tumor response. Here, we show that immunoradiotherapy efficacy is dependent on treatment sequence and migration of modulated dendritic cells from tumor to sentinel lymph nodes. Using a tamoxifen-inducible reporter paired with CITE-sequencing in a murine model of oral cancer, we comprehensively characterize tumor immune cellular migration through lymphatic channels to sentinel lymph nodes at single-cell resolution, revealing a unique immunologic niche defined by distinct cellular phenotypic and transcriptional profiles. Through a structured approach of sequential immunomodulatory radiotherapy and checkpoint inhibition, we show that sequenced, lymphatic-sparing, tumor-directed radiotherapy followed by PD-1 inhibition achieves complete and durable tumor responses. Mechanistically, this treatment approach enhances migration of activated CCR7+ dendritic cell surveillance across the tumor-sentinel lymph node axis, revealing a shift from their canonical role in promoting tolerance to driving antitumor immunity. Overall, this work supports rationally sequencing immune-sensitizing, lymphatic-preserving, tumor-directed radiotherapy followed by immune checkpoint inhibition to optimize tumor response to immunoradiotherapy by driving activated dendritic cells to draining sentinel lymph nodes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b99be24738c923c94e6ccc17f0f7d3c4a62f717" target='_blank'>
              The tumor-sentinel lymph node immuno-migratome reveals CCR7⁺ dendritic cells drive response to sequenced immunoradiotherapy
              </a>
            </td>
          <td>
            R. Saddawi-Konefka, R. A. Msari, Shiqi Tang, Chad Philips, S. Sadat, Lauren M. Clubb, Sarah Luna, Santiago Fassardi, Riley Jones, Ida Franiak Pietryga, F. Faraji, Shiruyeh Schokrpur, Bryan S. Yung, M. Allevato, Kelsey E Decker, C. Nasamran, Daisy R. Chilin-Fuentes, S. Rosenthal, S. M. Jensen, B. A. Fox, R. Bell, J. S. Gutkind, Andrew B Sharabi, Joseph A Califano
          </td>
          <td>2025-07-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="ABSTRACT Introduction Lung adenocarcinoma, the most common subtype of non‐small cell lung cancer, faces challenges such as drug resistance and tumor heterogeneity. N4‐acetylcytidine (ac4C) is an important RNA modification involved in cancer progression, but its role in lung adenocarcinoma remains unclear. Methods This study analyzed transcriptomic and single‐cell RNA sequencing data from public databases to investigate the expression and clinical significance of ac4C‐related genes in lung adenocarcinoma. Ten machine learning algorithms were applied to develop and validate an ac4C‐related gene signature (ARGSig) for prognosis prediction across multiple independent cohorts. Results Cells with high ac4C activity showed increased intercellular communication and activation of tumor‐associated pathways. The ARGSig model effectively stratified patients by survival outcomes and predicted sensitivity to immune checkpoint inhibitors and chemotherapy agents. Conclusion ac4C modification and its related genes play a critical role in lung adenocarcinoma development. The ARGSig model provides a promising molecular tool for prognosis evaluation and personalized treatment guidance in lung adenocarcinoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b6722e2c14c1989978a512d01d29289da459bd" target='_blank'>
              Integrating Single‐Cell Transcriptomics and Machine Learning to Define an ac4C Gene Signature in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Yuan Wang, Wei Su, Guangyao Zhou, Yijie Wang, Chunnuan Wu, Pengpeng Zhang, Lianmin Zhang
          </td>
          <td>2025-08-01</td>
          <td>Thoracic Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Breast cancer invasion and subsequent metastasis to distant tissues occur when cancer cells lose cell–cell contact, develop a migrating phenotype, and invade the basement membrane (BM) and the extracellular matrix (ECM) to penetrate blood and lymphatic vessels. The identification of the mechanisms which induce the development from a ductal carcinoma in situ (DCIS) to a minimally invasive breast carcinoma (MIBC) is an emerging area of research in understanding tumor invasion and metastatic potential. To investigate the progression from DCIS to MIBC, we analyzed peritumoral collagen architecture using correlative scanning electron microscopy (SEM) on histological sections from human biopsies. In DCIS, the peritumoral collagen organizes into concentric lamellae (‘circular fibers’) parallel to the ducts. Within each lamella, type I collagen fibrils align in parallel, while neighboring lamellae show orthogonal fiber orientation. The concentric lamellar arrangement of collagen may physically constrain cancer cell migration, explaining the lack of visible tumor cell invasion into the peritumoral ECM in DCIS. A lamellar dissociation or the development of small inter fiber gaps allowed isolated breast cancer cell invasion and exosomes infiltration in the DCIS microenvironment. The radially arranged fibers observed in the peri-tumoral microenvironment of MIBC biopsies develop from a bending of the circular fibers of DCIS and drive a collective cancer cell invasion associated with an intense immune cell infiltrate. Type I collagen fibrils represent the peri-tumoral nano-environment which can play a mechanical role in regulating the development from DCIS to MIBC. Collectively, it is plausible to suggest that the ECM effectors implicated in breast cancer progression released by the interplay between cancer, stromal, and/or immune cells, and degrading inter fiber/fibril hydrophilic ECM components of the peritumoral ECM, may serve as key players in promoting the dissociation of the concentric collagen lamellae.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca42ff86f8ea9dc2e92ec4093986b47c8d0700cf" target='_blank'>
              Ultrastructural Changes of the Peri-Tumoral Collagen Fibers and Fibrils Array in Different Stages of Mammary Cancer Progression
              </a>
            </td>
          <td>
            Marco Franchi, Valentina Masola, Maurizio Onisto, Leonardo Franchi, Sylvia Mangani, V. Zolota, Z. Piperigkou, Nikos K. Karamanos
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Stromal cells are found in all tissues of the body. Among them, lymphoid stromal cells (LSCs) correspond to the cell subsets found in secondary and tertiary lymphoid organs. LSC heterogeneity has been characterized in depth in mice based on cell-fate mapping, high-resolution imaging and single-cell RNAseq analysis, and more recently in humans despite the difficulty of accessing these rare cell populations. At steady-state, LSCs organize discrete anatomical niches in lymphoid organs and orchestrate adaptive immune response. Studies of LSCs at the single cell level have identified a wide role for these cells in various pathological conditions, including solid tumors, autoimmune diseases, and lymphomas. In this review, we will discuss the diversity and plasticity of LSCs and LSC-like cells as well as their functions in pathological settings, with a focus on cancer and autoimmune diseases. Altogether, it highlights the importance of increasing our understanding of these cells, to use them as a target for novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae385fc73dd8ff4540908af0f79aa0db93c9021" target='_blank'>
              Lymphoid stroma in all its states
              </a>
            </td>
          <td>
            Nicolas Barbier, Valentin Isen, Karin Tarte, David Roulois
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Innovative treatment strategies for pituitary tumors are necessary to limit the disease burden and to improve survival in cases of carcinomas. The paucity and inaccuracy of available preclinical models substantially hamper pituitary research and drug discovery. Hence, we describe a novel method to generate orthotopic pituitary tumors via stereotaxic injection of somatotroph GC cells into the pituitaries of immunocompetent Wistar Furth rats. Tumor growth was monitored by repeated 7 Tesla magnetic resonance imaging. The procedure consistently led to rapidly expanding intra- and suprasellar growth hormone-secreting tumors within their native anatomical environment. The generated tumors faithfully reproduced the microarchitecture of human somatotroph pituitary adenomas, including the immune infiltrates and other typical components of their microenvironment, which is a prerequisite for testing immunomodulating agents. This orthotopic model of proliferative pituitary tumors developed in immunocompetent hosts therefore unlocks new opportunities for preclinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a14d8df9146cc44ada8fa140ac68c8412f2d3b72" target='_blank'>
              Orthotopic pituitary tumors generated by stereotaxic GC cell injection in immunocompetent rats
              </a>
            </td>
          <td>
            Etienne Lefevre, Fanny Chasseloup, N. Ladurelle, Clément Janot, Jean Laurent Thibaud, Isabelle Beau, Clovis Adam, Céline des Courtils, P. Zizzari, Alexandre Carpentier, P. Kamenickỷ
          </td>
          <td>2025-07-05</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="For years, valuable clinical samples preserved in formalin-fixed paraffin-embedded tissues were underutilized. However, with advanced spatial multiomics profiling tools, crucial information has become increasingly accessible. Integrating genomic data with spatial information has unveiled crucial insights into cellular activities, enhancing our comprehension of biology. Measuring cellular gene expression while capturing spatial context – including morphology and intercellular relationships – is vital for understanding both normal and diseased biological processes. To date, this approach has illuminated the mechanisms of complex diseases, such as cancer and has facilitated the discovery of biomarkers for early disease detection and new therapeutic targets, accelerating progress in cancer immunotherapies. Cutting-edge single-cell analysis tools are rapidly emerging as the gold standard for investigating intricate biological systems and medical specimens, fueling a multi-billion-dollar industry. Single-cell spatial research, in particular, is inherently cross-disciplinary and addresses questions that remain hidden when focusing solely on the genome or transcriptome of large cell populations. Leveraging advances in single-cell spatial profiling can offer insights into improving cancer immunotherapy and other modern medical treatments. This review will delve into the diverse applications of spatial profiling technology, showcasing examples that demonstrate its ability to provide a detailed picture of the underlying molecular and cellular mechanisms within cells. As a comprehensive reference, this review empowers researchers and industry leaders to harness single-cell and spatial omics for breakthroughs in biomedicine and translational science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33ed8c77341ba16d0b5a75b9927877b08daea9ef" target='_blank'>
              In situ spatial profiling: Gaining molecular and cellular insights within content and context
              </a>
            </td>
          <td>
            Shu-Ti Lin, Ian Molyneaux, Chen Yeh
          </td>
          <td>2025-07-10</td>
          <td>Gene &amp; Protein in Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) is a leading cause of cancer-related deaths worldwide. Despite advancements in treatment, prognosis for patients with advanced stages remains poor. Metabolic reprogramming in the tumor microenvironment, particularly abnormal glycolysis, plays a crucial role in immune evasion and treatment response. We collected nine single-cell datasets to create a single-cell atlas of CD8 + T cells from 89 NSCLC patients, revealing ten distinct states of these cells. We employed a multimodal data analysis approach, integrating bulk transcriptomics, single-cell transcriptomics, spatial transcriptomics, and proteomics. Using 117 machine learning models, we identified key genes associated with NSCLC metastasis. Notably, the StepCox[forward] + Lasso model was instrumental in pinpointing key genes that significantly impact disease prognosis. Our analysis revealed that LTB + LDHA + CD8 + T cells have a distinct metabolic and immune phenotype, characterized by enhanced glycolysis and elevated lactate production. This not only facilitates tumor cell migration and invasion but also impairs the cytotoxic function of CD8 + T cells. Furthermore, our machine learning models identified four key genes significantly associated with NSCLC metastasis: TBCD, PTPRC, LDHA, and ACTR2. Of these, high LDHA expression was strongly linked to poorer responses to immunotherapy and a higher risk of therapy resistance. LTB + LDHA + CD8 + T cells also reduced antitumor immune responses by inhibiting the secretion of effector molecules like GNLY. Additionally, elevated LDHA expression was associated with reduced CD8 + T cell infiltration, which further promotes tumor immune evasion. This study highlights the heterogeneity of CD8 + T cells in NSCLC, emphasizing the unique role of the LTB + CD8 + Tn subpopulation in metastasis. LDHA is identified as a critical key gene with a significant impact on immunotherapy outcomes, presenting a potential therapeutic target. These insights offer new biomarkers and targeted strategies for personalized immune therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e078cc67ce9052ada65f40295ba6296864dfb9f6" target='_blank'>
              Metabolic alterations driven by LDHA in CD8 + T cells promote immune evasion and therapy resistance in NSCLC
              </a>
            </td>
          <td>
            Hao Wen, Panpan Zhang, Juan Zhao, Yakui Liu, Lei Wan, Haoran Li, Jun Yi, Xinqiang Li
          </td>
          <td>2025-07-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24a4c1ebff0edca5f86402765983e14a517f49f7" target='_blank'>
              Form Follows Function: A New Paradigm of Pancreatic Cancer Progression.
              </a>
            </td>
          <td>
            William Gasper, Giada Pontecorvi, Zhikai Chi, S. Enrico, Lily C. Goodman, Haneen Amarneh, Evelyn Mazzarelli, Joshua Thomas, Lucia Zhang, Rachel Handloser, Seth LeCates, A. L. Tomescu, Francesca Rossi, Myla Sykes, Anjali Bhatt, Nicholas Fazio, Alessandro Bifolco, Nafeesah Fatimah, Austin Seamann, Cheryl M Lewis, Shelby Melton, Herbert J Zeh, Megan B. Wachsmann, Dario Ghersi, Matteo Ligorio
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Patient-derived xenograft (PDX) models are essential for understanding the pathophysiology and developing treatment strategies for breast cancer brain metastasis (BCBM). While immunodeficient mouse models allow for human BCBM growth, their small size limits host survival, neurological imaging and therapeutic interventions. This study evaluated the immunodeficient SRG rat (Sprague Dawley Rag2−/−; Il2rg−/−) as a new intermediate-sized host for orthotopic modeling of human BCBM. The primary goal was to determine if the SRG rat brain presents a hospitable environment for orthotopic growth of patient BCBM cells. A secondary goal was to compare phenotypes of the patient and xenografted tumors. Adult SRG rats received stereotactic intracerebral implants of 1 million engineered patient BCBM cells. Bioluminesence imaging (BLI) and magnetic resonance imaging (MRI) provided metabolic and anatomic monitoring of tumor growth. Post-mortem histological analysis compared biomarker profiles in original patient and xenograft tumors. Orthotopic patient-derived BCBM tumors progressed in all SRG rats within 5 weeks post-implantation. BLI radiance increased 125-fold over the study period. MRI revealed tumor-induced brain edema and both patient and xenograft BCBMs exhibited pronounced vascularity and gadolinium enhancement. Histopathology confirmed that xenograft tumors maintained high proliferation indices and biomarker expression consistent with the parent tumor. The SRG rat provided a reliable intermediate-sized host for orthotopic growth of patient-derived BCBM xenografts, offering advantages over existing models for studying tumor behavior and therapeutic responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51569fb41107b3b3d9a9ec6748cc5af1d6d2b8bc" target='_blank'>
              The SRG rat as a novel host for an orthotopic patient-derived xenograft model of breast cancer brain metastasis
              </a>
            </td>
          <td>
            N. Fulcher, Hien Nguyen, A. Deweyert, M. Uzelac, Maryam Mozaffari, Qi Zhang, John Ronald, John J. Kelly, Ying Xia, Timothy J Scholl, Fallon K. Noto, D. Begemann, Michael J. Schlosser, Susanne Schmid, M. Hebb
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e9ad63d60d3424441f26d5aee0efbd26af0b9f" target='_blank'>
              A single-cell atlas of ribosomal protein heterogeneity across human tissues reveals phenotypes of biological and clinical significance
              </a>
            </td>
          <td>
            Aishwarya Murali, Chartsiam Tipgomut, Anand D. Jeyasekharan, Himanshu Sinha
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Lung cancer, predominantly non-small cell lung cancer (NSCLC), remains a principal driver of cancer-related morbidity and mortality worldwide. Despite advancements in surgery, radiotherapy, chemotherapy, and targeted treatments, outcomes remain poor in advanced NSCLC. The tumor microenvironment (TME) exerts a critical influence on therapy responses. Within the TME, immune cells such as T and B lymphocytes, dendritic cells, myeloid-derived suppressor cells, tumor-associated macrophages, neutrophils, and natural killer cells can drive both pro- and anti-tumor processes. This review integrates their classification, phenotypic plasticity, and roles in NSCLC, highlighting key preclinical and clinical evidence while discussing pathogenesis, prognostic significance, and therapeutic potential. We also summarize the current immunotherapeutic strategies for advanced NSCLC, including first- or second-line regimens with immune checkpoint inhibitors alone or combined with chemotherapy, anti-angiogenic agents, or additional checkpoint inhibitors, and future directions. By elucidating the interplay between the NSCLC immune microenvironment and emerging immunotherapies, this review emphasizes the need for novel combination regimens and robust predictive biomarkers to improve clinical outcomes and extend survival in advanced NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d594d55b67aa26998978e57ef768648090cd4cd" target='_blank'>
              Tumor-infiltrating lymphocytes in NSCLC: from immune surveillance to immunotherapy
              </a>
            </td>
          <td>
            Haiyi Xue, Yilan Fan, Yi Li, Qian Zhao, Xuelu Zhang, Pei Zhao, Zhenjun Liu
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background The N-formyl peptide receptor family (FPRs) is implicated in the progression of diverse cancer types, yet studies specifically exploring their roles in breast cancer remain scarce. Methods A comprehensive analysis integrating bulk RNA-seq transcriptomics, methylomics, single-cell transcriptomics, and spatial single-cell transcriptomics data was conducted to elucidate the distinctive characteristics of FPRs in breast cancer. This study particularly focused on delineating the e xpression profiles of FPR3 across distinct breast cancer subtypes, while systematically investigating its prognostic implications and association with macrophage polarization patterns in breast cancer patients. Furthermore, molecular docking analysis was performed to screen potential therapeutic compounds targeting FPR3, providing insights into its druggability. Results Notably, FPR3 was found to be highly expressed in macrophages within breast cancer tissues, with a notably elevated level in HER2-positive and triple-negative breast cancer (TNBC) subtypes, both of which are associated with poor prognosis. FPR3 expression inversely correlates with promoter methylation levels. Further analysis of pan-cancer immune infiltration patterns uncovered a striking association between FPR3 and macrophage infiltration, as well as their polarization status. Knockdown of FPR3 expression in macrophages markedly enhanced the expression of IL6, TNF-α, and TGF-β, while significantly reducing IL10 levels, indicative of a shift towards an M1-like macrophage phenotype. Through computational molecular docking analyses, Otamixaban and Rivaroxaban emerged as promising candidate inhibitors of FPR3. Conclusions These findings underscore the profound infiltration of FPR3 + macrophages in breast cancer patients with adverse prognoses, highlighting FPR3 as a potential therapeutic target for intervening breast cancer aggressiveness. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03942-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18854fe793c2d29227b32ba5f47639fad4edcf8c" target='_blank'>
              FPR3 orchestrates macrophage polarization in breast cancer: multi-omics dissection of prognostic relevance and therapeutic targeting
              </a>
            </td>
          <td>
            Ying Chen, Xin Tang, Liran Zhu, Yi Wang, Gaopeng Li, Wulin Yang
          </td>
          <td>2025-08-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC), the most common RCC subtype, displays significant intratumoral heterogeneity driven by metabolic reprogramming, which complicates our understanding of disease progression and limits treatment efficacy. This study aimed to construct a comprehensive cellular and transcriptional landscape of ccRCC, with emphasis on gene expression dynamics during malignant progression. An integrated analysis of 90 scRNA-seq samples comprising 534,227 cells revealed a progressive downregulation of sodium ion transport-related genes, particularly CHP1 (calcineurin B homologous protein isoform 1), which is predominantly expressed in epithelial cells. Reduced CHP1 expression was confirmed at both mRNA and protein levels using bulk RNA-seq, CPTAC proteomics, immunohistochemistry, and ccRCC cell lines. Survival analysis showed that high CHP1 expression correlated with improved prognosis. Functional analyses, including pseudotime trajectory, Mfuzz clustering, and cell–cell communication modeling, indicated that CHP1+ epithelial cells engage in immune interaction via PPIA–BSG signaling. Transcriptomic profiling and molecular docking suggested that CHP1 modulates amino acid transport through SLC38A1. ZNF460 was identified as a potential transcription factor of CHP1. Virtual screening identified arbutin and imatinib mesylate as candidate CHP1-targeting compounds. These findings establish CHP1 downregulation as a novel molecular feature of ccRCC progression and support its utility as a prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a152ff7c27c9a4d5b4b572fd6cc9cf8e33bcba91" target='_blank'>
              Targeting Sodium Transport Reveals CHP1 Downregulation as a Novel Molecular Feature of Malignant Progression in Clear Cell Renal Cell Carcinoma: Insights from Integrated Multi-Omics Analyses
              </a>
            </td>
          <td>
            Yun Wu, Ri-Ting Zhu, Jiaru Chen, Xiao-Min Liu, Guo-Liang Huang, Jin-Cheng Zeng, Hong-Bing Yu, Xin Liu, Cuifang Han
          </td>
          <td>2025-07-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The anti-tumor role of the immune system has long been associated with interferon-γ-mediated activation of immune cells and their ability to recognize and eliminate transformed cells. Fundamental principles of tumor immunoediting describe a dynamic interplay between the immune system and neoplastic cells, wherein immune pressure can paradoxically shape tumor evolution. Within this context, macrophages, natural killer cells, and T lymphocytes are central effectors of anti-tumor immunity. Traditionally, macrophages exhibiting M1 phenotype are characterized by high cytotoxic potential and considered important contributors to tumor eradication. In contrast, M2-polarized tumor-associated macrophages are associated with immune suppression and tumor progression. However, recent evidence challenges this binary paradigm. It is increasingly evident that M1 macrophages, while initially exerting anti-tumor effects, can also promote tumor progression by applying sustained cytotoxic pressure that selects for more malignant and immune-resistant tumor clones. This phenomenon represents an unexpected and overlooked contribution of cytotoxic macrophages to tumor progression. In this review, we examine the complex, context-dependent function of M1 macrophages and reassess current strategies aimed at enhancing their cytotoxicity. While such approaches may offer short-term benefits, they risk driving clonal selection of aggressive, immune-evasive tumor cells. Therefore, we propose a paradigm shift: instead of promoting M1 polarization alone, therapeutic strategies should consider the broader consequences of macrophage–tumor interactions. A nuanced understanding of macrophage plasticity and tumor dynamics is essential for designing effective immunotherapies. Recognizing the paradoxical role of M1 macrophages is critical to avoiding unintended support of tumor evolution and improving treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d17eeff6a62b58600242672f5016da3be414f80d" target='_blank'>
              M1 macrophages – unexpected contribution to tumor progression
              </a>
            </td>
          <td>
            Olga V. Kovaleva, M. Rashidova, V. V. Sinyov, O. S. Malashenko, Alexei N. Gratchev
          </td>
          <td>2025-07-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Head and neck squamous cell carcinoma is a challenging cancer that often resists treatment due to its ability to escape the body’s immune defense. This review explains how the tumor’s surrounding environment—made up of immune cells, blood vessels, and structural proteins—helps the cancer grow, spread, and avoid immune attack. We aim to highlight how different components in this environment, including fibroblasts, suppressive immune cells, and genetic changes like TP53 mutations, contribute to disease progression. By understanding these mechanisms, researchers can develop more effective therapies that target not just the cancer cells, but also the supportive environment around them. This review may guide future research in creating personalized treatments and improving outcomes for patients with this aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec6e0af15a343b955ed9de45b18fba4fac507f2" target='_blank'>
              Immune Evasion in Head and Neck Squamous Cell Carcinoma: Roles of Cancer-Associated Fibroblasts, Immune Checkpoints, and TP53 Mutations in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Chung-Che Tsai, Yi-Chiung Hsu, Tin-Yi Chu, Po-Chih Hsu, Chan-Yen Kuo
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="OBJECTIVE
Adamantinomatous craniopharyngioma (ACP) is an epithelial tumor that occurs in the sellar region of the human brain. Despite resection, relapse is frequent with poor prognosis. To facilitate the development of new therapy for ACP, the authors examined the spatial distribution, cell of origin, and potential biological functions of leukemia inhibitory factor (LIF), an important stem cell self-renewal regulator, in a series of ACP tumors.


METHODS
The transcriptional sites of LIF and LIF receptor (LIFR) were determined by single-cell sequencing and space transcriptome analysis. LIF and LIFR distribution characteristics in different histopathological regions were detected with immunohistochemistry and immunofluorescence analysis. The relationships between the regional distributions of different tissues and tumor imaging characteristics, tumor cell stemness, cell proliferation, LIF expression, and patient prognosis were analyzed.


RESULTS
The authors' analysis of 39 ACPs detected LIF overexpression that was selectively enriched in cell clusters. In addition to the discovery of the stromal cells in the interstitial region, palisade epithelium, and stellate reticulum as the source cells of LIF production, the authors also revealed that LIFR was primarily generated by the cell clusters. Examination of differentially expressed genes between LIF-high and LIF-low ACP tumors indicated that the binding of LIF to LIFR may lead to the activation of the PI3K-AKT signaling pathway. Further analysis showed enrichment of LIF expression in β-catenin-positive cell clusters expressing stem cell markers of CD44, supporting its role in stem cell self-renewal. Integrated analysis with diagnostic imaging found higher level expression of LIF in cystic tumors than in solid tumors, displaying a trend toward poorer prognosis.


CONCLUSIONS
This study confirmed for the first time that LIF in ACP mainly originates from tumor microenvironment stroma. The authors' data suggest that future efforts should also include tumor stromal cells as a novel cellular and/or molecular target when developing new anticancer therapies against ACP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90e38d490dae9950f240b893ac7de6f57947b7a0" target='_blank'>
              Use of integrated spatial transcriptomics and histopathological analysis in adamantinomatous craniopharyngiomas to identify stromal cells as a new cellular source of leukemia inhibitory factor.
              </a>
            </td>
          <td>
            Wenxin Hu, Chuan Zhao, Wenrong Zheng, Yi Lin, Ning Luo, Hongxing Liu, Xingfu Wang, Xueling Qi, Xianlong Wang, Xiao-Nan Li, Zhixiong Lin
          </td>
          <td>2025-08-01</td>
          <td>Journal of neurosurgery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d94128a4ca71f38899bbd2a679a0555681c49d19" target='_blank'>
              Microenvironment crosstalk and immune evasion of circulating tumor cells: From mechanism to clinical significance.
              </a>
            </td>
          <td>
            Shi Chen, Sirui Zhang, Ying Shi, Xiaowei Liu
          </td>
          <td>2025-08-08</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="As DNA variants accumulate in somatic stem cells, become selected or evolve neutrally, they may ultimately alter tissue function. When, and how, selection occurs in homeostatic tissues is incompletely understood. Here, we introduce SCIFER, a scalable method that identifies selection in an individual tissue, without requiring knowledge of the driver event. SCIFER also infers self-renewal and mutation dynamics of the tissue’s stem cells, and the size and age of selected clones. Probing bulk whole-genome sequencing data of nonmalignant human bone marrow and brain, we detected pervasive selection in both tissues. Selected clones in hematopoiesis, with or without known drivers, were initiated uniformly across life. In the brain, we found pre-malignant clones with glioma-initiating mutations and clones without known drivers. In contrast to hematopoiesis, selected clones in the brain originated preferentially from childhood to young adulthood. SCIFER is broadly applicable to renewing somatic tissues to detect and quantify selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6381edb3d3b2ef9cddaaae818d10d5852bbd41f" target='_blank'>
              Detecting and quantifying clonal selection in somatic stem cells
              </a>
            </td>
          <td>
            Verena Körber, N. A. Jakobsen, N. Ansari-Pour, Rachel Moore, Nina Claudino, M. Metzner, Eva Thielecke, Franziska Esau, B. Usukhbayar, Mirian Angulo Salazar, Simon Newman, B. Kendrick, Adrian H Taylor, Rasheed Afinowi-Luitz, R. Gundle, B. Watkins, Kim Wheway, D. Beazley, S. Dakin, Antony Palmer, Andrew J. Carr, Paresh Vyas, Thomas Höfer
          </td>
          <td>2025-07-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related mortality globally, with non-small cell lung cancer (NSCLC) representing 85% of cases. Advances in treatment modalities, including stereotactic radiation therapy, have improved outcomes. However, possible synergistic effects of these therapies remain underexplored at the molecular level. This study investigated high-dose radiation-induced proteomic changes in lung adenocarcinoma cell line HCC-44 grown adherently and cell line A549, grown as adherent cells or spheroids. Our hypothesis was that proteins upregulated by 10 Gy irradiation serve as resistance drivers in cancerous cells and can thus represent potential therapeutic targets. The label-free mass spectrometry revealed distinct proteomic responses to 10 Gy irradiation, varying by cell line and culturing conditions. Differentially expressed proteins elevated in the irradiated samples included ephrin type-A receptor 2 (EPHA2) in adherent cells and insulin-like growth factor 2 receptor, tetraspanin 3 as well as cathepsin D (CTSD) in spheroids. The validation of these targets was carried out via Western blot, immunofluorescence, viability assay and spheroid formation assay which demonstrated that irradiation sensitized adherent NSCLC cells to EPHA2 and CTSD inhibitors. These findings underscore the potential of integrating radiation and targeted therapies in NSCLC treatment, and highlight EPHA2 as a promising candidate for future therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-09285-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b394ae32bb3fba2e6d32961ffafe24b8e9f20a6" target='_blank'>
              Proteome changes associated with effect of high dose single-fractionation radiation on lung adenocarcinoma cell lines
              </a>
            </td>
          <td>
            Elina Leis, Vijayachitra Modhukur, H. Lust, M. Vardja, Merilin Saarma, Ivar Ilves, Margaret Pütsepp, Sergei Kopanchuk, Jana Jaal, D. Lavogina
          </td>
          <td>2025-07-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Medulloblastoma is an aggressive central nervous system tumor affecting children more commonly between the ages of 5-9. It is usually localized in the cerebellum, leading to diffusion of tumor cells through the cerebrospinal fluid and metastases to other portions of the brain and spinal cord. Conventional treatment consists of surgical resection followed by adjuvant radiation and/or chemotherapy. The side effects of these therapies are critical to consider, especially given that patients are in a distinct stage of their lives. In addition, the overall survival is not satisfactory ranging from 50-90% depending on the type of medulloblastoma. The molecular characterization has broadly subdivided medulloblastoma into four subgroups, and more recently, the single-cell transcriptomics studies have further identified several other subgroups. Important advances have been reported on the cell origin, their plasticity, heterogeneity of genetic and epigenetic alteration, and interaction with the immune and stromal components of the tumor microenvironment. Research studies on these key points are essential to make advances in planning the application of conventional therapies together with immunotherapies. Herein, we discuss the main advances recently obtained on medulloblastoma biology and immunotherapies. Overall, the biological and molecular features of medulloblastoma are briefly summarized to understand the reason for the application of the old and new immunotherapies. Immunotherapies considered include the identification of potential medulloblastoma neoantigens and tumor-associated antigens to generate antigen-specific T lymphocytes. The main antigens expressed by medulloblastoma cells and/or by components of the tumor microenvironment will be considered as the molecular targets of antibodies, antibody derivatives, and chimeric antigen receptor effector cells to improve the conventional therapies. In the last portion of this review, the brief analysis of the activating and inhibiting receptors expressed by antitumor T, natural killer, and unconventional T cells can give new insights into the potential treatment of medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66089ed496a881bda16d192ca239b61f3eeade79" target='_blank'>
              Medulloblastoma: biology and immunotherapy
              </a>
            </td>
          <td>
            Alessandro Poggi, Francesco Reggiani, Helena S. Azevedo, L. Raffaghello, Rui Cruz Pereira
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Cervical cancer’s tumor microenvironment (TME) was composed of a diverse array of immune cells that significantly influence tumor progression and response to treatment. Recent advancements in multi-omics and single-cell sequencing had provided valuable insights into the cellular heterogeneity and immune landscape of the TME, revealing critical interactions that shape tumor behavior and therapy outcomes. Method This study used multi-omics and single-cell sequencing to explore the immune landscape, cellular heterogeneity, and drug sensitivity in cervical cancer, focused on tumor subtypes and their interactions with immune cells, and aimed to understand therapy responses. Results The research presented a thorough single-cell analysis of cervical cancer, identified distinct tumor epithelial cell (EPC) subtypes, and explored their roles in tumor progression, immune evasion, and therapeutic response. It underscored the potential of tumor EPCs as valuable biomarkers for prognosis and as targets for personalized treatment approaches. Conclusion The immune landscape of cervical cancer and its interaction with tumor endothelial progenitor cells played crucial roles in determining the tumor’s progression and response to therapy. The classification of tumor subtypes based on immune characteristics and drug sensitivity was critical for personalized treatment. The identification of TSPAN1 as key biomarkers provided insight into tumor biology and potential therapeutic targets. Our findings emphasized the need for combining immune checkpoint modulation with precise drug sensitivity analysis to optimize treatment strategies, particularly in advanced cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db53e3e78ba135d282fa510797c202842bdafc26" target='_blank'>
              Revolutionizing cervical cancer treatment: single-cell sequencing of TSPAN1+ tumor EPCs and immune checkpoints to assess drug sensitivity and optimize therapy
              </a>
            </td>
          <td>
            Yumeng Li, Zhiheng Lin, Guangyao Lin, Zhijie Zhao, Zhikai Xiahou, Pingping Cai
          </td>
          <td>2025-07-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4144c066603d461af67a1609df3300fe3c044e71" target='_blank'>
              Single-cell and spatial transcriptomics reveal mechanisms of radioresistance and immune escape in recurrent nasopharyngeal carcinoma.
              </a>
            </td>
          <td>
            Rui You, Qunlun Shen, Chao Lin, Kangning Dong, Xiao Liu, Hanshi Xu, Wanming Hu, Yulong Xie, Ruoqi Xie, Xiaoyi Song, Chunliu Huang, Jinhui Wu, Tao Yu, Huifeng Li, Zining Wang, Keming Chen, Xiong Zou, Peiyu Huang, Yijun Hua, Youping Liu, Tianliang Xia, Shihua Zhang, Mingyuan Chen
          </td>
          <td>2025-07-21</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Investigating the heterogeneous responses of individual cancer cells to chemotherapeutic drugs is crucial for deciphering the mechanisms of cancer drug resistance. In recent years, single-cell proteomics has demonstrated its significant capability in exploring drug response in cancer cells. Meanwhile, there are increasing reports suggesting that the cellular morphology is potentially associated with drug resistance. However, integrating the single-cell proteomic results with morphological information remains challenging. Here, we present a morphology-aware single-cell proteomic analysis (Morp-SCP) platform to precisely capture cells of interest with real-time and high-resolution imaging, and to conduct deep proteomic analysis at the single-cell level, providing multidimensional information on the target single cells. The Morp-SCP platform was applied for exploring the time-dependent proteomic alterations of human nonsmall cell lung cancer cells (A549) upon cisplatin exposure. Subpopulations of drug-resistant A549 cells were identified, which exhibited distinct proteomic and morphological patterns when resisting cell death induced by cisplatin exposure. By revealing the proteomics-morphology relationship, the Morp-SCP platform offers an effective strategy to provide insights into the heterogeneity of drug resistance at the single-cell level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/439c211eca558887357b434ba49c5af9cf3c9266" target='_blank'>
              Integrated Single-Cell Proteomic and Morphometric Analysis Reveals Heterogeneous Drug-Resistant Subpopulations.
              </a>
            </td>
          <td>
            Qian-Xi Fan, Yi-Rong Jiang, Jian-Bo Chen, Yi-Xue Chen, Qin-Qin Xu, Jie Wu, Huifeng Wang, Yi Yang, Jian-Zhang Pan, Qun Fang
          </td>
          <td>2025-07-07</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT KRAS is mutationally activated in 45-50% of colorectal cancer (CRC) cases, and while KRAS-targeted therapies have shown some clinical promise, upfront and acquired resistance limit their efficacy. To explore the acute response and mechanisms underlying KRAS inhibitor resistance, we used targeted exome sequencing and single-cell spatial transcriptomics to analyze patient-matched pre-treatment, on-treatment, and progression biopsies from patients treated with combined KRAS G12C and EGFR inhibition. Acquired genetic events were identified in most patients at progression but were often subclonal and coexisted with transcriptional adaptive states. Mesenchymal, YAP, and fetal-like transcriptional signatures predominated in resistant tumors, while tumor cell-intrinsic inflammatory programs were induced in the early treatment phase. Single-cell spatial analysis revealed significant intratumoral heterogeneity, with diverse adaptive states predominating in different zones of individual tumors. Using human and murine organoid models, we show that these drug-induced inflammatory programs are cancer-cell autonomous and precede the emergence of regenerative fetal-like programs associated with drug resistance. We uncover TBK1 as a promising target to abrogate the early inflammatory adaptive phase and enhance responses to KRAS inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75a7967138a21f862a2f0477898f16663439bb11" target='_blank'>
              Concurrent genetic and non-genetic resistance mechanisms to KRAS inhibition in CRC
              </a>
            </td>
          <td>
            Salvador Alonso, Kevan Chu, Marie J Parsons, Elizabeth Granowsky, H. Gunasinghe, Jinru Shia, R. Yaeger, Lukas E. Dow
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8217431a238aa68e23274392bf64da3b8ab3791b" target='_blank'>
              An organotypic in vitro model of human papillomavirus-associated precancerous lesions allowing automated cell quantification for preclinical drug testing
              </a>
            </td>
          <td>
            R. Köhler, Hans-Jürgen Stark, Iris Martin, Joel Altmann, M. S. Kalteis, Magnus von Knebel Doeberitz, Elena-Sophie Prigge
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Colorectal cancer (CRC) occurs as the third most common cancer with high mortality across the world. Understanding the intratumoral immune cell heterogeneity and their responses to various therapies is crucial for enhancing patient outcomes. This study aimed to characterise and evaluate the immune microenvironment landscapes of CRC shaped by different therapies including CD73 inhibitor, PD‐1 blockade and photothermal therapy (PTT). Our investigation revealed that three therapies could commonly modulate the down‐regulation of Treg, M2 macrophage and Ptprj+ G4 granulocyte, up‐regulation of effector/memory T cell, M1 macorphage and Hilpda+ G1 granulocyte. Moreover, we identified the uniquely dis‐regulated cell types and pathway activities response to each therapy, such as CD73 inhibitor enriched more Cd8+ memory and central memory (CM) cell, PD‐1 blockade with more Cd8+ CTL and Cxcl3+ G2 granulocyte, and PTT with more Cd8+ effector memory and Rethlg+ G3 granulocyte cell. These responses disordered the glycolysis, angiogenesis, phagocytosis functions and cellular communication to reshape the CRC tumour immune microenvironment. We provide the detail insights into the intratumoral immunomodulation preferences of CRC mice treated with CD73 inhibitor, PD‐1 blockade and PTT therapies, which might contribute to the ongoing development of more effective anticancer strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ef48ff72788aaf2bb81620f30c069f2cf46018" target='_blank'>
              Characterising and Evaluating the Immune Microenvironment Landscapes of Colorectal Cancer Shaped by Different Therapies
              </a>
            </td>
          <td>
            Chen Zhou, Yifan Wang, Yuanyuan Li, Weitao Zhang, Yunmeng Bai
          </td>
          <td>2025-07-16</td>
          <td>IET Systems Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A compelling body of research has confirmed the critical role of the tumor microenvironment (TME) in the growth and aggressiveness of breast cancer (BC). Cancer-associated fibroblasts (CAF), the predominant cell component of the TME, significantly influences its behavior through complex crosstalk with cancer and other stromal cells in response to tumor-derived activating signals. Murine models are extensively used to investigate the mechanisms underlying CAF's role in BC progression, as they allow for the evaluation of the endogenous TME and the assessment of immune system involvement using syngeneic xenograft studies. Here, we present a set of methods for culturing BC-derived CAFs, manipulating gene expression, and evaluating their pro-tumorigenic functions using both in vitro and in vivo approaches. We also describe protocols for CAF gene expression profiling and transcriptomic analysis, alongside a computational procedure for inferring transcriptional signatures in primary tumors using publicly available bulk RNA-seq data. Additionally, we introduce a user-friendly, web-based tool for analyzing the transcriptomes of stromal and epithelial components from laser capture micro-dissected BC tumors. Ultimately, these methods can provide valuable insights into the molecular mechanisms driving CAF-tumor interactions and hold potential for identifying novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f94a97738d3f7846e0a2b6fcec7f408e4d3b367d" target='_blank'>
              Cancer-Associated Fibroblasts from Mouse Mammary Tumors as Tools for Molecular and Computational Studies.
              </a>
            </td>
          <td>
            Daniele Viavattene, Aurora Savino, Somayeh Mirzaaghaei, Andrea Lobascio, Valeria Poli, L. Avalle
          </td>
          <td>2025-07-03</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT In most cancers, T lymphocytes comprise an essential cellular component of the non-neoplastic microenvironment, where they have the capacity to both suppress and support tumor growth. One specialized T lymphocyte population is the CD8+ exhausted T cell, which has been intensely studied as an actionable therapeutic target. Unfortunately, there is currently no uniformly accepted classification scheme for these specialized T cells. To provide a potential model for classifying CD8+ exhausted T cells, we leveraged single cell transcriptomic analysis of a diverse collection of both human (n = 8) and mouse (n = 4) cancers to identify unique subpopulations shared across tumor types and species. By integrating data from both human and mouse cancer studies, as well as previously described CD8+ exhausted T cell subsets, we provide an integrated framework to characterize the heterogeneity of exhausted CD8+ T cells. As such, one of these subpopulations (cluster C1) increases following immune checkpoint inhibitor treatment in the setting of cancer in mice and patients. Taken together, this proposed classification scheme may be useful for the design and interpretation of current and future immune-based therapy studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9bb9b37e719dff3c4a062c99fe8116dd65c7d0" target='_blank'>
              Pan-cancer single cell transcriptomic clustering reveals heterogeneous CD8+ exhausted T cell populations with different immune checkpoint inhibitor responses
              </a>
            </td>
          <td>
            Rui Mu, Rasha Barakat, David H. Gutmann
          </td>
          <td>2025-08-03</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Background Triple‐negative breast cancer (TNBC) poses significant challenges due to its aggressive nature and lack of targeted therapies. Understanding the cellular behaviors of TNBC is crucial for developing effective treatments. Aims This study aims to compare the morphological characteristics of non‐tumorigenic MCF10A and aggressive MDA‐MB‐231 TNBC cell lines using advanced analytical techniques. Methods and Results Advanced techniques such as Principal Component Analysis (PCA), t‐Distributed Stochastic Neighbor Embedding (t‐SNE), and digital holographic microscopy were utilized. Cellular features such as area, migration, motility, irregularity, and optical thickness were thoroughly analyzed over time. Our results revealed significant morphological differences between the MCF10A and MDA‐MB‐231 cell lines. Specifically, MDA‐MB‐231 cells displayed enhanced motility and a smaller, more variable size, attributes that may facilitate their invasive potential. In contrast, MCF10A cells exhibited larger sizes and more regular migration patterns, suggesting stability in structured tissue environments. Additionally, temporal analysis highlighted consistent phenotypic behaviors over time, with MDA‐MB‐231 cells demonstrating higher optical thickness and irregularity, indicating potential structural complexities associated with malignant transformation. Correlative analysis further confirmed these results by revealing connections between cell size, motility, and optical properties crucial for understanding cell behavior within their microenvironment. Conclusion The profound differences in cellular dynamics between MCF10A and MDA‐MB‐231 cell lines underscore the unique adaptive mechanisms of TNBC cells. Our study provides valuable insights into the cellular foundations of TNBC aggressiveness, offering a foundation for future research aimed at understanding the mechanistic underpinnings of TNBC progression and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9f97f859a9a924b82dc01461c45eed54b60c098" target='_blank'>
              Dissecting Morphological and Functional Dynamics of Non‐Tumorigenic and Triple‐Negative Breast Cancer Cell Lines Using PCA and t‐SNE Analysis
              </a>
            </td>
          <td>
            Yazan A. Almahdi, Eva R Schwark, Aidan J. McLaughlin, Besa Xhabija
          </td>
          <td>2025-06-27</td>
          <td>Cancer Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Motivation Perturbations in biological tissues—e.g. due to inflammation, disease, or drug treatment—alter the composition of cell types and cell states in the tissue. These alterations are often spatially localized in different regions of a tissue, and can be measured using spatial transcriptomics technologies. However, current methods to analyze differential abundance in cell types or cell states, either do not incorporate spatial information—and thus cannot identify spatially localized alterations—or use heuristic and inaccurate approaches. Results We introduce Spatial Anomaly Region Detection in Expression Manifolds (Sardine), a method to estimate spatially localized changes in spatial transcriptomics data obtained from tissue slices from two or more conditions. Sardine estimates the probability of a cell state being at the same (relative) spatial location between different conditions using spatially localized density estimation. On simulated data, Sardine recapitulates the spatial patterning of expression changes more accurately than existing approaches. On a Visium dataset of the mouse cerebral cortex before and after injury response, as well as on a Visium dataset of a mouse spinal cord undergoing electrotherapy, Sardine identifies regions of spatially localized expression changes that are more biologically plausible than alternative approaches. Availability and implementation We implement Sardine in Python 3, with an open source implementation available at: https://github.com/raphael-group/spatial_anomaly_detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65acdb2dbe154c03be39a6cc5358a0a80c5d9d45" target='_blank'>
              Anomaly detection in spatial transcriptomics via spatially localized density comparison
              </a>
            </td>
          <td>
            Gary Hu, Julian Gold, Uthsav Chitra, Sunay Joshi, Benjamin J. Raphael
          </td>
          <td>2025-07-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Interstitial lung diseases (ILDs) encompass a diverse group of pulmonary disorders, with progressive fibrosis leading to poor prognosis. Here we aimed to identify key molecules involved in progressive fibrosis across various ILDs, using spatial transcriptomics (ST). ST analysis (Visium) was performed on lung cryobiopsy specimens from five patients with various ILDs. Two cases, rich in young fibrotic lesions, as defined by fibroblastic foci and destructive alveolar organization, were selected for spatial high‐dimensional weighted gene coexpression network analysis (hdWGCNA) to identify key gene networks with biological significance in active fibrosis. We utilized public single‐cell RNA sequencing datasets of various ILDs, performed enrichment analysis and trajectory‐based differential expression analysis, and quantified cell–cell communication to evaluate the involvement of the spatially extracted module in fibrosis. Immunohistochemical staining of the extracted molecules was performed. Using hdWGCNA, we identified a distinct gene module (the SM2 module) enriched in young fibrotic lesions. The SM2 module was characterized by distinct features of fibroblast activation that were represented across various lesions. Key hub genes within this module, including COL1A2, COL3A1, COL1A1, and SPARC, formed a robust coexpression network. Immunohistochemical staining showed that SPARC, a component of the SM2 module, was highly expressed in young fibrotic lesions, but not in old scarring lesions, across various ILDs. To assess the prognostic significance of SPARC immunohistochemical expression, we extended our analysis to a cohort of 71 patients with unclassifiable ILDs (uILDs), a particularly heterogeneous subtype with unclear pathogenesis and limited treatment options. Higher SPARC levels in the upper, lower, or both lung lobes in uILD were significantly associated with poor overall survival. In summary, an integrated cross‐disease approach using ST revealed key gene expression patterns central to active fibrosis and successfully identified SPARC as a potentially beneficial prognostic marker. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9a55a6327413c2017bbf7b10baf6a8049b2a4f0" target='_blank'>
              Spatial transcriptomics identifies SPARC as a prognostic marker in interstitial lung diseases
              </a>
            </td>
          <td>
            T. Niitsu, T. Kataoka, K. Fukushima, D. Motooka, Shigeyuki Shichino, Yayoi Natsume-Kitatani, H. Kitamura, Takashi Niwa, T. Baba, D. Okuzaki, Atsushi Kumanogoh, Shizuo Akira, K. Okudela, Takashi Ogura
          </td>
          <td>2025-07-28</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Purpose The heterogeneity of immune cells is a critical manifestation of gastric cancer (GC) heterogeneity and significantly contributes to immune therapy resistance. Although previous studies have focused on the roles of specific myeloid cells and exhausted CD8+ T cells in immune resistance, the immune cell interaction network and its spatiotemporal distribution in GC immune resistance remain underexplored. Methods This study integrated multiple GC single-cell RNA sequencing, spatial transcriptomics, bulk-RNA sequencing, and single-cell immunotherapy datasets of our cohort (NFHGC Cohort). Methods such as single-cell subpopulation identification, transcriptomic analysis, spatial colocalization, cell communication network analysis and tissue immunofluorescence of gastric cancer were employed to investigate immune cell interactions and their molecular mechanisms in immune resistance. Results By leveraging a comprehensive approach that integrates single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq profiles, we identified 20 immune subsets with potential prognostic and therapeutic implications. Our findings suggest a stromal immunosuppressive network orchestrated by Macro_SPP1/C1QC macrophages and CD8_Tex_C1 T cells, which may form a barrier impeding antitumor immunity. Macrophage-derived MIF signaling appears to drive immunosuppression via the MIF-CD74/CXCR4/CD44 axis. Based on these observations, we developed a preliminary TME classification system using a gene signature derived from barrier-associated immune cell markers and unsupervised clustering. Conclusions Our study identified a potential stromal immunosuppressive barrier in gastric cancer, driven by Macro_SPP1/C1QC macrophages and CD8_Tex_C1 T cells, which may contribute to immune dysfunction and therapy resistance. Molecular subtyping based on this barrier’s presence could inform personalized immune therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1cf3d75b61cafecd69ad5d518d7d286742cb1e5" target='_blank'>
              Identification of a stromal immunosuppressive barrier orchestrated by SPP1+/C1QC+ macrophages and CD8+ exhausted T cells driving gastric cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Guichuang Ma, Xiaohan Liu, Qinrui Jiang, Shaowei Li, Qijing Wu, B. Liang, F. Sun, Chunhui Gu, Wangjun Liao, Zhi-Huan Zhang, Min Shi, Qiong Huang
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Simple Summary Soft tissue sarcomas (STSs) are rare cancers that respond poorly to current treatments. As the immune system is increasingly recognized as a key player in cancer control, we investigate a specific type of immune cell—natural killer (NK) cells. We analyzed NK cells from blood samples of STS patients and healthy individuals. NK cells from STS patients were less functional, showing reduced degranulation and lower production of IFNγ, both essential for killing cancer cells. We also found alterations in key surface markers, which may explain their impaired function. Among these, CD27 and NKp44 were notably different between patients and healthy individuals and showed strong potential to distinguish between these groups. Higher CD27 levels were associated with NK cells that are less effective at killing cancer cells. Overall, our findings suggest that NK cells from STS patients may have a reduced capacity to fight cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14bb47c660c210aea5cd2c30473df6b270829f44" target='_blank'>
              Comprehensive Receptor Repertoire and Functional Analysis of Peripheral NK Cells in Soft Tissue Sarcoma Patients
              </a>
            </td>
          <td>
            L. M. Sousa, J. Almeida, Tânia Fortes-Andrade, Patrícia Couceiro, Joana Rodrigues, Rúben Fonseca, M. Santos-Rosa, Paulo Freitas-Tavares, J. Casanova, Paulo Rodrigues-Santos
          </td>
          <td>2025-07-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background Chimeric antigen receptor (CAR) T cell therapy has revolutionized the treatment of hematological malignancies; however, it faces significant challenges in treating solid tumors, including limited immune infiltration into tumor tissues and immunosuppressive tumor microenvironment. Oncolytic viruses (OVs), which selectively destroy cancer cells and trigger antitumor immune responses, offer a compelling solution to these challenges. Newcastle disease virus (NDV) is a natural OV that exhibits antitumor activity with minimal side effects in clinical studies. We hypothesized that combining NDV, engineered to express a chemokine, with tetracycline-inducible CAR T cells could synergistically enhance CAR T cell therapy efficacy against solid tumors. Methods We constructed a recombinant NDV expressing human CCL19 (rNDV19) and evaluated its therapeutic efficacy alongside doxycycline-inducible CAR T cells in an orthotopic lung cancer mouse model. Tumor burden, immune cell infiltration, and mouse survival were analyzed. Results rNDV19 retained potent oncolytic activity, significantly reducing tumor cell viability while achieving stable expression of human CCL19. rNDV19 triggered significant recruitment of CAR T cells into tumors and promoted their activity. Mechanistic analysis revealed that rNDV19 and CAR T cell combination therapy remodeled the tumor microenvironment. Transcriptomic profiling highlighted activation of critical immune pathways—including leukocyte chemotaxis, T cell differentiation, cytokine production, and immune response-activating signaling in combination therapy. These findings were further corroborated by upregulated expression of T cell activation markers like IL-2, TNF-α, IFN-γ, and cytotoxic effector molecules such as granzyme A and perforin. Therapeutically, the combination synergistically extended median survival time from 22 to 36 days, outperforming monotherapies. Conclusions The combination of rNDV19 and CAR T cells represents a promising strategy for overcoming the limitations of CAR T cell therapy for solid tumors. This approach enhances immune cell infiltration and activation, potentially converting “cold” tumors into “hot” tumors to improve therapeutic outcomes and offering a robust translational framework for solid tumor immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab12fafd95d3f595f4929ade8cd028f6ddc5cdfc" target='_blank'>
              hCCL19-expressing recombinant Newcastle disease virus boosts CAR T cell infiltration and efficacy in solid tumor
              </a>
            </td>
          <td>
            Man Liu, Can Li, Lan‐Ting Feng, Renyu Zhang, Wen-Liang Li, Ling Wang, Hai-Jiao Yang, Qian Li, Ze-Kun Liu, Yu‐Le Yong, Xiang-min Yang, H. Cui, L. Kong, Wen Yin, Zhi-Nan Chen, Huijie Bian, Ding Wei
          </td>
          <td>2025-07-25</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Cervical cancer remains a leading cause of cancer-related mortality among women worldwide. Despite advances in vaccination and early screening, late-stage diagnoses are common and associated with poor outcomes. This study aimed to identify novel prognostic biomarkers and therapeutic targets through a multi-omics approach, providing insights into the tumor immune microenvironment. Methods We integrated transcriptomic, mutational, and clinical data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) to construct a prognostic model. Differential gene expression, enrichment analysis, immune infiltration profiling, and drug response prediction were performed to explore molecular features and therapeutic relevance. Results Key high-risk biomarkers (EZH2, PCNA, BIRC5) and protective factors (CD34, ROBO4, CXCL12) were identified. The model effectively stratified patient survival in both cohorts and showed strong predictive performance. High-risk patients displayed distinct immune cell infiltration patterns and upregulated immune checkpoint expression, suggesting potential benefit from immunotherapy. Additionally, higher tumor mutational burden (TMB) was associated with improved survival. Drug sensitivity analysis indicated increased responsiveness of high-risk patients to agents such as Afuresertib and Venetoclax. Conclusion This study establishes a reliable prognostic model and identifies critical biomarkers associated with cervical cancer progression, offering valuable insights into personalized therapeutic strategies. The findings contribute to a more comprehensive understanding of the disease and provide a foundation for future clinical applications. Nevertheless, further large-scale validation is required to confirm these findings and enhance their clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afb16f94cd15c668a2f6b2b2c27ba4adc54c3b4c" target='_blank'>
              Analysis of immune cell infiltration in the tumor microenvironment of cervical cancer and its impact on immunotherapy
              </a>
            </td>
          <td>
            Fei Qin, Lu Huan, Xiaoxi Shi, Yuanyuan Hua, Yi Wu
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor evolution is shaped by selective pressures imposed by physiological factors as the tumor naturally progresses to colonize local and distant tissues, as well as by therapy. However, the distinction between these two types of pressures and their impact on tumor evolution remain elusive, mainly, due to extensive intra-tumor heterogeneity. To disentangle the effects of these selective pressures, we analyze data from diverse cohorts of patients, of both treated and untreated cancers. We find that, despite the wide variation across patients, the selection strength on tumor genomes in individual patients is stable and largely unaffected by tumor progression in the primary settings, with some cancer-specific signatures detectable in the progression to metastases. However, we identify a nearly universal shift toward neutral evolution in tumors that resist treatment and demonstrate that this regime is associated with worse prognosis. We validate these findings on both published and original datasets. We suggest that monitoring the selection regime during cancer treatment can assist clinical decision-making in many cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95ee20056cad563b0d086b8e70065c4daacf4f" target='_blank'>
              Genome-level selection in tumors as a universal marker of resistance to therapy
              </a>
            </td>
          <td>
            Erez Persi, P. Sudalagunta, Yuri I. Wolf, Rafael R. Canevarolo, Mehdi Damaghi, K. Shain, A. Silva, Eugene V. Koonin
          </td>
          <td>2025-07-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT Fusobacterium nucleatum (Fn) is commonly enriched in colorectal cancer (CRC) and associated with poor outcomes, though its mechanisms remain unclear. Our study investigated how Fn affects the tumor microenvironment through single-cell transcriptomic analyses of 42 CRC patient tissues, comparing Fn-positive and Fn-negative tumors. We discovered that Fn impairs IgA plasma cell development and secretory IgA (sIgA) production by disrupting communication with tumor-associated macrophages. Additional experiments in germ-free mice, together with our re-analysis of a publicly available single-cell RNA-seq data set from a CRC mouse model with an intact gut microbiome–both models having been orally gavaged with Fn–jointly validated the causal role of Fn in impairing sIgA induction. We identified a dysregulated IgA maturation (IGAM) module in Fn-positive patients, indicating compromised mucosal immunity and increased bacterial infiltration. This IGAM signature effectively stratified Fn-positive patients, suggesting potential for targeted therapeutic approaches. Our findings reveal that Fn disrupts sIgA production, increasing tumor microbial burden and worsening prognosis through chronic inflammation in Fn-positive CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b983088c09af7c7c8c02b6b79dd03932133b600d" target='_blank'>
              Secretory IgA dysfunction underlies poor prognosis in Fusobacterium-infected colorectal cancer
              </a>
            </td>
          <td>
            Ilseok Choi, Kyung-A Kim, Sang Cheol Kim, Donghwan Park, Ki Taek Nam, Junha Cha, S. Baek, Junha Cha, Hye-Yeong Jo, Minsun Jung, Melody Y. Zeng, Irina Matei, Susan Bullman, Joong Bae Ahn, Yoon Dae Han, Han Sang Kim, Insuk Lee
          </td>
          <td>2025-07-16</td>
          <td>Gut Microbes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Natural killer (NK) cells and T cells play crucial roles in liver metastatic cancer, particularly through their cytotoxic effects on tumor cells. Although existing evidence suggests a functional network of mutual regulation between NK and T cells, the nature of their interaction and its role in liver metastasis remain elusive. In this study, we analyzed single-cell transcriptomics of liver metastases and adjacent tissues from nasopharyngeal carcinoma (NPC), thyroid carcinoma (THCA), breast cancer (BC), colorectal cancer (CRC) and cervical cancer (CESC) to uncover the NK-T cell interaction network and its functional implications. In adjacent tissues, we observed increased infiltration of CD8+ NKT-like cells, γδT cells, and NK cells. The interaction between CD8+ NKT-like cells, CD8+ T cells, γδT cells, and NK cells were enhanced, with stronger signals associated with T and NK cell functions. Notably, CD8+ NKT-like cells, CD8+ T cells, and γδT cells exhibited an increased capacity to activate NK cells. These T cell subsets promoted NK cell anti-tumor activity via CD48-CD244 and TNF-TNFR signaling pathways, which in turn activated the ERK, JNK, and MAPK cascades. Our findings provide valuable insights into the NK-T cell interaction network in liver metastatic cancer, highlighting its potential as a therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-06241-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d630bd4e5bebe82c7ea2d9533a6c99b8e57bfc4" target='_blank'>
              Single-cell transcriptomics analysis reveals a disrupted NK-T cell interaction network in liver metastatic cancer
              </a>
            </td>
          <td>
            Xiaoshuang Wang, Zhongen Wu, Yan Zhou, Dehua Yang, Qingtong Zhou, Di Zhu, Mingwei Wang, Lu Wang
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Breast cancer is a leading cause of central nervous system (CNS) metastases in women, often associated with poor prognosis and limited therapeutic options. However, molecular differences between primary tumors and CNS metastases remain underexplored. We aimed to characterize transcriptomic differences between primary breast tumors and matched CNS metastases and identify immune-related biomarkers associated with metastatic progression and patient outcomes. Transcriptomic profiling was based on 11 matched FFPE sample pairs (primary tumor and CNS metastasis). Paired formalin-fixed paraffin-embedded (FFPE) samples from primary tumors (T1) and CNS metastases (T2) were analyzed using the NanoString nCounter® platform and the PanCancer IO 360™ Gene Expression Panel. Differential gene expression, Z-score transformation, and heatmap visualization were performed in R. In silico survival analyses for overall survival (OS) and recurrence-free survival (RFS) were conducted using publicly available TCGA and GEO datasets. Forty-five genes were significantly differentially expressed between the T1 and T2 samples. Immune-related genes such as CXCL9, IL7R, CD79A, and CTSW showed consistent downregulation in CNS metastases. High expression of CXCL9 and CD79A was associated with improved OS and RFS, whereas high IL7R and CTSW expression correlated with worse outcomes. These findings indicate immune suppression as a hallmark of CNS colonization. Comparative transcriptomic analysis further underscored the distinct molecular landscapes between primary and metastatic tumors. This study highlights transcriptional signatures associated with breast cancer CNS metastases, emphasizing the role of immune modulation in metastatic progression. The identified genes have potential as prognostic biomarkers and therapeutic targets, supporting the need for site-specific molecular profiling in metastatic breast cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f38a39c970e1e53e1acfac2524807ff0971811e" target='_blank'>
              Transcriptomic Profiling of Paired Primary Tumors and CNS Metastases in Breast Cancer Reveals Immune Modulation Signatures
              </a>
            </td>
          <td>
            A. J. Freitas, M. B. Varuzza, S. Calfa, R. Causin, Vinicius Duval da Silva, C. P. Souza, Marcia Maria Chiquitelli Marques
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Glioblastoma remains a significant healthcare challenge due to its aggressiveness and poor prognosis, underscoring the urgent need for innovative approaches to tackle this disease. While there is a growing emphasis on the use of in vitro models for these developments, animal models continue to play a pivotal role, as they provide insights into the complex tumor microenvironment (TME) and host interactions that in vitro systems cannot fully replicate. However, selecting the appropriate animal model based on the research question is fundamental to ensure the reliability and translational potential of preclinical findings. Advanced neuroimaging techniques, particularly magnetic resonance imaging (MRI) and spectroscopy (MRS), offer unique advantages for the in vivo characterization of glioblastoma models. These techniques offer comprehensive morphological, functional, and metabolomic insights, enabling the evaluation of critical tumor features, such as vascular permeability, infiltrative capacity, or metabolism, among others. In this study, different glioblastoma models were generated by orthotopic implantation of two widely used glioblastoma cell lines, C6 and F98, in Fischer, Sprague Dawley, and Wistar rats and characterized by multiparametric MRI, MRS, along with histological analysis. Results revealed that the F98-Fischer model closely mimics human glioblastoma regarding vascular permeability, infiltrative growth, and key metabolic hallmarks. In contrast, the C6-Wistar model, while exhibiting similarities in permeability and metabolism, lacked invasive growth. Moreover, the two cell lines showed different responses to the host environment, with F98 consistently forming infiltrative tumors across rat strains, while C6 showed variability in tumor growth patterns, suggesting a greater dependence on the host microenvironment. These findings underscore the complex interplay between cell line genetics and host factors in defining tumor phenotype, emphasizing the importance of considering host-tumor interactions in glioblastoma research. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-06684-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3d23b95edee5f48bfb60775fba190c1b1cb0465" target='_blank'>
              A multiparametric perspective on C6 and F98 cell lines in orthotopic rat models for glioblastoma research
              </a>
            </td>
          <td>
            Carlos Caro, Nuria Arias-Ramos, J. D. Urbano-Gámez, Raquel González-Alday, Pilar López-Larrubia, M. García-Martín
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cell type-specific transcriptional heterogeneity in embryonic mouse skin is well-documented, but few studies have investigated the regulatory mechanisms. Here, we present high-throughput single-cell chromatin accessibility and transcriptome sequencing (HT-scCAT-seq), a method that simultaneously profiles transcriptome and chromatin accessibility. We utilized HT-scCAT-seq to dissect the gene regulatory mechanism governing epidermal stratification, periderm terminal differentiation, and fibroblast specification. Results By linking chromatin accessibility to gene expression, we identify candidate cis-regulatory elements (cCREs) and their target genes which are crucial for dermal and epidermal development. We describe cells with similar gene expression profiles that exhibit distinct chromatin accessibility statuses during periderm terminal differentiation. Finally, we characterize the underlying lineage-determining transcription factors and demonstrate that ALX4 and RUNX2 are candidate transcription factors regulators of the dermal papilla lineage development through in silico perturbation analysis and CUT&Tag experiment. Conclusions Overall, HT-scCAT-seq represents a powerful tool for unraveling the spatiotemporal dynamics of gene regulation in single cells. Our results advance the understanding of embryonic skin development while providing a scalable framework for investigating regulatory mechanisms across diverse biological systems and disease contexts. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03652-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/747c145f394c82a8a3bbf273f6afa4e007611092" target='_blank'>
              Deciphering gene regulatory programs in mouse embryonic skin through single-cell multiomics analysis
              </a>
            </td>
          <td>
            Q. Deng, Pengfei Cai, Yingjie Luo, Zhongjin Zhang, Wen Ma, Zijie Huang, Xiaoya Chen, Shijie Hao, Weiguang Ma, Jiangshan Xu, Chunqing Wang, Mengnan Cheng, Xiumei Lin, Ru Zhou, Shanshan Duan, Junjie Chen, Ronghai Li, Xuyang Shi, Chang Liu, Liang Wu, Peng Gao, Jianting Li, Xiao Yang, Xiangdong Wang, Jun Xie, Longqi Liu, Yue Yuan, Chuanyu Liu
          </td>
          <td>2025-07-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Malignant pediatric brain tumors (MPBTs) are aggressive tumors that are the leading cause of mortality in pediatric neuro-oncology. Since surgical resection is often limited by tumor location, radiotherapy frequently serves as initial treatment. However, MPBTs might have spontaneously or can develop radioresistance (RR) where the cellular mechanisms underlying this resistance remain poorly understood. Single-Cell RNA sequencing (scRNA-seq), which characterizes tumor heterogeneity at the cellular level, might distinguish intrinsic and extrinsic specific transcriptomic signatures associated with RR before and after irradiation exposure. To investigate those relevant transcriptomic changes induced by irradiation, patient-derived tumoroids established from patient-derived xenograft models were exposed to a repetitive irradiation of 5x4Gys. scRNA-seq approach was performed on naïve tumoroids and 7 days after the end of irradiation. Based on this experimentation, we aim to identify gene expression changes occurring in response to irradiation. For this comparative purpose, we implemented a generative AI model based on Disentangled Variational Auto-Encoders (D-VAE) to learn irradiation-invariant representations of the transcriptome. These representations are then concatenated with controllable treatment attributes for data reconstruction. This model is generating realistic transcriptomic data of irradiated cells from both non-irradiated or irradiated tumor cells just by controlling the irradiation attribute sent to the generator of the model. The exploration of generated synthetic data through this model might be leveraged to understand the effects of irradiation treatments on gene expression profiles. From the generated transcriptomic data, our initial validation confirms that the D-VAE model accurately reproduces the radiation treatment effects and preserve cell clustering patterns. We are now exploring the opportunities provided by the ability to reconstruct "counterfactual" data to understand the fate of different cell clusters in response to irradiation. This process will allow to identify key genes and pathways that are consistently altered following irradiation or associated to RR. In particular, we are focusing on whether subset of cells display pre-existing RR biomarkers prior to treatment. Together, this work is demonstrating the utility of AI-driven approaches in modeling complex cellular responses to irradiation and is offering new insights of the molecular determinants of RR in MPBTs.



 Chinar Salmanli, Marlene Deschuyter, Natacha Entz-Werle, Julien Godet. Generative modeling (AI) for the analysis of Single-Cell RNA-seq data of radioresistant malignant pediatric brain tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B032.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fd82bb93df18507140cdacfa44fe230d899ab1e" target='_blank'>
              Abstract B032: Generative modeling (AI) for the analysis of Single-Cell RNA-seq data of radioresistant malignant pediatric brain tumors
              </a>
            </td>
          <td>
            Chinar Salmanli, Marlène Deschuyter, N. Entz-Werlé, Julien Godet
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Multi-modal spatial omics data are invaluable for exploring complex cellular behaviors in diseases from both morphological and molecular perspectives. Current analytical methods primarily focus on clustering and classification, and do not adequately examine the relationship between cell morphology and molecular dynamics. Here, we present MorphLink, a framework designed to systematically identify disease-related morphological-molecular interplays. MorphLink has been evaluated across a wide array of datasets, showcasing its effectiveness in extracting and linking interpretable morphological features with various molecular measurements in spatial omics analyses. These linkages provide a transparent view of cellular behavior heterogeneity within tissue regions with similar cell type compositions, characterizing tumor subtypes and immune diversity across different organs. Additionally, MorphLink is scalable and robust against cross-sample batch effects, making it an efficient method for integrative spatial omics data analysis across samples, cohorts, and modalities, and enhancing the interpretation of results for large-scale studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0fedcc34d8c73fe0f20a713c19658964fd9b5f5" target='_blank'>
              Bridging cell morphological behaviors and molecular dynamics in multi-modal spatial omics with MorphLink
              </a>
            </td>
          <td>
            Jing Huang, Chenyang Yuan, Jiahui Jiang, Jianfeng Chen, Sunil S Badve, Y. Gokmen-Polar, R. Segura, Xinmiao Yan, Alexander Lazar, Jianjun Gao, Bing Yao, Michael P. Epstein, Linghua Wang, Jian Hu
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumor immune microenvironment plays a crucial role in determining the prognosis of lung adenocarcinoma (LUAD), with the interaction of immune cells within this microenvironment contributing to a poorer prognosis. We sought to investigate the causal relationship and underlying biological mechanisms between immune cell characteristics and LUAD to offer new insights for enhancing treatment strategies. We evaluated the association between immune cell characteristics and LUAD using Mendelian randomization (MR) analysis based on genome-wide association studies summary statistics. Sensitivity analysis was performed to verify the robustness of MR results. Immune cell infiltration analysis and machine learning on bulk RNA-sequencing data were conducted to further identify immune cells associated with LUAD. Prognostic genes of LUAD were identified using single-cell RNA-sequencing data and high dimensional weighted gene co-expression network analysis. MR analysis identified three immune cell characteristics associated with LUAD, including CCR2 on granulocyte, CD25 on CD45RA + CD4 not Treg, and plasmacytoid dendritic cell, and sensitivity analysis confirmed the robustness of the associations. Machine learning identified neutrophils, hematopoietic stem cells, B cells, and myeloid progenitor cells as key immune cell characteristics related to LUAD. Neutrophil was identified as the target cell of LUAD based on the MR analysis and machine learning. Subcequently, single-cell RNA-sequencing mapped the immune microenvironment of LUAD and identified down-regulated neutrophil. In addition, robust neutrophil-macrophage communication in LUAD was revealed using the CellChat package. Finally, nine neutrophil-related prognostic genes of LUAD were identified, three of which potentially regulated neutrophil-macrophage communication. This study showed a significant correlation between neutrophil and LUAD, particularly highlighting the neutrophil-macrophage communication. This finding may enhance our comprehension of LUAD immune microenvironment and hopefully promote the discovery of new immunotherapeutic targets for LUAD and the development of related drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7e6b22a22c7a1990ef7a66682bcfe098b8e6b5" target='_blank'>
              Exploring the impact of neutrophils on lung adenocarcinoma using Mendelian randomization and transcriptomic study
              </a>
            </td>
          <td>
            Xiang Xiao, Xuan-Yu Wu, Jingqi Zhang, Wenyuan Li, Feng-Ming You, Jing Guo
          </td>
          <td>2025-07-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tissue function emerges from coordinated interactions among diverse cell types, but how these interactions are structured and rewired in disease remains unclear. In a recent study, Shi and colleagues introduce CoVarNet, a computational framework that maps reproducible multicellular modules (CMs) across 35 human tissues using single-cell and spatial transcriptomics. These CMs, spanning immune, stromal, and endothelial cells, exhibit functional organization across tissue systems and dynamically respond to biological transitions such as aging and menopause. Importantly, cancer progression is marked by a breakdown of tissue-specific CMs and the emergence of a convergent cancer-associated ecosystem, cCM02. This rewiring reflects a fundamental reorganization of tissue architecture during malignancy and provides new opportunities for diagnostics and therapeutic targeting. The study signifies a conceptual advance from cell-centric to ecosystem-level biology and offers a generalizable framework for integrating multimodal data to dissect tissue-level coordination. Here, we discuss how CoVarNet redefines our understanding of tissue organization, its translational implications in oncology, and unresolved questions in modular tissue biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77233b071e8184bbc674d8e5cc53c013d3b3c380" target='_blank'>
              From Harmony to Discord: Multicellular Coordination in Tissues and Its Rewiring in Cancer.
              </a>
            </td>
          <td>
            Merve Dede, Vakul Mohanty, Ken Chen
          </td>
          <td>2025-07-23</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Background/Aim: Tongue squamous cell carcinoma (TSCC), a highly aggressive subtype of head and neck cancers, is characterized by frequent lymphatic metastasis and poor prognosis. Recently, we showed that lymphatic vessel endothelial hyaluronan receptor 1 (LYVE-1) is involved in TSCC progression, yet the underlying molecular mechanisms remain unclear. Materials and Methods: CRISPR/Cas9 gene editing was employed to generate LYVE1 knockout (KO) TSCC cell lines. Single-cell clones were isolated, screened, and validated through sequencing and Inference of CRISPR Edits (ICE) analysis and qRT-PCR. RNA sequencing was performed on LYVE1 KO and wild-type (WT) cells to identify differentially expressed genes (DEGs). Bioinformatic analyses, including Gene Ontology (GO) enrichment and protein-protein interaction (PPI) network mapping, were conducted to explore affected pathways. Finally, network topology was examined using NetworkAnalyzer and cytoHubba plugins. Results: Transcriptomic analysis revealed significant down-regulation of pro-metastatic pathways, including epithelial-mesenchymal transition (EMT), extracellular matrix remodeling, and immune modulation. DEG analysis identified 263 genes, with key down-regulated targets such as WNT5A, TGFB2, and MMP2, and up-regulation of tumor-suppressive genes including PTGS2. GO and PPI analyses highlighted LYVE1’s pivotal role in regulating cell adhesion, migration, and immune response. Conclusion: LYVE1 KO reduces TSCC invasive potential by disrupting EMT and tumor-stroma interactions, aligning with previous experimental findings. These results suggest LYVE1 as a critical driver of metastasis, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff3c633bd83b5225f79493e53059643bbd3c2790" target='_blank'>
              CRISPR/Cas9-mediated Knockout of LYVE1 In Human Tongue Cancer Cells Reveals Transcriptomic Changes in Metastasis-associated Pathways
              </a>
            </td>
          <td>
            Sini Karinen, Johanna FORERO-RODRÍGUEZ, Annika Järvinen, Kari K Eklund, Gonçalo Barreto, A. Salem
          </td>
          <td>2025-06-26</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Background: Several studies have demonstrated the occurrence of secondary tumors as a rare but significant complication of chimeric antigen receptor T (CAR-T) cell therapy, underscoring the need for a detailed investigation. Given the limited variety of secondary tumor types reported to date, a comprehensive characterization of the various secondary tumors arising after CAR-T therapy is essential to understand the associated risks and to define the role of the immune microenvironment in malignant transformation. This study aims to characterize the immune microenvironment of a newly identified secondary tumor post-CAR-T therapy, to clarify its pathogenesis and potential therapeutic targets. Methods: In this study, the bone marrow (BM) samples were collected by aspiration from the primary and secondary tumors before and after CD19 CAR-T treatment. The CD45+ BM cells were enriched with human CD45 microbeads. The CD45+ cells were then sent for 10× genomics single-cell RNA sequencing (scRNA-seq) to identify cell populations. The Cell Ranger pipeline and CellChat were used for detailed analysis. Results: In this study, a rare type of secondary chronic myelomonocytic leukemia (CMML) were reported in a patient with diffuse large B-cell lymphoma (DLBCL) who had previously received CD19 CAR-T therapy. The scRNA-seq analysis revealed increased inflammatory cytokines, chemokines, and an immunosuppressive state of monocytes/macrophages, which may impair cytotoxic activity in both T and natural killer (NK) cells in secondary CMML before treatment. In contrast, their cytotoxicity was restored in secondary CMML after treatment. Conclusions: This finding delineates a previously unrecognized type of secondary tumor, CMML, after CAR-T therapy and provide a framework for defining the immune microenvironment of secondary tumor occurrence after CAR-T therapy. In addition, the results provide a rationale for targeting macrophages to improve treatment strategies for CMML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f62036ee9eb9eb457e3406f8e167f834148a9998" target='_blank'>
              Decoding the immune microenvironment of secondary chronic myelomonocytic leukemia due to diffuse large B-cell lymphoma with CD19 CAR-T failure by single-cell RNA-sequencing
              </a>
            </td>
          <td>
            Xudong Li, Hong Huang, Fang Wang, Mengjia Li, Binglei Zhang, Jianxiang Shi, Yuke Liu, Mengya Gao, Mingxia Sun, Haixia Cao, Danfeng Zhang, Na Shen, Weijie Cao, Zhilei Bian, Haizhou Xing, Wei Li, Linping Xu, Shiyu Zuo, Yongping Song
          </td>
          <td>2025-07-07</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Drug discovery and testing are both time-consuming and costly. Leveraging single-cell transcriptomic data, we developed a machine learning framework to model patient cells using scRNA-seq data, aiming to infer gene regulatory networks (GRNs) that drive cell states and plasticity. Our computational algorithm predicts cellular responses to drug perturbations at the transcriptional level, even without experimental testing. We utilized this algorithm to identify drug candidates that could shift anti-inflammatory macrophages to a pro-inflammatory state for remodeling the tumor microenvironment, reactivate exhausted CD8 T cells, and convert immunosuppressive CD4 Treg cells into central memory CD4 cells to enhance anti-tumor activity. The predicted outcomes were validated experimentally through scRNA-seq, flow cytometry, and cytokine profiling. In human hepatocellular carcinoma (HCC), tumor infiltrating lymphocytes (TILs) treated with the predicted drugs showed a decreased FoxP3+ CD4 Treg population compared to untreated control TILs from the same HCC (n = 3). Separately, TILs treated with another set of predicted drugs showed higher levels of IFNG+ CD8 T cells than untreated controls (n = 3). Lastly, in human THP1 macrophages differentiated to an anti-inflammatory state, treatment resulted in increased levels of TNFa and IL1b secretion compared to untreated controls, indicating a phenotypic shift towards a pro-inflammatory state. We then tested the same drug on tumor associated macrophages (TAMs) from HCC and showed an increased CD64+ HLA-DR+ macrophage subset upon treatment. scRNA-seq analysis further revealed key pathways induced by the drugs, that resulted in the desired change in immune phenotype, offering insight into immune cell function and plasticity. In conclusion, this study introduced a machine-learning driven approach to identify candidate small molecules from scRNA-seq data that are capable of modulating macrophage and T cell activation, providing new avenues for targeted interventions in cancer.



 N. Jannah M. Nasir, John F. Ouyang, Valerie Chew, Owen Rackham. Applying Machine Learning to Identify Enhancers of Immune Cell Functionality for Targeted Cancer Therapies [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P54.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb6e81b3bc27e9df1055642ea570cd3c3f46a059" target='_blank'>
              Abstract P54: Applying Machine Learning to Identify Enhancers of Immune Cell Functionality for Targeted Cancer Therapies
              </a>
            </td>
          <td>
            N. J. M. Nasir, John F. Ouyang, Valerie Chew, Owen Rackham
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumor microenvironment (Tumor Microenvironment, TME) is a core regulatory factor in the occurrence, development, and treatment resistance of tumors. Macrophages, as key immune cell components in the TME, have a profound impact on the tumor process (Visser and Joyce in Cancer Cell 41:374–403, 2023). This review aims to systematically elucidate the characteristics and functional differences of macrophage polarization into M1 and M2 phenotypes within the TME. Additionally, it endeavors to dissect the regulatory mechanisms by which metabolic products, cytokines, and extracellular matrix components secreted by tumor cells modulate macrophage polarization (Wang et al. in Mol Cancer 23:268, 2024). Moreover, the metabolic reprogramming of tumorassociated macrophage (TAM) is a core mechanism for their functional shift, and intervening in metabolic pathways holds promise for reprogramming TAMs to inhibit tumor progression (Jin et al. in Nat Cancer 6:239–252, 2025). Within the TME, macrophages can be polarized into classically activated M1 and alternatively activated M2 types (Ge and Wu in Zhongguo Fei Ai Za Zhi 26:228–237, 2023). Accumulating evidence indicates that classically activated M1 macrophages orchestrate anti-tumor immunity by secreting pro-inflammatory cytokines, including tumor necrosis factor-α (TNF-α) and interleukin-12 (IL-12), which collectively activate cytotoxic T lymphocyte responses, induce tumor cell apoptosis, and enhance immune surveillance (Luo et al. in Front Immunol 15:1352946, 2024). In contrast, M2 macrophages are induced in the TME and promote tumor angiogenesis, immune evasion, tumor cell proliferation, and metastasis by secreting factors such as vascular endothelial growth factor (VEGF) and transforming growth factor-beta (TGF-β) (Wang et al. in NPJ Precis Oncol 8:31, 2024). Therefore, in-depth research on the mechanisms of macrophage polarization in the tumor microenvironment provides an important basis for developing new tumor immunotherapy strategies and has significant clinical translational value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323da979d84b0f4625128d836a1d59f140970818" target='_blank'>
              Advances in the regulation of macrophage polarization by the tumor microenvironment
              </a>
            </td>
          <td>
            Minzhi Peng, Yuan Zhu, Yi Hu, Jianping Wen, Weiguo Huang
          </td>
          <td>2025-08-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Mammary tumors are the most frequent neoplasia in female dogs. Canine mammary cell lines represent a model of study for women breast cancer research that could benefit both species. The aim of our study was to establish and partially characterize canine mammary tumor cell lines. Ten cell cultures were generated from tumor tissue obtained from canines with mammary tumors. Seven cell lines were generated from primary mammary tumors and three cell lines from metastatic sites. Immunocytochemistry (ICC) analysis revealed a negative expression of hormonal receptors (estrogen receptor [ER] and progesterone receptor [PR]) in five cell lines and positive in one cell line, whereas six cell lines were human epidermal growth factor receptor 2 (HER2)-negative and positive for vimentin. Several cell lines showed tumorigenicity capacity in vitro. The susceptibility of five cell lines to different drugs was evaluated with the MTT (3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide) assay, where doxorubicin (DOX) showed the highest growth-inhibitory effect (DOX > Paclitaxel > Colchicine > 5-Fluorouracil [5-FU] > Carboplatin). The cell lines generated represent a model of study for breast cancer that can be used for developing and testing new treatments to improve the survival of canine and human patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb972b0b2cc1015936d305bfbb0e2027beeb58d9" target='_blank'>
              Establishment and Partial Characterization of Canine Mammary Tumor Cell Lines
              </a>
            </td>
          <td>
            Eliza Vazquez, Luis Dominguez, Brian Silverio, Geobanni Torres, A. Garibay-Escobar, F. Queiroga, Carlos Velazquez
          </td>
          <td>2025-07-01</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Spatial transcriptomics is able to acquire cellular gene expression while retaining spatial location. It is often accompanied by matched hematoxylin and eosin-stained histology whole-slide images. This retention of spatial information is critical for studying key issues in cell biology, developmental biology, neurobiology, and tumor biology. However, conventional sequencing technologies are costly and time-consuming, limiting the development of spatial transcriptomics research. Recently, deep learning methods have been widely applied to spatial transcriptomics prediction, but there are some problems in existing methods. To tackle the problem, we develop Reg2ST, a deep learning model to learn potential patterns in gene expression and apply them for spatial transcriptomics prediction. Reg2ST treats spatial transcriptomics and histology as different expressions of the same data. Contrastive learning is used to minimize the distance between them. Then image features are used to predict gene features, which aligns histology images with spatial transcriptomics. Reg2ST uses a novel way to capture relationships among spots instead of K-Nearest-Neighbors. Evaluations of Pearson correlation coefficient, statistical tests, computational efficiency using human breast cancer, and cutaneous squamous cell carcinoma datasets demonstrate the superior performance of Reg2ST for spatial gene expression prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33edc5eac27ac34326f96c844f5a9d94f37ebec2" target='_blank'>
              Reg2ST: recognizing potential patterns from gene expression for spatial transcriptomics prediction
              </a>
            </td>
          <td>
            Xu Wang, Yaqiu Wang, Xuan Wang, Yadong Wang, Junyi Li
          </td>
          <td>2025-07-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796b3d170cc4e2945a302ca6861debacf36381a0" target='_blank'>
              The State of Single-Cell Atlas Data Visualization in the Biological Literature.
              </a>
            </td>
          <td>
            Mark S. Keller, Eric Morth, Thomas C. Smits, Simon Warchol, Grace Guo, Qianwen Wang, Robert Krueger, Hanspeter Pfister, Nils Gehlenborg
          </td>
          <td>2025-06-27</td>
          <td>IEEE computer graphics and applications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="We previously showed that selective suppression of CD44 in the corneal epithelium leads to structural abnormalities in the mouse cornea. Our comparative studies of young and aged ocular biopsies revealed that CD44 expression is downregulated in aged corneas, while leucine-rich repeats and immunoglobulin-like domain 1 (Lrig1+) stem cells remain preserved in the peripheral limbus. These findings suggest an age-related shift in the corneal stem cell compartmentalization, characterized by impaired CD44 expression in the central cornea and preservation of Lrig1+ stem cells in the limbus, which become the main stem cells in the senescent cornea. To investigate this further, we performed topical tamoxifen-inducible, diphtheria toxin-mediated ablation of Lrig1+ stem cells in mouse corneas. We then assessed both activated and non-activated beta-catenin expression in wild-type (WT) and CD44 knockout (CD44KO) mice, given that CD44 modulates the Wingless-related integration site (Wnt) pathway. Our results indicate that two distinct stem cell populations operate in the mouse cornea: Lrig1-derived stem cells and Wnt-activity/CD44-dependent stem cells. The Lrig1-derived cells act as a reservoir of quiescent stem cells that regenerate the cornea upon injury, whereas under homeostatic conditions, the Wnt-activity/CD44-dependent stem cells are primarily responsible for corneal renewal. In the aged cornea, the loss of CD44 expression leads to reduced Wnt signaling, making the tissue increasingly dependent on Lrig1+ stem cells for regeneration. In mice, Lrig1+ stem cells are capable of sustaining permanent corneal renewal, even in the absence of CD44.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a4632715eba8ae15094380ddd88e83af0bc60a" target='_blank'>
              Identification of Two Distinct Stem Cell Clusters, Lrig1-Derived and Wnt/CD44-Dependent, in Corneal Epithelium
              </a>
            </td>
          <td>
            L. Barnes, Evangelia Konstantinou, J. Saurat, Alexandre Moulin, G. Kaya
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff04e163a7aa41f773b08a8fee437665a57ee718" target='_blank'>
              CB2R-induced differentiation epigenetically restrains cancer plasticity enabling adaptive therapy.
              </a>
            </td>
          <td>
            Nuria G. Martínez-Illescas, Gemma Pérez-García, María Rubert-Hernández, Camila Quezada-Gutiérrez, Estrella Martín-Zapater, Luis M Alonso-Colmenar, Carmen Hernández, Ángela Zarco-Cuadrillero, Eduardo Caleiras, Miguel Muñoz-Ruiz, Ana Payo-Payo, Laura Frías-Aldeguer, Cristina Sánchez, Mercedes Balcells-Camps, E R Edelman, María Salazar-Roa
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Nuclear morphology is a defining cellular feature, differing based on cell type, tissue type, and species. In healthy cells, nuclear morphology is generally tightly regulated and maintained; however, dynamic changes in nuclear morphology are observed under certain conditions, for instance in early embryos and in some immune cells. Deviations in normal nuclear morphology are linked to numerous diseases, including most cancers and premature aging syndromes. Many regulators of nuclear morphology have been identified, encompassing both intranuclear, cytoplasmic, and extracellular factors. Of note, recent studies have converged on chromatin and chromatin-associated proteins as key determinants of nuclear morphology and dynamics. In this review we discuss how the chromatin landscape regulates nuclear morphology in both normal and diseased cellular states. Additionally, we highlight emerging technologies that promise to bridge critical gaps in our understanding of nuclear morphology, including new approaches to probe nuclear structure and the use of synthetic cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adc2c44f577e2c2679b995d4aca9623ddc124e59" target='_blank'>
              How the chromatin landscape influences nuclear morphology
              </a>
            </td>
          <td>
            Sourabh Sengupta, Haritha Prabha, Daniel L. Levy
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad606f697b154e1d0963d77bb80a6ca95fd8151f" target='_blank'>
              Proteomic technologies for profiling cell-membrane/secretome interactions in brain metastatic cancer progression.
              </a>
            </td>
          <td>
            I. Lazar
          </td>
          <td>2025-07-17</td>
          <td>Expert review of proteomics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Periodontitis, a chronic inflammatory condition of the periodontium, is associated with over 60 systemic diseases. Despite advancements, precision medicine approaches have had limited success, emphasizing the need for deeper insights into cellular subpopulations and structural immunity, particularly gingival keratinocytes. This study employs autoencoder models and data augmentation techniques to explore the transcriptomic diversity of gingival keratinocytes at the single-cell level. Single-cell RNA sequencing data from GSE266897 were processed using the Scanpy library, with quality control implemented to filter cells based on predefined metrics. Clustering was performed using principal component analysis (PCA) and k-nearest neighbor (KNN) algorithms. Marker gene identification and differential expression analysis were used to characterize cell clusters. Visualization techniques, including UMAP, heatmaps, dot plots, and violin plots, provided insights into gene expression patterns. The autoencoder architecture featured an encoder reducing input size to 256 units with ReLU activation, a bottleneck layer, and a decoder restoring data dimensions. The basic Autoencoder (AE) demonstrated superior performance, achieving the lowest loss (0.725), the highest accuracy (0.695), and minimal false positives. The Test-Time Augmentation AE also performed robustly, achieving an F1 score of 0.642 and an AUC-ROC of 0.800. The Basic AE effectively modeled RNA-seq data complexity compared to Variational and Denoising Autoencoders. This study highlights advanced computational techniques to investigate gingival keratinocytes’ transcriptomic diversity, revealing distinct subpopulations and differential gene expression profiles. These findings underscore the active role of keratinocytes in periodontal health and inflammatory responses, contributing to precision medicine approaches in periodontology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54234971a14ce3e8edfa17d85d926435fc3b536" target='_blank'>
              Leveraging autoencoder models and data augmentation to uncover transcriptomic diversity of gingival keratinocytes in single cell analysis
              </a>
            </td>
          <td>
            P. K. Yadalam, P. Natarajan, Carlos M. Ardila
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Aim This study aims to explore the dynamic tumor microenvironment of hepatocellular carcinoma (HCC) through deep transcriptomic analysis and to identify key regulatory genes, among which MRE11 was further validated for its immunomodulatory and prognostic significance. Methods We performed Summary-data-based Mendelian Randomization (SMR) analysis to identify genes causally associated with HCC and intersected these with DNA damage repair (DDR) genes, leading to the identification of MRE11. A comprehensive evaluation of MRE11 expression in HCC was conducted using transcriptomic data analysis. We collected data from 92 HCC patient samples and validated MRE11 expression differences in HCC tissues through qPCR, immunohistochemistry, and Western blotting. Publicly available single-cell RNA sequencing (scRNA-seq) data and spatial transcriptomics were utilized to explore MRE11’s dynamic mechanisms in the tumor microenvironment (TME) of both primary and post-immunotherapy cases. We also screened for differentially expressed genes and constructed a robust HCC prognosis model using 101 machine-learning algorithms. Results Our results demonstrated that high MRE11 expression is strongly associated with poor prognosis in HCC. In the primary TME, MRE11 regulates immune responses, facilitating immune evasion. Single-cell analysis revealed significant tumor heterogeneity in MRE11 high-expression groups, particularly in macrophages and malignant cells, where MRE11 regulates immune evasion and tumor progression via the cGAS-STING pathway and HGF-MET axis. Under immunotherapy, high MRE11 expression facilitated epithelial-mesenchymal transition (EMT) and extensive remodeling of the TME. Furthermore, MRE11 dynamically enhanced macrophage regulation, exhibiting immunosuppressive and tumor-invasive features. Finally, our prognostic model exhibited strong predictive accuracy across multiple datasets. Conclusion High MRE11 expression is crucial in regulating the immune microenvironment in HCC, fostering immune evasion and driving tumor progression. MRE11 emerges as a promising biomarker for HCC diagnosis and a potential target for personalized immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03931-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60428c69be5bed6fbcd8373c293bad782a86b9ab" target='_blank'>
              Elucidating the dynamic tumor microenvironment through deep transcriptomic analysis and therapeutic implication of MRE11 expression patterns in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Ruiqiu Chen, Chaohui Xiao, Zizheng Wang, Guineng Zeng, Shaoming Song, Gong Zhang, Lin Zhu, Penghui Yang, Rong Liu
          </td>
          <td>2025-08-21</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Ovarian cancer (OV) is the deadliest gynecologic malignancy owing to its late diagnosis and high metastatic propensity. Current biomarkers lack sufficient sensitivity and specificity for the detection of early-stage cancer. To address this gap, we integrated single‐cell transcriptomic profiling of tumor tissues with analysis of circulating exosomal RNA, aiming to uncover candidate markers that reflect tumor heterogeneity and metastatic potential and that may serve as sensitive, non‐invasive diagnostics. Methods We integrated single-cell RNA sequencing (scRNA-seq) data from primary tumors and metastatic lesions with bulk tissue transcriptomes and plasma-derived exosomal RNA sequencing (RNA-seq). Differentially expressed genes (DEGs) shared across tumor cells, metastatic subpopulations, and exosomes were identified through intersection analysis. Candidate genes were validated in clinical specimens using qPCR and immunohistochemistry. We then applied ten machine learning algorithm to exosomal transcriptomic data to evaluate diagnostic performance and identify the optimal classifier. Tumor cell differentiation states were evaluated using CytoTRACE, and intercellular communication was analyzed with CellChat. Results Intersection analysis highlighted 52 overlapping DEGs, of which SCNN1A and EFNA1 emerged as the top prognostic indicators. Both genes were significantly upregulated in tumor tissues, metastatic foci, and plasma exosomes (P < 0.01). The exosome-based Adaboost model had an area under the curve of 0.955 in an independent test cohort. Single-cell subcluster analyses revealed high SCNN1A/EFNA1 expression correlated with stem-like differentiation states and enriched pathways associated with immune evasion and adhesion. CellChat analysis demonstrated that highly differentiated tumor cells extensively engaged with fibroblasts and endothelial cells, implying their role in niche formation. Conclusions By coupling single-cell, bulk tissue, and exosomal transcriptomics, we elucidated the key molecular drivers of OV metastasis and established SCNN1A and EFNA1 as promising non-invasive biomarkers. This multi-omics framework provides an effective strategy for early detection and a better understanding of metastatic progression in OV.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0c71607fd05b63fb6eb088e5cf81be9d264179" target='_blank'>
              Integrative single-cell and exosomal multi-omics uncovers SCNN1A and EFNA1 as non-invasive biomarkers and drivers of ovarian cancer metastasis
              </a>
            </td>
          <td>
            Liping Tang, Dong Pang, Chengbang Wang, Jiali Lin, Shaohua Chen, Jiangchun Wu, Junqi Cui
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tumor stroma can be actively shaped by tumor cells to adopt pro-tumorigenic properties, making it a focal point of cancer research. Among the stromal components, cancer-associated fibroblasts (CAFs) -- often the most abundant cell type in the tumor microenvironment (TME) -- play a pivotal role in promoting cancer progression through their interactions with tumor cells and other stromal components. Elucidating the mechanisms behind these interactions requires robust methods for isolating and characterizing CAFs, which can also lay the foundation for developing novel CAF-targeted cancer therapies. This study presents a method for isolating fibroblasts from both the tumor and the adjacent normal regions of radical prostatectomy specimens obtained from prostate cancer (PCa) patients. This approach enables the concurrent isolation of CAFs and their normal counterparts (NFs) from the same individual, providing a paired experimental system. Detailed protocols are provided for the maintenance of primary CAFs and NFs for downstream in vitro analyses, as well as for their immortalization to facilitate long-term studies. We also describe functional assays designed to compare the effects of CAFs and NFs on cancer cell behaviors, including proliferation, migration, and anchorage-independent growth. Two complementary approaches are detailed: treatment of cancer cells with fibroblast-derived conditioned media (CM), and direct co-culture of fibroblasts with tumor cells. Together, these methodologies represent a comprehensive toolkit for investigating the dynamic interplay between CAFs and tumor cells, not only in PCa but potentially across a range of human cancers. Moreover, molecular characterization of these interactions may reveal key mediators of CAF-driven oncogenesis, offering promising targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd2d8dc58b820e48882f34f8df6b8e5bd76919a2" target='_blank'>
              Isolation, Culture, and Characterization of Prostate Cancer-Associated Fibroblasts.
              </a>
            </td>
          <td>
            Somayeh Mirzaaghaei, L. Avalle, Chiara Verrengia, Jahnavi Srivatsa, Laura Conti, Marta Gai, Chiara Fiameni, Paolo Gontero, Umberto Mortara, Luca Molinaro, Mauro Papotti, Valeria Poli
          </td>
          <td>2025-08-01</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT Oropharyngeal candidiasis (OPC) caused by the fungus Candida albicans triggers a robust inflammatory response that rapidly eliminates the bulk of the fungal load in the oral mucosa. While pro-inflammatory responses to OPC have been extensively studied, little is known about the counterbalance of immunity in the infection milieu that mitigates inflammatory damage. We employed 10× Visium Spatial Transcriptomics, a next-generation technology that preserves the spatial integrity of infected tongue tissues, enabling high-resolution mapping of the host microenvironment during Candida infection in a murine OPC model. This approach provided a view of cellular interactions and immune dynamics, revealing intricate crosstalk between distinct immune cell populations. Our findings highlight a previously underappreciated role of the type-2 immune response and M2 macrophages in maintaining tissue homeostasis, and the predicted role of platelet degranulation in facilitating disease resolution. Finally, we identified a novel family of small proline-rich antimicrobial proteins with potent antifungal activity. These molecules emerge as promising therapeutic candidates, offering a new avenue for combating OPC. IMPORTANCE Oropharyngeal candidiasis (OPC), a fungal infection caused by Candida albicans, affects individuals with weakened immune systems. Our study used spatial transcriptomics, a cutting-edge technology that preserves tissue architecture while mapping immune interactions at high resolution. This approach allowed us to uncover previously unrecognized cellular crosstalk and regulatory pathways that shape the host response to OPC. We discovered that platelets, beyond their role in clotting, play a key role in antifungal defense. Additionally, M2 macrophages are important for resistance to OPC. Most notably, we identified a new family of antimicrobial proteins with strong antifungal properties, presenting promising therapeutic potential. By uncovering these overlooked immune mechanisms, our research findings may lead to better treatments for OPC, particularly in immunocompromised individuals. Oropharyngeal candidiasis (OPC), a fungal infection caused by Candida albicans, affects individuals with weakened immune systems. Our study used spatial transcriptomics, a cutting-edge technology that preserves tissue architecture while mapping immune interactions at high resolution. This approach allowed us to uncover previously unrecognized cellular crosstalk and regulatory pathways that shape the host response to OPC. We discovered that platelets, beyond their role in clotting, play a key role in antifungal defense. Additionally, M2 macrophages are important for resistance to OPC. Most notably, we identified a new family of antimicrobial proteins with strong antifungal properties, presenting promising therapeutic potential. By uncovering these overlooked immune mechanisms, our research findings may lead to better treatments for OPC, particularly in immunocompromised individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/901da1fcdc673e05f0604138f1185396e69199f8" target='_blank'>
              A spatial transcriptomic atlas of the host response to oropharyngeal candidiasis
              </a>
            </td>
          <td>
            Sunna Nabeela, Hayden McSwiggin, R. Magalhães, Eliciane Mattos, Mohammad Mannan, John T. Dillon, Ashley Barbarino, E. Youssef, Shakti Singh, Wei Yan, Ashraf S. Ibrahim, Heather R. Conti, Priya Uppuluri
          </td>
          <td>2025-06-30</td>
          <td>mBio</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tumor heterogeneity and the dynamic evolution of tumor immune microenvironment (TIME) contribute to therapeutic resistance and poor clinical prognosis. To elucidate this mechanism, we first established a murine tumor evolution model (TEM) and systematically identified evolutionary core genes demonstrating progressive alterations during evolution. Subsequently, we developed a single-cell TEM through integrative analysis of hepatocellular carcinoma (HCC) clinical specimens (n=10) with external cohorts (n=11), enabling dynamic characterization of tumor-immune interactions during evolution, while addressing ethical challenges associated with obtaining tumor tissues from multiple stages in a single patient. Through TEMs analyses, we identified a contrasting glucose metabolism pattern between malignant cells and CD8+ T cells during tumor evolution. Mechanistically, glucose metabolic dominance triggers NSUN2 upregulation in tumor cells, where this functional RNA methyltransferase stabilizes key glycolytic transcripts (GLUT1, HK2, PFKM) through mRNA methylation. The NSUN2-mediated GLUT1 stabilization enhances the competitive advantage of tumor cells in glucose acquisition, creating a positive feedback loop that accelerates malignancy and exacerbates CD8+ T cell dysfunction. Building on these insights, we designed a dual-targeting strategy combining GLUT1/NSUN2 axis inhibitor WZB117 with PD-L1 blockade, which synergistically suppressed tumor evolution and reversed immunosuppression in preclinical models, suggesting a novel synergistic therapeutic strategy for treatment-resistant HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b96376e6c37993b534f4d68956c9aa4ba9723b03" target='_blank'>
              The glucose sensor NSUN2-m5C modification regulates tumor-immune glucose metabolism reprogramming to drive hepatocellular carcinoma evolution
              </a>
            </td>
          <td>
            Jing He, Boqiang Liu, Weijun Zhao, Hao Shen, Yi Wang, Weiqi Li, Chenqi Jin, Yifan Wang, Xiujun Cai, Liang Shi
          </td>
          <td>2025-07-11</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The extracellular matrix (ECM) plays a critical role in colorectal cancer (CRC) progression and therapeutic resistance. Accurate characterization of ECM composition and architecture is essential for understanding how CRC evades therapy, yet most protocols either assess ECM components in isolation or remain technically challenging. Here we present a robust yet simple protocol for spatial characterization of collagen and glycosaminoglycan (GAG) organization within the CRC tumor microenvironment (TME). Our method combines Alcian Blue and Picrosirius Red staining procedures with standardized tissue processing and imaging protocols. The protocol enables simultaneous visualization, assessment, and quantification of collagen and GAG distribution patterns in formalin-fixed, paraffin-embedded tissue sections. Key methodological advances include optimized dual-staining approach with distinct blue-red coloration for straightforward spectral separation on digital imaging systems, standardized reagent preparations, and validated imaging parameters. The complementary wavelengths facilitate both visual interpretation and potential digital separation, offering advantages over multi-component stains with overlapping spectral ranges. Validation across multiple CRC patient specimens demonstrates excellent reproducibility with consistent staining intensity using standard histology equipment. Because the resulting spatial maps can be compared directly with engineered or ex vivo tumor models, the protocol also provides a practical benchmark for microenvironment validation. This standardized approach to ECM visualization will advance TME research, support morphological studies, and enable comparative analyses across CRC subtypes. The methodology can be adapted to other solid tumor types and integrated with complementary techniques including digital pathology workflows for comprehensive microenvironment characterization and enhanced analysis capabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00377ff7110d150be1ad2ae02040cd414655a3e" target='_blank'>
              Protocol for spatial characterization of ECM collagen-GAG in CRC tumor microenvironment
              </a>
            </td>
          <td>
            Libby A. Boykin, Sneha Jayashankar, Khidr Kishan K. Budhwani, Brahma Mubarak K. Budhwani, Diya Samal, Chelsea L. Crawford, Allan Tsung, Karim I. Budhwani
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Gastric cancer (GC) poses a significant threat to human health. Despite considerable advancements in immunotherapy for GC, the effectiveness of current immunotherapeutic targets remains constrained by the heterogeneity of the tumor microenvironment and mechanisms of immune evasion. Consequently, the identification of novel immunotherapy targets has emerged as a critical area of research. This study investigates the potential of Pygo2 as a target for immunotherapy in GC. Methods The expression and cell localization of Pygo2 in GC tissues were characterized by single cell sequencing, flow cytometry and mIHC. The relationship among Pygo2 expression and prognosis, immune microenvironment and immunotherapy effect was studied in 282 gastric cancer patients. Results The findings indicate a significant upregulation of Pygo2 expression in GC tissues, particularly within tumor cells and T cells. Pygo2 expression in T cells is not only correlated with the advanced T stage and N stage but also inversely associated with patient survival. Additionally, overexpression of T cell Pygo2 resulted in a significant increase in TCF7, which suggested Pygo2+ T cells might represent a subset of exhausted T cells. The study also demonstrated that the density of Pygo2+ CD8+ T cells is negatively correlated with the efficacy of immunotherapy. Conclusion Tumor-infiltrating Pygo2+ T cells could be applied as a clinical prognosticator and a predictive biomarker for immunotherapy responsiveness to GC. These findings offer new therapeutic targets for the treatment of GC and provide fresh insights into cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da9ab431143dec1c75f52a7552a2aca85e59fcb0" target='_blank'>
              Pygo2+ T cells possess immunosuppressive features and inferior immunotherapeutic response in gastric cancer
              </a>
            </td>
          <td>
            Weilong Chang, Huifang Yan, Yawei Zhang, Zibo Sang, Bei Bu, Rui Deng, Kaibo Li, Jiajing Li, Yang Fu, Jinyuan Cui
          </td>
          <td>2025-07-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Serous tubal intraepithelial carcinomas (lesions) in the human fallopian tube epithelium (hFTE) are theorized to give rise to high-grade serous ovarian cancers. Small extracellular vesicles (sEV) are known to mediate key signaling in both normal and cancerous tissues, but few ex vivo systems exist for studying the impact of sEV on hFTE tissue. In this study, we present a microfluidic tissue culture platform with combined spatial transcriptomic and proteomic readouts that allows us to profile dual responses in tissue exposed to sEV “messages”—capturing both short-term transcriptomic shifts in the tissue and long-term changes in protein cargo of secreted EVs (the “reply”). Using spatial transcriptomics, we show that the short-term 1-day exposure to ovarian cancer–derived sEVs alters expression of 68 transcripts in secretory cells, the progenitor of high-grade serous ovarian cancer, notably upregulating immune-related mRNA, including CXCL family chemokines, VCAM1, and pro-inflammatory mediators (NFKB1, IL1B, and IFNA7/17). Additionally, we observed that the long-term 14-day exposure to sEVs alters the expression of seven transcripts and 25 EV cargo proteins of fallopian tube–derived EVs (“secondary release EVs”) following stimulus from cancer EVs. Together, tissue transcriptomics and tissue-derived EV proteomics indicate that ovarian cancer–derived sEVs rewire target cell signaling to modify the tubal immune landscape. This study provides insights into the early molecular changes associated with the pathogenesis of ovarian cancer in its tissue of origin, providing a platform to study EV–tissue interactions and identify how sEVs drive cell signaling reprogramming in hFTE. Significance: We model the fallopian tube preneoplastic landscape using a microfluidic platform to study EV-induced stress and show that cancer EVs promote immune signaling changes representing the earliest stages of ovarian cancer pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7b9b7316e20845062da859010e1284d4df77437" target='_blank'>
              Defining the Ovarian Cancer Precancerous Landscape through Modeling Fallopian Tube Epithelium Reprogramming Driven by Extracellular Vesicles
              </a>
            </td>
          <td>
            Jared Sipes, Didi Zha, Sagar Rayamajhi, Leonidas E Bantis, Rashna Madan, Amrita Mitra, Rajni V Puri, M. Rahman, Foyez Ahmmed, H. Pathak, Angela Russo, Mihaela Sardiu, Brett C Isenberg, B. Cain, J. Coppeta, Pamoda M Galhenage, S. Pathania, Shannon MacLaughlan David, Joanna E. Burdette, Andrew K. Godwin
          </td>
          <td>2025-07-21</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ffde0bbfbd90bb0bf5e018161ce107f39f7740f" target='_blank'>
              Assessing the current molecular understanding of therapeutic targets in osteosarcoma.
              </a>
            </td>
          <td>
            Chao Zhang, Ji-Lan Yang
          </td>
          <td>2025-08-07</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background The unique stress response state of T cells (Tstr) is found in various cancers and correlates with altered Lung Adenocarcinoma (LUAD) tumor microenvironment and immunotherapy outcomes. However, fewer studies have been described on the relationship between Tstr and LUAD, and its mechanism of action in the pathogenesis of LUAD needs to be further investigated. Methods The categorization and validation of cluster types for Tstr signature genes were systematically carried out using transcriptomic data sourced from the TCGA and GEO databases. The focus was on elucidating distinct pharmacological characteristics, tumor microenvironmental traits, and immunotherapeutic benefits across various subtypes. Moreover, a predictive model was developed through Least Absolute Shrinkage and Selection Operator (LASSO) regression analysis, and subsequent cellular experiments confirmed the pivotal role of CREG2 in driving LUAD cell proliferation and migration. Results Tstr signature genes represented by the heat shock protein family are highly expressed at the end of T-cell differentiation and affect cellular communication. Patients with the C1 subtype have a poorer prognosis and high expression of HSPA1A and HSPA1B, which exhibit cold tumor characteristics and define them as important Tstr subtypes. CREG2 is positioned as a promising biological indicator as it plays an important part in enhancing the migratory and proliferative capacity of LUAD cells. Discussion This study reveals the characteristics of different Tstr types. A predictive model was built to forecast the prognosis of LUAD patients based on the DEGs of the subtype. The exploration delved into the bio-function of CREG2 in the progression of LUAD. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03170-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77dda53b77f8e92a63f02ef3e1d805a06740b60e" target='_blank'>
              Integration of single cell and bulk transcriptomes identifies T cell stress subtypes in LUAD
              </a>
            </td>
          <td>
            Shengping Min, Linfeng Pan, Xinyu Zhang, Huili Chen, Lixuan Qiu, Xinyu Wang, Yiluo Xie, Kai Zhang, Qiang Zhang, Chaoqun Lian, Jing Zhang
          </td>
          <td>2025-07-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The bone marrow microenvironment is intimately linked to the biology that underpins the development and progression of multiple myeloma. However, the complex cellular and molecular features that form bone marrow niches are poorly defined. Here, we used subcellular spatial transcriptomics to profile the expression of 5,001 genes in human bone marrow in the context of multiple myeloma. Using this approach, we explored the plasma cell and stroma ecosystem in bone marrow trephines from 21 individuals, including 7 with pre-malignant disease and 10 with newly diagnosed multiple myeloma. Using spatial transcriptomics in conjunction with an optimised trephine biobanking methodology, we could resolve major components of the human bone marrow microenvironment and reliably characterise distinct plasma cell populations in samples from healthy, pre-malignant disease and active myeloma. When plasma cells were visualised in the context of location, we detected spatially restricted subpopulations of plasma cells in five of ten newly diagnosed myeloma trephines. Surprisingly, the composition of haematopoietic and stromal microenvironments varied significantly between newly diagnosed myeloma trephines. Furthermore, these differences in microenvironments were also observed within trephines that had spatially restricted plasma cell subpopulations. Thus, these data are not consistent with the hypothesis that a universal bone marrow microenvironment supports the expansion of malignant plasma cells in myeloma. Instead, we propose that myeloma subpopulations form distinct microenvironments and can vary between both patients and spatial location.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb6467d36212cc2446a47ddf3e54593d16e09483" target='_blank'>
              Profiling the spatial architecture of multiple myeloma in human bone marrow trephine biopsy specimens with spatial transcriptomics.
              </a>
            </td>
          <td>
            R. K. Yip, Jeremy Er, Lei Qin, Quoc Hoang Nguyen, Allan Motyer, Joel S. Rimes, Amanda Light, Ruvimbo D. Mishi, Ling Ling, Casey J. A. Anttila, Ellen Tsui, D. Amann-Zalcenstein, Mark R. Dowling, K. Rogers, R. Bowden, Yunshun Chen, Simon J Harrison, Edwin D Hawkins
          </td>
          <td>2025-07-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Isocitrate dehydrogenase wild-type glioblastoma, a malignant brain tumour of glial origin, confers a poor prognosis with a median survival of 12 to 16 months from diagnosis. Glioblastomas are aggressive tumours that rapidly proliferate and diffusely infiltrate surrounding brain tissue. Current multimodal standard treatment is typically ineffective and despite gross total surgical resection, tumours recur with more aggressive sub-clonal populations of malignant cells. A defining characteristic of glioblastoma is its highly heterogeneous nature and acquirement of somatic mutations advantageous to tumour growth and suppression of apoptotic pathways. Pathogenesis of malignant brain tumours as well as its mode of transformation to a more aggressive subtype is still largely unknown. Although genomic studies have elucidated a plethora of genetic markers associated with glioblastoma subtypes, only a few have been utilised in a clinical setting. One of the emerging approaches to studying glioblastomas is by investigating how an active proteome contributes to its aggressive nature. Furthermore, through activation of specific pathways via post-translational modifications of proteins such as phosphorylation, glioblastomas create an intricate network of signalling pathways which favour tumour growth and proliferation. Here, we investigated the feasibility of diverse methodological approaches to describe abnormal protein signalling across distinct intra-tumour regions of primary glioblastoma tissue, including proliferative core, peripheral rim, and invasive margin. Whilst we observe a broadly comparable proteome relative to the human non-diseased brain, we identify cytoplasmic proteins α-trypsin, actin, apolipoprotein A1 and transthyretin which may putatively be associated with the GBM infiltrative tumour margin. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-09228-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26cc570a29b72848055078fed6cd3945f46575d9" target='_blank'>
              Regional profiling reveals a distinct glioblastoma infiltrative margin proteome
              </a>
            </td>
          <td>
            A. Kocoń, Stuart J. Smith, B. Morentin, Luis F. Callado, Wayne Carter, Ruman Rahman
          </td>
          <td>2025-07-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Immune evasion represents a significant challenge in oncology. It allows tumors to evade immune surveillance and destruction, thereby complicating therapeutic interventions and contributing to suboptimal patient outcomes. This review addresses the critical need to understand how cancers evade immune surveillance. It aims to provide a comprehensive overview of strategies of tumors to escape immune detection by examining tumor-induced immune suppression, immune checkpoint regulation, and genetic and epigenetic influences. Moreover, it explores the dynamic role of the tumor microenvironment (TME) in fostering immune resistance and highlights the impact of metabolic reprogramming on immune suppression. Additionally, this review focuses on how tumor heterogeneity influences immune evasion and discusses the limitations of current immunotherapies. The role of key signaling pathways, including programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte–associated antigen 4 (CTLA-4), transforming growth factor-β (TGF-β), nuclear factor kappa-light-chain-enhancer of activated B cells (NF-κB), and cyclic GMP-AMP synthase-stimulator of interferon genes (cGAS–STING) is analyzed to elucidate their contributions to immune escape. Emphasizing the complexities of immune evasion, this review underscores the importance of personalized approaches and the integration of multi-omics data to combat therapeutic resistance. Furthermore, it discusses novel and emerging therapeutic strategies, such as bispecific antibodies, oncolytic viruses, and nanotechnology-driven immunotherapies, showcasing innovative avenues in cancer treatment. The significance of this review lies in its potential to guide future research and innovations in immunotherapy, ultimately improving patient outcomes and advancing our understanding of cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b9bd056fe3296a46c5f79e0e0f8c410fd536528" target='_blank'>
              Immune evasion in cancer: mechanisms and cutting-edge therapeutic approaches
              </a>
            </td>
          <td>
            Muhammad Tufail, Canhua Jiang, Ning Li
          </td>
          <td>2025-07-31</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Oxaliplatin-based neoadjuvant chemotherapy (NAC) is the standard treatment for advanced colorectal cancer (CRC), yet resistance to NAC poses a significant clinical challenge. Methods To investigate the mechanisms of chemoresistance, we analyzed single-cell RNA sequencing (scRNA-seq) data from CRC patients undergoing NAC. Comprehensive analyses, including InferCNV, differentially expressed genes analysis, pathway enrichment, cell communication, and SCENIC were performed. High-throughput drug screening identified potential therapeutic candidates targeting chemoresistance pathways, and the efficacy of targeting the KLF5/PI3K/AKT axis in combination with oxaliplatin was explored in animal models. Results NAC effectively reduced tumor burden and enhanced T_NK cell infiltration in responsive tumors. Notably, NAC-resistant cell clusters exhibited activation of fatty acid-related metabolic pathways and demonstrated limited immune infiltration. Transcriptional analysis identified KLF5 as a potential driver of chemotherapy resistance. Based on these findings, we developed a KLF5 regulon-associated risk score model with significant potential for predicting CRC patient prognosis. Mechanistically, KLF5 activation of the PI3K/AKT pathway conferred chemoresistance in CRC cells. Through high-throughput screening, GDC-0941, a PI3K/AKT inhibitor, emerged as a promising therapeutic agent that synergistically enhanced oxaliplatin efficacy and overcame resistance in preclinical models. Conclusions Targeting the KLF5/PI3K/AKT axis may enhance chemotherapy efficacy and overcome drug resistance in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c89694e65cc06e36cf7753e13edc503c27ae0798" target='_blank'>
              Targeting the KLF5/PI3K/AKT axis as a therapeutic strategy to overcome neoadjuvant chemoresistance in colorectal cancer
              </a>
            </td>
          <td>
            Meiling Gao, Jiahao Qian, Ping Xia, Wancheng Liu, Yunjia Jiang, Yuchang Xia, Xin Yao, Q. Jiao, Minggang Wei
          </td>
          <td>2025-07-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48a5df41c5ba0be3a9d26dc85ce353b479996b4" target='_blank'>
              A patient-derived orthotopic xenograft model unveils metastatic dynamics in head and neck squamous cell carcinoma.
              </a>
            </td>
          <td>
            Yuchen Bai, Benjamin Blyth, Jarryd M Boath, Qiji Deng, Ruihong Huang, C. Darido
          </td>
          <td>2025-08-11</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Decades of research into the molecular signalling determinants of B cell fates, and recent progress in characterising the genetic drivers of lymphoma, has led to a detailed understanding of B cell malignancies but also revealed daunting heterogeneity. While current therapies for diffuse large B-cell lymphoma are effective for some patients, they are largely agnostic to the biology of each individual’s disease, and approximately one third of patients experience relapsed/refractory disease. Consequently, the challenge is to understand how each patient’s mutational burden and tumour microenvironment combine to determine their response to treatment; overcoming this challenge will improve outcomes in lymphoma. This mini review highlights how data-driven modelling, statistical approaches and machine learning are being used to unravel the heterogeneity of lymphoma. We review how mechanistic computational models provide a framework to embed patient data within knowledge of signalling. Focusing on recurrently dysregulated signalling networks in lymphoma (including NF-κB, apoptosis and the cell cycle), we discuss the application of state-of-the-art mechanistic models to lymphoma. We review recent advances in which computational models have demonstrated the power to predict prognosis, identify promising combination therapies and develop digital twins that can recapitulate clinical trial results. With the future of treatment for lymphoma poised to transition from one-size-fits-all towards personalised therapies, computational models are well-placed to identify the right treatments to the right patients, improving outcomes for all lymphoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6853f7a1c4a91fa28c35000a4f2aefb81e3d0bb5" target='_blank'>
              Computational modelling of aggressive B-cell lymphoma
              </a>
            </td>
          <td>
            E. Jayawant, Aimilia Vareli, Andrea G S Pepper, Chris Pepper, F. Simoes, Simon Mitchell
          </td>
          <td>2025-07-04</td>
          <td>Biochemical Society Transactions</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic cancer is a common cause of cancer mortality, and pancreatic ductal adenocarcinoma (PDAC) is the most common subtype. Tumor-associated neutrophils (TANs) have been recognized as potential therapeutic targets. In this study, we utilized bulk and single-cell RNA sequencing (scRNA‒seq) to identify seven distinct subtypes of neutrophils in PDAC. Oxidized low-density lipoprotein receptor 1 (OLR1)+ neutrophils and macrophage migration inhibitory factor (MIF)+ neutrophils were classified as TANs. The clinical relevance, dynamic transitional process, function, cell‒cell communication and transcription factor activity of neutrophil subclusters in PDAC were characterized. Furthermore, the novel MIF+ TANs were fully validated in PDAC tissues, an orthotopic pancreatic tumor model and a patient-derived xenograft (PDX) model. MIF+ TANs promote the proliferation and migration of PDAC cells through the activation of the ERK and AKT pathways and epithelial‒mesenchymal transition (EMT). This study provides insight into the potential of MIF+ TANs as therapeutic targets for PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63493c7d2e0e9105a107c3ea49cac4f828b092e4" target='_blank'>
              Single-cell transcriptome analysis identifies MIF as a novel tumor-associated neutrophil marker for pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yan Zeng, Jiaping Yu, Yutong Chen, Juan Zhuang, Xinyue Liang, Yaning Li, Shili Chen, Wenzheng Pang, Linjuan Zeng
          </td>
          <td>2025-08-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a55b2dafa57fa725379d7dc71b70c66988362113" target='_blank'>
              Clustered macrophages cooperate to eliminate tumors via coordinated intrudopodia.
              </a>
            </td>
          <td>
            L. Dooling, Alişya A Anlaş, Michael P. Tobin, Nicholas M. Ontko, Tristan Marchena, Maximilian Wang, Jason C. Andrechak, D.E. Discher
          </td>
          <td>2025-07-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Osteosarcoma, a prevalent malignant bone tumor in pediatric and adolescent populations, is characterized by a markedly diverse tumor microenvironment (TME) that complicates therapeutic intervention. Among the key components of the TME, M2 macrophages contribute to immune suppression, tumor progression, and resistance to therapy. This study aims to explore the immune microenvironment of osteosarcoma and identify key immunosuppressive components using integrated transcriptomic analysis.



 We integrated scRNA-seq (GSE152048) and bulk RNA-seq (TARGET) data to characterize the osteosarcoma tumor microenvironment. Key analyses included cell clustering and annotation (Seurat), pseudotime trajectory (Monocle 2), cell-cell communication (CellChat), and M2 macrophage deconvolution (CIBERSORT, BayesPrism). Gene modules were identified via WGCNA, and drug sensitivity was predicted using oncoPredict. Key genes were validated by qRT-PCR and Western blot.



 Our integrated analysis revealed that M2 macrophage infiltration was significantly associated with an immunosuppressive tumor microenvironment in osteosarcoma. Weighted gene co-expression network analysis (WGCNA) identified gene modules highly correlated with M2 macrophage abundance and immune evasion signatures. Reduced M2 infiltration was linked to altered transcriptional patterns and increased predicted sensitivity to Etoposide and Doxorubicin. Two immune-related genes, PTK2B and PTPRC, were significantly upregulated in osteosarcoma samples and validated by RT-qPCR and Western blotting in U2OS cells.



 This study demonstrates that M2 macrophage infiltration is a defining feature of the osteosarcoma immune microenvironment and is associated with immune suppression. Through integrative transcriptomic analysis, PTK2B and PTPRC were identified as key immune-related genes linked to M2 macrophage abundance and patient prognosis. These results establish a transcriptional basis for the immunosuppressive phenotype in osteosarcoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c9aeb21260637fca67e1110200fbde1ea1d4d" target='_blank'>
              Integrated RNA sequencing reveals tumor microenvironment heterogeneity and immunosuppressive role of M2 macrophages in osteosarcoma
              </a>
            </td>
          <td>
            Zhijun Ding, Yi Li, Cheng Zhong
          </td>
          <td>2025-08-01</td>
          <td>Oncologie</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Immune-related genes (IRGs) play a pivotal role in the tumor microenvironment of triple-negative breast cancer (TNBC). This study aimed to uncover novel prognostic markers for TNBC based on IRGs. Differential expression analysis, univariate Cox regression, and machine learning algorithms identified BPI, GPHA2, KIR3DL3, MASP1, RASGRP1, and SEMA4A as potential prognostic genes. A random survival forest (RSF) model was developed and subsequently validated, alongside the creation of a prognostic nomogram to assess gene diagnostic performance. Further, epithelial cells, fibroblasts, and T cells were recognized as the primary cell types through which RASGRP1 exerts its effects. Finally, an analysis of the immune microenvironment and drug predictions provided fresh insights for the treatment of TNBC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-15157-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f5a164db4384d30a7ce7d87a791eadd2b7becb9" target='_blank'>
              Immune microenvironment and prognostic genes of triple-negative breast cancer in the context of transcriptome and single-cell sequencing
              </a>
            </td>
          <td>
            Xiangqing Huang, Wei Chen, Zhiwu Lin, Hui Zhang, Ruijuan Wang, Cailing Lin
          </td>
          <td>2025-08-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Glioma, a tumor derived from neuroglial cells of the central nervous system, is characterized by its high malignancy, frequent recurrence, and high mortality rate. Recent advances in exosome research related to gliomas have propelled this emerging field as a pivotal area of clinical neuroscience investigation. Despite extensive focus on glioma exosome research, a systematic visual analysis of this field remains lacking. Using bibliometric analysis, this study maps the scientific landscape of glioma exosome research to identify current hotspots and emerging trends in neuro-oncology. Methods This bibliometric analysis extracted glioma exosome publications from the Web of Science Core Collection(WoSCC) spanning 2005–2025. Following systematic screening, 209 articles constituted the analytic corpus. Using VOSviewer(VOS), Scimago Graphica, and CiteSpace, we mapped contributions across geographic, institutional, authorship, and conceptual domains. This visualization delineates the field’s intellectual structure, identifies current research foci, and projects emerging trends in neuro-oncology. Results Annual publication volume on glioma exosomes has increased significantly since 2019. China and the United States are the predominant contributors, collectively driving field advancement. Leading institutions include Shandong University, Shandong Provincial Key Laboratory of Brain Function Reconstruction, and Capital Medical University, whose productivity metrics reflect substantial influence. Notably, Chinese institutions comprise 90% of the top 10 productive entities. Key investigators Li Gang and Xue Hao demonstrate exceptional scholarly output, advancing the field through high-impact research. NEURO-ONCOLOGY emerges as the premier journal, evidenced by its dominant citation frequency. Thematic analysis reveals three conceptual domains: Exosome biogenesis in gliomas, Glioblastoma extracellular vesicle signatures, and Exosome mediated drug delivery systems-representing current priorities and evolving research frontiers. Conclusion Exosomes have attracted considerable attention in glioma research, establishing themselves as a cutting edge thematic focus. This study applies bibliometric rigor to map research trajectories and conceptual evolution in glioma exosomes. The analysis establishes foundational frameworks that prioritize future investigation domains while objectively quantifying the field’s translational potential. These evidence-based insights will accelerate mechanistic discovery and clinical translation in neuro-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce5f890f1f8b8ce61524a09398cf2f3457fe7ab4" target='_blank'>
              Evolution of glioma exosome research: emerging trends and global collaborations (2005–2025)
              </a>
            </td>
          <td>
            Yijun Zeng, Ting Liu, Shixue Jing, Yongfeng Wang, Jihong He, Hui Cao, Xinlian Liu, Lushun Zhang
          </td>
          <td>2025-07-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Cancer stem-like cells (CSCs) represent a subset of tumor cells that have the ability to self-renew, a long lifespan and a relatively quiescent phenotype, and show resistance to conventional therapies. Various markers are used to identify CSCs, and have shown that different CSC subtypes may be present within a tumor. One functional property of CSCs is their relative lack of proteasomal activity compared to the tumor bulk. Methods We introduced an unstable fluorescent molecule into FaDu oropharyngeal squamous cell carcinoma cells and analyzed the association of proteasome activity with aldehydehyde dehydrogenase (ALDH) activity as another common CSC marker, and with other stem-cell related properties of glucose metabolism. We also analyzed publicly available gene expression profiling data of ALDH+ CSCs for alterations in mRNAs associated with proteostasis. Results We show that FaDu CSCs identified by low proteasome activity are associated with the population identified by high ALDH activity. Futher characterization shows that these CSCs have a relatively high mitochondrial membrane potential and low levels of glucose transporter, indicating a non-Warburg metabolic phenotype. We also show that proteasome-low FaDu CSCs exhibit decreased rates of protein synthesis. Gene expression profiling of other cancer cell lines reveal common statistically significant differences in proteostasis in ALDH+ CSCs compared to the bulk of the tumor cells, including reduced levels of Hsp70 and/or Hsp90 in CSCs defined by ALDH, together with reduced levels of UCHL5 mRNA. Conclusions These data provide additional insights into the functional characteristics of proteasome-low/ALDH-high CSCs, indicating a metabolic phenotype of reduced reliance on aerobic glycolysis and a decreased protein synthesis rate. We also identify specific chaperone and ubiquitin ligase activities that can be used to identify CSCs, with corresponding implications for therapeutic strategies that target CSCs through their altered metabolic properties. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14460-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1caf3233dadab8ec33f9106bfa9a067dc15c6bb" target='_blank'>
              Heterogeneous protein dynamics links to mitochondrial activity, glucose transporter, and ALDH cancer stem cell properties
              </a>
            </td>
          <td>
            Martin Krkoška, Zuzana Tylichová, P. Zatloukalová, Petr Müller, B. Vojtěšek, Philip J Coates
          </td>
          <td>2025-07-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="ABSTRACT Developing advanced preclinical models and targeted therapies is essential for reducing cancer-related deaths in children with solid tumors. Patient-derived xenografts (PDX) have the potential to replicate key elements of the original tumor, including morphology, genetic alterations, and microenvironment, making them valuable tools for studying tumor biology and drug response. We implanted 124 pediatric solid tumor samples, collected for 1 y, into NOD/SCID/IL2Rg (NSG) mice. Tumor fragments were placed subcutaneously, and the animals were monitored for up to 1 y. Histopathology, Short Tandem Repeat (STR) profiling, RT-PCR and/or RNA-sequencing were performed to confirm tumor identity and detect driver fusions. Fifty-five xenografts were successfully established (44.35% of implanted samples), representing 19 tumor types. Sarcomas, notably osteosarcoma, Ewing sarcoma, synovial sarcoma, and rhabdomyosarcoma, displayed first-generation engraftment rates above 55%. Central nervous system tumors had lower success, reflecting unique microenvironmental requirements. Histopathology and STR concordances were 85.45% and 81.1%, respectively, while 92.6% of sarcoma PDXs retained original fusion genes. Second-generation xenografts showed faster growth, suggesting adaptation to the murine host. Sporadic discrepancies, such as new fusions or lymphoproliferative expansions, indicated the need for ongoing molecular validation parallel to other techniques. A pediatric PDX biobank can effectively capture key tumor features while facilitating the study of therapeutic responses and tumor evolution. Our models confirm the feasibility of achieving stable histological and molecular profiles, offering a valuable resource for precision oncology research. Ultimately, these pediatric PDXs could accelerate the discovery of targeted therapy and significantly improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e2beb1824c0da93aca0d94e158f83285bfd918e" target='_blank'>
              Establishing a pediatric solid tumor PDX biobank for precision oncology research
              </a>
            </td>
          <td>
            Larissa Akemi Kido, Milena Rodrigues Marusco, Ellen Aparecida da Silva, Laís Do Carmo, Ana Beatriz Teodoro Borges, Felipe Luz Torres Silva, Juliana Silveira Ruas, Dieila Giomo de Lima, Larissa de Abreu Fernandes, Camila Maia Martin Daiggi, Izilda Aparecida Cardinalli, Mayara Ferreira Euzébio, Patricia Yoshioka Jotta, Mariana Maschietto, Priscila Pini Zenatti
          </td>
          <td>2025-08-13</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Multiple sclerosis (MS) is a chronic autoimmune disorder of the central nervous system (CNS) that predominantly affects young adults. However, current disease-modifying therapies demonstrate limited efficacy in addressing progressive disease subtypes, underscoring the urgent need for novel therapeutic strategies. Here, we systematically review the neuroimmune interactions underlying the pathogenesis of MS, with a focus on three key aspects: the immune niche, immune cell types, and cell-based therapies. We first discuss the evolution of brain-immune concepts, from early notions of immune privilege to modern understandings of brain-border immune niches (meninges, choroid plexus, and perivascular spaces). These compartments serve as critical interfaces where peripheral immune cells interact with CNS-resident immune cells. We then analyze the roles of specific immune cell subsets (e.g., T/B cells, myeloid cells and microglia) in disease progression, highlighting their functional heterogeneity across different MS subtypes. Furthermore, we highlight emerging MS immunotherapies-including chimeric antigen receptor (CAR) T regimens, mesenchymal stem cell interventions, microbiome modulation, and nanodelivery systems, which strategically target mechanistic nodes spanning neuroimmune niche regulation, inflammatory cascade blockade, and CNS neurorestorative capacities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27edf65b186f82cf8db863821048aa25425e97e" target='_blank'>
              Immunological microenvironment and targeted therapeutics in multiple sclerosis: new insights in crosstalk between immune niches and CNS
              </a>
            </td>
          <td>
            Xiaodi Sun, Feng Zhang, Luojinyun Wang, Gyeore Lee, Sibo Yang, Daqiang Zhou, Bohao Chang, Bo Hu, Yifan Zhou
          </td>
          <td>2025-08-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Liquid biopsy offers a promising, minimally invasive approach for cancer detection by providing valuable insights into tumor biology. Among the various sources used in liquid biopsy, platelets and their RNA stand out as a unique diagnostic tool, reflecting the body’s systemic response to cancer. However, during laboratory platelet extraction known as platelet washing, white blood cells (WBCs) retention can occur, potentially confounding platelet-derived RNA sequencing data.



 To address this challenge, we developed a method to quantify WBC enrichment in platelet RNA-seq datasets. Using the largest publicly available dataset (GEO GSE183635), which includes 2,351 samples from cancer patients, healthy donors, and individuals with benign, non-cancer conditions, we identified 3 sample clusters based on leukocyte marker levels. This allowed us to differentiate two entirely distinct sets of samples, with the lowest and the highest WBC retention, containing 377 matched samples in each set. We then assessed how the leukocyte presence influences the performance of machine learning models for cancer classification for the low and high leukocyte subgroup.



 Across the full test set (n=453), our model achieved an area under the curve (AUC) of 0.94. Interestingly, in the low-leukocyte subset (n=229), the AUC slightly decreased to 0.93, whereas in the high-leukocyte subset (n=224), it increased to 0.95.



 These results suggest that while leukocyte retention may slightly enhance classification performance, platelets alone provide substantial diagnostic value. Overall, our findings highlight the strong potential of platelet-based liquid biopsy and reveal how understanding leukocyte retention can further refine cancer detection strategies.



 Michał Sieczczynski, Krzysztof Pastuszak, Anna J. Zaczek, Matthew T. Rondina, Sjors GJG. in 't Veld, Myron G. Best, Anna Supernat. Decoding the Leukocyte Effect: How Cell Retention Shapes ML Outcomes in Platelet RNA-based Cancer Detection [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B034.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1028fbb68d5dca2426241f5c4eaf3adb3c531ac0" target='_blank'>
              Abstract B034: Decoding the Leukocyte Effect: How Cell Retention Shapes ML Outcomes in Platelet RNA-based Cancer Detection
              </a>
            </td>
          <td>
            Michal Sieczczynski, Krzysztof Pastuszak, A. Żaczek, Matt Rondina, S. I. '. In 't Veld, M. Best, A. Supernat
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Tumor Treating Fields (TTFields) therapy is an approved cancer treatment modality, based on non-invasive application of electric fields to the tumor region. Proteomic and cell biology methods revealed a versatile mechanism of action to be involved in the response to TTFields. In the current research we performed whole transcriptome analysis across tumor types to identify pan-cancer responses to TTFields. For this we collected samples from control and TTFields-treated human cancer cell lines of gastric cancer, pancreatic cancer, ovarian cancer, non-small cell lung carcinoma, pleural mesothelioma, and glioblastoma. The transcriptomic analysis supported previous reported effects: downregulation of pathways associated with cell cycle, cell growth, and proliferation; downregulation of DNA replication and the FA-BRCA DNA repair pathway; and upregulation of cellular responses to stress—senescence, autophagy, and apoptosis. Notably, previously unrecognized downstream effects of TTFields were revealed on cellular metabolism, with downregulation of protein and RNA metabolism, and upregulation of steroid biosynthesis. Additional DNA repair pathways were also found to be downregulated, including nucleotide excision repair, base excision repair, and mismatch repair. In conclusion, this study revealed similar response patterns to TTFields across different tumor types, re-enforcing some already pinpointed mechanisms, while revealing new mechanisms. Unlocking these new mechanisms may allow identification of potential new cancer treatments for application together with TTFields based on mechanistical compatibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1627b07ec04103438850967d9453d926d5ec1e3d" target='_blank'>
              The transcriptomic fingerprint of cancer response to Tumor Treating Fields (TTFields)
              </a>
            </td>
          <td>
            K. Wainer-Katsir, A. Haber, H. Fishman, L. Ding, Michael D. Story, Renfei Du, U. Kahlert, L. Mannarino, F. Mirimao, M. Lupi, Maurizio D’Incalci, G. Lavy-Shahaf, Hila Ene, R. Frechtel-Gerzi, Zeina Drawshy, A. Martinez-Conde, E. Dor-On, Y. Porat, M. Giladi, U. Weinberg, Y. Palti
          </td>
          <td>2025-07-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) remains a daunting global health concern. It is marked by aggressive progression and poor survival. While immunotherapy has emerged as a promising treatment modality, both primary and acquired resistance continue to limit its clinical impact, leaving many patients without durable benefits (e.g., CheckMate-648, ESCORT-1st). This review explains resistance mechanisms and suggests new strategies to improve outcomes. These mechanisms include immunosuppressive cells (Treg cells, myeloid-derived suppressor cells), inhibitory cytokines, molecular alterations involving programmed death 1/programmed death-ligand 1 signaling, and impaired antigen presentation. We also highlight key clinical trials—for example, CheckMate-648 and ESCORT-1st—that reveal both the potential and pitfalls of current immune checkpoint blockade strategies, underscoring the need for robust predictive biomarkers. Moreover, we examine cutting-edge tactics to overcome resistance, including combination regimens, tumor microenvironment remodeling, and tailored treatment approaches rooted in the patient’s unique genomic and immunologic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8755d6f455eb60021c79724f1c8204130606cf4" target='_blank'>
              Unmasking immune checkpoint resistance in esophageal squamous cell carcinoma: Insights into the tumor microenvironment and biomarker landscape
              </a>
            </td>
          <td>
            Zhe Wang, Rui-Ying Zhang, Yi-Fan Xu, Bingtong Yue, Jia-Yi Zhang, Feng Wang
          </td>
          <td>2025-08-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5a92276acbe2123f6e1cab5bdfe86dc9b524ae1" target='_blank'>
              Age and Generation-Based Model of Metastatic Cancer
              </a>
            </td>
          <td>
            Panagiotis Gavriliadis, Georgios Lolas, T. Matsoukas
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Simple Summary Although targeted therapies and immune checkpoint inhibitors have improved patient outcomes, most metastatic renal cell carcinoma tumors eventually become resistant to treatment and progress. In this study, we used a syngeneic mouse model of metastatic renal cell carcinoma to investigate how gene expression changes during disease progression and in response to treatment with cabozantinib, an anti-PD-1 antibody, or both. We found that different treatments activated distinct molecular pathways in metastatic tumors. These treatment-specific changes influenced how tumors responded to subsequent therapies. Our findings suggest that the biology of metastatic renal cell carcinoma evolves depending on prior treatment, and that selecting second-line therapies based on these molecular changes may improve outcomes. This research provides insights that could help guide future treatment strategies and support the development of more effective, personalized therapy sequences for patients with metastatic renal cell carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a2406df3fc5a264b372ca5f6f1b3b41d0f1310" target='_blank'>
              Treatment-Induced Gene Expression Changes in Metastatic Renal Cell Carcinoma: Insights from a Syngeneic Mouse Model
              </a>
            </td>
          <td>
            Kohshi Okabe, Toshiaki Tanaka, Tetsuya Shindo, Yuki Kyoda, Sachiyo Nishida, Kohei Hashimoto, K. Kobayashi, N. Masumori
          </td>
          <td>2025-07-01</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Background Cervical cancer remains a global public health challenge, particularly in regions with limited access to screening and vaccination. While high‐risk HPV infection is the primary cause, the genetic and molecular mechanisms driving cervical cancer progression are not fully understood. Objective This study integrates Mendelian randomization (MR) and single‐cell RNA sequencing (scRNA‐seq) to identify causal eQTL‐related genes and explore their roles in tumorigenesis. Functional experiments were conducted to validate key findings. Methods MR analysis identified eQTL‐related genes with significant causal associations with cervical cancer. Functional enrichment and Gene Set Variation Analysis (GSVA) revealed their involvement in key pathways. scRNA‐seq explored cell‐specific expression patterns and immune cell infiltration in the tumor microenvironment (TME). In vitro experiments, including qRT‐PCR, siRNA knockdown, migration, proliferation, and colony formation assays, validated the biological roles of pivotal genes. Results A total of 307 eQTL‐related genes were identified, enriched in pathways such as Th17 cell differentiation, TNF, and IL‐17 signaling. scRNA‐seq revealed cell‐specific expression of key genes, including FUOM, which was elevated in cervical cancer cells. FUOM knockdown significantly reduced cell proliferation (by 37%, p < 0.001), migration (by 43%, p < 0.001), and colony formation (by 62%, p < 0.001). Regulatory analysis identified miRNAs as upstream modulators of these genes. Conclusion This study identifies FUOM as a novel driver gene in cervical cancer progression and highlights its role in tumorigenesis and immune modulation. These findings provide insights into potential biomarkers and therapeutic targets, offering a foundation for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f2af3648a1778b7bc1979bb8e9f7cee9b957bcb" target='_blank'>
              Integrative Genomic and Functional Approaches Identify FUOM as a Key Driver and Therapeutic Target in Cervical Cancer
              </a>
            </td>
          <td>
            Wenzhi Jiao, Shanshan Liu, Jianwei Shi, Minmin Yu
          </td>
          <td>2025-08-01</td>
          <td>Cancer Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9d6d4cae3e0a9086b4188f54060ec83ad93a702" target='_blank'>
              Integrating scRNA-seq and spatial transcriptomics to explore the implication of G6PD on immune microenvironment in lymphatic metastasis of breast cancer.
              </a>
            </td>
          <td>
            Hongsen Liu, Mengting Chen, Bo Hong, Ruijin Liang, Lijie Fan, Yun Qian
          </td>
          <td>2025-07-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PLXNA3 is a transmembrane protein essential for axon guidance and synapse formation, gaining attention in cancer research for its roles in cell migration, invasion, and signal transduction. However, its biological significance in breast cancer remains underexplored. Here, bulk RNA sequencing and survival analysis revealed that elevated PLXNA3 levels were correlated with increased malignancy and poorer survival in breast cancer. Single-cell RNA sequencing demonstrated that PLXNA3 was predominantly enriched in tumor cell clusters within the tumor immune microenvironment, rather than in immune cell clusters. In vitro experiments further demonstrated that knockdown and overexpression of PLXNA3 significantly altered the proliferative, invasive, and migratory capacities of breast cancer cells. Additionally, the overexpression of PLXNA3 was negatively correlated with immune activation status and served as a predictor of poor response to anti-PD1 immunotherapy. Our findings suggested that high expression of PLXNA3 is associated with poor prognosis in breast cancer and plays a crucial role in cancer immunity, making it a promising novel target for intervention in breast cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13843-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f126a6deb0547875700a586994249ce5224fe17" target='_blank'>
              Comprehensive analysis of the PLXNA3 gene on prognosis and immune characteristics in breast cancer
              </a>
            </td>
          <td>
            Wen Sun, Yifan Xu, Qi Qi, Weiya Zhang, H. Pan, Guangshun Sun, Shui Wang, Wenbin Zhou
          </td>
          <td>2025-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Immunogenic cell death (ICD) is crucial for cancer development and enhances the effectiveness of immunotherapy by triggering an adaptive immune response. However, the impact of ICD-related genes (ICDRGs) on iCCA progression and patient prognosis remains unclear. iCCA samples from The Cancer Genome Atlas (TCGA) were categorized into two ICD-associated subtypes by consensus clustering, and patient prognosis in these subgroups was assessed. Single-cell RNA sequencing was conducted on viable cells from 8 iCCA patients to understand their transcriptomic profiles, heterogeneity, and immune microenvironment. Principal component analysis (PCA) and uniform manifold approximation and projection (UMAP) analysis were applied to identified 13 cell subpopulations. The gene set enrichment analysis (GSEA) was used to investigate pathway heterogeneity among these subpopulations. CellChat was employed for intercellular communication analysis. Pseudotime trajectory analyses with CytoTRACE and Monocle2 were used to explore differentiation trajectories and functional differences across various states. 30,485 single cells from human iCCA and normal tissues were sequenced, revealing two ICD-associated subtypes. The ICD-low subtype correlates with better clinical outcomes, whereas the ICD-high subtype is linked to increased immune cell infiltration and immune response signaling activity. Distinct biological functions and metabolic characteristics were observed in iCCA between high- and low-ICD groups. High infiltration of ICD-low subtype in iCCA is associated with poor prognosis, potentially through the ANXA-FPR pathway affecting macrophage proliferation and differentiation. Our study constructed an immunogenic cell death-related signature that can promise prognosis prediction and personalized medicine in iCCA. This finding might help guide clinicians for iCCA patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08602-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e45d84f628dac0dca7f3bfd929def33d5b1582" target='_blank'>
              Single cell RNA sequencing to identify an Immunogenic cell death related prognostic signature in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Bowen Sha, Fei Tong, Qinghua Shu, Miao Wu, Liang Zhao, Yufeng Zhang
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Emerging research emphasizes the critical role of local microbiota in shaping the tumor microenvironment (TME) and influencing cancer progression. Lung adenocarcinoma (LUAD) is distinguished by unique bacterial communities that appear to regulate immune responses, gene expression, and patient outcomes. Methods We compiled microbiome profiles from several cancer types—including LUAD, lung squamous cell carcinoma (LUSC), breast carcinoma (BRCA), and thyroid carcinoma (THCA)—using public databases. Non-negative matrix factorization (NMF) was employed to categorize LUAD cases based on TME features, while DESeq2 was used to pinpoint bacterial taxa with differing abundance. Multi-omics networks were developed to integrate microbial, transcriptomic, and clinical data. For in vitro verification, we conducted siRNA-mediated knockdown of the long non-coding RNA LCIIAR and ISG15 in Lewis lung carcinoma cells, followed by proliferation assays. Results In contrast to LUSC, BRCA, and THCA, LUAD exhibited distinct microbial populations, with notable enrichment of Cylindrospermopsis, Cyanothece, and Sulfolobus. NMF clustering identified two LUAD subtypes with differing prognoses. One longer survival cluster, marked by reduced bacterial presence and stronger antitumor immunity—reflected in stronger immune response, increased effector T cells activity, and greater immune cell infiltration. A competing endogenous RNA (ceRNA) network analysis established a link between LCIIAR and ISG15, both overexpressed in LUAD and associated with worse survival outcomes. Knockdown LCIIAR or ISG15 through siRNA significantly inhibited lung cancer cell proliferation, pointing to their roles in tumor growth and ceRNA-mediated regulation. Conclusion LUAD features a distinctive microbiota that engages with inflammatory and ceRNA regulatory pathways. These observations underscore the value of targeting microbiome-influenced mechanisms, such as the LCIIAR–ISG15 axis, as a promising approach to enhance treatment outcomes in lung adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93a45784f057c67e4efb8c56c492f819f5fd7032" target='_blank'>
              Distinct Lung Adenocarcinoma-Associated Microbiota Are Associated with Inflammatory Immune Landscapes and Tumor Cell Proliferation via LCIIAR–ISG15 Regulatory Networks
              </a>
            </td>
          <td>
            Shipu Liu, Zijian Zhang
          </td>
          <td>2025-07-01</td>
          <td>Cancer Management and Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Three‐dimensional (3D) cell culture models derived from patient tumors are currently under development to recapitulate in vivo physiological conditions and assess therapeutic responses. In vitro models often fail to replicate the drug sensitivity observed in humans. In this study, a standardized in vitro culturing method using a 3D bioprinted breast cancer tumor model was established. We compared traditional two‐dimensional (2D), spheroid, and 3D bioprinted in vitro models as well as in vivo growing syngeneic or xenograft tumors. Our aim was to determine whether 3D bioprinted in vitro cultures can represent tissue heterogeneity, growth capacity, and/or drug response as potential tools for personalized drug sensitivity tests. Our findings demonstrated that 3D bioprinted models closely mimic in vivo tumor morphology and drug responses, outperforming 2D cultures and patient‐derived xenografts (PDX) in severe combined immunodeficiency mice. Additionally, 3D bioprinted models showed similar drug sensitivity to syngeneic tumors regrown in BALB/c mice, highlighting their potential for better therapeutic response predictions. Our results support the use of 3D bioprinted tumor models for personalized oncology. The presented approach could significantly advance personalized cancer therapy by using 3D bioprinted tumor tissues, offering a more accurate representation of tumor behavior and treatment efficacy compared to currently used PDX models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ffacc0113987c6e08dc28232bcb9d0736ad5403" target='_blank'>
              Mimicking breast cancer tissue—3D bioprinted models in accurate drug sensitivity tests
              </a>
            </td>
          <td>
            G. Petővári, Dorottya Moldvai, R. Raffay, T. Dankó, Dániel Sztankovics, Andrea Reszegi, Risa Miyaura, Rebeka Gelencsér, András Rókusz, C. Tolnai‐Kriston, Zsófia Vilimi, Nikolett Kállai-Szabó, T. Visnovitz, A. Sebestyén
          </td>
          <td>2025-07-29</td>
          <td>VIEW</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Meningioma is the most prevalent primary brain tumor with extensive intra-tumoral heterogeneity and high recurrence rates, particularly in high-grade meningiomas. Despite advancements in understanding the molecular underpinnings of meningiomas, the longitudinal evolutionary trajectory and cellular diversity of recurrent tumors remain elusive. In this study, we perform single-nuclei sequencing of matched primary and recurrent meningiomas to explore the dynamic transcriptional heterogeneity and evolutionary trajectory of meningioma tumor cells, as well as their molecular interactions with tumor-associated immune cells that shape the complex milieu of the meningioma microenvironment. Our findings reveal that both primary and recurrent meningiomas constitute diverse cellular compositions and hierarchies, where recurrent tumor cells are characterized by enrichments of cell cycle activities and proliferative kinetics. Integrative RNA velocity and latent time uncover divergent transcriptional trajectories in recurrent tumors, demonstrating multidirectional transitions with the dominance of COL6A3, which confers higher risk vulnerability and treatment resistance. Tumor microenvironment analysis further reveals enrichments of COL6A3-mediated interactions between immunosuppressive macrophages and tumor cells in recurrent meningiomas. Collectively, these results provide profound insights into the complex evolutionary process of meningiomas and suggest potential therapeutic strategies for the treatment of recurrent tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46967d3ec0f9b8d5fcb467d9f78f81fb62b39f73" target='_blank'>
              Single-cell analysis reveals a longitudinal trajectory of meningioma evolution and heterogeneity
              </a>
            </td>
          <td>
            Ji Yoon Lee, Eun Jung Lee, Bo Yeon Seo, Jiwon Kim, Youjin Song, Dayoung Lee, Namsung Moon, Harim Koo, Chul-Kee Park, Min-Sung Kim, Serk In Park, Do-Hyun Nam, D. Kong, Jason K. Sa
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Liver metastasis remains a major cause of death in colorectal cancer (CRC). Interactions between tumor cells and components of the tumor microenvironment (TME) are integral to the progression of cancer cell migration and metastatic dissemination. Investigating these cellular dynamics is essential for identifying actionable therapeutic targets. Methods This study employed bioinformatics analysis, IHC, WB, and murine in vivo imaging to delineate the role of SPOCD1 in CRC. Additional in vitro co-culture systems, immunofluorescence, and RNA-seq were utilized to determine how SPOCD1 modulated epithelial-mesenchymal transition (EMT) through cancer-associated fibroblasts (CAFs). Mechanistic insight into SPOCD1-mediated transcriptional regulation of LAMA4 was obtained via dual-luciferase reporter assays, ChIP-qPCR, and Co-IP. Results SPOCD1 was found to be markedly upregulated in CRC cells and exhibited potent pro-metastatic activity in vivo. Integration of external datasets with experimental validation revealed a strong correlation between SPOCD1 expression and CAFs infiltration in the TME. Further analyses demonstrated that SPOCD1 enhanced CXCL12 expression in CAFs via upregulation of LAMA4, enabling CXCL12 to engage CXCR4 on CRC cells and activate EMT signaling, thereby driving metastasis. This signaling axis was effectively interrupted by CXCR4 inhibitors. Mechanistically, SPOCD1 promoted LAMA4 expression through DNMT1 recruitment, facilitating DNA methylation-dependent transcriptional regulation. Conclusion This study delineates a SPOCD1-centered interaction network between CRC cells and CAFs in colorectal cancer liver metastasis (CRLM), offering novel molecular candidates for therapeutic intervention in CRLM. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06863-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4d8174ccdaba2a7e9f4b9613ed271a53eaeae82" target='_blank'>
              Cancer cell SPOCD1 promotes colorectal cancer liver metastasis by activating the CXCL12/CXCR4 signaling pathway in cancer-associated fibroblasts
              </a>
            </td>
          <td>
            Geng Peng, Lin Zhong, Lina Luo, Yongle Ju, Yan Lu, L. Ng, D. Foo, Fu Cheng, Luji Huang, M. Ouyang
          </td>
          <td>2025-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Smooth muscle cells (SMCs) play a crucial role in atherosclerosis, undergoing proliferation, migration, and transdifferentiation into other cell types. Multiple SMC subsets exist, each with distinct roles that are increasingly being elucidated. To investigate the heterogeneity of SMC subsets in lesion, we integrated three single-cell RNA sequencing datasets. Bioinformatics analysis was conducted to evaluate the effects of the immune microenvironment, biological function, metabolic interactions, and differentiation potential of different SMC subsets. Atherosclerotic plaques from ApoE−/− mice were used for validation. We employed pySCENIC to predict the transcription factors associated with different SMC subsets and used bulk RNA sequencing data to identify the function of the_C5 marker gene (ALDOA). We identified a unique smooth muscle cell cluster 5 (SMC_C5) in human coronary plaques and confirmed its presence in aortic plaques of ApoE−/− mice. SMC_C5 interacts with macrophages and monocytes, influencing the microenvironment and participating in various pathways and metabolic processes. Additionally, SMC_C5 cells exhibit pluripotent differentiation potential. Mechanistically, we found a strong association of SMC_C5 and the transcription factor GA binding protein 1 (GABP1). The SMC_C5 marker gene (ALDOA) was involved in regulating metabolism, hypoxia response, and the immune microenvironment. This investigation provides novel insights into the complex biology of SMCs and reveals novel functions of SMC_C5 in atherogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f75d3eece9d205d449ff08ed145c865b6bbbff00" target='_blank'>
              Single-cell RNA sequencing in human atherosclerotic plaques reveals a novel smooth muscle cell subtype that possesses multi differentiation potential and shapes the microenvironment
              </a>
            </td>
          <td>
            Weijie Ye, Zhibing Chen, Zhiwei Xia, Dong-Jie Li
          </td>
          <td>2025-07-16</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND Colorectal cancer (CRC) remains a major global health burden due to its high incidence and mortality, with treatment efficacy often hindered by tumor heterogeneity, drug resistance, and a complex tumor microenvironment (TME). Lactate metabolism plays a pivotal role in reshaping the TME, promoting immune evasion and epithelial-mesenchymal transition, making it a promising target for novel therapeutic strategies and prognostic modeling in CRC. AIM To offer an in-depth analysis of the role of lactate metabolism in CRC, highlighting its significance in the TME and therapeutic response. METHODS Utilizing single-cell and transcriptomic data from the Gene Expression Omnibus and The Cancer Genome Atlas, we identified key lactate metabolic activities, particularly in the monocyte/macrophage subpopulation. RESULTS Seven lactate metabolism-associated genes were significantly linked to CRC prognosis and used to construct a predictive model. This model accurately forecasts patient outcomes and reveals notable distinct patterns of immune infiltration and transcriptomic profiles mutation profiles between high- and low-risk groups. High-risk patients demonstrated elevated immune cell infiltration, increased mutation frequencies, and heightened sensitivity to specific drugs (AZD6482, tozasertib, and SB216763), providing a foundation for personalized treatment approaches. Additionally, a nomogram integrating clinical and metabolic data effectively predicted 1-, 3-, and 5-year survival rates. CONCLUSION This report underscored the pivotal mechanism of lactate metabolism in CRC prognosis and suggest novel avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1cc03f92fc7424a656f3531e66027ab68b3f01d" target='_blank'>
              Deciphering lactate metabolism in colorectal cancer: Prognostic modeling, immune infiltration, and gene mutation insights
              </a>
            </td>
          <td>
            Xiao-Peng Wang, Jia-Xin Zhu, Chang Liu, Hao-Wen Zhang, Guan-Duo Sun, Jing-Ming Zhai, Hai-Jun Yang, De-Chun Liu
          </td>
          <td>2025-07-07</td>
          <td>World Journal of Gastroenterology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Macrodactyly is a congenital overgrowth disorder characterized by pathological adipose proliferation due to PIK3CA mutations in adipose-derived stem cells (ADSCs). Due to the somatic mosaicism, the affected tissues comprise a mixture of mutant and wild-type cells. However, how PIK3CA mutated ADSCs influence adjacent wild-type cells in macrodactyly remains poorly understood. In this study, we utilized coculture systems to investigate the effects of macrodactylous adipose-derived stem cells (Mac-ADSCs) on normal ADSCs, fibroblasts (FBs), and vascular endothelial cells (VECs). Our study demonstrated that activating PIK3CA mutations in Mac-ADSCs promotes the proliferation, migration, invasion, adipogenesis, and angiogenesis of wild-type ADSCs, FBs and VECs. Furthermore, using RNA sequencing and cytokine arrays, we revealed that these effects are primarily mediated by various secreted paracrine cytokines. These findings demonstrated that activating PIK3CA mutation alters the paracrine characteristics of Mac-ADSCs and reshapes the microenvironment of macrodactyly, driving adjacent wild-type cells to exhibit mutant-like phenotypes. Targeting PIK3CA with BYL-719 could influence the progression of macrodactyly by inhibiting the paracrine signaling of Mac-ADSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e67317efe81db7638c9468323480f426b18afb72" target='_blank'>
              Activating PIK3CA mutations in adipose-derived stem cells drive mutant-like phenotypes of wild-type cells in macrodactyly
              </a>
            </td>
          <td>
            Xiao Zhang, Yating Yin, Zhibo Wang, Yongkang Jiang, Aiping Yu, Xinyi Dai, Xiaoli Wang, Xuesong Guo, Hailei Mao, Bin Wang
          </td>
          <td>2025-07-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793beb6dae04857b331a1472ade96cbcdfbcb83e" target='_blank'>
              Integrated protocol for single-cell RNA sequencing, organoid culture, and RNA-protein spatial analysis in gastroesophageal tissues
              </a>
            </td>
          <td>
            Naveen Kumar, Pon Ganish Prakash, Rajendra Kumar Gurumurthy, Cindrilla Chumduri
          </td>
          <td>2025-07-07</td>
          <td>BMC Methods</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tongue Squamous Cell Carcinoma (TSCC) represents a significant subtype of malignant oral cancer, characterized by a heterogeneous tumor microenvironment (TME). The tongue, a complex muscular organ, naturally presents an initial microenvironment that is largely inhospitable to the initiation and progression of TSCC. However, advanced-stage TSCC exhibits a pronounced accumulation of cancer-associated fibroblasts (CAFs), indicative of a drastic microenvironmental transformation. In this study, through comprehensive analysis combining cell model assessments and single-cell RNA sequencing data from a 4-NQO-induced TSCC mouse model, along with a lineage tracing-in-transplant assay, we elucidate and confirm the process of transdifferentiation whereby tongue muscle cells (TMCs) convert into CAFs in the TSCC context. Furthermore, we demonstrate that targeting TSCC with an IL-17a inhibitor offers a viable strategy to inhibit the reprogramming of TMCs into CAFs. Additionally, our research also identifies four critical marker genes involved in the transdifferentiation of TMCs to CAFs, Thbs1, Crabp1, Ifi205, and Cxcl5. Collectively, these findings delineate a mechanism through which TSCC cells induce the transdifferentiation of TMCs into CAFs, thereby transforming the cancer-suppressive microenvironment of the tongue into one that is conducive to TSCC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/666df9c3fc7fca0534d4dd3d77f31ac2e7399d3f" target='_blank'>
              Transdifferentiation of tongue muscle cells into cancer-associated fibroblasts in response to tongue squamous cell carcinoma
              </a>
            </td>
          <td>
            Weifan Lin, Wu-hua Huang, Shiqiang Mei, Xiangwan Lu, Junheng Zheng, Hua Wang, Lingling Zheng, Yan Zhang
          </td>
          <td>2025-07-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Transcriptome profiling by RNA sequencing (RNAseq) can provide insightful information on the molecular processes underlying disease development and progression. Although fresh tissue represents the preferred source material for RNAseq, here, we investigated the feasibility of applying RNAseq analysis to single 10 μm thick formalin-fixed and paraffin-embedded (FFPE) lung slides from the lungs of control and bleomycin (BLM)-treated mice. This approach aims at providing spatial-oriented transcriptomic data, that can be integrated with in vivo and ex vivo readouts obtained on the same sample, as a way to enhance the mechanistic information and biomarker/target discovery potential of preclinical models of fibrotic lung diseases. Methods RNAseq analysis was conducted on individual FFPE slides from the lungs of both controls and BLM-treated mice. The results were initially validated by comparison with publicly available bulk data from fresh-frozen (FF) mouse tissues, both untreated and BLM-treated, as well as human idiopathic pulmonary fibrosis (IPF) biopsies. Unsupervised cluster analysis was performed on Differentially Expressed Genes (DEGs) distinguishing untreated and BLM-treated fibrotic lung samples. For each sample, Pearson correlation analysis was used to compare expression levels of individual gene clusters with Ashcroft Scores and aeration compartments quantitatively assessed on the matched 2D micro-CT coronal slice. Results Over 90% of annotated genes within the FFPE dataset were shared with gene signatures retrieved from FF bulk datasets. Differentially modulated gene clusters were mainly found to be associated with extracellular matrix (ECM) organization, tissue remodeling, and inflammatory response pathways. For each sample, expression levels of individual gene clusters were highly correlated with 2D histology readouts and aeration compartments determined on matched 2D coronal slices by micro-CT imaging. Conclusions FFPE lung tissue represents a valuable alternative to fresh tissue for RNAseq analysis, allowing to achieve a more precise, spatially oriented picture of pulmonary disease development. This approach is thus instrumental to a better characterization of the molecular changes associated to each sample. It can also contribute to a more informed interpretation of histology and micro-CT imaging data, paving the way to the identification of translationally relevant biomarkers as well as novel candidate targets for the development of more effective therapeutic interventions. Supplementary Information The online version contains supplementary material available at 10.1186/s12931-025-03300-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca3f0c49a6eeb93406bc59871126d70b0cd7e03" target='_blank'>
              Spatial transcriptomic and morpho-functional information derived from single mouse FFPE slides allows in-depth fingerprinting of lung fibrosis
              </a>
            </td>
          <td>
            Erica Ferrini, Costanza Bonfini, Giovanna Marchese, Martina Buccardi, M. Zoboli, P. Faccioli, N. Sverzellati, G. Villetti, Simone Ottonello, Maria Ravo, F. F. Stellari
          </td>
          <td>2025-07-02</td>
          <td>Respiratory Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>