<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-10-20 09:31:01 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>102</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Aggressive cancers, such as pancreatic ductal adenocarcinoma (PDAC), are often characterized by a complex and desmoplastic tumor microenvironment, a stroma rich supportive connective tissue composed primarily of extracellular matrix (ECM) and non-cancerous cells. Desmoplasia, a dense deposition of stroma, is a major reason for therapy resistance, acting both as a physical barrier that interferes with drug penetration and as a supportive niche that protects cancer cells through diverse mechanisms. Precise understanding of spatial cell interactions in stroma-rich tumors is essential for optimizing therapeutic responses. It enables detailed mapping of stromal-tumor interfaces, comprehensive cell phenotyping, and insights into changes in tissue architecture, improving assessment of drug responses. Recent advances in multiplexed immunofluorescence imaging have enabled the acquisition of large batches of whole-slide tumor images, but scalable and reproducible methods to analyze the spatial distribution of cell states relative to stromal regions remain limited. To address this gap, we developed an open-source computational pipeline that integrates QuPath, StarDist, and custom Python scripts to quantify biomarker expression at a single- and sub-cellular resolution across entire tumor sections. Our workflow includes: (i) automated nuclei segmentation using StarDist, (ii) machine learning-based cell classification using multiplexed marker expression, (iii) modeling of stromal regions based on fibronectin staining, (iv) sensitivity analyses on classification thresholds to ensure robustness across heterogeneous datasets, and (v) distance-based quantification of the proximity of each cell to the stromal border. To improve consistency across slides with variable staining intensities, we introduce a statistical strategy that translates classification thresholds by propagating a chosen reference percentile across the distribution of marker-related cell measurement in each image. We apply this approach to quantify spatial patterns of distribution of the phosphorylated form of the N-Myc downregulated gene 1 (NDRG1), a novel DNA repair protein that conveys signals from the ECM to the nucleus to maintain replication fork homeostasis, and a known cell proliferation marker Ki67 in fibronectin-defined stromal regions in PDAC xenografts. The pipeline is applicable for the analysis of markers of interest in stroma-rich tissues and is publicly available.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b6327fb55ba3776af18a8a1265a4ec0c3c69a0b" target='_blank'>
              An image analysis pipeline to quantify the spatial distribution of cell markers in stroma-rich tumors
              </a>
            </td>
          <td>
            Antoine A. Ruzette, Nina Kozlova, Kayla A. Cruz, Taru Muranen, S. Nørrelykke
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) presents diverse histomorphological features within the tumor microenvironment (TME) that influence prognosis and response to immunotherapy. Leveraging contrastive learning, we developed an unbiased atlas of cell neighborhoods to systematically explore the LUAD microenvironment at the cellular scale and investigate how these neighborhoods combine to form histologic patterns. This multiscale approach enables a comprehensive understanding of both cell-specific interactions and broader histologic architecture in LUAD. Our analysis identified distinct histomorphological phenotype clusters of cellular neighborhoods (cn-HPCs) with prognostic significance. For example, cn-HPC 0 was associated with immune activation and favorable survival, while cn-HPC 23 was enriched in necrotic, immune-excluded regions and aligned with poorer outcomes. Through associations with immunophenotypes, co-expressed gene modules, and pathway enrichment, we found that cn-HPCs reflect underlying processes of immune modulation, cellular growth, and inflammation, offering insights into the functional landscape of LUAD. In an independent LUAD immunotherapy cohort, we found that cn-HPC 23 showed promising predictive value for treatment response. These findings suggest that fine-grained spatial profiling of the TME may offer complementary value for identifying patients more likely to benefit from immunotherapy. Taken together, our study highlights the potential of cn-HPCs to bridge histopathological patterns and clinical outcomes, supporting their relevance for prognosis and treatment stratification in LUAD. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17117-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732a50286d99e4889f11b9dfeef8dfb5de086217" target='_blank'>
              Contrastive learning uncovers cellular interactions and morphologies in the tumor microenvironment of lung adenocarcinoma linked to immunotherapy response
              </a>
            </td>
          <td>
            H. Le, N. Coudray, A. Yeaton, Sitharam Ramaswami, A. Karimkhan, Wei-Yi Cheng, James Cai, Tai-Hsien Ou Yang, S. Punekar, Luis Chiriboga, Vamsidhar Velcheti, Kwok-Kin Wong, D. Sterman, Andre L. Moreira, Harvey I. Pass, A. Tsirigos
          </td>
          <td>2025-10-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Cancer immunotherapy has revolutionized clinical oncology; however, the inherent complexity and heterogeneity of cancer present substantial challenges to achieving broad therapeutic efficacy. Tumor heterogeneity manifests not only among different patients but also within individual tumors, further complicating personalized treatment approaches. Single-cell sequencing technologies encompassing genomics, transcriptomics, epigenomics, proteomics, and spatial omics have significantly enhanced our ability to dissect tumor heterogeneity at single-cell resolution with multi-layered depth. These approaches have illuminated tumor biology, immune escape mechanisms, treatment resistance, and patient-specific immune response mechanisms, thereby substantially advancing precision oncology strategies. This review systematically examines recent advances in single-cell multi-omics technologies across various cancer research areas, emphasizing their transformative impacts on understanding tumor heterogeneity, immunotherapy, minimal residual disease monitoring, and neoantigen discovery. Additionally, we discuss current technical and analytical limitations and unresolved questions associated with single-cell technologies. We anticipate single-cell multi-omics technologies will become central to precision oncology, facilitating truly personalized therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3005de124c813e8dd9ae4675ad8451e32f8f1a34" target='_blank'>
              Single-cell multi-omics in cancer immunotherapy: from tumor heterogeneity to personalized precision treatment
              </a>
            </td>
          <td>
            Jiayuan Le, Yating Dian, Deze Zhao, Ziyu Guo, Zehao Luo, Xiang Chen, Furong Zeng, Guangtong Deng
          </td>
          <td>2025-08-25</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="


 Tumors acquire a set of functional capabilities, known as the Hallmarks of Cancer, to thrive and grow. While extensive research has revealed how gene and protein expression networks contribute to these traits, the relationships between tumor cell states, spatial organization of immune and stromal cells in the tumor microenvironment, and tumor genetics, remains poorly understood. To address this, we constructed a cohort of primary tumor tissues from 117 pancreatic ductal adenocarcinoma (PDAC) patients. Using Multiplexed Ion-Beam Imaging (MIBI) with a 40-antibody panel, we mapped the spatial organization of major cell types and their phenotypes. We also conducted targeted genomic and bulk proteomic analyses to explore how spatial organization patterns align with tumor genetics and proteome profiles. We identified phenotypic subtypes of tumor epithelium that shape distinct microenvironmental niches: (i) Glycolytic tumors expressing proteins such as GLUT1, LDHA, MCT1 and HIF1α reside in hypoxic niches. They are encased by dense fibrotic stroma and lacking vasculature and immune cell infiltration. (ii) Tumor and metaplastic epithelium that exhibit an oxidative proteome profile, are highly enriched for ECAD and PD-L1 expression, and are situated near pancreatic epithelium and vasculature, with significant immune infiltration. (iii) Myeloid-interacting tumors expressing CXCL5, heavily infiltrated with neutrophils and characterized by low-collagen and high FAP positive fibroblasts. Notably, neutrophils accumulate within the lumen of tumor ducts where they exhibit increased apoptosis, DNA damage, and netosis. These highly neutrophil-infiltrated tumors, associate with poor disease outcome. The genetic landscape of PDAC influences these spatial and phenotypic patterns. Patients with TGFβ/SMAD pathway mutations, demonstrate an increased proportion of basal tumor cells accompanied by reduced lymphocytic infiltration. Similarly, distinct TP53 variants shape immune-cell infiltration: patients harboring missense mutations show increased lymphocytic infiltration compared to those with truncating mutations. These findings underscore the strong link between tumor genetics, phenotypic heterogeneity and cellular architecture of the PDAC microenvironment, offering new insights into collective tumor behavior.



 Ofer Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, Inti Zlobec, Tamar Geiger, Leeat Keren. Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B123.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb5d6211d2e33680561c9558492cf85ff16b33" target='_blank'>
              Abstract B123: Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization
              </a>
            </td>
          <td>
            O. Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, I. Zlobec, Tamar Geiger, Leeat Keren
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer cell plasticity, defined as the ability of tumor cells to reversibly adopt distinct functional states, plays a central role in tumor heterogeneity, therapy resistance, and disease relapse. This process enables cells to enter stem-like, dormant, or drug-tolerant persister states in response to treatment or environmental stress without undergoing genetic changes. Such reversible transitions complicate and limit current treatments. Conventional cancer models often fail to capture the complexities of these adaptive states. In contrast, patient-derived tumor organoids (PDOs), which retain the cellular diversity and structure of primary tumors, provide a unique system for investigating plasticity. This review describes how PDOs can model cellular plasticity, such as the emergence of drug-tolerant persister cells and the interconversion between cancer stem cell states across multiple tumor types. We particularly focused on colorectal cancer organoids, for which research on the mechanism of plasticity is the most advanced. Combined with single-cell analysis, lineage tracing, and functional assays, PDOs can help identify the molecular pathways that control plasticity. Understanding these mechanisms is important for developing therapies to prevent treatment failure and control disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd4e7e497e61cac197e5ff80d3e09c68cd18937b" target='_blank'>
              Patient-Derived Tumor Organoids to Model Cancer Cell Plasticity and Overcome Therapeutic Resistance
              </a>
            </td>
          <td>
            R. Coppo, Masahiro Inoue
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract The heterogeneous nature of cell populations in human tumors is a major contributor to tumor evolution, including and perhaps most importantly in response to treatment. Here, we review current knowledge on tumor heterogeneity and cell state plasticity in small cell lung cancer (SCLC), a fast growing and highly metastatic form of lung cancer which develops rapid resistance to therapy. There is a pressing need to expand treatment options for patients with SCLC, which requires a better understanding of the mechanisms by which this disease is able to rapidly grow and evolve in response to therapy. Our current understanding points to epigenetic rather than genetic factors in defining major aspects of inter- and intra-tumoral heterogeneity in SCLC. SCLC is overall considered to be a neuroendocrine (NE) cancer type but SCLC tumors harbor a wide diversity of cancer cell states, including both NE and non-neuroendocrine (non-NE) states, defined by their mutually exclusive expression of a set of transcription factors such as ASCL1, NEUROD1, and POU2F3. The immune microenvironments of SCLC tumors also contain a great deal of heterogeneity. Here, we discuss the different SCLC cell states associated with their defining transcription factors, as well as the epigenetic mechanisms regulating the ability of SCLC cells to switch from one state to another. We further discuss how the composition of SCLC tumors and the surrounding immune cells may affect the response to chemotherapy and immunotherapy. Being able to control plasticity and heterogeneity in SCLC may in the future offer unique opportunities to improve treatment efficacy in this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f9c75cb099eaa87426731186cf92ba20e24f46a" target='_blank'>
              Tumor Heterogeneity and Plasticity in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Gina Duronio, Julien Sage
          </td>
          <td>2025-10-01</td>
          <td>Lung Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spontaneously transformed mouse ovarian surface epithelial cell line ID8 generates a syngeneic mouse model, recapitulating the primary and metastatic stages of the malignancy hence, frequently used in pre-clinical evaluation. Due to inherent phenotypic plasticity, primary and metastatic tumors show contrasting phenotypic features dictated by the tissue niche. To address the tumor heterogeneity within the model we stratified it into a specific molecular subtype, which would provide an accurate setting for interpretation of experimental data. Stratification was performed on ID8 cell line and ID8 generated xenografts based on immunostaining, live cell imaging, histopathology, transcriptomics and proteomics. Tumor microenvironment (TME) investigation for each xenograft was done through flow cytometry analysis. ID8 cells present epithelial/mesenchymal (E/M) phenotype. Phenotypic plasticity retention in them causes class switching of ID8 cells in response to niche alteration, from E/M (in culture) to intermediate epithelial (iE) in orthotopic tumors while intermediate mesenchymal (iM) in subcutaneous tumors. Metastasis of ID8 primary tumor resulted into differentiated mesenchymal phenotype in ascites while differentiated epithelial phenotype in secondary intestinal tumors. Immune profiling of primary and secondary xenografts revealed differences in immune repertoire of tumors, highlighting subcutaneous tumor model to be inappropriate as compared to orthotopic model. Local tissue niche modulates the phenotype of tumors generated, transitioning through intermediate states towards a final commitment to either an epithelial or mesenchymal state that recapitulates the reported class switching in patient tumors. During disease progression, such heterogeneity of tumor cells at different niche(s) and their cross-talks with discrete TME at each site, will pose challenges for effective therapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18264-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20023e2da4c4317c3adb1e410526c772109c27da" target='_blank'>
              ID8 cells manifest phenotypic plasticity and molecular heterogeneity of high-grade serous ovarian cancer in response to the local tissue niche
              </a>
            </td>
          <td>
            Ritika Gupta, S. Shravan, Swapna Balaji Agwane, R. M. Bankar, S. Bapat
          </td>
          <td>2025-10-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has emerged as an advanced biological technology capable of resolving the complexity of cancer landscapes at single-cell resolution. Spatial transcriptomics(ST), as an innovative complementary approach, effectively compensates for the lack of spatial information inherent in scRNA-seq data. This review explores the rapidly evolving integration of scRNA-seq and ST and their transformative role in deciphering the tumor microenvironment (TME). We highlight how these technologies jointly uncover cellular heterogeneity, stromal-immune interactions, and spatial niches driving tumor progression and therapy resistance. Moving beyond previous reviews, we emphasize emerging computational strategies for data integration—including deconvolution and mapping approaches—and evaluate their applications in characterizing immune evasion, fibroblast diversity, and cell-cell communication networks. Ultimately, this review provides a forward-looking perspective on how spatial multi-omics are poised to advance precision oncology through spatially-informed biomarkers and diagnostic tools. We conclude that the full clinical potential of these technologies relies on closing the gap between analytical innovation and robust clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84801b1d17cdb40a9473ba25f6bac709b5c4c887" target='_blank'>
              Single-cell and spatial transcriptomics integration: new frontiers in tumor microenvironment and cellular communication
              </a>
            </td>
          <td>
            Wenxin Shi, Zhiqiang Zhang, Xiaotong Xu, Yanpeng Tian, Li Feng, Xianghua Huang, Yanfang Du, Zhongkang Li
          </td>
          <td>2025-10-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="The heterogeneity and dynamic evolution of the tumor microenvironment (TME) are key drivers of tumor progression, immune evasion, and therapeutic resistance. Traditional transcriptomic approaches are limited by their reliance on population-averaged data and the lack of spatial context, making it challenging to fully dissect the complex interactions among tumor cells, immune infiltrates, and stromal components within the TME. In recent years, the synergistic advancement of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has revolutionized our ability to construct high-resolution cellular atlases and delineate spatially organized functional modules within the TME. These technologies have enabled the identification of novel cell subtypes, cell-cell communication pathways, and spatial niches associated with disease progression and therapy response. This review systematically compiles recent technological breakthroughs in the integration of single-cell and spatial multi-omics, highlights key discoveries in TME research, and discusses their transformative impact on the development of precision oncology. Additionally, it addresses current technical limitations and explores emerging strategies and future directions for advancing this rapidly evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90a0c36ae523e1a4fc23b5c8bc8013f692b5b483" target='_blank'>
              Refining Tumor Microenvironment Insights via Single-Cell and Spatial Transcriptomics for Precision Medicine
              </a>
            </td>
          <td>
            Xianbing Wang
          </td>
          <td>2025-09-05</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is a molecularly diverse form of lung cancer characterized by distinct oncogenic driver mutations that influence both tumour biology and clinical outcomes. Understanding the interplay between these oncogenic drivers and the tumour microenvironment (TME) is crucial for improving therapeutic strategies and patient management. Here, we investigate the impact of driver mutations on the composition and spatial architecture of the TME in LUAD. Using imaging mass cytometry (IMC), we analyse tumour samples from 157 LUAD patients, integrating genomic and clinical data to link specific mutations with tumour characteristics. Unique patterns are associated with mutated KRAS and EGFR tumours with TP53 co-mutations, suggesting these co-mutations reshape the TME and promote resistance to tyrosine kinase inhibitors (TKIs). Overall, our findings highlight the complex interplay between oncogenic driver mutations and the TME in LUAD, underscoring the importance of integrating genomic and cellular data to understand the underlying tumour behaviour and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05eac26faa3f5fa852d857cbc84d4c9c3228ae87" target='_blank'>
              Oncogenic driver mutations underlie the spatial tumour immune landscape of non-small cell lung cancer
              </a>
            </td>
          <td>
            Saskia Hartner, H. Abolfathi, M. Rezanejad, Bridget Liu, E. Karimi, Dakota Rogers, Mark Sorin, Samuel Doré, Lysanne Desharnais, M. Orain, William Enlow, A. Gagné, Yuhong Wei, Y. Bossé, D. Quail, Philippe Joubert, L. Walsh
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Simple Summary Spatial transcriptomics is an advanced technology that allows for the study of gene activity in cancer tissues while preserving the original tissue structure. Unlike traditional methods that either analyze samples without location information or examine individual cells removed from their natural environment, spatial transcriptomic technology maps where genes are active within intact tumor tissue. This approach enables a greater understanding of how different areas of a tumor vary, how cancer cells interact with surrounding healthy tissue, and how cells are organized spatially within the tumor at high resolution. As spatial technology becomes more widely used, it may improve cancer diagnosis and treatment by helping to create more personalized and effective therapies for individual patients. This review outlines the current and future role of spatial transcriptomics methodologies in the context of genitourinary cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa8ee089aa47992263f8b69f0d5705b49d03f797" target='_blank'>
              Current Role and Future Frontiers of Spatial Transcriptomics in Genitourinary Cancers
              </a>
            </td>
          <td>
            Firas Hatoum, Adnan Fazili, Justin W Miller, Xuefeng Wang, Xiaoqing Yu, Xin Lu, Jeffrey S. Johnson, Philippe E. Spiess, J. Chahoud
          </td>
          <td>2025-08-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Adult gliomas are a diverse class of tumors with substantial molecular and pathological variation. While prior single-cell omics studies have characterized intratumoral heterogeneity, they often examined tumors in isolation, thus overlooking the recurrence of tumor states across subtypes. Given their shared environment, understanding the conservation of tumor states across gliomas is essential.



 To address this, we analyzed 310 tumor cores from 284 patients spanning oligodendroglioma, low- and high-grade astrocytoma, glioblastoma, and ependymoma, using single-cell spatial transcriptomics with a 344-gene panel (10x Xenium). This approach yielded a dataset of 2.8 million cells, of which 84% were tumor and 16% represented 18 non-malignant cell types from the tumor microenvironment (TME). To further investigate local cellular interactions, we applied an extended spatial transcriptomics panel (5,096 genes) alongside multiplexed immunofluorescence (57 proteins) to a validation cohort of low-grade astrocytomas.



 Our analysis revealed tumor states recurring across gliomas, albeit in varying proportions, organized into neighborhoods that reflect local cell-cell interactions. For instance, proliferating tumor cells tended to co-localize with oligodendro- and neuro-progenitor-like tumor states, suggesting that these programs may give rise to proliferating cells. Spatial neighborhoods were primarily subtype-specific, indicating that tissue organization and not cell composition alone, underlies glioma heterogeneity. We next assessed the relationship between tumor and immune cells, revealing entity-specific immune microenvironments: astrocytoma and ependymoma exhibited distinct immune responses (T cell- versus myeloid-driven inflammation), whereas oligodendroglioma displayed a low inflammatory landscape. Notably, microglia were consistently observed in proximity to tumor cells across all glioma subtypes, highlighting their conserved role in modulating anti-tumor immunity. Methylation profiling stratified these tumors into 13 molecular subtypes, each associated with distinct neighborhoods. Glioblastoma subtypes RTK1, RTK2, and MES, in particular, displayed unique tissue architectures consistent with their subtype-specific methylation profiles.



 Collectively, these findings link microanatomical features to glioma heterogeneity by connecting tumor states to their microenvironments. Regional differences, such as those between the tumor core and infiltrating edge, shape cellular composition with implications to treatment response. By capturing such spatially dependent patterns, our work provides a framework for advancing the understanding of glioma biology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34fbb58728bc44ec20e86f909612c57947c86652" target='_blank'>
              KS03.6.A THE RECURRENT TISSUE STATES ACROSS ADULT GLIOMAS AS REVEALED BY SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            A. Mathioudaki, Z. Seferbekova, M. Ritter, D. Calafato, G. Rukhovich, F. Hinz, F. Ippen, E. Popova, N. Koeberer, P. Sant, S. Schinkewitsch, J. Malm, S. Dietrich, C. Herold-Mende, N. Etminan, A. Wick, S. Krieg, M. Platten, A. von Deimling, F. Sahm, A. Suwala, M. Gerstung
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Objectives Small cell lung cancer (SCLC) accounts for approximately 15% of lung tumors and is marked by aggressive growth and early metastatic spread. In this study, we used two SCLC mouse models with differing tumor mutation burdens (TMB). To investigate tumor composition, spatial architecture, and interactions with the surrounding microenvironment, we acquired multiplexed images of mouse lung tumors using imaging mass cytometry (IMC). These data build upon a previously published characterization of the mouse model. Data description After tumor detection, mice were assigned to one of five treatment groups. Lung tumor tissues were imaged with a 37-marker IMC panel designed to identify major cell types—tumor, immune, and structural—as well as their functional states. When possible, each tumor was sampled both at its center and border regions. Tumor masks in the form of binary images are provided to delineate tumor areas. Additional metadata include tumor onset and endpoint dates to support downstream correlation or predictive analyses based on the image data. This dataset offers a valuable resource for studying the histological and cellular complexity of SCLC in a genetically controlled mouse model across multiple therapeutic conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597aa5e8dfd6901b69ffa28e7837ffa96623cc96" target='_blank'>
              Imaging mass cytometry dataset of small-cell lung cancer tumors and tumor microenvironments
              </a>
            </td>
          <td>
            France Rose, O. Ibruli, Luca Lichius, M. Kiljan, Gokcen Gozum, Manoela Iannicelli Caiaffa, Jiali Cai, Li‐na Niu, Jan M. Herter, Holger Grüll, Reinhard Büttner, F. Beleggia, Graziella Bosco, Julie George, G. Herter-Sprie, H. C. Reinhardt, Katarzyna Bożek
          </td>
          <td>2025-09-08</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Owing to the complexity of TME components and the heterogeneity of cancer cells, the relationship between the niches of TME and prognosis in breast ductal carcinoma remains unknown. The staged characteristics of corresponding cancer cell behaviors are unclear. Our study aims to reveal spatial structures and specific cellular information of TME and cancer cells subgroups during the progression from DCIS to IDC and lymph node metastasis. Methods Single-cell sequencing, spatial transcriptomics, bulk RNA sequencing datasets were used to explore the changes in microenvironmental components and transcriptional programs of tumor cells during the progression of breast ductal carcinoma. Immunohistochemistry, multiplex immunofluorescence, flow cytometry cell cycle detection, invasion migration experiments, and WB imprinting were employed for validation. Results Analysis of TME cell type subsets revealed the accumulation of TEX, iTreg, and stress-phenotype TAM in the mammary gland in situ during the invasion process. Lymphatic metastases exhibited enrichment of nTregs and a more naïve-like CD8 T cell population. Spatial analysis and survival analysis showed that the spatial niches of CD4 TN and phagocytic-phenotype macrophages were associated with a favorable prognosis, and these niches were lost during disease progression. The proliferative subpopulation of breast ductal carcinoma was enriched in lymphatic metastatic tissues, expressing high levels of FAM111B and exhibiting intense TCA and oxidative phosphorylation metabolism. Silencing FAM111B led to cell cycle arrest, decreased invasion and migration abilities, and downregulation of core mediator genes for cuproptosis and disulfidptosis. Conclusions The stage-specific microenvironmental characteristics of breast ductal carcinoma correspond to some extent to the behavior of tumor cells. During the progression of ductal carcinoma in breast tissue, the establishment of an immunosuppressive microenvironment occurs. The microenvironmental spectrum at lymph node metastases differs somewhat, corresponding to a more enriched turnover of cancer cell proliferation and death. Inhibitors of FAM111B and inducers of cuproptosis and disulfidptosis may serve as potential therapeutic targets for proliferative subgroups. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07010-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfce9a8ff47f521350ed382190ba1004f756bb5c" target='_blank'>
              Spatiotemporal microenvironment landscape and malignant epithelial pattern transition in breast ductal carcinoma progression
              </a>
            </td>
          <td>
            Xifu Cheng, Wenjuan Zeng, Bingzhe Yin, Jiawei Gui, Hengbin Zhang, Zhenxing Lv, Simin Zhang, Yao Zhou
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background The presence of tertiary lymphoid structures (TLS) in solid tumors, including Merkel cell carcinoma (MCC), is associated with a better prognosis and a better response to immunotherapy with immune checkpoint inhibition (ICI). The detailed mechanisms by which TLS influence antitumor immune responses are only partially understood. Methods Clinically annotated tumor tissues of 27 patients with MCC were obtained prior to ICI therapy. Tumor samples were subjected to transcriptomic and multiplex immuno-visual profiling, T-cell receptor (TCR) clonotype mapping, as well as—in selected cases—spatial transcriptomics to comprehensively characterize the tumor immune microenvironment. Results Weighted gene co-expression network analysis (WGCNA) of transcriptomic data in combination with topological overlap measures indicated a higher abundance of TLS in tumors of patients with MCC responding to ICI therapy. This concept was substantiated through immunomorphological analyses, revealing mature B-cell follicle-like structures characterized by high endothelial venules (HEVs). Further supporting HEVs as critical entry points for naïve T cells, the presence of TLS was correlated with a pronounced infiltration of CD4+ and CD8+ T cells, exhibiting both naïve and central memory phenotypes. The TCR repertoire of these infiltrates exhibited enhanced richness and diversity with a pronounced reactivity toward Merkel cell polyomavirus-derived T-cell epitopes. Spatially resolved RNA and V(D)J sequencing revealed the expression of genes associated with T-cell recruitment within TLS, alongside the presence of naïve and central memory T-cell markers. Notably, individual clonally expanded TCR transcripts were detected both within TLS and among tumor-infiltrating lymphocytes. The latter were associated with low expression of memory cell markers and high expression of effector cell markers. Additionally, a spatial gradient in the expression of genes linked to immune stress in MCC cells—such as those involved in the interferon-γ response and antigen processing and presentation machinery—originated in proximity to the TLS. Conclusion Our findings are consistent with a key role of TLS in shaping immune interactions within the MCC microenvironment, driving the recruitment of diverse tumor-reactive T cells. These insights hold promise for advancing immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13de7f3d541927495612436f916863e4107c4925" target='_blank'>
              Tertiary lymphoid structures in Merkel cell carcinoma facilitate naïve and central memory T-cell infiltration linked to immunotherapy response
              </a>
            </td>
          <td>
            Nalini Srinivas, Ivelina Spassova, Kuan Cheok Lei, Jiwei Gao, María José Pino, Giovanni Giglio, S. Kitanovski, Mazdak Dalkoohi, E. Livingstone, Ulrike Leiter, Peter Mohr, T. Gambichler, Ingo Stoffels, Selma Ugurel, Phyllis Fung-Yi Cheung, Camilla Engblom, Weng-Onn Lui, Jürgen C. Becker
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality, with tumor heterogeneity and drug resistance posing significant challenges to treatment. We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA-seq deconvolution to analyze BRCA samples. Our analysis identified 15 major cell clusters, including neoplastic epithelial, immune, stromal, and endothelial populations. Notably, low-grade tumors showed enriched subtypes, such as CXCR4+ fibroblasts, IGKC+ myeloid cells, and CLU+ endothelial cells, with distinct spatial localization and immune-modulatory functions. These subtypes were paradoxically linked to reduced immunotherapy responsiveness, despite their association with favorable clinical features. High-grade tumors exhibited reprogrammed intercellular communication, with expanded MDK and Galectin signaling. Bulk RNA-seq deconvolution further supported the prognostic significance of low-grade-enriched subtypes. Our findings highlight the heterogeneity of the tumor microenvironment and provide new insights into immune evasion and therapeutic resistance in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57925849e8c34466fe3132e9855985812e402540" target='_blank'>
              Machine learning analysis reveals tumor heterogeneity and stromal-immune niches in breast cancer
              </a>
            </td>
          <td>
            Junjie Kuang, Guofang Zhong, Linfeng Zhao, Xia Yuan, Yundong Zhou, Jun Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Purpose Microsatellite instability-high colorectal cancer is characterized by hypermutated genomes and high neoantigen loads, yet a significant proportion of patients exhibit resistance to immune checkpoint blockade. This study aims to investigate tumor cell functional heterogeneity and its role in immune evasion. Patients and Methods We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA sequencing from microsatellite instability-high colorectal cancer patients. After quality control, normalization, and clustering, malignant epithelial subpopulations were identified through copy number variation analysis and non-negative matrix factorization. Functional characterization employed gene set enrichment analysis. Spatial transcriptomics clarified immune cell and tumor subpopulation localization, and survival analyses assessed prognostic implications. Results We identified a glycolysis-enriched tumor subpopulation (MP2) that co-localized with immunosuppressive niches marked by Treg accumulation, effector T-cell depletion, and FOLR2+ tumor-associated macrophages. MP2-high tumors were associated with immune checkpoint blockade resistance and poor prognosis. Mechanistically, MP2 cells secreted lactate, promoting Treg differentiation and macrophage polarization toward an immunosuppressive phenotype. Spatial transcriptomics revealed the precise organization of these lactate-rich, immune-excluded niches within tumors. Conclusion These findings establish tumor cell-intrinsic glycolysis as a key driver of immune evasion in microsatellite instability-high colorectal cancer and propose metabolic targeting as a strategy to overcome immune checkpoint blockade resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773cdb1598a2f7975562a7cc2f1fda279ad89676" target='_blank'>
              Glycolysis-Driven Immune Evasion in Microsatellite Instability-High Colorectal Cancer: An Integrated Single-Cell and Spatial Transcriptomics Study
              </a>
            </td>
          <td>
            Chenchen Li, Peicong Cai, Hengda Zeng, Jianxia Li, Huabin Hu, Jianwei Zhang, Zehua Wu, Ge Qin, Yanhong Deng
          </td>
          <td>2025-09-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is highly heterogeneous, with five-year survival rates dropping from $\sim$90% in localized disease to $\sim$15% with distant metastases. Disease progression is shaped not only by tumor-intrinsic alterations but also by the reorganization of the tumor microenvironment (TME). Metabolic, compositional, and spatial changes contribute to this progression, but considered individually they lack context and often fail as therapeutic targets. Understanding their coordination could reveal processes to alter the disease course. Here, we combined multiplexed ion beam imaging (MIBI) with machine learning to profile metabolic, functional and spatial states of 522 colorectal lesions with single-cell resolution. We observed recurrent stage-specific remodeling marked by a lymphoid-to-myeloid shift, stromal-cancer cooperation, and malignant metabolic shifts. Spatial organization of epithelial, stromal, and immune compartments provided stronger stratification of disease stage than tumor-intrinsic changes or bulk immune infiltration alone. To systematically model these coordinated changes, we condensed multimodal features into 10 latent factors of TME organization. These factors tracked disease progression, were conserved across cohorts, and revealed frequent multicellular metabolic niches and distinct, non-exclusive TME trajectories. Our framework MuVIcell exposes the elements that together drive CRC progression by grouping co-occurring changes across cell types and feature classes into coordinated multicellular programs. This creates a rational basis to therapeutically target TME reorganization. Importantly, the framework is scalable and flexible, offering a resource for studying multicellular organization in other solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884e760c0de1e86aa10844e17508931cd572069b" target='_blank'>
              Robust multicellular programs dissect the complex tumor microenvironment and track disease progression in colorectal adenocarcinomas
              </a>
            </td>
          <td>
            Loan Vulliard, Teresa Glauner, Sven Truxa, Miray Cetin, Yu-Le Wu, R. Simon, Laura Behm, J. Tanevski, Julio Saez-Rodriguez, G. Sauter, Felix J. Hartmann
          </td>
          <td>2025-10-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Background Metastatic renal cell carcinoma (mRCC) still lacks tissue-based biomarkers that predict benefit from first-line immune checkpoint inhibitor (ICI) regimens. High-resolution spatial transcriptomics can dissect the tumor microenvironment (TME) and reveal prognostic niches invisible to bulk assays, offering a path toward precision immunotherapy. Methods Formalin-fixed, paraffin-embedded samples from 12 therapy-naïve patients with mRCC treated with ICI-based combinations were profiled with 10x Genomics Visium and NanoString COSMx. Six patients contributed matched metastatic lesions. Cell-type deconvolution used robust cell type decomposition with a public single-cell RNA-seq reference. Clinical outcomes (mean follow-up 16.75 months) were correlated with spatial features. Validation employed Phase 2/3 JAVELIN Renal 101 as well as Checkmate 010/025 transcriptomic data. Results Spatial transcriptomics uncovered pronounced divergences between primary tumors and metastases in cellular composition and microarchitectural organization. Canonical RCC molecular subtypes displayed marked intrapatient and interpatient spatial heterogeneity, underscoring limitations of bulk classifications. Five recurrent immune niches emerged across samples. One niche—rich in macrophages and CD8+ T cells and defined by chemokine signaling—was closely associated with objective response in metastatic lesions yet absent or infrequent in matched primaries. A gene signature reflecting this niche stratified responders within the immuno-oncology (IO) -treated arms of JAVELIN 101 and CheckMate 010/025 trials. On the other hand, signatures derived from angiogenic tumor cell-rich niches defined responders within the comparator arms (tyrosine kinase inhibitor/everolimus) in these trials. Conclusion Metastasis-specific immune niches, rather than primary-tumor characteristics, appear decisive for ICI efficacy in mRCC. Spatial transcriptomic profiling can identify these clinically relevant microenvironments and generate transferable gene signatures, supporting biomarker-driven patient selection and trial design. Integrating spatial TME analysis into future studies may accelerate personalized immunotherapy strategies for mRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bedf8d3b886332325de79302aa00547b1e48f37" target='_blank'>
              Spatial transcriptomic profiling of metastatic renal cell carcinoma identifies chemokine-driven macrophage and CD8+ T-cell interactions predictive of immunotherapy response
              </a>
            </td>
          <td>
            Charis Kalogirou, M. Krebs, Andreas S Kunz, Oliver Hahn, Hubert Kübler, Marcel Schwinger, Arndt Hartmann, Astrid Schmieder, Markus Eckstein
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Current colorectal cancer mouse models either lack colon specificity, limiting progression towards more advanced disease, or preclude evaluation of resident stem cells as cancer origins. Here we report the identification of NOX1 and NPY1R as cell-surface markers enriched in LGR5+ stem cells predominantly within the caecum and exclusively within the middle and distal colorectum, respectively. Selective dysregulation of Wnt signalling in NOX1+ or NPY1R+ stem cells using CreERT2 mouse lines drives colon cancer initiation, predominantly within the caecum and rectum respectively, establishing these stem cell populations as important sources of colon cancer. Selective conditional activation of Wnt signalling and oncogenic Kras in combination with loss of TRP53 in these stem cell compartments resulted in the development of advanced, invasive cancers. This study establishes CreERT2 drivers as valuable tools for studying stem cell contributions to colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77d17a800b951a14fec481a0422bbf1125414c43" target='_blank'>
              NOX1 and NPY1R mark regional colon stem cell populations that serve as cancer origins in vivo
              </a>
            </td>
          <td>
            Maxime Gasnier, Tanysha Chi-Ying Chen, Swathi Yada, Sowmya Sagiraju, Yusuke Yoshikawa, Stefano Perna, H. Lim, Bernett Lee, Nick Barker
          </td>
          <td>2025-09-02</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Repeated oncogenic mutations and polyclonal proliferation are evident in cancers. However, little is known about the polyclonal principles governing the systemic cancerous lineage during immunotherapy. Here, we examine a unique autopsy case of metastatic urothelial carcinoma that exhibits different treatment responses to anti-PD-1 therapy at each tumor site. By performing in-depth analyses of different multiregional bulk tumor masses, we reveal that subsets of subclones acquire potential driver mutations under treatment selection pressure. Spatial transcriptomics analysis reveals that subclones resistant to immunotherapy form distinct immunosuppressive environments consistent with their habitats. Furthermore, different cancer hallmarks are identified in each of the subclones that expand under immunotherapy at single-cell level; for example, one subclone is more proliferative, and another is more stem-cell-like. In summary, this study provides an overall picture of the polyclonal competition and changes in the immune microenvironment that are related to resistance to immunotherapy in patients with malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f357f9c7e57b9e95e8ee48a6378f3d690f3275bd" target='_blank'>
              Clonal diversity shapes the tumour microenvironment leading to distinct immunotherapy responses in metastatic urothelial carcinoma
              </a>
            </td>
          <td>
            Takashi Kamatani, K. Umeda, T. Iwasawa, Fuyuki Miya, K. Matsumoto, S. Mikami, Kensuke Hara, Masayuki Shimoda, Yutaka Suzuki, Jo Nishino, Mamoru Kato, Kazuhiro Kakimi, Nobuyuki Tanaka, M. Oya, T. Tsunoda
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease with diverse outcomes, and intra-tumoral heterogeneity plays a significant role in both diagnosis and treatment. Despite its importance, the spatial distribution of intra-tumoral heterogeneity is not fully elucidated. Spatial transcriptomics has emerged as a promising tool to study the molecular mechanisms behind many diseases. It offers accurate measurements of RNA abundance, providing powerful tools to correlate the morphologies of cellular neighborhoods with localized gene expression patterns. However, the spot-based spatial transcriptomic tools, including the most widely used platform, Visium, do not achieve single-cell resolution readouts, which hinders data interpretability. In this study, we present a computational pathology image analysis pipeline (i.e., computational tissue annotation, CTA) that utilizes machine learning algorithms to accurately map tumor, stroma, and immune compartments within Visium-assayed tumor sections. Using a cohort of 23 breast tumor sections from four patients, we demonstrate that CTA can provide high-resolution annotations on the hematoxylin-and-eosin-stained images alongside the paired sequencing data, support the evaluation of deconvolution methods, deepen insights into intra-tumoral heterogeneity by increasing data analysis resolution, assist with spatially resolved intrinsic subtyping, and enhance the visualization of lymphocyte clones at single-cell resolution. The proposed pipeline provides valuable insights into the complex spatial architecture of breast cancer, contributing to more personalized diagnostics and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5fa100fe1b095f296661cac47143cc93bae15b" target='_blank'>
              Computational pathology annotation enhances the resolution and interpretation of breast cancer spatial transcriptomics data
              </a>
            </td>
          <td>
            Tianyi Li, Qiao Yang, B. Ács, E. Sifakis, Hosein Toosi, Camilla Engblom, K. Thrane, Qirong Lin, Jeff E Mold, Wenwen Sun, Ceren Boyaci, Sanna Steen, Jonas Frisén, Jens Lagergren, J. Lundeberg, Xinsong Chen, Johan Hartman
          </td>
          <td>2025-09-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is characterized by exceedingly high rates of metastatic progression, with the liver representing the most common site of distant spread. Here, we established a platform for multisite immune profiling of human PDAC encompassing the tumor, peripheral circulation, and premetastatic liver, to more comprehensively study how various immune subsets might contribute to patient outcomes. Methods Tumor, liver, and blood samples were obtained from patients undergoing resection for non-metastatic PDAC. Derived immune cells underwent paired single-cell RNA and TCR sequencing. Immune composition, cell-type functional profiles, and T cell clonal expansion patterns were evaluated across tissue sites. Results In total, 106,539 immune cells were sequenced, of which 85,748 met criteria for analysis. We identified 32 cell populations, of which seven demonstrated significant enrichment within a particular tissue, highlighting that this workflow possesses the granularity needed for identifying potential future biomarkers. Functional profiling revealed tissue-specific differences in cell phenotypes. This included terminally differentiated exhausted CD8 T cells within the tumor, highly active Tregs within the premetastatic liver and tumor, and M1 versus M2 polarization of liver and tumor macrophage populations, respectively. Within the tumor, expanded Treg clones were uniquely abundant, and while expanded clones could be tracked to the blood and premetastatic liver, many of these mapped back to known viral antigens. Leveraging previously validated gene sets, we show how these can be applied to predict the tumor reactivity of intratumoral T cells using transcriptional signatures. We demonstrated a high degree of concordance between multiple independent signatures and tracked high-priority TCRs within the blood and liver. Conclusion This study demonstrates the feasibility of a platform, which has already been implemented into ongoing clinical protocols, for immune profiling of human PDAC across the sites most relevant to metastatic progression. Future applications of this work can monitor immune populations throughout metastatic progression to build a temporal database of immune phenotypes and track association with clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04146-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9fc3615c15a691adb382d776e1bcc8d09f65d7" target='_blank'>
              A platform for multisite immune profiling of premetastatic pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Elishama N. Kanu, Ashley Fletcher, Jiayin Bao, Ethan S Agritelley, Julia Button, A.M. Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, M. Lidsky, S. Zani, D. Blazer, Peter J Allen, Zhicheng Ji, Frank J. Lowery, Sri Krishna, Nicholas D. Klemen, Daniel P. Nussbaum, Erika J. Crosby
          </td>
          <td>2025-08-23</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9fa61bcf35b5d6c234bc5ce442e119522c741a" target='_blank'>
              Basal cell of origin resolves neuroendocrine-tuft lineage plasticity in cancer.
              </a>
            </td>
          <td>
            A. Ireland, Daniel A Xie, Sarah B Hawgood, Margaret W Barbier, Lisa Y. Zuo, Benjamin E Hanna, Scarlett Lucas-Randolph, Darren R Tyson, Benjamin L. Witt, R. Govindan, Afshin Dowlati, Justin C. Moser, Anish Thomas, Sonam Puri, C. Rudin, Joseph M Chan, Andrew Elliott, T. Oliver
          </td>
          <td>2025-09-17</td>
          <td>Nature</td>
          <td>2</td>
          <td>36</td>
        </tr>

        <tr id="


 Tumor-infiltrating lymphocyte (TIL) therapy has demonstrated efficacy in melanoma and is emerging as a promising modality for cell-based immunotherapy. However, its application in immunologically “cold” tumors like glioblastoma is limited due to sparse T cell infiltration, antigenic heterogeneity, and a suppressive tumor microenvironment. To identify genomic and spatial determinants of TIL expandability, we performed integrated, multimodal profiling of high-grade gliomas to identify molecular and spatial features associated with successful TIL expansion.



 We conducted comprehensive profiling of 18 TIL products using spectral flow cytometry and T Cell Receptor sequencing (TCRseq). Additionally, 4 tumors yielding ≥10⁸ viable TILs per 0.5 g tumor within two weeks of IL2 Rapid Expansion Protocol cultures, classified as TIL⁺, were analyzed and compared to 4 non-yielding tumors, classified as TIL⁻. Expanded TILs were subjected to spectral flow cytometry and TCRseq. These TIL⁺ and TIL⁻ tumors underwent single-cell RNA-seq (scRNAseq), in situ Xenium spatial transcriptomics using a customized 480-gene panel, and CODEX spatial proteomics.



 The TIL⁺ tumors yielded 0.5-3.2 × 10⁹ cells with 100-1,000-fold expansion. Flow cytometry showed CD4⁺ and CD8⁺ effector memory phenotypes with minimal exhaustion; Tregs were <10% and γδ T cells were detected. TCRseq revealed high frequency αβ clones and diverse γδ repertoires. Expanded γδ T cells had higher D50 and diversity indices, suggesting a broad, non-dominant repertoire. TIL⁺ tumors exhibited elevated IL7R and ACSS3 expression, enriched Th1/pro-inflammatory gene signatures, and lower Treg-associated transcripts. Spatial analysis showed perivascular immune clusters in TIL⁺ tumors, enriched for CD8⁺ T cells and antigen-presenting cells, with local upregulation of IL7R, MHC II, and CXCL13. These areas were associated with vascular markers (e.g., VCAM1) and T cell retention signatures. In contrast, TIL⁻ tumors displayed immune exclusion, neuronal lineage enrichment, and higher expression of immune suppressive genes like TOX and FERMT1. Tumor-connected TAMs expressing SOX2, NES, and AQP4 were more abundant in TIL⁻ tumors, suggesting a barrier to infiltration or active immune suppression.



 TIL⁺ tumors exhibit distinct molecular and spatial features associated with successful TIL expansion, including IL7R-driven T cell responsiveness, mesenchymal metabolic programs, and structured perivascular immune niches. These findings define biomarkers of therapeutic TIL production and inform strategies to stratify patients for TIL⁻ based therapies in glioblastoma and other immune-excluded tumors. Incorporating these biomarkers into patient selection and workflows could improve clinical trial design, product consistency, and therapeutic response rates.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98b47fb61580f74a05d0a598bf26243ed8f703f3" target='_blank'>
              KS03.4.A SPATIAL MULTI-OMIC PROFILING REVEALS DETERMINANTS OF TIL EXPANDABILITY IN HIGH-GRADE GLIOMAS
              </a>
            </td>
          <td>
            K. Hotchkiss, K. Zhang, A. Corcoran, E. Owens, P. Noldner, C. Railton, K. Van Batavia, Y. Zhou, J. Jepson, K. Singh, R. E. McLendon, K. Batich, A. Patel, K. Ayasoufi, M. Brown, E. Calabrese, J. Xie, J. Conejo-Garcia, B. Shaz, J. Hickey, M. Khasraw
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="High-grade serous carcinoma (HGSC) is the most common ovarian cancer subtype, typically diagnosed at late stages with poor prognosis. Understanding early molecular events driving HGSC progression is crucial for timely detection and development of effective treatment strategies. We performed and integrated spatial cell-type resolved proteomics and paired transcriptomics across 25 women with precursor lesions of the fallopian tube and/or HGSC. Epithelial cell signatures revealed early activation of SUMOylation machinery, increased ATR and Wnt signaling, and enhanced MHC-I antigen presentation along the disease trajectory. The stroma exhibited extracellular matrix remodeling and interferon-mediated inflammation. Serous tubal intraepithelial carcinomas (STICs) in cancer patients contained a pro-coagulative signature and reduced APOA1/2 compared to STICs in individuals without cancer. We functionally established important roles of epithelial-derived TRIP13 and SUMOylation, and cancer-associated fibroblast-derived SULF1 and BGN in HGSC progression. These findings provide unique molecular insights into HGSC pathogenesis and identify potential new therapeutic targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44488614a89cae7d4f29ffc136362eb7dd32e0db" target='_blank'>
              Integration of cell-type resolved spatial proteomics and transcriptomics reveals novel mechanisms in early ovarian cancer
              </a>
            </td>
          <td>
            Andreas Metousis, Hilary A Kenny, Aasa Shimizu, L. Schweizer, S. Ben-Moshe, A. Bilecz, Rahul Krishnan, Jingwen Zhang, Isabel Alcazar, Lucy Kelliher, Mallika Ravi, Tejas Samantaray, Sabrina Richter, Yan Li, Jiying Wang, Sophia Steigerwald, F. Theis, Florian A. Rosenberger, T. Nordmann, S. Yamada, Ricardo R. Lastra, M. Mann, E. Lengyel
          </td>
          <td>2025-08-28</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="It has become evident from decades of clinical trials that multimodal therapeutic approaches with focus on cell intrinsic and microenvironmental cues are needed to improve understanding and treat the rare, inoperable, and ultimately fatal diffuse intrinsic pontine glioma (DIPG), now categorized as a diffuse midline glioma. In this study we report the development and characterization of an in vitro system utilizing 3D Tumor Tissue Analogs (TTA), designed to replicate the intricate DIPG microenvironment. The innate ability of fluorescently labeled human brain endothelial cells, microglia, and patient-derived DIPG cell lines to self-assemble has been exploited to generate multicellular 3D TTAs that mimic tissue-like microstructures, enabling an in- depth exploration of the spatio-temporal dynamics between neoplastic and stromal cells. The 3D-TTA model recapitulates clinical patterns of DIPG growth, evidenced by resistance to chemotherapy, HDAC and proteasome inhibitors, as well as sensitization to the antibody-activated innate immune microenvironment including complement proteins and surrounding microglia. Multimodal fluorescence imaging platforms integrated with high-throughput omics revealed that alterations in tumor cell motility and growth in the 3D-TTA model compared to tumor cell only spheroids correlated with specific transcriptomic and proteomic changes. STAT3, ITGA5, LGALS1, SOD2, MVP, and CLIC1, associated with microenvironment signaling, DNA replication, and immune regulation, were identified as potential novel targets in the 3D model. The results indicate that the 3D TTA platform developed here represents a powerful tool for preclinical studies, paving the way for identification/validation of tissue specific biomarkers and novel drug targets, thus advancing disease management strategies for DIPG in children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b07b858c6fd1cb8139cba6229ad20ebbdfab9d" target='_blank'>
              Multicellular tumor-stromal interactions recapitulate aspects of therapeutic response and human oncogenic signaling in a 3D disease model for H3K27M-altered DIPG
              </a>
            </td>
          <td>
            Meenakshi Upreti, Astgik Petrosyan, M. Thornton, A. Hovsepyan, G. E. Fernandez, David S. Koos, S. Byrum, Samuel G Mackintosh, Jacob K Al-Husseini, Tania Porras, Joseph Ha, A. Tackett, Miqin Zhang, M. Johal, Anat Erdreich-Epstein, Susan Durham, Mark D Krieger, Ashley S Margol, Brendan H. Grubbs, T. Chambers, S. Asgharzadeh, Rex A. Moats, Peter A Chiarelli
          </td>
          <td>2025-09-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Abstract The tumor microenvironment (TME) is a critical determinant of tumor progression and therapeutic response in gliomas. While pediatric gliomas have historically been treated using strategies derived from the management of adult gliomas, emerging evidence reveals that pediatric gliomas possess a unique TME. The pediatric TME is distinct, characterized not only by differences in cellular composition but a lower mutational burden, diminished neoantigen presentation, and heightened immunosuppressive activity. The unique immune landscape, developmental trajectories, and immune escape mechanisms in the pediatric TME create barriers to effective therapy. Recent studies show promising results in novel and advanced therapeutic strategies, highlighting the potential for innovative immunotherapeutic approaches. Advances in methodologies for modeling the TME, including computational approaches and animal-based models, provide new insights into pediatric glioma biology. Utilization of computational models may provide opportunities to predict tumor response to specific therapies and tailor immunotherapy regimes to individuals, allowing for personalized care. Leveraging the unique features of the pediatric TME offers an opportunity to overcome current barriers to immunotherapy and develop more effective, age- and tumor-specific treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d12c4d7c16cb4f880257014b09a937ace0732f5a" target='_blank'>
              Targeting the tumor microenvironment in pediatric gliomas: Advances and future directions in immunotherapy
              </a>
            </td>
          <td>
            Cheyenne Ahamed, Lam Nguyen, Cayley S Brock, Ayla Farzamnia, Pierrick Millet, Keisaku Sato, Kevin K Kumar
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT The immune landscape of healthy prostate and its alterations during prostate cancer (PCa) progression remain poorly characterized. Using scRNA-sequencing and multiparametric flow-cytometry analysis, we comprehensively characterized immune cells in wild-type and PTEN(i)pe−/− mouse prostates, a model that closely recapitulates human PCa. PCa in PTEN(i)pe−/− is marked by the recruitment of tumor-associated neutrophils (TANs), which represent the dominant immune cell population and resolved into eight distinct states, Trem2+ tumor-associated-macrophages (TAMs), and exhausted CD8+ T cells. Trem2+ TAMs differ from the three main resident macrophage populations in the healthy prostate, exhibiting a strong metabolic and immunosuppressive signature, likely driven by the MIF/HIF1A-signaling axis. This study provides the first detailed characterization of immune cells in the healthy mouse prostate and reveals changes in the immune landscape associated with prostate cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb0c0592aa197777e78a6d1020c20a4157e6fc" target='_blank'>
              Characterization of the immune landscape in healthy mouse prostate and during prostate cancer progression
              </a>
            </td>
          <td>
            Despoina Pervizou, Joanna De Chiara, L. Spinelli, Maia Nestor-Martin, Lionel Chasson, Kateryna Len-Tayon, Darya Yanushko, Frédéric Fiore, Marc Bajénoff, Bernard Malissen, Daniel Metzger, Gilles Laverny, S. Henri
          </td>
          <td>2025-09-26</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract The ability to interrogate changes within the tumor microenvironment (TME) before, during and following therapeutic intervention could yield important understanding of treatment response and causes for disease progression. Yet, the role of investigational tissue analysis faces key challenges in the clinical setting and the value of integrating longitudinal biopsies with emerging multimodal molecular analyses (“Multi-omics”) remains to be defined. In this study, we conducted a multicenter phase 2 clinical trial examining the effect of a novel cytotoxic T-lymphocyte-associated protein 4/programmed death-ligand 1 bispecific antibody in combination with a dual-epitope blocking anti-human epidermal growth factor receptor 2 antibody in treatment-resistant metastatic breast cancer. We performed longitudinal sampling of patient tumor tissues before and following treatment. Single-cell RNA and T cell receptor sequencing from 334,183 cells from site-matched tumors reveals significant temporal shift of various immune cell populations and phenotypes within the TME associated with treatment responses. Conversely, regulatory T cells were activated while effector T cells, natural killer cells, and dendritic cells were significantly depleted in non-responding tumors. Taken together, these results support that longitudinal analysis of TME to generate multiomics data that can lead to rich insight into disease process and to provide clinical value in evaluating treatment responses. Trial registration number NCT04521179.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97d0a38c945abea56c7fcf08591e230c989d914" target='_blank'>
              Longitudinal tissue analysis reveals microenvironmental changes correlate with combined immunotherapy and targeted therapy response in metastatic breast cancer
              </a>
            </td>
          <td>
            , Jien-Ren Wang,, Hengyu Li, Zhijun Liu, Z. Tian, Xinying Lv, Jianjian Peng, Chuangui Song, Jieqiong Liu
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Heterogeneity within the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment (TME), across tumor locations, and between patients largely contributes to poor treatment response and PDAC’s status as the fourth leading cause of cancer death in the United States. In order to identify suitable targets for innovative therapeutic interventions, it is necessary to understand and mechanistically test TME constituents, their infiltration, and their interaction dynamics in the context of primary and metastatic tumor locations using pre-clinical modeling. Therefore, we designed a novel 46-marker multiplex immunofluorescent staining panel for murine formalin-fixed paraffin-embedded (FFPE) tissues. This panel enables the spatial quantification of immune cell populations and activation/exhaustion states, epithelial-to-mesenchymal phenotypes, cancer-associated fibroblast heterogeneity, hematologic and lymphatic vessel infiltration, neuronal infiltration, and extracellular matrix deposition in the TME. We quantified the composition and spatial infiltration of the TME on matched murine primary and liver tumor samples from tandem orthotopic and hemi-spleen transplants of three distinct primary murine PDAC cell lines derived from the KPC model (KrasG12D; TP53R172H; PDX1-Cre; RosaEYFP). The resulting primary tumors from each cell line showed inter-tumoral heterogeneity as their distinct levels of immune infiltration had varying clustering patterns. Spatial analysis further revealed heterogeneous cancer cell MHC-I expression, immune cell exhaustion, and myeloid and lymphoid populations correlating with cell line. However, all liver nodules were encased by a barrier of immune cells separating the tumor from normal liver parenchyma irrespective of cell line. This observation reflects our recent spatial transcriptomic analysis of patient-derived matched liver and primary tumors, supporting the use of these pre-clinical models for future mechanistic investigations. Altogether, these data highlight the complexity of PDAC TMEs and the site-specific differences between primary and metastatic tumor composition.



 Christina R. Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert Welner, Julienne L. Carstens. High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B110.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335836491342c8b5df5a982f9e0790e87ffd1d16" target='_blank'>
              Abstract B110: High-dimensional spatial analysis of murine PDAC microenvironment emphasizes primary and metastatic immune heterogeneity
              </a>
            </td>
          <td>
            Christina R Larson, Satwik Acharyya, Jace Baines, Ayushi Mandloi, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296f18d57dd6ad5d3ff3f75155cdc24fcd7506bf" target='_blank'>
              Cross-species analysis identifies genotype-driven vulnerabilities in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Angeliki Karatza, Maayan Pour, Jaylen M Powell, Yuan Hao, Dimitris Nasias, Yi Jer Tan, Alfonso Lopez, M. Ciampricotti, Chengwei Peng, Álvaro Quintanal Villalonga, Yeuan Ting Lee, Ruveyda Ayasun, Muhammad Elnaggar, Priyanka Sahu, Igor Dolgalev, Hai Hu, Luis Chiriboga, Cynthia Loomis, Daniel Andrussier, Magdalena A Ploszaj, A. Tsirigos, Peter Meyn, Fred P. Davis, Nicolas Stransky, Gromoslaw A. Smolen, John T. Poirier, Jonathan So, Mark Philips, Harvey I. Pass, Dafna Bar Sagi, Itai Yanai, Kwok-Kin Wong
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly heterogeneous malignancy, characterized by low tumor cellularity, a dense stromal response, and intricate cellular and molecular interactions within the tumor microenvironment (TME). Although bulk omics technologies have enhanced our understanding of the molecular landscape of PDAC, the specific contributions of non-malignant immune and stromal components to tumor progression and therapeutic response remain poorly understood. Methods: We explored genome-wide DNA methylation and transcriptomic data from the Cancer Genome Atlas Pancreatic Adenocarcinoma cohort (TCGA-PAAD) to profile the immune composition of the TME and uncover gene co-expression networks. Bioinformatic analyses included DNA methylation profiling followed by hierarchical deconvolution, epigenetic age estimation, and a weighted gene co-expression network analysis (WGCNA). Results: The unsupervised clustering of methylation profiles identified two major tumor groups, with Group 2 (n = 98) exhibiting higher tumor purity and a greater frequency of KRAS mutations compared to Group 1 (n = 87) (p < 0.0001). The hierarchical deconvolution of DNA methylation data revealed three distinct TME subtypes, termed hypo-inflamed (immune-deserted), myeloid-enriched, and lymphoid-enriched (notably T-cell predominant). These immune clusters were further supported by co-expression modules identified via WGCNA, which were enriched in immune regulatory and signaling pathways. Conclusions: This integrative epigenomic–transcriptomic analysis offers a robust framework for stratifying PDAC patients based on the tumor immune microenvironment (TIME), providing valuable insights for biomarker discovery and the development of precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec3422e79e1fa464a39c9ceade9afd8d95e43c98" target='_blank'>
              Deciphering the Heterogeneity of Pancreatic Cancer: DNA Methylation-Based Cell Type Deconvolution Unveils Distinct Subgroups and Immune Landscapes
              </a>
            </td>
          <td>
            Barbara Mitsuyasu Barbosa, Alexandre Todorovic Fabro, Roberto da Silva Gomes, C. Rainho
          </td>
          <td>2025-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Intratumoral heterogeneity in pancreatic ductal adenocarcinoma (PDAC) is characterized by the coexistence of classical epithelial, basal, and mesenchymal cancer cell states. Targeting cancer cell plasticity—the ability of tumor cells to adapt to extrinsic pressures via non-genetic mechanisms—has emerged as a promising therapeutic strategy. In parallel, the immunosuppressive tumor microenvironment remains a major barrier to effective treatment in PDAC. Yet, whether specific cell states possess distinct functional properties that drive key tumor behaviors—such as phenotypic plasticity or immune evasion—remains poorly understood. Using genetically engineered mouse models driven by oncogenic Kras G12D and loss of Trp53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We find that the KPC model harbors a basal state, which harbors transcriptional signature that aligns with the "basal" signature in human PDAC and is linked to poor prognosis and chemoresistance. The basal state co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. To functionally interrogate the basal state in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the basal state has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the basal state is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the basal state over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we find that the basal state is a key driver of therapeutic resistance: rather than being eliminated by treatment, it persists through phenotypic adaptation in response to both KRAS-targeted therapies and cytotoxic chemotherapy. Co-elimination of the basal state alongside these therapies leads to deeper and more durable responses, underscoring its functional role in tumor persistence under therapy. In parallel, we observe that the basal state plays a central role in enforcing immunosuppression in PDAC tumors. Basal state ablation rapidly reshapes the tumor immune microenvironment, and we observe loss of cancer-associated fibroblasts, polarization of PDAC-resident macrophages toward a pro-inflammatory phenotype, and increased infiltration of T and NK cells. Cytokine profiling of basal-ablated tumors revealed a dramatic reduction in multiple immune-modulatory and immunosuppressive cytokines. Taken together, our data indicate that PDAC harbors a striking dependency on the basal state for both therapeutic resistance and immune evasion, and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.



 Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C. Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Zeynep C. Tarcan, Olca Basturk, Rona Yaeger, James G. Christensen, Yan Yan, Elisa de Stanchina, Dana Pe'er, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B108.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6879ea5062e58b2315581060e5992acf02efdbb" target='_blank'>
              Abstract B108: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Z. Tarcan, O. Basturk, R. Yaeger, James G Christensen, Yan Yan, E. de Stanchina, D. Pe’er, Tuomas Tammela
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is classified into epithelial and mesenchymal subtypes based on transcriptional profiling, and these subtypes are associated with varied patient outcomes. Differences in the tumor microenvironment (TME), particularly in cancer-associated fibroblasts (CAFs), may contribute to this variation. scRNA-seq studies have observed heterogeneous CAF populations in PDAC, although the relationship between tumor subtype and CAF phenotype is not well understood. To investigate this relationship, we utilized epithelial and mesenchymal PDAC cell lines in a xenograft model to examine whether these subtypes influence CAF phenotypes.



 Xenograft models were generated by subcutaneously injecting BALB/c nude mice with PDAC cell lines of either the epithelial subtype (BxPC-3) or the mesenchymal subtype (MIA PaCa-2). Tumors were then harvested and dissociated for scRNA-seq to investigate the TME. Human tumor (graft) and murine stromal (host) cells were computationally separated using the Xenocell tool. Subsequent analyses focused on fibroblasts within the TME, which were subclustered to assess CAF heterogeneity between PDAC subtypes. To characterize transcriptional programs and identify CAF features, we performed high-dimensional weighted gene co-expression network analysis (hdWGCNA) and analyzed differentially expressed genes. Finally, spatial transcriptomics was conducted on tumor sections from BxPC-3 and MIA PaCa-2 xenografts to validate CAF clusters observed in scRNA-seq.



 Distinct CAF clusters were identified in the xenograft tumors, including LRRC15+ CAFs, RGS5+ CAFs, and inflammatory CAFs (iCAFs). LRRC15+ CAFs were predominant in tumors derived from the epithelial subtype (BxPC-3), while RGS5+ CAFs were enriched in tumors from the mesenchymal subtype (MIA PaCa-2). LRRC15+ CAFs exhibited a myofibroblastic CAF (myCAF) phenotype characterized by extracellular matrix production, with high expression of Lrrc15, Postn, and Fbln1. RGS5+ CAFs expressed canonical pericyte markers such as Rgs5 and Cox4i2, along with CAF-associated genes including Acta2. These CAF clusters also showed distinct expression patterns of collagens and secreted factors. In particular, Col11a1, Col12a1, and Wnt5a were upregulated in LRRC15+ CAFs, while Col4a1, Col10a1, and Tgfb1 were upregulated in RGS5+ CAFs. Spatial transcriptomics confirmed that gene modules associated with Lrrc15 and Rgs5, inferred from hdWGCNA, were differentially expressed between Acta2-positive stromal regions of epithelial and mesenchymal PDACs.



 These results suggest that epithelial and mesenchymal PDAC subtypes directly shape distinct CAF populations. Notably, the enrichment of RGS5+ CAFs in mesenchymal tumors may underlie their metastatic potential and poor clinical outcomes.



 Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, Jung Hyun Jo. Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B105.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020656bad62fdcf52b16552c5f7369030a0e1205" target='_blank'>
              Abstract B105: Distinct cancer-associated fibroblast populations induced by epithelial and mesenchymal PDAC subtypes in a xenograft model
              </a>
            </td>
          <td>
            Kyu Min Lim, Soo Been Park, Jung-hwan Park, Yoo-Keung Tae, Hee Won Choi, In Rae Cho, Hee Seung Lee, J. Jo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Biliary tract cancer (BTC) is a highly heterogeneous and aggressive gastrointestinal malignancy, marked by a high mortality rate and limited treatment efficacy. The primary contributing factors include the absence of reliable early detection methods, the anatomical intricacy of the biliary system, the inherently aggressive tumor biology, and the restricted effectiveness of systemic therapies. A profound understanding of molecular characteristics and clinically relevant emerging biomarkers is essential for advancing BTC treatment strategies. Recent developments in single-cell multi-omics technologies have enabled the analysis of genetic, transcriptomic, proteomic, and metabolomic data at the single-cell resolution, thereby uncovering the heterogeneity and complexity of tumor biology. These techniques provide critical insights into the diversity of immune cell populations within the tumor microenvironment (TME) and offer novel perspectives on tumor progression and potential therapeutic interventions. While single-cell technologies have significantly advanced the study of solid tumors, their application in BTC remains nascent, with a paucity of comprehensive reviews. This review systematically integrates single-cell genomics, transcriptomics, and epigenomics data to construct a cross-omics molecular atlas of BTC. It highlights the utility of single-cell multi-omics technologies in elucidating tumor heterogeneity, microenvironment remodeling, and clonal evolution in biliary tumors, while thoroughly analyzing their implications for clinical outcomes. Furthermore, this review explores personalized treatment strategies informed by single-cell technologies and underscores the significance of these technologies as indispensable tools for unraveling the complexity of BTC and fostering mechanism-based therapeutic innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c48afe626e59e1df497d34aa627d14168d0c25d2" target='_blank'>
              Single-cell multi-omics in biliary tract cancers: decoding heterogeneity, microenvironment, and treatment strategies
              </a>
            </td>
          <td>
            Nannan Tang, Jiatong Li, Ao Gu, Meng‐Yao Li, Yingbin Liu
          </td>
          <td>2025-10-15</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is composed of a diverse and dynamic spectrum of cell types, cellular activities, and cell–cell interactions (CCI). Understanding the complex CCI within the TME is critical for advancing cancer treatment strategies, including modulating or predicting drug responses. Recent advances in omics technologies, including spatial transcriptomics and proteomics, have allowed improved mapping of CCI within the TME. The integration of omics insights from different platforms may facilitate the identification of novel biomarkers and therapeutic targets. This review discusses the latest computational methods for inferring CCIs from different omics data and various CCI and drug databases, emphasizing their applications in predicting drug responses. We also comprehensively summarize recent patents, clinical trials, and publications that leverage these cellular interactions to refine cancer treatment approaches. We believe that the integration of these CCI-focused technologies can improve personalized therapy for cancer patients, thereby optimizing treatment outcomes and paving the way for next-generation precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb307e16bbb2fc4780518da53d8301520562b427" target='_blank'>
              Cell–cell interactions as predictive and prognostic markers for drug responses in cancer
              </a>
            </td>
          <td>
            Xuehan Lu, Xiao Tan, Eun Ju Kim, Xinnan Jin, Meg L. Donovan, Jazmina L. Gonzalez Cruz, Zherui Xiong, Maria Reyes Becerra de los Reyes Becerra Perez, Jialei Gong, J. Monkman, Divya Agrawal, A. Kulasinghe, Quan Nguyen, Z. Tuong
          </td>
          <td>2025-10-08</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Despite therapeutic advances, metastatic melanoma, and particularly brain metastasis (MBM), remains a lethal burden for patients. Existing single-cell studies offer a more detailed view of melanoma and its microenvironment, which is crucial to improve diagnosis and treatment. Results We here present a computational reanalysis of single-nucleus data comparing 15 MBM and 10 extracranial melanoma metastases (ECM), considering recent best practice recommendations. We used cell type-specific pseudobulking and omit imputation during patient integration to gain complementary insights. Interestingly, our analysis revealed high homogeneity in tumor cell expression profiles within and between MBM and ECM. However, MBM displayed even higher homogeneity but a more flexible energy metabolism, suggesting a specific metastatic adaptation to the putatively more restricted brain microenvironment. While tumor cells were homogeneous, the metastasis microenvironment, especially lymphocytes and related immune-tumor interaction pathways, exhibited greater divergence between MBM and ECM. Overall, this suggests that major differences between MBM and ECM are potentially driven by variations in their microenvironment. Finally, a comparison of single-cell data to previous bulk studies, including their deconvoluted putative cell types, showed significant differences, potentially causing divergent conclusions. Conclusion Our study contributed to refine the understanding of differences between MBM and ECM, suggesting these are potentially more influenced by their local microenvironments. Future research and therapies could possibly focus on the metabolic flexibility of melanoma brain metastases and patient-specific immune pathway alterations. Supplementary Information The online version contains supplementary material available at 10.1186/s13062-025-00691-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e755d874cd5c0ea8aa714bc1a90cc86adaa7c4ba" target='_blank'>
              Single-cell transcriptome analysis suggests cells of the tumor microenvironment as a major discriminator between brain and extracranial melanoma metastases
              </a>
            </td>
          <td>
            Konrad Grützmann, Michael Seifert
          </td>
          <td>2025-09-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Gliosarcoma, a rare and aggressive variant of IDH-wildtype glioblastoma, is characterized by striking intra-tumoral heterogeneity with alternating glial and mesenchymal differentiation. The mesenchymal component is often enriched at recurrence and associated with therapy resistance, suggesting treatment-driven clonal selection. Despite its poor prognosis and distinct histology, gliosarcoma remains underexplored at the molecular level, with no systematic profiling of its cellular or spatial architecture - limiting progress toward targeted therapies.



 Single-nucleus RNA sequencing (snRNA-seq) was performed on eight FFPE gliosarcoma samples, complemented by spatial transcriptomic profiling (10X Visium) in four matched cases. Multimodal integration enabled high-resolution mapping of malignant and non-malignant compartments. To enhance spatial context extraction and to model tumor-microenvironment interactions, advanced computational approaches, including graph neural networks (GNNs) and convolutional neural networks (CNNs), are being developed.



 Integration of snRNA-seq and spatial transcriptomic data revealed a distinct population of stromal-like malignant cells enriched for mesenchymal signatures. Compared to conventional glioblastoma, gliosarcoma samples exhibited a reduced presence of neural progenitor cell (NPC-like) and oligodendrocyte progenitor cell (OPC-like) states, suggesting early dominance or selective expansion of the mesenchymal compartment. Spatial annotation of the tumor microenvironment revealed immune-rich niches populated by disease-associated microglia (DIMs), damage-associated macrophages (DAMs), and macrophages (MACs). Preliminary spatial modeling indicates potential interactions between immune population and mesenchymal-like tumor cells that may support therapy-resistant niches.



 This ongoing study integrates single-nucleus and spatial transcriptomic profiling to characterize mesenchymal transition and resistance in gliosarcoma. Early findings highlight a distinct mesenchymal-like malignant population and associated immune environments that may drive resistance. Further computational modeling aims to uncover key cellular interactions and molecular programs, that could inform novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47d9bc10df09a19173093cb2520a4af4dc46d0d5" target='_blank'>
              P14.25.B DECODING MORPHO-MOLECULAR DIVERSITY IN GLIOSARCOMA AND INSIGHTS INTO RESISTANCE USING SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            D. Savran, N. Schoebe, N. Berghaus, D. Reuss, M. Ratliff, C. Herold-Mende, S. Krieg, W. Wick, A. von Deimling, D. Heiland, F. Sahm, P. Sievers
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of breast cancer is a complex ecosystem, in which cancer-associated fibroblasts (CAFs), as the most abundant stromal cell type, meticulously construct an ecological niche that supports tumor growth through mechanisms including extracellular matrix (ECM) remodeling, secretion of bioactive factors, and interactions with neighboring cells. High-resolution technologies, including single-cell sequencing and spatial transcriptomics, have revealed the high heterogeneity, functional diversity, and spatial distribution within the CAF population. Significant differences exist in the interactions between distinct CAF subpopulations and immune cells. Through complex crosstalk with the immune system, they collaboratively establish an immunosuppressive network, becoming a core driving force for tumor immune escape. This review focuses on the latest research advances in heterogeneous subpopulations of CAFs within the breast cancer microenvironment, delves into how the complex bidirectional crosstalk between different CAF subpopulations and immune cells collaboratively shapes the tumor immune microenvironment (TIME), and summarizes various CAF-based therapeutic strategies for breast cancer, aiming to provide critical theoretical basis and novel therapeutic perspectives for the clinical translation of CAF heterogeneity research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e74899699742a2ae95fa97ca3627d6bdb366035" target='_blank'>
              Crosstalk between heterogeneous cancer-associated fibroblast subpopulations and the immune system in breast cancer: key players and promising therapeutic targets
              </a>
            </td>
          <td>
            Lu Liu, Weijun Wan, Yilin Chang, Luoquan Ao, Yan Xu, Xiang Xu
          </td>
          <td>2025-09-01</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Intra-tumoral heterogeneity (ITH) is common in malignant tumors and arises from dynamic variations across genetic, epigenetic, transcriptomic, proteomic, metabolic, and microenvironmental factors. This complexity drives tumor evolution and treatment resistance, undermining the accuracy of clinical diagnosis, prognosis, and treatment planning. Despite recent advances in multi-omics technologies that enable comprehensive mapping of ITH across molecular layers, major challenges remain in clinical translation. This review outlines the principles and clinical applications of eight major omics modalities in the context of ITH: genomics, single-cell genomics, transcriptomics, epigenomics, proteomics, radiomics, microbiome, and metabolomics. We highlight the unique contributions of each omics platform to tumor profiling and emphasize how their integration enhances biological interpretation, patient stratification, and biomarker discovery. Furthermore, we will focus more extensively on the limitations of applying these approaches to ITH analysis. Instead of providing an exhaustive catalog, this review highlights major controversies, technical hurdles, and conceptual gaps that impede the clinical translation of multi-omics-based ITH analysis, with the aim of addressing ITH-related clinical challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3a8e32658c6f97ddac24dad4088bab77b9c897d" target='_blank'>
              Integration of multi-omics approaches in exploring intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            Mengmeng Dong, Liping Wang, Ning Hu, Yueli Rao, Zhen Wang, Yu Zhang
          </td>
          <td>2025-08-29</td>
          <td>Cancer Cell International</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a fundamental role in tumor progression. Cancer cells interact with their surroundings to establish a supportive niche through structural changes and paracrine signaling. Cells around transformed tumor cells contribute to cancer development, while infiltrating immune cells in this aggressive TME often become exhausted. Solid tumors, especially the most invasive types such as pancreatic ductal adenocarcinoma, are notably stiff mechanically, with cross-linking enzymes significantly affecting the survival of cancer cells in both primary tumors and metastatic sites. In this review, we highlight recent key contributions to the field, focusing on single-cell sequencing of stromal cells, which are increasingly seen as highly heterogeneous yet classifiable into distinct subtypes. These new insights enable the development of effective co-treatment approaches that could significantly enhance current and novel therapies against the most aggressive cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5047b2e8e1a1022055daa398bbad8bc791fe908c" target='_blank'>
              Microenvironment of Solid Tumors.
              </a>
            </td>
          <td>
            T. Korneenko, Nikolay B. Pestov, M. Shakhparonov, Nickolai A. Barlev
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Summary Kaposi’s sarcoma (KS) is a highly inflammatory, angiogenic tumor driven by Kaposi’s sarcoma-associated herpesvirus (KSHV), yet the origins of tumor cells and mechanisms of progression remain unclear. Here, we present the first spatial single-cell atlas of KS, profiling 256 samples across patch, plaque, and nodular lesions and normal controls. We identify CD34+ progenitor lymphatic endothelial cells (LECs) as the primary targets of KSHV, whose clonal expansion drives tumor growth. KSHV infection induces widespread cellular reprogramming across the tumor microenvironment, including LECs, vascular endothelial cells, fibroblasts, and macrophages, generating hybrid phenotypes that support angiogenesis, inflammation, and immune modulation. KSHV+ macrophages are enriched in tumor-proximal niches, further promoting a proangiogenic, immunosuppressive environment. Spatial analysis reveals evolving tumor-associated niches, with a core-to-periphery gradient correlating with infection, immune modulation, and cellular remodeling. We identify disease progression predictive signatures, offering mechanistic insights into KS pathogenesis and potential new therapeutic strategies by reprogramming the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/835b859927d90c9b0c5454b990e0f2840ed6f867" target='_blank'>
              Spatial Single-Cell Atlas Reveals KSHV-Driven Broad Cellular Reprogramming, Progenitor Expansion, Immune and Vascular Remodeling in Kaposi’s Sarcoma
              </a>
            </td>
          <td>
            Wen Meng, Arun Das, Harsh Sinha, Rana Naous, P. Bracci, Mike McGrath, Yufei Huang, Shou-Jiang Gao
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Abstract Immunotherapy, particularly immune checkpoint inhibitors (ICIs), has revolutionized cancer treatment, yet durable responses are achieved in only a subset of patients. The tumor microenvironment (TME) plays a key role in cancer progression and immune modulation, critically influencing the efficacy of ICIs. Recent advances in single‐cell technologies have enabled high‐resolution profiling of the TME, particularly tumor‐infiltrating immune cells, across diverse cancer types. However, our understanding of how immune cells shape ICI responses and how they are dynamically altered during treatment remains incomplete. In this review, we summarize recent progress in characterizing TME features associated with ICI responsiveness, highlighting key immune cell subsets involved in ICI therapy and emphasizing their phenotypic plasticity and functional adaptability following ICIs. Additionally, we outline the spatial architecture of the TME in terms of its effects on immune cell behavior and interactions, and discuss the critical role of stromal components and the microbiota in modulating the immune landscape and influencing ICI responsiveness. By integrating these insights, we aim to deepen our understanding of the cellular and molecular mechanisms underlying ICI responses, elucidate determinants of therapeutic sensitivity and resistance, and inform the development of more effective immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa3d10e168326d4a4d7bd5875ce8bcf47cc231e7" target='_blank'>
              Insights into the Mechanisms of Immune‐Checkpoint Inhibitors Gained from Spatiotemporal Dynamics of the Tumor Microenvironment
              </a>
            </td>
          <td>
            Yuanyuan Zhang, Zhihua Liu
          </td>
          <td>2025-08-28</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling.
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metastasis remains the leading cause of cancer-related death, yet the biological determinants that enable tumor cells to disseminate and colonize distant organs are incompletely understood. Emerging evidence identifies the microbiome, not merely as a bystander, but as an active architect of the metastatic cascade. Microbial communities residing in the gut, mucosal barriers, and within tumors shape metastatic progression by modulating immune surveillance, stromal remodeling, oncogenic signaling, and therapy response. Intratumoral and even intracellular microbes regulate epithelial–mesenchymal transition, angiogenesis, and immune escape, while gut-derived metabolites condition pre-metastatic niches and alter systemic immunity. Technological advances in spatial transcriptomics, single-cell multi-omics, and metagenomics have revealed a spatially organized, functionally integrated microbial ecosystem within tumors, challenging long-held assumptions of sterility in cancer biology. This review synthesizes five converging dimensions of this paradigm: microbial interactions in the metastatic tumor microenvironment; microbiome-mediated immunoediting and metastatic escape; the role of intratumoral and intracellular bacteria in dissemination; spatial-multi-omic approaches to map microbial niches; and microbial biomarkers predictive of metastasis and therapy outcomes. Collectively, these findings recast the microbiome as a critical and targetable determinant of metastasis. Deciphering the tumor–microbe–host triad holds transformative potential for biomarker development, therapeutic innovation, and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15720d6be643a5b1bcd7bdc88b245af42661e241" target='_blank'>
              Microbial signatures in metastatic cancer
              </a>
            </td>
          <td>
            Jhommara Bautista, María Paula Fuentes-Yépez, Joseth Adatty-Molina, Andrés López-Cortés
          </td>
          <td>2025-09-30</td>
          <td>Frontiers in Medicine</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="


 Diffuse midline glioma (DMG) is an aggressive pediatric brain tumor with no curative treatment. The tumor microenvironment (TME) of DMG, shaped by its unique anatomical location and developmental context, poses significant therapeutic hurdles. While the TME has been characterized by a predominance of tumor-associated microglia and sparse infiltration of blood-derived immune cells, a comprehensive understanding of immune-tumor cell interactions and how to effectively target them with immunotherapies remains challenging.



 To dissect the DMG TME, we employed a multi-omics approach that integrated single-nucleus RNA sequencing, spatial transcriptomics, and high-dimensional imaging on both patient samples and an immunocompetent in utero electroporation murine DMG model. We characterized the spatial and phenotypic architecture of the tumors and validated a novel immune target using syngeneic mouse models and ex vivo co-culture experiments.



 Our analysis revealed pronounced intratumoral heterogeneity, with two spatially distinct patterns of cancer-immune interaction. A mesenchymal-like niche promoted the recruitment of blood-derived immune cells, whereas another pattern exhibited an immunologically cold phenotype, reminiscent of the immune-privileged healthy brain. Spatially resolved analysis of cell-cell communication within the TME identified the VISTA immune checkpoint pathway as a key regulator of these patterns. Targeting of this axis in a syngeneic mouse model led to significant tumor reduction and 100% survival in mice, highlighting its therapeutic potential. Further validation demonstrated that microglia play a critical role in orchestrating immune cell recruitment and anti-tumor activity, and that modulating the VISTA pathway can reshape the immune response.



 Our findings uncover the spatial and cellular complexity of the DMG immune microenvironment and identify the VISTA pathway as a critical modulator of immune suppression. By demonstrating robust therapeutic efficacy upon VISTA blockade, this study lays a strong foundation for developing targeted immunotherapies in DMG and opens new avenues for overcoming immune resistance in pediatric brain tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3d8c86dfadea2de1a37fb282c8875e761a9bc8" target='_blank'>
              JS07.7.A VISTA IS A CRITICAL AND TARGETABLE IMMUNE CHECKPOINT PATHWAY IN DIFFUSE MIDLINE GLIOMA
              </a>
            </td>
          <td>
            R. Collot, C. Ruiz-Moreno, C. Honhoff, A. Wezenaar, H. Ariese, H. Johnson, B. M. Vervoort, J. Bunt, E. Bokobza, H. G. Rebel, B. T. Te Pas, P. Robe, M. Kool, M. Kranendonk, D. V. van Vuurden, E. Hulleman, E. J. Wehrens, A. Zomer, H. Stunnenberg, A. Rios
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal and immunologically "cold" cancer, with a 5-year survival rate of 13%. Despite the success of immunotherapy in other malignancies, its efficacy in PDAC remains limited due to complex immune evasion mechanisms and a pro-tumorigenic immune microenvironment. Several lines of evidence suggest that PDAC progresses from acinar-to-ductal metaplasia (ADM) to pancreatic intraepithelial neoplasia (PanIN) lesions to invasive cancer, accompanied by increasing immunosuppression. We hypothesize that understanding the spatial and temporal transitions of immune cell states during tumor evolution is key to uncovering mechanisms of immune dysfunction and guiding new therapeutic strategies. Using single-cell RNA sequencing (scRNA-seq) and multispectral imaging, we aim to uncover comprehensive, large-scale, and longitudinal changes in the immune microenvironment during PDAC progression. Here, we utilized the KrasG12D/+; Trp53R172H/+; Ptf1a-Cre (KPC) genetically engineered mouse model, and through longitudinal histological analysis of pancreas, we delineated the trajectory of disease progression, with PanIN I-II lesions appearing by 8–9 weeks, PanIN III by 11–12 weeks, and locally invasive PDAC by 16 weeks of age. To study the temporal evolution of immune cell-state transitions, we performed scRNA-seq on CD45+ enriched immune cells isolated from KPC mouse pancreas collected at 4, 8, 12, and 16 weeks using the 10x Genomics Chromium platform. To examine the spatial dynamics of immune cells, we performed multispectral imaging at defined time points using the Lunaphore COMET platform with a panel of over 15 antibodies, enabling simultaneous visualization of several markers on a single tissue section. In our analysis to date, we identified subsets of pro-tumorigenic macrophages emerging during PDAC development, along with a marked increase in myeloid-derived suppressor cells (MDSCs) in late-stage tumors. We also observed dynamic shifts in other immune cell populations evolving alongside tumor progression. Notably, we uncovered distinct subsets of monocytes, macrophages, and T cells exhibiting tumor-promoting potential, progressively enriched during the advanced stages of PDAC. Using multispectral imaging, we generated a comprehensive spatial map of the immune microenvironment across PDAC progression, capturing patterns of immune exclusion, infiltration, and cell–cell neighborhood interactions. Our study highlights the multi-layered immune dysfunction that evolves from early to late stages of PDAC - features that are challenging to capture in human samples. CD45+ enrichment improved the detection of rare and transient immune subsets by mitigating the masking effects caused by abundant cancer and stromal cells. Together, these findings provide an unbiased and detailed view of the evolving immune landscape in PDAC, laying the foundation for identifying microenvironmental and cell-intrinsic factors that either constrain or promote malignant progression, as well as for developing immune-based therapeutic strategies.



 Atul Verma, Doreen Klingler, Linda Bergmayr, Tanvi V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug. Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0f49d549158981070f5d866ac858fc605d13b" target='_blank'>
              Abstract A106: Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression
              </a>
            </td>
          <td>
            Atul Verma, Doreen Klingler, Linda Bergmayr, T. V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Most meningiomas are benign, slow-growing tumors. However, CNS WHO grades 2 and 3 meningiomas display significantly higher aggressiveness and poorer prognosis. Recent investigations into the molecular biology of meningiomas have greatly increased our understanding of these tumors. Furthermore, emergent spatial transcriptomic sequencing methods now provide an unprecedented opportunity to study molecular mechanisms of tumor development within a spatial context and on a single-cell level. This study aims to uncover specific microenvironmental features, cell-cell interactions and transcriptional regulatory mechanisms present in different molecular classes and subclones of meningiomas.



 We employed single-nuclei RNA sequencing and spatial transcriptomic profiling on a matching set of meningiomas (n=22). We developed a method based on non-negative matrix factorization (NMF) to analyze intra-tumor heterogeneity from both single-cell and spatial transcriptional profiles across samples. Trajectory analysis was performed to screen for expression signatures selectively up-regulated around specific morphological features of the tumors. We clustered cells into types of niches based on compositions of their spatial neighborhoods.



 We annotated cell types in the tumor microenvironment (TME), and identified cellular states of meningioma tumor cells in single-cell and spatial transcriptomic data. Tumors of different subclasses display drastic differences in spatial cell state compositions, suggesting distinct modes of microenvironmental organization. We identified conserved patterns of tumor-TME cell state colocalization and interaction. Benign NF2-mutant meningiomas are characterized by extensive infiltration of myeloid cells which trigger a series of immune and stress response pathways in neighboring tumor cells. Among more malignant tumors, a decrease of immune cell populations is coupled with a marked shift in the TME toward tumor-supportive phenotypes. Moreover, we observed reduced tumor-infiltrative capabilities among tumor-associated myeloid cells and the formation of compartmentalized spatial niches of interacting tumor cells. Additionally, various tumor transcriptional signatures related to extracellular matrix remodeling, fibroblastic markers, and meningothelial signatures were mapped to distinct morphological structures across samples.



 Our results reveal that the meningioma TME is heterogenous at the cell state level, with tumor cells and normal cells forming niches that have varying functions in tumor pathophysiology and characterize molecular classes of meningiomas. A series of corresponding RNA markers and spatial signatures were identified, which could potentially be used to increase precision in diagnosis and stratification of meningiomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb7c99c73e50db41072f3f2a1080df073fd8ef43" target='_blank'>
              OS13.6.A A SINGLE-CELL AND SPATIAL ATLAS OF MENINGIOMA HETEROGENEITY ACROSS TUMOR CLASSES
              </a>
            </td>
          <td>
            Y. Tang, R. Banan, S. Maas, D. Calafato, R. Michael, J. Kada Benotmane, R. Yan, G. Rukhovich, Z. Ferdinand, V. Venkataramani, A. Suwala, I. Bludau, S. Krieg, W. Wick, A. von Deimling, D. Heiland, M. Gerstung, B. Patel, F. Sahm
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 The tumour microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC) is comprised of heterogenous malignant epithelium and diverse immune infiltrates within a dense, fibrotic stroma. This multi-compartmental TME includes myeloid and lymphoid lineages, cancer-associated fibroblasts (CAFs), mural and endothelial cells, and tumour cells embedded in a rich extracellular matrix (ECM). Efforts to profile and perturb compartments or cellular phenotypes in isolation have often led to unintended adverse effects on neighbouring cells, compromising drug efficacy. This drives the need for an integrated, spatially-resolved understanding of the architecture and molecular features underlying the organization of the TME. To address this, we leveraged single-cell RNA sequencing (scRNAseq) data from eight distinct datasets to design three custom multiplexed imaging mass cytometry (IMC) panels against tumour, immune, and stromal targets. These were applied to three serial sections of a tissue microarray of 221 PDAC patients. Segmentation and clustering of 7.9 million cells revealed 83 distinct cell types and their functional states, including biophysical responses. Advanced image alignment across serial sections enabled the incorporation of TME-wide features and resolved eight spatially recurrent PDAC microenvironments neighbouring phenotypically distinct epithelial ducts. Two microenvironments, ECM-rich and immune-suppressed, were associated with worse overall survival and frequently co-occurred with a third, stiff matrix, characterized by stroma enriched in phospho-myosin light chain 2 expressing CAFs. In contrast, the immune infiltrated stroma microenvironment was linked to improved clinical outcomes and often co-occurred with CD105+ fibrovascular and immune infiltrated microenvironments. Matched whole-genome sequencing (n = 182/221) revealed that patients dominant in ECM-rich regions harboured 12p11.21 (KRAS) and 8q24.21 (MYC and POU5F1B) amplifications, and 17q22 (RNF43) deletions, and stiff matrix were associated with 13q33.3 (LATS2) deletion. To link spatial niches to proteomic signatures, we performed whole-slide IMC-guided laser capture microdissection and mass spectrometry, and further integrated differentially expressed proteins with transcriptomes from an external scRNAseq cohort (n = 163). ECM-rich and immune suppressed regions showed signatures of mechanically induced immune suppression, including NETosis and innate immune surveillance via pattern recognition receptors. Contrastingly, immune infiltrated microenvironments were enriched for cytotoxic activity and immune trafficking programs, while immune infiltrated stroma displayed signatures associated with oxidative stress responses. These findings define unique spatial ecosystems within the PDAC TME that are linked to specific genomic alterations, functional states and clinical outcomes, highlighting their therapeutic relevance and potential as targets to overcome the challenges of treating PDAC.



 Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J. Geuenich, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Cassandra J. Wong, Brendon Seale, Zhen Yuan Lin, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steven Gallinger, Hartland W. Jackson. Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B115.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e942fb4c97db1a6398873aeae226bf3656a085" target='_blank'>
              Abstract B115: Integrated spatial and multi-omic analysis reveals functional niches in the pancreatic cancer microenvironment
              </a>
            </td>
          <td>
            Noor Shakfa, Ferris Nowlan, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Michael J Geuenich, Sheng-Ben Liang, Barbara Gruenwald, A. Borgida, Cassandra J. Wong, Brendon Seale, Zhen Yuan Lin, Edward L. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, H. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Metastatic non-small cell lung cancer (mNSCLC) cells carry heterogeneity, not only among different subtypes but also within a single tumor. Most evidence suggests that mNSCLC exploits specific molecular drivers and mechanisms to maintain physiology, metabolism, and immune evasion during tumorigenesis. Genome-wide association studies also revealed particular mutations in the oncogenic drivers supporting tumor cell proliferation and survival, resulting in aggressive and drug-resistant phenotypes of mNSCLC. While significant progress has been made in understanding mNSCLC at the genetic and molecular levels, a considerable gap remains in understanding the dynamic interplay between intrinsic factors—particularly key tumor-associated cells—and tumor immune microenvironment (TIME) during metastasis. Hence, this review highlights histological and genetic characteristics, emphasizes the clinical relevance of metastasis, and the roles of tumor-associated cells in shaping the immunosuppressive tumor microenvironment (TME) in mNSCLC. Understanding these intricate features and mechanisms is crucial for identifying novel therapeutic targets and improving strategies to combat mNSCLC progression in diagnosed patients. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/931263caf4b3388e1caa012daaf5acb9be29f764" target='_blank'>
              Unraveling the molecular-pathological characteristics and cellular complexity of the tumor immune microenvironment in metastatic non-small cell lung cancer
              </a>
            </td>
          <td>
            S. Bharadwaj, J. M. Mierzwicka, Lucie Vaňková, Petr Malý
          </td>
          <td>2025-09-29</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 PDAC is the most hypovascular solid tumor, imposing harsh constraints on tumor metabolism. Cancer cells cope with poor access to vascular-derived nutrients and O2 by cooperating with abundant stromal cells, receiving alanine, proline, glycosaminoglycans, branched chain ketoacids, lysophosphatidic acid, and unsaturated fatty acids from fibroblasts. However, the extent of metabolic support provided by other cell types in the PDAC TME is unexplored. We reasoned that metabolically cooperative cell types would be spatially positioned near cancer cells in these tumors, and that cooperative spatial arrangements would be enriched under conditions of relative hypovascularity. We first performed Xenium spatial transcriptomics on a cohort of 8 treatment-naive human PDAC resections and generated a spatial atlas of nearly 700,000 cells across a range of expected cell types. Niche analysis uncovered 17 recurrent cellular neighborhoods in the data, including an endothelial-enriched niche. We find that tumor-associated neutrophils form dense aggregates within malignant glands in tumors with low endothelial content, and form in tumor regions which are most distal from vasculature. In a separate cohort of 4 additional PDAC tumors, the expression of neutrophil chemokines CXCL1/2/3/5 was highest in a subpopulation of malignant cells distant from endothelium. These data suggest metabolic stress secondary to poor vascularity drives recruitment of neutrophils via the CXCL1/2/3/5 axis. Indeed, when exposed to Tumor Interstitial Fluid Media (TIFM), cancer cells upregulated the transcription and/or secretion of these cytokines and G-CSF. Since the formation of neutrophil aggregates is driven by cancer cell nutrient derangement, we hypothesized that cancer cells could cooperate metabolically with neutrophils to alleviate nutrient stress driven by poor vascularity. Neutrophil co-culture promoted cancer cell proliferation under TIFM, but not RPMI, while under lethal combination TIFM and serum starvation, co-culture enabled enhanced survival of cancer cells. Lastly, to understand what metabolic processes could be fueled by neutrophil aggregates in pancreatic cancer, we designed a fully custom spatial transcriptomics panel targeting ∼360 metabolic enzymes and profiled 4 additional PDAC cases, performing dispersed whole-transcriptome scFFPEseq on an adjacent tissue section. We integrated data across the spatial and dispersed modalities to enable spatial projection of whole-transcriptome-derived information. Cancer cells located near neutrophil aggregates had significantly enhanced expression of vitamin B6 salvage pathway, suggesting that one critical shared metabolite between neutrophils and cancer cells is the B6 precursor pyridoxine. In total, we have deployed multiple spatial and single cell RNA profiling technologies to uncover a novel node of metabolic crosstalk in pancreatic cancer between neutrophils and cancer cells. Future work is needed to confirm transfer of specific metabolites and the downstream sequelae of their provision for tumor growth.



 Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M Celeste. Simon. Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec70e5707c093ceceed418e17d0f9e9a3b6c60c" target='_blank'>
              Abstract B127: Spatial profiling reveals cancer cell metabolic cooperation with aggregative neutrophils in pancreatic cancer
              </a>
            </td>
          <td>
            Carson D. Poltorack, Connor J. Hennessey, Sydney M. Shaffer, M. C. Simon
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e2642fd5880574a79a83aa30d56f3b2134c4d69" target='_blank'>
              Applications of single-cell transcriptomics: updated insights in endometrial cancer.
              </a>
            </td>
          <td>
            Shuyue Xiao, Huixin Li, Jianyao Liu, Xinyi Xie, Hanzi Xu, Zhen Gong, Shanliang Zhong
          </td>
          <td>2025-09-03</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f3009a3214b83a250848d6ef5905a4fb9e1a3db" target='_blank'>
              Unraveling the single cell spatial landscapes of melanoma brain metastases
              </a>
            </td>
          <td>
            A. Manukyan, K. Peters, H. Radbruch, F. Roßner, A. Lehmann, K. Kleo, T. Conrad, J. Altmueller, I. Plumbon, A. Akalin, K. Juerchott, E. Wyler, M. Landthaler, J. Radke, T. Redmer
          </td>
          <td>2025-09-02</td>
          <td>None</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Cancer-associated fibroblasts (CAFs) are critical mediators of tumor-stroma crosstalk in pancreatic ductal adenocarcinoma (PDAC). While bulk and single-cell transcriptomic studies have uncovered CAF heterogeneity, the spatial organization of CAF subtypes and their physical interactions with tumor epithelial subtypes remain incompletely understood. Here, we integrate bulk, single-cell, and spatial transcriptomic datasets to systematically interrogate the molecular, cell, and spatial interactions between tumor-promoting CAFs (proCAFs), tumor-restraining CAFs (restCAFs), and basal-like versus classical tumor subtypes across multiple biological scales. Ligand-receptor analysis of single-cell RNA-seq data revealed robust signaling between proCAFs and basal-like tumor cells, particularly through integrin-mediated (COL11A1–ITGA11) pathways, suggesting subtype-specific stromal-epithelial crosstalk within distinct tumor microenvironments. Histopathologic validation via immunofluorescence staining confirmed that proCAF regions are preferentially located adjacent to tumor epithelium and are characterized by reactive, myxoid stroma, whereas restCAF regions exhibit a fibrous, hypocellular morphology. To further elucidate spatial relationships, we performed 10x Genomics Visium spatial transcriptomics on seven primary PDAC tumors. Application of the DeCAF classifier to Visium spots demonstrated a distinct radial gradient of proCAF expression, peaking at the tumor-stroma interface and diminishing with distance—indicative of spatial colocalization with basal-like programs. Notably, proCAF gene signatures were specifically enriched in stromal regions adjacent to basal-like tumor zones, while classical tumor regions were spatially associated with heterogeneous CAF compositions. This subtype-specific spatial organization and paracrine signaling may underlie the clinical observation that basal-like tumors are significantly more likely to co-occur with proCAF stroma (64.5%, p = 2.17e–5). Importantly, patients harboring both basal-like and proCAF subtypes exhibited the poorest clinical outcomes (median overall survival: 11.0 months), compared to those with classical/restCAF profiles (median overall survival: 30.4 months, p < 0.001). Collectively, these multidimensional data define spatially organized and functionally interactive tumor-stroma niches in PDAC. Our findings highlight a co-evolutionary relationship between proCAFs and basal-like tumor cells that contributes to an aggressive disease phenotype, and underscore the potential of spatially resolved tumor-stroma profiling for improving patient stratification and guiding therapeutic targeting of the tumor microenvironment.



 Xianlu Laura. Peng, Ian McCabe, Elena Kharitonova, Ryan Zhao, Changfei Luan, Arthi Hariharan, Jaewon Lee, Nancy Kren, Yi Xu, Joseph Kearney, Justin Su, Michelle LaBella, Susan Tsai, Brian Belt, Roheena Panni, David Linehan, Yuliya Pylayeva-Gupta, Alina Iuga, Naim Rashid, Jen Jen Yeh. Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B100.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af6682e5a69c7a44f5aef0626bbff15428b91a16" target='_blank'>
              Abstract B100: Spatial relationships between tumor subtypes and tumor-promoting CAFs with clinical implications
              </a>
            </td>
          <td>
            X. Peng, Ian C McCabe, Elena V Kharitonova, Ryan Zhao, C. Luan, Arthi Hariharan, Jaewon J Lee, Nancy P. Kren, Yi Xu, Joseph F Kearney, Justin Su, M. LaBella, Susan Tsai, Brian Belt, R. Panni, David Linehan, Y. Pylayeva-Gupta, Alina C Iuga, Naim U Rashid, J. J. Yeh
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Introduction Leiomyosarcoma (LMS) is a rare, aggressive cancer with limited treatment options at the metastatic stage. The response to immune checkpoint inhibitors (ICIs) is inconsistent, likely due to intratumoral heterogeneity, which is more pronounced in large tumours such as retroperitoneal LMS. Methods This study examined heterogeneity in four large treatment-naive LMS tumours (ten samples per tumour) by analysing immune cells, tertiary lymphoid structures (TLSs), checkpoint molecules, and cytokine secretion across different tumour regions. Results Significant region-dependent differences were observed in immune components, with TLSs present only at tumour margins and inconsistently across samples from the same tumour. Expression levels of programmed cell death 1 (PD-1) and programmed death ligand 1 (PD-L1) varied within individual tumours, and shared immune patterns were identified in specific regions, including elevated indoleamine 2,3-dioxygenase 1, absence of a particular macrophage subpopulation, and reduced PD-1 and lymphocyte activation gene 3 (LAG-3) expression at organ-adjacent margins. Anti-LAG-3 blockade altered cytokine and checkpoint molecule levels in a region-specific manner. Conclusion These findings highlight substantial intratumoral heterogeneity, which may contribute to the variable response to ICI therapy. As immune checkpoint molecule expression influences treatment eligibility, multiple biopsies from different tumour regions may be necessary to assess immune infiltration accurately and guide therapy decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f47a854ab32ca2eceb50d1b8d1469818b11328c" target='_blank'>
              Spatial immunoprofiling of retroperitoneal leiomyosarcomas reveals intratumoral heterogeneity in immune cell infiltration, checkpoint molecule expression, and tertiary lymphoid structures
              </a>
            </td>
          <td>
            Iva Benesova, Jan Balko, V. Tovazhnianska, M. Rataj, R. Lischke, J. Bartůňková, K. Kopeckova, T. Buchler, W. V. van Houdt, Yvonne Schrage, David Moura, J. Broto, Zuzana Ozaniak Střížová, A. Ozaniak
          </td>
          <td>2025-10-13</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Adenoid cystic carcinoma of the salivary gland (SGACC) is a highly aggressive malignancy characterized by poor patient survival outcomes. While several studies have analyzed the transcriptome of the salivary gland at the bulk and single-cell level, no spatial transcriptomic analyses of this tissue have been published. Most of the existing publications on SGACC have predominantly relied on bulk and single cell RNA sequencing approaches, which do not resolve the spatially localized transcriptional heterogeneity nor have the resolution for defining molecular markers within tumor subpopulations. SGACC is clinically notable for the presence of multiple tumor clones, distinct spatial phenotypes, and its indolent yet invasive nature coupled with a high propensity for distant metastasis. These features may reflect co-expression of tumor-associated markers across diverse cellular niches, and a resultant biological complexity which causes standard treatment such as surgical resection, radiation therapy, and chemotherapy to be largely ineffective in significantly improving long-term survival, and highlights the need for more precise, targeted therapeutic strategies. Herein, we analyzed single cell (n = 4) and high-resolution spatial transcriptomics samples (n = 5) to characterize cancer cell populations in MYB- and non-MYB-expressing cell states, delineated gene expression signatures, and identified critical molecular interactions specific to SGACC. We used Visum HD to obtain spatial transcriptomics data at 2μm squared high resolution. This allowed a multi-omics approach comprising single cell and spatial transcriptomic methods to enable the discovery of novel transcriptional signatures and microenvironmental features not captured by conventional methods. Spatial mapping revealed marked cellular heterogeneity and demonstrated how tissue environments influence cellular transcriptomics. To tumor heterogeneity, we focused on tumorigenic cell populations, profiled plasma and T cell enrichment within the tumor microenvironment and identified key pathways and transcriptional drivers including the MYB-NFIB fusion underlying the tumor cluster formation. Our findings indicate an upregulation of genes involved in extracellular matrix remodeling, autophagy, and reactive stromal cell populations. We further found evidence of partial epithelial-mesenchymal transition (P-EMT) programming within MYB-expressing tumor clusters. Pathway analysis revealed that mutations in the spatial query sample prominently affect the PI3K-AKT and IL-17 signaling pathways, together with a downregulation of canonical Wnt signaling in some regions of the tissue architecture adjacent to immune cells. Collectively, these results underscore the complex regulatory landscape of SGACC and offer insights into its cellular dynamics and possible therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d936540aedb35ad533bb62a60043e3d08a9252f" target='_blank'>
              Gene-Expression Programs in Salivary Gland Adenoid Cystic Carcinoma Analyzed Using Single-Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ifeoma Ebinumoliseh, Gopikrishnan Bijukumar, Kendall Hoff, K. Brayer, Elaine L. Bearer, Scott A. Ness, Jeremy S. Edwards
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The tumor microenvironment (TME) plays a pivotal role in shaping immune responses and therapeutic outcomes in lung cancer, yet the diversity and functional specialization of tumor-associated macrophages (TAMs) remain poorly resolved. Here, we present a refined classification of TAM subtypes across large cohorts of cancer datasets using integrative analysis of single-cell RNA sequencing, spatial transcriptomics, and clinical datasets from lung adenocarcinoma and lung squamous cell carcinoma. By combining cell-based gene scoring with hierarchical classification, we defined 7 macrophage subtypes-each with distinct transcriptional programs and abundances. Notably, lipid-associated TAMs expand with disease progression and exhibit immunosuppressive and protumorigenic features, whereas tissue-resident macrophages decline. Spatial and survival analyses reveal that an increased lipid-associated to tissue-resident TAM ratio correlates with advanced disease and poor prognosis. Given that spatial transcriptomic assays rely on deconvolution techniques to infer cell type compositions, accurate gene expression signatures are essential, especially for fine-grained subpopulations of TAMs. Our refined subtype-specific signatures address this bottleneck and enhance the resolution of spatial mapping efforts. These findings offer new insights into macrophage heterogeneity and highlight lipid-associated TAMs as potential biomarkers and therapeutic targets in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e49c52a60b0a9da9a722e72cb8391f58dc458b" target='_blank'>
              Transcriptional states of lung cancer microenvironment reveal macrophage subtype dynamics linked to disease progression.
              </a>
            </td>
          <td>
            Duygu Keremitçi, Özlem Tuna, Aissa Houdjedj, Hilal Kazan, Yasin Kaymaz
          </td>
          <td>2025-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous malignancy, with its tumor immune microenvironment (TIME) playing a crucial role in tumor progression, immune evasion, and treatment response. Despite insights from single-cell technologies, challenges such as small samples and inconsistent annotations have limited a comprehensive understanding. Thus, a large-scale, high-resolution single-cell atlas is needed to fully explore the heterogeneity of CRC TIME and guide personalized treatment. Methods We analyzed single-cell transcriptomic data from 100 CRC samples (371,223 cells). Immune, stromal, and epithelial cells in the TIME were clustered, annotated, and functionally analyzed. Transcription factor networks were inferred using SCENIC, differentiation states assessed with CytoTRACE, and cell–cell communication analyzed via CellPhoneDB. TIME subtypes were constructed from bulk RNA-seq data, followed by survival analysis. SCRN1 was further validated through in vitro experiments. Results This study presents a comprehensive single-cell CRC atlas, revealing significant heterogeneity in T cells, B cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells within the TIME. We defined two immune ecological subtypes: subtype 1 (enriched in metabolic and motility pathways with poor prognosis) and subtype 2 (enriched in immune response pathways, with better prognosis and greater immunotherapy potential). High SCRN1 expression in malignant epithelial cells correlated with poor prognosis and metastasis. In vitro experiments confirmed the role of SCRN1 in promoting CRC cell proliferation and migration. Conclusions This study provides a comprehensive characterization of the CRC TIME, links TIME subtypes to prognosis, and identifies SCRN1 as a potential therapeutic target, offering insights for CRC precision subtyping and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04173-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c5be539fad2c0073ae6e62159eaef1e83a1ef" target='_blank'>
              The single-cell atlas of colorectal cancer reveals cellular heterogeneity and key molecular features of the tumor immune microenvironment
              </a>
            </td>
          <td>
            JunHu Li, ZhiQiang Liu, BoYu Zhang, Liuping You, Yuenan Huang
          </td>
          <td>2025-10-10</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="SUMMARY Ovarian high-grade serous carcinoma (HGSC) is characterized by extensive intra-peritoneal dissemination and tumor heterogeneity. In the metastatic cascade, tumors utilize several transcriptional programs to translocate and survive in distant tissues. Here, we analyzed multi-modal, real-world data from 350 tumor samples across 160 patients with HGSC to identify transcriptional programs that drive intra-peritoneal metastasis and heterogeneity. We identified nine transcriptional programs, including those regulating epithelial-mesenchymal transition and immune modulation and cytoskeletal reorganization, which shape distinct metastatic trajectories to solid and ascitic environments and are associated to treatment response. Our results reveal pronounced intra-patient transcriptional heterogeneity, which in some cases surpassed inter-patient heterogeneity, highlighting the importance of multi-site sampling for accurate prognostication and combinatorial treatments in HGSC. Our extensive characterization offers novel insights into intra-peritoneal metastasis with significant prognostic implications, reveals histomorphological biomarkers for patient stratification and paves the way for innovative therapeutic strategies aimed at impairing cancer cell adaptability and limiting metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32c6dc18825437bb12a5a64bda09f91c2c6d329" target='_blank'>
              Multi-modal characterization of transcriptional programs that drive metastatic cascades to solid sites and ascites in ovarian cancer
              </a>
            </td>
          <td>
            Kaiyang Zhang, Essi Kahelin, G. Marchi, Oskari Lehtonen, S. Salloum, K. Lavikka, Yilin Li, Felix Dietlein, A. Lahtinen, J. Oikkonen, S. Hietanen, J. Hynninen, Antti Häkkinen, A. Virtanen, S. Hautaniemi
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Ovarian cancer represents a typically immune “cold” tumor, where obvious immunosuppression, spatial T-cell exclusion, and cellular dysfunction collectively limit immunotherapy effectiveness. Especially in high-grade serous ovarian carcinoma (HGSOC), the immune low-response state is driven by complex interactions among tumor-associated macrophages (TAMs), suppressive stromal networks, and the T-cell compartment (regulatory T cells, Tregs, and exhausted effector T cells). Emerging multi-omics technologies—particularly single-cell RNA sequencing and spatial transcriptomics—have showed the heterogeneity and spatial immune organization underlying this suppressed state. Here, we integrate these datasets to describe TAM phenotypes and spatial niches, T-cell exhaustion, Tregs accumulation, NK-cell dysfunction, and stromal barriers that enforce exclusion. We then derive phenotype-guided combination strategies to remodel the tumor microenvironment and improve responsiveness to immune checkpoint blockade. This synthesis provides a concise, multi-dimensional framework for precision immunotherapy and for overcoming resistance in immune-low ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880f6c405f773eb38fd078c68256074438417fa2" target='_blank'>
              Decoding immune low-response states in ovarian cancer: insights from single-cell and spatial transcriptomics for precision immunotherapy
              </a>
            </td>
          <td>
            Yalan Yan, Jiaan Lu, Huanyu Luo, Zizhang Wang, Ke Xu, Lexin Wang, Qin Wang
          </td>
          <td>2025-09-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer characterized by a lack of hormone receptors, making it challenging to treat. Methods We generated a comprehensive spatial cell atlas of TNBC using a multi-omics integration approach that combined single-cell RNA sequencing (scRNA-seq) with spatial transcriptomics. This integration allowed us to characterize the spatial microenvironment and map the cell-type-specific distributions in TNBC tissues. Results Our analysis revealed significant heterogeneity in cell types and spatial distribution, with normal regions enriched in insulin resistance functions, whereas cancerous regions displayed diverse cell populations, including immune cells, cancer-associated fibroblasts (CAFs), and mesenchymal cells. By constructing transcription factor (TF) regulatory networks, we identified TFF3, RARG, GRHL1, RORC, and KLF5 as critical regulators of epithelial cells, whereas EMX2, TWIST1, TWIST2, NFATC4, and HOXC6 were found to play essential roles in mesenchymal cells. Immunohistochemical validation supported the involvement of these TFs in TNBC. Further analysis of receptor-ligand interactions highlighted the roles of KNG1_BDKRB2 and NRG1_ERBB4 signaling in promoting tumor aggression, suggesting potential therapeutic targets. GO enrichment analysis revealed overlapping pathways between epithelial and mesenchymal cells, focusing on migration, signaling, and development, indicating that the shared regulatory mechanisms contribute to cancer progression. Conclusion Our findings provide new insights into the TNBC microenvironment, emphasizing the complex spatial interactions between different cell types and highlighting key regulatory pathways that could be targeted for future therapeutic interventions. This spatial cell atlas lays the foundation for further exploration of tumor microenvironment dynamics and precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8697e1d4f457ce1a990f5f037a0dd30fbd923ee5" target='_blank'>
              Multi-omics integration uncovers key transcriptional regulators in triple-negative breast cancer spatial heterogeneity
              </a>
            </td>
          <td>
            Ning Zhang, Ye Zhang, Rui-Fei Yang, Hai-Qing Chu, Abdur Rehman, Lu-Yu Yang, Ya-Yu Li, Xin Zhou, Shou-Ping Gong, Hui-Ling Cao
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose The dysregulated immune microenvironment represents a key pathogenic driver in psoriatic lesions. However, the intricate cellular and molecular interactions underlying psoriasis remain incompletely elucidated. Therefore, we aim to employ integrated multi-omics approaches to characterize the immune microenvironment and pathogenic niche in psoriasis, thereby elucidating the cellular and molecular mechanisms of disease pathogenesis. Methods Integrated Single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and bulk RNA sequencing (RNA-seq) data to explore the heterogeneity of stromal cells and immune cells in psoriatic lesions and the complex spatial niches formed between them. Enrichment analysis, intercellular communication analysis, and spatial co-localization analysis were used to investigate the transcriptional changes and distribution characteristics of each cell type in the lesions of psoriasis patients. Results Using scRNA-seq, we identified a novel CD4+ tissue-resident memory T cell (TRM) subset that is exclusively present in lesional skin of psoriasis patients but absent in healthy skin. These cells exhibit elevated expression of genes including IL17RA, IL22, PD1 (PDCD1), CXCR6, ITGAE, CD69, TNFRSF9, TNFRSF4, IL7R, CD4, and STAT3. Additionally, we discovered a novel microvascular endothelial cell subset, designated Venous endo2, which highly expresses CD93, ACKR1, ICAM1, VCAM1, IL15, SELE, and SELP, while also overlapping with high endothelial venule (HEV)-associated transcriptional signatures. Integrated analysis of scRNA-seq and spatial transcriptomics further revealed strong spatial co-localization of Venous endo2 with fibroblast activation protein-positive fibroblasts (FAP+ Fbs), T cells, and antigen-presenting cells (APCs) in Psoriasis lesions—a pattern not observed in healthy control skin. Conclusion Through integrated multi-omics analysis, we identified a potential pathogenic niche in psoriasis patients, composed of Venous endo2, FAP+ Fbs, T cells, and APCs. This structure resembles tertiary lymphoid structures (TLS), suggesting a functional parallel in disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d8eeb3b234a229f3a9f78e22b42dca0befb2a94" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals a Pathological Niche Formed by FAP+ Fibroblasts, Immune, and Endothelial Cells in Psoriatic Lesions
              </a>
            </td>
          <td>
            Wuwei Zhuang, Qi Zhang, Qingtao Kong, Yun Hui, Jie Shen, Chen Zhang, Hong Sang, Qiao Ye
          </td>
          <td>2025-09-01</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0c307663da680bc20dfe3893c95285de2664ca" target='_blank'>
              Spatiotemporal analyses of the pan-cancer single-cell landscape reveal widespread profibrotic ecotypes associated with tumor immunity.
              </a>
            </td>
          <td>
            Ya Han, Lele Zhang, Dongqing Sun, Guang-xu Cao, Yuting Wang, Jiali Yue, Junjie Hu, Zhonghua Dong, Fang Li, Taiwen Li, Peng Zhang, Qiu Wu, Chenfei Wang
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>2</td>
          <td>12</td>
        </tr>

        <tr id="Tumor heterogeneity, a hallmark of cancer, frequently leads to treatment failure and relapse. However, the intricate communication between various cell types within the tumor microenvironment and their roles in tumor progression in vivo remain poorly understood. Here we establish a novel tumor heterogeneity model in the Drosophila larval eye disc epithelium and dissect the in vivo mechanisms by combining sophisticated genetics with single-cell RNA sequencing. We found that mutation of the tricellular junction protein M6 in cells surrounding RasV12 benign tumors promotes their malignant transformation. Mechanistically, early RasV12//M6−/− tumors secrete Pvf1, which activates the Pvr receptor on hemocytes, facilitating their recruitment to the tumor site. These tumor-associated hemocytes secrete the Spätzle (Spz) ligand to activate the Toll receptor within the RasV12 tumors. This enhanced activation of the Toll pathway synergizes with RasV12 to promote malignant transformation through the JNK-Hippo signaling cascade. In summary, our study elucidates the complex interplay between genetically distinct oncogenic cells and between tumors and hemocytes, highlighting how hemocytes exploit the ancient innate immune system to coordinate tumor heterogeneity and drive tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d320162041d03e1b05ee82602e945a36e317508" target='_blank'>
              Hemocytes facilitate interclonal cooperation-induced tumor malignancy by hijacking the innate immune system in Drosophila
              </a>
            </td>
          <td>
            Sihua Zhao, Yifan Guo, Xiaoyu Kuang, Xiaoqin Li, Chenxi Wu, Peng Lin, Qi Xie, Zongzhao Zhai, D. Kong, Xianjue Ma
          </td>
          <td>2025-08-22</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02937-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic cholangiocarcinoma (ICC) have not yet been fully characterized. In this study, we analyzed publicly available single-cell RNA sequencing (scRNA-seq) data derived from paired samples of two intrahepatic cholangiocarcinoma (ICC) tissues and two adjacent normal tissues, thoroughly examining their cellular composition. InferCNV analysis was employed to compare tumor cells and normal cells, and pseudotime analysis was used to identify the growth and differentiation trajectories of the cells. Additionally, intercellular communication analysis was conducted to elucidate the communication networks between cells. Our analysis delineated the cellular ecosystem of ICC, identifying cell subclusters with shared characteristics between ICC and normal tissues. Notably, we characterized a distinct C7-E-T subcluster that exhibited high expression of CXCR4 and BPTF, markers associated with cancer stem cells (CSCs). Further investigation revealed that the MIF intercellular signaling pathway promotes the progression of ICC by activating intracellular signals in the MYC pathway. This study highlights the dysregulation of intercellular signaling pathways within tumor clusters, which influences the onset and progression of ICC. The cancer stem cell subpopulation (CXCR4hiBPTFhiE-T) exerts a significant influence on ICC progression by secreting relevant signaling molecules via the MIF signaling pathway. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17102-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28caecb5ae60429cc1067831e07cb74217728059" target='_blank'>
              Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Xiaowei Gu, Fuzhong Liu, Junfeng Zhang, Nigeerayi Nuermaimaiti, Wenjia Guo, Xiaogang Dong
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Tumor resident microbes are a well-recognized component of the tumor microenvironment. Microbial subcellular location within different tumor types along with their functionality remains to be determined. Bulk microbial profiling techniques, however, lack subcellular and spatial transcriptomic resolution. To address these limitations, we developed advanced quantitative fluorescent imaging methodologies that allowed for visualization of microbial cellular compartmentalization across different tumor types. Furthermore, we performed spatial microbial transcriptomics at the regional and single cell levels to interrogate microbial regulation of signaling within pancreatic cancer cells. To confirm presence of viable microbes within tumors, we performed multiplexed culturomics of patient tissue samples followed by Whole Genomic Sequencing (WGS) analysis. We used in vitro co-culture assays together with in vivo animal experiments for functional biological validation. Overall, our results identified several pathways under microbial regulation within cancer cells that can drive immune evasion through impaired antigen presentation. In summary, using multiple complimentary novel methodologies we characterize the microbial niche of tumors (MiNT) that uncovers microbial regulation of host cells signaling and reshaping of the surrounding tumor microenvironment.



 Vidhi Chandra, Le Li, Seyda Baydogan, Haoyue Liu, Javier Gomez, Steven B. Maron, Daniel Carvajal-Hausdorf, Erick M. Riquelme, Mark W. Hurd, Anirban Maitra, Luis Diaz, Ismet Sahin, Jared Burks, Michael P. Kim, Florencia McAllister. Functional interrogation of the pancreatic intratumoral microbial niche reveals tumor modulation and reshaping of the microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B011.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9d435ae53244dd8930ff7ec47bc24ea3f53880a" target='_blank'>
              Abstract B011: Functional interrogation of the pancreatic intratumoral microbial niche reveals tumor modulation and reshaping of the microenvironment
              </a>
            </td>
          <td>
            Vidhi Chandra, Le Li, Seyda Baydogan, Haoyue Liu, Javier A Gomez, S. Maron, Daniel Carvajal-Hausdorf, Erick Riquelme, M. Hurd, Anirban Maitra, Luis A. Diaz, I. Sahin, Jared K. Burks, Michael P Kim, Florencia McAllister
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Pancreatic cancer, specifically pancreatic ductal adenocarcinoma (PDAC), is notorious for its aggressive nature and dismal prognosis, ranking as a leading cause of cancer-related mortality worldwide. Despite advancements in surgical techniques, chemotherapies, and novel therapeutic approaches, the overall survival rates remain bleak, primarily due to the complex and dynamic tumor microenvironment (TME) which fosters cancer progression and therapeutic resistance. Central to the TME are cancer-associated fibroblasts (CAFs), which exhibit significant heterogeneity and play multifaceted roles in tumor progression, ranging from remodeling the extracellular matrix to modulating immune responses. Recent breakthroughs in single-cell sequencing and spatial omics have unraveled the diverse functional states and spatial organization of various cell types within the TME, including distinct subtypes of CAFs which differ vastly in their influence on pancreatic cancer pathophysiology. This review delves into the intricate interactions between CAFs and pancreatic cancer cells, highlighting how metabolic reprogramming and immune regulation by CAFs contribute to the high malignancy of PDAC. By integrating single-cell and spatial resolution data, we offer new insights into the metabolic crosstalk and immune landscape remodeling driven by CAFs, establishing a foundation for targeting these stromal interactions in therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2add2bd80d02d372a76aab262dddc13647852fe6" target='_blank'>
              Metabolic and immune crosstalk between cancer-associated fibroblasts and pancreatic cancer cells
              </a>
            </td>
          <td>
            Qiyao Zhang, Z. Cao, Shangcheng Yan, Bingrong Chen, Hao Wu, Hongtao Cao, Chen Lin, Ziwen Liu
          </td>
          <td>2025-10-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pancreatic cancer remains a lethal disease despite advances in surgery and systemic treatment in the last two decades, underscoring the urgent need to better understand its biological underpinnings. Despite remarkable advances in the molecular characterization of pancreatic ductal adenocarcinoma (PDAC), clinically actionable biomarkers remain scarce, and current treatment remains empiric. Transcriptomic subtypes such as “classical” and “basal-like” offer some prognostic value, but their ability to guide real-time treatment decisions is limited. In this review, we explore the limitations of current biomarker strategies, in particular subtype-based classifications, and argue for a functional reframing of biomarker development in PDAC, centered on patient-derived organoids (PDOs). We explore four key domains in which PDOs deepen our understanding of therapeutic response and resistance, namely, drug response phenotyping, modeling chemoresistance, incorporating tumor microenvironmental complexity through co-culture systems, and more functional profiling through proteomic and metabolomic approaches. Together, these applications move PDOs beyond static avatars of the tumor to dynamic platforms capable of capturing clinically relevant biology. As functional precision medicine gains traction, PDOs may offer a path to more tailored, responsive treatment strategies in a cancer where new options are urgently needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243db33c2c4b51c3c745a2973296c1c4099759e6" target='_blank'>
              Functional Precision in Pancreatic Cancer: Redefining Biomarkers with Patient-Derived Organoids
              </a>
            </td>
          <td>
            C.A. Chew, Cheng Mun Wun, Yi Fang Lee, C.E. Chee, Khek Yu Ho, Glenn Kunnath Bonney
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality worldwide. However, the spatial and temporal dynamics underlying its development remain poorly characterized. This study employs spatial transcriptomics (ST) to investigate the progression of intestinal tumors in APC Min/+ mice across multiple time points. We identified distinct transcriptional profiles between tumor and normal tissues, resolving six major cell types through integrated dimensionality reduction and pathological annotation. Pseudo-time trajectory analysis revealed increased expression of MMP11 and MYL9 in later stages of tumor progression. Analysis of human CRC cohorts from the TCGA database further confirmed that high expression of these genes is associated with advanced clinical stages and promotes tumor proliferation and invasion. Temporal gene expression dynamics indicated enrichment of cancer-related pathways concurrent with suppression of lipid and amino acid metabolism. Notably, genes in the DEFA family were significantly upregulated in normal tissues compared to tumor tissues. Functional validation showed that DEFA3 inhibits colon cancer cell migration and proliferation in vitro. These demonstrate the value of ST in resolving spatiotemporal heterogeneity in CRC and identify both MMP11/MYL9 and DEFA3 as potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a441608f72da7375d3ddfdc034535ffa428b9ffc" target='_blank'>
              Investigating the Development of Colorectal Cancer Based on Spatial Transcriptomics
              </a>
            </td>
          <td>
            Zhaoyao Qi, Guoqing Gu, Huanwei Huang, Beile Lyu, Yibo Liu, Wei Wang, Xu Zha, Xicheng Liu
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is characterized by profound intratumoral heterogeneity and cellular plasticity, which drive therapeutic resistance. While molecular subtyping into classical and basal-like states has provided a framework, the biology of cells transitioning between these phenotypes remains poorly understood. This study aimed to characterize and validate a transitional, co-expressor cell state at the single-cell, spatial, and proteomic levels to elucidate mechanisms of subtype plasticity. We employed an integrative, multi-platform approach combining single-cell RNA sequencing (scRNA-seq), multiplex immunohistochemistry (mIHC), and laser microdissection with mass spectrometry-based proteomics (LMD-MS) to investigate tumor cells co-expressing the classical marker GATA6 and the basal-like marker KRT17. scRNA-seq analysis revealed that a significant population of tumor cells co-expresses GATA6 and KRT17, existing along a transcriptional continuum between the classical and basal-like poles. Pseudotime trajectory analysis identified a path from the GATA6+ classical state towards the KRT17+/GATA6+ transitional state, governed by the upregulation of pathways including MYC signaling and epithelial-mesenchymal transition (EMT). To validate these findings spatially, mIHC confirmed the presence of KRT17+/GATA6+ double-positive cells, frequently located at the interface between well-differentiated, GATA6-rich glandular structures and poorly differentiated, KRT17-rich infiltrative fronts. Furthermore, LMD-MS was used to provide proteomic confirmation. By isolating pure populations of GATA6+ and double-positive cells, we confirmed that transitional cells are enriched in proteins associated with metabolic reprogramming and cellular motility. Our findings demonstrate that PDAC plasticity is not a simple switch but a dynamic continuum involving a tangible transitional cell state. These co-expressor cells, with their unique metabolic and signaling dependencies, represent a key driver of tumor progression and a potential reservoir for therapeutic resistance. Targeting the specific vulnerabilities of this transitional phenotype offers a promising strategy to overcome plasticity, prevent relapse, and improve therapeutic outcomes in PDAC.



 Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, Natalia Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth Shroyer. Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B122.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a10f45ec5eb895ff99dbba41e30b63f2ad6188" target='_blank'>
              Abstract B122: Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, N.Ye Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth R. Shroyer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell transcriptomics has significantly advanced our ability to uncover the cellular heterogeneity of tumors. A key challenge in single-cell transcriptomics is identifying cancer cells and, in particular, distinguishing them from non-malignant cells of the same cell lineage. Focusing on features that can be measured by single-cell transcriptomics, this review explores the molecular aberrations of cancer cells and their observable readouts at the RNA level. Identification of bona fide cancer cells typically relies on three main features, alone or in combination: i) expression of cell-of-origin marker genes; ii) inter-patient tumor heterogeneity; iii) inferred copy-number alterations. Depending on the cancer type, however, alternative or additional features may be necessary for accurate classification, such as single-nucleotide mutations, gene fusions, increased cell proliferation, and altered activation of signaling pathways. We summarize computational approaches commonly applied in single-cell analysis of tumoral samples, as well as less explored features that may aid the identification of malignant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86abedac67c16f4a560937fab77dcbb2e53869a5" target='_blank'>
              Identification of malignant cells in single-cell transcriptomics data
              </a>
            </td>
          <td>
            M. Andreatta, J. Garnica, Santiago J. Carmona
          </td>
          <td>2025-08-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Cervical cancer (CC) is a highly heterogeneous malignancy primarily driven by persistent infection with high-risk human papillomavirus (HPV). However, comprehensive analyses of heterogeneity in the immune microenvironment, particularly its spatial heterogeneity, between HPV-positive and HPV-negative CC remain limited, despite their critical clinical significance. Methods We performed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) sequencing on collected cervical cancer samples, integrating scRNA-seq, ST, and bulk RNA-seq to analyze distinct cell subtypes and characterize their spatial distribution. Multiplex immunofluorescence analysis was further utilized to validate HPV status-specific expression patterns. Cox regression and LASSO regression analyses were used to identify the prognostic signature on the TCGA dataset. Results Through integrative analysis, we found that HPV-positive samples demonstrated elevated proportions of CD4+ T cells and cDC2s, whereas HPV-negative samples exhibited increased CD8+ T cell infiltration. In HPV-positive CC, epithelial cells acted as primary regulators of cDC2s via the ANXA1-FPR1/3 pathway, with cDC2s subsequently modulating CD4+ T cells and interferon-related CD8+ T cell subtypes. In contrast, HPV-negative CC featured epithelial cells predominantly influencing monocytes and macrophages, which then interacted with CD8+ T cells. Notably, the MDK-LRP1 ligand-receptor interaction emerged as a potential key mechanism for recruiting immunosuppressive cells into CC tumors, fostering an immunosuppressive microenvironment. Further, we constructed a risk score model based on an epithelial cell-related signature (ERS), which was significantly associated with patient survival. Noteworthy variations were observed in immune cell infiltration and immune microenvironment among distinct risk groups. Conclusion Based on integrated multi-omics data, we precisely delineated the spatial transcriptional features of the tumor microenvironment in CC with different HPV statuses, including identifying distinct CD8+ T cell states and cell-cell communication. In addition, we developed an ERS closely associated with the immune environment and prognosis of CC. These results increase our understanding of the molecular mechanisms of cervical cancer under different HPV statuses and provide assistance for the precise treatment of cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5152b47b483e0040237d8137dfb459c7f501aef2" target='_blank'>
              Integrated multi-omics analysis of single-cell and spatial transcriptomics reveals distinct hpv-associated immune microenvironment features and prognostic signatures in cervical cancer
              </a>
            </td>
          <td>
            Qiuyue Su, Xiangdong Tian, Fucheng Li, Xi Yu, Wenchen Gong, Yurong Chen, Jianan Wang, Siqi Yang, Shaojun Zhang, Qian Zhang, Shanshan Yang
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Significance Androgen deprivation therapy (ADT) is a mainstay in prostate cancer (PCa) treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer directions for PCa modeling and pave the way toward therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b171b1d175becacb6af2e7a071be6d1fa5ee752f" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2025-08-25</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>169</td>
        </tr>

        <tr id="Abdominal aortic aneurysm (AAA) is a life-threatening vascular disease with no effective pharmacological interventions. While single-cell transcriptomics has advanced our understanding of AAA, it lacks spatial context. Here, we employed Seq-Scope, an ultra-high-resolution spatial transcriptomic technology, to decipher the spatial landscape of angiotensin II–induced AAA in Apoe–/– mice. Our analysis revealed the heterogeneity of macrophages, fibroblasts, and smooth muscle cells (SMCs), with specific responses in different layers of the AAA tissue. SMCs in the inner layers showed associations with Mgp-expressing fibroblasts and GPNMB-expressing macrophages, whereas the outer layers had different dominant cell types. Notably, GPNMB-expressing macrophages were concentrated near SMCs in regions of severe elastic lamina damage. Immunofluorescent staining confirmed their colocalization, and scRNA-seq reanalysis independently validated the presence of GPNMB-high macrophages in AAA tissues, highlighting their involvement in inflammation and tissue remodeling. Moreover, we discovered that macrophage-derived soluble GPNMB induces SMC phenotypic switching, reducing contractile markers while increasing cytokines and metalloproteinases. This effect was partly mediated by CD44 signaling. These findings suggest that GPNMB-high macrophages contribute to AAA development by driving SMC dysfunction. This study highlights the importance of high-resolution spatial transcriptomics in complementing single-cell transcriptomics, offering valuable insights into molecular and cellular responses in the AAA microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5eda91d1a78d03cc2b4d2c27c04a3ac82f92378" target='_blank'>
              Single-cell spatial transcriptomics unravels the cellular landscape of abdominal aortic aneurysm
              </a>
            </td>
          <td>
            Guizhen Zhao, Chun-Seok Cho, Hongyu Liu, Yongha Hwang, Yichen Si, Myungjin Kim, Yongjie Deng, Yang Zhao, Chao Xue, Yanhong Guo, Lin Chang, D. Mizrak, Bo Yang, H. Kang, Jifeng Zhang, Jun Hee Lee, Y. Chen
          </td>
          <td>2025-08-22</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive disease with one of the lowest five-year survival rates of all cancers. Low grade, premalignant pancreatic intraepithelial neoplasias (PanINs) are common in the general population, but despite expression of mutant KRAS, the vast majority of these lesions will never progress to cancer. Understanding how additional genetic or epigenetic changes as well as changes to the tumor microenvironment contribute to PanIN to PDAC progression is critical and has the potential to reveal mechanisms of interception for this deadly disease. Our previous work identified the prolyl isomerase PIN1 as a critical factor in the tumor microenvironment for growth of a subset of pancreatic cancer cell lines. Loss of PIN1 in the tumor microenvironment restricted in vivo growth of multiple KPC cell lines. We identified distinct fibroblast phenotypes in the absence of microenvironmental PIN1 expression or function. These changes in fibroblast state drive significant changes in the extracellular matrix and growth factor expression, both of which impact the crosstalk to and behavior of the neoplastic cells. We are continuing to use conditional PIN1 knockout mice to interrogate the specific cell types responsible for the observed tumor-restrictive phenotype and are using single cell sequencing to better understand how changes to individual cell types affect multiple cell populations to inhibit tumor growth. In addition, we are comparing the fibroblast phenotypes observed in our mouse models to those found in tumor-promoting versus tumor-restrictive microenvironments in human tissues. These tissues include autopsy pancreata with low grade PanIN lesions as well as PDAC resection specimens with low and high grade PanINs. We are creating organoid and cell line models of epithelial and stromal cells from normal human pancreata to support complex co-culture engineered models of early disease progression. By integrating across mouse and human model systems, we strive to understand the crosstalk that permits progression of pancreatic cancer and to identify targets to intercept this process.



 Tana Gazdik, Madeline Kuhn, Chloe Bowman, Eric Carlson, Kyra Lindley, Ellen M. Langer. Interrogating microenvironmental changes that impact early pancreatic cancer development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4475acff7b0cbed38743a8484c9c3b1b2adb82c" target='_blank'>
              Abstract A031: Interrogating microenvironmental changes that impact early pancreatic cancer development
              </a>
            </td>
          <td>
            Tana Gazdik, Madeline R Kuhn, Chloe L Bowman, Eric J Carlson, Kyra A Lindley, Ellen M. Langer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The tumor microenvironment drives many malignant features of pancreatic ductal adenocarcinoma (PDAC). The fibroblasts within pancreatic tumors promote tissue remodeling, immune suppression, and resistance to therapy. However, the interactions between stromal populations and pancreatic cancer cells are less understood in the liver, the most frequent site of PDAC metastasis. To address this, we employ single cell transcriptomics to compare primary pancreatic vs. liver PDAC lesions. Here, we identify the expression of hepatocyte growth factor (HGF) in fibroblasts and its receptor MET in cancer cells are both markedly increased in the PDAC liver niche. Using functional assays, we validate that mitogenic MET signaling is activated in PDAC cells by liver-derived fibroblasts. Importantly, the inhibition of MET signaling leads to reduced tumor growth in immune competent mouse models. Collectively, our data demonstrates that liver stromal-epithelial crosstalk networks engage in signaling pathways distinct from primary pancreatic tumors, highlighting opportunities to develop new treatments for metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aa94d2601cca690cf0f25d219abda02e990ca8c" target='_blank'>
              Comparative Analysis of Primary and Liver Fibroblasts Reveals MET as a Potent Target in Pancreatic Cancer Metastasis
              </a>
            </td>
          <td>
            Rima Singh, Natalie Yousefian, Walker M. Allen, Cecily Anaraki, Alica K Beutel, Sabrina E. Calderon, Ian M Loveless, Oliver McDonald, Jennifer P Morton, D. Imagawa, Zeljka Jutric, Thomas F. Martinez, Nina G. Steele, Christopher J Halbrook
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Liver metastases represent a leading cause of mortality in pancreatic ductal adenocarcinoma (PDAC). Recent clinical evidence has revealed distinct immune ecosystems in the liver tumor microenvironment, yet mechanistic insight into the cellular and molecular mechanisms driving those diverse immune landscapes remains unclear. Here we report, computational cellular signaling analysis of single-cell RNA sequencing data from matched primary and liver metastatic samples isolated at early and late time points from the spontaneous genetically engineered mouse model KPC (Pdx1Cre; KrasLSL-G12D; Tp53LSL-R172H). This analysis uncovered several immunomodulatory axes between cancer cells-T cells and cancer cells- macrophages. Validation of these findings were performed using primary tumor cell lines, isolated from the same or littermate tumors as the scRNA-seq dataset. Bulk RNA-seq analysis of these cell lines showed a retention of cancer-derived ligands in culture. In vivo validation using the hemisplenic liver metastatic transplant model confirmed that cell lines isolated from primary tumors, which metastasized spontaneously to the liver, retained liver metastatic outgrowth potential. Flow cytometry and multiplex cytokine profiling of isolated cell lines, non-involved liver, and liver tumors revealed distinct immune population shifts within the metastatic niche and confirmed cancer cell derived immunomodulatory cytokines. This study highlights the interplay between cancer-intrinsic signaling and the immune landscape in PDAC liver metastases and supports future mechanistic exploration.



 Ayushi Mandloi, Jace Baines, Meet Patel, Christina R. Larson, Robert S. Welner, Julienne L. Carstens. Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A081.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fce5d9cdc25fe3d365bf9aa1c45eddedfbe9968" target='_blank'>
              Abstract A081: Dissecting the Tumor–Immune Landscape of PDAC Liver Metastases to Identify Candidate Drivers of Immune Evasion and Metastatic Outgrowth
              </a>
            </td>
          <td>
            Ayushi Mandloi, Jace Baines, Meet Patel, Christina R Larson, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous cancer with complex tumor–immune interactions. This heterogeneity, particularly in tumor and immune cells, complicates treatment and prognostic evaluation. Although recent studies have revealed distinct tumor cell states and immune dysfunction in HCC, the molecular basis underlying tumor aggressiveness remains poorly understood. A deeper understanding of the molecular and functional diversity of both tumor and immune cell populations, especially the identification of stem-like tumor subpopulations and immunosuppressive mechanisms, along with the development of robust prognostic biomarkers, is essential for advancing precision oncology and improving clinical outcomes.We integrated three publicly available single-cell RNA sequencing (scRNA-seq) datasets from GEO to delineate the cellular architecture of the HCC tumor microenvironment. Unsupervised clustering and dimensionality reduction were employed to identify major cell types and tumor subpopulations. Functional annotation was performed using canonical markers, Monocle, CytoTRACE, and AUCell scoring. H2AFZ was identified as a candidate oncogene and validated through in vitro knockdown experiments. The interaction between T cell subsets and tumor subpopulations were further characterized. A prognostic risk model was constructed using LASSO regression.Six major cell types were identified in HCC TME. Tumor cells were subdivided into three distinct clusters: Tumor_C0, Tumor_C1 and Tumor_C2. Tumor_C2 showed the highest stemness, pro-metastatic activity and immunogenic cell death signatures. H2AFZ was highly expressed in Tumor_C2 and associated with poor prognosis. The knockdown of H2AFZ reduced H2A.Z protein levels, inhibited proliferation, invasion, and induced apoptosis. T cell analysis revealed five subpopulations. It was found that Tumor_C2 interacts with the proliferative and exhausted T cell subpopulations, suggesting a potential functional relationship between them. The prognostic model based on tumor_C2 transcriptomic features effectively stratified patient survival across multiple cohorts, with robust AUCs and Kaplan-Meier survival distinctions.We identified a proliferative, stem-like tumor cell subpopulation (Tumor_C2) in HCC characterized by high H2AFZ expression, which drives tumor aggressiveness. T cell analysis revealed significant interactions with Tumor_C2. Moreover, a prognostic model based on Tumor_C2 features effectively stratified patient survival across multiple cohorts. Together, these findings highlight potential therapeutic targets for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7024320dd03972a4da66c548f19e56393e35ae99" target='_blank'>
              Single-cell transcriptomics identifies an H2AFZ-driven proliferative tumor subpopulation associated with poor prognosis in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jihui Huo, Tao Yang, Kai Lei, Zeyao Wang, Zebin Chen, Qi Zhou
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Breast cancer brain metastases are associated with a poor prognosis, and there is a high demand for improved personalized treatment strategies. It is still poorly understood which breast cancer brain metastases patients benefit from immune checkpoint blockade.



 Here, we multimodally characterized the immune niche of breast cancer brain metastases at single-cell resolution in a clinically annotated patient cohort (n = 156), combining multiplexed imaging and cell type quantification by tissue cytometry, single-nuclei RNA-sequencing, flow cytometry, spatial transcriptomics, and functional studies in patient-derived organoids.



 Overall, infiltration of breast cancer brain metastases by CD8 T cells was strongly heterogeneous between patients, and high numbers were prognostically favorable. In-depth characterization of cell types and integrative analyses including clinical data revealed distinct immune landscapes across patients, two of which independently predicted prolonged overall survival: i) Breast cancer brain metastases containing T and B cells accumulating together with characteristic cell communities in stromal tertiary lymphoid structures, and ii) breast cancer brain metastases characterized by a high content of tissue resident-like CD8 memory T cells (CD8 TRM-like cells) spatially enriched within tumor islands. CD8 TRM-like cells exhibited high levels of activation, checkpoint, and functional markers and showed tumor cell killing capacity ex vivo, suggesting them to be critical players of tumor immune control.



 Together, our work provides novel insights into anti-tumor immunity in breast cancer brain metastases and reveals prognostic immune biomarkers with potential translational relevance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb6c7f193a5bf8137e26dc814e6139f92f208a4" target='_blank'>
              KS03.5.A DISSECTING THE CELLULAR ARCHITECTURE OF BREAST CANCER BRAIN METASTASES REVEALS PROGNOSTICALLY DISTINCT IMMUNE LANDSCAPES
              </a>
            </td>
          <td>
            L. Jassowicz, F. Feng, R. Warta, F. Sor, J. Soller, A. Wöllner, C. Lotsch, J. Wong, J. Schlegel, L. Frank, P. Dao Trong, G. Jungwirth, A. Nohman, K. Maaß, M. Barthel, A. Unterberg, S. Krieg, K. Rippe, A. Abdollahi, A. von Deimling, J. Malm, P. Lichter, M. Zapatka, M. Seiffert, C. Herold-Mende
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="


 Emerging clinical evidence highlights the importance of tumor infiltration of activated immune cells, particularly T and B lymphocytes, for the efficacy of immune-based cancer therapies. However, many patients show limited immune cell presence in the tumor microenvironment. High endothelial venules (HEVs) are specialized blood vessels that enable lymphocyte trafficking from circulation into lymph nodes or tumors. Histological analyses of clinical tumor samples indicate that HEVs are positioned near dense aggregates of B and T lymphocytes, known as tertiary lymphoid structures (TLS), underscoring their role in immune cell recruitment and anti-tumor immunity. The abundant development of TLS and HEVs is associated with clinical response to immune checkpoint inhibition therapy and cancer survival. Recently, we found that simultaneous activation of innate immune effectors, STING and lymphotoxin-β receptor (LTβR) using their agonists induces the formation of TLS and HEVs in TLS/HEV-free tumors. In the current study, mouse pancreatic ductal adenocarcinoma (KPC) tumors were grown in the dorsal skin fold window chamber (DSWC) and received STING/LTβR agonist combination therapy to monitor the real-time TLS/HEV development and dynamic tumor microenvironment in intravital microscopy (IVM). Transgenic mice with fluorescent red B cells and green fluorescent CD31+ blood vessels were infused with MECA-79 antibody (HEV marker) to visualize B cell trafficking through HEVs. Four-color confetti mice crossed with B cell-specific Cre mice, allowed us to analyze the clonal expansion of B cells expressing one of four colors, with daughter cells retaining their parent color. Our findings demonstrated that tumoral HEVs closely resemble those found in lymph nodes, characterized by fewer branches, longer segment lengths, and increased diameters compared to non-HEV tumor blood vessels, which typically exhibit chaotic patterns. Circulating B cells traveling through tumoral HEVs displayed significantly slower rolling velocities than non-HEV tumor vessels, likely due to enhanced integrin and L-selectin expression induced by the combination therapy. By repeated intravital imaging, we were also able to observe a quadruple increase in B cell number in TLS during the day 4-7 treatment time frame. Following adoptive transfer of splenic leukocytes from naïve donor mice, we found that these leukocytes preferentially homed to TLS-rich areas, with B cells constituting the majority of the recruited cells. Importantly, analyses using the confetti mouse model indicate that tumor infiltrating B cells underwent clonal expansion in TLS, with different TLS having different colors. Overall, our study demonstrates that HEVs are the gateways for B cell entry to tumors, and the intratumoral TLS formation is driven by localized clonal expansions of these B cells, resembling lymph node germinal center activities, which drive the development of humoral immunity against tumors.



 Gabrielle Rowe-Brown, Masanobu Komatsu. B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36afed9d0acad24d10324fa459b1bfa3a197ed1" target='_blank'>
              Abstract A010: B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy
              </a>
            </td>
          <td>
            Gabrielle Rowe-Brown, Masanobu Komatsu
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Retinoblastoma is the most common intraocular malignant tumor in childhood. Although current treatments offer a high survival rate, treatment toxicity, tumor relapse, and treatment resistance require a deeper understanding of the disease mechanisms to develop adapted therapies. Microscopically, this tumor is characterized by different states of differentiation and proliferation, ranging from poorly differentiated to well-differentiated retinoblastoma. Retinocytoma, on the other hand, is a benign non-proliferative tumor. Recent next-generation multi-omics analyses classified retinoblastoma tumors in subtypes 1 or 2, subtype 2 presenting a later age of onset, more genetic alterations, and higher metastatic potential. In parallel, several single-cell transcriptomics studies demonstrated intratumoral heterogeneity. However, mapping the different cell populations directly on the tumor and comparing histological features and molecular subtypes remains an unmet need. Methods Spatial transcriptomics was used to characterize a primary enucleated case with two histologically distinct areas. The whole transcriptomic profile of sixteen regions of interest, covering the two differentiation patterns of the tumor and the non-tumoral retina, was analyzed. Results The clustering of the regions of interest based on whole transcriptome data correlated with the histological description: cluster 1 (6 regions of interest) corresponded to highly differentiated areas and cluster 2 (6 regions of interest) to poorly differentiated areas. They showed enrichment for phototransduction and proliferation respectively, confirmed by immunohistochemistry for markers of these pathways. The publicly available molecular signatures of the two retinoblastoma subtypes categorized our regions of interest into two groups, which correlated perfectly with histological observation and transcriptomic profiles. Further analysis of the expression of specific senescence markers in the well-differentiated area did not support the diagnosis of retinocytoma as it did not confirm the expected up-regulation seen in this tumor type. Conclusions This study demonstrates a strong correlation between histological observation and molecular profiling, representing the first mapping of retinoblastoma composed of adjacent molecular subtypes 1 and 2. It also highlights the diagnostic ambiguity between retinocytoma and well-differentiated non-proliferative subtype 1 retinoblastoma and emphasizes the need for specific biomarkers to differentiate these two tumor types. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14814-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/599058c9510122a1b04d9c9b60c21f33e8817b4e" target='_blank'>
              Spatial transcriptomics of retinoblastoma: a visual window on intra-patient heterogeneity
              </a>
            </td>
          <td>
            A. Moulin, J. Thevenet, L. Mazzeo, S. Tissot, C. Stathopoulos, F. L. Munier, A. Berger
          </td>
          <td>2025-09-02</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf8e063a68a88a96c8de0cef9efb9254b78c4b6" target='_blank'>
              A cellular and spatial atlas of TP53-associated tissue remodeling defines a multicellular tumor ecosystem in lung adenocarcinoma.
              </a>
            </td>
          <td>
            William Zhao, Thinh Nguyen, Atharva Bhagwat, Akhil Kumar, Bruno Giotti, Benjamin Kepecs, Jason L. Weirather, N. Mahadevan, Åsa Segerstolpe, Komal Dolasia, Jamshid Abdul-Ghafar, Naomi Besson, Stephanie M Jones, Brian Y. Soong, Chendi Li, S. Vigneau, M. Slyper, Isaac Wakiro, Mei-ju Su, K. Helvie, Allison Frangieh, Judit Jané-Valbuena, Orr Ashenberg, Mark Awad, Asaf Rotem, Raphael Bueno, O. Rozenblatt-Rosen, K. Pfaff, Scott J. Rodig, Aaron N. Hata, Aviv Regev, B. Johnson, A. Tsankov
          </td>
          <td>2025-10-07</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Background Human papillomavirus (HPV)-associated oropharyngeal cancer (OPC) is increasing in prevalence, but the drivers of metastasis remain poorly understood, which impacts the ability to personalise management decisions. Much of the genomic research to date focuses on the HPV-negative population. Here, we utilise single-cell and spatial single-cell techniques to understand the drivers of metastasis. Methods Patients with HPV-positive OPC and cervical lymph node metastases treated with curative surgery had matched samples from the primary and lymph nodes collected for research. Single-cell RNA sequencing, single-cell spatial sequencing (Visium) and in-situ spatial platforms were performed. Cancer clones were delineated using inferred copy number variation. Expression phenotypes and interactions with the tumour microenvironment were compared between the metastasising and non-metastasising cancer clones. Results Individual cancer clones have varied abilities to metastasise and undergo clonal expansion in the lymph node, with only a subset of clones present in the primary expanding in the lymph node. Four mechanisms were identified as defining the metastatic phenotype, including protein translation adaptation, immunoproteasome dysfunction and immune evasion, suppression of the IFN immune response and cap-independent protein translation. Conclusions This research elucidates multiple mechanisms driving the expansion of cancer clones in HPV-positive oropharyngeal cancer. By detailing the roles of translational adaptation, immunoproteasome dysfunction, suppression of the interferon immune response and cap-independent protein translation, we provide insights into how these processes contribute to immune evasion and tumour survival. Supplementary Information The online version contains supplementary material available at 10.1186/s12916-025-04392-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fda352f2dd3f74c2b6cb392920d418565101f0" target='_blank'>
              Protein translation dysregulation and immune cell evasion mediated by IFN and immunoproteasome downregulation define metastatic clones in HPV-related cancer of the oropharynx
              </a>
            </td>
          <td>
            Venessa T. Chin, W. Muskovic, Rachael A. McCloy, Drew R Neavin, José Alquicira-Hernández, Himanshi Arora, A. Senabouth, Kavitha Krishna Sudhakar, Patricia Keith, Ellie Spenceley, Dominik Kaczorowski, Peter Floros, Brett Leavers, Julia Crawford, Richard Gallagher, Joseph E. Powell
          </td>
          <td>2025-10-14</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The cellular plasticity of neuroblastoma is defined by a mixture of two major cell states, adrenergic and mesenchymal, which may contribute to therapy resistance. However, how neuroblastoma cells switch cellular states during therapy remains largely unknown, and how to eradicate neuroblastoma regardless of its cell state is a clinical challenge. To better understand the cellular plasticity of neuroblastoma in chemoresistance, we define the transcriptomic and epigenetic map of adrenergic and mesenchymal types of neuroblastomas using human and murine models treated with indisulam, a selective RBM39 degrader. We show that cancer cells not only undergo a bidirectional switch between adrenergic and mesenchymal states, but also acquire additional cellular states, reminiscent of the developmental pliancy of neural crest cells. These cell state alterations are coupled with epigenetic reprogramming and dependency switching of cell state–specific transcription factors, epigenetic modifiers, and targetable kinases. Through targeting RNA splicing, indisulam induces an inflammatory tumor microenvironment and enhances the anticancer activity of natural killer cells. The combination of indisulam with anti-GD2 immunotherapy results in a durable, complete response in high-risk transgenic neuroblastoma models, providing an innovative, rational therapeutic approach to eradicate tumor cells regardless of their potential to switch cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8bfb0658591ed4d276e6df844046df62c0e809" target='_blank'>
              RBM39 degrader invigorates innate immunity to eradicate neuroblastoma despite cancer cell plasticity
              </a>
            </td>
          <td>
            Shivendra Singh, Jie Fang, Hongjian Jin, Lee-Ann Van de Velde, Andrew Cortes, Jiani Chen, Sivaraman Natarajan, E. Poon, Qiong Wu, Christopher L. Morton, Mary A. Woolard, Waise Quarni, Jacob A Steele, Jon P Connelly, Liusheng He, Rebecca Thorne, Gregory Turner, Thomas Confer, Melissa Johnson, W. Caufield, Burgess Freeman, Timothy Lockey, A. Murphy, P. J. Murray, Takashi Owa, S. Pruett-Miller, Ruoning Wang, L. Chesler, Julie R. Park, A. Davidoff, John Easton, Xiang Chen, Paul G. Thomas, Jun Yang
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Medullary thyroid carcinoma (MTC) is a rare neuroendocrine tumor arising from parafollicular C-cells in the thyroid. Unlike other neuroendocrine tumors, knowledge about the MTC tumor microenvironment and its role in therapy resistance remains limited, mainly due to the lack of reliable in vitro models. We hypothesized that, as observed in other cancer types, MTC cells may exhibit a degree of phenotypic plasticity that could be modulated by various microenvironmental factors.



 The primary aim of this study was to establish patient-derived MTC in vitro models capable of growing under different three-dimensional (3D) culture conditions while retaining the ability to shift toward a more mesenchymal, stem-like phenotype.



 We successfully established eight patient-derived primary MTC cell lines from tumors with different genetic backgrounds and variable degrees of aggressiveness. These lines were characterized in terms of phenotype and responses to different microenvironmental stimuli and targeted therapies through Western Blot, confocal microscopy, ELISA assays, ELDA, RT-qPCR, and proliferation assays.



 Notably, the cells maintained their plasticity over extended passaging when cultured under stem-promoting conditions. In differentiation media, they preserved the ability to secrete high levels of classic MTC markers, such as calcitonin and chromogranin A. Under 3D culture conditions, all cell lines showed a significant increase in stem and neuroendocrine precursor markers, including SOX2, OCT4, TBB3 and FOXA1, along with spatially dependent shifts in Epithelial-to- Mesenchymal Transition (EMT) markers E-cadherin and Vimentin. These phenotypic variations partially recapitulated the molecular alterations observed in MTC tissue samples. Moreover, changes in culture conditions—such as inducing a pseudo- hypoxic state or culturing in non-adherent environments—profoundly affected both cell phenotype and sensitivity to therapeutic agents currently available for MTC treatment, including Pralsetinib, Selpercatinib, Cabozantinib, Lenvatinib, and Vandetanib.



 In conclusion, we present here the generation of a new valid patient- derived in vitro model that can provide new opportunities to explore MTC biology complexity and will allow future translational studies, including those involved in uncovering the mechanisms underlying the insurgence of therapy resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0c689353bda0dac4479e679a592360fdc7492a4" target='_blank'>
              OC48 - Generation of patient-derived in vitro models to unravel medullary thyroid cancer microenvironment and therapy resistance
              </a>
            </td>
          <td>
            V. Ghiandai, V. Cirello, G. Gazzano, G. Dionigi, C. Colombo, L. Fugazzola, L. Persani, E. S. Grassi
          </td>
          <td>2025-09-01</td>
          <td>European Journal of Endocrinology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Metastasis remains the principal cause of cancer-related mortality, yet its distribution across organs is far from random. Instead, tumor cells exhibit organotropism, a consistent preference for colonizing specific distant tissues, a phenomenon shaped by anatomical constraints, molecular crosstalk, and microenvironmental compatibility. Far beyond mere mechanical entrapment in vascular beds, metastatic dissemination reflects a coordinated interplay between tumor-intrinsic programs and organ-specific niches. Tumor-derived extracellular vesicles, cytokines, and matrix-remodeling enzymes actively precondition distant sites through pre-metastatic niche formation, creating permissive microenvironments primed for colonization. Simultaneously, tissue-specific immune landscapes, stromal compositions, and mechanical architectures determine the fate of disseminated tumor cells, whether they are eliminated, enter dormancy, or form macrometastases. Phenotypic plasticity, metabolic reprogramming, and immune evasion mechanisms equip subclones with the capacity to exploit these unique niches. Across cancer types, reproducible patterns of organotropic metastasis not only guide clinical surveillance and therapeutic stratification but also reveal vulnerabilities in the metastatic cascade. This review synthesizes emerging mechanistic insights across anatomical, immunological, and molecular domains to construct a comprehensive framework of organotropism, highlighting therapeutic opportunities to intercept metastasis at organ-specific checkpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7663dab540302726b6f7b6337c8bf27a6cf336" target='_blank'>
              Deciphering organotropism reveals therapeutic targets in metastasis
              </a>
            </td>
          <td>
            Jhommara Bautista, Andrés López-Cortés
          </td>
          <td>2025-10-10</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pancreatic cancer is an aggressive disease with a dense fibrotic stroma and is often accompanied by chronic inflammation. Peritumoral inflammation is typically viewed as a reaction to nearby tumor growth. Here, we report that the inflamed pancreatic lobules are frequently invaded by tumor cells, forming a distinct, non-fibrotic tumor niche. Using a semi-supervised machine learning approach for annotations of clinical samples and multiplex protein profiling, we show that tumor cells at the invasion front are closely associated with acinar cells undergoing damage-induced changes, and with activated fibroblasts expressing markers of injury. The invaded lobules are linked to classical tumor phenotypes, in contrast to fibrotic areas where tumor cells display a more basal profile, highlighting microenvironment-dependent tumor subtype differences. In female mice, lobular invasion similarly aligns with the classical tumor phenotype. Together, our data reveal that pancreatic tumors colonize injured lobules, creating a unique niche that shapes tumor characteristics and contributes to disease biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33fdbbbab057fd0e654aba5735a7c83d83362e84" target='_blank'>
              An injury-associated lobular microniche is associated with the classical tumor cell phenotype in pancreatic cancer
              </a>
            </td>
          <td>
            Sara Söderqvist, Annika Viljamaa, Natalie Geyer, Anna-Lena Keller, Kseniya Ruksha, Carina Strell, Neda Hekmati, Alexandra Niculae, J. Engstrand, E. Sparrelid, Caroline Salmén, Tânia D. F. Costa, Miao Zhao, Staffan Strömblad, Argyro Zacharouli, P. Ghorbani, Sara Harrizi, Yousra Hamidi, Olga Khorosjutina, Stefina Milanova, Bernhard Schmierer, Béla Bozóky, C. Fernández Moro, Marco Gerling
          </td>
          <td>2025-09-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Breast cancer progression is a consequence of intricate dynamics between cells and their matrix in the tumor microenvironment. However, most in vitro models are not amenable to studying the behavior of multiple cell types within a defined matrix architecture. In this study, we demonstrate a microporous matrix where breast cancer cells and adipose derived stromal cells are integrated to evaluate crosstalk between matrix parameters and heterotypic cell populations. To do this, we leveraged two biofabrication techniques, granular suspension matrices and drop-on-demand bioprinting, to deposit multiple cell types in a reproducible format amenable to high-throughput screening. 3D gelatin-methacryloyl microgels were used to create a yield stress granular suspension bath with tunable interstitial volume to mimic the porosity and densities of healthy and fibrotic microenvironments. Invasive and non-invasive breast cancer cells (MCF-7 and MDA-MB-231) were bioprinted at the interface of the ADSC-laden granular matrix to probe invasive processes and heterotypic crosstalk as a tumor–stroma model. We focused on cancer cell migration through model fibrotic tissue and ADSC transformations into cancer associated fibroblasts. α-smooth muscle actin expression indicated that the high density microgel matrices are more conducive to ADSC-CAF transformations, which in turn influenced the expression of molecular markers associated with tumorigenicity and chemoresistance in the resident cancer cells. Treatment with doxorubicin supported increased tumorigenicity in the co-cultures. Together, this work demonstrates how defined microengineered matrices can serve as platforms to evaluate cell behavior, with scope for translation to in vitro assays for biological discovery and drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbed15b3d1cf440dd524e75b1029862768a9edc2" target='_blank'>
              A biofabricated 3D cancer-stroma tumor microenvironment model
              </a>
            </td>
          <td>
            S. Romanazzo, Peilin Tian, Gagan K Jalandhra, Riddhesh Doshi, J. Gooding, K. Kilian
          </td>
          <td>2025-09-23</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The metastatic tumor microenvironment (TME) is a highly dynamic and heterogeneous ecosystem that plays a critical role in promoting cancer cell colonization, immune escape, and resistance to therapy. Recent advances in multi-omics technologies—including genomics, transcriptomics, epigenomics, proteomics, and metabolomics—have enabled a systems-level understanding of the molecular reprogramming that occurs in the TME following metastasis. In this review, we systematically summarize emerging findings from recent multi-omics studies that dissect cellular composition, signaling pathways, immune landscape, and metabolic rewiring within the metastatic TME. We highlight key molecular signatures and intercellular interactions that drive metastatic progression and therapy resistance. Furthermore, we discuss how integrative multi-omics data are being leveraged to identify actionable targets and to design novel immunotherapeutic and molecular precision strategies tailored to the metastatic niche. These insights provide a scientific rationale for the development of TME-targeted approaches in the treatment of advanced-stage cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1202cb3e5acfc52d10d5fc247d4fca996a2d6f2e" target='_blank'>
              Multi-omics analysis of the tumor microenvironment after metastasis: advancing toward personalized immunotherapy and molecular targeted strategies
              </a>
            </td>
          <td>
            Qing Gao, Huqiang Wu, Li Duan, Guanhua Lv, Dongmei Quan
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Primary prostate cancer presents with multifocal lesions and unpredictable clinical behavior, posing significant challenges for effective prognosis. To address this, we investigate the epigenomic landscape of prostate tumor biopsies from 25 treatment-naïve male patients by analyzing chromatin compartmentalization patterns. Our analysis reveals two distinct molecular subtypes: one with a Low Degree of Decompartmentalization (LDD) and another with a High Degree of Decompartmentalization (HDD). Here we show that the HDD subgroup exhibits extensive chromatin reorganization associated with diminished oncogenic potential. This subtype shows repression of molecular pathways involved in extracellular matrix remodeling and cellular plasticity. From this distinction, we derive an 18-gene transcriptional signature capable of differentiating HDD from LDD cases. Importantly, we validate the prognostic relevance of this signature in multiple independent cohorts totaling over 900 patients. Our findings suggest that epigenetic-derived signature at the time of diagnostic biopsy can offer a powerful tool for risk stratification in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696cb38bd331720bed277dcfa9f4701c78f389c7" target='_blank'>
              Chromatin remodeling restrains oncogenic functions in prostate cancer
              </a>
            </td>
          <td>
            V. Rosti, Giovanni Lembo, C. Petrini, Francesca Gorini, Roberto Quadri, C. Cordiglieri, Margherita Mutarelli, Elisa Salviato, Elisabetta Casari, Emanuele Di Patrizio Soldateschi, Emanuele Montanari, G. Albo, Francesco Ripa, Alessandra Fasciani, Mariacristina Crosti, E. De Lorenzis, M. Maggioni, V. Vaira, Maria Vivo, Francesco Ferrari, Chiara Lanzuolo
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Emerging clinical evidence highlights the importance of tumor infiltration of activated immune cells, particularly T and B lymphocytes, for the efficacy of immune-based cancer therapies. However, many patients show limited immune cell presence in the tumor microenvironment. High endothelial venules (HEVs) are specialized blood vessels that enable lymphocyte trafficking from circulation into lymph nodes or tumors. Histological analyses of clinical tumor samples indicate that HEVs are positioned near dense aggregates of B and T lymphocytes, known as tertiary lymphoid structures (TLS), underscoring their role in immune cell recruitment and anti-tumor immunity. The abundant development of TLS and HEVs is associated with clinical response to immune checkpoint inhibition therapy and cancer survival. Recently, we found that simultaneous activation of innate immune effectors, STING and lymphotoxin-β receptor (LTβR) using their agonists induces the formation of TLS and HEVs in TLS/HEV-free tumors. In the current study, mouse pancreatic ductal adenocarcinoma (KPC) tumors were grown in the dorsal skin fold window chamber (DSWC) and received STING/LTβR agonist combination therapy to monitor the real-time TLS/HEV development and dynamic tumor microenvironment in intravital microscopy (IVM). Transgenic mice with fluorescent red B cells and green fluorescent CD31+ blood vessels were infused with MECA-79 antibody (HEV marker) to visualize B cell trafficking through HEVs. Four-color confetti mice crossed with B cell-specific Cre mice, allowed us to analyze the clonal expansion of B cells expressing one of four colors, with daughter cells retaining their parent color. Our findings demonstrated that tumoral HEVs closely resemble those found in lymph nodes, characterized by fewer branches, longer segment lengths, and increased diameters compared to non-HEV tumor blood vessels, which typically exhibit chaotic patterns. Circulating B cells traveling through tumoral HEVs displayed significantly slower rolling velocities than non-HEV tumor vessels, likely due to enhanced integrin and L-selectin expression induced by the combination therapy. By repeated intravital imaging, we were also able to observe a quadruple increase in B cell number in TLS during the day 4-7 treatment time frame. Following adoptive transfer of splenic leukocytes from naïve donor mice, we found that these leukocytes preferentially homed to TLS-rich areas, with B cells constituting the majority of the recruited cells. Importantly, analyses using the confetti mouse model indicate that tumor infiltrating B cells underwent clonal expansion in TLS, with different TLS having different colors. Overall, our study demonstrates that HEVs are the gateways for B cell entry to tumors, and the intratumoral TLS formation is driven by localized clonal expansions of these B cells, resembling lymph node germinal center activities, which drive the development of humoral immunity against tumors.



 Gabrielle Rowe-Brown, Masanobu Komatsu. B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr PR-11.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c11ba138005b19c5e9d04c0e07daa4c79af69fe8" target='_blank'>
              Abstract PR-11: B cell infiltration and clonal expansion in tertiary lymphoid structures: Insights from intravital microscopy
              </a>
            </td>
          <td>
            Gabrielle Rowe-Brown, Masanobu Komatsu
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Background and Aims The relationship between chronic liver disease and liver cancer remains poorly understood, and treatment options for advanced liver disease remain limited. This study aims to elucidate the dynamic evolution of cellular and molecular alterations from normal liver to diseased liver. Methods Single‐cell RNA sequencing was performed to profile the dynamic cellular variations from normal liver to diseased liver. Bioinformatics analyses were employed to assess tumour heterogeneity and the evolution of tumour microenvironment. Molecular biology validations were performed to investigate the findings. Results Through single‐cell RNA sequencing, we constructed a human liver landscape consisting of more than 130 000 single cells, from normal, cirrhotic tissue, primary liver cancer and paired adjacent tissues. We profiled both inter‐ and intra‐tumour heterogeneity of hepatocellular carcinoma and intrahepatic cholangiocarcinoma. A dual role of the pro‐fibrosis and pro‐carcinogenesis FABP5high macrophage subset was shown to be conserved throughout the progression of liver fibrosis and carcinogenesis; markers, spatial localization, origin, and functional pathways associated with FABP5high macrophages were identified through in vitro and in vivo validation. Metabolic reprogramming of these conserved FABP5high macrophages is implicated in the formation of cirrhotic and cancerous niches. Additionally, we also identified a subset of RGCC+COL4A1+ endothelial cells enriched in HCC tissues, which might experience endothelial‐mesenchymal transition and orchestrate the proliferation and invasion of cancer cells via angiogenesis. Conclusions Our findings provide insights into the evolution from chronic liver disease to liver cancer. These insights will contribute to the development of novel cell subset‐based therapeutics based on immunotherapy and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec8b08f4e01c5f59d84102994dc20c75fd98f8a8" target='_blank'>
              A Conserved FABP5high Macrophage Subset Promotes Fibrosis and Carcinogenesis in Advanced Liver Disease
              </a>
            </td>
          <td>
            Wenhao Ge, Lifeng Ma, Yu Huang, W. E., Shujing Lai, Bijun Cui, Jingjing Wang, Yutian Ji, Daiyuan Liu, Song Ye, Yang Kong, Jiarong Zhou, Chengxuan Yu, Yingsheng Wu, Qiang Sun, Dongkai Zhou, Wei Zhu, Yuan Ding, Xiaoping Han, Guoji Guo, Weilin Wang
          </td>
          <td>2025-08-28</td>
          <td>Liver International</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Simple Summary Hepatoblastoma (HB), the most common childhood liver cancer, often shows aberrant expression and/or mutations of the β-catenin (B), YAP (Y) and/or NRF2 (N) transcription factors and can be modeled in mouse livers by overexpressing combinations of these (e.g., BY, BN, YN and BYN). Because very few human or murine HB cell lines exist, genetically defined murine HB cell lines would be a significant advance. We describe here the generation of cell lines from primary BN and YN tumors. Unexpectedly, one BN line shows a remarkable ability to trans-differentiate into endothelial cells and can accelerate in vivo tumorigenesis and angiogenesis. These and previously derived BY and BYN lines have similar sensitivities to four drugs used to treat HB. Our generic approach for cell line derivation should allow for the generation of additional lines driven by less common drivers. A collection of such genetically defined and well-characterized cell lines will facilitate studies that are impractical to perform in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31da89c86be409fe01133803515bcf28ea7e0273" target='_blank'>
              Derivation of Genetically Defined Murine Hepatoblastoma Cell Lines with Angiogenic Potential
              </a>
            </td>
          <td>
            Keyao Chen, Ahmet Toksoz, Colin Henchy, Jessica Knapp, Jie Lu, S. Ranganathan, Huabo Wang, E. V. Prochownik
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Metastatic pancreatic adenocarcinoma is the dominant clinical presentation with a grim 3% 5-year survival rate. Over 80% of metastatic disease occurs in the liver and has the poorest outcomes. Overall, there is little mechanistic understanding of what promotes liver metastatic outgrowth and organotropism. Our recent work using spatial transcriptomics on unique matched primary tumors and metastases revealed distinct cellular ecosystems, noting reduced desmoplasia, high proliferation, spatially constrained metabolism, and heightened pro-tumorigenic myeloid infiltration and T-cell dysfunction at the invasive border of liver metastases. However, moving beyond these observations requires establishing causal links between molecular drivers and metastatic competence. Causal studies are limited by current model systems, the spontaneous genetically engineered mouse models (GEMM) that form the basis of pre-clinical studies do not sufficiently model the clinical metastatic reality with robust matched pancreas and liver tumors, and are further hampered with inconsistent metastatic rates and unpredictable progression for timed analysis. Syngeneic transplants of GEMM-derived cancer lines into wild-type mice provide a rapid pre-clinical model system of liver disease. However, the liver-metastatic rates varies between cell lines, even with identical driving mutations. We suspected that the unexplored mechanisms driving these differences in metastatic outgrowth present an opportunity to understand critical biology. Here we report the development a consistent model system of matched pancreatic and liver tumors using syngeneic cell lines with high and low tropism for liver metastatic outgrowth in C57Bl/6 mice. These lines are transplanted at low cell numbers to better allow the evolution of the site-specific tumor microenvironment and provide a reliable model system to examine both cancer-cell intrinsic and site-specific microenvironmental factors dictating liver outgrowth. Our observations of comparable pancreatic growth, successful metastatic growth in other organs (peritoneum or lung), and micro-metastatic lesions in the liver at early time points, suggests the liver-tropic differences fall within the ability of these cells to successfully outgrow in the liver microenvironment, rather than in vivo proliferative or extravasation differences. Comparison of gene expression between high and low liver-tropic cell lines identified several immune regulatory genes and a general increase in lipid metabolism, consistent with our published patient data. Finally, spatial quantifications of these lesions using a novel 46-plex murine immunotyping panel on FFPE tissues show similar suppressive immune cells at the interface of the tumor and normal liver as observed in our patient data, but evolves across differing lesion size, suggestive of a spatiotemporal progression. Altogether, this model system provides a robust, efficient pre-clinical platform to dissect spatiotemporal drivers of liver metastatic disease.



 Christina R. Larson, Jace Baines, Ayushi Mandloi, Meet Patel, Tuan Tran, Nailah Jones, Ateeq M. Khaliq, Christopher A. Risley, Robert S. Welner, Satwik Acharyya, Julienne L. Carstens. Robust Pancreatic Cancer Liver Metastatic Model System Reveals Cancer Cell Dependent Organotropism and Site-specific Tumor Microenvironment Regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288b7e540e23f2b8aa282e16a6f01e2dc86d80d0" target='_blank'>
              Abstract A048: Robust Pancreatic Cancer Liver Metastatic Model System Reveals Cancer Cell Dependent Organotropism and Site-specific Tumor Microenvironment Regulation
              </a>
            </td>
          <td>
            Christina R Larson, Jace Baines, Ayushi Mandloi, Meet Patel, Tuan Tran, Nailah Jones, A. Khaliq, Christopher A. Risley, Robert S. Welner, Satwik Acharyya, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Papillary thyroid carcinoma (PTC) is the most prevalent form of thyroid cancer, yet its cellular heterogeneity and prognostic determinants remain poorly defined. Here, we integrate two single-cell RNA sequencing datasets comprising 20 human thyroid samples to construct a high-resolution cellular atlas of PTC. We identify 29 distinct cellular subpopulations and delineate their composition, dynamics, and interactions in healthy versus tumor tissues. Notably, epithelial and monocyte populations were markedly expanded in PTC, whereas adaptive immune subsets such as B and T cells were diminished. Cell–cell communication analysis revealed enhanced intercellular signaling in the tumor microenvironment, with epithelial and endothelial cells receiving the strongest inputs. Among monocyte-specific transcriptional signatures, we identified 65 prognostic genes via univariate Cox analysis. A LASSO-derived 14-gene risk score robustly stratified patient outcomes, with CSGALNACT1 emerging as a key epithelial-specific, independent prognostic gene. Pseudotime analysis further supported its role in epithelial cell differentiation. Functional validation demonstrated that CSGALNACT1 promotes proliferation in PTC cell lines, suggesting a potential oncogenic function. Immune deconvolution across risk groups revealed substantial divergence in innate and adaptive immune infiltration, indicating a close interplay between tumor-intrinsic transcriptional programs and immune microenvironment remodeling. Collectively, our study provides a comprehensive single-cell framework for PTC, identifies a clinically relevant risk model, and highlights CSGALNACT1 as a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62fcd940763c0c7abf39d3fd346b8a069b2e5dc" target='_blank'>
              Comprehensive analysis of the papillary thyroid carcinoma identifies CSGALNACT1 as a proliferation driver and prognostic biomarker
              </a>
            </td>
          <td>
            Yu Zhao, Juan Zhang, Yong Yao, Linna Yu
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea7d79855ee47627896db43f29cfa10fcc98ce7" target='_blank'>
              Single-cell multiomic modelling of early metastatic events promoted by the extracellular matrix.
              </a>
            </td>
          <td>
            Hyun Jin Lee, Jun Hyeong Lee, Junho Kang, Kyeonghui Kim, Youngwon Cho, Jihyun Park, Sang-Hyun Song, J. Kwon, Young-Joon Kim, Woong-Yang Park, Tae-You Kim, Jong-Eun Park, Pilnam Kim, Jung Kyoon Choi
          </td>
          <td>2025-09-15</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma (PDAC) exhibits a huge range of genomic plasticity due to prevalent genomic instability across tumors. Longitudinal whole genome sequencing of patients at primary diagnosis and metastatic disease progression have shown KRAS copy number amplification and allelic imbalance as drivers of classical to basal-like phenotypic switch, resulting in propensity for whole genome duplication, genomic instability and aggressive metastasis. High-plex spatial proteomics have revealed that primary tumors are phenotypically mixed, but the relationship between intratumorally distinct phenotypes and their genomics remain unclear as existing genomics relies on bulk captured tumors. Therefore, this drives the need to spatially resolve the intra-tumoral genomic heterogeneity associated with distinct tumor phenotypes. We leveraged whole-slide high-plex Imaging Mass Cytometry (IMC) to stain a phenotypically mixed primary tumor with 41 different markers covering the classical (AGR2, TFF1, CEACAM6) and basal (TP63, KRT5, S100A2, CAV1) phenotype, pan-immune markers, epithelial transcription factors (GATA6, FOXA2, PDX1) and stroma. Bulk whole genome sequencing (30X) reveals that this tumor had undergone losses of chr 1p, 6p, 9, 17 and 8, in addition to 12p focal amplification and whole genome doubling. To address the association of spatial phenotypes to regional genomics, IMC guided laser capture microdissection (LCM) was performed on a serial section aligned to 4 classical (C1-4), 4 basal (B1-4) and 4 hybrid-staining (H1-4) regions, followed by independent shallow whole genome sequencing (sWGS) at 2X coverage for each captured region (n = 12). Quality control, alignment and variant calling was performed utilizing standard whole-genome sequencing workflow. Integer copy number was determined through 100kb bins and segmentation. A full range of ploidy solutions (2N, 3N and 4N) was fit for each region using Absolute Copy Number Estimation (ACE) and the best fit was used for each region. We observed that in region C2, losses in many chromosomes (1p, 6, 9, 17, 18) but had no arm level amplifications, indicating a mitotic defect. At region C3, we observed a small increase in the 12p segment harboring the KRAS locus, suggesting that a sub-clonal event had occurred resulting in amplification. At region H3, a phenotypically mixed region, a strong 12p amplification was observed along with widespread genomic instability. Finally, at region B1, the whole genome duplication is observed. Our initial pilot reveals a stepwise, orchestrated genome evolution process in a phenotypically mixed primary tumor which was not obvious in bulk whole genome sequencing, highlighting the need for spatially informed regional capture of tumors to uncover phenotypically and potentially clinically important genomic alterations.



 Tiak Ju Tan, Deni Van. Sechie, Ferris Nowlan, Noor Shakfa, Elizabeth Sunnucks, Sibyl Drissler, Jenn Gorman, Michael J. Geuenich, Chengxin Yu, Grannie O. Kane, Faiyaz Notta, Rob Grant, Julie Wilson, Steve Gallinger, Kieran R. Campbell, Hartland W. Jackson. Imaging mass cytometry guided shallow whole genome sequencing for phenotype informed regional copy number profiling in a pancreatic tumor [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B113.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e0d82bf993f8cb23419e03138a40de013a09be1" target='_blank'>
              Abstract B113: Imaging mass cytometry guided shallow whole genome sequencing for phenotype informed regional copy number profiling in a pancreatic tumor
              </a>
            </td>
          <td>
            Tiak Ju Tan, Deni Van. Sechie, Ferris Nowlan, Noor Shakfa, Elizabeth Sunnucks, Sibyl Drissler, Jennifer L. Gorman, Michael J Geuenich, Chengxin Yu, Grannie O. Kane, F. Notta, Robert C. Grant, Julie M. Wilson, Stephen Gallinger, Kieran R. Campbell, H. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Chimeric antigen receptor T-cell (CAR-T) therapy has emerged as a breakthrough immunotherapy for multiple cancers, yet its efficacy remains limited in glioblastoma due to tumor heterogeneity, immune-evasion of the microenvironment, and rapid T-cell dysfunction. The aim of our study is to investigate the spatially differential expression of CAR-T targets in glioblastoma. We want to focus on differential expression of CAR-T targets in tumor cell subtypes, differences between tumor core and infiltration zone as well as reactive immune zones.



 We analyzed spatial transcriptomic data from 62 glioblastoma specimens using SPATA2 software, augmented by advanced computational tools such as scVI (data integration), Cell2Location (single-cell deconvolution), graph neural networks, gene set enrichment analysis (GSEA), and spatial annotation screening (SAS). CAR-T antigens were chosen from existing published clinical and pre-clinical CAR-T cell trials.



 Spatial mapping revealed a pronounced gradient of NPC-like cells with a higher abundance of NPC-like cells in the infiltration zone and resection margins—regions critical for residual disease persistence after resection. Clinical CAR-T targets (e.g., ERBB2, PTPRZ1) were highly expressed in AC-like and OPC-like dominated niches while nearly absent in NPC-rich areas, suggesting a blind spot for current CAR-T therapies. Single-cell deconvolution further correlated NPC-like dominance to increased T-cell infiltration, while GSEA and clonal dynamics analysis of the T-cell compartment suggested downregulated functioning subclones in NPC-rich spatial neighborhoods, underscoring potential immune evasion mechanisms in these regions.



 Our study exposes a fundamental limitation of current CAR-T designs: their inefficacy against NPC-like cells that dominate the infiltration zone and therefore the post-surgical glioblastoma ecosystem. By pinpointing spatial and phenotypic resistance hubs, we provide a roadmap for refining CAR-T targets to address this gap, potentially enhancing adjuvant immunotherapy for glioblastoma. Protein-level validation of key findings is currently ongoing to further substantiate the results.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe7508f42aeb973a2f44bba592abe40968ca576d" target='_blank'>
              P04.13.B SPATIAL HETEROGENEITY OF CAR-T ANTIGEN EXPRESSION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Y Kong, J. Zhang, F. Sahm, G. Kastner, M. Mühlbauer, E. Grabis, P. Heinrichs, J. Straehle, R. Sankowski, R. Roelz, J. Beck, D. Heiland, N. Neidert
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are pervasively present in human cancers and have a fundamental role in treatment failure and disease recurrence. Identifying critical elements that sustain the CSC phenotype may lead to novel strategies for cancer treatment. Here, we provide evidence of an essential link between the σ1 receptor (σ1R), a ligand-regulated chaperone protein residing preferentially at the endoplasmic reticulum-mitochondria contact sites, and CSCs in castration-resistant prostate cancers (CRPCs). Integrating functional assays in multiple preclinical models with transcriptomic and proteomic data, we found that σ1R controls CSC self-renewal capacity and tumorigenic proficiency by coordinating mitochondrial dynamics and mitochondrial-nuclear signaling. Inhibiting σ1R with synthetic antagonists and RNA interference led to the progressive exhaustion and loss of tumorigenicity of the CSC progeny. Mechanistically, interfering with σ1R function disrupted mitochondria homeostasis and triggered β-catenin degradation. Examining clinical CRPC samples, we found a tight correlation between σ1R and mitochondrial gene expression. Furthermore, σ1R and β-catenin protein levels were highly correlated in prostate tumors with significant upregulation in metastatic CRPCs, sustaining a role of the σ1R-mitochondria-β-catenin axis in disease progression. This σ1R-centered axis is essential for preserving the self-renewal and tumorigenic capability of CSCs and represents a critical vulnerability exploitable for discovering novel CSC-directed therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f3ac7d8f1f9723125587e98e6c28a4657680b4" target='_blank'>
              Integrated control of cancer stemness by σ1 receptor in advanced prostate cancer
              </a>
            </td>
          <td>
            G. Civenni, G. Sandrini, Jessica Merulla, C. Musumeci, Elisa Federici, Arianna Vallegra, A. Kokanovic, Simone Mosole, D. Shinde, Elisa Sorrenti, Alyssa J J Paganoni, Martina Marchetti, Riccardo Valzelli, D. Albino, Matteo Pecoraro, Andrea Rinaldi, Marco Bolis, Roger Geiger, Tobias Winge, Catharina Holtschulte, E. Laurini, S. Pricl, G. M. Carbone, Bernhard Wünsch, C. Catapano
          </td>
          <td>2025-09-02</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2affd9966e7d74eae7ebff40360fcc4663c6bf18" target='_blank'>
              Clonal expansion of cytotoxic CD8âº T cells in lecanemab-associated ARIA
              </a>
            </td>
          <td>
            Lance A. Johnson, Kai Saito, Akhil Pallerla, Jessica L Funnell, Ashley R. Ezzo, Chelsea L. Song, Douglas A Harrison, Noah J. Norton, Lauren C. Moore, L. Eldik, D. Fardo, Greg E. Cooper, Josh M Morganti
          </td>
          <td>2025-09-21</td>
          <td>None</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) chemotherapy regimens are largely uniform, tailored mainly to patient fitness or rare genomic alterations. This strategy fails to capitalize on the disease’s substantial molecular heterogeneity, including variations in copy number alterations, immune infiltration, stromal composition, and cancer cell phenotypes. These features are not independent; they reflect interconnected biological processes and co-evolving pathways that shape tumour progression. Multi-omic integration is therefore essential—not only to capture the complexity of PDAC biology, but also to clarify the molecular basis of tumours classified as ‘intermediate’ by current stromal and cancer taxonomies, thereby enabling rational therapeutic targeting. To this end, we performed imaging mass cytometry on three serial sections of a PDAC tissue microarray (221 resected tumours, ∼4 cores each), generating >800 multiplexed images (40–43 channels) focused on epithelial, immune, or stromal biomarkers. We profiled 76 immune and stromal cell types and states (hypoxic, proliferative, apoptotic, under tension), as well as six tumour phenotypes defined by expression of epithelial transcription factors (GATA6, FOXA2, PDX1), classical markers (AGR2, TFF1, CEACAM6), basal markers (TP63, KRT5, S100A2, CAV1), and other PDAC-associated proteins (S100A4, MMP7, MUC16). These six cancer cell types captured the classical and basal PDAC signatures, along with four discrete “intermediate” states with distinct associations to stromal heterogeneity, RNA subtype (n = 92), tumour ploidy (n = 182), and patient outcome. These phenotypes are also detectable in unmatched single-cell RNA-seq data (n = 163), though with more overlap in marker expression. Using matched 30X whole genome sequencing (n = 182), we identified mutations and copy number alterations linked to shifts in cancer and stromal cell phenotypes, raising the question of which molecular axis best informs clinical prognosis. To address this, we applied a modified version of Stabl, a Lasso-based machine learning approach, to compare across omic layers and identify features most strongly associated with overall survival. Per-modality analysis showed that models incorporating omics outperformed those based on clinical features alone, with imaging data slightly outperforming genomics. Cross-omic integration revealed several prognostically relevant copy number aberrations, fibroblast phenotypes, and cancer cell states. By consolidating information on tumour phenotypes, stromal niches, and genomic alterations, this work aims to focus future drug discovery on the most clinically impactful molecular features in PDAC.



 Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, Ayelet Borgida, Edward L. Chen, Golnaz Abazari, Miralem Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, Grainne M. O'Kane, Faiyaz Notta, Steve Gallinger, Hartland W. Jackson. Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B116.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5cd553302915b28eaa98522e24327f9aa96cbdc" target='_blank'>
              Abstract B116: Integrative proteogeonomic profiling of PDAC reveals updated epithelial subtypes and cross-omic predictors of survival
              </a>
            </td>
          <td>
            Ferris Nowlan, Noor Shakfa, Tiak Ju Tan, Sibyl Drissler, Elizabeth Sunnucks, Jennifer L. Gorman, Chengxin Yu, Sheng-Ben Liang, Barbara Gruenwald, A. Borgida, Edward L. Chen, Golnaz Abazari, M. Mrkonjic, Julie M. Wilson, Kieran R. Campbell, Robert C. Grant, Anne-Claude Gringas, G. O’Kane, F. Notta, Stephen Gallinger, H. Jackson
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Despite the availability of numerous approved immunotherapies for various cancers, durable progression-free survival remains relatively uncommon among patients with advanced cancer. As research into immunotherapy intensifies, the heterogeneity and complexity of the tumor microenvironment (TME) have emerged as critical determinants of treatment response and a major obstacle to understanding tumor resistance mechanisms. Recent advances in spatially resolved transcriptomics (SRT) enable transcriptome-wide measurement of gene expression while preserving essential spatial information, which supports the characterization of the features of the TME. This review outlines key challenges in cancer immunotherapy and emphasizes the importance of the TME in determining therapeutic efficacy. SRT strategies suitable is systematically introduced for TME profiling, comparing their relative advantages and limitations. Additionally, bioinformatics tools and methods used to reconstruct spatial tissue patterns are discussed from SRT data. Furthermore, how SRT is explored can enhance the understanding of the complexity and heterogeneity of the TME, facilitate the identification of novel therapeutic targets, and advance biomarker discovery to refine personalized treatment strategies for diverse cancer patients. Finally, future perspectives on the application of SRT in cancer immunotherapy and addressing the associated challenges are provided.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53e788cfc2c6f7fa7f419ffa32b647da6da2a8f3" target='_blank'>
              Spatially Resolved Transcriptomics: Revealing Tumor Microenvironment Heterogeneity to Advance Cancer Immunotherapy.
              </a>
            </td>
          <td>
            Dan Liu, Lu Xiao, Yuyuan Wu, Chensirong Yue, Mengmeng Li, Yuanpei Sun, Chao Yang
          </td>
          <td>2025-09-04</td>
          <td>Small methods</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The high intra-patient heterogeneity in multifocal primary prostate cancer (PCa) has curtailed the efficacy of current treatment options. By employing twin biopsies from multiple lesions with matched patient-derived organoids (PDO) models, the PCa molecular heterogeneity was investigated. We utilized genomics, transcriptomics and machine learning (ML) approaches to elucidate and predict the underlying mechanisms of pharmacological heterogeneity. Our data indicate a vulnerability of primary PCa organoids for small molecule inhibitors targeting receptor tyrosine kinases (MET, ALK, SRC). By exploring gene expression data from matched parental tissue in an unsupervised manner, we identified two distinct clusters of samples. Interestingly, the PDO drug responses were significantly different between the two clusters for 4/11 compounds tested. We developed a transcriptomics-based, cluster prediction model, which can accurately stratify samples into the two clusters. Notably, our prediction model is based on tissue profiles, therefore, it can be utilized to rapidly evaluate new cases and suggest promising drug candidates, even when PDO derivation is not feasible. Taken together, we propose a novel flexible stratified oncology approach that can swiftly and accurately highlight promising drug vulnerabilities of PCa patients. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03540-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181289a740f6c1249d22ea3b355cd097aedde6e0" target='_blank'>
              Multi-layer stratified oncology platform utilizing transcriptomics, prostate cancer organoids, and modeling of drug response
              </a>
            </td>
          <td>
            Juening Kang, P. Chouvardas, Andrew Maalouf, D. Hanhart, Laura Fernández Cerro, Wanli Cheng, Eva Compérat, Katja Ovchinnikova, Rahel Etter, Michaela Medová, Ulrich Schneeberger, Beat Roth, George N. Thalmann, S. Karkampouna, M. Kruithof-de Julio
          </td>
          <td>2025-10-16</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Cancer is extensively studied, yet its origins and progression remain unclear. A key question is why tumors of the same type vary in aggressiveness and treatment response. In colorectal cancer (CRC), the third most common cancer, this variability led to the identification of molecular subtypes (CMS). However, the tumor microenvironment remains poorly understood and may be crucial for understanding carcinogenesis and drug resistance. A promising approach is analyzing cell interactions through ligand-receptor expression. This study used bioinformatics to examine CRC in different anatomical locations, identify diagnostic and prognostic biomarkers, and propose targeted drugs. Methods Expression data were obtained from the TCGA-COAD database. All samples were filtered based on the tumor’s region of origin and purity. RNA-seq expression analysis was then conducted to assess molecular differences according to tumor location and purity, identifying region-specific ligands and receptors using the secretome list as a reference. Once these differences were identified, an interactome was constructed to depict cell interactions within the tumor microenvironment. The most relevant genes were then evaluated for their prognostic potential through survival analysis, and their susceptibility to pharmacological modulation was assessed to identify potential new drug candidates for CRC treatment. Results The integration of secretome data and the construction of the interactome proved to be a valuable approach for detecting novel biomarkers specific to right- and left-sided CRC. Through this approach, FGFR4, FLT1, and WNT5A were identified as key biomarkers involved in tumor carcinogenesis, modulating distinct processes in each region, such as fibroblast recruitment and cell division. Based on these biomarkers, Dovitinib and Nintedanib were predicted as potential therapeutic agents, as they target multiple identified markers. Conclusion This study highlights FGFR4, FLT1, and WNT5A as key diagnostic and therapeutic biomarkers for CRC, with their relevance varying based on the tumor’s site of origin. Leveraging these findings, we propose Dovitinib and Nintedanib as promising targeted therapies for CRC. These insights can enhance current treatment strategies and pave the way for future in vivo and in vitro studies, driving progress in CRC research and therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02424-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d86a3d9f8b9f730ccd2b60a1bbaaeddf8c7f0fad" target='_blank'>
              Integrating the secretome and interactome to identify novel biomarkers and therapeutic targets in colorectal cancer
              </a>
            </td>
          <td>
            Gabriel Henrique Caxali, Mirian Carolini Esgoti Aal, Catherine Wesselka Garcia Osvaldo, Jakeline Santos Oliveira, L. Bidinotto, Robson Francisco Carvalho, F. K. Delella
          </td>
          <td>2025-10-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Immune profiling has become a transformative tool in oncology, offering comprehensive information on tumor immune interactions and facilitating precision medicine. Recent advances such as mass cytometry (CyTOF), single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and liquid biopsy have greatly enhanced our ability to characterize immune heterogeneity and predict treatment responses. These innovations support the identification of new biomarkers, therapeutic targets, and resistance mechanisms, refining patient stratification and clinical results. Additionally, artificial intelligence (AI) driven models are now being employed to integrate multi-omics datasets and create predictive insights, thereby linking the gap between research and clinical decision-making. This review studies the evolution of immune profiling technologies, their integration into real-world oncology practice, and the associated technical and analytical challenges, including sample variability, data harmonization, and multi-omics integration. Although challenges such as cost, throughput, and standardization persist, the merging of advanced technologies, bioinformatics, and clinical frameworks promises to reshape cancer diagnosis, therapy selection, and disease monitoring through personalized and data-driven strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3549665d4cf233e9a7efcdd9f458593ee2c2ae65" target='_blank'>
              Immune profiling in oncology: bridging the gap between technology and treatment
              </a>
            </td>
          <td>
            Nanthini Ravi, G. Tye, S. Dhaliwal, Muhamad Yusri Musa, Matthew Tze Jian Wong, Ngit Shin Lai
          </td>
          <td>2025-08-26</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="PURPOSE
The Tumor Microenvironment (TME) has a vital role in cancer survival and progression, and may play roles in drug resistance and immune escape. To date, few studies have detailed the TME of Ewing sarcoma (EwS).


EXPERIMENTAL DESIGN
We performed spatially resolved transcriptomics of primary treatment-naïve EwS tumor biopsies from patients with or without clinical metastasis, complemented by high-plex spatial proteomic analysis.


RESULTS
We discovered greater stromal enrichment in localized EwS primary tumors compared to metastasis-associated EwS primary tumors. Through spatial ligand-receptor analysis, we show that the stromal enriched regions harbor unique extracellular matrix related cytokines, immune recruitment and proinflammatory microenvironmental signals, implying EwS stroma may play an anti-tumor role by acting as an immune recruitment center. All EwS tumors expressed pro-tumorigenic MIF-CD74 immune signaling connectivity, suggesting a potential immune-evasive mechanism.


CONCLUSIONS
In addition to the immune recruitment role of tumor-associated stroma, our findings provide spatial insight into the TME of EwS and provide a rationale for the preclinical investigation of MIF as a potential target for Ewing sarcoma immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f9c77e0153cfc111b2db6ef0b69632696638603" target='_blank'>
              Spatial profiling identifies tumor-associated stroma enrichment and MIF as potential immunotherapy targets in primary Ewing sarcomas.
              </a>
            </td>
          <td>
            Christopher Kuo, K. Giannikou, Nuoya Wang, Mikako Warren, Andrew Goodspeed, N. Shillingford, Masanori Hayashi, M. Raredon, James F. Amatruda
          </td>
          <td>2025-09-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Single-cell multi-omics has transformed T-cell biology by enabling the simultaneous analysis of T-cell receptor (TCR) sequences, transcriptomes, and surface proteins at the resolution of individual cells. These capabilities are critical for identifying antigen-specific T-cells and accelerating the development of TCR-based immunotherapies. Methods Here, we introduce TCRscape, an open-source Python 3 tool designed for high-resolution T-cell receptor clonotype discovery and quantification, optimized for BD Rhapsody™ single-cell multi-omics data. Results TCRscape integrates full-length TCR sequence data with gene expression profiles and surface protein expression to enable multimodal clustering of αβ and γδ T-cell populations. It also outputs Seurat-compatible matrices, facilitating downstream visualization and analysis in standard single-cell analysis environments. Discussion By bridging clonotype detection with immune cell transcriptome, proteome, and antigen specificity profiling, TCRscape supports rapid identification of dominant T-cell clones and their functional phenotypes, offering a powerful resource for immune monitoring and TCR-engineered therapeutic development. TCRscape can be found at https://github.com/Perik-Zavodskii/TCRscape/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eefe820a4c406c3e6561963250df919c0c97ebc" target='_blank'>
              TCRscape: a single-cell multi-omic TCR profiling toolkit
              </a>
            </td>
          <td>
            R. Perik-Zavodskii, O. Perik-Zavodskaia, M. Volynets, S. Alrhmoun, Sergey Sennikov
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 We present a novel integration of data-driven single-cell analysis with mechanistic mathematical modeling to predict the impact of cancer-associated fibroblasts (CAF) on pancreatic ductal adenocarcinoma (PDAC) invasion. Bioinformatic analyses of high-throughput single-cell and spatial molecular assays give us access to human tumor gene expression at the time of measurement, enabling direct characterization of the human tumor microenvironment (TME), complementing our understanding of cell behavior gained from preclinical models. Notably, applying our Bayesian non-negative matrix factorization algorithm CoGAPS to single-cell RNA-seq data from PDAC allowed us to associate a gene program of concurrent epithelial-to-mesenchymal transition (EMT) and inflammation in epithelial cells with the presence of CAFs, validated through organoid co-culture experiments. We hypothesize that this gene program represents a mechanism by which CAFs promote invasive behavior in neoplastic cells over time. Testing this hypothesis requires inference of temporal changes, which is not possible with single-timepoint data collection in genomics data. In contrast to data-driven bioinformatics, mechanistic mathematical modeling can capture processes that happen over time and make predictions about a system. Agent-based models (ABMs) are one form of such mechanistic mathematical models that are well suited for temporal modeling of cellular phenotypes. We developed new software for ABMs that abstract cells into software agents with individual states and their own rules of behavior from single-cell and spatial molecular assays for temporal inference. We apply this technique to understand how CAFs in the PDAC microenvironment support the series of phenotype transitions comprising tumor progression and invasion, by encoding the CAF-induced ability of tumor cells to migrate and a cessation of proliferation in tumor cells, induced by CAF-secreted, non-diffusable substrate as inferred from our genomics analysis. While this provides a virtual framework to test the impact of varying CAF density on tumor cell invasion, it initially lacked integration with the spatial architecture of human tumors. We then apply the ABM to human PDAC spatial transcriptomics data to forecast tumor behavior over time in a spatially-informed setting, varying the hypothesized mechanisms. This revealed that a substrate-driven mechanism results in a broad mesenchymal-like zone surrounding epithelial-like outgrowths, while a cell contact-mediated mechanism results in a single-cell-wide boundary surrounding more significant outgrowth. When we compared these to real tumor compositions, this favors the explanation that secreted CAF substrate drives EMT in PDAC neoplastic cells, which we then validate in organoids with CAF conditioned media. Our work demonstrates how integrating ABMs with genomic and spatial data enables testing of mechanistic hypotheses about tumor-stroma interactions and provides deeper insights into PDAC microenvironments, potentially informing strategies to intercept disease progression.



 Jeanette I. Johnson, Daniel R. Bergman, Tamara Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Yoseph W. Dance, Jacob T . Mitchell, Elizabeth M. Jaffee, Neeha Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, Ashley L. Kiemen, Luciane T. Kagohara, Young H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O'Brien, Elana J. Fertig. Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d57a26173bbcb8bbded169c7c20718d2482d35" target='_blank'>
              Abstract A025: Genomics-informed agent-based models drive new understanding of the role of neoplastic phenotype transitions in human PDAC invasion
              </a>
            </td>
          <td>
            J. Johnson, Daniel R Bergman, T. Lopez-Vidal, Eric Cramer, Fatemeh Shojaeian, Y. Dance, Jacob T. Mitchell, Elizabeth M. Jaffee, N. Zaidi, Elizabeth D. Thompson, Jacquelyn W. Zimmerman, A. Kiemen, Luciane T. Kagohara, Y. H. Chang, Jude M. Phillip, Laura D. Wood, Paul Macklin, Genevieve L. Stein-O’Brien, E. Fertig
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Tumor heterogeneity impacts invasive behaviors, treatment response, and clinical outcomes in triple-negative breast cancer (TNBC). However, this heterogeneity remains incompletely characterized. This study aims to utilize multi-scale data to investigate inter-tumoral heterogeneity and identify potential TNBC biomarkers. Methods Single-cell RNA expression profiles were analyzed using R packages. Specifically, the infercnv, Pyscenic, GeneNMF, SCP, Vector, CellChat, and hdWGCNA packages were employed to identify malignant cells and characterize heterogeneity in transcription factors, metaprograms, lineage evolution, developmental trajectories, cell–cell interactions, and co-expression networks. Bulk RNA datasets were incorporated to assess the prognostic value of cell clusters and candidate genes. G Protein Subunit Alpha 15 (GNA15) expression was determined via reverse transcription–quantitative PCR (RT–qPCR) and immunohistochemistry. Cell functional assays were performed to evaluate proliferation, migration, and invasion capabilities. Results A total of 14,335 malignant cells were isolated from epithelial cells across 15 single-cell RNA samples. Six tumor cell clusters were identified, which exhibited distinct prognoses, biological functions, driver transcription factors, and co-expression networks. Notably, the S2 cluster demonstrated association with multiple malignancy-related pathways and inferior survival outcomes. GNA15 emerged as the S2 cluster hub gene. In vitro experiments confirmed that GNA15 knockdown significantly attenuated proliferation, migration, and invasion in TNBC cell lines. Conclusions Our study comprehensively delineated TNBC tumor cell heterogeneity and established the critical role of GNA15 in TNBC progression. These findings enhance the understanding of TNBC heterogeneity and provide a theoretical foundation for TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e149422e4d37b7c64ca2e5cbee9d7c8957b702e6" target='_blank'>
              Multiomics profiling identifies the poor prognostic role of a tumor cluster with GNA15 overexpression in triple-negative breast cancer
              </a>
            </td>
          <td>
            Guixin Wang, Jun Cao, Chenglu Lu, Yu-Peng Cao, Shuo Wang, Ziyi Chen, Zhaohui Chen, Yingxi Li, Yue Yu, Yao Tian, Xin Wang
          </td>
          <td>2025-09-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) refer to ectopic lymphoid aggregates that form in non-lymphoid tissues at sites of chronic inflammation including cancers. TLSs have been recognized as significant predictors of the efficacy of immune checkpoint blockade (ICB) therapies and have the potential to elicit robust anti-tumor immune response. However, recent studies have revealed substantial heterogeneity in TLSs across different individuals and cancer types, which directly impacts the effectiveness of anti-tumor immunity. Concretely, the maturation status, localization, and density of TLSs profoundly influence the dynamic interactions among immune cells within these structures, potentially leading to adverse effects. This review provides an in-depth exploration of how the heterogeneity of TLSs influences cellular composition and immune dynamics, with the objective of influencing the efficacy of ICB therapies and modulating prognostic prediction accuracy. Additionally, the potential of combining TLSs with other biomarkers for predicting anti-tumor immunity outcomes is further investigated, alongside the introduction of advanced technologies for evaluating TLS heterogeneity. Collectively, these analyses aim to advance the understanding of TLS heterogeneity and facilitate its translation into clinical and translational medicine applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3182b4ad88a2ceacb5b488328272b01f2977fe0" target='_blank'>
              Dissecting Tertiary Lymphoid Structures in Cancer: Maturation, Localization and Density
              </a>
            </td>
          <td>
            Guang-Liang Su, Meng‐Jie Zhang, Hao Li, Zhijun Sun
          </td>
          <td>2025-08-30</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Pancreatic intraepithelial neoplasms (PanINs), precursor lesions of pancreatic ductal adenocarcinoma (PDAC) that are abundant healthy individuals, rarely progress to cancer despite universally possessing many of the driver alterations associated with malignancy. Recent single cell profiling efforts of PanIN progression in vivo have described marked heterogeneity in chromatin accessibility and transcriptional identity within these neoplasias, but it remains unclear how discrete epigenetic factors play a role in shaping tumorigenesis, or what distinguishes rare neoplastic cells capable of seeding invasive carcinomas from their benign neighbors. Given that more than 80% of patients with PDAC present with unresectable or metastatic disease, further studies on the molecular features of PanIN progression are needed to enable methods of early detection and intervention. Unfortunately, owing to their microscopic size and challenges associated with the isolation of PanINs from surrounding normal epithelium and stroma, it is particularly difficult to address these knowledge gaps in vivo. By generating complex murine derived organoid models, we have built a system that allows us to faithfully reconstitute the evolution from benign PanIN to PDAC in vitro, as well as in vivo through orthotopic implantation into syngeneic mice. As they are easily perturbable, readily generate large amounts of biomass, and closely resemble the stages of PDAC initiation in vivo, these models can be leveraged to directly assay how chromatin remodelers influence malignant transformation. Further, pairing these organoids with our highly multiplexed, transcribable DNA barcoding system and single cell epi multiomics will enable us to study the evolutionary dynamics, chromatin landscapes, and transcriptional identities of clonal populations of PanIN organoids as they progress to cancer. Through these studies, we will gain unprecedented insights into the underpinnings of PDAC initiation, and thus may open up new avenues for the early detection of tumorigenesis.



 Connor J. Hennessey, Minh T. Than, Ben Z. Stanger, Sydney M. Shaffer. Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90b593e2ad7a725999b9d873ea37a620072eeba" target='_blank'>
              Abstract B106: Elucidating the clonal dynamics and epigenetic regulators of PDAC initiation in inducible neoplastic organoid models
              </a>
            </td>
          <td>
            Connor J. Hennessey, Minh T Than, Ben Z. Stanger, Sydney M. Shaffer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a055b2099d39e961c79dc682be1c20375d7bba" target='_blank'>
              Tumor microenvironment response to neoadjuvant therapy in breast cancer: insights from single-cell and spatial omics.
              </a>
            </td>
          <td>
            Qingzhong Wu, Jiawei Yang, Da Zhang, Haowei Xu, Yuhang Yu, Yu Zheng
          </td>
          <td>2025-09-13</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Biological systems exhibit emergent phenotypes that arise from the collective behavior of individual components, such as whole-organ functions that arise from the coordinated activity of its individual cells, or organism-level phenotypes that result from the functional interplay of collections of genes in the genome. We present CELLECTION, a deep learning framework that learns to associate subgroups of instances with different emergent phenotypes. We show CELLECTION enables interpretable predictions for heterogeneous tasks, including disease classification, identification of disease-associated cell subtypes, alignment of developmental stages between human model systems, and even predicting relative hand-wing indices across the avian lineage. CELLECTION therefore provides a scalable and flexible framework for identifying key cellular or genetic signatures underlying complex traits in development, disease, and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc6dcd93998e52fb62e6d82980c4261acb342888" target='_blank'>
              Predicting emergent phenotypes from single cell populations using CELLECTION
              </a>
            </td>
          <td>
            Hongru Hu, Siddhant Sanghi, Gerald Quon
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is an epithelial cell cancer of the biliary tract. CCA can be further classified into intrahepatic cholangiocarcinoma (iCCA), perihilar cholangiocarcinoma (pCCA), and distal cholangiocarcinoma (dCCA) depending on the anatomical location. Until recently, the treatment for advanced CCA has remained highly reliant on chemotherapy, with Gemcitabine plus Cisplatin used in first-line treatment. Recent developments have led to the addition of immune checkpoint blockade (ICB) to the chemotherapy regimen, highlighting the promising potential of immunotherapies for CCA treatment. Despite these developments, most patients still do not benefit from current treatments, and response rates to ICB monotherapy remain modest. This underscores the need to develop more effective immunotherapeutic strategies. A major obstacle to this is the highly heterogenous nature of the disease. CCA tumors exhibit high inter-tumor heterogeneity in terms of anatomical locations, driver mutations, etiologies, and tumor microenvironment (TME) composition, making each patient immunologically distinct and difficult to benefit from a one-fit-all approach. There is a need to stratify patients according to individual disease status to identify immunotherapies and combination therapies that are most beneficial to them. Here we describe the different ways inter-tumor heterogeneity may arise in CCA, including stromal cell abundance, anatomical location, driver mutations, etiologies, TME profile, and tertiary lymphoid structure (TLS) presence. We also discuss what these factors mean to the immune microenvironment and their potential to be used as biomarkers. Careful stratification of patients is crucial in designing personalized medicine to improve survival outcomes and treatment efficacy for CCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb66e953d9482069c69df6565960e118d42cf3b5" target='_blank'>
              Stratifying Cholangiocarcinoma: Tumor Microenvironment, Molecular Drivers, and Novel Immunotherapeutic Approaches.
              </a>
            </td>
          <td>
            Cindy Xinqi Liu, 
          </td>
          <td>2025-10-14</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Objective
Advanced gastric cancer remains highly refractory to therapy, with limited immunotherapy efficacy due to tumor microenvironment heterogeneity. Primary cilia, microtubule-based organelles involved in tumor progression, remain insufficiently explored in gastric cancer. This study aimed to define primary cilia subtypes and establish prognostic signatures for personalized treatment strategies.


Methods
Bulk transcriptomic data from over 1,500 gastric cancer samples were integrated to define distinct primary cilia subtypes. A primary ciliary phenotype-associated signature (PCS) was established using a multi-machine learning survival framework incorporating ten algorithms. The prognostic predictive value and immunotherapy response prediction capability of PCS were validated across multiple independent cohorts. Single-cell RNA sequencing analysis was performed to identify cellular populations associated with high-PCS phenotype. Causal weighted gene co-expression network analysis (WGCNA) was employed to identify driving factors, followed by functional validation through cell culture experiments and xenograft models.


Results
Two distinct primary cilia subtypes were identified and validated across all cohorts, with C2 patients exhibiting significantly worse overall survival compared to C1 patients. PCS demonstrated robust predictive value for both prognosis and immunotherapy response, with superior accuracy compared to existing models across multiple validation cohorts. High-PCS patients showed reduced tumor purity, increased stromal cell infiltration, and poor response to immunotherapy. Single-cell analysis revealed that fibroblasts had the highest PCS scores and identified a novel secreted modular calcium-binding protein 2 (SMOC2)high myofibroblastic cancer-associated fibroblast (mCAF) population as the key driver of high-PCS phenotype. Functional experiments confirmed that SMOC2 knockdown significantly suppressed gastric cancer cell proliferation, migration, and invasion, while promoting mCAF-to-inflammatory cancer-associated fibroblasts (iCAF) transition.


Conclusions
PCS serves as a robust prognostic biomarker for gastric cancer patients. Additionally, targeting SMOC2 high mCAFs represents a potential therapeutic strategy for patients with high-PCS gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8325aa5e13eb71b8edb3f16c8e7ba4db0f0ba8" target='_blank'>
              SMOC2 high myofibroblastic cancer-associated fibroblast drives primary cilia-associated tumor microenvironment remodeling and poor prognosis in gastric cancer.
              </a>
            </td>
          <td>
            Qiqi Xu, Changjiang Yang, Jingyuan Ning, Yunze Niu, Xuesong Zhao, Long Zhao, Caihong Wang, Shan Wang, Y. Ye, Zhanlong Shen
          </td>
          <td>2025-08-30</td>
          <td>Chinese journal of cancer research = Chung-kuo yen cheng yen chiu</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Small cell lung cancer (SCLC) is classified into distinct molecular subtypes based on the expression patterns of four transcription regulators: achaete–scute homolog 1 (ASCL1), neuronal differentiation 1 (NEUROD1), POU class 2 homeobox 3 (POU2F3), and yes‐associated protein 1 (YAP1). MicroRNAs (miRNAs) play critical roles in cancer cellular processes but their subtype‐specific implications in SCLC remain underexplored. Out of 46 surgically resected SCLC samples, miRNA visualization through in situ hybridization identified high expression of miR‐375 in the ASCL1, NEUROD1, and ASCL1/NEUROD1 subtypes, and miR‐9‐5p in the POU2F3 subtype. Comprehensive enhancer profiling using SCLC cell lines indicated that miR‐375 and miR‐9‐5p were regulated by super‐enhancers in a subtype‐specific manner. Multiplex immunohistochemistry by imaging mass cytometry found that the miR‐9‐5p‐high SCLC was characterized by a higher stromal area ratio, increased numbers of CD8+ T cells and CD163− macrophages in the intra‐tumoral area, and an increased number of plasma cells in the stromal area, as compared with the miR‐9‐5p‐low SCLC. Finally, clinicopathological analysis revealed that the miR‐375‐high SCLC was associated with YAP1 downregulation, increased serum pro‐gastrin‐releasing peptide levels, and poor prognosis. These findings highlight the critical role of super‐enhancer‐related miRNAs in the diversity of SCLC, and underscore the potential for novel diagnostic and prognostic biomarkers based on these subtype‐specific miRNAs. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbf371b5212be407e64461ced537f826862e10f9" target='_blank'>
              Characteristic miRNA profiles represent clinicopathological diversity of small cell lung cancer
              </a>
            </td>
          <td>
            Masafumi Horie, Hiroshi Takumida, H. Koba, Tsukasa Ueda, Hidenori Tanaka, Masami Suzuki, Yukinobu Ito, Ayumi Ito, Mao Kondo, Hiroshi I Suzuki, Isao Matsumoto, Seiji Yano, Akira Saito, Daichi Maeda
          </td>
          <td>2025-08-22</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="T cells recognize antigens and induce specialized gene expression programs (GEPs), enabling functions like proliferation, cytotoxicity and cytokine production. Traditionally, different T cell classes are thought to exhibit mutually exclusive responses, including TH1, TH2 and TH17 programs. However, single-cell RNA sequencing has revealed a continuum of T cell states without clearly distinct subsets, necessitating new analytical frameworks. Here, we introduce T-CellAnnoTator (TCAT), a pipeline that improves T cell characterization by simultaneously quantifying predefined GEPs capturing activation states and cellular subsets. Analyzing 1,700,000 T cells from 700 individuals spanning 38 tissues and five disease contexts, we identify 46 reproducible GEPs reflecting core T cell functions including proliferation, cytotoxicity, exhaustion and effector states. We experimentally demonstrate new activation programs and apply TCAT to characterize activation GEPs that predict immune checkpoint inhibitor response across multiple tumor types. Our software package starCAT generalizes this framework, enabling reproducible annotation in other cell types and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdba0ab80686667b3dcec48b4fa56304b52cf36e" target='_blank'>
              Reproducible single-cell annotation of programs underlying T cell subsets, activation states and functions
              </a>
            </td>
          <td>
            Dylan Kotliar, Michelle Curtis, Ryan Agnew, Kathryn Weinand, A. Nathan, Y. Baglaenko, Kamil Slowikowski, Yu Zhao, P. Sabeti, Deepak A. Rao, Soumya Raychaudhuri
          </td>
          <td>2025-09-03</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Glioblastoma (GB) is a highly aggressive brain tumor with poor prognosis and limited response to immunotherapies. Tumor intrinsic features like low mutational burden, and extrinsic factors such as T-cell dysfunction and an immunosuppressive microenvironment, contribute to resistance. To better study immune-tumor interactions in a clinically relevant context, we have developed patient-derived tumor organoids (PDTOs) that support co-culture with autologous and CAR-engineered immune cells, enabling real-time functional analysis.



 PDTOs were generated from freshly resected GB tissue, preserving tumor-specific architecture and molecular features. Patient-matched PBMCs were isolated via density gradient centrifugation, and immune cells were enriched using negative selection with MACS sorting. T cells were expanded ex vivo and characterized by flow cytometry for activation and exhaustion markers, although not yet tested in co-culture. Cytokine secretion was analyzed via ELISA to complement phenotypic profiling. For functional studies, EGFR/EGFRvIII-retargeted CAR-NK cells were generated via lentiviral transduction of primary PBMC-derived NK cells and expanded ex vivo in the presence of (low levels of) IL-15. These cells were co-cultured with 3D tumor spheroids at defined effector-to-target ratios. Immune infiltration and tumor cell killing were assessed using real-time live-cell imaging and fluorescence-based readouts.



 As proof of concept, we co-cultured EGFR-diploid and EGFR-amplified 3D GB spheroids with naïve and CAR-NK cells targeting EGFR derived from a healthy donor. CAR-NK cells showed enhanced cytotoxicity against EGFR-amplified spheroids, demonstrating receptor-dependent immune activation. These results confirm the feasibility of functional immune co-culture assays and support the ongoing integration of this approach into our organoid-based platforms.



 Our organoid-immune cell co-culture system offers a physiologically relevant platform to investigate GB immunobiology. Future applications include drug screening and CRISPR-based perturbation to identify key modulators of immune response, with the goal of informing personalized immunotherapeutic strategies. Observations from the CAR2BRAIN trial (NCT03383978) have shown CD4⁺ and CD8⁺ T cell recruitment following CAR-NK therapy, highlighting the clinical relevance of combinatorial immune approaches and supporting the use of our model to explore such mechanisms ex vivo.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb41d2ad01808f179f77775b88ffca7888d1d44" target='_blank'>
              P18.15.A IMMUNOCOMPETENT GLIOBLASTOMA ORGANOIDS FOR PERSONALIZED IMMUNE PROFILING
              </a>
            </td>
          <td>
            T. Koenig, L. M. Reindl, P. Zeiner, K. Weber, A. Rombach, M. Armbrust, P. Cakmak, W. Wels, T. Broggini, K. Plate, M. Czabanka, J. Steinbach, M. Burger, E. Ullrich, T. Buhlmann, A. Hau
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) represent a small but critical subpopulation within breast tumors that drive tumor initiation, progression, metastasis, therapeutic resistance, and recurrence. BCSCs are characterized by cell surface markers such as CD44⁺/CD24⁻ and high aldehyde dehydrogenase 1 (ALDH1) activity, and are regulated by pluripotency-associated transcription factors including POU class 5 homeobox 1 (OCT4), SRY-Box transcription factor 2 (SOX2), and Nanog homeobox (NANOG). They exhibit remarkable plasticity and heterogeneity across different breast cancer subtypes, which is further enhanced by epithelial-mesenchymal transition and dynamic interactions with the tumor microenvironment, enabling immune evasion and adaptation to hostile conditions such as hypoxia. Functionally, BCSCs sustain tumorigenesis through self-renewal, contribute to metastatic dissemination as circulating tumor cells, and can enter dormancy in distant niches, leading to late recurrences. Their intrinsic resistance to chemotherapy, radiotherapy, and targeted therapies arises from quiescence, enhanced DNA damage repair, overexpression of drug efflux transporters, and metabolic reprogramming, posing major clinical challenges. Recent advances have improved BCSC identification and isolation through marker-based cell sorting, dye exclusion assays, mammosphere culture, and functional reporter systems. Conventional therapies often spare or enrich BCSCs, highlighting the need for novel treatments. Targeted strategies focusing on stemness-related signaling pathways such as Wnt, Notch, and Hedgehog, as well as regulators including NANOG, ALDH1 and programmed death-ligand 1 (PD-L1), are under investigation. Emerging approaches incorporating natural compounds, nanotechnology, and microenvironmental modulation show promise in overcoming therapeutic resistance. This review integrates current understanding of BCSC biology, heterogeneity, and microenvironmental interactions, providing valuable insights for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0824bd9fb688f1edb70aeb158a043b8788637c" target='_blank'>
              Breast cancer stem cells: Biological roles, microenvironmental interactions, and therapeutic targeting
              </a>
            </td>
          <td>
            Mou Yan
          </td>
          <td>2025-10-11</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer mortality, with minimal improvements in 5-year survival over the past decade. While PDAC originates from premalignant lesions, only a minority of these lesions advance further into malignancy. Distinguishing which lesions are likely to progress would transform enabling timely preventative surgical intervention, but remains a key challenge. In addition to the more well-characterized genetic mutations associated with PDAC onset, recent studies suggest that inflammation in the pancreas microenvironment promotes epithelial chromatin accessibility remodeling to increase their plasticity. This plasticity is thought to enhance cellular sensitivity in response to local signaling cues, which varies drastically across PDAC tumors. We propose that the cell lineage fate of the premalignant epithelial cells is heavily influenced by their surrounding microenvironment. To investigate this, we are using the CosMx spatial transcriptomics platform to profile cellular composition heterogeneity across pancreatic tissue cores from KrasLSL-G12D; Trp53LSL-R172H; Pdx1-Cre (KPC) mouse model of PDAC. By analyzing spatial heterogeneity within the tumor microenvironment, we aim to delineate how local microenvironmental differences might influence epithelial lineage fate decisions into PDAC. Our findings will help understand the factors driving progression from early lesions to invasive cancer, supporting efforts in early PDAC detection and informing clinical strategies for surgical resection.



 Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P. Morton, Crispin J. Miller. Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B118.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb056397ddb7c8e4e0421ffdba57d9fda0a6959d" target='_blank'>
              Abstract B118: Investigating the tumor microenvironment’s role in lineage fate determination in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jennifer Claire Muscat, Eva Freckmann, Andrew S. Papanastasiou, Jennifer P. Morton, Crispin J. Miller
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053a87c896e7b2228f019eac1392494ac26a840e" target='_blank'>
              A spatial map of MAPK-activated immunosuppressive myeloid populations in pediatric low-grade glioma.
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Romain Sigaud, Evan Puligandla, Bridget Liu, E. Karimi, Alva Annett, M. Rezanejad, Wajih Jawhar, Robert Taylor, Yi Cao, S. Schmid, F. Selt, P. Hernáiz Driever, Svea Horn, P. Sievers, M. Prinz, M. Glatzel, Christian Mawrin, C. Hartmann, C. Monoranu, Liza Konnikova, F. Sahm, Stefan M. Pfister, Olaf Witt, David T. W. Jones, A. Koch, Claudia L Kleinman, David Capper, L. Walsh, N. Jabado, T. Milde
          </td>
          <td>2025-09-15</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="


 Alaska Native peoples experience high colorectal cancer (CRC) incidence and mortality rates, which Alaska tribal health organizations are working to address. To reduce the high rates, it is important to understand the biological mechanisms that drive risk and survival outcomes. The tumor microenvironment (TME) comprises diverse cell types, including cancer, stromal, and immune cells, whose composition and spatial organization may impact prognosis and disease progression. We performed spatial proteomics profiling on tissue microarrays from 97 Alaska Native patients with CRC using Akoya Biosciences’ PhenoCycler-Fusion system. We included 35 patients who died of CRC and 62 matched patients who did not die of CRC and lived at least as long as the patient they were matched to. Patients were matched on age at diagnosis, sex, tumor site, tumor stage, and year at diagnosis. We classified 2.90 million cells into 16 distinct cell types. We quantified immune and stromal cells as proportions of non-epithelial cells and epithelial cells as proportions of total cells. For spatial analysis, we identified eight cellular neighborhoods with distinct local cell composition and calculated spatial proximities between different cell types. We performed multivariable-adjusted logistic regression with generalized estimating equations to investigate associations of CRC-specific death with proportions of cell types and spatial proximities. We estimated odds ratios (OR) with 95% confidence intervals (CI) and calculated the false discovery rate (FDR) to account for multiple comparison using the Benjamin-Hochberg method. We found that higher proportions of stromal cells (OR for doubling of proportion=2.29, 95%CI=1.00-5.26) and higher proportion of M2 macrophages at the tumor center (OR=2.90, 95%CI=1.14-8.22) were associated with increased risks of CRC-specific death among 76 patients with microsatellite stable (MSS) tumors. We observed higher proportions of podoplanin (PDPN)-expressing stromal cells (OR=2.97 for doubling of proportion, 95%CI=1.28-7.08) and M1 macrophages (OR=2.42, 95%CI=1.05-5.69) within cellular neighborhoods predominantly composed of tumor epithelial cells among patients with CRC deaths compared to survivors, although these associations did not remain significant at FDR 0.05. Additionally, we observed that colocalization of PDPN-expressing stromal cells with CD8+ T cells, and colocalization of antigen-presenting cells with plasma cells, was associated with increased risk of CRC-specific death. In contrast, colocalization of B cells with stromal cells was associated with decreased risk (adjusted P=0.04 for all associations). Those associations remained significant among patients with MSS (adjusted P=0.03). This is the first study focusing on the TME cellular landscape among Alaska Native patients with CRC. Our results highlight the prognostic relevance of cellular composition and spatial organization within the TME, suggesting specific stromal and immune cell interactions as potential biomarkers for CRC outcomes.



 Hang Yin, Diana Redwood, Kimberly Smythe, Kristin L. Robinson, Daniel Jones, McGarry Houghton, Kevin C. Barry, Amanda L. Koehne, Elizabeth Donato, Cecilia Yeung, Minggang Lin, James J. Tiesinga, Tabitha A. Harrison, Sushma S. Thomas, Li Hsu, Jane C. Figueiredo, Li Li, Christopher I. Li, Ulrike Peters, Jeroen R. Huyghe. Spatial-resolved single-cell analysis of the tumor microenvironment in Alaska Native colorectal cancer patients reveals novel biomarkers for prognosis [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr A087.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fe434533ff8a8160b5fcdfdf886e32d3184d6f4" target='_blank'>
              Abstract A087: Spatial-resolved single-cell analysis of the tumor microenvironment in Alaska Native colorectal cancer patients reveals novel biomarkers for prognosis
              </a>
            </td>
          <td>
            Hang Yin, Diana Redwood, Kimberly Smythe, Kristin L. Robinson, Daniel Jones, Mcgarry Houghton, K. C. Barry, Amanda L Koehne, Elizabeth Donato, Cecilia Yeung, MingGang Lin, James Tiesinga, T. Harrison, Sushma Thomas, Li Hsu, Jane C Figueiredo, Li Li, Christopher I Li, Ulrike Peters, J. Huyghe
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Cell type markers have been instrumental to physiological and molecular investigation of the human brain and remain essential for annotating cell type clusters in single-cell expression data and for targeted validation studies. However, expression of canonical markers in the target cell type (which we termed as the expression ‘fidelity’) as well as expression in unrelated cell types (which we termed as the ‘background expression’) across cortical regions remains poorly characterized. Using nearly 500,000 high-quality single-nucleus profiles from 19 studies, we quantified marker fidelity, revealing substantial regional variability. We developed a statistical framework that aggregates annotated barcodes into pseudo-bulk profiles, applied rigorous performance metrics, and identified markers with improved fidelity, reduced background, and consistent expression across regions. This approach extended the canonical marker set for six major brain cell types and yielded superior subtype-specific markers. The resulting marker lists, and a user-friendly analysis interface, provide a valuable resource for cell type annotation and validation in neurological research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40d3defdba63314b1d51e6d4e42495e28376f453" target='_blank'>
              Expanding canonical cortical cell type markers in the era of single-cell transcriptomics
              </a>
            </td>
          <td>
            Dennis Joshy, Soojin V Yi
          </td>
          <td>2025-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="MYCN amplification without concurrent RB1 mutations characterizes a rare yet highly aggressive subtype of retinoblastoma; however, its precise developmental origins and therapeutic vulnerabilities remain incompletely understood. Here, we modeled this subtype by lentiviral-mediated MYCN overexpression in human pluripotent stem cell-derived retinal organoids, revealing a discrete developmental window (days 70–120) during which retinal progenitors showed heightened susceptibility to transformation. Tumors arising in this period exhibited robust proliferation, expressed SOX2, and lacked CRX, consistent with origin from primitive retinal progenitors. MYCN-overexpressing organoids generated stable cell lines that reproducibly gave rise to MYCN-driven tumors when xenografted into immunodeficient mice. Transcriptomic profiling demonstrated that MYCN-overexpressing organoids closely recapitulated molecular features of patient-derived MYCN-amplified retinoblastomas, particularly through activation of MYC/E2F and mTORC1 signaling pathways. Pharmacological screening further identified distinct therapeutic vulnerabilities, demonstrating distinct subtype-specific sensitivity of MYCN-driven cells to transcriptional inhibitors (THZ1, Flavopiridol) and the cell-cycle inhibitor Volasertib, indicative of a unique oncogene-addicted state compared to RB1-deficient retinoblastoma cells. Collectively, our study elucidates the developmental and molecular mechanisms underpinning MYCN-driven retinoblastoma, establishes a robust and clinically relevant human retinal organoid platform, and highlights targeted transcriptional inhibition as a promising therapeutic approach for this aggressive pediatric cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7456982cd84b09cab50502a3b5ab5c8e7a74c4cd" target='_blank'>
              Human Retinal Organoid Modeling Defines Developmental Window and Therapeutic Vulnerabilities in MYCN-Amplified Retinoblastoma
              </a>
            </td>
          <td>
            Jinkyu Park, Gang Cui, Jiyun Hong, H. Jeong, Minseok Han, Min Seok Choi, Jeong Ah Lim, Sanguk Han, Christopher Seungkyu Lee, Min Kim, Sangwoo Kim, Junwon Lee, S. Byeon
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background/Aims
Patient-derived organoids (PDOs) are promising preclinical models that replicate critical tumor features. However, intratumoral heterogeneity challenges the clinical utility of PDOs, especially in capturing diverse tumor cell subpopulations.


Methods
Single-cell transcriptomics was used to analyze PDOs from distinct sites within a single gastric cancer tumor, aiming to assess their ability to reflect intratumoral heterogeneity.


Results
The PDOs displayed similarities in gene expression but also exhibited distinct profiles. Single-cell analysis of PDOs revealed upregulation of markers for neuroendocrine tumors, which was validated via immunohistochemistry staining of neuron-specific enolase in the primary tumor. Notably, heat shock proteins showed significant variability among the PDOs, impacting immune responses. Tumors with abundant heat shock proteins are reported to have increased cytotoxic T cell activity.


Conclusions
Intratumoral heterogeneity poses challenges for PDO-based models, highlighting the need for comprehensive assessment. Despite their limitations, PDOs offer valuable insights into precision medicine for patients with gastric cancer, aiding in the development of therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9154970e5ea83aed48cb2a2d344b23752aade163" target='_blank'>
              Patient-Derived Organoids from Multiple Sites of a Single Tumor Recapitulates Intratumoral Heterogeneity in Patients with Gastric Cancer.
              </a>
            </td>
          <td>
            Bo Kyung Yoon, Y. Bae, Yeonjin Je, Seyeon Joo, Yuna Kim, Su-Jin Shin, Sungsoon Fang, Jie-Hyun Kim
          </td>
          <td>2025-09-05</td>
          <td>Gut and liver</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Pediatric high‐grade gliomas (pHGG) were first defined as a distinct entity in the 2021 fifth edition of the WHO classification of tumors of the central nervous system. These tumors, designated primarily as Grade 4, include the following subtypes: (1) diffuse midline glioma with H3‐K27 alterations (DMG, H3‐K27M), (2) diffuse hemispheric glioma with H3‐G34 mutations (DHG, H3G34M), and (3) diffuse pediatric‐type high‐grade glioma with wild‐type H3 and isocitrate dehydrogenase (pHGG, H3‐WT/IDH WT). Clinically, pHGGs are known for their poor outcomes and marked tumor heterogeneity. Despite this, the characteristics of the tumor microenvironment (TME) and the processes governing tumor cell lineage transitions remain incompletely understood. In this study, we used single‐cell RNA sequencing (scRNA‐seq) to analyze pHGG tumor cells (excluding infant‐type hemispheric gliomas). Through comprehensive bioinformatic approaches—including cell proportion analysis, Gene Ontology (GO) enrichment, metabolic activity inference via scMetabolism, proliferation gene scoring, stemness assessment by CytoTRACE2, SCENT, and pseudotime trajectory analysis with Monocle2—we thoroughly investigated the TME features and heterogeneity of these aggressive brain tumors. Our findings highlight the presence of oligodendrocyte precursor cell (OPC)‐like subpopulations, with epidermal growth factor receptor (EGFR)‐expressing OPC‐like cells emerging as a potential tumorigenic origin in diffuse midline gliomas due to their distinct stemness properties. Notably, platelet‐derived growth factor receptor alpha (PDGFRA)‐positive cells exhibit high specificity in DMG, suggesting greater diagnostic and therapeutic potential than EGFR. Next‐generation sequencing (NGS) and multiplex immunofluorescence analyses confirmed their distinct expression pattern, supporting PDGFRA as a key molecular marker. Moreover, OPC‐like cells at different differentiation states may drive lineage transitions in DMG. Together, these findings enhance our understanding of pHGG—especially DMG—and point to new avenues for targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/746a8877d1a92b076c1950fd12d0bc8dc2480909" target='_blank'>
              Single‐Cell Profiling of Pediatric High‐Grade Gliomas Reveals OPC‐Like Subpopulations Driving Tumorigenic Lineage Transitions
              </a>
            </td>
          <td>
            Tian Tian, Chong Huang, Lusheng Li
          </td>
          <td>2025-09-01</td>
          <td>Pediatric Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance.
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Despite notable advances with immune checkpoint inhibitors (ICIs) in esophageal squamous cell carcinoma (ESCC), their clinical efficacy remains limited, largely due to CD8⁺T cell exhaustion. Among these, progenitor exhausted T cells (Tpex) represent a key subset with stem cell–like features that sustain durable anti-tumor immunity. Methods We applied multi-color immunohistochemistry (mIHC) to determine the spatial distribution and clinical significance of Tpex cells within the tumor microenvironment (TME) of ESCC. Publicly available single-cell RNA sequencing (scRNA-seq) datasets were further analyzed to characterize Tpex cell phenotypes, differentiation trajectories, and intercellular communication networks. Results Tpex cells constituted a distinct subset of infiltrating CD8⁺T cells and represented a transitional stage of the exhaustion continuum. A higher degree of Tpex infiltration was significantly associated with improved overall survival in ESCC patients. Moreover, scRNA-seq data from patients treated with PD-1 blockade revealed that responders harbored markedly enriched Tpex populations compared with non-responders. Conclusion Our findings identify Tpex cells as a critical prognostic and immunologically active CD8⁺T cell subset in ESCC. Their abundance and functional engagement are closely associated with favorable clinical outcomes and response to PD-1 blockade. Furthermore, their stem cell-like properties may be pivotal in shaping durable anti-tumor immunity and could provide novel therapeutic targets to enhance the efficacy of PD-1-based immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08c5eac76128741742f8a923e04cbd692a6d7928" target='_blank'>
              Progenitor-exhausted T cell as prognostic indicator in esophageal squamous cell carcinoma: illuminating their key contribution to tumor immunity
              </a>
            </td>
          <td>
            Yi Liu, Hongwei Jiang, Zhang Fang, Bin Xu, Junjun Chen, Xiao Zheng, Renhao Geng, Lujun Chen
          </td>
          <td>2025-09-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Interpatient tumor heterogeneity manifests as multimodal distributions across genomic, transcriptomic, and microenvironmental profiles. This fundamentally violates the unimodal assumption of conventional machine learning models, impairing immune checkpoint blockade (ICB) response prediction. To resolve this limitation, we propose a heterogeneity-optimized framework that applies K-means clustering to stratify patients into biologically distinct hot-tumor and cold-tumor subgroups, demonstrating superiority over hierarchical/DBSCAN clustering. Subsequently, heterogeneity-optimized predictive models–a support vector machine for hot-tumor subtypes and a random forest for cold-tumor subtypes–were developed utilizing seven heterogeneity-associated biomarkers to circumvent unimodal constraints. The proposed model significantly enhances ICB response prediction in melanoma, NSCLC, other cancer types, and pan-cancer datasets, achieving a mean accuracy gain of at least 1.24% compared to 11 baseline methods. This performance improvement was consistently validated in an independent external cohort. Furthermore, the approach enables biologically interpretable precision immunotherapy by explicitly modeling multimodal heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea2e7a884fb718694551d16dff86f3e29cf9498" target='_blank'>
              Heterogeneity-optimized method for predicting immune checkpoint blockade response
              </a>
            </td>
          <td>
            Juan Liang, Qihang Guo, Shan Xiang, Yun Wang, Juntao Li
          </td>
          <td>2025-09-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Uveal melanoma (UM), the most common primary intraocular malignancy in adults, presents significant clinical challenges due to its high metastatic potential, pronounced hepatic tropism, and poor prognosis upon systemic dissemination. Despite established local therapies, nearly half of patients develop distant metastases, highlighting an urgent need for more effective systemic strategies. Recent advances in single-cell omics technologies (e.g., scRNA-seq, scATAC-seq, spatial transcriptomics) have revolutionized our understanding of UM pathobiology. These approaches have meticulously delineated the complex tumor heterogeneity, immunosuppressive microenvironment, and key molecular drivers—including novel macrophage subsets (e.g., immunosuppressive MΦ-C4), senescent endothelial cells, and non-canonical immune checkpoint expression—providing unprecedented resolution for identifying actionable therapeutic targets. Concurrently, innovations in materials science and biomedical engineering offer transformative opportunities for precision therapy. Engineered nanocarriers, biodegradable implants, and advanced gene therapy vectors (e.g., tropism-enhanced AAVs, CRISPR-Cas9 systems) enabled targeted drug delivery, sustained release, and genetic modulation tailored to the eye’s unique anatomy and immune privilege. This review synthesizes these converging frontiers, outlining how the integration of multi-omics insights with smart biomaterials can overcome current therapeutic limitations. We catalog emerging material-based platforms applicable to UM and summarize validated molecular targets (e.g., GNAQ/GNA11, YAP/TAZ, BAP1, c-Met, CXCR4). Furthermore, we propose an interdisciplinary paradigm spanning combinatorial targeted therapies, immunomodulation, minimally invasive devices (e.g., robotic radiosurgery), and engineered delivery systems. By bridging mechanistic discovery with translational engineering, this synergy holds significant promise for advancing precision medicine and improving clinical outcomes in UM, ultimately facilitating the transition from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0323cee7d6c89a49fe569ae452832b7a890d38bd" target='_blank'>
              Integrating single-cell omics and materials science for uveal melanoma: from mechanistic insights to precision therapeutics
              </a>
            </td>
          <td>
            Shouyong Fu, Changfei Li
          </td>
          <td>2025-09-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Ovarian cancer (OvCa) remains largely refractory to immune checkpoint blockade (ICB), owing to a profoundly immunosuppressive tumor microenvironment (TME). Single-cell RNA sequencing (scRNA-seq) of patient tumors has revealed a TME enriched in immunosuppressive macrophages and characterized by exclusion of T cells from the tumor core, features strongly associated with ICB resistance. However, the molecular mechanisms underlying the establishment of this immune landscape remain poorly understood. To identify regulators of OvCa immunity, we employed Perturb-map, a first-of-its-kind spatial functional genomics platform that enables multiplexed CRISPR/Cas9 knockout of candidate genes while preserving the spatial context of tumor–TME interactions in situ. We targeted 35 genes, prioritized through ligand–receptor interaction analysis of human OvCa scRNA-seq data, in an immunocompetent OvCa mouse model. Among these, IL-4 emerged as a key driver of ICB resistance. While IL-4 knockout did not affect tumor growth in untreated animals, it substantially remodeled the local TME and rendered tumors responsive to anti-PD-1 therapy. Unexpectedly, we found that both mouse and human ovarian cancer cells are a source of IL-4, a cytokine traditionally restricted to immune cells. Analysis of patient tumors revealed that tumor-associated macrophages exhibit a robust IL-4, induced transcriptional signature, implicating IL-4 in shaping the immunosuppressive TME in human disease, consistent with our preclinical findings (Cell, 2024). Furthermore, we observed that standard-of-care chemotherapy regimens, including carboplatin, paclitaxel, and cisplatin—induce a dose-dependent increase in IL-4 expression in both murine and human OvCa cell lines. This chemotherapy-induced IL-4 upregulation may contribute to immune evasion and recurrence following initial therapeutic response. Therapeutically, combined blockade of IL-4Rα and PD-1 significantly delayed tumor progression in vivo compared to monotherapy, suggesting that IL-4 signaling is both a necessary and targetable mechanism of immune resistance. These findings are clinically actionable, as Dupilumab, an FDA-approved anti-IL-4Rα antibody, is already in clinical use and well-tolerated. Our results uncover a previously unrecognized, chemotherapy-induced immunoregulatory axis in ovarian cancer and provide a strong rationale for therapeutic strategies that combine IL-4 pathway inhibition with ICB to overcome resistance and improve patient outcomes.



 Gurkan Mullaoglu, Hunter Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian Brown, Alessia Baccarini. Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462188b4ccc770fdbf7b28c6a3fda711e0408972" target='_blank'>
              Abstract PR015: Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Gurkan Mullaoglu, Hunter T. Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian D. Brown, Alessia Baccarini
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Quantitative image analysis is essential for advancing stem cell biology, developmental studies, and drug discovery, yet most workflows still rely on manual or semi-quantitative scoring that is slow, subjective, and poorly scalable. A major challenge is converting complex colony morphologies into reproducible, high-dimensional datasets. To address this gap, we developed ColonyQuant, an open-source platform that integrates automated colony segmentation, alkaline phosphatase (AP) intensity quantification, morphometric profiling, and statistical classification into a single workflow. ColonyQuant computes per-colony functional readouts alongside comprehensive shape descriptors, capturing both staining intensity and structural features in a unified framework. Applied to embryonic stem cells (ESCs) treated with a selective KDM4 histone-demethylase inhibitor, ColonyQuant revealed dose-dependent reductions in colony area and integrated AP signal, together with systematic remodeling of morphometric metrics. Multivariate analyses robustly stratified treatment groups and identified intensity and solidity as principal features capturing dose-dependent colony responses. By transforming subjective scoring into objective, scalable, and biologically interpretable phenotyping, ColonyQuant provides a reproducible platform for stem cell research and high-content screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57357026b197a6c463b98555927514215f3da5a" target='_blank'>
              Automated Image-Based Profiling of Pluripotent Stem Cell Colonies
              </a>
            </td>
          <td>
            Rui Geng, Benjamin L. Kidder
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Glioblastoma is the most common malignant primary brain tumor in adults and associated with a poor prognosis and a median survival of approximately 15 months, largely due to the lack of effective therapies and inevitable recurrence. A key challenge is the spatial and cellular heterogeneity, evident on MRI as a necrotic center, a contrast-enhancing tumor core, and a non-enhancing, T2/FLAIR-hyperintense infiltration zone. The diffuse infiltration of tumor cells into surrounding brain tissue impedes complete surgical resection and drives relapse from persisting residual cells. While regional molecular differences have been well characterized, their functional implications - particularly drug sensitivities - remain insufficiently understood.



 To address this, we performed MRI- and 5-aminolevulinic acid (5-ALA)-guided tissue sampling from the tumor core and from the infiltration zone of patients with glioblastoma. We utilized pharmacoscopy—a single-cell, microscopy-based drug screening platform—to evaluate region- and tumor cell type-specific drug responses. Cancer cell types were defined based on differential expression of nestin (marking stem-like cells) and S100B (marking more differentiated tumor cells).



 Profiling a library of 59 drugs across region-specific tumor samples from 23 patients revealed pronounced heterogeneity in drug responses across patients, tumor regions and tumor cell states. To overcome this heterogeneity, we computationally predicted complementary drug pairs and validated 20 drug combinations in an independent cohort. The combinations demonstrated enhanced efficacy compared to single agents and effectively addressed glioblastoma’s spatial, cellular, and inter-patient heterogeneity. Notably, drug combinations of oncology and neuroactive drugs outperformed those involving agents from the same drug class (oncology + oncology or neuroactive + neuroactive). Ongoing mechanistical explorations reveal first insights into mechanisms of the complementary drug efficacy.



 Our findings underscore the limitations of monotherapies for patients with glioblastoma and highlight the potential of rationally designed drug combinations tailored to the tumor’s spatial and functional complexity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60062d1c4566c5af52ee0940de47f3ed99103b0" target='_blank'>
              OS12.5.A SPATIALLY RESOLVED FUNCTIONAL PROFILING FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Buck, S. Lee, I. Sakic, R. Wegmann, J. Mena, F. Vasella, J. Friesen, L. Regli, E. Le Rhun, A. Zeitlberger, N. Tatari, G. Hutter, M. Neidert, M. Weller, B. Snijder, T. Weiss
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The human brain, a pinnacle of biological complexity, comprises a diverse array of cell types that regulate cognition and maintain neural homeostasis. Advances in single-cell transcriptomics have revolutionized neuroscience by enabling high-resolution molecular profiling, revealing unprecedented insights into cellular heterogeneity, lineage dynamics, and disease-associated states. Large-scale brain-mapping initiatives have identified numerous novel cell types, yet their functional roles in health and disease remain poorly understood. This review synthesizes current knowledge of brain cell diversity, from neurogenesis to pathological states, and highlights key gene markers that define cellular identity and function. By integrating insights from single-cell transcriptomics, we explore how cellular diversity shapes brain function and contributes to disease mechanisms, providing a foundation for future research and translational applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c5944247d3c48d1d1cd248f9d8b0f5a130e7cb" target='_blank'>
              Mapping human brain cell type origin and diseases through single-cell transcriptomics
              </a>
            </td>
          <td>
            Anjana Soorajkumar, Bipin Balan, Nasna Nassir, Hosneara Akter, Zaha Shahin, B. Berdiev, Marc Woodbury-Smith, Reem Khalil, Babacar Cisse, Hani T S Benamer, Mohammed Uddin
          </td>
          <td>2025-10-06</td>
          <td>Translational Psychiatry</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background/Objectives: Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with poor prognosis. T cell exhaustion (TEX) is a key factor in tumor immune evasion and therapeutic resistance. In this study, we integrated single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data to characterize TEX-related transcriptional features in HCC. Methods: We first computed TEX scores for each sample using a curated 65-gene signature and classified them into high-TEX and low-TEX groups by the median score. Differentially expressed genes were identified separately in scRNA-seq and bulk RNA-seq data, then intersected to retain shared candidates. A 26-gene prognostic signature was derived from these candidates via univariate Cox and LASSO regression analysis. Results: The high-TEX group exhibited increased expression of immune checkpoint molecules and antigen presentation molecules, suggesting a tumor microenvironment that is more immunosuppressive but potentially more responsive to immunotherapy. Functional enrichment analysis and protein–protein interaction (PPI) network construction further validated the roles of these genes in immune regulation and tumor progression. Conclusions: This study provides a comprehensive characterization of the TEX landscape in HCC and identifies a robust gene signature associated with prognosis and immune infiltration. These findings highlight the potential of targeting TEX-related genes for personalized immunotherapeutic strategies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a1103280627c731b735d281df63ab3e636c5d0" target='_blank'>
              Bulk and Single-Cell Transcriptomes Reveal Exhausted Signature in Prognosis of Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Ruixin Chun, Haisen Ni, Ziyi Zhao, Chunlong Zhang
          </td>
          <td>2025-08-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73e8d2f1aaaa7f0e435a0bcfed4046ad25f2a01b" target='_blank'>
              Deciphering macrophage heterogeneity and factors driving M2 polarization in lung adenocarcinoma through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Meiling Sheng, Beiwei Yu, Qunzhi Wang, Yuanchao Xiao, Ming Ling, Xiaoming Wu
          </td>
          <td>2025-10-16</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Macrophages are highly plastic immune cells that adopt diverse functional states in response to the local microenvironment. The traditional M1/M2 polarization model that has long been used to describe macrophage activation is insufficient to capture the full spectrum of macrophage diversity observed in vivo. Advances in single-cell RNA sequencing (scRNA-seq) have revealed that macrophages exist in a continuum of transcriptional states formed by tissue-specific and disorder-specific cues. This insight has led to the recognition of disorder-specific macrophages, defined as macrophage subpopulations that emerge in response to pathological stimuli and play unique roles in disease progression. These macrophages exhibit distinct transcriptional signatures, epigenetic modifications, and functional properties shaped by their ontogeny and microenvironmental signals, arising from the reprogramming of resident macrophages or the differentiation of bone marrow–derived progenitors. Notable examples include macrophages in chronic infections (e.g., tuberculosis), immunosuppressive tumor-associated macrophages, lipid-associated macrophages in obesity, and disease-associated microglia in neurodegeneration. These subsets exhibit unique regulatory mechanisms, including enhancer remodeling driven by histone H3 lysine 27 acetylation in non-alcoholic steatohepatitis, CCAAT enhancer binding protein α-mediated differentiation in obesity, and Jmjd3-IRF4 axis control in allergic inflammation. Additionally, their function and fate are strongly influenced by their subtissular niche, as evidenced by crown-like structures in adipose tissue, tumor microenvironments, fibrotic lesions, and granulomas, where distinct microenvironmental cues shape macrophage behavior. Furthermore, interindividual heterogeneity in macrophage function, driven by genetic polymorphisms, is increasingly recognized, highlighting the role of host genetic background in disease susceptibility and macrophage-driven pathology. Here, we review the conceptual evolution of the disorder-specific macrophage, tracing its origins from the limited M1/M2 model to its refinement through scRNA-seq-based classification. We summarize the ontogeny, transcriptional regulation, and spatial heterogeneity of these macrophages across various disorders, emphasizing how the subtissular niche dictates functional specialization. Finally, we discuss potential therapeutic strategies targeting disorder-specific macrophage subsets, highlighting the need for integrative multi-omics approaches to refine their classification and functional characterization. Understanding the regulatory networks that govern disorder-specific macrophages will advance our knowledge of macrophage biology while facilitating the development of precision medicine for immune-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9feef19310b98a25e6e67c81bcaeb9daa262ec04" target='_blank'>
              Functional diversity of disorder-specific macrophages involved in various diseases
              </a>
            </td>
          <td>
            Yuichi Mitsui, Takashi Satoh
          </td>
          <td>2025-10-01</td>
          <td>Inflammation and Regeneration</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is characterized by fast progression, infiltrative growth pattern and a high rate of relapse. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, i.e. a hypoxic niche, a leading-edge niche, and a peri-vascular niche, in which malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment, including immune cells, astrocytes, and vascular cells.



 By analysis of single-cell transcriptomic data of human GBM and transgenic mouse models of GBM, we unexpectedly identified pericytes intimately associated with the endothelium as the most active paracrine signaling hub within the tumor parenchyma. Exclusive signaling axes emanating from pericytes were received by endothelial cells, malignant cells, astrocytes, and immune cells. Depletion of pericytes through genetic engineering in several different transgenic and orthotopic mouse models of GBM demonstrated accelerated tumor progression, a disrupted blood-brain-barrier, and premature death of pericyte-poor mice. Mechanistic studies revealed that pericyte deficiency altered the cellular composition of GBM, remodeled the endothelium, and impacted on the immune cell landscape. Specifically, endothelial cells deprived of pericyte association upregulated Colony Stimulating Factor (CSF)-1, which in turn attracted peri-vascular tumor-associated macrophages polarized towards an immune-suppressive phenotype. In the absence of pericytes, the recruited macrophages expressed Hepatocyte Growth Factor (HGF), which reinforced activation of its receptor tyrosine kinase c-Met on a subset of GBM cells harboring an extreme mesenchymal-like state. Indeed, orthotopic implantation of isolated c-Met-expressing GBM cells corroborated their superior tumor-initiating capability and invasive phenotype, compared to cells negative for c-Met.



 We infer that the pericyte represents a critical modulator of GBM development by orchestrating a tumor-suppressive microenvironment; our findings thus highlight the importance of pericyte preservation in the face of current and future GBM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a67e2a70cd46f28d9a303424953a15720ec17e" target='_blank'>
              P02.01.B PERICYTES ORCHESTRATE A TUMOR-SUPPRESSIVE MICROENVIRONMENT BY IMPINGING ON THE CROSSTALK BETWEEN MACROPHAGES AND TUMOR-INITIATING CELLS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            S. Braun, P. Bolivar, C. Oudenaarden, J. Sjölund, M. Bocci, K. Harbst, M. Safaee Talkhoncheh, B. Phung, E. Cordero, E. Johansson, R. Rosberg, G. B. Jönsson, A. Pietras, K. Pietras
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="T follicular helper (Tfh) cells are central to the adaptive immune response and exhibit remarkable functional diversity and plasticity. The complex nature of Tfh cell populations, inconsistent findings across experimental systems and potential differences across species have fueled ongoing debate regarding core regulatory pathways that govern Tfh differentiation. Many studies have experimentally investigated individual proteins and circuits involved in Tfh differentiation in limited contexts, each providing only a partial understanding of the process. To address this, we adopted a novel multi-scale network systems approach that incorporates both regulatory and protein-protein interactions. Our approach integrates diverse data types, captures regulation across multiple levels of immune system organization, and recapitulates known drivers. Further, we discover a core Tfh gene set that is conserved across tissue types and disease contexts, and is consistent across data modalities - bulk, single-cell and spatial. While components of this set have been individually reported, a novel aspect of our work lies in the discovery, characterization, and connectivity of this core signature using a single unbiased approach. Using this method, we also uncover a novel function of IL-12, a molecule with reported conflicting functions, in the regulation of Tfh differentiation. Notably, we find that, in both humans and mice, IL-12 is permissive for the differentiation of Tfh precursors, but blocks subsequent differentiation into GC Tfh cells. Overall, this work elucidates novel networks with unexplored roles in governing Tfh cell differentiation across species and tissues, paving the way for novel -therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cdef08e00798c3da4a3f940c2d60360ad82720" target='_blank'>
              A unified network systems approach uncovers a core novel program underlying T follicular helper cell differentiation
              </a>
            </td>
          <td>
            Alisa A. Omelchenko, Syed A. Rahman, Vinayak V. Viswanadham, Grace J. Yuen, Perla M. Del Rio Estrada, Valentino D’Onofrio, Yijia Chen, Na Sun, Hamid Mattoo, Chinmay G. Varma, Gonzalo Salgado, Maribel S. Nava, Lady C. Ruiz, Dafne D. Rivera, S. A. Rios, Sudhir P. Kasturi, Susan P. Ribeiro, M. Shlomchik, Amanda C. Poholek, Shiv S. Pillai, Rebecca A. Elsner, V. Mahajan, Jishnu Das
          </td>
          <td>2025-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="Despite significant efforts, Diffuse Midline Gliomas (DMGs) remain incurable. Although promising results are obtained in preclinical studies, most approaches have failed to improve survival in these young patients. In this context, by integrating elements of the biophysical tumor microenvironment known to regulate the response to therapies, a new preclinical tool is developed to better evaluate the efficacy of antitumoral strategies. For this purpose, a novel DMG-on-Chip (DoC) is engineered composed of a 3D dense tumor disk embedded in an extracellular matrix, which is accessible for real-time monitoring using wide-field phase contrast or confocal fluorescence microscopy. By driving the oxygen supply within the chip solely in a radial manner, a hypoxia gradient is established that can be associated with changes in DMG cell phenotype, proliferation, and rewiring of metabolic and stress transcriptomic signatures. Finally, DoC is used to analyze the spatial heterogeneity of the response of DMG cell lines and patient-derived 3D cultures to treatments through cell segmentation. Altogether, these interdisciplinary characterizations validate the new tool as an interesting healthcare solution to understand how cell responses are modulated by biophysical or biochemical cues in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a25e403c641a7729cb13c93c687243726c53a1" target='_blank'>
              Novel Diffuse Midline Glioma-on-Chip Recapitulating Tumor Biophysical Microenvironment to Assess the Heterogeneity of Response to Therapies.
              </a>
            </td>
          <td>
            Lisa Terrassoux, Calypso Hazard, Agathe Laratte, Joanne Balsamelli, Mélanie Arcicasa, Elisa Fong, Léo Baland, Fabrice Soncin, Giovanni Cappello, C. Maurage, Sebastien Janel, Antony Bazir, Fabrizio Cleri, Anthony Treizebré, Eric Lartigau, Yousr Rekik, Samuel Meignan, Alessandro Furlan
          </td>
          <td>2025-09-29</td>
          <td>Small</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains a highly lethal malignancy, in part due to its heterogeneous tumor microenvironment. Most PDAC tumors are poorly vascularized and embedded within densely fibrotic stroma; however, in prior computational analyses, we identified a subset of PDAC tumors with hyper-vascular features. In this study, we sought to determine the extent of vascular heterogeneity in PDAC and its functional consequences. Analysis of human tumor specimens revealed that approximately 10-20% of human tumors display features of hyper-vascularity. Using a panel of clonal murine cell lines that recapitulate this heterogeneity, we demonstrate that VEGFA-mediated angiogenesis does not account for the increased vessel density. Functionally, we further show that hyper-vascular tumors display enhanced sensitivity to both chemotherapy and anti-angiogenic therapy. These findings help explain the limited efficacy of VEGFA-targeted therapies in unselected PDAC cohorts and highlight vessel density as a potential biomarker for patient stratification.



 Samantha B. Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo I. Orlen, Samuel D. Zwernik, Kailee M. Hartway, Nina G. Steele, Agnieszka K. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger. Vascular heterogeneity influences treatment response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05cec4b9af7cf61e156104d3e634ae5d707e4e9c" target='_blank'>
              Abstract A127: Vascular heterogeneity influences treatment response in pancreatic cancer
              </a>
            </td>
          <td>
            Samantha B Kemp, William B. Sullivan, Benjamin M. Kahn, Jonathan H. Sussman, Il-Kyu Kim, Takeshi Katsuda, Sidney Mahan, Margo Orlen, Samuel D Zwernik, Kailee M Hartway, Nina G. Steele, A. Witkiewicz, Erik S. Knudsen, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Merging evidence suggests that cellular fitness competition between malignant cells and the tumor microenvironment (TME) dictates tumor progression. We recently identified Flower isoforms as molecular “fitness fingerprints,” where cancer cells expressing Flower-Win gain dominance over surrounding cells marked with Flower-Lose. In this study, we uncover that loss of stromal fitness is not an inherent trait but rather a cancer-driven event. Tumor cells secrete a specific long non-coding RNA, Tu-Stroma, via exosomes, which reprograms splicing of the Flower locus in TME cells, enforcing Flower-Lose expression and thereby reducing stromal competitiveness. This cancer-induced shift establishes a permissive niche that accelerates growth, metastatic dissemination, and ultimately shapes host survival in ovarian cancer. Therapeutically, blockade of Flower using a fully humanized monoclonal antibody (Aenya) markedly suppressed tumor expansion and peritoneal spread in patient-derived xenograft models. Importantly, prophylactic administration preserved the integrity of intraperitoneal organs even in the presence of highly aggressive tumor cells. These findings highlight a previously unrecognized mechanism of cancer-driven stromal takeover and establish Flower as a first-in-class therapeutic target with broad implications for preventing metastatic colonization.



 Rajan G. Gogna. Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0864f00ed3ab31470a0c2e77e5565e26d50bb11a" target='_blank'>
              Abstract IA008: Discovery of fitness fingerprints and flower monoclonal ab therapy in ovarian cancer
              </a>
            </td>
          <td>
            Rajan Gogna
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Treatment of locally advanced unresectable pancreatic cancer remains a major clinical challenge due to pronounced heterogeneity and resistance to standard regimens. Increasing evidence highlights the critical role of the tumor microenvironment (TME) in shaping therapeutic response and driving drug resistance. In this minireview, we summarize recent advances in TME phenotyping and its potential to guide precision therapy. A four-dimensional framework integrating stromal, immune, genomic, and metabolic features has been proposed to better characterize TME heterogeneity. Preclinical and clinical studies indicate that strategies targeting the stroma, modulating immunity, or exploiting genomic vulnerabilities such as homologous recombination deficiency may enhance the efficacy of chemotherapy, immunotherapy, and targeted agents. Dynamic biomarkers, including circulating tumor DNA and carbohydrate antigen 19-9, also show promise for real-time therapy adaptation, although their clinical application remains limited. By synthesizing current evidence, we emphasize the importance of individualized treatment strategies that account for TME complexity. While encouraging, the translation of multiomics phenotyping and biomarker monitoring into routine clinical practice requires standardization, prospective validation, and integration of novel technologies. Future research should focus on establishing reproducible TME-guided models to enable dynamic and personalized therapy for patients with unresectable pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9ebed50c75b679e8298f471ae2f8ee5933e4197" target='_blank'>
              Tumor microenvironment phenotyping guides precision therapy in unresectable pancreatic cancer
              </a>
            </td>
          <td>
            Kai Zhao, , Yong-Sheng Yang, Xiao Xiao
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Capillary malformation (CM) is a congenital vascular anomaly affecting the skin, mucosa, and brain, yet the understanding of its vascular pathogenesis remains limited. Methods: We applied spatial whole-transcriptome profiling (GeoMx) and gene set enrichment analysis within CM lesions at single vasculature level. Differentially expressed genes were validated by immunofluorescence staining. Phosphoproteomics was profiled to uncover lesion-wide phosphorylation sites on proteins. Single-cell RNA sequencing was performed on CM-derived induced pluripotent stem cells (iPSCs) to determine differentiation trajectories of lesional vascular lineages. In silico gene perturbation was used to predict candidate genes for modulating vascular pathological progression, followed by functional validation in CM iPSC-derived endothelial cells (ECs) using a Tet-on system. Results: A spatial transcriptomic atlas was constructed, and pathological landscape of individual CM vasculature was delineated. CM vessels exhibited hallmarks of endothelial-to-mesenchymal transition (EndMT), including disruption of adherens junctions (AJs), vascular identity transitions, and metabolic remodeling. Phosphoproteomics confirmed that differentially phosphorylated proteins were enriched in EndMT- and AJ-related pathways. Aberrant expression of venous transcriptional factor NR2F2 was observed in lesional ECs and correlated with progressive enlargement from capillaries to larger-caliber vessels containing multiple layers of smooth muscle cells (SMCs). In CM iPSCs, differentiation course yielded reduced ECs but increased SMCs. In silico knockout simulation predicted NR2F2 as a crucial regulator of facilitating SMC phenotype in CM. Consistently, enforced NR2F2 expression during iPSC differentiation suppressed endothelial markers while inducing SMC-associated genes. Conclusions: Single CM vasculature displays pathological hallmarks characterized by EndMT and AJ disruption, leading to progressive vascular remodeling. NR2F2 functions as a central regulatory factor orchestrating the acquisition of the SMC phenotype, thereby representing a potential therapeutic target in CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1fa98536c86180270a8a6d151af2637cfd03767" target='_blank'>
              Single-vessel transcriptome map pathological landscapes and reveal NR2F2-mediated smooth muscle cell phenotype acquisition in capillary malformations
              </a>
            </td>
          <td>
            Vi Nguyen, Irving Mao, Siwuxie He, Isabella Castellanos, Mackenzie Azuero, Marcelo L Hochman, Yin Rong, L. Pernomian, Elliott H Chen, Harold I Friedman, Yan-Hua Chen, Qun Lu, Daping Fan, Camilla F Wenceslau, Dong-bao Chen, Stuart Nelson, Anil G Jegga, Yunguan Wang, Wenbin Tan
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Brain metastases often originate from circulating tumor cells (CTCs) that become arrested within cerebral microvessels. Recent findings indicate that this process induces localized hypoxia, which in turn upregulates angiogenic mediators such as Angiopoietin-2 (Ang-2), thereby promoting vascular remodeling and facilitating extravasation of tumor cells into the brain parenchyma. Nevertheless, the early transcriptional landscape of the adjacent brain tissue—the so-called pre-metastatic niche—remains poorly defined.



 To simulate early metastatic seeding, syngeneic murine lung and breast carcinoma cells were injected into the right carotid artery of immunocompetent mice. The contralateral, non-injected hemisphere served as an internal control to allow intra-animal comparisons, reduce biological variability, and minimize animal usage. Brain tissue was collected at 1 and 4 days post-injection, processed via formalin fixation and paraffin embedding, and subjected to histological evaluation. Comparative bulk RNA sequencing of tumor-bearing and control hemispheres was performed, complemented by high-plex spatially resolved single-cell transcriptomic profiling to detect localized gene expression patterns.



 Bulk RNA sequencing revealed no significant transcriptional differences between hemispheres. In contrast, spatial transcriptomic analysis uncovered robust, spatially restricted gene expression changes in the tumor-bearing hemisphere. These alterations involved genes associated with vascular remodeling, hypoxia-responsive pathways, and neuronal connectivity. Notably, spatial mapping demonstrated that the expression of several key transcripts peaked near sites of arrested tumor cells and declined with increasing distance, indicative of a highly localized, tumor-induced host response.



 This study reveals distinct, spatially confined transcriptional alterations that define the early pre-metastatic niche in the brain. By leveraging advanced spatial single-cell transcriptomics, we provide a powerful framework to dissect the molecular cues underlying early metastatic interactions within the native tissue microenvironment. These insights not only deepen our understanding of early brain metastasis biology but also inform the development of innovative strategies for therapeutic intervention at the pre-metastatic stage.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6ab71d6adf66b6036a2c3fb3715feb43793308" target='_blank'>
              P18.25.B SPATIALLY CONFINED TRANSCRIPTIONAL RESPONSES CHARACTERIZE THE EARLY PRE-METASTATIC NICHE IN BRAIN METASTASIS
              </a>
            </td>
          <td>
            W. R. Hube, F. Schallerer, L. Bergmayr, R. Piecyk, T. Eska, K. J. Mueller, E. Nicolaishvili, L. Sevenich, K. Unger, P. Jurmeister, L. von Baumgarten, P. Harter
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Brain metastases (BM) critically reduce breast cancer (BC) patients’ survival. Extravasation is pivotal for BM development, but the underlying events at the blood-brain barrier (BBB) remain elusive. We aimed to unravel the players and mechanisms governing BC cells (BCCs)-BBB interaction. For that, mixed cultures of human brain microvascular endothelial cells (HBMECs), mimicking the BBB, and brain-tropic triple-negative BCCs (MDA-MB-231 Br4), or non-brain-tropic (MDA-MB-231) or non-metastatic cells (MCF-7) were established. Temporal and spatial analysis of BCCs-BBB interactions (live-cell imaging automated microscopy), and assessments of endothelial-to-mesenchymal transition (EndMT) markers, transcription factors, cytoskeletal proteins, and morphology (immunocytochemistry) were performed. BBB integrity (permeability, transendothelial electrical resistance) and endothelial migration (wound-healing) were also assessed. Our results revealed that contrasting with non-metastatic and non-brain-tropic cells, BCCs quickly developed an invasive, migratory phenotype, characterized by invadopodium formation and reduced roundness. Spatial analysis showed different positioning of BCCs relative to the BBB endothelium over time, with 14% of BCCs transmigrated after 3 h, compromising BBB integrity through endothelial holes, reduced tightness, and increased permeability. Prior to transmigration, alterations in adhesion markers (E-selectin, ICAM-1, CD24, CD34, β3-integrin, Sialyl-Lewis X) were observed. EndMT was also evident by decreased endothelial (β-catenin and pan cytokeratin) and increased mesenchymal (vimentin, neuronal-cadherin, Slug, ZEB1) markers, elongation (RhoA, α-SMA), nuclear deformation, and migratory capacity. Caveolin-1 silencing in HBMEC decreased BCCs transmigration. This study reveals significant BBB phenotypic and structural changes, facilitating both paracellular and transcellular BCCs transmigration. These findings provide advanced understanding of BCCs trafficking across the BBB, aiding strategy development to prevent extravasation and BM. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-01985-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58375842dd28bad8d863cc4ae660178921a26a5b" target='_blank'>
              Unveiling the intricate dynamics of the interplay between triple-negative breast cancer cells and the blood-brain barrier endothelium
              </a>
            </td>
          <td>
            Ana Rita Garcia, Ana Rita Vaz, Rui Malhó, Hugo M. Botelho, Inês Figueira, M. A. Brito
          </td>
          <td>2025-08-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The spatial organization and composition of the tumor-immune microenvironment (TME) play a critical role in shaping the progression of many solid cancers, but the organization of the TME in primary prostate cancer (PCa) remains poorly characterized. We therefore profiled the abundance and spatial distributions of major cell types involved in adaptive immunity in 29 radical prostatectomy specimens stratified into high (HGG; n=14) and low Gleason-grade (LGG; n=15). Compared to LGG, HGG PCa exhibited significantly greater B and T cell infiltration with many immune cells organized into clusters, some of which resembled tertiary lymphoid structures (TLSs). In HGG tumors, these clusters were dense, symmetric, rich in PD-1+ T cells, and frequently proximate to the tumor compartment. LGG clusters were less well organized, and T cell depleted. Thus, a subset of high-grade PCa harbor organized immune clusters that may play a role in tumor control and contain therapeutically targetable T and B cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e5e762dad0168906d48c0967522a47db6d0ac" target='_blank'>
              Spatially organized lymphocytic microenvironments in high grade primary prostate tumors
              </a>
            </td>
          <td>
            Ali Amiryousefi, Jeremiah Wala, Jia-Ren Lin, Brian Labadie, A. Atmakuri, Z. Maliga, Eamon Toye, K. Chaudagar, Madeleine S Torcasso, Shannon Coy, G. Fanelli, Brigette Kobs, F. Socciarelli, Andréanne Gagné, Eliezer Van Allen, Akash Patnaik, P. Sorger
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal tumor. PDAC also shows poor response to many conventional treatments and is refractory to immunotherapy. We hypothesize that these challenges arise from the highly heterogeneous and immunosuppressive tumor microenvironment (TME) in PDAC. Therefore, to comprehend the players behind this immunosuppressive TME, we integrated 10 publicly available datasets into a robust PDAC scRNA atlas of 941.608 cells covering 201 patients (159 PDAC, 24 non-cancer, and 18 metastatic PDAC). The TME showed distinct composition for each tissue, predominating a fibro-inflammatory phenotype with infiltrating Tregs and dysfunctional CD8 T cells in primary PDACs. In contrast, adjacent normal tissue exhibited enrichment of tissue-resident memory CD8 T cells (p < 0.05). Metastatic PDAC was characterized by a reduction in stromal cells combined with myeloid and naive T cells infiltration. Comparison between treated and untreated PDAC revealed the enrichment of cancer-associated fibroblasts (CAFs) in treated patients. Besides, CD8 T cells from untreated patients exhibited higher levels of genes previously linked to immunotherapy response compared to treated patients. Non-negative matrix factorization (NMF) of the malignant cells unraveled 8 distinct metaprograms (MP). MP3 and MP7 overlapped with the well-known Classical subtype, whereas MP5 correlated with the Basal subtype. MP2 exhibited high expression of MHC-II genes and was enriched in metastatic lesions. Next, we investigated the tumor architecture using public spatial transcriptomic datasets. Spot-level deconvolution revealed 12 distinct cellular niches. Deconvolution of these spatial niches in a large cohort of bulk RNA samples uncovered unique clusters associated with patient survival, providing novel insights into TME biology and its clinical implications.



 Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, Alessandro Bersch Osvaldt, Simone Marcia Dos Santos Machado, Eduardo Cremonese Filippi-Chiela, Cristina Bonorino. Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B126.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a30afbd64e95a911caaa3a76b2513c14671d0c3e" target='_blank'>
              Abstract B126: Single-cell and spatially resolved atlas of pancreatic cancer reveals immunophenotypes associated with clinical outcome
              </a>
            </td>
          <td>
            Gabriel Francisco Pozo de Mattos Pereira, Julia Crispim da Fontoura, Marvin Paulo Lins, A. Osvaldt, Simone Marcia dos Santos Machado, E. Filippi-Chiela, Cristina Bonorino
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Ovarian cancer remains the most lethal gynecologic malignancy and a leading cause of cancer-related deaths among women. Adoptive cell therapy (ACT) using tumor-infiltrating lymphocytes (TILs) has shown success in melanoma and correlates with survival in ovarian cancer. However, its application remains limited, partly due to inefficient expansion of functional, tumor-specific T cells.



 We developed a novel ex vivo expansion platform for ovarian TILs, integrating metabolic and phenotypic reprogramming with selective enrichment for tumor-reactive populations. Tissue from nine patients with stage III/IV high-grade epithelial ovarian carcinoma was collected for banking, NGS, and TIL expansion. Four had matched TILs expanded pre- and post-neoadjuvant chemotherapy. The initial priming phase employed our published cytokine cocktail designed to induce Th1/17 and Th17/Tc17 polarization, to promote effector differentiation while preserving stemness. TILs were cultured for five days in this cocktail to enhance metabolic fitness and anti-tumor potential. CD137+ (4-1BB+) T cells were isolated based on activation-induced co-stimulatory expression and subjected to rapid expansion using a G-Rex-based bioreactor system with irradiated feeder cells, anti-CD3, and low-dose IL-2. Expanded TILs were characterized by multicolor flow cytometry, CD107a degranulation, IFN-γ ELISA, and single-cell RNA sequencing (scRNA-seq), and benchmarked against TILs expanded using the conventional protocol from NCI.



 Our approach reliably produced 15–20 billion viable TILs per patient within 30 days, achieving >95% viability and ∼2500-fold expansion. CD137+ enrichment enhanced tumor reactivity, as shown by increased IFN-γ secretion and degranulation with autologous tumor and high-grade serous ovarian cancer cell lines. scRNA-seq revealed enriched cytotoxic CD8+ T cells and mucosal-associated invariant T (MAIT) cells, with reduced FOXP3+ regulatory and non-cytotoxic CD4+ subsets, compared to TILs from matched blood, tumor, and those expanded with high-dose IL-2 in the NCI protocol. CD8+ proliferative and cytotoxic T cell subsets were enriched under the hybrid CD137-sorted protocol. Transcriptomic profiling showed that CD8+ proliferating T cells from the hybrid group downregulated key metabolic genes, including SLC7A5 and LDHB, suggesting a rewired metabolic state that may support persistence. Similarly, CD8+ cytotoxic T cells showed chromatin remodeling, with upregulation of histone demethylases KDM5A and KDM7A, and downregulation of CHD7 and JARID2, indicating transcriptional plasticity. MAIT cells also showed increased activation markers (IFNG, CCL20). Expanded TILs exhibited reduced expression of exhaustion markers PD-1 and CD38 at the protein level.



 Metabolic reprogramming combined with CD137-based enrichment enables robust expansion of metabolically reprogrammed, tumor-specific TILs from ovarian tumors. This strategy offers a promising foundation for advancing TIL-based ACT in ovarian cancer.



 Paramita Chakraborty, Stephanie Mills, Jadan Zavestoski, Satyajit Das, Suganya Subramanian, Nicolina Mascia, Jian Liu, Stefano Berto, Brian Orr, Shikhar Mehrotra, Stephanie Mills. Metabolic reprogramming of ovarian CD137+ TILs for treatment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b37662ad71037e3dead97ad855c982dd8d046b2" target='_blank'>
              Abstract PR006: Metabolic reprogramming of ovarian CD137+ TILs for treatment
              </a>
            </td>
          <td>
            Paramita Chakraborty, Stephanie Mills, Jadan Zavestoski, Satyajit Das, S. Subramanian, Nicolina Mascia, Jian Liu, S. Berto, Brian Orr, Shikhar Mehrotra
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The swift advancement of single-cell RNA sequencing (scRNA-seq) technology has furnished a crucial instrument for investigating the tumor microenvironment (TME) and its response to immunotherapy. As immunotherapy becomes increasingly prevalent, the challenge of accurately predicting its efficacy has emerged as a prominent focus in contemporary research. In recent years, the utilization of scRNA-seq in the context of immunotherapy has demonstrated promising potential, particularly in the realms of efficacy prediction and biomarker discovery. The heterogeneity of immune cells within the TME exerts intricate and multifaceted influences on treatment response, necessitating comprehensive investigation. Furthermore, the integration of biomaterials into tumor immunotherapy presents novel research opportunities in this domain. scRNA-seq technology offers a systematic approach to evaluating the modifications in the TME induced by biomaterials. This article aims to review the current state of scRNA-seq in the context of immunotherapy, identify existing challenges within related research, and propose future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff1055fe0bb727cc5808bf5c9c95fef6d5984f2e" target='_blank'>
              Single-cell RNA sequencing: enhancing the predictive accuracy of tumor immunotherapy efficacy
              </a>
            </td>
          <td>
            Wei Zhou, Ziwei Huang, Zhiyun Wu, Mengyuan Tang, Linqi Zhu, Weifeng Shi, Qi Wang, Liangzhu Feng
          </td>
          <td>2025-08-22</td>
          <td>Essays in Biochemistry</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM), the most aggressive adult brain cancer, comprises a complex tumour microenvironment (TME) with diverse cellular interactions that drive progression and pathobiology. The aim of this study was to understand how these spatial patterns and interactions evolve with treatment.


METHODS
To explore these relationships, we employed imaging mass cytometry to measure the expression of 34 protein markers, enabling the identification of GBM-specific cell types and their interactions at single-cell protein level in paired primary (pre-treatment) and recurrent (post-treatment) GBM samples from five patients.


RESULTS
We find a significant post-treatment increase in normal brain cells alongside a reduction in vascular cells. Moreover, despite minimal overall change in cellular diversity, interactions among astrocytes, oligodendrocytes, and vascular cells increase post-treatment, suggesting reorganisation of the TME. The GBM TME cells form spatially organized layers driven by hypoxia pre-treatment, but this influence diminishes post-treatment, giving way to less organised layers with organisation driven by reactive astrocytes and lymphocytes.


CONCLUSIONS
These findings provide insight into treatment-induced shifts in GBM's cellular landscape, highlighting aspects of the evolving TME that appear to facilitate recurrence and are, therefore, potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fcab07cdc7f1fdc819d108cb3fdc13fdcffb099" target='_blank'>
              Spatial profiling of longitudinal glioblastoma reveals consistent changes in cellular architecture, post-treatment.
              </a>
            </td>
          <td>
            Shoaib Ajaib, Joshua Winter-Luke, Richard J Digby, Steven Pollock, Gemma Hemmings, A. Gusnanto, Aruna Chakrabarty, Azzam Ismail, Erica Wilson, B. Hunter, Andrew Filby, David McDonald, Asa A. Brockman, Rebecca A. Ihrie, Lucy F Stead
          </td>
          <td>2025-09-01</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Late recurrence of breast cancer poses a considerable threat to patient survival, often attributed to breast cancer dormancy. Dormancy, characterized by cancer cells remaining quiescent for extended periods, is influenced by genetic factors and modifications that directly impact cellular phenotype. Alterations in gene expression dynamically shape cellular behavior, often mediated through receptor signaling pathways. Moreover, interactions within the tumor microenvironment play a pivotal role, fostering either cancer cell dormancy or promoting their escape from dormancy. This review endeavors to provide a comprehensive synthesis of recent advancements in understanding breast cancer dormancy across genetic, receptor molecular, and cellular dimensions. By elucidating the intricate mechanisms underlying dormancy, we aim to shed light on potential therapeutic strategies to prevent late recurrences. Furthermore, we anticipate future research directions that may uncover novel insights into this complex phenomenon, ultimately improving patient outcomes and refining clinical management strategies for breast cancer recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a6a18864245f8cad5e8cc957c1b2db50aef88c" target='_blank'>
              Unraveling the Mystery of Breast Cancer Dormancy: Insights into Genetic, Receptor, and Cellular Interactions Driving Late Recurrence
              </a>
            </td>
          <td>
            Haochen Ma, Bingqiang Zhang, Mengmeng Chen, Zhendi Song, Yi Zhao
          </td>
          <td>2025-09-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background This study aimed to characterize measurable morphological features such as size, shape, presence of protrusions, and micronuclei in circulating tumor cells (CTCs) in prostate cancer (PCa) patients and assess their clinical relevance. Methods Peripheral blood (PB) and tumor-draining vein blood (TDVB) samples were collected from 118 PCa patients at the Medical University of Gdańsk between 2018 and 2023. CTCs were phenotyped using epithelial (pan-keratins, K) and mesenchymal (vimentin, V) markers, with exclusion of non-tumor cells using CD45, CD31, αSMA, and CD29 markers. Imaging flow cytometry and QuPath software were used for the acquisition of CTC images, phenotypic and morphological analysis. Statistical analysis was performed with IBM SPSS Statistics. Representative images of CTCs were shared via CTC Atlas (www.CTCAtlas.org). Results A total of 1,437 CTCs in TDVB and 578 in PB were identified across 4 phenotypes: K + V−, K + V+, K − V+, and K − V−. CTCs were larger than leukocytes, with K + V− and K − V− cells being the largest, and K − V+ the smallest. Epithelial-to-mesenchymal transition (EMT) CTCs were significantly smaller in PB. Cytoplasmic protrusions were detected in a small subpopulation of CTCs. When detected in TDVB-derived CTCs, protrusions were significantly more frequent in epithelial CTCs compared to EMT-CTCs (P < .001). Micronuclei were observed in < 1% of CTCs in TDVB (limited to K + V− and K + V+), increasing to 3% in epithelial CTCs in PB. Small CTCs were associated with the shorter time to biochemical recurrence (P = .041, HR 2.198, CI95% 1.034-4.669). PB-derived CTCs were significantly smaller (P < .0001) and more irregular (P < .0001) in patients with newly diagnosed metastatic PCa (n = 13) than in those with localized disease (n = 556) (median diameter: 9.9 µm, range 7-23 µm vs 16.5 µm, range 7-62 µm; circularity: 0.67 vs 0.76). Smaller CTCs detected in TDVB and/or PB indicated patients’ shorter time-to-biochemical recurrence (P = .041, HR 2.198, CI95% 1.034-4.669). Conclusions CTCs display morphological heterogeneity across phenotypes. This is the first report to quantify EMT-CTC morphology and identify features such as protrusions and micronuclei. Smaller and morphologically irregular CTCs appear to correlate with PCa aggressiveness and poorer clinical outcomes, suggesting that even such simple CTC characteristics might identify CTCs associated with disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee17dbd2d4b8133dbdc0eb919926013f490dd3ab" target='_blank'>
              IUC24390-83 Morphology of circulating tumor cells: how it might relate to biology and clinics in prostate cancer
              </a>
            </td>
          <td>
            R. Wenta, J. Richert, A. Muchlińska, B. Pieczyńska-Uziębło, K. Miszewski, M. Matuszewski, A. Żaczek, N. Bednarz-Knoll
          </td>
          <td>2025-08-29</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) are ectopic lymphoid aggregates often found in chronic inflammatory conditions, including cancer. These structures, which share many cellular and functional features with secondary lymphoid organs, can profoundly influence the tumor microenvironment by promoting local anti-tumor immune activation. TLSs have been observed in various cancers, including melanoma, and are associated with improved responses to immunotherapy and clinical outcomes. However, our understanding of the molecular mechanisms underlying TLS formation and function remains incomplete. This review summarizes the current findings on TLSs in human melanoma, drawing from multiple studies to provide an updated overview. We discuss the cellular composition, spatial distribution, and genetic signatures of TLSs at different stages of melanoma pathogenesis and in subtypes including acral, uveal, and desmoplastic melanoma. Additionally, we examine the influence of tumor mutational burden (TMB) and complement activation on TLS formation, as well as the role of TLSs in immune checkpoint inhibitor therapy. We also highlight the potential of TLSs as indicators for disease progression and treatment response, and review preclinical strategies aimed at inducing TLSs to improve therapeutic outcomes. This synthesis aims to support ongoing research into the role of TLSs in melanoma immunobiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cd5d327c6ce7e444f0890b04b9badc06dcf384c" target='_blank'>
              Tertiary Lymphoid Structures in Human Melanoma: Molecular Mechanisms and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Gelare Ghajar-Rahimi, Ishika Patel, Nabiha Yusuf
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 KRAS^G12D mutations represent a significant oncogenic driver, prevalent in highly aggressive malignancies such as pancreatic ductal adenocarcinoma (PDAC) and colorectal cancer (CRC). The recent development of direct KRAS^G12D inhibitors has ushered in a new era of targeted therapy, offering unprecedented opportunities to address these previously intractable cancers. This study aimed to explore and optimize therapeutic strategies by thoroughly evaluating the efficacy of MRTX1133, a novel selective non-covalent KRAS^G12D inhibitor, in conjunction with various immunotherapeutic modalities, particularly those involving natural killer (NK) cells.



 We utilized patient-derived xenograft and syngeneic murine cancer models to systematically characterize alterations of immune landscapes within tumor microenvironment (TME) upon KRAS^G12D inhibition. Comprehensive and in-depth assessment of immune cell function and single-cell level transcriptional activity was performed using flow cytometry and single-cell RNA sequencing analysis.



 Our findings reveal that MRTX1133 not only suppresses tumor progression but also profoundly enhances the activation of infiltrated natural killer (NK) cells within the tumor microenvironment. Functional depletion of NK cells partially, yet significantly, attenuated the anti-tumor activity of MRTX1133, underscoring the contribution of NK cells to the observed therapeutic benefit. Notably, KRAS^G12D inhibition led to a reduction in tumor-infiltrating Gr-1^+ myeloid-derived suppressor cells (MDSCs) and increased the proportion of effector NK cells. Furthermore, MRTX1133 facilitated greater intratumoral accumulation of adoptively transferred NK cells and counteracted tumor-induced systemic immunosuppression on NK cells. Combined treatment with MRTX1133 and NK cell immunotherapies produced synergistic anti-tumor effects and significantly extended survival in preclinical models. Mechanistically, ex vivo analyses demonstrated that MRTX1133 upregulates NK cell-activating ligands, including MICA/B and ULBPs, and suppresses the production of cytokines such as IL-6 and GM-CSF by cancer cells, thereby enhancing tumor cell susceptibility to NK cell-mediated cytotoxicity and reducing MDSC proliferation.



 Collectively, our results illuminate the interplay between KRAS^G12D inhibition and immuno-microenvironmental remodeling, highlighting rational combinations of targeted and NK cell-based immunotherapies as promising strategies to potentiate therapeutic responses in KRAS^G12D-driven cancers, potentially improving clinical outcomes for patients.



 Hongyuan Chen, Tuo Hu, Chao Xu; Liangjie Chi, Fangqin Xue, Chunbo He. KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aa888d3ef7c73bc5df1193380be3a3238a9fcd" target='_blank'>
              Abstract A016: KRASG12D inhibition enhances tumor cell sensitivity to natural killer cell-mediated cytotoxicity
              </a>
            </td>
          <td>
            Hongyuan Chen, Tuo Hu, Chao Xu, Liangjie Chi, Fangqin Xue, Chunbo He
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive and lethal brain tumor with very poor prognosis despite recent progress in multimodal treatments. Within glioma tissue, various niche cells such as macrophages and neutrophils form a unique glioma immune microenvironment (GIME) by interacting with heterogenous cancer cells, and this has been implicated in disease progression and responsiveness to immunomodulatory therapies. This study explores novel potential prognostic markers associated with the GIME using integrated bioinformatics analyses, including single-cell RNA-sequencing (scRNA-seq), and spatial transcriptome (ST) datasets of clinical GBM specimens. We first identified 42 genes as being associated with poor prognosis in GBM from 5 different cohorts, GBM vs. nontumor tissue, grade IV vs. grade II gliomas, isocitrate dehydrogenase (IDH)-wild-type vs. IDH-mutant variants, mesenchymal vs. proneural and classical subtypes, and hazard ratio for overall survival. Among these, 32 genes were positively correlated with ESTIMATEScore, infiltration of various immune cell types, expression of known immune-related genes, and representative immune-associated biological signals. On scRNA-seq analysis, 7 genes were relatively concentrated in tumor-associated macrophages rather than in malignant cells. ST analysis revealed that Collagen beta(1-O)galactosyltransferase 1 (COLGALT1), Integrin subunit beta 2 (ITGB2), and Myosin light chain 12A (MYL12A) were distributed in the interface between the tumor and the peritumoral area, overlapping with the expression of representative immune-related genes. These findings support the potential of COLGALT1, ITGB2 and MYL12A as biomarkers for predicting the prognosis and immune responses of GBM, which can help in the development of potential immunotherapeutic strategies for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76a83bcafe86bab51686dcebc0d98499aabea491" target='_blank'>
              Potential Prognostic Markers for Glioblastoma Associated with the Glioma Immune Microenvironment.
              </a>
            </td>
          <td>
            Kazuya Tokumura, E. Hinoi
          </td>
          <td>2025-09-01</td>
          <td>Biological & pharmaceutical bulletin</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Breast cancer biology is influenced by both genetic ancestry and the environment with implications for disease outcomes. However, we lack the knowledge how non-cancerous cells within tumors are functionally altered by these factors at single-cell resolution. Here, we created a single-nucleus (sn) dataset from 33 African American, 25 Kenyan, and 24 European American women with breast cancer using combined RNA and ATAC sequencing. We successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue using an optimized combination of enzymatic digestion and automated tissue homogenization. SnMultiome sequencing of 82 tumors was performed using the 10x Genomics platform. Following filtering, normalization, peak calling, and integration, our dataset includes a total of 292,458 nuclei. Cancerous (163,419 nuclei) and non-cancerous (129,039 nuclei) cells were distinguished based on DNA copy number. Within the microenvironment, 11 major immune, epithelial, and stromal cell types were identified. CD45+, EPCAM- cells were further analytically extracted, clustered, and manually annotated based on known marker genes, detecting 26 distinct immune subpopulations across our samples. After adjustment for potential confounders (age, body mass index, and tumor molecular subtype) in all analyses, we found that women of African descent possessed distinct, activated mesenchymal and endothelial cell subpopulations within the tumor vasculature that promote inflammation, angiogenesis, and cancer cell invasion. Women of African descent exhibited a more immune-suppressive tumor microenvironment, with elevated expression of immune checkpoint markers (PD-1, TIGIT, CTLA-4, and LAG-3), functional impairment of dendritic cells, and enrichment of regulatory and tolerogenic lymphocytic subpopulations. Further, malignant cells from African descent patients expressed a discrete set of CAR-T targets with potential implications for precision immunotherapy. Lastly, we developed three gene signatures within the microenvironment that are predictive of survival with external validation in both tumor-adjacent normal and cancerous tissues from the TCGA. Together, we uncovered molecular breast tumor characteristics of clinical significance in women of African ancestry.



 Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H. Dorsey, Eytan Ruppin, Shahin Sayed, Francis Makokha, Gretchen L. Gierach, Stefan Ambs. Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b4c16fead2784b8ef235041a0b9c641f582e01" target='_blank'>
              Abstract C109: Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H Dorsey, E. Ruppin, Shahin Sayed, Francis W. Makokha, Gretchen L. Gierach, S. Ambs
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="The awakening of dormant disseminated cancer cells appears to be responsible for the clinical relapses of patients whose primary tumors have been successfully cured months and even years earlier. In the present study, we demonstrate that dormant breast cancer cells lodged in the lungs reside in a highly mesenchymal, nonproliferative phenotypic state. The awakening of these cells is not triggered by a cancer cell-autonomous process. Instead, lung inflammation induced by the chemotherapeutic agent bleomycin effectively awakens dormant cancer cells, providing useful models for studying metastatic awakening. Mechanistically, the awakened cells shift from a highly mesenchymal to a quasi-mesenchymal phenotypic state in which they acquire tumorigenicity and proliferative ability. Once awakened, these cells can stably reside in this quasi-mesenchymal state and maintain their tumor-initiating ability, doing so without ongoing heterotypic signaling from the lung microenvironment. Epidermal growth factor receptor ligands released by the cells of the injured tissue microenvironment, including notably M2 type macrophages, promote dormant cancer cells to move toward this quasi-mesenchymal state, a transition that is critical for the awakening process. An understanding of the mechanisms of metastatic awakening may lead in the future to treatment strategies designed to prevent such awakening and resulting metastatic relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0978c66611eb319088a607dd5da99f325c7f34ff" target='_blank'>
              Inflammation awakens dormant cancer cells by modulating the epithelial-mesenchymal phenotypic state.
              </a>
            </td>
          <td>
            Jingwei Zhang, Jingwen Zhang, Longfei Han, Shiyi Wu, Jie Li, E. Eaton, Bingbing Yuan, Ferenc Reinhardt, Hao Li, Patrick C Strasser, Sunny Das, Joana Liu Donaher, Md Imtiaz Khalil, Haiping Jiang, Alexander Deuschel, Danni Lin, Carolin Sebastiany, Mariana Maranga, Salomé Shubitidze, Xiaofei Liu, Arthur W Lambert, Yun Zhang, Yana Liu, L. Sui, Sarah Elmiligy, Umberto Pozza, Rauf Günsay, Ranjan Mishra, José Velarde, Sonia Iyer, Whitney S. Henry, K. Weiskopf, Guihai Feng, Tobiloba E. Oni, R. Watnick, Xin Li, Robert A. Weinberg
          </td>
          <td>2025-09-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4146020be8bb663490d3534c28c485f753647c10" target='_blank'>
              Machine learning integration of bulk and single-cell RNA-seq data reveals glycolytic heterogeneity in colorectal cancer.
              </a>
            </td>
          <td>
            Yuanyuan Du, Zefeng Miao, Peng Li, Dan Feng, Mulin Liu, Aifang Ji, Shijun Li
          </td>
          <td>2025-08-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell molecular profiling has emerged as a cutting-edge technology in biological research, playing a pivotal role in elucidating fundamental life processes and cellular heterogeneity. However, traditional methods generally provide end point measurements and require cell disruption, posing challenges for long-term monitoring of living cell events, e.g., cancer therapy. Tumor Treating Fields (TTFields), a noninvasive therapeutic modality, exert inhibitory effects on tumor growth and invasion by applying low-intensity intermediate-frequency alternating electric fields to tumor regions, whose molecular mechanism on cancer physiology remains poorly understood. In this study, we propose a multifunctional microchip capable of continuous in situ monitoring of molecular signatures in living single cells. The platform integrates spatial cell positioning, single-cellular multimolecule tracking with surface-enhanced Raman scattering (SERS), and TTFields treatment functionalities on a microchip. Using multiplexed analysis of liver cancer cells treated with TTFields and chemotherapy drugs, we implement spatial-temporal multiomics profiling for cancer therapy. Temporal tracking of critical molecular events associated with glycolysis, genetic stability, and membrane integrity reveals significant differences between the groups with and without TTFields treatment. Causal network analysis of molecular profiles uncovers the underlying mechanisms through which TTFields modulate oncogenic pathways. Our work demonstrates the effectiveness of the proposed microchip platform for real-time monitoring of living cell molecular dynamics, offering a powerful tool for precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889ada64ff2d00892ebd5cde7eb385bb8b31694b" target='_blank'>
              Spatial-Temporal Multiomics Profiling of Living Single Cells on Chip for Cancer Therapy.
              </a>
            </td>
          <td>
            Kunru Yu, Sheng-Zong Chen, Rong Zhu
          </td>
          <td>2025-10-12</td>
          <td>ACS sensors</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Analyses of patient-derived cell lines have greatly enhanced discovery of molecular biomarkers and therapeutic targets. However, characterization of cellular morphological properties is limited. We studied cell morphologies of human pancreatic adenocarcinoma (PDAC) cell lines and their associations with drug sensitivity, gene expression, and functional properties. By integrating live cell and spatial messenger RNA imaging, we identified KRAS inhibitor–induced morphological changes specific for drug-resistant cells that correlated with gene expression changes. We then categorized a large panel of patient-derived PDAC cell lines into morphological and organizational subtypes and found differences in gene expression, therapeutic targeting potential, and metastatic proclivity. Patterns of cancer cell organization in human PDAC tissues stratified distinct gene expression signatures with clinical significance. In summary, we highlight the potential of cell morphological information in rapid, cost-effective assays to aid precision oncology efforts leveraging patient-derived in vitro models and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb047a3ab41c4e28c6535b371221906a6c927ab" target='_blank'>
              Integrated spatial morpho-transcriptomics predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, Rachel Liu, Yi Cui, Michael Rhodes, Jung Woo Bae, Joseph M. Beechem, W. L. Hwang
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Renal cancer, particularly clear cell renal cell carcinoma (ccRCC), is characterized by significant intratumoral heterogeneity, which poses challenges for diagnosis and treatment. Single-cell sequencing (SCS) provides unprecedented insights into the cellular landscape of renal cancer, allowing for detailed characterization of tumor heterogeneity at the single-cell level. This review highlights how SCS has been instrumental in elucidating the origins of different renal cancer subtypes, understanding mechanisms of tumor initiation and progression, and dissecting the complex tumor microenvironment (TME). It discusses the identification of novel biomarkers and therapeutic targets, as well as the potential of SCS to inform personalized treatment strategies. The review also explores the integration of SCS with spatial omics technologies, which enhances the understanding of cellular interactions within their spatial context. Moreover, it addresses the challenges and future directions in applying SCS to clinical practice, emphasizing its significance in advancing renal cancer biology and improving clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03a5f0a7b414a81333643cfcf0b78e2ab4708d98" target='_blank'>
              Single-cell sequencing technology in renal cancer: insights into tumor biology and clinical application
              </a>
            </td>
          <td>
            Hanzhong Zhang, Ying Liu, Wenqiang Liu, Anqi Lin, Yu Fang, Le Qu, Xu Zhang, Peng Luo, Linhui Wang, Aimin Jiang
          </td>
          <td>2025-10-01</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains largely refractory to therapy, due in part to the complex interplay between tumor cells and their microenvironment. Human antigen R (HuR/ELAVL1), a ubiquitously expressed RNA-binding protein, is emerging as an important regulator both of tumor-intrinsic and tumor-extrinsic pathways that govern PDAC progression. While the role of HuR in promoting cancer cell survival under stress is well established, recent studies reveal its broader role in shaping the tumor microenvironment (TME), including metabolic rewiring, stromal activation, angiogenesis, and immune modulation. In this review, we examine how tumor-intrinsic HuR drives epithelial-mesenchymal transition, stabilizes key transcripts involved in metabolic adaptation, and alters the tumor secretome to influence extracellular matrix deposition and fibroblast behavior. We further explore the role of HuR in regulating immune cell function and the immune landscape of PDAC. Notably, HuR-driven TME remodeling reinforces environmental stressors that further activate HuR, establishing a feed-forward loop that drives disease progression. These findings underscore HuR as a central regulator of the PDAC TME and therapeutic resistance, and thus, highlight its potential as a target in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be1b261ccac3070698258b8350c58e4ccb52e4c4" target='_blank'>
              The Role of Tumor-Intrinsic HuR in Modeling the Pancreatic Tumor Microenvironment: Molecular Mechanisms and Therapeutic Opportunities.
              </a>
            </td>
          <td>
            Katherine R. Pelz, Kyle P Gribbin, Tess Sevetson, Jonathan R. Brody
          </td>
          <td>2025-10-13</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract In its first decade, The Journal of Pathology: Clinical Research has become a leading source of translational studies advancing molecular diagnostics in cancer, particularly in urologic pathology. This commentary highlights recent contributions that collectively place precision oncology at the forefront of pathology research. One review examines cancer stem cells in renal cell carcinoma, emphasizing the complexity of cellular plasticity and the tumor microenvironment in driving resistance and recurrence. In prostate cancer, epithelial‐to‐mesenchymal transition (EMT) regulators, including Twist, Slug, and Snail, are identified as synergistic markers of poor prognosis, linked to hypoxia and invasiveness. Another review details the integration of homologous recombination repair gene testing into clinical workflows, supporting targeted treatment strategies with poly (ADP‐ribose) polymerase inhibitors. In pediatric oncology, TP53 alterations in Wilms tumor are shown to occur beyond anaplastic cases, expanding their prognostic significance. Advances in molecular subtyping are also demonstrated in bladder cancer, where transcriptomic profiling could enable tailored neoadjuvant therapy. In clear cell renal cell carcinoma, re‐evaluation of a prognostic model revealed that while necrosis or sarcomatoid differentiation correlated with poor outcomes, only DNA methylation markers improved prognostic accuracy, underscoring their utility for biopsy‐based risk stratification. Finally, digital spatial profiling of sarcomatoid urothelial carcinoma reveals an immunosuppressive microenvironment with CD163‐positive cells, implicating them in EMT and aggressive phenotype. Together, these studies highlight the transformative role of integrated molecular diagnostics in guiding individualized therapies and improving outcomes in urologic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9269338c6f4616fe82344276fa3a9645f1fc73c2" target='_blank'>
              Translational milestones in urologic pathology: integrating molecular diagnostics across cancer types
              </a>
            </td>
          <td>
            Andres Matoso, A. Acosta
          </td>
          <td>2025-09-01</td>
          <td>The Journal of Pathology: Clinical Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer remains the most prevalent malignancy among women, necessitating the development of novel therapeutic strategies. Experimental animal models that closely mimic human breast cancer are crucial for advancing these therapies. This study utilized the criteria of the tumour, node, metastasis (TNM) staging system and variations in metabolic rates to develop models representing stages II and IV of human breast cancer, using the MBL-6 mouse breast cancer cell line. We assessed tumor growth curves in vivo and investigated distant metastasis to organs such as the liver, lungs, lymph nodes, and spleen. Carcinoma-associated fibroblasts (CAFs) were isolated, and their proliferation rates, inflammatory enzyme expression, and matrix metalloproteinase levels were compared between stages II and IV. By analyzing tumor kinetics and metabolic differences, we were able to predict tumor size and progression at each stage. Our results revealed that CAFs isolated from both stages exhibited similar phenotypic characteristics. However, CAFs from stage II tumors showed higher expression of indoleamine 2,3-dioxygenase 1 (IDO1), while those from stage IV tumors had higher levels of inducible nitric oxide synthase (iNOS). These distinct expression patterns suggest unique microenvironmental features at different stages of tumor progression. Further investigation of the cancer microenvironment may provide valuable insights for selecting targeted therapies and improving disease management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d06703a306e6912c38523fa9c498e367782abad" target='_blank'>
              Stage-specific tumor microenvironment dynamics and cancer-associated fibroblast profiling in MBL-6 mouse models of breast cancer
              </a>
            </td>
          <td>
            L. Langroudi, Maryam Iranpour, Mojtaba Mollaei, Masoud Soleimani, Seyed Mahmoud Hashemi, Zuhair M. Hasan
          </td>
          <td>2025-10-10</td>
          <td>Veterinary Science Development</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding genetic dependencies in cancer is key to identifying novel actionable drug targets to advance precision medicine. Whole-genome CRISPR-knockout library screening methods have facilitated this goal. Pooled libraries of single guide RNAs (sgRNAs) targeting over 90% of the annotated protein coding genome are used to induce gene knockouts in pre-clinical cancer models. Novel genes of interest are identified by evaluating sgRNA dropout or enrichment following selection pressure application. This method is particularly beneficial for researching cancers where effective treatment strategies are limited. One example of a commonly chemoresistant cancer, particularly at relapse, is the low survival malignancy epithelial ovarian cancer (EOC), made up of multiple histotypes with distinct molecular profiles. CRISPR-knockout library screens in pre-clinical EOC models have demonstrated the ability to predict biomarkers of treatment response, identify targets synergistic with standard-of-care chemotherapy, and determine novel actionable targets which are synthetic lethal with cancer-associated mutations. Robust experimental design of CRISPR-knockout library screens, including the selection of strong pre-clinical cell line models, allows for meaningful conclusions to be made. We discuss essential design criteria for the use of CRISPR-knockout library screens to discover genetic dependencies in cancer and draw attention to discoveries with translational potential for EOC. Supplementary Information The online version contains supplementary material available at 10.1007/s13402-025-01102-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c4704796a9cf8557ad095d79824927f1f6b68e0" target='_blank'>
              Revealing genetic drivers of ovarian cancer and chemoresistance: insights from whole-genome CRISPR-knockout library screens
              </a>
            </td>
          <td>
            Tali S. Skipper, Kristie-Ann Dickson, Christopher E. Denes, Matthew A. Waller, Tian Y. Du, G. Neely, N. Bowden, Alen Faiz, Deborah J. Marsh
          </td>
          <td>2025-08-28</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Approximately 40% of patients with pancreatic ductal adenocarcinoma (PDAC) receive no benefit from surgical resection and systemic chemotherapy and experience metastatic disease recurrence within 12 months. The extreme heterogeneity of the cells that populate PDAC tumors has been a significant impediment to identifying the cells that likely seed early recurrence. Previous studies show that metastatic spread is driven by specific cells with hybrid epithelial-mesenchymal characteristics, but despite the identification of numerous molecular features associated with epithelial, hybrid, and mesenchymal PDAC cancer cells, it has not been possible to specifically identify the cells that seed metastatic recurrence in human tissue. We have previously shown that PDAC cell glycan signatures are altered as a direct result of the metabolic and transcription changes that occur as PDAC cancer cells undergo EMT. Now we integrated the glycan signatures with known protein markers and morphological analyses using our pipeline for multiplexed immunofluorescence analysis to identify distinct subpopulations of epithelial, hybrid, and mesenchymal PDAC cells. In cell-level analyses of immunofluorescence images of 24 PDAC cell lines, these subpopulations revealed epithelial to hybrid to mesenchymal conversions along 3 distinct trajectories, which were defined by transcription factors and proteins previously associated with epithelial or mesenchymal states—TP53, TP63, GATA6, and MYC—and further distinguished by specific glycan signatures and protein markers of metabolism and cellular differentiation. The integrated transcription factor, glycan, and protein signatures identified the previously described classical and squamous PDAC subpopulations and defined a new signature for a fully mesenchymal subpopulation. The signatures differentiated between PDAC subpopulations in primary human tumors and in metastatic liver lesions and showed a patient-matched correspondence between the primary tumors and metastatic lesions in specific hybrid PDAC cells within distinct trajectories, potentially revealing the PDAC cells seeding metastatic recurrence. These results suggest that the development of mesenchymal and metastatic PDAC cells occurs through distinct, stepwise trajectories with divergent morphological, metabolic, molecular, and glycan-based signatures. These findings will/have the potential to inform the development of treatments that specifically target the PDAC cells that seed early metastatic recurrence.



 ChongFeng Gao, Zachary Klamer, Ali Moursy, Galen Hostetter, Paul M. Grandgenett, Peter Allen, Brian Haab. Resolving PDAC cell subpopulations that seed metastatic recurrence [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B125.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246925305285b4b397bb54f4914db32c1466cbe8" target='_blank'>
              Abstract B125: Resolving PDAC cell subpopulations that seed metastatic recurrence
              </a>
            </td>
          <td>
            Chongfeng Gao, Zachary L. Klamer, Ali Moursy, G. Hostetter, P. Grandgenett, Peter Allen, Brian Haab
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="BACKGROUND
Changes in the cyst microenvironment in Polycystic Kidney Disease (PKD) may drive progressive cyst formation. Bulk- and single-cell RNA sequencing have advanced our understanding of altered signaling; however, the lack of spatial information has limited our insights into local gene expression and cellular communication near cysts.


METHODS
We used wild-type and Pkd1-deficient mouse kidneys to generate 10x Genomics Visium Spatial Gene Expression datasets. Utilizing our single-cell mouse kidney atlas and single cell sequencing data for spot deconvolution, we enhanced resolution and estimated enriched cell types. We analyzed spatial gene expression patterns and used a cyst-centered analysis to identify cyst-associated gene signature. Cell communication near cysts was investigated, identifying key ligand-receptors. Prioritized key factors were validated in tissues.


RESULTS
We observed enrichment of fibroblasts, injury-repair-related cell types, and diverse immune populations in PKD. Injury-repair-related cells were exclusively observed in PKD, predominantly localized within immune cell-dense regions near cysts. These cells collectively contributed to the altered gene expression profile in PKD, including cyst-associated signature genes related to inflammatory processes. Analysis of cellular communication in less-inflamed regions around cysts revealed the involvement of multiple cell types. Key ligand-receptor interactions were associated with cytokine signaling, fibrosis, cellular development, and repair. These included Angpt2, C3, Csf1, Cxcl12, Il34, Gas6, Il16, Mdk, Mif, Ptn, Sfrp2, Spp1, Sdc1, Tnc, Tnfsf12, Wnt5a. In addition, ECM proteins implicated in immune response, ECM remodeling, cell adhesion, and cell signaling were identified, such as Adam9, Adam10, Col1a1, Col3a1, Col4a2, Lamb2, Lamc1, Efnb1, Efnb2, Thbs1, Thbs2, and Vcam1. IHC confirmed expression of Syndecan-1-Collagen IV, Midkine-Integrin β1, CSF-1, Pleiotrophin, and Tenascin-C in cystic kidneys.


CONCLUSIONS
Spatial transcriptomics in PKD revealed enrichment of (myo)fibroblasts, immune, and injury-repair-related cells near cysts, creating a (pro)inflammatory and (pro)fibrotic niche. Key ligand-receptor and ECM interactions were identified and validated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecdcf5e98138976e4b0b6f2eafc40e1b84148be0" target='_blank'>
              Spatial Transcriptomics Reveals Injured Cells, Signature Genes, and Communication Patterns in the Cyst Microenvironment of Polycystic Kidney Disease.
              </a>
            </td>
          <td>
            Sevtap A Yasinoglu, Claudio Novella-Rausell, L. Wisse, K. Dijkstra, A. Mahfouz, Hans J. Baelde, DJM. Peters
          </td>
          <td>2025-10-10</td>
          <td>Journal of the American Society of Nephrology : JASN</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The thymidine analog EdU (5-ethynyl-2-deoxyuridine) is incorporated into DNA during replication and has traditionally been used as a marker of S-phase cells. In this study, we discovered that EdU fluorescence images display substantial cell-to-cell variability, which could be classified into multiple clusters by unsupervised machine learning. This suggests that seemingly random EdU patterns contain reproducible, computationally recognizable features. Building on our observation that distinct patterns emerged in response to radiation stress, we investigated whether radioresistant cancer cells exhibit specific EdU signatures. Analysis of PLK1-overexpressing cells, which acquire radioresistance through altered DNA replication, revealed radiation-induced EdU patterns distinct from control cells. Prompted by the observation that these cells displayed markedly enlarged and intensified γ-H2AX foci, a marker of DNA damage, we employed a supervised machine learning model based on γ-H2AX patterns to isolate the radioresistant cell subpopulation. We then extracted the EdU signals from these isolated cells and, through further unsupervised machine learning, successfully identified a characteristic pattern specific to radioresistance. This establishes a machine learning framework capable of extracting universal rules from the dynamic networks that vary among individual cells, which provides a novel platform for a screening system to identify molecules involved in radioresistance, focusing on cancer heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0447884e0a398eddbbba5abf08e4dfe806602b07" target='_blank'>
              Machine Learning Extracts Radiation Resistant-Specific EdU Fluorescence Pattern in Cancer Cells.
              </a>
            </td>
          <td>
            M. Ikura, Tsuyoshi Ikura, Kanji Furuya
          </td>
          <td>2025-09-01</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung cancer in smokers (LCIS) and lung cancer in never-smokers (LCINS) are different entities with distinct molecular features. However, their cellular heterogeneity still requires further investigation. Through an integrated analysis of single-cell RNA sequencing (scRNA-seq) and bulk sequencing data, we identified cell subpopulations associated with smoking and non-smoking patients. Downstream transcriptomic analyses were then performed to reveal differences in cell function and tumor microenvironment. We observed that smoking-associated cancer cells exhibited a higher degree of aggressiveness, which may correlate with an adverse prognosis in smoking patients. Additionally, immunosuppressive CXCL10+ macrophages may be involved in their tumorigenesis in smokers, and the immunoregulatory LGALS9-HAVCR2 axis could be a potential immunotherapeutic target. In non-smokers, the inflammatory microenvironment may be involved in their tumor formation. Moreover, the decreased anti-tumor cytotoxicity could be associated with their suboptimal immunotherapeutic response. Our study uncovered differences in oncogenic and immune escape mechanisms between LCIS and LCINS patients and suggests potential immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6369e8d4f2741d426c67fe24a46863f596640138" target='_blank'>
              Integrating bulk and single-cell sequencing reveals cellular heterogeneity between lung adenocarcinoma in smokers and never-smokers.
              </a>
            </td>
          <td>
            Zihuan Zhao, Pan Yang, Yuhan Liu, Kai Wang, Xianfeng Xu, Yuzhuo Wang, Meng Zhu, N. Qin, Cheng Wang, Weimin Li, Hongxia Ma, Zhoufeng Wang, Hongbing Shen
          </td>
          <td>2025-08-25</td>
          <td>Journal of biomedical research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLSs) in the tumor microenvironment (TME) act as local hubs for antigen presentation and lymphocyte activation, reinforcing antitumor immunity. They act as ectopic, lymph node-like niches that orchestrate local antitumor immunity through well-defined B-cell follicles, T-cell zones, follicular dendritic cells (FDCs), and high endothelial venules (HEVs). Although various cancer treatment methods can impact TLSs, thereby changing the state of the TME, the dynamic changes in TLSs caused by treatment and the mechanisms therein have not been comprehensively summarized to date. This review highlights the mechanisms of different cancer treatment effects on TLSs dynamic change, including the number and functionality of TLSs within tumors. We also summarize the value of TLSs in predicting prognosis and therapeutic effect, as well as the deficiencies in current research and future development directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b10ea5ef7732254210365008058829408071ec8" target='_blank'>
              Dynamic remodeling of tertiary lymphoid structures in response to cancer therapy: a recent review
              </a>
            </td>
          <td>
            Chongyu Tan, Jinliang Huang, Ning Gao, Bingquan Wu, Matsika Juliet, Jiatong Xiao, Jiao Hu, Ping Liu, Jinbo Chen
          </td>
          <td>2025-09-29</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0643ed748b04336ec828931abc9d5fbd2911609" target='_blank'>
              [Progress of scRNA-seq technology in nasopharyngeal carcinoma research].
              </a>
            </td>
          <td>
            Bin Zheng, Guanqiao Jin
          </td>
          <td>2025-09-01</td>
          <td>Lin chuang er bi yan hou tou jing wai ke za zhi = Journal of clinical otorhinolaryngology head and neck surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive malignancy driven by oncogenic KRAS and inflammation-induced cellular heterogeneity, yet the mechanisms underlying tumor-initiating cells (TICs) emergence and maintenance remain unclear. Our study establishes LY6D as a marker of persistent TICs that orchestrate pancreatic cancer progression across all disease stages. Single-cell RNA sequencing of inflammation-driven PDAC models reveals that LY6D+ TICs specifically originate from KRAS-mutant acinar-to-ductal metaplasia (ADM) lesions under inflammatory conditions, maintaining conserved stemness properties and exhibiting a distinctive oxidative phosphorylation (OXPHOS) dependency throughout tumor evolution. Moreover, genetic ablation of Ly6d in Kras G12D pancreata delays tumorigenesis, while forced Ly6d expression enhances tumorigenic potential and metastatic capability. Mechanistically, LY6D-despite lacking intracellular domains-scaffolds lipid raft-associated kinase networks and FOSL1-dependent epigenetic reprogramming to establish a ​stable pro-tumorigenic state. Clinically, LY6D+ cells are enriched in human PDAC and exhibit conserved stemness and epithelial-mesenchymal transition (EMT) properties. Strikingly, LY6D expression levels demonstrate PDAC-restricted prognostic power. Our work defines LY6D as pan-stage TICs marker linking cellular plasticity to PDAC initiation and progression, offering new avenues for early detection and interception of this lethal malignancy.



 Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue. LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B119.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f74adf609c9f9d814e1f1f768c348943a8c10b" target='_blank'>
              Abstract B119: LY6D identifies persistent tumor-initiating cells driving pancreatic tumorigenesis
              </a>
            </td>
          <td>
            Juanjuan Shi, Yingying Tang, Zhengyan Zhang, Xian Wang, Shixin Meng, Ping Lu, Junyi Xu, Zheng Wang, Yongwei Sun, Jing Xue
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor matrix stiffness plays a critical role in cancer progression, metastasis, and therapy resistance. Although traditional biophysical methods have shed light on the impact of matrix stiffness on tumor behavior, these techniques are confined to measuring the physical properties of the tumors. In this study, we leveraged RNA-seq data to predict tumor matrix stiffness, aiming to reveal mechanical properties by molecular signatures across various cancer types. To this end, we systematically analyzed RNA-seq data from tumors of varying stiffness levels to identify stiffness-associated gene signatures. With these molecular signatures, we developed a computational model for predicting tumor matrix stiffness and further applied it to The Cancer Genome Atlas (TCGA) dataset. Our analysis revealed significant differences in the tumor microenvironment as well as immune response between soft and stiff tumor samples, suggesting that tumor rigidity impacts not only cellular behavior but also characteristics of the tumor microenvironment. These findings underscore the potential of RNA-based stiffness models to enhance our comprehension of tumor mechanics and cancer biology, thereby facilitating the development of innovative targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd0a56130a8bfa1d6781b487c86488f3edefb603" target='_blank'>
              Pan-cancer analysis reveals molecular signatures for predicting matrix stiffness in solid tumors.
              </a>
            </td>
          <td>
            Gongyu Tang, Xinyi Liu, Yuanxiang Li, Yunfei Ta, Minsu Cho, Hua Li, Xiaowei Wang
          </td>
          <td>2025-09-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most frequent brain tumors in children and comprise a heterogeneous group of tumors with different locations, histologic subtypes, ages at presentation, and clinical behavior. Tumors frequently respond to treatment with chemotherapy or radiation therapy, but they can regrow after a period of quiescence, requiring further therapy. Thus, a deeper understanding of the molecular processes involved in these tumors is required to develop therapeutic strategies that are effective against their disease mechanisms. To better understand the cellular behaviors of this heterogenous group of tumors, we have employed single-cell and single-nuclei RNA sequencing technologies to analyze a large-scale dataset (>250,000 cells) of pLGGs. Analysis of this data identified a heterogenous population of cell types and cell states, detecting mature and progenitor-like astrocytes and oligodendrocytes, as well as cells exhibiting senescence or cycling programs. Moreover, we identify a significant immune infiltrate, comprised primarily of microglia. In addition to heterogeneity within pLGG tumors, heterogeneity between LGG subtypes represents another layer that stratifies pLGG biology. We performed a compositional analysis of the cell types present in these tumors and compared transcription signatures and gene expression programs across shared cellular populations of histologically and genetically distinct pLGGs. Finally, we optimized our integration and batch correction analyses by using external 293T cells as spike in controls during our single-cell and single-nuclei data generation steps to determine the most suitable method for batch-effect removal. Our analysis of human pLGGs at the single-cell and single-nuclei resolution provides critical insight into the heterogenous biological activities that constitute these tumors.



 Michelle Boisvert, Ashwyn Perera, Alexandra L. Condurat, John Jeang, Jessica W. Tsai, Dana Novikov, Kevin Zhou, Madison Chacon, Jeromy DiGiacomo, Rushil Kumbhani, Dayle Wang, Michael D. Taylor, Jordan R. Hansford, Louise Ludlow, Nada Jabado, Keith L. Ligon, Rameen Beroukhim, Pratiti Bandopadhayay, David T.W. Jones. Understanding the transcriptional heterogeneity of pediatric low-grade gliomas and its implication for tumor pathophysiology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87fff9552d8445c379c8cd3282dd22472d8f4d25" target='_blank'>
              Abstract A014: Understanding the transcriptional heterogeneity of pediatric low-grade gliomas and its implication for tumor pathophysiology
              </a>
            </td>
          <td>
            Michelle Boisvert, Ashwyn A. Perera, Alexandra L. Condurat, John Jeang, Jessica W Tsai, Dana Novikov, Kevin N Zhou, Madison S. Chacon, Jeromy J. Digiacomo, Rushil Kumbhani, Dayle K. Wang, Michael D. Taylor, J. Hansford, Louise E. Ludlow, Nada Jabado, Keith L. Ligon, R. Beroukhim, P. Bandopadhayay, David Jones
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>151</td>
        </tr>

        <tr id="Myocardial infarction (MI) often leads to ischemic cardiomyopathy, which is characterized by extensive cardiac remodeling and pathological fibrosis accompanied by inflammatory cell accumulation. Although inflammatory responses elicited by cardiac macrophages are instrumental in post-MI cardiac remodeling, macrophage microniche-mediated fibroblast activation in MI are not understood. Analyses of the spatial transcriptomics data of the hearts of patients with ischemic cardiomyopathy and a history of MI using a novel workflow combining Significant Latent Factor Interaction Discovery (SLIDE), which is an interpretable machine learning approach recently developed by us, regulatory network inference, and in-silico perturbations unveiled unique context-specific cellular programs and corresponding transcription factors driving these programs (that would have been missed by traditional analyses) in macrophages, and resting and activated cardiac fibroblasts. More nuanced analyses to examine the microniches comprising these cells in failed hearts uncovered additional cellular programs reflective of altered paracrine signaling among these cells. Silencing of niche-specific key genes and TFs from these cellular programs in both mouse and human macrophages altered the expression of pro-fibrotic genes. Furthermore, the secretomes from these macrophages suppressed myofibroblast differentiation. Finally, macrophage-specific in vivo silencing of Tvp23b, Tdrd6, and B3galnt1 and the transcription factors Mafk and Maz, which are differentially expressed in macrophage/activated fibroblast niches, using a novel lipidoid nanoparticle approach in mice with MI significantly improved cardiac function and suppressed fibrosis. Our study uncovers novel macrophage niche-mediated fibroblast activation mechanisms and provides a new generalizable framework, coupling interpretable machine learning, regulatory network inference, in-silico perturbations, and in vitro and in vivo testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c77d3462554e8ecfcd1884a030957783462e632" target='_blank'>
              Interpretable machine learning coupled to spatial transcriptomics unveils mechanisms of macrophage-driven fibroblast activation in ischemic cardiomyopathy
              </a>
            </td>
          <td>
            Niranjana Natarajan, Hanxi Xiao, Shagufta Haque, Mary D Cundiff, Mika Hara, Varsha Sriram, Jishnu Das, Partha Dutta
          </td>
          <td>2025-08-24</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a7545be94c7b15689eb454a33645943682e8cb" target='_blank'>
              Dissecting the heterogeneity and tumor-associated dynamics of human liver group I ILC via scRNA sequencing data.
              </a>
            </td>
          <td>
            Yumo Zhang, Jitian He, Xue Li, Zhibo Han, Huaiyong Chen, Zhouxin Yang, Youwei Wang
          </td>
          <td>2025-08-23</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4ae2d5f1f6f184ad5a2b71dcc7e35b4feeb422f" target='_blank'>
              Functional biomaterials and machine learning approaches for phenotyping heterogeneous tumor cells and extracellular vesicles.
              </a>
            </td>
          <td>
            Rutwik Joshi, R. Ahmad, Karl Gardner, Hesaneh Ahmadi, Chau-Chyun Chen, Shannon L Stott, Wei Li
          </td>
          <td>2025-09-18</td>
          <td>Biomaterials science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL)-therapy has received FDA approval for the treatment of advanced melanoma and shows potential for broader applications in solid tumors, including glioblastoma. In this study, tumor-reactive TILs (tr-TILs) are isolated and enriched for CD137 expression from cavitron ultrasonic aspirator (CUSA) emulsions of 161 adult patients diagnosed with diffuse gliomas. Tr-TILs are successfully expanded in 87 out of the 161 patients, reflecting an expansion rate of 54%. Notably, the presence of IDH1 mutation and the cumulative dose of steroids are identified as significant negative predictors of expansion efficacy. The expanded tr-TILs exhibit distinct phenotypic and molecular dysfunctional features yet show upregulated expression of progenitor/memory-like markers and polyclonal T-cell receptors. Importantly, these tr-TILs demonstrate specific antitumor reactivity against autologous tumor cells in both in vitro and in vivo xenograft models. These findings provide a compelling background for a personalized immunotherapeutic approach while tackling one of the most significant challenges in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fd8a91ad8ccb8e7a67febcd47df6b08cd98dc4" target='_blank'>
              Polyclonal expansion of functional tumor-reactive lymphocytes infiltrating glioblastoma for personalized cell therapy
              </a>
            </td>
          <td>
            M. Maffezzini, S. Musio, N. Di Ianni, Agnese Rumolo, M. Patanè, Andrea Galluzzo, Irene Sambruni, Arianna Berlendis, Domenico Aquino, Giacomo Baso, Manuela Zingarelli, M. Facciolla, L. Maddaloni, Rossella Valentino, R. Paterra, F. Agistri, M. Farinotti, Luca Mattei, Paola Coluccia, Francesco Acerbi, Francesco DiMeco, Bianca Pollo, Antonio Silvani, M. Eoli, Catia Traversari, Daniela Montagna, S. Pellegatta
          </td>
          <td>2025-08-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="This editorial discusses Alpsoy et al’s significant study of prognosis of pancreatic ductal adenocarcinoma (PDAC), which lacks histopathological markers. This study evaluated the synergistic prognolymphocytes. Peritumoral budding is significantly correlated with tumor volume, while intratumoral budding is closely related to lymph node metastasis. Peritumoral budding and intratumoral budding are confirmed as independent adverse prognostic factors, and their high levels of expression are associated with immature stromal phenotypes, suggesting the key role of epithelial-mesenchymal transition. These breakthrough findings provide a new multidimensional biomarker system for the prognostic assessment of PDAC, and promote the clinical transformation process of incorporating tumor budding indicators into the pathological reporting process. However, the complexity and spatiotemporal heterogeneity of the tumor microenvironment require us to go beyond traditional morphological analysis and move towards multiomics integration and dynamic monitoring. Through standardized pathological assessment, innovative treatment strategies and interdisciplinary collaboration, it is expected to transform tumor microenvironment-related markers into clinically applicable indicators, ultimately improving the treatment predicament of PDAC. This editorial intended to summarize relevant studies and share some of our views, in order to offer perspectives for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25a304affe959bbde58f99c98545a1534cc99601" target='_blank'>
              Microenvironmental code of pancreatic ductal adenocarcinoma: The prognostic symphony of budding, matrix and lymphocytes
              </a>
            </td>
          <td>
            Qing-Qing Zhou, Yi-Meng Ren, Su-Ming Shi, Teng-Fei Liu
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="We report the optimization of a 38-parameter spectral flow cytometry panel to identify, phenotype, and assess lineage plasticity of human regulatory T cells (Tregs). Tregs are an indispensable T cell lineage with pleiotropic tolerogenic functions whose activities contribute critically to numerous disease settings including cancer, autoimmunity, and infectious disease, among others. Phenotypic and functional heterogeneity within the Treg lineage has been appreciated, but the etiology and impact of Treg plasticity across disease states remain incompletely understood. Better tools are thus needed to deeply characterize human Treg phenotypic heterogeneity at the single cell level to advance discovery of Treg-based biomarkers in disease and to assess the effects of Treg-directed therapeutics. To this end, our 38-parameter panel consists of a 13-marker PBMC backbone module to broadly phenotype PBMCs while specifically discriminating Tregs, and a 25-marker Treg phenotyping module to determine Treg differentiation state, activation profile, and lineage subtype. In contrast to many high-parameter OMIPs that rely on surface staining only, we incorporated several intracellular targets in this panel. This afforded the opportunity to thoroughly evaluate binding characteristics of all antibodies in both pre-fixation and post-fixation and permeabilization settings; we identify several antibodies eligible for overnight post-fixation staining that require substantially reduced titers as compared to traditional pre-fixation staining, resulting in significant cost savings. Dimensionality reduction and semi-supervised clustering on healthy donor PBMC-derived Tregs profiled by our panel reveal up to 14 discrete Treg phenotypes. In sum, this panel enables deep phenotypic characterization of human Treg heterogeneity in peripheral blood specimens by spectral flow cytometry.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0d624e8d03927136ba204e575e7534e1cc07f41" target='_blank'>
              OMIP-118: A 38-Marker Spectral Flow Cytometry Panel to Assess Human Regulatory T Cell Phenotype and Lineage Plasticity.
              </a>
            </td>
          <td>
            Nilika Bhattacharya, Collin Jugler, Jessica C Hill, Kade R Copple, Sameena Nikhat, Mohsen Khosravi-Maharlooei, Casey R Ager
          </td>
          <td>2025-10-05</td>
          <td>Cytometry. Part A : the journal of the International Society for Analytical Cytology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="For over sixty years, blood researchers have been counting clones with every tool at their disposal. Inspired by phage and fly geneticists, Till and McCulloch irradiated mice to induce chromosomal aberrations. Using this labeling strategy, they demonstrated that different types of blood cells shared the same mutation in every spleen colony, thereby proving the existence of hematopoietic stem cells. Since their breakthrough, technological advances have enabled researchers to quantify hematopoiesis at single-cell resolution in increasingly complex samples across both mice and humans. With these modern sophisticated lineage tracing methods, our foundational understanding of the blood system is being reshaped. For instance, we now interpret hematopoietic architecture as arising from stem and progenitor cells of diverse developmental origins, each with distinct fate biases encoded by unique regulatory states. Interacting with this regulatory layer, genetic mutations and epimutations arise, expanding clonally and becoming pervasive with age. Together, clonal heterogeneity and age-driven clonal selection may underlie the perplexing diversity of therapy responses in cancer and beyond. As these paradigm-shifting insights gain traction, clonal tracing is being adopted across dozens of biological and clinical studies. Here, we review the modern toolbox of clonal tracking technologies, with a focus on next-generation sequencing-based approaches, and provide a practical guide for matching specific research questions with optimal experimental strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca9cbce80016ba1b72cbad0fc7cc9413794f150" target='_blank'>
              Clonal tracing of blood stem cells across mouse and human lifespans.
              </a>
            </td>
          <td>
            Alejo E. Rodriguez-Fraticelli
          </td>
          <td>2025-09-22</td>
          <td>Blood</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Notch is a contact-dependent signaling pathway that plays critical roles in spatiotemporal patterning during embryogenesis. Beyond its role in development, Notch signaling regulates tissue patterning in adult organs in response to tissue injury or cancer. In the context of cancer, the role of Notch signaling has been extensively characterized in cancer cells, T cells, and endothelial cells. Our previous work has implicated Notch signaling in the myeloid and fibroblast compartments of the pancreatic tumor microenvironment (TME) in the tumor immune response, an area that has been previously understudied. In our current work we use genetically engineered mouse models to inhibit canonical Notch activation in fibroblasts (Pdgfra CreERT2/+, LSL-DNMAML) and myeloid cells (LysM Cre/+, LSL-DNMAML) separately. In both an orthotopic transplant and genetic model (Ptf1a FlpO/+;FSF-Kras G12D/+) of pancreatic cancer progression, inhibition of fibroblast-specific Notch activation had limited effects on disease progression and the transcriptional profile of the fibroblasts themselves. This is consistent with previous evidence of more limited Notch activation in pancreatic cancer-associated fibroblasts and suggests a primary role for fibroblasts in providing Notch ligand (JAG1) to neighboring cells. In contrast, inhibition of myeloid-specific Notch activation resulted in smaller orthotopic tumors and slowed progression in a genetic model of pancreatic neoplasia. These samples had reduced overall macrophage infiltration, and a secondary alteration of the fibroblast phenotypes was also seen. Collectively, these results suggest that Notch signaling in the myeloid compartment supports tumor progression and regulates multiple components of the pancreatic TME. Future studies will further interrogate how myeloid-specific Notch activation promotes disease progression and affects Notch signaling activation in adjacent compartments.



 Allison Bischoff, Wei Yan, Carlos Espinoza, Emily Lasse-Opsahl, Nur Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar. Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A080.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df0eac24a706ec3b2888b93172ea892748b9176" target='_blank'>
              Abstract A080: Deciphering the hierarchy of Notch signaling in the pancreatic tumor microenvironment
              </a>
            </td>
          <td>
            Allison C Bischoff, Wei Yan, Carlos E Espinoza, Emily L. Lasse-Opsahl, Nur M Renollet, Yaqing Zhang, Marina Pasca di Magliano, Filip Bednar
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="PURPOSE
With surgery being the only potential cure for pancreatic cancer, high-risk premalignant pancreatic lesions often go unnoticed by palpation or white light visualization, leading to recurrence. We asked whether near-infrared fluorescence imaging of tumor-associated inflammation could identify high-risk premalignant lesions, leveraging the tumor microenvironment (TME) as a sentinel of local disease and, thus, enhance surgery outcomes.


EXPERIMENTAL DESIGN
Fluorescence-guided surgery was performed on genetically engineered mice (Ptf1a-Cre; LSL-KrasG12D/+; Smad4flox/flox [KSC]) at discrete stages of disease progression, histologically confirmed high-risk, premalignant lesions in postnatal mice to locally advanced pancreatic tumors in adults, using the imaging agent V-1520, a translocator protein (TSPO) ligand. Age-matched wild-type littermates were used as controls, while Ptf1a-Cre; LSL-KrasG12D/+(KC) mice modeled pancreatitis and precursors of low penetrance. Localization of V-1520 and tumor-associated macrophages amongst the TME was detected by immunofluorescence imaging.


RESULTS
V-1520 exhibited robust accumulation in the pancreata of KSC mice from the early postnatal stage. Increased accumulation was observed in the pancreata of adolescent- and adult-aged mice with greater ductal lesion and stromal burden. Confocal microscopy of ex vivo pancreas specimens co-localized V-1520 accumulation primarily with CD68-expressing macrophage in KSC mice. Unlike the pancreata of KSC mice, accumulation of V-1520 did not exceed background levels in the pancreata of KC mice with pancreatitis.


CONCLUSION
V-1520 exhibited differential accumulation in pancreatic cancer-associated inflammation compared to pancreatitis. Given the robust tracer uptake in tissues associated with early yet high-risk lesions, we envision V-1520 could enhance surgical resection and reduce the potential for recurrence from residual disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e1713383f0cb9b8f04471c53342e9b58b0edaa9" target='_blank'>
              Preclinical Fluorescence-Guided Imaging Leveraging Surrounding Sentinel Tumor Microenvironment Identifies High-Risk Premalignant Pancreatic Lesions.
              </a>
            </td>
          <td>
            Shilpa Sharma, Xiaoxia Wen, Jianbo Wang, Beibei Huang, Denise Hernandez, Cong-Dat Pham, Zhiwen Liu, S. J. Lin, Aiko Yamaguchi, Dimitra K. Georgiou, R. P. Coll, H. C. Manning
          </td>
          <td>2025-08-25</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cabe0cbcb8c826f9af8109abd06e0ae3e7cf0a6a" target='_blank'>
              Patient-derived organoids reveal hypoxia-driven plasticity and therapeutic vulnerabilities in pheochromocytomas and paragangliomas.
              </a>
            </td>
          <td>
            Xiaojing Wang, Maite Calucho, Cynthia M Estrada-Zuniga, Mehwish Noureen, Hector Gonzalez-Cantú, Samantha Mossuto, Jonathan N Levi, Bethany Landry, Qianjin Guo, Kailee A Rutherford, Zi-Ming Cheng, Andreanne Sannajust, Sapna Raghunathan, Sanskriti Balaji, Viviane Nascimento da Conceicao, Summer Norris, S. Y. Munawar, H. Nguyen, Jacob De Rosa, G. Tomlinson, James X Wu, Michael W. Yeh, M. Livhits, Anand Vaidya, James Powers, Ron Lechan, Art Tischler, Miao Zhang, Paul Graham, Camilo Jimenez, M. Peitzsch, Nicole Bechmann, Graeme Eisenhofer, Yanli Ding, Mio Kitano, A. Soragni, P. Dahia
          </td>
          <td>2025-08-23</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Pancreatic adenocarcinoma is one of the deadliest solid tumors, with a two-year survival rate below 5%. Progress in developing effective treatments has been limited, largely due to the absence of physiologically relevant disease models that accurately recapitulate the tumor microenvironment. Such models are critical for understanding disease mechanisms and evaluating therapeutic strategies. Organoids have recently emerged as promising platforms for modeling various diseases, including pancreatic cancer, as they retain the structural and functional features of their tissue of origin. Moreover, CRISPR-Cas9 gene editing allows precise introduction of cancer-associated mutations, providing a powerful approach for generating genetically defined models.



 We established normal pancreatic organoids from human pancreatic tissue samples and validated their identity through qPCR and histological analyses, confirming their resemblance to native pancreatic epithelium. These organoids demonstrated stable growth over several passages and retained viability after cryopreservation and thawing. To model pancreatic cancer, we used CRISPR-Cas9 technology to introduce TP53 mutations, a key tumor suppressor gene altered in over 70% of pancreatic ductal adenocarcinoma (PDAC) cases. Following Nutlin-3a selection, successful TP53 knockout was confirmed by Sanger sequencing. The engineered TP53-knockout (KO) organoids were expanded and characterized to assess cancer-like phenotypes.



 Human-derived pancreatic organoids retained their structural and functional identity across passages and after thawing. Following TP53 gene editing, the TP53 KO organoids exhibited increased proliferation and morphological changes consistent with early tumorigenesis. Importantly, qPCR analysis revealed significant upregulation of cancer-associated markers, including MUC5AC, KRT19, and CEACAM5, in TP53-mutated organoids compared to controls, further supporting their malignant transformation. These findings represent a critical step toward building a genetically defined pancreatic cancer model that closely reflects human disease.



 We successfully generated a pancreatic cancer organoid model by editing TP53 in normal human pancreatic organoids. The TP53 KO organoids showed enhanced growth, morphological transformation, and elevated expression of key cancer markers, recapitulating early features of pancreatic cancer. This model mimics critical aspects of pancreatic cancer biology and provides a robust platform for studying tumor initiation, progression, and drug response. Unlike conventional animal models or stem-cell-derived systems, our approach offers a human-based, physiologically relevant, and reproducible system for pancreatic cancer research and therapeutic screening.



 Eden Demere Amare, Soraya SALAS-SILVA, Ji Hyun Shin, Dongho Choi. CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59cdfa7b3d8511b93672f4beff88185a825b61c7" target='_blank'>
              Abstract A013: CRISPR-engineered human organoid-based pancreatic cancer model recapitulates early tumor features through TP53 inactivation
              </a>
            </td>
          <td>
            Eden Demere Amare, Soraya SALAS-SILVA, Jihyun Shin, Dongho Choi
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT Cellular senescence is a state of irreversible cell cycle arrest accompanied by a distinctive inflammatory secretory profile known as the senescence‐associated secretory phenotype (SASP). While various biomarkers, such as senescence‐associated beta‐galactosidase (SA‐βgal), EdU incorporation, p21 and p16, are used to identify senescent cells, no single biomarker universally defines cellular senescence and current methods often fail to address heterogeneity in biomarker expression levels. This study leverages single‐cell fluorescence imaging to assess multiple senescence markers including SA‐βgal enzymatic activity, p21 and IL‐6 expression and nuclear and cell area in chemotherapy‐induced (mitomycin C) and oxidative stress‐induced (D‐galactose) senescence models in human fibroblasts. Our findings reveal significant heterogeneity in SA‐βgal activity and distinct sub‐populations within senescent cells. Nuclear and cell area measurements emerged as robust indicators of cellular senescence, displaying similar variability across individual cells. Importantly, we identified specific nuclear area sub‐populations that strongly correlate with IL‐6 expression levels, demonstrating a relationship between the heterogeneous expression of senescence biomarkers and the SASP. To address this heterogeneity, we introduced an induction threshold method to more accurately quantify the percentage of cells expressing senescence biomarkers. Furthermore, in both senescence models, we observed that rapamycin, a well‐known senomorphic agent, selectively targets specific biomarker‐expressing sub‐populations. This study underscores the value of assessing cellular heterogeneity in senescence research and provides an improved approach for analysing senescence markers in diverse cellular contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86141b62ee10b7214b95e930ed5897691f6a58a" target='_blank'>
              Single‐Cell Fluorescence Imaging Reveals Heterogeneity in Senescence Biomarkers and Identifies Rapamycin‐Responsive Sub‐Populations
              </a>
            </td>
          <td>
            Vijayraghavan Seshadri, Charmaine Chng, Joel Tyler, Cestarangga Adikerta, Kaveh Baghaei, Yan Wang, Nuri Gueven, Sharon Ricardo, Iman Azimi
          </td>
          <td>2025-09-03</td>
          <td>Aging Cell</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (iCCA) is a highly aggressive and heterogeneous malignancy characterized by marked resistance to standard chemotherapy and poor prognosis. While the advent of immunotherapy has revolutionized the management of several solid tumors, including melanoma, breast cancer, and non-small cell lung cancer, its efficacy in iCCA remains limited. Recent clinical trials have demonstrated the efficacy of durvalumab in combination with chemotherapy for iCCA, leading to its approval as a first-line treatment. However, overall response rates remain low, largely due to its immunosuppressive tumor immune microenvironment (TIME). The immune-cold nature of iCCA is typified by a dominant presence of immunosuppressive cell populations, including M2-polarized tumor-associated macrophages, myeloid-derived suppressor cells, and T regulatory cells. In addition, traditional biomarkers such as PD-L1 expression, tumor mutational burden, and microsatellite instability have shown limited predictive value in iCCA, highlighting the need for novel biomarkers and immunotherapeutic strategies. Emerging approaches aimed at reprogramming the TIME, including combination therapies targeting suppressive cells, stromal remodeling, and novel immune effectors like CAR-T and cancer vaccines, hold significant promise for enhancing therapeutic efficacy. This review summarizes the distinct features of iCCA TIME, key mechanisms of immune evasion, current challenges, and future directions to overcome immune resistance, with the aim of developing personalized immunotherapies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1167c3442bd04894cf43de06826ab2ab3325aec4" target='_blank'>
              Immune Landscape of Intrahepatic Cholangiocarcinoma: Evasion and Therapeutic Insights
              </a>
            </td>
          <td>
            Nunzia Porro, Elena Spínola-Lasso, F. Marra, A. Gentilini
          </td>
          <td>2025-09-17</td>
          <td>Immuno</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Brain metastases are a significant complication of non-small-cell lung cancer (NSCLC), contributing to high morbidity and mortality rates. The introduction of immune checkpoint inhibitors (ICIs) has opened new therapeutic avenues for patients with NSCLC, including those with brain metastases. However, the distinct microenvironment of the brain presents unique challenges to the effectiveness of these treatments. This review examines the mechanisms by which ICIs impact brain metastases from NSCLC, with particular focus on immune cell trafficking across the blood–brain barrier (BBB), tumor microenvironment modulation, and transcriptomic evolution of brain-tropic tumor clones. Unlike prior reviews, we integrate emerging data from single-cell and spatial transcriptomic studies, BBB disruption mechanisms, and the tumor-supportive role of brain-resident glia. We also critically evaluate key clinical trials and real-world evidence, highlighting differences in ICI efficacy across patient subgroups and therapeutic contexts. Additionally, we address the evolving role of surgical resection, stereotactic radiosurgery, and cerebrospinal-fluid-based biomarkers in optimizing ICI-based treatment strategies. This synthesis provides a comprehensive, mechanistic, and clinically relevant framework for improving outcomes in patients with NSCLC brain metastases treated with immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c6ed65bc2f6ca5f07b82a4924c158eebc0c9a9" target='_blank'>
              Immune Checkpoint Inhibition in Patients with Brain Metastases from Non-Small-Cell Lung Cancer: Emerging Mechanisms and Personalized Clinical Strategies
              </a>
            </td>
          <td>
            Nicola J. Nasser, K. Sindhu, Loor Nasser, Z. Shafaee, Joshua Li, Lucas Resende Salgado, Baoqing Li
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background We previously defined a Genomic Dedifferentiation Signature (GDS) derived from differential gene expression analysis of sarcomatoid renal cell carcinoma (sRCC) tumor clones compared to clear cell RCC (ccRCC; Salgia et al. KCRS. 2024). GDS expression is negatively prognostic across clinical RCC cohorts, yet carries predictive capacity for identifying patients likely to achieve benefit from immune checkpoint blockade (ICB). Furthermore, we have also demonstrated that sRCCs are enriched for expression of tertiary lymphoid structure (TLS) signatures. Crucially, TLS signatures have been associated with response to ICB in ccRCC (Jammihal et al. Nature Cancer. 2025). Therefore, we sought to investigate if an interplay between expression of the GDS—a tumor-intrinsic gene signature—and TLS signatures—characterizing the inflammatory immune microenvironment—could best identify ICB sensitivity in patients with metastatic RCC. Methods Transcript-per-million normalized gene expression counts matrices and clinical metadata were collected from three trials of ICB regimens in patients with metastatic RCC: IMmotion 151 (Motzer et al. Cancer Cell. 2020), Javelin101 (Motzer et al. Nature Medicine. 2020), and HCRN-GU16-260 (Hugaboom et al. Cancer Discovery. 2025). Gene expression scores for the GDS and the Imprint TLS signature (Meylan et al. Immunity. 2022) were assigned to cases using single sample gene set enrichment analysis. Correlation between GDS and TLS scores was performed via Spearman’s rank-order correlation. Cases were stratified into -high and -low signature expression (TLShigh-GDShigh, TLShigh-GDSlow, TLSlow-GDShigh, and TLSlow-GDSlow) based on median signature expression within trial cohorts. Progression-free survival (PFS) was compared between groups and trial arms within each cohort using Cox proportional hazards models, while objective response rates (ORR) were compared using Fisher’s exact tests. Results The correlation between GDS & Imprint TLS expression was moderate across IMmotion151 (n = 822; R = 0.44; p < 2.2x10-16), Javelin101 (n = 723; R = 0.43; p < 2.2x10-16) and HCRN-GU16-260 (n = 72; R = 0.53; p = 2.6x10-6). Within IMmotion151, patients classified as TLShigh-GDShigh experienced prolonged PFS with the ICB-containing atezolizumab/bevacizumab regimen (median PFS: 10.9 months) compared to sunitinib (median PFS: 6.9 months; Hazard Ratio [HR]=0.68; P = .009; Figure 1), whereas no significant differences between atezolizumab/bevacizumab and sunitinib PFS were observed in TLShigh-GDSlow (HR = 1.18; P = .5), TLSlow-GDShigh (HR = 0.71; P = .09), or TLSlow-GDSlow (HR = 0.93; P = .6) subgroups. ORR to atezolizumab/bevacizumab within the TLShigh-GDShigh population (47.5%) was also significantly higher compared to atezolizumab/bevacizumab ORR within the TLSlow-GDSlow population (30.6%; P = .008) and compared to ORR with sunitinib in the TLShigh-GDShigh population (31.0%; P = .009). Conversely, ORR with sunitinib (43.6%) was significantly greater than with atezolizumab/bevacizumab (30.6%; P = .004) in the GDSlow-TLSlow population. Consistent results were observed in Javelin101: the TLShigh-GDShigh population experienced prolonged PFS with the ICB-containing axitinib/avelumab regimen (median PFS: 12.5 months) compared to sunitinib (median PFS: 6.9 months; HR = 0.47; P < .001). Again, no significant differences between axitinib/avelumab and sunitinib PFS were observed in TLShigh-GDSlow (HR = 1.04; P = .9), TLSlow-GDShigh (HR = 0.81; P = .4), or TLSlow-GDSlow (HR = 0.81; P = .3) subgroups. In the HCRN-GU16-260 trial of front-line nivolumab in metastatic RCC, TLShigh-GDShigh status was associated with prolonged PFS, albeit non-significantly, compared to TLShigh-GDSlow (HR = 2.02; P = .10), TLSlow-GDShigh (HR = 1.99; P = .13) and TLSlow-GDSlow (HR = 1.72; P = .09). Additionally, ORR to nivolumab was significantly enriched in the TLShigh-GDShigh cohort (76.9%) compared to those classified as TLShigh-GDSlow (23.1%; P = .0009), TLSlow-GDShigh (25.0%; P = .01), and TLSlow-GDSlow (18.5%; P < .0001) within HCRN-GU16-260. Conclusions Across metastatic RCC patient cohorts, simultaneous enrichment of tumor-intrinsic and TLS gene expression best predict response to ICB versus anti-angiogenic therapy. This suggests that a pro-inflammatory milieu is not sufficient for effective immune sensitivity, but that a biologic interplay between tumor cells and their surrounding microenvironment is necessary for effective immune surveillance and ICB responsiveness in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b4535064bf0393b2610743ad10a87355b938d65" target='_blank'>
              32Interplay of Tumor-Intrinsic and microenvironmental gene expression predicts immune checkpoint blockade response in metastatic renal cell carcinoma
              </a>
            </td>
          <td>
            Nicholas J. Salgia, Eric C. Kauffman, J. Muhitch
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb5a38dd74bce74d21cd64c9a442cf5e375b46f4" target='_blank'>
              Targeting Glioblastoma Cell State Plasticity for Enhanced Therapeutic Efficacy.
              </a>
            </td>
          <td>
            S. Cirigliano, R. Singhania, James G Nicholson, Isha Monga, Yushan Wan, C. Haywood, Ashlesha Muley, Skylar A. Giacobetti, Howard A. Fine
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da9274332e90da211d1dc7074d7c247b79d5167" target='_blank'>
              The Pdgfd-Pdgfrb axis orchestrates tumor-nerve crosstalk in pancreatic cancer.
              </a>
            </td>
          <td>
            Peter L. Wang, N. Lester, Ella Perrault, J. Su, Dennis Gong, C. Shiau, Jingyi Cao, Phuong T T Nguyen, Jung Woo Bae, Deniz G Olgun, H. Hoffman, Ashley Lam, Jean Huang-Gao, Saifur Rahaman, Jimmy A. Guo, J. Barth, Nicholas J Caldwell, P. Divakar, Jason Reeves, Arya Bahrami, Shanshan He, Michael Patrick, Eric M. Miller, M. Ganci, Grissel Cervantes-Jaramillo, Theodore Hong, Jennifer Wo, Hannah Roberts, Ralph Weissleder, Hongyoon Choi, C. Fernández-del-Castillo, Kathy Cormier, David T Ting, Tyler Jacks, Lei Zheng, M. Hemberg, Mari Mino-Kenudson, William L Hwang
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive and lethal malignancy characterized by a profoundly immunosuppressive tumor microenvironment. Within PDAC, tumor-associated macrophages (TAMs) are the most abundant immune cell population. Here, TAMs orchestrate the characteristic immunosuppression of the pancreatic tumor microenvironment, drive disease progression, and mediate therapeutic resistance in PDAC. Accordingly, successful development of approaches that reprogram or eliminate these pro-tumor immune populations have potential to improve patient outcomes. One of the most challenging aspects of developing approaches to modulate TAMs programming is the ability to perform mechanistic experiments in a system the recapitulates the complex pancreatic tumor microenvironment. We and others have leveraged primary bone marrow-derived macrophages (BMDMs) polarized by PDAC conditioned media as a discovery platform that has yielded translational findings. However, this system fails to account for the dynamic and reciprocal interactions between myeloid and cancer cells that occur within tumors. Accordingly, we sought to address if this model could be built into progressively more complex systems to assess if these better recapitulate TAMs using a systems biology approach. Using transcriptomic profiling, we compared different in vitro strategies, from direct co-cultures to conditioned media polarization, against TAMs harvested from orthotopic PDAC tumors. Our preliminary data show that allowing PDAC cells to engage in direct cellular contact and sustained paracrine signaling most closely resemble the transcriptomic profile of bona fide TAMs. Our further efforts to use these PDAC-macrophage culture models will be key to identifying relevant signaling pathways that can be mechanistically examined prior to more cumbersome in vivo validation. Collectively, our data can be used to facilitate more accurate and insightful investigations into PDAC immunology and the development of therapeutic targeting strategies.



 Cavina K. Lee, Cecily Anaraki, Austin D. Silva, Tiffany Lai, Hillary Tran, Arjun Sapru. Dissecting myeloid - cancer communication networks in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B104.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad71226813c7f5d3d378a55b84523f40aab08ce1" target='_blank'>
              Abstract B104: Dissecting myeloid - cancer communication networks in pancreatic cancer
              </a>
            </td>
          <td>
            Cavina K. Lee, Cecily Anaraki, Austin D. Silva, Tiffany Lai, Hillary Tran, Arjun Sapru
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In most solid tumors, hypoxia constitutes a defining microenvironmental feature that reprograms malignant cells into a highly metastatic state by driving cellular plasticity and exacerbating chromosomal instability (CIN). However, the mechanisms by which cancer cells concurrently co-opt these elements of hypoxic adaptation to promote metastasis remains poorly understood. Here, we report that hypoxia promotes metastasis by suppressing the JmjC-containing histone lysine demethylase Kdm8. CRISPR/Cas9-mediated targeting of Kdm8 in a Kras;Trp53-driven mouse model of pancreatic ductal adenocarcinoma (PDA) robustly rewires the malignant cell transcriptomic programs, leading to a profound loss of the epithelial morphology and widespread metastatic disease. In PDA patients, a high KDM8-induced gene signature is associated with reduced metastatic burden and better survival in advanced disease. Notably, Kdm8 suppression in normoxia recapitulates key aspects of the global epigenetic and transcriptomic rewiring, mitotic spindle defects, and CIN induced by hypoxia. Moreover, disruption of Kdm8’s demethylase activity phenocopies Kdm8 loss, whereas expression of hypermorphic Kdm8 variants resistant to hypoxic suppression markedly reduces metastasis beyond the levels achieved by the wildtype protein. Through the suppression of Kdm8 demethylase function, hypoxia unleashes a potent metastatic program by simultaneously advancing cellular plasticity and CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ead3df1c0ef44aa86f0839ae810b8a1fead329" target='_blank'>
              Hypoxia-induced metastatic heterogeneity in pancreatic cancer
              </a>
            </td>
          <td>
            Pradeep Moon Gunasekaran, Qianqian Wang, Yoke-Chen Chang, Polina Guseva, Rajika Chauhan, Alexander Kley, Gene Lee, Siddharth Ghosh Roy, Yousef Masoudpoor, Arthur Roberts, Kelly Watkins Walton, Lucyann Franciosa, Shafiq Bhat, Emmanuel Zachariah, Kishan Patel, Zhongren Zhou, Wenjin Chen, J. Ni, S. Gu, Cristina Montagna, Shin-Heng Chiou
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumor-immune interactions are shaped by both antigenic heterogeneity and stochastic perturbations in the tumor microenvironment, yet the mathematical mechanisms underlying immune phase transitions remain poorly understood. We propose a four-compartment dynamical model that incorporates antigen accumulation and immune escape mutations. Bifurcation analysis reveals bistability between immune surveillance and immune escape states, providing a mechanistic explanation for heterogeneous immune outcomes during tumor progression. In the multistable regime, the stable manifold of a saddle point partitions the state space into distinct basins of attraction, determining the long-term fate of the system. We further analyze how stochastic fluctuations in the tumor microenvironment perturb these separatrices, potentially triggering irreversible state transitions. By characterizing the critical noise intensity and estimating the tipping time, we establish a mathematical framework for assessing noise-induced transitions. The model further predicts that increasing tumor cell death can improve system resilience to stochastic perturbations, whereas stronger immune pressure may facilitate immune escape-highlighting the nonlinear and non-monotonic nature of tumor-immune dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/358f26ec76382702bbe9f0bdc46b3b599d87dd6f" target='_blank'>
              Bistability and Noise-Induced Evasion in Tumor-Immune Dynamics with Antigen Accumulation and Immune Escape
              </a>
            </td>
          <td>
            Mengfan Tan, Shaoqing Chen, Chunjin Wei, Da Zhou
          </td>
          <td>2025-09-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="As a common malignant tumor, the heterogeneity of colorectal cancer plays an important role in tumor progression and treatment response. In recent years, the rapid development of single-cell transcriptomics and spatial transcriptomics technologies has provided new perspectives for resolving the heterogeneity of colorectal cancer. These techniques can reveal the complexity of cellular composition and their interactions in the tumor microenvironment, and thus facilitate a deeper understanding of tumor biology. However, in practical applications, researchers still face technical challenges such as data processing and result interpretation. The aim of this paper is to explore how to use artificial intelligence (AI) technology to enhance the research efficiency of single-cell and spatial transcriptomics, analyze the current research progress and its limitations, and explore how combining AI approaches can provide new ideas for decoding the heterogeneity of colorectal cancer, and ultimately provide theoretical basis and practical guidance for the clinical precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48a5aab5c189afed70eaf2f7feb72d37acadd46e" target='_blank'>
              Multidimensional decoding of colorectal cancer heterogeneity: Artificial intelligence-enabled precision exploration of single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Wen-Yu Luan, Qi Zhao, Zheng Zhang, Zhen-Xi Xu, Si-Xiang Lin, 
          </td>
          <td>2025-10-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb8107340798d9682d8c6977b5d27a13471e822" target='_blank'>
              Integrated single-cell and spatial transcriptomics uncover the prognostic, epigenetic, and immunological roles of FANCC in low-grade glioma.
              </a>
            </td>
          <td>
            Zongye Zhang, Zhi Sha, Han Liu, Yusheng Chen, Zhendong Liu, Xingbo Cheng, Sujie Gu, Yanzheng Gao
          </td>
          <td>2025-09-09</td>
          <td>Neurological research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="In prostate cancer, it is recognized that adenocarcinoma can transdifferentiate into neuroendocrine prostate cancer (NEPC) owing to lineage plasticity; however, transdifferentiation into other histological types remains uncertain. We present a case of a patient who underwent surgery for adenocarcinoma, which later recurred as prostate carcinosarcoma. Genomic analysis revealed a TMPRSS2-ERG fusion, confirming a common clonal origin and transdifferentiation from adenocarcinoma to carcinosarcoma. Additionally, we identified a frameshift mutation in TP53 and the loss of PTEN and RB1. Transcriptome analysis revealed enriched epithelial-mesenchymal transition and immune-related pathways, a pattern distinct from both adenocarcinoma and NEPC. To our knowledge, this is the first report that comprehensively evaluated the clonal origin of the rare prostate carcinosarcoma and characterized it using genomic and transcriptomic sequencing. It enhances our understanding of prostate cancer lineage plasticity and highlights the importance of developing novel therapies specifically targeted at prostate carcinosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc3dc165d184c74305476d6dbd5e449b98486ca" target='_blank'>
              Case Report: Genomic insights into prostate adenocarcinoma transdifferentiation to carcinosarcoma due to lineage plasticity
              </a>
            </td>
          <td>
            Tomohiro Fukui, Arinobu Fukunaga, Yuki Teramoto, Maki Fujiwara, Kensuke Hikami, T. Sunada, Kei Mizuno, Yuki Kita, Takayuki Sumiyoshi, Takayuki Goto, R. Saito, Takashi Kobayashi, S. Akamatsu
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Despite the clear genetic evidence linking serous tubal intraepithelial carcinoma (STIC) and high-grade serous ovarian carcinoma (HGSOC), the specific conditions and events that promote the progression of STIC lesions into invasive disease remain poorly understood.



 As a critical initial step, we have assembled a cohort of incidental p53 signatures (p53.I), incidental STIC (STIC.I), and STIC with concurrent HGSOC (STIC.C). We have performed extensive multi-modal analysis using multiplexed tissue imaging and spatial transcriptomics to identify features of the immune system that play a vital role in the early steps of HGSOC development. We have processed 43 specimens using highly multiplexed tissue imaging at single-cell resolution (cyclic immunofluorescence, CyCIF), and 35 specimens for micro-regional spatial whole transcriptomics using the GeoMx (Nanostring) on over 500 pathologist-annotated regions of interest.



 We showed a temporal evolution of HGSOC, including the activation of the interferon (IFN) signaling pathway underpinning the progression from the very early stage, p53.I. Additional cancer hallmark pathways, such as Transforming Growth Factor Receptor-β, Hypoxia, were induced in the tumor microenvironment (TME), including STIC.C. One of the mechanisms of the early induction of IFN might be induced by DNA damage response and chromosomal instability, potentially activating the cGAS-STING pathway. We validated IFN activation at the early stage of HGSOC by tissue imaging. For instance, more epithelial cells expressing both HLA-A as well as HLA-E, p-TBK1 and p-STAT3 with disease progression than FT regardless of BRCA status (p<0.001, GLMMs). We found IFN signaling to be persistent at STIC.C and further upregulated in the tumor. To model the evolution of the cancer ecosystem, we then looked into immune composition at both protein and RNA levels. Our data suggested active immune surveillance and potential elimination for p53 signatures. Immune surveillance was potentially further increased at STIC.I by an increased activated cDC1, antigen-presenting cells (APCs), and >30% activated (PD1+) cytotoxic (CD8+ CD103-) and TRM with some level of exhaustion (LAG3+). STIC.C, presumably resembling late clonal expansion, had less intra-tumoral NK and cDC1 in both protein and RNA levels. There was a gradual decline in total CD8+ T cells with HGSOC progression, while 30-40% expressed Ki67/PD1 or LAG3 in cancer, indicating immune editing. Exhaustion state was further shown with HAVCR2 expression (TIM3). We found that one of the contributors of T cells exhaustion was chronic IFN.



 Epithelial cells with IFN activation seem to be under a natural selection with HSGSOC progression. One of the selective advantages might be the upregulation of MHC-class I. HLA-E upregulation by tumor cells was shown to be immune suppressive by reducing NK cell surveillance. Along with the gradual accumulation of cancer hallmark pathways, an emergence of immune suppressive mechanisms seems to arise at the very early precursor stage.



 Tanjina Kader, Jia-Ren Lin, Clemens Hug, Shannon Coy, Yu-An Chen, Ino de. Bruijin, Clarence Yapp, Jeremy L. Muhlich, Nikolaus Schultz, Charles W. Drescher, Peter K. Sorger, Ronny Drapkin, Sandro Santagata. Immune suppression in the fallopian tube: Insights from the pre-cancer atlas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA009.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bfa669bc19f1cf6d070d242569736aa782cd91a" target='_blank'>
              Abstract IA009: Immune suppression in the fallopian tube: Insights from the pre-cancer atlas
              </a>
            </td>
          <td>
            Tanjina Kader, Jia-Ren Lin, Clemens Hug, S. Coy, Yu-An Chen, Ino de. Bruijin, Clarence Yapp, J. Muhlich, N. Schultz, Charles W Drescher, P. Sorger, Ronny Drapkin, S. Santagata
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT Accumulating evidence suggests that the TP53 mutation, intratumoral microbiome, and tumour microenvironment (TME) are closely linked to tumourigenesis, yet the biological mechanisms underlying these connections remain unclear. To explore this, we collected multi‐omics data—including genome, transcriptome, and tumour microbiome data—from a wide range of cancer types in The Cancer Genome Atlas (TCGA). Through a pan‐cancer analysis, we identified significant correlations between intratumoral microbiota diversity and TP53 mutation status, particularly in hepatocellular carcinoma (HCC) and endometrial cancer (EC). Despite notable differences in microbiota composition between these two cancer types, we consistently observed that TP53 mutations were associated with reduced alpha‐diversity. Additionally, we found that TP53 mutation status significantly influenced stromal components within the TME, such as a strong correlation between decreased endothelial cell abundance and TP53 mutation. Our integrated approach reveals the complex interplay between TP53 and factors regulating the host TME, offering new insights into cancer progression and potential therapeutic targets for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db906f017cde5061fad12eabf32c649d299b5278" target='_blank'>
              Pan‐Cancer Integrative Analyses Reveal the Crosstalk Between the Intratumoral Microbiome, TP53 Mutation and Tumour Microenvironment
              </a>
            </td>
          <td>
            Baoling Wang, Bo Zhang, Chun Li
          </td>
          <td>2025-01-01</td>
          <td>IET Systems Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) is a highly vascularized and immunogenic malignancy with a complex tumor microenvironment (TME) that shapes disease progression and therapeutic resistance. Despite advances in immune checkpoint inhibitors (ICIs) and targeted therapies, clinical responses remain heterogeneous, underscoring the need for a deeper understanding of RCC immunobiology. This review comprehensively examines the immunosuppressive TME of RCC, emphasizing the roles of cytotoxic and immunosuppressive immune cells, carcinoma-associated fibroblasts (CAFs), abnormal vasculature, and extracellular matrix (ECM) remodeling in fostering immune evasion. This review summarized emerging biomarkers—including PD-L1 expression, tumor mutational burden (TMB), gene mutations, and immune-based subtypes—that may predict ICI response. Furthermore, we evaluate current immunotherapeutic strategies, such as ICIs, combination therapies, and novel approaches targeting immunosuppressive cells and metabolic pathways. While combination therapies have improved outcomes, challenges like toxicity and resistance persist, necessitating biomarker-driven patient stratification and optimized treatment sequencing. Future directions should focus on deciphering TME heterogeneity and developing precision immunotherapy strategies to enhance clinical efficacy in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c635f92429f0a0ad81f2c837971a966305d3f506" target='_blank'>
              Characteristics of the tumor microenvironment and potential immunotherapy strategies in renal cell carcinoma
              </a>
            </td>
          <td>
            Hui Wen, Shi Zheng, Xiaoqin Zhu, Ling Wang, Dongping Chen
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b038428041ef5d754c8d9cbe5398a0911b1f648" target='_blank'>
              Fusion-driven oncogenic programs shape the immune landscape in translocation renal cell carcinoma.
              </a>
            </td>
          <td>
            Prathyusha Konda, Cary N. Weiss, Yantong Cui, Sayed Matar, Jinyu Wang, Yasmin L Nabil, Riva Deodhar, Jiao Li, J. Horst, Sabrina Y. Camp, A. Sheshdeh, Jonathan L. Hecht, David J. Einstein, Yashika Rustagi, Anwesha Nag, A. Thorner, Cheng-Zhong Zhang, E. V. Van Allen, S. Signoretti, T. Choueiri, Srinivas R Viswanathan
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="The tumor microenvironment (TME) comprises neoplastic and stromal cells, and extracellular matrix elements, all engaging in a complex interplay that ultimately dictates tumorigenesis, cancer progression, and therapeutic response. While extensive research on the TME has been conducted in human oncology, data on its veterinary counterpart, particularly in feline mammary tumors (FMTs), are still scarce. In this review, we explore current understanding of feline mammary carcinoma (FMC) microenvironment, focusing on tumor necrosis, fibrosis, angiogenesis, adipose tissue tumor-associated inflammation, extracellular vesicles, and epithelial–mesenchymal transition (EMT) and their prognostic implications. In FMC, remodeling of collagen fibers, cancer-associated fibroblasts (CAFs), regulatory T cells (Tregs) and elevated serum leptin have been associated with poor prognosis, whereas stromal cytotoxic T cells correlate with more favorable outcomes. By contrast, findings on necrosis and pro-angiogenic factors remain inconsistent, and research on extracellular vesicles (EVs) is still in its early stages. This review presents insights from human breast cancer (HBC) that further support and elucidate the potential relevance of these TME components. As FMCs are highly aggressive tumors, a deeper understanding of their microenvironment could not only improve prognostic accuracy but also uncover novel therapeutic targets. Furthermore, due to their similarities, FMCs offer a potential valuable spontaneous model for HBC, particularly for the aggressive triple-negative phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acfc9c9171773b52e82ceb2afc7e5c11c59650bc" target='_blank'>
              Inside the Tumor: Decoding the Feline Mammary Tumor Microenvironment and Its Prognostic Value—A Review
              </a>
            </td>
          <td>
            J. Rodrigues-Jesus, Ana Canadas-Sousa, Hugo Vilhena, P. Dias-Pereira
          </td>
          <td>2025-10-08</td>
          <td>Veterinary Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Lineage plasticity, a critical hallmark of cancer progression, enables tumor cells to evade inhibition of primary oncogenic pathways through histologic transformation. This adaptive process, driven by stemness-associated features and epigenetic reprogramming, poses significant challenges in treatment. Using non-small cell lung cancer and prostate cancer as models, we examine the utility of tissue and liquid biopsies in detecting histologic transformations and tailoring treatments to specific subtypes, which has profound clinical implications, potentially improving outcomes in patients with advanced, therapy-resistant disease. We also discuss emerging therapeutic strategies, including novel molecular targets, and address ongoing clinical challenges in managing treatment-emergent histologic transformation.


SIGNIFICANCE
The advent of highly effective molecularly targeted therapies results in increased recognition of treatment-emergent histologic transformation. This review not only summarizes current evidence on diagnosis and management of lineage plasticity but also explores therapeutic strategies under study, outlining a framework for clinical translation and successful drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49dc4debf7b809a5797b2dd12ac58569636b53ca" target='_blank'>
              Histologic Transformation in Cancer: The Path for Clinical Translation.
              </a>
            </td>
          <td>
            Ioannis A. Vathiotis, Avisek Banerjee, Utsav Sen, Felix C Saalfeld, Debdatta Halder, A. Charpidou, Konstantin Syrigos, H. Horinouchi, Ashutosh K Tewari, Mark A Dawson, M. Wermke, Triparna Sen
          </td>
          <td>2025-09-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Colorectal laterally spreading tumors (LSTs) are defined as non-protruding neoplasms exceeding 10 mm in diameter that grow primarily along the intestinal wall. The morphogenetic mechanisms and evolutionary trajectories of granular-type LSTs (LST-Gs) towards colorectal carcinoma remain unclear. Methods In this study, we investigate the transcriptional features of LST-Gs using single-cell RNA sequencing technology by comparing them with protruded-type adenomas (PAs) and normal mucosal tissues. Results Adenomatous LST-Gs harbor an epithelial cell population with metaplastic differentiation, which are almost absent in PAs. Cells with a high degree of differentiation in LST-G demonstrate enhanced immunogenicity and robust adhesion/junction interactions. Furthermore, LST-Gs show upregulated expression of molecular chaperones and metallothioneins compared to PAs, reflecting a more hostile microenvironment similar to that observed in carcinoma stages. These characteristics suggest that LST-G exhibits greater heterogeneity compared to the earliest colorectal adenomas, mirroring the progression from precancerous states to cancer. Notably, the Arp2/3 complex is significantly upregulated in highly differentiated LST-G cell populations, potentially facilitating cell migration along the basement membrane, which highlights the similarities in cell motility between adenomatous LST-G and normal mucosal epithelium as well as serrated polyps (SERs). Conclusion The differentiation state of cells within LST-G exhibits a close correlation with their diverse characteristics. Metaplastic differentiation, as a prominent feature at the transcriptional level, demonstrates significant associations with the genomic features, morphogenesis, and tumor progression of LST-G.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21cfb702cab41be0ca69fc6ec0389c21e705f623" target='_blank'>
              Single-cell RNA profiling of colorectal granular-type laterally spreading tumor uncovers progression trajectory toward carcinoma and transcriptional signatures favoring lateral morphogenesis
              </a>
            </td>
          <td>
            Yueqing Gong, Yuxin Zhang, Xun Liu, Rongli Cui, Jingjing Lu, Jun Li, Fang Gu, Jing Zhang, Shigang Ding, Weiwei Fu
          </td>
          <td>2025-09-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer with a dismal five-year survival rate of 13%. Most patients present with metastatic disease, and 80% undergoing surgery relapse within two years. Research on PDAC has traditionally focused on primary tumors, thus most treatment derives from primary tumor biology. Recent evidence unveiled significant differences in the tumor immune microenvironment (TIME) between primary and metastatic lesions. Understanding these differences is crucial, as immunotherapies are predominantly administered to patients with refractory metastatic PDAC, yet traditional treatment fails to account for organ specificity and patient TIME signatures. In this study, we wanted to know why the same cancer cell type in different organs would create different microenvironments in the pancreas, liver, and lungs. To do this, we investigated the TIME in early and late-stage metastatic cohorts by multiplex immunohistochemistry (mIHC), transcriptomic assays and spatial analysis. To profile the TIME of different immune and stromal cells, we employed spatial protein expression using mIHC platform for the early cohort. In total, we applied a 30-plex panel to 56 samples. Our analysis revealed a higher density of COL1A1+ fibroblasts and CD163+ macrophages in the liver compared to other sites. Additionally, liver metastases exhibited significantly fewer CD8+ T cells compared to other organs within the tumor core. These results confirmed the earlier studies regarding the TIME of metastatic lesions, while also showing more macrophage and fibrosis in the liver. In the late-stage cohort, similar trends of decreased immune cell presence and exclusion phenotypes were observed using mIHC and Visium platforms. To explore the underlying mechanisms, we performed transcriptomic assays on the immune compartment. Our findings indicated an enrichment of interferon-positive macrophage subsets, particularly in liver metastases compared to primary PDAC. Additionally, we observed a higher presence of exhaustive CD8+ T cells and regulatory T cells in primary sites compared to liver and lung metastases, suggesting that both primary PDAC and liver metastases possess an immunosuppressive environment. Furthermore, spatial proximity analysis demonstrated that CD8+ T cells were closer to tumor cells in lung metastases compared to liver and primary tumors, reinforcing the notion that lung metastases are more immune-active. Pathway analysis revealed downregulation of antigen presentation and lymphocyte proliferation signaling in liver T cells compared to those in the lungs. Notably, immune infiltration was observed in some liver samples, but the percentage was significantly lower compared to lung samples. This divergence in TIME modulation between organs suggests a significant impact on disease progression in PDAC patients. The significant difference in immuno-response across organ shows why it is critical to consider distinct immune landscapes of different organs and patient histories when developing tailored therapies for advanced PDAC patients.



 Yu-Lan Kao, Olivia M. Dres, Blake G. Sells, Varun Shenoy, Xiang Li, Kapur Dhami, Paul M. Grandgenett, Li Ding, Liang-I Kang, David G. DeNardo. Dynamics of tumor immune environment across organs in early and late-stage metastatic pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A084.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/846e100bfc6ae1b16ca4e9e6b72859a2dc02155f" target='_blank'>
              Abstract A084: Dynamics of tumor immune environment across organs in early and late-stage metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Yu-Lan Kao, Olivia M Dres, Blake Sells, Varun V Shenoy, Xiang Li, Kapur B. Dhami, P. Grandgenett, Li Ding, Liang-I Kang, D. DeNardo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Abstract Background With immune checkpoint inhibitors (ICIs) now widely used as a primary treatment for metastatic RCC, understanding the composition and function of the tumor microenvironment (TME) has become more important. Numerous studies have focused on T cells to explain both treatment response and resistance, and single-cell RNA sequencing (scRNA-seq) has revealed the complex heterogeneity of T cell states. However, the contribution of myeloid cells (particularly tumor-associated macrophages (TAMs), which are known to suppress antitumor immunity) remains incompletely understood in the context of ICI therapy. In this study, we aimed to investigate TAM populations associated with ICI resistance in RCC using single-cell transcriptomic profiling. Methods We analyzed 70 tumor samples (58 clear cell and 12 non-clear cell) obtained from 63 patients with advanced RCC. The cohort included 9 untreated patients, 10 who received non-ICI systemic therapies, and 44 who were treated with ICI-based regimens. From the ICI-based therapy group, we excluded 17 patients who had stable disease and focused on 29 tumor samples from 27 patients with either pre-treatment (n = 15) or post-treatment (n = 14) samples. These patients received various ICI regimens, including mono-ICI (n = 11), ICI plus ICI (n = 11), ICI plus VEGF inhibitor (n = 6), and ICI plus IDO1 inhibitor (n = 1). We performed scRNA-seq (10x Genomics) on all samples and applied non-negative matrix factorization (NMF) to identify transcriptional programs within TAMs. We then compared these programs between responders (n = 18, complete or partial response) and non-responders (n = 11, progressive disease) based on RECIST criteria. Statistical significance was assessed using the Wilcoxon signed-rank test. Results A total of 443 337 high-quality viable cells were analyzed and classified into major cell types, including lymphoid, myeloid, tumor, endothelial, and fibroblast compartments. Within the TAM compartment, NMF uncovered 8 gene expression programs, such as “antigen presentation”, “S100A8/9 inflammation”, “stress response”, “C1Q/APOE/TREM2 signaling”, “CD163/MRC1-M2-like”, “hypoxia-related signaling”, “interferon-stimulated response”, and a distinct “LILRB/SIGLEC10” immunosuppressive program. This LILRB/SIGLEC10-enriched TAM subcluster was significantly more abundant in non-responders than in responders (P = .005). Importantly, this difference was also observed in pre-treatment samples alone (P = .014), suggesting it may be involved in primary resistance. These TAMs expressed higher expression levels of immunosuppressive LILRB1/2/3 genes, the inhibitory receptor SIGLEC10 (a recently characterized “don’t eat me” signal), and the immune checkpoint molecule VISTA, compared to other TAM subclusters (p < 2.22E-16 for each gene). Conclusions Our scRNA-seq-based analysis identified a distinct population of TAMs characterized by immunosuppressive transcriptional programs associated with poor response to ICI therapy in RCC. These findings provide insight into potential mechanisms of resistance and suggest that targeting this TAM subset may improve therapeutic efficacy. This study also demonstrates the utility of single-cell transcriptomics for uncovering key immunoregulatory populations in large clinical cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c5a6a9e73d777a1a4cc9b52941635d122ce502" target='_blank'>
              41Single-cell dissection of immunosuppressive myeloid subclusters driving resistance to immune checkpoint therapy in renal cell carcinoma (RCC)
              </a>
            </td>
          <td>
            S. Kashima, Rishabh Rout, M. Hugaboom, Zhaochen Ye, N. Schindler, A. Dighe, Maxine Sun, G. M. Lee, Wenxin Xu, S. Signoretti, Bradley A. McGregor, Rana McKay, Toni Choueri, D. Braun
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Intratumoral heterogeneity remains a major barrier to durable cancer therapies, largely driven by the persistence of cancer stem cells (CSCs). In this study, we employed an integrated, multi-scale approach to investigate how melanoma CSCs respond to siRNA-mediated silencing of three key regulatory genes: KLF4, SHH, and HIF1α. Using a combination of morphological, molecular, spectroscopic, and elemental analyses, we explored structural and biochemical consequences of gene knockdown. Gene silencing resulted in significant changes in cell shape and size, reduced F-actin organization, and decreased PFN1 expression, indicating a loss of stem-like properties. ATR-FTIR spectroscopy revealed shifts in biomolecular composition, notably a reduction in amide III intensity and an increase in lipid ester content. SEM-EDS point-based elemental analysis revealed SEM-EDS point-based elemental analysis revealed relative differences in carbon and nitrogen levels between selected central and peripheral regions of silenced and control cells, at the micron-scale working depth, reflecting broader elemental distribution trends rather than precise subcellular compartmentalization. XPS analysis further confirmed these differences, providing additional insights into the elemental composition of the cellular surface. The integration of FTIR spectroscopy into this study highlights the potential of infrared spectroscopy as a powerful tool in cancer research. These findings demonstrate that targeting critical regulatory pathways induces cytoskeletal and biochemical remodelling in melanoma CSCs, offering a multi-dimensional perspective on cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5835a6c3f340e01657d8039c9562c5a0b45cfc" target='_blank'>
              Integrated spectroscopic and morphological analyses reveal cellular shifts in gene-silenced melanoma CSCs
              </a>
            </td>
          <td>
            Berrin Ozdil, Günnur Güler, E. Ataman, H. Aktug
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality, with genetic, epigenetic, and immune-dependent mechanisms contributing to disease pathogenesis. Preventive strategies are especially important for individuals with hereditary syndromes such as familial adenomatous polyposis (FAP), in which Adenomatous Polyposis Coli (APC) mutations drive early tumor formation. The polyposis in rat colon (Pirc) model, harboring an Apc mutation, faithfully recapitulates the development of adenomatous polyps, a precursor stage of CRC, in both the colon and duodenum. Here, we report the generation and characterization of primary cell cultures derived from 30 colonic and duodenal adenomas (benign tumors) from Pirc rats. These Pirc colon adenoma and duodenal adenoma cultures demonstrated consistent morphology, epithelial marker expression (E-cadherin and pan-cytokeratin), and robust 3D spheroid formation. Growth kinetics revealed a doubling time of 50 ± 4 hours. Notably, histone deacetylase and bromodomain inhibitors reduced cell viability and colony formation significantly, highlighting the synergistic potential in targeting deregulated epigenetic signatures as early-stage prevention strategies in FAP. These primary cultures, maintained between passages 5 and 10 to preserve phenotypic integrity, offer a valuable ex vivo model for early-stage CRC research and the screening of preventive agents. By bridging genetic susceptibility and translational prevention, this platform provides a novel and reproducible tool for advancing cancer interception.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6739c771e5dffc00116bad98344e26be1545833" target='_blank'>
              Development of Primary Colon and Duodenal Tumor Cultures from the Apc-mutant Pirc Rat for Screening Preventive Agents
              </a>
            </td>
          <td>
            Ahmed Muhsin, Sultan Neja, Nivedhitha Mohan, Sabeeta Kapoor, J. E. Tovar Perez, W. Dashwood, Asif Rashid, Roderick H. Dashwood, P. Rajendran
          </td>
          <td>2025-09-10</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Despite decades of progress in ovarian cancer treatment, therapeutic resistance remains a persistent and lethal challenge. Emerging evidence suggests polyploid giant cancer cells (PGCCs) as key drivers of this resistance. PGCCs are distinguished by multinucleation or a single oversized nucleus containing multiple chromosome sets. Polyploid cell populations are present in premalignant lesions and expand with disease progression and therapeutic stress. PGCCs demonstrate heightened resistance to treatment relative to non-PGCCs and can regenerate non-PGCCs via budding once therapeutic pressure is alleviated, leading to relapse. Supported by extensive preclinical and clinical evidence, we hypothesize that elimination of PGCCs may reduce therapeutic resistance in ovarian cancer. Given their significant role in resistance and relapse, PGCCs remain largely untargeted due to a lack of scalable detection methods. Standard viability assays such as MTT, XTT, and ATP-based screens fail to capture the rare but critical PGCC subpopulation. Current gold-standard methods, fluorescence-activated cell sorting (FACS) coupled with manual microscopy, while accurate, are time-intensive and unsuitable for high-throughput drug screening. Thus, a rapid and accurate technique for studying PGCCs is essential. In this pilot study, we have developed a high-throughput single-cell morphological analysis pipeline capable of precisely quantifying PGCC numbers and proportions. The pipeline accurately quantifies PGCCs based on nuclear and cellular features, including size and DNA content. Leveraging this platform, we screened a library of 2,726 FDA-approved compounds to identify agents that effectively kill PGCCs. Hits from this screen were validated in 3D spheroid cultures, revealing compounds with potent anti-PGCC activity. Complementary transcriptomic analysis (RNA-Seq) of flow-sorted PGCCs versus non-PGCCs unveiled profound dysregulation of the FOXM1 signaling axis, implicating this pathway as a key regulator of PGCC formation and survival. Consistent with this, the high-throughput screen identified Thiostrepton, a known FOXM1 inhibitor, as a selective anti-PGCC agent. Across multiple ovarian cancer cell lines, FOXM1 inhibitors outperformed standard-of-care agents (Paclitaxel, Cisplatin, Olaparib) in eliminating PGCCs. siRNA knockdown of FOXM1 significantly reduced PGCC numbers, particularly under Paclitaxel stress. Moreover, FOXM1 inhibition triggered apoptosis in PGCCs, an effect reversible with the pan-caspase inhibitor Z-VAD-FMK. Together, these findings underscore PGCCs as a therapeutically actionable cell state driving resistance in ovarian cancer. Our integrative single-cell morphological and transcriptomic platform enables rapid discovery of anti-PGCC agents and establishes FOXM1 inhibition as a compelling strategy to eradicate this resilient subpopulation. This pipeline offers broad utility for overcoming therapy resistance across diverse malignancies.



 Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen. Combating ovarian cancer resistance via polyploid cancer cell targeted screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62f27ef6b0280aa103e9f9c96579bbfa985df28c" target='_blank'>
              Abstract A023: Combating ovarian cancer resistance via polyploid cancer cell targeted screening
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSC) typically originates in the Fallopian tube epithelium (FTE) and is characterized by genomic instability and near-universal TP53 defects. HGSC is typically diagnosed at an advanced stage and rapid progression once it becomes invasive. To date, immunotherapies have had limited efficacy in HGSC. Most current HGSC research focuses on tumor and immune profiling of fully fledged HGSC, where in the tumor microenvironment (TME) has already been established. Much less is known about the microenvironment of pre-neoplastic lesions and the role of the immune system in controlling HGSC emergence. Hence, there is an urgent need for in-depth research on the molecular interactions between precancerous cells and immune cells to better understand the process of malignant transformation and escape from host immunity.



 By using our genetically defined, syngeneic organoid platform, FTE organoids were engineered to Trp53 -/- ,Ccne1-overexpressing FTE organoids (“PC” organoids) and Brca1 +/- ;Trp53 -/- (PBrca1) and Brca2 -/- ;Trp53 -/- (PBrca2) to model Homologous recombination (HR)-deficient (HRD) and HR-Proficient (HRP) HGSC, respectively. Pre-tumor growth was monitored using an in vivo imaging system (IVIS) following luciferase-labeled precancerous cell ovarian bursal injection. In addition, adaptive transfer and antibody deletion were performed to determine which immune cell type(s) mediate immune surveillance.



 In a previous study, our laboratory found that PC organoids are not tumorigenic in wild-type (WT) C57BL6 mice. In this study, I found that PC cells were eliminated around 1 month after bursal injection in WT mice. Surprisingly, however, they kept growing and eventually formed tumors in Rag2 -/- mice, which lack B and T lymphocytes. Then, T cells were adaptive transfer from WT mice bearing PC cells to Rag2 -/- mice. I observed a marked reduction in PC cell signal as early as 24 hours post-transfer, which suggests T cells play a critical role in the clearance of precancerous cells. I also found compared to PC cell, PBrca1 and PBrca2 FTE organoids, both of which are non-tumorigenic, are eliminated much earlier (in 24 hours) and was completed in rapidly (∼15 days) following bursal injection into both WT and Rag2 -/- mice. Following NK cells deletion in Rag2 -/- mice, I observed that PBrca1 cells successfully established growth, whereas PBrca2 cells exhibited significantly delayed clearance compared to the IgG control group. Additionally, compared to PC cells, the expression of mouse MHC-Ib Qa-1b, which inhibits NK cell–mediated cytotoxicity, was markedly reduced in PBrca1/2 cells. Although other explanations are possible, these data raise the possibility that PBrca1/2 cells elicit a strong NK cell mediated innate immune response.



 This study represents a novel and highly feasible approach to interrogating the pre-cancer TME. My findings demonstrate that HRD (PBrca1/2) and HRP (PC) trigger markedly divergent immune surveillance mechanisms during the pre-malignant phase.



 Jianmei Hou, Benjamin Neel. Organoid models reveal distinct mechanisms of pre-tumor immune surveillance in high grade serous ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/263093496afb42b25c3ac1b6f0bdab9faf970516" target='_blank'>
              Abstract PR003: Organoid models reveal distinct mechanisms of pre-tumor immune surveillance in high grade serous ovarian cancer
              </a>
            </td>
          <td>
            Jianmei Hou, Benjamin G Neel
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the deadliest diseases, with a 5-year survival rate of only 13%. While rare mutations have been identified to drive PDAC therapy resistance, undiscovered potent, post-transcriptional regulatory mechanisms could be key to understanding this resistance. The RNA-binding protein HuR (ELAVL1) is known to play a critical role in PDAC oncogenesis by stabilizing mRNA transcripts involved in key oncogenic pathways. Our lab has shown that HuR is overexpressed in PDAC tumor and promotes multiple pro-survival signaling pathways. To study HuR-mediated PDAC progression in immune competent models that reflect the tumor microenvironment, we deleted ELAVL1 via CRISPR-Cas9 in Kras-p53 mutant-driven (KPC) murine PDAC cells and found that HuR-knockout (KO) KPC tumors grew more slowly compared to KPC wild-type (WT) tumors in orthotopic implantation settings. Using flow cytometry and multiplex immunohistochemistry, we found that HuR-WT tumors had significantly less T cell infiltration and greater endothelial cell infiltration compared to HuR-KO tumors. Deep transcriptomic and proteomic profiling revealed that HuR mediates the expression of multiple metabolic pathways and determines mRNA and protein cargo loading in secreted extracellular vesicles (EVs). In fact, when mice bearing HuR-KO tumors were injected with purified EVs from HuR-WT cells, both tumor size and endothelial cell function in leukocyte trafficking were rescued. In the context of T cell function, HuR directly enriched transcripts in the mTOR pathway, and inhibition of mTOR activity in HuR-WT tumor cells rescued T cell function in co-culture. Consistent with these observations linking tumor-intrinsic HuR to dysregulated endothelial and anti-tumor immune function, HuR-KO tumors were significantly more responsive to immune checkpoint blockade therapy as compared to WT-HuR tumors. Similarly, in a Kras-MYC driven mouse model, HuR-WT tumor cells metastasized at significantly higher rates to the liver compared to HuR-KO tumors. Finally, given the recent advancements in KRAS inhibitors, we tested their efficacy in these pre-clinical models, and surprisingly, we observed that HuR-KO tumors were up to >100-fold more resistant to KRAS inhibition than WT tumors. These observations could be explained by HuR expression and activity being closely linked to aberrant KRAS signaling, where loss of HuR leads to the reliance of compensatory pathways and drives PDAC cells away from a KRAS addiction phenotype, thus providing a setting of KRAS inhibitor resistance. Accordingly, surviving PDAC cells post-KRAS inhibitor treatment in vivo lost HuR expression compared to untreated tumors. Together, these findings highlight that HuR is a global post-transcriptional regulator that: 1) reshapes the tumor microenvironment to be immune evasive; 2) is involved in a feed-forward loop enhancing KRAS pathway activation; and 3) facilitates PDAC metastatic potential. Strategies to target HuR activity and use it as a potential biomarker could lead to improved therapeutic outcomes.



 Miffy Guo, Jennifer Finan, Hen Halamish, Katherine Pelz, Robert Eil, Rosalie Sears, Jonathan Brody. HuR drives immune evasion, metastases and KRAS addiction in pancreatic ductal adenocarcinoma: implications for IO-based and KRAS inhibitor therapies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9ebd5486262a2dcbf20ec9a786ef9c644e158e4" target='_blank'>
              Abstract A027: HuR drives immune evasion, metastases and KRAS addiction in pancreatic ductal adenocarcinoma: implications for IO-based and KRAS inhibitor therapies
              </a>
            </td>
          <td>
            Miffy Guo, Jennifer M. Finan, Hen Halamish, Katherine R. Pelz, Robert Eil, Rosalie C. Sears, Jonathan R. Brody
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a treatment-resistant subtype that arises through lineage plasticity, allowing tumor cells to bypass androgen receptor (AR)–targeted therapies. In a recent study, Lu et al. define a transcriptional and epigenetic hierarchy that drives this transdifferentiation. The pioneer factor FOXA2 initiates enhancer remodeling and regional DNA demethylation, while the neural lineage transcription factor NKX2-1 is required to complete the NE program. Together, these factors reorganize 3D chromatin architecture and activate lineage-specific genes through enhancer–promoter looping. Crucially, the histone acetyltransferase p300/CBP is essential cofactor in this process. Pharmacologic inhibition of p300/CBP with CCS1477 suppresses NE gene expression and impairs tumor growth in NEPC models. These findings offer a mechanistic insight of lineage plasticity and highlight p300/CBP as promising therapeutic targets. The study also raises key questions about the stability and reversibility of chromatin remodeling and sets a framework for understanding enhancer-driven plasticity in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3331955821f5eab1873b6edce226793e7651773a" target='_blank'>
              FOXA2 and NKX2-1 Orchestrate Neuroendocrine Lineage Plasticity in Prostate Cancer
              </a>
            </td>
          <td>
            Ka-wing Fong
          </td>
          <td>2025-10-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Intrahepatic cholangiocarcinoma (ICC) is a primary malignant tumor of the liver, second only to hepatocellular carcinoma. The scarcity of appropriate cell lines has impeded ICC-related research. This study aims to establish a novel patient-derived ICC cell line and systematically evaluate its biological characteristics, thereby providing a valuable cellular tool for ICC research. Methods A cell line named LPHC-6 of ICC was established using a patient-derived xenograft (PDX) tumor model. We comprehensively characterized this novel cell line through a series of techniques, including cell morphology observation, short tandem repeat (STR) analysis, karyotype analysis, cell proliferation, cell migration assays, western blot analysis, immunofluorescence staining, flow cytometry, gene mutation detection, spheroid formation assays, drug sensitivity testing, and xenograft tumor formation in nude mice. Results LPHC-6 cells exhibited typical characteristics of malignant epithelial cells, demonstrating robust proliferation and migration capabilities. These cells possessed a structurally complex triploid karyotype, which was confirmed to be free from cross-contamination with other human cell lines. Tumor biomarker analysis revealed positive expression of AFP and negative expression of GPC3. Additionally, LPHC-6 cells showed high expression levels of stem cell markers CD133 and CD44. The biliary epithelial cell marker CK19 was positive, whereas the hepatocyte marker Hep Par1 was negative. Genomic analysis identified an exon mutation in TP53, copy number loss in PTEN and RAD51, and copy number gain in NTRK1, PDCD1LG2, and VEGFA. Investigations confirmed that LPHC-6 cells exhibited excellent sphere formation ability and tumorigenic potential. Furthermore, LPHC-6 cells demonstrated sensitivity to Sorafenib, Lenvatinib, 5-fluorouracil, Oxaliplatin and Atezolizumab in both 2D and 3D culture system. Conclusion A novel intrahepatic cholangiocarcinoma cell line LPHC-6 has been successfully established, which provides a powerful cell tool for the research related to intrahepatic cholangiocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c7811061b86fee1a20c5c32421efca379f2d35f" target='_blank'>
              Establishment and characteristic analysis of a novel patient derived cell line of intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Shuang Dou, Minghui Gao, Quanwei Li, Mengyin Chai, B. Kou, Xiaoni Liu
          </td>
          <td>2025-10-15</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cytotoxic chemotherapy remains the standard-of-care treatment for patients with pancreatic ductal adenocarcinoma (PDAC). However, chemotherapy only has modest effects at improving patient survival due to primary or rapidly acquired chemoresistance. The biological underpinnings of PDAC therapy resistance are incompletely defined, but the tumor microenvironment is known to be a major contributor to chemoresistance. We have found chemoresistance is imprinted on PDAC cells by the tumor microenvironment and persists for a period of days after PDAC cells are removed from tumors. However, PDAC chemoresistance is lost upon long term culture in standard laboratory conditions. Interestingly, culture of PDAC cells in Tumor Interstitial Fluid Medium (TIFM), a culture medium we developed to recapitulate the nutrient availability of the tumor microenvironment, maintains PDAC cells in a chemo- and targeted therapy resistant state even after long term culture ex vivo. These findings suggest that microenvironmental metabolic stress keeps PDAC cells in a physiologically relevant, therapy resistant cell state that standard culture models fail to maintain. Using TIFM culture, we sought to understand how PDAC cells in this state resist therapeutic challenge. We found that chemo- and targeted therapies largely retain on-target activity within TIFM medium but fail to activate cell death, enabling a “chemotolerant” cell state, which is also observed in PDAC tumors. This chemotolerant state is driven by suppression of apoptotic priming and can be overcome by targeting the anti-apoptotic regulator BCL-XL. Taken together, these findings suggest that reprogramming of cell death mechanisms by the PDAC nutrient microenvironment is a key contributor to therapy resistance in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bea129494e792fbbf0176ec1d7ce93ff6acb889" target='_blank'>
              Tumor nutrient stress gives rise to a drug tolerant cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Colin Sheehan, Lyndon Hu, Guillaume Cognet, Grace Croley, Thao Trang Nguyen, Anika Thomas-Toth, Darby Agovino, Patrick B. Jonker, Mumina Sadullozoda, Leah M. Ziolkowski, James K. Martin, Alica K Beutel, Ranya Dano, Mohammed A. Khan, Christopher J Halbrook, Kay F. Macleod, Christopher R. Weber, James L LaBelle, Alexander Muir
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of chronic inflammation-associated cancers (CIACs) is shaped by cycles of injury and maladaptive repair, yet the principles organizing fibrotic stroma in these tumors remain unclear. Here, we applied the concept of hot versus cold fibrosis, originally credentialed in non-cancerous fibrosis of heart and kidney, to lung squamous cell carcinoma (LUSC), a prototypical CIAC. Single-cell transcriptomics of matched tumor and adjacent-normal tissue from 16 treatment-naive LUSC patients identified a cold fibrotic architecture in the LUSC TME: cancer-associated fibroblasts (CAFs) expanded and adopted myofibroblast and stress-response states, while macrophages were depleted. This macrophage-poor, CAF-rich stroma was maintained by CAF autocrine growth factor loops, including TIMP1, INHBA, TGFB1, and GMFB. In parallel, the immune compartment exhibited a hot tumor phenotype with abundant T and B cells, forming spatially distinct but molecularly engaged networks with CAFs. CAF gene programs typifying cold fibrosis in LUSC were conserved in other CIACs, including esophageal and gastric adenocarcinomas. These results redefine desmoplastic regions of tumors through the lens of a non-cancer fibrosis model, demonstrating that conserved stromal circuits constitute therapeutic vulnerabilities in CIACs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ed36e6ea62839a2c887c4f2b23f5bf7ed74c7d" target='_blank'>
              A fibroblast-centric network drives cold fibrosis in the tumor microenvironment of lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Shoval Miyara, Shachaf Frenkel, Avi Mayo, Philippe Gascard, Michael K. Strasser, David Gibbs, Eviatar Weizman, Itay Ben Shalom, Yan Stein, Deng Pan, Joseph A. Caruso, V. Sangwan, Nicholas Bertos, Julie Bérubé, Sophie Camilleri-Broet, Spyridon Oikonomopoulos, H. Djambazian, K. Umansky, Jacob Elkahal, Shimrit Mayer, Pierre Fiset, Jiannis Ragoussis, Miri Adler, Eldad Tzahor, Sui Huang, Lorenzo Ferri, T. Tlsty, Ruth Scherz-Shouval, U. Alon
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d43ce8f81cf0d90a92d73349eebd57717f9bf249" target='_blank'>
              Decoding adenomyosis pathogenesis using an assembloid model.
              </a>
            </td>
          <td>
            Yiliang Xu, Tao Cheng, Jianzhang Wang, Zhiruo Qiu, Xiaohong Guan, Yayuan Yu, Jiacheng Shen, Fangfang Xu, Xiaohong Jiang, Dandan Bai, Mingzhu Wang, Shuyan Mei, Hong Wang, Xiaocui Xu, Li Wu, Shaorong Gao, Xuan Che
          </td>
          <td>2025-08-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f06d8c3c1ce1c0e2c42d69d2aefb0ff5769089e" target='_blank'>
              STAT3 sustains tumorigenicity following mutant KRAS ablation.
              </a>
            </td>
          <td>
            S. D'Amico, V. Kirillov, Jingxuan Liu, Zhijuan Qiu, Xinyuan Lei, Hong Qin, Brian S. Sheridan, Nancy C. Reich
          </td>
          <td>2025-08-26</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Diffuse midline gliomas (DMGs) are universally fatal pediatric brain tumors with a median survival of just 11 months. Despite radiation therapy offering temporary benefit, there are no effective long-term treatments, and targeted therapies have shown minimal impact. The infiltrative nature of DMGs, combined with their critical brainstem location, underscores the urgent need to identify novel mechanisms driving tumor progression and resistance. Cellular senescence is a state of permanent cell-cycle arrest induced by stress and has traditionally been viewed as tumor-suppressive. However, recent evidence in adult cancers implicates senescent cells as active contributors to tumor growth, treatment resistance, and relapse through the secretion of pro-tumorigenic factors known as the senescence-associated secretory phenotype (SASP). While senescence has emerged as a hallmark of cancer in adults, its role in pediatric tumors such as DMGs remains largely unexplored. Using single-cell RNA sequencing of treatment-naïve DMG specimens, we identified rare populations of senescent glioma cells and microglia that would be undetectable using conventional approaches. Computational analyses revealed predicted paracrine interactions between senescent and non-senescent cells including pathways previously linked to tumor invasiveness and resistance. This study aims to define how senescent cells and their secretory programs shape the tumor microenvironment, promote DMG progression, and contribute to therapy resistance. By elucidating these mechanisms, we hope to uncover new therapeutic vulnerabilities in DMGs and expand our broader understanding of senescence as a driver of malignancy in pediatric cancers.



 Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya M. Rangaswamy, Claudia Kleinman, Pratiti Bandopadhayay. Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785f69efbe0f5fccc4e2b6c0500f58472565b0dc" target='_blank'>
              Abstract B040: Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance
              </a>
            </td>
          <td>
            Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya Rangaswamy, Claudia L Kleinman, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Glioblastoma multiforme represents the most common primary malignant brain tumor in adults, with a 90% 2-year recurrence rate despite standard therapy of surgical resection followed by chemo- and radiotherapy. Yet, the cellular mechanisms underlying therapeutic resistance remain poorly understood. By integrating analysis of single-cell RNA sequencing results of 41 primary and 43 recurrent tumor specimens with experimental validation, we identify neuro-oligodendrocyte precursors as the predominant cellular population driving tumor recurrence through establishment of protective microenvironmental niches. We discover that CTNNA3 and MBP exhibit paradoxical roles in promoting survival in primary tumors while conferring chemoresistance in recurrent GBM, revealing that these factors determine treatment efficacy rather than directly driving relapse. Functional assays confirmed their role as mediators of extracellular matrix remodeling, establishing a mechanistic link between tumor microenvironmental remodeling and therapeutic resistance. By revealing how recurrent tumors engineer self-protective niches, our study provides a new framework for developing combination therapies that disrupt these resistance mechanisms and provides foundation for overcoming treatment failure in this universally fatal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec11a04512f1a7cb7a16592e9fd8fa7748da070" target='_blank'>
              CTNNA3 and MBP Mediate Treatment Resistance Through Tumor Microenvironment Remodeling in Recurrent Glioblastoma
              </a>
            </td>
          <td>
            Wenna Mao
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intratumoral microbiota, a critical component of the tumor microenvironment, notably impacts tumor progression through various complex mechanisms such as metabolic regulation, immune system remodeling and genotoxicity. The present review focuses on eight prevalent solid tumors (breast, colorectal, lung, pancreatic, gastric, ovarian, prostate cancers and melanoma), detailing the intratumoral microbial compositional heterogeneity within these malignancies. The present review analyzes the heterogeneous carcinogenic mechanisms mediated by these microorganisms, including gene instability, immune microenvironment remodeling and metabolic intervention. The diagnostic value of microbial markers in liquid biopsy and in situ tissue detection is explored, and the potential for synergistic strategies combining microbial-targeted therapy and immunotherapy is discussed. Finally, the review suggests future research directions, such as spatiotemporal dynamic analysis and organoid-microorganism co-culture, offering new insights for precise cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09bce21033da16d36e409d313fbe6c3d34814bec" target='_blank'>
              Tumor-associated microbiota: Multi-cancer landscape, mechanistic insights and clinical translation (Review)
              </a>
            </td>
          <td>
            Jindi Yang, Yige Dong, Yi Chen, Hong Liang, Shengyu Rong, Zhe Liu, Qiulei Lang
          </td>
          <td>2025-09-16</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75865e4fe743320d35269c6c38da63e48d3392a1" target='_blank'>
              Lineage plasticity and histological transformation: tumor histology as a spectrum.
              </a>
            </td>
          <td>
            Xiaoling Li, Eric E. Gardner, Sonia Molina-Pinelo, Clare Wilhelm, Ping Mu, Á. Quintanal-Villalonga
          </td>
          <td>2025-09-30</td>
          <td>Cell research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Advancements in chemotherapy have improved outcomes for rhabdomyosarcoma (RMS), the most common soft tissue sarcoma in children. However, relapse-free survival rates remain below 20% in metastatic alveolar rhabdomyosarcoma (ARMS). Tumor-initiating cells (TICs) drive recurrence. Targeting this pool of cells has been shown to reduce relapse rates and metastasis in various cancers. Emerging evidence implicates lipid metabolism in supporting stem-like properties in cancer. However, TICs remain poorly characterized in ARMS. This study investigates the transcriptomic and metabolic profile of TICs in ARMS. Methods Transcriptomic and lipidomic profiling were performed on tumorsphere-derived and adherent cells from ARMS cells. Differential expression of metabolic genes and lipid species was assessed via RNA-Sequencing and mass spectrometry. Functional dependence on Phospholipase A2 (PLA2) signaling was tested on TICs in vitro and in vivo. Seahorse assays were used to measure glycolytic and mitochondrial flux. Statistical significance was determined using Student’s t-test or one-way ANOVA. Results Transcriptomic analysis of ARMS tumorspheres showed upregulation of lipid metabolism especially different PLA2 enzyme subtypes, and suppression of glycolytic gene expression. Lipidomic analyses revealed enrichment of linoleic acid-derived triglycerides and lysophospholipids, paralleled by increased PLA2 activity. Tumorspheres also showed elevated expression of lipid storage regulators PPARG and CD36 and correspondingly, significant lipid droplet accumulation. Inhibition of PLA2 with Darapladib reduced tumorsphere formation and cellular motility in vitro and tumor volume in vivo. These defects were rescued by exogenous linoleic acid (LA) supplementation, implicating PLA2-driven lipid signalling in TIC maintenance. Interestingly, TICs display high metabolic flexibility in energy consumption, as neither glycolysis nor fatty acid oxidation (FAO) inhibition significantly impaired tumorsphere formation. Conclusions PLA2-mediated lipid remodelling supports TIC by promoting linoleic acid–linked lipid signalling. PLA2 inhibition impairs self-renewal, motility, and tumorigenic potential, highlighting it as a promising therapeutic target in ARMS. These findings uncover lipid remodelling as a key metabolic adaptation in ARMS TICs, independent of canonical energy pathways, with implications for targeting the stem-like compartment in pediatric sarcomas. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02409-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfe186d79f283fafcded05829d4a4d4c46ef02bf" target='_blank'>
              PLA2 driven lipid signaling drives ARMS tumorigenic cell properties
              </a>
            </td>
          <td>
            Amogh Gupta, Bharathi Ramanathan, Dipanwita Das, Aiswariya Vadivellu, A. Loh, R. Taneja
          </td>
          <td>2025-10-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive solid malignancy with poor overall prognosis and limited response to standard treatments. Growing interest in the modulation of DNA repair mechanisms, including the homologous recombination (HR) repair pathway, has opened new avenues for therapeutic development. BARD1 (BRCA1-Associated RING Domain 1) plays a complex role in tumor biology, functioning either as a tumor suppressor or as an oncogenic driver, depending on isoform expression, cellular context, and regulatory environment. In this review, we examine the dual roles of BARD1, focusing on its regulation and paradoxical activities in PDAC. We summarize evidence that BARD1 and BARD1 isoforms differentially affect DNA repair, apoptosis, and drug resistance and evaluate the therapeutic potential of targeting BARD1 and other DNA damage response (DDR) proteins. We also review ongoing clinical trials and investigational agents designed to exploit DDR vulnerabilities in PDAC. Together, these insights highlight BARD1’s context-dependent roles in PDAC and support continued efforts to profile BARD1 isoforms, clarify their functions, and leverage DDR pathways through precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c070f9d1ec5a160a1dcacf90475e9772e358bc" target='_blank'>
              BARD1: A Friend or Foe in Pancreatic Ductal Adenocarcinoma?
              </a>
            </td>
          <td>
            Lily Zekavat, Aditi Jain
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Compared to sun-exposed melanomas, acral melanomas are genetically diverse and occur in areas with low sun exposure and high mechanical loads. During metastatic growth, melanomas invade from the epidermis to the dermis layers through dense tumor stroma and are exposed to fibrillar collagen architectures and mechanical stresses. However, the role of these signals during acral melanoma pathogenesis is not well understood. In this study, a novel 3D in vitro platform comprising heterogeneous, bundled collagen architectures recapitulates mechanical and architectural signals from the melanoma tumor microenvironment. YUSEEP patient-derived human acral melanoma and B16F10 mouse melanoma single cells and spheroids are embedded in collagen or bundled collagen hydrogels and mechanically compressed to quantitatively profile cellular responses to these cues, including viability, DNA damage and repair, proliferation, invasion, and nuclear and cellular morphologies. Spatial confinement of cells in a microfluidic platform, solid mechanics simulations, and pharmacological inhibition studies lend further mechanistic insights into these cues. Results reveal mechanical compression induces DNA damage and repair, while interactions with bundled collagen promote a malignant, protrusive phenotype. The findings further suggest that actin polymerization and contractility inhibitors may rescue DNA damage and mitigate malignancy upon compression, thereby potentially paving the way for novel therapeutic targets against acral melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a50277034a8bc6fee3365fef5db114e494ef3ed6" target='_blank'>
              Impact of Mechanical and Architectural Signals in the Tumor Microenvironment on Melanoma.
              </a>
            </td>
          <td>
            Z. M. Khan, Alejandro Rossello-Martinez, Michael Mak
          </td>
          <td>2025-09-07</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Surrogate selection is an experimental design that without sequencing any DNA can restrict a sample of cells to those carrying certain genomic mutations. In immunological disease studies, this design may provide a relatively easy approach to enrich a lymphocyte sample with cells relevant to the disease response because the emergence of neutral mutations associates with the proliferation history of clonal subpopulations. A statistical analysis of clonotype sizes provides a structured, quantitative perspective on this useful property of surrogate selection. Our model specification couples within-clonotype birth-death processes with an exchangeable model across clonotypes. Beyond enrichment questions about the surrogate selection design, our framework enables a study of sampling properties of elementary sample diversity statistics; it also points to new statistics that may usefully measure the burden of somatic genomic alterations associated with clonal expansion. We examine statistical properties of immunological samples governed by the coupled model specification, and we illustrate calculations in surrogate selection studies of melanoma and in single-cell genomic studies of T cell repertoires.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db201d19838bab76eccd97ad0f817ca6a6bc83d" target='_blank'>
              SURROGATE SELECTION OVERSAMPLES EXPANDED T CELL CLONOTYPES.
              </a>
            </td>
          <td>
            Peng Yu, Yumin Lian, Elliot Xie, Cindy L. Zuleger, Richard J Albertini, Mark R Albertini, Michael A. Newton
          </td>
          <td>2025-09-01</td>
          <td>The annals of applied statistics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 To identify novel drivers of pancreatic ductal adenocarcinoma (PDAC) malignancy, we employed a systems biology approach called regulatory network analysis, which accurately infers the activity of transcription factors and other regulatory proteins based on the integrated expression of their positive and negative target genes. This highly validated approach enables the identification of the most hyper-activated and hyper-repressed regulatory proteins (“master regulators”) that drive phenotypic distinctions. We applied this technique to a set of 200 laser capture microdissected human PDAC samples as well as 45 low-grade precursors for which we had matched histopathological, clinical, and epidemiological annotation. We identified the master regulators associated with 4 malignancy phenotypes: tumor initiation, tumor progression, survival post resection, and association with KRAS activity. Integrating across these phenotypes, the top master regulator was BMAL2, a member of the PAS family. Although BMAL2 is classically associated to the circadian rhythm, gene set enrichment analysis highlighted a potential role in hypoxia response, a key phenotype in PDAC evolution. We previously demonstrated that PDAC in humans and in the genetically engineered “KPC” mouse model is hypovascularized and hypoperfused, and we have since shown that KPC tumors are profoundly hypoxic. Sequence conservation analysis of BMAL2 demonstrated close analogy to hypoxia-related genes, its activity is induced in response to it and inhibited following treatment with multiple RAF, MEK, and ERK inhibitors, validating its computationally inferred association with RAS activity. Strikingly, knockout of BMAL2 in human PDAC cells led to defects in viability and invasion in the setting of hypoxia. Metabolically, loss of BMAL2 impairs the ability to induce glycolysis - key metabolic pathway unregulated during hypoxia response - upon exposure to severe hypoxia. Surprisingly, knockout of BMAL2 in multiple PDAC cell lines led to a complete loss of HIF1A stabilization in response to hypoxia. By contrast, HIF2A was further upregulated under hypoxia in the setting BMAL2 loss. Additionally, metabolomic studies showed an overrepresentation of glycolysis metabolites and related metabolic pathways in hypoxic control conditions but to a lesser degree in BMAL2 knockout cells. Finally, preliminary in vivo experiments in cell line derived xenograft models displayed lower tumor growth rates in BMAL2 knock out cells. Together, our data indicates that BMAL2 loss blunts hypoxia response providing also a potential answer to the long-standing question of how these proteins are differentially regulated. Certainly, a connection between RAS signaling and glycolysis has long been apparent, but our findings provide a novel mechanistic understanding of the relationship between RAS and hypoxic metabolism.



 Alvaro Curiel Garcia, Carlo H. Maurer, Sam Holmstrom, Cristina Castillo, Pasquale Laise, Carmine F. Palermo, Steven A. Sastra, Anthony Andren, Li Zhang, Tessa Le Large, Irina Sagalovsky, Daniel R. Ross, Vilma Rosario, Kate Lu, Ethan Ferraiuolo, Nicholas Spinosa, Winston Wong, Kaitlin Shaw, John A. Chabot, Jeanine Genkinger, Hanina Hibshoosh, Gulam A. Manji, Alina C. Iuga, Roland M. Schmid Schmid, Michael A. Badgley, Kristen C. Johnson, Andrea Califano, Yatrik M. Shah, Costas A. Lyssiotis, Kenneth P. Olive. Ras-dependent activation of BMAL2 regulates hypoxic metabolism in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A047.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b200421d854e96ea48e923482c3749c3e10676" target='_blank'>
              Abstract A047: Ras-dependent activation of BMAL2 regulates hypoxic metabolism in pancreatic cancer
              </a>
            </td>
          <td>
            Á. C. García, Carlo Maurer, Sam Holmstrom, Cristina Castillo, Pasquale Laise, Carmine F. Palermo, Stephen A. Sastra, A. Andren, Li Zhang, Tessa Y. Le Large, Irina Sagalovsky, Daniel R. Ross, V. Rosario, K. Lu, Ethan Ferraiuolo, Nicholas Spinosa, W. Wong, K. Shaw, John A. Chabot, Jeanine Genkinger, H. Hibshoosh, G. Manji, Alina C Iuga, Roland M. Schmid Schmid, Michael A. Badgley, Kristen C. Johnson, Andrea Califano, Yatrik M. Shah, C. Lyssiotis, K. Olive
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Tumor-infiltrating lymphocyte (TIL) therapy is a promising approach, earning U.S. Food and Drug Administration approval in patients with anti-PD-1-resistant melanoma. Extending TIL therapy to patients with cholangiocarcinoma (CCA), an aggressive and largely immune-refractory cancer, is an emerging area of interest. However, cost and manufacturing complexity constrain clinical scalability of TIL therapy for CCA, underscoring the need for a murine model to optimize efficacy. Here, we established a novel orthotopic model of TIL therapy for CCA and tested a new ex vivo expansion strategy. We first characterized the immune landscape of orthotopic CCA and then compared 2 TIL expansion methods: (1) a conventional protocol using CD3 agonist stimulation (CD3 TILs) and (2) a tumor antigen-based protocol using irradiated autologous CCA cells to enrich for tumor-reactive TILs (Tumor Ag TILs). Tumor Ag TILs displayed superior tumor lysis in vitro compared to CD3 TILs. While both TIL products engrafted in vivo, Tumor Ag TILs showed enhanced persistence. Despite this, monotherapy with either TIL product alone had only a modest impact on tumor growth rate, and infused cells had upregulation of inhibitory checkpoint receptors, including PD-1. Further investigations demonstrated that the in vivo antitumor efficacy of both Tumor Ag TILs and CD3 TILs was enhanced when combined with PD-L1 inhibitor therapy. Altogether, our study establishes a preclinical platform for modeling CCA TIL therapy, identifies a rational combination strategy that potentiates TIL efficacy, and provides the field with a foundation to advance adoptive T-cell transfer development for CCA and related solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c94988fa5b70c12c198d5f39ec050ac02e9b35b8" target='_blank'>
              Development of a murine tumor-infiltrating lymphocyte therapy model for cholangiocarcinoma.
              </a>
            </td>
          <td>
            Megen C. Wittling, Frances J Bennett, Emilie A K Warren, K. M. Oppat, M. Wyatt, Jacklyn Hammons, Yuan Liu, Shishir K Maithel, Chrystal M. Paulos, Gregory B. Lesinski
          </td>
          <td>2025-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Background Metastatic melanoma is a highly aggressive disease with poor survival rates despite recent therapeutic advancements with immunotherapy. The proteomic landscape of advanced melanoma remains poorly understood, especially regarding proteomic heterogeneity across metastases within patients. Methods We collected 83 melanoma metastases from 19 different metastatic sites in 24 patients with advanced metastatic melanoma almost exclusively from the pre‐immunotherapy era, using semi‐rapid autopsies. The metastases were subjected to histopathological evaluation, RNA‐sequencing and mass spectrometry‐based proteomics for protein quantitation and non‐reference peptide (NRP) sequence detection using a proteogenomic data integration approach. Results NRPs associated with mutations frequently occurred in proteins related to focal adhesion, vesicle‐mediated transport, MAPK signalling and immune response pathways across the cohort. Intrapatient heterogeneity was negligible when considering morphology and driver gene mutation status but was substantial at the proteogenomic level. This heterogeneity was not driven by metastasis location, albeit liver metastases exhibited distinct proteogenomic patterns, including upregulation of metabolic pathways. Cluster analysis outlined four proteomic clusters (C1–4) of the metastases, characterised by the upregulation of cell cycle and RNA‐splicing (C1), mitochondrial processes (C3), extracellular matrix (ECM) and immune pathways (C2) and ECM and vesicle‐mediated transport pathways (C4). Around two‐thirds of patients had metastases that had strongly distinct phenotypes. Patients in our cohort whose metastases were primarily assigned to clusters C1 and C3 exhibited shorter overall survival than patients whose metastases were categorised mainly into the C2 and C4 clusters. Conclusion Our unique multi‐metastasis cohort captured the proteogenomic heterogeneity of immunotherapy‐naïve melanoma distant metastases, establishing a foundation for future studies aimed at identifying novel therapeutic targets to complement current immunotherapies. Key points Comprehensive proteogenomic profiling of post‐mortem melanoma metastases, collected primarily before the immunotherapy era. Description of 1177 protein sequence variants predicted by RNA‐Seq and validated via mass spectrometry‐based proteomics. Empirical evidence of prominent intrapatient heterogeneity, driven by heterogeneous protein expression related to cell cycle‐ and mitochondrial processes, immune system and extracellular matrix organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c47896dcec6e305a1baa3208d22fb00cf053050b" target='_blank'>
              Distant metastases of melanoma exhibit varying extent of intrapatient proteogenomic heterogeneity
              </a>
            </td>
          <td>
            B. Szeitz, Y. Hagemeijer, Zoltán G Páhi, Zsuzsanna Ujfaludi, M. Kuras, Jimmy Rodriguez, V. Doma, R. Mohácsi, Magdolna Herold, Zoltán Herold, Z. Horváth, I. Pla, Y. Sugihara, Bo Baldetorp, H. Lindberg, Henriett Oskolás, M. Rezeli, Jeovanis Gil, R. Appelqvist, L. Kemény, Jéssica Guedes, Johan Malm, Aniel Sanchez, Imre M Boros, I. Németh, Victor Guryev, Tibor Pankotai, Krzysztof Pawłowski, E. Wieslander, A. M. Szasz, David Fenyö, Peter Horvatovich, József Timár, Gyorgy Marko-Varga, L. Betancourt
          </td>
          <td>2025-09-28</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Confident identification of human leukocyte antigen (HLA)-presented peptides is crucial for advancing cancer immunotherapy. We present MHCquant2, a scalable and modular Nextflow pipeline integrated into nf-core as a reproducible, portable, and standardized workflow for immunopeptidomics. This integration allows a community-driven, robust solution for high-throughput analyses across operating systems and cloud infrastructures. MHCquant2 integrates open-source tools including OpenMS, DeepLC, and MS2PIP, improving peptide identifications by up to 27% across diverse MS platforms, particularly enriching low-abundant peptides. MHCquant2 demonstrates state-of-the-art performance on our novel benignMHCquant2 dataset (n = 92) and expands the benign human immunopeptidome by over 160,000 unique naturally presented HLA peptides. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03763-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e678695db6815730d0674dcf746e931d4d6d4de2" target='_blank'>
              MHCquant2 refines immunopeptidomics tumor antigen discovery
              </a>
            </td>
          <td>
            Jonas Scheid, Steffen Lemke, Naomi Hoenisch-Gravel, Anna Dengler, Timo Sachsenberg, Arthur Declerq, R. Gabriels, J. Bauer, Marcel Wacker, Leon Bichmann, Lennart Martens, M. Dubbelaar, Sven Nahnsen, J. Walz
          </td>
          <td>2025-09-22</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="


 Tumor-associated macrophages (TAMs) represent the main immune cell population in various brain malignancies, but their role in the meningioma (MGM) microenvironment remains to be elucidated.



 Here, we investigated TAM frequencies, activation state, survival-associated changes, and their association with tumor-infiltrating T-lymphocytes (TILs) in two independent study samples comprising altogether 680 MGMs. To this end, we performed tissue cytometry analyses, quantified tissue cytokine levels, and integrated previously-published TIL infiltration and microarray datasets in the discovery cohort (n=195 clinically well-annotated cases). This was complemented by a DNA methylation-based deconvolution approach to predict TAM and TIL infiltration rates and survival associations in an independent validation cohort of n=485 MGMs.



 Our findings revealed substantial but heterogeneous TAM infiltration in newly-diagnosed MGMs, with increased numbers of anti-inflammatory TAMs in clinically-aggressive tumors. Cytokine and transcriptome analyses corroborated the presence of an immunosuppressive niche in TAM-enriched MGMs. Importantly, high TAM infiltration was identified as an independent prognostic factor for poor survival, counteracting the beneficial prognostic effect of TILs. Furthermore, deconvolution analyses confirmed the opposing prognostic roles of TAMs and TILs in the validation cohort.



 Methylation-based deconvolution to explore the prognostic impact of TAM and TIL composition in menigiomas may serve as a potential biomarker for immunotherapeutic treatment strategies. Furthermore, our findings highlight pro-tumoral TAMs to be an attractive treatment target in MGMs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b541053e2ec078b9d9af0d16f05da69f5740e693" target='_blank'>
              OS13.5.A INTEGRATIVE MULTIOMICS ANALYSIS IDENTIFIES TUMOR-ASSOCIATED MACROPHAGES AS NOVEL INDEPENDENT BIOMARKER FOR POOR SURVIVAL IN MENINGIOMAS
              </a>
            </td>
          <td>
            C. Lotsch, R. Warta, F. Liu, G. Jungwirth, C. Rommel, K. Lamszus, A. Kessler, M. Löhr, R. Ketter, C. Senft, N. Maas, P. Sievers, M. Westphal, S. Krieg, A. Unterberg, M. Simon, A. von Deimling, F. Sahm, D. Raleigh, C. Herold-Mende
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Background Cellular senescence plays a significant role in tumorigenesis and tumor progression. Substantial evidence indicates that senescence occurs in cancer-associated fibroblasts (CAFs), the predominant stromal component within the tumor microenvironment (TME), which profoundly impacts tumor biology. However, despite growing evidence of stromal cell involvement in cancer progression, the specific mechanisms and clinical implications of senescent CAFs (SCAFs) in hepatocellular carcinoma (HCC) have not been fully elucidated. Methods The senescence signature was utilized to evaluate the senescence status of cell types within the TME of HCC using the GSE149614 dataset. The CytoTRACE and cell-cell communication analysis were used to find the correlation between cancer stemness and SCAFs. A risk prediction model associated with SCAFs was constructed to investigate potential mechanisms by which SCAFs promote tumor progression. Single-cell RNA sequencing data was used to identify senescent CAF-related genes. Gene expression and clinical data for HCC were obtained from the Cancer Genome Atlas (TCGA), International Cancer Genome Consortium (ICGC), and National Omics Data Encyclopedia (NODE) databases. Using four machine-learning algorithms, crucial genes were identified to develop a CAF-senescence-related risk model, predicting prognosis, cancer stemness, immune infiltration, tumor mutation burden, and therapeutic responses in HCC patients. Next, we explored the role of Collagen Triple Helix Repeat Containing-1 (CTHRC1) in cancer stemness using both in vitro and in vivo experiments. Through various functional experiments, we elucidated the downstream signaling pathways of CTHRC1. Additionally, chromatin immunoprecipitation experiments were used to verify that key transcription factors bind to the CTHRC1 promoter region. Results CAFs exhibited high senescence status and a strong correlation with cancer stemness in HCC. A novel CAF-senescence-score (CSscore) prognostic model was established for HCC based on 10 genes: CTHRC1, SERPINE1, RNF11, ENG, MARCKSL1, ASAP1, FHL3, LAMB1, CD151, and OLFML2B. The survival prediction performance was validated on TCGA, ICGC, and NODE cohorts. Immune analysis revealed that the CSscore was positively correlated with immunosuppressive immune cell populations, including M2 macrophages and regulatory T cells. Conversely, a negative correlation was observed between the CSscore and anti-tumor immune cells such as CD8 + T cells, dendritic cells, and B cells HCC patients with a low CSscore had a lower tumor mutation burden and showed improved responsiveness to immunotherapy and transarterial chemoembolization. In vitro experiments and bioinformatics analysis further revealed that CTHRC1 was significantly elevated in SCAFs promoted cancer stemness and metastasis via the SRY-box transcription factor 4 (SOX4)-CTHRC1-Notch1 axis in HCC. Conclusion Our study revealed that SCAFs were strongly correlated with cancer stemness in HCC. A novel machine learning model based on senescent CAF-related genes was constructed to accurately predict prognosis in HCC patients. Furthermore, CTHRC1 was identified as a novel prognostic and therapeutic biomarker to predict poor prognosis in HCC and promote cancer stemness and metastasis through the Notch signaling pathway, with its expression being transcriptionally regulated by SOX4. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02369-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff7c6095fcfbd5cb01c70f78e80c25fa1a454693" target='_blank'>
              Senescent fibroblasts secrete CTHRC1 to promote cancer stemness in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hai Huang, Wang Peng, Qiaodan Zhou, Yuchong Zhao, Luyao Liu, H. Cui, Jingwen Liang, Mengdie Cao, Wei Chen, Ronghua Wang, Shiru Chen, S. Xiong, Bin Cheng, Shuya Bai
          </td>
          <td>2025-08-25</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Brain metastases (BM) represent a signiﬁcant unmet clinical need with poor survival and limited drug thera- pies. Recent evidence suggests a subset of BM have a more locally invasive phenotype. We sought to determine whether the tumour leading edge and tumour microenvironment (TME) differed between BM of different phe- notypes using spatial transcriptomics.



 Image guided surgical sampling of the tumour margin in operated BM cases using a previously described method (“Geo-Tagging”). Tumour microarray from FFPE blocks assembled using 2mm dots in triplicate. Spatial transcriptomics analysis using Nanostring GeoMX platform and immune-oncology panel (86 markers). Mart1 (melanoma BM) or CK7 (breast and lung adenocarcinoma BM) was used to delineate the tumour, GFAP for per- itumoral brain (TME) and CD45 for immune compartment. Annotation of tissue with clinical data for overall survival, local recurrence, primary cancer and assessment of BM leading edge using established criteria – dif- fuse or border.



 22 cases were analyzed in triplicate, 5 breast cancer, 8 metastatic melanoma and 9 lung adenocarcinoma. Data was analysed using established bioinformatics pipelines in R. As expected there were signiﬁcant differences in gene expression between all BM of different primary cancers but not in the portion of T cells, macrophages and NK cells. PDL1 was upregulated in the tumour compartment and HLA-E downregulated in the TME com- partment of BM without a clear border. Locally recurrent BM signiﬁcantly overexpressed a number of genes previously established as important in BM, including for adhesion (ICAM1, EPCAM), microglia-tumour interac- tion (STAT3) and local invasion (CCND1, VEGFA).



 BM have different phenotypes that are dependent on both tumour and local microenvironmental factors with the future potential for therapeutic targeting to modify tumour behaviour and reduce local invasion and recur- rence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef6992d8a9fcf5e2f41d2333a8c8118644a30147" target='_blank'>
              INVESTIGATING THE TUMOUR MICROENVIRONMENT FOR BRAIN METASTASES USING SPATIAL TRANSCRIPTOMICS AND SURGICALLY GUIDED SAMPLING
              </a>
            </td>
          <td>
            R. Zakaria, Michael Jenkinson, Ian Scott, Frances Greaney Davies, Michael Chvanov, Jamie Soul
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4006665d3be664d3d5517dbddb8bef8da7462da" target='_blank'>
              High-Resolution Spatial Transcriptomics Reveals Fibroblast and Neuroimmune Microenvironments in Endometriosis Lesions.
              </a>
            </td>
          <td>
            Caroline M Haney, Elaheh Alizadeh, Meryl Sullivan, Joshua Lee, Jasmina Kuljancic, W. Flynn, Paul Robson, Brian S. White, Danielle E Luciano, Elise T. Courtois
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Polymerase epsilon mutant (POLE-mut) endometrial cancers (EC) are characterized by a near 100% disease-specific survival rate, even when treated by surgery alone. This spectacular survival, combined with the ultramutated genome and high level of neoantigens in these tumors, indicates a substantial degree of immune control in preventing disease spread and recurrence. Although these features are intriguing, the immune infiltration of POLE-mut EC has predominantly been confined to immunohistochemistry studies. Here, we used state of the art single-cell RNA and TCR sequencing to characterize the immune landscape of POLE-mutant ECs. Moreover, we uniquely analyzed patient blood samples taken two to eight years after curative treatment to assess formation of long-term immune memory in circulation. We identified specialized tumor-infiltrating myeloid subsets at different stages of maturation, an array of lymphocytes ranging from immature to cytotoxic and adaptive natural killer (NK) as well as tumor-reactive exhausted and effector T cells, contributing to a highly inflammatory anti-tumor response. Remarkably, our analysis of blood samples taken years after curative treatment uncovered the presence of tumor-reactive T cell clones that matched the primary tumor. This indicates the formation of systemic long-term memory immune responses in POLE-mut EC survivors. Our study highlights the distinctive immunogenicity of POLE-mut EC and identifies key features associated with persistent anti-tumor immunity that may contribute to prolonged, relapse-free survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0198b259b8d5d685a1ba64c108955b1306a25438" target='_blank'>
              Spatiotemporal immune landscape and long-term immune memory in POLE-mutant endometrial cancer at the single-cell level.
              </a>
            </td>
          <td>
            K. Brummel, M. Requesens, N. van Rooij, H. Workel, F. Eggink, A. Plat, R. Wardenaar, D. Spierings, F. Foijer, David N. Church, J. Bart, H. Nijman, M. de Bruyn
          </td>
          <td>2025-08-29</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most frequent and malignant primary brain tumor, is characterized by a highly diverse and profoundly immunosuppressive tumor microenvironment (TME) that provides an unconstrained environment for tumor progression and significantly complicates therapeutic interventions. Despite advances in immunotherapeutic approaches, such as chimeric antigen receptor T cell and immune checkpoint inhibitors, efficacy remains limited due to the complexity of the GBM TME and robust immune evasion mechanisms. In this review, we elucidate the intricate interplay among cellular components within the TME that lead to this immunosuppressive state, including tumor-associated macrophages/microglia, myeloid-derived suppressor cells, regulatory T cells, and glioma stem cells, as well as other critical elements that contribute to TME complexity, such as the severe hypoxia associated with central necrosis, the blood-brain barrier, and the extracellular matrix. This review also highlights mechanisms of immune evasion and recent immunotherapeutic approaches along with their biologic rationale, underscoring the need for integrated therapeutic strategies that both target immunosuppressive elements and enhance immune activation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7b85891090b88db119cdf1eaf07337b4a452f03" target='_blank'>
              Immunopathology of Glioblastoma.
              </a>
            </td>
          <td>
            Jiabo Li, James L. Ross, D. Hambardzumyan, Daniel J Brat
          </td>
          <td>2025-10-08</td>
          <td>Annual review of pathology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c516594ae0175b41dbabb65d57d662cb8d6406" target='_blank'>
              Unveiling the role of the extracellular matrix in the osteosarcoma tumor microenvironment through integrated transcriptomics and experimental validation.
              </a>
            </td>
          <td>
            Yuyang Liu, Yuchen Han, Zixuan Guo, Yinglong Zhang, Xiuyuan Xu, Wenting Qi, Meng Xu, Jianxiong Li
          </td>
          <td>2025-10-07</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="We developed a computational pathology pipeline to extract and analyze collagen disorder architecture (CoDA) features from whole slide images (WSIs) of 2,212 colon cancer (CC) patients across multiple institutions. CoDA features—capturing collagen fragmentation, bundling, anisotropy, density, and rigidity, were evaluated for associations with clinical variables (overall stage, T/N/M stage), molecular classifications (Consensus Molecular Subtypes [CMS1–4]), and genetic mutations (KRAS, BRAF, NRAS) using the Mann-Whitney U test with Bonferroni correction. These analyses revealed significant differences in CoDA feature distributions across multiple subgroups, suggesting that collagen architecture varies meaningfully with tumor stage, molecular subtype, and mutation status.To assess how well CoDA features could distinguish between these subgroups, we implemented a Random Forest classification framework. High mean AUC values (≥0.7) across several variables indicated strong discriminatory performance of CoDA features in separating clinically and biologically distinct groups. For survival analysis, LASSO-Cox models were trained on the PLCO dataset to generate CoDA-based risk scores for overall survival (OS) and disease-free survival (DFS), which were used to stratify patients into high- and low-risk groups in a combined validation dataset (TCGA, UH, and Emory). Kaplan-Meier curves demonstrated significant survival differences across clinical stages, CMS subtypes, and KRAS mutation status. Multivariable Cox proportional hazards models further confirmed the independent prognostic value of CoDA features after adjusting for clinical, molecular, and genetic covariates. These findings highlight that CoDA features are significantly associated with key clinical and molecular characteristics, can distinguish relevant patient subgroups, and offer independent prognostic information, underscoring their potential utility in characterizing the tumor microenvironment and informing risk stratification in CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04db6c4e2bf678fd2c4ee0d19ebb24dcd6405bce" target='_blank'>
              Collagen disorder architecture features are associated with clinical, molecular, genetic factors and survival outcomes in colon cancer
              </a>
            </td>
          <td>
            R. Nag, Chuheng Chen, Haider Mejbel, Haojia Li, Aya Aqeel, Pingfu Fu, G. Corredor, S. Khalighi, Tilak Pathak, M. Mokhtari, Michelle D. Reid, Alyssa M. Krasinskas, K. Pandav, Joseph E. Willis, A. Madabhushi
          </td>
          <td>2025-08-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a696fa9baeadfe94d2a764547f5853c287c52816" target='_blank'>
              Hedgehog signaling drives glial cell plasticity and oncogenic reprogramming in gastroenteropancreatic neuroendocrine neoplasms.
              </a>
            </td>
          <td>
            Suzann Duan, AnneLeigh Twer, Ateh Zinkeng, Ricky A. Sontz, Juanita L. Merchant
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 RAS inhibitors (RASi) have the potential to change the clinical landscape for pancreatic ductal adenocarcinoma (PDAC) and improve patient survival. Nevertheless, clinical resistance is likely to reduce the long-term benefits of RASi, and combinations that can bypass or overcome resistance will be needed. Here, we studied the dynamics of resistance to the RAS multi(ON) inhibitor RMC-6236 (daraxonrasib). Prior to frank drug resistance, PDAC cells persist in a drug tolerant state characterized by reduced proliferation; however, only a fraction of the drug-tolerant persister cells contributes to resistance. To identify and study the relevant, resistance-contributing cells, we developed a highly multiplexed, lentiviral DNA barcode system that is compatible with single-cell technologies, enabling paired lineage tracing and phenotypic profiling throughout RASi treatment. Our lineage tracing studies suggest that drug persistence and resistance are fueled by a subset of cells that are primed to survive RASi treatment. Ongoing work is directed at defining and tracking the cell states associated with each stage of resistance emergence.



 Jayne C. McDevitt, Connor J. Hennessey, Robert Vander Velde, Sydney M. Shaffer, Ben Z. Stanger. Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B082.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c1bd513ed6b1024a532086d606e43f01e09bb7" target='_blank'>
              Abstract B082: Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition
              </a>
            </td>
          <td>
            Jayne C McDevitt, Connor J Hennessey, R. V. Velde, Sydney M. Shaffer, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 PDAC is defined by a dense, immunosuppressive TME composed of ECM proteins, CAFs, and diverse immune cells. ChIP-seq on resected human PDAC samples identified MICAL2 as a super-enhancer-associated gene and its high expression was corelated with poor survival of patients. MICAL2, a flavin monooxygenase, promotes actin depolymerization and SRF transcription. This study investigates how tumor-intrinsic MICAL2 alters the PDAC TME. Orthotopic implantation of KPC-MICAL2 knockdown (M2KD) inducible cells led to reduced tumor growth. Immunofluorescence, flow cytometry, and atomic force microscopy showed M2KD tumors were more cellular, with marked reduction in PDPN+/α-SMA+ CAFs, collagen, and fibronectin, resulting in reduced stiffness. MICAL2 loss also reprogrammed CAF gene expression and increased infiltration of activated CD8+ T cells, skewing from an M2 to M1 macrophage phenotype. scRNA-seq revealed major shifts in immune populations, including increased cycling T and plasma cells. CD8+ T cell depletion reversed tumor suppression in both immune-hot and immune-cold syngeneic models, and adoptive transfer of MICAL2-KD tumor-infiltrating T cells suppressed the growth of control tumors. RNAScope showed reduced IL-1α, IL-6, TGF-β expression and decreased p38 and STAT3 phosphorylation in MICAL2-KD tumors. To test if MICAL2 regulates PDAC TME transcriptionally, we generated KPCM tumors and lines (by crossing PDX1-Cre; LSL-KRAS G12D/+; P53R172H/WT (KPC) with Mf/f mice) and reintroduced WT MICAL2 and MICAL2 lacking nuclear localization signal (DNLS) or FAD enzymatic domain in KPCM line. DNLS and FAD point mutant MICAL2 cells were deficient in tumor growth, had reduced stromal deposition and SRF activity, and increased CD8+ T cells. Anti-PD-1 significantly reduced M2KD tumor size and 50% of M2KD tumor-bearing mice had complete regression with anti-PD-1 and IL-1α antibodies. MICAL2-ASO treatment in human PDAC ex-vivo slice cultures increased T cell activation and M1 polarization. MICAL2 drives immunosuppression in PDAC and could be a potential therapeutic target.



 Bharti Garg, Evangeline Sari. Mose, Edgar Esparza, Jay Patel, Kevin Gulay Gulay, Sarah Sass, Alexei Martsinkovskiy, Asmina Courelli, Carrie Bishop, Gisselle Gonzalez, Adam Engler, Parag Katira, Vivien Ileana . Maltez, Herve Tiriac, Andrew M. Lowy. Actin cytoskeleton dynamics in tumor cells mediate Immune-suppressive microenvironment and sensitize pancreas tumors to PD-1 blockade therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a79cb674dc0d1eba8719f4642ee7a1f1f6ec638" target='_blank'>
              Abstract A109: Actin cytoskeleton dynamics in tumor cells mediate Immune-suppressive microenvironment and sensitize pancreas tumors to PD-1 blockade therapy
              </a>
            </td>
          <td>
            Bharti Garg, Evangeline S Mose, Edgar Esparza, Jay Patel, Kevin Gulay Gulay, Sarah Sass, A. Martsinkovskiy, Asmina Courelli, Carrie Bishop, Gisselle Gonzalez, Adam Engler, Parag Katira, Vivien Ileana . Maltez, Hervé Tiriac, A. Lowy
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="


 Chemoresistance remains a major barrier to improving outcomes in pancreatic ductal adenocarcinoma (PDAC), particularly in response to frontline gemcitabine and nab-paclitaxel (GnP) therapy. We aimed to identify transcriptional drivers of acquired GnP resistance and evaluate their therapeutic potential in PDAC. Although GnP is a standard-of-care regimen, most preclinical models rely on monotherapy-based systems, limiting insight into combination-specific resistance mechanisms. To address this, we developed GnP-resistant (GPR) cell lines by continuously culturing six syngeneic mouse PDAC lines that span the epithelial-to-mesenchymal transition (EMT) spectrum using a clinically relevant 1:8 paclitaxel-to-gemcitabine ratio. Two lines served as EMT controls with EMT-loss via Snail1/Twist1 knockout or EMT-gain via E-cadherin knockout. The remaining four unmodified KPC lines exhibited stable, spontaneous EMT phenotypes. Parental GPR cell lines >10- to 30-fold increase in IC50 as compared to their parental counterparts. Stable resistance was established through serial passaging under continuous exposure to consistent drug concentrations of gemcitabine paclitaxel. To assess whether resistance persisted in vivo, GPR and parental cells were orthotopically implanted into the pancreas of immunocompetent C57BL/6 mice. Upon palpable tumor formation, mice were treated with gemcitabine and nab-paclitaxel twice weekly. GnP significantly extended survival in mice bearing parental tumors (median >60 days), whereas those with GPR-derived tumors showed reduced survival (25–30 days), like vehicle-treated controls, demonstrating that acquired resistance negated GnP efficacy in vivo. To uncover molecular mechanisms of resistance, we performed bulk RNA sequencing on all six matched parental-GPR pairs. Transcriptomic analysis revealed shared changes in gene expression across GPR lines, including activation of proliferative pathways and factors involved in transcriptional regulation and chromatin remodeling. These resistance-associated programs were further validated in GnP-treated tumors by immunohistochemistry and RNA sequencing. Analysis of patient datasets showed that elevated expression of chromatin remodeling factors was associated with poor prognosis in PDAC, underscoring their potential clinical relevance. Future studies will focus on defining chromatin-based mechanisms by which transcriptional reprogramming supports resistance to combination chemotherapy and identifying targetable epigenetic vulnerabilities in treatment-refractory PDAC.



 Meet Patel, Richard Laube, Ayushi Mandloi, Jace Baines, Robert S. Welner, Julienne L. Carstens. Transcriptional and Chromatin Remodeling Programs Underlie Resistance to Gemcitabine and Nab-Paclitaxel in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A062.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35935c1852c231b8da0c50cb3825fb6d98d5f58" target='_blank'>
              Abstract A062: Transcriptional and Chromatin Remodeling Programs Underlie Resistance to Gemcitabine and Nab-Paclitaxel in Pancreatic Cancer
              </a>
            </td>
          <td>
            Meet Patel, Richard Laube, Ayushi Mandloi, Jace Baines, Robert S. Welner, Julienne L. Carstens
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="that effectively predicts MMR status through a multi-regional combined approach. By integrating spatial heterogeneity within the radiomic model with transcriptomic data, we elucidated key biological differences between patients with dMMR vs . pMMR across sub-regions. Furthermore, the characteristic functions, genes, and immune cell types identified in each sub-region provide valuable insights into the molecular and cellular bases of GC heterogeneity">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b40f26074a634f846c9e94bcd6a3112c9d16dae1" target='_blank'>
              Radiomic model for preoperative prediction of mismatch repair deficiency in gastric cancer: a multicenter study integrating tumor sub-region radiomics and transcriptomics
              </a>
            </td>
          <td>
            Siwei Pan, Enze Li, Guoliang Zheng, Yi You, Yanqiang Zhang, Mengxuan Cao, Ruolan Zhang, Qing Yang, Yizhou Wei, Weiwei Zhu, Ke Shen, Chencui Huang, Jingxu Xu, Lijing Wang, Zaisheng Ye, Zhiyuan Xu, Can Hu
          </td>
          <td>2025-09-08</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Tunneling nanotubes (TNTs) are dynamic membranous protrusions that are critical for intercellular communication in cancer, yet their transient nature and lack of definitive molecular markers pose challenges for systematic study. To address this issue gap in knowledge, we used bulk and single-cell transcriptomic sequencing approaches to identify differentially expressed genes in ovarian cancer cells enriched for TNTs to identify molecular pathways and genes potentially implicated in TNT formation, maintenance and function. We used edgeR and Seurat to perform differential expression analysis on ovarian cancer cells grown in conditions favoring upregulation of TNT formation and further performed single-cell sequencing of the same cell type isolated via spatiotemporal genomic analysis (SaGA) and fluorescence-activated cell sorting (FACS). We found 771, 32 and 279 genes upregulated in bulk sequenced TNT forming SKOV3, COV362 and ES-2 ovarian carcinoma cell lines, respectively. Gene families that repeatedly came up across all three of the cell lines included carbonic anhydrases, matrix metalloproteinases, SERPIN (Serine Protease Inhibitor-30), and SNORA (Small Nucleolar RNAs). We narrowed down candidate genes of interest to 23 differentially expressed genes (DEGs), many of which are involved in cellular stress response, organization of the extracellular matrix or intracellular cytoskeleton, cell adhesion, and other pathways. Further analyzing the single cell sequenced ES-2 cells, we found the differential expression of KISS1 gene, a metastasis suppressor implicated in cytoskeletal organization, significantly upregulated in TNT-enriched cells, highlighting its potential role in TNT formation. Other genes, such as EPHB1, MMP3, and WNT7B, were also upregulated in TNT-enriched cells, suggesting their involvement in regulating cell adhesion, migration, and differentiation. Conversely, we observed downregulation of BGN, a gene associated with promoting metastatic potential and NFkB signaling in cancer cells, as well as SYNGR1, a gene traditionally linked to synaptic transmission but also involved in vesicle exocytosis, which may influence TNT formation.



 Rojina Sapkota. RNA-seq analysis to uncover drivers of cancer cells forming tunneling nanotubes for intercellular communication [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb5389ea72947e09856895a8de975c87a724ed10" target='_blank'>
              Abstract B015: RNA-seq analysis to uncover drivers of cancer cells forming tunneling nanotubes for intercellular communication
              </a>
            </td>
          <td>
            Rojina Sapkota
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Breast cancer is a complex disease influenced not only by genes but also by the way cancer cells generate and use energy. One rare change is the loss of a gene called fumarate hydratase (FH), which normally helps cells produce energy efficiently. We examine how this change influences how tumors grow and respond to treatment. By studying thousands of breast cancer samples, we found that tumors missing FH tend to create a special environment that helps them grow and survive, especially by encouraging new blood vessel formation. We also describe a patient with this FH change who had a remarkable and long-lasting response to a therapy that blocks blood vessel growth (known as anti-VEGF treatment). These findings suggest that identifying this rare energy-related alteration may help clinicians determine which patients are most likely to benefit from this therapy. Such insights could contribute to advancing more personalized approaches to breast cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c214b499120e8d0fc3e3396e45d4d3eb6e6c018" target='_blank'>
              Revealing the Angiogenic Signature of FH-Deficient Breast Cancer: Genomic Profiling and Clinical Implications
              </a>
            </td>
          <td>
            L. A. Sinberger, Noa Keren-Khadmy, Assaf Goldberg, T. Peretz-Yablonski, A. Sonnenblick, Mali Salmon-Divon
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Significant group-based variations in prostate cancer (PCa) outcomes represents a critical healthcare challenge in the USA. African American (AA) men face the highest incidence of PCa and are twice as likely to succumb to their disease compared to their Caucasian American (CA) counterparts. This disparity stems from a complex interplay involving social, environmental, and biological factors. While inequitable access to quality healthcare is a key contributor, even in settings with equal care, AA patients more often present with advanced disease at diagnosis, suggesting unique tumor biology. While some understanding of the epigenomic landscape has been recently identified, the full extent of global chromatin remodeling contribution towards differences in health outcomes across population groups remains poorly understood. To better elucidate how chromatin remodeling and gene regulation influences PCa development across CA and AA population groups, we employed single-cell spatial transposase-accessible chromatin technology followed by sequencing (scATAC-seq) and H3K27ac spatial-CUT&Tag using samples from men who underwent prostatectomy at the Center for Prostate Disease Research/Walter-Reed National Military Medical Center. Our current analysis revealed significantly greater chromatin accessibility in AA men compared to CA men. Leveraging a machine learning-based computational cell typing method called Cellcano to annotate scATACseq data, enabled the identification of unique cell types and distinct gene score signatures enriched in AA samples compared to CA counterparts. Furthermore, AA demonstrated significant lower accessible chromatin associated with CCCTC-binding factor (CTCF) which is a critical zinc finger protein crucial for regulating gene expression, epigenetic marks, and maintaining 3-dimensional genome structure. We have validated this finding by CTCF immunohistochemistry in an independent TMA that includes 99 patients with distribution of CA and AA prostatectomy samples. Moreover, integration of H3K27ac profiling revealed several differentially regulated enhancer regions near specific genes across samples. Collectively, our current findings provide critical insights into the divergent chromatin accessibility profiles between AA and CA men with localized PCa, which likely underlie distinct transcriptional responses that can determine therapeutic resistance and tumor progression. These data shed light on the complex epigenomic mechanisms driving the group-based variations in PCa outcomes and may inform the development of personalized treatment strategies to address this critical healthcare challenge.



 Beatriz German Falcon, Kun-Lin Ho, Tri Truong, Jennifer Garbarino, Jiaying Chen, Amina Ali, William Azadze, Sally Elsamanoudi, Colin Ng, Gregory Chesnut, Leigh Ellis. Analysis of the chromatin accessibility landscape in patients with localized prostate cancer [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C087.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f758d1a482aa1ee0457d64a79f2bbfa79c84e92e" target='_blank'>
              Abstract C087: Analysis of the chromatin accessibility landscape in patients with localized prostate cancer
              </a>
            </td>
          <td>
            Beatriz German Falcon, Kun-Lin Ho, Tri Truong, Jennifer Garbarino, Jiaying Chen, Amina Ali, William Azadze, S. Elsamanoudi, Colin Ng, Gregory T. Chesnut, Leigh Ellis
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lysosome-dependent cell death (LDCD) is an emerging form of regulated cell death with critical implications in tumor development, immune modulation, and therapy responsiveness. However, the role of LDCD-related genes in colon adenocarcinoma (COAD) remains poorly understood. We comprehensively analyzed LDCD-related gene expression profiles using transcriptomic data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus databases (GEO). Unsupervised clustering was performed to identify molecular subtypes. A prognostic signature was developed using LASSO and Cox regression analyses. Immune infiltration characteristics and immunotherapy responses were assessed via multiple algorithms. Single-cell RNA sequencing (scRNA-seq) analysis was conducted to explore the cellular distribution of LDCD genes. Functional assays, including colony formation, Transwell migration, and western blotting, were performed to validate the role of key LDCD regulators in COAD cell line. LDCD-related genes were differentially expressed in COAD tissues and stratified patients into two distinct subtypes with significant differences in survival, immune infiltration, and biological pathways. A five-gene prognostic signature demonstrated robust predictive ability for overall survival and was associated with tumor immune microenvironment characteristics. High-risk patients showed distinct immune checkpoint profiles and predicted sensitivity to immune checkpoint blockade therapy. scRNA-seq analysis revealed cell type-specific expression patterns of LDCD genes. In vitro experiments confirmed that modulation of a key LDCD regulator, SLC11A1, affected the proliferation and invasiveness of COAD cells. Moreover, the impact of SLC11A1 on apoptosis and intracellular ROS level confirmed the potential association between SLC11A1 and COAD cell apoptosis. Our study reveals the prognostic value and immunological relevance of LDCD-related genes in COAD. These findings provide new insights into molecular classification and may aid in the development of personalized therapeutic strategies targeting LDCD pathways in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18267-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9901f6ba7f419afbbf7d8a4c7ff078253e88b294" target='_blank'>
              Comprehensive characterization of lysosome-dependent cell death reveals prognostic significance and immune landscape in colon adenocarcinoma
              </a>
            </td>
          <td>
            Zhenhua Liu, Lianming Cai, B. Liao, Baowen Wang, T. Xia, Xiangxin Wu
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is the age-related expansion of mutated hematopoietic stem cells without other hematologic abnormalities. In patients with solid tumors, CH is associated with higher mortality and may evolve to therapy-related myeloid neoplasms; however, the mechanisms by which cancer treatments promote CH dynamics remain largely unknown. Here, we analyzed 392 serial samples from a prospective cohort of breast cancer patients and showed that cytotoxic treatments (chemotherapy {+/-} radiation) led to strong therapeutic bottlenecks, resulting in significant reductions in hematopoietic allelic populations and differential clonal selection. CH clones that were positively selected and expanded through dose-dependent therapeutic bottlenecks frequently harbored mutations in TP53, PPM1D, SRCAP, DNMT3A, and YLPM1. Patients with positively selected CH during treatment had the shortest progression-free and overall survival compared to patients with unchanging or negatively selected CH across all therapies. These findings, corroborated in independent breast cancer and pan-cancer cohorts, provide strong evidence for clinical relevance of changes in CH during cancer treatment which may help identify patients at high risk for inferior outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a553a5184b2aa0287d71bedac04551bf6a5da6" target='_blank'>
              Clonal Hematopoiesis Dynamics and Evolutionary Fitness During Cancer Treatment Impact Clinical Outcomes
              </a>
            </td>
          <td>
            Mona Arabzadeh, Yi-Han Tang, Christelle M. Colin-Leitzinger, S. Marzban, Daniel Walgenbach, Stefania Morganti, Vaidhyanathan Mahaganapathy, Erika Harper, Mingxiang Teng, Jacob Kresovich, Iman Washington, H. Parsons, Judy E. Garber, J. West, Shridar Ganesan, Hossein Khiabanian, Nancy Gillis
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d81b6af91799905ae613971966f2b549960f979" target='_blank'>
              The tumour microenvironment in pancreatic cancer - new clinical challenges, but more opportunities.
              </a>
            </td>
          <td>
            Heng-Chung Kung, Kevin W Zheng, Jacquelyn W. Zimmerman, Lei Zheng
          </td>
          <td>2025-10-03</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mutations in numerous genes contribute to human cancer, with different oncogenic lesions prevalent in different cancer types. However, the malignant phenotype is simple, characterized by unrestricted cell growth, invasion, and often metastasis. One possible hypothesis explaining this dichotomy is that cancer genes regulate common targets, which then function as master regulators of essential cancer phenotypes. To identify mechanisms that drive the most fundamental feature shared by all tumors-unrestricted cell proliferation-we used a multiomic approach, which identified translation and ribosome biogenesis as common targets of major oncogenic pathways across cancer types. Proteomic analysis of tumors and functional studies of cell cultures established nucleolar and coiled-body phosphoprotein 1 as a key node, whose convergent regulation, both transcriptionally and posttranslationally, is critical for tumor cell proliferation. Our results indicate that lineage-specific oncogenic pathways regulate the same set of targets for growth control, revealing key downstream nodes that could be targeted for therapy or chemoprevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e15820af1f4a4174736d844aaa4b18b01cd6fd" target='_blank'>
              Diverse oncogenes use common mechanisms to drive growth of major forms of human cancer.
              </a>
            </td>
          <td>
            O. Kauko, Mikko Turunen, Päivi Pihlajamaa, Antti Häkkinen, R. Queiroz, M. Pääkkönen, S. Ventelä, Massimilliano Gaetani, Susanna L. Lundström, Antonio Murgia, Biswajyoti Sahu, J. Routila, Gong-Hong Wei, H. Irjala, J.L. Griffin, Kathryn S. Lilley, Teemu Kivioja, S. Hautaniemi, J. Taipale
          </td>
          <td>2025-08-22</td>
          <td>Science advances</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="SUMMARY Spatiotemporal environmental variation results in marked heterogeneity in bacterial infection progression and disease outcome, with vital consequences for treatment success. For the globally important pathogen Mycobacterium tuberculosis (Mtb), while the pronounced intra-host spatial heterogeneity in lesion immune cell composition and phenotype has been well-described, the highly complex Mtb cell envelope has presented a particular challenge for the required equivalent insight into bacterial heterogeneity. Here, we develop hybridization chain reaction-fluorescence in situ hybridization (HCR-FISH)-based methodology for Mtb mRNA visualization in the context of intact lung and lesion architecture. In combination with a Mtb transcriptional/translational activity reporter, we reveal spatiotemporal differences in gene expression relating to Mtb lipid metabolism, response to key environmental signals, and the ESX-1 type VII secretion system. Our results establish a framework for in situ analysis of Mtb mRNA, opening the path to elucidating critical bacterial drivers that underlie the marked heterogeneity in Mtb-host interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a13c42f93d5cca6138cd9f381352612e75158b1b" target='_blank'>
              Direct visualization of bacterial transcripts in the infected lung illuminates spatiotemporal environmental adaptation of Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Anna-Lisa E. Lawrence, Shumin Tan
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Purpose Pancreatic ductal adenocarcinoma (PDA) remains one of the most lethal malignancies with limited early diagnostic and therapeutic options. Although receptor for activated C kinase 1 (RACK1) is an evolutionarily conserved scaffold protein, its functional role and mechanistic involvement in PDA pathogenesis remain elusive. Methods Using multimodal approaches including: (1) genetically engineered mouse models of pancreatitis and carcinogenesis, (2) patient-derived PDA tissues with matched normal specimens, (3) primary acinar cell 3D cultures, and (4) orthogonal gain/loss-of-function assays in PDA cell lines, we systematically investigated RACK1’s spatiotemporal expression patterns and functional impacts. Mechanistic dissection was performed through gene expression profiling and pathway enrichment analyses with functional validation. Results RACK1 exhibited progressive silencing across pancreatic lesion progression: acinar cells (normal) > ADM > pancreatic intraepithelial neoplasia (PanIN) > PDA. This acinar-specific protein was undetectable in ductal/islet lineages and was further suppressed under inflammatory challenge. Functionally, RACK1 depletion accelerated ADM initiation and enhanced PDA cell motility and metastatic dissemination in vivo, whereas its overexpression exerted tumor-suppressive effects. Mechanistically, caerulein/TGF-α stimulation and KrasG12D activation converged to inhibit RACK1 while activating MAP2K3-SRC-RELA(p65) signaling nodes, establishing a pro-inflammatory feedforward loop. Conclusions RACK1 serves as a gatekeeper restraining inflammation-driven ADM transformation, with its downregulation constituting an early molecular event in PDA pathogenesis. The RACK1-MAP2K3 axis orchestrates malignant transition through simultaneous NF-κB activation (inflammatory priming) and MAPK hyperactivation (proliferative drive). Our findings nominate RACK1 as both a stratification biomarker for high-risk pancreatic lesions and a druggable node for intercepting preneoplastic progression. Supplementary Information The online version contains supplementary material available at 10.1007/s13402-025-01084-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39be333f88be26f22fe55082d0e5477bb0e3c020" target='_blank'>
              RACK1 attenuates pancreatic tumorigenesis by suppressing acinar-to-ductal metaplasia through inflammatory signaling modulation
              </a>
            </td>
          <td>
            Wei Zhang, Tingting Jiang, Huiqing Zhang, Fang Wei, Xiaojia Li, Keping Xie
          </td>
          <td>2025-09-01</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is an aggressive brain tumour conferring poor prognosis and one of the highest recurrence rates despite multimodal treatment. The mechanisms underlying recurrence are not fully understood, but the tumour microenvironment is known to play a key role. Astrocytes, the most abundant glial cells in the brain, have been implicated in supporting tumour progression, yet their metabolic interactions with glioblastoma invasive margin cells remain poorly understood. This study aims to investigate metabolic interactions using single-cell spatial metabolomics to identify potential pathways involved in tumour adaptation and progression.



 The study employed atmospheric pressure matrix-assisted laser desorption/ionisation (AP-MALDI) imaging mass spectrometry at 10 μm spatial resolution to investigate metabolic interactions between primary glioblas- toma invasive margin cells and human cortical astrocytes in a co-culture model designed to mimic post-surgical residual disease.



 Metabolomic analysis revealed distinct metabolic changes in glioblastoma invasive margin (GIN) cells and hu- man cortical astrocytes (HA) under co-culture conditions. In GIN cells, pyroglutamic acid and glutamate levels were altered, along with changes in lipid metabolism, including a reduction in oleamide. In astrocytes, co- culture inﬂuenced nucleotide metabolism, with increased levels of deoxyguanosine, alongside shifts in lipid proﬁles. Findings indicate bidirectional metabolic interactions that may contribute to glioblastoma progres- sion.



 By mapping metabolite spatial distributions at the single-cell level, this study provides novel insights into tumour–astrocyte metabolic interactions at the glioblastoma invasive margin. These ﬁndings highlight poten- tial metabolic dependencies that could be therapeutically targeted to disrupt glioblastoma recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c48c731e4f1e7e12000fcb428faa049465c8fd4b" target='_blank'>
              SINGLE-CELL METABOLIC PROﬁLING OF TUMOUR ASTROCYTE INTERACTIONS WITHIN A CO-CULTURE MODEL OF THE GLIOBLASTOMA INVASIVE MARGIN
              </a>
            </td>
          <td>
            U. Kontrimaite, Kei F Carver Wong, Phoebe McCrorie, Sandra Martinez Jarquin, Stuart Smith, Dong-Hyun Kim, Ruman Rahman
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tissue microenvironment characteristics associated with elevated risk of colorectal cancer (CRC) in Lynch syndrome (LS) are poorly characterized. We applied the multimodal single cell sequencing platform ExCITE-seq to define the colonic cellular composition and transcriptome of LS carriers with and without a history of CRC compared with general population controls. Our analysis revealed widespread remodeling in LS that included striking expansion of epithelial stem and progenitor cells, and loss of fibroblast populations. Although clonally expanded and terminally exhausted CD8 T cells were more prominent in individuals with a history of CRC, LS carriers without CRC displayed enrichment of cytotoxic mucosal-associated invariant T (MAIT) cells associated with CCL20 expression in epithelial progenitors, validated by orthogonal techniques including demonstration of a protective function in a murine model of CRC. These findings highlight cellular features that distinguish LS carriers and suggest a protective role of MAIT cells in human CRC surveillance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79380d7adba5d58e3eea2a58ea5bb81cdc80c34a" target='_blank'>
              MAIT cell enrichment in Lynch syndrome is associated with immune surveillance and colorectal cancer risk
              </a>
            </td>
          <td>
            Ken Cadwell, Hairu Yang, Michaela Dungan, Keely Beyries, Xin Wang, Robert Kilpatrick, Baron Chen, Sangmi Oh, Marissa Berkowitz, David Smith, Sergei B Koralov, Jordan Axelrad, Christopher Lengner, Nicole Belle, Meenakshi Bewtra, Bryson W. Katona
          </td>
          <td>2025-08-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with limited therapeutic options and poor prognosis. For translational research, although patient-derived organoids (PDOs) have emerged as powerful platforms for modeling tumor biology and drug response, conventional tissue-derived organoids are often limited by low cellularity, contamination with non-tumor cells, and the invasiveness of tissue sampling. Here, we report the establishment and characterization of fluid-derived organoids (FDOs) from malignant pleural effusion and ascitic fluid of PDAC patients. Compared to tissue-derived PDOs, FDOs were established more rapidly and less invasively, with higher success rates and reduced contamination by non-tumor cells. FDOs retained key histopathological and molecular features of the primary tumor, including KRAS mutations and epithelial marker expression. Holotomography imaging revealed distinct morphological and subcellular phenotypes of fluid-derived cancer organoids, reflecting their divergence from normal organoids as well as temporal changes associated with treatment response and disease progression. In vivo xenograft assays confirmed the tumorigenic potential of FDOs, particularly those derived from advanced disease. Drug screening demonstrated that FDOs harboring KRAS G12D mutations responded to the KRAS inhibitor MRTX1133. Transcriptomic profiling identified a poor prognosis-associated gene signature that conferred gemcitabine resistance by suppressing apoptosis. Collectively, our findings establish FDOs as clinically relevant, minimally invasive, and scalable models that enable personalized therapy and biomarker discovery in PDAC.



 Hee Seung Lee, Yoo Keung Tae, Sang-Min Kim, Jung-hwan Park, Kyu Min Lim, HyunKi Kim, Jung Hyun Jo. Minimally invasive fluid-derived organoids enable modeling of chemotherapy resistance in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B095.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81dfc735c625271116241a3184d0aee8b2524e42" target='_blank'>
              Abstract B095: Minimally invasive fluid-derived organoids enable modeling of chemotherapy resistance in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Hee Seung Lee, Yoo-Keung Tae, Sang-Min Kim, Jung-hwan Park, Kyu Min Lim, HyunKi Kim, J. Jo
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Lymph nodes (LNs) play a pivotal role in colorectal cancer (CRC) progression and immunity, yet their molecular and functional diversity remains poorly understood. By analyzing 630 LNs and 88 primary tumors from 200 CRC patients across four independent cohorts using bulk and single-cell RNA sequencing, we identify four non-metastatic negative LNs (NLN) subtypes (NLN_C1-C4) exhibiting obviously different immune function and stromal expansion. NLN_C3/C4 are characterized by diminished T and B cell activity and fibroblast-driven fibrosis, with follicular dendritic cell loss contributing to B cell dysfunction. Immune checkpoint inhibitors partially reverse these effects, restoring FDC and B cell activity. LNs subtypes demonstrate heterogeneity across patients and within individuals, with higher NLN_C3/C4 proportions associated with advanced tumor stages, poorer survival, and recurrence. Here, we report LNs subtypes as critical manifestations of LN heterogeneity in CRC, providing a basis for improved clinical stratification and LN-targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7feee3a87f017d97fabbe5f56625b50a9e843d16" target='_blank'>
              Lymph nodes molecular subtypes unravel lymph nodes heterogeneity and clinical implications in colorectal cancer
              </a>
            </td>
          <td>
            Xu Guan, Yueyang Zhang, Ruifang Sun, Guiyu Wang, Xiaoman Bi, Zhen Zhang, L. Zan, Yongsheng Meng, Yun-Xiang Liu, Jinyu Guo, Jian Ma, Jiale Li, Hanqing Hu, Yixun Zhang, Xin Zhang, Maoxi Liu, Bo Jiang, Haiyi Liu, S. Zou, Yanfeng Xi, HAI-MENG Zhou, Wenqi Bai, Deng Wu, Xishan Wang
          </td>
          <td>2025-08-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) remains among the deadliest cancers, with over 80% diagnosed at advanced stages and a 5-year survival rate below 12%. Current treatments are largely ineffective, and even patients who undergo surgery often experience early relapse, highlighting the urgent need for better therapeutic strategies. Approximately 20–40% of PDACs harbor defects in DNA damage response (DDR) pathways, including mutations in ATM. ATM is a central kinase involved in repairing DNA double-strand breaks, and its loss promotes genomic instability and tumorigenesis. However, ATM deficiency also increases tumor sensitivity to DNA-damaging agents. In this context, cells become more dependent on the related DDR kinase ATR, presenting a promising target for synthetic lethality-based therapies. TRA-1-60 (TRA), a key cell-surface antigen involved in the reprogramming of inducible pluripotent stem cells, is overexpressed in PDAC, and correlates with poor survival. Tumor samples derived from our PDAC models showed differential TRA-1-60 expression, with BxPC-3-derived tumors exhibiting high levels and AsPC-1-derived tumors low levels. TRA expression increases before or during metastasis and decreases with cell differentiation, making it a promising biomarker for targeted radiopharmaceutical therapy (RPT) in PDAC. We developed a beta-emitting RPT by attaching 177Lutetium (t1/2 ∼ 6.67 d) to the antibody Bstrongomab, which targets TRA on PDAC cells and induces DNA strand break damage. Considering PDAC’s complex mutational profile, including DNA repair defects in 20–40% of cases, we evaluated TRA RPT alone and its combinations with PARP and ATR inhibitors in ATM- deficient and wild-type (WT) PDAC cell models. The in vitro analysis of PANC1 ATM knockout (KO) and AsPC1 ATM KO cells revealed increased sensitivity to non-targeted [177Lu] LuCl3 treatment compared to WT cells. This suggests that DDR-deficient PDAC cells may be more susceptible to DNA damage induced by the radionuclide. In vivo, treatment of these DDR-deficient models with TRA RPT significantly delayed tumor growth compared to WT, particularly in AsPC1 ATM KO xenografts, despite their low TRA expression. TRA RPT alone (p = 0.001) significantly suppressed tumor progression in the AsPC1 ATM KO model compared to WT xenografts. While the combination of TRA RPT with olaparib (PARPi) (p = 0.00025) also suppressed tumor growth compared to WT, it did not provide a statistically significant benefit over TRA RPT treatment alone. Notably, combining TRA RPT with the ATR inhibitor berzosertib resulted in a pronounced delay in tumor growth in AsPC1 ATM KO models, showing a significant improvement compared to TRA RPT alone. These results highlight the promise of combining TRA RPT with DDR-targeted therapies to improve treatment in DDR-deficient PDAC. Identifying effective drug combinations may be key to better patient outcomes and advancing personalized therapies, especially for cases with DDR mutations.



 Nasrin Movahhedin, Jacob Lindquist, Istri Ndoja, Chris Yunker, Nerissa T. Viola, Steve M. Patrick. A novel theranostic approach targeting ATM-deficient pancreatic ductal adenocarcinoma (PDAC) using 177Lu-labeled anti–TRA-1-60 [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A107.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a007a1a3f187552a4f3aa8416d4fc649bde2601" target='_blank'>
              Abstract A107: A novel theranostic approach targeting ATM-deficient pancreatic ductal adenocarcinoma (PDAC) using 177Lu-labeled anti–TRA-1-60
              </a>
            </td>
          <td>
            N. Movahhedin, Jacob Lindquist, Istri Ndoja, Chris Yunker, Nerissa T. Viola, Steve M. Patrick
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Understanding the cellular origins and early evolutionary dynamics that drive the initiation of carcinogenesis is critical to advancing early detection and prevention strategies. By characterizing key molecular, cellular and niche events at the precancerous tipping point of early gastric cancer (EGC), we aimed to develop more precise screening tools and design targeted interventions to prevent malignant transformation at this stage. We utilized our AI models to integrate spatial multimodal data from nine EGC endoscopic submucosal dissection (ESD) samples (covering sequential stages from normal to cancer), construct a spatial-temporal profile of disease progression, and identify a critical tipping point (PMC_P) characterized by an immune-suppressive microenvironment during early cancer development. At this stage, inflammatory pit mucous cells with stemness (PMC_2) interact with fibroblasts via NAMPT\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\longrightarrow$$\end{document}⟶ITGA5/ITGB1 and with macrophages via AREG\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$$\longrightarrow$$\end{document}⟶EGFR/ERBB2 signaling, fostering cancer initiation. We established gastric precancerous cell lines and organoids to demonstrate that NAMPT and AREG promote cellular proliferation in vitro. Furthermore, in the transgenic CEA-SV40 mouse model, targeting AREG and/or NAMPT disrupted key cell interactions, inhibited the JAK-STAT, MAPK, and NFκB pathways, and reduced PD-L1 expression, which was also confirmed by western blot in vitro. These interventions delayed disease progression, reversed the immunosuppressive microenvironment, and prevented malignant transformation. Clinical validation was conducted using endoscopically resected EGC specimens. Our study provides a precise spatiotemporal depiction of EGC development and identifies novel diagnostic markers and therapeutic targets for early intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0410e9abe73bbef3e06750644cca2f27fd69ccca" target='_blank'>
              Spatiotemporal multi-omics analysis uncovers NAD-dependent immunosuppressive niche triggering early gastric cancer
              </a>
            </td>
          <td>
            Pingting Gao, Chunman Zuo, Wei Yuan, Jiabin Cai, Xiaoqiang Chai, Ruijie Gong, Jia Yu, Lu Yao, Wei Su, Zuqiang Liu, Shengli Lin, Yun Wang, Mingyan Cai, Lili Ma, Quanlin Li, Pinghong Zhou
          </td>
          <td>2025-09-22</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Understanding gene regulatory networks (GRNs) is essential for deciphering biological processes and disease mechanisms. Single-cell multiome technologies now enable joint profiling of chromatin accessibility and gene expression, offering an powerful means to infer cell type-specific GRNs. However, existing methods analyze each cell type independently or aggregate data into pseudo-bulk profiles, limiting their ability to resolve rare populations and capture cellular heterogeneity. We introduce BayesCNet, a Bayesian hierarchical model that jointly infers enhancer-gene linkages across all cell types while leveraging their hierarchical relationships for information sharing. Through extensive simulations, BayesCNet consistently outperforms state-of-the-art methods, with the largest improvements in rare cell types. When applied to real datasets, BayesCNet identifies enhancer-gene linkages with higher accuracy validated by promoter-capture Hi-C data, and reconstructs cell type–specific GRNs that highlight key regulators, demonstrating its power to resolve gene regulatory programs across diverse cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4714272f9648202cbefffa98ee2a6c7e8d8bb3a" target='_blank'>
              BayesCNet: Bayesian inference for cell type-specific regulatory networks leveraging cell type hierarchy in single-cell data
              </a>
            </td>
          <td>
            Fengdi Zhao, Arkaprava Roy, Weijia Jin, Leeana D. Peters, Todd Brusko, Qing Lu, K. Esser, Ramon C. Sun, Li Chen
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) progression is driven by a series of sequential mutations in key driver genes; however, the factors underlying tumor progression and metastasis remain poorly understood. Mutations in TP53 and SMAD4, in particular, are associated with CRC progression. Although gut microbiome dysbiosis is implicated in CRC initiation and progression, the interactions between the microbiome and specific CRC driver mutations, especially those promoting metastasis, are not well defined. Methods In this study, we utilized triple mutant (Apc, Kras, Tp53; AKP) and quadruple mutant (Apc, Kras, Tp53, Smad4; AKPS) organoid-based orthotopic mouse models of CRC to investigate the impact of the SMAD4 mutation on microbiome composition. Results Our results reveal significant differences in metastatic potential and microbial community dynamics between the two tumor models. AKPS tumors exhibited metastasis to the lymph nodes, liver, and lungs, while AKP tumors remained confined to the colon. Longitudinal microbiome analysis showed shifts in microbial composition within each tumor model. Both AKP and AKPS models demonstrated enrichment of Faecalibaculum and a decrease in Dubosiella over time; however, additional shifts were noted with distinct taxa associated with late-stage tumors in each group. Notably, the AKPS model exhibited higher relative abundances of pro-inflammatory taxa, including Turicibacter, Romboutsia, and Akkermansia, suggesting that the SMAD4 mutation promotes a more immunosuppressive and pro-metastatic microbiome profile. Conclusions These findings emphasize the significance of SMAD4 mutation and microbiome modulation, revealing the interaction between host genetics and gut microbiota in driving colorectal cancer aggressiveness and suggesting potential microbial targets. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02580-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ea8e36d3fb22448abe7d7539393a947dae74443" target='_blank'>
              SMAD4 mutation drives gut microbiome shifts toward tumor progression in colorectal cancer
              </a>
            </td>
          <td>
            Travis J Gates, Dechen Wangmo, Kyra M Boorsma Bergerud, Bridget M Keel, Christopher Staley, Subbaya Subramanian
          </td>
          <td>2025-10-09</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/018e4ec4dd8ef81f5b174f65fbe205bcc992d85a" target='_blank'>
              The Impact of Polyploid Giant Cancer Cells: The Root of Stress Resilience.
              </a>
            </td>
          <td>
            Yuta Ogawa, Lydia Fisher, Tomonori Matsumoto
          </td>
          <td>2025-09-08</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 The University of Sydney, Sydney, NSWRhabdoid Tumours (RTs) are highly aggressive rare pediatric malignancies that primarily affect infants and young children, arising as malignant Rhabdoid Tumours (MRTs) in the kidney or Atypical Teratoid/Rhabdoid Tumours (AT/RTs) in the brain. These tumours are driven by SMARCB1 mutations or deletions and exhibit a low tumour mutation burden (TMB), traditionally suggesting poor immunogenicity. However, immune profiling has revealed substantial immune infiltration into the tumour immune microenvironment (TIME), including robust T-cell and myeloid cell presence, PD-L1 expression, and SMARCB1-dependent re-expression of endogenous retroviruses (ERVs), which can activate anti-tumour immune responses. Despite these features, RTs remain incurable, underscoring the need to elucidate their immune landscape to inform novel therapeutic strategies. Using data from the ZERO Childhood Cancer program, we applied bulk RNA sequencing and whole-genome sequencing to deconvolute the immune cell composition beyond T cells, preform IPASS, and assess associations between genomic alterations and immune features. Notably, we identified a strong correlation between high inflammation/IPASS scores and pathogenic SWI/SNF complex mutations, including SMARCB1 loss. Among immune modulators, leukemia inhibitory factor (LIF) emerged as a key player, displaying high expression in RTs and an inverse relationship with SWI/SNF gene expression. To spatially resolve the TIME of RTs, we performed spatial transcriptomics and proteomics, focusing on immune cell infiltration and exclusion mechanisms mediated by LIF-LIFR signalling. We found that LIF is predominantly expressed by tumour cells, while LIFR is enriched on myeloid cells, suggesting a tumour-driven immunosuppressive axis. Moreover, MRTs and AT/RTs exhibit distinct immune architectures: MRTs are highly inflamed, with dispersed T cells and myeloid cells across tumour and stromal regions, whereas AT/RTs are dominated by myeloid infiltration localized to stromal regions and perivascular niches. Xenium-based differential gene expression analysis of macrophages from AT/RT and MRT patients reveals that AT/RT-associated macrophages exhibit reduced interferon signalling and proliferation, but enhanced tissue remodelling and immunosuppressive M2-like polarization. This study underscores the power of multiomic approaches in dissecting the immune landscape of rare and incurable paediatric solid tumours. By leveraging the IPASS gene signature and LIF-LIFR axis, we identified immune exclusion mechanisms that could serve as therapeutic targets. Future investigations will focus on validating these findings and assessing their potential for combinatorial immunotherapies in rhabdoid tumours. Lastly, the dataset generated by this study is a rare and invaluable resource in the advancement of rhabdoid tumour research.



 Erin Coll, Chelsea Mayoh, Lauren Brown, Anne-Lise Gerard, Helen McGuire, Paul Ekert. Multi-omic characterisation of the immune microenvironment in Rhabdoid Tumours [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9035caab9c962f80d8ad66b2920e4557557a32b5" target='_blank'>
              Abstract B025: Multi-omic characterisation of the immune microenvironment in Rhabdoid Tumours
              </a>
            </td>
          <td>
            Erin Coll, Chelsea Mayoh, Lauren M. Brown, Anne-Lise Gerard, Helen M. McGuire, P. Ekert
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Tertiary lymphoid structures (TLSs) function as ectopic immune centers in non-lymphoid tissues and are a key area of research in tumor immunology, particularly in hepatocellular carcinoma (HCC). However, the role of TLSs in HCC shows significant heterogeneity; intratumoral TLSs (iTLSs) are linked to favorable outcomes and immune therapy responses, while peritumoral TLSs may be dysfunctional or promote tumor progression. This spatial paradox is a core scientific question for understanding and leveraging TLSs in HCC treatment. The liver's unique immunosuppressive and fibrotic microenvironment, influenced by hepatic stellcell-derived cancer-associated fibroblasts (CAFs) and local signals like tryptophan metabolism, profoundly regulates TLSs formation and functional polarization.


METHODS
This review integrates current research on TLSs in HCC, focusing on their functional heterogeneity based on spatial localization and the regulatory mechanisms involving the liver's microenvironment. It identifies major bottlenecks in TLSs clinical translation, including inconsistent assessment standards and the lack of non-invasive dynamic monitoring methods. Cutting-edge technological paradigms such as spatial multi-omics, AI-driven computational pathology, and multi-modal liquid biopsy are explored as feasible pathways for precise decoding and standardized assessment of TLSs functional states.


RESULTS
Existing evidence highlights the differential impact of iTLSs (favorable outcomes, improved immunotherapy response) versus peritumoral TLSs (dysfunctional or pro-tumorigenic) in HCC. The regulation of TLSs is shown to be deeply intertwined with the liver's unique microenvironment, particularly CAFs and tryptophan metabolism. The analysis points to the potential for integrating advanced technologies to overcome current assessment and monitoring challenges, paving the way for a more precise understanding of TLSs functional states.


CONCLUSION
Addressing the functional paradox of TLSs based on spatial localization in HCC is crucial for guiding treatment. Overcoming current assessment and monitoring bottlenecks through the integration of spatial multi-omics, AI-driven computational pathology, and multi-modal liquid biopsy offers a pathway for precise decoding and standardized evaluation of TLSs functional states. Future research should shift towards active therapeutic interventions targeting TLSs function and location to optimize personalized immunotherapy strategies for HCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a2db1fcae8b8c1c630e42c8bfd5c16f6322dbf4" target='_blank'>
              The Dichotomous Role of Tertiary Lymphoid Structures in Hepatocellular Carcinoma: From Spatial Location to Clinical Implications.
              </a>
            </td>
          <td>
            Jingyi Xu, Yunzhang Cheng, Yilin Wang, Yun Feng
          </td>
          <td>2025-08-31</td>
          <td>Immunological investigations</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Amyotrophic lateral sclerosis (ALS) progression rates vary dramatically between patients, yet the basis of this heterogeneity remains elusive, with no prognostic biomarkers existing to guide clinical decisions or stratify patients for therapeutic trials. Here, we identify a network of coordinated immune cell types, which exhibit differential disruption across progression groups. Using mass cytometry (CyTOF) to profile 2.2 million immune cells from 35 ALS patients stratified by progression rate and 9 healthy controls, we find that the extent of immune dysfunction cannot be reflected by examining differences in individual cell type frequencies. In contrast, analyses of correlation patterns between cell types revealed distinct immune organization patterns, where coordination complexity varied with disease progression. Across all progression groups, we observed striking immune reorganization in natural killer (NK) cells and a major shift from B cell/basophil coordination hubs in healthy controls to neutrophil/T cell-dominated patterns in ALS. Having established coordinated immune patterns, we developed machine learning models to further improve our ability to stratify between disease and non-disease cohorts, achieving superior performance compared to models using cell frequencies alone. Central and effector memory (CM/EM) CD4+ T cell interactions emerged as top discriminative features for disease status, while plasmacytoid dendritic cell (pDC) relationships, especially their ratio with regulatory T cells (T-regs), distinguished progression rates, supporting T-reg-based therapeutic approaches. These findings reframe ALS as a disease of immune coordination breakdown, pointing towards cell-type specific therapeutics and biomarkers that may extend beyond ALS to other neurodegenerative diseases characterized by immune dysfunction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55da039f7aa68cdb569b978ffb6e59b6e1b808d0" target='_blank'>
              Peripheral immune patterns enable robust cross-platform prediction of ALS onset and progression
              </a>
            </td>
          <td>
            Luvna Dhawka, B. Evangelista, Omeed K. Arooji, Marie A. Iannone, Kyle Pellegrino, Rebecca Traub, Xiaoyan Li, Richard S. Bedlack, Rick Meeker, Todd J. Cohen, Natalie Stanley
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Anaplastic thyroid carcinoma (ATC) is one of the most lethal human malignancies, often evolving from differentiated thyroid carcinoma (DTC) through a poorly understood dedifferentiation process. To elucidate this transition, spatial transcriptomic sequencing (spRNAseq) is performed on seven samples containing coexisting regions of ATC, poorly differentiated thyroid carcinoma, and DTC. SpRNAseq revealed that ATC regions were characterized by upregulated genes involved in immune suppression, angiogenesis, and extracellular matrix remodeling. Whole-exome sequencing and inferCNV analysis confirmed that adjacent DTC regions harbored mutational burdens comparable to those of ATC regions, suggesting early genomic priming for dedifferentiation. Trajectory analysis delineated a stepwise reprogramming process and identified four gene modules associated with the loss of thyroid differentiation, among which PDCD4 and TYMP emerged as key regulators. Notably, TYMP⁺ tumor-associated macrophages (TAMs) were highly enriched in ATC regions and contribute to an immunosuppressive microenvironment. Mechanistic experiments demonstrated that loss of PDCD4 led to eIF4A-dependent overexpression of immunosuppressive effectors, promoting the high infiltration of TYMP⁺TAMs in ATC. These findings support that coexisting DTC regions with ATC-like genomic alterations undergo sequential transcriptomic reprogramming and immune microenvironment remodeling to evolve into a full ATC pathological phenotype, in which PDCD4 loss-induced TAMs formation plays a critical role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4fbade6a7e0bbb1789c00df5213a5311bb8e73a" target='_blank'>
              Spatial Transcriptomics Reveals Transcriptomic and Immune Microenvironment Reprogramming during Thyroid Carcinoma Dedifferentiation.
              </a>
            </td>
          <td>
            K. Ning, Bu Zou, Yongchao Yu, Taonong Cai, Zhenyu Luo, Yu Guo, Yi Wu, Xiujiao Shen, Hao Li, Mengyuan Fang, J. Bu, Han Hong, Zan Jiao, Tong Wu, Yulong Wang, Tianrun Liu, Wei-chao Chen, Wanming Hu, Mingjie Jiang, A. Yang
          </td>
          <td>2025-09-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background The tertiary lymphoid structures (TLS) within the immune microenvironment of hepatocellular carcinoma (HCC) have been shown to significantly influence patient prognosis. Understanding the mechanisms behind their formation and maturation can help us refine therapeutic strategies and improve treatment outcomes. In certain cancers, intratumoral bacteria have been found to promote the development of TLS. Our study aims to investigate the impact of intratumoral bacteria on TLS-associated immune cells in HCC, as well as their relationship with TLS quantity and maturation status. Methods In this study, we collected samples from 153 patients with hepatocellular carcinoma. We employed fluorescence in situ hybridization (FISH), immunofluorescence (IF), and hematoxylin and eosin (H&E) staining to assess intratumoral bacterial load, as well as to evaluate the number and maturation status of tertiary lymphoid structures. Patient prognosis was also analyzed. Furthermore, we examined the bacterial load and distribution within tumors of patients presenting TLS, and explored the relationship between intratumoral bacteria and TLS-associated immune cell infiltration. Results Among 74 patients with hepatocellular carcinoma, tertiary lymphoid structures were present within the tumors. By integrating the expression profiles of both intratumoral and peritumoral TLS, prognostic analysis revealed that patients with structured tumor microenvironments (TME) and exclusionary TME had better outcomes. The intratumoral bacterial load varied among patients, with higher bacterial burden observed in regions enriched with TLS. Moreover, as TLS matured, the bacterial load within tumors was significantly greater compared to patients lacking TLS. Correspondingly, CD20, a major component of TLS, showed increased expression. These findings suggest that intratumoral bacteria can influence the immune response within the tumor microenvironment and are associated with the maturation of TLS. Conclusion Our study demonstrates that patients receiving structured TME and excluded TME subtypes exhibit superior overall survival (OS) and recurrence-free survival (RFS) following radical surgery. Furthermore, the intratumoral bacterial load showed a significant correlation with CD20+ B-cell density and was strongly correlated with both the number and maturity of TLS. These findings suggest that intratumoral bacteria may influence patient responses to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51afc77b8d734ec619b2147ecef00cb70414525c" target='_blank'>
              Intratumoral bacterial load and tertiary lymphoid structure density in hepatocellular carcinoma: association and prognostic significance
              </a>
            </td>
          <td>
            Shen Qu, Weili Jia, Xiaoyan Liu, Qianyun Yao, Chao Chen, Zihao Zhao, Ye Nie, Feng Chang, Zexu Yang, Chaosheng Peng, Yangang Wang, Wenjie Song
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Internal organs are encased by a supportive epithelial monolayer of mesodermal origin, termed mesothelium. The nature, evolution and function of mesothelial cells, and their genetic regulation impacting disease development are insufficiently understood. Here, we generate a comprehensive organ-wide single-cell transcriptomic compendium of mesothelium across healthy and diseased mouse and human organs, delineating the evolution of conserved activated states of mesothelial cells in response to disease. We uncover genetic drives behind each cell state and reveal a conserved metabolic gate into multipotent proteolytic, inflammatory and fibrotic cell differentiation, in mouse and human. Using lung injury models in mice, in combination with mesothelial cell-specific viral approaches, we show that direct metabolic reprogramming using Ifi27l2a and Crip1 on organ surfaces, blocks multipotent differentiation and protects mouse lungs from fibrotic disease. These findings place mesothelial cells as cellular exemplars and gateway to fibrotic disease, opening translational approaches to subvert fibrosis across a range of clinical indications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32dcba7e7305488d54f2d30bf4e811162db8c3c5" target='_blank'>
              A mesothelial differentiation gateway drives fibrosis
              </a>
            </td>
          <td>
            Safwen Kadri, A. Fischer, Martin A Mück-Häusl, Wei Han, Amal Kadri, Yue Lin, Lin Yang, Shaoping Hu, Haifeng Ye, Pushkar Ramesh, M. Ansari, Herbert B. Schiller, H.-G. Machens, Y. Rinkevich
          </td>
          <td>2025-09-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Tertiary lymphoid structures (TLSs) are ectopic lymphoid aggregates that form in non-lymphoid organs, frequently observed in conditions such as cancer, autoimmune diseases, transplant rejection, and chronic inflammation. Growing evidence suggests that TLSs are beneficial for patients’ prognosis with higher TLS density generally correlating with improved therapeutic response and survival outcomes across malignancies and might serve as a novel therapeutic target for cancer immunotherapy. However, the correlation between TLSs and tumor development is still ambiguous. The exact timing of TLS formation during tumorigenesis and their dynamic evolution throughout tumor progression remain under investigation. Recent studies have identified potential strategies for inducing TLSs, but there remains a considerable distance from clinical application. More advanced techniques such as high-resolution spatial multi-omics technologies combined with big data analysis will benefit understanding the complex interactions within TLSs and developing novel immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d68cd72d4e06f1532031fad0eb22d59671e56af" target='_blank'>
              Orchestration of tertiary lymphoid structures: decoding developmental mechanisms for next-generation cancer immunotherapies
              </a>
            </td>
          <td>
            Wenqin Hao, Ying Zhao
          </td>
          <td>2025-09-12</td>
          <td>Medical Review</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bb8a507427a3e0cfe9ea50f66dabfc18ac940f2" target='_blank'>
              Cancer Metastasis: Therapeutic Challenges and Opportunities.
              </a>
            </td>
          <td>
            Sri Sathya Sandilya Garemilla, Manisha Choudhary Kadambala, S. C. Gampa, Swetha Chinthala, S. Garimella
          </td>
          <td>2025-10-16</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related morbidity and mortality, with African-ancestry populations experiencing higher incidence and poorer outcomes. Although socioeconomic factors contribute, growing evidence suggests biological differences also play a role. However, ancestry-associated molecular features of CRC remain insufficiently defined, largely due to the underrepresentation of non-European populations in genomic studies. In this study, transcriptomic profiles of African-Black and European-White CRC patients from The Cancer Genome Atlas (TCGA) were compared to identify ancestry-associated molecular programs. Differential expression analysis identified dysregulated genes, and pathway enrichment revealed distinct biological processes. White patients exhibited coordinated immune–metabolic pathways, whereas Black patients showed fragmented modules centered on CRYBB2 and MAP9. Immune profiling using single-sample gene set enrichment analysis (ssGSEA) further demonstrated higher enrichment of interferon alpha response, allograft rejection, and complement activation in White patients, complemented by increased expression of immune checkpoint genes. Bootstrap resampling confirmed the robustness of these findings despite sample imbalance. These results provide novel evidence of ancestry-linked transcriptomic differences in CRC and underscore the importance of including diverse populations in cancer genomics to develop equitable biomarkers and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c54690c9d78f17a34e09a1ce349638b6ab3f2c1" target='_blank'>
              Divergent Molecular Networks Drive Ancestry-Related Colorectal Cancer Disparities
              </a>
            </td>
          <td>
            Salma Fareed
          </td>
          <td>2025-09-15</td>
          <td>Scholarly Review Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT RNA viruses have high mutation frequency, quick generation periods and vast population numbers, which promote fast evolution and host environment adaptation. We integrated scRNA‐seq and spatial transcriptomics to profile immune cells and viral gene expression in COVID‐19. Cell types and interactions were identified using Seurat‐based tools. Spatial transcriptomics analysis revealed viral hotspots, and GISAID data were used to track SARS‐CoV‐2 mutations. Single‐cell and spatial transcriptomics analyses revealed that immune cells such as Neutrophils, Monocyte:CD14 + , and T cell:CD4+ central memory are highly abundant in COVID‐19 patients, particularly in mild and severe cases, and are concentrated in the central and upper regions of lung tissue. Pseudotime and CellChat analyses indicated that cell differentiation trajectories and communication networks shift toward heightened inflammatory responses in severe conditions. Spatial analysis of viral gene expression showed that SARS‐CoV‐2 genes, especially Nucleoprotein, Spike and Envelope were highly expressed in central and upper‐right tissue regions, suggesting active viral replication. This localized viral activity was strongly associated with areas of immune cell infiltration and inflammation. The top 10 sustainable mutants in SARS‐CoV‐2 genome with high frequency were observed in NSP12 (P323L, 99%, Switzerland), Spike (D614G, 97%, Switzerland), NSP4 (T492I, 79%, Switzerland), NSP6 (T77A, 70%, Guangdong), Orf9c (G50N, 64%, England), Nucleoprotein (D377Y, 62%, United States), Orf9b (T60A, 61%, France), NSP14 (I42V, 55%, United States), Envelope (T9I, 51.3%, Trinidad and Tobago), and NSP5 (P132H, 51.2%, United States). Following to this approach is crucial for a strong epidemiological reaction against the changing SARS‐CoV‐2 outbreak.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41d4c6b68c2ba2cc60900b3817724ba0dc5ef871" target='_blank'>
              Spatial Transcriptomics and Single Cell‐RNASeq Reveals Cellular Heterogeneity of SARS‐CoV‐2 in Lung Tissues and Global Mutational Patterns in COVID‐19 Patients
              </a>
            </td>
          <td>
            Seyed Taleb Hosseini, M. Mahmanzar, K. Rahimian, Saleha Bayat, Amir Gholamzad, M. M. Sisakht, Amin Farhadi, D. L. Kuehu, Youping Deng
          </td>
          <td>2025-09-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>7</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>