<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-11-11 06:31:29 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Tumor progression is driven by dynamic interactions between cancer cells and their surrounding microenvironment. Investigating the spatiotemporal evolution of tumors can provide crucial insights into how intrinsic changes within cancer cells and extrinsic alterations in the microenvironment cooperate to drive different stages of tumor progression. Here, we integrate high-resolution spatial transcriptomics and evolving lineage tracing technologies to elucidate how tumor expansion, plasticity, and metastasis co-evolve with microenvironmental remodeling in a Kras;p53-driven mouse model of lung adenocarcinoma. We find that rapid tumor expansion contributes to a hypoxic, immunosuppressive, and fibrotic microenvironment that is associated with the emergence of pro-metastatic cancer cell states. Furthermore, metastases arise from spatially-confined subclones of primary tumors and remodel the distant metastatic niche into a fibrotic, collagen-rich microenvironment. Together, we present a comprehensive dataset integrating spatial assays and lineage tracing to elucidate how sequential changes in cancer cell state and microenvironmental structures cooperate to promote tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d58cdf3e4d61aebac0aebfdb8231e65833afc8fc" target='_blank'>
              Spatiotemporal lineage tracing reveals the dynamic spatial architecture of tumor growth and metastasis
              </a>
            </td>
          <td>
            Matthew G Jones, Dawei Sun, Kyung Hoi Joseph Min, William N. Colgan, Luyi Tian, Jackson A Weir, Victor Z. Chen, Luke W. Koblan, Kathryn E. Yost, Nicolas Mathey-Andrews, Andrew J. C. Russell, Robert R. Stickels, Karol S. Balderrama, William Rideout, Howard Y. Chang, T. Jacks, Fei Chen, Jonathan S. Weissman, Nir Yosef, Dian Yang
          </td>
          <td>2024-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Intratumoral heterogeneity is intrinsically comprised of molecular alterations of tumor cells and extrinsically from interconnections with microenvironments. This study explores the spatial heterogeneity of ovarian clear cell carcinoma (OCCC), a rare cancer with significance to East Asian women. We profiled 21 tumors from matched ovarian and metastatic sites with spatial transcriptomic (ST) platforms including GeoMx and Visium CytAssist. Three subclusters of OCCC tumor cells enriched in oxidative phosphorylation, epithelial-mesenchymal transition, and inflammation pathways were identified with preferential geospatial localizations in tumor centers and invasive margins/tumor-stromal interfaces. These ST subclusters correlated with epithelial-mesenchymal gradients and prognostic features in larger cohorts. Single-cell ST in CosMx identified 9 cancer cell populations with organ-specific tropism. Trajectory analysis indicated a NEAT1-high subclone having founder characteristics, progressively dominating the metastatic niche, displaying mesenchymal traits. This founder feature correlated with clinical outcomes in PanCancer datasets. Our findings highlight the adaptive and context-specific interactions between cancer cells and microenvironments within spatial complexities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8349570f5b5a5cc83b0effd91ea241eef3a2ce6c" target='_blank'>
              Spatial Transcriptomic Profiling Of Advanced Ovarian Clear Cell Carcinoma Reveals Intra-Tumor Heterogeneity In Epithelial-Mesenchymal Gradient
              </a>
            </td>
          <td>
            Thang Truong Le, Duncan Yi-Te Wang, Ko-Chen Chen, Sebastian Rui-Gu Yan, Pei-Yu Chu, Tuan Zea Tan, Yi-Chia Chiu, Jia-Yuh Sheu, Chen-Hao Huang, Jieru Ye, Ya-Ting Tai, Hsueh-Fen Juan, Feng-Chiao Tsai, Wei-Chou Lin, Lin-Hung Wei, Ruby Yun-Ju Huang
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In the ever-changing world of digital pathology, being able to extract a maximum amount of information from a patient tissue sample is of paramount importance for better diagnosis, disease characterization, and therapeutic strategies. Recent technologies such as multiplex immunofluorescence imaging and spatial transcriptomic now enable a deep analysis of protein and gene expression while retaining the spatial context of the tissue. Here, we describe an innovative approach combining a 34-protein Phenocycler panel and transcriptome analysis using Visium on a single head and neck squamous cell carcinoma section. While protein analysis reveals the complexity of the immune phenotypes involved in the disease, transcriptome analysis reveals the intricate cellular states of cancer cells that coexist within the patient’s tumor. Finally, integrating both omics modalities, we uncover unique comparison of gene and protein expression of spatially resolved cellular subspaces.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6713975853fd4c99d1beb66b49491a5d06627320" target='_blank'>
              A new spatial multi-omics approach to deeply characterize human cancer tissue using a single tissue section
              </a>
            </td>
          <td>
            Yasmine Adimi, Clement Levin, Emilia Puig Lombardi, Benjamin Morin, Claire Gervais, Sandrine Tolou, Anne-Claude Noirot-Jagu, Julie Hauret, Corinne Thomas, Patricia Tchepikoff, Fedor Grigorev, Anya Djafri, Sandrine Longuemaux, Anne Caron, Souad Naimi, Matteo Cesaroni, E. Tartour, Marion Classe, Cecile Badoual
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer progression to visceral organs such as lung and liver is regarded as a dreadful event, unequivocally associated with a poor prognosis. Yet, these vital sites are characterized by highly diverse cellular microenvironments and physiological functions, suggesting that they may influence cancer cells behavior in divergent ways. Unexpectedly, we find that while the liver microenvironment fosters metastasis-boosting properties of its metastatic seeds and favors secondary spread, the lungs impose a drastic roadblock to the same processes. Using patient data and mouse models of breast cancer that spontaneously metastasize to visceral organs, coupled with genetic barcode-mediated metastasis tracing, niche labeling technology and single cell analysis of both tumor cells and their direct microenvironment at each site, we characterize this dichotomy and identify differential microenvironmental factors that contribute to liver vs lung metastasis biology. Mechanistically, we uncover pulmonary cytokines, such as Granulin, whose activity naturally blocks metastatic properties. Conversely, we identify BMP2-producing endothelial cells within the liver metastatic niche, capable to enhance further spread. Together, we reveal an unexpected contrast in the site-specific behavior of visceral metastases in triple-negative breast cancer models, highlighting microenvironmental factors that contribute to its geographical diversity, as well as organ-specific opportunities for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9abbeb3d8e83ff0a32a06d5c39fa11a5d2c21ff" target='_blank'>
              Mapping the niche of breast cancer metastases in lung and liver
              </a>
            </td>
          <td>
            Magdalena K. Sznurkowska, Francesc Castro-Giner, Yongzhan Zhang, Alexander Ring, F. Schwab, Francesca Albrecht, K. D. Mertz, Karin Strittmatter, Lauren L. Ozimski, Selina Budinjas, G. Auray, C. Giachino, Verdon Taylor, Ana Gvozdenovic, I. Malanchi, Heike Frauchiger-Heuer, Andreas Wicki, M. Vetter, Nicola Aceto
          </td>
          <td>2024-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e27ecfb76604dddc8d09da4d1bbaaf7b66725bf" target='_blank'>
              Spatial multiplexed immunofluorescence analysis reveals coordinated cellular networks associated with overall survival in metastatic osteosarcoma
              </a>
            </td>
          <td>
            Ryan A Lacinski, Sebastian A. Dziadowicz, Vincent K. Melemai, Brody Fitzpatrick, John J. Pisquiy, Tanya Heim, Ines Lohse, Karen Schoedel, Nicolas J. Llosa, Kurt R Weiss, B. Lindsey
          </td>
          <td>2024-09-27</td>
          <td>Bone Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background: HCC is a highly vascular tumor, and many effective drug regimens target the tumor blood vessels. Prior bulk HCC subtyping data used bulk transcriptomes, which contained a mixture of parenchymal and stromal contributions. Methods: We utilized computational deconvolution and cell-cell interaction analyses to cell type-specific (tumor-enriched and vessel-enriched) spatial transcriptomic data collected from 41 resected HCC tissue specimens. Results: We report that the prior Hoshida bulk transcriptional subtyping schema is driven largely by an endothelial fraction, show an alternative tumor-specific schema has potential prognostic value, and use spatially paired ligand-receptor analyses to identify known and novel (LGALS9 tumor-HAVCR2 vessel) signaling relationships that drive HCC biology in a subtype-specific and potentially targetable manner. Conclusions: Our study leverages spatial gene expression profiling technologies to dissect HCC heterogeneity and identify heterogeneous signaling relationships between cancer cells and their endothelial cells. Future validation and expansion of these findings may validate novel cancer-endothelial cell interactions and related drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e5076806fef62fc7b47a9b3751a8891a984dc18" target='_blank'>
              HCC spatial transcriptomic profiling reveals significant and potentially targetable cancer-endothelial interactions
              </a>
            </td>
          <td>
            Chenyue Lu, Amaya Pankaj, Michael J. Raabe, Cole Nawrocki, Ann Liu, Nova Xu, Bidish Kumar Patel, Matthew J. Emmett, Avril K Coley, Cristina R Ferrone, Vikram Deshpande, I. Bhan, Y. Hoshida, David T. Ting, Martin J. Aryee, Joseph W Franses
          </td>
          <td>2024-09-27</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3095632bcf16399c0c6ee358ffba9b30df1dc77" target='_blank'>
              Integrative spatial and genomic analysis of tumor heterogeneity with Tumoroscope
              </a>
            </td>
          <td>
            S. Shafighi, Agnieszka Geras, Barbara Jurzysta, Alireza Sahaf Naeini, Igor Filipiuk, Alicja Ra Czkowska, Hosein Toosi, Łukasz Koperski, K. Thrane, Camilla Engblom, Jeff E. Mold, Xinsong Chen, Johan Hartman, Dominika Nowis, A. Carbone, Jens Lagergren, E. Szczurek
          </td>
          <td>2024-10-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PDAC) is a rapidly progressing cancer that responds poorly to immunotherapies. Intratumoral tertiary lymphoid structures (TLS) have been associated with rare long-term PDAC survivors, but the role of TLS in PDAC and their spatial relationships within the context of the broader tumor microenvironment remain unknown. We generated a spatial multi-omics atlas encompassing 26 PDAC tumors from patients treated with combination immunotherapies. Using machine learning-enabled H&E image classification models and unsupervised gene expression matrix factorization methods for spatial transcriptomics, we characterized cellular states within TLS niches spanning across distinct morphologies and immunotherapies. Unsupervised learning generated a TLS-specific spatial gene expression signature that significantly associates with improved survival in PDAC patients. These analyses demonstrate TLS-associated intratumoral B cell maturation in pathological responders, confirmed with spatial proteomics and BCR profiling. Our study also identifies spatial features of pathologic immune responses, revealing TLS maturation colocalizing with IgG/IgA distribution and extracellular matrix remodeling. GRAPHICAL ABSTRACT HIGHLIGHTS Integrated multi-modal spatial profiling of human PDAC tumors from neoadjuvant immunotherapy clinical trials reveal diverse spatial niches enriched in TLS. TLS maturity is influenced by tumor location and the cellular neighborhoods in which TLS immune cells are recruited. Unsupervised machine learning of genome-wide signatures on spatial transcriptomics data characterizes the TLS-enriched TME and associates TLS transcriptomes with survival outcomes in PDAC. Interactions of spatially variable gene expression patterns showed TLS maturation is coupled with immunoglobulin distribution and ECM remodeling in pathologic responders. Intratumoral plasma cell and immunoglobin gene expression spatial dynamics demonstrate trafficking of TLS-driven humoral immunity in the PDAC TME. Significance We report a spatial multi-omics atlas of PDAC tumors from a series of immunotherapy neoadjuvant clinical trials. Intratumorally, pathologic responders exhibit mature TLS that propagate plasma cells into malignant niches. Our findings offer insights on the role of TLS-associated humoral immunity and stromal remodeling during immunotherapy treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b4043873d18d3b758a620d9a4f90e32bea7034" target='_blank'>
              Spatial multi-omics reveal intratumoral humoral immunity niches associated with tertiary lymphoid structures in pancreatic cancer immunotherapy pathologic responders
              </a>
            </td>
          <td>
            Dimitri N. Sidiropoulos, Sarah M. Shin, Meredith Wetzel, Alexander Girgis, Daniel Bergman, Ludmila Danilova, Susheel Perikala, Daniel H. Shu, Janelle M. Montagne, Atul Deshpande, James Leatherman, Lucie Dequiedt, Victoria Jacobs, Aleksandra Ogurtsova, Guanglan Mo, Xuan Yuan, Dmitrijs Lvovs, G. Stein-O’Brien, M. Yarchoan, Qingfeng Zhu, Elizabeth I. Harper, A. Weeraratna, A. Kiemen, Elizabeth M. Jaffee, Lei Zheng, W. Ho, Robert A. Anders, E. Fertig, L. T. Kagohara
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Malignant melanoma is an aggressive cancer, with a high risk of metastasis and mortality rates, characterized by cancer cell heterogeneity and complex tumor microenvironment (TME). Single cell biology is an ideal and powerful tool to address these features at a molecular level. However, this approach requires enzymatic cell dissociation that can influence cellular coverage. By contrast, single nucleus RNA sequencing (snRNA-seq) has substantial advantages including compatibility with frozen samples and the elimination of a dissociation-induced, transcriptional stress response. To better profile and understand the functional diversity of different cellular components in melanoma progression, we performed snRNA-seq of 16,839 nuclei obtained from tumor samples along the growth of murine syngeneic melanoma model carrying a BRAFV600E mutation and collected 9 days or 23 days after subcutaneous cell injection. We defined 11 different subtypes of functional cell clusters among malignant cells and 5 different subsets of myeloid cells that display distinct global transcriptional program and different enrichment in early or advanced stage of tumor growth, confirming that this approach was useful to accurately identify intratumor heterogeneity and dynamics during tumor evolution. The current study offers a deep insight into the biology of melanoma highlighting TME reprogramming through tumor initiation and progression, underlying further discovery of new TME biomarkers which may be potentially druggable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d37af9fbd7e323bb1c7c0e6480590eefcaf22d16" target='_blank'>
              Single-Nuclei Transcriptome Profiling Reveals Intra-Tumoral Heterogeneity and Characterizes Tumor Microenvironment Architecture in a Murine Melanoma Model
              </a>
            </td>
          <td>
            S. Parab, Valery Sarlo, S. Capellero, Luca Palmiotto, A. Bartolini, Daniela Cantarella, M. Turi, Annamaria Gullà, Elena Grassi, Chiara Lazzari, Marco Rubatto, Vanesa Gregorc, Fabrizio Carnevale-Schianca, Martina Olivero, Federico Bussolino, V. Comunanza
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44ac4365f3d537cc727149bd2f0c2b4a261aed5" target='_blank'>
              The neuroendocrine transition in prostate cancer is dynamic and dependent on ASCL1.
              </a>
            </td>
          <td>
            Rodrigo Romero, Tinyi Chu, Tania J González Robles, Perianne Smith, Yubin Xie, Harmanpreet Kaur, Sara Yoder, Huiyong Zhao, Chenyi Mao, Wenfei Kang, Maria V. Pulina, K. Lawrence, Anuradha Gopalan, S. Zaidi, K. Yoo, Jungmin Choi, Ning Fan, Olivia Gerstner, W. Karthaus, E. deStanchina, Kelly V. Ruggles, P. Westcott, R. Chaligné, D. Pe’er, C. Sawyers
          </td>
          <td>2024-10-11</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is an immensely complex ecosystem1,2. This complexity underlies difficulties in elucidating principles of spatial organization and using molecular profiling of the TME for clinical use3. Through statistical analysis of 96 spatial transcriptomic (ST-seq) datasets spanning twelve diverse tumor types, we found a conserved distribution of multicellular, transcriptionally covarying units termed ‘Spatial Groups’ (SGs). SGs were either dependent on a hierarchical local spatial context – enriched for cell-extrinsic processes such as immune regulation and signal transduction – or independent from local spatial context – enriched for cell-intrinsic processes such as protein and RNA metabolism, DNA repair, and cell cycle regulation. We used SGs to define a measure of gene spatial heterogeneity – ‘spatial lability’ – and categorized all 96 tumors by their TME spatial lability profiles. The resulting classification captured spatial variation in cell-extrinsic versus cell-intrinsic biology and motivated class-specific strategies for therapeutic intervention. Using this classification to characterize pre-treatment biopsy samples of 16 non-small cell lung cancer (NSCLC) patients outside our database distinguished responders and non-responders to immune checkpoint blockade while programmed death-ligand 1 (PD-L1) status and spatially unaware bulk transcriptional markers did not. Our findings show conserved principles of TME spatial biology that are both biologically and clinically significant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a400ff77ff3e3d5b4307dd66b458357a7b860383" target='_blank'>
              Conserved principles of spatial biology define tumor heterogeneity and response to immunotherapy
              </a>
            </td>
          <td>
            Vivek Behera, Hannah Giba, Ue-yu Pen, Anna Di Lello, Benjamin A. Doran, Alessandra Esposito, Apameh Pezeshk, C. Bestvina, Justin Kline, M. Garassino, Arjun S. Raman
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0d10a131e1ec85b605aa6456ac30c2943dbeb4" target='_blank'>
              Spatial immunogenomic patterns associated with lymph node metastasis in lung adenocarcinoma
              </a>
            </td>
          <td>
            Fanjie Meng, Hao Li, Ruoyi Jin, Airong Yang, Hao Luo, Xiao Li, Peiyu Wang, Yaxing Zhao, O. Chervova, Kaicheng Tang, Sida Cheng, Bin Hu, Yun Li, Jianpeng Sheng, Fan Yang, David Carbone, Kezhong Chen, Jun Wang
          </td>
          <td>2024-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is heterogeneous with low tumor purity, prominent microenvironment and complex architecture, which preclude identification of shared tumor intrinsic biology within and across patients. We overcame these challenges by applying complementary spatial omics approaches – providing necessary resolution and context – to primary untreated PDAC tumors from 39 patients capturing 341,949 low-bulk and 531,718 single-cell spatial transcriptomes. Of these, 59,403 low-bulk profiles and 205,665 single-cells represented tumor cells. We leveraged this data to build on prior reports of tumor cell subtypes in PDAC. We identify 5 distinct malignant subtypes, excluding ductal-like and intraepithelial neoplasia cells, that span the classical-to-basal spectrum. One tumor subtype is highly proliferative and enriched for the DNA synthesis-condensation-mitosis transcriptional network suggesting this subset disproportionately contribute to tumor growth. Strikingly, the spectrum of tumor cell subtypes was present within all patients, suggesting that subtype-based treatment stratification may need refinement. We find that each subtype has its own distinct regulators and histology. The classical subtype has extensive cell intrinsic regulation, higher cell density and lower extracellular matrix content. In contrast, the basal subtype phenotype results from a combination of cell intrinsic and tumor microenvironment (TME) interactions, wherein the basal subtype has a lower cell density, is surrounded by activated stroma and dense collagen matrix, and is poised for hypoxic adaptation. We derive the composition of cellular neighborhoods providing an explanation for the complex architecture seen in PDAC. Tumor cell neighborhoods stratified based on which side of the classical-to-basal spectrum they fell. For example, cells towards the classical-side of the spectrum had more homogenous tumor neighborhoods that generally lacked intermediate and basal tumor cells. In contrast, cells towards the basal-side of the spectrum were likely to contain classical neighbors. More broadly, neighborhoods along this continuum also contained different proportions of immune and stromal cells, cell densities, collagen deposition, and TME adaptation. These observations support a model where tumor subtypes exist within disparate niches. Lastly, our atlas unifies the catalog of previously reported stromal cells (fibroblasts and stellate cells versus myofibroblastic and inflammatory cancer associated fibroblasts (CAF)) and describes a novel interferon stimulated gene (ISG) resistance signature (ISG.RS) fibroblast. We map the location of these various stromal cells relative to tumor subtypes. For example, myofibroblast CAFs and ISG.RS fibroblasts are enriched near the basal tumor subtype whereas inflammatory CAF abundance increases with distance away from tumor cells. In sum, our atlas provides a lens for stratifying tumor complexity in a cancer cell centric manner and provides a path for testing therapeutic hypothesis in the right cell subtype and context.
 Citation Format: Brian Rabe, Anna Lyubetskaya, Andrew Kavran, Yulong Bai, Andrew Fisher, Hannah Pliner, Alba Font-Tello, Anne Lewin, Ruifeng Hu, Alexandre P Alloy, Yelena Cheng, Chao Dai, Yunfan Fan, Constance Brett, Todd Brett, Lauren Giampapa, Soeren Stahlschmidt, Fayaz Seifuddin, Steven Vasquez Grinnell, Daniel Carrera, Carlos Rios, Tom Lila, Pradeep Kar, Abhishek Shukla, Rachael Bashford Rogers, Lara Heij, Mike Mason, Ryan Golhar, Isaac Neuhaus, Enas Abu Shah, Shivan Sivakumar, Jimena Trillo Tinoco, Benjamin J Chen, Konstantinos J Mavrakis, Eugene Drokhlyansky. In situ multi-modal characterization of pancreatic ductal adenocarcinoma reveals tumor cell identity as a defining factor of the surrounding microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B081.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566e9b3495ee9764c389c226c9c17e824614540e" target='_blank'>
              Abstract B081: In situ multi-modal characterization of pancreatic ductal adenocarcinoma reveals tumor cell identity as a defining factor of the surrounding microenvironment
              </a>
            </td>
          <td>
            Brian Rabe, Anna Lyubetskaya, Andrew Kavran, Yulong Bai, Andrew Fisher, Hannah Pliner, Alba Font-Tello, Anne Lewin, Ruifeng Hu, Alexandre P. Alloy, Yelena Cheng, Chao Dai, Yunfan Fan, Constance Brett, Todd Brett, Lauren Giampapa, Soeren Stahlschmidt, Fayaz Seifuddin, Steven Vasquez Grinnell, Daniel Carrera, Carlos Rios, Tom Lila, Pradeep Kar, Abhishek Shukla, Rachael Bashford Rogers, Lara R. Heij, Mike Mason, R. Golhar, Isaac Neuhaus, E. A. Shah, Shivan Sivakumar, Jimena Trillo Tinoco, Benjamin Chen, Konstantinos J Mavrakis, Eugene Drokhlyansky
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Diffuse Large B-cell Lymphoma (DLBCL) is the most prevalent subtype of non-Hodgkin’s lymphoma for which current therapeutic strategies remain insufficient. The diffuse nature of DLBCL, lacking distinct tissue structures, represents a challenge to elucidate the cellular organization and interactions within the tumor microenvironment (TME). In this study, we applied spatial transcriptomics to identify spatially-resolved gene expression profiles in 10 DLBCL tissue samples, identifying distinct immune cell infiltration and colocalization patterns. These profiles were classified into six cellular ecosystems (Cell-Eco) that differ in cellular composition, functional patterns, and neighborhood characteristics. The spatially-resolved Cell-Eco signatures provided prognostic scores that stratified patients with different overall survival rates. We also found that C1q+ tumor-associated macrophages are the primary cells interacting with malignant B cells and influencing the spatial architecture of the TME. This study provides novel biological insights into the complexity of the TME in DLBCL and highlights the potential prognostic value of its spatial organization. Graphical abstract Key findings Spatial transcriptomics classifies DLBCL tissues based on immune cell infiltration and colocalization patterns. DLBCL tumor microenvironment consists of cellular ecosystems (Cell-Eco) that differ in cellular composition, transcriptomic profiles and neighborhood characteristics. Spatially-resolved Cell-Eco signatures stratify patients with different overall survival. C1q+ tumor-associated macrophages primarily interact with malignant B cells and contribute to the spatial organization of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ebc56c31f2327f9141f086b678169242b28d1d" target='_blank'>
              Spatial transcriptomics unveils immune cellular ecosystems associated with patient survival in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Alba Díaz Herrero, Hector F Pelaez-Prestel, Lucile Massenet-Regad, Maëva Veyssiere, Julien Calvani, Caterina Cristinelli, Jacqueline Lehmann-Che, Véronique Meignin, Catherine Thieblemont, Véronique Blanc, V. Soumelis, P. Tonnerre
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background Smooth muscle cell (SMC) plasticity and phenotypic switching play prominent roles in the pathogenesis of multiple diseases, but their role in tumorigenesis is unknown. We investigated whether and how SMC diversity and plasticity plays a role in tumor angiogenesis and the tumor microenvironment. Methods and Results We use SMC-specific lineage-tracing mouse models and single cell RNA sequencing to observe the phenotypic diversity of SMCs participating in tumor vascularization. We find that a significant proportion of SMCs adopt a phenotype traditionally associated with macrophage-like cells. These cells are transcriptionally similar to ‘resolution phase’ M2b macrophages, which have been described to have a role in inflammation resolution. Computationally predicted by the ligand-receptor algorithm CellChat, signaling from BST2 on the surface of tumor cells to PIRA2 on SMCs promote this phenotypic transition; in vitro SMC assays demonstrate upregulation of macrophage transcriptional programs, and increased proliferation, migration, and phagocytic ability when exposed to BST2. Knockdown of BST2 in the tumor significantly decreases the transition towards a macrophage-like phenotype, and cells that do transition have a comparatively higher inflammatory signal typically associated with anti-tumor effect. Conclusion As BST2 is known to be a poor prognostic marker in multiple cancers where it is associated with an M2 macrophage-skewed TME, these studies suggest that phenotypically switched SMCs may have a previously unidentified role in this immunosuppressive milieu. Further translational work is needed to understand how this phenotypic switch could influence the response to anti-cancer agents and if targeted inhibition of SMC plasticity would be therapeutically beneficial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcad9e00e00b0b1e9b4d096ed61477beb04a0767" target='_blank'>
              BST2 induces vascular smooth muscle cell plasticity and phenotype switching during cancer progression
              </a>
            </td>
          <td>
            Caitlin F. Bell, Richard A. Baylis, N. G. Lopez, W. Ma, Hua Gao, Fudi Wang, Sharika Bamezai, Changhao Fu, Yoko Kojima, Shaunak S. Adkar, Lingfeng Luo, Clint L. Miller, N. Leeper
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Solid tumours exhibit a well‐defined architecture, comprising a differentiated core and a dynamic border that interfaces with the surrounding tissue. This border, characterised by distinct cellular morphology and molecular composition, serves as a critical determinant of the tumour's invasive behaviour. Notably, the invasive border of the primary tumour represents the principal site for intravasation of metastatic cells. These cells, known as circulating tumour cells (CTCs), function as ‘seeds’ for distant dissemination and display remarkable heterogeneity. Advancements in spatial sequencing technology are progressively unveiling the spatial biological features of tumours. However, systematic investigations specifically targeting the characteristics of the tumour border remain scarce. In this comprehensive review, we illuminate key biological insights along the tumour body‐border‐haematogenous metastasis axis over the past five years. We delineate the distinctive landscape of tumour invasion boundaries and delve into the intricate heterogeneity and phenotype of CTCs, which orchestrate haematogenous metastasis. These insights have the potential to explain the basis of tumour invasion and distant metastasis, offering new perspectives for the development of more complex and precise clinical interventions and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f6a6ffd5124276ada6cdb3c914a272e4284a04" target='_blank'>
              Single‐cell and spatial omics unravel the spatiotemporal biology of tumour border invasion and haematogenous metastasis
              </a>
            </td>
          <td>
            Xifu Cheng, Yuke Cao, Xiangyi Liu, Yuanheng Li, Qing Li, Dian Gao, Qiongfang Yu
          </td>
          <td>2024-09-30</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) and more committed progenitors (collectively referred to as HSPCs) emerge from vessels during development, via Endothelial-to-Hematopoietic Transition (EHT). Recently, using the zebrafish embryo, we showed that two EHT cell types emerge from the aorta, raising the question of their subsequent fate. To address this issue, we established a complex pipeline based on single-cell photoconversion and transgenic lines to characterize the transcriptomic profiles of single EHT cell type progenies. We obtained, at unprecedented resolution in the early larva, a cartography of HSPCs and highly diversified differentiated populations, notably NK-like cell types, innate lymphoid cells and early eosinophils. We show that the two EHT cell types previously characterized indeed lead to differentially fated cells, with significant differences in thymus colonization and T-lymphoid lineage commitment. Using HSPC signatures retrieved from our datasets – namely gata2b and cd34/podocalyxin -, and to address niches, we performed in situ gene expression analyses via RNAscope. Unexpectedly, we unveil a niche contacting the supra-intestinal artery. Finally, integration with previous datasets reveal that our populations contain potential developmental HSCs bearing signatures highly similar with adult HSCs. Summary Statement Single cell photoconversion of emerging hematopoietic precursor cells and transcriptomics unravel the diversity of hematopoietic stem and progenitor cell populations and homing in developmental niches in toto, during zebrafish development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f32a16581b4a45a2a208a9aad0febcec0dcd9d0" target='_blank'>
              Single-cell and in situ spatial analyses reveal the diversity of newly born hematopoietic stem cells and of their niches
              </a>
            </td>
          <td>
            Léa Torcq, Catherine Vivier, Sandrine Schmutz, Y. Loe-Mie, Anne A. Schmidt
          </td>
          <td>2024-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Recent progress in multiplexed tissue imaging is advancing the study of tumor microenvironments to enhance our understanding of treatment response and disease progression. Cellular neighborhood analysis is a popular computational approach for these complex image data. Despite its popularity, there are significant challenges, including high computational demands that limit feasibility for large-scale applications and the lack of a principled strategy for integrative analysis across images. This absence hampers the precise and consistent identification of spatial features and tracking of their dynamics over disease progression. To overcome these challenges, we introduce SpaTopic, a spatial topic model designed to decode high-level spatial architecture across multiplexed tissue images. This algorithm integrates both cell type and spatial information within a topic modelling framework, originally developed for natural language processing and adapted for computer vision. Spatial information is incorporated into the flexible design of documents, representing densely overlapping regions in images. The model employs an efficient collapsed Gibbs sampling algorithm for both statistical and computational inference. We benchmarked the performance against five state-of-the-art algorithms through various case studies using different single-cell spatial transcriptomic and proteomic imaging platforms across different tissue types. Our findings demonstrate that SpaTopic consistently identifies biologically and clinically significant spatial “topics” such as tertiary lymphoid structures (TLSs) and tracks dynamic changes in spatial features over disease progression. Its computational efficiency and broad applicability across various molecular imaging platforms will enhance the analysis of large-scale tissue imaging datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d101a10feced9713be21331ff827f237ec17306" target='_blank'>
              Decoding Spatial Tissue Architecture: A Scalable Bayesian Topic Model for Multiplexed Imaging Analysis
              </a>
            </td>
          <td>
            Xiyu Peng, J. Smithy, M. Yosofvand, Caroline E. Kostrzewa, MaryLena Bleile, Fiona D Ehrich, Jasme Lee, M. Postow, M. K. Callahan, K. Panageas, Ronglai Shen
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="T cells are the central players in antitumor immunity, and effective tumor killing depends on their ability to infiltrate into the tumor microenvironment (TME) while maintaining normal cytotoxicity. However, late-stage tumors develop immunosuppressive mechanisms that impede T cell movement and induce exhaustion. Investigating T cell migration in human tumors in vivo could provide novel insights into tumor immune escape, although it remains a challenging task. In this study, we developed ReMiTT, a computational method that leverages spatial transcriptomics data to track T cell migration patterns within tumor tissue. Applying ReMiTT to ovarian tumor samples, we identified key genes and pathways that enriched on algorithm-identified T cell migration trails, including leukocyte chemotaxis, cell-cell adhesion, and ECM remodeling. We also characterized the phenotypes of T cells on the migrating trails, suggesting that regulatory T cells may accompany cytotoxic T cells during migration. Our findings provide a novel approach to studying T cell migration and interactions within the TME, offering new insights into tumor-immune dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4223fe92cd99ee6cca24d42a0934cd6e2610c25" target='_blank'>
              Computational Identification of Migrating T cells in Spatial Transcriptomics Data
              </a>
            </td>
          <td>
            Lin Zhong, Bo Li, Siyuan Zhang, Qiwei Li, Guanghua Xiao
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Effective cancer immunotherapies restore anti-tumor immunity by rewiring cell-cell communication. Treatment-induced changes in communication can be inferred from single-cell RNA-sequencing (scRNA-seq) data, but current methods do not effectively manage heterogeneity within cell types. Here we developed a computational approach to efficiently analyze scRNA-seq-derived, single-cell-resolved cell-cell interactomes, which we applied to determine how agonistic CD40 (CD40ag) alters immune cell crosstalk alone, across tumor models, and in combination with immune checkpoint blockade (ICB). Our analyses suggested that CD40ag improves responses to ICB by targeting both immuno-stimulatory and immunosuppressive macrophage subsets communicating with T cells, and we experimentally validated a spatial basis for these subsets with immunofluorescence and spatial transcriptomics. Moreover, treatment with CD40ag and ICB established coordinated myeloid-T cell interaction hubs that are critical for reestablishing antitumor immunity. Our work advances the biological significance of hypotheses generated from scRNA-seq-derived cell-cell interactomes and supports the clinical translation of myeloid-targeted therapies for ICB-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63442f7112573b18ed8b87addf34034f405e0832" target='_blank'>
              Mapping intratumoral myeloid-T cell interactomes at single-cell resolution reveals targets for overcoming checkpoint inhibitor resistance
              </a>
            </td>
          <td>
            Kate Bridges, Gabriela A. Pizzurro, Alev Baysoy, Janani P. Baskaran, Ziyan Xu, Varsha Mathew, V. Tripple, Michael LaPorte, Koonam Park, W. Damsky, Harriet M Kluger, Rong Fan, S. Kaech, M. Bosenberg, K. Miller-Jensen
          </td>
          <td>2024-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Localised prostate cancers (PCa) are heterogeneous and multifocal, with diverse outcomes. Current prognostic methods are epithelium-centric, overlooking the complex cellular landscape within the tumour microenvironment (TME), which remains incompletely characterised. We performed a comprehensive analysis of cancerous and adjacent-benign cores from 24 patients with hormone therapy-naïve localised PCa using single-cell RNA-sequencing. By integrating copy number variation and transcriptional signatures, we classified epithelial cells across a malignant spectrum, revealing widespread molecular perturbation. We found an expansion of Club cell phenotypes, suggestive of Luminal dedifferentiation. We also performed a detailed annotation of stromal phenotypes, focusing on fibroblasts, and identified a novel peri-neural fibroblast population. Spatial transcriptomics elucidated the precise anatomical distribution of CAFs within the PCa TME. This study provides a valuable foundation for advancing our understanding of PCa pathobiology and developing a comprehensive cellular model of the disease. Statement of Significance Our study leverages single-cell RNA-sequencing and spatial transcriptomics to provide a comprehensive cellular annotation of hormone therapy-naïve localised PCa. We reveal widespread molecular perturbations in epithelial cells and map distinct fibroblast populations to specific anatomical niches. Notably, we identify a novel peri-neural phenotype associated with nerves, which merits further functional characterisation and exploration as a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b9e207aea660a1c4b72622107ccb70821b90a9" target='_blank'>
              Profiling of epithelial functional states and fibroblast phenotypes in hormone therapy-naïve localised prostate cancer
              </a>
            </td>
          <td>
            Eva Apostolov, D. Roden, Holly Holliday, Aurélie S. Cazet, K. Harvey, Hanyun Zhang, Sunny Z. Wu, Sophie van der Leij, Luke A Selth, Nenad Bartonicek, G. Al-Eryani, Mengxiao He, J. Lundeberg, John Reeves, James G Kench, Alison J Potter, Phillip Stricker, Anthony M Joshua, Lisa G. Horvath, Alexander Swarbrick
          </td>
          <td>2024-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Human health depends on complex processes of intercellular interactions and single-cell type-specific functions. Dramatic improvements in single-cell RNA sequencing (scRNA-seq) have opened up the possibility to identify functional states of single cells. Still, spatial methods are crucial for under-standing transcript location in relation to tissue architecture. Here, we con-structed a large-scale mapping strategy for proteins based on scRNA-seq and fluorescent-based multiplex immunohistochemistry (mIHC). We focused on the spatiotemporal landscape of 12 distinct germ cell states in human testis and performed an in-depth characterization of ∼500 germ cell proteins. Quantitative spatial localization data based on a custom-built image analysis pipeline allowed us to cluster proteins according to expression, forming the basis for functional analysis. mRNA and protein expression dynamics showed multiple cases with low levels of co-expression, highlighting the necessity of studying protein levels in single-cell mapping projects. The pre-sented workflow holds promise for proteome-wide tissue studies in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80790c055a4a9b5f49862e97b9882e5929f77890" target='_blank'>
              A spatiotemporal atlas of human spermatogenesis based on single-cell transcriptomics and multiplex antibody imaging
              </a>
            </td>
          <td>
            Feria Hikmet, Loren Méar, Jonas Gustavsson, Gisele Miranda, Cheng Zhang, Borbala Katona, Rutger Schutten, K. von Feilitzen, Mattias Forsberg, Jan-Bernd Stukenborg, Mathias Uhlén, Cecilia Lindskog
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary Phenotypic diversity in tumors makes treatment difficult and leads to poor prognosis. A better understanding of what causes intratumoral heterogeneity and differences in cells that are within the same tumor will advance therapeutic options and improve patient outcomes. This review will summarize recent studies that have examined the various cancer cell phenotypes along the epithelial–mesenchymal spectrum present in tumors and how epigenetic factors and the mechanical properties of tumor tissue mediate these phenotypes. This review will also discuss the therapeutic implications of intratumoral heterogeneity, such as drug resistance and tumor recurrence, and highlight opportunities to address current limitations in cancer treatments. Abstract The tumor microenvironment comprises various cell types and experiences dynamic alterations in physical and mechanical properties as cancer progresses. Intratumoral heterogeneity is associated with poor prognosis and poses therapeutic challenges, and recent studies have begun to identify the cellular mechanisms that contribute to phenotypic diversity within tumors. This review will describe epithelial–mesenchymal (E/M) plasticity and its contribution to phenotypic heterogeneity in tumors as well as how epigenetic factors, such as histone modifications, histone modifying enzymes, DNA methylation, and chromatin remodeling, regulate and maintain E/M phenotypes. This review will also report how mechanical properties vary across tumors and regulate epigenetic modifications and E/M plasticity. Finally, it highlights how intratumoral heterogeneity impacts therapeutic efficacy and provides potential therapeutic targets to improve cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b710fff2d564553c94ffb93440614a32c0f2e22e" target='_blank'>
              Epithelial–Mesenchymal Plasticity and Epigenetic Heterogeneity in Cancer
              </a>
            </td>
          <td>
            Jessica L Sacco, Esther W. Gomez
          </td>
          <td>2024-09-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metabolic enzymes perform life-sustaining functions in various cellular compartments. Anecdotally, metabolic activity is observed to vary between genetically identical cells, which impacts drug resistance, differentiation, and immune cell activation. However, no large-scale resource systematically reporting metabolic cellular heterogeneity exists. Here, we leverage imaging-based single-cell spatial proteomics to reveal the extent of non-genetic variability of the human enzymatic proteome, as a proxy for metabolic states. Nearly two fifths of enzymes exhibit cell-to-cell variable expression, and half localize to multiple cellular compartments. Metabolic heterogeneity arises largely autonomously of cell cycling, and individual cells reestablish these myriad metabolic phenotypes over several cell divisions. These results establish that cell-to-cell enzymatic heterogeneity is an organizing principle of cell biology and one that may rewire our understanding of drug resistance, treatment design, and other aspects of medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fc03cc47c0d94e752c4532dd652b1361ad12454" target='_blank'>
              Dissecting autonomous enzyme variability in single cells
              </a>
            </td>
          <td>
            Christian Gnann, Anthony J. Cesnik, A. Sigaeva, Trang Le, Sanem Sariyar, Diana Mahdessian, Rutger Schutten, Preethi Raghavan, Manuel D Leonetti, Cecilia Lindskog, Mathias Uhlén, Ulrika Axelsson, Emma Lundberg
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54370344e70d12f9e093a0f96e506dd839c6df0" target='_blank'>
              Deciphering breast cancer dynamics: insights from single-cell and spatial profiling in the multi-omics era
              </a>
            </td>
          <td>
            Xin Xiong, Xin Wang, Cui-Cui Liu, Zhi-Ming Shao, Ke-Da Yu
          </td>
          <td>2024-09-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background & Aims: Hepatocellular carcinoma (HCC) develops in the context of chronic inflammation, however, the opposing roles the immune system plays in both the development and control of tumours is not fully understood. Mapping immune cell interactions across the distinct tissue regions could provide greater insight into the role individual immune populations have within tumours. Approach & Results: Using highly multiplexed imaging mass cytometry, we have mapped the immune landscape of tumour, invasive margin, and adjacent non-tumour regions across sixteen resected tumours comprising of 144 regions of interest and over 10,000 parameters. We show immune cell densities remain largely consistent across these three regions, except for subsets of monocyte-derived macrophages which are enriched within the tumours. Mapping cellular interactions across these regions in an unbiased manner identifies immune neighbourhoods comprised of tissue-resident T cells, dendritic cells, and various macrophage populations around perivascular spaces. Importantly, we identify multiple immune cells within these neighbourhoods interacting with VEGF+ perivascular macrophages. Conclusions: Cellular interactions, but not cell densities, differ between intratumoural and adjacent non-tumour regions in HCC. Unique intratumoural immune neighbourhoods around the perivascular space points to an altered landscape within tumours. Enrichment of VEGF+ perivascular macrophages within these tumours could play a key role in angiogenesis and vascular permeability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f97ce36e43e0dfde11d3ba7e6fa851a271fe9a2" target='_blank'>
              Spatial mapping of the HCC landscape identifies unique intratumoral perivascular-immune neighborhoods
              </a>
            </td>
          <td>
            Felix Marsh-Wakefield, C. Santhakumar, A. L. Ferguson, T. Ashhurst, Joo-Shik Shin, Fiona H. X. Guan, Nicholas J. Shields, Barry J. Platt, Givanna H. Putri, Ruta Gupta, Michael Crawford, Carlo Pulitano, C. Sandroussi, Jerome M Laurence, Ken Liu, Geoffrey W McCaughan, U. Palendira
          </td>
          <td>2023-09-15</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4200779de8ffeea14186de4448271c5a8d0e9f0" target='_blank'>
              Developmental-status-aware transcriptional decomposition establishes a cell state panorama of human cancers
              </a>
            </td>
          <td>
            Yikai Luo, Han Liang
          </td>
          <td>2024-10-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Breast cancer is one of the most prevalent cancers in women globally. Its treatment and prognosis are significantly influenced by the tumor microenvironment and tumor heterogeneity. Precision therapy enhances treatment efficacy, reduces unwanted side effects, and maximizes patients’ survival duration while improving their quality of life. Spatial transcriptomics is of significant importance for the precise treatment of breast cancer, playing a critical role in revealing the internal structural differences of tumors and the composition of the tumor microenvironment. It offers a novel perspective in studying the spatial structure and cell interactions within tumors, facilitating more effective personalized treatments for breast cancer. This article will summarize the latest findings in the diagnosis and treatment of breast cancer from the perspective of spatial transcriptomics, focusing on the revelation of the tumor microenvironment, identification of new therapeutic targets, enhancement of disease diagnostic accuracy, comprehension of tumor progression and metastasis, assessment of drug responses, creation of high-resolution maps of tumor cells, representation of tumor heterogeneity, and support for clinical decision-making, particularly in elucidating the tumor microenvironment, tumor heterogeneity, immunotherapy and their correlation with clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1c95c6cdf84e47a145e58d9ad6429fccbeca441" target='_blank'>
              Spatial transcriptomics: a new frontier in accurate localization of breast cancer diagnosis and treatment
              </a>
            </td>
          <td>
            Yang Zhang, Shuhua Gong, Xiaofei Liu
          </td>
          <td>2024-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Deciphering shared features between patients through unsupervised analyses of tumor single-cell transcriptomes is hindered by the predominant clustering of malignant cells based on the patient’s tumor of origin. In contrast, cancer-associated non-malignant cells within tumors cluster according to their somatic cell type (e.g. macrophage, fibroblast, astrocyte), regardless of the individual patient. We investigated the origin of this contrasting clustering behavior using computational analyses and novel data sampling techniques across 160 tumors representing 9 distinct cancer types. This led to the unexpected disclosing of tumor-specific malignant cell types. We demonstrate that tumor-driven malignant cell clustering is independent from technical or computational biases, nor is it reducible to defined gene sets with tumor-specific expression across all examined cancers. On the opposite, we unveil redundant information dispersed across the transcriptome that encodes a unique identity shared by the malignant cells within each tumor. Additionally, we found a similar genomic encoding for the identity of normal cell types. Finally, we demonstrate that malignant cell identities are maintained across space and over time, as are the identities of normal somatic cells types. These findings suggest the establishment of a distinct type of malignant cells within each tumor, robustly and diffusively encoded across the entire transcribed genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb2c0940cbf36006f73fd3a210e9572d4abd3013" target='_blank'>
              A unique malignant cell type per patient tumor is encoded in each cancer cell transcriptome
              </a>
            </td>
          <td>
            M. Saurty-Seerunghen, Elias A. El-Habr, Yossi Eliaz, Léa Bellenger, Christophe Antoniewski, Hervé Chneiweiss, M. Junier
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Bone marrow (BM) is a common site for solid tumor metastasis, often causing poor outcome. Here, we define the characteristics of BM-disseminated tumor cells (DTCs) using neuroblastoma as a model. We combined single-cell RNA-sequencing (scRNA-seq) and cell-surface protein analysis using 7 paired BM and primary tumor (PT) samples and found that DTCs contain a higher percentage of cycling cells and higher expression of neurodevelopmental genes compared to corresponding PT cells. In 6 patients, the copy number variation profile differed between PT cells and DTCs, indicating spatial heterogeneity. Within the BM, we detected dormant DTCs with potentially reduced chemosensitivity; this population contained cells expressing low levels of the immunotherapeutic antigen GD2 and increased NGFR expression. In conclusion, we characterized DTCs that are particularly challenging to target, offering new avenues for developing therapeutic strategies designed to target all subpopulations within the highly complex metastatic site, thereby preventing the development of drug-resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fc1c99c4919cd41656eef13d151b98e6e5eb661" target='_blank'>
              Metastatic tumor cells in bone marrow differ from paired neuroblastoma tumor and contain subsets with therapy-resistant characteristics
              </a>
            </td>
          <td>
            Caroline Hochheuser, A. Boltjes, Kaylee M. Keller, Simon Tol, Marieke van de Mheen, Carolina Pita Barros, Zeinab van Gestel-Fadaie, André B. P. van Kuilenburg, Sander van Hooff, C. Voermans, Jan J. Molenaar, G. Tytgat, Ilse Timmerman
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Tumor tissues exhibit a complex spatial architecture within the tumor microenvironment (TME). Spatially resolved transcriptomics (SRT) is promising for unveiling the spatial structures of the TME at both cellular and molecular levels, but identifying pathology-relevant spatial domains remains challenging. Here, we introduce SpaTopic, a statistical learning framework that harmonizes spot clustering and cell-type deconvolution by integrating single-cell transcriptomics and SRT data. Through topic modeling, SpaTopic stratifies the TME into spatial domains with coherent cellular organization, facilitating refined annotation of the spatial architecture with improved performance. We assess SpaTopic across various tumor types and show accurate prediction of tertiary lymphoid structures and tumor boundaries. Moreover, marker genes derived from SpaTopic are transferrable and can be applied to mark spatial domains in other datasets. In addition, SpaTopic enables quantitative comparison and functional characterization of spatial domains across SRT datasets. Overall, SpaTopic presents an innovative analytical framework for exploring, comparing, and interpreting tumor SRT data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff3877840f97f1fe063aecd16595236351a205d9" target='_blank'>
              SpaTopic: A statistical learning framework for exploring tumor spatial architecture from spatially resolved transcriptomic data
              </a>
            </td>
          <td>
            Yuelei Zhang, Bianjiong Yu, Wenxuan Ming, Xiaolong Zhou, Jin Wang, Dijun Chen
          </td>
          <td>2024-09-27</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and aggressive primary brain cancer in adults, characterized by robust infiltrative growth and invasion within the neural microenvironment. While various stromal cells, including astrocytes, microglia, and vascular cells in the microenvironment, communicate with GBM cells to promote growth and invasion, relatively less is known about another critical component of the tumor stroma: the extracellular matrix (ECM). The matrisome comprises hundreds of ECM genes and affiliated adhesion and signaling components; however, little is understood about (i) the repertoire of matrisome genes and proteins expressed in GBM, (ii) the stromal and cancer cells of origin that express different matrisome factors, and (iii) how these components are interconnected to drive tumor malignancy. Here, we have used in situ single-cell transcriptomics to analyze the spatial expression of nearly 350 genes that constitute the human brain matrisome. Our efforts have identified several differentially expressed matrisome genes involved in ECM adhesion and signaling between cancer and stromal cells. Additionally, matrisome genes associated with low-grade glioma pathogenesis and progression to high-grade GBM have been identified and spatially mapped. Collectively, these experiments reveal the single-cell origins of differentially expressed ECM biomarkers as well as their spatial expression patterns in brain tumors. These findings may lead to new therapeutic efforts to inhibit matrisome components to benefit patients with malignant brain cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db27897e2873e1f72dc0c04bc6970233914ac8ad" target='_blank'>
              In Situ Single-Cell Spatial Profiling of Matrisome Gene Expression in Glioblastoma
              </a>
            </td>
          <td>
            Arpan De, Santiago A. Forero, Sumod Sebastian, John E. Morales, Marisol De La Fuente-Granada, Jason T Huse, L. Ballester, Jeffrey S. Weinberg, Frederick F. Lang, Kadir C. Akdemir, Joseph H. McCarty
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains an intractable cancer characterized by significant inter- and intra-patient heterogeneity. Recent research has established connections between key genetic drivers and distinct transcriptional profiles in GBM, yet the interactions between genetic changes and the transcriptional and spatial heterogeneity remain unclear. This study used genetically engineered mouse models, coupled with spatially resolved and single-cell RNA sequencing, to explore how specific genetic alterations influence the tumor microenvironment (TME) and cellular responses spatially.



 We used nestin-Tva C57BL/6 mice and generated de novo GBM using overexpression of oncogenes in combinations with short hairpin RNAs (shRNAs) or deletion to various tumor suppressors, including Tp53, Cdkn2A, and Pten. We modeled key driver mutations for GBM: PDGFB, NF1, and EGFR. We conducted single-cell RNA sequencing, combined with spatially resolved transcriptomics and proteomics. Data integration techniques, including horizontal (mutual nearest neighbors) and vertical (weighted nearest neighbors) approaches, were used and analyzed with the SPATA2 software. Advanced computational methods, such as graph neural networks and single-cell deconvolution, were applied to examine GBM genotypes and the spatial variability within tumors and their TMEs.



 Our findings revealed persistent cellular heterogeneity among tumor cells under various genetic conditions. However, specific genetic alterations influenced the cells toward dominant transcriptional states. NF1 and EGFR mutations resulted in shifts towards MES and AC-like states, respectively, unlike PDGFB, which promoted a NPC/OPC-like state. These transcriptional shifts were strongly associated with distinct microenvironmental changes, such as enhanced neuronal signaling in PDGFB-driven GBMs versus pronounced immune infiltration in NF1 and EGFR models. Utilizing weighted co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA) alongside spatial-niche deconvolution, we identified a mesenchymal transition leading to an immunosuppressive TME in NF1-driven GBMs and increased lymphocyte presence in EGFR-amplified GBMs, especially around blood vessels. Similar spatial patterns were observed in human spatially resolved transcriptomic data, highlighting the complexity and variability of genetic drivers in human GBMs.



 Our genetically engineered mouse models provide a valuable platform for investigating GBM with defined driver mutations. By leveraging sophisticated analytical techniques and computational modeling, this study underscores the potential to elucidate the interactions between genetic drivers and the tumor microenvironment. Our approach paves the way for developing targeted therapies and precision medicine strategies in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0056b4191c0de2f537409a208d3f06f5659352" target='_blank'>
              OS07.7.A DISSECTING SPATIAL DIVERSITY OF GLIOBLASTOMA GENOTYPES
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Hambardzumyan, D. Heiland
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Combining molecular profiling with imaging techniques has advanced the field of spatial biology, offering new insights into complex biological processes. Focusing on diffuse IDH-mutated low-grade glioma, this study presents a workflow for Spatial Multi-omics Integration, SMINT, specifically combining spatial transcriptomics and spatial metabolomics. Our workflow incorporates both existing and custom-developed computational tools to enable cell segmentation and registration of spatial coordinates from both modalities to a common coordinate framework. During our investigation of cell segmentation strategies, we found that nuclei-only segmentation, while containing only 40% of segmented cell transcripts, enables accurate cell type annotation, but does not account for multinucleated cells. Our integrative workflow including cell-morphology segmentation identified distinct cellular neighborhoods at the infiltrating edge of gliomas, which were enriched in multinucleated and oligodendrocyte-lineage tumor cells, that may drive tumor invasion into the normal cortical layers of the brain. Highlights Alignment and integrated analysis of spatial transcriptomic and metabolomic data Nuclei-only and cell-morphology segmentations are concordant for cell annotation Spatially distinct regions are conserved in transcriptomic and metabolomic datasets Multi-omic exploration of glioma leading edge identifies novel biological features">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d0727cfef9051214008921002b4929b8b054e5e" target='_blank'>
              An integrative spatial multi-omic workflow for unified analysis of tumor tissue
              </a>
            </td>
          <td>
            J. Kriel, J. Moffet, Tianyao Lu, Oluwaseun E. Fatunla, Vinod K. Narayana, Adam Valkovic, Ana Maluenda, Malcolm J. McConville, Ellen Tsui, J. Whittle, Sarah A. Best, S. Freytag
          </td>
          <td>2024-10-18</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20edd8d2bb6126475fe06f906514312b49ac959d" target='_blank'>
              Integrating histopathology and transcriptomics for spatial tumor microenvironment profiling in a melanoma case study
              </a>
            </td>
          <td>
            Óscar Lapuente-Santana, J. Kant, Federica Eduati
          </td>
          <td>2024-11-07</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Recent advancements in spatial biology have revolutionized our understanding of the organization and functional dynamics of cells and tissues. In this study, we applied Molecular Pixelation (MPX), a single-cell spatial proteomics assay, to investigate the modulation of the cell surface proteome in an in vitro drug screening model using the ETV6::RUNX1 acute lymphoblastic leukemia (ALL) cell line, Reh. Specifically, we focused on the in vitro response to fludarabine, a chemotherapeutic agent used prior to allogenic stem cell transplantation and chimeric antigen receptor (CAR)-T cell therapy in high-risk, refractory, or relapsed ALL patients. Using MPX, we quantified changes in protein abundance, spatial distribution, and colocalization of 76 targeted cell surface proteins in Reh cells before and after fludarabine treatment. Our analysis revealed 25 proteins with altered abundance, 24 proteins with increased polarity, and 138 protein pairs with modified colocalization following treatment. Notably, the tetraspanins CD82 and CD53, which are known for their roles in chemotherapy resistance, exhibited increased abundance, polarization, and colocalization post-treatment, suggesting their potential as a therapeutic scaffold. These findings underscore the unique ability of spatially resolved single-cell proteomics to uncover nuanced cellular responses that would otherwise remain undetected.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f5a352c544200f32c43afeccf0c1c6f723f3d81" target='_blank'>
              Mapping the Spatial Proteome of Leukemia Cells Undergoing Fludarabine Treatment
              </a>
            </td>
          <td>
            , Henrik Gezelius, Anna Pia Enblad, Anders Lundmark, Mariya Lysenkova Wiklander, O. Krali, Martin Åberg, Arja Harila, A. Raine, Claes Andersson, J. Nordlund
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f3fb8761d45bc6d0ac56290c268361e8b70736" target='_blank'>
              Dissecting human adipose tissue heterogeneity using single‐cell omics technologies
              </a>
            </td>
          <td>
            G. Di Rocco, A. Trivisonno, Giovanni Trivisonno, Gabriele Toietta
          </td>
          <td>2024-09-27</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cranial neural crest (NC) cells, which can migrate, adopt multiple fates, and form most of the craniofacial skeleton, are an excellent model for studying cell fate decisions. Using time-resolved single-cell multi-omics, spatial transcriptomics, and systematic Perturb-seq, we fully deciphered zebrafish cranial NC programs, including 23 cell states and three spatial trajectories, reconstructed and tested the complete gene regulatory network (GRN). Our GRN model, combined with a novel velocity-embedded simulation method, accurately predicted functions of all major regulons, with over a 3-fold increase in correlation between in vivo and in silico perturbations. Using our new approach based on regulatory synchronization, we discovered a post-epithelial-mesenchymal-transition endothelial-like program crucial for migration, identified motif coordinators for dual-fate priming, and quantified lineage-specific cooperative transcription factor functions. This study provides a comprehensive and validated NC regulatory landscape with unprecedented resolution, offering general regulatory models for cell fate decisions in vertebrates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d752e40a594fbc48d93390258fcd2d7f82846c" target='_blank'>
              Single-cell multi-omics, spatial transcriptomics and systematic perturbation decode circuitry of neural crest fate decisions
              </a>
            </td>
          <td>
            Zhiyuan Hu, Sarah Mayes, Weixu Wang, José M. Santos-Pereira, F. Theis, Tatjana Sauka-Spengler
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5b58b19c98f4afe3952f6247f4898d29bd39cb6" target='_blank'>
              Tumour evolution and microenvironment interactions in 2D and 3D space
              </a>
            </td>
          <td>
            Chia-Kuei Mo, Jingxian Liu, Siqi Chen, E. Storrs, A. L. N. Targino da Costa, Andrew Houston, M. Wendl, R. Jayasinghe, Michael D. Iglesia, Cong Ma, John M Herndon, Austin N. Southard-Smith, Xinhao Liu, Jacqueline L Mudd, A. Karpova, Andrew T Shinkle, S. Goedegebuure, Abdurrahman Abdelzaher, Peng Bo, L. Fulghum, Samantha Livingston, Metin Balaban, Angela Hill, Joseph E. Ippolito, V. Thorsson, Jason M. Held, Ian S. Hagemann, Eric H Kim, Peter Bayguinov, Albert H. Kim, Mary M Mullen, K. Shoghi, Tao Ju, M. Reimers, Cody Weimholt, Liang-I Kang, Sidharth V. Puram, D. Veis, Russell Pachynski, K.C. Fuh, M. Chheda, William E Gillanders, Ryan C. Fields, Benjamin J. Raphael, Feng Chen, Li Ding
          </td>
          <td>2024-10-01</td>
          <td>Nature</td>
          <td>2</td>
          <td>56</td>
        </tr>

        <tr id="The interaction of glioblastoma (GB) and microglia is critical due to its implications for tumor progression, immune response modulation, and potential therapeutic strategies. However, the role of microglia in GB pathogenesis remains unclear, especially regarding the in vivo dynamics of their interplay. Performing three-photon imaging in an autochthonous, immunocompetent mouse GB model, we examined tumor/microglia dynamics within previously inaccessible regions at the GB far infiltration zone in the corpus callosum. Initially, microglia increased tissue surveillance upon encountering GB-cells in sparsely infiltrated areas. In contrast, when GB-cell density increased, microglia reduced surveillance, suggesting a biphasic response to tumor invasion. Additionally, microglia were not uniformly attracted to infiltrating GB-cells; only a subset moved directionally toward them within a defined spatial range. This study provides insight into the heterogeneity of the immune response to tumor invasion and the dynamics of microglia-GB interactions in vivo. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39820cb3045ee584de7bf3d731fda31accb858a" target='_blank'>
              Dynamics of microglia-glioblastoma crosstalk at the far infiltration zone
              </a>
            </td>
          <td>
            F. Nebeling, Falko Fuhrmann, M. Mittag, Fabrizio Mussachio, Nala Gockel, Lea L Friker, S. Leonardelli, S. Filser, A. Deli, Miriam Stork, D. Bano, Torsten Pietsch, S. Parrinello, M. Hölzel, U. Herrlinger, P. Salomoni, M. Fuhrmann
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Background Breast cancer's complex transcriptional landscape requires an improved understanding of cellular diversity to identify effective treatments. The study of genetic variations among breast cancer subtypes at single‐cell resolution has potential to deepen our insights into cancer progression. Methods In this study, we amalgamate single‐cell RNA sequencing data from patient tumours and matched lymph metastasis, reduction mammoplasties, breast cancer patient‐derived xenografts (PDXs), PDX‐derived organoids (PDXOs), and cell lines resulting in a diverse dataset of 117 samples with 506 719 total cells. These samples encompass hormone receptor positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple‐negative breast cancer (TNBC) subtypes, including isogenic model pairs. Herein, we delineated similarities and distinctions across models and patient samples and explore therapeutic drug efficacy based on subtype proportions. Results PDX models more closely resemble patient samples in terms of tumour heterogeneity and cell cycle characteristics when compared with TNBC cell lines. Acquired drug resistance was associated with an increase in basal‐like cell proportions within TNBC PDX tumours as defined with SCSubtype and TNBCtype cell typing predictors. All patient samples contained a mixture of subtypes; compared to primary tumours HR+ lymph node metastases had lower proportions of HER2‐Enriched cells. PDXOs exhibited differences in metabolic‐related transcripts compared to PDX tumours. Correlative analyses of cytotoxic drugs on PDX cells identified therapeutic efficacy was based on subtype proportion. Conclusions We present a substantial multimodel dataset, a dynamic approach to cell‐wise sample annotation, and a comprehensive interrogation of models within systems of human breast cancer. This analysis and reference will facilitate informed decision‐making in preclinical research and therapeutic development through its elucidation of model limitations, subtype‐specific insights and novel targetable pathways. Key points Patient‐derived xenografts models more closely resemble patient samples in tumour heterogeneity and cell cycle characteristics when compared with cell lines. 3D organoid models exhibit differences in metabolic profiles compared to their in vivo counterparts. A valuable multimodel reference dataset that can be useful in elucidating model differences and novel targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b1b9e5e204010f0ff424f3f3d4b1b7c96f99e7a" target='_blank'>
              Single‐cell transcriptional atlas of human breast cancers and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, Brian D Lehmann, Jane E. Visvader, J. C. Harrell
          </td>
          <td>2024-10-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>162</td>
        </tr>

        <tr id="Predicting the progression of solid cancers based solely on genetics is challenging due to the influence of the tumor microenvironment (TME). For colorectal cancer (CRC), tumors deficient in mismatch repair (dMMR) are more immune infiltrated than mismatch repair proficient (pMMR) tumors and have better prognosis following resection. Here we quantify features of the CRC TME by combining spatial profiling with genetic analysis and release our findings via a spatially enhanced version of cBioPortal that facilitates multi-modal data exploration and analysis. We find that ∼20% of pMMR tumors exhibit similar levels of T cell infiltration as dMMR tumors and that this is associated with better survival but not any specific somatic mutation. These T cell-infiltrated pMMR (tipMMR) tumors contain abundant cells expressing PD1 and PDL1 as well as T regulatory cells, consistent with a suppressed immune response. Thus, like dMMR CRC, tipMMR CRC may benefit from immune checkpoint inhibitor therapy. SIGNIFICANCE pMMR tumors with high T cell infiltration and active immunosuppression are identifiable with a mid-plex imaging assay whose clinical deployment might double the number of treatment-naïve CRCs eligible for ICIs. Moreover, the low tumor mutational burden in tipMMR CRC shows that MMR status is not the only factor promoting immune infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb91d144afd07ee13d239cb16bf5689a34f70783" target='_blank'>
              Integrating spatial profiles and cancer genomics to identify immune-infiltrated mismatch repair proficient colorectal cancers
              </a>
            </td>
          <td>
            Jeremiah Wala, Ino de Bruijn, S. Coy, Andréanne Gagné, Sabrina Chan, Yu-An Chen, J. Hoffer, J. Muhlich, N. Schultz, S. Santagata, P. Sorger
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bca16ff9172b241a4b47443f7584ae07d099226" target='_blank'>
              Transcriptomic and proteomic spatial profiling of pediatric and adult diffuse midline glioma H3 K27-Altered
              </a>
            </td>
          <td>
            Sudarshawn Damodharan, Jack M. Shireman, Elliot Xie, Emily Distler, Christina Kendziorski, Mahua Dey
          </td>
          <td>2024-09-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d524bebda002b9fe5f1ee5e915d5586778f0b5b3" target='_blank'>
              A multi-modal single-cell and spatial expression map of metastatic breast cancer biopsies across clinicopathological features.
              </a>
            </td>
          <td>
            Johanna Klughammer, Daniel L. Abravanel, Åsa Segerstolpe, Timothy R Blosser, Y. Goltsev, Yi Cui, Daniel R Goodwin, Anubhav Sinha, Orr Ashenberg, M. Slyper, S. Vigneau, Judit Jané-Valbuena, Shahar Alon, Chiara Caraccio, Judy Chen, Ofir Cohen, Nicole Cullen, L. Dellostritto, Danielle Dionne, Janet L. Files, Allison Frangieh, K. Helvie, Melissa E Hughes, Stephanie Inga, Abhay Kanodia, Ana Lako, Colin Mackichan, Simon Mages, Noa Moriel, Evan Murray, Sara Napolitano, Kyleen Nguyen, Mor Nitzan, Rebecca Ortiz, Miraj Patel, K. Pfaff, C. Porter, Asaf Rotem, Sarah Strauss, Robert Strasser, A. Thorner, M. Turner, Isaac Wakiro, Julia Waldman, Jingyi Wu, Jorge Gómez Tejeda Zañudo, Diane Zhang, Nancy U Lin, S. Tolaney, Eric P. Winer, E. Boyden, Fei Chen, Garry Nolan, Scott J. Rodig, X. Zhuang, O. Rozenblatt-Rosen, Bruce E. Johnson, Aviv Regev, Nikhil Wagle
          </td>
          <td>2024-10-30</td>
          <td>Nature medicine</td>
          <td>1</td>
          <td>96</td>
        </tr>

        <tr id="Serrated lesions are common precancerous pathways in colorectal cancer (CRC), but the process by which they progress to malignancy remains unclear. We aimed to elucidate this progression through a single-cell RNA landscape. We conducted single-cell RNA sequencing on three normal colonic tissues and fifteen SLs (including HPs, SSLs, SSLD, and TSAs) and integrated these data with datasets containing tumor samples. We identified three invasive malignant epithelial cell subtypes related to CRC progression: SLC1, SLC2, and tumor cell. SLC1, specific to SSLs, is involved in cell proliferation and shows a continuum of malignancy in gene expression. TSA-specific SLC2 exhibited FOXQ1 upregulation and active EMT, indicating invasiveness. The trajectory analysis showed that HPs do not progress to cancer, and different SL types are linked to the MSI status of advanced CRCs. We validated molecular drivers in premalignant lesions and later carcinogenesis. In the tumor microenvironment, CAF and pre-CAF fibroblast subtypes associated with progression were identified. During the premalignant stage, SLC1 triggered CD8+ T cell responses, while at the advanced stage, CAFs promoted tumor invasion and metastasis via FN1-CD44, influencing tumor progression and the treatment response. Our findings highlight transcriptional changes across serrated pathway stages, aiding in early CRC diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f52080cc10a19d5a436d2837e1d1e6040efb30" target='_blank'>
              Single-Cell Transcriptomics Reveals Cellular Heterogeneity and Drivers in Serrated Pathway-Driven Colorectal Cancer Progression
              </a>
            </td>
          <td>
            Jiahui Wang, Yu Zhang, Xinyi Chen, Qi Sheng, Junluo Yang, Yuyao Zhu, Yuhang Wang, Fangrong Yan, Jingya Fang
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The abundance and biological contribution of natural killer (NK) cells in cancer are controversial. Here, we aim to uncover clinical relevance and cellular roles of NK cells in colon cancer liver metastasis (CCLM). Here, we integrated single-cell RNA-sequencing, spatial transcriptomics (ST), and bulk RNA-sequencing datasets to investigate NK cells’ biological properties and functions in the microenvironment of primary and liver metastatic tumors. Results were validated through an in vitro co-culture experiment based on bioinformatics analysis. Useing single-cell RNA-sequencing and ST, we mapped the immune cellular landscape of colon cancer and well-matched liver metastatic cancer. We discovered that GZMK+ resting NK cells increased significantly in tumor tissues and were enriched in the tumor regions of both diseases. After combining bulk RNA and clinical data, we observed that these NK cell subsets contributed to a worse prognosis. Meanwhile, KIR2DL4+ activated NK cells exhibited the opposite position and relevance. Pseudotime cell trajectory analysis revealed the evolution of activated to resting NK cells. In vitro experiments further confirmed that tumor-cell-co-cultured NK cells exhibited a decidual-like status, as evidenced by remarkable increasing CD9 expression. Functional experiments finally revealed that NK cells exhibited tumor-activating characteristics by promoting the dissociation of SCF (stem cell factor) on the tumor cells membrane depending on cell-to-cell interaction, as the supernatant of the co-culture system enhanced tumor progression. In summary, our findings revealed resting NK cells exhibited a clinical relevance with CCLM, which may be exploited for novel strategies to improve therapeutic outcomes for patients with CCLM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be15cf8ef30b7e56c663c4156f8b61fd676de622" target='_blank'>
              Resting natural killer cells promote the progress of colon cancer liver metastasis by elevating tumor-derived stem cell factor
              </a>
            </td>
          <td>
            Chenchen Mao, Yanyu Chen, Dong Xing, Teming Zhang, Yangxuan Lin, Cong Long, Jiaye Yu, Yunhui Luo, Tao Ming, W. Xie, Zheng Han, Dianfeng Mei, Dan Xiang, Mingdong Lu, Xian Shen, Xiangyang Xue
          </td>
          <td>2024-10-10</td>
          <td>eLife</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract The tumor microenvironment of brain metastases has become a focus in the development of immunotherapeutic drugs. However, countless patients with brain metastasis have not experienced clinical benefit. Thus, understanding the immune cell composition within brain metastases and how immune cells interact with each other and other microenvironmental cell types may be critical for optimizing immunotherapy. We applied spatial whole-transcriptomic profiling with extensive multiregional sampling (19–30 regions per sample) and multiplex IHC on formalin-fixed, paraffin-embedded lung cancer brain metastasis samples. We performed deconvolution of gene expression data to infer the abundances of immune cell populations and inferred spatial relationships from the multiplex IHC data. We also described cytokine networks between immune and tumor cells and used a protein language model to predict drug–target interactions. Finally, we performed deconvolution of bulk RNA data to assess the prognostic significance of immune–metastatic tumor cellular networks. We show that immune cell infiltration has a negative prognostic role in lung cancer brain metastases. Our in-depth multiomics analyses further reveal recurring intratumoral immune heterogeneity and the segregation of myeloid and lymphoid cells into distinct compartments that may be influenced by distinct cytokine networks. By using computational modeling, we identify drugs that may target genes expressed in both tumor core and regions bordering immune infiltrates. Finally, we illustrate the potential negative prognostic role of our immune–metastatic tumor cell networks. Our findings advocate for a paradigm shift from focusing on individual genes or cell types toward targeting networks of immune and tumor cells. Significance: Immune cell signatures are conserved across lung cancer brain metastases, and immune–metastatic tumor cell networks have a prognostic effect, implying that targeting cytokine networks between immune and metastatic tumor cells may generate more precise immunotherapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488bc40a00ebddc1b6645e7141cbe9f55941284e" target='_blank'>
              Spatial Multiomics Reveals Intratumoral Immune Heterogeneity with Distinct Cytokine Networks in Lung Cancer Brain Metastases
              </a>
            </td>
          <td>
            Gustav Christensson, M. Bocci, Julhash U. Kazi, Geoffroy Durand, Gustav Lanzing, Kristian Pietras, Hugo Gonzalez Velozo, C. Hagerling
          </td>
          <td>2024-10-14</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Astrocytes comprise ∼50% of all brain cells and present distinct morphological, molecular and functional properties in different brain regions. In glioblastoma (GBM), an aggressive primary brain tumour, tumour-associated astrocytes (TAAs) become activated and exhibit different transcriptomic profiles, morphology and functions supporting disease progression. Heterogeneity and specific roles of TAAs within various regions of tumours are poorly known. Advancements of single-cell and spatial transcriptomics allow to profile tumours at unprecedented resolution revealing cell phenotypes, hidden functionalities and spatial architecture in disease-specific context. We combined spatial transcriptomics and multiple immunofluorescent staining to visualize TAAs heterogeneity and location of various subpopulations in intracranial murine gliomas. Using distinct gene expression profiles, we identified subtypes of TAAs with distinct localization and inferred their specialized functionalities. Gene signatures associated with TAAs reflected their reprograming in the tumour microenvironment (TME), revealed their multiple roles and potential contributing factors shaping the local milieu. Using spatial correlation analysis of the spots, we inferred the interactome of Slc1a2 (encoding a glutamate transporter) with the other markers of TAAs based on segregated areas of the tumour. The designer RGD peptide blocking tumour-microglia communications, alters the spatial distribution of TAAs in experimental gliomas providing insights into potential mechanisms. Spatial transcriptomics combined with multiple staining unveils multiple functional phenotypes of TAAs and interactions within TME. It shows their distinct morphology and unveils different roles in various regions of the tumour. We demonstrate the glioma-induced heterogeneity of TAAs and their adaption to the pharmacologically-induced modification of the immunosuppressive TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2a092533402eea8774b3945747486b079a48e5e" target='_blank'>
              Integration of spatial transcriptomics with immunofluorescence staining reveals spatial heterogeneity and plasticity of astrocytes in experimental glioblastomas
              </a>
            </td>
          <td>
            Mitrajit Ghosh, P. Pilanc-Kudlek, Karol Jacek, Szymon Baluszek, Katarzyna Poleszak, P. Szadkowska, B. Gielniewski, Aleksandra Ellert Miklaszewska, Bozena Kaminska
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Accurately deciphering tumor cellular heterogeneity is crucial for developing effective treatments. While single-cell epigenomics assays offer powerful insights into studying tumor heterogeneity beyond the transcriptional level, their application remains a significant challenge. To address this, we introduce Multimodal-based Analysis of scATAC-Seq data (MAAS), a method designed to identify functional tumor cell subpopulations and reconstruct their lineages from single-cell sequencing assay for transposase-accessible chromatin (scATAC-seq). MAAS uniquely integrates multimodal information, encompassing chromatin accessibility, copy number variations (CNVs), and single-nucleotide variants, through self-expressive multimodal matrix factorization. MAAS outperforms existing methods in accuracy and robustness on both simulated and real datasets. When applied to multiple glioma scATAC-seq datasets, MAAS uncovered a previously hidden tumor cell subpopulation resistant to temozolomide treatment. Furthermore, MAAS effectively detected clinically relevant subpopulations with low CNVs, such as those found in pediatric ependymoma and B-cell lymphoma. Additionally, we developed a novel MAAS-derived clinical signature that provides superior prognostic prediction than traditional clinicopathologic features across multiple cancer types. In summary, MAAS significantly advances the identification of clinically relevant tumor cell subpopulations, thereby accelerating the discovery of potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b29ddaa37d46a0d04b1320e8bee3cd217df7912" target='_blank'>
              Multimodal integration of single cell ATAC-seq data enables highly accurate delineation of clinically relevant tumor cell subpopulations
              </a>
            </td>
          <td>
            Kewei Xiong, Ruofan Ding, Yangmei Qin, Xudong Zou, Jiguang Wang, Chen Yu, Lei Li
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) represents the complexity of interaction between cancer and cells of the tumor microenvironment (TME). Immune cells affect tumor cell behavior, thus driving cancer progression. Cancer-associated fibroblasts (CAFs) are responsible of the desmoplastic and fibrotic reaction by regulating deposition and remodeling of extracellular matrix (ECM). As tumor-promoting cells abundant in PDAC ECM, CAFs represent promising targets for novel anticancer interventions. However, relevant clinical trials are hampered by the lack of specific markers and elusive differences among CAF subtypes. Indeed, while single-cell transcriptomic analyses have provided important information on the cellular constituents of PDACs and related molecular pathways, studies based on the identification of protein markers in tissues aimed at identifying CAF subtypes and new molecular targets result incomplete.Herein, we applied multiplexed Imaging Mass Cytometry (IMC) at single-cell resolution on 8 human PDAC tissues to depict the PDAC composing cells, and profiling immune cells, endothelial cells (ECs), as well as endocrine cells and tumor cells.We focused on CAFs by characterizing up to 19 clusters distinguished by phenotype, spatiality, and interaction with immune and tumor cells. We report evidence that specific subtypes of CAFs (CAFs 10 and 11) predominantly are enriched at the tumor-stroma interface and closely associated with tumor cells. CAFs expressing different combinations of FAP, podoplanin and cadherin-11, were associated with a higher level of CA19-9. Moreover, we identified specific subsets of FAP+ and podoplanin+/cadherin-11+ CAFs enriched in patients with negative prognosis.The present study provides new general insights into the complexity of the PDAC microenvironment by defining phenotypic heterogeneities and spatial distributions of CAFs, thus suggesting different functions of their subtypes in the PDAC microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c403b727bc52805f8503133b3a285d436787caa0" target='_blank'>
              Depicting the cellular complexity of pancreatic adenocarcinoma by Imaging Mass Cytometry: focus on cancer-associated fibroblasts
              </a>
            </td>
          <td>
            M. Erreni, M. R. Fumagalli, Raffaella D’Anna, Mauro Sollai, S. Bozzarelli, G. Nappo, Damiano Zanini, Raffaella Parente, Cecilia Garlanda, L. Rimassa, Luigi Maria Terracciano, S. Biswas, A. Zerbi, Alberto Mantovani, Andrea Doni
          </td>
          <td>2024-11-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfb73fa1be8db27e4aafae1a4087794eca77af2e" target='_blank'>
              Spatial interactions modulate tumor growth and immune infiltration
              </a>
            </td>
          <td>
            S. Marzban, Sonal Srivastava, Sharon Kartika, Rafael Bravo, Rachel Safriel, Aidan Zarski, Alexander R. A. Anderson, Christine H Chung, Antonio L. Amelio, Jeffrey West
          </td>
          <td>2024-09-30</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma is the most hypovascular solid tumor, imposing harsh constraints on tumor metabolism. Cancer cells adapt to reduced access to vascular-derived nutrients and O2 by cooperating with abundant stromal cells. Specifically, critical nutrients including alanine, proline, glycosaminoglycans, branched chain ketoacids, lysophosphatidic acid, and unsaturated fatty acids can be provided by fibroblasts in vitro and in vivo. However, the extent of metabolic support provided by other cell types in the PDAC TME is relatively unexplored. We hypothesized that cell types that are cooperating metabolically with cancer cells will be spatially positioned near cancer cells in these tumors. We performed Xenium spatial transcriptomics on 8 treatment-naive PDAC resections and generated a spatial atlas of nearly 700,000 cells across a range of expected cell types. We noted striking neutrophil aggregates, nucleating in the glands of classical-like malignant cells and elaborating extraglandularly. Cellular neighborhood analysis demonstrates neutrophil aggregation is strongly correlated with malignant cell proliferation and low endothelial content, suggesting neutrophils may be an important source of pro-tumoral, non-vascular nutrients in the PDAC TME. Reanalysis of published scRNAseq data revealed 3 neutrophil metabolic states in human PDAC tumors. Metabolic cluster 7 neutrophils were found only in PDAC patient samples but not healthy controls and were enriched for Type I interferon response as well as pentose phosphate pathway and iron uptake. Metabolic cluster 2 neutrophils were strikingly depleted from PDAC tumor samples compared to blood, suggesting decreased infiltration or increased cell death. Cluster 2 neutrophils were enriched for several catabolic pathways including glycogenolysis, autophagy, and proteasome, and selectively express the NETosis gene PADI4. Multiplexed immunofluorescence demonstrated that neutrophil aggregates in PDAC are citH3+/MPO+ large neutrophil extracellular traps (NETs). We reasoned pancreatic cancer cells could proliferate under tumor-like metabolic stress by repurposing amino acids liberated by autophagy, glucose liberated by glycogenolysis, or simply consuming bulk neutrophil dead cell debris via macropinocytosis. Using tumor interstitial fluid media (TIFM) and 0.5% O2 to mimic tumor-like metabolic stress, we cocultured pancreatic cancer cells with human primary neutrophils and HL60-derived neutrophil-like cells. However, under tumor-like metabolic conditions, coculture with human primary neutrophils failed to substantially increase cancer cell proliferation and NETotic neutrophils were cytotoxic. However, we noticed enhanced neutrophil viability in tumor-like metabolic conditions, suggesting metabolic regulation of neutrophil cell death. Together, these data shed light on the metabolic phenotypes and interactions with cancer cells of neutrophils in the PDAC tumor microenvironment.
 Citation Format: Carson D Poltorack, Sydney M Shaffer, M Celeste Simon. Spatial and single cell transcriptomics uncovers metabolic reprogramming and NETosis of neutrophils in human pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A065.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e97afc3e27a38cdf1cbacb0a0b2dbe91b12facfe" target='_blank'>
              Abstract A065: Spatial and single cell transcriptomics uncovers metabolic reprogramming and NETosis of neutrophils in human pancreatic cancer
              </a>
            </td>
          <td>
            Carson D. Poltorack, Sydney M Shaffer, M. C. Simon
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Enhancers play an essential role in developmental processes by orchestrating the spatial and temporal regulation of gene expression. However, mapping the location of these regulatory elements in the genome and precisely characterizing their spatial and temporal activity remain important challenges. Here we introduce a novel in vivo and in silico method for spatial single-cell enhancer-reporter assays (spatial-scERA) designed to reconstruct the spatial activity of multiple candidate enhancer regions in parallel in a multicellular organism. Spatial-scERA integrates massively parallel reporter assays coupled with single-cell RNA sequencing (scRNA-seq) and spatial reconstruction using optimal transport, to map cell-type-specific enhancer activity at the single-cell level on a 3D virtual representation of the sample. We evaluated spatial-scERA in stage 6 Drosophila embryos using 25 candidate enhancers (including 19 previously uncharacterized regions), and validated the robustness of our predicted reconstructions by comparing them to microscopy images generated by in situ hybridization. Remarkably, spatial-scERA faithfully reconstructed the spatial activity of these enhancers, even when the enhancer-reporter construct was expressed in as few as 10 cells. Our results demonstrate the importance of integrating transcriptomic and spatial data for the accurate prediction of enhancer activity patterns in complex multicellular samples. Indeed, we found that chromatin modifications and open chromatin regions are often poor predictors of enhancer activity. Moreover, spatial data can often be essential for the accurate annotation of scRNA-seq clusters. Overall, spatial-scERA provides a scalable approach to map spatio-temporal enhancer spatial activity at single-cell resolution without the need for imaging or a priori knowledge of embryology and can be applied to any multicellular organism amenable to transgenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bac181df228c591b517e41c0b03a873f25a141a3" target='_blank'>
              Spatial reconstruction of single-cell enhancer activity in a multicellular organism
              </a>
            </td>
          <td>
            Baptiste Alberti, Séverine Vincent, Isabelle Stévant, Damien Lajoignie, Hélène Tarayre, Paul Villoutreix, Yad Ghavi-Helm
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67b3c3d2c93cdd85dd75bfc027eecd714525336" target='_blank'>
              Neuroblastoma plasticity during metastatic progression stems from the dynamics of an early sympathetic transcriptomic trajectory
              </a>
            </td>
          <td>
            Benjamin Villalard, A. Boltjes, F. Reynaud, Olivier Imbaud, Karine Thoinet, Ilse Timmerman, S. Croze, Emy Théoulle, Gianluigi Atzeni, Joël Lachuer, Jan J. Molenaar, G. Tytgat, C. Delloye-Bourgeois, Valérie Castellani
          </td>
          <td>2024-11-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0107b91d27e36a7693d55731e76f890903068027" target='_blank'>
              Single cell spatial profiling of FFPE splenic tissue from a humanized mouse model of HIV infection
              </a>
            </td>
          <td>
            Guoxin Wu, Samuel H. Keller, Luca Sardo, Brian Magliaro, Paul Zuck, C. Balibar, Claire Williams, Liuliu Pan, Mark Gregory, Kathy Ton, Jill Maxwell, Carol Cheney, Tom Rush, Bonnie J. Howell
          </td>
          <td>2024-10-08</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multiplex imaging technologies have developed rapidly over the past decades. The advancement of multiplex imaging has been driven in part by the recognition the spatial organisation of cells can represent important prognostic biomarkers and that simply studying the composition of cells in diseased tissue is often insufficient. There remains a lack of tools that can perform spatial analysis at the level of cellular aggregates (a common histopathological presentation) such as tumors and granulomas, with most analysis packages focussing on smaller regions of interest and potentially missing patterns in the overall lesion structure and cellular distribution. Here we present a protocol to quantify the cellular structure of entire tissue lesions built around two novel metrics. The Total Cell Preference Index reports whether a lesion tends to change in density in its central vs peripheral areas and can indicate the extent of necrosis across the entire lesion. The Immune Cell Preference Index then reports whether each immune cell-type is located more centrally or peripherally across the entire lesion. The output of both indexes is a single number readout for simple interpretation and visualization, and they can be applied to lesions of any size or shape. Additionally, this protocol can be applied to any slide-scanning multiplexed imaging system, either based on protein and nucleic acid staining. Finally, the protocol uses the open-source software QuPath and can be utilized by researchers with a basic understanding of QuPath with the full protocol able to be applied on pre-generated images within one hour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd048c5276460698536d41824ce54a133b750a5e" target='_blank'>
              A highly adaptable protocol for mapping spatial features of cellular aggregates in tissues
              </a>
            </td>
          <td>
            Andrew Sawyer, Nick Weingaertner, E. Patrick, Carl G Feng
          </td>
          <td>2024-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is the most prevalent type of kidney cancer characterized by its diverse tumor composition featuring various subclonal populations that hinder effective treatment responses. Tumoroids present an avenue for modelling this diversity and replicating the intricate tumor heterogeneity. Spatial transcriptomics preserves the spatial context of gene expression enabling us to study distinct tumor areas and the influence on overall diversity. Our spatial transcriptomics analysis uncovered tumor clusters with distinct genetic profiles that showcase various functional areas in depth and offer valuable understandings into the diversity of ccRCC types. Some of these tumor clusters exhibited activity in genes responsible for protein catabolism and reduced abundance of genes related to mitochondrial respiration processes. We also show isoform expression within tumoroids, in particular glutaminase (GLS) especially with the prevalence of the highly metabolically active GAC isoform that is expressed in regions where mitochondrial gene abundance is lower; whereas the KGA isoform displayed a more focal expression pattern. Combining long-read spatial transcriptomics with organoid models presents a novel strategy for unravelling gene and transcript level complexity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8b349aa856c9b8fd9c5542bf55647949ae547e0" target='_blank'>
              Long-read Spatial Transcriptomic Profiling of Patient-derived ccRCC Tumoroids Reveals Heterogeneity in Isoform and Gene Expression
              </a>
            </td>
          <td>
            Mustafa Elshani, Hazem Abdullah, Ying Zhang, Alexander Laird, Peter Mullen, David J. Harrison
          </td>
          <td>2024-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Cancer cell clusters have a higher capacity for metastasis than single cells, suggesting cancer cell clusters have biological properties different from those of single cells. The nature of de novo cancer cell clusters that are newly formed from tumor masses is largely unknown. Herein, we generated small cell clusters from colorectal cancer organoids and tracked the growth patterns of the clusters up to four cells. Growth patterns were classified into actively growing and poorly growing spheroids (PG). Notch signaling was robustly activated in small clusters immediately after dissociation, and Notch signaling inhibition markedly increased the proportion of PG spheroids. Only a limited number of PG spheroids grew under growth‐permissive conditions in vitro, but xenograft tumors derived from Notch inhibited clusters showed growth rates comparable to those of untreated spheroids. Thus, de novo clusters are composed of cells with interchangeable growth fates, which are regulated in a context‐dependent manner by Notch signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58323cfb508f873a7dd485a9b4e5d498413b2533" target='_blank'>
              Growth pattern of de novo small clusters of colorectal cancer is regulated by Notch signaling at detachment
              </a>
            </td>
          <td>
            Yi-Kai Lin, R. Coppo, K. Onuma, H. Endo, Jumpei Kondo, Sadahiro Iwabuchi, Shinichi Hashimoto, Yoshiro Itatani, K. Obama, Masahiro Inoue
          </td>
          <td>2024-09-19</td>
          <td>Cancer Science</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a7e2a6359020fadce11d67094c49cba1e4ed0b" target='_blank'>
              Multi-scale transcriptomics reveals that specific tumor cells promote lung adenocarcinoma metastasis through crosstalk with the microenvironment
              </a>
            </td>
          <td>
            Qi Cai, Lin Shi, Meng-ya Zhang, Peng Chen
          </td>
          <td>2024-10-03</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Recent advances in neuro-oncology highlighted the meninges as an important immunological hub at the crossroads between systemic immunity and the central nervous system. Meningiomas are predominantly benign tumors arising from meningeal cells. However, about 20% are high-grade cases, commonly refractory to standard treatment, showing high recurrence rates. Methylation-based classification has crucially improved meningioma patient stratification. However, the biological processes that govern differences between these classes remain unknown. Here, we investigate the role of differential immune infiltration, cellular communities, and their spatial organization. In light of previously found differential immune infiltration across methylation classes, this study aims to spatially resolve the role of myeloid cells and study their influence on tumor progression and recurrence.



 We deployed multiplexed ion beam imagining and spatial transcriptomics on a retrospective cohort with extensive clinical annotations. Analyzing single-cell transcriptomic data of a subset of patients (n=22), we delineated the expression of gene and protein markers that were used to design an antibody panel comprising 41 targets for cell-type identification, phenotypic and metabolic characterization at the protein level. Additionally, we curated a targeted gene panel for spatial transcriptomics to perform granular spatial analysis. An integration of histological, spatial transcriptomic, and protein layers provides a comprehensive multi-omics approach to studying spatial cellular interactions.



 Transcriptional programs and cell populations observed in single-nuclei data were captured in the spatial context revealing differences in the myeloid compartment across meningioma methylation classes. In specific classes, myeloid cells constitute a significant part of the tumor bulk warranting further investigation into their phenotypic differences. Accordingly, our analyses revealed a distinct phenotypic macrophage landscape along disease progression. Furthermore, spatial profiling of the metabolic regulome displayed distinct niche formation within perivascular and tumor compartments.



 In this study we confirm that macrophages undergo a phenotypic switch from more tolerogenic in low-grade to inflammatory in high-grade cases of meningioma. Correspondingly, specific phenotypic and metabolic features of myeloid and tumor cells contribute to tumor progressions. Considering the limited targeted therapy options and poor responses to immunotherapy in meningioma patients, we hope that our spatial multi-omics analysis may reveal recurrent infiltration patterns of phenotypically distinct cell types that could be informative of biological processes and predictive of disease outcome.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c6aee05db85854153c8ed4386e1bdf7d8947c0" target='_blank'>
              P02.23.A SPATIAL PROFILING OF THE MYELOID COMPARTMENT IN MENINGIOMA REVEALS DISTINCT PHENOTYPIC HETEROGENEITY ACROSS METHYLATION CLASSES
              </a>
            </td>
          <td>
            D. Calafato, Y. Tang, G. Rukhovich, B. Patel, A. Suwala, F. Sahm, M. Gerstung, F. Hartmann
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Distal metastases result from metastatic microenvironment and tumour epithelial cell interactions, the cellular heterogeneity of primary colorectal cancer (CRC) and liver metastases (LM) was evaluated by integrating single‐cell sequencing data, and the collected gene expression data from metastatic epithelial cell subsets was used to construct a prognostic model and to identify intercellular receptor‐ligand interactions between epithelial and immune cells in CRC and LM. Multiplex immunofluorescence staining, and in vitro wound healing, cell migration and cell apoptosis assays were performed to further explore the biological relevance of identified potential regulatory molecules. In this study, approximately 17 epithelial cell subtypes were detected, with Epi‐11 cells being highly expressed in LM tissues compared with CRC samples. Furthermore, patients with high expression of the metastasis‐related genetic profile of Epi‐11 had a poorer prognosis. By predicting receptor–ligand interactions, Epi‐11 cells were found to interact more with myeloid and T/natural killer cells in LM tissues when compared to primary CRC samples, which was mediated by the PLXNB1/SEMA4D axis. In addition, high SEMA4D expression was correlated with decreased overall survival of patients with CRC, whereas PLXNB1 was not. SEMA4D knockdown prevented the migration and promoted the apoptosis of HCT116 cells in vitro. In summary, Epi‐11 cells, an important subset of epithelial cells, may drive the LM of CRC and act by crosstalk with immune cells through the PLXNB1/SEMA4D signalling axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c1112eec1a1933f6595f6220f658ed440bccd24" target='_blank'>
              PLXNB1/SEMA4D signals mediate interactions between malignant epithelial and immune cells to promote colorectal cancer liver metastasis
              </a>
            </td>
          <td>
            Zixue Xuan, Yuan Zhang, Dan Li, Kai Wang, Ping Huang, Jiana Shi
          </td>
          <td>2024-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8531cded1ac1312b87ee916c1c14bac956c911de" target='_blank'>
              Computational immune synapse analysis reveals T-cell interactions in distinct tumor microenvironments
              </a>
            </td>
          <td>
            Victor G Wang, Zichao Liu, Jan Martinek, Ali Foroughi pour, Jie Zhou, Hannah Boruchov, Kelly Ray, Karolina Palucka, Jeffrey H. Chuang
          </td>
          <td>2024-09-28</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Adrenocortical carcinoma (ACC) is a rare endocrine malignancy with a poor prognosis and limited treatment options. Bulk genomic characterization of ACC has not yielded obvious therapeutic or immunotherapeutic targets, yet novel therapies are needed. We hypothesized that elucidating the intratumoral cellular heterogeneity by single nuclei RNA sequencing analyses would yield insights into potential therapeutic vulnerabilities of this disease. In addition to characterizing the immune cell and fibroblast landscape, our analyses of single nuclei gene expression profiles identified an adrenal cortex cell cluster exhibiting a program of replication stress and DNA damage response in primary and metastatic ACC. In vitro assessment of replication stress and DNA damage response using an ACC cell line and a series of newly-derived hormonally active patient-derived tumor organoids revealed ATR sensitivity. These findings provide novel mechanistic insight into ACC biology and suggest that an underlying dependency on ATR may be leveraged therapeutically in advanced ACC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1469f57c44a466002613ce12061fcafe073cc550" target='_blank'>
              Single Nuclei Sequencing Reveals Intratumoral Cellular Heterogeneity and Replication Stress in Adrenocortical Carcinoma
              </a>
            </td>
          <td>
            Liudmila V. Popova, Elizabeth A. R. Garfinkle, Daniel M. Chopyk, Jaye B. Navarro, Adithe Rivaldi, Yaoling Shu, Elena Lomonosova, J. Phay, Barbra S. Miller, Swati Sattuwar, Mary Mullen, Elaine R. Mardis, Katherine E. Miller, Priya H. Dedhia
          </td>
          <td>2024-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Gliomas are aggressive brain tumors associated with a poor prognosis. Cancer stem cells (CSCs) play a significant role in tumor recurrence and resistance to therapy. This study aimed to identify and characterize glioma stem cells (GSCs), analyze their interactions with various cell types, and develop a prognostic signature. Methods Single-cell RNA sequencing data from 44 primary glioma samples were analyzed to identify GSC populations. Spatial transcriptomics and gene regulatory network analyses were performed to investigate GSC localization and transcription factor activity. CellChat analysis was conducted to infer cell-cell communication patterns. A GSC signature (GSCS) was developed using machine learning algorithms applied to bulk RNA sequencing data from multiple cohorts. In vitro and in vivo experiments were conducted to validate the role of TUBA1C, a key gene within the signature. Results A distinct GSC population was identified, characterized by high proliferative potential and an enrichment of E2F1, E2F2, E2F7, and BRCA1 regulons. GSCs exhibited spatial proximity to myeloid-derived suppressor cells (MDSCs). CellChat analysis revealed an active MIF signaling pathway between GSCs and MDSCs. A 26-gene GSCS demonstrated superior performance compared to existing prognostic models. Knockdown of TUBA1C significantly inhibited glioma cell migration, and invasion in vitro, and reduced tumor growth in vivo. Conclusion This study offers a comprehensive characterization of GSCs and their interactions with MDSCs, while presenting a robust GSCS. The findings offer new insights into glioma biology and identify potential therapeutic targets, particularly TUBA1C, aimed at improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c2780868fec467184ab80544d5944cba2384cf" target='_blank'>
              From single-cell to spatial transcriptomics: decoding the glioma stem cell niche and its clinical implications
              </a>
            </td>
          <td>
            Lei Cao, Xu Lu, Xia Wang, Hao Wu, Xiaye Miao
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdce58909f9de06504a3ce2e2bc45ebf1454164f" target='_blank'>
              Spatial single-cell protein landscape reveals vimentinhigh macrophages as immune-suppressive in the microenvironment of hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Xinyao Qiu, Tao Zhou, Shuai Li, Jianmin Wu, Jing Tang, Guosheng Ma, Shuai Yang, Ji Hu, Kaiting Wang, Siyun Shen, Hongyang Wang, Lei Chen
          </td>
          <td>2024-09-26</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The development and progression of diseases in multicellular organisms unfold within the intricate three-dimensional body environment. Thus, to comprehensively understand the molecular mechanisms governing individual development and disease progression, precise acquisition of biological data, including genome, transcriptome, proteome, metabolome, and epigenome, with single-cell resolution and spatial information within the body’s three-dimensional context, is essential. This foundational information serves as the basis for deciphering cellular and molecular mechanisms. Although single-cell multi-omics technology can provide biological information such as genome, transcriptome, proteome, metabolome, and epigenome with single-cell resolution, the sample preparation process leads to the loss of spatial information. Spatial multi-omics technology, however, facilitates the characterization of biological data, such as genome, transcriptome, proteome, metabolome, and epigenome in tissue samples, while retaining their spatial context. Consequently, these techniques significantly enhance our understanding of individual development and disease pathology. Currently, spatial multi-omics technology has played a vital role in elucidating various processes in tumor biology, including tumor occurrence, development, and metastasis, particularly in the realms of tumor immunity and the heterogeneity of the tumor microenvironment. Therefore, this article provides a comprehensive overview of spatial transcriptomics, spatial proteomics, and spatial metabolomics-related technologies and their application in research concerning esophageal cancer, gastric cancer, and colorectal cancer. The objective is to foster the research and implementation of spatial multi-omics technology in digestive tumor diseases. This review will provide new technical insights for molecular biology researchers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77672e163c08fe2553e07120e8f19a354c4e1b31" target='_blank'>
              The burgeoning spatial multi-omics in human gastrointestinal cancers
              </a>
            </td>
          <td>
            Weizheng Liang, Zhenpeng Zhu, Dandan Xu, Peng Wang, Fei Guo, Haoshan Xiao, Chenyang Hou, Jun Xue, Xuejun Zhi, Rensen Ran
          </td>
          <td>2024-09-13</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1436facb9b04a543d668f4862e82c8d1f590873" target='_blank'>
              FBXO2 as a switch guides a special fate of tumor clones evolving into a highly malignant transcriptional subtype in oral squamous cell carcinoma.
              </a>
            </td>
          <td>
            Jingyi Cheng, O. Liu, Xin Bin, Zhangui Tang
          </td>
          <td>2024-11-02</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is an increasingly prevalent malignancy worldwide. This study aims to understand molecular alterations associated with lymph node metastasis of OSCC in order to improve treatment strategies. We analyzed a cohort of 46 patients with primary OSCC, including 10 with lymph node metastasis and 36 without. Using a comprehensive multi-omics approach—encompassing genomic, transcriptomic, proteomic, epigenetic, single-cell, and spatial analyses—we integrated data to delineate the molecular landscape of OSCC in the context of lymph node metastasis. Our genomic analysis identified significant mutations in key genes within the MAPK, TGF-β, and WNT signaling pathways, which are essential for tumor development. The proteogenomic analysis highlighted pathways critical for lymph node dissemination and factors contributing to an immunosuppressive tumor microenvironment. Elevated levels of POSTN were found to reorganize the extracellular matrix (ECM), interact with TGF-β, disrupt cell cycle regulation, and suppress the immune response by reducing VCAM1 activity. Integrated analyses of single-cell and spatial transcriptome data revealed that cancer-associated fibroblasts (CAFs) secrete TGF-β1/2, promoting cancer cell metastasis through epithelial-mesenchymal transition (EMT). Our integrated multi-omics analysis provides a detailed understanding of molecular mechanisms driving lymph node metastasis of OSCC. These insights could lead to more precise diagnostics and targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bdd901aa91cd3ba2711b681f62cf92ddfab3b2" target='_blank'>
              Proteogenomic Characterization of Primary Oral Squamous Cell Carcinomas Unveils the Extracellular Matrix Remodeling and Immunosuppressive Microenvironment Linked with Lymph Node Metastasis
              </a>
            </td>
          <td>
            Yu Liu, Zhenyu Yang, J. Pu, Jie Zhong, U. Khoo, Yu-Xiong Su, Gao Zhang
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ab5a7fdae99ad93f99247468e27d3a8b9633164" target='_blank'>
              Spatiotemporal modeling reveals high-resolution invasion states in glioblastoma
              </a>
            </td>
          <td>
            Varsha Thoppey Manoharan, Aly O. Abdelkareem, Gurveer Gill, Samuel Brown, Aaron H. Gillmor, Courtney Hall, Heewon Seo, Kiran Narta, Sean Grewal, Ngoc Ha Dang, Bo Young Ahn, Kata Osz, X. Lun, Laura Mah, Franz Zemp, Douglas J Mahoney, Donna L Senger, Jennifer A. Chan, A. Morrissy
          </td>
          <td>2024-10-10</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Immunotherapy has revolutionized clinical care for many cancers, yet the approach has not been fully realized in immunologically cold tumors like Pancreatic Ductal Adenocarcinoma (PDAC). One striking hallmark of PDAC is local immunosuppression (LIS) which leads to cytotoxic T cell inactivation and exclusion. There is great need for an increased understanding of the cellular crosstalk within tumors and the identification of stromal and immune populations involved in shaping the tumor microenvironment (TME). Oncogenic KRAS activation in tumor cells promotes the invasion and proliferation of tumor-supporting stromal cells, while excluding cancer-targeted cytotoxic T cells. LIS is mediated by multiple subtypes of cancer-associated fibroblasts (CAFs) and myeloid cells resident within the tumor parenchyma. Multiple prior attempts to reverse LIS in PDAC by targeting individual stromal cell populations have been unsuccessful, alluding to the complexity of stromal crosstalk within the TME. The stromal diversity of PDAC complicates investigating paracrine cascades involving multiple cell types. To decipher diverse drug effects on altering the TME, we employ in vivo studies in mouse models recapitulating the human disease, as well as a novel tumor explant model that enables the short-term culture of intact human or murine PDAC. Importantly, PDAC explants maintain their histopathological architecture and cellular diversity over time. This medium-throughput platform allows for testing of multiple drugs and mechanistic hypotheses in the native PDAC TME. We show in preliminary data that Smoothened inhibition (SMOi) decreases proliferation and activity of myCAFs, but provokes the expansion of CD11b-positive myeloid cells in vivo. Thus, we hypothesize that LIS in PDAC is maintained by a delicate balance between myCAFs and myeloid cells, preventing effective T cell invasion. Single cell RNA-seq data comparing ctrl vs. SMOi-treated murine PDAC elucidates stromal subpopulations involved in the LIS phenotype and guides the identification of myeloid subtypes emerging after SMOi. Strikingly, we demonstrated that simultaneous SMOi and targeting myeloid cells via anti-Gr1 or CCR1 inhibition (CCR1i) significantly elevates cytotoxic T cell numbers up to 20x within the TME. We are currently investigating whether the activity of these T cells may be further potentiated through combination with immunomodulatory agents. By testing various treatment combination in the same TME, we will identify the best synergistic effects for future immunotherapy approaches in human PDAC. In summary, we are elucidating the complex mechanism behind LIS in PDAC by employing our novel explant culture system alongside in vivo studies. We aim to develop a translatable regimen to neutralize LIS, reactivating the cytotoxic T cells in the tumor periphery to invade, proliferate, and attack cancer cells.
 Citation Format: Marie C Hasselluhn, Lukas Vlahos, Dafydd Thomas, Quin T Waterbury, Alvaro Curiel Garcia, Amanda R Decker-Farrell, Tanner C Dalton, Stephen A Sastra, Carmine F Palermo, Timothy C Wang, Andrea Califano, Kenneth P Olive. Disrupting local immunosuppression through combination myCAF/myeloid targeting in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A045.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1820ee724a7ae9f0d2e205032e44a12b20445026" target='_blank'>
              Abstract A045: Disrupting local immunosuppression through combination myCAF/myeloid targeting in pancreatic cancer
              </a>
            </td>
          <td>
            Marie C. Hasselluhn, Lukas J. Vlahos, Dafydd Thomas, Quin T. Waterbury, Á. C. García, Amanda R. Decker-Farrell, Tanner C. Dalton, Stephen A. Sastra, Carmine F. Palermo, Timothy C Wang, Andrea Califano, K. Olive
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d92fbde3dae7404c9f30bd22e44407b07e5261a" target='_blank'>
              Self-sustaining long-term 3D epithelioid cultures reveal drivers of clonal expansion in esophageal epithelium
              </a>
            </td>
          <td>
            A. Herms, D. Fernandez-Antoran, M. Alcolea, Argyro Kalogeropoulou, Ujjwal Banerjee, G. Piedrafita, E. Abby, J. A. Valverde-Lopez, Inês S Ferreira, Irene Caseda, M. T. Bejar, S. Dentro, Sara Vidal-Notari, S. Ong, B. Colom, K. Murai, C. King, Krishnaa Mahbubani, K. Saeb‐Parsy, Alan R Lowe, M. Gerstung, P. Jones
          </td>
          <td>2024-09-23</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>57</td>
        </tr>

        <tr id="The phenotypic and functional states of a cell are modulated by a complex interactive molecular hierarchy of multiple omics layers, involving the genome, epigenome, transcriptome, proteome, and metabolome. Spatial omics approaches have enabled the capture of information from different molecular layers directly in the tissue context. However, current technologies are limited to map one to two modalities at the same time, providing an incomplete representation of cellular identity. Such data is inadequate to fully understand complex biological systems and their underlying regulatory mechanisms. Here we present spatial-Mux-seq, a multi-modal spatial technology that allows simultaneous profiling of five different modalities, including genome-wide profiles of two histone modifications and open chromatin, whole transcriptome, and a panel of proteins at tissue scale and cellular level in a spatially resolved manner. We applied this technology to generate multi-modal tissue maps in mouse embryos and mouse brains, which discriminated more cell types and states than unimodal data. We investigated the spatiotemporal relationship between histone modifications, chromatin accessibility, gene and protein expression in neuron differentiation revealing the relationship between tissue organization, function, and gene regulatory networks. We were able to identify a radial glia spatial niche and revealed spatially changing gradient of epigenetic signals in this region. Moreover, we revealed previously unappreciated involvement of repressive histone marks in the mouse hippocampus. Collectively, the spatial multi-omics approach heralds a new era for characterizing tissue and cellular heterogeneity that single modality studies alone could not reveal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fb17f1f5e4732ac8e25cf0a1ea0dd59476fec81" target='_blank'>
              Multiplexed spatial mapping of chromatin features, transcriptome, and proteins in tissues
              </a>
            </td>
          <td>
            Pengfei Guo, Liran Mao, Yufan Chen, Chin Nien Lee, Angelysia Cardilla, Mingyao Li, Marek Bartošovič, Yanxiang Deng
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Single-cell transcriptomics has transformed our understanding of cellular diversity in biological systems. However, systematic noise, often introduced by low-quality cells, can obscure biological signals if not properly accounted for. Thus, one of the common quality control steps involves filtering out cells with a high percentage of mitochondrial RNA counts (pctMT), as high pctMT typically indicates cell death. Yet, commonly used filtering thresholds, primarily derived from studies on healthy tissues, may be overly stringent for malignant cells, which often naturally exhibit higher baseline mitochondrial gene expression. We analyzed public single-cell RNA-seq and spatial data to investigate if malignant cells with high pctMT are viable and functionally significant subpopulations. Results We analyzed nine single-cell RNA-seq datasets from uveal melanoma, breast, lung, kidney, head and neck, prostate, and pancreatic cancers, including 439,507 cells from 151 patients. Malignant cells exhibited significantly higher pctMT than nonmalignant cells without a significant increase in dissociation-induced stress signature scores. Malignant cells with high pctMT showed metabolic dysregulation, including increased xenobiotic metabolism, which is implicated in cancer therapeutic response. Our analysis of pctMT in cancer cell lines uncovered associations with resistance and sensitivity to certain classes of drugs. Additionally, we observed a link between pctMT and malignant cell transcriptional heterogeneity as well as patient clinical features. Conclusions This study provides a detailed exploration of the functional characteristics of malignant cells with elevated pctMT, challenging current quality control practices in single-cell RNA-seq analyses of tumors. Our findings have the potential to improve data interpretation and refine the biological conclusions of future cancer studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a829d72e1a8f6b7f963a581d9b84a3007bfbb96" target='_blank'>
              Filtering cells with high mitochondrial content removes viable metabolically altered malignant cell populations in cancer single-cell studies
              </a>
            </td>
          <td>
            Josephine Yates, Agnieszka Kraft, Valentina Boeva
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer-associated fibroblast (CAF) composition within the same organ varies across different cancer subtypes. Distinct CAF subtypes exhibit unique features due to interactions with immune cells and the tumor microenvironment. However, data on CAF subtypes in individuals with vestibular schwannoma (VS) are lacking. Therefore, we aimed to distinguish CAF subtypes at the single-cell level, investigate how stem-like CAF characteristics influence the tumor immune microenvironment, and identify CAF subtype-specific metabolic reprogramming pathways that contribute to tumor development. Data were analyzed from three patients with VS, encompassing 33,081 single cells, one bulk transcriptome cohort, and The Cancer Genome Atlas Pan-Cancer database (RNA sequencing and clinical data). Our findings revealed that antigen-presenting CAFs are linked to substantially heightened immune activity, supported by metabolic reprogramming, which differs from tumorigenesis. High expression of the stem-like CAF gene signature correlated with poor prognosis in low-grade gliomas within the pan-cancer database. This is the first study to classify CAF subtypes in VS patients and identify a therapeutic vulnerability biomarker by developing a stem-like CAF gene signature. Personalized treatments tailored to individual patients show promise in advancing precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd8e5819e3704f5476a3b3da2ab4be193ceaa21" target='_blank'>
              Cancer-Associated Fibroblast Subtypes Reveal Distinct Gene Signatures in the Tumor Immune Microenvironment of Vestibular Schwannoma
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jung Woo Lee
          </td>
          <td>2024-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Hepatocellular carcinomas (HCCs) are highly heterogeneous malignancies. They are characterized by a peculiar tumor microenvironment and dense vascularization. The importance of signaling between immune cells, endothelial cells, and tumor cells leads to the difficult recapitulation of a reliable in vitro HCC model using the conventional two-dimensional cell cultures. The advent of three-dimensional organoid tumor technology has revolutionized our understanding of the pathogenesis and progression of several malignancies by faithfully replicating the original cancer genomic, epigenomic, and microenvironmental landscape. Organoids more closely mimic the in vivo environment and cell interactions, replicating factors such as the spatial organization of cell surface receptors and gene expression, and will probably become an important tool in the choice of therapies and the evaluation of tumor response to treatments. This review aimed to describe the ongoing and potential applications of organoids as an in vitro model for the study of HCC development, its interaction with the host’s immunity, the analysis of drug sensitivity tests, and the current limits in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d0895466db0ea1058030a64c8d93345a743b1d0" target='_blank'>
              Hepatocellular-Carcinoma-Derived Organoids: Innovation in Cancer Research
              </a>
            </td>
          <td>
            C. Airola, M. Pallozzi, Eleonora Cesari, L. Cerrito, L. Stella, Claudio Sette, F. Giuliante, Antonio Gasbarrini, F. Ponziani
          </td>
          <td>2024-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Leukocyte extravasation is a key component of the innate immune response, while circulating tumor cell extravasation is a critical step in metastasis formation. Despite their importance, the mechanisms underlying leukocyte and tumor cell extravasation remain incompletely understood. Here, we developed an imaging pipeline that integrates fast, label-free live-cell imaging with deep learning-based image analysis to quantitatively track and compare the initial steps of extravasation—cell landing and arrest on an endothelial monolayer—under physiological flow conditions. We find that pancreatic ductal adenocarcinoma (PDAC) cells exhibit variable adhesion strength and flow sensitivity. Remarkably, some PDAC cells demonstrate comparable endothelial engagement as leukocytes, preferentially arresting at endothelial junctions, potentially due to increased stiffness at these sites, which leads to exposure to the underlying basal extracellular matrix. PDAC cells also tend to cluster in regions with high, heterogeneous expression of the endothelial CD44 receptor. Simulations suggest that clustering results from the combination of CD44-mediated attachment and localized flow disturbances that facilitate subsequent cell attachment. Targeting CD44 using siRNA or function-blocking antibodies, or degrading its ligand hyaluronic acid (HA), almost completely abolishes PDAC cell attachment. Overall, we demonstrate that cancer and immune cells share both common and unique features in endothelial adhesion under flow, and identify CD44 and HA as key mediators of PDAC cell arrest.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c66df57f0b64f339c753331f3baf8365af93874" target='_blank'>
              Fast label-free live imaging reveals key roles of flow dynamics and CD44-HA interaction in cancer cell arrest on endothelial monolayers
              </a>
            </td>
          <td>
            Gautier Follain, Sujan Ghimire, Joanna W. Pylvänäinen, Monika Vaitkevičiūtė, Diana Wurzinger, Camilo Guzmán, James R. W. Conway, Michal Dibus, S. Oikari, K. Rilla, Marko Salmi, Johanna Ivaska, Guillaume Jacquemet
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 The dynamic interplay between tumor cells and γδT cells within the tumor microenvironment (TME) significantly influences disease progression and immunotherapy outcome. Here, we delved into the modulation of γδT-cell activation by tumor cell ligands CD112 and CD155, which interact with the activating receptor DNAM-1 on γδT cells. Spatial and single cell RNA sequencing (scRNA-seq), as well as spatial metabolome analysis, from neuroblastoma (NB) revealed that the expression levels and localization of CD112 and CD155 varied across and within tumors, correlating with differentiation status, metabolic pathways, and ultimately disease prognosis and patient survival. Both in vivo tumor xenograft experiments and in vitro co-culture experiments demonstrated that a high CD112/CD155 expression ratio in tumors enhanced γδT-cell-mediated cytotoxicity, while a low-ratio fostered tumor resistance. Mechanistically, CD112 sustained DNAM-1-mediated γδT-cell activation, whereas CD155 downregulated DNAM-1 expression via TRIM21-mediated ubiquitin proteasomal degradation. By interacting with tumor cells differentially expressing CD112 and CD155, intratumoral γδT cells exhibited varying degrees of activation and DNAM-1 expression, representing three major functional subsets. This study underscores the complexity of tumor-immune crosstalk, offering insights into how tumor heterogeneity shapes the immune landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed0f304618ec33893cbfa169baffb49b70570809" target='_blank'>
              Spatial and Single Cell Analyses Reveal Heterogeneity of DNAM-1 Receptor-Ligand Interactions that Instructs Intratumoral γδT-Cell Activity
              </a>
            </td>
          <td>
            Xiaolin Wang, Hui Wang, Zhengjing Lu, Xiangjun Liu, Wenjia Chai, Wei Wang, Jun Feng, Shen Yang, Wei Yang, Haiyan Cheng, Chenghao Chen, Shihan Zhang, N. Sun, Qiaoyin Liu, Qiliang Li, Wenqi Song, Fang Jin, Qi Zeng, Shengcai Wang, Yan Su, Huanmin Wang, Xinxin Ni, Jingang Gui
          </td>
          <td>2024-11-08</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Immune responses in tissues display intricate patterns of gene expression that vary across space and time. While such patterns have been increasingly linked to disease outcomes, the mechanisms that generate them and the logic behind them remain poorly understood. As a tractable model of spatial immune responses, we investigated heterogeneous expression of antimicrobial peptides in the larval fly fat body, an organ functionally analogous to the liver. To capture the dynamics of immune response across the full tissue at single-cell resolution, we established live light sheet fluorescence microscopy of whole larvae. We discovered that expression of antimicrobial peptides occurs in a reproducible spatial pattern, with enhanced expression in the anterior and posterior lobes of the fat body. This pattern correlates with microbial localization via blood flow but is not caused by it: loss of heartbeat suppresses microbial transport but leaves the expression pattern unchanged. This result suggests that regions of the tissue most likely to encounter microbes via blood flow are primed to produce antimicrobials. Spatial transcriptomics revealed that these immune microenvironments are defined by genes spanning multiple biological processes, including lipid-binding proteins that regulate host cell death by the immune system. In sum, the larval fly fat body exhibits spatial compartmentalization of immune activity that resembles the strategic positioning of immune cells in mammals, such as in the liver, gut, and lymph nodes. This finding suggests that tissues may share a conserved spatial organization that optimizes immune responses for antimicrobial efficacy while preventing excessive self-damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1e820264076d865da870641b39e567dd5ebfc25" target='_blank'>
              Spatial microenvironments tune immune response dynamics in the Drosophila larval fat body
              </a>
            </td>
          <td>
            Brandon H. Schlomann, Ting-Wei Pai, Jazmin Sandhu, Genesis Ferrer Imbert, T. G. Graham, Hernán G. García
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Innovative identification technologies for hematopoietic stem cells (HSCs) have advanced the frontiers of stem cell biology. However, most analytical techniques capture only a single snapshot, disregarding the temporal context. A comprehensive understanding of the temporal heterogeneity of HSCs necessitates live-cell, real-time and non-invasive analysis. Here, we developed a prediction system for HSC diversity by integrating single-HSC ex vivo expansion technology with quantitative phase imaging (QPI)-driven machine learning. By analyzing single-cell kinetics with QPI, we discovered previously undetectable diversity among HSCs that snapshot analysis fails to capture. Our QPI-driven algorithm quantitatively evaluates the stemness of individual HSCs and incorporates temporal information to significantly improve prediction accuracy. This platform marks a paradigm shift from “identification” to “prediction”, enabling us to forecast HSC status by analyzing their past temporal kinetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41d59e33033757dcb2fbbe006d53b555326dd5cf" target='_blank'>
              Quantitative phase imaging with temporal kinetics predicts hematopoietic stem cell diversity
              </a>
            </td>
          <td>
            Takao Yogo, Yuichiro Iwamoto, Hans Jiro Becker, Takaharu Kimura, Ayano Sugiyama-Finnis, Tomomasa Yokomizo, Toshio Suda, Sadao Ota, Satoshi Yamazaki
          </td>
          <td>2024-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Whether metastasis in humans can be accomplished by most primary tumor cells or requires the evolution of a specialized trait remains an open question. To evaluate whether metastases are founded by non-random subsets of primary tumor lineages requires extensive, difficult-to-implement sampling. We have realized an unusually dense multi-region sampling scheme in a cohort of 26 colorectal cancer patients with peritoneal metastases, reconstructing the evolutionary history of on average 28.8 tissue samples per patient with a microsatellite-based fingerprinting assay. To assess metastatic randomness, we evaluate inter- and intra-metastatic heterogeneity relative to the primary tumor and find that peritoneal metastases are more heterogeneous than liver metastases but less diverse than locoregional metastases. Metachronous peritoneal metastases exposed to systemic chemotherapy show significantly higher inter-lesion diversity than synchronous, untreated metastases. Projection of peritoneal metastasis origins onto a spatial map of the primary tumor reveals that they often originate at the deep-invading edge, in contrast to liver and lymph node metastases which exhibit no such preference. Furthermore, peritoneal metastases typically do not share a common subclonal origin with distant metastases in more remote organs. Synthesizing these insights into an evolutionary portrait of peritoneal metastases, we conclude that the peritoneal-metastatic process imposes milder selective pressures onto disseminating cancer cells than the liver-metastatic process. Peritoneal metastases’ unique evolutionary features have potential implications for staging and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d18aa39a03328c1be1ca186cb7127b46d1c690b2" target='_blank'>
              A unique interplay of access and selection shapes peritoneal metastasis evolution in colorectal cancer
              </a>
            </td>
          <td>
            E. Wassenaar, A. Gorelick, Wei-Ting Hung, David M Cheek, Emre Kucukkose, I-Hsiu Lee, Martin Blohmer, Sebastian Degner, Peter Giunta, Rene MJ Wiezer, M. Raicu, I. Ubink, S. Klaasen, Nico Lansu, E. Watson, Ryan B. Corcoran, Genevieve Boland, G. Getz, Geert Jpl Kops, D. Juric, Jochen K Lennerz, D. Boerma, O. Kranenburg, Kamila Naxerova
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>195</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d593a4bab5e1bd9b8d5a096791ffe052f667c34" target='_blank'>
              Computational identification of surface markers for isolating distinct subpopulations from heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Andrea L. Gardner, T. Jost, Daylin Morgan, Amy Brock
          </td>
          <td>2024-10-17</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Plasticity in tumor cells represents the ability of cancer cells to transition between different phenotypic states, including epithelial and mesenchymal states, known as the epithelial-mesenchymal transition (EMT). This transition is critical for processes such as invasion, metastasis, and resistance to therapy. Understanding the dynamics of plasticity and the extrinsic factors influencing state transitions may be crucial for improving therapeutic strategies and patient outcomes. However, deciphering the complex interactions and factors driving these state changes remains challenging. We introduce PlastiNet, a graphical attention-based network (GAT) designed to create spatially aware embeddings that enable the identification of cellular neighborhoods and the plasticity spectrum, despite gene panel limitations. PlastiNet incorporates both self-attention and neighbor-attention mechanisms in its GAT. This approach allows us to account for the influence of surrounding cells in determining the state of a target cell. We validated our model by constructing differentiation paths in healthy colon data, starting from stem cell signatures. The attention mechanisms highlighted known interactions, such as those between fibroblasts and epithelial cells. When applied to pancreatic ductal adenocarcinoma (PDAC) tumors, PlastiNet identified the SPP1-CD44 axis as a potential mediator in the classical-to-basal transition of tumor cells. To further explore the impact of CD44, particularly the isoform switch that occurs during EMT, we studied a patient-derived a PDAC cell line using single-cell RNA sequencing (scRNAseq) and Multiplexed Arrays Isoform Sequencing (Mas-Iso-Seq) to analyze cells spanning a range of the plasticity spectrum. Looping this analysis back to spatial transcriptomics data, we observed that the signature associated with CD44v8-10 or CD44s isoforms is co-localized with the SPP1 ligand, suggesting a spatially confined interaction. This finding underscores the potential of PlastiNet to generate hypotheses directly from patient-derived samples, offering new insights into the cellular mechanisms driving tumor plasticity. PlastiNet effectively constructs spatially aware embeddings that capture the dynamics of tumor cell plasticity. By identifying key interactions and extrinsic factors, this method opens new avenues for understanding and potentially targeting the mechanisms underlying tumor progression and treatment resistance.
 Citation Format: Izabella L Zamora, Milan Parikh, Samuel Wright, Lynn Bi, Bidish Patel, Dana Pe’er, David Ting, Nir Hacohen, Arnav Mehta. PlastiNet: Understanding the Epithelial - Mesenchymal Transition Through Graphical Attention in Spatial Transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C065.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b321cb8d990031a14ea0e651c0cf868b86d2e213" target='_blank'>
              Abstract C065: PlastiNet: Understanding the Epithelial - Mesenchymal Transition Through Graphical Attention in Spatial Transcriptomics
              </a>
            </td>
          <td>
            Izabella Zamora, Milan Parikh, Samuel J Wright, Lynn Bi, Bidish Kumar Patel, D. Pe’er, David T Ting, N. Hacohen, Arnav Mehta
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="Esophageal cancer comprises two main subtypes: esophageal squamous cell carcinoma (ESCC) and esophageal adenocarcinoma (EAC). Previous studies have revealed distinct genomic characteristics between these subtypes, with ESCC sharing similarities with head and neck squamous cell carcinoma (HNSCC), and EAC aligning with gastric adenocarcinoma (GAC). Additionally, recent immunotherapy clinical trials have shown a higher response rate in ESCC and HNSCC compared to EAC and GAC. However, how the tumor microenvironment contributes to these varied immunotherapy responses remains unclear. In this study, through a comparative analysis of single-cell tumor transcriptomes from 35 patients with ESCC, EAC, HNSCC, or GAC, we identified two groups with tumor microenvironment distinctions: ESCC and HNSCC versus EAC and GAC, consistent with their genomic classifications. Malignant epithelial cells displayed distinct separations based on histological origin. In the tumor immune microenvironment, we observed an enrichment of CXCL13+CD8+ T cells and CXCL9+CXCL10+ tumor-associated macrophages (TAMs) in ESCC and HNSCC, which activate cellular immunity through interferon-γ. In contrast, EAC and GAC exhibited a high presence of heat-shock protein-expressing CD8+ T cells and MARCO+ TAMs. These immune signatures help explain the varied immunotherapy responses among these cancer subtypes and successfully predict immunotherapy outcomes across diverse cancer types, underscoring their clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251c1e9cc81c4dd68a06455379c16a74217063d7" target='_blank'>
              Tumor microenvironment distinctions between esophageal cancer subtypes explain varied immunotherapy responses
              </a>
            </td>
          <td>
            S. Baek, Junha Cha, M. Hong, Gamin Kim, Yoon Woo Koh, Dahee Kim, M. Hemberg, Seong Yong Park, Hye Ryun Kim, Insuk Lee
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162a418f3caf8634ccba24bf052ce0feed7db43f" target='_blank'>
              Unraveling molecular characteristics and tumor microenvironment dynamics of neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            David Heimdörfer, N. Artamonova, Zoran Culig, Isabel Heidegger
          </td>
          <td>2024-10-16</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eadaab3d293ff876f51403f3620a7ad83b9f29" target='_blank'>
              Spatial transcriptomic analysis of primary and metastatic pancreatic cancers highlights tumor microenvironmental heterogeneity.
              </a>
            </td>
          <td>
            A. Khaliq, Meenakshi Rajamohan, Omer Saeed, Kimia Mansouri, Asif Adil, Chi Zhang, Anita Turk, Julienne L. Carstens, Michael House, Sikander Hayat, G. Nagaraju, Sam G Pappas, Y. A. Wang, N. Zyromski, Mateusz Opyrchal, Kelvin P Lee, Heather M. O’Hagan, Bassel El Rayes, Ashiq Masood
          </td>
          <td>2024-09-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc9bc6fb64c943c109f90f687d279d53c1cce3e" target='_blank'>
              Integrating microarray-based spatial transcriptomics and RNA-seq reveals tissue architecture in colorectal cancer
              </a>
            </td>
          <td>
            Zheng Li, Xiao-Jie Zhang, Chongyuan Sun, Zefeng Li, He Fei, Dongbing Zhao
          </td>
          <td>2024-09-17</td>
          <td>J. Big Data</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Today, the intratumoral composition is a relevant factor associated with the progression and aggression of cancer. Although it suggests a metabolic interdependence among the subpopulations inside the tumor, a detailed map of how this interdependence contributes to the malignant phenotype is still lacking. To address this issue, we developed a systems biology approach integrating single-cell RNASeq and genome-scale metabolic reconstruction to map the metabolic cross-feeding among the subpopulations previously identified in the spheroids of MCF7 breast cancer. By calibrating our model with expression profiles and the experimental growth rate, we concluded that the reverse Warburg effect emerges as a mechanism to optimize community growth. Furthermore, through an in silico analysis, we identified lactate, alpha-ketoglutarate, and some amino acids as key metabolites whose disponibility alters the growth rate of the spheroid. Altogether, this work provides a strategy for assessing how space and intratumoral heterogeneity influence the metabolic robustness of cancer, issues suggesting that computational strategies should move toward the design of optimized treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c54b7152ffb9e525c998dbbf7f6c9c4f3769b3" target='_blank'>
              Intratumoral Heterogeneity and Metabolic Cross-Feeding in a Three-Dimensional Breast Cancer Culture: An In Silico Perspective
              </a>
            </td>
          <td>
            Jorge E. Arellano-Villavicencio, Aarón Vázquez-Jiménez, J. J. Oropeza-Valdez, Cristian Padron-Manrique, H. Prado-García, Armando R. Tovar, O. Reséndis-Antonio
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cellular plasticity plays critical roles in tissue regeneration, tumour progression and therapeutic resistance. However, the mechanism underlying this cell state transition remains elusive. Here, we show that the transcription factor SOX2 induces fetal reprogramming and reversible dormancy in colorectal cancer (CRC). SOX2 expression correlates with fetal reprogramming and poor prognosis in human primary and metastatic colorectal adenocarcinomas. In mouse CRC models, rare slow-cycling SOX2+SCA1+ cells are detected in early Apc-deleted mouse tumours that undergo slow clonal expansion over time. In contrast, the SOX2+ clones were found proliferative in advanced Apc-/-;KrasG12D/+;p53-/-;Tgfbr2-/-(AKPT) tumours, accompanied by dynamic cell state reprogramming from LGR5+ to LGR5-SCA1+ cells. Using transgenic mouse models, we demonstrate that ectopic expression of SOX2 inhibits intestinal lineage differentiation and induces fetal reprogramming. SOX2+ cells adopt dynamic cell cycle states depending on its expression level. High SOX2 expression results in hyperproliferation, whereas low SOX2 levels induce senescence-mediated dormancy. We find that loss of p53 can reverse SOX2-induced senescence, in line with the dormant cell state exit of the SOX2+ cells observed in advanced tumours. Finally, SOX2 expression is induced by 5-FU treatment in CRC. SOX2-expressing organoids exhibit increased tolerance to chemotherapy treatment, whilst deletion of SOX2 in AKPT tumour organoids sensitises drug responses. We propose that SOX2-induced plasticity and reversible dormancy promotes tumour progression and drug tolerance in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb2c2dcfb70e2e15d1ce94c401c626a55033b7a7" target='_blank'>
              SOX2 drives fetal reprogramming and reversible dormancy in colorectal cancer
              </a>
            </td>
          <td>
            Anna Baulies, V. Moncho-Amor, Diana Drago-Garcia, Anna Kucharska, Probir Chakravarty, M. Moreno-Valladares, Sara Cruces-Salguero, Florian Hubl, Colin Hutton, Hanue Kim, A. Matheu, Robin Lovell-Badge, V. Li
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="
 Cell morphologies represent a phenotypic integration of intrinsic cellular processes, changes in cell state, and interactions with neighboring cells. While analyses of large cell line collections such as the Cancer Dependency Map (DepMap) have greatly enhanced discovery of novel molecular biomarkers and therapeutic targets, morphological analysis of these models has not yet been performed. Hence, there is a rich opportunity to discover novel regulators and functional properties of cell morphologies by linking image-based profiling to -omics based assays. Here, we studied the transcriptional and functional properties of 16 human pancreatic cancer cell lines, associations with distinct cell morphologies, and treatment-induced remodeling. At baseline, we observed extensive morphological heterogeneity across and within cell lines, as well as differences in multicellular organization. We categorized cell line morphologies into epithelial, mesenchymal, and spheroid, and organizational patterns into tightly aggregated, multilayered, and dispersed. We performed differential expression analysis across morphological subtypes utilizing DepMap transcriptomics data and queried functional correlates including CRISPR dependency, drug sensitivity, and proclivity for metastasis. For example, we discovered that the mesenchymal morphology and multilayered organizational pattern [WH3] were associated with metastatic potential, while expressing reduced levels of tight junction and cell adhesion genes. We then explored treatment associated morphological changes and their potential as cost-effective biomarkers for cell-intrinsic resistance to chemotherapy (e.g., 5-FU and gemcitabine) and KRAS inhibitors (e.g., MRTX1133 and RMC- 6236). Cells were treated continuously and monitored with live cell imaging (Incucyte SX5) followed by Cell Painting at the three-day endpoint. We observed conserved morphological responses to therapy across cell lines including a shift towards spindle-like cell shapes with neurite-like projections in chemotherapy-treated conditions, and a cell bloating phenomena characterized by an increase in cell area in KRAS inhibitor treated conditions. To directly link transcriptional state to morphology, we performed 1000-plex spatial molecular imaging (Nanostring CosMx) at subcellular resolution to pools of cell lines, demonstrating that morphological diversity within cell lines corresponded to distinct transcriptional states. In conclusion, this study highlights the potential of harnessing cell morphological information in a rapid, cost-effective phenotyping assay to aid precision oncology efforts leveraging patient- derived in vitro models.
 Citation Format: Dennis Gong, William L Hwang. Morphological diversity predicts functional traits in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B076.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ee8c48f6e344128f81cda79e9fefc7cbd605a4f" target='_blank'>
              Abstract B076: Morphological diversity predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, W. L. Hwang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer death worldwide, with poor survival despite recent therapeutic advances. A better understanding of the complexity of the tumor microenvironment is needed to improve patients’ outcome.We applied a computational immunology approach (involving immune cell proportion estimation by deconvolution, transcription factor activity inference, pathways and immune scores estimations) in order to characterize bulk transcriptomics of 62 primary lung adenocarcinoma (LUAD) samples from patients across disease stages. Focusing specifically on early stage samples, we validated our findings using an independent LUAD cohort with 70 bulk RNAseq and 15 scRNAseq datasets and on TCGA datasets.Through our methodology and feature integration pipeline, we identified groups of immune cells related to disease stage as well as potential immune response or evasion and survival. More specifically, we reported a duality in the behavior of immune cells, notably natural killer (NK) cells, which was shown to be associated with survival and could be relevant for immune response or evasion. These distinct NK cell populations were further characterized using scRNAseq data, showing potential differences in their cytotoxic activity.The dual profile of several immune cells, most notably T-cell populations, have been discussed in the context of diseases such as cancer. Here, we report the duality of NK cells which should be taken into account in conjunction with other immune cell populations and behaviors in predicting prognosis, immune response or evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c27b25a059c3fcc66883874b31718552b5cb7d85" target='_blank'>
              Transcriptomics profiling of the non-small cell lung cancer microenvironment across disease stages reveals dual immune cell-type behaviors
              </a>
            </td>
          <td>
            Marcelo Hurtado, Leila Khajavi, Abdelmounim Essabbar, Michael Kammer, Ting Xie, Alexis Coullomb, A. Pradines, A. Casanova, Anna Kruczynski, Sandrine Gouin, E. Clermont, Léa Boutillet, M. Senosain, Yong Zou, Shillin Zhao, Prosper Burq, Abderrahim Mahfoudi, Jerome Besse, Pierre Launay, Alexandre Passioukov, Eric Chetaille, Gilles Favre, Fabien Maldonado, F. Cruzalegui, O. Delfour, Julien Mazieres, Vera Pancaldi
          </td>
          <td>2024-10-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f337210da04933a9ae37820fdc6d7b7fe43d5d5" target='_blank'>
              CALB2 drives pancreatic cancer metastasis through inflammatory reprogramming of the tumor microenvironment
              </a>
            </td>
          <td>
            Jinxin Tao, Yani Gu, Zeyu Zhang, Guihu Weng, Yueze Liu, Jie Ren, Yanan Shi, J. Qiu, Yuanyang Wang, Dan Su, Ruobing Wang, Yifan Fu, Tao Liu, Liyuan Ye, W. Luo, Hao Chen, Gang Yang, Zhe Cao, Hua Huang, Jianchun Xiao, Bo Ren, Lei You, Taiping Zhang, Yupei Zhao
          </td>
          <td>2024-10-03</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Acquiring detailed spatial information about intratumor microbiota in situ is challenging, which leaves 3D distributions of microbiota within entire tumors largely unexplored. Here, a modified iDISCO-CUBIC tissue clearing and D-amino acid microbiome labeling-based (miCDaL) strategy are proposed, that integrates microbiota in situ labeling, tissue clearing, and whole-mount tissue imaging to enable 3D visualization of indigenous intratumor microbiota. Leveraging whole-mount spatial resolution and centimeter-scale imaging depth, the 3D biogeography of microbiota is successfully charted across various tumors at different developmental stages, providing quantitative spatial insights in relation to host tumors. By incorporating an immunostaining protocol, 3D imaging of the immunologic microenvironment is achieved in both murine and human mammary tumors that is previously assumed to be bacteria-free. Notably, immune infiltrates, including T cells and NK cells, and tertiary lymphoid structures are conspicuously absent in bacteria-colonized regions. This 3D imaging strategy for mapping Indigenous intratumor microbiota offers valuable insights into host-microbiota interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6fdb761437877e43564b6889311f1059c2228cb" target='_blank'>
              Whole-Tumor Clearing and Imaging of Intratumor Microbiota in Three Dimensions with miCDaL Strategy.
              </a>
            </td>
          <td>
            Yuezhou Wang, Zile Jiang, Kai Zhang, Huimin Tang, Guimei Wang, Jinshan Gao, Guanghui He, Baoyue Liang, Li Li, Chaoyong Yang, Xianming Deng
          </td>
          <td>2024-10-08</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="While collective migration is shown to enhance invasive and metastatic potential in cancer, the mechanisms driving this behavior and regulating tumor migration plasticity remain poorly understood. This study provides a mechanistic framework explaining the emergence of different modes of collective migration under hypoxia-induced secretome. We focus on the interplay between cellular protrusion force and cell-cell adhesion using collectively migrating three-dimensional microtumors as models with well-defined microenvironment. Large microtumors show directional migration due to intrinsic hypoxia, while small microtumors exhibit radial migration in response to hypoxic secretome. Here, we developed the minimal multi-scale microtumor model (MSMM) to elucidate underlying mechanisms. We identified distinct migration modes within specific regions of protrusion force and cell-cell adhesion parameter space. We show that sufficient cellular protrusion force is crucial for both, radial and directional collective microtumor migration. Radial migration emerges when sufficient cellular protrusion force is generated, driving neighboring cells to move collectively in diverse directions. Within migrating tumors, strong cell-cell adhesion enhances the alignment of cell polarity, breaking the symmetric angular distribution of protrusion forces, and leading to directional microtumor migration. The integrated results from the experimental and computational models provide fundamental insights into collective migration in response to different microenvironment stimuli.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48bc996f280a2ab475dfcb73e972ab4b63f451ce" target='_blank'>
              Protrusion force and cell-cell adhesion play a critical role in collective tumor cluster migration
              </a>
            </td>
          <td>
            Huijing Wang, Catalina Ardila, Ajita Jindal, Vaishali Aggarwal, Weikang Wang, Jonathan Vande Geest, Yi Jiang, Jianhua Xing, Shilpa Sant
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oncogene amplification on extrachromosomal DNA (ecDNA) is strongly associated with treatment resistance and shorter survival for patients with cancer, including patients with glioblastoma. The non-chromosomal inheritance of ecDNA during cell division is a major contributor to intratumoral genetic heterogeneity. At present, the spatial dynamics of ecDNA, and the impact on tumor evolutionary trajectories, are not well understood. Here, we investigate the spatial-temporal evolution of ecDNA and its clinical impact by analyzing tumor samples from 94 treatment-naive human IDH-wildtype glioblastoma patients. We developed a spatial-temporal computational model of ecDNA positive tumors (‘SPECIES’) that integrates whole-genome sequencing, multi-region DNA FISH, and nascent RNAscope, to provide unique insight into the spatial dynamics of ecDNA evolution. Random segregation in combination with positive selection of ecDNAs induce large, predictable spatial patterns of cell-to-cell ecDNA copy number variation that are highly dependent on the oncogene encoded on the circular DNA. EGFR ecDNAs often reach high mean copy number (mean of 50 copies per tumor cell), are under strong positive selection (mean selection coefficient, s > 2) and do not co-amplify other oncogenes on the same ecDNA particles. In contrast, PDGFRA ecDNAs have lower mean copy number (mean of 15 copies per cell), are under weaker positive selection and frequently co-amplify other oncogenes on the same ecDNA. Evolutionary modeling suggests that EGFR ecDNAs often accumulate prior to clonal expansion. EGFR structural variants, including vIII and c-terminal deletions are under strong positive selection, are found exclusively on ecDNA, and are intermixed with wild-type EGFR ecDNAs. Simulations show EGFRvIII ecDNA likely arises after ecDNA formation in a cell with high wild-type EGFR copy number (> 10) before the onset of the most recent clonal expansion. This remains true even in cases of co-selection and co-amplification of multiple oncogenic ecDNA species in a subset of patients. Overall, our results suggest a potential time window in which early ecDNA detection may provide an opportunity for more effective intervention. Highlights ecDNA is the most common mechanism of focal oncogene amplification in IDHwt glioblastoma. EGFR and its variants on ecDNA are particularly potent, likely arising early in tumor development, providing a strong oncogenic stimulus to drive tumorigenesis. Wild-type and variant EGFR ecDNA heteroplasmy (co-occurrence) is common with EGFRvIII or c-terminal deletions being derived from EGFR wild-type ecDNA prior to the most recent clonal expansion. Tumors with ecDNA amplified EGFR versus PDGFRA exhibit different evolutionary trajectories. SPECIES model can infer spatial evolutionary dynamics of ecDNA in cancer. A delay between ecDNA accumulation and subsequent oncogenic mutation may give a therapeutic window for early intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e8d31fa2dc1841943bfa95252e831edb576e77" target='_blank'>
              Extrachromosomal DNA driven oncogene spatial heterogeneity and evolution in glioblastoma
              </a>
            </td>
          <td>
            I. Noorani, M. Haughey, J. Luebeck, A. Rowan, E. Grönroos, Francesco Terenzi, I. Wong, Jeanette Kittel, Chris Bailey, C. Weeden, Donald M Bell, Eric Joo, Vittorio Barbè, Matthew G Jones, E. Nye, M. Green, Lucy Meader, E. Norton, Mark Fabian, N. Kanu, M. Jamal-Hanjani, T. Santarius, James A R Nicoll, D. Boche, Howard Y. Chang, V. Bafna, Weini Huang, P. Mischel, C. Swanton, Benjamin Werner
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is one of the most lethal cancers, with a 5-year survival rate of only 13%. PDAC develops from precursor lesions, most commonly Pancreatic Intraepithelial Neoplasia (PanIN). Studying PanIN in human patients is difficult due to its microscopic nature and the lack of indication for sampling the pancreas in the absence of disease. In collaboration with Gift of Life - Michigan, a local organ and tissue donor organization, we obtained over 80 pancreata for research. Surprisingly, we found PanIN in the vast majority of the organs, regardless of the donor’s age; the prevalence of PanIN compared to the relatively low incidence of PDAC indicates that most lesions are destined not to progress. To gather insight into factors that might distinguish lesions designed to progress from lesions that are likely to remain dormant, we sought to characterize both epithelial cells and their supporting microenvironment. Using spatial transcriptomics, we determined that PanIN epithelial cells are transcriptionaally similar to pancreatic cancer; further, limited genetic characterization revealed that they contain similar oncogenic KRAS mutations as pancreatic malignancy. We then sought to determine whether factors in the microenvironment distinguish sporadic PanIN from those associated with overt malignancy. We used 10x Genomic CytAssist Visium technology and profiled an initial batch of 7 tissue sections from donor pancreata and an equal number from tumors. We employed the BayesPrism algorithm to deconvolute the Visium data (which lacked single-cell resolution), using our single-cell RNAseq atlas to define stroma- and epithelial-specific expression profiles. We then segmented the tissue into epithelial and stromal compartments. We further identified various stromal subpopulations, with macrophages and fibroblasts being the most abundant cell types. We defined subpopulations of epithelial cells, fibroblasts, and macrophages through unsupervised clustering. In both donor samples and tumors, we identified heterogeneous populations of stromal cells present in specific geographic patterns, with PanIN-adjacent stroma clearly distinct from stroma surrounding healthy ducts. Cell-cell neighborhood analysis revealed a distinct geographic arrangement of epithelial and stromal compartments, indicating potential communication patterns. Independent unsupervised statistical analysis using Non-negative Matrix Factorization (NMF) using Coordinated Gene Association in Pattern Sets (CoGAPS) R package also revealed similar patterns. Our next steps involve validating gene expression signatures that define each subpopulation and identifying tumor-specific or PanIN-specific genes. Ultimately, we aim to investigate interactions between epithelial cells, fibroblasts, and macrophages to identify potential tumor-promoting or tumor-restraining pathways.
 Citation Format: Ahmed M. Elhossiny, Jude Okoye, Imade Williams, Yaqing Zhang, Hannah Watkoske, Atul Deshpande, Elana Fertig, Arvind Rao, Eileen S. Carpenter, Timothy L. Frankel, Marina Pasca di Magliano. Spatial transcriptomics reveals heterogeneity of epithelial and stromal compartments in healthy and tumor pancreas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A046.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f587649d06619c2f34a308f750ab4e8707c4ec42" target='_blank'>
              Abstract A046: Spatial transcriptomics reveals heterogeneity of epithelial and stromal compartments in healthy and tumor pancreas
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, Jude Okoye, Imade Williams, Yaqing Zhang, Hannah R Watkoske, Atul Deshpande, E. Fertig, Arvind Rao, Eileen S Carpenter, Timothy L. Frankel, M. Pasca di Magliano
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Simple Summary Glioblastoma is a highly aggressive tumor, and despite standard-of-care therapy, tumor recurrence is inevitable. Recurrent glioblastoma (rGBM) remains a challenge to treat, given that it predominantly forms within a highly inflammatory tumor microenvironment (TME) that contributes to treatment resistance. However, the complex immune landscape of rGBM and how to effectively characterize it for therapeutic targeting remains poorly understood. To address this gap, this review exhaustively examines the existing body of literature on the immune characteristics of rGBM with a focus on the role of glioma-associated microglia and macrophages (GAMMs). We examine this inflammatory landscape through the lens of histological-based assessments given histological staining remains the gold standard of diagnostic identification of rGBM. We review canonical and emerging cell-specific markers of GAMMs and provide a guide of how these histological-based assessments can aid in characterizing the TME of rGBM and dissect how this landscape shifts in the context of treatment response. Abstract The glioblastoma (GBM) tumor microenvironment consists of a heterogeneous mixture of neoplastic and non-neoplastic cells, including immune cells. Tumor recurrence following standard-of-care therapy results in a rich landscape of inflammatory cells throughout the glioma-infiltrated cortex. Immune cells consisting of glioma-associated macrophages and microglia (GAMMs) overwhelmingly constitute the bulk of the recurrent glioblastoma (rGBM) microenvironment, in comparison to the highly cellular and proliferative tumor microenvironment characteristic of primary GBM. These immune cells dynamically interact within the tumor microenvironment and can contribute to disease progression and therapy resistance while also providing novel targets for emerging immunotherapies. Within these varying contexts, histological-based assessments of immune cells in rGBM, including immunohistochemistry (IHC) and immunofluorescence (IF), offer a critical way to visualize and examine the inflammatory landscape. Here, we exhaustively review the available body of literature on the inflammatory landscape in rGBM as identified through histological-based assessments. We highlight the heterogeneity of immune cells throughout the glioma-infiltrated cortex with a focus on microglia and macrophages, drawing insights from canonical and novel immune-cell histological markers to estimate cell phenotypes and function. Lastly, we discuss opportunities for immunomodulatory treatments aiming to harness the inflammatory landscape in rGBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4001488ad8c82d2b66d1744897ff8a71e5f6d40" target='_blank'>
              Unveiling the Inflammatory Landscape of Recurrent Glioblastoma through Histological-Based Assessments
              </a>
            </td>
          <td>
            Nicholas B. Dadario, D. Boyett, Damian E Teasley, Peter J. Chabot, Nathan J. Winans, Michael G. Argenziano, Colin P. Sperring, P. Canoll, Jeffrey N Bruce
          </td>
          <td>2024-09-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related deaths globally, with its complexity driven by intricate and intertwined genetic, epigenetic, and environmental factors. Despite advances in genomics, transcriptomics, and proteomics, understanding the phenotypic diversity of lung cancer has lagged behind. Next-generation phenomics, which integrates high-throughput phenotypic data with multiomics approaches and digital technologies such as artificial intelligence (AI), offers a transformative strategy for unraveling the complexity of lung cancer. This approach leverages advanced imaging, single-cell technologies, and AI to capture dynamic phenotypic variations at cellular, tissue, and whole organism levels and in ways resolved in temporal and spatial contexts. By mapping the high-throughput and spatially and temporally resolved phenotypic profiles onto molecular alterations, next-generation phenomics provides deeper insights into the tumor microenvironment, cancer heterogeneity, and drug efficacy, safety, and resistance mechanisms. Furthermore, integrating phenotypic data with genomic and proteomic networks allows for the identification of novel biomarkers and therapeutic targets in ways informed by biological structure and function, fostering precision medicine in lung cancer treatment. This expert review examines and places into context the current advances in next-generation phenomics and its potential to redefine lung cancer diagnosis, prognosis, and therapy. It highlights the emerging role of AI and machine learning in analyzing complex phenotypic datasets, enabling personalized therapeutic interventions. Ultimately, next-generation phenomics holds the promise of bridging the gap between molecular alterations and clinical and population health outcomes, providing a holistic understanding of lung cancer biology that could revolutionize its management and improve patient survival rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eff2978937e2ef89767871d93305b0ec9138e48" target='_blank'>
              Next-Generation Cancer Phenomics: A Transformative Approach to Unraveling Lung Cancer Complexity and Advancing Precision Medicine.
              </a>
            </td>
          <td>
            Sanjukta Dasgupta
          </td>
          <td>2024-10-22</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a lethal malignancy due to overwhelming metastatic burden. PDAC cells migrate and invade by two distinct epithelial-to-mesenchymal transition phenotypes: collective (partial-EMT) or single-cell (complete-EMT). The mechanism(s) by which these invasive phenotypes intravasate is unknown. The Tumor Microenvironment of Metastasis (TMEM) doorway, which is formed by a tumor cell, macrophage and an endothelial cell in direct contact, causes opening of the vasculature in which tumor cells intravasate. Intravasation via TMEM doorway is inhibited by Tie2 blockade. Whether this occurs in PDAC and whether the EMT phenotype affects intravasation is unknown. We hypothesize that PDAC cells intravasate through TMEM-dependent mechanisms regardless of EMT phenotype. Tumor samples from murine models were stained for TMEM doorways using a triple immunohistochemistry (IHC) stain (tumor cells expressing the actin regulatory protein Mena, macrophages expressing CD68, and endothelial cells expressing CD31). TMEM doorway opening was assessed in murine models of PDAC treated with or without Tie-2 inhibitor by analyzing TMEM-mediated extravasation of high-molecular weight dextran (155kD-TMR dextran) by Intravital Imaging, IHC and immunofluorescence (IF) of serial slides and circulating tumor cells. An intravasation trans-endothelial migration assay was also performed to assess the intravasation event by TMEM doorway in vitro. A student t-test was used for the analyses. TMEM doorways were observed in all PDAC murine models. For the first time in PDAC, focal extravasation of dextran events followed by a tumor cell intravasation was visualized by intravital imaging. PDAC tumor bearing mice treated with Tie2 inhibitor had a 2-fold decrease in the number of circulating tumor cells compared to control (958 ± 499.8 vs 534 ± 479.9 cells/mL blood, p=0.0257). The intravasation trans-endothelial migration assay showed that C-EMT cells have a higher migration rate than P-EMT cells (17.8 ± 5.53 vs 10.94 ± 3.84 migrating cells p=0.01), and this ratio increases with the presence of macrophages (23.75 ± 5 vs 15.54 ± 3.09 migrating cells p=0.005). The Tie-2 inhibition decreases the number of C-EMT and P-EMT migrating cells (7.69 ± 1.3 vs 3.67 ± 1.2 migrating cells p=0.01). Orthotopic tumors from C-EMT phenotype grow at a faster rate than the P-EMT (0.49mg ± 0.09 vs 0.12mg ± 0.05 p<0.05). Tumors from C-EMT and P-EMT cells will be analyzed for TMEM doorway score, TMEM activity and stemness signatures around TMEM doorway to determine if TMEM doorway is a niche for stemness in PDAC. TMEM doorways are present in PDAC mouse model. Inhibition of Tie2 leads to decreased TMEM doorway function and circulating tumor cells in PDAC. In addition, regarding the EMT phenotype, TMEM doorways are functional mechanisms of intravasation in vitro. This is a novel mechanism in PDAC and suggests a promising therapeutic target for decreasing intravasation and potential metastasis in PDAC.
 Citation Format: Erika Pereira Zambalde, Lina Ariyan, Francisco Puerta-Martinez, Yuqin Zhu, John Condeelis, Ben Stanger, John McAuliffe. Understanding the events that promote pancreatic cancer metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A059.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bb4fb69cd1b8a364a616f674f6098ab7848f20f" target='_blank'>
              Abstract A059: Understanding the events that promote pancreatic cancer metastasis
              </a>
            </td>
          <td>
            É. P. Zambalde, Lina Ariyan, Francisco Puerta-Martinez, Yuqin Zhu, J. Condeelis, Ben Z. Stanger, John C. McAuliffe
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Xenograft models are attractive models that mimic human tumor biology and permit one to perturb the tumor microenvironment and study its drug response. Spatially resolved transcriptomics (SRT) provide a powerful way to study the organization of xenograft models, but currently there is a lack of specialized pipeline for processing xenograft reads originated from SRT experiments. Xenomake is a standalone pipeline for the automated handling of spatial xenograft reads. Xenomake handles read processing, alignment, xenograft read sorting, and connects well with downstream spatial analysis packages. We additionally show that Xenomake can correctly assign organism specific reads, reduce sparsity of data by increasing gene counts, while maintaining biological relevance for studies.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bb323ed1a30af4078c10507f1902d83b76e6d0" target='_blank'>
              Xenomake: A pipeline for processing and sorting xenograft reads from spatial transcriptomic experiments.
              </a>
            </td>
          <td>
            Benjamin S Strope, Katherine E Pendleton, William Z Bowie, Gloria V. Echeverria, Qian Zhu
          </td>
          <td>2024-10-14</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Pancreatic cancer remains one of the most deadly cancers, with over 450,000 fatalities in 2022 alone. The most lethal variation of this cancer is Pancreatic Ductal Adenocarcinoma (PDAC), with a 5-year survival rate of under 10%. Combination chemotherapy has provided major advances in this disease but it remains challenging for many reasons including the immunosuppressive nature of the tumor microenvironment (TME). PDAC has a highly desmoplastic microenvironment that suppresses antitumor immunity and promotes tumor cell invasion, metastasis, and resistance through epithelial to mesenchymal transition (EMT) of cancer cells. Cancer cells can undergo various forms of cell death, notably apoptosis, necrosis, autophagy, pyroptosis, and ferroptosis. These mechanisms are triggered by different cellular stressors and shape the microenvironment. Through the use of 15 Visium spatial transcriptomics samples from publicly available primary PDAC tumors and paired single-cell RNA (scRNA) data, we aim to understand how cell stress and cell death shape the tumor microenvironment (TME) and epithelial-mesenchymal transition (EMT). By leveraging information from known pathways in literature, we will identify cellular programs related to stress, death, and EMT. We will explore intracellular correlations within single-cell data and extracellular interactions within the TME using spatial data to uncover how different forms of cell death and stress signatures influence the TME and EMT spectrum. The initial phase involves identifying the types of stress that lead to various forms of cell death intracellularly and validating these gene signatures. Subsequently, we will apply these validated signatures to the spatial data to correlate each score and search for significant overlapping programs. This will extend beyond stress and death, incorporating TME signatures corresponding to different cell types and the EMT spectrum. By understanding the signals that promote the death of various cell populations, we aim to identify combination treatments that target both ends of the EMT spectrum. Additionally, this research will provide insights into the inflammatory impact of cell death on the TME. Our approach aims to inform the development of therapies that disrupt resistance mechanisms, ultimately improving treatment outcomes for patients with PDAC. This research can potentially reveal new therapeutic targets and strategies, enhancing our ability to combat this deadly disease.
 Citation Format: Rahul Bansal, Izabella Zamora, Samuel Wright, Nir Hacohen, Arnav Mehta. Influence of Cell Death and Stress on the Tumor Microenvironment and EMT in Pancreatic Ductal Adenocarcinoma Through Single Cell and Spatial Transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba425dbbc0b9c870572a7ccded079a29b6e67232" target='_blank'>
              Abstract C074: Influence of Cell Death and Stress on the Tumor Microenvironment and EMT in Pancreatic Ductal Adenocarcinoma Through Single Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Rahul Bansal, Izabella Zamora, Samuel J Wright, N. Hacohen, Arnav Mehta
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="The rapid development of spatial transcriptomics (ST) technologies has enabled transcriptome-wide profiling of gene expression in tissue sections. Despite the emergence of single-cell resolution platforms, most ST sequencing studies still operate at a multi-cell resolution. Consequently, deconvolution of cell identities within the spatial spots has become imperative for characterizing cell type-specific spatial organization. To this end, we developed SpatialDeX, a regression model-based method for estimating cell type proportions in tumor ST spots. SpatialDeX exhibited comparable performance to reference-based methods and outperformed other reference-free methods with simulated ST data. Using experimental ST data, SpatialDeX demonstrated superior performance compared with both reference-based and reference-free approaches. Additionally, a pan-cancer clustering analysis on tumor spots identified by SpatialDeX unveiled distinct tumor progression mechanisms both within and across diverse cancer types. Overall, SpatialDeX is a valuable tool for unraveling the spatial cellular organization of tissues from ST data without requiring scRNA-seq references.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd5599937aad32ad900e85b0269ce97f19907f24" target='_blank'>
              SpatialDeX is a Reference-Free Method for Cell Type Deconvolution of Spatial Transcriptomics Data in Solid Tumors.
              </a>
            </td>
          <td>
            Xinyi Liu, Gongyu Tang, Yuhao Chen, Yuanxiang Li, Hua Li, Xiaowei Wang
          </td>
          <td>2024-10-10</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs), and circulating tumor related cells, are extremely rare cells that intravasate from the tumor into the circulatory system and can be captured via a liquid biopsy. Although CTCs contribute to the metastatic cascade, and diverse phenotypes of CTCs have been observed – including the cytokeratin expressing CTCs, CTC clusters, large polyploid CTCs, and CTCs undergoing epithelial to mesenchymal transition – little is known about their functionality. By virtue of CTCs being rare, a detection method that maximizes the information obtained per cell would be ideal to understand their biology and for use in diagnostic approaches. The challenge is that rare cell detection necessitates extensive processing steps, during which molecular content, such as RNA and DNA, needs to be preserved for downstream single cell analysis. We developed GEnomic, Transcriptomic, and MOrphological profiling of Rare cells (GeTMoR), a method that extends the High Definition Single Cell Assay for detecting rare cancer related cells to simultaneously image and profile the genome and transcriptome from single rare cells. We validated GeTMoR by spiking in cancer cell lines into whole blood to evaluate the quality of recovered gene expression and copy number profile from the same cell. The GeTMoR approach provides the ability to link the phenotype of rare cells, including CTCs, to their genome and transcriptome, thereby enabling insight into rare cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d6df3d1f532e0cca0520b19b2419d2d9cd51ad9" target='_blank'>
              GeTMoR: Simultaneous genomic, transcriptomic, and morphological profiling of rare single cells
              </a>
            </td>
          <td>
            Rishvanth K. Prabakar, Michael J. Schmidt, Peter Kuhn, James B. Hicks
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is a heterogenous disease comprised of malignant cell states and an immunosuppressive microenvironment that underlie radiotherapy (RT) resistance. Here we integrate genome wide and single cell in vivo functional genomic screens of preclinical models with single nuclei and spatial transcriptomics of human tumors to identify therapeutic vulnerabilities driving GBM radioresistance.



 Perturb-seq of intracranial GBM models was performed using human (GBM43) or mouse (GL261, SB28) GBM cells expressing CRISPR interference (CRISPRi) machinery, with sgRNA libraries delivered to intracranial tumors using convection enhanced delivery. Genome wide dual sgRNA CRISPRi screens were performed in GBM cells (GBM43, LN18, GL261, SB28) ± RT. Functional genomic analyses across preclinical models were integrated with single nuclei and spatial transcriptomics from human GBMs. Validation of genetic phenotypes was performed using the DNA-PK inhibitor peposertib to treat GBM cells and orthotopic mouse models (SB28, GBM43, GBM6), which were analyzed using IF and single cell RNA sequencing.



 Genome wide CRISPRi screens identified 114 radiation sensitizing targets. In vivo perturb-seq of radiation sensitizing targets reprogrammed 25 gene modules that spanned homeostatic, metabolic, and inflammatory pathways and nominated DNA-PK as a pleotropic radiation sensitizer that regulates interactions between malignant cells and the myeloid microenvironment. Genetic and pharmacologic inhibition of DNA-PK revealed radiotherapy dependent reprograming of cell states through cell intrinsic mechanisms (e.g. cell cycle, proteasome) and cell extrinsic pathways (e.g. TNFα, IFN). Peposertib sensitized GBMs to radiotherapy and extended survival in mice harboring intracranial xenografts. DNA-PK inhibition induced cGAS/STING and IFN signaling in malignant cells (n = 58,161) that were associated with pro-inflammatory polarization of infiltrating myeloid cells (n = 22,067). These alterations were synergistic with RT but were not induced by RT alone. Spatial transcriptomics of 8 matched primary-recurrent GBMs (20,793 transcriptomes) from patients treated with RT and peposertib, and single nuclei transcriptomics from 100 GBMs (76 matched primary-recurrent tumors, 4 after treatment with RT and peposertib, 356,997 nuclei) demonstrated malignant cell differentiation and enrichment of pro-inflammatory macrophages in the tumor microenvironment after inhibition of DNA-PK.



 Integration of genome wide CRISPRi screens and in vivo perturb-seq in preclinical models with single nuclei and spatial transcriptomic analyses of human tumors enables discovery and mechanistic validation of therapeutic targets in GBM. These results reveal DNA-PK as a target for radiation sensitization of tumor and myeloid microenvironment cells in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59313aeff0a382ec9e51c8a10cae8b225d6db462" target='_blank'>
              OS07.1.A INTEGRATIVE SINGLE NUCLEI, SPATIAL, AND FUNCTIONAL GENOMIC ANALYSES REVEAL DNA-PK DRIVES GLIOBLASTOMA RADIORESISTANCE THROUGH TUMOR CELLS AND THE MYELOID MICROENVIRONMENT
              </a>
            </td>
          <td>
            J. Liu, N. Majd, C. Zou, K. Mirchia, J. Pak, K. Foster, K. Lin, T. Casey-Clyde, K. Seo, S. Pan, E. Payne, H. Vasudevan, T. Ozawa, M. Berger, J. de Groot, W. Weiss, L. Gilbert, D. Raleigh
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor heterogeneity and plasticity, driven by Epithelial-Mesenchymal Transition (EMT), enable cancer therapeutic resistance. We previously showed that EMT promotes primary cilia formation, which enables stemness and tumorigenesis in triple-negative breast cancer (TNBC). Here, we establish a role for primary cilia in human TNBC chemotherapeutic resistance. We developed patient-derived organoids, and showed that these recapitulated the cellular heterogeneity of TNBC biopsies. Notably, one of the identified cell states bore a quasi-mesenchymal phenotype, primary cilia, and stemness signatures. We treated our TNBC organoids with chemotherapeutics and observed partial killing. The surviving cells with organoid-reconstituting capacity showed selective enrichment for the quasi-mesenchymal ciliated cell subpopulation. Genomic analyses argue that this enrichment reflects a combination of pre-existing cells and ones that arose through drug-induced cellular plasticity. We developed a family of small-molecule inhibitors of ciliogenesis and show that these, or genetic ablation of primary cilia, suppress chemoresistance. We conclude that primary cilia help TNBC to evade chemotherapy. Significance Cancer cells that activate EMT to acquire a quasi-mesenchymal state form primary cilia to evade chemotherapy in human triple-negative breast cancer. Pharmacological inhibition of primary ciliogenesis counteracts EMT-induced chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dd368daa4ef108513a9f36d1e4d1835f96618d2" target='_blank'>
              Primary cilia promote EMT-induced triple-negative breast tumor heterogeneity and resistance to therapy
              </a>
            </td>
          <td>
            Camille E. Tessier, J. Derrien, Aurore M M Dupuy, Thomas Pelé, Martin Moquet, Julie Roul, E. Douillard, Camille El Harrif, Xavier Pinson, Matthieu Le Gallo, Florence Godey, Patrick Tas, Roselyne Viel, Claude Prigent, Éric Letouzé, Peggy Suzanne, Patrick Dallemagne, M. Campone, Robert A. Weinberg, Jacqueline A. Lees, Philippe P. Juin, Vincent J Guen
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer progression and response to therapy are inextricably reliant on the co-evolution of a supportive tissue microenvironment. This is particularly evident in pancreatic ductal adenocarcinoma (PDAC), a tumor type characterized by expansive and heterogeneous stroma. Herein, we employed single cell RNAseq and spatial transcriptomics of normal, inflamed, and malignant pancreatic tissues to contextualize stromal dynamics associated with disease and treatment status, identifying temporal and spatial trajectories of fibroblast differentiation. Using analytical tools to infer cellular communication, together with a newly developed assay to annotate genomic alterations in cancer cells, we additionally explored the complex intercellular networks underlying tissue circuitry, highlighting a fibroblast-centric interactome that grows in strength and complexity in the context of malignant transformation. Our study yields new insights on the stromal remodeling events favoring the development of a tumor-supportive microenvironment and provides a powerful resource for the exploration of novel points of therapeutic intervention in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/106968e3649b3283e96d5bfd01ae138fc25d6cc4" target='_blank'>
              Dynamic evolution of fibroblasts revealed by single cell RNA sequencing of human pancreatic cancer.
              </a>
            </td>
          <td>
            S. Dimitrieva, Jon M Harrison, Jonathan Chang, M. Piquet, M. Mino-Kenudson, Millicent Gabriel, Vivek Sagar, Heiko Horn, K. Lage, Julie Kim, Gang Li, Shaobu Weng, Cynthia Harris, Anupriya S Kulkarni, David T Ting, M. Qadan, Peter J Fagenholz, Cristina R Ferrone, Angelo L Grauel, Tyler Laszewski, Alina Raza, Markus Riester, Tim Somerville, Joel P Wagner, Glenn Dranoff, J. Engelman, Audrey Kauffmann, Rebecca Leary, Andrew L Warshaw, K. Lillemoe, C. F. Fernandez-Del Castillo, David A Ruddy, Andrew S. Liss, Viviana Cremasco
          </td>
          <td>2024-11-01</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>130</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is one of the deadliest types of cancer with median overall survival of little more than a year. It is the prototypic example of a molecularly diverse tumor associated with multiple subclones and high transcriptional plasticity. We here set out to map the clonal architecture and link it with growth patterns, transcriptional states and drug response in-vitro.



 Seventy-eight multiregional samples from 24 GBM patients (F/M ratio = 0.60, median age= 65.5ys) were profiled using deep whole-exome sequencing with a nominal coverage of >300x. Clonal complexity was assessed by the Euclidean distance based on somatic mutations between regions. Molecular distance was integrated with 3D neuronavigation-based physical distances and patient survival. Fresh tumor slices of 8 patients were cultured in minimal essential media and treated with 4.2Gy plus 200µM temozolomide for 48h. Spatial Transcriptomics was performed on matched tumor slices with /without treatment.



 Integrative characterization revealed two evolutionary trajectories termed expansive and stochastic diversification based on the correlation between physical and molecular distance (RE=0.6, RS=-0.2). High molecular distance was associated with reduced PFS (R=-0.5538, p=0.026). Tumors with expansive evolution demonstrated volume-based growth that directly correlated with increased molecular diversification, while stochastic models showed irregular and discordant patterns. Patients with stochastic evolution were characterized by unfavorable prognosis (p= 0.35) and tumors appeared more frequently in the frontal lobe. In-vitro drug response varied considerably between patients based on cell proliferation (minus 0.5-13.8%), apoptosis (30.1-44.4%), and viability (minus 7.8-34.1%). Under treatment an increase of mesenchymal states with peculiar redistribution of transcriptional patterns and increased clonal complexity was observed.



 In sum, we report on an ongoing study in glioblastoma that seeks to link the clonal architecture with tumor growth, transcriptional patterns and in-vitro drug response. Our study will provide new insights into the clinical relevance of the clonal composition of this deadly cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f41aef776a5ac27081fe2094b9b9942292ad4e2a" target='_blank'>
              P01.02.B CLONAL ARCHITECTURE, 3D TUMOR GROWTH, TRANSCRIPTIONAL STATES AND IN-VITRO DRUG RESPONSE IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Y. Song, Y. Miller-Michlits, L. Greutter, K. Nenning, B. Kiesel, D. Loetsch-Gojo, V. Redl, J. Gesperger, J. Jana Breu, D. Nam, G. Widhalm, J. K. Sa, A. Woehrer
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cervical cancer remains the fourth most common cancer affecting women worldwide, and incidences of other HPV-related cancers continue to rise. For the development of effective prevention strategies in high-risk patients, we aimed to better understand the roles of inflammatory pathways and the tumour microenvironment as the main driver of progression to malignancy in HPV-infected tissues. We analysed the spatial organisation of seven samples of HPV+ high-grade squamous intraepithelial lesion (HSIL) and cervical intraepithelial neoplasia 3 (CIN3), comparing tumour heterogeneity and immune microenvironments between pre-malignant (neoplastic) and adjacent cervical tissues. We observed evidence of immune suppression within the neoplastic regions across all samples and identified distinct immune clusters for each dysplastic lesion. Previous single-cell data analyses in an HPV16 E7 oncoprotein-driven transgenic mouse model suggested a potential role for IL34-CSF1R signalling in immune modulation, where low IL34 expression was associated with Langerhans cell dysfunction, and, in cervical cancer, with poor patient outcome. Here, we observed that IL34-CSF1R co-expression was absent within HPV-associated neoplastic tissues but present in adjacent normal tissue regions. Additionally, we identified enrichment of an M2 gene signature in neoplastic tissue, while adjacent tissue was enriched with a pro-inflammatory M1 gene signature. Our findings provide bio-pathological insights into the spatial cellular and molecular mechanisms underlying HPV-associated cervical cancer immune regulation and suggest a strategy to modulate the immune system in HPV-positive neoplastic cervical and other tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78e69a13d70e65b0ae755f807b9334672667003c" target='_blank'>
              Spatial analysis of HPV associated cervical intraepithelial neoplastic tissues demonstrate distinct immune signatures associated with cervical cancer progression
              </a>
            </td>
          <td>
            Gianna Pavilion, Hani Vu, Zherui Xiong, Thi Viet Trinh Dang, Michael Walsh, Andrew Causer, J. Chandra, Quan Nguyen, Ian H. Frazer
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbcb18aaf8922f16ad19ad9eeb7b0ec8bc2789d" target='_blank'>
              Serial single-cell RNA sequencing unveils drug resistance and metastatic traits in stage IV breast cancer
              </a>
            </td>
          <td>
            K. Otsuji, Yoko Takahashi, Tomo Osako, Takayuki Kobayashi, T. Takano, S. Saeki, Liying Yang, Satoko Baba, Kohei Kumegawa, Hiromu Suzuki, Tetsuo Noda, Kengo Takeuchi, Shinji Ohno, T. Ueno, Reo Maruyama
          </td>
          <td>2024-10-03</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e8c0aed3c023b673b9bb62e9f661eb422606ebf" target='_blank'>
              A deep-learning model for characterizing tumor heterogeneity using patient-derived organoids
              </a>
            </td>
          <td>
            Kosuke Takagi, Motoki Takagi, Gen Hiyama, Kazuhito Goda
          </td>
          <td>2024-10-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Macrophages are primary cells of the innate immune system that mediate tumor progression. However, the motile behavior of macrophages and interactions with tumor cells are not well understood. Here, we exploit the optical transparency of larval zebrafish and perform real time imaging of macrophage-melanoma interactions. We found that macrophages are highly motile in the tumor microenvironment. Macrophages extend dynamic projections between tumor cells that precedes invasive melanoma migration. Modulating macrophage motility with a dominant inhibitory mutation in Rac2 inhibits recruitment to the tumor and impairs tumor invasion. However, a hyperactivating mutation in Rac2 does not affect macrophage recruitment but limits macrophage projections into the melanoma mass and reduces invasive melanoma cell migration. Taken together, these findings reveal a role for Rac2-mediated macrophage protrusive motility in melanoma invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d3962c28fb4273477445616dcfb2ada19a4a588" target='_blank'>
              Real-time imaging reveals a role for macrophage protrusive motility in melanoma invasion
              </a>
            </td>
          <td>
            Gayathri Ramakrishnan, Veronika Miskolci, Miranda Hunter, Morgan A. Giese, Daniela Münch, Yiran Hou, Kevin W. Eliceiri, Michael R. Lasarev, Richard M. White, Anna Huttenlocher
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The field cancerization theory suggests that a group of cells containing oncogenic mutations are predisposed to transformation1, 2. We previously identified single cells in BRAFV600E;p53-/-zebrafish that reactivate an embryonic neural crest state before initiating melanoma3–5. Here we show that single cells reactivate the neural crest fate from within large fields of adjacent abnormal melanocytes, which we term the “cancer precursor zone.” These cancer precursor zone melanocytes have an aberrant morphology, dysplastic nuclei, and altered gene expression. Using single cell RNA-seq and ATAC-seq, we defined a distinct transcriptional cell attractor state for cancer precursor zones and validated the stage-specific gene expression initiation signatures in human melanoma. We identify the cancer precursor zone driver, ID1, which binds to TCF12 and inhibits downstream targets important for the maintenance of melanocyte morphology and cell cycle control. Examination of patient samples revealed precursor melanocytes expressing ID1, often surrounding invasive melanoma, indicating a role for ID1 in early melanomagenesis. This work reveals a surprising field effect of melanoma initiation in vivo in which tumors arise from within a zone of morphologically distinct, but clinically covert, precursors with altered transcriptional fate. Our studies identify novel targets that could improve early diagnosis and prevention of melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96eaad85df9cc9f05779363a3326ab27df67629f" target='_blank'>
              An attractor state zone precedes neural crest fate in melanoma initiation
              </a>
            </td>
          <td>
            Alicia M. McConnell, Maggie H. Chassé, H. Noonan, Jeffrey K. Mito, Julia Barbano, Erika Weiskopf, Irene F. Gosselink, Meera Prasad, Song Yang, P. Abarzua, C. G. Lian, George F. Murphy, C. Trapnell, Leonard I. Zon
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal solid tumor with an unfavorable prognosis, often resistant to conventional treatments due to its dense stroma rich in Cancer- Associated Fibroblasts (CAFs). Limited T cell infiltration and prevalence of immunosuppressive cells within the tumor hinder the efficacy of immune checkpoint inhibitors (ICIs). The objective of our project is to modify immune landscape of PDAC tumor microenvironment (TME) to induce ICIs sensitization. We thus develop novel therapeutic strategies combining an immunomodulatory cytokine Interleukine-15 (IL15), with conventional chemotherapies (Gemcitabine or Folfirinox). To accurately replicate the complex cancer-stroma interactions within the pancreatic tumor microenvironment, we constructed a three-dimensional in vitro heterospheroid model composed of xenograft-derived tumor cells from PDAC patients, both primary or immortalized CAFs, and peripheral blood mononuclear cells (PBMCs) from healthy donors. Human 3D models were first set up and characterized by cytometry, imaging mass spectrometry and immunohistochemistry. We showed that, CAFs produce components of the extracellular matrix, promote tumor cell growth, improve resistance to chemotherapy and are able to down-regulate immune cell infiltration and modulate the nature of infiltrated immune cells. Spatial analysis of spheroids using imaging mass cytometry show that CAF influences the distribution of immune cells in our models. Interestingly, Interleukine 15 treatment prompted robust infiltration of PBMCs into heterospheroids. Immunophenotyping experiments revealed a substantial shift in the nature of immune infiltration, marked by a pronounced increase in CD4+ and CD8+ T lymphocytes, gDelta T cell and NK cell populations. When combined with Gemcitabine and IL15, immune effector cell infiltration is remarkably increased, resulting in antitumoral effects and control of tumor cell growth. Moreover, therapeutic combinations induce activation of infiltrated immune effector cells with an increase of activation, degranulation and cytotoxic markers, such as IFNγ, CD107a, and Granzyme B respectively. Thus, the heterotypic spheroids described in our study are a suitable in vitro model to both characterize the influence of CAF on therapeutic effects and the mechanisms that drives immune suppressive microenvironment. Moving forward, the next steps will consist to conduct in vivo experiments using a syngeneic PDAC orthotopic model to assess whether this combined therapeutic approach sensitize the PDAC microenvironment to anti-PD1 therapies.
 Citation Format: Thomas Bessede, Clara Freixinos, Jennifer Barrat, Véronique Garambois, Nadia Vie, Henri-Alexandre Michaud, Julien Faget, Nathalie Bonnefoy, Céline Gongora, Bruno Robert, Laurent Gros, Christel Larbouret. Sensitizing the PDAC tumor microenvironment to immune checkpoint therapies: characterization of a PDAC 3D model to decipher immune infiltration [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B045.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45910dcfd551f3dbd3e06cb65c7eddf239ebae12" target='_blank'>
              Abstract B045: Sensitizing the PDAC tumor microenvironment to immune checkpoint therapies: characterization of a PDAC 3D model to decipher immune infiltration
              </a>
            </td>
          <td>
            Thomas Bessede, Clara Freixinos, Jennifer Barrat, V. Garambois, N. Vié, H. Michaud, Julien Faget, Nathalie Bonnefoy, Céline Gongora, Bruno Robert, Laurent Gros, C. Larbouret
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Ethnic disparities in breast cancer (BC) outcomes have been reported, with lower survival rates among black women compared to white. Additionally, the unfavorable triple negative (TNBC) sub-type is more prevalent in black women and within this population, risk of mortality is again higher compared to white women. Biological contributors to this disparity are less clearly understood than socio-economic factors. Recent studies recognize different breast tumour immune microenvironments between black and white BC patients, necessitating further research into ethnicity-related differences in anti-tumor immune responses. Such responses begin in regional lymph nodes (LNs) with tumor neoantigen presentation. Here, we generate single-cell spatial maps of involved TNBC LN cores in treatment naïve black and white women. To guide design of a phenotyping panel for LNs, we profiled blood samples (n=95) using high-dimensional flow cytometry from newly diagnosed black and white BC patients. We found differential skewing of NK cell subsets, lower memory B cell (identified as IgM+CD27+IgD- & IgM+CD27+IgD+ cells) abundance and higher CD4+ T effector memory presence in black patients compared to white. Additionally, higher antigen presentation capacity was observed with increased intermediate monocytes and conventional DCs, confirmed by MS-based proteomics. Using highly multiplexed imaging with a panel of 26 markers, we performed single-cell spatial immune profiling to characterize cellular phenotypes, neighborhoods and interactions within a cohort of LN core biopsies from black patients and white patients. A total of 536,113 cells were segmented for analysis. Cell abundance analysis revealed higher CD8+ T effector memory and CD4+ T cell subsets in white patients, except for CD4+ T effector memory cells which were higher in black patients, consistent with blood profiling. Cellular neighborhood analysis revealed black patient tumor-rich regions were enriched for M2 macrophages while white patients had no such enrichment. In these white patients, a neighborhood resembling canonical follicular structures almost exclusively containing B cells was identified. Interestingly in these black patients, analogous B cell rich regions also comprised a heterogeneous array of other immune cells. Further analysis to explore B cell zones is ongoing. Preliminary interaction analysis identified immunosuppressive crosstalk in black patients with enriched Treg & CD8+ T cell and Treg & NK interactions, not observed in white patients. These initial findings suggest ethnicity-based variation in LN architectures from treatment naive patients with TNBC, supporting the hypothesis that variation in immune programming could translate into differences in pro and anti-tumor immunity, contributing to disparities in outcome amongst black and white patients.
 Citation Format: Paul R. Buckley, Melek Akay, Esme Carpenter, Rosalind Graham, Helen Kakkassery, Thanussuyah Alaguthurai, Priya Hossain, Patrycja Gazińska, Sheeba Irshad. Differential single-cell spatial immune cell profiles and immune cell interactions identified within lymph nodes metastases of black and white women diagnosed with triple negative breast cancers [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C142.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c431f2f91ac1c49d9f19dea4541265da8c85d8d6" target='_blank'>
              Abstract C142: Differential single-cell spatial immune cell profiles and immune cell interactions identified within lymph nodes metastases of black and white women diagnosed with triple negative breast cancers
              </a>
            </td>
          <td>
            Paul Buckley, Melek Akay, Esme Carpenter, Rosalind Graham, Helen Kakkassery, T. Alaguthurai, Priya Hossain, Patrycja Gazińska, S. Irshad
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Glioblastoma is the most common malignant primary brain tumor in adults with a poor survival and an unmet need for more effective therapies. Macroscopically, as observed through MR imaging and microscopy, these tumors are characterized by distinct tumor regions: a contrast-enhancing tumor core with a high cancer cell density and an infiltration zone that appears T2/FLAIR hyperintense without the uptake of contrast agent and consists of infiltrating tumor cells as well as non-malignant cells of the microenvironment. Previous genomic and transcriptomic studies revealed molecular differences between the tumor core and the infiltration zone. However, functional differences, such as drug sensitivity, between tumor regions are still largely unknown.



 To address this gap, we collected region-specific glioblastoma patient tissue samples from both the contrast-enhancing tumor core and the non-contrast-enhanced but 5-aminolevulinic acid (5-ALA) fluorescent infiltration zone directly from surgery. We utilized pharmacoscopy, a microscopy-based single-cell drug screening approach, to identify drug responses across regions and patients.



 By profiling a library of 60 drugs across regions in a cohort of 23 patients, we identified region-specific drug responses and developed a complementarity score to pinpoint drug combinations capable of targeting both the tumor core and the infiltration zone. These combination treatments exhibited highly promising anti-tumor activity across regions and patients.



 Given the high levels of functional spatial intra- and inter-tumor heterogeneity, relying on a single drug across patients may not be feasible for effective patient treatment. Thus, there is a strong rationale to administer drug combinations that not only consider regional heterogeneity but also account for patient-specific variations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fa1d38bc5a81ab129f0d7423cbbd80eb88f3cf3" target='_blank'>
              P01.13.B SPATIALLY RESOLVED FUNCTIONAL PROFILING FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Buck, S. Lee, R. Wegmann, J. Mena, F. Vasella, J. Friesen, L. Regli, E. Le Rhun, A. Zeitlberger, M. Neidert, N. Tatari, G. Hutter, M. Weller, B. Snijder, T. Weiss
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a937fbb57d8c42791d7a325a409f71c5d3e4aa5b" target='_blank'>
              Reconstruction of cancer marker analysis with holistic anatomical precision implicates heterogeneity development during breast tumor progression
              </a>
            </td>
          <td>
            Zhicheng Ge, Jing Wang, Libing He, Meng Zhao, Yang Si, Siyuan Chang, Guoyan Zhang, Shan Cheng, Wei Ding
          </td>
          <td>2024-10-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Even in its localized stages, pancreatic ductal adenocarcinoma (PDAC) is virtually incurable, characterized by early dissemination and disease recurrence. The liver is the most common site of metastasis, and in most cases hepatic recurrences drive survival outcomes. This underscores the critical need to target and prevent liver metastases. We leveraged a novel biobanking strategy to perform paired single-cell RNA (scRNA-seq) and TCR sequencing (scTCR-seq) to profile multisite immunity in patients with localized PDAC across the tumor, peripheral blood, and premetastatic liver. This analysis of three patients included three tumor, three blood, and nine liver (three per patient from separate segments) samples, and generated 85,748 cells for downstream analysis. We evaluated populations of tumor-expanded T cells as potential indicators of antitumoral immunity, and utilized validated computational approaches to identify tumor-reactive T cells based on key markers of dysfunction and exhaustion. Gene set enrichment analysis was performed on tumor infiltrating lymphocytes (TILs) using signatures specific to tumor-reactive CD4 and CD8 T cells (“NeoTCR4” and “NeoTCR8,” respectively), and scored these using the AUCell package in R. Overall, 153 CD4 and 22 CD8 T cells fit candidate NeoTCR4 and NeoTCR8 signature states, representing 5.6% and 0.8% of the total TIL population respectively. These nominated T cells expressed key marker genes associated with tumor specificity such as CXCL13, TIGIT, PDCD1, ENTPD1, and TOX. Thirteen of these predicted tumor-reactive T cell clones identified within the tumor were also detected within our liver samples, suggestive of a potential antitumoral immune response in the premetastatic liver space. We next performed 10x Xenium in situ spatial transcriptomic analysis of primary tumor, normal adjacent pancreas, and premetastatic liver from two of these patients, using tissue microarrays to incorporate replicate 2mm core biopsies from each of these sites. A custom designed gene panel, guided by our scRNA-seq dataset, targeted tumor-specific mutations (e.g. KRAS G12V, G12D, G12R, and Q61) and immune related genes with a specific focus on markers of T cell activity and exhaustion. Leveraging these latter marker genes, which represent a subset of the NeoTCR4 and NeoTCR8 signatures, we identified a population of exhausted TILs within the tumor microenvironment, representing potentially tumor-reactive T cells whose interactions can now be explored at spatial resolution. Future directions are aimed at experimentally validating the tumor reactivity of the predicted neoantigen-specific TILs within our dataset and characterizing the immune neighborhoods that exist in both the tumor and liver of PDAC patients. This work establishes exciting opportunities for novel T-cell therapy in the treatment of PDAC, and we have already implemented these experimental findings in a forthcoming clinical trial investigating the applicability of this predictive signature score for the generation of TCR-engineered lymphocyte therapy in patients with PDAC.
 Citation Format: Elishama N Kanu, Ashley A Fletcher, Sri Krishna, Frank J Lowery, Haotian Zhuang, Ethan S Agritelley, Jiayin Bao, Austin M Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, Michael E Lidsky, Sabino Zani, Dan G Blazer III, Peter J Allen, Nicholas D Klemen, Zhicheng Ji, Daniel P Nussbaum, Erika J Crosby. Single-cell and spatial analysis of the immune landscape unveils a subset of potentially tumor-reactive T cells in patients with localized PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb49745088796a4e1e8fb6d2e9a57e189de99728" target='_blank'>
              Abstract B052: Single-cell and spatial analysis of the immune landscape unveils a subset of potentially tumor-reactive T cells in patients with localized PDAC
              </a>
            </td>
          <td>
            Elishama N. Kanu, Ashley A. Fletcher, Sri Krishna, Frank J Lowery, Haotian Zhuang, Ethan S Agritelley, Jiayin Bao, A. Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, M. Lidsky, S. Zani, D. Blazer, Peter J. Allen, Nicholas D. Klemen, Zhicheng Ji, Daniel P. Nussbaum, Erika J. Crosby
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b28b9f10dc7dee52d960145a8bb408a48e4a08ef" target='_blank'>
              Multiparametric profiling of HER2-enriched extracellular vesicles in breast cancer using Single Extracellular VEsicle Nanoscopy
              </a>
            </td>
          <td>
            Nan Jiang, Andras Saftics, Eugenia Romano, I. Ghaeli, Cristal Resto, V. Robles, Saumya Das, K. Van Keuren-Jensen, Victoria L Seewaldt, T. Jovanović-Talisman
          </td>
          <td>2024-09-28</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Gliomas comprise a diverse spectrum of related tumor subtypes with varying biological and molecular features and clinical outcomes. Advances in detailed genetic and epigenetic characterizations along with an appreciation that subtypes associated with developmental origins, including brain location and patient age, have shifted glioma classification from the historical reliance on histopathological features to updated categories incorporating molecular signatures and spatiotemporal incidence. Within a subtype, individual gliomas show cellular heterogeneity, generally containing subpopulations resembling different types of normal glial and progenitor cells. In addition to tumor-autonomous mechanisms of aberrant growth regulation driven by genetic mutations and signaling between tumor cells, interactions with the tumor microenvironment, including neurons, astrocytes, oligodendrocyte precursor cells, and the immune microenvironment play important roles in driving glioma growth and influencing response to treatment. The emerging understanding of the complex contributions of normal brain to glioma growth represents new opportunities for therapeutic advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b4b46271e9cdc90000ffc4d7028f66e1ab4920c" target='_blank'>
              Glial Malignancies.
              </a>
            </td>
          <td>
            Suzanne J Baker, Hui Zong, Michelle Monje
          </td>
          <td>2024-09-16</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Understanding the mechanisms by which cancer cells switch between different adaptive states and evade therapeutic interventions is essential for clinical management. In this study, the in vivo cellular dynamics of a new chronic myeloid leukaemia cell line displaying altered phenotype and resistance to tyrosine kinase inhibitors were investigated in correlation with their parental cells for invasiveness/metastasis, angiogenic potential and population kinetics. We showed that the cells exhibiting drug resistance and plastic phenotype possess an increased capacity for invasion compared to their parental cells, that exposure to imatinib mesylate has the potential to enhance cellular motility and that in a leukaemic cell population, even a minority of plastic cells exhibit improved migratory ability. Furthermore, we show that these plastic cells have angiogenic and extravasation potential. The present study provides significant insights into the cellular dynamics displayed by a TKI‐resistant, phenotypically plastic CML cell line, using a zebrafish (Danio rerio) xenograft model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8debdaa7aac5be72ea83fbcd5d951e3b97abea7b" target='_blank'>
              Phenotypically plastic drug‐resistant chronic myeloid leukaemia cell line displays enhanced cellular dynamics in a zebrafish xenograft model
              </a>
            </td>
          <td>
            Seda Baykal, Zeynep Yuce, G. Ozhan
          </td>
          <td>2024-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Medulloblastoma (MB) is a pediatric brain tumor that develops in the cerebellum, representing one of the most common malignant brain cancers in children. Standard treatment includes surgery, chemotherapy, and radiation, but despite a 5-year survival rate of approximately 70%, these therapies often lead to significant neurological damage in the developing brain. This underscores the urgent need for less toxic, more effective therapeutic alternatives. Recent advancements in cancer immunotherapy, including immune checkpoint inhibitors and CAR-T cell therapy, have revolutionized cancer treatment. One promising avenue is the use of Gamma Delta (γδ)T cells, a unique T cell population with potential advantages such as non-alloreactivity, potent tumor cell lysis, and broad antigen recognition. However, their capacity to recognize and target MB cells remains underexplored. To investigate the therapeutic potential of γδT cells against MB, we analyzed the proportion and status of MB-infiltrated γδT cells within patient datasets. We next investigated the expression of γδT cell ligands on MB cells and identified EphA2 receptor and the phosphoantigen/Butyrophilin complex as key ligands, activating Vγ9Vδ1 and Vγ9Vδ2 T cells, respectively, leading to significant MB cell lysis in both monolayer and spheroid models. Importantly, preliminary safety data showed that γδT cells did not target differentiated neurons or neuroepithelial stem cells derived from induced pluripotent stem cells, underscoring the selectivity and safety of this approach. In conclusion, γδT cells trigger an efficient and specific killing of MB, and would offer a promising novel therapeutic strategy. Key messages Medulloblastoma patients often experience significant long-term side effects from current standard treatments. Immunotherapy has emerged as a promising alternative to conventional therapeutic approaches. In our study, we demonstrated that γδT cells can efficiently and specifically target medulloblastoma cells without causing harm to healthy neuronal tissue. These findings suggest that γδ T cell therapy may provide therapeutic benefits while potentially reducing treatment-related toxicity in medulloblastoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7246bc7c2d6cd1e3caa03b84ebe7960f735a32b" target='_blank'>
              EphA2 and Phosphoantigen-Mediated Selective Killing of Medulloblastoma by γδT Cells Preserves Neuronal and Stem Cell Integrity
              </a>
            </td>
          <td>
            Lola Boutin, Mingzhi Liu, Julie Déchanet Merville, Oscar Bedoya-Reina, M. Wilhelm
          </td>
          <td>2024-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b389c3620d8a50a335c990feb75f3a4d357ca8e6" target='_blank'>
              Integration of single-cell and spatial transcriptome sequencing identifies CDKN2A as a senescent biomarker in endothelial cells implicating hepatocellular carcinoma malignancy
              </a>
            </td>
          <td>
            Yue Ma, Chenhe Yi, Ning Cai, Jinhong Chen
          </td>
          <td>2024-11-06</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
The determinants of the response to checkpoint immunotherapy in hepatocellular carcinoma (HCC) remain poorly understood. The organisation of the immune response in the tumour microenvironment (TME) is expected to govern immunotherapy outcomes but spatial immunotypes remain poorly defined.


OBJECTIVE
We hypothesised that the deconvolution of spatial immune network architectures could identify clinically relevant immunotypes in HCC.


DESIGN
We conducted highly multiplexed imaging mass cytometry on HCC tissues from 101 patients. We performed in-depth spatial single-cell analysis in a discovery and validation cohort to deconvolute the determinants of the heterogeneity of HCC immune architecture and develop a spatial immune classification that was tested for the prediction of immune checkpoint inhibitor (ICI) therapy.


RESULTS
Bioinformatic analysis identified 23 major immune, stroma, parenchymal and tumour cell types in the HCC TME. Unsupervised neighbourhood detection based on the spatial interaction of immune cells identified three immune architectures with differing involvement of immune cells and immune checkpoints dominated by either CD8 T-cells, myeloid immune cells or B- and CD4 T-cells. We used these to define three major spatial HCC immunotypes that reflect a higher level of intratumour immune cell organisation: depleted, compartmentalised and enriched. Progression-free survival under ICI therapy differed significantly between the spatial immune types with improved survival of enriched patients. In patients with intratumour heterogeneity, the presence of one enriched area governed long-term survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/592e13ae9180b39b999935cfd28a102ececa8437" target='_blank'>
              Spatial single-cell profiling and neighbourhood analysis reveal the determinants of immune architecture connected to checkpoint inhibitor therapy outcome in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            H. Salié, Lara Wischer, A. D’Alessio, Ira Godbole, Yuan Suo, P. Otto-Mora, Juergen Beck, Olaf Neumann, A. Stenzinger, P. Schirmacher, C. Fulgenzi, Andreas Blaumeiser, Melanie Boerries, Natascha Roehlen, Michael Schultheiß, M. Hofmann, R. Thimme, D. Pinato, T. Longerich, B. Bengsch
          </td>
          <td>2024-09-30</td>
          <td>Gut</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/530d6f24e583e529ad460451ca4a394ebf382c85" target='_blank'>
              Mucosa-like differentiation of head and neck cancer cells is inducible and drives the epigenetic loss of cell malignancy
              </a>
            </td>
          <td>
            F. Oppel, S. Gendreizig, L. Martínez-Ruiz, J. Florido, Alba López-Rodríguez, H. Pabla, Lakshna Loganathan, Leonie Hose, P. Kühnel, P. Schmidt, M. Schürmann, J. M. Neumann, F. Viyof Ful, L. Scholtz, Dina Ligum, Frank Brasch, Karsten Niehaus, Germaine Escames, Tobias Busche, Jörn Kalinowski, Peter Goon, Holger Sudhoff
          </td>
          <td>2024-10-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e9aa014f46654cafa654936e981e4a60b2ea55" target='_blank'>
              DKK1+ tumor cells inhibited the infiltration of CCL19+ fibroblasts and plasma cells contributing to worse immunotherapy response in hepatocellular carcinoma
              </a>
            </td>
          <td>
            G. Fan, Ruyun Gao, Tongji Xie, Lin Li, Le Tang, Xiaohong Han, Yuankai Shi
          </td>
          <td>2024-11-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Modern cancer management comprises a variety of treatment strategies. Immunotherapy, while successful at treating many cancer subtypes, is often hindered by tumor immune evasion and T cell exhaustion as a result of an immunosuppressive tumor microenvironment (TME). In solid malignancies, the extracellular matrix (ECM) embedded within the TME plays a central role in T cell recognition and cancer growth by providing structural support and regulating cell behavior. Relative to healthy tissues, tumor associated ECM signatures include increased fiber density and alignment. These and other differentiating features contributed to variation in clinically observed tumor-specific ECM configurations, collectively referred to as Tumor-Associated Collagen Signatures (TACS) 1–3. TACS is associated with disease progression and immune evasion. This review explores our current understanding of how ECM geometry influences the behaviors of both immune cells and tumor cells, which in turn impacts treatment efficacy and cancer evolutionary progression. We discuss the effects of ECM remodeling on cancer cells and T cell behavior and review recent in silico models of cancer-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc9d5f20d0547b05a0b2945d91b671d9fb6945ef" target='_blank'>
              Understanding the interplay between extracellular matrix topology and tumor-immune interactions: Challenges and opportunities
              </a>
            </td>
          <td>
            Yijia Fan, Alvis Chiu, Feng Zhao, Jason T. George
          </td>
          <td>2024-11-07</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Testicular tumors are the most common malignancy of young men and tumors affecting the testis are caused by somatic mutations in germ or germ-like cells. The PI3K pathway is constitutively activated in about one third of testicular cancers. To investigate the role of the PI3K pathway in transforming stem-like cells in the testis, we investigated tumors derived from mice with post-natal, constitutive activation of PI3K signaling and homozygous deletion of tumor suppressor Pten, targeted to nestin expressing cells. Mice developed aggressive tumors, exhibiting heterogeneous histopathology and hemorrhaging. The tumors resemble the rare testis tumor type, testicular sex cord–stromal Leydig cell tumors. Single cell resolution spatial tissue analysis demonstrated that T-cells are the dominant tumor infiltrating immune cell type, with very few infiltrating macrophages observed in the tumor tissue, with CD8+ T-cells predominating. Further analysis showed that immune cells preferentially localize to or accumulate within stromal regions. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee742f58eb6122ac85c8bf499863f34f30c2b9b5" target='_blank'>
              Testicular sex cord–stromal tumors in mice with constitutive activation of PI3K and loss of Pten
              </a>
            </td>
          <td>
            Marija Dinevska, Lachlan Mcaloney, Samuel S. Widodo, G. Filiz, Jeremy Anderson, Sebastian Dworkin, Simon P. Windley, Dagmar Wilhelm, Theo Mantamadiotis
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Glioblastoma, a highly aggressive brain tumor, presents a daunting prognosis with an average survival of under 15 months due to its invasive nature. Understanding the specific behaviors of individual tumor cells in terms of volume, shape, and movement dynamics is crucial for improved patient stratification and understanding tumor mechanisms.



 we developed Single-Cell Behavior Live Imaging (ScBLI) analysis, a novel approach combining live imaging with advanced mathematical processing. Our method enables precise analysis of morphological and behavioral traits in patient-derived tumor cells cultured in vitro. Leveraging sophisticated cell tracking methods, we analyzed 21 glioblastoma and 1 healthy astrocyte (HA) cell cultures, to capture detailed movement information. Our analyses extend beyond mere tracking; we extrapolated an overarching mobility strategy employed by the cells within each culture. By measuring the delta scaling parameter, we discern between subdiffusive motion, random Brownian motion, and more organized efficient strategies such as a typical Levy Walk. We correlate variations in delta scaling with cell applied stimuli, such as invasion and antineoplastic agents, enhancing our understanding of tumor cell behavior.



 We analyzed 100 cell trajectories per culture, totaling 2200 trajectories. Parameters as distance traveled, volume, track displacement, directional change rate, circularity, mean velocity, and confinement ratio were used to identify 3 distinct cell populations with different distributions within each GBM and HA culture facilitating the stratification of samples into 3 clusters. Moreover, the delta scaling was divided into three groups (0.28 to 0.80), and its integration with clinical data revealed an increased percentage of patients with shorter overall survival as scaling increased. Higher delta scaling indicated a more organized system adapted to challenging environments. Transcriptome analysis unveiled gene expression differences, highlighting potential therapeutic targets.



 This approach provides a deeper understanding of how tumor cells navigate and occupy the available space, shedding light on their dynamic behavior in vitro. Our results indicate that cancer cells must preserve the control of their locomotion to explore space in an organized and efficient way for survival, similar to the organized and functional behavior of healthy cells. By integrating the unique characteristics of each patient’s tumor cell behavior with clinical data, this approach offers a new method for stratifying patients with different survival probabilities. Overall, by combining imaging, image processing algorithms, and dynamic modeling, we provide a valuable framework for understanding the complex dynamics of glioblastoma and paving the way for more personalized and effective treatment approaches.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6a935ed764568a05683204598acba4c1c6581d6" target='_blank'>
              OS07.8.A INNOVATIVE DYNAMIC STRATIFICATION IN GLIOBLASTOMA: ANALYZING TUMOR CELL CHARACTERISTICS AND MOBILITY STRATEGIES THROUGH SINGLE CELL BEHAVIOR LIVE IMAGING (SCBLI)
              </a>
            </td>
          <td>
            G. Ferri, M. Morelli, A. Pastore, F. Lessi, F. Di Lorenzo, S. Franceschi, F. Pieri, C. Gambacciani, Y. Shah, L. Tonello, O. Santonocito, A. D. Di Stefano, L. Palatella, P. Grigolini, C. Mazzanti
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background In the Tumor Microenvironment (TME), hypoxia stands as a significant factor that modulates immune responses, especially those driven by T cells. As T cell-based therapies often fail to work in solid tumors, this study aims to investigate the effects of hypoxia on T cell topo-distribution in the TME, gene expression association with T cell states, and clinical responses in melanoma. Methods To generate detailed information on tumor oxygenation and T cell accessibility, we utilized mathematical modeling of human melanoma tissue microarrays (TMAs) that incorporate oxygen supply from vessels, intratumoral diffusion, and cellular uptake. We created tumor maps and derived plots showing the fraction of CD4 and CD8 T cells against the distance to the nearest vessel and oxygen pressure. To assess their function and transcriptional changes caused by hypoxia, effector T cells were generated and cultured under hypoxia (0.5% oxygen) or normoxia (21% oxygen). The T cell hypoxia-transcriptional signature was compared against datasets from msigDB, iATLAS (clinical trials of melanoma patients treated with immune checkpoint inhibitors), ORIEN AVATAR (real-world melanoma patients treated with immune checkpoint inhibitors, ICIs), and a single-cell atlas of tumor-infiltrating lymphocytes (TILs). Results We made three specific observations: 1) in melanoma T cells preferentially accumulated in oxygenated areas close to blood vessels (50-100 micrometers from the vasculature in the regions of high oxygen availability) but not in hypoxic areas far from blood vessels. 2) Our analysis confirmed that under hypoxia, T cell functions were significantly reduced compared to normoxic conditions and accompanied by a unique gene signature. Furthermore, this hypoxic gene signature was prevalent in resting and non-activated T cells. Notably and clinically relevant, the hypoxic T cell gene set was found to correlate with reduced Overall Survival (OS) and reduced progression-free survival (PFS) in melanoma patients, which was more pronounced in non-responder patients undergoing ICI therapy. 3) Finally, compared with a single-cell atlas of tumor-infiltrating T cells, our hypoxia signature aligned with a population of cells at a state termed stress response state (TSTR). Conclusion Our study highlights the critical role of hypoxia in shaping T cell distribution and its correlation with clinical outcomes in melanoma. We revealed a preferential accumulation of T cells in oxygenated areas. Moreover, hypoxic T cells develop a distinct hypoxic gene signature prevalent in resting, non-activated T cells and TSTR that was also associated with poorer outcomes, particularly pronounced among non-responders to ICIs. Key Messages What is already known on this topic: Hypoxia significantly impairs T cell functions, including reduced proliferation and cytokine production. This impairment may contribute to immune evasion and resistance to immune therapies, such as ICIs, adoptive transfer of TILs, and Chimeric Antigen Receptor (CAR) T cells. Despite the established impact of hypoxia on T cell function, the precise spatial distribution of T cells in relation to oxygen availability within the TME and how this affects clinical outcomes in melanoma patients remains unclear. Additionally, the specific transcriptional changes in T cells induced by hypoxia and their prevalence in different T cell states, as well as the implications for resistance to ICIs, have not been thoroughly investigated. Understanding these aspects is crucial for developing targeted therapies to overcome hypoxia-induced resistance and improve immunotherapy efficacy. What this study adds: This study elucidates the profound impact of hypoxia on T cells in melanoma. Our findings reveal that T cells accumulate in well-oxygenated regions near blood vessels, whereas hypoxic conditions significantly impair their proliferation and cytokine production. By addressing the transcriptional changes induced by hypoxia, we demonstrated its prevalence in resting and non-activated T cells and TSTR. Moreover, we found that hypoxia is associated with shorter OS and PFS in melanoma patients, particularly in non-responders to ICIs. This research highlights the critical role of hypoxia in modulating T cell spatial behavior, contributing to immune evasion and poor clinical outcomes in melanoma. How this study might affect research, practice, or policy: The findings emphasizes hypoxia’s critical role in modulating T cell behavior and its potential as a biomarker for treatment outcomes. These insights could inform the development of therapeutic strategies aimed at improving T cell function in hypoxic TMEs, thereby enhancing the efficacy of immunotherapies for melanoma and other solid tumors. Additionally, the study’s results may influence policies regarding the evaluation and implementation of combination therapies that target hypoxia to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/070aba94c8974ffb34dd93c7c046374606f86370" target='_blank'>
              Effector T Cells under Hypoxia have an Altered Transcriptome Similar to Tumor-Stressed T Cells Found in Non-Responsive Melanoma Patients
              </a>
            </td>
          <td>
            Mate Z Nagy, Lourdes B. Plaza Rojas, Justin C Boucher, Elena Kostenko, Anna Austin, Ahmad Tarhini, Zhihua Chen, Dongliang Du, Awino M. E. Ojwang', Joshua Davis, Katarzyna A. Rejniak, Timothy I. Shaw, Jose Alejandro Guevara
          </td>
          <td>2024-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Objective This study aimed to explore the heterogeneity of tumor endothelial cells (TECs) in hepatocellular carcinoma (HCC) and their role in tumor progression, with the goal of identifying new therapeutic targets and strategies to improve patient prognosis. Methods Single-cell RNA sequencing data from nine primary liver cancer samples were analyzed, obtained from the Gene Expression Omnibus (GEO) database. Data preprocessing, normalization, dimensionality reduction, and batch effect correction were performed based on the Seurat package. HCC cell types were identified using uniform manifold approximation and projection (UMAP) and cluster analysis, and the different cell types were annotated using the CellMarker database. Pseudotime trajectory analysis was conducted with Monocle to explore the differentiation trajectory of TECs. MAPK signaling pathway activity and copy number variations (CNV) in TECs were analyzed in conjunction with data from The Cancer Genome Atlas (TCGA), the trans-well and wound healing assay was used for cell invasion and migration activity assessment. Results Two subgroups of TECs (TECs 1 and TECs 2) were identified, exhibiting distinct functional activities and signaling pathways. Specifically, TECs 1 may be involved in tumor cell proliferation and inflammatory responses, whereas TECs 2 is not only involved in cell proliferation pathways, but also enriched in pathways such as metabolic synthesis. Pseudotime analysis revealed dynamic changes in TECs subgroups during HCC progression, correlating specific gene expressions (such as PDGFRB, PGF, JUN, and NR4A1). Subsequently, the JUN gene was predicted by performing binding sites and was shown to act as a transcription factor that may regulate the expression of the PGF gene. CNV analysis highlighted key genes and pathways in TECs that might influence HCC progression, and the PGF as key regulatory factor mediated cell proliferation and migration. Conclusion The study revealed the heterogeneity of TECs in HCC and their potential roles in tumor progression, offering new perspectives and potential therapeutic targets for HCC molecular mechanisms. The findings emphasize the importance of further exploring TECs heterogeneity for understanding HCC pathogenesis and developing personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db89c0c7a388ba83d2034d24b60625ad1b46b81d" target='_blank'>
              Exploring tumor endothelial cells heterogeneity in hepatocellular carcinoma: insights from single-cell sequencing and pseudotime analysis
              </a>
            </td>
          <td>
            Jiachun Sun, Shujun Zhang, Yafeng Liu, Kaijie Liu, Xinyu Gu
          </td>
          <td>2024-10-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic breast cancer remains largely incurable, and the mechanisms underlying the transition from primary to metastatic breast cancer remain elusive. We analyzed the complex landscape of primary and metastatic breast cancer using scRNA-seq data from twenty-three female patients with either primary or metastatic disease to elucidate the genetic and molecular mechanisms underlying changes in the metastatic tumor ecosystem. We identify specific subtypes of stromal and immune cells critical to forming a pro-tumor microenvironment in metastatic lesions, including CCL2+ macrophages, cytotoxic T cells with an exhausted gene signature, and FOXP3+ regulatory T cells. Analysis of cell-cell communication highlights a marked decrease in tumor-immune cell interactions in metastatic tissues, likely strengthening the immunosuppressive microenvironment. In contrast, primary breast cancer samples displayed increased activation of the TNF- signaling pathway via NF-kB, indicating a potential therapeutic target. Our study comprehensively characterizes the transcriptional landscape encompassing primary and metastatic breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f83bcd576506ee2a8774cb0cfd1433b6936e5b1" target='_blank'>
              Single-cell RNA sequencing reveals immunosuppressive pathways associated with metastatic breast cancer
              </a>
            </td>
          <td>
            F. Ozmen, T. Y. Ozmen, A. Ors, M. Janghorban, M. J. Rames, X. Li, F. Behbod, G. B. Mills, H. Mohammed
          </td>
          <td>2024-09-26</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Non-melanoma skin cancer (NMSC) mainly includes basal cell carcinoma, cutaneous squamous cell carcinoma, and Merkel cell carcinoma, showing a low mortality rate but the highest incidence worldwide. In recent decades, research has focused on understanding the pathogenesis and clinical treatments of NMSC, leading to significant advances in our knowledge of these diseases and the development of novel therapies, including immunotherapy. Nevertheless, the low to moderate objective response rate, high recurrence, and therapeutic resistance remain persistent challenges, which are partly attributable to the intratumoral heterogeneity. This heterogeneity indicates that tumor cells, immune cells, and stromal cells in the tumor microenvironment can be reshaped to a series of phenotypic and transcriptional cell states that vary in invasiveness and treatment responsiveness. The advent of single-cell RNA sequencing (scRNA-seq) has enabled the comprehensive profiling of gene expression heterogeneity at the single-cell level, which has been applied to NMSC to quantify cell compositions, define states, understand tumor evolution, and discern drug resistance. In this review, we highlight the key findings, with a focus on intratumoral heterogeneity and the mechanism of drug resistance in NMSC, as revealed by scRNA-seq. Furthermore, we propose potential avenues for future research in NMSC using scRNA-seq.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a5efda06af7a4c8d9343615ef50b870bd5ba887" target='_blank'>
              Unraveling the landscape of non-melanoma skin cancer through single-cell RNA sequencing technology
              </a>
            </td>
          <td>
            G. Yan, Xiuli Wang, Guolong Zhang
          </td>
          <td>2024-11-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Asymmetric partition of fate determinants during cell division is a hallmark of cell differentiation. Recent works suggested that such a mechanism is hijacked by cancer cells to increase both their phenotypic heterogeneity and plasticity and in turn their fitness. To quantify fluctuations in the partitioning of cellular elements, imaging-based approaches are used, whose accuracy is limited by the difficulty of detecting cell divisions. Our work addresses this gap proposing a general method based on high-throughput flow cytometry measurements coupled with a theoretical frameworks. We applied our method to a panel of both normal and cancerous human colon cells, showing that different kinds of colon adenocarcinoma cells display very distinct extents of fluctuations in their cytoplasm partition, explained by an asymmetric division of their size. To test the accuracy of our population-level protocol, we directly measure the inherited fractions of cellular elements from extensive time-lapses of live-cell laser scanning microscopy, finding excellent agreement across the cell types. Ultimately, our flow cytometry-based method promise to be accurate and easily applicable to a wide range of biological systems where the quantification of partition fluctuations would help accounting for the observed phenotypic heterogeneity and plasticity">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89534ca95bbe7d2c00c135f356aa0991c7b534e4" target='_blank'>
              Robust assessment of asymmetric division in colon cancer cells
              </a>
            </td>
          <td>
            Domenico Caudo, Chiara Giannattasio, Simone Scalise, V. Turris, Fabio Giavazzi, G. Ruocco, Giorgio Gosti, Giovanna Peruzzi, Mattia Miotto
          </td>
          <td>2024-09-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Tumours are complex ecosystems comprised of tumour cells and various normal cell types. The interactions among these cells with diverse molecular characteristics lead to intratumoral heterogeneity and cancer evolution. However, little is known about the spatial organisation of the glioma tumour ecosystem, the cellular interactions therein, and their influence on patient outcomes. Spatially resolved transcriptomics, which measures the activity of hundreds of genes in situ, offers new avenues to address these long-standing questions.



 We assembled a clinically annotated cohort of glioma patients spanning 278 specimens from low- and high-grade IDH-mutant astrocytoma, IDH-mutant and 1p/19q co-deleted oligodendroglioma, ependymoma, and glioblastoma. Using tissue microarrays from archival FFPE, we performed spatial transcriptomics profiling with the 10x Xenium platform. A custom panel of 344 genes, selected based on publicly available single-cell RNA sequencing datasets of gliomas, was interrogated.



 The spatial transcriptomics data enabled annotation and spatial mapping of more than 20 tumour microenvironment (TME) cell types and various tumour cell types present across the 5 glioma entities. We observed broadly conserved transcriptional profiles of TME cell types across gliomas, but TME cell type composition and spatial organisation varied depending on tumour entity. In particular, lower amounts of macrophages and microglia as well as oligodendrocytes and oligodendrocyte progenitor cells were observed in oligodendroglioma tumours compared to other entities. High-grade astrocytoma samples contained more myeloid dendritic cells and monocytes in comparison to low-grade astrocytomas. A lower proportion of vascular cells correlated with better overall survival for glioblastomas and low- and high-grade astrocytomas. Tumour cells exhibited mostly entity-specific gene expression, but also distinct tumour cell states including AC-, OPC-, NPC-, and MES-like cell states.



 Our work demonstrates the utility of the 10x Xenium platform for high-resolution spatial profiling of expression in archival FFPE samples. Preliminary mapping of the TME and tumour cell types unravelled a complex spatial architecture of the glioma ecosystems. Distinct compositions of the TME were found in each glioma subtype, which may help understand different mechanisms of malignancy. Associations between cellular spatial patterns and clinical data available for FFPE samples have the potential to provide better insights for improved patient stratification and treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5877e1818dcb8b95627446b49e8f170fc56f766" target='_blank'>
              P01.03.A DECIPHERING THE TUMOUR ECOSYSTEM IN GLIOMAS USING SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            Z. Seferbekova, D. Calafato, G. Rukhovich, A. Patel, M. Ritter, F. Ippen, A. Suwala, F. Sahm, M. Gerstung
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Isocitrate dehydrogenase (IDH) mutant gliomas are progressive, invariably fatal brain tumors affecting young and middle-age adult patients. Previous single-cell transcriptomic characterization of IDH-mutant gliomas has uncovered a heterogeneity of cell states, with a developmental hierarchy that starts with neural progenitor-like stem cells that transition to more differentiated malignant cell states. Despite this understanding, the dynamics of these cell state transitions as tumors undergo grade progression, and the factors that facilitate or restrict cell state transitions remain poorly understood.



 Here, we characterize the cell states of IDH-mutant gliomas via single-cell gene expression (scRNA-seq) and single-cell analyses of accessible chromatin (scATAC-seq) in tumors spanning all grades using fresh patient samples and compare them with transcriptional programs and chromatin accessibility profiles of normal brain cells. We perform mechanism validation experiments using IDH-mutant glioma organoids.



 We find that tumor transcriptional cell states experience a marked shift as tumors undergo progression. In low-grade tumors, predominant cell states mimic slow-cycling oligodendrocyte progenitor-like cells (OPCs). With grade progression, predominant cell states transition to resemble highly proliferative neural progenitor-like cells (NPCs). Additionally, scATAC-seq analyses revealed novel permissive chromatin states, which could facilitate this transition. Genetic analyses identified several molecular alterations in high-grade tumors, including loss of DNA methylation, oncogene amplification, and the deletion of genes associated with the interferon (IFN) response. Those IFN genes, as well as tumor suppressors, are initially repressed by DNA hypermethylation in low-grade tumors. Lastly, we validate the role of DNA hypomethylation in favoring immune evasion with grade progression using an IDH-mutant glioma organoid model.



 Overall, our findings provide evidence that tumor progression in IDH-mutant gliomas is driven by an interplay of epigenetic and genetic mechanisms, with the former being determinant in low-grade tumors and the latter in high-grade tumors. The emergence of genetically-driven mechanisms promotes more mitotically active cell states as well as decreased anti-tumoral immune activation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13fd62dc7724c0acd329992464a530f1dbcd46aa" target='_blank'>
              KS02.6.A SINGLE-CELL MULTIMODAL ANALYSIS OF IDH-MUTANT GLIOMAS UNCOVERS SHIFT FROM EPIGENETIC TO GENETIC DRIVERS WITH TUMOR PROGRESSION
              </a>
            </td>
          <td>
            L. G. Gonzalez Castro, J. Wu, S. Battaglia, C. El Farran, J. D’Antonio, T. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 Hotspot mutations in the IDH genes serve as tumor-initiating events that define a subset of adult diffuse glioma that can further be divided into either oligodendroglioma or astrocytoma subtypes. Despite standard treatment, IDH-mutant gliomas inevitably recur. Previous studies suggest that therapeutic resistance may result from a combination of intratumoral cellular heterogeneity, epigenetic evolution, and acquired genetic alterations. However, how these multilayered molecular features interact to influence the evolutionary paths of IDH-mutant gliomas remains incompletely understood.



 To chart this evolution, 75 glioma samples were longitudinally collected from 35 patients (n = 13 oligodendroglioma, n = 22 astrocytoma) and profiled using single nucleus RNA sequencing, ATAC sequencing, and bulk DNA sequencing.



 Analysis of 331,016 nuclei (snRNA) revealed eight tumor microenvironment cell types and five pan-IDH-mutant malignant cellular states. The malignant states were distributed along a cellular hierarchy of stem-like populations (neural progenitor cell-like, oligodendrocyte progenitor cell-like, and undifferentiated) and more differentiated populations (mesenchymal-like and astrocyte-like) with cycling cells enriched in the stem-like populations. Joint open chromatin accessibility data (snATAC) was available for a subset of malignant cells (71,088 nuclei) and state-specific differentially accessible peaks supported that these states are epigenetically encoded. Across both subtypes, higher tumor grade was associated with reduced malignant cell differentiation and increased cycling populations. Between the two time points, we identified that there was a longitudinal increase in the cycling, undifferentiated, and mesenchymal-like populations with a corresponding decrease in the astrocyte-like population. Longitudinal genetic analysis revealed that 19 of 35 tumors acquired at least one of the following genetic alterations: treatment-associated hypermutation, cell cycle alteration, or large changes in copy number alterations. Importantly, tumors that acquired these key genetic alterations demonstrated significant shifts towards reduced differentiation and increased cycling populations while those tumors that did not had more stable malignant profiles.



 Collectively, our results suggest a common cellular hierarchy across IDH-mutant gliomas with a shift towards reduced differentiation, increased cycling populations during disease progression that is driven by acquired genetic alterations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d952023b3d2c6da79f53a908951847332e2e66b4" target='_blank'>
              KS02.7.A LONGITUDINAL SINGLE-CELL ANALYSES IDENTIFY DRIVERS OF GENETIC, EPIGENOMIC, AND CELLULAR EVOLUTION IN IDH-MUTANT GLIOMA
              </a>
            </td>
          <td>
            K. Johnson, A. Spitzer, F. Varn, M. Nomura, L. Garofano, T. Chowdhury, K. J. Anderson, F. D’Angelo, L. Bussema, S. Gritsch, Y. Oh, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, A. Lipsa, F. Hertel, Z. Zhao, Q. Wang, T. Jiang, B. Hermes, N. Sanai, A. Gołȩbiewska, S. Niclou, J. Huse, W. Yung, A. Lasorella, M. Suvà, A. Iavarone, I. Tirosh, R. Verhaak
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) often exhibit stem-like attributes that depend on an intricate stemness-promoting cellular ecosystem within their niche. The interplay between CSCs and their niche has been implicated in tumor heterogeneity and therapeutic resistance. Normal stem cells (NSCs) and CSCs share stemness features and common microenvironmental components, displaying significant phenotypic and functional plasticity. Investigating these properties across diverse organs during normal development and tumorigenesis is of paramount research interest and translational potential. Advancements in next-generation sequencing (NGS), single-cell transcriptomics, and spatial transcriptomics have ushered in a new era in cancer research, providing high-resolution and comprehensive molecular maps of diseased tissues. Various spatial technologies, with their unique ability to measure the location and molecular profile of a cell within tissue, have enabled studies on intratumoral architecture and cellular cross-talk within the specific niches. Moreover, delineation of spatial patterns for niche-specific properties such as hypoxia, glucose deprivation, and other microenvironmental remodeling are revealed through multilevel spatial sequencing. This tremendous progress in technology has also been paired with the advent of computational tools to mitigate technology-specific bottlenecks. Here we discuss how different spatial technologies are used to identify NSCs and CSCs, as well as their associated niches. Additionally, by exploring related public data sets, we review the current challenges in characterizing such niches, which are often hindered by technological limitations, and the computational solutions used to address them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/548e0b0064280f307e085a25804779bf2b1c655a" target='_blank'>
              Deciphering normal and cancer stem cell niches by spatial transcriptomics: opportunities and challenges.
              </a>
            </td>
          <td>
            Hirak Sarkar, Eunmi Lee, Sereno Lopez-Darwin, Yibin Kang
          </td>
          <td>2024-11-04</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6866382555382de9b8b8e3c83922b3e82d715085" target='_blank'>
              Defining heritability, plasticity, and transition dynamics of cellular phenotypes in somatic evolution.
              </a>
            </td>
          <td>
            Joshua S. Schiffman, Andrew R D'Avino, Tamara Prieto, Yakun Pang, Yilin Fan, Srinivas Rajagopalan, Catherine Potenski, Toshiro Hara, M. Suvà, Charles Gawad, Dan A Landau
          </td>
          <td>2024-09-24</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 The structured layers of the human neocortex stand in sharp contrast to the disordered arrangement found in malignant CNS tumors. Glioblastoma (GBM), with its diverse characteristics, presents formidable hurdles in both research and treatment, as successful approaches demand a uniform biological understanding. In this study, we capitalize on recent advances in spatial biology, employing integrated spatially resolved transcriptomics and metabolomics to identify the factors driving the unique architectural changes observed in GBM.



 We performed spatial metabolomics (MALDI) and transcriptomics (Visium n=43 and MERFISH n=25) of glioblastoma patients. Computational analysis was performed with the SPATA2 R package.



 We found that most spatially recurrent pattern in glioblastoma emerge in a gradient-like organization,particularly influenced by proximity to areas of necrosis and hypoxia. Samples without hypoxic regions demonstrated less spatial organization. Metabolomic and transcriptomic profiling revealed enhanced responses to hypoxia coupled with reduced cell proliferation and elevated expression indicative of the S-phase in cell division in regions near necrosis. The vicinity to hypoxic areas appeared to be associated with increased chromosomal alterations as well as extracellular matrix remodeling and epithelial-to-mesenchymal transition, implying that the invasive capabilities of GBM cells originate from these hypoxic niches. By further integrating spatially resolved TCR-sequencing, we identified that hypoxic gradients drive defined T cell response with higher likelihood of T cell dysfunction within a distance of 235 μm (sd: 132 μm) to hypoxia. This suggests that the presence of necrosis, hypoxia, and hypoxia associated metabolites could serve as potential predictive markers for T-cell behavior and exhaustion in glioblastoma.



 Our investigation provides a comprehensive examination of the impact of two common environments in glioblastoma: necrosis and hypoxia. We demonstrate that the intricate structure of glioblastoma is not merely shaped by genetic chance and random events. Rather, it is substantially molded by spatial proximity to critical elements such as necrosis and hypoxia. This understanding contributes to an advancing framework aimed at unraveling the complex biology of glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e118b0e7f81b796b7529c20a3b8d7d117791b7f5" target='_blank'>
              OS07.3.A DYNAMIC METABOLIC SHIFTS RESHAPE GLIOBLASTOMA: IMPACTS ON SPATIAL STRUCTURE AND IMMUNE RESPONSE
              </a>
            </td>
          <td>
            N. Conrad, J. Benotmane, G. Villa, O. Schnell, D. Heiland, J. Kueckelhaus
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Esophageal Squamous Cell Carcinoma (ESCC) poses a significant health challenge in China due to its high incidence and mortality. Single-cell transcriptomics has transformed the approach to cancer research, allowing detailed analysis of cellular and molecular heterogeneity. The interaction between the transcription factor STAT1 and the tumor suppressor LHPP is crucial, potentially influencing cancer progression and therapeutic outcomes. Objective This study aims to explore the molecular mechanisms and cellular dynamics underlying ESCC, with a focus on the role of transcription factors, particularly STAT1, and its regulatory relationship with LHPP, in the pathogenesis of ESCC. Methods Utilizing single-cell transcriptomics data sourced from the public database GEO (GSE160269), we identified major cell types and their transcriptomic changes in ESCC patients. Differential gene expression profiles were examined to understand the dynamics of the tumor microenvironment (TME). A cohort of 21 ESCC patients was recruited to validate the findings. Furthermore, in ESCC cell lines, we validated the transcriptional regulatory relationship between STAT1 and LHPP. Results Our analysis identified six major cell types within the ESCC microenvironment, revealing significant changes in cellular composition and gene expression profiles associated with tumorigenesis. Notably, a novel association between STAT1's regulatory role over LHPP was observed, suggesting a complex regulatory loop in ESCC pathogenesis. Elevated STAT1 and reduced LHPP expression were confirmed in patient samples with STAT1 negatively regulates LHPP expression at the promoter level; when the promoter binding motif regions were mutated, the transcriptional repression ability on LHPP was weakened. Conclusion The study highlights STAT1 as a core regulator in ESCC, directly influencing LHPP expression. The findings offer novel insights into the molecular mechanisms driving ESCC, shedding light on the cellular alterations and gene regulation dynamics within the ESCC microenvironment. This research provides a foundation for developing targeted therapeutic strategies, potentially utilizing the STAT1-LHPP axis as a focal point in ESCC treatment and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c86f49b4acf6a7d2f9a182e7693ea26179b8f22" target='_blank'>
              Single-Cell Transcriptomics Reveals STAT1 Drives LHPP Downregulation in Esophageal Squamous Cell Carcinoma Progression
              </a>
            </td>
          <td>
            Xia Chen, Zhiyun Xu, Wubi Zhou, Lili Zhang, Jian Huang, D. Bao
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is the third leading cause of cancer-related death in the US. This poor prognosis is related to the complex tumor microenvironment and tumor heterogeneity that drives resistance to chemotherapy. Here the heterogeneity of the stromal, immune, and malignant epithelial cell populations enables signaling and metabolic crosstalk mechanisms that drive immune suppression and resistance to therapy. Recently, our group unveiled that multiple distinct metabolic subclasses of cancer cells coexist within individual PDAC tumors. Among these, two PDAC clonal populations can engage in metabolic symbiosis that supports mitochondrial metabolism. One subtype has a constitutively high integrated stress response (ISR), enabling resistance to mitochondrial inhibitors. Further, these cells release asparagine that supports the mitochondrial metabolism of the other subtype that are more sensitive to mitochondrial inhibition. Given these observations, we are developing approaches to disrupt this metabolic symbiosis. Clonal PDAC cells with constitutively active ISR exhibit increased sensitivity to inhibition of the redox activity of Redox factor 1/Apurinic-apyrimidinic endonuclease 1 (Ref-1). Importantly, this can be leveraged to disrupt the support of the mitochondrial metabolism of clonal symbiotic pairs. Moreover, polyclonal PDAC cell lines representative of the heterogeneity seen in PDAC patients show a highly synergistic response to the combination of Ref-1 redox inhibitors with mitochondrial inhibitors. These data suggest this treatment combination will be synergistic in polyclonal tumors that exist in most PDAC patients. Overall, these data have the potential to lead to new avenues to directly target heterogenous populations of cancer cells to treat pancreatic cancer- an urgent clinical need.
 Citation Format: Taryn Morningstar, Cecily Anaraki, Lorenzo Scipioni, Giulia Tedeschi, Michelle Digman, Aimee Edinger, Claus Jorgensen, Melissa Fishel, Christopher J Halbrook. Targeting metabolic heterogeneity in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C045.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0412df26641d71ff1a5183fc1b222c74aa39cae6" target='_blank'>
              Abstract C045: Targeting metabolic heterogeneity in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Taryn Morningstar, Cecily Anaraki, Lorenzo Scipioni, Giulia Tedeschi, M. Digman, Aimee Edinger, Claus Jorgensen, M. Fishel, Christopher J. Halbrook
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Purpose: Optimization of pancreatic ductal adenocarcinoma (PDAC) patient survival will depend on advancing our understanding of the mechanisms governing tumor cell plasticity and effectively mitigating their invasive and metastatic capabilities. This study aims to investigate the correlation of cancer biomarkers crucial to these processes by integrating molecular signatures with histopathological characteristics using single-cell RNA sequencing (scRNA-seq) and multiplexed IHC/IF. Methods: Single-cell RNA sequencing (scRNA-seq) data from a comprehensive dataset of over 53,000 PDAC cells and validation of lead targets through 14-plex immunofluorescence, using a customized MICS platform (Miltenyi Biotec) and dual color IHC were employed to assess protein expression of defining biomarkers of basal and classical subtypes of PDAC including Keratin 17 (K17) and GATA6. Results: scRNAseq analysis revealed distinct patterns of K17 and GATA6 expression, highlighting intratumoral heterogeneity among PDAC tumors. Specifically, populations of cells expressing K17+/GATA6+ and K17+/GATA6- exhibited significant enrichment in pathways associated with invasion and cell migration, including PI3K/Akt/mTOR, TGF-β, and epithelial-mesenchymal transition (EMT) signaling. Conversely, GATA6-/K17- cells demonstrated enrichment in biosynthetic and metabolic pathways that govern cell cycle regulation. Additionally, mIF data assessment revealed correlations between histomorphological patterns and dynamic shifts in K17 and GATA6 expression within PDAC tumors. Diffusely infiltrative tumor cells (DITCs) uniformly expressed K17, contrasting with the gland-forming components (GFCs) of PDAC, which showed enrichment in GATA6 and sporadic expression of K17. E-cadherin, a key adhesion marker, exhibited the highest expression levels in the glandular and solid components of PDAC, underscoring their epithelial characteristics. Conversely, vimentin, a marker typically associated with epithelial-to-mesenchymal transition (EMT), was significantly expressed in DITCs, suggesting a mesenchymal-like phenotype in these infiltrative cells. Dual color IHC further validated these findings, reinforcing the correlation between morphologic plasticity and the expression patterns of K17 and GATA6, emphasizing their potential to distinguish distinct tumor cell populations within PDAC. Conclusions: These insights suggest that the interaction between these markers may indicate a distinct biological state characterized by altered cellular metabolism and potentially different therapeutic vulnerabilities. The robust association between K17 expression and diffusely infiltrative tumor cells underscores its potential as a marker for the most aggressive forms of this disease. Our results highlight the heterogeneous nature of PDAC and emphasize the critical importance of understanding these molecular distinctions in histopathological contexts to develop targeted therapies aimed at improving patient survival.
 Citation Format: Lyanne A Delgado- Coka, Brian Nelson, Michael Horowitz, Shayan Sarkar, Natalia Marchenko, Scott Powers, Luisa F Escobar-Hoyos, Kenneth Shroyer. Keratin 17 and GATA6 correlate with diffusely infiltrative versus gland-forming components of PDAC: Uncovering the transitional state in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C070.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10730db85bb1ae24ebbf29f9506b2592a344ce53" target='_blank'>
              Abstract C070: Keratin 17 and GATA6 correlate with diffusely infiltrative versus gland-forming components of PDAC: Uncovering the transitional state in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A Delgado- Coka, Brian J. Nelson, Michael Horowitz, Shayan Sarkar, N.Ye Marchenko, Scott Powers, Luisa F. Escobar‐Hoyos, Kenneth R. Shroyer
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract The heterogeneity of tumor clones drives the selection and evolution of distinct tumor cell populations, resulting in an intricate and dynamic tumor evolution process. While tumor bulk DNA sequencing helps elucidate intratumor heterogeneity, challenges such as the misidentification of mutation multiplicity due to copy number variations and uncertainties in the reconstruction process hinder the accurate inference of tumor evolution. In this study, we introduce a novel approach, REconstructing Tumor Clonal Heterogeneity and Evolutionary Relationships (RETCHER), which characterizes more realistic cancer cell fractions by accurately identifying mutation multiplicity while considering uncertainty during the reconstruction process and the credibility and reasonableness of subclone clustering. This method comprehensively and accurately infers multiple forms of tumor clonal heterogeneity and phylogenetic relationships. RETCHER outperforms existing methods on simulated data and infers clearer subclone structures and evolutionary relationships in real multisample sequencing data from five tumor types. By precisely analysing the complex clonal heterogeneity within tumors, RETCHER provides a new approach to tumor evolution research and offers scientific evidence for developing precise and personalized treatment strategies. This approach is expected to play a significant role in tumor evolution research, clinical diagnosis, and treatment. RETCHER is available for free at https://github.com/zlsys3/RETCHER.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f822108aa29d6372cc1d56eba22e5f8f0a1fffc3" target='_blank'>
              Reconstructing tumor clonal heterogeneity and evolutionary relationships based on tumor DNA sequencing data
              </a>
            </td>
          <td>
            Zhen Wang, Yanhua Fang, Ruoyu Wang, Liwen Kong, Shanshan Liang, Shuai Tao
          </td>
          <td>2024-09-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) disease progression involves complex cell state transitions in both malignant and non-malignant populations within the tumor microenvironment. To uncover PDAC cell states and epigenetic features associated with clinical outcome, we profiled 40 tumor samples (33 primary tumors and 7 metastases) from 39 patients with single cell ATAC-seq, along with 10 sample-matched single cell RNA-seq profiles. Through topic modeling of high dimensional chromatin data, we uncovered distinct cis-regulatory networks underlying the classical and basal-like molecular subtypes of PDAC, which showed differential activity in basal-classical hybrid cells. Furthermore, delineation of epigenetic alterations by basal, classical, and hybrid subtypes revealed a core set of chromatin regions opened in all PDAC subtypes relative to ductal and acinar cells. These universally open regions showed striking overlap with gastrointestinal enhancers, highlighting a common endoderm lineage infidelity process across pancreatic tumors. Within the stromal compartment, we identified novel regulators of myofibroblastic cancer-associated fibroblasts (myCAFs) and inflammatory CAFs (iCAFs) from enhancer networks active in each CAF subtype. Importantly, our data uncover a highly prognostic gene program active in only one of two distinct myCAF chromatin states found in our data, highlighting a putative tumor-promoting myCAF subpopulation. Our work provides a high-resolution description of epigenetic cell state heterogeneity in PDAC, revealing novel regulators and gene programs associated with PDAC subtypes and clinical outcome.
 Citation Format: Kevin MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, Carl Pelz, Trent Waugh, Ethan Agritelle, John Muschler, Andrew Adey, Rosalie Sears. Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55da9db59167776bfc809f1ebb52c0cfa3226d09" target='_blank'>
              Abstract C078: Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes
              </a>
            </td>
          <td>
            Kevin A MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, C. Pelz, Trent Waugh, Ethan Agritelle, John Muschler, Andrew C. Adey, Rosalie C. Sears
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Esophageal cancer (EC) is a major health issue, ranking seventh in incidence and sixth in mortality worldwide. Despite advancements in multidisciplinary treatment approaches, the 5-year survival rate for EC remains low at 21%. Challenges in EC treatment arise from late-stage diagnosis, high malignancy, and poor prognosis. Understanding the tumor microenvironment is critical, as it includes various cellular and extracellular components that influence tumor behavior and treatment response. Mast cells (MCs), as tissue-resident immune cells, play dual roles in tumor dynamics. High-throughput single-cell RNA sequencing offers a powerful tool for analyzing tumor heterogeneity and immune interactions, although its application in EC is limited. Methods In this study, we investigated the immune microenvironment of EC using single-cell RNA sequencing and established a comprehensive immune profile. We also performed analysis of upstream transcription factors and downstream pathway enrichment to further comprehensively decipher MCs in EC. Besides, we performed knockdown experiments to explore the role of epidermal growth factor receptor (EGFR) signaling pathway in MCs-tumor cell interactions, highlighting its potential as a prognostic marker. Finally, we constructed a prognostic model for EC, which provided valuable suggestions for the diagnosis and prognosis of EC. Results Our analysis identified 11 major cell types, of which MCs were particularly present in pericarcinoma tissues. Further grouping of the 5,001 MCs identified 8 distinct subtypes, including SRSF7-highly expressed MCs, which showed strong tumor preference and potential tumor-promoting properties. Moreover, we identified the key signaling receptor EGFR and validated it by in vitro knockdown experiments, demonstrating its cancer-promoting effects. In addition, we established an independent prognostic indicator, SRSF7+ MCs risk score (SMRS), which showed a correlation between high SMRS group and poor prognosis. Conclusion These findings illuminate the complex interactions within the tumor microenvironment of EC and suggest that targeting specific MCs subtypes, particularly via the EGFR signaling pathway, may present novel therapeutic strategies. This study establishes a comprehensive immune map of EC, offering insights for improved treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b52619ad1abc5e3af051c9a613c06b47617866c" target='_blank'>
              Unraveling the ecological landscape of mast cells in esophageal cancer through single-cell RNA sequencing
              </a>
            </td>
          <td>
            Shengyi Zhang, Xinyi Zhang, Zhikai Xiahou, Shunqing Zuo, Jialong Xue, Yi Zhang
          </td>
          <td>2024-10-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Background Locally advanced oral cavity squamous cell carcinoma (OCSCC) presents a significant clinical challenge despite being partially responsive to standard treatment modalities. This study investigates the prognostic implications of programmed death-ligand 1 (PD-L1) expression in these tumors, focusing on its association with treatment outcomes and the immune microenvironment. Methods We assessed tumor-infiltrating lymphocytes (TILs) in 132 patients with OCSCC to evaluate their impact on survival. Multiplex immunohistochemistry staining for CD3, CD68, CD11c, PD-L1, and P40 was used to explore correlations with clinical outcomes in patients with early-stage (n=22) and locally advanced (n=36) OCSCC. These initial findings were validated through differential gene expression analysis, gene set enrichment, and immune cell deconvolution in a The Cancer Genome Atlas cohort of 163 locally advanced OCSCC tumors. Additionally, single-cell RNA sequencing (scRNA-seq) on a smaller cohort (n=10) further characterized the PD-L1hi or PD-L1lo cancer cells in these tumors. Results Elevated PD-L1 expression was associated with poor outcomes in patients with locally advanced OCSCC undergoing standard adjuvant therapy, irrespective of “hot” or “cold” classification based on TILs assessment. PD-L1hi tumors exhibited an active immune response phenotype, enriched with M1 macrophages, CD8+ T cells and T regulatory cells in the tumor microenvironment. Notably, the negative impact of PD-L1 expression on outcomes was primarily attributed to its expression by cancer cells, rather than immune cells. Furthermore, scRNA-seq revealed that immune interactions were not essential for PD-L1 upregulation in cancer cells, instead, complex regulatory networks were involved. Additionally, PD-L1lo locally advanced tumors exhibited more complex pathway enrichment and diverse T-cell populations compared with those in the early-stage. Conclusion Our findings underscore the prognostic significance of PD-L1 expression in locally advanced OCSCC, and unveil the complex interplay between PD-L1 expression, immune responses, and molecular pathways in the tumor microenvironment. This study provides insights that may inform future therapeutic strategies, including the possibility of tailored immunotherapeutic approaches for patients with PD-L1hi locally advanced OCSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25eba868865a5d2a47d0e7cf7bcfd94b21e79403" target='_blank'>
              Cancer cell-specific PD-L1 expression is a predictor of poor outcome in patients with locally advanced oral cavity squamous cell carcinoma
              </a>
            </td>
          <td>
            Minyu Wang, Lei Qin, K. Thia, T. Nguyen, Sean Macdonald, Simone Belobrov, Sevastjan Kranz, David Goode, J. Trapani, David Wiesenfeld, Paul J Neeson
          </td>
          <td>2024-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Although the combination of immunotherapy and radiotherapy (RT) for the treatment of malignant tumors has shown rapid development, the insight of how RT remodels the tumor microenvironment to prime antitumor immunity involves a complex interplay of cell types and signaling pathways, much of which remains to be elucidated. Four tumor samples were collected from the same abdominal wall metastasis site of the patient with gastric cancer at baseline and during fractionated RT for single-cell RNA and T-cell receptor sequencing. The Seurat analysis pipeline and immune receptor analysis were used to characterize the gastric cancer metastasis ecosystem and investigated its dynamic changes of cell proportion, cell functional profiles and cell-to-cell communication during RT. Immunohistochemical and immunofluorescent staining and bulk RNA sequencing were applied to validate the key results. We found tumor cells upregulated immune checkpoint genes in response to RT. The infiltration and clonal expansion of T lymphocytes declined within tumors undergoing irradiation. Moreover, RT led to the accumulation of proinflammatory macrophages and natural killer T cells with enhanced cytotoxic gene expression signature. In addition, subclusters of dendritic cells and endothelial cells showed decrease in the expression of antigen present features in post-RT samples. More ECM component secreted by myofibroblasts during RT. These findings indicate that RT induced the dynamics of the immune response that should be taken into consideration when designing and clinically implementing innovative multimodal cancer treatment regimens of different RT and immunotherapy approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/523682bb135dcad73fb3cd605aea7d93bedb4b8c" target='_blank'>
              Dynamic single-cell sequencing unveils the tumor microenvironment evolution of gastric cancer abdominal wall metastases during radiotherapy.
              </a>
            </td>
          <td>
            Qianqian Mao, Zhenzhen Wu, Yonghong Lai, Ling Wang, Qiongzhi Zhao, Xi Xu, Xiansheng Lu, Wenjun Qiu, Zhi-Huan Zhang, Jiani Wu, Gaofeng Wang, R. Zhou, Jianhua Wu, Huiying Sun, N. Huang, Xiatong Huang, Luyang Jiang, Yiran Fang, Yuyun Kong, Li Liang, J. Bin, Yulin Liao, Min Shi, Wangjun Liao, D. Zeng
          </td>
          <td>2024-09-26</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Lung cancer is preceded by premalignant lesions, and what factors drive this transformation and the potential regulatory mode in the context of tumor initiation remain to be elucidated. In the course of precancerous lesions, we found a phasic shift in metabolic patterns. Macrophages are a heterogeneous cell population with high plasticity in the tumor microenvironment. Single-cell interaction and metabolic analyses highlighted a cellular state, S100a4+ alveolar macrophages, which exhibited distinct fatty acid metabolic activity, such as palmitic acid metabolism, at the atypical adenomatous hyperplasia stage, accompanied by an angiogenic promoting function in a pre-neoplastic setting. These findings were reproducible in human single-cell transcriptome and had been confirmed by histopathological staining and in vitro cell coculture assays. Taken together, the results from this study demonstrated that specific alveolar macrophage subset contributes to tumorigenesis by altering its metabolic state, suggesting that metabolic interventions targeting this cell state in the early stage of disease may delay neoplastic transformation of the lung epithelium.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/427d6ba88bd2071a9f16317edc51bd68702c55ba" target='_blank'>
              S100a4+ alveolar macrophages accelerate the progression of precancerous atypical adenomatous hyperplasia by promoting fatty acid metabolism
              </a>
            </td>
          <td>
            Hong Huang, Ying Yang, Qiuju Zhang, Yongfeng Yang, Zhenqi Xiong, Shengqiang Mao, Tingting Song, Yilong Wang, Zhiqiang Liu, Hong Bu, Li Zhang, Le Zhang
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Single-cell transcriptome sequencing technology has been applied to decode the cell types and functional states of immune cells, revealing their tissue-specific gene expression patterns and functions in cancer immunity. Comprehensive assessments of immune cells within and across tissues will provide us with a deeper understanding of the tumor immune system in general. Here, we present Cross-tissue Immune cell type or state Enrichment analysis of gene lists for Cancer (CIEC), the first web-based application that integrates database and enrichment analysis to estimate the cross-tissue immune cell type or state. CIEC version 1.0 consists of 480 samples covering primary tumor, adjacent normal tissue, lymph node, metastasis tissue, and peripheral blood from 323 cancer patients. By applying integrative analysis, we constructed an immune cell-type/state map for each context and adopted our previously developed Kyoto Encyclopedia of Genes and Genomes (KEGG) Orthology Based Annotation System (KOBAS) algorithm to estimate the enrichment for context-specific immune cell type/state. In addition, CIEC also provides an easy-to-use online interface for users to comprehensively analyze the immune cell characteristics mapped across multiple tissues, including expression map, correlation, similar genes detection, signature score, and expression comparison. We believe that CIEC will be a valuable resource for exploring the intrinsic characteristics of immune cells in cancer patients and for potentially guiding novel cancer-immune biomarker development and immunotherapy strategies. CIEC is freely accessible at http://ciec.gene.ac/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0854bfcb621205a0dacea3739cf8d7f741cf345" target='_blank'>
              CIEC: Cross-tissue Immune Cell Type Enrichment and Expression Map Visualization for Cancer.
              </a>
            </td>
          <td>
            Jinhua He, Haitao Luo, Wei Wang, Dechao Bu, Zhengkai Zou, Haolin Wang, Hongzhen Tang, Zeping Han, Wenfeng Luo, Jian Shen, Fangmei Xie, Yi Zhao, Zhiming Xiang
          </td>
          <td>2024-10-03</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) that is refractory to treatment frequently leads to extensive systemic and multi-organ metastases. PDAC is characterized as an immune “cold” cancer; however, the persistence of this “cold” tumor immune microenvironment (TIME) across distant metastases, and the mechanisms by which metastases influence their respective TIME, remain poorly understood. In this study, we conducted spatial protein expression profiling of the TIME using the Nanostring GeoMx Digital Spatial Profiling (DSP) platform. We employed a 43- plex panel on 54 tissue samples, including 14 primary tumors and 40 matched metastases from organs commonly affected by PDAC metastases (liver, lung, and peritoneum), obtained through a rapid (“warm”) autopsy program. We quantified protein expression in three cellular compartments: PanCK+ epithelial cells, α-SMA+ fibroblasts, and CD45+ immune cells, across 181 regions of interest (ROIs). Our analysis revealed a significant enrichment of CD45+ immune cells in lung metastases compared to other organ sites, with a notably lower presence of these cells in liver metastases. Focusing on the immune cell compartment, we analyzed the CD45 segments for further investigation. After normalization using housekeeping proteins, including Histone H3, GAPDH, and ribosomal protein S6, we found that the immune profiles were most distinct between liver and lung metastases. Specifically, liver metastases exhibited the greatest deviation in immune profile compared to primary PDAC, lung, and peritoneal metastases. We scored protein expression signatures for various immune cell subsets based on specific marker expressions. When comparing liver to lung metastases, scores for T cells, myeloid cells, checkpoint markers, macrophages, and interferon signaling were significantly lower in the liver, underscoring the immune “coldness” of the liver metastasis environment. Our cohort included six untreated patients and nine patients treated with chemotherapies. We compared protein expressions in each organ site between untreated and treated patients. Notably, treatment induced divergent expression patterns of multiple proteins between liver and lung metastases. While treatment significantly reduced the expression of several immune markers, including CD45 (total immune), CD3 (T cell), CD4 (T helper cell), CD45RO (memory T cell), CD163 (M2 macrophage), and B7-H3 (checkpoint marker), in lung metastases, these markers were significantly upregulated in liver metastases of treated patients. This finding indicates a pronounced divergence in TIME modulation in response to treatment between liver and lung metastases, potentially fostering a treatment-resistant environment in PDAC patients with multi- organ metastasis. Consequently, this urges the need to develop tailored therapeutic strategies for PDAC metastasis, taking into account the distinct immune landscapes of different metastasis sites.
 Citation Format: Jimin Min, Vittorio Branchi, Kimal I Rajapakshe, Nisha Narayanan, Guangsheng Pei, Linghua Wang, Jean L Grem, Thomas C Caffrey, Paul M Grandgenett, Michael A Hollingsworth, Paola A Guerrero, Anirban Maitra. Organ-specific diversity of tumor immune microenvironment in metastatic pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B073.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd4226ecd000fb900607928bb9c305d2b12ea81" target='_blank'>
              Abstract B073: Organ-specific diversity of tumor immune microenvironment in metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Jimin Min, Vittorio Branchi, K. Rajapakshe, Nisha Narayanan, Guangsheng Pei, Linghua Wang, Jean L. Grem, T. Caffrey, P. Grandgenett, Michael A Hollingsworth, Paola A Guerrero, Anirban Maitra
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Characterized by an immune-suppressive tumor microenvironment (TME), pancreatic ductal adenocarcinoma (PDAC) is well-known for its poor prognosis. Tumor associated macrophages (TAMs) play a critical role in PDAC TME. An in-depth understanding of TAMs is helpful to develop new strategies for immunotherapy. Methods A large number of single-cell RNA sequencing data and bulk RNA sequencing data of PDAC were collected for systematic bioinformatics analysis. Characterize subtypes of TAMs at single-cell resolution and its effect on prognosis. Differential gene analysis and cell-cell communication were used to describe the effect on prognosis and validated by the TCGA dataset. Results We used two prognosis-favorable genes, SLC12A5 and ENPP2, to identify a benign M2-like TAMs (bM2-like TAMs), which shared similarities with C1QC + TAMs, CXCL9+ TAMs and CD169+ TAMs, by analyzing scRNA-seq data and bulk RNA data of PDAC. The bM2-like TAMs were revealed to promote T cell activation and proliferation through ALCAM/CD6 interaction. Meanwhile, the bM2-like TAMs were responsible for stroma modeling by altering αSMA+/αSMA-cell ratio. On the contrast, the rest of the M2-like TAMs were defined as malignant M2-like TAMs (mM2-like TAMs), partly overlapping with SPP1+ TAMs. mM2-like TAMs were revealed to promote tumor progression by secretion of MIF and SPP1. Conclusion Our study used two prognosis-favorable genes to divide M2-like TAMs of PDAC into anti-tumor bM2-like TAMs and pro-tumor mM2-like TAMs. The bM2-like TAMs activate T cells through ALCAM/CD6 and generate prognosis-favorable αSMA+ myofibroblasts through secreting TGFβ, which brings insight into heterogeneity of TAMs, prognosis prediction and immunotherapy of PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7511ec98244dccf94f5f8c18ca6c256807241f26" target='_blank'>
              Single-cell transcriptome analysis identifies a novel tumor-associated macrophage subtype predicting better prognosis in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Xiaonan Wang, Dongyi Li, Bo Zhu, Zichun Hua
          </td>
          <td>2024-10-23</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) emerges as a crucial promoter of rapid intra-tumour heterogeneity and oncogene amplification, contributing to tumour progression and adaptability to resistance. Multiple types of ecDNA are often observed within the same cancer cell, promoting high levels of complexity. While the study of multiple ecDNA types is still in its early stages with exciting new clinical and experimental studies in human cancers, few theoretical models have been proposed to understand the mechanisms enabling the maintenance of multiple ecDNA types (mix cells). Here, based on our general framework of modeling multiple ecDNA types, we explore the conditions that allow for a stable coexistence of multiple ecDNA types within single cancer cells. We study how genetic mutations or phenotypic switching among different ecDNA types contribute to the maintenance of ecDNA diversity under different fitness regimes. We show that either positive epistasis or switching is required to stably maintain mix cell populations. Counter-intuitively, intermediate rather than high switching rates lead to the highest fraction of mix cells, which is further confirmed by our analysis of sub-population compositions, transition probabilities between pure and mix states as well as single-cell Shannon indices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/398d1444c0fac5c6ddf4fe45cb3ef1584d2da2f5" target='_blank'>
              The maintenance of multiple ecDNA types in single cancer cells
              </a>
            </td>
          <td>
            Elisa Scanu, Benjamin Werner, Weini Huang
          </td>
          <td>2024-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Cancer stem cells (CSCs) are widely acknowledged as the drivers of tumor initiation, epithelial‐mesenchymal transition (EMT) progression, and metastasis. Originating from both hematologic and solid malignancies, CSCs exhibit quiescence, pluripotency, and self‐renewal akin to normal stem cells, thus orchestrating tumor heterogeneity and growth. Through a dynamic interplay with the tumor microenvironment (TME) and intricate signaling cascades, CSCs undergo transitions from differentiated cancer cells, culminating in therapy resistance and disease recurrence. This review undertakes an in‐depth analysis of the multifaceted mechanisms underlying cancer stemness and CSC‐mediated resistance to therapy. Intrinsic factors encompassing the TME, hypoxic conditions, and oxidative stress, alongside extrinsic processes such as drug efflux mechanisms, collectively contribute to therapeutic resistance. An exploration into key signaling pathways, including JAK/STAT, WNT, NOTCH, and HEDGEHOG, sheds light on their pivotal roles in sustaining CSCs phenotypes. Insights gleaned from preclinical and clinical studies hold promise in refining drug discovery efforts and optimizing therapeutic interventions, especially chimeric antigen receptor (CAR)‐T cell therapy, cytokine‐induced killer (CIK) cell therapy, natural killer (NK) cell‐mediated CSC‐targeting and others. Ultimately use of cell sorting and single cell sequencing approaches for elucidating the fundamental characteristics and resistance mechanisms inherent in CSCs will enhance our comprehension of CSC and intratumor heterogeneity, which ultimately would inform about tailored and personalized interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d170930289915a8a14d31763d5c1c28676522808" target='_blank'>
              Intricate relationship between cancer stemness, metastasis, and drug resistance
              </a>
            </td>
          <td>
            T. C. Dakal, Ravi Bhushan, Caiming Xu, Bhana Ram Gadi, S. Cameotra, Vikas Yadav, Jarek Maciaczyk, Ingo G H Schmidt-Wolf, Abhishek Kumar, Amit Sharma
          </td>
          <td>2024-09-21</td>
          <td>MedComm</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="
 Although pancreatic cancer is generally refractory to immune checkpoint blockade (ICB), recent studies of tumor-infiltrating T cells in human tumor samples demonstrated the presence of in vivo expanded, tumor-reactive T cell receptor (TCR) clonotypes. Here, we explored the T cell repertoire in ICB-refractory murine pancreatic cancer models by combining single-cell transcriptomics with functional TCR characterization. Our initial studies in an orthotopic, transplantable model revealed a remarkable diversity of tumor-reactive TCR clonotypes that could be distinguished from bystander T cells based on gene expression signatures. We identified TCRs exclusively reactive against the tumor of origin, TCRs that also react against an independently derived pancreatic tumor line, and TCRs broadly reactive to tumor cells of diverse tissue origin (e.g. MC38), the latter category being predominant. Subsequent studies were performed in an autochthonous mouse model, in which single focal tumors were induced in the pancreas by means of in vivo electroporation of gene constructs encoding mutant Kras and inducing CRISPR/Cas9-mediated inactivation of tumor suppressor genes. We analyzed the T cell repertoire in tumors with different levels of mutational burden by also inactivating DNA damage repair genes. Interestingly, even in mismatch repair-deficient tumors that featured a high mutational load, the anti-tumor T cell response was primarily directed against recurrent tumor-associated antigens (TAAs) rather than ‘private’ mutanome-encoded neoantigens. Several TAA-reactive TCRs also mediated reactivity against non-transformed cells. Immunopeptidome analysis thus far revealed the identity of 5 cognate antigens targeted by the tumor-reactive TCR repertoire: a mutanome-encoded neoantigen, an epitope encoded by an ectopically expressed endogenous retroviral provirus, and three epitopes derived from TAAs: Lon Peptidase 2 (LONP2), ATPase Na+/K+ Transporting Subunit Beta 3 (ATP1B3) and SEC24B, a component of secretory vesicles. Notably, the overexpression of LONP2 and ATP1B3 was found associated with progression of human cancers, whereas silencing was shown to suppress tumor cell growth. Overexpression of SEC24B in cancers has not been reported, but this protein plays an essential role in the secretory pathway of acinar cells. In summary, in-depth analysis of the natural T cell response in murine pancreatic cancer models revealed a diverse tumor-reactive T cell repertoire that is predominantly directed against TAAs, some of which are also expressed in normal somatic cells. These findings underline the importance of uncovering the antigen-specificity of the natural tumor-reactive TCR repertoire, both in mouse models and human tumor samples, to assess its therapeutic potential and safety with regard to personalized immunotherapy.
 Citation Format: Stefan Zens, Hannes Kehm, Daniel Baumann, Aline Konrad, Zibo Meng, Arnoud H. de Ru, Julian Mochayedi, Francesca Lucato, Angelika B. Riemer, Isabel Poschke, Michael Volkmar, Peter A. van Veelen, Rienk Offringa. Dissection of the T cell infiltrate in mouse pancreatic tumors reveals a diverse tumor-reactive T cell repertoire targeting neoantigens and tumor-associated antigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B042.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426abb4187fdd15d9ff5304c1841ae2ba8d15e16" target='_blank'>
              Abstract B042: Dissection of the T cell infiltrate in mouse pancreatic tumors reveals a diverse tumor-reactive T cell repertoire targeting neoantigens and tumor-associated antigens
              </a>
            </td>
          <td>
            Stefan Zens, Hannes Kehm, Daniel Baumann, Aline S. Konrad, Z. Meng, Arnoud H. de Ru, Julian Mochayedi, Francesca Lucato, Angelika B. Riemer, I. Poschke, Michael Volkmar, P. V. van Veelen, R. Offringa
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="A significant challenge in analyzing spatial transcriptomics data is the effective and efficient detection of spatially variable genes (SVGs), whose expression exhibits non-random spatial patterns in tissues. Many SVGs show spatial variation in expression that is highly correlated with cell type categories or compositions, leading to the concept of cell type-specific spatially variable genes (ctSVGs). Existing statistical methods for detecting ctSVGs treat cell type-specific spatial effects as fixed effects when modeling, resulting in a critical issue: the testing results are not invariant to the rotation of spatial coordinates. Additionally, an SVG may display random spatial patterns within a cell type, and a ctSVG may exhibit random spatial patterns from a general perspective, indicating that an SVG does not necessarily have to be a ctSVG, and vice versa. This poses challenges in real analysis when detecting SVGs or ctSVGs. To address these problems, we propose STANCE, a unified statistical model developed to detect both SVG and ctSVGs in spatial transcriptomics. By integrating gene expression, spatial location, and cell type composition through a linear mixed-effect model, STANCE enables the identification of both SVGs and ctSVGs in an initial stage, followed by a second stage test dedicated to ctSVG detection. Its design ensures robustness in complex scenarios and the results are spatial rotation invariant. We demonstrated the performance of STANCE through comprehensive simulations and analyses of three public datasets. The downstream analyses based on ctSVGs detected by STANCE suggest promising future applications of the model in spatial transcriptomics and various areas of genome biology. A software implementation of STANCE is available at https://github.com/Cui-STT-Lab/STANCE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d67f5df398d264ed4c0dfa9fbb81e5f7b52f6c2a" target='_blank'>
              STANCE: a unified statistical model to detect cell-type-specific spatially variable genes in spatial transcriptomics
              </a>
            </td>
          <td>
            Haohao Su, Yuesong Wu, Bin Chen, Yuehua Cui
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Glioblastoma (GBM) is a dreadful brain tumor, with a particular relationship to the adult subventricular zone (SVZ) that has been described as relevant to disease initiation, progression, and recurrence. Methods We propose a novel strategy for the detection and tracking of xenografted GBM cells that are located in the SVZ, based on an intracerebroventricular (icv) recombinant adeno-associated virus (AAV)-mediated color conversion method. We used different patient-derived GBM stem-like cells (GSCs), which we transduced first with a retroviral vector (LRLG) that included a lox-dsRed-STOP-lox cassette, upstream of the eGFP gene, then with rAAVs expressing the Cre-recombinase. Red and green fluorescence is analyzed in vitro and in vivo using flow cytometry and fluorescence microscopy. Results After comparing the efficiency of diverse rAAV serotypes, we confirmed that the in vitro transduction of GSC-LRLG with rAAV-Cre induced a switch from red to green fluorescence. In parallel, we verified that rAAV transduction was confined to the walls of the lateral ventricles. We, therefore, applied this conversion approach in 2 patient-derived orthotopic GSC xenograft models and showed that the icv injection of an rAAV-DJ-Cre after GSC-LRLG tumor implantation triggered the conversion of red GSCs to green, in the periventricular region. Green GSCs were also found at distant places, including the migratory tract and the tumor core. Conclusions This study not only sheds light on the putative outcome of SVZ-nested GBM cells but also shows that icv injection of rAAV vectors allows to transduce and potentially modulate gene expression in hard-to-reach GBM cells of the periventricular area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076d45b092f9c2622d4d12f4631c9d6323027d0e" target='_blank'>
              Development of an intraventricular adeno-associated virus-based labeling strategy for glioblastoma cells nested in the subventricular zone
              </a>
            </td>
          <td>
            A. Lombard, D. Isci, Gilles Reuter, Emmanuel Di Valentin, Alexandre Hego, Didier Martin, B. Rogister, Virginie Neirinckx
          </td>
          <td>2024-09-28</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) patients with tumors enriched for the basal-like molecular subtype exhibit severe resistance to standard of care treatments and have significantly worse overall survival, compared to patients with classical subtype enriched tumors. It is therefore important to develop resources that allow identification of novel targets in a statistically rigorous way to overcome therapeutic challenges. Here we compiled a single cell RNA sequencing (scRNAseq) atlas of the human pancreas with 229 individual patient samples, aggregated from raw data obtained from published studies. We mapped cell-type specific scRNAseq gene signatures in newly generated (n=12) and existing (n=13) spatial transcriptomics (ST) data from PDAC samples. Analysis of tumor cells from our scRNAseq atlas revealed nine transcriptionally distinct populations, two of which strongly align with a basal gene signature, correlating with worse overall survival in PDAC bulk RNAseq datasets. Deconvolution showed one of the basal populations identified to be the predominant tumor subtype in non- dissociated ST tissues and in vitro tumor cell and organoid PDAC lines. In bulk RNAseq datasets, we uncovered that myofibroblasts (myFb) expressing CXCL10 consistently correlated with both basal subtypes and were found near basal tumor cells using immunostaining. We also identified that besides immune cells, ductal cells also express CXCR3, the receptor for CXCL10, further suggesting a relationship between these cell types in PDAC tumor microenvironment (TME). Overall, we show that our newly built scRNAseq atlas, containing over 700,000 cells, integrated with ST data is a powerful resource, given that it corroborates numerous PDAC TME observations in the literature, with increased statistical power. Moreover, we uncovered a novel association between CXCL10+ myFb and basal tumor cells, underscoring the presence of unique spatial niches in PDAC that should be explored for the development of targeted therapies for this dismal disease.
 Citation Format: Ian M Loveless, Samantha B Kempt, Yuesong Wu, Daniel J Salas-Escabillas, Kailee Hartway, Madison George, Yetunde Makinwa, Thais Stockdale, Samuel D Zwernik, Kendyll Gartrelle, Rohit G Reddy, Daniel Long, Allison Wombell, Julie M Clark, Albert M Levin, David Kwon, Ling Huang, Ralph Francescone, Débora B Vendramini-Costa, Ben Stanger, Adam Alessio, Andrew M Waters, Yuehua Cui, Brian Theisen, Howard C Crawford, Nina G Steele. Human pancreatic cancer single cell atlas reveals association of CXCL10 positive fibroblasts and basal subtype tumor cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B068.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb2cdcff90781c8782f8a20266c95fd99d51d7f6" target='_blank'>
              Abstract B068: Human pancreatic cancer single cell atlas reveals association of CXCL10 positive fibroblasts and basal subtype tumor cells
              </a>
            </td>
          <td>
            Ian Loveless, Samantha B Kempt, Yuesong Wu, Daniel J. Salas-Escabillas, Kailee Hartway, Madison George, Yetunde Makinwa, Thais Stockdale, Samuel Zwernik, K. Gartrelle, Rohit G Reddy, Daniel W Long, Allison Wombell, Julie Clark, Albert Levin, David Kwon, Ling Huang, R. Francescone, D. Vendramini-Costa, Ben Z. Stanger, Adam Alessio, Andrew M. Waters, Yuehua Cui, Brian Theisen, Howard C. Crawford, Nina G. Steele
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background: Gastrointestinal cancer cells display both morphology and physiology diversity, thus posing a significant challenge for precise representation by a single data model. We conducted an in-depth study of gastrointestinal cancer heterogeneity by integrating and analyzing data from multiple modalities. Methods: We used a modified Canny algorithm to identify edges from tumor images, capturing intricate nonlinear interactions between pixels. These edge features were then combined with differentially expressed mRNA, miRNA, and immune cell data. Before data integration, we used the K-medoids algorithm to pre-cluster individual data types. The results of pre-clustering were used to construct the kernel matrix. Finally, we applied spectral clustering to the fusion matrix to identify different tumor subtypes. Furthermore, we identified hub genes linked to these subtypes and their biological roles through the application of Weighted Gene Co-expression Network Analysis (WGCNA) and Gene Ontology (GO) enrichment analysis. Results: Our investigation categorized patients into three distinct tumor subtypes and pinpointed hub genes associated with each. Genes MAGI2-AS3, MALAT1, and SPARC were identified as having a differential impact on the metastatic and invasive capabilities of cancer cells. Conclusion: By harnessing multimodal features, our study enhances the understanding of gastrointestinal tumor heterogeneity and identifies biomarkers for personalized medicine and targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279c68f2f427231b32eddf4811a28d710e5bb1cf" target='_blank'>
              Tumor Heterogeneity in Gastrointestinal Cancer Based on Multimodal Data Analysis
              </a>
            </td>
          <td>
            Dongmei Ai, Yang Du, Hongyu Duan, Juan Qi, Yuduo Wang
          </td>
          <td>2024-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aaee723a7ca519349b96ee47e6c8c191b4d3f26" target='_blank'>
              Single-cell transcriptome analysis reveals immune microenvironment changes and insights into the transition from DCIS to IDC with associated prognostic genes
              </a>
            </td>
          <td>
            Yidi Sun, Zhuoyu Pan, Ziyi Wang, Haofei Wang, Leyi Wei, Feifei Cui, Quan Zou, Zilong Zhang
          </td>
          <td>2024-10-03</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Introduction: A complex immunosuppressive tumor microenvironment (TME) is a distinctive feature of pancreatic ductal adenocarcinoma (PDAC). The TME consists of an admixture of cell types including epithelial, endothelial, immune cells of both myeloid and lymphoid origin, and cancer-associated fibroblasts (CAFs). The traditionally defined subtypes of PDAC tumor cells include classical, associated with the retention of epithelial markers and a better prognosis, and basal, which is characterized by a quasi-mesenchymal phenotype with poorer responses to chemotherapy and worse overall survival. Additionally, CAF subtypes are associated with specific roles within the TME. These include inflammatory (iCAF), with the capacity to secrete IL-6, myofibroblastic (myCAF), defined by high levels of alpha smooth muscle actin (αSMA) expression, and antigen presenting (apCAF), defined by expression of HLA-DR and antigen processing and presentation capacity. We hypothesized that interactions between CAFs and tumor cells altered tumor cell phenotype and could be delineated. Methods: Using multi-omics platforms, we evaluated the impact of CAF populations on tumor cell phenotype. Our approach integrated multiplex immunohistochemistry, imaging mass cytometry (IMC) and RNA-scope of patient tumor tissues with functional in vitro studies from matched patient-derived organoid (PDO) and CAF cocultures. The dynamic impact of CAFs on PDOs was examined using a combination of bulk RNA sequencing, multiparametric flow cytometry and secretome analysis, in addition to RT-qPCR and western blot comparing cocultures to matched PDO monocultures. Results: IMC of 15 patient tumors revealed distinct neighborhoods of CAFs and tumor cells. iCAFs were in close proximity to classical tumor cells, while myCAFs were in close proximity to basal tumor cells. Further, in our PDO-CAF co-cultures the PDOs underwent a shift from classical to basal subtype in 42% of co-cultures, a phenomenon also inferred in patient tissues by associating collagen content with classical or basal designation. We also captured a tumor cell population expressing both basal and classical markers that may represent a transition state. The shift from classical to basal tumor cell subtype was accompanied by enhanced epithelial-mesenchymal transition and a loss of IL-8 secretion. Interestingly, when we examined immune cell distribution within tumor tissues, T cells were both the most proximal and the most abundant immune cell type in relation to tumor cells. Moreover, the highest frequency of total and activated T cells was seen in close association with basal tumor cells. Conclusions: Our study underscores the importance of understanding the complex interplay between CAFs and tumor cells in PDAC. Further, we identify a relationship between T cells and tumor cell subtypes. By leveraging high-dimensional data and patient-derived models, we provide new insights into the spatial and functional dynamics of the PDAC TME, paving the way for novel therapeutic strategies to reprogram the TME and enhance patient survival.
 Citation Format: Jacquelyn W Zimmerman, Samantha Guinn, Mili Ramani, Brayan Perez, Joseph A Tandurella, Daniel J Zabransky, Jae W Lee, Emma Kartalia, Jignasha Patel, Sarah Shin, Alexei Hernandez, Courtney Cannon, Nicole Gross, Soren Charmsaz, Elana J Fertig, Richard A Burkhart, Won J Ho, Elizabeth M Jaffee. Cancer associated fibroblasts drive classical to basal change associated with increased T cell presence in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A032.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b1b3de60c255399613042a94100d74cd1acaab3" target='_blank'>
              Abstract A032: Cancer associated fibroblasts drive classical to basal change associated with increased T cell presence in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jacquelyn W. Zimmerman, Samantha Guinn, Mili Ramani, Brayan Perez, Joseph A Tandurella, Daniel J. Zabransky, Jae W. Lee, Emma Kartalia, Jignasha Patel, Sarah M. Shin, Alexei Hernandez, Courtney D Cannon, Nicole E. Gross, Soren Charmsaz, E. Fertig, Richard A Burkhart, W. Ho, Elizabeth M. Jaffee
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 The pancreas is composed of the epithelial and mesenchymal cells. While mesenchymal fibroblasts are a minor component of the normal pancreas, fibroblast population expands drastically during tumorigenesis. In pancreatic ductal adenocarcinoma (PDAC), cancer associated fibroblasts (CAFs) play critical and complex roles in the tumor microenvironment. This study sought to define the origin, heterogeneity and function of pancreatic cancer associated fibroblasts. Recently we performed a series of lineage tracing studies in genetically engineered mouse models. This identified the splanchnic mesenchyme (a tissue layer adjacent to the developing fetal pancreatic epithelium) as the fetal origin of the adult pancreatic resident fibroblasts and pancreatic CAFs. Single cell transcriptomic analysis indicated persistent and dynamic gene expressions along the pancreatic mesenchymal trajectory during development, homeostasis, precancer and cancer. Intriguingly, we found that two splanchnic transcription factors, GATA6 and FOXF1, are expressed in only subsets of adult pancreatic fibroblasts in temporally and spatially distinct patterns. Similar patterns were observed in both mouse models and human patient samples. To determine the role of GATA6 in fibroblasts during tumorigenesis, we constructed a dual DNA recombinase mouse genetic model. DNA recombinase FlpO directs expression of an oncogene Kras (G12D mutation) and loss of a tumor suppressor p53 in pancreatic epithelial cells to induce spontaneous tumor formation in the pancreas, and DNA recombinase Cre deletes Gata6 specifically in fibroblasts. This fibroblast specific Gata6 deletion resulted in altered number and gene expression of CAFs as well as increased tumor burden in the pancreas. This suggests a non-cell autonomous function of GATA6 within CAFs to restrain pancreatic cancer progression. In summary, this study delineated a continuous cell trajectory of the mesenchymal lineage in the pancreas across different life stages. Additionally, this study provides evidence that persistent and selective gene expressions along the mesenchymal trajectory contributes to pancreatic CAF heterogeneity and such persistence may constitute an inherent host defense mechanism to restrain pancreatic cancer. The enhancement of this mechanism could be explored further for therapeutic benefits.
 Citation Format: Lu Han, Tom Walter, Joseph Beaudet, Caroline Everett, Gustavo Leone, Michael Ostrowski. Persistence of fetal splanchnic gene signature defines a tumor-restraining fibroblast subtype in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44596654399cf752308f05003ed354c1a1239f4c" target='_blank'>
              Abstract A044: Persistence of fetal splanchnic gene signature defines a tumor-restraining fibroblast subtype in pancreatic cancer
              </a>
            </td>
          <td>
            Lu Han, Tom Walter, Joseph Beaudet, Caroline Everett, Gustavo Leone, Michael C. Ostrowski
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Glioblastoma trigger systemic immune responses, resulting in elevated white blood cell counts and elevated temperatures, as the body attempts to combat tumor cells. However, glioblastoma reprograms immune cells to evade immune-mediated destruction and promote tumor growth. While targeted immunotherapies have shown promise in some cancers like melanoma, they have not been efficacious for malignant brain tumors, leading to significant societal costs due to high morbidity and mortality rates.



 Understanding immune cell trafficking is crucial for developing effective immunotherapies. We conducted spatially resolved single-cell analyses to investigate immune cell entry into glioblastoma. Integrating these data with multi-omic sequencing analyses of anatomically dissected specimens from glioblastoma-affected human brain border regions and tumor cores revealed distinct paths of immune cell trafficking into glioblastoma.



 One path involves an ordered entry of mononuclear cells through the subarachnoid and perivascular spaces, guided by chemotactic cues. Cells in these regions display markers of immune activation, suggesting immune priming. Another path occurs at sites of vascular leakage due to hypoxia and necrosis, where a mix of mononuclear cells and neutrophils accumulate. These cells exhibit immunosuppressive transcriptomic programs, regulated by the phagocytosis-inducing transcription factor MITF and downstream metabolic changes controlled by PPAR-gamma. Ongoing analyses aim to identify pharmacological agents that can modulate immune cell reprogramming at hypoxic niches.



 In summary, we have identified distinct modes of immune cell trafficking in glioblastoma. Chemotaxis-mediated trafficking via the brain border regions appears to prime immune cells, while entry at hypoxic niches is associated with immune suppression. Pharmacological agents shielding immune cells from immunosuppressive molecular cues in the tumor microenvironment show promise for glioblastoma immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8f5d39f4dab2e338bbcd19df4cc3dd4650f4fd" target='_blank'>
              P02.20.A NICHE-DEPENDENT MODES OF IMMUNE CELL TRAFFICKING IN GLIOBLASTOMA OFFER TARGETS FOR NOVEL IMMUNOTHERAPIES
              </a>
            </td>
          <td>
            R. Sankowski, A. Dalmau Gasull, M. Prinz
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b7649961c3767077277c2ff33140a98201a7bd6" target='_blank'>
              Characterization and spatial distribution of infiltrating lymphocytes in medullary, and lymphocyte-predominant triple negative breast cancers
              </a>
            </td>
          <td>
            A. Alfaro, C. Catelain, H. El-Masri, P. Rameau, M. Lacroix-Triki, JY. Scoazec, V. Marty, F. Mosele, B. Pistilli
          </td>
          <td>2024-09-14</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Although generally unknown, the age of a newly diagnosed tumor encodes valuable etiologic and prognostic information. Here, we estimate the age of breast cancers, defined as the time from the start of growth to detection, using a measure of epigenetic entropy derived from genome-wide methylation arrays. Based on an ensemble of neutrally fluctuating CpG (fCpG) sites, this stochastic epigenetic clock differs from conventional clocks that measure age-related increases in methylation. We show that younger tumors exhibit hallmarks of aggressiveness, such as increased proliferation and genomic instability, whereas older tumors are characterized by elevated immune infiltration, indicative of enhanced immune surveillance. These findings suggest that the clock captures a tumor’s effective growth rate resulting from the evolutionary-ecological competition between intrinsic growth potential and external systemic pressures. Because of the clock’s ability to delineate old and stable from young and aggressive tumors, it has potential applications in risk stratification of early-stage breast cancers and guiding early detection efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38dff77a79c0945793feb47e32141039b96a0e80" target='_blank'>
              Mapping the Temporal Landscape of Breast Cancer Using Epigenetic Entropy
              </a>
            </td>
          <td>
            Daniel L. Monyak, Shannon T. Holloway, Graham J. Gumbert, Lars J. Grimm, E. S. Hwang, Jeffrey R. Marks, Darryl Shibata, Marc D. Ryser
          </td>
          <td>2024-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Glioblastoma (GB) stem cells (GSCs) are described to recapitulate the expression signatures of stem and progenitor cell classes in the neurodevelopmental hierarchy and are implicated in driving therapeutic resistance. However, the heterogeneity of both the GSC subclasses and their expression of stemness markers hinders comprehensive labelling of these cells. Cellular dormancy is a key feature of GSCs that can be captured by label retention method to track slowly dividing cells. We used CellTrace (CT) dye to identify long term label retaining GB cells (LRGC) in orthotopic xenograft model. We analysed the distribution of LRGC versus the tumour bulk in histology. Additionally, we analysed the transcriptional signature by FAC-sorting LGRC for single cell RNA sequencing. Further, we demonstrated that LRCs survived through temozolomide (TMZ) treatment in vivo and re-initiated tumour proliferation in a faster pace than other surviving glioma cells, signifying its role in therapeutic relapse.



 To label LRGC, patient derived GB cell lines were stained with CT dye before transplantation into the striatum of immunodeficient mice. After 6 weeks of growth the brains were cryosectioned, histologically stained and imaged via confocal microscopy, followed by histologic AI assisted analysis with the AIVIA software. LRGC and non-label retaining tumor cells were FAC-sorted for single-cell RNA sequencing (SMART seq2). To explore the role of LGRC in therapeutic resistance, tumors were treated with TMZ. Re-initiation of tumor growth was tracked by subcutaneous surgical implantation of BrdU pumps at different timepoints after therapy. The proliferation patterns of LRGC versus other surviving tumor cells were analysed in histology again using an AI assisted software.



 LRGC could be identified in histology 6 weeks after tumor growth. LRGC were found near the centre of tumor but rarely at the invasion front. Single cell RNA-seq of LRGC showed differentially expressed pathways related to lipid metabolism and neurodevelopment. Re-initiation of tumor growth started 12-14 days after therapy. LRGC showed a more intense proliferation compared to other surviving tumor cells, indicating the ability of LRGC to not only survive TMZ treatment, but also drive GB relapse.



 LRGC could be visualized 6-8 weeks after tumor growth in an orthotopic xenograft model. On a transcriptional level these cells share similarities with stem cells in normal brain development but also show distinct features. An important role of LRGC in re-initiation of tumor growth after therapy could be confirmed.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863c1df5d02a6308a5d876cfe8995a6849735071" target='_blank'>
              P03.02.B DECIPHERING THE IDENTITY AND CHARACTERISTICS OF LONG-TERM LABEL RETAINING GLIOBLASTOMA CELLS AND THEIR ROLE IN TUMOUR RELAPSE
              </a>
            </td>
          <td>
            H. Chow, A. Kourtesakis, L. Hai, S. Schaffranek, S. Pusch, D. Hoffmann, D. Reibold, W. Wick, R. Wagener
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d829dac90ae6e0bee2354e14ecf9cb6f0b45ddf8" target='_blank'>
              Polyclonal-to-monoclonal transition in colorectal precancerous evolution.
              </a>
            </td>
          <td>
            Zhaolian Lu, Shanlan Mo, Duo Xie, Xiangwei Zhai, Shanjun Deng, Kantian Zhou, Kun Wang, Xueling Kang, Hao Zhang, Juanzhen Tong, Liangzhen Hou, Huijuan Hu, Xuefei Li, Da Zhou, L. T. Lee, Li Liu, Yaxi Zhu, Jing Yu, Ping Lan, Jiguang Wang, Zhen He, Xionglei He, Zheng Hu
          </td>
          <td>2024-10-30</td>
          <td>Nature</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fc425805bf39c40a323743bba1f215c7c5c8c55" target='_blank'>
              Digital spatial profiling for pathologists.
              </a>
            </td>
          <td>
            B. Donati, G. Manzotti, F. Torricelli, Cristian Ascione, Riccardo Valli, G. Santandrea, Moira Ragazzi, Eleonora Zanetti, A. Ciarrocchi, S. Piana
          </td>
          <td>2024-11-05</td>
          <td>Virchows Archiv : an international journal of pathology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4229ea75fc458fb4ee692eac3c38175128262ee2" target='_blank'>
              Effect of fibroblast heterogeneity on prognosis and drug resistance in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Tingjie Wang, Lingxi Tian, Bing Wei, Jun Li, Cuiyun Zhang, Ruitao Long, Xiaofei Zhu, Yougai Zhang, Bo Wang, Guangbo Tang, Jun Yang, Yongjun Guo
          </td>
          <td>2024-11-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Spatial transcriptomics technology allows for the detection of cellular transcriptome information while preserving the spatial location of cells. This capability enables researchers to better understand the cellular heterogeneity, spatial organization and functional interactions in complex biological systems. However, current technological methods are limited by low resolution, which reduces the accuracy of gene expression levels. Here, we propose scstGCN, a multimodal information fusion method based on Vision Transformer (ViT) and Graph Convolutional Network (GCN) that integrates histological images, spot-based spatial transcriptomics data and spatial location information to infer super-resolution gene expression profiles at single-cell level. We evaluated the accuracy of the super-resolution gene expression profiles generated on diverse tissue ST datasets with disease and healthy by scstGCN along with their performance in identifying spatial patterns, conducting functional enrichment analysis, and tissue annotation. The results show that scstGCN can predict super-resolution gene expression accurately, aid researchers in discovering biologically meaningful differentially expressed genes and pathways. Additionally, scstGCN can segment and annotate tissues at a finer granularity, with results demonstrating strong consistency with coarse manual annotations. Key Points scstGCN combines multi-modal information including histology image, spot-based spatial transcriptomics (ST) data, and physical spatial location through deep learning methods to achieve single-cell resolution of spot-based ST data without requiring single-cell references. scstGCN employs GCN to capture complex relationships between neighboring cells, facilitating the integration of multimodal feature information based on single-cell level, and then accurately infers single-cell resolution spatial gene expression. scstGCN can infer single-cell resolution gene expression across the entire tissue region. Through transfer learning, gene expression in three-dimensional tissues can be characterized efficiently. Furthermore, it demonstrates outstanding performance in spatial patterns enhancement, functional enrichment analysis, and annotate tissues at the high-resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11e229cf48d595d655cf9018d85b7b9b1f77f893" target='_blank'>
              Inferring single-cell resolution spatial gene expression via fusing spot-based spatial transcriptomics, location and histology using GCN
              </a>
            </td>
          <td>
            Shuailin Xue, Fangfang Zhu, Jinyu Chen, Wenwen Min
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Aim The dynamic interplay between cancer cells and the microenvironment involves a wide range of intricate relationships that evolve during different stages of tumor progression. Recent attention has focused on high endothelial venules (HEVs), specialized endothelial cells in tumors with a unique cuboidal shape similar to those in lymph nodes. Previous animal studies have shown that normalization of tumor angiogenesis through anti-VEGFR2 therapy promotes HEV formation. However, few reports exist regarding the relationship between HEVs and preexisting blood vessels or interstitial fibers. In this study, we histologically examined whether tumor vascular structure correlates with HEV neogenesis. Patients and Methods A total of 109 patients with pathological stage I lung adenocarcinoma who had undergone curative lung resection at our Institute between 2012 and 2016 were included. HEVs were identified by anti-peripheral node addressin (PNAd) staining. Immunostaining and Elastica-Masson-Goldner staining were performed on tumor sections and quantified. Results PNAd-positive cells were identified in 102 (93.6%) patients. Nearly all PNAd-positive cells were located within or near immune cell clusters. We investigated the correlation between microvessel structures or interstitial fibers and the number/density of PNAd-positive vessels, but no significant correlation was found. Since PNAd-positive cells were concentrated in immune cell aggregates, we focused our analysis specifically on these regions. Immune cell aggregates with abundant PNAd-positive vessels had a greater microvessel density along with by rich collagen fiber production, and displayed a more mature morphological phenotype of HEVs. Conclusion The generation of PNAd-positive cells in tumors is governed by an angiogenetic mechanism distinct from that of broader tumor microenvironment. Furthermore, the accumulation of immune cells is associated with increased HEV maturation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6515292d87876d38e023114fce1cab413aa327" target='_blank'>
              Histopathological Correlation Between High Endothelial Venule Neogenesis and the Tumor Microenvironment in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Ato Sugiyama, Tai Hato, Hiroaki Kashimada, Masatoshi Yamaguchi, Y. Inoue, K. Aoki, Hiroki Fukuda, M. Nakayama, Morihiro Higashi, Mitsutomo Kohno
          </td>
          <td>2024-10-29</td>
          <td>In Vivo</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a0fc0f5df38dda8be158f2e4cdef84f02b31da8" target='_blank'>
              Spatial proteomics: unveiling the multidimensional landscape of protein localization in human diseases
              </a>
            </td>
          <td>
            Mengyao Wu, Huihui Tao, Tiantian Xu, Xuejia Zheng, Chunmei Wen, Guoying Wang, Yali Peng, Yong Dai
          </td>
          <td>2024-09-20</td>
          <td>Proteome Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma multiform (GBM), a grade IV glioma, is the most common and aggressive cancer in the central nervous system. Current treatment for GBM includes surgical resection, radiation, and the frontline DNA alkylating drug, temozolomide (TMZ). The current median survival for GBM patients is about 14.5 months with 5% patients surviving up to 5 years. We propose that discerning distinct subsets within heterogeneous GBM will provide avenues for research to improve new therapies. We used different methods to isolate GBM cell subsets. These include stable transfectants of GBM cell lines with a lentiviral system in which green fluorescence protein (GFP) is regulated with tandem repeats of Oct4a and Sox2 response elements. Parallel studies with a plasmid using the full-length regulatory region of Oct4a indicated reduced efficiency in separating cell subsets, relative to SORE6-GFP lentivirus. Stem cell-linked gene expressions and function studies such as ALDH1, tumorsphere and in vivo passaging of GFP hi subsets confirmed the presence of cancer stem cells (CSCs). We also studied a more efficient method that could be relevant for primary GBM cells. We selected tumorspheres by plating heterogeneous GBM cells and then serially passaged the spheres. Studies for stem cell genes indicated that this method could be used for primary GBM cells. Overall, this study provided insights into methods to isolate GBM subsets, including primary GBM cells. The advantages of the methods are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492e350655b514b418d1c77aa6fb2234fa8abba3" target='_blank'>
              A Platform to Establish a Working Hierarchy of Glioblastoma Multiforme Cells:
              </a>
            </td>
          <td>
            V. H. Savanur, Anushka Sarkar, Andrew Petryna, Ky Nguyen, Jesus Benites-Sandoval, Marina Gergues, Arash Hatefi, P. Rameshwar
          </td>
          <td>2024-09-24</td>
          <td>International Journal of Translational Science</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Liver cancer represents a substantial global health challenge, contributing significantly to worldwide morbidity and mortality. It has long been understood that tumors are not composed solely of cancerous cells, but also include a variety of normal cells within their structure. These tumor-associated normal cells encompass vascular endothelial cells, fibroblasts, and various inflammatory cells, including neutrophils, monocytes, macrophages, mast cells, eosinophils, and lymphocytes. Additionally, tumor cells engage in complex interactions with stromal cells and elements of the extracellular matrix (ECM). Initially, the components of what is now known as the tumor microenvironment (TME) were thought to be passive bystanders in the processes of tumor proliferation and local invasion. However, recent research has significantly advanced our understanding of the TME’s active role in tumor growth and metastasis. Tumor progression is now known to be driven by an intricate imbalance of positive and negative regulatory signals, primarily influenced by specific growth factors produced by both inflammatory and neoplastic cells. This review article explores the latest developments and future directions in understanding how the TME modulates liver cancer, with the aim of informing the design of novel therapies that target critical components of the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d21309486c5cd52dddda7197045d0a0b43a28e44" target='_blank'>
              Beyond Cancer Cells: How the Tumor Microenvironment Drives Cancer Progression
              </a>
            </td>
          <td>
            Hussein Sabit, B. Arneth, Shaimaa Abdel-Ghany, Engy F. Madyan, Ashraf H. Ghaleb, Periasamy Selvaraj, Dong M. Shin, Ramireddy Bommireddy, Ahmed Elhashash
          </td>
          <td>2024-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Spatial transcriptomic technologies are becoming increasingly high-resolution, enabling precise measurement of gene expression at the subcellular level. Here, we introduce a computational method called subcellular expression localization analysis (ELLA), for modeling the subcellular localization of mRNAs and detecting genes that display spatial variation within cells in high-resolution spatial transcriptomics. ELLA creates a unified cellular coordinate system to anchor diverse cell shapes and morphologies, utilizes a nonhomogeneous Poisson process to model spatial count data, leverages an expression gradient function to characterize subcellular expression patterns, and produces effective control of type I error and high statistical power. We illustrate the benefits of ELLA through comprehensive simulations and applications to four spatial transcriptomics datasets from distinct technologies, where ELLA not only identifies genes with distinct subcellular localization patterns but also associates these patterns with unique mRNA characteristics. Specifically, ELLA shows that genes enriched in the nucleus exhibit an abundance of long noncoding RNAs or protein-coding mRNAs, often characterized by longer gene lengths. Conversely, genes containing signal recognition peptides, encoding ribosomal proteins, or involved in membrane related activities tend to enrich in the cytoplasm or near the cellular membrane. Furthermore, ELLA reveals dynamic subcellular localization patterns during the cell cycle, with certain genes showing decreased nuclear enrichment in the G1 phase while others maintain their enrichment patterns throughout the cell cycle. Overall, ELLA represents a calibrated, powerful, robust, scalable, and versatile tool for modeling subcellular spatial expression variation across diverse high-resolution spatial transcriptomic platforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e20fa135a1de302ac8c0f3a99633dfe53cde3ba4" target='_blank'>
              ELLA: Modeling Subcellular Spatial Variation of Gene Expression within Cells in High-Resolution Spatial Transcriptomics
              </a>
            </td>
          <td>
            Jade Xiaoqing Wang, Xiang Zhou
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7721878e5f49189d68fbf5ffadef4b78f908c349" target='_blank'>
              Deciphering the role of LGALS2: insights into tertiary lymphoid structure-associated dendritic cell activation and immunotherapeutic potential in breast cancer patients
              </a>
            </td>
          <td>
            Shuyu Li, Nan Zhang, Hao Zhang, Zhifang Yang, Q. Cheng, Kang Wei, Meng Zhou, Chen-shen Huang
          </td>
          <td>2024-09-30</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f00e386abbf367932af2ed3eee8d073da659417c" target='_blank'>
              Multi-stage mechanisms of tumor metastasis and therapeutic strategies
              </a>
            </td>
          <td>
            Zaoqu Liu, Jingqi Chen, Yuqing Ren, Shutong Liu, Yuhao Ba, Anning Zuo, Peng Luo, Q. Cheng, Hui-Xian Xu, Xinwei Han
          </td>
          <td>2024-10-11</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 KRAS mutations are common in non-small cell lung cancer (NSCLC) and often co-occur with TP53 mutations. Although the KRAS oncogene was once considered ‘undruggable,’ FDA approval of KRASG12C inhibitors and promising pre-clinical data from KRASG12D inhibitors highlight the potential for targeted therapies. The tumor microenvironment (TME) of NSCLC comprises immune and stromal cells, including antigen-presenting cancer-associated fibroblasts (for simplicity, hereafter ‘apCAFs,’ regardless of cancer association), a population that expresses markers that interact with and affect T cell activation, exhaustion and differentiation. However, the role of apCAFs in tumorigenesis, how they interact with T cells, and their response to KRAS inhibitors remain unknown. A crucial need exists to characterize apCAFs to optimize therapeutic approaches. We used an inducible and reversible KrasG12D lung cancer mouse model (L-iKras) to perform single cell RNA sequencing analyses. Lung tissue from two each (1 male/1 female) of control mice, mice with KrasG12D turned on with doxycycline (ON), and mice with KrasG12D turned on for 20 weeks and off for 1 week via doxycycline withdrawal (OFF) were included. Fibroblasts were clustered and stratified using the Seurat package. One population resembled apCAFs expressing Pdgfra, Clec3b, Col1a1, and Col1a2 and antigen-presenting markers H2-Ab1, H2-Aa, and Cd74. This population increased in ON and OFF groups compared to controls and expressed genes required for three signals of T cell activation: engagement of the T cell receptor with the peptide- MHC complex of an antigen-presenting cell (APC); binding of costimulatory proteins CD28 or CTLA4 (T cell) with CD80/CD86 (APC); and secreted cytokines (APC) initiating T cell differentiation. This apCAF population showed increased markers for immune checkpoints Cd274 (PDL1) and Cd86, which can inhibit or exhaust T cells, upon KRAS inhibition. Further, these apCAFs upregulated markers involved with the induction of Tregs upon KRAS inhibition (Tgfb1). Treg populations were increased, and CD8 T cell populations were decreased in the OFF group compared to the ON group. These findings suggest that apCAFs are induced in the TME during tumorigenesis and interact with tumor-infiltrating T cells. Further, we hypothesize that apCAFs become immunosuppressive upon KrasG12D inhibition, activate Tregs, and induce CD8 T cell exhaustion. Interestingly, upregulation of immune checkpoints in apCAFs suggests that these cells may respond well to checkpoint therapies, which could be harnessed for future therapeutic approaches in NSCLC. The focus of our future studies will be to validate our findings in an orthotopic lung cancer mouse model developed by our lab, which utilizes KrasG12D-inducible and reversible lung cancer cell lines derived from L- iKras. With this model we will further characterize apCAF-T cell interactions, investigate the impact of KRAS inhibitors on apCAFs, and identify the role of apCAFs in tumor responses to combinations of KRAS inhibitors and checkpoint inhibitors.
 Citation Format: Rachael Hinshaw, Carlos Espinoza, Yaqing Zhang, Marina Pasca di Magliano, Stefanie Galban. Investigating the role of antigen-presenting fibroblasts in the tumor microenvironment of non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr B019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beac9e00fd1ced3575c16790fafc36abcc21b523" target='_blank'>
              Abstract B019: Investigating the role of antigen-presenting fibroblasts in the tumor microenvironment of non-small cell lung cancer
              </a>
            </td>
          <td>
            Rachael Hinshaw, Carlos E Espinoza, Yaqing Zhang, M. Pasca di Magliano, Stefanie Galbán
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Immune dysregulation plays a major role in cancer progression. The quantification of lymphocytic spatial inflammation may enable spatial system biology, improve understanding of therapeutic resistance, and contribute to prognostic imaging biomarkers.In this paper, we propose a knowledge-guided deep learning framework to measure the lymphocytic spatial architecture on human H&E tissue, where the fidelity of training labels is maximized through single-cell resolution image registration of H&E to IHC. We demonstrate that such an approach enables pixel-perfect ground-truth labeling of lymphocytes on H&E as measured by IHC. We then experimentally validate our technique in a genetically engineered, immune-compromised Rag2 mouse model, where Rag2 knockout mice lacking mature lymphocytes are used as a negative experimental control. Such experimental validation moves beyond the classical statistical testing of deep learning models and demonstrates feasibility of more rigorous validation strategies that integrate computational science and basic science.Using our developed approach, we automatically annotated more than 111,000 human nuclei (45,611 CD3/CD20 positive lymphocytes) on H&E images to develop our model, which achieved an AUC of 0.78 and 0.71 on internal hold-out testing data and external testing on an independent dataset, respectively. As a measure of the global spatial architecture of the lymphocytic microenvironment, the average structural similarity between predicted lymphocytic density maps and ground truth lymphocytic density maps was 0.86 ± 0.06 on testing data. On experimental mouse model validation, we measured a lymphocytic density of 96.5 ± %1% in a Rag2+/- control mouse, compared to an average of 16.2 ± %5% in Rag2-/- immune knockout mice (p<0.0001, ANOVA-test).These results demonstrate that CD3/CD20 positive lymphocytes can be accurately detected and characterized on H&E by deep learning and generalized across species. Collectively, these data suggest that our understanding of complex biological systems may benefit from computationally-derived spatial analysis, as well as integration of computational science and basic science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85260d40c721d3e1f8d5b2764225eb5a4620514b" target='_blank'>
              Computational staining of CD3/CD20 positive lymphocytes in human tissues with experimental confirmation in a genetically engineered mouse model
              </a>
            </td>
          <td>
            Xiang Li, Casey C. Heirman, Ashlyn G Rickard, Gina Sotolongo, Rico Castillo, Temitayo Adanlawo, Jeffery I. Everitt, Jeffery B. Hodgin, Tammara L. Watts, Andrew Janowczyk, Yvonne Mowery, Laura Barisoni, K. Lafata
          </td>
          <td>2024-10-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE
Immunotherapies have led to a paradigm shift in the treatment of hepatocellular carcinoma (HCC). Studies have revealed the single-cell catalogues of tumor-infiltrating immune cells and the trajectories of their differentiation. Nevertheless, the spatial distribution of these immune cells with distinct phenotypes in tumor microenvironment and their clinicopathological significance in resectable and unresectable HCCs are still largely unclear.


EXPERIMENTAL DESIGN
We analyzed the spatial dynamics of intratumoral CD4 and CD8 T cells and their association with B and plasma cells using 283 surgically resected HCCs, 58 unresectable HCC samples before combined immunotherapy (atezolizumab plus bevacizumab [Atezo+Bev]), and autopsy specimens from 50 cases of advanced-stage HCCs through multiplex immunohistochemistry combined with transcriptomic and driver gene mutation analysis. Classification based on the spatial dynamics of T and B cell responses (refined immunosubtype) was developed and its clinicopathological significance was analyzed.


RESULTS
We found that stem-like CD4 and CD8 T cells were mainly observed in T cell aggregates. Differentiation of T follicular helper cells were associated with the development of tertiary lymphoid structures, whereas differentiation of CD4 TCXCL13 cells with phenotype resembling T peripheral helper cells were associated with the development of lymphoplasmacytic microenvironment. The refined immunosubtype could predict clinical outcomes of resectable HCC after surgery and unresectable HCC after Atezo+Bev therapy. The immune microenvironment of metastatic lesions tended to reflect those of primary lesions.


CONCLUSIONS
We revealed the spatial dynamics of T and B cell response in HCC, which is closely associated with the clinical outcome after surgical resection or Atezo+Bev therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30a9933c34558056c95ceeec869060a2ac77726f" target='_blank'>
              Spatial Dynamics of T and B Cell Response Predicts Clinical Outcome of Resectable and Unresectable Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Yutaka Kurebayashi, K. Sugimoto, Hanako Tsujikawa, Kosuke Matsuda, Rui Nomura, A. Ueno, Y. Masugi, Ken Yamazaki, Kathryn Effendi, Hirohito Takeuchi, Takao Itoi, Y. Hasegawa, Yuta Abe, M. Kitago, H. Ojima, Michiie Sakamoto
          </td>
          <td>2024-10-17</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Traditional drug discovery efforts have largely focused on targeting rapid, reversible protein-mediated adaptations to undermine cancer cells’ resistance to therapy. However, cancer cells also exploit DNA-based strategies, typically viewed as slow, irreversible, and unpredictable changes like point mutations or the selection of drug-resistant clones. Contrary to this perception, extrachromosomal DNA (ecDNA) represents a form of DNA alteration that is rapid, reversible, and predictable, playing a crucial role in cancer’s adaptive response. In this study, we present a novel post-processing pipeline for the automated detection and quantification of ecDNA in Fluorescence in situ Hybridization (FISH) images using the Microscopy Image Analyzer (MIA) tool. Our approach is particularly designed to monitor ecDNA dynamics during drug treatment, providing a quantitative framework to understand how ecDNA enables cancer cells to swiftly and reversibly adapt to therapeutic pressure. This pipeline not only offers a valuable resource for researchers aiming to automate ecDNA detection in FISH images but also sheds light on the adaptive mechanisms of ecDNA in response to epigenetic remodeling agents like JQ1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a33656032abe4b14c89c4da5e33bf1e9ab825af" target='_blank'>
              Leveraging AI to Automate Detection and Quantification of Extrachromosomal DNA (ecDNA) to Decode Drug Responses
              </a>
            </td>
          <td>
            Kohen Goble, Aarav Mehta, Damien Guilbaud, Jacob Fessler, Jingting Chen, William Nenad, Oliver Cope, Darby Cheng, William Dennis, Nithya Gurumurthy, Yue Wang, Kriti Shukla, Elizabeth Brunk
          </td>
          <td>2024-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Pancreatic cancer, recognized as one of the most lethal cancer types, exhibits a dismal five-year survival rate below 12%. The genetic and epigenetic heterogeneity in human cancer poses a tremendous challenge in understanding disease mechanisms and developing targeted therapeutics. Current mouse models in the field can recapitulate some features of human pathophysiology but have limitations. While tumor formation in patients is focal, most genetically engineered mouse models (GEMMs) express oncogenes throughout the pancreas. GEMMs also have limited flexibility to manipulate gene expression and model the heterogeneity of human cancer. Additionally, GEMMs require significant time for crossing and tumor formation. Tumor transplantation models offer greater flexibility, however, they often require immunodeficient mice, which limits their applications in modeling the tumor microenvironment. To overcome the limits of current mouse models, we combined the piggyBac transposon system, CRISPR/Cas9, and in vivo electroporation to engineer somatic pancreatic tissue to induce focal tumor formation in immune-competent mice. With this approach, we engineered wildtype pancreata to express oncogenic KrasG12D while knocking out multiple tumor suppressors. We optimized this protocol by generating multiplexed gRNA constructs and cell type-specific promoters to improve reproducibility. Low-grade tumors formed within 10 days after electroporation, progressed to higher grade within a month, and metastasized to peritoneal organs during disease progression. Mouse tumors also recapitulated histopathological features in human patients. Lastly, we used our novel mouse model to study how mutant Kras disrupts post-transcriptional regulation by microRNA to drive pancreatic tumorigenesis. This model offers a powerful platform for manipulating gene expression and functionality to model cancer heterogeneity, ultimately advancing precision oncology's prospects for treating human patients.
 Citation Format: Ziyue (Zoey) Yang, Mara Cardenas, Akihiko Miyauchi, Caroline Snider, Ronald Parchem. Generation of a flexible pancreatic cancer mouse model via somatic tissue engineering [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02653c15c1c4443d9e4685c7c408a134b796978a" target='_blank'>
              Abstract A006: Generation of a flexible pancreatic cancer mouse model via somatic tissue engineering
              </a>
            </td>
          <td>
            Z. Yang, Mara Cardenas, Akihiko Miyauchi, Caroline Snider, Ronald Parchem
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ebdd957ebc89410899ec23dbdacf2c9d724e672" target='_blank'>
              Differential cellular communication in tumor immune microenvironment during early and advanced stages of lung adenocarcinoma.
              </a>
            </td>
          <td>
            Mudita Shukla, R. Sarkar
          </td>
          <td>2024-10-26</td>
          <td>Molecular genetics and genomics : MGG</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/484efc8a4ee08b37da85a0f29cd67a40a824997b" target='_blank'>
              Spatial analysis reveals targetable macrophage-mediated mechanisms of immune evasion in hepatocellular carcinoma minimal residual disease.
              </a>
            </td>
          <td>
            Lea Lemaitre, Nia Adeniji, Akanksha Suresh, Reshma Reguram, Josephine Zhang, J. Park, Amit Reddy, Alexandro E. Trevino, Aaron T. Mayer, A. Deutzmann, Aida S Hansen, Ling Tong, Vinodhini Arjunan, N. Kambham, B.C. Visser, M. Dua, C. A. Bonham, N. Kothary, H. B. D'Angio, Ryan Preska, Yanay Rosen, James Y. Zou, Vivek Charu, Dean W Felsher, R. Dhanasekaran
          </td>
          <td>2024-09-20</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
 The complex biology of pancreatic ductal adenocarcinoma (PDAC) remains a significant challenge and limits overcoming its dismal prognosis. The tumor microenvironment (TME), shaped by dynamic interactions between cancer-associated fibroblasts (CAFs), immune and tumor cells, plays a crucial role in PDAC pathogenesis. Uncharted field lies in the impact of specific gene mutations within the tumor epithelium on orchestrating distinct oncogenic TME patterns. Our investigation into homologous recombination deficiency (HRD), caused by mutations in genes like ATM, reveals its role in promoting PDAC aggressiveness and triggering a pronounced desmoplastic reaction, suggesting a unique TME programing. Through single-cell-resolved and multiomics analyses of murine ATM and/or P53-depleted PDACs, and validation in human cases, we unveil transcriptional, translational, and secretory regulatory events, shed light on the intricate interplay between tumor genetics and PDAC microenvironment and pave the way for the elaboration of novel personalized therapies. Murine tumor analysis using single-nucleus multiomics RNA and ATAC sequencing, alongside histological validation in human tissues, revealed specific enrichment of myCAFs in ATM-deficient tumors and reduced populations of T cells and macrophages. Molecular signatures and cell-cell communication inference uncovered ATM-specific interactions within the TME, highlighting TGF-β signaling as pivotal in mediating dialog between CAFs and an aggressive EMT-like tumor cell subset prevalent in ATM-depleted tumors. Additionally, distinct communication patterns were observed between CAFs and immune cells, associated with specific epigenetic and molecular profiles. Mechanistically, transcriptomics, secretomics and proteomics investigations confirmed increased TGFβ release by ATM-deficient PDAC cells and identified a reactive oxygen species (ROS)-mediated mechanism affecting tumor cell behavior and interactions with CAFs. Consistently, a combinatorial therapy targeting TGF-β-mediated tumor-stroma dialog reversed cancer-promoting TME remodeling and exacerbated FOLFIRINOX cytotoxic effects, in vivo, exclusively in ATM-deficient HRD PDACs. Dissecting the biology of ATM-deficient HRD malignant cells, our study unveils their pivotal role in orchestrating oncogenic communication networks with CAFs and immune cells, along with specific tumor promoting stromal reprogramming. Our findings paint a comprehensive picture of how distinct tumor-stroma interactions reshape the TME, steering it towards a cancer-promoting outcome and shed light on the potential to exploit genotype-specific vulnerabilities via tailored tumor-stroma interference strategies.
 Citation Format: Elodie Roger, Hannah M. Mummey, Eleni Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, Lukas Perkhofer, Johann Gout, Alexander Kleger. Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fd93ca5c41a43eb8ee9eb151fa6f7aeb554c533" target='_blank'>
              Abstract A055: Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer
              </a>
            </td>
          <td>
            E. Roger, Hannah M. Mummey, E. Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, L. Perkhofer, J. Gout, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Spatial transcriptomics methods capture cellular measurements such as gene expression and cell types at specific locations in a cell, helping provide a localized picture of tissue health. Traditional visualization techniques superimpose the tissue image with pie charts for the cell distribution. We design an interactive visual analysis system that addresses perceptual problems in the state of the art, while adding filtering, drilling, and clustering analysis capabilities. Our approach can help researchers gain deeper insights into the molecular mechanisms underlying complex biological processes within tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c511a6033fe68a8e269b93e1ec6ba4d5c45da80" target='_blank'>
              A Part-to-Whole Circular Cell Explorer.
              </a>
            </td>
          <td>
            Siyuan Zhao, G. Marai
          </td>
          <td>2024-10-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="This study delves into the proteomic intricacies of drug-resistant cells (DRCs) within prostate cancer, which are known for their pivotal roles in therapeutic resistance, relapse, and metastasis. Utilizing single-cell proteomics (SCP) with an optimized high-throughput Data Independent Acquisition (DIA) approach with the throughput of 60 sample per day, we characterized the proteomic landscape of DRCs in comparison to parental PC3 cells. This optimized DIA method allowed for robust and reproducible protein quantification at the single-cell level, enabling the identification and quantification of over 1,300 proteins per cell on average. Distinct proteomic sub-clusters within the DRC population were identified, closely linked to variations in cell size. The study uncovered novel protein signatures, including the regulation of proteins critical for cell adhesion and metabolic processes, as well as the upregulation of surface proteins and transcription factors pivotal for cancer progression. Furthermore, by integrating SCP and single-cell RNA-seq (scRNA-seq) data, we identified six upregulated and ten downregulated genes consistently altered in drug-treated cells across both SCP and scRNA-seq platforms. These findings underscore the heterogeneity of DRCs and their unique molecular signatures, providing valuable insights into their biological behavior and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3153689159ed447ea718eb77046a432729554a1" target='_blank'>
              Single-Cell Proteomic and Transcriptomic Characterization of Drug-Resistant Prostate Cancer Cells Reveals Molecular Signatures Associated with Morphological Changes
              </a>
            </td>
          <td>
            Jong-Su Woo, Michael A. Loycano, Md Amanullah, Jiang Qian, Sarah Amend, Kenneth J Pienta, Hui Zhang
          </td>
          <td>2024-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Background Pancreatic ductal adenocarcinoma (PDAC) is the fourth leading cause of cancer- related deaths in the United States and is often not diagnosed until it has already metastasized. Our prior work established that calcium signaling pathways facilitate a pro-metastatic phenotype through the induction of an epithelial-to-mesenchymal transition (EMT) program in PDAC tumor cells, but the downstream signaling molecules mediating this phenotype have remained elusive. MethodsUsing a novel genetically engineered model of pancreatic cancer, we demonstrate that loss of Calcium/calmodulin-dependent kinase-II B (CaMK2B), a downstream effector kinase activated by calcium, enhances cellular plasticity, and increases metastasis. To this end, we generated a genetically engineered mouse model where Camk2b was knocked out in tumor cells in a lineage-labeled Kras- and p53-driven PDAC mouse model (herein KPCY-Camk2bcKO). Results Genetic deletion of Camk2b in the KPCY genetically engineered model accelerated tumor formation and dramatically reduced the overall median survival of KPCY- Camk2b cKO mice (78 days) relative to KPCY controls (136 days). Pathological and immunofluorescence analysis of KPCY-Camk2b cKO animals showed an increased number of poorly differentiated tumor regions and a higher metastatic burden, with >85% of KPCY- Camk2b cKO animals presenting with multi-focal metastases. Further analysis of tumor cells in KPCY-Camk2b cKO mice showed enhanced expression of markers of the highly aggressive squamous PDAC subtypes. Strikingly, Camk2b deletion combined with oncogenic Kras G12D (in the absence of p53 alterations), was sufficient to form lethal metastatic PDAC tumors, while age-matched control Kras G12D tumors with intact Camk2b only progressed to the acinar-to-ductal metaplasia (ADM) or Pancreatic Intraepithelial Neoplasia (PanIN) stage. Collectively, these results establish Camk2b loss as a new oncogenic and pro-metastatic mechanism in PDAC Conclusion This work demonstrates that Camk2b loss enhances tumor cell evolution and plasticity within PDAC to promote metastasis. In this context, we speculate that loss of Camk2b rewires calcium signaling to drive an aggressive subtype of PDAC with increased metastasis.
 Citation Format: Jessica Peura, Nikita Bhalerao, Calvin Johnson, Yamini Ogoti, Faith Keller, Emma Watson, Zhen Zhao, Jason Pitarresi. Loss of CaMK2B accelerates tumor formation and enhances metastatic competency in genetically engineered mouse models of PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C061.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7848880ff2d74aa871376f23a8aeef10bd0463c1" target='_blank'>
              Abstract C061: Loss of CaMK2B accelerates tumor formation and enhances metastatic competency in genetically engineered mouse models of PDAC
              </a>
            </td>
          <td>
            Jessica Peura, Nikita Bhalerao, Calvin Johnson, Yamini Ogoti, Faith Keller, Emma Watson, Zhen Zhao, Jason R. Pitarresi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d663e089c1c202b486de405e9b1fc737ad2ffa1" target='_blank'>
              SOX10 mediates glioblastoma cell-state plasticity
              </a>
            </td>
          <td>
            Ka-Hou Man, Yonghe Wu, Zhenjiang Gao, Anna-Sophie Spreng, Johanna Keding, Jasmin Mangei, P. Boskovic, Jan-Philipp Mallm, Hai-Kun Liu, Charles D. Imbusch, Peter Lichter, B. Radlwimmer
          </td>
          <td>2024-09-16</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="


 Astrocytoma is a diffuse and heterogeneous glioma that infiltrates the brain. Despite post-operative therapy including radiation and chemotherapy, 52% of patients’ tumours recur within five-years of surgery. These tumours are characterised by mutations in the IDH1 or IDH2 gene that confer metabolic alterations, most notably the production of the onco-metabolite 2-hydroxyglutarate. To reveal the metabolic alterations and their upstream regulation across the heterogeneous tumour microenvironment, we applied spatial metabolomics and spatial transcriptomics to sequential sections from resected tumour samples of patients with low grade astrocytoma.



 Spatial assessment of heterogeneity was performed on serial sections using spatial metabolomics (atmospheric pressure MALDI-TOF) and spatial transcriptomics (10X Genomics Xenium). This necessitated extensive optimisation of image registration across modalities and sequential sections to create three dimensional models to deeply interrogate heterogeneity within tumours.



 Combining spatial transcriptomics and spatial metabolomics across 8 matched sequential slices of the same patient tumour, enabled the identification of metabolic niches associated with distinct cell type composition and transcriptional regulation. Strikingly, normal domains that were present at the leading edge of tumours could be identified both metabolically and transcriptomically.



 Understanding the heterogeneity of astrocytomas on a spatial level reveals unique domains and organisation of tumour and immune cell types. Matching these transcriptomic and cell state information with metabolic insights leads to pathway insights that will be essential to the identification of druggable targets. Our pipeline combining spatial transcriptomics and metabolomics with tissue histology reveals these biologically important domains.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbf36889624e6059d6b35a03a0d2ffccf6ee4adc" target='_blank'>
              OS07.5.A MULTI-MODAL EXAMINATION OF SPATIAL HETEROGENEITY IN THE ASTROCYTOMA MICROENVIRONMENT
              </a>
            </td>
          <td>
            T. Lu, L. Freytag, J. Moffet, J. Kriel, J. Whittle, S. Freytag, S. A. Best
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9875ac28ea781567d1b798ee476f9d9fa7efb581" target='_blank'>
              Miniaturized protein profiling permits targeted signaling pathway analysis in individual circulating tumor cells to improve personalized treatment
              </a>
            </td>
          <td>
            Mahdi Rivandi, A. Franken, Liwen Yang, A. Abramova, Nadia Stamm, J. Eberhardt, B. Gierke, Meike Beer, Tanja Fehm, D. Niederacher, Michael Pawlak, Hans Neubauer
          </td>
          <td>2024-09-20</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b397d7b281fa7e047bec3772e9d91f03230a0f4" target='_blank'>
              Profiling the cancer-prone microenvironment in a zebrafish model for MPNST
              </a>
            </td>
          <td>
            Cheryl Cero, John S. House, Vincenzo Verdi, J. L. Ferguson, Dereje D. Jima, Aubrie A. Selmek, Olivia M. Patania, Jennifer E. Dwyer, Bih-Rong Wei, Dillon Lloyd, H. Shive
          </td>
          <td>2024-11-07</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="ABSTRACT Background Small cell lung cancer (SCLC) harbours the most aggressive phenotype of all lung cancers to correlate with its bleak prognosis. The aggression of SCLC is partially attributable to its strong metastatic tendencies. The biological processes facilitating the metastasis in SCLC are still poorly understood and garnering a deeper understanding of these processes may enable the exploration of additional targets against this cancer hallmark in the treatment of SCLC. Recent Findings This narrative review will discuss the proposed molecular mechanisms by which the cancer hallmark of activating invasion and metastasis is featured in SCLC through important steps of the metastatic pathway, and address the various molecular targets that may be considered for therapeutic intervention. The tumour immune microenvironment plays an important role in facilitating immunotherapy resistance, whilst the poor infiltration of natural killer cells in particular fosters a pro‐metastatic environment in SCLC. SCLC vasculogenesis is achieved through VEGF expression and vascular mimicry, and epithelial‐mesenchymal transition is facilitated by the expression of the transcriptional repressors of E‐cadherin, the suppression of the Notch signalling pathway and tumour heterogeneity. Nuclear factor I/B, selectin and B1 integrin hold important roles in SCLC migration, whilst various molecular markers are expressed by SCLC to assist organ‐specific homing during metastasis. The review will also discuss a recent article observing miR‐1 mRNA upregulation as a potential therapeutic option in targeting the metastatic activity of SCLC. Conclusion Treatment of SCLC remains a clinical challenge due to its recalcitrant and aggressive nature. Amongst the many hallmarks used by SCLC to enable its aggressive behaviour, that of its ability to invade surrounding tissue and metastasise is particularly notable and understanding the molecular mechanisms in SCLC metastasis can identify therapeutic targets to attenuate SCLC aggression and improve mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e819255be722f10508752f999900a9b1c75e2697" target='_blank'>
              Activating Invasion and Metastasis in Small Cell Lung Cancer: Role of the Tumour Immune Microenvironment and Mechanisms of Vasculogenesis, Epithelial‐Mesenchymal Transition, Cell Migration, and Organ Tropism
              </a>
            </td>
          <td>
            Carl He
          </td>
          <td>2024-10-01</td>
          <td>Cancer Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell fate decisions, such as proliferation, differentiation, and death, are driven by complex molecular interactions and signaling cascades. While significant progress has been made in understanding the molecular determinants of these processes, historically, cell fate transitions were identified through light microscopy that focused on changes in cell morphology and function. Modern techniques have shifted towards probing molecular effectors to quantify these transitions, offering more precise quantification and mechanistic understanding. However, challenges remain in cases where the molecular signals are ambiguous, complicating the assignment of cell fate. During viral infection, programmed cell death (PCD) pathways, including apoptosis, necroptosis, and pyroptosis, exhibit complex signaling and molecular crosstalk. This can lead to simultaneous activation of multiple PCD pathways, which confounds assignment of cell fate based on molecular information alone. To address this challenge, we employed deep learning-based image classification of dying cells to analyze PCD in single Herpes Simplex Virus-1 (HSV-1)-infected cells. Our approach reveals that despite heterogeneous activation of signaling, individual cells adopt predominantly prototypical death morphologies. Nevertheless, PCD is executed heterogeneously within a uniform population of virus-infected cells and varies over time. These findings demonstrate that image-based phenotyping can provide valuable insights into cell fate decisions, complementing molecular assays.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23ea913674617fc1b694df526cfac8afeb0fe5af" target='_blank'>
              Deep learning-based image classification reveals heterogeneous execution of cell death fates during viral infection
              </a>
            </td>
          <td>
            Edoardo Centofanti, Alon Oyler-Yaniv, Jennifer Oyler-Yaniv
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="High Grade Serous Ovarian Cancer (HG-SOC), the most prevalent and aggressive gynecological malignancy, is marked by ubiquitous loss of functional p53, largely due to point mutations that arise very early in carcinogenesis. These mutations often lead to p53 protein misfolding and subsequent aggregation, yet the alterations in intracellular p53 dynamics throughout ovarian cancer progression remain poorly understood. HG-SOC originates from the fallopian tube epithelium, with a well-documented stepwise progression beginning with early pre-malignant p53 signatures. These signatures represent largely normal cells that express and accumulate mutant p53, which then transform into benign serous tubal intraepithelial lesions (STIL), progress into late pre-malignant serous tubal intraepithelial carcinoma (STIC), and ultimately lead to HGSOC. Here, we show that the transition from folded, soluble to aggregated mutant p53 occurs during the malignant transformation of benign precursor lesions into HGSOC. We analyzed fallopian tube tissue collected from ten salpingo-oophorectomy cases and determined the proportion of cells carrying soluble versus mis-folded/mutant p53 through conformation-sensitive staining and quantification. Misfolded p53 protein, prone to aggregation, is present in STICs and HG-SOCs, but notably absent from preneoplastic lesions and surrounding healthy tissue. Overall, our results indicate that aggregation of mutant p53 is a structural defect that distinguishes preneoplastic early lesions from late premalignant and malignant ones, offering a potential treatment window for targeting p53 aggregation and halting ovarian cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e004df7ee189f194b9dc43a60b80d81862858bf" target='_blank'>
              Mutant p53 Misfolding and Aggregation Precedes Transformation into High-Grade Serous Ovarian Carcinoma
              </a>
            </td>
          <td>
            S. Sartini, Lexi Omholt, Neda A Moatamed, A. Soragni
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="BACKGROUND
Cancer-Associated Fibroblasts (CAFs) constitute a heterogeneous group of cells critical for the remodeling of the tumor microenvironment (TME). Given their significant impact on tumor progression, particularly in skin cancers, a deeper understanding of their characteristics and functions is essential.


METHODS
This study employed a single-cell transcriptomic analysis to explore the diversity of CAFs within three major types of skin cancer: basal cell carcinoma, melanoma, and head and neck squamous cell carcinoma. We applied analytical techniques, including Gene Ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), Gene Set Enrichment Analysis (GSEA), pseudotime tracking, metabolic profiling, and stemness assessment to delineate and define the functional attributes of identified CAF subgroups.


RESULTS
Our analysis successfully delineated nine distinct CAF subgroups across the studied tumor types. Of particular interest, we identified a novel CAF subtype, designated as C0, exclusive to basal cell carcinoma. This subtype exhibits phenotypic traits associated with invasive and destructive capabilities, significantly correlating with the progression of basal cell carcinoma. The identification of this subgroup provides new insights into the role of CAFs in cancer biology and opens avenues for targeted therapeutic strategies.


CONCLUSION
A pan-cancer analysis was performed on three cancers, BCC, MA, and HNSCC, focusing on tumor fibroblasts in TME. Unsupervised clustering categorized CAF into nine subpopulations, among which the C0 subpopulation had a strong correspondence with BCC-CAF and an invasive- destructive-related phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b65ae0c3c55b73c69971a741aaa77cb534354a3" target='_blank'>
              Pan-Cancer Single-Cell Analysis Revealing the Heterogeneity of Cancer-Associated Fibroblasts in Skin Tumors.
              </a>
            </td>
          <td>
            Yichi Zhang, Zhijie Zhao, Wenyi Huang, Byeong Seop Kim, Li Lin, Xin Li, Mengyuan Hou, Li Li, Yan Zhang, Wenjing Xi, G. Chai
          </td>
          <td>2024-09-25</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="While immune checkpoint blockade (ICB) therapy has shown promising results in a small subset of colorectal cancer patients with high microsatellite instability (MSI-H), the majority of patients with colorectal cancer do not respond to ICB therapy. The main obstacle to the success of immunotherapy in cancer treatment is the exhaustion of tumor-infiltrating lymphocytes (TILs). Elucidating the spatial organization of immune checkpoints within the tumor microenvironment could pave the way for the development of novel prognostic tools and therapeutic strategies to enhance antitumor immune responses. To clarify the spatial and functional diversity of tumor-infiltrating lymphocytes (TILs) in the colorectal tumor microenvironment (TME), we performed multiplexed IHC to examine the exhaustion of TILs in the TME (the expression of PD-1 and TIM-3 (T-cell immunoglobulin and mucin-domain-containing protein 3), which are major biomarkers of T-cell exhaustion) and Lasso-Cox analyses of the correlation between CRC prognosis and TME features. For proof of concept, the antitumor efficacy of TIM-3 and PD-1 dual blockade in CRC was further evaluated in a CT26 subcutaneous tumor model of human CRC. We found that the spatial context of PD-1 and TIM-3 successfully predicted the overall survival of CRC patients independent of TNM stage. Dual targeting of PD-1 and TIM-3 in mouse tumor models inhibited tumor progression and reduced T-cell exhaustion, indicating a potential strategy for improving the clinical treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b7e667f607ec054318ced952999594c8c938db" target='_blank'>
              Spatial context of immune checkpoints as predictors of overall survival in patients with resectable colorectal cancer independent of standard TNM stages.
              </a>
            </td>
          <td>
            Hao Kong, Qingxin Yang, Chunwei Wu, Xiangji Wu, Xinrui Yan, Li-Bin Huang, Lu Chen, Zongguang Zhou, Ping Wang, Hong Jiang
          </td>
          <td>2024-11-01</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract The human placenta is a vital organ, encompassing many distinct cell types, that maintains the growth and development of the fetus and is essential for substance exchange, defense, synthesis, and immunity. Abnormalities in placental cells can lead to various pregnancy complications, but the mechanisms remain largely unclear. Single‐cell and spatial transcriptomics technologies have been developed in recent years to demonstrate placental cell heterogeneity and spatial molecular localization. Here, we review and summarize the current literature, demonstrating these technologies and showing the heterogeneity of various placenta cells and cell–cell communication of normal human placenta, as well as placenta‐related diseases, such as preeclampsia, gestational diabetes mellitus, advanced maternal age, recurrent pregnancy loss, and placenta accreta spectrum disorders. Meanwhile, current weaknesses and future direction were discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b889ea7da8b6acb03f7b66bb0a59e2967532e104" target='_blank'>
              Single‐cell and spatial transcriptomics: Discovery of human placental development and disease
              </a>
            </td>
          <td>
            Mi Tang, Liling Xiong, Jianghui Cai, Li Fan, Cheng Huang, Shimao Zhang, Ying Jin, Erdan Luo, Shasha Xing, Xiao Yang
          </td>
          <td>2024-09-21</td>
          <td>FASEB BioAdvances</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4788ed4991cb05a8733cec69abe88f6738bf0a69" target='_blank'>
              Differential chromatin accessibility and transcriptional dynamics define breast cancer subtypes and their lineages.
              </a>
            </td>
          <td>
            Michael D. Iglesia, R. Jayasinghe, Siqi Chen, Nadezhda V. Terekhanova, John M Herndon, E. Storrs, A. Karpova, D. Zhou, Nataly Naser Al Deen, Andrew T Shinkle, Rita Jui-Hsien Lu, Wagma Caravan, Andrew Houston, Yanyan Zhao, Kazuhito Sato, Preet Lal, Cherease Street, Fernanda Martins Rodrigues, Austin N. Southard-Smith, A. L. N. Targino da Costa, Houxiang Zhu, Chia-Kuei Mo, Lisa Crowson, Robert S Fulton, Matthew A. Wyczalkowski, C. Fronick, L. Fulton, Hua Sun, Sherri R Davies, Elizabeth L. Appelbaum, S. Chasnoff, Madelyn Carmody, Candace Brooks, Ruiyang Liu, M. Wendl, C. Oh, Diane E Bender, C. Cruchaga, O. Harari, Andrea L Bredemeyer, K. Lavine, R. Bose, J. Margenthaler, Jason M. Held, Samuel Achilefu, F. Ademuyiwa, R. Aft, Cynthia X Ma, G. A. Colditz, Tao Ju, Stephen T. Oh, James A J Fitzpatrick, E. S. Hwang, K. Shoghi, M. Chheda, D. Veis, Feng Chen, Ryan C. Fields, William E Gillanders, Li Ding
          </td>
          <td>2024-10-30</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>65</td>
        </tr>

        <tr id="Approximately 10% of Renal Cell Carcinoma (RCC) tumors form a thrombus that invades into nearby vasculature and is associated with lower survivability. However, the mechanisms driving tumor thrombus formation and progression are poorly understood. We therefore examined a publicly available RNAseq dataset of samples from RCC patients containing tumors with and without thrombi, as well as associated normal kidney tissues, and compared them. Using cell deconvolution, we found indications that kidney tumors and thrombi are associated with loss of mature kidney-specific cell types and gain of proliferative characteristics. Moreover, we identified a set of transcripts that are alternatively spliced between tumors with and without thrombi. Using motif enrichment analysis for known RNA binding proteins we found putative splicing regulators that are presumably associated with thrombus formation. We believe that this study will assist in unraveling cellular and molecular mechanisms of tumor progression and thrombus formation in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45491fd142b0369bad9a9b9ac0d3e1b89ef193d6" target='_blank'>
              Characterization of alternative mRNA splicing associated with tumor thrombus in clear-cell renal cell carcinoma
              </a>
            </td>
          <td>
            Zeev Cohen, Harel Reinus, I. Ben-Dov, T. Kalisky
          </td>
          <td>2024-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbc3ebbaf27f078aaa2e9a9bf031bdb59bb995bd" target='_blank'>
              Bioinformatics analysis and experimental validation of the oncogenic role of COL11A1 in pan-cancer.
              </a>
            </td>
          <td>
            Xiaofeng Wan, Qingmei Deng, Anling Chen, Xinhui Zhang, Wulin Yang
          </td>
          <td>2024-11-04</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Pancreatic cancer is a highly lethal and aggressive disease, commonly characterized by a unique tumor microenvironment, where mutant KRAS signaling plays a pivotal role in its pathophysiology. To deepen our understanding of KRAS-dependent signaling in earliest tumor development, oncogenic KRAS (KRASG12D) expression was induced in human stem cell-derived pancreatic ductal-like organoids (PDLOs), followed by time-resolved single-cell RNA and bulk ATAC sequencing. This unique precancerous state model facilitated a comprehensive characterization of early oncogenic events at cellular resolution. Furthermore, a machine learning approach was applied to accurately predict KRAS-dependent cell identities. Within seven days, mutated KRAS triggers distinct gene signature programs related to extracellular matrix remodeling and inflammation. Although derived from dysplastic yet non-cancerous states, these gene signatures negatively correlated with overall survival in PDAC (pancreatic ductal adenocarcinoma) patients. The KRAS-dependent signatures retrieved with the machine learning approach allowed the grouping of 24 published PDAC cases, showing differences in niche composition and communication patterns. Various co-culture systems demonstrated that KRASG12D-expressing PDLOs could activate pancreatic stellate cells, inducing a cancer- associated fibroblast (CAF)-like state. Simultaneously, these KRAS-activated organoids were protected from T cell infiltration in vitro. Furthermore, in silico approaches were employed to reconstruct a virtual pancreatic cancer space, demonstrating that our PDLOs transcriptionally bridge healthy and malignant ductal states. This also revealed potential KRAS-driven pathways mediating CAF activation and T cell shielding preceding cancer onset. In summary, our human PDLO model enables us to explore the transition from normal to cancerous states, addressing gaps in our understanding of tumor development and tumor niche preparation.
 Citation Format: Chantal Allgoewer, Markus Breunig, Meike Hohwieler, Medhanie Mulaw, Alexander Kleger. Single-cell profiling unleashes KRAS-driven redrafting of the tumor microenvironment before cancer onset [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc4900be652d49c2d242f617b8845d915d29b" target='_blank'>
              Abstract C010: Single-cell profiling unleashes KRAS-driven redrafting of the tumor microenvironment before cancer onset
              </a>
            </td>
          <td>
            Chantal Allgoewer, M. Breunig, M. Hohwieler, Medhanie M Mulaw, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) it though to develop through induction of different precursor lesions, however, the normal cell type that gives rise to these precancerous lesions and PDAC is unclear. Recent studies using mouse models and ex vivo cultured human cells suggest that the cellular origin of PDAC may be either acinar or ductal cells. Potential transcriptional markers associated with PDAC of different cellular origins have been identified. Here, we interrogate whether these markers are predictive of cellular of origin across models using mouse models where acinar and ductal cells (Ptf1aCreER or Sox9CreER) expressing oncogenic Kras in the absence of Trp53 or Pten give rise to PDAC tumors. We found that acinar- and ductal-cell-derived PDAC exhibit transcriptional differences both in vivo and in vitro when comparing within one genotype or sample source type (e.g. bulk tumor or primary cell lines). However, previously predicted markers of cellular origin had variable expression across models and sample types suggestive of differences specific to those sample sets. Using our larger sample sets and multiple genotypes and source types, we proposed a more comprehensive set of cell of origin markers for consideration. Application of these markers to our own data sets, however, failed to properly segregate acinar- and ductal-cell-derived PDAC into separate groups. This suggests that the transcriptional heterogeneity observed in our sample set was not driven by the tumors arising from different cellular origins. We also examined the relationship between cell of origin and patient PDAC transcriptional subtypes using our mouse model datasets. Our findings support previous research indicating that mouse models of acinar-cell-derived PDAC favor those of a classical subtype. However, our mouse PDAC samples all express similar or very low levels of the genes denoting the basal subtype, suggesting that the basal subtype was not strongly present in any of the datasets we examined. In summary, our data suggest that acinar and ductal cells converge into a very similar PDAC transcriptional state and that, if differences can be identified, they will likely be identified by non-transcript based readouts.
 Citation Format: Ken Chu, Alex Lee, Karnjit Sarai, Yan Dou, Wesley Hunt, Emma Croft, Atefeh Samani, Farnaz Taghizedeh, Claire Dubois, Stephane Flibotte, Janel Kopp. Cell of origin does not underlie transcription heterogeneity in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7619576b4dd9f9188da32a5b5d7af49469c5cb88" target='_blank'>
              Abstract A025: Cell of origin does not underlie transcription heterogeneity in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ken Chu, Alex Y. L. Lee, Karnjit Sarai, Yan Dou, Wesley Hunt, Emma Croft, Atefeh Samani, Farnaz Taghizedeh, Claire L. Dubois, Stephane Flibotte, Janel L. Kopp
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Lymph node metastasis contributed to the leading cause and treatment failure in nasopharyngeal carcinoma (NPC). The microenvironment and the cellular communications of lymph node metastasized tumours determine the tumour progression and therapeutic effect, but the ecosystems about the lymph node metastasis (LNM) for NPC patients remain poorly characterized. Here, we integrated the transcriptomes of 47,618 single cells from eight samples related to NPC LNM. The dynamic immune ecosystems and immunosuppressive microenvironment including T cells, myeloid cells and B cells were observed in the lymph node metastatic samples compared with primary tumours. Additionally, the heterogeneity of epithelial cells was also revealed, and several clusters with expression programs that were associated with the progression‐free survival of NPC patients were identified. Additionally, our data revealed the complex intercellular communications from primary to lymph node metastasis. The rewiring of CCL signalling which plays an important role in tumour metastasis was further identified. Altogether, we systematically characterized the ecosystem of NPC primary and lymph node metastasized tumours, which may shed light on the development of a therapeutic strategy to improve clinical outcomes of NPC patients with lymph node metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64615538dd674063486c12bae779de9b1cec7e9e" target='_blank'>
              Single‐cell sequencing analysis reveals the dynamic tumour ecosystems of primary and metastatic lymph nodes in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Dahua Xu, Nihui Zhang, Yutong Shen, Dehua Zheng, Zhizhou Xu, Peihu Li, Jiale Cai, Guanghui Tian, Qingchen Wei, Hong Wang, Hongyan Jiang, Meng Cao, Bo Wang, Kongning Li
          </td>
          <td>2024-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3911738bf6c8895906cc93b1b96ca83b9bc71e1a" target='_blank'>
              Two distinct epithelial-to-mesenchymal transition programs control invasion and inflammation in segregated tumor cell populations.
              </a>
            </td>
          <td>
            Khalil Kass Youssef, Nitin Narwade, Aida Arcas, Ángel Márquez-Galera, Raúl Jiménez-Castaño, Cristina López-Blau, Hassan Fazilaty, David García-Gutierrez, A. Cano, Joan Galcerán, G. Moreno-Bueno, J. López-Atalaya, M. Á. Nieto
          </td>
          <td>2024-10-16</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="
 Background: Pancreatic ductal adenocarcinoma (PDAC) is highly resistant to chemoradiation and immune therapies. There is limited biomarker to identify subsets of PDAC that responds to immunotherapy or targeted therapy. We have identified a subset of PDACs (approximately 25%) carrying COMPASS-like complex (CLC) gene mutations. Analysis of the clinicopathologic and molecular features of this PDAC subset revealed that these tumors are an aggressive group of PDACs with poor or squamous histologic differentiation, increased rates of TP53 mutations, and poor patient survival compared to control PDACs. Our preliminary data showed that PDACs carrying mutations in two members of the CLC genes (KDM6A and KMT2D) have immunosuppressive tumor microenvironment (TME), suggesting that CLC genes are candidate biomarkers for immunotherapy. However, a comprehensive analysis of the immune composition, immune checkpoint expression, and the relationship between tumor cells and immune cells in CLC gene mutated PDACs compared to controls has not been performed. In this study, we characterized the immune composition in CLC gene mutated PDACs compared to paired control PDACs using tyramide signal amplification multiplex fluorescent immunohistochemistry (mfIHC). Methods: Human PDAC tissue slides were stained with 2 mfIHC panels and cell types were defined by different markers: panel 1 (PanCK, CD163, PD-L1, CD3, CD8, FoxP3) and panel 2 (PanCK, CD163, CD3, CD8, TIGIT, TIM3). Random image sections were selected in each sample (5∼8 sections per sample) and divided into wild-type (WT) group (n=57) and mutant group (n=41) and analyzed using InForm Cell Analysis software. Tissue segmentation training was performed to separate stromal and epithelial elements. Cell percentage, cell density, and nearest neighbor distance of each cell type were calculated using R program. Results: The CLC gene mutation is associated with elevated epithelial cells (ECs) and PDL1+ ECs and increased infiltration of exhausted cytotoxic T cells (TIGIT+/TIM3+ CD8 T cells). CLC gene mutated PDACs are characterized by spatial proximity between exhausted T cells (TIGIT+) and epithelial cells/antigen presenting cells (APCs). Higher number of epithelial cells is associated with reduced infiltration of helper T cells in the mutant PDAC tissue. We also observed that high numbers of APC in proximity to CD4 T cells were positively correlated with the proximity of EC and CD8 cytotoxic T cells as well as the proximity of CD4 T cells and cytotoxic T cells in the WT group but not in the CLC gene mutant group. Conclusion: Our results provide insight into the effects of CLC gene mutations on shaping the immune microenvironment of PDAC and suggest that CLC gene mutated PDACs have an immunosuppressive immune microenvironment than other PDACs and may respond differently to immunotherapy.
 Citation Format: Shungang Zhang, Elaina Daniels, Jake McGue, Hong Sun Kim, Dafydd Thomas, Timothy Frankel, Jiaqi Shi. Multiplex fluorescent immunohistochemistry reveals immunosuppressive tumor microenvironment in COMPASS-like complex gene mutated pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2790a085665b4a36a3c8b84d6bede9ac8c16c70" target='_blank'>
              Abstract A040: Multiplex fluorescent immunohistochemistry reveals immunosuppressive tumor microenvironment in COMPASS-like complex gene mutated pancreatic cancer
              </a>
            </td>
          <td>
            Shungang Zhang, Elaina Daniels, Jake McGue, H. Kim, Dafydd Thomas, Timothy L. Frankel, Jiaqi Shi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/000ea5f5a04e65fe0427f079acb1a4f46ea53f2a" target='_blank'>
              Deciphering the topological landscape of glioma using a network theory framework
              </a>
            </td>
          <td>
            Mengchao Yao, Yang Su, Ruiqi Xiong, Xile Zhang, Xiaomei Zhu, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is characterized by a dense desmoplastic stroma, comprised of extracellular matrix, cancer-associated fibroblasts (CAFs), and immune cells. Due to the rapid tumor growth and the unusually low blood vessel density of PDAC, both tumor cells and stromal cells experience hypoxia, or the low availability of oxygen. Although hypoxia is a crucial feature of the tumor milieu and predicts poor clinical outcome, the effects of hypoxia on the PDAC stromal cells, as well as the interplay between tumor and stroma, is not well understood. Cancer-associated fibroblasts (CAFs) are a predominant and heterogeneous stromal cell type in PDAC, and single-cell transcriptomics of human and mouse PDAC revealed a distinct CAF subpopulation, inflammatory CAFs (iCAFs). These inflammatory fibroblasts produce high levels of cytokines and chemokines in PDAC, potentially contributing to its immunosuppressive microenvironment and tumorigenesis. By injecting a hypoxia probe into PDAC mouse models, we previously showed that iCAFs predominantly reside in oxygen- starved regions of the tumor. Furthermore, we observed that the hypoxia-related gene signature is positively enriched in iCAFs in human PDAC tissues. By exposing three-dimensional (3D) cocultures of pancreatic tumor cells and fibroblasts to hypoxia, we found that hypoxia induces IL1ɑ from tumor cells, thereby promoting iCAF formation. Importantly, tumor cell IL1ɑ was further elevated in the presence of hypoxic fibroblasts, implicating hypoxia as a modulator of bidirectional tumor cell-CAF interactions. Efforts are ongoing to understand the mechanism by which hypoxia regulates CAF phenotype by modulating the bidirectional crosstalk between tumor cells and fibroblasts in PDAC. Together, our study identifies hypoxia as a crucial environmental factor in promoting the iCAF phenotype, thus highlighting an instructive role of hypoxia in shaping the stromal microenvironment.
 Citation Format: Tenzin Ngodup, Ashley Mello, Katelyn Donahue. Role of hypoxia in fibroblast reprogramming in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A037.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da5c5297876e5216c83c3617a0c5a2cee1274f49" target='_blank'>
              Abstract A037: Role of hypoxia in fibroblast reprogramming in pancreatic cancer
              </a>
            </td>
          <td>
            Tenzin Ngodup, Ashley M. Mello, Katelyn L. Donahue
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcbb969fcc77899a650652d987f291cd240a2f64" target='_blank'>
              Single-nucleus RNA-seq dissection of choroid plexus tumor cell heterogeneity.
              </a>
            </td>
          <td>
            Anthony D Hill, Konstantin Okonechnikov, Marla K Herr, Christian Thomas, Supat Thongjuea, M. Hasselblatt, Annarita Patrizi
          </td>
          <td>2024-10-31</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8356cbc45a5a461f151c806739a40b73285b289b" target='_blank'>
              Enhancing spatial domain detection in spatial transcriptomics with EnSDD
              </a>
            </td>
          <td>
            Hui-Sheng Li, Yu-Ting Tan, Xiao-Fei Zhang
          </td>
          <td>2024-10-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is a deadly malignancy. Cancer associated fibroblasts (CAFs) play a complex role in the PDAC tumor microenvironment (TME), with a subset contributing significantly to tumor progression. Phosphatase and tensin homolog (PTEN) is a tumor suppressor and its loss is associated with increased tumor aggressiveness. Our group previously demonstrated PTEN loss in SMA+ positive CAFS correlated with worsened outcomes in patients and in mouse PDAC models. Our group previously demonstrated the loss of PTEN in CAFS led to activation of Stat3, resulting in an immunosuppressive microenvironment (Lefler et al, 2022 PMID: 35803738). However, little is known of the signals produced by tumor cells or other cell types in the TME that trigger loss of PTEN in CAFs. This study aims to to elucidate the molecular mechanisms underlying PTEN loss in CAFs in PDAC. To address this gap, we employed single cell RNA sequencing of tumors from a mouse PDAC model and identified a (PDRGFRa+Lys6+Stat3+Pten-low) CAF subpopulation. Comparing normal pancreas epithelial cells to tumor cells in this data set, we identified the IL6 family of cytokines, (IL6; Oncostatin-M, OSM; and Leukemia Inhibitory Factor, LIF) and Transforming Growth Factor beta 1 (TGFb1) as potential ligands that regulated PTEN, while myeloid cells were a source of the IL6 family member Oncostatin M (OSM). Treatment of isolated CAFs with these factors revealed no significant difference in PTEN mRNA expression. Thus, we developed an in vitro reporter system in which a PTEN-GFP fusion protein is expressed in CAFs. Treating these CAFs with IL6, OSM and TGFb1 (but not LIF) led to a significant decrease in GFP fluorescence. Analysis of endogenous PTEN by western blot confirmed these results. We posited that expression of PTEN E3 ligases might be increased by treatment with these factors, leading to destruction of PTEN. In support of this hypothesis, the expression of E3 ligases WWP1 and WWP2 was increased by treatment with IL6 and OSM. Additionally, preliminary data has revealed that PTEN is oxidized in isolated CAFs, which causes loss of PTEN activity and potentially increased protein turnover. Current experiments are aimed at testing these potential mechanisms. To conclude, PTEN is dysregulated post-transcriptionally in CAFs. The mechanism of PTEN dysregulation could be via increased expression of E3 ligases that target PTEN and/or PTEN oxidation. Future efforts will be aimed at testing whether inhibition of these pathways can restore PTEN expression in mouse PDAC models. These pathways and interactions could be further exploited for therapeutic benefit in curbing PDAC progression.
 Citation Format: Ivo N. Woogeng, Lu Han, Samaneh Saberi, Cameron Bumbleburg, Joseph Beaudet, Sudarshana Sharma, Michael Ostrowski. Dysregulation of PTEN expression in a subset of Cancer Associated Fibroblasts by Tumor Secreted Factors in PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A060.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aaeeaaaa210347b9fca3c389f98eb266dbc318b" target='_blank'>
              Abstract A060: Dysregulation of PTEN expression in a subset of Cancer Associated Fibroblasts by Tumor Secreted Factors in PDAC
              </a>
            </td>
          <td>
            Ivo N. Woogeng, Lu Han, S. Saberi, Cameron Bumbleburg, Joseph Beaudet, Sudarshana Sharma, Michael C. Ostrowski
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 The tumor immune microenvironment shapes breast tumor evolution. We previously described the ductal carcinoma in situ (DCIS) to invasive breast carcinoma (IBC) transition as an evolutionary bottleneck and a key step of immune escape. Here, we dissected the cellular composition and spatial topology of DCIS and IBC patient samples and rat tumors that faithfully recapitulate human breast cancer. We identified differences in T cells as the main distinguishing feature between DCIS and IBC and discovered a novel cycling regulatory T cell population (cycTreg) as the primary orchestrator of the immune suppressive environment during invasive progression, especially enriched in triple-negative histological subtype. We showed that cycTregs are locally expanding Treg precursors in response to increased antigen presentation, and further demonstrated that modulating them by OX40 agonist in combination with anti-PD-L1 enhances CD8+ T cell infiltration and suppresses tumor growth. Characterizing therapeutic responses at the single cell level revealed a Treg and cancer-associated fibroblast (CAF) crosstalk via IL33-ST2 pathway that is critical for Treg expansion. Multimodal analysis of the patient cohorts demonstrated that cycTregs predict recurrence in low-grade DCIS and poor patient outcomes in IBC, and that increased CAF-immune-DCIS proximity is prognostic of progression to IBC. In sum, our study establishes cycTregs as key drivers of immune escape during invasive progression and implies that their targeting is essential for enhancing the efficacy of immune checkpoint inhibitors in IBC.
 Citation Format: Triet Minh Bui, Ernesto A Rojas, Pierre Foidart, Zheqi Vaciry Li, Pengze Yan, Jun Nishida, Robert West, Shelley Hwang, Simona Cristea, Kornelia Polyak. Cycling T regulatory cells architect an immune escape hub during in situ to invasive breast carcinoma transition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303a814c313c222a2e5b96739715f771fd2858ed" target='_blank'>
              Abstract A028: Cycling T regulatory cells architect an immune escape hub during in situ to invasive breast carcinoma transition
              </a>
            </td>
          <td>
            Triet Bui, Ernesto Rojas, P. Foidart, Zheqi Li, Pengze Yan, Jun Nishida, Robert West, Shelley Hwang, Simona Cristea, K. Polyak
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="
 Intraductal papillary mucinous neoplasms (IPMNs), a common precursor to pancreatic ductal adenocarcinoma, present as a spectrum of benign indolent behavior and progression to invasive carcinoma, which highlights the importance of identifying biomarkers of IPMN progression. This study utilized whole transcriptome spatial transcriptomics to define the molecular characteristics of Epithelial, Immune, and Fibroblast cell populations of the three distinct histological sub-types of IPMN to understand the dynamics between the cell types and identify biomarkers of neoplastic progression. Fifty-two surgically resected IPMN cores from 21 patients with resected IPMNs comprising 24 Gastric, 14 Intestinal, and 14 Pancreaticobiliary with a distribution of low-grade dysplasia (LGD), high-grade dysplasia (HGD), and Invasive carcinoma (IC) were analyzed. Epithelial, Immune, and Fibroblast components were analyzed using the GeoMx spatial profiler. Unsupervised hierarchical clustering of epithelial areas of interest (AOI) identified 3 distinct clusters demonstrating a spectrum of molecular phenotypes in IPMN that were correlated with the three histological subtypes. Comparing the gene expression profile based on the shared genes expressed on all three subtypes, the Intestinal type tends to have an overexpression of genes whereas the Pancreaticobiliary represents a repressed phenotype relative to the Gastric phenotype. Supervised clustering using Moffitt’s classifier to identify the Epithelial-Mesenchymal transition demonstrates the Pancreaticobiliary subtype to be more Basal-like versus the Gastric and Intestinal inclined towards the classical phenotype. Supervised clustering based on the degree of dysplasia within the epithelial compartments of each of these subtypes, demonstrated a spectrum of molecular heterogeneity and identified biomarkers specific to High-Grade dysplasia in the Gastric and Intestinal subtype. Immune AOI deconvolution analysis demonstrated that Tregs were associated with Pancreaticobiliary, Macrophage enrichment associated with the Intestinal phenotype, while there is enrichment of CD8 T-cell and CD4 T-cell gene signatures associated with the Gastric subtype. Fibroblast AOI analyses demonstrated heterogenous myCAF- iCAF signatures between the subtypes. In summary, Spatial whole transcriptome analysis of IPMN identifies molecular signatures in epithelial and immune compartments that differentiate between the three histological subtypes of IPMN and helps understand their progression from Low-grade dysplasia to High-grade dysplasia.
 Citation Format: Amaya Pankaj, Michael J Raabe, Yuhui Song, Bidish K Patel, Katherine Xu, Joshua R Kocher, Nicholas J Caldwell, Maria L Ganci, Linda T Nieman, Mari Mino-Kenudson, Vikram Deshpande, Nabeel Bardeesy, Carlos Fernandez-Del Castillo, Martin J Aryee, David T Ting, Yasmin G Hernandez-Barco. Spatial Transcriptomics Unveils Intraductal Papillary Mucinous Neoplasm Heterogeneity: From Novel Clusters to Immune Dynamics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B091.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a43f4d0c8d98a15ce28e74661a5da83f45aa30c" target='_blank'>
              Abstract B091: Spatial Transcriptomics Unveils Intraductal Papillary Mucinous Neoplasm Heterogeneity: From Novel Clusters to Immune Dynamics
              </a>
            </td>
          <td>
            Amaya Pankaj, Michael J. Raabe, Yuhui Song, Bidish Kumar Patel, Katherine Xu, Joshua R. Kocher, Nicholas J Caldwell, M. L. Ganci, Linda T. Nieman, M. Mino-Kenudson, Vikram Deshpande, N. Bardeesy, C. F. Fernandez-Del Castillo, Martin J. Aryee, David T. Ting, Yasmin G Hernandez-Barco
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="
 Triple-negative breast cancer (TNBC) is the most aggressive subtype of breast cancer. TNBC displays exceptional transcriptional heterogeneity, both between patients and intratumorally, and disproportionately affects underserved women. Hypoxia, a state of low oxygen levels, is often present in the solid tumor microenvironment. In TNBC, hypoxia is a key driver of aggressiveness and therapeutic resistance. Past bulk RNA sequencing (RNAseq) studies in cancer research have lacked spatial resolution, failing to uncover precise transcriptional differences within tumor regions. This spatial limitation has been especially problematic for studying transcriptional heterogeneity of the cellular hypoxia response within tumors. By integrating single-cell RNA sequencing and spatial transcriptomic analyses in a TNBC xenograft tumor model, our research has uncovered a novel hypoxia gene signature, specific to cancer cells. Elevated expression of this hypoxia signature predicts poorer prognosis in breast cancer patients, especially within TNBC patients and those of African ancestry. This suggests that hypoxia may play a role in the racial disparities observed in TNBC prognosis, potentially informing treatment strategies. This workflow introduces a novel framework for translating general spatial transcriptomics data into clinically relevant and accurate gene signatures.
 Citation Format: Kent Schechter, Long C. Nguyen, Geetha P. Yerradoddi, Rania Bassiouni, John D. Carpten, Marsha R. Rosner. Novel cancer-specific hypoxia signature predicts poor prognosis and highlights racial disparities in breast cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C105.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43ba633337a4ee769f8bd92a6f7364c821869174" target='_blank'>
              Abstract C105: Novel cancer-specific hypoxia signature predicts poor prognosis and highlights racial disparities in breast cancer
              </a>
            </td>
          <td>
            Kent Schechter, Long C Nguyen, G. P. Yerradoddi, R. Bassiouni, J. Carpten, M. Rosner
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Breast cancer is a complex and heterogeneous disease with varying cellular, genetic, epigenetic, and molecular expressions. The detection of intratumor heterogeneity in breast cancer poses significant challenges due to its complex multifaceted characteristics, yet its identification is crucial for guiding effective treatment decisions and understanding the disease progression. Currently, there exists no method capable of capturing the full extent of breast tumor heterogeneity. In this study, the aim is to identify and characterize metabolic heterogeneity in breast tumors using immune cells and an ultrafast laser-fabricated Immuno Nano Sensor. Combining spectral markers from both Natural Killer (NK) and T cells, a machine-learning approach is implemented to distinguish cancer from healthy samples, identify primary versus metastatic tumors, and determine estrogen receptor (ER)/progesterone receptor (PR) status at the single-cell level. The platform successfully distinguished heterogeneous breast cancer samples from healthy individuals, achieving 97.8% sensitivity and 92.2% specificity, and accurately identified primary tumors from metastatic tumors. Characteristic spectral signatures allow for discrimination between ER/PR-positive and negative tumors with 97.5% sensitivity. This study demonstrates the potential of immune cell-based metabolic profiling in providing a comprehensive assessment of breast tumor heterogeneity and paving the way for minimally invasive liquid biopsy approaches in breast cancer diagnosis and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e06ed42248f4cc295c91e854d17b922ee95ca30" target='_blank'>
              Profiling Breast Tumor Heterogeneity and Identifying Breast Cancer Subtypes Through Tumor-Associated Immune Cell Signatures and Immuno Nano Sensors.
              </a>
            </td>
          <td>
            D. Ishwar, Srilakshmi Premachandran, Sunit Das, K. Venkatakrishnan, B. Tan
          </td>
          <td>2024-10-26</td>
          <td>Small</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf64c81b2e5cd6a446e0a9c3f91b443d7d5cf63c" target='_blank'>
              Supercontinuum-tailoring multicolor imaging reveals spatiotemporal dynamics of heterogeneous tumor evolution
              </a>
            </td>
          <td>
            Xiujuan Gao, Xinyuan Huang, Zhongyun Chen, Liu Yang, Yifu Zhou, Zhenxuan Hou, Jie Yang, Shuhong Qi, Zheng Liu, Zhihong Zhang, Qian Liu, Qingming Luo, Ling Fu
          </td>
          <td>2024-10-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is among the most aggressive brain cancers, and it contains glioma stem cells (GSCs) that drive tumor initiation, progression, and recurrence. These cells resist conventional therapies, contributing to high recurrence rates in GBM patients. Developing in vitro models that mimic the tumor microenvironment (TME), particularly the GSC niche, is crucial for understanding GBM growth and therapeutic resistance. Three-dimensional (3D) spheroid models provide a more physiologically relevant approach than traditional two-dimensional (2D) cultures, recapitulating key tumor features like hypoxia, cell heterogeneity, and drug resistance. This review examines scaffold-free and scaffold-based methods for generating 3D GBM spheroids, focusing on their applications in studying the cancer stem cell niche. The discussion encompasses methods such as the hanging drop, low-adhesion plates, and magnetic levitation, alongside advancements in embedding spheroids within extracellular matrix-based hydrogels and employing 3D bioprinting to fabricate more intricate tumor models. These 3D culture systems offer substantial potential for enhancing our understanding of GBM biology and devising more effective targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea7b7a45a32163a4dbbbdd90c32adea5a5aa6ad9" target='_blank'>
              Recapitulating Glioma Stem Cell Niches Using 3D Spheroid Models for Glioblastoma Research
              </a>
            </td>
          <td>
            Hyunji Jo, Seulgi Lee, Min-Hyeok Kim, Sungsu Park, 
          </td>
          <td>2024-11-07</td>
          <td>Biosensors</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b71a024e8ea11ad0f8bde61ad5aaa4bc4e17f0cb" target='_blank'>
              A method for in silico exploration of potential glioblastoma multiforme attractors using single-cell RNA sequencing
              </a>
            </td>
          <td>
            Marcos Guilherme Vieira Junior, Adriano Maurício de Almeida Côrtes, Flávia Raquel Gonçalves Carneiro, Nicolas Carels, Fabrício Alves Barbosa da Silva
          </td>
          <td>2024-10-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6a7400a55e661d30ea0a596e56eadc3b19d76c4" target='_blank'>
              Single-cell RNA-sequencing reveals a unique landscape of the tumor microenvironment in obesity-associated breast cancer
              </a>
            </td>
          <td>
            Guanghui Zhao, Xiaodong Zhang, Liying Meng, Ke Dong, Shipeng Shang, Tengfei Jiang, Ziqian Liu, Haidong Gao
          </td>
          <td>2024-09-16</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The impact of aging on the immune landscape of luminal breast cancer (Lum-BC) is poorly characterized. Understanding the age-related dynamics of immune editing in Lum-BC is anticipated to improve the therapeutic benefit of immunotherapy in older patients. To this end, here we applied the 'multiple iterative labeling by antibody neo-deposition' (MILAN) technique, a spatially resolved single-cell multiplex immunohistochemistry method. We created tissue microarrays by sampling both the tumor center and invasive front of luminal breast tumors collected from a cohort of treatment-naïve patients enrolled in the prospective monocentric IMAGE (IMmune system and AGEing) study. Patients were subdivided into three nonoverlapping age categories (35-45 = 'young', n = 12; 55-65 = 'middle', n = 15; ≥70 = 'old', n = 26). Additionally, depending on localization and amount of cytotoxic T lymphocytes, the tumor immune types 'desert' (n = 22), 'excluded' (n = 19), and 'inflamed' (n = 12) were identified. For the MILAN technique we used 58 markers comprising phenotypic and functional markers allowing in-depth characterization of T and B lymphocytes (T&B-lym). These were compared between age groups and tumor immune types using Wilcoxon's test and Pearson's correlation. Cytometric analysis revealed a decline of the immune cell compartment with aging. T&B-lym were numerically less abundant in tumors from middle-aged and old compared to young patients, regardless of the geographical tumor zone. Likewise, desert-type tumors showed the smallest immune-cell compartment and were not represented in the group of young patients. Analysis of immune checkpoint molecules revealed a heterogeneous geographical pattern of expression, indicating higher numbers of PD-L1 and OX40-positive T&B-lym in young compared to old patients. Despite the numerical decline of immune infiltration, old patients retained higher expression levels of OX40 in T helper cells located near cancer cells, compared to middle-aged and young patients. Aging is associated with important numerical and functional changes of the immune landscape in Lum-BC. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769ef243851690165ce0b362f0b78f63d7b88eb4" target='_blank'>
              Dissecting the immune infiltrate of primary luminal B-like breast carcinomas in relation to age.
              </a>
            </td>
          <td>
            S. Hatse, Y. Lambrechts, Asier Antoranz Martinez, M. De Schepper, T. Geukens, H. Vos, L. Berben, J. Messiaen, Lukas Marcelis, Yannick Van Herck, P. Neven, A. Smeets, C. Desmedt, F. De Smet, Francesca M Bosisio, H. Wildiers, G. Floris
          </td>
          <td>2024-09-29</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13caafe064cf39a767e5adb9dbd05fb72dc6368e" target='_blank'>
              Pre-metastatic niche: formation, characteristics and therapeutic implication
              </a>
            </td>
          <td>
            Yuhang Wang, Jiachi Jia, Fuqi Wang, Yingshuai Fang, Yabing Yang, Quanbo Zhou, Weitang Yuan, X. Gu, Junhong Hu, Shuaixi Yang
          </td>
          <td>2024-09-25</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The purpose of this study was to evaluate the suitability of paraffin-embedded (FFPE) samples and fixed fresh (FF) samples for single-cell RNA sequencing (scRNAseq). To this end, we compared whether single-cell profiles from FFPE and matched FF tissue samples from one invasive carcinoma of no especial type carcinoma (invasive ductal carcinoma –IDC) and one invasive lobular carcinoma (ILC) were similar when captured using the common immunohistochemical (IHC) and immune features of the tumors. FF and FFPE tissue derived libraries showed high-quality parameters. The cell heterogeneity obtained was similar, although, there were some differences in the number of cells captured, but there were no specific cell populations captured exclusively by one of the two different approaches. The combined analysis of the four samples identified six types of epithelial cells (normal basal cells, four subpopulations of neoplastic cells in the IDC and one in the ILC), and different tumoral microenvironment populations. scRNAseq results from epithelial neoplastic cells were concordant with common IHC markers. In addition, after quantitative assessment of the whole slide images using QuPath, the proportion of immune cells identified by IHQ in FFPE sections were similar to those obtained by scRNAseq. Confirming the suitability of scRNAseq using FFPE tissue, this technique identified one subpopulation of neoplastic cells present only in the IDC, characterized by the expression of genes related with multi-ciliated cell (MCC) differentiation (FOXJ1, ROPN1L, TPPP3, CFAP45). IHC analysis of FOXJ1 expression confirmed the presence of positive cells only in the IDC in a similar proportion to that obtained by scRNAseq. Analysis of FOXJ1 in a cohort of 214 ER-positive invasive carcinomas demonstrated protein expression in at least 1% of cells in 33% of tumors, suggesting frequent focal MCC differentiation. Our results support the suitability of scRNAseq analysis using FFPE tissue and identified a subpopulation of neoplastic MCC in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2bd0963b0249a16a0871656c5b25b2c0524782" target='_blank'>
              Single-cell RNA sequencing on formalin-fixed and paraffin-embedded (FFPE) tissue identified multi-ciliary cells in breast cancer
              </a>
            </td>
          <td>
            Silvia González-Martínez, Irene Carretero-Barrio, Belén Pérez-Miés, Val Fernández-Lanza, Mónica García-Cosío, T. Caniego-Casas, David Hardisson, Isabel Esteban-Rodríguez, J. Cortés, José Palacios
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Integrin family transmembrane receptors mediate dynamic interactions between cells and their extracellular microenvironment. The heterogeneous interaction partners of integrins directly regulate cell adhesion, motility, proliferation, and intracellular signaling. Despite the recognized importance of protein-protein interactions and the formation of signaling hubs around integrins, the ability to detect and quantify these dynamic binding partners with high spatial and temporal resolution remains challenging. Here, we developed an integrin-family-directed quantitative photoproximity protein interaction (PhotoPPI) profiling method to detect and quantify native integrin-centered protein social networks on live cells and tissues without the need for genetic manipulation, antibodies, or non-physiologic cell culture conditions. We drafted quantitative maps of integrin-centered protein social networks, highlighting conserved and unique binding partners between different cell types and cellular microenvironments. Comparison of integrin social networks in cancer cell lines of diverse tissue of origin and disease state identified specific AND-gate binding partners involved cell migration, microenvironmental interactions and proliferation that serve as markers of tumor cell metastatic state. Finally, we identified unique combinations – or barcodes - of integrin-proximal proteins on the surface of pre- and post-metastatic triple negative breast cancer (TNBC) cells whose expression strongly correlate with both positive and negative disease progression and outcomes in TNBC patients. Taken together, these data provide the first family-wide high-resolution maps of native protein interactors on live cells and identify dynamic integrin-centered social networks as potential AND-gate markers of cell identity, microenvironmental context and disease state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5900ee56d3da605dca7c4bf45b4b9da559c49721" target='_blank'>
              Family-Wide Photoproximity Profiling of Integrin Protein Social Networks in Cancer
              </a>
            </td>
          <td>
            Anthony J. Carlos, Dongbo Yang, Deborah M. Thomas, Shuyuan Huang, Keira I. Harter, R. Moellering
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 In the United States and globally, prostate cancer (PCa) mortality rates are the highest among men of African descent. 24% of the disparities in PCa remains even when controlled for access to care and stage at presentation. Moreover, the tumor microenvironment plays an essential role in tumor progression, aggressiveness, therapeutic response, and patient outcomes. Additionally, the association of the genomic findings with patient ancestry and other characteristics, such as tumor biology and transcriptomic alterations, remains poorly understood. Here, we performed a multi-Omics approach (N=447) to unravel the complexity of tumor heterogeneity and understand disease progression & distinct tumor biology influenced by genetic ancestry. We performed Whole Exome (Normal/tumor paired) Sequencing matched with Methyl Seq and Whole transcriptomic Sequencing for three datasets of our African, African American Men (AAM), and European Men (EAM). Additionally, we ran Spatial NanoString high-plex GeoMx-DSP for a total of 118 treatment-naive PCa patients (around 500 ROIs). Simultaneously, we added matched whole transcriptome Sequencing for these patients. The cohort comprised 87 AAM, 3 unknown, and 28 EAM self-reported individuals. To verify the self-reported race, the genomic ancestry was qualified using genotype and Admixture analysis. To further validate differentially expressed genes at the protein level, we performed multi-plex histological staining of 40 markers to determine the spatial resolution and neighborhood clustering within the tumor and the microenvironment. In parallel, we performed scMultiOmics sequencing (scRNA & scATAC-Seq) at a single-cell resolution (6000 cells/sample at 25K reads per cell) from an additional 12 patients (9 AAM & 3 EAM). We reconstructed a 3D model of the distinct tumor microenvironment at subcellular resolution using serially sectioned hematoxylin and eosin- stained tissue sections (CODA technology). Our results demonstrate that patients who self-report as AAM or Nigerian are assigned to high African (> 70%) Ancestry with either Yoruba (Nigeria) and/or Bantu subpopulation in the Sub-Saharan area. Additionally, high African Ancestry patients are diagnosed at a younger age and show advanced pathology stages compared to patients with European Ancestry. AAM of Yoruba descent expresses significantly higher immune-inflammatory signatures (STAT/IFNG-signaling pathway) compared to the Nigerian-Yoruba subpopulation. Our scRNA-Seq analysis shows that AAM has suppressive myeloid cells infiltrate within the tumor cells. The infiltrations of these cells change with age, Gleason Grade, and pathology stage. Moreover, AAM-tumor cells (scATAC-Seq) have increased open Chromatin accessibility with STAT motifs enrichment (p-value;0.0001), which could be the driver regulators of the immune-suppressive signature as well as influence the upregulation of STAT/IFNG signature within African Ancestry patients. Our study provides new insight into how genetic ancestry impacts immune signatures in AAM/African and contributes to PCa racial disparities.
 Citation Format: Isra Elhussin, Ezra Baraban1, Ashley Kiemen, Tamara L Lotan, Cathy Handy Marshall, Emmanuel Antonarakis, Moray J Campbell, Melissa Davis, Michael Dixon, Isaac Kim, Stefan Ambs, Rick Kittles, Adam B Murphy, Clayton Yates. Integrative multi-omics profiling in patients of African descent diagnosed with prostate cancer reveals distinct tumor-promoting immunosuppressive niches at a single-cell level and spatial resolution [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C117.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4100a5d71a839b36fc483aaee40ec75da9220882" target='_blank'>
              Abstract C117: Integrative multi-omics profiling in patients of African descent diagnosed with prostate cancer reveals distinct tumor-promoting immunosuppressive niches at a single-cell level and spatial resolution
              </a>
            </td>
          <td>
            Isra Elhussin, Ezra G. Baraban, Ashley Kiemen, T. Lotan, Cathy Handy Marshall, E. Antonarakis, Moray J. Campbell, Melissa B. Davis, Michael Dixon, Isaac Kim, S. Ambs, Rick A. Kittles, Adam B. Murphy, Clayton C. Yates
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Background Pancreatic ductal adenocarcinoma is a devastating disease lacking targeted therapy. Basal-like pancreatic cancer is a tumor subtype with a worse clinical outcome than other subtypes. These cancers are characterized by the expression of basal keratins and mesenchymal markers. However, the genes that drive the basal-like phenotype are poorly understood. Ras homolog family member V (RhoV) is one of the Rho GTPases, and it has been demonstrated to drive the progression of lung and breast cancers. The current study aims to reveal the role of RhoV in basal-like pancreatic cancer. Design RhoV, p63, and CK5/6 were stained in sequential sections by immunohistochemistry. Expression of RhoV in pancreatic cancer and cell lines was reanalyzed based on the data from ProteinAtlas and Broad Insitute DepMap Portal. Basal cell line, BxPC3, and non-basal cell line, Panc1 were selected for functional evaluation based on their differentiated RhoV expression. RhoV gene was overexpressed and knocked down using the CRISPR/gRNA-based Lentiviral system, followed by sphere formation, colony formation, migration, and invasion assays. Results RhoV, p63, and CK5/6 were expressed in the same histologic location of basal pancreatic cancer. The gene expression level of RhoV was co-expressed with p63 (a classic basal marker) in four pancreatic cancer cell lines (broadinstitute.org). Knockout of RhoV by three runs of gRNAs in BxPC3 cells all significantly attenuated tumor sphere formation with less colony formation as compared to mock knockout. Overexpression of RhoV in non-basal-like cells, Panc 1, significantly increased the size of sphere formation, numbers of colony formation, and p38 activation compared to EV control. Cell migration and invasion assays showed the same RhoV-dependent migration and invasion in corresponding cells. Panc1 cells with RhoV knockout significantly generated smaller tumor xenografts in immunodeficient mice and RhoV overexpression led to larger tumor sizes. Conclusion Finding the driver of the basal-like phenotype helps to reveal the pathophysiology of the basal-like subtype of pancreatic cancer and facilitates the identification of new targets for therapeutic purposes. Our preliminary results indicated that RhoV, a small G protein from the Rho family of GTPases, played essential roles in tumor proliferation and progression, by activating the p38 signaling pathway. Further study will be focused on revealing Rhov's downstream pathways and exploring the role of RhoV in apoptosis/cell cycle, cell metabolism, autophagy, etc. in basel-like pancreatic cancer.
 Citation Format: Shang Wu, Jia Feng, Selina Gao, Ting Wang, Zebang Li, Yifei Liu, Chiung-Kuei Huang, Shaolei Lu. The role of RhoV in basal-like pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffc91f4069ed1467bf0a9573f938915fbe029125" target='_blank'>
              Abstract A077: The role of RhoV in basal-like pancreatic cancer
              </a>
            </td>
          <td>
            Shang Wu, Jia Feng, Selina Gao, Ting Wang, Z. Li, Yifei Liu, Chiung-Kuei Huang, Shaolei Lu
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Brain tumors exhibit distinct DNA methylation patterns that reflect their cellular origin, molecular subtype, and clinical behavior. DNA methylation signatures provide valuable insights into tumor heterogeneity, evolution, and treatment response, guiding personalized therapeutic strategies. Our case, glioblastoma, atypical mesenchymal type confirmed by DNA methylation profile and it reflects the clonal evolution of the glioblastoma metaplasia with minimal glial component which was histologically challenging.



 A 56-year-old gentleman presented with headaches. MRI brain demonstrated a heterogeneous enhancing mass with areas of central necrosis in the left temporal lobe, measuring approximately 49 x 34 mm, with surrounding T2 FLAIR hyperintensity extending into the left basal ganglia, causing compression of the left lateral ventricle and uncal herniation. He underwent left-sided craniotomy for tumor resection. Microscopic examination revealed hypercellular spindle cell neoplasm, forming fascicles. Brisk mitotic activity and geographic necrosis were evident. GFAP highlighted narrow margins of the tumor which could represent the entrapped glial tissue. CD99 stained the tumor cells, Other immunohistochemical stains for spindle cell work up, like S100, EMA, PR STAT6 and desmin were all negative. A diagnosis of intracranial mesenchymal tumor /sarcoma was considered.
 Methylation profiling results showed class calibrated scores of 0.9741, compatible with Glioblastoma, IDH-wildtype, atypical mesenchymal type with a cytogenetic profile including gain of chromosome 7 (including EGFR) and loss of chromosome 10 (including PTEN), and loss of CDKN2A/B.



 NA



 Although histopathological examination and immunohistochemical study are fundamental in diagnosing brain tumors, the rapid development of cytogenetic and molecular diagnostic modalities has significantly contributed to diagnostic accuracy and the potential for targeted treatment. Particularly, DNA methylation profiling has proven valuable in diagnosing morphologically challenging cases with prognostic value. Eventually, DNA methylation profiling will be adopted as a major cancer diagnostic criterion and will become a standard procedure performed at major academic centers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2df16d4fcd08f0a0e5c276568c97d9016f47f98f" target='_blank'>
              DNA Methylation Profiling Confirms a Challenging Glioblastoma, Atypical Mesenchymal Type Which Mimics Intracranial Mesenchymal Tumor
              </a>
            </td>
          <td>
            Z. Aljumaili, N. Ibrahim, Y. Gao, D. Rivera, M. Bhattacharjee, H. Takei, X. Zhang
          </td>
          <td>2024-10-01</td>
          <td>American Journal of Clinical Pathology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc4ff74f564f7b2a57419b04c8a2f80af41275c3" target='_blank'>
              Multiregional transcriptomic profiling provides improved prognostic insight in localized non-small cell lung cancer
              </a>
            </td>
          <td>
            Chenyang Li, Thinh T. Nguyen, Jian-Rong Li, Xingzhi Song, J. Fujimoto, L. Little, Curtis Gumb, C. Chow, Ignacio I. Wistuba, Andrew P Futreal, Jianhua Zhang, Shawna M. Hubert, J. Heymach, Jia Wu, Christopher I Amos, Jianjun Zhang, Chao Cheng
          </td>
          <td>2024-10-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa2963e25bac9e3ab08f5a5b7df7ab88d90b9cc" target='_blank'>
              Multiomic single cell sequencing identifies stemlike nature of mixed phenotype acute leukemia
              </a>
            </td>
          <td>
            C. Peretz, Vanessa E. Kennedy, Anushka Walia, C. Delley, Andrew Koh, Elaine Tran, Iain C Clark, Corey E. Hayford, Chris D’Amato, Yi Xue, Kristina Fontanez, Aaron A. May-Zhang, Trinity Smithers, Y. Agam, Qian Wang, Haixin Dai, Ritu Roy, Aaron C Logan, Alexander E Perl, A. Abate, Adam Olshen, Catherine C Smith
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7199657c55f0905635cd26958d5762b121b0a3f2" target='_blank'>
              Integrated multi-omics assessment of lineage plasticity in a prostate cancer patient with brain and dural metastases
              </a>
            </td>
          <td>
            Megan L Ludwig, David Moline, Alec Horrmann, Ella Boytim, Gabrianne Larson, Ali T Arafa, Masooma Sayeda, John R Lozada, Hannah E Bergom, Abderrahman Day, Sandhyarani Dasaraju, S. Dehm, Paari J. Murugan, Justin H. Hwang, J. Drake, E. S. Antonarakis
          </td>
          <td>2024-09-30</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) originates from precursor lesions, the most common of which are known as pancreatic intraepithelial neoplasia (PanIN). PanIN formation and progression is accompanied by accumulation of a precursor microenvironment (PME) composed of desmoplastic stroma that plays an important role in promoting tumor growth and immune evasion. The crosstalk between tumor cells and stromal cells is an area of active investigation. While PanIN and the PME have been extensively studied in mouse models, their nature in humans have remained elusive as there is no clinical indication to sample healthy pancreas. Through a unique collaboration with the Gift of Life Organ and Tissue donation program, we performed transcriptomic analysis on normal pancreata declined for transplant. We noted that epithelial cells in the PanIN and cancer microenvironment were transcriptomically similar; however, their surrounding stromal fibroblasts were markedly distinct. Given that most precursor lesions remain quiescent while some progress to tumor, we sought out to elucidate how the transcriptional programs in epithelial cells influence fibroblast cell fate. To do so, we developed a 3D ex-vivo organoid-fibroblast coculture system using primary organoid and fibroblast cultures successfully derived from Gift of Life pancreata or PDAC patients undergoing surgical resection. Our coculture system retains histological features of heterogeneity observed in vivo. Using single cell RNA-sequencing analysis of matched fibroblast-organoid cocultures we identified factors that were highly expressed in fibroblasts in tumor cocultures, but not in normal cocultures. We are currently interrogating candidate ligand/receptor interactions in our cocultures to determine the mediators of this epithelial-mediated fibroblast polarization. This model will allow us to gain an understanding about how components of the microenvironment restrain or promote malignant transformation and serve as a great tool to study therapies targeting tumor and stromal cells in a patient specific manner.
 Citation Format: Padma Kadiyala, Ahmed Elhossiny, Jianhua Liu, Alexander Bray, Katelyn Donahue, Arvind Rao, Jiaqi Shi, Yaqing Zhang, Filip Bednar, Timothy Frankel, Eileen Carpenter, Marina Pasca Di Magliano. Investigating Fibroblast-Epithelial crosstalk in pre-malignant and cancerous microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A057.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9841941d0df90af0a793863b27fe1e8b8a9497" target='_blank'>
              Abstract A057: Investigating Fibroblast-Epithelial crosstalk in pre-malignant and cancerous microenvironment
              </a>
            </td>
          <td>
            Padma Kadiyala, Ahmed M. Elhossiny, Jianhua Liu, Alexander W Bray, Katelyn L. Donahue, Arvind Rao, Jiaqi Shi, Yaqing Zhang, Filip Bednar, Timothy L. Frankel, Eileen S Carpenter, M. P. di Magliano
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/281649fc3a01628d835005fce4ae40aa70c5e3f2" target='_blank'>
              Oncometabolites at the crossroads of genetic, epigenetic and ecological alterations in cancer.
              </a>
            </td>
          <td>
            Letizia Lanzetti
          </td>
          <td>2024-10-23</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758b3e0bcca0b232b4864c10d9c2f0433098ac1f" target='_blank'>
              NF1 expression profiling in IDH-wildtype glioblastoma: genomic associations and survival outcomes
              </a>
            </td>
          <td>
            Michael Chang, Mohamed Sherief, Maria Ioannou, Viveka Chinnasamy, Lucy Chen, Michael Frost, Michelle Mattson-Hoss, Herb Sarnoff, D. Kamson, Matthias Holdhoff, Debraj Mukherjee, C. Bettegowda, Jordina Rincon-Torroella, Victoria Croog, Peng Huang, Fausto J Rodriguez, Calixto-Hope G Lucas, K. Schreck
          </td>
          <td>2024-10-29</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Intestinal fibrosis is a severe complication of Crohn’s disease, often requiring surgical intervention. Despite extensive research efforts, an effective treatment to prevent or reverse intestinal fibrosis remains elusive. In this issue of the JCI, Zhang, Wang, and colleagues employed single-cell RNA sequencing to uncover mechanisms of the fibrotic process. They identified a key fibroblast subset of TWIST1+FAP+ cells that interacts with CXCL9+ macrophages. TWIST1 emerged as a central regulator of the fibrotic microenvironment, representing a promising therapeutic target for effectively treating intestinal fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b4be1b024f7ef8eaf670f9817e0da809757562d" target='_blank'>
              UnTWISTing intestinal fibrosis: single-cell transcriptomics deciphers fibroblast heterogeneity, uncovers molecular pathways, and identifies therapeutic targets
              </a>
            </td>
          <td>
            Giovanni Santacroce, A. Di Sabatino
          </td>
          <td>2024-09-17</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Cellular diversity is a product of evolution acting to drive divergent regulatory programs from a common genome. Here, we use cross-cell-type epigenetic conservation to gain insight into the impact of selective constraints on genome function and phenotypic variation. By comparing chromatin accessibility across hundreds of diverse cell-types, we identify 1.4% of the human genome safeguarded by conserved domains of facultative heterochromatin, which we term regions under “cellular constraint”. We calculate single-base resolution cellular constraint scores and demonstrate robust prediction of functionally important coding and non-coding loci in a cell-type-, trait-, and disease-agnostic manner. Cellular constraint annotation enhances causal variant identification, drug discovery, and clinical diagnostic predictions. Furthermore, cell-constrained sequences share paradoxical evolutionary signals of positive and negative selection, suggesting a dynamic role in driving human adaptation. Overall, this study demonstrates that evolutionary chromatin dynamics can be leveraged to inform the translation of genetic discoveries into effective biological, therapeutic, and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0bc98afde8c329e9647a592102463c5d074b6c9" target='_blank'>
              Epigenetic constraint of cellular genomes evolutionarily links genetic variation to function
              </a>
            </td>
          <td>
            Enakshi Sinniah, Dalia Mizikovsky, Woo Jun Shim, Chris Siu Yeung Chow, Yassine Souilmi, Fei-Fei Cheng, Zhili Zeng, Jordan Laurie, Matthew Foster, Sonia Shah, Mikael Bodén, Jian Zeng, B. Llamas, N. Palpant
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4422b0759e87ed388f1ba9e5bcbfed55cb314237" target='_blank'>
              Spatial transcriptomics reveals strong association between SFRP4 and extracellular matrix remodeling in prostate cancer
              </a>
            </td>
          <td>
            M. K. Andersen, S. Krossa, Elise Midtbust, Christine A. Pedersen, M. Wess, T. S. Høiem, T. Viset, Øystein Størkersen, Ingunn Nervik, E. Sandsmark, H. Bertilsson, G. Giskeødegård, Morten B. Rye, May-Britt Tessem
          </td>
          <td>2024-11-08</td>
          <td>Communications Biology</td>
          <td>3</td>
          <td>27</td>
        </tr>

        <tr id="Purpose Bordering the central nervous system (CNS) parenchyma are the choroid (underlying the retina) and the leptomeninges (the inner layers of the meninges enveloping the brain). Although near the neural parenchyma, the choroid and leptomeninges are external to the immune privileged environment of the retina and brain and thus are distinct immune compartments. This study aimed to characterize the transcriptomic signatures of immune cells within the choroid and leptomeninges bordering the healthy adult mouse CNS. Methods Eyes and brains were obtained from 7-week-old C57Bl/6J mice. Choroid and leptomeninges were processed for isolation of CD45+ immune cells and single cell RNA-sequencing. Additionally, single cell RNA-sequencing was performed on immune cells isolated from choroid obtained from human donor eye tissue. Immunostaining and confocal microscopy of wholemount tissue were used to validate selected immune cell populations in situ. Results A total of 3606 cells were sequenced from mouse tissues, including 2125 CD45+ cells from choroid and 1481 CD45+ cells from leptomeninges. Clustering and differential gene expression analysis revealed heterogeneous subtypes of monocytes/macrophages, dendritic cells, T cells, and B cells. Whereas some clusters were common to both choroid and leptomeninges, others exhibited tissue-specific gene expression profiles and potential functional specializations. Analysis of 6501 CD45+ cells sequenced from human choroid identified similar immune cell populations to mouse choroid. Conclusions This study provides a detailed characterization of the molecular signatures of immune cells within the vascular connective tissues bordering the healthy retina and brain, and their potential roles in immune protection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ed2b98f968077381be62d762f11a12216fb13ee" target='_blank'>
              Tissue-Specific Immune Transcriptional Signatures in the Bordering Tissues of the Mouse Retina and Brain
              </a>
            </td>
          <td>
            Fazeleh Etebar, Paul Whatmore, Damien G. Harkin, Samantha J. Dando
          </td>
          <td>2024-10-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Patients with pancreatic ductal adenocarcinoma (PDAC) have a poor prognosis due to late presentation of disease and frequent resistance to therapy. Immunotherapy regimens have uniformly failed in PDAC, which has long been considered an immunologically cold tumor. It is believed that immunotherapy has been unsuccessful in PDAC due to its dense, fibrotic, and immunosuppressive microenvironment. Cancer-associated fibrosis may account for up to 90% of the PDAC microenvironment, thus further study of this immunosuppressive component is urgently needed. Methods Human PDAC surgical specimens were procured, sectioned, and allocated for histology, flow cytometry, tumor-infiltrating lymphocyte (TIL) expansion, and fibroblast culture. For histology, tumor slices were saved in 10% formalin and wax embedded. Histological PDAC slides were stained against fibroblasts markers alpha smooth muscle actin, fibroblast activation protein; and T-cell markers CD3, CD69, CD25, FOXP3. First, tumor sections were digested using mechanical and chemical techniques. For flow cytometry, digested tumor cells were stained with fluorescent conjugated antibodies targeting CD45, CD3, CD4, CD8, CD25, CD69, PD-1 and ICAM. For TIL isolation and expansion, resultant single-cells were bead-isolated for CD3 positive cells, cultured and expanded with 2000 IU interleukin-2 and 1.5 mg/mL anti-CD3/CD28 antibodies. For fibroblast culture, fibroblasts were bead isolated and sub-cultured in RPMI supplemented with FGF. TILs were co-cultured with patient-derived fibroblasts/fibroblast media with the PD-L1 inhibitor Atezolizumab. TILs were then analyzed by flowcytometry, and media from co-culture was analyzed via ELISA for gamma interferon release. Results TILs isolated from PDAC (N=5) co-cultured with patient-matched fibroblasts or media were phenotypically compared using flow cytometry and measure for gamma interferon release. Interestingly, TILs co-cultured with fibroblasts or fibroblast media showed improved expansion over peripheral blood derived control T-cells, but both groups showed highly impaired gamma interferon release. Co-cultured TILs had high levels of PD-1 and low levels of CD69, compared to control TILs, indicating a suppressed phenotype. TILs co-cultured with fibroblasts and Atezolizumab resulted in significantly increased CD69 expression and gamma interferon release (P>0.05), indicating an activated phenotype. Histological PDAC slides showed abundant T-cell infiltrates with minimal CD69 expression. CD25+ T-cells were identified throughout fibrotic areas of the TME. Conclusion Here, we challenge the long-held belief that PDAC is immunologically cold as our tumor cross-sections (N=5) were rich in T-cells. More importantly, our results show that tumor-derived fibroblasts appear to play a role in suppression of TIL function, which is partially reversible with immunomodulatory techniques (e.g., anti-PDL1therapy). Our results warrant further investigation into targeting CAFs in PDAC microenvironment to improve patient response to immunotherapy.
 Citation Format: Charles Bailey, Hannah Mcdonald, Anna Reagan, Michael Cavnar, Mautin Barry-Hundeyin, Prakash Pandalai, Joseph Kim. Pancreatic ductal adenocarcinoma derived cancer-associated fibroblasts suppress tumor infiltrating lymphocytes in the tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A056.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afa1491a3d04f2ef3985518fd8800e598091bc54" target='_blank'>
              Abstract A056: Pancreatic ductal adenocarcinoma derived cancer-associated fibroblasts suppress tumor infiltrating lymphocytes in the tumor microenvironment
              </a>
            </td>
          <td>
            Charles J. Bailey, Hannah G. McDonald, Anna Reagan, M. Cavnar, Mautin Barry-Hundeyin, Prakash Pandalai, Joseph Kim
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer metastasis, the process by which tumour cells migrate and colonize distant organs from a primary site, is responsible for the majority of cancer-related deaths. Understanding the cellular and molecular mechanisms underlying this complex process is essential for developing effective metastasis prevention and therapy strategies. To this end, we systematically analysed 1786 bulk tissue samples from 13 cancer types, 988 463 single cells from 17 cancer types, and 40 252 spots from 45 spatial slides across 10 cancer types. The results of these analyses are compiled in the metsDB database, accessible at https://relab.xidian.edu.cn/metsDB/. This database provides insights into alterations in cell constitutions, cell relationships, biological pathways, molecular biomarkers, and drug responses during cancer metastasis at bulk, single-cell, and spatial levels. Users can perform cell or gene searches to obtain multi-view and multi-scale metastasis-related data. This comprehensive resource is invaluable for understanding the metastasis process and for designing molecular therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d18e231fd68285bc1eb899acb7cd1bd947418508" target='_blank'>
              metsDB: a knowledgebase of cancer metastasis at bulk, single-cell and spatial levels.
              </a>
            </td>
          <td>
            Sijia Wu, Jiajin Zhang, Yanfei Wang, Xinyu Qin, Zhaocan Zhang, Zhennan Lu, Pora Kim, Xiaobo Zhou, Liyu Huang
          </td>
          <td>2024-10-21</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Patient-derived organoids (PDOs) have emerged as a powerful tool for modeling human cancers due to their ability to closely mirror in vivo architecture and preserve the genetic landscape of tumors. They offer a crucial platform for studying disease progression and treatment responses. In this study, we present a comprehensive characterization of a unique collection of PDO cultures derived from pancreatic ductal adenocarcinoma (PDAC), a particularly aggressive malignancy often diagnosed in advanced stages with limited treatment options and poor prognosis. Our investigation explores morphological and genetic profiling of 10 PDAC PDOs, specifically identifying distinct drug-response patterns. The morphological characteristics of these PDO cultures were investigated by microscopy, while the mutational status was acquired by panel sequencing. Subsequently, PDOs were treated with a panel of 10 inhibitors, either individually or in combination. The response to these treatments was assessed through live-cell imaging (Incucyte®) to capture and quantify changes in organoid morphology and endpoint measurements of cellular viability using a CellTiter-Glo® assay. Our detailed in vitro drug sensitivity testing revealed a remarkable diversity in the responses of the PDAC PDOs to different chemotherapeutic agents, including those currently under clinical trials. Of notable significance is the depth of understanding we have achieved, allowing us to establish potential correlations between PDO morphology, genetic mutations, and drug reactions. This finding provides a key insight into the complex interplay of these factors and their impact on treatment outcomes, offering valuable insights into understanding individual tumor subtypes' sensitivity or resistance to treatments. The PDOs exhibited a spectrum of cancer-associated mutations, including KRAS, TP53, SMAD4, CDKN2A, and ARID1A, reflecting the heterogeneity observed in PDAC patients. Morphologically, the organoids displayed various growth patterns, ranging from cystic and regular structures to filled or unfilled, granular, dark, and light membrane structures. Drug treatments underscored the diverse and unique drug-response profiles within the PDOs, with some demonstrating sensitivity to multiple treatments. In contrast, others exhibited resistance, highlighting the intricate nature of drug responses within this model. Overall, our findings emphasize the ability of PDAC organoids to faithfully capture the complexity and heterogeneity of the disease, making them promising platforms for guiding personalized oncology approaches. The diverse drug response profiles observed in PDOs advocate leveraging this model to develop precision medicine strategies tailored to patients' characteristics, ultimately paving the way for more effective and personalized treatment regimens in the fight against PDAC.
 Citation Format: Constanza Tapia Contreras, Günter Schneider, Denise Müller, Kimia Mirzakhani, Karly Conrads, Silke Kaulfuß, Tim Beißbarth, Gabriela Salinas, Nils Beyer, Sophia Reinländer, Ulrix Sax, Elisabeth Heßmann, Volker Ellenrieder, Philipp Ströbel, Michael Ghadimi. Patient-Derived Organoid Cultures for Personalized Therapies and Targeted Drug Screening Applications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A023.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a23581700d7eba1d7151f1cd1e85577199cc66e" target='_blank'>
              Abstract A023: Patient-Derived Organoid Cultures for Personalized Therapies and Targeted Drug Screening Applications
              </a>
            </td>
          <td>
            Constanza Tapia Contreras, Günter Schneider, Denise Müller, Kimia Mirzakhani, Karly Conrads, S. Kaulfuß, Tim Beißbarth, Gabriela Salinas, Nils Beyer, Sophia Reinländer, Ulrix Sax, E. Hessmann, V. Ellenrieder, Philipp Ströbel, Michael Ghadimi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a40aec86f303beb0c2a44fc542c17c17169d4e43" target='_blank'>
              Prognosis prediction via histological evaluation of cellular heterogeneity in glioblastoma
              </a>
            </td>
          <td>
            Mari Kirishima, Seiya Yokoyama, T. Akahane, Nayuta Higa, H. Uchida, Hajime Yonezawa, Kei Matsuo, Junkoh Yamamoto, Koji Yoshimoto, Ryosuke Hanaya, Akihide Tanimoto
          </td>
          <td>2024-10-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Single nucleotide variants (SNVs), as important components of genetic variation, affect gene expression, function and phenotype. Mining and summarizing the spatial distribution of SNVs in diseased and normal tissues for a better understanding of their characteristics and potential roles in cell-lineage determination, aging, or disease occurrence is significant. Herein, we have developed a comprehensive spatial mutation resource stSNV (http://bio-bigdata.hrbmu.edu.cn/stSNV/index.jsp), which provides an atlas of spatial SNVs in major diseased and normal tissues of human and mouse. stSNV documents 42 202 spatial mutated genes involving 898 908 SNVs called from 730 067 spots within 450 slices from 19 diseased and 28 normal tissues. Importantly, potential characteristics of SNVs are explored and provided by analyzing the perturbation of the SNVs to gene expression, spatial communication, biological function, region-specific mutated genes, spatial mutant signatures, SNV-cell co-localization and mutation core region. All these spatial mutation data and in-depth analyses have been integrated into a user-friendly interface, visualized through intuitive tables and various image formats. Flexible tools are developed to explore co-localization among clusters, genes, cell types and SNVs in the same slice. In summary, stSNV as a valuable resource helps to dissect intra-tissue genetic heterogeneity and lays the groundwork for understanding the SNVs' biological regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44ecc8b01ff77f4282637b8dc14044c728ac01de" target='_blank'>
              stSNV: a comprehensive resource of SNVs in spatial transcriptome.
              </a>
            </td>
          <td>
            Changbo Yang, Yujie Liu, Xiaohua Wang, Qing Jia, Yuqi Fan, Zhenglin Lu, Jingyi Shi, Zhaoxin Liu, Gengdong Chen, Jianing Li, Weijian Lu, Weiwei Zhou, Dezhong Lv, Haozhe Zou, Juan Xu, Yongsheng Li, Qinghua Jiang, Tao Wang, Tingting Shao
          </td>
          <td>2024-10-29</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive disease. In most cases, PDAC initiation results from oncogenic KRAS mutations, that act in part by directly increasing levels of a glycoprotein called Mucin-16 (MUC16). MUC16 is virtually absent in the normal pancreas, expressed on tumor cells in >65% of PDAC and is a marker of poor prognosis. The cell autonomous roles of MUC16 involve signaling through the ErbB-1/2/3 axes, and eliciting downstream AKT and GSK-3β activation to promote cell proliferation and survival. PDAC has a very interesting “immune cold” tumor microenvironment (TME) that is devoid of anti-tumor T cells and is instead flush with pro-tumor myeloid cells like macrophages. However, the role of MUC16 in influencing immune cells, particularly macrophages has not been investigated. Methods: We use immunohistochemistry to check for macrophage populations in spontaneous and orthotopic models of PDAC, KPC (LSL-Kras G12D/+ ; LSL-Trp53 R172H/+ ; Pdx-1- Cre) and a derived KPC-MUC16 knockout (KPC-M16KO). Orthotopic implantation with KPC and KPC-M16- cells was done to assess survival. In vitro cytokine screening from M16+/- cells was done to understand cytokine players in MUC16-mediated immunosuppression. Clodronate liposomes (CL) and neutral control liposomes (NL) were used to deplete macrophages in orthotopically implanted KPC and KPC-M16- tumor-bearing mice, and flow cytometry was conducted to profile macrophages and T cells in the pancreas and spleens at the experimental endpoint. Results: We show that KPC-M16KO tumors have decreased M2-like macrophages, and a modest increase in M1-like macrophages, leading to a high M1/M2 ratio suggesting reduced immune suppressed cells in the KPC-M16KO models. KPC-M16KO tumor-bearing mice showed longer survival compared to the KPC group. In addition, we detect low levels of CCL2 and Serpin-E1 in cell lines and tumor tissues from KPC-M16KO models suggesting that they could serve as potential regulators of macrophage infiltration and M2-like polarization downstream of MUC16. Siglec-E (murine homolog of human Siglec-7/9) was also downregulated in KPC-M16KO tumors, suggesting MUC16-Siglec mediated immunosuppression as another mechanism. CL administration (versus NL) resulted in significant macrophage depletion in the spleens of mice with KPC and KPC-M16KO tumors, however, such depletion was not robust in the pancreas. Immune profiling studies showed a high M1/M2 ratio in the spleen and pancreas of mice treated with CL, suggesting a tumor cell-macrophage crosstalk, that could mediate M2-like macrophage states in a macrophage-rich TME. Mice treated with CL have smaller tumors, and KPC-M16KO+CL group had the smallest tumors, indicative that dual disruption of MUC16 and macrophages lowers tumor burden. Conclusions: These results suggest that MUC16 plays a crucial role in modulating the tumor immune microenvironment by increasing the accumulation of immunosuppressive macrophages in PDAC.
 Citation Format: Christabelle Rajesh, Satish Sagar, Prakash Radhakrishnan. Influence of MUC16 on the immune tumor microenvironment of PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B051.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cdddd6fa80a3802aae69cece6c20e3daa98477b" target='_blank'>
              Abstract B051: Influence of MUC16 on the immune tumor microenvironment of PDAC
              </a>
            </td>
          <td>
            Christabelle Rajesh, S. Sagar, P. Radhakrishnan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer death by 2030. PDAC lacks effective treatment options and innovative strategies for therapeutic intervention are urgently needed. Chimeric Antigen Receptor (CAR)-T cell therapy has shown remarkable success in treating hematologic malignancies, yet achieving efficacy in solid tumors remains a significant challenge. Early results from clinical trials show limited efficacy of CAR T-cell therapy in PDAC. To uncover tumor-intrinsic mechanisms of resistance to CAR T-cell therapy in vivo, we performed iterative CRISPR/Cas9-based pooled screens in a fully immunocompetent, orthotopic model of PDAC. We identified a variety of genes influencing CAR T-cell treatment efficacy. In particular, loss of genes involved in redox biology and oxidative stress sensitized PDAC tumors to CAR T-cell therapy in vivo. Additionally, single-cell RNA sequencing of PDAC tumors subjected to CAR T-cell therapy unveiled a distinct subset of tumor cells resilient to the treatment, characterized by changes in oxidative stress. Our findings indicate a potential avenue for therapeutic synergy by targeting pathways involved in oxidative stress to increase the efficacy of CAR T cell therapy in PDAC.
 Citation Format: Julia Froese, Charlie Whittaker, Paul Leclerc, Adam Langenbucher, Riley Hellinger, Daniel Goulet, Michael T. Hemann. Elucidating the tumor-driven mechanisms of resistance to immunotherapies in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr B039.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81348949cfc84943625452ad98cf593484f76b30" target='_blank'>
              Abstract B039: Elucidating the tumor-driven mechanisms of resistance to immunotherapies in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening
              </a>
            </td>
          <td>
            Julia Froese, Charles A. Whittaker, Paul Leclerc, Adam Langenbucher, Riley D Hellinger, Daniel Goulet, Michael Hemann
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e6d29d20af097f819edb5c85ecc5de9f784bb73" target='_blank'>
              Inertial effect of cell state velocity on the quiescence-proliferation fate decision
              </a>
            </td>
          <td>
            Harish Venkatachalapathy, Cole Brzakala, Eric Batchelor, Samira M. Azarin, Casim A. Sarkar
          </td>
          <td>2024-10-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/869c734206e668652b18e8e50d0afc69d2273b39" target='_blank'>
              Intratumoral microbiota of pancreatic ductal adenocarcinoma impact patient prognosis by influencing tumor microenvironment
              </a>
            </td>
          <td>
            Jingze Leng, Hengyi Xu, Xiaoyu Liu, Yufan Yang, Chun Ning, Lejia Sun, Jiangming Qu, Xindi Ke, Xun Lan
          </td>
          <td>2024-09-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="As we age, many tissues become colonised by microscopic clones carrying somatic driver mutations. Some of these clones represent a first step towards cancer whereas others may contribute to ageing and other diseases. However, our understanding of the clonal landscapes of human tissues, and their impact on cancer risk, ageing and disease, remains limited due to the challenge of detecting somatic mutations present in small numbers of cells. Here, we introduce a new version of nanorate sequencing (NanoSeq), a duplex sequencing method with error rates <5 errors per billion base pairs, which is compatible with whole-exome and targeted gene sequencing. Deep sequencing of polyclonal samples with single-molecule sensitivity enables the simultaneous detection of mutations in large numbers of clones, yielding accurate somatic mutation rates, mutational signatures and driver mutation frequencies in any tissue. Applying targeted NanoSeq to 1,042 non-invasive samples of oral epithelium and 371 samples of blood from a twin cohort, we found an unprecedentedly rich landscape of selection, with 49 genes under positive selection driving clonal expansions in the oral epithelium, over 62,000 driver mutations, and evidence of negative selection in some genes. The high number of positively selected mutations in multiple genes provides high-resolution maps of selection across coding and non-coding sites, a form of in vivo saturation mutagenesis. Multivariate regression models enable mutational epidemiology studies on how carcinogenic exposures and cancer risk factors, such as age, tobacco or alcohol, alter the acquisition and selection of somatic mutations. Accurate single-molecule sequencing has the potential to unveil the polyclonal landscape of any tissue, providing a powerful tool to study early carcinogenesis, cancer prevention and the role of somatic mutations in ageing and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb1ed2352282f2639f8bc59358f511e72d35ab26" target='_blank'>
              Somatic mutation and selection at epidemiological scale
              </a>
            </td>
          <td>
            A. R. Lawson, F. Abascal, P. A. Nicola, S. Lensing, Amy L Roberts, Georgios Kalantzis, A. Baez-Ortega, Natalia Brzozowska, Julia S. El-Sayed Moustafa, Dovile Vaitkute, Belma Jakupovic, Ayrun Nessa, S. Wadge, Anna L. Paterson, Doris M. Rassl, R. E. Alcantara, L. O’Neill, S. Widaa, Siobhan Austin-Guest, M. Neville, M. Przybilla, Wei Cheng, M. Morra, Lucy Sykes, Matthew Mayho, Nicole Müller-Sienerth, Nick Williams, Diana Alexander, Luke M. R. Harvey, Thomas Clarke, Alex Byrne, Jamie R Blundell, Matthew D. Young, A. KrishnaaT., Mahbubani, K. Saeb-Parsy, Hilary C. Martin, M. R. Stratton, P. J. Campbell, R. Rahbari, K. Small, I. Martincorena
          </td>
          <td>2024-11-01</td>
          <td>None</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Diffuse high-grade gliomas are the most common malignant adult neuroepithelial tumors in humans and a leading cause of cancer-related death worldwide. The advancement of high throughput transcriptome sequencing technology enables rapid and comprehensive acquisition of transcriptome data from target cells or tissues. This technology aids researchers in understanding and identifying critical therapeutic targets for the prognosis and treatment of diffuse high-grade glioma.Spatial transcriptomics was conducted on two cases of isocitrate dehydrogenase (IDH) wild-type diffuse high-grade glioma (Glio-IDH-wt) and two cases of IDH-mutant diffuse high-grade glioma (Glio-IDH-mut). Gene set enrichment analysis and clustering analysis were employed to pinpoint differentially expressed genes (DEGs) involved in the progression of diffuse high-grade gliomas. The spatial distribution of DEGs in the spatially defined regions of human glioma tissues was overlaid in the t-distributed stochastic neighbor embedding (t-SNE) plots.We identified a total of 10,693 DEGs, with 5,677 upregulated and 5,016 downregulated, in spatially defined regions of diffuse high-grade gliomas. Specifically, SPP1, IGFBP2, CALD1, and TMSB4X exhibited high expression in carcinoma regions of both Glio-IDH-wt and Glio-IDH-mut, and 3 upregulated DEGs (SMOC1, APOE, and HIPK2) and 4 upregulated DEGs (PPP1CB, UBA52, S100A6, and CTSB) were only identified in tumor regions of Glio-IDH-wt and Glio-IDH-mut, respectively. Moreover, Kyoto Encyclopedia of Genes and Genomes (KEGG) and gene ontology (GO) enrichment analyses revealed that upregulated DEGs were closely related to PI3K/Akt signaling pathway, virus infection, and cytokine-cytokine receptor interaction. Importantly, the expression of these DEGs was validated using GEPIA databases. Furthermore, the study identified spatial expression patterns of key regulatory genes, including those involved in protein post-translational modification and RNA binding protein-encoding genes, with spatially defined regions of diffuse high-grade glioma.Spatial transcriptome analysis is one of the breakthroughs in the field of medical biotechnology as this can map the analytes such as RNA information in their physical location in tissue sections. Our findings illuminate previously unexplored spatial expression profiles of key biomarkers in diffuse high-grade glioma, offering novel insight for the development of therapeutic strategies in glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0de4338631623fe99431c55e90ce245032a07df9" target='_blank'>
              Spatial transcriptomics analysis identifies therapeutic targets in diffuse high-grade gliomas
              </a>
            </td>
          <td>
            Yongtao Yang, Yingzhou Hong, Kai Zhao, Minhao Huang, Wenhu Li, Kui Zhang, Ninghui Zhao
          </td>
          <td>2024-10-24</td>
          <td>Frontiers in Molecular Neuroscience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 The inherent adaptive capacity of GBM to metabolic stress represents a key clinical challenge, and the underlying mechanisms remain ill-defined. Our lab focuses on the emerging role of lipid metabolism in cancer cell stress adaptation. We have previously found increased scavenging of lipid particles, including extracellular vesicles (EVs) and lipoproteins, during acute stress, further resulting in a lipid droplet (LD) storage phenotype. Here, we propose a new mechanistic link between chondrotin sulfate proteoglycan (CSPG) remodelling, extracellular lipid scavenging, and LD formation that together serve to fine tune intracellular lipid metabolism to escape ferroptosis.



 We have comprehensively profiled GBM tissues and cells during different acidic stress adaptation conditions, including freshly resected GBM specimens, primary GBM cultures, and established GBM cell-lines. Methods/analyses include: Multiomics (RNA, proteome, glycome), laser capture microdissection for spatial profiling, snRNA-seq, immunophenotyping by FACS and imaging, and in vitro treatment studies (in 2D and 3D) with live imaging of cytotoxicity and ferroptosis. Mouse GBM models were employed for in vivo treatment studies.



 We provide first evidence of a functional role of cell-surface CSPGs during GBM adaptation to metabolic acidosis. Substantial CSPG induction coincides with LD formation in a process dependent on extracellular lipids and acidification. Specifically, CSPG encapsulation of stressed GBM cells acts as a shield against excessive lipid particle scavenging while LDs consitute an intracellular sink for lipid storage. Based on these findings, we introduce combined targeting of CSPGs and LDs as a strategy to challenge the stressed tumor niche. Concomitant disruption of CSPG and LD formation unleashes a metabolic vulnerability in acidic cells driven by uncontrolled lipid uptake and peroxidation, mitochondrial disintegration, and ferroptotic death.



 In conclusion, this study identifies a potentially targetable adaptative mechanism in the treatment resistant, acidic niche of GBM. We propose that cancer cell escape from acidic stress involves the induction of CSPGs - providing a shield against the lipid rich tumor microenvironment - and LDs - forming an intracellular lipid sink. Combined targeting of CSPG and LD formation to induce ferroptosis may offer new therapeutic opportunities in GBM and other solid tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94131c305e74656b4d8349f795b86e501414c82d" target='_blank'>
              OS09.5.A UNEXPECTED FUNCTIONAL ROLE OF GLYCOCALYX ENCAPSULATION DURING GBM METABOLIC ADAPTATION OPENS AVENUES FOR NEW TREATMENT CONCEPTS
              </a>
            </td>
          <td>
            A. Bång-Rudenstam, M. Cerezo-Magaña, K. Gonçalves de Oliveira, V. Governa, H. Talbot, L. Kjellen, E. Tykesson, A. Malmström, T. van Kuppevelt, K. Forsberg-Nilsson, J. Bengzon, M. Belting
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Disclosure: D. Zhang: None. M. Bergsneider: None. M.B. Wang: None. W. Kim: None. H. Vinters: None. A.P. Heaney: None. Most pituitary corticotroph adenomas are benign but ∼15% are refractory to conventional surgical, medical and even radiotherapeutic management. This tumor subset invade sellar adjacent structures, and in rare cases metastasize locally within the central nervous system or to neck lymph nodes (LN) or more distant organs such as the liver. To better understand the potential interactions between tumor cells and neighboring tissues during these progressive stages, we used spatial transcriptomics (ST, Visium, 10x Genomics) to profile the genetic composition and native architecture of 4 surgically resected corticotroph tumor paraffin-embedded samples collected from the same patient over several years. Samples 1& 2 invaded the sino-nasal mucosa, sample 3 the parietal dura, sample 4 the clival bone, and sample 5 had metastasized locally to a neck LN. Quality of the 4 samples was confirmed by DV200 score (51 ± 7%). An average of 2,473 tissue spots per tissue sample were obtained after sequencing with a median of 4,893 genes detected per spot. We then used the Seurat analytic pipeline to reconstitute spatial transcriptional distribution that authentically mirrored histologically verifiable structures. These included the main tumor area itself, it’s invading tumor edge, adjacent normal epithelium, dura, and LN tissue. Further deconvolution analysis revealed a distinct cellular composition at the invading tumor edge which was tissue context specific. For instance, a proliferative cell population was noted between the central tumor area and it’s invasive margin, B cell and fibroblast cell enrichment was prominent at sino-nasal mucosal invasive margins, whereas endothelial and pericyte cells were prominent along the tumor/ normal pituitary border. . As expected, the central tumor area expressed genes associated with corticotroph hormone production including POMC, NNAT, GAL and GNAS, genes associated with mitochondrial energy metabolism such as ATP5F1E, NDUFA8 and COX7B as well as transcripts related to lipid kinase activity (GKG, CERKL, PIK3C2G, DGKK). Most intriguingly, tumor cells at the fore-front of the invading edge expressed high level of cyclin-dependent kinases, potentially reflecting higher proliferative rates of these invasive margin cells. Additional studies to validate and define the role of theses invasive margin transcripts are now warranted to translate our profiling findings into potential drug targets. In summary, this is the first use of spatial transcriptomics to delineate the evolving transcriptomic features in a series of surgically resected pituitary corticotroph tumors from the same patient that progressed over time to a locally metastatic corticotroph carcinoma. It provides unparalleled insights into the differential transcriptional topography of corticotroph tumor tissue and most importantly, its invading edge and may identify novel therapeutic targets. Presentation: 6/3/2024">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52629cfca0e770c37dfc12e4ac5b75c2f708ab6e" target='_blank'>
              8133 Spatial Transcriptomic Profiling of Pituitary Corticotroph Carcinoma: Uncovering Architectural Complexity
              </a>
            </td>
          <td>
            Dongyun Zhang, M. Bergsneider, Marilene B. Wang, Won Kim, H. Vinters, A. Heaney
          </td>
          <td>2024-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background: Kidney renal clear cell carcinoma (KIRC) is the most prevalent subtype of malignant renal cell carcinoma and is well known as a common genitourinary cancer. Stratifying tumors based on heterogeneity is essential for better treatment options. Methods: In this study, consensus clusters were constructed based on gene expression, DNA methylation, and gene mutation data, which were combined with multiple clustering algorithms. After identifying two heterogeneous subtypes, we analyzed the molecular characteristics, immunotherapy response, and drug sensitivity differences of each subtype. And we further integrated bulk data and single-cell RNA sequencing (scRNA-Seq) data to infer the immune cell composition and malignant tumor cell proportion of subtype-related cell subpopulations. Results: Among the two identified consensus subtypes (CS1 and CS2), CS1 was enriched in more inflammation-related and oncogenic pathways than CS2. Simultaneously, CS1 showed a worse prognosis and we found more copy number variations and BAP1 mutations in CS1. Although CS1 had a high immune infiltration score, it exhibited high expression of suppressive immune features. Based on the prediction of immunotherapy and drug sensitivity, we inferred that CS1 may respond poorly to immunotherapy and be less sensitive to targeted drugs. The analysis of bulk data integrated with single-cell data further reflected the high expression of inhibitory immune features in CS1 and the high proportion of malignant tumor cells. And CS2 contained a large number of plasmacytoid B cells, presenting an activated immune microenvironment. Finally, the robustness of our subtypes was successfully validated in four external datasets. Conclusion: In summary, we conducted a comprehensive analysis of multi-omics data with 10 clustering algorithms to reveal the molecular characteristics of KIRC patients and validated the relevant conclusions by single-cell analysis and external data. Our findings discovered new KIRC subtypes and may further guide personalized and precision treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/203996654d7983016b85becfbf4c5b9e1aed4bef" target='_blank'>
              Integrative analysis of single-cell and bulk multi-omics data to reveal subtype-specific characteristics and therapeutic strategies in clear cell renal cell carcinoma patients
              </a>
            </td>
          <td>
            Xinjia Ruan, Chong Lai, Leqi Li, Bei Wang, Xiaofan Lu, Dandan Zhang, Jingya Fang, Maode Lai, Fangrong Yan
          </td>
          <td>2024-10-18</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/048c4b22ff35649ad9723e4c2a3778aaac4de791" target='_blank'>
              Coordinated, multicellular patterns of transcriptional variation that stratify patient cohorts are revealed by tensor decomposition.
              </a>
            </td>
          <td>
            Jonathan Mitchel, M. G. Gordon, Richard K. Perez, Evan Biederstedt, Raymund Bueno, C. Ye, P. V. Kharchenko
          </td>
          <td>2024-09-23</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ba4a11f7c854cb28f1695f90c8539432db45325" target='_blank'>
              Calibrating tumor growth and invasion parameters with spectral spatial analysis of cancer biopsy tissues
              </a>
            </td>
          <td>
            Stefano Pasetto, Michael Montejo, M. Zahid, Marilin Rosa, Robert A Gatenby, Pirmin Schlicke, Roberto Diaz, Heiko Enderling
          </td>
          <td>2024-10-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is especially lethal among cancers because current treatments are few and ineffective. One key stromal factor thought to contribute to the dismal prognosis of PDAC is the presence of cancer-associated fibroblasts (CAFs). However, CAF-targeting therapeutic strategies have so far failed. Recently, providing an explanation for this apparent contrasting observations, phenotypically distinct PDAC CAF subtypes have been discovered. We previously described two main PDAC CAF populations that are defined by distinct targetable molecular mechanisms: TGF-beta-dependent myofibroblastic CAFs (myCAFs) and IL-1-dependent inflammatory CAFs (iCAFs) (J Exp Med 2017). We also demonstrated how the activation of one pathway (i.e., TGF-beta signaling) blocks the activation of another (i.e., IL-1 signaling), leading to this CAF heterogeneity (Cancer Discovery 2019). Despite our progress, the roles of distinct PDAC CAF subtypes remain to be largely elucidated. Indeed, the ability of myCAFs and iCAFs to at least partially interconvert and the lack of genetically engineered mouse models (GEMMs) for their in vivo manipulation have so far hindered careful investigation of their functions in PDAC progression. To overcome these challenges, we have engineered new mouse models and ‘CAF-locked’ fibroblast lines that enable to alter CAF composition in vitro and in vivo. Leveraging these new models, we are starting to understand the respective crosstalk of iCAFs and myCAFs with each other, with PDAC malignant cells, with immune cells and with other stromal cells. Improving PDAC survival for all patients will likely require a better understanding of the complex nature of the tumor microenvironment in these heterogeneous cancers. While further genetic manipulation of myCAF (e.g., EGFR-activated myCAFs – Cancer Cell 2024) and iCAF subsets will be required to achieve that, our studies are crucial to begin to decipher this complexity.
 Citation Format: Giulia Biffi. Functional heterogeneity of fibroblast populations in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr IA-03.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5159b911b55b7d105859836df1e2efc50a3e69f" target='_blank'>
              Abstract IA-03: Functional heterogeneity of fibroblast populations in pancreatic cancer
              </a>
            </td>
          <td>
            Giulia Biffi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Glioblastoma is an extremely aggressive form of brain tumor. The median overall survival of patients is only 15 to 18 months, and the five-year survival rate is less than 10% despite aggressive multimodal treatment that includes total resection of the tumor followed by a combination of radio-, chemo- and Tumor Treating Fields (TTF)-therapy. Therefore, new therapeutic interventions are urgently needed. For this reason, it is essential to define novel therapeutic targets for this tumor entity. To this aim, we developed a combinatorial approach, termed “
 FLY dentification”, to identify and characterize novel driver genes in glioblastoma. This pipeline combines bioinformatics analysis of patients proteomics data together with functional screening in Drosophila models of glioblastoma, and validation with human glioblastoma cell models. This approach allowed us to initially uncover the top 300 most overexpressed proteins in glioblastoma tissues using in silico preselection algorithms. By taking advantage of a Drosophila model of glioblastoma ‒ driven by overexpression of the oncogenic forms of EGFR and PI3K ‒ and using an RNAi approach we tested the functional contribution of 192 of the gene candidates to glioblastoma lethality in vivo. This targeted screening allowed us to identify 41 clinically and functionally relevant gene candidates. Focusing on the top candidates with no reported function in glioblastoma, we demonstrated that the DNA binding and chromatin remodeling DEK protein contributes to human glioblastoma tumorigenesis in vitro. Some of these findings will be validated using in vivo models of human glioblastoma. Together, the established “FLY dentification” pipeline represents a combinatorial approach designed to uncover previously unrecognized glioblastoma driver genes, which has the power to reveal novel biomarkers and therapeutic targets for this tumor type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ca079226a82444c0dcc78c2ef1c66c6e5d84ed" target='_blank'>
              P06.05.B FLYDENTIFICATION - CHARACTERIZATION OF NOVEL MOLECULAR DRIVERS IN GLIOBLASTOMA BASED ON FUNCTIONAL GENOMICS SCREENING IN DROSOPHILA MELANOGASTER
              </a>
            </td>
          <td>
            O. Lewandowska, D. Picard, B. Hariri, T. Klein, G. Reifenberger, T. Reiff, G. Leprivier
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Over the last decade, single-cell genomics has revealed remarkable heterogeneity and plasticity of cell types in the lungs and airways. The challenge now is to understand how these cell types interact in three-dimensional space to perform lung functions, facilitating airflow and gas exchange while simultaneously providing barrier function to avoid infection. An explosion in novel spatially resolved gene expression technologies, coupled with computational tools that harness machine learning and deep learning, now promise to address this challenge. Here, we review the most commonly used spatial analysis workflows, highlighting their advantages and limitations, and outline recent developments in machine learning and artificial intelligence that will augment how we interpret spatial data. Together these technologies have the potential to transform our understanding of the respiratory system in health and disease, and we showcase studies in lung development, COVID-19, lung cancer, and fibrosis where spatially resolved transcriptomics is already providing novel insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24ca8a7b4002bcf62b81748ecefc28933b1e3587" target='_blank'>
              Spatial Transcriptomics of the Respiratory System.
              </a>
            </td>
          <td>
            Stathis Megas, A. Wilbrey-Clark, Aidan Maartens, Sarah A Teichmann, Kerstin B. Meyer
          </td>
          <td>2024-10-01</td>
          <td>Annual review of physiology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4cae5a686cb86b01a2b0b2288eccbe354b27458" target='_blank'>
              Quiescent cancer cells induced by high-density cultivation reveals cholesterol-mediated survival and lung metastatic traits.
              </a>
            </td>
          <td>
            Xingyang Liu, Qinjie Min, Xinxin Cheng, Weimin Zhang, Qingnan Wu, Xu Chen, Mengzhu Lv, Siqi Liu, Huihui Zhao, Di Yang, Yidi Tai, Xiao Lei, Yan Wang, Qimin Zhan
          </td>
          <td>2024-10-11</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer death by 2030. PDAC lacks effective treatment options and innovative strategies for therapeutic intervention are urgently needed. Chimeric Antigen Receptor (CAR)-T cell therapy has shown remarkable success in treating hematologic malignancies, yet achieving efficacy in solid tumors remains a significant challenge. Early results from clinical trials show limited efficacy of CAR T-cell therapy in PDAC. To uncover tumor-intrinsic mechanisms of resistance to CAR T-cell therapy in vivo, we performed iterative CRISPR/Cas9-based pooled screens in a fully immunocompetent, orthotopic model of PDAC. We identified a variety of genes influencing CAR T-cell treatment efficacy. In particular, loss of genes involved in redox biology and oxidative stress sensitized PDAC tumors to CAR T-cell therapy in vivo. Additionally, single-cell RNA sequencing of PDAC tumors subjected to CAR T-cell therapy unveiled a distinct subset of tumor cells resilient to the treatment, characterized by changes in oxidative stress. Our findings indicate a potential avenue for therapeutic synergy by targeting pathways involved in oxidative stress to increase the efficacy of CAR T cell therapy in PDAC.
 Citation Format: Julia Fröse, Charlie Whittaker, Paul Leclerc, Adam Langenbucher, Riley Hellinger, Daniel R. Goulet, Michael T. Hemann. Unveiling resistance mechanisms to CAR T-cell therapy in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B060.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a130506e26e2da6b9c6338dc0032b746f1f4ca46" target='_blank'>
              Abstract B060: Unveiling resistance mechanisms to CAR T-cell therapy in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening
              </a>
            </td>
          <td>
            Julia Fröse, Charles A. Whittaker, Paul Leclerc, Adam Langenbucher, Riley D Hellinger, Daniel Goulet, M. Hemann
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 The localization of gliomas in the brain is a critical prognostic factor, reﬂecting the genetic makeup of their originating cells. Variations in transcriptomic proﬁles across different regions and subtypes may further shed light on the mechanisms driving tumor development. Our research delves into the complex interplay between glioma locations and their transcriptomic signatures, seeking to dissect the spatial and genetic intricacies contributing to glioma diversity.



 In this study, we conducted a comprehensive analysis of preoperative anatomical MRI and transcriptomic data from 242 glioma patients to investigate the relationship between spatial distribution and tumor transcriptomic proﬁles. The MRI data were standardized by registration to the MNI-152 brain space. Tumors were segmented, and the brain was partitioned into 84 discrete subregions for detailed examination. Transcriptomic data underwent preprocessing, with principal component analysis employed to reduce dimensionality. Statistical analyses were used to assess the relationship between glioma description across different subtypes and PCA results. Additionally, we identify differentially expressed genes and common genes across regions for various glioma subtypes. Comparative analyses with the Allen Brain Atlas (divided into 84 subregions with expression data) were also performed to discern transcriptional differences between healthy individuals and glioma patients, further enhancing our understanding of glioma biology at the molecular level.



 We generated 21 frequency maps detailing regional glioma occurrences, categorized by biomarkers, subtypes, and grades. Notably, some of these maps showed signiﬁcant correlations with expression principal component analyses (PCAs). Our ﬁndings include a set of genes consistently expressed across subregions, alongside region speciﬁc differentially expressed genes. Additionally, we identiﬁed distinct gene expression patterns when comparing glioma patients to healthy controls in the Allen Brain Atlas, highlighting potential molecular underpinnings of glioma heterogeneity.



 Our analysis revealed notable correlations between glioma subtype locations and transcriptomic proﬁles, underscoring the role of transcriptomic signatures in explaining regional variations in glioma incidence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9710a81388df04aff0d2613923e0becf7f6f7df7" target='_blank'>
              INTEGRATING TRANSCRIPTOMICS WITH SPATIAL RADIOGRAPHIC ATLASES FOR GLIOMA ANALYSIS
              </a>
            </td>
          <td>
            Mr Yonghao Li, Mr Yizhou Wan, Prof Stephen Price
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Introduction Cancer-associated fibroblasts (CAFs) are key players in the fibrotic tumour microenvironment (TME) of pancreatic cancer. Recent evidence has demonstrated the existence of multiple CAF subtypes, however there is relatively little evidence validating and corroborating these CAF subtypes in resected human pancreatic cancer. Furthermore, the spatial interactions between CAF subtypes and the epithelial and immune compartments of the pancreatic cancer TME is not well understood. Methods We designed and optimised a bespoke 8-plex immunofluorescence panel for the purposes of CAF subtype profiling and elucidation of CAF spatial biology: panCK, FAP, SMA, CTGF, S100A4, IL6, LIF, PDGFR. This was applied to a cohort of n=215 resected pancreatic cancer, well characterised with highly granular clinico-pathological and transcriptomic data through the Australian Pancreatic Cancer Genome Initiative. Spatial analysis was performed with novel computational biology techniques. Results We demonstrate seven CAF subtypes which cluster to form distinct, highly prognostic “enrichment profiles” (EPs). The cold EP represents quiescent CAFs in close proximity to immune cells (multivariate HR 0.45, 95% CI 0.30-0.68, p<0.001), and the fibrotic EP represents activated CAFs in close proximity to epithelial cells (multivariate HR 2.22, 95% CI 1.48-3.35, p<0.001). We also demonstrate the prognostic power of high spatial entropy (loss of tissue architecture, correlating with the fibrotic EP) vs low spatial entropy (preservation of tissue architecture, correlating with the cold EP) (HR 2.22, 95% CI 1.47-3.35, p<0.001). Conclusion We begin to unravel the elusive spatial biology of pancreatic CAFs in a large resected human cohort. We see changes in immune cell infiltrate, modulation of cellular relationships, and greater tissue disorganisation as a result of CAF activation. These features are highly prognostic and we hypothesise they are driven by changes in crosstalk between TME compartments. Current work focuses on application of these techniques to a large resected neoadjuvantly-treated cohort, allowing further interrogation of the notorious fibrotic post-neoadjuvant pancreatic cancer TME. If accepted for presentation we look forward to presenting both cohorts simultaneously.
 Citation Format: Adam Bryce, Stephan Dreyer, Fieke Froeling, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, John Le Quesne, David Chang. Interrogating the spatial biology of pancreatic cancer-associated fibroblasts in resected human pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A053.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179192d42336b982da176e9aa5fe870344303cba" target='_blank'>
              Abstract A053: Interrogating the spatial biology of pancreatic cancer-associated fibroblasts in resected human pancreatic cancer
              </a>
            </td>
          <td>
            A. Bryce, S. Dreyer, F. Froeling, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, John Le Quesne, David Chang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47fd6c00b7eddf226853bfb45b4ea5ec0d0223f8" target='_blank'>
              Three-dimensional computer vision for exploring heterogeneity in collective Cancer Invasion
              </a>
            </td>
          <td>
            Yanlin Li, Ninghao Zhu, Mona Ahmed, Julio Urbina, Tai-Yin Huang, Pak Kin Wong
          </td>
          <td>2024-10-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Glioblastoma Multiforme (GBM) is a high-grade glioma known to infiltrate the surrounding brain parenchyma inducing a direct neuronal damage. Indeed, GBM mass is not an isolated and self-sustained entity but rather participates and interacts with the surrounding neuronal tissue. Specifically, tumoral cells induce neuron hyper-excitability while peritumoral cells seem to regulate tumor proliferation and progression. GBM affects 3 in 100,000 people and has a median survival of 15 months; it remains poorly understood and lethal. Therefore, gaining a deeper understanding of the alterations occurring in peritumoral tissues during GBM progression is of paramount importance.



 We investigate the plastic rearrangements of motor cortical areas in two different glioma-bearing mouse models using optogenetic stimulation, two photon imaging and immunohistochemical analyses. Specifically, we took advantage of Thy1-ChR2 transgenic mice, injected with two syngeneic lines of murine glioma cells (i.e. GL261 and PDGF+ Trp53-/- cells) into the motor cortex. Mice were optogenetically stimulated in 120 points per session (covering the entire extent of forelimb motor representation) at 3 different time points: baseline (before glioma cells injection), 14 and 21 days after tumor implantation.



 We show that both models of glioma-bearing mice underwent a strong remapping of forelimb motor territory. Namely, an increased threshold required for eliciting a forelimb movement was coupled with a decrease of forelimb specificity in the peritumoral area with respect to naïve mice. In addition, glioma-bearing mice showed a decrease in the number of dendritic spines, along with reduced density of perineuronal nets (PNNs) and inhibitory interneuron markers in the peritumoral cortex.



 We provide important insight into the interaction between GBM and surrounding tissue that can be used for developing more effective treatments to improve patients’ quality of life and survival.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6297cf9746d4f9b221b8e8f9f648c2b95db0018f" target='_blank'>
              P02.22.A REARRANGEMENTS OF PERITUMORAL TISSUE THAT OCCUR ALONG WITH GLIOMA PROGRESSION
              </a>
            </td>
          <td>
            E. Mori, C. Spalletti, M. Scalera, S. Haddad, D. Cangi, V. Pillai, E. Parmigiani, M. Mainardi, S. Landi, M. Caleo, E. Vannini
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intravital microscopy has enabled the study of immune dynamics in the pulmonary microvasculature, but many key events remain unseen because they occur in deeper lung regions. We therefore developed a technique for stabilized intravital imaging of bronchovascular cuffs and collecting lymphatics surrounding pulmonary veins in mice. Intravital imaging of pulmonary lymphatics revealed ventilation-dependence of steady-state lung lymph flow and ventilation-independent lymph flow during inflammation. We imaged the rapid exodus of migratory dendritic cells through lung lymphatics following inflammation and measured effects of pharmacologic and genetic interventions targeting chemokine signaling. Intravital imaging also captured lymphatic immune surveillance of lung-metastatic cancers and lymphatic metastasis of cancer cells. To our knowledge, this is the first imaging of lymph flow and leukocyte migration through intact pulmonary lymphatics. This approach will enable studies of protective and maladaptive processes unfolding within the lungs and in other previously inaccessible locations. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ad7292e3dc5444dbf480a1cf6ce97a64b4f2de" target='_blank'>
              Intravital imaging of pulmonary lymphatics in inflammation and metastatic cancer
              </a>
            </td>
          <td>
            S. Cleary, Longhui Qiu, Yurim Seo, P. Baluk, Dan Liu, N. Serwas, Jason G. Cyster, Donald M. McDonald, Matthew F. Krummel, M. Looney
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. Previous studies have identified m6A as a key player in glioma pathogenesis, promoting cell stemness and tumor cell proliferation. However, there is also evidence of a potential anti-cancer role of m6A. Here, we present a comprehensive overview of the m6A landscape in a glioma cell line model using Nanopore direct RNA sequencing platform. The study investigates functional and biological impact of targeted knockdown (KD) of key m6A regulators (reader: IGF2BP2, writer: METTL3, eraser: ALKBH5). Transcriptome wide mapping of m6A demonstrated enrichment in CDS and 3’UTR regions. Differential patterns of non-DRACH motifs were observed across the KD conditions. Our analysis identified 716 common m6A sites across all conditions and clusters of KD specific modifications enriched for distinct functional pathways. IGF2BP2 KD was associated with pathways involving protein metabolism and apoptosis regulation. METTL3 KD was linked to cell cycle and stress response regulation and ALKBH5 KD impacted ribosome biogenesis and protein synthesis. Here, for the first time we also report detection of full-length RNA transcripts in extracellular vesicles (EVs) derived RNA from knockdown and control conditions. These findings underscore the unique contributions of each m6A regulator to glioma cell transcriptomics and suggest potential pathways for elucidating the overall role of m6A machinery in glioma pathogenesis. Important clinical applications include identification of novel diagnostic, prognostic, and therapeutic m6A RNA biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3136f121112542cac680afcce8f3495635fccbb" target='_blank'>
              Exploring N6-Methyladenosine and long transcripts in glioma cells and extracellular vesicles
              </a>
            </td>
          <td>
            S. Batool, Ana K Escobedo, Hanna Lee, Prerna Khanna, Kase D. Haas, Tiffaney Hsia, Bob S Carter, L. Balaj
          </td>
          <td>2024-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with a five-year survival rate of only 13%, characterized by rapid proliferation, high metastatic potential, and resistance to conventional therapies. PDAC tumors are heavily reliant on nutrient-scavenging pathways such as autophagy and the lysosome – a degradative organelle that plays an essential role in the digestion and recycling of diverse cellular material. Despite their importance, the specific function of lysosomes over the course of tumor evolution and metastasis in PDAC tumors remains poorly understood. To understand the function of the lysosome in PDAC, we utilize a biochemical method known as LysoIP-based proteomics, which isolates intact lysosomes for subsequent analysis using mass spectrometry-based proteomics. This approach allows for a comprehensive analysis of lysosomal composition that can uncover unique features and functions of PDAC lysosomes that promote cellular adaptation to stress in support of tumor growth. A limitation of lysosome isolation from 2D cultured cells is that it does not recapitulate lysosomal features associated with tumors growing in vivo. To address this limitation, we generated a genetically engineered mouse model of PDAC incorporating conditional expression of ‘LysoTag’ (TMEM192-mRFP-3xHA) that enables capture and profiling of lysosomes at different stages of tumor evolution, including early-stage, late-stage, and metastatic disease. Employing this strategy, we successfully captured lysosomal proteins from primary tumors (pancreas), as well as lung and liver metastases. Our preliminary analysis revealed an enrichment of resident lysosomal membrane and luminal proteins in the datasets, confirming the successful isolation of intact lysosomes from these complex tissues. Additionally, we identified non-resident lysosomal proteins (termed cargo) that are enriched and co-evolved with advancing tumor stages. Notably, our results reveal distinct differences in the lysosomal proteome of tumor cells across different tissues. These findings emphasize the critical role of lysosomal function in tumor progression and suggest that targeting lysosomal components could be a promising therapeutic strategy for treating metastatic PDAC.
 Citation Format: Thuy T.P Nguyen, Grace A. Hernandez, Gilles Rademaker, Matthew Luy, Alexander Li, Longhui Qiu, Joao A. Paulo, Rushika M. Perera, Joseph D. Mancias. Defining the lysosome proteome during pancreatic cancer tumor evolution and metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A082.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccb1db833eb540121818b78d6999d64e82eb52f8" target='_blank'>
              Abstract A082: Defining the lysosome proteome during pancreatic cancer tumor evolution and metastasis
              </a>
            </td>
          <td>
            T. Nguyen, Grace A. Hernandez, Gilles Rademaker, Matthew Luy, Alexander Li, Longhui Qiu, Joao A. Paulo, Rushika M. Perera, J. Mancias
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Many solid tumors including breast cancer can exhibit early dissemination and dormancy-in which cancer cells spread early in the disease process and survive long periods without detectable growth. These early disseminated tumor cells sometimes reactivate and lead to incurable metastatic disease years or even decades after curative-intent therapy for the primary tumor. We are just beginning to understand the role of the immune system in this process in part because of improvements in immunocompetent models as well as technological advances such as single-cell genomics and spatial transcriptomics. In this issue of Cancer Research, Bushnell and colleagues showed that NK cells are important in this context. The authors found that disseminated tumor cells and quiescent cells express higher levels of MHC 1 but are resistant to NK-cell-mediated immunity. The proposed mechanism involves the STING pathway and transcription factors Sox2 and Bach1. As other studies have highlighted the importance of T-cell immunity, this work reaffirms the importance and diversity of immune regulation of dormancy and suggests the need for future studies to flesh out mechanistic details and predict when each type of immunity is most important. See related article by Bushnell et al., p. 3337.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9576e1dca44b3bd419b3eed082360a6cebe6d307" target='_blank'>
              Breast Cancer Disseminated Tumor Cells: Do They Stay and Fight or Run and Hide?
              </a>
            </td>
          <td>
            F. Cackowski, H. Korkaya
          </td>
          <td>2024-10-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4670cbb29b827ac111fd8c21b0ef2880316e9a14" target='_blank'>
              Cell type mapping reveals tissue niches and interactions in subcortical multiple sclerosis lesions.
              </a>
            </td>
          <td>
            Celia Lerma-Martin, Pau Badia-i-Mompel, R. O. Ramirez Flores, Patricia Sekol, P. S. L. Schäfer, Christian J. Riedl, Annika Hofmann, Thomas Thäwel, Florian Wünnemann, Miguel A. Ibarra-Arellano, Tim Trobisch, Philipp Eisele, Denis Schapiro, Maximilian Haeussler, Simon Hametner, Julio Saez-Rodriguez, Lucas Schirmer
          </td>
          <td>2024-11-05</td>
          <td>Nature neuroscience</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b108c069a149d86683e9a9aa9e66ae175bdebcb9" target='_blank'>
              MECOM Locus classical transcript isoforms affect tumor immune microenvironment and different targets in ovarian cancer
              </a>
            </td>
          <td>
            Ning Lan, Shuheng Bai, Min Chen, Xuan Wang, Zhaode Feng, Ying Gao, B. Hui, Wen Ma, Xiangxiang Zhang, Fengyuan Hu, Wanyi Liu, Wenyang Li, Fang Wu, Juan Ren
          </td>
          <td>2024-10-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Aberrant expression of repeat RNAs in pancreatic ductal adenocarcinoma (PDAC) mimics viral-like responses with implications for tumor cell state and the surrounding microenvironment. To better understand the relationship of repeat RNAs in human PDAC, we employed the NanoString CosMx™ spatial molecular imaging (SMI) platform, which utilizes a 1,000-plex RNA panel with custom repeat RNA probes targeting long interspersed nuclear element 1 (LINE-1) retrotransposon ORF1 and ORF2, HSATII satellite (SAT) repeat, and two human endogenous retroviruses (HERV-K and HERV-H) in 46 primary tumors. This analysis revealed correlations of high repeat RNA expression with alterations in the epithelial state of PDAC cells and the myofibroblast phenotype in cancer-associated fibroblasts (CAFs). The loss of cellular identity observed with dosing of extracellular vesicles (EVs) and individual repeat RNAs in PDAC and CAF cell culture models points to cell-cell intercommunication involving these viral-like elements. Differences in the PDAC and CAF response are driven by distinct innate immune signaling pathways through interferon regulatory transcription factor 3 (IRF3). Altogether, our data indicate that cell context-specific viral-like responses driven by tumor cells have broad impacts on single-cell heterogeneity within cancer cells and the pancreatic cancer microenvironment.
 Citation Format: Eunae You, Patrick Danaher, Chenyue Lu, Siyu Sun, Luli Zou, Ildiko Phillips, Alexandra Rojas, Natalie Ho, Yuhui Song, Michael Raabe, Katherine Xu, Peter Richieri, Hao Li, Natalie Aston, Rebecca Porter, Bidish Patel, Linda Nieman, Nathan Schurman, Briana Hudson, Khrystyna North, Sarah Church, Vikram Deshpande, Andrew Liss, Tae Kim, Yi Cui, Youngmi Kim, Benjamin Greenbaum, Martin Aryee, David Ting. Repeat RNA mediated disruption of cellular plasticity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A039.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffeaf733e03a440a3ea9633bad9509d48160c804" target='_blank'>
              Abstract A039: Repeat RNA mediated disruption of cellular plasticity in pancreatic cancer
              </a>
            </td>
          <td>
            Eunae You, Patrick Danaher, Chenyue Lu, Siyu Sun, Luli Zou, Ildiko E Phillips, Alexandra Rojas, Natalie Ho, Yuhui Song, Michael J. Raabe, Katherine Xu, Peter Richieri, Hao Li, Natalie Aston, Rebecca L. Porter, Bidish Kumar Patel, Linda T. Nieman, Nathan Schurman, Briana M. Hudson, Khrystyna North, Sarah Church, Vikram Deshpande, Andrew S. Liss, T. Kim, Yi Cui, Youngmi Kim, Benjamin Greenbaum, Martin J. Aryee, David T. Ting
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The tumor microenvironment (TME) can be regarded as a complex and dynamic microecosystem generated by the interactions of tumor cells, interstitial cells, the extracellular matrix, and their products and plays an important role in the occurrence, progression and metastasis of tumors. In a previous study, we constructed an IEO model (prI-, prE-, and pOst-metastatic niche) according to the chronological sequence of TME development. In this paper, to fill the theoretical gap in spatial heterogeneity in the TME, we defined an MCIB model (Metabolic, Circulatory, Immune, and microBial microenvironment). The MCIB model divides the TME into four subtypes that interact with each other in terms of mechanism, corresponding to the four major links of metabolic reprogramming, vascular remodeling, immune response, and microbial action, providing a new way to assess the TME. The combination of the MCIB model and IEO model comprehensively depicts the spatiotemporal evolution of the TME and can provide a theoretical basis for the combination of clinical targeted therapy, immunotherapy, and other comprehensive treatment modalities for tumors according to the combination and crosstalk of different subtypes in the MCIB model and provide a powerful research paradigm for tumor drug-resistance mechanisms and tumor biological behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6e1c21293602e7c8448df8e77945274f0be047" target='_blank'>
              The MCIB Model: A Novel Theory for Describing the Spatial Heterogeneity of the Tumor Microenvironment
              </a>
            </td>
          <td>
            Minghao Guo, Yinan Sun, Xiaohui Wang, Zikun Wang, Xun Yuan, Xinyi Chen, Xianglin Yuan, Lu Wang
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Clonal hematopoiesis of indeterminate potential (CHIP) is an acquired, genetic cardiovascular (CV) risk factor affecting about 10% of the general population at the age of 70 years. Most CHIP-driver mutations affect the epigenetic regulator gene DNMT3A. Experimental studies suggest a causal link between CHIP and inflammation-driven atherosclerosis, but the underlying mechanisms of how few mutant clones provoke adverse clinical outcomes in patients remain obscure.



 Single monocytes and bone marrow cells from DNMT3A-mutation carriers undergoing coronary angiography and open-heart surgery underwent RNA and genomic DNA sequencing in parallel on a single cell level. This approach enabled us to compare mutant and non-mutant cells directly within carriers, and to non-carrier cells. Atherosclerotic chimeric mice reconstituted with a mixture of Dnmt3a-deficient (CD45.2) and -competent (CD45.1) bone marrow served as an experimental model to compare mutant to non-mutant macrophages in atherosclerotic lesions.



 Surprisingly, RNA/gDNA parallel sequencing revealed that gene expression profiles of mutant monocytes resembled those of non-mutant monocytes in carriers of DNMT3A-CHIP mutations (n=4). Collectively, however, monocytes of DNMT3A mutation carriers were more inflammatory than those of matched non-carrier patients. Mutant hematopoietic stem and progenitor cells, analyzed by RNA/gDNA parallel sequencing in two carriers, expressed more activation and inflammatory markers compared to non-mutant cells within the same bone marrow, and intraindividual differences declined as cells differentiated into monocytes. Macrophages were isolated from atherosclerotic aortas of mixed bone marrow chimeric mice based on CD45.1 (non-mutant) or CD45.2 (mutant or non-mutant controls) expression and subjected to single cell RNA sequencing. Inflammation markers were higher expressed in non-mutant macrophages within carrier mice than in non-mutant macrophages of non-carrier chimeras, and showed relatively small differences compared to Dnmt3a-deficient macrophages.



 Our experimental findings in humans and mice support a pathomechanistic model in which few DNMT3A-CHIP mutation-carrying hematopoietic stem cells stimulate non-mutant cells within the bone marrow, thereby augmenting inflammation downstream in all circulating monocytes and their progeny in atherosclerotic lesions.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48931a326f84618148fc9b32717321cd9730d16b" target='_blank'>
              DNMT3A CHIP-driver mutations ignite the bone marrow
              </a>
            </td>
          <td>
            T. Dederichs, C. Ehlert, H. Becker, D. Wolf, M. Czerny, S. Jaiswal, D. Westermann, I. Hilgendorf
          </td>
          <td>2024-10-01</td>
          <td>European Heart Journal</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 Introduction Reprogramming of pancreas cell fate drives development of pancreatic ductal adenocarcinoma (PDAC). Acinar cells, the most probable origin of pancreatic cancer, undergo a rapid cell identity switch towards a duct-like phenotype upon KrasG12D expression and when combined with pancreatitis or the loss of acinar differentiation factors. Metaplastic and neoplastic duct-like cells are heterogeneous with a proportion acquiring features reminiscent of gastric lineages. While some gastric signatures are maintained in the classical PDAC subtype, they are eroded in the more aggressive, basal-like PDAC. Since subtype identity has a major impact on prognosis and therapeutic targetability, druggable targets that regulate cellular reprogramming in pancreatic cancer can be exploited to increase sensitivity to therapy. Methods To elucidate mechanisms responsible for early reprogramming, pancreatic tissue of mice with conditional KrasG12D expression and loss of Pdx1 was analyzed by single-nucleus ATAC-seq. Expression of identified target genes was studied in precancerous lesions and PDAC by using multiplex RNAscope and IHC staining. Computational analyses of publicly available sequencing data were used to establish correlation to PDAC subtype identity. Genes were functionally studied in PDAC cell lines. Results By performing snATAC-seq and RNA-seq of early transformed pancreatic tissue, we discovered that acinar cells with the combined expression of KrasG12D and loss of Pdx1 activate expression of a gastric metaplastic gene signature accompanied by elevated levels of the receptor kinase Ror2. Ror2 is also highly expressed in distinct subpopulations of metaplastic and neoplastic cells, associated with a gastric neck cell phenotype and enhanced proliferative capacity. In contrast, Ror2Low lesions are characterized by a gastric pit cell-like and a senescent phenotype. Genetic ablation of Ror2 resulted in a shift in lesion identity, enriching those with a pit cell and senescent identity. In PDAC, we found that Ror2 anti-correlates with a similar gastric pit cell phenotype that is maintained in the classical subtype PDAC, but is strongly associated with the more aggressive basal-like subtype. Overexpression of ROR2 in human PDAC cell lines with a classical differentiation induced loss of the classical gene signature as well as epithelial-to-mesenchymal transition. Moreover, ROR2 enforces a strong dependency on AKT signaling, causing increased vulnerability of ROR2-expressing cells to AKT inhibition, but increased resistance to the KRAS inhibitor MRTX1133. Conclusions We discovered Ror2 as a critical determinant of precancerous lesion as well as PDAC subtype identity. Its role in driving an aggressive PDAC phenotype that is inherently resistant to Kras inhibition suggests that inhibiting this receptor tyrosine kinase will enhance sensitivity to the new generation of targeted therapies.
 Citation Format: Simone Benitz, Alec Steep, Malak Nasser, Jonathan Preall, Ujjwal Mukund Mahajan, Holly McQuithey, Ian Loveless, Erick T. Davis, Hui-Ju Wen, Daniel W. Long, Thomas Metzler, Samuel Zwernik, Michaela Louw, Donald Rempinski, Daniel Salas-Escabillas, Sydney Brender, Linghao Song, Ling Huang, Brian K. Theisen, Zhenyu Zhang, Nina G. Steele, Ivonne Regel, Filip Bednar, Howard C. Crawford. Ror2, a Novel Key Regulator Driving Cell Fate Decisions throughout Pancreatic Carcinogenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C075.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada52d632c14a2e67876713dc628440690c3b38e" target='_blank'>
              Abstract C075: Ror2, a Novel Key Regulator Driving Cell Fate Decisions throughout Pancreatic Carcinogenesis
              </a>
            </td>
          <td>
            S. Benitz, Alexander Steep, Malak M Nasser, Jonathan Preall, U. Mahajan, Holly McQuithey, Ian Loveless, Erick T Davis, Hui-Ju Wen, Daniel W Long, Thomas Metzler, Samuel Zwernik, Michaela Louw, D. Rempinski, Daniel J. Salas-Escabillas, Sydney M Brender, Linghao Song, Ling Huang, Brian Theisen, Zhenyu Zhang, Nina G. Steele, I. Regel, Filip Bednar, Howard C. Crawford
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56281f9e2c04f45376f07cc22699136fff5305aa" target='_blank'>
              Exploring the interaction between immune cells in the prostate cancer microenvironment combining weighted correlation gene network analysis and single-cell sequencing: An integrated bioinformatics analysis
              </a>
            </td>
          <td>
            Danial Hashemi Karoii, Sobhan Bavandi, Melika Djamali, Ali Shakeri Abroudi
          </td>
          <td>2024-09-30</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="
 Pancreatic intraepithelial neoplasia (PanIN) lesions are the microscopic pre-malignant precursor to pancreatic ductal adenocarcinoma (PDAC). Animal models have demonstrated that PanINs arise from acinar cells which have undergone acinar-to-ductal metaplasia (ADM), but the exact molecular mechanisms which govern this transformation remain poorly understood. Novel in vitro models of PanIN genesis are necessary to better characterize pancreatic carcinogenesis and identify clinically actionable biomarkers of early disease. Organoids cultured from normal human pancreata have generated promise in this regard, but the cellular identity and malignant potential of their component cells have not been comprehensively characterized. We performed single-cell RNA sequencing (10X Genomics) on 6 normal pancreatic organoid lines maintained in culture for 3 weeks as well as their parent pancreata obtained from donors with no history of PDAC or other exocrine pancreatic pathology. Integration of whole pancreatic and organoid data using the R package Seurat generated a UMAP with clearly defined “ductal” and “acinar” clusters based on marker gene expression. The majority of organoid cells (47.3-81.5%) colocalized to the “ductal” cluster, and expressed SOX9, KRT19, and PDX1. However, a subset of cells (1.0-33.3%) present in each organoid sample aligned with the “acinar” cluster and expressed PRSS1, PRSS3, REG3A, MUC1, and SPINK1. In accordance with prior reports, organoid cells did not meaningfully express the acinar/endocrine markers PTF1A, CPA1, CHGA, or INS. Further examination of the “acinar” organoid cells revealed heightened expression of CLDN18, TFF1, TFF2, CTSE, CA2, and LYZ, genes associated with PanIN in prior spatial RNA sequencing studies. Other genes significantly enriched in this “acinar” organoid cluster include MUC5AC, a known PanIN marker, OLFM4, a tumor suppressor implicated in intestinal carcinogenesis, and the chemokine CXCL17, which is expressed in cells undergoing ADM. Overall, this data suggests that organoids derived from normal pancreata represent a heterogeneous population of epithelial cells with both acinar and ductal features. Furthermore, these organoids may represent an in vitro model of ADM and/or early PanIN formation, though additional functional mechanistic studies are required for confirmation.
 Citation Format: Alexander W Bray, Eileen S Carpenter, Timothy L Frankel, Marina Pasca di Magliano. Single-cell transcriptomic analysis of organoids derived from normal pancreata reveals epithelial cell heterogeneity and expression of PanIN-associated genes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A081.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67126c942c9917021e7682e3b9947f383b1d448b" target='_blank'>
              Abstract A081: Single-cell transcriptomic analysis of organoids derived from normal pancreata reveals epithelial cell heterogeneity and expression of PanIN-associated genes
              </a>
            </td>
          <td>
            Alexander W Bray, Eileen S Carpenter, Timothy L. Frankel, M. Pasca di Magliano
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7405dbfe981797b8a9d71c0696e2eb74696ed4d" target='_blank'>
              Genomic and transcriptomic landscape of human gastrointestinal stromal tumors
              </a>
            </td>
          <td>
            Feifei Xie, Shuzhen Luo, Dongbing Liu, Xiaojing Lu, Ming Wang, Xiaoxiao Liu, Fujian Jia, Yuzhi Pang, Yanying Shen, Chunling Zeng, Xin-Li Ma, Daoqiang Tang, L. Tu, Lin-Xi Yang, Yumei Cheng, Yuxiang Luo, Fanfan Xie, Hao Hou, Tao Huang, B-L Ni, C. Zhuang, Wen-Yi Zhao, Ke Li, Xufen Zheng, Wenbo Bi, X. Jia, Yi He, Simin Wang, Hui Cao, Kui Wu, Yue-Qing Wang
          </td>
          <td>2024-11-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 The immune microenvironment plays a critical role in cancer development, progression and resistance to therapy. KRAS mutations are near-ubiquitous in pancreatic ductal adenocarcinoma (PDAC) and several studies have demonstrated that KRASG12D can create an immunosuppressive tumor microenvironment. However, the immune effects of other oncogenic KRAS variants, such as KRASG12R (representing 20% of PDAC cases) are unknown. Given that specific KRAS mutants display distinct molecular phenotypes, we examined the immune cell composition of KRASG12R pancreatic tumors. Orthotopically implanted mouse tumors derived from KrasG12R/+;Trp53KO pancreatic organoids have a distinct immune profile in comparison to KrasG12D/+;Trp53KO tumors, characterized by an influx of intratumoral CD4+ T cells and mature dendritic cells. RNA sequencing of KRASG12R tumor cells revealed enrichment of Type I interferon (IFN) pathways accompanied by related chemokines, CXCL9 and CXCL10 in comparison to KRASG12D tumors. Analysis of publicly available human PDAC cohorts revealed enrichment of genes in inflammatory pathways in KRASG12R tumors. Collectively, these data link KRASG12R to a unique immune landscape that is distinct from KRASG12D tumors.
 Citation Format: Andrew wenger, Tal Baron, Erika Hissong, Adrian Vega-Perez, Whitney Sisso, Maria Paz Zafra, Rohit Chandwani, Lukas Dow, Despina Siolas. KRAS Mutation-Specific effects on the Tumor Immune Microenvironment Drive Tumor Progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7dad93aa0e8627285cb8c42371afc07798c76be" target='_blank'>
              Abstract A034: KRAS Mutation-Specific effects on the Tumor Immune Microenvironment Drive Tumor Progression
              </a>
            </td>
          <td>
            Andrew wenger, Tal Baron, Erika Hissong, Adrián Vega-Pérez, Whitney Sisso, Maria Paz Zafra, R. Chandwani, Lukas E. Dow, Despina Siolas
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is a highly diffusive tumor which restricts the efficacy of surgical resection and facilitates tumor recurrence. Matrix metalloproteinase 2 (MMP2) is the extracellular matrix protease that is activated by MMP14 expressed by microglia and plays a central role in tissue remodeling and tumor progression. The R132H mutation in isocitrate dehydrogenase 1 (IDH1R132H) is commonly observed in gliomas, results in hypermethylator phenotype and in GBMs is associated with better survival. The impacts of IDH1R132H on the tumor microenvironment of mIDH1/mATRX/mTP53 gliomas remains unexplored.



 We employed unique mouse glioma models which mimic human GBM pathology, coupled with high-dimensional technologies (CITEseq and spatial transcriptomics) to analyze and visualize the immune landscape in different gliomas. Additionally, we performed gelatin zymography, qPCR, wound healing assays, and DNA methylation qPCR techniques to validate the obtained results. MMP-2 expression/methylation was studied in the glioma TCGA dataset.



 The expression and activity of Mmp2 were different in experimental gliomas with different IDH1 status. Spatial transcriptomics demonstrated that Mmp2 expression is localized within the IDH1 wild-type tumors and absent in IDH1 mutated ones. Gelatin zymography showed the presence of Mmp2 activity in extracts from IDH1 wild-type tumors. Methylation of Mmp2 gene promoter was increased and resulted in decreased Mmp2 expression in IDH1 mutated gliomas. The IDH1 wild-type gliomas showed more diffuse growth patterns (tumors were stained with Nestin antibody) whereas IDH1 mutant gliomas showed more constrained growth patterns. In the TCGA glioma dataset, methylation of the MMP2 gene promoter was increased which was associated with decreased MMP2 expression in IDH1 mutated gliomas.



 We found that the IDH1 mutation results in hypermethylation of the Mmp2 gene promoter, leading to decreased Mmp2 expression and Mmp2 activity in experimental gliomas and human GBMs. Reduced Mmp2 activity and decreased tumor invasion result in more restricted tumor growth, which may facilitate resection, and radiotherapy, thereby improving patient survival. Studies were supported by National Science Center Poland grant 2020/39/B/NZ4/02683 (BK) and PASIFIC Call 1 PAS (MG).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81ebfa9c8baa458907e58b8a2d93e1bdfd385428" target='_blank'>
              P02.24.B THE IMPACT OF THE IDH1 R132H MUTATION ON MMP2 METHYLATION, EXPRESSION AND INVASION IN EXPERIMENTAL GLIOMAS AND HUMAN GBMS
              </a>
            </td>
          <td>
            A. M. Lenkiewicz, A. Ibragimova, M. Ghosh, P. Szadkowska, A. Ellert-Miklaszewska, S. Baluszek, K. Jacek, M. Castro, B. Kaminska
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In esophageal squamous cell carcinoma (ESCC), lymph node (LN) metastasis is associated with poor survival. Emerging evidence has demonstrated elevated CD8+ T-cell levels in metastatic LNs following immunotherapy and increased chemoresistance. However, the underlying regulatory mechanisms of CD8+ T cells in chemoresistant/metastatic patients have not been elucidated. Given that metastasis is linked to aberrant splicing patterns, transcripts with alternative splicing forms also play a critical role. With spatial transcriptomics (ST), spatial isoform transcriptomics (SiT), single-cell RNA sequencing (scRNA-seq), and T-cell receptor (TCR) sequencing, the spatial isoforms are analyzed quantitatively in human solid tumors and LNs. These isoforms are classified according to expression and spatial distribution patterns and proposed that the temporal heterogeneity in isoforms is attributed to isoform biogenesis dynamics. C1QC+ tumor-associated macrophages (TAMs) contribute to the formation of metastases in the context of both immunotherapy and chemotherapy. Additionally, CD74 isoforms can be targeted by mRNA drugs, such as antisense oligonucleotides (ASOs) and RNA interference (RNAi), to prevent chemoresistance and metastasis. Overall, this work suggests that C1QC+ TAMs interact with CD8+ CXCL13+ Tex cells via enrichment with the CD74 isoform in the ESCC 's metastatic lymph node.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b661dec030c9bd326290c3a36f5208dd2dc40655" target='_blank'>
              Spatial Isoforms Reveal the Mechanisms of Metastasis.
              </a>
            </td>
          <td>
            Yin Yin, Yuhao Wang, Xiaofei Yu, Yang Li, Yahui Zhao, Yanfeng Wang, Zhihua Liu
          </td>
          <td>2024-09-23</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Study of early pancreas neoplasia in humans had previously been limited by lack of available tissue. Recent work by our group on deceased donor pancreata identified pancreatic intraepithelial neoplasia (PanIN) lesions in non-diseased organs and defined a transcriptomic signature for the microenvironment of these pre-cancerous lesions. Prevalence of PanINs in healthy human tissue was higher than expected and established a novel model to expand understanding of the complex biology of these precursor lesions. Epigenetic changes in chromatin structure and the subsequent effects on gene expression are essential processes in neoplasia that have yet to be described at the single cell level in the human pancreas. We hypothesized that previously defined gene signatures could identify acinar, ductal, and PanIN cells in donor pancreata based on chromatin accessibility profiling by single nuclear transposase-accessible chromatin preparation followed by high-throughput sequencing (snATAC-seq). Comparison between the accessibility patterns, transcription factor motifs, and differences in accessibility versus expression profiles can then be used to expand prior knowledge of early pancreatic neoplasia and identify novel targets for further study and therapy selection. To test this hypothesis, single nuclei from seventeen donor pancreata (twenty total samples) were isolated for snATAC-seq using the 10x Genomics platform. Data analysis was conducted using CellRanger, Seurat, Signac, AUCell, and other programs to identify and label chromatin peaks in individual nuclei corresponding to acinar, ductal, and PanIN origin according to gene signatures identified by scRNA-seq and special transcriptomics on matched donor pancreata. Using this method, cells corresponding to these gene signatures, including PanIN- like cells, were indeed identifiable with differentially accessible regions of the genome then analyzed for transcription factor motif enrichment between these cell types. Motifs enriched in cells identified as PanINs compared to either ductal or acinar cells included several transcription factors implicated in tumorigenesis. These results are similar to previously demonstrated essential factors in pancreatic neoplasia, suggesting this strategy is a relevant method for data analysis and discovery of novel factors implicated in tumorigenesis. Ongoing studies include validation of these findings in vitro using human pancreatic normal epithelial cells, human tumor cell lines, and organoid models derived from human normal and tumor samples. Together, this study demonstrates that PanINs can be identified in human pancreas tissue by snATAC-seq and that differentially expressed motifs can be identified, providing opportunity for further integration with transcriptomic information to elucidate detailed understanding of early neoplasia in the pancreas for future drug development and targeted therapeutics studies.
 Citation Format: Jamie N Mills, Aaron Dendekker, Joyce K Thompson, Hannah Watkoske, Simone Benitz, Padma Kadiyala, Ahmed Elhossiny, Howard Crawford, Eileen Carpenter, Marina Pasca di Magliano, Filip Bednar. Characterization of chromatin accessibility patterns in the human pancreas and early pancreatic neoplastic lesions using snATAC-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C056.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eee7faf599c3d04827a3bdb6a5acb4e73be0ca2f" target='_blank'>
              Abstract C056: Characterization of chromatin accessibility patterns in the human pancreas and early pancreatic neoplastic lesions using snATAC-seq
              </a>
            </td>
          <td>
            Jamie N. Mills, Aaron Dendekker, Joyce K Thompson, Hannah R Watkoske, S. Benitz, Padma Kadiyala, Ahmed M. Elhossiny, Howard C. Crawford, Eileen S Carpenter, M. Pasca di Magliano, Filip Bednar
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Investigating mutations, including single nucleotide variations (SNVs), gene fusions, alternative splicing and copy number variations (CNVs), is fundamental to cancer study. Recent computational methods and biological research have demonstrated the reliability and biological significance of detecting mutations from single-cell transcriptomic data. However, there is a lack of a single-cell-level database containing comprehensive mutation information in all types of cancer. Establishing a single-cell mutation landscape from the huge emerging single-cell transcriptomic data can provide a critical resource for elucidating the mechanisms of tumorigenesis and evolution. Here, we developed scTML (http://sctml.xglab.tech/), the first database offering a pan-cancer single-cell landscape of multiple mutation types. It includes SNVs, insertions/deletions, gene fusions, alternative splicing and CNVs, along with gene expression, cell states and other phenotype information. The data are from 74 datasets with 2 582 633 cells, including 35 full-length (Smart-seq2) transcriptomic single-cell datasets (all publicly available data with raw sequencing files), 23 datasets from 10X technology and 16 spatial transcriptomic datasets. scTML enables users to interactively explore multiple mutation landscapes across tumors or cell types, analyze single-cell-level mutation-phenotype associations and detect cell subclusters of interest. scTML is an important resource that will significantly advance deciphering intra-tumor and inter-tumor heterogeneity, and how mutations shape cell phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44540801ce810af79670418ea8247b9a3e3dfa5" target='_blank'>
              scTML: a pan-cancer single-cell landscape of multiple mutation types.
              </a>
            </td>
          <td>
            Haochen Li, Tianxing Ma, Zetong Zhao, Yixin Chen, Xi Xi, Xiaofei Zhao, Xiaoxiang Zhou, Yibo Gao, Lei Wei, Xuegong Zhang
          </td>
          <td>2024-10-18</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The fibrotic stroma characterizing pancreatic ductal adenocarcinoma (PDAC) derives from a progressive tissue rigidification, which induces epithelial mesenchymal transition and metastatic dissemination. The aim of this study was to investigate the influence of matrix stiffness on PDAC progression by analyzing the proteome of PDAC-derived extracellular vesicles (EVs). PDAC cell lines (mPDAC and KPC) were grown on synthetic supports with a stiffness close to non-tumor (NT) or tumor tissue (T), and the protein expression levels in cell-derived EVs were analyzed by a quantitative MSE label-free mass spectrometry approach. Our analysis figured out 15 differentially expressed proteins (DEPs) in mPDAC-EVs and 20 DEPs in KPC-EVs in response to matrix rigidification. Up-regulated proteins participate to the processes of metabolism, matrix remodeling, and immune response, altogether hallmarks of PDAC progression. A multimodal network analysis revealed that the majority of DEPs are strongly related to pancreatic cancer. Interestingly, among DEPs, 11 related genes (ACTB/ANXA7/C3/IGSF8/LAMC1/LGALS3/PCD6IP/SFN/TPM3/VARS/YWHAZ) for mPDAC-EVs and 9 (ACTB/ALDH2/GAPDH/HNRNPA2B/ITGA2/NEXN/PKM/RPN1/S100A6) for KPC-EVs were significantly overexpressed in tumor tissues according to gene expression profiling interaction analysis (GEPIA). Concerning the potential clinical relevance of these data, the cluster of ACTB, ITGA2, GAPDH and PKM genes displayed an adverse effect (p < 0.05) on the overall survival of PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a05c9f844961832d60b66d635818946ef2b774" target='_blank'>
              Matrix stiffness regulates the protein profile of extracellular vesicles of pancreatic cancer cell lines.
              </a>
            </td>
          <td>
            Benedetta Ferrara, S. Bourgoin-Voillard, Damien Habert, Benoit Vallée, A. Nicolás-Boluda, Isidora Simanic, Michel Seve, Benoit Vingert, Florence Gazeau, Flavia Castellano, José Cohen, José Courty, Ilaria Cascone
          </td>
          <td>2024-09-16</td>
          <td>Proteomics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The genomic features of pancreatic ductal adenocarcinoma (PDAC) have been well described, yet the evolutionary contexts within which those features occur remains unexplored. We studied the genome landscapes, phylogenies and clonal compositions of 91 PDACs in relation to clinicopathologic features. There was no difference in the number of driver mutations or the evolutionary timing that each mutation occurred. High truncal density, a metric of the accumulation of somatic mutations in the lineage that gave rise to each PDAC, was significantly associated with worse overall survival. Polyclonal, monoclonal or mixed polyclonal/monoclonal metastases were identified across the cohort highlighting multiple forms of inter-tumoral heterogeneity. Advanced stage and treated PDACs had higher odds of being polyclonal, whereas oligometastatic PDACs had fewer driver alterations, a lower fractional allelic loss and increased likelihood of being monoclonal. In sum, our findings reveal novel insights into the dynamic nature of the PDAC genome beyond established genetic paradigms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bec3471b710f3087aaa9089b00fcdde1b15c6db" target='_blank'>
              The Evolutionary Forest of Pancreatic Cancer.
              </a>
            </td>
          <td>
            Katelyn M Mullen, Jungeui Hong, M. Attiyeh, Akimasa Hayashi, Hitomi Sakamoto, Z. Kohutek, Caitlin A McIntyre, Haochen Zhang, Alvin P. Makohon-Moore, Amanda Zucker, L. D. Wood, Matthew A. Myers, Brian J. Arnold, S. Zaccaria, Joanne F Chou, M. Capanu, N. Socci, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-10-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="


 Glioma stem cells (GSCs) are believed to underpin treatment resistance in glioblastoma by virtue of their self- renewal capabilities and presumed transcriptional plasticity. Understanding how cell state diversity is established and maintained is crucial to make therapeutic gains. We have previously shown mitochondrial membrane potential (MMP) to be a functionally relevant source of extrinsic noise in embryonic and haematopoietic stem cells. This study evaluates cellular MMP as an axis of variability in GSCs.



 MMP was measured by ﬂow cytometry in patient-derived GSC lines using cationic dyes that diffuse reversibly across the mitochondrial membrane in a voltage dependent manner. MMP-high and low populations (top and bottom 10-20%) were fractionated and subjected to RNA-sequencing and assays of proliferation and clonogenicity. Correlation between MMP and global transcription rate was investigated through a uridine analogue (5-EU) incorporation assay. The relationship between MMP and quiescence was assessed, with quiescence deﬁned as inducible histone2B-GFP label retention after 10 days. Timecourse data for MMP was obtained through sequential staining.



 RNA sequencing identiﬁed differential gene expression and exon utilization according to MMP status. MMP- high cells upregulated cell cycle genes, while MMP-low cells exhibited a diverse signature suggestive of a more differentiated state. A greater proportion of MMP-high cells were in cycle, but MMP-low cells formed more colonies. MMP positively correlated with global transcription rate in both cycling and quiescent GSCs. Time- course experiments revealed dynamic periodicity in cellular MMP.



 The MMP axis reveals transcriptional and functional heterogeneity in GSCs and importantly unmasks phenotypic variability within the treatment-refractory quiescent compartment. The positive correlation between MMP and transcription rate, along with splicing differences, suggests a mechanism underlying functional diversity. Given the oscillatory dynamics of MMP, we postulate that stable-low MMP may deﬁne a deeply quiescent, therapy-resistant state. Evaluating MMP as a tractable regulator of GSC fate is warranted.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40ebe042eeab2b2615496bc618026a728a541e4e" target='_blank'>
              MITOCHONDRIAL MEMBRANE POTENTIAL IS A FUNCTIONALLY RELEVANT AXIS OF NON-GENETIC CELLULAR HETEROGENEITY IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Dr Ruth Robinson, Ms Emily Gaizley, Dr Elitza Deltcheva, Dr Sam Marguerat, Dr Jason Wray, Prof Tariq Enver
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Limited information exists about the types of immune cells present in Kaposi sarcoma (KS) lesions, especially in KS in the gastrointestinal tract. Using previously reported RNA-sequencing results from Kaposi sarcoma lesions in skin and gastrointestinal tract with normal matched tissues from the same patients at the same time, we investigated changes in lymphocytes in these tissues. We employed a computational method that determines changes in cell type distributions using KS lesion transcriptome data compared to a reference set of RNA expression patterns of purified immune cells. Since secreted cytokines and chemokines from KSHV-infected cells may influence the microenvironment of Kaposi sarcoma lesions, we performed cytokine profiling of conditioned media from KSHV-infected primary human dermal lymphatic endothelial cells. We also measured how this conditioned media altered the differentiation of macrophages in cell culture assays. These results suggested that factors in conditioned media from KSHV-infected endothelial cells promoted differentiation of a promonocytic cell line to proinflammatory macrophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cb74bbfbe4d9179b00243bc79251726e404bfda" target='_blank'>
              Analysis of tumor infiltrating immune cells in Kaposi sarcoma lesions discovers shifts in macrophage populations.
              </a>
            </td>
          <td>
            Takanobu Tagawa, Guruswamy Mahesh, J. Ziegelbauer
          </td>
          <td>2024-10-31</td>
          <td>Global health & medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is the fourth leading cause of cancer death in men and women in the U.S., with a 5-year survival rate of 13%. Within the PDAC tumor microenvironment (TME), morphologically aberrant leaky vessels are responsible for hypoxia and impaired immune response, which likely reduces the efficacy of cancer therapies. We and others showed that pericyte (PC) coverage significantly correlates with vascular integrity, function, and intratumor hypoxia. Therefore, understanding the contribution of abnormal PC to the TME evolution is a critical step toward PDAC treatment options. Consequently, we hypothesize that correcting such undesirable phenotype via vascular normalization will enhance treatment efficacy. Our study revealed that tumor-associated pericytes across all PDAC tumor tissues exhibited ectopic alpha-smooth muscle actin (αSMA) cytoskeletal protein expression up to 10 times higher than normal pericytes, which was significantly correlated with vascular leakiness and hypoxia. Our previous publication showed that PDAC-derived extracellular vesicles are potent inducers of αSMA expression in pericytes characterized by biomechanical abnormalities. To evaluate the vascular functional defect, we performed a tube formation assay showing abnormal pericytes contribute to abnormal vasculatures in vitro. Our investigation aims to uncover the cause of the abnormal pericyte phenotype that disrupts the physical adhesion between pericytes and endothelial cells (EC), resulting in compromised vasculature. Our findings reveal consistently upregulated monocarboxylate transporters in vascular cells. To enhance EC-pericyte adhesion, we investigated lactate metabolic communication, which is tightly linked due to the proximity of ECs and pericytes. We have shown that in PDAC, MCT1 lactate exporters are upregulated in EC. Conversely, MCT12 lactate importers are highly expressed in pericytes. The high amount of lactate produced by ECs decreases the pH in the basement membrane. So, we knocked down MCT1 transporters in ECs to increase basement pH, leading to improved PC-EC attachment and pericyte homeostasis. Finally, we performed single-cell RNA sequencing to determine the pathological signature of tumor pericytes and identify potential target molecules for vascular normalization. In conclusion, our study indicates that tumor-associated pericytes undergo phenotype switching and disturbed metabolic communication under the influence of pancreatic cancer cells. These aberrant pericytes might contribute to non-optimal vascular integrity and function. Single-cell RNA sequencing helped understand the molecular signature of the PDAC pericyte, providing a powerful tool to develop novel targeting strategies for vascular normalization. Future work includes exploring the vascular normalization approach by suppressing pericyte phenotype switching to enhance vascular function and chemo- and immunotherapeutic efficacy.
 Citation Format: Vikneshwari Natarajan, Sangdeuk Ha, Jiha Kim. Aberrant pericytes in PDAC: effects on endothelial-pericyte adhesion, vascular integrity, and tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df6c1e264f9ed7f7a7d5fa987227d27f64ca0926" target='_blank'>
              Abstract B074: Aberrant pericytes in PDAC: effects on endothelial-pericyte adhesion, vascular integrity, and tumor microenvironment
              </a>
            </td>
          <td>
            V. Natarajan, Sangdeuk Ha, Jiha Kim
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Macrophages are essential components of the immune system’s response to tumors, engaging in intricate interactions shaped by factors such as tumor type, progression, and the surrounding microenvironment. These dynamic relationships between macrophages and cancer cells have become a focal point of research, as scientists seek innovative ways to harness the immune system, including macrophages, for cancer immunotherapy. In this study, we introduce a novel model that examines the interaction between tumor and macrophage cells. We provide an in-depth analysis of the equilibrium points and their stability, as well as a thorough investigation into the solution properties of the model. Moreover, by incorporating a stochastic approach, we account for inherent randomness and fluctuations within the system, offering a more comprehensive understanding of tumor-immune dynamics. Numerical simulations further validate the model, providing key insights into how stochastic elements may influence tumor progression and immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3388123f46efe1a467aec31f0d783d8f7573a13c" target='_blank'>
              A stochastic approach to tumor modeling incorporating macrophages
              </a>
            </td>
          <td>
            Sümeyra Uçar, Ilknur Koca, N. Özdemir, Tarık İnci
          </td>
          <td>2024-10-31</td>
          <td>Bulletin of Biomathematics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Introduction The mutated in colorectal cancer (MCC) gene was initially identified as a candidate tumor suppressor gene in colorectal cancer, acting as a negative regulator of cell cycle progression. However, its functional roles in brain tumors, particularly glioblastoma, remain largely unexplored. This study reveals a significant association between MCC status and glioblastoma. Methods We explored MCC expression in the glioblastoma database, patient samples, and cell lines. We investigated the proliferation and migration of the cell lines in MCC gene knockdown using small interfering RNA. Results In vitro analyses revealed elevated protein and mRNA levels of MCC in several glioblastoma cell lines (U118MG and T98G). Silencing MCC expression via siRNA-mediated knockdown resulted in increased proliferation and migration of these cell lines. Supporting these findings, analyses of The Cancer Genome Atlas (TCGA), Chinese Glioma Genome Atlas (CGGA), and Genotype-Tissue Expression (GTEx) databases confirmed higher MCC expression in glioblastoma tumors than in normal brain tissue. Importantly, we observed that high MCC expression was associated with poor prognosis in glioblastoma patients, highlighting its potential role in disease progression. Additionally, this study identifies a nuclear localization of MCC in the glioblastoma cell line. Discussion These findings indicate that MCC expression is significantly upregulated in glioblastoma and may play a role in its pathophysiology, warranting further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1756f336eb718e69a73fb3b12fe17a3f5407527e" target='_blank'>
              The mutated in colorectal cancer (MCC) gene can serve as a potential biomarker of glioblastoma
              </a>
            </td>
          <td>
            Huonggiang Nguyen, Qingzhi Huang, Uijin Juang, Suhwan Gwon, Woohyeong Jung, Soohyeon Lee, Beomwoo Lee, So-Hee Kwon, In Soo Kim, Jongsun Park, Seon-Hwan Kim
          </td>
          <td>2024-10-08</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90840562892bea01041a4d1992fe26fa50d57b5" target='_blank'>
              Single cell analysis revealed SFRP2 cancer associated fibroblasts drive tumorigenesis in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Qiwei Wang, Yinan Zhao, Guolin Tan, J. Ai
          </td>
          <td>2024-10-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 19],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>